
The p53 pathway: positive and negative feedback loops

Sandra L Harris¹ and Arnold J Levine¹,*

¹The Cancer Institute of New Jersey and the Institute for Advanced Study, New Jersey, NJ, USA

The p53 pathway responds to stresses that can disrupt the fidelity of DNA replication and cell division. A stress signal is transmitted to the p53 protein by post-translational modifications. This results in the activation of the p53 protein as a transcription factor that initiates a program of cell cycle arrest, cellular senescence or apoptosis. The transcriptional network of p53-responsive genes produces proteins that interact with a large number of other signal transduction pathways in the cell and a number of positive and negative autoregulatory feedback loops act upon the p53 response. There are at least seven negative and three positive feedback loops described here, and of these, six act through the MDM-2 protein to regulate p53 activity. The p53 circuit communicates with the Wnt-beta-catenin, IGF-1-AKT, Rb-E2F, p38 MAP kinase, cyclin-cdk, p14/19 ARF pathways and the cyclin G-PP2A, and p73 gene products. There are at least three different ubiquitin ligases that can regulate p53 in an autoregulatory manner: MDM-2, Cop-1 and Pirh-2. The meaning of this redundancy and the relative activity of each of these feedback loops in different cell types or stages of development remains to be elucidated. The interconnections between signal transduction pathways will play a central role in our understanding of cancer.

Oncogene (2005) 24, 2899–2908. doi:10.1038/sj.onc.1208615

Keywords: p53; feedback loops; autoregulatory

the clone of cells with these defects eliminated from the body. In addition to these responses to cellular stress at the single-cell level, the p53 pathway in a cell communicates with neighboring cells by secreting a series of proteins that may alter the cellular environment. P53-regulated and p53-secreted proteins may alter the extracellular matrix and influence angiogenic signals in a localized region of a tissue. After DNA damage in a cell, the p53 pathway produces a set of proteins that can aid directly in DNA repair processes. Finally, the activation of the p53 protein and its network of genes sets in motion an elaborate process of autoregulatory-positive or autoregulatory-negative feedback loops, which connect the p53 pathway to other signal transduction pathways in the cell, and through this broader communication permits the completion or the reversal of the p53 programmed responses to stress.

Nature of the stress signals that activate p53 and its pathway

Stresses, both intrinsic and extrinsic to the cell, can act upon the p53 pathway. Among the signals that activate the p53 protein is damage to the integrity of the DNA template. Gamma or UV irradiation, alkylation of bases, depurination of DNA or reaction with oxidative free radicals all alter the DNA in different ways, and for each damaging agent, a different detection and repair mechanism is employed by the cell (for reviews see Giaccia and Kastan, 1998; Gudkov and Komarova, 2003; Oren, 2003). In each case the proteins that detect and repair the DNA lesion contain enzyme activities that communicate to the p53 protein that the DNA is damaged. This is accomplished by post-translational modifications resulting in phosphorylation, acetylation, methylation, ubiquitination or sumolation of the p53 protein (Figure 1) (Appella and Anderson, 2001). It appears that different types of DNA damage activate different enzyme activities that modify the p53 protein at different amino-acid residues, and so the nature of the stress signal is transmitted to the protein, and presumably its activity, by a code inherent to the post-translational modifications that reflect the different types of stress (Colman et al., 2000). For example, gamma-radiation activates the ATM kinase and the CHK-2 kinase, both of which can phosphorylate the p53 protein, while UV-radiation activates ATR, CHK-1 and

Functions of the p53 pathway

The p53 pathway is composed of a network of genes and their products that are targeted to respond to a variety of intrinsic and extrinsic stress signals that impact upon cellular homeostatic mechanisms that monitor DNA replication, chromosome segregation and cell division (Vogelstein et al., 2000). In response to a stress signal, the p53 protein is activated in a specific manner by post-translational modifications, and this leads to either cell cycle arrest, a program that induces cell senescence or cellular apoptosis (Jin and Levine, 2001). In this way, a variety of intrinsic or extrinsic stresses that would result in a loss of fidelity in DNA replication, genome stability, cell cycle progression or faithful chromosome segregation can be accommodated or, alternatively,

*Correspondence: AJ Levine; E-mail: alevine@ias.edu

Feedback loops of p53 pathway  
SL Harris and AJ Levine  

Activation of p53  

Stress Signals  
DNA Damage  
Hypoxia  
Ribosome biogenesis  
rNTP Depletion  
Spindle Damage  
Heat or Cold shock  
Nitric Oxide (NO)  
Oncogene Activation  

Mediators  
Kinases  
ATM  
ATR  
CHK2  
CHK1  

Coactivators with Acetyltransferase Activity  
CBP  
p300  
PCAF  
TRAF  

Corepressors with Deacetylase Activity  
HDAC1/mSin3  
HDAC1/MTA  

Other p53 Activity Modulators  
PML  
SUMO-1  
HMG1  
WRN  

Core Regulation  
MDM2  
p53  
p14ARF  
E2F-1  

Figure 1 Diversity of cancer-related signals that activate p53 contributes to the central role the p53 protein as a tumor suppressor. See text for details  

casein kinase-2, which results in the modification of different amino-acid residues on the p53 protein (Appella and Anderson, 2001). Phosphorylation at the serine and threonine residues and acetylation, methylation, ubiquitination or sumolation at the epsilon amino groups of lysines, mostly at the carboxy-terminal domain of the p53 protein, produce a combinatorial set of modifications that may well be specific to the type of stress acting upon the cell. In this way, the location on the protein and the chemical nature of the protein modification on a single protein, p53, may integrate the collection of stress signals that the cell must cope with in its life cycle. Other types of stress signals also result in different patterns of p53 protein modifications. Ribonucleotide triphosphate pool sizes are monitored and reported to the p53 protein as well as the synthesis and biogenesis of ribosomes in the nucleolus. Very low ribonucleotide triphosphate levels or too few ribosomes to sustain the cell cycle are reported to p53 and cell cycle progression is halted. Hypoxic conditions activate the p53 protein and lead to cell cycle arrest, apoptosis or senescence. Both heat-and cold-shock conditions, which result in denatured proteins and RNA aggregation, activate the p53 pathway. Spindle poisons, which block chromosome segregation, also activate the p53 protein. Inflammation in tissues and its associated nitric oxide signaling can activate the p53 response. For most of these stress signals, the p53 protein is modified by phosphorylation and acetylation, but the nature of the protein kinases and histone acetylases that carry out these post-translational changes in response to these latter stress signals remains unclear (Vogelstein et al., 2000). These protein modifications appear to alter the p53 protein in two ways: first the half-life of the protein  

in a cell increases, from 6–20 min to hours, and this results in a 3–10-fold increased concentration of the p53 protein in a cell. Second the ability of the p53 protein to bind to specific DNA sequences and promote the transcription of genes regulated by those DNA sequences is enhanced. Collectively, these two changes are the definition of the activation of the p53 protein. The different types of stress that are responded to by the p53 protein have one thing in common: they all have the potential to disrupt the efficient and faithful duplication of the cell, resulting in an enhanced mutation rate or aneuploidy during cell division. Indeed gene amplifications, deletions and aneuploidy are commonly observed and have been correlated with mutations in the p53 gene (Overholtzer et al., 2003). Thus, the absence of the p53 gene or functional protein predisposes the organism to develop cancers at a young age (Malkin et al., 1990). When cells are removed from an animal and placed in culture, these cells will divide a limited number of times, with the cells showing a senescent phenotype as cell division stops. Cell division is limited by the length of the telomere and by the absence of telomerase to restore the proper length to the chromosome end. The ATM kinase appears to sense this problem and signals to p53, which then initiates a program of cellular senescence that stops cell division (termed M1). Inactivation of p53 by mutation or an oncogene like the SV40 T-antigen reverses this block to cell division and permits several more divisions, which then results in cells with very short telomeres, chromosomal abnormalities and massive cell death (termed M2). Any cell that survives these events has many mutations, is aneuploid and is transformed and immortalized (reviewed in Vaziri, 1997).

One of the most intriguing methods of activating the p53 protein in a cell results from the mutational inactivation of some tumor suppressor genes including retinoblastoma (Rb), and adenomatous polyposis coli (APC) or the mutational activation of some oncogenes such as ras and myc. Here, a mutation or alteration in a selected protein that signals inappropriately for entry into the cell cycle is detected by the p53 checkpoint and this cell is usually killed. This is an excellent example of the tumor suppressor phenotype of the p53 protein. This phenotype is mediated by a positive feedback loop in the p53 pathway that connects the p53 node to other signal transduction pathways that regulate cell division (Lowe and Sherr, 2003). Thus, the p53 gene appears to play a dual role in preventing cancers from arising in a cell population by halting progression of the cell cycle when a stressful event would enhance the error rate, and if a mutation should occur in a gene that regulates the cell cycle, the p53 protein activates the apoptotic pathway in that cell.

The activation of the p53 protein in response to stresses is mediated and regulated by protein kinases, histone acetyl-transferases, methylases, ubiquitin and sumo ligases. As the p53 protein is activated by these protein modifications, it can also be inactivated by phosphatases, histone deacetylases, ubiquitinases or even inhibitors of ubiquitin ligases. In addition, the activated p53 protein appears to interact with a number of proteins that are important for its transcriptional activity such as PML bodies (promyelocytic leukemia bodies) (Louria-Hayon et al., 2003; Zhu

Feedback loops of p53 pathway
SL Harris and AJ Levine

et al., 2003) and the Werner helicase (Blander et al., 2000) (Figure 1). Several of these associated proteins have been shown to be essential for p53 transcriptional activities or even the selective modulation of genes that p53 may regulate in its pathway to carry out its functions.

Downstream events of the p53 pathway

Once the p53 protein is activated, it initiates a transcriptional program that reflects the nature of the stress signal, the protein modifications and proteins associated with the p53 protein. The activated p53 protein binds to a specific DNA sequence, termed the p53-responsive element (RE), composed of RRRCWWGYYY (spacer of 0–21 nucleotides) RRRCWWGYYY, where R is a purine, W is A or T, and Y is a pyrimidine. Thus, two degenerate 10bp sequences separated in the genome by a variable length spacer are required to regulate the p53-responsive genes. These p53 REs have been found both 5′ to a gene and in the first or second introns of a gene. An algorithm that identifies p53-responsive genes in the human and mouse genome has been utilized to detect a number of new genes regulated by the p53 protein (Hoh et al., 2002). The genes in this p53 network initiate one of three programs that result in cell cycle arrest (G-1 or G-2 blocks are observed), cellular senescence or apoptosis (Figure 2).

---

**Downstream Events of p53**

![Diagram](#)

**Core Regulation**

- **G1-S**
  - Cyclin E
  - Cdk2
  - p21
  - Cdc25C
  - 14-3-3-σ
  - Reprimo
  - Cyclin B
  - Gadd45
  - B99
  - Cdc2

- **G2-M**

**Cell Cycle Arrest**

**Apoptosis**

- MDM2
- p14ARF
- p53
- E2F-1
- Fas
- PIDD
- Killer/DR5
- Casp 8
- p53AIP
- Noxa
- PIGs
- PUMA
- Bid
- Cyto C
- Apaf-1
- Casp9
- Casp 3

**Inhibition of Angiogenesis and Metastasis**

- IGF-BP3
- Scotin
- PAG608
- PERP
- Bax
- Siah
- PAI
- BAI-1
- p48
- KAI
- GD-AiF
- p53R2
- TSP1
- Maspin

**DNA Repair**

---

**Figure 2** Downstream targets of the p53 transcription factor mediate its different biological outcomes. See text for details.

Feedback loops of p53 pathway
SL Harris and AJ Levine

A major player in the p53-mediated G-1 arrest is the p21 gene product that inhibits cyclin E-cdk2. This cyclin-dependent kinase acts upon the Rb protein to derepress the E2F1 activity that promotes the transcription of genes involved in preparing a cell to progress from G-1 to S phase in the cell cycle. The p53-induced G-2 arrest is mediated in part by the synthesis of 14-3-3 sigma, a protein that binds to CDC25C and keeps it in the cell cytoplasm. CDC25C is a phosphatase that acts upon cyclin B-CDC2, a kinase that is essential for the G-2-to M-phase transition. Keeping CDC25C in the cytoplasm prevents it from activating cyclin B-CDC2 in the nucleus and these cells are blocked in the G-2 phase of the cell cycle (Figure 2). The role of the p53 pathway in checkpoint control has been recently reviewed (Iliakis et al., 2003). Little or nothing is presently understood about the genes p53 must regulate to bring about senescence of cells. Experimentally senescence has been studied, as normal cells taken from an in vivo setting and cultured in vitro undergo multiple divisions in the absence of telomerase activity. The shorter chromosome ends trigger (via the ATM kinase) the activation of p53, which then activates the program for cellular senescence. More recently, a second way of activating the p53-mediated program of cellular senescence has been described. The introduction of an activated ras oncogene into primary cells in culture mediates a p53-dependent senescence. The activation of p53 by a number of different oncogenes (E2F-1, beta-catenin, myc, ras and adenovirus E1A.) is mediated in part by a positive feedback loop that results in the transcription of p19 or p14ARF (Bates et al., 1998; de Stanchina et al., 1998; Palmero et al., 1998; Stott et al., 1998; Zindy et al., 1998), which in turn inhibits the HDM-2 ubiquitin ligase that is responsible for inactivating p53 (Honda and Yasuda, 1999), resulting in an increased level of p53 in the cell (Figure 3). While the p53-regulated genes that bring about senescence are less well characterized (Nakamura, 2004), by contrast a large number of genes directly regulated by p53 are known that contribute to the apoptosis of cells. Several p53-regulated genes enhance the secretion of cytochrome c into the cytoplasm from the mitochondria (bax, noxa, puma) often in a tissue-specific fashion. Cytochrome c interacts with APAF-1 (a p53-regulated gene) to initiate a protease cascade, leading to the activation of caspase 9 and then caspase-3 followed by apoptosis. This is the intrinsic apoptotic pathway that is initiated by a number of stress signals that activate the p53 pathway. In addition to the intrinsic pathway, p53 regulates a series of genes that initiate the extrinsic apoptotic pathway (Fas ligand, killer Dr receptor), resulting in the caspase-8 and -3 activities and apoptosis. Mouse gene knockout experiments with many of these individual genes on the p53 apoptotic pathway (Figure 2) have demonstrated a significant redundancy in this cell death program. Just which of these three pathways, cell cycle arrest, senescence or apoptosis, may be chosen for the fate of a cell appears to also depend upon the nature of the stress signal, the types and location of the protein modifications on the p53 protein, those proteins associated with p53 and even the pattern of gene expression occurring in the specific tissue undergoing a p53 response. In murine 3T3 cells with a temperature-sensitive p53 gene product, a shift in temperature from 39 to 32°C activates p53 and results in a G-1 cell cycle arrest. If these 3T3 cells express high levels of E2F-1 or Myc, then a shift to 32°C results in apoptosis (Wu and Levine, 1994). In some cell types, the addition of a cytokine, such as IL-6, can reverse a p53-mediated apoptosis demonstrating that the interaction of the p53 pathway with other signal transduction pathways can alter the p53 response (Lotem et al., 2003).

In addition to genes regulated by p53 that give rise to cell cycle arrest, senescence or apoptosis, the p53 protein regulates a set of genes that produce secreted proteins including PAI, maspin and thrombospondin. These secreted proteins may be employed to communicate signals to surrounding cells informing them of a stress response (Komarova et al., 1998), altering the extracellular matrix and even regulating angiogenic signals to endothelial cells (Nishizaki et al., 1999). In addition, those p53-activated cells undergoing apoptosis might well sensitize adjacent cells for an enhanced programmed cell death or an altered gene expression. Several experiments have indicated that p53-activated cells provide a 'bystander effect' to the surrounding cells in a tissue or cell culture dish as reviewed in Mothersill and Seymour (2004). Another set of p53-regulated genes, for example, p53R2 (a ribonucleotide reductase subunit), result in enhanced DNA repair and the inactivation of reactive oxidative intermediates and peroxides in the cell or in the culture media. Finally, a number of p53-regulated genes create a series of negative and positive feedback loops that ultimately fine tune the p53 activity in a cell or turn the p53 protein on or off.

Several lines of evidence have suggested that the p53 protein has an activity that permits it to interact with the mitochondria directly and promote apoptosis. It has recently been shown that in response to DNA damage, mitochondrial p53 translocation triggers a rapid apoptotic response that occurs prior to p53 target gene activation, lending support to the growing notion that p53 can play a role in apoptosis that is transcription independent (Erster et al., 2004). The prevailing notion

![Figure 3 p53/MDM-2/p19/14 ARF loop. See text for details. Arrows denote stimulatory interactions, whereas horizontal bars instead of arrowheads indicate inhibitory influences](#)

Oncogene

is that transcription-independent proapoptotic activities  
of p53 may result from the interaction between p53 and  
proteins known to be critical for the cascade, leading to  
apoptosis. Two such examples include the Fas/CD95  
death receptor that is translocated to the plasma  
membrane by p53 (Bennett *et al.*, 1998) and the p53-  
mediated translocation of Bax from the cytoplasm to the  
mitochondria, resulting in cytochrome c release (Schuler  
*et al.*, 2000). While a transcription-independent role for  
p53 has been observed repeatedly by a number of  
different research groups, it is still unclear as to how  
much this function of p53 contributes to apoptosis  
compared to the role of p53 in transcriptional activa-  
tion. The mutations that inactivate p53 function in  
cancer cells almost all appear to localize to the DNA-  
binding domain of the p53 protein and produce a  
protein that fails to transcribe p53-responsive genes.  
Thus, the cancer cell appears to select for the loss of  
transcription factor activity. When fragments of the p53  
protein inactive for transcription are overexpressed in  
HeLa cells in culture, they can induce an apoptosis  
(Haupt *et al.*, 1997). Whether this is a consequence of an  
overexpression activity and has no physiological sig-  
nificance or this means that p53 even without its  
transcription factor activity can act to kill cells remains  
a debatable issue in the literature. Similarly, p53 mutant  
proteins have been shown by many research groups to  
have a ‘gain-of-function’ phenotype (Dittmer *et al.*,  
1993; Blandino *et al.*, 1999; de Vries *et al.*, 2002),  
resulting in enhanced tumorigenic potential, enhanced  
drug resistance and even allele-specific phenotypes. P53  
missense mutant proteins exhibit altered transcriptional  
activities, suggesting a mechanism for these phenotypes  
(Kern *et al.*, 1991; Shaulian *et al.*, 1992; Epstein *et al.*,  
1998; Bullock *et al.*, 2000). Tumors with certain gain-of-  
function mutations in p53 are resistant to chemotherapy  
compared to cells that lack p53. Recent studies have  
implicated the p53 homologues p63 and p73, known to  
be capable of inducing at least some p53 transcriptional  
target genes (Jost *et al.*, 1997; Kaghad *et al.*, 1997; Yang  
*et al.*, 1998), to be important players in this phenomenon  
(Gaiddon *et al.*, 2001). Specific p53 mutants are able to  
bind to and inhibit p73, thus reducing p73-dependent  
apoptosis, and the ability of the mutant p53 protein to  
interact with p73 is influenced by the polymorphism at  
amino acid 72 in p53. Mutant p53 expressing the Arg72  
allele has an enhanced ability to inhibit p73 compared to  
the mutant p53 Pro72 allele (Marin *et al.*, 2000). This  
observation has been extended to a clinical correlation  
in head and neck cancers. P53 mutants associated with  
less favorable response to chemotherapy are those that  
most efficiently inhibit p73 *in vitro*. Additionally,  
response was less favorable when the p53 mutation  
occurred in the 72Arg allele as opposed to the 72Pro  
form. Thus, both the p53 mutation and the polymorph-  
ism were shown to influence progression-free overall  
survival in head and neck cancer (Bergamaschi *et al.*,  
2003). It remains to be seen if this correlation will hold  
for other types of cancer, but it is intriguing in that it  
provides a mechanism for the dominant-negative effect  
of certain p53 mutations.

---

Feedback loops of p53 pathway  
SL Harris and AJ Levine  

Positive and negative p53 feedback loops  

A variety of studies in the literature have identified 10  
positive or negative feedback loops in the p53 pathway  
(see Figures 3–10). Each of these loops creates a circuit  
composed of proteins whose activities or rates of  
synthesis are influenced by the activation of p53, and  
this in turn results in the alteration of p53 activity in a  
cell. Of these, seven are negative feedback loops that  
modulate down p53 activity (MDM-2, Cop-1, Pirh-2,  
p73 delta N, cyclin G, Wip-1 and Siah-1) and three are  
positive feedback loops (PTEN-AKT, p14/19 ARF and  
Rb) that modulate up p53 activity. All of these networks  
or circuits are autoregulatory in that they are either  
induced by p53 activity at the transcriptional level,  
transcriptionally repressed by p53 (p14/19 ARF,  

pro-apoptosis  

E2F-1  
↑  
p21  
↓  
cyclin E-cdk-2  
MDM-2  
Rb  
ATM  
DNA damage  

Figure 4 Cyclin/Cdk/Rb/MDM-2 loop. See text for details  

p53  
↑  
p38  
WIP-1  
MAPK  
Ras pathway-  
mediated  
phosphorylation  

Figure 5 Wip-1/p38 MAPK loop. See text for details

Feedback loops of p53 pathway
SL Harris and AJ Levine

Figure 6 Siah-1/beta-catenin/p14/19 ARF loop. See text for details.


p53
↓
SIAH-1
↓
β-catenin → P19/14 ARF


Figure 7 PTEN/AKT/MDM-2 loop. See text for details


PTEN
↑
PIP-3
↓
AKT → MDM-2 (nuclear)


Figure 8 Cyclin G/MDM-2 loop. See text for details


p53
→ MDM-2
↑
PP2A
cyclin G
↑
cyclin G


Figure 9 p73 delta N loop. See text for details


p73ΔN
↑
p53
→ P53 responsive genes


Figure 10 At least three ubiquitin ligases that promote p53 ubiquitination and subsequent proteasomal degradation are part of the autoregulatory feedback loops. See text for details


A p53 → MDM-2
B p53 → COP-1
C p53 → PIRH-2


of the p14/19 ARF gene is positively regulated by E2F-1 (Zhu et al., 1999) and beta-catenin (Damalas et al., 2001) and negatively regulated by p53 itself. In addition, the levels of p14/19 ARF protein are increased by Ras and Myc activities in a cell (Figure 3). The complexity of the regulation of p53 by p14/p19ARF has been recently reviewed (Lowe and Sherr, 2003). The p14/19 ARF-MDM-2 complexes are often localized in the nucleolus of the cell due to the nucleolar localization signals present within p14/p19ARF. The nucleolus is the site of ribosomal biogenesis and p14/19 ARF activity itself can alter the rate of RNA processing of the ribosomal RNA precursor into mature ribosomal subunits (Sugimoto et al., 2003). Thus, p14/19 ARF by controlling MDM-2 and p53 levels and coordinating this with ribosomal biogenesis plays an important role in cell cycle regulation. This has recently been reinforced by the demonstration that the p14/19 ARF protein can regulate Myc activity as well (and therefore cell size) (Datta et al., 2004). The MDM-2 in the nucleolus is not, however, a passive entity. The MDM-2 protein has been shown to bind specifically to three large ribosomal subunit proteins L5, L11 and L23 (Marechal et al., 1994; Lohrum et al., 2003; Zhang et al., 2003; Dai et al., 2004), and the binding of L5 (Dai and Lu, 2004) or L11 (Lohrum et al., 2003; Zhang et al., 2003) to MDM-2 lowers its ubiquitin ligase activity. In addition, the ring-finger domain of MDM-2 binds specifically to an RNA sequence found in the large ribosomal RNA subunit (Elenbaas et al., 1996). While all of these observations point to a central role for MDM-2 and p14/19ARF in the regulation of ribosome biogenesis and the cell cycle, we do not understand how these observations come together to form this regulatory loop.

The Rb protein can be found in cells in a complex with MDM-2 and p53, resulting in high p53 activity and enhanced apoptotic activity (Xiao et al., 1995). High levels of active E2F-1 not bound to Rb switches the p53 response from G-1 arrest to apoptosis. Both Rb and MDM-2 are phosphorylated and inhibited by cyclin E-cdk2 (Figure 4). When p53 is activated, it stimulates the synthesis of the p21 protein, which inhibits cyclin E-cdk2 activity, and this in turn acts upon the Rb-MDM-2 complex that promotes p53 activity and apoptosis. After DNA damage, both the MDM-2 protein and the p53 protein are modified by the ATM protein kinase (Figure 4). This enhances p53 activity in the same way

Figure 3) or are regulated by p53-induced proteins. Six of these feedback loops act through MDM-2 (MDM-2, cyclin G, Siah-1, p14/19 ARF, AKT and Rb) to modulate p53 activity.

An exciting finding is that the p53 pathway is intimately linked to other signal transduction pathways that play a significant role in the origins of cancer. One of the first connections studied involves p14/p19ARF and MDM-2. The p14/19 ARF protein binds to the MDM-2 protein and modulates down its ubiquitin ligase activity, increasing the levels of the p53 protein (Honda and Yasuda, 1999) (Figure 3). The transcription

that the p53-MDM-2-Rb complex increases p53 function and is proapoptotic. For a detailed recent review of the p53-Rb-E2F1 axis see Yamasaki (2003).

Part of the activation of the p53 protein involves the phosphorylation of the p53 protein at serines located at residues 33 and 46 by the p38 MAP kinase (Figure 5). This p38 MAP kinase is itself activated by phosphorylation (regulated by the Ras-Raf-Mek-Erk pathway) that can be reversed or inactivated by the Wip-1 phosphatase. Wip-1 is a p53-responsive or p53-regulated gene forming a negative autoregulatory loop and connecting the p53 and Ras pathways (Takekawa et al., 2000) (Figure 5). An activated p53 protein positively regulates the transcription of the ubiquitin ligase Siah-1 (Fiucci et al., 2004), which in turn acts to degrade the beta-catenin protein (Iwai et al., 2004) (Figure 6). Beta-catenin levels can regulate the p14/19 ARF gene, which in turn negatively regulates MDM-2 and results in higher p53 levels (a positive feedback loop) (Figure 6). Siah-1 thus connects the Wnt-beta-catenin-APC pathway to the p53 pathway. In some cell types, the p53 protein induces the transcription of the PTEN gene (Figure 7). The PTEN protein is a PIP-3 phosphatase. PIP-3 activates the AKT kinase, which has a number of antiapoptotic protein substrates including the MDM-2 protein. Phosphorylation results in the translocation of MDM-2 into the nucleus where it inactivates p53 (Figure 7). This connects the p53 pathway with the IGF-1-AKT pathway and forms a positive feedback loop for enhanced p53 activity and decreased AKT activity. This loop in p53 regulation has also recently been reviewed (Gottlieb et al., 2002). These positive and negative feedback loops accomplish two things: (1) they modulate p53 activity in the cell and (2) they coordinate p53 activity with other signal transduction pathways that regulate entry of the cell into the cell cycle (Rb-E2F-1, myc, Ras, beta-catenin, IGF-1 and cyclin E-cdk2 activities).

There are two additional p53 autoregulatory circuits that negatively feedback upon p53 function. One of the most active of the p53-responsive genes is the cyclin G gene. It is rapidly transcribed to high levels after p53 activation in a wide variety of cell types (Okamoto and Beach, 1994; Zauberman et al., 1995; Bates et al., 1996; Yardley et al., 1998). The cyclin G protein makes a complex with the PP2A phosphatase, which removes a phosphate residue from MDM-2 (Okamoto et al., 2002) (Figure 8), which is added to the MDM-2 protein by a cdk kinase (Zhang and Prives, 2001) (Figure 4). Phosphorylation of MDM-2 by cyclin A/cdk2 inhibits its activity, thus the cyclin G-PP2A phosphatase enhances MDM-2 activity and inhibits p53. Mice with the cyclin G gene knocked out are viable (Kimura et al., 2001), and cyclin G null mouse embryo fibroblasts have elevated p53 protein levels in the absence of stress (Okamoto et al., 2002), demonstrating that this feedback loop is operational *in vivo* and acts upon the basal levels of p53 in a cell not only the higher p53 activated levels after stress. The second negative feedback loop involves a member of the p53 family of transcription factors, which include p53, p63 and p73 that are related

Feedback loops of p53 pathway  
SL Harris and AJ Levine  

by structure and function and have evolved from a common precursor. After a stress response, the p53 gene is activated, which in turn stimulates the transcription of a particular spliced m-RNA from the p73 gene, called p73 delta N (Figure 9). This translates a p73 protein without its amino-terminal domain. All three of the p53 family of proteins have similar domain structures composed of an N-terminal transcriptional activation domain linked to a central core domain that binds to a specific DNA sequence discussed above. All three of the p53 family transcription factors recognize the same DNA sequence, even though p53, p63 and p73 are capable of initiating distinct transcriptional programs. There are, however, a large number of common genes that can be regulated by all three proteins as recently reviewed in Harms et al. (2004). Thus, when p53 activates the transcription of p73 delta N, the p73 delta N protein can bind to many of the p53-regulated genes, but the absence of a transactivation domain makes it act as a repressor or competitor of p53 transcriptional activation. In this way, a negative feedback loop is set up and p53 activity declines (Grob et al., 2001; Kartasheva et al., 2002) (Figure 9). Thus, five of these positive or negative feedback circuits (Rb, PTEN, Siah-1, Wip-1, p14/19 ARF) involve genes and proteins that are central members of other signal transduction pathways, while two (cyclin G and p73 delta N) form direct negative feedback loops.

The final negative feedback loops to be discussed come in the form of ubiquitin ligases. Surprisingly, there appears to be three different p53-ubiquitin ligase activities (MDM-2, Cop-1 and Pirh-1), each of which forms an autoregulatory loop resulting in lower p53 activity (Leng et al., 2003; Dornan et al., 2004) (Figure 10). Each gene is transcriptionally activated by p53. Just why there is this level of redundancy is at present unclear. Several possibilities are that these gene products are expressed or act optimally in different cell or tissue types or even at different stages of development. For example, the MDM-2 knockout mouse is lethal at about 6 days after fertilization just at implantation of the blastocyst. This may be triggered by the hypoxia that must occur at that stage, activating p53 in the absence of MDM-2 and causing apoptosis. Consistent with this interpretation is the observation that a p53, MDM-2 double knockout mouse is viable and is born as normal as a p53 knockout mouse (Jones et al., 1995; Montes de Oca Luna et al., 1995). This is therefore consistent with the idea that the MDM-2 protein acts without a backup ubiquitin ligase activity in the blastocyst stage, but these other proteins might permit more normal function at later stages of development. These ideas are now testable. It is also possible that one or more of these three ubiquitin ligases are involved in the maintenance of p53 levels in the nonstressed or basal state, while others act only after a stress-induced p53 is produced. The activated p53 and the stress-induced p53 proteins have very different protein modifications and the impact of this upon the activity of MDM-2, Cop-1 or Pirh-2 is at present unclear. It appears likely that each of these three

Feedback loops of p53 pathway
SL Harris and AJ Levine

ubiquitin ligases form protein complexes in the cell and the associated proteins may well differ for each of these ligases, connecting them to different regulatory circuits. At present, a great deal is known about MDM-2 and relatively little focus has been placed upon the role of Cop-1 and Pirh-2, which have only been reported in the literature in the past year or so. Additionally, very recently p53 was shown to be the substrate of yet another E3 ubiquitin ligase enzyme, topors (Rajendra *et al.*, 2004). It remains to be determined if topors is also a transcriptional target of p53, and thus should be added to the growing list of proteins contributing to autoregulatory control of the p53 pathway. The next few years of study should address these questions.

As mentioned above, many of the regulatory loops involve MDM-2, thus highlighting the central role of MDM-2 in the control of p53 activity. A genetic analysis of p53 and MDM-2 mutations that block this protein complex has identified critical amino-acid residues in each protein that are important for this binding interaction (Lin *et al.*, 1994; Freedman *et al.*, 1997). These same amino-acid residues have been shown to make these protein contacts in the crystal structure of the amino-terminus of HDM-2 (the human protein) and a peptide from the amino-terminus of p53 (Kussie *et al.*, 1996). Residues phenylalanine 19, tryptophan 23 and leucine 26 of p53 form the major contacts in the MDM-2 hydrophobic pocket. Phosphorylation of residues serine 20 and possibly serine 15 should weaken these contacts, and peptides and drugs that compete with these contacts block the p53 MDM-2 complex and promote apoptosis in cells (Klein and Vassilev, 2004). Thus, the p53-MDM-2 complex and the MDM-2 ubiquitin ligase activity have become a major drug target for some cancers. In about a third of human sarcomas and in some leukemias and glioblastomas, the HDM-2 gene has been amplified and this protein is overexpressed. The p53 gene is wild type and the p53 protein is apparently inactive, so that drugs that break the p53-HDM-2 complex should activate p53. In addition, many other cancers appear to express the HDM-2 gene product at high levels even when the HDM-2 gene is not amplified. In these types of cancers blocking HDM-2 activity or freeing p53 from this complex could well induce apoptosis selectively in the cancer cells. This could also enhance chemotherapeutic activity of some drugs that activate p53.

The p53-MDM-2 autoregulatory loop is predicted to set up an oscillator with p53 and MDM-2 levels increasing and decreasing with time and out of phase in the cell. This has been demonstrated first by measuring MDM-2 and p53 levels using Western blots of proteins from cells in culture undergoing a p53 stress response (Lev Bar-Or *et al.*, 2000). While oscillations are observed and dampen with time, this experiment averages the protein concentrations from many cells in culture that may be out of phase in their oscillations, giving rise to constructive or destructive interference. For this reason, fluorescently tagged p53 and HDM2 fusion proteins were imaged in individual cells to follow the changes in p53 and HDM-2 levels in cells undergoing a p53 stress response. The expected oscillations out of phase were observed and surprisingly the number of oscillations in a cell was proportional to the dose of radiation given to these cells (Lahav *et al.*, 2004). This suggests that p53 may measure the intensity of a stress signal in a digital manner (number of oscillations) and not in an analog manner (higher p53 concentrations). Similar oscillations have been observed in other signal transduction pathways that have negative autoregulatory loops such as the NF-κ-B pathway with the NF-κ-B and I-κ-B proteins (Scott *et al.*, 1993). These digital signals resulting from oscillations in the amount of transcription factors could well result in a pattern of periodic gene expression. However, how digital signals at the transcription factor level are translated into analog signals at the level of the amount of mRNA produced by a gene remain unclear. These experiments lead to the possibility that different numbers of oscillations, the timing or wavelength of the oscillations or the amplitude of these oscillations may impact upon the selected pattern of gene expression and the outcome (cell cycle arrest, apoptosis or senescence) of the p53 response.

Why are there so many feedback loops in the p53 pathway? There are many answers to this question. All of the mechanisms may not be active in the same cell or tissue type or at the same stages during development. Feedback loops of the type described here provide a means to connect the p53 pathway with other signal transduction pathways and coordinate the cellular signals for growth and division. Redundancies in a system can sometimes prevent errors and a backup system reduces the phenotype of mutations. On the other hand, not every one of the feedback loops shown in Figures 3–10 may stand the test of time and further experimentation. Many of these pathways have been elucidated by experiments carried out with cancer cells in culture that have mutations that alter these pathways. Even normal cells in culture or knockout mice (due to accommodation to the mutation) may not reflect all the conditions that occur in normal cells and organs *in vivo*. It is particularly difficult to prove that a specific protein kinase or phosphatase acts upon a specific substrate *in vivo* and has an outcome that can be measured quantitatively. Thus, we need continue to test and challenge the functions and pathways we believe operate in a cell. However, these constructs, shown in Figures 1–10, are useful in formulating hypotheses and testing ideas that will surely lead to novel insights into the nature of cancers and the design of drugs and agents that selectively kill cancer cells.

Acknowledgements
We thank Victor Jin and Diane DePiano for assistance with the figures. This work was supported by grants to AJL from the Breast Cancer Research Foundation and the NIH Program Project Grant #CA-87497.

References

Appella E and Anderson CW. (2001). *Eur. J. Biochem.*, 268, 2764–2772.

Bates S, Phillips AC, Clark PA, Stott F, Peters G, Ludwig RL and Vousden KH. (1998). *Nature*, 395, 124–125.

Bates S, Rowan S and Vousden KH. (1996). *Oncogene*, 13, 1103–1109.

Bennett M, Macdonald K, Chan SW, Luzio JP, Simari R and Weissberg P. (1998). *Science*, 282, 290–293.

Bergamaschi D, Gasco M, Hiller L, Sullivan A, Syed N, Trigiante G, Yulug I, Merlano M, Numico G, Comino A, Attard M, Reelfs O, Gusterson B, Bell AK, Heath V, Tavassoli M, Farrell PJ, Smith P, Lu X and Crook T. (2003). *Cancer Cell*, 3, 387–402.

Blander G, Zalle N, Leal JF, Bar-Or RL, Yu CE and Oren M. (2000). *FASEB J.*, 14, 2138–2140.

Blandino G, Levine AJ and Oren M. (1999). *Oncogene*, 18, 477–485.

Bullock AN, Henckel J and Fersht AR. (2000). *Oncogene*, 19, 1245–1256.

Colman MS, Afshari CA and Barrett JC. (2000). *Mutat. Res.*, 462, 179–188.

Dai MS and Lu H. (2004). *J. Biol. Chem.*, 279, 44475–44482.

Dai MS, Zeng SX, Jin Y, Sun XX, David L and Lu H. (2004). *Mol. Cell. Biol.*, 24, 7654–7668.

Damalas A, Kahan S, Shtutman M, Ben-Ze'ev A and Oren M. (2001). *EMBO J.*, 20, 4912–4922.

Datta A, Nag A, Pan W, Hay N, Gartel AL, Colamonici O, Mori Y and Raychaudhuri P. (2004). *J. Biol. Chem.*, 279, 36698–36707.

de Stanchina E, McCurrach ME, Zindy F, Shieh SY, Ferbeyre G, Samuelson AV, Prives C, Roussel MF, Sherr CJ and Lowe SW. (1998). *Genes Dev.*, 12, 2434–2442.

de Vries A, Flores ER, Miranda B, Hsieh HM, van Oostrom CT, Sage J and Jacks T. (2002). *Proc. Natl. Acad. Sci. USA*, 99, 2948–2953.

Dittmer D, Pati S, Zambetti G, Chu S, Teresky AK, Moore M, Finlay C and Levine AJ. (1993). *Nat. Genet.*, 4, 42–46.

Dornan D, Wertz I, Shimizu H, Arnott D, Frantz GD, Dowd P, O'Rourke K, Koeppen H and Dixit VM. (2004). *Nature*, 429, 86–92.

Elenbaas B, Dobbelstein M, Roth J, Shenk T and Levine AJ. (1996). *Mol. Med.*, 2, 439–451.

Epstein CB, Attiyeh EF, Hobson DA, Silver AL, Broach JR and Levine AJ. (1998). *Oncogene*, 16, 2115–2122.

Erster S, Mihara M, Kim RH, Petrenko O and Moll UM. (2004). *Mol. Cell. Biol.*, 24, 6728–6741.

Fiucci G, Beaucourt S, Duflaut D, Lespagnol A, Stumptner-Cuvelette P, Geant A, Buchwalter G, Tuynder M, Susini L, Lassalle JM, Wasyluk C, Wasyluk B, Oren M, Amson R and Teleman A. (2004). *Proc. Natl. Acad. Sci. USA*, 101, 3510–3515.

Freedman DA, Epstein CB, Roth JC and Levine AJ. (1997). *Mol. Med.*, 3, 248–259.

Gaiddon C, Lokshin M, Ahn J, Zhang T and Prives C. (2001). *Mol. Cell. Biol.*, 21, 1874–1887.

Giaccia AJ and Kastan MB. (1998). *Genes Dev.*, 12, 2973–2983.

Gottlieb TM, Leal JF, Seger R, Taya Y and Oren M. (2002). *Oncogene*, 21, 1299–1303.

Grob TJ, Novak U, Maisse C, Barcaroli D, Luthi AU, Pirnia F, Hugli B, Graber HU, De Laurenzi V, Fey MF, Melino G and Tobler A. (2001). *Cell Death Differ.*, 8, 1213–1223.

Gudkov AV and Komarova EA. (2003). *Nat. Rev. Cancer*, 3, 117–129.

Harms K, Nozell S and Chen X. (2004). *Cell Mol. Life Sci.*, 61, 822–842.

Haupt Y, Rowan S, Shaulian E, Kazaz A, Vousden K and Oren M. (1997). *Leukemia*, 11 (Suppl 3), 337–339.

Hoh J, Jin S, Parrado T, Edington J, Levine AJ and Ott J. (2002). *Proc. Natl. Acad. Sci. USA*, 99, 8467–8472.

Honda R and Yasuda H. (1999). *EMBO J.*, 18, 22–27.

Iliakis G, Wang Y, Guan J and Wang H. (2003). *Oncogene*, 22, 5834–5847.

Iwai A, Marusawa H, Matsuzawa SI, Fukushima T, Hijikata M, Reed JC, Shimotohno K and Chiba T. (2004). *Oncogene*, 23, 7593–7600.

Jin S and Levine AJ. (2001). *J. Cell Sci.*, 114, 4139–4140.

Jones SN, Roe AE, Donehower LA and Bradley A. (1995). *Nature*, 378, 206–208.

Jost CA, Marin MC and Kaelin Jr WG. (1997). *Nature*, 389, 191–194.

Kaghad M, Bonnet H, Yang A, Creancier L, Biscan JC, Valent A, Minty A, Chalon P, Lelias JM, Dumont X, Ferrara P, McKeon F and Caput D. (1997). *Cell*, 90, 809–819.

Kartasheva NN, Contente A, Lenz-Stoppler C, Roth J and Dobbelstein M. (2002). *Oncogene*, 21, 4715–4727.

Kern SE, Kinzler KW, Baker SJ, Nigro JM, Rotter V, Levine AJ, Friedman P, Prives C and Vogelstein B. (1991). *Oncogene*, 6, 131–136.

Kimura SH, Ikawa M, Ito A, Okabe M and Nojima H. (2001). *Oncogene*, 20, 3290–3300.

Klein C and Vassilev LT. (2004). *Br. J. Cancer*, 91, 1415–1419.

Komarova EA, Diatchenko L, Rokhlin OW, Hill JE, Wang ZJ, Krivokrysenko VI, Feinstein E and Gudkov AV. (1998). *Oncogene*, 17, 1089–1096.

Kussie PH, Gorina S, Marechal V, Elenbaas B, Moreau J, Levine AJ and Pavletich NP. (1996). *Science*, 274, 948–953.

Lahav G, Rosenfeld N, Sigal A, Geva-Zatorsky N, Levine AJ, Elowitz MB and Alon U. (2004). *Nat. Genet.*, 36, 147–150.

Leng RP, Lin Y, Ma W, Wu H, Lemmers B, Chung S, Parant JM, Lozano G, Hakem R and Benchimol S. (2003). *Cell*, 112, 779–791.

Lev Bar-Or R, Maya R, Segel LA, Alon U, Levine AJ and Oren M. (2000). *Proc. Natl. Acad. Sci. USA*, 97, 11250–11255.

Lin J, Chen J, Elenbaas B and Levine AJ. (1994). *Genes Dev.*, 8, 1235–1246.

Lohrum MA, Ludwig RL, Kubbatat MH, Hanlon M and Vousden KH. (2003). *Cancer Cell*, 3, 577–587.

Lotem J, Gal H, Kama R, Amariglio N, Rechavi G, Domany E, Sachs L and Givol D. (2003). *Proc. Natl. Acad. Sci. USA*, 100, 6718–6723.

Louria-Hayon I, Grossman T, Sionov RV, Alsheich O, Pandolfi PP and Haupt Y. (2003). *J. Biol. Chem.*, 278, 33134–33141.

Lowe SW and Sherr CJ. (2003). *Curr. Opin. Genet. Dev.*, 13, 77–83.

Malkin D, Li FP, Strong LC, Fraumeni Jr JF, Nelson CE, Kim DH, Kassel J, Gryka MA, Bischoff FZ and Tainsky MA. (1990). *Science*, 250, 1233–1238.

Marechal V, Elenbaas B, Piette J, Nicolas JC and Levine AJ. (1994). *Mol. Cell. Biol.*, 14, 7414–7420.

Marin MC, Jost CA, Brooks LA, Irwin MS, O’Nions J, Tidy JA, James N, McGregor JM, Harwood CA, Yulug IG, Vousden KH, Allday MJ, Gusterson B, Ikawa S, Hinds PW, Cook T and Kalin Jr WG. (2000). *Nat. Genet.*, 25, 47–54.

Montes de Oca Luna R, Wagner DS and Lozano G. (1995). *Nature*, 378, 203–206.

Mothersill C and Seymour CB. (2004). *Nat. Rev. Cancer*, 4, 158–164.

Nakamura Y. (2004). *Cancer Sci.*, 95, 7–11.

Nishizaki M, Fujiwara T, Tanida T, Hizuta A, Nishimori H, Tokino T, Nakamura Y, Bouvet M, Roth JA and Tanaka N. (1999). *Clin. Cancer Res.*, 5, 1015–1023.

Okamoto K and Beach D. (1994). *EMBO J.*, 13, 4816–4822.

Okamoto K, Li H, Jensen MR, Zhang T, Taya Y, Thorgeirsson SS and Prives C. (2002). *Mol. Cell*, 9, 761–771.

Oren M. (2003). *Cell Death Differ.*, 10, 431–442.

Overholtzer M, Rao PH, Favis R, Lu XY, Elowitz MB, Barany F, Ladanyi M, Gorlick R and Levine AJ. (2003). *Proc. Natl. Acad. Sci. USA*, 100, 11547–11552.

Palmero I, Pantoja C and Serrano M. (1998). *Nature*, 395, 125–126.

Rajendra R, Malegaonkar D, Pungaliya P, Marshall H, Rasheed Z, Brownell J, Liu LF, Lutzker S, Saleem A and Rubin EH. (2004). *J. Biol. Chem.*, 279, 36440–36444.

Schuler M, Bossy-Wetzel E, Goldstein JC, Fitzgerald P and Green DR. (2000). *J. Biol. Chem.*, 275, 7337–7342.

Scott ML, Fujita T, Liou HC, Nolan GP and Baltimore D. (1993). *Genes Dev.*, 7, 1266–1276.

Shaulian E, Zauberman A, Ginsberg D and Oren M. (1992). *Mol. Cell. Biol.*, 12, 5581–5592.

Stott FJ, Bates S, James MC, McConnell BB, Starborg M, Brookes S, Palmero I, Ryan K, Hara E, Vousden KH and Peters G. (1998). *EMBO J.*, 17, 5001–5014.

Sugimoto M, Kuo ML, Roussel MF and Sherr CJ. (2003). *Mol. Cell*, 11, 415–424.

Takekawa M, Adachi M, Nakahata A, Nakayama I, Itoh F, Tsukuda H, Taya Y and Imai K. (2000). *EMBO J.*, 19, 6517–6526.

Vaziri H. (1997). *Biochemistry (Mosc.)*, 62, 1306–1310.

Vogelstein B, Lane D and Levine AJ. (2000). *Nature*, 408, 307–310.

Wu X and Levine AJ. (1994). *Proc. Natl. Acad. Sci. USA*, 91, 3602–3606.

Xiao ZX, Chen J, Levine AJ, Modjtahedi N, Xing J, Sellers WR and Livingston DM. (1995). *Nature*, 375, 694–698.

Yamasaki L. (2003). *Cancer Treat. Res.*, 115, 209–239.

Yang A, Kaghad M, Wang Y, Gillett E, Fleming MD, Dotsch V, Andrews NC, Caput D and McKeon F. (1998). *Mol. Cell*, 2, 305–316.

Yardley G, Zauberman A, Oren M and Jackson P. (1998). *FEBS Lett.*, 430, 171–175.

Zauberman A, Lupo A and Oren M. (1995). *Oncogene*, 10, 2361–2366.

Zhang T and Prives C. (2001). *J. Biol. Chem.*, 276, 29702–29710.

Zhang Y, Wolf GW, Bhat K, Jin A, Allio T, Burkhart WA and Xiong Y. (2003). *Mol. Cell. Biol.*, 23, 8902–8912.

Zhu H, Wu L and Maki CG. (2003). *J. Biol. Chem.*, 278, 49286–49292.

Zhu JW, DeRyckere D, Li FX, Wan YY and DeGregori J. (1999). *Cell Growth Differ.*, 10, 829–838.

Zindy F, Eischen CM, Randle DH, Kamijo T, Cleveland JL, Sherr CJ and Roussel MF. (1998). *Genes Dev.*, 12, 2424–2433.
