
APPLIED AND ENVIRONMENTAL MICROBIOLOGY, May 2005, p. 2539–2547
0099-2240/05/$08.00+0 doi:10.1128/AEM.71.5.2539–2547.2005
Copyright © 2005, American Society for Microbiology. All Rights Reserved.

# Production of the Potent Antibacterial Polyketide Erythromycin C in Escherichia coli

Salvador Peirú,${}^{1,2}$ Hugo G. Menzella,${}^{1,2}$ Eduardo Rodríguez,${}^{2}$ John Carney,${}^{2}$ and Hugo Gramajo${}^{1,2*}$

Microbiology Division, IBR (Instituto de Biología Molecular y Celular de Rosario), Consejo Nacional de Investigaciones Científicas y Técnicas, Facultad de Ciencias Bioquímicas y Farmacéuticas, Universidad Nacional de Rosario, Suipacha 531, (S2002LRK) Rosario, Argentina,${}^{1}$ and Kosan Biosciences, Inc., 3832 Bay Center Place, Hayward, California 94545${}^{2}$

Received 13 August 2004/Accepted 8 December 2004

An *Escherichia coli* strain capable of producing the potent antibiotic erythromycin C (Ery C) was developed by expressing 17 new heterologous genes in a 6-deoxyerythronolide B (6dEB) producer strain. The megalomicin gene cluster was used as the source for the construction of two artificial operons that contained the genes encoding the deoxysugar biosynthetic and tailoring enzymes necessary to convert 6dEB to Ery C. The reconstructed mycarose operon contained the seven genes coding for the enzymes that convert glucose-1-phosphate (G-1-P) to TDP-L-mycarose, a 6dEB mycarosyl transferase, and a 6dEB 6-hydroxylase. The activity of the pathway was confirmed by demonstrating conversion of exogenous 6dEB to 3-O-α-mycarosylerythronolide B (MEB). The reconstructed desosamine operon contained the six genes necessary to convert TDP-4-keto-6-deoxyglucose, an intermediate formed in the mycarose pathway, to TDP-d-desosamine, a desosamine transferase, a 6dEB 12-hydroxylase, and the rRNA methyltransferase ErmE; the last was required to confer resistance to the host cell upon production of mature macrolide antibiotics. The activity of this pathway was demonstrated by conversion of MEB to Ery C. When the mycarose and desosamine operons were expressed in an *E. coli* strain engineered to synthesize 6dEB, Ery C and Ery D were produced. The successful production of Ery C in *E. coli* shows the potentiality of this model microorganism to synthesize novel 6-deoxysugars and to produce bioactive glycosylated compounds and also establishes the basis for the future use of *E. coli* both in the production of new glycosylated polyketides and for the generation of novel bioactive compounds through combinatorial biosynthesis.

Polyketides are a large family of structurally diverse natural products that are produced by bacteria, fungi, sponges, insects, and plants. They possess a broad range of pharmacological properties that have gained widespread use in human medicine (as antibacterial, antifungal, and anticancer agents and immunosuppressants), veterinary medicine (as antibiotics and anthelmintics), and agriculture (as insecticides). The enormous structural diversity of these compounds is dictated by polyketide synthases (PKS) through a number of programmed events that involve the selection of starter and extender units, carbon chain length, and degree of reduction (13). This natural diversity is expanded through post-PKS biosynthetic steps, such as acylation, alkylation, oxidation, and glycosylation. Many naturally occurring polyketides contain unusual sugars as essential components of the bioactive molecule, examples being the antibiotics erythromycin, oleandomycin, and megalomicin, the antifungal agent nystatin, the antiparasitic agent avermectin, and several anticancer compounds (e.g., doxorubicin and aclacinomycins). All of these polyketides contain deoxysugars attached in a glycosidic linkage to the aglycone core. These sugar components generally participate in the molecular recognition of the cellular target by the bioactive compound, and their presence is usually essential for bioactivity (33).

The need for new antimicrobial drugs is constant due to the inevitable development of resistance that follows the introduction of antibiotics to the clinics (30). Naturally occurring metabolites continue to be an important source for lead compounds in the discovery of novel therapeutic agents for infectious diseases and other illnesses (3). The low success rate during the last decades in finding novel compounds with useful antibiotic activity has moved scientists towards combinatorial biosynthesis in order to discover new natural products for drug development. The modular organization of the type I PKS has allowed the engineering of novel compounds by genetic modification of the catalytic domains that determine the choice or stereochemistry of extender units or degree of reduction at various points in the biosynthesis of the polyketide core (12, 18, 23). Due to the limited genetic tools available in most polyketide-producing organisms, engineering of the PKS genes in more genetically manageable heterologous hosts is preferred.

A significant achievement in this field has been the production of the 6-deoxyerythronolide B (6dEB) aglycone in an engineered strain of *Escherichia coli* (21). This was followed by the successful biosynthesis of yersiniabactin (22), a polyketide-nonribosomal peptide hybrid, and an ansamycin polyketide precursor (31). These results provide promise for the use of *E. coli* as a platform, not only for the expression of new clusters of PKS genes but also to greatly enhance the rate at which combinatorial biosynthesis tools are developed and PKS are engineered.

Although advances have been made in the biosynthesis of PKS in *E. coli*, the challenge of expressing the tailoring en-

2540 PEIRÚ ET AL.

APPL. ENVIRON. MICROBIOL.

![Megalomicin A](https://i.imgur.com/1.png)

![Erythromycin C](https://i.imgur.com/2.png)

FIG. 1. Structures of megalomicin A and erythromycin C.

zymes involved in the modification of the polyketide cores in order to produce fully active compounds remains. Megalomicin is a 6-O-megosaminyl derivative of erythromycin C (Ery C) (Fig. 1). The proposed biosynthetic pathway of erythromycin and its relationship to that of megalomicin is identical through the production of Ery C (29), the last common intermediate in the biosynthesis of erythromycin A (Ery A) and megalomicin A. In order to demonstrate the feasibility of employing *E. coli* to produce bioactive erythromycin analogs, and with the assumption that the enzymes involved in all the common steps of the erythromycin and megalomicin biosynthetic pathways would be similar, we chose the megalomicin (*meg*) gene cluster from *Micromonospora megalomicea* (Fig. 2a) as the source of the tailoring genes needed for the biosynthesis of Ery C. Here we describe the expression of 23 foreign genes in *E. coli* that enable the production of 6dEB and its subsequent conversion to the erythromycin analogs Ery C and Ery D. Although these end products currently represent ca. 12% of the total polyketide formed, the system provides a needed proof of principle that mature erythromycin analogs can be produced in a single heterologous expression system and provides the foundation for further yield improvement by process science and metabolic engineering.

### MATERIALS AND METHODS

**Bacterial strains, plasmids, and growth conditions.** The relevant plasmids and strains used throughout this study are described in Table 1. Luria-Bertani (LB) medium was used for growth of bacteria. The following antibiotics were added to the media when necessary: kanamycin (50 μg/ml), carbenicillin (100 μg/ml), chloramphenicol (20 μg/ml), streptomycin (25 μg/ml), and phleomycin (50 μg/ml).

**DNA manipulation.** DNA restriction and modifying enzymes were used as recommended by the manufacturer (New England Biolabs). Standard protocols were used for recombinant DNA techniques (Sambrook). DNA fragments were purified from agarose gels with the QIAquick purification kit (QIAGEN). Plasmids were prepared using a QIAprep spin miniprep kit (QIAGEN). Deep Vent DNA polymerase was used in all PCRs according to the supplier's instructions (New England Biolabs).

**Plasmid constructions.** Each gene of the mycarose and desosamine biosynthetic pathways, the genes for their specific glycosyltransferases (the *megBV* and *megCIII* genes, respectively) and the genes for the P450 C-6 and C-12 hydroxylases (*megF* and *megK*, respectively) were individually amplified by PCR from *M. megalomicea* genomic DNA and sequenced to confirm that they were free of errors. The 5′ primers (Table 2) were designed to have an NdeI site overlapping the translational initiation codon, changing GTG start codons to ATG when necessary. The 3′ primers (Table 2) contained EcoRI and adjacent SpeI sites downstream from the stop codon. PCRs were performed using a DNA thermal cycler 480 (Perkin-Elmer) with the cycling parameters described in Table 2. The

PCR products were digested with NdeI and EcoRI and cloned into identical sites of either pET24b or pET28a vectors in order to express proteins with their natural N terminus or as His-tag fusions, respectively. pKOS431-39-1 was derived from pRSF-1b (Novagen) to accept XbaI/EcoRI fragments as follows. (i) The XbaI site at nucleotide 2254 was eliminated by cleavage, filling-in, and blunt end ligation; (ii) the MluI/BlpI fragment of the polylinker was replaced with that from pET26b (S. Mutka, personal communication).

pKOS431-54-2 was derived from pCDF-1b (Novagen) to modify the antibiotic resistance and to accept XbaI/EcoRI fragments as follows. (i) The XbaI site at nucleotide 2206 was eliminated by cleavage, filling-in, and blunt end ligation; (ii) the resulting plasmid was digested with BspHI/DrdI, blunted with T4 DNA polymerase, and dephosphorylated with shrimp alkaline phosphatase in order to replace the streptomycin resistance gene by ligation to a blunted SpeI/HindIII bleomycin resistance cassette from pKOS183-112 (B. Julien, personal communication); (iii) the MluI/BlpI fragment of the polylinker was replaced with that from pET26b (S. Mutka, personal communication).

The general strategy for the construction of the mycarose and desosamine operons was carried out as follows. The XbaI/EcoRI fragment from the pET-derived construct harboring the second gene of the operon was cloned into the SpeI/EcoRI site of the pET-derived plasmid already containing the first gene of the final construction. This resulted in an intermediate vector harboring the first and second genes separated by a 43-bp sequence with an appropriately positioned ribosome-binding site (RBS). The process was repeated in a recursive fashion until the entire operon containing all genes in tandem was constructed. The mycarose operon contained in pET28a (pKOS452-25b) was removed from this plasmid as an XbaI/EcoRI fragment and cloned into pKOS431-39.1 to give pKOS452-53e. The desosamine operon harbored in pET24b (pKOS452-8b) was also removed as an XbaI/EcoRI fragment and cloned into pKOS431-54.2 to give pKOS452-54. The final operons are shown in Fig. 2b.

**Gene expression.** To analyze expression of the individual sugar biosynthetic genes cloned in pET24b or pET28a, cultures of transformed BL21(DE3) were grown overnight in tubes containing LB medium and appropriate antibiotics. Cultures were diluted 1:50 and grown at 37°C to mid-log phase (0.4 to 0.6 OD<sub>600</sub> unit/ml). Gene expression was induced with 0.5 mM IPTG (isopropyl β-D-thiogalactopyranoside), and cultures were grown for 6 h at the same temperature. Total cell extracts were analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) or Western blotting as described below.

For expression of the mycarose operon, BL21(DE3)/pKOS452-53e cultures were grown at 37°C to mid-log phase (0.4 to 0.6 OD<sub>600</sub> unit/ml), induced with 0.5 mM IPTG, and grown for 36 h at 22°C. Volumes of each culture equivalent to 3 OD<sub>600</sub> units were resuspended in 1 ml of 20 mM Tris, 150 mM NaCl, pH 7.5, containing 1 tablet of complete EDTA-free protease inhibitor cocktail (Roche) per 50 ml. Cells were disrupted by sonication and centrifuged at 14,000 rpm for 15 min. Pellets were resuspended in 0.5 ml lysis buffer, and soluble and insoluble fractions equivalent to a cell suspension with an OD<sub>600</sub> of 0.03 were analyzed by Western blotting.

**Protein analysis.** Cell extracts of induced cultures were analyzed by denaturing PAGE (SDS-PAGE) (14a) on Novex 12% Tris-glycine gels (Invitrogen) and visualized by staining with Coomassie blue. Western blotting was carried out as described by Nikolau et al. (20) by transferring proteins to an Invitrolon polyvinylidene difluoride membrane (Invitrogen). His-tagged proteins were identified using a monoclonal anti-His<sub>5</sub> antibody (QIAGEN) as a primary antibody and horseradish peroxidase-conjugated goat anti-mouse antibody as a secondary

Vol. 71, 2005

ERYTHROMYCIN BIOSYNTHESIS IN E. COLI 2541

a

megR megK megCV megClV megBVI
2K 4K 6K 8K 10K 12K 14K 16K 18K

megDI megDII megDIV megDVI megBV

Cosmid KOS079-138B

megAl megAlll

5K 8K 12K 16K 20K 24K 28K 32K 36K 40K 44K 48K 52K 56K 60K

Cosmid KOS205.57-2.3B

megCII megCIII megBII megH megF megBIII megM megL orf1
51K 52K 53K 54K 55K 56K 57K 58K 59K 60K

b

pKOS452-53e

megDIV* megBII* megM megL* megF megBVI* megBIIV megBV* megBIII*
0.7 1.4 2.1 2.8 3.5 4.2 4.9 5.6 6.3 7.0 7.7 8.4 9.1 9.8

pKOS452-54

megDII megCIII* megClV* megCI megCV* megDIII* megK* ermE
0.7 1.4 2.1 2.8 3.5 4.2 4.9 5.6 6.3 7.0 7.7 8.4 9.1

FIG. 2. Organization of the megalomicin gene cluster. (a) The nucleotide sequences of the left and right hand sides of the meg gene cluster were obtained from cosmids pKOS079-138B (29) and pKOS205.57-2.3B (McDaniel, personal communication). (b) Genetic organization of the mycarose and the desosamine operons. Synthetic operons constructed for expression of the mycarose (pKOS452-53e) and desosamine (pKOS452-54) pathways were cloned under the transcriptional control of the T7 promoter. Asterisks indicate genes with a 5′ translational fusion to a His₆-encoding DNA sequence.

antibody, and the reaction was visualized by enhanced chemiluminescence with ECL reagents (Amersham Pharmacia Biotech).

Bioconversion experiments. E. coli BL21(DE3) with or without the chaperone expression plasmids and each of the sugar pathway plasmids, pKOS452-53e (mycarose) and pKOS452-54 (desosamine), was cultured overnight at 37°C in LB medium with appropriate antibiotics, subcultured by a 1:50 dilution in the same medium, and grown to mid-log phase (0.4 to 0.6 OD₆₀₀ unit/ml). Chaperones and sugar gene expression were induced by addition of 2 mg/ml L-arabinose or 1 μg/ml tetracycline and 0.5 mM IPTG, respectively, and cultures were supplemented with 50 μg/ml 6dEB or 3-α-mycarosyl erythronolide B (MEB) as needed. Cultures were grown at 22°C for 2 or 5 days, their pH values were adjusted to 9, and the cultures were then extracted with an equal volume of ethyl acetate. The organic layer was removed, dried under vacuum, dissolved in 5 mM NH₄ acetate in 1:1 methanol-H₂O, and analyzed by liquid chromatography/mass spectrometry (LC/MS) as described below.

Bioconversion of 6dEB to Ery C was studied in BL21(DE3) containing pKOS452-53e (mycarose) and pKOS452-54 (desosamine) in the presence of

pGro7 as a source of the GroEL/GroES chaperone complex. The induction, feeding conditions, and product purification and analysis were carried out as described above.

Erythromycin production. E. coli K207-3 harboring pBP130, pKOS207-129, pKOS452-53e, pKOS452-54, and the chaperone expression plasmid pGro7 was cultured overnight at 37°C in LB medium with the appropriate antibiotics and then subcultured by a 1:50 dilution to mid-log phase (0.4 to 0.6 OD₆₀₀ unit/ml). Chaperones, deoxyerythronolide synthase, and sugar gene expression were induced by addition of 0.5 mM IPTG and 2 mg/ml arabinose; cultures were also supplemented with 50 mM glutamate, 50 mM succinate, and 5 mM propionate and then incubated for 5 days at 22°C. The cultures were processed as described for bioconversion experiments.

Polyketide analysis. Extracts were analyzed using a system consisting of a Gilson 215 sample handler, an Agilent 1100 high-performance liquid chromatography pump, and an Applied Biosystems Mariner time-of-flight mass spectrometer in a positive-ion mode configured with a Turbo-ionspray source. Samples (20 μl) were chromatographed on a Phenomenex Develosil ODS-

TABLE 1. Relevant plasmids and strains used in this study

| Plasmid or strain | Description or relevant genotype$^a$ | Source or reference |
|-------------------|-------------------------------------|---------------------|
| **Plasmids**      |                                     |                     |
| pET24b            | $E. coli$ expression vector, ColEl ori, kan | Novagen             |
| pET28a            | $E. coli$ expression vector, ColEl ori, kan | Novagen             |
| pKOS 431-54.2     | $E. coli$ expression vector, CloDF13ori, bleo | S. Mutka, personal communication |
| pKOS 431-39.1     | $E. coli$ expression vector, RSF1030ori, kan | S. Mutka, personal communication |
| pKOS 342-96       | $ermE$ in a pET28a background, kan | S. Ward, personal communication |
| pKOS 452-53e      | $megDIV$, $megBII$, $megM$, $megL$, $megF$, $megBVI$, $megBV$, and $megBIII$ in a pKOS 431-39.1 background, kan | This work |
| pKOS 452-54       | $megDII$, $megCIII$, $megCIV$, $megCII$, $megCV$, $megDIII$, $megK$ and $ermE$ in a pKOS 431-54.2 background, bleo | This work |
| pBP130            | $debs2$ and $debs3$ in a pET21b background, bla | 21                  |
| pKOS207-129       | $debs1$ in a pRSF1010 background, str | 19                  |
| pGro7             | $P_{BAD} groES-groEL$ in a pACYC 184 background, cat | Takara             |
| **E. coli strains** |                                     |                     |
| DH5$\alpha$       | $lacZ\Delta M15 recA1$ | Promega              |
| BL21 (DE3)        | $ompT hsdS(r_B^- m_B^-) gal$ (DE3) | Novagen             |
| K207-3            | $F^- ompT hsdS(r_B^- m_B^-) gal dcm$ (DE3) $panD::panDS2SA\Delta prpRBCD::T7 prom-sfp T7$ $prom-prpE ygfG::T7 prom-accA1-T7 prom-pccB$ | 19                  |

$^a$ Abbreviations: $kan$, kanamycin resistance gene; $bleo$, bleomycin resistance gene; $bla$, ampicillin resistance gene; $str$, streptomycin resistance gene; $cat$, chloramphenicol acetyltransferase gene.

TABLE 2. Oligonucleotides used in this study and conditions used for PCR amplification

| Gene    | Oligonucleotides$^a$                                                                                          | Annealing temp ($^\circ$C)$^b$ |
|---------|----------------------------------------------------------------------------------------------------------------|--------------------------------|
| megDIV  | Upper, 5'-CCGGGCATATGAGGGTCGAGGAGGCTG-3' <br> Lower, 5'-GGAATTCACTAGTCCGGGGTACGTCCGC-3'                           | 58                             |
| megBII  | Upper, 5'-CCGT CATATGAGCACCGACGCCAC-3' <br> Lower, 5'-AGGAATTCACTAGTGCGGGGCTCTCACC GTAG-3'                         | 56                             |
| desIII  | Upper, 5'-GGGGTCATATGAAGGCGCTTGTCTCTGCGG-3' <br> Lower, 5'-GGAATTCTTGTGACTAGTCGAGTAGTC-3'                          | 58                             |
| desIV   | Upper, 5'-GACCTCCA TATGACGACTCGACTCCTGGTC-3' <br> Lower, 5'-TGAATTCACTAGTCCCTCACACCATCGCCC G-3'                      | 54                             |
| megF    | Upper, 5'-TGGTCATATGAAA CTGCCCGATCTGGAGAG-3' <br> Lower, 5'-CGAATTCACTAGTCTCATCCGTT CGGTCGCA-3'                    | 58                             |
| megBVI  | Upper, 5'-GGCATATG GGGGATCGGGTCAACGGTCATG-3' <br> Lower, 5'-GGAATTCACTAGTTTCACGCCGTC GCCGGTTGAG-3'                 | 58                             |
| megBIV  | Upper, 5'-GCATATGACA AGACATGTCACACTTCTCGG-3' <br> Lower, 5'-CGAATTCACTAGTGTCACCTTGGT CGAGATGA-3'                   | 58                             |
| megBV   | Upper, 5'-TGTACATATGCGGGTCCTGCTCACCTCG-3' <br> Lower, 5'-AGAATTCACTAGTCACCTGT CGGGCGGGTGCTG-3'                | 58                             |
| megBIII | Upper, 5'-CAGCATATGCCC GAAACGAGATGCCG-3' <br> Lower, 5'-AGAATTCACTAGTTCATCAC ACCACTTCCAGG-3'                  | 56                             |
| megDII  | Upper, 5'-CACATATGACC ACTACGTCTGGT CCTATC-3' <br> Lower, 5'-CGAATTCACTAGTCACAGCCC GGTGATGACCTCC-3'               | 58                             |
| megCIII | Upper, 5'-CTCATATGCGCGGTCTTCTCCTCCATGGC-3' <br> Lower, 5'-TGAATTCACTAGTCATCCGACGGCGGTCCTCCC-3'                      | 58                             |
| megCIV  | Upper, 5'-GACCATATGAAGCGCGTACCGACC-3' <br> Lower, 5'-GGAATTCACTAGTCACGAACC GTTGC GTTTC-3'                     | 58                             |
| megCII  | Upper, 5'-AGAGCATATGAACACGACC GATCGCG-3' <br> Lower, 5'-CGGAATTCACTAGTCAGAGTT CGACC G-3'                        | 52                             |
| megCV   | Upper, 5'-CGCATATGAAGACC ACTCCTACGGCGACC G-3' <br> Lower, 5'-TCGAATTCACTAGTT CAGACGGCGG GATCAGGC-3'             | 58                             |
| megDIII | Upper, 5'-CCGCATATGCCGAACAGCCACTCGACCACG-3' <br> Lower, 5'-CGAATTCACTAGTCGACCCCTCACCGCCCCGGCTCC-3'                   | 54                             |
| megK    | Upper, 5'-GGAGACATATGACC ACTATCGAACAGATCC-3' <br> Lower, 5'-TCGAATTCACTAGTGGTTCAGGCGGACTCG-3'                       | 58                             |

$a$ The engineered restriction sites are shown in italics.

$b$ The genes were amplified under the following PCR conditions: 30 cycles of 30s at 94°C, 30s of annealing at the temperature indicated in the table, and 60s at 68°C.

TABLE 3. Deduced functions of all the auxiliary genes identified in the megalomicin gene cluster involved in the biosynthesis of Ery C

| Gene       | Closest protein match | % Sim$^a$ | Proposed pathway$^b$     | Proposed function                | Reference |
|------------|-----------------------|-----------|--------------------------|----------------------------------|-----------|
| megF       | EryF                  | 91        |                          | C-6 hydroxylase                 | 32        |
| megK       | EryK                  | 90        |                          | C-12 hydroxylase                | 25        |
| megL       | MtmD                  | 73        | Myc/Meg/Des              | TDP-D-glucose synthase          | 16        |
| megM       | GilE                  | 73        | Myc/Meg/Des              | TDP-glucose dehydratase         | 4         |
| megBVI     | EryBVI                | 86        | Myc/Meg                  | 2,3-Dehydratase                 | 27        |
| megBII     | EryBII                | 72        | Myc/Meg                  | 2,3-Reductase                   | 27        |
| megDIV     | EryBVII               | 68        | Myc/Meg                  | 5-Epimerase                     | 27        |
| megBIII    | EryBIII               | 89        | Myc                      | 3-C-Methyltransferase           | 27        |
| megBIV     | EryBIV                | 80        | Myc                      | 4-Ketoreductase                 | 27        |
| megBV      | EryBV                 | 85        | Myc                      | Glycosyltransferase             | 27        |
| megCII     | EryCII                | 78        | Des                      | 3,4-Isomerase                   | 27        |
| megCIV     | EryCIV                | 92        | Des                      | 2,3-Dehydratase                 | 27        |
| megCV      | EryCV                 | 87        | Des                      | 3,4-Enoylreductase              | 6         |
| megDIII    | EryCVI                | 65        | Des/Meg                  | Dimethyltransferase             | 27        |
| megCIII    | EryCIII               | 90        | Des                      | Glycosyltransferase             | 27        |
| megDII     | Med-ORF 20            | 83        | Des/Meg                  | Aminotransferase                | 9         |

$^a$ Sim, similarity. Determined by BLASTX using default parameters.

$^b$ Myc, mycarose; Meg, megosamine; Des, desosamine.

Vol. 71, 2005                                                                 ERYTHROMYCIN BIOSYNTHESIS IN *E. coli*  2545

![Diagram of Erythromycin C biosynthesis pathway]

FIG. 3. Erythromycin C biosynthesis pathway. All of the heterologous proteins present in *E. coli* K207/pBP130/pKOS207-129/pKOS452-53e/ pKOS452-54 that are involved in the biosynthesis of Ery C are shown. The common intermediates of the L-mycarose and D-desosamine pathways are indicated.

synthesis of TDP-rhamnose and dTDP-4-acetamido-4,6-dideoxygalactose, respectively (17, 26).

Production of Ery C in *E. coli* from its primary metabolites. To demonstrate the feasibility of producing Ery C without the requirement of feeding specific intermediates of the pathway, the 6dEB producer strain *E. coli* K207-3/pBP130/pKOS207-129 (19) was transformed with the expression plasmids pKOS452-53e, pKOS452-54, and pGro7. This new strain contains 23 heterologous genes (approximately 0.5% of the protein genes present in the chromosome of *E. coli*) all under the transcriptional control of T7 RNA polymerase (Fig. 3). LC/MS analysis of extracts obtained from 5-day cultures demonstrated production of the two erythromycins analogs Ery C (0.4 mg/liter) and Ery D (0.5 mg/liter) (Fig. 4). The sample also contained 6dEB (7 mg/liter), EB (15 mg/liter), MEB (0.14 mg/liter), and 0.28 mg/liter of the analog with one less methyl group in the neutral sugar.

Interestingly, we did not find a 3-desmethyl-Ery C derivative that should result from the desosamination of the desmethyl mycarosyl-EB compound. However, a 3-desmethyl-Ery A has been reported in *eryBIII* mutants of *S. erythraea* (7).

### DISCUSSION

In this study we have succeeded for the first time in the complete biosynthesis and export of a potent macrolide antibiotic in a genetically modified strain of *E. coli*. Although this microorganism presents a new environment for PKS expression (there are no PKS found in *E. coli*) and polyketide biosynthesis, we have demonstrated the utility and promise of the biosynthetic capabilities of this bacterium for producing glycosylated polyketides.

The development of new erythromycin analogs with altered aglycones (5, 10, 14) has been greatly facilitated by heterologous expression of the deoxyerythronolide synthase in *Streptomyces coelicolor* (11) and, more recently, in a metabolically engineered strain of *E. coli* (21). The aglycones thus obtained are purified and subsequently modified into promising new

2546 PEIRÚ ET AL.

![Figure](https://i.imgur.com/1234567.png)

FIG. 4. Analysis of erythromycin C produced by *E. coli*. (A and B) LC/MS-extracted ion chromatogram (*m/z* 720.5 ± 0.2) of the erythromycin C standard (A) and of an extract from a fermentation broth of *E. coli* K207/pBP130, pKOS207-129, pKOS452-53e, pKOS452-54, and pGro7 (B). (C and D) LC/MS of the erythromycin C standard (C) and of the peak corresponding to erythromycin C in the extract from a fermentation broth of *E. coli* K207/pBP130, pKOS207-129, pKOS452-53e, pKOS452-54, and pGro7 (D).

bioactive compounds by an erythromycin-producing strain of *S. erythraea* containing the necessary tailoring enzymes but in which the erythromycin PKS had been rendered inactive (2). Although the two-fermentation system is highly refined, it would clearly be advantageous to produce mature erythromycin analogs in a single organism. While the final titer for Ery C in our system is not yet useful for broader application, improved production of this compound can be approached by rational optimization of putative rate-limiting steps or by the empirical processes of titer development.

Interestingly, the distribution of products in the full producer strain does not correlate with that observed in the feeding experiments using the mycarose or the desosamine pathways and suggests the presence of differential amounts of substrates or enzyme activities. In other work, we have observed unpredictable differential expression of genes when multiple T7 promoters were used, presumably due to competition for systems needed in transcription/translation (unpublished results). The producer strain contains five different plasmids and probably hundreds of copies of the strong φ10 promoter from T7 phage, which might produce a heavy burden on the metabolism of the microorganism after its induction. We also demonstrated that *E. coli* provided the electron donor system, probably by the ferredoxin (flavodoxin) NADP⁺ oxidoreductase encoded by *fpr* (1), which is needed for the P450 hydroxylases MegF and MegK. However, the complete conversion of 6dEB to EB and the accumulation of Ery D in the final producing strain might indicate that the electron donor system is optimal for MegF but not for MegK. Expression of a more specific redox pair (ferredoxin/ferredoxin reductase) for MegK might also be necessary to optimize this production system.

Changing the sugar moieties in natural products might be an important contribution to the generation of novel derivatives with potentially novel pharmacological properties. Recently, a plasmid containing the genes involved in the biosynthesis of L-oleandrose was developed in *Streptomyces antibioticus* (24). By replacement of *oleU* with different 4-ketoreductase genes, and by using the “sugar-flexible” glycosyltransferase ElmGT, Rodriguez et al. were able to produce novel glycosylated compounds. Although this example is an important achievement towards the generation of combinatorial libraries, the restricted genetic tools available for *Streptomyces* and the relatively low growth rate of these bacteria limit the system. The idea of converting *E. coli* into a model system for generating combinatorial libraries, where the biosynthesis of different PKS aglycones could be combined with the expression of different sugar pathways and glycosyltransferases of relaxed substrate specificity, has gained growing interest. The results presented here represent a step towards the development of such a model system.

ACKNOWLEDGMENTS

We thank Chau Tran for assistance with the MS, Daniel Kurth for the construction of some of the intermediate plasmids, and Mark Burlingame for facilitating production of the 6dEB and MEB. We also thank Dan Santi, David Hopwood, Sarah Kodumal, Gary Ashley, and Leonard Katz for their comments on the manuscript.

REFERENCES

1. Bianchi, V., P. Reichard, R. Eliasson, E. Pontis, M. Krook, H. Jörnvall, and E. Haggard-Ljungquist. 1993. *Escherichia coli* ferredoxin NADP⁺ reductase: activation of *E. coli* anaerobic ribonucleotide reduction, cloning of the gene (*fpr*), and overexpression of the protein. J. Bacteriol. **175**:1590–1595.
2. Carreras, C., S. Frykman, S. Ou, L. Cadapan, S. Zavala, E. Woo, T. Leaf, J. Carney, M. Burlingame, S. Patel, G. Ashley, and P. Licari. 2002. Saccharopolyspora erythraea-catalyzed bioconversion of 6-deoxyerythronolide B analogs for production of novel erythromycins. J. Biotechnol. **92**:217–228.
3. Cragg, G. M., D. J. Newman, and K. M. Snader. 1997. Natural products in drug discovery and development. J. Nat. Prod. **60**:52–60.
4. Fischer, C., F. Lipata, and J. Rohr. 2003. The complete gene cluster of the antitumor agent gilvocarcin V and its implication for the biosynthesis of the gilvocarcins. J. Am. Chem. Soc. **125**:7818–7819.
5. Frykman, S., T. Leaf, C. Carreras, and P. Licari. 2001. Precursor-directed production of erythromycin analogs by *Saccharopolyspora erythraea*. Biotechnol. Bioeng. **76**:303–310.
6. Gaisser, S., G. A. Böhm, J. Cortés, and P. F. Leadlay. 1997. Analysis of seven genes from the *eryAI-eryK* region of the erythromycin biosynthetic gene cluster in *Saccharopolyspora erythraea*. Mol. Genet. **256**:239–251.
7. Gaisser, S., G. A. Bohm, M. Doumith, M. C. Raynal, N. Dhillon, J. Cortes, and P. F. Leadlay. 1998. Analysis of *eryBI*, *eryBIII* and *eryBVII* from the erythromycin biosynthetic gene cluster in *Saccharopolyspora erythraea*. Mol. Genet. **258**:78–88.
8. Gramajo, H. C., J. White, C. R. Hutchinson, and M. J. Bibb. 1991. Overproduction and localization of components of the polyketide synthase of *Streptomyces glauscens* involved in the production of the antibiotic tetramycin C. J. Bacteriol. **173**:6475–6483.
9. Ichinose, K., M. Ozawa, K. Itou, K. Kunieda, and Y. Ebizuka. 2003. Cloning, sequencing and heterologous expression of the medermycin biosynthetic gene cluster of Streptomyces sp. AM-7161: towards comparative analysis of the benzoisochromanquinone gene clusters. Microbiology **149**:1633–1645.
10. Jacobsen, J. R., C. R. Hutchinson, D. E. Cane, and C. Khosla. 1997. Precursor-directed biosynthesis of erythromycin analogs by an engineered polyketide synthase. Science **277**:367–369.
11. Kao, C. M., L. Katz, and C. Khosla. 1994. Engineered biosynthesis of a complete macrolactone in a heterologous host. Science **265**:509–512.
12. Katz, L. 1997. Manipulation of modular polyketide synthases. Chem. Rev. **97**:2557–2576.
13. Katz, L. 2002. Polyketide diversity, p. 157–175. In W. K. Shounfeld and H. A. (ed.), Macrolide antibiotics. Birkhauser Verlag, Basel, Switzerland.
14. Kennedy, J., S. Murli, and J. T. Kealey. 2003. 6-Deoxyerythronolide B analogue production in *Escherichia coli* through metabolic pathway engineering. Biochemistry **42**:14342–14348.
14a.Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature **227**:680–685.
15. Liu, M., and S. Douthwaite. 2002. Activity of the ketolide telithromycin is refractory to Erm monomethylation of bacterial rRNA. Antimicrob. Agents Chemother. **46**:1629–1633.
16. Lombó, F., K. Siems, A. F. Brana, C. Mendez, K. Bindseil, and J. A. Salas. 1997. Cloning and insertional inactivation of *Streptomyces argillaceus* genes involved in the earliest steps of biosynthesis of the sugar moieties of the antitumor polyketide mithramycin. J. Bacteriol. **179**:3354–3357.
17. Marolda, C. L., and M. A. Valvano. 1995. Genetic analysis of the dTDP-rhamnose biosynthesis region of the *Escherichia coli* VW187 (O7:K1) *rfb* gene cluster: identification of functional homologs of *rfbB* and *rfbA* in the *rfb* cluster and correct location of the *rfbE* gene. J. Bacteriol. **177**:5539–5546.
18. McDaniel, R., A. Thamchaipenet, C. Gustafsson, H. Fu, M. Betlach, M. Betlach, and G. Ashley. 1999. Multiple genetic modifications of the erythromycin gene cluster to produce a library of novel “unnatural” natural products. Proc. Natl. Acad. Sci. USA **96**:1846–1851.
19. Murli, S., J. Kennedy, L. C. Dayem, J. R. Carney, and J. T. Kealey. 2003. Metabolic engineering of *Escherichia coli* for improved 6-deoxyerythronolide B production. J. Ind. Microbiol. Biotechnol. **30**:500–509.
20. Nikolau, B. J., E. S. Wurtele, and P. K. Stumpf. 1985. Use of streptavidin to detect biotin-containing proteins in plants. Anal. Biochem. **149**:448–453.
21. Pfeifer, B. A., S. J. Admiraal, H. Gramajo, D. E. Cane, and C. Khosla. 2001. Biosynthesis of complex polyketides in a metabolically engineered strain of *E. coli*. Science **291**:1790–1792.
22. Pfeifer, B. A., C. C. Wang, C. T. Walsh, and C. Khosla. 2003. Biosynthesis of yersiniabactin, a complex polyketide-nonribosomal peptide, using *Escherichia coli* as a heterologous host. Appl. Environ. Microbiol. **69**:6698–6702.
23. Rodriguez, E., and R. McDaniel. 2001. Combinatorial biosynthesis of antimicrobials and other natural products. Curr. Opin. Microbiol. **4**:526–534.
24. Rodriguez, L., I. Aguirrezabalaga, N. Allende, A. F. Brana, C. Mendez, and J. A. Salas. 2002. Engineering deoxysugar biosynthetic pathways from antibiotic-producing microorganisms. A tool to produce novel glycosylated bioactive compounds. Chem. Biol. **9**:721–729.
25. Stassi, D., S. Donadio, M. J. Staver, and L. Katz. 1993. Identification of a *Saccharopolyspora erythraea* gene required for the final hydroxylation step in erythromycin biosynthesis. J. Bacteriol. **175**:182–189.
26. Stevenson, G., B. Neal, D. Liu, M. Hobbs, N. H. Packer, M. Batley, J. W. Redmond, L. Lindquist, and P. Reeves. 1994. Structure of the O antigen of *Escherichia coli* K-12 and the sequence of its *rfb* gene cluster. J. Bacteriol. **176**:4144–4156.
27. Summers, R. G., S. Donadio, M. J. Staver, E. Wendt-Pienkowski, C. R. Hutchinson, and L. Katz. 1997. Sequencing and mutagenesis of genes from the erythromycin biosynthetic gene cluster of *Saccharopolyspora erythraea* that are involved in L-mycarose and D-desosamine production. Microbiology **143**:3251–3262.
28. Vester, B., and S. Douthwaite. 1994. Domain V of 23S rRNA contains all the structural elements necessary for recognition by the ErmE methyltransferase. J. Bacteriol. **176**:6999–7004.
29. Volchegursky, Y., Z. Hu, L. Katz, and R. McDaniel. 2000. Biosynthesis of the anti-parasitic agent megalomicin: transformation of erythromycin to megalomicin in *Saccharopolyspora erythraea*. Mol. Microbiol. **37**:752–762.
30. Walsh, C. 2003. Where will new antibiotics come from? Nat. Rev. Microbiol. **1**:65–70.
31. Watanabe, K., M. A. Rude, C. T. Walsh, and C. Khosla. 2003. Engineered biosynthesis of an ansamycin polyketide precursor in *Escherichia coli*. Proc. Natl. Acad. Sci. USA **100**:9774–9778.
32. Weber, J. M., J. O. Leung, S. J. Swanson, K. B. Idler, and J. B. McAlpine. 1991. An erythromycin derivative produced by targeted gene disruption in *Saccharopolyspora erythraea*. Science **252**:114–117.
33. Weymouth-Wilson, A. C. 1997. The role of carbohydrates in biologically active natural products. Nat. Prod. Rep. **14**:99–110.
