
AUTOPHAGY IN METAZOANS:  
CELL SURVIVAL IN THE LAND  
OF PLENTY  

**Julian J. Lum, Ralph J. DeBerardinis and Craig B. Thompson**

Abstract | Cells require a constant supply of macromolecular precursors and oxidizable substrates to maintain viability. Unicellular eukaryotes lack the ability to regulate nutrient concentrations in their extracellular environment. So when environmental nutrients are depleted, these organisms catabolize existing cytoplasmic components to support ATP production to maintain survival, a process known as autophagy. By contrast, the environment of metazoans normally contains abundant extracellular nutrients, but a cell’s ability to take up these nutrients is controlled by growth factor signal transduction. Despite evolving the ability to maintain a constant supply of extracellular nutrients, metazoans have retained a complete set of autophagy genes. The physiological relevance of autophagy in such species is just beginning to be explored.

OXIDIZABLE SUBSTRATE  
A molecule that can be used  
in mitochondrial oxidation/  
reduction reactions to yield  
reduced electron carriers  
(NADH, FADH₂). In this  
review, we consider oxidizable  
substrates to be molecules  
that provide carbon to the  
tricarboxylic acid cycle.  

In all cells, maintaining a critical supply of intra-  
cellular metabolites is required for survival. OXIDIZABLE  
SUBSTRATES such as glucose and pyruvate sustain the  
cell’s most vital tasks, including ATP generation and  
the production of precursors for macromolecular  
synthesis, such as amino acids, glycerol and acetyl  
CoA, through biochemical reactions that occur in  
the cytoplasm and mitochondria. Maintaining these  
processes is essential, a fact that is highlighted by the  
high degree of evolutionary conservation of systems  
that regulate nutrient uptake and metabolism, and the  
consequences for cells that fail to take up sufficient  
nutrients to maintain themselves.  

Cells have at least two ways to regulate intracellular  
metabolite supply. The first involves the acquisition of  
EXTRACELLULAR NUTRIENTS. In mammalian cells, nutrient  
uptake is regulated by GROWTH FACTOR signalling path-  
ways, including the well-studied phosphatidylinositol  
3-kinase (PI3K)/Akt system. Stimulation of this system  
by insulin, insulin-like growth factors or other extracel-  
lular cues causes an increase in nutrient import, which  
provides the cell with a supply of oxidizable substrates  
to support ATP generation and biosynthesis through  

ANABOLIC METABOLISM¹⁻³. These effects allow cells to meet  
the metabolic demand of growth and proliferation,  
which are downstream effects of the pathway. The abil-  
ity of growth factor signalling to manage nutrient  
handling is particularly important in multicellular  
organisms, where tissue-specific growth and prolifera-  
tion must be regulated in response to the needs of the  
organism rather than individual cells.  

In the absence of stimuli for growth, quiescent  
cells have a lower biosynthetic demand but still need  
sufficient intracellular metabolites to support basic  
functions and maintain viability. In these situations,  
nutrient import is limited by decreased growth fac-  
tor signalling, and survival becomes dependent on  
the second mechanism by which cells maintain bio-  
energetics — the activation of CATABOLIC METABOLISM  
to degrade intracellular macromolecules. Cells have  
various catabolic pathways at their disposal, but the  
most extreme form is MACROAUTOPHAGY (herein referred  
to as autophagy), in which cytoplasmic components,  
including proteins and organelles, are sequestered  
into acidic vacuoles for bulk degradation. This proc-  
ess, which is conserved from yeast to humans, was  

Abramson Family Cancer  
Research Institute,  
Department of Cancer  
Biology, University of  
Pennsylvania School of  
Medicine, Philadelphia,  
Philadelphia 19104, USA.  
Correspondence to C.B.T.  
e-mail: craig@mail.  
med.upenn.edu  
doi:10.1038/nrm1660  
Published online 13 May 2005

govern the ability of metazoan cells to take up nutrients, and concurrently suppress autophagy. Conversely, deprivation of either growth factors or extracellular nutrients leads to a decreased intracellular metabolite supply, and the de-repression of autophagy as a means to maintain basic bioenergetic processes. This review traces the development of these ideas, paying particular attention to the key role that autophagy has in diverse aspects of cell physiology, and to the serine/threonine kinase target of RAPAMYCIN (Tor), the metabolic sensor that coordinates crosstalk among nutrient availability, growth factor signalling and autophagy. We also discuss the potential roles of autophagy in survival, metazoan development, non-apoptotic cell death and tumour suppression, both at the level of the individual cell and of the whole organism.

**Autophagy is induced by metabolic stress**

Autophagy is an evolutionarily conserved, lysosomal-mediated catabolic response pathway that is induced under nutrient-poor conditions⁷. During autophagy, cytoplasmic components, including proteins and organelles, are engulfed by double-membrane structures known as autophagosomes (FIG. 1), which fuse to lysosomal vesicles where the contents are degraded into their components (for example, from proteins to amino acids). Following degradation of the cargo, the breakdown products are released into the cytoplasm, where they presumably provide a source of metabolites to support biosynthesis and other processes. The molecular steps that are required for autophagosome formation, as studied in yeast, and that are mostly conserved in mammals, have been reviewed elsewhere⁹⁻¹².

Autophagy has now been studied in various organisms, and this work has clarified its role in responding to nutrient ‘emergencies’. In many organisms, survival during nutrient restriction requires adopting a metabolically quiescent phenotype. In *Saccharomyces cerevisiae*, deprivation of either carbon or nitrogen induces the formation of autophagosomes. Autophagosome formation has been linked to survival in nutrient-poor conditions; a yeast screen for mutants defective in survival on nitrogen-poor media led to the isolation of 15 genes that are required for autophagosome formation (*ATG1–15*)¹³. So, autophagy supports yeast survival during starvation¹³. In addition, these mutants also lacked the ability to sporulate — a state that is induced by nutrient starvation. When nutrients are re-supplied, yeast spores resume proliferation even after prolonged starvation.

Autophagy fulfills a similar role in other organisms, allowing them to adapt to and survive periods of starvation. In *Dictyostelium discoideum*, nutrient deprivation causes amoeboid cells to aggregate into a multicellular organism, which is a process that requires autophagy. Mutations in the *D. discoideum* orthologues of *ATG5*, *ATG6*, *ATG7* and *ATG8* do not affect amoeboid growth in nutrient-replete conditions, but limit differentiation and viability during nutrient deprivation¹⁴,¹⁵. In *Acer pseudoplatanus* root tip cells, sucrose deprivation triggers a cascade of responses, including growth arrest,

originally characterized as a response to nutrient deprivation. However, the regulation and consequences of autophagy have come under intense scrutiny in recent years, as a number of observations have revealed functions for autophagy in diverse processes, including cell death, cell survival, development and tumour suppression⁴⁻⁸.

Studies of the relationship between growth factor signalling and autophagy now indicate that the two processes are mechanistically linked, and that growth factors

Figure 1 | **Growth factor signalling regulates autophagy.** The mammalian homologue of ATG8, microtubule-associated protein light chain 3 (LC3), is a specific marker of autophagosome formation¹⁰⁷,¹⁰⁸. Cells that undergo autophagy redistribute LC3 from the cytoplasm to localized punctate structures. **a** | In the presence of a growth factor, for instance interleukin-3 (IL3), cells exhibit uniform cytoplasmic LC3 staining. **b** | Following growth factor withdrawal, LC3 redistributes to a punctate pattern that is characteristic of ongoing autophagy. **c,d** | Transmission electron microscopy confirmed that autophagosomes are scarce in cells cultured in the presence of growth factor but abundant in the absence of growth factor. **e,f** | Higher-power magnification photomicrographs of IL3-dependent cells deprived of IL3 show autophagosomes that contain intracellular contents. Autophagosomes are indicated by arrowheads.

decline in respiration, decreased glycolysis, global degradation of cellular proteins and accumulation of autophagosomes^{16,17}. In *Arabidopsis thaliana*, mutations in autophagy genes cause accelerated senescence and leaf yellowing (chlorosis), and limit lateral root elongation in starvation conditions^{18–20}. A study in *Caenorhabditis elegans* revealed a crucial role for autophagy in formation of the dauer diapause, a quiescent, long-term survival larval stage prompted by nutrient deprivation, increased temperature and mutations in the insulin-like signalling pathway (for example, *daf-2*). The dauer phenotype is associated with increased autophagic activity, and requires the orthologues of *ATG1*, ATG6, ATG7, ATG8 and *ATG10* (REF. 21).

In *C. elegans*, loss-of-function mutations in autophagy genes not only impair dauer formation, but also decreases adult lifespan in *daf-2* mutant animals^{21}. This raises the possibility that autophagy plays a part in survival of the whole organism in addition to survival of individual cells. Consistent with this idea, RNA interference (RNAi) knockdown of fat-body-specific Atg5 or organismal loss of Atgl decreases the survival of adult flies in low-nutrient conditions^{22}. Furthermore, mice deficient in Atg5 have a severe reduction in survival during neonatal starvation, whereas wild-type mice induce autophagy in numerous tissues after birth and survive longer periods of fasting^{23}. Such a result provides independent genetic evidence consistent with numerous reports that correlate the presence of autophagosomes with the breakdown of liver and muscle glycogen during early neonatal periods^{24–27}. Together, these experiments support the idea that autophagy is a conserved component of the response to nutrient deprivation on both a cellular and organismal level, and contributes to survival during such conditions.

Whether autophagy has a role in cell-autonomous survival in mammals has been difficult to determine, because when individual mammalian cells are subjected to either nutrient deprivation or growth factor withdrawal, they rapidly undergo apoptosis. Recently, this issue has been resolved by using cells resistant to apoptosis. Cells that lack the apoptotic proteins *Bax* and *Bak* resist various pro-apoptotic stimuli^{28–30}. In the absence of trophic factors, growth factor-responsive *bax⁻/⁻bak⁻/⁻* cells survive for weeks despite cell cycle arrest, and a progressive decline in the ability to maintain extracellular nutrient uptake and cell size. Loss of growth factor signalling in such cells induced autophagy^{31} (FIG. 1), and the interruption of autophagy by chemical inhibition or RNAi resulted in rapid cell death. Growth factor deprivation stimulated autophagy even in the presence of abundant extracellular nutrients, which implies either that growth factor withdrawal or nutrient withdrawal are sufficient to induce autophagy in metazoan cells. Decreased extracellular nutrients or growth factors correlated with decreased intracellular metabolite concentrations and stimulation of autophagy. The re-addition of growth factors to autophagic mammalian cells increased nutrient import and utilization, decreased autophagy and allowed cells to resume growth and proliferation — which is analogous to changes seen after nutrient replenishment in *C. elegans* dauer larvae or yeast spores. The survival supported by autophagy in mammalian cells was not indefinite, and after several weeks in culture, cells lost viability through a non-apoptotic mechanism that was associated with necrotic features, such as a progressive decrease in ATP levels.

The ability of growth factor signalling to suppress autophagy is consistent with previous studies that associated class-I PI3K activity with reduced autophagosome formation. Treating HT-29 cells with end products of class-I PI3K increased Akt activity and suppressed autophagic protein turnover^{32}. Furthermore, autophagy is suppressed by insulin-like growth factor activation of PI3K in *C. elegans*^{21}. By contrast, class-III PI3K activity is associated with increased autophagy activity^{32}. Moreover, beclin-1, the homologue of yeast ATG6, was identified as part of class-III PI3K complexes by immunoprecipitation^{33}, which indicates that class-III PI3K might affect crucial components of the autophagic machinery.

**Growth factors support anabolism through Akt**

The observation that growth factors suppress autophagy is consistent with their role in facilitating growth and proliferation. Because each round of replicative cell division requires a doubling of biomass, logic predicts that growth factor signalling should stimulate anabolism and suppress catabolism. A large body of evidence from cytokine-dependent haematopoietic cells supports this model. The cytokine interleukin-3 (IL3) regulates nutrient import by increasing the expression and/or surface localization of transporters for glucose, amino acids, iron and low-density lipoprotein^{34–36}. The availability of IL3 is directly proportional to several markers of metabolism, including the expression of glycolytic enzymes, glycolytic rate, lactate production and mitochondrial membrane potential^{35,37,38}. Furthermore, stimulation of glycolysis by IL3 is a primary effect rather than merely a homeostatic response to growth, because it exceeds proliferative demand and remains high even when new protein synthesis is inhibited^{38}.

Proliferation requires duplication of the protein, nucleic acid and lipid contents of a cell, which is a task that requires a substantial increase in the uptake of synthetic precursors. In addition to stimulating the uptake of glucose, amino acids and lipid, purified growth factors or serum stimulate nucleoside uptake as well as pyrimidine and purine synthesis^{39–42}. These data indicate that enhancing intracellular nucleotide pools for DNA synthesis is another metabolic effect of growth factor regulation.

The metabolic effects of IL3 are primarily due to its ability to activate PI3K and the serine/threonine kinase Akt (FIG. 2). Blockade of this pathway abrogates the effect of IL3 on glucose metabolism, whereas constitutive Akt activity supports glucose uptake, mitochondrial membrane potential and glucose-dependent ATP production, even in the absence of IL3 (REFS 43,44). These activities have beneficial effects on cell survival,

EXTRACELLULAR NUTRIENT

A resource in the environment that can be passively or actively taken up by cells and used in intermediate metabolism. These include amino acids, fatty acids, glucose, nucleotides and minerals.

GROWTH FACTOR

A small molecule that binds to lineage-specific cell-surface receptors and activates signal transduction pathways necessary for cell growth, proliferation and survival. Examples of growth factors include insulin, platelet-derived growth factor and cytokines.

ANABOLIC METABOLISM (anabolism). A collective term for metabolic pathways in which simple metabolites are used to synthesize macromolecules. Examples include the synthesis of proteins from amino acids and of lipids from acetyl CoA. Anabolism is required for cell growth, is stimulated by growth factors and generally consumes ATP.

CATABOLIC METABOLISM (catabolism). A collective term for metabolic pathways that degrade macromolecules to yield simpler constituents and/or ATP, usually during periods of nutrient deprivation. Examples include β-oxidation of fatty acids and glycogenolysis. Autophagy is considered as an arm of catabolism, because it includes protein degradation in response to nutritional stress.

MACROAUTOPHAGY (autophagy). A process whereby double-membrane structures sequester cytosolic material and fuse to a lysosome or vacuole. Material within these structures is degraded and recycled.

RAPAMYCIN (also known as sirolimus).

A peptide from the bacterium *Streptomyces hygroscopicus*, isolated from the soil on Easter Island. It is an inhibitor of Tor activity and functions as a potent immunosuppressant.

from one that supports self-maintenance to one that supports anabolic growth. The latter is dependent on glucose metabolism to maintain ATP production.

The mechanism by which constitutive Akt activation induces glucose ‘addiction’ is unclear, but might involve the Akt-dependent suppression of lipid and amino-acid degradation. In glioblastoma cells, constitutive Akt activity impairs the induction of β-oxidation, limiting the ability to generate ATP from fatty acids. This effect can be overcome by artificially stimulating AMP-activated protein kinase (AMPK), an important intracellular ‘fuel gauge’ that de-represses β-oxidation by relieving inhibition on the mitochondrial fatty-acid transporter carnitine palmitoyltransferase I (CPT1)⁴⁶. AMPK activation enhances β-oxidation and improves survival in the absence of glucose, which indicates that restoring the ability to catabolize macromolecules shunted into biosynthesis during Akt-stimulated cell growth can rescue cells from Akt-driven glucose dependence. But the intrinsic ability of cells to induce survival through AMPK activation is repressed by PI3K and Akt activation through mechanisms that are still not fully understood.

These studies emphasize the importance of coordinated regulation of anabolic and catabolic metabolism in growth factor signalling. *In vivo*, many cells are quiescent for long periods, and then enter a period of growth and/or proliferation in response to a growth factor. Whereas the survival of quiescent cells can be maintained by autophagy, β-oxidation and other catabolic pathways, growth factor stimulation leads to anabolic growth and the suppression of these survival-promoting catabolic pathways. An evolving area of current research is determining which signalling kinases are responsible for enhancing anabolism and suppressing autophagy and other forms of catabolism. The serine/threonine kinase Tor has emerged as an important factor in executing these functions.

**Control of autophagy by the nutrient sensor Tor**

Protein synthesis is a required component of growth and proliferation, and can consume a large amount of the total intracellular energy resources. The process must therefore be tightly regulated, and needs to integrate information from both nutrient abundance and growth factor signalling. This crucial function is carried out by the conserved protein Tor (and its orthologues mTOR in mammals and dTOR in *Drosophila melanogaster*), which functions downstream of PI3K and Akt in growth factor signalling⁴⁷,⁴⁸. The presence of growth factors or of abundant intracellular amino acids leads to the activation of mTOR and phosphorylation of its targets, ribosomal protein S6 kinase (S6K) and the eukaryotic initiation factor 4E binding protein-1 (4E-BP1)⁴⁹,⁵⁰. Phosphorylation of these targets facilitates CAP-DEPENDENT TRANSLATION⁴⁷,⁵¹,⁵². mTOR is therefore a component of both growth factor signalling and nutrient-sensing pathways, and induces protein synthesis when metabolic conditions (amino acid availability) are favourable and/or when stimulated by growth factors.
The molecular regulation of mTOR activity has been intensely studied and involves a number of proteins, including the tuberous sclerosis complex proteins TSC1 and TSC2, and the small GTPase Rheb (FIG. 3). Under amino-acid-rich conditions, GTP-bound Rheb activates mTOR, but during periods of amino acid depletion, the TSC1–TSC2 heterodimer blocks Rheb-mediated mTOR activation. It has been proposed that TSC2 functions as a GTPase-activating protein during low amino acid conditions, causing Rheb to remain in an inactive state bound to GDP and decreasing its ability to activate mTOR<sup>53,54</sup>.

The ability of mTOR to function as a nutrient sensor might involve responding to other variables apart from the amino acid concentration *per se*. In addition, ATP levels might modulate mTOR activity independently of amino acids, because glycolytic inhibition reduces cellular ATP and decreases mTOR-dependent phosphorylation of S6K1 and 4E-BP1 (REF. 55). Other work has uncovered an intriguing connection between mTOR and the nutrient sensor AMPK, which is activated

Metabolic stress  
Low ATP/AMP ratio  
AMPK  

![Diagram](  
Metabolic stress  
Low ATP/AMP ratio  
AMPK  
↓  
P P P  
TSC2  
TSC1  
Akt  
ATP  
Amino acids  
Glucose  
Growth factor  
→  
mTOR  
Rapamycin  
Hypoxia  
Autophagy  
)  

Figure 3 | Coordinated regulation of nutrient sensing by mTOR and AMPK. A model for how the mammalian orthologue target of rapamycin, mTOR, functions as a central sensor of cellular energy that responds to changes in the intracellular metabolite concentration. The activity of mTOR is controlled through Rheb, a GTP-binding protein, which stimulates mTOR when it is in the GTP-bound state. Rheb can be activated by abundant amino acids or inactivated by the TSC heterodimer. AMP-activated protein kinase (AMPK; a metabolic sensor that is activated by low energy states), metabolic stresses and pharmacological activators such as 5-amino-imidazole carboxamide riboside (AICAR), maintains TSC2 phosphorylation and binding to TSC1, thereby sustaining Rheb inactivation and suppressing mTOR activity. Conversely, constitutive Akt activity inhibits the TSC complex, which leads to increased Rheb function and suppression of autophagy. mTOR activation by high levels of ATP, amino acids, glucose or growth factors also suppresses autophagy, whereas rapamycin treatment or hypoxia induce autophagy.

during metabolic stress by the decreasing ratio of ATP to AMP<sup>56–59</sup> (FIG. 3). During low energy states, AMPK phosphorylates TSC2, which increases its ability to inactivate mTOR<sup>60</sup>, thereby providing one mechanistic explanation for the regulation of mTOR by TSC2.

Growth factor signalling also modulates mTOR activity through mechanisms that are independent of its effect on increasing amino acid import. Recent studies have shown that growth factor signalling activates mTOR through effects on TSC. In the unstimulated state, TSC1 and TSC2 heterodimerize and inhibit mTOR activity. In the presence of growth factors, Akt-dependent TSC2 phosphorylation disrupts the complex and relieves mTOR repression<sup>60,61</sup>. The key role of TSC2 in limiting mTOR activity is emphasized by studies with TSC2<sup>−/−</sup> cells, in which S6K and 4E-BP1 are constitutively phosphorylated.

In mammalian cells, the observation that rapamycin stimulates autophagy led to the discovery that mTOR activity also functions to suppress this pathway<sup>62</sup>, and similar results have been subsequently described in yeast<sup>63</sup>. This activity might be mediated in part by the ability of rapamycin to inhibit Tor-dependent phosphorylation of ATG13 and subsequent binding to ATG1, an organizational step that is necessary for the formation of the pre-autophagosomal membrane<sup>64,65</sup>. ATG1 and ATG13 might be regulated by SNF1, the yeast orthologue of AMPK<sup>66</sup>. Yeast that grows in stationary phase induces autophagy, but this activity is downregulated in yeast strains that lack SNF1, which indicates that in mammalian cells AMPK might have a role in modulating the autophagic response.

Tor-regulated suppression of autophagy also occurs in multicellular organisms. In *D. melanogaster*, loss-of-function dTOR mutants have constitutive autophagy in the fat body<sup>22</sup>. Furthermore, inactivation of dTOR by the removal of growth factor stimulation decreases nutrient use and allows cells to engage the autophagic survival programme. This phenotype could not be reversed by overexpression of PI3K, which indicates that dTOR is required for growth-factor-mediated autophagy suppression *in vivo*<sup>22</sup>. Together, these observations indicate a model in which Tor and its orthologues function as important players that orchestrate responses to growth factor and amino acid levels.

### How does autophagy support cell survival?

The role for autophagy as a survival pathway has been extensively described, and extensive evidence details the benefits it provides in response to various stresses (FIG. 4a). Some of these benefits might be metabolic in nature, because autophagy is induced during periods of metabolic bioenergetic stress, which is imposed either by decreased extracellular metabolite supply or by decreased intracellular metabolite concentrations that result from the loss of growth factor signalling.

But how exactly does autophagy provide a bioenergetic benefit to ‘starving’ cells? Considering the connection between amino acid starvation and increased protein degradation in autophagosomes, one probable  
REVIEW S

a

| Stress                                                                 | Benefit                                                                 |
|------------------------------------------------------------------------|-------------------------------------------------------------------------|
| Cellular damage (for example, chemotherapy)                              | Removal of damaged organelles                                           |
| Intracellular pathogens                                                | Limitation of infection                                               |
| Nutrient/growth factor deprivation                                      | Metabolic substrates for:                                              |
| Hypoxia                                                                | • Adaptive protein synthesis                                          |
|                                                                        | • Bioenergetic needs                                                  |

b

Autophagosome/lysosome

- Protein
- Membrane lipid

Fatty acids → Acetyl CoA  
Pyruvate ← Amino acids: alanine, glutamate  
Adaptive protein synthesis  

TCA cycle  
OAA  
NADH/FADH₂  
αKG  
Glutamate  
NAD⁺/FAD  
Electron transport chain  
H⁺  
ATP  

Mitochondrion  

Figure 4 | Model for the metabolic benefits of autophagy.  
a | Induction of autophagy by various stresses provides many benefits that support the survival of metazoan cells and organisms.  
b | To survive periods of reduced metabolite supply, cells require amino acids for adaptive protein synthesis and cytoplasmic ATP to fuel bioenergetic needs. Autophagy might fulfill both of these needs. Autophagosomes/lysosomes contain protein and lipid constituents to be recycled. Presumably, lipid degradation of autophagosome/lysosomal material provides fatty acids for mitochondrial β-oxidation, yielding acetyl CoA. Amino acids from degraded proteins are substrates both for new protein synthesis and for mitochondrial oxidation through the tricarboxylic acid (TCA) cycle. The efficiency of amino acid oxidation depends on both the amino acid and the tissue. Alanine and several other amino acids can be converted to pyruvate, a substrate for oxidative decarboxylation, and to acetyl CoA, which can then enter the TCA cycle. Other amino acids such as glutamate enter the TCA cycle as dicarboxylic acids, for example, α-ketoglutarate (αKG), and can be partially oxidized through TCA reactions, or converted to acetyl CoA for complete oxidation (reactions not shown). Isoleucine and leucine can be metabolized to acetyl CoA in the muscle, or their α-ketoacids can be converted to acetyl CoA in the liver. Ongoing TCA function produces electrons for the electron transport chain and, ultimately, oxidative phosphorylation to produce ATP. This model is supported by data in which methylpyruvate, a pyruvate analogue, supports viability in cells in which autophagy is inhibited. OAA, oxaloacetic acid.

benefit is that amino acids that are generated by autophagy contribute to synthesis of new proteins that are needed to adapt to starvation. However, protein synthesis decreases during periods when autophagy is active, which indicates that other metabolic mechanisms might also contribute (FIG. 4b). For example, some amino acids that are generated through autophagy might be further processed to support the tricarboxylic acid (TCA) cycle. The TCA cycle carries out two crucial functions to maintain viability in cells that have been withdrawn from nutrients or growth factors. First, it is an important source of FADH₂ and NADH for the electron transport chain and therefore helps to generate ATP. Second, the TCA cycle is a biosynthetic machine, because intermediates in the cycle can be converted into various metabolites, including cytosolic acetyl CoA, sterols, nucleotides and even glucose in certain specialized tissues. Loss of TCA cycle activity would compromise the ability of the cell to generate all of these substances. The importance of constitutive TCA activity in autophagic cells is supported by studies showing that providing cells with pyruvate, a TCA substrate, can suppress autophagy and can maintain cell viability³,¹⁷,³¹,⁶⁷.

If maintaining TCA flux is a fundamental effect of autophagy, then the turnover of other cell constituents, particularly lipids, might also be important. Acetyl CoA is an end product of both β-oxidation and glycolysis, and combines with oxaloacetate in the entry step of the TCA cycle (also refer to FIG. 4b). Failure to provide acetyl CoA will result in decreased TCA activity, even in the presence of sufficient oxaloacetate. In nutrient- or growth-factor-deprived cells, when glucose uptake and glycolysis are decreased, β-oxidation might play an increasingly important part in supplying acetyl CoA. The role of β-oxidation in autophagy has not been carefully studied, although rapamycin treatment of human BJAB B-lymphoma cells induces the expression of β-oxidation genes⁶⁸. A method to measure and track various metabolites would be of great value in clarifying the metabolic effects of autophagy and its potential synergy with other metabolic processes.

Intracellular damage activates autophagy

In addition to its role in promoting cell survival in response to nutrient deprivation, autophagy also seems to function as a waste disposal system, clearing organelles that accumulate from the cytoplasm. Whether this process occurs stochastically or involves the selective targeting of particular organelles is unknown, but a possible model is that autophagy preferentially clears aged or oxidatively damaged organelles⁶⁹–⁷² and protein aggregates associated with neurodegenerative disorders (see below). This could be particularly important for damaged mitochondria, which have a decreased ability to produce ATP and an increased production of toxic reactive oxygen species⁷⁰. In yeast, mitochondrial destruction by autophagosomes involves the outer mitochondrial membrane protein **Uth1**, and strains that are deficient in this gene have decreased mitochondrial degradation during autophagy⁷³.
Another important organelle that is targeted for turnover through autophagy are peroxisomes. These single membrane organelles function in lipid metabolism and breakdown of hydrogen peroxide and are ‘marked’ for turnover when they are in excess or damaged ${ }^{74}$. The nature of selective peroxisome degradation has been best described in *Hansenula polymorpha*. The absence of an N-terminal domain in the peroxin protein Pex14 impaired peroxisome degradation ${ }^{75}$. Whether or not similar systems are at work in mammals has not been evaluated. It is not known whether Uth1, Pex14 or any other protein helps target damaged rather than undamaged organelles. Understanding the regulation of this process is of clinical interest, because the steady accumulation of oxidized mitochondria and loss of peroxisome function are believed to contribute to many pathological conditions.

### Autophagy in human disease

The deregulation of cell growth, proliferation and apoptosis are hallmarks of tumorigenesis. A substantial body of investigation has determined a role for mTOR in promoting oncogenesis ${ }^{51,76,77}$. However, the relationship between autophagy and cancer is less obvious, because multiple lines of evidence have indicated both positive and negative roles for autophagy in tumorigenesis. Autophagy could support survival of cells within a tumour subjected to decreased nutrient availability. Conversely, mice that are haploinsufficient for *beclin-1* have decreased levels of autophagy and increased frequency of spontaneous tumours ${ }^{78,79}$. The mechanism by which *beclin-1* functions as a tumour suppressor is not known. Inactivation of autophagy could result in a state of decreased protein degradation, so that anabolic processes and growth predominate. Alternatively, *beclin-1* associates with Bcl2 proteins, and therefore *beclin-1* knockout might result in impaired apoptosis, facilitating tumorigenesis. In addition, autophagy might have other less direct roles in cancer, such as the removal of damaged mitochondria and as a protective response to chemotherapy or radiation-directed therapies ${ }^{80-83}$. Furthermore, several other oncogenes have been implicated in autophagy regulation (BOX 1), but further *in vivo* testing will be required to clarify the importance of autophagy in cancer.

Considering the conserved role of autophagy in supporting viability during periods of stress, it is interesting that in certain contexts, autophagy is a process that is associated with a form of non-apoptotic programmed cell death, known as type II programmed cell death ${ }^{83,84}$. In addition, the requirement for autophagy during embryonic development remains an uncertain issue. Mice that are homozygous for null alleles of *beclin-1* die early in embryogenesis. Moreover, autophagy is required in the fat body of flies during metamorphosis, and RNAi of *Atg1* results in lethality associated with improper development of that organ, which indicates that autophagic cell death is important in developing flies ${ }^{85}$. However, mice deficient for *Atg5* survive postnatally but die from starvation after birth. One interpretation for these differences might be the role of *beclin-1* in processes other than autophagy. An important initiative in the future study of autophagy will involve understanding how it functions as a survival pathway in certain situations and a death pathway in others (BOX 2).

---

**Box 1 | Role of oncogenes and tumour suppressors in autophagy**

The role of oncogenes and tumour suppressors in regulating autophagy has been extensively studied, yet most findings remain correlative. The oncoprotein Ras is known to function downstream of growth factor receptor tyrosine kinases. In response to nutrient deprivation, Ras suppresses transformation in a rapamycin-sensitive manner and is counteracted by a dominant-negative Ras. In addition, Ras decreases autophagic protein degradation, which indicates that Ras might prevent autophagy through class-I phosphatidylinositol 3-kinase (PI3K) ${ }^{96}$. By contrast, overexpression of Myc in rat fibroblasts had the opposite effect by promoting autophagy ${ }^{97}$. However, expression of a Myc mutant lacking box II, which has no effect on Myc-induced apoptosis or tumorigenesis, did not affect autophagy, which indicates that tumorigenesis and autophagy might be not be linked ${ }^{97}$. Autophagy has also been shown to be under the regulation of the tumour suppressor PTEN (phosphatase and tensin homologue). HT-29 cells that overexpress PTEN were able to overcome interleukin-13 suppression of autophagy ${ }^{98}$. By contrast, a constitutively active form of Akt resulted in reduced levels of autophagy. These collective data indicate opposing functions of oncogenes and tumour suppressors in modulating the autophagic response, and together they reveal the potential for therapeutic intervention of autophagy in cancers depending on their genetic lesion(s) within the class-I PI3K pathway.

More recent work has shown that *beclin-1* (the homologue of yeast Atg6) is a novel Bcl2-interacting protein ${ }^{99}$. Decreases in *beclin-1* levels have been associated with human breast carcinomas, and targeted inactivation of *beclin-1* predisposes mice to increased tumour susceptibility ${ }^{78,79}$. These mice display lower autophagic activity, prompting the idea that *beclin-1* functions as a tumour suppressor by promoting autophagic clearance and degradation of cells susceptible to forming tumours. Alternatively, *beclin-1* might participate in the clearance of damaged mitochondria that produce reactive oxygen species, inducing DNA damage and accelerating malignant conversion.

The idea that autophagy has a role in tumorigenesis is appealing, given the importance of balancing protein synthesis and degradation. Also, diminishing the capacity for protein degradation through autophagy could stimulate growth and proliferation to an abnormal degree. Rapamycin, which potently suppresses translation, B and T cell proliferation, and survival of cells from selected tumours ${ }^{100}$, is now being tested in clinical trials for efficacy toward human cancers. Perhaps these effects are due in part to de-repression of autophagy.
Box 2 | Autophagy as a form of programmed cell death

In addition to its role in cell survival, autophagy has been observed in the non-apoptotic cell death (also known as type II programmed cell death) of numerous cell types following various stimuli. Treatment of L929 fibroblasts with the caspase inhibitor zVAD causes a form of cell death that is characterized by autophagosome formation. RNA interference of Atg7 or *beclin-1* attenuates this reduction in cell number, which indicates that autophagy is a key mediator of death in response to zVAD<sup>84</sup>. In wild-type mouse embryonic fibroblasts (MEFs), staurosporine causes cell death that is predominantly apoptotic, but MEFs that lack Bax and Bak undergo *beclin-1*- and Atg5-dependent non-apoptotic death<sup>101</sup>. Surprisingly, treatment with small interfering RNA against Bcl-x<sub>L</sub> protects these cells against non-apoptotic death, whereas overexpression of Bcl2 or Bcl-x<sub>L</sub> sensitizes wild-type MEFs to autophagy<sup>101</sup>. Together, these data raise the possibility that Bcl2 proteins regulate autophagy, and that the mode of programmed cell death is determined in part by the relative levels of pro- and anti-apoptotic Bcl2 proteins.

Glioblastoma cells treated with arsenic trioxide also acquire changes characteristic of autophagy. In these cells, death is blocked by bafilomycin A1, a V-type ATPase inhibitor that prevents acidification of lysosomal-based compartments, including autophagosomes<sup>80</sup>. In MCF-7 cells, tamoxifen or irradiation induces autophagic death blocked by the autophagy inhibitor 3-methyladenine<sup>102</sup>.

There is overlap in the molecules involved in apoptotic and non-apoptotic death, including certain substances produced by dying cells themselves. Ceramide is thought to be involved in both apoptosis and autophagy because endogenous levels increase after pro-apoptotic insults. Synthetic ceramide analogues (for example, C<sub>2</sub>-ceramide) induce cell death<sup>103,104</sup> and the accumulation of autophagic vacuoles in human colon cancer cells<sup>105</sup>. In addition, in tamoxifen-treated breast cancer cells, autophagy is blocked by a ceramide synthesis inhibitor<sup>105</sup>. In both cell lines, the induction of autophagy is associated with decreased Akt phosphorylation at Ser473 and the induction of *beclin-1* expression. Perhaps ceramide induces autophagy by relieving the anti-autophagic effects of class-I phosphatidylinositol 3-kinase (PI3K)/Akt. In malignant glioma cells, C<sub>2</sub>-ceramide induces autophagic changes and increases expression of BNIP3, a mitochondria-associated cell death protein also involved in apoptotic responses<sup>106</sup>. Surprisingly, BNIP3 overexpression also induces autophagic changes, which indicates that part of the pro-autophagic activity of ceramide might be due to its induction of this gene.

Alterations in autophagy have also been documented in other pathologies including several neurodegenerative disorders, such as Alzheimer’s disease, Huntington’s disease and Parkinsonism<sup>86–90</sup>. Furthermore, the neurotransmitter dopamine triggers autophagic cell death in some systems<sup>91,92</sup>. In many neurodegenerative disorders, abnormal protein folding or increased hydrophobicity cause proteins to aggregate and resist cytosolic protease degradation, apparently contributing to pathogenesis. Autophagy might facilitate the removal of aggregated proteins. In animal models of Huntington’s disease as well as in brains from human patients, mTOR is sequestered in these aggregates, which results in impaired translation and the induction of autophagy<sup>88</sup>. This evidently functions as a protective role, because an increase in mTOR activity enhances polyglutamine repeat toxicity, whereas rapamycin reduces the rate of neurodegeneration. This work suggests that autophagy helps clear protein aggregates associated with neurodegeneration, thereby reducing neurotoxicity.

Autophagy has also emerged as a possible arm of the innate immune system. *In vitro*, group A Streptococcus (GAS) associates with autophagosome-like vacuoles following infection of various nonphagocytic cell types<sup>93</sup>. This process requires Atg5, because cells derived from Atg5<sup>−/−</sup> mice are deficient in GAS vacuolar uptake, which results in persistence of the bacterium, and indicates a role for autophagy in limiting the virulence of intracellular pathogens. Consistent with this idea, activation of autophagy by starvation or rapamycin treatment causes *Mycobacterium tuberculosis* phagosomes to associate with microtubule-associated protein light chain 3 (LC3), a marker of autophagosome formation, and reduces bacterial survival within macrophages. Furthermore, interferon-γ treatment of macrophages induces autophagy and suppresses *M. tuberculosis* virulence. Autophagy has also been implicated in the elimination of *Listeria monocytogenes* and *Shigella flexneri*<sup>4,94,95</sup>. *Shigella flexneri* is particularly intriguing as one of its own proteins, VirG, seems to trigger autophagy by interacting with ATG5, whereas another protein, IcsB, interferes with autophagy, perhaps supporting intracellular bacterial survival. Together, these studies indicate that regulation of autophagy by the innate immune response might be one mechanism host cells use to eradicate intracellular pathogens.

### Concluding remarks

In recent years, there has been a renewed interest in autophagy as an important cellular function in various organisms. Its ability to support survival in quiescent cells deprived of growth factor signalling indicates that autophagy might play a part in the homeostasis of mammalian tissues. Maintaining intracellular metabolite levels is the result of reciprocal controls that are exerted by growth factor signalling and autophagy.

Understanding how autophagy is regulated by growth factors remains an active area of research. Other important questions include determining the fates of constituents of the macromolecules that are degraded in autophagosomes: are these used directly or further catabolized to maintain bioenergetics? Do these metabolites contribute substantially either to cell survival or cell death *in vivo*? Are autophagic cells, similar to apoptotic cells, invisible to the immune system?

Perhaps the most intriguing and important remaining question is to understand how autophagy supports survival in some cells but functions as a death pathway in others. Does the ‘amplitude’ of autophagy determine whether it promotes life or death, or is the relationship more subtle, involving the unmasking of a death signal that exists in some contexts but not in others? Solving this puzzle will help explain the importance of autophagic death in mammalian development and whether manipulating autophagy could be of therapeutic benefit.

1. Edinger, A. L., Cinalli, R. M. & Thompson, C. B. Rab7 prevents growth factor-independent survival by inhibiting cell-autonomous nutrient transporter expression. *Dev. Cell* **5**, 571–582 (2003).
2. Alessi, D. R. & Downes, C. P. The role of PI3-kinase in insulin action. *Biochim. Biophys. Acta* **1436**, 151–164 (1998).
3. Barata, J. T. *et al.* Activation of PI3K is indispensable for interleukin 7-mediated viability, proliferation, glucose use, and growth of T cell acute lymphoblastic leukemia cells. *J. Exp. Med.* **200**, 659–669 (2004).
4. Kirkegaard, K., Taylor, M. P. & Jackson, W. T. Cellular autophagy: surrender, avoidance and subversion by microorganisms. *Nature Rev. Microbiol.* **2**, 301–314 (2004).
5. Meijer, A. J. & Codogno, P. Regulation and role of autophagy in mammalian cells. *Int. J. Biochem. Cell Biol.* **36**, 2445–2462 (2004).
6. Yoshimori, T. Autophagy: a regulated bulk degradation process inside cells. *Biochem. Biophys. Res. Commun.* **313**, 453–458 (2004).
7. Levine, B. & Klionsky, D. J. Development by self-digestion: molecular mechanisms and biological functions of autophagy. *Dev. Cell* **6**, 463–477 (2004).
8. Gozuacik, D. & Kimchi, A. Autophagy as a cell death and tumor suppressor mechanism. *Oncogene* **23**, 2891–2906 (2004).
9. Huang, W. P. & Klionsky, D. J. Autophagy in yeast: a review of the molecular machinery. *Cell Struct. Funct.* **27**, 409–420 (2002).
10. Noda, T., Suzuki, K. & Ohsumi, Y. Yeast autophagosomes: de novo formation of a membrane structure. *Trends Cell Biol.* **12**, 231–235 (2002).
11. Ohsumi, Y. Molecular dissection of autophagy: two ubiquitin-like systems. *Nature Rev. Mol. Cell Biol.* **2**, 211–216 (2001).
12. Petiot, A. *et al.* Diversity of signaling controls of macroautophagy in mammalian cells. *Cell Struct. Funct.* **27**, 431–441 (2002).
13. Tsukada, M. & Ohsumi, Y. Isolation and characterization of autophagy-defective mutants of *Saccharomyces cerevisiae*. *FEBS Lett.* **333**, 169–174 (1993). Shows the involvement of autophagy in maintaining survival of unicellular organisms during nutrient deprivation. Using a genetic screen to isolate yeast cells that are unable to form autophagic vacuoles, the authors identified the first set of autophagy genes, which provided the foundation for the discovery of homologues in higher eukaryotes.
14. Otto, G. P. *et al.* Macroautophagy is required for multicellular development of the social amoeba *Dictyostelium discoideum*. *J. Biol. Chem.* **278**, 17636–17645 (2003).
15. Otto, G. P. *et al.* *Dictyostelium* macroautophagy mutants vary in the severity of their developmental defects. *J. Biol. Chem.* **279**, 15621–15629 (2004).
16. Journet, E. P., Bligny, R. & Douce, R. Biochemical changes during sucrose deprivation in higher plant cells. *J. Biol. Chem.* **261**, 3193–3199 (1986).
17. Aubert, S. *et al.* Ultrastructural and biochemical characterization of autophagy in higher plant cells subjected to carbon deprivation: control by the supply of mitochondria with respiratory substrates. *J. Cell Biol.* **133**, 1251–1263 (1996). Work by the group in reference 16 showed that plants experience decreased glycolytic enzymatic activity, growth arrest and a decline in respiration during nutrient withdrawal. Reference 17 shows that autophagy is controlled by the levels of oxidizable substrates for the mitochondria and not by carbon sugar deprivation.
18. Hanaoka, H. *et al.* Leaf senescence and starvation-induced chlorosis are accelerated by the disruption of an *Arabidopsis* autophagy gene. *Plant Physiol.* **129**, 1181–1193 (2002).
19. Yoshimoto, K. *et al.* Processing of ATG8s, ubiquitin-like proteins, and their deconjugation by ATG4s are essential for plant autophagy. *Plant Cell* **16**, 2967–2983 (2004).
20. Doelling, J. H., Walker, J. M., Friedman, E. M., Thompson, A. R. & Vierstra, R. D. The APG8/12-activating enzyme APG7 is required for proper nutrient recycling and senescence in *Arabidopsis thaliana*. *J. Biol. Chem.* **277**, 33105–33114 (2002).
21. Melendez, A. *et al.* Autophagy genes are essential for dauer development and life-span extension in *C. elegans*. *Science* **301**, 1387–1391 (2003).
22. Scott, R. C., Schuldiner, O. & Neufeld, T. P. Role and regulation of starvation-induced autophagy in the *Drosophila* fat body. *Dev. Cell* **7**, 167–178 (2004).
23. Kuma, A. *et al.* The role of autophagy during the early neonatal starvation period. *Nature* **432**, 963 (2004). Describes the first genetic demonstration for the *in vivo* requirement of autophagy in early neonatal mammal survival during nutrient starvation. Atg5-knockout neonatal mice have a severe reduction in the ability to survive up to the establishment of their ability to be nursed by their mothers. It follows from several observations (see references 24–27) that associated the degradation of glycogen in autophagosomes during early neonatal periods.
24. Schiaffino, S. & Hanzlikova, V. Autophagic degradation of glycogen in skeletal muscles of the newborn rat. *J. Cell Biol.* **52**, 41–51 (1972).
25. Devos, P. & Hers, H. G. Random, presumably hydrolytic, and lysosomal glycogenolysis in the livers of rats treated with phlorizin and of newborn rats. *Biochem. J.* **192**, 177–181 (1980).
26. Iwamasa, T., Tsuru, T., Hamada, T. & Takeuchi, T. Physicochemical and ultrastructural studies on glycogenosomes in newborn rat hepatocytes. *Pathol. Res. Pract.* **167**, 363–373 (1980).
27. Kondomerkos, D. J., Kalamidas, S. A. & Kotoulas, O. B. An electron microscopic and biochemical study of the effects of glucagon on glycogen autophagy in the liver and heart of newborn rats. *Microsc. Res. Tech.* **63**, 87–93 (2004).
28. Wei, M. C. *et al.* Proapoptotic BAX and BAK: a requisite gateway to mitochondrial dysfunction and death. *Science* **292**, 727–730 (2001).
29. Zong, W. X. *et al.* Alkylating DNA damage stimulates a regulated form of necrotic cell death. *Genes Dev.* **18**, 1272–1282 (2004).
30. Zong, W. X. *et al.* BH3-only proteins that bind pro-survival Bcl-2 family members fail to induce apoptosis in the absence of Bax and Bak. *Genes Dev.* **15**, 1481–1486 (2001).
31. Lum, J. J. *et al.* Growth factor regulation of autophagy and survival in the absence of apoptosis. *Cell* **120**, 237–248 (2005).
32. Petiot, A. *et al.* Distinct classes of phosphatidylinositol 3'-kinases are involved in signaling pathways that control macroautophagy in HT-29 cells. *J. Biol. Chem.* **275**, 992–998 (2000). Shows that class-I and class-III PI3K cascades have distinct effects on autophagy. Autophagy can be blocked by class-I PI3K inhibitors, which allows new insight into the signal transduction cascades of how autophagy is regulated.
33. Kihara, A. *et al.* Beclin–phosphatidylinositol 3-kinase complex functions at the trans-Golgi network. *EMBO Rep.* **2**, 330–335 (2001).
34. Whetton, A. D., Bazill, G. W. & Dexter, T. M. Haemopoietic cell growth factor mediates cell survival via its action on glucose transport. *EMBO J.* **3**, 409–413 (1984).
35. Vander Heiden, M. G. *et al.* Growth factors can influence cell growth and survival through effects on glucose metabolism. *Mol. Cell. Biol.* **21**, 5899–5912 (2001).
36. Edinger, A. L. & Thompson, C. B. Akt maintains cell size and survival by increasing mTOR-dependent nutrient uptake. *Mol. Biol. Cell* **13**, 2276–2288 (2002).
37. Bentley, J. *et al.* Interleukin-3-mediated cell survival signals include phosphatidylinositol 3-kinase-dependent translocation of the glucose transporter GLUT1 to the cell surface. *J. Biol. Chem.* **278**, 39337–39348 (2003).
38. Bauer, D. E. *et al.* Cytokine stimulation of aerobic glycolysis in hematopoietic cells exceeds proliferative demand. *FASEB J.* **18**, 1303–1305 (2004).
39. Hirai, S. *et al.* Fibroblast growth factor-dependent metabolism of hypoxanthine via the salvage pathway for purine synthesis in porcine aortic endothelial cells. *Biochem. Pharmacol.* **45**, 1695–1701 (1993).
40. Ishijima, S. *et al.* Evidence for early mitogenic stimulation of metabolic flux through phosphoribosyl pyrophosphate into nucleotides in Swiss 3T3 cells. *J. Biochem.* **104**, 570–575 (1988).
41. del Santo, B. *et al.* Differential expression and regulation of nucleoside transport systems in rat liver parenchymal and hepatoma cells. *Hepatology* **28**, 1504–1511 (1998).
42. Becker, M. A., Dicker, P. & Rozengurt, E. Mitogenic enhancement of purine base phosphoribosylation in Swiss mouse 3T3 cells. *Am. J. Physiol.* **244**, C288–C296 (1983).
43. Plas, D. R. *et al.* Akt and Bcl-xL promote growth factor-independent survival through distinct effects on mitochondrial physiology. *J. Biol. Chem.* **276**, 12041–12048 (2001).
44. Rathmell, J. C. *et al.* Akt-directed glucose metabolism can prevent Bax conformation change and promote growth factor-independent survival. *Mol. Cell. Biol.* **23**, 7315–7328 (2003).
45. Elstrom, R. L. *et al.* Akt stimulates aerobic glycolysis in cancer cells. *Cancer Res.* **64**, 3892–3899 (2004).
46. Hardie, D. G., Carling, D. & Carlson, M. The AMP-activated/SNF1 protein kinase subfamily: metabolic sensors of the eukaryotic cell? *Annu. Rev. Biochem.* **67**, 821–855 (1998).
47. Hay, N. & Sonenberg, N. Upstream and downstream of mTOR. *Genes Dev.* **18**, 1926–1945 (2004).
48. Brown, E. J. & Schreiber, S. L. A signaling pathway to translational control. *Cell* **86**, 517–520 (1996).
49. Tokunaga, C., Yoshino, K. & Yonezawa, K. mTOR integrates amino acid- and energy-sensing pathways. *Biochem. Biophys. Res. Commun.* **313**, 443–446 (2004).
50. Raught, B., Gingras, A. C. & Sonenberg, N. The target of rapamycin (TOR) proteins. *Proc. Natl Acad. Sci. USA* **98**, 7037–7044 (2001).
51. Bjornsti, M. A. & Houghton, P. J. The TOR pathway: a target for cancer therapy. *Nature Rev. Cancer* **4**, 335–348 (2004).
52. Fingar, D. C. & Blenis, J. Target of rapamycin (TOR): an integrator of nutrient and growth factor signals and coordinator of cell growth and cell cycle progression. *Oncogene* **23**, 3151–3171 (2004).
53. Tee, A. R. *et al.* Tuberous sclerosis complex gene products, Tuberin and Hamartin, control mTOR signaling by acting as a GTPase-activating protein complex toward Rheb. *Curr. Biol.* **13**, 1259–1268 (2003).
54. Inoki, K. *et al.* Rheb GTPase is a direct target of TSC2 GAP activity and regulates mTOR signaling. *Genes Dev.* **17**, 1829–1834 (2003).
55. Dennis, P. B. *et al.* Mammalian TOR: a homeostatic ATP sensor. *Science* **294**, 1102–1105 (2001).
56. Krause, U., Bertrand, L. & Hue, L. Control of p70 ribosomal protein S6 kinase and acetyl-CoA carboxylase by AMP-activated protein kinase and protein phosphatases in isolated hepatocytes. *Eur. J. Biochem.* **269**, 3751–3759 (2002).
57. Dubbelhuis, P. F. & Meijer, A. J. Hepatic amino acid-dependent signaling is under the control of AMP-dependent protein kinase. *FEBS Lett.* **521**, 39–42 (2002).
58. Bolster, D. R. *et al.* AMP-activated protein kinase suppresses protein synthesis in rat skeletal muscle through down-regulated mammalian target of rapamycin (mTOR) signaling. *J. Biol. Chem.* **277**, 23977–23980 (2002).
59. Larsen, A. K. *et al.* Naringin-sensitive phosphorylation of plectin, a cytoskeletal cross-linking protein, in isolated rat hepatocytes. *J. Biol. Chem.* **277**, 34826–34835 (2002).
60. Inoki, K., Zhu, T. & Guan, K. L. TSC2 mediates cellular energy response to control cell growth and survival. *Cell* **115**, 577–590 (2003).
61. Potter, C. J., Pedraza, L. G. & Xu, T. Akt regulates growth by directly phosphorylating Tsc2. *Nature Cell Biol.* **4**, 658–665 (2002).
62. Blommaart, E. F. *et al.* Phosphorylation of ribosomal protein S6 is inhibitory for autophagy in isolated rat hepatocytes. *J. Biol. Chem.* **270**, 2320–2326 (1995).

63. Noda, T. & Ohsumi, Y. Tor, a phosphatidylinositol kinase homologue, controls autophagy in yeast. *J. Biol. Chem.* **273**, 3963–3966 (1998).

64. Abeliovich, H. *et al.* Dissection of autophagosome biogenesis into distinct nucleation and expansion steps. *J. Cell Biol.* **151**, 1025–1034 (2000).

65. Kamada, Y., Sekito, T. & Ohsumi, Y. Autophagy in yeast: a TOR-mediated response to nutrient starvation. *Curr. Top. Microbiol. Immunol.* **279**, 73–84 (2004).

66. Wang, Z. Antagonistic controls of autophagy and glycogen accumulation by Snf1p, the yeast homolog of AMP-activated protein kinase, and the cyclin-dependent kinase Pho85p. *Mol. Cell. Biol.* **21**, 5742–5752 (2001).

67. Danial, N. N. *et al.* BAD and glucokinase reside in a mitochondrial complex that integrates glycolysis and apoptosis. *Nature* **424**, 952–956 (2003).

68. Peng, T., Golub, T. R. & Sabatini, D. M. The immunosuppressant rapamycin mimics a starvation-like signal distinct from amino acid and glucose deprivation. *Mol. Cell. Biol.* **22**, 5575–5584 (2002).

69. Elmore, S. P. *et al.* The mitochondrial permeability transition initiates autophagy in rat hepatocytes. *FASEB J.* **15**, 2286–2287 (2001).

70. Lang-Rollin, I. C. *et al.* Mechanisms of caspase-independent neuronal death: energy depletion and free radical generation. *J. Neurosci.* **23**, 11015–11025 (2003).

71. Xue, L., Fletcher, G. C. & Tolkovsky, A. M. Mitochondria are selectively eliminated from eukaryotic cells after blockade of caspases during apoptosis. *Curr. Biol.* **11**, 361–365 (2001).

72. Lemasters, J. J. *et al.* The mitochondrial permeability transition in cell death: a common mechanism in necrosis, apoptosis and autophagy. *Biochim. Biophys. Acta* **1366**, 177–196 (1998).

73. Kissova, I. *et al.* Uth1p is involved in the autophagic degradation of mitochondria. *J. Biol. Chem.* **279**, 39068–39074 (2004).

74. Kiel, J. A. *et al.* Macropexophagy in *Hansenula polymorpha*: facts and views. *FEBS Lett.* **549**, 1–6 (2003).

75. Bellu, A. R. *et al.* Peroxisome biogenesis and selective degradation converge at Pex14p. *J. Biol. Chem.* **276**, 44570–44574 (2001).

76. Bjornsti, M. A. & Houghton, P. J. Lost in translation: dysregulation of cap-dependent translation and cancer. *Cancer Cell* **5**, 519–523 (2004).

77. Sawyers, C. L. Will mTOR inhibitors make it as cancer drugs? *Cancer Cell* **4**, 343–348 (2003).

78. Qu, X. *et al.* Promotion of tumorigenesis by heterozygous disruption of the beclin 1 autophagy gene. *J. Clin. Invest.* **112**, 1809–1820 (2003).

79. Yue, Z. *et al.* Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor. *Proc. Natl Acad. Sci. USA* **100**, 15077–15082 (2003).

References 78 and 79 describe the first experimental evidence that genetic inactivation of *beclin-1* resulted in a high frequency of tumour formation. These mice had both lower basal levels of autophagy and defective autophagic responses.

80. Kanzawa, T. *et al.* Induction of autophagic cell death in malignant glioma cells by arsenic trioxide. *Cancer Res.* **63**, 2103–2108 (2003).

81. Kanzawa, T. *et al.* Role of autophagy in temozolomide-induced cytotoxicity for malignant glioma cells. *Cell Death Differ.* **11**, 448–457 (2004).

82. Paglin, S. *et al.* A novel response of cancer cells to radiation involves autophagy and formation of acidic vesicles. *Cancer Res.* **61**, 439–444 (2001).

83. Bursch, W. The autophagosomal-lysosomal compartment in programmed cell death. *Cell Death Differ.* **8**, 569–581 (2001).

84. Yu, L. *et al.* Regulation of an ATG7-beclin 1 program of autophagic cell death by caspase-8. *Science* **304**, 1500–1502 (2004). The authors describe how autophagic cell death is regulated by the caspase-8 pathway. Induction of autophagy by a general caspase inhibitor promotes autophagy and is prevented by knockdown of *ATG7* and *beclin-1*.

85. Juhasz, G. *et al.* The *Drosophila* homolog of *Aut1* is essential for autophagy and development. *FEBS Lett.* **543**, 154–158 (2003).

86. Zhu, J. H. *et al.* Localization of phosphorylated ERK/MAP kinases to mitochondria and autophagosomes in Lewy body diseases. *Brain Pathol.* **13**, 473–481 (2003).

87. Stefanis, L. *et al.* Expression of A53T mutant but not wild-type α-synuclein in PC12 cells induces alterations of the ubiquitin-dependent degradation system, loss of dopamine release, and autophagic cell death. *J. Neurosci.* **21**, 9549–9560 (2001).

88. Ravikumar, B. *et al.* Inhibition of mTOR induces autophagy and reduces toxicity of polyglutamine expansions in fly and mouse models of Huntington disease. *Nature Genet.* **36**, 585–595 (2004).

89. Qin, Z. H. *et al.* Autophagy regulates the processing of amino terminal huntingtin fragments. *Hum. Mol. Genet.* **12**, 3231–3244 (2003).

90. Anglade, P. *et al.* Apoptosis and autophagy in nigral neurons of patients with Parkinson's disease. *Histol. Histopathol.* **12**, 25–31 (1997).

91. Petersen, A. *et al.* Expanded CAG repeats in exon 1 of the Huntington's disease gene stimulate dopamine-mediated striatal neuron autophagy and degeneration. *Hum. Mol. Genet.* **10**, 1243–1254 (2001).

92. Gomez-Santos, C. *et al.* Dopamine induces autophagic cell death and α-synuclein increase in human neuroblastoma SH-SY5Y cells. *J. Neurosci. Res.* **73**, 341–350 (2003).

93. Nakagawa, I. *et al.* Autophagy defends cells against invading group A Streptococcus. *Science* **306**, 1037–1040 (2004).

94. Rich, K. A., Burkett, C. & Webster, P. Cytoplasmic bacteria can be targets for autophagy. *Cell. Microbiol.* **5**, 455–468 (2003).

95. Ogawa, M. *et al.* Escape of intracellular shigella from autophagy. *Science* **307**, 727–731 (2004).

96. Furuta, S. *et al.* Ras is involved in the negative control of autophagy through the class I PI3-kinase. *Oncogene* **23**, 3898–3904 (2004).

97. Tsuneoka, M. *et al.* c-myc induces autophagy in rat 3Y1 fibroblast cells. *Cell Struct. Funct.* **28**, 195–204 (2003).

98. Arico, S. *et al.* The tumor suppressor PTEN positively regulates macroautophagy by inhibiting the phosphatidylinositol 3-kinase/protein kinase B pathway. *J. Biol. Chem.* **276**, 35243–35246 (2001).

99. Liang, X. H. *et al.* Beclin 1 contains a leucine-rich nuclear export signal that is required for its autophagy and tumor suppressor function. *Cancer Res.* **61**, 3443–3449 (2001).

100. Schreiber, S. L. Chemistry and biology of the immunophilins and their immunosuppressive ligands. *Science* **251**, 283–287 (1991).

101. Shimizu, S. *et al.* Role of Bcl-2 family proteins in a non-apoptotic programmed cell death dependent on autophagy genes. *Nature Cell Biol.* **6**, 1221–1228 (2004).

102. Bursch, W. *et al.* Active cell death induced by the anti-estrogens tamoxifen and ICI 164 384 in human mammary carcinoma cells (MCF-7) in culture: the role of autophagy. *Carcinogenesis* **17**, 1595–1607 (1996).

103. van Blitterswijk, W. J. *et al.* Ceramide: second messenger or modulator of membrane structure and dynamics? *Biochem. J.* **369**, 199–211 (2003).

104. Suzuki, E. *et al.* Sphingosine-dependent apoptosis: a unified concept based on multiple mechanisms operating in concert. *Proc. Natl Acad. Sci. USA* **101**, 14788–14793 (2004).

105. Scarlatti, F. *et al.* Ceramide-mediated macroautophagy involves inhibition of protein kinase B and up-regulation of beclin 1. *J. Biol. Chem.* **279**, 18384–18391 (2004).

106. Daido, S. *et al.* Pivotal role of the cell death factor BNIP3 in ceramide-induced autophagic cell death in malignant glioma cells. *Cancer Res.* **64**, 4286–4293 (2004).

107. Mizushima, N. *et al.* In vivo analysis of autophagy in response to nutrient starvation using transgenic mice expressing a fluorescent autophagosome marker. *Mol. Biol. Cell* **15**, 1101–1111 (2004).

108. Kabeya, Y. *et al.* LC3, a mammalian homologue of yeast Apg8p, is localized in autophagosome membranes after processing. *EMBO J.* **19**, 5720–5728 (2000).

References 107 and 108 describe the mammalian homologue of ATG8, LC3, as a specific marker of autophagosome formation.

Acknowledgements

The authors wish to thank Daniel E. Bauer for insightful comments and suggestions during the preparation of this review. Julian J. Lum is supported by a fellowship from the Leukemia and Lymphoma Society. Ralph J. DeBerardinis is supported by a grant from the National Institutes of Health. Grants for the Thompson laboratory are provided in part by the National Institutes of Health.

Competing interests statement

The authors declare no competing financial interests.

### Online links

#### DATABASES

The following terms in this article are linked online to:

- Entrez Gene: http://www.ncbi.nlm.nih.gov/entrez
- ATG1 | ATG5 | ATG6 | ATG7 | ATG8 | ATG10 | daf-2 | PTEN
- Swiss-Prot: http://us.expasy.org/sprot/
- Bak | Bax | CPT1 | 4E-BP1 | Pex14 | Rheb | S6K | TSC1 | TSC2 | Uth1

Access to this interactive links box is free online.
