
The Bacillus subtilis YdfHI two-component system regulates the transcription of *ydfJ*, a member of the RND superfamily

Masakuni Serizawa and Junichi Sekiguchi

Department of Applied Biology, Faculty of Textile Science and Technology, Shinshu University, 3-15-1 Tokida, Ueda-shi, Nagano 386-8567, Japan

Correspondence  
Junichi Sekiguchi  
jsekigu@gipc.shinshu-u.ac.jp  

Received 13 September 2004  
Revised 1 February 2005  
Accepted 22 February 2005  

The *ydfHI* genes encode a sensor kinase and a response regulator forming a two-component system. *ydfJ* is located downstream of *ydfHI*, and belongs to the RND (resistance-nodulation-cell division) superfamily, which is present in most major organisms. Four genes (*secDF*, *yerP*, *ydfJ* and *ydgH*) in *Bacillus subtilis* belong to this family. This study revealed that the YdfHI two-component system regulates *ydfJ* transcription. A gel shift assay using histidine-tagged YdfI (h-YdfI) showed that it directly binds to the *ydfJ* promoter region. Moreover, DNase I footprinting analysis revealed a tandem repeat sequence consisting of two conserved 12-mer sequences (GCCCRAAYGTAC) within the h-YdfI-binding site.

INTRODUCTION

In micro-organisms, a two-component system is a very important signal transduction system for adaptation to drastic and immediate changes in external or internal environmental conditions. This system is usually composed of a sensor kinase and a response regulator. The sensor kinase monitors some type of signal caused by changes in environmental conditions, and transmits the information to the response regulator via a phosphoryl-transfer reaction. The response regulator is shifted to its active state by the phosphate group received from the sensor kinase, and directly binds to the promoter region of its target genes as a transcriptional regulator (Stock *et al.*, 1995).

Twenty-nine adjacent sets of genes encoding the sensor kinase and response regulator are recognized in *Bacillus subtilis*. The functions of thirteen systems, namely CitST (Yamamoto *et al.*, 2000), ComPA (Msadek *et al.*, 1995), CssSR (Hyyrylainen *et al.*, 2001), DctSR (Asai *et al.*, 2000), DegSU (Msadek *et al.*, 1995), DesKR (Aguilar *et al.*, 2001), PhoRP (Hulett, 1996), ResED (Sun *et al.*, 1996), BceRS (Ohki *et al.*, 2003), YufLM (Doan *et al.*, 2003; Tanaka *et al.*, 2003), LiaRS (Mascher *et al.*, 2004), YxdJK (Joseph *et al.*, 2004) and YycFG (Fukuchi *et al.*, 2000), have been reported. It has been recognized that those systems are necessary for micro-organisms to survive under various conditions. Sixteen other systems may also have certain important functions.

Genes regulated by a two-component system tend to be located adjacent to the genes encoding the two-component system. Moreover, there is a close association between the signal that a sensor kinase recognizes and the function of the regulated genes. For example, BceRS regulates *bceAB* transcription, and *bceAB* is located downstream of *bceRS*. *bceAB* encodes the subunits of the ABC transporter that expels bacitracin from inside the cell. The transcription of *bceAB* is induced by BceRS, which is activated by the addition of extracellular bacitracin (Ohki *et al.*, 2003). The same phenomenon is also observed in CitST, CssSR, DctSR and DesKR. We studied the genes that are located adjacent to the 16 unknown two-component systems. Genes associated with substance transport were recognized adjacent to 9 two-component systems (YcbAB, YcbML, YccGH, YclKJ, YdfHI, YesMN, YfiJK, YvcPQ and YvfTU). ABC transporter homologues were recognized adjacent to 6 two-component systems (YcbML, YccGH, YclKJ, YfiJK, YvcPQ and YvfTU). *ydfJ* is located downstream of *ydfHI*. *ydfJ* (724 aa; Mr 76 850) has 12 transmembrane segments, two MMPL (Tekaia *et al.*, 1999) domains and an SS (sterol sensing) domain, and belongs to the RND (resistance-nodulation-cell division) superfamily that is present in most major organisms. Most genes assigned to this superfamily encode proteins that catalyse substrate efflux via a H⁺ antiport mechanism. Members of this family are associated with multidrug resistance, heavy-metal ion export, transport of oligosaccharides, and solvent tolerance (Tseng *et al.*, 1999). However, little is known about noxious compound resistance systems mediated by genes involved in this family in low-G+C Gram-positive bacteria. Therefore, the analysis of genes in the RND superfamily of *B.* subtilis as a model bacterium is important.

In DNA microarray experiments under the overexpression of a response regulator (YdfI), *ydfJ* is regulated by a YdfHI two-component system (Kobayashi *et al.*, 2001). Moreover, there are some reports of an association between

Abbreviation: RND, resistance-nodulation-cell division.
M. Serizawa and J. Sekiguchi

two-component systems and drug efflux genes (Grkovic *et al.*, 2002). Thus, we were interested in determining the association between the YdfHI two-component system and *ydfJ* transcription, and also the roles of YdfHI and YdfJ in antibiotic resistance and substance transport. In this analysis, we report the positive regulation of *ydfJ* transcription mediated by the YdfHI two-component system.

### METHODS

**Bacterial strains, plasmids, primers and culture conditions.** The bacterial strains and plasmids used in this study are listed in Table 1. The primers used are listed in Table 2. The bacterial strains were cultured in LB medium (*g l⁻¹*: yeast extract, 5; polypeptone, 10; NaCl, 5; pH 7·2) at 37°C. When necessary, ampicillin, spectinomycin or kanamycin was added at final concentrations of 50, 50 and 10 µg ml⁻¹, respectively. *B. subtilis* strains were also grown in Difco sporulation medium (DSM) (Schaeffer *et al.*, 1965).

**Construction of the spectinomycin-resistance-gene harbouring plasmid.** A *Bam*HI-*Xho*I fragment from plasmid pDG1727 including the spectinomycin resistance gene was ligated into the corresponding sites of pBluescriptII-SK(+) to construct pBlueSPR.

**Construction of the *ydfH ydfJ* double null mutant.** A *ydfH ydfJ* double null mutant was constructed by the PCR ligation method. Fragments ΔYDFH and ΔYDFI contained the 5′-end sequence of *ydfH* and the 3′-end sequence of *ydfJ*, respectively. These fragments were amplified from *B. subtilis* 168 chromosomal DNA, and the primers YDFH-BF1 and YDFH-R1, and YDFI-F2 and YDFI-XR2, respectively. Fragment SPR was amplified from plasmid pBlueSPR, containing the spectinomycin resistance gene, and the primers

PB-M13Rev and PB-M13-20. The 5′-end sequences of YDFH-R1 and YDFI-F2 are complementary to PB-M13-20 and PB-M13Rev, respectively. Fragment YDFHI-SPR was amplified from a mixture of fragments ΔYDFH, ΔYDFI and SPR, and the primers YDFH-BF1 and YDFI-XR2. Fragment YDFHI-SPR was used for the transformation of *B. subtilis* 168, and the *ydfH* null mutant YDFHΔSp was selected on an agar plate containing spectinomycin.

**Construction of the *ydfJ* disruptant.** A *ydfJ* internal fragment amplified from 168 chromosomal DNA and the primers YDFJ-FE and YDFJ-RB was digested with *Eco*RI and *Bam*HI. Then the digested fragment was ligated into the *Eco*RI and *Bam*HI sites of pMUTIN4, and *Escherichia coli* JM109 was transformed with the ligation mixture to produce pM4YDFJ. Subsequently, YDFJΔPM4 was constructed by Campbell-type integration of the plasmid DNA prepared from *E. coli* C600 cells harbouring pM4YDFJ. The proper integration of the plasmid was confirmed by PCR.

**Construction of a plasmid for producing 6×His-tag-fused YdfI in *E. coli*.** For the overproduction and purification of the 6×His-tag-fused response regulator YdfI in *E. coli*, the expression plasmid pQEYDFI was constructed. A 680 bp PCR fragment obtained with the primers YDFI-HisFB and YDFI-HisRP was digested with *Bam*HI and *Pst*I, followed by ligation into the corresponding sites of plasmid pGEM-3Zf(+). *E. coli* JM109 cells were transformed with the ligation mixture to produce pGYDFI. The nucleotide sequence was confirmed with a DNA sequencer (ABI Prism 310 Genetic Analyser; Applied Biosystems). A *Bam*HI-*Pst*I fragment of pGYDFI including *ydfJ* was ligated in the corresponding sites of the histidine-tagged plasmid pQE-30. The resultant plasmid pQEYDFI was used for the transformation of *E. coli* M15 harbouring the plasmid pREP4. Consequently, the synthesis of the histidine-tagged YdfI (h-YdfI) was controlled by IPTG induction.

---

### Table 1. Bacterial strains and plasmids

| Strain or plasmid | Relevant genotype | Source or reference* |
|-------------------|-------------------|-----------------------|
| *B. subtilis*     |                   |                       |
| 168               | *trpC2*           | S.D. Ehrlich          |
| YDFHIAΔSp         | *trpC2 ydfHI::spc* | This study            |
| YDFJΔPM4          | *trpC2 ydfJ::pM4YDFJ* | This study            |
| *E. coli*         |                   |                       |
| JM109             | *recA1 supE44 endA1 hsdR17 gyrA96 relA1 thi-1* | Takara Bio |
|                   | Δ(*lac-proAB*) / F'(*traD36 proAB⁺ lacIq lacZΔM15*) |           |
| C600              | *supE44 hsdR17 thi-1 thr-1 leuB6 lacY1 tonA21* | Laboratory stock |
| M15               | *Nal⁸ Str⁸ Rif⁸ lac ara gal mtlF⁻ recA⁺ uvr⁺* | Qiagen      |
| Plasmid           |                   |                       |
| pGEM-3Zf(+)       | *blalacZ*         | Promega               |
| pBluescriptII-SK(+) | *blalacZ*        | Stratagene            |
| pDG1727           | *blaspc*          | BGSC                  |
| pREP4             | *lacIneo*         | Qiagen                |
| pQE30             | *bla*             | Qiagen                |
| pMUTIN4           | *ermblalacZlacIPspac* | Vagner *et al.* (1998) |
| pBlueSPR          | *blaspc*          | This study            |
| pGYDFI            | *blaydfI*         | This study            |
| pQEYDFI           | *blaydfI*         | This study            |
| pDG148ydfI        | *blakan ydfI*     | Kobayashi *et al.* (2001) |
| pM4YDFJ           | *pMUTIN4::ΔydfJ*  | This study            |

*BGSC, Bacillus Genetic Stock Center, Ohio State University, Columbus, USA; S.D. Ehrlich, INRA, Jouy-en-Josas, France.

1770 Microbiology 151
The YdfH system regulates ydfJ transcription

Table 2. Primers

| Primer       | Sequence (5'→3') *                                                                                   | Restriction enzyme |
|--------------|----------------------------------------------------------------------------------------------------|---------------------|
| YDFH-BF1     | gcgcGGATCCTTACTCACATGATAGGCGG                                                                       | BamHI               |
| YDFH-R1      | CACTGGCCGTCGTTTACATACGCCGAATTATGACCA                                                                |                     |
| YDFI-F2      | CATGGTCATAGCTGTTCCGGATATCTGTAAAAGATACG                                                              |                     |
| YDFI-XR2     | gccgCTCGAGTTGTCTATTGTCAGAATACCT                                                                     | XhoI                |
| PB-M13-20    | GTAAAAACGACGGGCCAGTG                                                                                |                     |
| PB-M13Rev    | GGAAACAGCTATGACCATG                                                                                |                     |
| YDFH-FE      | gcgcGAATTCTTACTCACATGATAGGCGG                                                                       | EcoRI               |
| YDFH-RB      | GTAATACGACTCACTATAGGGCGGATCCTCTGTCTGCCATTCCTATAA                                                    | BamHI               |
| YDFI-FE      | gcgcGAATCCGTTGATGACCATCTTGTC                                                                        | EcoRI               |
| YDFI-RB      | GTAATACGACTCACTATAGGGCGGATCCTCTGATGGCTGCATCCATT                                                     | BamHI               |
| YDFJ-FE      | gcgcGAATCATGGGTTGCTCGCAATCG                                                                         | EcoRI               |
| YDFJ-RB      | GTAATACGACTCACTATAGGGCGGACTTGCAATCGAGTCAAC                                                          | BamHI               |
| YFKC-FE      | gcgcGAATTCTCGTCGATATTCTCCTCGT                                                                       | EcoRI               |
| YFKC-RB      | GTAATACGACTCACTATAGGGCGGATCCTATCTTGGCCAAATCATGCA                                                   | BamHI               |
| YDFJP-F      | gcgcgcCTGCAGGAATTCATGGCCATGCAAAAAGGTATT                                                             | PstI, EcoRI         |
| YDFJP-R      | gcgccgCAATTGGTCGACTGACATGATAATCTCCTTCA                                                              | MunI, SalI          |
| YDFI-HisFB   | gccgGGATCCAATAAGGTTTTAATCGTTGATG                                                                    | BamHI               |
| YDFI-HisRP   | gccgCTGCAGGGCAATTTTCAAATATGCAGT                                                                     | PstI                |
| YDFJ-PEX     | GATCCACGACACATATTCGCT                                                                               |                     |
| PET-T7       | TAATACGACTCACTATAGGG                                                                                |                     |

*The additional sequences (lower case), restriction sites (underlined) and T7 RNA polymerase recognition sequences (bold italics) are indicated.

Transformation of E. coli and B. subtilis. This was performed as described previously (Ishikawa et al., 1998).

Northern blot analysis. Total RNA preparation and purification were performed as described by Serizawa et al. (2004). The Northern blot analysis of RNAs fractionated by electrophoresis in agarose-formaldehyde gels was performed as described by Ishikawa et al. (1998). The transfer of RNAs onto nylon membranes was performed with a vacuum blotter (model BS-31; Bio Craft). Probe labelling was performed with a DIG RNA labelling kit (Roche Diagnostics) according to the manufacturer's instructions, but with some minor modifications. Templates for the *ydfH-*, *ydfI-*, *ydfJ-* and *yfkC*-specific probes were constructed by PCR. *ydfH-*, *ydfI-*, *ydfJ-* and *yfkC-* internal fragments were amplified from *B. subtilis* 168 chromosomal DNA, and the primers YDFH-FE and YDFH-RB, YDFI-FE and YDFI-RB, YDFJ-FE and YDFJ-RB, and YFKC-FE and YFKC-RB, respectively. The 5'-end sequences of YDFH-RB, YDFI-RB, YDFJ-RB and YFKC-RB contain the sequence of the T7 RNA polymerase recognition site. The amplified fragments were digested with EcoRI, and then used as templates for in vitro run-off transcription with T7 RNA polymerase. Hybridization and detection were performed with a DIG luminescent detection kit (Roche Diagnostics), according to the manufacturer's instructions.

Primer extension analysis. This was performed as described previously (Yoshida et al., 1997). Total RNA (40 μg) was hybridized with a primer YDFJ-PEX that had been labelled at the 5'-end using T4 polynucleotide kinase (Megalabel kit, Takara Bio) and [γ-³²P]ATP (Amersham Biosciences). Unincorporated [γ-³²P]ATP was removed using a MicroSpin S200HR column (Amersham Biosciences). Primer extension reactions were performed with Molony murine leukaemia virus reverse transcriptase (Reverse transcriptase M-MLV; Takara Bio). Dideoxy sequence ladders for use as size markers were prepared with the Takara taq cycle sequencing kit (Takara Bio).

Overexpression and purification of h-YdfI. *E. coli* M15 harbouring pREP4 and pQEYDFJ was cultured in 500 ml LB medium containing ampicillin (50 μg ml⁻¹), kanamycin (20 μg ml⁻¹) and glucose (1%, w/v) until the cell density reached approximately OD₆₀₀ 0·6 at 37°C. Then, IPTG was added to the culture at a final concentration of 1 mM, followed by further incubation for 1·5 h. The culture was centrifuged and the pellet was resuspended in 10 ml 10 mM imidazole-NPB solution [10 mM imidazole and 1 M NaCl in 20 mM sodium phosphate buffer (pH 7·4)]. After ultrasonication, the suspension was centrifuged, and the supernatant was filtered through a 0·45 μm membrane filter (DISMIC-13CP cellulose acetate 0·45 μm; Advantec), and applied onto a HiTrap chelating column (1 ml; Amersham Biosciences). The column was washed with 20 ml 10 mM imidazole-NPB solution, and the h-YdfI protein was then eluted with NPB solution containing a stepwise gradient of imidazole, from 20 to 500 mM. The calculated molecular mass of h-YdfI was 25212 Da. The purified h-YdfI protein was dialysed in PD buffer [40 mM Tris/HCl (pH 7·2) containing 6 mM MgCl₂, 1 mM DTT, 50 mM KCl, 0·5 mM CaCl₂, 10% (v/v) glycerol, 200 mM NaCl, 20 mM imidazole] using a dialysis membrane (size 18; Wako).

Preparation of radioactively labelled DNA probes for gel shift assay and DNase I footprinting analysis. The probes YDFJ-Pc and YDFJ-Pn are 3'-end radioactively labelled probes of the coding and noncoding strands of the *ydfJ* upstream region, respectively. The probe PG3-MCS was used as a negative control in the gel shift assay. To construct YDFJ-Pc and YDFJ-Pn, a 196 bp DNA fragment containing the *ydfJ* promoter region was amplified from *B. subtilis* 168 chromosomal DNA, and the primers YDFJP-F and YDFJP-R. For the construction of PG3-MCS, a 135 bp fragment was amplified from pGEM-3Zf(+), and the primers PET-T7 and PB-M13Rev. These fragments were purified with a QIAquick gel extraction kit (Qiagen). YDFJ-Pc, YDFJ-Pn and PG3-MCS were digested with EcoRI and SalI, PstI and MunI and EcoRI, respectively. Accordingly, 170, 182 and 113 bp fragments were obtained. Five picomoles each of YDFJ-Pc, YDFJ-Pn and PG3-MCS DNAs were labelled by incubation at 10°C for 1 h with 1·85 MBq [α-³²P]dATP
(Amersham Biosciences) and 3 U of the Klenow fragment of DNA polymerase I (Takara Bio). Unincorporated nucleotides were removed using a MicroSpin S400HR column (Amersham Biosciences). The fragments were then ethanol precipitated, and the pellet was dissolved in 100 µl sterilized ultrapure water. The YDFJ-Pc, YDFJ-Pn and PG3-MCS DNAs were digested with *SalI*, *PstI* and *EcoRI*, respectively. After incubation at 37°C for 6 h, the restriction enzymes were completely removed with Micropure-EZ (Millipore) and the DNA was ethanol precipitated. The pellets were dissolved in 200 µl sterilized ultrapure water at a final concentration of 0·025 pmol µl⁻¹.

**Gel shift assay.** Radioactively labelled YDFJ-Pc and PG3-MCS DNAs were used. Binding reactions were performed with a fixed amount of probe DNA (0·025 pmol), 4 µg of poly(dI-dC) (Amersham Biosciences) and different amounts of h-YdfI (0, 6·25, 12·5, 18·75 and 25 pmol) at 20°C for 1 h in GB buffer [40 mM Tris/HCl (pH 7·2) containing 6 mM MgCl₂, 1 mM DTT, 50 mM KCl, 0·5 mM CaCl₂, 10% (v/v) glycerol, 200 mM NaCl, 20 mM imidazole, 1% (v/v) Tween 20] in 20 µl total volume. Two microlitres of loading dye [0·2% (w/v) bromophenol blue, 0·1% (w/v) xylene cyanol, 40% (v/v) glycerol, 60% (v/v) TBE buffer diluted to one fourth (TBE buffer: 50 mM Tris, 48·5 mM boric acid, 2 mM EDTA)] was added to each reaction mixture, and the samples were loaded onto an 8% native polyacrylamide gel [the ratio of acrylamide to bisacrylamide was 75 to 1 (w/w)]. The competition assay was performed with a fixed amount of radioactively labelled YDFJ-Pc (0·025 pmol), different amounts of non-labelled YDFJ-Pc (2·5, 5, 7·5 and 10 pmol), 4 µg poly(dI-dC), and a fixed amount of h-YdfI (25 pmol) at 20°C for 1 h in GB buffer in a total volume of 20 µl.

**DNase I footprinting analysis.** Radioactively labelled YDFJ-Pc (coding strand) and YDFJ-Pn (noncoding strand) were used in this analysis. Binding reactions were performed with a fixed amount of radioactive probe (0·125 pmol), 20 µg poly(dI-dC) and different amounts of h-YdfI (0, 62·5, 125, 187·5 and 250 pmol) at 20°C for 1 h in GB buffer in 100 µl total volume. Subsequently, 1 µl DNase I solution (3 units µl⁻¹; Takara Bio) was added to each reaction mixture, followed by incubation for 1 min at 20°C. DNase I digestion was terminated by the addition of 100 µl ice-cold DNase I stop solution [1·5 M sodium acetate (pH 5·2) containing 0·02 M EDTA, 0·02 µg µl⁻¹ sonicated salmon sperm DNA (Stratagene)], and then the mixture was subjected to phenol/chloroform/isoamyl alcohol (25:24:1, by vol.) extraction and ethanol precipitation. The pellet was dissolved in 5 µl loading buffer [0·01% (w/v) bromophenol blue, 0·01% (w/v) xylene cyanol, 1 mM EDTA, in 90% (v/v) formamide], and the solution was then separated on a denaturing 6% polyacrylamide gel. G and A+G Maxam–Gilbert sequence ladders, used as size markers, were prepared according to Maxam & Gilbert (1977, 1980).

**Computational analysis.** To search for paralogous and orthologous genes of *ydfH*, *ydfI* and *ydfJ*, SSDB (sequence similarity database; http://www.genome.ad.jp/kegg/ssdb/) in KEGG (Kyoto Encyclopedia of Genes and Genomes databases) was used. To search for genes with a YdfI putative recognition sequence in their promoter region, a web-based application, GRASP-DNA, constructed for whole-genome sequence searches was used (Schilling *et al.*, 2000; http://www2.genomatica.com/grasp-dna/).

**Disk diffusion experiments.** *B.* subtilis cells were inoculated into 200 ml LB medium at OD₆₀₀ 0·01 and then cultured at 37°C for 2·5 h up to approximately OD₆₀₀ 1·5. Two hundred microlitres of culture was spread onto LB agar plates. A filter paper disk with antibiotics then was placed on top of the plates. The plates were incubated at 37°C for 18 h, and the diameter of the blocking zones was measured with a digital slide gauge.

In this analysis, we used filter paper disks with thirty-one types of

---

antibiotic. Twenty-eight types of Sensi-disc (Becton Dickinson) were used, namely the fluoroquinolone group [levofloxacin (5 µg) and sparfloxacin (5 µg)], penicillin group [carbenicillin (100 µg), piperacillin (100 µg) and ampicillin (10 µg)], monobactam group [aztreonam (30 µg)], cephem group [ceftazidime (30 µg), cefepime (30 µg) and cefoperazone (75 µg)], aminoglycoside group [spectinomycin (100 µg), kanamycin (30 µg) and streptomycin (10 µg)], macrolide group [erythromycin (15 µg), azithromycin (15 µg), clarithromycin (15 µg), roxithromycin (15 µg) and josamycin (30 µg)], lincomycin group [lincomycin (2 µg) and clindamycin (2 µg)], tetracycline group [tetracycline (30 µg) and minocycline (30 µg)], polypeptide group [vancomycin (30 µg), bacitracin (10 U), polymyxin B (300 U) and colistin (10 µg)] and others [chloramphenicol (30 µg), rifampicin (5 µg) and fosfomycin (50 µg)], as well as an SN disk (Nissui Pharmaceutical) with teicoplanin (30 µg) (polypeptide group). We also used hand-made filter paper disks (6 mm diameter) with surfactin (50 µg) and fusidic acid (100 µg). Since the YDFHIΔSp strain was constructed by inserting the spectinomycin resistance gene into *ydfHI* loci, and the YDFJΔPM4 strain harboured the erythromycin resistance gene, we could not use spectinomycin and macrolide antibiotics for YDFHIΔSp and YDFJΔPM4, respectively.

---

## RESULTS

### Transcriptional analysis of the *ydfHI* genes of *B.* subtilis

The *B.* subtilis genome sequence shows that *ydfH* and *ydfI* are transcribed in the same direction. In addition, ρ-independent terminators are located downstream of *ydfI* (Fig. 1). To determine the transcriptional units and expression patterns of *ydfH* and *ydfI* in the wild-type strain 168 and the *ydfHI* double null mutant YDFHIΔSp, we performed Northern blot analysis using DIG-labelled *ydfH*- and

![Figure 1](https://i.imgur.com/1234567.png)

---

**Fig. 1.** Feature map of the *ydfHI* region. The largest arrows show the six deduced ORFs, and indicate their transcriptional and translational directions. Vertical open arrowheads indicate primer positions. The shaded bars indicate probe positions. White and black boxes indicate the regions of fragments ΔYDFH and ΔYDFI, respectively. The shaded arrow indicates the direction of insertion of a spectinomycin resistance gene to construct the *ydfHI* double null mutant YDFHIΔSp. The small black arrow indicates the direction of Pₛₚₐc insertion in YDFJΔPM4. Numbers indicate the position with respect to the origin of replication, coinciding with the base numbering start point.
The YdfHI system regulates ydfJ transcription

Fig. 2. Northern blot analysis of strains *ydfH* and *ydfI* in 168 and YDFHIΔSp. These strains were inoculated into 200 ml DSM at an OD<sub>600</sub> of 0·01. Cells were harvested at various time points to allow isolation of total RNA. *t<sub>-x</sub>* means *x* hours before *t<sub>0</sub>* (time of onset of sporulation). Ten micrograms of each RNA from strains 168 and YDFHIΔSp was loaded onto 1% (w/v) formaldehyde-agarose gels. Signals were detected using DIG-labelled *ydfH*- and *ydfI*-specific RNA probes. The exposure time to the X-ray film (RX-U; Fujifilm) for detection was 4·5 h. Arrows indicate the positions of mRNA signals. The rRNA positions are also indicated (23S, 2·93 kb 23S rRNA; 16S, 1·55 kb 16S rRNA).

*ydfI*-specific probes. The X-ray film required an exposure time of 4·5 h, which indicates that the amounts of the transcripts were very small. Fig. 2 shows that *ydfH*- and *ydfI*-specific probes both hybridized to 2·0 and 2·1 kb mRNAs during the exponential growth phase of strain 168. The result indicated the polycistronic transcription of *ydfH* and *ydfI*. Neither transcript was detected for the *ydfHI* double null mutant YDFHIΔSp. Incidentally, mRNA signals corresponding to the polycistronic transcription of *ydfH* and *ydfI* were not detected after *t<sub>0</sub>* (the time of onset of sporulation) (data not shown).

### *ydfJ* transcription is specifically induced by YdfI overexpression

It has been demonstrated that the overexpression of the response regulator in the background of deficiency of its cognate sensor kinase leads to the expression of its target genes (Kobayashi *et al.*, 2001; Ogura *et al.*, 2001). When the signal recognized by the sensor kinase is unknown, this feature is useful for analysing the association between a two-component system and its presumed target genes. In DNA microarray experiments where YdfI is overexpressed in the background of the *ydfH* disruptant, it has been reported that the YdfHI two-component system may regulate 27 genes including *ydfJ* (Kobayashi *et al.*, 2001). Thus, we performed Northern blot analysis to confirm that *ydfJ* transcription is specifically induced by YdfI overexpression.

*ydfJ* transcription was fairly weak under normal conditions (Fig. 3a). This transcription was not influenced by *ydfHI* disruption. To detect this transcription in Northern analysis, 20 μg RNA instead of 10 μg was loaded onto the gels. Moreover, more than 1 h was needed for exposure to the X-ray film.

We transformed the *ydfHI* double null mutant YDFHIΔSp with the multicopy plasmid pDG148ydfI. The plasmid pDG148ydfI was used for YdfI overexpression under the control of the IPTG-inducible *spac* promoter (Kobayashi *et al.*, 2001). The 0·6 kb mRNA signal corresponding to *ydfI*

monocistronic transcription was strongly induced by the addition of IPTG (Fig. 3b, lanes 4–6). In the absence of IPTG, *ydfJ* transcription was not detected (Fig. 3c, lanes 1–3). However, when YdfI was overexpressed on addition of IPTG, we detected a stronger 2·2 kb mRNA signal corresponding to the *ydfJ* monocistronic transcript (Fig. 3c, lanes 4–6). Ten minutes exposure to the X-ray film was sufficient to detect this transcript. Judging from the above results, it is clear that *ydfJ* transcription is regulated by the YdfHI two-component system.

#### Gel shift assays with histidine-tagged YdfI protein (h-YdfI) and the *ydfJ* promoter region

A gel shift assay was performed to examine the direct interaction between the YdfI protein and the region upstream of *ydfJ*. We used a histidine-tagged YdfI protein (h-YdfI) in this analysis. The h-YdfI protein (224 aa; *M<sub>r</sub>* 25 212) was expressed in the presence of IPTG (inducing conditions) in *E. coli* M15(pREP4, pQEYDFI), and purified by nickel-affinity chromatography (data not shown). An end-labelled DNA fragment (YDFJ-Pc, 170 bp) encompassing a region from –154 to +6 bp with respect to the 5′ end of *ydfJ* was used as a radioactive probe. In the presence of 6·25 pmol of h-YdfI, YDFJ-Pc exhibited a mobility shift (Fig. 4, lane 2). A clearer mobility shift of YDFJ-Pc was observed with an increase in the amount of h-YdfI. In the presence of 18·75 and 25 pmol of h-YdfI, almost all the probes formed a complex with h-YdfI (Fig. 4, lanes 4 and 5). Furthermore, we performed similar gel shift assays for 12 genes (*ydfE*, *ydgJ*, *ydjM*, *pyrR*, *proH*, *ysbA*, *rbsR*, *lctE*, *yqxL*, *ywcJ*, *ywbH* and *cydA*) that had been detected as candidate genes regulated by YdfHI as a result of DNA microarray analysis (Kobayashi *et al.*, 2001). Although the regions upstream of the 12 genes were selected as putative promoter regions and gel shift assays were performed, no gel shift was found for any of the genes (data not shown). It may be that these 12 genes are related to YdfHI indirectly. In this analysis, we focused only on genes that were indicated to have a direct interaction with the YdfI response regulator;

M. Serizawa and J. Sekiguchi

(a)  
**ydfJ DIG-labelled RNA probe**  
168 YDFHIA Sp  
1 2 3 4 5 6  

(b)  
**ydfI DIG-labelled RNA probe**  
IPTG: – +  
1 2 3 4 5 6  

(c)  
**ydfJ DIG-labelled RNA probe**  
IPTG: – +  
1 2 3 4 5 6  

Fig. 3. (a) Northern blot analysis of *ydfJ* in strains 168 and YDFHIA Sp. These strains were inoculated into 200 ml LB at an OD<sub>600</sub> of 0·01. Cells were harvested at optical densities of approximately 0·4, 0·9 and 1·8 to allow isolation of total RNA. Twenty micrograms of each RNA was loaded onto 1% (w/v) formaldehyde-agarose gels. Signals were detected with a DIG-labelled *ydfJ*-specific RNA probe. The exposure time to the X-ray film (RX-U; Fujifilm) for detection was 1 h. Arrows and bars indicate the positions of mRNA signals and rRNAs, respectively. (b, c) Northern blot analysis of *ydfI* (b) and *ydfJ* (c) transcripts in YDFHIA Sp harbouring plasmid pDG148ydfI, used for overexpression of *ydfI*. After inoculation of these strains into 200 ml LB medium containing kanamycin (10 μg ml<sup>-1</sup>) at an OD<sub>600</sub> 0·01, the cells were grown at 37°C for 1 h. IPTG was added to one of the flasks at a final concentration of 1 mM, and the cells were harvested after 1, 1·5 and 2 h. Ten micrograms of each RNA was loaded onto 1% (w/v) formaldehyde-agarose gels. Signals were detected with DIG-labelled *ydfI-* (b) and *ydfJ-* (c) specific RNA probes. The exposure times to the X-ray film (RX-U; Fujifilm) for detection were 10 s (b) and 10 min (c).

therefore, a detailed analysis was not performed for these 12 genes.

The addition of excess amounts of the non-radioactive YDFJ-Pc fragment suppressed radioactively labelled YDFJ-Pc

**Competition**  
**Non-Rl**  
**YDFJ-Pc**  
None h-YdfI h-YdfI None h-YdfI  
1 2 3 4 5 6 7 8 9 10 11 12  

h-YdfI  
and  
YDFJ-Pc  
complex  

Free  
YDFJ-Pc  

Free  
PG3-MCS  

complex formation (Fig. 4, lanes 6–9). These results suggest that complex formation between YDFJ-Pc and h-YdfI is caused by specific interaction. Moreover, h-YdfI did not bind to an unrelated 113 bp fragment PG3-MCS, which belongs to the multi-cloning site of pGEM-3Zf(+) (Fig. 4, lanes 10–12). Therefore, it seems reasonable to conclude that the formation of the complex between h-YdfI and YDFJ-Pc is specific.

Fig. 4. Gel shift assays of h-YdfI protein and YDFJ-Pc probe DNA including the *ydfJ* promoter region. Radioactively labelled YDFJ-Pc (0·025 pmol) (lanes 1–5) or PG3-MCS (lanes 10–12), and 4 μg poly(dI-dC) as the competitor DNA were incubated with different amounts of h-YdfI (lanes 1 and 10, 0 pmol; lane 2, 6·25 pmol; lanes 3 and 11, 12·5 pmol; lane 4, 18·75 pmol; lanes 5 and 12, 25 pmol). PG3-MCS was used as a negative control. A competition assay was performed with constant amounts of radioactively labelled YDFJ-Pc probe (0·025 pmol) and h-YdfI (25 pmol), and different amounts of nonlabelled YDFJ-Pc probe (lane 6, 2·5 pmol; lane 7, 5 pmol; lane 8, 7·5 pmol; lane 9, 10 pmol).

The YdfHI system regulates *ydfJ* transcription

DNase I footprinting analysis of the interaction of h-YdfI with the *ydfJ* promoter region and primer extension analysis of *ydfJ*

We carried out DNase I footprinting analysis to determine the h-YdfI-binding site(s), and primer extension analysis of *ydfJ* (Figs 5 and 6). Fig. 6 shows that the transcriptional initiation site of *ydfJ* is 'G' and we could not detect any other significant transcripts within the region of 144 bp upstream of the 5'-end of *ydfJ*. These results suggest the putative

promoter sequences (-35 and -10) of *ydfJ* are TTttCA and TAcAAT, respectively (capital letters indicating the sigma A consensus sequences). h-YdfI-protected sequences in both the coding and noncoding strands of the *ydfJ* promoter region are located just upstream of the putative *ydfJ* promoter sequence (-35 and -10) (Figs 5 and 6). Moreover, we found the tandem repeat sequences, GCCCaAAcGTAC and GCCCgAAgtGTAC, in the h-YdfI-protected region (from -67 to -35 bp with respect to the *ydfJ* transcriptional initiation site). The sequence GCCCRaAYGTAC, which is the putative recognition sequence of YdfI, was called the 'YdfI-box'. The above result suggests that the direct binding of YdfI to the *ydfJ* promoter region regulates *ydfJ* transcription.

Disk diffusion analysis for detecting sensitivity to various antibiotics

Since YdfJ belongs the RND superfamily, there is a possibility that YdfJ is associated with resistance to antibiotics. Thus, we investigated the sensitivity to antibiotics of strains 168 (wild-type), YDFHIAΔSp (*ydfHI* null mutant) and YDFJΔPM4 (*ydfJ* disruptant). Filter paper disks with 31 types of antibiotic, described in Methods, were used in this analysis. However, the results obtained were contrary to our expectations: no significant difference in sensitivity to the 31 types of antibiotic between the three strains (168, YDFHIAΔSp and YDFJΔPM4) was detected (data not shown).

DISCUSSION

Based on the classification of sensor kinases (in terms of the sequence around the phosphorylated histidine) and response regulators (in terms of the sequence of the output domain), YdfH and YdfI have been classified into Class II and NarL groups, respectively (Fabret *et al.*, 1999). The YdfH sensor kinase (407 aa; *M*<sub>r</sub> 46 345) has five transmembrane domains in its N-terminal region, which suggests that YdfH is a membrane-anchored protein. The C-terminal region of YdfH consists of the His PTase (histidine phosphotransferase) domain. The YdfI response regulator (213 aa; *M*<sub>r</sub> 23 826) consists of the PA (phosphorylated aspartate) domain in the N-terminal region, and an output domain in the C-terminal region. A gel shift was observed with the h-YdfI protein and a DNA fragment containing the upstream region of *ydfJ* (YDFJ-Pc). Since the molar ratio of h-YdfI to YDFJ-Pc for stable binding was more than 750 (Fig. 4), more efficient binding may be found after phosphorylation of h-YdfI with a chemical phosphodonor such as acetylphosphate.

DNase I footprinting analysis revealed the region protected by h-YdfI (Fig. 5). The protected area was from -67 to -35 bp with respect to the *ydfJ* transcriptional initiation site (Fig. 6). We found two tandem repeat sequences consisting of conserved 12-mer sequences (YdfI-box; GCCCRaA-YGTAC). Accordingly, we presume that the tandem repeat sequence is the putative YdfJ recognition sequence. It has

Fig. 5. DNase I footprinting analysis of *ydfJ* promoter region. The left and right panels are DNase I footprints of the 3'-end-labelled coding (YDFJ-Pc) and noncoding (YDFJ-Pn) probe DNAs. Arrowheads and vertical lines on the right side of each panel indicate the enhanced cleavage sites and protected regions, respectively. The numbers are with respect to the *ydfJ* transcriptional initiation site. The amount of h-YdfI protein in lanes 1–6 was 0, 62·5, 125, 187·5, 250 and 0 pmol, respectively. Lanes G and A+G indicate Maxam–Gilbert sequence ladders.

M. Serizawa and J. Sekiguchi

Noncoding strand  
ATTG CCATGCAAAA AGGTATTCTG ACAATAGACA ACTAAACTGC ATATTTGAAA ATTGCCAAAC GTIACATGCC GGAATGTACG TTTTTTICAT  

Coding strand  
TAAC GGTACGGTTT TCCATAAGAC TGTTATCTGT TGATTTGACG TATAAACAAA AACGGGTTTG CATGIAACGG GCITIACATG C AAAAAAAGTA  

I A M Q K G I L T I D N -67 -61 -40 -35  

ydfI  

P  
ydfJ  

Noncoding strand  
TTCATTGTCA ACTAGAATGA GAAAAGAATG TGTGTTGAAA GGAGATTATC ATGTCA  

Coding strand  
AAGTAACAGT TGATGTTACT CTTCGTACA CTAGTTCGTT ACACAACTTT CCTCTAATAG TACAGT  

(-10) RBS ydfJ  

Fig. 6. Summary of the results of DNase I footprinting and primer extension analyses. The nucleotide sequences of the coding and noncoding strands of the *ydfJ* promoter region are given. The region exhibiting affinity to h-Ydfl is boxed in black. Two horizontal arrows with numbers denote the tandem repeat sequences consisting of the two conserved 12-mer sequences (GCCCRRAAYGTAC). The numbers below the sequence are with respect to the *ydfJ* transcriptional initiation site. Converging arrowheads indicate the putative *ydfI* terminator. A potential ribosome-binding site (RBS) is denoted by an open box. Vertical arrowheads and a bar indicate the enhanced cleavage sites and the 3′ end of *ydfI*, respectively. Putative *ydfJ* promoter sequences (-35 and -10) are underlined. The -35 region (TTTTCA) and -10 region (TACAAT), with a gap of 13 bp, are similar to those of the σ<sup>A</sup> consensus sequence (TTGACA for the -35 region and TATAAT for the -10 region, with a gap of 17 bp) (Haldenwang, 1995). ‘+1’ indicates the transcriptional initiation site of *ydfJ* detected by primer extension analysis. Total RNAs (40 μg) extracted from *B.* subtilis YDFHIΔSp harbouring pDG148ydfI (grown in the presence of IPTG for overexpression of Ydfl) were used as RNA samples (they were the same RNAs as used in Fig. 3c, lane 5). Signals were detected with <sup>32</sup>P-labelled primer YDFJ-PEX, which was designed based on the sequence from +54 to +72 with respect to the 5′-end of *ydfJ*. Dideoxy DNA sequencing reaction mixtures with the same primer were electrophoresed in parallel (lanes C, G, T and A) as size markers.

already been reported that in five systems (PhoPR, YxdJK, CitST, DctSR and LiaRS) of *B.* subtilis, tandem or direct repeat sequences are located within the target sites of their response regulators (DBTBS: http://dbtbs.hgc.jp/; Joseph *et al.*, 2004; Yamamoto *et al.*, 2000; Asai *et al.*, 2000; Mascher *et al.*, 2004).

To find genes other than *ydfJ* regulated by YdfHI, a whole-genome sequence search was performed using the web-based software GRASP-DNA. Seq-1 (GCCCAAACGTAC) or seq-2 (GCCCGAATGTAC), which are found in the putative *cis*-acting regulatory sequences of *ydfJ*, were used as query sequences in the GRASP-DNA search. Then, we studied regions in which two or more query sequences existed in tandem in the region upstream of the genes detected in the GRASP-DNA search. Two sequences similar to the Ydfl-box were found to exist in tandem within the region upstream of *yfkC*, when seq-2 was used as a query sequence. Moreover,

the σ<sup>A</sup> consensus sequence was found upstream of *yfkC*. If YdfHI controls *yfkC*, the monocistronic transcription of *yfkC* may be specifically induced by Ydfl overexpression. Therefore, we performed Northern blot analysis of *yfkC* under Ydfl overexpression conditions in *ydfHI* double null mutant YDFHIΔSp. However, an mRNA signal corresponding to the monocistronic transcription of *yfkC* was not detected.

Since YdfJ belongs to the RND superfamily, there is a possibility that YdfJ is associated with resistance to antibiotics. However, we have not yet found any antibiotics that have significant effects on YDFHIΔSp (*ydfHI* null mutant) and YDFJΔPM4 (*ydfJ* disruptant) strains. There may be an association between YdfJ and the transport of some noxious compounds (for example, heavy-metal ions and dyes) other than antibiotics. Moreover, it may be necessary to investigate the association between YdfJ and the transport
and biosynthesis of lipids. YdfJ belongs to the HAE2 family, which is a subfamily of the RND superfamily (Tseng *et al.*, 1999). The HAE2 family is specific to Gram-positive bacteria and is involved in not only drug resistance but also the transport and biosynthesis of lipids (transport classification database; http://tcdb.ucsd.edu/tcdb/). For example, *mmpL7* and *mmpL8* of *Mycobacterium tuberculosis* are required for the efficient translocation of phthiocerol dimycocerosate (Camacho *et al.*, 2001) and for sulfolipid-1 biosynthesis (Converse *et al.*, 2003), respectively.

CAC3431 of *Clostridium acetobutylicum* ATCC 824 shows a high amino acid sequence similarity with *ydfJ* (Smith–Waterman similarity score, 2026; identity 45·4%). CAC3431, as well as *ydfJ*, belongs to the HAE2 family, and is predicted to function as a multidrug-efflux pump (transportDB; http://www.membranetransport.org/). Moreover, CAC3429 and CAC3430, which are located adjacent to CAC3431, have high amino acid sequence similarities to YdfI (response regulator) and YdfH (sensor kinase), respectively. Consequently, it is quite likely that CAC3431 is regulated by a two-component system consisting of CAC3429 and CAC3430 in *C.* *acetobutylicum* ATCC 824. The gene organization of *C.* *acetobutylicum* was similar to that of *ydfH-ydfI* (two-component system) and *ydfJ* (RND superfamily member), and it was the only case in the sequence similarity database (SSDB: 160 bacteria are currently registered).

In this study, we have demonstrated that *ydfJ* of the RND superfamily is regulated by the YdfHI two-component system. YdfI (the response regulator) directly binds to the *ydfJ* promoter region, and tandem repeat sequences are present in the YdfI-binding site. However, we could not identify the signal recognized by the YdfH sensor kinase. Most RND superfamily members have a role in resistance to a wide range of noxious compounds. As a future study, it will be necessary to investigate the association between YdfJ and the transport of lipids, heavy-metal ions and dyes.

## ACKNOWLEDGEMENTS

We are grateful to M. Watanabe for the construction of pQEYDFI. We also thank H. Yamamoto and S. Tojo (Fukuyama University) for kind technical advice, and A. Chaki and K. Kodama for experimental assistance. For radioisotope and DNA sequence analyses, some equipment in the Gene Research Center of Shinshu University was used. This work was supported by a Grant-in-aid for Scientific Research on Priority Areas, Genome Biology (12206005) (to J.S.), Grant-in-aid for JSPS Fellows (15061544) (to M.S.) from the Ministry of Education, Culture, Sports, Science and Technology of Japan, and JSPS Research Fellowships for Young Scientists (to M.S.) from the Japan Society for the Promotion of Science.

## REFERENCES

Aguilar, P. S., Hernandez-Arriaga, A. M., Cybulski, L. E., Erazo, A. C. & de Mendoza, D. (2001). Molecular basis of thermosensing: a two-component signal transduction thermometer in *Bacillus subtilis*. EMBO J 20, 1681–1691.

Asai, K., Baik, S. H., Kasahara, Y., Moriya, S. & Ogasawara, N. (2000). Regulation of the transport system for $C_{4}$-dicarboxylic acids in *Bacillus subtilis*. Microbiology 146, 263–271.

Camacho, L. R., Constant, P., Raynaud, C., Laneelle, M. A., Triccas, J. A., Gicquel, B., Daffe, M. & Guilhot, C. (2001). Analysis of the phthiocerol dimycocerosate locus of *Mycobacterium tuberculosis*. Evidence that this lipid is involved in the cell wall permeability barrier. J Biol Chem 276, 19845–19854.

Converse, S. E., Mougous, J. D., Leavell, M. D., Leary, J. A., Bertozzi, C. R. & Cox, J. S. (2003). MmpL8 is required for sulfolipid-1 biosynthesis and *Mycobacterium tuberculosis* virulence. Proc Natl Acad Sci U S A 100, 6121–6126.

Doan, T., Servant, P., Tojo, S., Yamaguchi, H., Lerondel, G., Yoshida, K., Fujita, Y. & Aymerich, S. (2003). The *Bacillus subtilis ywkA* gene encodes a malic enzyme and its transcription is activated by the YufL/YufM two component system in response to malate. Microbiology 149, 2331–2343.

Fabret, C., Feher, V. A. & Hoch, J. A. (1999). Two-component signal transduction in *Bacillus subtilis*: how one organism sees its world. J Bacteriol 181, 1975–1983.

Fukuchi, K., Kasahara, Y., Asai, K., Kobayashi, K., Moriya, S. & Ogasawara, N. (2000). The essential two-component regulatory system encoded by *yycF* and *yycG* modulates expression of the *ftsAZ* operon in *Bacillus subtilis*. Microbiology 146, 1573–1583.

Grkovic, S., Brown, M. H. & Skurray, R. A. (2002). Regulation of bacterial drug export systems. Microbiol Mol Biol Rev 66, 671–701.

Haldenwang, W. G. (1995). The sigma factors of *Bacillus subtilis*. Microbiol Rev 59, 1–30.

Hulett, F. M. (1996). The signal-transduction network for Pho regulation in *Bacillus subtilis*. Mol Microbiol 19, 933–939.

Hyyrylainen, H. L., Bolhuis, A., Darmon, E. & 8 other authors (2001). A novel two-component regulatory system in *Bacillus subtilis* for the survival of severe secretion stress. Mol Microbiol 41, 1159–1172.

Ishikawa, S., Hara, Y., Ohnishi, R. & Sekiguchi, J. (1998). Regulation of a new cell wall hydrolase gene, *cwlF*, which affects cell separation in *Bacillus subtilis*. J Bacteriol 180, 2549–2555.

Joseph, P., Guiseppi, A., Sorokin, A. & Denizot, F. (2004). Characterization of the *Bacillus subtilis* YxdJ response regulator as the inducer of expression for the cognate ABC transporter YxdLM. Microbiology 150, 2609–2617.

Kobayashi, K., Ogura, M., Yamaguchi, H., Yoshida, K., Ogasawara, N., Tanaka, T. & Fujita, Y. (2001). Comprehensive DNA microarray analysis of *Bacillus subtilis* two-component regulatory systems. J Bacteriol 183, 7365–7370.

Mascher, T., Zimmer, S. L., Smith, T. A. & Helmann, J. D. (2004). Antibiotic-inducible promoter regulated by the cell envelope stress-sensing two-component system LiaRS of *Bacillus subtilis*. Antimicrob Agents Chemother 48, 2888–2896.

Maxam, A. M. & Gilbert, W. (1977). A new method for sequencing DNA. Proc Natl Acad Sci U S A 74, 560–564.

Maxam, A. M. & Gilbert, W. (1980). Sequencing end-labeled DNA with base-specific chemical cleavages. Methods Enzymol 65, 499–560.

Msadek, T., Kunst, F. & Rapoport, G. (1995). A signal transduction network in *Bacillus subtilis* includes the DegS/DegU and ComP/ComA two-component systems. In *Two-Component Signal Transduction*, pp. 447–471. Edited by J. A. Hoch & T. J. Silhavy. Washington, DC: American Society for Microbiology.

Ogura, M., Yamaguchi, H., Yoshida, K., Fujita, Y. & Tanaka, T. (2001). DNA microarray analysis of *Bacillus subtilis* DegU, ComA and PhoP regulons: an approach to comprehensive analysis of *B.* *subtilis* two-component regulatory systems. Nucleic Acids Res 29, 3804–3813.

http://mic.sgmjournals.org   1777
M. Serizawa and J. Sekiguchi

Ohki, R., Giyanto, Tateno, K., Masuyama, W., Moriya, S., Kobayashi, K. & Ogasawara, N. (2003). The BceRS two-component regulatory system induces expression of the bacitracin transporter, BceAB, in *Bacillus subtilis*. *Mol Microbiol* 49, 1135–1144.

Schaeffer, P., Millet, J. & Aubert, J. P. (1965). Catabolic repression of bacterial sporulation. *Proc Natl Acad Sci U S A* 54, 704–711.

Schilling, C. H., Held, L., Torre, M. & Saier, M. H., Jr (2000). GRASP-DNA: a web application to screen prokaryotic genomes for specific DNA-binding sites and repeat motifs. *J Mol Microbiol Biotechnol* 2, 495–500.

Serizawa, M., Yamamoto, H., Yamaguchi, H., Fujita, Y., Kobayashi, K., Ogasawara, N. & Sekiguchi, J. (2004). Systematic analysis of SigD-regulated genes in *Bacillus subtilis* by DNA microarray and Northern blotting analyses. *Gene* 329, 125–136.

Stock, J. B., Surette, M. G., Levit, M. & Park, P. (1995). Two-component signal transduction systems: structure-function relationships and mechanisms of catalysis. In *Two-Component Signal Transduction*, pp. 25–52. Edited by J. A. Hoch & T. J. Silhavy. Washington, DC: American Society for Microbiology.

Sun, G., Sharkova, E., Chesnut, R., Birkey, S., Duggan, M. F., Sorokin, A., Pujic, P., Ehrlich, S. D. & Hulett, F. M. (1996). Regulators

of aerobic and anaerobic respiration in *Bacillus subtilis*. *J Bacteriol* 178, 1374–1385.

Tanaka, K., Kobayashi, K. & Ogasawara, N. (2003). The *Bacillus subtilis* YufLM two-component system regulates the expression of the malate transporters MeaN (YufR) and YflS, and is essential for utilization of malate in minimal medium. *Microbiology* 149, 2317–2329.

Tekaia, F., Gordon, S. V., Garnier, T., Brosch, R., Barrell, B. G. & Cole, S. T. (1999). Analysis of the proteome of *Mycobacterium tuberculosis* in silico. *Tuber Lung Dis* 79, 329–342.

Tseng, T. T., Gratwick, K. S., Kollman, J., Park, D., Nies, D. H., Goffeau, A. & Saier, M. H., Jr (1999). The RND permease superfamily: an ancient, ubiquitous and diverse family that includes human disease and development proteins. *J Mol Microbiol Biotechnol* 1, 107–125.

Vagner, V., Dervyn, E. & Ehrlich, S. D. (1998). A vector for systematic gene inactivation in *Bacillus subtilis*. *Microbiology* 144, 3097–3104.

Yamamoto, H., Murata, M. & Sekiguchi, J. (2000). The CitST two-component system regulates the expression of the Mg-citrate transporter in *Bacillus subtilis*. *Mol Microbiol* 37, 898–912.

Yoshida, K., Aoyama, D., Ishio, I., Shibayama, T. & Fujita, Y. (1997). Organization and transcription of the myo-inositol operon, *iol*, of *Bacillus subtilis*. *J Bacteriol* 179, 4591–4598.
