
Chemistry & Biology, Vol. 12, 665–675, June, 2005, ©2005 Elsevier Ltd All rights reserved. DOI 10.1016/j.chembiol.2005.04.010

# Biosynthesis of the Unique Amino Acid Side Chain of Butirosin: Possible Protective-Group Chemistry in an Acyl Carrier Protein-Mediated Pathway

Yanyan Li, Nicholas M. Llewellyn, Ramesh Giri, Fanglu Huang, and Jonathan B. Spencer*

Department of Chemistry  
University of Cambridge  
Lensfield Road  
Cambridge CB2 1EW  
United Kingdom

## Summary

Butirosins A and B are naturally occurring aminoglycoside antibiotics that have a (2S)-4-amino-2-hydroxybutyrate (AHBA) side chain. Semisynthetic addition of AHBA to clinically valuable aminoglycoside antibiotics has been shown both to improve their pharmacological properties and to prevent their deactivation by a number of aminoglycoside-modifying enzymes involved in bacterial resistance. We report here that the biosynthesis of AHBA from L-glutamate, encoded within a previously identified butirosin biosynthetic gene cluster, proceeds via intermediates tethered to a specific acyl carrier protein (ACP). Five components of the pathway have been purified and characterized, including the ACP (BtrI), an ATP-dependent ligase (BtrJ), a pyridoxal phosphate-dependent decarboxylase (BtrK), and a two-component flavin-dependent monooxygenase system (BtrO and the previously unreported BtrV). The proposed biosynthetic pathway includes a γ-glutamylolation of an ACP-derived γ-aminobutyrate intermediate, possibly a rare example of protective group chemistry in biosynthesis.

## Introduction

Aminoglycoside antibiotics constitute a clinically valuable class of compounds including kanamycin, neomycin, and gentamicin. The bactericidal activity of such antibiotics derives from their affinity for the bacterial 30S ribosomal subunit and their consequent interference with protein synthesis by induction of codon misreading and inhibition of translocation [1, 2]. Aminoglycoside antibiotics are most commonly used to treat infections by Gram-negative bacilli [3], but they also show potentially useful activity against human immunodeficiency virus (HIV) [4, 5] and inhibit catalytic RNAs such as the group I intron, the hammerhead ribozyme, and the hepatitis delta virus (HΔV) ribozyme [6]. Aminoglycoside resistance among target pathogens [7, 8] is conferred in most cases by the expression of at least one enzyme that chemically modifies the antibiotic molecule [9] by O-phosphorylation, O-nucleotidylylation, or N-acetylation, thereby weakening rRNA binding. This has stimulated the search for semisynthetic

*Correspondence: jbs20@cam.ac.uk

aminoglycoside analogs, such as amikacin, (Figure 1) which can overcome these resistance mechanisms [10]. An attractive alternative approach to the development of new, more potent antibiotics is the genetic manipulation of existing biosynthetic pathways to incorporate beneficial structural modifications. To pursue this, we have sought to elucidate the biosynthetic pathway for butirosin.

The butirosins (Figure 1) from *Bacillus circulans* [11] contain a 4,5-disubstituted 2-deoxystreptamine (2-DOS) core with a neosamine C ring at 4-O and either a xylose (Butirosin A) or ribose (Butirosin B) sugar at 5-O. They are unique among naturally occurring aminoglycosides in that they bear a (2S)-4-amino-2-hydroxybutyryl (AHBA) moiety on the C-1 amine of the 2-DOS ring. This modification has been shown to confer resistance against several aminoglycoside-modifying enzymes [12]. Indeed, the butirosins have demonstrated activity toward many strains that have developed resistance to other structurally similar aminoglycosides [11, 13]. Although the butirosins are not themselves clinically useful drugs, recognition of the valuable AHBA modification has led to successful synthetic alteration of kanamycin and dibekacin to produce amikacin and arbekacin, respectively. The C-1 amine is a frequent target of aminoglycoside *N*-acyltransferases, and the AHBA modification prevents drug deactivation by this class of enzymes while also improving the pharmacological profile of the drug beyond that of the parent molecule. The biosynthesis of AHBA has been the subject of considerable speculation, and a partial gene cluster for butirosin biosynthesis containing 17 ORFs (*btrA-btrQ*) in a region of 18 kbp has been identified in *B. circulans* [14]. However, the enzymatic pathway leading to AHBA and to its integration into the complete butirosin molecule has not been clearly characterized. Feeding studies have shown that L-glutamate and γ-aminobutyrate (GABA) are incorporated into the butirosins in *B. circulans* but not AHBA, suggesting that free AHBA is not a precursor of the butirosins [15]. This finding contradicts an earlier report suggesting the direct addition of AHBA to the DOS amine based on an apparent in vitro enzymatic AHBA-dependent ATP-PPi exchange in extracts of butirosin-producing *B. circulans* [16]. The observation of ATP-PPi exchange is suggestive of carboxyl-group activation and, possibly, amide-bond formation as seen in nonribosomal peptide synthetases. In this paper, we report the heterologous expression and functional characterization of the enzymes responsible for the synthesis of AHBA via acyl carrier protein (ACP)-mediated intermediates.

## Results

A gene cluster putatively responsible for butirosin biosynthesis was previously reported, and three genes, *btrI*, *btrJ*, and *btrK* were proposed to encode proteins involved in the synthesis of AHBA based on the sim-

Chemistry & Biology
666

![Molecular Structures of Aminoglycoside Antibiotics](image1.png)

Figure 1. Molecular Structures of Aminoglycoside Antibiotics

ilarity of their deduced peptide sequences to those of known enzymes [14]. Comparison with the neomycin biosynthetic gene cluster [17] shows that btrG, btrH, btrl, btrJ, btrK, and btrO are unique to the butirosin cluster (Figure 2), suggesting that some or all of these genes relate to the synthesis and transfer of AHBA, which is not present in neomycin. Further, gene walking upstream of the btrA gene in the butirosin cluster has now revealed another open reading frame unique to the butirosin cluster, btrV. The genes btrl, btrJ, btrK, btrO, and btrV have been successfully amplified from the genomic DNA of B. circulans NR3312 and cloned into pET-28(a)+ with appropriate restriction sites. Proteins were heterologously expressed in Escherichia coli BL-

![Butirosin Biosynthetic Gene Cluster in Bacillus circulans](image2.png)
![Neomycin Biosynthetic Gene Cluster in Streptomyces fradiae](image3.png)

- **2-deoxy-scyllo-inosose synthase**
- **aminomutase**
- **glycosyltransferase**
- **sugar oxidoreductase**
- **L-glutamine:2-deoxy-scyllo-inosose aminotransferase**
- **purine ribosyltransferase**
- **resistance genes**
- **enzymes for (2S)-2-hydroxy-4-aminobutyric acid synthesis**
- **alcohol dehydrogenase**
- **ABC transporter**
- **phosphomutase**
- **regulatory or unknown function**
- **nucleotidyltransferase**

| ORF          | Size (amino acids) | Proposed function / homology         | ORF          | Size (amino acids) | Proposed function / homology           |
|--------------|---------------------|--------------------------------------|--------------|---------------------|----------------------------------------|
| **btrG**     | 156                 | Unknown                              | **btrH**     | 302                 | Unknown                                |
| **btrl**     | 87                  | acyl-carrier protein                 | **btrJ**     | 419                 | biotin carboxylase                     |
| **btrK**     | 428                 | ornithine decarboxylase              | **btrO**     | 341                 | FMN-dependent monooxygenase            |
| **btrV**     | 226                 | FMN-NAD(P)H oxidoreductase           |              |                     |                                        |

Figure 2. Comparison of Butirosin and Neomycin Gene Clusters

Biosynthesis of (2S)-4-Amino-2-Hydroxybutyrate
667

A
Relative Abundance
100
50
30 32 34 36 38 40
Time (min)
33.24
37.11
RT = 37.11
Relative Abundance
100
50
11500 11600 11700 11800 11900 12000
mass
11812.0
11989.0

B
Relative Abundance
100
50
30 32 34 36 38 40
Time (min)
33.46
apo-Btrl
Relative Abundance
100
50
12000 12100 12200 12300 12400 12500
mass
12153.0
RT = 33.46
12329.0

Figure 3. 4′-Phosphopantetheinylation of Btrl

LC and MS traces of (A) apo-Btrl and (B) holo-Btrl produced by incubation of apo-Btrl with Sfp and CoA.

21(DE3) with N-terminal His₆ tags. Proteins were initially purified by nickel-affinity chromatography and further purified to homogeneity as necessary.

Characterization of Btrl

Carrier proteins such as acyl, peptidyl, and aryl carrier proteins (ACPs/PCPs/ArCPs) are central entities of several multifunctional enzyme complexes, including fatty acid synthases (FASs), polyketide synthases (PKSs), and nonribosomal polypeptide synthetases (NRPSs) [18, 19]. The functional forms of these carrier proteins bear an essential 4′-phosphopantetheine (4′-PP) prosthetic arm covalently bound to an active site serine residue. The terminal cysteamine thiol of the 4′-PP moiety serves as a nucleophile to attack the activated acyl group, thereby linking the substrate and the carrier protein via a thioester bond. Btrl is a small protein of 87 amino acids (9.8 kDa) that exhibits homology to several ACPs from PKS systems and conserves the signature LGXDSX sequence of the 4′-PP binding domain. Btrl is most homologous to an ACP domain from a mixed type I PKS/NRPS in a symbiont bacterium of Paederus fuscipes beetles (33% identity, 58% similarity) [20] and to a putative methoxymalonyl-ACP involved in geldanamycin biosynthesis in Streptomyces hygroscopicus (34% identity, 57% similarity) [21]. Btrl is also distantly related to a D-alanyl carrier protein involved in lipoteichoic-acid biosynthesis [22]. Btrl does not demonstrate significant homology to other carrier proteins (PCPs/ArCPs); this is unsurprising because sequence analysis has revealed that ACPs share little similarity

with PCPs beyond the highly conserved region around the conserved serine residue of the 4′-PP binding domain to which 4′-PP transferases (PPTases) transfer 4′-PP from coenzyme A (CoASH) [23].

For confirmation that Btrl is an acyl carrier protein, in vivo and in vitro phosphopantetheinylation assays were carried out to test whether Btrl is a substrate for PPTases. N-terminally His₆-tagged Btrl was well expressed in E. coli with high yields (10 mg/l culture). LC-ESI-MS confirmed that the majority of the purified Btrl has a molecular mass of 11,812.0 Da (Figure 3A), corresponding to the apo-form (calculated at 11,813.25 Da, His₆ tag inclusive). Less than 10% was shown to be in the active holo-form. Thus, it appears that Btrl is a poor substrate of endogenous E. coli PPTases such as AcpS [24]. Therefore, the broad-spectrum PPTase Sfp from B. subtilis, which exhibits activity toward PCPs as well as ACPs in both primary and secondary metabolism [25], was chosen for the phosphopantetheinylation assay. Sfp was overexpressed in E. coli and purified to near homogeneity as described in Experimental Procedures. Upon incubation with CoASH and Sfp for 15 min, apo-Btrl was almost fully converted to the holo-form, having a molecular mass of 12,153.0 Da in accordance with the calculated 12,152.57 Da as shown by LC-ESI-MS (Figure 3B). Btrl was coexpressed with Sfp in E. coli in order to prove that it is a substrate of PPTase in vivo as well. The coexpression vector was derived from pET-28a(+) with each gene under the control of its own T7 promoter. The coexpressed Btrl yielded one major peak (>95%) by LC-ESI-MS, corresponding to the holo-form, and a minor peak corresponding to the apo-form.

Chemistry & Biology
668

![Figure 4. γ-L-Glutamylation of BtrI](image.png)

**Figure 4. γ-L-Glutamylation of BtrI**

LC and MS traces of γ-L-glutamyl-S-BtrI produced by incubation of holo-BtrI with BtrJ and L-glutamate.

corresponding to the apo-form. These results demonstrate that the PPTase Sfp can recognize apo-BtrI as a substrate and transfer the 4′-PP moiety from CoASH into holo-BtrI both in vivo and in vitro.

### Activation and Loading of L-Glutamate

The deduced amino acid sequence of BtrJ (419 amino acids, 48.6 kDa) shows weak local similarity to that of biotin carboxylase (32% identity) [26] and of the large subunit of carbamoyl phosphate synthase (26% identity) [27]. Both enzymes belong to the growing ATP-grasp superfamily as classified by their three-dimensional structures. Other members of this superfamily include D-alanine:D-alanine ligase [28], glutathione synthetase [29], glycine ribonucleotide transformylase [30], and succinyl-CoA synthetase [31]. Despite low sequence homology, enzymes in this superfamily exhibit a characteristic motif for ATP binding and invariably possess ATP-dependent carboxyl-amine or carboxyl-thiol ligase activity. The reaction mechanism is proposed to involve the generation of an acylphosphate intermediate via hydrolysis of ATP to ADP [28, 29].

The N-terminal His₆-tagged BtrJ is a monomeric protein with a molecular mass of 50,962.0 Da (calculated at 50,969.05) as measured by LC-ESI-MS. The purified protein was liable to aggregation in solution, and the presence of dithiothreitol (DTT) was found to be necessary to prevent this. BtrJ was initially proposed to catalyze formation of the amide bond between AHBA and ribostamycin, but when BtrJ was incubated with ATP, AHBA, and ribostamycin in the presence of Mg²⁺, no production of butirosin was detectable, apparently indicating that BtrJ is involved in a different ligation reaction. Because recent feeding studies revealed that labeled L-glutamate and GABA but not AHBA were incorporated into butirosin [15], we reasoned that BtrJ might catalyze activation of a carboxyl precursor and its subsequent transfer to holo-BtrI, yielding an acyl-S-BtrI product. The ADP-forming activity of BtrJ was coupled to the reactions of pyruvate kinase (PK) and NADH-dependent lactate dehydrogenase (LDH) and measured spectrophotometrically to test this hypothesis. A significant decrease in absorbance at 340 nm was observed only when L-glutamate was added to the reaction mixture, which contained 1 mM ATP, 5 mM MgCl₂, 5 mM MnCl₂, 50 mM KCl, BtrJ, and the PK/LDH coupling components. Incubation with GABA, AHBA, or D-glutamate did not lead to significant spectral changes compared to a blank control. Divalent ions, such as Mg²⁺ or Mn²⁺, are essential for the L-glutamate-dependent ATPase activity of BtrJ, and the presence of Mn²⁺ was found to stimulate the reaction rate. These results imply the generation of a glutamyl-phosphate intermediate. Loading of L-glutamate onto BtrI was conducted in a 50 μl reaction composed of 10 mM L-glutamate, 5 mM ATP, 5 mM Mg²⁺, 5 mM Mn²⁺, 50 mM KCl, holo-BtrI, and BtrJ. The mixture was analyzed by LC-ESI-MS and revealed a new peak eluting 1 min earlier than holo-BtrI (Figure 4). It was observed to have a molecular mass of 12,281.0 Da, matching closely that of L-Glu-S-BtrI (calculated at 12,281.68 Da). The yield of glutamyl-ation of BtrI was ~30% under these conditions. Because the pathway to AHBA from glutamate is most likely to include a decarboxylation step, we reasoned that the α-carboxyl group of L-glutamate would remain free while the γ-carboxyl group would be thioester bound to BtrI.

### Decarboxylation and Hydroxylation of the Acyl-ACP

FMNH₂-dependent two-component monooxygenase systems have been reported to catalyze monooxygenation of aliphatic substrates. Members of such systems include bacterial luciferase [32], pristinamycin II_A synthase [33], nitrilotriacetate monooxygenase [34], and alkanesulfonate monooxygenase [35], which are involved in chemiluminescence reactions, antibiotic biosynthesis, xenobiotic degradation, and sulfur metabolism, respectively. Each enzyme system features a two-component organization, composed of an oxidoreductase that catalyzes the reduction of FMN with NAD(P)H as an electron donor and a monooxygenase that hydroxylates the substrate in the presence of FMNH₂ and molecular oxygen. The oxidoreductase provides the monooxygenase with the necessary reduced FMN cosubstrate. Although low sequence similarity is found between monooxygenases in these systems at the amino acid level, available three-dimensional structures, including luciferase from *Vibrio harveyi* (LuxAB) and alkanesulfonate monooxygenase from *E. coli* (SsuD), exhibit strikingly similar structural organization distinct from the structures of enzymes that utilize FMN as a prosthetic group and not as a substrate. BtrO (341 aa, 38.7 kDa) exhibits 26% sequence identity to the alkanesulfonate monooxygenase SsuD in *B. subtilis*, which is involved in oxygen-mediated cleavage of sulfonate to sulfite via an α-hydroxy intermediate. BtrO is also predicted to be structurally related to SsuD and

Biosynthesis of (2S)-4-Amino-2-Hydroxybutyrate
669

LuxAB. The gene *btrV* resides 20 kb upstream to the gene *btrO* in the butirosin cluster and encodes a protein homologous to an NAD(P)H oxidoreductase in *Bacillus cereus* (48% identity). Therefore, it is proposed that BtrO and BtrV might together constitute a two-component monooxygenase system.

The purified BtrV was found to be a homotetramer with a monomeric molecular mass of 26.1 kDa (226 aa). It is bright yellow in color, and UV-visible spectroscopy reveals absorbance maxima at 377 and 454 nm, typical of a flavin cofactor. In order to determine the identity of the flavin cofactor, BtrV was precipitated with acetone. HPLC analysis of the supernatant showed a single peak with a retention time identical to that of the FMN standard rather than the FAD standard. ESI-MS confirmed the cofactor to be FMN. Assays of oxidoreductase activity were followed spectrophotometrically with FMN and NADH, and the effect of pH on BtrV activity was investigated over the range of 4.0–10.5. BtrV is most active in sodium acetate buffer at pH 5.0. Kinetic studies indicate that BtrV obeys typical Michaelis-Menten saturation kinetics (for NADH in the presence of FMN, *k*~cat~ = 1243.13 min⁻¹, *K*~M~ = 6.86 μM). BtrV is able to oxidize both NADH and NADPH at comparable rates; however, substrate inhibition was observed for NADPH, suggesting that NADH is the preferred substrate. Kinetic data also indicated that FMN is favored over FAD (for NADH in the presence of FAD, *k*~cat~ = 887.48 min⁻¹, *K*~M~ = 7.25 μM). Therefore, BtrV is characterized as an NAD(P)H:FMN oxidoreductase.

The purified BtrO was colorless, showing no spectral absorbance typical of flavin cofactors. This observation agrees with the proposition that BtrO utilizes FMNH₂ as a free coenzyme instead of a bound prosthetic group. SDS-PAGE showed the 40 kDa His₆-tagged BtrO to account for ~70% of the eluted protein. ESI-MS confirmed its molecular mass to be 40,771.0 Da (calculated at 40,772.32). BtrO was tested against several free acids, including L-glutamate, L-aspartate, GABA, and L-ornithine, under a variety of conditions; no hydroxylation products were detectable by TLC or HPLC, consistent with the proposed ACP-mediated pathway. It was expected that Sfp, which has been reported to promiscuously load various acyl- and peptidyl-CoA substrates onto carrier proteins [25], could modify Btrl with synthetic acyl-CoA derivatives to afford various acyl-S-Btrl complexes in order to assay BtrO against authentic ACP-bound substrates. However, attempts to assay BtrO activity against a synthetic sample of γ-L-Glu-S-Btrl were hindered by the lability of γ-L-Glu-CoA. Upon incubation of BtrO and BtrV with γ-L-Glu-S-Btrl generated by BtrJ, no hydroxylation product was detected, which would suggest that decarboxylation occurs first and that GABA-S-Btrl is the substrate for BtrO. Although GABA-CoA also proved unstable, enough synthetically prepared GABA-S-Btrl was obtained to assay with BtrO; surprisingly, no hydroxylation product was detected in this case, either. Given the apparent unreactivity of BtrO toward either of the likely substrates, an activity assay was devised with butyryl-S-Btrl. Butyryl-S-Btrl was selected as a model substrate based on three considerations: (1) the butyryl moiety is a structural analog of GABA; (2) butyryl-S-Btrl is easily produced in large quantity by the action of the PPTase.

Figure 5. Hydroxylation of Butyryl-S-Btrl

LC and MS traces of butyryl-S-Btrl and 2-hydroxybutyryl-S-Btrl produced by incubation of apo-Btrl with Sfp and butyryl-CoA followed by addition of BtrO and BtrV.

Sfp with apo-Btrl and butyryl-CoA; (3) and it is convenient to determine the position and stereochemistry of the hydroxyl group by assay with LDH. Apo-Btrl was modified by Sfp in the presence of butyryl-CoA to efficiently generate butyryl-S-Btrl. Incubation of butyryl-S-Btrl with BtrO and BtrV led to a new peak by LC-ESI-MS with a mass of 12,240.0 Da eluting earlier than butyryl-S-Btrl, whose measured mass was 12,224.0 Da (Figure 5). The calculated masses for hydroxybutyryl- and butyryl-S-Btrl are 12,238.65 and 12,222.66 Da, respectively. Therefore, the new product, obtained in an estimated yield of ~50%, corresponds to hydroxybutyryl-S-Btrl. When the reaction was conducted under anaerobic conditions or when FMN was replaced with FAD, no hydroxylation product was detected, indicating that FMNH₂ and molecular oxygen are required for BtrO activity. Moreover, BtrO showed no activity at all with free butyrate or with butyryl-CoA, further supporting the idea that Btrl is crucial for recognition and that a significant protein-protein interaction between BtrO and Btrl must exist. BtrO was also observed to hydroxylate octyryl-S-Btrl (data not shown), suggesting that the enzyme can accept longer chains as substrates.

Chemistry & Biology
670

![](image1.png)

Figure 6. Decarboxylation of γ-L-Glu-S-Btrl

LC and MS traces of the BtrJ γ-glutamylase reaction mixture after addition of (A) BtrK, producing γ-L-Glu-GABA-S-Btrl, (B) EDTA and then BtrK, producing GABA-S-Btrl, and (C) BtrK and then BtrO and BtrV, producing γ-L-Glu-AHBA-S-Btrl.

The position and stereochemistry of the hydroxyl group were determined enzymatically by incubation with LDH. Hydroxybutyrate was hydrolyzed from Btrl under alkaline conditions and extracted as described in **Experimental Procedures**. The extracts were dissolved in 100 μL buffer containing NAD⁺ and L- or D-LDH. After 70 min incubation, the absorbance at 340 nm of the L-LDH reaction increased from 0.12 to 0.60, whereas that of the D-LDH reaction remained nearly unchanged (0.12 to 0.17). These results indicate that the extracted hydroxybutyrate is a substrate of L-LDH but not D-LDH, therefore confirming that hydroxylation occurs at the α-carbon of the butyryl moiety and that the product has the S configuration, in agreement with the known structure of butirosin. Given the reluctance of the monooxygenase system to react with any of the expected native substrates and the efficient reactivity toward butyryl-S-Btrl, which lacks a γ-amine, we began to suspect that an alternative substrate may be involved in the biosynthetic pathway and turned to the decarboxylation step in search of answers.

BtrK (428 aa, 47.8 kDa) exhibits significant homology to the ornithine/arginine/diaminopimelate (DAP) decarboxylase family members, which are pyridoxal 5′-phosphate (PLP) dependent. In these enzymes, the PLP cofactor is covalently bound to an essential lysine residue via a Schiff base. The purified BtrK appears bright yellow in color. UV-visible spectroscopy of BtrK reveals a maximum absorbance at 420 nm, consistent with the spectrum of an internal aldimine of PLP. Gel filtration indicates that PLP and BtrK co-elute, confirming that the PLP cofactor is tightly bound to BtrK. The His₆-tagged BtrK was found to be a homodimer with a monomer mass of 49,846.0 Da (calculated at 49,847.68 inclusive of bound PLP), as confirmed by LC-ESI-MS.

Initial activity assays demonstrated that BtrK was able to decarboxylate L-ornithine, L-arginine, L-lysine, and DAP but not L-glutamate or any D-amino acids. However, this very weak decarboxylation activity was only seen at high substrate concentrations (>10 mM) and after extended incubation (>12 hr), as compared to blank reactions without BtrK. Because of the low efficiency and low specificity of decarboxylation, we concluded that these free amino acids are probably not the real substrates of the enzyme. When BtrK was added directly to the BtrJ reaction mixture and incubated for 30 min, LC-ESI-MS revealed that γ-L-Glu-S-Btrl was completely converted to a new product (**Figure 6A**). Most surprisingly, the mass of this product (observed at 12,370.0 Da) was significantly greater than that of the expected GABA-S-Btrl product (calculated at 12,237.67 Da). The mass difference could correspond to a glutamate-

Biosynthesis of (2S)-4-Amino-2-Hydroxybutyrate
671

![Figure 7. Biosynthetic Pathway of AHBA Leading to Butirosins A and B](https://i.imgur.com/yourimageurl.png)

mate residue; thus, the new product is proposed to be γ-L-Glu-GABA-S-Btrl (calculated at 12,366.85 Da). One possible explanation of this result could be that the still-present BtrJ is able to catalyze ligation of a second L-glutamate molecule to GABA-S-Btrl, forming an amide bond. The GABA-S-Btrl is closely related to holo-Btrl in structure; therefore, it is reasonably likely to fit into the active site of BtrJ. Another possibility is that it could result from a nonenzymatic nucleophilic attack by the amino group of GABA-S-Btrl toward the activated acyl-phosphate intermediates generated by BtrJ. However, the latter scenario is unlikely based on the observation that there is no γ-L-Glu-γ-L-Glu-S-Btrl produced in the BtrJ assay. For the same reason, it is also unlikely that the observed BtrK product is GABA-γ-L-Glu-Btrl. It is expected, then, that if BtrJ is inactivated before addition of BtrK, GABA-S-Btrl should be obtained. Indeed, when the reaction mixture of BtrJ was incubated with EDTA prior to addition of BtrK, the BtrK assay yielded the expected GABA-S-Btrl product with an observed mass of 12,240.0 Da, coeluting with unreacted γ-L-Glu-S-Btrl (12,284.0 Da) (Figure 6B).

The production of γ-L-Glu-GABA-S-Btrl by the combined action of BtrJ and BtrK strongly supports the contention that BtrO, showing no activity toward the most likely substrates, must act on an alternative substrate. The γ-glutamylation of GABA neutralizes the expected positively charged amine of GABA, consistent with the observed reactivity of BtrO toward the electronically neutral butyryl substrate as well as the longer neutral octyryl substrate. Indeed, BtrO is able to efficiently hydroxylate γ-L-Glu-GABA-S-Btrl: after a 30 min reaction, complete conversion of available γ-L-Glu-GABA-S-Btrl to γ-L-Glu-AHBA-S-Btrl (observed at 12,384.0 Da; calculated at 12,382.85 Da) was observed (Figure 6C). Therefore, we conclude that γ-L-Glu-GABA-S-Btrl is the real substrate for BtrO.

To date, BtrG and BtrH have not yet been successfully expressed in *E. coli* as soluble, active enzymes, and efforts toward the characterization of these two enzymes continue.

Discussion

We report here the characterization of five enzymes involved in the biosynthesis of (2S)-4-amino-2-hydroxybutyrate (AHBA). Our results suggest a novel ACP-mediated biosynthetic route (Figure 7). The L-glutamate is first activated and tethered to Btrl via a thioester linkage between its γ-carboxyl group and the terminal sulfhydryl group of the 4′-PP arm of Btrl. Decarboxylation and α-hydroxylation occur sequentially on acyl-S-Btrl intermediates. Most distinctly, AHBA is produced exclusively with a γ-glutamyl substituent on the amino group. This may be a protective mechanism that prevents the amino group of GABA-S-Btrl from undergoing nucleophilic attack on the carbonyl group, leading to self-cyclization and premature cleavage from the ACP. A similar protection mechanism appears to operate in a recently described bacterial putrescine utilization pathway in which putrescine is γ-glutamylated prior to oxidation to γ-glutamyl-γ-aminobutyraldehyde [36]; without the γ-glutamyl protective group, the γ-aminobutyraldehyde would likely self-cyclize, interfering with further transformations. Furthermore, the γ-glutamyl moiety may provide additional protection to the producing organism against the butirosin it produces. It is therefore proposed that the cleavage of the γ-glutamyl group occurs in the final step of butirosin biosynthesis. This pathway agrees with the results of feeding studies demonstrating that L-glutamate is a precursor of butirosin, whereas free AHBA is not. Although these feeding studies also indicated GABA to be a precursor, this is likely the result of the in vivo interconversion of GABA and L-glutamate via a succinate intermediate [37].

Btrl plays an essential central role in the AHBA synthetic pathway, carrying the γ-glutamyl moiety via a thioester linkage and functions as a platform on which modifications of L-glutamate occur. Btrl shows sequence homology to ACPs but not PCPs. Unlike neutral PCPs, which are buried among multiple domains, Btrl is discrete and acidic (calculated pI 4.30), resembling type II ACPs. The charged protein surface may be cru-

Chemistry & Biology
672

cial for interactions between Btrl and its partner proteins. γ-carboxyl-activated glutamate is liable to undergo cyclization, forming pyrrolidone carboxylate, or hydrolysis; however, compared to chemically synthesized γ-glutamyl-CoA thioester, γ-L-Glu-S-Btrl demonstrates unusual stability. This stability suggests that Btrl may provide essential hydrogen bonding sites to form a stabilizing environment for the γ-glutamyl moiety. The solution structure of a type II PKS ACP (actinorhodin ACP) reveals that residue Arg72 is partially buried in the hydrophobic core, suggesting a potential hydrogen bonding site for unstable polyketide intermediates [38]. This arginine residue is conserved in all PKS ACPs as well as in Btrl (Arg73) and, therefore, might serve the same stabilizing function.

Btrl is absolutely required for the function of other enzymes in this pathway, suggesting strong protein-protein interactions between Btrl and its partner enzymes. Although the structures of the PPTase Sfp [39], the phenylalanine-activating A domain of gramicidin S synthetase I [40], and several ACP/PCPs are available, just how PCPs and ACPs interact with other proteins remains unknown. BtrJ is unable to γ-glutamylate one of Btrl's predicted structural homologs, the actinorhodin ACP. Likewise, Btrl cannot replace the actinorhodin ACP in actinorhodin biosynthesis (Dr. H. Hong, personal communication), implying that specificity of recognition may result from rather subtle structural differences. A recent docking simulation of an acyl-ACP dehydrogenase (FkbI) and its cognate ACP (FkbJ) revealed that the conserved Leu residue in the D-S-L motif of the ACP was placed into a hydrophobic pocket formed by FkbI [41]. Moreover, a conserved Asp residue of FkbJ formed a salt bridge to an Arg residue conserved in FkbI orthologs but not in most acyl-CoA dehydrogenases. Such conserved interface interactions ought to exist between Btrl and its partner proteins. Taken together, our results suggest that further decoding of the protein-protein interactions in the AHBA biosynthetic pathway might well facilitate construction of hybrid enzyme systems that may allow attachment of AHBA to different substrates and, hence, produce novel aminoglycosides directly by fermentation.

BtrJ is a enzyme of dual function that selectively activates the γ-carboxyl group of L-glutamate and catalyzes two ligation reactions of γ-L-glutamate to holo-Btrl and to GABA-S-Btrl, respectively. BtrJ is functionally analogous to adenylation (A) domains in NRPSs; however, BtrJ does not show sequence similarity to any A domains. Moreover, BtrJ activates the carboxyl group via an acylphosphate intermediate, a different mechanism from that of the A domains, which utilize aminoacyl-AMP intermediates. To our knowledge, BtrJ is the first characterized enzyme that catalyzes activation of the γ-carboxyl group of L-glutamate and formation of a γ-L-Glu-S-ACP species. There is only one corresponding example known in the A domains: the A domain of the mixed PKS/NRPS McyE in microcystin biosynthesis is proposed to load the γ-carboxyl group of D-glutamate onto a PCP domain [42]. The amide-forming activity of BtrJ resembles that of γ-glutamylcysteine synthetase in glutathione biosynthesis, which catalyzes γ-glutamylation of cysteine [43]. Both enzymes are predicted to belong to the ATP-grasp superfamily based

on their similar carboxyl activation mechanism. The structural similarity between the terminal cysteamine moiety of the 4′-PP arm and the terminal amine of GABA may be a prerequisite determinant for the dual function of BtrJ.

The molecular logic utilized by these five enzymes (Btrl, BtrJ, BtrK, BtrO, and BtrV) to activate and modify a precursor docked on an ACP has been observed analogously in other biosynthetic pathways. For example, the sets of genes asm13–asm17, fkbG–fkbK, and gdmG–gdmK have been identified as factors in the biosynthesis of the 2-methoxymalonyl extender unit in the ansamitocin [44], FK520 [45], and geldanamycin [21] pathways, respectively. These reactions have been proposed to proceed through activation and ligation of a glycolytic intermediate onto an ACP followed by subsequent transformations. Analogously, in some NRPSs β-hydroxylation occurs on amino acids preattached to A-PCP didomains. Our results lend further weight to the suggestion that the utilization of acyl-S-enzyme or aminoacyl-S-enzyme species is a common strategy to sequester primary metabolites for secondary metabolism [46]. These results may also implicate similar ACP-mediated pathways in the biosynthesis of nonstandard starter units in polyketide synthesis. For example, in the biosynthetic cluster for the polyketide macrolactam vi-cenistatin in Streptomyces halstedii, among the gene products thought to be involved in the formation of the precursor of the starter unit (2R)-2-methyl-3-aminopropionate are a putative discrete ACP (VinL) and a BtrK homolog (VinO) [47]. Based on our results, one may propose a similar pathway of decarboxylation and epimerization occurring on the precursor (2S, 3S)-3-methylaspartate thioester carried by VinL. The (2R)-2-methyl-3-aminopropionyl moiety tethered to VinL is then specifically transferred to the loading module of the PKS by so-called ACP-to-ACP skipping [47].

The activities of the remaining two enzymes from this gene cluster, BtrG and BtrH, have not yet been characterized. Neither enzyme bears significant homology to any known proteins. Given that the five enzymes studied here are sufficient for the biosynthesis of γ-L-Glu-GABA-S-Btrl, we postulate that these two enzymes are responsible for the transfer of the acyl chain to ribostamycin/xylostasin and deglutamylation as the final steps in butirosin biosynthesis.

Significance

The modification of aminoglycoside antibiotics with (2S)-4-amino-2-hydroxybutyrate (AHBA) has previously been shown to help overcome bacterial resistance mechanisms and to improve pharmacological properties. Here, AHBA is shown to be biosynthesized from L-glutamate via an ACP-mediated pathway encoded within the butirosin synthetic gene cluster. Biosynthesis is demonstrated to proceed by γ-glutamylation of the ACP Btrl by BtrJ, decarboxylation of the resulting acyl-ACP by the PLP-dependent decarboxylase BtrK, and subsequent hydroxylation by a two-component FMN-dependent monooxygenase system composed of BtrO and BtrV. Surprisingly, BtrJ is found to catalyze a second γ-glutamylation of the

GABA-S-Btrl intermediate produced in the decarboxylation step; this activity may represent an unusual example of protective-group chemistry in an enzymatic system. The number of protein-protein interactions apparent in this system offers a valuable opportunity to study the determinants of such interactions in this and similar ACP-mediated systems. Furthermore, the characterization of this pathway will potentially allow the movement of these genes into other aminoglycoside-producing organisms to provide both known and novel derivatives directly by fermentation.

### Experimental Procedures

#### Cloning of Butirosin Biosynthetic Genes and Heterologous Overexpression of Enzymes

The genes *btrG*, *btrH*, *btrl*, *btrJ*, *btrK*, *btrO*, and *btrV* were amplified by PCR from genomic DNA of *Bacillus circulans* NR3312 with the following pairs of primers, respectively (restriction sites are underlined): forward pG1 5′-CGTAAAGTGAGTTTAAGCATATGAT-TAG CTGGACGAA-3′ (*Ndel*) and reverse pG2 5′-GCGATTTTCTGCA CTCGAGAAATTCTCGTCATAAC-3′ (*Xhol*); forward pH1 5′-AGA-GGAGAAGCCATATGTGCCACTCGTTA-TGACG-3′ (*Ndel*) and reverse pH2 5′-TCCTTCTGTATGGGATCCC GGGTTTCCGCACAGC GGT-3′ (*BamHI*); forward pl1 5′-CAGAAGGAGTTGAATCATATGTCT TTAGAAACTC-3′ (*Ndel*) and reverse pl2 5′-TCCAGCGGATGAG GGATCCTCATAGGG-3′ (*BamHI*); forward pj1 5′-ACGTCTATGCT AGCATGAAAGTTCACTCATCCGCTGG-3′ (*Nhel*) and reverse pj2 5′- CTCGATCCTCGAGCCCTCCCTTCTTTCATGAGTTAAT-3′ (*Xhol*); forward pK1 5′-GGGATTTGCA TATGAACCTGGATCAAGCTG-3′ (*Ndel*) and reverse pK2 5′-CGTATAT TGGAAATTCCTATTTGTCCAT-3′ (*EcoRI*); forward pO1 5′-ATACGAAAAAGCAGGCATATGATTGCATTGG-3′ (*Ndel*) and reverse pO2 5′-GTTTGCTCGCGGT CGGATCCCTAGAC-3′ (*BamHI*); and forward pV1 5′-AAAAATAACATATGGGAAAGGAGAGT-GAT-3′ (*Ndel*) and reverse pV2 5′-TTATACTCTCTCTCGAGATCCGAACGTCA CAT-3′ (*Xhol*). The gene *Sfp* was PCR amplified from genomic DNA of *Bacillus subtilis* with the forward primer 5′-AATCAACATATG AAGATTTACGGAATTTATATGGACC GC-3′ (*Ndel*) and the reverse primer 5′-CCTACTCTCGAGTTGAGCGCGGT GTCAAGCTGTTG-3′ (*Xhol*).

PCR reactions were carried out as follows with *Pfu* polymerase (Stratagene): 4 min at 94°C, 1 min at 94°C, 1 min at 60°C, 2 min at 72°C for 25 cycles, and the final extension at 72°C for 10 min. The amplified genes were cloned into appropriate restriction sites of pET-28a(+) and transformed into *E. coli* BL21(DE3) for overexpression. All constructs were checked by DNA sequencing.

#### Overexpression and Purification of Recombinant Btrl, BtrJ, BtrK, BtrO, BtrV, and Sfp

One liter of culture in LB medium of *E. coli* BL21(DE3) harboring recombinant plasmid was grown at 37°C for 3 hr with kanamycin selection (50 μg/ml). Protein overexpression was induced by addition of 0.2 mM isopropyl-α-D-thiogalactopyranoside (IPTG), and cell growth continued at 16°C overnight. Ni²⁺-NTA affinity chromatography purification was performed according to the manufacturer’s protocols (Novagen). The harvested cells were resuspended in 50 ml 1× binding buffer (20 mM Tris-HCl [pH 7.9], 0.5 M NaCl, 5 mM imidazole, and 10% glycerol) and disrupted by sonication. N-terminally His₆-tagged proteins were purified in one step by Ni²⁺-NTA affinity column and exchanged into storage buffer by ultrafiltration. The purified proteins were stored in appropriate buffers (see below) containing 50% glycerol at –20°C, except Btrl and BtrV, which were stored at 4°C: apo-Btrl, 50 mM HEPES (pH 7.0), 50 mM NaCl, 2 mM DTT, and 0.5 mM EDTA; Sfp, 50 mM HEPES (pH 7.6) and 200 mM NaCl; BtrJ, 50 mM HEPES (pH 7.6), 50 mM KCl, 5 mM MgCl₂, 5 mM DTT, and 0.5 mM EDTA; BtrK, 50 mM HEPES (pH 7.8) and 2 mM DTT; BtrO, 50 mM HEPES (pH 7.6); BtrV, 50 mM Tris-HCl (pH 7.5) and 50 mM NaCl.

#### LC-ESI-MS Analysis of Enzyme Assays

Online LC-ESI-MS (helium as collision gas) analysis of protein samples was performed on a Finnigan LCQ (Thermo Finnigan) coupled with an HP1100 HPLC system (Agilent). Samples were separated on a 4.6 × 250 mm Jupiter 5μ C4 column (Phenomenex) by the following gradient at a flow rate of 0.2 ml/min and column temperature of 40°C: 0–5 min 5% B, 5–6 min 5%–35% B, 6–31 min 35%–65% B, 31–36 min 65% B, and 36–41 min 65%–95% B (buffer A, H₂O with 0.1% TFA; buffer B, CH₃CN with 0.1% TFA). Absorbance at 214 nm was monitored. Mass spectra were acquired from 600 to 2000 Da.

#### In Vitro 4′-Phosphopantetheinylation of Apo-Btrl

A reaction mixture (total volume 10 μl) containing 50 mM HEPES (pH 7.0), 5 mM DTT, 1 mM Coenzyme A or acyl-CoA, 1 mM MgCl₂, 0.1–0.5 nmol Sfp, and 0.5–1.0 nmol apo-Btrl was incubated at 37°C for 15–30 min and analyzed directly by LC-ESI-MS.

#### In Vivo 4′-Phosphopantetheinylation of Apo-Btrl

The plasmid for coexpression of the genes *btrl* and *Sfp* was constructed as follows: PCR product of the gene *sfp* fused with the T7 promoter, *lac* operon, and T7 terminator was amplified from the pET28-a(+)-derived plasmid pSfp with *pfu* polymerase. The *SphI* restriction site was introduced to both primers (underlined), the forward Col-1 5′-GTAGGAGCATGCTTAATACGACTCACTATA-GGG GAA-3′ and the reverse Col-2 5′-TGAGTTGCATGCAAAAAACCCC TCAAGAGCCCGTTTA-3′. The digested PCR product was cloned into the pET-28a(+)-derived plasmid pBtrl with the *SphI* site, yielding the coexpression plasmid pCo-I, which contains both *btrl* and *Sfp* genes each under the control of their own T7 promoter. Holo-Btrl was overexpressed and purified as described above. The molecular mass was confirmed by online LC-ESI-MS analysis.

#### Enzymatic Assays of BtrJ

The γ-L-glutamylation activity of BtrJ was assayed in a typical 50 μl reaction containing 10 mM L-Glu, 5 mM MgCl₂, 5 mM MnCl₂, 5 mM ATP, 50 mM KCl, 1 mM DTT, 1–2 nmol holo-Btrl, and 0.1–0.5 nmol BtrJ in 50 mM HEPES buffer (pH 7.6).

The L-glutamate-dependent production of ADP by BtrJ was coupled with reactions of pyruvate kinase and L-lactate dehydrogenase and monitored by the oxidation of NADH with UV/Visible spectrophotometry at 340 nm. A 1 ml reaction contained 50 mM HEPES (pH 7.6), 20 mM KCl, 5 mM MgCl₂, 2 mM DTT, 2 mM ATP, 10 mM MnCl₂, 0.5 mM phospho(enol)pyruvate (PEP), 0.15 mM NADH, 10 units of pyruvate kinase (type VII from rabbit muscle), 10 units of L-lactate dehydrogenase (type II from rabbit muscle), 0.5–5.0 μmol BtrJ, and 5–50 mM L-Glu. The mixture was preincubated without L-Glu at 25°C for 5 min and transferred to a 1 ml disposable polystyrene cuvette (Fisher). The reaction was incubated at 25°C and initiated by addition of L-Glu. The decrease of absorbance at 340 nm was recorded on a CARY 100 Bio spectrophotometer (Varian).

#### Enzymatic Assays of BtrK

To the γ-L-glutamylation reaction described above, 10–50 pmol BtrK to a final volume of 55 μl was added. The reaction was incubated at 30°C for 30 min and analyzed directly by LC-ESI-MS. 100 mM EDTA (pH 7.8) was added to the γ-L-glutamylation reaction to a final concentration of 10 mM prior to addition of BtrK to detect GABA-S-Btrl.

#### Determination of BtrV FMN Cofactor

BtrV in 500 μl solution (140 μM) was denatured by addition of acetone. Upon removal of acetone, the supernatant was analyzed by UV-visible spectrophotometry and LC-ESI-MS on a reverse phase Prodigy C18 column (250 × 4.6 mm, Phenomenex).

#### Enzymatic Assay of BtrV

A typical 1 ml reaction mixture was composed of 200 μM NAD(P)H, 10 μM FMN or FAD, and 715 nmol BtrV in 45 mM Tris-HCl buffer (pH 7.5). Reaction progress was followed spectrophotometrically at 340 or 375 nm at 25°C.

#### Kinetic Studies of BtrV

Kinetic parameters were measured in the presence of the external electron acceptor iodonitrotetrazolium chloride (INT). The initial

rate of reaction was measured at 530 nm in a 1 ml reaction containing FMN or FAD (10 μM), BtrV (14.8 nM), INT (1 mM), and various concentration of NADH or NADPH in 20 mM sodium acetate buffer (pH 5.0) at 530 nm at 25°C.

### Enzymatic Assay of BtrO/BtrV

γ-L-Glu-GABA-S-Btrl and GABA-S-Btrl were prepared in a 60 μl reaction immediately before use. The reaction was initiated by adding 10 μM FMN, 2 mM NADH, 10–50 pmol BtrO, 1–5 pmol BtrV, and incubated at 30°C for 30 min.

### Monoxygenation of Butyryl-S-Btrl by BtrO/BtrV

A standard 10 μl butyrylation reaction of Btrl contained 0.5–2 nmol apo-Btrl, 0.1–0.5 nmol Sfp, 1 mM MgCl₂, 5 mM DTT, and 1 mM butyryl-CoA in 50 mM HEPES buffer (pH 7.0) and was incubated at 37°C for 15 min. 10 μM FMN, 2 mM NADH, 10–50 pmol BtrO, and 1–5 pmol BtrV were added, and the reaction was continued at 37°C for 30 min.

### Determination of the Position and Stereochemistry of BtrO/BtrV Hydroxylation

α-hydroxy-butyryl-S-Btrl was produced in a scaled-up 3 ml reaction that contained 150 nmol apo-Btrl at 30°C for 1 hr. A control reaction was set up without butyryl-CoA. A few drops of 5 M NaOH were added, and the mixture was incubated at 40°C for 30 min. Proteins were removed by centrifugation, and α-hydroxybutyrate was extracted from the supernatant as described in the literature [48]. The extracts were redissolved in 100 μl 50 mM HEPES buffer (pH 7.6) and incubated with 1 mM NAD⁺ and 10 units D- or L-lactate dehydrogenase at 25°C. The reaction was monitored at 340 nm over 70 min.

### Synthesis of GABA-CoA and γ-L-Glu-CoA

N-Boc-L-glutamic acid α-tert-butyl ester or N-Boc-γ-aminobutyric acid (0.7 mmol) was dissolved in 5 ml distilled THF under nitrogen. The solution was cooled in an ice bath, and ethylchloroformate (70 μl) was added. The reaction was stirred for 5 min at 0°C, and then coenzyme A (0.5 mmol) was added in 3.5 ml of 0.3 M NaOH solution. The reaction continued to stir for 1 hr at room temperature and then was neutralized with 6 M HCl and concentrated in vacuo. After lyophilization, the resulting white solid was purified by HPLC with a preparative C18 column with a gradient of 0%–100% buffer B over 30 min (buffer A, 0.1% aqueous TFA; buffer B, 100% methanol). Product elution was monitored at 214 nm. Deprotection of the product was carried out in 1 ml 100% TFA at room temperature for 1 hr. The finished reacton was diluted with water to 4 ml and purified by HPLC as above. Product mass was confirmed by ESI-MS.

### Acknowledgments

Y.L. wishes to thank the Cambridge Overseas Trust for financial support. N.M.L. acknowledges financial support from the Marshall Scholarships. R.G. thanks the Shell Centenary Scholarship Fund for financial support. J.B.S. thanks the BBSRC for a project grant in support of this work. Grateful thanks are given to Dr. Dieter Spiteller and Dr. Tsung-Lin Li for assistance with mass spectroscopy.

Received: March 11, 2005  
Revised: April 12, 2005  
Accepted: April 12, 2005  
Published: June 24, 2005  

### References

1. Edelmann, P., and Gallant, J. (1977). Mistranslation in *E. coli*. Cell 10, 131–137.
2. Carter, A.P., Clemons, W.M., Brodersen, D.E., Morgan-Warren, R.J., Wimberly, B.T., and Ramakrishnan, V. (2000). Functional insights from the structure of the 30S ribosomal subunit and its interactions with antibiotics. Nature 407, 340–348.
3. Dax, S.L. (1997). Antibacterial Chemotherapeutic Agents, First Edition (London: Blackie Academic & Professional).
4. Tok, J.B., Dunn, L.J., and Des Jean, R.C. (2001). Binding of dimeric aminoglycosides to the HIV-1 rev responsive element (RRE) RNA construct. Bioorg. Med. Chem. Lett. 11, 1127–1131.
5. Litovchick, A., Lapidot, A., Eisenstein, M., Kalinkovich, A., and Borkow, G. (2001). Neomycin B-arginine conjugate, a novel HIV-1 Tat antagonist: synthesis and anti-HIV activities. Biochemistry 40, 15612–15623.
6. Schroeder, R., Waldsich, C., and Wank, H. (2000). Modulation of RNA function by aminoglycoside antibiotics. EMBO J. 19, 1–9.
7. Neu, H.C. (1992). The crisis in antibiotic resistance. Science 257, 1064–1073.
8. Mingeot-Leclercq, M.P., Glupczynski, Y., and Tulkens, P.M. (1999). Aminoglycosides: activity and resistance. Antimicrob. Agents Chemother. 43, 727–737.
9. Wright, G.D. (1999). Aminoglycoside-modifying enzymes. Curr. Opin. Microbiol. 2, 499–503.
10. Kawaguchi, H., Naito, T., Nakagawa, S., and Fujisawa, K.I. (1972). BB-K 8, a new semisynthetic aminoglycoside antibiotic. J. Antibiot. (Tokyo) 25, 695–708.
11. Howells, J.D., Anderson, L.E., Coffey, G.L., Senos, G.D., Underhill, M.A., Vogler, D.L., and Ehrlich, J. (1972). Butirosin, a new aminoglycosidic antibiotic complex: bacterial origin and some microbiological studies. Antimicrob. Agents Chemother. 2, 79–83.
12. Nagabhushan, T.L., Miller, G.H., and Weinstein, M.J. (1982). Structure-activity relationships in aminoglycoside-aminocyclitol antibiotics. In The Aminoglycosides, Microbiology, Clinical Use, and Toxicology, Volume A, A. Whelton and H.C. Neu, eds. (New York: M. Dekker), pp. 3–27.
13. Heifetz, C.L., Fisher, M.W., Chodubski, J.A., and DeCarlo, M.O. (1972). Butirosin, a new aminoglycosidic antibiotic complex: antibacterial activity in vitro and in mice. Antimicrob. Agents Chemother. 2, 89–94.
14. Ota, Y., Tamegai, H., Kudo, F., Kuriki, H., Koike-Takeshita, A., Eguchi, T., and Kakinuma, K. (2000). Butirosin-biosynthetic gene cluster from *Bacillus circulans*. J. Antibiot. (Tokyo) 53, 1158–1167.
15. Yukita, T., Nishida, H., Eguchi, T., and Kakinuma, K. (2003). Biosynthesis of (2R)-4-amino-2-hydroxybutyric acid, unique and biologically significant substituent in butirosins. J. Antibiot. (Tokyo) 56, 497–500.
16. Aubert-Pivert, E., el-Samaraie, A., and von Dohren, H. (1993). A 4-amino-2-hydroxybutyrate activating enzyme from butirosin-producing *Bacillus circulans*. Biochem. Biophys. Res. Commun. 197, 1334–1339.
17. Huang, F., Haydock, S.F., Mironenko, T., Spiteller, D., Li, Y., and Spencer, J.B. (2005). The neomycin biosynthetic gene cluster of *Streptomyces fradiae* NCIMB 8233: characterization of an aminotransferase involved in the formation of 2-deoxystreptamine. Org. Biomol. Chem. 3, 1410–1418.
18. Cane, D.E., Walsh, C.T., and Khosla, C. (1998). Harnessing the biosynthetic code: combinations, permutations, and mutations. Science 282, 63–68.
19. Cane, D.E., and Walsh, C.T. (1999). The parallel and convergent universes of polyketide synthases and nonribosomal peptide synthetases. Chem. Biol. 6, R319–R325.
20. Piel, J. (2002). A polyketide synthase-peptide synthetase gene cluster from an uncultured bacterial symbiont of *Paederus* beetles. Proc. Natl. Acad. Sci. USA 99, 14002–14007.
21. Rascher, A., Hu, Z., Viswanathan, N., Schirmer, A., Reid, R., Nierman, W.C., Lewis, M., and Hutchinson, C.R. (2003). Cloning and characterization of a gene cluster for geldanamycin production in *Streptomyces hygroscopicus* NRRL 3602. FEMS Microbiol. Lett. 218, 223–230.
22. Debabov, D.V., Kiriukhin, M.Y., and Neuhaus, F.C. (2000). Biosynthesis of lipoteichoic acid in *Lactobacillus rhamnosus*: role of DltD in D-alanylation. J. Bacteriol. 182, 2855–2864.
23. Lambalot, R.H., Gehring, A.M., Flugel, R.S., Zuber, P., LaCelle, M., Marahiel, M.A., Reid, R., Khosla, C., and Walsh, C.T. (1996). A new enzyme superfamily—the phosphopantetheinyl transferases. Chem. Biol. 3, 923–936.

24. Flugel, R.S., Hwangbo, Y., Lambalot, R.H., Cronan, J.E., Jr., and Walsh, C.T. (2000). Holo-(acyl carrier protein) synthase and phosphopantetheinyl transfer in *Escherichia coli*. J. Biol. Chem. 275, 959–968.

25. Quadri, L.E., Weinreb, P.H., Lei, M., Nakano, M.M., Zuber, P., and Walsh, C.T. (1998). Characterization of Sfp, a *Bacillus subtilis* phosphopantetheinyl transferase for peptidyl carrier protein domains in peptide synthetases. Biochemistry 37, 1585–1595.

26. Climent, I., and Rubio, V. (1986). ATPase activity of biotin carboxylase provides evidence for initial activation of HCO3- by ATP in the carboxylation of biotin. Arch. Biochem. Biophys. 251, 465–470.

27. Meister, A. (1989). Mechanism and regulation of the glutamine-dependent carbamyl phosphate synthetase of *Escherichia coli*. Adv. Enzymol. Relat. Areas Mol. Biol. 62, 315–374.

28. Mullins, L.S., Zawadzke, L.E., Walsh, C.T., and Raushel, F.M. (1990). Kinetic evidence for the formation of D-alanyl phosphate in the mechanism of D-alanyl-D-alanine ligase. J. Biol. Chem. 265, 8993–8998.

29. Gushima, H., Miya, T., Murata, K., and Kimura, A. (1983). Purification and characterization of glutathione synthetase from *Escherichia coli* B. J. Appl. Biochem. 5, 210–218.

30. Thoden, J.B., Firestine, S., Nixon, A., Benkovic, S.J., and Holden, H.M. (2000). Molecular structure of *Escherichia coli* PurT-encoded glycineamide ribonucleotide transformylase. Biochemistry 39, 8791–8802.

31. Wolodko, W.T., Fraser, M.E., James, M.N., and Bridger, W.A. (1994). The crystal structure of succinyl-CoA synthetase from *Escherichia coli* at 2.5-A resolution. J. Biol. Chem. 269, 10883–10890.

32. Hastings, J.W., Potrikus, C.J., Gupta, S.C., Kurfurst, M., and Makemson, J.C. (1985). Biochemistry and physiology of bioluminescent bacteria. Adv. Microb. Physiol. 26, 235–291.

33. Thibaut, D., Ratet, N., Bisch, D., Faucher, D., Debussche, L., and Blanche, F. (1995). Purification of the two-enzyme system catalyzing the oxidation of the D-proline residue of pristinamycin IIB during the last step of pristinamycin IIA biosynthesis. J. Bacteriol. 177, 5199–5205.

34. Witschel, M., Nagel, S., and Egli, T. (1997). Identification and characterization of the two-enzyme system catalyzing oxidation of EDTA in the EDTA-degrading bacterial strain DSM 9103. J. Bacteriol. 179, 6937–6943.

35. Eichhorn, E., van der Ploeg, J.R., and Leisinger, T. (1999). Characterization of a two-component alkanesulfonate monooxygenase from *Escherichia coli*. J. Biol. Chem. 274, 26639–26646.

36. Kurihara, S., Oda, S., Kato, K., Kim, H.G., Koyanagi, T., Kumagai, H., and Suzuki, H. (2005). A novel putrescine utilization pathway involves gamma-glutamylated intermediates of *Escherichia coli* K-12. J. Biol. Chem. 280, 4602–4608.

37. Metzer, E., Levitz, R., and Halpern, Y.S. (1979). Isolation and properties of *Escherichia coli* K-12 mutants impaired in the utilization of gamma-aminobutyrate. J. Bacteriol. 137, 1111–1118.

38. Crump, M.P., Crosby, J., Dempsey, C.E., Parkinson, J.A., Murray, M., Hopwood, D.A., and Simpson, T.J. (1997). Solution structure of the actinorhodin polyketide synthase acyl carrier protein from *Streptomyces coelicolor* A3(2). Biochemistry 36, 6000–6008.

39. Reuter, K., Mofid, M.R., Marahiel, M.A., and Ficner, R. (1999). Crystal structure of the surfactin synthetase-activating enzyme sfp: a prototype of the 4'-phosphopantetheinyl transferase superfamily. EMBO J. 18, 6823–6831.

40. Conti, E., Stachelhaus, T., Marahiel, M.A., and Brick, P. (1997). Structural basis for the activation of phenylalanine in the nonribosomal biosynthesis of gramicidin S. EMBO J. 16, 4174–4183.

41. Watanabe, K., Khosla, C., Stroud, R.M., and Tsai, S.C. (2003). Crystal structure of an Acyl-ACP dehydrogenase from the FK520 polyketide biosynthetic pathway: insights into extender unit biosynthesis. J. Mol. Biol. 334, 435–444.

42. Tillett, D., Dittmann, E., Erhard, M., von Dohren, H., Borner, T., and Neilan, B.A. (2000). Structural organization of microcystin biosynthesis in *Microcystis aeruginosa* PCC7806: an integrated peptide-polyketide synthetase system. Chem. Biol. 7, 753–764.

43. Orlowski, M., and Meister, A. (1971). Partial reactions catalyzed by -glutamylcysteine synthetase and evidence for an activated glutamate intermediate. J. Biol. Chem. 246, 7095–7105.

44. Yu, T.W., Bai, L., Clade, D., Hoffmann, D., Toelzer, S., Trinh, K.Q., Xu, J., Moss, S.J., Leistner, E., and Floss, H.G. (2002). The biosynthetic gene cluster of the maytansinoid antitumor agent ansamitocin from *Actinosynnema pretiosum*. Proc. Natl. Acad. Sci. USA 99, 7968–7973.

45. Wu, K., Chung, L., Revill, W.P., Katz, L., and Reeves, C.D. (2000). The FK520 gene cluster of *Streptomyces hygroscopicus* var. ascomyceticus (ATCC 14891) contains genes for biosynthesis of unusual polyketide extender units. Gene 251, 81–90.

46. Chen, H., Thomas, M.G., O'Connor, S.E., Hubbard, B.K., Burkart, M.D., and Walsh, C.T. (2001). Aminoacyl-S-enzyme intermediates in beta-hydroxylations and alpha,beta-desaturations of amino acids in peptide antibiotics. Biochemistry 40, 11651–11659.

47. Ogasawara, Y., Katayama, K., Minami, A., Otsuka, M., Eguchi, T., and Kakinuma, K. (2004). Cloning, sequencing, and functional analysis of the biosynthetic gene cluster of macrolactam antibiotic vicenistatin in *Streptomyces halstedii*. Chem. Biol. 11, 79–86.

48. Lees, W.J., Benson, T.E., Hogle, J.M., and Walsh, C.T. (1996). (E)-enolbutyryl-UDP-N-acetylglucosamine as a mechanistic probe of UDP-N-acetylenolpyruvylglucosamine reductase (MurB). Biochemistry 35, 1342–1351.
