
INFECTION AND IMMUNITY, July 2005, p. 4272–4280
0019-9567/05/$08.00+0 doi:10.1128/IAI.73.7.4272–4280.2005
Copyright © 2005, American Society for Microbiology. All Rights Reserved.

### Identification of a Novel Adhesion Molecule Involved in the Virulence of *Legionella pneumophila*

**Bin Chang, Fumiaki Kura, Junko Amemura-Maekawa, Nobuo Koizumi, and Haruo Watanabe\***

*Department of Bacteriology, National Institute of Infectious Diseases, Toyama 1-23-1, Shinjuku-ku, Tokyo 162-8640, Japan*

Received 2 September 2004/Returned for modification 23 November 2004/Accepted 24 February 2005

*Legionella pneumophila* is an intracellular bacterium, and its successful parasitism in host cells involves two reciprocal phases: transmission and intracellular replication. In this study, we sought genes that are involved in virulence by screening a genomic DNA library of an *L. pneumophila* strain, 80-045, with convalescent-phase sera of Legionnaires’ disease patients. Three antigens that reacted exclusively with the convalescent-phase sera were isolated. One of them, which shared homology with an integrin analogue of *Saccharomyces cerevisiae*, was named *L. pneumophila* adhesion molecule homologous with integrin analogue of *S. cerevisiae* (LaiA). The *laiA* gene product was involved in *L. pneumophila* adhesion to and invasion of the human lung alveolar epithelial cell line A549 during in vitro coculture. However, its presence did not affect multiplication of *L. pneumophila* within a U937 human macrophage cell line. Furthermore, after intranasal infection of A/J mice, the *laiA* mutant was eliminated from lungs and caused reduced mortality compared to the wild isolate. Thus, we conclude that the *laiA* gene encodes a virulence factor that is involved in transmission of *L. pneumophila* 80-045 and may play a role in Legionnaires’ disease in humans.

---

Legionellae are gram-negative, rod-shaped bacteria that are ubiquitous inhabitants of biofilms in aquatic environments or moist soil and replicate as intracellular parasites of protozoa (12, 45, 46). Although approximately 50 species of *Legionella* are known, the majority of Legionnaires’ disease cases are attributed to *Legionella pneumophila*, particularly serogroup 1 (12, 53). *L. pneumophila* causes two distinct clinical infections in humans: a potentially lethal pneumonia called Legionnaires’ disease (13) and a self-limiting, flu-like disease known as Pontiac fever (17). Upon transmission to individuals through aerosols containing *L. pneumophila*, the bacteria adhere to and enter into the alveolar pulmonary macrophages and/or epithelial cells (8, 34, 36, 55) by coiling or conventional phagocytosis (22, 40). Subsequently, the bacteria replicate within the phagosomes, which interact with mitochondria, smooth vesicles, and ribosomes, but the phagosomes do not acidify or fuse with the lysosomes for at least 8 h after infection (21, 53). Lastly, when the amino acids of the host cells become scarce, the cells eventually lyse and intracellular *L. pneumophila* organisms escape from the depleted cells and are transmitted to a new host (21, 53). Adhesion to and entry into host cells are considered to be the primary and indispensable steps for infection by *L. pneumophila*.

In many microbial infections, expression of virulence genes is modulated in response to the different environments encountered at the site of infection (28, 31). The discovery and characterization of genes specifically induced in patients upon infection are indispensable in studying bacterial virulence at the molecular level. To identify the genes that are involved in virulence of pathogenic microorganisms, various techniques, such as in vivo expression technology, signature-tagged mutagenesis, subtractive and differential hybridization, and in vivo induced antigen technology, have been developed (2, 18, 19, 42). Signature-tagged mutagenesis has also been used for the identification of the *L. pneumophila* genes that are necessary for infection of the guinea pig (10) and the free-living amoeba *Acanthamoeba castellanii* (38). The *mip* gene of *L. pneumophila* was identified and characterized by screening a genomic library with an anti-*L. pneumophila* rabbit serum (11). In vivo induced antigen technology is a new technique for studying genes that are specifically expressed in vivo by immunoscreening with sera (18). By using the absorbed sera collected during different stages of the disease or from patients infected via different routes, the genes expressed at specific stages can be identified. The technology has been used to study the oral pathogen *Actinobacillus actinomycetemcomitans* (18) and *Mycobacterium tuberculosis* (9).

In the current study, we identified three *L. pneumophila* genes whose products specifically reacted with the convalescent-phase sera of Legionnaires’ disease patients but not with the sera of healthy people. One of them, *laiA*, was isolated and shown to be necessary for the virulence of *L. pneumophila* in mice.

---

### MATERIALS AND METHODS

Bacterial strains, plasmids, media, and growth conditions. Bacterial strains and plasmids used in this study are summarized in Table 1. *L. pneumophila* serogroup 1 strain 80-045, which was isolated from a Legionnaires’ disease patient in Japan (48), was used for studying pathogenesis in this work. *L. pneumophila* strains were grown at 37°C either on buffered charcoal yeast extract (BCYE) agar (Difco) or in ACES [N-(2-acetamido)-2-aminoethanesulfonic acid]-buffered yeast extract broth (AYE) (43). If needed, *L. pneumophila* recovered from the BCYE plates was inoculated in 5 ml of AYE medium, incubated at 37°C with shaking, and collected at an optical density at 600 nm of 2.0. *Escherichia coli* K-12 strains DH10B (Invitrogen), XL1-Blue MRF′, and XLOLR (Stratagene) were used as recipient cells for recombinant DNAs. All of these *E. coli* strains were grown at 37°C on Luria-Bertani (LB) agar plates or in LB broth. For plaque formation, NZY broth (GibcoBRL) supplemented with 1.5% or 0.7% agar (Difco) was used for agar plates or top agar, respectively.

TABLE 1. Bacterial strains and plasmids used in this study

| Strain or plasmid | Genotype and features$^a$ | Reference or source |
|-------------------|----------------------------|---------------------|
| $L$. pneumophila | Serogroup 1, wild-type isolate | 48 |
| 80-045 |  |  |
| LAM 0101 | 80-045$laiA::Km^{r}$ | This study |
| LAM 0102 | LAM 0101 with pMMBLG 0503 | This study |
| LAM 0103 | LAM 0101 with pMMB207C | This study |
| $E$. coli |  |  |
| XL1-Blue MRF' | $\Delta(mcrA)183\Delta(mcrCB-hsdSMR-mrr)173endA1supE44thi-1recA1gyrA96relA1lac[F'proAB lacI^{q}ZDM15Tn10(Tet^{r})]$ | Stratagene |
| XLOLR | $\Delta(mcrA)183\Delta(mcrCB-hsdSMR-mrr)173endA1thi-1recA1gyrA96relA1lac[F'proAB lacI^{q}Z\Delta M15Tn10(Tet^{r})]$ | Stratagene |
| DH10B | $F^{-}mcrA(mrr-hsdRMS-mcrBC)\phi80lacZ\Delta M15\Delta lacX74deoRrecA1endA1araD139\Delta(ara-leu)7697galUgalK\lambda^{-}rpsLnupG$ | Invitrogen |
| Plasmids |  |  |
| pUC19 | $oriR(ColE1)$ MCS $Ap^{r}$ | 56 |
| pACYC177 | $oriR(P15A)Km^{r}$ | 1a |
| pBK-CMV | $oriR(ColE1)$ MCS $Neo^{r}Km^{r}$ | Stratagene |
| pCR-XL-TOPO | $oriR(ColE1)ccdBMCSKm^{r}Zeocin^{r}$ | Invitrogen |
| pLAW344 | $sacB$ MCS $oriT(RK2)Cm^{r}loxPorir(ColE1)Ap^{r}loxp$ | 54 |
| pMMB207C | RSF1010 derivative, IncQ $lacI^{q}Cm^{r}PtacoriTMCSmobA::Km^{s}$ | 32 |
| pBKLG04 | $Km^{r}$, contains the region from base 1477 to base 4542 of $laiA$ on the phagemid pBK-CMV | This study |
| pCRLG06 | $Km^{r}$, contains the region from base 1 to base 3913 of $laiA$ on pCR-XL-TOPO | This study |
| pLAWLG0501 | $Cm^{r}$, contains the region between the BamHI and SphI sites of pUCLG0501 in the EcoRV site of pLAW344 | This study |
| pMMBLG0503 | $Cm^{r}$, contains the region from base 1 to base 4542 of $laiA$ between the SmaI and SalI sites of pMMB207C | This study |
| pUCLG05 | $Ap^{r}$, contains the region from base 2318 to base 4542 of $laiA$ between the HincII and XbaI sites of pUC19 | This study |
| pUCLG06 | $Ap^{r}$, contains the region from base 1 to base 3609 of $laiA$ between the PstI and HincII sites of pUC19 | This study |
| pUCLG0501 | $laiA::Km^{r}$ of pUCLG05 | This study |
| pUCLG0503 | $Ap^{r}$, contains the region from base 1 to base 4542 of $laiA$ between the PstI and HincII sites of pUC19 | This study |

$a$ MCS, multiple cloning sites; Neo, neomycin.

4274 CHANG ET AL. INFECT. IMMUN.

80-045

f3  
Ac Ban Ao r8  
-135 +1 4542 5030 bases  

laiA  

pBKLG04  

pUCLG05  

pUCLG0501  

pLAWLG0501  

pCRLG06  

pUCLG06  

pUCLG0503  

pMMBLG0503  

FIG. 1. Schematic diagrams of laiA in the chromosome of L. pneumophila 80-045 and of plasmids constructed in this study. The chromosome of 80-045 is presented as a solid line, and the vectors are presented as broken lines. The region corresponding to the laiA gene is shown as heavy lines. The putative translational start codon of the LaiA protein is indicated as 1. The arrow indicates the direction of transcription of the laiA gene. Sites for primers f3 and r8 used in PCR analysis are shown. Triangles show the insertion of the Km<sup>r</sup> cassette. Abbreviations for the restriction enzymes: Ac, AccII; Ao, Aor51HI; Bam, BamHI; Ban, BanII; Ps, PstI; Pv, PvuII; Sp, SphI; Xb, XbaI; Xh, XhoI.

and/or 0.01% Triton X-100 at 37°C for 30 min before lysis with the sample buffer as described by Hazlett et al. (20).

Construction of plasmids for allelic exchange. The constructed plasmids used in this study are shown in Fig. 1. First, a DNA fragment of the recombinant plasmid pBKLG04, which contained the 3′ terminus (from the base 1477 to 4542) of laiA in the pBK-CMV phagemid vector, was digested with XbaI and AccII (base 2318 of laiA). Thereafter, the fragment containing part of laiA was cloned into the XbaI and HincII sites of pUC19 to generate pUCLG05. To knock out the laiA gene, a Km resistance (Km<sup>r</sup>) cassette from pACYC177 (1a) was inserted. The Km<sup>r</sup> cassette was ligated to the Aor51HI site (base 3609 of laiA) of pUCLG05, and the resultant plasmid was called pUCLG0501. Subsequently, the pUCLG0501 DNA was digested with BamHI and SphI. The fragment, which carried the laiA gene containing the Km<sup>r</sup> cassette, was blunt ended and ligated to the EcoRV site of pLAW344 (54) to generate pLAWLG0501. Lastly, pLAWLG0501 DNA was introduced into 80-045 by electroporation to mediate allelic exchange, and the strain was plated on BCYE agar containing Km after growth in AYE at 37°C for 5 h (54). Km<sup>r</sup> Suc<sup>r</sup> Cm<sup>s</sup> colonies, which should be Km<sup>r</sup> mutants of the laiA gene on the chromosome, were selected, and one of them was named LAM0101.

Construction of plasmids for complementation. To obtain a fragment that contained the entire gene, a portion of the 5′ terminus of laiA was amplified by PCR and ligated to the 3′ terminus. The DNA fragment corresponding to the 5′ terminus of laiA (approximately 3.9 kb in length) was amplified by PCR with primers f3 (5′-CTGCCCTGCTCTTTATAC-3′) and r8 (5′-CGAGTTACTCGC AAAACTCG-3′), using total cellular DNA from 80-045 cells as the template. The nucleotide sequence of primer f3, which is located upstream of the laiA gene, was determined by the sequence of an L. pneumophila Philadelphia-1 isolate (http://genome3.cpmc.columbia.edu/~legion/). The PCR product was cloned into pCR-XL-TOPO (Invitrogen) to produce pCRLG06. After digestion of pCRLG06 DNA with PstI and Aor51HI (base 3609 of laiA), the fragment corresponding to the 5′ terminus of laiA was cloned into the PstI and HincII sites of pUC19 to generate pUCLG06. Subsequently, the fragment containing the 3′ terminus of the laiA gene was ligated to the 5′ terminus; the BanII fragment (from base 2326 of laiA to the BanII site of the vector) of pUCLG06 was replaced by the 3′-terminal fragment of laiA, which was derived from a BanII fragment of pUCLG05. The constructed plasmid containing the entire laiA gene was named pUCLG0503. The pUCLG0503 DNA was digested with PvuII and XhoI, and the obtained fragment was cloned into the SmaI and SalI sites of the pMMB207C vector to generate pMMBLG0503. The sequence of laiA of pMMBLG0503 was confirmed to be the same as that of 80-045, and expression of the laiA gene in pMMBLG0503 was confirmed by Western blot analysis with patient sera. pMMBLG0503 and empty pMMB207C DNAs were introduced into LAM0101 to generate LAM0102 and LAM0103, respectively.

Growth and motility of bacteria in the laboratory. Growth of L. pneumophila was examined as follows. L. pneumophila was inoculated from a BCYE agar plate into 5 ml of AYE broth at a concentration of 10<sup>8</sup> CFU/ml. After overnight incubation at 37°C, the bacteria were suspended in 5 ml of fresh medium at an optical density at 600 nm of 0.3 and further cultured. The optical density of the culture was measured at 600 nm every 2 h for 24 h. The presence of pili and flagella in L. pneumophila was assessed by transmission electron microscopy after negative staining, and motility was assessed using microscopy (6).

Cell culture method. A549 cells (JCRB0076), a type II alveolar epithelial cell line, and U937 cells (JCRB9021), a human monocytic cell line, were obtained from the Japanese Collection of Research Bioresources cell bank (Osaka, Japan). All of the cells were maintained in RPMI 1640 medium (Invitrogen) plus 10% heat-inactivated fetal calf serum (HyClone). For experiments, the A549 cells were seeded into 24-well flat-bottom tissue plates (Corning) at a concentration of 5 × 10<sup>5</sup> cells/ml and incubated for 18 h at 37°C in a humid atmosphere containing 5% CO<sub>2</sub> in air. The U937 cells were seeded into 24-well plates at a concentration of 1 × 10<sup>6</sup> cells/ml and differentiated into macrophages by incubating for 48 h in RPMI medium with 10<sup>-8</sup> M phorbol 12-myristate 13-acetate in the same atmosphere.

Adhesion and invasion assays. A suspension of L. pneumophila culture in RPMI 1640 medium was added to A549 monolayers at multiplicities of infection of 20 to 2,000 and was further cultured for 1.5 h. The monolayers were washed twice with PBS to remove nonadherent bacteria. For the adhesion assay, the cells were peeled from the well by addition of 0.1% Triton X-100 and sonicated with Bioruptor UCD-200T (Cosmo Bio, Tokyo, Japan) at 130 W for 24 s. A series of

10-fold dilutions of the sonicated solution was plated on BCYE plates, and the number of bacteria was counted. For the invasion assay, after washing twice with PBS, 0.5 ml of the culture medium supplemented with 100 μg/ml of gentamicin (Wako Pure Chemical Industries, Osaka, Japan) was added to each well and incubated for 2 h to kill extracellular bacteria. The cells were washed again with PBS, and the number of intracellular bacteria was determined as described above. To correct for the variation in level between experiments, adhesion or invasion of the tested strains was indicated as a ratio to that of the wild 80-045 isolate [i.e., relative adhesion = (percent adhesion of test strain/percent adhesion of 80-045) × 100%].

Intracellular growth assay. Approximately \(1 \times 10^6\) bacteria were added to each well inoculated with U937 cells. After incubation with the bacteria for 1 h at 37°C, the cells were washed twice with PBS and treated with 100 μg/ml of gentamicin for 2 h. The initial time point (\(t = 0\)) is the first time point immediately after gentamicin treatment and represents intracellular bacteria that were enumerated following treatment of the host cell with Triton X-100. Infections were allowed to proceed until the indicated time. U937 cells were then lysed, and bacteria that grew intracellularly were enumerated on BCYE plates.

Mouse infection and LD₅₀. For examination of the difference in virulence of *L. pneumophila* strains, 6-week-old male A/J mice (SLC, Shizuoka, Japan), in whose peritoneal macrophages *L. pneumophila* is able to grow (57), were used. After anesthetization by injection under the skin with a mixture of ketamine hydrochloride (60 mg/kg; Sigma) and xylazine (12 mg/kg; Sigma), A/J mice were infected by intranasal inoculation with 40 μl of *L. pneumophila*. Lungs of mice were removed at 1, 24, and 48 h after the inoculation and homogenized. The homogenate was diluted with saline and plated on BCYE plates for determination of bacterial number. Mortality of mice was monitored for 30 days after the infection, and the 50% lethal dose (LD₅₀) was calculated by the Reed-Muench method (41). All animal experiments complied with the National Institute of Infectious Diseases guidelines regarding the use of animals in research.

Statistical analyses. All in vitro experiments were carried out in triplicate and repeated at least three times. The significance of the results was analyzed using Student's \(t\) test. In the LD₅₀ experiments, the significance of survival curve differences was examined by both log rank test and Wilcoxon test methods. Differences were considered significant at a \(P\) value of <0.05.

Nucleotide sequence accession numbers. Nucleotide sequences reported in this paper are available in the GenBank database under accession no. AB107985, AB107986, AB107987, AB185448, and AB196938.

---

**RESULTS**

Identification of novel *L. pneumophila* antigens reacted with convalescent-phase sera by immunoscreening of the 80-045 expression library. In order to identify antigens that were expressed during infection caused by *Legionella*, a lambda ZA-PII phage expression library of 80-045 was generated and screened with the convalescent-phase mouse sera. Forty-five positive clones appeared from approximately 100,000 plaques, and the positive lambda ZAP phages were converted into the phagemid form in XL1-Blue MRF'. In order to confirm the expression, total proteins of bacteria harboring the phagemids were reacted with convalescent-phase sera of Legionnaires' disease patients. Twelve clones were positive (data not shown).

Subsequently, the nucleotide sequences of the inserts were determined and the putative open reading frames (ORFs) within them were identified. The fragments corresponding to each ORF of the 12 recombinant phagemids were independently subcloned into the relevant sites of pUC19. Expression of these ORFs was tested again by immunoreaction with the convalescent-phase patient sera. Nine clones that reacted, i.e., N1, N2, N4, N5, N8, N25, N31, N32, and N34 (isolate numbers), were selected (data not shown). Clones N2, N4, N31, and N32 and clones N5, N8, and N34 expressed the same antigenic proteins. Protein expression of the representative positive clones N1, N5, N25, and N31 was further examined by reaction with sera of healthy people. Three clones, N1, N5, and

N31, expressed proteins that reacted only with the patient sera (data not shown).

The ORF of N1 encoded a 29-kDa polypeptide which contained 253 amino acid residues and lacked the 5'- and 3'-terminal ends. The ORF of N5 encoded a 33-kDa polypeptide which consisted of 292 amino acid residues. The two peptides corresponded to Lpg2803 and Lpg2327 of *L. pneumophila* Philadelphia-1 (3). The two genes were named *lgN1-1* and *lgN5-1* for *L. pneumophila* genes N1-1 and N5-1, respectively. The ORF of N31 encoded a 134-kDa polypeptide, which contained 1,021 amino acid residues, at the 3'-terminal portion of a presumed open reading frame. The C terminus of the polypeptide shared homology (41% similarity over 454 residues) with the integrin analogue of *Saccharomyces cerevisiae* (25). This result suggested that the gene might be involved in adhesion of *L. pneumophila*. The gene was named *laiA* for *L. pneumophila* adhesion molecule homologous to the integrin analogue of *S. cerevisiae*. The role of *laiA* in infection was further investigated in this study.

Cloning of the entire *laiA* gene from the 80-045 isolate. The DNA fragment containing the entire *laiA* gene of strain 80-045 was cloned into pMMB207C. The sequence was determined in both directions by using appropriate primers synthesized according to the analyzed sequences. The *laiA* gene consisted of 4,542 nucleotides and was predicted to encode a 1,513-residue polypeptide with a theoretical molecular mass of 170 kDa. A homology search revealed that the *laiA* gene shared 80.5% identity at the nucleotide level with the corresponding gene (*lpg2157*) of Philadelphia-1. There was 76.9% identity between the amino acid residues encoded by the two genes. Furthermore, five paralogs of *lpg2157* (*lpg2153*, *lpg2154*, *lpg2156*, *lpg0234*, and *lpg2509*) were present on the chromosome of Philadelphia-1 (3). We also determined the nucleotide sequences of these *laiA* paralogs of 80-045, which corresponded to those of Philadelphia-1. Three of them, *laiB*, *laiC*, and *laiE*, were located upstream of *laiA*, whereas the other two genes, *laiD* and *laiF*, were located separately in another region of chromosome (Fig. 2A). The *laiB*, *laiC*, *laiD*, *laiE*, and *laiF* genes consisted of 5,763, 4,605, 4,545, 1,131, and 1,149 nucleotides, respectively. They were predicted to encode 1,920-, 1,534-, 1,514-, 376-, and 382-residue polypeptides with theoretical molecular masses of 217, 173, 172, 43, and 43 kDa, respectively. At the nucleotide level, the *laiA*, *laiB*, *laiC*, and *laiD* genes shared approximately 80% homology with each other, but they had lower homology (45% to 49%) with the *laiE* and *laiF* genes.

During the preparation of this paper, Luo and Isberg (27) reported five paralogs, i.e., *sdeA*, *sdeB*, *sdeC*, *sidE*, and *sdeD*, of *L. pneumophila* strain LP02, a derivative of Philadelphia-1. The five genes corresponded to and were identical to *lpg2157*, *lpg2156*, *lpg2153*, *lpg0234*, and *lpg2509*, respectively (3). Those authors indicated that these proteins are translocated by a Dot/Icm type IV secretion system (27, 47, 51). At the amino acid level, LaiD, LaiE, and LaiF shared higher homology (98.3%, 97%, and 99%, respectively) with the corresponding SidE (Lpg0234), Lpg2154, and SdeD (Lpg2509) polypeptides. LaiC and LaiB possessed 89% and 83.1% homology with the corresponding polypeptides SdeC (Lpg2153) and SdeB (Lpg2156), respectively. However, there was 76.9% homology between LaiA and SdeA (Lpg2157). Compared to the other

A.

![Diagram](https://i.imgur.com/1234567.png)

B.
NH2 - [Box] - [Gray Box] - [Black Boxes] - COOH
1          505                          1513 residues

C.
EF hand consensus
1   2   3       4           5           6       7
(D) X (DNS) [ILVFYW] (DENSTG) (DNQGHKR) [GP]
8       9      10  11  12     13
(ILVMC) (DENQTSGCA) X X (DE) (ILVMFYW)

D.
LaiA<sub>493-505</sub> D P S R G G P Y E N P Y S
αInt1p I<sub>283-295</sub> N N N N S K N V S D M D S
αInt1p II<sub>601-613</sub> D S N D G D R E D N D D I
αM I<sub>449-461</sub> D V D S N G S T D L V L I
αM II<sub>513-524</sub> D V N G D K L T D - V A I
αM III<sub>576-595</sub> D L T M D G L V D L T V G
β3<sub>119-131</sub> D L S Y S M K D D L W S I

FIG. 2. (A) Schematic representation of the chromosomal region containing *laiA* and its five paralogs in *L. pneumophila* 80-045. The corresponding genes of Philadelphia-1 (3) and Lp02 (27) are presented under the predicted ORFs. (B) Domain structure of LaiA. The putative divalent-cation-binding site is shown as a hatched box, the transmembrane domain is shown as a gray box, and the DXSX motifs are shown as black boxes. (C and D) Comparison of divalent-cation-binding motifs. (C) The consensus amino acid sequences for the 13 amino acid residues of EF hand divalent-cation-binding motifs (1). Parentheses, acceptable amino acids; brackets, unacceptable amino acids; X, any amino acid. Cation-coordinating sites are shown in boldface. (D) Alignment of the putative cation-binding site of LaiA, two cation-binding sites in *C. albicans* αInt1p, three cation-binding sites in α<sub>M</sub>, and one cation-binding site in β<sub>3</sub> (14, 15, 24). The standard single-letter code is used. A dash indicates a gap. The figure was designed according to the work of Hostetter (24).

paralogs, *laiA* (*sdeA*, *lpg2157*) seems to be more diverse in the various *L. pneumophila* isolates.

Characteristics of the LaiA polypeptide. We searched LaiA for the characteristic motifs present in the integrins of *Candida albicans* and vertebrate cells (15, 24). We could not find the putative I domain of integrins and integrin analogues in LaiA. However, LaiA carried five homologues of potential MIDAS motifs (DXSX) for the coordination of divalent cations. Two of them were located at the N terminus, and the remaining three were in the middle of the LaiA protein (Fig. 2B). Chou-Fasman analysis (Genetyx) indicated that LaiA has multiple α-helices, which are present in integrins and integrin analogues (data not shown). The divalent-cation-binding sites in the α subunits are known to facilitate the ligand binding of integrins to the tripeptide Arg-Gly-Asp (RGD) (15, 24). One putative divalent-cation-binding site was found at residues 493 to 505 of the LaiA polypeptide. Comparison to the consensus sequence for the divalent-cation-binding site (Fig. 2C) revealed that there were four mismatches of amino acids at positions 7, 8, 12, and 13 (Fig. 2D). Similar to the case for the other integrins, a transmembrane domain (hydrophobic sequence) was located at amino acids 1436 to 1463 of LaiA as determined by Hydro-

Cluster analysis (Genetyx), and a single tyrosine residue was also present in the C-terminal end (Fig. 2B). Thus, LaiA contained several characteristic integrin motifs.

Isolation of a *laiA* mutant by allelic exchange. To examine the involvement of the *laiA* gene in infection caused by *L. pneumophila*, a *laiA* mutant (LAM0101) was constructed by introduction of a Km<sup>r</sup> cassette. LAM0101 was selected, and the presence of the Km<sup>r</sup> cassette in the *laiA* gene on the chromosomal DNA was confirmed by PCR and DNA sequence analysis.

Characteristic phenotypes of LAM0101 were compared with those of 80-045 under laboratory conditions. The colony morphology, growth rate in AYE broth, and sodium sensitivity of LAM0101 were not significantly different from those of the wild isolate (data not shown). Furthermore, there were no obvious differences in the presence of pili and flagella or in motility between 80-045 and LAM0101 in AYE broth (data not shown).

Exposure of LaiA to the cell surface of *L. pneumophila*. To determine the cellular location of the LaiA protein, the PK accessibility method was employed. Total cellular proteins from 80-045, which were treated with PK, PK and Triton

adhere to and invade A549 epithelial cells was examined. *L.* pneumophila 80-045 adhered to and invaded the A549 cells at rates of approximately \(7 \times 10^{-3}\) and \(3 \times 10^{-6}\) with respect to the number of inoculated bacteria, respectively. The adherence rate of the *laiA* mutant was 70% lower than that of the wild type (Fig. 4A). The level of adhesion of LAM0101 was complemented to that of 80-045 by addition of pMMBLG0503 but not by addition of the empty pMMB207C vector (Fig. 4A). The invasion ability of LAM0101 was also 70% lower than that of the 80-045 isolate (Fig. 4B). The *laiA*-complemented strain LAM0102 recovered the level of invasive ability of the wild isolate (Fig. 4B). These results suggest that the *laiA* gene is involved in the adhesion of 80-045 to A549 alveolar epithelial cells and that adhesion may be one of the most important prerequisites for invasion of *L.* pneumophila.

The gentamicin sensitivities of *L.* pneumophila wild-type 80-045, LAM0101, LAM0102, and LAM0103 were compared by incubating the strains with gentamicin at 100 μg/ml for 2 h. No significant differences in the susceptibility to gentamicin were found (data not shown).

The *laiA* gene does not affect intracellular multiplication of *L.* pneumophila in macrophage cells. Intracellular growth of 80-045 and LAM0101 in the human monocytic U937 cells was examined (Fig. 5). The number of *laiA* mutant cells in the U937 cells was approximately 40% of that of the wild type strain at 0 h (the first time point following the gentamicin treatment to kill extracellular bacteria). The numbers of the two strains growing in the U937 cells increased approximately 1,000 and 100,000 times at 24 and 48 h after infection, respectively. The rates of bacterial growth in U937 cells were not significantly different for 80-045 and LAM0101. This result shows that the *laiA* gene does not significantly affect the growth rate of *L.* pneumophila in U937 cells.

*laiA* affects the virulence of *L.* pneumophila 80-045 in the A/J mouse model. In order to elucidate whether the reduction of adherence to epithelial cells in the *laiA* mutant affects the virulence of *L.* pneumophila, we performed two assays: elimination of bacteria from mouse lungs and mouse lethality after

FIG. 3. Immunoblot of the total proteins of *L.* pneumophila with mouse monospecific anti-LaiA convalescent-phase sera. After suspension in SDS-PAGE sample buffer, samples were electrophoresed on an 8% running gel, blotted, and reacted as described in Materials and Methods. An open arrowhead indicates the full-length LaiA protein. Bands shown by closed arrowheads seem to be the degraded fractions of LaiA. Positions of molecular mass markers are indicated on the right. Lane 1, total cellular protein of 80-045; lane 2, total cellular protein of PK-treated 80-045; lane 3, total cellular protein of LAM0101.

X-100, or Triton X-100 only, were analyzed by Western blotting with the monospecific anti-LaiA antibody. As shown in Fig. 3, a band corresponding to the full-length LaiA protein was located at a molecular mass of approximately 170 kDa (lane 1). There were several bands that reacted with the LaiA-specific antibody (lane 1). These seemed to be degradation products of LaiA, because no bands were detected in the total cellular protein of LAM0101 (lane 3). After digestion of intact 80-045 bacteria with PK, the band corresponding to the full-length LaiA protein disappeared (lane 2). The same result as seen in lanes 1 and 2 was obtained when the 80-045 cells were treated with Triton X-100 only and with both PK and Triton X-100 (data not shown). These results suggest that a portion of LaiA that is sensitive to PK was exposed to the cell surface.

Adhesion and invasion of the *L.* pneumophila 80-045 isolate depends on the *laiA* gene. The ability of *L.* pneumophila to

FIG. 4. Ability of *L.* pneumophila to adhere to (A) and invade (B) A549 alveolar epithelial cells. Data points and error bars represent the means and standard errors. Shown here are the results with the wild *L.* pneumophila strain 80-045, the *laiA* mutant strain LAM0101, the *laiA*-complemented (pMMBLG0503) strain LAM0102, and LAM0103, the strain carrying the empty pMMB207C vector. All experiments were performed more than three times with triplicate cultures in each experiment.

4278 CHANG ET AL.

**FIG. 5. Growth of *L. pneumophila* in U937 human monocytic cells.** Differentiated U937 cells (1 × 10⁶ cells/well) were infected with approximately 1 × 10⁶ bacteria/well for 1 h. The data shown are representative of those from three experiments, which showed similar results. Error bars indicate the standard errors of the mean. The growth rates of 80-045 (squares) and LAM0101 (circles) at 0, 24, and 48 h after infection showed no significant difference.

intranasal inoculation. LAM0102 lost pMMBLG0503 (*Cm*<sup>r</sup>) in mouse lungs under nonselective conditions for the plasmid. Approximately 1/2 and less than 1/500 of total bacterial counts surviving in the lungs retained the plasmid at 48 and 96 h after infection, respectively (data not shown). Therefore, we examined the elimination of bacteria from lungs within 48 h after infection and did not determine the LD₅₀ of LAM0102 for mice.

A representative result for the elimination of *Legionella* from mouse lungs is presented in Fig. 6. The numbers of LAM0101 cells at the 24- and 48-h time points were significantly less than those of 80-045 and LAM0102. The results for mouse lethality after infection with 80-045 and LAM0101 are summarized in Table 2. The LD₅₀s of 80-045 and LAM0101 for the A/J mice were calculated to be 2.2 × 10⁶ and 2.3 × 10⁷ CFU, respectively, demonstrating that the *laiA* mutant was less virulent than the 80-045 wild-type strain. These results show that the *laiA* gene is involved in the virulence of *L. pneumophila* 80-045 in the A/J mouse model.

**DISCUSSION**

*L. pneumophila* can multiply intracellularly within a wide range of host cells, including protozoa and macrophages and/or epithelial cells of animals. Although the pathway for a productive *L. pneumophila* infection is a multistep process, the life cycle of the bacteria can be simply presented in two stages, replication and transmission (35). Based on the virulence traits, most of the virulence genes of *L. pneumophila* can be divided into two distinct classes: one is for the promotion of replication, and the other is for transmission. LaiA, identified in this study, appears to be involved in transmission, but not in replication, of *L. pneumophila*.

Adhesion and invasion are considered to be the primary mechanisms for the pathogenesis of intracellular pathogens,

**FIG. 6. Elimination of *L. pneumophila* from the lungs of intranasally inoculated mice.** A/J mice were inoculated with 10⁶ cells of *L. pneumophila*. In each experiment, three mice were used for the test at the 1-h infection point and four mice each were used for the 24- and 48-h infection points. At the time points indicated, the mice were sacrificed and the number of *L. pneumophila* in the lungs was determined as described in Materials and Methods. The number of bacteria at 1 h postinoculation was set at 1, and the numbers at the 24- and 48-h time points are presented as the relative ratio. Shown here are the results for the wild isolate 80-045 (squares), the *laiA* mutant LAM0101 (circles), and the *laiA*-complemented strain LAM0102 (triangles). Results indicate the means ± standard deviations of the bacterial counts at the indicated time point. The numbers of LAM0101 bacteria at the 24- and 48-h time points are significantly less than those of 80-045 and LAM0102. Three independent experiments showed similar results.

and recent observations have suggested that the initial entry plays an important role in subsequent intracellular survival and replication of *L. pneumophila* (4, 6). It has been reported that the epithelial cells also play a crucial role in the infection caused by *L. pneumophila* (34). Although the mechanism of entry is not well understood, several factors are involved in the adhesion of *L. pneumophila*, including opsonization of complement (26, 29, 37) and some specific antibodies (23, 26, 37). In addition to these factors, nonopsonic adhesive molecules (16, 44), such as a type IV pilus (52), are also involved in the adherence. More recently, two gene clusters, *enh1* and *enh2*, have been reported to enhance the entry of *L. pneumophila* into cells (6). The *rtxA* gene, which is present at the *enh1* locus, is involved in adhesion to macrophages and epithelial cells and also in virulence of *L. pneumophila* (5, 6). LaiA was involved in adherence and entry into epithelial cells (Fig. 4) and also in virulence of *L. pneumophila* infection in mice (Fig. 6 and Table 2). This study provides new evidence that adhesion and entry of *L. pneumophila* are critical steps for *Legionella* virulence and that LaiA is directly involved in these essential steps.

Integrins have multiple functions in adhesion, morphogenesis, and signaling in vertebrate cells. The functions of the integrin homologues of cyanobacteria and *S. cerevisiae* have not been well studied, whereas the integrin homologue of *C. albicans* was determined to be involved in adhesion, filamentous growth, and virulence (14, 15). A549 cells were established from a human lung cancer and have the characteristics of well-differentiated type II alveolar epithelial cells (33). The

TABLE 2. Survival of A/J mice infected with 80-045 and LAM 0101 and their LD₅₀ values$^a$

| Strain     | $3.7 \times 10^{8}$ | $1 \times 10^{8}$ | $3.7 \times 10^{7c}$ | $1 \times 10^{7c}$ | $3.7 \times 10^{6c}$ | $1 \times 10^{6}$ | $3.7 \times 10^{5}$ | LD₅₀ (CFU) |
|------------|---------------------|------------------|----------------------|-------------------|----------------------|------------------|---------------------|-------------|
| 80-045     | ND$^b$             | ND               | 0/10                 | 0/10              | 7/17                | 14/16            | 10/10               | $2.2 \times 10^{6}$ |
| LAM 0101   | 0/10               | 1/12             | 5/11                 | 12/15             | 10/10               | 10/10            | ND                  | $2.3 \times 10^{7}$ |

$a$ Survival of A/J mice was monitored for 30 days after infection. The targeted amount of bacteria inhaled into lungs of mice is indicated; the actual doses of 80-045 and LAM 0101 inhaled into lungs were approximately 1.1- and 1.23-folds the targeted doses, respectively. The LD₅₀ was calculated with actual doses.

$b$ ND, not determined.

$c$ $P < 0.05$ for 80-045 versus LAM 0101. The statistical significance of survival curve differences was examined by both log rank test and Wilcoxon test methods.

14. Gale, C. A., C. M. Bendel, M. McClellan, M. Hauser, J. M. Bercher, J. Berman, and M. K. Hostetter. 1998. Linkage of adhesion, filamentous growth, and virulence in *Candida albicans* to a single gene, *INT1*. Science 279:1355–1358.
15. Gale, C., D. Finkel, N. Tao, M. Meinke, M. McClellan, J. Olson, K. Kendrick, and M. Hostetter. 1996. Cloning and expression of a gene encoding an integrin-like protein in *Candida albicans*. Proc. Natl. Acad. Sci. USA 93:357–361.
16. Gibson, F. C., III, A. O. Tzianabos, and F. G. Rodgers. 1994. Adherence of *Legionella pneumophila* to U-937 cells, guinea-pig alveolar macrophages, and MRC-5 cells by a novel, complement-independent binding mechanism. Can. J. Microbiol. 40:865–872.
17. Glick, T. H., M. B. Gregg, B. Berman, G. Mallison, W. W. Rhodes, Jr., and I. Kassanoff. 1978. Pontiac fever. An epidemic of unknown etiology in a health department. I. Clinical and epidemiologic aspects. Am. J. Epidemiol. 107:149–160.
18. Handfield, M., L. J. Brady, A. Progulske-Fox, and J. D. Hillman. 2000. IVIAT: a novel method to identify microbial genes expressed specifically during human infections. Trends Microbiol. 8:336–339.
19. Handfield, M., and R. C. Levesque. 1999. Strategies for isolation of in vivo expressed genes from bacteria. FEMS Microbiol. Rev. 23:69–91.
20. Hazlett, K. R. O., T. J. Sellati, T. T. Nguyen, D. L. Cox, M. L. Clawson, M. J. Caimano, and J. D. Radolf. 2001. The TprK protein of *Treponema pallidum* is periplasmic and is not a target of opsonic antibody or protective immunity. J. Exp. Med. 193:1015–1026.
21. Horwitz, M. A. 1983. The Legionnaires’ disease bacterium (*Legionella pneumophila*) inhibits phagosome-lysosome fusion in human monocytes. J. Exp. Med. 158:2108–2126.
22. Horwitz, M. A. 1984. Phagocytosis of the Legionnaires’ disease bacterium (*Legionella pneumophila*) occurs by a novel mechanism: engulfment within a pseudopod coil. Cell 36:27–33.
23. Horwitz, M. A., and S. C. Silverstein. 1981. Interaction of the Legionnaires’ disease bacterium (*Legionella pneumophila*) with human phagocytes. II. Antibody promotes binding of *L. pneumophila* to monocytes but does not inhibit intracellular multiplication. J. Exp. Med. 153:398–406.
24. Hostetter, M. K. 1999. Integrin-like proteins in *Candida* spp. and other microorganisms. Fungal Genet. Biol. 28:135–145.
25. Hostetter, M. K., N. J. Tao, C. Gale, D. J. Herman, M. McClellan, R. L. Sharp, and K. E. Kendrick. 1995. Antigenic and functional conservation of an integrin I-domain in *Saccharomyces cerevisiae*. Biochem. Mol. Med. 55: 122–130.
26. Husmann, L. K., and W. Johnson. 1992. Adherence of *Legionella pneumophila* to guinea pig peritoneal macrophages, J774 mouse macrophages, and undifferentiated U937 human monocytes: role of Fc and complement receptors. Infect. Immun. 60:5212–5218.
27. Luo, Z. Q., and R. R. Isberg. 2004. Multiple substrates of the *Legionella pneumophila* Dot/Icm system identified by interbacterial protein transfer. Proc. Natl. Acad. Sci. USA 101:841–846.
28. Mahan, M. J., J. M. Slauh, and J. J. Mekalanos. 1993. Selection of bacterial virulence genes that are specifically induced in host tissues. Science 259:686–688.
29. Marra, A., M. A. Horwitz, and H. A. Shuman. 1990. The HL-60 model for the interaction of human macrophages with the Legionnaires’ disease bacterium. J. Immunol. 144:2738–2744.
30. May, A. P., and C. P. Ponting. 1999. Integrin α- and β4-subunit-domain homologues in cyanobacterial proteins. Trends Biochem. Sci. 24:12–13.
31. Mekalanos, J. J. 1992. Environmental signals controlling expression of virulence determinants in bacteria. J. Bacteriol. 174:1–7.
32. Miyamoto, H., S. Yoshida, H. Taniguchi, and H. A. Shuman. 2003. Virulence conversion of *Legionella pneumophila* by conjugal transfer of chromosomal DNA. J. Bacteriol. 185:6712–6718.
33. Mizutani, Y., T. Nakajima, S. Morinaga, M. Gotoh, Y. Shimosato, T. Akino, and A. Suzuki. 1988. Immunohistochemical localization of pulmonary surfactant apoproteins in various lung tumors. Special reference to nonmucus producing lung adenocarcinomas. Cancer 61:532–537.
34. Mody, C. H., R. Paine III, M. S., Shahrabadi, R. H. Simon, E. Pearlman, B. I. Eisenstein, and G. B. Toews. 1993. *Legionella pneumophila* replicates within rat alveolar epithelial cells. J. Infect. Dis. 167:1138–1145.
35. Molofsky, A. B., and M. S. Swanson. 2004. Differentiate to thrive: lessons from the *Legionella pneumophila* life cycle. Mol. Microbiol. 53:29–40.
36. Nash, T. W., D. M. Libby, and M. A. Horwitz. 1984. Interaction between the Legionnaires’ disease bacterium (*Legionella pneumophila*) and human alveolar macrophages: influence of antibody lymphokines and hydrocortisone. J. Clin. Investig. 74:771–782.
37. Payne, N. R., and M. A. Horwitz. 1987. Phagocytosis of *Legionella pneumophila* is mediated by human monocyte complement receptors. J. Exp. Med. 166:1377–1389.
38. Polesky, A. H., J. T. Ross, S. Falkow, and L. S. Tompkins. 2001. Identification of *Legionella pneumophila* genes important for infection of amoebas by signature-tagged mutagenesis. Infect. Immun. 69:977–987.
39. Ransom, D. G., M. D. Hens, and D. W. DeSimone. 1993. Integrin expression in early amphibian embryos: cDNA cloning and characterization of *Xenopus* beta 1, beta 2, beta 3, and beta 6 subunits. Dev. Biol. 160:265–275.
40. Rechnitzer, C., and J. Blom. 1989. Engulfment of the Philadelphia strain of *Legionella pneumophila* within pseudopod coils in human phagocytes. Comparison with other *Legionella* strains and species. APMIS 97:105–114.
41. Reed, L. J., and H. Muench. 1938. A simple method of estimating fifty percent endpoints. Am. J. Hyg. 27:493–497.
42. Reidl, J. 1999. Methods and strategies for the detection of bacterial virulence factors associated with pathogenicity islands, plasmids, and bacteriophages, p. 13–32. In J. B. Kaper, and J. Hacker (ed.), Pathogenicity islands and other mobile virulence elements. American Society for Microbiology, Washington, D.C.
43. Ristroph, J. D., K. W. Hedlund, and R. G. Allen. 1980. Liquid medium for growth of *Legionella pneumophila*. J. Clin. Microbiol. 11:19–21.
44. Rodgers, F. G., and F. C. Gibson, 3rd. 1993. Opsonin-independent adherence and intracellular development of *Legionella pneumophila* within U-937 cells. Can. J. Microbiol. 39:718–722.
45. Rogers, J., and C. W. Keevil. 1992. Immunogold and fluorescein immunolabelling of *Legionella pneumophila* within an aquatic biofilm visualized by using episcopic differential interference contrast microscopy. Appl. Environ. Microbiol. 58:2326–2330.
46. Rowbotham, T. J. 1980. Preliminary report on the pathogenicity of *Legionella pneumophila* for freshwater and soil amoebae. J. Clin. Pathol. 33:1179–1183.
47. Sadosky, A. B., L. A. Wiater, and H. A. Shuman. 1993. Identification of *Legionella pneumophila* genes required for growth within and killing of human macrophages. Infect. Immun. 61:5361–5373.
48. Saito, A., T. Shimoda, M. Nagasawa, H. Tanaka, N. Ito, Y. Shigeno, K. Yamaguchi, M. Hirota, M. Nakatomi, and K. Hara. 1981. The first case of Legionnaires’ disease in Japan. Kansenshogaku Zasshi 55:124–128.
49. Sambrook, J., and D. W. Russell. 2001. Molecular cloning: a laboratory manual, 3rd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.
50. Sanders, C. L., T. A. Jackson, R. R. Adee, G. J. Powers, and P. W. Wehner. 1971. Distribution of inhaled metal oxide particles in pulmonary alveoli. Arch. Intern. Med. 127:1085–1089.
51. Segal, G., M. Purcell, and H. A. Shuman. 1998. Host cell killing and bacterial conjugation require overlapping sets of genes within a 22-kb region of the *Legionella pneumophila* genome. Proc. Natl. Acad. Sci. USA 95:1669–1674.
52. Stone, B. J., and Y. Abu Kwaik. 1998. Expression of multiple pili by *Legionella pneumophila*: identification and characterization of a type IV pilin gene and its role in adherence to mammalian and protozoan cells. Infect. Immun. 66:1768–1775.
53. Swanson, M. S., and B. K. Hammer. 2000. *Legionella pneumophila* pathogenesis: a fateful journey from amoebae to macrophages. Annu. Rev. Microbiol. 54:567–613.
54. Wiater, L. A., A. B. Sadosky, and H. A. Shuman. 1994. Mutagenesis of *Legionella pneumophila* using Tn903dIIlacZ: identification of a growth-phase-regulated pigmentation gene. Mol. Microbiol. 11:641–653.
55. Winn, W. C., Jr., and R. L. Myerowitz. 1981. The pathology of the *Legionella pneumonias*: a review of 74 cases and the literature. Hum. Pathol. 12:401–422.
56. Yanisch-Perron, C., J. Vieira, and J. Messing. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119.
57. Yoshida, S., Y. Goto, Y. Mizuguchi, K. Nomoto, and E. Skamené. 1991. Genetic control of natural resistance in mouse macrophages regulating intracellular *Legionella pneumophila* multiplication in vitro. Infect. Immun. 59:428–432.

Editor: V. J. DiRita
