
JOURNAL OF BACTERIOLOGY, July 2005, p. 4813–4821
0021-9193/05/$08.00+0 doi:10.1128/JB.187.14.4813–4821.2005
Copyright © 2005, American Society for Microbiology. All Rights Reserved.

**Enhancement of Glutamine Utilization in *Bacillus subtilis* through the GlnK-GlnL Two-Component Regulatory System**

Takenori Satomura,$^{1}$ Daisuke Shimura,$^{2}$ Kei Asai,$^{2}$ Yoshito Sadaie,$^{2}$ Kazutake Hirooka,$^{1}$ and Yasutaro Fujita$^{1*}$

Department of Biotechnology, Faculty of Life Science and Biotechnology, Fukuyama University, Fukuyama 729-0292,$^{1}$ and Department of Biochemistry and Molecular Biology, Faculty of Science, Saitama University, Saitama 338-8570,$^{2}$ Japan

Received 11 January 2005/Accepted 13 April 2005

During DNA microarray analysis, we discovered that the GlnK-GlnL (formerly YcbA-YcbB) two-component system positively regulates the expression of the *glsA-glnT* (formerly *ybgJ-ybgH*) operon in response to glutamine in the culture medium on Northern analysis. As a result of gel retardation and DNase I footprinting analyses, we found that the GlnL protein interacts with a region (bases –13 to –56; +1 is the transcription initiation base determined on primer extension analysis of *glsA-glnT*) in which a direct repeat, TTTGTN4T TTTGT, is present. Furthermore, the *glsA* and *glnT* genes were biochemically verified to encode glutaminase and glutamine transporter, respectively.

Various organisms have developed sophisticated signaling systems for eliciting a variety of adaptive responses to their environment. The two-component regulatory system is one such signaling system in prokaryotes, lower eukaryotes, and plants, and it consists of at least two signal transduction proteins (the sensor kinase and the response regulator). The sensor kinase senses environmental or intercellular signals and then transmits them to the response regulator through phosphoryl group transfer. The response regulator often binds to the promoter regions of target genes and regulates their transcription through activation or repression (28).

Genome sequencing of *Bacillus subtilis* has revealed the presence of 36 sensor kinases and 35 response regulators, each of 30 kinase-regulator pairs residing in an operon on the genome (20). According to the classification of sensor kinases as to the sequence around the phosphorylated histidine, CitS, DctS, MalK, and GlnK (formerly YcbA) belong to the same group of kinases (group IV), which are paired with CitT, DctT, MalR, and GlnL (formerly YcbB) on the genome, respectively (9). Out of the four kinase-regulator pairs, three (the exception being GlnKL) have been reported to sense the presence of the respective tricarboxylic acid cycle intermediates in the culture medium and to positively regulate each system of their utilization. The CitST system positively regulates the expression of *citM*, which encodes the Mg-citrate transporter (36), whereas DctST enhances the transcription of the *ydbEFGH* operon, which is involved in the utilization of fumarate and succinate (3). Recently, the MalKR system was reported to trigger the transcription of *ywkA*, which encodes a malic enzyme (8), and that of *maeN* and *yflS* (30), which encodes malate transporters, indicating that this system is involved in the transport and utilization of malate. Thus, only the function of the fourth

system (GlnKL) belonging to this group remained to be characterized.

Glutamine, the best nitrogen source for *B. subtilis*, serves as an amino acid for protein synthesis and a nitrogen donor for the synthesis of various nitrogenous molecules in the cell (10). It also functions as a gauge of the nitrogen supply level in the cell, being a feedback inhibitor of glutamine synthetase. Glutamine synthetase interacting with glutamine is able to trap and inhibit TnrA, which functions as a global regulator under nitrogen-limited conditions (34). Therefore, uptake of glutamine from the medium and its de novo synthesis by glutamine synthetase, as well as glutamine utilization, should be rigorously and highly regulated in the cell.

In this communication, we report that the GlnK-GlnL two-component system positively regulates the expression of the *glsA-glnT* (formerly *ybgJ-ybgH*) operon in response to glutamine in the culture medium. It was verified that the *glsA* and *glnT* genes encode the glutaminase and glutamine transporter, respectively, indicating that the GlnKL system is involved in glutamine utilization.

**MATERIALS AND METHODS**

Bacterial strains and cultivation media. The *B. subtilis* strains used in this study are listed in Table 1. *Escherichia coli* BL21(DE3) (Novagen, Inc.) was used for overproduction of His-tagged GlnL and GluA. *E. coli* BL21(DE3) was cultured in Luria-Bertani (LB) medium (26). *B. subtilis* cells were precultured at 30°C overnight on tryptose blood agar base (Difco) plates supplemented with 0.18% glucose (TBABG) and then cultivated in a liquid minimal medium (MM) plus tryptophan (50 μg/ml) (38) containing 13.6 mM glutamine or 13.6 mM Na glutamate plus 20 mM NH₄Cl as the nitrogen source at 37°C, with shaking. If necessary, ampicillin (50 μg/ml), erythromycin (0.3 μg/ml), or chloramphenicol (5.0 μg/ml) was added to the medium.

*B. subtilis* mutant construction. Strain FU707 carrying a *glnQ* deletion (∆*glnQ::cat*) was constructed as follows. The PCR fragment in which the regions flanking the *glnQ* gene sandwiched the *cat* gene was prepared by means of long-flanking homology PCR (32). The two regions upstream and downstream of *glnQ* were amplified by PCR using primer pairs (*glnQ*-F1 and *glnQ*-F2 and *glnQ*-R1 and *glnQ*-R2, respectively) (Table 2) and DNA of strain 168 as a template. In addition, the chloramphenicol acetyltransferase gene (*cat*) cassette was amplified by PCR using a primer pair (*cat*-M1 and *cat*-M2) (Table 2) and DNA of plasmid pCBB31, a derivative of plasmid pUC118 carrying *cat* from plasmid pC194 (17). Joint PCR (32) involving the above three PCR products and

* Corresponding author. Mailing address: Department of Biotechnology, Faculty of Life Science and Biotechnology, Fukuyama University, 985 Sanzo, Higashimura-cho, Fukuyama-shi, Hiroshima 729-0292, Japan. Phone: 81 84 936 2111. Fax: 81 84 936 2023. E-mail: yfujita@bt.fubt.fukuyama-u.ac.jp.

4814 SATOMURA ET AL.

TABLE 1. *B. subtilis* strains used in this study

| Strain | Genotype | Source |
|--------|----------|--------|
| 168    | *trpC2*  | Wild type (laboratory stock) |
| YCBBd  | *glnL::pMUTIN trpC2* | JAFAN$^a$ |
| YBGJd  | *glsA::pMUTIN trpC2* | JAFAN |
| YBGHd  | *glnT::pMUTIN trpC2* | JAFAN |
| 1A71   | *gltA1 trpC2* | BGSC$^b$ |
| FU707  | *ΔglnQ::cat trpC2* | This study |
| FU708  | *ΔglnQ::cat glnT::pMUTIN trpC2* | This study |
| FU801  | *gltA1 glsA::pMUTIN trpC2* | This study |

$a$ JAFAN, Japan Function Analysis Network for *B. subtilis*.  
$b$ BGSC, Bacillus Genetic Stock Center (Ohio University).

were harvested, washed twice with 50 mM Tris-Cl buffer (pH 8.0), and then suspended in 50 mM Tris-Cl buffer (pH 8.0) containing 10% (vol/vol) glycerol. The cells were disrupted by sonication, and a supernatant was obtained by centrifugation (10,000 × g; 10 min). His-tagged GlnL was purified to homogeneity by nickel affinity chromatography with the use of a His-binding kit (Novagen).

To produce the GlmA protein in *E. coli*, the *glsA* region was amplified by PCR using a primer pair (*glsA-O1–glsA-O2*) (Table 2) and DNA of strain 168 as a template; the reverse primer (*glsA-O2*) was designed to eliminate the authentic NdeI site by replacement of A by G. The resulting PCR product was ligated with vector pCR2.1-TOPO, and the ligated DNA was used for transformation of *E. coli* TOP10, according to the instructions for the TA-Cloning kit (Invitrogen) to generate plasmid pTA-GLSA. After correct cloning of the fragment into plasmid pTA-GLSA had been confirmed by nucleotide sequencing, the plasmid DNA was digested with NdeI and NotI. The resulting fragment was ligated with the NdeI-NotI arm of vector pET-22b (Novagen), and the ligated DNA was used for transformation of *E. coli* BL21(DE3) to produce plasmid pGLSA. The GlmA protein was overexpressed in *E. coli* BL21(DE3) by the addition of IPTG to the medium, and a crude cell extract was prepared as described above.

Gel retardation and DNase I footprinting experiments. Gel retardation and DNase I footprinting experiments were performed as described previously (40). The purified His-tagged GlnL protein was prepared as described above. This protein was treated at 37°C for 1 h in 50 μl of a reaction mixture containing 50 mM Tris-Cl (pH 7.5), 5 mM MgCl₂, 50 mM acetylphosphate (Sigma), and 1 mM dithiothreitol, essentially as described previously (25). For gel retardation analysis, the probe DNA containing the *glsA-glnT* promoter region (bases –205 and +25) was a PCR product amplified from DNA of strain 168 using a primer pair (*glsA-F1–glsA-F2*). The probe was labeled with a BcaBEST labeling kit and [α-³²P]dCTP (Amersham Biosciences). For DNase I footprinting, the same probe DNA was prepared by PCR amplification using the above primers (*glsA-F1–glsA-F2*), either of which were labeled at the 5′ terminus with a MEGA-LABEL kit and [γ-³²P]ATP (Amersham Biosciences).

Paper chromatography to identify the enzyme reaction product of GlmA. The reaction mixture (50 μl) comprising 50 mM potassium phosphate buffer (pH 7.2) and 20 mM L-glutamine was incubated overnight at 30°C after the addition of a crude cell protein extract (10 μg), which had been prepared from *E. coli* BL21(DE3) cells carrying either plasmid pGLSA or pET-22b, as described above. An aliquot (5 μl) of the reaction mixture was subjected to paper chromatography together with authentic L-glutamate and L-glutamine solutions, the developing solvent being n-butanol:acetic acid:H₂O at a ratio of 4:1:1. Spraying with 0.7% ninhydrin localized the spots of the reaction product and two other authentic amino acids on the chromatogram.

a primer pair—a nested primer pair (*glnQ-F11* and *glnQ-R22*) (Table 2)—resulted in a product, which was used for transformation of strain 168 to chloramphenicol resistance (5 μg/ml). Correct deletion of the *glnQ* gene from strain 168 was confirmed by means of PCR and sequencing.

Strain 708 (*ΔglnQ::cat ybgH::pMUTIN*) was constructed by transformation of strain FU707 (*ΔglnQ::cat*) with DNA of strain YBGHd (*glnT::pMUTIN*) to erythromycin resistance (0.3 μg/ml). Strain FU801 (*gltA1 glsA::pMUTIN*) was constructed by transformation of strain 1A71 with PCR products amplified by the use of primer pairs (*glsA-M1/glsA-M2*) (Table 2) to erythromycin resistance (0.3 μg/ml).

β-Galactosidase (β-Gal) assay. *B. subtilis* cells carrying *lacZ* fusions were grown overnight at 30°C on TBABG containing erythromycin (0.3 μg/ml). The cells were inoculated into 50 ml of MM medium containing glutamine or glutamate plus ammonium as the nitrogen source and then incubated at 37°C. During growth, 1-ml aliquots of the culture were withdrawn, and β-Gal activity was determined as described previously (38).

Northern analysis. *B. subtilis* cells were grown in MM medium containing glutamine or glutamate plus ammonium as the nitrogen source at 37°C to optical density at 600 nm (OD₆₀₀) of 0.6 (mid-logarithmic phase). Total RNA was extracted and purified from cell pellets as described previously (39). For Northern analysis, RNA was electrophoresed in a glyoxal gel and then transferred to a Hybond-N membrane (Amersham Biosciences) (26). To prepare probes for the detection of transcripts carrying *glsA*, *glnT*, and *ybgG*, the respective products amplified by PCR using primer pairs (*glsA-N1–glsA-N2*, *glnT-N1–glnT-N2*, and *ybgG-N1/ybgG-N2*) (Table 2) and chromosomal DNA of strain 168 as a template were labeled with a BcaBEST labeling kit (Takara Shuzo Co., Ltd., Kyoto, Japan) and [α-³²P]dCTP (Amersham Biosciences). Hybridization and transcript detection were carried out as described previously (26).

Primer extension. Primer extension analysis was performed essentially as described previously (37). To map the 5′ end of the *glsA-glnT* transcript by primer extension, each of the total RNAs (each, 45 μg) used for Northern analysis was annealed to primer EX-glsA (Table 2), corresponding to bases (+76 to +95) that had been labeled at the 5′ end with a MEGALABEL kit (Takara Shuzo) and [γ-³²P]ATP (Amersham Biociences). Primer extension was carried out with a thermostable reverse transcriptase (Thermoscript; Invitrogen Corp.) in 40 μl of a reaction mixture comprising 50 mM Tris-acetate (pH 8.4), 75 mM potassium acetate, 8 mM magnesium acetate, 1 mM each deoxyribonucleotide triphosphate, 1 mM dithiothreitol, 40 U RNase OUT (Invitrogen), and 30 U Thermoscript at 58°C for 1 h. The resultant cDNAs were subjected to urea-polyacrylamide gel electrophoresis as described previously (12). A template for the dideoxy sequencing reactions for ladder preparation starting from the same end-labeled primer was prepared by PCR using a primer pair (*glsA-F1–EX-glsA*) (Table 2) and DNA from strain 168 as a template.

Production and purification of GlnL and GlmA. To clone and express the *glnL* gene in *E. coli*, the *glnL* region was amplified by PCR using a primer pair (*glnL-O1–glnL-O2*) and DNA of strain 168 as a template. The PCR product, which had been digested with BamHI and SphI, was cloned in *E. coli* JM109 into vector pDGHisN2 with selection of an ampicillin-resistant transformant (50 μg/ml), which was a derivative of plasmid pDG148 (a shuttle between *E. coli* and *B. subtilis*) (29). The *glnL* gene cloned in plasmid pDGHisN2 carrying *lacI* was placed under the control of the isopropyl-β-d-thiogalactopyranoside (IPTG)-inducible Pspac promoter, and its product was His tagged at the N terminus. (Construction of plasmid pDGHisN2 by K. Asai remains to be reported.) When *E. coli* BL21(DE3) cells bearing the resultant plasmid pGLNL had grown to an OD₆₀₀ of 0.4 at 37°C in LB medium containing ampicillin (50 μg/ml), 1 mM IPTG was added to induce His-tagged GlnL. After 4 h of incubation, the cells

TABLE 2. Primers used in this study

| Primer | Sequence |
|--------|----------|
| *glnT-N1* | TGCAATCTTCTGGATGTGGA |
| *glnT-N2* | TTGAAAAGATTCCCCGTCTG |
| *glsA-N1* | GAGCCTGATGGAACGATGAT |
| *glsA-N2* | TCGTGTCGAATCGGATGATA |
| *ybgG-N1* | AACGCAAGGGCTGTAATTTG |
| *ybgG-N2* | TTTGGCTTCAGTCAGACACG |
| Ex-glsA | CGAATTCCTCCTCACTGATT |
| *glsA-F1* | AGGGCCAGTATATATCCGTC |
| *glsA-F2* | AACGCTTTCATAATAGCGGA |
| *glsA-O1* | GAGGAATCATATGAAAGAATTGATAAAAGA |
| *glsA-O2* | TTA AAAAATGCTGAGCTCCCATTCCCTGCGCCATGTG$^b$ |
| *glnL-O1* | GGAGGATCCATGCGTTTTTTATTGCAGAT$^a$ |
| *glnL-O2* | CTCCTCGAGCATGCTTATATCCTTCTTGTTCATC G$^a$ |
| *glsA-M1* | CTTCATGAATCTCTCCGGCCAGTCTCAGGA |
| *glsA-M2* | GGCATCCGAAAAACAGAATGATAAAAGCGGT |
| *glnQ-F1* | CAACAAGAACCGCCTTGACT |
| *glnQ-F11* | TCGCAATTAACAAACGCAGGAACAGCTCCGA |
| *glnQ-F2* | GTTATCCGCTCACAAATTCCGCAATCCTCCTATCAGAA |
| *glnQ-R1* | CGTCGTGACTGGAAAAACTAAAAACAGAAATGGGGGAGC |
| *glnQ-R2* | ACTGCCGGCCAGCGCGATA |
| *glnQ-R22* | ACATTGCGAGCAAAGGTATGGTAAAAACCG |
| cat-M1 | GAATTGTGAGCGGATAAC |
| cat-M2 | GTTTTCCCAGTCACGACG |

$a$ Restriction enzyme sites are underlined.  
$b$ The boldface G was introduced in place of A to eliminate the NdeI site.

Measurement of L-glutamine uptake. L-Glutamine uptake by strains of *B. subtilis* was measured essentially by a previously described method (13, 40). After cells had been grown at 37°C to an OD<sub>600</sub> of 1.0 in MM medium containing glutamate plus ammonium as the nitrogen source with or without 5 mM IPTG, 6 ml of the culture was harvested and washed in MM medium plus chloramphenicol (100 μg/ml). The washed cells were suspended in 4 ml of the same medium plus chloramphenicol (100 μg/ml) and then used for measurements. L-Glutamine uptake assays were performed with a final L-glutamine concentration of 500 μM. The assay tubes held 0.7 ml of MM medium plus chloramphenicol (100 μg/ml) containing L-[U-<sup>14</sup>C] glutamine (22.2 kBq; 9,250 MBq/mmol) (American Radiolabeled Chemicals, Inc.) and 930 μM L-glutamine and were then kept at 37°C. L-Glutamine uptake was initiated by the addition of 0.6 ml of a cell suspension to each assay tube. After incubation at 37°C for 3 min, the suspension was filtered through a moistened glass microfiber filter (2.4-cm diameter) (GF/C; Whatman). The filter was immediately washed three times with 5 ml of MM medium plus chloramphenicol (100 μg/ml). The filter was dried, and radioactivity was counted in 10 ml of ASCII scintillant (Amersham Biosciences).

**RESULTS**

Induction of the *glsA-glnT* operon by glutamine, which is positively regulated through the GlnK-GlnL two-component regulatory system. Comprehensive DNA microarray analysis of *B. subtilis* two-component regulatory systems (19) revealed that one of the candidate target genes positively regulated by the GlnKL system is *glnT*, which is adjacent to and located divergently from the *glnKL* genes and appears to constitute an operon with *glsA* (Fig. 1A). The GlsA protein exhibited high similarities to glutaminases from various organisms in a protein similarity search involving the BLASTP 2.2.9 program (2; data not shown). A GlsA homolog of *Rhizobium etli* has been verified to be a glutaminase (5). So, we performed Northern analysis with specific probes for *glsA*, *glnT*, and *ybgG* to reveal the operon organization of the three genes and to determine if these genes are induced by glutamine and positively regulated by GlnKL (Fig. 1B). *B. subtilis* cells grow almost equally well on glutamine and glutamate plus ammonium as the nitrogen source in MM medium (11). When total RNA samples prepared from cells of strain 168 grown on glutamine (Fig. 1B, lanes 1, 4, and 7) and glutamate plus ammonium (lanes 2, 5, and 8) as nitrogen sources and from cells of strain YCBBd (*glnL::pMUTIN*) grown on glutamine (lanes 3, 6, and 9) were subjected to Northern analysis (Fig. 1B), a transcript (2.5 kb) was only detected in total RNA from strain 168 grown on glutamine with the *glsA* and *glnT* probes (lanes 1 and 4). A 1.1-kb transcript, however, was only detected with the *ybgG* probe in all total RNAs (lanes 7 to 9); the 2.5- and 1.1-kb transcripts were considered to include *glsA* and *glnT*, and *ybgG*, respectively (Fig. 1A). The results indicated that an operon comprising the *glsA* and *glnT* genes (not *ybgG*) was induced by glutamine in the medium; this induction was under positive regulation through the GlnKL system. Moreover, β-Gal synthesis in strain YBGHd (*glnT::pMUTIN*) with a *lacZ* fusion with *glnT* was confirmed to be induced by glutamine in the medium (data not shown). Thus, it is highly possible that GlnK undergoes autophosphorylation when it senses glutamine in the medium and transfers the phosphoryl group to the cognate response regulator of GlnL, which triggers transcription of the *glsA-glnT* operon. Nevertheless, strain YCBBd (*glnL::pMUTIN*) was able to grow well on glutamine as the sole nitrogen source at the same doubling rate as the wild-type strain 168 grew (data not shown).

5′ mapping of the *glsA-glnT* transcript. Northern analysis (Fig. 1B) suggested that the *glsA-glnT* operon was likely transcribed from a site upstream of *glsA* to produce the 2.5-kb transcript. To determine the 5′ end of this transcript, primer extension analysis was performed with the same RNA samples as those used for Northern analysis. As shown in Fig. 2, a specific band of runoff cDNA was only detected with total RNA prepared from cells of strain 168 grown on glutamine (lane 2), i.e., not with total RNA from ones grown on glutamate plus ammonium (lane 1), which coincided with the results of Northern analysis; the size of the cDNA indicated that the 5′ end of the *glsA-glnT* transcript was 95 bp upstream of the translation initiation site for *glsA*. This mapping allowed us to predict the sequences of the −10 and −35 regions of the *glsA-glnT* promoter as TATACA and TTGTAT, respectively, with a 17-bp gap, which is likely recognized by σ<sup>A</sup>-RNA polymerase (14).

In addition, we found a ρ-independent terminator located between 2,544 and 2,576 bases from the initiation base for *glsA-glnT* transcription (Fig. 1A), the location of which was immediately downstream of *glnT* and coincided well with the appearance of the 2.5-kb long *glsA-glnT* transcript (Fig. 1B).

In vitro localization of the binding site for GlnL in the *glsA-glnT* promoter region. The *glsA-glnT* transcription induced by glutamine was positively regulated by the response regulator of GlnL, which is cognate to the sensor kinase of GlnK (Fig. 1B). To determine if GlnL binds to the *glsA-glnT* promoter region for its activation by means of gel retardation assaying, we induced the His-tagged GlnL protein in *E. coli* BL21(DE3) cells carrying plasmid pGLNL by the addition of IPTG and purified it by nickel affinity chromatography to homogeneity (Fig. 3A). As shown in Fig. 3B, the probe spanning bases −205 to +25 became increasingly retarded as the amount of His-tagged GlnL in the assay mixture increased, forming a band corresponding to a protein-DNA complex. The results clearly indicated that the His-tagged GlnL protein bound to the promoter region between bases −205 and +25. The equilibrium dissociation constant for the interaction of His-tagged GlnL with the probe under our assay conditions was roughly estimated to be 500 nM. Thus, a half shift of the probe was observed when the protein/DNA concentration ratio was 625; this suggests that, as demonstrated for other response regulators (22), GlnL must be phosphorylated to be fully active in DNA binding.

To raise the affinity of His-tagged GlnL for its DNA-binding site to physiologically relevant levels, we treated the protein with acetylphosphate, which acts as a substrate for autophosphorylation of many response regulators (33). As shown in Fig. 3C, acetylphosphate-treated GlnL exhibited only a 1.5-fold increase in binding affinity to the *glsA-glnT* promoter region compared to that treated without acetylphosphate. However, the equilibrium dissociation constant for GlnL protein treated without acetylphosphate was 600 nM, which was higher than that of the untreated protein, possibly due to partial denaturation of the GlnL protein during this treatment.

To further localize the GlnL-binding site, DNase I footprinting was performed with the same probe as that used for gel retardation analysis. As shown in Fig. 4A, specific interaction of His-tagged GlnL with the coding and noncoding strands was

FIG. 1. Organization and transcription of the *glsA-glnT* operon. (A) The *glsA*, *glnT*, and *ybgG* genes are oriented divergently from the *glnKL* genes. Northern and primer extension analyses in this work revealed that the *glsA* and *glnT* genes constituting an operon are transcribed from the promoter of the *glsA-glnT* (*P<sub>glsA-glnT</sub>*) promoter to produce a 2.5-kb mRNA, whereas the *ybgG* gene is likely transcribed monocistronically to form a 1.1-kb transcript. The two hairpin structures considered ρ-independent transcription terminators were found immediately downstream of *glnT* and *ybgG*, respectively. (B) Northern analysis of transcription of the *glsA* and *glnT* genes. The RNA samples from strains 168 (lanes 1, 2, 4, 5, 7, and 8) and YCBBd (*glnL::pMUTIN*) (lanes 3, 6, and 9) grown on glutamine (lanes 1, 3, 4, 6, 7, and 9) and glutamate plus ammonium (lanes 2, 5, and 8) were subjected to Northern analysis using *glsA*, *glnT*, and *ybgG* probes. The arrows indicate a 2.5-kb transcript detected with the *glsA* and *glnT* probes and a 1.1-kb transcript detected with the *ybgG* probe; the former transcript (2.5 kb) was induced with glutamine. The two faint bands (2.8 and 1.7 kb), indicated by arrowheads, resulted from unspecific hybridization to 23S and 16S rRNAs, respectively.

observed in a concentration-dependent manner (Fig. 4B). The protected region (bases –13 to –56) covered the –35 region of the *glsA-glnT* promoter, and sequence examination revealed a direct repeat, TTTTGTTN4TTTTGT (Fig. 4B). Furthermore,

the acetylphosphate-treated GlnL specifically protected the same region as the untreated protein did, at a lower concentration than that of the protein treated without acetylphosphate (data not shown).

FIG. 2. Mapping of the 5′ end of the *glsA-glnT* transcript by means of primer extension. Total RNAs from strain 168 grown in glutamate and ammonium (lane 1) and glutamine (lane 2) were annealed with the Ex-*glsA* primer, and then primer extension was performed as described in the text. Lanes G, A, T, and C contained the products of the respective dideoxy sequencing reactions, as described in the text. The arrow indicates the runoff cDNA resulting from primer extension. The part of the nucleotide sequence of the coding strand corresponding to the ladder is shown with the transcription initiation base (+1) determined in this analysis, and the corresponding –10 and –35 regions are underlined.

FIG. 3. Gel retardation analysis of GlnL interaction with the *glsA-glnT* promoter region. (A) Sodium dodecyl sulfate-polyacrylamide gel electrophoresis of pure His-tagged GlnL. The purified His-tagged GlnL protein (lane 1) and a crude protein extract of cells of *E. coli* BL21(DE3) bearing plasmid pGLNL exposed to IPTG (lane 2) were subjected to sodium dodecyl sulfate-polyacrylamide gel electrophoresis on a 12.5% gel. The preparation of the crude protein extract and purification of the His-tagged GlnL are described in the text. The arrow indicates His-tagged GlnL. (B) Gel retardation analysis of the binding of His-tagged GlnL to the 230-bp probe containing its recognition site. The analysis was performed as described in the text. The probe (0.02 pmol) was incubated with 0, 0.9, 1.7, 3.4, 6.9, and 13.7 pmol of His-tagged GlnL (lanes 1, 2, 3, 4, 5, and 6, respectively) in a 25-μl reaction mixture. The addition of more His-tagged GlnL to the reaction mixture was not performed due to the low concentration of the protein preparation. The His-tagged GlnL/probe complexes (bound) and free probe (free) are indicated on the right. (C) The increased binding affinity of autophosphorylated His-tagged GlnL to the probe. His-tagged GlnL was treated in the absence and presence of 50 mM acetylphosphate, respectively (*GlnL* treated and *GlnL~P* treated), as described in the text. The probe (0.02 pmol) was incubated with 0, 3.3, 6.6, 9.9, 13.2, and 16.5 pmol of His-tagged GlnL treated without and with acetylphosphate (lanes 1 to 6) in a 25-μl reaction mixture. Lane 7 contains the probe incubated with 33.0 pmol of His-tagged GlnL treated without acetylphosphate.

Functional analysis of the *glsA* and *glnT* genes. As the *glsA-glnT* operon is specifically induced by glutamine in the culture medium, we examined if the two genes are actually involved in glutamine utilization. To verify that *glsA* encodes glutaminase, we cloned and expressed it in *E. coli*. A reaction mixture containing glutamine was incubated with a protein extract prepared from cells bearing either plasmid pGLSA possessing *glsA* or vector pET-22b, both of which had been grown with IPTG, and then submitted to paper chromatography, together with authentic glutamine and glutamate solutions (Fig. 5). A ninhydrin spot corresponding to glutamate was observed on the paper chromatogram only on incubation with the protein extract from cells bearing plasmid pGLSA (Fig. 5, lane 4), clearly indicating that GlsA is a glutaminase.

Glutaminase produces glutamate from glutamine; glutamate synthase, whose large subunit is encoded by *gltA* (4, 7), forms two glutamate molecules from glutamine and α-ketoglutarate through the aminotransferase reaction. However, not only strain YBGJd carrying the pMUTIN disruption of the *glsA* gene encoding glutaminase but also strain 1A71 possessing the *gltAl* mutation grew in MM medium containing glutamine as the sole nitrogen source, as well as the wild-type strain 168 (Fig. 6). Thus, we constructed strain FU801 harboring both the *glsA::pMUTIN* and *gltAl* mutations to see if it could grow on glutamine as the sole nitrogen source. As shown in Fig. 6, strain FU801 grew slowly with a prolonged doubling time, in contrast to the two strains carrying *glsA::pMUTIN* and *gltAl*, respectively. These results indicated that the *glsA* gene is actually involved in glutamine utilization.

A BLAST similarity search regarding the GlnT protein indicated that this protein belongs to the sodium:alanine symporter family (24). Since another constituent of the *glsA-glnT*

4818 SATOMURA ET AL.

J. BACTERIOL.

A

coding strand
[GlnL]
G A T C 1 2 3 4

non-coding strand
[GlnL]
G A T C 1 2 3 4

B

TGCTTTTGACCCAGAACATGAGGTTTTTATAAAACTTACTGCTTATTTTCAAAATGTTTTTGTGCTATTTTGTATGA
ACGAAAACCTGGGTCTTGTACTCCAAAAATATTTAGAATGACGAATAAAAGTTTTACAAAAAACAGCATAAAACATACT
-10 +1
TTCTGTAGTCTCCATTATACAGTAATTAACCTTCCGCTATTATGAAAGCGTTTAACGCCCCCTTAGCCTTCGTTT
AAGACATCAGAGGTAATATGTCATTAATATGGAAAGGCGATAATACTTTTGCACAAATTGCGGGGGGAATCGGAAAGCAAAA
GATACACCTCAGTCCTTTAAGAAAAAAAAATCAGTGAGGAGGAATTCGATG
CTATGTGGAGTCAGGAAATTCTTTTTTTAGTCACTCCTCCTTAAGCTAC

FIG. 4. DNase I footprinting analysis of the interaction of GlnL with the *glsA-glnT* region. (A) The analysis was performed as described in the text. The left and right panels are DNase I footprints of the 5′-end-labeled coding and noncoding strands of the DNA probe, respectively. Lanes 1 to 4 contained 0.04 pmol of the <sup>32</sup>P-labeled probe DNA in the reaction mixture (50 μl). Lane 1 contained no His-tagged GlnL. Lanes 2, 3, and 4 contained 27.4, 20.5, and 13.7 pmol of His-tagged GlnL. Lanes G, A, T, and C contained the products of the respective sequencing reactions performed with the same primers as those used for the probe preparation. The protected areas of the coding and noncoding strands are boxed. (B) The nucleotide sequences of the coding and noncoding strands of the region upstream of the *glsA-glnT* promoter, with the protected sequences (in boldface type), and the −35 and −10 regions of the *glsA-glnT* promoter (+1 is the transcription initiation base) indicated. The boxed T and ATG are the transcription initiation base and translation initiation codon, respectively. Arrows indicate directly repeated sequences (TTTTGT).

operon, *glsA*, was found to encode glutaminase, which is involved in glutamine utilization, it is highly likely that the *glnT* gene encodes the sodium:glutamine symporter. To determine if the GlnT protein is a sodium:glutamine symporter, we measured glutamine uptake by cells of strains YBGJd (*glsA::pMUTIN*) and YBGHd (*glnT::pMUTIN*) (Fig. 7). The *glnT* gene was disrupted through the integration of plasmid pMUTIN, resulting in strain YBGHd, whereas it was under the control of an IPTG-inducible *Pspac* promoter in strain YBGJd. As shown in Fig. 7, glutamine uptake into YBGJd cells was significantly induced by IPTG in the culture medium. At a glutamine concentration of 500 μM, rates of glutamine uptake were calculated to be 12.1 ± 0.4 and 8.0 ± 0.4 nmol/min per OD<sub>600</sub> unit with and without IPTG induction, respectively. But glutamine uptake into YBGHd cells was not induced by IPTG and remained at a level similar to that into YBGJd cells not exposed to IPTG. These results suggested that the *glnT* gene encodes a

glutamine transporter, probably a sodium:glutamine symporter.

As shown in Fig. 7, the cells of strain YBGHd (*glnT::pMUTIN*) still contained two-thirds the glutamine transporter activity detected in cells of strain YBGJd (*glsA::pMUTIN*) grown with IPTG, indicating that *B. subtilis* cells possess another glutamine transporter(s). *B. subtilis* likely possesses the ATP-binding cassette (ABC) transporter system (15) for glutamine, probably comprising GlnQHMP proteins, as was inferred from a BLAST search that showed that the GlnQH proteins (probable ATP- and glutamine-binding proteins) exhibited high similarities to the corresponding ABC glutamine transporter proteins of *B. stearothermophilus* (35; data not shown). Actually, strains YBGHd (*glnT::pMUTIN*) and FU707 (*glnQ::cat*) grew on glutamine as the sole nitrogen source, as well as the wild-type strain 168 (data not shown). So, we constructed strain FU708, doubly mutated in the *glnT* and *glnQ*

IV comprise four sensor kinases (CitS, DctS, MalK, and GlnK), and their cognate response regulators are CitT, DctR, MalR, and GlnL; the functions of all four two-component systems are now known to be in the utilization of citrate, fumalate and succinate, malate, and glutamine, respectively. The former three systems are especially involved in utilization of the intermediates of the tricarboxylic acid cycle, which is different in the case of the GlnK-GlnL system. It is notable that, except for GlnL and GlnK, the sensor kinases (CitS, DctR, and MalK) and response regulators (CitT, DctR, and MalR) belong to the CitA and CitB subfamilies in *Klebsiella pneumoniae*, respectively (18).

Using the SOSUI system (16) to predict the transmembrane segments for the GlnK protein (387 amino acids) and a search for its protein sequence motif ([http://motif.genome.jp](http://motif.genome.jp)) revealed that this protein possesses four transmembrane segments at the N terminal, as reported previously (9), and a histidine phosphotransferase domain (106 amino acids) at the C terminal. To examine if the external domains of GlnK resemble substrate-binding sites of transport proteins, we subjected the amino acid sequences of the corresponding regions to the protein similarity search using the BLASTP 2.2.3 program (2), which revealed no significant similarity to any protein. On the other hand, the GlnL protein (314 amino acids) carries a response regulator receiver domain (193 amino acids) at the N terminal, which was detected with the above motif search, and a helix-turn-helix motif consisting of 22 amino acids (amino acids 216 to 237), which was predicted by NPS@ (network protein sequence analysis) (6).

GlnK probably senses glutamine in the medium, because it possesses four transmembrane segments, as described above. It is most likely that GlnK autophosphorylated at a histidine residue transfers a phosphoryl group to an aspartate residue of

FIG. 5. Catalyzation of deamination of glutamine to glutamate by the GlsA protein. After incubation of the reaction mixture containing glutamine with the crude protein extract prepared from *E. coli* BL21(DE3) cells carrying either plasmid pET-22b (lane 3) or pGLSA (lane 4) exposed to IPTG, an aliquot of the mixture was subjected to paper chromatography as described in the text. Lanes 1 and 2 contained authentic Na glutamate and glutamine, respectively. The spots were visualized by spraying a ninhydrin solution. Arrows indicate the positions of the glutamine and glutamate spots.

genes, but it grew as well on glutamine as the sole nitrogen source as the wild-type strain 168 did (data not shown). These results strongly suggest that *B. subtilis* possesses a glutamine transport system(s) other than the GlnQHMP complex and GlnT protein.

## DISCUSSION

On comprehensive DNA microarray analysis of *B. subtilis* two-component regulatory systems (19), we revealed that glutamine in the medium is able to trigger expression of the *glsA-glnT* operon, which encodes glutaminase and the glutamine transporter, through signal transduction via a two-component regulatory system (GlnK-GlnL) (Fig. 1, 5, 6, and 7). The other candidate target genes for positive regulation of GlnK-GlnL, which had been indicated on DNA microarray analysis (19), were found to be false positives on in vivo and in vitro analysis (K. Asai and Y. Sadaie, unpublished data). On primer extension and retardation and DNase I footprinting analyses, we found that the GlnL protein interacts with a region (bases –13 to –56) in which a direct repeat, TTTTGTN 4TTTGT, is present (Fig. 2, 3, and 4). We were unable to find this sequence on the *B. subtilis* genome, implying that the *glsA-glnT* operon might be the only target of the GlnK-GlnL system.

The sensor kinases of the two-component regulatory systems are classified into four groups according to the sequence around the phosphorylated histidine (9). The members of group

![Graph](attachment:graph.png)

Time (h)

FIG. 6. Some defect in utilization of glutamine as the nitrogen source in *B. subtilis* strain mutated in the *gltA* and *glsA* genes, which encoded glutamate synthase and glutaminase, respectively. Cells of strains 168 (open circles), YBGJd (*glsA::pMUTIN*) (open squares), 1A71 (*gltA1*) (open triangles), and FU801 (*glsA::pMUTIN gltA1*) (closed circles) were grown on TBABG plates at 30°C for 14 h. Then the cells were inoculated, to an OD<sub>600</sub> of 0.05, into MM medium (38) containing 13.6 mM glutamine as the sole nitrogen source and incubated at 37°C with shaking.

FIG. 7. Glutamine transport into cells of the pMUTIN integrant strains YBGJd (glsA::pMUTIN) and YBGHd (glnT::pMUTIN). Cells of strains YBGJd (bars 1 and 2) and YBGHd (bars 3 and 4) were grown with (bars 1 and 3) or without (bars 2 and 4) IPTG, and the glutamine transport into cells was measured as described in the text. The values are averages with standard deviations and were obtained in duplicate experiments.

GlnL, which positively regulates the expression of the glsA-glnT operon through binding to the promoter region (Fig. 3 and 4). The glsA-glnT operon is adjacent to glnKL (Fig. 1A) but is transcribed diversely with respect to the direction of transcription. A search for orthologs of the glnK, L, glsA, and glnT genes in complete genome sequences of bacteria in the MBGD database (http://mbgd.genome.ad.jp) revealed that glsA orthologs are adjacent to oppositely directed glnKL ones in the Bacillus cereus, Bacillus thuringiensis, and Bacillus halodurans genomes, as well as in the B. subtilis genome, implying that these glnKL orthologs might possibly be involved in glutamine transport and utilization in these bacilli.

Acetylphosphate-treated GlnL exhibited only a 1.5-fold increase in binding affinity for the glsA-glnT promoter region. Hence, it is uncertain whether or not acetylphosphate acts as an autophosphorylation substrate, as reported for other response regulators (1, 21, 33). Of the B. subtilis response regulators, the binding affinity of ComA (25) and CitT (36) to the targets was reported to be enhanced approximately 4 and 1.2 fold, respectively. B. subtilis cultures growing in the presence of excess carbohydrate normally excrete acetate as one of the major by-products during exponential growth phase (27). The ackA and pta genes encode acetate kinase and phosphotransacetylase, respectively, which catalyze the conversion of acetyl-coenzyme A to acetate via an acetyl phosphate intermediate and are direct targets of CcpA-dependent positive regulation occurring during glycolic cell growth (23, 31). Thus, the in vivo concentration of acetyl phosphate is supposed to be elevated during cell growth in the presence of excess carbohydrate, which might cause autophosphorylation of several response regulators to adjust to these physiological conditions, just as in E. coli (33). Further investigation of autophosphorylation of the response regulator of B. subtilis would be necessary to unveil the physiological role of the autophosphorylation of the response regulators.

B. subtilis appears to contain multiple systems for glutamine utilization, as inferred from the fact that mutant strains deficient in glsA and gltT, respectively, were able to normally utilize glutamine as the sole nitrogen source. Glutamine serves as an amino acid for protein synthesis and a nitrogen donor for the synthesis of various nitrogenous molecules in the cell (10), so this organism has developed several compensatory systems for glutamine supply and utilization to maintain the concentration of glutamine in the cell. Glutamine is now known to be metabolized not only by glutamate synthetase encoded by gltA but also by glutaminase encoded by glsA. Moreover, B. subtilis strains doubly mutated in the glsA and gltA genes were able to grow on glutamine, although the doubling time was longer than those of the singly mutated strains (Fig. 6). This suggests that a glutamine utilization enzyme(s) other than glutaminase and glutamate synthetase should be present in B. subtilis, such as another candidate glutaminase (YlaM) and/or nitrogen-donating enzymes for the synthesis of various nitrogenous molecules in the cell. On the other hand, this organism was shown to possess a glutamine transporter (GlnT), probably a sodium:glutamine symporter (Fig. 7). Besides GlnT, this organism likely contains an ATP-binding cassette transporter system for glutamine, probably comprising GlnQHMP proteins; GlnQ is a probable glutamine-binding protein. However, even a strain doubly mutated in glnQ and glnT grew well on glutamine as the sole nitrogen source, like wild-type strain 168, indicating that B. subtilis possesses an unidentified glutamine transport system(s) other than the GlnQHMP complex and GlnT proteins.

ACKNOWLEDGMENTS

We are grateful to Naotake Ogasawara for his critical suggestions regarding the functions of the unanalyzed two-component regulatory systems based on the ortholog locations on several genome sequences of gram-positive bacteria. We also thank S. Miyazaki and K. Taniguchi for their help.

This work was supported by a Grant-in-Aid for Scientific Research on Priority Areas and “High-Tech Research Center” Project for Private Universities from the Ministry of Education, Culture, Sports Science and Technology of Japan.

REFERENCES

1. Abo-Amer, A. E., J. Munn, K. Jackson, M. Aktas, P. Golby, D. J. Kelly, and S. C. Andrews. 2004. DNA interaction and phototransfer of the C4-

dicarboxylate-responsive DcuS-DcuR two-component regulatory system from *Escherichia coli*. J. Bacteriol. 186:1879–1889.

2. Altschul, S. F., T. L. Madden, A. A. Schaffer, J. Zhang, Z. Zhang, W. Miller, and D. J. Lipman. 1997. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 25:3389–3402.

3. Asai, K., S. H. Baik, Y. Kasahara, S. Moriya, and N. Ogasawara. 2000. Regulation of the transport system for C₄-dicarboxylic acids in *Bacillus subtilis*. Microbiology 146:263–271.

4. Bohannon, D. E., M. S. Rosenkrantz, and A. L. Sonenshein. 1985. Regulation of *Bacillus subtilis* glutamate synthase genes by the nitrogen source. J. Bacteriol. 163:957–964.

5. Calderon, J., A. Huerta-Saquero, G. Du Pont, and S. Duran. 1999. Sequence and molecular analysis of the *Rhizobium etli glsA* gene, encoding a thermolabile glutaminase. Biochim. Biophys. Acta 1444:451–456.

6. Combet, C., C. Blanchet, C. Geourjon, and G. Deleage. 2000. NPS@: network protein sequence analysis. Trends Biochem. Sci. 25:147–150.

7. Deshpande, K. L., and J. F. Kane. 1980. Glutamate synthase from *Bacillus subtilis*: in vitro reconstitution of an active amidotransferase. Biochem. Biophys. Res. Commun. 93:308–314.

8. Doan, T., P. Servant, S. Tojo, H. Yamaguchi, G. Lerondel, K. Yoshida, Y. Fujita, and S. Ameyrich. 2003. The *Bacillus subtilis ywkA* gene encodes a malic enzyme and its transcription is activated by the YufL/YufM two-component system in response to malate. Microbiology 149:2331–2343.

9. Fabret, C., V. A. Feher, and J. A. Hoch. 1999. Two-component signal transduction in *Bacillus subtilis*: how one organism sees its world. J. Bacteriol. 181:1975–1983.

10. Fisher, S. H. 1999. Regulation of nitrogen metabolism in *Bacillus subtilis*: vive la différence. Mol. Microbiol. 32:223–232.

11. Fisher, S. H., and L. V. Wray, Jr. 2002. Mutations in the *Bacillus subtilis glnRA* operon that cause nitrogen source-dependent defects in regulation of TnrA activity. J. Bacteriol. 184:4636–4639.

12. Fujita, Y., T. Fujita, Y. Miwa, J. Nishashi, and Y. Aratani. 1986. Organization and transcription of the gluconate operon, *gnt*, of *Bacillus subtilis*. J. Biol. Chem. 261:13744–13753.

13. Fujita, Y., J. Nishashi, and T. Fujita. 1986. The characterization and cloning of a gluconate (*gnt*) operon of *Bacillus subtilis*. J. Gen. Microbiol. 132:161–169.

14. Haldenwang, W. G. 1995. The sigma factors of *Bacillus subtilis*. Microbiol. Rev. 59:1–30.

15. Higgins, C. F. 1992. ABC transporters: from microorganisms to man. Annu. Rev. Cell Biol. 8:67–113.

16. Hirokawa, T., S. Boon-Chieng, and S. Mitaku. 1998. SOSUI: classification and secondary structure prediction system for membrane proteins. Bioinformatics 14:378–379.

17. Horinouchi, S., and B. Weisblum. 1982. Nucleotide sequence and functional map of pC194, a plasmid that specifies inducible chloramphenicol resistance. J. Bacteriol. 150:815–825.

18. Kaspar, S., R. Perozzo, S. Reinelt, M. Meyer, K. Pfister, L. Scapozza, and M. Bott. 1999. The periplasmic domein of the histidine autokinase CitA functions as a highly specific citrate receptor. Mol. Microbiol. 33:858–872.

19. Kobayashi, K., M. Ogura, H. Yamaguchi, K. Yoshida, N. Ogasawara, T. Tanaka, and Y. Fujita. 2001. Comprehensive DNA microarray analysis of *Bacillus subtilis* two-component regulatory systems. J. Bacteriol. 183:7365–7370.

20. Kunst, F., N. Ogasawara, I. Moszer, et al. 1997. The complete genome sequence of the gram-positive bacterium *Bacillus subtilis*. Nature 390:249–256.

21. Meyer, M., P. Dimroth, and M. Bott. 1997. In vitro binding of the response regulator CitB and of its carboxy-terminal domain to A + T-rich DNA target sequences in the control region of the divergent *citC* and *citS* operons of *Klebsiella pneumoniae*. J. Mol. Biol. 269:719–731.

22. Perego, M., and J. A. Hoch. 2002. Two-component systems, phosphorelays,

and regulation of their activities by phosphatases, p. 473–481. *In* A. L. Sonenshein, J. A. Hoch, and R. Losick (ed.), *Bacillus subtilis* and its closest relatives: from genes to cells. American Society for Microbiology, Washington, D.C.

23. Prescan-Siedel, E., A. Galinier, R. Longin, J. Deutscher, A. Danchin, P. Glaser, and I. Martin-Verstraete. 1999. Catabolite regulation of the *pta* gene as part of carbon flow pathways in *Bacillus subtilis*. J. Bacteriol. 181:6889–6897.

24. Reizer, J., A. Reizer, and M. H. Saier, Jr. 1994. A functional superfamily of sodium/solute symporters. Biochim. Biophys. Acta 1197:133–166.

25. Roggiant, M., and D. Dubnau. 1993. ComA, a phosphorylated response regulator protein of *Bacillus subtilis*, binds to the promoter region of *srfA*. J. Bacteriol. 175:3182–3187.

26. Sambrook, J., and D. W. Russell. 2001. Molecular cloning: a laboratory manual, 3rd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

27. Speck, E. L., and E. Freese. 1973. Control of metabolite secretion in *Bacillus subtilis*. J. Gen. Microbiol. 78:261–275.

28. Stock, J. B., M. G. Surette, M. Levit, and P. Park. 1995. Two-component signal transduction system: structure-function relationships and mechanisms of catalysis, p. 24–51. *In* J. A. Hoch and T. J. Silhavy (ed.), Two-component signal transduction. American Society for Microbiology, Washington, D.C.

29. Stragier, P., C. Bonamy, and C. Karmazyn-Campelli. 1988. Processing of a sporulation sigma factor in *Bacillus subtilis*: how morphological structure could control gene expression. Cell 52:697–704.

30. Tanaka, K., K. Kobayashi, and N. Ogasawara. 2003. The *Bacillus subtilis* YufLM two-component system regulates the expression of the malate transporters MaeN (YufR) and YflS, and is essential for utilization of malate in minimal medium. Microbiology 149:2317–2329.

31. Turinsky, A. J., F. J. Grundy, J. Kim, G. H. Chambliss, and T. M. Henkin. 1998. Transcriptional activation of the *Bacillus subtilis ackA* gene requires sequences upstream of the promoter. J. Bacteriol. 180:5961–5967.

32. Wach, A. 1996. PCR-synthesis of marker cassettes with long flanking homology regions for gene disruptions in *S. cerevisiae*. Yeast 12:259–265.

33. Wanner, B. L. 1995. Signal transduction and cross regulation in the *Escherichia coli* phosphate regulon by PhoR, CreC, and acetyl phosphate, p. 203–221. *In* J. Hoch and T. Silhavy (ed.), Two-component signal transduction. American Society for Microbiology, Washington, D.C.

34. Wray, L. V., Jr., J. M. Zalieckas, and S. H. Fisher. 2001. *Bacillus subtilis* glutamine synthetase controls gene expression through a protein-protein interaction with transcription factor TnrA. Cell 107:427–435.

35. Wu, L., and N. E. Welker. 1991. Cloning and characterization of a glutamine transport operon of *Bacillus stearothermophilus* NUB36: effect of temperature on regulation of transcription. J. Bacteriol. 173:4877–4888.

36. Yamamoto, H., M. Murata, and J. Sekiguchi. 2000. The CitST two-component system regulates the expression of the Mg-citrate transporter in *Bacillus subtilis*. Mol. Microbiol. 37:898–912.

37. Yoshida, K., D. Aoyama, I. Ishio, T. Shibayama, and Y. Fujita. 1997. Organization and transcription of the myo-inositol operon, *iol*, of *Bacillus subtilis*. J. Bacteriol. 179:4591–4598.

38. Yoshida, K., I. Ishio, E. Nagakawa, Y. Yamamoto, M. Yamamoto, and Y. Fujita. 2000. Systematic study of gene expression and transcription organization in the *gntZ-ywaA* region of the *Bacillus subtilis* genome. Microbiology 146:573–579.

39. Yoshida, K., K. Kobayashi, Y. Miwa, C. Kang, M. Matsunaga, H. Yamaguchi, S. Tojo, M. Yamamoto, R. Nishi, N. Ogasawara, T. Nakayama, and Y. Fujita. 2001. Combined transcriptome and proteome analysis as a powerful approach to study genes under glucose repression in *Bacillus subtilis*. Nucleic Acids Res. 29:683–692.

40. Yoshida, K., Y. Yamamoto, K. Omae, M. Yamamoto, and Y. Fujita. 2002. Identification of two myo-inositol transporter genes of *Bacillus subtilis*. J. Bacteriol. 184:983–991.
