
ANTIMICROBIAL AGENTS AND CHEMOTHERAPY, Aug. 2005, p. 3129–3135
0066-4804/05/$08.00+0 doi:10.1128/AAC.49.8.3129–3135.2005
Copyright © 2005, American Society for Microbiology. All Rights Reserved.

**Role of the AcrAB-TolC Efflux Pump in Determining Susceptibility of Haemophilus influenzae to the Novel Peptide Deformylase Inhibitor LBM415**

Charles R. Dean,${}^{1*}$ Shubha Narayan,${}^{1}$ Denis M. Daigle,${}^{1}$ JoAnn L. Dzink-Fox,${}^{1}$ Xiaoling Puyang,${}^{1}$ Kathryn R. Bracken,${}^{1}$ Karl E. Dean,${}^{1}$ Beat Weidmann,${}^{1}$ Zhengyu Yuan,${}^{2}$ Rakesh Jain,${}^{2}$ and Neil S. Ryder${}^{1}$

Novartis Institutes for Biomedical Research, Inc., Cambridge, Massachusetts 02139,${}^{1}$ and Vicuron Pharmaceuticals, Fremont, California 94555${}^{2}$

Received 26 October 2004/Returned for modification 12 January 2005/Accepted 5 May 2005

*Haemophilus influenzae* isolates vary widely in their susceptibilities to the peptide deformylase inhibitor LBM415 (MIC range, 0.06 to 32 μg/ml); however, on average, they are less susceptible than gram-positive organisms, such as *Staphylococcus aureus* and *Streptococcus pneumoniae*. Insertional inactivation of the *H. influenzae acrB* or *tolC* gene in strain NB65044 (Rd strain KW20) increased susceptibility to LBM415, confirming a role for the AcrAB-TolC pump in determining resistance. Consistent with this, sequencing of a PCR fragment generated with primers flanking the *acrRA* region from an LBM415-hypersusceptible *H. influenzae* clinical isolate revealed a genetic deletion of *acrA*. Inactivation of *acrB* or *tolC* in several clinical isolates with atypically reduced susceptibility to LBM415 (MIC of 16 μg/ml or greater) significantly increased susceptibility, confirming that the pump is also a determinant of decreased susceptibility in these clinical isolates. Examination of *acrR*, encoding the putative repressor of pump gene expression, from several of these strains revealed mutations introducing frameshifts, stop codons, and amino acid changes relative to the published sequence, suggesting that loss of pump repression leads to decreased susceptibility. Supporting this, NB65044 *acrR* mutants selected by exposure to LBM415 at 8 μg/ml had susceptibilities to LBM415 and other pump substrates comparable to the least sensitive clinical isolates and showed increased expression of pump genes.

The continuing emergence and spread of cellular target-based antibiotic resistance mechanisms is a serious and increasing threat to the effective treatment of microbial infections. There is an obvious need for the development of new compounds, directed at novel cellular functions, which will be active against current resistant strains. In general, target-based antibiotic development strategies are very effective for identifying potent and specific inhibitors of target proteins; however, historically there have been significant hurdles to the effective use of these inhibitors as broad-spectrum antibiotics. This has much to do with intrinsic resistance imparted by bacterial membrane impermeability and efflux. These issues are especially problematic in the case of gram-negative bacteria, where the outer membrane and efflux pumps have been shown to act synergistically to minimize intracellular accumulation of a variety of structurally unrelated compounds (reviewed in references 17, 18, and 19).

There are five general groups of bacterial efflux pumps currently described: the major facilitator superfamily, small multidrug resistance family, the multidrug and toxic compound extrusion family, the ATP-binding cassette family, and the resistance-nodulation-division (RND) family (25). The RND family appears to have the broadest substrate range, and these pumps are therefore most generally relevant vis-a-vis drug resistance in gram-negative bacteria. Architecturally, they consist of an inner membrane proton-drug antiporter, an outer membrane channel, and a so-called membrane fusion protein that is thought to function in facilitating the interaction between the inner and outer membrane components in the periplasm. Substrate extrusion is driven by the proton motive force, and recent data indicate that many substrates may be pumped from the periplasm or the cytoplasmic membrane (9, 14, 33).

Along with intrinsic resistance conferred by efflux, regulatory mutations turning on or increasing efflux pump expression (presumably selected for by exposure to antimicrobial agents or biocides) can confer increased resistance to several or all of the substrates for a given pump (5). Efflux pump overexpressors have been isolated clinically (3, 15, 34); therefore, while cross-resistance to novel agents may not preexist in the form of target-based mutations selected by commonly used antibiotics, these exposures may select pump mutants with decreased susceptibility to novel antibiotics.

*Pseudomonas aeruginosa*, an important emerging opportunistic pathogen, represents one end of the spectrum of efflux-based resistance, having multiple RND family pumps of overlapping substrate range and a notably impermeable outer membrane which has been shown to significantly increase the efficiency of the pumps by limiting influx (24). Perhaps representing the other end is *Haemophilus influenzae*, an important respiratory pathogen (7, 11, 22, 29) that has only one known RND family (AcrAB-TolC homolog) pump (10, 27) and is characterized by a relatively permeable outer membrane. The permeability of the outer membrane has been implicated in limiting the efficiency of the efflux pump even for relatively

* Corresponding author. Mailing address: Infectious Diseases, Novartis Institute for Biomedical Research, Inc., 100 Technology Square, Cambridge, MA 02139. Phone: (617) 871-3150. Fax: (617) 871-7047. E-mail: charlesr.dean@novartis.com.

3130 DEAN ET AL.

ANTIMICROB. AGENTS CHEMOTHER.

LBM415

LBK611

FIG. 1. Structures of the novel peptide deformylase inhibitors LBM415 and LBK611.

large substrates such as erythromycin (27). Therefore, *H. influenzae* may represent an example of a gram-negative pathogen where efflux-based intrinsic and acquired resistance may be expected to pose less of a problem. Despite this and consistent with erythromycin being a substrate of the AcrAB-TolC pump of *H. influenzae* (27), moderate levels of intrinsic resistance to macrolides in *H. influenzae* clinical isolates has been associated with efflux (21). Recently, high-level resistance to macrolides related to mutations in the L22 ribosomal protein has also been shown to require a contribution from efflux (20).

A new class of antimicrobial compounds, typified by the reverse hydroxamates LBM415 and LBK611 (Fig. 1), are potent inhibitors of bacterial peptide deformylase (PDF), an antibacterial target that has been the focus of much recent interest (2, 6, 32). LBM415 shows significant promise, particularly towards gram-positive bacteria, including well-characterized resistant strains (10a). We have noticed that *H. influenzae* isolates exhibit, overall, somewhat reduced susceptibilities to LBM415 (MIC<sub>90</sub> of 4 μg/ml against a panel of 33 isolates) and related compounds, although they also have a very wide range of susceptibilities (LBM415 MIC range, 0.06 to 32 μg/ml). Therefore, we have investigated the contribution of AcrAB-TolC-mediated efflux in determining the susceptibility of several *H. influenzae* strains to LBM415 and the structurally related compound LBK611.

**MATERIALS AND METHODS**

Bacterial strains, plasmids, and growth media. Bacterial strains and plasmids used in this study are listed in Table 1. Luria-Bertani (LB) broth or LB agar (Difco) was used for routine growth of *Escherichia coli*. Chocolate agar plates (Remel) were used for routine growth of *H. influenzae*. Brain heart infusion broth (Remel) supplemented with 10 μg/ml of β-NAD (Fluka) and 10 μg/ml hemin, provided from a hemin-vitamin K solution (sBHI) (Remel), was used for liquid broth cultivation of *H. influenzae*. For induction of natural competence in *H. influenzae*, nutritional downshift was induced using M-IV medium as described previously (23). Kanamycin was added to growth media at 50 μg/ml (*E. coli*) or 5 μg/ml (*H. influenzae*) as required.

Antimicrobial susceptibility testing. Antibiotic MICs were determined by broth microdilution using twofold dilution in Haemophilus test medium (HTM; Remel) in accordance with the procedures established by the CLSI (formerly NCCLS) (16). Peptide deformylase inhibitors were synthesized at the Novartis Institutes for Biomedical Research, Cambridge, MA. All remaining antibiotics were obtained from Sigma (St. Louis, MO).

DNA manipulations. *H. influenzae* genomic DNA was isolated using the Puregene tissue kit (Genta Systems, Inc., Minneapolis, MN) according to the manufacturer's instructions. Oligonucleotides for PCR and sequencing were obtained from Genelink (Hawthorne, NY) and are listed in Table 2. PCRs were carried out using the Easystart mix-in-a-tube system (Molecular Bio-Products,

Inc., San Diego, CA) according to the supplied instructions, with prepared genomic DNA or cells from isolated colonies used as the template. Restriction endonucleases and modifying enzymes were used according to the instructions supplied with the enzymes. DNA fragments were purified or isolated following agarose gel electrophoresis, using the QIAquick PCR cleanup or gel extraction kit (QIAGEN, Inc., Valencia, CA) as specified in the instructions. Nucleotide sequencing was performed by Agencourt, Inc. (Beverly, MA).

In vitro insertion mutagenesis. For use in generating chromosomal insertion knockouts of *acrB* in *H. influenzae*, the *acrB* gene was inactivated in vitro as follows: primers AcrBHIF and AcrBHIR3 (Table 2) were used to generate a PCR fragment containing *acrB* from *H. influenzae* NB65001 genomic template DNA. This was directly cloned into pCR 2.1-Topo (Invitrogen, Carlsbad, CA) according to the instructions provided with the kit, then excised using EcoRI, and ligated into pEX18Tc. The resulting construct was linearized at the unique MfeI site within *acrB*, blunt ended with T4 DNA polymerase, and ligated to a blunt PCR fragment encompassing the kanamycin resistance marker from pACYC177 to give plasmid pCDBKm. This construct has the resistance determinant in the orientation opposite that of *acrB*. The presence of an *H. influenzae* DNA uptake sequence has been shown to facilitate much greater levels of natural transformation, which help to introduce DNA into various isolates which may not efficiently take up DNA by natural transformation (1). To introduce the uptake sequence into pCDBKm, a 177-bp DNA fragment containing the uptake sequence was amplified from NB65044 genomic DNA using previously described primers (1). The product was cloned into pCR2.1-Topo (Invitrogen, Carlsbad, CA), excised with KpnI, and cloned into the KpnI site within the multicloning site of pCDBKm to give pCDBKmUS.

To obtain an insertion in the open reading frame (ORF) encoding TolC (30), primers HI1462IF and HI1462IR (Table 2) were used to generate a PCR fragment encompassing most of *tolC* from *H. influenzae* NB65001 genomic DNA template. This was ligated directly into pCR 2.1-Topo (Invitrogen, Carlsbad, CA), recovered as an EcoRI fragment, and ligated into the EcoRI site of pBluescript SK. A kanamycin resistance gene, isolated from pBAD18Kan as a 1.8-kb HaeII fragment and blunt ended with T4 DNA polymerase, was then ligated into the unique MluI site within *tolC*, which had been rendered blunt, to generate pCD14Km. This construct has the kanamycin resistance determinant in the same orientation as HI1462 (Fig. 2).

Introduction of insertions onto the chromosome by gene replacement. For introduction of the *acrB::Km* insertion into the genome, *H. influenzae* strains were grown to early log phase (optical density at 600 nm of approximately 0.2) in sBHI, and natural competence was induced by nutritional downshift into M-IV medium by the method of Poje and Redfield (23). Competent cells were transformed with pCDBKmUS (linearized with XbaI) or pCDRKm (linearized with ScaI) as previously described (23) and plated on chocolate agar containing 5 μg/ml kanamycin. For introduction of the *tolC::Km* insertion into the genome of *H. influenzae* strain NB65044, competent cells were transformed with pCD14Km (linearized with ScaI) and selected as described above. For introduction of this insertion into strains NB65027 and NB65051, the recipient cells were made competent as described above, transformed with genomic DNA isolated from strain NB65044-CDS0001 (which has the insertion) and selected on chocolate agar plates containing 5 μg/ml kanamycin. All insertions were confirmed by PCR, and sequencing of fragments was generated using primers flanking the insertion sites.

TABLE 1. Bacterial strains and plasmids used in this study

| Strain or plasmid | Genotype or relevant characteristic(s) | Reference or source |
|-------------------|---------------------------------------|---------------------|
| H. influenzae     |                                       |                     |
| NB65044           | RdKW20                                | 10                  |
| NB65001           |                                       | ATCC 49247          |
| NB65062           | Clinical isolate, LBM 415 hypersusceptible | R.N. Jones          |
| NB65016           | Clinical isolate, decreased LBM 415 susceptibility | I.A. Chopra         |
| NB65027           | Clinical isolate, decreased LBM 415 susceptibility | B. Willinger       |
| NB65051           | Clinical isolate, decreased LBM 415 susceptibility | R.N. Jones          |
| NB65063           | Clinical isolate, decreased LBM 415 susceptibility | R.N. Jones          |
| NB65069           | Clinical isolate, decreased LBM 415 susceptibility | R.N. Jones          |
| NB65076           | Clinical isolate, decreased LBM 415 susceptibility | R.N. Jones          |
| NB 65062-CDS 0038 | NB 65062 derivative, complemented for acrA deficiency | This study         |
| NB 65062-CDS 0039 | NB 65062 derivative, complemented for acrA deficiency | This study         |
| NB 65044-CDS 0001 | NB65044acrB::Km                        | This study         |
| NB 65044-CDS 0020 | NB65044tolC::Km                        | This study         |
| NB 65044-CDS 0011 | NB65044acrR mutant selected on 8 μg/ml LBM 415 | This study         |
| NB 65044-CDS 0014 | NB65044acrR mutant selected on 8 μg/ml LBM 415 | This study         |
| NB 65016-CDS 0004 | NB65016acrB::Km                        | This study         |
| NB 65027-CDS 0021 | NB65027acrB::Km                        | This study         |
| NB 65027-CDS 0003 | NB65027tolC::Km                        | This study         |
| NB 65051-CDS 0002 | NB65051acrB::Km                        | This study         |
| NB 65051-CDS 0022 | NB65051tolC::Km                        | This study         |
| NB 65063-CDS 0005 | NB65063acrB::Km                        | This study         |
| NB 65069-CDS 0007 | NB65069acrB::Km                        | This study         |
| NB 65076-CDS 0006 | NB65076acrB::Km                        | This study         |
| E. coli           |                                       |                     |
| Top 10            |                                       | Invitrogen, Inc.    |
| Plasmids          |                                       |                     |
| pCR 2.1topo       | Cloning vector                         | Invitrogen, Inc.    |
| pEX18Tc           | Cloning vector                         | 13                  |
| pCDBKmUS          | pEX18Tc derivative containing the H. influenzae uptake sequence and acrB interrupted by a Tn 903-derived Km' cassette | This study         |
| pCD14Km           | pBluescript containing ORF tolC interrupted by a Tn 903-derived Km' cassette | This study         |
| pACYC 177         | Cloning vector, source of Tn903Km' cassette | NEB                 |
| pBAD18Kan         | Cloning/expression vector, source of Tn903Km' marker | 12                  |
| pBluescript SK    | Cloning vector                         | Stratagene, Inc.   |

*NEB, New England BioLabs.

PCR analysis of hypersusceptible strain NB 65062 and complementation of the acrA deletion. For analysis of the region encompassing the acrA gene in LBM 415-hypersusceptible strain NB 65062, primers AcrRHIF1 and AcrAHIR (Table 2) were used to generate an approximately 1.1-kb PCR fragment from H. influenzae NB 65062 genomic DNA. The fragment was gel purified, and the nucleotide sequence was determined. For complementation of the acrA defect of strain NB 65062, genomic DNA from NB 65044, which contains a functional AcrAB-TolC pump, was used to transform NB 65062 using the nutritional downshift method described above. Transformed cells were plated on chocolate agar containing either 4 μg/ml LBM415 or 2 μg/ml erythromycin, both substrates of the AcrAB-TolC efflux pump, and representative isolated colonies arising after 24h were picked for further examination.

RNA isolation. H. influenzae cultures were grown in triplicate in HTM (Remel) liquid medium with shaking at 37°C to late log phase (optical density at 600 nm of 0.8 to 1.0) and collected by centrifugation, and the cell pellets were frozen at -80°C. Total RNA was isolated from cell pellets using the Purescript RNA

TABLE 2. Oligonucleotides used for PCRs in this study

| Primer pair | Sequence                                      | Product size and target gene |
|-------------|-----------------------------------------------|------------------------------|
| AcrBHIF     | 5'-CCACTTTAATGTATGAGGAAATCCG-3'                | 3.1kb, acrB                   |
| AcrBHIR3    | 5'-CGAATTACGTAAGATAACCTAAGTGCG-3'              |                              |
| HI1462IF    | 5'-CACGCTTGCTTTGTTGATGTCTGGTGC-3'              | 1.3kb, tolC                   |
| HI1462IR    | 5'-TCCC GCCATTGAGCTATATACCGCA-3'               |                              |
| AcrRHIF2    | 5'-TAATGATGAAAAGTGCGGTTAATT-3'                 | 0.752kb, acrR                 |
| AcrRHIR     | 5'-TTTCTGAATCGCACGCCAAGAGCGT-3'                |                              |
| AcrRHIF1    | 5'-GTGCGGGTGCCACCGCAAGGACATA-3'                | 2kb, acrR-acrA                |
| AcrAHIR     | 5'-TGCA GGCCTATTGCACCCACAATG-3'                |                              |

acrRHIF1                                                                 acrAHIR
![Diagram](#)

FIG. 2. Genetic arrangement of the *acrAB* and *tolC* efflux pump genes in *H. influenzae* showing the positions and orientations of kanamycin resistance markers (Km) used for insertional inactivation. The positions of PCR primers AcrRHIF1 and AcrAHIR (arrows) are shown.

isolation kit (Gentra Systems, Minneapolis, MN) according to the supplied instructions. Approximately 100 μg of total RNA was then treated with 10 units RNase-free DNase I (RQ1; Promega, Madison, WI) for 1 h at 37°C to remove contaminating genomic DNA. Treated samples were then purified by processing over RNeasy minicolumns (QIAGEN, Inc., Valencia, CA) with an additional on-column DNase I treatment, according to the supplied protocol. RNA integrity was confirmed by formaldehyde agarose gel electrophoresis. Standard PCR using 20 ng RNA as template and primers specific for *acrA* was carried out (45 cycles) to confirm that there was no measurable contaminating genomic DNA.

GeneChip analysis. Efflux pump gene expression was examined using custom Affymetrix GeneChips designed to interrogate all predicted ORFs for *H. influenzae* strain RdKW20 (10). RNA was reverse transcribed, and 1 μg of cDNA was fragmented, labeled, and hybridized to GeneChips using the standard prokaryotic GeneChip protocol supplied by Affymetrix. GeneChip data were obtained by scanning with an Affymetrix autoloading scanner, and data were normalized and compared using Genespring (Silicon Genetics) analysis software.

Real-time RT-PCR. Primers and probes for real-time reverse transcription-PCR (RT-PCR) (Table 3) were designed using Primer Express v. 2.0 software (Applied Biosystems, Foster City, CA) and were synthesized by Applied Biosystems Assays by Design service. The levels of *acrB* transcripts were monitored by real-time RT-PCR analysis using Applied Biosystems' EZ RT-PCR Core Reagents kit based on a one-step RT-PCR for RNA quantitation on an Applied Biosystems PRISM model 7500 Sequence Detection system. Relative quantitation was done by the comparative cycle threshold method using the endogenous internal control *rpsL* (ribosomal protein S12) for sample normalization which had been shown to be invariant in this study (data not shown). Cycle threshold values were calculated using Applied Biosystems Sequence Detection software v.1.2.2. For each one-step RT-PCR run, 10 μl (10 ng) of total RNA was added to a reaction mixture prepared on ice containing 1× EZ RT-PCR TaqMan buffer, 3 mM manganese acetate, 300 μM dATP, dCTP, and dGTP, 600 μM dUTP, 0.9 μM of forward and reverse primers, 0.25 μM fluorogenic TaqMan-labeled probe, and 5 U of rTth DNA polymerase in a final volume of 50 μl. The thermocycling conditions were as follows: 60°C for 30 min, 95°C for 5 min, followed by 45 cycles of 95°C for 15 s and 60°C for 1 min. A preliminary experiment was performed to show that both the target and endogenous control transcripts (*acrB* and *rpsL*, respectively) were amplified with approximately equal efficiencies.

RESULTS AND DISCUSSION

Role of the AcrAB-TolC pump in determining susceptibility to LBM415. Insertional inactivation of *acrB* or *tolC* in the *H. influenzae* laboratory strain NB65044 (Fig. 2) increased susceptibility to LBM415 16- to 32-fold (NB65044-CDS0001 and NB65044-CDS0020 [Table 4]). The insertion also increased susceptibility to the structurally related PDF inhibitor LBK611 and the known pump substrate erythromycin (27), and more dramatically to clindamycin, while not significantly affecting susceptibility to the non-pump substrate tetracycline (27) (Table 4). Another known non-pump substrate, chloramphenicol, was also unaffected by pump deletion (data not shown). This confirms that the AcrAB-TolC pump of *H. influenzae* plays a significant role in reducing intrinsic susceptibility to LBM415 and LBK611.

Decreased susceptibility to antibiotics in clinical isolates of a number of bacteria is frequently associated with overexpression of efflux pumps. Identification of pump repressor mutations and/or pump gene overexpression in resistant clinical isolates is a strong indicator of efflux-based resistance; however, there is not always a clear association between pump status and resistance to specific antibiotics (28). Therefore, to directly address whether the AcrAB-TolC pump plays a significant role in decreasing the susceptibility of *H. influenzae* clinical isolates with the lowest susceptibilities to the PDF inhibitors, *acrB* was inactivated in strains NB65016, NB65027, NB65051, NB65063, NB65069, and NB65076, all of which exhibit decreased susceptibilities to LBM415, LBK611, and clindamycin. In all cases, pump loss substantially increased susceptibility.

TABLE 3. Primers used for real-time RT-PCR

| Primer       | Sequence                          | Target                              |
|--------------|-----------------------------------|-------------------------------------|
| HIAcrBST1F   | 5′-AAAATGATAATTCGCGGTGGCTACGG-3′  | *acrB*, unlabeled                   |
| HIAcrBST1R   | 5′-GAGGTTGGATTGATGGCTAACAC-3′     | *acrB*, unlabeled                   |
| HIRpsLST1F   | 5′-AACCTTCAAGAGCACAGTGTGTGTAT-3′  | *rpsL*, unlabeled                   |
| HIRpsLST1R   | 5′-GCACACCCGGTAAGTCTTTAA-3′       | *rpsL*, unlabeled                   |
| HIAcrBST1M2  | 5′-AACGGACTCTGCAACATT-3′          | *acrB*, TaqMan MGB fluorogenic (FAM labeled) $^a$ |
| HIRpsLST1M2  | 5′-ACGACCACCACGGATTA-3′           | *rpsL*, TaqMan MGB fluorogenic (FAM labeled) |

$a$ MGB, minor groove binder; FAM, 6-carboxyfluorescein.

TABLE 4. Impact of AcrAB-TolC efflux pump status on susceptibility to LBM 415 and LBK611 in *H. influenzae* clinical isolates and role of AcrR mutation in decreasing susceptibility

| Strain or isolate | Pump status | MIC (µg/ml) |
|-------------------|--------------|--------------|
|                   |              | LBM 415      | LBK 611    | ERY        | CLI        | TET        |
| Strains           |              |              |            |            |            |            |
| NB65044           |              | 4            | 4          | 4          | 8          | 0.5        |
| NB 65044-CDS 0001 | Derivative of NB 65044 lacking *acrB* | 0.25       | 0.25       | 0.25       | 0.5        | 0.5        |
| NB 65044-CDS 0020 | Derivative of NB 65044 lacking *tolC* | 0.125      | 0.125      | 0.25       | 0.5        | 1          |
| NB 65044-CDS 0011<sup>b</sup> |              | 32         | 32         | 8          | 32         | 1          |
| NB 65044-CDS 0014<sup>b</sup> |              | 32         | 32         | 8          | 32         | 1          |
| NB65062           | LBM 415-hypersusceptible clinical isolate, lacks *acrA* | ≤0.25      | ND         | ND<sup>c</sup> | ≤0.25      | 0.5        |
| NB 65062-CDS 0038<sup>d</sup> | NB 65062 complemented for *acrA* | 8          | ND         | ND<sup>c</sup> | 16         | 1          |
| NB 65062-CDS 0039<sup>e</sup> | NB 65062 complemented for *acrA* | 8          | ND         | ND<sup>c</sup> | 16         | 1          |
| Clinical isolates with decreased susceptibility |              |              |            |            |            |            |
| NB65016           |              | 16          | 16         | 16         | 16         | 1          |
| NB 65016-CDS 0004 | Derivative of NB 65016 lacking *acrB* | 0.25       | 0.25       | 0.25       | 0.5        | 1          |
| NB65027           |              | 32          | 32         | 16         | 32         | 1          |
| NB 65027-CDS 0021 | Derivative of NB 65027 lacking *acrB* | 0.5        | 0.5        | 0.25       | 2          | 0.5        |
| NB 65027-CDS 0003 | Derivative of NB 65016 lacking *tolC* | 0.5        | 0.25       | 0.125      | 2          | 1          |
| NB65051           |              | 32          | 32         | 16         | 32         | 1          |
| NB 65051-CDS 0002 | Derivative of NB 65051 lacking *acrB* | 0.25       | 0.25       | 0.25       | 0.5        | 1          |
| NB 65051-CDS 0022 | Derivative of NB 65051 lacking *tolC* | 0.25       | 0.25       | 0.125      | 0.5        | 1          |
| NB65063           |              | 32          | 64         | 8          | 16         | 1          |
| NB 65063-CDS 0005 | Derivative of NB 65063 lacking *acrB* | 1          | 1          | 0.25       | 0.5        | 1          |
| NB65069           |              | 16          | 32         | 16         | 16         | 1          |
| NB 65069-CDS 0007 | Derivative of NB 65069 lacking *acrB* | 0.25       | 0.5        | 0.125      | 0.5        | 1          |
| NB65076           |              | 16          | 32         | 16         | 32         | 2          |
| NB 65076-CDS 0006 | Derivative of NB 65076 lacking *acrB* | 0.25       | 0.25       | 0.125      | 0.5        | 1          |

<sup>a</sup> Abbreviations: ERY, erythromycin; CLI, clindamycin; TET, tetracycline; ND, not determined.

<sup>b</sup> Strains NB 65044-CDS 0011 and NB 65044-CDS 0014 are *acrR* mutants selected on chocolate agar plates containing 8 µg/ml LBM 415. NB 65044-CDS 0011 *acrR* has a C-to-T nucleotide change at position 253, introducing a stop codon. NB 65044-CDS 0014 *acrR* has a T-to-C nucleotide change at position 164, resulting in an L-to-P amino acid substitution.

<sup>c</sup> MICs determined by Etest (AB Biodisk, Solna, Sweden); NB 65062, 0.19 µg/ml; NB 65062-CDS 0038 and NB 65062-CDS 0039, 12 µg/ml.

<sup>d</sup> Transformant selected on ERY.

<sup>e</sup> Transformant selected on LBM 415.
lack the efflux pump. A recent report indicates that certain target mutations in L22 ribosomal proteins confer resistance to macrolides only in the presence of efflux (20). Further examination of macrolide resistance in certain LBM415-hypersusceptible strains will shed more light on the relationship between efflux and target-based macrolide resistance.

AcrR is a repressor of AcrAB expression, and mutations in *acrR* are related to susceptibility to LBM415. The demonstration that the AcrAB-TolC efflux pump is a major contributor to decreased susceptibility to LBM415 and other antimicrobials in *H. influenzae* clinical isolates suggests that increased pump expression may lead to decreases in susceptibility. Although the emerging picture of efflux pump regulation is becoming increasingly complex, there are many cases where pump overexpression is related to simple mutations in regulatory genes. For example, *P. aeruginosa nalB* strains overexpress MexAB-OprM due to mutations in the *mexR* gene encoding a repressor, located immediately upstream of the *mexAB-oprM* genes (26). In *H. influenzae*, a putative *acrR* gene (*HI0983*), located immediately upstream of *acrAB* (Fig. 2), encodes an AcrR/TetR family repressor which may be involved in controlling the expression of *acrAB*. Nucleotide sequencing of *acrR* from *H. influenzae* strains NB65016, NB65027, NB65051, and NB65063 revealed the presence of insertion/deletions or point mutations generating either frameshifts or stop codons, as follows: NB65016, 1-base (C) insertion after nucleotide 442 (frameshift); NB65027, 8-bp deletion and GTT insertion after nucleotide 366 (frameshift) and an additional 1-base insertion downstream; NB65051, 4-bp deletion after nucleotide 322 (frameshift); and NB65063, C252T substitution (stop). The *acrR* genes from NB65069 and NB65076 revealed point mutations leading to amino acid changes relative to the published sequence for the *acrR* gene. The preponderance of *acrR* mutations, combined with the clear role of the AcrAB-TolC efflux pump in decreasing susceptibility to LBM415, erythromycin, and clindamycin in these strains strongly suggests that the AcrAB-TolC efflux pump is being overexpressed due to loss of AcrR repressor function.

To further examine the relationship between AcrR and decreased susceptibility to LBM415, we tested whether exposure of *H. influenzae* NB65044 to LBM415 at 8 μg/ml would select mutants with altered *acrR* genes. Mutants of strain NB65044 were selected on chocolate agar containing 8 μg/ml of LBM415 (typical frequency of 10⁻⁷), and examination of the *acrR* genes from 10 isolated mutants revealed mutations in all 10 isolates (data not shown). Susceptibility testing of two of these mutants (NB65044-CDS0011 and NB65044-CDS0014, possessing an introduced stop codon and an amino acid change, respectively) (see footnote b of Table 4) revealed an eightfold decrease in susceptibility to LBM415 and LBK611 and a fourfold decrease in susceptibility to clindamycin with no change in susceptibility to tetracycline (Table 4). To relate the increase in resistance to increased pump expression, transcriptional profiling revealed that there was a modest increase in expression of *acrR* (approximately 3.44-fold), *acrA* (approximately 2.65-fold), and *acrB* (approximately 1.88-fold) in strain NB65044-CDS0011 compared to the parent strain NB65044. Real-time RT-PCR analysis for *acrB* (2.17-fold ± 0.03-fold upregulated) confirmed the increase measured by GeneChips. Similar results were obtained for a strain insertionaly inactivated for *acrR* using a kanamycin cassette (data not shown).

Taken together, these data show that decreased susceptibility to LBM415 can be acquired mutationally in the form of *acrR* mutations, presumably by affecting expression of AcrAB. It also strongly supports the notion that the mutations in *acrR* found in the less susceptible clinical strains examined here result in pump overexpression. It was previously reported that inactivation of *acrR* in *H. influenzae* did not alter susceptibility to a wide range of compounds, including erythromycin (30). In our investigation the change observed in erythromycin susceptibility upon *acrR* mutation was small, but susceptibility to clindamycin (and LBM415) was apparently much more responsive to AcrAB pump level, potentially reflecting differences between the compounds as regards recognition by the pump and/or influx across the outer membrane. The GeneChip data also indicate that *acrR* is autoregulated, which is typical for many efflux pump repressors.

Conclusions. In summary, we have shown that the AcrAB-TolC efflux pump of *H. influenzae* is responsible for decreasing susceptibility to the PDF inhibitors LBM415 and the structurally related compound LBK611. Susceptibility is further decreased upon mutational loss of AcrR. Significantly, *acrR* mutants apparently exist within the clinical population, suggesting that previous exposure to antimicrobials may have selected for decreased susceptibility to these novel PDF inhibitors. Furthermore, exposure of *H. influenzae* to LBM415 in vitro selected for *acrR* mutations and corresponding decreases in susceptibility to both LBM415 and other pump substrates. This highlights the potential impact of efflux with respect to novel antimicrobial compound development, even in bacteria, such as *H. influenzae*, with relatively less efficient efflux.

In light of the comparative inefficiency of the *H. influenzae* AcrAB-TolC pump and the relative permeability of the outer membrane, it may be possible to modify novel compounds to influx rapidly and/or escape recognition by the pump. Indeed, glycyl modifications of minocycline were previously shown to alter the relative contributions of pumps mediating efflux in *Pseudomonas aeruginosa* from MexAB-OprM to MexXY-OprM (8), suggesting that a specific pump can be evaded through chemical modification. Since many *H. influenzae* strains likely have only a single RND-type pump, this strategy may be more successful in this organism. Alternatively, efflux may be overcome through the development of specific pump inhibitors. At least one compound, MC 04,124 has been reported to inhibit macrolide efflux in *H. influenzae* (4, 31). The increasing prevalence and importance of gram-negative infections demand a more concerted effort directed at overcoming efflux in order to increase the odds of successful development of novel antimicrobials targeted at these bacteria.

ACKNOWLEDGMENTS

We are grateful to Stacey Esterow for pulsed-field gel analysis. We also thank Herbert Schweizer and Jon Beckwith for kindly providing pEX18Tc and pBAD18Kan, respectively, and Ian Chopra, Ronald Jones, and Birgit Willinger for providing *H. influenzae* clinical isolates. We are also grateful to Genedata, Inc., for assistance in developing the custom-designed GeneChip.

REFERENCES

1. Akerley, B. J., E. J. Rubin, V. L. Novick, K. Amaya, N. Judson, and J. J. Mekalanos. 2002. A genome-scale analysis for identification of genes re-
required for growth or survival of *Haemophilus influenzae*. Proc. Natl. Acad. Sci. USA 99:966–971.

2. Apfel, C. M., H. Locher, S. Evers, B. Takacs, C. Hubscherlen, W. Pirson, M. G. Page, and W. Keck. 2001. Peptide deformylase as an antibacterial drug target: target validation and resistance development. Antimicrob. Agents Chemother. 45:1058–1064.

3. Beinlich, K. L., R. Chuanchuen, and H. P. Schweizer. 2001. Contribution of multidrug efflux pumps to multiple antibiotic resistance in veterinary clinical isolates of *Pseudomonas aeruginosa*. FEMS Microbiol. Lett. 198:129–134.

4. Cho, D., D. Lofland, J. Blais, K. Tangen, D. Cotter, O. Lomovskaya, S. Chamberland, and M. N. Dudley. 2000. An efflux pump inhibitor (EPI), MC-04,124, enhances the activity of macrolides against gram-negative bacteria, abstr. 1497, p. 208. Program Abstr. 40th Intersci. Conf. Antimicrob. Agents Chemother. 2000. American Society for Microbiology, Washington, D.C.

5. Chuanchuen, R., K. Beinlich, T. T. Hoang, A. Becher, R. R. Karkhoff-Schweizer, and H. P. Schweizer. 2001. Cross-resistance between triclosan and antibiotics in *Pseudomonas aeruginosa* is mediated by multidrug efflux pumps: exposure of a susceptible mutant strain to triclosan selects *nfxB* mutants overexpressing MexCD-OprJ. Antimicrob. Agents Chemother. 45: 428–432.

6. Clements, J. M., R. P. Beckett, A. Brown, G. Catlin, M. Lobell, S. Palan, W. Thomas, M. Whittaker, S. Wood, S. Salama, P. J. Baker, H. F. Rodgers, V. Barynin, D. W. Rice, and M. G. Hunter. 2001. Antibiotic activity and characterization of BB-3497, a novel peptide deformylase inhibitor. Antimicrob. Agents Chemother. 45:563–570.

7. Dagan, R., C. E. Johnson, S. McLinn, N. Abughali, J. Feris, E. Leibovitz, D. J. Burch, and M. R. Jacobs. 2000. Bacteriologic and clinical efficacy of amoxicillin/clavulanate vs. azithromycin in acute otitis media. Pediatr. Infect. Dis. J. 19:95–104.

8. Dean, C. R., M. A. Visalli, S. J. Projan, P. E. Sum, and P. A. Bradford. 2003. Efflux-mediated resistance to tigecycline (GAR-936) in *Pseudomonas aeruginosa* PAO1. Antimicrob. Agents Chemother. 47:972–978.

9. Elkins, C. A., and H. Nikaido. 2002. Substrate specificity of the RND-type multidrug efflux pumps AcrB and AcrD of *Escherichia coli* is determined predominantly by two large periplasmic loops. J. Bacteriol. 184:6490–6498.

10. Fleischmann, R. D., M. D. Adams, O. White, R. A. Clayton, E. F. Kirkness, A. R. Kerlavage, C. J. Bult, J. F. Tomb, B. A. Dougherty, J. M. Merrick, et al. 1995. Whole-genome random sequencing and assembly of *Haemophilus influenzae* Rd. Science 269:496–512.

10a. Fritsche, T. R., H. S. Sader, R. Cleeland, and R. N. Jones. 2005. Comparative antimicrobial characterization of LBM415 (NVP PDF-713), a new peptide deformylase inhibitor of clinical importance. Antimicrob. Agents Chemother. 49:1468–1476.

11. Gotfried, M. H. 2001. Epidemiology of clinically diagnosed community-acquired pneumonia in the primary care setting: results from the 1999–2000 respiratory surveillance program. Am. J. Med. 111(Suppl. 9A):25S–29S.

12. Guzman, L. M., D. Belin, M. J. Carson, and J. Beckwith. 1995. Tight regulation, modulation, and high-level expression by vectors containing the arabinose *P BAD* promoter. J. Bacteriol. 177:4121–4130.

13. Hoang, T. T., R. R. Karkhoff-Schweizer, A. J. Kutchma, and H. P. Schweizer. 1998. A broad-host-range Flp-FRT recombination system for site-specific excision of chromosomally-located DNA sequences: application for isolation of unmarked *Pseudomonas aeruginosa* mutants. Gene 212:77–86.

14. Li, X. Z., D. Ma, D. M. Livermore, and H. Nikaido. 1994. Role of efflux pump(s) in intrinsic resistance of *Pseudomonas aeruginosa*: active efflux as a contributing factor to beta-lactam resistance. Antimicrob. Agents Chemother. 38:1742–1752.

15. Mazzariol, A., Y. Tokue, T. M. Kanegawa, G. Cornaglia, and H. Nikaido. 2000. High-level fluoroquinolone-resistant clinical isolates of *Escherichia coli* overproduce multidrug efflux protein AcrA. Antimicrob. Agents Chemother. 44:3441–3443.

16. National Committee for Clinical Laboratory Standards. 2003. Methods for dilution antimicrobial susceptibility tests for bacteria that grow aerobically. Approved standard M7–A5. National Committee for Clinical Laboratory Standards, Wayne, Pa.

17. Nikaido, H. 1996. Multidrug efflux pumps of gram-negative bacteria. J. Bacteriol. 178:5853–5859.

18. Nikaido, H. 1998. Multiple antibiotic resistance and efflux. Curr. Opin. Microbiol. 1:516–523.

19. Nikaido, H., and H. I. Zgurskaya. 2001. AcrAB and related multidrug efflux pumps of *Escherichia coli*. J. Mol. Microbiol. Biotechnol. 3:215–218.

20. Peric, M., B. Bozdogan, C. Galderisi, D. Krissinger, T. Rager, and P. C. Appelbaum. 2004. Inability of L22 ribosomal protein alteration to increase macrolide MICs in the absence of efflux mechanism in *Haemophilus influenzae* HMC-S. J. Antimicrob. Chemother. 54:393–400.

21. Peric, M., B. Bozdogan, M. R. Jacobs, and P. C. Appelbaum. 2003. Effects of an efflux mechanism and ribosomal mutations on macrolide susceptibility of *Haemophilus influenzae* clinical isolates. Antimicrob. Agents Chemother. 47:1017–1022.

22. Pfaller, M. A., A. F. Ehrhardt, and R. N. Jones. 2001. Frequency of pathogen occurrence and antimicrobial susceptibility among community-acquired respiratory tract infections in the respiratory surveillance program study: microbiology from the medical office practice environment. Am. J. Med. 111(Suppl. 9A):4S–12S.

23. Poje, G., and R. J. Redfield. 2003. Transformation of *Haemophilus influenzae*. Methods Mol. Med. 71:57–70.

24. Poole, K. 2001. Multidrug efflux pumps and antimicrobial resistance in *Pseudomonas aeruginosa* and related organisms. J. Mol. Microbiol. Biotechnol. 3:255–264.

25. Poole, K., and R. Srikumar. 2001. Multidrug efflux in *Pseudomonas aeruginosa*: components, mechanisms and clinical significance. Curr. Top. Med. Chem. 1:59–71.

26. Poole, K., K. Tetro, Q. Zhao, S. Neshat, D. E. Heinrichs, and N. Bianco. 1996. Expression of the multidrug resistance operon *mexA-mexB-oprM* in *Pseudomonas aeruginosa*: *mexR* encodes a regulator of operon expression. Antimicrob. Agents Chemother. 40:2021–2028.

27. Sanchez, L., W. Pan, M. Vinas, and H. Nikaido. 1997. The *acrAB* homolog of *Haemophilus influenzae* codes for a functional multidrug efflux pump. J. Bacteriol. 179:6855–6857.

28. Sobel, M. L., G. A. McKay, and K. Poole. 2003. Contribution of the MexXY multidrug transporter to aminoglycoside resistance in *Pseudomonas aeruginosa* clinical isolates. Antimicrob. Agents Chemother. 47:3202–3207.

29. Sokol, W. 2001. Epidemiology of sinusitis in the primary care setting: results from the 1999–2000 respiratory surveillance program. Am. J. Med. 111(Suppl. 9A):19S–24S.

30. Trepod, C. M., and J. E. Mott. 2004. Identification of the *Haemophilus influenzae tolC* gene by susceptibility profiles of insertionally inactivated efflux pump mutants. Antimicrob. Agents Chemother. 48:1416–1418.

31. Watkins, W. J., Y. Landavery, R. Leger, R. Litman, T. E. Renau, N. Williams, R. Yen, J. E. Zhang, S. Chamberland, D. Madsen, D. Griffith, V. Trembe, K. Huie, and M. N. Dudley. 2004. The relationship between physicochemical properties, in vitro activity and pharmacokinetic profiles of analogues of diamine-containing efflux pump inhibitors. Bioorg. Med. Chem. Lett. 13:4241–4244.

32. Wise, R., J. M. Andrews, and J. Ashby. 2002. *In vitro* activities of peptide deformylase inhibitors against gram-positive pathogens. Antimicrob. Agents Chemother. 46:1117–1118.

33. Yu, E. W., G. McDermott, H. I. Zgurskaya, H. Nikaido, and D. E. Koshland, Jr. 2003. Structural basis of multiple drug-binding capacity of the AcrB multidrug efflux pump. Science 300:976–980.

34. Ziha-Zarifi, I., C. Llanes, T. Kohler, J. C. Pechere, and P. Plesiat. 1999. In vivo emergence of multidrug-resistant mutants of *Pseudomonas aeruginosa* overexpressing the active efflux system MexA-MexB-OprM. Antimicrob. Agents Chemother. 43:287–291.
