
# Structures and reaction mechanisms of riboflavin synthases of eubacterial and archaeal origin

M. Fischer¹, W. Römisch, B. Illarionov, W. Eisenreich and A. Bacher  
Lehrstuhl für Organische Chemie und Biochemie, Technische Universität München, Lichtenbergstr. 4, D-85747 Garching, Germany

## Abstract

The biosynthesis of one riboflavin molecule requires one molecule of GTP and two molecules of ribulose 5-phosphate as substrates. GTP is hydrolytically opened, converted into 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione by a sequence of deamination, side chain reduction and dephosphorylation. Condensation with 3,4-dihydroxy-2-butanone 4-phosphate obtained from ribulose 5-phosphate leads to 6,7-dimethyl-8-ribityllumazine. The dismutation of 6,7-dimethyl-8-ribityllumazine catalysed by riboflavin synthase produces riboflavin and 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione. A pentacyclic adduct of two 6,7-dimethyl-8-ribityllumazines has been identified earlier as a catalytically competent reaction intermediate of the *Escherichia coli* enzyme. Acid quenching of reaction mixtures of riboflavin synthase of *Methanococcus jannaschii*, devoid of similarity to riboflavin synthases of eubacteria and eukaryotes, afforded a compound whose optical absorption and NMR spectra resemble that of the pentacyclic *E. coli* riboflavin synthase intermediate, whereas the CD spectra of the two compounds have similar envelopes but opposite signs. Each of the compounds could serve as a catalytically competent intermediate for the enzyme by which it was produced, but not vice versa. All available data indicate that the respective pentacyclic intermediates of the *M. jannaschii* and *E. coli* enzymes are diastereomers. Whereas the riboflavin synthase of *M. jannaschii* is devoid of similarity with those of eubacteria and eukaryotes, it has significant sequence similarity with 6,7-dimethyl-8-ribityllumazine synthase and the archaeal riboflavin synthase appear to have diverged early in the evolution of Archaea from a common ancestor.

Flavocoenzymes are essential cofactors for the catalysis of a wide variety of redox reactions (for a review, see [1]). Moreover, they are involved in numerous other physiological processes involving light sensing, bioluminescence, circadian time-keeping and DNA repair [2–8].

Vitamin B₂ (riboflavin) (9) is the universal precursor of flavocoenzymes (for a review, see [9]). The pathway of riboflavin biosynthesis summarized in Scheme 1 starts off from one molecule of GTP (1) and two molecules of ribulose 5-phosphate (6). GTP cyclohydrolase II (A) catalyses the release of formate from the imidazole ring and the release of pyrophosphate from the side chain of the nucleotide precursor producing 2,5-diamino-6-ribosylamino-4(3H)-pyrimidinone 5′-phosphate (2). Hydrolytic release of the position 2 amino group, reduction of the ribose side chain and dephosphorylation lead to 5-amino-6-ribitylamino-2,4-(1H,3H)-pyrimidinedione (5). The sequential order of deamination and side chain reduction (B, C) varies in different taxonomic groups. 5-Amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione phosphate (5) needs to be dephosphorylated before the enzymatic formation of 6,7-dimethyl-8-ribityllumazine (8). The pyrimidine derivative 5 produces 6,7-dimethyl-8-ribityllumazine (8) by condensation with 3,4-dihydroxy-2-butanone 4-phosphate (7) which is obtained from ribulose 5-phosphate (6) by a skeletal rearrangement. The final step of the biosynthetic pathway involves an unusual dismutation of the pteridine derivative 8 affording riboflavin (9) and the pyrimidine 5.

Phosphorylation of riboflavin by flavokinase (H) is invariably required in prototrophic as well as in auxotrophic species in order to obtain riboflavin phosphate (FMN, 10) and FAD (11) (for review, see [10]).

In the final reaction step of the riboflavin pathway, 6,7-dimethyl-8-ribityllumazine (8) undergoes an unusual dismutation in the formation of riboflavin (9) and 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione (1) (Scheme 1). Riboflavin synthase (G) catalysing that reaction was the first enzyme of the riboflavin pathway to be discovered more than five decades ago [11–14]. The enzyme was first purified to near-homogeneity (~4000-fold) from baker’s yeast [15,16].

The reaction involves the formation of 1 mol each of riboflavin and 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione from 2 mols of 6,7-dimethyl-8-ribityllumazine. Formally, the reaction can be described as a dismutation involving the exchange of a C₄ fragment between the two substrate molecules. However, both the size of the exchanged fragment and the mechanistic complexity set the reaction clearly apart from the classical dismutation reactions.

Interestingly, the formation of riboflavin and 5 from 8 can proceed in the absence of any catalyst under quite mild

Key words: archaea, eubacteria, evolution, reaction mechanism, riboflavin biosynthesis, riboflavin synthase.
¹To whom correspondence should be addressed (email markus.fischer@ch.tum.de).

©2005 Biochemical Society

Scheme 1 | The biosynthesis of riboflavin and flavocoenzymes

A, GTP cyclohydrolase II; B, 2,5-diamino-6-ribosylamino-4(3H)-pyrimidinone 5-phosphate deaminase; C, 5-amino-6-ribosylamino-2,4(1H,3H)-pyrimidinedione-5'-phosphate reductase; D, 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione phosphatase; E, 3,4-dihydroxy-2-butanone 4-phosphate synthase; F, 6,7-dimethyl-8-ribityllumazine synthase; G, riboflavin synthase; H, riboflavin kinase; I, FAD synthetase.

1, GTP; 2, 2,5-diamino-6-ribosylamino-4(3H)-pyrimidinone 5'-phosphate; 3, 5-amino-6-ribosylamino-2,4(1H,3H)-pyrimidinedione 5'-phosphate; 4, 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione; 5, 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione; 6, ribulose 5-phosphate; 7, 3,4-dihydroxy-2-butanone 4-phosphate; 8, 6,7-dimethyl-8-ribityllumazine; 9, riboflavin; 10, FMN; 11, FAD.


The chemical structures and reaction pathways are included in the image but cannot be directly transcribed into Markdown format due to their complexity. However, the textual information has been accurately OCR'd as requested.
Scheme 2 | Proposed mechanism and stereochemistry of 6,7-dimethyl-8-ribityllumazine conversion into riboflavin catalysed by eubacterial (A, red) respectively archaeal (B, green) riboflavin synthase. R, ribityl.

The substrate, 8, has a number of unusual properties. The position 7 methyl group is highly acidic with a pK of approx. 8.5 [22]. Deprotonation affords an anion mixture where the position 7 exomethylene form is present at a level of approx. 0.5%. Predominantly, however, the mixture contains several tricyclic molecular species arising by the nucleophilic attack of one of the side chain hydroxy groups on the ring carbon 7 [23–26]. The acidic protons of the position 7 methyl group are easily exchanged with solvent water [15,23,27], and this exchange is accelerated by riboflavin synthase [15].

On the basis of these findings, the involvement of anionic molecular species in the formation (both enzymatic and non-enzymatic) of riboflavin from 8 was proposed (Scheme 2, 8a). Suggested reaction mechanisms are all based on the hypothesis that a carbanion obtained by deprotonation of the position 7 methyl group of the substrate nucleophilically attacks the second substrate molecule in the formation of a dimer (Scheme 2). In support of that, Plaut and co-workers could show that substrate analogues carrying a position 7 oxo-group are potent inhibitors of riboflavin synthase [28]. Ligand binding studies also showed that each subunit of riboflavin synthase can bind two substrate molecules [16,29,30].

Recent work with an S41A (Ser⁴¹ → Ala) mutant of riboflavin synthase of Escherichia coli produced a pentacyclic dimer of 6,7-dimethyl-8-ribityllumazine (Scheme 2, compound Q) after acid quench of enzyme/substrate mixtures under single turnover conditions [20]. The compound, designated Q, fulfils the requirements for a kinetically competent reaction intermediate. Compound Q was shown to be a pentacyclic dimer of 6,7-dimethyl-8-ribityllumazine by NMR analysis. Notably, the dimerization is conducive to two centres of chirality (in addition to the chiral centres in the respective ribityl side chains). The dimer can be cleaved by riboflavin synthase in two different ways leading to two molecules of 8 (backward reaction) or one molecule each of riboflavin (9) and 5 (forward reaction). Riboflavin is formed more rapidly from compound Q than from 8 [20]. Stopped-flow as well as quenched-flow experiments documented the transient formation of the pentacyclic reaction intermediate. In these experiments, no other transient species were sufficiently populated to allow detection [31,32].

Riboflavin synthases of eubacteria and of Arabidopsis thaliana are homotrimers of approx. 75 kDa. The riboflavin synthases of plants, fungi and eubacteria show internal sequence similarity, which suggested the presence of two domains with similar folding topology [33–37]. It was proposed that each domain could bind one substrate molecule, and that the active site could be formed at the interface of two domains where two substrate molecules could be brought together in an approximately c₂ symmetric arrangement

that would lead naturally to the antiparallel regiochemistry observed in the product.

The three-dimensional structures of riboflavin synthases from *E. coli* and *Schizosaccharomyces pombe* have been determined by X-ray crystallography [35,38]. The two domains of each respective subunit are indeed related by pseudo-c₂ symmetry, but the interface region of that pseudosymmetric ensemble does not harbour the active site. Surprisingly, however, the N-terminal domain of one subunit of the *E. coli* enzyme is in close contact with the C-terminal domain of an adjacent subunit. The two respective domains are also related by pseudo-c₂ symmetry. Notably, only two out of the six folding domains of the molecule are involved in that type of interaction, and the trimer is devoid of trigonal symmetry.

Comparison between the riboflavin synthases of *E. coli* and *S. pombe* show unequivocally that each folding domain can bind one substrate molecule. At the interface between the respective N- and C-terminal domains of two adjacent subunits in the homotrimer, the two respective substrate-binding sites are in close proximity. Moreover, the two bound substrate molecules in that configuration are related by pseudo-c₂ symmetry (dictated by the pseudo-c₂ symmetry of the protein environment) that is required by the regiospecificity of the enzyme-catalysed reaction. The pentacyclic reaction intermediate, compound Q, fits perfectly into the cavity between these two subunits. The topology of the protein environment dictates a *cis*-configuration for the dimerization. The model suggests further that, ultimately, the substrate molecule at the C-terminal domain is the one that donates a C₄ unit to the substrate molecule bound at the N-terminal domain resulting in the formation of the product, riboflavin [35,38]. Thus the donor and acceptor sites can now be unequivocally assigned to the C-terminal and N-terminal folding domains respectively.

An unusual riboflavin synthase lacking the apparent sequence similarity and the domain structure of the enzymes from eubacteria, fungi and plants has been cloned and characterized from *Methanobacterium thermoautotrophicum* and *Methanococcus jannaschii* [21,39]. In close parallel with the studies reported above, acid quench of reaction mixtures under single turnover conditions afforded a compound, subsequently designated Q′ (Scheme 2), whose optical spectrum closely resembled that of the pentacyclic intermediate of *E. coli* riboflavin synthase, compound Q (Figure 1A).

The isolated compound Q′ was converted to riboflavin at a rate of 21 nmol·mg⁻¹·min⁻¹ by riboflavin synthase of *M. jannaschii* under steady-state conditions, whereas 6,7-dimethyl-8-ribityllumazine was converted into riboflavin at a rate of 11 nmol·mg⁻¹·min⁻¹. Hence, Q′ fulfils the criteria for a kinetically competent intermediate. However, the compound was unable to serve as substrate for the riboflavin synthase of *E. coli* (specific activity, <10 pmol·mg⁻¹·min⁻¹). Moreover, compound Q, i.e. the reaction intermediate isolated in the studies with *E. coli* riboflavin synthase, failed to serve as a substrate for *M. jannaschii* riboflavin synthase (specific activity <10 pmol·mg⁻¹·min⁻¹).

---

**Figure 1 | Absorbance and CD spectra of compound Q (dotted line) and compound Q′ (solid line)**

The measurements were performed at pH 10.2 (A), pH 11.0 (B) respectively.

![Graph A](https://i.imgur.com/placeholder.png)
![Graph B](https://i.imgur.com/placeholder.png)

---

CD spectra of compounds Q and Q′ revealed similar envelopes but opposite signs (Figure 1B). Based on the comparison of the CD and ¹³C NMR data of compounds Q and Q′, it can be concluded that both molecular species share (i) the same constitution, (ii) the same stereochemistry of their ribityl side chains (which is of course not affected in the course of the riboflavin synthase catalysed reaction) and (iii) the *cis*-linkage of the two pteridine systems, but that the pentacyclic ring systems of the two compounds are related by mirror symmetry. Both compounds have eight chiral centres (the ring carbon atoms 6* and 7* and the chiral centres of each respective ribityl side chain). Since the chiral centres of both ribityl side chains have the same configuration, compounds Q and Q′ differ by their configurations at the ring carbon atoms 6* and 7* and must be designated as diastereomers (Scheme 2).

The riboflavin synthases of archaea have no detectable sequence similarity with those of eubacteria, yeasts and plants. However, their sequences closely resemble those of 6,7-dimethyl-8-ribityllumazine synthases. Completely sequenced archaeal genomes typically comprise sets of two

similar genes, which specify paralogous riboflavin synthases and 6,7-dimethyl-8-ribityllumazine synthases. On the other hand, in eubacteria, plants and fungi, single genes of the 6,7-dimethyl-8-ribityllumazine synthase type are accompanied by genes specifying riboflavin synthases of the eubacterial/eukaryotic type characterized by internal sequence similarity and no similarity to 6,7-dimethyl-8-ribityllumazine synthase. Sequence arguments also showed that the divergence between the two domains of eubacteria/eukaryotic riboflavin synthase occurred very early in evolution; similarly, the divergence between 6,7-dimethyl-8-ribityllumazine synthase and archaeal bacterial riboflavin synthase occurred early in evolution [21].

Hydrodynamic studies show that archaeal riboflavin synthases form homopentamers, in close agreement with the quaternary structure properties of 6,7-dimethyl-8-ribityllumazine synthases [21]. This reflects the fact that archaeal riboflavin synthases are members of the lumazine synthase superfamily. Lumazine synthases form c5-symmetric homopentamers or icosahedral capsids, which are best described as dodecamers of pentamers [40–47].

The multiple active sites of lumazine synthases are located at each respective interface between two adjacent subunits in the pentameric modules, and it is likely that the active sites of archaeal bacterial riboflavin synthases have the same approximate location. The orientation of the product, 6,7-dimethyl-8-ribityllumazine, in the enzyme-product complex of 6,7-dimethyl-8-ribityllumazine synthase is well documented by X-ray structures. The heterocyclic ring system of the product has been shown to form a π complex with a highly conserved aromatic amino acid residue (phenylalanine or tryptophan) [40,41]. Based on sequence arguments, it is highly plausible that one of the substrate molecules in the paralogous riboflavin synthase could occupy the same place as the product lumazine in 6,7-dimethyl-8-ribityllumazine synthase. Moreover, there is enough space in the active site cavity to accommodate a second molecule 6,7-dimethyl-8-ribityllumazine in order to enable the dimerization and subsequent dimer fragmentation to produce riboflavin. However, whereas the pseudo-c2 symmetry of the respective N- and C-terminal domains that together form the active site of the E. coli enzyme is by necessity conducive to the required quasi-c2 topology of the substrate molecules, the archaeal enzyme is devoid of any symmetry properties that would inherently enforce the required symmetry properties of the substrate topology.

### References

1. Müller, F. (ed.) (1992) Chemistry and Biochemistry of Flavoenzymes, vol. 3, CRC Press, Boca Raton, FL
2. Meighen, E.A. (1991) Microbiol. Rev. **55**, 123–142
3. Meighen, E.A. (1993) FASEB J. **7**, 1016–1022
4. O’Kane, D.J. and Prasher, D.C. (1992) Mol. Microbiol. **6**, 443–449
5. Sancar, A. (2000) Annu. Rev. Biochem. **69**, 31–67
6. Sancar, A. (2003) Chem. Rev. **103**, 2203–2237
7. Kavakli, I.H. and Sancar, A. (2002) Mol. Interv. **2**, 484–492
8. Cheng, P., He, Q., Yang, Y., Wang, L. and Liu, Y. (2003) Proc. Natl. Acad. Sci. U.S.A. **100**, 5938–5943

9. Fischer, M. and Bacher, A. (2005) Nat. Prod. Rep. **22**, 324–350
10. Bacher, A. (1991) in Chemistry and Biochemistry of Flavoenzymes (Müller, F., ed.), vol. 1, pp. 349–370, CRC Press, Boca Raton, FL
11. Masuda, T. (1956) Pharm. Bull. **4**, 71–72
12. Masuda, T. (1956) Pharm. Bull. **4**, 375–381
13. Kuwada, S., Masuda, T., Kishi, T. and Asai, M. (1958) J. Vitaminol. (Kyoto) **4**, 217–225
14. Maley, G.F. and Plaut, G.W. (1959) J. Biol. Chem. **243**, 641–647
15. Plaut, G.W., Beach, R.L. and Aogaichi, T. (1970) Biochemistry **9**, 771–785
16. Harvey, R.A. and Plaut, G.W. (1966) J. Biol. Chem. **241**, 2120–2136
17. Beach, R. and Plaut, G.W. (1969) Tetrahedron Lett. **40**, 3489–3492
18. Rowan, T. and Wood, H.C. (1968) J. Chem. Soc. C **4**, 452–458
19. Rowan, T. and Wood, H.C.S. (1963) Proc. Chem. Soc. London 21–22
20. Illarionov, B., Eisenreich, W. and Bacher, A. (2001) Proc. Natl. Acad. Sci. U.S.A. **98**, 7224–7229
21. Fischer, M., Schott, A.K., Römisch, W., Ramsperger, A., Augustin, M., Fidler, A., Bacher, A., Richter, G., Huber, R. and Eisenreich, W. (2004) J. Mol. Biol. **343**, 267–278
22. Pfleiderer, W. and Hutzenlaub, W. (1973) Chemische Berichte **106**, 3149–3174
23. Beach, R. and Plaut, G.W. (1970) Biochemistry **9**, 760–770
24. Beach, R.L. and Plaut, G.W.E. (1971) J. Org. Chem. **36**, 3937–3943
25. Bown, D.H., Keller, P.J., Floss, H.G., Sedlmaier, H. and Bacher, A. (1986) J. Org. Chem. **51**, 2461–2467
26. Pfleiderer, W., Mengel, R. and Hemmerich, P. (1971) Chemische Berichte **104**, 2273–2292
27. Paterson, T. and Wood, H.C.S. (1969) Chem. Commun. (London) **6**, 290–291
28. Winestock, C.H., Aogaichi, W.T. and Plaut, G.W. (1963) J. Biol. Chem. **238**, 2866–2874
29. Otto, M.K. and Bacher, A. (1981) Eur. J. Biochem. **115**, 511–517
30. Plaut, G.W.E. (1971) in Comprehensive Biochemistry (Florkin, M. and Stotz, E.H., eds.), vol. 21, pp. 11–45, Elsevier, Amsterdam
31. Illarionov, B., Haase, I., Bacher, A., Fischer, M. and Schramek, N. (2003) J. Biol. Chem. **278**, 47700–47706
32. Illarionov, B., Haase, I., Fischer, M., Bacher, A. and Schramek, N. (2005) Biol. Chem. **386**, 127–136
33. Schott, K., Kellermann, J., Lottspeich, F. and Bacher, A. (1990) J. Biol. Chem. **265**, 4204–4209
34. Eberhardt, S., Richter, G., Gimbel, W., Werner, T. and Bacher, A. (1996) Eur. J. Biochem. **242**, 712–719
35. Gerhardt, S., Schott, A.K., Karies, N., Cushman, M., Illarionov, B., Eisenreich, W., Bacher, A., Huber, R., Steinbacher, S. and Fischer, M. (2002) Structure (Camb.) **10**, 1371–1381
36. Fischer, M., Schott, A.K., Kemter, K., Feicht, R., Richter, G., Illarionov, B., Eisenreich, W., Gerhardt, S., Cushman, M., Steinbacher, S. et al. (2003) BMC Biochem. **4**, 18
37. Fischer, M., Haase, I., Feicht, R., Schramek, N., Köhler, P., Schieberle, P. and Bacher, A. (2005) Biol. Chem. **386**, 417–428
38. Liao, D.I., Wawrzak, Z., Calabrese, J.C., Viitanen, P.V. and Jordan, D.B. (2001) Structure (Camb.) **9**, 399–408
39. Eberhardt, S., Korn, S., Lottspeich, F. and Bacher, A. (1997) J. Bacteriol. **179**, 2938–2943
40. Gerhardt, S., Haase, I., Steinbacher, S., Kaiser, J.T., Cushman, M., Bacher, A., Huber, R. and Fischer, M. (2002) J. Mol. Biol. **318**, 1317–1329
41. Koch, M., Breithaupt, C., Gerhardt, S., Haase, I., Weber, S., Cushman, M., Huber, R., Bacher, A. and Fischer, M. (2004) Eur. J. Biochem. **271**, 3208–3214
42. Braden, B.C., Velikovsky, C.A., Cauerhff, A.A., Polikarpov, I. and Goldbaum, F.A. (2000) J. Mol. Biol. **297**, 1031–1036
43. Meining, W., Mörtl, S., Fischer, M., Cushman, M., Bacher, A. and Ladenstein, R. (2000) J. Mol. Biol. **299**, 181–197
44. Persson, K., Schneider, G., Douglas, B.J., Viitanen, P.V. and Sandalova, T. (1999) Protein Sci. **8**, 2355–2365
45. Haase, I., Mörtl, S., Köhler, P., Bacher, A. and Fischer, M. (2003) Eur. J. Biochem. **270**, 1025–1032
46. Mörtl, S., Fischer, M., Richter, G., Tack, J., Weinkauf, S. and Bacher, A. (1996) J. Biol. Chem. **271**, 33201–33207
47. Fischer, M., Haase, I., Kis, K., Meining, W., Ladenstein, R., Cushman, M., Schramek, N., Huber, R. and Bacher, A. (2003) J. Mol. Biol. **326**, 783–793

Received 12 April 2005
