
Available online at www.sciencedirect.com

**SCIENCE DIRECT**

Virology 340 (2005) 307–317

VIROLOGY

www.elsevier.com/locate/yviro

---

### Changes in bacteriophage T7 virion structure at the initiation of infection

Priscilla Kemp${}^{a}$, L. René Garcia${}^{a,1}$, Ian J. Molineux${}^{a,b,*}$

${}^{a}$Molecular Genetics and Microbiology, University of Texas, Austin, TX 78712, USA  
${}^{b}$Institute for Cell and Molecular Biology, University of Texas, Austin, TX 78712, USA  

Received 3 May 2005; returned to author for revision 23 June 2005; accepted 28 June 2005  
Available online 28 July 2005

---

#### Abstract

Five proteins are ejected from the bacteriophage T7 virion at the initiation of infection. The three known proteins of the internal core enter the infected cell; all three must both disaggregate from their structure in the mature virion and also almost completely unfold in order to leave the head and pass through the head–tail connector. Two small proteins, the products of genes 6.7 and 7.3, also are ejected from the infecting virion. Gp6.7 and gp7.3 were not previously described as structural virion components, leading to a re-appraisal of the stoichiometry of virion proteins. Gp6.7 is found in tail-less particles and is defined as a head protein, whereas gp7.3 is localized in the tail. Gene 6.7 may be important in morphogenesis; mutants defective in this late gene yield a reduced burst of progeny. Gene 7.3 is essential for virion assembly but, although normally present, its product gp7.3 is not required in a mature particle. Particles assembled in the absence of gp7.3 contain tail fibers but fail to adsorb to cells.

© 2005 Elsevier Inc. All rights reserved.

#### Keywords: Bacteriophage T7; Mutant virion composition; Ejected proteins

---

#### Introduction

The virions of bacteriophages T3 and T7 are very similar and have a fairly simple structure (Fig. 1). The icosahedral head has a diameter of 60 to 61 nm with a shell that is 2 nm thick (Stroud et al., 1981; Rontó et al., 1983). The outer shell is composed of two forms of the gene 10 protein, which are made via a programmed translational frameshift near the 3′ end of the shorter gene 10A (Dunn and Studier, 1983; Condrey et al., 1989, Condron et al., 1991a, 1991b). Inserted at one vertex is the head–tail connector, composed of 12 gp8 molecules (Carazo et al., 1986; Cerritelli and Studier, 1996; Kocsis et al., 1995; Valpuesta et al., 1992). The connector has a 12-lobed wide domain inserted into the head cavity and a narrower domain that interacts with the tail (Valpuesta et al., 1992, 2000). A channel, which is closed in mature virions, runs through the center of the connector (Donate et al., 1988). Inside the head, and attached to the head–tail connector in the coaxial orientation, is a 26 nm × 21 nm cylindrical structure that is usually referred to as the internal core (Serwer, 1976; Serwer et al., 1997; Steven and Trus, 1986). The core has recently been shown to exhibit 8-fold symmetry (Cerritelli et al., 2003), it consists of stacked rings and contains three distinct proteins, the products of genes 14, 15, and 16. The 40-kb genome is spooled around the internal core in six coaxial shells (Cerritelli et al., 1997). Unlike most other tailed phages, the T7 tail is not assembled as a separate structure but forms directly on the DNA-filled head (Studier, 1972; Serwer, 1976; Roeder and Sadowski, 1977; Matsuo-Kato et al., 1981). The stubby tail is 23 nm long, tapering from a diameter of 21 nm at the connector to 9 nm at its distal end, and is known to consist of two major proteins gp11 and gp12 (Studier, 1972; Steven and Trus, 1986). Attached near the head proximal end of the tail are six symmetrically positioned tail fibers. Each fiber is composed of a trimer of gp17 that forms a kinked structure (Kato et al., 1985, 1986; Steven et al., 1988). The N-terminal 150 residues of gp17

* Corresponding author. Molecular Genetics and Microbiology, University of Texas, Austin, TX 78712, USA. Fax: +1 512 471 7088.
    * E-mail address: molineux@mail.utexas.edu (I.J. Molineux).
* Present address: Department of Biology, Texas A&M University, College Station, TX 77843, USA.

0042-6822/$ - see front matter © 2005 Elsevier Inc. All rights reserved.  
doi:10.1016/j.virol.2005.06.039

P. Kemp et al. / Virology 340 (2005) 307–317

![Fig. 1. Schematic diagram of the T7 virion.](image.png)

link the fiber to the tail, the next 117 residues fold into an α-helix, forming a 16.4-nm rod that is flexibly joined to the 15.5-nm distal half fiber. The latter consists of a linear array of four globules that is thought to bind directly to the bacterial cell.

Bacteriophage T7 initiates an infection of *Escherichia coli* by the interaction of its tail fibers with the lipopolysaccharide (LPS) on the cell surface. Interaction of all six tail fibers with LPS would orient the phage tail perpendicular to the cell surface, conferring efficiency to subsequent stages of infection. However, the stubby T7 tail is too short to span the *E. coli* cell envelope and a channel needs to be made to allow the phage genome to travel from the virion into the cytoplasm. It was proposed that virion proteins are ejected into the cell, functionally endowing T7 with an extensible tail, in contrast to the well-known contractile tail of T4 and other *Myoviridae* (Molineux, 2001). In testing this suggestion, we show here that five T7 proteins are ejected from the virion into the cell at the initiation of infection. Two of these proteins were not previously known to be part of the virion, an observation that prompted a re-evaluation of the protein composition of the T7 particle.

### Genes 6.7, 7, and 7.3

Gene 6.7 was first revealed by the genome sequence; gene 6.7 mutants have not previously been described, but a deletion mutant that lacks both gene 6.5 and gene 6.7 had previously been generously provided by F. W. Studier. Gene 6.5 is non-essential (A.H. Rosenberg and Studier, personal communication) and was not found in this study as a virion protein. T7 gene 7 was originally defined by a number of strains containing mutations that mapped between genes 6 and 8 (Studier, 1969). The genome sequence eventually revealed that the mutants actually represented two genes, genes 7 and 7.3 (Dunn and Studier, 1983). As shown below, Δ(6.7–7.7) mutants plate normally on cells that do not supply gene 7 function, indicating that gene 7 is not required for growth on the *E. coli* B or K-12 strains used here. The gene 7 protein was not reproducibly observed in purified T7 particles; when present gp7 was found only at very low levels and was not further examined. The closely related phage T3 lacks gene 7 (Beck et al., 1989; Pajunen et al., 2002) and it seems unlikely that T7 gp7 has a critical role in either virion formation or the infection process.

The two small virion proteins that are normally degraded after infection by a cellular protease were isolated from purified virions by SDS-PAGE; they were then partially digested with trypsin and peptides were isolated by HPLC. N-terminal sequencing and mass spectrometry analyses unequivocally showed them to be the products of genes 6.7 and 7.3. We found no evidence in purified T7 particles for the presence of gp13, which has long been described as an internal virion protein (Studier, 1972). Neither gp6.7 nor gp7.3 have previously been identified as T7 virion proteins and it was therefore necessary to determine their stoichiometry and locations within the particle, and to try to elucidate their functions.

P. Kemp et al. / Virology 340 (2005) 307–317

A
B
C

Phage
Cells+Phage
Eluate

Phage
Cells+Phage
Eluate

Phage
Cells+Phage
Eluate

T7⁺
Δ(6.5-6.7)
13am89

Fig. 2. Elution and SDS-PAGE separation of \[^{35}S\]-labeled T7 virion proteins from infected IJ1133 cells. (A) Wild-type T7 virions. (B) Δ(6.5–6.7) defective virions. (C) 13am89 defective virions.

Virions prepared from gene 6.7 mutants adsorb moderately well to IJ1133 but ejection of the internal core proteins gp14, gp15, and gp16 into the cell seems less efficient than after infection by a wild-type particle (Fig. 2B). Phage particles assembled in the absence of gene 13 activity also adsorb to cells, albeit less effectively than 6.7 mutant particles. However, the majority of the internal core proteins of a \(13^-\) virion do not stably associate with cells and can be eluted (Fig. 2C).

In *E. coli* BL21, gene 6.7 mutants form pinprick plaques at low frequency, but in the K-12 strain IJ1133, the same minute plaques are found at close to normal efficiency (Table 1). In part, the minute plaque phenotype is due to a significantly reduced burst size. In both BL21 and IJ1133 infected by Δ(6.5–6.7), only ~20 particles per cell are produced that make plaques on the complementing host IJ1133(pAR775). This is about 10% of the burst given by wild-type phage during infections conducted in parallel experiments. In contrast, gene 7.3 is absolutely required on both B and K-12 strains. Gene 7.3 deletion mutants produce plaques only at mutational frequencies. Mutant phage particles do not form plaques even when the host cell contains a complementing plasmid; thus, a T7 virion must contain or be assembled in the presence of gp7.3 in order to initiate an infection. Extragenic suppressors of the growth defect can be isolated that bypass the necessity for phages to contain gene 7.3.

The requirement for gene 6.7 appears less stringent when the host cell contains a gene 7.3 plasmid (Table 1). The relative efficiency of plaque formation by Δ(6.7–7.7) mutants is close to unity on BL21 containing either pAR3476 or pOP1, and the morphology of plaques produced is only slightly smaller than when the cell also contains a gene 6.7 plasmid. Presumably, the lack of gp6.7 in infected cells is largely compensated by overexpression of the plasmid-carried gene 7.3 in the infected cell.

Morphologically normal particles are produced after infection of non-permissive cells by mutant phages that contain amber mutations in either gene 7.3 or gene 13 (Studier, 1972; Roeder and Sadowski, 1977). We show here that defective virions can be prepared from gene 7.3 or 13 deletion mutants and also from a gene 6.7–7.7 deletion mutant that lacks both gp6.7 and gp7.3 (gp7.7 is not considered a structural protein). Virions were purified by equilibrium density gradient centrifugation in CsCl and their proteins were examined by SDS-PAGE (Fig. 3). The electrophoretic mobilities of gp6.7, gp7, gp7.3, and gp13 are all similar and the relative order of migration changes in gels of only slightly different composition. For accurate identification, it is necessary that each gel includes markers made from cells containing appropriate expression plas-

Table 1

| T7 strain | Plasmid               | Plasmid-borne T7 genes | Relative titer |
|-----------|-----------------------|------------------------|----------------|
| Δ(6.5–6.7) | pAR775               | 6.5+6.7                | BL21: 1        |
|           |                       |                        | IJ1133: 1.0    |
|           | pAR3687              | 6.7                    | BL21: 1.1      |
|           |                       |                        | IJ1133: 0.94   |
|           | None                 |                        | BL21: $1.8 \times 10^{-3}$ (pp)$^a$ |
|           |                       |                        | IJ1133: 0.79 (pp) |
| Δ(6.7–7.7) | pAR3687+pPK95        | 6.7+7.3                | BL21: 1        |
|           |                       |                        | IJ1133: 0.9    |
|           | pAR3687              | 6.7                    | BL21: $2.3 \times 10^{-6}$ |
|           |                       |                        | IJ1133: $7 \times 10^{-7}$ |
|           | pAR3476              | 7.3                    | BL21: 0.97     |
|           |                       |                        | IJ1133: 0.9    |
|           | pOP1                 | 7.3                    | BL21: 0.72     |
|           |                       |                        | IJ1133: 1.1    |
|           | None                 |                        | BL21: $3.2 \times 10^{-9}$ |
|           |                       |                        | IJ1133: $6 \times 10^{-10}$ |
| Δ(7.3–7.7) | pAR3476              | 7.3                    | BL21: 1        |
|           |                       |                        | IJ1133: 0.94   |
|           | pOP1                 | 7.3                    | BL21: 1.0      |
|           |                       |                        | IJ1133: 1.0    |
|           | None                 |                        | BL21: $9.0 \times 10^{-7}$ |
|           |                       |                        | IJ1133: $2.6 \times 10^{-5}$ |
| Δ7.3      | pOP1                 | 7.3                    | BL21: 1        |
|           |                       |                        | IJ1133: 1.3    |
|           | None                 |                        | BL21: $8.4 \times 10^{-7}$ |
|           |                       |                        | IJ1133: $1.7 \times 10^{-5}$ |
| Δ(11–12)  | pAR5435              | 11+12                  | BL21: 1        |
|           |                       |                        | IJ1133: 1      |
|           | None                 |                        | BL21: $5.9 \times 10^{-7}$ |
|           |                       |                        | IJ1133: $5.9 \times 10^{-7}$ |
| Δ13       | pPK64                | 13                     | BL21: 1        |
|           |                       |                        | IJ1133: 1      |
|           | None                 |                        | BL21: $8.4 \times 10^{-6}$ (pp) |
|           |                       |                        | IJ1133: $8.9 \times 10^{-6}$ (pp) |

$a$ pp = pinprick-sized.

virions assembled in the absence of the tail fiber also lack gp7.3. At first blush, this observation implies that gp7.3 may be part of the tail fiber. However, this reasoning cannot be correct; the structure of tail fibers purified from cells containing a clone of gene 17 is the same as fibers isolated directly from mature particles (Steven et al., 1988).

Tail fibers can be added in vitro to lysates of fiber-less particles to produce infective virions (Studier, 1969; Issinger and Hausmann, 1973; Steven et al., 1988), a result not readily compatible with the absence of gp7.3 in fiber-less virions when gene 7.3 has been shown to be essential (Table 1). One possibility is that purifying fiber-less virions by equilibrium density gradient centrifugation in CsCl for SDS-PAGE analysis caused gp7.3 to be lost. We therefore added a crude preparation of gp17, prepared from cells containing a plasmid that overexpressed gene 17, to both lysates and highly purified fiber-less particles. In agreement with the earlier observations, lysates of fiber-less particles can be complemented by exogenous gp17 to yield infective particles (Fig. 5). However, purified phage virions, known by SDS-PAGE analysis to lack gp7.3 in addition to gp17, were complemented equally well. Thus, mature virions do not require gp7.3 to initiate an infection. We thought that gene 7.3 could be essential only for tail assembly. When assembly is complete, gp7.3 may no longer be required even though it normally remains a part of the mature virion. This idea was tested by constructing a Δ7.3, 17am double mutant; defective virions prepared under non-permissive conditions were unable to be complemented in vitro by the addition of gp17, or by the addition of gp17 and gp7.3 (Fig. 5 and data not shown). Defective virions prepared from the single mutant Δ7.3 were also unable to be complemented in vitro. We conclude that gp7.3 is necessary only to assemble infective particles.

Non-infectious, tail fiber-less particles were prepared using various gene 17 amber mutants and purified by CsCl density gradients. None of these virions contain any associated gp17-related protein (Fig. 4), even though 17am75 and 17am61 make products that are, respectively, 85% and 88% of full-length. Both amber peptides are stable in infected cells (not shown). It seems likely that the entire protein must be synthesized before an assembly-competent form of the tail fiber can be generated. Surprisingly, purified

Fig. 4. Mutant 17am virions lack gp7.3 in addition to gp17.

Virions lacking the tail fiber gp17 also lack gp7.3

Fig. 5. In vitro complementation of tail-less virions. Lysates (open symbols) or CsCl-purified (closed symbols) tail-less virions of 17am8 (○, ●), 17am145 (□, ■), and Δ7.3, 17am145 (▲) were incubated with a crude cell extract of BL21(pAR3685). After 30 min, the mixture was titered on the Su2⁺ strain IJ511 or IJ511 containing the gene 7.3 plasmid pOP1.

Adsorption assays

In order to determine the reason for the lack of infectivity of various mutants, ${}^{35}$S-labeled mutant T7 virions were prepared and purified using discontinuous and then equilibrium density gradient centrifugation in CsCl. Virions were added to exponential phase $E$. coli BL21 cells grown at 30 °C to a density of $2 \times 10^8$ per ml at an estimated ($A_{260}$ of 1 = $\sim 5 \times 10^{11}$ virions per ml) multiplicity of 5. After 5 min incubation at 30 °C to allow for adsorption, cells and unadsorbed phages were separated by centrifugation and the radioactivity associated with both cells and supernatant fraction was measured by scintillation counting. More than 90% of wild-type T7 adsorbed to cells under these conditions (Table 2); in contrast, all the mutant virions tested were completely or partially defective. Surprisingly, virions made in the absence of gp7.3 are completely defective in adsorption, even though they contain a normal complement of tail fibers. Thus, the tail fibers of a virion assembled in the absence of gp7.3 cannot adequately interact with the cell LPS in order to make the phage–bacterium complex stable to centrifugation. No significant difference in adsorption was observed between particles that lacked gp7.3 and those lacking both gp7.3 and gp17, both were as defective as particles lacking any tail-associated proteins. Virions made in the absence of gene 6.7 adsorbed less efficiently than wild-type, although they were not as defective as virions made in the absence of gene 13, which also lack gp6.7. This suggests that gene 13 activity is required for correct assembly or incorporation of virion components in addition to gp6.7. Virions assembled in the absence of gene 13 contain variable amounts of gp7.3 (Fig. 3), and the difference in adsorption properties between $6.7^{-}$ and $13^{-}$ virions could be explained if a fraction of the latter completely lack gp7.3 in addition to lacking gp6.7; such virions would be incapable of adsorption.

Stoichiometry of proteins in the T7 virion

In order to determine the relative proportions of proteins in the virion, proteins were continuously labeled with [${}^{35}$S]methionine during infection, and virions were purified by two successive equilibrium CsCl gradients. Proteins were separated on SDS-gels (Fig. 6), and radioactive bands were quantitated by phosphorimagery. Band intensities were then converted into relative amounts of protein molecules using the predicted sequences of T7 proteins, after correction for the retention or removal of the N-terminal Met. We normalized protein copy number to the 12 copies of the portal protein gp8, as this number is known precisely from cryoelectron microscopy reconstructions (Valpuesta et al., 1992, 2000; Agirrezabala et al., 2005). The data presented in Table 3 are an average of 11 separate gels, each containing samples prepared from three independent stocks of phage. About 445 copies of the capsid protein were estimated, in good agreement with the 415 copies expected for the T = 7 T7 capsid. About 4% of the capsid protein is present in the minor gp10B species, which results from a programmed translational frameshift near the end of the gene $10A$ coding sequence (Dunn and Studier, 1983; Condron et al., 1991a, 1991b). The number of copies of gp15 was found to be

Table 2  
Adorption of T7 mutant particles to BL21  

| Mutant       | Protein(s) missing         | % Adsorbed phage${}^{a}$ |
|--------------|----------------------------|---------------------------|
| Wild-type    | None                       | 93.4                      |
| $\Delta 6.7-7.7$ (+gp7.3) | gp6.7                     | 73.9                      |
| $\Delta 7.3$      | gp7.3                     | 7.0                       |
| $\Delta 11-12$    | gp7.3, gp11, gp12, gp17   | 4.4                       |
| $13am89$     | gp6.7                      | 27.4                      |
| $17am8$      | gp7.3, gp17                | 6.4                       |

${}^{a}$ Average of at least two experiments.

Table 3  
Composition of the T7 virion  

| Copies | Function                   | Expectation from symmetry |
|--------|----------------------------|---------------------------|
| **Head proteins**                  |                           |                           |
| gp6.7  | DNA completion?            | ?                         |
|        | Ejected protein            |                           |
| gp8    | Head-tail connector        | 12                        |
| gp10A  | Major capsid shell         | 415${}^{a}$               |
| gp10B  | Minor capsid shell         |                           |
| gp14   | Internal core; ejected protein | 10 (8 or 12?)          |
| gp15   | Internal core; ejected protein | 8                        |
| gp16   | Internal core; ejected protein | 4                        |
| **Tail proteins**                  |                           |                           |
| gp7.3  | Adsorption; ejected protein | ?                         |
| gp11   | Structural tail            | 12                        |
| gp12   | Structural tail            | 6                         |
| gp17   | Tail fiber; adsorption     | 18                        |

${}^{a}$ gp10A + gp10B.

close to eight, consistent with the observed 8-fold symmetry of the internal core (Cerritelli et al., 2003). In agreement with Serwer (1976) and within the range discussed by Cerritelli et al. (2003), four copies of gp16 are present in the virion, which could also be consistent with the eightfold symmetry of the core if each copy of gp16 was associated with two gp15 molecules. However, ten copies of the third core component, gp14, were found. This is the value originally calculated by Adolph and Haselkorn (1972), but is less than the revised estimate from the same data by Cerritelli et al. (2003). However, the value of 10.9 ± 2.4 obtained by these authors is consistent with the value of 9.7 ± 0.6 found in this work. If the actual value is ten, the protein may be matched with the fivefold symmetry of the capsid protein at the vertex where the head–tail connector is inserted, and a symmetry mismatch must exist between gp14 and the other two internal core proteins. The mass of the internal core is dominated by gp15 and gp16, and any deviation from a strict 8-fold symmetry would be difficult to observe. The specific location within the head of the 18 copies of gp6.7 is not known, and thus no expectation with principles of symmetry can be made at this time.

In large agreement with earlier estimates (Adolph and Haselkorn, 1972; Serwer, 1976; Cerritelli et al., 2003), six copies of gp12 and 12 copies of gp11 are found in each virion. These two proteins form the bulk of the sixfold symmetrical tail. About 30 copies of gp7.3 were found but the relationship of gp7.3 to the remainder of the tail structure is not yet known and symmetry estimations cannot be made. Somewhat less than the expected 18 copies of the gp17 tail fiber were found; an even lower estimate came from the re-evaluation by Cerritelli et al. (2003) of the data of Adolph and Haselkorn (1972). The latter authors used ¹⁴C-labeling of phage proteins, comparable to the ³⁵S-labeling employed here. In contrast, both Serwer (1976) and Cerritelli et al. (2003) estimated about 20 copies from densitometry of band stained with Coomassie Blue. The structure of the tail fiber is clearly a trimer of gp17, and there are equally clearly six tail fibers positioned symmetrically around the tail (Steven et al., 1988). We have no explanation for the lower values obtained with radioactive labeling procedures. It is possible that not all particles contained a full complement of tail fibers. However, phage growth in the presence of a gene 17 plasmid, which would be expected to increase the intracellular concentration of fibers, did not affect the number of gp17 molecules found associated with each virion.

portal vertex, at least in tail-less virions. The 18 copies of gp6.7 in the virion suggest that it could display sixfold symmetry in the virion, an idea that would be compatible with the protein being in contact with the 12-fold symmetrical portal, where a plug function is to be expected. However, other possibilities — including gp6.7 not being symmetrically organized in the capsid — cannot be ruled out from the available data. Infections by gene 6.7 deletion mutant virions into non-permissive hosts lead to minute plaques at approximately normal efficiency on K-12 strains, but at reduced levels on B strains. Both a reduced burst of infective particles and an adsorption defect of mutant virions contribute to the growth defect. Thus, gp6.7 has two functions, it is involved in virion morphogenesis and is also ejected from the infecting particle into the bacterial cell. If the protein does serve as a plug that helps stabilize the packaged phage genome in the head, its removal would be obligatory.

Gene 7.3 was first defined by a collection of “leaky” mutants originally grouped together as “gene 7” (Studier, 1969). The genome sequence (Dunn and Studier, 1983) eventually revealed that “gene 7” mutants represented two genes, which were consequently renamed 7 and 7.3. Both genes were suggested to affect host range (Studier, 1975) but no further studies appear to have been conducted. We have now shown that gp7.3 is ejected into the cell at the initiation of infection, suggesting that it is located either within the channel that runs through the core of the tail or at its tip. One activity of gp7.3 may therefore be to protect the internal portion of the tail from environment insults. However, gp7.3 must do more than play a passive protective role. Virions lacking gp7.3 are defective in adsorption, suggesting that the protein may normally interact with a component on surface of the infected cell. An analogous protein could be T4 gp12, which forms the short tail fibers of the T4 baseplate. Gp12 binds to LPS (Riede, 1987), and stabilizes the phage–cell complex to allow successful tail tube penetration. However, unlike T7 7.3⁻ virions, those of T4 12⁻ do adsorb and undergo normal tail contraction but are then released from the infected cells (King, 1968; Simon et al., 1970).

The initial productive interaction of most tailed phages with the bacterial cell surface is understood to utilize their tail fibers, and there is no compelling reason to think that T7 is an exception. Tail fiber-less virions prepared from gene 17 amber mutants also lack gp7.3 but it seems clear that gp7.3 cannot be part of the tail fiber itself. Identical fibers to those found on virions can be assembled in vitro from purified protein synthesized from the cloned gene (Steven et al., 1988), a result suggesting that gp7.3 is stabilized in the tail by the addition of tail fibers. However, 17⁻ virions, that lack both gp17 and gp7.3, can be rendered infective by the in vitro addition of gp17 alone. It is not known whether this complementation reaction is as efficient as normal tail fiber addition in vivo, or whether higher than normal concentrations of gp17 drive the in vitro reaction in the absence of

Discussion

This study has revealed two proteins of the T7 virion that were not previously described. Tail-less particles contain gp6.7, which defines it as a head protein; in contrast, the same particles lack gp7.3, which is therefore a component of the tail. It is possible that gp6.7 serves as the plug at the

gp7.3. Nevertheless, as fiber-less particles assembled in the absence of gp7.3 cannot be complemented in vitro with fibers (with or without gp7.3), the essential activity of gp7.3 must lie during virion morphogenesis. We suggest that gp7.3 is an assembly factor for the tail, perhaps forming a scaffold around which gp11 and gp12 polymerize, and that after it performs its essential function it passively remains associated with the mature virion. It is noted that, unlike most other tailed phages, T7 tail assembly occurs on the DNA-filled head ([Studier and Maizel, 1969](https://doi.org/10.1073/pnas.63.4.1058); [Matsuo-Kato et al., 1981](https://doi.org/10.1016/0022-2836(81)90356-8)). The failure of virions assembled in the absence of gp7.3 to adsorb stably to cells therefore means that adsorption of T7 virions requires, in addition to a tail fiber-LPS complex, a productive interaction of at least one of the tail proteins gp11 or gp12 with a cell surface component in order to become stable to brief centrifugation. The initial interaction of T4 tail fibers with LPS is also known to be weak ([Wilson et al., 1970](https://doi.org/10.1073/pnas.67.1.1)), in fact a weak interaction allows a phage to randomly diffuse over a cell surface in a search for a favorable site to initiate infection (for a review, see [Goldberg et al., 1994](https://doi.org/10.1016/0022-2836(94)90149-9)).

After adsorption, five T7 proteins are ejected from the virion into the cell. With the exception of gp7.3, these proteins must pass through the head-tail connector. The T7 and T3 connectors are similar in structure, and both are similar to connectors of unrelated phages ([Agirrezabala et al., 2005](https://doi.org/10.1074/jbc.M502688200)). An open channel runs through the T3 connector purified from the cloned gene, but the channel is closed when the connector is isolated from virions ([Donate et al., 1988](https://doi.org/10.1016/0022-2836(88)90488-8)), suggesting that another protein—perhaps gp6.7?—is present. The channel has an average diameter of ~3.7 nm but with a constriction down to ~2.2 nm. The proteins ejected from the T7 capsid must presumably pass through this constricted channel as they travel into the cell. The major functions of the ejected proteins are to form a channel across the cell envelope, including localized degradation of the cell wall ([Molineux, 2001](https://doi.org/10.1016/S0022-2836(01)00595-6); [Moak and Molineux, 2000](https://doi.org/10.1016/S0022-2836(00)97240-3)), and to translocate the leading end of the phage genome by what appears to be an enzyme-catalyzed reaction ([García and Molineux, 1995, 1996](https://doi.org/10.1016/0022-2836(95)90386-9); [Struthers-Schlinke et al., 2000](https://doi.org/10.1016/S0022-2836(00)97240-3); [Kemp et al., 2004](https://doi.org/10.1016/j.virol.2004.04.036)).

A remarkable series of conformational changes must occur in these ejected proteins at the initiation of infection. The ten copies of gp14, four of gp16, and eight of gp15 that comprise the internal core structure inside the mature capsid must disaggregate and unfold almost completely in order to pass through the connector. The energy required for these processes may be provided during packaging in the previous host and stored within the mature virus particle. T4 particles are thought to be in a metastable state, the energy required for the hexagon to star transition of the baseplate and for tail contraction seemingly residing within the particle (for a review, see [Coombs and Arisaka, 1994](https://doi.org/10.1016/0022-2836(94)90149-9)). It is likely that T7 and perhaps most phage virions are in a comparable metastable state. More puzzling is the process by which about forty distinct T7 protein molecules sequentially exit the capsid through the connector and tail to enter the infected cell. The total mass of these molecules is more than 10% of the total protein content, and they constitute more than 60% of the non-head shell proteins of the virion. The problem is not so much in the energetics of movement but how does each protein molecule find the exit channel? Several phages insert a genome end partway down the tail during morphogenesis ([Chattoraj and Inman, 1974](https://doi.org/10.1073/pnas.71.10.3951); [Saigo, 1975](https://doi.org/10.1073/pnas.72.10.4006); [Saigo and Uchida, 1974](https://doi.org/10.1073/pnas.71.10.3951)), thereby ensuring that genome ejection from the capsid is vectorially controlled. However, this simple but elegant solution for a phage genome does not readily lend itself to multiple—and different—protein molecules. One possibility is that each end of a protein chain has an affinity for an end of a separate molecule, making the ensemble of molecules equivalent to a single long polypeptide. However, inspection of the amino acid sequences of the proteins ejected does not reveal an obvious source of the proposed affinity between molecules. An alternative idea is that one end of the first molecule to leave the virion lies within the channel and exits through the tail, as the trailing end of the same molecule begins to exit it exposes the leading end of the second molecule to the channel. No specific interaction between different protein molecules would be required, only that the compact structure of the internal core be constructed such that the end of one polypeptide chain be located in the vicinity of the end of a second chain.

One function of the ejected internal core proteins must be to extend the length of the normal tail of the virion and to create a channel that extends from the head-tail connector into the cell cytoplasm. A channel is necessary to guide genome translocation across the cell envelope; a channel can also presumably protect the entering DNA from the periplasmic endonuclease I. Mature T7 virions may therefore be considered functionally to maintain their tail in a retracted conformation; tail extension occurs only after adsorption of virions to the surface of a susceptible bacterium. This process is in contrast to phage T4, whose tail is extended in mature virions. After the T4 tail fibers bind to cell surface LPS and the baseplate reorients to allow the tail pins to lock onto the cell, the outer sheath of the tail contracts. The internal tail tube, which does not change its length during infection, then penetrates the cell.

**A model for the initiation of infection by T7**

In the following scheme, gp7.3 is assumed to fill the channel running the length of the T7 tail, whose major components are gp11 and gp12. As discussed above, it is possible that gp7.3 actually makes contact with the cell surface but contact is not a pre-requisite for a successful infection, only that the tail was assembled in the presence of gp7.3.

The initial productive interaction between T7 and a bacterium involves the tail fibers and the cell surface LPS. This interaction is weak, perhaps serving to constrain phage diffusion to a 2-dimensional walk over the cell surface in search of a site where a successful infection can occur. This site has not yet been defined but is where one or both of the tail proteins gp11 and gp12 interact with an unknown cell surface component. It is therefore expected that host range mutants might contain a gene 11 or 12 mutation(s). Such mutants have not yet been described for T7 although an expanded host range mutant of the salmonellaphage SP6 that has acquired the ability to grow on *E. coli* has been shown to contain mutations in its tail protein genes (A.L. Stell and I.J.M., unpublished data). The two SP6 tail proteins are 32 and 36% identical in sequence with, respectively, T7 gp11 and gp12 (Scholl et al., 2004). The combined interaction of both the tail and tail fibers with the cell causes a conformational change in the tail fibers that allows release of gp7.3 from the tail into the cell. Loss of gp7.3 is normally followed by the ejection of gp6.7 from the head. In the situation where tail fibers have been added to defective virions in vitro and where there is therefore no gp7.3 in the tail, a signal must still be transmitted to gp6.7 to cause its ejection. It seems unlikely that this signal involves only the tail fibers as these are connected to the tail, below where the latter is attached to the head–tail connector (Studier, 1972; Serwer, 1976; Steven et al., 1988). In this situation, the signal may be transduced from the tail to the connector, and that the connector passes the signal on to gp6.7.

An alternative thought satisfies the puzzling observations that gp7.3 is required during virion morphogenesis and is a significant component of the mature tail in wild-type phage but is not required for the infectivity of a mature particle. Perhaps gp7.3 assembles the tail and tail fiber proteins in a metastable state, in preparation for the irreversible steps that occur at the initiation of infection. In this view, the signal to eject head proteins is transmitted via a conformational change in the tail once its tip has bound to a cell surface component. In the absence of gp7.3 during morphogenesis, the tail proteins gp11 and gp12 are therefore in a stable conformation that allows them to adsorb to the cell but makes them unable to transmit a signal from the cell surface to the phage head.

Loss of gp6.7 from the infecting particle presumably allows disaggregation of the internal core structure and ejection of gp14, gp15, and gp16 into the cell. At this stage, adsorption can clearly be defined as irreversible; gp16 plays an essential role in the subsequent translocation of the phage genome (García and Molineux, 1995, 1996; Struthers-Schlinke et al., 2000; Moak and Molineux, 2000, 2004; Kemp et al., 2004), and comparable important functions are likely to be associated with gp14 and gp15. Loss of the internal core from a virion inevitably makes it unable to infect another cell.


Methods of procedure

Bacteria, phages, and plasmids

BL21 (*E. coli* B *gal hsdS*) and IJ1133 [*E. coli* K-12 Δ*lacX74 thi Δ(mcrC-mrr)102::Tn10*] were used for most studies. IJ511 (*E. coli* K-12 Δ*lacX74 supE44 galK2 galT22 mcrA rfbD1 mcrB1 hsdS3*) was used as the permissive host for T7 amber mutants. Wild-type T7 isolates included the strain maintained in this laboratory for many years (GenBank accession #AY264774) and a stock recently obtained from F.W. Studier (Accession # V01146). T7 amber mutants, Δ(6.5–6.7) and 10B⁻, Δ(11–12) were from the collection of F.W. Studier. Δ(6.7–7.7) was made by ligating the left-end *StuI* fragment to the filled-in *BstEII* right-end fragment, followed by transfection into a complementing host to obtain phage particles. Δ(7.3–7.7) was made similarly except that the left end was the *BanII* fragment ligated via a linker oligonucleotide to the right-end *BstEII* fragment. A complete deletion of gene 7.3 was made on a plasmid using overlap PCR, nucleotides 19532 to 19838 of T7 were replaced with a *BamHI* sequence. The DNA sequence between the unique *StuI* and *BstEII* sites was determined to confirm the accuracy of the PCR reactions. The *StuI-BstEII* fragment of the plasmid was then ligated to the left-end *StuI* and the right-end *BstEII* fragments of wild-type T7 DNA. A deletion of gene 13 (removing T7 nucleotides 27310–27731) was made on a plasmid that contained T7 DNA from the unique *PacI* (previously converted into an *EcoRI* site, García and Molineux, 1995) and *NcoI* sites. After confirmation of the desired sequence, the *PacI(EcoRI)-NcoI* sequence was ligated to the left-end *PacI(EcoRI)* and right-end *NcoI* fragments of T7 DNA. This mutant also contains an uncharacterized mutation that affects the electrophoretic mobility of gp11 in Tris-glycine but not tricine-buffered gels. Plasmids are listed in Table 4.

Phage manipulations

Standard procedures were used for routine phage propagation; one-step growth curves were performed at a multiplicity of 0.1. The screen for temperature-sensitive mutants was conducted as follows. Individual plaques of


Table 4  
Plasmids  

| Plasmid   | Vector    | Promoter | T7 genes | Source       |
|-----------|-----------|----------|----------|--------------|
| pAR775    | pBR322    | φ6.5     | 6.5 + 6.7 | F.W. Studier |
| pAR3476   | pET-1     | φ10      | 7.3      | F.W. Studier |
| pAR3685   | pBR322    | φ17      | 17       | F.W. Studier |
| pAR3687   | pET-1     | φ10      | 6.7      | F.W. Studier |
| pAR5435   | pET-1     | φ10      | 11 + 12  | F.W. Studier |
| pOP1      | pACYC184  | φ10      | 7.3      | This work    |
| pPK64     | pUC19     | lac      | 13       | This work    |
| pPK95     | pACYC184  | φ10      | 7.3      | This work    |

13am89 on IJ511 were resuspended and titered on the non-permissive host BL21. Plaques of $am^+$ revertants were then gridded onto BL21 using pairs of tryptone plates without NaCl and incubated at 30 °C or 43 °C. Phages that failed to grow at 43 °C were purified at 30 °C and re-tested for temperature-sensitivity. Only one Ts mutant found in each initial plaque on BL21 was studied further. The mutation in each strain was then localized by marker rescue using wild-type T7 DNA carried on various plasmids.

### Adsorption assay

Wild-type and mutant virions were labeled using media containing $[^{35}S]$methionine, and purified by equilibrium CsCl density gradient centrifugation. Concentrations of phages were determined by absorbance (1 A$_{260}$ corresponds to $\sim 5 \times 10^{11}$ particles per ml) and, if possible, by titering. Assays used a multiplicity of infection of 5 in exponential phase BL21 cells at 30 °C. After 10 min of infection, an aliquot was taken to calculate the total radioactivity, and then cells and unadsorbed phages were separated by centrifugation. The radioactivity associated with the whole culture, cells and supernatant fractions was determined after adding an equal volume of 10% TCA and collecting the precipitate on a glass fiber filter.

### In vitro complementation

Crude preparations of gp17 or gp7.3 were prepared from stationary phase cultures of BL21(DE3) containing, respectively, pAR3685 or pAR3476. No induction of gene 1 was employed. Cells were concentrated 20-fold by centrifugation and resuspended in a buffer containing 50 mM Tris-HCl, 2 mM EDTA, 4% (v/v) glycerol, pH 8.0, and incubated for 30 min at 30 °C with 200 μg per ml egg-white lysozyme. The mixture was freeze-thawed twice to complete lysis, 10 mM MgCl₂ and 20 μg per ml pancreatic DNase I was added, and the mixture was incubated for a further 10 min. Between 1 and 100 μl of the clarified cell extract was then added to 1 ml of a mutant T7 lysate in L broth, or to CsCl-purified mutant phage appropriately diluted with broth. After 30 min incubation at 30 °C, dilutions of the mixtures containing 17am8 or 17am145 mutant particles were plated using IJ511 as indicator bacteria, the mixture containing Δ7.3, 17am145 mutant particles were plated on IJ511(pOP1).

### SDS-PAGE

Most gels, including those shown in Fig. 6, used 17% acrylamide, 0.085% bis-acrylamide, 0.1% SDS, in 0.375 M Tris-HCl, pH 8.75 (20 °C) for separation with a stacking gel of 4% acrylamide, 0.75% bis-acrylamide, 0.1% SDS, in 0.125 M Tris-HCl, pH 6.8. The electrode buffer contained 0.05 M Tris glycine, 0.2% SDS, pH 8.3 and was circulated during electrophoresis at 4 °C for 19 h using a constant current of 12.5 mA.

## Acknowledgments

We are grateful to Drs. A.H. Rosenberg and F.W. Studier for their generous gift of T7 mutants and complementing plasmids. We thank Olivia Perng for construction of the T7 Δ7.3 mutant. This work was supported by grant GM32095 from the National Institutes of Health.

## References

Adolph, K.W., Haselkorn, R., 1972. Comparison of the structures of blue-green algal viruses LPP-1M and LPP2 and bacteriophage T7. Virology 47, 701–710.

Agirrezabala, X., Martín-Benito, J., Valle, M., González, J.M., Valencia, A., Valpuesta, J.M., Carrascosa, J.L., 2005. Structure of the connector of bacteriophage T7 at 8 Å resolution: structural homologies of a basic component of a DNA translocating machinery. J. Mol. Biol. 347, 895–902.

Beck, P.J., Gonzalez, S., Ward, C.L., Molineux, I.J., 1989. Sequence of bacteriophage T3 DNA from gene 2.5 through gene 9. J. Mol. Biol. 210, 687–701.

Carazo, J.M., Fujisawa, H., Nakasu, S., Carrascosa, J.L., 1986. Bacteriophage T3 gene 8 product oligomer structure. J. Ultrastruct. Mol. Struct. Res. 94, 105–113.

Cerritelli, M.E., Studier, F.W., 1996. Purification and characterization of T7 head-tail connectors expressed from the cloned gene. J. Mol. Biol. 258, 299–307.

Cerritelli, M.E., Cheng, B., Rosenberg, A.H., McPherson, C.E., Booy, F.P., Steven, A.C., 1997. Encapsidated conformation of bacteriophage T7 DNA. Cell 91, 271–280.

Cerritelli, M.E., Trus, B.L., Smith, C.S., Cheng, N., Conway, J.F., Steven, A.C., 2003. A second symmetry mismatch at the portal vertex of bacteriophage T7: 8-fold symmetry in the procapsid core. J. Mol. Biol. 327, 1–6.

Chattoraj, D.K., Inman, R.B., 1974. Location of DNA ends in P2, P4, and lambda bacteriophage heads. J. Mol. Biol. 87, 11–22.

Condrey, J.P., Wright, S.E., Molineux, I.J., 1989. Nucleotide sequence and complementation studies of the gene 10 region of bacteriophage T3. J. Mol. Biol. 207, 555–561.

Condron, B.G., Atkins, J.F., Gesteland, R.F., 1991a. Frameshifting in gene 10 of bacteriophage T7. J. Bacteriol. 173, 6998–7003.

Condron, B.G., Gesteland, R.F., Atkins, J.F., 1991b. An analysis of sequences stimulating frameshifting in the decoding of gene 10 of bacteriophage T7. Nucleic Acids Res. 19, 5607–5612.

Coombs, D.H., Arisaka, F., 1994. T4 tail structure and function. In: Karam, J.D. (Ed.), Molecular Biology of Bacteriophage T4. American Society for Microbiology, Washington, DC, pp. 259–281.

Donate, L.E., Herranz, L., Secilla, J.P., Carazo, J.M., Fujisawa, H., Carrascosa, J.L., 1988. Bacteriophage T3 connector: three-dimensional structure and comparison with other viral head-tail connecting regions. J. Mol. Biol. 201, 91–100.

Dunn, J.J., Studier, F.W., 1983. Complete nucleotide sequence of bacteriophage T7 DNA and the locations of genetic elements. J. Mol. Biol. 166, 477–535.

García, L.R., Molineux, I.J., 1995. Rate of translocation of bacteriophage T7 DNA across the membranes of Escherichia coli. J. Bacteriol. 177, 4066–4076.

García, L.R., Molineux, I.J., 1996. Transcription-independent DNA trans-

location of bacteriophage T7 DNA into *Escherichia coli*. J. Bacteriol. 178, 6921–6929.

Goldberg, E.B., Grinius, L., Letellier, L., 1994. Recognition, attachment, and injection. In: Karam, J.D. (Ed.), Molecular Biology of Bacteriophage T4. American Society for Microbiology, Washington, DC, pp. 347–356.

Israel, V., 1977. E proteins of bacteriophage P22. J. Virol. 23, 91–97.

Issinger, O., Hausmann, R., 1973. Synthesis of bacteriophage-coded gene products during infection of *Escherichia coli* with amber mutants of T3 and T7 defective in gene I. J. Virol. 11, 465–472.

Kato, H., Fujisawa, H., Minagawa, T., 1985. Purification and characterization of gene 17 product of bacteriophage T3. Virology 146, 22–26.

Kato, H., Fujisawa, H., Minagawa, T., 1986. Subunit arrangement of the tail fiber of bacteriophage T3. Virology 153, 80–86.

Kemp, P., Gupta, M., Molineux, I.J., 2004. Bacteriophage T7 DNA ejection into cells is initiated by an enzyme-like mechanism. Mol. Microbiol. 53, 1251–1265.

King, J., 1968. Assembly of the tail of bacteriophage T4. J. Mol. Biol. 32, 231–262.

Kocsis, E., Cerritelli, M.E., Trus, B.L., Cheng, N., Steven, A.C., 1995. Improved methods for determination of rotational symmetries in macromolecules. Ultramicroscopy 60, 219–228.

Luria, S.E., 1953. General Virology. Wiley, New York.

Matsuo-Kato, H., Fujisawa, H., Minagawa, T., 1981. Structure and assembly of bacteriophage T3 tails. Virology 109, 157–164.

Meerman, H.J., Georgiou, G., 1994. Construction and characterization of a set of *E. coli* strains deficient in all known loci affecting the proteolytic stability of secreted recombinant proteins. Bio/Technology (NY) 12, 1107–1110.

Moak, M., Molineux, I.J., 2000. Role of the Gp16 lytic transglycosylase motif in bacteriophage T7 virions at the initiation of infection. Mol. Microbiol. 37, 345–355.

Moak, M., Molineux, I.J., 2004. Peptidoglycan hydrolytic activities associated with bacteriophage virions. Mol. Microbiol. 51, 1169–1183.

Molineux, I.J., 2001. No syringes please, ejection of T7 DNA from the virion is enzyme-driven. Mol. Microbiol. 40, 1–8.

Morita, M., Tasaka, M., Fujisawa, H., 1994. Analysis of functional domains of the packaging proteins of bacteriophage T3 by site-directed mutagenesis. J. Mol. Biol. 235, 248–259.

Morita, M., Tasaka, M., Fujisawa, H., 1995. Structural and functional domains of the DNA packaging protein of bacteriophage T3: importance of the C-terminal region of the large subunit in prohead binding. J. Mol. Biol. 245, 635–644.

Pajunen, M.I., Elizondo, M.R., Skurnik, M., Kieleczawa, J., Molineux, I.J., 2002. Complete nucleotide sequence and likely recombinatorial origin of bacteriophage T3. J. Mol. Biol. 319, 1115–1132.

Puck, T.T., Garen, A., Cline, J., 1951. The mechanism of virus attachment to host cells. J. Exp. Med. 93, 65–80.

Riede, I., 1987. Receptor specificity of the short tail fibres (gp12) of T-even type *Escherichia coli* phages. Mol. Gen. Genet. 206, 110–115.

Roeder, G.S., Sadowski, P.D., 1977. Bacteriophage T7 morphogenesis: phage-related particles in cells infected with wild-type and mutant T7 phage. Virology 76, 263–285.

Rontó, G., Agamalyan, M.M., Drabkin, G.M., Feigin, L.A., Lvov, Y.M., 1983. Structure of bacteriophage T7. Biophys. J. 43, 309–314.

Saigo, K., 1975. Tail-DNA connection and chromosome structure in bacteriophage T5. Virology 68, 154–165.

Saigo, K., Uchida, H., 1974. Connection of the right-hand terminus of DNA to the proximal end of the tail in bacteriophage lambda. Virology 61, 524–536.

Scholl, D., Kieleczawa, J., Kemp, P., Rush, J., Richardson, C.C., Merril, C., Adhya, S., Molineux, I.J., 2004. Genomic analysis of bacteriophages SP6 and K1-5, an estranged subgroup of the T7 supergroup. J. Mol. Biol. 335, 1151–1171.

Serwer, P., 1976. Internal proteins of bacteriophage T7. J. Mol. Biol. 107, 271–291.

Serwer, P., Khan, S.A., Hayes, S.J., Watson, R.H., Griess, G.A., 1997. The conformation of packaged bacteriophage T7 DNA: informative images of negatively stained T7. J. Struct. Biol. 120, 32–43.

Simon, L.D., Swan, J.G., Flatgaard, J.E., 1970. Functional defects in T4 bacteriophages lacking the gene II and gene 12 products. Virology 41, 77–90.

Steven, A.C., Trus, B.L., 1986. Electron microscopy of proteins. In: Harris, J.R., Horne, R.W. (Eds.), Viral Structure, vol. 5. Academic Press, New York, pp. 1–35.

Steven, A.C., Trus, B.L., Maizel, J.V., Unser, M., Parry, D.A., Wall, J.S., Hainfeld, J.F., Studier, F.W., 1988. Molecular substructure of a viral receptor-recognition protein. The gp17 tail-fiber of bacteriophage T7. J. Mol. Biol. 200, 351–365.

Stone, J.C., Miller, R.C., 1985. Spontaneous temperature-sensitive mutations in bacteriophage T7. J. Virol. 54, 886–888.

Stroud, R.M., Serwer, P., Ross, M.J., 1981. Assembly of bacteriophage T7. Biophys. J. 36, 743–757.

Struthers-Schlinke, J.S., Robins, W.P., Kemp, P., Molineux, I.J., 2000. The internal head protein gp16 of bacteriophage T7 controls DNA ejection from the virion. J. Mol. Biol. 301, 35–45.

Studier, F.W., 1969. The genetics and physiology of bacteriophage T7. Virology 39, 562–574.

Studier, F.W., 1972. Bacteriophage T7. Science 176, 367–376.

Studier, F.W., 1975. Gene 0.3 of bacteriophage T7 acts to overcome the DNA restriction system of the host. J. Mol. Biol. 94, 283–295.

Studier, F.W., Maizel, J.V., 1969. T7-directed protein synthesis. Virology 39, 575–586.

Studier, F.W., Rosenberg, A.H., 1981. Genetic and physical mapping of the late region of bacteriophage T7 DNA by use of cloned fragments of T7 DNA. J. Mol. Biol. 153, 503–525.

Tolmach, L.J., 1957. Attachment and penetration of cells by viruses. Adv. Virus Res. 4, 63–110.

Valpuesta, J.M., Fujisawa, H., Marco, S., Carazo, J.M., Carrascosa, J.L., 1992. Three-dimensional structure of T3 connector purified from overexpressing bacteria. J. Mol. Biol. 224, 103–112.

Valpuesta, J.M., Sousa, N., Barthelemy, I., Fernández, J.J., Fujisawa, H., Ibarra, B., Carrascosa, J.L., 2000. Structural analysis of the bacteriophage T3 head-to-tail connector. J. Struct. Biol. 131, 146–155.

Wilson, J.H., Luftig, R.B., Wood, W.B., 1970. Interaction of bacteriophage T4 tail fiber components with a lipopolysaccharide fraction from *Escherichia coli*. J. Mol. Biol. 51, 423–434.
