
Microbiology (2005), 151, 2515–2528

DOI 10.1099/mic.0.28138-0

---

**Identification of tailoring genes involved in the modification of the polyketide backbone of rifamycin B by *Amycolatopsis mediterranei* S699**

Jun Xu,$^{1}$ Eva Wan,$^{1,2}$ Chang-Joon Kim,$^{1,t}$ Heinz G. Floss$^{1}$ and Taifo Mahmud$^{1,2}$

$^{1}$Department of Chemistry, University of Washington, Box 351700, Seattle, WA 98195-1700, USA  
$^{2}$Department of Pharmaceutical Sciences, College of Pharmacy, Oregon State University, Corvallis, OR 97331-3507, USA

---

Rifamycin B biosynthesis by *Amycolatopsis mediterranei* S699 involves a number of unusual modification reactions in the formation of the unique polyketide backbone and decoration of the molecule. A number of genes believed to be involved in the tailoring of rifamycin B were investigated and the results confirmed that the formation of the naphthalene ring moiety of rifamycin takes place during the polyketide chain extension and is catalysed by Rif-Orf19, a 3-(3-hydroxyphenyl)propionate hydroxylase-like protein. The cytochrome P450-dependent monooxygenase encoded by *rif-orf5* is required for the conversion of the Δ12, 29 olefinic bond in the polyketide backbone of rifamycin W into the ketal moiety of rifamycin B. Furthermore, Rif-Orf3 may be involved in the regulation of rifamycin B production, as its knock-out mutant produced about 40% more rifamycin B than the wild-type. The work also revealed that many of the genes located in the cluster are not involved in rifamycin biosynthesis, but might be evolutionary remnants carried over from an ancestral lineage.

---

### INTRODUCTION

Rifamycin continues to play a significant role in clinical medicine. Synthetically modified derivatives, such as rifampicin, rifabutin and rifapentine, remain the principal chemotherapeutic agents used for combating tuberculosis, leprosy and AIDS-related mycobacterial infections (Maggi *et al.*, 1966; Ramos-e-Silva & Rebello, 2001; Sepkowitz *et al.*, 1995). The potent antibacterial activity of this class of antibiotics is due to their specific inhibition of bacterial DNA-dependent RNA polymerases (Campbell *et al.*, 2001; Wehrli & Staehelin, 1969). At the same time, they have relatively low if any activity against eukaryotic RNA polymerases. Due to their high selectivity for their molecular target, the rifamycins have become a safe and effective medication. Unfortunately, as with many other antibiotics, the incidence of resistance of *Mycobacterium tuberculosis*, the causative agent of tuberculosis, to rifamycins is continuing to increase over time, due largely to mutational alterations of the target molecule, the β subunit of RNA polymerase (Kirschbaum & Gotte, 1993; Suzuki *et al.*, 1995). This high-level resistance has contributed to the recent re-emergence of tuberculosis as a major health problem and the consequent increase in the death toll among world populations (Dye *et al.*, 2002). Therefore, new drug discovery and continued development of the existing drugs to combat tuberculosis is indispensable.

Despite the preparation of a large number of rifamycin derivatives by semi-synthetic approaches, the structural modifications have been limited primarily to one region of the molecule, the C-3 or C-4 positions of the aromatic core unit (Cricchio *et al.*, 1974, 1975; Maggi *et al.*, 1965; Wehrli *et al.*, 1987). Alterations at other locations are difficult to accomplish chemically due to the complexity of the molecule, and require the implementation of alternative methodology, including combinatorial biosynthesis or mutasynthesis, to achieve additional structural diversity.

The biosynthesis of rifamycin has been studied extensively by both classical feeding and mutagenesis experiments, as well as by contemporary genetic and biochemical approaches (August *et al.*, 1998; Floss & Yu, 2005; Ghisalba & Nuesch, 1981; Schupp *et al.*, 1998). The polyketide framework of rifamycin B is assembled from

---

tPresent address: Department of Chemical and Biological Engineering, Gyeongsang National University, Jinju 660-701, Korea.

Abbreviations: ACP, acyl carrier protein; AHBA, 3-amino-5-hydroxybenzoic acid; DMDARS, 27-O-demethyl-25-O-desacylrifamycin S; DMDARSV, 27-O-demethyl-25-O-desacylrifamycin SV; DMRSV, 27-O-demethylrifamycin SV; ESI-MS, electrospray ionization mass spectrometry; $hyg^R$, hygromycin-resistance gene; MCS, multiple cloning site; NRPS, non-ribosomal peptide synthetase; PKS, polyketide synthase.

J. Xu and others

3-amino-5-hydroxybenzoic acid (AHBA), which arises from the aminoshikimate pathway (Kim et al., 1998; Yu et al., 2001), two molecules of acetate and eight molecules of propionate. Synthesis of the core structure of rifamycin is catalysed by five multifunctional proteins (RifA–RifE) and an amide synthase (RifF) (Fig. 1A) (August et al., 1998; Schupp et al., 1998). A loading module that contains domains homologous to adenylation and thiolation (A-T) domains of non-ribosomal peptide synthetases (NRPS) is located on RifA and recognizes AHBA as the starter unit (Admiraal et al., 2001, 2002). This is progressively extended by malonyl-CoA and methylmalonyl-CoA via ten individual modules of a type I polyketide synthase (PKS) to give rise to the polyketide backbone of rifamycin. The amide synthase (RifF) catalyses the release of this linear product from the PKS complex and its cyclization via intramolecular amide formation. Inactivation of RifF results in the premature release of a series of polyketides of different chain length from the enzyme complex (Stratmann et al., 1999; Yu et al., 1999), and reveals that the formation of the naphthalene ring of rifamycin occurs during the chain extension. However, further studies are required to determine the mechanism and genes responsible for the construction of the naphthalene core motif.

Proansamycin X has been proposed to be the earliest

(A)

I. PKS

| L | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| orfO | rifA | rifB | rifC | rifD | rifE | rifF |

II. AHBA biosynthesis

| orf1 | rifG | rifH | rifK | rifL | rifN |
| --- | --- | --- | --- | --- | --- |
| rifM | AHBA |

III. Regulatory and post-PKS modification genes

| rifO | rifP | rifQ | orf4 | orf5 | orf6 | orf7 | orf9 | orf11 | orf18 | orf20 | orf13 | orf15 | orf16 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| orf2 | orf3 | orf8 | orf10 | orf17 | orf19 | orf12 | orf14 | rifJ |

(B)

| Proansamycin X | Rifamycin W | Rifamycin B |
| --- | --- | --- |
| hydroxylation of C-34a | acetylation |  |
| dehydrogenation | methylation |  |
|  | oxidative removal of C-34a |  |
|  | oxidative cleavage |  |
|  | quinone reduction |  |
|  | glycolate unit addition |  |
|  | connection of 6-O/C-12 |  |

Fig. 1. (A) Genetic organization and polyketide chain extension of rifamycin B biosynthesis and (B) modifications required to convert proansamycin X into rifamycin B. Asterisks indicate non-functional or inactive domains.
macrocyclic intermediate in rifamycin B biosynthesis (Fig. 1B) (August *et al.*, 1998). Further tailoring reactions of proansamycin X would first give rise to rifamycin W, which is known as an early central intermediate in biosynthesis and the precursor of a variety of other rifamycin derivatives (White *et al.*, 1974). The basic skeleton of rifamycin W is essentially different from that of rifamycin B. The former compound has an olefinic moiety in its backbone structure, while the latter has a ketal structure. Thus, bioconversion from rifamycin W to rifamycin B must involve a major polyketide backbone rearrangement, which includes an oxidative cleavage of the olefin moiety, an oxidative removal of C-34a and a reduction of the quinone. These are followed by the attachment of side chain moieties to C-4, C-25 and C-27 to complete the final decoration of the molecule. A recent study provides evidence that the acetylation of C-25 takes place prior to the methylation of C-27 (Xu *et al.*, 2003). Rif-Orf20, a homologue of the *M. tuberculosis* PapA5 protein, has been found to be responsible for the acetylation of 27-O-demethyl-25-O-desacylrifamycin SV (DMDARSV) to give 27-O-demethylrifamycin SV (DMRSV) (Xiong *et al.*, 2005), whereas Rif-Orf14, a methyltransferase, is responsible for the methylation of DMRSV to rifamycin SV (Xu *et al.*, 2003). Although the bioconversion from DMDARSV to rifamycin SV, and to some extent from rifamycin SV to rifamycin B, has been investigated, little is known about the details of other tailoring reactions and their corresponding enzymes, in particular, those from proansamycin X to DMDARSV. To remedy this, we have constructed a number of mutants of *Amycolatopsis mediterranei* S699 in which one or more genes located in the regulatory and post-PKS modification region are inactivated, and the mutants were analysed for their ability to produce secondary metabolites. The results of the present study provide evidence that implicates *rif-orf19* and *rif-orf5* in the formation of the naphthalene moiety and the unusual polyketide backbone rearrangement in rifamycin B biosynthesis, respectively.

### METHODS

#### Micro-organisms, culture conditions and vectors. The rifamycin B-producing strain, *A. mediterranei* S699, was obtained from Professor Giancarlo Lancini at the former Lepetit Laboratories, Geranzano, Italy. For sporulation it was grown at 30°C on agar plates containing YMG (yeast extract, malt extract and glucose) medium (Kim *et al.*, 1998). YMG liquid medium was used for both seed and production cultures. *Escherichia coli* XL-1 Blue (Stratagene) was used as host for subcloning. pBluescript II SK(−) (Stratagene) was used as vector for subcloning and gene inactivation. *A. mediterranei* transformants were selected with 70 µg hygromycin B ml⁻¹ in both solid and liquid media. *E. coli* strains were grown in LB media supplemented with carbenicillin (100 µg ml⁻¹) and/or hygromycin B (70 µg ml⁻¹) for selection of plasmids.

#### Instrumentation. Electrospray ionization mass spectrometry (ESI-MS) was carried out on a Bruker-Esquire ion trap mass spectrometer with electrospray, atmospheric pressure chemical ionization (APCI) or nanospray ionization sources, and high-resolution ESI-MS was carried out on a Fison VG Quattro II electrospray ionization mass spectrometer. Proton NMR spectra were recorded on a Bruker DRX-499 spectrometer with an SGI O2 computer. An ISF-4-V culture shaker (Adolf Kuhner AG) was used for the fermentation of *A. mediterranei* S699. Ultraviolet spectroscopy was carried out on a Hewlett Packard 8452A diode array spectrophotometer. For HPLC, a Beckman System Gold Programmable Solvent Module was used with a Beckman System Gold Diode Array Detector Module.

#### DNA manipulation. Routine genetic procedures such as genomic and plasmid DNA isolations, restriction endonuclease digestions, alkaline phosphatase treatments, DNA ligations, and other DNA manipulations were performed according to standard techniques (Kieser *et al.*, 2000; Sambrook *et al.*, 1989). DNA fragments were excised from agarose gels and residual agarose was removed with the QiaQuick Gel Extraction Kit (Qiagen). For Southern hybridizations, the genomic DNA was immobilized on a Hybond-N+ membrane (Amersham Pharmacia Biotech). Hybridization was performed at 68°C for 5 h using nick-translated [³²P]dCTP (PerkinElmer Life Sciences)-labelled DNA probes. Stringency washes were done with 0·1 × SSC at 68°C.

#### Plasmid DNA construction

Plasmids carrying disrupted or truncated genes used in the inactivation experiments were constructed as follows.

##### pTM0031AB. A 5·6 kb *FspI/Spel* fragment containing the region from part of *orf16* to part of the *rpoB* gene, the very end of the rifamycin B biosynthetic gene cluster, and a 6·8 kb *FspI/FspI* fragment containing the region from part of *rifN* to part of *orf5*, were individually cloned from the cosmid clone pFKN108 into the pBluescript II SK(−) vector at *SmaI/Spel* and *EcoRV* multiple cloning sites, respectively, providing plasmids pTM0031A and pTM0031B (Fig. 2). Combination of the blunt ends *FspI* with *SmaI* and *FspI* with *EcoRV* generated new recleavable sites *AclI* and *SfaNI* on pTM0031A and pTM0031B, respectively. Plasmid pTM0031B was then digested with *KpnI* and the resulting 4·5 kb *KpnI/KpnI* fragment was inserted into *KpnI*-digested pTM0031A to give pTM0031AB.

##### Disruption of *rif-orf3*. A 7·8 kb *EcoRV/BstXI* fragment of pFKN108 (Fig. 3) was subcloned into pBluescript II SK(−) at the *EcoRV/BstXI* sites. The resulting plasmid pTM4502 was then digested with *KpnI* to give a 8·2 kb, a 2·3 kb and a 0·2 kb fragment. The 8·2 kb and 2·3 kb fragments were recovered from agarose gel. The 8·2 kb fragment was dephosphorylated by alkaline phosphatase and then religated to the 2·3 kb fragment with T4 DNA ligase (NEB) so as to delete a 0·2 kb fragment within *rif-orf3* and give pTM45023.

##### Disruption of *rif-orf4*. pTM4502 was partially digested with *NotI* (0·01 unit µl⁻¹), and the resulting complementary single-stranded sticky ends were filled by treatment with DNA polymerase I, Klenow fragment (Gibco) and 1 µmol of each deoxynucleotide triphosphate (Gibco), to generate blunt ends at the cut site. The formed blunt-end linear plasmid was religated with T4 DNA ligase such that the translated reading frame was shifted to give pTM45024.

##### Disruption of *rif-orf5*. pTM4502 was digested with *ApaI* and then incubated with Klenow enzyme in the same way as *rif-orf4* to give pTM45025.

##### Disruption of *rif-orf9*. A 7·1 kb *BsiWI/BsiWI* DNA fragment from pFKN108 (Fig. 3) containing part of *rif-orf9, rif-orfs 10, 11, 17, 18* and part of *rif-orf19* was blunt-ended with Klenow enzyme and subcloned into pBluescript II SK(−) at the *EcoRV* site to give pTM0035. Deletion of a 3·3 kb *SphI/SphI* fragment of pTM0035 provided pTM0035S, which was then digested with *SmaI* to give a
J. Xu and others

Regulatory and post-PKS modification genes

![Diagram](attachment:diagram1.png)

Fig. 2. Construction of plasmid pTM0031ABH. All circular plasmids were derived from pBluescript II SK(−) containing the ampicillin resistance gene (bla).

3·1 kb and a 3·7 kb fragment. The 3·7 kb fragment, containing the vector and part of *rif-orf9*, was recovered from agarose gel and religated to give pTM0035S9.

Disruption of *rif-orf10*. pTM0035S was digested with *SacI/NcoI* to give a 6·4 kb and a 0·4 kb fragment. The 0·4 kb fragment (part of *orf10*) was recovered from the agarose gel, blunt-ended using Klenow enzyme and inserted into pBluescript II SK(−) at the *EcoRV* site to give pTM0035S10.

Disruption of *rif-orfs 11 and 18*. pTM0035 was partially digested with *MluI* (0·1 unit μl⁻¹) to produce linear DNA with a cutting site within *orf11*, or within *orf18*. The cut plasmid DNA was blunt-ended using Klenow enzyme and dNTP mixture, and then religated. The obtained plasmids had either *orf11* or *orf18* disrupted and could be discriminated by digesting the plasmids with *MluI* and *XbaI*. The one with a disrupted *orf11* was designated pTM00351, and that with a disrupted *orf18*, pTM00358.

Disruption of *rif-orf13*. A 3·4 kb *StuI/XmnI* DNA fragment from pFKN108 (Fig. 3) containing part of *rif-orf20*, *rif-orfs 12, 13, 14* and part of *rif-orf15* was cloned into pBluescript II SK(−) at the *EcoRV*

![Diagram](attachment:diagram2.png)

Fig. 3. Restriction map of part of the rifamycin biosynthetic gene cluster on pFKN108. Solid bars indicate DNA fragments cloned into pBluescript II SK(−) or pAMR1 vectors. Ap, *ApaI*; Bg, *BglII*; BH, *BamHI*; BW, *BsiWI*; BX, *BstXI*; EI, *EcoRI*; EV, *EcoRV*; Kp, *KpnI*; Mu, *MluI*; Nc, *Ncol*; Nt, *NotI*; Sm, *SmaI*; Sa, *ScaI*; Sc, *Scal*; Sp, *SphI*; St, *Stul*; Xm, *XmnI*.
site to give pTM1304. The plasmid was partially digested with Ncol (1 unit µl⁻¹) and the cut plasmid DNA was blunt-ended using Klenow enzyme and dNTP mixture, and then religated with T4 DNA ligase. The obtained plasmids had either orf12 or orf13 disrupted and could be discriminated by digesting the plasmids with NcoI/EcoNI or NcoI/XbaI. The desired construct was designated pTM1304K.

Disruption of rif-orf16. A 3·6 kb NotI/ApaI DNA fragment from pFKN108 (Fig. 3), containing part of rif-orf15, rif-orf16 and rifJ, was cloned into pBluescript II SK(−) at the NotI/ApaI sites to give pTM1601. The plasmid was digested with BamHI and the cut plasmid DNA was blunt-ended using Klenow enzyme and dNTP mixture, and then religated to give pTM1601K.

Disruption of rif-orf19. pTM1303 (Xu et al., 2003; Table 1) was digested with HindIII, StuI and EcoRI to produce a 3·2 kb and a 5·9 kb fragment. The 3·2 kb fragment containing rif-orf19 was inserted into pBluescript II SK(−) at the HindIII/StuI site to give pTM1901. The 5·9 kb fragment containing rif-orf14 was inserted into pBluescript II SK(−) at the StuI/EcoRI site to give pTM1401 (Xu et al., 2003). pTM1901 was then digested with BglII and incubated with Klenow enzyme as previously described to give pTM1901B.

Introduction of the hygromycin B resistance gene (hygᴿ). All constructed plasmids were marked for selection by hygᴿ by cloning a 1·7 kb KpnI/KpnI fragment from plasmid pIJ5607 (Yu et al., 2001) into the KpnI site of pTM0031AB, pTM45023, pTM45024, pTM45025, pTM0035S9, pTM0035S10, pTM00351, pTM00358, pTM1304K, pTM1601K and pTM1901B, resulting in pTM0031ABH, pTM45023H, pTM45024H, pTM45025H, pTM0035S9H, pTM0035S10H, pTM00351H, pTM00358H, pTM1304KH, pTM1601KH and pTM1901BH, respectively.

Construction of pTM1312. pTM1303 was digested with MluI to produce a 5·7 kb fragment containing part of rif-orf14, rif-orfs 13, 12, 20, 19, and part of rif-orf18. The fragment was inserted into pAMR1 at the MluI site to give pTM1312.

Electrotransformation of the plasmid DNAs. All plasmid DNAs were methylated with A. mediterranei S699 cell extract and S-adenosyl-L-methionine before use. Transformation of heat-denatured methylated plasmid DNAs into A. mediterranei S699 was carried out by electroporation at 12·5 kV cm⁻¹, and the transformants were immediately transferred to a 500 ml Erlenmeyer flask containing 50 ml YMG medium and incubated at 200 r.p.m. and 28 °C for 8–24 h. Single crossover mutants were selected by plating out the cells onto YMG plates containing hygromycin B, which were incubated at 30 °C for 7–14 days. The genotypes of mutant candidates were confirmed by Southern hybridization. Double crossover mutants were selected after three rounds of cultivation of a single crossover mutant in YMG liquid medium by replica plating on YMG agar plates with and without hygromycin B. Colonies that lost the antibiotic resistance were selected and genotypically verified by Southern hybridization as either mutants or revertants. The expected mutant strains were used for further analysis.

Isolation and structure elucidation of rifamycin intermediates. Spores of the mutant strain were initially grown in YMG medium for 3 days at 28 °C and 200 r.p.m. This seed culture was used to inoculate (10 %, v/v) twenty 500 ml Erlenmeyer flasks containing 50 ml of YMG medium. After incubation for 10 days under the same conditions, the cultures were centrifuged, the pooled supernatants acidified to pH 3 with 1 M HCl, and the metabolites extracted with 3 × 100 ml ethyl acetate. Rifamycin-related compounds

were purified consecutively by SiO₂ gravity column chromatography (step gradient ethyl acetate to ethyl acetate/acetone 9:1, v/v) and HPLC (YMC.PAK ODS C₁₈ 10 × 250 mm, MeOH/water 78:22, v/v).

Rifamycin W: ESI-MS m/z 654 (M-H)⁻. NMR: ¹H (CD₃OD, 499 MHz) δ (p.p.m.): 0·44 (3H, d, J=7 Hz, H-32), 0·72 (3H, d, J=7 Hz, H-33 or H-34), 0·92 (3H, d, J=7 Hz, H-31), 1·06 (3H, d, J=7 Hz, H-33 or H-34), 1·76–1·81 (2H, m, H-24 and H-26), 1·84–1·87 (1H, m, H-22), 1·95 (3H, s, H-14), 2·04 (3H, br s, H-13), 2·08 (3H, br s, H-30), 2·30–2·36 (1H, m, H-20), 2·59–2·64 (1H, m, H-28), 3·52 (1H, dd, J=2 and 10 Hz, H-25), 3·56 (1H, dd, J=7·5 and 11 Hz, H-34aa), 3·62 (1H, dd, J=6·5 and 11 Hz, H-34ab), 4·04 (1H, dd, J=1 and 10 Hz, H-23), 4·22 (1H, dd, J=1 and 10 Hz, H-21), 4·34 (1H, s, H-27)*, 6·11 (1H, dd, J=6·5 and 16 Hz, H-19), 6·25 (1H, d, J=11 Hz, H-17), 6·46 (1H, d, J=10 Hz, H-29), 6·54 (1H, dd, J=11 and 16 Hz, H-18), and 7·46 (1H, s, H-3). *J₂₆,₂₇ and J₂₇,₂₈ = 0 Hz (Nakata et al., 1990).

Treatment of A. mediterranei S699 with ancymidol. Ancymidol (12 mg) was added in two portions to cultures of wild-type strain S699 at 48 and 96 h after inoculation and the products were harvested after 7–10 days’ cultivation. Extraction, purification and structure elucidation of the products were carried out according to the procedures described above.

---

**RESULTS**

Delineation of the rifamycin post-PKS modification genes

The rifamycin biosynthetic gene cluster is organized into three regions: the PKS genes, the AHBA biosynthetic genes, and the post-PKS modification genes (August et al., 1998). To determine the extent of the involvement of genes located in the proposed post-PKS modification region of the rif cluster, we generated a mutant strain of A. mediterranei S699, in which a 21 kb KpnI/KpnI fragment downstream of the AHBA biosynthetic genes is deleted from the chromosome. The construction was initiated by the subcloning of two DNA fragments located upstream and downstream of the deleted region from the cosmid clone pFKN108 (August et al., 1998) into the pBluescript II SK(−) vector, providing plasmids pTM0031B and pTM0031A, respectively (Fig. 2). Plasmid pTM0031B was then digested with KpnI and the resulting 4·5 kb fragment was inserted into KpnI-digested pTM0031A, replacing the 0·9 kb KpnI/KpnI fragment of pTM0031A, to give pTM0031AB, which harbours the hybrid 9·2 kb DNA fragment (Fig. 2). The constructed plasmid pTM0031AB was marked for selection by hygᴿ (Yu et al., 2001), which was obtained from plasmid pIJ5607, and the product was transformed into A. mediterranei S699. Double homologous recombination within the 9·2 kb DNA fragment gave rise to a number of mutants and revertants (genotypically identical to the wild-type). As expected, the mutant strain (MT0031ABH) lacked the 21 kb DNA fragment as confirmed by Southern hybridization (data not shown). The deleted region was located downstream of genes encoding a putative regulator (rifO), an efflux protein (rifP) and a transcriptional repressor (rifQ), and upstream of the aminodehydroquinate dehydratase gene (riff), which is known to be involved in the biosynthesis of
Table 1. Plasmids and strains used in this study

| Plasmids          | Relevant characteristics                                                                                          | Reference or source               |
|-------------------|-----------------------------------------------------------------------------------------------------------------|----------------------------------|
| pFKN 108          | Cosmid clone of part of rif PKS genes, AHBA biosynthetic genes, and regulatory and post-PKS modification genes in vector pOJ 446 | August et al. (1998)             |
| pBluescript II SK(−) | ColE1-based phagemid vector with fl(−) and pUC origins; T3, T7 and lac promoters; bla | Stratagene                       |
| pTM0031A          | pBluescript II SK(−) derivative containing the 5·6kb FspI/SpeI fragment of pFKN 108 including part of orf16, rifJ, orf36, orf37 and part of rpoB genes | This study                      |
| pTM0031B          | pBluescript II SK(−) derivative containing the 6·8kb FspI/FspI fragment of pFKN 108 including part of rifN, rifO, orf2, rifP, rifQ, orf3, orf4 and part of orf5 genes | This study                      |
| pTM0031AB         | pTM0031A derivative, in which a 0·9kb KpnI/KpnI fragment was replaced with the 4·5kb KpnI/KpnI fragment of pTM0031B | This study                      |
| pTM0031ABH        | pTM0031AB+hyg<sup>R</sup>                                                                                     | This study                      |
| pTM0035           | pBluescript II SK(−) derivative containing the 7·1kb BsiWI/BsiWI fragment of pFKN 108 including part of orf9, orf10, orf11, orf17, orf18 and part of orf19 genes | This study                      |
| pTM0035S          | pTM 0035 derivative, in which a 3·3kb SphI/SphI fragment has been deleted                                          | This study                      |
| pTM0035S9         | pBluescript SK(−) derivative containing the 0·8kb BsiWI/SmaI fragment of orf9 gene                                | This study                      |
| pTM0035S9H        | pTM0035S9+hyg<sup>R</sup>                                                                                     | This study                      |
| pTM0035S10        | pBluescript II SK(−) derivative containing the 0·8kb SacI/NcoI fragment of orf10 gene                            | This study                      |
| pTM0035S10H       | pTM0035S10+hyg<sup>R</sup>                                                                                    | This study                      |
| pTM4502           | pBluescript II SK(−) derivative containing the 7·8kb EcoRV/BstXI fragment of pFKN 108 including part of rifP, rifQ, orf3, orf4, orf5, orf6 and part of orf7 genes | This study                      |
| pTM45023          | pTM 4502 derivative, in which the 0·2kb KpnI/KpnI fragment of orf3 has been deleted                               | This study                      |
| pTM45023H         | pTM45023+hyg<sup>R</sup>                                                                                      | This study                      |
| pTM45024          | pTM 4502 derivative, in which the NotI site on orf4 has been Klenow filled-in                                     | This study                      |
| pTM45024H         | pTM45024+hyg<sup>R</sup>                                                                                      | This study                      |
| pTM45025          | pTM 4502 derivative, in which the ApaI site on orf5 has been Klenow filled-in                                     | This study                      |
| PTM45025H         | pTM45025+hyg<sup>R</sup>                                                                                      | This study                      |
| pTM1303           | pBluescript II SK(−) derivative containing the 9·1kb ScaI/EcoRI fragment of pFKN 108 including part of orf18, orf19, orf20, orf12, orf13, orf14, orf15 and part of orf16 genes | Xu et al. (2003)                |
| pTM1304           | pBluescript II SK(−) derivative containing the 3·4kb StuI/XmnI fragment of pFKN 108 including part of orf20, orf12, orf13, orf14 and part of orf15 genes | This study                      |
| pTM1304K          | pTM 1304 derivative, in which the NcoI site on orf13 has been Klenow filled-in                                    | This study                      |
| pTM1304KH         | pTM1304K+hyg<sup>R</sup>                                                                                      | This study                      |
| pTM1601           | pBluescript II SK(−) derivative containing the 3·6kb NotI/ApaI fragment of pFKN 108 including part of orf15, orf16 and rifJ genes | This study                      |
| pTM1601K          | pTM 1601 derivative, in which the BamHI site on orf16 has been Klenow filled-in                                   | This study                      |
| pTM1601KH         | pTM1601K+hyg<sup>R</sup>                                                                                      | This study                      |
| pTM1901           | pTM 1303 derivative containing a 3·2kb HindIII/StuI fragment including part of orf18, orf19 and part of orf20 genes | This study                      |
| pTM1901B          | pTM 1901 derivative, in which the BglII site on orf19 has been Klenow filled-in                                   | This study                      |
| pTM1901BH         | pTM1901B+hyg<sup>R</sup>                                                                                      | This study                      |
| pIJ5607           | Plasmid containing hyg<sup>R</sup>                                                                            | Yu et al. (2001)                 |
| pAMR1             | pULVK 2 (Kumar et al., 1996) derivative, in which hyg<sup>R</sup> and a MCS have been introduced                  | C.R. Hutchinson, personal communication |
| pTM1312           | pAMR1 derivative containing the 5·7kb MluI/MluI fragment of pTM 1303 harbouring orf13, orf12, orf20 and orf19 genes | This study                      |

Table 1. cont.

| Plasmids | Relevant characteristics | Reference or source |
|----------|-------------------------|---------------------|
| Strains |  |  |
| E. coli |  |  |
| XL-1 Blue | recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 lac [F' proAB lacIq ZDM 15 Tn10(Tetr)] | Stratagene |
| Amycolatopsis mediterranei |  |  |
| S699 | Wild-type rifamycin B producing strain | Dr G. Lancini |
| MT0031ABH | 21kb post-PKS genes deleted mutant that produces SY4b and desacetyl-SY4b | This study |
| MT45023H | rif-orf3 knock-out mutant that produces 40% more rifamycin B than the wild-type | This study |
| MT45024H | rif-orf4 knock-out mutant that produces rifamycin B | This study |
| MT45025H | rif-orf5 knock-out mutant that produces rifamycin W | This study |
| MT0035S9H | rif-orf9 knock-out mutant that produces rifamycin B | This study |
| MT0035S10H | rif-orf10 knock-out mutant that produces rifamycin B | This study |
| MT00351H | rif-orf11 knock-out mutant that produces rifamycin B | This study |
| MT00358H | rif-orf18 knock-out mutant that produces rifamycin B | This study |
| MT1304KH | rif-orf13 knock-out mutant that produces rifamycin B | This study |
| MT1601KH | rif-orf16 knock-out mutant that produces rifamycins SV and B | This study |
| MT1901BH | rif-orf19 knock-out mutant that produces SY4b and desacetyl-SY4B | This study |
| MT1901BH/1312 | MT1901BH mutant that was transformed with pTM1312; produces rifamycin B | This study |

the starter unit AHBA (Yu et al., 2001). The mutant strain appeared to be morphologically different from the wild-type, with no apparent rifamycin production on the YMG agar plates. Analysis of the products by HPLC and ESI-MS revealed that the mutant (MT0031ABH) was unable to produce rifamycin B or to produce proansamycin X as one would have expected for mutants lacking the entire set of tailoring enzymes. Instead, the mutant strain produced the tetraketide products SY4b and desacetyl-SY4b (the cyclic forms of P8/l-OG, Fig. 4B), which have previously been shown to be premature release products when the amide synthase (RifF) is inactivated (Stratmann et al., 1999; Yu et al., 1999). Negative-ion ESI-MS tandem analysis of desacetyl-SY4b gave fragment ions at m/z 246, 228, 190, 172, 153, 136, which are consistent with the expected fragmentation patterns for desacetyl-SY4b (Fig. 4C). The results suggested that the gene(s) responsible for naphthalene ring formation might be located in the deleted region.

### Inactivation of the P450-dependent monooxygenases (Rif-Orfs 4, 5, 13 and 16)

There are four genes in the regulatory and post-PKS modification region that encode proteins with homology to the cytochrome P450 family. rif-orf4 encodes a protein homologous to the putative cytochrome P450 oxidoreductases from several Streptomyces species (Trower et al., 1992). rif-orfs 5 and 13 encode proteins similar to cytochrome P450 EryF involved in erythromycin biosynthesis (Weber et al., 1991), and rif-orf16 encodes a protein that shares homology with cytochrome P450 SubC from Streptomyces griseolus (Omer et al., 1990). To evaluate the roles of these genes in rifamycin B biosynthesis, all four cytochrome P450 genes

http://mic.sgmjournals.org

J. Xu and others

(A)

Abs.
1.00
WT
0.75
0.50
0.25
0
0   10    20    30    40    50 min
Rifamycin B
254 nm
320 nm

(B)

Abs.
1.00
MT0031AB
0.75
0.50
0.25
0
0   10    20    30    40    50 min
desacetyl
SY4b
(major)
+ 
OH
NH₂
O
OH
H
O
C
OH
NHAc
OH
H
O
C
(minor)
254 nm
320 nm

(C)

Intens.
2500
190.0
153
H
136
O
246
190
O
C
m/z 290
(negative ion)
109.3
136.2
153.1
172.1
-H₂O
228.0
246.0
-H₂O
75    100   125   150   175   200   225   250   275   300 m/z

Fig. 4. Products of the mutant strain MT0031ABH. (A) HPLC chromatogram of the wild-type (WT) fermentation products; (B) HPLC chromatogram of the mutant fermentation products; (C) ESI-MS/MS of the mutant product desacetyl-SY4b. Abs., absorbance; Intens., intensity.

obtained by mutagenesis of spores with nitroguanidine (White et al., 1974). Rifamycin W production was confirmed by mass spectrometry and 'H-NMR analysis. Thus we conclude that rif-orf5 is critical for the conversion of rifamycin W to DMDARSV during rifamycin B biosynthesis. However, detailed biochemical studies of the enzyme have not been achieved, as attempts to produce recombinant Rif-Orf5 in *E. coli* have so far been unsuccessful.

Inhibition of cytochrome P450 monooxygenases by ancymidol

To confirm the involvement of a cytochrome P450 monooxygenase in the conversion of rifamycin W, cultures of wild-type *A. mediterranei* S699 were treated with ancymidol, an inhibitor of cytochrome P450 monooxygenases (Ralston et al., 2001), and analysed for compound production. No apparent cell growth inhibition was detected in cultures with ancymidol, relative to those without ancymidol. However, cultures in the presence of ancymidol produced a dark brown colour, which turned to bright yellow when

the pH of the culture was adjusted to 2·5 with 0·1 M H₂SO₄, whereas those without ancymidol have a light brown colour, indicating a normal production of rifamycin B. TLC, HPLC and ESI-MS analyses of the products revealed that in the presence of ancymidol the bacteria accumulated rifamycin W, supporting the role of a cytochrome P450, presumably Rif-Orf5, in the conversion of rifamycin W to DMDARSV.

Inactivation of rif-orf3

Based on results of the present study and others (Stratmann et al., 1999; Yu et al., 1999), it was clear that the core naphthalene ring moiety of rifamycin is constructed during the polyketide chain extension process. However, the fact that the reaction may require a catalytic enzyme, whose gene is located in the post-PKS modification region and functions *in trans* with the PKS, is rather unusual. This notion has led us to a closer inspection of a functionally unknown gene (*rif-orf3*) that is located in the cluster directly upstream of the cytochrome P450-dependent monooxygenase genes

rif-orfs 4 and 5. This gene originally showed no similarity to any known proteins in the database (but recently has been found to share high homology with a number of hypothetical proteins predicted from genome sequences; see Discussion). *In silico* analysis of the amino acid sequence revealed the presence in the protein of a putative flavin-nucleotide-binding region (Marchler-Bauer *et al.*, 2003), which might be involved in a phenolic hydroxylation reaction, making this gene an intriguing candidate for gene inactivation experiments. Therefore, disruption of *rif-orf3* was implemented by an internal deletion of a 0·2 kb *KpnI/* *KpnI* fragment from the gene, and the mutant phenotype was analysed by HPLC and ESI-MS. The mutant gave an identical phenotypic pattern to that of the wild-type strain, but produced about 40% more rifamycin B than the wild-type, suggesting that *rif-orf3* is involved in the regulation of rifamycin B production rather than in the formation of the naphthalene ring.

### Involvement of Rif-Orf19 in naphthalene ring formation

As neither the P450 monooxygenases nor *rif-orf3* are directly involved in naphthalene ring formation, we then focused on *rif-orf19*, which encodes a protein homologous to 3-(3-hydroxyphenyl)propionate hydroxylases, a group of flavoproteins that catalyse the hydroxylation of phenol structures. This includes the 3-(3-hydroxyphenyl)propionic acid hydroxylases (HppA or MhpA) from *Rhodococcus globerulus* (32% identity, 46% similarity) (Barnes *et al.*, 1997), *E. coli* strain K-12 (35% identity, 51% similarity) (Blattner *et al.*, 1997) and *Comamonas testosteronei* TA441 (37% identity, 50% similarity) (Arai *et al.*, 1999). Genes similar to *rif-orf19* have also been reported to be involved in a number of secondary metabolite biosyntheses, such as in the biosyntheses of the antitumour antibiotic geldanamycin (*gdmM*; 46% identity and 59% similarity at the protein level) (Rascher *et al.*, 2003) and of the aromatic myxobacterial electron transport inhibitor stigmatellin (*stiL*; 27% identity, 41% similarity at the protein level) (Gaitatzis *et al.*, 2002). *In silico* analysis of the amino acid sequences revealed that all the proteins within this family contain typical nucleotide-binding motifs for flavoprotein hydroxylases with the amino acid sequences GxGxxG, DGxxSxxR and GDxxH (Fig. 5) (Enroth *et al.*, 1998; Eppink *et al.*, 1997).

To inactivate *rif-orf19*, a 3·2 kb *HindIII/StuI* DNA fragment derived from pTM1303 harbouring part of *rif-orf18*, *rif-orf19*, and part of *rif-orf20* was subcloned into vector pBluescript II SK(−) followed by introduction of *hyg*<sup>R</sup> to give plasmid pTM1901BH. Inactivation of *rif-orf19* was achieved by Klenow fill-in frameshift at a *BglII* site. Introduction of this mutation into the wild-type strain abolishes the ability of the micro-organism to produce rifamycin B, and as expected, the mutant produces desacetyl-SY4b, which is identical to the compound isolated from the 21 kb deletion mutant strain MT0031ABH.

---

### Complementation of *rif-orf19*

To confirm the direct involvement of *rif-orf19* in rifamycin B biosynthesis, a complementation experiment was carried out by introducing the wild-type *rif-orf19* gene into the mutant strains using a replicating vector derived from plasmid pULVK2 (Kumar *et al.*, 1994). Thus, a 5·7 kb *MluI/MluI* fragment of pTM1303 harbouring the complete operon of *rif-orfs* 13, 12, 20 and 19 was cloned into pAMR1, a pULVK2 (Kumar *et al.*, 1996) plasmid modified by the laboratory of Professor C. R. Hutchinson at the University of Wisconsin by introducing *hyg*<sup>R</sup> and a multiple cloning site (MCS). The resulting plasmid (pTM1312) was electrotransformed into mutant strain MT1901BH and the transformants were selected on YMG plates containing the antibiotic hygromycin. Positive transformants could be easily identified by the diffusion of red-brown colour around the colonies on agar plates, which was further analysed by ESI-MS to show the recovery of rifamycin B production in these transformants. To confirm their genetic make-up, recovery of the plasmids was attempted by ‘electroduction’ (Tuteja *et al.*, 2000) of the plasmid from the transformants into *E. coli* using procedures suggested by Lal and co-workers (Tuteja *et al.*, 2000); however, no plasmid could be recovered. On the other hand, Southern hybridization of the total cellular DNA using a 1·5 kb *BglII/HindIII* fragment of pTM1901 containing parts of *rif-orf18* and *rif-orf19* and a 1·7 kb fragment containing the *hyg*<sup>R</sup> gene as probes gave strong signals, albeit in DNA of sizes that were different from those predicted for either the free plasmid or for plasmid that has been accidentally incorporated into the chromosome. This could be due to rearrangement and/or deletion of DNA fragments of pTM1312 in *A. mediterranei*, as such events have previously been reported for pULVK2 derivatives (Tuteja *et al.*, 2000). While the restriction patterns could not be readily explained, the strong hybridization signals for both probes indicated the presence of both *rif-orf19* and *hyg*<sup>R</sup> genes in the cells. No signal could be detected in the Southern hybridization of wild-type and mutant MT1901BH DNA using the *hyg*<sup>R</sup> gene as a probe.

### Inactivation of *rif-orfs* 9, 10, 11 and 18

Finally, to determine if any of the following genes – *rif-orf9* (putative aminotransferase), *rif-orf10* (putative oxidoreductase), *rif-orf11* (putative reductase) and *rif-orf18* (putative 2,3-dehydratase) – are involved in rifamycin B biosynthesis, they were individually inactivated by either single or double crossover recombination. Mutants generated by these experiments were grown in YMG medium and the products analysed by LC-MS. All mutants produced rifamycin B in amounts comparable to the wild-type, indicating that those gene products are not involved in rifamycin B biosynthesis.

---

### DISCUSSION

The genetic organization of the rifamycin biosynthetic gene cluster comprises a set of PKS modules (*rifA–E*), an amide

J. Xu and others

| HmpA-E.coli | 1 MA---IQHPDIQPAVNHSVQVAIAGAGPVGLMMANYLGQMGIIDVLVVVEKLDKLIDYPRAT |
| --- | --- |
| MhpA-C.test | 1 ----MNAPQENQASQDDADVLITGAGPVGLTLANTLGMAGVRVIVAEKLPRIIDYPRAI |
| HppA-R.glob | 1 --------MKTTQDTTEVLIVGAGPAGLMLANILGMYGKQTVVLEAMDTLIDYPRGV |
| Stil-S.aur | 1 ------------MDVAIVGGGPVGLMAANLLGSYGVGVLVERESSPHTYRAV |
| GdmM-S.hyg | 1 MTGKEAAVDTARETDSLEAEVLIVGYGPVGQLLSVLLAQRRRTVVVERWPEPYRHPRAV |
| Orf19-A.med | 1 -------------LTDTDVVIVGNGPTIGATLSVLLAQRGWRVTVLERRPRPYKLPRAT |
| consensus | 1 . .*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*dNTP-hexose dehydratase (RfbH) of Salmonella enterica and aminotransferase (YokM) of Bacillus subtilis, respectively, results in mutants that produce rifamycin B, indicating that there is no direct involvement of these genes in rifamycin B biosynthesis (August *et al.*, 1998; M. Müller and others, unpublished results). These findings have been confirmed by the results of our experiments, as individual inactivation of *rif-orfs* 9, 10, 11, and 18 also does not affect rifamycin B production.

The first cyclization product released from the *rif* PKS is believed to be proansamycin X, a hypothetical molecule that has never been isolated or identified. It was expected that deletion of the entire set of post-PKS modification genes from the chromosome might give mutants that produce proansamycin X. However, mutant strain MT0031ABH, in which a 21 kb fragment of the post-PKS modification genes has been deleted, did not accumulate proansamycin X, but instead the tetraketides SY4b and desacetyl-SY4b were produced. These compounds have previously been isolated from knock-out mutants of the amide synthase RifF, as a result of premature release of the polyketide chain intermediates from the enzyme, starting from the tetraketide through the undecaketide (Stratmann *et al.*, 1999; Yu *et al.*, 1999). Structure identification of these intermediates showed that from the pentaketide onwards they contain a naphthalene ring, suggesting that the formation of the naphthalene moiety of rifamycin B takes place during the chain extension of the polyketide, specifically between the tetraketide and the pentaketide stage. Therefore, the production of desacetyl-SY4b by MT0031ABH indicates

that a gene(s) responsible for the formation of the naphthalene ring is located in the deleted region. However, attempts to complement the missing genes with a 28·1 kb EcoRV/SpeI fragment, harbouring all genes of the deleted region (Fig. 2), have not been successful. It has been proposed that the formation of the naphthalene ring would require the introduction of an oxygen into the aromatic ring at the position para to the hydroxyl group of AHBA, followed by quinone formation and cyclization by a Michael addition (Floss & Yu, 1999). The initial reaction might be catalysed by one of the cytochrome P450 monooxygenases (Fig. 6). However, individual inactivation of *rif-orfs* 4, 13 and 16 did not significantly affect rifamycin B production. Interestingly, the *rif-orf16* mutant produces both rifamycins SV and B, which raises a question whether *rif-orf16* may to some extent be involved in the formation and/or attachment of the glycolate side chain.

Another P450 gene in the cluster, *rif-orf5*, was shown to be involved in the modification of rifamycin W, a rifamycin analogue that has been proposed to be a progenitor of rifamycin B. White and co-workers experimentally showed that washed mycelium from a rifamycin B-producing strain of *A. mediterranei* transformed rifamycin W into rifamycin B (White *et al.*, 1974). As inactivation of *rif-orf5* resulted in mutants that produce rifamycin W, we propose that Rif-Orf5 functions in the oxidative cleavage of the C12–C29 double bond of rifamycin W, leading to a rearrangement reaction to form a ketal structure (Fig. 7). This is followed by an oxidation of the C-34a alcohol to the carboxylic acid and then a decarboxylation-dehydration

![Diagram](https://i.imgur.com/yourimageurl.png)

Fig. 6. Proposed mechanism for naphthalene ring formation by Rif-Orf19. The yellow box shows the hydroxylation of 3-(3-hydroxyphenyl)propionate to 3-(2,3-dihydroxyphenyl)propionate catalysed by 3-(3-hydroxyphenyl)propionate hydroxylase (HppA) from *R. globerulus*. Asterisks indicate non-functional or inactive domains.

http://mic.sgmjournals.org   2525
J. Xu and others

![image](attachment://chemical_structure_1.png)

Rifamycin W

![image](attachment://chemical_structure_2.png)

27-O-demethyl-25-O-desacetyl-rifamycin S (DMDARS)

Fig. 7. Proposed mechanism for the polyketide backbone rearrangement in rifamycin B biosynthesis. The P450 enzyme Rif-Orf5 may be involved in the oxidative cleavage of the olefinic moiety of rifamycin W.

to give DMDARS, which may be converted to its redox counterpart, DMDARSV, in the cells (Xu *et al.*, 2003). Alternative scenarios for the conversion of the C-34a alcohol into DMDARS could involve a retro-aldol cleavage-dehydration to release C34a as formaldehyde, or oxidation to the C-34a aldehyde and a retro-Claisen reaction-dehydration to liberate C-34a as formic acid. The C-34a alcohol of rifamycin W may be derived from the C-34a methyl group of proansamycin X by one of the P450-dependent monooxygenases. None of the P450 genes located in the post-PKS modification gene cluster seems to be responsible for this reaction. However, another P450 gene, *rif0*, located immediately upstream of *rifA*, could be responsible for the reaction. Mutational analysis of this gene by the group of Hutchinson did not give conclusive results (C. R. Hutchinson, personal communication). Alternatively, the P450 enzymes may have relaxed substrate specificity, so that when one of the enzymes is disrupted, its function will be complemented by the others.

Disruption of *rif-orf19* ceased production of rifamycin B and caused the accumulation of the tetraketides SY4b and desacetyl-SY4b, implicating *rif-orf19* in the formation of the naphthalene ring of rifamycin B. The use of a tailoring enzyme *in trans* for the formation of the naphthalene ring during polyketide chain extension is unprecedented in polyketide biosynthesis. To date, a number of unusual examples have been reported in PKS systems, in which the acyltransferase domain is located outside the PKS modules and functions *in trans* with the modular enzymes (Cheng *et al.*, 2003; Piel, 2002). However, there are no examples of an oxidation reaction that takes place during polyketide chain assembly nor of an oxidation domain located within a modular PKS system. This kind of reaction is more commonly seen in the NRPS systems, in which oxidation domains are usually located within NRPS modules (Du *et al.*, 2000; Weinig *et al.*, 2003). Nevertheless, based on the present results, it is proposed that Rif-Orf19 functions as a separate enzyme that interacts with the PKS, introducing a hydroxyl group into the acyl carrier protein (ACP)-bound tetraketide. This sets the stage for the further enzyme-catalysed or spontaneous cyclization reaction to form the naphthalene ring. Whether a single enzyme or a complex of enzymes is responsible for the entire process remains to be investigated. Failure of this cyclization to take place, as in the *rif-orf19* mutant, leads to rejection of the nascent ketide by module 4 of the PKS and release of the tetraketide.

The hydroxylation of ACP-bound tetraketide by an FAD-dependent hydroxylase also raises an important question regarding the nature of substrate access to the active site. Mechanistically, this class of enzymes requires the cofactors FAD and NADPH, molecular oxygen, and substrate. FAD is in general incapable of catalysing the hydroxyl transfer reaction free in solution; therefore, the catalytic site must be shielded from solvent during the critical step of the hydroxylation reaction. As the substrate is rather bulky and is bound to a large complex protein (PKS), it is less likely that Rif-Orf19 adopts a regular ‘open-closed’ channel mechanism for substrates and products to enter and leave the active site. The most attractive hypothesis would be that Rif-Orf19 docks onto the PKS and ‘engulfs’ the tetra-ketide portion of the ACP-bound substrate, whereas the PKS itself serves as a giant lid to the active site of the hydroxylase.
If this is the case, how and when NADPH and molecular oxygen enter the active site would be open questions. Interestingly, alignment analysis of the amino acid residues of Rif-Orf19 with those of related enzymes showed no significant disparity at the sequence levels (Fig. 5). Instead, all proteins seem to be highly similar, particularly Rif-Orf19 and GdmM (46 % identity, 59 % similarity); the latter is involved in geldamycin biosynthesis (Rascher *et al.*, 2003). However, there are a number of amino acid residues in Rif-Orf19 which are different from the conserved residues found in other hydroxylases (indicated by large arrows in Fig. 5) and which may play a role in its potentially distinct structural and/or catalytic nature. Detailed biochemical and structural investigations of the protein are critical to address these intriguing questions.

## ACKNOWLEDGEMENTS

The authors thank Pall Yoon, Marilynne Sarkissian and Jo Choi for their technical assistance, as well as Professor C. R. Hutchinson for providing plasmid pAMR1. Professor Giancarlo Lancini is thanked for providing *A. mediterranei* S699. This work was supported by NIH grant AI 20264 and funds from the Medical Research Foundation of Oregon.

## REFERENCES

Admiraal, S. J., Walsh, C. T. & Khosla, C. (2001). The loading module of rifamycin synthetase is an adenylation-thiolation didomain with substrate tolerance for substituted benzoates. *Biochemistry* 40, 6116–6123.

Admiraal, S. J., Khosla, C. & Walsh, C. T. (2002). The loading and initial elongation modules of rifamycin synthetase collaborate to produce mixed aryl ketide products. *Biochemistry* 41, 5313–5324.

Arai, H., Yamamoto, T., Ohishi, T., Shimizu, T., Nakata, T. & Kudo, T. (1999). Genetic organization and characteristics of the 3-(3-hydroxyphenyl)propionic acid degradation pathway of *Comamonas testosteroni* TA441. *Microbiology* 145, 2813–2820.

August, P. R., Tang, L., Yoon, Y. J. & 9 other authors (1998). Biosynthesis of the ansamycin antibiotic rifamycin: deductions from the molecular analysis of the rif biosynthetic gene cluster of *Amycolatopsis mediterranei* S699. *Chem Biol* 5, 69–79.

Barnes, M. R., Duetz, W. A. & Williams, P. A. (1997). A 3-(3-hydroxyphenyl)propionic acid catabolic pathway in *Rhodococcus globerulus* PWD1: cloning and characterization of the *hpp* operon. *J Bacteriol* 179, 6145–6153.

Blattner, F. R., Plunkett, G., 3rd, Bloch, C. A. & 14 other authors (1997). The complete genome sequence of *Escherichia coli* K-12. *Science* 277, 1453–1474.

Campbell, E. A., Korzhev, N., Mustaev, A., Murakami, K., Nair, S., Goldfarb, A. & Darst, S. A. (2001). Structural mechanism for rifampicin inhibition of bacterial RNA polymerase. *Cell* 104, 901–912.

Cheng, Y. Q., Tang, G. L. & Shen, B. (2003). Type I polyketide synthase requiring a discrete acyltransferase for polyketide biosynthesis. *Proc Natl Acad Sci U S A* 100, 3149–3154.

Cricchio, R., Lancini, G., Tamborini, G. & Sensi, P. (1974). Oximes of 3-formylrifamycin SV. Synthesis, antibacterial activity, and other biological properties. *J Med Chem* 17, 396–403.

Cricchio, R., Arioli, V. & Lancini, G. C. (1975). Hydrazones of 3-formylrifamycin SV. I – Hydrazones with *N*-amino-*N'*-substituted piperazines: synthesis, antibacterial activity and other biological properties. *Farmaco [Sci]* 30, 607–619.

Du, L., Sanchez, C., Chen, M., Edwards, D. J. & Shen, B. (2000). The biosynthetic gene cluster for the antitumor drug bleomycin from *Streptomyces verticillus* ATCC15003 supporting functional interactions between nonribosomal peptide synthetases and a polyketide synthase. *Chem Biol* 7, 623–642.

Dye, C., Williams, B. G., Espinal, M. A. & Raviglione, M. C. (2002). Erasing the world’s slow stain: strategies to beat multidrug-resistant tuberculosis. *Science* 295, 2042–2046.

Enroth, C., Neujahr, H., Schneider, G. & Lindqvist, Y. (1998). The crystal structure of phenol hydroxylase in complex with FAD and phenol provides evidence for a concerted conformational change in the enzyme and its cofactor during catalysis. *Structure* 6, 605–617.

Eppink, M. H., Schreuder, H. A. & Van Berkel, W. J. (1997). Identification of a novel conserved sequence motif in flavoprotein hydroxylases with a putative dual function in FAD/NAD(P)H binding. *Protein Sci* 6, 2454–2458.

Floss, H. G. & Yu, T.-W. (1999). Lessons from the rifamycin biosynthetic gene cluster. *Curr Opin Chem Biol* 3, 592–597.

Floss, H. G. & Yu, T.-W. (2005). Rifamycin – mode of action, resistance and biosynthesis. *Chem Rev* 105, 621–632.

Gaitatzis, N., Silakowski, B., Kunze, B., Nordsiek, G., Blocker, H., Hofle, G. & Muller, R. (2002). The biosynthesis of the aromatic myxobacterial electron transport inhibitor stigmatellin is directed by a novel type of modular polyketide synthase. *J Biol Chem* 277, 13082–13090.

Ghisalba, O. & Nuesch, J. (1981). A genetic approach to the biosynthesis of the rifamycin-chromophore in *Nocardia mediterranei*. IV. Identification of 3-amino-5-hydroxybenzoic acid as a direct precursor of the seven-carbon amino starter-unit. *J Antibiot* 34, 64–71.

Kieser, T., Bibb, M. J., Buttner, M. J., Chater, K. F. & Hopwood, D. A. (2000). *Practical Streptomyces Genetics*. Norwich: John Innes Foundation.

Kim, C. G., Yu, T.-W., Fryhle, C. B., Handa, S. & Floss, H. G. (1998). 3-Amino-5-hydroxybenzoic acid synthase, the terminal enzyme in the formation of the precursor of mC<sub>7</sub>N units in rifamycin and related antibiotics. *J Biol Chem* 273, 6030–6040.

Kirschbaum, T. M. & Gotte, R. F. (1993). Rifampicin resistance of Mu lysogenic strains by *rpoB* mutations. *Biol Chem Hoppe-Seyler* 374, 657–664.

Kishi, T., Harada, S., Asai, M., Muroi, M. & Mizuno, K. (1969). Tolypomycin. II. Structures of tolyposamine and tolypomycin Y. *Tetrahedron Lett* 2, 97–100.

Kishi, T., Yamana, H., Muroi, M., Harada, S. & Asai, M. (1972). Tolypomycin, a new antibiotic. 3. Isolation and characterization of tolypomycin Y. *J Antibiot* 25, 11–15.

Kumar, C. V., Coque, J. R. & Martin, J. F. (1994). Efficient transformation of the cephamycin C producer *Nocardia lactamdurans* and development of shuttle and promoter-probe cloning vectors. *Appl Environ Microbiol* 60, 4086–4093.

Kumar, V., de la Fuente, J. L., Leitao, A. L., Liras, P. & Martin, J. F. (1996). Effect of amplification or targeted disruption of the beta-lactamase gene of *Nocardia lactamdurans* on cephamycin biosynthesis. *Appl Microbiol Biotechnol* 45, 621–628.

Maggi, N., Pallanza, R. & Sensi, P. (1965). New derivatives of rifamycin SV. *Antimicrobial Agents Chemother* 5, 765–769.

Maggi, N., Pasqualucci, C. R., Ballotta, R. & Sensi, P. (1966). Rifampicin: a new orally active rifamycin. *Chemotherapy* 11, 285–292.
J. Xu and others

Marchler-Bauer, A., Anderson, J. B., DeWeese-Scott, C. & 24 other authors (2003). CDD: a curated Entrez database of conserved domain alignments. *Nucleic Acids Res* 31, 383–387.

Nakata, M., Akiyama, N., Kamata, J., Kojima, K., Masuda, H., Kinoshita, M. & Tatsuta, K. (1990). The total synthesis of rifamycin W. *Tetrahedron* 46, 4629–4652.

Omer, C. A., Lenstra, R., Litle, P. J., Dean, C., Tepperman, J. M., Leto, K. J., Romesser, J. A. & O'Keefe, D. P. (1990). Genes for two herbicide-inducible cytochromes P-450 from *Streptomyces griseolus*. *J Bacteriol* 172, 3335–3345.

Piel, J. (2002). A polyketide synthase-peptide synthetase gene cluster from an uncultured bacterial symbiont of *Paederus beetles*. *Proc Natl Acad Sci U S A* 99, 14002–14007.

Ralston, L., Kwon, S. T., Schoenbeck, M., Ralston, J., Schenk, D. J., Coates, R. M. & Chappell, J. (2001). Cloning, heterologous expression, and functional characterization of 5-epi-aristolochene-1,3-dihydroxylase from tobacco (*Nicotiana tabacum*). *Arch Biochem Biophys* 393, 222–235.

Ramos-e-Silva, M. & Rebello, P. F. (2001). Leprosy. Recognition and treatment. *Am J Clin Dermatol* 2, 203–211.

Rascher, A., Hu, Z., Viswanathan, N., Schirmer, A., Reid, R., Nierman, W. C., Lewis, M. & Hutchinson, C. R. (2003). Cloning and characterization of a gene cluster for geldanamycin production in *Streptomyces hygroscopicus* NRRL 3602. *FEMS Microbiol Lett* 218, 223–230.

Sambrook, J., Fritsch, E. F. & Maniatis, T. (1989). *Molecular Cloning: a Laboratory Manual*, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Schupp, T., Toupet, C., Engel, N. & Goff, S. (1998). Cloning and sequence analysis of the putative rifamycin polyketide synthase gene cluster from *Amycolatopsis mediterranei*. *FEMS Microbiol Lett* 159, 201–207.

Sepkowitz, K. A., Raffalli, J., Riley, L., Kiehn, T. E. & Armstrong, D. (1995). Tuberculosis in the AIDS era. *Clin Microbiol Rev* 8, 180–199.

Stratmann, A., Toupet, C., Schilling, W., Traber, R., Oberer, L. & Schupp, T. (1999). Intermediates of rifamycin polyketide synthase produced by an *Amycolatopsis mediterranei* mutant with inactivated *rifF* gene. *Microbiology* 145, 3365–3375.

Suzuki, Y., Katsukawa, C., Inoue, K., Yin, Y., Tasaka, H., Ueba, N. & Makino, M. (1995). Mutations in *rpoB* gene of rifampicin resistant clinical isolates of *Mycobacterium tuberculosis* in Japan. *Kansenshogaku Zasshi* 69, 413–419.

Trower, M. K., Lenstra, R., Omer, C., Buchholz, S. E. & Sariaslani, F. S. (1992). Cloning, nucleotide sequence determination and expression of the genes encoding cytochrome P-450soy (*soyC*) and ferredoxinsoy (*soyB*) from *Streptomyces griseus*. *Mol Microbiol* 6, 2125–2134.

Tuteja, D., Dua, M., Khanna, R., Dhingra, N., Khanna, M., Kaur, H., Saxena, D. M. & Lal, R. (2000). The importance of homologous recombination in the generation of large deletions in hybrid plasmids in *Amycolatopsis mediterranei*. *Plasmid* 43, 1–11.

Weber, J. M., Leung, J. O., Swanson, S. J., Idler, K. B. & McAlpine, J. B. (1991). An erythromycin derivative produced by targeted gene disruption in *Saccharopolyspora erythraea*. *Science* 252, 114–117.

Wehrli, W. & Staehelin, M. (1969). The rifamycins – relation of chemical structure and action on RNA polymerase. *Biochim Biophys Acta* 182, 24–29.

Wehrli, W., Zimmermann, W., Kump, W., Tosch, W., Vischer, W. & Zak, O. (1987). CGP 4832, a new semisynthetic rifamycin derivative highly active against some gram-negative bacteria. *J Antibiot (Tokyo)* 40, 1733–1739.

Weinig, S., Hecht, H. J., Mahmud, T. & Muller, R. (2003). Melithiazol biosynthesis: further insights into myxobacterial PKS/NRPS systems and evidence for a new subclass of methyl transferases. *Chem Biol* 10, 939–952.

White, R. J., Martinelli, E. & Lancini, G. (1974). Ansamycin biogenesis: studies on a novel rifamycin isolated from a mutant strain of *Nocardia mediterranei*. *Proc Natl Acad Sci U S A* 71, 3260–3264.

Xiong, Y., Wu, X. & Mahmud, T. (2005). A homolog of *Mycobacterium tuberculosis* PapA5 protein, Rif-Orf20, is an acetyltransferase involved in the biosynthesis of antitubercular drug rifamycin B by *Amycolatopsis mediterranei* S699. *ChemBioChem* 6, 834–837.

Xu, J., Mahmud, T. & Floss, H. G. (2003). Isolation and characterization of 27-O-demethylrifamycin SV methyltransferase provides new insights into the post-PKS modification steps during the biosynthesis of the antitubercular drug rifamycin B by *Amycolatopsis mediterranei* S699. *Arch Biochem Biophys* 411, 277–288.

Yu, T.-W., Shen, Y., Doi-Katayama, Y., Tang, L., Park, C., Moore, B. S., Richard Hutchinson, C. & Floss, H. G. (1999). Direct evidence that the rifamycin polyketide synthase assembles polyketide chains processively. *Proc Natl Acad Sci U S A* 96, 9051–9056.

Yu, T.-W., Muller, R., Muller, M., Zhang, X., Draeger, G., Kim, C. G., Leistner, E. & Floss, H. G. (2001). Mutational analysis and reconstituted expression of the biosynthetic genes involved in the formation of 3-amino-5-hydroxybenzoic acid, the starter unit of rifamycin biosynthesis in *Amycolatopsis mediterranei* S699. *J Biol Chem* 276, 12546–12555.
