
APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Aug. 2005, p. 4437–4445
0099-2240/05/$08.00+0 doi:10.1128/AEM.71.8.4437–4445.2005
Copyright © 2005, American Society for Microbiology. All Rights Reserved.

**Allophanate Hydrolase, Not Urease, Functions in Bacterial Cyanuric Acid Metabolism**

Gang Cheng,²,³,⁴ Nir Shapir,¹,²,³ Michael J. Sadowsky,²,³,⁴ and Lawrence P. Wackett¹,²,³*

Department of Biochemistry, Molecular Biology, and Biophysics,¹ BioTechnology Institute,² Center for Microbial and Plant Genomics,³ and Department of Soil, Water, and Climate,⁴ University of Minnesota, St. Paul, Minnesota 55108

Received 12 January 2005/Accepted 3 March 2005

Growth substrates containing an *s*-triazine ring are typically metabolized by bacteria to liberate 3 mol of ammonia via the intermediate cyanuric acid. Over a 25-year period, a number of original research papers and reviews have stated that cyanuric acid is metabolized in two steps to the 2-nitrogen intermediate urea. In the present study, allophanate, not urea, was shown to be the 2-nitrogen intermediate in cyanuric acid metabolism in all the bacteria examined. Six different experimental results supported this conclusion: (i) synthetic allophanate was shown to readily decarboxylate to form urea under acidic extraction and chromatography conditions used in previous studies; (ii) alkaline extraction methods were used to stabilize and detect allophanate in bacteria actively metabolizing cyanuric acid; (iii) the kinetic course of allophanate formation and disappearance was consistent with its being an intermediate in cyanuric acid metabolism, and no urea was observed in those experiments; (iv) protein extracts from cells grown on cyanuric acid contained allophanate hydrolase activity; (v) genes encoding the enzymes AtzE and AtzF, which produce and hydrolyze allophanate, respectively, were found in several cyanuric acid-metabolizing bacteria; and (vi) TrzF, an AtzF homolog found in *Enterobacter cloacae* strain 99, was cloned, expressed in *Escherichia coli*, and shown to have allophanate hydrolase activity. In addition, we have observed that there are a large number of genes homologous to *atzF* and *trzF* distributed in phylogenetically distinct bacteria. In total, the data indicate that *s*-triazine metabolism in a broad class of bacteria proceeds through allophanate via allophanate hydrolase, rather than through urea using urease.

The microbial metabolism of *s*-triazine herbicides and the resin building block melamine proceeds through cyanuric acid as an intermediate (6, 11, 21, 25, 30 31, 35, 36, 38, 39, 40, 44). Cyanuric acid is degraded by a large number of soil bacteria and fungi (3, 8, 12, 13, 20, 23, 24, 29, 32, 45). Cyanuric acid metabolism was concluded to occur via biuret and urea (Fig. 1, top) in *Enterobacter cloacae* (formerly *Klebsiella pneumoniae*) strain 99, *Acidovorax avenae* subsp. *citrulli* (also known as strain NRRLB-12227 and *Pseudomonas* strain A), and *Pseudomonas huttonis* NRRLB-12228 (Table 1). The proposed metabolic pathway was logical; urea was detected in protein extracts incubated with cyanuric acid, and urea was hydrolyzed to ammonia in the same extracts (4, 5). In further studies, biuret and urea were detected in culture extracts from unique transposon mutants of *Acidovorax avenae* subsp. *citrulli* incubated with melamine (10). The same pathway (Fig. 1, upper pathway) has also been ascribed to the cyanuric acid-metabolizing *Rhodococcus corallinus* strain 11 and an unidentified bacterium designated C3E (3). In subsequent years, the cyanuric acid pathway shown in Fig. 1 (upper pathway) has been attributed to other bacteria that grow on melamine or *s*-triazine herbicides (12, 30, 34, 36).

In 2001, an alternative pathway for cyanuric acid metabolism was inferred for *Pseudomonas* sp. strain ADP (Fig. 1, lower pathway) based on the expression of recombinant *Pseudomonas* sp. strain ADP proteins in *Escherichia coli* (27). The hydrolytic ring opening of cyanuric acid to produce biuret was identified, but crude protein extracts from recombinant *E. coli* containing *atzE* from *Pseudomonas* sp. strain ADP hydrolyzed biuret to allophanate (*N*-carboxyurea). Allophanate was degraded to carbon dioxide and ammonia in cell extracts from recombinant *E. coli* expressing AtzF. Urea was detected only in trace amounts.

In this context, the goal of the present study was severalfold: (i) to determine if *Pseudomonas* sp. strain ADP metabolizes cyanuric acid via allophanate in vivo, (ii) to determine if other cyanuric acid-metabolizing bacteria generate allophanate, and (iii) to discern if cyanuric acid metabolism might generate both allophanate and urea, or produce one intermediate uniquely. The favored hypothesis, based on our previous work with *Pseudomonas* sp. strain ADP and our observations on the chemical instability of allophanate, is that bacteria generate biuret and allophanate exclusively during cyanuric acid metabolism (Fig. 1, lower pathway). To test this hypothesis, metabolic trapping experiments, enzyme assays, gene cloning, and recombinant enzyme expression studies were performed. The data indicated that allophanate was produced as a metabolic intermediate in the metabolism of cyanuric acid by all of the *s*-triazine-metabolizing bacteria tested, and there was no evidence for the occurrence of an alternative pathway producing urea. Additionally, sequence analysis suggests that allophanate hydrolase is widely distributed in phylogenetically divergent bacteria.

---

*MATERIALS AND METHODS*

Bacteria and growth media. Bacteria used in these studies are shown in Table 1. Bacteria were grown in minimal R medium (9) containing 13.9 mM glucose as the sole carbon source and either 5 mM cyanuric acid, 7.5 mM urea, or 15 mM ammonium chloride as the sole nitrogen source. In solid media, cyanuric acid (12 g/l) was used as the sole nitrogen source.

2N Intermediates

$\begin{array}{ccccc}
 & \mathrm{H}_{2} \mathrm{~N}-\mathrm{C}-\mathrm{NH}_{2} \\
 & | \\
 & \mathrm{O} \\
 & \text { Urea } \\
 & \text { Allophanate }
\end{array}$

FIG. 1. Proposed metabolic pathways for the degradation of cyanuric acid in bacteria. The lower pathway is that found in *Pseudomonas* sp. strain ADP and operates via enzymes encoded by the *atzDEF* operon.

Chemicals. Cyanuric acid (98% purity) was purchased from Acros Organics (Morris Plains, NJ). Biuret (99% purity) was obtained from Fluka (Milwaukee, WI), and ultrapure urea (99% purity) was obtained from Invitrogen-Gibco-BRL (Carlsbad, CA). [UL-<sup>14</sup>C]cyanuric acid (radiochemical purity, 97.3%; specific activity, 12.2 μCi/mg) was generously provided by Syngenta Crop Protection (Greensboro, NC). Potassium allophanate was prepared by hydrolyzing ethyl allophanate (Fisher Scientific, Pittsburgh, Pa.) as described by Whitney and Cooper (43).

DNA methods. DNA was sequenced at the University of Minnesota Advanced Genetic Analysis Center (St. Paul, MN). Primers for sequencing nearly full-length homologs of the *atzE* and *atzF* genes in bacterial strains were developed from those genes in *Pseudomonas* sp. strain ADP (GenBank accession no. U66917). The following primers were used: for *atzE*, *atzEf* (5′-GGTATCGCC TCTGGCAGAAC-3′) and *atzEr* (5′-GGCGATACCGGGTGCTTGT-3′); for *atzF*, *atzFf* (5′-AAGATCTGGTCGAGTCAC-3′) and *atzFr* (5′-TATTGAGCC GCGAGGTATGC-3′). Additional sequence data for *atzF* were obtained using primer *atzFf2* (5′-AGCGTCCTCGCCCATA C-3′). The junction between the *atzD-atzE* and the *atzE-atzF* genes was sequenced following PCR amplification of the 4.5-kb operon using primer pair *atzDf* (5′-CCGAATCCCTTGCCACAG- 3′)-*atzFr* (5′-CGATGAAAGTGACGCCAAAT-3′). The junction between the *atzD* and *atzE* genes was sequenced using primer *atzDEf* (5′-CATTGGCCTCGG TAGTTGGG-3′), and the junction between the *atzE* and *atzF* genes was sequenced using primers *atzEFf* (5′-TTCTCACTCAGCCGGTCTCCTTC-3′) and *atzFER* (5′-GCAACGCTTCTCCCTTGTCCTT-3′). The *atzD* and *trzD* genes were amplified using a specific set of primers as described by Fruchey et al. (13).

HPLC analyses. Urea, biuret, allophanate, and cyanuric acid were separated by high-performance liquid chromatography (HPLC) using a Hewlett-Packard HP 1100 system equipped with a photodiode array detector and interfaced to an HP ChemStation. A Waters IC-Pak A HC anion-exchange column (150 by 4.6 mm; Waters Corp., Milford, MA) was used with an isocratic mobile phase consisting of 5 mM sodium phosphate buffer, pH 8.0, at a flow rate of 0.5 ml per min and was monitored at 194 nm. Under these conditions, the retention times of urea, biuret, allophanate, and cyanuric acid were 8, 10.5, 18.5, and 22.5 min, respectively. The concentrations of biuret, allophanate, and cyanuric acid in samples were quantified by integrating peak areas compared to synthetic standards.

Allophanate stability assays. The stability of 3 mM potassium allophanate in 10, 50, or 100 mM sodium phosphate buffer, pH 8.0, or 100 mM sodium phosphate buffer, pH 7.3, was investigated. Samples were incubated at 25°C for various times and subjected to HPLC as described above. The stability of allo-

TABLE 1. Strains used in these studies and genes involved in cyanuric acid catabolism

| Strain<sup>a</sup> | Original growth substrate | Presence of genes<sup>b</sup> | Previous intermediate identified | Reference |
| --- | --- | --- | --- | --- |
| *Pseudomonas* sp. strain ADP | Atrazine | trzD | + | Allophanate | 1 |
|  |  | atzD | + (100) |  |  |
|  |  | atzE | + (100) |  |  |
|  |  | atzF | + (100) |  |  |
| *Ralstonia pickettii* strain D | Atrazine | – | + | None | 2 |
|  |  | atzD | + (100) |  |  |
|  |  | atzE | + (100)<sup>c</sup> |  |  |
|  |  | atzF | + (100)<sup>c</sup> |  |  |
| *Agrobacterium radiobacter* J14a | Atrazine | – | + | None | 3 |
|  |  | atzD | + (100) |  |  |
|  |  | atzE | + (100) |  |  |
|  |  | atzF | + (100)<sup>c</sup> |  |  |
| *Ralstonia basilensis* M91-3 | Atrazine | + | – | None | 4 |
| (*Ralstonia* sp. strain M91-3) |  | atzD | – |  |  |
|  |  | atzE | – |  |  |
|  |  | atzF | – |  |  |
| *Alcaligenes* sp. strain SG1 | Atrazine | + | – | None | 5 |
|  |  | atzD | – |  |  |
|  |  | atzE | – |  |  |
|  |  | atzF | – |  |  |
| *Pseudomonas huttoniensis* | Ammeline | + | – | Urea | 6 |
| NRRLB-12228 |  | atzD | – |  |  |
| (*Pseudomonas* sp. strain NRRLB-12228, |  | atzE | – |  |  |
| *Pseudomonas* sp. strain D) |  | atzF | – |  |  |
| *Acidovorax avenae* subsp. | N-Isopropylammelide | + | – | Urea | 6 |
| citrulli NRRLB-12227 |  | atzD | – |  |  |
| (*Pseudomonas* sp. strain NRRLB-12227, |  | atzE | – |  |  |
| *Pseudomonas* sp. strain A) |  | atzF | – |  |  |
| *Enterobacter cloacae* strain 99 | Cyanuric acid | + | – | Urea | 6 |
| (*Klebsiella pneumoniae* |  | atzD | – |  |  |
| strain 99) |  | atzE | – |  |  |
|  |  | atzF | – |  |  |

<sup>a</sup> Names in parentheses refer to previously used strain designations.  
<sup>b</sup> As determined by PCR. +, presence; –, absence. Values in parentheses refer to percent amino acid sequence identity as determined by BLASTP.  
<sup>c</sup> Amino acid identity determined from 1,797 bp of 1,818-bp gene region sequenced.

phanate under acid conditions was investigated by adding 44 μl of cold perchloric acid to 1 ml of 3 mM allophanate in 7.3 mM sodium phosphate buffer, initial pH 7.3. The precipitate was removed by centrifugation at 14,000 × g at 4°C, the supernatant was neutralized by addition of 0.075 ml of 5 M KOH, and the resulting solution was centrifuged at 20,000 × g for 10 min at 4°C. The final supernatant fluid was filtered through a 0.2-μm polytetrafluoroethylene (PTFE) membrane filter (Gelman, Ann Arbor, MI) and analyzed by HPLC as described above. The concentration of allophanate remaining in solution was quantified by integrating the peak area. The stability of allophanate in the thin-layer chromatography (TLC) mobile phase previously used by Cook et al. (5) was determined. Potassium allophanate was added to ethyl acetate–acetic acid–methanol (5:1:1:1 by volume) at 25°C to make 50 mM. The solution was analyzed for allophanate by HPLC as described above.

Whole-cell metabolic studies using *Pseudomonas* sp. strain ADP. The ability of whole cells of *Pseudomonas* sp. strain ADP to transform cyanuric acid to allophanate was investigated. *Pseudomonas* sp. strain ADP was grown in R medium containing 500 μg cyanuric acid per ml as the sole N source. Cells were harvested at mid-log phase by centrifugation at 7,000 × g for 10 min at 4°C, and the pellet was washed twice in 10 mM sodium phosphate buffer, pH 8.0, and resuspended in the same buffer to a final optical density at 600 nm (*OD*~600~) of 1.5. Malonic acid and cyanuric acid were added to final concentrations of 10 mM and 3 mM, respectively, and the reaction mixture was incubated at 30°C. At different time points, 3-ml samples of the reaction mixture were treated with 12 μl of 3 mM NaOH, and cells were broken by four cycles of sonication for 1 min each. Cell debris was removed by centrifugation at 20,000 × g for 10 min at 4°C, and the supernatants were filtered through a 0.2-μm PTFE filter (Gelman, Ann Arbor, MI) and analyzed by HPLC as described above.

In vitro transformation of cyanuric acid to allophanate by *Pseudomonas huttiensis* NRRLB-12228 and *Pseudomonas* sp. strain ADP. *Pseudomonas huttiensis* NRRLB-12228 and *Pseudomonas* sp. strain ADP were grown in minimal R medium containing 5 mM cyanuric acid and were harvested at mid-log phase by centrifugation at 7,000 × g for 10 min at 4°C. Cell pellets were washed twice and resuspended in 10 mM sodium phosphate buffer, pH 8.0, cells were broken by sonication (four cycles of 1 min each), and cell extracts were prepared by centrifugation at 20,000 × g for 30 min at 4°C. Reaction mixtures contained 0.5 mM [UL-¹⁴C]cyanuric acid (3,177,120 cpm), 500 μg of crude protein per ml, and 10 mM sodium phosphate buffer, pH 8.0. Reaction mixtures were incubated at 25°C, 3-ml samples were taken at various times, and the reactions were stopped by the addition of 12 μl 3 mM NaOH. Proteins were precipitated by centrifugation at 14,000 × g for 10 min, and supernatants were passed through a 0.2-μm PTFE HPLC filter and analyzed by HPLC as described above. Fractions from the HPLC were collected at 20-s intervals into 15-ml scintillation vials containing 5 ml Ecolume (ICN). Samples were mixed and were kept in the dark overnight, and radioactivity in fractions was measured by using a Beckman (Fullerton, CA) LS 3801 liquid scintillation system.

Enzyme assays. Allophanate hydrolase and urease activities were determined by measuring the release of ammonia (42). *Enterobacter cloacae* strain 99, *Pseudomonas huttiensis* NRRLB-12228, and *Ralstonia pickettii* strain D were grown to mid-log phase at 30°C in minimal R medium containing 5 mM cyanuric acid, 7.5 mM urea, or 15 mM ammonium chloride as the sole nitrogen source. Crude cell protein extracts were prepared as described above. Reaction mixtures contained 200 μg of crude protein per ml and either 10 mM urea or 10 mM potassium allophanate in 5 mM sodium phosphate buffer, pH 8.0. Reaction mixtures were incubated at 30°C, and reactions were terminated by addition of 0.5 M H₂SO₄. Samples were taken at several time points within 60 min, and the ammonium ion concentration was measured as described previously (42).

Construction of a recombinant plasmid for the expression of TrzF. The *trzF* gene from *Enterobacter cloacae* strain 99 (formerly *Klebsiella pneumoniae* 99) was amplified, without its native promoter, by PCR using primers trzF-Topo86f (5′-CACCGGATATAACTGCAATATG-3′) and trzF-Topo1916r (5′-GCTATT GATCGGCAAGATATG-3′). The forward primer contained CACC at the 5′ end to facilitate directional cloning into topoisomerase I-based cloning sites. The *trzF* gene was cloned as a fusion protein with thioredoxin, bridged by an enterokinase protease cleaving site, in vector pBAD102/D-TOPO (Invitrogen, Carlsbad, CA). The plasmid was transformed into *E. coli* OneShot TOPO10 chemically competent cells (Invitrogen, Carlsbad, CA), and its sequence was verified.

Expression of TrzF in crude cell extracts. *E. coli* OneShot TOPO10 (pBAD102/D-TOPO::trzF) was grown in LB medium (33) containing 50 μg of ampicillin per ml at 25°C, with shaking at 150 rpm. When the culture reached an *OD*~600~ of 0.5, L-arabinose was added to a final concentration of 0.002%, and the induced cells were grown for an additional 8 h under the same conditions. Cultures were centrifuged at 10,000 × g for 10 min at 4°C and washed three times

FIG. 2. Instability of synthetic allophanate under differing pH and buffer conditions. Buffers used were 10 (∇), 50 (○), or 100 (▽) mM sodium phosphate buffer, pH 8.0, or 100 mM sodium phosphate buffer, pH 7.3 (●). Allophanate solutions were sampled at the indicated time points and analyzed by HPLC using conditions under which the remaining allophanate is stable.

in 0.85% NaCl, and cell pellets (2.5 g [wet weight]) were resuspended in 25 ml 0.5 M NaCl in 20 mM sodium phosphate buffer, pH 7.4. The cell suspension was passed four times through a chilled French pressure cell operated at 140 MPa, and the crude cell lysates were cleared by centrifugation at 18,000 × g for 90 min at 4°C. The activity of TrzF was determined as described above, except that the crude cell extract was incubated for 20 min at 37°C.

RESULTS

Chemical instability of allophanate. In earlier studies, urea was identified in cell extracts of cyanuric acid-metabolizing strains NRRLB-12227 and NRRLB-12228 by colorimetric assay, TLC, and HPLC (5, 10). However, in those studies, acetic acid was used in the developing solvent for TLC, and metabolite extracts were acidified with 0.5 N perchloric acid prior to analysis by HPLC. Both of these conditions are likely to cause allophanate to undergo a quantitative decarboxylation to urea, based on previous observations (26).

Studies conducted here using a synthetic standard showed that allophanate readily underwent decarboxylation to form urea at neutral and acidic pHs and in moderate to high buffer concentrations. Figure 2 shows the nonenzymatic decomposition of allophanate under different conditions. In 10 mM phosphate buffer at pH 8.0, allophanate showed a half-life of 50 h. However, with increasing buffer concentrations, the half-life decreased significantly to 17 h, and at pH 7.3, the half-life decreased to 3 h. When solutions of allophanate were treated with 0.5 N perchloric acid, a treatment previously used during sample preparation for cyanuric acid metabolites (5, 16), allophanate could no longer be detected immediately following the treatment; thus, the kinetics of allophanate decay could not be measured, but we predict a half-life of seconds to minutes. When synthetic allophanate was dissolved in the TLC developing solvent containing acetic acid that was used previously for thin-layer chromatography (5), allophanate could not be detected 5 min later. In the allophanate decomposition experiments conducted in this study, HPLC analysis showed that urea formed concomitantly with allophanate decomposition.

These data suggested that it would be difficult to distinguish

led to the detectable accumulation of allophanate (data not shown).

Other *s*-triazine-degrading bacteria contain identical *atzE* and *atzF* genes linked to a cyanuric acid hydrolase gene. The bacteria used in this study have been isolated by different groups on different *s*-triazine ring compounds, but all grow on cyanuric acid as their sole nitrogen source (Table 1). First, PCR was used to determine if these bacteria contained genes with significant DNA sequence identity to *atzE*. *Pseudomonas* sp. strain ADP, *Ralstonia pickettii* strain D, and *Agrobacterium radiobacter* strain J14a were found to contain a homolog to *atzE* (Table 1), suggesting that those strains also contain an enzyme that cleaves the terminal carbon-nitrogen bond of biuret to yield allophanate. New primers were designed to amplify nearly the entire *atzE* genes from *R. pickettii* strain D and *A. radiobacter* J14a, and the PCR-amplified DNA was subjected to sequencing. The sequences of the *atzE* genes from both strains were found to be 100% identical to the *atzE* gene from *Pseudomonas* sp. strain ADP (Table 1), strongly suggesting that the biuret hydrolase in all three strains makes the identical product, allophanate.

To follow up on that observation, PCR was used to amplify an *atzF* gene homolog in bacterial strains *R. pickettii* strain D and *A. radiobacter* J14a. Both sequences, which covered 98% of the gene, were identical to each other and to the *atzF* gene of *Pseudomonas* sp. strain ADP, which had previously been shown to encode an enzyme that hydrolyzes allophanate (27) but has no activity with urea (N. Shapir et al., submitted).

The presence of identical *atzE* and *atzF* genes in these three bacteria suggested that they all might contain a cyanuric acid metabolism operon similar to that identified in *Pseudomonas* sp. strain ADP. To test this hypothesis, PCR primers were designed to amplify nearly the entire operon, beginning within the *atzD* gene and ending within the *atzF* gene. This led to the amplification of DNA fragments of the same size in *Pseudomonas* sp. strain ADP, *R. pickettii* strain D, and *A. radiobacter* strain J14a. The entire gene regions and junctions were sequenced and found to be identical in all three strains. These data were consistent with these strains containing the same operon and expressing the same metabolic pathway for the metabolism of cyanuric acid.

Other bacteria (Table 1) did not contain DNA that was amplified by PCR primers for *atzD*, *atzE*, or *AtzF*. However, strains lacking the *atz* genes contained *trzD*, which is known to encode a cyanuric acid hydrolase that produces biuret as a product (23). Some of those strains (NRRLB-12227, NRRLB-12228, and *E. cloacae* strain 99) had previously been proposed to metabolize biuret to urea. Further investigation of these strains required the direct identification of metabolites under conditions in which both urea and allophanate could be detected if they were produced.

Metabolite identification in *Pseudomonas huttiensis* NRRLB-12228. In previous studies, urea was detected in extracts of *Pseudomonas huttiensis* NRRLB-12228 metabolizing cyanuric acid (5), and genes homologous to *atzE* and *atzF* were not PCR amplified using total DNA obtained from strain 12228 (Table 1). In this context, metabolite experiments were conducted with strain 12228 using workup and chromatographic conditions under which synthetic allophanate was stable.

In initial experiments, allophanate, and not urea, was de-

FIG. 4. Kinetic course of [UL-¹⁴C]cyanuric acid metabolism by Pseudomonas huttiensis NRRLB-12228 and radiometric detection of metabolites separated by HPLC. (A) HPLC radiochromatogram at one time point, with each bar showing the amount of radioactivity collected in that fraction. Positions of elution for standards of urea, biuret, bicarbonate, allophanate, and cyanuric acid are shown by the arrows. (B) Concentrations of cyanuric acid, biuret, allophanate, and bicarbonate shown as a function of time as determined by multiple HPLC analyses as illustrated in panel A.

FIG. 5. Urease and allophanate hydrolase activity levels in crude protein extracts from (A) *Pseudomonas huttiensis* NRRLB-12228 and (B) *Enterobacter cloacae* strain 99, each measured after growth on ammonia, urea, or cyanuric acid as the sole nitrogen source. Black bars, urease activity; gray bars, allophanate hydrolase activity.

ammonia, levels of both enzymes were low (Fig. 5A). In contrast, when the strain was grown on urea as the nitrogen source, both activity levels were moderately higher and comparable. However, when the cells were grown on cyanuric acid, urease levels were somewhat higher but allophanate hydrolase activity was elevated more than 10-fold over the level found in ammonia-grown cells (Fig. 5A). Similar results were obtained with *Ralstonia pickettii* strain D (data not shown).

Different results were obtained with *Enterobacter cloacae* strain 99 (Fig. 5B), which had previously been reported to contain low but detectable levels of allophanate hydrolase activity (5). Experiments conducted here confirmed the previous observation that levels of allophanate hydrolase activity were lower than levels of urease activity. However, urease activity was similar in cells grown on cyanuric acid or ammonia. Growth on urea, by contrast, led to significantly higher levels of urease. In cells grown on cyanuric acid, both urease and allophanate hydrolase activities were comparable to the urease and allophanate hydrolase activities observed in ammonia-grown cells. In resting-cell experiments with *Enterobacter cloacae* strain 99 incubated with cyanuric acid, biuret was observed to accumulate to high levels (data not shown). Biuret accumulation by strain 99 was also reported by Cook et al. (5). This suggested that enzymes downstream of cyanuric acid hydrolase are not expressed at high levels, consistent with our observation here and that of Cook et al. (5) that allophanate hydrolase activity is low. Thus, it was not possible to demon-

strate significant levels of either allophanate or urea by direct metabolic experiments using *Enterobacter cloacae* strain 99.

**Cloning and expression of allophanate hydrolase from *Enterobacter cloacae* strain 99.** The recent deposit of the sequence of a gene region from *E. cloacae* strain 99 that is involved in s-triazine ring metabolism (GenBank accession no. AF342826) revealed a protein with significant (68%) amino acid sequence identity to allophanate hydrolase from *Pseudomonas* sp. strain ADP. To test whether this protein was indeed an allophanate hydrolase, we PCR amplified the gene region from *E. cloacae* strain 99 genomic DNA and cloned and expressed the protein in *E. coli*. Protein extracts from the recombinant *E. coli* strain showed allophanate hydrolysis activity and no urease activity. Control *E. coli* strains without the recombinant protein showed neither urease nor allophanate hydrolase activity. Both the location of the gene, directly upstream of other genes implicated in s-triazine ring metabolism, and the activity of the enzyme demonstrated here suggested that this enzyme is involved in cyanuric acid metabolism in a pathway that proceeds through allophanate.

**Sequence analysis of allophanate hydrolases and putative allophanate hydrolases.** The experimental studies presented here demonstrated that phylogenetically distinct bacteria express allophanate hydrolase activity. In this context, sequences in databases were compared with the identical allophanate hydrolase sequences found in this study for *R. pickettii* strain D, *A. radiobacter* J14a, and *Enterobacter cloacae* strain 99 (Fig. 6). Some of the genes in Fig. 6 have been functionally demonstrated, by either biochemical or genetic analysis, to encode allophanate hydrolases. For example, the *Enterobacter cloacae* gene had previously been identified as a urea amidolyase by genetic studies and was demonstrated biochemically in this study to be a bona fide allophanate hydrolase (accession no. AF342826). Furthermore, some of the genes in Fig. 6 had been suggested to encode allophanate hydrolases based on their genetic proximity to a metabolically cofunctional urea carboxylase gene (22, 22a). Other proteins in Fig. 6 have had functions attributed to them only by computational genome annotation and have been hypothesized to encode asparagine or glutamine tRNA amidases or amidases of unknown function. It is proposed here that these could be allophanate hydrolases. Not all allophanate hydrolases in prokaryotes are involved in cyanuric acid metabolism; those identified by Kanamori et al. (22, 22a) are likely involved in urea metabolism that proceeds via carboxylation and subsequent hydrolysis of the resultant allophanate to yield ammonia and carbon dioxide (18).

---

**DISCUSSION**

Previous studies concluded that the metabolism of s-triazine ring compounds, such as melamine and atrazine, proceeds through cyanuric acid, biuret, and urea, with the last metabolic step carried out via the enzyme urease (1, 3, 4, 10, 24, 36). The present study provides evidence that allophanate, not urea, is the last intermediate in this pathway and that allophanate hydrolase, not urease, is the enzyme involved. The role of allophanate hydrolase, only recently demonstrated in bacterial metabolism (22a, 27), might be much broader than previously appreciated.

The previous proposal that urea is an intermediate in bacterial cyanuric acid metabolism was based largely on sound experimental observations. First, bacteria that grew on cyanuric acid were also observed to grow with urea as the sole nitrogen source (5, 10). Second, urease activity was detected when cells were grown on cyanuric acid (1, 5, 10). Third, metabolism of compounds structurally analogous to cyanuric acid, such as barbituric acid, proceeds via urea (37). Fourth, mutants blocked in complete cyanuric acid metabolism accumulated an organic product that, after workup and chromatography, was identified as urea (10). These data were interpreted with the proposal that a biuret hydrolase enzyme was cleaving the subterminal carbon-nitrogen bond of biuret to yield urea and carbamate (Fig. 7, lower pathway). Carbamate decomposes to ammonia and bicarbonate with a half-life of 70

FIG. 6. Dendrogram of protein sequence relatedness comparing allophanate hydrolase homologs in all bacteria indicated. Boxed entries are found in bacteria that contain a urea carboxylase or s-triazine biodegradation genes, consistent with an assignment as an allophanate hydrolase. Those strains highlighted in gray have been shown in this study to specifically function as allophanate hydrolases involved in cyanuric acid metabolism. The protein sequences used for the dendrogram are as follows: *Pseudomonas* sp. strain ADP AtzF (NP_862539), *R. pickettii* strain D (this study), *A. radiobacter* J14a (this study), *Enterobacter cloacae* (AAK11683), *Oleomonas sagaranensis* (BAD16655), *Novosphingobium aromaticivorans* (ZP_00304984), *Caulobacter crescentus* (NP_420637), *Gloeobacter violaceus* (NP_923907), *Magnetospirillum magnetotacticum* (ZP_00208237), *Rhodopseudomonas palustris* (CAE26847), *Bradyrhizobium japonicum* (NP_772446), *Mesorhizobium loti* (NP_107459.1), *Rhizobium* sp. strain NGR234 (AAQ87317.1), *Wolinella succinogenes* (NP_907309), *Microbulbifer degradans* (ZP_00314767), *Pseudomonas fluorescens* (ZP_00266691), *Burkholderia cepacia* (ZP_00211646), *Dechloromonas aromatica* (ZP_00152874), *Ralstonia eutropha* (ZP_00202612), *Rubrivivax gelatinosus* (ZP_00241447), *Xanthomonas axonopodis* (NP_644621), *Erwinia carotovora* (YP_050236), *Burkholderia fungorum* (ZP_00282169), *Pseudomonas syringae* (NP_791188), *Saccharomyces cerevisiae* (AAC41643), *Streptomyces avermitilis* (NP_827873), and *Kineococcus radiotolerans* (ZP_00198103.1).

Vol. 71, 2005                                                                                       ALLOPHANATE HYDROLASE IN CYANURIC ACID METABOLISM 4443

![Chemical Reaction Diagram](https://i.imgur.com/chemical_reaction.png)

FIG. 7. Theoretical pathways of amide bond hydrolysis from biuret. Only the upper pathway is experimentally verified. Urea derived from allophanate decarboxylation (downward arrow).

ms at pH 7.0 (41). Thus, either pathway in Fig. 7 is equally plausible chemically.

However, more-recent studies called the previous conclusions into question. The plasmid containing atrazine degradation genes from *Pseudomonas* sp. strain ADP was completely sequenced. An operonic cluster of three genes, *atzDEF*, involved in cyanuric acid metabolism was identified, and each gene was expressed individually in *E. coli*. The *atzE* gene encoded an activity, biuret hydrolase, that produced allophanate, and the *atzF* gene encoded an activity that hydrolyzed allophanate but not urea. In these experiments with the recombinant *atzE* clone, urea was observed to form when the extracts were handled at neutral or acidic pHs but not at alkaline pHs (Fig. 7). Allophanate has been reported to undergo decarboxylation at acidic pHs, in the absence of enzymes, to yield urea (43). The possibility that cyanuric acid is metabolized via allophanate (Fig. 7, upper pathway) was investigated in the present study and confirmed in six different types of experiments.

First, the data presented in the present study demonstrated that the experimental conditions used in previous studies (5, 10, 16) would have caused rapid decarboxylation of allophanate to the stable product urea. This necessitated that all metabolic experiments conducted here be done under alkaline pH conditions, under which allophanate was shown to be stable. In the second set of experiments, using alkaline extraction and analytical conditions, allophanate was recovered from incubation mixtures of cyanuric acid with *Pseudomonas* sp. strain ADP.

Third, the kinetic course of metabolite formation and disappearance was consistent with a direct precursor-product relationship between biuret and allophanate, respectively, with the latter intermediate giving rise to carbon dioxide. No urea was observed in these experiments. Moreover, the stoichiometry of the intermediates was consistent with no additional intermediates being formed. In addition, purified allophanate hydrolase from *Pseudomonas* sp. strain ADP has been shown to produce ammonia and carbon dioxide directly, without the intermediary of urea (35a).

In the fourth set of experiments, protein extracts from cells grown on cyanuric acid contained allophanate hydrolase activity, and urease was shown not to be reactive with allophanate. Urease activity with urea as the substrate was detected, consistent with previous reports that urease is expressed when cells are grown on cyanuric acid (5). However, in our experiments, urease activity was detected under all growth conditions. Urease activity is often elevated under conditions of nitrogen limitation in *Pseudomonas* and other gram-negative bacteria (2, 19, 28), and growth on s-triazine compounds has been shown to represent a nitrogen-limited state in *Pseudomonas* sp. strain ADP (14, 15). Thus, the presence of urease activity would be expected even if urease was not involved in cyanuric acid metabolism.

In the fifth set of experiments, *atzE* and *atzF* gene homologs from different s-triazine-degrading bacteria were detected and sequenced. The genes were 100% identical in sequence to *atzE* and *atzF* from *Pseudomonas* sp. strain ADP. We are aware of only one case in which enzymes with identical amino acid sequences, but binding different metal ions, catalyze different reactions (7). However, AtzF does not bind a metal ion (35a), so even this rare exception seems to be excluded. Thus, it is almost certain that these different cyanuric acid-metabolizing bacteria containing AtzE and AtzF all express a biuret hydrolase that yields allophanate and an allophanate hydrolase that cleaves allophanate to yield 2 mol of ammonia.

In the sixth set of experiments, a gene designated as a urea amidolyase, from *E. cloacae* strain 99, previously thought to generate and metabolize urea during cyanuric acid metabolism, was expression cloned into *E. coli* and shown to encode allophanate hydrolase. The allophanate hydrolase gene clustered with other genes involved in the metabolism of s-triazine ring compounds, consistent with its involvement in the same metabolic pathway.

In total, the present studies present a compelling case that cyanuric acid metabolism proceeds through allophanate and uses the enzyme allophanate hydrolase to produce ammonia and carbon dioxide. This is also highly plausible from the vantage point of metabolic logic. A recent commentary has raised the issue that bacteria may not be able to make urease under all conditions, even when urea is present as a sole nitrogen source (18). An alternative strategy is thus to use an ATP-dependent carboxylase to transform urea to allophanate and then use allophanate hydrolase to release the two nitrogen atoms to support growth. It is now emerging that allophanate hydrolase may be more common in bacteria than previously thought (22, 22a). The present study extended this analysis to examine the distribution of genes homologous, and with high

sequence identity, to experimentally determined allophanate hydrolases described in this study (Fig. 6). These data strongly suggest that a number of genes in the databases, identified as amidases of various types, in fact may function to hydrolyze allophanate. Allophanate might be an intermediate in the bacterial catabolism of other nitrogen heterocyclic rings, which are found in thousands of natural products (17). Allophanate is more easily hydrolyzed than urea (18), and the former does not require a complex nickel-containing enzyme. Thus, the intermediary of allophanate in metabolic pathways is a potential logical alternative to urea for the breakdown of the thousands of ring compounds containing the string of atoms “C-N-C-N” within their chemical structure.

**Conclusions.** Taken together, these experiments are consistent with the conclusion that cyanuric acid metabolism proceeds through allophanate exclusively in all the strains tested. This is important to establish because almost all papers discussing s-triazine ring degradation over the last 20 years have presumed that urea is the pathway intermediate in all bacteria isolated for their ability to mineralize melamine, ammeline, and s-triazine herbicides (1, 6, 12, 30, 34, 36). The hydrolysis of biuret at the subterminal carbon-nitrogen bond is chemically plausible, so it is certainly possible that some bacteria might hydrolyze biuret to yield urea. However, reaching this conclusion, in light of the present evidence, would be dependent on isolating metabolites using workup and analytical conditions that preserve allophanate and showing that enzymes generating and consuming allophanate are absent.

**ACKNOWLEDGMENTS**

This work was supported, in part, by grant USDA/CSREES/NRI 2202-35107-12508 and by Department of Energy Office of Biological and Environmental Research grant DE-FG02-01ER63268. We thank Jack Richman for helpful discussions and Gilbert Johnson for synthesis of allophanate.

**REFERENCES**

1. Beilstein, P., and R. Hutter. 1980. Enzymatic cleavage of cyanuric acid by a hydrolase. Experientia 36:1457.
2. Collins, C. M., D. M. Gutman, and H. Laman. 1993. Identification of a nitrogen-regulated promoter controlling expression of *Klebsiella pneumoniae* urease genes. Mol. Microbiol. 8:187–198.
3. Cook, A. M. 1987. Biodegradation of s-triazine xenobiotics. FEMS Microbiol. Rev. 46:93–116.
4. Cook, A. M., H. Grossenbacher, P. Beilstein, and R. Hutter. 1983. The degradative pathway of cyanuric acid. Experientia 39:1432.
5. Cook, A. M., P. Beilstein, H. Grossenbacher, and R. Hutter. 1985. Ring cleavage and degradative pathway of cyanuric acid in bacteria. Biochem. J. 231:25–30.
6. Cook, A. M., and R. Hütter. 1981. s-Triazines as nitrogen sources for bacteria. J. Agric. Food Chem. 29:1135–1143.
7. Dai, Y., P. C. Wensink, and R. H. Abeles. 1999. One protein, two enzymes. J. Biol. Chem. 274:1193–1195.
8. de Souza, M. L., N. R. Pechacek, L. P. Wackett, M. J. Sadowsky, and B. L. Hoyle. 1998. Diversity in bacterial atrazine degradation pathways, abstr. Q-195, p. 453. Abstr. 98th Gen. Meet. Am. Soc. Microbiol.
9. Eaton, R. W., and D. W. Ribbons. 1982. Metabolism of dibutylphthalate and phthalate by *Micrococcus* sp. strain 12B. J. Bacteriol. 151:48–57.
10. Eaton, R. W., and J. S. Karns. 1991. Cloning and analysis of s-triazine catabolic genes from *Pseudomonas* sp. strain NRRLB-12227. J. Bacteriol. 173:1215–1222.
11. Eaton, R. W., and J. S. Karns. 1991. Cloning and comparison of the DNA encoding ammelide aminohydrolase and cyanuric acid amidohydrolase from three s-triazine-degrading bacterial strains. J. Bacteriol. 173:1363–1366.
12. Erickson, L. E., and H. K. Lee. 1989. Degradation of atrazine and related s-triazines. Crit. Rev. Environ. Cont. 19:1–13.
13. Fruchey, I., N. Shapir, M. J. Sadowsky, and L. P. Wackett. 2003. On the origins of cyanuric acid hydrolase: purification, substrates, and prevalence of AtzD from *Pseudomonas* sp. strain ADP. Appl. Environ. Microbiol. 69:3653–3657.
14. Garcia-Gonzalez, V., F. Govantes, L. J. Shaw, R. G. Burns, and E. Santero. 2003. Nitrogen control of atrazine utilization in *Pseudomonas* sp. strain ADP. Appl. Environ Microbiol. 69:6987–6993.
15. Garcia-Gonzalez, V., F. Govantes, O. Porrua, and E. Santero. 2005. Regulation of the *Pseudomonas* sp. strain ADP cyanuric acid degradation operon. J. Bacteriol. 187:155–167.
16. Grossenbacher, H., A. M. Cook, and R. Hutter. 1985. Determination of biuret and urea by high-performance liquid chromatography. J. Chromatogr. 331:161–167.
17. Harborne, J. B. 1988. Introduction to ecological biochemistry, 3rd ed. Academic Press, San Diego, Calif.
18. Hausinger, R. P. 2004. Metabolic versatility of prokaryotes for urea decomposition. J. Bacteriol. 186:2520–2522.
19. Janssen, D. B., W. J. A. Habeis, J. T. Marugg, and C. van der Drift. 1982. Nitrogen control in *Pseudomonas aeruginosa*: mutants affected in the synthesis of glutamine synthetase, urease, and NADP-dependent glutamate dehydrogenase. J. Bacteriol. 151:22–28.
20. Jensen, H. L., and A. S. Abdel-Ghaffer. 1969. Cyanuric acid as nitrogen source for microorganisms. Arch. Mikrobiol. 67:1–5.
21. Jutzi, K., A. M. Cook, and R. Hutter. 1982. The degradative pathway of the s-triazine melamine. The steps to ring cleavage. Biochem. J. 208:679–684.
22. Kanamori, T., N. Kanou, H. Atomi, and T. Imanake. 2004. Enzymatic characterization of a prokaryotic urea carboxylase. J. Bacteriol. 186:2532–2539.
22a. Kanamori, T., N. Kanou, S. Kusakabe, H. Atomi, and T. Imanaka. 2005. Allophanate hydrolase of *Oleomonas sagaranensis* involved in an ATP-dependent degradation pathway specific to urea. FEMS Microbiol. Lett. 245:61–65.
23. Karns, J. S. 1999. Gene sequence and properties of an s-triazine ring-cleavage enzyme from *Pseudomonas* sp. strain NRRLB-12227. Appl. Environ. Microbiol. 65:3512–3517.
24. Karns, J. S., and R. W. Eaton. 1997. Genes encoding s-triazine degradation are plasmid-borne in *Klebsiella pneumoniae* strain 99. J. Agric. Food Chem. 45:1017–1022.
25. Mandelbaum, R. T., D. T. Allan, and L. P. Wackett. 1995. Isolation and characterization of a *Pseudomonas* sp. that mineralizes the s-triazine herbicide atrazine. Appl. Environ. Microbiol. 61:1451–1457.
26. Martinez, B. 2001. Characterization, complete nucleotide sequence and organization of atrazine catabolic plasmid, pADP-1 from *Pseudomonas* sp. strain ADP. Ph.D. thesis. University of Minnesota, Saint Paul.
27. Martinez, B., J. Tomkins, L. P. Wackett, R. Wing, and M. J. Sadowsky. 2001. Complete nucleotide sequence and organization of the atrazine catabolic plasmid pADP-1 from *Pseudomonas* sp. strain ADP. J. Bacteriol. 183:5684–5697.
28. Mobley, H. L., M. D. Island, and R. P. Hausinger. 1995. Molecular biology of microbial ureases. Microbiol. Rev. 59:451–480.
29. Myskow, W., T. Lasota, and A. Stachyra. 1982. Cyanuric acid: a s-triazine derivative as a nitrogen source for some soil microorganisms. Acta Microbiol. Pol. 32:177–183.
30. Radosevich, M., S. J. Traina, Y. Hao, and O. H. Tuovinen. 1995. Degradation and mineralization of atrazine by a soil bacterial isolate. Appl. Environ. Microbiol. 61:297–302.
31. Rousseaux, S., A. Hartmann, and G. Soulas. 2001. Isolation and characterization of new gram-negative and gram-positive atrazine degrading bacteria from different French soils. FEMS Microbiol. Ecol. 36:211–222.
32. Saldick, J. 1974. Biodegradation of cyanuric acid. Appl. Microbiol. 28:1004–1008.
33. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.
34. Sanchez, M., C. Garbi, R. Martinez-Alvarez, L. T. Ortiz, J. L. Allende, and M. Martin. 2005. *Klebsiella planticola* strain DSZ mineralizes simazine: physiological adaptations involved in the process. Appl. Microbiol. Biotechnol. 66:589–596.
35. Shapir, N., J. P. Osborne, G. Johnson, M. J. Sadowsky, and L. P. Wackett. 2002. Purification, substrate range and metal center of AtzC: the N-isopropylammelide hydrolase involved in bacterial atrazine metabolism. J. Bacteriol. 184:5376–5384.
35a. Shapir, N., M. J. Sadowsky, and L. P. Wackett. 2005. Purification and characterization of allophanate hydrolase (AtzF) from *Pseudomonas* sp. strain ADP. J. Bacteriol. 187:3731–3738.
36. Shelton, D. R., J. S. Karns, G. W. McCarty, and D. R. Durham. 1997. Metabolism of melamine by *Klebsiella terragena*. Appl. Environ. Microbiol. 63:2832–2835.
37. Soong, C. L., J. Ogawa, and S. Shimizu. 2001. Novel amidohydrolytic reactions in oxidative pyrimidine metabolism: analysis of the barbiturase reaction and discovery of a novel enzyme, ureidomalonase. Biochem. Biophys. Res. Commun. 286:222–226.
38. Struthers, J. K., K. Jayachandran, and T. B. Moorman. 1998. Biodegradation of atrazine by *Agrobacterium radiobacter* J14a and use of this strain in bioremediation of contaminated soil. Appl. Environ. Microbiol. 64:3368–3375.

39. Topp, E., H. Zhu, S. M. Nour, S. Houot, M. Lewis, and D. Cuppels. 2000. Characterization of an atrazine-degrading *Pseudaminobacter* sp. isolated from Canadian and French agricultural soils. Appl. Environ. Microbiol. **66**:2773–2782.

40. Topp, E., W. M. Mulbry, H. Zhu, S. M. Nour, and D. Cuppels. 2000. Characterization of s-triazine herbicide metabolism by a *Nocardioides* sp. isolated from agricultural soil. Appl. Environ. Microbiol. **66**:3134–3141.

41. Wang, T. T., S. H. Bishop, and A. Himoe. 1972. Detection of carbamate as a product of the carbamate kinase-catalyzed reaction by stopped flow spectro-photometry. J. Biol. Chem. **247**:4437–4440.

42. Weatherburn, M. W. 1967. Phenol-hypochlorite reaction for determination of ammonia. Anal. Chem. **39**:791–794.

43. Whitney, P. A., and T. G. Cooper. 1972. Urea carboxylase and allophanate hydrolase. Two components of adenosine triphosphate:urea amido-lyase in *Saccharomyces cerevisiae*. J. Biol. Chem. **247**:1349–1353.

44. Zeyer, J., J. Bodmer, and R. Hütter. 1981. Microbial degradation of amme-line. Zentbl. Bakteriol. Hyg. Abt. 1 Orig. C **2**:289–298.

45. Zeyer, J., J. Bodmer, and R. Hütter. 1981. Rapid degradation of cyanuric acid by *Sporothrix schenckii*. Zentbl. Bakteriol. Hyg. Abt. 1 Orig. C **2**:99–110.
