
Int. J. Dev. Biol. 49: 491-500 (2005)  
doi: 10.1387/ijdb.041968pm  

# Circadian clock signaling in *Arabidopsis thaliana*:  
from gene expression to physiology and development  

PALOMA MÁS*  
Consorcio CSIC-IRTA, Laboratorio de Genética Molecular Vegetal, Instituto de Biología Molecular de Barcelona (IBMB-CSIC),  
Barcelona, Spain  

**ABSTRACT** The daily rotation of the earth on its axis leads to predictable periodic fluctuations of environmental conditions. Accordingly, most organisms have evolved an internal timing mechanism, the circadian clock, which is able to recognize these 24-hour rhythmic oscillations. In plants, the temporal synchronization of physiology with the environment is essential for successful plant growth and development. The intimate connection between light signaling pathways and the circadian oscillator allows the anticipation of the environmental transitions and the measurement of day-length as an indicator of changing seasons. In recent years, significant advances have been made in the genetic and molecular dissection of the plant circadian system, mostly in *Arabidopsis thaliana*. The overall plant clock organization is highly complex; the system seems to include several input pathways, tightly regulated central oscillators and a myriad of outputs. The molecular cloning and characterization of a number of clock components has greatly improved our view of the plant central oscillator and additional players will most likely come into place very soon. Molecular mechanisms underlying circadian clock function are also beginning to be characterized. The emerging model relies on negative feedback loops at the core of the oscillator. Additional levels of post-transcriptional and post-translational regulation also contribute to the generation and maintenance of the rhythms. Globally, these studies have shed new light on how the clock coordinates plant physiology and development with the daily and seasonal environmental cycles.

**KEY WORDS:** biological clock, circadian rhythms, *Arabidopsis thaliana*

The biological clock entails a 24-hour rhythm on biochemical and physiological processes such that they occur at specific, advantageous times of the day-night cycle (Dunlap, 1999, Young and Kay, 2001). The circadian movements of leaves and flowers, the fragrance emission, the stomatal opening, hypocotyl expansion or the photoperiodic control of flowering time are some examples of processes tightly regulated by the plant clockwork (Barak *et al.*, 2000, McClung, 2001). Underlying all these physiological rhythms are endogenous circadian oscillations of gene expression. Indeed, genomic approaches have identified in *Arabidopsis* hundreds of genes under clock control, with peaks of expression at all phases of the day/night cycle (Harmer *et al.*, 2000, Schaffer *et al.*, 2001). Although the rhythmic oscillations persist in constant conditions (i.e. in the absence of environmental transitions) the circadian clock does not run in isolation from the environment. The clock includes a resetting mechanism by which it is synchronized each day to the correct time (Devlin and Kay, 2001). The presence of an endogenous timing system provides an adaptive advantage, enabling the anticipation of the environmental transitions and the temporal coordination of physiological events to occur at specific phase relationships with the environment (Johnson, 2001). Thus, the circadian clock can be considered as an internal processor of environmental signals (such as light and temperature) that coordinates the appropriate timing of metabolic and developmental activities in the plant (Harmer *et al.*, 2001, McClung, 2001).

Classically, the circadian system has been divided into three main components: the *input pathways* involved in the perception and transmission of environmental signals to synchronize the *central oscillator* or pacemaker that generates and maintains rhythmicity through multiple *output pathways* that connect the oscillator to physiology and metabolism (Figure 1). Clearly, this is an oversimplified conceptual model of the clock. Several lines of evidence reveal the existence of a far more complicated circadian system, with output elements modulating the pace of the oscillator and input elements being themselves tightly controlled by the clock. The existence of different free-running periods in the expression of diverse clock outputs is indicative of separate

---

*Address correspondence to:* Dr. Paloma Más. Consorcio CSIC-IRTA, Laboratorio de Genética Molecular Vegetal, Instituto de Biología Molecular de Barcelona (IBMB-CSIC), C/ Jordi Girona 18-26, 08034 Barcelona, Spain. Fax: +34-94-204-5904. e-mail: pmmgmc@ibmb.csic.es  

0214-6282/2005/$25.00  
© UBC Press  
Printed in Spain  
www.intjdevbiol.com

light conditions (Millar *et al.*, 1995). The *toc1-1* mutant plants also exhibited a day-length-insensitive, early-flowering phenotype, under cycles of 24 h, but not under 21 h environmental cycles (which match the endogenous period of this mutant (Strayer *et al.*, 2000). This indicates that the photoperiodic insensitivity is due to the improper functioning of the clock caused by the *toc1-1* mutation. Cloning of the gene revealed that TOC1 encoded a nuclear protein containing a receiver domain similar to the one found in plant response regulators (Strayer *et al.*, 2000). However, the conserved phospho-accepting aspartate residues present in bona fide response regulators is missing in TOC1 (also denominated pseudo-response regulator), suggesting that it does not function in a canonical phosphor-relay mechanism (Mizuno, 2004). In addition, TOC1 contains a distinctive COOH-terminal motif (CCT motif) which is conserved within the CONSTANS (CO) family of plant transcription factors (Strayer *et al.*, 2000). Analysis of TOC1 expression revealed that the mRNA rhythmically cycled and participated in a negative feedback loop mechanism to control its own expression (Strayer *et al.*, 2000). Together, these observations led to the idea that TOC1 was an important component of the core of the oscillator, rather than part of the light input pathway to the clock. However, a detailed characterization of TOC1 function using RNAi plants and a strong allele of *TOC1* (*toc1-2*) provided evidence of unpredicted roles for TOC1 in the control of circadian and photo-morphogenic responses (Más *et al.*, 2003a). These studies showed that silencing of the *TOC1* gene caused arrhythmia in constant darkness and in various intensities of red light. In addition, *TOC1* RNAi and *toc1-2* mutant plants displayed an important reduction in sensitivity to red and far-red light in the control of hypocotyl elongation whereas increments in *TOC1* gene dosage clearly enhanced light sensitivity (Más *et al.*, 2003a). Thus, based on these studies, it was postulated a new role for TOC1 in the integration of light signals from phytochromes to clock outputs, controlling circadian gene expression and other light-dependent developmental processes in the plant.

Two transcription factors, CCA1 (CIRCADIAN CLOCK ASSOCIATED 1) and LHY (LATE ELONGATED HYPOCOTYL), with a single MYB-like domain, have also been closely associated with the plant circadian system (Schaffer *et al.*, 1998, Wang and Tobin, 1998). CCA1 was identified as a binding factor to a region of the promoter of *Arabidopsis LHCB* gene. These initial studies indicated that CCA1 protein was an important element in the functioning of the phytochrome signal transduction leading to increased transcription of the *LHCB* gene (Wang *et al.*, 1997). Later studies also revealed an additional role of CCA1 in the circadian clockwork. Loss of function *cca1* mutants displayed a shorter than wild-type period of genes expressed at different circadian times while CCA1 constitutive over-expression clearly disrupted rhythmicity in various clock outputs including hypocotyl elongation, leaf movements and circadian gene expression (Green and Tobin, 1999, Wang and Tobin, 1998).

The *hy* mutation was caused by the insertion of a transposon (Ds) within the 5' untranslated region (5'UTR) of the *LHY* gene, causing its over-expression. Studies of these LHY over-expressing plants as well as loss of function *hy* mutants showed clear alterations in circadian rhythmicity for leaf move-
ment and for the circadian regulation of gene expression. Both $CCA1$ and $LHY$ transcripts rhythmically oscillate, with a peak of expression early in the morning, shortly after dawn. The rhythms persist in constant conditions (constant light and constant darkness) indicating that both genes are under circadian control (Schaffer *et al.*, 1998, Wang and Tobin, 1998). The circadian expression of the $LHY$ and $CCA1$ transcripts is abolished in $hy1$ and CCA1-overexpressing (CCA1-ox) plants, suggesting that the rhythmic expression of $LHY$ and $CCA1$ is required for normal circadian function. Interestingly, CCA1 and LHY regulated their own and each other’s expression. Over-expressing plants of either gene exhibited arrhythmic expression of the endogenous $LHY$ and $CCA1$ transcripts. Furthermore, the expression of both endogenous genes was repressed to levels similar to trough in wild-type plants. These results suggested that LHY and CCA1 might function redundantly as components of a negative feedback loop (Schaffer *et al.*, 1998, Wang and Tobin, 1998). Indeed, recent analysis comparing phenotypes in single and double mutant $hy-cca1$ plants indicated that the $LHY$ and $CCA1$ genes are partially redundant and that they are required for maintenance of circadian rhythmicity in *Arabidopsis* (Alabadi *et al.*, 2002, Mizoguchi *et al.*, 2002).

More recently, another player has joined the clockwork team, in very close association with the circadian oscillator. $ELF4$ ($EARLY-FLOWERING4$) which encodes a protein of 111 amino acids without predictable protein domains, was identified by its involvement in photoperiodic perception and circadian regulation (Doyle *et al.*, 2002). Analysis of plants containing a T-DNA insertion in the $ELF4$ gene revealed an early flowering phenotype in short photoperiods while under long-day conditions, the mutant plants flowered at about the same time as wild-type (Doyle *et al.*, 2002). The $elf4$ mutation reduced $CCA1$ expression and affected the rhythms of clock-controlled genes expressed at different circadian times. Their expression rapidly became arrhythmic under constant conditions, but individual seedlings were transiently rhythmic with highly variable periods. These results suggest a role for ELF4 in maintaining the accuracy of the rhythms. Consistent with the early flowering phenotype in short days, the $elf4$ mutation increased expression of $CO$, a gene that has a relevant role in floral induction (see below).

### Synchronization with external time: light input to the clock

The earliest experimental evidence of the existence of an internal time-keeping mechanism came with the observations that the daily leaf movement of Mimosa plants continued even when plants were placed in constant darkness. These observations demonstrated that the environmental changes were not required to trigger the rhythmic response and revealed one of the main properties characterizing circadian rhythmicity: its ability to persist in the absence of environmental cues. In constant conditions, the period of the rhythms is approximately, but not exactly 24-hours. To generate an accurate 24-hours period, the endogenous oscillator must be synchronized with the external, environmental time (Johnson, 2001). Environmental transitions between dawn and dusk help to adjust the

endogenous period of the clock to exactly match the 24-hour period that we find in nature (Devlin and Kay, 2001). As light is an essential environmental signal in the synchronization of the clock with the outside world, a long-standing goal in the plant circadian field has been the identification of the components in the signal transduction pathway responsible for resetting the clock. Studies of photoreceptor-deficient mutants crossed into the $CAB::LUC$ plants provided evidence that two classes of photoreceptors, phytochromes (PHY) and cryptochromes (CRY), participated in the light-driven entrainment of the *Arabidopsis* clock (Somers *et al.*, 1998a). Four of the five phytochromes identified in *Arabidopsis* (PHYA, PHYB, PHYD and PHYE) act additively in the red-light input to the clock while *Arabidopsis* CRYPTOCHROME 1 (CRY1) acts as a clock photoreceptor for high and low fluences of blue light and both CRY1 and CRY2 act redundantly at intermediate fluences of blue light (Devlin and Kay, 2000, Somers *et al.*, 1998a). Mutations in $cry2$ were also shown to affect entrainment in white light conditions (Más *et al.*, 2000) suggesting an interaction of CRY2 with the phytochrome signaling pathway. Indeed, using Fluorescent Resonance Energy Transfer (FRET) microscopy, PHYB and CRY2 were shown to interact *in vivo*, co-localizing in nuclear speckles in response to light treatments (Más *et al.*, 2000). Surprisingly, cryptochrome mutant plants also showed altered entrainment under red light suggesting that CRY1 is required for PHYA signaling to the clock in both red and blue light (Devlin and Kay, 2000). Quadruple mutants (lacking $phyA$, $phyB$, $cry1$ and $cry2$) maintained rhythmicity (Yanovsky *et al.*, 2000), indicating that additional components participate in the light input pathway to reset the clock. Some possible candidates to perform this function are the gene family ZEITLUPE ($ZTL$), FLAVIN-BINDING, KELCH REPEAT, F-BOX 1 ($FKF1$) and LOV, KELCH PROTEIN 2 ($LKP2$). The genes encode proteins that contain six kelch repeats as well as a LOV and an F-box domain. The LOV domain is highly similar to the one found in the blue light photoreceptors NPH1 (*Arabidopsis*) and WHITE COLLAR 1 (WC-1; *Neurospora*). The F-box is a motif found in proteins that act as adapters and bring specific substrates to ubiquitin protein ligase subunits for degradation (Craig and Tyers, 1999). The unique combination of these domains in the ZTL family of proteins suggests that they might mediate light-dependent protein degradation of critical clock components. Indeed, a recent report has shown TOC1 as one of the ZTL substrates (see below and Más *et al.*, 2003b). Analysis of $zt1$ mutant plants revealed a long period circadian phenotype that was dependent on light intensity (Somers *et al.*, 2000) while over-expression of LKP2 resulted in arrhythmic expression of several clock-controlled genes (Schultz *et al.*, 2001). The $fkf1$ mutant was initially identified by its late flowering phenotype (Nelson *et al.*, 2000) and more recently, its role in the photoperiodic control of $CO$ expression has been described (Imaizumi *et al.*, 2003). The possible function of these proteins in photo-entrainment might result from their direct interaction with photoreceptors. Indeed, ZTL has been shown to interact *in vitro* with PHYB and CRY1 (Jarillo *et al.*, 2001) although it remains to be determined the biological relevance of these interactions *in vivo*.

The central oscillator can be also entrained by temperature (warm/cold) cycles. This feature has been used to analyze whether circadian mutants affect specifically light signaling
input to the clock (Millar, 2004). A detailed phase response curve to temperature was recently published (Michael *et al.*, 2003a) and new evidence suggest that the light- and temperature-sensing systems in plants might be connected, since photoreceptor signaling pathways can be temperature sensitive (Halliday and Whitelam, 2003, Mazzella *et al.*, 2000). In spite of these new advances, we are far from understanding the mechanisms of clock entrainment by temperature cues.

### Additional genes: searching for functional roles

ELF3 (EARLY FLOWERING 3) is an important component of the light signaling pathway to the clock. Initial studies showed that *elf3* mutants displayed arrhythmic clock output expression in continuous light whereas rhythmicity was retained in constant darkness, suggesting a defect in the light input pathway to the clock (Hicks *et al.*, 1996). In addition, the *elf3* mutant plants did not discriminate different photoperiods, flowering as early in short days as in long days. In wild-type plants, *CAB* induction is rhythmically repressed by the clock during the night, allowing a higher induction during the light period of the day (Millar and Kay, 1996). The circadian «gating» of *CAB* acute induction is lost in *elf3* mutants, leading to constitutive CAB activation (McWatters *et al.*, 2000). In a series of elegant studies, it was shown that the arrhythmicity in constant light of *elf3* mutants was due to a defect in the repression of light signals during the subjective night (McWatters *et al.*, 2000). Both *ELF3* transcript and protein rhythmically oscillate with a peak of expression at dusk, just when ELF3 is required to antagonize light signals to the clock (Covington *et al.*, 2001, Hicks *et al.*, 2001, Liu *et al.*, 2001b, McWatters *et al.*, 2000). Despite these significant advances, the exact mechanisms of ELF3 function in the light signaling pathway to the clock is not well understood. The *elf3* mutants show more apparent phenotypes under red light conditions, indicating a connection with PHYB signaling. A physical interaction between PHYB and ELF3 has been described (Liu *et al.*, 2001b). However, analysis of double mutant plants *phyB-elf3* showed an additive behavior, suggesting additional routes of ELF3 independently of PHYB (Reed *et al.*, 2000).

Another gene with partially overlapping functions with *ELF3* is *TIME FOR COFFEE* (*TIC*, Hall *et al.*, 2003). *Tic* mutant plants were shown to disrupt circadian rhythmicity of several clock outputs, arresting clock function in the subjective morning. Similar to ELF3 function at night, it was suggested a possible role for TIC in gating the clock during the day. Studies with double mutant *tic/elf3* showed complete arrhythmia demonstrating that both genes are important in the functioning of the clock. The intimate connection between light signals and the circadian clock is also represented in SRR1 (sensitivity to red light reduced 1) function. *Srr1* mutants were shown to be defective in PHYB-mediated signaling as well as in the normal expression of clock outputs (Staiger *et al.*, 2003). The circadian phenotypes of *srr1* mutants suggest that SRR1 activity might be required for normal oscillator function.

GIGANTEA (*GI*) is another gene that has been linked to the circadian clock. GI encodes a novel, putative membrane protein that was initially identified by its highly delayed flowering phenotype under long days (Fowler *et al.*, 1999). Additional experiments showed fluence-rate defects of the *gi* mutants in

the circadian regulation of leaf movement and transcript oscillations, suggesting a connection of GI with the light input to the clock (Fowler *et al.*, 1999, Park *et al.*, 1999). Rhythmic expression of *GI* transcript was shown to be altered in *gi* mutants, suggesting that GI could form a feedback loop required for normal clock function (Fowler *et al.*, 1999, Park *et al.*, 1999). In plants over-expressing CCA1 or LHY, the circadian expression of *GI* was disrupted and reciprocally, the absence of GI caused a reduction in the *CCA1* and *LHY* expression. The identification of interacting partners of GI might help to clarify GI function in this puzzled network. Recently, a report showed the interaction of SPINDLY (SPY), a negative regulator of gibberellin, with GI (Tseng *et al.*, 2004). The report describes their involvement in light responses, flowering and rhythms in cotyledon movements. A further *gi* allele was identified in a screen for alterations in hypocotyl elongation under red light (Huq *et al.*, 2000). These studies revealed that GI was also associated with PHYB function although the correlation of GI roles in photomorphogenesis and in the light input to the clock remains to be elucidated.

### Transcriptional feedback loops

Knowledge of circadian clock mechanisms in *Arabidopsis thaliana* has been aided by the formulation of circadian function in other circadian systems. The general mechanism of the clockwork seems to be conserved among organisms and it is based on negative feedback loops at the core of the oscillator (Dunlap, 1999, Young and Kay, 2001). This common theme, at its simplest, involves positive and negative components that mutually regulate their rhythmic abundance and/or activity. The oscillatory expression and regulation of clock components generates circadian rhythms that are translated to multiple clock outputs. While this mechanism has been firmly established in *D. melanogaster*, mammals and *Neurospora crassa* (Harmer *et al.*, 2001), we are just beginning to ascertain clock mechanisms in higher plants (Figure 2). In *Arabidopsis*, the first working model for the plant clockwork came with the observation of a reciprocal regulation between CCA1/LHY and TOC1 (Alabadí *et al.*, 2001). The two MYB proteins (LHY and CCA1) negatively regulated *TOC1* expression that in turn positively induced *LHY/CCA1*. According to this model, the MYB transcription factors CCA1 and LHY might function as negative elements within this transcriptional loop, with a similar role to the one described for PERIOD (PER) and TIMELESS (TIM) in the circadian system of *Drosophila*, or FREQUENCY (FRQ) in the *Neurospora* clock. TOC1 might work as a positive component, similar to the clock components CYCLE (CYC) and CLOCK (CLK) in *Drosophila*, or WHITE COLLAR 1 and 2 (WC1 and WC2) in the circadian system of *Neurospora*. The repression of *TOC1* might take place by the direct binding of CCA1/LHY proteins to a region in the *TOC1* promoter (denominated evening-element) (Alabadí *et al.*, 2001, Harmer *et al.*, 2000). Decreasing CCA1/LHY levels throughout the day causes a relieved in that repression, resulting in rising levels of *TOC1* mRNA. The positive action of TOC1 on *CCA1* and *LHY* expression thereby reinitiate the oscillatory cycle. The *TOC1* activation of *CCA1/LHY* is probably indirect because at least three other components also positively participate in *CCA1/*

$LHY$ expression: ELF3 (Schaffer *et al.*, 1998) GI (Fowler *et al.*, 1999) and ELF4 (Doyle *et al.*, 2002).

Additional levels of complexity are added to this regulatory loop since $CCA1/LHY$ as well as $TOC1$ are members of multi-gene families. Several homologues of $CCA1/LHY$ (denominated REVEILLES) have been shown to circadianly oscillate with a peak of expression around dawn (Andersson *et al.*, 1999). These additional MYB proteins could form homo- and hetero-interactions among them that could contribute to the oscillatory loop. $TOC1$ is also a member of a gene family composed of 4 more components (denominated pseudo-response regulators, $PRR$ 3, 5, 7 and 9). Phenotypic analysis of mutant and over-expressing plants of individual $PRR$ has revealed alterations of period, phase and/or amplitude in some overt rhythms, changes in flowering time and affected sensitivity to red-light control of hypocotyl elongation (Mizuno, 2004). $PRR$ transcripts accumulate rhythmically in the order $PRR9-PRR7-PRR5-PRR3-TOC1$ with peak levels after dawn from 2 h ($PRR9$) to 10 h ($TOC1$, also known as $PRR1$) (Matsushika *et al.*, 2000, Strayer *et al.*, 2000). It is suggested that $PRR$ participate in the light signaling within the circadian clock but are not required for rhythm generation (Mizuno, 2004).

Several studies have reported different circadian periods in various output genes suggesting regulation by multiple oscillators (Hall *et al.*, 2002, Michael *et al.*, 2003a, Thain *et al.*, 2002). Based on their sensitivity to temperature, it was postulated the presence of two circadian clocks differentially controlling $CAB$

and $CAT3$ expression in Arabidopsis (Michael *et al.*, 2003a). The partial overlapping expression of these genes opens up the possibility of the co-existence of at least two oscillators in the same cell although further experiments will be required to unequivocally confirm this hypothesis.

### Post-translational regulation of the plant clockwork

Protein phosphorylation has been shown to be necessary in the functioning of the circadian clock in *Drosophila*, *Neurospora* and in humans. Various studies reveal that casein kinases (CK1 and CK2) are the main protein kinases involved in the phosphorylation of essential clock components (Dunlap, 2004, Edery, 1999, Loros and Dunlap, 2001, Toh *et al.*, 2001). In *Arabidopsis*, the protein kinase CK2 was shown to phosphorylate *in vitro* the clock-associated protein $CCA1$ (Sugano *et al.*, 1998). Over-expression of a CK2 regulatory subunit (denominated CKB3) affected the regulation of circadian rhythmicity, shortening the period of expression of several output genes (Sugano *et al.*, 1999). By examining the effects of a constitutively-expressing $CCA1$ mutant that could not be phosphorylated by CK2, it was demonstrated that $CCA1$ phosphorylation by CK2 is important for the normal functioning of the *Arabidopsis* circadian clock (Daniel *et al.*, 2004).

Detailed studies of $LHY$ over-expressing plants showed that in constant light conditions, circadian rhythmicity was abolished although rhythmic expression was preserved under light-dark

![Diagram](attachment:diagram.png)

Fig. 2. Current molecular model of circadian clock signaling in the *Arabidopsis*. Light signals are perceived by a set of multiple photoreceptors, including phytochromes (PHYs) and cryptochromes (CRYs), that participate in the resetting of the clock. Light effects on the clock are also modulated by the action of both positive (GI) and negative light-signaling components (ELF3). At the core of the oscillator, a feedback loop generates rhythmicity by the reciprocal regulation between $CCA1/LHY$ and $TOC1/ELF4$. $CCA1$ phosphorylation by CK2 is important for the normal functioning of the *Arabidopsis* circadian clock. The ZTL family (ZTL, FKF1 and LKP2) mediates between photoreceptors and the circadian clock. The targeted degradation of $TOC1$ protein by ZTL is essential in the control of circadian period by the clock. In the scheme, some of the genes and the physiological/metabolic processes regulated by the clock are represented: photosynthetic processes (CAB), lipid metabolism (CHS, CHALCONE SYNTHASE), cell elongation (EXP, EXPANSIN) and flowering-time regulation (CO). The transcription of CO is regulated by FKF1 and GI, whose transcription in turn is under the control of the circadian clock. The gray shaded circle represents the feedback loop that is central for clock function in *Arabidopsis*; dotted arrows indicate gene expression regulated by the clock.
cycles (Kim *et al.*, 2003a). The rhythmic oscillations were correlated with light-induced regulation of *LHY* translation, causing high amplitude changes in LHY protein levels. The authors suggested that both translational induction and transcriptional repression of *LHY* at dawn contribute to the robustness and accuracy of the circadian oscillations.

Recent genetic and molecular evidence have also revealed that a precise post-translational regulation of the TOC1 protein is essential for clock function in *Arabidopsis*. The F-box protein ZEITLUPE (ZTL) was shown to be involved in the targeted degradation of TOC1 (Más *et al.*, 2003b). In these studies, TOC1's physical interaction with ZTL was abolished by the *ztl-1* mutation resulting in constitutive levels of TOC1 protein expression. The proteosome-mediated degradation of TOC1 protein occurs mainly during the dark period and requires a functional ZTL. It was demonstrated that the TOC1-ZTL interaction was important in the control of TOC1 protein stability and responsible for the accurate regulation of circadian period by the clock (Más *et al.*, 2003b). The abundance of ZTL protein is also regulated by the proteosome. ZTL is more rapidly degraded at dawn when the protein reaches trough levels (Kim *et al.*, 2003b). Post-translational regulation of protein stability through the proteosome pathway has been shown to be essential in clock function in the mammalian system and in *Drosophila* (Grima *et al.*, 2002, Ko *et al.*, 2002, Yagita *et al.*, 2002). Together, these results suggest that tightly regulated protein degradation by the proteosome might be a conserved aspect in the regulation of the eukaryotic circadian clocks.

### Clock control of output gene expression

Global analysis of clock-controlled gene expression is a useful tool for identifying new components of the circadian system and finding co-expressed genes integrated into common metabolic or physiologic pathways. The use of oligonucleotide microarrays has provided new insights into global transcription networks regulated by the *Arabidopsis* clock (Harmer *et al.*, 2000, Schaffer *et al.*, 2001). Clustering based on functional features of these genes has revealed how the clock enables plants to anticipate and adapt to the daily and seasonal fluctuations of light and temperature. A large set of photosynthetic genes were shown to be under clock control. All these genes exhibited a co-regulated expression, peaking around midday. Nine genes involved in energetically demanding processes such as nitrogen assimilation were expressed early in the day, when energy levels are increased by light harvesting. The expression of 23 genes encoding enzymes involved in the biosynthesis of photo-protective pigments was shown to peak coordinately before dawn. This pattern of expression might help to protect plants from the damaging effects of UV-B light. Another cluster of coordinately expressed genes was represented by a group involved in lipid modification which has been shown to oscillate and correlate with chilling tolerance. These genes have a peak of expression when most needed, just before dusk. These results illustrate the importance of the clock in compartmentalizing metabolic events to occur rhythmically at most advantageous time of the day (Harmer *et al.*, 2000, Schaffer *et al.*, 2001). The current challenge is the follow-up of all these data to unravel the clock-controlled gene regulatory networks and their connection to physiology and development.

Microarray experiments combined with computational analysis have also helped to identify common motifs in the upstream regions of co-regulated genes (Harmer *et al.*, 2000, Schaffer *et al.*, 2001). A nine base-pair sequence, denominated EVENING ELEMENT (*EE*) was found to be over-represented in a cluster of genes, whose expression peaked at the end of the day. Mutational analysis confirmed the importance of the *EE* in the oscillatory expression of *COLD, CIRCADIAN RHYTHM 2 (CCR2)* and in *CATALASE 3 (CAT3)* (Harmer *et al.*, 2000, Michael and McClung, 2002). A region of the *CCR2* promoter containing the *EE* was shown to confer circadian rhythmicity to the *β*-GLUCURONIDASE reporter gene. The sequence of the *EE* (AAAATATCT) is identical to the one recognized in the *TOC1* promoter by the transcriptional repressors CCA1 and LHY. The sequence is also highly similar to the one present in the *CAB* promoter where CCA1 binds acting as a positive regulator. These results suggest that CCA1 and LHY might control the expression of genes with opposite phases.

### Circadian regulation of plant growth

The regulation of hypocotyl length is controlled by environmental signals (light and temperature) as well as by endogenous signaling pathways (e.g., gibberellin acid, brassinosteroids, ethylene, abscisic acid, cytokinins). In constant light conditions, the pattern of hypocotyl growth displays rhythmic pauses near subjective dawn with a rapid elongation at subjective dusk. This rhythmicity is entrained by light-dark cycles and its period was shown to be shortened in the *toc1-1* mutant, indicating that it is controlled by the circadian clock. TOC1 RNAi, *toc1-2* mutants and mutants that over-express either CCA1 or LHY exhibit long hypocotyl phenotypes. Mutations in several other clock-associated components (such as *ZTL, FKF1, ELF3, ELF4, GI, SRR1*) also cause both circadian and hypocotyl-length phenotypes. The

**Fig. 3. Correlation of CO protein and mRNA expression with the photoperiodic control of Arabidopsis flowering time.** CO mRNA levels are increased under long-days by the action of FKF1 protein that functions as a photoreceptor. Under long-days, the CO mRNA peak of expression at the end of the day is abolished in the *fkf1* mutant plants (dotted line). The solid line indicates CO mRNA levels in wild-type plants. Blue and far-red light stabilize CO protein by the action of CRYs and PHYA. PHYB antagonizes the activity of CRYs and PHYA, especially in the morning. The combination of these regulatory activities results in robust floral promotion in long-days.altered hypocotyl length in these clock mutants was shown to be due to severe circadian defects on the regulation of cell expansion (Dowson-Day and Millar, 1999). Interestingly, the correlation between the clock and hypocotyl length is not absolute; the *toc1-1* mutant exhibits a short circadian period but does not have a hypocotyl phenotype. The fact that *toc1-1* mutant plants shorten multiple circadian outputs but still have wild-type hypocotyl length suggests that hypocotyl elongation during seedling de-etiolation could be controlled by different routes, dependent and independent of the clock. *toc1-1* represents a mutation with affected clock function but with normal light-mediated de-etiolation response independent of the clock.

The role of the clock in the regulation of hypocotyl growth has been proposed to be based on the "gating" of the light signaling pathways in a similar way to the gated induction of CAB expression. When the gate is closed, the light signaling pathways are inhibited, even in the presence of light. This repression leads to the growth of hypocotyls. When the gate is open, the clock allows the light input signals, inhibiting hypocotyl elongation. As described above, ELF3 has an essential function in controlling this gating mechanism. Accordingly, loss-of-function *elf3* mutant alleles result in a loss of rhythmic hypocotyl growth.

### Photoperiodic regulation of flowering time: a role for the circadian clock

A crucial step of the plant life cycle is the transition from a vegetative stage to a reproductive mode (Mouradov *et al.*, 2002, Searle and Coupland, 2004). Initiation of flowering occurs as a response to a number of environmental signals, including seasonal changes in day-length (Hayama and Coupland, 2004). The role of the circadian clock in the photoperiodism is illustrated by the fact that several mutants identified in *Arabidopsis* on the basis of their defective photoperiodic regulation of flowering also display altered circadian rhythms (e.g. ELF3, Hicks *et al.*, 1996; LHY, Schaffer *et al.*, 1998; GI, Fowler *et al.*, 1999, Park *et al.*, 1999; ELF4, Doyle *et al.*, 2002). Mutants originally isolated for their circadian defects, such as TOC1 (Millar *et al.*, 1995, Somers *et al.*, 1998b, Strayer *et al.*, 2000) and ZTL (Somers *et al.*, 2000), also showed a reduced sensitivity to day-length. Recently, an important progress has been made in the molecular understanding the circadian clock function in the photoperiodic control of flowering time. These studies set up the basis for a molecular understanding of the interactions between the circadian clock and the developmental control of flowering (Hayama and Coupland, 2004, Yanovsky and Kay, 2003). One of the key genes mediating the transition to flowering in *Arabidopsis* is *CONSTANS* (*CO*) (Putterill *et al.*, 1995). Analysis of *CO* expression reveals a rhythmic oscillation of both transcript and protein (Hayama and Coupland, 2003). An early flowering phenotype is observed in *CO* over-expressing plants although no circadian phenotypes are identified, indicating that *CO* is a clock output and mediates between the circadian system and initiation of flowering (Hayama and Coupland, 2003). The gene encodes a nuclear protein that contains a *CCT* motif (also found in *TOC1* and in *CO*-like genes) and two B-box type zinc-finger domains, believed to mediate protein-protein interaction. Under long days, the peak of *CO* expression is at the end of the day and during the night. A different pattern is observed under short-day conditions with a majority of

$CO$ expression during the night (Figure 3). The coincidence of higher levels of $CO$ with the light period has been proposed to be essential for the induction of flowering in long-days (Suárez-López *et al.*, 2001). The mechanism of CO action seems to be related with its involvement in *FLOWERING LOCUS T* (*FT*) activation. *FT* is an essential gene triggering flowering in *Arabidopsis* and its expression has been tightly linked to the presence of active CO (Kardailsky *et al.*, 1999, Kobayashi *et al.*, 1999, Samach *et al.*, 2000). The importance of $CO$ phase of expression has been demonstrated in the *toc1-1* mutant (with a short circadian period of 21 h). $CO$ expression in this mutant is phase-shifted such that expression occurs during the light period in both long- and short-days. This pattern of $CO$ expression results in a photoperiodic-insensitive early flowering. However, the photoperiodic control is restored when the *toc1-1* mutant plants are grown under light:dark cycles that match *toc1-1* endogenous period of 21 hours (Yanovsky and Kay, 2002). These studies reveal the importance of the clock in controlling the phase of $CO$ expression. The coincidence of light with the $FT$ activation by $CO$ is a key event in the photoperiodic control of flowering time (Roden *et al.*, 2002, Yanovsky and Kay, 2003) and two photoreceptors, PHYA and CRY2, have been shown to be involved in this process (Yanovsky and Kay, 2002).

Recent studies have also provided new insights into the mechanism that generates the diurnal pattern of $CO$ transcription. In the *fkf1* mutant, the high levels of $CO$ mRNA are strongly reduced and the daytime CO peak is abolished. Under long-days, FKF1 protein levels exhibit a diurnal pattern with a peak in the late day. However, under short-days the peak protein expression occurs during the early to mid nighttime, with low FKF1 protein levels during the day (Imaizumi *et al.*, 2003). A model was proposed suggesting that under long-days, high levels of FKF1 protein and its direct activation by light occur simultaneously, generating the daytime peak of $CO$ mRNA.

Analysis of CO over-expressing plants revealed that CO protein accumulates under continuous white light, whereas $CO$ levels are strongly reduced in constant darkness (Valverde *et al.*, 2004). The dark-dependent instability of CO protein is due to an active degradation of CO protein by the proteosome. Furthermore, CO protein accumulates to high levels in continuous blue and far-red light, while the protein is rapidly degraded under constant red light. Under long-day conditions, CO protein shows a strong peak at the end of the day, whereas under short-days the protein displays a much weaker peak of expression at the early nighttime (Valverde *et al.*, 2004). The regulation of CO protein abundance during the day seems to be mediated by PHYB, which decreases CO protein levels early in the morning. In contrast, PHYA and CRYs stabilize CO protein at the end of the day (Figure 3). These studies led to the conclusion that PHYB promotes CO protein degradation and antagonizes the action of PHYA and CRYs in the morning (Hayama and Coupland, 2004).

Gene homologous to $CO$ and $FT$ have been identified in many species suggesting a conservation in the components of the *Arabidopsis* photoperiod pathway (Liu *et al.*, 2001a, Yano *et al.*, 2000, Kojima *et al.*, 2002, Griffiths *et al.*, 2003). Molecular and mechanistic studies in other plant species might help us to understand how the diversity in photoperiodic pathways was generated during evolution of different photoperiodic responses in plants.
498 P. Más

# Perspectives

The precise control of gene expression by the circadian clock modulates the rhythmicity of physiological and developmental processes that are essential for plant survival. The clock is able to precisely recognize the time of the day in the light/dark cycle, allowing the anticipation of the environmental transitions. Elegant experiments performed in bacteria (Ouyang *et al.*, 1998) and more recently in plants (Green *et al.*, 2002, Michael *et al.*, 2003b) have shown that this anticipation provides an adaptive advantage and increases the fitness of the organisms. The *Arabidopsis* circadian oscillator might be comprised by multiple interlocked feedback loops that generate and maintain circadian rhythmicity through a dynamic signaling network. Identifying clock components, their different levels of regulation and the interaction among them will be crucial in understanding mechanisms of action and influences of the clock on all circadianly-regulated processes. The first steps are already taken. New tools and experimental approaches will help to assemble the pieces into place and solve the intricate puzzle of the *Arabidopsis* circadian clock.

## Acknowledgments

I thank T. Stratmann for critical reading of the manuscript. Our work on circadian rhythms is supported by grants (BIO2004-02144) from Spanish MEC and (FP6-012239) from EC.

## References

ALABADÍ, D., OYAMA, T., YANOVSKY, M.J., HARMON, F.G., MÁS, P. and KAY, S.A. (2001). Reciprocal regulation between *TOC1* and *LHY/CCA1* within the *Arabidopsis* circadian clock. *Science* 293: 880-883.

ALABADÍ, D., YANOVSKY, M.J., MÁS, P., HARMER, S.L. and KAY, S.A. (2002). Critical role for *CCA1* and *LHY* in maintaining circadian rhythmicity in *Arabidopsis*. *Curr. Biol.* 12: 757-761.

ANDERSSON, C.R., HARMER, S.L., SCHULTZ, T.F. and KAY, S.A. (1999). The REVEILLE (*REV*) family of DNA binding proteins and the circadian clock. *In:* Abstracts of the 10th international conference on *Arabidopsis* research, Melbourne, 4-8 July 1999. http://Arabidopsis.org/abstract_australia.pdf.

BARAK, S., TOBIN, E.M. and RONIS, C., SUGANO, S. and GREEN, R.M. (2000). All in good time: The *Arabidopsis* circadian clock. *Trends Plant Sci.* 5: 517-522.

COVINGTON, M.F., PANDA, S., LIU, X.L., STRAYER, C.A., WAGNER, D.R. and KAY, S.A. (2001). *ELF3* modulates resetting of the circadian clock in *Arabidopsis*. *Plant Cell* 13: 1305-1315.

CRAIG, K.L. and TYERS, M. (1999). The F-box: A new motif for ubiquitin dependent proteolysis in cell cycle regulation and signal transduction. *Prog. Biophys. Mol. Biol.* 72: 299-328.

DANIEL, X., SUGANO, S. and TOBIN, E.M. (2004). CK2 phosphorylation of *CCA1* is necessary for its circadian oscillator function in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA* 101: 3292-7.

DEVLIN, P.F. and KAY, S.A. (2000). Cryptochromes are required for phytochrome signaling to the circadian clock but not for rhythmicity. *Plant Cell* 12: 2499-2510.

DEVLIN, P.F. and KAY, S.A. (2001). Circadian photoperception. *Annu. Rev. Physiol.* 63: 677-694.

DOWSON-DAY, M.J. and MILLAR, A.J. (1999). Circadian dysfunction causes aberrant hypocotyl elongation patterns in *Arabidopsis*. *Plant J.* 17: 63-71.

DOYLE, M.R., DAVIS, S.J., BASTOW, R.M., MCWATTERS, H.G., KOZMA-BOGNAR, L., NAGY, F., MILLAR, A.J. and AMASINO, R.M. (2002). The *ELF4* gene controls circadian rhythms and flowering time in *Arabidopsis thaliana*. *Nature* 419: 74-7.

DUNLAP, J.C. (1999). Molecular bases for circadian clocks. *Cell* 96: 271-290.

DUNLAP, J.C. (2004). Kinases and circadian clocks: Per goes it alone. *Development*

tal Cell 6: 160-161.

EDERY, I. (1999). Role of posttranscriptional regulation in circadian clocks: Lessons from *Drosophila*. *Chronobiol. Int.* 16: 377-414.

ERIKSSON, M.E. and MILLAR, A.J. (2003). The circadian clock. A plant's best friend in a spinning world. *Plant Physiol.* 132: 732-8.

FOWLER, S., LEE, K., ONOUCHI, H., SAMACH, A., RICHARDSON, K., MORRIS, B., COUPLAND, G. and PUTTERILL, J. (1999). *GI/GIGANTEA*: A circadian clock-controlled gene that regulates photoperiodic flowering in *Arabidopsis* and encodes a protein with several possible membrane-spanning domains. *EMBO J.* 18: 4679-88.

GREEN, R.M., TINGAY, S., WANG, Z.Y. and TOBIN, E.M. (2002). Circadian rhythms confer a higher level of fitness to *Arabidopsis* plants. *Plant Physiol.* 129: 576-84.

GREEN, R.M. and TOBIN, E.M. (1999). Loss of the CIRCADIAN CLOCK-ASSOCIATED PROTEIN 1 in *Arabidopsis* results in altered clock-regulated gene expression. *Proc. Natl. Acad. Sci. USA* 96: 4176-9.

GRIFFITHS, S., DUNFORD, R.P., COUPLAND, G. and LAURIE, D.A. (2003). The evolution of CONSTANS-LIKE gene families in barley, rice and *Arabidopsis*. *Plant Physiol.* 131: 1855-1867.

GRIMA, B., LAMOUROUX, A., CHELOT, E., PAPIN, C., LIMBOURG-BOUCHON, B. and ROUYER, F. (2002). The F-box protein slimb controls the levels of clock proteins period and timeless. *Nature* 420: 178-182.

HALL, A., BASTOW, R.M., DAVIS, S.J., HANANO, S., MCWATTERS, H.G., HIBBERD, V., DOYLE, M.R., SUNG, S., HALLIDAY, K.J., AMASINO, R.M. (2003). The *TIME FOR COFFEE* gene maintains the amplitude and timing of *Arabidopsis* circadian clocks. *Plant Cell* 15: 2719-2729.

HALL, A., KOZMA-BOGNAR, L., BASTOW, R.M., NAGY, F. and MILLAR, A.J. (2002). Distinct regulation of *CAB* and *PHYB* gene expression by similar circadian clocks. *Plant J.* 32: 529-37.

HALLIDAY, K.J. and WHITELAM, G.C. (2003). Changes in photoperiod or temperature alter the functional relationships between phytochromes and reveal roles for PHYD and PHYE. *Plant Physiol.* 131: 1913-1920.

HARMER, S.L., HOGENESCH, J.B., STRAUME, M., CHANG, H., HAN, B., ZHU, T., WANG, X., KREPS, J.A. and KAY, S.A. (2000). Orchestrated transcription of key pathways in *Arabidopsis* by the circadian clock. *Science* 290: 2110-2113.

HARMER, S.L., PANDA, S. and KAY, S.A. (2001). Molecular bases of circadian rhythms. *Annu. Rev. Cell Dev. Biol.* 17: 215-253.

HAYAMA, R. and COUPLAND, G. (2003). Shedding light on the circadian clock and the photoperiodic control of flowering. *Curr. Opin. Plant Biol.* 6: 13-9.

HAYAMA, R. and COUPLAND, G. (2004). The molecular basis of diversity in the photoperiodic flowering responses of *Arabidopsis* and rice. *Plant Physiol.* 135: 677-84.

HICKS, K.A., ALBERTSON, T.M. and WAGNER, D.R. (2001). EARLY FLOWERING 3 encodes a novel protein that regulates circadian clock function and flowering in *Arabidopsis*. *Plant Cell* 13: 1281-1292.

HICKS, K.A., MILLAR, A.J., CARRÉ, I.A., SOMERS, D.E., STRAUME, M.D., MEEKS-WAGNER, R. and KAY, S.A. (1996). Conditional circadian dysfunction of the *Arabidopsis early-flowering 3* mutant. *Science* 274: 790-792.

HUQ, E., TEPPERMAN, J.M. and QUAIL, P.H. (2000). *GI/GIGANTEA* is a nuclear protein involved in phytochrome signaling in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA* 97: 9789-9794.

IMAIZUMI, T., TRAN, H.G., SWARTZ, T.E., BRIGGS, W.R. and KAY, S.A. (2003). FKF1 is essential for photoperiodic-specific light signalling in *Arabidopsis*. *Nature* 426: 302-6.

JARILLO, J.A., CAPEL, J., TANG, R.H., YANG, H.Q., ALONSO, J.M., ECKER, J.R. and CASHMORE, A.R. (2001). An *Arabidopsis* circadian clock component interacts with both CRY1 and PHYB. *Nature* 410: 487-490.

JOHNSON, C.H. (2001). Endogenous timekeepers in photosynthetic organisms. *Annu Rev Physiol* 63: 695-728.

KARDAILSKY, I., SHUKLA, V.K., AHN, J.H., DAGENAIS, N., CHRISTENSEN, S.K., NGUYEN, J.T., CHORY, J., HARRISON, M.J. and WEIGEL, D. (1999). Activation tagging of the floral inducer FT. *Science* 286: 1962-1965.

KIM, J.Y., SONG, H.R., TAYLOR, B.L. and CARRE, I.A. (2003a). Light-regulated translation mediates gated induction of the *Arabidopsis* clock protein *LHY*. *EMBO J* 22: 935-944.

KIM, W.-Y., GENG, R. and SOMERS, D.E. (2003b). Circadian phase-specific degra-

dation of the F-box protein ZTL is mediated by the proteasome. *Proc. Natl. Acad. Sci. USA* 100: 4933-4938.

KO, H.W., JIANG, J. and EDERY, I. (2002). Role for SLIMB in the degradation of *Drosophila* PERIOD protein phosphorylated by DOUBLETIME. *Nature* 420.: 673-678.

KOBYASHI, Y., KAYA, H., GOTO, K., IWABUCHI, M. and ARAKI, T. (1999). A pair of related genes with antagonistic roles in mediating flowering signals. *Science* 286: 1960-1962.

KOJIMA, S., TAKAHASHI, Y., KOBYASHI, Y., MONNA, L., SASAKI, T., ARAKI, T. and YANO, M. (2002). Hd3a, a rice ortholog of the *Arabidopsis FT* gene, promotes transition to flowering downstream of Hd1 under short-day conditions. *Plant Cell Physiol.* 43: 1096-105.

LIU, J., YU, J., MCINTOSH, L., KENDE, H. and ZEEVAART, J.A. (2001a). Isolation of a CONSTANS ortholog from *Pharbitis nil* and its role in flowering. *Plant Physiol.* 125: 1821-30.

LIU, X.L., COVINGTON, M.F., FANKHAUSER, C., CHORY, J. and WAGNER, D.R. (2001b). ELF3 encodes a circadian clock-regulated nuclear protein that functions in an *Arabidopsis* PHYB signal transduction pathway. *Plant Cell* 13: 1293-1304.

LOROS, J.J. and DUNLAP, J.C. (2001). Genetic and molecular analysis of circadian rhythms in *Neurospora*. *Annu. Rev. Physiol.* 63: 757-794.

MÁS, P., ALABADI, D., YANOVSKY, M.J., OYAMA, T. and KAY, S.A. (2003a). Dual role of TOC1 in the control of circadian and photomorphogenic responses in *Arabidopsis*. *Plant Cell* 15: 223-236.

MÁS, P., DEVLIN, P.F., PANDA, S. and KAY, S.A. (2000). Functional interaction of PHYTOCHROME B and CRYPTOCHROME 2. *Nature* 408: 207-211.

MÁS, P., KIM, W.J., SOMERS, D.E. and KAY, S.A. (2003b). Targeted degradation of TOC1 by ZTL modulates circadian function in *Arabidopsis*. *Nature* 426: 567-570.

MATUSHIKA, A., MAKINO, S., KOJIMA, M. and MIZUNO, T. (2000). Circadian waves of expression of the APRR1/TOC1 family of pseudo-response regulators in *Arabidopsis thaliana*: Insight into the plant circadian clock. *Plant Cell Physiol.* 41: 1002-1012.

MAZZELLA, M.A., BERTERO, D. and CASAL, J.J. (2000). Temperature-dependent internode elongation in vegetative plants of *Arabidopsis thaliana* lacking PHYTO-CHROME B and CRYPTOCHROME 1. *Planta* 210: 497-501.

MCCLUNG, C.R. (2001). Circadian rhythms in plants. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* 52: 139-162.

MCWATTERS, H.G., BASTOW, R.M., HALL, A. and MILLAR, A.J. (2000). The ELF3 zeitnehmer regulates light signaling to the circadian clock. *Nature* 408: 716-20.

MICHAEL, T.P. and MCCLUNG, C.R. (2002). Phase-specific circadian clock regulatory elements in *Arabidopsis*. *Plant Physiol.* 130: 627-38.

MICHAEL, T.P., SALOME, P.A. and MCCLUNG, C.R. (2003a). Two *Arabidopsis* circadian oscillators can be distinguished by differential temperature sensitivity. *Proc. Natl. Acad. Sci. USA* 100: 6878-83.

MICHAEL, T.P., SALOME, P.A., YU, H.J., SPENCER, T.R., SHARP, E.L., MCPEEK, M.A., ALONSO, J.M., ECKER, J.R. and MCCLUNG, C.R. (2003b). Enhanced fitness conferred by naturally occurring variation in the circadian clock. *Science* 302: 1049-53.

MILLAR, A.J. (2004). Input signals to the plant circadian clock. *J Exp Bot* 55: 277-83.

MILLAR, A.J., CARRE, I.A., STRAYER, C.A., CHUA, N.H. and KAY, S.A. (1995). Circadian clock mutants in *Arabidopsis* identified by luciferase imaging. *Science* 267: 1161-3.

MILLAR, A.J. and KAY, S.A. (1996). Integration of circadian and phototransduction pathways in the network controlling cab gene transcription in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA* 93: 15491-6.

MIZOGUCHI, T., WHEATLEY, K., HANZAWA, Y., WRIGHT, L., MIZOGUCHI, M., SONG, H.R., CARRE, I.A. and COUPLAND, G. (2002). *LHY* and *CCA1* are partially redundant genes required to maintain circadian rhythms in *Arabidopsis*. *Developmental Cell* 2: 629-641.

MIZUNO, T. (2004). Plant response regulators implicated in signal transduction and circadian rhythm. *Curr. Opin. Plant Biol.* 7: 499-505.

MOURADOV, A., CREMER, F. and COUPLAND, G. (2002). Control of flowering time: Interacting pathways as a basis for diversity. *Plant Cell* 14 Suppl: S111-30.

NELSON, D.C., LASSWELL, J., ROGG, L.E., COHEN, M.A. and BARTEL, B. (2000). FKF1, a clock-controlled gene that regulates the transition to flowering in *Arabidopsis*. *Cell* 101: 331-340.

OUYANG, Y. and ERSSON, C.R., KONDO, T., GOLDEN, S.S. and JOHNSON, C.H. (1998). Resonating circadian clocks enhance fitness in cyanobacteria. *Proc. Natl. Acad. Sci. USA* 95: 8660-8664.

PARK, D.H., SOMERS, D.E., KIM, Y.S., CHOY, Y.H., LIM, H.K., SOH, M.S., KIM, H.J., KAY, S.A. and NAM, H.G. (1999). Control of circadian rhythms and photoperiodic flowering by the *Arabidopsis gigantea* gene. *Science* 285: 1579-1582.

PUTTERILL, J., ROBSON, F., LEE, K., SIMON, R. and COUPLAND, G. (1995). The CONSTANS gene of *Arabidopsis* promotes flowering and encodes a protein showing similarities to zinc finger transcription factors. *Cell* 80: 847-57.

REED, J.W., NAGPAL, P., BASTOW, R.M., SOLOMON, K.S., DOWSON-DAY, M.J., ELUMALAI, R.P. and MILLAR, A.J. (2000). Independent action of ELF3 and PHYB to control hypocotyl elongation and flowering time. *Plant Physiol.* 122: 1149-60.

RODEN, L.C., SONG, H.R., JACKSON, S., MORRIS, K. and CARRE, I.A. (2002). Floral responses to photoperiod are correlated with the timing of rhythmic expression relative to dawn and dusk in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA* 99: 13313-8.

SAMACH, A., ONOUCHI, H., GOLD, S.E., DITTA, G.S., SCHWARZ-SOMMER, Z., YANOFFSKY, M.F. and COUPLAND, G. (2000). Distinct roles of CONSTANS target genes in reproductive development of *Arabidopsis*. *Science* 288: 1613-1616.

SCHAFFER, R., LANDGRAF, J., ACCERBI, M., SIMON, V., LARSON, M. and WISMAN, E. (2001). Microarray analysis of diurnal and circadian-regulated genes in *Arabidopsis*. *Plant Cell* 13: 113-23.

SCHAFFER, R., RAMSAY, N., SAMACH, A., CORDEN, S., PUTTERILL, J., CARRÉ, I.A. and COUPLAND, G. (1998). The late elongated hypocotyl mutation of *Arabidopsis* disrupts circadian rhythms and the photoperiodic control of flowering. *Cell* 93: 1219-1229.

SCHULTZ, T.F., KIYOSUE, T., YANOVSKY, M., WADA, M. and KAY, S.A. (2001). A role for LKP2 in the circadian clock of *Arabidopsis*. *Plant Cell* 13: 2659-70.

SEARLE, I. and COUPLAND, G. (2004). Induction of flowering by seasonal changes in photoperiod. *EMBO J.* 23: 1217-22.

SOMERS, D.E., DEVLIN, P.F. and KAY, S.A. (1998a). Phytochromes and cryptochromes in the entrainment of the *Arabidopsis* circadian clock. *Science* 282: 1488-1490.

SOMERS, D.E., SCHULTZ, T.F., MILNAMOW, M. and KAY, S.A. (2000). ZEITLUPE encodes a novel clock-associated PAS protein from *Arabidopsis*. *Cell* 101: 319-329.

SOMERS, D.E., WEBB, A.A.R., PEARSON, M. and KAY, S.A. (1998b). The short-period mutant *toc1-1*, alters circadian clock regulation of multiple outputs throughout development in *Arabidopsis thaliana*. *Development* 125: 485-494.

STAIGER, D., ALLENBACH, L., SALATHIA, N., FIECHTER, V., DAVIS, S.J., MILLAR, A.J., CHORY, J. and FANKHAUSER, C. (2003). The *Arabidopsis SRR1* gene mediates PHYB signaling and is required for normal circadian clock function. *Genes Dev.* 17: 256-268.

STRAYER, C.A., OYAMA, T., SCHULTZ, T.F., RAMAN, R., SOMERS, D.E., MÁS, P., PANDA, S., KREPS, J.A. and KAY, S.A. (2000). Cloning of the *Arabidopsis* clock gene *TOC1*, an autoregulatory response regulator homolog. *Science* 289: 768-771.

SUÁREZ-LÓPEZ, P., WHEATLEY, K., ROBSON, F., ONOUCHI, H., VALVERDE, F. and COUPLAND, G. (2001). CONSTANS mediates between the circadian clock and the control of flowering in *Arabidopsis*. *Nature* 410: 1116-1120.

SUGANO, S. and RONIS, C., GREEN, R.M., WANG, Z.Y. and TOBIN, E.M. (1998). Protein kinase CK2 interacts with and phosphorylates the *Arabidopsis* circadian clock-associated 1 protein. *Proc. Natl. Acad. Sci. USA* 95: 11020-11025.

SUGANO, S. and RONIS, C., ONG, M.S., GREEN, R.M. and TOBIN, E.M. (1999). The protein kinase CK2 is involved in regulation of circadian rhythms in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA* 96: 12362-12366.

THAIN, S.C., MURTAS, G., LYNN, J.R., MCGRATH, R.B. and MILLAR, A.J. (2002). The circadian clock that controls gene expression in *Arabidopsis* is tissue specific. *Plant Physiol.* 130: 102-10.

TOH, K.L., JONES, C.R., HE, Y., EIDE, E.J., HINZ, W.A., VIRSHUP, D.M., PTACEK, L.J. and FU, Y.-H. (2001). An hPER2 phosphorylation site mutation in familial advanced sleep phase syndrome. *Science* 291: 1040-1043.

TSENG, T.-S., SALOME, P.A., MCCLUNG, C.R. and OLSZEWSKI, N.E. (2004). SPINDLY and GIGANTEA interact and act in *Arabidopsis thaliana* pathways involved in light responses, flowering and rhythms in cotyledon movements. *Plant*

Cell 16: 1550-1563.

VALVERDE, F., MOURADOV, A., SOPPE, W., RAVENSCROFT, D., SAMACH, A., and COUPLAND, G. (2004). Photoreceptor regulation of CONSTANS protein in photoperiodic flowering. *Science* 303: 1003-6.

WANG, Z.Y., KENIGSBUCH, D., SUN, L., HAREL, E., ONG, M.S. and TOBIN, E.M. (1997). A myb-related transcription factor is involved in the phytochrome regulation of an *Arabidopsis LHCB* gene. *Plant Cell* 9: 491-507.

WANG, Z.Y. and TOBIN, E.M. (1998). Constitutive expression of the circadian clock associated 1 (*CCA1*) gene disrupts circadian rhythms and suppresses its own expression. *Cell* 93: 1207-17.

YAGITA, K., TAMANINI, F., YASUDA, M., HOEIJMAKERS, J.H.J., VAN DER HORST, G.T.J. and OKAMURA, H. (2002). Nucleocytoplasmic shuttling and mCRY-dependent inhibition of ubiquitylation of the mPER2 clock protein.

EMBO J. 21: 1301-1314.

YANO, M., KATAYOSE, Y., ASHIKARI, M., YAMANOUCHI, U., MONNA, L., FUSE, T., BABA, T., YAMAMOTO, K., UMEHARA, Y., NAGAMURA, Y. (2000). Hd1, a major photoperiod sensitivity quantitative trait locus in rice, is closely related to the *Arabidopsis* flowering time gene CONSTANS. *Plant Cell* 12: 2473-2484.

YANOVSKY, M.J. and KAY, S.A. (2002). Molecular basis of seasonal time measurement in *Arabidopsis*. *Nature* 419: 308-12.

YANOVSKY, M.J. and KAY, S.A. (2003). Living by the calendar: How plants know when to flower. *Nat. Rev. Mol. Cell Biol.* 4: 265-275.

YANOVSKY, M.J., MAZZELLA, M.A. and CASAL, J.J. (2000). A quadruple photoreceptor mutant still keeps track of time. *Curr. Biol.* 10: 1013-1015.

YOUNG, M.W. and KAY, S.A. (2001). Time zones: A comparative genetics of circadian clocks. *Nat. Rev. Gen.* 2: 702-715.
