
Molecular Membrane Biology, January-April 2005; 22(1-2): 51-61

**Structural and dynamic properties of bacterial Type IV secretion systems (Review)**

PETER J. CHRISTIE & ERIC CASCALES

Department of Microbiology and Molecular Genetics, UT-Houston Medical School, Houston, Texas, USA

(Received 22 November 2004, and in revised form 6 January 2005)

**Abstract**

The type IV secretion systems (T4SS) are widely distributed among the Gram-negative and -positive bacteria. These systems mediate the transfer of DNA and protein substrates across the cell envelope to bacterial or eukaryotic cells generally through a process requiring direct cell-to-cell contact. Bacteria have evolved T4SS for survival during establishment of pathogenic or symbiotic relationships with eukaryotic hosts. The *Agrobacterium tumefaciens* VirB/D4T4SS and related conjugation machines serve as models for detailed mechanistic studies aimed at elucidating the nature of translocation signals, machine assembly pathways and architectures, and the dynamics of substrate translocation. The *A. tumefaciens* VirB/D4T4SS are polar-localized organelles composed of a secretion channel and an extracellular T pilus. These T4SS are assembled from 11 or more subunits, whose membrane topologies, intersubunit contacts and, in some cases, 3-dimensional structures are known. Recently, powerful *in vivo* assays have identified C-terminal translocation signals, defined for the first time the translocation route for a DNA substrate through a type IV secretion channel, and supplied evidence that ATP energy consumption contributes to a late stage of machine morphogenesis. Together, these recent findings describe the mechanics of type IV secretion in unprecedented detail.

**Keywords:** Conjugation, pathogenesis, traffic ATPases, macromolecular transport, DNA transfer

**Introduction**

The bacterial type IV secretion systems (T4SS) comprise a large family of translocation systems dedicated to the movement of DNA or protein substrates across the cell envelope [1, 2]. The T4SS are broadly distributed among most species of Gram-negative and -positive bacteria [1, 3]. In one classification scheme, the T4SS have been grouped according to sequence relatedness of machine components. Accordingly, systems assembled from proteins related to the archetypal VirB/D4T4SS of *Agrobacterium tumefaciens* are classified as type IVA and those related to the Dot/IcmT4SS of *Legionella pneumophila* are classified as type IVB [4]. Other T4SS bearing little or no sequence relationships with the types IVA and IVB systems, e.g., those encoded by conjugative transposons and plasmids of Gram-positive bacteria [3], will be classified once their subunit compositions become better defined. The T4SS can also be grouped according to function. A large number of T4SS mediate the conjugative transfer of plasmids and other mobile DNA elements to bacterial recipient cells. Others, designated as the 'effector translocator' systems, deliver protein or DNA substrates to eukaryotic target cells. The conjugation systems and most effector translocator systems deliver their substrates through direct contact with target cells. By contrast, a third subgroup termed 'DNA release or uptake' systems exchange DNA substrates with the extracellular milieu by contact-independent mechanisms [1]. To date, most structure-function studies have focused on the type IVA conjugation systems, although rapid progress is being made on type IVA and IVB effector translocator systems of medically important pathogens. Various aspects of T4SS biology have been covered in recent excellent reviews [2, 5–9]. Here, we will focus mainly on the type IVA T4SS, first summarizing novel systems identified in the past year whose functions reinforce the concept that these secretion machines have evolved for a wide range of biological activities. The remainder of the review will highlight recent progress toward answering long-standing questions of how and where at the cell surface the T4SS assemble, how T4SS recognize and recruit secretion substrates, and how type IV secretion substrates traverse the Gram-negative cell envelope.

---

Correspondence: Peter J. Christie, Department of Microbiology and Molecular Genetics, UT-Houston Medical School, 6431 Fannin, Houston, Texas 77030, USA. Tel: +1 713 500 5440. Fax: 713 500 5499. E-mail: Peter.J.Christie@uth.tmc.edu

ISSN 0968-7688 print/ISSN 1464-5203 online © 2005 Taylor & Francis  
DOI:10.1080/09687860500063316

Type IV family members

The *A. tumefaciens* VirB/D4 T4SS will serve as a framework for this discussion. The VirB/D4 T4SS is a *bona fide* bacterial conjugation apparatus adapted for the novel purpose of delivering an oncogenic T-DNA substrate as well as effector proteins to susceptible plant cells [10]. The ~12-kilobase *virB* operon encodes 11 proteins, VirB1 through VirB11, and the *virD* operon encodes 5 proteins including VirD4 [11]. The VirB and VirD4 proteins together elaborate the transenvelope secretion channel, whereas independently of VirD4 the VirB proteins elaborate an extracellular filament termed the T pilus [12]. The VirB proteins are termed the mating pair formation (Mpf) proteins, and VirD4 is termed the coupling protein (T4CP). The T-DNA is processed for transfer at the T-DNA border repeat sequences by the VirD2 relaxase and auxiliary factors, collectively termed the DNA transfer and replication (Dtr) proteins. The *A. tumefaciens* Dtr, T4CP, and Mpf subunits are related in sequence and function to subunits of conjugation systems, and the T-DNA borders to origin-of-transfer (*oriT*) sequences, of Gram-negative bacterial plasmids, e.g., F, RP4, R388, and pKM101 [6, 13].

Many T4SS have been shown to deliver DNA and/or protein substrates to bacterial or eukaryotic target cells. In the past year, this list was expanded to include T4SS carried by *Mesorhizobium loti* [14], *Burkholderia cenocepacia* [15], *Xanthomonas axonopodis pv. citri* [16], and *Campylobacter jejuni* and *C. coli* [17]. The *M. loti* and *B. cenocepacia* T4SS contribute to the colonization of plant host cells. In *M. loti* strain R27, the 11 *virB* and *virD4* genes are carried on a symbiosis island. In addition to extensive similarities in regulation, operon structure, and subunit composition, the *M. loti* strain R27 and *A. tumefaciens* VirB/D4 T4SS export functionally redundant effector proteins to plant cells. For example, *M. loti* exports the effector protein Msi061 and *A. tumefaciens* exports VirF. Both effectors possess F-box motifs that, upon delivery into the plant cell, are postulated to mimic the activities of eukaryotic F-box receptors for attracting specific proteins to the SCF (Skp1-Cdc53-F-box protein) complexes for ubiquitination and proteolysis [14, 18]. The *A. tumefaciens* VirB/D4 T4SS also was shown to translocate *M. loti* Msi061 to plant cells, suggesting that the *A. tumefaciens* and *M. loti* VirB/D4 T4SS recognize common translocation signals [14]. Interestingly, through adaptation of these closely related T4SS, *A. tumefaciens* induces a pathogenic response whereas *M. loti* establishes a symbiotic relationship with the eukaryotic host.

*B. cenocepacia* strain K56-2 carries two T4SS, the Ptw system encoded by plasmid pK56-2 and a second encoded by chromosome II [15]. The Ptw system is composed of homologs of VirB4, VirB6 through VirB11, and VirD4, as well as homologs of the F plasmid TraN and TraU subunits and a homolog of the R388 plasmid TrwC relaxase. The chromosomal T4SS is highly similar in genetic composition and arrangement to the *A. tumefaciens* VirB/D4 T4SS, with the exception that *virD4* is probably part of the same transcriptional unit as the 11 *virB* genes. Mutational studies provided evidence for the involvement of the Ptw system in eliciting the plant tissue water-soaking phenotype [15]. *Erwinia amylovora* and *Pantoea stewartii* subsp. *stewartii* also induce a plant water-soaking phenotype, but both of these species are thought to export the elicitors DspE and WtsE, respectively, to plant cells via type III secretion systems (T3SS) (see [15]). *B. cenocepacia*, *E. amylovora* and *P. stewartii* thus appear to have coopted either T3SS or T4SS to elicit a common pathogenic outcome on susceptible plant hosts.

A very recent study supplied compelling evidence that *Helicobacter pylori* activates the NF-κB signaling pathway through delivery of peptidoglycan to gastric epithelial cells [19]. The host intracellular protein Nod1 is involved in epithelial cell sensing of peptidoglycan as well as recognition of *cagPAI*-positive *H. pylori*. Though not yet shown directly, these findings suggest that *H. pylori* Cag T4SS exports not only the previously-identified effector protein CagA, but also peptidoglycan [19]. If this turns out to be true, the type IV substrate repertoire now includes a non-protein or non-DNA signaling molecule.

Type IV machine components: Cellular locations, interaction networks, and structures

At this time, there is no high-resolution structure for any T4SS analogous to that recently presented for a T3SS [20]. Nevertheless, fairly comprehensive architectural models of the T4SS can be generated through topological, structural, and interaction studies of machine components. This combination of approaches supplies evidence for at least three stable subassemblies of VirB/D4 components (Figure 1).

Energy subcomplex – VirD4, VirB4, VirB11

VirD4 T4CP, VirB11, and VirB4 are the three energetic components of the VirB/D4 T4SS. Each of these subunits possesses a characteristic nucleoside triphosphate binding site (Walker A) motif required for substrate translocation [7]. Homologs of each subunit bind ATP, but ATP hydrolysis

Mechanistic features of Type IV secretion 53

![Diagram](attachment:diagram.png)

Figure 1. Protein interaction networks between type IV secretion subunits. The localization and topologies of the components of the archetypal *Agrobacterium tumefaciens* VirB/D4 T4SS are represented. Shaded areas correspond to subcomplexes – energy (yellow shading), core (cream), pilus (green), F plasmid-specific (tan) – described in the text. Arrows indicate interactions detected among the subunits by biochemical/structural approaches – (co-immunoprecipitation, co-purification, crystal structure; blue arrow), yeast two-hybrid or peptide linkage (red arrow) and lambda cI repressor fusion or bacterial two hybrid (green arrow). The letter(s) accompanying the arrow denotes the system where the binary interaction was detected (*A*, *A. tumefaciens*; *B*, *Bartonella henselae*; *Hp*, *Helicobacter pylori*; *P*, *Bordetella pertussis*; *R*, *Rickettsia sibirica*; *Xa*, *Xanthomonas axonopodis* pv. *citri*; *W*, *N*, *F*, *X* and *H*, IncW, IncN, IncF, IncX and IncH plasmid incompatibility group conjugation systems). Interactions detected by two-hybrid screens between components localizing in different compartments are not shown, e.g., VirB11-VirB1, VirB11-VirB10 C-terminal domain, VirB11-VirB8 periplasmic domain, VirB11-VirB9. The F-plasmid transfer system Tra/Trb components are probably involved in pilus extension/retraction (*F*, *H*, *U*, *W* represent TraF, TraH, TraU and TraW; *B* and *I* represent TrbB and TrbI). For studies reporting these interactions, see refs. [6,7,16,24,28,29,31–35,38–40,44,46,50,69].

activity has been demonstrated convincingly only for homologs of VirB11 (*H. pylori* HP0525, RP4 TrbB, R388 TrwD, R64 PilQ, *L. pneumophila* DotB) [21–23]. Members of the VirD4 T4CP family form ring structures discernible by electron microscopy. An X-ray structure solved for a soluble domain of TrwB<sub>R388</sub> presents as a homohexameric ring with a central channel of 20 Å in diameter. The structure bears a striking resemblance to the F1-ATPase α3β3 heterohexamer [24]. The VirB11 homologs TrbB<sub>RP4</sub>, TrwD<sub>R388</sub>, and *H. pylori* HP0525<sub>Cag</sub> also form homohexameric rings. The double-stacked ring of HP0525 seen by X-ray crystallography appears to be highly conserved, even among distantly-related ATPases associated with other transport or fimbrial biogenesis systems, e.g., *L. pneumophila* DotB of the Dot/Icm T4SS [22],

Aquifex aeolicus PilT associated with type IV pilus retraction [25], Vibrio cholerae EpsE associated with type II secretion [26], and Actinobacillus actinomyctemcomitans TadA associated with Flp pilus biogenesis [27]. Both EpsE and HP0525 are structurally similar to members of the AAA ATPase superfamily [9,26]. Little structural information is available for the VirB4 subunit family, with the exception that *A. tumefaciens* VirB4 self-interacts and its N terminus likely spans the inner membrane [28–30].

The three energetic subunits form pairwise interactions, as shown by two-hybrid interaction assays (Figure 1). For example, *A. tumefaciens* VirB4 interacts with VirB11 [29], *Rickettsa sibirica* VirB4 with VirD4 [31], and *Xanthomonas campestris* VirD4 with VirB11 [16]. Additionally, native and Walker A mutant forms of VirD4, VirB4, and

VirB11 interact, as shown by co-immunoprecipitation, suggesting that ATP binding or hydrolysis does not drive at least the initial assembly of these ternary complexes [32]. Several studies also have demonstrated VirD4–VirB10 and VirB4–VirB10 interactions [29, 31–36]. The physiological significance of the VirD4–VirB10 interaction is discussed below.

**Core subcomplex – VirB6, VirB7, VirB8, VirB9, VirB10**

Five VirB proteins are implicated as forming a ‘core’ transenvelope structure on the basis of phylogenetic relationships and cell localization and protein-protein interaction data [7]. Two recent advances in our understanding of this core complex include the development of a VirB6 topology model and evidence that VirB10 resembles the energy transducer protein TonB both in overall structure and function. *A. tumefaciens* VirB6 is highly hydrophobic and spans the membrane several times, as deduced by results of computer analyses and reporter protein fusion studies [37, 38]. However, these studies left ambiguities in the topology that now have been resolved with Cys accessibility studies of phenotypically-silent Cys substitution mutations [37]. Together, the data support a model for VirB6 as a polytopic membrane protein with a periplasmic N terminus, 5 TMS, and a cytoplasmic C terminus. A particularly notable feature of the topology model is a large central periplasmic loop, designated loop P2, which is now known to play an important role in substrate translocation (see below). VirB6 contacts with other channel subunits have not been detected by yeast two-hybrid screens, probably because of its hydrophobic character. However, VirB6 was shown to interact with the VirB7 lipoprotein and VirB9 by co-immunoprecipitation [39], and with other VirB ‘core’ subunits by gel filtration chromatography and blue native gel electrophoresis [40]. Additionally, VirB6 exerts stabilizing effects on other VirB subunits and it co-localizes with VirD4 and the T pilus at the cell poles of *A. tumefaciens* [12, 39, 41]. These findings together with results of DNA transfer assays described below strongly indicate that VirB6 assembles as a central component of the ‘core’ structure that likely corresponds to the VirB/D4 secretion channel.

The VirB7 lipoprotein forms a disulfide bridge with VirB9, and the heterodimer sorts to the OM where it exerts stabilizing effects on other machine subunits [7]. VirB8 and VirB10 are bitopic IM subunits. Recently, structures of periplasmic fragments of both subunits were resolved by X-ray crystallography (G. Waksman, pers. comm.). Over its length, VirB10 shares several structural features

with TonB, including a small N-terminal cytoplasmic domain, a single TMS, a Pro-rich region, and a region of sequence conservation at the C-terminal end [36]. For TonB, the Pro-rich motif contributes to a rigid, extended structure in the periplasm that might permit simultaneous contacts with partner subunits at the inner and outer membranes [42, 43]. Similarly, *A. tumefaciens* VirB10 interacts with the IM subunits VirB8, VirD4, and VirB4, and with the OM-associated VirB7–VirB9 heterodimer [32–34, 36, 44] (Figure 1). VirB10 also functionally resembles TonB by linking energy at the IM to the assembly or gating of the type IV secretion channel for substrate translocation (see below).

Recent studies also have identified protein–protein interactions among machine components of the F plasmid T4SS [45–47]. This T4SS is a chimeric system composed of several VirB/D4 subunits and other components of an unrelated ancestry [6]. Studies of the F-specific subunits have identified a protein interaction network, composed of TrbB, TrbI, TraW, TraU, TraF, and TraH, that likely spans the cell envelope [47] (Figure 1). This putative structure is of special interest because the F plasmid-encoded T pilus extends and retracts, presumably to draw donor and recipient cells together for mating pair formation [6]. Mutations in the Trb/Tra components listed above disrupt pilus outgrowth and retraction, suggesting that this subcomplex regulates dynamic aspects of pilus formation [47]. It is likely that an F-specific protein subassembly serves such a function because other conjugation systems, e.g., the *A. tumefaciens* VirB/D4, RP4 Tra, and R388 Trw systems, appear to slough rather than retract their pili [7].

**T pilus subcomplex – VirB2, VirB5, VirB7**

VirB2 is the major subunit of the T pilus [48]. The VirB7 lipoprotein and VirB5 are minor pilus components, although it is not known how these subunits associate with the T pilus [49, 50]. Recently, a crystal structure of a VirB5 homolog, TraC of pKM101, was presented [51]. This is a single domain protein with a mostly α-helical, elongated structure. Analyses of TraC mutants conferred altered conjugation frequencies as well as defects in infection by bacteriophages PRD1 and Ike. These findings led to a proposal that the VirB5-like proteins bind along the pilus, including the pilus tip (the site of Ike attachment) and base (the site of PRD1 attachment), and that these sites of contact might promote both pilus biogenesis and adherence to recipient cells [9, 51].

Dynamics of T4SS machine assembly and function

The available structural data and protein interaction networks present a somewhat static view of the type IV systems (Figure 1). Recent studies, however, have begun to capitalize on development of novel in vivo assays to explore dynamic aspects of machine assembly and function.

### Substrate recognition and recruitment

For example, studies examining the role of the T4CP in substrate recruitment exploited the finding that the VirD4T4CP localizes to the *A. tumefaciens* cell poles [41]. Accordingly, to assay for T4CP receptor activity, the VirE2 effector protein was tagged with GFP, or both VirD4 and VirE2 were tagged with complementary half-GFP's. Cells producing these constructs displayed polar fluorescence that was dependent on VirD4 but independent of the VirB Mpf subunits, confirming the proposal that the VirD4T4CP functions as a receptor for protein substrates [52]. Additionally, it was discovered that T4SS will translocate Cre recombinase [53] or adenylate cyclase when fused to the N termini of secretion substrates [54]. By monitoring intercellular translocation of reporter proteins, investigators are beginning to define the nature of type IV secretion signals carried by the exported protein substrates. In general, these translocation signals are located at the C termini of the substrates. Yet, the signals identified to date show little sequence conservation other than clusters of Arg residues, as shown for substrates of the *A. tumefaciens* VirB/D4T4SS [53], or hydrophobic residues, as shown for substrates of the *L. pneumophila* Dot/Icm T4SS [54]. In *Bartonella henselae*, the VirB/D4 T4SS exports substrates (*Bartonella* effector proteins or Beps) through recognition of a bipartite motif composed of a positively-charged C terminus and one or more BID (Bep intracellular delivery) domains. These BID domains are often, but not exclusively, located at the C termini of the Beps [55].

### Definition of the T-DNA translocation pathway by TrIP

Recently, another novel assay termed transfer DNA immunoprecipitation (TrIP) was developed to trace the path of a DNA substrate through the type IV secretion channel [56]. Adapted from the chromatin immunoprecipitation (ChIP) technique, TrIP employs a three-stage reaction involving: (i) formaldehyde treatment of intact cells to crosslink channel subunits to the T-DNA substrate as it exits the cell, (ii) detergent-solubilization and immunoprecipitation of individual T4SS subunits, and (iii) PCR amplification for detection of the T-DNA in the immunoprecipitates. With this assay, it was shown that the T-DNA substrate forms close contacts with 6 of the 12 VirB/D4 components, VirD4T4CP, VirB11 ATPase, polytopic VirB6, bitopic VirB8, VirB2 pilin, and VirB9 [56]. Further TrIP studies of T4SS mutants enabled formulation of a sequentially-and spatially-ordered translocation pathway for the T-DNA substrate. This pathway is described below and depicted in Figure 2.

#### Step I – Substrate recruitment to the T4SS

In the initial TrIP studies, VirD4 was shown to interact with the T-DNA independently of the VirB Mpf subunits, establishing that VirD4 is a receptor not only for protein substrates (see above) but also for the T-DNA [56]. Moreover, a VirD4 Walker A mutant was shown to retain both T-DNA as well as protein substrate receptor activity [32] (K. Atmakuri, P.J. Christie, unpublished findings). VirD4 thus appears to function as a general receptor for DNA and protein substrates through a mechanism(s) that operates independently of other Mpf components or ATP energy.

#### Step II – Transfer to the VirB11 hexameric ATPase

TrIP studies of various T4SS mutant strains established that VirD4 transfers the T-DNA substrate to the VirB11 ATPase [56]. Moreover, a VirD4 Walker A mutant protein successfully transferred the DNA substrate to VirB11 and, similarly, native VirD4 transferred the substrate to a VirB11 Walker A mutant. These findings argue against a requirement for ATP utilization by either of these energetic subunits for this transfer step [32]. Two further lines of study identified additional requirements for DNA transfer to VirB11. First, analyses of *virB* gene deletion mutants showed that VirB7 synthesis is essential for this transfer step [56]. As noted above, VirB7 contributes to assembly of a stable secretory apparatus through dimerization with VirB9. VirB7 might act indirectly through stabilization of the transenvelope structure to enable productive contacts between VirD4 and VirB11 [56], although it is also interesting to consider that the lipoprotein exerts its effects by interacting directly with the VirD4-VirB11 complex. Second, an in vivo reconstitution study showed that strains producing certain subsets of VirB/D4 components support DNA transfer from VirD4 to VirB11. Specifically, DNA transfer to VirB11 was observed in strains producing VirD4 and VirB11 plus VirB7 and either VirB8 and VirB9, or VirB10. Precisely how these ‘core’ components interface with the energetic components at the IM to stimulate substrate transfer from the

56 P. J. Christie & E. Cascales

VirB6 to substrate transfer, various VirB6 insertion and deletion mutations were assayed for effects on substrate translocation. Results indicated that: (i) periplasmic loop P2 contributes to formation of substrate contacts with VirB6, (ii) a domain adjacent to loop P2 contributes to substrate transfer from VirB6 to VirB8, and (iii) two terminal domains of VirB6 contribute to substrate transfer to VirB2 and VirB9 [37]. It can be speculated that loop P2 might form part of the secretion channel structure, whereas other domains of VirB6 form critical intersubunit contacts required for assembly of the distal portions of the channel. A combination of T4SS mutant analyses and *in vivo* reconstitution studies further showed that substrate transfer to VirB6 and VirB8 requires each of the three energetic components—VirD4, VirB4, and VirB11—together with VirB6, VirB7, VirB8, and either VirB9 or VirB10. These ‘core’ subunits likely promote formation of critical contacts between the IM channel subunits, but again the molecular details of these interactions await further study. Finally, Walker A mutations in any one of the energetic components abolished substrate transfer from VirB11 to VirB6 and VirB8, strongly indicating that VirD4, VirB4, and VirB11 mediate this step of substrate transfer through a coordination of ATP binding or hydrolysis activities [32, 56].

Step IV – Transfer to the periplasmic and outer-membrane-associated proteins VirB2 and VirB9. The TrIP studies place VirB2 and VirB9 at the distal end of the proposed DNA translocation pathway. Additionally, as observed for VirB6 and VirB8, both VirB2 and VirB9 must be co-produced for establishment of DNA substrate contacts with either subunit, and thus VirB2 and VirB9 likely function together to mediate the latter step(s) of substrate transfer [56]. Only a portion of VirB2 pilin produced by a cell polymerizes as the T pilus, whereas the remaining cell-associated form—the fraction resistant to extraction from the cell by shearing—co-partitions with the IM, OM, and periplasmic proteins [57]. An interesting set of mutations in VirB11, VirB6, and VirB9 were shown to selectively block assembly of the T pilus without affecting substrate transfer [39, 57] (S. Jakubowski, P.J. Christie, unpublished findings). These so-called ‘uncoupling’ mutations do not affect accumulation of cellular VirB2, strongly indicating that it is this cellular pilin pool and not the T pilus that contributes to substrate translocation to recipient cells. Finally, several T4SS subunits, including VirB3, VirB5, and VirB10 do not form detectable interactions with the T-DNA, suggesting that these subunits are probably not channel components. Nevertheless, the corresponding gene deletions abolish substrate

Figure 2. The T-strand translocation pathway, as identified by TrIP. Immunoprecipitation of the VirD2-T-strand transfer intermediate with channel components (dark gray) from wild-type and mutant strains define a temporally- and spatially-ordered transfer pathway (Hatched arrow, blue arrow online). The remaining VirB subunits (light gray) participate in machine biogenesis and/or regulate substrate passage among the channel components [56]. This figure is reproduced in colour in Molecular Membrane Biology online.

VirD4 receptor to VirB11 is a fascinating topic for further study.

Step III – Transfer to the integral inner membrane proteins VirB6 and VirB8. The TrIP studies place VirB6 and VirB8 at an intermediate point in the postulated DNA translocation pathway, dispensable for DNA transfer to VirD4 and VirB11 but necessary for transfer to VirB2 and VirB9. Additionally, VirB6 and VirB8 must be co-synthesized for establishment of DNA substrate contacts with either of these subunits, suggesting that VirB6 and VirB8 coordinate their activities to mediate this transfer step [56]. To further define the contributions of

translocation from VirB6 and VirB8 to VirB2
and VirB9, indicating that VirB3, VirB5, and
VirB10 still are essential for substrate trafficking
through the periplasm [56].

Energetics of DNA translocation: VirB10, a
TonB-like ATP energy sensor subunit

The energetics of DNA translocation across bacterial
membranes has been investigated in bacteriophage,
transformation, and conjugation systems [58–
61]. In each of these systems, DNA transfer is
dependent in part on proton motive force, as
demonstrated by an inhibition of transfer in the
presence of protonophores such as carbonyl cyanide
m-chlorophenyl hydrazone (CCCP). In early studies
of conjugation, CCCP-treated *E. coli* donors carrying
the F plasmid failed to form stable mating
aggregates, suggesting that transmembrane potential
(Δψ) or a pH gradient energizes some aspect of
machine assembly or function [58]. Additional
electrochemical measurements showed that the addition
of arsenate, an ATP synthase inhibitor, to
RP4-carrying *E. coli* cells conferred defects (e.g.,
resistance to phage binding) suggestive of a block in
assembly of the Mpf structure. Thus, evidence exists
for contributions of both proton motive force and
ATP energy to morphogenesis of the conjugation
machines.

Very recently, it was reported that *A. tumefaciens*
VirB10 undergoes a structural transition in response
to ATP utilization by the VirD4 T4CP and the
VirB11 ATPase [36]. Moreover, VirB10 was shown
to interact with VirD4 independently of ATP energy,
whereas stable complex formation between VirB10
and the OM-associated VirB7-VirB9 dimer requires
ATP utilization by both VirD4 and VirB11. Accompanying
the energy-dependent formation of this
presumptive transenvelope VirD4-VirB10-VirB7-
VirB9 structure, the T-DNA substrate translocates
from the IM portion of the secretion channel
composed of VirB6 and VirB8 to that in the
periplasm composed of VirB2 and VirB9. VirB10
thus supplies a TonB-like function by converting
energy at the IM into a mechanical force required for
a latter stage of machine biogenesis. Reminiscent of
TonB, VirB10 might transduce IM energy to the
OM-associated VirB7-VirB9 complex to mediate
formation or opening of a channel gate for passage of
the DNA substrate to the cell surface (Figure 3)
[36, 62, 63].

It is noteworthy that TonB couples IM proton
motive force to OM transport [62, 63], while VirB10
couples ATP energy to latter stages of DNA substrate
translocation [36]. This is the first report of an
ATP energy sensing mechanism by a TonB-like


Mechanistic features of Type IV secretion 57
protein. However, studies of several T2SS as well
as the bacteriophage f1 extrusion system also have
implicated bitopic subunits as energy coupling sub-
units required for translocation (Figure 3). For
example, the GspC subunits of T2SS interact with
IM protein platforms composed in part of VirB11-
like GspE ATP-binding subunits and the GspD
channel-forming secretin complex at the OM
[64, 65]. In the *Aeromonas hydrophila* T2SS, complex
formation between the ExeA ATPase and TonB-like
ExeB promotes multimerization of the secretin
ExeD, prompting the proposal that the ExeAB
complex energizes morphogenesis and/or gating of
the OM secretin [66, 67]. Finally, protein pI of
bacteriophage f1 appears to be a chimeric protein
composed of an N-terminal ATP-binding subunit
and a C-terminal TonB-like subunit. Both cellular
ATP and an intact Walker A motif are required for
phage extrusion, suggestive of pI-mediated energy
transduction to gate the pIV secretin complex at the
OM [68]. Taken together, these observations point
to specific functional relationships between IM
ATPases and cognate TonB-like subunits for assembly
of evolutionarily diverse macromolecular trafficking
systems.


Conclusion and perspectives

Bacteria have evolved T4SS to translocate a diverse
array of macromolecular substrates across the cell
surface. Recent studies have begun to describe
the assembly pathways and architectures of these
complex machines. Enabling *in vivo* assays also
are supplying exciting new information about the
dynamics of substrate recognition and recruitment,
and the genetic and energetic requirements
for defined steps of substrate translocation. Among
the important remaining questions are the following.
How are DNA and protein substrates configured
for export and how are C-terminal (or other)
translocation signals recognized? How do two ring-
shaped homohexameric ATP-binding subunits
and a third ATP-binding subunit interface to
drive substrate translocation across the IM and
beyond? Are protein and DNA substrates delivered
through the same translocation pathway? What is
the structural and functional relationship between
the secretion channel and extracellular pili or
other surface structures? What determinants specify
assembly of type IV secretion machines at the
cell poles? The current pace of research with
multi-faceted approaches, together with an influx
of youthful energy, should secure answers to
these fascinating questions within the coming few
years.

58 P. J. Christie & E. Cascales

Figure 3. Schematic representation of energetic components in macromolecular trafficking systems. Proton-motive force driven machines (upper panel): *E. coli* Tol-Pal system (left) involved in Group A colicins import and membrane stability, *E. coli* TonB system (right) required for group B colicins, B12 vitamin and iron-charged siderophores import. The energy is powered by two accessory proteins (TolQR and ExbBD) that probably act as a proton-conducting complex [70]. ATP-driven machines (lower panel): T4SS (left), T2SS (center) required for export of protein or nucleoprotein substrates across the envelope, and f1 filamentous phage extrusion system (right). The T4SS is exemplified by the *A. tumefaciens* VirB/D4 apparatus, whereas T2SS are exemplified by the Terminal Branch of the General Secretory Pathway (GSP) and the *Aeromonas aerophila* Exe machine. Subunits that bind or hydrolyze ATP are indicated with an arrow. The crystal or cryo-microscopy protein structures are reproduced from the following references [24,26,43,67,71–76] with permission from the authors, Elsevier Ed, MacMillan Publishers Ltd, Nature Publishing Group, and the American Society for Biochemistry and Molecular Biology. This figure is reproduced in colour in *Molecular Membrane Biology* online.

### Acknowledgements

The authors thank members of the Christie laboratory for helpful critiques of this manuscript. Work in this laboratory is funded by NIH Grant GM47846. We also thank Drs G. Waksman and C. Dehio for permission to cite unpublished findings, and the following investigators for permission to reproduce published images presented in Figure 3: M. Coll, S. Darst, H. Deisenhofer, W. Hol, B. Luisi, A. Pluckthun, T. Pugsley, and G. Waksman.

### References

[1] Cascales E, Christie PJ. The versatile bacterial type IV secretion systems. Nat Rev Microbiol 2003;1:137–150.

[2] Llosa M, O'Callaghan D. Euroconference on the biology of type IV secretion processes: Bacterial gates into the outer world. Mol Microbiol 2004;53:1–8.

[3] Grohmann E, Muth G, Espinosa M. Conjugative plasmid transfer in Gram-positive bacteria. Microb Mol Biol Rev 2003;67:277–301.

[4] Christie PJ, Vogel JP. Bacterial type IV secretion: conjugation systems adapted to deliver effector molecules to host cells. Trends Microbiol 2000;8:354–360.

[5] Ochman H, Lawrence JG, Groisman EA. Lateral gene transfer and the nature of bacterial innovation. Nature 2000;405:299–304.

[6] Lawley TD, Klimke WA, Gubbins MJ, Frost LS. F factor conjugation is a true type IV secretion system. FEMS Microbiol Lett 2003;224:1–15.

[7] Christie PJ. Bacterial type IV secretion: The Agrobacterium VirB/D4 and related conjugation systems. Biochem Biophys Acta 2004;1694:219–234.

[8] Funnel BE, Phillips GJ, editors. Plasmid biology. Washington, DC: ASM Press, 2004.

[9] Yeo HJ, Waksman G. Unveiling molecular scaffolds of the type IV secretion system. J Bacteriol 2004;186:1919–1926.

[10] Gelvin SB. Agrobacterium-mediated plant transformation: the biology behind the ‘gene-jockeying’ tool. Microbiol Mol Biol Rev 2003;67:16–37.

[11] Zhu Y, Nam J, Humara JM, Mysore KS, Lee LY, Cao H, Valentine L, Li J, Kaiser AD, Kopecky AL, et al. Identification of Arabidopsis rat mutants. Plant Physiol 2003;132:494–505.

[12] Lai EM, Chesnokova O, Banta LM, Kado CI. Genetic and environmental factors affecting T-pilin export and T-pilus biogenesis in relation to flagellation of Agrobacterium tumefaciens. J Bacteriol 2000;182:3705–3716.

[13] Lessl M, Lanka E. Common mechanisms in bacterial conjugation and Ti-mediated T-DNA transfer to plant cells. Cell 1994;77:321–324.

[14] Hubber A, Vergunst AC, Sullivan JT, Hooykaas PJ, Ronson CW. Symbiotic phenotypes and translocated effector proteins of the Mesorhizobium loti strain R7A VirB/D4 type IV secretion system. Mol Microbiol 2003;54:561–574.

[15] Engledow AS, Medrano EG, Mahenthiralingam E, LiPuma JJ, Gonzalez CF. Involvement of a plasmid-encoded type IV secretion system in the plant tissue watersoaking phenotype of Burkholderia cenocepacia. J Bacteriol 2004;186:6015–6024.

[16] Alegria MC, Docena C, Khater L, Ramos CHI, da Silva ACR, Farah CS. Identification of new protein-protein interactions involving products of the chromosome-and plasmid-encoded type IV secretion loci of the phytopathogen Xanthomonas axonopodis pv. citri. J Bacteriol 2004; in press.

[17] Batchelor RA, Pearson BM, Friis LM, Guerry P, Wells JM. Nucleotide sequences and comparison of two large conjugative plasmids from different Campylobacter species. Microbiol 2004;150:3507–3517.

[18] Tzfira T, Vaidya M, Citovsky V. Involvement of targeted proteolysis in plant genetic transformation by Agrobacterium. Nature 2004;431:87–92.

[19] Viala J, Chaput C, Boneca IG, Cardona A, Girardin SE, Moran AP, Athman R, Memet S, Huerre MR, Coyle AJ, et al. Nod1 responds to peptidoglycan delivered by the Helicobacter pylori cag pathogenicity island. Nat Immunol 2004;5:1166–1174.

[20] Marlovits TC, Kubori T, Sukhan A, Thomas DR, Galan JE, Unger VM. Structural insights into the assembly of the type III secretion needle complex. Science 2004;306:1040–1042.

[21] Krause S, Pansegrau W, Lurz R, de la Cruz F, Lanka E. Enzymology of type IV macromolecule secretion systems: The conjugative transfer regions of plasmids RP4 and R388 and the cag pathogenicity island of Helicobacter pylori encode structurally and functionally related nucleoside triphosphate hydrolases. J Bacteriol 2000;182:2761–2770.

[22] Sexton JA, Pinkner JS, Roth R, Heuser JE, Hultgren SJ, Vogel JP. The Legionella pneumophila PilT homologue DotB

exhibits ATPase activity that is critical for intracellular growth. J Bacteriol 2003;185:1658–1666.

[23] Sakai D, Horiuchi T, Komano T. ATPase activity and multimer formation of PilQ protein are required for thin pilus biogenesis in plasmid R64. J Biol Chem 2001;276:17968–17975.

[24] Gomis-Ruth FX, de la Cruz F, Coll M. Structure and role of coupling proteins in conjugal DNA transfer. Res Microbiol 2002;153:199–204.

[25] Herdendorf TJ, McCaslin DR, Forest KT. Aquifex aeolicus PilT, homologue of a surface motility protein, is a thermostable oligomeric NTPase. J Bacteriol 2002;184:6465–6471.

[26] Robien MA, Krumm BE, Sandkvist M, Hol WG. Crystal structure of the extracellular protein secretion NTPase EpsE of Vibrio cholerae. J Mol Biol 2003;333:657–674.

[27] Planet PJ, Kachlany SC, DeSalle R, Figurski DH. Phylogeny of genes for secretion NTPases: Identification of the widespread tadA subfamily and development of a diagnostic key for gene classification. Proc Natl Acad Sci USA 2001;98:2503–2508.

[28] Dang TA, Zhou X-R, Graf B, Christie PJ. Dimerization of the Agrobacterium tumefaciens VirB4 ATPase and the effect of ATP-binding cassette mutations on assembly and function of the T-DNA transporter. Mol Microbiol 1999;32:1239–1253.

[29] Ward DV, Draper O, Zupan JR, Zambryski PC. Peptide linkage mapping of the Agrobacterium tumefaciens vir-encoded type IV secretion system reveals protein subassemblies. Proc Natl Acad Sci USA 2002;99:11493–11500.

[30] Rabel C, Grahn AM, Lurz R, Lanka E. The VirB4 family of proposed traffic nucleoside triphosphatases: Common motifs in plasmid RP4 TrbE are essential for conjugation and phage adsorption. J Bacteriol 2003;185:1045–1058.

[31] Malek JA, Wierzbowski JM, Tao W, Bosak SA, Saranga DJ, Doucette-Stamm L, Smith DR, McEwan PJ, McKernan KJ. Protein interaction mapping on a functional shotgun sequence of Rickettsia sibirica. Nucleic Acids Res 2004;32:1059–1064.

[32] Atmakuri K, Cascales E, Christie PJ. Energetic components VirD4, VirB11, and VirD4 mediate early DNA transfer reactions required for bacterial type IV secretion. Mol Microbiol 2004;54:1199–1211.

[33] Gilmour MW, Gunton JE, Lawley TD, Taylor DE. Interaction between the IncHI1 plasmid R27 coupling protein and type IV secretion system: TraG associates with the coiled-coil mating pair formation protein TrhB. Mol Microbiol 2003;49:105–116.

[34] Llosa M, Zunzunegui S, de la Cruz F. Conjugative coupling proteins interact with cognate and heterologous VirB10-like proteins while exhibiting specificity for cognate relaxosomes. Proc Natl Acad Sci USA 2003;100:10465–10470.

[35] Terradot L, Durnell N, Li M, Li D, Ory J, Labigne A, Legrain P, Colland F, Waksman G. Biochemical characterization of protein complexes from the Helicobacter pylori protein interaction map: strategies for complex formation and evidence for novel interactions within type IV secretion systems. Mol Cell Proteomics 2004;3:809–819.

[36] Cascales E, Christie PJ. Agrobacterium VirB10, an ATP energy sensor required for type IV secretion. Proc Natl Acad Sci USA 2004;101:17228–17233.

[37] Jakubowski SJ, Krishnamoorthy V, Cascales E, Christie PJ. Agrobacterium tumefaciens VirB6 domains direct the ordered export of a DNA substrate through a type IV secretion system. J Mol Biol 2004;341:961–977.

[38] Das A, Xie YH. Construction of transposon Tn3phoA: Its application in defining the membrane topology of the

60 P. J. Christie & E. Cascales

Agrobacterium tumefaciens DNA transfer proteins. Mol Microbiol 1998;27:405–414.

[39] Jakubowski SJ, Krishnamoorthy V, Christie PJ. Agrobacterium tumefaciens VirB6 protein participates in formation of VirB7 and VirB9 complexes required for type IV secretion. J Bacteriol 2003;185:2867–2878.

[40] Krall L, Wiedemann U, Unsin G, Weiss S, Domke N, Baron C. Detergent extraction identifies different VirB protein subassemblies of the type IV secretion machinery in the membranes of Agrobacterium tumefaciens. Proc Natl Acad Sci USA 2002;99:11405–11410.

[41] Kumar RB, Das A. Polar location and functional domains of the Agrobacterium tumefaciens DNA transfer protein VirD4. Mol Microbiol 2002;43:1523–1532.

[42] Higgs PI, Myers PS, Postle K. Interactions in the TonB-dependent energy transduction complex: ExbB and ExbD form homomultimers. J Bacteriol 1998;180:6031–6038.

[43] Postle K, Kadner RJ. Touch and go: Tying TonB to transport. Mol Microbiol 2003;49:869–882.

[44] Das A, Xie Y-H. The Agrobacterium T-DNA transport pore proteins VirB8, VirB9, and VirB10 interact with one another. J Bacteriol 2000;182:758–763.

[45] Harris RL, Sholl KA, Conrad MN, Dresser ME, Silverman PM. Interaction between the F plasmid TraA (F-pilin) and TraQ proteins. Mol Microbiol 1999;34:780–791.

[46] Harris RL, Hombs V, Silverman PM. Evidence that F-plasmid proteins TraV, TraK and TraB assemble into an envelope-spanning structure in Escherichia coli. Mol Microbiol 2001;42:757–766.

[47] Harris RL, Silverman PM. Tra proteins characteristic of F-like type IV secretion systems constitute an interaction group by yeast two-hybrid analysis. J Bacteriol 2004;186:5480–5485.

[48] Lai EM, Kado CI. The T-pilus of Agrobacterium tumefaciens. Trends Microbiol 2000;8:361–369.

[49] Schmidt-Eisenlohr H, N. D, C. A, Wanner G, Zambryski PC, Baron C. Vir proteins stabilize VirB5 and mediate its association with the T pilus of Agrobacterium tumefaciens. J Bacteriol 1999;181:7485–7492.

[50] Sagulenko V, Sagulenko E, Jakubowski S, Spudich E, Christie PJ. VirB7 lipoprotein is extracellular and associates with the Agrobacterium tumefaciens T pilus. J Bacteriol 2001;183:3642–3651.

[51] Yeo H-J, Yuan Q, Beck MR, Baron C, Waksman G. Structural and functional characterization of the VirB5 protein from the type IV secretion system encoded by the conjugative plasmid pKM101. Proc Natl Acad Sci USA 2003;100:15947–15052.

[52] Atmakuri K, Ding Z, Christie PJ. VirE2, a type IV secretion substrate, interacts with the VirD4 transfer protein at cell poles of Agrobacterium tumefaciens. Mol Microbiol 2003;49:1699–1713.

[53] Vergunst AC, van Lier M, den Dulk-Ras A, Stuve TAG, Ouwehand A, Hooykaas PJ. Positive charge is an important feature of the C-terminal transport signal of the VirB/D4-translocated proteins of Agrobacterium. Proc Natl Acad Sci USA 2005;102:832–837.

[54] Nagai H, Kagan JC, Amor JC, Kahn RA, Roy CR. A C-terminal translocation signal required for Dot/Icm-dependent delivery of the Legionella RalF protein to host cells. Proc Natl Acad Sci USA 2005;102:826–831.

[55] Schulein R, Guye P, Rhomberg TA, Schmid MC, Schroder G, Vergunst AC, Carena I, Dehio C. A bipartite signal mediates the transfer of type IV secretion substrates of Bartonella henselae into human cells. Proc Nat Acad Sci USA 2005;102:856–861.


[56] Cascales E, Christie PJ. Definition of a bacterial type IV secretion pathway for a DNA substrate. Science 2004;304:1170–1173.

[57] Sagulenko Y, Sagulenko V, Chen J, Christie PJ. Role of Agrobacterium VirB11 ATPase in T-pilus assembly and substrate selection. J Bacteriol 2001;183:5813–5825.

[58] Palmen R, Driessen AJ, Hellingwerf KJ. Bioenergetic aspects of the translocation of macromolecules across bacterial membranes. Biochim Biophys Acta 1994;1183:417–451.

[59] Daugelavicius R, Bamford JKH, Grahn AM, Lanka E, Bamford D. The IncP plasmid-encoded cell envelope-associated DNA transfer complex increases cell permeability. J Bacteriol 1997;179:5195–5202.

[60] Maier B, Chen I, Dubnau D, Sheetz MP. DNA transport into Bacillus subtilis requires proton motive force to generate large molecular forces. Nat Struct Mol Biol 2004;11:643–649.

[61] Letellier L, Boulanger P, Plancon L, Jacquot P, Santamaria M. Main features on tailed phage, host recognition and DNA uptake. Front Biosci 2004;9:1228–1339.

[62] Larsen RA, Thomas MG, Postle K. Protonmotive force, ExbB and ligand-bound FepA drive conformational changes in TonB. Mol Microbiol 1999;31:1809–1824.

[63] Germon P, Ray MC, Vianney A, Lazzaroni JC. Energy-dependent conformational change in the TolA protein of Escherichia coli involves its N-terminal domain, TolQ, and TolR. J Bacteriol 2001;183:4110–4114.

[64] Bleves S, Gerard-Vincent M, Lazdunski A, Filloux A. Structure-function analysis of XcpP, a component involved in general secretary pathway-dependent protein secretion in Pseudomonas aeruginosa. J Bacteriol 1999;181:4012–4019.

[65] Possot OM, Vignon G, Bomchil N, Ebel F, Pugsley AP. Multiple interactions between pullulanase secretion components involved in stabilization and cytoplasmic membrane association of PulE. J Bacteriol 2000;182:2142–2152.

[66] Howard SP, Meiklejohn HG, Shivak D, Jahagirdar R. A TonB-like protein and a novel membrane protein containing an ATP-binding cassette function together in exotoxin secretion. Mol Microbiol 1996;22:595–604.

[67] Schoenhofen IC, Stratilo C, Howard SP. An ExeAB complex in the type II secretion pathway of Aeromonas hydrophila: effect of ATP-binding cassette mutations on complex formation and function. Mol Microbiol 1998;29:1237–1247.

[68] Feng JN, Model P, Russel M. A trans-envelope protein complex needed for filamentous phage assembly and export. Mol Microbiol 1999;34:745–755.

[69] Shamaei-Tousi A, Cahill R, Frankel G. Interaction between protein subunits of the type IV secretion system of Bartonella henselae. J Bacteriol 2004;186:4796–4801.

[70] Cascales E, Lloubès R, Sturgis JN. The TolQ–TolR proteins energize TolA and share homologies with the flagellar motor motA–notB. Mol Microbiol 2001;42:795–807.

[71] Yeo H-J, Savvides SN, Herr AB, Lanka E, Waksman G. Crystal structure of the hexameric traffic ATPase of the Helicobacter pylori type IV system. Mol Cell 2000;6:1461–1472.

[72] Nouwen N, Ranson N, Saibil H, Wolpensinger B, Engel A, Ghazi A, Pugsley AP. Secretin PulD: association with pilot PulS, structure, and ion-conducting channel formation. Proc Natl Acad Sci USA 1999;96:8173–8177.

[73] Opalka N, Beckmann R, Boisset N, Simon MN, Russel M, Darst SA. Structure of the filamentous phage pIV multimer by cryo-electron microscopy. J Mol Biol 2003;325:461–470.

[74] Chang C, Mooser A, Pluckthun A, Wlodawer A. Crystal structure of the dimeric C-terminal domain of TonB reveals a novel fold. J Biol Chem 2001;276:27535–27540.

[75] Buchanan SK, Smith BS, Venkatramani L, Xia D, Esser L, Palnitkar M, Chakraborty R, van der Helm D, Deisenhofer J. Crystal structure of the outer membrane active transporter FepA from *Escherichia coli*. Nat Struct Biol 1999;6:56–63.

[76] Witty M, Sanz C, Shah A, Grossmann JG, Mizuguchi K, Perham RN, Luisi B. Structure of the periplasmic domain of *Pseudomonas aeruginosa* TolA: evidence for an evolutionary relationship with the TonB transporter protein. EMBO J 2002;21:4207–4218.
