
The TagB Protein in *Bacillus subtilis* 168 Is an Intracellular Peripheral Membrane Protein That Can Incorporate Glycerol Phosphate onto a Membrane-bound Acceptor *in Vitro*

Received for publication, July 1, 2005, and in revised form, September 1, 2005 Published, JBC Papers in Press, September 2, 2005, DOI 10.1074/jbc.M507154200

Amit P. Bhavsar$^{1}$, Ray Truant, and Eric D. Brown$^{2}$

From the Antimicrobial Research Centre and Department of Biochemistry and Biomedical Sciences, McMaster University, Hamilton, Ontario L8N 3Z5, Canada

Genes involved in the synthesis of poly(glycerol phosphate) wall teichoic acid have been identified in the *tag* locus of the model Gram-positive organism *Bacillus subtilis* 168. The functions of most of these gene products are predictable from sequence similarity to characterized proteins and have provided limited insight into the intracellular synthesis and translocation of wall teichoic acid. Nevertheless, critical steps of poly(glycerol phosphate) teichoic acid polymerization continue to be a puzzle. TagB and TagF, encoded in the *tag* locus, do not show sequence similarity to characterized proteins. We recently showed that recombinant TagF could catalyze glycerol phosphate polymerization *in vitro*. Based largely on homology to TagF, the TagB protein has been proposed to catalyze either an intracellular glycerophosphotransfer reaction or the extracellular teichoic acid/peptidoglycan ligation reaction. Here we have taken steps to characterize TagB, particularly through *in vivo* localization studies and *in vitro* biochemical assay, in order to make a case for either role in teichoic acid biogenesis. We have shown that TagB associates peripherally with the intracellular face of the cell membrane *in vivo*. We have also produced recombinant TagB and used it to demonstrate the enzymatic incorporation of labeled glycerol phosphate onto a membrane-bound acceptor. The data collected from this and the accompanying study (Schertzer, J. W., Bhavsar, A. P., and Brown, E. D. (2005) J. Biol. Chem. 280, 36683–36690) are strongly supportive of a role for TagB in *B. subtilis* 168 teichoic acid biogenesis as the CDP-glycerol:N-acetyl-β-D-mannosaminyl-1,4-N-acetyl-D-glucosaminyldiphosphoundecaprenyl glycerophosphotransferase. Here we use the trivial name “Tag primase.”

on the basis of homology to characterized enzymes. It is noteworthy that biochemical characterization of these functions has been restricted so far to TagD, the CDP-glycerol pyrophosphorylase (9, 10), TagF, the teichoic acid glycerol phosphate (Tag)$^{3}$ polymerase (11), and MnaA, the N-acetylglucosamine 2-epimerase (12). A model for teichoic acid biogenesis has been proposed based largely on the functional predictions for the *tag* genes (13). Significant gaps remain, however, in our understanding of teichoic acid biogenesis. Most notably, the step immediately preceding glycerol phosphate polymerization, the transfer of glycerol phosphate to an undecaprenyl-pyrophosphoryl-N-acetylglicosaminyl-N-acetylmannosamine acceptor, and also the final step in teichoic acid biogenesis, the ligation of teichoic acid to peptidoglycan (14), remain essentially uncharacterized.

We are particularly interested in identifying the unknown enzymes catalyzing the glycerophosphotransfer and ligation reactions outlined above. The reactions are chemically similar (see Fig. 1), since they both entail a transesterification reaction where the acceptor substrate is a sugar hydroxyl (either the 4-hydroxyl of N-acetylmannosamine or the 6-hydroxyl of N-acetylmuramic acid) and the activated donor has a pyrophosphate moiety (either CDP-glycerol or prenyl-pyrophosphate-disaccharide-polymer) that will undergo cleavage. Indeed, the enzymes catalyzing both reactions might be expected to share mechanistic similarity and perhaps homology with the glycerol phosphate polymerase, TagF.

Intriguingly, TagF shows no homology to characterized enzymes but does share extended homology over its carboxyl terminus, ~30% identity and 50% similarity, to the TagB protein (7). Unfortunately, little is known about how the structure of TagF, a nearly 90-kDa protein, relates to its activity as a glycerol phosphate polymerase, so its carboxyl-terminal sequence similarity to TagB does not immediately yield an accurate functional prediction for TagB. Nevertheless, the thesis that TagB catalyzes either the glycerophosphotransfer to the undecaprenyl-pyrophosphate-disaccharide or the polymer ligation to muramic acid (peptidoglycan) is a reasonable one based on homology to TagF, as noted by Karamata’s group (15).

The major class of anionic polymer in Gram-positive bacteria is teichoic acid. Growing evidence suggests that these phosphate-rich polyol polymers are essential to the viability of the model Gram-positive, *Bacillus subtilis* (1–5). The bulk of the genes for teichoic acid biosynthesis in *B. subtilis* 168 are indispensable and situated in the *tag* operons, *tagAB*, *tagDEF*, and *tagGH*, which code for the synthesis of a poly(glycerol phosphate) polymer (6, 7). Indeed, we recently verified that the *tagB* and *tagF* gene products were essential to the viability of *B. subtilis* 168 (8). Functional predictions have been made for most of the gene products

The physiological literature on TagB is also unclear. Characterization of a temperature-sensitive *B. subtilis* mutant, *tag-1*, generated by random mutagenesis and attributed to a lesion in *tagB*, demonstrated that the strain accumulated cytosolic pools of CDP-glycerol when grown at the restrictive temperature (16). This would be consistent with a role for TagB as a glycerophosphotransferase that acts downstream of the glycerol phosphate cytidylyltransferase (TagD). It was also reported, however, that the same strain retained susceptibility to phage Φ29, which recognizes glucosylated teichoic acid as a receptor (16). This implies

* This work was supported in part by Canadian Institutes of Health Research (CIHR) Grant MOP-15496 (to E. D. B.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
$^{1}$ Supported by a Canada Graduate Scholarship from the CIHR.
$^{2}$ Holder of a Canada Research Chair in Microbial Biochemistry. To whom correspondence should be addressed: Dept. of Biochemistry and Biomedical Sciences, McMaster University, 1200 Main St. W., Hamilton, Ontario L8N 3Z5, Canada. Tel.: 905-525-9140 (ext. 22392); Fax: 905-522-9033; E-mail: ebrown@mcmaster.ca.

$^{3}$ The abbreviations used are: Tag, teichoic acid glycerol phosphate; GFP, green fluorescent protein; PBS, phosphate-buffered saline; CHAPS, 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate; HPLC, high performance liquid chromatography.

In Vivo and in Vitro Characterization of B. subtilis TagB

![Diagram](attachment:diagram.png)

FIGURE 1. Chemical similarity of the reactions carried out by the teichoic acid glycerol phosphate polymerase (Tag polymerase) and the unidentified teichoic acid glycerol phosphate primase (Tag primase) and teichoic acid/peptidoglycan ligase. The double line indicates the cell membrane, and the proposed subcellular location of each reaction is indicated. Reactions are listed in temporal order as denoted by the arrows. Intermediate steps are not shown. The enzyme catalyzing each reaction is listed. $C_{55}$, undecaprenyl; $R$, glucose or alanine; $AcN$, $N$-acetyl.

that teichoic acid was synthesized and exported to the extracellular face of the membrane and would be consistent with a teichoic acid/peptidoglycan ligase function for TagB.

Predictions based on sequence analysis of TagB (TMHMM version 2.0) (17) and TMPred (18) indicate the presence of a transmembrane helix situated between residues 6 and 25. Furthermore, topological predictions suggest that TagB is a Type II membrane protein, where the bulk of the protein is localized extracellularly. Also of interest is the identification by Karamata and co-workers (7) of a potential lipoprotein modification signal, LANC, which matches the consensus lipobox sequence, LXXC, identified by Hayashi and Wu (19). Lipid modification in *B. subtilis* typically results in localization of modified proteins to the extracellular face of the membrane. Thus, the predictions from sequence analysis are most consistent with a role for TagB as the teichoic acid/peptidoglycan ligase.

In this work, we have investigated both the in vivo and in vitro properties of TagB to determine its biochemical role in teichoic acid biosynthesis. To characterize the localization of TagB we have developed a preparation of pure recombinant TagB and produced TagB polyclonal antiserum. We have provided experimental evidence for membrane localization and have characterized this association to show that it is peripheral and intracellular. Using fluorescence microscopy and a fusion of TagB to the green fluorescent protein (GFP), we have provided the first in vivo visualization of a teichoic acid biosynthetic protein, which showed localization of TagB to the cell periphery and septa. Further, we have demonstrated that recombinantly purified TagB is capable of catalyzing the incorporation of glycerol phosphate from CDP-glycerol to a membrane-bound acceptor in vitro. Thus, all experimental results support a role for TagB as the CDP-glycerol:$N$-acetyl-$\beta$-$D$-mannosaminyl-1,4-$N$-acetyl-$D$-glucosaminyldiphosphoundecaprenyl glycerophosphotransferase. Here we refer to this enzyme as the “Tag primase,” due to its temporal and functional relationship with the Tag polymerase, TagF.

### EXPERIMENTAL PROCEDURES

**Bacterial Strains, Reagents, and General Methods**—The strains used in this study are listed in TABLE ONE. All chemicals, unless otherwise noted, were purchased from Sigma. General cloning methods for *Escherichia coli* and *B. subtilis* were performed as outlined by Sambrook et al. (20) and Cutting and Youngman (21), respectively. Reagents for molecular cloning were purchased from New England Biolabs (Beverly, MA). Unless otherwise noted, the growth medium used was a derivative of Luria-Bertani, and antibiotic selection was as follows: 50 $\mu$g/ml ampicillin and 25 $\mu$g/ml kanamycin.

**Construction of Strains Used in This Study**—To generate the amino-terminal hexahistidine-tagged TagB, the *tagB* open reading frame was amplified by PCR using primers *tagBhisfor* (5′-GGGGACAAGTTTGTACAAAAAAAGCAGGCTTAATGAAAATAAGATCACTAATGGCGAATTGC-3′) and *tagBhisrev* (5′-GGGGACCACCTTTGTAAACAA-GAAAGCTGGGTCTCAGCTTATTAATTTTCGATGAAATTC-AATAAATTTTGGGC-3′). The amplified product was recombined into vector pDONR201 and subsequently vector pDEST17 of the Gateway expression system (Invitrogen Canada Inc., Burlington, Canada) to yield an amino-terminal hexahistidine fusion. Following recombination into pDEST17, the additional residues added to the amino terminus of TagB were MSYYHHHHHHLESTSLYKKAGL.

To allow protein expression of carboxyl-terminal hexahistidine-tagged TagB or TagBH126A, the open reading frames *tagBhis* and *tagBH126Ahis* were amplified from pBStagBoptrbs (22) and pRBtagBH126A (22) with primers *tagBoptrbsfor* (22) and *tagBfullrev2his* (5′-GGAAAAGCTTTTAGTGGTGTTGGTGGTGGGCTTATTAATTTTCG-3′). The products were cloned into the EcoRV site of

TABLE ONE

Strains and plasmids used in this study

| Strains/Plasmids | Genotype/Description                                                                 | Reference       |
|------------------|-------------------------------------------------------------------------------------|-----------------|
| B. subtilis strains |                                                                                   |                 |
| EB6              | hisA1argC4metC 3                                                                    | L5087(5)        |
| EB889            | EB 6 transformed with pRB 374                                                      | This work       |
| EB890            | EB 6 transformed with pRBtagB                                                      | This work       |
| EB892            | EB 6 transformed with pRBtagBgf                                                   | This work       |
| E. coli strains   |                                                                                   |                 |
| Novablue         | endA1hsdR17(rK12–mK12+) supE44 thi-1 recA1 gyrA 96 relA1 lac [F' proA+B+ lacIq ZΔM15::Tn10(TcR)] | Novagen         |
| MC1061           | [araD139] Δ(araA-leu) 7697 Δ(codB-lacI) 3 galK 16 galE15N mcrA e14 relA1 rpsL150(strR) spoT1 mcrB–hsdR2 | Ref. 25         |
| BL21-AI          | F– ompT hsdSB (rB–mB–) gal dcm araB::T7 RNAP-tetA                                   | Invitrogen      |
| BL21-SI          | F– ompT hsdSB (rB–mB–) gal dcm endA1 lon– proU::T7 RNAP::malQ-lacZ(TetS)             | Invitrogen      |
| Plasmids         |                                                                                   |                 |
| pPL51            | specR derivative of pDL50b                                                        | Ref. 24         |
| pBluescript II SK+ | General cloning vector                                                           | Stratagene      |
| pBSgfp           | GFP cloned from pPL51in pBluescript II SK+                                          | This work       |
| pBStagBoptrbs    | pBluescript with wildtype tagB from B. subtilis                                     | Ref. 22         |
| pBStagBgf        | In frame fusion of tagB in pBSgfp                                                  | This work       |
| pBStagBhis       | pBluescript with tagB carboxyl-terminal hexahistidine tag insert                    | This work       |
| pBStagBH126Ahis  | pBluescript with tagBH126A carboxyl-terminal hexahistidine tag insert                | This work       |
| pRB374           | Replicative expression plasmid for B. subtilis                                     | Ref. 23         |
| pRBtagB          | pRB 374 with wildtype tagB insert                                                  | This work       |
| pRBtagBgf        | pRB 374 with tagBgf fusion insert                                                  | This work       |
| pRBtagBhis       | pRB 374 with tagB carboxyl-terminal hexahistidine tag insert                        | This work       |
| pRBtagBH126A     | pRB 374 with tagBH126A insert                                                      | Ref. 22         |
| pDEST 17         | E. coli amino-terminal hexahistidine fusion expression vector                       | Invitrogen      |
| pDESTtagBhis     | pDEST 17 with tagB insert                                                          | This work       |

pBluescript, generating pBStagBhis and pBStagBH126Ahis, respectively, and selected such that the 5'-end of the coding strand was proximal to the KpnI site of the polylinker. All expression constructs were verified by nucleotide sequencing of the open reading frames.

For in vivo TagB localization experiments, the tagBhis open reading frame was cloned from pBStagBhis into pRB 374 (23), using SalI and BamHI restriction sites to generate pRBtagBhis. The tagBhis insert was replaced by tagB from pBStagBoptrbs to generate pRBtagB.

To generate carboxyl-terminal fusions of GFP to TagB, the gfp open reading frame was cloned from pPL 51 (a spectinomycin-resistant derivative of pDL50b (24)) into pBluescript II SK+ to create pBSgfp. Gene tagB was amplified by PCR with primers T7 (5'-TAATACGACTCAC-TATAGGG-3') and tagBRImutgfprev 2 (5'-GGGCTCGAGGGCTTAT-TAAATTTCGATGAAATTCAATAAATTTTGGCTTGAGTTC-CATCAG-3'), which introduces a silent mutation at the EcoRI restriction site in the coding sequence of tagB) and cloned into pBSgfp upstream of, and in frame with, the gfp gene. The resulting plasmid, pBStagBgf, has the additional amino acids leucine and glutamate between the TagB and GFP open reading frames. The entire open reading frame was then excised and used to replace tagB in pRBtagB, generating plasmid pRBtagBgf. All pRB 374-derived constructs were transformed into B. subtilis 168 following passage through E. coli MC 1061 (25). Transformants were selected for kanamycin resistance and verified by diagnostic digestion of plasmid isolated from the transformed cells.

spectrometry at the McMaster Regional Centre for Mass Spectrometry. HisTagB protein was buffer-exchanged into phosphate-buffered saline (PBS) plus 20% glycerol and sent for antibody production in rabbits (Cocalico Biologicals, Reamstown, PA). Affinity purification of anti-TagB antiserum was performed according to the manufacturer's instructions using purified HisTagB coupled to Affi-Gel 10 matrix (Bio-Rad).

We took advantage of the T7 promoter on pBluescript to express the carboxyl-terminal hexahistidine-tagged TagB proteins. BL21-AI-competent cells (Invitrogen) were transformed with pBStagBhis and pBStagBH126Ahis. For protein purification, cells were grown in LB medium supplemented with ampicillin for 3 h at 37°C with shaking at 250 rpm and induced with 0.2% arabinose ($A_{600} \sim 1$). Cultures were grown an additional 4 h at 30°C with shaking at 200 rpm. The purification was similar to that outlined for HisTagB (above) except that the soluble lysate was used for purification, rendering detergent solubilization unnecessary. Pooled proteins from the nickel affinity column were further purified on an S200 preparative gel filtration column (Amersham Biosciences) using GF Buffer (25 mM Hepes, pH 7.5, 100 mM NaCl, 1 mM dithiothreitol, 10% glycerol). Pooled fractions were frozen at $-80^\circ$C. Protein concentrations were determined by the method of Bradford using bovine serum albumin as a protein standard (26).

Fluorescence Microscopy—Both fixed and live *B. subtilis* cells were examined by fluorescence microscopy. Live cells were from a midlog phase culture that was diluted 10-fold and applied to an acid-etched, poly-L-lysine-treated coverslip, adhered to the bottom of a 1.5-inch Petri dish. The Petri dish was centrifuged at $\sim 300$ rpm in a Beckman Allegra 6 centrifuge to adhere cells to the coverslip. *B. subtilis* cell fixation was performed according to the procedure of Harry *et al.* (27). Fixed samples were adhered to freshly prepared poly-L-lysine-treated coverslips and sealed. Images were captured on a Nikon TE200 inverted fluorescence microscope with a 175-watt xenon lamp source (Sutter Instruments, Novato, CA), ×63 plan apochromat oil objective (Nikon Canada, Mississauga, Canada), computer-controlled shutters, and filter wheels (Sutter Instruments), with Compix SimplePCI version 5.2 software (Nikon Canada). Enhanced GFP fluorescence was filtered with the enhanced GFP-specific excitation and emission filter sets (Chroma Technologies, Rockingham, VT), and images were captured using a Hamamatsu ORCA 100 digital camera at $1024 \times 1024$ 8-bit pixel resolution.

Immunodetection of TagB in *B. subtilis*—Strains EB889 and EB890 were grown in 150 ml of LB medium supplemented with kanamycin with shaking (250 rpm) at 30°C until an approximate $A_{600}$ value of 1.2. Cells were harvested, washed, and stored at $-20^\circ$C. Cell pellets were resuspended in Bacillus lysis buffer (PBS, pH 7.3, 1 mM EDTA, 1 mM dithiothreitol, 10 $\mu$g/ml DNase, 10 $\mu$g/ml RNase, Complete protease inhibitor mixture) and normalized to the same $A_{600}$ value. Cells were disrupted by passage through a French pressure cell, and the lysate was centrifuged at 11,000 rpm in a Beckman MLA-80 rotor for 10 min to remove unbroken cells. The supernatant was centrifuged at 24,000 rpm in a MLA-80 rotor for 20 min, and the resulting supernatant was centrifuged at 45,000 rpm in an MLA-80 rotor for 1 h. Supernatant from this centrifugation step was considered soluble lysate, and the pellet was resuspended in an equivalent volume of membrane resuspension buffer (PBS, pH 7.3, 1 mM dithiothreitol, 1 mM EDTA) and considered the membrane fraction. Samples were separated by either 12 or 15% SDS-PAGE followed by transfer to a nitrocellulose membrane. Immunodetection was performed using either affinity-purified anti-TagB antibody, anti-FtsY polyclonal antiserum, anti-TagD polyclonal antiserum, or anti-EzrA polyclonal antiserum. Donkey anti-rabbit IgG conjugated

to horseradish peroxidase (BIO/CAN Scientific, Mississauga, Canada) was used as secondary antibody. Immunoblots were detected using the Western Lightning chemiluminescence reagent kit (PerkinElmer Life Science Products) according to the manufacturer’s specification.

Extraction of *B. subtilis* Membranes—Membrane fractions from EB890 were precentrifuged at 90,000 rpm in a Beckman TLA-120.1 rotor for 30 min, and the pellet was resuspended in membrane resuspension buffer. One volume of membrane suspension was added to three volumes of extraction agent, and extractions were performed as previously reported (28), except that 0.1 M NaOH was used for alkali conditions. Following extraction, samples were layered onto a 0.5 M sucrose cushion (12 volumes) containing the appropriate extraction agent. Samples were centrifuged at 45,000 rpm in a MLA-80 rotor for 30 min. The samples were fractionated into top (sample volume), middle (sucrose cushion volume), and bottom (pellet) fractions. The top and middle fractions were precipitated with ice-cold trichloroacetic acid, followed by an ethanol/ether wash. The trichloroacetic acid precipitates and pellet fraction were suspended in the same volume of membrane resuspension buffer. Top, middle, and bottom fractions were separated by 12% SDS-PAGE followed by transfer to a nitrocellulose membrane. Immunodetection was performed as outlined above.

Protease Protection Analysis of TagB in *B. subtilis*—Protoplasts were generated from strain EB890 using essentially the method of Breitling and Dubnau (29), except that the STM buffer contained 28% sucrose. Membranes from EB890 (generated as inverted membrane vesicles via passage through a French pressure cell as outlined above) were centrifuged at 55,000 rpm in a Beckman TLA 120.1 rotor for 30 min, and the pellet was resuspended in an equivalent volume of STM. Both membranes and protoplasts were treated with either 0.5 mg/ml Proteinase K or 1% Triton X-100 plus 0.5 mg/ml Proteinase K. The final concentration of sucrose in all reactions was $\sim 25\%$ to maintain the integrity of the protoplasts. Reactions were incubated for 30 min at room temperature and halted by the addition of phenylmethanesulfonyl fluoride to 4 mM. Protoplast samples were centrifuged at maximum speed in a microcentrifuge for 5 min, and the resulting supernatant was saved for analysis. The pellet was washed three times in STM plus phenylmethanesulfonyl fluoride (4 mM) and resuspended in an equivalent volume of STM. Membrane samples were centrifuged at 55,000 rpm in a TLA-120.1 rotor for 30 min, and the resulting supernatant was saved for analysis. The pellet was washed three times with STM plus phenylmethanesulfonyle fluoride using 8-min centrifugation steps at 106,000 rpm in a TLA-120.1 rotor. Following washes, the pellet was resuspended in an equivalent volume of STM. Both supernatant and pellet samples were separated by 15% SDS-PAGE followed by transfer to a nitrocellulose membrane. Immunodetection was performed as outlined above.

Synthesis of [$\beta^{-32P}$]CDP-glycerol—[$\beta^{-32P}$]CDP-glycerol was synthesized stepwise with the initial formation of [$^{32}$P]glycerol phosphate. This reaction contained 100 $\mu$M ATP, 100 $\mu$M glycerol, 250 $\mu$Ci of [$\gamma^{-32}$P]ATP ($\sim 3000$ Ci/mmol; Amersham Biosciences), and one unit of glycerokinase (Sigma). The reaction buffer contained 50 mM Hepes, pH 8, 10 mM MgCl$_2$, and 1 mM dithiothreitol. The reaction was incubated for 2 h at room temperature and subsequently centrifuged twice through an Ultrafree-MC 5000 NMWL centrifugal filter (Fisher) to remove glycerokinase. The reaction product was then converted to CDP-glycerol according to Ref. 11.

In Vitro Assay for TagB—The assay was essentially performed according to Schertzer and Brown (11), except that *B. subtilis* membranes were preheated at $100^\circ$C for 20 min prior to use in the assay to decrease the background activity. Unless otherwise stated, the assay (250-$\mu$l volume) consisted of 200 nM TagBhis, 1.5 mg *B. subtilis* 168

membrane protein/ml, and 50 μM CDP-glycerol (~1 μCi [β-32P]CDP-
glycerol) in assay buffer (25 mM Tris-HCl, pH 7.5, 40 mM MgCl₂) incu-
bated for 30 min at room temperature and quenched with urea to 4 M.
Kinetic parameters were calculated using Sigma Plot 8.0 (SPSS Scien-
tific, Chicago, IL).

**Extraction of TagB Reaction Product—**TagBhis reactions were car-
ried out as above, and following the final wash membranes were resus-
pended in assay buffer (200 μl for butanol extractions and 250 μl for
mild acid extraction). Butanol extractions were performed according to
Amer and Valvano (30), and mild acid extraction was performed
according to McArthur *et al.* (31) (*i.e.* 0.1 M HCl for 10 min at 100 °C).
Reactions were neutralized prior to analysis.

**Size Analysis of TagB Reaction Product—**The TagB reaction was
modified to ~700 nM TagBhis, 3 mg of *B. subtilis* 168 membrane pro-
tein/ml, and 10 μM CDP-glycerol (~15 μCi of [β-32P]CDP-glycerol)
and incubated for 15 h at room temperature. A control reaction omit-
ting TagBhis was also performed. To reduce background radioactivity,
the membranes were washed four times with assay buffer. Membranes,
recovered by ultracentrifugation, were subjected to mild acid extraction
as outlined above. The soluble fraction was isolated after ultracentrifu-
gation and filtration through an Ultrafree-MC 5000 NMWL centrifugal
filter. A portion of the recovered material was acidified to 1 M HCl and
heated at 100 °C for 3 h to reduce the TagB reaction product to its
glycerol phosphate component (11, 32). All samples were neutralized
and analyzed by high performance liquid chromatography (HPLC)
using a Waters Ultrahydrogel 120 column (Mississauga, Canada) with
GF2 elution buffer (50 mM Tris-HCl, pH 7.5, 100 mM NaCl, 10 mM
EDTA) at 0.5 ml/min.

---

**RESULTS**

**Purification of Recombinant TagB—**We report here the first purifica-
tion of recombinant TagB from *E. coli*. Amino-terminally hexahisti-
dine-tagged TagB (HisTagB) was enriched in the membrane fraction of
*E. coli*, and we found that approximately 20–30% of HisTagB could be
solubilized from the membrane in the presence of a nonionic detergent
(CHAPS). Furthermore, solubilized HisTagB could be purified to
homogeneity using a single nickel-chelating affinity chromatography
step (data not shown). To verify the identity of TagB, we performed
immunodetection analysis of the purified protein and determined that it
was immunoreactive to a commercially available anti-hexahistidine
antibody (data not shown). We also subjected the purified protein to
analysis by mass spectrometry and determined a molecular mass of
47,079 daltons (data not shown), in exact agreement with the calculated
mass of HisTagB after cleavage of the initiating methionine residue. We
used this protein as antigen to produce an anti-TagB polyclonal anti-
body. The HisTagB protein was also used to affinity-purify the anti-
TagB polyclonal antiserum.

**TagB Is Localized to the Membrane in *B. subtilis—***To facilitate detec-
tion of TagB *in vivo*, the *tagB* gene was cloned into a plasmid that allows
constitutive expression from the Pₐₑ₉ₕⅡ promoter (23). Lysates generated
from strains EB889 and EB890 were analyzed for immunodetection of
TagB. Fig. 2A shows clear detection of TagB in EB890 lysate but no
corresponding signal from EB889 lysate. Equivalent lysate loading was
verified by comparison of FtsY protein levels. Fig. 2A also highlights our
inability to detect endogenous levels of TagB in *B. subtilis*, presumably
due to a low copy number of this protein, a possibility that has been
previously noted upon analysis of the ribosome binding site and codon
usage of the *tagB* gene (7).

To determine the localization of TagB *in vivo*, lysate from strain
EB890 was fractionated by differential centrifugation. Immunodetec-

---

**FIGURE 2.** TagB is enriched in the membrane of *B. subtilis* 168. *A*, immunodetection of
TagB in lysate from strains EB889 (lane 1) and EB890 (lane 2). Immunodetection of FtsY
protein levels was used as a loading control. *B*, immunodetection of TagB from lysate
(lane 1), soluble lysate (lane 2), and membrane (lane 3) fractions of EB890 following dif-
ferential centrifugation. Immunodetections of TagD and EzrA proteins were used as
soluble lysate and membrane fractionation controls, respectively.

tion analysis of the fractionated lysate showed that TagB was almost
exclusively localized to the membrane of *B. subtilis* (Fig. 2B). Detection
of two control proteins, TagD, a soluble glycerol phosphate cytidylyl-
transferase involved in teichoic acid biosynthesis (9), and EzrA, a mem-
brane protein involved in cell division (33), was used to test the integrity
of the cellular fractionation. Fig. 2B shows fractionation between soluble
lysate and membrane samples with low levels of membrane contamina-
tion in the soluble lysate (as judged by EzrA detection). The latter find-
ing suggests that TagB may be exclusively membrane-associated.

**TagB Is Peripherally Associated with the Membrane—**Bioinforma-
tics-based prediction programs (e.g. TMHMM version 2.0 (17) and
TMPred (18)) indicated the presence of a transmembrane helix between
residues 6 and 25 of TagB. Closer inspection of this sequence indicated
an amphipathic nature to the putative helix (Fig. 3A). To discriminate
between transmembrane and peripheral membrane association of
TagB, the membrane fraction derived from strain EB890 was extracted
with chaotropic salt or alkaline treatment. Extracted samples were cen-
trifuged through a sucrose cushion to enhance separation of extracted
protein from the membrane fraction. In contrast to the unextracted
control, samples extracted with either chaotropic salt (1.5 M NaI) or
alkaline conditions (0.1 N NaOH) showed complete extraction of TagB
from the membrane pellet (Fig. 3B). These results suggest that TagB is
peripherally associated with the cell membrane, an unexpected finding
given sequence-based predictions of transmembrane topology.

**TagB Associates with the Cytosolic Face of the Membrane—**We con-
ducted a topological analysis of the interaction of TagB with the mem-
brane to determine whether TagB associated with the cytoplasmic or
extracellular face of the cell membrane. Protoplasts and inverted mem-
brane vesicles generated from strain EB890 were used in a traditional
protease protection assay. Reactions were separated by centrifugation
to discriminate between protected species and protease-resistant spe-
cies. Following protease treatment of EB890 protoplasts, the majority of
TagB protein was detected in the pelleted fraction and observed to be of
similar size to full-length protein (Fig. 3C, top panel, compare lanes 2
and 4). A minor amount of a faster migrating, immunoreactive TagB
species was detected in the supernatant fraction. We believe that this
represents a protease-resistant form that probably derived from the
small amount of TagB found in the supernatant fraction of the
untreated sample. Neither the protected nor the resistant TagB species

NOVEMBER 4, 2005 • VOLUME 280 • NUMBER 44
JOURNAL OF BIOLOGICAL CHEMISTRY 36695

In Vivo and in Vitro Characterization of B. subtilis TagB

FIGURE 3. TagB is peripherally associated with the intracellular face of the cell membrane. A, helical wheel representation of residues 6–25 from TagB. Note the alignment of the polar residues (Asn-9, Ser-16, and Gln-23) to one face of the helix (shown in black). B, immunodetection of TagB in EB890-derived membranes extracted with chaotropic agent (Nal) or alkali treatment (NaOH). Reactions were centrifuged through a sucrose cushion and separated into top (lane 1), middle (lane 2), and bottom (lane 3) fractions (see "Experimental Procedures"). C, immunodetection of TagB in EB890-derived protoplasts (top panels) or inverted membrane vesicles (IMVs, bottom panels) following treatment with protease (Proteinase K; PK), in the presence or absence of detergent (Triton X-100; TX). Reactions were separated into supernatant (lanes 1, 3, and 5) and pellet (lanes 2, 4, and 6) fractions by centrifugation. Full-length TagB is the slower migrating species.

was evident when protoplasts were incubated with detergent prior to incubation with protease.

In contrast to the protoplast samples, a full-length species of TagB was not evident when inverted membrane vesicles were incubated with protease in the absence of detergent (Fig. 3C, bottom panel, compare lanes 2 and 4). We noted an increase in the protease-resistant immunoreactive species of TagB found in the supernatant. Again, this proteaseresistant form of TagB was not observed in the presence of the detergent. Thus, the data from the inverted membrane vesicle experiment complements that of the protoplast studies and clearly demonstrates that TagB is localized to the cytoplasmic face of the cell membrane. This topology has also been independently demonstrated using the reciprocal reporter enzyme methodology of Manoil (34), where fusions to the *E. coli lacZ* and *phoA* genes were made to *tagB* after the 30th (following the putative amphipathic helix) and penultimate codons (data not shown).

Fluorescence Microscopy Allows in Vivo Visualization of TagBGFP Fusion Protein—To localize TagB in vivo, a translational fusion was made to the green fluorescent protein at the carboxyl terminus of TagB. Upon examination of strain EB892 by fluorescence microscopy, we found GFP fluorescence predominantly localized to the cell periphery of *B. subtilis* (Fig. 4A). Imaging of live EB892 cells showed fluorescence signal localized in a ringlike structure when cells were examined in cross-section (Fig. 4C). These data suggest that TagB is localized to the cytoplasmic face of the membrane in *B. subtilis*, consistent with the results obtained by biochemical analysis. Indeed, immunodetection of TagBGFP in soluble lysate and membrane fractions of strain EB892 showed essentially the same enrichment in the membrane fraction as that seen with strain EB890 (data not shown). Thus, we conclude that the GFP fusion does not impact on the localization of TagB. Interestingly, we noted that GFP fluorescence was significantly increased at the junction between tightly apposed cells, even in the absence of an obvious division between adjacent cells as judged by light microscopy (arrows in Fig. 4, A and B).

The TagBGFP fusion protein was also tested for its ability to complement a *B. subtilis* strain (L5058) carrying the *tag-1* allele that contains a lethal thermosensitive mutation in *tagB* (2, 7). We observed that the TagBGFP fusion was able to complement the *tag-1*-containing strain when grown at the restrictive temperature in a manner similar to unfused TagB (data not shown). This indicates that the fusion protein was functional in vivo and, therefore, that the localization of the TagB-GFP fusion protein was physiologically relevant.

FIGURE 4. TagBGFP is localized to the cell periphery and septa of *B. subtilis* 168. EB892 cells were examined by fluorescence microscopy (*A* and *C*) and by light microscopy (*B* and *D*). The arrows highlight putative cell division sites. Bar, 2 μm. Note that cells shown in *C* and *D* were imaged live, whereas cells in *A* and *B* were fixed.

In Vitro Incorporation of Labeled Glycerol Phosphate onto a *B. subtilis* Membrane Acceptor—Schertzer and Brown (11) reported on an assay that monitored the incorporation of labeled glycerol phosphate, from substrate CDP-glycerol, onto an uncharacterized acceptor in *B. subtilis* membranes. That assay was robust in demonstrating the polymerization of glycerol phosphate by TagF. We reasoned that the same assay might permit the detection of a single glycerol phosphate molecule incorporated onto the presumed undecaprenyl-pyrophosphate-disaccharide acceptor in *B. subtilis* membranes. However, we were cognizant that the anticipated signal from a “primase” activity would be much lower than that for the polymerase. In question is the amount of prenylpyrophosphate-disaccharide acceptor in the membrane preparations. In addition, the anticipated signal generated by the primase would be significantly lower than the polymerase, due to the fact that the latter incorporates some 35 residues of glycerol phosphate for every residue incorporated by the primase. Accordingly, we modified the assay to utilize a ${}^{32}$P-labeled CDP-glycerol substrate. [${}^{32}$P]glycerol phosphate starting material was not commercially available, so we employed a two-step enzymatic synthesis for the production of [$\beta$-${}^{32}$P]CDP-glycerol. In the first enzymatic reaction, glycerol was phosphorylated with [$\gamma$-${}^{32}$P]ATP using glycerokinase, a step that went to 99% completion based upon HPLC analysis (data not shown). The labeled glycerol phosphate was then converted to CDP-glycerol using glycerol-3-phosphate cytidylyltransferase and CTP (see “Experimental Procedures”). Typically, this reaction went to ~95% completion (data not shown).

Given the membrane localization of TagB in vivo and the prediction for a membrane targeting sequence at the amino terminus of TagB, we were mindful that our recombinantly purified HisTagB had a significant extension (22 amino acids) at its amino terminus. Therefore, we designed a second recombinant TagB construct that placed a hexahistidine fusion at the carboxyl terminus of the protein, leaving native sequence at the amino terminus intact (TagBhis). We noted that expression of this construct yielded more soluble protein than the amino-terminal fusion. Using recombinantly purified TagBhis and [$\beta$-${}^{32}$P]CDP-glycerol in the membrane incorporation assay, we were able to observe low but significant incorporation of radiolabel into the membrane fractions of both *B. subtilis* 168 and W23. As shown in Fig. 5A, this assay was linear for at least 30 min with 50, 100, and 200 nM TagBhis. Assay linearity was also observed with membranes isolated from *B. subtilis* W23 (data not shown). A turnover number of 0.25 h${}^{-1}$ for TagBhis was calculated (Fig. 5A, inset) from this finding.

Given the slow turnover for this enzyme, we attempted to determine whether the amount of membrane acceptor (*i.e.* *B. subtilis* membranes)

In Vivo and in Vitro Characterization of B. subtilis TagB

**TABLE TWO**

Chemical characterization of TagB reaction product

| Treatment | Sample | Radioactivity extracted |
|-----------|--------|-------------------------|
|           |        | %                       |
| Mild acid$^a$ | TagB reaction product$^b$ | 65.2 ± 7.3 |
| Butanol   | TagB reaction product$^c$ | 64.0 ± 16.8 |
|           | CDP-glycerol$^c$ | 0.071 ± 0.009 |

$a$ 0.1 M HCl for 10 min at 100°C.  
$b$ Reaction performed in duplicate.  
$c$ Reaction performed in triplicate.

TagBhis could be extracted by either treatment. As a control, the butanol partitioning of the labeled CDP-glycerol substrate was also tested and found to be 900-fold less efficient (see TABLE TWO).

We further characterized the TagBhis reaction product via size analysis of the mild acid hydrolysate using size exclusion chromatography. As shown in Fig. 6A, the cleaved TagBhis reaction product eluted in two overlapping peaks. The larger of the two peaks centered on 12.7 min and co-eluted with a polycytidine 5-mer (12.7-min elution; data not shown). The smaller peak eluted at ~14 min and co-eluted with a glycerol phosphate standard (13.9 min elution; data not shown). Prolonged concentrated acid treatment of the original acid hydrolysate resulted in the elution of the bulk of the radioactive signal at 14 min (Fig. 6B). Control reactions showed no appreciable radioactive signal in the absence of TagB protein.

**In Vitro Primase Activity Is Specifically Attributable to TagBhis—To verify that the activity we observed in the membrane incorporation assay was due to TagBhis and not to a contaminating enzyme, we attempted to “knock down” the TagBhis activity. In the accompanying paper (22), we showed that two conserved histidine residues in TagB and TagF were strictly required for their ability to complement a temperature-sensitive lesion in their respective allele *in vivo*. In addition, the histidine variants severely abrogated TagF activity *in vitro* (22). We thought to extend these results to TagB and purified a carboxyl-terminal hexahistidine-tagged version of TagBH126A (TagBH126Ahis) to determine its activity in the *in vitro* primase assay. Wild type (TagBhis) and variant (TagBH126Ahis) enzymes were assayed, and the results are shown in Fig. 7. TagBH126Ahis showed ~75% less activity relative to the wild type protein. A representative aliquot of both enzyme preparations was examined by SDS-PAGE and Coomassie staining to verify similar levels of TagB protein in the assay (Fig. 7, inset). Although TagB levels were similar between the two enzyme preparations, it was readily apparent that the relative amount of contaminating protein in the TagBH126Ahis preparation was significantly higher. We reasoned that if the observed assay activity was due to a contaminating factor(s) in the sample preparation, one would expect a higher activity from the TagBH126Ahis preparation. That we actually observed a decrease in activity with this sample strengthens our conclusion that the observed glycerophosphotransfer activity is attributable to TagB.

**DISCUSSION**

The synthesis of polyol phosphate polymers linked to peptidoglycan, wall teichoic acid in Gram-positive bacteria, remains largely uncharacterized by biochemical experimentation. In particular, the critical steps of priming, polymerization, and linkage of poly(glycerol phosphate) teichoic acid to the N-acetylmuramic acid residues of peptidoglycan continue to be a puzzle. Based largely on its homology to TagF, the Tag polymerase, it has been posited that the TagB protein may catalyze either the primase or ligase reaction, where these functions are expected

In Vivo and in Vitro Characterization of B. subtilis TagB

the first direct biochemical characterization of TagB and implicate TagB as the Tag primase in *B. subtilis* 168.

Our *in vivo* studies of TagB highlighted several interesting observations that may lead to a better understanding of its role in teichoic acid biogenesis. Fractionation, immunodetection, and *in vivo* fluorescence microscopy of TagB were consistent with membrane association of this teichoic acid biosynthetic protein. This is perhaps not unexpected. Teichoic acid biosynthesis occurs on an isoprenoid molecule embedded in the cell membrane, and, as such, the assembly of teichoic acid is thought to occur in close proximity to the cell membrane. TagO, the first enzyme acting in the biosynthetic pathway, is predicted to span the membrane multiple times (35). In addition, the Tag polymerase, TagF, has been shown to associate with membranes purified from *B. subtilis* (11, 15). The notable exception to this trend is the cytosolic synthesis of nucleotide-activated precursors used in the assembly of teichoic acid.

Our analysis of the association of TagB with *B. subtilis* membranes indicated a peripheral interaction, in contrast to what was suggested by sequence-based prediction programs. These predictions indicated the presence of an α-helix situated between residues 6 and 25 of TagB that would be responsible for insertion of the protein through the membrane bilayer. Although we have demonstrated that TagB is not an integral membrane protein, we were interested to note that examination of this sequence revealed a putative amphipathic α-helical motif. A similar motif has been identified at the carboxyl terminus of the MinD and FtsA proteins, both involved in cell division, and shown to be responsible for the peripheral association of these proteins with the membrane in *E. coli* (36–38). Unlike the MinD amphipathic helix, the putative helix in TagB does not contain any charged residues, an uncommon characteristic for an amphipathic helix. It may be that the polar but uncharged face of the putative TagB helix may facilitate interaction with an unidentified receptor protein situated on the membrane. Investigations into the amphiphaticity and requirement for a membrane-bound receptor protein are currently under way. However, it should be noted that even under conditions of high expression, TagB was found associated with the membrane, suggesting that membrane binding may not be saturable. This would suggest that membrane association is independent of a membrane-bound receptor.

We have also demonstrated by both protease protection experiments and reciprocal reporter enzyme activity assays that TagB is localized inside the cell and must therefore associate with the cytoplasmic face of the cell membrane. This finding makes TagB an unlikely candidate for lipid modification. Lipoproteins are thought to be restricted to the extracellular face of the cell membrane in Gram-positive bacteria (39, 40). This suggests that the LANC sequence found between residues 7 and 10 of TagB is not a *bona fide* lipobox, despite matching the canonical LXXC sequence. In support of our findings, a comprehensive bioinformatic study of the lipid-modified protein complement in *B. subtilis*, conducted by van Dijl and co-workers (41), failed to identify TagB.

We have confirmed the membrane localization of TagB *in vivo* by visualization of a fusion of TagB to the green fluorescent protein. This represents the first *in vivo* localization of a protein involved in teichoic acid synthesis and paves the way toward investigating the potential interactions between the teichoic acid biosynthetic machinery and other cellular components involved in cell wall biosynthesis in *B. subtilis*. Of note, we did not observe TagBGFP localization into specific structures such as filaments or helices. This would suggest that TagB does not interact with the newly identified cytoskeletal proteins Mbl and MreB that have been shown to influence nascent peptidoglycan incorporation (42, 43). It should be noted, however, that our fluorescent construct was highly overexpressed with respect to native protein levels,

FIGURE 6. Size analysis of *in vitro* TagB reaction product. Membranes from *B. subtilis* 168 and labeled CDP-glycerol were incubated with TagBhis (solid lines) or control buffer (dashed lines). Membranes were collected by ultracentrifugation and washed prior to extraction with mild acid treatment (see "Experimental Procedures"). The soluble material was recovered after ultracentrifugation and analyzed by HPLC-based gel filtration chromatography (*A*). The mild acid-extractable material was subjected to prolonged concentrated acid hydrolysis (see "Experimental Procedures") and reanalyzed by HPLC-based gel filtration chromatography (*B*). Note that 45 and 15 μl of sample were injected for chromatograms *A* and *B*, respectively.

FIGURE 7. Effect of H126A mutation on TagBhis activity *in vitro*. Wild type TagBhis (200 nM; dark gray bar), TagBH126Ahis (equivalent amount; light gray bar), and an uncatalyzed control (white bar) were reacted with 3 mg/ml *B. subtilis* W23 membranes for 90 min. Reactions were performed in quadruplicate. Inset, volumes of TagBhis (lane 1) and TagBH126Ahis (lane 2) equivalent to those used in the assay were analyzed by SDS-PAGE and Coomassie staining. Approximate protein sizes (in kDa) are listed. TagB protein is the prominent species below the 45-kDa marker.

to be intracellular and extracytoplasmic, respectively. In the work described here, we have provided the first evidence for membrane association of this protein and have shown that TagB associates peripherally with the cytoplasmic face of the cell membrane. These observations, together with data demonstrating the enzymatic incorporation of glycerol phosphate from precursor CDP-glycerol to a membrane-bound acceptor that is specifically attributable to recombinant TagB, represent

and this may have precluded our ability to observe discrete localization patterns. Nevertheless, we were interested to note a relatively large signal for TagBGFP protein at potential division sites, even when individual cells could not be discriminated by light microscopy. Given the indispensable nature of teichoic acid biosynthesis, this observation tempts us to renew the suggestion that there may be a role for Tag enzymes in cell septation. Karamata's group (44) has previously suggested that specific Tag enzymes are involved in septum biogenesis, leading to a difference in the teichoic acid structure between septum and cell cylinder. However, it is unclear if the increased intensity of GFP signal corresponded to TagB protein in the closely apposed membranes of newly divided cells.

Prior to this work, the only clue to the biochemical function of TagB was its homology to the TagF protein and preliminary analysis of the *tag-1* mutant strain. These two proteins are part of a growing family of TagF-like proteins, identified bioinformatically, although possessing no sequence similarity to proteins of known function (22). With the recent demonstration that recombinant TagF could catalyze poly(glycerol phosphate) polymerization *in vitro* (11), it was plausible that, based on its homology to TagF, the TagB protein was a candidate to catalyze the uncharacterized priming or peptidoglycan attachment reactions, both of which are chemically related to that carried out by the polymerase.

The demonstration that TagB could catalyze the incorporation of \[^{32}P\] glycerol phosphate from \[\beta-{^{32}}P\]CDP-glycerol to a membrane-bound acceptor is the first observed direct biochemical activity for TagB to our knowledge. Although the level of signal relative to background was modest, the assay was robust, as evidenced by the linearity of incorporation with both time and enzyme concentration. Whereas the estimation of TagB turnover number is significantly lower than that for TagF (~4000-fold), it is important to note that the reaction dependence on CDP-glycerol concentration was not determined in this study, and thus, we cannot be confident that the turnover number was determined at saturating concentration of this substrate. Indeed, to ensure the incorporation of a detectable amount of labeled glycerol phosphate in the TagB *in vitro* assay, the concentration of unlabeled CDP-glycerol was fixed at 50 $\mu$M, whereas the saturating concentration of CDP-glycerol in the TagF polymerase assay was determined to be 1.2 mM (11). Alternatively, the low TagB turnover value calculated in this study may be a consequence of the heat treatment of the membranes prior to use in the *in vitro* assay. We cannot rule out the possibility that such treatment reduces the efficiency of glycerol phosphate incorporation or, perhaps, inactivates other factors that may impact upon TagB activity. Interestingly, the dependencies on membrane acceptor of both TagB and TagF were very similar (apparent Michaelis constants of 400 versus 650 $\mu$g/ml, respectively), suggesting that the two reactions may be coordinated *in vivo* as suggested in models of teichoic acid biosynthesis.

Characterization of the *in vitro* TagB reaction product demonstrated that it was extractable from the membrane using both mild acid and butanol treatment. These results are consistent with the proposed composition of the reaction product (specifically, that it contains lipophilic (undecaprenyl) and acid-sensitive (pyrophosphate linkage) chemical components). It stands to reason that these two components would also be found in the membrane-bound acceptor. If anything, the addition of glycerol phosphate to the acceptor should decrease the lipophilicity of the molecule (31) while leaving the mild acid susceptibility relatively unaffected.

Furthermore, our size analysis of the TagB reaction product indicated that both glycerol phosphate and a larger molecule were liberated by mild acid extraction. In the absence of molecular standards for the proposed reaction intermediates of teichoic acid biogenesis, we previously compared the elution of the TagF reaction product with polycytidine standards. That reaction product eluted in a very broad peak centered on a 20-mer of polycytidine, consistent with a role for TagF as the poly(glycerol phosphate) polymerase (11). Here we demonstrate co-elution of the TagB reaction product with a 5-mer of polycytidine, a product considerably smaller than the TagF product and consistent with the addition of a single glycerol phosphate to the lipid-linked disaccharide. Nevertheless, we acknowledge here that, due to the inherent difficulties in characterizing the reaction product by molecular size, we cannot rule out the possibility that TagB transfers more than a single glycerol phosphate molecule to the disaccharide intermediate.

In aggregate, the data generated in this study are consistent with the hypothesis that TagB catalyzes the transfer of glycerol phosphate from CDP-glycerol to an undecaprenyl-pyrophosphoryl-disaccharide acceptor molecule. However, we cannot rule out the possibility that the Tag primase activity resides within another protein that requires TagB for activity.

Initial *in vitro* activity experiments were performed with the amino-terminally tagged version of TagB (HisTagB). We were unsuccessful at detecting meaningful incorporation of labeled glycerol phosphate onto the membrane acceptor using this protein. It is unclear if the lack of activity was due to the position of the hexahistidine fusion, perhaps impairing membrane association, or instead due to inactivation following solubilization in nonionic detergent. The latter consideration prompted us to utilize the more soluble carboxyl-terminal hexahistidine fusion protein. Our studies also indicated that membrane acceptor from both *B. subtilis* 168 and W23 strains were functional in the TagB assay. This is not surprising, given that the glycerol phosphate primase reaction is thought to occur on identical membrane-bound substrates in the two organisms, since the point of divergence between the two pathways occurs after the addition of at least one molecule of glycerol phosphate to the undecaprenyl-pyrophosphate-disaccharide unit (13). In support of this observation, the putative ortholog of TagB from *B. subtilis* W23 (TarB) shares almost 50% protein sequence identity with TagB.

The decreased activity of the TagBH126Ahis variant indicated that TagB was specifically responsible for the observed enzymatic activity in this study. In the accompanying paper (22), we describe the identification of key conserved residues in both TagB and TagF (including His^126^) whose mutation abrogates the conditional suppression of a temperature-sensitive teichoic acid biosynthesis lesion in each respective allele. The 25% residual enzymatic activity for the TagBH126Ahis variant was surprising, given that the analogous histidine variant in TagF (TagFH474A) was catalytically impaired by ~5000-fold (22). Currently, our understanding of the reaction pathways for both TagB and TagF are limited, such that it is unclear what steps in these pathways are rate-limiting to either enzyme. Given that TagF is a polymerase and TagB is a primase, it is conceivable that the rate-limiting steps may be different. Nevertheless, the histidine residue in question (TagFH474 and TagBH126) may have the same role in both reaction pathways (e.g., as a catalytic base) but be rate-limiting in our *in vitro* assay only in the case of TagF. Interestingly, despite 25% residual activity of TagBH126Ahis *in vitro*, this variant was unable to functionally complement the *tag-1* strain (22). However, it is not known what threshold level of TagB activity is required for complementation of this allele.

Nevertheless, the phenotypic parity between TagB and TagF implies a conservation of mechanism that, in addition to the evidence provided in this paper, strongly suggests that TagB is the CDP-glycerol:N-acetyl-$\beta$-D-mannosaminyl-1,4-N-acetyl-D-glucosaminyldiphosphoundecaprenyl glycerophosphotransferase, or Tag primase. Furthermore, these studies directly implicate, for the first time, the homologous

domain of the “TagF-like” proteins as a catalytic domain involved in glycerophosphotransfer.

Acknowledgments—We acknowledge the generosity of the following: Dr. Petra Levin, who provided pPL51 and anti-EzrA antibody; Dr. David Andrews who provided anti-FtsY antibody; and members of the laboratory for helpful discussions and assay reagents.

---

**REFERENCES**

1. Bhavsar, A. P., Beveridge, T. J., and Brown, E. D. (2001) *J. Bacteriol.* 183, 6688–6693
2. Boylan, R. J., and Mendelson, N. H. (1969) *J. Bacteriol.* 100, 1316–1321
3. Boylan, R. J., Mendelson, N. H., Brooks, D., and Young, F. E. (1972) *J. Bacteriol.* 110, 281–290
4. Mauel, C., Young, M., Margot, P., and Karamata, D. (1989) *Mol. Gen. Genet.* 215, 388–394
5. Briehl, M., Pooley, H. M., and Karamata, D. (1989) *J. Gen. Microbiol.* 135, 1325–1334
6. Lazarevic, V., and Karamata, D. (1995) *Mol. Microbiol.* 16, 345–355
7. Mauel, C., Young, M., and Karamata, D. (1991) *J. Gen. Microbiol.* 137, 929–941
8. Bhavsar, A. P., Erdman, L. K., Schertzer, J. W., and Brown, E. D. (2004) *J. Bacteriol.* 186, 7865–7873
9. Park, Y. S., Sweitzer, T. D., Dixon, J. E., and Kent, C. (1993) *J. Biol. Chem.* 268, 16648–16654
10. Badurina, D. S., Zolli-Juran, M., and Brown, E. D. (2003) *Biochim. Biophys. Acta* 1646, 196–206
11. Schertzer, J. W., and Brown, E. D. (2003) *J. Biol. Chem.* 278, 18002–18007
12. Soldo, B., Lazarevic, V., Pooley, H. M., and Karamata, D. (2002) *J. Bacteriol.* 184, 4316–4320
13. Lazarevic, V., Abellan, F. X., Moller, S. B., Karamata, D., and Mauel, C. (2002) *Microbiology* 148, 815–824
14. Pooley, H. M., and Karamata, D. (1994) in *Bacterial Cell Wall* (Ghuysen, J. M., and Hakenbeck, R., eds) pp. 187–198, Elsevier, Amsterdam
15. Pooley, H. M., Abellan, F. X., and Karamata, D. (1992) *J. Bacteriol.* 174, 646–649
16. Pooley, H. M., Abellan, F. X., and Karamata, D. (1991) *J. Gen. Microbiol.* 137, 921–928
17. Krogh, A., Larsson, B., von Heijne, G., and Sonnhammer, E. L. (2001) *J. Mol. Biol.* 305, 567–580

18. Hofmann, K., and Stoffel, W. (1993) *Biol. Chem. Hoppe-Seyler* 374, 166
19. Hayashi, S., and Wu, H. C. (1990) *J. Bioenerg. Biomembr.* 22, 451–471
20. Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) *Molecular Cloning: A Laboratory Manual*, 2nd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY
21. Cutting, S. M., and Youngman, P. (1994) in *Methods for General and Molecular Bacteriology* (Murray, R. G. E., Krieg, N. R., Wood, W. A., and Gerhardt, P., eds) pp. 348–364, American Society for Microbiology, Washington, D. C.
22. Schertzer, J. W., Bhavsar, A. P., and Brown, E. D. (2005) *J. Biol. Chem.* 280, 36683–36690
23. Bruckner, R. (1992) *Gene (Amst.)* 122, 187–192
24. Lin, D. C., Levin, P. A., and Grossman, A. D. (1997) *Proc. Natl. Acad. Sci. U. S. A.* 94, 4721–4726
25. Casadaban, M. J., and Cohen, S. N. (1980) *J. Mol. Biol.* 138, 179–207
26. Bradford, M. M. (1976) *Anal. Biochem.* 72, 248–254
27. Harry, E. J., Pogliano, K., and Losick, R. (1995) *J. Bacteriol.* 177, 3386–3393
28. Kao, M. C., Di Bernardo, S., Matsuno-Yagi, A., and Yagi, T. (2003) *Biochemistry* 42, 4534–4543
29. Breitling, R., and Dubnau, D. (1990) *J. Bacteriol.* 172, 1499–1508
30. Amer, A. O., and Valvano, M. A. (2002) *Microbiology* 148, 571–582
31. McArthur, H. A., Hancock, I. C., Roberts, F. M., and Baddiley, J. (1980) *FEBS Lett.* 111, 317–323
32. Anderson, R. G., Hussey, H., and Baddiley, J. (1972) *Biochem. J.* 127, 11–25
33. Levin, P. A., Kurtser, I. G., and Grossman, A. D. (1999) *Proc. Natl. Acad. Sci. U. S. A.* 96, 9642–9647
34. Manoil, C. (1990) *J. Bacteriol.* 172, 1035–1042
35. Soldo, B., Lazarevic, V., and Karamata, D. (2002) *Microbiology* 148, 2079–2087
36. Pichoff, S., and Lutkenhaus, J. (2005) *Mol. Microbiol.* 55, 1722–1734
37. Szeto, T. H., Rowland, S. L., Rothfield, L. I., and King, G. F. (2002) *Proc. Natl. Acad. Sci. U. S. A.* 99, 15693–15698
38. Hu, Z., and Lutkenhaus, J. (2003) *Mol. Microbiol.* 47, 345–355
39. Sutcliffe, I. C., and Russell, R. R. (1995) *J. Bacteriol.* 177, 1123–1128
40. Nielsen, J. B., and Lampen, J. O. (1982) *J. Bacteriol.* 152, 315–322
41. Tjalsma, H., Kontinen, V. P., Pragai, Z., Wu, H., Meima, R., Venema, G., Bron, S., Sarvas, M., and van Dijl, J. M. (1999) *J. Biol. Chem.* 274, 1698–1707
42. Jones, L. J., Carballido-Lopez, R., and Errington, J. (2001) *Cell* 104, 913–922
43. Daniel, R. A., and Errington, J. (2003) *Cell* 113, 767–776
44. Mauel, C., Bauduret, A., Chervet, C., Beggah, S., and Karamata, D. (1995) *Microbiology* 141, 2379–2389

Glycobiology and Extracellular Matrices:  
The TagB Protein in *Bacillus subtilis* 168 Is  
an Intracellular Peripheral Membrane  
Protein That Can Incorporate Glycerol  
Phosphate onto a Membrane-bound  
Acceptor in Vitro  

Amit P. Bhavsar, Ray Truant and Eric D.  
Brown  
J. Biol. Chem. 2005, 280:36691-36700.  
doi: 10.1074/jbc.M507154200 originally published online September 2, 2005  

Access the most updated version of this article at doi: [10.1074/jbc.M507154200](http://dx.doi.org/10.1074/jbc.M507154200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).  

Alerts:  
- [When this article is cited](#)  
- [When a correction for this article is posted](#)  

[Click here](#) to choose from all of JBC's e-mail alerts  

This article cites 41 references, 23 of which can be accessed free at  
[http://www.jbc.org/content/280/44/36691.full.html#ref-list-1](http://www.jbc.org/content/280/44/36691.full.html#ref-list-1)
