
Transcription in the  
Osteoblast: Regulatory  
Mechanisms Utilized by  
Parathyroid Hormone and  
Transforming Growth  
Factor-Beta  

CHRISTINE E. BOUMAH,  
NAGARAJAN SELVAMURUGAN,  
AND NICOLA C. PARTIDGE  

Department of Physiology and Biophysics,  
University of Medicine and Dentistry of  
New Jersey-Robert Wood Johnson Medical  
School, Piscataway, New Jersey 08854  

I. Introduction ..... 288  
II. Regulation of Transcription by Parathyroid  
Hormone (PTH) ..... 288  
A. Role of Osteoblasts in PTH-Induced Bone Resorption ..... 288  
B. Protein Kinase A Dependence of PTH-Induced  
Transcriptional Activation ..... 289  
C. Structure of the Rat MMP-13 Promoter ..... 293  
D. Role of AP-1 Transcription Factors in PTH Activation of  
MMP-13 Gene ..... 294  
E. Role of the RD-Binding Proteins (Runx2) in PTH  
Activation of MMP-13 Gene ..... 298  
F. Interaction Between AP-1 Factors and RD-Binding Proteins in  
PTH-Activated MMP-13 Promoter ..... 300  
G. Nucleosome Structure of the MMP-13 Promoter ..... 301  
H. PTH-Induced Changes in Acetylation of Histones  
in MMP-13 Promoter ..... 302  
III. Summary of PTH Induction of MMP-13 ..... 303  
IV. Regulation of Transcription by Transforming  
Growth Factor-beta (TGF-$\beta$) ..... 304  
A. TGF-$\beta$ in Bone ..... 304  
B. TGF-$\beta$ Signaling ..... 304  
C. TGF-$\beta$1 Regulation of MMP-13 Gene in Osteoblastic Cells ..... 305  
V. Summary of TGF-$\beta$ Actions on MMP-13 ..... 309  
VI. Conclusions ..... 310  
References ..... 311  

Progress in Nucleic Acid Research  
and Molecular Biology, Vol. 80  
DOI: 10.1016/S0079-6603(05)80007-8  
Copyright 2005, Elsevier Inc.  
All rights reserved.  
0079-6603/05 $35.00  


The osteoblast is a cell of mesenchymal origin sharing a common progenitor stem cell with chondrocytes, myoblasts, and bone marrow stromal cells including adipocytes. Preosteoblasts, however, are capable of responding to a variety of extracellular signaling pathways and further differentiating into bone-lining mature osteoblasts and osteocytes; this process of osteoblast differentiation or osteogenesis, crucial for skeletal tissue formation, repair, and maintenance, is under extensive investigation. Osteoblast differentiation is often subdivided into three stages: (1) a proliferation stage, during which the cells exit from the cell cycle and fully commit to the osteoblast phenotype; in this stage, the osteoblasts express high levels of immediate early genes (*c-fos*, *c-jun*, *c-myc*), histones, cyclins, and several other genes specific to proliferating cells ([1–3]); (2) a growth arrest stage, accompanied by development and maturation of extracellular matrix, downregulation of growth-associated genes, and expression of high levels of alkaline phosphatase and collagen ([4, 5]); and (3) a mineralization stage, with maximal osteoblastic expression of extracellular matrix components including noncollagenous bone matrix proteins such as osteocalcin, osteopontin, and bone sialoprotein ([4, 6, 7]). Osteoblast differentiation and the mechanisms responsible for the temporal and sequential display of distinct subsets of transcriptionally active osteoblastic genes were established using *in vitro* osteoblast culture models and, more recently, in *in vivo* studies with mice. Osteoblast activities are regulated, in a stage-specific manner, by hormones including parathyroid hormone (PTH) ([8, 9]), 1,25(OH)₂ vitamin D₃ ([10]), estrogen ([11, 12]), and glucocorticoids ([13, 14]). Osteoblast differentiation is also regulated by cytokines and various local factors such as transforming growth factor-beta (TGF-β) ([15, 16]) and fibroblast growth factor ([17]).

## II. Regulation of Transcription by Parathyroid Hormone (PTH)

### A. Role of Osteoblasts in PTH-Induced Bone Resorption

Parathyroid hormone (PTH) is a polypeptide hormone essential in the regulation of bone remodeling and calcium homeostasis. Secreted in response to decreases in serum calcium levels, PTH acts rapidly on the kidney to stimulate calcium reabsorption and synthesis of 1,25(OH)₂ vitamin D₃, which enhances absorption of calcium in the intestine ([18]). The other major mechanism utilized by PTH to restore serum calcium levels involves degradation of bone matrix (bone resorption) with concomitant release of stored calcium ([19]). Bone is a mineralized tissue made largely of extracellular matrix and containing

three major cell types: osteoblasts or bone-forming cells, osteocytes which are fully differentiated osteoblasts embedded in bone matrix, and osteoclasts or bone-resorbing cells. PTH-induced destruction of bone matrix has been intensively studied and the role of osteoblasts clearly established although not entirely elucidated. Osteoclasts produce cysteine proteinases (particularly high cathepsin K levels) and several matrix metalloproteinases (MMPs) such as MMP-9 and MT1-MMP (20, 21) but not the collagenase MMPs capable of specifically degrading the most abundant bone matrix protein, type I collagen (22). The membrane receptor through which PTH acts in bone, the PTH 1 receptor (PTH1R), is not present on osteoclasts but is found exclusively on cells of the osteoblast lineage. Furthermore, *in vitro* studies on osteoclast attachment and pit formation indicated that collagenases produced by osteoblasts during bone remodeling effectively degraded the layer of matrix on bone surfaces, thereby allowing the macrophage-like osteoclasts to attach and initiate resorption (23, 24). Osteoblasts also produce RANKL (receptor activator of NF-κB ligand), a transmembrane protein that stimulates osteoclasts' differentiation and activity (25, 26). Thus, the bone-forming osteoblasts play a crucial role in bone resorption by preparing the matrix for osteoclast attachment, by recruiting and activating osteoclasts to initiate degradation by acid-pH acting proteases such as cathepsin K (20). The aim of this section of the chapter is to report our progress toward the elucidation of the transcriptional processes utilized by PTH to alter the function and phenotype of osteoblasts from bone-forming cells to cells involved with the bone resorption process.

## B. Protein Kinase A Dependence of PTH-Induced Transcriptional Activation

When osteoblastic cells are exposed to PTH, they cease producing collagen, alkaline phosphatase, osteonectin, and osteopontin (27, 28) and start synthesizing osteocalcin, cytokines, and neutral proteases (collagenases) (29–31). The first indication that these changes were due to effects on mRNA abundance was provided by studies done on rats that had been injected with doses of PTH sufficient to cause serum hypercalcemia (32, 33); the bones from PTH-treated rats produced considerably higher collagenolytic activities than did those from uninjected rats (32) and hypercalcemia was abolished by co-treatments with actinomycin D (34). In a later study, the involvement of transcriptional and post-transcriptional events in PTH action on osteoblastic cells was established by studying the UMR 106-01 (UMR) rat osteosarcoma cell line. This cell line displays many osteoblastic markers including PTH1 receptors, type I collagen synthesis, and high alkaline phosphatase and, when treated with PTH, UMR cells typically decrease collagen synthesis and begin producing collagenase (35, 36). This rodent enzyme was later found to be

homologous to human collagenase-3, finally assigned the name matrix metal-
loproteinase-13 or MMP-13. Northern blots and nuclear run-on assays with
PTH-treated UMR cells showed 100-fold induction in the rate of synthesis of
MMP-13 mRNA at 2 h after PTH treatment, with steady state levels of mRNA
increased 180-fold at 4 h (37); mRNA levels were shown to decline to approxi-
mately 30% of maximum by 8 h. The induction of MMP-13 transcription was
detected only after a lag period of at least 1 h and was completely abolished by
cycloheximide; these observations suggested that PTH effects on the MMP-13
gene in UMR cells are secondary responses requiring expression of other
genes. A comprehensive analysis of gene expression profile changes in UMR
cells treated with $10^{-8}$ M rat PTH (1-34) was conducted in our laboratory using
DNA microarray analysis (38). There were 125 known genes and 30 unknown
expressed sequence tags (ESTs) found to have at least 2-fold expression
changes after PTH treatment at 4, 12, and 24 h. Subsequent real-time reverse
transcriptase-PCR confirmed that 90% of these genes were regulated more
than 2-fold; these included hormones, cytokines/growth factors, receptors,
signal transduction pathway proteins, transcription factors, proteases, and
structural and matrix proteins (Table I). The mechanisms by which PTH
affects expression of these genes in osteoblasts is under investigation in our
laboratory; the present discussion will focus on transcriptional activation of
MMP-13.

Close examination of the structure of the membrane receptor, PTH1R, has
provided insights into the potential signal transduction pathways utilized by
PTH. The PTH1R in osteoblastic cells belongs to the group of seven trans-
membrane domain receptors that are coupled to heterotrimeric G proteins
(39). One potential signaling pathway for PTH action involves the G-alpha
proteins ($G_{\alpha_s}$) which are known to activate adenylate cyclase and stimulate
formation of cyclic 3', 5'-adenosine monophosphate (cAMP); cAMP, in turn,
binds the regulatory subunits of protein kinase A (PKA) to release the catalytic
subunits of the enzyme. The catalytic subunit of PKA phosphorylates proteins
on serine residues in the recognition sequence R-X-X-S and causes changes in
structure and function of several proteins, particularly in transcription factors
such as Cbfal (core binding factor $\alpha$)/Runx2 and cAMP-response element-
binding protein CREB, a family of proteins regulating the activity of several
other genes. Alternatively, interaction of PTH with PTH1R also leads to the
activation of phospholipase C$\beta$ by another set of G proteins, the $G_{\alpha_q}$ proteins
which, from the breakdown of phosphatidylinositolphosphate and production
of diacylglycerol (DAG) and 1,4,5-inositol trisphosphate (40, 41), causes the
release of intracellular calcium ions and activation of protein kinase C (PKC)
(Fig. 1). PTH has also been shown to stimulate non-phospholipase C$\beta$ depen-
dent activation of the PKC pathway (42). Although both the cAMP-PKA and
the PKC pathways play a role in various PTH effects (separately or conjointly),

TRANSCRIPTION IN THE OSTEOBLAST

TABLE I  
SELECTED GENES REGULATED BY PTH IN OSTEOBLASTIC CELLS$^a$

| Hormone, growth factor | Signal transduction |
|------------------------|---------------------|
| frizzled related protein (frpAP) | cAMP phosphodiesterase |
| amphiregulin | MKP-1 |
| TGF-alpha | gene 33 |
| Jagl | serum and glucocorticoid-regulated kinase |
| preprorelaxin | Ruk |
| IL-18 | OST-PTP |
| placental pre-proGHRH | protein-tyrosine kinase, JAK3 |
| BMP3 | protein phosphatase inhibitor-1 |
| BMP4 | Transcription factor and regulator |
| BMP7 | c-fos |
| vascular endothelial growth factor B | rNFI L-6, C/EBP $\beta$ |

| Receptor |  |
|----------|---|
| fast nerve growth factor receptor | Lot1 |
| c-kit receptor tyrosine kinase | TIEG |
| NOR-1 | CREMdeltaC-G gene |
| Ret | CREM repressor |
| syndecan 4 | Nurr77 |
| interferon gamma receptor | hairless |
| urinary plasminogen activator receptor | LIM protein, FHL2 |
| Rev-ErbA-alpha | Maf1 |
| growth hormone receptor | Maf2 |
| nonselective-type endothelin receptor | Protease and regulator |
| PTH receptor | MMP-13 |
| Trk | tissue-type plasminogen activator |
| 5-hydroxytryptamine receptor | TIMP-1 |

| Cell cycle and apoptosis |  |
|---------------------------|---|
| cyclin D1 | Pace4 |

---

$^a$ UMR 106-01 cells were treated with $10^{-8}$M PTH (1–34) for 4, 12, and 24 h. RNA was collected and analyzed with Affymetrix RG U34A chips. Fold changes in gene expression were determined by quantitative real-time RT-PCR.

evidence accumulated thus far has failed to show any direct involvement of the PKC pathway in PTH-induced changes in mRNA of most osteoblastic genes and it appears that the physiological actions of PTH are, for the most part, mediated through the cAMP and the PKA pathway. In calvariae from fetal rats, cAMP levels increased significantly within 1 min of PTH addition, reached

Ca<sup>++</sup>
AC
α<sub>s</sub>
β
γ
α
PLCβ
PIP<sub>2</sub>
DAG
PKC
Ca<sup>++</sup>
ATP
CAMP
PKA
PKA (active)
GSK
ser129
ser133
CREB
CRE
fos
Rurα2
AD
MMP-13
Nucleus
PTH
IP<sub>3</sub>
Ca<sup>++</sup>
ER

Fig. 1. PTH receptor signaling in osteoblastic cells.

maximum after 5 min, and the magnitude of the response was proportional to PTH concentration (43). Expression of bone sialoprotein *bsp* (a major noncollagenous protein) in primary cultures of embryonic osteoblasts was shown to increase nearly 4-fold after PTH treatment; co-transfection of a *bsp* promoter/reporter construct with a plasmid encoding the inhibitor of cAMP-dependent protein kinase abolished PTH-induced *bsp* transcription and also decreased basal transcription. PKC was found capable of stimulating *bsp* expression independently of PTH since inhibitors of PKC had no effect on PTH induction of *bsp* (44, 45). Our studies with MMP-13 in UMR cells have shown that the cAMP analog 8-bromo-cAMP (8Br-cAMP) was capable of inducing gene expression to levels close to those of PTH while neither the PKC activator, PMA, nor the calcium ionophore ionomycin, when used alone, caused any change (37). Therefore, in UMR cells, PTH induces MMP-13 transcription primarily through activation of cAMP and the PKA signaling pathway, rather than through PKC or calcium signaling, and this response requires expression of other genes.

### C. Structure of the Rat MMP-13 Promoter

Rat interstitial collagenase or collagenase-3 (MMP-13) belongs to the family of MMPs which are responsible for degrading components of the extracellular matrix (46, 47). The MMP family includes gelatinases, stromelysins, membrane-type MMPs, and collagenases. Three types of collagenase have been identified in humans: collagenase-1, -2, and -3; the rat and mouse interstitial collagenase shows 86% homology with human collagenase-3 and has been termed collagenase-3, also called MMP-13 (35). Rat and mouse MMP-13 are expressed by a variety of cells including fibroblasts (48), hypertrophic chondrocytes (49), and osteoblasts (50, 51) and have been shown to be important in endochondral ossification and bone cell function. MMP-13 cleaves a single site in type I, II, III, and X collagens, which are the major components of cartilage and bone (47). Mutation of the collagenase cleavage site in the type I collagen *Collal* gene results in decreased PTH-induced bone resorption, decreased PTH-induced calcemic responses, and thicker bones in homozygous mutant mice (52). MMP-13 expression is stimulated by bone resorbing agents such as PTH and interleukin-6, and is suppressed by factors associated with bone growth and formation such as bone morphogenic proteins, BMPs, and insulin-like growth factors (53–55). Improper regulation of MMP-13 contributes to several pathological conditions including tumor metastasis, rheumatoid arthritis, and osteoarthritis (56–59).

The rat MMP-13 cDNA was cloned in our laboratory using a cDNA library prepared from the rat UMR cells treated with \(10^{-8}\) M PTH: the gene consists of 10 exons spanning approximately 12 kbp (60); the 2.9 kb mRNA encodes the proenzyme with a predicted molecular weight of 52 kDa (35). A large stretch

of promoter region was isolated, sequenced, and used to identify the regulatory elements involved in the PTH response as well as the primary responsive genes which transduce the hormonal signal and bind to this region of the MMP-13 gene. Transient transfections of UMR cells with various 5′-deleted and internally mutated rat collagenase-3 promoter constructs revealed that the minimum PTH-responsive region is within 148 base pairs upstream of the transcriptional start site (61, 62). This region contains a typical TATA box and consensus binding sites for several transcription factors including C/EBP (CAAT enhancer-binding site), the AP (activator protein)-1 and -2, RD (runt domain-binding sites), PEA-3 (polyoma enhancer activator-3), p53, as well as a homopolymeric(dA:dT) element identified as the binding site for the transcription factor Nmp4/CIZ (nuclear matrix protein-4/cas interacting zinc finger protein) (61, 63–66) (Fig. 2). Mutation-deletion studies within the minimum PTH-responsive region indicated that while PEA-3 and possibly p53 sites did not contribute to PTH stimulation, both the AP-1 site and RD site were required for basal expression and PTH activation of the gene (Fig. 3). Using gel-shift analysis, the AP-1 factors and runt proteins binding the collagenase-3 promoter AP-1 site and RD in the PTH-treated UMR cells were identified as c-Fos/c-Jun and members of the acute myelogenous leukemia (AML)/runt domain transcription factor family, respectively.

# D. Role of AP-1 Transcription Factors in PTH Activation of MMP-13 Gene

The AP-1 transcription factors are immediate early response genes rapidly and transiently induced upon cellular stimulation by growth factors, chemokines, and cytokines, as well as non-mitogenic stimuli such as PTH (67, 68). The AP-1 complex consists of a heterodimer composed of members of the Fos (c-Fos, Fos-B, Fra-1, and Fra-2) and Jun (c-Jun, JunB, and JunD) family proteins (69, 70) or a homodimer of the Jun members (c-Jun, Jun B, Jun D). This complex binds the consensus DNA sequence TGA(G/C)TCA, termed 12-O-tetradecanoylphorbol-13-acetate (TPA)-response element, TRE, or AP-1 site, present in a variety of promoter regions including MMP-13 (71). Fos and Jun are members of the basic region-leucine zipper (bZIP) family of sequence-specific dimeric DNA-binding proteins (72). The carboxy-terminal region of the bZIP domain is amphipathic and contains a heptad repeat of leucines that is critical for the dimerization of bZIP proteins, whereas the amino-terminal region contains the basic domain that is required for sequence-specific DNA binding. The most stable, most transcriptionally active, and most predominant AP-1 complexes are those composed of c-Fos/c-Jun factors and are often simply termed the AP-1 factors. The presence of AP-1 binding sites in the MMP-13 promoter and the fact that the AP-1 site is required for the basal and

Rat MMP-13 Promoter

-219 -216 -204 -201 -132 -126 -119 -110 -101 -92 -78 -71 -62 -55 -48 -42 -36 -31 -26 +1
SBE     SBE     RD      Nmp4/CIZ p53    PEA3   AP-2   RD AP-1 C/EBP TATA

FIG. 2. Schematic representation of the promoter of the rat MMP-13 gene. From right to left are sites for the following consensus sequences: C/EBP, AP-1, Runx2 proximal, AP-2, PEA-3, p53, Nmp4/CIZ, Runx2 distal, SBE.

FIG. 3. Analysis of the CAT activity of rat MMP-13 promoter constructs with RD site and/or AP-1 site mutation. The wild type and mutant constructs were transiently transfected into UMR 106-01 cells, treated with control or rat PTH (1–34, $10^{-8}$M) for 24 h, and then assayed for CAT activity. Basal CAT activities (inset) are shown relative to the WT (-148) construct. The PTH stimulation is shown as —fold stimulation over control. Data represent mean ± S.E. of three replicate plates (61).

PTH response suggests that the AP-1 factors play an important role in bone remodeling; indeed, MMP-13 expression was shown to be considerably lower in c-Fos deficient (fos$^{-/-}$) mouse embryos compared to wild type embryos (62). Moreover, evidence from breast cancer and other cell lines suggests that the differential expression of and regulation of MMP-13 in osteoblastic compared to non-osteoblastic cells depends largely upon the expression of AP-1 factors. Indeed, overexpression of c-Fos in transgenic mice has been associated with development of osteochondromas and osteosarcomas (73) and mice containing the c-Fos knockout exhibit skeletal abnormalities related to osteoclast dysfunction including osteopetrosis (74).

PTH transiently increases the mRNA expression of c-Fos and c-Jun in UMR cells with concomitant increases in binding to the AP-1 site in the collagenase-3 promoter (75); these increases were rapid, appeared to be primary responses not requiring protein synthesis, and reached maximum levels within 30 min of PTH addition, well before the maximum transcription time for collagenase-3 (68, 75). These observations, along with the fact that overexpression of c-Fos and c-Jun in proliferating osteoblasts increased collagenase-3 promoter activity, strongly suggest that c-Fos and c-Jun are the immediate early response genes required for collagenase-3 activation by PTH. The mechanisms involved in PTH regulation of c-Fos in osteoblastic cells were investigated earlier in transient transfection experiments with c-Fos promoter 5′-deletion constructs and demonstrated a requirement for the major cAMP response element (CRE) present in the c-Fos promoter region (68). Point mutations in the major CRE inhibited both PTH-stimulated and basal expression. Gel retardation and Western blotting revealed that CRE-binding protein (CREB) constitutively binds the major CRE but becomes phosphorylated at its cAMP-dependent PKA consensus recognition site following PTH treatment. In other studies from our group, it was shown that PKA phosphorylated CREB at serine 133 (S133) and that this event was required for PTH-induced c-Fos expression. S133 phosphorylation correlated in a time- and dose-dependent manner with the time and level of activation of PKA. Specific inhibitors of calcium/calmodulin-dependent protein kinases and PKC did not inhibit CREB phosphorylation at S133 nor did they affect c-Fos expression in response to PTH; however, H-89, a specific inhibitor of PKA, abolished both CREB phosphorylation and c-Fos gene activation (76). Other phosphorylation events associated with CREB include phosphorylation by casein kinase II (CK II) at serines 108, 111, 114, 117, and 121 and phosphorylation by glycogen synthase kinase-3 (GSK-3) at serine 129 (S129) in both unstimulated and PTH-treated UMR cells. However, while CK II phosphorylation events were found unnecessary, S129 phosphorylation by GSK-3 was essential for the basal and (together with PKA-dependent S133 phosphorylation) PTH-induced trans-activation of CREB (77).

E. Role of the RD-Binding Proteins (Runx2) in PTH Activation of MMP-13 Gene

The RD-binding site is bound by members of the Cbfa (core-binding factor α) family, which are also named Runx proteins, α-subunits of polyoma enhancer binding protein 2 (PEBP2α), AML, as well as osteoblast specific factor-2 (Osf-2). Members of this family include the *Drosophila* Runt and Lozenge proteins, and the human and murine Cbfa1, Cbfa2, and Cbfa3, also named Runx2, Runx1, and Runx3 (78, 79). Runx proteins contain a DNA-binding domain of 128 amino acids called the runt domain and are capable of binding to DNA as monomers, but Runx2 can also heterodimerize with Cbf-β (core-binding factor β), a ubiquitously expressed nuclear factor (80). Cbf-β does not directly bind to DNA, but increases the affinity of Runx2 for DNA (81, 82). Three major isoforms for Runx2 have been described and have been designated Types I, II, and III; they differ in their 5′ N-terminus using different translational start sites but share similar functional protein domains: the runt domain, a C-terminal proline-serine-threonine-rich (PST) domain, a short Myc-related nuclear localization sequence, and a glutamine-alanine (QA)-rich region (83, 84) (Fig. 4). Deletion analysis of Type III Runx2 revealed one repression domain and three activation domains (AD1, AD2, and AD3). The main repression domain of Runx2 is composed of the last 154 amino acids of the C-terminus. Additionally, the last five amino acids, the VWRPY domain, alone can act as a transcriptional repressor, possibly through interaction with TL2. This is the human homolog of Groucho, a *Drosophila* repressor protein shown previously to interact with the VWRPY domain of Runt (84, 85). The first activation domain is unique to the Type III isoform and is located in the first 19 amino acids of the protein. The second and third activation domains are located in the QA and PST domains, respectively. The PST domain contains consensus phosphorylation sites for ERK, PKC, CKII, and PKA (84, 86). All three isoforms of Runx2 have been detected in bone, but only types II and III have been identified in the osteoblast (83, 87). The Runx2 gene has been cloned; the promoter contains several putative RD binding sites, as well as binding sites for AP-1, Ets-1, and the glucocorticoid receptor (88). Runx2 gene expression can be stimulated by Runx2, c-Jun, BMP 4/7, and α2-integrin signaling, and is inhibited by 1,25(OH)₂D₃ and Smad 4 (89–91).

The importance of Runx2 to bone formation is well recognized. Runx2 knockout mice die at birth and lack both skeletal ossification and mature osteoblasts (83, 92). Mutations that result in inactive or deleted Runx2 genes have been linked to cleidocranial dysplasia (CCD), a human disorder characterized by defective endochondral and intramembranous bone formation (93). Transfection of Runx2 into non-osteogenic cells, such as primary skin fibroblasts, induces transcription of osteoblast-specific genes, while disruption

TRANSCRIPTION IN THE OSTEOBLAST 299

A  
(nucleotides) (1-138) (139-260)  
Exons -1 0  
(261-647) (648-805) (806-909) (910-1084) (1085-1246) (1247-1312)(1313-1791)  
1 2 3 4 5 6 7  
MLHSPH  
(Type III)  
MRIPVD  
(Type I)  
MASNSL  
(Type II)  

B  
DNA  
binding  
1 19 49 77 241 528  
Runx2 QA RUNT PST VWRRPY  
AD1 AD2  
transactivation  
NLS NMTS  
AD3  
transactivation  
ERK ERK  
241 374  
PKC PKC ERK PKC CK2 PKC  
PKA  

FIG. 4. Structure of the mouse Runx2 cDNA (A) and domain organization of mouse type III Runx2 protein (B). The translational start sites are encoded by exons-1, 0, and 1. The runt domain is encoded by exons 1 through 3. The PKA phosphorylation site in the activation domain (AD3) is located in exon 5. NLS, nuclear localization signal; NMTS, nuclear matrix targeting signal.

by antisense oligonucleotides in osteoblastic cultures prevents development of the mature osteoblast phenotype (83).

Evidence accumulated in our studies shows that Runx2 is essential for activation of MMP-13 transcription by PTH. In transient transfection experiments, we found that overexpression of a Runx2 repressor protein inhibited MMP-13 promoter activation by PTH whereas overexpression of c-Fos, c-Jun, Runx2, and Cbfβ increased PTH response, but not with mutation of either or both the AP-1 and RD binding sites (61). This suggests a cooperative interaction between Runx2 binding to the RD site and the AP-1 factors binding to the AP-1 site in the rat MMP-13 gene of osteoblastic cells in response to PTH. The mechanism of action of Runx2 as a transcription initiator is not fully understood. Analysis of the expression pattern of Runx2 in the rat osteoblastic cell line UMR 106-01 indicated that, in contrast to the expression of AP-1 factors, there is no change in the levels of Runx2 protein or RNA either under control

conditions or after PTH treatment. Our most recent results using chromatin immunoprecipitation (ChIP) assays, however, show moderate increase (2-fold) in levels of Runx2 protein associated with MMP-13 promoter in lysates of UMR cells treated with PTH. Similar assays conducted with c-Fos and c-Jun revealed transient but substantial (over 10-fold) increases in amounts of protein bound to the rat MMP-13 promoter in UMR cells. The discrepancy between Western blot results and those from ChIP assays regarding the PTH-induced changes in levels of Runx2 associated with MMP-13 promoter could be explained by the higher sensitivity of the ChIP assays. Furthermore, the PTH-responsive region of MMP-13 contains a second putative RD site, not contributing to the PTH response (66) but possibly capable of binding amounts of Runx2 sufficient to be detected in ChIP assays combined with real time PCR. To date, the major action of PTH on Runx2 we have identified is activation of Runx2's third activation domain (AD3), found in the PST region, through phosphorylation of the PKA site. We also showed that purified PKA catalytic subunit could phosphorylate the AD3 region of Runx2 *in vitro* but not if the PKA site at amino acid 344 in AD3 was mutated (86). To study phosphorylation of Runx2 in response to PTH *in vivo*, we transfected COS-7 cells with an expression plasmid of c-Myc-tagged Runx2 or a control vector, then labeled with radioactive phosphate. The cells were treated with 8-BrcAMP, lysed, and immunoprecipitated with anti-Myc. The band showing phosphorylated Runx2 was excised and analyzed by TLC. The results indicated there is phosphorylation at serine in both control and 8-BrcAMP treated lysates, but there appears to be more serine phosphorylation after 5 min of 8-BrcAMP treatment and this phosphorylation was retained up to 30 min.

F. Interaction Between AP-1 Factors and RD-Binding Proteins in PTH-Activated MMP-13 Promoter

Numerous studies demonstrate the ability of transcription factors to bind each other and to mediate cooperative DNA binding and promoter activation when their respective binding sites are juxtaposed (94, 95). The proximity of the AP-1 (-51/-45) and RD (-135/-128) sites, and their cooperative involvement in the activation of the MMP-13 promoter by PTH, suggests that proteins binding to these sites (c-Fos, c-Jun, and Runx2) also physically interact. All three transcription factors have previously been shown to interact with other transcription factors or with components of the basic transcriptional machinery. For Runx2, this includes the androgen and glucocorticoid receptors (96), HES-1 (97), TLE2 (84), histone deacetylase 6 (98) and, more recently, TAZ (99). The list is much more extensive for Fos and Jun and includes both nuclear proteins binding to enhancer elements (100), coactivators such as CBP (101) and p300 (102), or members of the basal transcription

machinery such as TBP and the TFIID complex (103, 104). In our laboratory, gel shift analysis demonstrated that both Runx2 and c-Fos/c-Jun proteins can bind an oligonucleotide containing both the RD and the AP-1 binding sites. Co-immunoprecipitation experiments with \[^{35}\text{S}\]-methionine-labeled c-Fos, c-Jun, Runx2, Runx1, and Cbfβ proteins expressed *in vitro* by using a coupled transcription-translation procedure showed that Runx2 interacts with both c-Fos and c-Jun (105). This interaction occurred for both Runx2 and Runx1, thus suggesting that motifs common to both proteins can be responsible for AP-1 interaction. In fact, we showed that the runt domain of Runx2 alone interacts with c-Fos and c-Jun *in vitro* (105). *In vivo* yeast two-hybrid studies confirmed interaction between Runx2 and c-Fos/c-Jun proteins and this in the absence of phosphorylation and without binding to DNA (105). The importance of physical interaction between the AP-1 and the RD sites in the MMP-13 response to PTH was confirmed by nucleotide insertion analysis. The insertion of 3 nucleotides, which disrupts helical phasing between the AP-1 and the RD binding sites, decreased both the basal and PTH-induced MMP-13 promoter activity. In contrast, the insertion of 10 nucleotides, which maintains the helical phasing relationship, did not alter the activity of the promoter (105).

### G. Nucleosome Structure of the MMP-13 Promoter

In eukaryotes, genomic DNA is enclosed within nucleosomes, the fundamental building blocks of chromatin, which consists of 146 base pairs of DNA wrapped around an octamer of core histone proteins (2 copies each of histones H2A, H2B, H3, and H4). Transcriptional activation of a previously repressed gene therefore requires reorganization and, in some cases, disruption of local chromatin structure so as to give transcription factors and the transcription machinery access to DNA. In the majority of genes, the nucleosome is either dissolved completely, moved to adjacent locations, or remains at the same location while adopting a more open conformation due to changes in its components (106). We analyzed nucleosome organization of the MMP-13 promoter in nuclei from UMR cells untreated and treated with PTH for 0.5 h or for 4 h (peak of PTH-induced activation of MMP-13 gene). In the absence of PTH, we mapped DNase I hypersensitive sites (DHS) at three locations in the MMP-13 promoter: one at −10/−2, adjacent to the transcription start site, a second DHS is located in a small AT-rich section at −166/−154, and the third DHS was found at −200/−191. These findings indicate that the proximal promoter of MMP-13 gene exists in a nucleosome structure between −154 and −10, a region which encompasses the minimum PTH-responsive element (−148/−38). There was no DHS elsewhere in the stretch of the promoter we analyzed (1329 base pairs), suggesting that the MMP-13 promoter in UMR cells is entirely shielded in nucleoprotein complexes. Treatment with PTH did

not alter the position of the DHS and the PTH-responsive region remained essentially nuclease resistant (107). We concluded that PTH caused neither nucleosome dissolution nor nucleosome sliding in the activation of MMP-13 gene expression in UMR cells but there was apparent relaxation of the nucleosome.

# H. PTH-Induced Changes in Acetylation of Histones in MMP-13 Promoter

Post-translational modifications of histones, e.g., acetylation of histone H3 and H4, are believed to cause the nucleosome relaxation that precedes and is essential for transcription initiation (108). We have shown that PTH-induced activation of MMP-13 gene transcription in the rat osteoblastic cell line, UMR 106-01, is associated with changes in histone acetylation patterns in the promoter region of the gene (107). Two- to 3-fold increases in amounts of acetylated histone H4 were observed as early as 5 min after PTH addition and the amounts steadily rose to nearly 20-fold after 60 min. ChIP assays using primers to various regions of the promoter further indicated specific association of acetylated histone H4 with the −176/−68 region which includes the Runt domain (RD) binding site. There were no significant changes in acetylated histone H4 around the AP-1 binding site (nt −115/−5), in the distal promoter region (nt −1237/−970), or within the coding region (exon 10). On the other hand, changes in acetylated histone H3 were not observed until 90 to 120 min after PTH addition and histone H3 acetylation levels were uniform around the entire proximal promoter region (nt −207/−5). Histone acetylation levels were unchanged around the promoter region of the osteopontin gene, a PTH nonresponsive gene expressed in UMR cells. These observations suggest different histone acetyl transferases (HATs) functioning specifically on different histones at the two sites, perhaps reflecting differential recruitment of the enzymes by DNA-bound proteins such as Runx2.

Endogenous HAT activity from both p300 and CREB binding protein (CBP), the most common eukaryotic HAT enzymes, were elevated in UMR cells; however, p300-catalyzed HAT activity was stimulated by PTH treatment while CBP-dependent HAT activity was unaffected. Western blots from UMR cell lysates using anti-CBP or anti-p300 antibodies showed a small but significant increase in HAT protein levels as early as 15 min after PTH addition. RNA levels for both CBP and p300, as measured by real time RT-PCR compared with actin or histone H4, were increased (4- to 5-fold) only after 60 min of PTH treatment (unpublished data).

PTH-induced increases in both acetylated histone H4 and p300-dependent HAT activity were abolished when cells were preincubated with cycloheximide, suggesting that the changes require protein synthesis. Signaling by PKA was essential for the PTH-induced stimulation of acetylated histone H4

levels of the MMP-13 promoter. Inhibitors of the ERK phosphorylation pathway had no effect on acetylated histone H4 levels while PKC-dependent phosphorylation had a stimulatory effect, causing up to 90-fold increases, suggesting that PKC has a negative effect on HAT activity associated with the MMP-13 promoter (107).

# III. Summary of PTH Induction of MMP-13

A model illustrating our current understanding of the early events in the PTH induction of the MMP-13 gene in rat osteoblastic cells is shown in Fig. 5. Briefly, immediately after binding to its receptors, PTH activates the cAMP/PKA pathway and phosphorylation of several transcription factors, mainly, Runx2 and CREB. Proteins with HAT activity such as p300 or p300-related factors are synthesized *de novo* and early in the sequence of events; these associate with DNA-bound phosphorylated Runx2 in a manner similar to that described for vitamin D stimulation of the osteocalcin gene in osteoblastic cells (109). There follows increases in amounts of acetylated histone H4 associated with the RD binding site, relaxation in nucleosomal

FIG. 5. Model for the early events of PTH induction of the MMP-13 gene in rat osteoblastic cells.

organization with enhanced accessibility of the AP-1 binding site to increasing amounts of c-Fos/c-Jun dimers, release of transcription repressors (such as ATF-4), and recruitment of activator proteins which are currently being identified in our laboratory.

# IV. Regulation of Transcription by Transforming Growth Factor-beta (TGF-$\beta$)

## A. TGF-$\beta$ in Bone

The discovery of large amounts of transforming growth factor-beta (TGF-$\beta$) in bone led to the conclusion in numerous studies that TGF-$\beta$ is a major player in bone remodeling. In mammals, there are three isoforms, TGF-$\beta$1, TGF-$\beta$2, and TGF-$\beta$3, all of which are expressed by bone cells (15, 110) and interact with the known TGF-$\beta$ receptor types I, II, and III (beta-glycan) (111). Both osteoblasts and osteoclasts are able to synthesize TGF-$\beta$ (112–115). Although osteoblasts are presumably responsible for production of most of the TGF-$\beta$ entrapped in bone matrix (116), the majority of active TGF-$\beta$ appears to be generated by bone resorbing osteoclasts either through release of TGF-$\beta$ stored in bone matrix or by biosynthesis of new TGF-$\beta$ (117). Active TGF-$\beta$ may then be made accessible to osteoblasts through a process involving transcytosis.

In some osteoblast cultures, TGF-$\beta$ stimulates proliferation, and in others inhibits proliferation (1). The two major functions of TGF-$\beta$ are its effect on cell proliferation and matrix formation (118–121). TGF-$\beta$1 knock-out mice display about 30% decrease in tibial length and a reduction in bone mineral content (122), consistent with the idea that TGF-$\beta$ functions as a bone-forming agent. In transgenic mice where the endogenous TGF-$\beta$ signaling is inhibited in bone by overexpression of the dominant negative TGF-$\beta$ type II receptor, there is increased trabecular bone mass due to decreased bone resorption by osteoclasts (123). Depending on particular experimental parameters, TGF-$\beta$ modulates various bone cell activities in opposite ways (124). Both increases and decreases in osteoclast formation, bone resorption, osteoblast proliferation, and osteoblast differentiation have been reported (118, 120). In general, TGF-$\beta$ has been suggested to function as a putative coupling factor between bone resorption and bone formation (118, 119).

## B. TGF-$\beta$ Signaling

TGF-$\beta$ signals through sequential activation of two cell surface serine-threonine kinase receptors (type II and type I), which phosphorylate Smad2 and Smad3 (regulatory Smads; R-Smads) within their conserved C-terminal

TRANSCRIPTION IN THE OSTEOBLAST 305

SSXS motif (125, 126). These activated Smads, together with Smad4 (common Smad; Co-Smad), translocate to the nucleus. In the nucleus, the R-Smad–Co-Smad interacts with various transcription factors and transcriptional coactivators/corepressors, leading to transcriptional regulation of target genes. Smad2 and Smad3 serve as R-Smads transducing TGF-$\beta$/activin-like signals, whereas Smads 1, 5, and 8 act as R-Smads transducing BMP-like signals. Smad4 is an essential component in many of the Smad-dependent responses (127), serving both to stabilize the Smad-transcription factor complex (128) and to form functional interactions with critical transcriptional adapter proteins including CBP and p300 (129). In general, a wide range of biological activities of TGF-$\beta$ may result from the ability of Smads to interact with various transcription factors. More than 30 transcription factors, including FAST1, c-Jun, Spl, TFE3, Mixer, vitamin D receptor, and GATA-3, have been reported to interact with Smads (130). Even though Smads can bind directly to DNA, this binding is of low affinity and low specificity (131). Smad DNA-binding does not appear to be absolutely required on all promoters (132). Instead Smad-dependent regulation of gene expression requires interactions with specific DNA-binding partners. For example, Smad2 and Smad3 DNA-binding partners include, FAST (FoxH1), Fos/Jun, ATF-2, TFE3, VDR, Runx/Cbfa, Mixer, LEF1/TCF, NF-kB, HNF4, and Spl (133, 134). By selecting these transcription factors as their interaction partners, Smads may exhibit specific biological effects on target cells.

TGF-$\beta$ regulates the transcription of a wide array of target genes. Based on transcriptional profiling, between 300 and 500 TGF-$\beta$-regulated genes have been identified (135–138). The spectrum of TGF-$\beta$ regulated genes varies greatly among different cell types. Different subsets of genes can be defined based on the kinetics of TGF-$\beta$-induction or repression. These studies include induction of expression of cyclin-dependent kinase inhibitors p15${}^{\text{INK4B}}$ (139, 140), p21${}^{\text{CIP1}}$ (141), Smad7 (142), fibronectin (143), AP-1, plasminogen activator inhibitor (PAI-1), interstitial collagen, and other extracellular matrix proteins (144). TGF-$\beta$ represses the transcription of c-myc (48), CDK4 and CDC25A, and Bcl-X${}_{\text{L}}$ (144, 145). The extent of TGF-$\beta$ transcriptional activation is controlled by co-repressors such as, for example, Evi-1, c-ski, SnoN, TGIF, SNIP, SIP, and co-activators, which include c-Jun, ATF-2, CREB, bHLH, TFE3, vitamin-D-, glucocorticoid-, and androgen receptors (145).

## C. TGF-$\beta$1 Regulation of MMP-13 Gene in Osteoblastic Cells

As we described earlier, MMP-13 is expressed as a late-differentiation gene in osteoblasts, being primarily responsible for the degradation of extracellular bone matrix components (type I, II, and III fibrillar collagens).

MMP-13 gene expression is regulated by bone-resorbing agents, such as PTH, cytokines such as interleukins (IL-1, –6), and growth factors that promote bone turnover (37, 146–149). The regulation of this gene is likely to have important consequences for both normal and pathological remodeling of bone where the balance between bone resorption and bone formation is disrupted. Using mutant mice homozygous for a targeted mutation in *Collal* that are resistant to collagenase cleavage of type I collagen, Zhao *et al.* showed that bone resorption and calcemic responses were markedly diminished (52). This study suggests that collagenases are coupled with osteoclast-mediated bone resorption by (1) acting on a layer of hypomineralized collagen on bone surfaces to permit osteoclasts to attach and then resorb bone by mechanisms utilizing other proteinases, (2) releasing biologically active peptides on bone resorption that might prevent osteoclast apoptosis, (3) producing type I collagen fragments which might themselves be biologically active for generation and activity of osteoclasts, and (4) releasing other biologically active ligands previously trapped in the matrix that could stimulate the generation and activity of osteoclasts (52, 150).

Both TGF-$\beta$1 and MMP-13 could function as putative coupling factors between bone formation and bone resorption. We have shown that TGF-$\beta$1 stimulates MMP-13 expression in rat osteoblastic osteosarcoma UMR 106-01 cells (151) (Fig. 6A,B). TGF-$\beta$1 inhibited MMP-13 mRNA expression at earlier time periods and stimulated its expression at later time periods (Fig. 6B), suggesting that TGF-$\beta$1-altered expression of other factors may be responsible for this. Transcriptional activation of MMP-13 by PTH and TGF-$\beta$ in osteoblastic cells is quite different and both use different signaling and molecular mechanisms (61, 105, 151). TGF-$\beta$1 effect on MMP-13 induction is secondary and early gene activation or *de novo* protein synthesis is required for this purpose (Fig. 6C) and the effects are slower. The activator protein-1-binding sites (AP-1) are responsible for early gene activation, and many TGF-$\beta$1-inducible genes contain these sites, which have been functionally linked to transcriptional activation by TGF-$\beta$1 (61, 152). TGF-$\beta$1-induced differential expression of Fos and Jun family members may be a primary effect and these transcription factors would bind to the AP-1 site of the rat MMP-13 promoter. In addition to the AP-1 factors, other factors such as Ets, Nmp4/CIZ, C/EBP, and p53 that were induced or repressed or modified by TGF-$\beta$1 (64–67) would bind to their respective sites of the MMP-13 promoter and form a multiprotein assemblage (enhanceosome) for MMP-13 gene expression.

The TGF-$\beta$1 signaling pathway for MMP-13 stimulation in bone appears to be significantly different from fibroblast and chondrocyte MMP-13 expression (153, 154) and is regulated by several intracellular pathways and components. Smad proteins are the main cytoplasmic signaling pathways in

TRANSCRIPTION IN THE OSTEOBLAST 307

(A)

Control

TGF-β1 (ng/ml)

0.1 1 5 10 30

C'ase-3

18 S

(B)

Time (h)

1 2 4 8 24 48

TGF-β1

(10 ng/ml)

C'ase-3

18 S

(C)

24 h

C T C + CHX T + CHX

C'ase-3

18 S

FIG. 6. (A) Dose effect of the TGF-β1 stimulation of MMP-13 mRNA. UMR 106-01 cells were serum-starved for 24 h and treated with control medium or medium containing TGF-β1 at different concentrations for 24 h, as indicated. Total RNA was isolated and subjected to Northern blot analysis using labeled rat MMP-13 (*Case-3*) or 18 S ribosomal cDNA. (B) Time course of the TGF-β1 stimulation of MMP-13 mRNA. UMR 106-01 cells were serum-starved for 24 h and treated with control medium or medium containing 10 ng/ml TGF-β1 for different time periods, as indicated. Total RNA was isolated and subjected to Northern blot analysis using labeled rat MMP-13 or 18 S ribosomal cDNA. (C) TGF-β1-stimulated MMP-13 mRNA expression requires *de novo* protein synthesis. UMR 106-01 cells were serum-starved for 24 h and treated with control medium (C) or medium containing 10 ng/ml TGF-β1 (T) for 8 and 24 h in the presence or absence of 30 μg/ml cycloheximide (CHX) added 1 h before TGF-β1 treatment, and total RNA was subjected to Northern blot analysis using labeled rat MMP-13 or 18 S ribosomal cDNA ([151]).

TGF-β1-stimulated MMP-13 expression in osteoarthritic chondrocytes ([154]). A cross-talk between the Smad and MAPK pathways may be necessary to confer cell-specific expression of MMP-13 by TGF-β1. The enhancement of MMP-13 expression by TGF-β1 is dependent on p38 MAPK activity in human gingival and skin fibroblasts, transformed human epidermal keratinocytes, and the human cutaneous squamous cell carcinoma cell line ([153], [155]). We have reported that TGF-β1-induced activation of p38 MAPK and ERK1/2 is essential for Smad-mediated MMP-13 promoter activity and that the interaction between the Smad and MAPK pathways is necessary for maximal promoter activation in UMR 106-01 cells ([151]).

The explanation for cross-talk between the Smad and MAPK pathways for TGF-β1-stimulated MMP-13 expression in osteoblastic cells is that components of these pathways interact directly in the transcriptional complex. Since there are two SBE sites in the rat MMP-13 promoter ([Fig. 2]), Smad

proteins could mediate this TGF-$\beta$1-response directly by binding to one or two of these SBE sites. Runx2 could be a substrate for the MAPK pathway, and this pathway can be stimulated by a variety of signals (156). The PST and C-terminal regions of Runx2 are important for its activity (84, 98, 99, 157–159). It appears that Runx2 expression and activity are regulated by post-translational modifications and protein–protein interactions. In our laboratories, we have demonstrated that TGF-$\beta$1 stimulates Runx2 phosphorylation at threonine and tyrosine residues in rat osteoblastic cells. Because Runx2 phosphorylation also occurs at tyrosine residues, the epidermal growth factor receptor could also be responding to TGF-$\beta$1, as evidenced by epidermal growth factor receptor transactivation in TGF-$\beta$1-mediated fibronectin expression in mesangial cells (160). Runx2 is phosphorylated at threonine residues after TGF-$\beta$1 treatment. The ERK1/2 signaling pathway is required for TGF-$\beta$1 stimulation of MMP-13 promoter activity in rat osteoblastic cells. These two pieces of evidence strongly suggest that Runx2 is a substrate for ERK1/2. ERK1/2 is a common target of TGF-$\beta$1 and BMP-2 and activates Runx2-dependent transcription without affecting the expression of Runx2 (99). The activation of p38 MAPK is also involved in the induction of Runx2 by TGF-$\beta$1 and BMP-2 stimulation (161). Because both the AP-1 and Runx2 transcription factors are important targets of interleukin-induced p38 MAPK, leading to MMP-13 expression in a human chondrosarcoma cell line (162), it is possible that Runx2 could also be a substrate for the p38 MAPK pathway in osteoblastic cells.

To determine the requirement of Runx proteins for TGF-$\beta$1-stimulated collagenase-3 promoter activity in rat osteoblastic cells, we overexpressed Runx repressor protein AML/ETO in co-transfection experiments (151). The results demonstrated that both the basal and the TGF-$\beta$1-response were greatly reduced (Fig. 7). Since there is an overlapping Runx binding site with the AP-1 site in the rat MMP-13 promoter (Fig. 2), overexpression of AML/ETO could block the binding of preexisting Runx2, Fos, and Jun family members at this site, resulting in reduced basal MMP-13 promoter activity. The reduction of both the basal and TGF-$\beta$1-stimulated MMP-13 promoter activity in rat osteoblastic cells could also be due to interaction of AML/ETO protein with corepressors such as HDACs, N-CoR, mSin3A, and binding of AML/ETO protein with the distal Runx site and the proximal Runx/AP-1 site.

So far, the significance of the Smad and Runx2 proteins and the mechanisms of their interactions with other proteins for MMP-13 expression in either fibroblasts or osteoblasts have not been studied. In fibroblasts, Smad3 mediates the induction of MMP-13 expression by TGF-$\beta$, whereas Smad2 is not involved in this context (154). In osteoblasts, only Smad2 (not Smad3) mediates its effect for MMP-13 promoter activation. We have provided evidence of functional cross-talk between the Smad and MAPK pathways by

TRANSCRIPTION IN THE OSTEOBLAST 309

![Image Description](image.png)

FIG. 7. Runx2 is required for TGF-β1-stimulated MMP-13 promoter activity. The -500 MMP-13 promoter construct was transiently cotransfected with the pCMV-AML-1/ETO (a repressor protein for Runx) construct at increasing concentrations into UMR 106-01 cells, treated with control or TGF-β1 (5 ng/ml)-containing medium for 24 h, and assayed for CAT activity. The total amount of DNA used for transfection with or without the expression constructs was equalized with pCMV. Data represent means ± S.E. of three experiments. pSVO represents the promoterless vector.

co-transfection of their components (Smad2 and Runx2) along with the MMP-13 promoter construct in UMR 106-01 cells (151). Even though the interaction of Smad2 and Runx2 is ligand-independent, TGF-β1 treatment is required to confer maximal MMP-13 promoter activity in rat osteoblastic cells, indicating that TGF-β1-induced, TGF-β1-repressed, or TGF-β1-modified factors may be necessary for interaction between Smad2 and Runx2 proteins. This functional interaction may be stabilized or mediated by CBP and p300, which could act as transcriptional adapter proteins (163, 164).

In addition to the MAPK pathway, the Smad pathway also interacts with other signaling pathways, including the Wnt-β-catenin pathway (145). Cross-talk with cytokine signaling has also been reported; interferon-γ (IFN-γ) induces expression of Smad7 on some cells through the Jak-STAT signaling pathway, and Smad7 then inhibits TGF-β signaling pathways (165). Another important interaction between the Smad and JAK-STAT signaling pathways has been reported, where TGF-β superfamily signaling is indirectly regulated by STAT1 through the Runx proteins (166).

## V. Summary of TGF-β Actions on MMP-13

In this section, we have focused on TGF-β1 stimulation of MMP-13 expression in rat osteoblastic cells. It appears that TGF-β1-stimulated MMP-13 expression in these cells involves a signaling cascade and a complex

TGF-β
TGF-β receptor

MAPK
(ERK, p38)

Smad2
Smad4

Smad2
Smad4

Runx2
Smad2
Smad4

MMP-13 Promoter

FIG. 8. A schematic diagram of TGF-β-stimulation of MMP-13 expression. TGF-β-stimulated MMP-13 expression requires both the Smad and the MAPK pathways (ERK and p38) and their components Smad2 and Runx2, respectively. Both Runx2 and Smad2 interact with each other and activate MMP-13 gene expression in rat osteoblastic cells.

of proteins and their activities, i.e., induction of early gene products, modulation of Runx2 expression by phosphorylation, and interaction of Smad and other proteins (Fig. 8). Elucidation of the signaling and molecular mechanisms of TGF-β1 regulation of MMP-13 is critical to understanding the molecular events governing bone remodeling and bone-related diseases. Especially in view of the importance of Runx2 as a pivotal transcription factor of bone and bone-related genes, this chapter will further advance our understanding of how Runx2 physiologically functions in bone metabolism.

VI. Conclusions

Despite the many questions that remain unanswered about PTH and TGF-β regulation of osteoblastic genes, in this chapter, we have mostly addressed regulation of MMP-13 expression by these factors in rat osteoblastic cells. PTH is the principal modulator and can be considered a “bone remodeler,” working to stimulate both bone resorption and bone formation. Similarly, TGF-β has effects on osteoblastic cell proliferation and bone matrix formation. In the cascade of bone remodeling, MMP-13 plays a critical role in mediating the acute osteoclastic bone resorption induced by PTH, *in vivo*. The

regulation of this gene is likely to have important consequences for both normal and pathological remodeling of bone where the balance between bone resorption and bone formation is disrupted. Even though PTH and TGF-β are systemic and local factors, respectively, both stimulate MMP-13 expression in rat osteoblastic cells. Since bone resorption releases TGF-β from bone matrix, it appears that TGF-β could be a mediator for PTH action in bone. The regulation of the MMP-13 gene by PTH and TGF-β in rat osteoblastic cells is a complex process and involves several distinct intracellular pathways. Thus, the determination of molecular mechanisms of osteoblastic gene expression, especially MMP-13 by PTH and TGF-β in cells of the osteoblastic lineage, may lead in the future to the development of new therapeutic strategies aimed at improving bone and addressing bone-related diseases.

---

**REFERENCES**

1. Stein, G. S., Lian, J. B., and Owen, T. A. (1990). Relationship of cell growth to the regulation of tissue-specific gene expression during osteoblast differentiation. *FASEB J.* 4, 3111–3123.
2. Stein, G. S., Lian, J. B., Stein, J. L., Van Wijnen, A. J., and Montecino, M. (1996). Transcriptional control of osteoblast growth and differentiation. *Physiol. Rev.* 76, 593–629.
3. Shalhoub, V., Conlon, D., Tassinari, M., Quinn, C., Partridge, N., Stein, G. S., and Lian, J. B. (1992). Glucocorticoids promote development of the osteoblast phenotype by selectively modulating expression of cell growth and differentiation associated genes. *J. Cell Biochem.* 50, 425–440.
4. Owen, T. A., Aronow, M., Shalhoub, V., Barone, L. M., Wilming, L., Tassinari, M. S., Kennedy, M. B., Pockwinse, S., Lian, J. B., and Stein, G. S. (1990). Progressive development of the rat osteoblast phenotype *in vitro*: Reciprocal relationships in expression of genes associated with osteoblast proliferation and differentiation during formation of the bone extracellular matrix. *J. Cell Physiol.* 143, 420–430.
5. Birnbaum, R. S., Bowsher, R. R., and Wiren, K. M. (1995). Changes in IGF-I and -II expression and secretion during the proliferation and differentiation of normal rat osteoblasts. *J. Endocrinol.* 144, 251–259.
6. Weinreb, M., Shinar, D., and Rodan, G. A. (1990). Different pattern of alkaline phosphatase, osteopontin, and osteocalcin expression in developing rat bone visualized by *in situ* hybridization. *J. Bone Miner. Res.* 5, 831–842.
7. Boskey, A. L., Wright, T. M., and Blank, R. D. (1999). Collagen and bone strength. *J. Bone Miner. Res.* 14, 330–335.
8. Dempster, D. W., Cosman, F., Parisien, M., Shen, V., and Lindsay, R. (1993). Anabolic actions of parathyroid hormone on bone. *Endocr. Rev.* 14, 690–709.
9. Swarthout, J. T., D’Alonzo, R. C., Selvamurugan, N., and Partridge, N. C. (2002). Parathyroid hormone-dependent signaling pathways regulating genes in bone cells. *Gene* 282, 1–17.
10. Lian, J. B., Stein, G. S., Stein, J. L., and van Wijnen, A. J. (1999). Regulated expression of the bone-specific osteocalcin gene by vitamins and hormones. *Vitam. Horm.* 55, 443–509.
11. Turner, R. T., Riggs, B. L., and Spelsberg, T. C. (1994). Skeletal effects of estrogen. *Endocr. Rev.* 15, 275–300.
12. Boyce, B. F., Hughes, D. E., Wright, K. R., Xing, L., and Dai, A. (1999). Recent advances in bone biology provide insight into the pathogenesis of bone diseases. *Lab. Invest.* 79, 83–94.

13. Delany, A. M., Dong, Y., and Canalis, E. (1994). Mechanisms of glucocorticoid action in bone cells. *J. Cell Biochem.* **56**, 295–302.

14. Ishida, Y., and Heersche, J. N. (1998). Glucocorticoid-induced osteoporosis: Both *in vivo* and *in vitro* concentrations of glucocorticoids higher than physiological levels attenuate osteoblast differentiation. *J. Bone Miner. Res.* **13**, 1822–1826.

15. Bonewald, L. F., and Mundy, G. R. (1990). Role of transforming growth factor-beta in bone remodeling. *Clin. Orthop.* **26**, 1–276.

16. Breen, E. C., Ignatz, R. A., McCabe, L., Stein, J. L., Stein, G. S., and Lian, J. B. (1994). TGF beta alters growth and differentiation related gene expression in proliferating osteoblasts *in vitro*, preventing development of the mature bone phenotype. *J. Cell Physiol.* **160**, 323–335.

17. Tang, K. T., Capparelli, C., Stein, J. L., Stein, G. S., Lian, J. B., Huber, A. C., Braverman, L. E., and DeVito, W. J. (1996). Acidic fibroblast growth factor inhibits osteoblast differentiation *in vitro*: Altered expression of collagenase, cell growth-related, and mineralization-associated genes. *J. Cell Biochem.* **61**, 152–166.

18. Silverberg, S. J., Shane, E., Jacobs, T. P., Siris, E., and Bilezikian, J. P. (1999). A 10-year prospective study of primary hyperparathyroidism with or without parathyroid surgery. *N. Engl. J. Med.* **341**, 1249–1255.

19. Potts, J. T., Jr., Gardella, T. J., Juppner, H., and Kronenberg, H. M. (1997). Structure-based design of parathyroid hormone analogs. *J. Endocrinol.* **154**, S15–S21.

20. Rantakokko, J., Aro, H. T., Savontaus, M., and Vuorio, E. (1996). Mouse cathepsin K: cDNA cloning and predominant expression of the gene in osteoclasts, and in some hypertrophying chondrocytes during mouse development. *FEBS Lett.* **393**, 307–313.

21. Sato, T., del Carmen Ovejero, M., Hou, P., Heegaard, A. M., Kumegawa, M., Foged, N. T., and Delaisse, J. M. (1997). Identification of the membrane-type matrix metalloproteinase MT1-MMP in osteoclasts. *J. Cell Sci.* **110**, 589–596.

22. Delaisse, J. M., Eeckhout, Y., Neff, L., Francois-Gillet, C., Henriet, P., Su, Y., Vaes, G., and Baron, R. (1993). (Pro)collagenase (matrix metalloproteinase-1) is present in rodent osteoclasts and in the underlying bone-resorbing compartment. *J. Cell Sci.* **106**, 1071–1082.

23. Chambers, T. J., Darby, J. A., and Fuller, K. (1985). Mammalian collagenase predisposes bone surfaces to osteoclastic resorption. *Cell Tissue Res.* **421**, 671–675.

24. Yasuda, H., Shima, N., Nakagawa, N., Yamaguchi, K., Kinosaki, M., Mochizuki, S., Tomoyasu, A., Yano, K., Goto, M., Murakami, A., Tsuda, E., Morinaga, T., Higashio, K., Udagawa, N., Takahashi, N., and Suda, T. (1998). Osteoclast differentiation factor is a ligand for osteoprotegerin/osteoclastogenesis-inhibitory factor and is identical to TRANCE/RANKL. *Proc. Natl. Acad. Sci. USA* **95**, 3597–3602.

25. Lacey, D. L., Timms, E., Tan, H. L., Kelley, M. J., Dunstan, C. R., Burgess, T., Elliott, R., Colombero, A., Elliott, G., Scully, S., Hsu, H., Sullivan, J., Hawkins, N., Davy, E., Capparelli, C., Eli, A., Qian, Y. X., Kaufman, S., Sarosi, I., Shalhoub, V., Senaldi, G., Guo, J., Delaney, J., and Boyle, W. J. (1998). Osteoprotegerin ligand is a cytokine that regulates osteoclast differentiation and activation. *Cell* **93**, 165–176.

26. Kong, Y. Y., Yoshida, H., Sarosi, I., Tan, H. L., Timms, E., Capparelli, C., Morony, S., Oliveira-dos-Santos, A. J., Van, G., Itie, A., Khoo, W., Wakeham, A., Dunstan, C. R., Lacey, D. L., Mak, T. W., Boyle, W. J., and Penninger, J. M. (1999). OPGL is a key regulator of osteoclastogenesis, lymphocyte development, and lymph-node organogenesis. *Nature* **397**, 315–323.

27. Dietrich, J. W., Canalis, E. M., Maina, D. M., and Raisz, L. G. (1976). Hormonal control of bone collagen synthesis *in vitro*: Effects of parathyroid hormone and calcitonin. *Endocrinology* **98**, 943–949.

28. Partridge, N. C., Dickson, C. A., Kopp, K., Teitelbaum, S. L., Crouch, E. C., and Kahn, A. J. (1989). Parathyroid hormone inhibits collagen synthesis at both ribonucleic acid and protein levels in rat osteogenic sarcoma cells. *Mol. Endocrinol.* **3**, 232–239.

29. Delaisse, J. M., Eeckhout, Y., and Vaes, G. (1988). Bone-resorbing agents affect the production and distribution of procollagenase as well as the activity of collagenase in bone tissue. *Endocrinology* **123**, 264–276.

30. Hamilton, J. A., Lingelbach, S., Partridge, N. C., and Martin, T. J. (1985). Regulation of plasminogen activator production by bone-resorbing hormones in normal and malignant osteoblasts. *Endocrinology* **116**, 2186–2191.

31. Wucherpfennig, A. L., Li, Y. P., Stetler-Stevenson, W. G., Rosenberg, A. E., and Stashenko, P. (1994). Expression of 92 kD type IV collagenase/gelatinase B in human osteoclasts. *J. Bone Miner. Res.* **9**, 549–556.

32. Walker, D. G., Lapierre, C. M., and Gross, J. (1964). A collagenolytic factor in rat bone promoted by parathyroid extract. *Biochem. Biophys. Res. Commun.* **15**, 397–402.

33. Rasmussen, H., Arnaud, C., and Hawker, C. (1964). Actinomycin D and the response to parathyroid hormone. *Science* **144**, 1019–1021.

34. Tashjian, A. H., Jr., Ontjes, D. A., and Goodfriend, T. L. (1964). Mechanism of parathyroid hormone action. Effects of actinomycin D on hormone-stimulated ion movement *in vivo* and *in vitro*. *Biochem. Biophys. Res. Commun.* **16**, 209–215.

35. Quinn, C. O., Scott, D. K., Brinckerhoff, C. E., Matrisian, L. M., Jeffrey, J. J., and Partridge, N. C. (1990). Rat collagenase. Cloning, amino acid sequence comparison, and parathyroid hormone regulation in osteoblastic cells. *J. Biol. Chem.* **265**, 22342–22347.

36. Partridge, N. C., Jeffrey, J. J., Ehlich, L. S., Teitelbaum, S. L., Fliszar, C., Welgus, H. G., and Kahn, A. J. (1987). Hormonal regulation of the production of collagenase and a collagenase inhibitor activity by rat osteogenic sarcoma cells. *Endocrinology* **120**, 1956–1962.

37. Scott, D. K., Brakenhoff, K. D., Clohisy, J. C., Quinn, C. O., and Partridge, N. C. (1992). Parathyroid hormone induces transcription of collagenase in rat osteoblastic cells by a mechanism using cyclic adenosine 3′,5′-monophosphate and requiring protein synthesis. *Mol. Endocrinol.* **6**, 2153–2159.

38. Qin, L., Qiu, P., Wang, L., Li, X., Swarthout, J. T., Soteropoulos, P., Tolias, P., and Partridge, N. C. (2003). Gene expression profiles and transcription factors involved in parathyroid hormone signaling in osteoblasts revealed by microarray and bioinformatics. *J. Biol. Chem.* **278**, 19723–19731.

39. Juppner, H. (1999). Receptors for parathyroid hormone and parathyroid hormone-related peptide: Exploration of their biological importance. *Bone* **25**, 87–90.

40. Civitelli, R., Reid, I. R., Westbrook, S., Avioli, L. V., and Hruska, K. A. (1988). PTH elevates inositol polyphosphates and diacylglycerol in a rat osteoblast-like cell line. *Am. J. Physiol.* **255**, E660–E667.

41. Babich, M., Choi, H., Johnson, R. M., King, K. L., Alford, G. E., and Nissenson, R. A. (1991). Thrombin and parathyroid hormone mobilize intracellular calcium in rat osteosarcoma cells by distinct pathways. *Endocrinology* **129**, 1463–1470.

42. Takasu, H., Guo, J., and Bringhurst, F. R. (1999). Dual signaling and ligand selectivity of the human PTH/PTHrP receptor. *J. Bone Miner. Res.* **14**, 11–20.

43. Chase, L. R., and Aurbach, G. D. (1970). The effect of parathyroid hormone on the concentration of adenosine 3′-5′-monophosphate in skeletal tissue *in vitro*. *J. Biol. Chem.* **245**, 1520–1526.

44. Yang, R., Gotoh, Y., Moore, M. A., Rafidi, K., and Gerstenfeld, L. C. (1995). Characterization of an avian bone sialoprotein (BSP) cDNA: Comparisons to mammalian BSP and identification of conserved structural domains. *J. Bone Miner. Res.* **10**, 632–640.

45. Yang, R., and Gerstenfeld, L. C. (1996). Signal transduction pathways mediating parathyroid hormone stimulation of bone sialoprotein gene expression in osteoblasts. *J. Biol. Chem.* **271**, 29839–29846.

46. Woessner, J. F., Jr. (1991). Matrix metalloproteinases and their inhibitors in connective tissue remodeling. *FASEB J.* **5**, 2145–2154.

47. Matrisian, L. M. (1992). The matrix-degrading metalloproteinases. *Bioessays* **14**, 455–463.

48. Roswit, W. T., Halme, J., and Jeffrey, J. J. (1983). Purification and properties of rat uterine procollagenase. *Arch. Biochem. Biophys.* **225**, 285–295.

49. Mattot, V., Raes, M. B., Henriet, P., Eeckhout, Y., Stehelin, D., Vandenbunder, B., and Desbiens, X. (1995). Expression of interstitial collagenase is restricted to skeletal tissue during mouse embryogenesis. *J. Cell Sci.* **108**, 529–535.

50. Meikle, M. C., Bord, S., Hembry, R. M., Compston, J., Croucher, P. I., and Reynolds, J. J. (1992). Human osteoblasts in culture synthesize collagenase and other matrix metalloproteinases in response to osteotropic hormones and cytokines. *J. Cell Sci.* **103**, 1093–1099.

51. Gack, S., Vallon, R., Schmidt, J., Grigoriadis, A., Tuckermann, J., Schenkel, J., Weiher, H., Wagner, E. F., and Angel, P. (1995). Expression of interstitial collagenase during skeletal development of the mouse is restricted to osteoblast-like cells and hypertrophic chondrocytes. *Cell Growth Differ.* **6**, 759–767.

52. Zhao, W., Byrne, M. H., Boyce, B. F., and Krane, S. M. (1999). Bone resorption induced by parathyroid hormone is strikingly diminished in collagenase-resistant mutant mice. *J. Clin. Invest.* **103**, 517–524.

53. Delany, A. M., Rydziel, S., and Canalis, E. (1996). Autocrine down-regulation of collagenase-3 in rat bone cell cultures by insulin-like growth factors. *Endocrinology* **137**, 4665–4670.

54. Varghese, S., and Canalis, E. (1997). Regulation of collagenase-3 by bone morphogenetic protein-2 in bone cell cultures. *Endocrinology* **138**, 1035–1040.

55. Gazzero, E., Rydziel, S., and Canalis, E. (1999). Skeletal bone morphogenetic proteins suppress the expression of collagenase-3 by rat osteoblasts. *Endocrinology* **140**, 562–567.

56. Aznavoorian, S., Murphy, A. N., Stetler-Stevenson, W. G., and Liotta, L. A. (1993). Molecular aspects of tumor cell invasion and metastasis. *Cancer* **71**, 1368–1383.

57. Billinghurst, R. C., Dahlberg, L., Ionescu, M., Reiner, A., Bourne, R., Rorabeck, C., Mitchell, P., Hambor, J., Diekmann, O., Tschesche, H., Chen, J., Van Wart, H., and Poole, A. R. (1997). Enhanced cleavage of type II collagen by collagenases in osteoarthritic articular cartilage. *J. Clin. Invest.* **99**, 1534–1545.

58. Mitchell, P. G., Magna, H. A., Reeves, L. M., Lopresti-Morrow, L. L., Yocum, S. A., Rosner, P. J., Geoghegan, K. F., and Hambor, J. E. (1996). Cloning, expression, and type II collagenolytic activity of matrix metalloproteinase-13 from human osteoarthritic cartilage. *J. Clin. Invest.* **97**, 761–768.

59. Wernicke, D., Seyfert, C., Hinzmann, B., and Gromnica-Ihle, E. (1996). Cloning of collagenase 3 from the synovial membrane and its expression in rheumatoid arthritis and osteoarthritis. *J. Rheumatol.* **23**, 590–595.

60. Rajakumar, R. A., and Quinn, C. O. (1996). Parathyroid hormone induction of rat interstitial collagenase mRNA in osteosarcoma cells is mediated through an AP-1-binding site. *Mol. Endocrinol.* **10**, 867–878.

61. Selvamurugan, N., Chou, W. Y., Pearman, A. T., Pulumati, M. R., and Partridge, N. C. (1998). Parathyroid hormone regulates the rat collagenase-3 promoter in osteoblastic cells through the cooperative interaction of the activator protein-1 site and the runt domain binding sequence. *J. Biol. Chem.* **273**, 10647–10657.

62. Porte, D., Tuckermann, J., Becker, M., Baumann, B., Teurich, S., Higgins, T., Owen, M. J., Schorpp-Kistner, M., and Angel, P. (1999). Both AP-1 and Cbfal-like factors are required for the induction of interstitial collagenase by parathyroid hormone. *Oncogene* **18**, 667–678.

63. Pendas, A. M., Balbin, M., Llano, E., Jimenez, M. G., and Lopez-Otin, C. (1997). Structural analysis and promoter characterization of the human collagenase-3 gene (MMP13). *Genomics* **40**, 222–233.

64. Karsenty, G. (2000). Role of Cbfal in osteoblast differentiation and function. *Semin. Cell. Dev. Biol.* **11**, 343–346.

65. Thunyakitpisal, P., Alvarez, M., Tokumaga, K., Onyia, J. E., Hock, J., Ohashi, N., Feister, H., Rhodes, S. J., and Bidwell, J. P. (2001). Cloning and functional analysis of a family of nuclear matrix transcription factors (NP/NMP4) that regulate type I collagen expression in osteoblasts. *J. Bone Miner. Res.* **16**, 10–23.

66. Shah, R., Alvarez, M., Jones, D. R., Torrungruang, K., Watt, A. J., Selvamurugan, N., Partridge, N. C., Quinn, C. O., Pavalko, F. M., Rhodes, S. J., and Bidwell, J. P. (2004). Nmp4/CIZ regulation of matrix metalloproteinase 13 (MMP-13) response to parathyroid hormone in osteoblasts. *Am. J. Physiol. Endocrinol. Metab.* **287**, E289–E296.

67. Janknecht, R., Cahill, M. A., and Nordheim, A. (1995). Signal integration at the c-fos promoter. *Carcinogenesis* **16**, 443–450.

68. Pearman, A. T., Chou, W. Y., Bergman, K. D., Pulumati, M. R., and Partridge, N. C. (1996). Parathyroid hormone induces c-fos promoter activity in osteoblastic cells through phosphorylated cAMP response element (CRE)-binding protein binding to the major CRE. *J. Biol. Chem.* **271**, 25715–25721.

69. Angel, P., and Karin, M. (1991). The role of Jun, Fos and the AP-1 complex in cell-proliferation and transformation. *Biochim. Biophys. Acta* **1072**, 129–157.

70. Zhou, X. F., Shen, X. Q., and Shemshedini, L. (1999). Ligand-activated retinoic acid receptor inhibits AP-1 transactivation by disrupting c-Jun/c-Fos dimerization. *Mol. Endocrinol.* **13**, 276–285.

71. Franceschi, R. T. (1999). The developmental control of osteoblast-specific gene expression: Role of specific transcription factors and the extracellular matrix environment. *Crit. Rev. Oral Bio. Med.* **10**, 40–57.

72. Rajaram, N., and Kerppola, T. K. (1997). DNA bending by Fos-Jun and the orientation of heterodimer binding depend on the sequence of the AP-1 site. *EMBO J.* **16**, 2917–2925.

73. Ruther, U., Komitowski, D., Schubert, F. R., and Wagner, E. F. (1989). c-fos expression induces bone tumors in transgenic mice. *Oncogene* **4**, 861–865.

74. Wang, Z. Q., Ovitt, C., Grigoriadis, A. E., Mohle-Steinlein, U., Ruther, U., and Wagner, E. F. (1992). Bone and hematopoietic defects in mice lacking c-fos. *Nature* **360**, 741–745.

75. Clohisy, J. C., Scott, D. K., Brakenhoff, K. D., Quinn, C. O., and Partridge, N. C. (1992). Parathyroid hormone induces c-fos and c-jun messenger RNA in rat osteoblastic cells. *Mol. Endocrinol.* **6**, 1834–1842.

76. Tyson, D. R., Swarthout, J. T., and Partridge, N. C. (1999). Increased osteoblastic c-fos expression by parathyroid hormone requires protein kinase A phosphorylation of the cyclic adenosine 3′,5′-monophosphate response element-binding protein at serine 133. *Endocrinology* **140**, 1255–1261.

77. Tyson, D. R., Swarthout, J. T., Jefcoat, S. C., and Partridge, N. C. (2002). PTH induction of transcriptional activity of the cAMP response element-binding protein requires the serine 129 site and glycogen synthase kinase-3 activity, but not casein kinase II sites. *Endocrinology* **143**, 674–682.

78. Daga, A., Karlovich, C. A., Dumstrei, K., and Banerjee, U. (1996). Patterning of cells in the *Drosophila* eye by Lozenge, which shares homologous domains with AML1. *Genes Dev.* **10**, 1194–1205.

79. Bae, S. C., Takahashi, E., Zhang, Y. W., Ogawa, E., Shigesada, K., Namba, Y., Satake, M., and Ito, Y. (1995). Cloning, mapping and expression of PEBP2 alpha C, a third gene encoding the mammalian Runt domain. *Gene* **159**, 245–248.

80. Kanno, T., Kanno, Y., Chen, L. F., Ogawa, E., Kim, W. Y., and Ito, Y. (1998). Intrinsic transcriptional activation-inhibition domains of the polyomavirus enhancer binding protein

2/core binding factor alpha subunit revealed in the presence of the beta subunit. *Mol. Cell. Biol.* **18**, 2444–2454.

81. Wang, S., Wang, Q., Crute, B. E., Melnikova, I. N., Keller, S. R., and Speck, N. A. (1993). Cloning and characterization of subunits of the T-cell receptor and murine leukemia virus enhancer core-binding factor. *Mol. Cell. Biol.* **13**, 3324–3339.

82. Golling, G., Li, L., Pepling, M., Stebbins, M., and Gergen, J. P. (1996). *Drosophila* homologs of the proto-oncogene product PEBP2/CBF beta regulate the DNA-binding properties of Runt. *Mol. Cell. Biol.* **16**, 932–942.

83. Ducy, P., Zhang, R., Geoffroy, V., Ridall, A. L., and Karsenty, G. (1997). Osf2/Cbfal: A transcriptional activator of osteoblast differentiation. *Cell* **89**, 747–754.

84. Thirunavukkarasu, K., Mahajan, M., McLaren, K. W., Stifani, S., and Karsenty, G. (1998). Two domains unique to osteoblast-specific transcription factor Osf2/Cbfal contribute to its transactivation function and its inability to heterodimerize with Cbfbeta. *Mol. Cell. Biol.* **18**, 4197–4208.

85. Tintut, Y., Parhami, F., Le, V., Karsenty, G., and Demer, L. L. (1999). Inhibition of osteoblast-specific transcription factor Cbfal by the cAMP pathway in osteoblastic cells. Ubiquitin/proteasome-dependent regulation. *J. Biol. Chem.* **274**, 28875–28879.

86. Selvamurugan, N., Pulumati, M. R., Tyson, D. R., and Partridge, N. C. (2000). Parathyroid hormone regulation of the rat collagenase-3 promoter by protein kinase A-dependent transactivation of core binding factor alphal. *J. Biol. Chem.* **275**, 5037–5042.

87. Xiao, G., Wang, D., Benson, M. D., Karsenty, G., and Franceschi, R. T. (1998). Role of the alpha2-integrin in osteoblast-specific gene expression and activation of the Osf2 transcription factor. *J. Biol. Chem.* **273**, 32988–32994.

88. Fujiwara, M., Tagashira, S., Harada, H., Ogawa, S., Katsumata, T., Nakatsuka, M., Komori, T., and Takada, H. (1999). Isolation and characterization of the distal promoter region of mouse Cbfal. *Biochim. Biophys. Acta* **1446**, 265–272.

89. Tsuji, K., Ito, Y., and Noda, M. (1998). Expression of the PEBP2alphaA/AML3/CBFA1 gene is regulated by BMP4/7 heterodimer and its overexpression suppresses type I collagen and osteocalcin gene expression in osteoblastic and nonosteoblastic mesenchymal cells. *Bone* **22**, 87–92.

90. Li, J., Tsuji, K., Komori, T., Miyazono, K., Wrana, J. L., Ito, Y., Nifuji, A., and Noda, M. (1998). Smad2 overexpression enhances Smad4 gene expression and suppresses CBFA1 gene expression in osteoblastic osteosarcoma ROS17/2.8 cells and primary rat calvaria cells. *J. Biol. Chem.* **273**, 31009–31015.

91. Ducy, P., Starbuck, M., Priemel, M., Shen, J., Pinero, G., Geoffroy, V., Amling, M., and Karsenty, G. (1999). A Cbfal-dependent genetic pathway controls bone formation beyond embryonic development. *Genes Dev.* **13**, 1025–1036.

92. Komori, T., Yagi, H., Nomura, S., Yamaguchi, A., Sasaki, K., Deguchi, K., Shimizu, Y., Bronson, R. T., Gao, Y. H., Inada, M., Sato, M., Okamoto, R., Kitamura, Y., Yoshiki, S., and Kishimoto, T. (1997). Targeted disruption of Cbfal results in a complete lack of bone formation owing to maturation arrest of osteoblasts. *Cell* **89**, 755–764.

93. Lee, B., Thirunavukkarasu, K., Zhou, L., Pastore, L., Baldini, A., Hecht, J., Geoffroy, V., Ducy, P., and Karsenty, G. (1997). Missense mutations abolishing DNA binding of the osteoblast-specific transcription factor OSF2/CBFA1 in cleidocranial dysplasia. *Nat. Genet.* **16**, 307–310.

94. Gutman, A., and Wasylyk, B. (1991). Nuclear targets for transcription regulation by oncogenes. *Trends Genet.* **7**, 49–54.

95. Fry, C. J., and Farnham, P. J. (1999). Context-dependent transcriptional regulation. *J. Biol. Chem.* **274**, 29583–29586.

TRANSCRIPTION IN THE OSTEOBLAST 317

96. Ning, Y. M., and Robins, D. M. (1999). AML3/CBFalphal is required for androgen-specific activation of the enhancer of the mouse sex-limited protein (Slp) gene. *J. Biol. Chem.* **274**, 30624–30630.

97. McLaren, K. W., Lo, R., Grbavec, D., Thirunavukkarasu, K., Karsenty, G., and Stifani, S. (2000). The mammalian basic helix loop helix protein HES-1 binds to and modulates the transactivating function of the runt-related factor Cbfal. *J. Biol. Chem.* **275**, 530–538.

98. Westendorf, J. J., Zaidi, S. K., Cascino, J. E., Kahler, R., van Wijnen, A. J., Lian, J. B., Yoshida, M., Stein, G. S., and Li, X. (2002). Runx2 (Cbfal, AML-3) interacts with histone deacetylase 6 and represses the p21(CIP1/WAF1) promoter. *Mol. Cell. Biol.* **22**, 7982–7992.

99. Cui, C. B., Cooper, L. F., Yang, X., Karsenty, G., and Aukhil, I. (2003). Transcriptional coactivation of bone-specific transcription factor Cbfal by TAZ. *Mol. Cell. Biol.* **23**, 1004–1013.

100. Chen, L., Glover, J. N., Hogan, P. G., Rao, A., and Harrison, S. C. (1998). Structure of the DNA-binding domains from NFAT, Fos and Jun bound specifically to DNA. *Nature* **392**, 42–48.

101. Bannister, A. J., and Kouzarides, T. (1995). CBP-induced stimulation of c-Fos activity is abrogated by E1A. *EMBO J.* **14**, 4758–4762.

102. Kamei, Y., Xu, L., Heintz, T., Torchia, J., Kurokawa, R., Gloss, B., Lin, S. C., Heyman, R. A., Rose, D. W., Glass, C. K., and Rosenfeld, M. G. (1996). A CBP integrator complex mediates transcriptional activation and AP-1 inhibition by nuclear receptors. *Cell* **85**, 403–414.

103. Metz, R., Bannister, A. J., Sutherland, J. A., Hagemeier, C., O'Rourke, E. C., Cook, A., Bravo, R., and Kouzarides, T. (1994). c-Fos-induced activation of a TATA-box-containing promoter involves direct contact with TATA-box-binding protein. *Mol. Cell. Biol.* **14**, 6021–6029.

104. Martin, M. L., Lieberman, P. M., and Curran, T. (1996). Fos-Jun dimerization promotes interaction of the basic region with TFIIE-34 and TFIIIF. *Mol. Cell. Biol.* **16**, 2110–2118.

105. D'Alonzo, R. C., Selvamurugan, N., Karsenty, G., and Partridge, N. C. (2002). Physical interaction of the activator protein-1 factors c-Fos and c-Jun with Cbfal for collagenase-3 promoter activation. *J. Biol. Chem.* **277**, 816–822.

106. John, S., and Workman, J. L. (1998). Just the facts of chromatin transcription. *Science* **282**, 1836–1837.

107. Boumah, C. E., Selvamurugan, N., and Partridge, N. C. (2004). Transcriptional activation of collagenase-3 by PTH: Changes in chromatin structure in the proximal promoter region. *J. Bone Min. Res.* **19**, S218.

108. Kurdistani, S. K., Tavazoie, S., and Grunstein, M. (2004). Mapping global histone acetylation patterns to gene expression. *Cell* **117**, 721–733.

109. Lian, J. B., Javed, A., Zaidi, S. K., Lengner, C., Montecino, M., van Wijnen, A. J., Stein, J. L., and Stein, G. S. (2004). Regulatory controls for osteoblast growth and differentiation: Role of Runx/Cbfa/AML factors. *Crit. Rev. Eukaryotic Gene Exp.* **14**, 1–41.

110. Horner, A., Kemp, P., Summers, C., Bord, S., Bishop, N. J., Kelsall, A. W., Coleman, N., and Compston, J. E. (1998). Expression and distribution of transforming growth factor-beta isoforms and their signaling receptors in growing human bone. *Bone* **23**, 95–102.

111. Hartsough, M. T., and Mulder, K. M. (1997). Transforming growth factor-beta signaling in epithelial cells. *Pharmacol. Therapeutics* **75**, 21–41.

112. Centrella, M., and Canalis, E. (1985). Transforming and nontransforming growth factors are present in medium conditioned by fetal rat calvariae. *Proc. Natl. Acad. Sci. USA* **82**, 7335–7339.

113. Oursler, M. J. (1994). Osteoclast synthesis and secretion and activation of latent transforming growth factor beta. *J. Bone Min. Res.* **9**, 443–452.

114. Oreffo, R. O., Mundy, G. R., Seyedin, S. M., and Bonewald, L. F. (1989). Activation of the bone-derived latent TGF beta complex by isolated osteoclasts. *Biochem. Biophys. Res. Comm.* **158**, 817–823.

115. Bonewald, L. F., Wakefield, L., Oreffo, R. O., Escobedo, A., Twardzik, D. R., and Mundy, G. R. (1991). Latent forms of transforming growth factor-beta (TGF beta) derived from bone cultures: Identification of a naturally occurring 100-kDa complex with similarity to recombinant latent TGF beta. *Mol. Endocrin.* **5**, 741–751.

116. Bonewald, L. F., and Dallas, S. L. (1994). Role of active and latent transforming growth factor beta in bone formation. *J. Cell. Biochem.* **55**, 350–357.

117. Pfeilschifter, J., and Mundy, G. R. (1987). Modulation of type beta transforming growth factor activity in bone cultures by osteotropic hormones. *Proc. Natl. Acad. Sci. USA* **84**, 2024–2028.

118. Centrella, M., Horowitz, M. C., Wozney, J. M., and McCarthy, T. L. (1994). Transforming growth factor-beta gene family members and bone. *Endocrine Reviews* **15**, 27–39.

119. Noda, M., and Camilliere, J. J. (1989). *In vivo* stimulation of bone formation by transforming growth factor-beta. *Endocrinology* **124**, 2991–2994.

120. Garcia, C., Boyce, B. F., Gilles, J., Dallas, M., Qiao, M., Mundy, G. R., and Bonewald, L. F. (1996). Leukotriene B4 stimulates osteoclastic bone resorption both *in vitro* and *in vivo*. *J. Bone Min. Res.* **11**, 1619–1627.

121. Rosier, R. N., O'Keefe, R. J., and Hicks, D. G. (1998). The potential role of transforming growth factor beta in fracture healing. *Clin. Orthopaedics Related Res.* S294–S300.

122. Geiser, A. G., Zeng, Q. Q., Sato, M., Helvering, L. M., Hirano, T., and Turner, C. H. (1998). Decreased bone mass and bone elasticity in mice lacking the transforming growth factor-beta1 gene. *Bone* **23**, 87–93.

123. Filvaroff, E., Erlebacher, A., Ye, J., Gitelman, S. E., Lotz, J., Heilman, M., and Derynck, R. (1999). Inhibition of TGF-beta receptor signaling in osteoblasts leads to decreased bone remodeling and increased trabecular bone mass. *Development* **126**, 4267–4279.

124. Rydziel, S., Varghese, S., and Canalis, E. (1997). Transforming growth factor beta1 inhibits collagenase 3 expression by transcriptional and post-transcriptional mechanisms in osteoblast cultures. *J. Cell. Physiol.* **170**, 145–152, [erratum appears in *J. Cell. Physiol.* 1997, May; **171(2):234**].

125. Zhang, Y., Feng, X., We, R., and Derynck, R. (1996). Receptor-associated Mad homologues synergize as effectors of the TGF-beta response. *Nature* **383**, 168–172.

126. Engel, M. E., McDonnell, M. A., Law, B. K., and Moses, H. L. (1999). Interdependent SMAD and JNK signaling in transforming growth factor-beta-mediated transcription. *J. Biol. Chem.* **274**, 37413–37420.

127. de Caestecker, M. P., Hemmati, P., Larisch-Bloch, S., Ajmera, R., Roberts, A. B., and Lechleider, R. J. (1997). Characterization of functional domains within Smad4/DPC4. *J. Biol. Chem.* **272**, 13690–13696.

128. Liu, F., Pouponnot, C., and Massague, J. (1997). Dual role of the Smad4/DPC4 tumor suppressor in TGFbeta-inducible transcriptional complexes. *Genes Devel.* **11**, 3157–3167.

129. de Caestecker, M. P., Yahata, T., Wang, D., Parks, W. T., Huang, S., Hill, C. S., Shioda, T., Roberts, A. B., and Lechleider, R. J. (2000). The Smad4 activation domain (SAD) is a proline-rich, p300-dependent transcriptional activation domain. *J. Biol. Chem.* **275**, 2115–2122.

130. Miyazawa, K., Shinozaki, M., Hara, T., Furuya, T., and Miyazono, K. (2002). Two major Smad pathways in TGF-beta superfamily signalling. *Genes Cells* **7**, 1191–1204.

TRANSCRIPTION IN THE OSTEOBLAST 319

131. Shi, Y., Wang, Y. F., Jayaraman, L., Yang, H., Massague, J., and Pavletich, N. P. (1998). Crystal structure of a Smad MH1 domain bound to DNA: Insights on DNA binding in TGF-beta signaling. *Cell* **94**, 585–594.

132. Pardali, E., Xie, X. Q., Tsapogas, P., Itoh, S., Arvanitis, K., Heldin, C. H., ten Dijke, P., Grundstrom, T., and Sideras, P. (2000). Smad and AML proteins synergistically confer transforming growth factor beta responsiveness to human germ-line IgA genes. *J. Biol. Chem.* **275**, 3552–3560.

133. Moustakas, A., and Kardassis, D. (1998). Regulation of the human p21/WAF1/Cip1 promoter in hepatic cells by functional interactions between Sp1 and Smad family members. *Proc. Natl. Acad. Sci. USA* **95**, 6733–6738.

134. Attisano, L., Silvestri, C., Izzi, L., and Labbe, E. (2001). The transcriptional role of Smads and FAST (FoxH1) in TGFbeta and activin signalling. *Mol. Cell Endocrin.* **180**, 3–11.

135. Chen, C. R., Kang, Y., and Massague, J. (2001). Defective repression of c-myc in breast cancer cells: A loss at the core of the transforming growth factor beta growth arrest program. *Proc. Natl. Acad. Sci. USA* **98**, 992–999.

136. Verrecchia, F., Chu, M. L., and Mauviel, A. (2001). Identification of novel TGF-beta/Smad gene targets in dermal fibroblasts using a combined cDNA microarray/promoter transactivation approach. *J. Biol. Chem.* **276**, 17058–17062.

137. Zavadil, J., Bitzer, M., Liang, D., Yang, Y. C., Massimi, A., Kneitz, S., Piek, E., and Bottinger, E. P. (2001). Genetic programs of epithelial cell plasticity directed by transforming growth factor-beta. *Proc. Natl. Acad. Sci. USA* **98**, 6686–6691.

138. Kang, Y., Chen, C. R., and Massague, J. (2003). A self-enabling TGFbeta response coupled to stress signaling: Smad engages stress response factor ATF3 for Id1 repression in epithelial cells. *Mol. Cell* **11**, 915–926.

139. Hannon, G. J., and Beach, D. (1994). p15INK4B is a potential effector of TGF-beta-induced cell cycle arrest. [see comment]. *Nature* **371**, 257–261.

140. Reynisdottir, I., Polyak, K., Iavarone, A., and Massague, J. (1995). Kip/Cip and Ink4 Cdk inhibitors cooperate to induce cell cycle arrest in response to TGF-beta. *Genes Devel.* **9**, 1831–1845.

141. Datto, M. B., Yu, Y., and Wang, X. F. (1995). Functional analysis of the transforming growth factor beta responsive elements in the WAF1/Cip1/p21 promoter. *J. Biol. Chem.* **270**, 28623–28628.

142. Hill, C. S. (1999). The Smads. *Int. J. Biochem. Cell Bio.* **31**, 1249–1254.

143. Qian, S. W., Burmester, J. K., Sun, P. D., Huang, A., Ohlsen, D. J., Suardet, L., Flanders, K. C., Davies, D., Roberts, A. B., and Sporn, M. B. (1994). Characterization of mutated transforming growth factor-betas which possess unique biological properties. *Biochemistry* **33**, 12298–12304.

144. Massague, J. (1998). TGF-beta signal transduction. *Ann. Rev. Biochem.* **67**, 753–791.

145. Derynck, R., Akhurst, R. J., and Balmain, A. (2001). TGF-beta signaling in tumor suppression and cancer progression. *Nat. Genet.* **29**, 117–129, [erratum appears in *Nat. Genet.* 2001, Nov; **29(3)**, 351].

146. Kusano, K., Miyaura, C., Inada, M., Tamura, T., Ito, A., Nagase, H., Kamoi, K., and Suda, T. (1998). Regulation of matrix metalloproteinases (MMP-2, -3, -9, and -13) by interleukin-1 and interleukin-6 in mouse calvaria: Association of MMP induction with bone resorption. *Endocrinology* **139**, 1338–1345.

147. Varghese, S., Delany, A. M., Liang, L., Gabbitas, B., Jeffrey, J. J., and Canalis, E. (1996). Transcriptional and posttranscriptional regulation of interstitial collagenase by platelet-derived growth factor BB in bone cell cultures. *Endocrinology* **137**, 431–437.

148. Varghese, S., Ramsby, M. L., Jeffrey, J. J., and Canalis, E. (1995). Basic fibroblast growth factor stimulates expression of interstitial collagenase and inhibitors of metalloproteinases in rat bone cells. *Endocrinology* **136**, 2156–2162.

149. Varghese, S., Rydziel, S., and Canalis, E. (2000). Basic fibroblast growth factor stimulates collagenase-3 promoter activity in osteoblasts through an activator protein-1-binding site. *Endocrinology* **141**, 2185–2191.

150. Zhao, W., Byrne, M. H., Wang, Y., and Krane, S. M. (2000). Osteocyte and osteoblast apoptosis and excessive bone deposition accompany failure of collagenase cleavage of collagen. *J. Clin. Invest.* **106**, 941–949.

151. Selvamurugan, N., Kwok, S., Alliston, T., Reiss, M., and Partridge, N. C. (2004). Transforming growth factor-beta 1 regulation of collagenase-3 expression in osteoblastic cells by cross-talk between the Smad and MAPK signaling pathways and their components, Smad2 and Runx2. *J. Biol. Chem.* **279**, 19327–19334.

152. Uria, J. A., Jimenez, M. G., Balbin, M., Freije, J. M., and Lopez-Otin, C. (1998). Differential effects of transforming growth factor-beta on the expression of collagenase-1 and collagenase-3 in human fibroblasts. *J. Biol. Chem.* **273**, 9769–9777.

153. Zhang, Y., and Derynck, R. (2000). Transcriptional regulation of the transforming growth factor-beta-inducible mouse germ line Ig alpha constant region gene by functional cooperation of Smad, CREB, and AML family members. *J. Biol. Chem.* **275**, 16979–16985.

154. Leivonen, S. K., Chantry, A., Hakkinen, L., Han, J., and Kahari, V. M. (2002). Smad3 mediates transforming growth factor-beta-induced collagenase-3 (matrix metalloproteinase-13) expression in human gingival fibroblasts. Evidence for cross-talk between Smad3 and p38 signaling pathways. *J. Biol. Chem.* **277**, 46338–46346.

155. Tardif, G., Reboul, P., Dupuis, M., Geng, C., Duval, N., Pelletier, J. P., and Martel-Pelletier, J. (2001). Transforming growth factor-beta induced collagenase-3 production in human osteoarthritic chondrocytes is triggered by Smad proteins: Cooperation between activator protein-1 and PEA-3 binding sites. *J. Rheum.* **28**, 1631–1639.

156. Johansson, N., Ala-aho, R., Uitto, V., Grenman, R., Fusenig, N. E., Lopez-Otin, C., and Kahari, V. M. (2000). Expression of collagenase-3 (MMP-13) and collagenase-1 (MMP-1) by transformed keratinocytes is dependent on the activity of p38 mitogen-activated protein kinase. *J. Cell Sci.* **2**, 227–235.

157. Franceschi, R. T., Xiao, G., Jiang, D., Gopalakrishnan, R., Yang, S., and Reith, E. (2003). Multiple signaling pathways converge on the Cbfal/Runx2 transcription factor to regulate osteoblast differentiation. *Conn. Tiss. Res.* **44**, 109–116.

158. Xiao, G., Jiang, D., Gopalakrishnan, R., and Franceschi, R. T. (2002). Fibroblast growth factor 2 induction of the osteocalcin gene requires MAPK activity and phosphorylation of the osteoblast transcription factor, Cbfal/Runx2. *J. Biol. Chem.* **277**, 36181–36187.

159. Choi, J. Y., Pratap, J., Javed, A., Zaidi, S. K., Xing, L., Balint, E., Dalamangas, S., Boyce, B., van Wijnen, A. J., Lian, J. B., Stein, J. L., Jones, S. N., and Stein, G. S. (2001). Subnuclear targeting of Runx/Cbfa/AML factors is essential for tissue-specific differentiation during embryonic development. *Proc. Natl. Acad. Sci. USA* **98**, 8650–8655.

160. Uchiyama-Tanaka, Y., Matsubara, H., Mori, Y., Kosaki, A., Kishimoto, N., Amano, K., Higashiyama, S., and Iwasaka, T. (2002). Involvement of HB-EGF and EGF receptor transactivation in TGF-beta-mediated fibronectin expression in mesangial cells. *Kidney International* **62**, 799–808.

161. Lee, K. S., Hong, S. H., and Bae, S. C. (2002). Both the Smad and p38 MAPK pathways play a crucial role in Runx2 expression following induction by transforming growth factor-beta and bone morphogenetic protein. *Oncogene* **21**, 7156–7163.

TRANSCRIPTION IN THE OSTEOBLAST 321

162. Mengshol, J. A., Vincenti, M. P., and Brinckerhoff, C. E. (2001). IL-1 induces collagenase-3 (MMP-13) promoter activity in stably transfected chondrocytic cells: Requirement for Runx-2 and activation by p38 MAPK and JNK pathways. *Nucleic Acids Res.* **29**, 4361–4372.

163. Pouponnot, C., Jayaraman, L., and Massague, J. (1998). Physical and functional interaction of SMADs and p300/CBP. *J. Biol. Chem.* **273**, 22865–22868.

164. Feng, X. H., Zhang, Y., Wu, R. Y., and Derynck, R. (1998). The tumor suppressor Smad4/DPC4 and transcriptional adaptor CBP/p300 are coactivators for smad3 in TGF-beta-induced transcriptional activation. *Genes Devel.* **12**, 2153–2163.

165. Ulloa, L., Doody, J., and Massague, J. (1999). Inhibition of transforming growth factor-beta/SMAD signalling by the interferon-gamma/STAT pathway. *Nature* **397**, 710–713.

166. Kim, S., Koga, T., Isobe, M., Kern, B. E., Yokochi, T., Chin, Y. E., Karsenty, G., Taniguchi, T., and Takayanagi, H. (2003). Stat1 functions as a cytoplasmic attenuator of Runx2 in the transcriptional program of osteoblast differentiation. *Genes Devel.* **17**, 1979–1991.
