
The Journal of Clinical Investigation. More information at www.jci.org/articles/view/26390

Review series

Autophagy in cell death: an innocent convict?

Beth Levine¹ and Junying Yuan²

¹Departments of Internal Medicine and Microbiology, University of Texas Southwestern Medical Center, Dallas, Texas, USA.
²Department of Cell Biology, Harvard Medical School, Boston, Massachusetts, USA.

The visualization of autophagosomes in dying cells has led to the belief that autophagy is a nonapoptotic form of programmed cell death. This concept has now been evaluated using cells and organisms deficient in autophagy genes. Most evidence indicates that, at least in cells with intact apoptotic machinery, autophagy is primarily a pro-survival rather than a pro-death mechanism. This review summarizes the evidence linking autophagy to cell survival and cell death, the complex interplay between autophagy and apoptosis pathways, and the role of autophagy-dependent survival and death pathways in clinical diseases.

Cell biologists have long recognized the possibility that eukaryotic cells may undergo nonapoptotic forms of programmed cell death. Autophagy, a lysosomal pathway involving the bulk degradation of cytoplasmic contents, has been identified as a prime suspect in such death, and recent studies have implicated the autophagy pathway as a cause of nonapoptotic cellular demise. However, most evidence linking autophagy to cell death is circumstantial. Now, with new tools to assess causality, it is an opportune time to revisit the case of autophagy in cell death. Is autophagy an innocent bystander, a direct death execution pathway, a defense mechanism that ultimately fails in its mission to preserve cell viability, and/or a garbage disposal mechanism that cleans up remnants of a cell already committed to die?

### Autophagy is a regulated lysosomal degradation pathway

The term *autophagy* (Greek, “to eat oneself”) does not refer to a death process; it denotes the process of self-cannibalization through a lysosomal degradation pathway. Autophagy is the cell’s major regulated mechanism for degrading long-lived proteins and the only known pathway for degrading organelles (reviewed in refs. 1, 2). During autophagy, an isolation membrane forms, presumably arising from a vesicular compartment known as the preautophagosomal structure, invaginates, and sequesters cytoplasmic constituents including mitochondria, endoplasmic reticulum, and ribosomes (Figure 1). The edges of the membrane fuse to form a double or multilamellar structure, known as the autophagosome or autophagic vacuole. The outer membrane of the autophagosome fuses with the lysosome (in mammalian cells) or vacuole (in yeast and plants) to deliver the inner membranous vesicle to the lumen of the degradative compartment. Degradation of the sequestered material generates nucleotides, amino acids, and free fatty acids that are recycled for macromolecular synthesis and ATP generation.

Autophagy occurs at low basal levels in all cells to perform homeostatic functions (e.g., cytoplasmic and organelle turnover) but is rapidly upregulated when cells need to generate intracellular nutrients and energy (e.g., during starvation or trophic factor withdrawal), undergo architectural remodeling (e.g., during developmental transitions),

or rid themselves of damaging cytoplasmic components (e.g., during oxidative stress, infection, and accumulation of protein aggregates). Nutritional status, hormonal factors, and other cues like temperature, oxygen concentrations, and cell density are important in the control of autophagy. Two evolutionarily conserved nutrient sensors play roles in autophagy regulation: (a) the target of rapamycin (TOR) kinase is the major inhibitory signal that shuts off autophagy during nutrient abundance (reviewed in ref. 3), and (b) the eukaryotic initiation factor 2α (eIF2α) kinase Gcn2 and its downstream target Gcn4, a transcriptional transactivator of autophagy genes, turn on autophagy during nutrient depletion (4). The class I PI3K/Akt signaling molecules link receptor tyrosine kinases to TOR activation and thereby repress autophagy in response to insulin-like and other growth factor signals (reviewed in ref. 3).

Downstream of TOR kinase, there are approximately 17 gene products essential for autophagy and related pathways in yeast, referred to as the ATG genes (5), and most yeast ATG genes have orthologs in higher eukaryotes (reviewed in ref. 2). The ATG genes encode proteins needed for the induction of autophagy, and the generation, maturation, and recycling of autophagosomes. These proteins are composed of 4 functional groups, including a protein serine/threonine kinase complex that responds to upstream signals such as TOR kinase (Atg1, Atg13, Atg17), a lipid kinase signaling complex that mediates vesicle nucleation (Atg6, Atg14, Vps34, and Vps15), 2 novel ubiquitin-like conjugation pathways that mediate vesicle expansion (the Atg8 and Atg12 systems), and a recycling pathway that mediates the disassembly of Atg proteins from matured autophagosomes (Atg2, Atg9, Atg18) (Figure 2).

### Autophagy as a cell death mechanism

The term “autophagic cell death” describes a form of programmed cell death morphologically distinct from apoptosis and presumed to result from excessive levels of cellular autophagy (6). In classical apoptosis, or type I programmed cell death, there is early collapse of cytoskeletal elements but preservation of organelles until late in the process. In contrast, in autophagic, or type II, programmed cell death, there is early degradation of organelles but preservation of cytoskeletal elements until late stages. Whereas apoptotic cell death is caspase-dependent and characterized by internucleosomal DNA cleavage, caspase activation and DNA fragmentation occur very late (if at all) in autophagic cell death (Figure 3). In contrast with necrosis, both apoptotic and autophagic cell death are characterized by the lack of a tissue inflammatory response.

Large numbers of autophagic vacuoles have been observed in dying cells of animals of diverse taxa (reviewed in refs. 6–9) (Table 1).

Nonstandard abbreviations used: Htt, Huntingtin; 3-MA, 3-methyladenine; MEF, murine embryonic fibroblast; MPT, mitochondrial permeability transition; polyQ, polyglutamine; RNAi, RNA interference; TOR, target of rapamycin; TRAIL, TNF-related apoptosis-inducing ligand.

Conflict of interest: The authors have declared that no conflict of interest exists.

Citation for this article: J. Clin. Invest. 115:2679–2688 (2005).
doi:10.1172/JCI26390.

review series

![Diagram](attachment:diagram.png)

Figure 1  
The autophagy pathway and its role in cellular adaptation to nutrient deprivation. Starvation or growth factor deprivation results in a decrease in intracellular nutrients and activation of nutrient-sensing signaling pathways (reviewed in ref. 97) that stimulate autophagy. Autophagy involves the sequestration of cytoplasmic material by an isolation membrane (derived from the preautophagosomal structure) to form a double-membrane vacuole, the autophagosome. The autophagosome undergoes fusion with a late endosome or lysosome, to form an autolysosome, in which the sequestered material is degraded. Degradation of membrane lipids and proteins by the autolysosome generates free fatty acids and amino acids that can be reused by the cell to maintain mitochondrial ATP energy production and protein synthesis and thereby promote cell survival. Disruption of this pathway by autophagy gene inactivation prevents cell survival in diverse organisms (Table 2). The same molecular machinery and overlapping dynamic membrane rearrangement events that occur during starvation may also be used in other settings to degrade unwanted cytoplasmic contents, including damaged mitochondria, protein aggregates, and intracellular pathogens. See text for discussion. TCA cycle, tricarboxylic acid cycle.

The consensus view has been that autophagic cell death occurs primarily when the developmental program (e.g., insect metamorphosis) or homeostatic processes in adulthood (e.g., mammary gland postlactational involution, prostate involution following castration) require massive cell elimination. Recently, studies have also described autophagic cell death in diseased mammalian tissues and in tumor cell lines treated with chemotherapeutic agents (Table 1). In many of these cases, morphologic features of autophagic and apoptotic cell death or of autophagic and necrotic cell death are observed in the same cell.

What is the evidence that autophagy is a death execution mechanism in autophagic cell death? If cell death is truly due to autophagy, then pharmacologic or genetic inhibition of autophagy should prevent the death. Yet, for most of the developmental, disease-associated, and toxic stimulus-induced deaths that are presumed to be autophagic (Table 1), the evidence for its role is only correlative. Moreover, in certain cases of autophagic cell death, the available evidence calls into question a causative role of autophagy. For example, in *Drosophila*, autophagic cell death but not autophagy observed during salivary gland regression is prevented by mutations in the ecdysone-regulated transcription factors BR-C and E74A (10). In the slime mold *Dictyostelium*, a null mutation in the autophagy gene *atg1* blocks vacuolization but not cell death in an in vitro model of autophagic cell death (11). Thus, in these model systems, autophagy per se is neither sufficient nor required for autophagic cell death. Furthermore, the caspase inhibitor p35 blocks metamorphic cell death in *Drosophila* without complete inhibition of autophagy, suggesting that it is caspase-mediated apoptosis, rather than autophagy, that plays a key role in this death process (10).

There is, however, some evidence in certain in vitro settings that pharmacologic or genetic inhibition of autophagy can prevent cell death. The pharmacologic inhibitor of autophagy 3-methyladenine (3-MA), a nucleotide derivative that blocks class III PI3K activity (12–14), delays or partially inhibits death in starved hepatocytes from carcinogen-treated rats (15), in anti-estrogen-treated human mammary carcinoma cells (16), in chloroquine-treated cortical neurons.

The Journal of Clinical Investigation. More information at www.jci.org/articles/view/26390

review series

**Figure 2**

The molecular mechanisms of autophagy. The autophagy (Atg) proteins can be divided into 4 functional groups, including (A) a protein kinase autophagy regulatory complex that responds to upstream signals, including nutrient limitation; (B) a lipid kinase signaling complex that mediates vesicle nucleation; (C) ubiquitin-like protein conjugation pathways that are required for vesicle expansion and completion; and (D) a retrieval pathway required for the disassembly of Atg protein complexes from matured autophagosomes. Shown are the yeast Atg proteins that participate in each functional group. Yeast Atg proteins with known orthologs in higher eukaryotes are underlined. PI, phosphatidylinositol; PI3-P, phosphatidylinositol 3-phosphate; PE, phosphatidylethanolamine.

blocked cell death in mouse L929 cells treated with the caspase inhibitor zVAD (23). Further, RNAi against autophagy genes *atg5* and *beclin 1* blocked death of *bax⁻/⁻*, *bak⁻/⁻* murine embryonic fibroblasts (MEFs) treated with staurosporine or etoposide (24). Notably, in both of these studies, *atg* gene RNAi blocked the death of cells whose apoptotic pathway had been crippled. Although these findings exclude the possibility that autophagy is triggering death through apoptosis induction, they raise the question of whether autophagy is a death mechanism in cells whose apoptotic machinery is intact.

Interestingly, in etoposide-treated wild-type MEFs (which die by apoptosis), only minimal autophagic activity and no inhibition of death by 3-MA is seen, indicating that autophagy is not involved in the death process unless apoptosis is blocked (24). These data are consistent with the theory previously proposed by Lockshin and Zakeri that cells preferentially die by apoptosis but will die by any alternative available route, including autophagy, if exposed to harsh enough stimuli (9). A related possibility is that apoptotic death is faster than autophagic death and, therefore, autophagy is only witnessed playing a role in cell death in apoptotic-deficient cells. This hypothesis is consistent with recent data indicating that growth factor-deprived wild-type cells undergo a rapid apoptotic death, whereas growth factor-deprived *bax⁻/⁻*, *bak⁻/⁻* cells undergo a slow demise characterized by progressive self-cannibalization (25).

Given the uncertain physiologic relevance of autophagy gene-dependent cell death in zVAD-treated cells or in *bax⁻/⁻*, *bak⁻/⁻* cells, it seems premature to conclude that autophagy is a physiologically important cause of cell death. To prove that autophagy is an important cell death pathway in normal cells, it will be necessary to demonstrate cell death resistance phenotypes in apoptotic-competent cells lacking autophagy genes.

**Autophagy as a cell survival mechanism**

A contradictory, but equally plausible, explanation for the presence of autophagy in dying cells is that activation of autophagy is a cellular survival strategy. This concept was first proposed in 1977 (26) and considered radical (8) but now is supported by studies demonstrating increased death in cells or organisms lacking gene products essential for autophagy (Table 2). The pro-survival function of autophagy is an evolutionarily ancient process, conserved from yeast to mammals, and best characterized in nutrient deficiency. During nutrient deficiency, degradation of membrane lipids and proteins by the autolysosome generates free fatty acids and amino acids that can be reused to fuel mitochondrial ATP energy production and maintain protein synthesis (Figure 1). Presumably, this recycling function of autophagy is linked mechanistically to its ability to sustain life during starvation.

review series

![Figure 3](image.png)

**Figure 3**

Ultrastructural examples of apoptotic and autophagic cell death. Electron micrographs of a FasL-treated Jurkat cell undergoing cell death with apoptotic features (A) and of a tamoxifen-treated MCF7 human breast carcinoma cell undergoing cell death with autophagic features (B). In A, note chromatin condensation (cell in center) and cytoplasmic vacuolization (cell in upper right). In B, note absence of chromatin condensation and presence of numerous autophagosomes. Images in A and B reproduced with permission from *Nature Cell Biology* (98) and Landes Bioscience (90), respectively.

Unicellular organisms with null mutations in autophagy genes are viable in normal growth conditions; however, unlike their wild-type counterparts, they die rapidly during starvation (Table 2) (27–29). In plants, deletion of autophagy genes (e.g., ATG6, ATG7, and ATG9) results in loss of chlorophyll and accelerated senescence following nutrient deprivation (30–32). Mice lacking Atg5, an acceptor molecule for the ubiquitin-like molecule Atg12, die during the neonatal period, when the placental blood supply is interrupted and they undergo a form of starvation (33). Atg5⁻/⁻ mice have decreased amino acid levels, decreased cardiac ATP production, and myocardial damage. Although the death of individual cells has not yet been assessed in atg5⁻/⁻ mice, it is predicted that the recycling function of autophagy is critical to maintain cellular energy homeostasis and cellular survival during the neonatal period. This is particularly likely in tissues such as the heart and diaphragm that have sudden increases in energy needs and exhibit increased autophagy immediately following birth. Similarly, the inability of *Caenorhabditis elegans* with RNAi-silenced autophagy genes (e.g., *unc-51*, *bec-1*, *atg8*, and *atg18*) to survive during dauer diapause (34) likely reflects an inability to recycle nutrients at the organismal level.

Autophagy genes may also be critical for maintaining cellular bioenergetics and survival when cells are unable to take up external nutrients (i.e., during growth factor deprivation). In the absence of growth factors such as IL-3, there is decreased surface expression of nutrient transporters, decreased nutrient uptake, and an intracellular deficiency of nutrients (35). Growth factor withdrawal usually results in rapid apoptotic cell death, but recent studies in apoptotic-deficient *bax⁻/⁻, bak⁻/⁻* cells have unraveled an essential role for autophagy genes (e.g., *atg5, atg7*) in maintaining cellular survival following IL-3 deprivation (25). As in nutrient starvation in yeast, autophagy is a self-limited survival strategy during growth factor deprivation. IL-3-deprived *bax⁻/⁻, bak⁻/⁻* cells eventually die, presumably because of excessive self-consumption and bioenergetic failure. However, at any point before death, the addition of growth factor reverses the catabolic process and maintains cell viability. These observations are consistent with the concept that autophagy is a self-limited survival strategy, rather than a primary or irreversible death execution program.

Like the pro-death function of autophagy genes in etoposide-treated *bax⁻/⁻, bak⁻/⁻* knockout cells, it will be important to determine whether this pro-survival function of autophagy genes during growth factor deprivation in *bax⁻/⁻, bak⁻/⁻* cells is conserved in cells with intact apoptotic machinery. It will also be interesting to examine whether autophagy genes play a similar cytoprotective role during withdrawal of hormonal support or growth factors besides IL-3. Perhaps, rather than contributing to death execution, autophagy delays initiation of the apoptotic death pathway in cells deprived of trophic support. Although studies have not been performed in apoptosis-competent cells deprived of trophic factors, autophagy genes prevent the onset of apoptosis during nutrient deprivation. RNAi against *beclin 1, atg5, atg10,* and *atg12* enhances starvation-induced, but not staurosporine-induced, apoptotic cell death (36). Thus, the mechanism by which autophagy genes promote survival during nutrient deprivation may involve suppression of the canonical apoptotic death pathway.

The mechanisms by which autophagy promotes cell survival are not restricted to its role in maintaining cellular energy homeostasis during starvation. Autophagy is also involved in removing damaged mitochondria and other organelles, in degrading intracellular pathogens, and in degrading protein aggregates too large to be removed by the ubiquitin-proteasomal system. These functions of autophagy could promote cellular survival during aging, infectious diseases, and neurodegenerative processes. In addition to a cell-autonomous role for autophagy in promoting survival, autophagy may regulate programmed cell death during physiologic processes in vivo. For example, during the plant innate immune response, silencing of autophagy genes *beclin 1, vps34, atg3,* and *atg7* does not alter the death of infected cells or pathogen spread but results in uncontrolled spread of programmed cell death beyond sites of pathogen infection (32). This suggests that autophagy limits cell death to the site of infection, allowing plant innate immunity to contain pathogen spread without death of innocent bystander cells. It is not known whether autophagy alters the production of death-promoting signals, prevents the movement of death-promoting signals into uninfected tissues, or protects uninfected tissues against death induced by these signals, or whether a similar function of autophagy plays a role in the spatial restriction of development and stress-induced programmed cell death in other eukaryotic organisms.

### Autophagy as a self-clearance mechanism

A third explanation for high levels of autophagy in dying cells is that it is a clean-up or self-clearance mechanism in cells committed to die by apoptosis or necrosis. This theory might explain why only selected populations of dying apoptotic cells have morphologic features of autophagy. The dogma is that most apoptotic cells are engulfed by phagocytes, with the lysosomes of the phagocyte responsible for the final degradation of dead cell bodies. However, in some forms of developmental programmed cell death (e.g., embryogenesis, insect metamorphosis), the availability of engulfment cells may be insufficient for clearance of dead cells. In such cases, dying cells may activate autophagy to target the cell’s contents for degradation by its own lysosomes.

This need might contribute to the overlap between signaling pathways that activate apoptosis and autophagy. It has been shown that the proapoptotic signaling molecule TNF-related apoptosis-inducing ligand (TRAIL) regulates autophagy in an in vitro model of mammary gland formation (37). Here, TRAIL-dependent induction of autophagy occurs in parallel with apoptosis. Suppression

Table 1  
Examples of cell death with morphologic features of autophagy$^A$

| Species | Tissue/cell type | Setting | References |
| --- | --- | --- | --- |
| **Developmental programmed cell death** |  |  |  |
| **Insects** |  |  |  |
| Chironomus tentans | Salivary gland | Metamorphosis | (8) |
| Manduca sexta | Intersegmental muscle, labial gland, prothoracic gland, larval fat body, motoneuron | Metamorphosis | (99, 100; reviewed in ref. 7) |
| Gryllus bimaculatus | Prothoracic gland | Metamorphosis | (7, 101) |
| Calliphora vomitoria | Salivary gland | Metamorphosis | (103) |
| Drosophila melanogaster | Salivary gland, midgut cells, larval fat body | Metamorphosis | (10, 103–105) |
| Orgyia leucostigma | Epithelial wing | Adult reduction in females | (reviewed in ref. 7) |
| **Birds** |  |  |  |
| Chicken | Limb bud, neurons, mesonephros, müllerian duct epithelium, feather melanocytes, heart | Embryonic development | (reviewed in refs. 7, 8) |
| Quail | Ovarian follicle | Atresia | (7) |
| **Amphibians** |  |  |  |
| Frog | Tail nerve cord | Metamorphosis | (8) |
|  | Gills, neurons | Larval development |  |
| **Mammals** |  |  |  |
| Mouse | Palatal epithelium | Embryonic development | (8) |
| Human | Mammary epithelial cells | Lumen formation | (37, 106) |
| **Disease-associated cell death** |  |  |  |
| **Mammals** |  |  |  |
| Mouse | Brain (cerebellar Purkinje cells) | Lurcher mutation | (67) |
|  | Striatal neuronal cell line | Mutant Huntingtin expression | (107) |
|  | Macrophages | Salmonella infection | (83) |
| Rat | PC12 neuronal cell line | Mutant α-synuclein expression | (108) |
| Hamster | Brain | Experimentally induced transmissible spongiform | (65) |
| Gerbil | Hippocampus | Ischemic injury | (109) |
| Human | Heart | Dilated cardiomyopathy | (110) |
|  | Dopaminergic neurons | Aortic stenosis | (111) |
|  |  | Parkinson disease | (63) |
| **Drug-, toxin-, or stress stimulus–associated cell death** |  |  |  |
| **Protozoans** |  |  |  |
| Leishmania donovani |  | Antimicrobial peptide treatment | (112) |
| Tetrahymena thermophila |  | Staurosporine treatment | (113) |
| Dictyostelium discoideum |  | Dual exposure to starvation and differentiation-induced factor | (114) |
| **Mammals** |  |  |  |
| Mouse | Heart | Diphtheria toxin treatment | (115) |
|  | Cortical neurons | Chloroquine treatment | (17) |
|  | Neural precursor cells | FGF withdrawal | (50) |
| Rat | Sympathetic neurons | Nerve growth factor withdrawal | (18) |
|  | Cultured cerebellar granule cells | Serum and potassium deprivation | (19) |
|  | Retinal explant | Anisomycin treatment | (116) |
|  | Hippocampal neurons | N-methyl-D-aspartate treatment | (117) |
|  | PC12 pheochromocytoma cells | Serum deprivation | (118) |
|  | Oral keratinocytes | 5-Fluorouracil treatment | (119) |
| Human | Ovarian carcinoma cell lines | Resveratrol treatment | (120) |
|  | Mammary carcinoma cell line | Anti-estrogen treatment | (16, 121) |
|  | Glioma cell lines | Arsenic trioxide treatment | (95, 122) |
|  | Endothelial cells | Ceramide treatment | (43) |
|  |  | Endostatin treatment | (123) |

$^A$Includes only those references in which (a) autophagy is observed at the ultrastructural level, and (b) autophagy is postulated by the authors to be a cause of death; excludes references in which (a) autophagy is assessed only at the light microscopic or biochemical level (e.g., by monodansylcadaverine staining, GFP-Atg8/LC3 staining, or LC3I-to-LC3II conversion), or (b) autophagy is not postulated to be mechanistically important in death.

review series

Table 2  
Examples of autophagy gene–dependent cell death, cell survival, and metazoan survival  

| Species | Tissue/cell type | ATG gene | Setting<sup>A</sup> | References |
|---------|------------------|----------|--------------------|------------|
| **Cell death** |  |  |  |  |
| Mammals |  |  |  |  |
| Mouse | $bax^{-/-}, bak^{-/-}$ embryonic fibroblasts | atg5, beclin 1 | Etoposide or staurosporine treatment | (24) |
| Human | L929 fibroblast cells, U937 monocyte-like cells | atg7, beclin 1 | Caspase inhibition | (23) |
| **Cell survival** |  |  |  |  |
| Yeast | *Saccharomyces cerevisiae* | All ATG genes | Starvation | (27) |
| Protozoans | *Dictyostelium discoideum* | ATG1, ATG5, ATG6, ATG8, ATG12 | Starvation | (28, 29) |
| Plants |  |  |  |  |
| *Arabidopsis thaliana* | Leaves | ATG7, ATG9 | Starvation | (30, 31) |
| *Nicotiana benthamiana* | Leaves | ATG3, ATG7, beclin 1, vps34 | Pathogen infection/innate immune response | (32) |
| Mammals |  |  |  |  |
| Mouse | $bax^{-/-}, bak^{-/-}$ hematopoietic cells | atg5, atg7 | IL-3 withdrawal | (25) |
| Brain |  | beclin 1<sup>B</sup> | Alphavirus encephalitis | (79) |
| Human | HeLa cells | atg5, beclin 1, atg10, atg12 | Starvation | (36) |
| **Metazoan survival** |  |  |  |  |
| *Caenorhabditis elegans* |  | unc-51, bec-1, atg7, atg8, atg18 | Dauer development | (34) |
|  |  | bec-1, atg8, atg18 | Larval development | (34) |
| *Drosophila* |  | ATG1, ATG3 | Larval/pupal development | (85, 124) |
| Mouse |  | beclin 1 | Embryonic development | (88, 89) |
|  |  | atg5 | Early postnatal starvation | (33) |

<sup>A</sup>Setting indicates conditions in which mutation in the indicated autophagy gene blocks cell death, cell survival, or metazoan survival. <sup>B</sup>The role of this gene was assessed by overexpression; in all other studies referenced in the table, the role of each gene was assessed by RNAi silencing or loss-of-function mutation.

of either apoptosis alone or TRAIL signaling does not prevent lumen formation, but simultaneous inhibition of apoptosis and TRAIL signaling prevents cell clearance.

These findings suggest that both apoptosis and autophagy may be involved in cavitation during mammary gland morphogenesis. However, to confirm a role for autophagy, it will be important to observe whether luminal filling occurs if autophagy genes are inactivated in cells with intact TRAIL signaling. It is not clear whether autophagy is required for luminal cell death or for removal of cells committed to death by an apoptotic pathway. Similarly, it is not known whether autophagy is required for caspase-dependent death and/or for the clearance of dying cells during insect metamorphosis.

### Cross-talk between apoptotic signaling, autophagy, and mitochondria

Several proapoptotic signals induce autophagy — e.g., components of the extrinsic apoptosis pathway, TRAIL, TNF, and FADD (21, 37–40); the calcium/calmodulin-regulated serine/threonine kinases DRP-1 and DAPk (41); and ceramide (42, 43). Conversely, antiapoptotic signaling pathways suppress autophagy — e.g., the class I PI3K/Akt/TOR signaling pathway (reviewed in refs. 3, 44). Coordinated regulation of apoptosis and autophagy is also reflected in the results of genome-wide analyses of transcriptional changes during developmental programmed cell death of the *Drosophila* salivary gland (45, 46).

The mitochondrion may integrate cell death signals and autophagy activation. Mitochondria generate apoptotic signals but are removed when damaged by autophagy; therefore, mitochondria represent a nexus at which autophagy and apoptosis pathways may interact. Accordingly, genes involved in mitochondrial physiology and/or mitochondrial regulation of apoptosis interact with the autophagy pathway. One example is the yeast gene, *UTH1*, that encodes an outer mitochondrial membrane protein involved in mitochondrial biogenesis and stress responses. *Uth1* mutants are defective in degrading mitochondria during autophagy (47) and survive and proliferate when expressing the mammalian proapoptotic cell death gene *bax* or when treated with the autophagy inducer rapamycin (48). These findings led Camougrand and colleagues to suggest that Uth1p mediates mitochondrial autophagy and autophagic death. However, it is not yet clear whether rapamycin induces cell death versus cell cycle arrest in wild-type yeast, and whether the phenotype of rapamycin-treated *uth1* mutant yeasts is due to direct effects of *UTH1* and the autophagy pathway in death regulation.

In mammalian cells, Bcl-2 family members in the outer mitochondrial membrane modulate autophagy. Bcl-2 downregulation increases autophagy in a caspase-independent manner in human leukemic cells (49), and Bcl-2 overexpression inhibits both autophagy and caspase-independent death in growth factor-deprived neural progenitor cells and in serum- and potassium-deprived cultured cerebellar granule cells (19, 50). Recent evidence suggests
that Bcl-2 inhibits autophagy through a direct interaction with the Beclin 1 autophagy protein and that the interaction between Bcl-2 and Beclin 1 may function as a rheostat that maintains autophagy at levels that are compatible with cell survival rather than cell death (51). In contrast, Bcl-2 or Bcl-xL overexpression potentiates autophagy and autophagy gene-dependent death in MEFs treated with the proapoptotic stimulus etoposide (24). The basis for the opposite effects of Bcl-2 family members on autophagy in different settings is unclear. Furthermore, it is not yet clear that Bcl-2 proteins function at the mitochondrion to regulate autophagy, since autophagy is inhibited by Bcl-2 targeted to endoplasmic reticulum but not by Bcl-2 targeted to mitochondria (51).

The role of proapoptotic Bcl-2 family members in autophagy gene–dependent life-and-death decisions is also controversial. As discussed earlier, $bax^{-/-}, bak^{-/-}$ cells undergo autophagy gene–dependent death when treated with etoposide (24) but undergo autophagy gene–dependent survival when deprived of trophic factor support (25). It is possible that in the setting of $bax/bak$ deficiency, the stimulus plays a critical role in determining cell fate, and that etoposide, but not growth factor deprivation, can target an intracellular pathway that turns autophagy into a deadly process. Some atypical Bcl-2 family members, including BNIP3 and Hspin, also activate autophagy and nonapoptotic cell death (52–54), but it is not yet known whether this caspase-independent cell death requires autophagy genes.

Another question is how the autophagy pathway recognizes damaged mitochondria. The mitochondrial permeability transition (MPT) may trigger the engulfment of depolarized mitochondria by autophagy (55). However, it is not known whether inhibition of autophagy increases the numbers of depolarized mitochondria in mammalian cells or how a depolarized mitochondrion might be targeted to autophagosomes. The MPT may represent a point of convergence of apoptotic and autophagy pathways, since Bcl-2 family members regulate the MPT. The proapoptotic family member Bax interacts with the voltage-dependent anion channel (56) and/or the adenine nucleotide translocator (57) to induce the MPT in cells and isolated mitochondria upon induction of apoptosis. It is currently not clear, however, whether the MPT regulated by Bax triggers mitochondrial turnover by autophagy.

In many canonical apoptosis pathways, the MPT is caspase–dependent (58). Thus, if the MPT is a critical signal for mitochondrial degradation through autophagy, inhibition of caspases should prevent the loss of mitochondria. However, Tolkovsky and coworkers reported that, although caspase inhibitors effectively inhibit neuronal cell death, they fail to prevent the formation of autophagosomes or the degradation of mitochondria. In fact, the long-term culturing of neurons in the presence of proapoptotic stimuli and caspase inhibitors leads to the loss of mitochondria (59, 60). Thus, the relationship among MPT, caspase-dependent cell death, and mitochondrial autophagy remains unclear.

### Autophagy in neurodegenerative diseases

The accumulation of mutant or toxic proteins plays a major role in chronic neurodegenerative diseases (61). Morphologic evidence of autophagy has been reported in neurodegenerative diseases including Parkinson, Huntington, and Alzheimer diseases, and transmissible spongiform encephalopathies (62–65). It is possible that autophagy activation contributes to neurodegeneration (66, 67), but the evidence is correlative. A contrasting view is that autophagy may be a protective mechanism to degrade mutant or toxic proteins. According to this model, defects in autophagy-related pathways contribute to the accumulation of neurotoxic proteins and the ensuing neuronal cell death. Although the exact roles of autophagy in neurodegenerative diseases are not fully defined, recent studies have provided some insights.

The protein α-synuclein is a major component of neuronal cytoplasmic inclusions that characterize Parkinson and other neurodegenerative diseases (68). Although earlier studies suggested that α-synuclein is degraded through both the proteasome and classical autophagy pathways (69), a recent study demonstrated that the turnover of α-synuclein is regulated by chaperone-mediated autophagy, which involves the direct lysosomal targeting of proteins containing specific pentapeptide recognition motifs (70). Interestingly, pathogenic α-synuclein mutants associated with familial, autosomal-dominant forms of Parkinson disease (71, 72) are inefficiently degraded by chaperone-mediated autophagy. Since the accumulation of wild-type α-synuclein in neuronal inclusions is common in adult-onset neurodegenerative diseases, these experiments suggest that defects in autophagy-related pathways may contribute to multiple neurodegenerative diseases.

Consistent with this hypothesis, autophagy has also been implicated in regulating the turnover of Huntingtin (Htt), the protein involved in Huntington disease, an autosomal-dominant neurodegenerative disorder caused by the expansion of a polyglutamine (polyQ) tract in Htt. Although the mechanism of neurotoxicity mediated by expanded polyQ is still controversial, expanded polyQ provokes a dominant gain-of-function neurotoxicity, regardless of the specific protein context within which it resides. The accumulation of expanded polyQ-containing proteins in insoluble aggregates in affected neurons is a hallmark feature of Huntington and other polyQ expansion diseases (73).

Although neuronal Htt proteins in inclusions are highly ubiquitinated, polyQ is a poor substrate for proteasomes (74). Thus, the highly ubiquitinated state of Htt inclusions may indicate the inability of proteasomes in affected neurons to clear abnormal Htt proteins. In contrast, there is pharmacologic evidence to suggest a role for autophagy in the degradation of the N-terminus of Htt. For example, 3-MA increases the aggregation of Htt with expanded polyQ in clonal striatal cells (62). Rapamycin, an inducer of autophagy, reduces the aggregation of expanded polyQ in transfected cells (75), protects against neurodegeneration in a fly model of Huntington disease, and improves performance on behavioral tests and decreases aggregate formation in a mouse model of Huntington disease (76). These results suggest a possible role of autophagy in the turnover of expanded polyQ proteins and in protection of neurons against their toxicity.

### Autophagy and infectious diseases

The autophagic machinery is used to degrade intracellular pathogens (reviewed in refs. 77, 78) including intracellular bacteria (e.g., Shigella flexneri and Mycobacterium tuberculosis), mammalian viruses that produce encephalitis (e.g., alphaviruses and herpes simplex virus), and plant viruses (32, 78–81). It also may be used to degrade invading extracellular pathogens such as group A Streptococcus (82). It is reasonable to propose that autophagy might promote cellular survival during pathogen invasion because of either enhanced degradation of intracellular pathogens and consequent decreases in microbial replication; enhanced degradation of specific cytotoxic microbial virulence products; or preservation of cellular nutrient status during a period of microbial parasitism (which mimics nutrient starvation). However, with the exception of the finding that
Downloaded on June 10, 2014. **The Journal of Clinical Investigation**. More information at [www.jci.org/articles/view/26390](http://www.jci.org/articles/view/26390)

---

### review series

#### forced expression of the *beclin 1* autophagy gene protects against Sindbis virus–induced apoptosis in mouse brains (in parallel with decreasing viral replication) (79), direct proof of a cell-autonomous, pro-survival role of autophagy in pathogen infection is lacking. Moreover, it has been proposed that a virulence protein, SipB, of the intracellular pathogen *Salmonella enterica* causes macrophage death by inducing autophagy, perhaps by triggering mitochondrial fusion with autophagosomes (83). Yet, in this study, there was no direct evidence that macrophage cell death was caused by, rather than simply associated with, autophagy. Further studies in autophagy-deficient host organisms are required to determine the role of autophagy in life-and-death decisions during pathogen infection.

#### Autophagy and cancer

Cancer results from the dysregulation of pathways that regulate cell differentiation, cell proliferation, and cell survival. Autophagy may protect against cancer by sequestering damaged organelles, permitting cellular differentiation, increasing protein catabolism, and/or promoting autophagic death. Alternatively, autophagy may contribute to cancer by promoting the survival of nutrient-starved cells. Recent data are most consistent with a model in which autophagy contributes to tumor suppression and defects in autophagy contribute to oncogenesis. Biochemical evidence in mammalian cells and genetic evidence in *C. elegans* and *Drosophila* indicate that autophagy is positively regulated by the PTEN tumor suppressor gene and negatively regulated by the oncogenic class I PI3K signaling pathway (14, 34, 84, 85). Furthermore, the mammalian autophagy gene *beclin 1* has tumor suppressor activity in breast carcinoma cells (86), is commonly deleted in human breast ovarian and prostate cancer (87), and is a haploinsufficient tumor suppressor gene in mice (88, 89).

Several theories regarding the role of autophagy-dependent death and autophagy-dependent survival in cancer biology have been proposed (3, 66, 90–94). One is that autophagy-dependent death is a mechanism of tumor suppression. However, there are no direct data to support this hypothesis. In contrast, studies in cells and animals with a deficiency in *beclin 1* suggest that death induction may not be involved in the tumor suppressor function of this autophagy gene. *Beclin 1⁻/⁻* ES cells are not resistant to death triggered by UV irradiation or serum withdrawal, and *beclin 1⁻/⁻* null animals die early during embryogenesis with massive cell death (89). Moreover, in *beclin 1* heterozygous-deficient mice (with reduced tissue levels of autophagy), there is hyperproliferation of mammary epithelial cells during glandular morphogenesis and increased antigen-driven proliferation of B cells without decreased cell death (88). Together, these observations suggest that the role of the *beclin 1* autophagy gene in tumor suppression is related not to cell death induction, but rather to inhibition of cellular proliferation.

It is possible that autophagy is involved in the spontaneous or chemotherapy-induced death of existing tumor cells. Although the role of autophagy in cell death in apoptosis-competent cells is unclear, autophagy gene–dependent death in cells crippled in apoptosis (e.g., zVAD-treated cells; *bax⁻/⁻*, *bak⁻/⁻* cells) may have relevance for cancer biology and therapy, since human tumor cells frequently contain mutations that render them resistant to apoptosis. One prediction is that such cells have an increased dependency on autophagy pathways for self-destruction, and that the impact of decreased autophagy-dependent cell death on tumor progression may be greater in tumor cells that are resistant to apoptosis.

Another prediction is that the enhanced autophagy-dependent death potential of apoptosis-resistant tumor cells might be exploited therapeutically by the administration of autophagy-inducing agents. Indeed, there are examples of putative autophagic cell death in cancer cell lines treated with chemotherapeutic agents (Table 1). During tamoxifen-induced death of MCF7 cells (a cell type that contains a mutation in caspase-3), there is a marked upregulation of Beclin 1 autophagy protein expression (42, 90), and, in some examples, chemotherapy-induced autophagic cell death is inhibited by 3-MA (16, 42, 95). However, evidence proving that autophagy is a bona fide death pathway in chemotherapy-treated cancer cells is lacking. In addition, rapamycin, an inhibitor of TOR kinase that has promising antitumor effects in human clinical trials (96), is one of the most potent known inducers of autophagy but is not known to induce autophagic cell death.

In contrast to potential pro-death effects, more clearly established pro-survival effects of autophagy during nutrient starvation might foster tumor initiation and/or progression (3, 91, 93). As tumor cells grow beyond their blood supply, they are exposed to nutrient-limiting conditions, and it is possible that transformed cells use autophagy as a survival strategy in this setting. It has been proposed that such a need for autophagy in tumor initiation might explain the retention of the wild-type allele in all tumors arising in *beclin 1⁺/⁻* mice (92). However, the role of autophagy in tumor cell survival in vivo has not been tested experimentally. Moreover, in considering the net effect of autophagy on tumorigenesis, it is important to recognize its other functions that could contribute to restricting tumorigenesis (e.g., the degradation of certain proteins or organelles required for cell growth and/or the degradation of damaged mitochondria and other organelles that generate genotoxic stress and increase the likelihood of oncogenic mutations).

#### Conclusion

Autophagy functions across a diverse range of species as a pro-survival pathway during nutrient deprivation and other forms of cellular stress. Paradoxically, in cells that cannot die by apoptosis and, more speculatively, in cells that cannot be removed by engulfment cells, the autophagic machinery may also be used for self-destruction. The challenge for scientists will be to understand the molecular basis of this paradox. The challenge for clinicians will be to selectively turn on or turn off autophagy gene–dependent survival and death pathways in the treatment of different clinical diseases.

#### Acknowledgments

The work from the authors’ laboratories was supported by NIH grants RO1 CA084254, RO1 AI151367, and RO1 CA109618 (to B. Levine) and R37 NIA12859 (to J. Yuan); American Cancer Society grant RSG 98-339 (to B. Levine); and an Ellison Medical Foundation Senior Scholar Award in Infectious Diseases (to B. Levine). We thank Alexi Degetrev for providing electron micrographs and Sophie Pattingre and Renee Talley for help with manuscript preparation.

Address correspondence to: Beth Levine, Division of Infectious Diseases, University of Texas Southwestern Medical Center, 5323 Harry Hines Boulevard, Dallas, Texas 75390-9113, USA. Phone: (214) 648-0493; Fax: (214) 648-0284; E-mail: beth.levine@utsouthwestern.edu.

1. Klionsky, D.J., and Emr, S.D. 2000. Autophagy as a regulated pathway of cellular degradation. *Science*. **290**:1717–1721.
2. Levine, B., and Klionsky, D.J. 2004. Development by self-digestion: molecular mechanisms and biological functions of autophagy. *Dev. Cell*. **6**:463–477.
3. Lum, J.J., DeBarardinis, R.J., and Thompson, C.B. 2005. Autophagy in metazoans: cell survival in the land of plenty [review]. *Nat. Rev. Mol. Cell Biol*. **6**:439–448.
4. Talloczy, Z., et al. 2002. Regulation of starvation- and virus-induced autophagy by the eIF2α kinase signaling pathway. *Proc. Natl. Acad. Sci. U. S. A*. **99**:190–195.
5. Klionsky, D.J., et al. 2003. A unified nomenclature for yeast autophagy-related genes. *Dev. Cell*. **5**:539–545.
6. Schweichel, J.-U., and Merker, H.-J. 1973. The morphology of various types of cell death in prenatal tissues. *Teratology*. **7**:253–266.
7. Bursch, W. 2001. The autophagosomal-lysosomal compartment in programmed cell death. *Cell Death Differ*. **8**:569–581.
8. Clarke, P.G. 1990. Developmental cell death: morphological diversity and multiple mechanisms [review]. *Anat. Embryol. (Berl.)* **181**:195–213.
9. Lockshin, R.A., and Zakeri, Z. 2004. Apoptosis, autophagy, and more. *Int. J. Biochem. Cell Biol*. **36**:2405–2419.
10. Lee, C.Y., and Baehrecke, E.H. 2001. Steroid regulation of autophagic programmed cell death during development. *Development*. **128**:1443–1455.
11. Kosta, A., et al. 2004. Autophagy gene disruption reveals a non-vacuolar cell death pathway in dictyostelium. *J. Biol. Chem*. **279**:48404–48409.
12. Seglen, P.O., and Gordon, P.B. 1982. 3-Methyladenine: specific inhibitor of autophagic/lysosomal protein degradation in isolated rat hepatocytes. *Proc. Natl. Acad. Sci. U. S. A*. **79**:1889–1892.
13. Blommaart, E.F., Krause, U., Schellens, J.P., Vreeling-Sindelarova, H., and Meijer, A.J. 1997. The phosphatidylinositol 3-kinase inhibitors wortmannin and LY294002 inhibit autophagy in isolated rat hepatocytes. *Eur. J. Biochem*. **243**:240–246.
14. Petiot, A., Ogier-Denis, E., Blommaart, E.F., Meijer, A.J., and Codogno, P. 2000. Distinct classes of phosphatidylinositol 3'-kinases are involved in signaling pathways that control macroautophagy in HT-29 cells. *J. Biol. Chem*. **275**:992–998.
15. Schwarze, P.E., and Seglen, P.O. 1985. Reduced autophagic activity, improved protein balance and enhanced in vitro survival of hepatocytes isolated from carcinogen-treated rats. *Exp. Cell Res*. **157**:15–28.
16. Bursch, W., et al. 1996. Active cell death induced by the anti-estrogens tamoxifen and ICI 164 384 in human mammary carcinoma cells (MCF-7) in culture: the role of autophagy. *Carcinogenesis*. **17**:1595–1607.
17. Kaidi, A.U., et al. 2001. Chloroquine-induced neuronal cell death is p53 and Bcl-2 family-dependent but caspase-independent. *J. Neuropathol. Exp. Neurol*. **60**:937–945.
18. Xue, L., Fletcher, G.C., and Tolkovsky, A.M. 1999. Autophagy is activated by apoptotic signalling in sympathetic neurons: an alternative mechanism of death execution. *Mol. Cell. Neurosci*. **14**:180–198.
19. Canu, N., et al. 2005. Role of the autophagic-lysosomal system on low potassium-induced apoptosis in cultured cerebellar granule cells. *J. Neurochem*. **92**:1228–1242.
20. Uchiyama, Y. 2001. Autophagic cell death and its execution by lysosomal cathepsins. *Arch. Histol. Cytol*. **64**:233–246.
21. Jia, L., et al. 1997. Inhibition of autophagy abrogates tumour necrosis factor alpha induced apoptosis in human T-lymphoblastic leukaemic cells. *Br. J. Haematol*. **98**:673–685.
22. Xue, L., Borutaite, V., and Tolkovsky, A.M. 2002.

Inhibition of mitochondrial permeability transition and release of cytochrome c by anti-apoptotic nucleoside analogues. *Biochem. Pharmacol*. **64**:441–449.
23. Yu, L., et al. 2004. Regulation of an ATG7-beclin 1 program of autophagic cell death by caspase 8. *Science*. **304**:1500–1502.
24. Shimizu, S., et al. 2004. Role of Bcl-2 family proteins in a non-apoptotic programmed cell death dependent on autophagy genes. *Nat. Cell Biol*. **6**:1221–1228.
25. Lum, J.J., et al. 2005. Growth factor regulation of autophagy and cell survival in the absence of autophagy. *Cell*. **120**:237–248.
26. Hourdry, J. 1977. Cytological and cytochemical changes in the intestinal epithelium during anuran metamorphosis. *Int. Rev. Cytol. Suppl*. **5**:337–385.
27. Tsukada, M., and Ohsumi, Y. 1993. Isolation and characterization of autophagy-defective mutants of *Saccharomyces cerevisiae*. *FEBS Lett*. **333**:169–174.
28. Otto, G.P., Wu, M.Y., Kazgan, N., Anderson, O.R., and Kessin, R.H. 2003. Macroautophagy is required for multicellular development of the social amoeba *Dictyostelium discoideum*. *J. Biol. Chem*. **278**:17636–17645.
29. Otto, G.P., Wu, M.Y., Kazgan, N., Anderson, O.R., and Kessin, R.H. 2004. *Dictyostelium* macroautophagy mutants vary in the severity of their developmental defects. *J. Biol. Chem*. **279**:15621–15629.
30. Doelling, J.H., Walker, J.M., Friedman, E.M., Thompson, A.R., and Vierstra, R.D. 2002. The APG8/12-activating enzyme APG7 is required for proper nutrient recycling and senescence in *Arabidopsis thaliana*. *J. Biol. Chem*. **277**:33105–33114.
31. Hanaoka, H., et al. 2002. Leaf senescence and starvation-induced chlorosis are accelerated by the disruption of an *Arabidopsis* autophagy gene. *Plant Physiol*. **129**:1181–1193.
32. Liu, Y., et al. 2005. Autophagy genes are essential for limiting the spread of programmed cell death associated with plant innate immunity. *Cell*. **121**:567–577.
33. Kuma, A., et al. 2004. The role of autophagy during the early neonatal starvation period. *Nature*. **432**:1032–1036.
34. Melendez, A., et al. 2003. Autophagy genes are essential for dauer development and lifespan extension in *C. elegans*. *Science*. **301**:1387–1391.
35. Edinger, A.L., Cinnalli, R.M., and Thompson, C.B. 2003. Rab7 prevents growth factor-independent survival by inhibiting cell-autonomous nutrient transport expression. *Dev. Cell*. **5**:571–582.
36. Boya, P., et al. 2005. Inhibition of macroautophagy triggers apoptosis. *Mol. Cell. Biol*. **25**:1025–1040.
37. Mills, K.R., Reginato, M., Debnath, J., Queenan, B., and Brugge, J.S. 2004. Tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) is required for induction of autophagy during lumen formation in vitro. *Proc. Natl. Acad. Sci. U. S. A*. **101**:3438–3443.
38. Thorburn, J., et al. 2005. Selective inactivation of FADD-dependent apoptosis and autophagy pathway in immortal epithelial cells. *Mol. Biol. Cell*. **16**:1189–1199.
39. Prins, J., et al. 1998. Tumour necrosis factor induced autophagy and mitochondrial morphological abnormalities are mediated by TNFR-1 and/or TNFR-II and do not invariably lead to cell death. *Biochem. Soc. Trans*. **26**:S314.
40. Pyo, J.-O., et al. 2005. Essential roles of Atg5 and FADD in autophagic cell death: dissection of autophagic cell death into vacuole formation and cell death. *J. Biol. Chem*. **280**:20722–20729.
41. Inbal, B., Bialik, S., Sabanay, I., Shani, G., and Kimchi, A. 2002. DAP kinase and DRP-1 mediate membrane blebbing and the formation of autophagic vesicles during programmed cell death. *J. Cell Biol*. **157**:455–468.
42. Scarlatti, F., et al. 2004. Ceramide-mediated macroautophagy involves inhibition of protein kinase B and upregulation of Beclin 1. *J. Biol. Chem*.

43. Daido, S., et al. 2004. Pivotal role of the cell death factor BNIP3 in ceramide-induced autophagic cell death in malignant glioma cells. *Cancer Res*. **64**:4286–4293.
44. Meijer, A.J., and Codogno, P. 2004. Regulation and role of autophagy in mammalian cells. *Int. J. Biochem. Cell Biol*. **36**:2445–2462.
45. Lee, C.-Y., et al. 2003. Genome-wide analyses of steroid- and radiation-triggered programmed cell death in *Drosophila*. *Curr. Biol*. **13**:350–357.
46. Gorski, S.M., et al. 2003. A SAGE approach to discovery of genes involved in autophagic cell death. *Curr. Biol*. **13**:358–363.
47. Kissova, I., Deffieu, M., Manon, S., and Camougrand, N. 2004. Uth1p is involved in the autophagic degradation of mitochondria. *J. Biol. Chem*. **279**:39068–39074.
48. Camougrand, N., et al. 2003. The product of the *UTH1* gene, required for Bax-induced cell death in yeast, is involved in the response to rapamycin. *Mol. Microbiol*. **47**:495–506.
49. Saeki, K., et al. 2000. Bcl-2 down-regulation causes autophagy in caspase-independent manner in human leukemic HL60 cells. *Cell Death Differ*. **7**:1263–1269.
50. Cardenas-Aguayo, M.D.C., Santa-Olalla, J., Baizabal, J.M., Salgado, L.M., and Covarrubias, L. 2003. Growth factor deprivation induces an alternative non-apoptotic death mechanism that is inhibited by Bcl2 in cells derived from neural precursor cells. *J. Hematother. Stem Cell Res*. **12**:735–748.
51. Pattingre, S., et al. 2005. Bcl-2 antiapoptotic proteins inhibit Beclin 1-dependent autophagy. *Cell*. In press.
52. Velde, C.V., et al. 2000. BNIP3 and genetic control of necrosis-like cell death through the mitochondrial permeability transition pore. *Mol. Cell*. **20**:5454–5468.
53. Yanagisawa, H., Miyashita, T., Nakano, Y., and Yamamoto, D. 2003. HSpin1, a transmembrane protein interacting with Bc-2/Bcl-xL, induces a caspase-independent autophagic cell death. *Cell Death Differ*. **10**:798–807.
54. Nakano, Y., et al. 2001. Mutations in the novel membrane protein spinster interfere with programmed cell death and cause neural degeneration in *Drosophila melanogaster*. *Mol. Cell. Biol*. **21**:3775–3788.
55. Elmore, S.P., Qian, T., Grissom, S.F., and Lemasters, J.J. 2001. The mitochondrial permeability transition initiates autophagy in rat hepatocytes. *FASEB J*. **15**:2286–2287.
56. Jacotot, E., et al. 1999. Mitochondrial membrane permeabilization during the apoptotic process. *Ann. N. Y. Acad. Sci*. **887**:18–30.
57. Marzo, I., et al. 1998. Bax and adenine nucleotide translocator cooperate in the mitochondrial control of apoptosis. *Science*. **281**:2027–2031.
58. Li, H., Zhu, H., Xu, C.J., and Yuan, J. 1998. Cleavage of BID by caspase 8 mediates the mitochondrial damage in the Fas pathway of apoptosis. *Cell*. **94**:491–501.
59. Tolkovsky, A.M., Xue, L., Fletcher, G.C., and Borutaite, V. 2002. Mitochondrial disappearance from cells: a clue to the role of autophagy in programmed cell death and disease [review]? *Biochimie*. **84**:233–240.
60. Xue, L., Fletcher, G.C., and Tolkovsky, A.M. 2001. Mitochondria are selectively eliminated from eukaryotic cells after blockade of caspases during apoptosis. *Curr. Biol*. **11**:361–365.
61. Grune, T., Jung, T., Merker, K., and Davies, K.J. 2004. Decreased proteolysis caused by protein aggregates, inclusion bodies, plaques, lipofuscin, ceroid, and “aggresomes” during oxidative stress, aging, and disease. *Int. J. Biochem. Cell Biol*. **36**:2519–2530.
62. Qin, Z.H., et al. 2003. Autophagy regulates the processing of amino terminal huntingtin fragments. *Hum. Mol. Genet*. **12**:3231–3244.

63. Anglade, P., et al. 1997. Apoptosis and autophagy in nigral neurons of patients with Parkinson’s disease. *Histol. Histopathol.* **12**:25–31.

64. Yu, W.H., et al. 2004. Autophagic vacuoles are enriched in amyloid precursor protein-secretase activities: implications for beta-amyloid peptide over-production and localization in Alzheimer’s disease. *Int. J. Biochem. Cell Biol.* **36**:2531–2540.

65. Liberski, P.P., Sikorska, B., Bratosiewicz-Wasik, J., Gajdusek, D.C., and Brown, P. 2004. Neuronal cell death in transmissible spongiform encephalopathies (prion diseases) revisited: from apoptosis to autophagy. *Int. J. Biochem. Cell Biol.* **36**:2473–2490.

66. Shintani, T., and Klionsky, D.J. 2004. Autophagy in health and disease: a double-edged sword. *Science.* **306**:990–995.

67. Yue, Z., et al. 2002. A novel protein complex linking the δ2 glutamate receptor and autophagy: implications for neurodegeneration in lurcher mice. *Neuron.* **35**:921–933.

68. Maries, E., Dass, B., Collier, T.J., Kordower, J.H., and Steece-Collier, K. 2003. The role of alpha-synuclein in Parkinson’s diseases: insights from animal models. *Nat. Rev. Neurosci.* **4**:727–738.

69. Webb, J.L., Ravikumar, B., Atkins, J., Skepper, J.N., and Rubinsztein, D.C. 2003. Alpha-synuclein is degraded by both autophagy and the proteasome. *J. Biol. Chem.* **278**:25009–25013.

70. Cuervo, A.M., Stefanis, L., Fredenburg, R., Lansbury, P.T., and Sulzer, D. 2004. Impaired degradation of mutant α-synuclein by chaperone-mediated autophagy. *Science.* **305**:1292–1295.

71. Polymeropoulos, M.H., et al. 1997. Mutation in the α-synuclein gene identified in families with Parkinson’s disease. *Science.* **276**:2045–2047.

72. Kruger, R., et al. 1998. Ala30Pro mutation in the gene encoding alpha-synuclein in Parkinson’s disease. *Nat. Genet.* **18**:106–108.

73. DiFiglia, M., et al. 1997. Aggregation of huntingtin in neuronal intranuclear inclusions and dystrophic neurites in brain. *Science.* **277**:1990–1993.

74. Venkatraman, P., Wetzel, R., Tanaka, M., Nukina, N., and Goldberg, A.L. 2004. Eukaryotic proteasomes cannot digest polyglutamine sequences and release them during degradation of polyglutamine-containing proteins. *Mol. Cell.* **14**:95–104.

75. Ravikumar, B., Duden, R., and Rubinsztein, D.C. 2002. Aggregate-prone proteins with polyglutamine and polyalanine expansions are degraded by autophagy. *Hum. Mol. Genet.* **11**:1107–1117.

76. Ravikumar, B., et al. 2004. Inhibition of mTOR induces autophagy and reduces toxicity of polyglutamine expansions in fly and mouse models of Huntington disease. *Nat. Genet.* **36**:585–595.

77. Kirkegaard, K., Taylor, M.P., and Jackson, W.T. 2004. Cellular autophagy: surrender, avoidance and subversion by microorganisms [review]. *Nat. Rev. Microbiol.* **2**:301–314.

78. Seay, M., Dinesh-Kumar, S., and Levine, B. 2005. Digesting oneself and digesting microbes: autophagy as a host response to viral infection. In *Modulation of host gene expression and innate immunity by viruses*. P. Palese, editor. Springer. Dordrecht, The Netherlands. 245–279.

79. Liang, X.H., et al. 1998. Protection against fatal Sindbis virus encephalitis by Beclin, a novel Bcl-2-interacting protein. *J. Virol.* **72**:8586–8596.

80. Ogawa, M., et al. 2005. Escape of intracellular Shigella from autophagy. *Science.* **307**:727–731.

81. Gutierrez, M.G., et al. 2004. Autophagy is a defense mechanism inhibiting BCG and *Mycobacterium tuberculosis* survival in infected macrophages. *Cell.* **119**:753–766.

82. Nakagawa, I., et al. 2004. Autophagy defends cells against invading group A Streptococcus. *Science.* **306**:1037–1040.

83. Hernandez, L.D., Pypaert, M., Flavell, R.A., and Galan, J.E. 2003. A Salmonella protein causes macrophage cell death by inducing autophagy. *J. Cell Biol.* **163**:1123–1131.

84. Rusten, T.E., et al. 2004. Programmed autophagy in the *Drosophila* fat body is induced by ecdysone through regulation of the PI3K pathway. *Dev. Cell.* **7**:179–192.

85. Scott, R.C., Schuldiner, O., and Neufeld, T.P. 2004. Role and regulation of starvation-induced autophagy in the *Drosophila* fat body. *Dev. Cell.* **7**:167–178.

86. Liang, X.H., et al. 1999. Induction of autophagy and inhibition of tumorigenesis by beclin 1. *Nature.* **402**:672–676.

87. Aita, V.M., et al. 1999. Cloning and genomic organization of *beclin 1*, a candidate tumor suppressor gene on chromosome 17q21. *Genomics.* **59**:59–65.

88. Qu, X., et al. 2003. Promotion of tumorigenesis by heterozygous disruption of the *beclin 1* gene. *J. Clin. Invest.* **112**:1809–1820. doi:10.1172/JCI200320039.

89. Yue, Z., Jin, S., Yang, C., Levine, A.J., and Heintz, N. 2003. Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor. *Proc. Natl. Acad. Sci. U.S.A.* **100**:15077–15082.

90. Furuya, N., Liang, X.H., and Levine, B. 2004. Autophagy and cancer. In *Autophagy*. D.J. Klionsky, editor. Landes Bioscience. Georgetown, Texas, USA. 244–253.

91. Ogier-Denis, E., and Codogno, P. 2003. Autophagy: a barrier or an adaptive response to cancer? *Biochim. Biophys. Acta.* **1603**:113–128.

92. Edinger, A.L., and Thompson, C.B. 2004. Death by design: apoptosis, necrosis and autophagy. *Curr. Opin. Cell Biol.* **16**:663–669.

93. Gozuacik, D., and Kimchi, A. 2004. Autophagy as a cell death and tumor suppressor mechanism. *Oncogene.* **23**:2891–2906.

94. Nelson, D.A., and White, E. 2004. Exploiting different ways to die. *Genes Dev.* **18**:1223–1226.

95. Kanzawa, T., Kondo, Y., Ito, H., Kondo, S., and Germano, I. 2003. Induction of autophagic cell death in malignant glioma cells by arsenic trioxide. *Cancer Res.* **63**:2103–2108.

96. Huang, S., and Houghton, P.J. 2003. Targeting mTOR signaling for cancer therapy. *Curr. Opin. Pharmacol.* **3**:371–377.

97. Codogno, P., and Meijer, A.J. 2004. Signaling pathways in mammalian autophagy. In *Autophagy*. D.J. Klionsky, editor. Landes Bioscience. Georgetown, Texas, USA. 26–47.

98. Degterev, A., et al. 2001. Identification of small-molecule inhibitors of interaction between the BH3 domain and BclxL. *Nat. Cell Biol.* **3**:173–182.

99. Kinch, G., Hoffman, K.L., Rodrigues, E.M., Zee, M.C., and Weeks, J.C. 2003. Steroid-triggered programmed cell death of a motoneuron is autophagic and involves structural changes in mitochondria. *J. Comp. Neurol.* **457**:384–403.

100. Muller, F., Adori, C., and Sass, M. 2004. Autophagic and apoptotic features during programmed cell death in the fat body of the tobacco hornworm (*Manduca sexta*). *Eur. J. Cell Biol.* **83**:67–78.

101. Romer, F., and Martau, T. 1998. Degeneration of mouling glands in male crickets. *J. Insect Physiol.* **44**:981–989.

102. Bowen, I.D., Mullarkey, K., and Morgan, S.M. 1996. Programmed cell death during metamorphosis in the blow-fly *Calliphora vomitoria*. *Microsc. Res. Tech.* **34**:202–217.

103. Jones, H.E., and Bown, I.D. 1993. Acid phosphatase activity in the larval salivary glands of developing *Drosophila melanogaster*. *Cell Biol. Int.* **17**:305–315.

104. Lee, C.-Y., Cooksey, B.A.K., and Baehrecke, E.H. 2002. Steroid regulation of midgut cell death during *Drosophila* development. *Dev. Biol.* **250**:101–111.

105. Butterworth, F.M., and LaTendresse, B.L. 1973. Quantitative studies of cytochemical and cytological changes during cell death in the larval fat body of *Drosophila melanogaster*. *J. Insect. Physiol.* **19**:1487–1500.

106. Debnath, J., et al. 2002. The role of apoptosis in creating and maintaining luminal space within normal and oncogene-expressing mammary acini. *Cell.* **111**:29–40.

107. Kegel, K.B., et al. 2000. Huntingtin expression stimulates endolysosomal-activity, endosome tubulation, and autophagy. *J. Neurosci.* **20**:7268–7278.

108. Stefanis, L., Larsen, K.E., Rideout, H.J., Sulzer, D., and Greene, L.A. 2001. Expression of A53T mutant but not wild-type α-synuclein in PC12 cells induces alterations of the ubiquitin-dependent degradation, loss of dopamine release, and autophagic cell death. *J. Neurosci.* **21**:9549–9560.

109. Nitatori, T., et al. 1995. Delayed neuronal death in the CA1 pyramidal cell layer of the gerbil hippocampus following transient ischemia is apoptosis. *J. Neurosci.* **17**:1001–1011.

110. Kostin, S. 2004. Types and mechanisms of myocyte cell death in diseased human heart. *Cardiovasc. J. S. Afr.* **15**(4 Suppl. 1):S1.

111. Hein, S., et al. 2003. Progression from compensated hypertrophy to failure in the pressure-overloaded human heart: structural deterioration and compensatory mechanisms. *Circulation.* **107**:984–991.

112. Bera, A., Singh, S., Nagaraj, R., and Vaidya, T. 2003. Induction of autophagic cell death in *Leishmania donovani* by antimicrobial peptides. *Mol. Biochem. Parasitol.* **127**:23–35.

113. Christensen, S.T., et al. 1998. Staurosporine-induced cell death in *Tetrahymena thermophila* has mixed characteristics of both apoptotic and autophagic degeneration. *Cell Biol. Int.* **22**:591–598.

114. Cornillon, S., et al. 1994. Programmed cell death in *Dictyostelium*. *J. Cell Sci.* **107**:2691–2704.

115. Akazawa, H., et al. 2004. Diphtheria toxin-induced autophagic cardiomyocyte death plays a pathogenic role in mouse model of heart failure. *J. Biol. Chem.* **279**:41095–41103.

116. Guimaraes, C.A., Benchimo, M., Amarante-Mendes, G.P., and Linden, R. 2003. Alternative programs of cell death in developing retinal tissue. *J. Biol. Chem.* **278**:41938–41946.

117. Borsello, T., Croquelois, K., Hornung, J.P., and Clarke, P.G.H. 2003. N-methyl-D-aspartate triggered neuronal death in organotypic hippocampal cultures is endocytic, autophagic and mediated by the c-Jun N-terminal kinase pathway. *Eur. J. Neurosci.* **18**:473–485.

118. Ohsawa, Y., et al. 1998. An ultrastructural and immunohistochemical study of PC12 cells during apoptosis induced by serum deprivation with special reference to autophagy and lysosomal cathepsins. *Arch. Histol. Cytol.* **61**:395–403.

119. von Bultingslowen, I., Jontell, M., Hurst, P., Nanmark, U., and Kardos, T. 2001. 5-Fluorouracil induces autophagic degeneration in rat oral keratinocytes. *Oral Oncol.* **37**:537–544.

120. Opipari, A.W., Jr., et al. 2004. Resveratrol-induced autophagocytosis in ovarian cancer cells. *Cancer Res.* **64**:696–703.

121. Bursch, W., et al. 2000. Autophagic and apoptotic types of programmed cell death exhibit different fates of cytoskeletal filaments. *J. Cell Sci.* **113**:1189–1198.

122. Kanzawa, T., et al. 2005. Arsenic trioxide induces autophagic cell death in malignant glioma cells by upregulation of mitochondrial cell death protein BNIP3. *Oncogene.* **24**:980–991.

123. Chau, Y.-P., Lin, S.-Y., Chen, J.H.-C., and Tai, M.-H. 2003. Endostatin induced autophagic cell death in EAhy926 human endothelial cells. *Histol. Pathol.* **18**:715–726.

124. Juhasz, G., Csikos, G., Sinka, R., Erdelyi, M., and Sass, M. 2003. The *Drosophila* homolog of Atg1 is essential for autophagy and development. *FEBS Lett.* **543**:154–158.
