
PERSPECTIVE

SLIM Trims STATs: Ubiquitin E3 Ligases Provide Insights for Specificity in the Regulation of Cytokine Signaling

Daniela Ungureanu¹ and Olli Silvennoinen¹,²*

(Published 4 October 2005)

Cellular homeostasis requires the precise coordination of proliferation, apoptosis, and differentiation, and the cytokines that bind to type I and type II transmembrane receptors play a critical role in this regulation (1, 2). The cytokine family consists of over 50 polypeptides that mediate highly specific functions ranging from blood formation, embryogenesis, and the regulation of metabolism to the orchestration of inflammatory and immune responses; thus, the mechanisms and regulation of cytokine actions are of wide biological and clinical interest (3, 4).

The principal signaling route of cytokines was discovered over a decade ago, and its basic mechanisms are currently well understood (5, 6). Ligand-induced receptor dimerization activates receptor-associated tyrosine kinases, the Janus kinases (JAKs), and initiates tyrosine phosphorylation of intracellular signaling proteins, including latent cytoplasmic transcription factors: the signal transducers and activators of transcription (STATs). The STAT family consists of seven mammalian members, which are critical mediators of functional responses and specificity in cytokine signaling. At the receptor complex, STATs become phosphorylated on a conserved tyrosine residue, which induces their dimerization, nuclear translocation, and DNA binding, and leads to the induction of cytokine-responsive genes (2, 7). However, appropriate biological responses require stringent regulation of the activation signals, and cytokine signaling is controlled by several regulatory proteins, such as SOCS (suppressor of cytokine signaling), PIAS (protein inhibitor of activated STAT), and protein tyrosine phosphatases (PTPs) (8). These regulatory proteins have evolved important functions in modulating various posttranslational modifications, such as phosphorylation by PTPs, sumoylation by PIAS proteins, and ubiquitination by SOCS proteins. Indeed, ubiquitin-mediated proteasomal degradation has been implicated in the regulation of the abundance of cytokine receptors at the cell surface, the activation of JAKs, and the turnover of STATs (9).

Ubiquitination is an efficient mechanism for targeting proteins for degradation and is involved in the regulation of numerous cellular events, ranging from progression through the cell cycle, transcription, and DNA repair, to antigen presentation and cell death by apoptosis (10). The ubiquitin-proteasomal pathway involves three steps: The ubiquitin residues are first activated by ubiquitin-activating enzyme (E1); they are then transferred to ubiquitin-conjugating enzyme (Ubc) (E2); and finally, ubiquitin ligase (E3) catalyzes the transfer of ubiquitin to the ε-amino group of lysine residues in the substrate protein. The polyubiquinated proteins are recognized and degraded by the 26S proteasomes in an adenosine triphosphate–dependent manner. Other types of ubiquitin modifications, such as monoubiquitination or multiubiquitination, do not result in protein degradation but instead alter the activity of the modified protein (11).

The key determinants of specificity are the E3 ligases, which represent the largest and most diverse group of enzymes in the ubiquitination process. Ubiquitin ligases are composed of several functional domains: A protein-protein interaction domain captures substrates for degradation, a recognition site recruits ubiquitin-bound E2 protein, and a functional ubiquitin ligase domain mediates the ubiquitin transfer from E2 to the substrate. The signature motifs of E3 ligases are the ubiquitin ligase domains, and the family can be divided into two main types: HECT (homologous to E6-AP C terminus)–domain and RING (really interesting new gene)–domain ligases. More recently, RING-like domain-containing E3 ligases have been identified, such as the PIAS family of SUMO ligases, U box E3s, and plant homeodomain (PHD) E3s (12).

Information about the role of ubiquitination in cytokine signaling has emerged. Initially, proteasome inhibitors were shown to prolong the activation of the JAK/STAT pathway and regulate the turnover of several cytokine receptors (13). Functional studies indicate that erythropoietin receptor internalization and degradation require both proteasomes and lysosomes, and the kinase activity of JAK2 is required for lysosomal targeting and degradation of the receptor but not for proteasome-mediated regulation (14).

The initial link between JAKs and ubiquitination was provided by the finding that the conserved domain in SOCS proteins, the SOCS box, interacts with elongins B and C, which form a part of a ubiquitin ligase complex (15, 16). Activation of JAK2, either by cytokine stimulation or by oncogenic TEL-JAK2 fusion, induces polyubiquitination and degradation of JAK2. SOCS-1 interacts with the activated form of JAK2 and enhances proteasome-mediated degradation of JAK2 in a SOCS box–dependent manner (17) (Fig. 1). Furthermore, the SOCS box is required to rescue the full suppressive effect in SOCS1⁻/⁻ mice and TEL-JAK2–transformed cells (16, 18, 19). Thus, the ubiquitin pathway appears to serve as the final fail-proof mechanism to control cytokine signaling. The SOCS-1–elongins B/C complex is likely to also contain RING-finger protein Rbx1 and a scaffold protein, Cullin-2. SOCS-1 governs the specificity and substrate recognition of the complex. Interestingly, JAK2 is also monoubiquitinated in unstimulated cells, but the functional role and regulation of this modification are currently unknown (17).

The results on JAKs provide an example of cross-talk between various posttranslational modifications. Phosphorylation of the activation loop Tyr¹⁰⁰⁷ on JAK2 is critical for the interaction with SOCS-1 and subsequent ubiquitination and degradation.

---

¹Institute of Medical Technology, University of Tampere, 33014, Tampere, Finland. ²Department of Clinical Microbiology, Tampere University Hospital, Tampere, Finland.

Corresponding author. Telephone, 358-3-3551-7845; fax, 358-3-3551-7332; e-mail, olli.silvennoinen@uta.fi

Science’s stke www.stke.org/cgi/content/full/sigtrans;2005/304/pe49 Page 1


This mechanism is analogous to the c-Cbl–mediated desensitization of epidermal growth factor receptor (EGFR), in which phosphorylated Tyr<sup>1045</sup> on EGFR serves as a docking site for the RING-finger protein c-Cbl, which regulates the ubiquitination of the receptor (20). The JAK2–SOCS-1 interaction does not involve direct tyrosine phosphorylation of SOCS-1, but the function of SOCS-1 appears to be regulated by other posttranslational modifications. Ser/Thr phosphorylation of SOCS-1 on its N terminus by the kinase Pim results in stabilization of SOCS-1 and disrupts the interaction with elongins B and C (21, 22). In accordance, SOCS-1 protein is reduced in *Pim-1*<sup>-/-</sup> and *Pim-2*<sup>-/-</sup> mice. Also, v-Abl signaling impairs SOCS-1 function in a similar fashion by indirectly inducing SOCS-1 phosphorylation on Ser/Thr residues. However, various SOCS have adopted different regulatory mechanisms. For example, the turnover of SOCS-3 is regulated through ubiquitination of an N-terminal Lys<sup>6</sup> residue and proteasomal degradation (23), and tyrosine phosphorylation decreases the half-life of SOCS-3 by disrupting the interaction with elongin C (24).

Ubiquitination is also implicated in the regulation of STATs. The first report from Kim and Maniatis showed ubiquitination of STAT1 (25), but the underlying molecular mechanisms remained unknown. Proteasome inhibitors were also shown to stabilize the tyrosine-phosphorylated forms of STAT4, STAT5, and STAT6 (26). However, recent findings have clarified the role of ubiquitination in the regulation of STATs and provided insights into the specificity of this modification.

Paramyxoviruses efficiently use the ubiquitin pathway as a mechanism to escape the antiviral activities of interferons (27). Paramyxoviruses are RNA viruses and include the human pathogens mumps, measles, and Nipah viruses. Their host evasion mechanisms are largely attributed to the V proteins. At least three V proteins function as E3 ligases with high specificity for STAT1, STAT2, or STAT3. The V proteins form a multisubunit E3 enzyme complex that shows homology to the SOCS-E3 complex. The current model of the E3 ligase suggests a complex that contains the V protein and the cellular V-interaction proteins (VIPs), as well as additional proteins, including DDB1 (an ultraviolet-damaged DNA binding protein) and members of the Cullin family, especially Cullin 4A (Cul4A). The E3 complex is termed VDC (for V/DDB1/CUL4A or V-dependent degradation complex). VDC mediates STAT degradation using a combination of virus-encoded and cell-derived factors, where VIPs are the core degradation complex and the other cellular cofactors are responsible for the variations in V-protein target specificities.

The most recent piece in the ubiquitin-specificity puzzle comes from the Grusby laboratory, which has identified a STAT4-interacting nuclear protein, SLIM (STAT-interacting LIM protein), from a Th1 cell library (28). SLIM contains a PDZ domain and a LIM domain and interacts in the nucleus with tyrosine-phosphorylated STAT molecules and specifically inhibits gene expression mediated by either STAT4 or STAT1. The LIM domain forms a zinc-finger structure related to the RING finger and PHD structures, thus suggesting E3 ligase activity for SLIM. As expected, SLIM promoted both the ubiquitination and degradation of STAT1 and STAT4. SLIM also impaired the tyrosine phosphorylation of STAT4, and this function of SLIM was independent of its role in promoting the proteasome-dependent degradation of STAT. These results suggest that SLIM may have an additional function as an adaptor molecule to recruit nuclear tyrosine phosphatases to dephosphorylate STATs. Alternatively, the ubiquitination of STATs may induce dephosphorylation through conformational changes that would allow interaction with nuclear phosphatases. The concept of SLIM as a regulator of STAT signaling and protein turnover is supported by the data from *SLIM*-deficient mice, which show enhanced STAT1 and STAT4 protein abundance and increased production of interferon-γ by Th1 cells. These findings suggest that SLIM is the first known cellular ubiquitin ligase with specificity toward STAT proteins, and they open a new chapter for investigation of STAT-specific degradation and the quest for additional LIM proteins.

It is becoming evident that ubiquitin-mediated proteosomal degradation plays an important role in the regulation of cytokine signaling at multiple points, from the receptor to the nucleus (Fig. 1). The first insights into the specificity and complexity of this regulation have been gained. This information will stimulate the field and further investigation, and it is also providing the framework for novel therapeutic applications targeting the cytokine signaling system.

![Figure 1](https://www.stke.org/cgi/content/full/sigtrans;2005/304/pe49/Figure_1)

Fig. 1. Schematic model of JAK/STAT regulation through the ubiquitin-proteasome pathway. Ligand binding induces receptor tyrosine phosphorylation by tyrosine kinases of the JAK family, producing a docking site for the STAT family of transcription factors. Phosphorylated STAT dimers translocate to the nucleus to initiate the transcription of target genes. At the receptor complex, activated JAKs interact with SOCS proteins and become polyubiquitinated and degraded in a SOCS box–dependent manner mediated by the elongin B/C (EloB and EloC)–Cul2–Rbx1 E3 ligase complex. Rubulaviruses can target STATs for ubiquitin-mediated proteasomal degradation using a VDC E3 ligase complex that requires the participation of V protein, VIPs, DDB1, and Cul4A. In the nucleus, STAT interaction with SLIM E3 ligase can result in STAT ubiquitination and degradation or dephosphorylation.

References

1. W. J. Leonard, J. J. O'Shea, Jaks and STATs: Biological implications. *Annu. Rev. Immunol.* **16**, 293–322 (1998).
2. J. N. Ihle, The Stat family in cytokine signaling. *Curr. Opin. Cell Biol.* **13**, 211–217 (2001).
3. C. Schindler, I. Strehlow, Cytokines and STAT signaling. *Adv. Pharmacol.* **47**, 113–174 (2000).
4. C. W. Schindler, Series introduction. JAK-STAT signaling in human disease. *J. Clin. Invest.* **109**, 1133–1137 (2002).
5. C. M. Horvath, J. E. Darnell, The state of the STATs: Recent developments in the study of signal transduction to the nucleus. *Curr. Opin. Cell Biol.* **9**, 233–239 (1997).
6. J. N. Ihle, Cytokine receptor signalling. *Nature* **377**, 591–594 (1995).
7. W. J. Leonard, Role of Jak kinases and STATs in cytokine signal transduction. *Int. J. Hematol.* **73**, 271–277 (2001).
8. K. Shuai, B. Liu, Regulation of JAK-STAT signalling in the immune system. *Nat. Rev. Immunol.* **3**, 900–911 (2003).
9. J. A. Johnston, Are SOCS suppressors, regulators, and degraders? *J. Leukoc. Biol.* **75**, 743–748 (2004).
10. A. Hershko, A. Ciechanover, The ubiquitin system. *Annu. Rev. Biochem.* **67**, 425–479 (1998).
11. L. Hicke, Protein regulation by monoubiquitin. *Nat. Rev. Mol. Cell Biol.* **2**, 195–201 (2001).
12. Y. C. Liu, Ubiquitin ligases and the immune response. *Annu. Rev. Immunol.* **22**, 81–127 (2004).
13. F. Verdier, P. Walrafen, N. Hubert, S. Chrétien, S. Gisselbrecht, C. Lacombe, P. Mayeux, Proteasomes regulate the duration of erythropoietin receptor activation by controlling down-regulation of cell surface receptors. *J. Biol. Chem.* **275**, 18375–18381 (2000).
14. P. Walrafen, F. Verdier, Z. Kadri, S. Chretien, C. Lacombe, P. Mayeux, Both proteasomes and lysosomes degrade the activated erythropoietin receptor. *Blood* **105**, 600–608 (2005).
15. T. Kamura, S. Sato, D. Haque, L. Liu, W. G. Kaelin Jr., R. C. Conaway, J. W. Conaway, The elongin BC complex interacts with the conserved SOCS-box motif present in members of the SOCS, Ras, WD-40 repeat, and ankyrin repeat families. *Genes Dev.* **12**, 3872–3881 (1998).
16. J. G. Zhang, D. Metcalf, S. Rakar, M. Asimakis, C. J. Greenhalgh, T. A. Williams, R. Starr, S. E. Nicholson, W. Carter, W. S. Alexander, D. J. Hilton, N. A. Nicol, The SOCS box of suppressor of cytokine signaling-1 is important for inhibition of cytokine action in vivo. *Proc. Natl. Acad. Sci. U.S.A.* **98**, 13261–13265 (2001).
17. D. Ungureanu, P. Saharinen, I. Juntila, D. J. Hilton, O. Silvennoinen, Regulation of Jak2 through the ubiquitin-proteasome pathway involves phosphorylation of Jak2 on Y1007 and interaction with SOCS-1. *Mol. Cell. Biol.* **22**, 3316–3326 (2002).

18. S. Kamizono, T. Hanada, H. Yasukawa, S. Minoguchi, R. Kato, M. Minoguchi, K. Hattori, S. Hatakeyama, M. Yada, S. Morita, T. Kitamura, H. Kato, K. Nakayama, A. Yoshimura, The SOCS box of SOCS-1 accelerates ubiquitin-dependent proteolysis of TEL-JAK2. *J. Biol. Chem.* **276**, 12530–12538 (2001).
19. J. Frantsve, J. Schwalller, D. W. Sternberg, J. Kutok, D. G. Gilliland, Socs-1 inhibits TEL-JAK2-mediated transformation of hematopoietic cells through inhibition of JAK2 kinase activity and induction of proteasome-mediated degradation. *Mol. Cell. Biol.* **21**, 3547–3557 (2001).
20. G. Levkowitz, H. Waterman, S. A. Ettenberg, M. Katz, A. Y. Tsygankov, I. Alroy, S. Lavi, K. Iwai, Y. Reiss, A. Ciechanover, S. Lipkowitz, Y. Yarden, Ubiquitin ligase activity and tyrosine phosphorylation underlie suppression of growth factor signaling by c-Cbl/Sli-1. *Mol. Cell* **4**, 1029–1040 (1999).
21. X. P. Chen, J. A. Losman, S. Cowan, E. Donahue, S. Fay, B. Q. Vuong, M. C. Nawijn, D. Capece, V. L. Cohan, P. Rothman, Pim serine/threonine kinases regulate the stability of Socs-1 protein. *Proc. Natl. Acad. Sci. U.S.A.* **99**, 2175–2180 (2002).
22. K. J. Peltola, K. Paukkunen, T. L. Aho, M. Ruuska, O. Silvennoinen, P. J. Koskinen, Pim-1 kinase inhibits STAT5-dependent transcription via its interactions with SOCS1 and SOCS3. *Blood* **103**, 3744–3750 (2004).
23. A. Sasaki, H. Yasukawa, A. Suzuki, S. Kamizono, T. Syoda, I. Kinjyo, M. Sasaki, J. A. Johnston, A. Yoshimura, Cytokine-inducible SH2 protein-3 (CIS3/SOCS3) inhibits Janus tyrosine kinase by binding through the N-terminal kinase inhibitory region as well as SH2 domain. *Genes Cells* **4**, 339–351 (1999).
24. S. Haan, P. Ferguson, U. Sommer, M. Hiremath, D. W. McVicar, P. C. Heinrich, J. A. Johnston, N. A. Cacalano, Tyrosine phosphorylation disrupts elongin interaction and accelerates SOCS3 degradation. *J. Biol. Chem.* **278**, 31972–31979 (2003).
25. T. K. Kim, T. Maniatis, Regulation of interferon-gamma-activated STAT1 by the ubiquitin-proteasome pathway. *Science* **273**, 1717–1719 (1996).
26. D. Wang, R. Moriggl, D. Stravopodis, N. Carpino, J. C. Marine, S. Teglund, J. Feng, J. N. Ihle, A small amphipathic alpha-helical region is required for transcriptional activities and proteasome-dependent turnover of the tyrosine-phosphorylated Stat5. *EMBO J.* **19**, 392–399 (2000).
27. C. M. Horvath, Silencing STATs: Lessons from paramyxovirus interferon evasion. *Cytokine Growth Factor Rev.* **15**, 117–127 (2004).
28. T. Tanaka, M. A. Soriano, M. J. Grusby, SLIM is a nuclear ubiquitin E3 ligase that negatively regulates STAT signaling. *Immunity* **22**, 729–736 (2005).

Citation: D. Ungureanu, O. Silvennoinen, SLIM trims STATs: Ubiquitin E3 ligases provide insights for specificity in the regulation of cytokine signaling. *Sci. STKE* **2005**, pe49 (2005).

PERSPECTIVE

Science’s stke  
www.stke.org/cgi/content/full/sigtrans;2005/304/pe49
Page 4

SLIM Trims STATs: Ubiquitin E3 Ligases Provide Insights for Specificity in the Regulation of Cytokine Signaling

Daniela Ungureanu and Olli Silvennoinen (September 27, 2005)

Science Signaling 2005 (304), pe49. [doi: 10.1126/stke.3042005pe49]

---

The following resources related to this article are available online at http://stke.sciencemag.org. This information is current as of June 4, 2015.

---

**Article Tools**  
Visit the online version of this article to access the personalization and article tools:  
http://stke.sciencemag.org/content/2005/304/pe49

**References**  
This article cites 28 articles, 13 of which you can access for free at:  
http://stke.sciencemag.org/content/2005/304/pe49#BIBL

**Glossary**  
Look up definitions for abbreviations and terms found in this article:  
http://stke.sciencemag.org/cgi/glossarylookup

**Permissions**  
Obtain information about reproducing this article:  
http://www.sciencemag.org/about/permissions.dtl

---

Science Signaling (ISSN 1937-9145) is published weekly, except the last December, by the American Association for the Advancement of Science, 1200 New York Avenue, NW, Washington, DC 20005. Copyright 2015 by the American Association for the Advancement of Science; all rights reserved.
