
The Plant Journal (2005) 44, 314–333

doi: 10.1111/j.1365-313X.2005.02530.x

# Immediate-early and delayed cytokinin response genes of *Arabidopsis thaliana* identified by genome-wide expression profiling reveal novel cytokinin-sensitive processes and suggest cytokinin action through transcriptional cascades

Wolfram G. Brenner${}^{1}$, Georgy A. Romanov${}^{2}$, Ireen Kollmer${}^{3}$, Lukas Burkle${}^{1,\dagger}$ and Thomas Schmülling${}^{3,*}$

${}^{1}$Max Planck-Institute for Molecular Genetics, Ihnestraße 63–73, D-14195 Berlin, Germany,  
${}^{2}$Institute of Plant Physiology, Russian Academy of Sciences, Botanicheskaya 35, 127276 Moscow, Russia, and  
${}^{3}$Institute of Biology/Applied Genetics, Freie Universität Berlin, Albrecht-Thaer-Weg 6, D-14195 Berlin, Germany

Received 26 May 2005; revised 18 July 2005; accepted 2 August 2005.  
*For correspondence (fax +49 30 838 54345; e-mail tschmue@zedat.fu-berlin.de).  
†Present address: ETH Zürich, Swiss Federal Institute of Technology Zürich, Institute of Plant Science, Universitätstr.2, CH - 8092 Zürich, Switzerland.*

---

## Summary

Cytokinins are hormones that regulate many developmental and physiological processes in plants. Recent work has revealed that the cytokinin signal is transduced by two-component systems to the nucleus where target genes are activated. Most of the rapid transcriptional responses are unknown. We measured immediate-early and delayed cytokinin responses through genome-wide expression profiling with the Affymetrix ATH1 full genome array (Affymetrix Inc., Santa Clara, CA, USA). Fifteen minutes after cytokinin treatment of 5-day-old *Arabidopsis* seedlings, 71 genes were upregulated and 11 genes were downregulated. Immediate-early cytokinin response genes include a high portion of transcriptional regulators, among them six transcription factors that had previously not been linked to cytokinin. Five plastid transcripts were rapidly regulated as well, indicating a rapid transfer of the signal to plastids or direct perception of the cytokinin signal by plastids. After 2 h of cytokinin treatment genes coding for transcriptional regulators, signaling proteins, developmental and hormonal regulators, primary and secondary metabolism, energy generation and stress reactions were overrepresented. A significant number of the responding genes are known to regulate light (*PHYA*, *PSK1*, *CIP8*, *PAT1*, *APRR*), auxin (*Aux/IAA*), ethylene (*ETR2*, *EIN3*, *ERFs/EREBPs*), gibberellin (*GAI*, *RGA1*, *GA20* oxidase), nitrate (*NTR2*, *NIA*) and sugar (*STP1*, *SUS1*) dependent processes, indicating intense crosstalk with environmental cues, other hormones and metabolites. Analysis of cytokinin-deficient 35S:*AtCKX1* transgenic seedlings has revealed additional, long-lasting cytokinin-sensitive changes of transcript abundance. Comparative overlay-analysis with the software tool MAPMAN identified previously unknown cytokinin-sensitive metabolic genes, for example in the metabolism of trehalose-6-phosphate. Taken together, we present a genome-wide view of changes in cytokinin-responsive transcript abundance of genes that might be functionally relevant for the many biological processes that are governed by cytokinins.

Keywords: cytokinin, gene expression, immediate-early genes, response genes, transcriptome analysis.

---

## Introduction

Cytokinins are a class of plant hormones that play a role in many aspects of plant growth and development, including apical dominance, the formation and activity of shoot meristems, leaf senescence, nutrient mobilization, seed germination, root growth and stress responses. They also appear to mediate a number of light-regulated processes, such as de-etiolation and chloroplast differentiation (Mok, 1994; Schmülling, 2004). Recently, major steps of cytokinin metabolism and cytokinin signaling have been elucidated in *Arabidopsis*. Seven cytokinin-synthesizing isopentenyl transferase (*IPT*) genes and seven genes coding for cytokinin-degrading cytokinin oxidase/dehydrogenase (*CKX*) genes are present in the *Arabidopsis* genome (Bilyeu et al., 2001; Kakimoto, 2001; Takei et al., 2001; Werner et al., 2001). CKX genes have been used to generate cytokinin-deficient plants. Cytokinin deficiency causes a reduced activity of the

shoot meristems and the formation of an enhanced root system (Werner *et al.*, 2003). The cytokinin signal is perceived by three membrane-located sensor histidine kinases (Inoue *et al.*, 2001; Suzuki *et al.*, 2001; Ueguchi *et al.*, 2001). It is further transmitted by a multistep phosphorelay system via phosphotransmitter proteins to type B response regulator proteins (Arabidopsis response regulators, ARRs), which act as transcription factors (Lohrmann *et al.*, 2001; Sakai *et al.*, 2000). Type B ARR proteins activate immediate-early response genes. The best known class of immediate-early genes in Arabidopsis are the type A ARR genes, which are upregulated within minutes after a cytokinin pulse (Brandstatter and Kieber, 1998; D'Agostino *et al.*, 2000; Taniguchi *et al.*, 1998). Type A response regulators act as negative regulators on the cytokinin signaling pathway (Hwang and Sheen, 2001; To *et al.*, 2004). By using the Affymetrix 8k Arabidopsis Genome Array, rapid response genes have been identified in Arabidopsis (Rashotte *et al.*, 2003). Several other studies identified late cytokinin responsive genes in Arabidopsis and diverse other species (Che *et al.*, 2002; Hoth *et al.*, 2003; Kiba *et al.*, 2004, 2005; Schäfer and Schmülling, 2002; Schmülling *et al.*, 1997; Zimmermann *et al.*, 2004). A comprehensive analysis of the overall transcriptional response to cytokinin is still missing and, in particular, the number of immediate-early cytokinin responsive genes identified so far is rather limited. Very rapid changes of transcript abundance, which are mediated by already expressed signal transmitters and transcription factors, are of particular interest. We, therefore, used the Affymetrix ATH1 full genome array, which represents approximately 23,750 Arabidopsis genes (Redman *et al.*, 2004), first to identify immediate-early genes. Some immediate-early proteins may subsequently act as transcriptional regulators and trigger a second wave of gene expression changes that may control more specific developmental or physiological processes. Therefore, we analyzed also the transcriptional changes caused by a longer cytokinin treatment (120 min). These changes were compared with those occurring under constitutive cytokinin-deficiency in order to identify genes and processes that are regulated in an opposite manner under these two conditions. The datasets were mined with the software tool MAPMAN (Thimm *et al.*, 2004) in order to search for specific correlations and in this way discover novel processes that are under cytokinin control. Our data suggest that cytokinin acts through transcriptional cascades and shares numerous pathways with other signals.

### Results and discussion

#### Identification of immediate-early response genes

We decided to use young seedlings to search for genes that are rapidly regulated by cytokinins as these contain a relatively high portion of meristematic tissues, where cytokinins are thought to exert many of their functions. Analysis of plants harboring a $P_{ARR5}::GUS$ fusion gene revealed that almost all cells in the seedling at this stage are able to react to cytokinins (D'Agostino *et al.*, 2000; Romanov *et al.*, 2002). Two independent experiments were performed for each condition. The correlation coefficients for the replicates as well as for the pairs used for ratio calculation are summarized in Table S1. There was a good agreement between independent biological replications. Approximately 10,000 genes were called present (P) in each of the experiments. To increase the chance to detect differentially expressed genes, transcripts with a ratio of change of >1.8 in both replicates and a P classification for the higher expression value were classified as regulated.

The transcripts of 71 genes were reproducibly upregulated >1.8-fold 15 min after cytokinin treatment (Figure 1). The majority of all upregulated transcripts were present (P) before the cytokinin treatment, and nine genes were induced from absent to present (A → P; Table 1). The induced genes

(a)

| Col-0 BA 15 vs Col-0 BA 0 | Col-0 BA 120 vs Col-0 BA 0 | CKX1 BA 15 vs CKX1 BA 0 | CKX1 BA 0 vs Col-0 BA 0 |
|-----------------------------|-------------------------------|---------------------------|--------------------------|
| ![Image](#) 71             | ![Image](#) 667               | ![Image](#) 27            | ![Image](#) 130          |
| ![Image](#) 11145          | ![Image](#) 9816              | ![Image](#) 8860           | ![Image](#) 10879        |
| ![Image](#) 11             | ![Image](#) 1169              | ![Image](#) 17             | ![Image](#) 341          |

(b)

| Col-0 BA 15 vs Col-0 BA 0 | Col-0 BA 120 vs Col-0 BA 0 | Col-0 BA 15 vs Col-0 BA 0 |
|-----------------------------|-------------------------------|-----------------------------|
| ![Image](#)                | ![Image](#)                  | ![Image](#)                |
| ![Image](#)                | ![Image](#)                  | ![Image](#)                |
| ![Image](#)                | ![Image](#)                  | ![Image](#)                |

| Col-0 BA 15 vs Col-0 BA 0 | Col-0 BA 120 vs Col-0 BA 0 | CKX1 BA 15 vs CKX1 BA 0 |
|-----------------------------|-------------------------------|---------------------------|
| ![Image](#)                | ![Image](#)                  | ![Image](#)              |
| ![Image](#)                | ![Image](#)                  | ![Image](#)              |
| ![Image](#)                | ![Image](#)                  | ![Image](#)              |

**Figure 1. Diagrams showing a compilation of gene expression data.**

(a) Diagrams showing the total number of expressed and regulated genes in the different experiments. Upregulation and downregulation is indicated by the arrows in the colored circles. From left to right, changes of gene expression in 5-day-old wild-type seedlings: after 15 min of cytokinin treatment (5 μM BA); after 120 min of cytokinin treatment; after 15 min of cytokinin treatment of cytokinin-deficient seedlings; and of cytokinin-deficient seedlings (35S:CKX1 transgenic line).

(b) Venn diagrams showing the overlapping and non-overlapping sets of genes for the different conditions as indicated by numbers and color codes.

Table 1 Immediate-early cytokinin response genes of Arabidopsis

| Col-0 | CKX1 | BA15 | BA120 | Affymetrix ID | AGI ID | Sequence description | Ratio Col-0 BA15 versus Col-0 BA0 | Call Col-0 BA15 versus BA0 | Ratio CKX1 BA120 versus Col-0 BA0 | Call CKX1 BA15 versus Col-0 BA0 |
|------|------|------|------|----------------|-------|-----------------------|----------------------------------|-------------------------------|----------------------------------|-------------------------------|
| ↑    | ↑    |      |      | 252374_at      | At3g48100 | Two component response regulator 5 (ARR5) | 13.4                          | P                               | 5.7                            | P                             |
| ↑    | ↑    |      |      | 247406_at      | At5g62920 | Two component response regulator 6 (ARR6) | 7.4                           | P                               | 2.4                            | P                             |
| ↑    | ↑    |      |      | 262488_at      | At1g21820 | Unknown protein EST gbl/AA586241          | 5.0                           | P                               | 8.6                            | P                             |
| ↑    | ↑    |      |      | 263236_at      | At1g10470 | Two component response regulator 4 (ARR4) | 1.9                           | P                               | 4.0                            | P                             |
| ↑    | ↑    |      |      | 262399_at      | At1g49500 | Unknown protein                              | 4.2                           | P                               | 4.8                            | P                             |
| ↑    | ↑    |      |      | 267591_at      | At2g39705 | Expressed protein                            | 3.3                           | P                               | 3.7                            | P                             |
| ↑    | ↑    |      |      | 265067_at      | At1g03850 | Hypothetical protein                         | 3.2                           | P                               | 3.4                            | P                             |
| ↑    | ↑    |      |      | 263382_at      | At2g40230 | Putative anthranilate N-hydroxycinnamoyl/ benzoyltransferase | 2.1                           | P                               | 3.8                            | P                             |
| ↑    | ↑    |      |      | 251436_at      | At3g59900 | Putative protein                             | 1.8                           | P                               | 2.9                            | P                             |
| ↑    | ↑    |      |      | 262212_at      | At1g74890 | Two component response                       | 16.0                          | A → P                           | 15.0                           | A                             |
| ↑    | ↑    |      |      | 257635_at      | At4g26280 | Cytochrome P450 monooxygenase (CYP71B4)     | 13.1                          | A → P                           | 27.5                           | A                             |
| ↑    | ↑    |      |      | 254250_at      | At4g23290 | Serine/threonine kinase-like protein         | 7.5                           | P                               | 6.4                            | P                             |
| ↑    | ↑    |      |      | 253253_at      | At4g34750 | Putative protein auxin-regulated gene        | 5.4                           | A → P                           | 5.9                            | A → P                         |
| ↑    | ↑    |      |      | 252462_at      | At3g47250 | Putative protein                             | 5.3                           | P                               | 6.9                            | P                             |
| ↑    | ↑    |      |      | 259466_at      | At1g19050 | Two component response regulator 9 (ARR9)   | 5.0                           | P                               | 2.6                            | P                             |
| ↑    | ↑    |      |      | 244961_at      | ycf5     | Hypothetical protein                         | 4.6                           | P                               | 4.9                            | P                             |
| ↑    | ↑    |      |      | 246099_at      | At5g20230 | Blue copper binding protein                  | 4.4                           | P                               | 12.8                           | P                             |
| ↑    | ↑    |      |      | 257964_at      | At3g19850 | Hypothetical protein                         | 3.6                           | P                               | 4.8                            | P                             |
| ↑    | ↑    |      |      | 251665_at      | At3g57040 | Two component response                       | 3.2                           | P                               | 2.8                            | P                             |
| ↑    | ↑    |      |      | 263737_at      | At1g60010 | Unknown protein                              | 3.1                           | P                               | 3.8                            | P                             |
| ↑    | ↑    |      |      | 266313_at      | At2g26980 | CBL-interacting protein kinase 3 (CIPK3)    | 3.1                           | P                               | 2.8                            | P                             |
| ↑    | ↑    |      |      | 262259_s_at    | At1g53870 | Hypothetical protein                         | 3.1                           | P                               | 3.5                            | P                             |
| ↑    | ↑    |      |      | 256299_at      | At1g69530 | Expansin (AtEXP1)                           | 3.0                           | P                               | 2.4                            | P                             |
| ↑    | ↑    |      |      | 261110_at      | At1g75440 | E2, ubiquitin-conjugating enzyme 16 (UBC16) | 2.8                           | P                               | 4.3                            | P                             |
| ↑    | ↑    |      |      | 258262_at      | At3g15770 | Hypothetical protein                         | 2.7                           | P                               | 3.8                            | P                             |
| ↑    | ↑    |      |      | 266078_at      | At2g40670 | Two component response regulator 16 (ARR16) | 2.7                           | P                               | 4.3                            | P                             |
| ↑    | ↑    |      |      | 251356_at      | At3g61060 | F-box protein (lectin-related)              | 2.6                           | P                               | 4.3                            | P                             |
| ↑    | ↑    |      |      | 246251_at      | At4g37220 | Cold acclimation protein homolog             | 2.6                           | P                               | 7.5                            | P                             |


| Col-0 | BA15 | CKX1 | BA120 | Affymetrix ID | Sequence description | Ratio Col-0 BA15 versus Col-0 BA0 | Call Col-0 BA15 versus BA0 | Ratio CKX1 BA120 versus Col-0 BA0 | Call CKX1 BA15 versus BA0 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 255283_at | At4g04620 | Putative symbiosis-related protein | 2.5 | P | 3.4 | P | 1.1 | P | 1.9 | P |
| 266799_at | At2g22860 | Unknown protein | 2.5 | P | 2.7 | P | 1.5 | P | 0.8 | P |
| 254197_at | At4g24040 | Trehalase-like protein trehalase 1 (GM TRE 1), Glycine max | 2.4 | A → P | 6.0 | A → P | 1.1 | A | 2.0 | A |
| 256069_at | At1g13740 | Hypothetical protein predicted by genscan+ | 2.3 | P | 2.9 | P | 1.5 | P | 1.3 | P |
| 259982_at | At1g76410 | Putative RING zinc finger protein | 2.3 | P | 5.6 | P | 1.8 | P | 1.0 | P |
| 260527_at | At2g47270 | Unknown protein | 2.3 | P | 2.9 | P | 3.3 | A | 0.4 | A |
| 250287_at | At5g13330 | Putative protein AP2 domain containing protein RAP2.6 | 2.0 | P | 2.8 | P | 1.6 | P | 1.5 | P |
| 250124_at | At5g16480 | Putative protein contains similarity to tyrosine phosphatase | 2.0 | P | 2.0 | P | 1.3 | A | 1.6 | A |
| 253038_at | At4g37790 | Homeobox protein HAT22 | 2.4 | P | 1.4 | P | 2.3 | P | 0.7 | P |
| 245048_at | matK | Orf within trnK intron | 2.2 | P | 1.3 | P | 1.9 | P | 0.6 | P |
| 260268_at | At1g68490 | Unknown protein | 2.2 | P | 2.0 | P | 2.3 | P | 0.8 | P |
| 253692_at | At4g29720 | Amine oxidase family | 2.0 | P | 1.0 | P | 2.1 | P | 0.9 | P |
| 265342_at | At2g18300 | bHLH064 putative bHLH transcription factor | 2.9 | P | 2.7 | P | 2.0 | A | 0.3 | P → A |
| 257339_s_at | Mitochondria 1 ATP synthase subunit 9 | 2.8 | P | 2.9 | P | 0.9 | P | 4.9 | P | 0.9 | A |
| 262374_s_at | At1g72930 | Similar to flax rust resistance protein GI: 4588066 from *Linum usitatissimum* | 3.5 | P | 0.4 | P → A | 0.8 | A | 0.9 | A |
| 245776_at | At1g30260 | Expressed protein | 9.0 | A → P | 7.0 | A | 2.4 | A | 1.5 | A |
| 259728_at | At1g60970 | Coatomer zeta subunit-related | 8.7 | A → P | 13.1 | A | 2.5 | A | 3.8 | A |
| 244971_at | clpP | Auxin-induced protein, putative | 3.0 | P | 1.4 | P | 0.5 | A | 2.4 | A |
| 244965_at | ORF31 | Hypothetical protein | 3.6 | P | 3.1 | P | 1.2 | P | 0.9 | P |
| 267644_s_at | At2g32880 | Expressed protein | 3.1 | A → P | 5.0 | A | 0.2 | A | 5.3 | A |
| 263656_at | At1g04240 | ATP-dependent protease subunit | 3.0 | P | 3.9 | P | 1.1 | P | 0.6 | P |
| 259365_at | ycf10_cemA | Hypothetical protein | 2.8 | P | 5.0 | P | 0.5 | P | 0.6 | A |
| 245020_at | petA | Expressed protein | 2.7 | P | 4.2 | P | 1.1 | P | 2.0 | P |
| 252076_at | At3g51660 | Cytochrome f | 2.6 | P | 1.4 | P | 0.6 | A | 1.0 | A |
| 258468_at | At3g06070 | Expressed protein | 2.4 | P | 1.7 | P | 0.9 | A | 1.5 | A |
| 264899_at | At1g23130 | Macrophage migration inhibitory factor (MIF) family Bet v1 allergen family | 2.4 | P | 3.2 | P | 1.4 | P | 0.7 | P | 1.8 | A |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |  |  |  |
|  |  |  |
| Col-0 | CKX1 | BA15 | BA120 | Affymetrix ID | Sequence description | Ratio Col-0 BA15 versus Col-0 BA0 | Call Col-0 BA15 versus Col-0 BA0 | Ratio CKX1 BA120 versus BA0 versus Col-0 BA0 | Call CKX1 BA15 versus Col-0 BA0 |
|-------|------|------|-------|----------------|---------------------|------------------------------------|-----------------------------------|----------------------------------------|-----------------------------------|
| 257206_at | At3g16530 | Lectin-related | 0.7 | P | 0.6 | A | A |
| 264782_at | At1g08810 | myb family transcription factor | 2.2 | A → P | 1.8 | A | A |
| 253872_at | At4g27410 | No apical meristem (NAM) protein family | 2.2 | P | 1.8 | P | P |
| 267459_at | At2g33850 | Expressed protein | 2.2 | A → P | 0.8 | A | 6.2 | A |
| 254998_at | At4g09760 | Choline kinase GmCK2p-related protein | 2.1 | P | 0.8 | P | 1.7 | P |
| 264040_at | At2g03730 | ACT domain-containing protein | 2.0 | P | 1.6 | P | 1.4 | P |
| 252093_at | At3g51500 | Expressed protein | 2.0 | P | 2.9 | P | 1.9 | A |
| 254016_at | At4g26150 | GATA zinc finger protein | 2.0 | P | 1.4 | P | 0.9 | A |
| 260639_at | At1g53180 | Expressed protein | 2.0 | P | 2.0 | P | 0.8 | A |
| 246502_at | At5g16240 | Acyl-(acyl-carrier-protein) desaturase (stearyl-ACP)-related | 2.0 | P | 1.5 | P | 0.8 | P → A | 0.9 | A |
| 245011_at | psbG | Photosystem II G protein | 1.9 | P | 1.8 | P | 0.6 | P → A | 1.5 | A |
| 263048_s_at | At2g05310 | Expressed protein | 1.9 | P | 1.5 | P | 1.0 | P | 1.1 | P |
| 264285_at | At1g62040 | Symbiosis-related protein-binding protein | 1.8 | P | 2.0 | P | 1.6 | P | 1.0 | P |
| 262883_at | At1g64780 | Ammonium transporter, putative | 0.5 | P | 0.2 | P | 0.5 | P | 0.8 | P |
| 262092_at | At1g56150 | Auxin-induced (indole-3-acetic acid induced) protein family | 0.5 | P | 0.3 | P → A | 0.7 | P | 1.0 | P |
| 254962_at | At4g11050 | Glycosyl hydrolase family 9 (endo-1,4-beta-glucanase) | 0.5 | P | 0.4 | P → A | 0.7 | P | 1.2 | P |
| 248857_at | At5g46640 | Expressed protein | 0.5 | P | 0.8 | A | 0.9 | A |
| 264026_at | At2g21060 | Glycine-rich protein (AtGRP2) | 0.4 | P | 0.7 | P | 1.3 | P |
| 266232_at | At2g02310 | F-box protein (SKP1 interacting partner 3-related) | 0.4 | P → A | 0.9 | P → A | 0.8 | P |
| 252892_at | At4g39420 | Hypothetical protein | 0.4 | P → A | 0.9 | P → A | 1.0 | P → A | 1.8 | A |

Ratios are the average of changes in transcript abundance after 15 min of cytokinin treatment of two independent biological replications. Expression of the immediate-early response genes after 120 min of cytokinin treatment of wild type (Col-0 BA120), after cytokinin treatment of cytokinin-deficient seedlings compared with wild type (Col-0 CKX1) are included as well and expression changes above the threshold are indicated by arrows in the left columns. Robust response genes that are upregulated or downregulated in cytokinin-deficient seedlings and showed an opposite regulation in cytokinin-treated wild-type seedlings were placed on top of the list. An extended version of this Table reporting the results of two independent experiments is included as Table S2.
contain seven of the 10 known rapidly regulated type A ARR genes, which belong to the genes with the highest induction factors (Table 1 and Table S2). Induction of these genes confirms earlier reports (Brandstatter and Kieber, 1998; D'Agostino et al., 2000; Taniguchi et al., 1998) and validates the experimental system.

A rapidly increased transcript abundance of ARR5 and three other less well induced genes was confirmed by Northern blot (Figure 2a) and the upregulation of several low expressed transcription factor genes was confirmed by semiquantitative RT-PCR (Figure 2b). Importantly, there is a partial overlap between the immediate-early cytokinin response genes identified here and genes upregulated in plants overexpressing type B ARR genes (Hass et al., 2004; Kiba et al., 2005), supporting the idea that these transcription factors are functionally involved in the immediate-early cytokinin response.

Table 2 shows that the immediate-early response genes contain a high percentage of genes involved in transcriptional control, namely 24% of the regulated genes, compared with 5.5% in the whole genome (Riechmann et al., 2000). In particular, single members of six different transcription factor families, including an APETALA2 (AP2) domain protein (At5g13330), the homeodomain leucine-zipper (HD-ZIP) protein HAT22 (At4g37790), a basic helix-loop-helix (bHLH) transcription factor (At2g18300), a Myb-domain protein (At1g08810), a member of the NAC-domain family (A4g27410), and a GATA zinc finger protein (At4g26150)

(a)
| Time of BA treatment (min) | 0 | 15 | 45 | 120 |
|-----------------------------|---|----|----|-----|
| At3g48100 (ARR5)            |   |    |    |     |
| At1g03850 (glutaredoxin)    |   |    |    |     |
| At2g26980 (CIPK3)           |   |    |    |     |
| At3g16770 (AP2 family)      |   |    |    |     |
| At3g18780 (actin2)          |   |    |    |     |

(b)
| Genotype Time of BA treatment (min) | Col-0 | CKX1 |
|-------------------------------------|-------|------|
| At2g18300 (bHLH64)                  | 0     | 15   | 120 | 0   |
| At4g37790 (HAT22)                   |       |      |     |     |
| At4g26150 (GATA)                    |       |      |     |     |
| At4g16780 (HAT4)                    |       |      |     |     |
| actin                               |       |      |     |     |

Figure 2. Expression analysis of selected upregulated genes.
(a) Northern blot analysis of immediate-early response genes.
(b) Semiquantitative RT-PCR of low expressed and cytokinin-induced transcription factor genes.

© Blackwell Publishing Ltd, The Plant Journal, (2005), 44, 314–333

Cytokinin response genes of Arabidopsis 319 were rapidly upregulated. Regulation of transcription factor genes is of special interest because they can activate the coordinated expression of many downstream target genes and in this way govern entire metabolic and developmental processes. These specific transcription factors, with the exception of At4g26150 (Kiba et al., 2005), have previously not been linked to cytokinin action and to our knowledge no specific function has been attributed to any of these proteins. One interesting known transcriptional regulator is IAA3/SHY2, which was also identified as cytokinin regulated by Rashotte et al. (Rashotte et al., 2003). IAA3/SHY2 belongs to the early auxin response gene family Aux/IAA (Hagen and Guilfoyle, 2002) and has roles in root growth, root branching and root hair formation (Knox et al., 2003; Tian and Reed, 1999). IAA3/SHY2 acts not only as a negative regulator of auxin signaling (Tian et al., 2002) but influences also photomorphogenic processes (Colón-Carmona et al., 2000), indicating a broader function of the protein. Regulation of IAA3/SHY2 by cytokinin may contribute to modulate the cytokinin effect in processes depending on the cytokinin-to-auxin ratio, such as root growth and branching, or it may be involved in the modulation of light-dependent processes by cytokinin.

A second over-represented class comprises genes associated with development (Table 2). An interesting developmental gene code for the phytosulfokine precursor 2 (PSK2) gene. PSK2 encodes a prohormone, from which the plant peptide growth factor PSK-α is released (Yang et al., 2001). PSK-α promotes plant cell proliferation in cultured cells and appears to cooperate with cytokinin and auxin to regulate diverse differentiation processes (Yang et al., 2001). It is possible that cytokinin induction of the PSK2 gene is functionally relevant for cell cycle regulation and other regulatory activities of the hormone. Another gene of this functional class is related to the major birch allergen Betv1 (Table 1). The Betv1 protein binds a variety of compounds, among them cytokinins, an ability that is thought to be relevant for its biological activity (Mogensen et al., 2002).

To the regulated genes encoding signal transduction proteins belongs a highly induced (ca. eightfold) serine/threonine-like protein kinase gene (At4g23290) of unknown function. A calcineurin B-like protein (CBL) interacting protein kinase (CIPK3) indicates the involvement of calcium as a second messenger in the early cytokinin response (see also below). A putative tyrosine phosphatase (At5g16480), an enzyme type that is particularly rare in plants and has recently been shown to regulate activity of cyclin-dependent kinase (CDK) and 14-3-3 proteins (Giacometti et al., 2004; Landrieu et al., 2004), is upregulated as well (Table 1).

Genes involved in control of protein turnover were also rapidly upregulated, encoding an E2 class ubiquitin-conjugating enzyme 16 (UBC16), an F-box protein (At3g61060), a putative RING zinc finger protein and an ATP-dependent protease subunit. Regulation of protein turnover is important in plants and it may be expected that it is also relevant for
Table 2 Cytokinin regulates genes with diverse functions

| Functional class | Col-0 BA 15 up | Col-0 BA 15 down | Col-0 BA 120 up | Col-0 BA 120 down | Col-CKX up | Col-CKX down | Whole genome |
| --- | --- | --- | --- | --- | --- | --- | --- |
|  | raw no. % | raw no % | raw no % | raw no. % | raw no. % | raw no. % | raw no. % |
| Total number (hits) | 71 | 10 | 713 | 1284 | 130 | 375 | 64685 |
| Unclassified | 21 | 2 | 218 | 239 | 39 | 75 | 21506 |
| Signal transduction | 5 | 65 | 118 | 7 | 16.3 | 4173 |  |
| Transcriptional control | 17 | 2 | 85 | 89 | 6 | 11.5 | 3561 |
| Development | 7 | 31 | 58 | 5 | 17 | 4.5 | 3303 |
| Stress/defense/detoxification | 4 | 42 | 152 | 21 | 16.2 | 37 | 9.9 |
| Protein targeting & transport | 1 | 6 | 13 | 1 | 0.8 | 6 | 1.6 |
| Cytoskeleton | 1 | 13 | 16 | 8 | 2.1 | 1874 |  |
| Translation |  | 10 | 13 | 6 | 4.6 | 12 | 3.2 |
| Protein modification | 2 | 3 | 2 | 0.2 |  |  | 1621 |
| Carbohydrate catabolism | 1 | 30 | 32 | 3 | 2.3 | 5 | 1.3 |
| Secondary metabolism | 3 | 1 | 25 | 92 | 12 | 9.2 | 29 |
| Unspecified/other transport |  | 9 | 46 | 1 | 0.8 | 3 | 0.8 |
| Lipid & fatty acid metabolism | 1 | 10.0 | 22 | 63 | 6 | 4.6 | 6 |
| Cell cycle control |  |  | 11 | 0.9 |  | 8 | 2.1 |
| Electron transport | 2 | 8 | 6 | 0.5 | 4 | 3.1 | 2 |
| Transposable element activation |  | 1 | 1 | 0.1 |  |  |  |
| Protein folding & stabilization |  | 4 | 13 | 1 | 0.8 | 3 | 0.8 |
| Cell wall modification | 1 | 1 | 20 | 89 | 4 | 3.1 | 11 |
| Phosphate metabolism |  |  | 1 | 0.1 |  |  |  |
| Carbohydrate anabolism |  | 7 | 16 | 1.3 |  |  |  |
| Cofactor, vitamin and prosthetic group metabolism |  | 2 | 2 | 0.2 |  | 1 | 0.3 |
| Chromatin remodeling | 1 |  | 9 | 0.7 | 2 | 0.5 | 594 |
| DNA repair | 1 |  | 6 | 0.5 | 1 | 0.3 | 407 |
| Ionic homeostasis |  |  | 1 | 0.1 |  |  | 375 |
| Plant hormonal regulation | 19 |  | 21 | 1.6 | 6 | 1.6 | 332 |
| Aerobic respiration | 2 |  | 12 | 0.9 | 2 | 0.5 | 272 |
| Photosynthesis | 1 | 1 | 6 | 9 | 0.8 |  | 111 |
| ATP/energy metabolism | 1 | 1 | 4 | 2 | 1 | 0.8 | 110 |

Genes were classified into functional categories based on established or putative functions as annotated in the MIPS database (http://mips.gsf.de). Association of individual genes with several functional classes is allowed. Classes that were over- or under-represented in a given experimental condition by a factor >1.8 compared with their genomic proportion are marked in dark and light gray, respectively. Only selected functional classes were included in the table.

degradation pathway (Schuster *et al.*, 1999). It could be, therefore, that the enhanced detection of plastid-located polyadenylated RNA indicates enhanced degradation of these RNA species. In any case, the fast response of plastid genes to a cytokinin signal necessitates a very rapid transfer of the cytokinin signal to plastids. Alternatively, the cytokinin signal may be perceived and interpreted directly by the plastids themselves.

Less genes (11) were rapidly and strongly downregulated following cytokinin treatment (Table 1). Fifteen minutes after cytokinin treatment, differences were generally in the range of twofold compared to time point zero. Transcripts of two genes, coding for an SKP1-interacting partner related F-box protein and a hypothetical protein, were rapidly suppressed to an undetectable level (Table 1). How this rapid decay of specific mRNAs is achieved deserves further analysis.

The immediate-early cytokinin response is dampened in cytokinin-deficient plants

We tested whether the same or a different set of genes would be rapidly induced by exogenous cytokinin in cytokinin-deficient plants. In total, 44 transcripts reacted after 15 min of cytokinin-treatment with a rapid increase or decrease >1.8-fold (Figure 1; Table 1 and Table S3). Thirteen of the induced genes were identical to those identified to be rapidly upregulated in wild type. Among the known genes are three of the type A ARRs (namely ARR4, ARR5, and ARR6) and the Homeobox (HB) gene HAT22. Among the 14 novel genes are several zinc finger proteins (including a CONSTANS family member), a putative AP2 transcription factor, a No apical meristem (NAM) protein family member, and a member of the 1-aminocyclopropane-1-carboxylate (ACC) synthase gene family (Table S5). In wild type, some of these genes were not detectable; others were weakly induced but remained below the threshold of 1.8. The fact that there is less gene regulation in the CKX1 overexpressing plants is surprising, as one might have anticipated a sensitized signaling system to compensate for the lower cytokinin content. On the contrary, the overall response of CKX1 transgenics to cytokinins appears to be somewhat dampened. One reason could be that the receptor gene CRE1/AHK4 is downregulated in cytokinin-deficient plants and CYTOKININ INDEPENDENT 2 (CKI2), encoding a histidine kinase, which can feed into the cytokinin response pathway, is upregulated (see below). These constitutive changes may cause on the one hand a different responsiveness for a subset of the cytokinin response genes and on the other hand an enhanced basal transcript level of another set of genes.

Identification of delayed response genes

After 120 min of cytokinin treatment, the number of induced and repressed genes increased strongly (Figure 1). The

Cytokinin response genes of Arabidopsis 321

transcript abundance of 667 genes increased >1.8-fold and the abundance of 1169 transcripts decreased >1.8-fold in two independent repetitions. Thirty-six of the 71 upregulated immediate-early response genes still have increased transcript levels after 120 min of cytokinin treatment (Table 1). Similarly, after 120 min four of the 11 rapidly reduced transcripts still showed a significant reduction of transcript levels while the remaining ones were no longer differentially expressed (Table 1).

The genes were ordered according to their changes in transcript levels 15 and 120 min after cytokinin induction: those genes whose transcript levels were significantly upregulated or downregulated at both investigated time points are highlighted in gray in Table 1. They were classified as 'robust response genes', while the other genes were classified as 'transient response genes'. Further criteria to judge the robustness of response genes was their regulation in cytokinin-deficient plants. Based on these criteria ARR4, ARR5, ARR6 and a gene of unknown function (At1g26280) can be classified as the most reliable and informative reporters about the cytokinin status (Table 1).

After 120 min, different functional classes of genes were over- or under-represented among the regulated genes (Table 2). Generally over-represented functional classes were the signaling genes with a total of 18% of the regulated genes, compared with a genomic portion of 6.5%, transcriptional regulators (12% versus 5.5%), stress-related genes, cell wall modifying genes and genes involved in hormonal regulation and photosynthesis. Genes of carbohydrate catabolism, photosynthesis and energy metabolism were over-represented among the upregulated genes, while genes of secondary metabolism and lipid metabolism were more abundant among the downregulated genes (Table 2). Together this indicates that 2 h after cytokinin treatment specific changes have occurred that reflect reprogramming of multiple processes. Table S4 shows regulated genes of different functional categories (see Table S5 for the complete list). Some are discussed in more detail in the subsequent sections.

Cytokinin-regulated genes involved in transcriptional regulation

Most of the transcription factor genes that were strongly upregulated 15 min after cytokinin treatment had fallen below the threshold after 120 min, suggesting that they function transiently, activating a second wave of transcriptional regulators. An exception is the AP2 domain protein RAP2.6 encoding gene At5g13330, whose transcript level remains elevated (Table 1). Four other members of the AP2 class and four members of the closely related ERF/EREBP class were upregulated (Riechmann and Meyerowitz, 1998). Ectopic expression of one of these genes, TINY, in the *tiny* mutant affects cell size and shape and causes phenotypic

changes reminiscent of ethylene and cytokinin mutants (Wilson et al., 1996).

Three members of a plant-specific subset of the homeobox (HB) gene superfamily, HAT1, HAT3 and HAT4, were upregulated (Table S4). Ectopic expression of HAT4 (identical to AtHB1) causes a series of developmental changes including hypocotyl elongation, early flowering, altered leaf morphology and dark green pigmentation (Schena et al., 1993). HAT4 is strongly regulated by different light qualities and thought to direct morphological changes during the shade avoidance response (Jiao et al., 2003).

Other transcription factor genes with increased transcript levels were 10 genes encoding Zn-finger proteins (including members of the GATA, Dof and CONSTANS families) three GRAS, two NAC, two Myb, two bHLH and one bZIP family member. Even more transcription factor genes were down-regulated. Among approximately 35 genes, genes coding for Zn-finger proteins (12) and proteins of the WRKY class (7) were the most abundant (Table S5). Several of the regulated genes are known to play roles in auxin, ethylene, gibberellin and light signaling and are discussed further in the specific sections below.

### Cytokinin-regulated genes associated with signal transduction

There is evidence from inhibitor studies that protein phosphorylation, other than the phosphorelay of the two-component system, plays a role in mediating the cytokinin response (Dominov et al., 1992; Schäfer and Schmülling, 2002), and transcriptional regulation of numerous genes encoding for kinases and phosphatases found in this study lends further support.

One particularly well represented class of signaling proteins is associated with calcium, which is an important convergence point of many signaling pathways. Genes encoding calcineurin B-like protein (CBL) interacting Ser/Thr protein kinases CIPK1, CIPK3 and CIPK20 are upregulated by cytokinin, while the CIPK5, CIPK6, CIPK8, CIPK14 and CIPK16 genes are downregulated (Tables S4 and S5). CIPK proteins interact specifically with calcium-sensing CBL proteins and are likely their regulatory targets (Albrecht et al., 2001). The function of most CBL and CIPK proteins has not been determined. A putative calcium-dependent protein kinase (CDPK) is strongly downregulated as well (Table S4). Reports about the involvement of Ca²⁺ in cytokinin signal transduction are rare, but evidence for a role for Ca²⁺ signaling in cytokinin-induced cell division has accumulated in the moss *Funaria hygrometrica* (Saunders, 1994). Stimulation of Ca²⁺ channels by cytokinin and the significance of Ca²⁺ in cytokinin-mediated bud formation was also shown in *Physcomitrella* (Schumaker and Dietrich, 1998). Release of Ca²⁺ from internal stores is stimulated by inositol-1,4,5-triphosphate (IP₃), a second messenger of the

phosphatidylinositol (PI) pathway (Meijer and Munnik, 2003). The function of the PI pathway depends on recycling of the phospholipase substrate phosphatidylinositol-4,5-bisphosphate (PIP₂) from IP₃. Two genes encoding putative members of the phosphatidylinositol 3- and 4-kinase family (PI3,4K) were upregulated and one phosphatidylinositol-4-phosphate kinase (PIP4K) gene was downregulated (Table S4). These kinases catalyze the formation of intermediates during the recycling process. The regulation of the encoding genes suggests that cytokinins may regulate the level of active phospholipids. Consistent with a link between cytokinins and PI signaling is the recent finding that knockout mutants of the *Physcomitrella PLC1* gene have lost their cytokinin sensitivity (Repp et al., 2004). Moreover, several putative phospholipase genes were identified as cytokinin regulated (Lin et al., 2004). The inhibition of phospholipase D activity significantly reduces early responses to cytokinin in Arabidopsis and *Amaranthus* indicating that several links to phospholipid-based signaling may exist (Romanov et al., 2000, 2002).

#### Cytokinin-regulated genes associated with metabolism and signaling of other hormones

The most prevalent changes in gene transcript levels known to be associated with other plant hormones are those related to auxin, ethylene and gibberellin. Genes related to abscisic acid, jasmonate and salicylic acid were less affected.

Several auxin-regulated genes including members of all three major auxin response gene families, Aux/IAA, SAUR and GH3 are upregulated (Tables S4 and S5). Also the putative auxin transporter gene AUX1 (Bennett et al., 1996) is upregulated. In contrast, genes encoding the auxin response factor ARF8, the putative auxin transport protein EIR1 (Luschnig et al., 1998) and an auxin biosynthesizing nitrilase gene (Bartling et al., 1994) are downregulated by cytokinin (Table S4). Regulation of these genes may contribute to realize the multiple agonistic and antagonistic interactions between auxin and cytokinin (Coenen and Lomax, 1997).

Cytokinin acts partially through ethylene and regulation of ethylene synthesis by cytokinin has been demonstrated (Cary et al., 1995). More specifically, at least two 1-aminocyclopropane-1-carboxylic acid (ACC) synthases (ACS5 and ACS9) were shown to be stabilized by cytokinin (Chae et al., 2003; Vogel et al., 1998), indicating the relevance of post-transcriptional regulation in cytokinin–ethylene interaction. In our study, transcripts of two out of four detected ACS genes, ACS6 and ACS8, were upregulated and downregulated by cytokinin (Table S4). One (At1g77330) of five putative ACC oxidase genes responded to cytokinin with a decrease of transcript abundance (Table S4). Furthermore, increases in transcript abundance of genes encoding the ethylene receptors ETR2 and ERS2, the key transcription factor in the
ethylene signaling pathway EIN3, and several ERFs/EREBPs transcription factors were identified (Table S4). These findings are novel and suggest that transcriptional regulation of metabolic and signaling genes is also part of the regulatory links between cytokinin and ethylene.

Two gibberellin biosynthesis genes, GA20 oxidase and the 3β-hydroxylase gene GA4, are downregulated by cytokinin. The GA20 oxidase catalyzes the formation of the inactive gibberellin precursors GA<sub>20</sub> and GA<sub>9</sub>, which are then converted by 3β-hydroxylases to the active GA<sub>1</sub> and GA<sub>4</sub>, respectively (Hedden and Kamiya, 1997). On the contrary, two genes encoding negative regulators of gibberellin signal transduction, GAI and RGA (Dill and Sun, 2001), are upregulated (Table S4). A model for the influence of cytokinin on GA synthesis and signaling is shown in Figure 3. The model predicts that cytokinin reduces gibberellin activity by downregulation of gibberellin biosynthesis genes and upregulation of suppressors of GA-signaling genes. In accordance with this model, we have found a reduced seed dormancy, indicative of increased gibberellin activity, in cytokinin-deficient seeds (M. Riefler and TS, FU Berlin, Germany, unpublished data). Conversely, reduced activity of the gibberellin pathway promotes shoot meristem activity in Arabidopsis, similar to an enhanced cytokinin production (Hay et al., 2002). Earlier work has reported about antagonistic activities of cytokinin and gibberellin (Biddington and Thomas, 1977). Most recently, the GA response inhibitor SPINDLY was identified as a central regulator of their crosstalk (Greenboim-Wainberg et al., 2005).

Cytokinin-regulated genes involved in light signaling

Soon after the discovery of cytokinin, Miller and co-workers described similarities between cytokinin and light effects for

$\begin{aligned}& \text { GA precursor } \\& \text { GA20 oxidase } \\& \text { GA4 } \\& \text { CK } \\& \text { GAI } \\& \text { RGA }\end{aligned}$

$\begin{aligned}& \text { GA }_{20} / \text { GA }_{9} \\& \text { active GA }_{1} / \text { GA }_{4} \\& \text { GA dependent } \\& \text { reactions }\end{aligned}$

Figure 3. Influence of cytokinin on transcript abundance of genes of gibberellin metabolism and signaling. The model predicts that an increase in cytokinin concentration decreases the pool of biologically active gibberellin and reduces gibberellin signaling.

© Blackwell Publishing Ltd, *The Plant Journal*, (2005), **44**, 314–333

seed germination and leaf expansion of lettuce (Miller, 1956), and numerous reports have documented mutual influences of cytokinin and light on gene expression (Schmülling et al., 1997). Our study has shown that the *PHYA* gene is upregulated 3.3-fold by cytokinin, as well as the phytochrome kinase substrate *PKS1* gene (Table S4). PKS1 interacts with PHYA, PHYB and PKS2 and is phosphorylated by phytochrome (Fankhauser et al., 1999; Lariguet et al., 2003). PKS1 and PKS2 were shown to have a role in providing PHYA-signaling homeostasis in a discrete branch of light signaling, the PHYA-mediated very low fluence response (VLFR; Lariguet et al., 2003). Another light-regulating gene, *PAT1*, a positive regulator of phytochrome signal transduction (Bolle et al., 2000), was upregulated (Table S4). Furthermore, the *CLP8* gene, encoding a ubiquitin ligase that interacts with the negative regulator of photomorphogenesis COP1 (Hardtke et al., 2002), was upregulated. CLP8 may participate in proteasome mediated degradation of HY5, a positive regulator of photomorphogenesis (Hardtke et al., 2002). The overlap between cytokinin-regulated and light-regulatory genes implies shared regulatory activities between cytokinin and light. In agreement with this, altered light responses of cytokinin mutants have been reported (To et al., 2004; Riefler and Schmülling, unpublished data).

Cytokinin-regulated genes associated with metabolism and transport

Only very few metabolic genes were found among the immediate-early response genes, but among the genes that are regulated 120 min after cytokinin treatment are many that have a function in primary or secondary metabolism. Extensive transcriptional reprogramming of metabolism, as well as transport of diverse molecules, indicates that regulation of these processes is a function of cytokinin. The discovered genes include those with a known link to cytokinin function, such as the nitrate reductase (*NIA1*; Lu et al., 1992), an invertase-like gene (At1g35580; Ehneß and Roitsch, 1997) and a PEP carboxylase gene (Schmitt and Piepenbrock, 1992). There are several novel regulated genes with a previously unknown link to cytokinin. These include genes coding for a formate dehydrogenase (FDH), a pyruvate decarboxylase (PDC1), a putative fructokinase, a fructose-bisphosphate aldolase, a UDP-glucosyltransferase, and alcohol, aldehyde, malate and glycerine aldehyde dehydrogenases (Tables S4 and S5). In total, few genes involved in glycolysis, the TCA cycle, chloroplast biogenesis, the Calvin cycle and photorespiration were regulated by cytokinin (data not shown).

Several genes assigned to sugar metabolism and transport were repressed or induced. Two sucrose synthase genes, one of them being *SUS1* (Martin et al., 1993), and several genes for sugar transporters, including the H<sup>+</sup>/monosaccharide symporter STP1 (Stadler et al., 2003), were
upregulated. In contrast, the sucrose transporter gene SUC1 (Sauer and Stolz, 1994) was downregulated. Downregulation of a hexokinase gene (At4g29130) indicates that cytokinin might also interfere with sugar signaling (Table S4).

Several genes of nitrogen metabolism and transport were cytokinin regulated. The 24- and 15-fold upregulation of genes encoding a glutamine-dependent asparagine synthetase and a glutamate dehydrogenase, respectively, indicate an increased need for NH₄⁺ assimilation. Increased NH₄⁺ levels after cytokinin treatment are the likely consequence of enhanced NIA levels. The NIA gene transcript abundance was increased 2.6-fold after 120 min cytokinin treatment (Table S4). It is noteworthy that three high-affinity nitrate transporter genes, NRT2.1, NRT2.3 and NRT2.6, are strongly downregulated after 120 min. Similarly, the ammonium transporter genes AMT1;1, AMT1;2 and AMT1;3 were repressed up to one-third of their original levels (Tables S4 and S5). Collectively, these data support the earlier notion that cytokinin plays an important role in regulating N-utilization and that early responses to cytokinin and changed nitrogen availability overlap in part (Kiba et al., 1999; Sakakibara, 2003; Wang et al., 2003).

Beside genes encoding nitrate transporters (see above), those encoding low affinity chloroplast phosphate transporters PHT2 and PT2 (Daram et al., 1999; Rausch et al., 2004; Versaw and Harrison, 2002) and several sulfate transporters (SULTR1;1, SULTR1;2 and SULTR4;2) are also downregulated to about one-quarter of their original levels. In contrast, the auxin-reactive potassium transporter KAT1 gene (Philippar et al., 2004) is strongly (approximately 8.5-fold) upregulated (Tables S4 and S5).

Cytokinin-deficient plants show specific alterations of transcript abundance

Next we analyzed the transcript profile of cytokinin-deficient 35S:CKX1 overexpressing seedlings and compared it to the profile of wild-type plants. The correlation coefficients of the two independent experiments indicate good reproducibility and are shown in Table S1. One hundred and thirty genes were constitutively upregulated in cytokinin-deficient seedlings and 341 were downregulated (Figure 1; Tables S6 and S7). Of the 667 genes that were upregulated 120 min after cytokinin treatment, only 18 were constitutively downregulated in cytokinin-deficient seedlings (Figure 1). A similar small overlap exists for the opposite regulation: of the 1169 cytokinin-downregulated genes, only 10 are upregulated in 35S:CKX1 overexpressing plants (Figure 1). This finding may lead to the conclusion that a totally different set of genes reacts to long-term cytokinin deficiency than to cytokinin induction. However, detailed examination of the raw data shows that, for example, the extent of downregulation of most of the immediate-early cytokinin response genes was very close to the exclusion criteria used in this study.

The over-represented classes of upregulated and downregulated genes were in general those that were also found after cytokinin treatment and included transcriptional control, signal transduction, stress response and different metabolic gene classes (Table 2).

Changes in transcript abundance of several novel genes may be relevant for the establishment of the cytokinin-deficiency syndrome. For example, the transcript level of KNAT1 is reduced in cytokinin-deficient seedlings to half of its wild-type level (Table S6). Phenotypic similarities between plants overexpressing KNAT1 and cytokinin overproducers have suggested a functional link between this transcription factor and the hormone (Chuck et al., 1996; Rupp et al., 1999). One other downregulated gene is SCARECROW (SCR). SCR is, beside RGA and GAI, the third cytokinin-regulated prototypical member of the GRAS family. SCR has important roles in regulating the radial organization of the Arabidopsis root and is required to maintain the activity of the quiescent center and stem cell identity of the root meristem (Sabatini et al., 2003). Given the regulatory activities of cytokinin in determining the size of the root meristem (Werner et al., 2003), it may be that this regulatory pathway also involves modulation of SCR transcript level. Of note is also the downregulation of PICKLE (PKL) in cytokinin-deficient plants. PKL encodes a CHD3-chromatin-remodeling factor that is required to repress embryonic traits during post-embryonic development and was shown to mediate a subset of GA-dependent responses (Henderson et al., 2004; Ogas et al., 1997).

Interestingly, numerous genes coding for translational proteins were affected in cytokinin deficient plants (Table 2). Arabidopsis has 249 genes encoding ribosomal proteins, most of them belonging to small gene families (Barakat et al., 2001). Ribosomal proteins stabilize the structure of ribosomes and are involved in controlling protein synthesis. Some ribosomal proteins may play roles in regulating cell growth, proliferation and death (Maguire and Zimmermann, 2001; Naora and Naora, 1999). Not all ribosomal genes were regulated simultaneously by cytokinin although the proteins are functional in one distinct process. This may lead to heterogeneous populations of ribosomal proteins. Heterogeneity of ribosome composition may contribute to context-dependent efficiency of translation (Szick-Miranda and Bailey-Serres, 2001). Hence, altered relative abundance of ribosomal proteins may lead to different translational efficiency in cytokinin-deficient plants.

Noteworthy is the upregulation of four of the 15 members of the lipid transfer proteins (LTP) gene family. LTPs are defined by their ability to facilitate transfer of phospholipids between membranes *in vitro* and it has been suggested that they play roles in cutin formation, embryogenesis, biotic and abiotic stress responses (Arondel et al., 2000; Kader, 1996).

This dataset was primarily used for comparison with the data generated after exogenous cytokinin addition.
reasoned that genes involved in cytokinin-sensitive processes may be affected in an opposite fashion under cytokinin-deficiency and following cytokinin treatment. In the following sections, we report the outcome of this comparison.

### Data comparison and discovery of cytokinin-regulated processes using MAPMAN

The MAPMAN software allows one to view microarray data in a biocentric manner. The visual display of MAPMAN generated data organizes genes in contextual groups (called bins), which facilitates the discovery of general trends in the displayed blocks (Thimm *et al.*, 2004). In the current study, we used MAPMAN with the data of all conditions presented above on the one hand to visualize gene expression data for complex processes, e.g. cytokinin metabolism and signaling, and on the other hand as a discovery tool to search for novel cytokinin-regulated processes.

### Regulation of cytokinin-related genes

The reactions of all known Arabidopsis genes that have a role in cytokinin metabolism or signaling were compared under the different experimental conditions. The result is shown as a MAPMAN sketch combined with a schematic view of cytokinin metabolism and signaling in Figure 4.

All cytokinin-synthesizing *IPT* genes are weakly expressed and under most conditions expression of only two or three of these genes was detected (Figure 4). More *IPT* genes are detected in *CKX1* overexpressing seedlings, which may indicate a modest compensatory upregulation of cytokinin synthesis in response to constitutive cytokinin deficiency. No strong changes were induced by cytokinin treatment, except for *IPT4*, which was significantly downregulated after cytokinin treatment of the *CKX1* transgenics. *IPT2* and *IPT9* were also regulated under this condition, however the products of these genes do not synthesize free cytokinin but presumably modify bases in tRNAs (Kakimoto, 2001; Takei *et al.*, 2001). Only five of the seven *CKX* genes are present on the ATH1 GeneChip; *CKX6* and *CKX7* are missing. Only *CKX5* was rapidly upregulated by cytokinin, on the average 1.9-fold after 15 min and 3.8-fold after 120 min (Figure 4; Table S4). Consistent with the microarray data, GUS staining was strongly increased in $P_{CKX5}:GUS$ plants after cytokinin treatment (Köllmer, 2004).

In Arabidopsis, a purine permease (*PUP*) gene family comprising 15 genes has been described and some members may be involved in cytokinin transport (Bürkle *et al.*, 2003). The abundance of the *PUP4* transcript was lowered by cytokinin, but none of the other detected *PUP* transcripts was significantly altered by cytokinin (Table S5). As the molecule transported by a transporter protein is often a regulator of the corresponding gene (Price *et al.*, 2004), this may indicate a role for *PUP4* in cytokinin transport.

Among the three cytokinin receptor genes, *AHK2* showed the most significant change, i.e. a strong downregulation 15 min after cytokinin treatment of *CKX1* transgenic seedlings. The *CRE1/AHK4* transcript level was lower in *CKX1* seedlings compared with wild-type seedlings, which could contribute to the dampened cytokinin response of the former (see above). Significant changes were found for other histidine (His) kinase genes (Figure 4). *CKI2*, which codes for a His-kinase lacking ligand-binding and membrane-spanning domains but conferring cytokinin-independent growth to callus tissue upon overexpression (Kakimoto, 1996), was strongly downregulated 15 and 120 min after cytokinin treatment of wild type and its steady-state level was lowered in *CKX* plants. The cytokinin-responsiveness may indicate that this gene has a specific function in responding to changes in ligand concentration for the related cytokinin receptors.

Genes coding for Arabidopsis histidine phosphotransmitter proteins (AHPs) or type B ARRs were not regulated under our experimental conditions, which is consistent with previous results (Imamura *et al.*, 1999; Kiba *et al.*, 1999; Figure 4). In contrast and as expected, a rapid regulation was found for most type A *ARR* genes. *ARR3* and *ARR8* respond with a delay confirming the individually different kinetics of regulation of these genes (D’Agostino *et al.*, 2000). Consistent with a role as indicators of cytokinin status, most of the type A *ARR* genes are strongly downregulated to one-tenth to one-half of their wild-type levels in the cytokinin-deficient *CKX1* line, but they are still inducible by cytokinin (Figure 4).

Surprisingly, some Arabidopsis pseudo response regulator genes (*APRR* genes), which are not thought to be functionally linked to cytokinins, were regulated by the hormone (Figure 4). *APRR* proteins belong, like type B response regulators, to the Myb superfamily of transcription factors but they lack the canonical phospho-accepting Asp residue. *APRR* proteins modulate various developmental processes in plants, including circadian rhythms and other light responses (Mizuno, 2004). The best studied family member is *APRR1/TOC1*, which activates the *CCA1* and *LHY* genes (Alabadi *et al.*, 2001; Mas *et al.*, 2003). Conversely, *CCA1* and *LHY* repress the transcription of *APRR1/TOC1* by directly binding to its promoter region. Interestingly, the *CCA1* gene transcript is strongly downregulated under cytokinin-deficiency (Table S6). Analysis of other *APRR* gene family members, including the cytokinin-regulated *APRR5* and *APRR7* genes, has also shown functions in regulating circadian rhythm and red-light sensitivity (Mizuno, 2004; Yamamoto *et al.*, 2003). Together, the transcriptional regulation of *APRR* genes might contribute to co-regulation of light-dependent processes by cytokinin. Consistent with a role of cytokinin in modulating circadian rhythm is the induction of the *CONSTANS-LIKE1 (COL1)* gene (Table S4). Enhanced *COL1* expression shortens the period of two distinct circadian rhythms (Ledger *et al.*, 2001).

© Blackwell Publishing Ltd, *The Plant Journal*, (2005), **44**, 314–333
326 Wolfram G. Brenner et al.

ATP/ADP/AMP + DMAPP

IPT

CKX

Transport across membranes

Cytokinin receptors and related kinases

AHP

type B ARR

type A ARR

pseudo ARR

Figure 4. Regulation of cytokinin metabolism and cytokinin signaling genes by cytokinin.
Genes associated with functions in cytokinin metabolism or signaling are listed on the left side and a schematic overview of cytokinin metabolism and signaling is shown on the right side. The MAPMAN output for each of the five comparisons, indicated above the color bars, was saved and the color bars were composed and aligned using CorelDraw. Changes in transcript levels are indicated by color codes as shown on the color gradient with blue indicating an increase and red indicating a decrease of transcript abundance. A color scale is included in the figure, the colors saturate at eightfold changes. Genes called absent by Affymetrix software and uncertain genes are shown in gray. A star indicates a gene that is targeted by an ambiguous Affymetrix probeset. Abbreviations: AHK, Arabidopsis histidine kinase gene; ARR, Arabidopsis response regulator gene; CKI, CYTOKININ INDEPENDENT; CKX, cytokinin oxidase/dehydrogenase gene; CRE1, CYTOKININ RESPONSE 1; DMAPP, dimethylallyl diphosphate; EIN, ERS, ETR, PHY, genes encoding ethylene and light receptors, respectively; IPT, isopentenyl transferase gene; PUP, purine permease gene.

Opposite regulation of genes after cytokinin treatments and conditions of cytokinin deficiency

To search for novel cytokinin-regulated metabolic processes, we compared the data of cytokinin-induction after 120 min and those obtained from cytokinin-deficient plants. We looked first for strong and opposite regulated regions in the bins shown by the MAPMAN software, then extracted the genes belonging to a particular region and subsequently built new maps with a more detailed view of the particular process. Figure S1 shows that distinct reaction patterns of several bins were found. For example, a large portion of the genes that are classified as light-related in MAPMAN are upregulated in wild type by cytokinin and downregulated under cytokinin deficiency. Likewise, many genes related to auxin, ethylene and different ion transport gene families are upregulated or downregulated by cytokinin. In contrast, other bins containing for example various anion transporter genes ('misc. anions') or genes involved in starch degradation appear to be less responsive to cytokinin (Figure S1 and data not shown).

Further detailed analysis of MAPMAN sketches identified two metabolic processes for which an opposite regulation of central steps was found after cytokinin induction and under cytokinin deficiency. One of these processes is trehalose metabolism, which includes trehalose-6-phosphate (T6P) as a precursor and biologically active molecule (Figure 5). T6P was shown to be essential for carbon utilization in Arabidopsis (Schlüepmann et al., 2003). Plants with a low T6P content grow slowly and accumulate respiratory intermediates, whereas plants with a high T6P content show the

opposite phenotype (Schlüepmann et al., 2003). We identified a number of trehalose-6-phosphate synthase (TPS) gene family members that were upregulated upon cytokinin treatment (Figure 5). In contrast, in cytokinin-deficient plants a number of TPS genes is downregulated. The strongest regulation, which was opposite under both experimental conditions, was found for *TPS8*. In addition, one member of the catabolizing trehalose-6-phosphate phosphatase (TPP) gene family was strongly downregulated by cytokinin (Figure 5) and a trehalase gene was among the immediate-early response genes (Table 1). The gene expression data suggest a lower T6P content in cytokinin-deficient plants. In view of the proposed role of T6P in plant growth regulation, the slow shoot growth of cytokinin-deficient plants and the fact that cytokinin-deficiency causes a reduction of sink strength in the shoot (K. Holst and T. Werner, FU Berlin, Germany, unpublished data), we hypothesize that control of T6P homeostasis by cytokinin may be causally involved in establishing the cytokinin deficiency syndrome. An altered growth response of cytokinin-deficient seedlings on trehalose-containing medium is consistent with this proposal (K. Holst and T. Werner, personal communication). Another process identified by the described procedure in which many of the genes are regulated by cytokinin is the metabolism of valine and leucine is described in Figure S2.

### Conclusions

The data presented here describe an extensive assessment of the Arabidopsis cytokinin transcriptional response at a

**Figure 5. Regulation of genes associated with trehalose metabolism by cytokinin.**

A schematic overview of trehalose metabolism is shown. Raw data of the experiments were used for the analysis. Changes of transcript levels shown in the display were obtained from seedlings treated for 120 min with cytokinin (Col-0 BA 120) and cytokinin-deficient seedlings (CKX1).

© Blackwell Publishing Ltd, *The Plant Journal*, (2005), **44**, 314–333
genome-wide level. A large set of novel immediate-early and delayed cytokinin response genes has been discovered and shows an unprecedented view of the regulatory activities of cytokinin. Earlier publications analyzed long-term changes in tissue culture (Che *et al.*, 2002) or transcriptional changes after 3–24 h of cytokinin treatment (Che *et al.*, 2002; Hoth *et al.*, 2003; Kiba *et al.*, 2004, 2005; Schäfer and Schmülling, 2002; Schmülling *et al.*, 1997; Zimmermann *et al.*, 2004). The work that is closest to ours is the one by Rashotte *et al.* (2003), who used the 8k GeneChip and older plant material.

One significant outcome of this study is the finding that transcriptional regulators are predominant among both the immediate-early and the delayed genes. This suggests a scenario in which a stepwise activation of different cytokinin response pathways is initiated by upregulation or downregulation of transcriptional control factors, presumably in a context-dependent manner. It also suggests that cytokinin signaling must not necessarily involve exclusively type B ARR proteins as transcriptional activators, but may be mediated through other transcription factors as well. Indeed, it has been shown that cytokinin-signaling phosphotransmitter proteins interact with other transcriptional regulators, e.g. the TCP-domain containing putative transcription factor TCP10 (Suzuki *et al.*, 2001). To our knowledge, none of the immediate-early transcription factors identified has previously been linked to cytokinin responses. They are now clearly major candidates for a functional assessment of their involvement in mediating cytokinin downstream responses.

Another common theme is crosstalk with other signals. Crosstalk may be achieved through pathways converging on the same transcriptional regulators or signaling component or through regulation of metabolic enzymes and/or transporters, finally affecting the active concentrations of other signaling molecules. In keeping with this, some of the early cytokinin-regulated transcription factors are involved in signaling pathways of light, auxin, ethylene or gibberellin, or regulate metabolism or transport of nitrate and sugar. Several previously unknown relations were identified: for example, the influence of cytokinin on the transcript levels of gibberellin metabolizing and signaling genes (Figure 2). A future task will be to understand the network and link gene regulation to specific processes.

The analyses have confirmed close connections between cytokinin and C- and N-utilization in plants and indicated novel metabolic and transport processes that may be under the influence of cytokinin. Particularly, by comparing the datasets of regulated genes after cytokinin addition and under cytokinin depletion, we were able to identify opposite regulation of some metabolic steps, e.g. in trehalose metabolism. These data have identified novel genes and molecules to test experimentally the hypothesis that an important part of growth control by cytokinin is realized through regulating sink-source transitions and does not involve necessarily the better understood direct control of cell cycling.

The identification of plastid genes among the immediate-early and delayed cytokinin response genes is an important result. It was shown soon after the discovery of cytokinin that the hormone is able to retard loss of leaf chlorophyll during senescence (Richmond and Lang, 1957), indicating that plastids are a target of cytokinin action. Numerous later reports have shown that plastid development is affected by cytokinin in many different ways, including a positive role of cytokinin in regulating plastid division (Reutter *et al.*, 1998). Plastids contain a specific set of cytokinin metabolites (Benkova *et al.*, 1999) and four nuclear-encoded cytokinin-synthesizing IPT proteins have been localized in the plastids of Arabidopsis (Kasahara *et al.*, 2004). However, the molecular basis for the regulatory activities of cytokinin in plastids has remained enigmatic. Plastid genes that react within minutes to changes in cytokinin concentration are an important tool for further analyses. We hypothesize that the plastids may be partially autonomous with respect to their cytokinin response. This is an exciting possibility as the plastid-related genome of cyanobacteria contains sequences with similarity to cytokinin metabolic genes (Kakimoto, 2003; Schmülling *et al.*, 2003), indicating cytokinin-related plastid functions may have been conserved during evolution.

Of course a change in transcript abundance can be only a first indication but not proof of a functional relevance as it may or may not be correlated to changes in the level and/or activity of the corresponding protein. However, these data provide the framework for further investigations of molecular mechanisms used in plants to interpret the cytokinin signal and to unravel the molecular mechanisms that lead to the many different outputs generated by the hormone.

## Experimental procedures

### Plant material, culture conditions and treatments

Arabidopsis thaliana ecotype Col-0 wild-type plants and transgenic homozygote 35S:CKX1 plants (Werner *et al.*, 2003) were used. The seeds (ca 5 mg) were incubated in liquid Murashige and Skoog (MS) Medium (1/2 MS salts; Sigma Aldrich Chemie GmbH, Munich, Germany; 1 g l⁻¹ sucrose, 0.5 g l⁻¹ MES, pH 5.7) in Petri dishes (diameter 95 mm) sealed with Leukopor (Beiersdorf, Hamburg, Germany) for two days at 4°C. Subsequently, the Petri dishes were transferred to a growth cabinet (Percival AR66L; Percival Scientific, Perry, IA, USA) at 24°C and light/dark periods of 16/8 h. The seedlings were grown for 5 to 7 days to stage 1.00 (Boyes *et al.*, 2001).

6-Benzyladenine (BA) was dissolved in 1 M KOH at a concentration of 1 mg ml⁻¹. From this solution, a 50 μM stock was prepared and titrated to pH 7 with HCl. This stock was directly added to the seedling cultures to give a final concentration of 5 μM BA in 1.13 mM KCl.

The timing of the treatments was such that the plants were harvested between 6 and 8 h after the onset of light. Col-0 seedlings were treated for 15 and 120 min before sampling; for 35S:CKX1 transgenic seedlings, only a 15-min treatment was performed. Control samples were treated for 2 h with 1.13 mM KCl in liquid MS. The seedlings were harvested by pouring the contents of a Petri dish

© Blackwell Publishing Ltd, *The Plant Journal*, (2005), **44**, 314–333
onto a filter paper placed on a vacuum manifold and removing the liquid by suction. The seedlings were then gently transferred into a tube and immediately frozen in liquid nitrogen.

### RNA preparation

Total RNA was extracted from seedlings with the TRIzol method. TRIzol reagent (38% phenol, 0.8 M guanidinium thiocyanate, 0.4 M ammonium thiocyanate, 0.1 M sodium acetate, pH 5, 5% glycerol) was made as described in the GIBCO TRIzol manual of Invitrogen (Invitrogen, Karlsruhe, Germany). RNA was precipitated by the modified salt precipitation method for proteoglycan and polysaccharide contamination as described in the GIBCO troubleshooting guide.

The total RNA was further purified using RNeasy mini-columns including the on-column DNase digestion as described in appendix D of the Qiagen RNeasy Mini Handbook (3rd edition; Qiagen). The purified RNA was quantified by 260/280/230 nm UV light absorption with a Nanodrop ND-1000 spectrophotometer (Nanodrop Technologies, Wilmington, DE, USA). An additional quality control was performed using non-denaturing agarose gel electrophoresis.

### Northern blot analysis

Twenty-five micrograms RNA was separated in a denaturing 1.5% agarose-formaldehyde gel, transferred to nylon membranes (Amersham Hybond N; Amersham, Little Chalfont, UK) and hybridized with radioactive-labeled DNA probes. Hybridization was performed in phosphate buffer with 7% SDS at 68°C. The lowest stringency wash was done in 0.2 × SSC, 0.1 % SDS at 65°C. For radioautography, blots were exposed with BioMax MS films (Kodak GmbH, Stuttgart, Germany) at –70°C for 1–3 days. As a control for loading, blots were rehybridized with an *actin* 2 probe. Quantification of signals was done on an integrative densitometer CD-50 (Desaga, Nürnbrecht, Germany).

### Semiquantitative RT-PCR

An RT-PCR kit (One Step RT-PCR kit) was used according to the manufacturer’s instructions (Qiagen). The amplification procedure was slightly modified by using 30 sec 94°C for denaturing, 30 sec 57°C for annealing and finally 2 min 72°C for the elongation step. The following primers were used: At4g26150, 5′-GATACCCATATCTCCCCAACCTC-3′ and 5′-AACCATTCCGTGCGATAGAG-3′; At2g18300, 5′-AAGACAAGGGGAAGA GCAAAAGAC-3′ and 5′-TTA-GTAATGAAAACCGAGGCAGA-3′; At4g16780, 5′-CGA GAAAGAC-GATCTGGGTCTA-3′ and 5′-CTAGGACGAAGAGCGTCAAAG-3′; At4g37790, 5′-TGATGATTCATGCAACACAGGT-3′ and 5′-ACATG-CTGCAGAA GGATTAGTG-3′; and At5g09810 (*actin*), 5′-TACAA-CGAGCTTCGTGTTGC-3′ and 5′-GATTGATCCTCCGATCCAGA-3′. Different numbers of cycles were performed for the different genes to avoid saturation of the amplification reaction as follows: At4g26150, 30 cycles; At2g18300, 25 cycles; At4g16780 27 cycles; At4g37790, 25 cycles; and actin, 23 cycles. Ten microliters of the reaction was loaded on a 1% agarose gel, electrophoretically separated and stained with ethidium bromide. Each experiment was repeated at least twice with independent biological replicates.

### cRNA synthesis and microarray hybridization

cDNA was synthesized from 5 μg of purified total RNA with SuperScriptII (Invitrogen), and a mixture of RNase H, *E. coli* DNA polymerase I, and *E. coli* DNA ligase (Invitrogen). The cDNA was purified using the QIAquick PCR purification procedure (Qiagen). The double-stranded cDNA was used as a template in the Bioarray High Yield Transcript Labeling reaction (Enzo, Farmingdale, NY, USA) resulting in biotin-labeled cRNA. The labeled cRNA was purified using the RNeasy cleanup protocol (Qiagen) and quantified by absorbance at 260 nm. Twenty micrograms of biotin-labeled cRNA was fragmented for 35 min at 94°C in fragmentation buffer (40 mM Tris-acetate, pH 8.1, 125 mM KOAc, and 30 mM MgOAc). Staining, hybridization, scanning and data pre-processing was performed by the RZPD Deutsches Ressourcenzentrum für Genomforschung GmbH (Berlin, Germany). Normalized raw expression values were obtained as tables in Microsoft Excel format. Other data types were obtained in Affymetrix standard formats. The data used in this paper are available through ArrayExpress (http://www.ebi.ac.uk/arrayexpress/) under the accession number E-MEXP-344.

### Calculations

MAS 5.0 normalized expression values were obtained as Excel worksheets from the RZPD Deutsches Ressourcenzentrum für Genomforschung GmbH and were taken for all subsequent procedures. Expression ratios were calculated separately for each repetition. The mean values of the single-repetition ratios were subsequently calculated. Additionally, the expression calls of the Affymetrix MAS 5.0 software (present, P, marginal, M, or absent, A) were computed to give expression classes as follows: a gene was classified as present (P) when the expression call was P for all four gene chips; a gene was classified as ‘turning from present to absent’ (P → A) when the expression calls were P for both gene chips of the reference samples and either A or M for both gene chips of the experiment samples; a gene was classified as ‘turning from absent to present’ (A → P) when the expression calls were either A or M for both gene chips of the reference samples and P for both replicates of the experiment samples. All genes that did not match these definitions were classified as ‘absent’ and eliminated from further analysis.

For p-value calculations, the raw data from the .cel files were used. The data were RMA (robust multichip averaging) normalized (Irizarry et al., 2003) using the Affy package and analyzed using the Limma package of the bioconductor software (Gentleman et al., 2004). A linear model with coefficients representing each treatment group was fitted for each gene and contrasts of interest were made. The duplicateCorrelation function was used to account for the paired nature of the data. This package uses an empirical Bayes approach to moderate t-statistics (Irizarry et al., 2003). A false discovery rate (FDR) correction for multiple testing was used (Benjamini and Hochberg, 1995). The biological variance between the replicates reduced the significance of this statistical approach.

To increase the chance of detecting differentially expressed genes under these circumstances, transcripts with a ratio of induction of >1.8 in both replicates and either a P or an A → P classification were regarded as upregulated. Transcripts with a ratio of induction of <1/1.8 in both replicates and either a P or a P → A classification were regarded as downregulated. To provide annotation data in the tables, the complete list of annotations was obtained from the Affymetrix web site as tab-delimited text, imported into Excel and mapped to the expression data.

### Usage of MAPMAN

The normalized raw data obtained from the RZPD Deutsches Ressourcenzentrum für Genomforschung GmbH were used for the

© Blackwell Publishing Ltd, *The Plant Journal*, (2005), **44**, 314–333

calculations to generate input tables for MAPMAN. In detail, after importing the normalized raw data into an Excel sheet, ratios were calculated. To avoid problems with division by zero, a gene with a denominator of 0 was flagged invalid and excluded from further operations. Next, absence or presence of a gene was determined using the MAS 5.0 expression call. Genes with a ratio of ≥1 and an absent expression call in the data serving as the numerator or with a ratio of <1 and an absent expression call in the data serving as the denominator were flagged invalid. Next, the log ratio was calculated from the remaining valid genes. Up to this point, the procedure was performed for both replicates individually. Finally, the average of both log ratios was calculated for those genes that were valid in both replicates. These data were exported into a new Excel sheet containing two columns: the first containing Affymetrix identifiers and the second containing the averaged log ratios or the invalid flags. The invalid genes are displayed with a gray box in MAPMAN. For this study, we used a local installation of MAPMAN version 1.4.2. The software is available at http://gabi.rzpd.de/projects/MapMan/.

### Acknowledgements

We thank Oliver Thimm and Yves Gibon for the installation of the MAPMAN software prior to publication and an introduction to it. We thank as well Matthew Hannah for carrying out the bioinformatics. We acknowledge Tomáš Werner, Alexander Heyl and Michael Riefler for critical reading of the manuscript. This work was supported by grants of the Deutsche Forschungsgemeinschaft (Sfb 429 and Arabidopsis Functional Genomics Network), the EU project Compendium of Arabidopsis gene expression (CAGE) (project no. QLK3-CT-2002-02035) and the Russian Federation of Basic Research NN 03-04-04001/04-04-49120, INTAS N 01-0602.

### Supplementary Material

The following supplementary material is available for this article online:

- Figure S1. Groups of functionally related genes and their regulation by cytokinin.
- Figure S2. Regulation of genes of valine and leucine metabolism by cytokinin.
- Table S1 Correlation coefficients between the replicates of each condition (a), and of the pairs used for ratio calculations (b). Present genes in single experiments (c) and in both replicates for a condition (d)
- Table S2 Immediate-early cytokinin response genes of Arabidopsis
- Table S3 Immediate-early cytokinin response genes in cytokinin-deficient Arabidopsis plants
- Table S4 Delayed cytokinin response genes of Arabidopsis
- Table S5 Delayed cytokinin response genes of Arabidopsis
- Table S6 Changes in gene expression in cytokinin-deficient Arabidopsis plants
- Table S7 Changes in gene expression in cytokinin-deficient Arabidopsis plants

This material is available as part of the online article from http://www.blackwell-synergy.com

### References

Alabadi, D., Oyama, T., Yanovsky, M.J., Harmon, F.G., Mas, P. and Kay, S.A. (2001) Reciprocal regulation between TOC1 and LHY/CCA1 within the Arabidopsis circadian clock. Science, 293, 880–883.

Albrecht, V., Ritz, O., Linder, S., Harter, K. and Kudla, J. (2001) The NAF domain defines a novel protein-protein interaction module conserved in Ca2+-regulated kinases. EMBO J. 20, 1051–1063.

Aravind, L. and Koonin, E.V. (1999) Gleaning non-trivial structural, functional and evolutionary information about proteins by iterative database searches. J. Mol. Biol. 287, 1023–1040.

Arondel, V., Vergnolle, C., Cantrel, C. and Kader, J.C. (2000) Lipid transfer proteins are encoded by a small multigene family in Arabidopsis thaliana. Plant Sci. 157, 1–12.

Barakat, A., Szick-Miranda, K., Chang, I.F., Guyot, R., Blanc, G., Cooke, R., Delseny, M. and Bailey-Serres, J. (2001) The organization of cytoplasmic ribosomal protein genes in the Arabidopsis genome. Plant Physiol. 127, 398–415.

Bartling, D., Seedorf, M., Schmidt, R.C. and Weiler, E.W. (1994) Molecular characterization of two cloned nitrilases from Arabidopsis thaliana: key enzymes in biosynthesis of the plant hormone indole-3-acetic acid. Proc. Natl Acad. Sci. USA, 91, 6021–6025.

Benjamini, Y. and Hochberg, Y. (1995) Controlling the false discovery rate – a practical and powerful approach to multiple testing. J. R. Stat. Soc. B Met., 57, 289–300.

Benkova, E., Witters, E., Van, D.W., Kolar, J., Motyka, V., Brzobohaty, B., Van, O.H.A. and Machackova, I. (1999) Cytokinins in tobacco and wheat chloroplasts. Occurrence and changes due to light/dark treatment. Plant Physiol. 121, 245–251.

Bennett, M.J., Marchant, A., Green, H.G., May, S.T., Ward, S.P., Millner, P.A., Walker, A.R., Schulz, B. and Feldmann, K.A. (1996) Arabidopsis AUX1 gene: a permease-like regulator of root gravitropism. Science, 273, 948–950.

Biddington,N.L. and Thomas,T.H. (1977) Interactions of abscisic acid, cytokinin and gibberellin in the control of betacyanin synthesis in seedlings of Amaranthus caudatus. Physiol. Plant. 40, 312–314.

Bilyeu, K.D., Cole, J.L., Laskey, J.G., Riekhof, W.R., Esparza, T.J., Kramer, M.D. and Morris, R.O. (2001) Molecular and biochemical characterization of a cytokinin oxidase from maize. Plant Physiol. 125, 378–386.

Bolle, C., Koncz, C. and Chua, N.H. (2000) PAT1, a new member of the GRAS family, is involved in phytochrome A signal transduction. Genes Dev. 14, 1269–1278.

Boyes, D.C., Zayed, A.M., Ascenzi, R., McCaskill, A.J., Hoffman, N.E., Davis, K.R. and Görlach, J. (2001) Growth stage-based phenotypic analysis of Arabidopsis: a model for high throughput functional genomics in plants. Plant Cell, 13, 1499–1510.

Brandstatter, I. and Kieber, J.J. (1998) Two genes with similarity to bacterial response regulators are rapidly and specifically induced by cytokinin in Arabidopsis. Plant Cell, 10, 1009–1020.

Bürkle, L., Cedzich, A., Döpke, C., Stransky, H., Okumoto, S., Gillissen, B., Kühn, C. and Frommer, W.B. (2003) Transport of cytokinins mediated by purine transporters of the PUP family expressed in phloem, hydathodes, and pollen of Arabidopsis. Plant J. 34, 13–26.

Cary, A.J., Liu, W. and Howell, S.H. (1995) Cytokinin action is coupled to ethylene in its effects on the inhibition of root and hypocotyl elongation in Arabidopsis thaliana seedlings. Plant Physiol. 107, 1075–1082.

Chae, H.S., Faure, F. and Kieber, J.J. (2003) The eto1, eto2, and eto3 mutations and cytokinin treatment increase ethylene biosynthesis in Arabidopsis by increasing the stability of ACS protein. Plant Cell, 15, 545–559.

Che, P., Gingerich, D.J., Lall, S. and Howell, S.H. (2002) Global and hormone-induced gene expression changes during shoot development in Arabidopsis. Plant Cell, 14, 2771–2785.

Chipman, D.M. and Shaanan, B. (2001) The ACT domain family. Curr. Opin. Struct. Biol. 11, 694–700.

Chuck, G., Lincoln, C. and Hake, S. (1996) KNAT1 induces lobed leaves with ectopic meristems when overexpressed in Arabidopsis. *Plant Cell*, 8, 1277–1289.

Coenen, C. and Lomax, T.L. (1997) Auxin-cytokinin interactions in higher plants: old problems and new tools. *Trends Plant Sci.*, 2, 351–356.

Colijn, C.M., Sijmons, P., Mol, J.M.M., Kool, A.J. and Nijkamp, H.J.J. (1982) Light and benzylaminopurine induce changes in ultrastructure and gene expression in plastids of *Petunia hybrida* cell cultures. *Curr. Genet.* 6, 129–135.

Colón-Carmona, A., Chen, D.L., Yeh, K.C. and Abel, S. (2000) Aux/IAA proteins are phosphorylated by phytochrome in vitro. *Plant Physiol.* 124, 1728–1738.

D'Agostino, I.B., Deruere, J. and Kieber, J.J. (2000) Characterization of the response of the *Arabidopsis* response regulator gene family to cytokinin. *Plant Physiol.* 124, 1706–1717.

Daram, P., Brunner, S., Rausch, C., Steiner, C., Amrhein, N. and Bucher, M. (1999) Pht2;1 encodes a low-affinity phosphate transporter from *Arabidopsis*. *Plant Cell*, 11, 2153–2166.

Dill, A. and Sun, T.P. (2001) Synergistic derepression of gibberellin signaling by removing RGA and GAI function in *Arabidopsis thaliana*. *Genetics*, 159, 777–785.

Dominov, J.A., Stenzler, L., Lee, S., Schwarz, J.J., Leisner, S. and Howell, S.H. (1992) Cytokinins and auxins control the expression of a gene in *Nicotiana plumbaginifolia* cells by feedback regulation. *Plant Cell*, 4, 451–461.

Ehneß, R. and Roitsch, T. (1997) Co-ordinated induction of mRNAs for extracellular invertase and a glucose transporter in *Chenopodium rubrum* by cytokinins. *Plant J.* 11, 539–548.

Fankhauser, C., Yeh, K.C., Lagarias, J.C., Zhang, H., Elich, T.D. and Chory, J. (1999) PKS1, a substrate phosphorylated by phytochrome that modulates light signaling in *Arabidopsis*. *Science*, 284, 1539.

Gentleman, R.C., Carey, V.J., Bates, D.M. *et al.* (2004) Bioconductor: Open software development for computational biology and bioinformatics. *Genome Biol.* 5, R80.

Giacometti, S., Camoni, L., Albumi, C., Visconti, S., De Michelis, M.I. and Aducci, P. (2004) Tyrosine phosphorylation inhibits the interaction of 14-3-3 proteins with the plant plasma membrane H+-ATPase. *Plant Biol.* 6, 422–431.

Greenboim-Wainberg, Y., Maymon, I., Borochov, R., Alvarez, J., Olszewski, N., Ori, N., Eshed, Y. and Weiss, D. (2005) Cross talk between gibberellin and cytokinin: the *Arabidopsis* GA response inhibitor SPINDLY plays a positive role in cytokinin signaling. *Plant Cell*, 17, 92–102.

Hagen, G. and Guilfoyle, T. (2002) Auxin-responsive gene expression: genes, promoters and regulatory factors. *Plant Mol. Biol.* 49, 373–385.

Hardtke, C.S., Okamoto, H., Stoop, M.C. and Deng, X.W. (2002) Biochemical evidence for ubiquitin ligase activity of the *Arabidopsis* COP1 interacting protein 8 (CIP8). *Plant J.* 30, 385–394.

Hass, C., Lohrmann, J., Albrecht, V. *et al.* (2004) The response regulator 2 mediates ethylene signalling and hormone signal integration in *Arabidopsis*. *EMBO J.* 23, 3290–3302.

Hay, A., Kaur, H., Phillips, A., Hedden, P., Hake, S. and Tsiantis, M. (2002) The gibberellin pathway mediates KNOTTED1-type homeobox function in plants with different body plans. *Curr. Biol.* 12, 1557–1565.

Hedden, P. and Kamiya, Y. (1997) GIBBERELLIN BIOSYNTHESIS: enzymes, genes and their regulation. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* 48, 431–460.

Henderson, J.T., Li, H.C., Rider, S.D., Mordhorst, A.P., Romero, S.J., Cheng, J.C., Robey, J., Sung, Z.R., de Vries, S.C. and Ogas, J. (2004) PICKLE acts throughout the plant to repress expression of

embryonic traits and may play a role in gibberellin-dependent responses. *Plant Physiol.* 134, 995–1005.

Hoth, S., Ikeda, Y., Morgante, M., Wang, X., Zuo, J., Hanafey, M.K., Gaasterland, T., Tingey, S.V. and Chua, N.H. (2003) Monitoring genome-wide changes in gene expression in response to endogenous cytokinin reveals targets in *Arabidopsis thaliana*. *FEBS Lett.* 554, 373–380.

Hsieh, M.H. and Goodman, H.M. (2002) Molecular characterization of a novel gene family encoding ACT domain repeat proteins in *Arabidopsis*. *Plant Physiol.* 130, 1797–1806.

Hwang, I. and Sheen, J. (2001) Two-component circuitry in *Arabidopsis* cytokinin signal transduction. *Nature*, 413, 383–389.

Imamura, A., Hanaki, N., Nakamura, A., Suzuki, T., Taniguchi, M., Kiba, T., Ueguchi, C., Sugiyama, T. and Mizuno, T. (1999) Compilation and characterization of *Arabidopsis thaliana* response regulators implicated in His-Asp phosphorelay signal transduction. *Plant Cell Physiol.* 40, 733–742.

Inoue, T., Higuchi, M., Hashimoto, Y., Seki, M., Kobayashi, M., Kato, T., Tabata, S., Shinozaki, K. and Kakimoto, T. (2001) Identification of CRE1 as a cytokinin receptor from *Arabidopsis*. *Nature*, 409, 1060–1063.

Irizarry, R.A., Bolstad, B.M., Collin, F., Cope, L.M., Hobbs, B. and Speed, T.P. (2003) Summaries of Affymetrix GeneChip probe level data. *Nucleic Acids Res.* 31, e15.

Jiao, Y., Yang, H., Ma, L. *et al.* (2003) A genome-wide analysis of blue-light regulation of *Arabidopsis* transcription factor gene expression during seedling development. *Plant Physiol.* 133, 1480–1493.

Kader, J.C. (1996) Lipid-transfer proteins in plants. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* 47, 627–654.

Kakimoto, T. (1996) CKI1, a histidine kinase homolog implicated in cytokinin signal transduction. *Science*, 274, 982–985.

Kakimoto, T. (2001) Identification of plant cytokinin biosynthetic enzymes as dimethylallyl diphosphate: ATP/ADP isopentenyltransferases. *Plant Cell Physiol.* 42, 677–685.

Kakimoto, T. (2003) Biosynthesis of cytokinins. *J. Plant Res.* 116, 233–239.

Kasahara, H., Takei, K., Ueda, N., Hishiyama, S., Yamaya, T., Kamiya, Y., Yamaguchi, S. and Sakakibara, H. (2004) Distinct isoprenoid origins of cis- and trans-zeatin biosyntheses in *Arabidopsis*. *J. Biol. Chem.* 279, 14049–14054.

Kiba, T., Taniguchi, M., Imamura, A., Ueguchi, C., Mizuno, T. and Sugiyama, T. (1999) Differential expression of genes for response regulators in response to cytokinins and nitrate in *Arabidopsis thaliana*. *Plant Cell Physiol.* 40, 767–771.

Kiba, T., Aoki, K., Sakakibara, H. and Mizuno, T. (2004) *Arabidopsis* response regulator, ARR22, ectopic expression of which results in phenotypes similar to the wol cytokinin-receptor mutant. *Plant Cell Physiol.*, 45, 1063–1077.

Kiba, T., Naitou, T., Koizumi, N., Yamashino, T., Sakakibara, H. and Mizuno, T. (2005) Combinatorial microarray analysis revealing arabidopsis genes implicated in cytokinin responses through the His → Asp Phosphorelay circuitry. *Plant Cell Physiol.* 46, 339–355.

Knox, K., Grierson, C.S. and Leyser, O. (2003) AXR3 and SHY2 interact to regulate root hair development. *Development*, 130, 5769–5777.

Köllmer, I. (2004) Regulation der AtCKX-Genexpression und Charakterisierung cytokinindefizienter *Arabidopsis thaliana*-Pflanzen. Diploma Thesis. Free University, Berlin.

Landrieu, I., da Costa, M., De Veylder, L. *et al.* (2004) A small CDC25 dual-specificity tyrosine-phosphatase isoform in *Arabidopsis thaliana*. *Proc. Natl Acad. Sci. USA*, 101, 13380–13385.

Lariguet, P., Boccalandro, H.E., Alonso, J.M., Ecker, J.R., Chory, J., Casal, J.J. and Fankhauser, C. (2003) A growth regulatory loop

that provides homeostasis to phytochrome a signaling. Plant Cell, 15, 2966–2978.

Ledger, S., Strayer, C., Ashton, F., Kay, S.A. and Putterill, J. (2001) Analysis of the function of two circadian-regulated CONSTANS-LIKE genes. Plant J. 26, 15–22.

Lerbs, S., Lerbs, W., Klyachko, N.L., Romanko, E.G., Kulaeva, O.N., Wollgiehn, R. and Parthier, B. (1984) Gene expression in cytokinin- and light-mediated plastogenesis of cucurbita cotyledons: ribulose-1,5-bisphosphate carboxylase/oxygenase. Planta, 162, 289–298.

Lin, W.H., Ye, R., Ma, H., Xu, Z.H. and Xue, H.W. (2004) DNA chip-based expression profile analysis indicates involvement of the phosphatidylinositol signaling pathway in multiple plant responses to hormone and abiotic treatments. Cell Res. 14, 34–45.

Lohrmann, J., Sweere, U., Zabaleta, E., Baeurle, I., Keitel, C., Kozma, B.L., Brennicke, A., Schaefer, E., Kudla, J. and Harter, K. (2001) The response regulator ARR2: a pollen-specific transcription factor involved in the expression of nuclear genes for components of mitochondrial complex I in Arabidopsis. Mol. Gen. Genet. 265, 2–13.

Lu, G., Campbell, W., Lindqvist, Y. and Schneider, G. (1992) Crystallization and preliminary crystallographic studies of the FAD domain of corn NADH: nitrate reductase. J. Mol. Biol. 224, 277–279.

Luschnig, C., Gaxiola, R.A., Grisafi, P. and Fink, G.R. (1998) EIR1, a root-specific protein involved in auxin transport, is required for gravitropism in Arabidopsis thaliana. Genes Dev. 12, 2175–2187.

Maguire, B.A. and Zimmermann, R.A. (2001) The ribosome in focus. Cell, 104, 813–816.

Martin, T., Frommer, W.B., Salanoubat, M. and Willmitzer, L. (1993) Expression of an Arabidopsis sucrose synthase gene indicates a role in metabolism of sucrose both during phloem loading and in sink organs. Plant J. 4, 367–377.

Mas, P., Alabadi, D., Yanovsky, M.J., Oyama, T. and Kay, S.A. (2003) Dual role of TOC1 in the control of circadian and photomorphogenic responses in Arabidopsis. Plant Cell, 15, 223–236.

Meijer, H.J. and Munnik, T. (2003) Phospholipid-based signaling in plants. Annu. Rev. Plant Biol. 54, 265–306.

Miller, C.O. (1956) Similarity of some kinetin and red light effects. Plant Physiol. 13, 318–319.

Mizuno, T. (2004) Plant response regulators implicated in signal transduction and circadian rhythm. Curr. Opin. Plant Biol. 7, 499–505.

Mogensen, J.E., Wimmer, R., Larsen, J.N., Spangfort, M.D. and Otzen, D.E. (2002) The major birch allergen, Bet v 1, shows affinity for a broad spectrum of physiological ligands. J. Biol. Chem. 277, 23684–23692.

Mok, M.C. (1994) Cytokinins and plant development: an overview. In Cytokinins: Chemistry, Activity, and Function (Mok, D.W. and Mok, M.C., eds). Boca Raton, FL: CRC Press, pp. 155–166.

Naora, H. and Naora, H. (1999) Involvement of ribosomal proteins in regulating cell growth and apoptosis: translational modulation or recruitment for extraribosomal activity? Immunol. Cell Biol. 77, 197–205.

Ogas, J., Cheng, J.C., Sung, Z.R. and Somerville, C. (1997) Cellular differentiation regulated by gibberellin in the Arabidopsis thaliana pickle mutant. Science, 277, 91–94.

Philippar, K., Ivashikina, N., Ache, P., Christian, M., Luethen, H., Palme, K. and Hedrich, R. (2004) Auxin activates KAT1 and KAT2, two K+-channel genes expressed in seedlings of Arabidopsis thaliana. Plant J. 37, 815–827.

Price, J., Laxmi, A., St Martin, S.K. and Jang, J.C. (2004) Global transcription profiling reveals multiple sugar signal transduction mechanisms in Arabidopsis. Plant Cell, 16, 2128–2150.

Rashotte, A.M., Carson, S.D.B., To, J.P.C. and Kieber, J.J. (2003) Expression profiling of cytokinin action in Arabidopsis. Plant Physiol. 132, 1998–2011.

Rausch, C., Zimmermann, P., Amrhein, N. and Bucher, M. (2004) Expression analysis suggests novel roles for the plastidic phosphate transporter Pht2;1 in auto- and heterotrophic tissues in potato and Arabidopsis. Plant J. 39, 13–28.

Redman, J.C., Haas, B.J., Tanimoto, G. and Town, C.D. (2004) Development and evaluation of an Arabidopsis whole genome Affymetrix probe array. Plant J. 38, 545–561.

Repp, A., Mikami, K., Mittmann, F. and Hartmann, E. (2004) Phosphoinositide-specific phospholipase C is involved in cytokinin and gravity responses in the moss Physcomitrella patens. Plant J. 40, 250–259.

Reutter, K., Atzorn, R., Hadeler, B., Schmülling, T. and Reski, R. (1998) Expression of the bacterial ipt gene in Physcomitrella rescues mutations in budding and in plastid division. Planta, 206, 196–203.

Richmond, A.E. and Lang, A. (1957) Effect of kinetin on protein content and survival of detached Xanthium leaves. Science, 125, 650.

Riechmann, J.L. and Meyerowitz, E.M. (1998) The AP2/EREBP family of plant transcription factors. Biol. Chem. 379, 633–646.

Riechmann, J.L., Heard, J., Martin, G. et al. (2000) Arabidopsis transcription factors: Genome-wide comparative analysis among eukaryotes. Science, 290, 2105–2110.

Rolland, N., Dorne, A.J., Amoroso, G., Sultemeyer, D.F., Joyard, J. and Rochaix, J.D. (1997) Disruption of the plastid ycf10 open reading frame affects uptake of inorganic carbon in the chloroplast of Chlamydomonas. EMBO J. 16, 6713–6726.

Romanov, G.A., Getman, I.A. and Schmülling, T. (2000) Investigation of early cytokinin effects in a rapid Amaranthus seedling test. Plant Growth Regul. 32, 337–344.

Romanov, G.A., Kieber, J.J. and Schmülling, T. (2002) A rapid cytokinin response assay in Arabidopsis indicates a role for phospholipase D in cytokinin signalling. FEBS Lett. 515, 39–43.

Rupp, H.M., Frank, M., Werner, T., Strnad, M. and Schmülling, T. (1999) Increased steady state mRNA levels of the STM and KNAT1 homeobox genes in cytokinin overproducing Arabidopsis thaliana indicate a role for cytokinins in the shoot apical meristem. Plant J. 18, 557–563.

Sabatini, S., Heidstra, R., Wildwater, M. and Scheres, B. (2003) SCARECROW is involved in positioning the stem cell niche in the Arabidopsis root meristem. Genes Dev. 17, 354–358.

Sakai, H., Aoyama, T. and Oka, A. (2000) Arabidopsis ARR1 and ARR2 response regulators operate as transcriptional activators. Plant J. 24, 703–711.

Sakakibara, H. (2003) Nitrate-specific and cytokinin-mediated nitrogen signaling pathways in plants. J. Plant Res. 116, 253–257.

Sauer, N. and Stolz, J. (1994) SUC1 and SUC2: Two sucrose transporters from Arabidopsis thaliana; expression and characterization in baker’s yeast and identification of the histidine-tagged protein. Plant J. 6, 67–77.

Saunders, M.J. (1994) Cytokinin and signal transduction. In Cytokinins: Chemistry, Activity and Function (Mok, D.W.S. and Mok, M.C., eds). Boca Raton, FL: CRC Press, pp. 155–166.

Schäfer, S. and Schmülling, T. (2002) The CRK1 receptor-like kinase gene of tobacco is negatively regulated by cytokinin. Plant Mol. Biol. 50, 155–166.

Schena, M., Lloyd, A.M. and Davis, R.W. (1993) The HAT4 gene of Arabidopsis encodes a developmental regulator. Genes Dev. 7, 367–379.

Schlüepmann, H., Pellny, T., Van, D.A., Smeeakens, S. and Paul, M. (2003) Trehalose 6-phosphate is indispensable for carbohydrate

utilization and growth in *Arabidopsis thaliana*. Proc. Natl Acad. Sci. USA, 100, 6849–6854.

Schmitt, J.M. and Piepenbrock, M. (1992) Regulation of phosphoenolpyruvate carboxylase and crassulacean acid metabolism induction in *Mesembryanthemum crystallinum* L. by cytokinin. Plant Physiol. 99, 1664–1669.

Schmülling, T. (2004) Cytokinin. In *Encyclopedia of Biological Chemistry* (Lennarz, W. and Lane, M.D. eds). Amsterdam: Academic Press/Elsevier Science.

Schmülling, T., Schaefer, S. and Romanov, G. (1997) Cytokinins as regulators of gene expression. Physiol. Plant. 100, 505–519.

Schmülling, T., Werner, T., Riefler, M., Krupková, E. and Bartrina y Manns, I. (2003) Structure and function of cytokinin oxidase/dehydrogenase genes of maize, rice, *Arabidopsis* and other species. J. Plant Res. 116, 241–252.

Schumaker, K.S. and Dietrich, M.A. (1998) Hormone-induced signaling during moss development. Annu. Rev. Plant Physiol. Plant Mol. Biol. 49, 501–523.

Schuster, G., Lisitsky, I. and Klaff, P. (1999) Polyadenylation and degradation of mRNA in the chloroplast. Plant Physiol. 120, 937–944.

Seyer, P. and Lescure, A.M. (1984) Evidence for changes in plastid mRNA populations during cytokinin-induced chloroplast differentiation in tobacco cell suspensions. Plant Sci. Lett. 36, 59–66.

Smalle, J., Kurepa, J., Yang, P., Babiychuk, E., Kushnir, S., Durski, A. and Vierstra, R.D. (2002) Cytokinin growth responses in *Arabidopsis* involve the 26S proteasome subunit RPN12. Plant Cell, 14, 17–32.

Stadler, R., Büttner, M., Ache, P., Hedrich, R., Ivashikina, N., Melzer, M., Shearson, S.M., Smith, S.M. and Sauer, N. (2003) Diurnal and light-regulated expression of AtSTP1 in guard cells of *Arabidopsis*. Plant Physiol. 133, 528–537.

Suzuki, T., Sakurai, K., Ueguchi, C. and Mizuno, T. (2001) Two types of putative nuclear factors that physically interact with histidine-containing phosphotransfer (HPt) domains, signaling mediators in His-to-Asp phosphorelay, in *Arabidopsis thaliana*. Plant Cell Physiol. 42, 37–45.

Szick-Miranda, K. and Bailey-Serres, J. (2001) Regulated heterogeneity in 12-kDa P-protein phosphorylation and composition of ribosomes in maize (*Zea mays* L.). J. Biol. Chem. 276, 10921–10928.

Takei, K., Sakakibara, H. and Sugiyama, T. (2001) Identification of genes encoding adenylate isopentenyltransferase, a cytokinin biosynthesis enzyme, in *Arabidopsis thaliana*. J. Biol. Chem. 276, 26405–26410.

Taniguchi, M., Kiba, T., Sakakibara, H., Ueguchi, C., Mizuno, T. and Sugiyama, T. (1998) Expression of *Arabidopsis* response regulator homologs is induced by cytokinins and nitrate. FEBS Lett. 429, 259–262.

Thimm, O., Blaesing, O., Gibon, Y., Nagel, A., Meyer, S., Krueger, P., Selbig, J., Mueller, L.A., Rhee, S.Y. and Stitt, M. (2004) MAPMAN: a user-driven tool to display genomics data sets onto diagrams of metabolic pathways and other biological processes. Plant J. 37, 914–939.

Tian, Q. and Reed, J.W. (1999) Control of auxin-regulated root development by the *Arabidopsis thaliana* SHY2/IAA3 gene. Development, 126, 711–721.

Tian, Q., Uhlir, N.J. and Reed, J.W. (2002) *Arabidopsis* SHY2/IAA3 inhibits auxin-regulated gene expression. Plant Cell, 14, 301–319.

To, J.P.C., Haberer, G., Ferreira, F.J., Deruere, J., Mason, M.G., Schaller, G.E., Alonso, J.M., Ecker, J.R. and Kieber, J.J. (2004) Type-A *Arabidopsis* response regulators are partially redundant negative regulators of cytokinin signaling. Plant Cell, 16, 658–671.

Ueguchi, C., Koizumi, H., Suzuki, T. and Mizuno, T. (2001) Novel family of sensor histidine kinase genes in *Arabidopsis thaliana*. Plant Cell Physiol. 42, 231–235.

Versaw, W.K. and Harrison, M.J. (2002) A chloroplast phosphate transporter, PHT2;1, influences allocation of phosphate within the plant and phosphate-starvation responses. Plant Cell, 14, 1751–1766.

Vogel, J.P., Woeste, K.E., Theologis, A. and Kieber, J.J. (1998) Recessive and dominant mutations in the ethylene biosynthetic gene ACS5 of *Arabidopsis* confer cytokinin insensitivity and ethylene overproduction, respectively. Proc. Natl Acad. Sci. USA, 95, 4766–4771.

Wang, G., Li, C. and Zhang, F. (2003) Effects of different nitrogen forms and combination with foliar spraying with 6-benzylaminopurine on growth, transpiration, and water and potassium uptake and flow in tobacco. Plant Soil, 256, 169–178.

Werner, T., Motyka, V., Strnad, M. and Schmülling, T. (2001) Regulation of plant growth by cytokinin. Proc. Natl Acad. Sci. USA, 98, 10487–10492.

Werner, T., Motyka, V., Laucou, V., Smets, R., Van, O.H. and Schmülling, T. (2003) Cytokinin-deficient transgenic *Arabidopsis* plants show multiple developmental alterations indicating opposite functions of cytokinins in the regulation of shoot and root meristem activity. Plant Cell, 15, 2532–2550.

Wilson, K., Long, D., Swinburne, J. and Coupland, G. (1996) A dissociation insertion causes a semidominant mutant that increases expression of *TINY*, an *Arabidopsis* gene related to *APETALA2*. Plant Cell, 8, 659–671.

Xie, Z. and Merchant, S. (1996) The plastid-encoded ccsA gene is required for heme attachment to chloroplast c-type cytochromes. J. Biol. Chem. 271, 4632–4639.

Yamamoto, Y., Sato, E., Shimizu, T., Nakamich, N., Sato, S., Kato, T., Tabata, S., Nagatani, A., Yamashino, T. and Mizuno, T. (2003) Comparative genetic studies on the APRR5 and APRR7 genes belonging to the APRR1/TOC1 quintet implicated in circadian rhythm, control of flowering time, and early photomorphogenesis. Plant Cell Physiol. 44, 1119–1130.

Yang, H., Matsubayashi, Y., Nakamura, K. and Sakagami, Y. (2001) Diversity of *Arabidopsis* genes encoding precursors for phytosulfokine, a peptide growth factor. Plant Physiol. 127, 842–851.

Zimmermann, P., Hirsch-Hoffmann, M., Hennig, L. and Gruissem, W. (2004) GENEVESTIGATOR. *Arabidopsis* microarray database and analysis toolbox. Plant Physiol. 136, 2621–2632.
