
JOURNAL OF BIOSCIENCE AND BIOENGINEERING  
Vol. 94, No. 6, 511–517. 2002

---

**REVIEW**

**Biosynthesis, Bioproduction and Novel Roles of Ubiquinone**

MAKOTO KAWAMUKAI¹  

*Faculty of Life and Environmental Science, Shimane University,*  
*1060 Nishikawatsu, Matsue, 690-8504 Japan*

Received 7 August 2002/Accepted 6 September 2002

Ubiquinone (coenzyme Q) is a well-known component of the electron transfer system in living organisms. It is known that ubiquinone transfers electrons from Complex I (or Complex II) to Complex III in the respiratory chain. However, recent evidence indicates that an involvement in respiration is not the sole role of ubiquinone, and various novel roles have been elucidated. A role as a lipid-soluble antioxidant is now widely accepted. The relationship between lifespan and ubiquinone has attracted much interest based on the study of a *Caenorhabditis elegans clk-1* mutant. The connection between disulfide bond formation and ubiquinone (or menaquinone) in *Escherichia coli* has been well studied. The production of hydrogen sulfide in a ubiquinone-deficient fission yeast is an interesting phenotype recently observed. These are some examples of the novel roles of ubiquinone and this review summarizes the recent findings relating to the biosynthesis, bioproduction, and novel roles of ubiquinone.

[Key words: ubiquinone, coenzyme Q, antioxidant, lifespan, sulfide]

Ubiquinone, a natural compound widely distributed in living organisms, is composed of a benzoquinone moiety and an isoprenoid side chain. The primary role of ubiquinone is as a component of the respiratory chain which transfers electrons from Complex I (or Complex II) to Complex III. Almost all living organisms including humans can produce ubiquinone, and ubiquinone was first thought to be a vitamin. The roles of ubiquinone, other than as a component of the respiratory chain, have been extensively studied (1). Living organisms possess different species of ubiquinone depending on the length of the side chain. For example, humans and *Schizosaccharomyces pombe* produce UQ-10, mouse and *Zea mays* produce UQ-9, *Escherichia coli* produces UQ-8, and *Saccharomyces cerevisiae* produces UQ-6 (2). The side chain of ubiquinone is supplied by the enzyme polyprenyl diphosphate synthase (3, 4). Among various ubiquinone species, UQ-10 (CoQ10) has been used as a medicine (1, 5) and recently as a nutritional supplement. UQ-10 (CoQ10) is a lipid antioxidant solely biosynthesized in humans, and therefore the side effects of UQ-10 (CoQ10) supplementation are few.

### I. BIOSYNTHESIS OF UBIQUINONE

The biosynthetic pathway of ubiquinone was proposed from the results of the genetic analysis of mutants of *E. coli* and *S. cerevisiae*. Eight alleles of *E. coli ubi* mutants and *S. cerevisiae coq* mutants were used to define the biosynthesis of ubiquinone in both organisms (6). Until recently, the genes for all eight *ubi* mutants and *coq* mutants have been identified, and almost all functions of *ubi* genes were elucidated in *E. coli*, but not all functions of *COQ* genes were unraveled in *S. cerevisiae*. Starting from the synthesis of para-hydroxybenzoate (PHB), 8 enzymes and 9 steps are thought to be involved in ubiquinone biosynthesis (Fig. 1, Table 1). In addition, the condensation step of the isoprenoid for the side chain of ubiquinone is involved in the biosynthesis (7). The pathways in *E. coli* and *S. cerevisiae* are slightly different in the order of biosynthesis. The synthesis of PHB, which is catalyzed by chorismate lyase in *E. coli*, does not occur in *S. cerevisiae*. PHB-polyprenyl transferase (UbiA/Coq2), which transfers isoprenoid to PHB, has broad substrate specificity (8). The *ubiG/COQ3* and *ubiE/COQ5* genes encode O-methyltransferase and C-methyltransferase, respectively (9, 10). The *ubiB* gene and the *ubiH/COQ6* genes encode a monooxygenase (11). Both the *ubiD* and *ubiX* genes encode a decarboxylase, but amino acid similarity was not observed between the gene products. The *COQ7* gene, which encodes a monooxygenase, is a homolog of *clk-1*, which was found to be involved in the lifespan of *Caenorhabditis elegans* (12). The *ispB/COQ1* genes encode polyprenyl diphosphate synthase (13). The *COQ4* gene has been identified, but a role for its gene product is not clear. The *COQ8* gene has been identified as being the same as the *ABC1* gene, which was isolated in the screening to find the factor involved in the translational regulation of cytochrome *b* (14).

The reaction steps in ubiquinone biosynthesis are easily estimated in *ubi* mutants in *E. coli* because precursor accumulation occurred in *ubi* mutants, but it is rather difficult in

e-mail: kawamuka@life.shimane-u.ac.jp  
phone: +81-(0)852-32-6587 fax: +81-(0)852-32-6092

FIG. 1. Biosynthesis of ubiquinone. The biosynthetic pathways of ubiquinone in *E. coli* and *S. cerevisiae* are shown. All *E. coli ubi* genes involved in the ubiquinone synthesis are assigned in the pathway, while two *S. cerevisiae COQ* genes are not defined (Table 1).

TABLE 1. The genes and enzymes involved in ubiquinone biosynthesis

| *E. coli* | *S. cerevisiae* | *S. pombe* | *C. elegans* | Human (mouse) | *Arabidopsis* | Enzyme |
|-----------|-----------------|-------------|--------------|----------------|---------------|--------|
| ispB      | COQ1            | dpsl-dpl1   | C24A11.9     | HuDPSI         | AtSPSI,2      | Polyisoprenyl diphosphate synthase |
| ubiA      | COQ2            | pptl        | F57B9.4b     | HuPPTI         | AtPPTI        | PHB-polyisoprenyltransferase |
| ubiG      | COQ3            | SPCC162.05  | CeCOQ3       | HuCOQ3         | AtCOQ3        | O-Methyltransferase |
|           | COQ4<sup>a</sup>| SPAC1687.12c| T03F1.2      | LOC51117       | At2g03690     | Unknown |
| ubiE      | COQ5            | SPCC4G3.04c | ZK652.9      | MGC4767        | At5g57300     | C-Methyltransferase |
| ubiH      | COQ6            | SPBC146.12  | K07B1.2      | MGC23201       | At3g24200     | Monooxygenase |
| ubiF      | COQ7            | SPBC337.15c | clk-1        | HuCOQ7         |               | Monooxygenase |
| ubiB      | COQ8<sup>a</sup>| abcl<sup>a</sup> | C35D10.4    | AAH13114       | AtABC1<sup>a</sup> | Monooxygenase |
| ubiC      |                 |             |              |                |               | Chorismate lyase |
| ubiD/ubiX |                 |             |              |                |               | Decarboxylase |

The genes and enzymes for ubiquinone biosynthesis have been best studied in *E. coli* and *S. cerevisiae* (7–16). The genes which are functionally or structurally homologous are aligned. The genes experimentally examined are italicized, but others are indicated by the annotated name on databases.

<sup>a</sup> Functions are not known.

*S. cerevisiae* because the accumulation of precursor was not observed. Based on this finding, Clarke proposed that the COQ enzymes form a complex in *S. cerevisiae* (15). Several genes for the biosynthesis of ubiquinone in animals and plants have been reported, but basically genes are similar to those from *S. cerevisiae* (Table 1). It is thus considered that the eukaryotic type of biosynthesis is common. However, intriguingly, a homolog of COQ7 from plants cannot be detected.

**Prenyldiphosphate synthase and mechanism to determine the side chain length of ubiquinone** The isoprenoid side chain of ubiquinone is synthesized through the mevalonate pathway in eukaryotes, but is synthesized through the 2-C-methyl-D-erythritol 4-phosphate (MEP) (or non-mevalonate) pathway in prokaryotes or in plant chloroplasts (6). The MEP pathway has been extensively studied and recently most of the reactions of this pathway were elucidated (16). Dimethylallyl diphosphate (DMAPP), the isomer of isopentenyl diphosphate (IPP) is a primer for the condensation of one unit of IPP to construct isoprenoid of a certain length (Fig. 2) (17). A side chain longer than C<sub>25</sub> is general for ubiquinone (or menaquinone) and a side chain shorter than C<sub>20</sub> is rare in natural ubiquinones. It is known that farnesyl diphosphate (FPP) is the precursor of squalene and geranylgeranyl diphosphate (GGPP) is used for carotenoid biosynthesis. FPP is synthesized by FPP synthase and GGPP by GGPP synthase (Table 2). Many genes for FPP synthase and GGPP synthase from various organisms have been cloned and analyzed (18). Based on the analysis of the respective enzymes, it was proposed that the 5th position from the first aspartate-rich motif of prenyl diphosphate synthase is particularly important for the chain length determination of FPP and GGPP (19).

To establish how the length of the side chain of ubiquinone is determined, our group collectively cloned many different types of prenyldiphosphate synthase genes and expressed them in *E. coli* and *S. cerevisiae* (13, 20). Each transformant produced a ubiquinone species in which the corresponding prenyldiphosphate synthase was expressed. Our group has successfully produced UQ5-UQ10 in *S. cerevisiae* and UQ7-UQ10 in *E. coli* (13, 21). Based on the analysis, it was concluded that the side chain of ubiquinone was

MEP pathway
MVA pathway

Cytokinin

Monoterpenoids

Steroids
Cholesterol
Sesquiterpenoids
Farnesylated proteins
Heme a
Vitamin K₂

Carotenoids
Retinoids
Diterpenes
Geranylgeranylated proteins
Chlorophylls

Archeal ether-linked lipids

UQ6

UQ7

UQ8
MK8

UQ9

UQ10

Dimethylallyl diphosphate
Isopentenyl diphosphate

Geranyl diphosphate
IPP

Farnesyl diphosphate
IPP

Geranylgeranyl diphosphate
IPP

Geranylgeranyl diphosphate
IPP

Hexaprenyl diphosphate
IPP

Heptaprenyl diphosphate
IPP

Octaprenyl diphosphate
IPP

Nonaprenyl diphosphate
IPP

Decaprenyl diphosphate
IPP

Undecaprenyl diphosphate
Dehydrololichyl diphosphate
Natural rubber

Chicle ~100
Gulta-percha ~700

FIG. 2. The synthesis of various isoprenoids. Various isoprenoid compounds synthesized either via the MVA pathway or the MEP pathway are shown. Dimethylallyl diphosphate (DMAPP) is a C₅ unit compound that serves as the precursor to condense multiple units of isopentenyl diphosphate (IPP). Isoprenoid with more than C₂₅ units are generally used for the synthesis of the side chain of ubiquinone.

determined by the supplied isoprenoid.

Thus, which region of prenyl diphosphate synthase is important for determination of the length of the isoprenoid product? My group has investigated this by constructing various mutants of DdsA (decaprenyl diphosphate synthase) from Gluconobacter suboxydans (21) and IspB (octaprenyl diphosphate synthase) from E. coli (22). There are two aspartate-rich regions in the enzymes. The most significant site for the determination of the length was the 5th position from the first aspartate-rich region as proposed from the analysis of FPP synthase and GGPP synthase. It was also found that the formation of an IspB dimer is essential for determination of the length. My group has recently identified a different type of prenyldiphosphate synthase which is composed of a Dps1 and Dlp1 heteromer (Fig. 3). Dlp1 lacks the aspartate-rich motif and is bound to Dps1 to constitute the heteromeric enzyme (Kawamukai et al., unpublished results). The variety of prenyldiphosphate synthase types is summarized in Table 2. Most of the enzymes are homodimers while some heteromeric enzymes were found.

II. BIOPRODUCTION OF UBIQUINONE

Industrial-scale bioproduction of ubiquinone by microorganisms has already been established by some Japanese companies. Microorganisms that produce UQ-10 including photosynthetic bacteria and yeasts, were selected and used for the fermentation production of UQ-10 (23). This fer-

TABLE 2. Various prenyl diphosphate synthases

| Enzyme                          | Subunit Structure | Length | Origin                                      |
|----------------------------------|-------------------|--------|---------------------------------------------|
| Geranyl diphosphate synthase     | Heterotetramer   | $C_{10}$ | Mentha piperita                             |
| Geranyl diphosphate synthase     | Homodimer        | $C_{10}$ | Arabidopsis, Abies grandis                  |
| Farnesyl diphosphate synthase    | Homodimer        | $C_{15}$ | Bacteria, Plants, Animals                   |
| Geranylgeranyl diphosphate synthase | Homodimer$^a$ | $C_{20}$ | Bacteria, Plants, Animals                   |
| Farnesylgeranyl diphosphate synthase | ?           | $C_{25}$ | Natronobacterium pharanois                 |
| Hexaprenyl diphosphate synthase  | Heterodimer      | $C_{30}$ | Micrococcus luteus                         |
| Heptaprenyl diphosphate synthase | Heterodimer      | $C_{35}$ | Bacillus subtilis, B. stearothermophilus   |
| Heptaprenyl diphosphate synthase | Homodimer        | $C_{35}$ | Haemophilus influenzae                     |
| Octaprenyl diphosphate synthase  | Homodimer        | $C_{40}$ | Escherichia coli                           |
| Nonaprenyl diphosphate synthase  | Homodimer        | $C_{45}$ | Synechocystis                              |
| Decaprenyl diphosphate synthase  | Homodimer        | $C_{50}$ | Gluconobacter suboxydans                   |
| Decaprenyl diphosphate synthase  | Heteromer        | $C_{50}$ | Schizosaccharomyces pombe                  |
| Undecaprenyl diphosphate synthase | Homodimer       | $C_{55}$ | Micrococcus luteus                         |

Polyprenyl diphosphate synthases responsible for the synthesis of menaquinone and ubiquinone are mainly shown. Many different types of FPP synthases and GGPP synthases were analyzed. Most of the polyprenyl diphosphate synthases are homodimers, while some are heterodimers (17–22, 49, 50).

$^a$ Some have been proposed to be homo-multimers.

mentation production of UQ-10 has become more widely used than chemical synthesis. The application of genetic engineering to the production of ubiquinone was once attempted in *E. coli* as a pilot experiment (24). A genetically engineered microorganism producing UQ-10 was also constructed (13).

### III. NOVEL ROLES OF UBIQUINONE

#### Antioxidant function of ubiquinone
While the electron transfer system is found in mitochondria, the distribution of ubiquinone is not only limited to mitochondria. Ubiquinone can be found in many other membranes including the Golgi, lysosome, microsome, peroxisome, and cell membranes (1). This wide distribution of ubiquinone suggests roles other than electron transfer. An antioxidant function of ubiquinone has been searched for in many organisms, and observed in most of them. The antioxidant function was found to protect against lipid peroxidation in the membrane. Ubiquinol functions as an antioxidant to the same extent as vitamin E to prevent lipid peroxidation (25), DNA damage and LDL peroxidation (26). Co-supplementation of ubiquinol and vitamin E synergistically prevents lipid peroxidation (1). In support of the hypothesis that ubiquinone functions as an antioxidant, *E. coli*, *S. cerevisiae*

![Structure of Dps1 and Dlp1](https://i.imgur.com/yourimage.png)

FIG. 3. Structure of Dps1 and Dlp1. Decaprenyl diphosphate synthase subunit 1 (Dps1) from fission yeast possesses a structure typical of polyprenyl diphosphate synthase, while Dlp1 does not have the aspartate-rich motif but shares homology with Dps1. Dps1 and Dlp1 form a complex to generate an active decaprenyl diphosphate synthase.

and *S. pombe* lacking ubiquinone are very sensitive to oxidative stress (27–29). Typically, in ubiquinone-less *S. cerevisiae*, cells became sensitive to lipid peroxide (28), and in ubiquinone-less *S. pombe*, cells became sensitive to $\mathrm{Cu}^{2+}$ and $\mathrm{H}_{2} \mathrm{O}_{2}$ (29).

#### Ubiquinone and the Arc two-component system
Recently, it was reported that the Arc two-component system in *E. coli* senses the ubiquinone or menaquinone redox signal (30). Arc is the redox-sensing two-component system comprised of ArcA as a DNA-binding response regulator and ArcB as a membrane sensor kinase (30). ArcB is autophosphorylated as a result of redox state sensing, and transfers its phosphate to ArcA. The phosphorylated form of ArcA binds to DNA and regulates the expression of some genes involved in the TCA cycle. The autophosphorylation of ArcB is inhibited by the oxidized form of ubiquinone or menaquinone. In mutants defective in ubiquinone synthesis, expression of *cydA-lacZ* was de-repressed. These results support the proposal that quinone is involved in the ArcA-ArcB system.

Interestingly, it was reported very recently that *bvgS* and *evgS* encoding the histidine kinases of *Bordetella pertussis* and *E. coli*, respectively, are also negatively regulated by ubiquinone (31). BvgS and EvgS comprise a two-component system with response regulators BvgA and EvgA, respectively. This second example of the regulation of a two-component system by ubiquinone predicts such regulatory conservation in other organisms.

#### Involvement of ubiquinone in disulfide bond formation
It was reported that menaquinone and ubiquinone provide the electron of the disulfide bond of proteins in *E. coli* (Fig. 4). The DsbA-DsbB system operates in this protein disulfide bond formation (32) and DsbA is reduced by forming the disulfide bond. The electron is transferred from DsbA to DsbB, then DsbA is re-oxidized by DsbB. Under anaerobic conditions, menaquinone is used and under aerobic conditions ubiquinone is used to oxidize DsbB. The precise location of the binding site of ubiquinone was recently analyzed (33). Isomerization of the disulfide bond in protein is supported by the DsbC-DsbD system. The electron of cellular NADPH is transferred to DsbC and DsbG through DsbD,

FIG. 4. The electron flow in disulfide bond formation of proteins. The electrons generated through disulfide bond formation are transferred to DsbA, and then to DsbB. Electrons from DsbB are further transferred to either ubiquinone or menaquinone in *E. coli*.

and DsbC and DsbG are reduced by DsbD. This role of ubiquinone as an electron acceptor in disulfide bond formation is not observed in eukaryotes.

Hydrogen sulfide and sulfide ubiquinone reductase

When the ubiquinone-less fission yeast mutants were constructed, various phenotypes were observed. One of the interesting phenotypes was a high amount of production of hydrogen sulfide (34). Two mutants that do not synthesize ubiquinone, whether due to deletion of the gene responsible for the production of the side chain (28) or deletion of the gene responsible for the transfer of the isoprenoid to PHB (34), were examined (Fig. 5). The phenotype of hydrogen sulfide accumulation was first discovered in fission yeast, but not in budding yeast. Fission yeast possess sulfide-ubiquinone reductase (Hmt2) and this enzyme was found in higher eukaryotes, including humans, but not in budding yeast (35). Sulfide-ubiquinone reductase is an enzyme that oxidizes sulfide via ubiquinone. This enzyme was initially found in a cyanobacterium and a phototrophic bacterium. In these microorganisms sulfide-ubiquinone reductase converts sulfide to sulfur as a part of an energy conversion system for photosynthetic growth (36, 37). Ubiquinol, the reduced form, is associated with the reduction of cytochrome *bc1* complex, and then bacteriochlorophyll. In cyanobacteria and phototrophic bacteria, sulfide-ubiquinone reductase utilizes sulfide for photosynthetic energy production, but it is likely that a similar enzyme in fission yeast and other eukaryotes is used to detoxify sulfide. Further analysis will be required in this respect.

Extension of lifespan due to a lack of ubiquinone

The finding that the *clk-1* mutant of *C. elegans* exhibits an extended lifespan has become the topic of interest in the field of ubiquinone research. The *clk-1* mutant was initially isolated as a mutant that displayed prolonged lifespan, developmental retardation, and low egg production in *C. elegans* (38). The *clk-1* gene encodes the enzyme responsible for the penultimate step in ubiquinone synthesis. The *clk-1* mutant accumulates demethoxy ubiquinone (DMUQ). There are several explanations why the *clk-1* mutant has an extended lifespan. One is that the *clk-1* mutant uses DMUQ as an electron carrier to some extent and this allows *C. elegans* to survive without using ubiquinone and to consume low levels of oxygen (39). Since low oxygen consumption reduces the production of oxygen radicals, lifespan is thought to increase in this mutant. The recent finding that a *coq-3* mutant of *C. elegans* does not survive without ubiquinone conflicts with the previous finding (40). Other reports describe the fact that *C. elegans* utilizes ubiquinone from an *E. coli* source and this organism has an extended lifespan when ubiquinone is not supplemented from *E. coli* (41, 42). However, when a knockout mouse of *clk-1* was bred, it was shown that the progeny died in the embryonic stage (43, 44). These results are rather understandable than to think lifespan is prolonged due to a lack of ubiquinone. Ubiquinone is very important for electron transfer as well as anti-oxidation and such an important compound should not be dispensable. It would be interesting to determine whether the finding in *C. elegans* is common to other species.

Requirement of ubiquinone by uncoupling protein (UCP)

As one of the novel roles of ubiquinone, uncoupling protein (UCP1) involved in thermogenesis has been found to be dependent on ubiquinone. Ubiquinone is an obligatory cofactor of UCP1 (45), which functions in the H⁺

pumping in mitochondria to control cell temperature. The redox state of ubiquinone regulates the function of UCP1. In addition to UCP1, it was recently shown that UCP2 and UCP3 also require ubiquinone for their functions (46).

Disease caused by ubiquinone deficiency UQ-10 (CoQ10) has been used for the treatment of heart failure. The association of a lack of UQ-10 with disease has also been reported. A patient who possessed UQ-10 at 10% the amount in normal people suffered from familial mitochondrial myopathy (47). This patient possessed very little ubiquinone in muscle, but normal levels in other tissues such as fibroblasts and serum. UQ-10 supplementation is very effective in this patient suggesting that the patient has some problem in the metabolic pathway of ubiquinone. A role of ubiquinone in blocking apoptosis was also reported (48).

Ubiquinone (or coenzyme Q) has been used as medicine but is now widely used as a nutritional supplement. Since humans synthesize ubiquinone, it is no doubt a natural compound. The biosynthetic pathway of ubiquinone has been predominantly defined from genetic and biochemical analyses in *E. coli* and yeast. However the synthetic pathway of ubiquinone is still not sufficiently clear in higher organisms including humans, although it is expected that a similar type of biosynthesis occurs. Analysis of ubiquinone synthesis in higher organisms represents an interesting future work. Various roles of ubiquinone have been observed but some roles are not yet well explained. The involvement of ubiquinone in lifespan is now being studied extensively in many laboratories and studies on the other roles of ubiquinone are also underway.

---

**REFERENCES**

1. Ernster, L. and Dallner, G.: Biochemical, physiological and medical aspects of ubiquinone function. Biochim. Biophys. Acta, 1271, 195–204 (1995).
2. Collins, M.D. and Jones, D.: Distribution of isoprenoid quinone structural types in bacteria and their taxonomic implications. Microbiol. Rev., 45, 316–354 (1981).
3. Grünler, J., Ericsson, J., and Dallner, G.: Branch-point reactions in the biosynthesis of cholesterol, dolichol, ubiquinone and prenylated proteins. Biochim. Biophys. Acta, 1212, 259–277 (1994).
4. Okada, K., Suzuki, K., Kamiya, Y., Zhu, X., Fujisaki, S., Nishimura, Y., Nishino, T., Nakagawa, T., Kawamukai, M., and Matsuda, H.: Polyprenyl diphosphate synthase essentially defines the length of the side chain of ubiquinone. Biochim. Biophys. Acta, 1302/3, 217–223 (1996).
5. Crane, F.L.: Biochemical functions of coenzyme Q10. J. Am. Coll. Nutr., 20, 591–598 (2001).
6. Meganathan, R.: Ubiquinone biosynthesis in microorganisms. FEMS Microbiol. Lett., 203, 131–139 (2001).
7. Okada, K., Minehira, M., Zhu, X-F., Suzuki, K., Nakagawa, T., Matsuda, H., and Kawamukai, M.: The *ispB* gene encoding octaprenyl diphosphate synthase is essential for growth of *Escherichia coli*. J. Bacteriol., 179, 3058–3060 (1997).
8. Suzuki, K., Ueda, M., Yuasa, M., Nakagawa, T., Kawamukai, M., and Matsuda, H.: Evidence that *Escherichia coli ubiA* product is a functional homolog of yeast COQ2, and the regulation of *ubiA* gene expression. Biosci. Biotechnol. Biochem., 58, 1814–1819 (1994).
9. Poon, W.W., Barkovich, R.J., Hsu, A.Y., Frankel, A., Lee, P.T., Shepherd, J.N., Myles, D.C., and Clarke, C.F.:
Yeast and rat Coq3 and *Escherichia coli* UbiG polypeptides catalyze both O-methyltransferase steps in coenzyme Q biosynthesis. J. Biol. Chem., 274, 21665–21672 (1999).
10. Barkovich, R.J., Shtanko, A., Shepherd, J.A., Lee, P.T., Myles, D.C., Tzagoloff, A., and Clarke, C.F.: Characterization of the *COQ5* gene from *Saccharomyces cerevisiae*. Evidence for a C-methyltransferase in ubiquinone biosynthesis. J. Biol. Chem., 272, 9182–9188 (1997).
11. Poon, W.W., Davis, D.E., Ha, H.T., Jonassen, T., Rather, P.N., and Clarke, C.F.: Identification of *Escherichia coli ubiB*, a gene required for the first monooxygenase step in ubiquinone biosynthesis. J. Bacteriol., 182, 5139–5146 (2000).
12. Jonassen, T., Proft, M., Randez-Gil, F., Schultz, J.R., Marbois, B.N., Entian, K.D., and Clarke, C.F.: Yeast Clk1 homologue (Coq7/Cat5) is a mitochondrial protein in coenzyme Q synthesis. J. Biol. Chem., 273, 3351–3357 (1998).
13. Okada, K., Kainou, T., Matsuda, H., and Kawamukai, M.: Biological significance of the side chain length of ubiquinone in *Saccharomyces cerevisiae*. FEBS Lett., 43, 241–244 (1998).
14. Do, T.Q., Hsu, A.Y., Jonassen, T., Lee, P.T., and Clarke, C.F.: A defect in coenzyme Q biosynthesis is responsible for the respiratory deficiency in *Saccharomyces cerevisiae abc1* mutants. J. Biol. Chem., 276, 18161–18168 (2001).
15. Hsu, A.Y., Do, T.Q., Lee, P.T., and Clarke, C.F.: Genetic evidence for a multi-subunit complex in the O-methyltransferase steps of coenzyme Q biosynthesis. Biochim. Biophys. Acta, 1484, 287–297 (2000).
16. Rohdich, F., Hecht, S., Gartne, K., Adam, P., Krieger, C., Amslinger, S., Arigoni, D., Bacher, A., and Eisenreich, W.: Studies on the nonmevalonate terpene biosynthetic pathway: metabolic role of IspH (LytB) protein. Proc. Natl. Acad. Sci. USA, 99, 1158–1163 (2002).
17. Ogura, K. and Koyama, T.: Enzymatic aspects of isoprenoid chain elongation. Chem. Rev., 98, 1263–1276 (1998).
18. Koyama, T.: Molecular analysis of prenyl chain elongating enzymes. Biosci. Biotechnol. Biochem., 63, 1671–1676 (1999).
19. Wang, K. and Ohnuma, S.: Chain-length determination mechanism of isoprenyl diphosphate synthases and implications for molecular evolution. Trends Biochem. Sci., 24, 445–451 (1999).
20. Okada, K., Kamiya, Y., Zhu, X., Suzuki, K., Tanaka, K., Nakagawa, T., Matsuda, H., and Kawamukai, M.: Cloning of the *sdsA* gene encoding solanesyl diphosphate synthase from *Rhodobacter capsulatus* and its functional expression in *Escherichia coli* and *Saccharomyces cerevisiae*. J. Bacteriol., 179, 5992–5998 (1997).
21. Okada, K., Kainou, T., Tanaka, K., Nakagawa, T., Matsuda, H., and Kawamukai, M.: Molecular cloning and mutational analysis of the *ddsA* gene encoding decaprenyl diphosphate synthase from *Gluconobacter suboxydans*. Eur. J. Biochem., 255, 52–59 (1998).
22. Kainou, T., Okada, K., Suzuki, K., Nakagawa, T., Matsuda, H., and Kawamukai, M.: Dimer formation of octaprenyl diphosphate synthase (IspB) is essential for chain length determination of ubiquinone. J. Biol. Chem., 276, 7876–7883 (2001).
23. Yoshida, H., Kotani, Y., Ochiai, K., and Araki, K.: Production of ubiquinone-10 using bacteria. J. Gen. Appl. Microbiol., 44, 19–26 (1998).
24. Zhu, X.F., Yuasa, M., Okada, K., Suzuki, K., Nakagawa, T., Kawamukai, M., and Matsuda, H.: Production of ubiquinone in *Escherichia coli* by expression of various genes responsible for ubiquinone biosynthesis. J. Ferment. Bioeng., 79, 493–495 (1995).
25. Frei, B., Kim, M.C., and Ames, B.N.: Ubiquinol-10 is an effective lipid-soluble antioxidant at physiological concentrations. Proc. Natl. Acad. Sci. USA, 87, 4879–4883 (1990).
26. Kontush, A., Hubner, C., Finckh, B., Kohlschutter, A., and

Beisiegel, U.: Antioxidative activity of ubiquinol-10 at physiologic concentrations in human low density lipoprotein. Biochim. Biophys. Acta, 1258, 177–187 (1995).

27. Søballe, B. and Poole, R. K.: Ubiquinone limits oxidative stress in *Escherichia coli*. Microbiology, 145, 1817–1830 (1999).

28. Do, T.Q., Schultz, J.R., and Clarke, C.F.: Enhanced sensitivity of ubiquinone-deficient mutants of *Saccharomyces cerevisiae* to products of autoxidized polyunsaturated fatty acids. Proc. Natl. Acad. Sci. USA, 93, 7534–7539 (1996).

29. Suzuki, K., Okada, K., Kamiya, Y., Zhu, X-F., Nakagawa, T., Kawamukai, M., and Matsuda, H.: Analysis of the decaprenyl diphosphate synthase (*dps*) gene in fission yeast suggests a role of ubiquinone as an antioxidant. J. Biochem., 121, 496–505 (1997).

30. Georgellis, D., Kwon, O., and Lin, E.C.C.: Quinones as the redox signal for the arc two-component system of bacteria. Science, 292, 2314–2317 (2001).

31. Bock, A. and Gross, R.: The unorthodox histidine kinases BvgS and EvgS are responsive to the oxidation status of a quinone electron carrier. Eur. J. Biochem., 269, 3479–3484 (2002).

32. Bader, M., Muse, W., Ballou, D.P., Gassner, C., and Bardwell, J.C.A.: Oxidative protein folding is driven by the electron transport system. Cell, 98, 217–227 (1999).

33. Xie, T., Yu, L., Bader, M.W., Bardwell, J.C., and Yu, C.A.: Identification of the ubiquinone-binding domain in the disulfide catalyst disulfide bond protein B. J. Biol. Chem., 277, 1649–1652 (2002).

34. Uchida, N., Suzuki, K., Saiki, R., Kainou, T., Tanaka, K., Matsuda, H., and Kawamukai, M.: Phenotypes of fission yeast defective in ubiquinone production due to disruption of the gene for *p*-hydroxybenzoate polyprenyl diphosphate transferase. J. Bacteriol., 182, 6933–6939 (2000).

35. Vande Weghe, J.G. and Ow, D.W.: Accumulation of metal-binding peptides in fission yeast requires *hmt2⁺*. Mol. Microbiol., 42, 29–36 (2001).

36. Bronstein, M., Schutz, M., Hauska, G., Padan, E., and Shahak, Y.: Cyanobacterial sulfide-quinone reductase: cloning and heterologous expression. J. Bacteriol., 182, 3336–3344 (2000).

37. Shibata, H., Takahashi, M., Yamaguchi, I., and Kobayashi, S.: Sulfide oxidation by gene expression of sulfide-quinone oxidoreductase and ubiquinone-8 biosynthesis in *Escherichia coli*. J. Biosci. Bioeng., 88, 244–249 (1999).

38. Ewbank, J.J., Barnes, T.M., Lakowski, B., Lussier, M., Bussey, H., and Hekimi, S.: Structural and functional conservation of the *Caenorhabditis elegans* timing gene *clk-1*. Science, 275, 980–983 (1997).

39. Miyadera, H., Amino, H., Hiraishi, A., Taka, H., Murayama, K., Miyoshi, H., Sakamoto, K., Ishii, N.,

Hekimi, S., and Kita, K.: Altered quinone biosynthesis in the long-lived *clk-1* mutants of *Caenorhabditis elegans*. J. Biol. Chem., 276, 7713–7716 (2001).

40. Hihi, A.K., Gao, Y., and Hekimi, S.: Ubiquinone is necessary for *Caenorhabditis elegans* development at mitochondrial and non-mitochondrial sites. J. Biol. Chem., 277, 2202–2206 (2002).

41. Jonassen, T., Larsen, P.L., and Clarke, C.F.: A dietary source of coenzyme Q is essential for growth of long-lived *Caenorhabditis elegans clk-1* mutants. Proc. Natl. Acad. Sci. USA, 98, 421–426 (2001).

42. Larsen, P.L. and Clarke, C.F.: Extension of life-span in *Caenorhabditis elegans* by a diet lacking coenzyme Q. Science, 295, 120–123 (2002).

43. Levavasseur, F., Miyadera, H., Sirois, J., Tremblay, M.L., Kita, K., Shoubridge, E., and Hekimi, S.: Ubiquinone is necessary for mouse embryonic development but is not essential for mitochondrial respiration. J. Biol. Chem., 276, 46160–46164 (2001).

44. Nakai, D., Yuasa, S., Takahashi, M., Shimizu, T., Asaumi, S., Isono, K., Takao, T., Suzuki, Y., Kuroyanagi, H., Hirokawa, K., Koseki, H., and Shirasawa, T.: Mouse homologue of *coq7/clk-1*, longevity gene in *Caenorhabditis elegans*, is essential for coenzyme Q synthesis, maintenance of mitochondrial integrity, and neurogenesis. Biochem. Biophys. Res. Commun., 289, 463–471 (2001).

45. Echtay, K.S., Winkler, E., and Klingenberg, M.: Coenzyme Q is an obligatory cofactor for uncoupling protein function. Nature, 408, 609–613 (2000).

46. Echtay, K.S., Winkler, E., Frischmuth, K., and Klingenberg, M.: Uncoupling proteins 2 and 3 are highly active H(+) transporters and highly nucleotide sensitive when activated by coenzyme Q (ubiquinone). Proc. Natl. Acad. Sci. USA, 98, 1416–1421 (2001).

47. Ogasahara, S., Engel, A.G., Frens, D., and Mack, D.: Muscle coenzyme Q deficiency in familial mitochondrial encephalomyopathy. Proc. Natl. Acad. Sci. USA, 86, 2379–2382 (1989).

48. Alleva, R., Tomasetti, M., Andera, L., Gellert, N., Borghi, B., Weber, C., Murphy, M.P., and Neuzil, J.: Coenzyme Q blocks biochemical but not receptor-mediated apoptosis by increasing mitochondrial antioxidant protection. FEBS Lett., 503, 46–50 (2001).

49. Burke, C. and Croteau, R.: Interaction with the small subunit of geranyl diphosphate synthase modifies the chain length specificity of geranylgeranyl diphosphate synthase to produce geranyl diphosphate. J. Biol. Chem., 277, 3141–3149 (2002).

50. Liang, P.H., Ko, T.P., and Wang, A.H.: Structure, mechanism and function of prenyltransferases. Eur. J. Biochem., 269, 3339–3354 (2002).
