
Microbiology (2005), 151, 3649–3656

Methylcrotonyl-CoA and geranyl-CoA carboxylases are involved in leucine/isovalerate utilization (Liu) and acyclic terpene utilization (Atu), and are encoded by *liuB/liuD* and *atuC/atuF*, in *Pseudomonas aeruginosa*

**Birgit Höschle,**¹ **Volker Gnau**² **and Dieter Jendrossek**¹

¹Institut für Mikrobiologie, Universität Stuttgart, Allmandring 31, 70550 Stuttgart, Germany  
²Universität Tübingen, Tübingen, Germany

---

Correspondence  
Dieter Jendrossek  
dieter.jendrossek@imb.uni-stuttgart.de  

Received 10 June 2005  
Revised 29 July 2005  
Accepted 5 August 2005  

---

*Pseudomonas aeruginosa* is able to grow on acyclic monoterpenes (citronellol, citronellate, geraniol and geranylate), and on other methyl-branched compounds such as leucine or isovalerate. The catabolic pathway of citronellol (Atu, acyclic terpene utilization) enters that of leucine/isovalerate (Liu, leucine and isovalerate utilization) at the level of methylcrotonyl-CoA. Key enzymes of the combined pathways are geranyl-CoA carboxylase (GCase) and methylcrotonyl-CoA carboxylase (MCase). In this study, isovalerate-grown cells specifically expressed MCase (apparent molecular mass of the biotin-containing subunit, 74 kDa) only, and the GCase biotin-containing subunit (71 kDa) was not detected. Citronellol- or citronellate-grown cells produced both carboxylases. Biotin-dependent proteins were purified from crude extracts by avidin-affinity chromatography, and assigned to the corresponding coding genes by trypsin fingerprint analysis. The two subunits of MCase corresponded to *liuB/liuD* (PA2014/PA2012) of the *P. aeruginosa* genome database, and *atuC/atuF* (PA2888/PA2891) encoded GCase subunits. This finding is contrary to that reported by others. The identified genes are part of two separate gene clusters [*liuRABCDE* (PA2011–PA2016) and *atuABCDEFGH* (PA2886–PA2893)] that are thought to encode most of the genes of the Atu and Liu pathways.

---

### INTRODUCTION

Citronellol (3,7-dimethyloct-6-en-1-ol) and geraniol (3,7-dimethylocta-2,6-dien-1-ol) are model compounds of acyclic monoterpenes, which occur naturally in plants. Citronellol and geraniol are of industrial interest due to their commercial use in the food (flavour) and perfume (odour) industries. Moreover, citronellol is one of very few naturally occurring repellents of insects (mosquitoes) (*Hierro et al.*, 2004; *Pawar et al.*, 1991; *Rice & Coats*, 1994). Geraniol is structurally related to citronellol, and differs only by the presence of an additional double bond. Recently, it was found that geraniol and related compounds have significant effects on eukaryotic and human cells, and are able to induce apoptosis *in vitro* in pancreatic cancer cells (*Burke et al.*, 1997, 2002; *Carnesecchi et al.*, 2002a, b, 2004; *Duncan et al.*, 2004; *Izumi et al.*, 1999). The intensity and variety of biological activities of monoterpenes and related compounds have not been fully investigated. A lack of information on the biochemical routes of monoterpenes in organisms might be one reason for the shortage of knowledge. Citronellol is the only acyclic monoterpene for which some information exists on its biochemistry in micro-organisms. This knowledge is based mainly on investigations performed by Seubert and coworkers with *Pseudomonas citronellolis* more than 40 years ago (*Seubert*, 1960; *Seubert et al.*, 1963; *Seubert & Remberger*, 1963; *Seubert & Fass*, 1964a, b), and these were continued by Fall and coworkers in the late 1970s (*Cantwell et al.*, 1978; *Fall & Hector*, 1977; *Fall et al.*, 1979; *Hector & Fall*, 1976a, b). The first steps of the catabolic pathway of citronellol and geraniol are the oxidation of the alcohols to citronellate or geranylate, and subsequent activation to the corresponding CoA esters citronellyl-CoA and geranyl-CoA. Due to the structural similarity of citronellol and geraniol, it is assumed that oxidation of citronellol and geraniol is catalysed by the same enzymes (Fig. 1). However, recent data suggest that oxidation of geraniol and/or geraniol is dependent on molybdenum, and that the reaction is different from molybdenum-independent oxidation of citronellol (*Höschle & Jendrossek*, 2005). Citronellyl-CoA can be converted to geranyl-CoA by a dehydrogenase step, and all

---

Abbreviations: Atu, acyclic terpene utilization; GCase, geranyl-CoA carboxylase; Liu, leucine and isovalerate utilization; MCase, methylcrotonyl-CoA carboxylase.

B. Höschle, V. Gnau and D. Jendrossek

Fig. 1. Putative Atu and Liu pathways in *P. citronellolis*, according to Seubert & Fass (1964a). Genes of *P. aeruginosa* PAO1 identified in this study (*atuC/atuF* and *liuD/liuD*), and putative isohexenyl-glutaconyl-CoA hydratase (*atuE*) and methylglutaconyl-CoA hydratase (*liuC*) genes, are indicated.

subsequent reactions are the same for citronellol and geraniol utilization. Geranyl-CoA is difficult to metabolize because of the presence of β-methyl groups that inhibit β-oxidation. The key enzyme of the citronellol/geraniol degradation pathway is geranyl-CoA carboxylase (GCase) (Seubert et al., 1963). GCase converts the branched-chain β-methyl group of geranyl-CoA to an acetate group that can be cleaved off more easily than a methyl group. For details of the catabolic pathway of citronellol/geraniol see Fig. 1.

The metabolite 7-methyl-3-oxo-6-octenoyl-CoA can be degraded by two rounds of subsequent β-oxidation reactions, without hindrance by β-methyl groups. The resulting metabolite (3-methylcrotonyl-CoA) is degraded via the leucine/isovalerate utilization (Liu) pathway (Fig. 1), which involves another carboxylase (methylcrotonyl-CoA carboxylase, MCase) that is unable to utilize geranyl-CoA as a substrate, but is able to utilize carboxylate methylcrotonyl-CoA (Fall & Hector, 1977; Fall, 1981; Hector & Fall, 1976a,

b). The two branched-chain-specific carboxylases GCase and MCase have been purified and partially characterized in *P. citronellolis* (Fall & Hector, 1977).

*Pseudomonas aeruginosa* and *Pseudomonas mendocina* are two additional bacteria with the capacity to utilize citronellol and other linear terpenes (Cantwell *et al.*, 1978). Recently, utilization of citronellol by *P. aeruginosa* was investigated by Diaz-Perez and coworkers (Diaz-Perez *et al.*, 2004). Those authors cloned a gene cluster consisting of six genes (PA2011–PA2016) that are necessary for growth on linear terpenes in *P. aeruginosa*. The cluster contained two genes (*gnyA/gnyB*, PA2012/PA2014) for putative carboxylase subunits, and other genes that possibly encode proteins of the acyclic (mono-) terpene utilization (Atu) pathway. In our present study, we initiated investigation of the Atu pathway by a biochemical approach, and we came to different conclusions than those of Diaz-Perez and coworkers.

**METHODS**

**Bacterial strains, plasmids and culture conditions.** The bacterial strains and plasmids used in this study are shown in Table 1. Cultures of *P. aeruginosa* PAO1 were routinely grown in Luria-Bertani (LB) medium, or in mineral salt medium (Schlegel *et al.*, 1961) containing different carbon sources [0·1% (v/v) sodium citronellate, 0·1% (v/v) sodium geranilate or 0·1% (v/v) isovaleric acid], at 30°C. Water-insoluble carbon sources (citronellol and geraniol) were added in the space between the Petri dish and the agar (20 μl per plate), and the plates were sealed with Parafilm. Growth on solid media with liquid carbon sources was performed in separate incubators to avoid cross-contamination by vapours. Liquid cultures contained 0·5% glucose or 0·1% glucose, and either 0·2% sodium citronellate or 0·2% isovalerate. The *Escherichia coli* strain was grown in LB medium at 37°C. Solid media contained 1·5% (w/v) agar.

For conjugation experiments, *P. aeruginosa* PAO1 Sm<sup>r</sup> and *E. coli* S17-1, containing the respective plasmids, were grown in LB medium. Equal volumes of the donor and the recipient cultures were mixed, spotted onto an LB agar plate, and incubated for 6 h at 37°C. The cells were resuspended in 0·5 ml saline, plated on selective agar containing 500 μg streptomycin ml<sup>-1</sup> and 50 μg gentamicin ml<sup>-1</sup>, and incubated overnight at 37°C.

**Insertion mutagenesis.** Disruption of *atuF* (PA2891) in *P. aeruginosa* was carried out using pKnockout-G for rapid gene inactivation (Windgassen *et al.*, 2000). A 3′- and 5′-truncated fragment of *atuF* was obtained by PCR-mediated amplification of *atuF* from *P. aeruginosa* genomic DNA using the synthetic oligonucleotides PA2891-EcoRI-Fwd (5′-GGAATTCCGCGCCGAGCTGATCCTGGAGAGG-3′) and PA2891-EcoRI-Rev (5′-GGAATTCCTGGGGTCTTCGGCGAAGCG-TTC-3′) as primers. The EcoRI-digested purified 778 bp PCR fragment of *atuF* was cloned in the EcoRI site of pKnockout-G, and transformed in *E. coli* S17-1. The plasmid pKnockout-G::*atuF* was transferred to *P. aeruginosa* PAO1 Sm<sup>r</sup> via conjugation, and transconjugants were selected on LB agar containing 500 μg streptomycin ml<sup>-1</sup> and 50 μg gentamicin ml<sup>-1</sup>. The disruption of *atuF* was verified by (i) PCR using one *atuF*-specific primer and one pKnockout-specific primer, and (ii) two Southern blot hybridization experiments of chromosomal mutant DNA with a DIG-labelled probe specific for the gentamicin-resistance gene, and a DIG-labelled probe specific for *atuF*.

**Isolation of biotin-containing proteins, Western blot analysis and trypsin fingerprint analysis.** Late-exponential cells were collected by centrifugation at 4°C, and washed with mineral salt medium that did not contain a carbon source. The pellet was resuspended in 25 mM Tris/HCl buffer, pH 7·5, at 1 ml (g cells)<sup>-1</sup>. The suspension was passed two times through a French press cell at 800 p.s.i. (5·5 MPa). Cell debris and membrane fragments were removed by centrifugation at 80 000 g in a Beckman rotor Ti50 for 1 h at 4°C. The supernatant was used for further analysis. The isolation of biotin-containing proteins was done with immobilized monomeric avidin. A small column was packed with 250 μl immobilized monomeric avidin, and equilibrated with PBS containing 0·1 M phosphate buffer, 0·15 M NaCl, pH 7·0. A 250 μl volume of the crude extract was applied to the column, which was then washed six times with 1 vol. PBS. Biotin-containing proteins were eluted from the column with elution buffer (2 mM D-biotin in PBS). The column was regenerated by washing two times with 0·1 M glycine HCl, pH 2·8.

Samples were separated by 10% SDS-PAGE, and either stained with Coomassie blue, or transferred to PVDF membranes for Western blot analysis. Protein bands of interest were cut out, and subjected to trypsin digestion and subsequent LC-MS/MS analysis. Western blotting was done by standard procedures. The blotted biotin-containing proteins were tagged with Streptavidin-AP conjugate (Roche), and the colour development was done with nitroblue tetrazolium chloride (NBT) and 5-bromo-4-chloro-3-indolyl-phosphate *p*-toluidine salt (BCIP). Prestained dual-colour SDS-PAGE standard proteins (Bio-Rad) were used as marker proteins.

**Nanoflow liquid chromatography tandem MS (nano-LC-MS/MS).** Reverse-phase nano-LC-MS/MS was performed using an Ultimate nanoflow LC system (LC Packings). The LC system was coupled to a QSTAR Pulsar hybrid qTOF mass spectrometer (AB-MDS Sciex), which used a modified nanoelectrospray ion source (Proxeon Biosystems) interface. Briefly, the tryptic peptide mixtures

**Table 1. Bacterial strains and plasmids**

| Strain or plasmid          | Relevant characteristics                                                                                   | Reference or source               |
|----------------------------|-----------------------------------------------------------------------------------------------------------|------------------------------------|
| *E. coli* S17-1            | Mobilizing strain                                                                                        | Simon *et al.* (1983)             |
| *P. aeruginosa* PAO1       | Wild-type                                                                                                | ATCC 15692                        |
| *P. aeruginosa* PAO1 Sm<sup>r</sup> | Spontaneous streptomycin-resistant mutant of PAO1; Sm<sup>r</sup> (≥500 μg ml<sup>-1</sup>) | Höschle & Jendrossek (2005)       |
| *P. aeruginosa* PAO1 Sm<sup>r</sup> ins-*atuF* | Knockout mutant in *atuF* with pKnockout-G::*atuF* in the genome; Sm<sup>r</sup>, Gm<sup>r</sup> | This study                         |
| pKnockout-G                | Suicide vector for rapid gene inactivation in *P. aeruginosa*                                              | Windgassen *et al.* (2000)         |
| pKnockout-G::*atuF*        | Including a 3′- and 5′-truncated fragment of *atuF*; Ap<sup>r</sup>, Gm<sup>r</sup>                          | This study                         |

http://mic.sgmjournals.org

B. Höschle, V. Gnau and D. Jendrossek

were autosampled at a flow rate of 50 μl min⁻¹ in 0·1% aqueous trifluoroacetic acid, and desalted on a PepMap C18 trapping cartridge (LC Packings). The trapped peptides were eluted and separated on the analytical column (PepMap C18, 75 μm i.d. × 15 cm; LC Packings) using a linear gradient of 7–50% acetonitrile in 0·5% formic acid for 30 min at a flow rate of 200 nl min⁻¹, and ionized by an applied voltage of 2·2 kV to the emitter.

The mass spectrometer was operated in data-dependent acquisition mode to automatically switch between MS and MS/MS. Survey MS spectra were acquired for 1 s, and the three most intense ions (doubly, triply or quadruply charged) were isolated, and sequentially fragmented for 1·5 s by low-energy collision-induced dissociation. All MS and MS/MS spectra were acquired with the Q2-pulsing function switched on, and optimized for enhanced transmission of ions in the MS (*m/z* 400–1000) and MS/MS (*m/z* 75–1300) mass ranges.

---

### RESULTS

Biotin-containing proteins of *P. aeruginosa* PAO1 were purified from (i) glucose-, (ii) citronellate- and (iii) isovalerate-grown cells, separated by reducing SDS-PAGE (Fig. 2A), and screened for biotin by Western blot analysis (Fig. 2B), as described in Methods. Only one biotin-containing polypeptide (apparent molecular mass 63 kDa) was found in glucose-grown cells. This protein was constitutively expressed, and apparently is not involved in carboxylation of methyl-branched compounds (see below). One additional biotin-containing polypeptide appeared at about 74 kDa in isovalerate-grown cells. This protein presumably corresponds to the biotin-containing subunit of MCase, because the catabolic pathway of leucine/isovalerate involves only one carboxylase step (MCase). Specific induction of MCase by leucine and isovalerate has been shown for *P. citronellolis* (Fall & Hector, 1977; Hector & Fall, 1976a, b). In citronellate-grown cells of *P. aeruginosa*, a third biotin-dependent protein at about 71 kDa appeared in addition to the 63 and 74 kDa proteins (Fig. 2B). Since GCase is necessary only in citronellol/citronellate-grown cells, but not in isovalerate-grown cells, the 71 kDa protein is thought to correspond to GCase.

The purified biotin-containing proteins of citronellate-grown cells (Fig. 2A) were cut out from the Coomassie-stained SDS-polyacrylamide gel, and subjected to trypsin fingerprint analysis and ESI-MS-MS (double or multiple mass spectra)-based amino acid sequencing. At least 13 peptides of each isolated biotin-containing protein were analysed, and compared with the *P. aeruginosa* database. The 63 kDa biotin band (band no. 3, Fig. 2A) perfectly matched to the putative product of ORF PA5435 (putative acetyl-CoA carboxylase/transcarboxylase subunit), and the 74 kDa (no. 1) and 71 kDa (no. 2) biotin-containing proteins were unequivocally identified as the gene products of *liuD* (PA2012; 74 kDa) and *atuF* (PA2891; 71 kDa), respectively (Figs 2 and 3). The three non-biotin-containing polypeptides (band nos 4/5 and 6, Fig. 2A) corresponded to *liuB* (PA2014), *atuC* (PA2888) and the product of PA5436,

---

**Fig. 2.** SDS-PAGE and Western blot analysis of *P. aeruginosa* PAO1 for biotin-containing proteins. After avidin purification, soluble cell extracts were separated on 10% SDS-polyacrylamide gels and stained with Coomassie blue (A), or transferred to PVDF membranes and screened for biotin-containing proteins (B), as described in Methods. To obtain sufficient resolution in the range above 50 kDa, electrophoresis was carried out for about 90 min (20–25 mA) so that proteins of less than ~25 kDa had run out of the gel. Gene products identified by trypsin fingerprint LC-MS analysis of isolated protein bands are indicated. The numbering of the protein bands in (A) is described in the text.

(A) P. aeruginosa atu cluster


atuR? (PA2885)
atuA (PA2886)
atuB (PA2887)
atuC (PA2888)
atuD (PA2889)
atuE (PA2890)
atuF (PA2891)
atuG (PA2892)
atuH (PA2893)

Probable transcriptional regulator
MW: 22057

Hypothetical protein
MW: 64436

Probable short-chain dehydrogenase
MW: 30751

Probable acyl-CoA dehydrogenase
MW: 42714

Probable enoyl-CoA hydratase/isomerase
MW: 27718

Geranyl-CoA carboxylase, alpha chain
(biotin-containing)
MW: 71748

Geranyl-CoA carboxylase, beta chain
MW: 57328

Probable short-chain dehydrogenase
MW: 29631

Probable very-long-chain
acyl-CoA synthase
MW: 67648


(B) P. aeruginosa liu cluster


liuE (PA2011)
liuD (PA2012)
liuC (PA2013)
liuB (PA2014)
liuA (PA2015)
liuR (PA2016)

Hydroxymethylglutaryl-CoA lyase
MW: 31837

Methylcrotonyl-CoA carboxylase
beta chain
MW: 57434

Methylcrotonyl-CoA carboxylase
alpha chain (biotin-containing)
MW: 71283

Probable acyl-CoA dehydrogenase
MW: 42207

Probable enoyl-CoA hydratase/isomerase
MW: 28947

Probable transcriptional regulator
MW: 15631


(C) P. putida, putative liu cluster


liuA/fvd (PP4064)
liuB (PP4065)
liuC (PP4066)
liuD (PP4067)
? (PP4068)

Isovaleryl-CoA dehydrogenase
MW: 47797

Acetyl-CoA carboxylase, biotin
carboxylase, putative
MW: 69729

3-Methylcrotonyl-CoA carboxylase,
beta subunit, putative
MW: 57505

Transcriptional regulator,
Cro/Cl family
MW: 26361

Enoyl-CoA hydratase, putative
MW: 30015


Fig. 3. Gene clusters of P. aeruginosa PAO1 (A, B) (atu and liu, respectively), and putative liu cluster of P. putida (C) (data and annotations are from http://www.pseudomonas.com and http://www.tigr.org). Note that the involvement of PA2885 (atuR?) in Atu is hypothetical, and is not supported by experimental data. The deduced molecular weight (MW) of each gene product in Da is shown.

respectively, and represent the non-biotin-containing carboxylase subunits (LiuB and AtuC were not separated in the double band no. 4/5). The genes liuA and atuF have been annotated as biotin-containing subunits of carboxylases in the P. aeruginosa database (Fig. 3). LiuB and liuD have been proposed to encode GCase (corresponding to gnyA and gnyB; Diaz-Perez et al., 2004), and PA2888 and PA2891 (atuC and atuF) have been suggested to encode MCase; this is contrary to our findings. Our Western blot results shown in Fig. 2, i.e. the findings that the 71 kDa band was expressed only in citronellate-grown cells, and that the 74 kDa band was expressed in both citronellate- and isovalerate-grown cells, suggest that liuB and liuD encode MCase (74 kDa), and that atuC and atuF encode GCase (71 kDa). The DNA-deduced molecular masses of LiuD and AtuF are 71 283 and 71 748 Da, respectively. The reason for the discrepancy in the theoretical values compared with the observed apparent molecular masses is unknown, but it is often observed in SDS-PAGE, and may depend on the individual amino acid sequence/composition.

B. Höschle, V. Gnau and D. Jendrossek

What could be the reason for the contrary conclusions concerning the assignment of LiuB/LiuD and AtuC/AtuF to MCase and GCase subunits? Diaz-Perez *et al.* (2004) also noticed the relatedness of LiuB/LiuD and AtuC/AtuF; they assumed that the compound 6-methyl-5-hepten-2-one would enter the combined citronellol/leucine degrading pathway at the level of methylcrotonyl-CoA, and considered growth of insertion mutants in the *liu* gene cluster on 6-methyl-5-hepten-2-one as evidence for the functionality of the leucine utilization pathway. However, it is unclear how catabolism of the eight-carbon-atom-containing compound 6-methyl-5-hepten-2-one concurs with the five-carbon-atom compound methylcrotonylate (CoA-derivative). Nevertheless, we tested growth of *P*. *aeruginosa* PAO1 on 6-methyl-5-hepten-2-one, and, surprisingly, we did not observe significant growth on this compound. We tested six other strains of *P*. *aeruginosa* (HABS1, CF-3MR, PAC1, B104, PA81 and HAB911), and we used different concentrations and different methods of carbon source application (filter paper soaked in 6-methyl-5-hepten-2-one, and inserted into the lid of the Petri dish; 20 μl 6-methyl-5-hepten-2-one spotted into the space between the bottom of the Petri dish and the agar; and 6-methyl-5-hepten-2-one applied by saturated atmosphere in a sealed container). We did not find growth for any of the seven strains that was stronger than that of controls without the carbon source (30 and 37 °C, and periods of up to 1 week, were tested). We concluded that *P*. *aeruginosa* is unable to use 6-methyl-5-hepten-2-one as a sole source of carbon and energy, and that growth on 6-methyl-5-hepten-2-one is not a suitable tool to test the functionality of the Liu pathway in *P*. *aeruginosa*. The assay for growth on isovalerate and/or leucine appears to be a more reliable tool for this purpose, since leucine and isovalerate are precursors of methylcrotonyl-CoA in catabolic pathways of many organisms (Fig. 1). Inspection of the genome of another related species, *Pseudomonas putida*, a bacterium that is able to grow on isovalerate, but unable to utilize citronellol or citronellate, and therefore is unlikely to have the genes for the Atu pathway, revealed the presence of a cluster containing four genes (PP4064–PP4067, Fig. 3). The deduced amino acid sequences of ORFs PP4064–PP4067 showed a much higher degree of similarity (67–87% amino acid identity) to LiuA–LiuE of *P*. *aeruginosa*, but only a medium level of similarity to AtuA–AtuH (35–51% identity). We assume that ORFs PP4064–PP4067 code for several genes of the leucine/isovalerate-degrading pathway in *P*. *putida*, and we suggest naming them *liu* genes (Fig. 3).

To find further experimental evidence for our interpretation, insertion mutagenesis of *atuF* (putative GCase biotin subunit, 71 kDa) was performed as described in Methods. The success of insertion mutagenesis was confirmed by PCR and Southern blot analysis of genomic DNA isolated from the respective mutants (data not shown). When the *P*. *aeruginosa* mutant with an insertion in *atuF* was tested for utilization of methyl-branched compounds, no growth was found on aliphatic monoterpenes such as citronellol, citronellate, geraniol and geranylate, confirming the involvement of *atuF* in utilization of citronellol and related monoterpenes. However, utilization of isovalerate and leucine was the same in the *atuF* mutant as in the wild-type. Growth on substrates unrelated to methyl-branched compounds (e.g. glucose, succinate or acetate) was also not affected. This result clearly shows that MCase and a functional Liu pathway are present in the *atuF* insertion mutant (Fig. 1). Accordingly, Western blot analysis of isovalerate-grown mutant cells for the presence of biotin-containing proteins showed the expected 74 kDa band, and confirmed that MCase was expressed (Fig. 4). After growth on isovalerate, the biotin-containing proteins of the *atuF* mutant were purified by avidin-affinity chromatography. The identity of the isolated 74 kDa band with the *liuA* gene product, and with the 74 kDa protein of the wild-type, was confirmed by trypsin fingerprint analysis of the avidin-isolated protein. The 63 kDa protein was again identified as PA5435 product (putative acetyl-CoA carboxylase/transcarboxylase subunit). The 71 kDa biotin protein was not found in mutant cells that had been incubated in the presence of citronellate for different time periods (Fig. 4), and this confirmed the absence of GCase in the *atuF* insertion mutant. The non-biotin-containing subunits of the purified proteins (58 and 48 kDa) were identified to correspond to LiuB and PA5436 product by trypsin fingerprint analysis MS (data not shown).

## DISCUSSION

In this study, we continued our investigation of the catabolism of acyclic (mono-) terpenes in *P*. *aeruginosa*. A

![Figure 4](attachment://image.png)

**Fig. 4.** Western blot analysis of biotin-containing proteins in avidin-purified crude extracts of *P*. *aeruginosa* (*atuF*-insertion mutant) grown on glucose (0·5%), glucose (0·1%) plus citronellate (0·2%), and glucose (0·1%) plus isovalerate (0·2%).


putative pathway leading from citronellol to acetate, acetyl-CoA and acetoacetate as end products, based on biochemical investigations in *P. citronellolis* by Seubert and coworkers more than 40 years ago (Seubert, 1960; Seubert *et al.*, 1963; Seubert & Remberger, 1963; Seubert & Fass, 1964a, b), is shown in Fig. 1. The pathway includes three phases. (i) Citronellol is first oxidized and activated to geranyl-CoA. Geranyl-CoA is subsequently carboxylated by GCase, hydrated, and the resulting product is cleaved into acetate and 7-methyl-3-oxo-6-octenoyl-CoA (Atu pathway). (ii) 7-Methyl-3-oxo-6-octenoyl-CoA can be degraded by two rounds of subsequent β-oxidation reactions without hindrance by β-methyl groups. (iii) The product 3-methylcrotonyl-CoA concurs with the Liu pathway, which includes a second carboxylation step by MCase. The latter carboxylase is specific for 3-methylcrotonyl-CoA, and is unable to utilize geranyl-CoA as a substrate (Fall & Hector, 1977). Acetate, acetyl-CoA and acetoacetate are the end products of the combined pathways. While the Atu pathway is present presumably in citronellol-degrading bacteria only (*P. citronellolis*, *P. aeruginosa* and *P. mendocina*), the Liu pathway probably is present in many organisms, at least in isovalerate- and leucine-utilizing bacteria. The finding that GCase (71 kDa) is expressed only in citronellate-grown cells, and that MCase (74 kDa) is expressed in both citronellate-grown and isovalerate-grown cells, is in agreement with the postulated pathway: MCase as a key enzyme of the Liu pathway is needed in equal measure for catabolism of citronellol, and for catabolism of leucine and isovalerate; GCase is needed only for utilization of acyclic (mono-) terpenes such as citronellol and geraniol. MCase can not replace GCase, since it is specific for methylcrotonyl-CoA, and does not accept geranyl-CoA as a substrate (Fall & Hector, 1977; Hector & Fall, 1976a, b).

To determine whether the key enzymes GCase and MCase of the combined pathways are expressed in a similar manner in *P. aeruginosa* and *P. citronellolis*, we performed Western blot analysis for biotin-containing proteins (Fig. 2). Indeed, two biotin-containing polypeptides (74 and 71 kDa) are additionally expressed in citronellate-grown cells compared with glucose-grown cells. Isovalerate-grown cells, however, contain only one additional biotin-containing band (74 kDa) compared with glucose-grown cells. These results show that expression of GCase and MCase in *P. aeruginosa* is the same as in *P. citronellolis*. Analysis of the DNA sequences of the biotin-containing subunits of the two candidate carboxylase genes (*liuD* and *atuF*) shows that the deduced amino acid sequences are highly similar (51% identity), and that the polypeptides have almost the same theoretical molecular masses of 71·3 and 71·7 kDa, respectively. A difference of less than 1 kDa is not sufficient to distinguish the products simply by their migration distance in SDS-PAGE. We therefore identified the respective gene products by trypsin fingerprint analysis, and subsequent ESI-MS^n^ sequencing of isolated peptides. This procedure enabled us to assign the bands in biotin blots to the two citronellate-specific biotin-containing proteins AtuF and LiuD (71 and 74 kDa

apparent molecular masses, respectively). This assignment is different from the earlier findings of Diaz-Perez *et al.* (2004).

Considering the results of Diaz-Perez and coworkers (Diaz-Perez *et al.*, 2004), and the findings of this contribution, it is evident that both clusters (*atuABCDEF*GH and *liuRABCDE*) are essential for catabolism of methyl-branched compounds such as citronellol, citronellate, leucine and isovalerate. We propose to name the cluster PA2886–PA2893 as the *atu* cluster (acyclic (mono-) terpene utilization, *atuABCDEF*GH), and the PA2011–PA2016 cluster as the *liu* gene cluster (leucine and isovalerate utilization, *liuRABCDE*) (Fig. 3). The name *gny* (geranyl), as suggested by Diaz-Perez *et al.* (2004), does not appear to be suitable for the *liu* cluster, since our results clearly show that the *liu* genes are only indirectly involved in the degradation pathway of acyclic terpenes.

Utilization of geraniol apparently requires the presence of additional gene products such as MoeA2, since oxidation of geranial and/or geraniol, in contrast to oxidation of citronellol, includes a molybdenum-dependent step (Höschle & Jendrossek, 2005). We predict that mutants defective in *liuD*, and probably most mutants defective in one of the other *liu* genes, will be shown to be unable to utilize leucine or isovalerate as a carbon source, in addition to the already known minus phenotype on acyclic monoterpenes (Diaz-Perez *et al.*, 2004).

## ACKNOWLEDGEMENTS

We thank A. Jakob for performing trypsin fingerprint and LC-MS analyses.

## REFERENCES

Burke, Y. D., Stark, M. J., Roach, S. L., Sen, S. E. & Crowell, P. L. (1997). Inhibition of pancreatic cancer growth by the dietary isoprenoids farnesol and geraniol. *Lipids* 32, 151–156.

Burke, Y. D., Ayoubi, A. S., Werner, S. R., McFarland, B. C., Heilman, D. K., Ruggeri, B. A. & Crowell, P. L. (2002). Effects of the isoprenoids perillyl alcohol and farnesol on apoptosis biomarkers in pancreatic cancer chemoprevention. *Anticancer Res* 22, 3127–3134.

Cantwell, S. G., Lau, E. P., Watt, D. S. & Fall, R. R. (1978). Biodegradation of acyclic isoprenoids by *Pseudomonas* species. *J Bacteriol* 135, 324–333.

Carnesecchi, S., Bradaia, A., Fischer, B., Coelho, D., Scholler-Guinard, M., Gosse, F. & Raul, F. (2002a). Perturbation by geraniol of cell membrane permeability and signal transduction pathways in human colon cancer cells. *J Pharmacol Exp Ther* 303, 711–715.

Carnesecchi, S., Langley, K., Exinger, F., Gosse, F. & Raul, F. (2002b). Geraniol, a component of plant essential oils, sensitizes human colon cancer cells to 5-fluorouracil treatment. *IARC Sci Publ* 156, 407–409.

Carnesecchi, S., Bras-Goncalves, R., Bradaia, A., Zeisel, M., Gosse, F., Poupon, M. F. & Raul, F. (2004). Geraniol, a component of plant essential oils, modulates DNA synthesis and potentiates 5-fluorouracil efficacy on human colon tumor xenografts. *Cancer Lett* 215, 53–59.

Diaz-Perez, A. L., Zavala-Hernandez, A. N., Cervantes, C. & Campos-Garcia, J. (2004). The *gnyRDBHAL* cluster is involved in

B. Höschle, V. Gnau and D. Jendrossek

acyclic isoprenoid degradation in *Pseudomonas aeruginosa*. Appl Environ Microbiol 70, 5102–5110.

Duncan, R. E., Lau, D., El-Sohemy, A. & Archer, M. C. (2004). Geraniol and beta-ionone inhibit proliferation, cell cycle progression, and cyclin-dependent kinase 2 activity in MCF-7 breast cancer cells independent of effects on HMG-CoA reductase activity. Biochem Pharmacol 68, 1739–1747.

Fall, R. R. (1981). 3-Methylcrotonyl-CoA and geranyl-CoA carboxylases from *Pseudomonas citronellolis*. Methods Enzymol 71, 791–799.

Fall, R. R. & Hector, M. L. (1977). Acyl-coenzyme A carboxylases. Homologous 3-methylcrotonyl-CoA and geranyl-CoA carboxylases from *Pseudomonas citronellolis*. Biochemistry 16, 4000–4005.

Fall, R. R., Brown, J. L. & Schaeffer, T. L. (1979). Enzyme recruitment allows the biodegradation of recalcitrant branched hydrocarbons by *Pseudomonas citronellolis*. Appl Environ Microbiol 38, 715–722.

Hector, M. L. & Fall, R. R. (1976a). Multiple acyl-coenzyme A carboxylases in *Pseudomonas citronellolis*. Biochemistry 15, 3465–3472.

Hector, M. L. & Fall, R. R. (1976b). Evidence for distinct 3-methylcrotonyl-CoA and geranyl-CoA carboxylases in *Pseudomonas citronellolis*. Biochem Biophys Res Commun 71, 746–753.

Hierro, I., Valero, A., Perez, P., Gonzalez, P., Cabo, M. M., Montilla, M. P. & Navarro, M. C. (2004). Action of different monoterpenic compounds against *Anisakis simplex* s.l. L3 larvae. Phytomedicine 11, 77–82.

Höschle, B. & Jendrossek, D. (2005). Utilization of geraniol is dependent on molybdenum in *Pseudomonas aeruginosa*: evidence for different metabolic routes for oxidation of geraniol and citronellol. Microbiology 151, 2277–2283.

Izumi, S., Takashima, O. & Hirata, T. (1999). Geraniol is a potent inducer of apoptosis-like cell death in the cultured shoot primordia of *Matricaria chamomilla*. Biochem Biophys Res Commun 259, 519–522.

Pawar, P. V., Sharma, R. N., Phadnis, A. P., Nanda, B. & Patwardhan, S. A. (1991). Action of some insect growth regulators on mosquito vectors: part I – citronellol based diethers. *J Commun Dis* 23, 118–122.

Rice, P. J. & Coats, J. R. (1994). Insecticidal properties of several monoterpenoids to the house fly (*Diptera: Muscidae*), red flour beetle (*Coleoptera: Tenebrionidae*), and southern corn rootworm (*Coleoptera: Chrysomelidae*). *J Econ Entomol* 87, 1172–1179.

Schlegel, H. G., Kaltwasser, H. & Gottschalk, G. (1961). A submersion method for culture of hydrogen-oxidizing bacteria: growth physiological studies. *Arch Mikrobiol* 38, 209–222.

Seubert, W. (1960). Degradation of isoprenoid compounds by microorganisms. I. Isolation and characterization of an isoprenoid-degrading bacterium, *Pseudomonas citronellolis* n. sp. *J Bacteriol* 79, 426–434.

Seubert, W. & Fass, E. (1964a). Studies on the bacterial degradation of isoprenoids. V. The mechanism of isoprenoid degradation. *Biochem Z* 341, 35–44.

Seubert, W. & Fass, E. (1964b). Studies on the bacterial degradation of isoprenoids. IV. The purification and properties of beta-isohexenylglutaconyl-CoA-hydratase and beta-hydroxy-beta-isohexenylglutaryl-CoA-lyase. *Biochem Z* 341, 23–34.

Seubert, W. & Remberger, U. (1963). Studies on the bacterial degradation of isoprenoid compounds. II. The role of carbon dioxide. *Biochem Z* 338, 245–264.

Seubert, W., Fass, E. & Remberger, U. (1963). Studies on the bacterial degradation of isoprenoid compounds. III. Purification and properties of geranyl carboxylase. *Biochem Z* 338, 265–275.

Simon, R., Priefer, U. & Pühler, A. (1983). A broad host range mobilization system for *in vivo* genetic engineering: transposon mutagenesis in Gram-negative bacteria. *Bio/Technology* 1, 784–791.

Windgassen, M., Urban, A. & Jaeger, K. E. (2000). Rapid gene inactivation in *Pseudomonas aeruginosa*. *FEMS Microbiol Lett* 193, 201–205.
