
# Cell-Cycle Control of Gene Expression in Budding and Fission Yeast

**Jürg Bähler**

Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom; email: jurg@sanger.ac.uk

---

### Key Words

Saccharomyces cerevisiae, Schizosaccharomyces pombe, transcription, forkhead, cell division, microarray

---

### Abstract

Cell-cycle control of transcription seems to be a universal feature of proliferating cells, although relatively little is known about its biological significance and conservation between organisms. The two distantly related yeasts *Saccharomyces cerevisiae* and *Schizosaccharomyces pombe* have provided valuable complementary insight into the regulation of periodic transcription as a function of the cell cycle. More recently, genome-wide studies of proliferating cells have identified hundreds of periodically expressed genes and underlying mechanisms of transcriptional control. This review discusses the regulation of three major transcriptional waves, which roughly coincide with three main cell-cycle transitions (initiation of DNA replication, entry into mitosis, and exit from mitosis). I also compare and contrast the transcriptional regulatory networks between the two yeasts and discuss the evolutionary conservation and possible roles for cell cycle-regulated transcription.

Contents

INTRODUCTION 70
REGULATION OF GENE EXPRESSION PEAKING DURING G1/S 71
MBF and SBF Transcription Factor Complexes 71
Cell-Cycle Regulation of MBF and SBF Activity 72
Genes Regulated by SBF and MBF 73
REGULATION OF GENE EXPRESSION PEAKING DURING G2/M 74
Forkhead Transcription Factor Complexes 74
Cell Cycle-Regulation of Forkhead Activity 74
Forkhead-Dependent Transcription in Fission Yeast 75
REGULATION OF GENE EXPRESSION PEAKING DURING M/G1 77
The Mcm1p Transcription Factor 77
The Ace2p and Swi5p Transcription Factors 77
Cell Cycle-Regulation of Ace2p and Swi5p Activity 77
Ace2p in Fission Yeast 78
TRANSCRIPTIONAL REGULATORY NETWORKS 78
CONSERVATION AND IMPORTANCE OF CELL CYCLE-REGULATED TRANSCRIPTION 80

specific cell-cycle phases. This first became evident with the discovery that histone mRNA levels oscillate as a function of the cell cycle and peak during DNA replication in both budding and fission yeast (67, 106). Since this finding, hundreds of genes that are periodically expressed during the cell cycle have been identified in yeast and other organisms. Many of these genes have specific roles in the cell cycle, and their expression peaks coincide with the phase during which they function. Much has also been learned about transcription factors (TFs) that are involved in the regulated expression of periodic genes, and about mechanisms to integrate TF activity with overall cell-cycle control. Phase-specific control of gene expression appears to be a universal feature of the cell-division cycle. Genome-wide studies of transcription and its regulation in proliferating cells are now starting to provide a global perspective on this aspect of cell-cycle control (29, 63, 159, 168a, 171).

This review summarizes the current knowledge of cell cycle-regulated transcription in both budding yeast (*Saccharomyces cerevisiae*) and fission yeast (*Schizosaccharomyces pombe*), revealing similarities as well as intriguing differences between these two distantly related model organisms. Cell cycle-regulated transcription and its integration with overall cell-cycle control has been worked out in greater detail for budding yeast, but fission yeast provides a valuable complementary system that can give unique insight and help to define universal principles, as it does in other fields (59). A main focus is on recent data obtained through global genomic approaches; for more comprehensive surveys of the older data, the reader is referred to previous reviews (26, 99, 157). In both yeasts, transcriptional control during the G1/S transition (initiation of DNA replication) has been most widely studied, but recently much progress has also been made in dissecting the regulation of transcription during other cell-cycle phases such as the G2/M transition (entry into

---

**Transcription factor (TF):** a protein that regulates the initiation of transcription for a specific set of genes

---

INTRODUCTION

Proliferation of all cells is mediated through the cell-division cycle, which consists of four main phases: genome duplication (S-phase) and nuclear division (mitosis or M-phase), separated by two gap phases (G1 and G2). Transcription of a number of genes peaks at

mitosis) and the M/G1 transition (exit from
mitosis).

**REGULATION OF GENE**
**EXPRESSION PEAKING DURING**
**G1/S**

MBF and SBF Transcription Factor
Complexes

At the end of G1-phase, cells decide whether
to commit to cell division in a process called
Start in yeast or restriction point in mam-
malian cells. The transcript levels of several
genes increase during late G1, promoting the
initiation of DNA replication and other events
of the G1/S transition. In budding yeast, two
related TF complexes activate the expression
of these genes: MBF (MCB-binding factor;
also known as DSC1, for DNA synthesis con-
trol complex) and SBF (SCB-binding factor).
The MBF complex consists of two protein
components, Mbp1p and Swi6p, and binds to
regulatory promoter sites named MCB (MluI
cell-cycle box), because they are related to
the recognition site of the MluI restriction
enzyme (Table 1) (26, 81). These MCB el-
ements are both necessary and sufficient for
cell-cycle-regulated transcription (94, 113).
The SBF complex consists of Swi4p and
Swi6p and binds to promoter sites named SCB
(Swi4/6-dependent cell-cycle box) (Table 1),

which are sufficient to promote G1/S-specific
transcription in heterologous genes (8, 9,
27, 124). The highly similar Mbp1p and
Swi4p proteins are the DNA-binding com-
ponents of MBF and SBF, respectively, while
Swi6p plays a regulatory role in both TF
complexes (52, 93, 124, 133). All three pro-
teins contain ankyrin repeats within a cen-
tral domain, indicating a common ancestry
(23, 27).

In fission yeast, a related MBF/DSC1TF
complex also activates gene expression dur-
ing the G1/S transition by binding to MCB
promoter sites, which are conserved between
the two yeasts (Table 1) (10, 32, 95, 102,
116, 155, 174). The fission yeast MBF com-
plex contains Cdc10p and at least two addi-
tional, related ankyrin-repeat proteins, Res1p
and Res2p, which bind to Cdc10p at their
C termini, whereas their N termini contain
DNA-binding domains and are involved in
heterodimer formation with each other (11,
12, 165, 175). The fission yeast MBF complex
also plays an important role during premei-
otic DNA replication, after minor modifica-
tions in its composition and with altered target
gene specificity (11, 44, 50, 105, 110). During
budding yeast meiosis, a subset of DNA repli-
cation genes are regulated by MBF, while the
meiotic regulation of the cyclin gene CLB5 is
independent of MBF yet still requires MCB
promoter elements (134).

---

**MCB:** MluI
cell-cycle box

**MBF:** MCB-binding factor

**SCB:** Swi4/6-dependent
cell-cycle box

**SBF:** SCB-binding
factor

**MBF/SBF:** yeast
TF complexes that
regulate the
initiation of DNA
replication;
functional equivalent
of human E2F-DP
complexes

---

**Table 1 Selected binding sites for cell-cycle transcription factors (TFs)**

| Name       | Phase   | Budding yeast          | Fission yeast         |
|------------|---------|------------------------|-----------------------|
|            |         | Binding site<sup>a</sup> | TF                    | Binding site<sup>a</sup> | TF                   |
| MCB        | G1/S    | ACGCGT                 | MBF                  | ACGCGT                | MBF                  |
| SCB        | G1/S    | CRCGAAA<sup>b</sup>     | SBF                  | Not described         | Not described        |
| SFF/Forkhead | G2/M    | Mcm1<sup>c</sup>+GTAAACAA<sup>b</sup> | Mcm1p+Fkh2p           | GTAAACAA<sup>b</sup>  | Sep1p?Fkh2p?         |
| PCB        | G2/M    | Not described          | Not described         | GNAACR               | PBF, Mbx1?           |
| ECB        | M/G1    | YAATTA+Mcm1<sup>c</sup> | Yox1/Yhp1p+Mcm1p      | Not described         | Not described        |
| Swi5/Ace2  | M/G1    | ACCAGCR<sup>b</sup>     | Swi5p, Ace2p          | ACCAGCCNT<sup>b</sup>  | Ace2p                |

<sup>a</sup> Different versions of binding sites have been described, and typical sequence patterns were selected for this table to represent each
binding site.

<sup>b</sup> These sites are also present in reverse orientation.

<sup>c</sup> A typical Mcm1 site is: TTWCCYAAWNNGGWAA.
N is any base; R is A or G; Y is C or T; W is A or T. See main text for references on the various binding sites.

www.annualreviews.org • Cell-Cycle Transcription in Yeasts 71

AR260-GE39-05 ARI 10 October 2005 18:9

**Cell-Cycle Regulation of MBF and SBF Activity**

Knowledge of specific roles for various sub-units of the MBF and SBF complexes as well as the cell-cycle regulation of their activities is still limited. Regulation is complex and coordinated at different levels (Figure 1). Both *S. cerevisiae* Swi6p and *S. pombe* Cdc10p appear to have negative as well as positive roles in MCB-mediated transcription (26, 111). In budding yeast, the Cln3p-Cdk1p/Cdc28p CDK (cyclin-dependent kinase) is a key regulator for transcriptional activation of MBF- and SBF-dependent genes (51, 149, 160), and genetic data point to Swi6p as a critical, but indirect, target of Cln3p (168). SBF and MBF bind to their target genes in early G1, but the RNA polymerase II machinery is only recruited to these promoters upon activation of Cln3p-Cdk1p in late G1-phase (40). In a recent breakthrough, Whi5p has been identified as the long-sought intermediate factor between Cln3p and SBF/MBF: It inhibits G1/S transcription through association with SBF until CDK-dependent phosphorylation during late G1 drives Whi5p out of the nucleus and relieves its inhibition (42, 46). This regulatory pathway is satisfyingly analogous to the cyclin D-Rb-E2F circuit that controls G1/S transcription in mammals and is deregulated in most cancer cells (140). Other activators such as Bck2p (167) and Stb1p (43, 68) regulate G1/S transcription in parallel to Cln3p, but their roles are less clear. The onset of SBF- and MBF-mediated transcription is dependent on a minimal cell-size threshold. Relatively poorly understood is how the cell measures and signals cell size requirements to SBF and MBF, although recent research gives fascinating insight into the coordination of cell division with cell growth (reviewed in 77).

Swi4p is also regulated at the transcriptional level, both by ECB binding sites in its promoter acting during M/G1 progression (see below) and by a positive feedback involving SBF and MBF (60, 108, 112, 145). In G2-phase, MBF- and SBF-mediated transcription is switched off through the Clb1/2p-Cdk1p CDK complex (6, 82, 148), which is itself activated by a combination of events following transcriptional activation of CLN1 and CLN2 by SBF and MBF (Figure 1) (reviewed in 63). Moreover, phosphorylation of Swi6p by the Clb6p-Cdk1p CDK, which is also transcriptionally activated by SBF and MBF, leads to nuclear export of Swi6p during S-phase, whereas dephosphorylation of Swi6p by the mitotic exit phosphatase Cdc14p leads to nuclear import of Swi6p during G1-phase.

---

**CDK:**  
cyclin-dependent kinase  

**Binding site:** short regulatory DNA motif located upstream of a protein coding region (promoter) that is recognized by a specific TF to regulate gene activity  

---

**Figure 1**  
Scheme of main regulatory circuits that drive the gene expression program during the budding yeast cell cycle. TFs are shown in red. Red and blue lines indicate transcriptional and posttranslational regulation, respectively. Arrows: positive regulation; bars: negative regulation. The cell-cycle transitions where the main TFs act are indicated on the left side. See main text for details and references.
(64, 143). The Paf1-RNA polymerase II complex is specifically required for full expression of many SBF- and MBF-regulated genes but not for their periodic regulation (131).

In fission yeast, MBF is constitutively bound throughout the cell cycle to the MCB site in the promoter of the *cdc18* target gene (169), although it activates transcription specifically in late G1-phase. The role of CDK in MBF regulation is less clear than for budding yeast. It has been suggested that activation of MBF-dependent gene expression is triggered by Cdk1p/Cdc2p CDK during G1-phase (135), and Cdk1p activity has been reported to be required for Cdc10p/Res1p complex formation (38). However, other data indicate that MBF activity itself is unlikely to be controlled by CDK (18). Instead, the Pcl-like cyclin Pas1p, when associated with its kinase partner Pef1p, may activate MBF during the G1/S transition (154). Additional proteins such as Rep2p are crucial for regulating the activity of the MBF complex (18, 122, 151, 164). Transcript levels of Rep2p, Cdc10p, and Res2p slightly increase coincident with MBF target genes (126, 138). It has also been hypothesized that an activator of MBF-regulated transcription is degraded by the SCF ubiquitin ligase during S-, G2-, and M-phases, when MBF is inactive (172). Moreover, the cyclin Cig2p, whose periodic transcription is dependent on MBF, inhibits MBF activity in G2-phase by negative feedback through binding to Res2p and phosphorylation of Res1p by Cdk1p-Cig2p CDK (13). *cig2* expression is slightly delayed relative to the expression of other MBF target genes, suggesting that *cig2* may be subject to somewhat different cell-cycle controls (17).

and SBF have been globally mapped using a combination of chromatin immunoprecipitation and microarrays (75, 145). These studies have identified about 200 new putative targets, although genome-wide datasets can be noisy when used on their own. MBF and SBF have partially redundant functions as reflected by the substantial overlap between their target genes. However, both factors also play distinct roles: Genes bound by SBF predominantly function in budding as well as membrane and cell-wall biosynthesis, while MBF binds many genes involved in DNA replication and repair (75, 145). The functional specialization of MBF and SBF may allow for independent regulation of distinct molecular processes that normally occur at the same time during the budding yeast cell cycle.

Although SBF and MBF bind to the promoters of more than 200 genes, they are associated with only a minority of the genes whose transcript levels peak at the G1/S transition in budding yeast. Additional regulators, such as Skn7p and Stb1p, appear to contribute directly to G1/S transcription (15, 25, 43, 90, 91). Some functions of Mbp1p such as bud emergence require its interaction with the response regulator Skn7p instead of Swi6p (25, 118). Moreover, SBF activates at least nine additional TFs that in turn induce the expression of further genes during the G1/S transition (71). Although these factors functionally overlap to some degree with SBF and MBF, they each also have specialized functions. Most of these TFs bind to the promoters of yet other TFs involved in cell-cycle progression and/or differentiation. Thus, SBF launches a complex transcriptional cascade, and a large network of TFs is involved in coordinating processes at the beginning of the budding yeast cell cycle (71).

In fission yeast, more than ten MBF target genes have been reported (102, and references cited therein). Global gene expression profiling recently identified dozens of additional potential MBF target genes, although the number of genes regulated by fission yeast MBF seems to be substantially lower than in

**Genes Regulated by SBF and MBF**

A number of SBF and MBF target genes have been identified by traditional approaches in budding yeast (reviewed in 26). More recently, the genomic binding sites of MBF

**Microarray:** technology for genome-wide measurements of transcript levels or TF binding regions by hybridization to DNA spots arrayed at high density on slides.
budding yeast (127, 138). Genes whose expression is regulated by MBF mainly function in DNA replication or cell-cycle control. Unlike in budding yeast, genes for membrane and cell wall formation do not appear to be regulated, possibly reflecting differences in the growth cycles and the absence of an SBF complex in fission yeast.

**REGULATION OF GENE EXPRESSION PEAKING DURING G2/M**

Although the factors regulating G1/S transcription have been known for many years, the identification of factors that regulate the G2/M transcriptional program promoting entry into mitosis was more difficult, because genetic analyses proved elusive. Studies of the SWI5 promoter in budding yeast revealed that its periodic expression requires Mcm1p, which binds to promoters as a dimer together with a ternary complex factor initially termed SFF (Swi five factor) (4, 97, 100). Mcm1p is a ubiquitous MADS-box TF similar to human serum response factor; MADS box proteins are combinatorial TFs that often derive their regulatory specificity from accessory factors (115, 142, 158).

Both Forkhead and Mcm1 binding sites are conserved in Metazoa (79, 129, 169). Fkh1p and Fkh2p appear to have partially redundant yet distinct functions in the control of G2/M-regulated genes. Fkh2p is the main factor that forms a ternary complex with Mcm1p on gene promoters, while Fkh1p can bind promoters in the absence of Mcm1p and also plays additional roles in cell-type determination (69, 70, 150). DNA bending induced by Mcm1p and a region upstream of the FKH DNA-binding domain of Fkh2p are instrumental in recruiting Fkh2p into the ternary transcription complex (24, 92). Surprisingly, both Fkh1p and Fkh2p also associate with coding regions of active genes and regulate, in opposing ways, transcriptional elongation and termination (120), probably reflecting a function independent of their roles in cell-cycle transcription.

**Cell Cycle-Regulation of Forkhead Activity**

Fkh2p and Mcm1p remain bound at promoters of G2/M-regulated genes throughout the cell cycle (4, 83). The periodic activity of this transcription complex is regulated by an additional factor, the coactivator Ndd1p (83, 96, 136). Ndd1p is essential for the G2/M transition and for activating gene expression of G2/M-regulated genes, and it associates with promoters of these genes in a Fkh2p-Mcm1p-dependent manner. Ndd1p is regulated both transcriptionally and via protein stability, and protein levels peak during S- and G2-phases. Loss of Fkh2p function suppresses the requirement for Ndd1p, indicating that Fkh2p also plays a negative regulatory role in transcription (83).

Amon et al. (6) have proposed that the Clb1/2p-Cdk1p kinase functions in a positive feedback loop to induce transcription of the cyclin genes CLB1 and CLB2 (Figure 1). Recent data give insight into the mechanism of this regulatory circuit. Fkh2p is phosphorylated by Clb5p-Cdk1p CDK, thus triggering recruitment of Ndd1p to promoters

**MADS box TF:** TF with conserved MADS box domain, including *S. cerevisiae* Mcm1p and *S. pombe* Mbx1, that often function in combination with various accessory factors

**SFF:** Swi five factor

**Forkhead TF:** member of a TF family that contains a conserved DNA-binding domain of winged helix structure (including *S. cerevisiae* Fkh1p and Fkh2p and *S. pombe* Sep1p and Fkh2p), named after the *Drosophila* forkhead factor; forkhead TFs function during development, cell differentiation, and the cell cycle

**Forkhead Transcription Factor Complexes**

At the beginning of the new millennium, a series of elegant studies unmasked the identity and control of SFF in budding yeast, making it the best understood mechanism of cell cycle-regulated transcription. The two proteins Fkh1p and Fkh2p, members of the conserved forkhead family of TFs (33), are required to regulate transcription of genes during the G2/M transition (69, 83, 85, 129, 148, 173; reviewed in 28, 62, 76). These TFs recognize Forkhead/SFF binding sites (4, 79, 129, 173), which are frequently located next to Mcm1 binding sites in the promoters of CLB2 cluster genes that are expressed during G2/M (Table 1) (2, 24, 62, 129, 148, 170).

bound by the Fkh2p-Mcm1p complex (128). The Ndd1p-Fkh2p interaction is further stabilized upon phosphorylation of Ndd1p by Clb2p-Cdk1p CDK, leading to a specific association with the forkhead-associated (FHA) domain of Fkh2p (45, 136), which is a conserved phosphopeptide recognition site (56, 57). Transcription of *CLB1* and *CLB2* during the G2/M transition is directly activated through this association of Ndd1p with Fkh2p-Mcm1p (28, 62, 76). In addition, Fkh2p itself becomes phosphorylated before mitosis (129), but whether this contributes to the regulation of G2/M transcription is not known. Morris et al. (121) showed that Cks1p, a conserved CDK-interacting protein, activates G2/M transcription of the *CLB2* cluster gene *Cdc20p* by recruiting the proteasome to its promoter. The proteasome has been implicated in transcriptional control in addition to its well-known proteolytic role (65), but whether Cks1p also contributes to the regulation of other *CLB2* cluster genes remains to be determined.

### Forkhead-Dependent Transcription in Fission Yeast

Knowledge of the transcriptional regulation during the G2/M transition is more limited in fission yeast, although a partially similar picture is emerging. The fission yeast genome contains four TFs of the forkhead family: Mei4p appears to function specifically in gene transcription during the meiotic nuclear divisions (1, 72, 105). SPAC1142.08 is highly similar to budding yeast Fhl1p that controls ribosomal protein gene expression (104; and references therein), and it does not appear to affect cell-cycle transcription (30; G. Rustici & J. Bähler, unpublished data). Evidence is accumulating that the two other forkhead proteins, Sep1p (137, 146) and SPBC16G5.15c (Fkh2p), are involved in transcriptional activation during M-phase. Sep1p is required for periodic expression of *cdc15* whose transcript levels peak during early mitosis (58, 176). Microarray analyses revealed that Sep1p is involved in activating transcription of dozens of additional genes during mitosis (Figure 2) (138). These genes are also significantly enriched for TF binding sites similar to the conserved binding sites of forkhead proteins (Table 1) (72, 79, 129, 130, 138, 173). Fkh2p, which is the closest homolog to budding yeast Fkh2p, also appears to function in transcriptional regulation during the cell cycle, because several mitotically induced genes show constitutive expression in its absence (30, 31). Intriguingly, basal transcription levels of mitotic genes are increased in the absence of Fkh2p, whereas they are decreased in the absence of Sep1p (31, 138). This indicates that Fkh2p, unlike Sep1p, has roles in repressing transcription at some stages of the cell cycle. Overproduction of Fkh2p is lethal in wild-type cells but not in *sep1* mutants, suggesting that Sep1p is required for Fkh2p function (30). Further work is needed to establish whether and how Sep1p and Fkh2p function together to regulate periodic cell-cycle transcription during mitosis in fission yeast.

As in budding yeast, there is evidence that a MADS-box TF (*Mbx1p*) functions together with Fkh2p in transcriptional regulation (30). Unlike in budding yeast, however, Forkhead binding sites do not seem to be accompanied by Mcm1 sites (127, 138). *Mbx1p* is part of an uncharacterized TF complex named PBF (PCB binding factor) that binds to a short promoter site called PCB (pombe cell-cycle box; Table 1) (7, 30). The genes with a PCB site also contain Forkhead binding sites (3, 31, 138), and it is possible that the combination of these two motifs provides the platform required for combinatorial control by forkhead and MADS-box TFs. The polo kinase Plolp, whose transcript is induced during mitosis, regulates PBF activity and may therefore provide a positive feedback mechanism to activate mitotic gene transcription (7).

Fission yeast contains no clear orthologs of budding yeast Ndd1p, and it is not known how transcription during mitosis is activated.

AR260-GE39-05 ARI 10 October 2005 18:9

![Expression Ratio](image.png)

**Figure 2**

Expression profiles of 39 Sep1p-dependent genes with peak transcript levels around mitosis in fission yeast. Some of these genes are direct targets of Sep1p, whereas others are indirectly activated by Sep1p via its target gene Ace2p. Blue profiles: cell-cycle synchronization using *cdc25* block-release (whole culture synchronization); green profiles: cell-cycle synchronization using centrifugal elutriation (selective synchronization). Two cell cycles were followed in each experiment, and transcript levels relative to unsynchronized cells were measured in 15-min intervals using DNA microarrays (138). The approximate timing and duration of M-phases are indicated by bars above the graphs.

Intriguingly, only Fkh2p contains a potential FHA domain while Sep1p does not (31). Transcription of *fkh2* but not of *sep1* is regulated during the cell cycle, although expression levels peak relatively late, coinciding with those of the target genes (31, 138, 176). Accordingly, the *fkh2* promoter contains a Forkhead binding site (138), raising the possibility that Sep1p and/or Fkh2p activate *fkh2* transcription, either as a positive feedback or, given the negative role of Fkh2p in transcription, to switch off mitotic transcription in a negative feedback. Similarly, there is also evidence in budding yeast that Fkh1p and Fkh2p themselves activate *FKH2* transcription (70). As in budding yeast, Fkh2p is phosphorylated during mitosis coincident with increased transcription of target genes (31). It has also been proposed that the Cdk1p CDK controls the activity of Sep1p based on genetic interactions between *sep1* mutants and mutants in *cdk1* or in genes for Cdk1p regulators (*cdc25* and *wee1*) (66). The TFIIH-associated CDK-activating kinase (Mcs6-complex) seems to be particularly important for Sep1p-regulated gene expression in fission yeast, a function that is probably independent of CDK activation (89).

REGULATION OF GENE  
EXPRESSION PEAKING  
DURING M/G1  

The Mcm1p Transcription Factor  

In budding yeast, the MADS-box protein Mcm1p also plays a role in the transcription of genes whose expression levels peak during mitotic exit. A promoter element named ECB (early cell-cycle box) contains a Mcm1 binding site and functions in the periodic transcription of genes during the M/G1 transition (112, 148). Genes for regulatory proteins such as the SBF component SWI4 and the cyclin CLN3, which are transcriptionally activated in late mitosis through ECB elements, are rate limiting for G1 progression (see above) (98, 101, 112). Mcm1p binds constitutively to ECB sites (101), and M/G1 specificity may be conferred by timed transcriptional repression: Two homeodomain proteins, Yox1p and Yhp1p, bind to typical homeodomain binding sites upstream of the Mcm1 binding sites in ECB (Table 1), and they act as repressors restricting gene transcription from ECB promoters to the M/G1-phase of the cell cycle (132). Yox1p itself is transcriptionally activated by the SBF complex (71, 75, 145); Mcm1p is thus setting up a negative feedback loop that determines the duration of ECB activity (Figure 1). Yhp1p is transcribed later in the cell cycle and helps to maintain ECB repression until late M-phase (132).

arates mother and daughter cells, and cellular identity or fate can change. Swi5p and Ace2p activate several genes with roles in cell separation (20, 53, 55). Moreover, these two TFs also function in specifying the identities of mother and daughter cells, together with one of their target genes, the TF Ash1p. Both Ace2p and Ash1p accumulate exclusively in daughter cells and control daughter cell-specific gene expression (19, 22, 36, 144, 153, 162). Ace2p also functions in the G1 delay specific to daughter cells (87). G1-specific transcription of the HO endonuclease gene that initiates mating-type switching requires both Swi5p and SBF; the ordered stepwise recruitment of chromatin remodeling complexes, TFs, and RNA polymerase II complexes to the HO promoter during the cell cycle has been studied in some detail (41, 84).

Cell Cycle-Regulation of Ace2p  
and Swi5p Activity  

ACE2 and SWI5 are themselves transcriptionally activated during the G2/M transition by the Mcm1p-Fkh2p complex described above (Figure 1) (97, 145, 173). In addition, the nuclear localization of Swi5p is negatively regulated by Clb2p-Cdk1p CDK: Proteolysis of Clb2p during anaphase leads to dephosphorylation of Swi5p, which then accumulates in the nucleus to activate transcription of target genes (117, 123, 161). Ace2p, on the other hand, is positively regulated by phosphorylation: A signaling network named RAM (regulation of Ace2p activity and cellular morphogenesis) culminating in activation of the Mob1p-Cbk1p kinase complex activates both Ace2p and polarized growth (20, 125, 162). The functional architecture of RAM signaling is similar to the mitotic exit network (MEN) and septation initiation network (SIN) in budding and fission yeast, respectively (16, 109). RAM acts together with MEN to promote daughter cell-specific localization of Ace2p, and it coordinates Ace2p-dependent transcription with MEN activation at the end of mitosis (162).

The Ace2p and Swi5p Transcription  
Factors  

The two related TFs Swi5p and Ace2p also activate transcription of genes during the M/G1 transition in budding yeast (53). Like the MBF/SBF and Fkh1p/Fkh2p TF pairs described above, Swi5p and Ace2p have overlapping but also specialized functions in transcriptional control (54, 55, 108, 145). Swi5p and Ace2p target genes are associated with a Swi5 binding site (Table 1) (54, 80, 148). During the M/G1 transition, cytokinesis sep-

Ace2p/Swi5p: yeast TF paralogs that regulate mitotic exit, cell division, and daughter cell identity  
ECB: early cell-cycle box

www.annualreviews.org • Cell-Cycle Transcription in Yeasts 77

AR260-GE39-05 ARI 10 October 2005 18:9

### Ace2p in Fission Yeast

A fission yeast homolog of Ace2p has recently been characterized and given the same name (103). As in budding yeast, Ace2p is required for cell separation, and it activates the transcription of *eng1* that functions in degradation of the cell division septum. Ace2p controls the expression of several additional genes whose transcript levels peak in M/G1-phase (Figure 2), and these genes are enriched for a binding site similar to the Swi5 site in budding yeast (Table 1) (3, 138). Moreover, *ace2* expression is itself regulated during mitosis by the forkhead TFs Sep1p and Fkh2p (3, 30, 138). This transcriptional cascade is reminiscent of the one in budding yeast (see above), and it explains that *sep1* and *ace2* mutants show similar defects in cell separation (103, 146). It is possible that the Sep1p-Ace2p pathway contributed to the evolution of single-celled fungi (14).

---

### TRANSCRIPTIONAL REGULATORY NETWORKS

The three main transcriptional waves described above do not act in isolation but are integrated with the cell-cycle machinery, with each other, and with additional clusters of periodically transcribed genes at other phases of the cell cycle. In recent years, much has been learned about transcriptional regulatory networks of the cell cycle through elegant genome-wide approaches pioneered with budding yeast (reviewed in 29, 63, 159, 168a, 171). This work revealed a serial regulation of TFs, whereby transcriptional activators functioning during one stage of the cell cycle regulate activators functioning during the next stage, thus resulting in a continuous cycle of interdependent waves of transcription (Figure 3). In late G1, the MBF and SBF complexes activate transcription of *NDD1*, whose gene product in turn activates

---

### Budding yeast

![Diagram](https://i.imgur.com/1234567.png)

### Fission yeast

![Diagram](https://i.imgur.com/7654321.png)

---

#### Figure 3

Transcriptional regulatory networks during the cell cycles in budding and fission yeast. Orthologous TFs or complexes are indicated by corresponding frames and colors, and approximate cell-cycle phases where TFs function are given within the circles. Continuous and hatched lines indicate transcriptional and posttranslational regulation, respectively. See main text for details and references.

---

78 Bähler
the Fkh2p-Mcm1p complex leading to transcription of SWI5 and ACE2 during G2/M. Mcm1p, Swi5p, and Ace2p, which all act in M/G1, then close the circle either directly by inducing transcription of the SBF gene SWI4, or indirectly by inducing transcription of the cyclin gene CLN3, whose product then activates both SBF and MBF. In addition, the transcriptional network is driven by complex overlapping layers of posttranslational regulation such as phosphorylation, protein degradation, and protein localization, some details of which were described in the previous sections (Figure 1). As illustrated by the regulation of the SBF/MBF and forkhead TF complexes described above, promoter prebinding of inhibited TFs that wait for a cell-cycle signal to activate transcription seems to be a general regulatory principle (78, 168a, 171).

Recent microarray experiments with fission yeast identified periodically transcribed genes clustered into four major waves of expression according to the time of their peak expression levels (127, 138). The forkhead protein Sep1p regulates mitotic genes in the first cluster including Ace2p, which then activates transcription in the second cluster during the M/G1 transition. Other genes in the second cluster required for G1/S progression are regulated by MBF independently of Sep1p and Ace2p (Figure 3). The third cluster coincides with S-phase, while a fourth cluster contains genes weakly regulated during G2-phase.

The three main cell-cycle transcriptional regulators or complexes are conserved between fission and budding yeast, and at least some of these TFs and promoter binding sites are also conserved in Metazoa. For example, forkhead proteins regulate the expression of mitotic genes such as cyclin B and polo kinase also in mammalian cells (5, 88), and many parallels are evident between the yeast MBF complex and the human E2F-DP heterodimer with regard to structure, function, target genes, and binding sites (29, 86, 156). In budding yeast, pairs of transcriptional

activators exhibit partially redundant functions (Ace2p and Swi5p, MBF and SBF), while fission yeast appears to rely on only one complex at each stage (Figure 3). Despite the similarities between the main transcriptional activators, the regulatory networks show major differences in circuitry reflecting differences in cell-cycle phasing between the two yeasts (Figure 3). In fission yeast, M- and G1-phases take place within ~20% of total cell-cycle duration (99). Accordingly, peak transcription levels of ~80% of the highly regulated periodic genes are concentrated in a short window, during which all known cell-cycle TFs function (138). In budding yeast, the functions of the main transcriptional regulators are spread more evenly over the cell cycle due to longer G1- and M-phases but short G2-phase (Figure 3). The transcriptional cascade from forkhead to Ace2p regulators acting during M- and G1-phases is conserved in fission yeast (3, 30, 138), whereas other aspects of the transcriptional network are clearly different. The MBF complex, which regulates G1/S progression in both yeasts, acts downstream of Ace2p and Swi5p and upstream of the forkhead complex in budding yeast (145, and references cited therein), while it functions in parallel to, and independently of, Ace2p and Sep1p in fission yeast (138). The absence of a distinct G1-phase in fission yeast means that mitotic exit (controlled by Ace2p) and initiation of S-phase (controlled by MBF) coincide, which is reflected in the regulatory circuit (Figure 3). Unlike in budding yeast, transcriptional activators in fission yeast probably do not form a fully connected cyclic network. The long G2-phase (~70% of cell cycle), during which mainly weakly regulated genes and no obvious transcriptional activators are expressed, is unlikely to be bridged by transcriptional control (138). Moreover, Ace2p is the only known cell-cycle TF in fission yeast whose transcription is strongly regulated and also peaking ahead of its target genes. It is therefore likely that in fission yeast posttranslational mechanisms are relatively more important to

www.annualreviews.org • Cell-Cycle Transcription in Yeasts 79
regulate periodic transcription during the cell cycle.

There are still large gaps in our knowledge, and the real picture of the transcriptional cell-cycle network is certainly more complex than conveyed in Figures 1 and 3. For many periodically expressed genes, the mechanisms responsible for their cell-cycle regulation are not known. For example, the TFs activating histone gene expression are not known in either yeast, even though these were the first genes that had been identified as periodically expressed, and potential TF binding sites have been described (10, 61, 67, 106, 107). Unlike the binding sites for the main transcriptional regulators described above (Table 1), the histone promoter sites are not conserved between budding and fission yeast. It is possible that SBF contributes to histone transcription in budding yeast as it binds to several histone gene promoters (145). Two proteins, Hir1p and Hir2p, act as transcriptional repressors that function in periodic transcription of histone genes in budding yeast (141, 147). Fission yeast Hip1p, which is closely related to Hir1p and Hir2p, is similarly required for repression of histone gene expression outside of S-phase, but histone mRNA levels also appear to be positively regulated by an unknown factor (21). Little is known about the regulation of genes whose transcript levels peak during G2-phase in fission yeast. It is possible that the transcript levels of some genes are regulated at the level of mRNA stability.

Furthermore, many genes are regulated by different combinations of TFs, and several regulators such as Mcm1p can act at various cell-cycle stages, adding additional gene expression modules to the network (15, 71, 73, 90, 145). For example, some genes appear to be regulated by a combination of SBF and forkhead, and some genes are regulated by forkhead independently of Mcm1p. CDC20 is an example for genes with a promoter that contains binding sites for the Mcm1p-Fkh2p complex together with an ECB element that binds Yox1p/Yhp1p and Mcm1p. These genes are expressed at intermediate times between

the two transcriptional waves promoted by either individual regulator complex, and they continue to be periodically transcribed if either regulatory pathway is disrupted (132, 173). Kato et al. (78) recently showed that regulation is frequently relayed from a TF acting earlier to a TF acting later via combinatorial, joint-phase control of the two TFs to regulate gene expression at intermediate times; this means that two cooperating TFs can generate at least three peaks of gene expression, which could explain the continuum of peaks observed over the cell cycle. Thus, combinatorial control of cell cycle-regulated transcription greatly contributes to the complexity and sophistication of periodic gene expression.

**CONSERVATION AND IMPORTANCE OF CELL CYCLE–REGULATED TRANSCRIPTION**

Genome-wide studies of the gene expression program during the budding yeast cell cycle identified about 400 to 800 periodically transcribed genes (35, 148). A corresponding study in fission yeast revealed about 400 genes, but only 136 of these genes exhibited high-amplitude cell cycle-dependent changes comparable to those in budding yeast (138). Another microarray analysis reported a less conservative list of >700 cell cycle-regulated *S. pombe* genes (127), although no data on reproducibility are available and many of these genes show marginal periodicity in expression profiles. Overall, more genes appear to be strongly regulated during the cell cycle in budding yeast compared with fission yeast. For example, transcriptional regulation of nine different cyclin genes helps to drive both the transcriptional network (Figure 1) and the cell cycle in budding yeast (29, 63, 119, 145, 148). In fission yeast, only one cyclin gene (*cig2*) is strongly regulated and two others (*cdc13*, *cig1*) show weak regulation (37, 126, 138). The weak transcriptional regulation of *cdc13* and *cig1* might be an

evolutionary remnant, or this regulation could become more important, and more pronounced, when the cell cycle slows down, e.g., during fluctuations in available nutrients. These observations indicate that fission yeast relies less on transcriptional control for cell-cycle regulation than does budding yeast, and therefore posttranslational mechanisms may play relatively more important roles.

Periodic gene expression that has been conserved through evolution is likely of more general biological significance for proper cell-cycle function. It is striking that genes for glycosylphosphatidylinositol-modified (GPI) cell surface proteins (47) are significantly enriched among periodically transcribed genes in both yeasts (14 of 33 *S. pombe* genes and 27 of 66 *S. cerevisiae* genes encoding GPI proteins; \(P < 10^{-4}\); J. Mata & J. Bähler, unpublished data). De Lichtenberg et al. (49) reported that periodically expressed budding yeast genes encode proteins that tend to share combinations of certain protein features such as phosphorylation, glycosylation, and instability that help to distinguish these proteins with high confidence. On the other hand, periodic gene expression is not necessarily conserved between budding and fission yeast (163). To obtain a conservative estimate of orthologous genes that are periodically expressed in both yeasts, Rustici et al. (138) determined the overlap between the genes reported as periodic in both studies of budding yeast (35, 147) and the genes periodically expressed with high amplitude in fission yeast (Figure 4). Although the resulting overlap of about 40 genes is highly significant, it is also surprisingly small. Less conservative comparisons reveal ~80–140 genes that are cell cycle-regulated in both yeasts, but a substantial number of these would be expected by chance given the sizes of gene lists involved (Figure 4) (127, 138). This suggests that cell cycle-regulated transcription of the majority

# Core cell cycle-regulated genes

**Budding yeast** | **Fission yeast** | **Budding yeast** | **Fission yeast**
--- | --- | --- | ---
| | |
| | |
| | |

**Figure 4**

Conservation of cell cycle-regulated transcription between budding and fission yeast. Left Venn diagram, conservative estimate: The 301 genes that were periodically expressed in both budding yeast microarray studies (35, 148) were compared with the 136 highly regulated genes that were periodically expressed in a fission yeast microarray study (138). Among the fission yeast genes that have budding yeast orthologs, 42 genes were also periodic in the budding yeast studies. Right Venn diagram, less conservative estimate: The 800 genes that were periodically expressed in the study of Spellman et al. (147) were compared with all 407 periodic genes identified by Rustici et al. (138). Among the fission yeast genes that have budding yeast orthologs, 81 genes were also periodic in the budding yeast studies. In brackets: numbers of genes expected to overlap by chance, given the sizes of the gene sets compared and the total number of 2981 genes with orthologs. The overlaps are highly significant (\(P \sim 2e-35\) and \(2e-22\) for left and right diagrams, respectively). Only numbers of genes with orthologs in the other yeast are shown in the comparisons.

Table 2 Selected core genes periodically expressed in both budding and fission yeast

| Budding yeast ortholog | Fission yeast ortholog | Function |
|-----------------------|------------------------|----------|
| DNA replication       |                        |          |
| POL1 and POL2        | pol1 and cdc20         | DNA polymerases α and ε |
| RFA1                 | ssb1                   | Single-stranded DNA-binding protein |
| CDC6                 | cdc18                  | Regulator of DNA replication initiation |
| MRC1                 | mrc1                   | DNA replication checkpoint protein |
| RNR1                 | cdc22                  | Ribonucleotide reductase |
| SMC3 and MCD1        | psm3 and rad21         | Cohesins |
| HTZ1                 | pht1                   | Histone variant |
| 8 histone genes      | 9 histone genes        | Histones H2A, H2B, H3, and H4 |
| Mitosis and cell division |                    |          |
| CDC5, IPL1, and KIN3 | polo, ark1, and fin1   | Polo, Aurora, and NimA kinases |
| CDC20                | slp1                   | Activator of anaphase promoting complex |
| SPO12                | wis3                   | Putative cell-cycle regulator |
| KAR3, KIP1           | klp5, klp6, klp8        | Kinesin microtubule motor |
| MOB1 and DBF2        | mob1 and sid2          | Proteins involved in mitotic exit/septation |
| MYO1                 | myo3                   | Myosin II heavy chain |
| BUD4                 | mid2                   | Protein involved in cytokinesis |
| ACE2                 | ace2                   | Transcription factor |
| HOF1                 | imp2                   | Protein involved in cell division |
| DSE4                 | eng1                   | Glucanase for cell separation |
| CHS2                 | chs2                   | Protein involved in septum formation |
| TOS7                 | mac1                   | Putative role in cell separation |
| Others               |                        |          |
| SWE1                 | mik1                   | Kinase inhibiting cyclin-dependent kinase |
| CLB1-CLB6            | cig2                   | B-type cyclins |
| MSH6                 | msh6                   | Mismatch-repair protein |
| RAD51                | rhp51                  | DNA repair protein |
from the cell cycle is not necessarily lethal for cells (141), but production of histones in correct stoichiometric amounts is important as increased expression of one class of histone relative to another causes increased chromosome loss (114). In addition, restriction of histone gene transcription to S-phase could be necessary to prevent regular histones from competing with histone variants for incorporation into chromosomes of nonreplicating cells. Consistent with this idea, the gene for the fission yeast centromere-specific histone variant CENP-A (*npl*) is periodically expressed earlier in the cell cycle than the regular histones (152).

What is the biological significance of cell cycle-regulated transcription, especially for those genes whose periodic expression is not conserved? The standard experiment to check the importance of transcription, constitutive expression of genes that are normally cell cycle-regulated, has produced only a few examples of its significance (26). In a *fkh1 fkh2* double mutant, cell-cycle regulation of many G2/M-phase genes becomes constitutive, but the cells remain viable and are quite healthy (173). It is therefore possible that regulation of most periodic genes is not absolutely required, but together they may enhance the fidelity and efficiency of the cell-division cycle. It is reasonable to assume that timely supply of products reflecting the demand of the cells will help the cell cycle to run smoothly and efficiently, which is as important for a reiterated essential process as execution of the process itself.

Clearly, many cell-cycle proteins are regulated at multiple and partly redundant levels, and it is the overall regulation that is likely to be important. Recent data show that protein complexes functioning during the cell cycle consist of both periodically and constitutively expressed subunits in budding yeast; the former are frequently also regulated by additional mechanisms such as CDK phosphorylation and targeted degradation and may thus play crucial and dynamic roles in controlling protein complex activity during the cell cycle

(48). This system ensures the supply of new, unmodified CDK targets with special regulatory roles in each cell cycle. Transcriptional control should therefore not be considered in isolation. Although the absence of transcriptional control in rapidly proliferating yeast cells could be buffered by faster acting posttranscriptional and posttranslational regulation, transcription might become more important in driving the cell cycle when nutrient limitations slow down growth and proliferation, and/or it might help to kick start the cell cycle when nutrients increase.

It has been argued that the large number of reported cell cycle-regulated genes should be treated with scepticism, because the observed periodicity of many genes might reflect a stress response induced by cell-cycle synchronization (39). In fission yeast, this could be the case for the mainly weakly regulated genes whose transcripts peak during G2-phase, and which overlap significantly with genes modulated during environmental stress (34, 138). However, this would imply that cells are more responsive or more sensitive to stress during G2-phase and periodically repeat the stress response in subsequent cell cycles.

It has been suggested that parsimony may explain much of cell cycle-regulated transcription (148, 166), whereby genes are expressed when there is a special need for their products at a particular phase in the cell cycle, and these needs can differ between organisms. For example, the periodic induction of several genes functioning in metabolism and growth occurs during early G2-phase in fission yeast, at a time when these cells increase their overall growth rate after a switch from monopolar to bipolar growth mode (138). On the other hand, genes for bud emergence and growth in budding yeast are expressed during G1-phase, reflecting the particular biology of these cells (148).

Genes for spatial landmark proteins in budding yeast illustrate that periodic transcription can regulate organism-specific roles such as cellular morphogenesis (138). Bud8p and Bud9p are homologous membrane
proteins that mark the distal and proximal cell
poles, respectively, and their transcripts peak
at different times in the cell cycle. The impor-
tance of transcriptional timing was shown by
elegant promoter-swap experiments between
BUD8 and BUD9, which results in mislocal-
ization of both gene products to the opposite
poles of the cell (139).

An intriguing possibility is that CDKs
and cyclins originally functioned in transcrip-
tional control before evolving into the key
regulators capable of driving the cell cycle
through a free-running oscillation in activity.
This transition could have led to a cell-cycle
machinery that relies less on transcriptional
regulation to drive cell proliferation. In ac-
cordance with this hypothesis, some members
of the CDK family can control the initiation
and elongation of transcription, and cyclins
are structurally similar to the basal transcrip-
tion factor TFIIB and the RB transcriptional
regulator (reviewed in 74, 119). Even the main
cell-cycle CDK in S. cerevisiae, Cdk1p, has
additional roles in transcriptional regulation
(121). In S. pombe, the Mcs6 CDK complex
not only activates the Cdk1p CDK but is also
critical for cell cycle-regulated transcription
(89).

---

### SUMMARY POINTS

1. Major cell-cycle TFs (MBF, forkhead, and Ace2p) and their binding sites are conserved
   in budding and fission yeast and to varying degree also in Metazoa, and they function
   at corresponding cell-cycle phases (initiation of DNA replication, entry and exit from
   mitosis).

2. Despite functional conservation of three major cell-cycle TFs, there are clear differ-
   ences in the circuitry of the regulatory networks between budding and fission yeast,
   probably reflecting a rewiring during evolution to adjust for differences in cell-cycle
   phasing between the two yeasts.

3. More genes appear to be strongly regulated during the cell cycle in budding yeast
   compared to fission yeast, raising the possibility that the latter relies less on transcrip-
   tional control for cell-cycle regulation, and posttranscriptional mechanisms may play
   relatively more important roles.

4. Surprisingly, periodic transcription of most genes is not conserved between the two
   yeasts, except for a relatively small core set of genes that mainly function in basic
   cell-cycle processes such as mitosis, cytokinesis, and DNA replication. Most genes of
   this core set appear to be universally regulated during the eukaryotic cell cycle, and
   they may play key roles in controlling cell-cycle progression.

5. DNA microarrays and other genome-wide methods are now providing global perspec-
   tives on the system underlying cell cycle–regulated gene expression and its integration
   with other aspects of the cell-cycle machinery.

6. As in other research areas, comparative studies between the two distantly related
   yeasts are proving fruitful for the elucidation of basic features and critical targets
   of transcriptional networks during eukaryotic cell cycles. Such studies also aid our
   understanding of the evolutionary plasticity of transcriptional regulation, and provide
   insight into the overall biological significance of cell cycle–regulated transcription.

# ACKNOWLEDGMENTS

I thank Christopher McInerny, Brian Morgan, and Mike Tyers for communicating data prior to publication, and José Ayté, Christopher McInerny, Brian Morgan, and Brian Wilhelm for comments on the manuscript. Research in the Bähler laboratory is funded by Cancer Research UK [CUK], Grant No. C9546/A5262 and by DIAMONDS, a EU FP6 Lifescihealth STREP.

# LITERATURE CITED

1. Abe H, Shimoda C. 2000. Autoregulated expression of *Schizosaccharomyces pombe* meiosis-specific transcription factor Mei4 and a genome-wide search for its target genes. *Genetics* 154:1497–508
2. Acton TB, Zhong H, Vershon AK. 1997. DNA-binding specificity of Mcm1: operator mutations that alter DNA-bending and transcriptional activities by a MADS box protein. *Mol. Cell Biol.* 17:1881–89
3. Alonso-Nunez ML, An H, Martin-Cuadrado AB, Mehta S, Petit C, et al. 2005. Ace2p controls the expression of genes required for cell separation in *Schizosaccharomyces pombe*. *Mol. Biol. Cell.* 16:2003–17
4. Althoefer H, Schleiffer A, Wassmann K, Nordheim A, Ammerer G. 1995. Mcm1 is required to coordinate G2-specific transcription in *Saccharomyces cerevisiae*. *Mol. Cell Biol.* 15:5917–28
5. Alvarez B, Martinez AC, Burgering BM, Carrera AC. 2001. Forkhead transcription factors contribute to execution of the mitotic programme in mammals. *Nature* 413:744–47
6. Amon A, Tyers M, Futcher B, Nasmyth K. 1993. Mechanisms that help the yeast cell cycle clock tick: G2 cyclins transcriptionally activate G2 cyclins and repress G1 cyclins. *Cell* 74:993–1007
7. Anderson M, Ng SS, Marchesi V, MacIver FH, Stevens FE, et al. 2002. *pol1+* regulates gene transcription at the M-G(1) interval during the fission yeast mitotic cell cycle. *EMBO J.* 21:5745–55
8. Andrews BJ, Herskowitz I. 1989. Identification of a DNA binding factor involved in cell-cycle control of the yeast HO gene. *Cell* 57:21–29
9. Andrews BJ, Herskowitz I. 1989. The yeast SWI4 protein contains a motif present in developmental regulators and is part of a complex involved in cell-cycle-dependent transcription. *Nature* 342:830–33
10. Aves SJ, Durkacz BW, Carr A, Nurse P. 1985. Cloning, sequencing and transcriptional control of the *Schizosaccharomyces pombe cdc10 ‘start’* gene. *EMBO J.* 4:457–63
11. Ayté J, Leis JF, DeCaprio JA. 1997. The fission yeast protein p73res2 is an essential component of the mitotic MBF complex and a master regulator of meiosis. *Mol. Cell Biol.* 17:6246–54
12. Ayté J, Leis JF, Herrera A, Tang E, Yang H, DeCaprio JA. 1995. The *Schizosaccharomyces pombe* MBF complex requires heterodimerization for entry into S phase. *Mol. Cell Biol.* 15:2589–99
13. Ayté J, Schweitzer C, Zarzov P, Nurse P, DeCaprio JA. 2001. Feedback regulation of the MBF transcription factor by cyclin Cig2. *Nat. Cell Biol.* 3:1043–50
14. Bähler J. 2005. A transcriptional pathway for cell separation in fission yeast. *Cell Cycle* 4:39–41
15. Banerjee N, Zhang MQ. 2003. Identifying cooperativity among transcription factors controlling the cell cycle in yeast. *Nucleic Acids Res.* 31:7024–31

16. Bardin AJ, Amon A. 2001. Men and sin: What's the difference? *Nat. Rev. Mol. Cell Biol.* 2:815–26

17. Baum B, Nishitani H, Yanow S, Nurse P. 1998. Cdc18 transcription and proteolysis couple S phase to passage through mitosis. *EMBO J.* 17:5689–98

18. Baum B, Wuarin J, Nurse P. 1997. Control of S-phase periodic transcription in the fission yeast mitotic cycle. *EMBO J.* 16:4676–88

19. Bertrand E, Chartrand P, Schaefer M, Shenoy SM, Singer RH, Long RM. 1998. Localization of ASH1 mRNA particles in living yeast. *Mol. Cell* 2:437–45

20. Bidlingmaier S, Weiss EL, Seidel C, Drubin DG, Snyder M. 2001. The Cbk1p pathway is important for polarized cell growth and cell separation in *Saccharomyces cerevisiae*. *Mol. Cell Biol.* 21:2449–62

21. Blackwell C, Martin KA, Greenall A, Pidoux A, Allshire RC, Whitehall SK. 2004. The *Schizosaccharomyces pombe* HIRA-like protein Hip1 is required for the periodic expression of histone genes and contributes to the function of complex centromeres. *Mol. Cell Biol.* 24:4309–20

22. Bobola N, Jansen RP, Shin TH, Nasmyth K. 1996. Asymmetric accumulation of Ash1p in postanaphase nuclei depends on a myosin and restricts yeast mating-type switching to mother cells. *Cell* 84:699–709

23. Bork P. 1993. Hundreds of ankyrin-like repeats in functionally diverse proteins: mobile modules that cross phyla horizontally? *Proteins* 17:363–74

24. Boros J, Lim FL, Darieva Z, Pic-Taylor A, Harman R, et al. 2003. Molecular determinants of the cell-cycle regulated Mcm1p-Fkh2p transcription factor complex. *Nucleic Acids Res.* 31:2279–88

25. Bouquin N, Johnson AL, Morgan BA, Johnston LH. 1999. Association of the cell cycle transcription factor Mbp1 with the Skn7 response regulator in budding yeast. *Mol. Biol. Cell* 10:3389–400

26. Breeden L. 1996. Start-specific transcription in yeast. *Curr. Top. Microbiol. Immunol.* 208:95–127

27. Breeden L, Nasmyth K. 1987. Cell cycle control of the yeast HO gene: cis- and trans-acting regulators. *Cell* 48:389–97

28. Breeden LL. 2000. Cyclin transcription: Timing is everything. *Curr. Biol.* 10:R586–88

29. Breeden LL. 2003. Periodic transcription: a cycle within a cycle. *Curr. Biol.* 13:R31–38

30. Buck V, Ng SS, Ruiz-Garcia AB, Papadopoulou K, Bhatti S, et al. 2004. Fkh2p and Sep1p regulate mitotic gene transcription in fission yeast. *J. Cell Sci.* 117:5623–32

31. Bulmer R, Pic-Taylor A, Whitehall SK, Martin KA, Millar JB, et al. 2004. The forkhead transcription factor Fkh2 regulates the cell division cycle of *Schizosaccharomyces pombe*. *Eukaryot. Cell* 3:944–54

32. Caligiuri M, Beach D. 1993. Sct1 functions in partnership with Cdc10 in a transcription complex that activates cell cycle START and inhibits differentiation. *Cell* 72:607–19

33. Carlsson P, Mahlapuu M. 2002. Forkhead transcription factors: key players in development and metabolism. *Dev. Biol.* 250:1–23

34. Chen D, Toone WM, Mata J, Lyne R, Burns G, et al. 2003. Global transcriptional responses of fission yeast to environmental stress. *Mol. Biol. Cell* 14:214–29

35. Cho RJ, Campbell MJ, Winzeler EA, Steinmetz L, Conway A, et al. 1998. A genome-wide transcriptional analysis of the mitotic cell cycle. *Mol. Cell.* 2:65–73

36. Colman-Lerner A, Chin TE, Brent R. 2001. Yeast Cbk1 and Mob2 activate daughter-specific genetic programs to induce asymmetric cell fates. *Cell* 107:739–50

37. Connolly T, Beach D. 1994. Interaction between the Cig1 and Cig2 B-type cyclins in the fission yeast cell cycle. *Mol. Cell Biol.* 14:768–76

38. Connolly T, Caligiuri M, Beach D. 1997. The Cdc2 protein kinase controls Cdc10/Sct1 complex formation. *Mol. Biol. Cell* 8:1105–15

39. Cooper S, Shedden K. 2003. Microarray analysis of gene expression during the cell cycle. *Cell Chrom.* 2:1

40. Cosma MP, Panizza S, Nasmyth K. 2001. Cdk1 triggers association of RNA polymerase to cell cycle promoters only after recruitment of the mediator by SBF. *Mol. Cell.* 7:1213–20

41. Cosma MP, Tanaka T, Nasmyth K. 1999. Ordered recruitment of transcription and chromatin remodeling factors to a cell cycle– and developmentally regulated promoter. *Cell* 97:299–311

42. Costanzo M, Nishikawa JL, Tang X, Millman JS, Schub O, et al. 2004. CDK activity antagonizes Whi5, an inhibitor of G1/S transcription in yeast. *Cell* 117:899–913

43. Costanzo M, Schub O, Andrews B. 2003. G1 transcription factors are differentially regulated in *Saccharomyces cerevisiae* by the Swi6-binding protein Stb1. *Mol. Cell Biol.* 23:5064–77

44. Cunliffe L, White S, McInerny CJ. 2004. DSC1-MCB regulation of meiotic transcription in *Schizosaccharomyces pombe*. *Mol. Genet. Genom.* 271:60–71

45. Darieva Z, Pic-Taylor A, Boros J, Spanos A, Geymonat M, et al. 2003. Cell cycle-regulated transcription through the FHA domain of Fkh2p and the coactivator Ndd1p. *Curr. Biol.* 13:1740–45

46. de Bruin RA, McDonald WH, Kalashnikova TI, Yates J 3rd, Wittenberg C. 2004. Cln3 activates G1-specific transcription via phosphorylation of the SBF bound repressor Whi5. *Cell* 117:887–98

47. De Groot PW, Hellingwerf KJ, Klis FM. 2003. Genome-wide identification of fungal GPI proteins. *Yeast* 20:781–96

48. de Lichtenberg U, Jensen LJ, Brunak S, Bork P. 2005. Dynamic complex formation during the yeast cell cycle. *Science* 307:724–27

49. de Lichtenberg U, Jensen TS, Jensen LJ, Brunak S. 2003. Protein feature based identification of cell cycle regulated proteins in yeast. *J. Mol. Biol.* 329:663–74

50. Ding R, Smith GR. 1998. Global control of meiotic recombination genes by *Schizosaccharomyces pombe rec16 (rep1)*. *Mol. Genet.* 258:663–70

51. Dirick L, Bohm T, Nasmyth K. 1995. Roles and regulation of Cln-Cdc28 kinases at the start of the cell cycle of *Saccharomyces cerevisiae*. *EMBO J.* 14:4803–13

52. Dirick L, Moll T, Auer H, Nasmyth K. 1992. A central role for SWI6 in modulating cell cycle Start-specific transcription in yeast. *Nature* 357:508–13

53. Dohrmann PR, Butler G, Tamai K, Dorland S, Greene JR, et al. 1992. Parallel pathways of gene regulation: homologous regulators SWI5 and ACE2 differentially control transcription of HO and chitinase. *Genes Dev.* 6:93–104

54. Dohrmann PR, Voth WP, Stillman DJ. 1996. Role of negative regulation in promoter specificity of the homologous transcriptional activators Ace2p and Swi5p. *Mol. Cell Biol.* 16:1746–58

55. Doolin MT, Johnson AL, Johnston LH, Butler G. 2001. Overlapping and distinct roles of the duplicated yeast transcription factors Ace2p and Swi5p. *Mol. Microbiol.* 40:422–32

56. Durocher D, Henckel J, Fersht AR, Jackson SP. 1999. The FHA domain is a modular phosphopeptide recognition motif. *Mol. Cell* 4:387–94

57. Durocher D, Jackson SP. 2002. The FHA domain. *FEBS Lett.* 513:58–66

58. Fankhauser C, Reymond A, Cerutti L, Utzig S, Hofmann K, Simanis V. 1995. The *S. pombe cdc15* gene is a key element in the reorganization of F-actin at mitosis. *Cell* 82:435–44

59. Forsburg SL. 1999. The best yeast? *Trends Genet.* 15:340–44

60. Foster R, Mikesell GE, Breeden L. 1993. Multiple SWI6-dependent *cis*-acting elements control SWI4 transcription through the cell cycle. *Mol. Cell Biol.* 13:3792–801

61. Freeman KB, Karns LR, Lutz KA, Smith MM. 1992. Histone H3 transcription in *Saccharomyces cerevisiae* is controlled by multiple cell cycle activation sites and a constitutive negative regulatory element. *Mol. Cell Biol.* 12:5455–63

62. Futcher B. 2000. Microarrays and cell cycle transcription in yeast. *Curr. Opin. Cell Biol.* 12:710–15

63. Futcher B. 2002. Transcriptional regulatory networks and the yeast cell cycle. *Curr. Opin. Cell Biol.* 14:676–83

64. Geymonat M, Spanos A, Wells GP, Smerdon SJ, Sedgwick SG. 2004. Clb6/Cdc28 and cdc14 regulate phosphorylation status and cellular localization of swi6. *Mol. Cell Biol.* 24:2277–85

65. Gonzalez F, Delahodde A, Kodadek T, Johnston SA. 2002. Recruitment of a 19S proteasome subcomplex to an activated promoter. *Science* 296:548–50

66. Grallert A, Grallert B, Ribar B, Sipiczki M. 1998. Coordination of initiation of nuclear division and initiation of cell division in *Schizosaccharomyces pombe*: genetic interactions of mutations. *J. Bacteriol.* 180:892–900

67. Hereford LM, Osley MA, Ludwig TR 2nd, McLaughlin CS. 1981. Cell-cycle regulation of yeast histone mRNA. *Cell* 24:367–75

68. Ho Y, Costanzo M, Moore L, Kobayashi R, Andrews BJ. 1999. Regulation of transcription at the *Saccharomyces cerevisiae* Start transition by Stb1, a Swi6-binding protein. *Mol. Cell Biol.* 19:5267–78

69. Hollenhorst PC, Bose ME, Mielke MR, Muller U, Fox CA. 2000. Forkhead genes in transcriptional silencing, cell morphology and the cell cycle. Overlapping and distinct functions for FKH1 and FKH2 in *Saccharomyces cerevisiae*. *Genetics* 154:1533–48

70. Hollenhorst PC, Pietz G, Fox CA. 2001. Mechanisms controlling differential promoter occupancy by the yeast forkhead proteins Fkh1p and Fkh2p: implications for regulating the cell cycle and differentiation. *Genes Dev.* 15:2445–56

71. Horak CE, Luscombe NM, Qian J, Bertone P, Piccirrillo S, et al. 2002. Complex transcriptional circuitry at the G1/S transition in *Saccharomyces cerevisiae*. *Genes Dev.* 16:3017–33

72. Horie S, Watanabe Y, Tanaka K, Nishiwaki S, Fujioka H, et al. 1998. The *Schizosaccharomyces pombe mei4+* gene encodes a meiosis-specific transcription factor containing a forkhead DNA-binding domain. *Mol. Cell Biol.* 18:2118–29

73. Ihmels J, Friedlander G, Bergmann S, Sarig O, Ziv Y, Barkai N. 2002. Revealing modular organization in the yeast transcriptional network. *Nat. Genet.* 31:370–77

74. Ingolia NT, Murray AW. 2004. The ups and downs of modeling the cell cycle. *Curr. Biol.* 14:R771–77

75. Iyer VR, Horak CE, Scafe CS, Botstein D, Snyder M, Brown PO. 2001. Genomic binding sites of the yeast cell-cycle transcription factors SBF and MBF. *Nature* 409:533–38

76. Jorgensen P, Tyers M. 2000. The fork'ed path to mitosis. *Genome Biol.* 1: reviews1022

77. Jorgensen P, Tyers M. 2004. How cells coordinate growth and division. *Curr. Biol.* 14:R1014–27

78. Kato M, Hata N, Banerjee N, Futcher B, Zhang MQ. 2004. Identifying combinatorial regulation of transcription factors and binding motifs. *Genome Biol.* 5:R56

79. Kaufmann E, Knöchel W. 1996. Five years on the wings of forkhead. *Mech. Dev.* 57:3–20

80. Knapp D, Bhoite L, Stillman DJ, Nasmyth K. 1996. The transcription factor Swi5 regulates expression of the cyclin kinase inhibitor p40SIC1. *Mol. Cell Biol.* 16:5701–7

81. Koch C, Moll T, Neuberg M, Ahorn H, Nasmyth K. 1993. A role for the transcription factors Mbp1 and Swi4 in progression from G1 to S phase. *Science* 261:1551–57

82. Koch C, Schleiffer A, Ammerer G, Nasmyth K. 1996. Switching transcription on and off during the yeast cell cycle: Cln/Cdc28 kinases activate bound transcription factor SBF (Swi4/Swi6) at start, whereas Clb/Cdc28 kinases displace it from the promoter in G2. *Genes Dev.* 10:129–41

83. Koranda M, Schleiffer A, Endler L, Ammerer G. 2000. Forkhead-like transcription factors recruit Ndd1 to the chromatin of G2/M-specific promoters. *Nature* 406:94–98

84. Krebs JE, Kuo MH, Allis CD, Peterson CL. 1999. Cell cycle-regulated histone acetylation required for expression of the yeast HO gene. *Genes Dev.* 13:1412–21

85. Kumar R, Reynolds DM, Shevchenko A, Goldstone SD, Dalton S. 2000. Forkhead transcription factors, Fkh1p and Fkh2p, collaborate with Mcm1p to control transcription required for M-phase. *Curr. Biol.* 10:896–906

86. La Thangue NB, Taylor WR. 1993. A structural similarity between mammalian and yeast transcription factors for cell-cycle-regulated genes. *Trends Cell Biol.* 3:75–76

87. Laabs TL, Markwardt DD, Slattery MG, Newcomb LL, Stillman DJ, Heideman W. 2003. ACE2 is required for daughter cell-specific G1 delay in *Saccharomyces cerevisiae*. *Proc. Natl. Acad. Sci. USA* 100:10275–80

88. Laoukili J, Kooistra MR, Bras A, Kauw J, Kerkhoven RM, et al. 2005. FoxM1 is required for execution of the mitotic programme and chromosome stability. *Nat. Cell Biol.* 7:126–36

89. Lee KM, Miklos I, Du H, Watt S, Szilagyi Z, et al. 2005. Impairment of the TFIIH-associated CDK-activating kinase selectively affects cell. *Mol. Biol. Cell.* 16:2734–45

90. Lee TI, Rinaldi NJ, Robert F, Odom DT, Bar-Joseph Z, et al. 2002. Transcriptional regulatory networks in *Saccharomyces cerevisiae*. *Science* 298:799–804

91. Liao JC, Boscolo R, Yang YL, Tran LM, Sabatti C, Roychowdhury VP. 2003. Network component analysis: reconstruction of regulatory signals in biological systems. *Proc. Natl. Acad. Sci. USA* 100:15522–27

92. Lim FL, Hayes A, West AG, Pic-Taylor A, Darieva Z, et al. 2003. Mcm1p-induced DNA bending regulates the formation of ternary transcription factor complexes. *Mol. Cell Biol.* 23:450–61

93. Lowndes NF, Johnson AL, Breeden L, Johnston LH. 1992. SWI6 protein is required for transcription of the periodically expressed DNA synthesis genes in budding yeast. *Nature* 357:505–8

94. Lowndes NF, Johnson AL, Johnston LH. 1991. Coordination of expression of DNA synthesis genes in budding yeast by a cell-cycle regulated trans factor. *Nature* 350:247–50

95. Lowndes NF, McInerny CJ, Johnson AL, Fantes PA, Johnston LH. 1992. Control of DNA synthesis genes in fission yeast by the cell-cycle gene cdc10+. *Nature* 355:449–53

96. Loy CJ, Lydall D, Surana U. 1999. NDD1, a high-dosage suppressor of cdc28–1N,

is essential for expression of a subset of late-S-phase-specific genes in *Saccharomyces cerevisiae*. Mol. Cell Biol. 19:3312–27

97. Lydall D, Ammerer G, Nasmyth K. 1991. A new role for MCM1 in yeast: cell cycle regulation of SWI5 transcription. Genes Dev. 5:2405–19

98. MacKay VL, Mai B, Waters L, Breeden LL. 2001. Early cell cycle box-mediated transcription of CLN3 and SWI4 contributes to the proper timing of the G1-to-S transition in budding yeast. Mol. Cell Biol. 21:4140–48

99. MacNeill SA, Nurse P. 1997. Cell cycle control in fission yeast. In *The Molecular and Cellular Biology of the Yeast Saccharomyces: Life Cycle and Cell Biology*, ed. JR Pringle, J Broach, EW Jones, pp. 697–763. Cold Spring Harbor, NY: Cold Spring Harbor Lab. Press

100. Maher M, Cong F, Kindelberger D, Nasmyth K, Dalton S. 1995. Cell cycle-regulated transcription of the CLB2 gene is dependent on Mcm1 and a ternary complex factor. Mol. Cell Biol. 15:3129–37

101. Mai B, Miles S, Breeden LL. 2002. Characterization of the ECB binding complex responsible for the M/G1-specific transcription of CLN3 and SWI4. Mol. Cell Biol. 22:430–41

102. Maqbool Z, Kersey PJ, Fantes PA, McInerny CJ. 2003. MCB-mediated regulation of cell cycle-specific cdc22+ transcription in fission yeast. Mol. Genet. Genom. 269:765–75

103. Martín-Cuadrado AB, Dueñas E, Sipiczki M, Vázquez de Aldana CR, Del Rey F. 2003. The endo-beta-1,3-glucanase eng1p is required for dissolution of the primary septum during cell separation in *Schizosaccharomyces pombe*. J. Cell Sci. 116:1689–98

104. Martin DE, Soulard A, Hall MN. 2004. TOR regulates ribosomal protein gene expression via PKA and the Forkhead transcription factor FHL1. Cell 119:969–79

105. Mata J, Lyne R, Burns G, Bähler J. 2002. The transcriptional program of meiosis and sporulation in fission yeast. Nat. Genet. 32:143–47

106. Matsumoto S, Yanagida M. 1985. Histone gene organization of fission yeast: a common upstream sequence. EMBO J. 4:3531–38

107. Matsumoto S, Yanagida M, Nurse P. 1987. Histone transcription in cell cycle mutants of fission yeast. EMBO J. 6:1093–97

108. McBride HJ, Yu Y, Stillman DJ. 1999. Distinct regions of the Swi5 and Ace2 transcription factors are required for specific gene activation. J. Biol. Chem. 274:21029–36

109. McCollum D, Gould KL. 2001. Timing is everything: regulation of mitotic exit and cytokinesis by the MEN and SIN. Trends Cell Biol. 11:89–95

110. McInerny CJ. 2004. Cell cycle-regulated transcription in fission yeast. Biochem. Soc. Trans. 32:967–72

111. McInerny CJ, Kersey PJ, Creanor J, Fantes PA. 1995. Positive and negative roles for cdc10 in cell cycle gene expression. Nucleic Acids Res. 23:4761–68

112. McInerny CJ, Partridge JF, Mikesell GE, Creemer DP, Breeden LL. 1997. A novel Mcm1-dependent element in the SWI4, CLN3, CDC6, and CDC47 promoters activates M/G1-specific transcription. Genes Dev. 11:1277–88

113. McIntosh EM, Atkinson T, Storms RK, Smith M. 1991. Characterization of a short, cis-acting DNA sequence which conveys cell cycle stage-dependent transcription in *Saccharomyces cerevisiae*. Mol. Cell Biol. 11:329–37

114. Meeks-Wagner D, Hartwell LH. 1986. Normal stoichiometry of histone dimer sets is necessary for high fidelity of mitotic chromosome transmission. Cell 44:43–52

115. Messenguy F, Dubois E. 2003. Role of MADS box proteins and their cofactors in combinatorial control of gene expression and cell development. Gene 316:1–21

116. Miyamoto M, Tanaka K, Okayama H. 1994. res<sup>2+</sup>, a new member of the cdc10<sup>+</sup>/SWI4 family, controls the ‘start’ of mitotic and meiotic cycles in fission yeast. *EMBO J.* 13:1873–80

117. Moll T, Tebb G, Surana U, Robitsch H, Nasmyth K. 1991. The role of phosphorylation and the CDC28 protein kinase in cell cycle-regulated nuclear import of the S. cerevisiae transcription factor SWI5. *Cell* 66:743–58

118. Morgan BA, Bouquin N, Merrill GF, Johnston LH. 1995. A yeast transcription factor bypassing the requirement for SBF and DSC1/MBF in budding yeast has homology to bacterial signal transduction proteins. *EMBO J.* 14:5679–89

119. Morgan DO. 1997. Cyclin-dependent kinases: engines, clocks, and microprocessors. *Annu. Rev. Cell Dev. Biol.* 13:261–91

120. Morillon A, O’Sullivan J, Azad A, Proudfoot N, Mellor J. 2003. Regulation of elongating RNA polymerase II by forkhead transcription factors in yeast. *Science* 300:492–95

121. Morris MC, Kaiser P, Rudyak S, Baskerville C, Watson MH, Reed SI. 2003. Cks1-dependent proteasome recruitment and activation of CDC20 transcription in budding yeast. *Nature* 424:1009–13

122. Nakashima N, Tanaka K, Sturm S, Okayama H. 1995. Fission yeast Rep2 is a putative transcriptional activator subunit for the cell cycle ‘start’ function of Res2-Cdc10. *EMBO J.* 14:4794–802

123. Nasmyth K, Adolf G, Lydall D, Seddon A. 1990. The identification of a second cell cycle control on the HO promoter in yeast: cell cycle regulation of SW15 nuclear entry. *Cell* 62:631–47

124. Nasmyth K, Dirick L. 1991. The role of SWI4 and SWI6 in the activity of G1 cyclins in yeast. *Cell* 66:995–1013

125. Nelson B, Kurischko C, Horecka J, Mody M, Nair P, et al. 2003. RAM: a conserved signaling network that regulates Ace2p transcriptional activity and polarized morphogenesis. *Mol. Biol. Cell* 14:3782–803

126. Obara-Ishihara T, Okayama H. 1994. A B-type cyclin negatively regulates conjugation via interacting with cell cycle ‘start’ genes in fission yeast. *EMBO J.* 13:1863–72

127. Peng X, Murthy Karuturi RK, Miller LD, Lin K, Jia Y, et al. 2005. Identification of cell cycle-regulated genes in fission yeast. *Mol. Biol. Cell* 16:1026–42

128. Pic-Taylor A, Darieva Z, Morgan BA, Sharrocks AD. 2004. Regulation of cell cycle-specific gene expression through cyclin-dependent kinase-mediated phosphorylation of the forkhead transcription factor Fkh2p. *Mol. Cell Biol.* 24:10036–46

129. Pic A, Lim FL, Ross SJ, Veal EA, Johnson AL, et al. 2000. The forkhead protein Fkh2 is a component of the yeast cell cycle transcription factor SFF. *EMBO J.* 19:3750–61

130. Pierrou S, Hellqvist M, Samuelsson L, Enerback S, Carlsson P. 1994. Cloning and characterization of seven human forkhead proteins: binding site specificity and DNA bending. *EMBO J.* 13:5002–12

131. Porter SE, Washburn TM, Chang M, Jaehning JA. 2002. The yeast pafl-RNA polymerase II complex is required for full expression of a subset of cell cycle-regulated genes. *Eukaryot. Cell* 1:830–42

132. Pramila T, Miles S, GuhaThakurta D, Jemiolo D, Breeden LL. 2002. Conserved homeodomain proteins interact with MADS box protein Mcm1 to restrict ECB-dependent transcription to the M/G1 phase of the cell cycle. *Genes Dev.* 16:3034–45

133. Primig M, Sockanathan S, Auer H, Nasmyth K. 1992. Anatomy of a transcription factor important for the start of the cell cycle in *Saccharomyces cerevisiae*. *Nature* 358:593–97

134. Raithatha SA, Stuart DT. 2005. Meiosis-specific regulation of the S.cerevisiae S-phase cyclin CLB5 is dependent upon MCB elements in its promoter but is independent of MBF activity. *Genetics* 169:1329–42

135. Reymond A, Marks J, Simanis V. 1993. The activity of S. pombe DSC-1-like factor is cell cycle regulated and dependent on the activity of p34cdc2. *EMBO J.* 12:4325–34

136. Reynolds D, Shi BJ, McLean C, Katis F, Kemp B, Dalton S. 2003. Recruitment of Thr 319-phosphorylated Ndd1p to the FHA domain of Fkh2p requires Clb kinase activity: a mechanism for CLB cluster gene activation. *Genes Dev.* 17:1789–802

137. Ribar B, Banrevi A, Sipiczki M. 1997. sep1+ encodes a transcription-factor homologue of the HNF-3/forkhead DNA-binding-domain family in Schizosaccharomyces pombe. *Gene* 202:1–5

138. Rustici G, Mata J, Kivinen K, Lio P, Penkett CJ, et al. 2004. Periodic gene expression program of the fission yeast cell cycle. *Nat. Genet.* 36:809–17

139. Schenkman LR, Caruso C, Page N, Pringle JR. 2002. The role of cell cycle-regulated expression in the localization of spatial landmark proteins in yeast. *J. Cell Biol.* 156:829–41

140. Sherr CJ, McCormick F. 2002. The RB and p53 pathways in cancer. *Cancer Cell* 2:103–12

141. Sherwood PW, Tsang SV, Osley MA. 1993. Characterization of HIR1 and HIR2, two genes required for regulation of histone gene transcription in Saccharomyces cerevisiae. *Mol. Cell Biol.* 13:28–38

142. Shore P, Sharrocks AD. 1995. The MADS-box family of transcription factors. *Eur. J. Biochem.* 229:1–13

143. Sidorova JM, Mikesell GE, Breeden LL. 1995. Cell cycle-regulated phosphorylation of Swi6 controls its nuclear localization. *Mol. Biol. Cell* 6:1641–58

144. Sil A, Herskowitz I. 1996. Identification of asymmetrically localized determinant, Ash1p, required for lineage-specific transcription of the yeast HO gene. *Cell* 84:711–22

145. Simon I, Barnett J, Hannett N, Harbison CT, Rinaldi NJ, et al. 2001. Serial regulation of transcriptional regulators in the yeast cell cycle. *Cell* 106:697–708

146. Sipiczki M, Grallert B, Miklos I. 1993. Mycelial and syncytial growth in Schizosaccharomyces pombe induced by novel septation mutations. *J. Cell Sci.* 104:485–93

147. Spector MS, Raff A, DeSilva H, Lee K, Osley MA. 1997. Hir1p and Hir2p function as transcriptional corepressors to regulate histone gene transcription in the Saccharomyces cerevisiae cell cycle. *Mol. Cell Biol.* 17:545–52

148. Spellman PT, Sherlock G, Zhang MQ, Iyer VR, Anders K, et al. 1998. Comprehensive identification of cell cycle-regulated genes of the yeast Saccharomyces cerevisiae by microarray hybridization. *Mol. Biol. Cell* 9:3273–97

149. Stuart D, Wittenberg C. 1995. CLN3, not positive feedback, determines the timing of CLN2 transcription in cycling cells. *Genes Dev.* 9:2780–94

150. Sun K, Coic E, Zhou Z, Durrens P, Haber JE. 2002. Saccharomyces forkhead protein Fkh1 regulates donor preference during mating-type switching through the recombination enhancer. *Genes Dev.* 16:2085–96

151. Tahara S, Tanaka K, Yuasa Y, Okayama H. 1998. Functional domains of rep2, a transcriptional activator subunit for Res2-Cdc10, controlling the cell cycle “start”. *Mol. Biol. Cell* 9:1577–88

152. Takahashi K, Chen ES, Yanagida M. 2000. Requirement of Mis6 centromere connector for localizing a CENP-A-like protein in fission yeast. *Science* 288:2215–19

153. Takizawa PA, Sil A, Swedlow JR, Herskowitz I, Vale RD. 1997. Actin-dependent localization of an RNA encoding a cell-fate determinant in yeast. *Nature* 389:90–93

154. Tanaka K, Okayama H. 2000. A pcl-like cyclin activates the Res2p-Cdc10p cell cycle “start” transcriptional factor complex in fission yeast. *Mol. Biol. Cell* 11:2845–62

155. Tanaka K, Okazaki K, Okazaki N, Ueda T, Sugiyama A, et al. 1992. A new cdc gene required for S phase entry of *Schizosaccharomyces pombe* encodes a protein similar to the cdc 10+ and SWI4 gene products. *EMBO J.* 11:4923–32

156. Taylor IA, Treiber MK, Olivi L, Smerdon SJ. 1997. The X-ray structure of the DNA-binding domain from the *Saccharomyces cerevisiae* cell-cycle transcription factor Mbp1 at 2.1 Å resolution. *J. Mol. Biol.* 272:1–8

157. Toone WM, Aerne BL, Morgan BA, Johnston LH. 1997. Getting started: regulating the initiation of DNA replication in yeast. *Annu. Rev. Microbiol.* 51:125–49

158. Treisman R. 1994. Ternary complex factors: growth factor regulated transcriptional activators. *Curr. Opin. Genet. Dev.* 4:96–101

159. Tyers M. 2004. Cell cycle goes global. *Curr. Opin. Cell Biol.* 16:602–13

160. Tyers M, Tokiwa G, Futcher B. 1993. Comparison of the *Saccharomyces cerevisiae* G1 cyclins: Cln3 may be an upstream activator of Cln1, Cln2 and other cyclins. *EMBO J.* 12:1955–68

161. Visintin R, Craig K, Hwang ES, Prinz S, Tyers M, Amon A. 1998. The phosphatase Cdc14 triggers mitotic exit by reversal of Cdk-dependent phosphorylation. *Mol. Cell* 2:709–18

162. Weiss EL, Kurischko C, Zhang C, Shokat K, Drubin DG, Luca FC. 2002. The *Saccharomyces cerevisiae* Mob2p-Cbk1p kinase complex promotes polarized growth and acts with the mitotic exit network to facilitate daughter cell-specific localization of Ace2p transcription factor. *J. Cell Biol.* 158:885–900

163. White JH, Barker DG, Nurse P, Johnston LH. 1986. Periodic transcription as a means of regulating gene expression during the cell cycle: contrasting modes of expression of DNA ligase genes in budding and fission yeast. *EMBO J.* 5:1705–9

164. White S, Khaliq F, Sotiriou S, McInerny CJ. 2001. The role of DSC1 components cdc10+, rep1+ and rep2+ in MCB gene transcription at the mitotic G1-S boundary in fission yeast. *Curr. Genet.* 40:251–59

165. Whitehall S, Stacey P, Dawson K, Jones N. 1999. Cell cycle-regulated transcription in fission yeast: Cdc10-Res protein interactions during the cell cycle and domains required for regulated transcription. *Mol. Biol. Cell* 10:3705–15

166. Whitfield ML, Sherlock G, Saldanha AJ, Murray JI, Ball CA, et al. 2002. Identification of genes periodically expressed in the human cell cycle and their expression in tumors. *Mol. Biol. Cell* 13:1977–2000

167. Wijnen H, Futcher B. 1999. Genetic analysis of the shared role of CLN3 and BCK2 at the G1-S transition in *Saccharomyces cerevisiae*. *Genetics* 153:1131–43

168. Wijnen H, Landman A, Futcher B. 2002. The G1 cyclin Cln3 promotes cell cycle entry via the transcription factor Swi6. *Mol. Cell Biol.* 22:4402–18

168a. Wittenberg C, Reed SI. 2005. Cell cycle-dependent transcription in yeast: promoters, transcription factors, and transcriptomes. *Oncogene* 24:2746–55

169. Wuarin J, Buck V, Nurse P, Millar JB. 2002. Stable association of mitotic cyclin B/Cdc2 to replication origins prevents endoreduplication. *Cell* 111:419–31

170. Wynne J, Treisman R. 1992. SRF and MCM1 have related but distinct DNA binding specificities. *Nucleic Acids Res.* 20:3297–303

171. Wyrick JJ, Young RA. 2002. Deciphering gene expression regulatory networks. *Curr. Opin. Genet. Dev.* 12:130–36

172. Yamano H, Kitamura K, Kominami K, Lehmann A, Katayama S, et al. 2000. The spike of S phase cyclin Cig2 expression at the G1-S border in fission yeast requires both APC and SCF ubiquitin ligases. *Mol. Cell* 6:1377–87

173. Zhu G, Spellman PT, Volpe T, Brown PO, Botstein D, et al. 2000. Two yeast forkhead genes regulate the cell cycle and pseudohyphal growth. *Nature* 406:90–94

174. Zhu Y, Takeda T, Nasmyth K, Jones N. 1994. pctl+, which encodes a new DNA-binding partner of p85cdc10, is required for meiosis in the fission yeast *Schizosaccharomyces pombe*. *Genes Dev.* 8:885–98

175. Zhu Y, Takeda T, Whitehall S, Peat N, Jones N. 1997. Functional characterization of the fission yeast Start-specific transcription factor Res2. *EMBO J.* 16:1023–34

176. Zilahi E, Salimova E, Simanis V, Sipiczki M. 2000. The S. *pombe sep1* gene encodes a nuclear protein that is required for periodic expression of the cdc15 gene. *FEBS Lett.* 481:105–8

---

**RELATED RESOURCES**

The Regulation of Histone Synthesis in the Cell Cycle  
MA Osley  
Annual Review of Biochemistry. Volume 60, Page 827–861, July 1991  

Yeast Transcriptional Regulatory Mechanisms  
K Struhl  
Annual Review of Genetics. Volume 29, Page 651–674, December 1995  

Getting Started: Regulating the Initiation of DNA Replication in Yeast  
WM Toone, BL Aerne, BA Morgan, LH Johnston  
Annual Review of Microbiology. Volume 51, Page 125–149, October 1997  

Cyclin-Dependent Kinases: Engines, Clocks, and Microprocessors  
DO Morgan  
Annual Review of Cell and Developmental Biology. Volume 13, Page 261–291, November 1997  

Transcription of Eukaryotic Protein-Coding Genes  
TI Lee, RA Young  
Annual Review of Genetics. Volume 34, Page 77–137, December 2000  

The Dynamics of Chromosome Organization and Gene Regulation  
DL Spector  
Annual Review of Biochemistry. Volume 72, Page 573–608, July 2003

Contents

John Maynard Smith  
Richard E. Michod ………………………………………… 1  

The Genetics of Hearing and Balance in Zebrafish  
Teresa Nicolson ………………………………………… 9  

Immunoglobulin Gene Diversification  
Nancy Maizels ………………………………………… 23  

Complexity in Regulation of Tryptophan Biosynthesis in  
*Bacillus subtilis*  
Paul Gollnick, Paul Babitzke, Alfred Antson, and Charles Yanofsky …………………… 47  

Cell-Cycle Control of Gene Expression in Budding and Fission Yeast  
Jürg Bähler ………………………………………… 69  

Comparative Developmental Genetics and the Evolution of Arthropod  
Body Plans  
David R. Angelini and Thomas C. Kaufman ………………………………………… 95  

Concerted and Birth-and-Death Evolution of Multigene Families  
Masatoshi Nei and Alejandro P. Rooney ………………………………………… 121  

*Drosophila* as a Model for Human Neurodegenerative Disease  
Julide Bilen and Nancy M. Bonini ………………………………………… 153  

Molecular Mechanisms of Germline Stem Cell Regulation  
Marco D. Wong, Zhigang Jin, and Ting Xie ………………………………………… 173  

Molecular Signatures of Natural Selection  
Rasmus Nielsen ………………………………………… 197  

T-Box Genes in Vertebrate Development  
L.A. Naiche, Zachary Harrelson, Robert G. Kelly, and Virginia E. Papaioannou …………………… 219  

Connecting Mammalian Genome with Phenome by ENU Mouse  
Mutagenesis: Gene Combinations Specifying the Immune System  
Peter Papathanasiou and Christopher C. Goodnow ………………………………………… 241  

Evolutionary Genetics of Reproductive Behavior in *Drosophila*:  
Connecting the Dots  
Patrick M. O'Grady and Therese Anne Markow ………………………………………… 263

Sex Determination in the Teleost Medaka, *Oryzias latipes*  
Masura Matsuda ………………………………………… 293  

Orthologs, Paralogs, and Evolutionary Genomics  
Eugene V. Koonin ………………………………………… 309  

The Moss *Physcomitrella patens*  
David Cove ……………………………………………… 339  

A Mitochondrial Paradigm of Metabolic and Degenerative Diseases, Aging, and Cancer: A Dawn for Evolutionary Medicine  
Douglas C. Wallace ………………………………………… 359  

Switches in Bacteriophage Lambda Development  
Amos B. Oppenheim, Oren Kobiler, Joel Stavans, Donald L. Court, and Sankar Adhya ………………………………………… 409  

Nonhomologous End Joining in Yeast  
James M. Daley, Phillip L. Palmbos, Dongliang Wu, and Thomas E. Wilson ………………………………………… 431  

Plasmid Segregation Mechanisms  
Gitte Ebersbach and Kenn Gerdes ………………………………………… 453  

Use of the Zebrafish System to Study Primitive and Definitive Hematopoiesis  
Jill L.O. de Jong and Leonard I. Zon ………………………………………… 481  

Mitochondrial Morphology and Dynamics in Yeast and Multicellular Eukaryotes  
Koji Okamoto and Janet M. Shaw ………………………………………… 503  

RNA-Guided DNA Deletion in Tetrahymena: An RNAi-Based Mechanism for Programmed Genome Rearrangements  
Meng-Chao Yao and Ju-Lan Chao ………………………………………… 537  

Molecular Genetics of Axis Formation in Zebrafish  
Alexander F. Schier and William S. Talbot ………………………………………… 561  

Chromatin Remodeling in Dosage Compensation  
John C. Lucchesi, William G. Kelly, and Barbara Panning ………………………………………… 615  

INDEXES  

Subject Index ……………………………………………… 653  

ERRATA  

An online log of corrections to *Annual Review of Genetics* chapters may be found at http://genet.annualreviews.org/errata.shtml
