
REVIEW

NORMAL HUNTINGTIN FUNCTION:  
AN ALTERNATIVE APPROACH TO  
HUNTINGTON'S DISEASE  

Elena Cattaneo, Chiara Zuccato and Marzia Tartari  

Abstract | Several neurological diseases are characterized by the altered activity of one or a few ubiquitously expressed cell proteins, but it is not known how these normal proteins turn into harmful executors of selective neuronal cell death. We selected huntingtin in Huntington's disease to explore this question because the dominant inheritance pattern of the disease seems to exclude the possibility that the wild-type protein has a role in the natural history of this condition. However, even in this extreme case, there is considerable evidence that normal huntingtin is important for neuronal function and that the activity of some of its downstream effectors, such as brain-derived neurotrophic factor, is reduced in Huntington's disease.

RNA INTERFERENCE (RNAi). A method by which double-stranded RNA that is encoded on an exogenous vector can be used to interfere with normal RNA processing, causing rapid degradation of the endogenous RNA and thereby precluding translation. This provides a simple way of studying the effects of the absence of a gene product in simple organisms and in cells.

The devastating neurodegenerative disorder Huntington's disease (HD) is caused by a mutation in a curious protein called huntingtin. Unlike many other proteins of a similar size (348 kDa), huntingtin is completely soluble, essential for embryogenesis, and ubiquitously expressed in moderate amounts in and outside the nervous system¹. The disease-causing gene mutation consists of an expanded CAG tract (>35 repeats) at the 5′ end, which is translated into a corresponding polyglutamine stretch (polyQ)² that makes medium spiny neurons in the striatum particularly vulnerable to cell death, and also leads to the dysfunction and death of neurons in other brain regions, including the cortex³.

The disease manifests at a mean age of 35 years, and is fatal after 15–20 years of progressive neurodegeneration. Its symptoms include motor dysfunction and cognitive abnormalities⁴⁵. Its dominant inheritance pattern and important experimental evidence, reviewed elsewhere⁶⁷, indicate that the mutant huntingtin protein is toxic and initiates the disease insofar as there is an inverse correlation between CAG length and disease onset, mainly in relation to the largest CAG size⁸⁹.

More recently, it has been suggested that the loss of normal huntingtin function might also contribute to the pathogenesis of HD¹. The reduced physiological activity of huntingtin in HD might also affect the development of therapeutic strategies: for example, RNA INTERFERENCE (RNAi) strategies aimed at blocking the disease-triggering mutation would have to be selective for the mutant allele¹⁰. Furthermore, the identification of the affected downstream targets of wild-type huntingtin might allow the development of drugs that can restore their activity¹¹, in addition to the current strategies aimed at blocking the toxicity of the mutant protein¹²¹³.

Here, we first discuss the structure and evolution of huntingtin, which might be important in understanding its cellular activity. We then consider studies that have investigated the biological function of huntingtin during development, and highlight its physiological role in the brain and its known downstream targets. Finally, we discuss evidence that points to the importance of the loss of wild-type huntingtin in HD. It is known that increased wild-type huntingtin expression improves brain cell survival¹⁴⁻¹⁸; removal of wild-type huntingtin generates some of the phenotypes observed in the presence of mutant huntingtin¹⁹⁻²³; wild-type huntingtin expression mitigates the effect of the mutant protein¹⁶,²³⁻²⁶; and deletion of the wild-type allele in an animal model of HD causes more damage²⁷ (BOX 1).

Department of  
Pharmacological Sciences  
and Center of Excellence on  
Neurodegenerative  
Diseases, University of  
Milan, Via Balzaretti 9,  
20133 Milano, Italy.  
Correspondence to E.C.  
e-mail: elena.cattaneo@unimi.it  
doi:10.1038/nrn1806  
Published online  
15 November 2005

Box 1 | Wild-type huntingtin and neuronal pathology

Various experimental approaches have been used to investigate normal huntingtin function and its possible involvement in the pathogenesis of Huntington’s disease. The data show that increased wild-type huntingtin expression leads to improved brain cell survival; removal of the wild-type protein generates some of the phenotypes observed in the presence of mutant huntingtin; wild-type huntingtin expression mitigates the effect of the mutant protein; and depletion of the wild-type protein in a Huntington’s disease background causes more damage. Wild-type huntingtin is shown in green, mutant huntingtin in orange.

Increased expression of wild-type huntingtin leads to:

- Protection from apoptotic neuronal cell death in response to toxic stimuli *in vitro*<sup>14,66,87</sup>
- Reduced apoptosis after toxic stimuli *in vivo* in mice<sup>17,25</sup>
- Dose-dependent neuroprotection from excitotoxicity *in vivo* in mice<sup>18</sup>
- Increased transcription of brain-derived neurotrophic factor (BDNF) and other Repressor element 1/neuron-restrictive silencer element (RE1/NRSE)-controlled neuronal genes in mice<sup>15,16</sup>

Reduced levels of wild-type huntingtin lead to:

- Embryonic lethality in huntingtin-knockout (*Hdh*<sup>-/-</sup>) mice<sup>71–73</sup>
- Abnormal brain development with huntingtin levels of <50% in mice<sup>80,81</sup>
- Neuronal cell death and neurological dysfunctions in *Hdh*<sup>+/−</sup> mice<sup>20,71</sup>
- Apoptosis in the brain and testes, and a Huntington’s disease-like phenotype in adult conditional *Hdh*<sup>-/-</sup> mice (in which *Hdh* is knocked out in both endogenous alleles only in post-mitotic neurons)<sup>20</sup>
- Fewer haematopoietic and neuronal progenitors from cultured *Hdh*<sup>-/-</sup> embryonic stem cells<sup>82,83</sup>
- Abnormal distribution and morphology of cellular organelles<sup>33</sup>
- Reduced vesicle and mitochondria transport in *Drosophila melanogaster* huntingtin (*htt*) RNAi (in which huntingtin mRNA is depleted by RNA interference) and in neurons from conditional *Hdh*<sup>-/-</sup> mice<sup>21,22</sup>
- Reduced BDNF vesicular transport in a mouse cell line after *htt* RNAi<sup>23</sup>

Increased expression of wild-type huntingtin in a mutant huntingtin background causes:

- Reduced toxicity in peripheral cells overexpressing mutant huntingtin<sup>24</sup>
- Inactivation of the RE1/NRSE silencer in heterozygous mutant huntingtin knock-in cells<sup>16</sup>
- Reduced neuronal toxicity mediated by NMDA (*N*-methyl-*D*-aspartate) or kainate receptors in rodents<sup>26</sup>

Depletion of wild-type huntingtin in a mutant huntingtin background causes:

- Increased apoptotic cell death in the testes of YAC72<sup>-/-</sup> mice (in which *Hdh* is knocked out in both endogenous alleles, and an additional yeast-derived artificial chromosome (YAC) containing the entire human mutant huntingtin 72-CAG gene is inserted) in comparison with YAC72<sup>+/+</sup> mice (in which *Hdh* is wild-type in both endogenous alleles, and an additional YAC containing the entire human mutant huntingtin gene is inserted)<sup>25</sup>
- Worsening of behavioural phenotype and increased apoptosis in the testes of YAC128<sup>-/-</sup> mice compared with YAC128<sup>+/+</sup> mice<sup>27</sup>

Although it does not yet show conclusively that the loss of normal huntingtin functions has an impact on human pathology, this growing list of findings provides strong support for this proposal and offers a useful example of how investigating the biology of disease-linked proteins can increase our knowledge of brain function and dysfunction. Examples of other diseases whose natural history might be at least partially due to the loss of physiological function of the normal protein are discussed briefly in BOX 2.

From structure to potential function

Huntingtin is a completely soluble protein of 3,144 amino acids. It has many potential domains, the boundaries and activities of which are not fully understood. It has no sequence homology with other proteins and is expressed ubiquitously in humans and rodents, with the highest levels in CNS neurons and the testes<sup>28–30</sup>. Intracellularly, huntingtin is associated with various organelles, including the nucleus, endoplasmic reticulum and Golgi complex<sup>31–35</sup>, and it is also found in neurites and at synapses, where it associates with vesicular structures such as clathrin-coated vesicles, endosomal compartments or caveolae, and microtubules<sup>36</sup>. This widespread subcellular localization does not facilitate the definition of its function.

The primary amino acid sequence of huntingtin reveals little, as there are only a few known sequence motifs and no structural domains with defined functions (FIG. 1). One obviously important portion of the

Box 2 | Normal PrP<sup>c</sup> and FMRP function in human pathology

Two types of disease whose natural history might be at least partially due to the loss of physiological function of the normal protein, despite the involvement of greater toxicity of the altered protein, are the transmissible spongiform encephalopathies (TSEs) and fragile X syndrome.

TSEs are fatal neurodegenerative diseases, such as **Creutzfeldt–Jakob disease**, that affect humans and other mammals. They are caused by the accumulation of PrP<sup>sc</sup>, an abnormally folded form of the prion protein. PrP<sup>sc</sup> acts in a dominant-negative manner by recruiting and modifying the conformation of the normal cellular form (PrP<sup>c</sup>), therefore leading to loss of its normal function in the absence of genetic insufficiency. PrP<sup>c</sup> is conserved across species, which suggests that it has some relevance to basic physiological processes. Although its precise function is not known, accumulating evidence shows that it is involved in: copper and/or zinc ion transport or metabolism<sup>139,140</sup>; protection from oxidative stress<sup>141</sup>; cell signalling<sup>142,143</sup>; membrane excitability and synaptic transmission<sup>144,145</sup>; neuritogenesis<sup>146</sup>; and apoptosis<sup>147</sup>. Like huntingtin, these functions can be achieved by means of PrP<sup>c</sup> binding to different interactors at different cellular locations<sup>148</sup>. The co-expression of both cytosolic and membrane-bound PrP<sup>c</sup> with Bax abolishes Bax-mediated cell death, which indicates that PrP<sup>c</sup> is involved in anti-apoptotic activity<sup>147</sup>.

Fragile X syndrome is an X-linked dominant disorder that involves the **fragile X mental retardation protein** (FMRP). In the presence of the mutation, FMRP loses its physiological activity, and this has been associated with the synaptic dysfunction observed in the disease. FMRP acts as a translational repressor of specific synaptic mRNAs (brain cytoplasmic RNA 1, BC1 RNA), and its mutation impairs this normal and necessary ‘break’ in translation and leads to an abnormally increased synaptic presence of active proteins<sup>149</sup>.

mammalian protein is the polyQ region itself, which is also present in many transcription factors and aberrantly expanded in at least eight other disease-causing proteins<sup>37</sup>. Because each of these diseases is characterized by the loss of a specific subset of neurons, we have previously proposed that sequences around the glutamine tract might have a crucial role in disease pathogenesis<sup>1</sup>.

The polyQ stretch in huntingtin begins at the eighteenth amino acid, and, in unaffected individuals, contains up to 34 glutamine residues<sup>2</sup>. In 1994, Perutz *et al.* showed that this portion forms a POLAR ZIPPER structure, and suggested that its physiological function is to bind transcription factors that contain a polyQ region<sup>38</sup>; it has now been shown that wild-type huntingtin interacts with several partners and that the polyQ tract is a key regulator of such binding<sup>39–41</sup>.

In higher vertebrates, the polyQ region is followed by a polyproline (polyP) stretch, which might help to keep the protein in solution<sup>42</sup>. Downstream of these regions are the so-called HEAT repeats, which are ~40-amino-acid-long sequences that occur multiple times within a given protein and are involved in protein–protein interactions<sup>43,44</sup> (FIG. 1). Bioinformatics analysis has found 37 putative HEAT repeats in huntingtin<sup>45</sup> and, despite contradictory findings concerning the number of functionally active repeats<sup>46</sup>, three main clusters have been identified<sup>43</sup>. The repeats are well conserved in huntingtin from all vertebrates, which may indicate that it interacts with the same proteins across vertebrates, whereas 28 putative consensus repeats have been found in *Drosophila melanogaster* huntingtin, whose degree of conservation with respect to humans has yet to be fully defined<sup>46</sup>.

Huntingtin contains a functionally active carboxy (C)-terminal nuclear export signal (NES) sequence and a less active nuclear localization signal (NLS), which might indicate that the protein (or a portion of it) is involved in transporting molecules from the nucleus to the cytoplasm<sup>47</sup>. This hypothesis is supported by huntingtin’s perinuclear and nuclear distribution, and the recent demonstration that the 17 amino acids before the polyQ region interact with the nuclear pore protein TPR (translocated promoter region), which exports proteins from the nucleus. Removal of these amino acids causes huntingtin to accumulate in the nucleus<sup>48</sup>.

Huntingtin also contains three well-characterized protease cleavage consensus sites<sup>49–53</sup> (FIG. 1), where cleavage generally leads to fragments of both normal and mutant huntingtin, although the latter is more susceptible to proteolysis and generates fragments that are found in the cytoplasm and nucleus<sup>54–58</sup>. Caspase cleavage sites are typically conserved in vertebrate huntingtin, but not in that of *D. melanogaster*<sup>49–51,58,59</sup>. Functionally active calpain cleavage sites have been described for mouse and human huntingtin<sup>52,53</sup>. Other sites, the exact amino acid positions of which are not well defined, are preferentially cleaved in some brain regions<sup>60</sup> (FIG. 1).

The contribution of huntingtin proteolysis to cell function is unclear. However, modifications in the activity of caspase and calpain reduce the proteolysis and toxicity of the mutant protein, and delay disease progression<sup>51,53,61</sup>.

Huntingtin is subject to four types of post-translational modification: the amino (N)-terminal lysines K6, K9 and K15 compete for sumoylation and ubiquitination<sup>42,62</sup>, and phosphorylation at serines 421 and 434 influences cleavage and toxicity, and is reduced in HD<sup>63–66</sup>. Huntingtin is also palmitoylated by its co-partner, huntingtin-interacting protein 14 (HIP14, a palmitoyl transferase), but the precise amino acid position involved is unknown. The palmitoylation of huntingtin is consistent with its proposed role in regulating vesicular trafficking, because palmitoylated proteins are often involved in the dynamic assembly of components that control vesicle trafficking and synaptic vesicle function<sup>31,67</sup>.

POLAR ZIPPER  
A term used by Max Perutz to describe a three-dimensional protein motif consisting of interactions between polar residues on separate subunits or separate proteins.

born with no apparent defects^{76,77}, which suggests that the CAG tract does not affect the physiological function of the protein during development, and also shows that mutant huntingtin can compensate for the loss of wild-type function in development (FIG. 2).

Patients with **Wolf-Hirschhorn syndrome**, a rare condition in which there is a deletion on chromosome 4 that comprises the CAG triplet repeats region (IT15), are born and do not develop HD^{78}. In addition, a rare patient with a balanced translocation with a breakpoint between exons 40 and 41 shows no disease phenotype^{79}. This evidence indicates that the presence of one fully functional allele (at least 50% huntingtin expression) is compatible with life in humans and that HD is not caused by a simple loss of function of the huntingtin gene. However, we now know that, in addition to its early extra-embryonic function, huntingtin is involved in other important activities that, if brought below a certain level (<50%), might render cells more vulnerable to the genetic stress imposed by the CAG mutation^{1}.

Taken together, these studies have identified potential activities of wild-type huntingtin, the most remarkable of which is its ability to bind several proteins in both its normal and mutated forms^{39,40}. Huntingtin might, therefore, have a flexible or multifunctional structure capable of assuming specific conformations and activities depending on its subcellular location and time of maturation in a given cell^{6,39,41,45,68}. Alternatively, it might be born as a natively unfolded protein. The evidence in favour of the structural flexibility of the huntingtin backbone is that different protein epitopes are differently accessible to various antibodies depending on their subcellular compartments^{69,70}, and that the elongated polyQ region can modify the three-dimensional structure of the entire protein and so change its interactors^{38,39}.

**EXTRA-EMBRYONIC TISSUE**
All tissues that do not contribute to the embryo but support its development.

**VISCERAL ENDODERM**
Extra-embryonic tissue in pre-gastrulation stages of the mouse embryo.

**ONTOGENESIS**
The events involved in the development of an organism, from the earliest embryonic stage to maturity.

**PHYLOGENESIS**
The events involved in the evolution of a species.

**NEUROGENESIS**
The birth of new neurons, which occurs not only in developing organisms but also throughout adult life in both vertebrates and invertebrates. Ongoing neurogenesis is thought to be an important mechanism underlying neuronal plasticity, allowing organisms to adapt to environmental changes, and influencing learning and memory throughout life.

**CHIMAERA**
An organism that is composed of cells derived from at least two genetically different zygotes.

**Huntingtin function during development**

Huntingtin is essential for embryonic development: its complete inactivation in huntingtin-knockout mice (*Hdh*⁻/⁻) causes embryonic death before day 8.5 (before gastrulation and the formation of the nervous system)^{71–73}. This correlates with defects in the organization of EXTRA-EMBRYONIC TISSUE^{73}, possibly as a consequence of an alteration in the nutritive function of the VISCERAL ENDODERM^{74}.

On the basis of Haeckel’s suggestion in 1899 that mammalian ONTOGENESIS recapitulates PHYLOGENESIS^{75}, the early non-neuronal activity of huntingtin can be likened to its ancestral function in species with a poorly organized or no nervous system. The extra-embryonic function of huntingtin (that is, in tissues such as the placenta) does not depend on the CAG tract and ancestral huntingtin has no CAG domain (BOX 3). Furthermore, the expression of mammalian huntingtin with a pathological polyQ expansion (even up to 128 CAG repetitions) rescues huntingtin-null mice from embryonic lethality^{25,27}. Finally, patients with HD who are homozygous for the CAG expansion do not express wild-type huntingtin but develop normally and are

Analyses of CHIMAERAS created by injection of *Hdh*⁻/⁻ embryonic stem cells into the blastocyst have shown that wild-type huntingtin is also crucial for establishing and maintaining neuronal identity, especially in the cortex and striatum^{84}. Although some adult brain regions were appropriately colonized by *Hdh*⁻/⁻ neurons (their functional activity has not been evaluated), few donor cells were found in the cerebral cortex, striatum and basal ganglia, which indicates that huntingtin might have a region-specific role in neuronal survival. Preliminary analyses of chimaeras at embryonic day 12.5 showed ongoing *Hdh*⁻/⁻ cell degeneration specific to the striatum, cortex and thalamus, which supports the view that neuroblasts in these areas need to synthesize huntingtin if they are to progress in development and differentiation^{85}.

These data indicate that huntingtin is required at different stages of development, and that its total absence or a >50% reduction in its levels generates an early phenotype in mice (BOX 1; FIG. 1). By contrast, like humans, mice carrying 50% full-length wild-type huntingtin (that is, one allelic dose) reach normal adult life^{71–73}. However, in adulthood, one in three heterozygous knockout mice shows behavioural

Box 3 | Huntingtin during evolution: a route to its function?

| Insects (Drosophila)                                                                 | Vertebrates (from fishes to human)                                                                 |
|-------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|
| Nematodes                                                                          | Halocynthia roretzi                                                |
| Protostoma                                                                         | Strongylocentrotus purpuratus                                      |
|                                                                                    | Heliocidaris erythrogramma                                          |
| Dictyostelium                                                                      | Deuterostoma                                                        |
| Saccharomyces cerevisiae                                                           |                                                                    |
| Fungi/Myxomycetes                                                                  |                                                                    |

In the absence of information about its three-dimensional structure, comparisons of huntingtin homologues might help to define conserved or newly emergent functional domains in mammalian cells. Huntingtin homologues in the vertebrate subphylum are highly conserved (80%)<sup>150</sup>, whereas *Drosophila melanogaster* huntingtin, the only entirely known invertebrate sequence, is characterized by an additional region and by five 20–50% conserved regions<sup>59</sup>; these may represent a residue of the ancestral huntingtin at the origin of the protostoma–deuterostoma branches (yellow-green). In support of this, huntingtin seems to be present in *Dictyostelium discoideum*, but not in the older *Saccharomyces cerevisiae* (fungi, light blue) or in previously divergent plants, which suggests that huntingtin is an evolutionarily old gene that has been lost in some later animals<sup>46,59</sup> (for example, *Caenorhabditis elegans*<sup>151</sup>). Only partial sequences are available from other deuterostoma invertebrates (one tunicata, two echinodermata), in which expression of the partially identified carboxy-terminal huntingtin seems to be confined to non-neural tissues<sup>152</sup>. By contrast, in vertebrates huntingtin is expressed throughout life and in all tissues, but is particularly enriched in the brain, which suggests that it may have acquired individual activities important for the newly formed nervous system and essential for post-mitotic neurons.

Although huntingtin passages linking vertebrates and *D. melanogaster* along the evolutionary tree have not been fully described, the available information raises the possibility that *D. melanogaster* huntingtin has developed some different functions from those of the mammalian protein<sup>21,22</sup>.

The polyglutamine (polyQ) region seems to have appeared in huntingtin for the first time in fishes and has been maintained during vertebrate evolution, becoming highly polymorphic only in humans (*D. melanogaster* huntingtin has no polyQ region<sup>59</sup>). The four glutamine residues in fishes' huntingtin are not followed by a polyproline (polyP) tract, which appeared later in vertebrate evolution, possibly providing structural advantages<sup>150,153,154</sup>. Because polyP helps to maintain protein solubility<sup>42</sup>, it is also possible that, during evolution, an expanded polyQ has conferred important molecular function(s) to the protein, in part because of cooperation with the emerging polyP tract.

abnormalities, cognitive deficits and significant neural loss in the subthalamic nucleus, suggesting that reduced huntingtin levels could contribute to HD pathogenesis<sup>19,71</sup>. This phenotype, which is not replicated in the other two knockout mice, might reflect a DOMINANT-NEGATIVE effect of a remaining 20 kDa N-terminal fragment over the full-length wild-type protein, regardless of the CAG tract.

From these data we can conclude that wild-type huntingtin has important functions that build up during mammalian development, and possibly reflect the evolutionary steps of the protein (FIG. 2; BOX 3). In higher vertebrates (late deuterostoma species, see BOX 3), huntingtin might have acquired activities that

DOMINANT-NEGATIVE  
Describes a mutant molecule that can form a heteromeric complex with the normal molecule, knocking out the activity of the entire complex.

are important for the newly formed nervous system and essential for post-mitotic neurons. This hypothesis has been tested in gene overexpression and deletion studies of cultured brain cells and *in vivo*, and the results suggest that wild-type huntingtin is important for the survival of mammalian neurons, in which it controls many cell functions, such as neuronal gene transcription, and axonal and vesicular transport.

### Huntingtin is anti-apoptotic

Various data highlight the continued importance of wild-type huntingtin function in post-mitotic brain cells and adulthood (BOX 1; FIG. 1). *In vitro* studies of conditionally immortalized striatum-derived cells over-expressing the human wild-type protein<sup>14</sup> provided the first direct demonstration that wild-type huntingtin is neuroprotective in brain cells exposed to various apoptotic stimuli, such as serum deprivation, mitochondrial toxins or the transfection of death genes<sup>14</sup>. It has also been shown that the *in vitro* anti-apoptotic activity of wild-type huntingtin is contained within the 548 N-terminal amino acids of the normal protein, the first recognized domain endowed with biological function<sup>14</sup>. These data also suggest that this specific function of huntingtin is cell-autonomous.

The *in vivo* function of the wild-type protein in neurons was tested using the Cre/loxP SITE-SPECIFIC RECOMBINATION SYSTEM driven by the neuron-specific α-subunit of the calcium/calmodulin-dependent protein kinase II (α-CaMKII) promoter to generate a null mutation of huntingtin at late stages of mouse forebrain development<sup>20</sup>. The neuronal inactivation of huntingtin led to neurological abnormalities and progressive degeneration (apoptotic cells in the hippocampus, cortex and striatum, and a lack of axons), with death occurring by the age of 1 year<sup>20</sup>. Earlier morphometric and ultrastructural analyses of one heterozygous knockout mouse strain yielded similar results<sup>19</sup>.

The overexpression of wild-type huntingtin also protects against ischaemic injury *in vivo*<sup>17</sup> and against EXCITOTOXICITY<sup>18</sup>, to which cells are made more susceptible by the full-length mutant protein<sup>18,86</sup>. Neuroprotection is enhanced with a progressive increase in the level of wild-type huntingtin, which indicates a gene-dosage effect<sup>18</sup>. All of these studies indicate that wild-type huntingtin can trigger molecular events that lead to increased cell survival well beyond the developmental period. Investigations of the underlying molecular mechanisms have shown that wild-type huntingtin prevents cells from dying by inhibiting the processing of pro-caspase 9 (REFS 14,87), and that it also inhibits the formation of the pro-apoptotic protein interactor of huntingtin-interacting protein 1 (HIPPI)—HIP1 (huntingtin-interacting protein 1) complex<sup>88</sup>. Huntingtin can also be a substrate for Akt, a serine/threonine kinase that activates pro-survival pathways<sup>63,89</sup>.

### Huntingtin controls BDNF production

Wild-type huntingtin has also been linked to brain-derived neurotrophic factor (BDNF), a neurotrophin that is particularly important for the survival of striatal

REVIEW S

In rodents, the *Bdnf* gene contains four 5′ exons (I–IV) that are linked to separate promoters to produce four different transcripts (mRNAs I–IV), which are spliced to the fifth 3′ exon (V) to produce the BDNF protein. The different promoter-like sequences upstream of exons I–IV act independently to modulate the tissue transcription of mRNAs I–IV in a development- and stimulus-specific manner<sup>105,106</sup>. These different transcripts may also have differential subcellular localization and targets<sup>107</sup>. Human *BDNF* is even more complex and contains a larger number of exons, some of which are still under investigation (GenBank accession number NT_009237.17, nucleotide positions 26463685–26530516). The use of promoter reporter assays and polymerase chain reaction (PCR) for the specific mRNAs I–IV showed that enhanced transcription from *Bdnf* exon II accounts for the increased amount of BDNF found in the presence of wild-type huntingtin, whereas transcription from exons I, III and IV is not affected<sup>15</sup>. Conversely, reduced BDNF mRNA levels are found in samples without wild-type huntingtin (C. Z., unpublished observations), as well as in heterozygous huntingtin-knockout mice<sup>16</sup>. Huntingtin mutation causes a similar reduction in the transcription of BDNF mRNA II, and also decreases transcription from *Bdnf* exons III and IV (REFS 15,16).

These findings have led to the conclusion that wild-type huntingtin contributes to the pool of BDNF protein produced in the cerebral cortex, and that a loss of or reduction in wild-type huntingtin activity might affect the stability of the cortical afferents and decrease BDNF support to striatal targets. Other studies have concentrated on BDNF trafficking and suggest that, in addition to controlling BDNF mRNA production, wild-type huntingtin may also regulate BDNF transport along the cortico-striatal afferents, and that huntingtin mutation affects the ability of BDNF to reach its striatal targets.

**Huntingtin facilitates vesicular transport**

Wild-type huntingtin specifically enhances the vesicular transport of BDNF along microtubules<sup>23</sup> (FIG. 4). *In vitro*, full-length wild-type huntingtin stimulates (but mutant huntingtin represses) BDNF vesicular trafficking in neuronal cells, and BDNF transport can be attenuated by reducing the levels of wild-type huntingtin using RNAi. The ability of wild-type huntingtin to enhance vesicular transport involves huntingtin-associated protein 1 (*HAP1*, the first identified huntingtin interactor) and the p150(Glued) subunit of dynactin, which is an essential component of molecular motors. Huntingtin normally interacts with the p150(Glued) subunit via HAP1, and thereby stimulates BDNF transport<sup>23</sup>.

Support for this idea is provided by earlier studies in *D. melanogaster* in which levels of the normal protein were experimentally reduced<sup>21</sup>. The loss of huntingtin in *D. melanogaster* neurons causes defects in FAST AXONAL TRAFFICKING and degeneration of the eye, a phenotype that is identical to that obtained after the expression of an exogenous mutant huntingtin fragment<sup>21,108</sup>. Huntingtin is found predominantly in the

**Cre/loxP SITE-SPECIFIC RECOMBINATION SYSTEM**

A system derived from the *Escherichia coli* bacteriophage P1. Two short DNA sequences (*loxP* sites) are engineered to flank the target DNA. Activation of the Cre-recombinase enzyme catalyses recombination between the *loxP* sites, which leads to excision of the intervening sequence. This tool is used at late stages of maturation or in the adult to study the function of genes whose deletion causes embryonic lethality.

**EXCITOTOXICITY**

Cellular toxicity involving the activation of glutamate receptors in the CNS. Glutamate, an excitatory amino acid neurotransmitter, activates different types of ionotropic (ion channel-forming) and metabotropic (G-protein-coupled) receptor. Excessive activation of these receptors by high concentrations of glutamate or by neurotoxins acting at the same receptors leads to cell death.

**CORTICO-STRIATAL SYNAPSE**

Cortical afferents reaching the striatum are intermingled with other cellular elements (the striatal targets, dopaminergic inputs and glial cells), which may be able to influence the output of the cortex. The activity of each of these cellular elements is finely regulated through a complex interplay between the receptor systems they express.

**FAST AXONAL TRAFFICKING**

Several motor proteins move various cargoes on microtubule tracks — such as membrane organelles, protein complexes, complexes of nucleic acids, signalling molecules, neuroprotective and repair molecules, and vesicular and cytoskeletal components — to deliver them from the neuronal cell body through the long axon to their final destination.

Figure 2 | **Huntingtin function over a lifetime.** Wild-type huntingtin function during development and in adults may arise from the use of different protein domains (pale or dark green), and by recruiting different sets of interactors (coloured circles). During embryogenesis, huntingtin is essential for gastrulation and the development of the nervous system; during development, mutant huntingtin has the same activities as the wild-type protein (and is therefore coloured green); in adulthood, wild-type huntingtin controls the activity of mammalian neurons. In Huntington’s disease, the altered three-dimensional structure of huntingtin bearing an expanded polyglutamine region (orange) gives rise to aberrant interactions that cause toxicity and also possibly interfere with normal huntingtin function.

Wild type

NTYAGMRCCNNRGMSAG

I—IV indicate BDNF promoter exons in rodent *Bdnf*; V indicates the coding region.

Figure 3 | Wild-type but not mutant huntingtin facilitates cortical BDNF mRNA production. Wild-type huntingtin contributes to brain-derived neurotrophic factor (*Bdnf*) transcription in the cortical neurons that project to the striatum by inhibiting the Repressor element 1/neuron-restrictive silencer element (RE1/NRSE) that is located in BDNF promoter exon II. I–IV indicate BDNF promoter exons in rodent *Bdnf*; V indicates the coding region. The RE1/NRSE consensus sequence is shown. Inactivation of the RE1/NRSE in *Bdnf* leads to increased mRNA transcription and protein production in the cortex. BDNF, which is also produced through translation from exons III and IV is then made available to the striatal targets via the cortico-striatal afferents. Wild-type huntingtin might also facilitate vesicular BDNF transport from the cortex to the striatum.

Huntingtin controls neuronal gene transcription

Further investigations of the mechanism by which wild-type huntingtin stimulates *Bdnf* transcription have concentrated on BDNF promoter exon II (FIG. 4), in which a conserved 21–23 base pair DNA Repressor element 1 (RE1; also known as the neuron-restrictive silencer element, NRSE) is recognized by the RE1-silencing transcription factor (*REST*; also known as neuronal restrictive silencing factor, NRSF) transcriptional regulator, which is proposed to act as a transcriptional silencer<sup>111–115</sup>. RE1/NRSE sequences are also found in genes involved in neuronal development and normal neuronal functions, such as ion channels, neurotransmitter receptors and their synthesizing enzymes, and synaptic vesicle proteins. Recent computational analyses of human and mouse genomes have found 1,892 and 1,894 RE1/NRSE sites, respectively<sup>116</sup>, so it is possible that wild-type huntingtin stimulates the transcription of many RE1/NRSE-controlled neuronal genes, other than *Bdnf*, by silencing the RE1/NRSEs.

We have found that wild-type huntingtin promotes *Bdnf* transcription because it sequesters the available REST/NRSF in the cytoplasm, thereby preventing it from forming the nuclear co-repressor complex at the RE1/NRSE nuclear site and allowing gene transcription<sup>16</sup>. Mutant huntingtin causes the pathological entry of REST/NRSF into the nucleus, so leading to repressor complex formation and reduced transcription (FIG. 4). Further data indicate that wild-type huntingtin has a broader role in regulating neuronal gene transcription — cells and mice expressing increased wild-type but not mutant huntingtin levels also show higher levels of mRNAs transcribed from many other RE1/NRSE-containing neuronal genes, whereas mutant huntingtin is associated with reduced transcription from neuronal genes with RE1/NRSEs<sup>16</sup> (C. Z., unpublished observations). Huntingtin might therefore act in the nervous system as a general facilitator of neuronal gene transcription.

Huntingtin at synaptic terminals

Huntingtin is involved in the cell machinery that controls synaptic transmission. Normal huntingtin interacts with a number of cytoskeletal and synaptic vesicle proteins that are essential for exo- and endocytosis at synaptic terminals<sup>117</sup>.

One key molecule in synaptic transmission is PSD-95 (postsynaptic density protein 95), a member of the membrane-associated guanylate kinase (MAGUK) family of proteins that binds the NMDA (*N*-methyl-*D*-aspartate) and kainate receptors at the postsynaptic density<sup>118</sup>. Wild-type huntingtin directly binds the Src homology 3 (SH3) domains of PSD-95 (REF. 26). The decreased interaction of mutant huntingtin with PSD-95 indicates that more PSD-95 is released in HD, which affects the activity of NMDA receptors and might lead to their overactivation or sensitization, and excitotoxicity<sup>26</sup>. By contrast, the overexpression of wild-type huntingtin seems to attenuate the neuronal toxicity induced by NMDA receptors and mutant huntingtin<sup>26</sup>.

Wild-type huntingtin reduces mutant huntingtin toxicity. Expression of mutant huntingtin in the absence of wild-type huntingtin results in massive apoptotic cell death in the testes of male mice²⁵. This cell death can be modulated by the expression of normal huntingtin. In fact, no evidence of apoptosis is seen in the testes of mice expressing human mutant huntingtin when wild-type huntingtin is expressed from both *Hdh* alleles²⁵; the same data also indicate that wild-type huntingtin protects non-neuronal cells from death. Similar results were obtained in *in vitro* experiments involving non-neuronal cells, which showed that wild-type protein overexpression reduces the polyQ toxicity induced by an exogenous mutant huntingtin construct²⁴. Another approach considers crossing mice expressing exogenous wild-type huntingtin with mice carrying the mutant gene. Restoration of at least some parameters is expected (B. Leavitt, unpublished observations).

Reduced wild-type huntingtin increases HD damage. Van Raamsdonk *et al.* compared the phenotypic severity of YAC128⁺/⁺ mice with YAC128⁻/⁻ mice. YAC128⁺/⁺ animals express, over the two endogenous wild-type alleles (+/+), a dose of mutant 128-CAG huntingtin that corresponds to 75% of endogenous levels, whereas YAC128⁻/⁻ mice do not express endogenous wild-type huntingtin (−/−) but express the same amount (75% of endogenous huntingtin levels) of the mutant protein. The loss of wild-type huntingtin in the YAC128⁻/⁻ mice led to a slight worsening of striatal atrophy and neuronal loss, and a small but significant decrease in neuronal cross-sectional area²⁷. YAC128⁻/⁻ mice performed worse than YAC128⁺/⁺ mice in the rotarod test of motor coordination and were hypoactive at 2 months of age. In addition, testicular atrophy and degeneration were markedly worsened in the absence of wild-type huntingtin. Finally, YAC128⁺/⁺ mice showed a male-specific deficit in survival at 12 months of age, which was exacerbated in YAC128⁻/⁻ mice. It is concluded that elimination of wild-type huntingtin expression in YAC128 mice results in the exacerbation of behavioural deficits and survival, with a mild worsening of neuropathology at 12 months. The absence of more severe striatal abnormalities suggests that the striatal phenotype is dependent primarily on mutant huntingtin toxicity. However, it might be important to characterize these mice at a molecular level and at later time points. In addition, it is possible that YAC128⁻/⁻ mice carry a fragment of wild-type huntingtin (not deleted in the original *Hdh⁻/⁻* mice)⁷¹.

The fact that a reduction of wild-type huntingtin worsens the behaviour of HD mice in the absence of a clear striatal pathology also suggests that a simple increase in the wild-type huntingtin allelic dose might not be sufficient to counteract all phenotypes in HD, and that more appropriate strategies should be aimed at restoring the activity of the downstream targets of wild-type huntingtin¹¹,¹²². However, it should be noted that the total amount of intact huntingtin protein in this study was always ≥50%, whereas this might not be true in patients. Another mouse model that stably

Figure 4 | Molecular mechanism of wild-type huntingtin function. Huntingtin activity on (a) the transcription of brain-derived neurotrophic factor (*BDNF*) and of many other neuronal genes and (b) BDNF transport. **a** | Wild-type huntingtin sequesters Repressor element 1 (RE1)-silencing transcription factor (REST, also known as neuronal restrictive silencing factor, NRSF) in the cytoplasm in such a way that the co-repressor complex does not form at the RE1 (also known as the neuron-restrictive silencer element, NRSE) sites and the BDNF gene is transcribed. The asterisk indicates that wild-type huntingtin facilitates the transcription of many other RE1/NRSE-controlled neuronal genes (for example, synapsin I, pro-enkephalin, muscarinic acetylcholine receptor 4 and dynamin 1) as well as BDNF. **b** | In Huntington’s disease, mutant huntingtin is less capable of retaining REST/NRSF in the cytoplasm, and it enters the nucleus where it leads to the formation of the repressor complex and reduced transcription of neuronal genes, including BDNF. **c** | Wild-type huntingtin is also part of a motor complex that controls vesicle BDNF transport along microtubules: it binds to huntingtin-associated protein 1 (HAP1) and indirectly regulates the assembly of p150(Glued) (grey) with the dynein (blue) and dynactin complexes (pink) that ultimately control the microtubule transport of BDNF vesicles (yellow). Arrows indicate the movement of the BDNF vesicles. The transient binding of wild-type huntingtin with cytoskeletal proteins might allow the movement of the BDNF vesicles. **d** | Under pathological conditions, mutant huntingtin binds more tightly to HAP1, reducing the transport of BDNF vesicles along microtubules. coREST, REST co-repressor; HDAC, histone deacetylase; SIN3A, SIN3 homologue A, a transcription regulator.

Reduced huntingtin activity in HD

The above data indicate that wild-type huntingtin has beneficial functions in the mature brain, so it is possible that its loss in patients with HD reduces the ability of neurons to survive the toxic effects of the mutant protein and thereby contributes significantly to disease pathogenesis. In mouse models, homozygosity for the HD mutation leads to a more severe phenotype than heterozygosity for an equivalent CAG expansion in the HD gene¹¹⁹,¹²⁰. Similarly, a small cohort of patients with HD who were homozygous for the CAG expansion showed more severe disease progression than those who were heterozygous for the mutation¹²¹. However, these studies were unable to determine the relative contributions of the loss of wild-type huntingtin and the increased expression of mutant huntingtin to the worsening phenotype.

In an attempt to address this question, a new set of experiments were carried out to investigate the ability of increased wild-type huntingtin to mitigate the damage caused by the mutant protein; the impact of reducing wild-type huntingtin levels in mice carrying the mutant gene; and the specific consequences of reduced levels of wild-type huntingtin downstream effectors (for example, BDNF) in a mouse model of HD.
expresses <50% of total huntingtin protein (wild-type plus mutant) shows an early phenotype at a striatal level<sup>81</sup>; although this analysis was mainly morphological and focused on the early stages compatible with life, the data seem to indicate the presence of striatal defects when huntingtin is below an acceptable level.

### Reduced BDNF levels in HD

The finding that normal huntingtin stimulates *Bdnf* gene transcription/protein production and its axonal transport has prompted analyses of BDNF levels in the cortex and striatum of transgenic mice and patients with HD. As BDNF is synthesized in the cerebral cortex but not in the striatum, mutant huntingtin should reduce mRNA and protein levels in the cortex and lead to reduced striatal BDNF levels. With the exceptions discussed below, a consistent reduction in cortical BDNF protein and its mRNA has been reported in all cases. In fact, reduced BDNF mRNA and protein levels are found in the cerebral cortex from: YAC mice expressing the full-length mutant protein<sup>15,123</sup>; R6/2 mice, which express a 63 amino acid N-terminal portion of mutant huntingtin<sup>17,122,124</sup>; mice expressing a 517 amino acid N-terminal portion of huntingtin with 82 polyQ repeats<sup>125</sup>; mutant huntingtin knock-in mice<sup>126</sup>; and post-mortem cortical tissue from a small cohort of patients with HD<sup>15</sup>.

These data indicate that reduced *Bdnf* transcription occurs in HD. This is due to reduced wild-type huntingtin activity on *Bdnf* exon II. Furthermore, mutant huntingtin also reduces transcription of exons III and IV, which suggests that the lower cortical BDNF levels in HD are due to both the loss of wild-type huntingtin function and increased toxicity of the mutant protein<sup>15,122</sup>.

Wild-type huntingtin also stimulates BDNF transport, and analysis of BDNF protein in a mouse model of HD shows that the mutation causes a reduction in the efficiency of BDNF vesicle transport along microtubules<sup>23</sup>. This study was conducted using heterozygous knock-in mice. However, another study that used the same mice, but in homozygosity, showed a 2.2-fold reduction in BDNF production in the cortex with respect to control mice, possibly affecting its transport along the cortico-striatal afferents<sup>126</sup>. The possibility that a defect in BDNF transport from cortex to striatum occurs in human HD was inferred from western blot data showing reduced BDNF levels in the striatum but not in the cortex of post-mortem brain extracts from ten patients with HD<sup>23</sup>. An earlier study also reported reduced amounts of BDNF in the striatum but not in the cortex<sup>127</sup>. Reduced striatal but not cortical BDNF has also been reported in the R6/1 mouse line, which shows a milder and more delayed phenotype than R6/2 mice<sup>128</sup>. However, in another study, R6/1 mice show normal levels of BDNF in the cortex and striatum<sup>129</sup>. In addition, the results of studies of many other mouse models and a limited number of human cases of HD (reviewed above)<sup>15,17,122–126</sup> have shown reduced levels of cortical BDNF protein (and mRNA), which is consistent with reduced transcriptional activity in HD.

Despite these differences, which subsequent studies will clarify, the data reviewed so far suggest that the reduction in the amount of BDNF reaching striatal targets might cause the preferential vulnerability of striatal neurons in HD. The impact of reduced cortical BDNF has been studied in *Emx–Bdnf* conditional knockout mice, in which cortical BDNF is almost completely eliminated<sup>98</sup>. In addition to finding that striatal and cortical volumes were decreased and the morphology of medium-sized spiny striatal neurons was altered, the study revealed a long-term *in vivo* requirement for cortical BDNF to support striatal neuronal survival. Adult *Emx–Bdnf*-knockout mice also have a hindlimb clasping phenotype similar to that observed in mouse models of HD<sup>98</sup>.

In another experiment, an earlier onset and enhanced severity of motor alterations was detected in mice expressing only one functional *Bdnf* allele on a background of an overexpressed N-terminal mutant huntingtin fragment (the R6/1 line)<sup>129</sup>. In particular, it was found that an insufficient level of BDNF in the striatum of R6/1 mice leads to degeneration of the enkephalinergic striatal projection neurons (those most affected in HD), a phenotype that could be restored by exogenous BDNF administration<sup>129</sup>. Reduced BDNF also seems to exacerbate dopaminergic neuronal dysfunction in the same mouse model<sup>130</sup>.

Together, these data indicate that wild-type huntingtin contributes to the pool of BDNF found in the brain, and that reduced BDNF levels affect the development of HD in mice. Reduced huntingtin function in HD might ultimately occur as a result of a dominant-negative mechanism in the presence of the mutant protein<sup>1,131</sup> and/or wild-type huntingtin sequestration in aggregates<sup>132,133</sup>, or because of its degradation by active caspases and calpains<sup>17,52,53,56,134,135</sup>.

## Concluding remarks

Huntingtin is a large protein that interacts with numerous other proteins, the roles of many of which still await biological confirmation<sup>39,41</sup>. Huntingtin has been found in different cellular locations, where it may act as a scaffold protein for the activation and correct assembly, in time and space, of specific molecular partners and pathways<sup>45,68</sup>. It is ubiquitously expressed and has anti-apoptotic properties in many intra- and extracerebral cell types.

But why is Huntington’s disease limited to neurons, and particularly to striatal neurons? Given the function of wild-type huntingtin in the brain, its reduced activity on specific downstream effectors, such as BDNF, might exacerbate the sensitivity of the striatal neurons to the expanded glutamine repeats<sup>86,136</sup>. In addition, reduced BDNF might severely affect the activity of the cortico-striatal synapse, therefore contributing to selective vulnerability. It is known that striatal neurons from HD mice are much more sensitive to excitotoxins and that changes in BDNF levels can influence the glutamatergic output of the cortex<sup>94,86,137</sup>. Other neurons besides those in the striatum are also affected in HD<sup>3,138</sup>, and reduced wild-type huntingtin activity on
axonal transport, synaptic activity and neuronal gene
transcription might contribute to this generalized
neuronal dysfunction. The neuronal specificity of the
RE1/NRSE–REST/NRSF pathway might be related to
the limited presence of REST/NRSF in the brain (where
its nuclear translocation is regulated by huntingtin),
compared with its level in non-neuronal tissues, where
it represses neuronal genes regardless of the presence
of wild-type or mutant huntingtin<sup>16</sup>.

Is the loss of wild-type huntingtin involved in the
disease process? If wild-type huntingtin has a function
in mammalian neurons and this is inhibited by mutant
huntingtin, the defects observed in mutant mice and
cells should also be observed in the absence or reduced
presence of wild-type huntingtin. Evidence reviewed
here indicates that reducing wild-type huntingtin levels
or expressing the mutant protein might have similar
effects on specific phenotypes, which suggests that
at least some of the molecular dysfunctions observed
in HD are a consequence of reduced wild-type hunt-
ingtin activity. Discrimination of the abnormalities
that depend on the activity of the mutant protein versus
the reduced function of wild-type huntingtin becomes
crucial. In the first case, to restore the normal phenotype

one would have to interfere with the events elicited by
the mutant protein. In the second case, compounds able
to mimic normal huntingtin activity on its downstream
targets would be ideal. The next phase of research on
huntingtin should also consider how this protein has
acquired such important activities; if there are specific
functional domains in the protein and what their
boundaries are; how the activity of these domains relates
to the natural history of HD; and whether there are
wild-type huntingtin effectors that can be exploited in
drug-screening strategies. The RE1/NRSE–REST/NRSF
pathway could be a first target.

Although the structure of huntingtin remains
unknown, a lot of information is now available about
its function in brain cells and the interactions of its
3,144 amino acids with other proteins<sup>39,41</sup>. Research
on huntingtin physiology will continue to contribute
to our understanding of neuronal development and
function. It will also give us molecular cues as to how
a protein with a range of important functions becomes
detrimental for HD neurons. But the next big challenge
will be to transform the study of normal huntingtin
function into an additional avenue for the treatment
of HD.


1. Cattaneo, E. *et al.* Loss of normal huntingtin function: new
   developments in Huntington’s disease research. *Trends Neurosci.*
   **24**, 182–188 (2001).
2. Huntington’s Disease Collaborative Research Group.
   A novel gene containing a trinucleotide repeat that is
   expanded and unstable on Huntington’s disease
   chromosomes. *Cell* **72**, 971–983 (1993).
3. Reiner, A. *et al.* Differential loss of striatal projection
   neurons in Huntington disease. *Proc. Natl Acad. Sci. USA*
   **85**, 5733–5737 (1988).
4. Vonsattel, G. J. P. & DiFiglia, M. Huntington Disease.
   *J. Neuropathol. Exp. Neurol.* **57**, 369–384 (1998).
5. Harper, P. S. *Huntington’s Disease* 2nd edn
   (W. B. Saunders, London, 1996).
6. Sipione, S. & Cattaneo, E. Modeling Huntington’s
   disease in cells, flies, and mice. *Mol. Neurobiol.*
   **23**, 21–51 (2001).
7. Bates, G., Harper, P. & Jones, L. (eds) *Huntington’s
   Disease* 3rd edn (Oxford Univ. Press, Oxford, UK, 2002).
8. Snell, R. G. *et al.* Relationship between trinucleotide repeat
   expansion and phenotypic variation in Huntington’s
   disease. *Nature Genet.* **4**, 393–397 (1993).
9. Andrew, S. E. *et al.* The relationship between trinucleotide
   (CAG) repeat length and clinical features of Huntington’s
   disease. *Nature Genet.* **4**, 398–403 (1993).
10. Harper, S. Q. *et al.* RNA interference improves motor and
    neuropathological abnormalities in a Huntington’s disease
    mouse model. *Proc. Natl Acad. Sci. USA* **102**, 5820–5825
    (2005).
11. Zuccato, C., Tartari, M., Goffredo, D., Cattaneo, E. &
    Rigamonti, D. From target identification to drug screening
    assays for neurodegenerative diseases. *Pharmacol. Res.*
    **52**, 245–251 (2005).
12. Hughes, R. E. & Olson, J. M. Therapeutic opportunities in
    polyglutamine disease *Nature Med.* **7**, 419–423 (2001).
13. McMurray, C. T. Huntington’s disease: new hope for
    therapeutics. *Trends Neurosci.* **24**, S32–S38 (2001).
14. Rigamonti, D. *et al.* Wild-type huntingtin protects from
    apoptosis upstream of caspase-3. *J. Neurosci.* **20**,
    3705–3713 (2000).
15. Zuccato, C. *et al.* Loss of huntingtin-mediated BDNF gene
    transcription in Huntington’s disease. *Science* **293**,
    493–498 (2001).
    Together with reference 16, this paper shows that
    wild-type but not mutant huntingtin contributes to
    the transcription of the BDNF gene. Cortex taken
    from HD mice and patients with HD has reduced
    BDNF mRNA and protein.
16. Zuccato, C. *et al.* Huntingtin interacts with REST/NRSF to
    modulate the transcription of NRSE-controlled neuronal
    genes. *Nature Genet.* **35**, 76–83 (2003).

17. Zhang, Y. *et al.* Depletion of wild-type huntingtin in mouse
    models of neurologic diseases. *J. Neurochem.* **87**,
    101–106 (2003).
    Levels of full-length huntingtin are reduced in the
    brains of transgenic mouse models of HD, ischaemia
    and trauma, and in the spinal cord after injury. Wild-
    type huntingtin protects against neurodegeneration
    after ischaemia.
18. Leavitt, B. R., *et al.* Wild-type huntingtin protects neurons
    from excitotoxicity. *J. Neurochem.* (in the press).
19. O’Kusky, J. R., Nasir, J., Cicchetti, F., Parent, A. &
    Hayden, M. R. Neuronal degeneration in the basal ganglia
    and loss of pallido–subthalamic synapses in mice with
    targeted disruption of the Huntington’s disease gene.
    *Brain Res.* **818**, 468–479 (1999).
20. Dragatsis, I., Levine, M. S. & Zeitlin, S. Inactivation of *Hdh*
    in the brain and testis results in progressive
    neurodegeneration and sterility in mice. *Nature Genet.*
    **26**, 300–306 (2000).
    A Cre/loxP site-specific recombination strategy was
    used to inactivate *Hdh* expression in postnatal
    mouse neurons. This resulted in a progressive
    degenerative neuronal phenotype.
21. Gunawardena, S. *et al.* Disruption of axonal transport by
    loss of huntingtin or expression of pathogenic PolyQ
    proteins in *Drosophila*. *Neuron* **40**, 25–40 (2003).
    Reduction of *D. melanogaster* huntingtin or
    expression of proteins containing pathogenic polyQ
    repeats disrupts axonal transport.
22. Trushina, E. *et al.* Mutant huntingtin impairs axonal
    trafficking in mammalian neurons *in vivo* and *in vitro*.
    *Mol. Cell Biol.* **24**, 8195–8209 (2004).
    Expression of full-length mutant huntingtin or
    reduction in the level of wild-type huntingtin impairs
    vesicular and mitochondrial trafficking in
    mammalian neurons *in vitro* and *in vivo*.
23. Gauthier, L. R. *et al.* Huntingtin controls neurotrophic
    support and survival of neurons by enhancing BDNF
    vesicular transport along microtubules. *Cell* **118**,
    127–138 (2004).
    This work indicates that wild-type huntingtin
    enhances vesicular transport of BDNF along
    microtubules. BDNF transport (but not mitochondrial
    transport) is specifically attenuated in HD.
24. Ho, L. W., Brown, R., Maxwell, M., Wyattenbach, A. &
    Rubinsztejn, D. C. Wild type Huntingtin reduces the cellular
    toxicity of mutant huntingtin in mammalian cell models of
    Huntington’s disease. *J. Med. Genet.* **38**, 450–452 (2001).
25. Leavitt, B. R. *et al.* Wild-type huntingtin reduces the
    cellular toxicity of mutant huntingtin *in vivo*. *Am. J. Hum.*
    *Genet.* **68**, 313–324 (2001).


26. Sun, Y., Savanenin, A., Reddy, P. H. & Liu, Y. F.
    Polyglutamine-expanded huntingtin promotes sensitization
    of N-methyl-D-aspartate receptors via post-synaptic
    density 95. *J. Biol. Chem.* **276**, 24713–24718 (2001).
27. Van Raamsdonk, J. M. *et al.* Loss of wild-type huntingtin
    influences motor dysfunction and survival in the YAC128
    mouse model of Huntington disease. *Hum. Mol. Genet.*
    **14**, 1379–1392 (2005).
    YAC128<sup>-/-</sup> mice that do not express wild-type
    huntingtin show defects in motor coordination and
    are hypoactive. Striatal neuropathology is only mildly
    worsened, but testis atrophy and degeneration is
    markedly worsened.
28. Trottier, Y. *et al.* Polyglutamine expansion as a pathological
    epitope in Huntington’s disease and four dominant
    cerebellar ataxias. *Nature* **378**, 403–406 (1995).
29. Ferrante, R. J. *et al.* Heterogeneous topographic and
    cellular distribution of huntingtin expression in the normal
    human neostriatum. *J. Neurosci.* **17**, 3052–3063 (1997).
30. Fusco, F. R. *et al.* Cellular localization of huntingtin in
    striatal and cortical neurons in rats: lack of correlation with
    neuronal vulnerability in Huntington’s disease. *J. Neurosci.*
    **19**, 1189–1202 (1999).
31. DiFiglia, M. *et al.* Huntingtin is a cytoplasmic protein
    associated with vesicles in human and rat brain neurons.
    *Neuron* **14**, 1075–1081 (1995).
    Shows the detection of huntingtin in synaptosomes
    and vesicles, where its immunoreactivity overlaps
    with the distribution of vesicle membrane proteins.
32. Velier, J. *et al.* Wild-type and mutant huntingtins function in
    vesicle trafficking in the secretory and endocytic pathways.
    *Exp. Neurol.* **152**, 34–40 (1998).
33. Hilditch-Maguire, P. *et al.* Huntingtin: an iron-regulated
    protein essential for normal nuclear and perinuclear
    organelles. *Hum. Mol. Genet.* **22**, 2789–2797 (2000).
34. Hoffner, G., Kahlem, P. & Djian, P. Perinuclear localization
    of huntingtin as a consequence of its binding to
    microtubules through an interaction with β-tubulin:
    relevance to Huntington’s disease. *J. Cell Sci.* **115**,
    941–948 (2002).
35. Kegel, K. B. *et al.* Huntingtin is present in the nucleus,
    interacts with the transcriptional corepressor C-terminal
    binding protein, and represses transcription. *J. Biol. Chem.*
    **277**, 7466–7746 (2002).
36. Li, J. Y., Plomann, M. & Brundin, P. Huntington’s disease:
    a synaptopathy? *Trends Mol. Med.* **9**, 414–420 (2003).
    Provides details on the impairment of exocytosis,
    endocytosis and synaptic transmission in HD.
37. Everett, C. M. & Wood, N. W. Trinucleotide repeats and
    neurodegenerative disease. *Brain* **127**, 2385–2405
    (2004).

38. Perutz, M. F., Johnson, T., Suzuki, M. & Finch, J. T. Glutamine repeats as polar zippers: their possible role in inherited neurodegenerative diseases. *Proc. Natl Acad. Sci. USA* **91**, 5355–5358 (1994).

39. Harjes, P. & Wanker, E. E. The hunt for huntingtin function: interaction partners tell many different stories. *Trends Biochem. Sci.* **28**, 425–433 (2003).

40. Li, S. H. & Li, X. J. Huntingtin-protein interactions and the pathogenesis of Huntington’s disease. *Trends Genet.* **20**, 146–154 (2004).

41. Goehler, H. *et al.* A protein interaction network links GIT1, an enhancer of huntingtin aggregation, to Huntington’s disease. *Mol. Cell* **15**, 853–865 (2004). Extensive analysis of protein–protein interactions has led to the discovery of GIT1, a G-protein-coupled receptor kinase-interacting protein, which enhances huntingtin aggregation by recruitment of the protein to membranous vesicles.

42. Steffan, J. S. *et al.* SUMO modification of huntingtin and Huntington’s disease pathology. *Science* **304**, 100–104 (2004). Sumoylation of an N-terminal fragment of huntingtin reduces the ability of the protein to form aggregates, therefore exacerbating its toxicity. Aggregated huntingtin is considered less pathogenic than soluble huntingtin.

43. Andrade, M. A. & Bork, P. HEAT repeats in the Huntington’s disease protein. *Nature Genet.* **11**, 115–116 (1995).

44. Neuwald, A. F. & Hirano, T. HEAT repeats associated with condensins, cohesins, and other complexes involved in chromosome-related functions. *Genome Res.* **10**, 1445–1452 (2000).

45. MacDonald, M. E. Huntingtin: alive and well and working in middle management. *Sci. STKE*. **207**, pe48 (2003).

46. Takano, H. & Gusella, J. F. The predominantly HEAT-like motif structure of huntingtin and its association and coincident nuclear entry with dorsal, an NF-κB/Rel/ dorsal family transcription factor. *BMC Neurosci.* **3**, 15 (2002).

47. Xia, J., Lee, D. H., Taylor, J., Vandelft, M. & Truant, R. Huntingtin contains a highly conserved nuclear export signal. *Hum. Mol. Genet.* **12**, 1393–1403 (2003).

48. Cornett, J. *et al.* Expansion of huntingtin impairs its nuclear export. *Nature Genet.* **37**, 198–204 (2005).

49. Goldberg, Y. P. *et al.* Cleavage of huntingtin by apopain, a proapoptotic cysteine protease, is modulated by the polyglutamine tract. *Nature Genet.* **13**, 442–449 (1996).

50. Wellington, C. L. *et al.* Caspase cleavage of gene products associated with triplet expansion disorders generates truncated fragments containing the polyglutamine tract. *J. Biol. Chem.* **273**, 9158–9167 (1998).

51. Wellington, C. L. *et al.* Inhibiting caspase cleavage of huntingtin reduces toxicity and aggregate formation in neuronal and nonneuronal cells. *J. Biol. Chem.* **275**, 19831–19838 (2000).

52. Gafni, J. & Ellerby, L. M. Calpain activation in Huntington’s disease. *J. Neurosci.* **22**, 4842–4849 (2002).

53. Gafni, J. *et al.* Inhibition of calpain cleavage of huntingtin reduces toxicity: accumulation of calpain/caspase fragments in the nucleus. *J. Biol. Chem.* **279**, 20211–20220 (2004).

54. Davies, S. W. *et al.* Formation of neuronal intranuclear inclusions underlies the neurological dysfunction in mice transgenic for the HD mutation. *Cell* **90**, 537–548 (1997).

55. DiFiglia, M. *et al.* Aggregation of huntingtin in neuronal intranuclear inclusions and dystrophic neurites in brain. *Science* **277**, 1900–1903 (1997).

56. Kim, Y. J. *et al.* Caspase 3-cleaved N-terminal fragments of wild-type and mutant huntingtin are present in normal and Huntington’s disease brains, associate with membranes, and undergo calpain-dependent proteolysis. *Proc. Natl Acad. Sci. USA* **98**, 12784–12789 (2001).

57. Lunkes, A. *et al.* Proteases acting on mutant huntingtin generate cleaved products that differentially build up cytoplasmic and nuclear inclusions. *Mol. Cell* **10**, 259–269 (2002).

58. Wellington, C. L. *et al.* Caspase cleavage of mutant huntingtin precedes neurodegeneration in Huntington’s disease. *J. Neurosci.* **22**, 7862–7872 (2002).

59. Li, Z. *et al.* A putative Drosophila homolog of the Huntington’s disease gene. *Hum. Mol. Genet.* **8**, 1807–1815 (1999).

60. Mende-Mueller, L. M., Toneff, T., Hwang, S. R., Chesselet, M. F. & Hook, V. Y. Tissue-specific proteolysis of Huntingtin (htt) in human brain: evidence of enhanced levels of N- and C-terminal htt fragments in Huntington’s disease striatum. *J. Neurosci.* **21**, 1830–1837 (2001).

61. Ona, V. O. *et al.* Inhibition of caspase-1 slows disease progression in a mouse model of Huntington’s disease. *Nature* **399**, 263–267 (1999).

62. Kalchman, M. A. *et al.* Huntingtin is ubiquitinated and interacts with a specific ubiquitin-conjugating enzyme. *J. Biol. Chem.* **271**, 19385–19394 (1996).

63. Humbert, S. *et al.* The IGF-1/Akt pathway is neuroprotective in Huntington’s disease and involves huntingtin phosphorylation by Akt. *Dev. Cell* **2**, 831–837 (2002). Huntingtin is phosphorylated on Ser421 and this modification protects against the toxicity of polyQ-expanded huntingtin in cell culture.

64. Warby, S. C. *et al.* Huntingtin phosphorylation on serine 421 is significantly reduced in the striatum and by polyglutamine expansion *in vivo*. *Hum. Mol. Genet.* **14**, 1569–1577 (2005). Phosphorylation at Ser421 is reduced in HD mice.

65. Luo, S., Vacher, C., Davies, J. E. & Rubinsztein, D. C. Cdk5 phosphorylation of huntingtin reduces its cleavage by caspases: implications for mutant huntingtin toxicity. *J. Cell Biol.* **169**, 647–656 (2005).

66. Hackam, A. S. *et al.* Huntingtin interacting protein 1 induces apoptosis via a novel caspase-dependent death effector domain. *J. Biol. Chem.* **275**, 41299–41308 (2000).

67. Huang, K. *et al.* Huntingtin-interacting protein HIP14 is a palmitoyl transferase involved in palmitoylation and trafficking of multiple neuronal proteins. *Neuron* **44**, 977–986 (2004).

68. Marcora, E., Gowan, K. & Lee, J. E. Stimulation of NeuroD activity by huntingtin and huntingtin-associated proteins HAP1 and MLK2. *Proc. Natl Acad. Sci. USA* **100**, 9578–9583 (2003). The authors propose that huntingtin, together with HAP1, functions as a scaffold protein for the activation of the transcription factor NeuroD by mixed-lineage kinase 2 (MLK2).

69. Trettel, F. *et al.* Dominant phenotypes produced by the HD mutation in STHdhQ111 striatal cells. *Hum. Mol. Genet.* **9**, 2799–2809 (2000).

70. Ko, J., Ou, S. & Patterson, P. H. New anti-huntingtin monoclonal antibodies: implications for huntingtin conformation and its binding proteins. *Brain Res. Bull.* **56**, 319–329 (2001).

71. Nasir, J. *et al.* Targeted disruption of the Huntington’s disease gene results in embryonic lethality and behavioral and morphological changes in heterozygotes. *Cell* **81**, 811–823 (1995).

72. Duyao, M. P. *et al.* Inactivation of the mouse Huntington’s disease gene homolog Hdh. *Science* **269**, 407–410 (1995).

73. Zeitlin, S., Liu, J. P., Chapman, D. L., Papaioannou, V. E. & Efstratiadis, A. Increased apoptosis and early embryonic lethality in mice nullizygous for the Huntington’s disease gene homologue. *Nature Genet.* **11**, 155–163 (1995).

74. Dragatsis, I., Efstratiadis, A. & Zeitlin, S. Mouse mutant embryos lacking huntingtin are rescued from lethality by wild-type extraembryonic tissues. *Development* **125**, 1529–1539 (1998).

75. Haeckel, E. *Riddle of the Universe at the Close of the Nineteenth Century* (1899).

76. Wexler, N. S. *et al.* Homozygotes for Huntington’s disease. *Nature* **326**, 194–197 (1987).

77. Myers, R. H. *et al.* Homozygote for Huntington disease. *Am. J. Hum. Genet.* **45**, 615–618 (1989).

78. Gottfried, M., Lavine, L. & Roessmann, U. Neuropathological findings in Wolf–Hirschhorn (4p-) syndrome. *Acta Neuropathol. (Berl.)* **55**, 163–165 (1981).

79. Ambrose, C. M. *et al.* Structure and expression of the Huntington’s disease gene: evidence against simple inactivation due to an expanded CAG repeat. *Somat. Cell Mol. Genet.* **20**, 27–38 (1994).

80. White, J. K. *et al.* Huntingtin is required for neurogenesis and is not impaired by the Huntington’s disease CAG expansion. *Nature Genet.* **17**, 404–410 (1997).

81. Auerbach, W. *et al.* HD mutation causes progressive lethal neurological disease in mice expressing reduced levels of huntingtin. *Hum. Mol. Genet.* **10**, 2515–2523 (2001).

82. Metzler, M. *et al.* Life without huntingtin: normal differentiation into functional neurons. *J. Neurochem.* **72**, 1009–1018 (1999).

83. Metzler, M. *et al.* Huntingtin is required for normal hematopoiesis. *Hum. Mol. Genet.* **12**, 387–394 (2000).

84. Reiner, A. *et al.* Neurons lacking huntingtin differentially colonize brain and survive in chimeric mice. *J. Neurosci.* **21**, 7608–7619 (2001).

85. Reiner, A., Dragatsis, I., Zeitlin, S. & Goldowitz, D. Wild-type huntingtin plays a role in brain development and neuronal survival. *Mol. Neurobiol.* **28**, 259–276 (2003).

86. Zeron, M. M *et al.* Increased sensitivity to N-methyl-D-aspartate receptor-mediated excitotoxicity in a mouse model of Huntington’s disease. *Neuron* **33**, 849–860 (2002).

87. Rigamonti, D. *et al.* Huntingtin neuroprotective activity occurs via inhibition of pro-caspase 9 processing. *J. Biol. Chem.* **276**, 14545–14548 (2001).

88. Gervais, F. G. *et al.* Recruitment and activation of caspase-8 by the Huntingtin-interacting protein Hip-1 and a novel partner Hipp1. *Nature Cell Biol.* **4**, 95–105 (2002).

89. Rangone, H. *et al.* The serum- and glucocorticoid-induced kinase SGK inhibits mutant huntingtin-induced toxicity by phosphorylating serine 421 of huntingtin. *Eur. J. Neurosci.* **19**, 273–279 (2004).

90. Widmer, H. R. & Hefti, F. Neurotrophin-4/5 promotes survival and differentiation of rat striatal neurons developing in culture. *Eur. J. Neurosci.* **6**, 1669–1679 (1994).

91. Nakao, N., Brundin, P., Funa, K., Lindvall, O. & Odin, P. Trophic and protective actions of brain-derived neurotrophic factor on striatal DARPP-32-containing neurons *in vitro*. *Brain Res. Dev. Brain Res.* **90**, 92–101 (1995).

92. Alcantara, S. *et al.* TrkB signaling is required for postnatal survival of CNS neurons and protects hippocampal and motor neurons from axotomy-induced cell death. *J. Neurosci.* **17**, 3623–3633 (1997).

93. Ivkovic, S. & Ehrlich, M. E. Expression of the striatal DARPP-32/ARPP-21 phenotype in GABAergic neurons requires neurotrophins *in vivo* and *in vitro*. *J. Neurosci.* **19**, 5409–5419 (1999).

94. Jovanovic, J. N., Czernik, A. J., Fienberg, A. A., Greengard, P. & Sihra, T. S. Synapsins as mediators of BDNF-enhanced neurotransmitter release. *Nature Neurosci.* **3**, 323–329 (2000).

95. Fusco, F. R. *et al.* Co-localization of brain-derived neurotrophic factor (BDNF) and wild-type huntingtin in normal and quinolinic acid-lesioned rat brain. *Eur. J. Neurosci.* **18**, 1093–1102 (2003).

96. Hofer, M., Pagliusi, S. R., Hohn, A., Leibrock, J. & Barde, Y. A. Regional distribution of brain-derived neurotrophic factor mRNA in the adult mouse brain. *EMBO J.* **9**, 2459–2464 (1990).

97. Altar, C. A. *et al.* Anterograde transport of brain-derived neurotrophic factor and its role in the brain. *Nature* **389**, 856–860 (1997). BDNF is widely distributed in nerve terminals, is present in the cortex and even in the striatum. The striatum lacks BDNF mRNA; the cortex is the primary source of striatal BDNF.

98. Baquet, Z. C., Gorski, J. A. & Jones, K. R. Early striatal dendrite deficits followed by neuron loss with advanced age in the absence of anterograde cortical brain-derived neurotrophic factor. *J. Neurosci.* **24**, 4250–4258 (2004).

99. Mizuno, K, Carnahan, J. & Nawa, H. Brain-derived neurotrophic factor promotes differentiation of striatal GABAergic neurons. *Dev. Biol.* **165**, 243–256 (1994).

100. Ventimiglia, R., Mather, P. E, Jones, B. E. & Lindsay, R. M. The neurotrophins BDNF, NT-3 and NT-4/5 promote survival and morphological and biochemical differentiation of striatal neurons *in vitro*. *Eur. J. Neurosci.* **7**, 213–222 (1995).

101. Bemelmans, A. P. *et al.* Brain-derived neurotrophic factor-mediated protection of striatal neurons in an excitotoxic rat model of Huntington’s disease, as demonstrated by adenoviral gene transfer. *Hum. Gene Ther.* **10**, 2987–2997 (1999).

102. Canals, J. M. *et al.* Expression of brain-derived neurotrophic factor in cortical neurons is regulated by striatal target area. *J. Neurosci.* **21**, 117–124 (2001).

103. Perez-Navarro, E., Gavalda, N., Gratacos, E. & Alberch, J. Brain-derived neurotrophic factor prevents changes in Bcl-2 family members and caspase-3 activation induced by excitotoxicity in the striatum. *J. Neurochem.* **92**, 678–691 (2005).

104. Hodgson, J. G. *et al.* A YAC mouse model for Huntington’s disease with full-length mutant huntingtin, cytoplasmic toxicity, and selective striatal neurodegeneration. *Neuron* **23**, 181–192 (1999).

105. Metsis, M., Timmusk, T., Arenas, E. & Persson, H. Differential usage of multiple brain-derived neurotrophic factor promoters in the rat brain following neuronal activation. *Proc. Natl Acad. Sci. USA* **90**, 8802–8806 (1993).

106. Timmusk, T. *et al.* Identification of brain-derived neurotrophic factor promoter regions mediating tissue-specific, anatomy-, and neuronal activity-induced expression in transgenic mice. *J. Cell Biol.* **128**, 185–199 (1995). Analyses the activity of BDNF regulatory regions, and shows that a tissue-specific activation of the different BDNF promoter exons occurs in the brain and the periphery.

107. Pattabiraman, P. P *et al.* Neuronal activity regulates the developmental expression and subcellular localization of cortical BDNF mRNA isoforms *in vivo*. *Mol. Cell. Neurosci.* **28**, 556–570 (2005).

108. Szebenyi, G. *et al.* Neuropathogenic forms of huntingtin and androgen receptor inhibit fast axonal transport. *Neuron* **40**, 41–52 (2003).

109. Block-Galarza, J. *et al.* Fast transport and retrograde movement of huntingtin and HAP 1 in axons. *Neuroreport* **8**, 2247–2251 (1997).

110. Gunawardena, S. & Goldstein, L. S. Cargo-carrying motor vehicles on the neuronal highway: transport pathways and neurodegenerative disease. *J. Neurobiol.* **58**, 258–271 (2004).

111. Maue, R. A., Kraner, S. D., Goodman, R. H. & Mandel, G. Neuron-specific expression of the rat brain type II sodium channel gene is directed by upstream regulatory elements. *Neuron* **4**, 223–321 (1990).

112. Kraner, S. D., Chong, J. A., Tsay, H. J. & Mandel, G. Silencing the type II sodium channel gene: a model for neural-specific gene regulation. *Neuron* **9**, 37–44 (1992).

113. Mori, N., Schoenherr, C., Vandenbergh, D. J. & Anderson, D. J. A common silencer element in the SCG10 and type II Na⁺ channel genes binds a factor present in nonneuronal cells but not in neuronal cells. *Neuron* **9**, 45–54 (1992).

114. Chong, J. A. *et al.* REST: a mammalian silencer protein that restricts sodium channel gene expression to neurons. *Cell* **80**, 949–957 (1995).

115. Schoenherr, C. J. & Anderson, D. J. The neuron-restrictive silencer factor (NRSF): a coordinate repressor of multiple neuron-specific genes. *Science* **267**, 1360–1363 (1995).

116. Bruce, W. A. *et al.* Genome-wide analyses of repressor element 1 silencing transcription factors/neuron restrictive silencing factor (REST/NRSF) target genes. *Proc. Natl Acad. Sci. USA* **101**, 10458–10463 (2004). *A report on in silico and biochemical approaches that revealed the presence of 1,892 RE1/NRSE sites in the human genome.*

117. Smith, R., Brundin, P. & Li, J. Y. Synaptic dysfunction in Huntington’s disease: a new perspective. *Cell. Mol. Life Sci.* **62**, 1901–1912 (2005).

118. Sheng, M. & Kim, M. J. Postsynaptic signaling and plasticity mechanisms. *Science* **298**, 776–780 (2002).

119. Reddy, P. H. *et al.* Behavioural abnormalities and selective neuronal loss in HD transgenic mice expressing mutated full-length HD cDNA. *Nature Genet.* **20**, 198–202 (1998).

120. Wheeler, V. C. *et al.* Length-dependent gametic CAG repeat instability in the Huntington’s disease knock-in mouse. *Hum. Mol. Genet.* **8**, 115–122 (1999).

121. Squitieri, F. *et al.* Homozygosity for CAG mutation in Huntington disease is associated with a more severe clinical course. *Brain* **126**, 946–955 (2003).

122. Zuccato, C. *et al.* Progressive loss of BDNF in a mouse model of Huntington’s disease and rescue by BDNF delivery. *Pharmacol. Res.* **52**, 133–139 (2005).

123. Hermel, E. *et al.* Specific caspase interactions and amplification are involved in selective neuronal vulnerability in Huntington’s disease. *Cell Death Differ.* **11**, 424–438 (2004).

124. Luthi-Carter, R. *et al.* Dysregulation of gene expression in the R6/2 model of polyglutamine disease: parallel changes in muscle and brain. *Hum. Mol. Genet.* **11**, 1911–1926 (2002).

125. Duan, W. *et al.* Dietary restriction normalizes glucose metabolism and BDNF levels, slows disease progression, and increases survival in huntingtin mutant mice. *Proc. Natl Acad. Sci. USA* **100**, 2911–2916 (2003).

126. Gines, S. *et al.* Specific progressive cAMP reduction implicates energy deficit in presymptomatic Huntington’s disease knock-in mice. *Hum. Mol. Genet.* **12**, 497–508 (2003).

127. Ferrer, I., Goutan, E., Marin, C., Rey, M. J. & Ribalta, T. Brain-derived neurotrophic factor in Huntington disease. *Brain Res.* **866**, 257–261 (2000).

128. Spires, T. L. *et al.* Environmental enrichment rescues protein deficits in a mouse model of Huntington’s disease, indicating a possible disease mechanism. *J. Neurosci.* **24**, 2270–2276 (2004).

129. Canals, J. M. *et al.* Brain-derived neurotrophic factor regulates the onset and severity of motor dysfunction associated with enkephalinergic neuronal degeneration in Huntington’s disease. *J. Neurosci.* **24**, 7727–7739 (2004).

130. Pineda, J. R. *et al.* Brain-derived neurotrophic factor modulates dopaminergic deficits in a transgenic mouse model of Huntington’s disease. *J. Neurochem.* **93**, 1057–1068 (2005).

131. Rubinsztein, D. C. How does the Huntington’s disease mutation damage cells? *Sci. Aging Knowl. Environ.* **37**, PE26 (2003).

132. Dyer, R. B. & McMurray, C. T. Mutant protein in Huntington disease is resistant to proteolysis in affected brain. *Nature Genet.* **29**, 270–278 (2001).

133. Busch, A. *et al.* Mutant huntingtin promotes the fibrillogenesis of wild-type huntingtin: a potential mechanism for loss of huntingtin function in Huntington’s disease. *J. Biol. Chem.* **278**, 41452–41461 (2003).

134. Goffredo, D. *et al.* Calcium-dependent cleavage of endogenous wild-type huntingtin in primary cortical neurons. *J. Biol. Chem.* **277**, 39594–39598 (2002).

135. Kim, M. *et al.* Huntingtin is degraded to small fragments by calpain after ischemic injury. *Exp. Neurol.* **183**, 109–115 (2003).

136. Levine, M. S. *et al.* Enhanced sensitivity to N-methyl-D-aspartate receptor activation in transgenic and knockin mouse models of Huntington’s disease. *J. Neurosci. Res.* **58**, 515–532 (1999).

137. Cepeda, C. *et al.* Increased GABAergic function in mouse models of Huntington’s disease: reversal by BDNF. *J. Neurosci. Res.* **78**, 855–867 (2004).

138. Rosas, H. D. *et al.* Evidence for more widespread cerebral pathology in early HD: an MRI-based morphometric analysis. *Neurology* **60**, 1615–1620 (2003).

Despite the fact that most clinical symptoms of HD have been attributed to striatal degeneration, patients with HD have a significant reduction in the volume of almost all brain structures.

139. Pauly, P. C. & Harris, D. A. Copper stimulates endocytosis of the prion protein. *J. Biol. Chem.* **273**, 33107–33110 (1998).

140. Watt, N. T. & Hooper, N. M. The prion protein and neuronal zinc homeostasis. *Trends Biochem. Sci.* **28**, 406–410 (2003).

141. Brown, D. R. Copper and prion disease. *Brain Res. Bull.* **55**, 165–173 (2001).

142. Mouillet-Richard, S. *et al.* Signal transduction through prion protein. *Science* **289**, 1925–1928 (2000).

143. Chiarini, L. B. *et al.* Cellular prion protein transduces neuroprotective signals. *EMBO J.* **21**, 3317–3326 (2002).

144. Collinge, J. *et al.* Prion protein is necessary for normal synaptic function. *Nature* **370**, 295–297 (1994).

145. Mallucci, G. R. *et al.* Post-natal knockout of prion protein alters hippocampal CA1 properties, but does not result in neurodegeneration. *EMBO J.* **21**, 202–210 (2002).

146. Graner, E. *et al.* Cellular prion protein binds laminin and mediates neuritogenesis. *Brain Res. Mol. Brain Res.* **76**, 85–92 (2000).

147. Roucou, X. *et al.* Cellular prion protein inhibits proapoptotic Bax conformational change in human neurons and in breast carcinoma MCF-7 cells. *Cell Death Differ.* **12**, 783–795 (2005).

148. Lee, K. S. *et al.* Towards cellular receptors for prions. *Rev. Med. Virol.* **13**, 399–408 (2003).

149. Zalfa, F. *et al.* The fragile X syndrome protein FMRP associates with BC1 RNA and regulates the translation of specific mRNAs at synapses. *Cell* **112**, 317–327 (2003).

150. Baxendale, S. *et al.* Comparative sequence analysis of the human and pufferfish Huntington’s disease genes. *Nature Genet.* **10**, 67–76 (1995).

151. Holbert, S. *et al.* Cdc42-interacting protein 4 binds to huntingtin: neuropathologic and biological evidence for a role in Huntington’s disease. *Proc. Natl Acad. Sci. USA* **100**, 2712–2717 (2003).

152. Kauffman, J. S., Zinovyeva, A., Yagi, K., Makabe, K. W. & Raff, R. A. Neural expression of the Huntington’s disease gene as a chordate evolutionary novelty. *J. Exp. Zool. B Mol. Dev. Evol.* **297**, 57–64 (2003).

153. Pecheux, C., Gall, A. L., Kaplan, J. C. & Dode, C. Sequence analysis of the CAG triplet repeats region in the Huntington disease gene (IT15) in several mammalian species. *Ann. Genet.* **39**, 81–86 (1996).

154. Karlovich, C. A., John, R. M., Ramirez, L., Stainier, D. Y. & Myers, R. M. Characterization of the Huntington’s disease (HD) gene homologue in the zebrafish *Danio rerio*. *Gene* **217**, 117–125 (1998).

### Acknowledgements

The work described in this review has been supported by grants from the Huntington’s Disease Society of America (HDSA; USA), the Hereditary Disease Foundation (HDF; USA), HighQ Foundation (USA), Telethon (Italy), the Cariplo Foundation (Italy), Ministero dell’Istruzione dell’Università e della Ricerca (Italy) and the European Commission Framework VI Programme (NeuroNE). E.C. is member and coordinator of the ‘Huntingtin Function’ HDSA Coalition For the Cure Team. We apologize for the omission of a number of significant papers that could not be cited because of space limitations.

### Competing interests statement

The authors declare no competing financial interests.

### Online links

#### DATABASES

The following terms in this article are linked online to:

- Entrez Gene: http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gene
  - BDNF | HAP1 | HIP1 | HIP14 | HIPPI | huntingtin | PSD-95 | REST | TPR
- Entrez Nucleotide: http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=Nucleotide
  - BDNF
- OMIM: http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=OMIM
  - Creutzfeldt-Jakob disease | Fragile X mental retardation | Huntington’s disease | Wolf-Hirschhorn syndrome

#### FURTHER INFORMATION

Cattaneo’s lab: http://users.unimi.it/~spharm/cattaneo/

Access to this interactive links box is free online.
