
Bile salt biotransformations by human intestinal bacteria

Jason M. Ridlon, Dae-Joong Kang, and Phillip B. Hylemon<sup>1</sup>

Department of Microbiology/Immunology, Medical College of Virginia, Virginia Commonwealth University, Richmond, VA

Abstract   Secondary bile acids, produced solely by intestinal bacteria, can accumulate to high levels in the enterohepatic circulation of some individuals and may contribute to the pathogenesis of colon cancer, gallstones, and other gastrointestinal (GI) diseases. Bile salt hydrolysis and hydroxy group dehydrogenation reactions are carried out by a broad spectrum of intestinal anaerobic bacteria, whereas bile acid 7-dehydroxylation appears restricted to a limited number of intestinal anaerobes representing a small fraction of the total colonic flora. Microbial enzymes modifying bile salts differ between species with respect to pH optima, enzyme kinetics, substrate specificity, cellular location, and possibly physiological function. Crystallization, site-directed mutagenesis, and comparisons of protein secondary structure have provided insight into the mechanisms of several bile acid-biotransforming enzymatic reactions. Molecular cloning of genes encoding bile salt-modifying enzymes has facilitated the understanding of the genetic organization of these pathways and is a means of developing probes for the detection of bile salt-modifying bacteria. The potential exists for altering the bile acid pool by targeting key enzymes in the 7α/β-dehydroxylation pathway through the development of pharmaceuticals or sequestering bile acids biologically in probiotic bacteria, which may result in their effective removal from the host after excretion.

Ridlon, J. M., D.-J. Kang, and P. B. Hylemon. Bile salt biotransformations by human intestinal bacteria. *J. Lipid Res.* 2006. 47: 241–259.

Supplementary key words bile acids • deoxycholic acid • 7α-dehydroxylation • gallstone disease • colon cancer • bile salt hydrolase • probiotics • hydroxysteroid dehydrogenase • hydrogen sulfide

The human large intestine harbors a complex microbial flora (1). Bacterial density in the human colon is among the highest found in nature, approaching \(10^{12}\) bacteria/g wet weight of feces (2, 3). In contrast, the host suppresses significant bacterial colonization of the small intestine by a variety of mechanisms, including rapid transit times, antimicrobial peptides, proteolytic enzymes, and bile (4). Failure of these mechanisms leads to bacterial overgrowth

of the small intestine, resulting in malabsorption as bacteria compete with the host for nutrients. Under normal conditions, bacterial fermentation in the colon represents an important salvage mechanism. Complex carbohydrates, which are intrinsically indigestible or which escape digestion and absorption in the proximal gut, are fermented by colonic bacteria to yield short-chain fatty acids. It has been estimated that these short-chain fatty acids constitute 3–9% of our daily caloric intake (4). Colonic bacteria also contribute to the salvage of bile salts that escape active transport in the distal ileum. The major bile salt modifications in the human large intestine include deconjugation, oxidation of hydroxy groups at C-3, C-7, and C-12, and 7α/β-dehydroxylation (Fig. 1). Deconjugation and 7α/β-dehydroxylation of bile salts increases their hydrophobicity and their \(P_{\mathrm{ka}}\), thereby permitting their recovery via passive absorption across the colonic epithelium. However, the increased hydrophobicity of the transformed bile salts also is associated with increased toxic and metabolic effects. High concentrations of secondary bile acids in feces, blood, and bile have been linked to the pathogenesis of cholesterol gallstone disease and colon cancer (5). We present here a current review of the microbiology of bile acid metabolism in the human GI tract, focusing on understanding the biochemical mechanisms and physiological consequences of such metabolism on both the bacterium and the human host.

THE ENTEROHEPATIC CIRCULATION OF BILE ACIDS

Bile acids are saturated, hydroxylated C-24 cyclopentanephenanthrene sterols synthesized from cholesterol in hepatocytes. The two primary bile acids synthesized in the

---

Abbreviations: bai, bile acid-inducible; BSH, bile salt hydrolase; CA, cholic acid; CBAH-1, conjugated bile acid hydrolase from *C. perfringens*; CDCA, chenodeoxycholic acid; DCA, deoxycholic acid; GDCA, glycodeoxycholate; HSDH, hydroxysteroid dehydrogenase; LCA, lithocholic acid; TDCA, taurodeoxycholate; UDCA, ursodeoxycholic acid.

<sup>1</sup>To whom correspondence should be addressed.  
e-mail: hylemon@hsc.vcu.edu

Manuscript received 11 October 2005 and in revised form 17 November 2005.  
Published, *JLR Papers in Press*, November 18, 2005.  
DOI 10.1194/jlr.R500013-JLR200

Copyright © 2006 by the American Society for Biochemistry and Molecular Biology, Inc.  
This article is available online at http://www.jlr.org

Fig. 1. Bacterial bile salt-biotransforming reactions in the human intestinal tract. Hydroxy group carbons of cholate are numbered and the AB rings are identified. The 3, 7, and 12 carbons of cholic acid (CA) are numbered. Nomenclature is that of Hofmann et al. (160). BSH, bile salt hydrolase; HSDH, hydroxysteroid dehydrogenase.

human liver are cholic acid (CA; 3α,7α,12α-trihydroxy-5β-cholan-24-oic acid) and chenodeoxycholic acid (CDCA; 3α,7α-dihydroxy-5β-cholan-24-oic acid). Bile acids are further metabolized by the liver via conjugation (N-acyl amidation) to glycine or taurine, a modification that decreases the Pka to ~5. Thus, at physiological pH, conjugated bile acids are almost fully ionized and may be termed bile salts (6). Bile salts are secreted actively across the canalicular membrane and are carried in bile to the gallbladder, where they are concentrated during the interdigestive period. After a meal, release of cholecystokinin from the duodenum stimulates the gallbladder to contract, causing bile to flow into the duodenum (7). Bile salts are highly effective detergents that promote solubilization, digestion, and absorption of dietary lipids and lipid-soluble vitamins throughout the small intestine. High concentrations of bile salts are maintained in the duodenum, jejunum, and proximal ileum, where fat digestion and absorption take place. Bile salts are then absorbed through high-affinity active transport in the distal ileum (6). Upon entering the bloodstream, bile salts are complexed to plasma proteins and returned to the liver. Upon reaching the liver, they are cleared efficiently from the circulation by active transporters on the sinusoidal membrane of hepatocytes and rapidly secreted into bile. This process is known as the enterohepatic circulation. Figure 2 depicts the enterohepatic circulation in the context of the gastrointestinal anatomy and also indicates the relative numbers and genera of the predominant bacteria inhabiting each section of the GI tract.

During the enterohepatic circulation, bile salts encounter populations of facultative and anaerobic bacteria of relatively low numbers and diversity in the small bowel. Bile salt metabolism by small bowel microbes consists mainly of deconjugation and hydroxy group oxidation. Ileal bile salt transport is highly efficient (~95%), but approximately 400–800 mg of bile salts escapes the enterohepatic circulation daily and becomes substrate for

Small Intestine

A: Duodenum (25cm)
pH 5.7~6.4
~10³ bacteria/ml
Lactobacillus
Streptococcus

B: Jejunum (1.0m)
pH 5.9~6.8
~10⁴ bacteria/ml
Lactobacillus
Streptococcus Staphylococcus
Veillonella

C: Ileum (2.0m)
pH 7.3~7.7
10⁶~10⁸ bacteria/ml
Enterobacteria
Enterococcus
Bacteroides
Clostridium
Lactobacillus
Veillonella

Large Intestine

D. Cecum/Colon (150cm)
pH 5.7-6.8
~10¹¹ bacteria/g
Bacteroides, Eubacterium, Bifidobacterium,
Ruminococcus, Peptostreptococcus,
Propionibacterium, Clostridium,
Lactobacillus, Escherichia, Streptococcus,
Methanobrevibacter

Rectum
pH 6.6~6.8
(300-600mg/day)

Fig. 2. Anatomy, physiology, and microbiology of the gastrointestinal tract. Large arrows denote the enterohepatic circulation of bile acids, which begins with contraction of the gallbladder, releasing bile into the duodenum. Small arrows denote the passive absorption of bile acids that escape active transport. * Secondary bile acids produced by 7α-dehydroxylation are passively absorbed in the large intestine and returned to the liver (see Fig 3). The genera of predominant bacteria isolated from each region of the lower gastrointestinal tract are listed.

significant microbial biotransforming reactions in the large bowel (6). Comparison of bile acid composition in the gallbladder and feces illustrates the extent of microbial bile acid metabolism in the large intestine (Fig. 3). The secondary bile acids deoxycholic acid (DCA; 3α,12α-dihydroxy-5β-cholan-24-oic acid) and lithocholic acid (LCA; 3α-hydroxy-5β-cholan-24-oic acid) are produced solely by microbial biotransforming reactions in the human large intestine. DCA accumulates in the bile acid pool (LCA to a much lesser extent) as a result of passive absorption through the colonic mucosa and the inability of the human liver to 7α-hydroxylate DCA and LCA to their respective primary bile acids. LCA is sulfated in the human liver at the 3-hydroxy position, conjugated at C-24, and excreted back into bile (6). The resultant bile acid sulfate is poorly reabsorbed from the gut. Even though 3-sulfo-LCA/LCA is lost in feces and does not normally accumulate in the enterohepatic circulation (8).

DECONJUGATION OF BILE SALTS

Characteristics of bile salt hydrolase(s)

Deconjugation refers to the enzymatic hydrolysis of the C-24 N-acyl amide bond linking bile acids to their amino acid conjugates. This reaction is substrate-limiting and goes to completion in the large bowel. Bile salt hydrolases (BSHs) are in the cholyglycine hydrolase family (EC 3.5.1.24) and have been isolated and/or characterized from several species of intestinal bacteria (Table 1). The importance of the position, charge, shape, and chirality of various analogs of taurine/glycine conjugates on the rate of hydrolysis by BSHs has also been investigated (9). BSHs differ in subunit size and composition, pH optimum,

Bile salt biotransformations by human intestinal bacteria 243

of a lack of conservation observed in residues making up the substrate binding pocket of the conjugated bile acid hydrolase gene product of *C. perfringens* (CBAH-1) and the corresponding residues predicted in amino acid multiple sequence alignment with other BSHs (Fig. 4). The sterol moiety is bound primarily through hydrophobic interactions in the CBAH-1 (residues highlighted in gray in Fig. 4) as well as hydrogen bonds to the carboxylate group. Although the crystal structure of CBAH-1 did not reveal specific recognition of the taurine/glycine moiety, kinetic data from several BSHs suggest that the conjugates are important in substrate specificity (Table 1). Therefore, additional crystallization and site-directed mutagenesis (preferably with mutagenesis of Cys2) of BSHs from different species will be helpful in explaining the kinetic observations of substrate specificity.

## Distribution, genetic organization, and regulation of BSH

Genes encoding BSHs have been cloned from *C. perfringens* (15), *Lactobacillus plantarum* (16), *La. johnsonii* (12, 17), *Bi. longum* (10), *Bi. bifidum* (18), *Bi. adolescentis* (19), and *Listeria monocytogenes* (20, 21). Homologs and putative *bsh* genes have also been identified recently through microbial genome analysis. The organization and regulation of genes encoding BSH differ between species and genera. Monocistronic BSH genes have been reported in *La. plantarum* (16), *La. johnsonii* (12), *Li. monocytogenes* (21), and *Bi. bifidum* (18). A gene encoding BSH (CBAH-1) cloned from *C. perfringens* (15) differed significantly in size and amino acid sequence from a BSH purified from a different strain of *C. perfringens* (13). The inactivation of the gene encoding CBAH-1 resulted in only partial reduction in BSH activity (BSH activity was 86% of that in the wild type), suggesting multiple BSH genes in *C. perfringens*. Furthermore, the crystal structure showed that the enzyme encoded by the CBAH-1 gene forms an active homotetramer (11). These observations, coupled with the detection of both intracellular and extracellular BSHs, provide further evidence for multiple isoforms, although the organization and regulation of the *bsh* gene(s) from *C. perfringens* are not known at present (22). Polycistronic operons encoding three genes involved in bile salt deconjugation (*cbsT1*, *cbsT2*, and *cbsHβ*) have been characterized in *La. johnsonii* and *La. acidophilus* (12). Genes *cbsT1* and *cbsT2* appear to be gene duplications that encode taurocholate/CA antiport proteins of the major facilitator superfamily, whereas *cbsHβ* encodes the BSH β-isoform (23). In addition, an uncharacterized extracellular factor has been detected in *La. johnsonii* 100-100, which stimulates BSH activity and uptake of conjugated bile salts during the stationary growth phase (12, 24). BSH expression is also growth phase-dependent. Stationary phase expression has been reported in *Bacteroides fragilis* (25), and exponential phase expression was reported for *Bi. longum* (10).

## Benefits of BSHs to the bacterium

BSHs appear to enhance the bacterial colonization of the lower gastrointestinal tract of higher mammals.

TABLE 1. Characteristics of BSHs from human intestinal bacteria

| Organism | Native Molecular Mass (kDa) | Subunit Molecular Mass (mM) | TCA | TCDC A | TDCA | GCA | GCD CA | GDCA | pH Optimum | Reference(s) |
|----------|---------------------------|----------------------------|-----|--------|------|------|--------|-------|-------------|---------------|
| Bacteroides fragilis | 250 | 32.5 | 0.45 | 0.29 | 0.17 | 0.35 | 0.26 | 0.2 | 4.2-4.5 | 25 |
| Bacteroides vulgatus | 140 | 36 | + | + | + | - | - | - | ND | 163 |
| Clostridium perfringens MCV 185 | 250 | 56 | 37 | 3 | 3.5 | 3.6 | 14 | 1.2 | 5.8-6.4 | 13 |
| Clostridium perfringens 13 | 147 | 36.1 | + | ND | + | + | ND | ND | 4.5-5.5 | 15 |
| Lactobacillus johnsonii 100-100 | 42(α), 38(β) |  |  |  |  |  |  |  |  | 12, 17, 23 |
| Isozyme A | 115 | 42 | 0.76 | + | + | + | ND | + | 4.2-4.5 |  |
| Isozyme B | 105 | 42, 38 | 0.95 | + | + | + | ND | + | 4.2-4.5 |  |
| Isozyme C | 95 | 42, 38 | 0.45 | ND | ND | ND | ND | ND | 4.2-4.5 |  |
| Isozyme D | 80 | 38 | 0.37 | ND | ND | ND | ND | ND | 4.2-4.5 |  |
| Lactobacillus plantarum 80 | ND | 37.1* | TR | TR | TR | + | + | + | 4.7-5.5 | 28 |
| Lactobacillus acidophilus | ND | 35.0* | + | ND | ND | + | ND | ND | ND | 12, 27 |
| Bifidobacterium longum BB 536 | 250 | 40 | 0.875 | 1.61 | 0.516 | 1.33 | 1.4 | 1.37 | 5.5-6.5 | 14 |
| Bifidobacterium longum SBT 2928 | 125-130 | 35 | 1.12 | 0.33 | 0.79 | 0.16 | 0.13 | 0.28 | 5.0-7.0 | 10 |
| Bifidobacterium bifidum ATCC 11863 | 150 | 35 | + | + | + | ++ | ++ | ++ | ND | 18 |
| Bifidobacterium adolescentis | ND | 35* | ND | ND | ND | ND | ND | ND | ND | 19 |
| Listeria monocytogenes | ND | 36.8* | ND | ND | ND | ND | ND | + | ND | 20, 164 |

BSH, bile salt hydrolase; GCA, glycocholate; GCDCA, glycodeoxycholate; GDCA, glycodeoxycholate; ND, not determined; TCA, taurocholate; TCDC A, taurochenodeoxycholate; TDCA, taurodeoxycholate; TR, trace of activity; +, activity detected; –, no activity detected.

“Value derived from the Protparam program (http://www.expasy.ch/tools/protparam.html) using the deduced amino acid sequence.

Cl. perfringens
Li. monocytogenes
En. faecalis
La. plantarum
La. gasseri
La. johnsonii (α)
Bi. longum
Bi. adolescentis
Bi. bifidum
La. johnsonii (β)
La. acidophilus
Ba. sphaericus

M--CTGLALETKDGLHLFGRNM DIEYSFNQSIIFIPRNFKCVNKS-NKKELTTKYAVLGM 57
M--CTSITYTTK--DHYFGRNFDEYLSYKEVVVTPKNYPHFRK--VEDIEKHyalIGI 54
M--CTAITYVSK--DHYFGRNFDEISYNEVVTITPRNYKFSFRE--VGNLDHHFAIIGI 54
M--CTAITYQSY--NNYFGRNFDEISYNEMVTITPRKYLVFRK--VENLDHHYAIIGI 54
M--CTSILYSPK--DHYFGRNLDEIAYGQKVITPKNYEFEFTD--LPAEKSHYAMIGV 54
M--CTSIVYSSNN-HHYFGRNLDEISEGEHPVITPRNYEFQYRK--LPNKKAKYAMVGM 55
M--CTGVRFSDDEGNTYFGRNLDWSFSYGETILVTPRGHYDTVFG-AGGKAkPNAIGV 57
M--CTGVRFSDEEGNMYFGRNLDWSFSYGESILATPRGYHYDNVFG-AERKATPNAIGV 57
M--CTGVRFSDEEGNMYFGRNLDWSFSYGETILVTPRGYQDYEEYG-AEGKSEPNAlGV 57
M--CTGLRFTDDQGNLYFGRNLDVGQDYEGVIITPRNYPLPYKF--LDNTTTKKAVIGM 56
M--CTGLRFTDDQGNLYFGRNLDVGQDYEGVIITPRNYPLPYKF--LDNTTTKKAVIGM 56
MLGCSSLsIRTDDKSLFARTMDFTMEEDSKVIIVPRYGIRLLEKENVVINNSYAFVGM 60

Cl. perfringens
Li. monocytogenes
En. faecalis
La. plantarum
La. gasseri
La. johnsonii (α)
Bi. longum
Bi. adolescentis
Bi. bifidum
La. johnsonii (β)
La. acidophilus
Ba. sphaericus

GTIFDDYPTFADGMNEKGLGCAgLNFPVYVSYSKEDIEGKTNI PVNFLWLVLANFSSVE 117
AAVMENPLYDYDATNEKGLSMAGLNPSGNADYKDFAEg-KDNVTPFEFIPWLGQCATVK 113
AAGIADYPLYDAINEKGLGMAGLNPSGYADYKKIEEG-KENVSPFEFIPWLGQCSTVD 113
TADVESPLYDYDAMNEKGLCIAgLNFPAGYADYKKYDAD-KVNITPFELIPWLLGQFSSVR 113
AAVADNTPLYCDAINEKGLGVAGLSFPAGQGKYFPNAAD-KKNIASFEFISYLLATYETVD 113
AIVEBNPLYFDA SNEEGLGIAgLNFDGPCHYFPENAE-KNNVTPFELIPYLLSQCTTVA 114
GVVMADRPMYFDCANEHGLAIAgLNFPGYASFVHEPVEGTENVATFEFLWVARNFDSVD 117
GVVMADRPMYFDCANEHGLAIAgLNFPGYAEFVHEPVEGTDNVATFEFLWVARNFDSVD 117
GVVMTDRPMYFDCANEHGLAIAgLNFPGYASFAHEPVEGTENVATFEFLWVARNFDSVD 117
GIVVDGYPSYFDCYNE DGLGIAgLNFPHFAKFSDGPIDGKINLASYEIMLWVTQNFTHS 116
GIVVDGYPSYFDCFNE DGLGIAgLNFPHFAKFSDGPIDGKINLASYEIMLWVTQNFTKVS 116
GSTDITS PVLYDGVNEKGLMGAMLYYATFATYADEPKKGT TGINPVYVISQVLGNCVTVD 120

Cl. perfringens
Li. monocytogenes
En. faecalis
La. plantarum
La. gasseri
La. johnsonii (α)
Bi. longum
Bi. adolescentis
Bi. bifidum
La. johnsonii (β)
La. acidophilus
Ba. sphaericus

EVKEALKNANIVDIPSENIPNTTHWMISDITGKSIVVEQTKE-KLNVF DNNIGVLTS 176
EARRLLQRINLVNISESNLPLSPDHWLMADQN-ESIVVEC VKD-GLHIYDNPVGVLTN 171
EAKKL LKNLN LVNISEDELPLSPDHWLLADKE-QSIVVESTKE-GLRVFDNPVGVLTN 171
EVKKNIQKLN LN VINESEQ LPLSPDHWL VADKQ-ESIVIESVKE-GLKIYDNPVGVLTN 171
QVKESLTNANISNSFAKNTPASELHWLVGDKTGKSIVVESDEK-GLHVYNNPVNALTA 172
EVKDALKDVSLVNINSEKLPLSPDHWLMADKTGESIVVESTLS-GLHVYDNPVHVLTN 173
EVEEALRNVTLSQIVP-GQQESLLHWFIGDK-RSIVVEQMAD-GMHVHHDDVDVLTNQ 174
EVEEALKNVTLVSQIVP-GQQESLLHWIIGDSE-RSIVVEQMAD-GMHVHHDDVDVLTNQ 174
EVEEALKNVTLVSQVVP-GQQESLLHWFIGDT-RSIVVEQMAD-GMHVHHDDVDVLTNQ 174
EVKEALKNVNLVNEAINTSFAVAPLHWIISDSD-EAIIVEVSKQYGMKVFDDKVGVLTS 175
DVKEALKNVNLVNEAINS SFAVAPLHWIISDKD-EAIIVEISKQYGMKVFDDRLGVLTNS 175
DVIEK LTSYT LLNEANII LGFAPPHYTF T DASGESIVIEPDKT-GITIHRKTIGVMTNS 179

Cl. perfringens
Li. monocytogenes
En. faecalis
La. plantarum
La. gasseri
La. johnsonii (α)
Bi. longum
Bi. adolescentis
Bi. bifidum
La. johnsonii (β)
La. acidophilus
Ba. sphaericus

PTFDWHVANLNQYVGLRYNQVPEFKLGQSQLTALGQGTGLVGLPGDFTPASRFIRVAFLR 236
PTFDYQLFN LNNYRVLSSETPEN NFSNEIDLDA YSRGMGGIGLPGDLSSMSRFVKATFTK 231
PTFDYQLFN LNNYRVLSTRPKNNFSDQIELDIYSRGMGGIGLPGDLSSVSRFVKATFTK 231
PNFDYQLFN LNNYRALNSNTPQN SFSEKVDLDS YSRGMGGLGLPGDLSSMSRFVRAFTK 231
PLFP EQLTNLAN YASVVPGE PDNNFLPGVNLKLYRS LGTHHLP GGMDES RFVKVC FAL 232
PEFP GQLRN LANYS NIA PAQKNTLVPGVDLNLYSRGLGTHFLPGGMDSASRFVKIAFVR 233
PTFD FH MENLRNYMCVSNE MAEPTSWGKASLTA WAGAGVMGHIGPDVSSPSRFVRVAYTN 234
PTFG FH MENLRNYMCVSGNE MAEPATWGKASLSAW GAGVSMHGIPGDVSSPSRFVRVAYAN 234
PTFD FH MENLRNYMCVSNE MAEPTTWGKAELS AWGAGVSMHGIPGDVSSPSRFVRVAYTN 234
PDFN WHLTNLG NYTGLNPHDATAQS WNGQKVAPWGVGT GSLGLPGDSIPADRFVKAAYLN 235
PDFN WHLTNLG NYTGLDPHDATAQS WNGQKVAPWGVGT GSLGLPGDSIPADRFVKAAYLN 235
PGYEWHQTNL RAYIGVT PNPPQDIMMGDLDLT PFGQGAGGLGLPGDFTPSARFLRVAYWK 239

Cl. perfringens
Li. monocytogenes
En. faecalis
La. plantarum
La. gasseri
La. johnsonii (α)
Bi. longum
Bi. adolescentis
Bi. bifidum
La. johnsonii (β)
La. acidophilus
Ba. sphaericus

DAMIKNDKDSIDLEFFHILNNVAMVRGSTRTVEEKSDLTQYTSCMCEKGIIYYNTYEN 296
LNSVSGDSESESIGQFFHILGSVEQQKGLCDVGGEKYEH TIYSSCCNIDKGIIYYRTYGN 291
LNSVSR SSEYESISQFFHILSSVEQQKGLCDVGDEKYEH TIYSSCCNLEKGIIYYRTYDN 291
LNSLSMQTESGSVSQFFHILGSVEQQKGLCEVTDGKYEH TIYSSCCDMDKGVYYYRTYDN 291
NHAPKDSDEVNVTNFFHILESVEQAKGMDQIGPNSFEYTMYSCMNLEKGILYFNCYDD 292
AHSPQGNNELSSVTNYFHILHSVEQP KGTDEVGPNSYEYTIYSDGTNLETGTFYYTNYEN 293
AHYPQ QNDEAANVSR LFHTLGSVQMVDGMAKMGDGFERTLFTSGYSSKTNTYYMNTYDD 294
THYPQQEGEAANVSRLFHTLGSVQMVDGMAKMSNQFER TLFTSGYSSKTNTYYMNTYDD 294
THYPQQNNEAANVSRLFHTLVSVQMVDGMSKMNGQFER TLFTSGYSGKTNTYYMNTYED 294
ANYPTVKGEKANVAKFFNILKSVAMIKGSVNDQGSD EYTVTACYSSGSKTYYCNFEDD 295
VNYPTVKGKKANVAKFFNILKSVAMIKGSVNNKQGSNEYTVTACYSAATKTYYCNEEND 295
KYTEKAKNETEGVTNLFHILSSVNIPKGVVLTNEGKTDYTIYTSAMCAQSKNYYFKLYDN 299

Cl. perfringens
Li. monocytogenes
En. faecalis
La. plantarum
La. gasseri
La. johnsonii (α)
Bi. longum
Bi. adolescentis
Bi. bifidum
La. johnsonii (β)
La. acidophilus
Ba. sphaericus

NQINAIDMNKENLDGNEIKTYKYNKTLSINHVN-------- 329
SQITGVDMHQEDLESKELAIYPLVNEQQLNIVNK-------- 325
SQITAVDMNKENLEKDSLIVYPMVETQQINYAN-------- 324
SQINSVSLNHEHLDTTELISYPLRSEAQYAVN-------- 324
SRISAVDMNKELDSSDLVVYDLFKQDISFIN-------- 325
NQINAIELNKENLNGDELTDYKLI EQTINYQN-------- 326
PAIRSYAMADYDMDSSELISVAR------------------- 317
PAIRSYAMADFDMDSSELITAA------------------- 316
PAIRSFAMSDFDMDSSELITAD------------------- 316
FELKTYKLDDHTMNSTSLVTY------------------- 316
FELKTYKLDDETMNADKLITY------------------- 316
SRISAVSLMAENLNSQDLITFEWDRKQDIKQLNQNVMS 338

Fig. 4. Multiple sequence alignment of cholyglycine hydrolases. Protein sequences were obtained from the National Center for Biotechnology Information (http://www.ncbi.nlm.nih.gov/). Alignments were made with the ClustalW program (http://www.ebi.ac.uk/clustalw/) using the GONNET 250 matrix. Residues highlighted in yellow are predicted active site amino acids based on the crystal structure of the BSH from *C. perfringens* (11) as well as on site-directed mutagenesis and biochemical data (10, 12–14). Residues highlighted in gray correspond to residues involved in substrate binding in the BSH from *C. perfringens* (11). The secondary structural elements, which are based on the conjugated bile acid hydrolase from *C. perfringens* (CBAH-1) crystal structure, are shown above the alignment. The α and β designations of *Lactobacillus johnsonii* refer to the two isoforms of the genes found in this bacterium.
with a wide range of substrate specificities, genetic regulation, and the occurrence of multiple isoforms in certain strains have created conflicting reports regarding the physiological benefit to the bacterium in hydrolyzing bile acid conjugates. Determining the mechanism(s) by which BSHs aid bacteria in the colonization of the mammalian intestine will be of great interest, especially with regard to bacterial pathogenesis.

### Taurine, hydrogen sulfide production, and colon cancer

The bile acid conjugates glycine and taurine serve as substrates in microbial metabolism. Unlike glycine, taurine contains a sulfonic acid moiety that is reduced and dissimilated to hydrogen sulfide after deconjugation (38, 39). Hydrogen sulfide is highly toxic and has been shown to increase colonocyte turnover (40). Activation and upregulation of the extracellular signal-regulated kinase 1 and 2 (ERK1/2) signaling pathway has been suggested as a possible mechanism for sulfide-induced colonocyte proliferation (41). Hydrogen sulfide also inhibits butyrate metabolism in colonocytes, a key nutrient and regulator of cell turnover in the gut (40). Levitt et al. (42) demonstrated that colonocytes have evolved a highly efficient mechanism to detoxify volatile reduced sulfides through oxidation to thiosulfate. Defects in this detoxification system are suggested to play a role in the pathogenesis of ulcerative colitis, a known risk factor for colon cancer (42, 43). Recently, sulfide was implicated in preventing apoptosis in the adenocarcinoma cell line HCT116 after exposure of cells to β-phenylethyl isothiocyanate, a phytochemical found in cruciferous vegetables, which has been shown to prevent colon carcinogenesis (44, 45).

A diet high in meat has been shown to significantly increase both the levels of taurine conjugation to bile acids (46, 47) and the production of hydrogen sulfide in the colon (48). A relationship exists between the generation of hydrogen sulfide in the colon and chronic GI illness, such as inflammatory bowel disease and colon cancer (5, 49). Populations such as native black Africans with low incidence of colon cancer consume low-meat diets (50). Native black Africans also have low ratios of taurine to glycine conjugation (1:9) and low hydrogen sulfide production compared with populations consuming a “Western diet” (46, 47). In human fecal slurries obtained from individuals consuming a Western diet, taurine addition generated some of the highest sulfide levels of any organic or inorganic sulfur source added (43). Taurine addition to a coculture of a species of bacteroides and an unidentified 7α-dehydroxylating bacterium resulted in significant sulfide production, which stimulated increased rates of DCA production (34).

Although the extent to which taurine metabolism contributes to total colonic sulfide production has yet to be established, several key points have been made: 1) the extent of taurine conjugation in the bile acid pool is largely affected by diet; 2) the same dietary factors that increase taurine conjugation are hypothesized to increase colon cancer risk; 3) taurine metabolism by intestinal bacteria results in hydrogen sulfide generation; 4) sulfide generation is linked to the carcinogenesis process through enhanced cell proliferation, inhibition of butyrate metabolism, and activation of cell signaling pathways; and 5) sulfide generation may enhance DCA formation in the gut through stimulation of the microbial bile acid 7α-dehydroxylation pathway.

---

### MICROBIAL BILE ACID HYDROXYSTEROID DEHYDROGENASE(S)

#### Oxidation and epimerization

Oxidation and epimerization of the 3-, 7-, and 12-hydroxy groups of bile acids in the GI tract are carried out by hydroxysteroid dehydrogenase (HSDH) expressed by intestinal bacteria (Fig. 1). Epimerization of bile acid hydroxy groups is the reversible change in stereochemistry from α to β configuration (or vice versa) with the generation of a stable oxo-bile acid intermediate. Epimerization requires the concerted effort of two position-specific, stereochemically distinct HSDHs of intraspecies or interspecies origin. For example, the presence of both 7α- and 7β-HSDH in *C. absonum* allows epimerization by a single bacterium (51), whereas epimerization also can be achieved in cocultures of intestinal bacteria, one possessing 7α-HSDH and the other 7β-HSDH (52, 53).

The extent of the reversible oxidation and reduction of bile acid hydroxy groups by HSDH depends in part on the redox potential of the environment. Addition of oxygen to the culture medium increases the accumulation of oxo-bile acids (51). Generation of oxo-bile acids may be more favorable under the higher redox potentials found on the mucosal surface (4), whereas reduction of oxo-bile acids may be more favorable under the low redox potential (-200 to -300 mV) in the large intestinal lumen. Thus, although the redox potential of the colon is net reductive, microenvironments at the mucosa may provide oxidizing conditions favorable for certain microbial reactions. HSDHs differ in their reductive and oxidative pH optima, NAD(H) or NADP(H) requirements, molecular weight, and gene regulation (Table 2).

#### 3α- and 3β-HSDHs

3α/β-HSDHs specifically catalyze the reversible, stereospecific oxidation/reduction between 3-oxo-bile acids and 3α- or 3β-hydroxy bile acids. 3α-HSDHs have been detected in some of the most prevalent intestinal bacteria, including *C. perfringens* (54), *Peptostreptococcus productus* (55), and *Eggerthella lenta* (formerly *Eubacterium lentum*) (56, 57), as well as in intestinal bacteria present in lower numbers (≤10⁵/g wet weight of feces), including *C. scindens* (58) and *C. hiranonis* (59), and in nonintestinal bacteria, including *Pseudomonas testosteroni* (60, 61). 3β-HSDH activity has been described in species of *Clostridium* and *Ruminococcus* (62–64). It appears that intraspecies 3-epimerization favors the 3α-position. In fact, growing cultures of *C. perfringens* in the presence of 3-oxo-CDCA formed CDCA (84%) preferentially over iso-CDCA (16%) under anaerobic conditions (65).

---

**Bile salt biotransformations by human intestinal bacteria** 247
Pyridine nucleotide cofactor requirements differ between 3α/β-HSDHs. 3α-HSDHs require NAD(H), with the exception of the enzyme purified from *C. perfringens*, which uses NADP(H), and that purified from *C. scindens*, which can use either NAD(H) or NADP(H) (54, 56–58, 61). 3β-HSDHs have been shown to preferentially require NADP(H), with the exception of *C. innocuum*, which uses NAD(H) (62–64). Dihydroxy bile acids [DCA, CDCA, and ursodeoxycholic acid (UDCA)] are generally better substrates than trihydroxy bile acids (CA) (62, 65).

3α/β-HSDHs characterized to date are constitutively expressed with the exception of those from *C. scindens* and *C. hiranonis*, which are induced by the primary bile acids CA and CDCA. In fact, three copies of 3α-HSDH genes (the bile acid-inducible genes *baiA1*, *baiA2*, and *baiA3*) have been identified from *C. scindens* (66, 67), and *baiA1* has been expressed in *Escherichia coli* and characterized (58). The *baiA* gene products are unique among 3α/β-HSDHs as a result of their high specificity toward CA-CoA and CDCA-CoA conjugates and relatively low activity toward free bile acids (58).

### 7α- and 7β-HSDHs

7α/β-HSDHs catalyze the reversible, stereospecific oxidation/reduction of the 7α- and 7β-hydroxyl groups of bile acids. Although 7α/β-HSDHs are common among intestinal bacteria, the extent of 7α/β-dehydrogenation in the intestine, or in mixed fecal suspensions, is difficult to interpret because of the competing, and irreversible, 7α/β-dehydroxylation of bile acids (see below) (68).

7α/β-HSDHs are widespread among the bacteroides and clostridia as well as in *E. coli* and *Ruminococcus* species (Table 2) (54, 64, 69–74). In addition, several intestinal clostridia express both 7α- and 7β-HSDHs and have been shown to epimerize the 7α/β-hydroxy group (75, 76–78). 7α/β-HSDHs have been partially purified from intestinal bacteria, including *Ba. fragilis* (71, 79), *Ba. thetaiotamicron* (74), *C. scindens* (80), *C. sordellii* (70), and *E. coli* (73), as well as from the soil isolates *Xanthomonas maltophilia* (81), *C. absonum* (82), and *C. bifermentans* (83). Interestingly, several intestinal bacteria with 7α/β-HSDH activity also possess BSH, including *Ba. fragilis* (25), *C. sordellii* (70), *C. perfringens* (84), and *C. innocuum* (63), as well as the soil isolate *C. bifermentans* (83).

7α-HSDHs generally use NADP(H) as a cofactor, with the exception of *E. coli* (85) and *Ba. thetaiotamicron* (74). *C. bifermentans*, *C. absonum*, and *Ba. fragilis* 7α-HSDHs can use either NAD(H) or NADP(H) as a cofactor (76, 79, 83). 7β-HSDH enzymes characterized to date use NADP(H) as a cofactor (53, 62, 75, 76). 7α/β-HSDH enzymes have

---

**TABLE 2. Characteristics of HSDHs from intestinal bacteria**

| Organism                          | Stereospecificity | Cofactor       | pH Optimum          | Native Molecular Mass (kDa)         | Gene Regulation           | Reference(s) |
|-----------------------------------|-------------------|----------------|---------------------|--------------------------------------|---------------------------|---------------|
| *Eggerthella lenta*               | 3α (CE)           | NAD(H)         | 11.3                | 205                                  | R by BA                   | 56, 57        |
|                                   | 12α (CE)          | NAD(H)         | 8.0–10.5            | 125                                  | R by BA                   |               |
| *Clostridium perfringens*         | 3α (CE)           | NADP(H)        | 11.3                | ND                                   | NI                        | 54, 65        |
|                                   | 12α (CE)          | NAD(H)         | 10.5                | ND                                   | NI                        |               |
| *Peptostreptococcus productus*    | 3α (CE)           | NAD(H)         | 8.5                 | 132                                  | NI                        | 55, 165, 166  |
|                                   | 3β (CE)           | NAD(H)         | 9.5                 | 95                                   | NI                        |               |
|                                   | 7β (PP)           | NADP(H)        | 9.8                 | 53                                   | I by 7-oxo-BA             |               |
| *Bacteroides fragilis*            | 7α (P)            | NAD(H)         | 8.5 (oxid); 6.5 (red) | 110 (tetramer)                      | I (CA)                    | 71, 79, 167   |
|                                   | 7α (PP)           | NADP(H)        | 7.0–9.0             | 127                                  | I (CA)                    |               |
| *Bacteroides thetaiotamicron*     | 7α (PP)           | NAD(H)         | ND                  | 320                                  | ND                        | 74            |
| *Escherichia coli*                | 7α (P)            | NAD(H)         | 8.5                 | 120 (tetramer)                       | NI                        | 73            |
| *Clostridium sp.* 25.11.C         | 3β (CE)           | NADP(H)        | 7.5                 | 104                                  | NI                        | 62            |
|                                   | 7α (CE)           | NADP(H)        | 8.6                 | 82                                   | I (CA, CDCA, 7-oxo-CA)     |               |
|                                   | 7β (CE)           | NADP(H)        | 8.6                 | 115                                  | I (CA, CDCA, 7-oxo-CA)     |               |
| *Clostridium absonum*             | 7α (PP)           | NAD(H)         | 9.5–11.5            | ND                                   | I (CDCA, DCA); R (UDCA)    | 76, 82        |
|                                   | 7β (PP)           | NADP(H)        | 9.0–10.0            | 200 (tetramer)                       | I (CDCA, DCA); R (UDCA)    |               |
| *Clostridium sordellii*           | 7α (P)            | NADP(H)        | ND                  | 120 (tetramer)                       | I (CDCA, CA, DCA)         | 70            |
| *Clostridium innocuum*            | 3β (PP)           | NAD(H)         | 10.0–10.2           | 56                                   | NI                        | 63            |
| *Clostridium scindens*            | 3α (P)            | NAD(H)/NADP(H) | 5.0–9.0             | 108 (tetramer)                       | I (CA, CDCA)              | 58, 80        |
| VPI 12708                         | 7α (P)            | NADP(H)        | ND                  | 124 (tetramer)                       | NI                        |               |
| *Clostridium bifermentans*        | 7α (PP)           | NAD(H)/NADP(H) | 11.0                | ND                                   | I (7-oxo-LCA, CDCA, DCA)  | 83            |
| *Clostridium limosum*             | 7α (CE)           | NAD(H)/NADP(H) | 10.5                | ND                                   | I (CA, CDCA, UDCA)        | 75            |
|                                   | 7β (CE)           | NADP(H)        | 10.5                | ND                                   | I (CA, CDCA, UDCA)        |               |
| *Clostridium leptum*              | 12α (PP)          | NADP(H)        | 8.5–9.0             | 225                                  | NI                        | 89            |
| *Clostridium group*               | 12α (PP)          | NADP(H)        | 7.8                 | 100                                  | R (CDCA, CA)              | 90            |
| P strain C 48-50                  | 12β (CE)          | NADP(H)        | 7.8 (oxid); 10 (red) | 126                                  | NI                        | 91            |
| *Clostridium paraputrificum*      | 7β (CE)           | NADP(H)        | 10.5                | 45                                   | R (UDCA)                  | 165           |

BA, bile acids; CA, cholic acid; CDCA, chenodeoxycholic acid; CE, cell extract; HSDH, hydroxysteroid dehydrogenase; I, induced; LCA, lithocholic acid; NI, noninduced by bile acids; ND, not determined; P, purified; PP, partially purified; R, repressed; UDCA, ursodeoxycholic acid.

higher affinity for dihydroxy bile acids (CDCA and 7-oxo-LCA) than for trihydroxy bile acids (CA and 7-oxo-DCA). *C. limosum* cell extracts use both free and conjugated bile acids, whereas whole cells only oxidize free bile acids (75, 86). This is attributable to the intracellular location of the 7α-HSDH and the inability of the organism to take up a conjugated bile salt. The genes encoding 7α-HSDHs have been cloned from *E. coli* (73), *C. scindens* (69), and *C. sordellii* (70). Sequence similarity suggests that these enzymes belong to the short-chain polyol dehydrogenase family. Regulation of 7α/β-HSDH expression is generally growth phase-dependent and inducible by bile acid substrates (Table 2). *C. scindens* and *E. coli* constitutively express 7α-HSDHs and are noninducible (69, 85). Unexpectedly, the nonsubstrate DCA can induce 7α-HSDH expression in *C. absonum* and *C. sordellii*, although the reason for this induction remains unclear (70, 76). Macdonald, White, and Hylemon (82) observed the expression of five soluble and two membrane polypeptides upon exposure of *C. absonum* to DCA and CDCA in the culture medium, although the functions of these additional polypeptides have not been determined.

Crystal structures of the *E. coli* 7α-HSDH binary and ternary complexes have been solved and a mechanism for 7α-dehydrogenation proposed (Fig. 5) (87). Binding of 7α-hydroxy bile acids elicits major conformational changes at the substrate binding loop and C-terminal domain. A two-step mechanism is proposed in which Tyr159 acts as a catalytic base removing the C-7 hydroxy hydrogen. Hydrogen bonding by serine 146 (Ser146) is hypothesized to stabilize the intermediate. Regeneration of the catalyst occurs through the transfer of the acquired hydride from the phenolic group at Tyr159 to lysine 163 (Lys163) to position 4 of NAD⁺. Lys163 serves two important roles: anchoring NAD⁺ through bifurcated hydrogen bonding, and indirect hydride transfer from Tyr159 to position 4 of NAD⁺ (87). Site-directed mutagenesis confirmed the role of these amino acids in 7α-HSDH catalysis (88). Analysis of the 7-oxo-GLCA bile acid substrate/enzyme complex revealed tight binding of the sterol with loose association for the glycine conjugate. Binding of glycine and taurine conjugates of CDCA was not significantly different from that of the free bile acid (87).

### 12α- and 12β-HSDHs

12α/β-HSDHs have been detected mainly among members of the genus *Clostridium*. NADP-dependent 12α-HSDHs have been detected in *C. leptum* (89) in *Clostridium* group P (90), whereas NAD-dependent 12α-HSDH activity was reported in *Eg. lenta* (56) and *C. perfringens* (54). 12β-HSDHs have been detected in *C. tertium*, *C. difficile*, and *C. paraputrificum* (91, 92). 12α/β-HSDHs characterized to date are constitutively expressed and noninducible, with the exception of the 12β-HSDH from *C. paraputrificum*, which is induced by 12-oxo-bile acid substrates (92). 12α/β-HSDHs generally have higher affinity for dihydroxy bile acids (DCA) than for trihydroxy bile acids (CA and iso-CA) and for free versus conjugated bile acids. The 12α-HSDH from *C. leptum* is an exception, demonstrating higher affinity for CA conjugates than for free CA (89). 12α-HSDHs appear to be repressed by the addition of bile acid substrates (DCA > CDCA > CA) to the growth medium at 1 mM concentrations. It has been suggested that these enzyme activities should be repressed in bacteria colonizing the large intestine (56, 92), although 12-oxo-bile acids have been detected at low levels in the feces of healthy individuals (Fig. 3) (93, 94).

#### Benefits of bile acid hydroxysteroid oxidoreductases to the bacterium

The oxidation of bile acid hydroxyl groups generates reducing equivalents for cellular biosynthetic reactions and possibly electron transport phosphorylation. Bile acid

![Figure 5](attachment:Figure_5.png)

**Fig. 5.** Proposed catalytic mechanism of bile acid 7α-dehydrogenation based on the crystal structure and site-directed mutagenesis of active site amino acids of the 7α-dehydrogenase from *E. coli*. See text for a description of catalysis. Reprinted with permission from Tanaka et al. (87). Copyright © 1996 American Chemical Society.

dehydrogenation is hypothesized to generate energy in *Ba. thetaiotamicron* (74). Sherod and Hylemon (74) suggested that reduced pyridine nucleotides generated from 7α-hydroxy oxidation serve to generate ATP via a cytochrome-linked electron transport chain in the presence of electron acceptors (i.e., fumarate).

Bile acids are potent antimicrobial agents provided that the proper concentration and proportion of hydrophobic bile acids (CDCA, LCA, and DCA) are present (95). Alteration of hydroxy group stereochemistry has a marked influence on the physiochemical properties of bile acids (96, 97). The epimerization of the 7α-hydroxy group of CDCA decreases the hydrophobicity and toxicity of the bile acid (97). Macdonald, White, and Hylemon (82) observed that *C. absonum* grew on plates containing 1 mM UDCA, although it was unable to grow on plates containing 1 mM CDCA. Furthermore, when cultured in the presence of 7-oxo-bile acids, only low concentrations of CA and CDCA were formed, whereas the majority of 7-oxo-bile acids were reduced to 7-epicholic acid and UDCA, respectively, by log-phase *C. absonum* (51). UDCA has also been shown to act as a repressor of 7α/β-HSDH production in *C. absonum*, suggesting that UDCA is an end product (76). The enzyme also displays markedly higher affinity for CDCA than for CA, the former being more toxic. In summary, dehydrogenation may serve functions related to energy generation as well as attempts to maintain low concentrations of more hydrophobic bile acids in the bacterium's microenvironment.

**Interplay between HSDH enzymes in human liver and intestinal bacteria**

The coevolution between host and gut flora is evident when observing the interplay between liver and bacterial biotransforming reactions. The liver synthesizes bile acids in which the hydroxy groups are in the α orientation. In the α-hydroxy orientation, one face of the molecule is hydrophobic and the other side is hydrophilic. This translates to efficient solubilization of lipid molecules through the formation of mixed micelles capable of efficient emulsification while remaining soluble in aqueous environments. Generation of β-hydroxy bile acids by microbial enzymes alters the efficiency of micelle formation as a result of hydrophilic groups on both faces of the sterol molecule. The differences observed between the composition of bile acids in serum and bile are a result of the continual interplay between liver and bacterial enzymes. Exposure of bile salts to intestinal bacteria results in ~50% of bile acids requiring reconjugation and low levels of bile acids returned to the liver in the 3β-hydroxy orientation (98). Without a means of epimerizing bile acid hydroxy groups in the human liver, β-hydroxy bile acids would accumulate in the bile acid pool. Interestingly, the liver seems to “allow” the accumulation of UDCA (7β-hydroxy) in the biliary pool, as in the case of therapeutic administration of CDCA (99). The protective effects of UDCA observed in clinical studies (100) as well as cell culture studies (97, 101) may provide an evolutionary explanation for this phenomenon.

---

THE BIOCHEMISTRY AND MOLECULAR BIOLOGY OF BILE ACID 7α/β-DEHYDROXYLATION

**Introduction**

Secondary bile acids (DCA and LCA) predominate in human feces (Fig. 3). Therefore, 7α-dehydroxylation is the most quantitatively important bacterial bile salt biotransformation in the human colon. The rapid rate of conversion of primary to secondary bile acids is surprising given current estimates that this metabolic pathway is found in ~0.0001% of total colonic flora (102–104). Human intestinal bacteria capable of bile acid 7α-dehydroxylation have been isolated (104, 105), and 16S rDNA phylogenetic analysis has led to their classification to the genus *Clostridium* (106–108).

Unlike bile acid oxidation and epimerization, 7α/β-dehydroxylation appears restricted to free bile acids. Removal of glycine/taurine bile acid conjugates via BSH enzymes is thus a prerequisite for 7α/β-dehydroxylation by intestinal bacteria (109–112). Some intestinal bacteria are capable of both 7α/β-dehydroxylating activities (113), whereas 7β-dehydroxylation activity is absent in other intestinal 7α-dehydroxylating bacteria (113). Epimerization of UDCA (7β-hydroxy) to CDCA (7α-hydroxy) via 7β-HSDHs produced by members of the gut flora results in subsequent 7α-dehydroxylation. In this regard, it appears that the presence of 7β-dehydroxylation activity is more of a luxury than a necessity.

**Elucidating the bile acid 7α/β-dehydroxylation pathway**

Samuelsson (114) administered \[6α-³H,6β-³H,8β-³H\] \[24-¹⁴C\]CA to bile duct-cannulated rabbits and rats. Analysis of the products recovered after exposure to intestinal bacteria revealed a differential loss of the 6β-³H during 7α-dehydroxylation of CA. Previous work showed complete retention of the 7β-³H in \[7β-³H\]\[24-¹⁴C\]CA during 7α-dehydroxylation in the rat intestine (115, 116). These data led Samuelsson (114) to propose a mechanism for CA 7α-dehydroxylation involving two steps: diaxial trans-elimination of the 7α-hydroxy group and 6β-hydrogen atom, followed by reduction through trans-hydrogenation of the 6β and 7α positions of the cholen-6-oic acid intermediate forming DCA. Björkhem et al. (117) showed the formation of a 3-dehydro-4-cholenoic acid intermediate after the differential loss of the 5βH in vitro and in vivo using \[3β-³H\]\[24¹⁴C\]CA and \[5β-³H\]24¹⁴C\]CA. Hylemon et al. (118) subsequently observed the accumulation of multiple bile acid intermediates in cell extracts of *C. scindens* induced by CA (Fig. 6). These radiolabeled CA intermediates were identified by mass spectrometry, then chemically synthesized and added to cell extracts of CA-induced *C. scindens*. Each 24\[^{14}\text{C}\]CA intermediate was converted to 24\[^{14}\text{C}\]DCA in cell extracts prepared from CA-induced cultures of *C. scindens*. These observations suggested that the 7α-dehydroxylation mechanism was more complex than the two-step mechanism proposed by Samuelsson (114). Furthermore, these data demonstrated that bile acid 7α-dehydroxylation was a multistep pathway in *C. scindens* and suggested the presence of multiple *bai* genes.

by one- and two-dimensional SDS-PAGE (119, 120). Purification and N-terminal sequencing of these *bai* polypeptides facilitated the cloning of *bai* genes through the design of degenerate probes (121–123). Northern blot analysis indicated the presence of a large CA-inducible (≥10 kb) mRNA transcript and a smaller transcript (≤1.5 kb) in *C.* scindens (121, 124). These studies led to the discovery of a *bai* regulon encoding at least 10 open reading frames (Fig. 7). Individual *bai* genes have been subcloned into *E.* coli and the functions of many of them determined (58, 67, 123, 125–130; P. B. Hylemon, unpublished data). The proposed bile acid 7α/β-dehydroxylation pathway in *C.* scindens is shown in Fig. 8. A *bai* operon has also been characterized from *C.* hiranonis (59), although the discussion below of the 7α/β-dehydroxylation pathway will center on *C.* scindens, from which the functions of the gene products have been determined.

### *bai* genes: a regulon for 7α/β-dehydroxylation

The transport of unconjugated primary bile acids into *C.* scindens is facilitated by the *baiG* gene product, which belongs to a major pump/facilitator superfamily of protein transporters (126). The *baiG* gene has been cloned into *E.* coli and shown to encode a 50 kDa H⁺-dependent bile acid transporter (126). BaiG facilitates the transport of unconjugated CA and CDCA but not of the secondary bile acids DCA and LCA (126). Computer-aided modeling suggests that the *baiG* polypeptide contains 14 membrane-spanning domains (126).

---

**Fig. 6.** Accumulation of \[^{24-14}C\]CA intermediates during 7α-dehydroxylation in cell extracts of *C.* scindens VPI 12708. Cell extracts were prepared from either CA-induced (I) or control (C) cells. This figure was modified from Hylemon et al. (118). Copyright © 1992 American Society for Biochemistry and Molecular Biology, Inc.

The induction of 7α-dehydroxylation activity in *C.* scindens by unconjugated C₂₄ primary bile acids resulted in the appearance of several new polypeptides, as observed

---

**Fig. 7.** Gene organization of the bile acid-inducible (*bai*) 7α/β-dehydroxylation operons characterized in *C.* scindens VPI 12708. P indicates the promoter region.

Fig. 8. Proposed bile acid 7α/β-dehydroxylation pathways in *C. scindens* VPI 12708 for CDCA and UDCA. Reaction steps in brackets indicate enzymatic steps for CDCA and UDCA intermediates. For simplicity, only CDCA intermediates are shown for the general 7-dehydroxylation pathway. The *baiCD* and *baiE* gene products are proposed to encode stereospecific enzymes for 7α-hydroxy bile acids. The *baiH* and *baiI* gene products are proposed to encode stereospecific enzymes for 7β-hydroxy bile acids. The *baiF* gene product has bile acid-CoA hydrolase activity but is hypothesized to encode a bile acid CoA transferase (see text).

After transport, ligation to CoA is the first step in activating CA and CDCA for 7α-dehydroxylation, as several subsequent enzymatic steps are specific for CoA conjugates. The *baiB* gene was shown to encode a 58 kDa bile acid CoA ligase (125). CoA ligation is ATP-, CoA-, and Mg²⁺-dependent and also requires a free carboxyl group on C₂₄ bile acids (125). CoA ligation may function both to sterically hinder the constitutive 7α-HSDHs, committing the bile acid to 7α-dehydroxylation, and to trap the bile acid inside the cell.

The 3α-hydroxy group is oxidized after CoA ligation. Oxidation of the 3-hydroxy group inhibits 7α-hydroxy group dehydrogenation, favoring 7-dehydroxylation over constitutively expressed 7α-HSDHs in *C. scindens* (58, 80). The *baiA* gene products encode 27 kDa polypeptides that have significant similarity with the short-chain alcohol/polyol dehydrogenase gene family (58, 124). Amino acid multiple sequence alignment and comparison between *baiA* gene products and other members of the short-chain alcohol dehydrogenase family revealed a possible NAD(P) binding site and catalytic active site (58). Three *baiA* genes have been cloned from *C. scindens*: the *baiA1* and *baiA3*

genes are monocistronic, whereas the *baiA2* gene is part of the polycistronic *bai* operon (66, 67, 124). The *baiA* genes from *C. scindens* were cloned in *E. coli* and shown to encode 3α-HSDHs (58). The enzymes only recognize bile acid CoA conjugates and can use either NAD⁺ or NADP⁺ as electron acceptors (58). Interestingly, the different 3α-HSDHs of *C. scindens* share 92% amino acid sequence identity with one another, suggesting gene duplication. The physiological importance of multiple *baiA* genes remains unclear.

The *baiCD* gene from *C. scindens* has been cloned and expressed in *E. coli* and was recently demonstrated to encode a steroid oxidoreductase specific for the CoA conjugates of 3-dehydro-4-cholenoic acid and 3-dehydro-4-chenodeoxycholenoic acid (Fig. 8) (P. B. Hylemon et al., unpublished data). The ~70 kDa enzyme requires FMN and NAD⁺ or NADP⁺ for activity and shows stereospecificity toward 7α-hydroxy bile acids. The BaiCD polypeptide shows considerable amino acid sequence identity with the Old Yellow Enzyme family, a putative NADH oxidase from *Li. monocytogenes*, several 2,4-dienoyl CoA reductases, and *baiH* from *C. scindens* (121, 129). The *baiH* gene en-

Fig. 9. Model of the bile acid 7α-dehydratase (baiE) based on structural data from scytalone dehydratase, nuclear transport factor 2, and steroid Δ⁵-isomerase. The α-helices are denoted by white strands, β-sheets are denoted by the yellow backbone, and the blue structure represents a steroid molecule modeled into the binding/active site pocket. Model kindly provided by Dr. Alexey G. Murzin (Cambridge University).

codes a 72 kDa polypeptide containing 661 amino acids (121). The BaiH protein exists as a homotrimer with NADH:flavin oxidoreductase activity (121). The baiH gene has been subcloned into *E. coli*, purified, and shown to contain 1 mol of FAD, 2 mol of iron, and 1 mol of copper per mole polypeptide subunit (129). The baiH gene product from *C. scindens* was recently determined to encode a steroid oxidoreductase specific for CoA conjugates of 3-dehydro-4-ursodeoxycholenoic acid and 3-dehydro-4-epicholenoic acid (Fig. 8) (P. B. Hylemon et al., unpublished data). Oxidation of CA-CoA, CDCA-CoA, and UDCA-CoA to their respective 3-dehydro-4-bile acid CoA conjugates appears to make the bile acid chemically labile for 7α/β-dehydration.

The baiF gene encodes a 47.5 kDa polypeptide containing 426 amino acids that was shown to have bile acid CoA hydrolase activity (122, 130). However, baiF is hypothesized to encode a CoA transferase because of energy conservation (Fig. 8) and homology to the type III family of CoA transferases (131). The first few cycles of 7α-dehydroxylation would require ATP hydrolysis (Fig. 8), although ATP-independent recycling of the thioesterase intermediates via transfer of CoA from 3-dehydro-4-cholenoic acid to CA, 3-dehydro-4-chenodeoxycholenoic acid to CDCA, or 3-dehydro-4-ursodeoxycholenoic acid to UDCA by the baiF would significantly conserve energy. However, this hypothesis remains to be tested.

7α-Dehydration of 3-dehydro-4-cholenoic acid and 3-dehydro-4-chenodeoxycholenoic acid results in the generation of a conjugated double bond in rings A and B, forming stable 3-dehydro-4,6-deoxycholdienoic acid and 3-dehydro-4,6-lithocholdienoic acid intermediates, respectively. The 7α-dehydration step results in the largest calculated energy change (−9.4 kcal/mol) of any reaction in this pathway, and the reverse reaction was not detected in vitro (128). The 19.5 kDa bile acid 7α-dehydratase is encoded by the baiE gene (128). This enzyme showed no activity with 3-dehydro-4-ursodeoxycholenoic acid (128). The baiE gene product was modeled on the crystal structure of the protein homologs (secondary structure) keto-steroid isomerase and scytalone dehydratase (Fig. 9) (K. Woodford et al., unpublished data). The putative active site/binding pocket is shown in Figure 10. A catalytic mechanism has been proposed for the 7α-dehydration step based on the conservation in secondary structure and site-directed mutagenesis of the key active site amino acids. Tyr30 acts as a general acid withdrawing electron density from the 3-oxo group. This shift in electron density is hypothesized to make the 6β-hydrogen labile for removal by the general base histidine 83 (His83) (assisted by Asp35). His83 is thought to donate its hydrogen to the 7α-hydroxy forming the water-leaving group that is stabilized by Asp106. Site-directed mutagenesis based on the proposed mechanism supports the important role of the putative active site amino acids in enzyme catalysis. It is hypothesized that the baiL gene encodes a bile acid 7β-dehydratase, because of amino acid sequence homologies between the baiE and baiL gene products.

Genes involved in the reductive arm of the 7α/β-dehydroxylation pathway have not been isolated. These genes should encode oxidoreductases catalyzing the reduction of 3-dehydro-4,6-deoxycholdienoic acid to 3-dehydro-4-deoxycholenoic acid to 3-dehydro-deoxycholic acid to DCA as well as a bile acid exporter to remove secondary bile acid end products from the bacterium (Fig. 8). Genes encoding putative transcriptional regulators have been detected upstream of the bile acid-inducible promoter region (Table 3) (D. H. Mallonee and P. B. Hylemon, unpublished data). Additional studies will be required to determine the mechanism of induction.

TABLE 3. $b a i$ genes characterized from C. scindens VPI 12708

| $b a i$ Gene | Molecular Mass ($k D a$) | Catalytic Activity/Function | Gene Family | Reference |
|--------------|--------------------------|----------------------------|-------------|-----------|
| $b a i A$    | 27                       | $3 \alpha-$HSDH            | Short-chain alcohol/polyol dehydrogenase | 58        |
| $b a i B$    | 58                       | Bile acid CoA ligase       | AMP binding | 125       |
| $b a i C D$  | 70                       | $3-$Dehydro-$4-$CDCA/$C A$<br>steroid oxidoreductase | Pyridine nucleotide-disulfide oxidoreductase;<br>NADH:flavin oxidoreductase | $-^{a}$   |
| $b a i H$    | 72                       | $3-$Dehydro-$4-$UDCA/$7-$epiCA<br>steroid oxidoreductase | Pyridine nucleotide-disulfide oxidoreductase;<br>NADH:flavin oxidoreductase | 129       |
| $b a i E$    | 19.5                     | $7 \alpha-$Dehydratase     | COG 4875    | 128       |
| $b a i I$    | 22                       | $7 \beta-$Dehydratase$^{b}$ | COG 4876    | 168       |
| $B a i F$    | 47.5                     | Bile acid CoA hydrolase<br>Bile acid CoA transferase$^{b}$ | Type III CoA transferase | 130       |
| $b a i G$    | 50                       | $H^{+}-$dependent bile acid transporter | Major facilitator superfamily | 126       |
| $b a r A$    | 46                       | Transcriptional regulation$^{b}$ | AraC/XylS | $-^{a}$   |
| $b a r B$    | 22                       | Transcriptional regulation$^{b}$ | RpoB; permeases of the major facilitator superfamily | $-^{a}$   |

$b a i$, bile acid-inducible; $b a r$, bile acid-regulatory.

$a$ P.B. Hylemon et al., unpublished data.

$b$ Hypothesized function.
that bile acids may cause DNA damage and act as carcinogens in humans (138). Higher levels of DCA are found in the blood of colon cancer patients compared with control patients (98, 143). Moreover, DCA is a logical candidate for promoting colon carcinogenesis for the following reasons: 1) it is found in fecal water in high concentrations (>100 μM) (138); 2) it can cross biological membranes via passive diffusion; and 3) it can activate mammalian cell signaling pathways that are known to be involved in promoting carcinogenesis. In this regard, cell signaling pathways activated by DCA in mammalian epithelial cells include protein kinase C (144), ERK1/2 via the epidermal growth factor receptor (101, 145, 146), β-catenin (147), and Jun-N-terminal kinase 1 and 2 (JNK1/2) (148). Secondary bile acids have been shown to cause apoptosis in colonic epithelial cells, and high concentrations of DCA and LCA in stool may promote carcinogenesis by exerting selective pressure for the emergence of epithelial cell mutants that are resistant to apoptosis (e.g., via loss of p53) (138). LCA has been found to be an excellent activator of the vitamin D receptor (149, 150). Activation of this receptor in intestinal epithelial cells activates genes that metabolize LCA (150). This may be a protective mechanism that evolved to limit LCA toxicity to intestinal epithelial cells.

DISCUSSION

Bile salt metabolism is a widespread and fundamental property of the gastrointestinal microbiota encompassing the most commonly isolated species of intestinal bacteria, including but not limited to the genera *Bacteroides*, *Clostridium*, *Lactobacillus*, *Bifidobacterium*, *Eubacterium*, and *Escherichia*. However, our current understanding of the intestinal microbiology of bile salt modifications is limited to cultivated species. Discovering novel genes in the gut microbiome (collective genomes of the gastrointestinal flora) encoding BSHs, HSDHs, and enzymes involved in the 7α/β-dehydroxylation pathway through molecular techniques will facilitate a much greater understanding of the diversity and complexity of these reactions in the human colon. Techniques such as PCR-denaturing gradient gel electrophoresis can be used to measure the diversity of organisms based on specific phylogenetic markers, such as 16S rDNA, functional genes, and potentially bile salt-modifying enzymes (151, 152). Measuring the true diversity of bile salt-modifying bacteria is crucial in studying the relationship between the levels and activities of these bacteria and disease risk.

Determining the conditions in which secondary bile acids are formed in significant quantities and retained in the enterohepatic circulation of certain individuals is suggested to be important in the etiology of cholesterol gallstone disease and colon cancer. Because secondary bile acids are formed exclusively through bacterial enzymatic reactions, the study of microbes capable of bile acid 7α/β-dehydroxylation is important in understanding these chronic GI illnesses. The goal of such research is to find ways to block the source of secondary bile acid production. Long-term use of antibiotics to prevent 7α-dehydroxylation of bile acids would be impractical. The design of pharmaceuticals to block the 7α-dehydroxylation pathway is a possibility. However, this approach requires targeting microbial enzymes and runs the risk of eventual drug resistance, as with antibiotics. Alternatively, reducing secondary bile acid production may be achieved by administering specialized CA-accumulating probiotic bacteria (153). Studies with bifidobacteria and lactobacilli isolated from human feces have shown their ability to assimilate CA spontaneously in vitro (153, 154). The mechanism for CA uptake in lactic acid bacteria appears to be diffusion of a hydrophobic weak acid through the membrane via the transmembrane proton motive force (153, 154). The higher intracellular pH causes the bile acids to become trapped as a result of ionization.

The use of specific bacteria as “drugs” to treat chronic GI illnesses caused in part by other intestinal bacteria certainly has potential, although rigorous studies are needed to demonstrate the effectiveness of such therapies. Overall, clinical trials using probiotics to solve specific health disorders have met with mixed results. Discrepancies between studies are attributable in part to different methodologies, choice of probiotic strains, colony-forming units administered per day, and patient characteristics. One particular variable that would have to be addressed when determining bile acid assimilation by lactic acid bacteria in clinical trials is viability in vivo after passage through the gastric juice and bile. A bacterium must be alive to create a membrane potential capable of accumulating bile acids. Targeted delivery of probiotic bacteria to the intestine in microencapsulated form has shown promise in improving viability in the presence of gastric conditions and bile (155, 156). Another issue is whether the introduction of billions of probiotic bacteria to the small bowel will have an effect on bile acid input into the colon. Significant bile salt hydrolysis proximal to the terminal ileum reduces the efficiency of bile salt uptake through high-affinity transport, allowing enhanced excretion of bile acids in feces. This principle is behind attempts to decrease serum cholesterol using probiotics with bile salt hydrolytic activity (157–159). Thus, the choice of probiotic delivery mechanisms is important in addressing issues of bacterial viability and in preventing increased bile salt hydrolysis proximal to the terminal ileum. The potential impact of probiotic bacteria in reducing secondary bile acid formation in the human colon by sequestering bile acids would be greatly reduced if these same bacteria cause an increase in bile acid input into the large intestine greater than their capacity for sequestering bile acids.

The authors acknowledge Dr. Alexey G. Murzin of the Centre for Protein Engineering (Cambridge University) for computer modeling a three-dimensional structure of the *baiE* gene. The authors also thank Mikyung Kang for graphic assistance with several figures. The authors are indebted to Elaine Studer and Drs. Douglas M. Heuman, William M. Pandak, James E. Wells, Darrell H. Mallonee, and Gregorio Gil for their constructive

Bile salt biotransformations by human intestinal bacteria 255
comments and advice during the preparation of the manuscript. Finally, the authors thank the reviewers for their valuable suggestions, which have improved this article. This work was supported by a Grant from the National Institutes of Health (PO1DK38030).

# REFERENCES

1. Eckburg, P. B., E. M. Bik, C. N. Bernstein, E. Purdom, L. Dethlefsen, M. Sargent, S. R. Gill, K. E. Nelson, and D. A. Relman. 2005. Diversity of the human intestinal flora. *Science*. **308**: 1635–1638.
2. Savage, D. C. 1977. Microbial ecology of the gastrointestinal tract. *Annu. Rev. Nutr.* **31**: 107–133.
3. Whitman, W. B., D. C. Coleman, and W. J. Wiebe. 1998. Prokaryotes: the unseen majority. *Proc. Natl. Acad. Sci. USA*. **95**: 6578–6583.
4. Wilson, M. 2005. Microbial Inhabitants of Humans. Cambridge University Press, Cambridge, UK. 1–47, 251–313, 375–392.
5. McGarr, S. E., J. M. Ridlon, and P. B. Hylemon. 2005. Diet, anaerobic bacterial metabolism and colon cancer risk: a review of the literature. *J. Clin. Gastroenterol.* **39**: 98–109.
6. Vlahcevic, Z. R., D. M. Heuman, and P. B. Hylemon. 1996. Physiology and pathophysiology of enterohepatic circulation of bile acids. *In* Hepatology: A Textbook of Liver Disease. 3rd edition. Vol. 1. D. Zakim and T. Boyer, editors. Saunders, Philadelphia, PA. 376–417.
7. Hofmann, A. F. 1999. The continuing importance of bile acids in liver and intestinal disease. *Arch. Intern. Med.* **159**: 2647–2658.
8. Cowen, A. E., M. G. Korman, A. F. Hofmann, O. W. Cass, and S. B. Coffin. 1975. Metabolism of lithocholate in healthy man. II. Enterohepatic circulation. *Gastroenterology*. **69**: 67–76.
9. Heijghebaert, S. M., and A. F. Hofmann. 1986. Influence of the amino acid moiety on deconjugation of bile acid amidates by cholyglycine hydrolase on human fecal cultures. *J. Lipid Res.* **27**: 742–752.
10. Tanaka, H., H. Hashiba, J. Kok, and I. Mierau. 2000. Bile salt hydrolase of *Bifidobacterium longum* biochemical and genetic characterization. *Appl. Environ. Microbiol.* **66**: 2502–2512.
11. Rossocha, M., R. Schultz-Heienbrok, H. von Moeller, J. P. Coleman, and W. Saenger. 2005. Conjugated bile acid hydrolase is a tetrameric N-terminal thiol hydrolase with specific recognition of its choly but not of its tauryl product. *Biochemistry*. **44**: 5739–5748.
12. Elkins, C. A., S. A. Moser, and D. C. Savage. 2001. Genes encoding bile salt hydrolases and conjugated bile salt transporters in *Lactobacillus johnsonii* 100-100 and other *Lactobacillus* species. *Appl. Environ. Microbiol.* **147**: 3403–3412.
13. Gopal-Srivastava, R., and P. B. Hylemon. 1988. Purification and characterization of bile salt hydrolase from *Clostridium perfringens*. *J. Lipid Res.* **29**: 1079–1085.
14. Grill, J. P., F. Schneider, J. Crociani, and J. Ballongue. 1995. Purification and characterization of conjugated bile salt hydrolase from *Bifidobacterium longum* BB536. *Appl. Environ. Microbiol.* **61**: 2577–2582.
15. Coleman, J. P., and L. L. Hudson. 1995. Cloning and characterization of a conjugated bile acid hydrolase gene from *Clostridium perfringens*. *Appl. Environ. Microbiol.* **61**: 2514–2520.
16. Christiaens, H., R. J. Leer, P. H. Pouwels, and W. Verstraete. 1992. Cloning and expression of a conjugated bile acid hydrolase gene from *Lactobacillus plantarum* by using a direct plate assay. *Appl. Environ. Microbiol.* **58**: 3792–3798.
17. Elkins, C. A., and D. C. Savage. 1998. Identification of genes encoding conjugated bile salt hydrolase and transport in *Lactobacillus johnsonii* 100-100. *J. Bacteriol.* **180**: 4344–4349.
18. Kim, G. B., C. M. Miyamoto, E. A. Meighen, and B. H. Lee. 2004. Cloning and characterization of the bile salt hydrolase genes (*bsh*) from *Bifidobacterium bifidum* strains. *Appl. Environ. Microbiol.* **70**: 5603–5612.
19. Kim, G. B., M. Brochet, and B. H. Lee. 2005. Cloning and characterization of a bile salt hydrolase (*bsh*) from *Bifidobacterium adolescentis*. *Biotechnol. Lett.* **27**: 817–822.
20. Glaser, P., L. Frangeul, C. Buchrieser, C. Rusniok, A. Amend, F. Baquero, P. Berche, H. Bloecker, P. Brandt, T. Chakraborty, et al. 2001. Comparative genomics of *Listeria* species. *Science*. **294**: 849–852.
21. Dussurget, O., D. Cabanes, P. Dehoux, M. Lecuit, C. Buchrieser, P. Glaser, P. Cossart, and the European Listeria Genome Consortium. 2002. *Listeria monocytogenes* bile salt hydrolase is a PrfA-regulated virulence factor involved in the intestinal and hepatic phases of listeriosis. *Mol. Microbiol.* **45**: 1095–1106.
22. Kishinaka, M., A. Umeda, and S. Kuroki. 1994. High concentrations of conjugated bile acids inhibit bacterial growth of *Clostridium perfringens* and induce its extracellular cholyglycine hydrolase. *Steroids*. **59**: 485–489.
23. Elkins, C. A., and D. C. Savage. 2003. CbsT2 from *Lactobacillus johnsonii* 100-100 is a transport protein of the major facilitator superfamily that facilitates bile acid antiport. *J. Mol. Microbiol. Biotechnol.* **6**: 76–87.
24. Lundeen, S. G., and D. C. Savage. 1992. Multiple forms of bile salt hydrolase from *Lactobacillus sp.* strain 100-100. *J. Bacteriol.* **174**: 7217–7220.
25. Stellwag, E. J., and P. B. Hylemon. 1976. Purification and characterization of bile salt hydrolase from *Bacteroides fragilis* subsp. *fragilis*. *Biochim. Biophys. Acta*. **452**: 165–176.
26. De Smet, I., L. Van Hoorde, M. Vande Woestyne, H. Christiaens, and W. Verstraete. 1995. Significance of bile salt hydrolytic activities of lactobacilli. *J. Appl. Bacteriol.* **79**: 292–301.
27. Corzo, G., and S. E. Gilliland. 1999. Measurement of bile salt hydrolase activity from *Lactobacillus acidophilus* based on disappearance of conjugated bile salts. *J. Dairy Sci.* **82**: 466–471.
28. De Boever, P., and W. Verstraete. 1999. Bile salt deconjugation by *Lactobacillus plantarum* 80 and its implication for bacterial toxicity. *J. Appl. Microbiol.* **87**: 345–352.
29. Grill, J. P., C. Cayuela, J. M. Antoine, and F. Schneider. 2000. Isolation and characterization of a *Lactobacillus amylovorus* mutant depleted in conjugated bile salt hydrolase activity: relation between activity and bile salt resistance. *J. Appl. Microbiol.* **89**: 553–563.
30. Tannock, G. W., M. P. Dashkevicz, and S. D. Feighner. 1989. Lactobacilli and bile salt hydrolysis in the murine intestinal tract. *Appl. Environ. Microbiol.* **55**: 1848–1851.
31. Flahaut, S., J. Frere, P. Boutibonnes, and Y. Auffray. 1996. Comparison of the bile salts and sodium dodecyl sulfate stress response in *Enterococcus faecalis*. *Appl. Environ. Microbiol.* **62**: 2416–2420.
32. Rince, A., Y. Le Breton, N. Verneuil, J. C. Giard, A. Hartke, and Y. Auffray. 2003. Physiological and molecular aspects of bile salt response in *Enterococcus faecalis*. *Int. J. Food Microbiol.* **88**: 207–213.
33. Lin, J., O. Sahin, L. O. Michel, and Q. Zhang. 2003. Critical role of multidrug efflux pump CmeABC in bile resistance and in vivo colonization of *Campylobacter jejuni*. *Infect. Immun.* **71**: 4250–4259.
34. Van Eldere, J., P. Celis, G. De Pauw, E. Lesaffre, and H. Eyssen. 1996. Tauroconjugation of cholic acid stimulates 7α-dehydroxylation by fecal bacteria. *Appl. Environ. Microbiol.* **62**: 656–661.
35. Cook, A. M., and K. Denger. 2002. Dissimilation of the C₂ sulfonates. *Arch. Microbiol.* **179**: 1–6.
36. Lengeler, J. W., G. Drews, and H. G. Schlegel. 1999. Biology of the Prokaryotes. Blackwell Science, New York. 756–758.
37. Adamowicz, M., P. M. Kelley, and K. W. Nickerson. 1991. Detergent (sodium dodecyl sulfate) shock proteins in *Escherichia coli*. *J. Appl. Bacteriol.* **137**: 229–233.
38. Laue, H., M. Friedrich, J. Ruff, and A. M. Cook. 2001. Dissimilatory sulfite reductase (desulfovirdin) of the taurine-degrading, non-sulfate-reducing bacterium *Bilophila wadsworthia* RZATAU contains a fused DsrB-DsrD subunit. *J. Bacteriol.* **183**: 1727–1733.
39. Lie, T. J., M. L. Clawson, W. Godchaux, and E. R. Leadbetter. 1999. Sulfidogenesis from 2-aminoethanesulfonate (taurine) fermentation by a morphologically unusual sulfate-reducing bacterium, *Desulforhopalus singaporensis* sp. nov. *Appl. Environ. Microbiol.* **65**: 3328–3334.
40. Christl, S. U., H. D. Eisner, G. Dusel, H. Kasper, and W. Scheppach. 1996. Antagonistic effects of sulfide and butyrate on proliferation of colonic mucosa: a potential role for these agents in the pathogenesis of ulcerative colitis. *Dig. Dis. Sci.* **41**: 2477–2481.
41. Deplancke, B., and H. R. Gaskins. 2003. Hydrogen sulfide induces serum-independent cell cycle entry in nontransformed rat intestinal epithelial cells. *FASEB J.* **17**: 1310–1312.
42. Levitt, M. D., J. Furne, J. Springfield, F. Suarez, and E. DeMaster. 1999. Detoxification of hydrogen sulfide and methanethiol in the cecal mucosa. *J. Clin. Invest.* **104**: 1107–1114.
43. Levine, J., C. J. Ellis, J. K. Furne, J. Springfield, and M. D. Levitt.

1998. Fecal hydrogen sulfide production in ulcerative colitis. *Am. J. Gastroenterol.* **93**: 83–87.

44. Rose, P., P. K. Moore, S. H. Ming, O. C. Nam, J. S. Armstrong, and M. Whiteman. 2005. Hydrogen sulfide protects colon cancer cells from chemopreventive agent β-phenylethyl isothiocyanate induced apoptosis. *World J. Gastroenterol.* **11**: 3990–3997.

45. Zhang, Y., and P. Talalay. 1994. Anticarcinogenic activities of organic isothiocyanates: chemistry and mechanisms. *Cancer Res.* **54**: 1976–1981.

46. Hardison, W. G. 1978. Hepatic taurine concentration and dietary taurine as regulators of bile acid conjugation with taurine. *Gastroenterology.* **75**: 71–75.

47. Sjövall, J. 1959. Dietary glycine and taurine on bile acid conjugation in man: bile acids and steroids 75. *Proc. Soc. Exp. Biol. Med.* **100**: 676–678.

48. Magee, E. A., C. J. Richardson, R. Hughes, and J. H. Cummings. 2000. Contribution of dietary protein to sulfide production in the large intestine: an in vitro and a controlled feeding study in humans. *Am. J. Clin. Nutr.* **72**: 1488–1494.

49. Gibson, G. R., J. H. Cummings, and G. T. Macfarlane. 1991. Growth and activities of sulfate-reducing bacteria in gut contents of healthy subjects and patients with ulcerative colitis. *FEMS Microbiol. Ecol.* **86**: 103–112.

50. O’Keefe, S. D. J., M. Kidd, G. Espitalier-Noel, and P. Owira. 1999. Rarity of colon cancer in Africans is associated with low animal product consumption, not fiber. *Am. J. Gastroenterol.* **94**: 1373–1380.

51. Sutherland, J. D., and I. A. Macdonald. 1982. The metabolism of primary, 7-oxo- and 7β-hydroxy bile acids by *Clostridium absonum*. *J. Lipid Res.* **23**: 726–732.

52. Hirano, S., and N. Masuda. 1981. Epimerization of the 7-hydroxyl group of bile acids by the combination of two kinds of microorganisms with 7α- and 7β-hydroxysteroid dehydrogenase activity, respectively. *J. Lipid Res.* **22**: 1060–1068.

53. Macdonald, I. A., Y. P. Rochon, D. M. Hutchison, and L. V. Holdeman. 1982. Formation of ursodeoxycholic acid from chenodeoxycholic acid by a 7β-hydroxysteroid dehydrogenase-elaborating *Eubacterium aerofaciens* strain when co-cultured with 7α-hydroxysteroid dehydrogenase-elaborating organisms. *Appl. Environ. Microbiol.* **44**: 1187–1195.

54. Macdonald, I. A., E. C. Meier, D. E. Mahony, and G. A. Costain. 1976. 3α, 7α- and 12α-hydroxysteroid dehydrogenase activities from *Clostridium perfringens*. *Biochim. Biophys. Acta.* **450**: 466–476.

55. Edenharder, R., A. Pfützner, and R. Hammann. 1989. Characterization of NAD-dependent 3α- and 3β-hydroxysteroid dehydrogenase and of NADP-dependent 7β-hydroxysteroid dehydrogenase from *Peptostreptococcus productus*. *Biochim. Biophys. Acta.* **1004**: 230–238.

56. Macdonald, I. A., D. E. Mahony, J. F. Jellet, and C. E. Meier. 1977. NAD-dependent 3α- and 12α-hydroxysteroid dehydrogenase from *Eubacterium lentum* ATCC 25559. *Biochim. Biophys. Acta.* **489**: 466–476.

57. Macdonald, I. A., J. F. Jellet, D. E. Mahony, and L. V. Holdeman. 1979. Bile salt 3α- and 12α-hydroxysteroid dehydrogenases from *Eubacterium lentum* and related strains. *Appl. Environ. Microbiol.* **37**: 992–1000.

58. Mallonee, D. H., M. A. Lijewski, and P. B. Hylemon. 1995. Expression in *Escherichia coli* and characterization of a bile acid-inducible 3α-hydroxysteroid dehydrogenase from *Eubacterium sp.* strain VPI 12708. *Curr. Microbiol.* **30**: 259–263.

59. Wells, J. E., and P. B. Hylemon. 2000. Identification and characterization of a bile acid 7α-dehydroxylating operon in *Clostridium sp.* strain TO931, a highly active 7α-dehydroxylating strain isolated from human feces. *Appl. Environ. Microbiol.* **66**: 1107–1113.

60. Skalhegg, B. A. 1974. On the 3α-hydroxysteroid dehydrogenase from *Pseudomonas testosteroni*: purifications and properties. *Eur. J. Biochem.* **46**: 117–125.

61. Skalhegg, B. A. 1975. 3α-Hydroxysteroid dehydrogenase from *Pseudomonas testosteroni*: kinetic properties with NAD and its thionicotinamide analogue. *Eur. J. Biochem.* **50**: 603–609.

62. Edenharder, R., M. Pfützner, and R. Hammann. 1989. NADP-dependent 3β-, 7α- and 7β-hydroxysteroid dehydrogenase activities from a lecithinase-lipase-negative *Clostridium* species 25.11.c. *Biochim. Biophys. Acta.* **1002**: 37–44.

63. Edenharder, R., and M. Pfutzner. 1989. Partial purification and characterization of an NAD-dependent 3β-hydroxysteroid dehy-

drogenase from *Clostridium innocuum*. *Appl. Environ. Microbiol.* **55**: 1656–1659.

64. Akao, T., T. Akao, M. Hattori, T. Namba, and K. Kobashi. 1987. Enzymes involved in the formation of 3β,7β-dihydroxy-12-oxo-5β-cholanolic acid from dehydrocholic acid by *Ruminococcus sp.* obtained from human intestine. *Biochim. Biophys. Acta.* **921**: 275–280.

65. Macdonald, I. A., D. M. Hutchison, T. P. Forrest, V. D. Bokkenheuser, J. Winter, and L. V. Holdeman. 1983. Metabolism of primary bile acids by *Clostridium perfringens*. *J. Steroid Biochem.* **18**: 97–104.

66. Coleman, J. P., W. B. White, M. Lijewski, and P. B. Hylemon. 1988. Nucleotide sequence and regulation of a gene involved in bile acid 7-dehydroxylation by *Eubacterium sp.* strain VPI 12708. *J. Bacteriol.* **170**: 2070–2077.

67. Gopal-Srivastava, R., D. H. Mallonee, W. B. White, and P. B. Hylemon. 1990. Multiple copies of a bile acid-inducible gene in *Eubacterium sp.* strain VPI 12708. *J. Bacteriol.* **172**: 4420–4426.

68. Hylemon, P. B., and T. L. Glass. 1983. Bile acid and cholesterol metabolism. *In Human Intestinal Microflora in Health and Disease*. D.J. Heneges, editor. Academic Press, New York. 189–213.

69. Baron, S. F., C. V. Franklund, and P. B. Hylemon. 1991. Cloning, sequencing, and expression of the gene coding for bile acid 7α-hydroxysteroid dehydrogenase from *Eubacterium sp.* strain VPI 12708. *J. Bacteriol.* **173**: 4558–4569.

70. Coleman, J. P., L. L. Hudson, and M. J. Adams. 1994. Characterization and regulation of the NADP-linked 7α-hydroxy-steroid dehydrogenase gene from *Clostridium sordellii*. *J. Bacteriol.* **176**: 4865–4874.

71. Bennett, M. J., S. L. McKnight, and J. P. Coleman. 2003. Cloning and characterization of the NAD-dependent 7α-hydroxysteroid dehydrogenase from *Bacteroides fragilis*. *Curr. Microbiol.* **47**: 475–484.

72. Warchol, M., L. Car, J. P. Grill, and F. Schneider. 2003. Metabolic changes in *Clostridium absonum* ATCC 27555 accompanying induction of epimerization of a primary bile acid. *Curr. Microbiol.* **47**: 425–430.

73. Yoshimoto, T., H. Higashi, A. Kanatani, X. Sheng Lin, H. Nagai, H. Oyama, K. Kurazono, and D. Tsuru. 1991. Cloning and sequencing of the 7α-hydroxysteroid dehydrogenase gene from *Escherichia coli* HB101 and characterization of the expressed enzyme. *J. Bacteriol.* **173**: 2173–2179.

74. Sherod, J. A., and P. B. Hylemon. 1977. Partial purification and characterization of NAD-dependent 7α-hydroxysteroid dehydrogenase from *Bacteroides thetaiotaomicron*. *Biochim. Biophys. Acta.* **486**: 351–358.

75. Sutherland, J. D., and C. N. Williams. 1985. Bile acid induction of 7α- and 7β-hydroxysteroid dehydrogenases in *Clostridium limosum*. *J. Lipid Res.* **26**: 344–350.

76. Macdonald, I. A., and P. D. Roach. 1981. Bile salt induction of 7α- and 7β-hydroxysteroid dehydrogenases in *Clostridium absonum*. *Biochim. Biophys. Acta.* **665**: 262–269.

77. Macdonald, I. A., D. M. Hutchison, and T. P. Forrest. 1981. Formation of urso- and ursodeoxycholic acids from primary bile acids by *Clostridium absonum*. *J. Lipid Res.* **22**: 458–466.

78. Edenharder, R., and T. Knaflic. 1981. Epimerization of chenodeoxycholic acid to ursodeoxycholic acid by human intestinal lecithinase-lipase negative clostridia. *J. Lipid Res.* **22**: 652–658.

79. Hylemon, P. B., and J. A. Sherrod. 1975. Multiple forms of a 7α-hydroxysteroid dehydrogenase in selected strains of *Bacteroides fragilis*. *J. Bacteriol.* **122**: 418–424.

80. Franklund, C. V., P. de Prada, and P. B. Hylemon. 1990. Purification and characterization of a microbial, NADP-dependent bile acid 7α-hydroxysteroid dehydrogenase. *J. Biol. Chem.* **265**: 9842–9849.

81. Medici, A., P. Pedrini, E. Bianchini, G. Fantin, A. Guerrini, B. Natalini, and R. Pellicciari. 2002. 7α-OH epimerization of bile acids via oxido-reduction with *Xanthomonas maltophilia*. *Steroids.* **67**: 51–56.

82. Macdonald, I. A., B. A. White, and P. B. Hylemon. 1983. Separation of 7α- and 7β-hydroxysteroid dehydrogenase activities from *Clostridium absonum* ATCC# 27555 and cellular response of this organism to bile acid inducers. *J. Lipid Res.* **24**: 1119–1126.

83. Sutherland, J. D., C. N. Williams, D. M. Hutchison, and L. V. Holdeman. 1987. Oxidation of primary bile acids by a 7α-hydroxysteroid dehydrogenase elaborating *Clostridium bifermen-tans* soil isolate. *Can. J. Microbiol.* **33**: 663–669.

84. Hirano, S., N. Masuda, H. Oda, and H. Mukai. 1981. Transformation of bile acids by *Clostridium perfringens*. *Appl. Environ. Microbiol.* **42**: 394–399.

85. Prabha, V., M. Gupta, and K. G. Gupta. 1989. Kinetic properties of 7α-hydroxysteroid dehydrogenase from *Escherichia coli* 080. *Can. J. Microbiol.* **35**: 1076–1080.

86. Sutherland, J. D., L. V. Holdeman, C. N. Williams, and I. A. Macdonald. 1984. Formation of urso- and ursodeoxy-cholic acids from primary bile acids by a *Clostridium limosum* soil isolate. *J. Lipid Res.* **25**: 1084–1089.

87. Tanaka, N., T. Nonaka, T. Tanabe, T. Yoshimoto, D. Tsuru, and Y. Mitsui. 1996. Crystal structures of the binary and ternary complexes of 7α-hydroxysteroid dehydrogenase from *Escherichia coli*. *Biochemistry.* **35**: 7715–7730.

88. Tanabe, T., N. Tanaka, K. Uchikawa, T. Kabashima, K. Ito, T. Nonaka, Y. Mitsui, M. Tsuru, and T. Yoshimoto. 1998. Roles of the Ser146, Tyr159, and Lys163 residues in the catalytic action of 7alpha-hydroxysteroid dehydrogenase from *Escherichia coli*. *J. Biochem. (Tokyo).* **124**: 634–641.

89. Harris, J. N., and P. B. Hylemon. 1978. Partial purification and characterization of NADP-dependent 12α-hydroxysteroid dehydrogenase from *Clostridium leptum*. *Biochim. Biophys. Acta.* **528**: 148–157.

90. Macdonald, I. A., J. F. Jellet, and D. E. Mahony. 1979. 12α-Hydroxysteroid dehydrogenase from *Clostridium* group P strain C48-50 ATCC 29733: partial purification and characterization. *J. Lipid Res.* **37**: 992–1000.

91. Edenharder, R., and J. Schneider. 1985. 12β-Dehydrogenation of bile acids by *Clostridium paraputrificum*, *C. tertium*, and *C. difficile* and epimerization at carbon-12 of deoxycholic acid by cocultivation with 12α-dehydrogenating *Eubacterium lentum*. *Appl. Environ. Microbiol.* **49**: 964–968.

92. Edenharder, R., and A. Pfützner. 1988. Characterization of NADP-dependent 12β-hydroxysteroid dehydrogenase from *Clostridium paraputrificum*. *Biochim. Biophys. Acta.* **962**: 362–370.

93. Reddy, B. S. 1981. Dietary fat and its relationship to large bowel cancer. *Cancer Res.* **41**: 3700–3705.

94. Ali, S. S., A. Kuksis, and J. M. R. Beveridge. 1966. Excretion of bile acids by three men on a fat-free diet. *Can. J. Biochem.* **44**: 957–969.

95. Begley, M., C. G. M. Gahan, and C. Hill. 2005. The interaction between bacteria and bile. *FEMS Microbiol. Rev.* **29**: 625–651.

96. Armstrong, M. J., and M. C. Carey. 1982. The hydrophobic-hydrophilic balance of bile salts. Inverse correlation between reverse-phase high performance liquid chromatographic mobilities and micellar cholesterol-solubilizing capacities. *J. Lipid Res.* **23**: 70–80.

97. Heuman, D. M., W. M. Pandak, P. B. Hylemon, and Z. R. Vlahcevic. 1991. Conjugates of ursodeoxycholate protect against cytotoxicity of more hydrophobic bile salts: in vitro studies in rat hepatocytes and human erythrocytes. *Hepatology.* **14**: 920–926.

98. Bayerdörffer, E., G. A. Mannes, T. Ochsenkühn, P. Dirschedl, B. Wiebecke, and G. Paumgartner. 1995. Unconjugated secondary bile acids in the serum of patients with colorectal adenomas. *Gut.* **36**: 268–273.

99. Salen, G., G. S. Tint, B. Eliav, N. Deering, and E. H. Mosbach. 1974. Increased formation of ursodeoxycholic acid in patients treated with chenodeoxycholic acid. *J. Clin. Invest.* **53**: 612–621.

100. Hofmann, A. F. 1995. Bile acids as drugs: principles, mechanisms of action and formulations. *Ital. J. Gastroenterol.* **27**: 106–113.

101. Im, E., and J. D. Martinez. 2004. Ursodeoxycholic acid (UDCA) can inhibit deoxycholic acid (DCA)-induced apoptosis via modulation of EGFR/Raf-1/ERK signaling in human colon cancer cells. *J. Nutr.* **134**: 483–486.

102. Stellwag, E. J., and P. B. Hylemon. 1978. Characterization of 7α-dehydroxylase in *Clostridium leptum*. *Am. J. Clin. Nutr.* **31** (Suppl. 10): 243–247.

103. Ferrari, A., C. Scolastino, and L. Baretta. 1977. Activity on bile salts of a *Clostridium bifermentans* cell-free extract. *FEBS Lett.* **75**: 166–168.

104. Wells, J. E., F. Berr, L. A. Thomas, R. H. Dowling, and P. B. Hylemon. 2000. Isolation and characterization of cholic acid 7α-dehydroxylating fecal bacteria from cholesterol gallstone patients. *J. Hepatol.* **32**: 4–10.

105. Hirano, S., R. Nakama, M. Tamaki, N. Masuda, and H. Oda. 1981. Isolation and characterization of thirteen intestinal microorganisms capable of 7α-dehydroxylating bile acids. *Appl. Environ. Microbiol.* **41**: 737–745.

106. Wells, J. E., K. B. Williams, T. R. Whitehead, D. M. Heuman, and P. B. Hylemon. 2003. Development and application of a polymerase chain reaction assay for the detection and enumeration of bile acid 7α-dehydroxylating bacteria in human feces. *Clin. Chim. Acta.* **331**: 127–134.

107. Kitahara, M., F. Takamine, T. Imamura, and Y. Benno. 2000. Assignment of *Eubacterium* sp. VPI 12708 and related strains with high bile acid 7α-dehydroxylating activity to *Clostridium scindens* and proposal of *Clostridium hylemonae* sp. nov., isolated from human feces. *Int. J. Syst. Evol. Microbiol.* **50**: 971–978.

108. Kitahara, M., F. Takamine, T. Imamura, and Y. Benno. 2001. *Clostridium hiranonis* sp. nov., a human intestinal bacterium with bile acid 7α-dehydroxylating activity. *Int. J. Syst. Evol. Microbiol.* **51**: 39–44.

109. Stellwag, E. J., and P. B. Hylemon. 1979. 7α-Dehydroxylation of cholic acid and chenodeoxycholic acid by *Clostridium leptum*. *J. Lipid Res.* **20**: 325–333.

110. Batta, A. K., G. Salen, R. Arora, S. Shefer, M. Batta, and A. Person. 1990. Side chain conjugation prevents bacterial 7α-dehydroxylation of bile acids. *J. Biol. Chem.* **265**: 10925–10928.

111. White, B. A., R. H. Lipsky, R. J. Fricke, and P. B. Hylemon. 1980. Bile acid induction specificity of 7α-dehydroxylase activity in an intestinal *Eubacterium* sp. *Steroids.* **35**: 103–109.

112. Schmassmann, A., H. F. Fehr, J. Locher, J. Lillienau, C. D. Schteinigart, S. S. Rossi, and A. F. Hofmann. 1993. Cholylsarcosine, a new bile acid analogue: metabolism and effect on biliary secretion in humans. *Gastroenterology.* **104**: 1171–1181.

113. White, B. A., R. J. Fricke, and P. B. Hylemon. 1982. 7β-Dehydroxylation of ursodeoxycholic acid by whole cells and cell extracts of the intestinal anaerobic bacterium, *Eubacterium* species VPI 12708. *J. Lipid Res.* **23**: 145–153.

114. Samuelsson, B. 1960. On the mechanism of the biological formation of deoxycholic acid from cholic acid. *J. Biol. Chem.* **235**: 361–366.

115. Lindstedt, S., and B. Samuelsson. 1959. Bile acids and steroids. LXXXIII. On the inter-conversion of cholic and deoxycholic acid in the rat. *J. Biol. Chem.* **234**: 2026–2030.

116. Bergstrom, S., S. Lindstedt, and B. Samuelsson. 1959. Bile acids and steroids. LXXXII. On the mechanism of deoxycholic acid formation in the rabbit. *J. Biol. Chem.* **234**: 2022–2025.

117. Björkhem, I., K. Einarsson, P. Melone, and P. B. Hylemon. 1989. Mechanism of intestinal formation of deoxycholic acid from cholic acid in humans: evidence for a 3-oxo-Δ⁴-steroid intermediate. *J. Lipid Res.* **30**: 1033–1039.

118. Hylemon, P. B., P. D. Melone, C. V. Franklund, E. Lund, and I. Björkhem. 1991. Mechanism of intestinal 7α-dehydroxylation of cholic acid: evidence that allo-deoxycholic acid is an inducible side-product. *J. Lipid Res.* **32**: 89–95.

119. Paone, D. A. M., and P. B. Hylemon. 1984. HPLC purification and preparation of antibodies to cholic acid-inducible polypeptides from *Eubacterium* sp. VPI 12708. *J. Lipid Res.* **25**: 1343–1349.

120. White, B. A., A. F. Cacciapuoti, R. J. Fricke, T. R. Whitehead, E. H. Mosbach, and P. B. Hylemon. 1981. Cofactor requirements for 7α-dehydroxylation of cholic and chenodeoxycholic acid in cell extracts of the intestinal anaerobic bacterium, *Eubacterium* species VPI 12708. *J. Lipid Res.* **22**: 891–898.

121. Franklund, C. V., S. F. Baron, and P. B. Hylemon. 1993. Characterization of the *baiH* gene encoding a bile acid-inducible NADH:flavin oxidoreductase from *Eubacterium* sp. strain VPI 12708. *J. Bacteriol.* **175**: 3002–3012.

122. White, W. B., J. P. Coleman, and P. B. Hylemon. 1988. Molecular cloning of a gene encoding a 45,000-dalton polypeptide associated with bile acid 7-dehydroxylation in *Eubacterium* sp. strain VPI 12708. *J. Bacteriol.* **170**: 611–616.

123. Coleman, J. P., W. B. White, and P. B. Hylemon. 1987. Molecular cloning of bile acid 7-dehydroxylase from *Eubacterium* sp. strain VPI 12708. *J. Bacteriol.* **169**: 1516–1521.

124. White, W. B., C. V. Franklund, J. P. Coleman, and P. B. Hylemon. 1988. Evidence for a multigene family involved in bile acid 7-dehydroxylation in *Eubacterium* sp. strain VPI 12708. *J. Bacteriol.* **170**: 4555–4561.

125. Mallonee, D. H., J. L. Adams, and P. B. Hylemon. 1992. The bile acid-inducible *baiB* gene from *Eubacterium* sp. strain VPI 12708 encodes a bile acid-coenzyme A ligase. *J. Bacteriol.* **174**: 2065–2071.

126. Mallonee, D. H., and P. B. Hylemon. 1996. Sequencing and expression of a gene encoding a bile acid transporter from *Eubacterium* sp. strain VPI 12708. *J. Bacteriol.* **178**: 7053–7058.

127. Mallonee, D. H., and P. B. Hylemon. 1999. Use of a short A/T-rich

cassette for enhanced expression of cloned genes in *Escherichia coli*. Mol. Biotechnol. **11**: 27–35.

128. Dawson, J. A., D. H. Mallonee, I. Björkem, and P. B. Hylemon. 1996. Expression and characterization of a C<sub>24</sub> bile acid 7α-dehydratase from *Eubacterium* sp. strain VPI 12708 in *Escherichia coli*. *J. Lipid Res.* **37**: 1258–1267.

129. Baron, S. F., and P. B. Hylemon. 1995. Expression of the bile acid-inducible NADH:flavin oxidoreductase gene of *Eubacterium* sp. VPI 12708 in *Escherichia coli*. *Biochim. Biophys. Acta.* **1249**: 145–154.

130. Ye, H. Q., D. H. Mallonee, J. E. Wells, I. Björkhem, and P. B. Hylemon. 1999. The bile acid-inducible *baiF* gene from *Eubacterium* sp. strain VPI 12708 encodes a bile acid-coenzyme A hydrolase. *J. Lipid Res.* **40**: 17–23.

131. Heider, J. 2001. A new family of CoA-transferases. *FEBS Lett.* **509**: 345–349.

132. Berr, F., G. A. Kullak-Ublick, G. Paumgartner, W. Munzig, and P. B. Hylemon. 1996. 7 Alpha-dehydroxylating bacteria enhance deoxycholic acid input and cholesterol saturation of bile in patients with gallstones. *Gastroenterology.* **111**: 1611–1620.

133. Dowling, R. H., M. J. Veysey, S. P. Pereira, S. H. Hussaini, L. A. Thomas, J. A. Wass, and G. M. Murphy. 1997. Role of intestinal transit in the pathogenesis of gallbladder stones. *Can. J. Gastroenterol.* **11**: 57–64.

134. Thomas, L. A., M. J. Veysey, G. French, P. B. Hylemon, G. M. Murphy, and R. H. Dowling. 2001. Bile acid metabolism by fresh human colonic contents: a comparison of caecal versus faecal samples. *Gut.* **49**: 835–842.

135. Low-Beer, T. S., and S. Nutter. 1978. Colonic bacterial activity, biliary cholesterol saturation, and pathogenesis of gallstones. *Lancet.* **2**: 1063–1065.

136. Marcus, S. N., and K. W. Heaton. 1988. Deoxycholic acid and the pathogenesis of gall stones. *Gut.* **29**: 522–533.

137. Hussaini, S. H., S. P. Pereira, G. M. Murphy, and R. H. Dowling. 1995. Deoxycholic acid influences cholesterol solubilization and microcrystal nucleation time in gallbladder bile. *Hepatology.* **22**: 1735–1744.

138. Bernstein, H., C. Bernstein, C. M. Payne, K. Dvorakova, and H. Garewal. 2005. Bile acids as carcinogens in human gastrointestinal cancers. *Mutat. Res.* **589**: 47–65.

139. Reddy, B. S., T. Narasawa, J. H. Weisburger, and E. L. Wynder. 1976. Promoting effect of sodium deoxycholate on colon adenocarcinomas in germfree rats. *J. Natl. Cancer Inst.* **56**: 441–442.

140. Pereira, M. A., W. Wang, P. M. Kramer, and L. Tao. 2004. DNA hypomethylation induced by non-genotoxic carcinogens in mouse and rat colon. *Cancer Lett.* **212**: 145–151.

141. Narisawa, T., N. E. Magadia, J. H. Weisburger, and E. L. Wynder. 1974. Promoting effect of bile acids on colon carcinogenesis after intrarectal instillation of N-methyl-N'-nitro-N-nitrosoguanidine in rats. *J. Natl. Cancer Inst.* **53**: 1093–1097.

142. Zusman, I., M. Chevion, and N. Kitrosski. 1992. Effects of N'-methyl-N'-nitro-N-nitrosoguanidine and deoxycholic acid on the content of free radicals in rat serum. *Exp. Toxicol. Pathol.* **44**: 187–189.

143. Bayerdörf fer, E., G. A. Mannes, W. O. Richter, T. Ochsenkühn, B. Wiebecke, W. Kopcke, and G. Paumgartner. 1993. Increased serum deoxycholic acid levels in men with colorectal adenomas. *Gastroenterology.* **104**: 145–151.

144. Zhu, Y., P. Hua, S. Rafiq, E. J. Waffner, M. E. Duffey, and P. Lance. 2002. Ca<sup>2+</sup> and PKC-dependent stimulation of PGE2 synthesis by deoxycholic acid in human colonic fibroblasts. *Am. J. Physiol. Gastrointest. Liver Physiol.* **283**: G503–G510.

145. Rao, Y. P., E. J. Studer, R. T. Stravitz, S. Gupta, L. Qiao, P. Dent, and P. B. Hylemon. 2002. Activation of the Raf-1/MEK/ERK cascade by bile acids occurs via the epidermal growth factor receptor in primary rat hepatocytes. *Hepatology.* **35**: 307–314.

146. Qiao, L., E. Studer, K. Leach, R. McKinstry, S. Gupta, R. Decker, R. Kukreja, K. Valerie, P. Nagarkatti, W. El Deiry, et al. 2001. Deoxycholic acid (DCA) causes ligand-independent activation of epidermal growth factor receptor (EGFR) and FAS receptor in primary hepatocytes: inhibition of EGFR/mitogen-activated protein kinase-signaling module enhances DCA-induced apoptosis. *Mol. Biol. Cell.* **12**: 2629–2645.

147. Pai, R., A. S. Tarnawski, and T. Tran. 2004. Deoxycholic acid activates beta-catenin signaling pathway and increases colon cell cancer growth and invasiveness. *Mol. Biol. Cell.* **15**: 2156–2163.

148. Gupta, S., R. Natarajan, S. G. Payne, E. J. Studer, S. Spiegel, P.


Dent, and P. B. Hylemon. 2004. Deoxycholic acid activates the c-Jun N-terminal kinase pathway via FAS receptor activation in primary hepatocytes. Role of acidic sphingomyelinase-mediated ceramide generation in FAS receptor activation. *J. Biol. Chem.* **279**: 5821–5828.

149. Makishima, M., T. T. Lu, W. Xie, G. K. Whitfield, H. Domoto, R. M. Evans, M. R. Haussler, and D. J. Mangelsdorf. 2002. Vitamin D receptor as an intestinal bile acid sensor. *Science.* **296**: 1313–1316.

150. Adachi, R., Y. Honma, H. Masuno, K. Kawana, I. Shimomura, S. Yamada, and M. Makishima. 2005. Selective activation of vitamin D receptor by lithocholic acid acetate, a bile acid derivative. *J. Lipid Res.* **46**: 46–57.

151. Dar, S. A., J. G. Kuenen, and G. Muyzer. 2005. Nested PCR-denaturing gradient gel electrophoresis approach to determine the diversity of sulfate-reducing bacteria in complex microbial communities. *Appl. Environ. Microbiol.* **71**: 2325–2330.

152. Mai, V., and J. G. Morris, Jr. 2004. Colonic bacterial flora: changing understandings in the molecular age. *J. Nutr.* **134**: 459–464.

153. Kurdi, P., H. Tanaka, H. W. Van Veen, K. Asano, F. Tomita, and A. Yokota. 2003. Cholic acid accumulation and its diminution by short-chain fatty acids in bifidobacteria. *Microbiology.* **149**: 2031–2037.

154. Kurdi, P., H. W. van Veen, H. Tanaka, I. Mierau, W. N. Konings, G. W. Tannock, F. Tomita, and A. Yokota. 2000. Cholic acid is accumulated spontaneously, driven by membrane ΔpH, in many lactobacilli. *J. Bacteriol.* **182**: 6525–6528.

155. Chandramouli, V., K. Kailasapathy, P. Peiris, and M. Jones. 2004. An improved method of microencapsulation and its evaluation to protect *Lactobacillus* spp. in simulated gastric conditions. *J. Microbiol. Methods.* **56**: 27–35.

156. Guerin, D., J. C. Vuillemaud, and M. Subirade. 2003. Protection of bifidobacteria encapsulated in polysaccharide-protein gel beads against gastric juice and bile. *J. Food Prot.* **66**: 2076–2084.

157. De Smet, I., P. De Boever, and W. Verstraete. 1998. Cholesterol lowering in pigs through enhanced bacterial bile salt hydrolase activity. *Br. J. Nutr.* **79**: 185–194.

158. Hepner, G., R. Fried, S. St. Jeor, L. Fusetti, and R. Morin. 1979. Hypocholesteremic effect of yogurt and milk. *Am. J. Clin. Nutr.* **32**: 19–24.

159. Hlivak, P., J. Odraska, M. Ferencik, L. Ebringer, E. Jahnova, and Z. Mikes. 2005. One-year application of probiotic strain *Enterococcus faecium* M-74 decreases serum cholesterol levels. *Bratisl. Lek. Listy.* **106**: 67–72.

160. Hofmann, A. F., J. Sjövall, G. Kurz, A. Radominska, C. D. Schteingart, G. S. Tint, Z. R. Vlahcevic, and K. D. Setchell. 1992. A proposed nomenclature for bile acids. *J. Lipid Res.* **33**: 599–604.

161. Reddy, B. S., and E. L. Wynder. 1977. Metabolic epidemiology of colon cancer: fecal bile acids and neutral sterols in colon cancer patients and patients with adenomatous polyps. *Cancer.* **39**: 2533–2539.

162. Paumgartner, G., and A. Stiehl. 1976. Falk Symposium 24: Bile Acid Metabolism in Health and Disease. University Park Press, Baltimore, MD. 167–172.

163. Kawamoto, K., I. Horibe, and K. Uchida. 1989. Purification and characterization of a new hydrolase for conjugated bile acids, chenodeoxycholytaurine hydrolase, from *Bacteroides vulgatus*. *J. Biochem. (Tokyo).* **106**: 1049–1053.

164. Sue, D., K. J. Boor, and M. Wiedmann. 2003. σ<sup>B</sup>-dependent expression patterns of compatible solute transporter genes *opuCA* and *Imo1421* and the conjugated bile salt hydrolase gene *bsh* in *Listeria monocytogenes*. *Microbiology.* **149**: 3247–3256.

165. Hirano, S., and N. Masuda. 1982. Characterization of NADP-dependent 7β-hydroxysteroid dehydrogenases from *Peptostreptococcus productus* and *Eubacterium aerofaciens*. *Appl. Environ. Microbiol.* **43**: 1057–1063.

166. Masuda, N., H. Oda, and H. Tanaka. 1983. Purification and characterization of NADP-dependent 7β-hydroxysteroid dehydrogenase from *Peptostreptococcus productus* strain b-52. *Biochim. Biophys. Acta.* **755**: 65–69.

167. Macdonald, I. A., C. N. Williams, D. E. Mahony, and W. M. Christie. 1975. NAD- and NADP-dependent 7α-hydroxysteroid dehydrogenase from *Bacteroides fragilis*. *Biochim. Biophys. Acta.* **384**: 12–24.

168. Hylemon, P. B., and J. Harder. 1999. Biotransformation of monoterpenes, bile acids, and other isoprenoids in anaerobic ecosystems. *FEMS Microbiol. Rev.* **22**: 475–488.
