
# Biotin Synthesis in Plants. The First Committed Step of the Pathway Is Catalyzed by a Cytosolic 7-Keto-8-Aminopelargonic Acid Synthase

Violaine Pinon, Stéphane Ravanel, Roland Douce, and Claude Alban*

Laboratoire de Physiologie Cellulaire Végétale, Centre National de la Recherche Scientifique/Institut National de la Recherche Agronomique (Jeune Equipe INRA)/Université Joseph Fourier/Commissariat à l’Energie Atomique-Grenoble, F–38054 Grenoble cedex 9, France

Biochemical and molecular characterization of the biotin biosynthetic pathway in plants has dealt primarily with biotin synthase. This enzyme catalyzing the last step of the pathway is localized in mitochondria. Other enzymes of the pathway are however largely unknown. In this study, a genomic-based approach allowed us to clone an Arabidopsis (*Arabidopsis thaliana*) cDNA coding 7-keto-8-aminopelargonic acid (KAPA) synthase, the first committed enzyme of the biotin synthesis pathway, which we named *AtbioF*. The function of the enzyme was demonstrated by functional complementation of an *Escherichia coli* mutant deficient in KAPA synthase reaction, and by measuring in vitro activity. Overproduction and purification of recombinant AtbioF protein enabled a thorough characterization of the kinetic properties of the enzyme and a spectroscopic study of the enzyme interaction with its substrates and product. This is the first characterization of a KAPA synthase reaction in eukaryotes. Finally, both green fluorescent protein-targeting experiments and western-blot analyses showed that the Arabidopsis KAPA synthase is present in cytosol, thus revealing a unique compartmentation of the plant biotin synthesis, split between cytosol and mitochondria. The significance of the complex compartmentation of biotin synthesis and utilization in the plant cell and its potential importance in the regulation of biotin metabolism are also discussed.

Biotin is an essential, water-soluble vitamin found in all living cells (Dakshinamurti and Cauhan, 1989). Plants, like most microorganisms, have the ability to synthesize biotin. In contrast, other multicellular eukaryotic organisms are biotin auxotroph. Biotin acts as a cofactor for a set of enzymes that catalyze carboxylation, decarboxylation, and transcarboxylation reactions in a number of crucial metabolic processes (Knowles, 1989).

Biotin biosynthesis has been well characterized in bacteria such as *Escherichia coli*, *Bacillus subtilis*, and *Bacillus sphaericus*, by combined biochemical and genetic studies (for review, see Marquet et al., 2001). In all known microbes, the cofactor is synthesized from pimeloyl-CoA through four enzymatic steps comprising 7-keto-8-aminopelargonic acid (KAPA) synthase, 7,8-diaminopelargonic acid (DAPA) aminotransferase, dethiobiotin synthase, and biotin synthase coded by *bioF*, *bioA*, *bioD*, and *bioB* genes, respectively. Enzymes required for biotin synthesis in *E. coli* and *B. sphaericus* have been purified and their activities characterized in vitro (for review, see Streit and Entcheva, 2003). KAPA synthase, the first enzyme of this pathway, catalyzes the decarboxylative condensation of pimeloyl-CoA and L-Ala to produce KAPA, CoASH, and carbon dioxide (Fig. 1). The structure and reaction mechanism of KAPA synthase places it in the subfamily of α-oxoamine synthases, a small group of pyridoxal 5′-phosphate (PLP)-dependent enzymes of the α-family (Ploux and Marquet, 1996; Alexeev et al., 1998; Webster et al., 2000).

In plants, the biosynthetic pathway beginning with pimeloyl-CoA appears to follow the same pattern as identified for bacteria. This was deduced from measuring pools of the different intermediates of biotin biosynthesis, employing lavender (*Lavandula vera*) cell cultures treated with radiolabeled precursors (Baldet et al., 1993b). Initial genetic information on biotin synthesis in higher plants came from analysis of the *bio1* biotin auxotroph of Arabidopsis (*Arabidopsis thaliana*; Schneider et al., 1989). Plant growth was rescued by biotin, dethiobiotin, or DAPA, but not KAPA supply, or by genetic complementation by *E. coli bioA* gene coding DAPA aminotransferase, demonstrating that mutant plants are defective in this enzyme (Shellhammer and Meinke, 1990; Patton et al., 1996b). More recently, a second biotin auxotroph of Arabidopsis has been identified. Arrested embryos from this *bio2* mutant are defective in the final step of biotin synthesis, i.e. the conversion of dethiobiotin to biotin (Patton et al.,

* Corresponding author; e-mail claude.alban@cea.fr; fax (33)–438–78–50–91.
The author responsible for distribution of materials integral to the findings presented in this article in accordance with the policy described in the Instructions for Authors (www.plantphysiol.org) is: Claude Alban (claude.alban@cea.fr).
Article, publication date, and citation information can be found at www.plantphysiol.org/cgi/doi/10.1104/pp.105.070144.

1666 **Plant Physiology**, December 2005, Vol. 139, pp. 1666–1676, www.plantphysiol.org © 2005 American Society of Plant Biologists Downloaded from www.plantphysiol.org on March 16, 2015 - Published by www.plant.org Copyright © 2005 American Society of Plant Biologists. All rights reserved.

Plant KAPA Synthase Is a Cytosolic Enzyme

1997; Picciocchi et al., 2001). Characterization of plant enzymes situated upstream of biotin synthase and elucidation of their subcellular distribution have not yet been reported.

In this study, we describe the identification, the cloning, and the functional characterization of a cDNA encoding KAPA synthase in Arabidopsis that we termed AtbioF. The recombinant protein was overexpressed in *E. coli* and purified, allowing a thorough characterization of the kinetic properties of the enzyme and a spectroscopic study of the enzyme interaction with its substrates and product. We also show that the plant KAPA synthase is located in the cytosol, thus revealing a complex compartmentation of biotin biosynthesis in plants.

**RESULTS**

**Identification and Cloning of the bioF Homolog in Arabidopsis**

Searches of the Arabidopsis genome database detected a single gene (At5g04620) encoding a predicted protein with 27% to 32% identity (44%–50% similarity) to the protein sequences of *E. coli* and *B. sphaericus* KAPA synthase (bioF), respectively. The product of this gene annotated as an aminotransferase class I and II family protein (The Arabidopsis Information Resource annotation) and as an 8-amino-7-oxononanoate synthase-like protein (Munich Information Center for Protein Sequences annotation) was designated AtbioF.

**Figure 2. Alignment of the deduced amino acid sequence of AtbioF with bioF proteins of *B. sphaericus* (GenBank accession no. AAA22271) and *E. coli* (AAC73863). The alignment was generated with the ClustalW program (www.expasy.ch). Identical amino acids are colored in red and similar amino acids in green. Dashes are gaps introduced to maximize the alignment. Stars indicate conserved residues implicated in catalysis and in substrates, or PLP-cofactor binding in the bacterial enzymes.**

| A. thaliana | MADHSWDKTVEEAVNVLESRQILRSLRPICMSRQNEEEIVKS RANGGDGYEVFDGLCQWD |
| B. sphaericus | MNDR-----FRRELQVIEEQGLTRKLR--LFSTGNES EVVMNG--------------- |
| E. coli | MSWQ-----EKINAALDARRAADALRRYPVAQGAGRWLVADD------------------- |

| A. thaliana | RTSVEVSVSIPTFQKWLHDEPSNGEEIFSGDALAECRKGRFKKL LLFSGNDYLGLSSHP |
| B. sphaericus | ------------------------------------------------------KKFL LFSSNNYLGLATDSR |
| E. coli | ------------------------------------------------------RQYLN FSSNDYLGLSHHPQ |

| A. thaliana | ISNAAANAVKEYMGPKGSALICGYTTYHRLLESSLAQLKKEDCLVCPTGFAANMAAMV |
| B. sphaericus | LKKKATEGISKYGAGGSRLLTTGNFDIHEQLESEIADFKTEAAIVFSSGYLAN----V |
| E. coli | IIRAWQQGAEQFGIGSGGSGHVS GYSVVHQALEEELAEWLGYSRALLFISGFAAN------- |

| A. thaliana | AIGSVASLLAASGKPLKNEKVAIFSDALNHASIIDGVRLAERQGNVEVFVYRHCDMYHLN |
| B. sphaericus | GV--ISSVMKAGDT--------IFSDAWNHASIIDGCRLS----KAKTIVYEHAMVDLE |
| E. coli | -QAVIAAMMAKERD--------IAADRLSHASLLEAASLS----PSQLRRFAHNDVTHLA |

| A. thaliana | SLLSNCKMK-RKVVTDSLFSMDGDFAPMEELSRLKKYGFLLVIDDAHGT FVCGENGGG |
| B. sphaericus | RKLRQSHGDGLKFIVTDGVFSMDGDIAPLPKIVELAKEYKAYIMDDAHATGVLGNDGCG |
| E. coli | RLLASPCPG-QQMVVTEGVFSMDGDSAPLAEIQQTQQHNGWLMVDDAHGTGVIGEQGRG |

| A. thaliana | VAEEFNCEADV DLCVGTLSKAAGCHGGFIACSKWKQLIQSRGRSFIFSTAIPVPMAAAA |
| B. sphaericus | TADYFGLKDEIDFTVGTLSKAIGAEGGFVSTSIAKNYLLNNARSFIFQTALSPSAIEAA |
| E. coli | --SCWLQKVKPELLVVTFGKGFGVSGAA VLCSSTVADYLLQFARHLIYSTSMPPAQAQAL |

| A. thaliana | YAAVVVARKEI--WRRKAIWERVKEFK---ELSGVDIS---SPIISLVVGNQE KALKASR |
| B. sphaericus | REGISIIQNEP--ERRKQLKNAQLRLKLEESGFVMKEGETPIISLIIGGSHEAMQFSA |
| E. coli | RASLAVIRSDEGDARREKLAALITRFRAGVQDLPFTLADSCSAIQPLIVGDN SRALQLAE |

| A. thaliana | YLLKSGFHVMAIRPPTVPNSCRLRVTLSAAHTTEDVKKLITALSSCLDFDNTATHIPSF |
| B. sphaericus | KLLDEGVFIPAIRPPTVPKGSSRLRITVMATHTIEQLDMVISKIK---KIGKEMGIV--- |
| E. coli | KLRQQGCWVTAIRPPTVPAGTARLRLTLTA AHEMQDIRLLEVLH------GNG--------- |

| A. thaliana | LFPKL |
| B. sphaericus | ----- |
| E. coli | ----- |

Plant Physiol. Vol. 139, 2005
Downloaded from www.plantphysiol.org on March 16, 2015 - Published by www.plant.org
Copyright © 2005 American Society of Plant Biologists. All rights reserved.

Pinon et al.

The corresponding cDNA was cloned by PCR amplification of a whole-plant Arabidopsis cDNA library. Sequencing and analysis of this cDNA indicated that it contains a 1,428-bp open reading frame encoding a 476-amino acid protein for a predicted molecular mass of 52,171 D. This sequence is consistent with that predicted from gene model At5g04620.2 presented at The Arabidopsis Information Resource Web site (http://Arabidopsis.org/) and with the sequence of a full-length cDNA available on Arabidopsis databases (GenBank accession no. BX830281). Alignment of the Arabidopsis deduced protein sequence with that of prokaryotes revealed two features (Fig. 2). First is the strict conservation of residues inferred to be crucial for substrate binding and catalysis, from crystallographic and mechanistic studies of the *E. coli* and *B. sphaericus* enzymes (Ploux and Marquet, 1996; Alexeev et al., 1998; Webster et al., 2000). These residues are Arg-24, Asn-110, His-210, and Lys-319, numbered according to the Arabidopsis sequence, Lys-319 being the PLP cofactor-binding site. The second feature is the presence of an insertion of about 60 residues in the N-terminal region of the plant sequence compared to its bacterial counterparts (Fig. 2). One should also note the report on Arabidopsis databases of a splice variant of the At5g04620 gene (At5g04620.1). This cDNA, isolated at the RIKEN Genomic Sciences Center (GenBank accession no. AK119095), contains unspliced intron 1, which introduces stop codons in the 5′ coding region. As a result, this sequence is predicted to encode a protein of only 343 amino acids, the first initiating Met corresponding to Met-134 of the larger form. This putative N terminus truncated form retains intact PLP-binding domain but lacks regions conserved among all known KAPA synthases, particularly the essential amino acids Arg-24 and Asn-110 of complete AtbioF. Therefore, this shorter AtbioF form, if it is actually produced, is probably catalytically inactive.

**AtbioF cDNA Complements an *E. coli* KAPA Synthase Mutant**

To test whether AtbioF is a functional enzyme, the full-length isolated sequence was cloned into *E. coli* expression vector pTrc99, which allows low to moderate levels of protein production. This construct was used to transform an *E. coli bioF−* mutant (strain bioF110), which lacks KAPA synthase activity and shows no growth on minimal medium unless biotin or a vitamer downstream pimeloyl-CoA is supplied (Rolfe and Eisenberg, 1968). Expression of the AtbioF protein restored the ability of the mutant to grow well without biotin (Fig. 3). The growth of the complemented cells was similar to that of mutant cells supplemented with biotin. No complementation was seen with the vector alone (Fig. 3). Retransformation of the *E. coli* mutant with rescued plasmid containing the Arabidopsis cDNA from complemented cells restored

![Figure 3](#fig3)

Figure 3. Complementation of an *E. coli bioF−* mutant by an AtbioF construct. Cells of the mutant transformed with vector alone (v) or vector containing full-length Arabidopsis cDNA (At) were cultured at 37°C on plate for 2 d (A) or in liquid medium for 12 h (B) on synthetic minimal medium plus or minus 1 mM biotin.

biotin prototrophy, further confirming that complementation was due to the encoded plant protein (data not shown). Taken together, these data indicate that plant AtbioF has KAPA synthase activity.

**Overexpression and Purification of the AtbioF Protein**

To validate the results described above and to characterize further the plant KAPA synthase reaction, the complete AtbioF protein was overexpressed with an N-terminal His tag using the pET28b plasmid and the *E. coli* host strain Rosetta (DE3). When bacteria were grown under optimum overexpression conditions, AtbioF was strongly expressed (approximately 5%–10% of total protein) in soluble form (Fig. 4A). Then the AtbioF protein was purified to near homogeneity by standard nickel-affinity chromatography (Fig. 4B). Typically, 50 to 80 mg of pure enzyme was obtained from 2 L of cell culture. The apparent molecular mass of the native recombinant protein, as estimated by gel-filtration chromatography on a Sephacryl S200 column, was found to be about 50 kD, which is close to the mass of the enzyme subunit (Fig. 4, C and D). These results are consistent with a monomeric structure of the recombinant enzyme in solution, similar to that previously reported for KAPA synthase from *B. sphaericus* (Ploux and Marquet, 1992) but different from that of *E. coli*, which crystallizes as a symmetric homodimer (Alexeev et al., 1998).

Plant Physiol. Vol. 139, 2005
Downloaded from www.plantphysiol.org on March 16, 2015 - Published by www.plant.org
Copyright © 2005 American Society of Plant Biologists. All rights reserved.

Plant KAPA Synthase Is a Cytosolic Enzyme

Figure 4. Purification procedure of AtbioF and native molecular mass determination. A to C, Polypeptides were separated by SDS-PAGE and stained with Coomassie Brilliant Blue R250. Lanes M, Molecular mass markers. A, Documentation of AtbioF overexpression. Lanes 1 and 2, Soluble proteins (25 μg) from E. coli Rosetta cells harboring the pET-AtbioF construct grown in the absence or in the presence of IPTG, respectively. B, Documentation of AtbioF purification on nickel-nitrilotriacetic acid-agarose resin. Lane 1, Soluble proteins (25 μg) from E. coli Rosetta cells expressing AtbioF; lane 2, proteins eluted from the column (5 μg). C and D, Molecular mass estimation of native AtbioF by gel filtration. Purified protein was chromatographed onto a Sephacryl S200 column (1.6 × 60 cm; Pharmacia). Eluted fractions (1.5 mL) were analyzed by SDS-PAGE (C). Standard proteins for column calibration were as follows: 1, aldolase (160 kD); 2, bovine serum albumin (66 kD); 3, ovalbumin (43 kD); 4, chymotrypsinogen (25 kD); and 5, ribonuclease A (13.7 kD) (D). Ve, Elution volume; V0, void volume.

Kinetic and Spectral Properties of the Recombinant Enzyme

Purified AtbioF exhibited the characteristic yellow color observed for PLP-containing enzymes. UV-visible spectra exhibited peaks at 330 and 423 nm that could be assigned, respectively, to the noncoplanar and coplanar forms of the AtbioF-PLP internal aldimine complex, i.e. the Schiff base formed between PLP and a Lys residue in the protein (Webster et al., 2000; Fig. 5A). The stoichiometric ratio of PLP per enzyme subunit was determined by measuring the absorption of the cofactor at 390 nm released on exposure of the native holoenzyme to alkaline conditions (0.1 M NaOH; Leussing, 1986). By this method, the average PLP content of the holoenzyme was found to be 1 mol/mol of enzyme monomer.

To investigate KAPA synthase activity catalyzed by AtbioF in vitro, we employed a colorimetric assay, which relies on monitoring the release of CoASH from pimeloyl-CoA during the reaction using 5,5′-dithiobis(2-nitrobenzoic acid) (DTNB). This assay, formerly developed for measuring acetyl-CoA synthetase activity in the direction of acetate formation (Sanchez and Müller, 1996), was more sensitive and reliable than published KAPA synthase assay procedures for the E. coli and the B. sphaericus enzymes. Under optimized assay conditions, the pure overexpressed Arabidopsis enzyme exhibited a specific activity of 90 to 110 nmol CoASH released min⁻¹ mg⁻¹ protein. The reaction was strictly dependent upon the presence of L-Ala in the reaction medium, and, as expected from PLP/enzyme stoichiometry measurements, the purified holoenzyme was fully active even in the absence of exogenously added PLP. Identification of KAPA as a reaction product of the in vitro assay was confirmed by HPLC analysis on a ion-exchange column, by comparison of its retention time value with that of authentic KAPA sample (Fig. 6) and by its coelution with KAPA standard in coinjection experiments (data not shown).

The recombinant Arabidopsis KAPA synthase obeyed Michaelis-Menten kinetics with respect to pimeloyl-CoA and L-Ala (Table I). Kinetic constants obtained compare well with those determined for the corresponding B. sphaericus enzyme (Ploux and Marquet, 1996) but differ somewhat from those determined for the E. coli enzyme (Webster et al., 2000). Particularly, the Km determined for pimeloyl-CoA was much lower than that for the corresponding E. coli enzyme (1.6 μM compared to 25 μM), indicating that the plant enzyme had a higher specificity for pimeloyl-CoA.

KAPA synthase forms together with 5-aminolevulinate synthase, Ser palmitoyltransferase, and 2-amino-3-oxobutyrate CoA ligase a small subfamily of α-oxamine synthases. These PLP-dependent enzymes catalyze condensations between amino acids and carboxylic

Pinon et al.

Figure 5. Spectroscopic properties of AtbioF. Absorption spectra were acquired at 30°C and pH 7.5 in the presence of 36.8 μM recombinant enzyme. A, Absorption properties of purified enzyme. B to D, Changes in cofactor absorption spectra of AtbioF. B, Binding of L-Ala to AtbioF. The enzyme was titrated with L-Ala. Spectra were recorded after each addition of the titrant. The increase in \(A_{423}\) was then plotted versus the titrant concentration. Data were fitted (inset) to a hyperbolic saturation curve (Eq. 1). C, Time-course changes in absorption spectra of the external aldimine form of the enzyme induced by pimeloyl-CoA. Spectra were recorded immediately and then every minute after addition of 600 μM pimeloyl-CoA to the enzyme previously equilibrated with 10 mM L-Ala. D, Absorption properties of purified enzyme in the presence of 4 mM KAPA.

acid CoA thioesters with concomitant decarboxylation of the amino acid. To exclude the possibility that AtbioF actually encodes another enzyme of this family with KAPA synthase side activity (AtbioF presents less than 35% identity to bacterial KAPA synthases), the plant enzyme was assayed for substrate specificity. None of the amino acids tested, except to some extent L-Ser (35% activity relative to L-Ala as pimeloyl-CoA condensing substrate), acted as an alternate substrate to L-Ala for the reaction. Similarly, none of the CoA derivatives tested, among acetyl-CoA, malonyl-CoA, propionyl-CoA, butyryl-CoA, succinyl-CoA, and palmitoyl-CoA, could substitute for pimeloyl-CoA in the assay. Furthermore, none of these compounds inhibited significantly KAPA synthase activity (data not shown). Also, AtbioF did not show any 5-aminolevulinate synthase, Ser palmitoyltransferase, or 2-amino-3-oxobutyrate CoA ligase activity. Altogether, these results confirm that AtbioF cDNA encodes a functional KAPA synthase with high substrate specificities.

To complete the characterization of plant KAPA synthase reaction, we analyzed the spectral changes induced by the interaction of AtbioF with its substrates and product using UV-visible spectroscopy. The increase in the absorbance of the 423-nm band on addition of L-Ala (Fig. 5B) was attributed to the external aldimine formation between PLP and the amino acid, in the active site (Jenkins, 1986). Plotting the absorption increase versus L-Ala concentration gave a hyperbolic curve with a dissociation constant \(K_d\) of 1.8 mM (Fig. 5B), which is consistent with the \(K_m\) value of 1.4 mM (Table I). This result is in broad agreement with those obtained for the corresponding *B. sphaericus* and *E. coli* enzymes (Ploux et al., 1999; Webster et al., 2000). Addition of pimeloyl-CoA to the external aldimine form of the enzyme resulted in the rapid formation of bands at 490 and 518 nm, which decreased slowly over the course of several minutes (Fig. 5C). The formation of these bands was observed only after addition of pimeloyl-CoA to the enzyme already complexed with L-Ala. The appearance of the 490-nm band implied formation of a quinoid (carbanion) intermediate (Webster et al., 2000), whereas the 518-nm band could be assigned to the product-PLP complex, as shown by incubation of AtbioF with KAPA, the end product of the reaction (Fig. 5D).

Subcellular Localization of Arabidopsis KAPA Synthase

Biotin synthase, the enzyme that catalyzes the final step in the biotin biosynthetic pathway, was previously localized into the mitochondrial matrix by both

Plant KAPA Synthase Is a Cytosolic Enzyme

botanik.uni-koeln.de predicted a mitochondrial or a plastidial targeting for AtbioF. On the other hand, the software program HMMTOP (Tusnády and Simon, 2001) detected a putative transmembrane domain in the C-terminal region of the protein (position 344–366). To clarify the situation, the subcellular localization of AtbioF was first investigated by fusing its full-length sequence upstream to the green fluorescent protein (GFP) marker protein. As shown in Figure 7, the expression of AtbioF-GFP in Arabidopsis protoplasts resulted in green fluorescence spread throughout the cytoplasm, a pattern similar to the one observed with GFP alone, but clearly distinct from punctuate patterns of green fluorescence observed with GFP fused to the transit peptide of the Rubisco small subunit (that colocalized with the red autofluorescence of chlorophyll) or fused to the mitochondrial transit peptide of an authentic mitochondrial protein. These results strongly suggest that AtbioF is a cytosolic protein.

As a complementary approach, we investigated the subcellular distribution of AtbioF in Arabidopsis by western blot. Intact chloroplasts and mitochondria from Arabidopsis leaves were purified on Percoll density gradients, thus providing organelles devoid of contamination from the other compartments. Also, a cytosolic-enriched fraction was prepared. Soluble proteins from purified chloroplasts (stroma), mitochondria (matrix), and the cytosolic-enriched fractions were then analyzed. Affinity-purified antibodies raised against the recombinant AtbioF protein identified a 52-kD polypeptide only in the total soluble protein extract and the cytosolic-enriched fraction (Fig. 8). No labeling was revealed in membrane subfractions (data not shown). These results confirmed that AtbioF is a cytosolic protein in Arabidopsis.

DISCUSSION

In this study, we report the cloning of an Arabidopsis cDNA coding a functional KAPA synthase, the first enzyme of the biotin biosynthetic pathway common to all biotin autotrophic organisms studied to date. To our knowledge, no other KAPA synthase has been characterized in eukaryotes. The data reported here substantiate earlier in vivo radiotracer evidence for the presence of this enzyme in plants (Baldet et al., 1993b) and validate the tentative annotation of the At5g04620 locus in the Arabidopsis genome database. Despite the relatively low overall amino acid identity with its

Figure 6. HPLC detection of KAPA produced by recombinant AtbioF. Reactions were carried out at 30°C and stopped by acidification with HCl as described in “Materials and Methods.” Assays contained 5 mM L-Ala, 1 mM pimeloyl-CoA, and 0.4 mg/mL pure recombinant enzyme. At the indicated times (T = 0 min, T = 30 min, and T = 120 min), a portion of the reaction mixtures was injected onto a Supelco Supelcosyl LC-SAX1 column. The uppermost trace is authentic KAPA (retention time 6.7 min). Metabolite traces are displayed at 210 nm because KAPA and L-Ala (retention time 8.6 min) are visible at this wavelength. Under the HPLC conditions used, pimeloyl-CoA and CoASH were not eluted from the column.

immunological and biochemical approaches (Baldet et al., 1997; Picciocchi et al., 2001, 2003). This raises the question of a mitochondrial localization for the whole pathway in the plant cell. However, none of the subcellular targeting prediction programs available and gathered on the Web site http://aramemnon.

Table I. Steady-state rate constants for Arabidopsis KAPA synthase

Kinetic parameters (±SD) were determined by nonlinear regression analysis of the data to the Michaelis-Menten equation.

| Substrate     | $K_{\mathrm{m}}$   | $V_{\max }$               | $k_{\text {cat }}$ | $k_{\text {cat }} / K_{\mathrm{m}}$ |
|---------------|--------------------|----------------------------|--------------------|-------------------------------------|
| Pimeloyl-CoA  | $\mu \mathrm{M}$   | $1.6 \pm 0.3$             | $110 \pm 4$        | $0.10 \pm 0.003$                   | $62,500$                              |
| L-Ala         |                    | $1,400 \pm 100$           | $125 \pm 5$        | $0.11 \pm 0.004$                   | $78$                                  |

Plant Physiol. Vol. 139, 2005

Downloaded from www.plantphysiol.org on March 16, 2015 - Published by www.plant.org  
Copyright © 2005 American Society of Plant Biologists. All rights reserved.

Pinon et al.

Bright Field    GFP         Chlorophyll

35S-GFP  
ATS1A-GFP  
HPPK DHPS-GFP  
AtBioF-GFP  

Figure 7. Expression of AtbioF fused to GFP in Arabidopsis protoplasts. Constructs encoding GFP alone (35S-GFP), GFP fused to the N terminus of the small subunit of Rubisco (a chloroplastic marker) from Arabidopsis (ATS1A-GFP), GFP fused to the N terminus of dihydrotorin pyrophosphokinase-dihydropteroate synthase (a mitochondrial marker) from pea (HPPK-DHPS-GFP), and GFP fused to full-length AtbioF (AtbioF-GFP) were introduced into Arabidopsis protoplasts. Images are optical photomicrographs (Bright Field), GFP fluorescence (GFP; green pseudocolor), and chlorophyll fluorescence (Chlorophyll; red pseudocolor).

between at least cytosol and mitochondria. Our current knowledge of biotin synthesis compartmentation in other eukaryotic organisms is rather limited and relates only to yeasts. *Saccharomyces cerevisiae* apparently contains only the three last enzymes of the pathway, DAPA aminotransferase, dethiobiotin synthase and biotin synthase, coded by the *BIO3*, *BIO4*, and *BIO2* genes, respectively (Zhang et al., 1994; Phalip et al., 1999a). Biotin synthase of *S. cerevisiae* is a mitochondrial protein (Mühlenhoff et al., 2002), but nothing has been reported concerning the subcellular localization of the other enzymes. The fission yeast *Schizosaccharomyces pombe* contains only the biotin synthase gene that is also predicted to encode a mitochondrial protein (Phalip et al., 1999b), whereas homologs of *BIO3* and *BIO4* are not found in the genome.

Our evidence that the synthesis of biotin is compartmentalized in Arabidopsis underscores the complex spatial organization of plant biotin metabolism. Biotin is present in most plant cell compartments; for instance, in pea (*Pisum sativum*) leaves, biotin is distributed in equimolar ratios among mitochondria, chloroplasts, and cytosol, but not in the same state. Most of the free biotin pool accumulates in cytosol of mesophyll cells, while protein-bound biotin (associated to biotin-dependent carboxylases) mainly resides within organelles (Baldet et al., 1993a). These biotin-dependent carboxylases in plants include cytosolic acetyl-CoA carboxylase, chloroplastic geranyl-CoA and acetyl-CoA carboxylases, and mitochondrial methylcrotonoyl-CoA carboxylase (for review, see Alban et al., 2000; Nikolau et al., 2003). This complex compartmentation of biotin, biotin-mediated reactions, and, as demonstrated in this study, biotin synthesis in the plant cell implies an intracellular trafficking of biotin and precursors, thus requiring transport mechanisms (Fig. 9). These transport steps include transfer of an intermediate, KAPA, DAPA, or dethiobiotin, between the cytosol and mitochondria. Preliminary bioinformatics analyses of conceptually translated Arabidopsis sequences retrieved from database searches

(kD)   T     St    Ma    Cy

Figure 8. Immunolocalization of AtbioF in Arabidopsis. Proteins (200 μg for total soluble extract [T], 90 μg for chloroplast stroma [St], 60 μg for mitochondrial matrix [Ma], and 200 μg for the cytosolic enriched fraction [Cy]) were separated by SDS-PAGE and analyzed by western blotting using affinity-purified antibodies raised against AtbioF.

bacterial counterparts, the plant protein was able to complement an *E. coli bioF*- mutant and to catalyze KAPA synthase reaction when assayed using pimeloyl-CoA and L-Ala as condensing substrates. Biochemical, kinetic, and spectroscopic studies of purified recombinant enzyme evidenced high substrate specificities and were consistent with the mechanistic proposal provided by x-ray analysis and detailed mechanistic studies of bacterial KAPA synthases (Ploux and Marquet, 1996; Ploux et al., 1999; Webster et al., 2000). An essential step of this mechanism is formation of an external aldimine between PLP and the substrate L-Ala. Abstraction of the C2-H proton of the aldimine, possibly by Lys-319 (Arabidopsis numbering), leads to a quinonoid intermediate, which then attacks the thioester carbonyl of pimeloyl-CoA. Release of CoASH produces a β-ketoacid aldimine, which after decarboxylation is converted into the product.

The most important finding from our study of plant KAPA synthase concerns its cellular distribution. Both GFP-targeting experiments and western-blot analyses showed that Arabidopsis KAPA synthase is specifically present in the cell cytosol. This contrasts markedly with the mitochondrial location of plant biotin synthase (Baldet et al., 1997; Picciocchi et al., 2001, 2003). Therefore, our results evidence an unexpected compartmentation of the plant biotin synthesis, split

Plant Physiol. Vol. 139, 2005

Downloaded from www.plantphysiol.org on March 16, 2015 - Published by www.plant.org

Copyright © 2005 American Society of Plant Biologists. All rights reserved.

Plant KAPA Synthase Is a Cytosolic Enzyme

Figure 9. Working model for the compartmentation of biotin metabolism in the plant cell. Biotin-synthesizing enzymes are as follows: 1, KAPA synthase; 2, DAPA aminotransferase; 3, dethiobiotin synthase; and 4, biotin synthase. The occurrence of reaction 2 in the cytosol and of reaction 3 in mitochondria is predicted from DNA sequence data. The compartmentation of holocarboxylase synthetase(s) catalyzing biotinylation reactions (reaction 5) is discussed by Tissot et al. (1997).

predict that putative DAPA aminotransferase lacks targeting peptide and is apparently cytosolic, and that putative dethiobiotin synthase is addressed to mitochondria. If so, this would imply that mitochondria import DAPA (Fig. 9). Also, once synthesized biotin must be exported outside mitochondria to cytosol, maybe in exchange with the imported intermediate, and then to chloroplasts, where protein biotinylation reactions were previously found to occur, together with mitochondria (Tissot et al., 1997, 1998). Such a situation probably reflects a major implication of organelle-specific transporters in the maintenance of the balance between the synthesis and the requirements in biotin for the plant cell.

Interestingly, this complex spatial organization of biotin metabolism in plants can be paralleled with that of tetrahydrofolate, another water-soluble vitamin synthesized only by plants and microorganisms. Tetrahydrofolate and its derivatives are vital cofactors for enzymes that mediate one-carbon transfer reactions. Folates are tripartite molecules, composed of pterin, p-aminobenzoate, and Glu moieties. In plants, the pterin moiety is produced in the cytosol, p-aminobenzoate is formed in plastids, and the two are coupled together and glutamylated in mitochondria, whereas all these steps are cytosolic in other organisms (for review, see Ravanel et al., 2004b). Like biotin, folates are also present in most plant cell compartments, in agreement with the subcellular distribution of folate-dependent reactions. Folates and their biosynthetic intermediates must therefore also move in and out of organelles (Basset et al., 2005). Thus, the synthesis of these metabolites and their transport across the inner mitochondrial membrane are also probably factors of regulation for synthesis of this vitamin in plants.

Biotin transporters have been characterized and cloned from animals, which typically depend on the dietary intake of biotin (Prasad et al., 1998; Wang et al., 1999; Grafe et al., 2003). *S. cerevisiae* and *S. pombe* are also auxotrophic for biotin since they present truncated biosynthetic pathways. However, these organisms contain specific transporters allowing growth in the presence of appropriate biotin intermediates (Phalip et al., 1999a, 1999b; Stolz, 2003, and refs. therein). In plants one biotin transporter has been identified so far. Arabidopsis AtSuc5 is a plasma membrane Suc/biotin cotransporter, possibly involved in biotin uptake and allocation between cells and plant tissues (Ludwig et al., 2000). Intracellular trafficking of biotin and intermediates is, however, largely unknown, despite its potential importance in the control of metabolic fluxes within the cell. Future elucidation of these specific transport systems at the biochemical and molecular levels will therefore be helpful for better understanding the mechanisms of regulation of biotin synthesis and utilization in plants.

Table II. Synthetic oligonucleotides used in this study

| Primer Name           | Primer Sequence                     |
|-----------------------|--------------------------------------|
| AtbioF cDNA cloning   |                                      |
| AtbioF/Ncol           | 5′ATTCTGTTTGCACCTTACCATGGCGGATCATT CGTGG3′ |
| AtbioF/Ndel           | 5′ATTCTGTTTGCACCTCATATGGCGGATCATT CGTGG3′ |
| AtbioF/EcoRI          | 5′GCGACAACAGAAAGAATTCTTATAATTTGGG AAATAG3′ |
| AtbioF-GFP fusion cloning |                                  |
| AtbioF-GFP5′          | 5′GCAGTCGACATGGCGGATCATT CGTGGG3′    |
| AtbioF-GFP3′          | 5′ATTCCATGGATAATTTGGGAAATAGAAAAGG3′ |

Plant Physiol. Vol. 139, 2005
Downloaded from www.plantphysiol.org on March 16, 2015 - Published by www.plant.org
Copyright © 2005 American Society of Plant Biologists. All rights reserved.

Pinon et al.

# MATERIALS AND METHODS

## Plant Materials

Arabidopsis (*Arabidopsis thaliana*; ecotype Wassilewskija) seeds were grown on soil under greenhouse conditions (23°C with a 16-h photoperiod and a light intensity of 100 μE m⁻² s⁻¹) until harvested for analysis. Arabidopsis (ecotype Columbia) cell suspension cultures were grown under continuous white light (40 μE m⁻² s⁻¹) at 23°C with rotary agitation at 125 rpm in Gamborg's B5 medium supplemented with 1 μM 2-naphtalene acetic acid and 1.5% (w/v) Suc.

## Chemicals

Pimeloyl-CoA was chemically synthesized and purified essentially as described by Ploux and Marquet (1992), starting from pimeloyl chloride as precursor. KAPA was obtained as described previously (Baldet et al., 1993b). Isopropylthio-β-d-galactoside (IPTG) was from Bioprobe Systems. His-binding resin was from Qiagen. All other biochemicals were obtained from Sigma-Aldrich and were the purest grade available.

## cDNA Isolation and Expression Constructs

Sequences of primers used in this study are listed in Table II. Full-length cDNAs were obtained by PCR amplification of an Arabidopsis (var Columbia) cDNA library constructed in pYES (Elledge et al., 1991). Primers were designed on the basis of the predicted genomic sequence of KAPA synthase and generated appropriate restriction sites for the cloning of cDNAs into different expression vectors. The PCR fragments were first cloned into pPCR-Script (Stratagene) for maintenance and sequenced in both strands (Genome Express). The cloned sequence encoding Arabidopsis KAPA synthase was designated *AtbioF* (the equivalent of bacterial *bioF* gene product) and was deposited in the GenBank database under GenBank accession number DQ017966. The *Escherichia coli* expression vector for functional complementation experiments was pTrc99A (Pharmacia), which allows tight control of gene expression by IPTG. The complete open reading frame of *AtbioF* cDNA was inserted between its *NcoI* and *EcoRI* sites, yielding the plasmid pTrc-AtbioF. For overexpression, the *AtbioF* sequence was inserted between the *NdeI* and *EcoRI* sites of pET-28b (Novagen). This procedure added an N-terminal hexa-His tag. The resulting pET-AtbioF construct was introduced first into *E. coli* DH5α, then into *E. coli* Rosetta (DE3) cells (Stratagene).

## Functional Complementation

*E. coli bioF-* mutant \[thr-, LeuB6(Am), glnV44, bioF110, rfbD1, thi-1\] harboring pTrc99A constructs was cultured either in liquid medium or on plate at 37°C in M9 synthetic minimal medium containing 0.2% (w/v) Glc, 0.4% (w/v) casein hydrolysate, 1 μg mL⁻¹ thiamin-HCl, 100 μg mL⁻¹ carbenicillin, and 1 mM IPTG, plus or minus 1 mM biotin.

## Overexpression and Purification of Recombinant AtbioF

*E. coli* Rosetta cells transformed with the pET-AtbioF construct were grown at 37°C in Luria-Bertani medium supplemented with 10 μM pyridoxine, 15 μM thiamin, and the appropriate antibiotics until A₆₀₀ was 0.6, at which point IPTG was added to a final concentration of 1 mM. Incubation was continued for 16 h at 28°C. Pelleted cells from 2-L cultures were resuspended in 15 mL of buffer A (20 mM Tris-HCl, pH 8, 500 mM NaCl) supplemented with 10 mM imidazole and disrupted by sonication with a Vibra-Cell disrupter. The soluble protein extract was separated from the cell debris by centrifugation at 18,000g for 30 min and applied onto a nickel-nitrilotriacetic acid-agarose column (1 × 3 cm) previously equilibrated with the same buffer. The column was washed with buffer A supplemented with 30 mM imidazole. The recombinant protein was then eluted with buffer A supplemented with 250 mM imidazole. Fractions containing the recombinant AtbioF protein were pooled and dialyzed against Tris-buffered saline. Purified enzyme was aliquoted and stored at –80°C until use.

## Assays for KAPA Synthase and Kinetic Data Analysis

KAPA synthase activity was measured using three different assays with similar results: (1) by monitoring the disappearance of the pimeloyl-CoA

thioester bond A₂₃₀, as described for the *Bacillus sphaericus* enzyme (Ploux and Marquet, 1996); (2) by monitoring the release of CoASH from pimeloyl-CoA using a linked assay with α-ketoglutarate dehydrogenase, as described for the *E. coli* enzyme (Webster et al., 2000); and (3) by following the release of CoASH from pimeloyl-CoA with Ellman’s thiol reagent, DTNB (Ellman, 1959). Since the last assay was the most sensitive, it was used subsequently for all kinetic determinations. Briefly, the standard assay mixture contained 50 mM potassium phosphate, pH 7.5, 100 μM pimeloyl-CoA, 10 mM L-Ala, 5 to 20 μg of AtbioF, and 0.1 mM DTNB in a final volume of 400 μL. The reaction was started with the addition of L-Ala, and the increase in absorbance due to the formation of thiophenolate was continuously monitored at 412 nm (*ε*₄₁₂ = 13,600 M⁻¹ cm⁻¹) and 30°C. The initial velocities measured in this procedure were linear for at least 2 min and rates varied linearly with enzyme concentration, demonstrating adherence to steady-state conditions. Kinetic data were fitted to the Michaelis-Menten rate equation using nonlinear regression data analysis software (Kaleidagraph; Abelbeck Software). Production of KAPA, the end product of the reaction, was controlled by HPLC analysis according to the procedure described by Baldet et al. (1993b). Briefly, reaction mixtures (100 μL) containing 10 mM potassium phosphate, pH 7.5, 1 mM pimeloyl-CoA, 5 mM L-Ala, and 40 μg of AtbioF were run for 30 to 120 min at 30°C. Reactions were stopped with 0.15 M HCl and centrifuged at 12,000g for 5 min. Supernatants (20-μL aliquots) were injected onto a Supelco Supelcosyl LC-SAX1 column (5 μm, 250 × 4.6 mm) and eluted isocratically with 25 mM HCl at a flow rate of 0.4 mL min⁻¹. Elution of KAPA was monitored at 210 nm using a diode-array detector (DAD 1100; Agilent).

## Spectroscopic Studies

Absorption spectra of KAPA synthase were recorded in a 1-cm optical-path-length quartz cuvette (200 μL) using a Uvikon 923 (UVK-Lab) spectrophotometer thermostated at 30°C. The protein concentration in all analyses was 36.8 μM in 100 mM potassium phosphate, pH 7.5. For determination of L-Ala dissociation constant, varying amounts of L-Ala (0–22.8 mM) were used. After addition of the substrate, the reactants were mixed and allowed to equilibrate for 5 min at 30°C before recording spectra. Small baseline changes at 550 nm were corrected. Changes in A₄₂₃ were plotted against L-Ala concentrations, and data points were fitted to a hyperbolic saturation curve (Eq. 1) using Kaleidagraph software:

\[
\Delta A_{\text{obs}} = \frac{\Delta A_{\text{max}}[\text{Ala}]}{K_{\text{d}} + [\text{Ala}]},
\]

where ΔAₘₐₓ is the maximal absorbance change, [Ala] is the L-Ala concentration, and Kₔ is the dissociation constant.

## Preparation of a Cytosolic-Enriched Fraction

Arabidopsis protoplasts were prepared by enzymatic digestion of 6-d-old cell suspension cultures, using the procedure described by Tissot et al. (1997). Protoplasts were gently ruptured by passing through a 20-μm nylon mesh and subsequently through a 10-μm nylon mesh. The protoplast lysate was centrifuged successively at 100g for 5 min, 900g for 5 min, and 13,000g for 20 min. The 13,000g supernatant fraction was centrifuged further at 100,000g for 1 h to remove membranes. Pellets were pooled and used, together with the 13,000g supernatant fraction, to measure marker enzyme activities (Tissot et al., 1997). Most of the chloroplast (94% ± 2%, mean of three independent experiments ± SE) and mitochondria (96% ± 2%) marker activities were recovered in the pooled pellets, whereas the 13,000g supernatant fraction contained 72% ± 3% of the cytosolic marker activity. This cytosolic-enriched fraction contained a small proportion of mitochondrial (4% ± 2%) and chloroplastic (6% ± 2%) marker enzymes.

## Purification of Chloroplast and Mitochondria

Chloroplasts and mitochondria were purified from 4-week-old Arabidopsis leaves as described by Ravanel et al. (2004a) and Kruff et al. (2001), respectively. Intact chloroplasts were purified on preformed Percoll gradients and lysed in 20 mM MOPS, pH 7.5, 4 mM MgCl₂, 1 mM phenylmethanesulfonyl fluoride, 1 mM benzamide-HCl, and 5 mM ε-aminocaproic acid. Chloroplast subfractions were separated on a step gradient of 0.93 to 0.6 M Suc in 10 mM MOPS, pH 7.5, by centrifugation at 70,000g for 1 h. The soluble fraction (stroma) was collected at the top of the 0.6 M Suc layer. Mitochondria were

purified on discontinuous gradients consisting of 18%, 29%, and 45% Percoll layers in 0.3 M Suc and 10 mM MOPS, pH 7.2. Gradients were centrifuged at 70,000g for 45 min, and mitochondria were collected at the 29%/45% Percoll interface. To obtain soluble proteins (matrix), purified mitochondria were lysed by a 10-fold dilution in 10 mM MOPS, pH 7.2, 1 mM dithiothreitol and submitted to three freeze-thaw cycles. Membranes were removed by centrifugation at 100,000g for 1 h.

### Protein Determination, Antibody Production, and Immunoblot Analysis

Proteins were measured either by the Bradford method (Bradford, 1976) using Bio-Rad protein assay reagent, with γ-globulin as a standard, or for pure proteins by measuring the $A_{205}$ (Scopes, 1974). The recombinant AtbioF protein purified by metal-chelate column chromatography was purified further by SDS-PAGE (LaemmlI, 1970) and injected into guinea pigs to raise antibodies (Elevage Scientifique des Dombes). Total soluble proteins from Arabidopsis cells were extracted by grinding powdered samples in 20 mM MOPS, pH 7.5, 5% (w/v) glycerol, 1 mM dithiothreitol, 1 mM EDTA, 1 mM benzamidine-HCl, and 5 mM ε-aminocaproic acid. Samples were centrifuged at 130,000g for 20 min at 4°C, and the supernatant was used as a source of soluble proteins. Proteins from Arabidopsis cell subfractions were resolved by SDS-PAGE and electroblotted to nitrocellulose membrane. The blots were probed by using the affinity-purified AtbioF antibodies (dilution 1:1,000), horseradish peroxidase-conjugated anti-guinea pig IgGs, and detection was achieved by chemiluminescence.

### Transient Expression of GFP Fusion Protein in Arabidopsis Protoplasts

Primers were designed to amplify the sequence encoding for the complete AtbioF protein (Table II) and allowed its cloning in frame upstream of the GFP-coding region in the 35Ω-sGFP(S65T) vector, at SalI and NcoI sites, yielding pGFP-AtbioF construct. The GFP reporter plasmid 35Ω-sGFP(S65T) expressed an engineered version of the GFP under the control of the cauliflower mosaic virus 35S promoter (Chiu et al., 1996). The transit peptide sequences of the small subunit of Rubisco from Arabidopsis (55 residues, *ats1A* gene, GenBank accession no. X13611) and dihydropterin pyrophosphokinase-dihydropteroate synthase (HPPK-DHPS) from pea (*Pisum sativum*; 28 residues; Rébeillé et al., 1997) were used as controls for the targeting of GFP to plastids and mitochondria, respectively. Transient transformation of Arabidopsis protoplasts prepared from a 4-d-old cell suspension culture was achieved using 20 μg of plasmid construct by the polyethylene glycol method, essentially as described (Abel and Theologis, 1994). Transformed cells were incubated at 23°C for 36 h and analyzed by epifluorescence microscopy using a Zeiss Axiosplan2 fluorescence microscope, and images were captured with a digital CCD camera (Hamamatsu). The filter sets used were Zeiss filter set 13, 488013-0000 (exciter BP 470/20, beamsplitter FT 493, emitter BP 505–530) and Zeiss filter set 15, 488015-0000 (exciter BP 546/12, beamsplitter FT 580, emitter LP 590), for GFP and chlorophyll fluorescence, respectively.

Sequence data from this article can be found in the GenBank/EMBL data libraries under accession number DQ017966.

### ACKNOWLEDGMENTS

We are grateful to Drs. Michel Matringe and Renaud Dumas for helpful discussions.

Received August 19, 2005; revised September 30, 2005; accepted September 30, 2005; published November 18, 2005.

### LITERATURE CITED

- Abel S, Theologis A (1994) Transient transformation of Arabidopsis leaf protoplasts: a versatile experimental system to study gene expression. Plant J 5: 421–427
- Alban C, Job D, Douce R (2000) Biotin metabolism in plants. Annu Rev Plant Physiol Plant Mol Biol 51: 17–47

Plant Physiol. Vol. 139, 2005

Downloaded from www.plantphysiol.org on March 16, 2015 - Published by www.plant.org Copyright © 2005 American Society of Plant Biologists. All rights reserved.

Pinon et al.

pombe bio2 by heterologous complementation of *Saccharomyces cerevisiae* mutant. Curr Microbiol 39: 348–350

Picciocchi A, Douce R, Alban C (2001) Biochemical characterization of the Arabidopsis biotin synthase reaction. The importance of mitochondria in biotin synthesis. Plant Physiol 127: 1224–1233

Picciocchi A, Douce R, Alban C (2003) The plant biotin synthase reaction. Identification and characterization of essential mitochondrial accessory protein components. J Biol Chem 278: 24966–24975

Ploux O, Breyne O, Carillon S, Marquet A (1999) Slow-binding and competitive inhibition of 8-amino-7-oxopelargonate synthase, a pyridoxal-5′-phosphate-dependent enzyme involved in biotin biosynthesis, by substrate and intermediate analogs. Kinetic and binding studies. Eur J Biochem 259: 63–70

Ploux O, Marquet A (1992) The 8-amino-7-oxopelargonate synthase from *Bacillus sphaericus*. Purification and preliminary characterization of the cloned enzyme overproduced in *Escherichia coli*. Biochem J 283: 327–331

Ploux O, Marquet A (1996) Mechanistic studies on the 8-amino-7-oxopelargonate synthase, a pyridoxal-5′-phosphate-dependent enzyme involved in biotin biosynthesis. Eur J Biochem 236: 301–308

Prasad PD, Wang H, Kekuda R, Fujita T, Fei YJ (1998) Cloning and functional expression of a cDNA encoding a mammalian sodium-dependent vitamin transporter mediating the uptake of pantothenate, biotin, and lipoate. J Biol Chem 273: 7501–7506

Ravanel S, Block MA, Rippert P, Jabrin S, Curien G, Rébeillé F, Douce R (2004a) Methionine metabolism in plants: chloroplasts are autonomous for *de novo* methionine synthesis and can import S-adenosylmethionine from the cytosol. J Biol Chem 279: 22548–22557

Ravanel S, Douce R, Rebeille F (2004b) The uniqueness of tetrahydrofolate synthesis and one-carbon metabolism in plants. In D Day, J Whelan, H Millar, eds, Advances in Photosynthesis and Respiration. Plant Mitochondria, from Genome to Function, Vol 17. Kluwer Academic Publishers, Dordrecht, The Netherlands, pp 277–292

Rébeillé F, Macherel D, Mouillon JM, Garin J, Douce R (1997) Folate biosynthesis in higher plants: purification and molecular cloning of a bifunctional 6-hydroxymethyl-7,8-dihydropterin pyrophosphokinase/7,8-dihydropteroate synthase localized in mitochondria. EMBO J 16: 947–957

Rolfe B, Eisenberg MA (1968) Genetic and biochemical analysis of the biotin loci of *Escherichia coli* K-12. J Bacteriol 96: 515–524

Sanchez LB, Müller M (1996) Purification and characterization of the acetate forming enzyme, acetyl-CoA synthetase (ADP-forming) from the amitochondriate protist, *Giardia lamblia*. FEBS Lett 378: 240–244

Schneider T, Dinkins R, Robinson K, Shellhammer J, Meinke DW (1989) An embryo-lethal mutant of *Arabidopsis thaliana* is a biotin auxotroph. Dev Biol 131: 161–167

Scopes RK (1974) Measurement of protein by spectrophotometry at 205 nm. Anal Biochem 59: 277–282

Shellhammer J, Meinke D (1990) Arrested embryos from the *bio1* auxotroph of *Arabidopsis thaliana* contain reduced levels of biotin. Plant Physiol 93: 1162–1167

Stolz J (2003) Isolation and characterization of the plasma membrane biotin transporter from *Schizosaccharomyces pombe*. Yeast 20: 221–231

Streit WR, Entcheva P (2003) Biotin in microbes, the genes involved in its biosynthesis, its biochemical role and perspectives for biotechnological production. Appl Microbiol Biotechnol 61: 21–31

Tissot G, Douce R, Alban C (1997) Evidence for multiple forms of biotin holocarboxylase synthetase in pea (*Pisum sativum*) and in *Arabidopsis thaliana*: subcellular fractionation studies and isolation of a cDNA clone. Biochem J 323: 179–188

Tissot G, Pépin R, Job D, Douce R, Alban C (1998) Purification and properties of the chloroplastic form of biotin holocarboxylase synthetase from *Arabidopsis thaliana* overexpressed in *Escherichia coli*. Eur J Biochem 258: 586–596

Tusnády GE, Simon I (2001) The HMMTOP transmembrane topology prediction server. Bioinformatics 17: 849–850

Wang H, Huang W, Fei YJ, Xia H, Yang-Feng TL, Leibach FH, Devoe LD, Ganapathy V, Prasad PD (1999) Human placental Na+-dependent multivitamin transporter. Cloning, functional expression, gene structure, and chromosomal localization. J Biol Chem 274: 14875–14883

Weaver LM, Yu F, Wurtele ES, Nikolau BJ (1996) Characterization of the cDNA and gene coding for the biotin synthase of *Arabidopsis thaliana*. Plant Physiol 110: 1021–1028

Webster SP, Alexeev D, Campopiano DJ, Watt RM, Alexeeva M, Sawyer L, Baxter RL (2000) Mechanism of 8-amino-7-oxononanoate synthase: spectroscopic, kinetic, and crystallographic studies. Biochemistry 39: 516–528

Zhang S, Sanyal I, Bulboaca GH, Rich A, Flint DH (1994) The gene for biotin synthase from *Saccharomyces cerevisiae*: cloning, sequencing, and complementation of *Escherichia coli* strains lacking biotin synthase. Arch Biochem Biophys 309: 29–35
