
Biochemical analysis of the EJC reveals two new factors and a stable tetrameric protein core

THOMAS Ø. TANGE,$^{1}$ TOSHIHARU SHIBUYA,$^{1}$ MELISSA S. JURICA,$^{2}$ and MELISSA J. MOORE$^{1}$

$^{1}$Howard Hughes Medical Institute, Department of Biochemistry, Brandeis University, Waltham, Massachusetts 02454, USA

---

**ABSTRACT**

The multiprotein exon junction complex (EJC) is deposited on mRNAs upstream of exon-exon junctions as a consequence of pre-mRNA splicing. In mammalian cells, this complex serves as a key modulator of spliced mRNA metabolism. To date, neither the complete composition nor the exact assembly pathway of the EJC has been entirely elucidated. Using in vitro splicing and a two-step chromatography procedure, we have purified the EJC and analyzed its components by mass spectrometry. In addition to finding most of the known EJC factors, we identified two novel EJC components, Acinus and SAP18. Heterokaryon analysis revealed that SAP18 is a shuttling protein whereas Acinus is restricted to the nucleus. In MS2 tethering assays Acinus stimulated gene expression at the RNA level, while MLN51, another EJC factor, stimulated mRNA translational efficiency. Using tandem affinity purification (TAP) of proteins overexpressed in HeLa cells, we demonstrated that Acinus binds directly to another EJC component, RNPS1, while stable association of SAP18 to form the trimeric apoptosis and splicing associated protein (ASAP) complex requires both Acinus and RNPS1. Using the same methodology, we further identified what appears to be the minimal stable EJC core, a heterotetrameric complex consisting of eIF4AIII, Magoh, Y14, and MLN51.

Keywords: exon junction complex; pre-mRNA splicing; mass spectrometry

---

**INTRODUCTION**

In the cell, messenger RNAs (mRNAs) coexist with numerous bound proteins in the form of ribonucleoprotein particles (mRNPs). It is now well established that many mRNP proteins are acquired during transcription and pre-mRNA processing in the nucleus and then accompany the mature mRNA to the cytoplasm. In particular, the act of pre-mRNA splicing leads to deposition on spliced mRNAs, ~20 nucleotides (nt) upstream of exon-exon junctions, of a set of proteins known as the exon junction complex (EJC) (Le Hir et al. 2000a; Tange et al. 2004). As EJCs travel with mRNAs to the cytoplasm, they modulate such processes as subcellular mRNA localization, translational efficiency, and nonsense-mediated mRNA decay (NMD; see below). To date, more than a dozen proteins are thought to interact with or be core components of the mammalian EJC. These include the splicing coactivators/alternative splicing factors SRm160, RNPS1, and Pinin; the mRNA export factors UAP56, REF/Aly, and NXF1/TAP:p15; homologs of the *Drosophila* mRNA localization factors Y14, Mago, Barentsz, and eIF4AIII (known in humans as Y14, Magoh, MLN51, and eIF4AIII); and the NMD factors Upf3a/b, Upf2, and Upf1 (Kataoka et al. 2000, 2001; Le Hir et al. 2000a, 2001a,b; Lykke-Andersen et al. 2000, 2001; Zhou et al. 2000; Gatfield et al. 2001; Kim et al. 2001a,b; Luo et al. 2001; Li et al. 2003; Chan et al. 2004; Degot et al. 2004; Ferraiuolo et al. 2004; Palacios et al. 2004; Shibuya et al. 2004).

Several EJC factors are known to function in mRNA export or subsequent mRNA localization in the cytoplasm. Although pre-mRNA splicing may not be essential for mRNA export, the presence of an EJC has been shown to enhance nuclear export of a bound RNA compared with an otherwise identical RNA not carrying an EJC. The EJC interacting proteins UAP56, Aly/REF, and NXF1/TAP:p15 are all known mRNA export factors that facilitate export of both intronless and spliced mRNAs. UAP56 is thought to recruit Aly/REF, which in turn interacts with NXF1/TAP:p15, which directly contacts components of the nuclear pore complex (Luo and Reed 1999; Zhou et al.

2000; Gatfield et al. 2001; Le Hir et al. 2001b; Luo et al. 2001; Rodrigues et al. 2001; Gatfield and Izaurralde 2002; Stutz and Izaurralde 2003). Once in the cytoplasm, many mRNAs are subject to asymmetric localization. In *Drosophila*, Y14, Mago, Barentsz, and eIF4AIII are all essential for proper localization and/or regulated translation of *oskar* mRNA at the posterior pole of developing oocytes. Mammalian MLN51 additionally exhibits dendritic localization in mature hippocampal neurons, implicating it in localization of dendritic mRNAs (Mohr et al. 2001; van Eeden et al. 2001; Palacios 2002; Macchi et al. 2003; Hachet and Ephrussi 2004; Palacios et al. 2004).

In mammalian cells, the EJC is a critical determinant for NMD. The currently accepted model for mammalian NMD posits that if the first in-frame stop codon occurs >25–30 nt upstream of any EJC deposition site(s), then the EJC interacting proteins and known NMD factors Upf3a/b (related products of different genes), Upf2, and Upf1 recruit the general mRNA degradation machinery, resulting in significant mRNA destabilization (Lykke-Andersen et al. 2000; Ishigaki et al. 2001; Lejeune et al. 2002; Conti and Izaurralde 2005; Lejeune and Maquat 2005). Evidence supporting this model includes data from RNAi and protein tethering experiments. For example, RNAi knockdown of either MLN51 or eIF4AIII in mammalian cells leads to defects in NMD (Ferraiuolo et al. 2004; Palacios et al. 2004; Shibuya et al. 2004). Further, Upf1, Upf2, Upf3a/b, Y14, Magoh, MLN51, and RNPS1 can all stimulate NMD when tethered to the 3′-UTR of a reporter mRNA as MS2 or λN fusion proteins (Lykke-Andersen et al. 2000, 2001; Fribourg et al. 2003; Gehring et al. 2003; Palacios et al. 2004). Conversely, many of these same factors can stimulate mRNA translation when tethered inside an open reading frame (ORF) (Nott et al. 2004). This latter activity replicates the effect of EJC deposition inside an ORF, which has likewise been shown to increase mRNA translational yield (Nott et al. 2003; Wiegand et al. 2003). Taken together, the above data indicate that the EJC and its binding partners interface with, if not all, then most of, the cellular machineries that act on mRNAs once they are deemed competent for nuclear export.

Structurally, the EJC likely consists of a few stably bound core factors initially loaded during pre-mRNA splicing. Other EJC factors interact more transiently and are subject to dynamic exchange either in the nucleus or the cytoplasm (Tange et al. 2004). Of the known EJC proteins, only eIF4AIII has been shown to interact directly with spliced mRNA (Shibuya et al. 2004). eIF4AIII is a member of the DEAD-box family of RNA-stimulated ATPases and is very closely related to the canonical eukaryotic translation initiation factor eIF4AI. Most likely it is eIF4AIII that creates the 8-to 10-nt EJC footprint, as this is the known binding site size for single-stranded nucleic acids exhibited by other members of this protein family (Tanner and Linder 2001). Thus eIF4AIII has been proposed to function as an RNA “placeholder” or platform upon which the rest of the EJC assembles (Shibuya et al. 2004). Because other DEAD-box proteins exhibit more dynamic RNA association than eIF4AIII, one might predict that other factors in the EJC core function to prevent it from dissociating from spliced mRNA. Recent reports have indicated that the factors Magoh, Y14, and MLN51/Barentsz all interact directly with eIF4AIII (Palacios et al. 2004; Shibuya et al. 2004). For example, Magoh and Y14, which themselves form a tight heterodimer, could be co-precipitated with GST-tagged eIF4AIII when the three factors were co-expressed in *Escherichia coli*, and two-hybrid studies indicated that *Drosophila* eIF4AIII directly interacts with the N-terminal half of Barentsz (Palacios et al. 2004). Thus, available evidence suggests that Magoh, Y14, and MLN51/Barentsz might all be part of the EJC core.

Despite much progress in EJC composition analysis over the past 5 yr, the factors so far identified are limited to proteins either previously known to associate with spliced mRNA or to interact physically or genetically with other known EJC factors. Given these limitations and the emerging realization that the EJC interfaces with so many other cellular machineries, it seemed likely to us that there could still be unidentified proteins either constituting part of the core or interacting transiently with the EJC. In recent years mass spectrometry has been widely employed to probe the composition of large protein complexes. For example, a plethora of polypeptides associated with pre-mRNA splicing complexes has been identified using this strategy (Jurica and Moore 2003). Here we report the purification and mass spectrometric analysis of an EJC assembled by in vitro pre-mRNA splicing. This work led to the identification of two novel EJC factors, Acinus and SAP18. Further, we have isolated two subcomplexes of EJC factors that can form stably in HeLa cells. One of these appears to be the minimal tetrameric EJC core and consists of eIF4AIII, Y14, Magoh, and MLN51.

## RESULTS

### In vitro purification and mass spectrometry of the EJC

In order to establish the complete set of factors stably associated with the in vitro-derived EJC, we undertook a large-scale purification based on protocols previously developed for purifying C complex spliceosomes and intact spliced mRNPs (Fig. 1A; Jurica et al. 2002; Reichert et al. 2002). Specifically, a single intron AdML pre-mRNA splicing substrate containing six MS2 stem-loops in the 3′ exon was spliced in HeLa nuclear extract that had been supplemented with recombinant MS2-maltose binding protein fusion (MS2-MBP) and recombinant GST-Magoh:Y14 heterodimer, both purified from *E. coli*. Stable association of MS2-MBP with its recognition sequences in the 3′ exon allowed for amylose affinity purification of the spliced

Downloaded from rnajournal.cshlp.org on April 22, 2015 - Published by Cold Spring Harbor Laboratory Press

Tetrameric EJC core

A

![Diagram](https://i.imgur.com/yourimageurl.png)

B

| kDa | EJC | control |
| --- | --- | --- |
| 97 |  |  |
| 66 |  |  |
| 45 |  |  |
| 31 |  |  |
| 21 |  |  |

and then applied to a GST-column to allow specific isolation of the EJC, which contained GST-Magoh:Y14.

Purified EJC proteins from 24 mL of splicing reaction were separated in one lane of a denaturing polyacrylamide gel (Fig. 1B, lane 1). A parallel control purification including all steps (also 24 mL splicing reaction total) was performed using an AdML substrate that lacked the intron (Fig. 1B, lane 2). The gel lanes for the EJC and control purifications were each excised in three sections and proteins analyzed by nanoscale microcapillary liquid chromatography tandem mass spectrometry (LC-MS/MS). The results of this analysis are summarized in Table 1, which lists the number of unique peptides identified for each protein in the EJC prep (EJC [Ad]). For the purposes of comparison, peptide numbers are also shown for our previous mass spectrometry analysis of C complex (C [Ad]) (Jurica et al. 2002), as well as two spliced mRNP purifications (mRNP [Ad] and mRNP [PIP]) for which we had heretofore published only a very limited subset of the data (Reichert et al. 2002). For the unspliced control purification (Fig. 1B, lane 2), the only proteins identified were MS2-MBP and CBP20; thus no peptides are noted in Table 1 for that preparation.

Note that for each protein, many factors in addition to protein abundance contribute to the number of unique peptides detected. Larger proteins will usually produce more unique peptides than smaller proteins, and individual peptides vary in their ability to be detected by mass spectrometry (Jurica and Moore 2003). Further, the complexity of the sample can adversely affect protein detection (Peng et al. 2003). Nonetheless, the number of unique peptides observed for a particular protein can give a very rough estimate of abundance if both the size of the protein and number of potential peptides are taken into account.

Previously known EJC proteins

Many of the proteins previously reported to associate with the EJC were identified by multiple peptides hits in our purified EJC preparation (Table 1; known EJC proteins found). All of these proteins also yielded more peptide hits in the purified EJC than we had previously detected in C complex (Jurica et al. 2002). Thus the current purification led to a significant enrichment of known EJC proteins.

Tange et al.

TABLE 1. Mass spectrometry (LC-MS/MS) analysis of the purified EJC

| Name | MW | Accession # | EJC(Ad) | mRNP (Ad) | mRNP (PIP) | C(Ad) |
|------|----|-------------|---------|-----------|------------|-------|
| Known EJC proteins found |  |  |  |  |  |  |
| eIF4AIII | 47 | P38919 | 12 | 14 | 11 | 10 |
| Pinin | 82 | Q6P5X4 | 11 | 3 | 5 | 6 |
| MLN 51/Btz | 76 | O15234 | 9 | 0 | 0 | 0 |
| Y14 | 20 | Q9Y5S9 | 7 | 6 | 9 | 3 |
| Magoh | 17 | P61326 | 6 | 5 | 7 | 1 |
| Aly/REF | 25 | Q86V81 | 4 | 2 | 2 | 1 |
| RNPS1 | 34 | Q15287 | 2 | 5 | 4 | 1 |
| EJC associated proteins not found |  |  |  |  |  |  |
| SRm 160 | 94 | O60585 | 0 | 0 | 0 | 1 |
| UAP 56 | 49 | Q13838 | 0 | 0 | 0 | 0 |
| NXF1/TAP | 70 | Q9UBU9 | 0 | 0 | 0 | 0 |
| p15 | 16 | Q9UKK6 | 0 | 0 | 0 | 0 |
| Upf3b | 58 | Q9BZ17 | 0 | 0 | 0 | 0 |
| Upf3a | 55 | Q9H1J1 | 0 | 0 | 0 | 0 |
| Upf2 | 148 | Q9HAU5 | 0 | 0 | 0 | 0 |
| Upf1 | 124 | Q92900 | 0 | 1 | 0 | 0 |
| Novel EJC proteins |  |  |  |  |  |  |
| AcinusL, S', S | 152, 71, 68 | Q9UKV3 | 19 | 12 | 9 | 0 |
| SAP18 | 18 | O00422 | 5 | 0 | 0 | 0 |
| Other proteins co-purified |  |  |  |  |  |  |
| U5-116 | 109 | Q15029 | 8 | 14 | 11 | 26 |
| CDC5L | 95 | Q99974 | 8 | 6 | 6 | 18 |
| SRm300 | 252 | Q9UHA8 | 6 | 0 | 0 | 17 |
| SF2/ASF | 28 | Q07955 | 5 | 7 | 0 | 1 |
| PRP19 | 55 | Q9UMS4 | 5 | 6 | 2 | 21 |
| Cycloph.E | 33 | Q9UNP9 | 4 | 7 | 4 | 18 |
| GCIP-IP | 29 | O95926 | 4 | 1 | 0 | 13 |
| SF3b155 | 146 | O75533 | 4 | 1 | 5 | 7 |
| CCDC9 | 60 | Q9Y3X0 | 4 | 0 | 0 | 0 |
| ARS2 | 101 | Q9BXP5 | 4 | 4 | 8 | 0 |
| CBP80 | 92 | Q09161 | 4 | 21 | 17 | 12 |
| PRP8 | 274 | Q6P2Q9 | 4 | 0 | 1 | 39 |
| Cactin | 94 | O75229 | 3 | 3 | 2 | 0 | 8 |
| U5-200 | 195 | O75643 | 3 | 1 | 1 | 50 |
| 9G8 | 27 | Q16629 | 3 | 3 | 0 | 1 |
| Calgran.B | 13 | P06702 | 3 | 0 | 0 | 0 |
| CBP20 | 18 | P52298 | 3 | 2 | 2 | 1 |
| hnRNPA0 | 31 | Q13151 | 3 | 0 | 0 | 0 |

The number of unique peptides identified for each factor in the purified EJC (EJC [Ad]) is compared with the number of unique peptides found for the same factor in the purified AdML mRNP(mRNP [Ad]) or PIP mRNP(mRNP [PIP]) (Reichert et al. 2002; previously unpublished) or in the purified AdML C complex (Jurica et al. 2002). Accession # indicates UniProt accession numbers. MW indicates the calculated molecular weight in kilodaltons. The “Known EJC proteins found” and “EJC associated proteins not found” groups comprise proteins previously reported to be part of or to interact with the EJC (see Introduction). “Novel EJC proteins” are proteins identified and confirmed by immunoprecipitation as novel EJC proteins in this study. “Other proteins co-purified” include all other proteins identified by more than three unique peptides in the purified EJC (excluding MS2-MBP).

This set included eIF4AIII, Pinin, MLN 51, Y14, Magoh, Aly/REF, and RNPS1. Except for MLN 51, peptides for all of these proteins were also detected in the two spliced mRNP preps. MLN 51 was nonetheless readily detectable by Western blotting in the spliced AdML mRNP used for the EJC prep (Fig. 2A), and α-MLN 51 antibodies selectively precipitated EJC carrying AdML and β-globin mRNAs from in vitro splicing reactions (Fig. 3A, lane 13; Fig. 3B, lane 11). Thus MLN 51 is a bona fide component of the spliced mRNP. Perhaps the greater protein complexity in the mRNP samples compared with the purified EJC somehow masked MLN 51's presence in the mass spec analysis.

Other proteins previously reported as EJC associated were either undetectable in all four mass spec analyses or were represented by a single peptide in just one sample (Table 1; EJC associated proteins not found). This set included SRm 160, UAP 56, Upf3a/b, Upf2, Upf1, and NXF1/TAP:p15. We have previously shown by Western blotting that the failure to detect peptides for SRm 160 and UAP 56 in purified spliced AdML mRNP was due to the absence of these proteins from the preparation (Reichert et al. 2002). This also seems to be the case with NXF1/TAP and Upf3b, as neither could be detected in purified spliced AdML mRNP by Western blotting (Fig. 2A; data not shown). Indeed, co-immunoprecipitation of spliced mRNA by NXF1/TAP and Upf3a/b antibodies has only been observed in samples derived from intact cells (Lykke-Andersen et al. 2000, 2001; Le Hir et al. 2001b; Gehring et al. 2003) or in which Upf3a/b was present at elevated levels (Kim et al. 2001a). Because NXF1/TAP and p15 form a tight heterodimer, the absence of NXF1/TAP can be inferred to reflect the absence of p15 from purified spliced mRNP. Likewise, Upf3b is thought to serve as the bridge linking Upf2 and Upf1 to the EJC, so its absence from the purified spliced mRNP supports the mass spec data indicating that neither Upf2 nor Upf1 is present.

Taken together, the data indicate that SRm 160, UAP 56, Upf3a/b, Upf2, Upf1, and NXF1/TAP:p15 either do not associate stably enough with the in vitro spliced mRNP to survive size exclusion and amylose affinity purification, or that EJC assembly in in vitro splicing reactions does not recapitulate all of the protein-protein interactions occurring in vivo. Nevertheless, the absence of this set of proteins from the in vitro-derived EJC indicates that their association is not necessary for EJC stability.

Downloaded from rnajournal.cshlp.org on April 22, 2015 - Published by Cold Spring Harbor Laboratory Press

Tetrameric EJC core

specific to AcinusL (data not shown). Western blotting with an antibody recognizing all three Acinus isoforms demonstrated that both the long and short isoforms were present in the purified EJC (Fig. 2B). However, since AcinusS and AcinusS' migrated at almost the same position on the gel, we could not distinguish if one of these isoforms was more prominent in the EJC prep than the other.

Recently, AcinusL and S' were found to associate in HeLa cells with RNPS1 and another factor, SAP18 (Zhang et al. 1997) in a stable heterotrimeric complex dubbed the “ASAP complex” for apoptosis and splicing associated proteins (Schwerk et al. 2003). Several unique peptides for SAP18 were also present in the EJC preparation (Table 1; novel EJC proteins). This suggested that the ASAP trimer was part of the in vitro-derived EJC. To confirm this, we performed a series of immunoprecipitation experiments from in vitro splicing reactions. Previous studies have shown that splicing-dependent association of EJC proteins with the region upstream of the exon-exon junction is maintained even after RNase H digestion of the spliced product to either side of the EJC deposition site. However, the 5′ exon must be initially of sufficient length to accept the complex. That is, whereas a 38-nt 5′ exon is long enough to accommodate the EJC, a 17-nt 5′ exon is not and antibodies against known EJC proteins do not co-precipitate spliced mRNAs with 5′ exon lengths ≤17 nt (Le Hir et al. 2000a, 2001b).

We used these same strategies to verify that Acinus and SAP18 are bona fide EJC factors. Incubation of splicing reactions containing AdML spliced products with a DNA

FIGURE 2. Immuno-analysis of purified mRNP and EJC. (A) Western blot of the HeLa nuclear extract (lanes 1,2) used for the EJC purification and the amylose affinity purified mRNP (lane 3). The blot was first probed with antibody against MLN51 (upper panel), then stripped and reprobed with antibody against Upf3b (lower panel). (B) Western blot of HeLa nuclear extract (lanes 1,2) used for the EJC purification and the purified EJC (lane 3) with an antibody recognizing all three Acinus isoforms (positions as indicated).

### Identification of Acinus and SAP18 as novel, late-associating EJC factors

Unexpectedly, the protein identified in the EJC prep by the highest number of unique peptide hits (19) was a polypeptide not previously known to be EJC associated. This factor, Acinus, was previously described as a protein that, upon proteolytic cleavage by caspases, can induce apoptosis and chromatin condensation in pre-apoptotic cells (Sahara et al. 1999). Acinus is expressed in three isoforms, AcinusL, AcinusS, and AcinusS', with the two shorter isoforms constituting the C-terminal region of the longer form (Sahara et al. 1999). Peptides identified in the EJC preparation included some common to all three isoforms and others

FIGURE 3. Immunoprecipitation of in vitro splicing reactions. (A) Denaturing 15% gel showing AdML splicing products and immunoprecipitated RNA fragments. The positions of pre-mRNA, mRNA, debranched lariat, and the two fragments resulting from RNase H cleavage of the mRNA are indicated on the left. The round object depicts the deposited EJC. The line beneath the mRNA 5′ exon indicates the position of the DNA oligo used for RNase H digestion. Odd-numbered lanes are reactions with pre-mRNA and even-numbered lanes are reactions with unspliced control mRNA. (Lanes 1,2) Completed splicing reactions. (Lanes 3,4) RNase H digested reactions. (Lanes 5,6) Markers created by RNase H digestion of unspliced pre-mRNA and control mRNA for 10 min in NE supplemented with the DNA oligo (the center of the 12-nucleotide oligo is 49 nt upstream of the exon junction). (Lanes 7–22) Immunoprecipitated reactions using antibodies indicated above lanes. Lanes 1–4 contain half as much input RNA as lanes 7–22. (B) Denaturing 15% gel showing β-globin splicing products and immunoprecipitated RNA fragments. Odd- and even-numbered lanes contain reactions with 38- and 17-nt 5′ exon β-globin RNA, respectively. The migration positions of pre-mRNAs and mRNAs are indicated to the left. (Lanes 1,2) Unspliced pre-mRNAs. (Lanes 3,4) Completed splicing reactions. (Lanes 5–22) Immunoprecipitated reactions using the indicated antibodies. Lanes 3 and 4 contain half as much input RNA as lanes 5–22. The high background of β-globin pre-mRNA and lariat observed in all immunoprecipitations likely indicates a general affinity of the β-globin intron for protein G beads.

Tange et al.

oligo complementary to a sequence 5′ of the EJC deposition site resulted in RNase H cleavage of the spliced mRNA into two fragments, the longer of which carried the EJC. Antibodies against two known EJC proteins, Y14 and MLN51, served as positive controls and selectively co-precipitated this longer fragment of the spliced mRNA, but precipitated neither fragment from an intronless control mRNA (Fig. 3A, lanes 7,8,13,14). Antibodies against Acinus and SAP18 yielded similar results (Fig. 3A, lanes 9–12). Like the Y14 and MLN51 antibodies, Acinus and SAP18 antibodies also efficiently co-precipitated spliced β-globin mRNA having a 38-nt 5′ exon, but not a version containing a 17-nt 5′ exon (Fig. 3B, lanes 5–12). Thus, Acinus and SAP18 appear to be general EJC factors in that they can associate with EJCs deposited on at least two different spliced mRNAs.

### Other proteins

The mass spectrometry analysis also yielded a large group of additional factors. Those for which three or more unique peptides were detected in the EJC prep are also listed in Table 1 (other proteins co-purified). Many of these were U5 and U2 snRNP specific proteins (U5–116, U5–200, PRP8, SF3b155), or components of the spliceosome-associated PRP19 complex (CDC5L, PRP19), suggesting some co-purification of earlier splicing complexes and/or partly disassembled spliceosomes (Jurica and Moore 2003). A subset of these factors, including CDC5L, SF3b155, U5–116, PRP19, and SF2/ASF, was examined in the above immunoprecipitation assays (Fig. 3A,B). With the exception of SF2/ASF, no antibody recognizing a member of this set of proteins specifically co-precipitated any EJC-associated RNA. Rather, antibodies against U5–116 and SF3b155 efficiently co-precipitated either the unspliced pre-mRNA or intron product (Fig. 3A, lanes 19–22; Fig. 3B, lanes 17–20). Antibodies against CDC5L and PRP19 exhibited similar co-precipitation profiles, although at much reduced efficiencies (Fig. 3A, lane 17,18; Fig. 3B, lanes 15,16,21,22). Whereas SF2/ASF antibodies weakly co-precipitated the EJC-containing fragment of spliced AdML mRNA (Fig. 3A, lanes 15,16), no specific precipitation of β-globin mRNA containing the 38-nt 5′ exon was obtained (Fig. 3B, lanes 13,14).

On the basis of these results, we conclude that most of the other proteins detected in the mass spectrometry analysis of the purified EJC represented contaminants from earlier splicing complexes. In most cases, this was consistent with greater numbers of peptide hits being detected for these proteins in C complex than in the EJC preparation and mRNP purification. However, SF2/ASF may constitute a peripheral EJC interacting protein whose EJC association is dependent on flanking exonic sequences. Consistent with this, SF2/ASF was only detected in the purified AdML mRNP and not the PIP mRNP (Table 1; other proteins co-purified).

SAP18 is a shuttling protein whereas Acinus is restricted to the nucleus

Many EJC factors, including eIF4AIll, Magoh, Y14, MLN51, RNPS1, and Aly/REF, have been shown to shuttle between the nucleus and the cytoplasm, with Y14, Magoh, eIF4AIll, and MLN51 having been directly demonstrated to associate with spliced mRNA in the cytoplasm (Kataoka et al. 2001; Kim et al. 2001b; Le Hir et al. 2001a,b; Lykke-Andersen et al. 2001; Degot et al. 2004; Palacios et al. 2004). The novel EJC factors Acinus and SAP18 are both predominantly nuclear, co-localizing with the splicing factor SC35 in nuclear speckles (Schwerk et al. 2003). To determine whether SAP18 and Acinus are also shuttling proteins, we performed heterokaryon analyses. Briefly, HeLa cells were transfected with GFP-tagged SAP18, AcinusL, or AcinusS′ together with either a nonshuttling or shuttling control, MS2-DEK or MS2-hnRNP Al, respectively (Fig. 4A–C). Subsequently, the HeLa cells were fused to mouse NIH3T3 cells in the presence of the translational inhibitor cycloheximide to produce heterokaryons.

In the GFP-SAP18 heterokaryons, a significant proportion of GFP-SAP18 migrated to the mouse nucleus, whereas the nonshuttling control MS2-DEK remained restricted to the HeLa nucleus (Fig. 4A). In contrast, similar analysis performed with AcinusL-GFP (Fig. 4B) and AcinusS′-GFP (Fig. 4C) showed that both AcinusL and AcinusS′ remained restricted to the human nucleus, whereas the shuttling control MS2-hnRNP Al migrated to the mouse nucleus. To rule out the possibility that the C-terminal GFP tag had an inhibitory effect on Acinus localization, we repeated the heterokaryon assays with a small N-terminal Flag tag on either AcinusL or AcinusS′. In those assays, co-transfected eIF4AIll-GFP served as the shuttling control (Shibuya et al. 2004). Again, both Flag-tagged Acinus constructs remained restricted to the nucleus (Fig. 4D,E). We therefore conclude that SAP18 is a shuttling protein, whereas AcinusL and AcinusS′ are nuclear restricted proteins.

### Tethered Acinus and MLN51 stimulate gene expression in different ways

To investigate potential functions of the new EJC factors, we next turned to a previously described Renilla luciferase reporter system containing MS2 binding loops internal to the Renilla open reading frame (Fig. 5A; Nott et al. 2004). The effect of tethering MS2 fusion proteins at this position can be evaluated at the RNA level by RNase protection assays (RPAs) and at the protein level by measuring luciferase activity. A co-transfected plasmid encoding firefly luciferase was used to normalize Renilla luciferase mRNA and protein levels in each transfection. MS2 alone and MS2-hnRNP Al were used as neutral controls, and all expression levels were normalized to those obtained with MS2 alone (Fig. 5B,C).

Downloaded from rnajournal.cshlp.org on April 22, 2015 - Published by Cold Spring Harbor Laboratory Press

Tetrameric EJC core

**A**
- SAP18-GFP
- MS2-DEK
- DAPI

**B**
- AcinusL-GFP
- MS2-hnRNP A1
- DAPI

**C**
- AcinusS'-GFP
- MS2-hnRNP A1
- DAPI

**D**
- Flag-AcinusL
- eIF4AIII-GFP
- DAPI

**E**
- Flag-AcinusS'
- eIF4AIII-GFP
- DAPI

FIGURE 4. Heterokaryon analysis. SAP18 is a shuttling protein; AcinusL and AcinusS' are nonshuttling proteins. (A) Localization of GFP-SAP18 (green) and nonshuttling control MS2-DEK (red) in a fused HeLa and a mouse NIH3T3 heterokaryon. The mouse nucleus is indicated by a white arrow in the DAPI-stained panel. (B,C) Localization of AcinusL-GFP (green) and AcinusS'-GFP (green) and the shuttling control MS2-hnRNP A1 (red) in a fused HeLa and a mouse NIH3T3 heterokaryon. The mouse nuclei are indicated by white arrows in the DAPI-stained panels. (D,E) Localization of Flag-AcinusL (red) and Flag-AcinusS' (red) and the shuttling control eIF4AIII-GFP (green) in a fused HeLa and a mouse NIH3T3 heterokaryon. The mouse nuclei are indicated by white arrows in the DAPI-stained panels.

As we previously reported, co-expression of MS2-RNPS1 with this reporter significantly enhanced overall gene expression (Nott et al. 2004). This enhancement was attributable to increases both in mRNA levels and in translational yield (i.e., the number of protein molecules made per mRNA molecule). Whereas we saw no effect of co-expressing MS2-SAP18, co-expression of MS2-AcinusS' increased the level of reporter mRNA but had no additional effect on translational yield (Fig. 5B,C). This is consistent with the above data showing that Acinus is confined to the nucleus.

Tethering of MS2-MLN51 also had a positive effect on luciferase expression, but only at the level of translational yield (Fig. 5B,C). This is consistent with MLN51 being a shuttling protein and is similar to the effects produced upon tethering of Y14 and Magoh (Nott et al. 2004). Thus, these data show that, when tethered to an mRNA reporter as MS2 fusion proteins, Acinus and MLN51 act to promote gene expression, but in different ways.

### Stable association of RNPS1 and SAP18 requires Acinus

With the complete set of proteins stably associated with the in vitro-derived EJC in hand, we next wanted to determine which subset might represent the minimal stable EJC core. To do so, we set about overexpressing various subsets of EJC factors in HeLa cells, with one factor containing a

**A**

![Diagram](https://example.com/diagram1.png)

**B**

![RNA Analysis](https://example.com/rna_analysis.png)

**C**

![Histogram](https://example.com/histogram.png)

FIGURE 5. MS2 tethering assay. (A) Schematic of Renilla reporter with six MS2 binding loops at the 5′ end of the open reading frame and co-transfected firefly control. (B) RNase protection assay showing Renilla, firefly, and endogenous cyclophilin RNAs. Co-transfected MS2 fusion protein plasmids are indicated above. (C) Histogram depicting the combined results of three independent experiments. (White bars) Renilla RNA normalized to firefly RNA relative to cells expressing MS2 alone. (Gray bars) Renilla luciferase activity normalized to firefly activity relative to cells expressing MS2 alone. (Black bars) Gray bars/white bars = net effect on translational yield.

tandem affinity purification (TAP)-tag for purification of  
that protein plus its stable interaction partners (Puig et al.  
2001). This approach allowed us to assess which subcom-  
plexes were stable in vivo, and circumvent the possibility  
that some eukaryotic proteins may not express and/or fold  
properly in bacteria.  

Since RNPS1, Acinus, and SAP18 were already known to  
form a stable trimeric complex in vivo (Schwerk et al.  
2003), this set served as our positive control (Fig. 6). Co-  
expression of RNPS1-TAP and untagged AcinusS' followed  
by tandem affinity purification yielded the two proteins in  
approximately equimolar amounts (as estimated by silver  
staining) with no other major co-purifying bands (Fig. 6,  
lane 1). Similar results were obtained for the RNPS1-TAP  
and AcinusL combination, although AcinusL did not ex-  
hibit very high expression levels (data not shown). Thus  
RNPS1-TAP and AcinusS' interact directly and form a  
stable heterodimer. In contrast, when RNPS1-TAP was  
co-expressed with untagged SAP18, SAP18 did not co-  
purify (Fig. 6, lane 3). SAP18 did co-purify with RNPS1-  
TAP, however, when AcinusS' was also co-expressed (Fig.  
6, lane 2). Western blot analysis of the HeLa lysate used for  
the purifications showed that SAP18 was highly over-  
expressed in both cases (Fig. 6, cf. Western blot in lane 1  
with lanes 2,3). Similar interactions between AcinusS',  
RNPS1, and SAP18 could be observed when the TAP-tag  
was placed on AcinusS'; that is, whereas SAP18 did not co-  
purify with AcinusS'-TAP alone, it did stably associate  
with the AcinusS'-TAP:RNPS1 heterodimer (data not  
shown). Thus, the ASAP trimer appears to form upon  
SAP18 interaction with a preformed Acinus:RNPS1 het-  
erodimer.

A stable tetramer of MLN51, Magoh, Y14, and eIF4AIII  

An earlier report demonstrated that the highly stable  
Magoh:Y14 heterodimer can associate with eIF4AIII when  
all three are co-expressed in bacteria, and two-hybrid anal-  
ysis indicated that MLN51 interacts directly with eIF4AIII  
(Palacios et al. 2004). To test if stable interactions among  
these factors could be observed in HeLa cells, TAP-tagged  
Y14 was co-expressed with various combinations of  
untagged Magoh, eIF4AIII, and Flag-MLN51 (Fig. 7A).  
When TAP-tagged Y14 was co-expressed with Magoh and  
eIF4AIII, only Magoh was co-purified (Fig. 7A, lane 1).  
Thus, despite being overexpressed, eIF4AIII failed to co-  
purify with the heterodimer (Fig. 7A, cf. Western blot lanes  
1,2). Next Y14-TAP, Magoh and Flag-MLN51 were co-  
expressed. Under these conditions, faint bands of eIF4AIII  
and MLN51 could now be observed co-purifying with  
Magoh:Y14, suggesting that overexpression of Flag-  
MLN51 had induced association of itself as well as endo-  
genous eIF4AIII with Magoh:Y14 (Fig. 7A, lane 2). When  
Y14-TAP, Magoh, Flag-MLN51, and eIF4AIII were all over-  

FIGURE 6. Silver-stained gel of purified complexes containing  
RNPS1-TAP, AcinusS', and/or SAP18. HeLa cells were transfected  
with combinations of expression plasmids for the indicated proteins.  
The indicated TAP-tagged protein was purified along with bound  
factors by way of the Protein A (ProtA) and calmodulin binding  
protein (CBP) modules separated by a Tev protease cleavage site,  
which together constitute the TAP tag. (Lane 1) Co-expression of  
RNPS1-TAP and AcinusS'. (Lane 2) Co-expression of RNPS1-TAP,  
AcinusS', and SAP18. (Lane 3) Co-expression of RNPS1-TAP and  
SAP18. Western blot shows the input levels of SAP18 in the three  
lysates prior to TAP purification.  

expressed, both of the two latter proteins clearly co-purified  
with Magoh:Y14 (Fig. 7A, lane 3).  

A visual estimation of relative protein quantities in the  
Coomassie-stained gel suggested that Y14-TAP and Magoh  
purified in a 1:1 ratio, and when all four factors were co-  
expressed, eIF4AIII and Flag-MLN51 also co-purified in a  
1:1 ratio (Fig. 7A, lane 3). However, the Magoh:Y14 hetero-  
dimer was clearly much more abundant in lane 3 than the  
heterotetrameric complex with Flag-MLN51 and eIF4AIII.  
Similar results were obtained when the TAP tag was placed  
on eIF4AIII. In that case the pattern was reversed with the  
quantity of Magoh:Y14 co-purified being significantly less  
than that of eIF4AIII and Flag-MLN51. Further, stable  
association of Magoh:Y14 with eIF4AIII-TAP required co-  
expression of Flag-MLN51 (data not shown).  

To allow specific purification of the heterotetrameric  
complex, the TAP-tag was split between Y14 and eIF4AIII,  
with the calmodulin binding protein (CBP) module on  
eIF4AIII (eIF4AIII-CBP) and the Protein A module and  
its attendant Tev protease cleavage site on Y14 (Y14-Tev-  
ProtA). Under these conditions, a balanced ratio of all four  
co-expressed proteins was co-purified (Fig. 7B). To rule out  
the possibility that the Magoh:Y14 heterodimer bound  
unspecifically to the calmodulin resin, we performed the  
same experiment co-expressing Y14-TevProtA, Magoh, and  


This Markdown output includes the OCR text from the image without any additional formatting or interpretation. The figure caption and text have been transcribed exactly as they appear in the image.
Downloaded from rnajournal.cshlp.org on April 22, 2015 - Published by Cold Spring Harbor Laboratory Press

Tetrameric EJC core

A
Y14-TAP

B

Magoh/ elF4AIII
Magoh/ Flag-MLN51
Magoh/ Flag-MLN51/ elF4AIII

kDa
97
66
45
31
21

Flag-MLN51
elF4AIII
Y14-CBP
Magoh
Flag-MLN51
elF4AIII

Western (input)

1   2   3

kDa
97
66
45
31
21

Flag-MLN51
elF4AIII-CBP
Y14
Magoh
Flag-MLN51
elF4AIII-CBP
elF4AIII
Y14-TevProtA

Western (input)

1   2

FIGURE 7. Coomassie-stained gels of purified complexes containing Y14, Magoh, eIF4AIII, and/or MLN51. (A) Co-expression of Y14-TAP, Magoh, and eIF4AIII (lane 1); co-expression of Y14-TAP, Magoh, and Flag-MLN51 (lane 2); and co-expression of Y14-TAP, Magoh, eIF4AIII, and Flag-MLN51 (lane 3). Western blot panels show input levels of Flag-MLN51 and eIF4AIII in all lysates prior to TAP purification. (B) Proteins purified upon co-expression of Y14-TevProtA, Magoh, eIF4AIII-CBP, and Flag-MLN51. Western blot panels show input levels of Flag-MLN51, eIF4AIII-CBP, eIF4AIII (endogenous), and Y14-TevProtA (detected by α-eIF4AIII via the ProtA tag) in the lysate prior to TAP purification.

Flag-MLN51 without eIF4AIII-CBP. Under those conditions, we did not detect any binding of Y14 and Magoh (Fig. 7B, lane 2). Thus, the Y14, Magoh, and Flag-MLN51 that are present in Figure 7B, lane 1, all originate from binding of these proteins to eIF4AIII-CBP.

Taken together, the above experiments demonstrate that eIF4AIII, MLN51, and Magoh:Y14 are able to form a stable tetrameric complex when co-expressed in vivo. Further, both eIF4AIII and MLN51 are required for the stable association of either with the Magoh:Y14 heterodimer. That these four factors can form a stable heterotetramer suggests that they constitute the minimal EJC core.

DISCUSSION

In the present study we used mass spectrometry to investigate the biochemical composition of the EJC purified from an in vitro splicing reaction. This analysis led to the identification of at least two novel EJC factors, Acinus and SAP18, whose association with the EJC was confirmed by immunoprecipitation assays. These novel factors can form a stable heterotrimeric complex with the known EJC factor RNPS1 in vivo. Further, another subcomplex consisting of

the previously identified EJC factors eIF4AIII, MLN51, Magoh, and Y14 can form between recombinant proteins ectopically expressed in vivo. The stable assembly of this heterotetrameric complex, which likely constitutes the core of the EJC, depends on all four factors being available simultaneously. The present analysis also lends significant insight into which factors are important for the structural integrity of the EJC and which associate more transiently (Fig. 8).

Novel EJC factors Acinus and SAP18

Acinus was previously characterized as an apoptosis-inducing protein that, upon cleavage by caspases, is capable of inducing morphological changes, such as chromatin condensation, which are characteristic for cell death (Sahara et al. 1999). SAP18, on the other hand, was originally identified as a component of the SIN3-HDAC complex, which causes transcriptional repression by deacetylating histones in promoter regions (Zhang et al. 1997; Cheng and Bishop 2002). A trimeric complex consisting of Acinus, SAP18, and RNPS1 was recently isolated from HeLa cells by conventional chromatography. This complex was named the ASAP complex for apoptosis and splicing associated proteins

www.rnajournal.org 1877

Downloaded from rnajournal.cshlp.org on April 22, 2015 - Published by Cold Spring Harbor Laboratory Press

**Tange et al.**

![Diagram](https://i.imgur.com/yourimageurl.png)

FIGURE 8. Three spheres of EJC factors. The minimal EJC core likely consists of a tetrameric complex containing eIF4AIII, MLN51, Magoh, and Y14 (this study), with eIF4AIII providing direct contact to the mRNA (Shibuya et al. 2004). All factors in this core are shuttling proteins and most likely follow the mRNA to the cytoplasm (Kataoka et al. 2000, 2001; Le Hir et al. 2001a,b; Macchi et al. 2003; Degot et al. 2004; Palacios et al. 2004; Shibuya et al. 2004). Proteins in the outer shell were all found by mass spectrometry of the in vitro-derived EJC (this study). RNPS1, Acinus, and SAP18 can stably associate (Schwerk et al. 2003) and may bind the EJC core as a trimeric complex. However, RNPS1 may also bind alone, e.g., via interactions with Pinin (Sakashita et al. 2004). SAP18, RNPS1, and Aly/REF are shuttling proteins (this study; Zhou et al. 2000; Lykke-Andersen et al. 2001), whereas Acinus and Pinin are nuclear restricted (this study; Li et al. 2003). Transiently interacting factors are proteins not identified in the in vitro-derived EJC, but which likely interact dynamically with either the EJC core or outer sphere proteins (see text).

(Schwerk et al. 2003). In line with that report, we were able to generate a stable heterotrimeric complex by co-expressing these three EJC factors together in HeLa cells and purifying them with a TAP tag fused to RNPS1. Further, we found that RNPS1 and Acinus can form a stable heterodimer and SAP18 can stably associate only with this heterodimer. The inability of SAP18 to interact stably with either RNPS1 or Acinus alone is in agreement with earlier experiments by Reinberg and coworkers (Schwerk et al. 2003) showing that neither RNPS1 nor Acinus are co-immunoprecipitated with SAP18 after induction of apoptosis. Since cleavage of Acinus accompanies apoptosis, this cleavage also likely leads to disassembly of the ASAP complex (Sahara et al. 1999; Schwerk et al. 2003).

What implications do the presence of Acinus and SAP18 have for EJC function? Microinjection of purified ASAP complex into HeLa cells significantly enhanced apoptosis when induced by external stimuli (Schwerk et al. 2003). This effect was most likely attributable to the actions of Acinus (Sahara et al. 1999). Whether the EJC has any involvement in apoptosis or chromatin remodeling is as yet unknown. Previous reports regarding SAP18 suggest that it is involved in transcriptional regulation. Notably, tethering of SAP18 to a promoter via a DNA binding domain represses transcription, suggesting that tethered SAP18 can recruit the SIN3-HDAC complex leading to transcriptional repression via histone deacetylation (Zhang et al. 1997). Thus, one intriguing possibility is that the EJC could be involved in transcriptional regulation. For example, under certain conditions EJCs deposited on newly transcribed and spliced RNAs might serve to recruit repressor complexes to active genes.

### A stable tetrameric EJC core

A central question regarding EJC structure is how it assembles and remains so tightly bound to mRNA in a sequence-independent fashion. Recently, we demonstrated that eIF4AIII interacts directly with mRNA at the position of the EJC in a splicing-dependent manner (Shibuya et al. 2004). Further, the 8- to 10-nt EJC footprint (Le Hir et al. 2000a) is remarkably similar to the known binding site size for other DEAD-box proteins (Tanner and Linder 2001). Therefore, eIF4AIII likely constitutes part of the platform upon which the rest of the EJC forms. However, other factors close to eIF4AIII in the EJC may assist its binding to mRNA or prevent its dissociation. The factors Magoh, Y14, and MLN51 had all been previously reported to directly contact eIF4AIII (Palacios et al. 2004; Shibuya et al. 2004). Although Y14 contains an apparent RNA recognition motif (RRM), this sequence is buried at the interface of Y14’s interaction with Magoh (Fribourg et al. 2003; Lau et al. 2003; Shi and Xu 2003), suggesting that Y14 is not involved in binding mRNA. MLN51, on the other hand, has been reported to have some nonspecific affinity for RNA (Degot et al. 2004) and may thus assist in binding the EJC core to mRNA.

The TAP purification studies presented here demonstrate that eIF4AIII, Magoh, Y14, and MLN51 can assemble as a tetrameric complex with apparently equal stoichiometries in vivo. Notably, all four factors had to be simultaneously co-expressed in order to form the stable complex in HeLa cells. This tetramer is in all likelihood the minimal inner EJC core (Fig. 8). The eIF4AIII:Y14:Magoh:MLN51 heterotetramer was purified in the presence of RNase A, indicating that flanking RNA is not required for its stability. This does not exclude the possibility, however, that a small RNA fragment is protected by the tetramer and that this fragment is required for stability of the complex. Whether the assembly of the tetramer requires the splicing process and/or RNA will be addressed in future experiments.

Although MLN51 is a nucleocytoplasmic shuttling protein, it exhibits predominantly cytoplasmic localization (Macchi et al. 2003). This contrasts with eIF4AIII, Magoh, and Y14, which also shuttle but are predominantly nuclear (Kataoka et al. 2000, 2001; Ferraiuolo et al. 2004; Shibuya et al. 2004). It seems possible that the low nuclear concentration of MLN51 may function to prevent formation of the stable EJC tetramer core inadvertently on RNA without the splicing process to orchestrate its assembly. Thus, the splicing process may be required to recruit MLN51 from its low nuclear pool to form a stable EJC core on spliced mRNA.

All four core factors have been shown to associate with spliced mRNA in the cytoplasm (Kataoka et al. 2001; Kim et al. 2001b; Le Hir et al. 2001a,b; Degot et al. 2004; Palacios et al. 2004; Shibuya et al. 2004), where they imprint the positions of exon-exon boundaries and exert functions attributable to the EJC. In line with this, we have shown in this and a previous report that tethered Y14, Magoh, and MLN51 all have significant positive effects on mRNA translational efficiency (Nott et al. 2004). This strongly suggests that these factors function to facilitate mRNA association with the translational machinery as part of the EJC (Wiegand et al. 2003; Nott et al. 2004). Given the role of these factors in asymmetric mRNA localization in *Drosophila* and human neuronal cells (Mohr et al. 2001; van Eeden et al. 2001; Palacios 2002; Macchi et al. 2003; Hachet and Ephrussi 2004; Palacios et al. 2004), it seems reasonable to propose that they may also facilitate recruitment of the translation machinery once they have participated in mRNA localization.

It should be noted that the effect of tethering certain EJC factors differs according to where on the reporter mRNA the tethering occurs. Tethering of the EJC and NMD factors Magoh, Y14, RNPS1, MLN51, Upf3b, Upf2, and Upf1 in the 3′ UTR has been shown to down-regulate an mRNA reporter, presumably because the upstream stop codon is recognized as premature when any of these factors are present in the 3′ UTR (Lykke-Andersen et al. 2000, 2001; Fribourg et al. 2003; Gehring et al. 2003; Nott et al. 2004; Palacios et al. 2004). However, when the same factors are tethered inside an ORF, they all stimulate mRNA translational efficiency (Nott et al. 2004, this study). Thus, the function of these EJC and NMD factors depends on their specific location on the mRNA (Wilkinson 2005).

When tethered as an MS2-fusion, we found that eIF4AIII had only minor effects on gene expression (data not shown). A λN-eIF4AIII fusion also failed to elicit NMD when tethered downstream of a stop codon (Palacios et al. 2004) despite this factor's demonstrated importance for NMD in mammalian cells (Ferraiuolo et al. 2004; Palacios et al. 2004; Shibuya et al. 2004). This may indicate that outside of the context of the full EJC, tethered eIF4AIII cannot adopt the appropriate conformation to efficiently recruit other factors necessary for either translational enhancement or NMD. Perhaps within the EJC, eIF4AIII functions mainly as a structural element to coordinate complex assembly and stability, while the other core factors, which can promote mRNA translational efficiency and NMD when tethered independently, provide the binding interfaces for the translation and NMD machineries.

EJC structure and dynamics

If eIF4AIII, Magoh, Y14, and MLN51 constitute the EJC inner core, then the other factors identified in the purified EJC can be considered EJC outer shell components (Fig. 8).

This set includes RNPS1, Acinus, SAP18, Pinin, and Aly/REF. The molecular weight of the in vitro-derived EJC was earlier estimated by gel filtration to be ~335 kDa (Le Hir et al. 2000a). Based on the theoretical molecular weights of the individual factors (Table 1), the weight of the tetrameric core should be 160 kDa. Addition of Pinin, Aly/REF, RNPS1, SAP18, and either AcinusS′ or AcinusL would bring the total size of the EJC up to 390 or 470 kDa, respectively, which is within the range of the earlier molecular weight estimate.

Currently, it is unknown how the factors in the outer shell associate with the inner core, as no direct interactions have yet been demonstrated between the two groups. Among the outer shell factors though, several interactions are known. In addition to being a component of the ASAP trimer, RNPS1 has been shown by two-hybrid analysis to interact with Pinin (Sakashita et al. 2004). Both RNPS1 and Pinin were found in purified C complex, whereas Acinus and SAP18 were not (Table 1; Jurica et al. 2002). This suggests that RNPS1, perhaps together with Pinin, initially associates with the partially assembled EJC in C complex and then recruits Acinus and SAP18 upon completion of exon ligation and/or mRNP release. It is also possible that the preassembled ASAP complex joins the EJC as a discrete unit, perhaps through the RNPS1:Pinin interaction. If this is the case, however, Acinus and SAP18 were either missed in the mass spec analysis of C complex (perhaps due to the complexity of the sample) or their association at that stage is not stable enough to survive the rather rigorous purification conditions applied (heparin + high salt + EDTA). It is also currently unknown if the RNPS1:Pinin interaction is permissive or mutually exclusive of ASAP complex formation.

In comparison to the four core factors, which have all been shown to remain associated with spliced mRNA in the cytoplasm (see above), some of the outer shell factors appear to dissociate from spliced mRNP prior nucleocytoplasmic export. Whereas SAP18, RNPS1, and Aly/REF are all shuttling proteins (Fig. 4; Zhou et al. 2000; Lykke-Andersen et al. 2001), Acinus and Pinin are restricted to the nucleus (Fig. 4; Li et al. 2003). Therefore, a major restructuring of the EJC outer shell must occur prior to or concomitant with mRNA export, with at least Pinin and Acinus dissociating prior to passage through the nuclear pore. Since Acinus is required for the stable association of SAP18 with RNPS1, and presumably therefore with the EJC, SAP18 might be expected to dissociate from the spliced mRNP during or soon after export. In mammalian cell extracts, RNPS1 could be detected in a cytoplasmic fraction in association with other EJC components and CBP80, the large subunit of the nuclear 7-methyl-G cap binding complex, but not with eIF4E, the major cytoplasmic cap binding protein. This was taken to indicate that RNPS1 remains associated with spliced mRNA in the cytoplasm until the first or “pioneering” round of translation.

sweeps off the vast majority of EJCs (Lejeune et al. 2002). This is consistent with data indicating that tethered RNPS1 can recruit both the translation and NMD machineries, presumably in the cytoplasm (Lykke-Andersen et al. 2001; Nott et al. 2004). Conversely, RNPS1 was not detectably associated with spliced RNA in the cytoplasm of Xenopus oocytes (Le Hir et al. 2001b), suggesting either fundamental differences between the mammalian and oocyte systems (Lejeune et al. 2002) or differences due to experimental design. Finally, Aly/REF does not appear to maintain any stable association with the cytoplasmic EJC, either in mammalian cells or oocytes (Kim et al. 2001b; Le Hir et al. 2001b; Lejeune et al. 2002), suggesting that it dissociates during or soon after mRNA export.

The present analysis of the in vitro-derived EJC failed to identify Upf3a/b, Upf2, Upf1, SRm160, UAP56, or NXF1/TAP:p15, which earlier reports had indicated as EJC components or interacting factors (see Introduction). The absence of this set of proteins indicates that they are not required for the structural integrity of the EJC. Functionally, Upf3b is crucial for the EJC's role in mammalian NMD and several reports support Upf3a/b's interaction with the EJC in vivo (Kim et al. 2001a; Le Hir et al. 2001b; Lykke-Andersen et al. 2001; Lejeune et al. 2002; Gehring et al. 2003). Their absence from the in vitro-derived spliced mRNP and EJC, however, suggests that Upf3a/b's association with the EJC is either transient in nature or that the in vitro system does not entirely recapitulate all of the EJC protein interactions that occur in vivo. An intriguing possibility is that association of Acinus and SAP18 with the in vitro-derived EJC is mutually exclusive of Upf3a/b binding. This possibility is consistent with data showing that Flag-tagged Upf3b can be incorporated into in vitro-derived EJCs if it is overexpressed in the cells from which the splicing extract is derived (Kim et al. 2001a; Kataoka and Dreyfuss 2004). Since Upf2 and Upf1 are thought to be recruited to the EJC by Upf3a/b (Lykke-Andersen et al. 2000; Conti and Izaurralde 2005; Lejeune and Maquat 2005), their absence from our EJC prep can be explained by the absence of Upf3a/b.

The data supporting SRm160's association with the EJC are based almost entirely on immunoprecipitation analyses with α-SRm160 antibodies (Le Hir et al. 2000a,b, 2001b). However, SRm160 does co-precipitate with Upf3a/b in nuclear fractions in an RNase-insensitive manner, suggesting a physical connection to the EJC in vivo (Lejeune et al. 2002). In vitro, SRm160's interaction with C complex has proven quite sensitive to purification conditions, and Western blotting revealed that it does not co-purify with spliced mRNP (Reichert et al. 2002). Most likely, SRm160's EJC association is transient, as are the EJC interactions of UAP56 and NXF1/TAP:p15. These latter proteins interact genetically and physically with Aly/REF, before and after Aly/REF's association with mRNA, respectively (Stutz and Izaurralde 2003). Thus, these factors probably also interact only peripherally with the EJC and belong to processes upstream and downstream of EJC assembly. Clearly additional studies will be required to

fully elucidate the processes of EJC assembly and structural evolution as it moves from the nucleus to the cytoplasm as a key component of the spliced mRNP.

# MATERIALS AND METHODS

## Plasmids

Plasmid pMS2-MBP for expressing MS2-MBP in *E. coli* was a gift from Josep Vilardell, while plasmids pGEXCS-hsMGN and pET28c-hsY14 for expressing GST-Magoh and untagged human Y14 in *E. coli* were gifts from Elisa Izaurralde (Le Hir et al. 2001a). pAdML-6MS2 encoding the AdML splicing substrate with six MS2 binding loops in the 3′ exon was derived from pHMS388 (Reichert et al. 2002) by adding three additional MS2 binding loops immediately 3′ to those in pHMS388. The control pAdMLmRNA-6MS2 was then constructed by deleting the intron from pAdML-6MS2. Plasmids encoding the AdML splicing substrate and mRNA control (AdML mRNA) used for co-immunoprecipitation analyses were as described (Le Hir et al. 2000a). Plasmids for transcribing β-globin splicing substrates with 38- and 17-nt 5′ exons (pUCT7-β/38 and pUCT7-β/17) were as previously described (Le Hir et al. 2001b). For MS2 tethering assays, plasmids encoding the firefly luciferase control (pcDNA3.1-firefly) and the *Renilla* luciferase reporter with MS2 stem-loops near the 5′ end of the *Renilla* open reading frame (pAN101) were as previously described (Nott et al. 2004). Plasmids pcDNA3.1-TAP, pcDNA3.1-TevProtA, and pcDNA3.1-CBP for constructing C-terminally TAP-tagged and split-TAP-tagged transgenes for mammalian expression were made by inserting fragments generated by PCR of pBS1479 (gift from Bertrand Seraphin) (Puig et al. 2001) into pcDNA3.1. The indicated MS2-tagged, GFP-tagged, Flag-tagged, TAP-tagged, and untagged mammalian expression constructs for MLN51, eIF4AIII, Magoh, Y14, AcinusL, AcinusS′, RNPS1, SAP18, and hnRNP A1 were constructed by inserting cDNA fragments generated by PCR of pNMS2-hnRNP A1, -RNPS1, -Magoh, -Y14, -DEK (gifts from Joan Steitz) (Lykke-Andersen et al. 2001), or pCAGGS-Flag-AcinusL, -AcinusS′ (gifts from Yoshihide Tsujimoto) (Sahara et al. 1999) or a HeLa cDNA library (Clontech) into pcDNA3.1-FlagMS2 (Nott et al. 2004), pEGFP-N3 or pEGFP-C3 (Clontech), pCMV10 3xFlag (Sigma), pcDNA3.1-TAP, pcDNA3.1-tevProtA, pcDNA3.1-CBP, or pcDNA3.1 (Invitrogen). All constructs were verified by sequencing.

## Protein expression and purification from *E. coli*

The GST-Magoh:Y14 heterodimer was generated by co-expressing pGEX-CS-hsMGN (Amp<sup>R</sup>) and pET28c-hsY14 (Kana<sup>R</sup>) in BL21(DE3) CodonPlus *E. coli* (Stratagene). Heterodimer purification was performed as previously described (Le Hir et al. 2001a). The MS2-MBP fusion protein was expressed in DH5α *E. coli* containing pMS2-MBP and purified as previously described (Jurica et al. 2002).

## EJC purification and mass spectrometry analysis

The protocol for EJC purification was adapted from the purification protocols used for C complex and for mRNP purification (Jurica et al. 2002; Reichert et al. 2002). Briefly, AdML-6MS2 RNA

or the control AdMLmRNA-6MS2 RNA were incubated under splicing conditions for 75 min at 30°C in reactions supplemented with both MS2-MBP and the GST-Magoh:Y14 heterodimer (40% HeLa nuclear extract, ~10 nM RNA, ~0.5 μM MS2-MBP, ~1.2 μM GST-Magoh:Y14, 20 mM HEPES [pH 7.9], 80 mM KCl, 8% glycerol, 2 mM magnesium acetate, 1 mM ATP, 10 mM creatine phosphate, 0.5 mM DTT, 0.2 U/μL RNasin [Promega]). To ensure efficient splicing, reactions were performed in 2-mL batches split into 10 × 200-μL aliquots; generally at least 70% splicing was obtained (data not shown). After 75 min, a mixture of six DNA oligos (0.4 μM) complementary to the bulk of the intron was added and the incubation continued for 20 min at 30°C to induce RNase H degradation of the intron. Nonspecific protein interactions were subsequently disrupted by adding 0.5 μg/μL heparin followed by incubation for 5 min more.

All subsequent steps were performed in wash buffer: 20 mM HEPES (pH 7.9), 100 mM KCl, 8% glycerol, 2 mM magnesium acetate, 0.025% NP40, 1 mM DTT. To separate the mRNP from the bulk of proteins in nuclear extract, splicing reactions (1 mL per run) were applied to a 4-mL S-300 gel filtration column; the radioactivity eluted as two peaks, with the first containing mRNP. Subsequently, the mRNP-containing peak was passed over an 80-μL amylose resin minicolumn (NEB) three times, followed by washing with ~25 column volumes. Bound mRNP was eluted with wash buffer containing 10 mM maltose. The eluted mRNP was treated with 0.05 μg/μL RNase A for 15 min at 30°C to digest the mRNA. This material was then applied to a 30-μL Glutathione Sepharose mini-column (Amersham) to purify the GST-Magoh:Y14-containing EJC. As above, the sample was passed over the column three times followed by washing with ~25 column volumes. Proteins were eluted in 10 mM HEPES (pH 7.9), 8 M urea.

Purified EJC proteins from a total of 24 mL splicing reaction (purifications performed in 2-mL batches) were combined for mass spectrometry analysis. Parallel purifications were performed on 24 mL of splicing reaction containing the control AdMLmRNA substrate without intron. Purified EJC and control proteins were TCA precipitated and loaded in single lanes of a 10% SDS-PAGE gel, which was subsequently stained with Coomassie G 250 (Sigma). The lanes for the EJC and control purifications were each cut out as three gel slab sections and the protein contents analyzed by nanoscale microcapillary liquid chromatography tandem mass spectrometry (LC-MS/MS) at the Taplin Mass Spectrometry facility at Harvard Medical School, Boston.

**Antibodies, Western blots, and immunoprecipitation**

Western blotting of HeLa nuclear extract, mRNP, and EJC samples were done with the following antibodies: α-Acinus (Ab-2, rabbit polyclonal, Oncogene Research Products); α-MLN51 (rabbit polyclonal, gift from Michael Kiebler) (Macchi et al. 2003); and α-Upf3b (rabbit polyclonal, gift from Jens Lykke-Andersen). Stripping of blot was performed with a kit (Chemicon). Immunoprecipitation of EJC factors and RNA from in vitro splicing reactions was performed essentially as described in Le Hir et al. (2000a, 2001b), using the following antibodies: α-Y14 (4C4, mouse monoclonal, ImmuQuest); α-Acinus (Ab-2, rabbit polyclonal, Oncogene Research Products); α-SAP18 (C-20, goat polyclonal, Santa Cruz Biotechnology); α-MLN51 (rabbit polyclonal, gift from Michael Kiebler) (Macchi et al. 2003); α-SF2/ASF (mAb96,

mouse monoclonal, gift from Adrian Krainer) (Hanamura et al. 1998); α-SAP155/SF3b155 (rabbit polyclonal, gift from Robin Reed) (Wang et al. 1998); α-U5–116 (rabbit polyclonal, gift from Reinhard Lührmann) (Fabrizio et al. 1997); α-CDC5L (mouse monoclonal, BD Biosciences); and α-Prp19 (rabbit polyclonal, Bethyl Laboratories). Co-precipitated RNAs were separated on 15% denaturing PAGE gels. Radioactivity was detected using a Molecular Dynamics PhosphorImager scanner.

**MS2 tethering assay**

For MS2 tethering assays, HeLa CCL-2 cells (ATCC) were grown in DMEM medium (Gibco-Invitrogen) containing 10% fetal bovine serum (Gibco-Invitrogen). Cells were seeded in six-well plates and transfected using Lipofectamine 2000 (Invitrogen) according to the manufacturer’s instructions. Each well was transfected with 250 ng 6MS2Renilla reporter, 250 ng firefly control, 350 ng pcDNA3.1, and 750 ng of the appropriate MS2 fusion protein plasmid and then incubated for 24 h at 37°C. To harvest cells, the wells were first washed with PBS, and then the cells were scraped off in 150 μL PBS using a cell scraper. Thirty microliters cells were lysed in 100 μL Passive Lysis Buffer (Promega), and 20 μL lysate was used for quantifying Renilla and firefly luciferase activities using the Dual Luciferase Assay (Promega). Total RNA was purified from 120 μL cells using TriReagent (MRC) and subjected to RNase protection analysis of Renilla, firefly, and cyclophilin mRNAs as described in Nott et al. (2004). For each transfection, Renilla protein and RNA levels were normalized to firefly protein and RNA, respectively, and then normalized relative to MS2 alone.

**Heterokaryon assay and immunofluorescence**

Heterokaryon assays were carried out as described in Shibuya et al. (2004). HeLa cells were co-transfected with plasmids expressing GFP-SAP18 or AcinusS′-GFP or AcinusL-GFP or Flag-AcinusS′ or Flag-AcinusL and nonshuttling controls MS2-DEK or shuttling controls MS2-hnRNP A1 or eIF4AIII-GFP. Next, the HeLa cells were fused with untransfected mouse NIH3T3 in the presence of cyclohexamide. For immunofluorescence, cells were fixed, permeabilized, incubated with primary and secondary antibodies, and visualized as described in Muhlemann et al. (2001). MS2-hnRNP A1 or MS2-DEK proteins were detected with α-MS2 (rabbit polyclonal) and rhodamine-labeled secondary antibody. Flag-tagged AcinusS′ or AcinusL were detected with α-Flag antibody (rabbit polyclonal) and rhodamine-labeled secondary antibody.

**TAP purification and Western analysis**

For purification of TAP-tagged proteins and co-expressed EJC factors, HeLa cells were seeded in 60 cm² plates in DMEM medium (Gibco-Invitrogen) containing 10% fetal bovine serum (Gibco-Invitrogen). Cells were transfected using Lipofectamine 2000 (Invitrogen) according to the manufacturer’s directions. Each plate was transfected with 20 μg plasmid total and then incubated at 37°C for 24 h. RNPS1-TAP, AcinusS′, and SAP18 complexes: One plate was transfected with 6.7 μg pcDNA3.1-RNPS1-TAP, 6.7 μg pcDNA3.1-AcinusS′ or pcDNA3.1, and 6.7 μg pcDNA3.1-SAP18 or pcDNA3.1.

Y14-TAP, Magoh, eIF4AIII, Flag-MLN51 complexes: Two plates were transfected with 3 μg pcDNA3.1-Y14-TAP, 3 μg pcDNA3.1-Magoh, 7 μg pcDNA3.1-eIF4AIII or pcDNA3.1, and 7 μg pCMV10–3xFlag-MLN51 or pcDNA3.1.

Y14-TevProtA, Magoh, eIF4AIII-CBP, Flag-MLN51 complexes: Two plates were transfected with 3 μg pcDNA3.1-Y14-TevProtA, 3 μg pcDNA3.1-Magoh, 7 μg pcDNA3.1-eIF4AIII-CBP, and 7 μg pCMV10–3xFlag-MLN51.

The purification protocol was adapted from Puig et al. (2001). Washed cells from one plate were resuspended in 1 mL lysis buffer (50 mM Tris [pH 8.0], 1% Triton X-100, 150 mM NaCl, 1 mM magnesium acetate, 0.1 mM ATP, 20 mM sodium fluoride, 1 mM sodium vanadate, 10 mM EGTA, 1 mM PMSF, 1 μg/mL leupeptin, 1 μg/mL aprotinin, 1 μg/mL pepstatin A) and incubated on ice for 30 min with frequent mixing. Cell lysate was clarified by centrifugation at 16,000g for 30 min at 4°C, and then added to 20 μL IgG Sepharose beads (Amersham Biosciences) equilibrated in lysis buffer. After 2 h gentle mixing at 4°C, beads were washed with 3 × 1 mL wash buffer (WB: 10 mM Tris [pH 8.0], 0.1% NP-40, 150 mM NaCl, 1 mM magnesium acetate, 0.1 mM ATP) and 1 mL wash buffer plus 1 mM DTT and 0.5 mM EDTA (WB+D+E). Beads were then incubated for 2 h at 16°C with frequent mixing in 100 μL WB+D+E containing 10 units AcTev proteinase (Invitrogen) and 0.5 μg/μL RNase A. The supernatant was transferred to a fresh tube and the beads washed with 200 μL WB+D+E. The 300-μL combined eluate was supplemented with calcium chloride to 2 mM and then combined with 700 μL calmodulin binding buffer (CBB: 10 mM Tris [pH 8.0], 0.1% NP-40, 150 mM NaCl, 1 mM magnesium acetate, 1 mM imidazole, 2 mM calcium chloride, 10 mM β-mercaptoethanol). Addition of 20 μL calmodulin affinity beads (Stratagene) equilibrated in CBB was followed by 2 h of rotation in the cold room. Beads were washed with 3 × 1 mL CBB and then eluted with 3 × 50 μL calmodulin elution buffer (CEB: 10 mM Tris [pH 8.0], 0.1% NP-40, 150 mM NaCl, 1 mM magnesium acetate, 1 mM imidazole, 2 mM EGTA, 10 mM β-mercaptoethanol). Purified proteins were TCA precipitated, separated by 13% SDS-PAGE, and visualized by silver staining (BioRad) or staining with colloidal Coomassie G250 (Sigma).

Western analyses of SAP18, Flag-MLN51, eIF4AIII, eIF4AIII-CBP, and Y14-TevProtA were performed by separating 15 μL cleared lysate on a 10% or 13% SDS-PAGE, which was then blotted onto a PVDF membrane (Immobilon). Membranes were incubated with the following primary antibodies: α-SAP18 (goat polyclonal, Santa Cruz Biotechnology); α-eIF4AIII (rabbit polyclonal, gift from N. Sonenberg) (Li et al. 1999); α-Flag (mouse monoclonal, Sigma), and the appropriate AP-conjugated secondary antibodies, visualized by ECF (Amersham Biosciences) and scanned on a Molecular Dynamics chemoluminescence scanner.

### ACKNOWLEDGMENTS

We thank E. Izaurralde and H. Le Hir for communicating results prior to publication. We are grateful to E. Izaurralde, J. Vilardell, M. Kiebler, R. Lührmann, R. Reed, A. Krainer, J. Steitz, Y. Tsuji-moto, B. Seraphin, Jens Lykke-Andersen, and N. Sonenberg for the kind gifts of constructs and antibodies as noted in the text. We acknowledge our laboratory members for helpful advice and discussions. This work was supported in part by NIH grant RO1 GM53007 (M.J.M.) and a postdoctoral fellowship from the Alfred

Benzon Foundation, Denmark (T.O.T.). M.J.M. is a Howard Hughes Medical Institute investigator.

Received July 6, 2005; accepted September 20, 2005.

#### NOTE ADDED IN PROOF

Ballut et al. recently reported that a stable complex consisting of recombinant Magoh, Y14, EIF4AIII, and MLN51 can assemble in vitro (Ballut et al. 2005).

### REFERENCES

Ballut, L., Marchadier, B., Baguet, A., Tomasetto, C., Seraphin, B., and Le Hir, H. 2005. The exon junction core complex is locked onto RNA by inhibition of EIF4AIII ATPase activity. *Nat. Struct. Mol. Biol.* **12**: 861–869.

Chan, C.C., Dostie, J., Diem, M.D., Feng, W., Mann, M., Rappsilber, J., and Dreyfuss, G. 2004. eIF4A3 is a novel component of the exon junction complex. *RNA* **10**: 200–209.

Cheng, S.Y. and Bishop, J.M. 2002. Suppressor of Fused represses Gli-mediated transcription by recruiting the SAP18-mSin3 corepressor complex. *Proc. Natl. Acad. Sci.* **99**: 5442–5447.

Conti, E. and Izaurralde, E. 2005. Nonsense-mediated mRNA decay: Molecular insights and mechanistic variations across species. *Curr. Opin. Cell Biol.* **17**: 316–325.

Degot, S., Le Hir, H., Alpy, F., Kedinger, V., Stoll, I., Wendling, C., Seraphin, B., Rio, M.C., and Tomasetto, C. 2004. Association of the breast cancer protein MLN51 with the exon junction complex via its speckle localizer and RNA binding module. *J. Biol. Chem.* **279**: 33702–33715.

Fabrizio, P., Laggerbauer, B., Lauber, J., Lane, W.S., and Luhrmann, R. 1997. An evolutionarily conserved U5 snRNP-specific protein is a GTP-binding factor closely related to the ribosomal translocase EF-2. *EMBO J.* **16**: 4092–4106.

Ferraiuolo, M.A., Lee, C., Ler, L.W., Hsu, J.L., Costa-Mattioli, M., Luo, M.J., Reed, R., and Sonenberg, N. 2004. A nuclear translation-like factor eIF4AIII is recruited to the mRNA during splicing and functions in nonsense-mediated decay. *Proc. Natl. Acad. Sci.* (in press).

Fribourg, S., Gatfield, D., Izaurralde, E., and Conti, E. 2003. A novel mode of RBD-protein recognition in the Y14-Mago complex. *Nat. Struct. Biol.* **10**: 433–439.

Gatfield, D. and Izaurralde, E. 2002. REF1/Aly and the additional exon junction complex proteins are dispensable for nuclear mRNA export. *J. Cell Biol.* **159**: 579–588.

Gatfield, D., Le Hir, H., Schmitt, C., Braun, I.C., Kocher, T., Wilm, M., and Izaurralde, E. 2001. The DExH/D box protein HEL/UAP56 is essential for mRNA nuclear export in *Drosophila*. *Curr. Biol.* **11**: 1716–1721.

Gehring, N.H., Neu-Yilik, G., Schell, T., Hentze, M.W., and Kulokiz, A.E. 2003. Y14 and hUpf3b form an NMD-activating complex. *Mol. Cell* **11**: 939–949.

Hachet, O. and Ephrussi, A. 2004. Splicing of oskar RNA in the nucleus is coupled to its cytoplasmic localization. *Nature* **428**: 959–963.

Hanamura, A., Cáceres, J.F., Mayeda, A., Franza Jr., B.R., and Krainer, A.R. 1998. Regulated tissue-specific expression of antagonistic pre-mRNA splicing factors. *RNA* **4**: 430–444.

Ishigaki, Y., Li, X., Serin, G., and Maquat, L.E. 2001. Evidence for a pioneer round of mRNA translation: mRNAs subject to nonsense-mediated decay in mammalian cells are bound by CBP80 and CBP20. *Cell* **106**: 607–617.

Jurica, M.S. and Moore, M.J. 2003. Pre-mRNA splicing: Awash in a sea of proteins. *Mol. Cell* **12**: 5–14.

Jurica, M.S., Licklider, L.J., Gygi, S.R., Grigorieff, N., and Moore, M.J. 2002. Purification and characterization of native spliceosomes suitable for three-dimensional structural analysis. *RNA* **8**: 426–439.

Kataoka, N. and Dreyfuss, G. 2004. A simple whole cell lysate system for in vitro splicing reveals a stepwise assembly of the exon-exon junction complex. *J. Biol. Chem.* **279**: 7009–7013.

Kataoka, N., Yong, J., Kim, V.N., Velazquez, F., Perkinson, R.A., Wang, F., and Dreyfuss, G. 2000. Pre-mRNA splicing imprints mRNA in the nucleus with a novel RNA-binding protein that persists in the cytoplasm. *Mol. Cell* **6**: 673–682.

Kataoka, N., Diem, M.D., Kim, V.N., Yong, J., and Dreyfuss, G. 2001. Magoh, a human homolog of *Drosophila* mago nashi protein, is a component of the splicing-dependent exon-exon junction complex. *EMBO J.* **20**: 6424–6433.

Kim, V.N., Kataoka, N., and Dreyfuss, G. 2001a. Role of the nonsense-mediated decay factor hUpf3 in the splicing-dependent exon-exon junction complex. *Science* **293**: 1832–1836.

Kim, V.N., Yong, J., Kataoka, N., Abel, L., Diem, M.D., and Dreyfuss, G. 2001b. The Y14 protein communicates to the cytoplasm the position of exon-exon junctions. *EMBO J.* **20**: 2062–2068.

Lau, C.K., Diem, M.D., Dreyfuss, G., and Van Duyne, G.D. 2003. Structure of the y14-magoh core of the exon junction complex. *Curr. Biol.* **13**: 933–941.

Le Hir, H., Izaurralde, E., Maquat, L.E., and Moore, M.J. 2000a. The spliceosome deposits multiple proteins 20–24 nucleotides upstream of mRNA exon-exon junctions. *EMBO J.* **19**: 6860–6869.

Le Hir, H., Moore, M.J., and Maquat, L.E. 2000b. Pre-mRNA splicing alters mRNP composition: Evidence for stable association of proteins at exon-exon junctions. *Genes & Dev.* **14**: 1098–1108.

Le Hir, H., Gatfield, D., Braun, I.C., Forler, D., and Izaurralde, E. 2001a. The protein Mago provides a link between splicing and mRNA localization. *EMBO Rep.* **2**: 1119–1124.

Le Hir, H., Gatfield, D., Izaurralde, E., and Moore, M.J. 2001b. The exon-exon junction complex provides a binding platform for factors involved in mRNA export and nonsense-mediated mRNA decay. *EMBO J.* **20**: 4987–4997.

Lejeune, F. and Maquat, L.E. 2005. Mechanistic links between nonsense-mediated mRNA decay and pre-mRNA splicing in mammalian cells. *Curr. Opin. Cell Biol.* **17**: 309–315.

Lejeune, F., Ishigaki, Y., Li, X., and Maquat, L.E. 2002. The exon junction complex is detected on CBP80-bound but not eIF4E-bound mRNA in mammalian cells: Dynamics of mRNP remodeling. *EMBO J.* **21**: 3536–3545.

Li, Q., Imataka, H., Morino, S., Rogers Jr., G.W., Richter-Cook, N.J., Merrick, W.C., and Sonenberg, N. 1999. Eukaryotic translation initiation factor 4AIII (eIF4AIII) is functionally distinct from eIF4AI and eIF4AII. *Mol. Cell. Biol.* **19**: 7336–7346.

Li, C., Lin, R.I., Lai, M.C., Ouyang, P., and Tarn, W.Y. 2003. Nuclear Pnn/DRS protein binds to spliced mRNPs and participates in mRNA processing and export via interaction with RNPS1. *Mol. Cell. Biol.* **23**: 7363–7376.

Luo, M.J. and Reed, R. 1999. Splicing is required for rapid and efficient mRNA export in metazoans. *Proc. Natl. Acad. Sci.* **96**: 14937–14942.

Luo, M.L., Zhou, Z., Magni, K., Christoforides, C., Rappsilber, J., Mann, M., and Reed, R. 2001. Pre-mRNA splicing and mRNA export linked by direct interactions between UAP56 and Aly. *Nature* **413**: 644–647.

Lykke-Andersen, J., Shu, M.D., and Steitz, J.A. 2000. Human Upf proteins target an mRNA for nonsense-mediated decay when bound downstream of a termination codon. *Cell* **103**: 1121–1131.

———. 2001. Communication of the position of exon-exon junctions to the mRNA surveillance machinery by the protein RNPS1. *Science* **293**: 1836–1839.

Macchi, P., Kroening, S., Palacios, I.M., Baldassa, S., Grunewald, B., Ambrosino, C., Goetze, B., Lupas, A., St Johnston, D., and Kiebler, M. 2003. Barentsz, a new component of the Staufen-containing ribonucleoprotein particles in mammalian cells, interacts with Staufen in an RNA-dependent manner. *J. Neurosci.* **23**: 5778–5788.

Mohr, S.E., Dillon, S.T., and Boswell, R.E. 2001. The RNA-binding protein Tsunagi interacts with Mago Nashi to establish polarity and localize oskar mRNA during *Drosophila* oogenesis. *Genes & Dev.* **15**: 2886–2899.

Muhlemann, O., Mock-Casagrande, C.S., Wang, J., Li, S., Custodio, N., Carmo-Fonseca, M., Wilkinson, M.F., and Moore, M.J. 2001.

Precursor RNAs harboring nonsense codons accumulate near the site of transcription. *Mol. Cell* **8**: 33–43.

Nott, A., Meislin, S.H., and Moore, M.J. 2003. A quantitative analysis of intron effects on mammalian gene expression. *RNA* **9**: 607–617.

Nott, A., Le Hir, H., and Moore, M.J. 2004. Splicing enhances translation in mammalian cells: an additional function of the exon junction complex. *Genes & Dev.* **18**: 210–222.

Palacios, I.M. 2002. RNA processing: splicing and the cytoplasmic localisation of mRNA. *Curr. Biol.* **12**: R50–52.

Palacios, I.M., Gatfield, D., St Johnston, D., and Izaurralde, E. 2004. An eIF4AIII-containing complex required for mRNA localization and nonsense-mediated mRNA decay. *Nature* **427**: 753–757.

Peng, J., Elias, J.E., Thoreen, C.C., Licklider, L.J., and Gygi, S.P. 2003. Evaluation of multidimensional chromatography coupled with tandem mass spectrometry (LC/LC-MS/MS) for large-scale protein analysis: The yeast proteome. *J. Proteome Res.* **2**: 43–50.

Puig, O., Caspary, F., Rigaut, G., Rutz, B., Bouveret, E., Bragado-Nilsson, E., Wilm, M., and Seraphin, B. 2001. The tandem affinity purification (TAP) method: A general procedure of protein complex purification. *Methods* **24**: 218–229.

Reichert, V.L., Le Hir, H., Jurica, M.S., and Moore, M.J. 2002. 5′ Exon interactions within the human spliceosome establish a framework for exon junction complex structure and assembly. *Genes & Dev.* **16**: 2778–2791.

Rodrigues, J.P., Rode, M., Gatfield, D., Blencowe, B.J., Carmo-Fonseca, M., and Izaurralde, E. 2001. REF proteins mediate the export of spliced and unspliced mRNAs from the nucleus. *Proc. Natl. Acad. Sci.* **98**: 1030–1035.

Sahara, S., Aoto, M., Eguchi, Y., Imamoto, N., Yoneda, Y., and Tsujimoto, Y. 1999. Acinus is a caspase-3-activated protein required for apoptotic chromatin condensation. *Nature* **401**: 168–173.

Sakashita, E., Tatsumi, S., Werner, D., Endo, H., and Mayeda, A. 2004. Human RNPS1 and its associated factors: A versatile alternative pre-mRNA splicing regulator in vivo. *Mol. Cell. Biol.* **24**: 1174–1187.

Schwerk, C., Prasad, J., Degenhardt, K., Erdjument-Bromage, H., White, E., Tempst, P., Kidd, V.J., Manley, J.L., Lahti, J.M., and Reinberg, D. 2003. ASAP, a novel protein complex involved in RNA processing and apoptosis. *Mol. Cell. Biol.* **23**: 2981–2990.

Shi, H. and Xu, R.M. 2003. Crystal structure of the *Drosophila* Mago nashi-Y14 complex. *Genes & Dev.* **17**: 971–976.

Shibuya, T., Tange, T.O., Sonenberg, N., and Moore, M.J. 2004. eIF4A-III binds spliced mRNA in the exon junction complex and is essential for nonsense-mediated decay. *Nat. Struct. Mol. Biol.* **11**: 346–351.

Stutz, F. and Izaurralde, E. 2003. The interplay of nuclear mRNP assembly, mRNA surveillance and export. *Trends Cell Biol.* **13**: 319–327.

Tange, T.O., Nott, A., and Moore, M.J. 2004. The ever-increasing complexities of the exon junction complex. *Curr. Opin. Cell Biol.* **16**: 279–284.

Tanner, N.K. and Linder, P. 2001. DExD/H box RNA helicases: From generic motors to specific dissociation functions. *Mol. Cell* **8**: 251–262.

van Eeden, F.J., Palacios, I.M., Petronczki, M., Weston, M.J., and St Johnston, D. 2001. Barentsz is essential for the posterior localization of oskar mRNA and colocalizes with it to the posterior pole. *J. Cell Biol.* **154**: 511–523.

Wang, C., Chua, K., Seghezzi, W., Lees, E., Gozani, O., and Reed, R. 1998. Phosphorylation of spliceosomal protein SAP 155 coupled with splicing catalysis. *Genes & Dev.* **12**: 1409–1414.

Wiegand, H.L., Lu, S., and Cullen, B.R. 2003. Exon junction complexes mediate the enhancing effect of splicing on mRNA expression. *Proc. Natl. Acad. Sci.* **100**: 11327–11332.

Wilkinson, M.F. 2005. A new function for nonsense-mediated mRNA-decay factors. *Trends Genet.* **21**: 143–148.

Zhang, Y., Iratni, R., Erdjument-Bromage, H., Tempst, P., and Reinberg, D. 1997. Histone deacetylases and SAP18, a novel polypeptide, are components of a human Sin3 complex. *Cell* **89**: 357–364.

Zhou, Z., Luo, M.J., Straesser, K., Katahira, J., Hurt, E., and Reed, R. 2000. The protein Aly links pre-messenger-RNA splicing to nuclear export in metazoans. *Nature* **407**: 401–405.

Downloaded from rnajournal.cshlp.org on April 22, 2015 - Published by Cold Spring Harbor Laboratory Press

![RNA Society Logo](https://example.com/rna-society-logo.png) RNA  
A PUBLICATION OF THE RNA SOCIETY

**Biochemical analysis of the EJC reveals two new factors and a stable tetrameric protein core**

THOMAS Ø. TANGE, TOSHIHARU SHIBUYA, MELISSA S. JURICA, et al.

RNA 2005 11: 1869-1883

---

### References
This article cites 58 articles, 31 of which can be accessed free at:  
[http://rnajournal.cshlp.org/content/11/12/1869.full.html#ref-list-1](http://rnajournal.cshlp.org/content/11/12/1869.full.html#ref-list-1)

### Email Alerting Service
Receive free email alerts when new articles cite this article - sign up in the box at the top right corner of the article or [click here](click here).

---

**Unique RNA sequencing in serum and plasma**  
EXIQON

---

To subscribe to RNA go to:  
[http://rnajournal.cshlp.org/subscriptions](http://rnajournal.cshlp.org/subscriptions)

---

Copyright 2005 by RNA Society
