
Mol Gen Genomics (2006) 275: 125–135  
DOI 10.1007/s00438-005-0070-z  

**ORIGINAL PAPER**

Naho Yoneyama · Hanayo Morimoto · Chuang-Xing Ye  
Hiroshi Ashihara · Kouichi Mizuno · Misako Kato  

**Substrate specificity of *N*-methyltransferase involved in purine alkaloids synthesis is dependent upon one amino acid residue of the enzyme**  

Received: 27 May 2005 / Accepted: 24 October 2005 / Published online: 7 December 2005  
© Springer-Verlag 2005  

**Abstract** Caffeine (1,3,7-trimethylxanthine) and theobromine (3,7-dimethylxanthine) are the major purine alkaloids in plants. To investigate the diversity of *N*-methyltransferases involved in purine alkaloid biosynthesis, we isolated the genes homologous for caffeine synthase from theobromine-accumulating plants. The predicted amino acid sequences of *N*-methyltransferases in theobromine-accumulating species in *Camellia* were more than 80% identical to caffeine synthase in *C. sinensis*. However, there was a little homology among the *N*-methyltransferases between *Camellia* and *Theobroma*. The recombinant enzymes derived from theobromine-accumulating plants had only 3-*N*-methyltransferase activity. The accumulation of purine alkaloids was, therefore, dependent on the substrate specificity of *N*-methyltransferase determined by one amino acid residue in the central part of the protein.

**Keywords** Caffeine · Caffeine synthase · Camellia, Theobromine · Theobromine synthase · Theobroma cacao  

---

**Communicated by R. Hagemann**

N. Yoneyama · H. Morimoto · M. Kato (✉)  
Graduate School of Humanities and Sciences,  
Ochanomizu University, Otsuka, Bunkyo-ku,  
112-8610 Tokyo, Japan  
E-mail: mkato@cc.ocha.ac.jp  
Fax: +81-1-359785351  

C.-X. Ye  
School of Life Sciences, Sun Yat-Sen University,  
510275 Guangzhou, China  

H. Ashihara  
Department of Biology, Faculty of Science,  
Ochanomizu University, Bunkyo-ku, 112-8610 Tokyo, Japan  

K. Mizuno  
Faculty of Bioresource of Science, Akita Prefectural University,  
010-0195 Akita City, Akita, Japan  

---

**Introduction**

Purine alkaloids, caffeine(1,3,7-trimethylxanthine) and theobromine(3,7-dimethylxanthine), have been reported in more than 80 plant species (Ashihara and Crozier 1999). Most of the purine alkaloid-containing plants, e.g., tea (*Camellia sinensis*) and coffee (*Coffea arabica*), accumulate caffeine. In contrast to tea and coffee plants, some purine alkaloid-producing species, e.g., *Camellia irrawadiensis* (Nagata and Sakai 1985; Ashihara and Kubota 1987), *Camellia ptilophylla* (Ashihara et al. 1998) *Theobroma cacao* (Hammerstone et al. 1994; Naik 2001; Koyama et al. 2003) accumulate theobromine rather than caffeine.

Extensive metabolic studies with purine alkaloids have elucidated the caffeine biosynthetic pathway in some detail in leaves of tea and coffee. Transient incorporation of label into theobromine was observed prior to an exponential increase in the rate of \[^{14}C\] caffeine accumulation by pulse-chase experiments with \[8-\(^{14}\) C\]adenine and \[8-\(^{14}\) C\] guanosine. The available data support the operation of a xanthosine → 7-methylxanthosine → 7-methylxanthine → theobromine → caffeine pathway as the major route to caffeine. On the other hand, the lack of detectable incorporation of label into paraxanthine in the pulse-chase experiments suggested caffeine biosynthetic pathway via paraxanthine was minor (Suzuki 1972; Ashihara et al. 1997; Kato et al. 1996). Therefore, 7-methylxanthine → paraxanthine(1,7-dimethylxanthine) → caffeine, which is one of a number of minor pathways operating in tea leaves (Kato et al. 1999)(Fig. 1). Three methylation steps occur in caffeine biosynthesis. S-adenosyl-L-methionine (SAM), which is the methyl donor of numerous biological reaction, is the purine base methyl donor in caffeine biosynthesis (Suzuki 1972). SAM-dependent *N*-methyltransferases play an important role in the regulation of caffeine biosynthesis (Fujimori et al. 1991). Previously, we purified caffeine synthase, which was an *N*-methyltransferase catalyzing

SAM
SAH

Xanthosine

Ribose

7-Methylxanthosine synthase

7-Methylxanthosine

Ribose

7-Methylxanthine

Theobromine synthase
Caffeine synthase

Camellia irrawadiensis
Camellia ptilophylla
Theobroma cacao

Theobromine
(3,7-Dimethylxanthine)

Paraxanthine
(1,7-Dimethylxanthine)

SAM
SAH

SAM
SAH

Camellia sinensis
Coffea arabica

Caffeine
(1,3,7-Trimethylxanthine)

Fig. 1 Pathways for the biosynthesis of theobromine and caffeine. Solid arrows indicate major biosynthesis route. 7-Methylxanthosine synthase and theobromine synthase have closely substrate specificity and only catalyze the conversion of xanthosine to 7-methylxanthosine and of 7-methylxanthine to theobromine, respectively.

Caffeine synthase has broad substrate specificity. Plant species in the square are located at the left side of their major accumulated compounds. Abbreviations: SAM S-adenosyl-L-methionine; SAH S-adenosyl-L-homocysteine

the second and third methylation steps (Kato et al. 1999), and cloned a gene encoding caffeine synthase from young tea leaves (Kato et al. 2000). Furthermore, many genes orthologous to caffeine synthase from coffee have been reported (Mizuno et al. 2001; Mizuno et al. 2003a; 2003b).

The isolated coffee clones that belonged to caffeine synthase family are classified into three groups with respect to the substrate specificity of the enzyme protein; that is, 7-methylxanthosine synthase (CmXRS1), theobromine synthase (CTS1, CTS2) and caffeine synthase (CCS1). 7-Methylxanthosine synthase converts only xanthosine to 7-methylxanthosine. The cloning of 7-methylxanthosine synthase was reported to be in coffee and this enzyme was specific for xanthosine and XMP or xanthine could not be used as a substrtate (Mizuno et al. 2003b; Uefuji et al. 2003). 7-Methylxanthosine formed from 7-N-methylation of xantho-

sine by 7-methylxanthosine synthase is hydrolysed by N-methyl nucleosidase to 7-methylxanthine (Negishi et al. 1988). Theobromine synthases only catalyze 3-N-methylation of 7-methylxanthine to form theobromine and did not have 1-N-methylation activity. 7-Methylxanthine was the specific methyl acceptor, while 3-N-methylation activity of paraxanthine was very low (Mizuno et al. 2001; Ogawa et al. 2001). Caffeine synthase catalyzes 1-N and 3-N methylations of mono/di methylxanthines. When dimethylxanthines were used as substrates for caffeine synthase, paraxanthine was the best methyl acceptor, followed by theobromine. Of the three monomethylxanthines investigated, 7-methylxanthine was methylated most readily. The order of the purine base with caffeine synthase from tea and coffee was N-3 > N-1 >> N-7. Paraxanthine is the best methyl acceptor for caffeine synthase, but not for theobromine synthase (Kato et al. 2000; Mizuno et al.

Table 1 Oligonucleotide primers for PCR

| Name       | Sequence                                                                 | Method                              | Source      |
|------------|-------------------------------------------------------------------------|-------------------------------------|-------------|
| TCS-F1     | 5'-GGACTTGGGGTTGTGTCAGC-3'                                              | 3'-RACE                             | Camellia    |
| TCS-R1     | 5'-CAGCAATGGGCCATAGCTAATAG-3'                                           | cDNA synthesis for 5'-RACE          | Camellia    |
| TCS-R2     | 5'-GGAGGGGCTTGTCTTTGATATG-3'                                            | cDNA synthesis for 5'-RACE          | Camellia    |
| TCS-F2     | 5'-CCGGGGGTCTTTCCATGGCCG-3'                                             | 5'-RACE                             | Camellia    |
| TCS-R3     | 5'-GCCTTTGAAGAGGGGTAT-3'                                                | 5'-RACE                             | Camellia    |
| TCS-F3     | 5'-GGCTTACTCAGGCACCAAAGG-3'                                             | 5'-RACE                             | Camellia    |
| TCS-R4     | 5'-CCCTGCATTTCTTTCCATC-3'                                               | 5'-RACE                             | Camellia    |
| TCS-SAM1   | 5'-GAYTTRGGITGYKCIKCIGGICCIAAYAC-3'                                     | 3'-RACE                             | Theobroma   |
| BTS-RT     | 5'-ACCCTCCTTGTTCACTAA-3'                                                | cDNA synthesis for 5'-RACE          | Theobroma   |
| BTS-F1     | 5'-GCAGTGTTGGAAGGCATACC-3'                                              | 5'-RACE                             | Theobroma   |
| BTS-R1     | 5'-ACCTTAGAGAGGCCATTGTA-3'                                              | 5'-RACE                             | Theobroma   |
| BTS-F2     | 5'-GGACTGATTGATGAAGAGAA-3'                                              | 5'-RACE                             | Theobroma   |
| BTS-R2     | 5'-AAGAGGAATGGATGAGATGC-3'                                              | 5'-RACE                             | Theobroma   |
| TCS2-NF1   | 5'-GCCATGGGCAAGGGAGAAG-3'                                               | Construction of expression plasmid | TCS2        |
| TCS2-R1    | 5'-CGGCCATGGAAAAGACCCCGG-3'                                             | Construction of expression plasmid | TCS2        |
| ICSN-1     | 5'-ACCATGGGGAAGGTGAAC-3'                                                | Construction of expression plasmid | ICS1        |
| ICS-2R     | 5'-TTGGACCCGCTGCACAAC-3'                                                | Construction of expression plasmid | ICS1, ICS2   |
| ICS2-N1    | 5'-GCCATGGAGGTGAAAGAAG-3'                                               | Construction of expression plasmid | ICS2        |
| PCS1-N1    | 5'-GCCATGGGGAAGGTGAAC-3'                                                | Construction of expression plasmid and site-directed mutagenesis | PCS1        |
| PCS-R      | 5'-CGTTTTGGACCCGCTGCAC-3'                                               | Construction of expression plasmid | PCS1, PCS2  |
| PCS2-N1    | 5'-GCCATGGGAGGGTGAAAG-3'                                                | Construction of expression plasmid | PCS2        |
| BTS-N1     | 5'-GCCATGGAGGTGAAGAAA-3'                                                | Construction of expression plasmid | BTS1        |
| BTS-CR     | 5'-AAGAGGAATGGATGAGATGC-3'                                              | Construction of expression plasmid | BTS1        |
| BTS-F3     | 5'-GGTTATCTGTGCATTCAG-3'                                                | Construction of expression plasmid | BTS1        |
| BTS-R3     | 5'-GGAATTCCTAGAGGTACCG-3'                                               | Construction of expression plasmid | BTS1        |
| PCS1-HRR   | 5'-GCCTACCACGAAGTA-3'                                                   | Site-directed mutagenesis           | PCS1/ICS1   |
| PCS1-HRF   | 5'-TACTTCGTGGTAGGC-3'                                                   | Site-directed mutagenesis           | PCS1/ICS1   |
| PCS1-SCR   | 5'-TCAGAACATTGCCCT-3'                                                   | Site-directed mutagenesis           | PCS1/ICS1   |
| PCS1-SCF   | 5'-AGGCAATGTTCTGA-3'                                                    | Site-directed mutagenesis           | PCS1/ICS1   |
| PCS1-EQR   | 5'-CAGCTCTGCACTCT-3'                                                    | Site-directed mutagenesis           | PCS1/ICS1   |
| PCS1-EQF   | 5'-AGAGATGCAGAGCTG-3'                                                   | Site-directed mutagenesis           | PCS1/ICS1   |
| T7-R       | 5'-CTAGTTATTGCTCAGCGG-3'                                                | Site-directed mutagenesis           | Vector site |

2003a). In other words, theobromine synthase is almost specific to 7-methylxanthine.

If three types of N-methyltransferases are present in theobromine-accumulating species as well as the coffee plant, the end product is caffeine, not theobromine. Caffeine synthase in these species may possess 3-N but not 1-N-methylation activity. This possibility is supported by only an activity of a desalted extract from *C. ptilophylla* to methylate 7-methylxanthine but not the dimethylxanthines (Ashihara et al. 1998). Little is known about the N-methyltransferase in theobromine-accumulating plants because it is too difficult to isolate extremely labile enzymes in theobromine-accumulating species.

This is the first report about the cloning of N-methyltransferase in theobromine-accumulating plants. The predicted amino acid sequences of N-methyltransferase in theobromine-accumulating species in *Camellia* are more than 80% identical to caffeine synthase in *C. sinensis*. However, the N-methyltransferases in *Camellia* show low homology to those in *Theobroma*. The substrate specificity of the recombinant enzyme with no tags suggests that the accumulation of theobromine depends on the nature of N-methyltransferases. In addition, the recognition site for the purine derivatives in N-methyltransferase is discussed.

Materials and methods

Screening of cDNA library derived from *Camellia sinensis*

The cDNA library derived from young tea leaves (Takeuchi et al. 1994) was screened by the plaque hybridization method (Benton and Davis 1977). The probes used were the cDNA fragment of TCS1 with MseI digestion (350bp) and MseI/StyI digestion (418bp). The labelled probe was prepared with [α-32P]dCTP (110TBq/mmol, Amersham Biosciences) and BcaBEST™ Labelling Kit (TaKaRa). Plaque lifts were prehybridized at 55°C in 5x SSPE (1x SSPE=10mM sodium phosphate, pH 7.7, 0.18M NaCl and 1mM EDTA), 0.02% Ficoll 400, 0.02% polyvinylpyrrolidone, 0.02% BSA and 0.1% SDS, and then hybridized at 55°C overnight in the prehybridization buffer containing denatured salmon testis DNA (0.2mg/ml) and probes (each 2×10^6 cpm/ml). The filters were washed twice in 2x SSC (1x SSC=15mM sodium citrate, pH 7.0, and 0.15M NaCl) at room temperature for 10min, twice in 2x SSC containing 0.1% SDS at 55°C for 15min and 2x SSC at room temperature for 10min, before autoradiography at -80°C. Positive

clones were plaque-purified, and the cDNA inserts were subcloned into the EcoRI site of pUC19 for further characterization.

Amplification of the coding sequences of caffeine synthase homologous cDNAs using rapid amplification of cDNA ends (3'-RACE and 5'-RACE)

Total cellular RNAs from *C. irrawadiensis*, *C. ptilo-phylla* and *Theobroma cacao* were extracted using a cetyltrimethylammonium bromide (CTAB) solution according to the method of Chang et al. (1993) with the exception of the use of 5% 2-mercaptoethanol. First-strand cDNA was synthesized using a 3'-RACE core set and oligo-dT 3 sites adaptor primer (oligo-dT 3SAP) (TaKaRa).

To obtain the 3' regions of cDNAs from *Camellia*, we used TCSF-1 as a primer in 3'-RACE. The polymerase chain reaction (PCR) was conducted in a thermal cycler (GeneAmp PCR System 2400, Perkin Elmer) for 30 cycles (60s at 94°C, 30s at 59°C, and 120s at 72°C) with the first-strand cDNAs and primers, TCSF-1 and TCSF-3 sites adaptor primer (3SAP) included in the 3'-RACE core set sequence. The nucleotide sequences of 5' regions of ICS and PCS clones were obtained by 5'-RACE with the 5'full RACE set (TaKaRa). TCS-R1 or TCS-R2 was used for reverse transcription. The primer sets for 5'-RACE in the first PCR were TCS-F2 and TCS-R3, and those in the second PCR were TCS-F3 and TCS-R4. Both amplifications were conducted for 30 cycles (30s at 94°C, 30s at 60°C, and 60s at 72°C) with a thermal cycler.

To obtain the 3' region of cDNA from *Theobroma cacao*, we used TCS-SAM1 as a primer in 3'-RACE. PCR was conducted in the thermal cycler for 30 cycles (60s at 94°C, 60s at 57°C, and 90s at 72°C) with the first-strand cDNAs and primers, TCS-SAM1 and 3 sites adaptor primer (3SAP) included in the 3'-RACE core set sequence. The nucleotide sequences of 5' regions of BTS1 clone were obtained by 5'-RACE with the 5'full RACE set (TaKaRa). BTS-RT was used for reverse transcription. The DNA fragments were amplified in the above thermal cycler for 30 cycles (30s at 94°C, 30s at 55°C, and 60s at 72°C), using BTS-F1 and BTS-R1 as the primer sets for 5'-RACE in the first PCR; and for 30s at 94°C, 30s at 58°C, and 60s at 72°C, using BTS-F2 and BTS-R2 as the primer set in the second PCR.

The amplified DNA fragments were purified by polyacrylamide gel electrophoresis (PAGE) and were subcloned into pT7 Blue® Vector (Novagen).

Construction of expression plasmids

Expression plasmids for TCS1 homologous genes were constructed in a pET23d vector (Novagen). Only TCS2 was constructed in pET23d and pET32a vector (Nov-

agen). To prepare expression plasmids for TCS1 homologous genes, we carried out PCR-directed mutagenesis to produce an NcoI site at the translation initiation sites in the cDNA sequence of clones. The pairs of primers were TCS2-NF1/TCS2-R1 for TCS2, ICSN-1/ICS-2R for ICS1, ICS2-N1/ICS-2R for ICS2, PCS1-N1/PCS-R for PCS1, PCS2-N1/PCS-R for PCS2, BTS-N1/BTS-CR for BTS1. The reaction programs were 30 cycles of 95°C for 60s, 64°C for 60s, and 72°C for 60s (for TCS2), 30 cycles of 95°C for 60s, 60°C for 60s, and 72°C for 60s (for ICS1, ICS2, PCS1, PCS2), 94°C for 60s, 53°C for 60s, and 72°C for 120s (for BTS1). The PCR products were subcloned into pT7 Blue® Vector (Novagen). The subclones of the PCR products for TCS2 were digested with NcoI and StuI. The subclones of the PCR products for ICS1, ICS2, PCS1 and PCS2 were digested with NcoI and AvaII. The subclones of the PCR products for BTS1 were digested with NcoI and EcoT22I. The NcoI/StuI fragment and StuI/EcoRI fragment derived from subcloned TCS2 in pUC19 were introduced into the pET23d or pET32a vector at the NcoI and EcoRI sites, respectively (named pET23d-TCS2 and pET23d-TCS2, respectively). The NcoI/AvaII fragment and AvaII/HindIII fragment derived from ICS1, ICS2, PCS1 and PCS2 were introduced into the pET23d vector at the NcoI and HindIII sites, respectively (named pET23d-ICS1, pET23d-ICS2, pET23d-PCS1, pET23d-ICS2). For BTS1, we carried out PCR-directed mutagenesis to produce an EcoRI site at the translation-termination sites in the cDNA sequence of clones. The primer pairs used were: BTS-F3/BTS-R3. The reaction programs were 30 cycles of 94°C for 60s, 53°C for 60s, and 72°C for 120s. The subclone of the PCR product was digested with EcoT22I and EcoRI. The NcoI/EcoT22I fragment and EcoT22I/EcoRI fragment derived from BTS1 were introduced into pET23d at the NcoI and EcoRI sites (named pET23d-BTS1).

Construction of chimeric expression plasmids

pET23d-TCS1 (Kato et al 2000) and pET23d-PCS1 were digested with NcoI. The NcoI/NcoI fragment at the 5' region of the inserted gene and the vector attached to the 3' region of the inserted gene were obtained. These fragments were purified by polyacrylamide gel electrophoresis (PAGE). The NcoI/NcoI fragment obtained from pET23d-PCS1 was ligated to pET23d-TCS1 from which was deleted the NcoI/NcoI fragment. The obtained expression plasmid was named chimera A. On the other hand, the NcoI/NcoI fragment from pET23d-TCS1 was ligated to pET23d-PCS1 from which was deleted NcoI/NcoI fragment. The obtained expression plasmid was named chimera B.

pET23d-TCS1 and pET23d-PCS1 were digested with StuI and MunI. These fragments were purified by PAGE. The StuI/MunI fragment (173bp) from pET23d-TCS1 was ligated to pET23d-PCS1 from which
was deleted *StuI/MunI* fragment. The obtained expression plasmid was named chimera C. The *StuI/MunI* fragment (173 bp) from pET23d-PCS1 was ligated to pET23d-TCS1 from which the *StuI/MunI* fragment was deleted. The obtained expression plasmid was named chimera D.

pET23d-TCS1 and pET23d-TCS2 were digested with *NcoI*. The *NcoI/NcoI* fragment at the 5′ region of the inserted gene and the vector attached to the 3′ region of the inserted gene were obtained. These fragments were purified by PAGE. The obtained *NcoI/NcoI* fragment from pET23d-TCS1 was ligated to pET23d-TCS2 from which the *NcoI/NcoI* fragment was deleted. This expression plasmid was named chimera E. The obtained *NcoI/NcoI* fragment from pET23d-TCS2 was ligated to pET23d-TCS1 from which the *NcoI/NcoI* fragment was deleted. This expression plasmid was named chimera F.

All plasmids were digested with *RsaI* to identify the direction of the inserted *NcoI/NcoI* fragment. The substitutions and the frame were confirmed by DNA sequencing of the obtained expression plasmids.

### Site-directed mutagenesis

Site-directed mutagenesis of specific amino acids was carried out using the PCR method (Mizuno et al. 2003a). The external primers for all PCS1 mutants were PCS1-N1 (5′ end) and T7-R (3′ end), and pET23d-PCS1 was used as a template. The mutagenic primers for each mutation were designed in complementary pairs as follows: PCS1-HRR/PCS1-HRF (*H*²²¹ to R, named H221R), PCS1-SCR/PCS1-SCF (*S*²²⁵ to C, named S225C), PCS1-EQR/PCS1-EQF (*E*²³² to Q, named E232Q). To obtain the 5′ region of PCS1-221, we used PCS1-N1 and PCS1-HRR as primers. To obtain the 3′ region of H221R, we used PCS1-HRF and T7-R as primers. Both PCRs were conducted in the thermal cycler for 30 cycles (60 s at 94°C, 60 s at 56°C, and 120 s at 72°C). To obtain the full length of site-directed mutant, we used PCS1-N1 and T7-R as primers and the 5′ fragment and 3′ fragment, which were purified by PAGE, as templates. PCR was conducted in the thermal cycler for 30 cycles (60 s at 95°C, 60 s at 55°C, and 120 s at 72°C). The amplified DNA fragment was purified by PAGE and was subcloned into pT7Blue® Vector (Novagen). The subclones of the PCR products were digested with *NcoI* and *EcoRI*. The *NcoI/NcoI* fragment and *NcoI/EcoRI* fragment were introduced into pET23d at the *NcoI* and *EcoRI* sites. The obtained plasmids were digested with *RsaI* to identify the direction of the inserted *NcoI/NcoI* fragment. The fidelity of the PCR reactions and the mutations were confirmed by DNA sequencing of the obtained expression plasmids. All manipulations for the construction of S225C and E232Q were carried out as described above except that we used different internal primers.

---

Production of recombinant enzymes in *Escherichia coli*

The resulting expression plasmids were introduced into the expression host *E. coli* BL21(DE3). A single colony of the transformants was cultured at 37°C overnight in 3 ml Luria broth containing 0.1 mg/ml ampicillin (LA) with constant shaking. A portion (200 μl) of the bacterial culture was added to 12 ml of fresh LA, and incubated at 37°C for 2 h with shaking. Production of the recombinant proteins was induced by adding 36 μl of 0.1 M isopropyl-β-D-thiogalactopyranoside (final concentration is 0.3 mM), and cells were kept at 30°C for 4 h. *E. coli* cells were harvested by centrifugation at 600 g for 5 min, and then washed with 20 mM Tris-HCl, pH 7.5 containing 0.1 M NaCl, 1 mM EDTA and 2 mM 2-mercaptoethanol (TES). This cell paste was suspended in an 800 μl TES, frozen at –80°C, sonicated to disrupt cells and then centrifuged at 13,000 g for 5 min at 4°C. The supernatant was assayed for N-methyltransferase. For the analysis of the kinetic parameter of the recombinant enzymes, the desalted extract, prepared by NAP™-10 column (Amersham Biosciences) was used as enzyme source.

#### Assay for N-methyltransferase

Enzymatic activity of N-methyltransferase was determined as described previously with a slight modification (Kato et al. 1999). The final concentration of the substrate (purine derivatives) was 1 mM when recombinant BTS1 extract was used as the enzyme.

#### Analytical procedure

Protein concentrations were measured by the method of Bradford (1976). Polyacrylamide gel electrophoresis in the presence of 0.1% sodium dodecyl sulphate (SDS-PAGE) was carried out by the method of Laemmli (1970). Western blot analysis was carried out as described previously (Mizuno et al. 2003a). Nucleotide sequencing was carried out by the dideoxy chain-termination method of Sanger et al. (1977) using DSQ-2000L (Shimadzu) and Thermo Sequenase fluorescent labeled primer cycle sequencing kit with 7-deaza-dGTP(Amersham Biosciences). Nucleotide and protein sequences were analyzed by computer using GENETYX software (GeneTyx Co., Tokyo, Japan).

---

### Results

cDNA cloning of genes homologous to *TCS1* from *Camellia* plants

To isolate the homologous genes to *TCS1*, we used two types of *TCS1* cDNA fragments to screen an approxi-
approximately \(6.8 \times 10^5\) recombinant phage \(\lambda\)gt11 from cDNA library from young leaves. Probe A, which was a 350 bp fragment after digestion with MseI, was the 5'-region of TCS1 cDNA. Probe B, which was a 420 bp after digestion with MseI and StyI, was the central region of TCS1 cDNA. Twenty-three positive plaques were isolated and were sequenced after the subcloning. There was no difference in the pattern of clones with the probe and we obtained two types of clones. One clone corresponded to TCS1 and the other was a novel one termed TCS2 (accession number AB031281). The full length of TCS2 cDNA consists of 1,405 bp and contains a single open reading frame of 1,095 bp with a first ATG initiation codon at position 95 coding for a protein 365 amino acids. The deduced protein has a molecular mass of 41 kDa and a calculated isoelectric point of pH 5.3.

Since we obtained only a few young leaves from *C. irrawadiensis* and *C. ptilophylla*, PCR cloning was carried out. Based on the conserved motif A of the binding site of the methyl donor SAM in plant methyltransferases (Joshi and Chiang 1998), a gene-specific PCR primer was used to amplify the 3' region of the TCS

homologous gene from *C. irrawadiensis* and *C. ptilophylla*. We carried out 3'-RACE using total RNA from leaves as a template. Two fragments from each species were obtained and the full-length cDNAs were isolated by the 5'-RACE method. The total length of ICS1 and ICS2 from *C. irrawadiensis* (accession numbers AB056108 and AB207816), PCS1 and PCS2 from *C. ptilophylla* (accession numbers AB207817 and AB207818), consist of 1,432, 1,413, 1,394, and 1,357 bp, respectively, and encode 363–365 amino acids. The deduced amino acid sequences from all genes homologous to TCS1 have motif A, motif B', motif C and YFFF-region (Fig. 2). The constituents of motif C and YFFF-region were identical in series I (TCS1, PCS1, ICS1) and series II (TCS2, PCS2, ICS2). On the other hand, the second alanine of motif A in series I was substituted for valine in TCS2 and PCS2. In motif B', phenylalanine in series I was substituted for proline in series II as same as SAMT and BAMT. As a whole, series I (TCS1, PCS1, ICS1) differs from series II (TCS2, PCS2, ICS2) in that a different 20-amino acid sequence was conserved in each series.

---

**Motif A**

| TCS1 | 1   | MELATAGKVNEVLFMNRGEGESSYAQNSSFTQQVASMAQPALENAVETLFS-RDFHL-OALNAAADLGCAAAGPNNTFAVISTIKRMMEKKCRELN--COTLEL | 97 |
|------|-----|-----------------------------------------------------------------------------------------------------------------|----|
| ICS1 | 1   | -------MGKVNEVLFMNRGEGEISYAQNSAFTQKVASMAMPALENAVETLFS-KDFHLLQALTAADLGCAAAGPNNTFAVISTIKRMMEKKCRELY--COTLEL | 93 |
| PCS1 | 1   | -------MGKVNEVLFMNRGEGEISYAQNSAFTQKVASMAMPALENAVETLFS-KDFHLLQALNAADLGCAAAGPNNTFAVISTIKRMMEKKCRELN--COTLEL | 93 |
| TCS2 | 1   | -------MKEVKAEALFMNKGEGESSYAQNSSFQTQVTSMTPVLENAVETLFS-KDFHLLQALNAVDLGCAAAGPTTFTVISTIKRMMEKKCRELN--COTLEL | 93 |
| ICS2 | 1   | -------MEVKAEALFMNKGEGESSYAQNSSFTEVTSMTPVLENAVETLFS-KDFHLLQALNAADLGCAAAGPTTFTVISTIKRMMEKKCRELN--COTLEL | 92 |
| PCS2 | 1   | -------MKEVKAEALFMNKGEGESSYAQNSSFTEVTSMTPVLENAVETLFS-KDFHLLQALNAVDLGCAAAGPTTFTVISTIKRMMEKKCRELN--COTLEL | 93 |
| BTS1 | 1   | -------MEVKEMLFMNKGDGENSYVKTSGYTQKVAAVTQPVVYRAAQSLFTGRNSCSYQLNVADLGCSSGPNNTLMTVRDIVQSIDKVGGQEKKNELERPTI | 93 |
| CmXRS1 | 1  | -------MELQEVLRMNGGEGDTSYAKNSAYNQLVLAKVPVLEQCVERELLRANLPNINKCIKVADLGCAASGPNTLLTVRDIVQSIDKVGGQEKKNELERPTI | 95 |
| CTS1 | 1   | -------MELQEVLHMNEGEGDTSYAKNASYN-LALAKVKPFLEQCIRELLRANLPNINKCIKVADLGCAASGPNTLLTVRDIVQSIDKVGGQEENELERPTI | 94 |
| CCS1 | 1   | -------MELQEVLHMNGGEGDTSYAKNSSYN-LFLIRVKPVLEQCIQELLRANLPNINKCFVGDLGCAASGPNTFSTVRDIVQSIDKVGGQEKKNELERPTI | 94 |

---

**Motif B'**

| TCS1 | 98  | QVYLNDLFGNDFNTLFKGLSSEVIGNKCEEV-------PCYVMGVPGSFHGRLFPNSLHLVHSSYSVHWLTQAPKGLTSREGL-ALNKGKIYISKTSPPVV | 191 |
|------|-----|---------------------------------------------------------------------------------------------------------|----|
| ICS1 | 94  | QVYLNDLFGNDFNTLFKGLSSEVVGNKCEEV-------SCYVMGVPGSFHGRLFPNSLHLVHSSYSVHWLTQAPKGLTSREGL-ALNKGKIYISKTSPPVV | 187 |
| PCS1 | 94  | QVYLNDLFGNDFNTLFKGLSSQVVGNKCEEV-------SCYVMGVPGSFHGRLFPNSLHLVHSSYSVHWLTQAPKGLTSREGL-ALNKGKIYISKTSPPVV | 187 |
| TCS2 | 94  | QVYLNDLFGNDFNTLFKGLPSKVVGNKCEEV-------SCYVVGVPGSFHGRLFPNSLHLVHSCYSVHWLTQAPKGLTSKEGL-ALNKGKIYISKTSPPVV | 187 |
| ICS2 | 93  | QVYLNDLPGNDFNTLFKGLSSKVVGNKCEEV-------PCYVVGVPGSFHGRLFPNSLHLVHSCYSVHWLTQAPKGLTSKEGL-ALNKGKIYISKTSPPVV | 186 |
| PCS2 | 94  | QVYLNDLPGNDFNTLFKGLSSKVVGNNCEEV-------SCYVVGVPGSFHGRLFPNSLHLVHSCYSVHWLTQAPKGLTSKEGL-ALNKGKIYISKTSPPVV | 187 |
| BTS1 | 94  | QVYLNDLVGDNFNTLFKGLS---VIQDKYKNVS-------CFAMGAPGSFYISRLFPEESMHFLHSCYCLOWLSQVPSGLVTELGIST-NKGSIYSSKAASRLPV | 185 |
| CmXRS1 | 96 | QIFLNDLFPNDFNSVFKLLPS--FYRKLEKENGRKIGSCLIGAMPGSFYGRLLFPEESMHFLHSCYSVHWLSQVPSGLVIELGIGA-NKGSIYSSKGCRPPV | 193 |
| CTS1 | 95  | QIFLNDLFQNDFNSVFKLLPS--FYRKLEKENGRKIGSCLISAMPGSFYGRLLFPEESMHFLHSCYSVHWLSQVPSGLVIELGIGA-NKGSIYSSKGCRPPV | 192 |
| CCS1 | 95  | QIFLNDLFQNDFNSVFKLLPS--FYRNLEKENGRKIGSCLIGAMPGSFYSRLFBEESMHFLHSCYLCHWLSQVPSGLVTELGISA-NKGCIYSSKASGPII | 192 |

---

**YFFF region**

| TCS1 | 192 | REAYLSQFHEDFTMFLNARSQEVVPNGCMVLIILRGRQCSDBSDMQSCFTWELLAMAIAELVSOGILIDEDKLDTFNIPSYFASLEEVDIVERDGSETIDHI | 292 |
|------|-----|---------------------------------------------------------------------------------------------|----|
| ICS1 | 188 | KEAYLSQFHEDFTMFLNARSQEVVPNGCMVLIILHGRQSSDPSEMESCFTWELLAIATAIELVSOGILIDEDKLDTFNVPSPWPSLEEVDIVERDGSETIDHL | 288 |
| PCS1 | 188 | KEAYLSQFHEDFTMFLNARSQEVVPNGCMVLIILHGRQSSDPSEMESCFTWELLAIATAIELVSOGILIDDKLDTFNVPSYWPSLEEVDIVERDGSETIDHL | 288 |
| TCS2 | 188 | REAYLSQFHEDFTMFLNARSQEVVPNGCMVLIILRGRQSSDPDMGSCFTWELLAVAIAELVSOGILIDEDKLDTFNVPSPFPSLEEVDIVERNGSETIDHM | 288 |
| ICS2 | 187 | REAYLSQFHEDFTMFLNARSQEVVPNGCMVLIILRGRLLSSDPDMESCFTWELLAAAAIAELVSOGILIDEDKLDTFNVPSPFPSLEEVDIVERNGSETIDHM | 287 |
| PCS2 | 188 | REAYLSQFHEDFTMFLNARSQEVVPNGCMVLIILRGRLLSSDPDMESCFTWELLAAAAIAELVSOGILIDEDKLDTFNVPSPFPSLEEVDIVERNGSETIDHM | 288 |
| BTS1 | 186 | WRAYLSQFQEDFLSLRCRSPSELVPDGRMVLIHGRKSADPTTRESCYTWEVLADAISYOVSQLIDEEKLNFSFNPYYIPSQEEVRDLVNKEGSF---LT | 283 |
| CmXRS1 | 194 | QKAYLDQFTKDTFTFLRIHSEELFSHGRMLL---TCICKGVELD---ARNAAIDLLEMAINDLVEGHLEEKELDSFLNPVYIPSAEEVKCIVEEEGSGFEILYL | 290 |
| CTS1 | 193 | QKAYLDQFTKDTFTFLRIHSKELFSRRGMLL---TCICKVDVFD---EPNPLDLLDMAINDLIVEGLLEEEKELDSFNPIPFTPSAAEEVKCIVEEEGSCETILYL | 289 |
| CCS1 | 193 | KKAYLDQFTKDTFTFLRIHSEEELISRGRMLL---TFICKEDEFD---HPNSMDLLEMSINDLVEGHLEEEKELDSFNVPIYAPSTEEVKRIVEEEGSGFEILYL | 289 |

---

**Motif C**

| TCS1 | 293 | EGFDLDSVE---MQENDK-WVRG-----------------EKFTKVRAFTEPII-SNQFGPEIMDKLYDKF-THIVVSDLEA-KLPK-TTSTIILVLSKIDG | 369 |
|------|-----|---------------------------------------------------------------------------------------------|----|
| ICS1 | 289 | EGFELDSLE---MQENDK-WVRG-----------------DKFAKMVARAFTEPII-SNOFGHEIMDKLYDKF-THILVSDLEA-ELPK-TTSIILVLSKIVG | 365 |
| PCS1 | 289 | EGFELDSLE---MQEDDK-WVRG-----------------DKFAKMVARAFTEPII-SNOFGQEIMDKLYDKF-THILVSDLEA-ELPK-TTSIILVLSKIVG | 365 |
| TCS2 | 289 | EGFELDSPE---MQENDK-WVRG-----------------EKFATVARAFTEPII-SNOFGHEIMDKLYEKF-THIVVSDFEA-KIPK-ITSIILVLSKIVG | 365 |
| ICS2 | 288 | EGFELDSPQ---MQENDK-WVRG-----------------EKFATVARAFTEPII-SNOFGHEIMDKLYEKF-THIVVSDLEA-KIPK-ITSIILVLSKIVG | 364 |
| PCS2 | 289 | EGFELDSPQ---MQENDK-WVRG-----------------EKFATVARASTEPII-SNOFGHEIMDKLYEKF-THIVISDLEA-KIPK-VTSIILVLSKIVG | 365 |
| BTS1 | 284 | E-F-VDTIE---VELEGIWTFP------------------ENGAKNLRSFTEPMI-SHQFGEEVMDKLYDKV-KDILVEDCKOEKQSTRGVSIVLELKKEESHLS | 363 |
| CmXRS1 | 1291 | ETFKVLYDAGFSIDDEH--------------------IKAEEYVASSVRAVEPILASH-FGEAIIIPDIFHRFAKHA-AKVLPLGKGFYNNLIISL-AKKPEKSDV | 372 |
| CTS1 | 290 | ETFKAHYDAAFSIDDDY-PVRSH---------------EQIKAEEYVASLIRSVEPILASH-FGEAIMPDLFHRALKHA-AKVLHMGKGCYNNLIISL-AKKPEKSDV | 378 |
| CCS1 | 290 | ETFYAPYDAGFSIDDDYQGRSHSPVSCDEHARAADVSVVRSIYEPIILASH-FGEAILPDLSHRIAkNA-AKVLRSKGKFYDSVIISL-AKKPEKADM | 384 |

---

Fig. 2 Comparison of the amino acid sequences of caffeine synthases and its related enzymes. Alignment of the amino acid sequences for TCS1 and TCS2 from tea, ICS1 and ICS2 from *C. irrawadiensis*, PCS1 and PCS2 from *C. ptilophylla*, BTS1 from cocoa, and CmXRS1, CTS1 and CCS1 from coffee are indicated. Shaded boxes represent conserved amino acid residues, and dashes represent gaps that have been inserted for optimal alignment. The proposed SAM-binding motifs (A, B' and C) and the conserved region of nominated as “YFFF-region” are shown by open boxes

(Mizuno et al. 2003a). Asterisks indicate Tyrosine (Y) or Phenylalanine (F) residues in the region. The nominated amino acids in substrate binding are indicated by closed circles, and additional active site residue is indicated by arrow head (Zubieta et al. 2003). Sources of the sequences are as follows: TCS1, AB031280 (Kato et al. 2000); TCS2, AB031281; BTS1, AB096699; PCS1, AB207817; PCS2, AB207818; ICS1, AB056108; ICS2, AB207816; CmXRS1, AB034699 (Mizuno et al. 2003b); CTS1, AB034700 (Mizuno et al. 2001); CCS1, AB086414 (Mizuno et al. 2003a)
Cloning of cDNA homologous to TCS from Theobroma cacao

The amino acid sequence of N-methyltransferase genes from Camellia showed only 40% of sequence identity with those from C. arabica. The total sequence homology of N-methyltransferase between Camellia and Theobroma was suggested to be low and the SAM binding motifs to be conserved. We amplified the cDNA homologous to the TCS1 gene using total RNA from young leaves by the 3'-RACE method with the degenerate primer, which corresponded to motif A region of N-methyltransferases. The obtained fragment had the motifs A, B', C and YFFF-region, suggesting that this cDNA fragment was a TCS1-homologous gene. The full length of cDNA was isolated by the 5'-RACE method and named BTS1. The full-length BTS1 cDNA consists of 1,264 bp and contains a single open reading frame of 1,092 bp with a first ATG initiation codon at position 48 coding for a protein of 364 amino acids. BTS1 has an overall 55% homology to orthologous genes derived from Camellia plants and only 40% homology to orthologous genes from C. arabica.

Properties of recombinant enzyme of theobromine-accumulating plants

The cDNAs were cloned and expressed in E. coli, to elucidate whether the proteins encoded by the cloned cDNAs are related to the caffeine biosynthetic pathway. Mizuno et al. (2003b) indicated the difference in substrate specificity between the native protein and the fused protein. Therefore, expression plasmids were constructed in the pET23d vector, which allows the expression of the native form. Recombinant proteins from the expression vector were produced in E. coli BL21(DE3). When the crude extracts of these E. coli were incubated with a variety of xanthine substrates of caffeine in the presence of [methyl-¹⁴C]SAM as a methyl donor, ICS1, PCS1 and BTS1 had N-methyltransferase activity. We could not detect N-methyltransferase activity in series II recombinant proteins. In particular, recombinant TCS2 protein which was fused to thioredoxin/His-tag with the construction of pET32a vector had no activity against xanthine derivatives.

Table 2 summarizes the substrate specificity of the recombinant enzymes. When various xanthine derivatives were used as substrates, 7-methylxanthine was the specific methyl acceptor, followed by paraxanthine, which was 10% active as 7-methylxanthine in ICS1 and PCS1. This suggests that ICS1 and PCS1 catalyzed only 3-N-methyltransferase activity. The obtained result in BTS1 was similar to those in ICS1 and PCS1 except that no BTS1 activity was detected with paraxanthine. In other words, these recombinant proteins were theobromine synthase. To compare enzyme-substrate affinity, kinetic parameters were determined. Lineweaver-Burk plots gave a Km value of 51 μM (ICS1), 85 μM (PCS1) and 2.4 mM (BTS1) for 7-methylxanthine in the presence of 50 μM SAM. The pH-dependent activity curve for the methylation of 7-methylxanthine showed a broad maximum at pH 8.0–8.5 for ICS1 and PCS1, pH 9.0–9.5 for BTS1.

Changing substrate preference of theobromine synthase to that of caffeine synthase by the hybrid protein and site-directed mutagenesis

The protein sequence of theobromine synthase (ICS1, PCS1) from Camellia plants was 90% identical to caffeine synthase(TCS1) from C. sinensis. We replaced the 139-amino acid fragment containing N-terminal of TCS1 with the corresponding region of PCS1 (Fig. 3). Substrate specificity of the hybrid enzymes reflected the region containing the C-terminal. Furthermore, the replacement in the central part of the enzymes suggested that the 173-amino acids of TCS1 played an important role in determining the substrate specificity (Table 3). There was only a conserved YFFF-region in this sequence. Alignment of the sequences in this 173-amino acid region, revealed a difference in amino acids only in nine positions between TCS1 and PCS1 and they were not clustered.

### Table 2 Comparison of the substrate specificity of recombinant N-methyltransferases

| Recombinant enzyme | Substrate/methylation position |
|--------------------|--------------------------------|
|                    | 7-mX/3N                       | 3-mX/1N                       | 1-mX-3N                      | Tb/1N                        | pX/3N                        | Tp/7N                        | XR/7N                        |
| TCS1               | 100                           | 1.0                           | 12.3                         | 18.5                         | 230.0                        | tr                           | ND                           |
| TCS2               | ND                            | ND                            | ND                           | ND                           | ND                           | ND                           | ND                           |
| ICS1               | 100                           | ND                            | ND                           | 0.7                          | 10.9                         | ND                           | ND                           |
| ICS2               | ND                            | ND                            | ND                           | ND                           | ND                           | ND                           | ND                           |
| PCS1               | 100                           | tr                            | ND                           | ND                           | 11.0                         | ND                           | ND                           |
| PCS2               | ND                            | ND                            | ND                           | ND                           | ND                           | ND                           | ND                           |
| BTS1               | 100                           | ND                            | ND                           | ND                           | ND                           | ND                           | ND                           |
| Methylated product | Tb                            | Tp                            | Cf                           | Cf                           | 7-mXR                        |                              |                              |

The relative activity is indicated as the percentage of the activity with 7-mX. Abbreviations as follows; ND not detected, Cf caffeine, mX methylxanthine, Tb theobromine, Tp theophylline, pX paraxanthine, XR xanthosine

* Taken from ref. [14]
Fig. 3 Construction of hybrid enzymes and site-directed mutagenesis

| Recombinant protein | Protein construction | Substrate specificity |
|---------------------|----------------------|-----------------------|
| TCS1                |                      | Caffeine synthase     |
| PCS1                |                      | Theobromine synthase  |
| TCS2                |                      | No activity           |
| Chimera A           |                      | Caffeine synthase     |
| Chimera B           |                      | Theobromine synthase  |
| Chimera C           |                      | Caffeine synthase     |
| Chimera D           |                      | Theobromine synthase  |
| Mutant 1            |                      | Theobromine/caffeine synthase |
| Mutant 2            |                      | Theobromine synthase  |
| Mutant 3            |                      | Theobromine synthase  |
| Chimera E           |                      | Caffeine synthase     |
| Chimera F           |                      | Caffeine synthase     |

To further determine whether or not these residues participate in determining substrate discrimination, we performed site-directed mutagenesis to change individual residues. Residues in PCS1 were changed in three mutants: H221R (mutant 1), S225C, (mutant 2), E232Q, (mutant 3) (Fig. 3). Table 3 summarizes the substrate specificity of the mutants. The activity against para-xanthine in mutant 1 increased five-fold in relation to PCS1, while, that in mutants 2 and 3 slightly decreased. However, the profile of the substrate specificity in mutant 1 was not consistent with the profile derived for caffeine synthase. The obtained results suggested that H²¹ played a critical role in substrate discrimination and more than one amino acid residue is involved in the recognition of the purine derivatives.

The recombinant proteins TCS2, ICS2 and PCS2 did not show N-methyltransferase activities using xanthine derivatives as substrates. No caffeine synthetic activity was detected in coffee recombinant proteins, CtCS3 and CtCS4, as in series II proteins (Mizuno et al. 2003a). To investigate the reason why the recombinant enzyme has no N-methyltransferase activity, we chose TCS1 (caffeine synthase) and TCS2 (no activity) which share 89% sequence identity. We replaced the 139-amino acid fragment containing N-terminal of TCS1 with the corresponding region of TCS2. Both enzyme preparations from chimera E and chimera F had caffeine synthase activity (Fig. 3), suggesting that the primary structure of TCS2 is sufficient to function as an N-methyltransferase.

Table 3 Comparison of the substrate specificity of the hybrid enzymes and mutants

| Recombinant enzyme   | Substrate/Methylation position |
|----------------------|-------------------------------|
|                     | 7-mX/3N                       | 3-mX/1N                       | 1-mX-3N                       | Tb/1N                        | pX/3N                        | Tp/7N                        |
| Chimera A           | 100                           | ND                            | 11.2                         | 14.3                         | 323.0                        | ND                           |
| Chimera B           | 100                           | ND                            | ND                           | ND                           | 4.8                          | ND                           |
| Chimera C           | 100                           | 2.1                           | 18.6                         | 3.1                          | 153.6                        | ND                           |
| Chimera D           | 100                           | ND                            | ND                           | ND                           | 7.3                          | ND                           |
| Mutant 1            | 100                           | ND                            | 3.4                          | ND                           | 50.8                         | ND                           |
| Mutant 2            | 100                           | –                             | –                            | ND                           | 5.2                          | –                            |
| Mutant 3            | 100                           | ND                            | 3.3                          | ND                           | 6.3                          | –                            |
| Chimera E           | 100                           | 20.3                          | 13.1                         | 3.4                          | 358.6                        | –                            |
| Chimera F           | 100                           | ND                            | ND                           | tr                           | 528.6                        | –                            |
| Methylated product  | Tb                            | Tp                            | Cf                           | Cf                           |                              |                              |

The relative activity is indicated as the percentage of the activity with 7-mX. Abbreviations as follows; ND not detected, –, not determined, Cf caffeine, mX methylxanthine, Tb theobromine, Tp theophylline, pX paraxanthine, XR xanthosine

Discussion

Comparison of the sequences of genes homologous to caffeine synthase

Two types of genes homologous to caffeine synthase were isolated from three *Camellia* plants. TCS1, previously reported as an *N*-methyltransferase involved in caffeine biosynthesis from *C. sinensis*, had 3- and 1-*N*-methyltransferase activities (Kato et al. 2000). Twenty amino acids in the amino acid sequence of series I were conserved in series II. Within the same species, the amino acid sequences of series I shared a high degree of sequence identity (87–89%) with those of series II. There were four highly conserved regions, motif A, motif B′, motif C and YFFF region, previously reported in coffee *N*-methyltransferases (Mizuno et al. 2003a). The three conserved motifs (A, B and C) of the binding site of the methyl donor SAM have been reported in most plant SAM-dependent *O*-methyltransferases (Joshi and Chiang 1998). Motif A and motif C are conserved regions in *O*-methyltransferases and motif B′ methyltransferase families. The motif B′ and YFFF region were specific to the motif B′ methyltransferase family which was a novel family including caffeine synthase (Kato and Mizuno 2004). Most members of this newly characterized motif B′ methyltransferase family catalyze the formation of small molecule methyl esters using SAM as a methyl donor and substrates with a carboxyl group as methyl acceptors. SAMT (SAM:salicylic acid carboxylmethyltransferase) (Ross et al. 1999; Fukami et al. 2002; Negre et al. 2002) BAMT (SAM:benzoic acid carboxyl methyltransferase) (Dudareva et al. 2000) and

JAMT (SAM:jasmonic acid carboxyl methyltransferase) (Seo et al. 2001) belong to this family.

BTS1 also belongs to the motif B′ methyltransferase family and share 55% homology to NMTs derived from *Camellia* plants and 40% homology with those from *C. arabica*. Figure 4 shows the phylogenetic tree analysis of motif B′ methyltransferase family and other plant *N*-methyltransferases. *N*-methyltransferases involved in caffeine biosynthesis are more closely related to motif B′ carboxylmethyltransferase than to plant *N*-methyltransferase involved in the biosynthesis of nicotine (Hibi et al. 1994) and coclaurine (Choi et al. 2002). However, it is likely that the biosynthetic pathway for purine alkaloids in tea, coffee and cacao evolved in parallel. In *Camellia*, members of the same enzyme group, in either series I or series II, were highly conserved beyond species. The homology between caffeine synthase and theobromine synthase in the same genus is very high; on the other hand, the homology between the orthologous genes is low. This indicates that caffeine biosynthesis evolved independently among plants.

Purine alkaloids biosynthesis in theobromine-accumulating plants

Genes homologous to caffeine synthase isolated from theobromine-accumulating plants encoded theobromine synthases. Theobromine synthase is thought to be the major type of caffeine synthase family which was expressed in young leaves because RT-PCR products were semi-quantitative corresponding to the amount of mRNA. The theobromine content was more than 0.5%, while the caffeine content was below 0.02% in *C. irra-*

Fig. 4 Evolutionary relationship of caffeine synthases and its related enzymes. The unrooted tree was created with ClustalW using the neighbour joining method (http://clustalw.genome.jp/). Substrates of the enzymes are indicated in parentheses. The substrates of TCS2, ICS2, PCS2 and CtCS3 are unknown. Abbreviations of substrates are as follows: *XR* xanthosine, *7mX* 7-methylxanthine, *Tb* theobromine, *SA* salicylic acid, *BA* benzoic acid, *JA* jasmonic acid. Sources of the sequences are as follows: CtCS3, AB054842 (Mizuno et al. 2003a); CbSAMT, AF133053 (Ross et al. 1999); AtJAMT, AY008434 (Seo et al. 2001); AmBAMT, AF198492 (Dudareva et al. 2000), and those of the other sequences are indicated in Fig. 3

![Evolutionary Tree](attachment:tree_image.png)
wadiensis (Nagata and Sakai 1985). In *C. ptilophylla*, theobromine was the major purine alkaloid and caffeine was not detected (Ashihara et al. 1998). Theobromine synthase activity was detected only in the crude extract from *C. ptilophylla* among the theobromine-accumulating plants (Ashihara et al. 1998). There was no doubt that the molecular species of purine alkaloids depended on the substrate specificity of *N*-methyltransferase.

We could not detect *N*-methyltransferase activity in the crude extract from *T. cacao* because a large amount of polysaccharides interfered with the isolation of the protein. The level of purine alkaloids in cacao leaves was much lower than in *Camellia* leaves. The major purine alkaloid in young cacao leaves was theobromine, but a small amount of caffeine was also present. Although no caffeine biosynthesis from \[^{14}C\]-labelled purine bases and nucleosides was observed during an 18-h incubation, exogenously supplied \[8-\[^{14}C\]\]theobromine was converted to caffeine in young leaves, suggesting that theobromine was converted to caffeine very slowly (Koyama et al. 2003). The *Km* value of BTS1 against theobromine was higher than that of other caffeine/theobromine synthases, and so one of the physiological properties may be caused by the character of theobromine synthase. However, judging from the significant concentration of caffeine in cacao leaves, another caffeine synthase gene which was expressed at an extremely low level might be present in cacao leaves.

### *N*-methyltransferase activity and enzyme structure

The hybrid enzyme analysis showed the central part of the enzyme determined the substrate specificity against xanthine derivatives. TCS1 had longer *N*-terminal sequence than other homologues and this specific sequence was unrelated to the substrate specificity. Zubieta et al. 2003 reported the 3.0 Å crystal structure of *Clarkia breweri* SAMT, which belonged to motif B' methyltransferase in complex with the substrate salicylic acid, and the demethylated product S-adenosyl-L-homocysteine revealed a protein structure that possessed a helical active site capping domain and a unique dimerization. The experimentally determined *Clarkia* SAMT structure nonetheless served as an effective template for modelling the active sites of the *N*-methyltransferases involved in caffeine biosynthesis. They pointed out the critical role of R^210^ of TCS1 on the substrate discrimination. In our experiment using site-directed mutagenesis, we have also recognized the same amino acid which appears to be responsible for the difference in the substrate discrimination. The final substrate specificity of caffeine synthase and theobromine synthase was supposed to be determined by more than two separated residues. This theory, namely that one more residue was responsible for the substrate discrimination, was consistent with the observation on *Clarkia* SAMT (Zubieta et al. 2003). In contrast, there is approximately 40% homology between the amino acid sequence of tea TCS1 and those of caffeine synthetic enzymes from coffee. Although the H221R-mutant of PCS1 that is tea TCS1 orthologue changes from theobromine synthase to caffeine synthase, there is no corresponding amino acid residue in coffee caffeine synthases (Fig. 2). It is likely that the substrate specificity is operated by a different structure between tea and coffee in this respect. On the other hand, chavicol *O*-methyltransferase and eugenol *O*-methyltransferase from sweet basil were 90% identical with each other at the primary sequence level and they were easily interconvertible via a single amino acid change (Gang et al. 2002). These enzymes do not belong to the motif B' methyltransferase family.

TCS1 and TCS2 are highly homologous with each other. TCS1 is a caffeine synthase, but the function of TCS2 has not been identified yet. Both types of hybrid enzymes had caffeine synthase activity, suggesting that TCS2 may function as a caffeine synthase *in vivo*. The unidentified factor, i.e. protein modification, the folding of protein and so on, thought to be underestimated the precise activity of the recombinant protein *in vitro*. Although we isolated seven kinds of coffee tentative caffeine synthases (CtCS series) from *C. arabica* and the enzyme activities of CtCS3 and CtCS4 could not be detected (Mizuno et al. 2003a). It is quite likely that they are *N*-methyltransferases involved in the methylation of other substances. However, series II proteins in *Camellia* species and CtCS3/CtCS4 in *C. arabica* shared more than 80% amino acid homology in their paralogous genes, respectively. If they are *N*-methyltransferases involved in the metabolism except caffeine biosynthesis, they are very interested in the molecular evolution of the enzymes and the branched metabolism. To further examine the correlation of the enzyme activity and the protein conformation, we are conducting experiments with coffee enzymes.

#### Acknowledgements

We deeply thank Dr. Y. Takeda (National Institute of Plant Vegetable and Tea Science) for the generous supply of *C. irrawadiensis*. We also would like to thank Dr. A. Takeuchi (National Institute of Plant Vegetable and Tea Science) for providing cDNA library of *C. sinensis* and Mr. Y. Tomoda (Meiji Seika Kaisya Co Ltd.) for the supply of cacao trees. This work is supported by a Grant-in-Aid for Scientific Research from the Japan Society for the Promotion of Science (No. 14540590 to MK., No. 15780076 to KM.).

---

### References

Ashihara H, Crozier A (1999) Biosynthesis and metabolism of caffeine and related purine alkaloids in plants. Adv Bot Res 30:117–205  
Ashihara H, Gillies FM, Crozier A (1997) Metabolism of caffeine and related purine alkaloids in leaves of tea (*Camellia sinensis* L.). Plant Cell Physiol 38:413–419  
Ashihara H, Kato M, Chuang-xing Y (1998) Biosynthesis and metabolism of purine alkaloids in leaves of cocoa tea (*Camellia ptilophylla*). J Plant Res 111:599–604  
Ashihara H, Kubota H (1987) Biosynthesis of purine alkaloids in *Camellia* plants. Plant Cell Physiol 28:535–539  
Ashihara H, Monterio AM, Gillies FM, Crozier A (1996) Biosynthesis of caffeine in leaves of coffee. Plant Physiol 111:747–753  
Benton WD, Davis RW (1977) Screening λgt recombinant clones by hybridization to single plaque *in situ*. Science 196:180–182

Bradford MM (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem 72:248–254

Chang S, Puryear J, Cairney J (1993) A simple and efficient method for isolation of RNA from pine trees. Plant Mol Biol Rep 11:113–116

Choi KB, Morishige T, Shitan N, Yazaki K, Sato F (2002) Molecular cloning and characterization of coclaurine N-methyltransferase from cultured cells of *Coptis japonica*. J Biol Chem 277:830–835

Dudareva N, Murfitt LM, Mann CJ, Gorenstein N, Kolosova N, Kish CM, Bonham C, Wood K (2000) Developmental regulation of methyl benzoate biosynthesis and emission in snapdragon flowers. Plant Cell 12:949–961

Fujimori N, Suzuki T, Ashihara H (1991) Seasonal variations in biosynthetic capacity for the synthesis of caffeine in tea leaves. Phytochemistry 30:2245–2248

Fukami H, Asakura T, Hirano H, Abe K, Shimomura K, Yamakawa T (2002) Salicylic acid carboxyl methyltransferase induced in hairy root cultures of *Atropa belladonna* after treatment with exogenously added salicylic acid. Plant Cell Physiol 43:1054–1058

Gang DR, Lavid N, Zubieta C, Chen F, Beuerle T, Lewinsohn E, Noel JP, Pichersky E (2002) Characterization of phenylpropene O-methyltransferases from sweet basil: Facile change of substrate specificity and convergent evolution within plant O-methyltransferase family. Plant Cell 14:505–519

Hammerstone JF Jr, Romanczyk LJ Jr, Aitkent WM (1994) Purine alkaloid distribution within *Herrania* and *Theobroma*. Phytochemistry 35:1237–1240

Hibi N, Higashiguchi S, Hashimoto T, Yamada Y (1994) Gene expression in tobacco low-nicotine mutants. Plant Cell 6:723–735

Joshi CP, Chiang VL (1998) Conserved sequence motifs in plant S-adenosyl-L-methionine-dependent methyltransferases. Plant Mol Biol 37:663–674

Kato M, Kanehara T, Shimizu H, Suzuki T, Gillies FM, Crozier A, Ashihara H (1996) Caffeine biosynthesis in young leaves of *Camellia sinensis*: in vitro studies on N-methyltransferase activity involved in the conversion of xanthosine to caffeine. Physiol Plant 98:629–636

Kato M, Mizuno K (2004) Caffeine synthase and related methyltransferases in plants. Front Biosci 9:1833–1842

Kato M, Mizuno K, Fujimura T, Iwama M, Irie M, Crozier A, Ashihara H (1999) Purification and characterization of caffeine synthase from tea leaves. Plant Physiol 120:579–586

Kato M, Mizuno K, Crozier A, Fujimura T, Ashihara H (2000) Caffeine synthase gene from tea leaves. Nature 406:956–957

Koyama Y, Tomoda Y, Kato M, Ashihara H (2003) Metabolism of purine bases, nucleosides and alkaloids in theobromine-forming *Theobroma cacao* leaves. Plant Physiol Biochem 41:977–984

Laemmli UK (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T-4. Nature 227:680–685

Mizuno K, Okuda A, Kato M, Yoneyama N, Tanaka H, Ashihara H, Fujimura T (2003a) Isolation of a new dual-functional caffeine synthase gene encoding an enzyme for the conversion of 7-methylxanthine to caffeine from coffee (*Coffea arabica* L.) FEBS Lett 534:75–81

Mizuno K, Kato M, Irino F, Yoneyama N, Fujimura T, Ashihara H (2003b) The first committed step reaction of caffeine biosynthesis: 7-methylxanthosine synthase is closely homologous to caffeine synthases in coffee (*Coffea arabica* L.) FEBS Lett 547:56–60

Mizuno K, Tanaka H, Kato M, Ashihara H, Fujimura T (2001) cDNA cloning of caffeine (theobromine) synthase from coffee (*Coffea arabica* L.,) In: International Scientific Colloquium on Coffee, ASIC, Paris, 19:815–818

Nagata T, Sakai S (1985) Purine base pattern of *Camellia irrawadiensis*. Phytochemistry 24:2271–2272

Naik JP (2001) Improved high-performance liquid chromatography method to determine theobromine and caffeine in cocoa and cocoa products. J Agric Food Chem 49:3579–3583

Negishi O, Ozawa T, Imagawa H (1985) Methylation of xanthosine by tea leaf extracts and caffeine biosynthesis. Agric Biol Chem 49:887–890

Negishi O, Ozawa T, Imagawa H (1988) N-methyl nucleosidase from tea leaves. Agric Biol Chem 52:169–175

Negre F, Kolosova N, Knoll J, Kish CM, Dudareva N (2002) Novel S-adenosyl-L-methionine:salicylic acid carboxyl methyltransferase, an enzyme responsible for biosynthesis of methyl salicylate and methyl benzoate, is not involved in floral scent production in snapdragon flowers. Arch Biochem Biophys 406:261–270

Ogawa M, Herai Y, Koizumi N, Kusano T, Sano H (2001) 7-Methylxanthine methyltransferase of coffee plants. J Biol Chem 276:8213–8218

Ross JR, Nam KH, D'Auria JC, Pichersky E (1999) S-Adenosyl-L-methionine:Salicylic acid carboxyl methyltransferase, an enzyme involved in floral scent production and plant defense, represents a new class of plant methyltransferases. Arch Biochem Biophys 367:9–16

Sanger F, Nicklen S, Coulson AR (1977) DNA sequencing with chain-terminating inhibitors. Proc Natl Acad Sci USA 74:5463–5467

Seo HS, Song JT, Cheong JJ, Lee YH, Iwang I, Lee JS, Choi YD (2001) Jasmonic acid carboxyl methyltransferase: a key enzyme for jasmonate-regulated plant responses. Proc Natl Acad Sci USA 98:4788–4793

Suzuki T (1972) The participation of S-adenosylmethionine in the biosynthesis of caffeine in the tea plants. FEBS Lett 24:18–20

Suzuki T, Ashihara H, Waller GR (1992) Purine and purine alkaloid metabolism in *Camellia* and *Coffea* plants. Phytochemistry 31:2575–2584

Takeuchi A, Matsumoto S, Hayatsu M (1994) Chalcone synthase from *Camellia sinensis*: isolation of the cDNAs and the organ-specific and sugar-responsive expression of the genes. Plant Cell Physiol 35:1011–1018

Uefuji H, Ogita S, Ymaguchi Y, Koizumi N, Sano H (2003) Molecular cloning and functional characterization of three distinct N-methyltransferase involved in the caffeine biosynthetic pathway in coffee plants. Plant Physiol 132:372–380

Zubieta C, Ross JR, Koscheski P, Yang Y, Pichersky E, Noel JP (2003) Structural basis for substrate recognition in the salicylic acid carboxyl methyltransferase family. Plant Cell 15:1704–1716
