
Zoology 105 (2002): 297–312  
© by Urban & Fischer Verlag  
http://www.urbanfischer.de/journals/zoology  

---

**REVIEW**

**The circadian system of *Drosophila melanogaster* and its light input pathways***  

Charlotte Helfrich-Förster*  

Institut für Zoologie, Universität Regensburg, Germany  

---

### Summary

The fruit fly *Drosophila melanogaster* has been a grateful object for circadian rhythm researchers over several decades. Behavioral, genetic, and molecular studies in the little fly have aided in understanding the bases of circadian time keeping and rhythmic behaviors not only in *Drosophila*, but also in other organisms, including mammals. This review summarizes our present knowledge about the fruit fly’s circadian system at the molecular and neurobiological level, with special emphasis on its entrainment by environmental light-dark cycles. The results obtained for *Drosophila* are discussed with respect to parallel findings in mammals.

#### Key words: insects, circadian clock, neurogenetics, rhythm generation, *Drosophila*

---

### Introduction

The cycling environmental conditions on earth force organisms to cope with large daily changes in lighting conditions, temperature, humidity and other physical parameters. Instead of merely reacting passively to the changes between day and night, most organisms have endogenous clocks, which allow them to prepare in advance for the changes in the environment. Furthermore, such clocks enable organisms to time specific (metabolic) activities to certain phases in the 24-hour cycle and to internally coordinate different physiological parameters. The adaptive advantage of endogenous circadian clocks is probably the reason underlying their broad distribution amongst the organisms living on our planet, and has been recently demonstrated experimentally for cyanobacteria (Ouyang et al., 1998) and fruit flies (Beaver et al., 2002). In the absence of any rhythmic environmental fluctuations, endogenous clocks drive rhythmic physiological processes and behaviors autonomously with an approximately 24-hour period. Therefore, they are also called circadian clocks. Nonetheless to function as timekeepers in the normal world they have to be synchronized by environmental Zeitgebers to a period of exactly 24 hours. One of the most important Zeitgebers in nature are the daily light-dark cycles, which are perceived by specialized photoreceptors which then convey this information to the clock.

The molecular mechanisms by which endogenous clocks operate and by which they are entrained to the external light-dark cycles remained largely a mystery for a long period of time and early chronobiologists had to work mainly at the descriptive level. However, during the past 20 years the myth once connected with circadian timekeeping has been uncovered at an ever increasing pace by genetic and molecular studies, a process that some researchers dubbed the “Clockwork Explosion” (Reppert, 1998). Systematic genetic screens yielded rhythm mutants and finally led to the molecular identification of key components of the circadian clock.

---

*Corresponding author: Charlotte Helfrich-Förster, Universität Regensburg, Institut für Zoologie, Universitätsstraße 31, 93053 Regensburg, Germany; phone: +49-941-943-3117; fax: +49-941-943-3325; e-mail: charlotte.foerster@biologie.uni-regensburg.de  

**Presented at the 95th Annual Meeting of the Deutsche Zoologische Gesellschaft in Halle/S., May 20–24, 2002

Ch. Helfrich-Förster

cation of several clock components in a few model organisms: the blue green alga Synechococcus, the bread mold Neurospora crassa, the higher plant Arabidopsis thaliana, the fruit fly Drosophila melanogaster and the mouse Mus musculus (for review: Dunlap, 1999; Edery, 2000; Young and Kay, 2001). Although the actual clock components may differ substantially between phylogenetic groups, all these clock components participate in a common fundamental mechanism of negative autoregulation, and in the case of mice and flies, even the molecules involved are remarkably conserved (for review: Allada et al., 2001; Stanewsky, 2003).

In the fruit fly Drosophila melanogaster, not only were the molecular mechanisms of rhythm generation revealed with the help of mutants, but also the anatomical localization of the endogenous clock. Unlike unicellular life forms and plants, where each cell contains an autonomous circadian clock (e.g. Thain et al., 2000), animals possess a circadian pacemaker center in the brain whereas many peripheral cells and tissues of the body have lost their autonomous oscillatory capabilities. Furthermore, animals have developed specialized photoreceptor organs to perceive and transmit light to this circadian pacemaker center. Neuroanatomical mutants have greatly contributed to the localization of the circadian pacemaker center and its associated photoreceptors within the fruitfly brain. This paper reviews the current view of the molecular functioning of the clock in Drosophila as well as the main steps involved in the anatomical identification of the circadian pacemaker center and related photoreceptors.

### The first rhythm mutants were isolated in *Drosophila melanogaster*

In 1971, Konopka and Benzer performed the first genetic screen for rhythm mutants with the help of a chemical mutagen (ethyl methyl sulfonate), which induces point mutations. They found three mutants. Two had altered free-running periods of 19 hours and 29 hours, respectively, and the third showed no overt rhythmicity at all (Fig. 1). All three mutations mapped to the same locus on the X-chromosome and were named *period*<sup>Short</sup> (*per<sup>S</sup>*), *per*<sup>Long</sup> (*per<sup>L</sup>*), and *per*<sup>0</sup> (Konopka and Benzer, 1971).

The *period* gene turned out to be an integral part of the central clock, and the success of Konopka's screen prompted the search for other such factors with similar strategies (for review: Stanewsky 2002). Many years later, a second gene – *timeless* (*tim*) – which was similarly important for circadian rhythms was found (Sehgal et al., 1994). Three others – *Clock* (*Clk*), *cycle* (*cyc*) and *double-time* (*dbt*) – were discovered 4 years later (Allada et al., 1998; Rutila et al., 1998; Price et al.,

1998) followed by the finding of *vrille* (*vri*) (Blau and Young, 1999) and *shaggy* (*sgg*) (Martinek et al., 2001). All these genes and their products participate in complex negative and positive feedback loops that generate stable circadian oscillations at the molecular level in individual cells.

#### Rhythm generation depends on transcriptional molecular feedback loops

The molecular basis of the clock relies strongly, among other posttranscriptional and posttranslational mechanisms, on daily oscillations in the transcription of particular genes. One of the key features of these molecular oscillations is a negative feedback loop in which the protein products of the *per* and *tim* genes (PER and TIM) eventually turn off their own transcription. This feedback loop is roughly described below. For details the reader is referred to current reviews and the references therein (e.g. Stanewsky 2002).

The whole molecular cycle starts in the late morning when transcription of the *per* and *tim* genes is activated by the CLK and CYC proteins (Fig. 2; Allada et al., 1998; Rutila et al., 1998). The CLK and CYC proteins belong to the family of bHLH-PAS transcription factors (bHLH standing for a “basic Helix-loop-Helix” domain, which is involved in DNA binding; PAS has its name from PER-ARNT-SIM, the founding members of the PAS-protein family, and is a protein dimerization domain). CLK and CYC proteins form heterodimers, which bind to specific sequences (E-boxes) within the promoters of the *per* and *tim* genes to activate their transcription (Fig. 2). Transcriptional activation results in increasing amounts of *per* and *tim* mRNA throughout the day, that reach their maxima at the beginning of the night (So and Rosbash, 1997). Translation of the proteins probably starts shortly afterwards and the TIM protein reaches its peak level at around midnight (Marrus et al., 1996). However, the PER protein accumulates much more slowly and reaches its maximum in the second half of the night at around Zeitgeber Time (ZT) 21 (Fig. 2; Marrus et al., 1996). This delay in PER protein accumulation is due to the activity of the *dbt*-encoded protein kinase (DBT). DBT permanently phosphorylates PER and in this way promotes its degradation (Kloss et al., 1998; Price et al., 1998). As a result PER accumulates only slowly in the cytoplasm. This leads to a time delay between mRNA and protein accumulation, which is essential for the establishment of a stable negative feedback loop. When TIM is at its peak level, PER is finally stabilized by forming dimers with TIM (the PAS domain of PER is involved in the dimerization with TIM). Although DBT continues to phosphorylate PER, PER is now protected against degradation.

tion. TIM also gets progressively phosphorylated by the sgg encoded protein kinase (SGG). However, this phosphorylation does not lead to degradation of TIM, but instead promotes the nuclear entry of the PER-TIM dimer (Martinek et al., 2001). In the middle of the night the PER-TIM dimer enters the nucleus perhaps as a complex with DBT and SGG, since PER and TIM are progressively phosphorylated in the nucleus in a DBT and SGG dependent manner (Edery et al., 1994; Curtin et al., 1995; Price et al., 1998; Kloss et al., 2001; Martinek et al., 2001). After nuclear entry, this complex inhibits CLK-CYC induced transcription of the *per* and *tim* genes. The actual repression probably occurs by direct interactions of the PER-TIM dimer with the CLK-CYC dimer, resulting in a loss of the DNA-binding ability of the positive transcription factors (Lee et al., 1998; 1999). The TIM protein starts to be degraded shortly after nuclear entry of the PER-TIM dimer (Marrus et al., 1996) probably through the action of SGG kinase and possibly other tyrosine-kinases (Naidoo et al., 1999; Martinek et al., 2001). PER continues to repress the CLK-CYC dimer (Rothenfluh et al., 2000) until it is again degraded via DBT dependent phosphorylation. PER degradation allows a new round of *per* and *tim* transcription to occur in the late morning.

This oscillatory loop is reinforced by a second one (Glossop et al., 1999), in which the VRI protein rhythmically represses the transcription of the *Clk* gene. The VRI protein is a basic-Zipper (bZip) transcription factor, and the expression of the *vri* gene is regulated in a manner similar to that of the *per* and *tim* genes, meaning that its transcription is activated by CLK-CYC and repressed by PER-TIM (Blau and Young, 1999). Thus the level of VRI-protein is high at the end of the night and low at the end of the day. Consequently, *Clk* mRNA and CLK protein levels are low at the end of the night and high at the end of the day. In contrast to the rhythmically regulated *Clk* gene, the *cyc* gene is constitutively expressed and the CYC protein is highly abundant throughout the day. Therefore, CLK is the limiting factor in constituting the CLK-CYC dimer and it dictates the amount of functional dimers that are available at any time to activate the transcription of the *per* and *tim* genes (Bae et al., 2000). Interestingly, the transcription of both genes has been observed to be already substantially decreased even before nuclear translocation of the PER-TIM repressors (So and Rosbash, 1997). A likely explanation for this are the low CLK levels in the second half of the night that could cause the initial drop in *per* and *tim* transcription (Lee et al., 1998; Bae et al., 2000). Thus, the PER-TIM repressor complex and the PER protein alone would rather prevent newly accumulating CLK-CYC dimers from reinitiating transcription at the end of the night.

Zoology 105 (2002) 4

Circadian system of *Drosophila*

Fig. 1. The period mutants of *Drosophila melanogaster*. Typical actograms are shown for *period*<sup>Short</sup> (*per*<sup>S</sup>), *period*<sup>Long</sup> (*per*<sup>L</sup>), and *period*<sup>zero</sup> (*per*<sup>0</sup>) mutants. Locomotor activity of the flies was first recorded for 7 days in 12 hour : 12 hour light-dark cycles (LD) and subsequently for 3 weeks in constant darkness (DD). The bars on top of the actograms indicate the light program. Under LD conditions, the *per*<sup>S</sup> and *per*<sup>L</sup> mutants show a bimodal activity pattern, whereas that of the *per*<sup>0</sup> mutants is uniformly distributed during daylight hours. Under DD conditions, the *per*<sup>S</sup> mutant free-runs with a period of ~19 hours and the *per*<sup>L</sup> mutant shows a period of ~29 hours. The *per*<sup>0</sup> mutant shows an arrhythmic activity pattern. To better visualize the short and long free-running rhythms of *per*<sup>S</sup> and *per*<sup>L</sup> mutants the actograms are shown as double plots, meaning that the activity of two consecutive days is plotted in one row, additionally to plotting the activity of each consecutive day (day 1 to 29) vertically one below the other. For comparison with the activity pattern of a wild-type fly see figure 4.

ZT 10

cytoplasm

CRY
CYC CLK
E-box
per Gen
mRNA
CYC CLK
E-box
tim Gen
mRNA
CYC CLK
E-box
ccg Gen
mRNA
output
nucleus


ZT 18

cytoplasm

PER
CRY
TIM
CRY
TIM PER
CYC CLK
E-box
per gene
TIM
CRY
TIM PER
CYC CLK
E-box
tim gene
SGG
TIM PER
CYC CLK
E-box
ccg gene
output
nucleus


ZT 0

cytoplasm

CRY
TIM PER
DBT
CRY
CYC CLK
E-box
per gene
CRY
TIM PER
CYC CLK
E-box
tim gene
CRY
TIM PER
CYC CLK
E-box
ccg gene
SOG
nucleus


expression profiles

ZT


Fig. 2. Model of the molecular feedback loop, which leads to rhythmic accumulation of *per* and *tim* mRNAs and the respective proteins. The state of the feedback loop is shown at three time points during the LD-cycle (ZT = Zeitgeber Time). ZT 10 means 10 hours after lights-on (late afternoon), ZT 18 is in the middle of the night, and ZT 0 is just at lights-on (early morning). The relevant levels of PER (red) and TIM (blue) proteins measured on Western blots of whole head extracts can be seen in the lower diagram (solid curves with squares; after Marrus et al. 1996). The dotted curves represent the transcription rate of *per* (red) and *tim* (blue) genes determined with nuclear run-on assays (after So and Rosbash, 1997). At ZT 10 transcription of the *per* and *tim* genes is activated by the CYCLE-CLOCK dimers (CYC-CLK) that bind to E-boxes present in the upstream region of both genes. Transcription rate is maximal at this time and the mRNAs accumulate in the cytoplasm. Translation of the proteins has started, but the latter cannot accumulate because of the activity of the DOUBLE-TIME (DBT) kinase, which phosphorylates PER leading to its degradation. TIM also cannot accumulate because CRYPTOCHROME (CRY) is activated by light, binds to TIM and leads to its degradation via the proteasome. At ZT 18 PER is still degraded by DBT, but CRY is now inactive. Consequently, TIM accumulates and stabilizes PER by forming TIM-PER dimers. TIM gets phosphorylated by the SHAGGY (SGG) kinase and the TIM-PER complexes enter the nucleus. There, the TIM-PER dimers bind to CYC-CLK and thus inactivate the transcription of their own genes. At ZT 0 the TIM-PER dimers are entirely localized to the nucleus blocking transcription completely, at this time, no *per* and *tim* mRNAs can be found. At lights on CRY is activated once again leading quickly to the degradation of TIM (see steep decline in TIM level in the right diagram). PER persists for longer, blocking transcription until the middle of the day, then it gets degraded by DBT activity and the whole cycle starts from the beginning. In addition to rhythmically controlling transcription of the *per* and *tim* genes, the clock proteins also regulate the activity of *clock controlled genes* (ccgs), that then lead to rhythmic output in the cell. In the present example the CCG protein would be produced from late afternoon to late night, but not at other times.

The clock proteins do not only regulate the expression of their own genes in a rhythmic manner, but also that of other genes called “clock-controlled genes” (ccgs) (Fig. 2; van Gelder and Krasnow, 1996; Rouyer et al., 1997; Claridge-Chang et al., 2001; McDonald and Rosbash, 2001; Stempfl et al., 2002). Such ccgs could either be controlled by the first or the second oscillatory loop and thus have opposite phases of their rhythms (high protein levels at the end of the night and low levels at the end of the day, or vice versa). If we assume that the ccg encoded proteins are involved in different physiological processes, this would finally lead to rhythmic outputs of different phases at the cellular level.

### The molecular oscillations are entrained to light-dark cycles via light-dependent degradation of TIM

The feedback loops are entrained to the light-dark cycle by a light-dependent degradation of TIM via the proteasome (Naidoo et al., 1999). As soon as the fly has seen light, TIM protein levels of head extracts are drastically reduced (Fig. 2; Hunter-Ensor et al., 1996; Marrus et al. 1996; Myers et al., 1996) and as a consequence also PER levels drop, since PER is not very stable in the absence of TIM. By this way the molecular feedback loop can be reset by light-pulses and entrained by light-dark cycles. Since the TIM protein is not a light-sensitive molecule *per se*, the light signals have to be transferred via photoreceptor molecules and transduction cascades. One molecule involved in this process is the blue-light photoreceptor cryptochrome (CRY). In contrast to mammals, flies have only one cryptochrome and this is not implicated in rhythm generation, at least not in the pacemaker neurons (Hall, 2000). Instead, it is involved in photoreception. In a cell based *in vitro* assay, CRY interacts with TIM in a light dependent manner, leading to the degradation of TIM (Ceriani et al., 1999). Flies which overexpress the *cry* gene, are hypersensitive to light (Emery et al., 1998), whereas flies with a point mutation in the *cry* gene (*cry<sup>b</sup>* flies) are less light-sensitive (Stanewsky et al., 1998; Emery et al., 2000a; Helfrich-Förster et al., 2001; see also Fig. 7). Western blots of head extracts show that TIM and PER protein levels remain at a more less constant level in *cry<sup>b</sup>* mutants under LD-conditions, meaning that the molecular feedback loop is no longer entrainable in these mutants (see Fig. 7; Stanewsky et al., 1998). However, this is not true in every cell that expresses the clock genes. In some brain neurons that are relevant for rhythmic behavior, TIM is still degraded upon illumination. Thus, pathways besides *cry* must exist that lead to light-dependent TIM degradation and to entrainment of the feedback loop (see below).

---

**The molecular clock ticks in many cells and tissues**

The cloning of the *per* and *tim* genes and the generation of antibodies against PER and TIM proteins made it possible to demonstrate a broad distribution of both mRNAs and proteins within and outside the *Drosophila* CNS (reviewed by Hall, 1998). Cyclic expression of both clock genes was detected in a few brain neurons (see below), in many glial cells in the brain, as well as in the photoreceptor cells of the compound eyes and ocelli. Outside the brain *per/tim* cycling was nicely revealed in the Malpighian tubules – the fly’s “kidney” (Hege et al., 1997; Giebultowicz et al., 2000). Furthermore *per/tim* cycling was detected in most segments of the alimentary canal, in the fat body and in specific parts of the female and male reproductive system (re-

---

**Fig. 3.** The head of a *sine oculis* (*so*) mutant (below) in comparison to a wild-type head (top). Note that the *so* mutant lacks compound eyes as well as ocelli.

Ch. Helfrich-Förster

WT

SO

sol;so

disco

pdf$^{01}$

Fig. 4. Brains and actograms of flies with mutations affecting the optic system. WT wild-type, so sine oculis, sol;so double mutant of small optic lobes and sine oculis, disco disconnected. so and sol;so mutants have no ocelli or eyes (compare Fig. 3), disco mutants have eyes, but the connection between eye and brain is severely disturbed during development (Steller et al., 1987). As a consequence the optic lobes of all these mutants are reduced in size, most severely in sol;so mutants that have only ~5% of the optic lobes left (Fischbach and Technau, 1984). Nevertheless, sol;so mutants are still rhythmic, but their activity pattern in DD seems to be composed of 2 free-running rhythms: a prominent one with long period and a weaker one with short period, (after Helfrich, 1986). Most disco mutants are arrhythmic and their activity pattern looks more disturbed than that of mutants that lack the pigment-dispersing factor (pdf$^{0l}$ mutants), actograms of which are shown at the bottom. pdf$^{01}$ mutants show a rather normal activity pattern under LD conditions. After transfer into DD conditions they first free-run with a short period and then become arrhythmic.
viewed by Giebultowicz, 2000; 2001). Studies with transgenic flies, that express the luciferase gene of the firefly under *per* or *tim* regulatory sequences, revealed rhythmic expression of both genes also in the prothoracic gland (Emery et al., 1997) as well as in chemosensory hairs located on the flies' antennae, proboscis, wing margin, and legs (Plautz et al., 1997). Rhythmic clock gene expression in all these tissues continued even *in vitro* and was entrainable by light-dark cycles, speaking for brain independent autonomous clocks in many cells. These cell autonomous clocks were shown to have physiological functions in the antennae and in the male gonads. In the antennae Krishnan et al. (1999) found a circadian rhythm in the electroantennogram which is likely to be responsible for circadian differences in sensitivity to olfactory stimuli. A similar role for the circadian clock has also been proposed in the modulation of the photic sensitivity of the compound eye (Chen et al., 1992). In addition the male gonads spermatophore production and sperm mobility was found to occur in a rhythmic manner, and the possession of an intact rhythm in these functions significantly increases the reproductive fitness (Beaver et al., 2002). It is quite imaginable that coordinated rhythms in gut, malpighian tubules and fat body have similar adaptive advantages for digestion, detoxification and fat metabolism. Nevertheless, *per/tim* oscillations outside the brain do not contribute to circadian regulation of behavioral rhythms such as pupal eclosion or adult locomotor activity.

### Neuroanatomical studies localized the circadian pacemaker center controlling rhythmic behavior to a few cells located between the optic lobes and the central brain

The early experiments of Konopka et al. (1983) with mosaic animals, which carried the *per<sup>o</sup>* mutation in some tissues and were wild-type in others, had already shown that the circadian clock which controls rhythmic behavior is located in the head. Brain transplantation experiments corroborated this view and additionally suggested the involvement of the flies' neurosecretory system (Handler and Konopka, 1979). Given the broad distribution of clock gene expressing cells in the brain it was, however, initially hard to reveal the exact location of the pacemaker center. Help came from anatomical brain mutants that lacked the eyes and consequently large parts of the optic lobes (Fig. 3, 4). The optic lobes had previously been shown to contain the circadian pacemaker center in cockroaches and crickets (see Page, 1985 for review). Similarly, *Drosophila* mutants with tiny optic lobes showed altered activity rhythms:

Circadian system of *Drosophila*

(1) The locomotor activity profiles of “small optic lobes; sine oculis” (*sol;so*) double mutants frequently split into two components, suggesting that the underlying circadian system was less stable (Fig. 4; Helfrich, 1986), while (2) “disconnected” (*disco*) mutant flies were found to be arrhythmic (Fig. 4; Dushay et al., 1989).

The results suggested that *sol;so* mutants still possess the circadian pacemaker neurons – although somehow altered, whereas *disco* mutants lack these cells. Indeed, *disco* mutants lacked a group of neurosecretory cells located at the lateral margin of the central brain just between the brain and the optic lobes (Helfrich-Förster, 1991). This neurosecretory cell group was later shown to contain the neuropeptide “pigment dispersing factor” (PDF), a homolog of the crustacean pigment dispersing hormone (PDH) (Helfrich-Förster and Homberg, 1993). The PDF expressing cells could not be stained in *disco* mutants, showed altered arborizations in *sol;so* mutants, and were unaltered in other optic lobe mutants which showed normal behavioral rhythmicity (Fig. 5; Helfrich-Förster and Homberg, 1993). Taken together these data suggested that the PDF expressing cells were part of *Drosophila’s* circadian pacemaker and this view was further corroborated by the fact that the occasional presence of a single PDF-neuron in *disco* mutants was sufficient to engender robust activity rhythms in the corresponding fly (Helfrich-Förster, 1998).

But do the PDF neurons also express the clock genes? With the help of an antiserum against the PER protein Siwicki et al. (1988) labeled several groups of neurons; some were located in the lateral brain and were named Lateral Neurons (LNs), others in the dorsal brain and were named Dorsal Neurons (DNs) (Fig. 6). The LNs were in a position similar to the PDF-neurons and double-staining for PER and PDF showed that both cell groups partially overlapped (Helfrich-Förster, 1995): a ventral group of the LN (LN<sub>v</sub>) were both PER and PDF-positive, whereas a dorsal group (LN<sub>d</sub>) contained only PER. Besides PER, the PDF-cells were shown to contain the clock proteins TIM and VRI (Kaneko et al., 1997; Blau and Young, 1999). Unfortunately, no adequate antisera were available for DBT, CLK and CYC, but RNA “in situ” hybridizations revealed a broad distribution of cells expressing the corresponding genes throughout the cortical regions of the brain and optic lobes (Kloss et al., 1998; So et al., 2000). The PDF-cells are probably a subset of these labeled cells. Thus, it can be argued that the molecular feedback loop in which these genes are normally involved is probably working also in the PDF neurons. But what would the role of the other clock gene expressing neurons (the LN<sub>d</sub>s and the DNs) be? Numerous studies show that the LNs (LN<sub>v</sub>s and LN<sub>d</sub>s) are sufficient and essential for robust rhythms in adult activity, whereas the DNs seem to

Zoology **105** (2002) 4   303
Ch. Helfrich-Förster

WT  
SO  
sol;so  
disco  

play a minor role. Flies that express *per* only in the LNs (both clusters) but not in the DNs show a rhythmic locomotor activity (Frisch et al., 1994); by contrast *per* expression in brain cells other than the LNs is not sufficient to produce robust activity rhythms (Ewer et al., 1992). *disco* mutants that have no functional LNs (both clusters are lacking), but have normal DNs, show arrhythmic pupal eclosion and adult activity patterns (Fig. 5; Dushay et al., 1989); Nevertheless, the DNs are involved in the circadian system, since transgenic flies in which *per* is only expressed in the LNs have rhythms with longer period and lower power than wild-type flies (Frisch et al., 1994). Similarly, *glass*<sup>60j</sup> mutants that lack most DN<sub>1</sub>s have “noisier” rhythms with lower power but normal period (Helfrich-Förster et al., 2001). Furthermore, many *disco* flies show residual rhythmicity for the first few days under constant conditions, indicating that they still possess a dampened circadian clock (Dushay et al., 1989, Hardin et al., 1992; Helfrich-Förster, 1998). Indeed, PER levels vary cyclically in the DNs and in the glial cells of *disco* mutants (Zerr et al., 1990) and this cycling persists even in the absence of the LNs for at least 2 days under constant conditions (Blanchardon et al., 2001). Therefore, the DNs may be responsible for the residual rhythmicity of *disco* flies.

Although the LNs appear to be the main circadian pacemakers in the fruit fly, it is not clear whether both clusters of LNs (LN<sub>d</sub>s, LN<sub>v</sub>s) are equally important. Only the role of the LN<sub>v</sub>s has been studied so far, because these cells became genetically manipulable after the cloning of the *pdf*-gene (Park and Hall, 1998). PDF probably serves as mediator for circadian signals to downstream neurons (Helfrich-Förster et al., 2000; Park et al., 2000; Renn et al., 1999). Flies that carry a null mutation in the *pdf* gene – and thus interrupt the neuropeptide output from these pacemaker cells – become arrhythmic after some days in constant darkness (Renn et al., 1999). Similarly, transgenic flies, in which the LN<sub>v</sub>s are ablated specifically by activation of cell death or apoptosis genes are also arrhythmic (Blanchardon et al., 2001; Renn et al., 1999). Nevertheless, compared to *disco* mutants, ablation of LN<sub>v</sub> causes a much milder effect on behavioral rhythmicity: most *pdf*<sup>0I</sup> mutants and LN<sub>v</sub> ablated flies maintain a rhythm with a short period for at least one week under constant conditions before becoming arrhythmic (Fig. 5; Blanchardon et al., 2001; Renn et al., 1999), whereas *disco* flies in which the function of the LN<sub>d</sub> as well as the LN<sub>v</sub> have been eliminated show such rhythms for at most 1–3 days after being transferred into constant conditions (Dushay et al., 1989; Hardin et al., 1992; Helfrich-Förster, 1998). This indicates that in addition to the LN<sub>v</sub>s also the LN<sub>d</sub>s contribute to the control of the activity rhythm. Interestingly, all clock gene expressing

Fig. 5. Arborization pattern of neurons that contain the pigment-dispersing factor (PDF) in the brain of the different mutants with reduced optic lobes. PDF is present in two types of neurons that are called large and small ventral Lateral Neurons (l-LN<sub>v</sub>s and s-LN<sub>v</sub>s). The l-LN<sub>v</sub>s arborize on the surface of the second optic ganglion – the medulla (Me) – and send fibers to the contralateral optic lobe via the posterior optic tract (POT). The s-LN<sub>v</sub>s run into the dorsal central brain and terminate dorso-frontally of the calyces (Ca) of the mushroom bodies. Both types of neurons have arborizations in the accessory medulla (aMe), a small neuropil that was found to contain the circadian pacemaker center in various insects (for review: Helfrich-Förster et al., 1998). The PDF arborization pattern is more or less normal in *so* mutants, but disturbed in *sol;so* mutants. Here the fibers originating from the l-LN<sub>v</sub>s run into the dorsal protocerebrum instead of terminating on the surface of the medulla, which is severely reduced. In *disco* flies the PDF neurons are not stained at all (after Helfrich-Förster and Homberg, 1993).

Circadian system of *Drosophila*

R7/8  
R1-6  

H-B eyelet  

aMe  

DN₁ DN₂  
DN₃  
LNᵈ  
I-LNᵛ  
s-LNᵛ  

compound eye  

R7/8 terminals  
I-LNᵛ  
S-LNᵛ  
aMe  
H-B eyelet tract  

Fig. 6. Arborization pattern of all Lateral Neurons (LNᵈs, I-LNᵛs and s-LNᵛs) in relation to the location of the Dorsal Neurons (DN₁, DN₂, DN₃) and to light-input pathways from the compound eyes (photoreceptor cells R1–6 and R7/8) and the Hofbauer-Buchner eyelet (H-B eyelet). The lower pictures represent confocal images of wild-type brains stained with an antiserum against PDF (red) and against photoreceptor cells (green) (insets indicated above with dotted lines). On the left, the terminations of R7/R8 in the medulla and the tract from the H-B eyelet are shown. R7/R8 terminate in close vicinity to the arborizations of the I-LNᵛs on the surface of the medulla and the H-B eyelet tract runs into the accessory medulla and overlaps completely with putative dendritic fibers of the I-LNᵛs and s-LNᵛs in this neuropil. On the right, neurons expressing the cryptochrome gene are shown in transgenic flies that express the reporter green fluorescent protein (GFP) under the control of the cryptochrome promoter (*cry*-GAL4; UAS-*gfp* flies, Emery et al., 2000b). It is evident that the cryptochrome gene is expressed in all LN and DN. The yellow staining is the result of GFP(green) and anti-PDF(red) fluorescence colocalizing to the LNᵛs.

Zoology **105** (2002) 4

Ch. Helfrich-Förster

![Diagram](image1.png)

Fig. 7. In the cryptochrome mutant $cry^b$ the molecular feedback loop is still entrained in the majority of neurons, but not in the photoreceptor cells of the compound eyes. The diagram shows PER protein levels of whole head extracts during LD-cycles representing predominantly the PER levels in the compound eyes (see text). In $cry^b$ mutants no significant cycling is present (after Stanewsky et al., 1998). The histograms show PER-protein levels revealed with an antiserum against the PER-protein in individual neurons at ZT 10 (hatched bars) and ZT22 (filled bars). The different neurons are shown in color in the upper right cartoon and the colors match the bars in the histograms. In wild-type brains staining was prominent in all neurons at ZT 22 and no staining at all was found at ZT 10, whereas in $cry^b$ mutants staining was found at both time points. Nevertheless, there was a significant difference in staining intensity especially in the s-LN$_v$s and DN$_1$ between both time points, indicating that these neurons are still entrained (after Helfrich-Förster et al., 2001). Accordingly, the activity rhythm of $cry^b$ mutants can still be entrained by light-dark cycles as shown in the actograms at the bottom. The light period is indicated in yellow directly on the actograms. It was shifted by 8 hours on day 8, mimicking the effect of a fictitious intercontinental flight from Frankfurt to Washington. Wild-type flies adjust their activity pattern immediately to the new light-regime, whereas $cry^b$ mutants take several days to readapt. This shows that $cry^b$ mutants are less sensitive to light than wild-type flies, but that their circadian system is not blind.
neurons (DNs and LNs) project into the dorsal brain (see Fig. 6 for projections of the LNs and Kaneko and Hall, 2000 + Helfrich-Forster, 2003 for the projections of the DNs). This brain area has connections to most sites of the brain and furthermore houses much of the neurosecretory system of the fly. Thus, circadian signals arising from the clock-gene expressing cells may be transferred electrically and/or via humoral pathways to effector organs.

### The circadian pacemaker neurons differ in their molecular oscillations from other clock-gene expressing cells

The mechanisms of the described molecular feedback loop (Fig. 2) were deduced from oscillations of clock gene mRNAs or proteins observed on Northern- or Western blots of whole head extracts. The most abundant clock-gene expressing cells in the head are the photoreceptor cells of the compound eyes. One compound eye has ~800 ommatidia each containing 8 photoreceptor cells. This makes 6400 photoreceptor cells per eye and 12800 cells per head in contrast to a maximum of 150 *per/tim* expressing neurons in the whole brain (see Fig. 6). Thus, the latter neurons hardly contribute to the mRNA or protein levels measured in total head extracts. This is especially evident in the results obtained for the *cry<sup>b</sup>* mutant: Western blots of head extracts did not reveal any cycling in TIM or PER levels under LD conditions (Fig. 7; Stanewsky et al., 1998). Nevertheless, antibody staining at the cellular level showed that clock protein cycles were entrainable in all LNs and the DN<sub>1</sub> (Fig. 7). This suggests that, besides additional photoreceptors that lead to TIM degradation in the *cry<sup>b</sup>* mutants, other molecular components may also contribute to the clock-gene oscillations in the clock neurons. Indeed, there are preliminary indications that the feedback loops are differentially regulated in different neurons, and that some of these cells might not co-express *Clk* and *cyc* (Kaneko and Hall, 2000; Park et al., 2000). According to the feedback loop model, CLK and CYC are necessary to activate *per*- and *tim* expression (Allada et al., 1998; Rutila et al., 1998). Consequently, mutants in the *Clk* or *cyc* genes, like *Clk<sup>Jrk</sup>* and *cyc<sup>OL</sup>*, show virtually no *per/tim* mRNA or protein. This is true for the photoreceptor cells, the DNs, LN<sub>d</sub>s and s-LN<sub>v</sub>s (Kaneko, 2000). However, in the l-LN<sub>v</sub>s, TIM-immunoreactivity and *tim*- or *per*-GAL4 driven GFP expression are still present in *Clk<sup>Jrk</sup>* or *cyc<sup>OL</sup>* backgrounds, although at a reduced level (Kaneko, 2000; Kaneko and Hall, 2000). This suggests that other factors are involved in activating *per/tim* expression in these cells. Furthermore, the timing of TIM and PER nuclear accumulation in the clock neurons is

---

**Circadian system of *Drosophila***

not only different from the photoreceptor cells, but, for example, also differs between the large and small LN<sub>v</sub>s (Shafer et al., 2002). In both cell types PER was observed in the nucleus already in the early night (4 hours after lights-off; ZT 16), a time at which TIM was still completely cytoplasmic. TIM started to appear in the nucleus of the l-LN<sub>v</sub>s around 7 hours after lights-off (= ZT 19), but nuclear translocation did not occur before ZT 21 in the s-LN<sub>v</sub>s. Although these results do not preclude a catalytic role for TIM in PER nuclear entry, they demonstrate that PER and TIM need not always

![gl<sup>60J</sup> cry<sup>b</sup>](#)

| ZT 22 | ZT 10 |
| --- | --- |

![Histogram and Actogram](#)

---

Fig. 8. The circadian system of *glass<sup>60J</sup> cry<sup>b</sup> (gl<sup>60J</sup> cry<sup>b</sup>)* mutants in which all known photoreceptors have been eliminated is circadian blind. The molecular feedback loop in the pacemaker neurons of *gl<sup>60J</sup> cry<sup>b</sup>* mutants cannot be entrained to LD-cycles as can be seen in the upper histogram showing PER-staining at two relevant time points (see Fig. 7). PER staining was not found at all in the DN<sub>1</sub> suggesting that these neurons are eliminated by the mutation in the *glass* gene. Consistent with the histological results, the activity rhythm of *gl<sup>60J</sup> cry<sup>b</sup>* mutants does not entrain to LD-cycles as shown in the actogram (labeling as in Fig. 7; after Helfrich-Forster et al., 2001).

Zoology **105** (2002) 4   307
Ch. Helfrich-Forster

act as an obligate heterodimer. Furthermore, they show that TIM nuclear entry could be differently regulated in the two cell types, in which case other still unknown clock factors could contribute to this regulation. The differential control of clock components in the large and small LN<sub>v</sub>s is not restricted to the clock core, but also concerns the light input pathways (see below) and the output pathways. The neuropeptide PDF is a circadian output factor of the LN<sub>v</sub>s (Renn et al., 1999; Helfrich-Forster et al., 2000; Park et al., 2000). The clock proteins CLK, CYC and VRI are involved in the control of PDF-levels: CLK and CYC are necessary for transcription of the *pdf*-gene (Park et al., 2000) whereas the VRI-Protein suppresses the accumulation of the PDF peptide (Blau and Young, 1999). Importantly, this only applies to the case of the s-LN<sub>v</sub>s. In the lLN<sub>v</sub>s the PDF-level is completely independent of CLK, CYC or VRI. Furthermore, PER and TIM provoke a rhythmic accumulation of PDF in the axonal terminals of the s-LN<sub>v</sub>s but not in those of the lLN<sub>v</sub>s (Park et al., 1999). Together, these results indicate that the different neurons may serve different functions requiring a differential control of the molecular oscillations. The results also demonstrate that it is not enough to look at Northern and Western blots of head extracts to understand the functioning of the entire circadian system. Unraveling the interplay of different neurons at the cellular level is indispensable for a thorough comprehension.

The circadian pacemaker neurons are entrained to the environmental light-dark cycles by several photoreceptors

As mentioned above, the *per/tim*-expressing neurons do not only use the blue light sensitive photopigment cryptochrome to entrain their molecular oscillations to LD-cycles, although the *cry* gene appears to be expressed in all neuronal cell types (Fig. 6; Emery et al., 2000b). *cry<sup>b</sup>* mutants that lack functional cryptochrome (Stanewsky et al., 1998) still show entrainment of the molecular oscillations in most neurons, and almost wild-type entrainment in the s-LN<sub>v</sub>s and DN<sub>1</sub>s (Fig. 7). Accordingly, behavioral rhythmicity is also entrained in *cry<sup>b</sup>* mutants (Fig. 7; Stanewsky et al., 1998; Helfrich-Forster et al., 2001). There are several other photoreceptors that could entrain the circadian system of fruit flies: the compound eyes, the ocelli, and a pair of extraretinal photoreceptors called the H-B eyelets (Hofbauer and Buchner, 1989). Indeed, all photoreceptors appear to contribute to entrainment, because only flies that lack all these photoreceptors as well as CRY are not able to entrain to light-dark cycles (Fig. 8; Helfrich-Forster et al., 2001). Both the compound eyes and the H-B eyelets possess anatomical connections to the

pacemaker neurons (Fig. 6). The H-B eyelets project directly toward the LN<sub>v</sub>s (Hofbauer and Buchner, 1989; Yasuyama and Meinertzhagen, 1999) and their terminals overlap completely with putative dendritic arborizations of the s-LN<sub>v</sub>s and lLN<sub>v</sub>s (Helfrich-Forster et al., 2002). The compound eyes appear to contact the tangential network of lLN<sub>v</sub>s on the surface of the medulla, either directly through the photoreceptors R7 and R8 or indirectly via lamina monopolar cells that are innervated by photoreceptor cells R1–6 (Fig. 6). Only in the case of the ocelli no direct input to the LNs or DN<sub>1</sub>s is evident. Interestingly, the H-B eyelets stem from Bolwig’s organ, the larval photoreceptor (Helfrich-Forster et al., 2002). The latter together with cryptochrome has been shown to entrain the larval circadian system (Kaneko et al., 2000). Thus, in principal the adult fly maintains the larval circadian photoreceptor system and in addition recruits compound eyes and

![Figure 9](https://example.com/figure9.png)

Fig. 9. Action spectra conducted for entrainment of the activity rhythm of wild-type flies and different photoreceptor mutants. *so* mutants lack the eyes and ocelli but retain the H-B eyelets and cryptochrome, *so;gl<sup>60j</sup>* mutants possess only cryptochrome, and *so;cry<sup>b</sup>* mutants only the H-B eyelets. Some *so;cry<sup>b</sup>* mutants retain tiny rudiments of the compound eyes (10–50 ommatidia). The absorption maxima of cryptochrome and of the different fly rhodopsins are indicated at the top of the diagram. The compound eyes express rhodopsin 1 (Rh1) in photoreceptor cells R1–6 (the relevant metarhodopsin (M Rh1) is sensitive to long wavelengths and is also indicated) and rhodopsins 3–6 in the photoreceptor cells R7/R8. The ocelli use rhodopsin 2 (Rh2); the H-B eyelets Rh6 and possibly also Rh5 (Yasuyama and Meinertzhagen, 1999; Malpel et al., 2002). The results suggest that cryptochrome is responsible for the sensitivity peak at 420 nm, the H-B eyelets for that around 500 nm and the compound eyes for sensitivity throughout the tested spectral range, but especially for sensitivity beyond 550 nm (after Helfrich-Forster et al., 2002).

Zoology **1056** (2002) 4
ocelli which participate in the process of entrainment of  
the activity rhythm (for a more detailed review see Hel-  
frich-Förster and Engelmann, 2002).  
Action spectra for entrainment of the adult locomotor  
activity rhythm indicate that the H-B eyelets are re-  
sponsible for a sensitivity peak of the circadian system  
to blue-green light (around 480 nm), whereas cryp-  
tochrome is responsible for a sensitivity peak at 420 nm  
(Fig. 9; Helfrich-Förster et al., 2002). The compound  
eyes, on the other hand, contribute to circadian sensitiv-  
ity throughout the spectral range tested (350–650 nm).  
Multiple photoreceptors with different spectral sensi-  
tivities likely enable the fly to employ the entire spec-  
trum of light to accurately time the circadian clock to  
the natural environment. Additional work is necessary  
to further dissect the specific roles of the different pho-  
toreceptors in entrainment of the adult circadian sys-  
tem.

### Comparison of *Drosophila*'s circadian system  
with the circadian system of mammals  

Although flies and mammals appear to have little in  
common, their circadian systems show remarkable sim-  
ilarities. Their circadian pacemaker systems lie in close  
vicinity to the optic system: above the optic chiasma in  
mammals and in front of the optic tract that connects  
the optic lobe with the central brain in flies. Both pace-  
maker systems are tightly connected with the brain hor-  
monal system: with the pineal organ and the hypoph-  
ysis of the hypothalamus in mammals and with the neu-  
rosecretory system of the *pars intercerebralis* and *pars*  
*lateralis* in flies. Both pacemaker systems receive di-  
rect light input from the eye, but this input does not  
come from the conventional photoreceptors of the  
retina. The mammalian circadian system is innervated  
by the retinohypothalamic tract (RHT) that originates  
from photoreceptive ganglion cells but not from the  
rods and cones of the retina. In flies the pacemaker cen-  
ter receives input from the H/B eyelets, 4 photoreceptor  
cells that lie underneath the photoreceptor cells of the  
compound eye. Even at the molecular level there are  
great similarities. Both clocks employ negative au-  
toregulative feedback loops to generate circadian oscil-  
lations, and even the participating molecules are largely  
homologous.  
Nevertheless, there are slight differences between both  
systems. In flies, TIM is the dimerization partner of  
PER, whereas in mammals this role has been taken by  
CRY and the function of mammalian TIM is largely un-  
known. Furthermore, mammals have 3 PERs and 2  
CRYs whereas *Drosophila* possesses only one type of  
either gene. In addition the feedback loops of mammals  
and flies differ in the way they are entrained to light-  

Circadian system of *Drosophila*  
dark cycles. In mammals light triggers the transcription  
of the *per* genes, whereas in flies the TIM protein is de-  
graded by light. Whereas the pacemaker cells of  
*Drosophila* are directly sensitive to light, the mam-  
malian ones are not. The cellular light-sensitivity in  
flies is mediated by CRY, which appears to be localized  
in all clock-cells. Mammalian CRY is similar to the fly  
homolog, but in mammals it has lost its photoreceptive  
capabilities and has been recruited instead into the  
molecular mechanism of rhythm generation. This ap-  
pears reasonable considering that the pacemaker sys-  
tem of mammals is completely screened from direct  
light by the skull. Nonetheless, recent studies suggest  
that CRY may have retained its photoreceptive function  
in the ganglion cell layer of the mammalian retina and  
that CRY together with melanopsin – an opsin, which is  
also located in the retinal ganglion cells (Hattar et al.,  
2002; Berson et al., 2002) – mediates circadian pho-  
toreception via the RHT (Miyamoto and Sancar, 1998;  
Sancar, 2000; Selby et al., 2000). Interestingly, CRY  
seems to have a similar dual role also in flies. Whereas  
it obviously does not participate in the molecular clock-  
work of the pacemaker cells, it serves such functions in  
the periphery (Hall, 2000; Ivanchenko et al., 2001;  
Krishnan et al., 2001). Together these results suggest  
that the molecular composition of the clock core is vari-  
able and may be adapted to the requirements of the each  
individual system: it may not only differ between dif-  
ferent tissues but even between neuronal cell clusters in  
the same species.

### Acknowledgments  

I am thankful to Alois Hofbauer for comments on the  
manuscript and Ralf Stanewsky for discussions on the  
molecular feedback loop. The reported studies of my  
group were supported by the Deutsche Forschungs-  
gemeinschaft.

### References  

Allada, R., N. E. White, W. V. So, J. C. Hall and M. Rosbash. 1998. A mutant *Drosophila* homolog of mammalian Clock dis-  
rupts circadian rhythms and transcription of *period* and *time-  
less*. Cell 93: 791–804.  
Allada, R., P. Emery, J. S. Takahashi and M. Rosbash. 2001.  
Stopping time: the genetics of fly and mouse circadian clocks.  
Annu. Rev. Neurosci. 24: 1091–1119.  
Bae, K., C. Lee, P. E. Hardin and I. Edery. 2000. dCLOCK is pre-  
sent in limiting amounts and likely mediates daily interactions  
between the dCLOCK-CYC transcription factor and the PER-  
TIM complex. J. Neurosci. 20: 1746–1753.  
Beaver L.M., B. O. Gvakharia, T. S. Vollintine, D. M. Hege, R.  
Stanewsky and J. M. Giebultowicz. 2002. Loss of circadian  
clock function decreases reproductive fitness in males of  
Ch. Helfrich-Förster

Drosophila melanogaster. Proc. Nat. Acad. Sci. USA 99: 2134–2139.

Berson D.M., F. A. Dunn and M. Takao. 2002. Phototransduction by retinal ganglion cells that set the circadian clock. Science 295: 1070–1073.

Blanchardon, E., B. Grima, A. Klarsfeld, E. Chelot, P. E. Hardin, T. Préat and F. Rouyer. 2001. Defining the role of Drosophila lateral neurons in the control of circadian activity and eclosion rhythms by targeted genetic ablation and PERIOD overexpression. Eur. J. Neurosci. 13: 871–888.

Blau, J. and M. W. Young. 1999. Cycling *vrille* expression is required for a functional *Drosophila* clock. Cell 99: 661–671.

Ceriani, M. F., T. K. Darlington, D. Staknis, P. Más, A. A. Petti, C. J. Weitz, and S. A. Kay. 1999. Light-dependent sequestration of TIMELESS by CRYPTOCHROME. Science 285: 553–556.

Chen, D.-M., J. S. Christianson, R. J. Sapp, W. S. Stark. 1992. Visual receptor cycle in normal and *period* mutant *Drosophila*: microspectrophotometry, electrophysiology, and ultrastructural morphometry. Vis. Neurosci. 9: 125–135.

Claridge-Chang A., H. Wijnen, F. Naef, C. Boothroyd, N. Rajewsky and M. W. Young. 2001. Circadian regulation of gene expression systems in the *Drosophila* head. Neuron 32: 657–671.

Curtin K.D., Z. J. Huang, and M. Rosbash. 1995. Temporally regulated nuclear entry of the *Drosophila* period protein contributes to the circadian clock. Neuron 14: 363–372.

Dunlap, J. C. 1999. Molecular bases for circadian clocks. Cell 96: 271–290.

Dushay, M. S., M. Rosbash and J. C. Hall. 1989. The *disconnected* visual system mutations in *Drosophila* drastically disrupt circadian rhythms. J. Biol. Rhythms 4: 1–27.

Edery, I. 2000. Circadian rhythms in a nutshell. Phys. Genomics 3: 59–64.

Edery I., L. J. Zwiebel, M. E. Dembinska and M. Rosbash. 1994. Temporal phosphorylation of the *Drosophila* period protein. Proc. Natl. Acad. Sci. USA 91: 2260–2264.

Emery, I. F., J. M. Noveral, C. F. Jamison and K. K. Siwicki. 1997. Rhythms of *Drosophila* period gene expression in culture. Proc. Natl. Acad. Sci. USA 94: 4092–4096.

Emery P., W. V. So, M. Kaneko, J. C. Hall and M. Rosbash. 1998. CRY, a *Drosophila* clock and light-regulated cryptochrome, is a major contributor to circadian rhythm resetting and photosensitivity. Cell 95: 669–679.

Emery P., R. Stanewsky, J. C. Hall and M. Rosbash. 2000a. A unique circadian-rhythm photoreceptor. Nature 404: 456–457.

Emery, P., R. Stanewsky, C. Helfrich-Forster, M. Emery-Le, J. C. Hall and M. Rosbash. 2000b. *Drosophila* CRY is a deep brain circadian photoreceptor. Neuron 26: 493–504.

Ewer, J., B. Frisch, M. J. Hamblen-Coyle, M. Rosbash and J. C. Hall. 1992. Expression of the *period* clock gene within different cell types in the brain of *Drosophila* adults and mosaic analysis of these cells’ influence on circadian behavioral rhythms. J. Neurosci. 12: 3321–3349.

Fischbach, K.-F. and G. Technau. 1984. Cell degeneration in the developing optic lobes of the *sine oculis* and *small optic lobes* mutants of *Drosophila melanogaster*. Dev. Biol. 104: 219–239.

Frisch, B., P. E. Hardin, M. J. Hamblen-Coyle, M. Rosbash and J. C. Hall. 1994. A promoterless *period* gene mediates behavioral rhythmicity and cyclical *per* expression in a restricted subset of the *Drosophila* nervous system. Neuron 12: 555–570.

Giebultowicz, J. M. 2000. Molecular mechanism and cellular distribution of insect circadian clocks. Ann. Rev. Entomol. 45: 767–791.

Giebultowicz, J. M. 2001. Peripheral clocks and their role in circadian timing: insights from insects. Philos. Trans. Roy. Soc. Lond. B 356: 1791–1799.

Giebultowicz, J. M., R. Stanewsky, J. C. Hall, D. M. Hege. 2000. Transplanted *Drosophila* excretory tubules maintain circadian clock cycling out of phase with the host. Curr. Biol. 10: 107–110

Glossop, N. R. J., L. C. Lyons and P. E. Hardin. 1999. Interlocked feedback loops within the *Drosophila* circadian oscillator. Science 286: 766–768.

Hall, J. C. 1998. Molecular neurogenetics of biological rhythms. J. Neurogenet. 12: 115–181.

Hall, J. C. 2000. Cryptochromes: sensory reception, transduction, and clock functions subserving circadian systems. Curr. Opinion Neurobiol. 10: 456–466.

Handler, A. M. and R. J. Konopka. 1979. Transplantation of a circadian pacemaker in *Drosophila*. Nature 279: 236–238.

Hardin, P.E., J. C. Hall and M. Rosbash, M. 1992. Behavioral and molecular analyses suggest that circadian output is disrupted by disconnected mutants in *D. melanogaster*. EMBO J. 11: 1–6.

Hattar, S., H.-W. Liao, M. Takao, D.M. Berson, K.-W. Yau. 2002. Melanopsin-containing retinal ganglion cells: architecture, projections, and intrinsic photosensitivity. Science 295: 1065–1070.

Hege D. M., R. Stanewsky, J. C. Hall, J. M. Giebultowicz. 1997. Rhythmic expression of a PER-reporter in the Malpighian tubules of decapitated *Drosophila*: evidence for a brain-independent circadian clock. J. Biol. Rhythms 12: 300–308.

Helfrich, C. 1986. Role of the optic lobes in the regulation of the locomotor activity rhythm of *Drosophila melanogaster*: Behavioral analysis of neural mutants. J. Neurogenet. 3: 321–343.

Helfrich-Förster, C. 1991. Are neurosecretory cells involved in the circadian control of locomotor activity of *Drosophila melanogaster?* In: Chronobiology and Chronomedicine (C. Gutenbrunner, G. Hildebrandt and R. Moog, eds.). Peter Lang Verlag, Frankfurt: 134–140.

Helfrich-Förster, C. 1995. The *period* clock gene is expressed in CNS neurons which also produce a neuropeptide that reveals the projections of circadian pacemaker cells within the brain of *Drosophila melanogaster*. Proc. Nat. Acad. Sci. USA 92: 612–616.

Helfrich-Förster, C. 1998. Robust circadian rhythmicity of *Drosophila melanogaster* requires the presence of lateral neurons: a brain-behavioral study of *disconnected* mutants. J. Comp. Physiol. A 182: 435–453.

Helfrich-Förster, C. 2003. The neuroarchitecture of the circadian clock in the *Drosophila* brain. Microsc. Res. Tech. (in press).

Helfrich-Förster C. and U. Homberg. 1993. Pigment-dispersing hormone-immunoreactive neurons in the nervous system of wild-type *Drosophila melanogaster* and of several mutants with altered circadian rhythmicity. J. Comp. Neurol. 337: 177–190.

Helfrich-Förster C. and W. Engelmann. 2002. Photoreceptors for the circadian clock of the fruit fly. In: Circadian Rhythms (V. Kumar, ed.), Narosa Publishing House, New Delhi: 94–106.

Helfrich-Förster, C., M. Stengl and U. Homberg. 1998. Organization of the circadian system in insects. Chronobiol. Internat. 15: 567–594.

Helfrich-Förster, C., M. Täuber, J. H. Park, M. Mühlig-Versen, S. Schneuwly and A. Hofbauer. 2000. Ectopic expression of the neuropeptide pigment-dispersing factor alters behavioral rhythms in *Drosophila melanogaster*. J. Neurosci. 20: 3339–3353.

Helfrich-Forster, C., C. Winter, A. Hofbauer, J. C. Hall and R. Stanewsky. 2001. The circadian clock of fruit flies is blind after elimination of all known photoreceptors. Neuron 30: 249–261.

Helfrich-Forster, C., T. Edwards, K. Yasuyama, S. Schneuwly, R. Stanewsky, I. Meinertzhagen and A. Hofbauer. 2002. The extraretinal eyelet of *Drosophila*: development, ultrastructure and putative circadian function. J. Neuroscience 22: 9255–9266.

Hofbauer A. and E. Buchner. 1989. Does *Drosophila* have seven eyes? Naturwissenschaften 76: 335–336.

Hunter-Ensor, M., A. Ousley and A. Sehgal. 1996. Regulation of the *Drosophila* protein timeless suggests a mechanism for resetting the circadian clock by light. Cell 84: 677–685.

Ivanchenko M., R. Stanewsky and J. M. Giebultowicz. 2001. Circadian photoreception in *Drosophila*: functions of cryptochrome in peripheral and central clocks. J. Biol. Rhythms 16: 205–215.

Kaneko, M. 2000. Neural substrates of circadian rhythms in developing and adult *Drosophila*. Dissertation, Brandeis University, USA.

Kaneko, M. and J. C. Hall. 2000. Neuroanatomy of cells expressing clock genes in *Drosophila*: transgenic manipulation of the *period* and *timeless* genes to mark the perikarya of circadian pacemaker neurons and their projections. J. Comp. Neurol. 422: 66–94.

Kaneko, M., C. Helfrich-Forster and J. C. Hall. 1997. Spatial and temporal expression of the *period* and *timeless* genes in the developing nervous system of *Drosophila*: newly identified pacemaker candidates and novel features of clock gene cycling. J. Neurosci. 17: 6745–6760.

Kaneko, M., M. J. Hamblen and J. C. Hall. 2000. Involvement of the *period* gene in developmental time-memory: effect of the *per<sup>short</sup>* mutation on phase shifts induced by light pulses delivered to *Drosophila* larvae. J. Biol. Rhythms 15: 13–30.

Kloss, B., J. L. Price, L. Saez, I. Blau, A. Rothenfluh and M. W. Young. 1998. The *Drosophila* clock gene *double-time* encodes a protein closely related to human casein kinase Iε. Cell 94: 97–107.

Kloss, B., A. Rothenfluh, M. W. Young and L. Saez. 2001. Phosphorylation of PERIOD is influenced by cycling physical associations of DOUBLE-TIME, PERIOD, and TIMELESS in the *Drosophila* clock. Neuron 30: 699–706.

Konopka, R.J. and S. Benzer. 1971. Clock mutants of *Drosophila melanogaster*. Proc. Natl. Acad. Sci. USA 68: 2112–2116.

Konopka, R. J., S. Wells and T. Lee. 1983. Mosaic analysis of a *Drosophila* clock mutant. Molec. Gen. Genet. 190: 284–288.

Krishnan B., S. E. Dryer, P. E. Hardin. 1999. Circadian rhythms in olfactory responses of *Drosophila melanogaster*. Nature 400: 375–378.

Krishnan B., J. D. Levine, K. S. Lynch, H. B. Dowse, P. Funes, J. C., P. E. Hardin and S. E. Dryer. 2001. A new role for cryptochrome in a *Drosophila* circadian oscillator. Nature 411: 313–317.

Lee C., K. Bae and I. Edery. 1998. The *Drosophila* CLOCK protein undergoes daily rhythms in abundance, phosphorylation and interactions with the PER-TIM complex. Neuron 21: 857–867.

Lee C., K. Bae and I. Edery. 1999. PER and TIM inhibit the DNA binding activity of a *Drosophila* CLOCK-CYC/dBMAL1 heterodimer without disrupting formation of the heterodimer: a basis for circadian transcription. Mol. Cell. Biol. 19: 5316–5325.

Malpel, S., A. Klarsfeld, and F. Rouyer. 2002. Larval optic nerve and adult extra-retinal photoreceptors sequentially associate

Circadian system of *Drosophila*

with clock neurons during *Drosophila* brain development. Development 129: 1443–1453.

Marrus, S. B., H. Zeng and M. Rosbash. 1996. Effect of constant light and circadian entrainment of *per<sup>s</sup>* flies: evidence for light-mediated delay of the negative feedback loop in *Drosophila*. EMBO J. 15: 6877–6886.

McDonald M.J., M. Rosbash. 2001. Microarray analysis and organization of circadian gene expression in *Drosophila*. Cell 107: 567–578.

Martinek, S., S. Inonog, A. S. Manoukian and M. W. Young. 2001. A role for the segment polarity gene *shaggy/GSK-3* in the *Drosophila* circadian clock. Cell 105: 769–779.

Miyamoto Y. and A. Sancar. 1998. Vitamin B<sub>2</sub>-based blue-light photoreceptors in the retinohypothalamic tract as the photoactive pigments for setting the circadian clock in mammals. Proc. Natl. Acad. Sci. USA 95: 6097–6102.

Myers, M.P., K. Wagner-Smith, A. Rothenfluh-Hilfiker and M. W. Young. 1996. Light-induced degradation of TIMELESS and entrainment of the *Drosophila* circadian clock. Science 271: 1736–1740.

Naidoo, N., W. Song, M. Hunter-Ensor, A. Sehgal. 1999. A role for the proteasome in the light response of the *timeless* clock protein. Science 285: 1737–1741.

Ouyang, Y., C. R. Andersson, T. Kondo, S. S. Golden and C. H. Johnson. 1998. Resonating circadian clocks enhance fitness in cyanobacteria. Proc. Natl. Acad. Sci. USA 95: 8660–8664.

Page, T. L. 1985. Clocks and circadian rhythms. Comp. Insect Physiol. Biochem. Pharmacol. 6: 557–653.

Park, J. H. and J. C. Hall. 1998. Isolation and chronobiological analysis of a neuropeptide pigment-dispersing factor gene in *Drosophila melanogaster*. J. Biol. Rhythms 13: 219–228.

Park, J. H., C. Helfrich-Forster, G. Lee, L. Liu, M. Rosbash and J. C. Hall. 2000. Differential regulation of circadian pacemaker output by separate clock genes in *Drosophila*. Proc. Nat. Acad. Sci. US 97: 3608–3613.

Plautz, J. D., M. Kaneko, J. C. Hall and S. A. Kay. 1997. Independent photoreceptive circadian clocks throughout *Drosophila*. Science 278: 1632–1635.

Price, J. L., J. Blau, A. Rothenfluh, M. Abodeely, B. Kloss and M. W. Young. 1998. *double-time* is a new *Drosophila* clock gene that regulates PERIOD protein accumulation. Cell 94: 83–95.

Renn, S. C. P., J. H. Park, M. Rosbash, J. C. Hall and P. H. Taghert. 1999. A *pdf* neuropeptide gene mutation and ablation of PDF neurons each cause severe abnormalities of behavioral circadian rhythms in *Drosophila*. Cell 99: 791–802.

Reppert, S. M. 1998. A clockwork explosion! Neuron 21: 1–4.

Rothenfluh, A., M. W. Young and L. Saez. 2000. A TIMELESS-independent function for PERIOD proteins in the *Drosophila* clock. Neuron 26: 505–514.

Rouyer, F., M. Rachidi, C. Pikielny and M. Rosbash. 1997. A new gene encoding a putative transcription factor regulated by the *Drosophila* circadian clock. EMBO J. 16: 3944–3954.

Rutila, J.E., V. Suri, M. Le, W. V. So, M. Rosbash and J. C. Hall. 1998. CYCLE is a second bHLH-PAS clock protein essential for circadian rhythmicity and transcription of *Drosophila period* and *timeless*. Cell 93: 805–814.

Sancar A. 2000. Cryptochrome: the second photoactive pigment in the eye and its role in circadian photoreception. Annu. Rev. Biochem. 69: 31–67.

Sehgal, A., J. L. Price, B. Man and M. W. Young. 1994. Loss of circadian behavioral rhythms and *per* RNA oscillations in the *Drosophila* mutant *timeless*. Science 263: 1603–1606.

Selby C. P., C. Thompson, T. M. Schmitz, R. N. Van Gelder, A.

Ch. Helfrich-Forster

Sancar. 2000. Functional redundancy of cryptochromes and classical photoreceptors for nonvisual ocular photoreception in mice. Proc. Natl. Acad. Sci. USA 97: 14697–14702.

Shafer, O. T., M. Rosbash and J. W. Truman. 2002. Sequential nuclear accumulation of the clock proteins period and timeless in the pacemaker neurons of *Drosophila melanogaster*. J. Neurosci. 22: 5946–5954.

Siwicki, K.K., C. Eastman, G. Petersen, M. Rosbash and J. C. Hall. 1988. Antibodies to the *period* gene product of *Drosophila* reveal diverse tissue distribution and rhythm changes in the visual system. Neuron 1: 141–150.

So W.V. and M. Rosbash. 1997. Post-transcriptional regulation contributes to *Drosophila* clock gene mRNA cycling. EMBO J. 16: 7146–7155.

So, W.V., L. Sarov-Blat, C. K. Kotarski, M. J. McDonald, R. Alada and M. Rosbash. 2000. *takeout*, a novel *Drosophila* gene under circadian clock transcriptional regulation. Mol. Cell. Biol. 20: 6935–6944.

Stanewsky, R. 2002. Clock mechanisms in *Drosophila*. Cell Tiss. Res. 309: 11–26.

Stanewsky, R. 2003. Genetic Analysis of the Circadian System in *Drosophila melanogaster* and mammals. J. Neurobiol.: in press.

Stanewsky, R., M. Kaneko, P. Emery, B. Beretta, K. Wager-Smith, S. A. Kay, M. Rosbash and J. C. Hall. 1998. The *cry<sup>b</sup>* mutation identifies cryptochrome as a circadian photoreceptor in *Drosophila*. Cell 95: 681–692.

Steller, H., K.-F. Fischbach and G. M. Rubin. 1987. *Disconnected*: A locus required for neuronal pathway formation in the visual system of *Drosophila*. Cell 50: 1139–1153.

Stempfl T., M. Vogel, G. Szabo, C. Wülbeck, J. Liu, J.C. Hall and R. Stanewsky. 2002. Identification of circadian-clock-regulated enhancers and genes of *Drosophila melanogaster* by transposon mobilization and luciferase reporting of cyclical gene expression. Genetics 160: 571–593.

Thain, S.C., A. Hall and A. J. Millar. 2000. Functional independence of circadian clocks that regulate plant gene expression. Curr. Biol. 10: 951–956.

Van Gelder R.N. and M. A. Krasnow. 1996. A novel circadianly expressed *Drosophila melanogaster* gene dependent on the *period* gene for its rhythmic expression. EMBO J. 15: 1625–1631.

Yasuyama, K., I. A. Meinertzhagen. 1999. Extraretinal photoreceptors at the compound eye's posterior margin in *Drosophila melanogaster*. J. Comp. Neurol. 412: 193–202.

Young, M. W. and S. A. Kay. 2001. Time zones: a comparative genetics of circadian clocks. Nature Rev. Genet. 2: 702–715.

Zerr, D. M., J. C. Hall, M. Rosbash and K. K. Siwicki. 1990. Circadian fluctuations of *period* protein immunoreactivity in the CNS and the visual system of *Drosophila*. J. Neurosci. 10: 2749–2762.
