
Molecular Microbiology (2006) 59(3), 923–935

doi:10.1111/j.1365-2958.2005.04953.x

First published online 12 December 2005

# The vancomycin resistance VanRS two-component signal transduction system of Streptomyces coelicolor

Matthew I. Hutchings,${}^{*†‡}$ Hee-Jeon Hong${}^{\dagger}$ and Mark J. Buttner

Department of Molecular Microbiology, John Innes Centre, Norwich Research Park, Colney, Norwich NR4 7UH, UK.

## Summary

We took advantage of the vancomycin-dependent phenotype of *Streptomyces coelicolor femX* null mutants to isolate a collection of spontaneous, drug-independent *femX* suppressor mutants that expressed the vancomycin-resistance (*van*) genes constitutively. All of the suppressor mutations were in *vanS* but, unexpectedly, many were predicted to be loss-of-function mutations. Confirming this interpretation, a constructed *vanS* deletion mutation also resulted in constitutive expression of the *van* genes, suggesting that VanS negatively regulated VanR function in the absence of drug. In contrast, a *vanS pta ackA* triple mutant, which should not be able to synthesize acetyl phosphate, failed to express the *van* genes, whereas a *pta ackA* double mutant showed wild-type, regulated induction of the *van* genes. These results suggest that in the absence of vancomycin, acetyl phosphate phosphorylates VanR, and VanS acts as a phosphatase to suppress the levels of VanR~P. On exposure to vancomycin, VanS activity switches from a phosphatase to a kinase and vancomycin resistance is induced. In *S. coelicolor*, the *van* genes are induced by both vancomycin and the glycopeptide A47934, whereas in *Streptomyces toyocaensis* (the A47934 producer) resistance is induced by A47934 but not by vancomycin. We exploited this distinction to replace the *S. coelicolor vanRS* genes with the *vanRS* genes from *S. toyocaensis*. The resulting strain acquired the inducer profile of *S. toyocaensis*, providing circumstantial evidence that the VanS effector ligand is the drug itself, and not an intermediate in cell wall biosynthesis that accumulates as result of drug action. Consistent with this suggestion, we found that non-glycopeptide inhibitors of the late steps in cell wall biosynthesis such as moenomycin A, bacitracin and ramoplanin were not inducers of the *S. coelicolor VanRS* system, in contrast to results obtained in enterococcal VanRS systems.

## Introduction

Vancomycin is clinically important for treating enterococcal infections arising after abdominal surgery and is vital as the only widely effective treatment for infections caused by methicillin-resistant *Staphylococcus aureus* (MRSA), a major killer in hospital-acquired infections. Vancomycin and other glycopeptide antibiotics inhibit cell wall biosynthesis by binding to the D-alanyl-D-alanine (D-Ala-D-Ala) terminus of lipid-attached peptidoglycan precursors on the outside of the cytoplasmic membrane (Williams *et al.*, 1983; Barna and Williams, 1984). This interaction blocks formation of mature peptidoglycan, principally by denying transpeptidase access to its substrate, thereby preventing formation of the peptide crosslinks between polysaccharide strands that give the cell wall its rigidity. The first clinical isolates of vancomycin-resistant strains of pathogenic *Enterococcus faecalis* and *Enterococcus faecium* (VRE) appeared in the late 1980s, and were shown to reprogramme cell wall biosynthesis such that the stem pentapeptide of peptidoglycan precursors terminated in D-alanyl-D-lactate (D-Ala-D-Lac), rather than in D-Ala-D-Ala (Bugg *et al.*, 1991; Walsh *et al.*, 1996; Healy *et al.*, 2000; Pootoolal *et al.*, 2002a). The affinity of vancomycin for precursors terminating in D-Ala-D-Lac is ~1000-fold lower than for precursors terminating D-Ala-D-Ala (Bugg *et al.*, 1991), rendering the modified bacteria resistant. This remodelling of cell wall precursors requires three enzymes: VanH, which converts pyruvate into D-lactate; VanA, a D-Ala-D-Lac ligase; and VanX, a D-Ala-D-Ala dipeptidase that cleaves any residual circulating D-Ala-D-Ala dipeptide, ensuring that peptidoglycan precursors terminate uniformly in D-Ala-D-Lac. Because vancomycin is the front-line therapy for treating problematic infections caused by MRSA, the spread of vancomycin resistance through bacterial populations is an acute public health issue, highlighted by the recent emergence of vancomycin-resistant, methicillin-resistant *Staphylococcus aureus* (VRSA) in hospitals (Pearson, 2002; Chang *et al.*, 2003; Weigel *et al.*, 2003; Tenover *et al.*, 2004).

Accepted 10 October, 2005. *For correspondence. E-mail* matt.hutchings@bbsrc.ac.uk; *Tel.* (+44) 1603 450757; *Fax* (+44) 1603 450778. ${}^{\dagger}$These two authors contributed equally to this work. ${}^{\ddagger}$Present address: School of Biological Sciences, University of East Anglia, Norwich NR4 7TJ.

© 2005 The Authors

Journal compilation © 2005 Blackwell Publishing Ltd

924 M. I. Hutchings, H.-J. Hong and M. J. Buttner

We previously showed that the non-pathogen, *Streptomyces coelicolor*, carries a gene cluster conferring inducible, high-level resistance to vancomycin (Hong et al., 2004). *S. coelicolor* is the model species of a genus of Gram-positive, mycelial soil bacteria responsible for production of two-thirds of the commercially important antibiotics. *S. coelicolor* itself does not make a glycopeptide, but all of the known glycopeptide antibiotics are produced by actinomycetes, the bacterial order to which the streptomycetes belong. As most non-pathogenic actinomycetes live in the soil, it seems likely that *S. coelicolor* encounters glycopeptide producers and that the *van* gene cluster therefore confers a selective advantage. The *S. coelicolor* resistance genes are clearly associated with a laterally acquired DNA element (H.-J. Hong, unpublished); it therefore seems likely that *S. coelicolor* ultimately acquired the *van* genes from an actinomycete glycopeptide producer, which must carry these genes to avoid auto-toxicity.

The *S. coelicolor* cluster consists of seven genes, *vanSRJKHAX*, divided into four transcription units: *vanRS*, *vanJ*, *vanK* and *vanHAX* (Fig. 1; Hong et al., 2004). The VanRS two-component signal transduction system, the principal focus of the work reported here, activates transcription from the four *van* promoters in response to extracellular vancomycin. VanJ is a membrane protein of unknown function, and VanHAX, which are orthologous to the VRE enzymes, alter the cell wall precursors to terminate D-Ala-D-Lac (Hong et al., 2004; 2005).

*vanK* is a particularly interesting gene because it is essential for vancomycin resistance in *S. coelicolor*, despite the fact that it has no orthologues in the vancomycin-resistance gene clusters of pathogenic enterococci (Hong et al., 2004). VanK is a member of the Fem family of enzymes, which add the branch amino acid(s) to the stem pentapeptide of PG precursors. In *S. coelicolor*, the branch is a single glycine residue and, in the absence of vancomycin, this glycine is added by an enzyme called FemX (Hong et al., 2005). However, the constitutive FemX activity of *S. coelicolor* can recognize only precursors that terminate in D-Ala-D-Ala as a substrate, and VanK is therefore required for vancomycin resistance because it is the only enzyme that can add the Gly branch

to precursors terminating in D-Ala-D-Lac (production of precursors lacking the Gly branch is lethal in Streptomycetes because it prevents cross-linking of the peptidoglycan by transpeptidase, leading to cell lysis) (Hong et al., 2005).

*femX* is essential under normal growth conditions (Hong et al., 2005). However, because VanK replaces the function of FemX during vancomycin-induced cell wall precursor remodelling, *femX* null mutants are viable in the presence of vancomycin, but die in its absence, when VanK is not expressed (Hong et al., 2005). We took advantage of this drug-dependent phenotype to isolate a collection of spontaneous, drug-independent *femX* suppressor mutants that expressed the vancomycin-resistance genes constitutively, all of which mapped to the *vanS* sensor kinase gene. Here we report the resulting genetic and biochemical dissection of the VanRS two-component signal transduction system.

## Results

### Suppressors of *femX* map to *vanS*

When the vancomycin-dependent *femX* null mutant strain of *S. coelicolor* was plated for confluent lawns on medium lacking vancomycin, it gave rise to spontaneous suppressor mutants that grew in the absence of the antibiotic (Fig. 2). Using S1 nuclease protection analysis we found that, in two suppressor mutants chosen at random, the *vanH* promoter was transcribed in the absence of vancomycin, and that transcription was not induced to higher levels by addition of the drug, contrasting with the inducible transcription of the *van* genes seen in the wild type (Fig. 3). Based on a classical model of two-component systems (Hakenbeck and Stock, 1996), we expected these suppressor mutants to carry gain-of-function mutations in either *vanR* or *vanS*, causing constitutive activity of the signal transduction system. Accordingly, we isolated 23 independent spontaneous suppressor mutants and from each sequenced the *vanRS* operon plus 100 bp of flanking DNA on either side. In every case, the *vanR* sequence was wild type but a mutation was found in *vanS* (Table 1). Significantly, most of the sequenced changes

![Figure 1](#fig1)

**Fig. 1.** Organization and regulation of the vancomycin resistance (*van*) gene cluster of *S. coelicolor*. The *van* genes are organized into four transcription units, *vanRS*, *vanJ*, *vanK* and *vanHAX*, and these transcripts are induced by vancomycin in a *vanR*-dependent manner (Hong et al., 2004). The SCO designations for the *van* genes in the genome annotation are *vanS* (SCO3589), *vanR* (SCO3590), *vanJ* (SCO3592), *vanK* (SCO3593), *vanH* (SCO3594), *vanA* (SCO3595) and *vanX* (SCO3596).

wild type
$\Delta( pta \ ackA)$
$\Delta femX$
$\Delta femX$ suppressor
$\Delta vanS$
$\Delta vanRS$
$\Delta vanR$
$\Delta vanRS$
+ $vanR$ D51A
$\Delta vanRS$
+ $vanR$

wild type
$\Delta( pta \ ackA)$
$\Delta femX$
$\Delta femX$ suppressor
$\Delta vanS$
$\Delta vanRS$
$\Delta vanR$
$\Delta vanRS$
+ $vanR$ D51A

No drug
+ 10 $\mu$g/ml vancomycin

Fig. 2. Phenotypes of the wild type and mutant derivatives of S. coelicolor on MMCGT medium. Note that the $femX$ null mutant is vancomycin-dependent.

were not compatible with gain-of-function mutations. For example, two suppressors (J3135 and J3137) had IS element insertions in $vanS$, and many contained frame-shift alleles that could not conceivably give rise to a functional VanS protein (Table 1).

The $van$ genes are transcribed constitutively in a $vanS$ null mutant

Sequence analysis suggested that most (and therefore conceivably all) of the 23 characterized spontaneous

0   30   60   90   min
MIC
wild type
>200 $\mu$g/ml
$\Delta femX$ suppressor 1
>200 $\mu$g/ml
$\Delta femX$ suppressor 2
>200 $\mu$g/ml
$\Delta vanS$
160 $\mu$g/ml
$\Delta vanS$ ackA pta
<10 $\mu$g/ml
$\Delta ackA$ pta
>200 $\mu$g/ml

Fig. 3. Response of the $vanH$ promoter to vancomycin in the wild type and mutant derivatives of S. coelicolor. Strains were grown to mid-late exponential phase in NMMP liquid medium and exposed to 10 $\mu$g ml$^{-1}$ vancomycin. RNA was extracted from samples taken immediately before the addition of vancomycin and at subsequent 30 min intervals and analysed by S1 nuclease protection assays. The six panels are equivalent exposures and are therefore directly comparable. $\Delta femX$ suppressor 1 is J3134 and $\Delta femX$ suppressor 2 is J3135 (Table 1). The vancomycin MICs of the strains (determined on MMCGT solid medium) are shown on the right.

© 2005 The Authors
Journal compilation © 2005 Blackwell Publishing Ltd, Molecular Microbiology, 59, 923–935

926 M. I. Hutchings, H.-J. Hong and M. J. Buttner

Table 1. femX suppressor mutations.

| Suppressor | vanS mutation/comments |
|------------|------------------------|
| J3134      | Single base pair deletion in codon 92 |
| J3135      | Insertion element (SCO0022) between codons 266 and 267 |
| J3136      | Six base pair insertion in codon 197 |
| J3137      | Insertion element IS466A (SCO3490) between codons 302 and 303 |
| J3138      | P91L (in HAMP linker domain) |
| J3139      | A18V (in first transmembrane domain) |
| J3180      | L191P (in H-box domain) |
| J3181      | Single base pair insertion in codon 256 |
| J3182      | Single base pair deletion in codon 25 |
| J3183      | Single base pair insertion in codon 315 |
| J3184      | Single base pair insertion in codon 92 |
| J3185      | L216P (in ATPase domain) |
| J3186      | G271V (in ATPase domain) |
| J3188      | Single base pair insertion in codon 272 |
| J3189      | Six base pair insertion in codon 340 (identical to J3197) |
| J3190      | Single base pair insertion in codon 92 |
| J3192      | Single base pair insertion in codon 2 |
| J3193      | Single base pair insertion in codon 2 and single base pair deletion in codon 92 |
| J3194      | A124T (in HAMP linker domain) |
| J3195      | Single base pair insertion between codon 8 and codon 9 |
| J3196      | G322V (in ATPase domain) |
| J3197      | Six base pair insertion in codon 340 (identical to J3189) |
| J3198      | Single base pair insertion in codon 91 |

mutant. The vancomycin-sensitive phenotype of the vanRS deletion strain was complemented in trans by wild-type vanR but not by the D51A allele (Fig. 2), suggesting that phosphorylation of D51 is required for van gene activation.

Construction of a vanS pta ackA triple mutant suggests that VanR can be activated by acetyl phosphate in vivo.

An alternative explanation for constitutive expression of the van genes in a vanS null mutant would be that VanR~P can be generated in a VanS-independent manner and that VanS acts as a VanR~P phosphatase in the absence of vancomycin. VanS-independent synthesis of VanR~P might arise through crosstalk with one of the other 83 sensor-kinases in S. coelicolor (Hutchings et al., 2004). Alternatively, VanR could be activated by a small molecule phosphodonor such as acetyl phosphate. Acetyl phosphate is routinely used to phosphorylate response regulators in vitro in the absence of their cognate sensor kinases (Hakenbeck and Stock, 1996) and, in Escherichia coli at least, there is genetic evidence that acetyl phosphate can act as a phosphodonor in vivo, for example to the response regulator RssB, the σS regulator (Bouché et al., 1998). Synthesis and degradation of acetyl phosphate is controlled by acetate kinase (Ack) and phosphotransacetylase (Pta); in E. coli, an ackA mutant over-produces acetyl phosphate, a pta mutant has very low levels of acetyl phosphate, and an ackA pta double mutant cannot produce acetyl phosphate from acetate or acetyl CoA (Pruss and Wolfe, 1994). To see if acetyl phosphate might be responsible for generating VanR~P in a vanS null mutant, we introduced a further lesion into the vanS null mutant background, removing the pta and ackA genes, which are adjacent to each other in S. coelicolor. In contrast to the vanS mutant that grew readily on vancomycin, the vanS pta ackA triple mutant was vancomycin-sensitive (Fig. 2) and transcription from the vanH promoter was virtually undetectable in the presence or absence of vancomycin (Fig. 3), suggesting that acetyl phosphate can act as a phosphodonor to VanR in vivo.

VanS is both a kinase and phosphatase

The data given above were consistent with VanS acting as a VanR phosphatase in the absence of vancomycin, but they did not show whether exposure to vancomycin converted VanS into an active VanR kinase, or simply turned off its phosphatase activity. We addressed this question genetically and biochemically. First, we constructed a pta ackA double mutant in a vanS+ background and found that this strain was vancomycin resistant (Fig. 2) and that transcription of the van genes was induced in response to vancomycin (Fig. 3), suggesting

© 2005 The Authors
Journal compilation © 2005 Blackwell Publishing Ltd, Molecular Microbiology, 59, 923–935
that VanS can generate VanR~P in vivo. Second, we over-expressed VanR and the cytosolic domain of VanS (c-VanS, corresponding to residues 85–364) in E. coli and purified them to homogeneity as his-tagged proteins. Incubation of c-VanS with [γ-³²P] ATP resulted in autophosphorylation, with maximum phosphorylation occurring after 60 min (Fig. 4A). When radiolabelled c-VanS was mixed with VanR in the presence of ATP, rapid phosphotransfer was observed (Fig. 4B, lane 2), confirming that c-VanS has kinase activity. Purified VanR protein could also be phosphorylated using radiolabelled acetyl phosphate (Fig. 4B, lane 3). Incubation of VanR~P generated in this way with c-VanS resulted in removal of the phosphate group from VanR, confirming that c-VanS has phosphatase activity. This dephosphorylation is unlikely to involve reverse phosphotransfer to VanS because no radiolabelled c-VanS was detected in this reaction.

The femX mutant provides a novel bioassay for inducers of VanS

In enterococci, the nature of the effector ligand that binds the sensor domain of VanS to activate the signal transduction pathway is controversial. Although glycopeptides activate the enterococcal van genes, several structurally unrelated compounds that inhibit late steps in cell wall biosynthesis, such as moenomycin A, bacitracin and ramoplanin, have also been reported to induce van gene expression (Handwerger and Kolokathis, 1990; Allen and Hobbs, 1995; Baptista et al., 1996; 1999; Lai and Kirsch, 1996; Ulijasz et al., 1996; Grissom-Arnold et al., 1997;

A 0 0.5 1 2 5 15 30 60 min

B
![Picture 1]



Fig. 4. In vitro phosphorylation of VanS and VanR.
A. Autophosphorylation of the cytosolic domain of VanS (c-VanS) when incubated with [γ-³²P]ATP.
B. Lane 1, [γ-³²P]ATP-radiolabelled c-VanS; lane 2, phosphotransfer from radiolabelled c-VanS to VanR after 5 min coincubation; lane 3, radiolabelling of VanR using ³²P acetyl phosphate after 15 min of incubation.

S. coelicolor VanRS 927 Mani et al., 1998). Because these compounds have no structural similarity to each other or to the glycopeptides, the general consensus has been that the VanS effector ligand must be an intermediate in cell wall biosynthesis, such as Lipid II, which accumulates as a result of the action of the inhibition of cell wall biosynthesis, rather than the antibiotic itself.

We previously established a bioassay for inducers of the S. coelicolor VanRS signal transduction system by making a multicopy construct in which the vanJ promoter (vanJp) drove expression of the neo gene, which confers resistance to neomycin and kanamycin (Hong et al., 2004). Using this bioassay, we showed that a variety of glycopeptides induced van gene expression, but that cephalosporins and penicillins did not. However, we did not test moenomycin A, bacitracin or ramoplanin. In constructing the drug-dependent femX null mutant, we created an optimal bioassay for inducers in S. coelicolor. This strain is viable only in the presence of compounds that activate the VanRS signal transduction system, the read-out is simple growth, and there are no plasmids or reporter genes involved. We therefore extended our previous analysis of van gene inducer specificity in S. coelicolor using the femX null mutant. Confluent lawns of spores of the femX null mutant (J3130) were spread on MMCGT plates and potential inducers were applied on paper discs to the freshly spread plates. Inducers of the VanRS signal transduction system caused a halo of growth around the disc (Fig. 5). As we had previously found using the vanJp-neo bioassay (Hong et al., 2004), vancomycin, ristocetin, chloroeremomycin and A47934 acted as inducers. Significantly, however, moenomycin A, bacitracin and ramoplanin completely failed to rescue the growth defect of the femX null mutant (Fig. 5), showing that these compounds are not inducers of the VanRS signal transduction system in S. coelicolor.

The origin of the VanRS signal transduction system, not the host, determines inducer specificity

Streptomyces toyocaensis produces the glycopeptide antibiotic A47934, and the sequence of the entire A47934 biosynthetic gene cluster has been determined (Pootoolal et al., 2002b). Associated with the cluster are homologues of the S. coelicolor vanSRJKHAX genes. In S. coelicolor, the van genes are induced by both A47934 and vancomycin (Fig. 5; Hong et al., 2004). In contrast, in S. toyocaensis, resistance is induced by A47934 but not by vancomycin (Neu and Wright, 2001), perhaps implying that the ligands recognized by the VanS proteins from these two Streptomyces species are different. To test this hypothesis, we swapped the signal transduction systems of the two species by introducing the S. toyocaensis vanRS genes (vanRSSst) into the S. coelicolor vanRS dele-

© 2005 The Authors
Journal compilation © 2005 Blackwell Publishing Ltd, Molecular Microbiology, 59, 923–935
Fig. 5. Use of the vancomycin-dependent femX null mutant (J3130) as a bioassay for inducers of VanS. Approximately \(10^7\) spores of S. coelicolor J3130 were spread on a 12 cm × 12 cm plate and antibiotics were applied on 6-mm paper discs to the freshly spread plate. Inducers of VanS led to expression of *vanK* and hence induced a halo of growth around the disc. Kanamycin, which targets the ribosome, served as a negative control; all of the other antibiotics used target cell wall biosynthesis. The amount of antibiotic in each disc is indicated.

| Ristocetin (30µg) | Teicoplanin (30µg) | Vancomycin (30µg) | Chloroeremycin (30µg) |
|--------------------|--------------------|--------------------|------------------------|
| A47934 (30µg)      | Moenomycin A (30µg)| Bacitracin (5U)    | Ramoplanin (30µg)     |
| Tunicamycin (30µg) | Enduracidin (30µg) | Polymycin (30µg)   | D-cycloserine (30µg)   |
| Phosphomycin (30µg)| Methicillin (10µg) | Duramycin (100µg)  | Kanamycin (20µg)       |

tion mutant, using the integrative vector pRT801 (Gregory et al., 2003). Significantly, the resulting strain was resistant to A47934, but sensitive to vancomycin (Fig. 6A). Consistent with this result, vancomycin did not induce transcription from the *vanH* promoter in this strain, but A47934 did (Fig. 6B). Thus, switching the VanRS signal transduction system from one species to the other also switched inducer specificity.

### Discussion

Taken together, our data suggest the following model for the regulation of the VanRS signal transduction system in S. coelicolor (Fig. 7). In the absence of vancomycin, acetyl phosphate phosphorylates D51 of VanR, and VanS acts as a phosphatase to suppress the levels of VanR-P. In the presence of vancomycin, VanS is converted from a phosphatase to a kinase, leading to accumulation of VanR-P and activation of the four promoters of the *van* gene cluster. Transcription of the *vanRS* operon is itself under VanRS control (Hong et al., 2004) and so there will be very little VanR or VanS protein in S. coelicolor growing constantly in the absence of vancomycin. Thus, the 'futile cycle' of VanR phosphorylation and dephosphorylation shown occurring in the absence of vancomycin in Fig. 7 will occur at a high level only after the organism has been transiently exposed to the antibiotic. The duration of high level futile cycling will therefore depend on the half-lives of VanR and VanS *in vivo*.

An unresolved question is how much, if at all, acetyl phosphate contributes to VanR phosphorylation when the kinase activity of VanS is turned on in the presence of vancomycin. Certainly an *ackA pta vanS⁺* strain is resistant to vancomycin (Fig. 2) and still shows strong induction of the *van* genes (Fig. 3). If acetyl phosphate does make a significant contribution and thereby influences the organism's ability to respond to challenge by the antibiotic, then further questions arise: how widely do acetyl phosphate levels fluctuate in S. coelicolor, and what factors influence that fluctuation. It will be interesting to discover how many of the other 79 response-regulators in S. coelicolor can be phosphorylated by acetyl phosphate *in vivo* (Hutchings et al., 2004).

Fig. 6. The origin of the VanRS signal transduction system, not the host, determines inducer specificity.

A. Wild-type *S. coelicolor* is resistant to both vancomycin and A47934. However, when the *vanRS* genes of *S. coelicolor* are deleted and replaced with the *vanRS* genes from *S. toyocaensis*, the strain is resistant to A47934 but not to vancomycin, the same phenotype displayed by *S. toyocaensis* itself. Strains were grown on MMCGT medium.

B. S1 nuclease protection analysis of *vanH* transcripts in *S. coelicolor* M600 or *S. coelicolor ΔvanRS* carrying the *S. toyocaensis vanRS* genes. Strains were grown to mid-late exponential phase in NMM liquid medium and exposed to 10 μg ml⁻¹ vancomycin or 10 μg ml⁻¹ A479324. RNA was extracted from samples taken immediately before the addition of drug and at subsequent 30 min intervals. The three panels are equivalent exposures and are therefore directly comparable.

Similar results have previously been obtained in pathogenic enterococci. Depardieu *et al.* (2003) identified an *E. faecium* clinical isolate that expressed vancomycin resistance constitutively and showed that it carried an 18-bp in-frame deletion in *vanS* that resulted in a VanS protein deficient in phosphatase activity. Arthur *et al.* (1997) showed that the *van* promoters of *E. faecium* were constitutively activated by VanR in the absence of VanS, and concluded that VanS negatively controls VanR in the absence of glycopeptide inducer, presumably by dephosphorylation. Furthermore, Haldimann *et al.* (1997) demonstrated that heterologous expression of *E. faecium*

VanR in an *E. coli ackA* strain harbouring a *vanH–lacZ* fusion stimulated high levels of β-galactosidase production, suggesting that acetyl phosphate could act as a phosphodonor to the *E. faecium* VanR protein in *E. coli* in vivo.

In enterococci, the identity of the VanS effector ligand has been the subject of debate. Screens for inducers have been established by coupling a promoter under the control of VanRS to suitable reporter genes (Lai and Kirsch, 1996; Ulijasz *et al.*, 1996; Grissom-Arnold *et al.*, 1997; Mani *et al.*, 1998), by assaying VanX activity in cell extracts (Baptista *et al.*, 1996; 1999), by monitoring induc-

930 M. I. Hutchings, H.-J. Hong and M. J. Buttner

no drug
inducer

phosphatase
VanS
kinase
VanR
VanR
acetyl phosphate
expression
of van genes

Fig. 7. A model for the function of the vancomycin resistance VanRS two-component signal transduction system in *S. coelicolor*. In the absence of antibiotic, acetyl phosphate phosphorylates D51 of VanR, and VanS acts as a phosphatase to suppress the levels of VanR~P. In the presence of antibiotic, VanS is converted from a phosphatase into a kinase, leading to the accumulation of VanR~P and activation of the four promoters of the *van* gene cluster. Transcription of the *vanRS* operon is itself under VanRS control (Hong et al., 2004) and so there will be very little VanR or VanS protein in *S. coelicolor* growing constantly in the absence of vancomycin. Thus, the ‘futile cycle’ of VanR phosphorylation and dephosphorylation shown in the absence of vancomycin will occur at a significant level only after the organism has been transiently exposed to the antibiotic.

tion of Lac-containing precursors (Allen and Hobbs, 1995), or by looking for induced vancomycin resistance in pretreated cultures (Handwerger and Kolokathis, 1990; Allen and Hobbs, 1995). Most have concluded that the VanS effector ligand is an intermediate in cell wall biosynthesis, such as Lipid II, which accumulates as a result of blocking peptidoglycan biosynthesis. The key observation supporting this conclusion is that structurally unrelated, non-glycopeptide inhibitors of late steps in cell wall biosynthesis, such as moenomycin A, bacitracin and ramoplanin, have been reported to induce *van* gene expression in enterococci. However, it is important to note that these reports are not wholly consistent with each other: for example, some conclude that bacitracin is an inducer (Allen and Hobbs, 1995; Lai and Kirsch, 1996; Ulijasz et al., 1996; Grissom-Arnold et al., 1997), while others conclude that it is not (Baptista et al., 1996; Mani et al., 1998). Similarly, Grissom-Arnold et al. (1997) conclude that ramoplanin is an inducer, while Baptista et al. (1996) conclude that it is not. To address this issue in Streptomycetes, we exploited the phenotype of our constructed *femX* deletion mutant.

Streptomyces coelicolor *femX* null mutants are viable only in the presence of compounds that activate the VanRS signal transduction system, because they rely on expression of VanK for survival. We took advantage of this discovery to create a simple bioassay for inducers of the *van* genes in *S. coelicolor*. The structurally closely related glycopeptide antibiotics vancomycin, ristocetin, chloroeremomycin and A47934 all acted as inducers. However, in contrast to the enterococcal VanRS systems, moenomycin A, bacitracin and ramoplanin did not act as inducers of the VanRS system in *S. coelicolor*.

To address the effector ligand issue further, we carried out a ‘VanRS swap’ experiment between two glycopeptide-resistant Streptomycetes species with differing spectra

of inducer molecules, to see if inducer specificity was determined by VanRS itself or by the host background. In *S. coelicolor*, the *van* genes are induced by both A47934 and vancomycin, while in *S. toyocaensis*, resistance is induced by A47934 but not by vancomycin (Neu and Wright, 2001). Introducing the *S. toyocaensis* VanRS signal transduction system into an *S. coelicolor* *vanRS* null mutant switched inducer specificity to that of *S. toyocaensis*. Thus, inducer specificity is determined by the origin of VanRS. There are two potential explanations for this observation. If Streptomyces VanS is activated by accumulation of a cell wall intermediate, vancomycin must induce a radically different spectrum of cell wall intermediates in *S. coelicolor* and *S. toyocaensis*, which seems unlikely. The more likely alternative is that VanS is directly activated by binding antibiotic (or possibly antibiotic bound to D-Ala-D-Ala-containing cell wall precursors, such as Lipid II), and that *S. toyocaensis* VanS interacts productively with A47934 but not with vancomycin. This would also be consistent with the fact that structurally unrelated compounds that also inhibit late steps in cell wall biosynthesis, such as moenomycin A, bacitracin and ramoplanin, are not inducers of VanS in Streptomycetes.

Whether the VanS effector ligand is a cell wall intermediate or the drug itself, the ability to respond differentially to vancomycin and A47934 must reside in differences between the sensor domains of the *S. toyocaensis* and *S. coelicolor* VanS proteins. Alignment of the VanS primary sequences from these two species shows that the proteins are very similar, with 65% identity overall (Fig. 8). It is striking, however, that this high level of identity breaks down in the 26–27-residue stretch between the two predicted transmembrane helices, corresponding to the putative VanS sensor domain (Fig. 8).

© 2005 The Authors
Journal compilation © 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **59**, 923–935

S. coelicolor VanRS 931

TM helix 1                      extracytoplasmic sensor domain                     TM helix 2

VanSsc: M--DRRPGLSVRLKLTLSYAGFLTLAGVLLLVAVGVFLLDQGWLLLTNERGAVRATPG-TVFLRSFAPTAAWVMAELLVFGLVG : 80  
VanSst: MRVHRAAPGMSVRLKFTLSYAGFVMLAGTMLMLVTWVYLLRG--LPDAASSPONGLVPVRAIVLHAYAPATAVVLALLLVFGLLG : 81  

VanSsc: GWFLAGRMLAPLDRITEATRTAAATGSLSHRIRLPGRRDEYRELADAFDEMLARLEAHVAEQRRFAANASHELRTPLAVSKAIL : 163  
VanSst: GWLLAGRMLAPLTRITDATRTAAANGSLSHRIELEGRKDEFRELADAFDTMLARLEAHVAEQRRFAATASHELRTPLAITKALL : 164  

VanSsc: DVARTDPHQDPGEIIDRLHAVNTRAIDLTEALLLLSRAGQRSFTREQVDMSLLAEEATEETLLPFAEKHGVTLETTRGHVTLLALG : 246  
VanSst: EVTINGPGRDLGELIGRLHNVNARAIDLTEALLLLSRADQRSFTREPVDLSLNVEEAVENTLLPLAEKHGVAIIEASGDITPAMG : 247  

VanSsc: SPALLLQLTTNLVHNAIVHNLPGRRRVWIHTAAGPRTTTRLVVENTGDLTISP HQASTLTEPFQRGTERIHTDHPGVGLGLAIVN : 329  
VanSst: SRALLLQLTVNLIHNAIVHNLHGRGIVRVTTGIESGSVTLSVENTGEVLPAQLVSTLTEPERRGTERIRQDHAGVGLGLAIVK : 330  

VanSsc: TITQAHDGTLTLLTPRHSGGLRVTVELPAAAAPHT--GR : 364  
VanSst: SITEAHDGTLTIVPRPGGGLOVTVRLPAAAPPPETLNR : 367  

Fig. 8. Alignment of the VanS proteins from *S. coelicolor* (VanSsc) and in *S. toyocaensis* (VanSst). The two predicted transmembrane helices and the 26–27 amino acid putative extracytoplasmic sensor domain lying between are highlighted. The histidine that is the putative site of autophosphorylation is marked with an asterisk.

**Experimental procedures**

**Bacterial strains, plasmids, oligonucleotides and growth conditions**

Strains, plasmids and oligonucleotides are described in Table 2. MMCGT is agar minimal medium (MM; Kieser et al., 2000) supplemented with 0.6% (w/v) Difco Casaminoacids, 0.75% (v/v) Tiger Milk (Kieser et al., 2000), 0.5% (w/v) glucose. Except where described below, media and culture conditions were as given previously (Kieser et al., 2000; Hong et al., 2002).

**Protoplast transformation and conjugal plasmid transfer from E. coli to Streptomyces spp.**

To bypass the methyl-specific restriction system of *S. coelicolor*, cosmids and plasmids were passed through the *dam dcm hsdS E. coli* strain ET12567 prior to protoplast transformation or conjugation. *E. coli* ET12567 carrying the non-transmissible, *oriT*-mobilizing ‘driver’ plasmid pUZ8002 was used for conjugation. Streptomyces protoplast transformation and conjugation from *E. coli* to Streptomyces were carried out as described by Kieser et al. (2000).

**Isolating suppressors of femX**

J3130 was streaked on MS agar containing 10 μg ml⁻¹ vancomycin and 23 single colonies were restreaked on MS agar without vancomycin. A single *femX* suppressor was then isolated from each of these plates to ensure the mutants were not clonal. Genomic DNA was isolated from each suppressor strain as described (Kieser et al., 2000) and the *vanRS* genes, including 100 bp flanking DNA on either side, were amplified by polymerase chain reaction (PCR) using the

primers vanRFLANKI plus vanRFLANKII, and vanSFLANKI plus vanSFLANKII (Table 2) and sequenced using the same primers.

**Construction of insertion and deletion mutants**

Δ*vanS::apr* (J3199), Δ*vanRS::apr* (J2175) and Δ(*pta ackA*)::apr (J3202) insertion mutants were constructed by replacing most or all of the entire coding sequence with a cassette carrying the apramycin resistance gene (*apr*) and *oriT* of RK2, using the PCR-targeting method of Gust et al. (2003). The appropriate cosmid (H66 or IF6) was introduced into *E. coli* BW25113 carrying pIJ790, and the target gene was disrupted by electroporation of the cells with the PCR-amplified *apr-oriT* cassette, generated using primers carrying the appropriate gene-specific extensions (Table 2). The resulting cosmids [H66Δ*vanS::apr*, H66Δ*vanRS::apr* and IF6Δ(*pta ackA*)::apr] were introduced into *E. coli* ET12567 carrying pUZ8002 and transferred into *S. coelicolor* M600 by conjugation. Apramycin-resistant (*Apr*⁸), kanamycin-sensitive (*Kan*⁵) exconjugants were identified and purified, and the disruptions were confirmed by PCR and Southern analysis. Unmarked Δ*vanS* (J3200) and Δ*vanRS* (J3201) deletion mutants were also constructed; cosmids H66Δ*vanS::apr* and H66Δ*vanRS::apr* were individually introduced into *E. coli* BT340 and deletion of the *apr-oriT* cassette (which is flanked by FRT sites) was obtained after induction of the FLP recombinase (Gust et al., 2003). The mutant cosmids thus obtained, H66Δ*vanS* and H66Δ*vanRS*, were introduced into J3199 and J2175, respectively, by protoplast transformation, and *Kan*⁸ transformants were selected. After growth in the absence of antibiotic selection, colonies that had lost both apramycin and kanamycin resistance were selected and purified (J3200 and J3201; Table 2) and analysed by PCR to confirm replacement of *apr-oriT* cassette by the non-polar ‘scar’ sequence (Gust et al., 2003). J3203 [Δ(*pta ackA*)::apr]

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **59**, 923–935

Table 2. Bacterial strains, plasmids and primers used in this study.

| Strains | Relevant genotype/comments | Source/reference |
| --- | --- | --- |
| S. coelicolor A3(2) | SCP1⁻ SCP2⁻ | Kieser *et al.* (2000) |
| M600 | ΔfemX::apr SCP1⁻ SCP2⁻ | Hong *et al.* (2005) |
| J3130 | ΔvanS::apr SCP1⁻ SCP2⁻ | This study |
| J3199 | ΔvanS SCP1⁻ SCP2⁻ | This study |
| J3200 | ΔvanRS::apr SCP1⁻ SCP2⁻ | This study |
| J2175 | ΔvanRS SCP1⁻ SCP2⁻ | This study |
| J3201 | Δ(pta ackA)::apr SCP1⁻ SCP2⁻ | This study |
| J3202 | Δ(pta ackA)::apr ΔvanS SCP1⁻ SCP2⁻ | This study |
| E. coli |  |  |
| ET12567 (pUZ8002) | ET12567 containing helper plasmid pUZ8002 | Paget *et al.* (1999) |
| BT340 | DH5α containing pCP20 (temperature-sensitive FLP recombination plasmid) | Datsenko and Wanner (2000) |
| BW25113 (pIJ790) | BW25113 containing λ RED recombination plasmid pIJ790 | Gust *et al.* (2003) |

| Plasmids | Relevant genotype/comments | Source/reference |
| --- | --- | --- |
| pIJ773 | pBluescript KS (+) containing the apramycin resistance gene *apr* and the *oriT* of plasmid RP4, flanked by FRT sites | Gust *et al.* (2003) |
| pMS82 | φBT1 *attP-int* derived integration vector for the conjugal transfer of DNA from *E. coli* to Streptomyces spp. (*Hyg*ᴿ) | Gregory *et al.* (2003) |
| pRT801 | φBT1 *attP-int* derived integration vector for conjugal transfer of DNA from *E. coli* to Streptomyces spp. (*Apr*ᴿ) | Gregory *et al.* (2003) |
| pIJ6961 | pMS82 carrying *vanR* | This study |
| pIJ6959 | pMS82 carrying *vanR* D51A | This study |
| pIJ10277 | pRT801 carrying *vanRSst* | This study |

| Primers | Relevant genotype/comments |  |
| --- | --- | --- |
| vanH S1 FOR | TTCGACCTCTATATGAAGCGACGT |  |
| vanH S1 REV | TGAGAGTCGCCTCGACGCCGAAA |  |
| vanRFOR | CATATGGATAGGCGCCCCGG |  |
| vanRREV | GGATCCCTATCCACCAGTCGC |  |
| vanSFOR | CATATGGGACGCATGCTCGC |  |
| vanSREV | GGATCCTCACCTGCGCCGGTGT |  |
| vanRFLANKI | GTGCAGCATCATCACGAGGC |  |
| vanRFLANKII | TCAGACCGGGGCGCCTATCC |  |
| D51A1 | GCCGTCCCTCGCCCGGCACATC |  |
| D51A2 | GATGTCGCGGGCGAGGACGGC |  |
| vanSFLANKI | GGCCCTGCGCCTCGACCCCGTT |  |
| vanSFLANKII | TCACCTGCGGGTGTTGCGGGAG |  |
| VanS KO I | GGGGCGGTGAGAGCGACTCCGGGCACAGTCTTCCCT TCGCATTCCGGGGATCCGTGACC |  |
| VanS KO II | GTCAGAGCTACCGCTGAGCCGATGGGCGGGCCCTG GCGCTGTAGGCTGGGAGCTGCTTC |  |
| vanR KO I | TGCCGACGATATGTGGCGACTCGTAATCTCGACCCA TGATTCGGGGGATCCGTGACC |  |
| pta KO I | AGGATTCCCGCTCACC CCTACGAAACGGAGAGCACGC GTGATTCGGGGATCCGTGACC |  |
| ackA KO II | GGCCACGAGGGGTGTGCC TCGACCGCGACGTCCGCT CACCTGTAGGCTGGGAGCTGCTTC |  |

Δ*vanS* was constructed by introducing the Δ(*pta ackA*)::apr allele into the unmarked Δ*vanS* mutant, J3200, using the method described above.

**Construction of a D51A allele of *vanR* and complementation of a *vanR* null mutant**

A D51A allele of *vanR* was created using a pair of complementary primers containing the point mutation (D51A1 and D51A2; Table 2) and two flanking primers from the 5′-and 3′ regions of *vanR* (*vanRFLANKI* and *vanRFLANKII*; Table 2). Briefly, two PCR reactions were set up using *vanRFLANKI* plus D51A2 and *vanRFLANKII* plus D51A1. The two PCR products were gel purified, mixed, and used as template for a third PCR reaction containing only the flanking primers, and the resulting full-length PCR product was gel purified. The wild-type *vanR* allele was amplified from M600 genomic DNA using only *vanRFLANKI* and *vanRFLANKII*. The PCR products were cloned into Smal-cut pUC19, sequenced and sub-cloned into HindIII-KpnI-cut pMS82 (Gregory *et al.*, 2003) to create pIJ6959 (D51A) and pIJ6961 (wild-type), which were
introduced into the *vanRS* mutant J2175 by conjugal transfer from *E. coli* strain ET12567 (pUZ8002), selecting exconjugants with hygromycin.

### Overexpression of VanS and VanR

Full-length *vanR* was amplified using primers vanRFOR and vanRREV, and the cytosolic domain of VanS (*c*-VanS, corresponding to codons 85–364) was amplified using primers vanSFOR and vanSREV. The forward primers contained Ndel sites in frame with the translation start codon. The PCR products were cloned into Smal-cut pUC19, sequenced and subcloned into Ndel-HindIII-cut pET15b (Stratagene). The His-tagged proteins were overexpressed in BL21λDE3 (2 l) by induction with 0.1 mM IPTG at OD<sub>600</sub> = 0.6 at 37°C for 3 h.

### Purification of VanR

Cells were resuspended in 38 ml of binding buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM imidazole), passed twice through a French Press, and the lysate was clarified by centrifugation. The supernatant and pellet both contained VanR, but only the soluble protein was further purified. The supernatant was filtered through a 0.2 μm filter (Millipore) and applied to a 5 ml HiTrap Chelating Ni<sup>2+</sup> column. The column was washed with binding buffer until the UV trace was stable, and VanR was eluted with a linear 5–500 mM imidazole gradient (30 ml). Fractions containing VanR were stored at 4°C in elution buffer.

### Purification of *c*-VanS

Cells were resuspended in 35 ml of 20 mM Tris-HCl (pH 8.0), passed twice through a French Press, and the lysate was clarified by centrifugation. *c*-VanS was found only in the pellet, which was washed with distilled water and solubilized in sarkosyl buffer [50 mM Tris-HCl, pH 7.5, 250 mM NaCl, 0.2% (w/v) sarkosyl (*N*-lauroylsarcosine)]. After solubilization overnight at 4°C, the solution was clarified by centrifugation and filtered through a 0.2-μm filter, and *c*-VanS was purified by nickel affinity chromatography as described for VanR. Fractions containing *c*-VanS were stored at 4°C in elution buffer.

### In vitro phosphorylation assays

Immediately before use, purified VanR protein (0.5 ml) was applied to a Millipore Ultrafree-0.5 centrifugal filter device and centrifuged for 10 min at 10 000 r.p.m. in an Eppendorf 5415 D microcentrifuge for 10 min. The concentrated protein was diluted back to 0.6 ml with 50 mM Tris-HCl, pH 8.0, 250 mM NaCl, and this was repeated three times. For the autophosphorylation reaction, VanS (40 pmol) was incubated with 10 μCi [γ<sup>32</sup>P] ATP at room temperature for 1 h in 50 mM HEPES, pH 7.2, 5 mM MgCl<sub>2</sub> (50 μl total volume) and samples were taken at 0.5, 1, 2, 5, 15, 30 and 60 min and quenched immediately by addition of an equal volume of SDS-PAGE loading dye. For the phosphotransfer reaction, 40 pmol VanS was incubated for 60 min with 10 μCi [γ<sup>32</sup>P] ATP and then mixed with VanR protein (20 pmol) at room

S. coelicolor VanRS 933

temperature (50 μl total volume). Samples were taken at 1, 5, 15 and 30 min and quenched by addition of SDS-PAGE loading dye. [³²P] acetyl phosphate was prepared as described previously (Molle and Buttner, 2000) and incubated with VanR (20 pmol) at room temperature for 15 min (5 μl total volume). Reactions were quenched by addition of an equal volume of SDS-PAGE loading dye. All samples were loaded onto 15% SDS-polyacrylamide gels and separated at 200 V for 60 min at room temperature. Gels were dried and exposed to X-ray film overnight.

#### Complementation of the S. coelicolor vanRS null mutant with S. toyocaensis vanRS (vanRSst)

A 3 kb Sacl fragment carrying the *S. toyocaensis vanRS* operon was made blunt-ended using DNA Poll and was cloned into the conjugative vector pRT801 cut with PvuII. The resulting plasmid, pIJ10277, was introduced into the *S. coelicolor vanRS* null mutant J3201 by conjugal transfer from *E. coli* strain ET12567/pUZ8002, and exconjugants were selected with apramycin.

#### RNA preparation and S1 nuclease protection assays

RNA was prepared as described previously (Hong *et al.*, 2002; 2004). Germinated spores of *S. coelicolor* strains were inoculated into NMMP (Kieser *et al.*, 2000) and grown to an OD<sub>450</sub> of 0.3–0.6 at 30°C. Immediately after the first 10 ml sample was taken, inducer (10 μg ml⁻¹ vancomycin or 10 μg ml⁻¹ A47934) was added to the remaining 40 ml of culture broth and 10 ml samples were taken at subsequent 30 min intervals up to 90 min. The *vanH* probe was a 270 bp 5′ end-labelled fragment generated by PCR using primers vanH S1 FOR and vanH S1 REV (Table 2) and *S. coelicolor* M600 chromosomal DNA as a template. For all assays, 30 μg RNA and 25 pmol labelled probe were dissolved in 20 μl of NaTCA buffer and hybridized at 45°C overnight after denaturation at 65°C for 15 min.

### Acknowledgements

We thank Nancy Sheeler, Alan Wolfe, David Hopwood, Ray Dixon, Mark Paget and Gerry Wright for helpful discussions and comments on the manuscript, and Xiao Dong Wang and Gerry Wright for the gift of A47934 and the *S. toyocaensis vanRS* clone. This work was funded by BBSRC Grants 208/P20040 (to H.-J.H. and M.J.B.) and 208/P14575 (to M.J.B.) and by a grant-in-aid to the John Innes Centre from the BBSRC.

### References

Allen, N.E., and Hobbs, J.N. (1995) Induction of vancomycin resistance in *Enterococcus faecium* by non-glycopeptide antibiotics. *FEMS Microbiol Lett* **132**: 107–114.

Arthur, M., Depardieu, F., Gerbaud, G., Galimand, M., Leclercq, R., and Courvalin, P. (1997) The VanS sensor negatively controls VanR-mediated transcriptional activation of glycopeptide resistance genes of Tn1546 and related elements in the absence of induction. *J Bacteriol* **179**: 97–106.

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **59**, 923–935
Baptista, M., Depardieu, F., Courvalin, P., and Arthur, M. (1996) Specificity of induction of glycopeptide resistance genes in *Enterococcus faecalis*. Antimicrob Agents Chemother 40: 2291–2295.

Baptista, M., Rodrigues, P., Depardieu, F., Courvalin, P., and Arthur, M. (1999) Single-cell analysis of glycopeptide resistance gene expression in teicoplanin-resistant mutants of a VanB-type *Enterococcus faecalis*. Mol Microbiol 32: 17–28.

Barna, J.C.J., and Williams, D.H. (1984) The structure and mode of action of glycopeptide antibiotics of the vancomycin group. Annu Rev Microbiol 38: 339–357.

Bouché, S., Klauck, E., Fischer, D., Lucassen, M., Jung, K., and Hengge-Aronis, R. (1998) Regulation of RssB-dependent proteolysis in *Escherichia coli*: a role for acetyl phosphate in a response regulator-controlled process. Mol Microbiol 27: 787–795.

Bugg, T.D.H., Wright, G.D., Dutka-Malen, S., Arthur, M., Courvalin, P., and Walsh, C.T. (1991) Molecular basis for vancomycin resistance in *Enterococcus faecium* BM4147. Biosynthesis of a depsipeptide peptidoglycan precursor by vancomycin resistance proteins VanH and VanA. Biochemistry 30: 10408–10415.

Chang, S., Sievert, D.M., Hageman, J.C., Boulton, M.L., Tenover, F.C., Downes, F.P., et al. (2003) Infection with vancomycin-resistant *Staphylococcus aureus* containing the *vanA* resistance gene. N Engl J Med 348: 1342–1347.

Datsenko, K.A., and Wanner, B.L. (2000) One step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. Proc Natl Acad Sci USA 97: 6640–6645.

Depardieu, F., Courvalin, P., and Msadek, T. (2003) A six amino acid deletion, partially overlapping the VanS<sub>B</sub> G2 ATP-binding motif, leads to constitutive glycopeptide resistance in VanB-type *Enterococcus faecium*. Mol Microbiol 50: 1069–1083.

Gregory, M.A., Till, R., and Smith, M.C.M. (2003) Integration site for *Streptomyces* phage φBT1 and development of site-specific integrating vectors. J Bacteriol 185: 5320–5323.

Grissom-Arnold, J., Alborn, W.E., Nicas, T.I., and Jaskunas, S.R. (1997) Induction of VanA vancomycin resistance genes in *Enterococcus faecalis*: use of a promoter fusion to evaluate glycopeptide and nonglycopeptide induction signals. Microb Drug Resist 3: 53–64.

Gust, B., Challis, G.L., Fowler, K., KieserT., and Chater, K.F. (2003) PCR-targeted *Streptomyces* gene replacement identifies a protein domain needed for biosynthesis of the sesquiterpene soil odor geosmin. Proc Natl Acad Sci USA 18: 1541–1546.

Hakenbeck, R., and Stock, J.B. (1996) Analysis of two-component signal transduction system involved in transcriptional regulation. Methods Enzymol 273: 281–300.

Haldimann, A., Fisher, S.L., Daniels, L.L., Walsh, C.T., and Wanner, B.L. (1997) Transcriptional regulation of the *Enterococcus faecium* BM4147 vancomycin resistance gene cluster by the VanS-VanR two-component regulatory system in *Escherichia coli* K-12. J Bacteriol 179: 5903–5913.

Handwerger, S., and Kolokathis, A. (1990) Induction of vancomycin resistance in *Enterococcus faecium* by inhibition of transglycosylation. FEMS Microbiol Lett 58: 167–170.

Healy, V.L., Lessard, I.A., Roper, D.I., Knox, J.R., and Walsh, C.T. (2000) Vancomycin resistance in enterococci: reprogramming of the D-Ala-D-Ala ligases in bacterial peptidoglycan biosynthesis. Chem Biol 7: R109–R119.

Hong, H.-J., Paget, M.S.B., and Buttner, M.J. (2002) A signal transduction system in *Streptomyces coelicolor* that activates the expression of a putative cell wall glycan operon in response to vancomycin and other cell wall-specific antibiotics. Mol Microbiol 44: 1199–1211.

Hong, H.-J., Hutchings, M.I., Neu, J.M., Wright, G.D., Paget, M.S.B., and Buttner, M.J. (2004) Characterisation of an inducible vancomycin resistance system in *Streptomyces coelicolor* reveals a novel gene (*vanK*) required for drug resistance. Mol Microbiol 52: 1107–1121.

Hong, H.J., Hutchings, M.I., Hill, L.M., and Buttner, M.J. (2005) The role of the novel Fem protein VanK in vancomycin resistance in *Streptomyces coelicolor*. J Biol Chem 280: 13055–13061.

Hutchings, M.I., Hoskisson, P.A., Chandra, G., and Buttner, M.J. (2004) Sensing and responding to diverse extracellular signals? Analysis of the sensor kinases and response regulators of *Streptomyces coelicolor* A3(2). Microbiology 150: 2795–2806.

Kieser, T., Bibb, M.J., Buttner, M.J., and Chater, and Hopwood, D.A. (2000) Practical Streptomyces. Norwich: The John Innes Foundation.

Lai, M.H., and Kirsch, D.R. (1996) Induction signals for vancomycin resistance encoded by the *vanA* gene cluster in *Enterococcus faecium*. Antimicrob Agents Chemother 40: 1645–1648.

Mani, N., Sancheti, P., Jiang, Z.D., McNaney, C., DeCenzo, M., Knight, B., et al. (1998) Screening systems for detecting inhibitors of cell wall transglycosylation in *Enterococcus*. Cell wall transglycosylation inhibitors in *Enterococcus*. J Antibiot 51: 471–479.

Molle, V., and Buttner, M.J. (2000) Different alleles of the response regulator gene *bldM* arrest *Streptomyces coelicolor* development at distinct stages. Mol Microbiol 36: 1265–1278.

Neu, J.M., and Wright, G.D. (2001) Inhibition of sporulation, glycopeptide antibiotic production and resistance in *Streptomyces toyocaensis* NRRL 15009 by protein kinase inhibitors. FEMS Microbiol Lett 199: 15–20.

Paget, M.S.B., Chamberlin, L., Atrih, A., Foster, S.J., and Buttner, M.J. (1999) Evidence that the extracytoplasmic function sigma factor, σ<sup>E</sup>, is required for normal cell wall structure in *Streptomyces coelicolor* A3(2). J Bacteriol 181: 204–211.

Pearson, H. (2002) ‘Superbug’ hurdles key drug barrier. Nature 418: 469–470.

Pootoolal, J., Neu, J., and Wright, G.D. (2002a) Glycopeptide antibiotic resistance. Annu Rev Pharmacol Toxicol 42: 381–408.

Pootoolal, J., Thomas, M.G., Marshall, C.G., Neu, Hubbard, B.K., Walsh, C.T., and Wright, G.D. (2002b) Assembling the glycopeptide antibiotic scaffold: the biosynthesis of A47934 from *Streptomyces toyocaensis*. Proc Natl Acad Sci USA 99: 8962–8967.

Pruss, B.M., and Wolfe, A.J. (1994) Regulation of acetyl phosphate synthesis and degradation, and the control of

flagellar expression in *Escherichia coli*. Mol Microbiol 12: 973–984.

Tenover, F.C., Weigel, L.M., Appelbaum, P.C., McDougal, L.K., Chaitram, J., McAllister, S., *et al.* (2004) Vancomycin-resistant *Staphylococcus aureus* isolate from a patient in Pennsylvania. Antimicrob Agents Chemother 48: 275–280.

Ulijasz, A.T., Grenader, A., and Weisblum, B. (1996) A vancomycin-inducible *lacZ* reporter system in *Bacillus subtilis*: induction by antibiotics that inhibit cell wall synthesis and by lysozyme. J Bacteriol 178: 6305–6309.

Walsh, C.T., Fisher, S.L., Park, I.S., Prahalad, M., and Wu, Z. (1996) Bacterial resistance to vancomycin: five genes

and one missing hydrogen bond tell the story. Chem Biol 3: 21–28.

Weigel, L.M., Clewell, D.B., Gill, S.R., Clark, N.C., McDougal, L.K., Flannagan, S.E., *et al.* (2003) Genetic analysis of a high-level vancomycin-resistant isolate of *Staphylococcus aureus*. Science 28: 1569–1571.

Williams, D.H., Williamson, M.P., Butcher, D.W., and Hammond, S.J. (1983) Detailed binding sites of the antibiotics vancomycin and ristocetin A: determination of intermolecular distances in antibiotic/substrate complexes by use of the time-dependent NOE. J Am Chem Soc 105: 1332–1339.
