
Structure of the Hydrophilic Domain of Respiratory Complex I from  
**Thermus thermophilus**  
Leonid A. Sazanov and Philip Hinchliffe  
Science 311, 1430 (2006);  
DOI: 10.1126/science.1123809  

This copy is for your personal, non-commercial use only.

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by [clicking here](https://www.sciencemag.org).

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines [here](https://www.sciencemag.org).

The following resources related to this article are available online at www.sciencemag.org (this information is current as of November 14, 2014):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:  
[http://www.sciencemag.org/content/311/5766/1430.full.html](http://www.sciencemag.org/content/311/5766/1430.full.html)

Supporting Online Material can be found at:  
[http://www.sciencemag.org/content/suppl/2006/02/06/1123809.DC1.html](http://www.sciencemag.org/content/suppl/2006/02/06/1123809.DC1.html)

This article cites 49 articles, 20 of which can be accessed free:  
[http://www.sciencemag.org/content/311/5766/1430.full.html#ref-list-1](http://www.sciencemag.org/content/311/5766/1430.full.html#ref-list-1)

This article has been cited by 172 article(s) on the ISI Web of Science

This article has been cited by 80 articles hosted by HighWire Press; see:  
[http://www.sciencemag.org/content/311/5766/1430.full.html#related-urls](http://www.sciencemag.org/content/311/5766/1430.full.html#related-urls)

This article appears in the following subject collections:  
Biochemistry  
[http://www.sciencemag.org/cgi/collection/biochem](http://www.sciencemag.org/cgi/collection/biochem)

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2006 by the American Association for the Advancement of Science; all rights reserved. The title **Science** is a registered trademark of AAAS.

RESEARCH ARTICLE

# Structure of the Hydrophilic Domain of Respiratory Complex I from Thermus thermophilus

Leonid A. Sazanov* and Philip Hinchliffe

Respiratory complex I plays a central role in cellular energy production in bacteria and mitochondria. Its dysfunction is implicated in many human neurodegenerative diseases, as well as in aging. The crystal structure of the hydrophilic domain (peripheral arm) of complex I from *Thermus thermophilus* has been solved at 3.3 angstrom resolution. This subcomplex consists of eight subunits and contains all the redox centers of the enzyme, including nine iron-sulfur clusters. The primary electron acceptor, flavin-mononucleotide, is within electron transfer distance of cluster N3, leading to the main redox pathway, and of the distal cluster N1a, a possible antioxidant. The structure reveals new aspects of the mechanism and evolution of the enzyme. The terminal cluster N2 is coordinated, uniquely, by two consecutive cysteines. The novel subunit Nqo15 has a similar fold to the mitochondrial iron chaperone frataxin, and it may be involved in iron-sulfur cluster regeneration in the complex.

Complex I [dihydronicotinamide adenine dinucleotide (NADH)-ubiquinone oxidoreductase, EC 1.6.5.3] is the first enzyme of the mitochondrial and bacterial respiratory chains. It catalyzes the transfer of two electrons from NADH to quinone, coupled to the translocation of about four protons across the membrane, helping to provide the proton-motive force required for the synthesis of adenosine triphosphate (ATP) (1, 2). The mitochondrial enzyme contains 46 different subunits (3) and is one of the largest known membrane protein complexes. The prokaryotic enzyme is simpler and has 14 subunits with a combined molecular mass of about 550 kD. Analogs of all subunits of bacterial complex I (also referred to as NDH-1) are found in the mitochondrial enzyme (1), and they contain equivalent redox components (2). Mitochondrial and bacterial enzymes have a characteristic L-shaped structure, with the hydrophobic arm embedded in the membrane and the hydrophilic peripheral arm protruding into the mitochondrial matrix or the bacterial cytoplasm (4–6). Thus, NDH-1 is a “minimal” model of complex I. Until now, its atomic structure and the mechanism of electron transfer have not been known. Because of its central role in respiration, mutations in subunits of complex I can lead to many human neurodegenerative diseases (7). Also, complex I has been suggested to be a major source of reactive oxygen species (ROS) in mitochondria, which can damage mitochondrial DNA and may be one of the causes of aging (8).

The hydrophilic domain of complex I contains the NADH-binding site, the flavin-mononucleotide (FMN), and eight or nine iron-sulfur (Fe-S) clusters (2, 9), whereas the proton-pumping machinery is probably in the membrane arm. FMN accepts two electrons simultaneously (as a hydride) from NADH and transfers them one at a time to one-electron carriers, the Fe-S clusters. These subsequently reduce membrane-embedded quinone to quinol in two one-electron steps. In models of the subunit arrangement (10–13), subunits Nqo1 to 3 form the dehydrogenase domain, and subunits Nqo4 to 6 and Nqo9 connect it to the membrane arm (*Thermus* nomenclature is used throughout, with bovine mitochondrial nomenclature shown in parentheses in section headings). Fe-S clusters identified by electron paramagnetic resonance spectroscopy (EPR) include the binuclear clusters N1a and N1b, and the tetranuclear clusters N2, N3, N4, N5, N6a, and N6b (2, 9). Complex I from *Thermus thermophilus* and from some other bacteria contains the additional tetranuclear cluster N7 (14). At pH 7, the two-electron midpoint redox potential of NADH is about –320 mV, of ubiquinone +110 mV (*Thermus* uses menaquinone, –80 mV), and of FMN about –340 mV. The one-electron potential of cluster N1a is about –370 mV and of cluster N2 about –100 mV, and all the other clusters appear to be isopotential at about –250 mV (2, 9). It is likely that cluster N3 accepts electrons from FMN, whereas the high-potential cluster N2 reduces the quinone at the interface with the membrane domain (1, 2, 9).

We have solved the crystal structure of the hydrophilic domain (peripheral arm) of complex I from *T. thermophilus* HB-8. This subcomplex consists of seven known hydrophilic subunits (Nqo1 to 6 and Nqo9) and a previously unknown subunit, which we have identified as part of the complex and named Nqo15 (15, 16). The domain contains all the redox centers and represents more than half the molecular mass of the entire complex (280 out of 520 kD).

## Structure Determination and Overall Architecture

Two heavy atom derivatives were useful in providing phase information [in addition to previous Fe-edge data (15)], so that, after density modification, the resolution could be extended to 3.3 Å (17). Side chains became clearly visible (fig. S1), and the atomic model was built into the electron density. The model was refined to \( R_{\text{cryst}} = 26.5\% \) and \( R_{\text{free}} = 29.8\% \), with good stereochemistry (table S1). Some N- and C-terminal residues, as well as a few external loops, were not observed in the density and so were not modeled (17). The current model contains 2333 residues (out of 2510 predicted from the sequences), 9 Fe-S clusters, and 1 FMN molecule.

In the overview of the structure shown in Fig. 1A, the peripheral arm of complex I is a Y-shaped assembly about 140 Å high. One uppermost tip of the molecule is formed by the subunits Nqo1 and Nqo2, and the other by the C-terminal domain of Nqo3. The main stem is formed by the N-terminal domain of Nqo3 and the connecting subunits. Its lower part consists of subunits Nqo4 and Nqo6 (which coordinates the terminal Fe-S cluster N2), and it forms an interface with the membrane domain. The overall arrangement of subunits and clusters is consistent with our previous interpretation, apart from the assignment of clusters N4 and N5 in subunit Nqo3 (15), which are now reversed (Fig. 1B). The FMN is coordinated by subunit Nqo1 at the deep end of a solvent-exposed cavity containing an apparent NADH-binding site. FMN is within 14 Å, the maximum distance for physiological electron transfer (18), of both tetranuclear cluster N3 and binuclear cluster N1a. Thus, electrons can be transferred effectively from the flavin to cluster N3 and then, as judged by shortest edge-to-edge distances (Fig. 1B), through a series of five isopotential clusters to the terminal cluster N2, located next to the likely quinone-binding site. This pathway, NADH-FMN-N3-N1b-N4-N5-N6a-N6b-N2-quinone, is likely to be the main route for electron transfer within the enzyme. Cluster N7, in subunit Nqo3, is too far from the other clusters to accept electrons effectively. As proposed (15), binuclear cluster N1a may act as an antioxidant.

At the interface with the membrane domain, subunit Nqo6 contains a rigid (as judged from B-factors) N-terminal α helix H1 (Fig. 1A), involved in crystal contacts. This amphipathic helix protrudes about 25 Å from the complex and has a relatively polar upper surface and a

Medical Research Council Dunn Human Nutrition Unit, Wellcome Trust/MRC Building, Hills Road, Cambridge CB2 2XY, U.K.

*To whom correspondence should be addressed. E-mail: sazanov@mrc-dunn.cam.ac.uk
hydrophobic lower surface. It is, therefore, likely to extend into the surface region of the membrane domain. This interpretation is consistent also with the fit of the structure into a low resolution model of the intact *Escherichia coli* complex I, obtained by cryo-electron microscopy (19). Thus, helix H1 can help in the orientation of the model relative to the intact complex; the membrane domain is likely to extend in the direction of the N terminus of this helix (to the right in Fig. 1A). Extensive intersubunit contacts are present in the structure (table S2) and are listed in (17).

### Protein Subunits and Redox Centers

#### Subunit Nqo1 (51 kD)
This subunit contains the NADH-binding site, the primary electron acceptor FMN, cluster N3, and as a whole, it has no significant sequence similarity to any proteins of known structure. The subunit can be separated roughly into four domains (Fig. 2A): an N-terminal domain (residues 7 to 72, *Thermus* numbering throughout) ending with a glycine-rich loop, followed by a Rossman-fold domain (73 to 240), a ubiquitin-like domain (241 to 335) and a C-terminal four-helical bundle containing cluster N3 (336 to 438).

The N-terminal domain wraps around the Rossman-fold domain on the surface of the complex and contains three short α helices. Searches with SSM [secondary structure matching (20)] and DALI (21) at the European Bioinformatics Institute servers did not reveal significant structural analogs of this domain. Below and in (17), alignments performed with SSM are listed, although DALI was used for the initial searches. The N-terminal domain ends with a glycine-rich loop (residues 62 to 72) that was previously suggested to be involved in binding the adenosine diphosphate (ADP) moiety of NADH, as part of a Rossman nucleotide-binding fold (1). Indeed, the Rossman fold–like domain is found after this loop, but it is not a classical Rossman fold and it appears to bind FMN, and presumably NADH as well, in a novel mode. Instead of a six-stranded parallel β sheet with a 6, 5, 4, 1, 2, 3 order of strands (22), there are only four strands in the order 4, 1, 2, 3, flanked by α helices (Fig. 2A).

In a classical Rossman fold, the loop between the first strand and nascent helix is glycine rich and binds one of the nucleotide phosphate groups. However, in Nqo1, this loop (residues 93 to 102) does not contain glycine; it is mostly shielded from the solvent and interacts with the buried edge of the isoalloxazine ring of FMN. Flavin is bound at the deep end of a solvent-exposed cavity [previously called channel 1 (15)], which contains an apparent binding site for NADH (Fig. 3A). FMN interacts mostly with strand four of the parallel β sheet and with loops emerging from strands one and three. It is held in place by a hydrogen-bonding network (17) and interacts mainly with residues from a previously predicted flavin-binding domain [residues 175 to 220 (1)], but also with residues among those previously suggested to bind NADH [64 to 70 and 90 to 100 (1)].

The FMN cavity can accommodate one NADH molecule comfortably, as indicated by manual docking (not shown on figure), and the residues likely to interact with NADH are shown in Fig. 3A. The invariant residues Glu⁹⁷ and Tyr¹⁸⁰ are exposed to the solvent at the deep end of the cavity and can interact with the nicotinamide carboxyamide. Residues 66 to 69 from the glycine-rich loop can bind the phosphate groups of the substrate. Consistently, our preliminary data obtained from NADH-containing crystals (at about 5 Å resolution), show a strong additional electron density in this area. The invariant Glu¹⁸⁵ (and/or Glu¹⁸⁴) can make hydrogen bonds to the ribose of the adenosine moiety, similar to the aspartate residue conserved in other dehydrogenases (22). Notably, the aromatic rings of Phe⁷⁸ and Phe²⁰⁵, near the entrance to the cavity, are nearly parallel and about 8.5 Å apart. These residues are well positioned to coordinate the adenine ring between side chains, by aromatic stacking interactions. If NADH binds as proposed, its nicotinamide ring will be adjacent to the isoalloxazine ring of FMN, which allows hydride transfer. Similar elements of NAD(P)(H) binding are found in other nucleotide-binding proteins, but in different structural environments. All the residues involved in binding FMN and NADH are strongly conserved, which suggests that the binding pocket is conserved. Thus, in complex I, uniquely, a Rossman-fold domain has evolved to bind both FMN and NADH, by the addition of an extra glycine-rich loop at the N terminus.

The subsequent ubiquitin-like domain has a structural role, but it may also be involved in degradation of complex I, as the domain is exposed at the surface near the tip of the molecule (17). Finally, the C-terminal domain coordinates cluster N3 and consists of a four-helix bundle. Although this fold is quite common [fold a.24 in SCOP (23)], it has not been observed previously to bind Fe-S clusters. The [4Fe-4S] cluster is coordinated in the cubane geometry by Cys³⁵⁴, Cys³⁵⁶, and Cys³⁵⁹ from the loop between the first and second helices of the bundle and by Cys⁴⁰⁰ from the loop between the third and fourth helices (fig. S2A).

#### Subunit Nqo2 (24 kD)
This subunit can be divided into two domains: an N-terminal four-helical bundle (residues 3 to 75) (17) and a thioredoxin-like C-terminal domain (residues 76 to 180) (Fig. 2B). The N-terminal domain is involved in interactions with subunits Nqo1 and Nqo3. The C-terminal domain coordinates the binuclear cluster N1a and consists of a mixed β sheet flanked by two α helices. It is very similar to the thioredoxin-like [2Fe-2S] ferredoxin from *Aquifex aeolicus* [Protein Data Bank

---

**Fig. 1. Architecture of the hydrophilic domain of *T. thermophilus* complex I. (A) Side view, with the membrane arm likely to be beneath and extending to the right, in the direction of helix H1. Each subunit is colored differently; FMN is shown as magenta spheres, metal sites as red spheres for Fe atoms and yellow spheres for S atoms. A possible quinone-binding site (Q) is indicated by an arrow. (B) Arrangement of redox centers. The overall orientation is similar to that in (A), tilted to provide an improved view of the FMN and the clusters. Cluster N1a is in subunit Nqo2; N3 and FMN in Nqo1; N1b, N4, N5, and N7 in Nqo3; N6a/b in Nqo9; and N2 in Nqo6. The main pathway of electron transfer is indicated by blue arrows, and a diversion to cluster N1a by a green arrow. The distances between the centers given in angstroms were calculated both center-to-center and edge-to-edge (shown in parentheses). Clusters N3 and N4 are separated by 17.6 Å (13.8 Å edge-to-edge), and clusters N1b and N5 by 19.2 Å (16.7 Å edge-to-edge).**


www.sciencemag.org     SCIENCE     VOL 311     10 MARCH 2006     1431

(PDB) identifier 1f37 (24), root mean square deviation (RMSD) = 1.7 Å over 80 Cα atoms, with 19% sequence identity, including cluster-ligating cysteines]. This similarity was noted previously from sequence comparisons (25). The stability of the fold is enhanced by a disulfide link between Cys¹⁴⁴ from the last β strand and Cys¹⁷² from a surface-exposed loop.

Cluster N1a is coordinated by cysteines 83, 88, 124, and 128. The cluster is only about 12 Å from the isoalloxazine ring, and so can accept electrons from FMN, but cannot pass them on directly to the nearest cluster N3, which is 19 Å away. Polar residues in the environment of this and other clusters are listed in (17). Such residues, located near the clusters, could help adjust redox potentials (26, 27). N1a appears to have a more hydrophobic environment than other clusters, which may explain its low midpoint potential.

Subunit Nqo3 (75 kD). This is the largest subunit in complex I, comprising two main parts, an N-terminal [FeFe]-hydrogenase-like domain (residues 1 to 240) and a domain similar to molybdopterin-containing enzymes (residues 241 to 767, subdomains I to IV) (Fig. 2C).

As predicted from sequence similarities (1, 25), the N-terminal domain of Nqo3 is similar to the N-terminal domain of [FeFe]-hydrogenases; residues 1 to 240 align with an RMSD = 2.0 Å and 23% sequence identity over 177 Cα atoms, with residues 1 to 206 from [FeFe]-hydrogenase [PDB 1feh (28)]. The coordination of clusters N1b, N4, and N5 within Nqo3 (fig. S2C) is similar (in its ligands and the distances between clusters) to the coordination of clusters FS2, FS4B, and FS4C, respectively, in the [FeFe]-hydrogenase (28). In Nqo3, binuclear cluster N1b is coordinated in a [2Fe-2S] ferredoxin-like fold (28) by cysteines 34, 45, 48, and 83. Tetranuclear cluster N4 is coordinated in a 2×[4Fe-4S] ferredoxin-like fold (28) by cysteines 181, 184, 187, and 230 (fig. S2C). In [FeFe]-hydrogenases, this fold contains an additional tetranuclear cluster FS4A, which is absent from complex I, although the fold is conserved.

The tetranuclear cluster N5 is coordinated by Cys¹¹⁹, Cys¹²², Cys¹²⁸, and His¹¹⁵ from a loop between two α helices (Fig. 3B). Similar to the [FeFe]-hydrogenase, the Nε atom of His¹¹⁵ is a ligand for iron from cluster N5, in contrast to other known examples where Nδ is the coordinating ligand (28). It has been discussed that the histidine ligand may tune the midpoint potential (28). In complex I, cluster N5 belongs to the equipotential group of clusters, but its EPR properties are unusual, as it exhibits very fast spin relaxation and exists in a mixed high-spin ground state (29). Cluster FS0 in nitrate reductase A is also ligated by three cysteines and one histidine (26) and has a fast spin relaxation and an unusual high-spin ground state (30). Thus, it is likely that the unusual properties of cluster N5 are due to its histidine ligation. Several polar residues are within hydrogen-bonding distances to the cluster [Fig. 3B and (17)]. Such an environment in the protein interior may indicate that cluster N5 is not simply a “stepping stone” in the electron transfer pathway, but that it could have an additional role in the mechanism of complex I.

Contrary to predictions (25), the C-terminal domain of Nqo3 (residues 241 to 767) does not have any structural similarity to the C-terminal active-site domain of [FeFe]-hydrogenases. However, it is clearly similar to a family of molybdenum (molybdopterin guanine dinucleotide, MGD) cofactor-containing enzymes.

---

**Fig. 2.** The folds of individual subunits. Fe-S centers are shown as red spheres for Fe atoms and yellow spheres for S atoms, with cluster names in red. Subunits are not drawn to the same scale. (A) Nqo1. Its N-terminal domain is in purple, a Rossmann-fold domain in blue, an ubiquitin-like domain in green, and the C-terminal helical bundle, coordinating cluster N3, in red. FMN is shown in stick representation. (B) Nqo2. The N-terminal helical bundle is shown in blue, the thioredoxin-like domain coordinating cluster N1a in green. (C) Nqo3. The N-terminal [FeFe]-hydrogenase-like domain coordinating clusters N1b, N4, and N5 is magenta, subdomains of the C-terminal molybdoenzyme-like domain are shown in I (coordinating cluster N7), blue; II, green; III, yellow; and IV, red. (D) Nqo9, coordinating clusters N6a and N6b, is shown in rainbow representation, colored blue to red from N to C terminus. (E) Nqo6, coordinating cluster N2, is shown in rainbow representation, with helix H1 indicated. (F) Nqo4. The N-terminal αβ domain is shown in blue, the α-helical bundle in green, the extended helix H2 in yellow, and the C-terminal αβ domain in orange. Clusters are shown for orientation only. (G) Nqo5. The N-terminal α/β domain interacting with Nqo4 is shown in blue, the domain interacting with Nqo9 in green, and the C-terminal loop interacting with Nqo3 in yellow. Clusters are shown for orientation only. (H) Nqo15, shown in rainbow representation. The histidines exposed inside the putative iron storage cavity are shown.

For example, residues 241 to 767 of Nqo3 superimpose with nitrate reductase [PDB 2nap (31), RMSD = 2.7 Å and 18% sequence identity over 428 Cα atoms]. The alignments include the position and Cys ligands of cluster N7, matching the analogous [4Fe-4S] cluster in molybdopterin-containing enzymes. Weak sequence homology with molybdopterin-containing enzymes in this region was noted previously (32).

The C-terminal domain of Nqo3 can be separated into four subdomains similar to those of nitrate reductase (31): They are domains I (residues 241 to 308), II (309 to 373 and 540 to 670), III (374 to 539), and IV (671 to 767). Domain I coordinates cluster N7, in a mostly hydrophobic environment. The related domains II and III each resemble a Rossman fold and consist mostly of a parallel five-stranded β sheet flanked by helices on each side. These two domains face each other, with the edges of β sheets creating a cavity in the middle. In nitrate reductase, this cavity is occupied by the MGD cofactor. In Nqo3, the cavity is capped

by domain IV, which is connected to domain II by a long, isolated β strand-type segment (residues 673 to 691), running over the surface of the protein. Domain IV has a mostly irregular structure, with one antiparallel β sheet. Its analog in nitrate reductase also acts as a lid for the MGD cavity. However, complex I does not contain a molybdenum cofactor. The NAD+-reducing formate dehydrogenase complex from *Ralstonia eutropha* includes an MGD-containing analog of the full-length Nqo3 (subunit FdsA), as well as analogs of Nqo1 and Nqo2 (33). This and similar enzymes could share a common ancestor with complex I, and it appears that the C-terminal part of Nqo3 evolved from an MGD-containing ancestor, after losing the molybdenum cofactor. However, cluster N7, which is too far away from the main redox chain to participate in electron transfer, has been retained in *T. thermophilus* and some other bacteria, probably because it confers structural stability on the fold in domain I. Sequence similarities indicate that the fold of the Nqo3 subunit is conserved in other species,

although most prokaryotes and all eukaryotes have lost cluster N7.

The functional role of the C-terminal domain of Nqo3 is unclear. Upon reduction with NADH, a large conformational change in *E. coli* complex I (and specifically in Nqo3) to an “open,” expanded conformation was observed (34). It is conceivable that, in an open conformation, the lid formed by domain IV moves, which provides access to the cavity between domains II and III. The B-factors show that the C-terminal part of Nqo3 is one of the most flexible areas of the complex. In the structure described here (presumably the oxidized “closed” conformation), there is a small cavity in the protein surface near His591, corresponding to the edge of the MGD cavity in nitrate reductase. No effectors are known to bind in this area. The Nqo3 cavity is too far from the main redox chain for electron transfer, and there is no obvious second NAD(H)-binding site elsewhere in the structure. Therefore, the suggestion that complex I has two different catalytic binding sites for NAD(H), with the second site operating during reverse electron flow (35), is inconsistent with the structure. Observed kinetic differences (35) may be due to conformational changes in complex I. However, because the former MGD-binding site is present, it is conceivable that NAD(H) or another effector binds to Nqo3 in a regulatory role, especially in an “open” conformation.

Subunit Nqo9 (TYKY). This subunit coordinates the tetranuclear clusters N6a and N6b, which follow cluster N5 in the redox chain. As predicted from sequence comparisons (1), the fold (Fig. 2D) is similar to that of the 2× [4Fe-4S] ferredoxin family (17). The clusters are linked by short α helices, so that cluster N6a is coordinated by cysteines 53, 56, 59, and 108, and cluster N6b is coordinated by cysteines 63, 98, 101, and 104 (fig. S2D), with several polar residues nearby (17). We propose that this naming convention should be used for all N6a/b clusters in complex I from other species. Residues at the N and C termini of Nqo9 (not in the ferredoxin fold) are involved in contacts with other subunits, mainly Nqo6. Overall, the role of this subunit appears to provide a “connecting chain” of two clusters between cluster N5 and the terminal cluster N2, and to stabilize the structure of the complex by interacting with other subunits (Fig. 1 and table S2).

Subunits Nqo4 (49 kD) and Nqo6 (PSST). As expected from their similarity to subunits of [NiFe]-hydrogenases, these subunits interact extensively; hence, they are discussed together. Subunit Nqo6 superimposes with the N-terminal part (coordinating the proximal cluster) of the small subunit of the [NiFe]-hydrogenase from *Desulfovibrio gigas* [PDB 2frv (36), RMSD = 2.8 Å and 24% sequence identity over 102 Cα atoms]. The fold consists of a parallel β sheet flanked by α helices, resembling a flavodoxin (36). Notably, the extended N-terminal helix

![Figure 3](#fig3)

**Fig. 3.** The environments of the FMN-binding site and of selected Fe-S clusters. (A) The binding site for FMN and NADH, viewed from the solvent-exposed side. Residues involved in FMN binding are shown in stick representation with carbon in yellow and hydrogen bonds as dotted lines. Residues likely to be involved in NADH binding are shown in stick representation, with carbon in magenta. Prefixes to residue names indicate the subunit number. Cluster N3 is visible to the left. Subunits are colored as in Fig. 1. A σA-weighted \(2F_{\text{obs}} - F_{\text{calc}}\) map contoured at 1σ is shown around the FMN. (B) Cluster N5 and (C) cluster N2. Cluster ligands and polar residues nearby are shown. The backbones of subunits are colored as in Fig. 1. Electron density is from a σA-weighted \(2F_{\text{obs}} - F_{\text{calc}}\) map contoured at 1σ. Clusters are shown as spheres of 0.3 van der Waals radius.

RESEARCH ARTICLE

H1 found in Nqo6 (Figs. 1 and 2E) is absent from the hydrogenase. The fold at the interface of Nqo6 and Nqo4 and the relative positions of these subunits are similar to the arrangement of small and large subunits in the hydrogenase. Similar to the proximal cluster in the hydrogenase, cluster N2 is coordinated only within Nqo6, but is at the interface with Nqo4. One of the four cysteine ligands of the hydrogenase proximal cluster is not conserved in complex I, which has instead two invariant adjacent cysteines, $_{6}$Cys$^{45}$ and $_{6}$Cys$^{46}$ (the prefix indicates subunit number). As there is no previous example in the PDB of an Fe-S cluster coordinated by two consecutive cysteines, it has been proposed that the fourth ligand may be provided by Nqo4, although mutagenesis indicated otherwise (37). Cluster N2 is coordinated by $_{6}$Cys$^{45}$ and $_{6}$Cys$^{46}$, which are in the first turn of a helix, connected through a β strand to helix H1. The coordination is completed by $_{6}$Cys$^{111}$ and $_{6}$Cys$^{140}$ (Fig. 3C). The Chi-1 angles of $_{6}$Cys$^{45}$ and $_{6}$Cys$^{46}$ deviate from optimal values, and the experimental electron density connecting these ligands with cluster N2 is noticeably lower than for other cysteines, which may reflect a suboptimal geometry and associated higher mobility.

Cluster N2 has also probably the most polar environment of all complex I clusters. It is hydrogen-bonded with the conserved residues $_{4}$Arg$^{84}$ and $_{4}$His$^{169}$ (similar to Arg$^{63}$ and His$^{219}$ in the hydrogenase) and is about 4 Å from $_{6}$Arg$^{83}$. Five backbone amides are also within NH-S hydrogen bond distance. It has been suggested that an increase in the number of such bonds and interactions with backbone amide dipoles (27), as well as interactions with polar head groups (26), leads to higher redox potentials. Most other clusters in the structure are involved in two to four interactions with backbone amides. The combined effect of interactions with the backbone and with polar head groups could account for the high redox potential of N2 relative to other clusters. Mutations of $_{4}$Arg$^{84}$ and $_{4}$His$^{169}$ analogs led to an apparent loss of cluster N2 with retention of activity (38). In view of the structure, it is more likely that the cluster was retained, but with the redox potential lowered below the detection level. N2 is only 8 Å from the membrane domain interface, but is, as are all the clusters, shielded from the solvent.

Subunit Nqo4 superimposes with the large subunit of the [NiFe]-hydrogenase [PDB 2frv (36), RMSD = 2.2 Å and 18% sequence identity over 280 C$_{\alpha}$ atoms]. The alignment includes N- and C-terminal αβ-type domains (residues 35 to 106 and 334 to 409, respectively) and a very long four-helix bundle (residues 107 to 320, Fig. 2F). A few short helices, a β sheet, and less regular segments in the hydrogenase are absent from Nqo4. Some of these areas are occupied by subunit Nqo5, without structural similarity to the hydrogenase. The location of a Ni-binding site in the hydrogenase is conserved in complex I, adjacent to cluster N2 and partly exposed toward an elongated cavity at the interface with the membrane domain, in the cleft between subunits Nqo4 and Nqo6 (Fig. 4, A and B). Judging from this location, this cavity is probably the binding site for a quinone, consistent with the observation that a semiquinone species forms within 12 Å of cluster N2 (39). Mutations of analogs of the conserved residues $_{4}$Asp$^{401}$ and $_{4}$Val$^{403}$, which face the cavity, lead to resistance to quinone-like inhibitors (38, 40). The hydroxyl of invariant $_{4}$Tyr$^{87}$ also faces the cavity, and is 8 Å from cluster N2 and 6 Å from the sulfur atom of its ligand $_{6}$Cys$^{45}$ (Fig. 3C). Mutation of this tyrosine in *Yarrowia lipolytica* completely abolished the activity (38). Therefore, the quinone head group is likely to interact with these residues and, possibly, also with exposed $_{4}$Arg$^{42}$ and $_{4}$Arg$^{350}$. Most of the other residues lining the cavity are hydrophobic, which allows it to accommodate part of the isoprenoid tail, although most of the tail is probably in the membrane domain. Thus, the structure confirms suggestions that the quinone-binding site evolved from the Ni-containing active center of the hydrogenase (38).

The interface with the membrane domain is mostly hydrophobic near helix H1 of Nqo6, but comprises a mixture of hydrophobic and acidic residues in the area of subunit Nqo4 (Fig. 4B). This suggests that binding of the peripheral arm of complex I to the membrane domain may involve both hydrophobic and polar interactions.

Notably, helix H2 (residues 321 to 333), which is absent from the hydrogenase, is extended far outside the main body of Nqo4, approaching clusters N5 and N6a (Fig. 2F). The sequence in this region is highly conserved, including the invariant residues $_{4}$Glu$^{322}$ and $_{4}$His$^{327}$, which interact (17) with clusters N5 and N6a, respectively (Fig. 3B and fig. S2D). Mutation of $_{4}$His$^{327}$ led to a 50% loss of activity, indicating an important functional role for this unusually arranged helix (38).

**Subunit Nqo5 (30 kD)**. The N-terminal domain of this subunit (residues 1 to 153) wraps around Nqo4 on one side, and the subunit interacts also with Nqo9 via a two-stranded β sheet (residues 154 to 171) and with Nqo3 via an extended C-terminal loop (residues 172 to 196) (Figs. 1A and 2G; table S2). The main fold consists of a five-stranded mixed β sheet (interacting with a β sheet in Nqo4), flanked on the outside by α helices. One possible role for Nqo5 appears to be in the stabilization of the complex. Mutation in *Yarrowia* of the invariant $_{4}$Arg$^{409}$, which in *Thermus* is involved in a salt bridge with the invariant $_{5}$Asp$^{120}$, led to destabilization of the enzyme (41).

---

**Fig. 4. Possible quinone- and iron-binding sites.** The solvent-accessible surface area is shown, calculated with the APBS plug-in in PyMOL using a sphere of 1.4 Å radius as a probe. It is colored red for negative, white for neutral, and blue for positive surface charges. (A) Close-up of the quinone-binding site, viewed from the membrane space up toward the peripheral arm. Residues likely to interact with the quinone are shown. Prefixes before residue names indicate the subunit number. Subunits are colored as in Fig. 1. (B) Overview of the interface with the membrane domain in surface representation. The orientation is similar to that in (A). The quinone-binding site is indicated by an arrow and Q. Helix H1 and subunit names are indicated for orientation. (C) A possible iron-binding and/or iron storage site. Histidines and other polar residues lining the cavity are shown.

Subunit Nqo15. This 129-residue protein [National Center for Biotechnology Information (NCBI) GenInfo (GI) identifier 55771878] is the first subunit of bacterial complex I identified in addition to the 14 conserved subunits (16). Nqo15 is bound to the side of the complex near the N terminus of Nqo3, where it interacts with subunits Nqo3, Nqo2, Nqo1, Nqo9, and Nqo4 (Fig. 1A and table S2). Without Nqo15, this region would be rather narrow compared with the rest of the structure. Thus, one of its probable functions is to stabilize the complex. Unexpectedly, the fold of Nqo15 was found to be similar to the unique fold of the frataxin family (Fig. 2H). Despite the low sequence similarity, Nqo15 superimposes very well with the *E. coli* frataxin homolog CyaY [PDB 1ew4 (42), RMSD = 2.5 Å, 13% sequence identity over 88 $C_{\alpha}$ atoms] and with human frataxin [PDB 1ekg (43), RMSD = 3.3 Å, 11% sequence identity over 96 $C_{\alpha}$ atoms]. The fold consists of a large, twisted, six-stranded antiparallel $\beta$ sheet, flanked on one side by N- and C-terminal $\alpha$ helices.

Mitochondrial matrix frataxin and its homologs in bacteria are thought to be iron chaperones, participating in the storage of iron and in the maturation of Fe-S and heme-containing proteins (43, 44). Frataxin deficiency in humans leads to Friedreich’s ataxia, a severe neurodegenerative disorder. Frataxin serves as an iron donor to aconitase, converting an inactive 3Fe-4S cluster to an active 4Fe-4S cluster (44).

Nqo15 interacts with complex I mainly through the exposed face of its $\beta$ sheet. Between this $\beta$ sheet and the rest of the complex there is a hydrophilic channel [with an elliptical cross section of about 7 Å by 15 Å, previously named channel 2 (15)] leading from the solvent to within about 10 Å of cluster N3, at its deep end, and cluster N1a, at its side. Remarkably, this cavity is lined with six exposed histidine residues, four of them provided by Nqo15 [histidines 90, 92, 94, and 96 (Fig. 2H)], as well as $_{1}$His${}^{350}$ and $_{3}$His${}^{208}$ (Fig. 4C). At the deep end of the channel, in an area with negative surface charge, there is a possible binding pocket for iron (as suggested by similarity to frataxin), formed by ${}_{15}$His${}^{92}$, ${}_{3}$His${}^{208}$, and ${}_{3}$Glu${}^{204}$. Additionally, ${}_{1}$His${}^{350}$ and ${}_{15}$His${}^{90}$ would only require a side-chain rotation to contribute to metal binding. Remaining Nqo15 histidines 94 and 96 are in a good position to serve as anchors that guide iron into the binding site. Thus, another possible role for Nqo15 is the storage of iron. It could be used for the reconstitution of the nearby Fe-S clusters N3 and N1a, if iron is lost from them. No electron density for bound metal was observed in the cavity, possibly because Fe is lost during purification. Binding of frataxin to its target proteins is mediated by iron (45), consistent with our proposal for an iron-binding site at the interface between Nqo15 and the rest of complex I. Also, frataxin is suggested to bind to its partners via the conserved exposed face of the

$\beta$ sheet (42), similar to Nqo15. Therefore, Nqo15 may represent the first example of a frataxin-like protein permanently bound to its target.

No other frataxin analogs were found in *T. thermophilus*, and analogs of Nqo15 were found only in close relatives, *Deinococcus geothermalis* and *Deinococcus radiodurans*. The permanent attachment of an Nqo15-like protein to complex I may be unique to thermophiles and may be a consequence of their harsh environment.

### Functional Implications

One of the most puzzling questions about complex I is why it contains so many Fe-S clusters. Evolution of the complex from many smaller building blocks is probably part of the answer, as the structure shows. One of the clusters, N7, is not conserved and is likely to be an evolutionary remnant. However, the number of conserved clusters retained appears to be in excess of the minimum number needed just to connect the NADH- and quinone-binding sites. Electron transfer could, in principle, proceed between clusters N3 and N4 (edge-to-edge distance 13.8 Å), without N1b participation. Additional reasons for complexity may lie in the mechanism of the enzyme.

Complex I flavin, after reduction by NADH to FMNH$_{2}$, releases two electrons in two steps with one-electron redox potentials (at pH 7.0) of about –300 mV (FMNH$_{2}$/flavosemiquinone) and of about –390 mV (flavosemiquinone/oxidized flavin) (46). Electrons can be transferred effectively to the nearest cluster N3 (potential –250 mV) and along the cluster chain to membrane-embedded quinone (Fig. 1B). Under steady-state NADH oxidation, the Fe-S clusters in complex I are almost fully reduced, and the rate-limiting step is likely the oxidation of N2 by quinone (47). When cluster N3 is reduced, electron transfer from flavosemiquinone (potential –390 mV) to cluster N1a (potential –370 mV) is thermodynamically favorable. As soon as N3 is reoxidized during quinone reduction, electrons from N1a can proceed via FMN into the main cluster chain. A pair of electrons from FMN may thus be donated nearly simultaneously to clusters N3 and N1a, and the flavosemiquinone radical may be short-lived. Cluster N1a is shielded from the solvent (fig. S2B), whereas FMN is exposed (Fig. 3A), and flavosemiquinone, as well as FMNH$_{2}$, can generate ROS by reduction of oxygen (48). Therefore, such temporary storage of electrons by N1a is likely to minimize ROS production during turnover of complex I. Apart from Fe-S cluster ligands, ${}_{1}$Cys${}^{182}$ is the only other conserved cysteine in complex I, and it has been identified as the site of the oxidative attack and as a redox-sensitive thiol (49). It is only 6 Å from the FMN, consistent with flavin serving as a source of ROS.

Considering the coupling between electron transfer and proton translocation, two main modes of coupling have been proposed: direct (redox-driven, e.g., Q-cycle variations) or indirect (through conformational changes) (1, 2, 34, 50, 51). Some features of the structure support direct coupling. There is a clear electron transfer pathway from FMN to cluster N2. Polar residues, found in the vicinity of nearly all clusters, may adjust redox potentials for optimal rates. The unique coordination of cluster N2 by two consecutive cysteines is conserved in complex I and is likely to be important for the mechanism. The midpoint potential of N2 is pH-dependent, indicating that its reduction is coupled to proton binding (2, 9). This may be due to the polar environment of the cluster, in particular ${}_{4}$His${}^{169}$. However, the unusual coordination of the cluster may allow its direct protonation, coupled to reduction. In this respect, N2 may resemble [3Fe-4S] clusters or redox-active [4Fe-4S] clusters, coordinated by only three cysteines, which can be directly protonated (52). The invariant ${}_{4}$Tyr${}^{87}$ can interact with one of the two consecutive N2 ligands, ${}_{6}$Cys${}^{45}$, as well as with the quinone (Figs. 3C and 4A). This arrangement could facilitate electron transfer and possibly proton transfer or coupling. As most elements would be needed for any NADH to quinone pathway, these features do not support the direct mechanism exclusively.

However, some elements of the structure can be interpreted as pointing to long-range conformational interactions. Notably, the conserved helix H2 extends away from Nqo4 to clusters N5 and N6a, and it interacts with them through polar residues (Fig. 2F and fig. S2D). One role of such an unusual arrangement could be to provide a way of communicating the redox state of these clusters to Nqo4 and further on to the membrane domain, possibly through conformational changes. In addition, the distance between clusters N5 and N6a is the longest in the redox chain (Fig. 1B), which may reflect a transfer-controlling step. Cluster N2 is linked through one $\beta$ strand to the rigid helix H1, which is likely to extend on the membrane domain surface toward antiporter-like subunits Nqo12 to 14 (11). This helix could also provide a connection between the clusters and the proton-pumping machinery, possibly as a kind of “lever.” As the conformation of Nqo3 is changed when enzyme reduction takes place (34), its large C-terminal domain could participate in a conformational mechanism. The very long redox chain of complex I may help to coordinate large-scale conformational changes (34, 51). As four protons are moved across the membrane in each catalytic cycle, a combination of direct and indirect mechanisms could operate in complex I.

---

**References and Notes**

1. J. E. Walker, *Q. Rev. Biophys.* **25**, 253 (1992).
2. T. Yagi, A. Matsuno-Yagi, *Biochemistry* **42**, 2266 (2003).
3. J. Carroll, I. M. Fearnley, R. J. Shannon, J. Hirst, J. E. Walker, *Mol. Cell. Proteomics* **2**, 117 (2003).

4. N. Grigorieff, J. Mol. Biol. 277, 1033 (1998).
5. G. Peng et al., Biochemistry 42, 3032 (2003).
6. V. Guenebaut, A. Schlitt, H. Weiss, K. Leonard, T. Friedrich, J. Mol. Biol. 276, 105 (1998).
7. A. H. Schapira, Biochim. Biophys. Acta 1364, 261 (1998).
8. R. S. Balaban, S. Nemoto, T. Finkel, Cell 120, 483 (2005).
9. T. Ohnishi, Biochim. Biophys. Acta 1364, 186 (1998).
10. M. Finel, J. M. Skehel, S. P. Albracht, I. M. Fearnley, J. E. Walker, Biochemistry 31, 11425 (1992).
11. P. J. Holt, D. J. Morgan, L. A. Sazanov, J. Biol. Chem. 278, 43114 (2003).
12. T. Friedrich, D. Scheide, FEBS Lett. 479, 1 (2000).
13. H. Leif, V. D. Sled, T. Ohnishi, H. Weiss, T. Friedrich, Eur. J. Biochem. 230, 538 (1995).
14. E. Nakamaru-Ogiso, T. Yano, T. Yagi, T. Ohnishi, J. Biol. Chem. 280, 301 (2005).
15. P. Hinchliffe, L. A. Sazanov, Science 309, 771 (2005).
16. P. Hinchliffe, J. Carroll, L. A. Sazanov, in preparation.
17. Materials and methods are available as supporting material on Science Online.
18. C. C. Page, C. C. Moser, X. Chen, P. L. Dutton, Nature 402, 47 (1999).
19. D. J. Morgan, L. A. Sazanov, unpublished data.
20. E. Krissinel, K. Henrick, Acta Crystallogr. D Biol. Crystallogr. 60, 2256 (2004).
21. L. Holm, C. Sander, Science 273, 595 (1996).
22. A. M. Lesk, Curr. Opin. Struct. Biol. 5, 775 (1995).
23. A. G. Murzin, S. E. Brenner, T. Hubbard, C. Chothia, J. Mol. Biol. 247, 536 (1995).
24. A. P. Yeh et al., J. Mol. Biol. 300, 587 (2000).

25. P. M. Vignais, B. Billoud, J. Meyer, FEMS Microbiol. Rev. 25, 455 (2001).
26. M. G. Bertero et al., Nat. Struct. Biol. 10, 681 (2003).
27. G. Fritz et al., Proc. Natl. Acad. Sci. U.S.A. 99, 1836 (2002).
28. J. W. Peters, W. N. Lanzilotta, B. J. Lemon, L. C. Seefeldt, Science 282, 1853 (1998).
29. T. Yano et al., J. Biol. Chem. 278, 15514 (2003).
30. R. A. Rothery et al., Biochemistry 43, 5324 (2004).
31. J. M. Dias et al., Struct. Fold. Design 7, 65 (1999).
32. M. Finel, Biochim. Biophys. Acta 1364, 112 (1998).
33. J. I. Oh, B. Bowien, J. Biol. Chem. 273, 26349 (1998).
34. A. A. Mamedova, P. J. Holt, J. Carroll, L. A. Sazanov, J. Biol. Chem. 279, 23830 (2004).
35. V. G. Grivennikova, R. Roth, N. V. Zakharova, C. Hagerhall, A. D. Vinogradov, Biochim. Biophys. Acta 1607, 79 (2003).
36. A. Volbeda et al., Nature 373, 580 (1995).
37. D. Flemming, A. Schlitt, V. Spehr, T. Bischof, T. Friedrich, J. Biol. Chem. 278, 47602 (2003).
38. N. Kashani-Poor, K. Zwicker, S. Kerscher, U. Brandt, J. Biol. Chem. 276, 24082 (2001).
39. T. Yano, W. R. Dunham, T. Ohnishi, Biochemistry 44, 1744 (2005).
40. E. Darrouzet, J. P. Issartel, J. Lunardi, A. Dupuis, FEBS Lett. 431, 34 (1998).
41. L. Grgic, K. Zwicker, N. Kashani-Poor, S. Kerscher, U. Brandt, J. Biol. Chem. 279, 21193 (2004).
42. S. J. Cho et al., Proc. Natl. Acad. Sci. U.S.A. 97, 8932 (2000).
43. S. Dhe-Paganon, R. Shigeta, Y. I. Chi, M. Ristow, S. E. Shoelson, J. Biol. Chem. 275, 30753 (2000).
44. A.-L. Bulteau et al., Science 305, 242 (2004).
45. T. Yoon, J. A. Cowan, J. Biol. Chem. 279, 25943 (2004).

46. V. D. Sled, N. I. Rudnitzky, Y. Hatefi, T. Ohnishi, Biochemistry 33, 10069 (1994).
47. A. B. Kotlyar, V. D. Sled, D. S. Burbaev, I. A. Moroz, A. D. Vinogradov, FEBS Lett. 264, 17 (1990).
48. A. P. Kudin, N. Y. Bimpong-Buta, S. Vielhaber, C. E. Elger, W. S. Kunz, J. Biol. Chem. 279, 4127 (2004).
49. Y. R. Chen, C. L. Chen, L. Zhang, K. B. Green-Church, J. L. Zweier, J. Biol. Chem. 280, 37339 (2005).
50. T. Friedrich, J. Bioenerg. Biomembr. 33, 169 (2001).
51. G. Belogrudov, Y. Hatefi, Biochemistry 33, 4571 (1994).
52. J. L. C. Duff, J. L. J. Breton, J. N. Butt, F. A. Armstrong, A. J. Thomson, J. Am. Chem. Soc. 118, 8593 (1996).
53. This work was funded by the Medical Research Council. We thank J. E. Walker, A. G. W. Leslie, and A. G. Murzin for helpful discussions of the manuscript; the European Synchrotron Radiation Facility for provision of synchrotron radiation facilities; and the staff of beamlines ID23 and ID29 for assistance. The coordinates and the structure factors have been deposited in the Protein Data Bank (accession code 2FUG).

**Supporting Online Material**

www.sciencemag.org/cgi/content/full/1123809/DC1

Materials and Methods  
SOM Text  
Figs. S1 and S2  
Tables S1 and S2  
References  

13 December 2005; accepted 30 January 2006  
Published online 9 February 2006;  
10.1126/science.1123809  

Include this information when citing this paper.

---

**REPORTS**

### Ultrafast Interfacial Proton-Coupled Electron Transfer

Bin Li,$^{1}$ Jin Zhao,$^{1}$ Ken Onda,$^{1}$ Kenneth D. Jordan,$^{2}$ Jinlong Yang,$^{3}$ Hrvoje Petek$^{1*}$

The coupling of electron and nuclear motions in ultrafast charge transfer at molecule-semiconductor interfaces is central to many phenomena, including catalysis, photocatalysis, and molecular electronics. By using femtosecond laser excitation, we transferred electrons from a rutile titanium dioxide (110) surface into a CH₃OH overlayer state that is 2.3 ± 0.2 electron volts above the Fermi level. The redistributed charge was stabilized within 30 femtoseconds by the inertial motion of substrate ions (polaron formation) and, more slowly, by adsorbate molecules (solvation). According to a pronounced deuterium isotope effect (CH₃OD), this motion of heavy atoms transforms the reverse charge transfer from a purely electronic process (nonadiabatic) to a correlated response of electrons and protons.

T **itanium dioxide (TiO₂)** is a large-band gap semiconductor with important applications in photocatalysis and solar energy conversion (1–5). Electron-hole pair excitation within semiconductors such as TiO₂ can initiate photocatalytic reactions in chemisorbed molecular overlayers through interfacial charge transfer. In response to charge injection into a molecular overlayer, the interfacial lattice ions and molecules will undergo inertial motion to minimize the free energy (dielectric screening) or even chemical reactions, which are in competition with the charge population decay by reverse transfer into the semiconductor. Of particular interest as a source of abundant clean energy and for environmental remediation is the activation of bonds that contain hydrogen atoms through the coupling of electrons and protons, which can catalyze disproportionation of H₂O into H₂ and O₂ (1–3).

Excess electrons in molecular overlayers on metal oxide surfaces are strongly attracted to the dangling hydrogen atoms of surface-bound -OH groups and on coadsorbed protic solvent molecules that are not engaged in strong hydrogen bonds (6, 7). The injection of electrons into the molecular overlayer can even release hydrogen atoms from the existing hydrogen bonds to create new dangling hydrogen atom centers. Such correlated dynamics of electrons and protons are fundamentally important in chemistry and biology. They can drive photocatalytic reactions as well as modulate charge transport at the semiconductor/protic solvent interfaces, and they are known to play a role in photosynthesis, respiration, and transport through biological membranes (8–13).

In a recent time-resolved two-photon photoemission (TR-2PP) study of H₂O/TiO₂(110) surfaces, we discovered an excited electronic state 2.4 ± 0.1 eV above the Fermi level $E_{\mathrm{F}}$. This state attains the maximum intensity when -OH species, which are formed by the dissociation of H₂O at defect sites on TiO₂(110), coexist with chemisorbed H₂O at ~1-monolayer (ML) coverage. On the basis of chemical evidence and density functional theory (DFT) calculations, we assigned this resonance to partially hydrated, or “wet,” electron states. Electrons injected into the molecular overlayer are delocalized over several hydrogen atoms associated with -OH groups and neighboring water molecules. The injected electrons decay into the conduction band of TiO₂ through reverse charge transfer in less than 15 fs, faster than the dielectric response of the interface can stabilize them (6, 7).

---

1Department of Physics and Astronomy, 2Department of Chemistry, University of Pittsburgh, Pittsburgh, PA 15260, USA. 3Hefei National Laboratory for Physical Sciences at Microscale, University of Science and Technology of China, Hefei, Anhui 230026, China.

*To whom correspondence should be addressed. E-mail: petek@pitt.edu

1436 10 MARCH 2006 VOL 311 SCIENCE www.sciencemag.org
