
Microbiology (2006), 152, 667–673

**DOI 10.1099/mic.0.28680-0**

# In vivo characterization of the dTDP-D-desosamine pathway of the megalomicin gene cluster from *Micromonospora megalomicea*

Eduardo Rodriguez,$^{1,2}$ Salvador Peirú,$^{1}$ John R. Carney$^{2}$ and Hugo Gramajo$^{1,2}$

**Correspondence**  
Hugo Gramajo  
gramajo@ibr.gov.ar  

$^{1}$ Microbiology Division, IBR (Instituto de Biología Molecular y Celular de Rosario), Consejo Nacional de Investigaciones Científicas y Técnicas, Facultad de Ciencias, Bioquímicas y Farmacéuticas, Universidad Nacional de Rosario, Suipacha 531, (S2002LRK) Rosario, Argentina  

$^{2}$ Kosan Biosciences, Inc., 3832 Bay Center Place, Hayward, CA 94545, USA  

Received 9 November 2005  
Revised 24 November 2005  
Accepted 20 December 2005  

---

*In vivo* reconstitution of the dTDP-D-desosamine pathway of the megalomicin gene cluster from *Micromonospora megalomicea* was achieved by expression of the genes in *Escherichia coli*. LC/MS/MS analysis of the dTDP-sugar intermediates produced by operons containing different sets of genes showed that production of dTDP-D-desosamine from dTDP-4-keto-6-deoxy-D-glucose requires only four biosynthetic steps, catalysed by MegCIV, MegCV, MegDII and MegDIII, and that MegCII is not involved. Instead, bioconversion studies demonstrated that MegCII is needed together with MegCIII to catalyse transfer of D-desosamine to 3-*α*-mycarosylerythronolide B.

---

## INTRODUCTION

The megalomicin polyketides, originally discovered as antibacterial agents, are produced by the actinomycete *Micromonospora megalomicea* (Weinstein *et al.*, 1969). Like the erythromycins (Waitz *et al.*, 1969), they inhibit protein synthesis through selective binding to the bacterial 50S rRNA, but with weaker activity. Therapeutic interest in the megalomicins has recently increased after it was found that they interfere with protein trafficking through inhibition of vesicular transport between the medial and trans Golgi, resulting in defectively sialylated proteins (Bonay *et al.*, 1996, 1997). This anomalous glycosylation may be the basis of the antiviral activity of megalomicins against enveloped viruses such as herpes simplex virus (HSV), Semliki Forest virus, vesicular stomatitis virus and, more importantly, the human immunodeficiency virus type 1 (HIV-1) (San Jose *et al.*, 1997; Alarcon *et al.*, 1988). Anti-HIV activity seems to depend on inhibition of the processing of gp160 to gp120 and gp41, resulting in noninfectious virions (San Jose *et al.*, 1997). Notably, megalomicins also showed antiparasitic activity against the epimastigote stage of *Trypanosoma cruzi*, *Leishmania* spp. and *Plasmodium falciparum* (Bonay *et al.*, 1998), although the mechanism of action is still unknown. All these interesting activities open possibilities for the development of new megalomicin-based drugs for treatment of these important human diseases.

Megalomicins consist of a 14-membered macrolactone ring carrying three deoxysugar residues, L-mycarose,

---

D-desosamine and L-megosamine, with acetyl or propionyl groups at the 3" or 4" hydroxyls of the mycarose moiety. The high similarity of megalomicin to erythromycin at the structural (Fig. 1) and biosynthetic levels is reflected in the overall organization of their gene clusters (Table 1) (Volchegursky *et al.*, 2000). The biosynthetic pathway of erythromycin has been studied extensively over the last 15 years, but the enzymic activity for each of the proteins involved in its biosynthesis, except for the polyketide synthase, has been assigned, almost entirely, on the basis of similarities with related enzymes in databases and through analysis of the compounds accumulated by mutants affected in selected deoxysugar genes (Gaisser *et al.*, 1997, 1998; Summers *et al.*, 1997; Salah-Bey *et al.*, 1998). Therefore, the assignment of enzyme activity to the proteins involved in the biosynthetic pathway of megalomicin (Volchegursky *et al.*, 2000; Peirú *et al.*, 2005), based on the erythromycin pathway, needed to be validated.

Desosamine, a 3-dimethylamino-3,4,6-trideoxyhexose, is synthesized from dTDP-activated hexoses via the common intermediate dTDP-4-keto-6-deoxy-D-glucose. In *M. megalomicea*, and based on protein homologies, formation of this intermediate would require activation of D-glucose 1-phosphate into dTDP-D-glucose by MegL, a putative dTDP-D-glucose synthetase (Fig. 2), followed by dehydration at C-4 and C-6 catalysed by MegM, a dTDP-D-glucose 4,6-dehydratase (Volchegursky *et al.*, 2000). Based on early studies of the biosynthesis of dTDP-D-desosamine in different systems, the enzymic activity of several megalomicin proteins has been proposed: MegCII as an isomerase

---

Abbreviation: MEB, 3-*α*-mycarosylerythronolide B.

E. Rodriguez and others

![Structures of megalomicin A and erythromycins C and D](image.png)

Fig. 1. Structures of megalomicin A and erythromycins C and D.

involved in tautomerization of dTDP-4-keto-6-deoxy-D-glucose to dTDP-3-keto-6-deoxy-D-glucose (Fig. 2, route A); MegCIV as a 3,4-dehydratase; MegCV as a 3,4-reductase; MegDII as a 3-aminotransferase; and MegDIII as a 3-N-dimethyltransferase (Fig. 2) (Salah-Bey et al., 1998). Since the last two steps of L-megosamine and D-desosamine biosynthesis are mechanistically similar, MegDII and MegDIII were proposed to be involved in the aminotransferase and dimethyltransferase steps of L-megosamine biosynthesis (Volchegursky et al., 2000).

Once dTDP-D-desosamine is synthesized, the sugar moiety has to be attached to its substrate, in this case 3-α-mycarosylerythronolide B (MEB), by a dedicated glycosyltransferase (Gtf), predicted to be MegCIII. Biochemical characterization of the erythromycin dTDP-D-desosaminyl transferase EryCIII, the homologue of MegCIII, showed that this enzyme is highly specific for its natural substrate MEB (Lee et al., 2004) and does not require any accessory protein for its activity *in vitro*. Interestingly, Hung-wen Liu and colleagues found that DesVII (a homologue to EryCIII and MegCIII, Table 1), the desosaminyltransferase from the methymycin/pikromycin gene cluster of Streptomyces venezuelae, is dependent on the auxiliary protein DesVIII (a homologue to EryCII and MegCII, Table 1) for activity.

(Borisova et al., 1999). Other examples of glycosyltransferase auxiliary proteins involved in macrolide production have been reported (Lu et al., 2005; Melancon et al., 2004).

The establishment of deoxy and amino sugars as vital components for the efficacy and specificity of biologically active glycoconjugates (Weymouth-Wilson, 1997) suggests that altering and/or exchanging these crucial sugar structures can enhance or vary the biological activities of their parent molecules. Such an approach has become an appealing strategy for developing a new generation of therapeutic agents, and requires a detailed knowledge of the biosynthetic role of each individual protein in a sugar biosynthetic pathway of interest. Since a more accurate assignment of the D-desosamine biosynthetic genes from the megalomicin cluster had to be established, in particular for MegCII (as a putative isomerase or desosaminyltransferase auxiliary protein), we studied the D-desosamine biosynthetic pathway from the megalomicin gene cluster *in vivo*.

Herein we report expression of each putative gene of the D-desosamine pathway from *M. megalomicia* in Escherichia coli and analysis by LC/MS/MS of the dTDP-sugar intermediates produced by operons containing different sets of genes.

---

Table 1. Homology of proteins involved in the desosamine pathway from the megalomicin, erythromycin and pikromycin gene clusters

| Meg     | Ery    | Identity (%) | Pik   | Identity (%) | Proposed function         | Reference                                                                 |
|---------|--------|--------------|-------|--------------|---------------------------|--------------------------------------------------------------------------|
| MegCIII | EryCIII| 82·7         | DesVII| 47·6         | Glycosyltransferase       | Salah-Bey et al. (1998); Summers et al. (1997); Xue et al. (1998)      |
| MegCII  | EryCII | 71·3         | DesVIII| 27·3        | Tautomerase/glycosyltransferase | Salah-Bey et al. (1998); Summers et al. (1997); Xue et al. (1998)      |
| MegCIV  | EryCIV | 83·3         | DesI  | 57·7         | C-4 deoxygenase           | Summers et al. (1997); Xue et al. (1998)                              |
| MegCV   | EryCV  | 79·6         | DesII | 60·8         | C-4 deoxygenase           | Gaisser et al. (1997); Xue et al. (1998)                               |
| MegDII  | EryCI  | 41·9         | DesV  | 43           | Aminotransferase          | Weber et al. (1990); Xue et al. (1998)                                 |
| MegDIII | EryCVI | 50·2         | DesVI | 49·8         | Dimethyltransferase       | Summers et al. (1997); Xue et al. (1998)                              |

Microbiology 152

dTDP-desosamine pathway from the megalomicin cluster

![Diagram](attachment:diagram.png)

Fig. 2. The dTDP-D-desosamine biosynthetic pathway. The originally proposed dTDP-D-desosamine pathway involved the intermediate dTDP-3-keto-6-deoxy-D-glucose generated by a putative isomerase MegCII (route A). The new dTDP-desosamine pathway proposes that MegCII works instead with MegCIII as the desosaminyltransferase (route B).

---

### METHODS

#### Bacterial strains and growth conditions
The strains used in this study are shown in Table 2. Luria-Bertani (LB) medium was used to grow the bacteria. The following antibiotics were added to the medium when necessary: kanamycin (50 μg ml⁻¹), chloramphenicol (20 μg ml⁻¹) and streptomycin (25 μg ml⁻¹). Induction experiments were performed as described by Peirú et al. (2005). Briefly, *E. coli* K207-3 harbouring pGro7 and different combinations of plasmids were grown at 37°C in LB with appropriate antibiotics until mid-exponential phase (0·4–0·6 OD₆₀₀ units ml⁻¹). Expression of chaperone and sugar genes was induced by adding 2 mg l-arabinose ml⁻¹ and 0·5 mM IPTG, respectively. Cultures were grown at 22°C for 2 days and cell pellets were resuspended in 20 mM Tris buffer pH 7·5 and disrupted by sonication. The supernatants were analysed by LC/MS/MS.

#### DNA manipulation
Restriction enzymes were used as recommended by the manufacturer (New England Biolabs). Standard protocols were used for recombinant DNA techniques (Sambrook et al., 1989). DNA fragments were purified from agarose gels with the QIAquick Purification Kit (Qiagen). Plasmids were prepared using a QIAPrep Spin Miniprep Kit (Qiagen). Deep Vent DNA polymerase was used in all PCR reactions according to the supplier’s instructions (New England Biolabs).

#### Plasmid constructions
Each gene of the D-desosamine biosynthetic pathway was amplified from *M.* megalomicae genomic DNA and cloned into pET24b or pET28a vectors as described by Peirú et al. (2005). In order to allow various combinations of genes to be expressed in the same strain, different sets of genes were cloned in compatible vectors. For this, *megCIV* was cloned from pLB134 as an XbaI-EcoRI fragment into pKOS431-54-2, a pCDF-1b derivative vector (Peirú et al., 2005) to give pLB316. Then *megCV* was cloned downstream of *megCIV* as described by Peirú et al. (2005) to give pLB318. In parallel, *megDIII* from pLB147 was cloned as a XbaI-NotI fragment downstream of *megDII* into pLB255, a pET-derivative, to give pKOS506-56. In the same way, the *megCIII-ermE* operon was cloned as an XbaI-EcoRI fragment into the pRSF-1b-derived pKOS431-39-1 (Peirú et al., 2005), compatible with both pET- and pCDF-derived vectors. Then two expression vectors were constructed as described by Peirú et al. (2005) containing the sets of genes *megCIV*, *megCV*, *megDII*, *megDIII*, *megCIII* and *ermE* (pKOS506-72B) and *megCIV*, *megCV*, *megDII*, *megDIII*, *megCII*, *megCIII* and *ermE* (pKOS506-72A) in a pCDF1b backbone. The plasmids are shown in Table 2.

#### LC/MS/MS analyses of sugar nucleotides
A system comprising an Applied Biosystems API-3000 triple quadrupole mass spectrometer equipped with a Turbo-ionspray source, Agilent 1100 series HPLC pump and HTC PAL autosampler was used for LC/MS/MS analyses. The mass spectrometer was operated in negative-ion mode with the electrospray needle voltage (−3200 V), source temperature (400°C), declustering potential (−46 V), focusing potential (−290 V) and collision energy (−34 eV) set to the values indicated. Nitrogen was used as the collision gas. For precursor scans, resolutions in Q1 and Q3 were set to unit and low, respectively, while for multiple reaction monitoring (MRM) experiments, low resolution was used in both quadrupoles. A Waters YMC ODS-A column (S-5, 2·0 × 250 mm) maintained at 37°C was used for separations. Mobile phase A was 10 mM triethylamine in H₂O (pH 5·60), and mobile phase B was MeCN. Typically 10 μl samples were injected and eluted with a linear gradient from 2 to 13% B over 10 min, then held at 13% for 2·5 min. The flow rate was 0·200 ml min⁻¹.

E. Rodriguez and others

Table 2. Strains and plasmids

| Strain or plasmid | Relevant genotype | Source or reference |
|-------------------|-------------------|---------------------|
| $E. coli$         |                   |                     |
| DH5$\alpha$       | $lacZ\Delta M15 recA1$ | Promega             |
| K207-3           | $F^- ompT hsdS_B (r^- m^-) gal dcm (DE3), panD::panDS25A \Delta prpRBCD::T7prom-sfp$ | Murli et al. (2003) |
|                  | $T7prom-prpE ygfG::T7prom-accA1-T7prom-pccB$ |                     |
| K506-121         | $F^- ompT hsdS_B (r^- m^-) gal dcm (DE3) panD::panDS25A$ | S. Peiru, unpublished |
|                  | $\Delta prpRBCD::T7prom-sfp T7prom-prpE ygfG::T7prom-accA1-T7prom-pccB \Delta rmlC$ |                     |
| Plasmid          | Description*      |                     |
| pET24b           | $E. coli$ expression vector, ColEl ori, kan | Novagen            |
| pET28a           | $E. coli$ expression vector, ColEl ori, kan | Novagen            |
| pLB99            | $megCII in a pET24b$ backbone, kan | Peiru et al. (2005) |
| pLB255           | $megDII in a pET24b$ backbone, kan | Peiru et al. (2005) |
| pLB318           | $megCIV-megCV in a pCDF1b$ backbone, str | This work          |
| pKOS506-56       | $megDII-megDIII in a pET24b$ backbone, kan | This work          |
| pKOS506-72B      | $megCIV, megCV, megDII, megDIII, ermE in a pCDF1b$ backbone, str | This work          |
| pKOS506-72A      | $megCIV, megCV, megDII, megDIII, megCIII, ermE in a pCDF1b$ backbone, str | This work          |
| pGro7            | $PBAD groES-groEL in a pACYC 184$ backbone, cat | Takara            |

*kan, kanamycin resistance gene; str, streptomycin resistance gene; cat, chloramphenicol acetyltransferase gene.

The column eluate was introduced unsplit into the mass spectrometer source. dTDP-glucose was purchased from Sigma, and dTDP-D-desosamine, prepared by chemical synthesis, was kindly provided by Hung-wen Liu (University of Texas at Austin).

Bioconversion experiments. $E. coli$ K207-3 (Murli et al., 2003) derivatives harbouring pGro7 (Takara) and the different combinations of plasmids were cultured overnight at $37^\circ C$ in LB with appropriate antibiotics, subcultured by 1:50 dilution in the same medium and grown to mid-exponential phase ($0 \cdot 4 - 0 \cdot 6 OD_{600}$ units ml$^{-1}$). Expression of chaperone and sugar genes was induced by adding $2 mg L$-arabinose ml$^{-1}$ and $0 \cdot 5 mM$ IPTG, respectively, and cultures were supplemented with $50 \mu g MEB$ ml$^{-1}$ as needed. Cultures were grown at $22^\circ C$ for 2 days, centrifuged at $14000$ r.p.m. for $5$ min, and the clarified culture broths were analysed by LC/MS.

LC/MS for bioconversion studies. The LC/MS system consisted of an Applied Biosystems Mariner time-of-flight mass spectrometer operated in positive-ion mode and configured with a Turboionspray source, an Agilent 1100 HPLC pump and a Gilson 215 sample handler. A Develosil ODS-UG-5 column ($2 \cdot 0 \times 150 mm$) at $60^\circ C$ was used for the chromatography. Mobile phase A was $5 mM NH_4OAc$ in water, and mobile phase B was $5 mM NH_4OAc$ in $4:1$ (v/v) MeCN/MeOH. Samples ($20 \mu l$) were injected and eluted with a gradient from $35$ to $100\% B$ over $10$ min at $0 \cdot 250 ml$ min$^{-1}$. The eluate was delivered unsplit into the mass spectrometer source.

RESULTS

In vivo characterization of the dTDP-D-desosamine biosynthetic pathway from $M. megalomicea$

We have recently shown production of erythromycin C in $E. coli$ by expressing genes from the ery and meg clusters of Saccharopolyspora erythraea and $M. megalomicea$, respectively (Peiru et al., 2005). Bioconversion of MEB into erythromycin C was achieved by expressing six proteins of

the putative megD-desosamine pathway, MegCII, MegCIV, MegCV, MegDII, MegDIII and MegCIII, in addition to the C-12 hydroxylase MegK and the erythromycin resistance methyltransferase ErmE (Peiru et al., 2005). These results validated the role of MegDII and MegDIII in the biosynthesis of D-desosamine and demonstrated that $E. coli$ BL21 produces the common precursor dTDP-4-keto-6-deoxy-D-glucose which, in this Gram-negative bacterium, is an intermediate for the synthesis of 4-acetamido-4,6-dideoxygalactose, a component of the enterobacterial common antigen (Rahman et al., 2001).

In order to determine if MegCII was involved in tautomerization of dTDP-4-keto-6-deoxy-D-glucose to dTDP-3-keto-6-deoxy-D-glucose before the C-4 deoxygenase steps catalysed by MegCIV/MegCV (Fig. 2, route A), protein expression experiments were performed in $E. coli$ K207-3. For the analysis of the putative products of the sugar biosynthesis pathways reconstituted in $E. coli$, we required a sensitive and selective method to detect sugar nucleotides in vivo. LC/MS/MS fulfilled our requirements. dTDP-sugars ionized well in negative-ion mode, and collisionally induced dissociation of the pseudomolecular ion produces a prominent daughter ion of $m/z 321$ corresponding to thiamine monophosphate (TMP). By performing a precursor scan of the TMP daughter ion, we were able to selectively screen for new or unexpected dTDP-sugar intermediates. In situations where we were screening for specific dTDP-sugars, multiple reaction monitoring (MRM) was performed by using the expected psuedomolecular ion based on the dTDP-sugar's molecular formula and $m/z 321$ as the parent/daughter pair. Both MS methods gave excellent selectivity, and based on the standards available, the MRM method provided detection limits in the low ng ml$^{-1}$ range.
The compatible plasmids pLB99 and pLB318 (Table 2) were used for co-expression of *megCII* and *megCIV-megCV*, respectively, and cell-free extracts were screened by LC/MS/MS for dTDP-activated hexose pools in *E. coli*. Cell-free extracts of *E. coli* expressing only MegCII did not show production of any new dTDP-sugar intermediate. However, extracts of *E. coli* expressing MegCIV and MegCV showed a new compound with a parent/daughter pair of *m/z* 529/321 (Fig. 3a) that corresponds to the mass of dTDP-3-keto-4,6-dideoxy-D-glucose. This is expected to be the intermediate substrate for the aminotransferase MegDII, the next step in the D-desosamine biosynthetic pathway, as demonstrated for the pikromycin aminotransferase DesV (Rahman *et al.*, 2001; Zhao *et al.*, 2001). It is important to note that no new compounds were detected in extracts of *E. coli* expressing MegCII or MegCV. Co-expression of MegCII with MegCIV and MegCV did not modify the production levels of dTDP-3-keto-4,6-dideoxy-D-glucose, nor cause accumulation of any other dTDP-sugar intermediate.

To confirm whether the activated sugar which accumulated (*m/z* 529) was the substrate for the aminotransferase MegDII followed by the methyltransferase MegDIII (Fig. 2, route B), two additional plasmids were constructed: pLB255 containing *megDII* and pKOS506-56 containing the pair of genes *megDII-megDIII*. *E. coli* extracts co-expressing MegCIV and MegCV together with MegDII (pLB318 and pLB255) showed the compound observed previously (parent/daughter *m/z* 529/321) and a new compound with a parent/daughter pair of *m/z* 530/321 which corresponds to the expected intermediate dTDP-3-amino-3,4,6-trideoxy-D-glucose (Fig. 3b), the product of the aminotransferase reaction. Analysis of *E. coli* extracts expressing MegCIV, MegCV, MegDII and MegDIII (pLB318 and pKOS506-56) showed accumulation of a compound with a parent/daughter pair of *m/z* 558/321, which corresponds to dTDP-D-desosamine (Fig. 3c). The identity of the compound was confirmed by comparison to an authentic standard of dTDP-D-desosamine. These results unambiguously demonstrate that production of dTDP-D-desosamine from dTDP-4-keto-6-deoxy-D-glucose requires only four biosynthetic steps, catalysed by MegCIV, MegCV, MegDII and MegDIII (Fig. 2, route B), and that MegCII is not necessary for this pathway.

Role of MegCII in the D-desosamine pathway of *M. megalomicae*

Since MegCII was clearly not involved in the biosynthesis of dTDP-D-desosamine, and considering its high homology with proteins recently described as part of a two-component desosaminyltransferase system (Borisova *et al.*, 1999), we hypothesized that MegCII could also be required, together with MegCIII, to catalyse the transfer of D-desosamine. To test this hypothesis, we fed MEB to an *E. coli* strain containing the D-desosamine operon, in the presence or absence of MegCII. The *E. coli* strain used was K506-121 (S. Peirú, unpublished) carrying an *rmlC* deletion and either of the following two plasmids. The first, pKOS506-72B, contained the dTDP-D-desosamine biosynthetic genes described before (*megCIV*, *megCV*, *megDII* and *megDIII*), plus the putative glycosyltransferase (*megCIII*) and the erythromycin resistance gene *ermE* in a single operon. The second, pKOS506-72A, was almost identical to pKOS506-72B but contained *megCII* in addition. The growth and induction conditions of the cultures were the same as described previously except that the cultures were supplemented with 50 mg MEB l⁻¹. After 3 days of induction, cell-free extracts and culture broths were analysed by LC/MS. Analysis of cell-free extracts showed that both strains produced dTDP-D-desosamine as described before, but erythromycin D production occurred only in the strain harbouring MegCII (Fig. 4). Protein analysis showed that MegCIII is expressed as a soluble protein and its expression is not

http://mic.sgmjournals.org
E. Rodriguez and others

In the current work we used LC/MS/MS to characterize dTDP-sugar pools in *E. coli* cell-free extracts that express the proteins of interest. This approach enabled us to establish that in *M. megalomicea*, production of dTDP-D-desosamine from dTDP-4-keto-6-deoxy-D-glucose requires only four biosynthetic steps, catalysed by MegCIV, MegCV, MegDII and MegDIII (Fig. 2, route B). This result excluded MegCII for the tautomerization step originally proposed for its homologue EryCII in the erythromycin pathway and agrees with the proposal that the pikromycin DesDI and DesII (homologues to MegCIV and MegCV, Table 1) work together to catalyse the C-4 deoxygenation of dTDP-3-keto-4,6-dideoxy-D-glucose, presumably via an amino sugar intermediate, and do not require a previous step of isomerization by DesVIII (a MegCII homologue, Table 1) (Szu *et al.*, 2005; Zhao *et al.*, 2001). However, in our system, expression of MegCIV by itself did not cause accumulation of any intermediate *in vivo*.

Bioconversion experiments demonstrated that the glycosyltransferase MegCIII requires MegCII for *in vivo* transfer of dTDP-D-desosamine to MEB. This experiment disagreed with the initial *in vitro* characterization of EryCIII (82·7% identical to MegCIII, Table 1), which showed activity in the absence of EryCII (71·3% identical to MegCII, Table 1) (Lee *et al.*, 2004). However, a more recent publication (Yuan *et al.*, 2005) suggests that EryCIII appears to be activated by EryCII by a still unknown mechanism; once this activation has occurred the presence of EryCII is no longer necessary. Other glycosyltransferases that require an auxiliary protein for efficient glycosyltransfer in macrolide antibiotics biosynthesis have recently been described. Activities of TylMII from the tylosin pathway of *Streptomyces fradiae* and MycB from the mycinamicin pathway of *Micromonospora griseorubida* were significantly enhanced by co-expression of TylMIII and MydC, respectively (Melancon *et al.*, 2004). Another two-component system was described for the aclacinomycin cluster, where AknT stimulates AknS 40-fold for transfer of L-2-deoxyfucose to the aglycone aklavinose (Lu *et al.*, 2005).

Although the mechanism of action of MegCIII and MegCII in the glycosyltransfer reaction is unknown, our results clearly establish the correct protein assignment for macrolide elaboration, which may help in the generation of novel macrolide agents. In addition, we have demonstrated the utility of *in vivo* characterization of dTDP-activated sugar intermediates by LC/MS/MS and its potential in the study of gene/enzyme activity of the sugar biosynthetic pathways of the growing collection of macrolide gene clusters.

**DISCUSSION**

The past few years have witnessed rapid growth in the number of new polyketide synthase gene clusters cloned and sequenced, prompting the need for new techniques to assign protein functions in the biosynthesis of polyketides, rather than relying on protein sequence comparisons. Moreover, since glycosylation by deoxy and amino sugars is vital for the efficacy and specificity of biological activity, the development of a new generation of therapeutic agents should follow from the alteration and/or exchange of these crucial sugar moieties.

In the *meg* gene cluster two genes encode putative glycosyltransferases, *megCIII* and *megDI*. Because MegCIII most closely resembles EryCIII, a recently characterized desaminyl transferase (Lee *et al.*, 2004), and because *megCIII* has the same location on the *meg* cluster as its counterpart in the *ery* cluster, *megCIII* was assigned to the D-desosamine pathway, leaving *megDI* for the mesosamine pathway (Volchegursky *et al.*, 2000).

**ACKNOWLEDGEMENTS**

We would like to acknowledge David Hopwood and Gary Ashley for their useful comments on the manuscript, Chau Tran for assistance with LC/MS experiments and Ben Liu for providing us with the chemically synthesized dTDP-D-desosamine.

REFERENCES

Alarcon, B., Gonzalez, M. E. & Carrasco, L. (1988). Megalomicin C, a macrolide antibiotic that blocks protein glycosylation and shows antiviral activity. FEBS Lett 231, 207–211.

Bonay, P., Munro, S., Fresno, M. & Alarcon, B. (1996). Intra-Golgi transport inhibition by megalomicin. J Biol Chem 271, 3719–3726.

Bonay, P., Fresno, M. & Alarcon, B. (1997). Megalomicin disrupts lysosomal functions. J Cell Sci 110, 1839–1849.

Bonay, P., Duran-Chica, I., Fresno, M., Alarcon, B. & Alcina, A. (1998). Antiparasitic effects of the intra-Golgi transport inhibitor megalomicin. Antimicrob Agents Chemother 42, 2668–2673.

Borisova, S. A., Zhao, L., Sherman, D. H. & Liu, H. W. (1999). Biosynthesis of desosamine: construction of a new macrolide carrying a genetically designed sugar moiety. Org Lett 1, 133–136.

Gaisser, S., Bohm, G. A., Cortes, J. & Leadlay, P. F. (1997). Analysis of seven genes from the eryAI-eryK region of the erythromycin biosynthetic gene cluster in Saccharopolyspora erythraea. Mol Gen Genet 256, 239–251.

Gaisser, S., Bohm, G. A., Doumith, M., Raynal, M. C., Dhillon, N., Cortes, J. & Leadlay, P. F. (1998). Analysis of eryBI, eryBIII and eryBVII from the erythromycin biosynthetic gene cluster in Saccharopolyspora erythraea. Mol Gen Genet 258, 78–88.

Lee, H. Y., Chung, H. S., Hang, C., Khosla, C., Walsh, C. T., Kahne, D. & Walker, S. (2004). Reconstitution and characterization of a new desosaminyl transferase, EryCIII, from the erythromycin biosynthetic pathway. J Am Chem Soc 126, 9924–9925.

Lu, W., Leimkuhler, C. Gatto, G. J., Jr, Kruger, R. G., Oberthur, M., Kahne, D. & Walsh, C. T. (2005). AknT is an activating protein for the glycosyltransferase AknS in L-aminodeoxysugar transfer to the aglycone of aclacinomycin A. Chem Biol 12, 527–534.

Melancon, C. E., III, Takahashi, H. & Liu, H. W. (2004). Characterization of tylM3/tylM2 and mydCl/mycB pairs required for efficient glycosyltransfer in macrolide antibiotic biosynthesis. J Am Chem Soc 126, 16726–16727.

Murli, S., Kennedy, J., Dayem, L. C., Carney, J. R. & Kealey, J. T. (2003). Metabolic engineering of Escherichia coli for improved 6-deoxyerythronolide B production. J Ind Microbiol Biotechnol 30, 500–509.

Peirú, S., Menzella, H. G., Rodriguez, E., Carney, J. & Gramajo, H. (2005). Production of the potent antibacterial polyketide erythromycin C in Escherichia coli. Appl Environ Microbiol 71, 2539–2547.

Rahman, A., Barr, K. & Rick, P. D. (2001). Identification of the structural gene for the TDP-Fuc4NAc:lipid II Fuc4NAc transferase involved in synthesis of enterobacterial common antigen in Escherichia coli K-12. J Bacteriol 183, 6509–6516.

Salah-Bey, K., Doumith, M., Michel, J. M., Haydock, S., Cortes, J., Leadlay, P. F. & Raynal, M. C. (1998). Targeted gene inactivation for the elucidation of deoxysugar biosynthesis in the erythromycin producer Saccharopolyspora erythraea. Mol Gen Genet 257, 542–553.

Sambrook, J., Fritsch, E. F. & Maniatis, T. (1989). Molecular Cloning: a Laboratory Manual, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

San Jose, E., Munoz-Fernandez, M. A. & Alarcon, B. (1997). Megalomicin inhibits HIV-1 replication and interferes with gp160 processing. Virology 239, 303–314.

Summers, R. G., Donadio, S., Staver, M. J., Wendt-Pienkowski, E., Hutchinson, C. R. & Katz, L. (1997). Sequencing and mutagenesis of genes from the erythromycin biosynthetic gene cluster of Saccharopolyspora erythraea that are involved in L-mycarose and D-desosamine production. Microbiology 143, 3251–3262.

Szu, P. H., He, X., Zhao, L. & Liu, H. W. (2005). Biosynthesis of TDP-D-desosamine: identification of a strategy for C4 deoxygenation. Angew Chem Int Ed Engl 44, 6742–6746.

Volchegursky, Y., Hu, Z., Katz, L. & McDaniel, R. (2000). Biosynthesis of the anti-parasitic agent megalomicin: transformation of erythromycin to megalomicin in Saccharopolyspora erythraea. Mol Microbiol 37, 752–762.

Waitz, J. A., Moss, E. L., Jr, Oden, E. M. & Weinstein, M. J. (1969). Biological activity of megalomicin, a new Micromonospora-produced macrolide antibiotic complex. J Antibiot 22, 265–272.

Weber, J. M., Leung, J. O., Maine, G. T., Potenz, R. H., Paulus, T. J. & DeWitt, J. P. (1990). Organization of a cluster of erythromycin genes in Saccharopolyspora erythraea. J Bacteriol 172, 2372–2383.

Weinstein, M. J., Wagman, G. H., Marquez, J. A., Testa, R. T., Oden, E. & Waitz, J. A. (1969). Megalomicin, a new macrolide antibiotic complex produced by Micromonospora. J Antibiot 22, 253–258.

Weymouth-Wilson, A. C. (1997). The role of carbohydrates in biologically active natural products. Nat Prod Rep 14, 99–110.

Xue, Y., Zhao, L., Liu, H. W. & Sherman, D. H. (1998). A gene cluster for macrolide antibiotic biosynthesis in Streptomyces venezuelae: architecture of metabolic diversity. Proc Natl Acad Sci USA 95, 12111–12116.

Yuan, Y., Chung, H. S., Leimkuhler, C., Walsh, C. T., Kahne, D. & Walker, S. (2005). In vitro reconstitution of EryCIII activity for the preparation of unnatural macrolides. J Am Chem Soc 127, 14128–14129.

Zhao, L., Borisova, S., Yeung, S. M. & Liu, H. (2001). Study of C-4 deoxygenation in the biosynthesis of desosamine: evidence implicating a novel mechanism. J Am Chem Soc 123, 7909–7910.
