
COMMUNICATION

Characterisation of *Streptomyces spheroides* NovW and revision of its functional assignment to a dTDP-6-deoxy-D-xylo-4-hexulose 3-epimerase

Mónica Tello,$^{a,b}$ Piotr Jakimowicz,$^{+c}$ James C. Errey,$^{a}$ Caren L. Freel Meyers,$^{\ddagger d}$ Christopher T. Walsh,$^{d}$ Mark J. Buttner,$^{c}$ David M. Lawson$^{b}$ and Robert A. Field*$^{a}$

Received (in Cambridge, UK) 7th November 2005, Accepted 14th December 2005  
First published as an Advance Article on the web 24th January 2006  
DOI: 10.1039/b515763c

Characterisation of recombinant *Streptomyces spheroides* NovW in vitro suggests that it is not a kinetically competent dual action dTDP-6-deoxy-D-xylo-4-hexulose 3,5-epimerase, but possesses only significant 3-epimerase activity.

The aminocoumarin antibiotic novobiocin (Fig. 1) is active against Gram-positive bacteria by virtue of its ability to inhibit the type II topoisomerase DNA gyrase.$^{1}$ It shares structural features in common with other *Streptomyces*-derived aminocoumarins, such as clorobiocin and coumermycin A$_{1}$. Data from several crystallographic$^{2,3}$ and structure–activity relationship studies$^{4-7}$ show that noviose, the sugar moiety of these antibiotics, is crucial for the binding and renders all three natural products competitive inhibitors of DNA gyrase with respect to ATP binding.

Following the identification of the S. *spheroides* gene cluster responsible for novobiocin production,$^{8}$ a biosynthetic pathway leading to noviose was proposed (see Fig. 2, central pathway). Our attention was drawn to NovW, proposed to be the third enzyme of the pathway, which was thought to serve as a dTDP-6-deoxy-D-xylo-4-hexulose 3,5-epimerase. NovW has high sequence homology to the validated 3,5-epimerase RmlC$^{9-11}$ from the dTDP-L-rhamnose pathway and to the 5-epimerase EvaD$^{12}$ from

![Figure 1](https://i.imgur.com/chemical_structure_of_novobiocin.png)

**Fig. 1** Chemical structure of novobiocin.

---

$^{a}$Centre for Carbohydrate Chemistry, University of East Anglia, Norwich, NR4 7TJ, UK. E-mail: r.a.field@uea.ac.uk; Fax: +44-1603-592003  
$^{b}$Department of Biological Chemistry, John Innes Centre, Norwich, NR4 7UH, UK  
$^{c}$Department of Molecular Microbiology, John Innes Centre, Norwich, NR4 7UH, UK  
$^{d}$Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, MA 02115, US  
$^{+}$ Present address: Department of Microbiology, Institute of Immunology and Experimental Therapy, Polish Academy of Sciences, Weigla 12 Wroclaw, PL-53-114, Poland.  
$\ddagger$ Present address: Department of Pharmacology and Molecular Sciences, Johns Hopkins University School of Medicine and Johns Hopkins Health System, 720 Rutland Avenue, Baltimore, Maryland 21205, USA.

This journal is © The Royal Society of Chemistry 2006

Chem. Commun., 2006, 1079–1081 | 1079

Table 1 Sequence identities for RmlC family epimerases$^a$

| EvaD | RmlC | RmlC |
| --- | --- | --- |
| $A. orientalis$ | $S. enterica$ | $M. tuberculosis$ |
| NovW $S. spheroides$ | 46% | 38% |
| EvaD $A. orientalis$ | 33% | 42% |
| RmlC $S. enterica$ | 33% | 36% |

$^a$ Values generated using NCBI BLAST version 2.2.12.

---

Table 2 Kinetic parameters for double epimerase activity

|  | $K_{\mathrm{m}} / \mathrm{mM}$ | $k_{\mathrm{cat}} / \mathrm{s}^{-1}$ | $(k_{\mathrm{cat}} / K_{\mathrm{m}}) /$ |
| --- | --- | --- | --- |
|  |  |  | $\mathrm{s}^{-1} \mathrm{mM}^{-1}$ |
| NovW $S. spheroides$ | $0.97 \pm 0.01$ | $0.020 \pm 0.001$ | $0.021$ |
| EvaD $A. orientalis^{12}$ | $0.27 \pm 0.03$ | $0.08 \pm 0.01$ | $0.33$ |
| RmlC $S. enterica^{17}$ | $0.71 \pm 0.03$ | $39 \pm 6.6$ | $54.9$ |

Kinetic parameters (Table 2) for the double epimerisation activity of NovW acting on dTDP-6-deoxy-D-xylo-4-hexulose were measured by the coupled assay method used previously to assay RmlC$^{11,17}$ and EvaD.$^{12}$ That is, the formation of 3,5-epimerised product was coupled with in situ reduction by RmlD, with concomitant NADPH consumption. The $k_{\mathrm{cat}}$ for NovW determined in this manner was nearly 3 orders of magnitude lower than for RmlC of $S. typhimurium$ and 4 times lower than that of EvaD. On the other hand, $K_{\mathrm{m}}$ values for all 3 enzymes are broadly of the same magnitude. The $k_{\mathrm{cat}} / K_{\mathrm{m}}$ ratio for these enzymes situates NovW and EvaD at a significantly lower level than RmlC ($ca.$ 100–1000 fold). On this basis, the assignment of NovW as a dTDP-6-deoxy-D-xylo-4-hexulose 3,5-epimerase is implausible.

To get direct evidence for the site of NovW action on dTDP-6-deoxy-D-xylo-4-hexulose,$^{18}$ solvent isotope incorporation experiments were performed, following the method employed by Stern et al$^{9}$ to study RmlC. Since the equilibrium position for RmlC$^{9}$/EvaD/NovW action is very heavily in favour of substrate (data not shown), deuterium incorporation into substrate was monitored. Following a reduction-cleavage-reduction-acetylation work-up procedure, GC-MS analysis of NovW reactions performed in D$_2$O with dTDP-6-deoxy-D-xylo-4-hexulose showed two main peaks (6.57 min and 6.65 min) in the GC trace (Fig. 4). The peak at 6.57 min corresponded to the fucitol standard (axial product arising from 4-keto substrate reduction) and at 6.65 min to the quinovositol standard (equatorial product arising from 4-keto substrate reduction). MS analysis of both peaks gave characteristic 218/232 peaks arising from incorporation of a single deuterium atom specifically at position 3. GC-MS results for parallel experiments on RmlC showed two major GC peaks at the same retention times as for NovW reactions. However, MS analysis of both peaks revealed the 218/233 profile of deuterium incorporation at both positions 3 and 5, as expected for RmlC activity. These results clearly indicate that under conditions where an equimolar quantity of RmlC proved capable of effecting complete deuterium

---

(A)

![Figure A](attachment://image1.png)

(B)

![Figure B](attachment://image2.png)

Fig. 3 (A) Overlap of sugar-nucleotide epimerase backbone structures: RmlC of *Salmonella typhimurium* in blue (PDB accession code 1DZR), NovW of *Streptomyces spheroides* in green (PDB accession code 2C0Z) and EvaD of *Amylocalyptis orientalis* in red (PDB accession code 1OFN). (B) Overlap of the 4 main residues participating in the epimerization reaction.

Foundation (RAF). Lutz Heide is acknowledged for the provision of cosmid 10-9C containing the novobiocin cluster of *S. spheroides*. Jim Naismith is thanked for helpful discussions and for the provision of *Rml* clones.

### Notes and references

§ The residue numbering used herein corresponds to that of NovW from *S. spheroides*.

1. A. Maxwell and D. M. Lawson, *Curr. Top. Med. Chem.* (Sharjah, United Arab Emirates), 2003, **3**, 283.
2. V. Lamour, L. Hoermann, J. M. Jeltsch, P. Oudet and D. Moras, *Acta Crystallogr.*, Sect. *D*, 2002, **D58**, 1376.
3. R. J. Lewis, O. M. Singh, C. V. Smith, T. Skarzynski, A. Maxwell, A. J. Wonacott and D. B. Wigley, *EMBO J.*, 1996, **15**, 1412.
4. D. C. Hooper, J. S. Wolfson, G. L. McHugh, M. B. Winters and M. N. Swartz, *Antimicrob. Agents Chemother.*, 1982, **22**, 662.
5. B. Musicki, A. M. Periers, P. Laurin, D. Ferroud, Y. Benedetti, S. Lachaud, F. Chatreaux, J. L. Haesslein, A. Iltis, C. Pierre, J. Khider, N. Tessot, M. Airault, J. Demassey, C. Dupuis-Hamelin, P. Lassaigne, A. Bonnefoy, P. Vicat and M. Klich, *Bioorg. Med. Chem. Lett.*, 2000, **10**, 1695.
6. B. Musicki, A. M. Periers, L. Piombo, P. Laurin, M. Klich, C. Dupuis-Hamelin, P. Lassaigne and A. Bonnefoy, *Tetrahedron Lett.*, 2003, **44**, 9259.
7. D. Ferroud, J. Collard, M. Klich, C. Dupuis-Hamelin, P. Mauvais, P. Lassaigne, A. Bonnefoy and B. Musicki, *Bioorg. Med. Chem. Lett.*, 1999, **9**, 2881.
8. M. Steffensky, A. Muhlenweg, Z. X. Wang, S. M. Li and L. Heide, *Antimicrob. Agents Chemother.*, 2000, **44**, 1214.
9. R. J. Stern, T. Y. Lee, T. J. Lee, W. Yan, M. S. Scherman, V. D. Vissa, S. K. Kim, B. L. Wanner and M. R. McNeil, *Microbiology*, 1999, **145**, 663.
10. M. F. Giraud, G. A. Leonard, R. A. Field, C. Bernlind and J. H. Naismith, *Nat. Struct. Biol.*, 2000, **7**, 398.
11. C. Dong, L. L. Major, A. Allen, W. Blankenfeldt, D. J. Maskell and J. H. Naismith, *Structure*, 2003, **11**, 715.
12. A. B. Merkel, L. L. Major, J. C. Errey, M. D. Burkart, R. A. Field, C. T. Walsh and J. H. Naismith, *J. Biol. Chem.*, 2004, **279**, 32684.
13. P. N. Kirkpatrick, W. Scaife, T. M. Hallis, H. S. Liu, J. B. Spencer and D. H. Williams, *Chem. Commun.*, 2000, 1565.
14. T. T. Thu Thuy, H. C. Lee, C. G. Kim, L. Heide and J. K. Sohng, *Arch. Biochem. Biophys.*, 2005, **436**, 161.
15. N-terminal His-tagged NovW was produced by amplifying the *novW* gene from *S. spheroides* genomic DNA and cloning it into pET-15b (Novagen). NovW was coexpressed with GroES/GroEL<sup>19</sup> chaperone system in *E. coli* BL21 (DE3) and was purified by Ni-NTA affinity chromatography (Hi-Trap column, Amersham), followed by size exclusion chromatography (HiLoad™ Superdex™ 16/60, Amersham). Expression level: ca. 13 mg L<sup>−1</sup>.
16. P. Jakimowicz, M. Tello, C. L. Freel Meyers, C. T. Walsh, M. J. Buttner, R. A. Field and D. M. Lawson, *Proteins: Struct., Funct., Bioinf.*, 2005, DOI: 10.1002/prot.20818.
17. M. Graninger, B. Nidetzky, D. E. Heinrichs, C. Whitfield and P. Messner, *J. Biol. Chem.*, 1999, **274**, 25069.
18. The reaction substrate, dTDP-6-deoxy-D-xylo-4-hexulose, was prepared enzymatically by overnight incubation of dTDP-glucose (10 mg) with dTDP-glucose-4,6-dehydratase (RmlB) (1.5 mg) and 1 mM NAD<sup>+</sup> at 37 °C. Substrate purity was monitored by <sup>1</sup>H NMR spectroscopy; data were in accord with the literature.<sup>20</sup> D-Fucitol, D-quinovositol and L-rhamnitol were used as standards for alditol analysis by GC-MS.
19. H. Grallert and J. Buchner, *J. Struct. Biol.*, 2001, **135**, 95.
20. A. Naundorf and W. Klaffke, *Carbohydr. Res.*, 1996, **285**, 141.
