
Available online at www.sciencedirect.com

SCIENCE DIRECT®

Progress in Lipid Research 45 (2006) 237–249

Review

Fatty acid elongases in mammals: Their regulation and roles in metabolism

Andreas Jakobsson, Rolf Westerberg, Anders Jacobsson*

The Wenner-Gren Institute, The Arrhenius Laboratories F3, Stockholm University, SE-10691 Stockholm, Sweden

Received 9 January 2006; received in revised form 17 January 2006; accepted 25 January 2006

---

### Abstract

A significant amount of the fatty acids synthesized by the cytosolic enzyme complex fatty acid synthase (FAS) or taken up by the diet are further elongated into very long chain fatty acids (VLCFA) in a four-step reaction cycle by membrane-bound enzymes predominantly located in the endoplasmic reticulum. Members of the *Elovl* (elongation-of-very-long-chain-fatty acids) gene family encode for enzymes (elongases), which are believed to perform the first, regulatory, step (condensation) in the elongation cycle in mammals. The family of enzymes consists of at least six members in mouse and human, believed to carry out substrate-specific elongation with fatty acids of different lengths and degrees of unsaturation.

The ability to synthesize VLCFA is a ubiquitous system found in different organs and cell types. However, VLCFAs seldom occur unesterified. Instead, they are joined either by an ester or amide linkage to a broad variety of different lipid species. VLCFA are most commonly found as building blocks in sphingolipids, although they are also important constituents of glycerophospholipids, triacylglycerols, sterol- and wax-esters.

To generalize, the fatty acid elongases can be divided into two major groups: (a) enzymes which are suggested to be involved in the elongation of saturated and monounsaturated VLCFA (ELOVL1, 3 and 6) and (b) enzymes which are elongases of polyunsaturated fatty acids (PUFA) (ELOVL2, 4 and 5). All the elongases exhibit specific spatial and temporal expression.

In this review, we will present and discuss the regulation of the mammalian fatty acid elongases and their potential role in lipid metabolism. We will consider both the biochemical functions of the proteins, as well as their role in a more physiological context.

© 2006 Elsevier Ltd. All rights reserved.

Keywords: Fatty acid elongation; Elongase; VLCFA; Lipid metabolism

---

### Abbreviations:

ACC, acetyl-CoA carboxylase; BAT, brown adipose tissue; CIG30, cold induced glycoprotein of 30 kDa; CNS, central nervous system; DGAT, diacylglycerol acyltransferase; DHA, docosahexaenoic acid; Elo, yeast elongase; ELOVL, elongation of very long chain fatty acids; EPA, eicosapentaenoic acid; FACE, fatty acid elongase; FAE, plant fatty acid elongase; FAS, fatty acid synthase; FIAF, fasting induced adipose factor; HEK, human embryonic kidney cells; HELO, human elongase; KAR, 3-ketoacyl-CoA reductase; LXR, liver X receptor; NADH, nicotinamide adenine dinucleotide; NADPH, nicotinamide adenine dinucleotide phosphate; PEA, Caenorhabditis elegans elongase; PPAR, peroxisome proliferator-activated receptor; PUFA, polyunsaturated fatty acids; rELO, rat elongase; SCD, steroyl-CoA desaturases; SREBP, sterol regulatory element binding protein; SSC, sequence similarity to cig30; TER, trans-2,3-enoyl-CoA reductase; VLCFA, very long chain fatty acids; WAT, white adipose tissue.

* Corresponding author.
E-mail address: anders.jacobsson@wgi.su.se (A. Jacobsson).

0163-7827/$ - see front matter © 2006 Elsevier Ltd. All rights reserved.
doi:10.1016/j.plipres.2006.01.004

Contents

1. Introduction ..... 238
2. The ELOVL family ..... 239
   2.1. Mammalian elongases ..... 239
   2.2. The ELOVL proteins ..... 240
   2.3. Functional analysis of the ELOVL enzymes ..... 241
3. Regulation of Elovl expression ..... 244
4. ELOVL and PUFA synthesis ..... 245
5. General conclusions ..... 246
Acknowledgements ..... 247
References ..... 247

---

1. Introduction

Much of our basic knowledge regarding fatty acid chain elongation in the endoplasmic reticulum (ER) derives from the pioneering work of Nugteren [72]. By using different rat liver cell fractions, he found that microsomes containing the ER fraction possessed the highest enzyme activity for elongating labeled fatty acyl-CoAs. This chain elongation process was clearly distinguished from de novo fatty acid synthesis found in the supernatant.

In mammals, fatty acids consisting of up to 16 carbons (palmitic acid) in length are synthesized by fatty acid synthase (FAS). Animal FAS functions as a homodimeric and multifunctional complex of ca 250 kDa found in the cytoplasm, which harbors seven different enzymatic activities in two catalytic centers. Synthesis of fatty acids by FAS is initiated by the elongation of a primer (i.e., acetyl or propionyl) with two-carbon units donated from malonyl-CoA and utilizes NADPH as reductant in the elongation reaction. Repeating this reaction seven times in a cyclic manner enables FAS to ultimately produce the saturated C16 fatty acid, palmitic acid [20, 87, 103, 104].

A significant amount of the fatty acids produced by FAS, as well as fatty acids taken up from the diet, are further elongated into long-chain fatty acids containing 18 carbon atoms or longer, i.e., very long-chain fatty acids (VLCFA). Formation of VLCFA is mainly performed in the ER by membrane-bound enzymes and the enzymatic steps involved in these processes are principally the same as described for FAS (Fig. 1). In contrast to the FAS complex, the four principal successive steps of VLCFA elongation are performed by individual proteins, which may be physically associated [19, 51, 72]. Three of the four enzymatic activities in VLCFA elongation are localized to the cytoplasmic side of ER membranes, while the enzyme performing the third step is suggested to be embedded in the membrane [74].

The reaction process begins with the condensation of an acyl-CoA molecule and malonyl-CoA, resulting in β-ketoacyl-CoA. The second step is a reduction reaction, which requires NADPH, where β-ketoacyl-CoA is converted to β-hydroxyacyl-CoA. β-Hydroxyacyl-CoA is subsequently dehydrated in the third step, resulting in enoyl-CoA, which needs to be reduced by enoyl-reductase in the fourth step to complete the elongation cycle and generate an extended acyl-chain.

Further, the microsomes may utilize NADH, but under rate-limiting conditions, a clear preference for NADPH was noticed [72]. The presence of fatty acid binding proteins, such as for example albumin, is necessary for optimal reaction [8, 9].

Several attempts have been made to purify the enzymes involved in the microsomal elongation process, largely with an unsuccessful outcome owing to the hydrophobic properties of the proteins. In 1979, Bernert and Sprecher were able to solubilize and partially purify β-hydroxyacyl-CoA dehydrase from rat liver microsomes. The enzyme was free of condensation and enoyl-CoA reductase activity, which again shows that the chain elongation system consists of discrete enzymes in contrast to cytosolic FAS [7].

A. Jakobsson et al. / Progress in Lipid Research 45 (2006) 237–249

Fig. 1. Fatty acid elongation in mammals. Fatty acids, which are synthesized by the fatty acid synthase (FAS) within the cytosol or taken up by the diet, are further elongated in the endoplasmic reticulum (ER) by four individual enzymes. The elongases, ELOVL (elongation of very long chain fatty acids), are the condensing enzymes suggested to determine the substrate specificity and are the rate-limiting step in the elongation cycle. KAR, 3-ketoacyl-CoA reductase; TER, trans-2,3-enoyl-CoA reductase.

Based on the rates obtained for the four individual reactions in the elongation process, Nugteren was the first to postulate that the first reaction is the rate-limiting step. By omitting NADPH from the elongation reaction in rat liver microsomes, it became evident that the conversion of β-ketopalmitic acid to palmitate proceeded approximately 10 times faster than the condensation reaction [72]. In addition, ectopic expression of the plant condensing enzyme FAE1 in yeast and plants leads to accumulation of VLCFAs, as shown by Millar and Kunst in 1997 [63]. This supports the theory that introduction of a condensing enzyme alone is sufficient for specific induction of VLCFA synthesis, thus it is the limiting factor for VLCFA synthesis.

## 2. The ELOVL family

### 2.1. Mammalian elongases

Extensive biochemical studies of the VLCFA elongation machinery, suggested the existence of several distinct elongation pathways in mammals long before genomic sequences for the enzymes were available [90]. However, studies of the separate enzymes and their respective properties have now become possible through the recent cloning of the genes involved.

The genes and proteins involved in the elongation of VLCFA have been studied in several model systems including plants (reviewed by Leonard et al. [56]). To date, six enzymes, termed ELOVL 1–6 (Elongation of very long chain fatty acid), which are suggested to perform the condensation reaction in the elongation cycle, have been identified (Table 1). Based on the presence of similar motifs in their corresponding protein sequences, such as KXXEXXDT, HXXHH, HXXMYXYY, TXXQXXQ, they are grouped as a gene/protein family, which we will refer to as members of the Elongase gene and ELONGASE protein families, respectively. The ELONGASE family members from mouse and yeast have approximately 30% sequence similarity in each mouse-mouse or mouse-yeast protein pair [99]. This rather high sequence similarity is also seen when comparing all members of the ELONGASE family with each other, including nematode, yeast, mouse and human sequences [56].

The first gene to be identified in our laboratory was *Elov3*, originally termed *Cig30* (Cold-induced glycoprotein of 30 kDa) because massively increased mRNA levels were found in the brown adipose tissue

Table 1  
Fatty acid elongases in mammals  

| Recommended nomenclature | Previous names |
|--------------------------|----------------|
| ELOVL1                   | SSC1(m)        |
| ELOVL2                   | SSC2(m)        |
| ELOVL3                   | CIG30(m)       |
| ELOVL4                   |                |
| ELOVL5                   | HELO1(h), rELO1(r) |
| ELOVL6                   | LCE(m), FACE(m), rELO2(r) |

(m) = mouse, (r) = rat, (h) = human.

of cold-exposed mice [98]. In silico searches for *Elovl3/Cig30* homologues resulted in the discovery of two other novel mouse genes, which were designated *Ssc1* and *Ssc2* (sequence similarity to *cig30*) [99]. The official names for *Ssc1* and *Ssc2* are now *Elovl1* and *Elovl2*, respectively. At about the same time, the first corresponding human cDNA sequence, *ELOVL5/HELO1* (human elongase 1) [54] and *ELOVL1*, the human ortholog of mouse *Elovl1/Ssc1* (Asadi and Jacobsson, GenBank Accession No. AF336793), was identified. Shortly afterwards, cloning of a mouse long-chain-elongase, *Lce/FACE* now termed *Elovl6*, was reported [60,68]. The orthologous gene in rat was independently discovered by Inagaki et al. and denoted *rELO2* (rat elongase 2) [39]. The rat elongase *rELO1* have been reported to have similar properties to the human *ELOVL5* [39]. The human *ELOVL4* was described as being implicated in the pathogenesis of macular dystrophy, indicating a crucial role of VLCFA synthesis in the retina, which also applies to the mouse orthologue *Elovl4* [115,116]. Human *ELOVL2* was cloned and described by Leonard et al. [55] as a player in the synthesis of polyunsaturated fatty acids (PUFA), consistent with published data for murine *Elovl2* [68,99].

### 2.2. The ELOVL proteins

The membrane topology of the members of the ELONGASE family is predicted to be five or six transmembrane-spanning, due to the presence of several hydrophobic amino acid stretches. Due to the long hydrophobic stretch in place of the third putative trans-membrane region, it is unclear whether the total number of membrane-spanning regions is five or six [65,99]. Similar to the yeast and nematode elongases, the mouse and human ELOVL proteins all contain lysine or arginine residues at the carboxyl terminus, which are proposed to function as ER retrieval signals [5,21,62,95,99,115]. This is supported by immuno-fluorescence data showing that human *ELOVL4* is located in close vicinity to the ER [32]. We have found that ectopic expression of the murine *ELOVL3* fused to green fluorescent protein in mammalian cells, also indicates localisation to the ER (A. Jakobsson, unpublished observations). These results are in line with data showing that the yeast elongases *Elo2p* and *Elo3p* and the yeast reductases *Ybr159p* (3-ketoacyl-CoA reductase) and *Tsc13p* (enoyl-CoA reductase) co-localize to areas in the yeast cell that correspond to the ER, or perhaps early Golgi [4,21,51,75].

Previously, we have shown that *ELOVL3* is glycosylated in positions 6–9 of the N-terminal domain [98], which is efficiently modified with a high mannose oligosaccharide both in vivo and in vitro [65]. The N-terminal tail together with the first trans-membrane region enables correct insertion of the protein into the ER membrane [65,66]. *ELOVL4* has also been shown to be glycosylated near the N-terminal, although this modification seems not to be essential for protein expression or localization [32]. *ELOVL1* also contains a potential N-glycosylation motif, but since this site is buried in a putative membrane-spanning region, *ELOVL1* is most likely not glycosylated. Furthermore, no glycosylation site is found in *ELOVL2*. The absence of functional glycosylation sites in these proteins is also supported by experiments where no mobility shift of in vitro translated *ELOVL1* and *ELOVL2* polypeptides can be detected in the presence of canine microsomal membranes (R. Westerberg, unpublished observations). The functional consequences of divergent glycosylation patterns within the ELOVL family have not been investigated.

By co-immunoprecipitation experiments, Grayson and Molday [32] have recently shown that *ELOVL4* forms multisubunit complexes indicating that this enzyme functions as homodimer similar to what has been suggested for FAS [18]. Whether this mechanism applies for all ELOVL proteins is not clear. So far, specific antibodies of relatively high-quality have only been presented for *ELOVL4* [32].

Arguments concerning whether the members of the ELONGASE family are reductases or condensing enzymes have been forwarded, partly as a consequence of the conserved histidine motif also present in desaturases. Today, a plethora of evidence exists that convincingly demonstrates the ELOVL proteins to be the condensing enzymes and thereby also the rate-limiting enzymes for the overall elongation of VLCFAs. For example, HEK-293 cells transfected with an expression plasmid containing the *Elov16* open reading frame showed a sixfold higher elongation activity than cells transfected with empty vector alone. The end product was primarily 3-ketostearoyl-CoA as the cells were fed palmitoyl-CoA in the absence of NADPH, indicative of an end product from the condensing reaction [68].

Further, heterologous expression of the ELOVL-related enzyme PEA1 (*Caenorhabditis elegans*) together with fatty acid desaturases in yeast, resulted in the synthesis of PUFAs which are not normally synthesized in yeast [5].

Additional evidence for a well-defined role of the ELOVL proteins in the condensation step, comes from the characterization of the two cloned mammalian reductases implicated in the second and fourth steps of the elongation cycle, 3-ketoacyl-CoA reductase (KAR) and *trans*-2,3-enoyl-CoA reductase (TER) [67]. These reductases show very little sequence similarity to the ELOVL members, rather they instead show similarity to other dehydrogenases and reductases, such as 17β-hydroxysteroid dehydrogenase and steroid-Δ5-reductase [43]. In contrast to the divergent tissue expression pattern and substrate-specific activity of the ELOVLs, the reductases are expressed in all human and murine tissues tested so far and show no fatty acyl-chain specificity [67]. Secondly, co-expression of ELOVL6 and KAR or TER in cellular systems resulted in the disappearance of 3-ketostearoyl-CoA and *trans*-2,3-stearoyl-CoA intermediates, respectively. To date, only these two reductases have been identified in mammalian systems, which should be compared with the much larger number of ELOVL family members identified so far. However, yeast mutants lacking the reductase Ybr159p, which is responsible for the first reduction step in the elongation machinery, are still viable, albeit that they grow at a slower rate [4]. This might point to the existence of yet-to-be identified ketoacyl-CoA reductases or redundant functions of overlapping enzymatic systems.

A protein harboring a β-hydroxyacyl-CoA dehydrase activity that is committed to the VLCFA elongation cycle still remains to be cloned.

Published data on the function of ELOVL family members suggests involvement in distinct metabolic processes delimited by tissue- and substrate-specificity for the individual enzymes as described below.

### 2.3. Functional analysis of the ELOVL enzymes

*Elovl1* was discovered through its sequence homology to *Elovl3*, while the suggested enzymatic function correlates primarily to that described for yeast Elo3p, which is the synthesis of saturated C26 fatty acids and sphingolipid formation [73,99]. Saturated VLCFA are suggested to be important for the tight packaging of membrane lipids in so-called raft structures, which may have a function as organisers in intracellular transport and signaling events by creating micro-environments [85,86]. Yeast VLCFA elongases are shown to be important in several aspects of cell growth [3,21] and the overlapping functions of the yeast Elo3p and the murine ELOVL1 suggest that mammalian enzymes may be equally important in membrane-related functions with regard to sphingolipids. Changes in acyl-chain length of membrane lipid constituents, due to variations in elongase activity in yeast, affect membrane-residing enzyme activities, as well as membrane fusion/budding events [22,25,26,29].

The ubiquitous expression pattern of *Elovl1* in murine tissues may relate to the housekeeping-need for ELOVL1-derived fatty acid products and related lipids in the maintenance of membrane integrity. Relatively high levels of *Elovl1* mRNA are found in highly myelinated parts of the CNS, such as the corpus callosum and the spinal cord (A. Jacobsson, unpublished observations). Microsomal VLCFA elongation activity, using C22:0 as substrate, is high during myelin development in mice (around day 20 after birth) [12]. Quaking (*qk*) and *Jumpy* (*jp*) are two mutant mouse strains that are deficient in nerve myelination throughout the CNS, resulting in severe motor problems and early death. Although the deficiency in the two strains is defined as mutation in the myelin-associated glycoprotein (*Mag*) [28] and the proteolipid protein (*Plp*) gene [71], respectively, both strains nonetheless show markedly decreased elongation activity when tested with arachidoyl-CoA (20:0) or behenoyl-CoA (22:0) as substrate [92], and VLCFA levels in the brain [11,12], which
correlate well with the reduced *Elov11* mRNA expression [99]. Interestingly, the more severe *jumpy* phenotype correlates with a more pronounced decrease in *Elov11* mRNA levels compared to the *quaking* mice, suggesting that the severity of the elongation defect is proportional to the reduction in *Elov11* expression.

*Elov2* was hypothesized to have a role in PUFA synthesis on the basis of complementation studies in yeast and the testis-specific mRNA expression as the gene was first described in 2000 [99], although no functional characterization of this enzyme was performed at that time. Subsequently, both mouse and human ELOVL2 were shown to elongate arachidonic acid (20:4n-6), eicosapentaenoic acid (20:5n-3), docosatetraenoic acid (22:4n-6) and docosapentaenoic acid (22:5n-3) in transfected yeast and mammalian HEK 293 cells, whereas no activity was detected for saturated/monounsaturated substrates [55,68]. However, mouse ELOVL2 was able to elongate γ-linolenic acid (C18:3n-6) to some degree, while human ELOVL2 could not, which may indicate a minor functional divergence between the species. Although ELOVL2 is implied to have a role in the synthesis of peroxisome proliferator-activated receptor α (PPARα) ligands [45], at present no data exist on the physiological relevance of the enzyme in any tissue.

In rat liver, ELOVL2 is suggested to have overlapping function with ELOVL5 which uses a broader spectrum of PUFA substrate [106] (see below).

*Elov3* was cloned as an mRNA species highly elevated in brown adipose tissue (BAT) from cold-exposed mice, during processes of BAT recruitment [98]. Prolonged cold exposure for one month gradually decreased the elevated *Elov3* expression, but the level nonetheless remained far above control levels. The induced expression was qualitatively mimicked by continuous administration via osmotic pumps of norepinephrine, the endogenous sympathetic nervous activator of BAT, to mice kept at 30°C, or by high caloric intake [98]. Increased *Elov3* expression was also detected during perinatal development, reaching a maximum level immediately after birth, demonstrating that *Elov3* expression is highly correlated with the recruitment of brown fat [98]. Low levels of *Elov3* expression have also been detected in white adipose tissue and in adipose cell-lines such as 3T3-L1 and 3T3-F442A [38] (Jacobsson, unpublished observations).

The generation of an *Elov3*-ablated mouse model has enabled studies on the specific role of this elongase [109,110]. The elongation activity in the microsomal fraction of isolated BAT *Elov3*-ablated mice displayed decreased ability to elongate saturated C16–C22 substrates in response to cold exposure for three days [109]. Surprisingly, the difference in elongation activity could no longer be detected in fully cold-acclimated mice, exposed to 4°C for three weeks [109]. Analysis of fatty acid composition in triglycerides and sterol-esters during cold exposure also indicate a transient divergence from wild-type samples, where several different fatty acid species, including saturated, monounsaturated and polyunsaturated VLCFA, were altered in the 3-day cold-exposed mice but not after 3 weeks of cold acclimation. Furthermore, in *Elov3*-ablated mice housed under thermoneutral conditions (30°C) BAT was almost depleted of fat droplets [109]. Thus, even the relatively low level of *Elov3* expression seen in the absence of cold stress does apparently have significant impact on tissue lipid homeostasis.

The rationale for a shift towards very long saturated fatty acids in the cold is unclear, although long hydrophobic fatty acids are important components in the formation of fat droplets [79]. Even if there is a decrease in triglyceride content secondary to activation of fatty acid oxidation during cold exposure [2], there is at the same time a huge increase in triacylglycerol/fatty acid turnover in BAT [13,57]. Taken together, the data imply that ELOVL3 has a function in BAT to synthesize saturated and/or monounsaturated VLCFA for triglyceride formation in order to generate lipid droplets (Fig. 2).

The most prominent phenotypic trait coupled to the ablation of *Elov3* was found in the skin of these mice, where the barrier function was impaired as a consequence of altered VLCFA synthesis [110]. Considering that VLCFAs play an important structural role in skin-barrier function [108] and that most of the identified fatty acids elongases are expressed in murine skin [55,99,106], not much is known concerning the specific roles of the individual enzymes in skin. In situ hybridization analysis revealed that *Elov3* is expressed in the sebocytes of the sebaceous glands and in the inner layer of the outer root sheath in the hair follicles [110].

In agreement with data obtained in BAT, the triglyceride and sterol-ester fractions of sebum from *Elov3*-ablated mice exhibited a marked reduction in saturated and monounsaturated fatty acids in the range of C22–C26, concomitantly with an accumulation of eicosenoic acid (20:1n-9) [110] suggesting a similar function of ELOVL3 in skin as in BAT. Although ELOV3 partly complemented Elo2p in yeast as an elongase in the synthesis of saturated and monounsaturated VLCFA, the role of this mammalian enzyme in sphingolipid
A. Jakobsson et al. / Progress in Lipid Research 45 (2006) 237–249

Warm BAT

Cold BAT

Fig. 2. ELOVL3-induced fatty acid elongation in brown adipocytes. Norepinephrine (NE) induces fatty acid oxidation, heat production and ELOVL3-dependent very long chain fatty acid (VLCFA) synthesis in brown adipose tissue (BAT). The suggested function of *Elov3* in cold-stimulated BAT is to replenish intracellular pools of specific VLCFAs when the FA turnover rate is high.

synthesis is probably of minor importance since epidermal lipids from *Elov3*-ablated mice have normal ceramide composition despite the apparent skin phenotype [110].

Although *Elov5* mRNA levels are also somewhat up-regulated in BAT from cold-exposed, wild-type mice, no compensatory changes in *Elov5* expression, or any other *Elov* gene, were seen in the *Elov3*-ablated mice [109] (A. Jakobsson, unpublished observation).

Interestingly, several enzymes, such as steroyl-CoA desaturases (SCD), diacylglycerol acyltransferase (DGAT), ELOVL6 and ELOVL3, which are involved in the synthesis of saturated and monounsaturated fatty acids and triglyceride formation, when mutated, give rise to abnormalities in sebaceous lipid composition, impaired skin defects and metabolic irregularities in adipose tissue [16,34,93].

*Elov4* was initially discovered as a target gene in patients with the eye diseases Stargardt-like dystrophy and autosomal dominant macular dystrophy [115]. These diseases are suggested to be caused by a point mutation in *ELOVL4*, resulting in a frame-shift and thereby synthesis of a truncated peptide which accumulates in the cytosol, instead of being correctly inserted into the ER [32]. *ELOVL4* is specifically expressed in the human retina and to a lower extent in the brain and testis, which also applies to *Elov4* in murine tissues [59,116]. The high expression of *ELOVL4* in the retina indicates that it might be involved in the elongation steps required for synthesis of docosahexaenoic acid (22:6n–3) [115]. Cloning and characterization of *ELOVL4* was also reported in cynomolgus monkey (*Macaca fascicularis*), which serves as model for studying the eye disease. In agreement with the previously reported expression pattern of human *ELOVL4* by Zhang et al. in 2001 [115], mRNA encoding the enzyme was detected in retina and brain. In addition, expression was also detected in the thymus and skin [100].

*Elov5*, is involved in the elongation of various polyunsaturated long-chain fatty acids of C18–C20 [54]. However, *ELOVL5* does not appear to have the capacity to elongate PUFA substrates beyond C22. Although *ELOVL5* expression can be detected in most human tissues, the highest mRNA levels were found in the testis and adrenal gland, consistent with the fact that these two tissues contain high levels of docosapentaenoic acid (22:5n–6), a PUFA metabolite. Rat *ELO1/ELOVL5* (*rELO1*), was shown to elongate several different species of PUFA, especially stearidonic acid (18:4n–3) and γ-linolenic acid (18:3n–6), similar to the substrate preference of the human homolog *HELO1*. Further, in accordance with the expression profile of *HELO1*, *rELO1* mRNA was detected in most tissues examined, with the highest levels observed in lung and brain [39].

Peroxisomal β-oxidation is also involved in VLCFA anabolic processes, as seen for 22-carbon PUFA synthesis, where elongation, presumably by *ELOVL2* and/or *ELOVL5*, in the ER is followed by one round of chain-shortening in peroxisomes to produce the end products (e.g., 22:6n–3 and 22:5n–6) destined for esterification in the ER [69,102].

Elov16, is involved in the elongation of saturated fatty acids with 12–16 carbons to C18 and does not possess the capacity to elongate beyond C18. Elov16 is ubiquitously expressed, especially in tissues with high lipid content such as BAT, white adipose tissue (WAT), liver and brain [68]. In agreement with the function described for ELOVL6, rELO2 also acts on C16:0 substrates to synthesize C18:0. rELO2 is predominantly expressed in liver and to a smaller extent in brain [39].

### 3. Regulation of Elov1 expression

Little is known concerning the regulation of the ELOVL family members, both regarding mRNA/protein expression, as well as enzymatic activity. Although several of the fatty acid elongases contain identified modification sites, such as for glycosylation, so far no data exist on post-transcriptional control of any ELOVL protein or FAS. Available data rather favor the opposite; the activity is controlled by transcriptional regulation.

Differential transcriptional regulation of Elov1 gene family members suggests that they are involved in divergent synthetic pathways [40,106], putatively by affecting different cellular fatty acid pools in response to various stimuli (Fig. 3). The balance between different degrees of unsaturation and chain length within the fatty acid pool is important for transcriptional regulation of metabolic events [44,77,81,94]. Different cellular stimuli which have been shown to alter the expression of Elov1 genes, are briefly discussed in a metabolic context below.

As noted above, ELOVL1 is suggested to control VLCFA synthesis up to 26:0 for e.g., sphingolipid formation [73,99]. The gene is highly expressed in tissues possessing a barrier function such as stomach, lung,

![Diagram](attachment://image.png)

**Fig. 3. Regulation of Elov1 expression and VLCFA synthesis.** A schematic picture based on data on regulation of Elov1 expression in adipose tissue and liver. Elov11 and Elov16 expression are controlled in a lipogenic fashion similar to fatty acid synthase (Fas) by factors such as liver X receptor (LXR) and sterol regulatory element-binding protein 1 (SREBP-1). In contrast, Elov13 expression is induced by factors stimulating fatty acid oxidation, such as NE, while it is inhibited by LXR. Both Elov12 and Elov15 control the elongation of polyunsaturated fatty acids (PUFAs) which in turn regulate the activity of e.g., PPARα. Under certain conditions, such as starvation, specific PUFAs via PPARα promote the synthesis of enzymes involved in fatty acid oxidation, while inhibiting lipogenesis via the active, nuclear form of SREBP-1 (nSREBP-1). Stimulation of PPARα has also been shown to promote the expression of Elov11, Elov13, Elov15 and Elov16, but not Elov12, in a more long-term perspective. Glucocorticoids (GC) are required only for induced ELOVL3 expression.
kidney, skin and myelinated nerve tissue in CNS. However, there exist no data on how *Elovll* expression is regulated in these tissues. We have earlier suggested a tight coupling between *Elovll* expression and control of cell proliferation, based on the genomic tail-to-tail arrangement of *Elovll* and *p55cdc* (a cell cycle gene involved in proliferative processes) [1], which is in agreement with functional data implying that *Elovll* is involved in membrane formation [73, 99].

In liver, Wang et al. [106] have shown that *Elovll* expression is induced in conjunction with increased expression of *Elovl5* and *Elovl6* after administration of the PPARα agonist Wy14,643 to rats, which suggests a nutritional regulation of the gene in certain tissues. However, in the same study, no effect was seen on *Elovll* expression either in fasted animals or in animals refed with olive oil or fish oil. In contrast, there was a clear induction of *Elovl6* and *Fas* after refeeding. This is in line with earlier data showing that *Elovl6* and *Fas* are target genes for sterol regulatory element binding protein-1c (SREBP-1c) and the liver X receptor α (LXRα) [6, 52, 58, 60, 68], which regulate fatty acid synthesis in a lipogenic manner [41, 53, 80].

Regulation of genes involved in saturated and monounsaturated fatty acid synthesis mainly for energy storage, such as *Fas*, steroyl-CoA desaturase (*Scd1*) and acetyl-CoA carboxylase (*Acc*), has been extensively investigated. Food deprivation suppresses the synthesis of FAS protein, while refeeding with high carbohydrate diet induces the synthesis [76, 88]. This is mainly mediated by SREBP-1, partly as a downstream target of LXRα [14, 36, 37, 105, 112–114]. The connection between ELOVL-regulated fatty acid elongation, *Fas* expression and feeding patterns becomes interesting in this respect.

In 2001, Moon et al. [68] described the identification of *Elovl6*, based on its high expression in transgenic mice overexpressing SREBP-1. This led us to investigate if similar regulatory mechanisms applied to other members of the VLCFA elongase family, by stimulating the upstream transcriptional regulator of SREBP-1, LXRα, with the synthetic agonist TO901317 [82] as a stimulant of lipogenic processes. While the LXR agonist had a stimulatory effect on the mRNA levels of *Elovll*, *Elovl6* and *Fas*, in agreement with the literature, the opposite was seen for *Elovl3* [40]. Coherent with this observation, a suppressive effect was seen on the LXR-induced expression of *Elovll*, *Elovl6* and *Fas* in response to a cocktail of agents including norepinephrine and the synthetic glucocorticoid dexamethasone, which induce *Elovl3* expression. In conjunction with this, *Elovl3* expression displayed similar alterations in response to the PPARα agonist Wy-14643 [40], as reported by Wang et al. for *Elovll*, 5 and 6 [106]. The mechanistic basis for this is still unclear but LXR- and PPARα-activation appear to exert antagonistic transcriptional regulation of each others’ target genes, putatively by forming atypical heterodimers [30, 64]. In addition, Weinhofer et al. recently showed that LXRα can act as a negative modulator of transcription by acting through a novel regulatory mechanism involving overlapping SREBP and LXRα binding sites [107]. This is of particular interest since the suppression of *Elovl3* by the LXR agonist does not involve SREBP-1 [40]. This is in line with data showing that LXRs are able to regulate processes involving both energy dissipation and fat storage in response to dietary stimuli [46]. However, the induction by Wy14,643 of *Elovl* expression in liver and BAT is probably not a direct effect of PPARα activity but rather due to an adaptive response to changes in lipid metabolism by Wy14,643.

The massive induction of *Elovl3* during BAT activation can in part be mimicked by continuous in vivo administration of norepinephrine, which leads to increased lipolysis and fatty acid oxidation. This suggests an important role for adrenergic stimuli in the transcriptional regulation of BAT *Elovl3* expression, which may be coupled to intense fatty acid turnover. The importance of different VLCFA elongases may subsequently change in brown adipocytes during processes of augmented fatty acid combustion, as cells need to replenish specific fatty acids. The in vivo data are complemented by results from cultured primary brown adipocytes where chronic treatment of differentiated cells with NE for three days results only in a small increase of *Elovl3* mRNA, much lower than that in cold-stimulated animals. However, in combination with the synthetic glucocorticoid dexamethasone and a PPARα agonist, norepinephrine strongly potentiated the *Elovl3* expression, along with induced expression of enzymes controlling fatty acid β-oxidation [40].

### 4. ELOVL and PUFA Synthesis

Several nuclear receptors have been implicated in various aspects of lipid metabolism, including complex cross-talk between fatty acid ligands and the receptors which govern a “higher” level of metabolic feedback control [15, 23].
Various types of fatty acids have been shown to activate PPARα, primarily to initiate catabolic events. Particularly PUFAs (e.g., linoleic acid) and their derivatives can function as high affinity ligands for PPARs (reviewed in [23]), while the equilibrium between free fatty acids and their CoA-esters is suggested to be an important determinant of the activation process, as non-hydrolysable acyl-CoA analogs show antagonistic properties in binding and activation studies [27]. The fatty acid composition of stored lipids in adipose tissue directly reflects the long-term diet [47], such that the release of potential PPAR high affinity ligands (e.g., PUFAs) as a response to, for example hormonal stimuli, may vary with diet composition. In addition, prolonged energy deprivation in mammals will lead to a gradual shift in fuel utilization from carbohydrates and fat in the fed state to nearly only fat after fasting, a mechanism controlled by PPARα [101].

In the liver of PPARα-null mice, in addition to impaired fatty acid β-oxidation, fatty acid uptake is diminished [50]. The mechanism behind this is unclear, although, it may involve altered regulation of angiopoietin-like proteins such as the fasting-induced-adipose-factor (FIAF) which has been shown to regulate fatty acid uptake in liver and adipose tissue and is a PPARα target gene [48, 49]. Recent data on the connection between control of lipogenesis and regulation of PUFA synthesis by ELOVLs have given more insight into the topic. It is known that both insulin and LXR agonists increase the nuclear content of SREBP-1 (nSREBP-1) and lipogenesis [82, 84], while dietary PUFA such as 20:5 (n–3) (EPA) and 22:6 (n–3) (DHA) suppress lipogenesis via phosphorylation of nSREBP-1 and a concomitantly increased nSREBP-1 breakdown in liver [10, 35]. Overexpression of ELOVL2 in cultured primary hepatocytes enhances 20:5(n–3) conversion to 24:5(n–3), which leads to reduced non-esterified 20:5 an thus attenuates 20:5 (n–3)-mediated induction of PPARα-regulated transcripts and enhanced SREBP-1c nuclear content [45]. In liver, ELOVL2 has a limited substrate specificity and is constitutively expressed at low levels while ELOVL5 is highly expressed is in adult rats and uses a broader spectrum of n–3 PUFAs. In addition, Wang et al. recently showed that ELOVL5 is co-ordinately regulated along with Δ5 and Δ6 desaturases during development and by diet suggesting that ELOVL5 is responsible for the nutritional control of PUFA elongation in rat liver [106].

### 5. General conclusions

The physiological significance of VLCFA is indisputable [24, 83, 91] and there are a number of mammalian disorders known to be related to abnormal levels of VLCFA, such as peroxisomal disorders involving the adrenal cortex and the nervous system, myelin deficiency and several types of skin disorders, metabolic disorders, etc. [61, 70, 96, 97]. There are also several pathological situations, such as e.g., tumor invasion, which are found to have a disturbed metabolism of sphingolipids (e.g., gangliosides) and ceramides, of which VLCFA are a part [33, 78, 89].

Similar to what is seen for FAS, regulation of ELOVL-mediated fatty acid biosynthesis is exerted in concert with the metabolic status of the organism through hormonal signalling and metabolite feed-back [31, 111].

A positive role for PUFAs in the field of dyslipidemia and diabetes has been suggested from several studies, and the involvement of VLCFA elongation enzymes in PUFA metabolism was recently reviewed by Leonard et al. [56]. Additional data since then emphasize the importance of PUFA elongases in the control of lipid metabolism [45, 106]. Interestingly, saturated and monounsaturated VLCFAs also appear to exert exciting regulatory effects on cellular metabolism, which may be coupled to structural functions when incorporated into specific lipids, or to allosteric regulation of enzymatic activities [42, 85]. This is also stressed by analysis of the elongases ELOVL1 and ELOVL3, which are suggested to be involved in membrane and triglyceride formation, respectively [99, 109].

However, although the general function of the *Elolv* genes is partially understood, very little is known about the role of fatty acids of specific chain length in cellular and developmental processes in a physiological context. The concept “essential” fatty acids needs to be considered in the sense of what kinds of lipids do we really have to synthesize ourselves, for what purpose and how is this synthesis regulated? Furthermore, what impact do dietary fatty acids have on this regulation?

Generation of FAS null mice, resulting in embryonic lethality, has proven that the enzyme complex is essential very early in development [17]. So far, mutations in *Elolv* genes only exist for *Elolv3* and *Elolv4*, which do not influence viability. As the ELOVL enzymes are putative key enzymes in metabolic processes, the diversity in transcriptional regulation of *Elolv* genes may provide some hints as to the physiological functions
of downstream fatty acid products in mammals. For example, EST mouse databases show that Elovl1 is already expressed in the unfertilised egg, which implies that this ubiquitously expressed elongase is essential for life, while those with a more infrequent expression profile, such as Elovl3 and Elovl4, have a more “subordinate” role. Although this has still to be proven, alterations in fatty acid profile as a consequence of Elovl gene deletion, for example as those seen in Elovl3 and Elovl4 mutant mice [110, 115], emphasize the importance of these enzymes in maintaining fatty acid homeostasis for proper cellular functioning under certain conditions in life. In addition, the data obtained from the brown fat of Elovl3-ablated mice housed at thermoneutral temperature, clearly demonstrate that even a minor amount of expression may have significant impact on tissue function.

In summary, spatial- and temporal-specific expression of VLCFA elongases may provide cells with tools for the fine-tuning of metabolically related processes at several levels. Changes in membrane properties, towards more-or-less ordered states depending on the mean acyl chain length, will alter the activity of trans-membrane transport, as well as signaling events. In addition, alterations in the synthesis of certain VLCFAs may exert feed-back regulation on cellular function through nuclear receptors.

Although the key enzymes are now identified, we are still at an early stage in understanding how these enzymes function and how they are regulated. The general assumption is that each elongase performs the condensation step in the fatty acid elongation cycle by having explicit substrate and product specificity in order to support/fulfill the function of the cell. However, most experiments performed in order to delineate the function of an elongase show a broad spectrum of substrate preferences as well as of end products. In order to gain a more detailed understanding of the function of the enzymes, the proteins have to be further analysed in detail.

Acknowledgments

We are grateful to Drs. Barbara Cannon and Jan Nedergaard for critical reading of the manuscript.

References

[1] Asadi A, Jørgensen J, Jacobsson A. J Biol Chem 2002; 277: 18494–500.

[2] Baht HS, Saggerson ED. Biochem J 1989; 258: 369–73.

[3] Bauer F, Urdaci M, Aigle M, Crouzet M. Mol Cell Biol 1993; 13: 5070–84.

[4] Beaudoin F, Gable K, Sayanova O, Dunn T, Napier JA. J Biol Chem 2002; 277: 11481–8.

[5] Beaudoin F, Michaelson LV, Hey SJ, Lewis MJ, Shewry PR, Sayanova O, et al. Proc Natl Acad Sci USA 2000; 10: 6421–6.

[6] Bennett MK, Lopez JM, Sanchez HB, Osborne TF. J Biol Chem 1995; 270: 25578–83.

[7] Bernert JJ, Sprecher H. J Biol Chem 1979; 254: 11584–90.

[8] Bernert JT, Sprecher H. J Biol Chem 1977; 252: 6736–44.

[9] Bernert JT, Sprecher H. Biochim Biophys Acta 1978; 531: 44–55.

[10] Botolin D, Wang Y, Christian B, Jump DB. J Lipid Res 2006; 47: 181–92.

[11] Bourre JM, Daudu O, Baumann N. Biochim Biophys Acta 1976; 424: 1–7.

[12] Bourre JM, Patureau JM, Daudu OL, Baumann NA. Eur J Biochem 1977; 72: 41–7.

[13] Brooks BJ, Arch JR, Newsholme EA. Biosci Rep 1983; 3: 263–7.

[14] Brown MS, Goldstein JL. Cell 1997; 89: 331–40.

[15] Chawla A, Repa JJ, Evans RM, Mangelsdorf DJ. Science 2001; 294: 1866–70.

[16] Chen HC, Smith SJ, Tow B, Elias PM, Farese Jr RV. J Clin Invest 2002; 109: 175–81.

[17] Chirala SS, Chang H, Matzuk M, Abu-Elheiga L, Mao J, Mahon K, et al. Proc Natl Acad Sci USA 2003; 100: 6358–63.

[18] Chirala SS, Huang WY, Jayakumar A, Sakai K, Wakil SJ. Proc Natl Acad Sci USA 1997; 94: 5588–93.

[19] Cinti DL, Cook L, Nagi MN, Suneja SK. Prog Lipid Res 1992; 31: 1–51.

[20] Colli W, Hinkle PC, Pullman ME. J Biol Chem 1969; 244: 6432–43.

[21] David D, Sundarababu S, Gerst JE. J Cell Biol 1998; 143: 1167–82.

[22] Desfarges L, Durrens P, Juguelin H, Cassagne C, Bonneu M, Aigle M. Yeast 1993; 9: 267–77.

[23] Desvergne B, Wahli W. Endocr Rev 1999; 20: 649–88.

[24] Driver PJ, Lemp MA. Surv Ophthalmol 1996; 40: 343–67.

[25] Durrens P, Revardel E, Bonneu M, Aigle M. Curr Genet 1995; 27: 213–6.

[26] El-Sherbeini M, Clemas JA. J Bacteriol 1995; 177: 3227–34.

[27] Elholm M, Dam I, Jorgensen C, Krogsdam AM, Holst D, Kratchmarova I, et al. J Biol Chem 2001; 276: 21410–6.

[28] Fujita N, Sato S, Kurihara T, Inuzuka T, Takahashi Y, Miyatake T. FEBS Lett 1988; 232: 323–7.
[29] Garcia-Arranz M, Maldonado AM, Mazon MJ, Portillo F. J Biol Chem 1994; 269: 18076–82.
[30] Gbaguidi GF, Agellon LB. Biochim Biophys Acta 2002; 1583: 229–36.
[31] Goodridge AG. J Biol Chem 1973; 248: 4318–26.
[32] Grayson C, Molday RS. J Biol Chem 2005; 280: 32521–30.
[33] Hannun YA, Obeid LM. Trends Biochem Sci 1995; 20: 73–7.
[34] Herron BJ, Bryda EC, Heverly SA, Collins DN, Flaherty L. Mamm Genome 1999; 10: 864–9.
[35] Hirano Y, Yoshida M, Shimizu M, Sato R. J Biol Chem 2001; 276: 36431–7.
[36] Hua X, Nohturfft A, Goldstein JL, Brown MS. Cell 1996; 87: 415–26.
[37] Hua X, Sakai J, Brown MS, Goldstein JL. J Biol Chem 1996; 271: 10379–84.
[38] Hummasti S, Laffitte BA, Watson MA, Galardi C, Chao LC, Ramamurthy L, et al. J Lipid Res 2004; 45: 616–25.
[39] Inagaki K, Aki T, Fukuda Y, Kawamoto S, Shigeta S, Ono K, et al. Biosci Biotechnol Biochem 2002; 66: 613–21.
[40] Jakobsson A, Jorgensen JA, Jacobsson A. Am J Physiol Endocrinol Metab 2005; 289: E517–26.
[41] Janowski BA, Willy PJ, Devi TR, Falck JR, Mangelsdorf DJ. Nature 1996; 383: 728–31.
[42] Jepson CA, Yeaman SJ. FEBS Lett 1992; 310: 197–200.
[43] Johnston IG, Rush SJ, Gurd JW, Brown IR. J Neurosci Res 1992; 32: 159–66.
[44] Jump DB, Botolin D, Wang Y, Xu J, Christian B, Demeure O. J Nutr 2005; 135: 2503–6.
[45] Jump DB, Wang Y, Botolin D, Xu J, Demeure O, Busik J, et al. Chem Phys Lipids 2005; 136: 86–7.
[46] Kalaany NY, Gauthier KC, Zavacki AM, Mammen PPA, Kitazume T, Peterson AJ, et al. Cell Metab 2005; 1: 231–43.
[47] Katan MB, van Birgelen A, Deslypere JP, Penders M, van Staveren WA. Ann Nutr Metab 1991; 35: 249–52.
[48] Kersten S. Biochem Soc Trans 2005; 33: 1059–62.
[49] Kersten S, Mandard S, Tan NS, Escher P, Metzger D, Chambon P, et al. J Biol Chem 2000; 275: 28488–93.
[50] Kersten S, Seydoux J, Peters JM, Gonzalez FJ, Desvergne B, Wahli W. J Clin Invest 1999; 103: 1489–98.
[51] Kohlwein SD, Eder S, Oh C-S, Martin CE, Gable K, Bacikova D, et al. Mol Cell Biol 2001; 21: 109–25.
[52] Latasa MJ, Moon YS, Kim KH, Sul HS. Proc Natl Acad Sci USA 2000; 97: 10619–24.
[53] Lehmann JM, Lenhard JM, Oliver BB, Ringold GM, Kliewer SA. J Biol Chem 1997; 272: 3406–10.
[54] Leonard AE, Bobik EG, Dorado J, Kroeger PE, Chuang L-T, Thurmond JM, et al. Biochem J 2000; 350: 765–70.
[55] Leonard AE, Kelder B, Bobik EG, Chuang LT, Lewis CJ, Kopchick JJ, et al. Lipids 2002; 37: 733–40.
[56] Leonard AE, Pereira SL, Sprecher H, Huang YS. Prog Lipid Res 2004; 43: 36–54.
[57] Lindberg O, Bieber LL, Houstek J. In: Jansky L, Musacchia XJ, editors. Regulation of depressed metabolism and thermogenesis. Ohio: Thomas Springfield; 1976. p. 117–36.
[58] Magana MM, Osborne TF. J Biol Chem 1996; 271: 32689–94.
[59] Mandal MN, Ambasudhan R, Wong PW, Gage PJ, Sieving PA, Ayyagari R. Genomics 2004; 83: 626–35.
[60] Matsuzaka T, Shimano H, Yahagi N, Yoshikawa T, Amemiya-Kudo M, Hasty AH, et al. J Lipid Res 2002; 43: 911–20.
[61] McGuinness MC, Lu JF, Zhang HP, Dong GX, Heinzer AK, Watkins PA, et al. Mol Cell Biol 2003; 23: 744–53.
[62] Michelsen K, Yuan H, Schwappach B. EMBO Rep 2005; 6: 717–22.
[63] Millar AA, Kunst L. Plant J 1997; 12: 121–31.
[64] Miyata KS, McCaw SE, Patel HV, Rachubinski RA, Capone JP. J Biol Chem 1996; 271: 9189–92.
[65] Monne M, Gafvelin G, Nilsson R, von Heijne G. Eur J Biochem 1999; 263: 264–9.
[66] Monne M, Hessa T, Thissen L, von Heijne G. FEBS J 2005; 272: 28–36.
[67] Moon YA, Horton JD. J Biol Chem 2003; 278: 7335–43.
[68] Moon YA, Shah NA, Mohapatra S, Warrington JA, Horton JD. J Biol Chem 2001; 276: 45358–66.
[69] Moore S, Hurt E, Yoder E, Sprecher H, Spector A. J Lipid Res 1995; 36: 2433–43.
[70] Mosser J, Douar AM, Sarde CO, Kioschis P, Feil R, Moser H, et al. Nature 1993; 361: 726–30.
[71] Nave KA, Bloom FE, Milner RJ. J Neurochem 1987; 49: 1873–7.
[72] Nugteren DH. Biochim Biophys Acta 1965; 106: 280–90.
[73] Oh CS, Toke DA, Mandala S, Martin CE. J Biol Chem 1997; 272: 17376–84.
[74] Osei P, Suneja SK, Laguna JC, Nagi MN, Cook L, Prasad MR, et al. J Biol Chem 1989; 264: 6844–9.
[75] Pan X, Roberts P, Chen Y, Kvam E, Shulga N, Huang K, et al. Mol Biol Cell 2000; 11: 2445–57.
[76] Paulauskis JD, Sul HS. J Biol Chem 1988; 263: 7049–54.
[77] Pegorier JP, Le May C, Girard J. J Nutr 2004; 134: 2444S–9S.
[78] Pettus BJ, Chalfant CE, Hannun YA. Biochim Biophys Acta 2002; 1585: 114–25.
[79] Raclot T. Prog Lipid Res 2003; 42: 257–88.
[80] Repa JJ, Liang G, Ou J, Bashmakov Y, Lobaccaro JM, Shimomura I, et al. Genes Dev 2000; 14: 2819–30.
[81] Sampath H, Ntambi JM. Annu Rev Nutr 2005; 25: 317–40.
[82] Schultz JR, Tu H, Luk A, Repa JJ, Medina JC, Li L, et al. Genes Dev 2000; 14: 2831–8.
[83] Sharp P, Johnson D, Poulos A. J Neurochem 1991; 56: 30–7.
[84] Shimomura I, Bashmakov Y, Ikemoto S, Horton JD, Brown MS, Goldstein JL. Proc Natl Acad Sci USA 1999; 96: 13656–61.
[85] Simons K, Ikonen E. Nature 1997; 387: 569–72.
[86] Simons K, Toomre D. Nat Rev Mol Cell Biol 2000; 1: 31–9.
[87] Smith S. FASEB J 1994; 8: 1248–59.
[88] Soncini M, Yet SF, Moon Y, Chun JY, Sul HS. J Biol Chem 1995; 270: 30339–43.
[89] Spiegel S, Merrill AJ. FASEB J 1996; 10: 1388–97.

[90] Sprecher H. Biochim Biophys Acta 1974; 360:113–23.
[91] Stewart ME, Downing DT. Adv Lipid Res 1991; 24:263–301.
[92] Suneja SK, Nagi MN, Cook L, Cinti DL. J Neurochem 1991; 57:140–6.
[93] Sweet H, Lane P. Mouse News Lett 1978; 59:287.
[94] Thewke DP, Panini SR, Sinensky M. J Biol Chem 1998; 273:21402–7.
[95] Toke DA, Martin CE. J Biol Chem 1996; 271:18413–22.
[96] Tsuji S, Ohno T, Miyatake T, Suzuki A, Yamakawa T. J Biochem (Tokyo) 1984; 96:1241–7.
[97] Tsuji S, Sano T, Ariga T, Miyatake T. J Biochem (Tokyo) 1981; 90:1233–6.
[98] Tvrdik P, Asadi A, Kozak LP, Nedergaard J, Cannon B, Jacobsson A. J Biol Chem 1997; 272:31738–46.
[99] Tvrdik P, Westerberg R, Silve S, Asadi A, Jakobsson A, Cannon B, et al. J Cell Biol 2000; 149:707–17.
[100] Umeda S, Ayyagari R, Suzuki MT, Ono F, Iwata F, Fujiki K, et al. Exp Anim 2003; 52:129–35.
[101] van den Berghe G. J Inherit Metab Dis 1991; 14:407–20.
[102] Voss A, Reinhart M, Sankarappa S, Sprecher H. J Biol Chem 1991; 266:19995–20000.
[103] Wakil SJ. Biochemistry 1989; 28:4523–30.
[104] Wakil SJ, Stoops JK, Joshi VC. Annu Rev Biochem 1983; 52:537–79.
[105] Wang X, Sato R, Brown MS, Hua X, Goldstein JL. Cell 1994; 77:53–62.
[106] Wang Y, Botolin D, Christian B, Busik J, Xu J, Jump DB. J Lipid Res 2005; 46:706–15.
[107] Weinhofer I, Kunze M, Rampler H, Bookout AL, Forss-Petter S, Berger J. J Biol Chem 2005; 280:41243–51.
[108] Wertz PW, Downing DT. J Lipid Res 1983; 24:759–65.
[109] Westerberg R, Månsson JE, Golozoubova V, Shabalina IG, Backlund EC, Tvrdik P, et al. J Biol Chem 2006; 281:4958–68.
[110] Westerberg R, Tvrdik P, Undén AB, Månsson JE, Norlén L, Jakobsson A, et al. J Biol Chem 2004; 279:5621–9.
[111] Witters LA, Kemp BE. J Biol Chem 1992; 267:2864–7.
[112] Yabe D, Brown MS, Goldstein JL. Proc Natl Acad Sci USA 2002; 99:12753–8.
[113] Yabe D, Komuro R, Liang G, Goldstein JL, Brown MS. Proc Natl Acad Sci USA 2003; 100:3155–60.
[114] Yang T, Espenshade PJ, Wright ME, Yabe D, Gong Y, Aebersold R, et al. Cell 2002; 110:489–500.
[115] Zhang K, Kniazeva M, Han M, Li W, Yu Z, Yang Z, et al. Nat Genet 2001; 27:89–93.
[116] Zhang XM, Yang Z, Karan G, Hashimoto T, Baehr W, Yang XJ, et al. Mol Vis 2003; 9:301–7.
