
APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Apr. 2006, p. 2491–2495  
0099-2240/06/$08.00+0 doi:10.1128/AEM.72.4.2491–2495.2006  
Copyright © 2006, American Society for Microbiology. All Rights Reserved.

**Purification and Characterization of TrzF: Biuret Hydrolysis by Allophanate Hydrolase Supports Growth**

Nir Shapir,$^{1,2}$† Gang Cheng,$^{2,3,4}$† Michael J. Sadowsky,$^{2,3,4}$ and Lawrence P. Wackett$^{1,2,3*}$

Department of Biochemistry, Molecular Biology and Biophysics,$^{1}$ BioTechnology Institute,$^{2}$ Center for Microbial and Plant Genomics,$^{3}$ and Department of Soil, Water and Climate,$^{4}$ University of Minnesota, St. Paul, Minnesota 55108

Received 1 November 2005/Accepted 21 January 2006

TrzF, the allophanate hydrolase from *Enterobacter cloacae* strain 99, was cloned, overexpressed in the presence of a chaperone protein, and purified to homogeneity. Native TrzF had a subunit molecular weight of 65,401 and a subunit stoichiometry of α₂ and did not contain significant levels of metals. TrzF showed time-dependent inhibition by phenyl phosphorodiamidate and is a member of the amidase signature protein family. TrzF was highly active in the hydrolysis of allophanate but was not active with urea, despite having been previously considered a urea amidolyase. TrzF showed lower activity with malonamate, malonamide, and biuret. The allophanate hydrolase from *Pseudomonas* sp. strain ADP, AtzF, was also shown to hydrolyze biuret slowly. Since biuret and allophanate are consecutive metabolites in cyanuric acid metabolism, the low level of biuret hydrolase activity can have physiological significance. A recombinant *Escherichia coli* strain containing *atzD*, encoding cyanuric acid hydrolase that produces biuret, and *atzF* grew slowly on cyanuric acid as a source of nitrogen. The amount of growth produced was consistent with the liberation of 3 mol of ammonia from cyanuric acid. In vitro, TrzF was shown to hydrolyze biuret to liberate 3 mol of ammonia. The biuret hydrolyzing activity of TrzF might also be physiologically relevant in native strains. *E. cloacae* strain 99 grows on cyanuric acid with a significant accumulation of biuret.

Genome sequencing projects and recent biochemical investigations have demonstrated that allophanate hydrolase is widely distributed in taxonomically diverse bacteria (3, 14, 18, 27). Previously, allophanate hydrolase was known in yeast and algae (15, 29) but was unknown in prokaryotes. It has now been determined that allophanate hydrolase has at least two distinct metabolic roles in bacteria. It functions in concert with a urea carboxylase to metabolize urea after its activation by carboxylation (17). This two-enzyme sequence provides bacteria with an alternative to urease that hydrolyzes urea directly to 2 mol of ammonia (14). Allophanate hydrolase also functions in the bacterial metabolism of cyanuric acid (3, 27). Recently, a number of cyanuric acid metabolizing bacteria have been shown to hydrolytically open the s-triazine ring of cyanuric acid to produce biuret, catalyzed by AtzD or TrzD (Fig. 1). It had been proposed that biuret was hydrolyzed to urea in different bacteria (4, 5, 8), but a more recent study showed that biuret hydrolysis in these bacteria produced allophanate (3). Allophanate was subsequently cleaved by an allophanate hydrolase. Thus, all three nitrogen atoms of the s-triazine ring are liberated to support bacterial growth on cyanuric acid as the sole nitrogen source.

One bacterium that has been demonstrated to grow on cyanuric acid, *Enterobacter cloacae* strain 99, was shown to have an allophanate hydrolase activity, but this strain was unusual in that it accumulated significant levels of biuret in the growth medium (4). The gene regions encoding cyanuric acid hydrolase (TrzD) and urea amidolyase have been reported previously (GenBank accession number AF342826), but no biuret hydrolase gene or enzyme has been found to date in that bacterium.

In the present work, the allophanate hydrolase from *E. cloacae* strain 99 was cloned with a chaperone protein and then purified and characterized. It was comparable in terms of a number of properties to AtzF from *Pseudomonas* sp. strain ADP (27) and malonamidase from *Bradyrhizobium japonicum*, a protein catalyzing a similar reaction with a structurally analogous substrate, malonamate (20, 28). Surprisingly, we found that purified, recombinant TrzF and AtzF proteins each showed a low, but significant, biuret hydrolase activity. In recombinant *Escherichia coli*, the biuret hydrolase activity was shown to function physiologically with cyanuric acid hydrolase to liberate ammonia from cyanuric acid and support growth.

---

**MATERIALS AND METHODS**

Chemicals. Potassium allophanate was prepared from diethyl allophanate (Acros Organics, Pittsburgh, PA) as described previously (31). Acetylurea was purchased from Lancaster Synthesis, Inc. (Pelham, NH), and all other commercially available compounds were purchased from Aldrich Chemical Co. (Milwaukee, WI).

Enzymatic assays. The amount of ammonia released from allophanate and related compounds was determined using the Berthelot reaction as described previously (30). One unit of enzyme activity was defined as the amount of enzyme converting 1 μmol of allophanate to 2 μmol of ammonia in 1 min.

Construction of recombinant plasmid for the expression of TrzF with a His tag. The putative urea amidolyase gene from *E. cloacae* strain 99 (GenBank accession number AF342826) was amplified, without its native promoter, by PCR using the primers trzFf-His (5′-CGGCTAGCATGAATTTGACAATTACA-3′) and trzFr-His (5′-CGAAGCCTTCTATTGATCGGCAAGA-3′). Primers were synthesized by Integrated DNA Technologies (Coralville, IA). The primers contained NheI and HindIII restriction enzyme cloning sites at their 5′ and 3′ ends, respectively. The gene was cloned into vector pET28b+ (Novagen, Madison, WI), downstream of the *lac* promoter and His tag coding sequence, to create plasmid pGC1. Plasmid pGC1 was transformed into *E. coli* BLR(DE3) (Novagen, Madison, WI), and the sequence of the cloned gene was verified.

* Corresponding author. Mailing address: Department of Biochemistry, Molecular Biology and Biophysics, 140 Gortner Lab, 1479 Gortner Ave., University of Minnesota, St. Paul, MN 55108. Phone: (612) 625-3785. Fax: (612) 625-5780. E-mail: wacke003@umn.edu.
† N.S. and G.C. contributed equally to this work.

Pseudomonas sp. ADP

AtzD

AtzE

AtzF

Cyanuric Acid

Biuret

Allophanate

Enterobacter cloacae 99

TrzD

Unknown

TrzF

FIG. 1. Cyanuric acid catabolic pathway in *Pseudomonas* sp. strain ADP and *E*. *cloacae* 99.

Expression and purification of TrzF. Plasmid pAG containing the chaperone genes *groEL* and *groES* (24) was transformed into *E*. *coli* BLR(DE3)(pGC1) in order to increase the solubility of TrzF. *E*. *coli* BLR(DE3)(pAG, pGC1) was grown in LB medium (25) containing 50 μg of kanamycin and 30 μg of chloramphenicol per ml at 15°C with shaking at 150 rpm. L-Arabinose (0.0015%) was added when cultures reached an optical density of 0.5 at 600 nm, and 1 mM IPTG (isopropyl-β-d-thiogalactopyranoside) was added after an additional 90 min of incubation. Induced cells were grown for an additional 16 h under the same conditions. Cultures were centrifuged at 10,000 × g for 10 min at 4°C and washed three times with 0.85% NaCl, and cell pellets were resuspended in 20 ml of 50 mM 3-(N-morpholino)propanesulfonate (MOPS) buffer, pH 7.4. All buffers contained 10% glycerol. Cells were broken by passage three times through a chilled French pressure cell operated at 140 MPa. Cell extracts were obtained by centrifugation at 18,000 × g for 90 min at 4°C. Lysates were applied to a 5-ml HisTrap chelating column (Amersham Pharmacia Biotech, Piscataway, NJ) complexed with Ni²⁺ according to the manufacturer's instructions. The column was washed with 15 ml of 50 mM MOPS buffer, pH 7.4, followed by a second wash with the same buffer supplemented with 0.125 M imidazole. The enzyme was eluted from the column with 15 ml of 0.25 M imidazole in 50 mM MOPS buffer, pH 7.4. The purified enzyme was concentrated using a Centricon-30 filtration unit (Amicon, Beverly, MA), and enzyme purity and subunit molecular weight were estimated via sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) (25).

Growth of *E*. *coli* containing the *atzD* and *atzF* genes on minimal medium with cyanuric acid as a nitrogen source. An *E*. *coli* strain containing the *atzD* and *atzF* genes from *Pseudomonas* sp. strain ADP was constructed by transforming plasmid pKT230::*atzD* (11, 21) into *E*. *coli* BLR(DE3) containing pET28b+::*atzF* (27). The ability of this recombinant *E*. *coli* to use AtzF to grow on biuret as a nitrogen source (produced by the hydrolysis of cyanuric acid by AtzD) was measured. Starter cultures grown in LB medium were centrifuged at 10,000 × g at 4°C, washed three times in 0.85% NaCl, and inoculated into 100 ml of minimal R medium (10) containing 1 mM proline and supplemented with 5 mM cyanuric acid as the nitrogen source. The culture was incubated at 30°C, with shaking at 150 rpm, and growth was measured by following optical density at 600 nm (OD₆₀₀). Cultures of *E*. *coli* containing each of the plasmids separately were used as negative controls. Minimal medium without cyanuric acid served as a control to measure the contribution of proline as a nitrogen source for growth. Medium containing 15 mM ammonium chloride, providing an equivalent amount of nitrogen as 5 mM cyanuric acid, was used to measure the optimum growth potential of the recombinant strain.

RESULTS

TrzF optimized expression and purification. TrzF activity was 0.06 umol/min per mg of protein in initial clones containing plasmid pGC1. Subsequent transformation of plasmid pGC1 into *E*. *coli* BLR(DE3) expressing the chaperone proteins GroEL and GroES led to a 20-fold increase in allophanate hydrolase activity in crude lysates. In crude extracts, a major protein band was observed by SDS-PAGE that migrated slightly more slowly than the intense high-molecular-weight chaperone protein. In initial purification efforts, significant amounts of the chaperone protein migrated with TrzF. Ultimately, conditions were found to elute the His-tagged TrzF uniquely from a nickel column to yield a homogeneous protein with a 10-fold purification and a 20% yield. Yield was sacrificed to obtain purity.

Physical characterization of TrzF. The purified TrzF protein appeared homogeneous by SDS-PAGE, with an estimated subunit molecular weight of 68,000. Based on translation of the gene sequence, the native TrzF protein without the His tag was calculated to have a subunit molecular weight of 65,401 (Table 1). This is very similar to the subunit molecular weight of AtzF and allophanate hydrolase from *Oleomonas sagaranensis*. The protein had a calculated pI of 5.1, comparable to other allophanate hydrolases. Gel filtration indicated a holoenzyme molecular weight of 144,000, indicating a subunit stoichiometry of α₂, similar to malonamide. No transition metals were observed; a stoichiometry of <0.1 metal per subunit was observed for Al, B, Ca, Cd, Co, Cr, Cu, Fe, K, Mg, Mn, Na, Ni, Pb, and Zn.

Characterization of TrzF enzyme activity. TrzF was incubated with potassium allophanate at pH 7.4, and the specific activity was found to be 9.3 μmol per min per mg protein. The *k*~cat~ was 6 s⁻¹, the *K*~m~ was 0.68 mM, and the *k*~cat~/*K*~m~ was 8.8 ×

VOL. 72, 2006                                                                 BIURET HYDROLYSIS BY ALLOPHANATE HYDROLASE  2493

TABLE 1. Comparison of physicochemical properties of TrzF, AtzF, malonamidase, and allophanate hydrolase from *O. sagaranensis*

| Property           | TrzF<sup>a</sup> | AtzF<sup>b</sup> | MAE2<sup>c</sup> | OSAH<sup>d</sup> |
|--------------------|-----------------|------------------|------------------|------------------|
| Mol wt            | 65,401          | 64,058           | 43,682           | 61,999           |
| Subunit stoichiometry | α₂             | α₄               | α₂               | α₂               |
| pI                | 5.1             | 5.3              | 7.8              | 4.8              |
| Metal             | None            | None             | ND<sup>e</sup>   | ND               |
| K<sub>m</sub> (mM) | 0.68            | 1.5              | 10               | 0.04             |
| k<sub>cat</sub>/K<sub>m</sub> (s⁻¹ M⁻¹) | 8.8 × 10³        | 1.1 × 10⁴        | ND<sup>f</sup>   | 2.7 × 10⁶         |
| PPDA<sup>g</sup> inhibition | Time dependent | Time dependent  | Covalent<sup>h</sup> | ND               |

<sup>a</sup> Allophanate hydrolase from *E. cloacae* strain 99.  
<sup>b</sup> Allophanate hydrolase from *Pseudomonas* sp. strain ADP.  
<sup>c</sup> Malonamidase from *Bradyrhizobium japonicum* USDA 110.  
<sup>d</sup> Allophanate hydrolase from *O. sagaranensis*.  
<sup>e</sup> ND, not determined.  
<sup>f</sup> Kinetic parameters are reported for the nonphysiological hydroxylamination reaction catalyzed by malonamidase MAE2, but not the physiological amide hydrolysis reaction.  
<sup>g</sup> PPDA, phenyl phosphorodiamidate.  
<sup>h</sup> Tested with crystals and shown to react with active-site serine.

---

10³ s⁻¹ M⁻¹. This was comparable with values previously determined for AtzF from *Pseudomonas* sp. strain ADP (27), but k<sub>cat</sub> was higher and K<sub>m</sub> was lower with the allophanate hydrolase from *Oleomonas* (Table 1). When the reaction was allowed to proceed to completion for 20 min with limiting concentrations of allophanate, 2.0 mol of ammonia was detected per mol of substrate consumed.

TrzF showed strong time-dependent inhibition with phenyl phosphorodiamidate, a compound shown to covalently modify the homologous enzymes AtzF (27) and malonamidase (28) (Table 1). These data are consistent with the structural relatedness of TrzF with other members of the amidase signature family. With both AtzF and malonamidase, phenyl phosphorodiamidate was shown to react specifically with a single serine residue. Other data have demonstrated that this serine residue acts as the nucleophile that displaces the leaving amino group, a mechanism characteristic of members of the amidase signature family (23, 28).

Nonsubstrates and slow substrates for TrzF. Urea was not observed to be a substrate for TrzF. In addition, the following urea derivatives were not found to be substrates: N-methylurea, acetylene, 1-acetyl-2-thiourea, methyl carbamate, and semicarbazide. The following allophanate analogs were not detectably reactive with TrzF: methyl allophanate, hydantoic acid, and oxamic acid. Thus, TrzF was observed here to have stringent substrate specificity.

Subsequently, other substrates for TrzF were uncovered, but they were all highly isosteric with allophanate, and activity was significantly lower than that observed with allophanate (Table 2). All compounds showing reactivity contained the following string of five functional groups in the precise order: (i) amino group, (ii) carbonyl carbon, (iii) amino group or methylene carbon, (iv) carbonyl carbon, and (v) hydroxyl or amino group.

Allophanate was by far the most reactive compound. However, malonamate was hydrolyzed by TrzF to yield 1 mol of ammonia per mol of substrate consumed, with a specific activity of 22 nmol per min per mg of protein. In contrast, malonamide was hydrolyzed by TrzF to yield 2 mol of ammonia per mol of substrate consumed with a specific activity of 1.5 nmol per min per mg of protein. Biuret was hydrolyzed by TrzF to yield 3 mol of ammonia per mol of substrate consumed (Fig. 2), and it was

---

TABLE 2. Structurally analogous compounds shown to be hydrolyzed by TrzF

| Name             | Structure                                      | Activity (nmol/min/mg) |
|------------------|------------------------------------------------|------------------------|
| Allophanic acid  | ![Allophanic Acid](https://via.placeholder.com/150) | 9,300                  |
| Malonamic acid   | ![Malonamic Acid](https://via.placeholder.com/150) | 22                     |
| Malonamide       | ![Malonamide](https://via.placeholder.com/150)     | 1.5                    |
| Biuret           | ![Biuret](https://via.placeholder.com/150)         | 0.5                    |
| General structure | ![General Structure](https://via.placeholder.com/150) | 0.5–9300               |

hydrolyzed with a specific activity of 0.5 nmol per min per mg of protein.

AtzF also hydrolyzes biuret. We had previously reported that biuret was not a substrate for AtzF, the allophanate hydrolase homolog of TrzF (27), but the low activity of TrzF with biuret observed here caused us to reexamine this. With longer incubation times, we determined that AtzF was similarly active with biuret, showing a specific activity of 1 nmol per min per mg of protein.

Potential physiological relevance of low reactivity of allophanate hydrolase with biuret. *E. cloacae* strain 99 grew with 5 mM cyanuric acid as the sole nitrogen source and accumulated 4.7 mM biuret, slightly less than stoichiometric accumulation. The difference, 0.3 mM cyanuric acid, can give rise to 0.9 mM ammonia, and that is sufficient nitrogen to support substantial growth. However, the accumulation of biuret indicates that it is inefficiently hydrolyzed by *E. cloacae* strain 99. This bacterium did not show a gene that amplified via PCR using *atzE* gene primers (data not shown), and a *trzE* gene has not been identified in this or any other bacterium (Fig. 1).

In this context, we considered the idea that the low level of biuret hydrolase activity conferred by TrzF or AtzF could support slow bacterial growth on biuret since consecutive hydrolysis liberating 3 mol of ammonia by TrzF from *E. cloacae* 99 and AtzF from *Pseudomonas* sp. strain ADP.

FIG. 2. Biuret hydrolysis liberating 3 mol of ammonia by TrzF from *E. cloacae* 99 and AtzF from *Pseudomonas* sp. strain ADP.

TABLE 3. Growth of *E. coli* containing the *atzD* and *atzF* genes on cyanuric acid as a nitrogen source

| Plasmid                          | Nitrogen source        | Final OD<sub>600</sub> |
|-----------------------------------|------------------------|-----------------------|
| pET28b+:::atzF                    | 5 mM cyanuric acid    | 0.22                  |
| pKT230::atzD                      | 5 mM cyanuric acid    | 0.27                  |
| pET28b+:::atzF, pKT230::atzD      |                        | 0.22                  |
| pET28b+:::atzF, pKT230::atzD      | 15 mM NH<sub>4</sub>Cl<sub>2</sub> | 1.07                 |
| pET28b+:::atzF, pKT230::atzD      | 5 mM cyanuric acid    | 0.79                  |

substrates shown in Table 2, and they are evolutionarily related members of the amidase signature protein family. TrzF is clearly identified as a member of this family by sequence comparison and time-dependent inhibition by phenyl phosphorodiamidate, as shown in this study. Phenyl phosphorodiamidate has been shown to covalently modify an essential serine residue in the active site of certain members of this superfamily (13, 28). Both allophanate hydrolases, AtzF (27) and TrzF, which was tested here, are highly sensitive to this reagent.

Data derived from hundreds of microbial genome sequencing projects also contribute to the idea that allophanate hydrolases are widespread. It was previously reported that bacteria from diverse taxonomic groups contain homologs to AtzF (27) and to the allophanate hydrolase from *Oleomonas* sp. (18). Allophanate hydrolase genes that are closely linked with a urea carboxylase gene are typically found on a chromosomal element. The allophanate hydrolase from *Pseudomonas* sp. strain ADP, involved in cyanuric acid metabolism, is found on plasmid pADP-1 (21). It comprises part of a cyanuric acid catabolic operon where it is flanked by *atzD*, encoding cyanuric acid hydrolase, and *atzE*, encoding biuret hydrolase. The allophanate hydrolase TrzF characterized in this study is encoded by a gene also found on plasmid DNA, and the flanking sequences show evidence for a cyanuric acid hydrolase, but nothing resembling the known biuret hydrolase gene, *atzE*, has been identified.

The functioning of allophanate hydrolase in the catabolism of cyanuric acid, derived from *s*-triazine herbicides that have only lately been present in the environment, suggests that this may be a more recently acquired function for these enzymes. The plasmid localization of the allophanate hydrolase genes that act in *s*-triazine metabolism is also consistent with recent recruitment of these genes for this function. To our knowledge, there is no known natural product *s*-triazine compound. In contrast to these observations, the allophanate hydrolase gene, *atzF*, in *Pseudomonas* sp. strain ADP is clustered with the *atzD* and *atzE* genes in an operon. The cyanuric acid metabolism in *Pseudomonas* sp. strain ADP has been shown to be expressed optimally in the presence of cyanuric acid and under general nitrogen starvation conditions (12). In contrast, *E. cloacae* strain 99 showed similar levels of allophanate hydrolase expression in the presence or absence of cyanuric acid and regardless of the presence of ammonium ion (3). Furthermore, the allophanate hydrolase levels in cell extracts of *E. cloacae* strain 99 were low under all growth conditions. This suggested that allophanate hydrolase is constitutively expressed at low levels and may be at an earlier stage of evolution than the physiologically controlled cyanuric acid operon observed in *Pseudomonas* sp. strain ADP. Moreover, *Pseudomonas* sp. strain ADP has a separate functional biuret hydrolase, AtzE, that is contiguous on plasmid pADP-1 (21). An analogous protein has never been identified in *E. cloacae* strain 99. The low biuret hydrolase activity for the allophanate hydrolases observed in this study may be relevant to the evolution of the cyanuric acid catabolic pathway in *E. cloacae* strain 99 and other bacteria.

The observation that homologous enzymes catalyze consecutive reactions and act on structurally analogous substrates is reminiscent of the retrograde model of catabolic pathway evolution originally proposed in 1945 (16). This model has not been shown to explain the genetic composition underlying

DISCUSSION

Bacterial allophanate hydrolases participate in two distinct metabolic processes: (i) urea metabolism, in concert with urea carboxylase, and (ii) the catabolism of *s*-triazine ring compounds. The catabolism of *s*-triazine pesticides and the resin intermediate melamine occurs via cyanuric acid as an intermediate. For 20 years, cyanuric acid metabolism was thought to proceed through urea (4, 19). Recently, bacterial metabolism of cyanuric acid was shown to proceed through allophanate, and urea was found not to be an intermediate in this metabolism (3). A significant number of soil bacteria are capable of growing on cyanuric acid (32), suggesting that the involvement of allophanate hydrolases in cyanuric acid metabolism is widespread. Only one such enzyme, AtzF, had been purified previously, and the present study extends our knowledge of this broadly distributed enzyme class. TrzF is principally, or completely, involved in cyanuric acid metabolism in vivo, based on its measured activities and linkage of the *trzF* gene with the *trzD* and *trzC* genes involved in the metabolism of *s*-triazine compounds. TrzC encodes ammelide hydrolase, and TrzD encodes cyanuric acid hydrolase. These two enzymes catalyze consecutive metabolic reactions and produce biuret (9).

The allophanate hydrolases, the enzyme malonamidase (28) (Table 1), and AtzE (21) work on the structurally analogous

most metabolic pathways; rather, a patchwork model is more commonly invoked (1, 6). Still, enzyme catalytic promiscuity is increasingly recognized as a key element for evolving new enzymatic activities (2, 7, 22), which can contribute to metabolic pathway evolution. In this context, the observation here that allophanate hydrolase has activity with biuret shows that it can catalyze the two terminal reactions in the cyanuric acid catabolic pathway (Fig. 1 and 2). Since these are the two reactions that liberate ammonia, allophanate hydrolase can act in concert with TrzD or AtzD and allow bacterial growth on cyanuric acid. This was demonstrated here with a recombinant *E. coli* strain expressing AtzD and AtzF. Control experiments with *E. coli* containing the *atzD* gene alone showed that no indigenous biuret hydrolase was present as this would have allowed growth in the absence of *atzF*, which was not observed. Moreover, biuret hydrolase activity was observed with highly purified TrzF, obtained from a recombinant bacterium thought not to contain an indigenous biuret hydrolase activity. It is interesting to consider the possibility that TrzF contributes to biuret hydrolysis in *E. cloacae* strain 99, promoting growth on cyanuric acid. Thus, it might be another example of catalytic promiscuity providing a selectable enzyme activity with evolutionary pressure that could act over time to improve activity. Although a biuret hydrolase has yet to be purified to homogeneity, the sequence of AtzE indicates that it is also a member of the amidase signature family.

### ACKNOWLEDGMENTS

We thank Gil Johnson for the synthesis of allophanate and José Carrascosa for the GroEL GroES construct.

This work was supported in part by U.S. Department of Agriculture grant 2002-35107-12508 and by a grant from the University of Minnesota Biocatalysis Initiative.

### REFERENCES

1. Babbitt, P. C., and J. A. Gerlt. 1997. Understanding enzyme superfamilies: chemistry as the fundamental determinant in the evolution of new catalytic activities. J. Biol. Chem. **272**:30591–30594.
2. Bornscheuer, U., and R. J. Kazlauskas. 2004. Catalytic plasticity in biocatalysis: Using old enzymes to form new bonds and follow new pathways. Angew. Chem. Int. Ed. Engl. **43**:6032–6040.
3. Cheng, G., N. Shapir, M. J. Sadowsky, and L. P. Wackett. 2005. Allophanate hydrolase, not urease, functions in bacterial cyanuric acid metabolism. Appl. Environ. Microbiol. **71**:4437–4445.
4. Cook, A. M. 1987. Biodegradation of s-triazine xenobiotics. FEMS Microbiol. Rev. **46**:93–116.
5. Cook, A. M., P. Beilstein, H. Grossenbacher, and R. Hütter. 1985. Ring cleavage and degradative pathway of cyanuric acid in bacteria. Biochem. J. **231**:25–30.
6. Copley, S. D. 2002. Evolution of a metabolic pathway for degradation of a toxic xenobiotic: the patchwork approach. Trends Biochem. Sci. **25**:261–265.
7. Copley, S. D. 2003. Enzymes with extra talents: moonlighting functions and catalytic promiscuity. Curr. Opin. Chem. Biol. **7**:265–272.
8. Eaton, R. W., and J. S. Karns. 1991. Cloning and analysis of s-triazine catabolic genes from *Pseudomonas* sp. strain NRRLB-12227. J. Bacteriol. **173**:1215–1222.
9. Eaton, R. W., and J. S. Karns. 1991. Cloning and comparison of the DNA encoding ammelide amidohydrolase and cyanuric acid amidohydrolase from three s-triazine-degrading bacterial strains. J. Bacteriol. **173**:1363–1366.
10. Eaton, R. W., and D. W. Ribbons. 1982. Metabolism of dibutylphthalate and phthalate by *Micrococcus* sp. strain 12B. J. Bacteriol. **151**:48–57.
11. Fruchey, I., N. Shapir, M. J. Sadowsky, and L. P. Wackett. 2003. On the origins of cyanuric acid hydrolase: purification, substrates, and prevalence of AtzD from *Pseudomonas* sp. strain ADP. Appl. Environ. Microbiol. **69**:3653–3657.
12. Garcia-Gonzalez, V., F. Govantes, O. Porrua, and E. Santero. 2005. Regulation of the *Pseudomonas* sp. strain ADP cyanuric acid degradation operon. J. Bacteriol. **187**:155–167.
13. Gopalakrishna, K. N., B. H. Stewart, M. M. Kneen, A. D. Andricopulo, G. L. Kenyon, and M. J. McLeish. 2004. Mandelamide hydrolase from *Pseudomonas putida*: characterization of a new member of the amidase signature family. Biochemistry **43**:7725–7735.
14. Hausinger, R. P. 2004. Metabolic versatility of prokaryotes for urea decomposition. J. Bacteriol. **186**:2520–2522.
15. Hodson, R. C., S. K. Williams II, and W. R. Davidson, Jr. 1975. Metabolic control of urea catabolism in *Chlamydomonas reinhardi* and *Chlorella pyrenoidosa*. J. Bacteriol. **121**:1022–1035.
16. Horowitz, N. H. 1945. On the evolution of biochemical syntheses. Proc. Natl. Acad. Sci. USA **31**:153–155.
17. Kanamori, T., N. Kanou, H. Atomi, and T. Imanaka. 2004. Enzymatic characterization of a prokaryotic urea carboxylase. J. Bacteriol. **186**:2532–2539.
18. Kanamori, T., N. Kanou, S. Kusakabe, H. Atomi, and T. Imanaka. 2005. Allophanate hydrolase of *Oleomonas sagaranensis* involved in an ATP-dependent degradation pathway specific to urea. FEMS Microbiol. Lett. **245**:61–65.
19. Karns, J. S., and R. W. Eaton. 1997. Genes encoding s-triazine degradation are plasmid-borne in *Klebsiella pneumoniae* strain 99. J. Agric. Food Chem. **45**:1017–1022.
20. Koo, H. M., S. O. Choi, H. M. Kim, and Y. S. Kim. 2000. Identification of active-site residues in *Bradyrhizobium japonicum* molonamidase E2. Biochem. J. **349**:501–507.
21. Martinez, B., J. Tomkins, L. P. Wackett, R. Wing, and M. J. Sadowsky. 2001. Complete nucleotide sequence and organization of the atrazine catabolic plasmid pADP-1 from *Pseudomonas* sp. strain ADP. J. Bacteriol. **183**:5684–5697.
22. O'Brien, P. J., and D. Herschlag. 1999. Catalytic promiscuity and the evolution of new enzymatic activities. Chem. Biol. **6**:R91–R105.
23. Patricelli, M. P., and B. F. Cravatt. 2000. Clarifying the catalytic roles of conserved residues in the amidase signature family. J. Biol. Chem. **275**:19177–19184.
24. Pérez-Pérez, J., and J. Gutiérrez. 1995. An arabinose-inducible expression vector, pAR3, compatible with ColE1-derived plasmids. Gene **158**:141–142.
25. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.
26. Segel, I. H. 1975. Other methods of plotting enzyme kinetics data, p. 208–224. *In* I. H. Segal (ed.), Enzyme kinetics: behavior and analysis of rapid equilibrium and steady-state enzyme systems. Wiley-Interscience, New York, N.Y.
27. Shapir, N., M. J. Sadowsky, and L. P. Wackett. 2005. Purification and characterization of allophanate hydrolase (AtzF) from *Pseudomonas* sp. strain ADP. J. Bacteriol. **187**:3731–3738.
28. Shin, S., T. H. Lee, C. N. Ha, H. M. Koo, S. Y. Kim, H. S. Lee, Y. S. Kim, and B. H. Oh. 2002. Structure of malonamidase E2 reveals a novel Ser-cisSer-Lys catalytic triad in a new serine hydrolase fold that is prevalent in nature. EMBO J. **21**:2509–2516.
29. Sumrada, R. A., and T. G. Cooper. 1982. Urea carboxylase and allophanate hydrolase are components of a multifunctional protein in yeast. J. Biol. Chem. **257**:9119–9127.
30. Weatherburn, M. W. 1967. Phenol-hypochlorite reaction for determination of ammonia. Anal. Chem. **39**:791–794.
31. Whitney, P. A., and T. G. Cooper. 1972. Urea carboxylase and allophanate hydrolase. Two components of adenosine triphosphate:urea amido-lyase in *Saccharomyces cerevisiae*. J. Biol. Chem. **247**:1349–1353.
32. Zeyer, J., J. Bodmer, and R. Hütter. 1981. Microbial degradation of ammeline. Zentbl. Bakteriol. Hyg. Abt. 1 Orig. C **2**:289–298.
