
Transcriptional Regulation of Metabolism

BÉATRICE DESVERGNE, LILIANE MICHALIK, AND WALTER WAHLI

Center for Integrative Genomics, National Centre of Competence in Research “Frontiers in Genetics,”  
University of Lausanne, Lausanne, Switzerland

---

I. Introduction                                                                                                   466
II. Transcriptional Control by Glucose                                                                           467
   A. Introduction                                                                                               467
   B. Transcriptional regulation of metabolism by high glucose levels                                           468
   C. Transcriptional regulation of metabolism by low glucose levels                                            472
III. Transcriptional Control of Amino Acid and Protein Metabolism                                                474
   A. Introduction                                                                                               474
   B. Transcriptional regulation of glutamine production and homeostasis                                        474
   C. Transcriptional regulation of urea and ammonia homeostasis                                                 476
   D. Amino acid deprivation and induction of gene expression                                                    477
IV. Transcriptional Control of Lipid Metabolism                                                                  478
   A. Transcriptional control of fatty acid metabolism                                                          478
   B. Transcriptional control of cholesterol homeostasis                                                         484
   C. Intricate regulation of cholesterol and fatty acid metabolism                                              487
V. Transcription Factor Interplay in the Fasting-Feeding Cycle                                                   489
   A. Fasting-feeding: metabolic adjustment in the WAT                                                          490
   B. Fasting-feeding: metabolic adjustment in muscles                                                          491
   C. Fasting-feeding and gluconeogenesis: metabolic adjustment in the liver, kidney,                             492
      and small intestine
   D. Metabolic adjustment and circadian rhythm                                                                  493
VI. Transcription Factors and Insulin Resistance: a Major Focus on Peroxisome Proliferator                        494
    Activated Receptors
   A. Introduction                                                                                               494
   B. Insulin resistance: a mixed defect in glucose and lipid metabolism                                         494
   C. Insulin resistance: on the molecular nature of the causal mechanism                                         495
   D. The thiazolidinediones as a tool for understanding the physiopathogeny                                     496
      of the metabolic syndrome
VII. Transcription Factors as Targets for Therapeutic Approaches of Metabolic Disorders                          498
   A. How to modulate transcription factor activity                                                              498
   B. Nuclear receptors as targets for new therapeutic approaches                                                499
   C. Adverse effects of drugs on energy metabolism                                                             500
VIII. Conclusions                                                                                                500

---

Desvergne, Béatrice, Liliane Michalik, and Walter Wahli. Transcriptional Regulation of Metabolism. *Physiol Rev* 86: 465–514, 2006; doi:10.1152/physrev.00025.2005.—Our understanding of metabolism is undergoing a dramatic shift. Indeed, the efforts made towards elucidating the mechanisms controlling the major regulatory pathways are now being rewarded. At the molecular level, the crucial role of transcription factors is particularly well-illustrated by the link between alterations of their functions and the occurrence of major metabolic diseases. In addition, the possibility of manipulating the ligand-dependent activity of some of these transcription factors makes them attractive as therapeutic targets. The aim of this review is to summarize recent knowledge on the transcriptional control of metabolic homeostasis. We first review data on the transcriptional regulation of the intermediary metabolism, i.e., glucose, amino acid, lipid, and cholesterol metabolism. Then, we analyze how transcription factors integrate signals from various pathways to ensure homeostasis. One example of this coordination is the daily adaptation to the circadian fasting and feeding rhythm. This section also discusses the dysregulations causing the metabolic syndrome, which reveals the intricate nature of glucose and lipid metabolism and the role of the transcription factor PPARγ in orchestrating this association. Finally, we discuss the molecular mechanisms underlying metabolic regulations, which provide new opportunities for treating complex metabolic disorders.

466 BEÁTRICE DESVERGNE, LILIANE MICHALIK, AND WALTER WAHLI

**I. INTRODUCTION**

Our knowledge of metabolism mainly results from the long-standing and very extensive work of a myriad of biochemists. They first achieved the identification of enzymatic steps, their functional characterization, and the discovery of regulatory loops with which they are associated. For decades, allosteric controls linked to substrate availability constituted the best of our knowledge of metabolic control systems. A crucial step was then accomplished with the deciphering of the galactose operon in bacteria, which represented a major discovery for the study of metabolism. It revealed how organisms can adapt their metabolic activity to environmental nutritional changes by modifying the level of expression of specific enzymes and linked modulation of enzymatic activity to the transcriptional control of gene expression for the first time.

It is now commonly accepted that metabolic regulation in complex organisms relies on three main types of control. The first corresponds to the classic allosteric control of the activity of a key enzyme along a metabolic pathway triggered by the binding of an activator, which often is the enzyme substrate itself. The second mechanism involves various posttranslational modifications such as proteolytic cleavage, phosphorylation, glycosylation, sumoylation, and acetylation, which may shift the equilibrium between an inactive and active enzyme within seconds and/or affect protein stability. In these two types of control, subsequent changes in protein-protein interaction may participate in producing the active/nonactive enzymatic complex. The third mechanism is transcriptional regulation, which affects the level of expression of key enzymes and is effective on a longer time scale. It clearly appears that most metabolic regulations benefit from a coordination of these various mechanisms. The purpose of this review is to highlight the recent progress in understanding when and how transcriptional regulation participates in the control of metabolic homeostasis.

Transcriptional control requires specific signals to be transduced to the cell nucleus where defined sets of genes are targeted. Thus understanding the transcriptional control of metabolism relies on three complementary pieces of information: 1) events upstream of transcriptional activity, which define the signals involved and their route to the nucleus; 2) the molecular mechanisms by which transcription factors operate; and 3) events downstream of transcriptional activity, which depend on the groups of genes that are targeted and how further signals are generated to reach the dynamic equilibrium of homeostasis. Virtually all transcription factor families are in one way or another involved in metabolic regulation. However, a few of them have a clear predominant role and seem mainly dedicated to metabolic regulation. For the sake of clarity, the transcription factors most often cited herein are briefly presented. Appendix A presents the nuclear receptor family and is accompanied by Figure 1A, which shows the main characteristics of the transcription factors belonging to this family. The notion of “metabolic sensor” receptors was more particularly developed with respect to these nuclear receptors, as also explained in Appendix A with the accompanying Figure 1B. Appendices B–F describe the main features of some of these “sensors,” which belong to the nuclear receptor family, with the peroxisome proliferator activated receptor (PPAR) in Appendix B, the liver X receptor (LXR) in Appendix C, the farnesol X receptor (FXR) in Appendix D, the hepatocyte nuclear factor 4 (HNF4) in Appendix E, and the retinoid X receptor (RXR) in Appendix F. Appendix G details the amazing characteristics of the sterol response element binding proteins (SREBPs), which play a major role in lipid and cholesterol metabolism. Finally, the heterogeneous family of proteins initially grouped under the name of liver-enriched transcription factors and which comprises the CAAT enhancer binding proteins (C/EBP) is discussed in Appendix H.

In recent years, interest has increased in cofactors that bridge proteins and allow the DNA-bound transcription factors to transmit their activation or repression properties to the transcriptional machinery. These cofactors are characterized by 1) their ability to interact with a wide variety of transcription factors and 2) their ability to assemble a protein complex that will be the transcriptional effector. Importantly, these cofactors are direct targets of certain signaling pathways, as seen with the insulin-dependent phosphorylation of the CREB-binding protein (CBP). With respect to the role of cofactors in transcriptional regulation of metabolism, for example, a clear picture has emerged from study of the PPAR gamma coactivator 1 (PGC1), which is implicated in thermogenesis and in associated metabolic responses, and of CBP in contributing to neoglucogenesis (see appropriate sections). Specific metabolic roles for steroid receptor coactivator 1 (SRC-1), transcriptional intermediary factor 2 (TIF2), and the receptor interacting protein 140 (RIP40) are also emerging, and further work should guarantee several important new developments in this field. However, it is beyond the scope of this review to discuss transcriptional cofactors specifically, and we refer to reviews that have recently been devoted to the subject (e.g., Refs. 85, 207, 250).

The aim of this review is to summarize recent knowledge concerning the transcriptional control of metabolic homeostasis. Analyses of the main transcriptional controls occurring in the regulation of intermediary metabolism is followed by an integrative approach which illustrates how these regulations can take place during the alternation between fasting and feeding to achieve energy homeostasis. In a pathological context, the disruption of energy homeostasis reflected by the metabolic syndrome highlights the intricate link between glucose and lipid

TRANSCRIPTIONAL REGULATION OF METABOLISM

A

Nuclear Receptor

Transcription activation (AF-1)
DNA Binding

Dimerization
Ligand binding
Ligand-dependent transcription activation (AF-2)
Dimerization

NR RXR

Transcription

(Nn)

DR-n

Corepressor

Histone deacetylation
ATP-dependent chromatin condensation
Inhibition of transcription

Coactivator

Histone acetylation/
methylation
ATP-dependent chromatin opening
Activation of transcription

FIG. 1. The nuclear receptor family. This figure accompanies Appendix A (see text of Appendix A).

B

Classic Receptors
GR
MR
AR
PR
ERα, β
VDR
TRα, β
RARα, β, γ

"Sensor" Receptors
PPARα, β, γ
HNF-4*
LXRα, β
FXR
PXR
CAR
RXRα, β, γ

Orphan Receptors
ARP-1
COUP-TF
EAR2
SVP46
REVERBα, β
SF-1
LRH-1
NGFI-B
NURR1
TR2
TR4
RZRα, β
ERR1
ERR2
GCNF
and others...

metabolism. The last section discusses transcription factors as targets for treating complex metabolic disorders.

II. TRANSCRIPTIONAL CONTROL BY GLUCOSE

A. Introduction

Glycemia is a parameter over which the organism establishes tight control. In humans, blood glucose levels are kept constant in a narrow range from 4 to 7 mM, despite discontinued supply due to the alternation between feeding and fasting. One main danger of prolonged hypoglycemia is acute brain damage. At the other end of the scale, acute hyperglycemia is a serious complication of decompensated diabetes mellitus. The associated ketoacidosis and hyperosmolar hyperglycemic state might be fatal due to dehydration and electrolyte imbalance. Chronic hyperglycemia is a major cause of neuropathy and vasculopathy, as seen in diabetes.

Glucose homeostasis is maintained by a hormonal network in which insulin and glucagon are the main agents. Synthesis and secretion of insulin are stimulated by increased glucose levels, particularly after feeding. Insulin release allows the quick removal of glucose from circulation by stimulating the entry of glucose into peripheral tissues, mainly in muscle and adipose tissue cells. In parallel, insulin increases energy storage by inducing glycogen synthesis in liver and muscle, and fatty acid synthesis in liver and adipose tissue. When insulin levels are low, between meals or upon fasting, the hormone glucagon increases the hepatic production and release of glucose by increasing glycogenolysis and stimulating gluconeogenesis. The pancreas is the chief organ of these dual regulations, as it senses glucose levels and produces insulin and glucagon accordingly. The liver functions as the main "buffer," providing glucose when nutrients are scarce and storing glucose as glycogen when food is abundant. Once the liver glycogen store is full, the adipose tissue converts glucose into triacylglycerol for longer term storage as fat. Muscles mainly consume rather than store energy, although they efficiently accumulate glycogen for their own use. The brain is a particular target organ that can use glucose and/or ketone

Physiol Rev • VOL 86 • APRIL 2006 • www.prv.org

bodies as an energy source. However, the fact that glucose represents the sole source of energy for some of its cells imposes a tight control over glycemia. In this organ, the entry of glucose in cells is mediated by the Glut3 transporter, which maintains a constant supply of glucose to brain cells until glycemia drops to very low levels close to its \( K_{\mathrm{m}} \) value, i.e., when approaching 2.2 mM.

The aim of this section, which cannot be exhaustive, is to discuss the main threads of the transcriptional network which result in glucose homeostasis. As is the case for many pathways regulated by nutrients, glucose is both an end product and the nutrient substrate that triggers regulation. Therefore, two opposing situations are considered for simplicity: that of high and that of low glucose levels. For each situation, we will describe the signals that are triggered and their action in transcription.

## B. Transcriptional Regulation of Metabolism by High Glucose Levels

High glucose levels influence gene expression either directly or through the stimulation of insulin production by the \(\beta\)-cells of the pancreas. We first analyze the transcriptional regulation of *insulin*¹ and the regulation of insulin secretion. We then review the mechanisms by which glucose and insulin, independently or together, modulate gene transcription.

### 1. Transcriptional control of insulin expression and secretion

Pro-insulin is synthesized in the \(\beta\)-cells of the pancreatic Langerhans islets and is then cleaved by proconvertases into insulin and peptide C. Insulin is stored in secretory vesicles, and its secretion is directly linked to a mechanism sensing glucose availability via an increase in the intracellular ATP/ADP ratio that correlates with the entry and metabolism of glucose in the \(\beta\)-cells (Fig. 2, Ref. 230). The entry of glucose into the \(\beta\)-cells requires a glucose transporter, Glut2 in rodents but Glut1 rather than Glut2 in humans (56), whose expression and membrane localization are independent of glucose or insulin signaling. The posttranscriptional control of insulin expression and processing, as well as the control over the secretory mechanism, which is dependent on glucose sensing, are key features of the regulation of insulin signaling. However, the pathologies exhibited by patients in whom the regulation of *insulin* gene expression is altered emphasize the importance of the transcriptional level of control.

While insulin mRNA and protein expression have been found in various tissues (151) in different diabetic

---

¹ Name in italics systematically refers to the gene.

HIGH GLUCOSE

Pancreatic β cell

mGlut2/hGlut1

HNF1β
[MODY 5]

HNF3β

HNF4α
[MODY 1]

Glucokinase
[MODY 2]

HNF1
[MODY 3]

PDX1
[MODY 4]

Glucose

(PI3K)

β cell-specific
TFs

Insulin

Insulin Secretory
Granules

Factor X?

INSULIN

Peripheral cells
(adipose tissue, liver, and
muscle cells)

G6P

F6P

F1,6P₂

ATP/ADP

Ca²⁺

K⁺

Ca²⁺

depolarization

FIG. 2. Transcriptional regulation of the insulin gene in the presence of high glucose. The roles of the HNF family of proteins and the central role of the transcription factor PDX1 in the regulation of the expression of the insulin gene are highlighted. Gene alterations responsible for the maturity onset diabetes of the young (MODY) are shown in red. For further details, see section II B. Insulin secretion is triggered by an increase in cytosolic ATP/ADP, which closes K⁺ channels in the plasma membrane, thereby causing membrane depolarization and opening of voltage-gated Ca²⁺ channels. The resulting rise in the cytosolic Ca²⁺ concentration activates exocytosis of insulin-containing granules (230). Factor X, hypothetical factor acting on insulin secretion; F6P, fructose-6-phosphate; F1,6-P2, fructose 1,6-diphosphate; Glut, glucose transporter; G6P, glucose-6-phosphate; HNF, hepatocyte nuclear factor; PDX1, pancreatic duodenum homeobox; PI3K, phosphatidylinositol 3-kinase; TFs, transcription factors.

pressed in most cells in vertebrates. This interaction drives the activation of the Ras/mitogen-activated protein kinase (MAPK) pathway. The cascade of phosphorylation events starts with the phosphorylation of insulin receptor substrate 1 and/or 2 (IRS1, IRS2). The successive activation of son-of-sevenless (SOS), Ras, and Raf-1 subsequently activates MEK (mitogen-activated, ERK-activating kinase), which in turn phosphorylates MAPK. Activation of this pathway seems to mostly target cellular growth and proliferation, rather than direct metabolic actions, and will not be discussed further.

An important relay of the metabolic action of insulin is the activation of PI3K leading to that of PDK1, which in turn phosphorylates the serine/threonine kinase Akt (also called PKB). Activated Akt/PKB phosphorylates several factors, including GSK3β and FOXOs, which directly or indirectly mediate the effects of insulin on the transcription of genes involved in glucose metabolism. There are three highly homologous Akt/PKB members, Akt1/PKBα, Akt2/PKBβ, and Akt3/PKBγ. Specific gene deletion or gene silencing of Akt1 and Akt2 demonstrates the primary role of Akt2 in insulin signaling and glucose metabolism, while significant redundancy of Akt1 and Akt2 still exists (80, 129). In addition to activating Akt, insulin triggers the tyrosine phosphorylation of the protooncogene Cbl in a PI3K-independent manner. Phosphorylated Cbl stimulates the translocation of the glucose transporter 4

Physiol Rev • VOL 86 • APRIL 2006 • www.prv.org

INSULIN

GLUCOSE

cAMP
↓↓
Inhibition
Glucagon
action

RAS
↓
Raf
↓
MEK
↓
ERK
↓
MAPK

IRS1/IRS2
↓
PI₃K
(PDK1)
↓
AKT1

Cbl
Translocation
(muscle, adipose tissue)

GLUT4
↓
Glucose

GK
→
G6P

SREBP1c
Xylulose 5P?
G6P?
Hexosamine?

ChREBP/ChoRF
USF1:USF2
?

MAPK dependent
transcription
↑ Glucagon
synthesis
↑ Gene
expression
↑ Protein
synthesis
↑ Glucose
transport

Insulin action on
cell growth (non-
metabolic action)

Role in insulin-mediated
negative regulation of
gene expression?

Insulin and glucose target
genes : L-PK, S14, FAS

FIG. 3. Insulin and glucose action on the regulation of gene expression. Four main outcomes of the interaction of insulin with the membrane insulin receptor (IR) receptor and insulin receptor substrate 1 and 2 (IRS1/IRS2) are represented. The important reduction of intracellular levels of cAMP counteracts the action of glucagon (see also Fig. 5). The RAS/MAPK pathway leads to the activation of genes, which are mainly involved in cell growth (not reviewed herein). The activation of the phosphatidylinositol 3-kinase (PI3K) mediates most of the action of insulin on intermediary metabolism, via activation of Akt and activation of the SREBP-1c gene expression. PI3K-dependent activation of SREBP-1c is presently considered to be a major event for insulin-mediated gene induction. In contrast, Akt/PKB activation inhibits the activity of the transcription factors FOXO. This insulin-mediated inhibition of FOXO mainly results in target gene repression. Other consequences of Akt activation are also indicated but are not discussed in the present review, due to the fact that little is known about their action at the transcriptional level. The mechanism of insulin action on Glut4 expression remains unknown (dotted line), while Cbl is involved in the insulin-mediated increase of Glut4 translocation in muscle and adipose tissue. The intricate roles of glucose and insulin as regulators of gene transcription are shown on the right of the scheme and comprise three aspects: the insulin-mediated translocation of the glucose transporter Glut4 as it occurs in the the adipose tissue and muscle cells but not in the liver, while expression of Glut2 in the liver is dependent on the transcription factor FOXA3 (HNF3γ); the increased glucokinase (GK) expression in the liver; the independent activation of common target genes by both insulin via SREBP-1c and by glucose via carbohydrate response element binding protein/carbohydrate response factor (ChREBP/ChoRF). Xylulose-5-phosphate (xylulose 5P), glucose-6-phosphate (G6P), and hexosamine are glucose metabolites indicated as possible signal molecules directly responsible for the transcriptional response of the cell to glucose. For more details, see section II.B.

(Glut4) to the cell surface membrane, independently of transcriptional regulation (reviewed in Refs. 176, 265). Together, the biological effects of the transduction cascades are Glut4 translocation to increase glucose uptake in adipose tissue and muscle, the activation of glycogen synthase and thus of glycogen synthesis, and the activation of the S6-kinase that will increase protein synthesis.

In addition to or as a result of these signaling cascades in target cells, the expression of more than 150 genes expressed in various tissues is transcriptionally regulated by insulin. The diversity of mechanisms of the insulin-mediated transcriptional regulation is extremely wide, as indicated by the variety of promoter sequences that are responsible for insulin-mediated action. A classification of the insulin response sequences or insulin response elements (IRS/IRE) into seven groups has been proposed (212). Whereas there is still much to learn about the factors that are modified by insulin and bind to these elements, recent results have identified sterol response element binding protein 1c (SREBP-1c) as a major contributor. SREBPs are transcription factors of the helix-loop-helix family highly expressed in the liver. Their three forms, SREBP-1a, SREBP-1c, and SREBP-2, were first explored for their role in cholesterol and lipid homeostasis (see Appendix G). Interestingly, SREBP-1c expression in the liver is upregulated by insulin independently of glucose levels

Physiol Rev • VOL 86 • APRIL 2006 • www.prv.org

(73, 278). This regulation occurs at the transcriptional level, and detailed functional analyses of the SREBP-1c promoter revealed a complex interplay of transcription factors (24, 33). Whereas SREBP itself and nuclear factor Y act to maintain the basal level of SREBP-1c expression, the LXR (see Appendix C), a nuclear receptor activated by oxidized derivatives of cholesterol, plays a crucial role in its insulin-mediated increased expression. The proposed mechanism by which insulin-increased LXR activity would involve the insulin-dependent production of a ligand for LXR (33) is yet to be demonstrated. In addition to this transcriptional up-regulation of SREBP-1c expression, insulin triggers the proteolytic cleavage of SREBP-1c in a PI3K-dependent manner to produce the mature active form of this transcription factor (104).

In turn, SREBP-1c acts as an important mediator of insulin action, at least with respect to the transcriptional regulation of glycolytic and lipogenic genes in the liver. Overexpression of a dominant negative form of SREBP-1c counteracts insulin-mediated induction of the expression of liver pyruvate kinase (L-PK), spot 14 (S14), and fatty acid synthase (FAS), three canonical genes with respect to glucose*insulin² responsiveness (see below and Ref. 73). The expression of glucokinase (GK) in the liver is also regulated by insulin, independently of extracellular glucose levels. GK phosphorylates glucose in glucose-6-phosphate (G6P), a first reaction required for any further intracellular metabolism of glucose. This regulation requires an intact PI3K pathway (124) and might also occur via SREBP-1c (73, 145). Less is known of insulin-mediated activation of SREBP-1c in the adipose tissue. The overlap of the gene expression profile of 3T3-L1 adipocytes subjected to insulin treatment with that of cells overexpressing the mature form or a dominant negative form of SREBP-1c strengthened the notion of a correlation between insulin-induced gene expression and SREBP-1c activity. It also revealed that, in these cells, the transcription factor CAAT/enhancer binding protein β (C/EBPβ) is responsive to insulin via stimulation of SREBP-1c (169a).

Insulin also negatively regulates transcription, particularly that of genes involved in hepatic glucose production, such as those encoding IRS2, phosphoenolpyruvate carboxykinase (PEPCK), insulin growth factor binding protein-1 (IGFBP-1), and glucose-6-phosphatase (G6Pase) (for review, see Ref. 213). A particular sequence element, often contained in a broader insulin response unit, was identified in the promoter region of these genes as a mediator of negative regulation by insulin. Several transcription factors, such as members of the C/EBP, HNF-3/

FOXA, and FOXO families, can bind to this element. Of particular interest are the FOXOs, represented by three members, FOXO1, FOXO3a, and FOXO4 (previously called FKHR, FKHRL, and AFX, respectively), which are phosphorylated by Akt-1 upon insulin-mediated activation of the PI3K pathway (reviewed in Ref. 299). Phosphorylated FOXO has a high affinity for protein 14-3-3, which relocates FOXO from the nucleus to the cytosol. In addition, insulin enhances ubiquitination of phosphorylated FOXO and its further degradation (193). Thus, in a simple mechanistic model, insulin would mediate repression via removal from the nucleus and accelerated degradation of the positive transcriptional regulator FOXO. Whereas the correlation between the activity levels of FOXO, HNF3, and C/EBP in mammals and gene repression by insulin remains unclear, their complex intertwined functions are illustrated in insulin-mediated PDX1 regulation. In the pancreatic β-cells, nuclear FOXO1 acts as a repressor of the positive activity of HNF3β on the PDX1 promoter, while insulin signaling relieves this repression by excluding FOXO1 from the nucleus (149).

It is interesting to note that the insulin signaling pathway going through PI3K and Akt activation is also present in *Drosophila* and in *Caenorhabditis elegans*. Indeed, the identification of FOXO (as DAF-16) and its involvement in insulin signaling was first described in *C. elegans*, triggering its characterization in mammals. A unique homolog of FOXO, dFOXO, has now been reported in *Drosophila* (134, 235) where insulin plays a crucial role in cellular growth. As seen in mammals and *C. elegans*, *Drosophila* Akt (dAkt) sequesters dFOXO in the cytoplasm when the insulin pathway is active. This results in an inhibition of dFOXO target gene expression, including that of the insulin receptor gene itself.

In summary, the general pathways followed by insulin to trigger many changes in gene expression are beginning to be understood. However, a lot more needs to be done to decipher the molecular mechanisms of this transcriptional regulation. The initial signal is at the cell membrane, and all subsequent events occur via phosphorylation cascades that mainly go through the PI3K pathway, but also possibly via the phosphorylation of the Cbl protooncogene. Thus it is possible that most of the insulin action on gene expression results from posttranslational modifications of various transcription factors, a process that would account for the pleiotropic effects of this hormone.

---

2 The experimental difficulty in discriminating between insulin and glucose effects in some of the gene transcriptional responses is reflected in the present review by the usage of the associated words glucose*insulin.

---

3. **Glucose*insulin regulated transcription of genes involved in glucose metabolism**

As mentioned above, glucose, independently of insulin, can regulate the expression of genes involved in carbohydrate metabolism. Upon entry into the cells, glucose is phosphorylated to G6P by GK in hepatocytes and by

Physiol Rev • VOL 86 • APRIL 2006 • www.prv.org
hexokinase in all other cells. This step is required for glucose to either undergo glycolysis, be used in the glycogen synthesis pathway, or enter the pentose phosphate pathway. This first metabolic transformation of glucose is also required for generating the signal that acts in transcriptional regulation. Some reports suggest that G6P itself might be the signaling molecule. Alternatively, other metabolites such as xylitol produced by the pentose phosphate pathway or intermediates of the hexosamine biosynthetic pathway might also act in tissue-specific regulations (reviewed in Ref. 305).

The analysis of glucose signaling is, however, often difficult to dissociate from insulin signaling (see Fig. 3). First, the entry of glucose into muscle and adipose tissue cells, which are two main insulin target organs, operates through the translocation of the Glut4 transporter via an insulin-mediated transduction signal. In contrast, the expression of the glucose transporter Glut2 in the liver and pancreas, Glut3 in the brain, and the widely distributed Glut1 are insulin independent, and their translocation is constitutive. Second, in the liver and to a lesser extent in the pancreas, the initial metabolic modification of glucose into G6P by GK is required for transcriptional regulation by glucose and is strongly dependent on insulin. Thus the actions of glucose and insulin are often interdependent and, in this review, we refer to this ambiguity by the use of the associated words glucose*insulin when relevant.

Three genes have been important tools for the analysis of the ability of glucose to direct transcriptional regulation; they encode the L-PK (acting on the glycolytic pathway from glucose to pyruvate), S14 (associated to lipogenesis but with an unclear function), and FAS (a key enzyme in lipogenesis). Analyses of the promoter region of these genes have identified response elements called carbohydrate response elements (ChoRE) or glucose response elements (GlRE), which have similarities. The main common feature is the presence of at least one E-box. The GlRE/ChoRE of $L$-PK and $S14$ comprises two E-boxes in tandem, in addition to a binding site for an ancillary factor which is HNF4 in the case of $L$-PK (57, 175, 276). The glucose*insulin responsiveness of FAS also requires a complex array of promoter elements, a complexity that has generated some controversy. Three glucose*insulin response sites are now proposed. A region between $-150/+50$ centered around an E-box was the first proposed IRE/GlRE/ChoRE. This sequence efficiently binds SREBP-1c and mainly represents an insulin responsive element. A second element is located at $-332$, but its role in vivo is unclear. Finally, a third far-upstream element located around $-7 \mathrm{~kb}$ closely resembles the GlRE/ChoRE found in $L$-PK and $S14$ (152, 201, 261).

E-boxes are binding motifs for helix-loop-helix transcription factors and can bind the abundant and ubiquitous upstream stimulatory factor (USF), whose two forms USF1 and USF2 can heterodimerize. However, various studies in vivo (with knock-out animals) and in vitro (electromobility shift assays) aimed at elucidating the link between the ability of USFs to bind to the E-box and glucose responsiveness have failed to prove the concept accurate. An alternate hypothesis involves the negative transcription factor COUP-TFII, which is also able to bind to the $L$-PK GlIRE/ChoRE. The equilibrium resulting from the competition between USF1:USF2 and COUP-TFII would then create the dynamic modulation and glucose-dependent regulation (305). A new factor, initially cloned as WBSCR14 (Williams-Beuren syndrome deleted DNA region, Ref. 51), exhibits a GlRE/ChoRE binding activity that could account for glucose responsiveness. Based on its interaction with the $L$-PK ChoRE, this helix-loop-helix factor has been renamed the ChoRE binding protein (ChREBP) (329). ChREBP is mainly expressed in liver, kidney, and adipose tissue. A mouse line null mutant for ChREBP provided evidence for a direct and dominant role of ChREBP in the glucose-mediated upregulation of $LPK, ACC$, and $FAS$ gene transcription, coordinating synthesis of fatty acids and triglycerides in vivo in response to high levels of glucose (118, 123). Under basal conditions, ChREBP is phosphorylated by protein kinase A (PKA) and remains cytosolic. The glucose-dependent activation of ChREBP is a two-step process, with a first dephosphorylation at serine-196 which triggers its nuclear translocation, and a second dephosphorylation in the nucleus at serine-568 and threonine-666, which allows it to bind to DNA. This activation requires $G K$ expression, as demonstrated in $G K$ knock-out mice (53), and the glucose metabolite xylulose 5-phosphate from the pentose phosphate pathway is the proposed functional link between high glucose and ChREBP activation (136). Indeed, xylulose 5-phosphate can activate protein phosphatase 2, triggering ChREBP dephosphorylation in the cytosol as well as in the nucleus (reviewed in Ref. 52). Finally, the transcriptional activity of ChREBP requires its heterodimerization with the bHLH/LZ factor Max-like protein X (Mlx) (183), which would allow the complex to specifically target E-box binding sites in glucose-responsive gene promoters.

Thus we have gained extremely interesting new understanding of glucose*insulin regulation of gene expression in the last few years. Present works are now aimed in part at understanding when and how these factors respond to an altered metabolic environment, such as in obesity, insulin resistance, or type 2 diabetes.

### C. Transcriptional Regulation of Metabolism by Low Glucose Levels

The prevalence of diabetes, i.e., a deregulation characterized by high glucose levels due to impaired insulin signaling, demonstrates the preeminent role of insulin over the action of all counteracting hormones. This fact possibly explains why less is known about the hormonal
regulation of genes in situations of low glucose availability. In periods of starvation, even between regularly spaced meals, the liver and to a lesser extent the kidney are responsible for the glucose production required for a sufficient supply to the brain. The small intestine also provides glucose upon prolonged starvation. Hormonal controls of this adaptation associate increased glucocorticoids, decreased insulin levels, and, importantly, glucagon secretion by the pancreas in response to low glucose.

Glucagon is processed from proglucagon in the α-cells of the pancreatic islets and is secreted in response to low blood glucose levels. In the liver, glucagon interacts with a membrane receptor coupled to GTP-binding proteins, inducing a rise in intracellular cAMP (Fig. 4), which in turn activates PKA. By this mechanism, glucagon counteracts some of the glucose-insulin-mediated responses. For example, increasing cAMP levels in primary hepatocytes decrease SREBP-1c expression via a mechanism requiring de novo protein synthesis (75, 279). Also, the PKA-dependent phosphorylation of ChREBP sequesters it in the cytosol and inhibits its lipogenic activity (140). Modulation of cAMP levels is the major mechanism by which the liver adjusts glycogenolysis and gluconeogenesis, which produce and release hepatic glucose in the blood. Several transcription factors such as the cAMP response element binding protein (CREB), the cAMP response element modulator (CREM), and the activation transcription factor-1 (ATF-1) are positively activated upon phosphorylation. All three belong to the bZIP family of transcription factors and share a conserved phosphorylation box and glutamine-rich transactivation domain (reviewed in Ref. 48). Their PKA-dependent phosphorylation allows the recruitment of the CREB-binding protein (CBP) coactivator, which contributes to the transcriptional activity of the DNA-bound complexes. CREB is a ubiquitously expressed transcription factor that induces the expression of key genes involved in the gluconeogenesis pathway, such as those encoding PEPCK, G6Pase, and pyruvate carboxylase. Accordingly, CREB binding sites have been identified in the PEPCK and G6Pase promoters (81, 112, 223), but not in that of pyruvate carboxylase. Additional mechanisms for cAMP-mediated transcriptional response are required to explain the full range of responsive genes and the specificity of the response in gluconeogenic tissues, i.e., liver, kidney, and small intestine. For example, the gene for the cofactor PGC1 is strongly activated by CREB in the liver (106). As a cofactor, PGC1 was shown to increase the transcriptional activity mediated by both HNF4α and the glucocorticoid receptor bound to the PEPCK promoter (334) (Fig. 4). The occurrence of such an indirect mechanism via PGC1 for the positive glucagon-dependent induction of pyruvate carboxylase is yet to be examined. HNF4α together with C/EBPα and C/EBPβ also constitutively binds to the G6Pase promoter. In this context, glucagon further induces gene transcription via CREB binding to its cognate site and further recruitment of CBP (81). CBP might be of prime importance for cessation of gluconeogenesis upon feeding, as it is also a target of insulin-dependent phosphorylation at Ser-436 (339). This phosphorylation impairs CBP recruitment to CREB, thereby inhibiting CREB target genes, as demonstrated for PGC1 (341).

LOW GLUCOSE

Glucagon Adrenaline

Glucagon receptor β-adrenergic receptor

C/EBPα → cAMP ↑ (PKA)

(Inhibition of insulin-mediated effects)

SREBP-1c

C/EBPα → Gluconeogenic enzyme gene expression G6Pase, PC, PEPCK

Glucose production

Glucose efflux

Liver cells

CREB PGC-1

Gluconeogenic enzyme gene expression

C/EBP HNF4 GR

PEPCK

FIG. 4. Transcriptional adaptation of the metabolism in liver cells upon low levels of glucose. Most of the responses to low glucose are mediated by the lack of insulin, associated with increased levels of glucagon and, to some extent, by the stimulation of adrenergic receptors. The subsequent increase in cAMP levels triggers the phosphorylation of the transcription factor cAMP response element binding protein (CREB), responsible for increased expression of gluconeogenic enzyme genes. In addition, C/EBPβ participates in increasing cAMP levels, whereas C/EBPα independently activates genes involved in gluconeogenesis. The inset shows the combined interaction of transcription factors and the coactivator PGC-1 involved in the transcriptional regulation of the phosphoenolpyruvate carboxykinase (PEPCK) gene, which plays a crucial role in gluconeogenesis. For more details, see section II.C.

The importance of C/EBPα (see Appendix H and Fig. 4) in the transcriptional control of gluconeogenesis has been revealed by the phenotype of C/EBPα null mice (312). The major metabolic disturbance seen in these mice is a lethal neonatal hypoglycemia. This hypoglycemia is due to the combination of two deficiencies; first, reduced glycogen synthase gene expression is responsible for the absence of a glycogen store; second, the very low levels of liver glu- coneogenic enzymes, such as G6Pase, PEPCK, and tyrosine aminotransferase, cause the lack of gluconeogenesis (reviewed in Ref. 253). A tissue-specific deletion of C/EBPα in the adult liver confirms that these three genes are under the control of C/EBPα in adulthood (165). C/EBPβ null mice have a high susceptibility to hypoglycemia, but survive. In these mice, there is a glycogen store, but glycogenolysis is impaired. This phenotype correlates with decreased levels of cAMP, which could explain an impaired glucagon responsiveness (44).

Finally, the AMP-activated protein kinase (AMPK) seems to play a major role in metabolic homeostasis. Its activation, upon stress or starvation, is caused by a drop in ATP levels with an increased AMP/ATP ratio. In the liver, activation of AMPK leads to an inhibition of lipogenic pathways and affects the glucose*insulin-dependent activation of FAS, S14, and L-PK. Conversely, the knock-out of the α₂-subunit of AMPK triggers a metabolic disturbance associated with high glucose and low insulin levels. This perturbation does not seem to be cell-autonomous, as assessed both in pancreatic islet and muscle cells in vitro, but rather caused by a perturbed autonomous nervous system (307). However, one form of AMPK is expressed in the cell nucleus (reviewed in Ref. 318), and AMPK could therefore act directly on transcriptional regulation by inhibiting the DNA binding activity of ChREBP via phosphorylation (139). This is supported by the fact that a specific short-term overexpression of AMPK in the liver decreased the refeeding-induced transcriptional activation of ChREBP, in parallel with a decreased expression of SREBP-1c (72).

In conclusion, this short presentation highlights some of the best-characterized features of the regulation of glucose homeostasis via the transcription of key genes. However, this summary cannot take into account the specificity of these regulations within each tissue, which is essential for the homeostasis at the level of the whole organism. An effort to integrate some of the pathways described above into the global balance of metabolic regulations will therefore be presented in section VI.

nucleotides, and polyamines, amino acids (AA) also play an important role in energy supply. Among the 20 AA involved in protein synthesis, around half can be synthesized de novo and 9 essential AA must come from the diet. However, during growth or other situations of high-energy expenditure, some of the nonessential AA such as arginine, whose synthesis is energy demanding, should rather be provided by food (reviewed in Ref. 241).

In western countries, under normal nutritional and physiological conditions, proteins/AA are often ingested in excess and are neither stored nor excreted but catabolized and used as an important source of energy that fuels the production of glucose and fatty acids. With a western diet, degradation of AA provides 10–15% of the total energy requirement of the organism. During starvation, the use of AA degradation for energy supply is increased. In the liver, carbon skeletons of the gluconeogenic AA (e.g., alanine and serine in the liver and glutamine in the kidney and the gut) are catabolized into pyruvate or into one of the metabolites of the citric cycle, which can then be converted into glucose. In contrast, ketogenic AA (e.g., leucine, isoleucine, phenylalanine) degraded into acetyl-CoA or acetooacetate are precursors of the ketone bodies that can be used as an alternate source of energy, in particular by brain cells. The AA carbon skeleton used as the energy source results from an AA deamination process, which leads to the cytosolic accumulation of toxic free ammonia (NH₄⁺). For further processing, NH₄⁺ is transported as glutamate and/or glutamine to the liver and kidneys (see Fig. 5), where ammonia is freed and processed in the urea cycle. Urea is a diffusible molecule, which is then excreted in the urine.

Regulation of these pathways, i.e., regulation of AA synthesis and degradation, depends heavily on substrate availability and allosteric mechanisms, which are mainly used in the short-term regulation of energy homeostasis. However, transcriptional regulation of genes for key enzymes is important for long-term adaptation to specific diets. Here, we will emphasize three aspects of AA metabolism for which transcriptional regulation has been shown to play a major role. We will first describe the regulation of glutamine homeostasis and of the urea cycle, two key determinants of the maintenance of the nitrogen balance under different physiological conditions. Then the specific regulatory pathways triggered by AA deprivation, which might be encountered in cases of general malnutrition or deficiency in any of the essential AA, will be discussed.

III. TRANSCRIPTIONAL CONTROL OF AMINO ACID AND PROTEIN METABOLISM

A. Introduction

In addition to being substrates for the synthesis of specialized products such as neurotransmitters, hemes,

B. Transcriptional Regulation of Glutamine Production and Homeostasis

As described above, glutamine has a particular status, and it is by far the most abundant AA in the human

TRANSCRIPTIONAL REGULATION OF METABOLISM

AA  
NH₄⁺  
Glutamate  

Metabolic  
Acidosis  
(in kidney)  

GLUTAMINE  

Storage of NH₃  
for AA synthesis  

Energy Source  
Transport and  
detoxification NH₃  

Transcriptional control  
via cAMP (i.e. energy  
deprivation)  

Peripheral Tissues  

Gluconeogenesis  
(kidney)  

Transcriptional control  
via glucocorticoids  

Liver cells  

Transcriptional  
control (HNF4α,  
C/EBPα, PPARα)  

UREA cycle  

Citric acid cycle  

FIG. 5. Transcriptional regulation of  
amino acid metabolism, with a particular  
emphasis on the central role of glutamine.  
Top: in peripheral tissues, particularly in  
the lungs and muscles, amino acid (AA)  
deamination leads to the formation of glu-  
tamate, which is converted to glutamine,  
via glutamine synthase. In these tissues,  
glutamine synthase expression is trans-  
scriptionally controlled by glucocorti-  
coids. Glutamine can be directly used as  
an energy substrate by tissues such as the  
gut. However, glutamine is a major carrier  
of ammonia being delivered to the liver  
where it can be disposed of (see bottom  
panel). Glutamine has a particularly im-  
portant role in the kidneys during meta-  
bolic acidosis as seen upon fasting (green  
pattern): the reverse reaction from glu-  
tamine to glutamate and α-ketoglutarate  
helps in excreting acid and provides a  
substrate for gluconeogenesis. Bottom: in  
the liver, glutaminase activity, under the  
control of cAMP, releases NH₃ for urea  
formation, which can be eliminated in the  
kidney. The urea cycle itself is also sub-  
jected to transcriptional control, as  
shown in Fig. 6. For more details, see  
section III B.

organism. In addition to being an important energetic and  
metabolic substrate, it provides sufficient amino groups  
for AA and nucleotide synthesis. It is also the main trans-  
porter of ammonia in the blood towards the liver, and  
glutamine generation in the brain is crucial for avoiding  
highly neurotoxic hyperammonemia. Thus it has a dual  
importance: during growth for anabolism and under cat-  
abolic conditions for limiting ammonia levels in periph-  
eral tissues and blood. Glutamine is required for normal  
growth and proliferation of cells, particularly of entero-  
cytes. Additionally, glutamine requirements are particu-  
larly high at times of severe sepsis, when proliferation of  
the immune cells is necessary (reviewed in Ref. 208).  
Conversely, depletion of glutamine due to a high cellular  
metabolic rate, often associated with high catabolism of  
AA, also occurs in cancer patients.

Glutamine formation via the ATP-dependent glu-  
tamine synthase (GS) occurs in most tissues (Fig. 5), with,  
in adults, the greatest activity in skeletal muscle, lung,  
brain, and adipose tissue. Whereas the GS turnover is  
increased by a high concentration of the end-product  
glutamine, regulation of GS activity also occurs at the  
transcriptional level, as best characterized by the respon-  
siveness of GS to glucocorticoids in the lungs and mus-  
cles. Two broad regions mediating this response have  
been identified in the upstream promoter and in the first  
intron of GS. Detailed studies of GS expression in chicken  
brains showed that glucocorticoids act by relieving the  
repression mediated by a silencer element located up-  
stream of the glucocorticoid receptor binding site (5).  
This glucocorticoid-mediated induction of GS occurs par-  

Physiol Rev • VOL 86 • APRIL 2006 • www.prv.org

α-ketoglutarate is linked to increased gluconeogenesis, most notably via the increased activity of PEPCK. Whereas the increased expression of glutaminase is mainly the result of mRNA stabilization via an element located in the 3′-UTR region of its mRNA (160, reviewed in Ref. 156), the expression and response to acidosis of PEPCK in the kidney depends on an HNF1 binding site present in its promoter (28).

Thus, whereas glutamine homeostasis seems to be one physiologically important knot of AA metabolism, most of the molecular mechanisms governing the regulation of these enzymatic activities remain to be analyzed. As glutamine synthesis is required, particularly in conditions of cell proliferation, decreasing glutamine intake has been proposed to control cell growth in cancer. It now appears that depletion of glutamine in cancer patients contributes to a global degradation of their health status. Consequently, a supplementation in glutamine is now on trial in these patients, as well as during severe sepsis for reinforcing the immune system (195).

### C. Transcriptional Regulation of Urea and Ammonia Homeostasis

Urea production in the liver, via the activity of the five enzymes of the urea cycle, carbamoyl phosphate synthetase 1 (CPS-1), ornithine transcarbamoylase (OTC), argininosuccinate synthase (ASS), argininosuccinate lyase (ASL), and arginase, is the main pathway for ammonia detoxification (Fig. 6). There are two main sources of ammonia production: the diet with its protein content, and endogenous protein degradation which occurs when there is a relatively low energy supply. The regulations occurring on a short time scale are mainly allosteric reactions, such as CPS-1 activation in the presence of N-acetylglutamate. However, there is a coordinated regulation of the expression of these five enzymes in response to dietary changes or to metabolic challenges during development and in adulthood. Hints regarding the molecular mechanism of this transcriptional regulation and the nature of the coordinating factor are starting to emerge.

Some important transcription factors for the regulation of ureagenesis have been revealed by the phenotype of several mutant mice. Liver-specific deletion of the nuclear receptor HNF4α results in hyperammonemia and hypouremia due to a dramatic reduction of the expression of OTC, consistent with the presence of two functional HNF4 response elements in the promoter of this gene. In contrast, the expression of the four other genes of the urea cycle remains unchanged or is only slightly increased (122). Thus the loss of the HNF4α function in ammonia detoxification might be the cause of the premature death of the mice carrying a liver-specific disruption of *HNF4α*. PPARα, on the other hand, is a negative regulator of the urea cycle, consistent with the PPARα-mediated downregulation of CPS-1, OTC, ASS, and ASL in mice treated with a fibrate (141). While the mechanism by which PPARα exerts this coordinated downregulation of urea enzyme gene expression is not yet known, it has been proposed that HNF4α positively regulates *PPARα* expression via a binding site for HNF4α in the promoter of this gene (231). Accordingly, the relatively low levels of PPARα in *HNF4α* null mice could thus explain the normal or higher levels of CPS-1, ASS, and ASL enzymes in these mice.

Severe hyperammonemia has been observed in mice carrying a liver-specific deletion of *C/EBPα*. This phenotype correlates with a dramatic fall in CPS-1 and arginase expression, together with a disturbed intrahepatic lobular distribution of OTC expression (147). Glucocorticoids,

FIG. 6. Transcriptional control of the urea cycle by HNF4α, PPARα, and C/EBPα. Amino acid catabolism leads to the formation of ammonia (NH₃) whose toxic accumulation is prevented by its processing to urea in the liver. The five enzymes coordinating the urea cycle are shown in blue: carbamoyl phosphate synthetase 1 (CPS-1), ornithine transcarbamoylase (OTC), argininosuccinate synthase (ASS), argininosuccinate lyase (ASL), and arginase. The transcriptional regulation of the expression of each enzyme by C/EBPα, PPARα, and HNF4α is indicated by solid, open, and hatched arrows, respectively. The amino acid carbon skeletons are further metabolized, generating substrates entering the citric acid cycle for energy production. For more details, see section III.C.

which have a strong impact on protein degradation in muscle, are efficient inducers of the urea cycle in the liver, allowing excretion in the form of urea of the excess ammonia produced by protein degradation. Intriguingly, the response of *arginase* to glucocorticoids via binding of the glucocorticoid receptor to its specific response element requires C/EBPβ expression and the integrity of the C/EBP binding sites in the 5′-flanking region of the gene (89). The same C/EBPβ dependency of the glucocorticoid response is also true of *CPS-1* (146). The *CPS-1* promoter contains a complex regulatory module composed of multiple sites for glucocorticoid receptor, HNF3 and C/EBP family members, as well as for unknown factors that control its specific pattern of expression and regulation. The glucocorticoid response unit itself combines a glucocorticoid response element with sites for HNF3β and C/EBP (41). Whereas these in vitro studies performed in primary hepatocytes in culture attributed a preferential role to C/EBPβ, C/EBPβ null mice did not present any perturbation of ureagenesis (40, 147), and it seems reasonable to propose that in vivo C/EBPα is the functional partner of the glucocorticoid receptor. With respect to HNF3β, an embryonic lethality of *HNF3β* null mutation preempts the analysis of the role of this factor in vivo, whereas *HNF3α* and *HNF3γ* null mice do not exhibit any perturbation of ammonia metabolism. Thus the phenotype of liver-specific KO of *C/EBPα* together with the subordination of the glucocorticoid response to C/EBP support the hypothesis that C/EBPα is the main coordinator of the expression of urea cycle genes.

The above discussion on the search for a coordinator of the ureagenesis pathway underscores the complex interplay established by transcription factors and pinpoints a general mode of homeostasis regulation, in which equilibrium is obtained via the simultaneous control of opposite pathways. One example here is the glucocorticoids that activate the urea cycle via C/EBPs, but at the same time increase the levels of PPARα, which is an inhibitor of the same cycle. Another example is the regulation of PPARα expression by HNF4α, which contributes to keeping a balance between activation and inhibition of urea cycle activity. A third example is that of *GS* and *glutaminase*, which have opposite activities but are both upregulated in the liver by glucocorticoids (see sect. III.B). While somewhat counterintuitive, such a mode of regulation should lead to a fine-tuning that limits the oscillations of feed-back and feed-forward regulations.

D. Amino Acid Deprivation and Induction of Gene Expression

1. The search for an amino acid response element

Two genes have been extensively explored for their ability to respond to amino acid deprivation. C/EBP homologous protein (*CHOP*) is induced by various stresses. One of these is the unfolded protein response pathway triggered by the accumulation of unfolded proteins in the endoplasmic reticulum (ER), which activates chaperones resident in the ER. CHOP is related to the C/EBP family of nuclear factors with which it forms heterodimers. Interestingly, global AA deprivation or starvation in individual AA induces *CHOP* expression independently of the ER stress pathway. A short promoter sequence or AA response element (AARE) conveys the AA sensitivity of *CHOP*. This sequence combines the features of a C/EBP consensus element and a cAMP response element (22). The second gene that is used as a model to explore the transcriptional response to amino acid deprivation is the gene encoding asparagine synthetase (AS). AS catalyzes the glutamine- and ATP-dependent conversion of aspartate to asparagine. AS mRNA accumulates in mammalian cell cultures in response to asparagine starvation. Amazingly, deprivation of a wide range of individual AA also induces this accumulation, therefore suggesting that AS responds to a signal reflecting AA deprivation more broadly. Two discrete response elements, called nutrient sensing response elements NSRE-1 and NSRE-2, were found in a short promoter region of human AS. Both of them are required for AS activation and form a nutrient-sensing response unit that mediates not only the response to AA deprivation, but also the response to glucose deprivation (284 and references therein).

The identity of the factors that bind to these elements and are thus responsible for the response to nutrient deprivation remains disputed. They belong to the activating transcription factor (ATF)/CREB family, which includes members sharing a basic leucine zipper motif and a consensus ATF/CRE DNA binding site “TGACGTC” (reviewed in Ref. 97). The CRE-binding protein 1 (CREBP1 or ATF2) binds to the C/EBP-ATF composite site forming the AARE of *CHOP*, either as a homodimer or as a heterodimer with an unknown dimerization partner (22). ATF4, but not ATF2, binds as a complex with C/EBP to NSRE-1 of *CHOP* and is required for the response of AS to nutrient deprivation (283). ATF4 itself is transcriptionally and posttranscriptionally regulated by both AA and glucose deprivation (100, 283), which suggests that ATF4 is an important transcriptional mediator of the nutrient-sensing response. However, the response of AS to glucose deprivation is mediated via the unfolded protein response pathway and may be independent of the response to AA deprivation.

Taken together, the differences between and similarities in the regulation of these two genes underline the existence of at least two related but independent pathways, via ATF2 and ATF4, respectively, which control gene expression in response to AA deprivation (21).

478 BEÁTRICE DESVERGNE, LILIANE MICHALIK, AND WALTER WAHLI

2. TOR as a master switch for catabolism versus anabolism

Studies carried out to decipher the yeast response to AA deprivation have identified the target of rapamycin (TOR) proteins as master switches for protein/AA catabolism versus anabolism (125). TOR belongs to the PI3K-related kinase family and appears to function as a nutrient-sensing check-point by controlling many aspects of mRNA translation. Inhibition of TOR proteins by rapamycin in yeast mimics nutrient deprivation. In fact, TOR modulates the transcription of genes involved in AA biosynthesis and the activity of permeases that allow AA transport into the cells. It also inhibits autophagy in yeast and in mammalian cells, a process that degrades cytoplasmic proteins and organelles for scavenging AA when nutrient levels are low. When a sufficient amount of nutrients is sensed, TOR proteins act as a permissive signal for growth and protein synthesis. A single mammalian homologous TOR protein, alternatively called mTOR, FRAP, or RAFT1, has been cloned in various species with a remarkable level of AA identity, suggesting well-conserved functions of the TOR-dependent regulatory pathways. In mammalian cells, as well as in *Drosophila*, TOR is not only sensitive to the presence of sufficient levels of AA, but also integrates energy and growth signals through the AMPK and PI3K pathways, respectively. Increased Akt activity, via insulin signaling for example, would trigger the phosphorylation of the tuberous sclerosis complex (TSC) relieving the constitutive inhibition that TSC exerts on mTOR activity (reviewed in Ref. 125). In addition to this regulatory interaction, recent evidence has demonstrated that rapamycin-sensitive mTOR kinase activity requires the direct interaction of the small GTPase Rheb-GTP with the TOR-containing complex TORC1 (177).

The best-known molecular mechanism of TOR action is a posttranscriptional action on the phosphorylation status of the initiation and elongation factors involved in translational control (reviewed in Ref. 240). However, it also regulates the abundance of the components of the translation machinery both at the transcriptional and translational levels. This results, for example, in controlling the translational events that regulate mammalian cell size (71). At the transcriptional level, TOR modulates the expression of numerous enzymes involved in multiple metabolic pathways. In yeast, this transcriptional control is mainly exerted by the cytoplasmic sequestration of transcription factors. TOR controls *ribosomal protein* (*RP*) gene transcription by maintaining the corepressor CRF1 in the cytoplasm, thereby allowing the forkhead-like transcription factor FHL1 and its coactivator IFH1 to efficiently activate *RP* gene transcription. Upon TOR inhibition, phosphorylated CRF1 rapidly translocates to the nucleus inhibiting *RP* transcription (131, 190).

While TORC1 and forkhead-associated domain-containing forkhead transcription factors are conserved from yeast to humans, little is known about the transcriptional mechanisms involved in multicellular organisms. Gene expression profiling in lymphocyte cell lines demonstrated that rapamycin, which inhibits TOR, upregulates genes involved in AA oxidation, fatty acid oxidation, and nucleotide salvage pathways, while it downregulates genes involved in lipid and protein biosynthesis. Furthermore, it was shown that rapamycin and AA deprivation act on overlapping but not identical sets of genes (227). Glutamine deprivation resulted in a broader overlap with rapamycin in terms of gene expression profiles. This reinforces the notion of a parallel between the high increase in the demand for glutamine when the immune system is challenged and the potent immunosuppressive effect of rapamycin. However, the molecular mechanisms of these transcriptional regulations remain entirely unexplained.

IV. TRANSCRIPTIONAL CONTROL OF LIPID METABOLISM

Lipids occur in three major classes of molecules in multicellular organisms: 1) fatty acids, as a source of energy, mainly stored in form of triglycerides in fat tissues; 2) phospholipids and cholesterol, which are major structural components of the cellular membranes; and 3) lipid-derived small bioactive molecules, such as the steroid hormones, the arachidonic acid derivatives (prostaglandins and leukotrienes), and other intracellular messengers that are present in small quantities and have crucial roles in signaling. In this section, we concentrate on the transcriptional mechanisms that govern the synthesis, storage, release, uptake, and oxidation of fatty acids as the main participants in energy homeostasis and on the intricate control of cholesterol metabolism.

A. Transcriptional Control of Fatty Acid Metabolism

Transcriptional regulation of the genes involved in fatty acid metabolism is presently considered as the major long-term regulatory mechanism controlling lipid homeostasis. It is executed by a variety of transcription factors among which the SREBPs, the C/EBPs, and members of the nuclear receptor family are particularly active agents.

1. Fatty acid synthesis and storage: control by an intricate array of transcription factors

Adipogenesis is a developmental process by which cells become determined and differentiate to fulfill adipose tissue specific functions, among which are fat stor-

age and production of adipose hormones (also called adipokines). Lipogenesis results in cellular lipid accumulation, via the uptake of lipogenic substrate from the diet, endogenous fatty acid synthesis, and fatty acid storage as triglycerides. It should be noted that lipolysis, which releases fatty acids into the blood, is the reciprocal important function of the white adipose tissue (WAT). However, little is known on the transcriptional regulation of lipolysis that will be discussed in the context of adaptation to fasting (see sect. VII).

A) SREBP-1, A MAJOR TRANSCRIPTION FACTOR INVOLVED IN FATTY ACID SYNTHESIS. A brief overview of the SREBPs is shown in Appendix G. The maturation process of SREBPs via a cholesterol sensing mechanism initially emphasized their role in cholesterol homeostasis (see Fig. 7). However, whereas the main role of SREBP-2 is indeed geared toward cholesterol metabolism, SREBP-1a and -1c have been clearly associated with both cholesterol and fatty acid homeostasis. SREBP-1a appears to be constitutively expressed at quite low levels and mainly in the liver (116, 280). Overexpression of the nuclear form of SREBP-1a in transgenic mice leads to a massive engorgement of the liver with triglycerides and to a lesser extent with cholesteryl (277). SREBP-1c, whose lower transcriptional activity has been related to its shorter NH₂-terminal tail, acts more specifically on genes involved in fatty acid synthesis. It was initially identified in the WAT and named adipocyte determination and differentiation factor 1 (ADD-1) (297). It is highly expressed in the liver and the WAT and is sensitive to multiple regulatory signals. Overexpression of its nuclear form strongly increases the triglyceride content of the liver, with no parallel accumulation of cholesterol. While a null-mutation of SREBP-1 (deleting both SREBP-1a and SREBP-1c) provokes an embryonic lethality that is partially penetrant, analyses of surviving SREBP-1⁻/⁻ mice show reduced synthesis of fatty acids, whereas the cholesterol synthesis pathway is increased due to a compensatory expression of SREBP-2 (see Ref. 114 for review).

Although the posttranscriptional maturation of SREBPs is an important regulatory event, the transcriptional regulation of SREBP-1c expression parallels the activity of the transcription factor. As discussed in section II.B, one major metabolic signal that upregulates SREBP-1c is insulin, whereas glucagon represses it. Insulin, whose release is stimulated by high blood glucose levels, induces the production of fatty acids from glucose-derived pyruvate in the liver and the adipose tissue. Most of the lipogenic effects of insulin are dependent on the induced expression of SREBP-1c and the subsequent stimulation of the fatty acid synthesis pathway. SREBP-1c also positively regulates its own production in a feed-forward loop that ensures higher SREBP-1c production when the levels of the mature nuclear form are elevated (1). SREBP-1c expression is also stimulated by the LXR, via two LXR binding sites (LXRE) present in the SREBP-1c promoter. The primary role of LXR was thought to be in the control and protection of cells from cholesterol overload. Thus the reason why LXR would trigger a parallel increase in fatty acid synthesis via transcriptional regulation of SREBP-1c remains unclear. One hypothesis is that the blood transport and cell storage of excess cholesterol requires the formation of cholesterol ester. This increases the demand for oleate production,

ER membrane

![Diagram](attachment:diagram_1.png)

Golgi apparatus

![Diagram](attachment:diagram_2.png)

FIG. 7. Maturation process of the SREBP transcription factor. This figure accompanies Appendix G (see text of Appendix G).

which is indeed stimulated upon overexpression of SREBP-1c (243, 272).

Conversely to these positive signals, high levels of polyunsaturated fatty acids (PUFA) repress SREBP-1c expression. Several mechanisms have been proposed for this downregulation. A functional analysis of the SREBP-1c promoter demonstrated a PUFA-dependent inhibition of LXR binding to the promoter (219, 336). PUFA may also accelerate the decay of SREBP-1c transcripts as demonstrated in rat hepatocytes (326) or may act on the process of SREBP-1c protein maturation, thereby decreasing SREBP-1c activity. Indeed, whereas the post-translational maturation of SREBP-2 depends tightly on the abundance of membrane cholesterol, the inhibition of the SREBP-1 cleavage and maturation seems to depend on the presence of PUFAs (reviewed in Ref. 63). However, little is known about the relative importance of this regulation. Further understanding the role of the PUFA-mediated repression of SREBP-1c is of importance as it may be the main mechanism by which dietary PUFAs inhibit lipogenesis.

As shown in Figure 8, the action of SREBP-1c in fatty acid synthesis is mediated by the activation of a set of target genes. The DNA elements in the promoter of SREBP-responsive genes are surprisingly diverse, and defining a consensus SRE is thus difficult. The intricate localization of an SRE with an E-box motif as seen in the FAS promoter does not seem to be the rule. In addition, SREBP-1 can bind to both an E-box motif or to a non-E-Box SRE (143). Nonetheless, the transcriptional activity of SREBPs often requires cooperation with other DNA binding transcription factors such as SP1, NF-Y, and CREB as well as with coactivators (reviewed in Ref. 63). Whereas further work is on-going to elucidate the molecular interactions involved in SREBP-mediated transcriptional activation, the evidence discussed above clearly emphasizes the role of SREBP-1c in increasing fatty acid synthesis in liver and in adipose tissue, particularly in response to insulin.

B) PPARγ: A MAJOR REGULATOR OF FATTY ACID STORAGE AND ADIPOGENESIS. Mature adipocytes are cells that sustain efficient lipogenesis and triglyceride synthesis and storage, but also release fatty acids and glycerol into the blood via lipolysis. The mature adipose tissue also secretes many endocrine signals whose importance and mechanism of action are presently intensively scrutinized in the context of whole body energy metabolism.

It is believed that the developing and mature adult adipose tissues constitute a life-long reservoir of preadipocytes that can be triggered to differentiate in adipocytes if the need for fat storage is signaled. The peroxisome proliferator activated receptor γ (PPARγ) has been clearly linked to the adipocyte differentiation program (reviewed in Ref. 258). At the protein level, there are two PPARγ isoforms, PPARγ1 and PPARγ2. PPARγ2 has 30 and 28 additional NH₂-terminal amino acids in mice and humans, respectively. Both isoforms are produced from the same gene by alternative promoter usage and mRNA splicing. PPARγ1 is mainly expressed in adipose tissues but is also detected in the colon, spleen, retina, hematopoietic cells, liver, and skeletal muscle, whereas PPARγ2 expression is restricted to the adipose tissue (18, 66 and reviewed in Ref. 54). However, the two isoforms have identical ligand-binding properties and share the same target genes (257), for instance, those coding for the adipocyte fatty acid binding protein (aP2), lipoprotein lipase (LPL), acyl-CoA synthase (ACS), and fatty acid transport protein (FAT/CD36). PPARγ is a late marker of

FIG. 8. Sterol response element binding protein (SREBP)-2 and SREBP-1c at the branching point of cholesterol and fatty acid metabolism. SREBP-1c is a transcription factor activating genes encoding enzymes involved in fatty acid synthesis and driving the formation of triglycerides and phospholipids (right). In contrast, SREBP-2 stimulates the transcription of genes encoding enzymes involved in cholesterol synthesis (left). These two pathways require the generation of NADPH, provided by the reactions shown in the middle of the figure. Both SREBP-1c and SREBP-2 positively regulate the three enzymes participating in these reactions, i.e., malic enzyme (ME), glucose-6-phosphate dehydrogenase (G6PDH), and 6-phosphogluconate dehydrogenase (PGDH). For more details, see section IV.

adipocyte differentiation, and its artificial expression is sufficient to force fibroblasts into the adipogenic program. Whereas $PPAR\gamma$ null mice are not viable, due to defects in placenta formation (9), the lack of adipocytes carrying the genotype $PPAR\gamma -/-$ in chimeric $PPAR\gamma +/+ : PPAR\gamma -/-$ mice has demonstrated the importance, in vivo, of $PPAR\gamma$ for adipogenesis (255).

The role of $PPAR\gamma$ in the transcriptional control of metabolism in mature adipose tissue is less clear. The overexpression of a dominant negative mutant of $PPAR\gamma$ in fully differentiated mature 3T3-L1 adipocytes results in increased lipolysis and decreased fatty acid uptake, concomitant with a decreased expression of known $PPAR\gamma$ target genes, including $aP2$, $LPL$, $ACS$, and $CD36$ (293). In vivo, synthetic $PPAR\gamma$ ligands, the glitazones, increase the number of adipocytes rather than modulate the mature adipocyte functions. The availability of full $PPAR\gamma$ antagonists should be an interesting tool for deciphering, in vivo, the role of $PPAR\gamma$ in adult tissues. However, the antagonists described so far either have a residual agonist activity or are hardly soluble and/or toxic (204, 211, 251). Recently, a new $PPAR\gamma$ antagonist has been tested in vivo, demonstrating that inhibition of $PPAR\gamma$ activity results in a decrease in white and brown adipose tissue fat deposits to a similar extent as that observed in $PPAR\gamma$ heterozygous mice. This reduction reflects a decrease in the average adipocyte volume (247), which could either be due to a lesser fat accumulation or to an increased lipolytic activity. The important role of $PPAR\gamma$ in the mature tissue is now confirmed by the phenotype of mice with an adipose tissue-specific deletion of $PPAR\gamma$. As the expression of the CRE enzyme responsible for the gene deletion is under the activity of the $aP2$ promoter, the deletion occurs after adipogenesis has taken place. This deletion results in a severe reduction of the number of mature adipocyte, both in white and brown adipose tissues, whereas small and likely nascent adipocytes are appearing. Thus $PPAR\gamma$ seems to be essential for the survival of mature adipocytes (103, 121). The implication of $PPAR\gamma$ in the stimulation or inhibition of lipolysis has not yet been addressed.

Adipogenesis via $PPAR\gamma$ activation may occur in other tissues than the adipose tissue. A forced expression of $PPAR\gamma 1$ in hepatocytes induces the classic pattern of $PPAR\gamma$-mediated gene activation and results in steatosis (338). These observations recapitulate the liver phenotype of murine models of diabetes and obesity that are marked by elevated levels of $PPAR\gamma$ in the liver. However, the normal expression of $PPAR\gamma$ in the liver is very low and, consistently, a liver specific deletion of $PPAR\gamma$ gives no phenotype unless this deletion is performed in a murine model of obesity such as $ob/ob$ mice (192). Specific deletion of $PPAR\gamma$ in the skeletal muscle also results in a lack of phenotype with respect to growth and lipid distribution, whereas there is increased adiposity and development of insulin resistance upon exposure to a high-fat diet (107, 209) (see also sect. vi). This emphasizes the link between muscle metabolism and adipose tissue response, the nature of which remains to be elucidated. Accumulation of lipids via $PPAR\gamma$ might also occur in smooth muscle cells and/or in activated macrophages. At the same time, $PPAR\gamma$ has also been implicated in cholesterol export from macrophages, thereby counterbalancing the detrimental effects of lipid loading (reviewed in Ref. 162). However, the complex role of $PPAR\gamma$ in foam cell formation and atherosclerosis goes beyond the scope of the present review.

A puzzling observation made a decade ago was that glitazones, which were developed for treatment of insulin resistance, are $PPAR\gamma$-selective ligands. The link between the promotion of adipocyte differentiation and lipid storage by $PPAR\gamma$ and the antidiabetic effects of these compounds is not fully understood and several hypotheses, based on a wealth of experiments, have been proposed. These are discussed in section vi.

C) THE PART PLAYED BY C/EBP IN ADIPOGENESIS AND LIPOGENESIS. The involvement of C/EBPs in adipogenesis can be summarized as follows. C/EBP$\beta$ and $\delta$ are expressed transiently in the earliest steps of adipocyte differentiation; they activate the expression of $PPAR\gamma$ and C/EBP$\alpha$, both of which are involved in the terminal differentiation of the adipocyte. Overexpression of C/EBP$\beta$ triggers the differentiation of preadipocytes into adipocytes in the absence of adipogenic hormones, whereas an inhibition of C/EBP$\alpha$ results in a blockade of late adipocyte differentiation (173, 267).

Compared with these cell culture studies, the phenotype of mice carrying null allele mutations for the various C/EBPs is more difficult to interpret. C/EBP$\alpha$ null mice exhibit a severe reduction of brown and white fat mass (312), which may result from impaired lipogenesis, and thus little lipid accumulation, rather than from altered adipocyte differentiation. This distinction is also suggested by the phenotype of either C/EBP$\beta$ or C/EBP$\delta$ null mice. Whereas little alteration is seen in the adipose tissue of these animals, mice carrying the double null allele mutation exhibit an important hypoplasia of WAT reflecting an impaired adipocyte differentiation program, and a moderate reduction of the brown adipose tissue mass attributed to reduced lipogenesis (295). Interestingly, these observations point to distinct regulations that may occur through the same transcription factors in brown and white adipose tissues (reviewed in Ref. 253).

The target genes of the C/EBPs that mediate their effects in fatty acid metabolism remain largely unknown. The transcription factor cascade described for adipogenesis suggests that C/EBPs coordinate the differentiation program by controlling the expression of transcription factors such as C/EBP$\alpha$ itself and $PPAR\gamma$ rather than by acting on direct effectors (274). Furthermore, because

C/EBPs are considered as constitutive regulators, their activity has to be regulated by increased or decreased expression. The two main hormonal signals known so far to trigger C/EBP expression in the liver are glucocorticoids and glucagon, hormones that are secreted in response to stress, fasting, or hypoglycemia. Glucocorticoids increase C/EBPα and C/EBPβ levels in the liver (45, 89), which in turn stimulate gluconeogenesis (see sect. III.C). Glucagon, and more particularly increased cAMP levels, are also an important metabolic signal that upregulates C/EBPβ expression in hepatocytes (44, 45). The regulation in fat tissue is somewhat different, since glucocorticoids decrease C/EBPα expression in WAT (184). In addition, and as mentioned above, insulin induces C/EBPβ in 3T3-L1 cells, via an increased SREBP-1c expression. Finally, C/EBPs are highly phosphorylated; however, the many studies aimed at deciphering how the phosphorylation pattern of C/EBPs might alter their metabolic-directed activities have not delivered a clear answer so far (238).

In summary, C/EBPs are linked to fatty acid metabolism via their implications in adipogenesis. However, their molecular mode of action remains elusive.

D) PPARβ: A ROLE IN ADIPOGENESIS? Reasons for the scarce knowledge gained so far of PPARβ functions are its ubiquitous expression and the lack of specific synthetic and natural ligands until recently. Carbaprostacyclin (cPGI), a stable analog of prostacyclin (PGI₂), acts as an agonist of PPARβ, supporting the notion that the cyclooxygenase-2 arachidonate metabolite PGI₂ might itself act as a bona fide natural ligand for PPARβ (273). Like PPARα and PPARγ, PPARβ binds fatty acids and, therefore, is also most likely a sensor of dietary lipids and lipid derivatives (reviewed in Ref. 196). In preadipocytes, PPARβ mediates long-chain fatty acid effects on the expression of adipose-related genes (2). Together with two additional transcription factors, C/EBPβ and C/EBPδ, PPARβ appears to be implicated in the induction of PPARγ expression (12, 110). In turn, high expression of PPARγ and C/EBPα in adipocytes establishes and maintains the terminal differentiation program. According to this scheme, activation of PPARβ by dietary lipids in preadipocytes would contribute to the expansion of the adipose tissue, a role consistent with the decreased amount of brown and white adipose tissues reported in the *PPARβ* null mice (8, 228). However, these observations contrast somewhat with other evidence reported by the same investigators. Adipose tissue-specific deletion of *PPARβ* does not alter fat mass (8), whereas PPARβ overexpression and overactivation in the WAT trigger fatty acid mobilization and oxidation (313). Overexpression of PPARβ in C2C12 myoblasts participates in their transdifferentiation in adipocytes (110), while a PPARβ agonist induces fatty acid oxidation in differentiated muscle cells (111). Indeed, there is now compelling evidence which supports a role

for PPARβ in fatty acid oxidation in muscle, as is discussed below. However, this activity in mature muscle cells does not preclude a specific role of PPARβ in the very early stages of adipocyte differentiation.

E) DIVERSE ARRAYS OF SIGNALS PARTICIPATE IN THE TUNING OF THE ADIPOGENESIS PROCESS. We have seen that adipogenesis is a highly regulated process in which C/EBPs and PPARγ play a major part. However, there is a diverse array of factors that completes the scheme. For example, HNF3β/FOXA2 is expressed in preadipocytes as well as in adipocytes and inhibits adipocyte differentiation (321). FOXC2 is a winged helix/forkhead transcription factor that induces the expression of the type Iα regulatory subunit of cAMP-dependent PKA (29). Consequently, FOXC2 increases the sensitivity to PKA activation (29) and was shown to inhibit white adipocyte differentiation (49). Other transducing signals acting on adipocyte differentiation involve SMAD3, GATA2, and GATA3 (reviewed in Ref. 101), but further work is needed to integrate these factors in a global representation of the transcriptional regulation network controlling adipogenesis.

2. Fatty acid oxidation: regulation by PPARα and PPARβ

In contrast with the numerous factors involved in fatty acid synthesis and storage, our knowledge of the transcriptional regulation of fatty acid oxidation has long been dominated by the major role of PPARα. However, in addition to recent work suggesting the potential role of PPARβ in fatty acid oxidation, other transcription factors certainly participate in the coordinated induction of the fatty acid oxidation enzymes. To gain some insight as to the nature of these factors, the regulation of the expression of the carnitine palmitoyltransferase I (*CPT-1*) gene, encoding a major rate-limiting enzyme in fatty acid oxidation, is used as a model and will be discussed in this section.

A) PPARα: A MAJOR TRANSCRIPTIONAL REGULATOR OF FATTY ACID OXIDATION. PPARα expression levels are the highest in brown adipose tissue and the liver, then in the heart, kidneys, enterocytes, and muscles. PPARα target genes constitute a comprehensive set of genes that participate in many if not all aspects of lipid catabolism. This includes fatty acid transport across the cell membrane (fatty acid transporter protein genes), intracellular binding (liver fatty acid binding protein gene), activation via the formation of acyl-CoA (long chain fatty acid acyl CoA synthase gene), catabolism by β-oxidation in peroxisomes and mitochondria, and catabolism by ω-oxidation in microsomes (acyl-CoA oxidase gene, CYP4A1 and CYP4A6 genes, medium-chain acyl-CoA dehydrogenase, and 3-hydroxy 3-methylglutaryl-CoA synthase genes) (reviewed in Ref. 54). The role of PPARα in fatty acid oxidation is particularly highlighted during fasting that results in an

enhanced load of fatty acids in the liver, to be used as energy source. Food deprivation provokes an increased expression and activity of PPARα, which stimulates β-oxidation. PPARα null mice, which are viable and exhibit only subtle abnormalities in lipid metabolism when kept under normal laboratory confinement and diet (163, 224), cannot sustain fasting. Their inability to enhance fatty acid oxidation results in hypoketonemia, associated with severe hypothermia and hypoglycemia (142, 170). Thus PPARα is crucial for the organism to adapt to increased demand in fatty acid oxidation, while it seems to play a marginal role in the basal situation with normal diets.

Most of the initial studies of PPARα and its role in fatty acid oxidation have been performed in the liver. However, the major site of fatty acid oxidation at rest and during exercise is the skeletal muscle, which expresses PPARα. Interestingly, a clinical study in lean women demonstrated that endurance training increases PPARα expression in muscle, together with increased expression of medium-chain acyl-CoA dehydrogenase (MCAD) and very-long-chain acyl-CoA dehydrogenase (VLCAD) (113), thereby increasing the oxidative capacity of the muscle. Surprisingly, PPARα null mice have a relatively mild phenotype in this respect, characterized by a precocious exhaustion when exercising. In normally fed animals, the fatty acid oxidative capacity in muscle is similar in wild-type and PPARα null mice. Upon starvation, the fatty acid oxidative capacity in muscle of null mice is reduced by only 30%, suggesting that other factors are important for the regulation of this pathway (206). This is in agreement with the coordinated induction of many genes involved in fatty acid oxidation in the muscles of streptozotocin-induced diabetic mice, which is paradoxically associated with a decreased expression of PPARα (333).

In cardiac muscle, PPARα is increased at the transition from fetal to adult cardiomyocytes. This step is characterized by a switch in the source of energy from glucose and lactate to fatty acids and by high expression levels of the PPARα target genes involved in their oxidation. PGC-1, which increases in parallel, may cooperate with PPARα for the induction of the genes encoding enzymes of the fatty acid oxidation pathway (11). In pathological cardiac hypertrophy, the expression of PPARα is downregulated, the utilization of fatty acids as energy substrate is decreased, and the genes implicated in the utilization of glucose as the main energy source are reinduced (10). It is presently unclear whether the decline in PPARα activity and fatty acid oxidation is a cause or a consequence of cardiac hypertrophy. It is therefore also unclear if PPARα ligands would be beneficial or detrimental to the heart in this pathological context (337; reviewed in Ref. 167). In this context, it is interesting to note that PGC-1α is a crucial regulator of mitochondrial biogenesis (236). It interacts with the nuclear respiratory factor NRF1, which positively regulates the nuclear genes that are involved in

the biogenesis of the respiratory chain (68, 323). The expression of PGC-1α also dramatically increases at birth when the heart has an increased requirement for mitochondrial respiration (166). Overall, in the heart, PGC-1 could coordinate the increased capacity of the respiratory chain via its interaction with NRF1 and the increased fatty acid oxidation which provides the fuel for the respiratory chain via a functional interaction with PPARα.

B) PPARβ AND FATTY ACID OXIDATION: OVERLAPS AND SPECIFICITIES WITH RESPECT TO PPARα FUNCTIONS. The first studies linking PPARβ to fatty acid oxidation in muscle were performed in cultures of primary muscle cells or muscle cell lines, where activation of PPARβ induced UCP2 and UCP3 gene expression (38, 206). In C2C12 cells, a PPARβ-dependent upregulation of H-FABP and FAT, and to a lesser extent of LPL, ACS, and CPT1 was observed (58, 110). In vivo, transgenic mice that overexpress PPARβ or a constitutively active PPARβ-VP16 fusion protein in muscle exhibit an enrichment of the muscle in red oxidative fibers, with an increased oxidative capacity assessed both at the gene expression and functional levels (182, 314). Similar results were obtained in wild-type mice treated with the PPARβ agonist (GW501516). Such a treatment results in a dose-dependent activation of fatty acid β-oxidation in the quadriceps muscles, sustained by the higher expression of genes encoding enzymes involved in mitochondrial fatty acid catabolism, such as fatty acid transport proteins (FAT and LCAD) as well as UCP2 and UCP3 (294). Overlapping activities of PPARβ with PPARα with respect to fatty acid oxidation are also described in the heart, where a mutation of PPARα is partially compensated by PPARβ (36, 50). In this organ, PPARβ may have a dominant role, as a cardiac specific disruption of PPARβ results in a cardiomyopathy that develops as early as at 4 mo of age and is associated with a general decrease in the cardiac expression of all genes involved in β-oxidation (35).

In the adipose tissue, and in contrast to the proposed role of PPARβ in adipogenesis (see previous section), PPARβ may also be involved in fatty acid oxidation. Overexpression of the PPARβ-VP16 fusion protein in brown and white adipose tissues produces lean mice and increases the mobilization and oxidation of fatty acids (313). This is due, at least in part, to the concomitant increased expression of genes involved in fatty acid β-oxidation (long-chain acyl-CoA dehydrogenase, LCAD; VLCAD), lipolysis (hormone-sensitive lipase, HSL), and energy uncoupling in mitochondria (uncoupling proteins UCP1 and UCP3). The increased catabolism of fatty acids in these transgenic mice is further illustrated by the protective effects of adipose PPARβ-VP16 expression against obesity, hyperlipidemia, and liver steatosis upon high-fat diet feeding.

An interesting aspect here is the consistent upregulation of the uncoupling proteins in various tissues. This suggests that in the adipose tissue and the skeletal muscle, PPARβ expression and activity are also linked to

energy dissipation, in addition to facilitating the use of fatty acid as an energy source (reviewed in Ref. 13).

C) TRANSCRIPTIONAL REGULATION OF CPT-I: A COMPLEX ARRAY OF TRANSCRIPTION FACTORS. CPT-I is a major rate-limiting enzyme of fatty acid oxidation, and functional analyses of *CPT-I* promoter have revealed the contribution of other transcription factors to the regulation of fatty acid oxidation. The CPT system controls the transfer of long-chain fatty acids inside the mitochondria. CPT-I, which is localized in the outer mitochondrial membrane, is considered as a rate-limiting enzyme for the oxidation of long-chain fatty acids. The major regulator of CPT-I activity consists of an allosteric control via malonyl-CoA, a substrate for fatty acid synthesis whose accumulation inhibits CPT-I activity.

CPT-I is expressed as two distinct isoforms, L-CPT-I (also called CTP-Iα in the liver) and M-CPT-I (also called CPT-Iβ in muscles and adipose tissue). Transcription of L-CPT-I is under the control of the thyroid hormone receptor and PPARα, through appropriate response elements found in the promoter of the gene (128). A sequence element in the first intron is also proposed to mediate a PPAR-independent transcriptional regulation by long-chain fatty acids (126, 178, 180). In addition, glucagon can induce L-CPT-I expression through increased cAMP levels. Intriguingly, the cAMP response unit is a composite of the DR1 sequence, previously shown to mediate PPARα induction, and of a CREB binding element. As already mentioned (see sect. III.C), PGC1 is strongly activated by CREB in liver in response to cAMP and potentiates HNF4α and glucocorticoid/glucocorticoid receptor action onto the promoter of the gluconeogenic enzyme PEPCK (106). A similar mechanism is now proposed to activate L-CPTI via the formation of a multiprotein complex containing PGC1, CREB, and HNF4α (179). Chromatin immunoprecipitation might now be of help to analyze the composition of the transcriptional complex that binds to this composite element.

Much less is known about the regulation of M-CPTI in muscle. An interaction of PGC1 with the muscle-specific myocyte enhancer factor 2 (MEF2) has been proposed to increase the expression of M-CPT. USF proteins would in contrast repress M-CPT-I gene expression, possibly by interacting with the cofactor PGC1, and therefore reduce MEF2 activity (202). Thus the functional network of transcription factors must be broadened to include that of the coactivators. While not unexpected, this leads the way for further understanding of the complexity of metabolic regulations.

Overall, our broad understanding of the mechanisms controlling lipogenesis and adipogenesis differs strikingly from our relatively poor understanding of the transcriptional control of fatty acid oxidation. Does this reflect the respective diversity and paucity of the mechanisms involved in these two processes? Recent progress towards

understanding the transcriptional control of fatty acid oxidation will help to build a more comprehensive model of the necessary balance between storage and consumption.

### B. Transcriptional Control of Cholesterol Homeostasis

#### 1. An outline of cholesterol metabolism and its main regulatory factors

Cholesterol is an essential component of cell membranes and is therefore critical for cell viability and growth. It is also the precursor from which steroids and bile acids are synthesized.

The main source of cholesterol is the diet. Its inclusion in micelles, formed by bile acids in the intestine, facilitates its uptake as well as that of fatty acids by the enterocytes. In these cells, lipids are incorporated into chylomicrons, then secreted into the lymph for delivery into the bloodstream. Chylomicrons deliver triglycerides to peripheral tissues via LPL-mediated hydrolysis, and their cholesterol-enriched remnants are taken up by the hepatocytes. In the liver, cholesterol is channeled to bile acid synthesis or is reincorporated into the very-low-density lipoprotein (VLDL) particles that are redelivered into systemic circulation. Lipoproteins are complex particles that allow the transport and circulation of lipids such as phospholipids, free cholesterol, esterified cholesterol, and triglycerides, in the blood. The proteins associated with the particles, or apoproteins, serve as lipid acceptors, enzyme cofactors, or ligands for receptor-mediated cellular uptake. Each lipoprotein particle has distinct properties and distinct destinations. Among them, VLDL particles are assembled in the liver and redistribute triglycerides and cholesterol to peripheral cells. They become LDL after a partial depletion in triglycerides has occurred, due to peripheral lipoprotein lipase activity. LDL are considered as the “bad” cholesterol compartment, due to their poor clearance from the blood, in contrast to HDL, which contain apoAI and apoAII proteins that serve as cholesterol acceptor and which are efficiently cleared from the blood by the liver. VLDL form the primary source of cholesterol for peripheral tissues, mainly via the low-density lipoprotein (LDL) receptor expression which is a main regulatory step for adjusting the import of cholesterol (reviewed in Ref. 340). Its export from cells requires the expression of transporters of the ATP binding cassette (ABC) superfamily and the extracellular presence of specific apolipoproteins as free cholesterol acceptors. These proteins are major components of the reverse cholesterol pathway and are therefore crucial for the efflux of excess cellular cholesterol. At the level of the whole organism, if cholesterol is in excess, its

conversion into bile acids is increased, allowing its elimination in the feces, which is the sole way of excreting cholesterol. This excretion is further regulated by the enterohepatic cycle of bile acids [i.e., secretion in the gut lumen, followed by reabsorption in the ileum and reentry to the liver (see Fig. 9)], which is in part aimed at saving complex and energy-costly molecules such as cholesterol and its bile acid derivatives. In contrast, if the cholesterol supply is low, de novo synthesis takes place mainly in the liver and the intestine.

The factors acting on the transcriptional regulation of cholesterol homeostasis can be presented schematically as follows. When there are sufficient amounts of cholesterol, membrane-bound SREBP-2 is not processed to its mature form, and cholesterol synthesis is not stimulated. In addition, if the cholesterol supply is high, LXR is activated by the high levels of oxysterols that are the intermediate compounds in the process of cholesterol degradation. In peripheral cells, LXR activity enhances cholesterol efflux, while in the liver it increases the formation of bile acids to avoid an accumulation of cholesterol in excess. In turn, FXR that senses high levels of bile acids inhibits their synthesis, thereby limiting the elevation of bile acid levels and their deleterious effects on liver cell functions. Conversely, low cholesterol levels are also sensed by the cells, which respond by increasing the processing and activity of SREBP-2, thereby stimulating cholesterol de novo synthesis.

---

### Cholesterol synthesis

#### Liver

- **SREBP-2** → Cholesterol
- SHP
- NTCP
- Cyp7A
- BSEP
- FXR ← Bile Acids (BA)
- Cyp7A (mouse)

#### Gut

- IBABP
- Bile acids reabsorbed
- Bile Acids + cholesterol
- excretion

#### LDL Receptor

- LXR
- CETP
- ApoE
- ABCG5
- ABCG8

#### Cholesterol (diet)

- Cholesterol reabsorbed

---

FIG. 9. The coordinated action of FXR, LXR, and SREBP-2 in the cholesterol metabolism pathway. The major source of cholesterol is the diet, while de novo synthesis of cholesterol is stimulated by SREBP-2 if supplies are too low. If cholesterol is in excess, its efflux from the cells and its conversion into bile acids for excretion in the feces are favored by the activation of liver X receptor (LXR). High bile acid production in turn activates farnesol X receptor (FXR), which limits the toxic accumulation of these metabolites in the liver, by increasing their cell efflux and limiting their production. The plain blue arrows correspond to the action of these transcription factors on the genes acting in the cholesterol metabolic pathway. The gray arrows correspond to the action of these gene products. Some bile acid and cholesterol metabolites are ligands for FXR and LXR, respectively (blue dotted line), while high cholesterol levels directly inhibit SREBP-2. For more details, see section IVB.

---

2. SREBP-2 is required for the transcriptional activation of the cholesterol synthesis pathway

SREBP-2 is present in most tissues at rather low levels but is more particularly expressed in the liver. As mentioned previously, high levels of cholesterol in the ER membrane inhibit the activity of the SREBP-cleavage activating protein (SCAP). This occurs through Insig-1 and Insig-2, which are ER proteins that bind SCAP in the presence of high levels of cholesterol. Insig interaction retains the SCAP/SREBP complex in the ER (331) (see Appendix G and Fig. 7). When cholesterol is low, this interaction is released, and the activated SCAP triggers the transport of the SCAP/SREBP complex from the ER to the Golgi membrane where proteases free the cytosolic domain of SREBP-2. This domain then translocates to the nucleus and affects the expression of key enzymes in cholesterol biosynthesis and uptake (19). In addition to this control, SREBP-2 positively regulates its own gene transcription via an SRE present in its promoter region (268). In contrast, upregulation of SREBP-2 in LXR null mice suggests that LXR has a negative action on SREBP-2 expression and, consequently, on the expression of its target genes (226). However, the mechanism of this down-regulation and its relevance in humans has not yet been addressed. The tight posttranscriptional control of SREBP-2 by cholesterol also raises the question of whether transcriptional regulation of SREBP-2 participates in the regulation of its activity.

In transgenic mice, overexpression of the active nuclear form of SREBP-2 dramatically increases the expression of the genes involved in cholesterol synthesis, particularly that encoding HMG-CoA reductase. In addition to cholesterol synthesis, SREBP-2 enhances the expression of the LDL receptor gene. A subsequent massive cholesterol overload of the liver is induced, with only a comparatively mild increase in fatty acid accumulation (reviewed in Ref. 114). Whereas little knowledge has been gained from SREBP-2 null mice which die in utero for nonelucidated reasons, a liver-specific mutation of SCAP almost completely blunts cholesterol synthesis (191).

In short, the synthesis of cholesterol by the liver and peripheral cells is strictly dependent on the presence of SREBP-2, whose activity is tightly regulated by the levels of membrane cholesterol sensed by SCAP.

### 3. LXR: a player in the reverse cholesterol pathway

LXR (see Appendix C) was initially characterized by its role in the positive regulation of the gene encoding cholesterol 7α-hydroxylase (CYP7A), the rate-limiting enzyme in the neutral bile acid biosynthetic pathway. The nature of LXR endogenous ligands, the cholesterol metabolite oxysterols, further emphasized the importance of this receptor in cholesterol metabolism (127, 168). This function has been confirmed by the phenotype of LXRα null mice, which appear healthy and fertile when fed on a standard mouse diet. However, when fed with a cholesterol-enriched diet, these mice display a severely impaired cholesterol and bile acid metabolism. Indeed, they fail to induce CYP7A and consequently suffer from a dramatic accumulation of cholesteryl ester in the liver with no increase in bile acid production (226). LXR in mice was thus credited for a major role in the disposal of excess cholesterol by increased elimination via bile acid synthesis (see Fig. 9). However, it is now clear that the human CYP7A is not responsive to LXR and might even be repressed by LXRα activation (34, 87). This difference between mouse and human is of interest as it might explain, at least in part, both the higher capability of mice to face a high-cholesterol diet and their increased resistance to developing atherosclerosis.

As a consequence, research now focuses more on other important LXR-mediated regulations that converge on the reverse cholesterol transport pathway. This pathway limits the exposure of peripheral cells to cholesterol excess, and its modulation by LXR has been reported in both humans and mice, at at least three levels (see Fig. 9). First, LXR upregulates the expression of several genes coding for members of the ABC transporter family. ABCG5 and ABCG8, which are expressed almost exclusively in the liver and small intestine, favor the secretion of sterols from the liver epithelial cells to the bile duct and from the gut epithelial cells to the intestinal lumen (14).

Activation of these two genes by LXR is considered to be the main mechanism by which an LXR agonist in mice causes a total blockade of cholesterol absorption (232, 242). Another important target is the widespread ABCA1 transporter, which promotes the efflux of intracellular and plasma membrane cholesterol to the nascent high-density lipoprotein (HDL) particles via interaction with ApoA1, thereby increasing HDL levels (245). Second, LXR increases ApoE expression. Effluxed cholesterol from cell membranes can also be charged on HDL particles by ApoE, increasing their total capacity for accepting cholesterol. In addition, ApoE-containing particles can interact with the scavenger receptor that increases uptake of these particles in the liver. Third, LXR increases the expression of the cholesteryl ester transfer protein (CETP), which promotes cholesteryl ester transfer from VLDL to HDL and from HDL to LDL, a lipoprotein which is also efficiently taken up by the liver. By these means, LXR increases cholesterol clearance from the blood (reviewed in Ref. 292). Some of these regulations are shared by LXRα and LXRβ. If LXRβ null mice do not have the dramatic phenotype described for LXRα, the double mutant mice are more strongly affected than the LXRα null mice (244). However, no functional compensation by LXRβ is seen in LXRα null mutant mice, and a specific role for LXRβ has not yet been clearly defined.

In short, LXR is the major transcription factor that acts as a sensor of cholesterol levels via its interaction with oxysterols and, in turn, drives the disposal of excess cholesterol. It also acts at the level of individual cells by increasing the ABC transporter molecules responsible for cholesterol efflux, and at the level of the organism by decreasing the cholesterol uptake from the diet. In mice livers, it also increases the conversion of cholesterol into bile acid.

### 4. FXR and the inhibition of bile acid synthesis

The farnesol X receptor (FXR) senses bile acids and responds by inhibiting bile acid synthesis (see Appendix D), as illustrated in FXR null mice. These mice, which have no overt phenotype except increased bile acid levels in the blood, cannot sustain a cholic acid-enriched diet. They suffer a severe wasting syndrome with hypothermia, and ~30% of them die by day 7 on such a diet (282), which is probably mainly due to the known toxicity provoked by accumulation of bile acids in hepatocytes (252).

As discussed above, CYP7A expression controls the neutral pathway of bile acid synthesis (Fig. 9). CYP7A, which is positively regulated by LXR in mice livers, is negatively regulated by FXR via an indirect mechanism. The constitutive expression of CYP7A in the liver depends on the nuclear factor LRH-1 (NR5A2), which binds to a response element located in the promoter of the CYP7A gene. Activated FXR does not bind to the promoter of this

gene but upregulates the expression of the short heterodimerization partner (SHP-1, NR0B2), which binds to LRH-1 to form an inactive heterodimer. This results in downregulation of CYP7A expression, even in the presence of activated LXR (86, 181). Increased expression of SHP-1 may also be responsible for the FXR-mediated decreased expression of the sodium taurocholate cotransporting polypeptide (NTCP). In hepatocytes, this transmembrane protein mediates the uptake of bile acids that are returning from the gut to the liver in the enterohepatic cycle. Finally, a direct action of FXR:RXR positively regulates the gene encoding the bile salt export pump (BSEP). This protein, only expressed in the liver, belongs to the ABC transporter superfamily and allows the extrusion of bile acids from hepatocytes into the biliary canalculus. These coordinated actions of FXR on CYP7A, NTCP, and BSEP result in lowering the potentially deleterious high levels of bile acids to which hepatocytes are exposed (reviewed in Ref. 77). In contrast, the FXR-mediated induction of the ileal bile acid binding protein (IBABP), an intracellular carrier of bile acids expressed in the ileal epithelial cells, favors the reuptake of bile acids from the gut lumen (91). One proposed hypothesis is that reabsorption of bile acids in the ileum would diminish the micelle formation and solubilization of cholesterol and thus limit its absorption. Because bile acids are more abundant in times of cholesterol excess, this would limit further cholesterol uptake by the intestine. This hypothesis might seem incomplete at first glance, as bile acid reabsorption mainly occurs in the ileum, whereas that of cholesterol mainly happens in the jejunum. However, ABCG5 and ABCG8, which are important regulators of cholesterol absorption, are also expressed in the ileum (59), and clinical studies in subjects with ileal pouch-anal anastomosis suggest that ileal dysfunction diminishes cholesterol absorption (153).

FXR might also act directly on circulating lipoproteins, by inducing the expression and secretion of hepatic apoCII (138), and by increasing the levels of the secreted enzyme phospholipid transfer protein (PLTP), which facilitates the transfer of cholesterol and phospholipids, respectively, from triglyceride-rich lipoproteins to HDL (303). However, the prominent effects of FXR on bile acid metabolism in part masks the specific effects of the increased apoCII and PLTP expression.

In short, FXR is the transcription factor that senses the intracellular levels of bile acids and is required for limiting liver bile acid accumulation. It inhibits bile acid synthesis via the downregulation of CYP7A and increases bile acid efflux in the bile via increased BSEP expression. In the ileal enterocytes, the reabsorbed bile is taken in charge by the cytosolic IBABP whose expression is also increased by FXR.

In conclusion, because cholesterol synthesis is quite energy-demanding, an efficient transport system for picking up cholesterol from the diet is probably advantageous for any organism. This efficiency is reflected in the increased cellular cholesterol levels that correlate with a cholesterol-rich diet. Once again, the mammalian organism is quite well-adapted for sparing the important and high-cost cholesterol molecules, but is ill-prepared for coping with an excess of cholesterol intake, which adds to cardiovascular risks.

### C. Intricate Regulation of Cholesterol and Fatty Acid Metabolism

Recent progress in understanding the regulatory pathways of fatty acid metabolism and cholesterol homeostasis are progressively unraveling more and more intricate regulatory events, shrinking the gap between these two main branches of lipid metabolism. In this section, we present these intersecting pathways, although some have already been mentioned in the appropriate sections above.

This meshing can be seen from two points of view. First, lipoprotein metabolism itself participates in both cholesterol and fatty acid metabolism and controls the circulation and tissue delivery of these molecules in a very fine-tuned and orchestrated manner. Second, transcription factors might themselves be knitting points in the intricate pattern of regulation. This is the case for the two SREBP isoforms, but also for LXR, which was initially credited with a major role in cholesterol homeostasis but is also now integrated in many aspects of fatty acid metabolism. RXR, by being a promiscuous partner for each of the nuclear receptors that participate in the control of lipid metabolism, could also act as a converging regulatory node. However, this is only partially true as discussed below.

#### 1. Regulation of the lipoprotein system and the particular role of PPARs

As discussed above, activation and repression of various components of the reverse cholesterol pathways are regulated by LXR and FXR, thereby directly affecting the circulating lipoproteins. We will discuss below how PPARs, which mainly target fatty acid metabolism, also affect cholesterol metabolism via their action on lipoproteins.

The blood lipoprotein profile is first modulated by PPARs via PPAR-mediated increased expression of LPL (reviewed in Ref. 15). Both PPARα and PPARγ can upregulate *LPL* expression. This is associated with a decreased expression of apoCIII, an inhibitor of LPL activity. These joint effects increase the free fatty acid delivery to tissues such as muscle and adipose tissue. PPARα also participates in an increase in the reverse cholesterol transport by upregulating the expression of the genes encoding the cholesterol acceptor apolipoprotein apoAI
and apoAII. These two proteins participate in the formation of HDL particles, which carry the cholesterol transferred from chylomicron and VLDL remnants to the liver. Finally, PPARα increases the hepatic expression of the scavenger receptor BI (SR-BI)/CLA-1, thereby increasing the selective uptake of HDL cholesteryl esters from the blood (reviewed in Ref. 54). These actions of PPARα on the lipoprotein profile, associated with its stimulation of fatty acid oxidation, counteract the highly atherogenic situation characterized by high levels of triglycerides and low levels of HDL in the blood, often associated with clinical dyslipidemia. Indeed, these observations unveil the molecular mechanisms by which fibrates have hypolipidemic properties that are very useful for the treatment of human lipid disorders.

The role of PPARβ in regulating the lipoprotein transport system has been best demonstrated in obese rhesus monkeys, used as a relevant animal model for human obesity and the associated metabolic disorders. Treatment of these animals with a selective PPARβ agonist caused an increase in the level of serum HDL cholesterol, while lowering the level of small-dense LDL, fasting triglycerides, and fasting insulin (216). In mice, the administration of a PPARβ-selective synthetic agonist to obese and diabetic *db/db* mice raised total plasma cholesterol levels primarily associated with HDL particles and decreased the expression of the lipoprotein lipase in WAT (169). However, under nonchallenging conditions, PPARβ null mice have normal plasma lipid composition with no changes in the levels of total cholesterol, HDL-cholesterol, triglycerides, and free fatty acids compared with wild-type littermates. Therefore, the contribution of PPARβ to lipoprotein metabolism in healthy mice remains to be investigated, especially in the context of different diets and levels of physical exercise (reviewed in Ref. 13).

### SREBPs and LXR at the branching point between fatty acid and cholesterol metabolism

The combined role of SREBP-1c and SREBP-2 in fatty acid and in cholesterol metabolisms can be effectively highlighted using a scheme representing the two pathways emerging from the single initial precursor acetyl-CoA (63, 114) (see Fig. 8). Acetyl-CoA is either condensed in acetoacetyl CoA via acetoacetyl CoA thiolase for cholesterol synthesis, or is transformed to malonyl-CoA by acetyl-CoA carboxylase for fatty acid synthesis. Alternatively, acetyl-CoA is oxidized in the citric acid cycle for ATP production. The relative importance of each of these three pathways is determined by the levels of activity of their respective enzymes. These are, at least in part, dependent on the transcriptional expression of the corresponding genes controlled by SREBP-1 and SREBP-2. In addition, both cholesterol and fatty acid syn-

thesis require NADPH. NADPH is produced either by the malic enzyme (ME), which transforms cytosolic malate into pyruvate, or via the pentose phosphate pathway, which produces ribulose and xylulose from G6P by activation of the glucose-6-phosphate dehydrogenase (G6PDH) and 6-phosphogluconate dehydrogenase (PGDH). Interestingly, the genes coding for these three enzymes (ME, G6PDH, PGDH) are targets of both SREBP-2 and SREBP-1c (281).

Links between fatty acid and cholesterol metabolisms also exist via the mechanisms regulating VLDL assembly. SREBP-1 and SREBP-2 are both capable of inducing the expression of the liver microsomal triglyceride transfer protein (MTP), which is essential in the process of VLDL assembly. VLDL particles are composed of the apoB protein, phospholipids and free cholesterol at the surface, and triglycerides and cholesterol esters in the core. The more that cholesterol is synthesized or present in the liver, the more fatty acids are associated with the forming particles. In addition, cholesterol esterification also requires fatty acids, and more particularly oleate. The unusually high level of oleate in mice overexpressing SREBP-1c might reflect a role for SREBP-1c in promoting the supply of the appropriate fatty acid. SREBP-1 and SREBP-2 also increase the expression of the LDL receptor in hepatic cells (115). Thus, by increasing the turnover of the VLDL particles, SREBPs contribute towards enhancing the supply of fatty acids and cholesterol to peripheral cells and limiting hyperlipoproteinemia.

The elegant design of a microarray experiment has given a recapitulation of most of these observations and casts light on new features with respect to SREBP target genes (115). The set of upregulated genes in transgenic mice overexpressing either SREBP-1a or SREBP-2 were compared with that of downregulated genes in mice specifically deleted of SCAP in the liver. The small subset of genes found in both is likely composed of genes directly regulated by SREBP activation. All others, in contrast, likely respond to secondary events. Two major LXR target genes involved in cholesterol metabolism, ABCG5 and ABCG8, but not LXR itself, are found in the latter class. Indeed, SREBP activation leads to the production of endogenous LXR ligand and increases its activity. Reciprocally, activated LXR also acts on fatty acid synthesis via stimulation of SREBP-1c. In addition, experiments performed in HepG2 cells demonstrate that LXR plays a direct role in enhancing the expression of FAS (132). Together these observations highlight the intertwined regulation of cholesterol and fatty acid metabolisms and might explain the steatosis and the massive increase in VLDL and triglyceride blood levels observed in mice treated with pharmacological doses of an LXR ligand (90). In contrast to the results discussed above and with the general pattern of LXR activities, a study performed in 3T3-L1 aimed at understanding the role of LXRα in the adipocytes reports an increased release of nonesterified
fatty acids. In the same report, treatment of mice with an LXRα agonist increased blood levels in glycerol and non-esterified fatty acids, which suggests an LXR-mediated increased lipolysis in the WAT (259). These discrepancies remain to be elucidated.

3. **RXR: a pivotal element of sensor-regulated pathways**

    RXR occupies a particular position, as it participates in most of the fatty acid and cholesterol metabolic regulations via its interaction with several of the nuclear receptors, a simple means of creating regulatory cross-talk. The role of RXR in metabolic regulation must be analyzed from two different angles.
    
    RXR is an obligatory partner for many nuclear receptors (186). Thus its deletion putatively affects a very diverse array of developmental and physiological pathways. This has been clearly underlined by the thorough phenotypic analyses of the developmental defects occurring in mice carrying various RXR gene deletions or mutations. Whereas RXRβ and RXRγ null mutations give rise to quite minor developmental defects, RXRα null mice die at early embryonic stages (188). This lethality might be due to defects in the PPARβ and PPARγ signaling pathways. Indeed, invalidation of either PPARβ or PPARγ provokes an embryonic lethality due to placental defects, appearing at the same early point as in RXRα null placentas. Other defects analyzed in the various RXR mutants and seen in tissues such as the skin, the eye, the heart, and the testis, reflect alterations in the pathways of many other receptors, such as the thyroid hormone, vitamin D, and retinoic acid receptors (188, 316). From the metabolic point of view, analyzing the importance of RXRα in adult tissues required the generation of tissue-specific knockouts. Specific invalidation of RXRα has been generated in the liver, and metabolic studies were performed to identify which pathways were most affected. As expected, many PPARα-mediated functions in fatty acid oxidation were altered by the lack of RXRα. However, other pathways that include LXR and FXR pathways were also compromised, at least partially, by the absence of RXRα. These effects could not be compensated for by RXRβ and RXRγ (309). RXRα was also invalidated in the adipose tissue. The deletion of the two RXRα alleles in the adipocytes of adult animals resulted in an alteration of preadipocyte differentiation as well as in a resistance to induced obesity. These results are reminiscent of the observations obtained with PPARγ heterozygous mice, suggesting that most of the effects due to the lack of RXRα expression in adipocytes reflect altered PPARγ functions (120). Finally, as mentioned above, the impaired lipolysis observed in these mice might be related to an alteration to LXR:RXR heterodimer signaling.

    RXR might also be important not only as a partner for other receptors but also as a bona fide receptor. As already stated, the nature of the endogenous ligand of RXR remains elusive, and all observations made so far were in a pharmacological context, treating mice with 9-cis-retinoic acid (RA) or synthetic RXR agonists. Upon such treatments, RXR may regulate transcription as homodimers (RXR:RXR), binding to DR-1-like response elements. While the formation of these homodimers is difficult to assert in vivo and has been neglected, new approaches by chromatin immunoprecipitation demonstrated that RXR homodimers do form in vivo and efficiently trigger the activation of PPRE-containing genes, such as ME in the liver (119). In contrast, there is major interest in exploiting the ability of RXR ligands to activate the so-called permissive heterodimers. These RXR-containing heterodimers allow RXR to bind its ligand and subsequently turn on the transcriptional activation of the gene to which the heterodimer binds. Thus, in many experimental reports, 9-cis-RA was used in combination with a PPAR ligand to amplify the PPAR biological response. In in vivo studies, potentiation of PPARα action by RXR ligands is mainly observed in the liver (220). In another example, in vivo administration of synthetic RXR ligands mimics, and increases when given in combination with TZD, the metabolic effects of PPARγ ligands, by decreasing hyperglycemia, and improving insulin sensitivity (205). However, these benefits seem to be accompanied in humans and in some animal models by a severe hypertriglyceridemia (43, 248). Exploring the LXR and FXR pathways under such treatment also highlights a coordinated regulation of both pathways leading to a very efficient inhibition of cholesterol absorption (245). These observations underline the problem that faces the researcher who works with such a promiscuous agent. On the one hand, it is difficult to predict the scope of changes that a specific RXR ligand may provoke in the whole organism with respect to metabolic homeostasis. On the other hand, it also gives hope for the identification of RXR ligands specific for a given heterodimer, which would broaden the spectrum of therapies available for targeting metabolic diseases.

V. TRANSCRIPTION FACTOR INTERPLAY IN THE FASTING-FEEDING CYCLE

Metabolic homeostasis requires at least two levels of integration. The first level is the overall metabolic homeostasis, at the time of growth during development, and in adulthood when the focus is on adapting food intake to the need of the organism. The many hormones and hormone-like substances that signal to the brain are the major agents in appetite control. The best indication of successful homeostasis in adulthood is the maintenance of body weight. The second level is the daily call on

adaptive mechanisms to cope with the alternation between feeding and fasting, with a main objective of keeping glucose levels high enough for brain cells and other glucose-dependent tissues while maintaining an energy supply sufficient for all organs. These regulations mainly focus on the production, distribution, use, and saving of energy-rich molecules, and not on constituting optimal energy stores. Here, transcription factors serve two main purposes: regulating specific metabolic adjustments within each peripheral organ and coordinating intertissue regulation for robust homeostasis. This section is aimed at presenting the role of transcription factors involved in the functional pathways detailed in the previous sections, in the context of this daily challenge constituted by the alternating status of fasting and feeding.

Fasting represents a situation that is reflected in the blood by decreased levels of glucose, and accordingly decreased levels of insulin and increased levels of glucagon. To maintain energy supply during fasting, the organism must 1) have efficient lipolysis in the WAT for the systemic release of fatty acids and glycerol, 2) promote the use of fat over glucose in all tissues where possible, and 3) provide glucose in sufficient amounts for glucose-dependent tissues such as the brain and erythrocytes. Upon refeeding, a large glucose load from nutrients taken up by the intestine triggers the hormonal response with high insulin and low glucagon, which coordinates metabolic adjustments in peripheral tissues. This situation is often used to analyze insulin-mediated metabolic action. During this period, energy stores are replenished, in the liver (glycogen and triglycerides), in muscle (glycogen), and in WAT (triglycerides).

A. Fasting-Feeding: Metabolic Adjustment in the WAT

In the WAT, free fatty acid release (i.e., lipolysis), in response to fasting, is a critical step aimed at maintaining whole body energy homeostasis in the absence of external energy supply.

Catecholamines are the most studied signals that trigger lipolysis, but other hormonal signals in the adipose tissue include thyroid hormone, leptin, and growth hormone (reviewed in Ref. 79). HSL is a key enzyme for the mobilization of triglycerides deposited in adipose tissue, and lipolysis is triggered by cAMP/PKA-dependent phosphorylation of HSL, as provoked by adrenergic stimuli (reviewed in Ref. 109). Interestingly, FOXC2, which increases the sensitivity to cAMP/PKA-dependent signals, induces an upregulation of *HSL* mRNA levels in WAT. Free fatty acids can then be funneled to the mitochondria whose biogenesis is also increased by FOXC2 (29). In this context, the associated increased expression of *Ucp1* redirects the energy in thermogenesis (99). It now remains to be determined whether FOXC2 is involved in the non-

adrenergic events controlling lipolysis especially in response to fasting or starvation.

A role for nuclear receptors is suggested from the results of the adipose tissue-specific deletion of *RXRα* in adult mice, which results in impaired lipolysis. This translates into an absence of free fatty acid increase in the blood upon starvation, a lack of adipose tissue reduction, and profound hypothermia, all effects which cause premature death if starvation is prolonged (120). RXR is thought to act in partnership with many other nuclear receptors, and the responsible heterodimer complex(es) that fail to form in the adipose tissue of these mutant mice is not yet known. It is unlikely to be PPARγ, since PPARγ1 and PPARγ2 are strongly downregulated during fasting (306). Based on microarray analyses, LXR has been proposed to be one of these potential RXR partners (259). However, this is difficult to reconcile with the strong LXR-mediated lipogenesis found in both liver and WAT of mice treated with a LXR agonist. Whereas the expression of LXR in the WAT of fasted mice has not been reported, the downregulation of both *Glut4* and *SREBP-1c*, which are considered as LXR target genes (157), suggest that LXR activity itself is also downregulated upon fasting. The question of the RXR partner involved in fasting-induced lipolysis is still open. It might be worth considering PPARβ as a candidate, since its overexpression in the adipose tissue results in an increased fatty acid consumption (313), although *PPARβ* null mice have a reduced adiposity (8). Leptin also drives lipolysis leading to increased plasma levels of glycerol that are not accompanied by increased free fatty acids. The fact that leptin-mediated lipolysis is inefficient in *PPARα* null mice (164) suggests that in wild-type mice the increased expression of *PPARα* caused by leptin triggers an immediate oxidation of the released fatty acids, and the subsequent release of glycerol in the blood.

Refeeding is first characterized by the immediate action of insulin on Glut4 translocation and the coordinated control of enzymatic activities along the glycolytic and fatty acid synthesis pathways. At the transcriptional level, insulin action translates into higher expression of PPARγ in the WAT, with reciprocal positive interaction of SREBP-1c and PPARγ on their respective gene expression, both leading to increased fatty acid synthesis and storage as triglycerides. Fat accumulation is also permitted by the insulin-mediated upregulation of the expression of the transporter Glut4. Although the corresponding transcriptional mechanism involved is not fully elucidated (reviewed in Ref. 31), the insulin-stimulated expression of *Glut4* in mouse epididymal WAT is dependent on *LXRα* expression, consistent with the induced *Glut4* expression by LXR agonists (46). In addition, insulin is strongly antilipolytic via at least two mechanisms. First, insulin lowers cAMP levels through activation of phosphodiesterase

TRANSCRIPTIONAL REGULATION OF METABOLISM

3B. Second, insulin also provokes the immediate reesterification of the triglycerides which constitutively undergo hydrolysis in WAT cells. These few understood general features mask our ignorance of many facts concerning the transcriptional regulation of the metabolic adjustment to refeeding. The time courses of the patterns of expression in the WAT of the main transcription factors, such as C/EBPα, SREBP, and PPARγ, remain largely unexplored. In contrast, there is a lot of interest in the identification of adipokines, which are molecules released by the WAT, depending on its level of replenishment. These cytokines are thought to play an important role as paracrine and/or endocrine signaling molecules (78, 79). Leptin, which has major effects in reducing food consumption, is the most studied. Other adipokines include adiponectin, resistin, and fasting-induced adipose factor (FIAF), which reach the systemic circulation, but also other cytokines such as tumor necrosis factor-α, interleukin-6, and plasminogen activator inhibitor-1 which may rather act as paracrine factors. While their roles and molecular mechanisms of their actions are beyond the scope of this review, there are high expectations concerning our understanding of energy homeostasis from analyses of the functions of these signaling molecules.

### B. Fasting-Feeding: Metabolic Adjustment in Muscles

In muscles, adaptation to fasting is aimed at sparing glucose, first by switching from glucose to fatty acid oxidation. In that respect, analysis of the regulation of the pyruvate dehydrogenase complex (PDC) during starvation is of principal importance. Pyruvate dehydrogenase activity irreversibly converts pyruvate to acetyl-CoA in the mitochondrion, leading to a complete glucose oxidation into CO₂ and H₂O, concomitant with ATP production (see Fig. 10A). This reaction links glycolysis to the citric cycle in organs of high energy demand and provides the carbon for fatty acid and ketone body synthesis in adipose tissue and in the liver. In the opposite situation, i.e., during starvation, the inhibition of PDC in tissues that can meet their energy needs independently of the complete oxidation of glucose is essential to spare pyruvate that can be channeled to the liver. This inhibition is particularly efficient in skeletal muscle via activation of pyruvate dehydrogenase kinase 4 (PDK4), which phosphorylates and thus inactivates PDC. This results in the conversion of pyruvate into lactate, which is used in the liver for gluconeogenesis. PPARα and glucocorticoids increase the expression of PDK4, in both muscles and the liver (102, 117). Thus the activation of PPARα during stress or starvation spares pyruvate for gluconeogenesis via increased levels of PDK4, while it concomitantly stimulates the use of fatty acids as an energy source via increased β-oxidation. Simultaneously, lipolysis of the triglyceride store is also activated in the muscle, but remains a minor source of free fatty acids compared with the supply coming from the adipose tissue and transported to the liver as albumin-bound molecules. However, as mentioned above, the fatty

---

**FIG. 10. Metabolic adjustment of glucose metabolism upon fasting. A: in the liver, pyruvate carboxylase directs pyruvate toward gluconeogenesis, producing glucose for other demanding tissues. In contrast, the pyruvate dehydrogenase complex favors the complete oxidation of pyruvate to provide energy to the cells. The pyruvate dehydrogenase kinase 4 (PDK4) can repress this pathway by inhibiting the activity of the pyruvate dehydrogenase complex. PDK4 is under the positive transcriptional control of PPARα. During fasting in muscle, the activation of PDK4 spares the pyruvate for its redirection to the liver, depicted in B. B: upon starvation, the glucose-alanine cycle in the muscle redirects the pyruvate formed by glycolysis to the liver in the form of the amino acid alanine that will be used as a gluconeogenic substrate for hepatic glucose production.**

**Liver cell**
- **Glycerol**
- **Alanine**
- **Glucose**
  - → **Pyruvate**
    - **Pyruvate Carboxylase**
      - → **Oxaloacetate** → **Gluconeogenesis** → **Glucose** (for export)
    - **Pyruvate dehydrogenase complex**
      - → **Acetyl CoA** → **Citric acid cycle** → **ENERGY**
  - **PPARα** → **PDK4**

**Liver cell**
- **Glucose**
  - → **Pyruvate**
    - **Alanine**
      - → **Amino acids**
        - **α-ketoglutarate**
          - **Glutamate**
            - **α-cetonic acid**
              - **Muscle cell**
                - **Pyruvate**
                  - **Glycolysis**


**Physiol Rev • VOL 86 • APRIL 2006 • www.prv.org**
acid oxidative capacity of muscle is reduced by only 30% upon starvation in PPARα null mice, suggesting that PPARβ may compensate for the lack of PPARα in these mice (206). This hypothesis is further supported by the demonstration that PPARβ is a transcriptional regulator of fatty acid oxidation in muscle (see above), although the role of PPARβ during fasting in wild-type mice remains to be specifically explored.

Amino acid metabolism in the muscle is also affected by starvation. It must be kept in mind that, while fatty acid oxidation participates in glucose formation by providing the required energy, fatty acids cannot form glucose. In contrast, glycerol from triglyceride lipolysis normally contributes up to ~10% of hepatic gluconeogenesis. Thus, once glycogen is depleted, amino acids, which mainly come from contractile proteins of skeletal muscles, remain the major source of substrate to be converted into glucose. In the glucose-alanine cycle (see Fig. 10B), oxidation of the branched chain amino acids is the privileged source of amino groups that are transferred to α-ketoglutarate to form glutamate. Glutamate in turn either donates its amino group to pyruvate forming alanine for the hepatic gluconeogenesis pathway, or is further aminated into glutamine that is used as an energy source by the intestine. In muscles, the transcriptional regulation of contractile protein degradation and amino acid metabolic adaptations are not yet clarified.

Refeeding completely reverses PDK4 induction in both heart and skeletal muscle. The insulin-mediated increase of Glut4 expression and membrane translocation facilitates glucose uptake and glycogen synthesis. Depending on the nature of the fiber, glycolytic or oxidative, preferential consumption of glucose or fatty acids can resume.

### C. Fasting-Feeding and Gluconeogenesis: Metabolic Adjustment in the Liver, Kidney, and Small Intestine

While hepatic gluconeogenesis is essential upon fasting, the kidney and small intestine also contribute to glucose output, more particularly upon prolonged fasting (64, 200). In these tissues, the main gluconeogenic substrate is glutamine which undergoes a partial decarboxylation, releasing a three-carbon compound that enters the mitochondrial tricarboxylic acid cycle. In these tissues the expression of G6Pase, which hydrolyzes G6P in glucose and Pi, is essential for glucose secretion. G6Pase expression is highly dependent on an insulin-deprived status and is markedly induced during fasting and diabetes.

However, in the normal postabsorptive state, the liver is the site of high glucose production, via glycogenolysis and gluconeogenesis. As we have seen, this requires the use of substrates such as glycerol, which comes from lipolysis, and gluconeogenic amino acids among which are alanine coming from the muscles, and lactate provided by muscle glycolysis. Glucocorticoids and glucagon are increased upon fasting, and both hormones increase C/EBPα as well as C/EBPβ (reviewed in Ref. 253), which participate in the enzymatic induction responsible for this increased hepatic glucose production. A main conveyor of this adaptive response is CREB, whose PKA-dependent phosphorylation, triggered by glucagon, turns on its activity as a key regulator of gluconeogenic gene transcription. Gluconeogenesis is energy-demanding and requires that fatty acid oxidation provides the necessary amounts of acetyl-CoA, NADH, and ATP as an energy source and a proper directional flux of the enzymatic reactions from pyruvate to glucose. In this respect, PPARα plays a major role in allowing an adaptive response to fasting. Its stimulation by glucocorticoids and sustained increased activity upon fasting correlates with the enzymatic induction of the fatty acid oxidation pathway and with the concomitant production of ketone bodies. At the same time, the lipogenic transcription factor gene SREBP-1c is repressed (172), and its activity further inhibited by increased Insig levels, which retain unprocessed SREBP in the ER membrane. In fact, Insig-2 is expressed via two mRNA species Insig-2a and Insig-2b, the former being upregulated upon fasting (328). With respect to amino acid metabolism, even though there is an increased degradation of muscle proteins, the consequence of fasting is an overall decrease in urea formation, reflecting the lack of dietary proteins that degrade. It is thus interesting to note that both C/EBPα, which increases urea cycle enzyme activities, and PPARα, which is an inhibitor of the urea cycle, are induced upon fasting. These concomitant activities are likely to permit a fine tuning of urea production, according to the needs rather than unilaterally imposing a directional flux in one direction or the other.

The phenotype of PPARα null mice is extremely revealing regarding the intricate links between the regulatory pathways of fatty acid, glucose, and amino acid metabolisms during fasting. Unchallenged PPARα null mutant mice do not present any overt phenotype. However, they cannot sustain fasting due to a severe hypoglycemia and lack of ketone body production, while levels of urea production remain high (141). The severe hypoglycemia of fasted PPARα null mice has been initially attributed to two facts: the low levels of the hepatic glycogen store and an impaired gluconeogenesis, probably due in part to the inability of fatty acids to provide the necessary energy. If we take the example of the PGC1 interaction with HNF4 and the glucocorticoid receptor for induction of the gluconeogenic PEPCK gene, it might be possible that PGC1, which also efficiently interacts with PPARα, also serves PPARα target genes that would belong to the gluconeogenic pathway. However, the hypothesis of an impaired gluconeogenesis in PPARα null mice is challenged by the
observation that PPARα null mice have an increased hepatic glucose production (327). This production might rely on glycerol coming from the adipose tissue as a result of starvation-induced lipolysis of triglycerides. However, the upregulation by fasting of several genes involved in the hepatic metabolism of glycerol (glycerol transporters aquaporin-3 and -9, glycerol kinase, and cytosolic and mitochondrial glycerol-3-phosphate dehydrogenase) is abolished in PPARα null mice (225). Alternatively, it is possible that amino acids are increasingly used for gluconeogenesis in PPARα null mice, which would be consistent with the maintenance of high urea levels in the blood of fasted mice (141). The association between a higher production of glucose and severe hypoglycemia in PPARα null animals indicates that a high disposal of glucose is maintained in the peripheral tissue, due to the lack of an alternative energy source such as ketone bodies and to the impaired ability to directly use fatty acids. As glucose sparing, via PDK4 activation, may not work in fasted PPARα null mice, the muscle could be a glucose sink, participating in the death-frightening hypoglycemia. Such inhibition of the fasting-induced PDK4 expression has indeed been observed in the liver of PPARα null mice (290). The analyses of stable isotopomer flux demonstrated a diminished response of these mice to meal-induced regulation, resulting in a decreased substrate/futile cycling of glucose, despite increased gluconeogenesis (325). However, other mechanisms might be involved. A recent study demonstrates a lack of Glut4 downregulation in the adipose tissue of fasted PPARα null mice. This is associated with an increased transport of glucose in the WAT, which participates in the synthesis of new fat (R. Burcelin and B. Thorens, personal communication).

Refeeding is marked by a delay in the metabolic adjustment of the liver, which is likely due to a lingering action of the gluconeogenic and lipolytic signals. This characterizes an early fed state during which fatty acid β-oxidation remains high and the liver continues in its gluconeogenic mode using lactate and dietary amino acids as gluconeogenic precursors. After this initial stage, the major transcriptional coordination comes on one hand from the downregulation of PPARα and on the other from the upregulation of SREBP-1c, which mediates many of the insulin-dependent actions on lipid but also glucose metabolism (88). Intriguingly, while Insig-2a is strongly repressed by insulin, Insig-1 is itself a target gene of SREBP-1c, thus increasing in parallel during refeeding (331). This stimulation results in controlling SREBP processing and limiting the insulin-mediated high lipogenic activity at the time of refeeding (4, 65). Finally, the feeding-induced flooding of the liver with lipoprotein particles triggers the main pathways of lipid and cholesterol redistribution, as described previously.

D. Metabolic Adjustment and Circadian Rhythm

Circadian rhythms correspond to rhythms established over the day, observed at the level of gene expression, physiology, and behavior. These rhythms are essential for our daily well-being, allowing for anticipatory regulations, concerning for example heartbeat, blood pressure, hormone secretion, to prepare the upcoming activity or rest. The suprachiasmatic nucleus (SCN) is the main controller of these rhythms and is itself regularly reset by the mean of light intensity sensors, located in the retina (246). However, each tissue, if not each cell, contains molecular oscillators, as demonstrated by the brief exposure of fibroblasts to high concentrations of serum that provokes circadian gene expression which persisted for several days (7, 269). The nature of the mammalian circadian clock proteins that compose the molecular oscillator system is now being unraveled and will not be presented here (for a recent review, see Ref. 246). However, one major question is how all these countless oscillators can become synchronized in a complex organism. Indeed, reciprocal interactions between feeding, metabolism, and circadian rhythm correspond to an important part of this systemic cycling.

The metabolism of the liver and kidneys is in part under the control of circadian regulation. In these tissues, genes whose expression follows a circadian rhythm mainly encode enzymes or regulatory proteins involved in food processing or energy homeostasis, such as the cholesterol 7α-hydroxylase, PEPCK, glycogen synthase, and PPARα. As shown in the liver, many clusters of circadian-regulated genes are rate-limiting steps in their respective pathways (222). In the intact organism, the SCN seems to be the main synchronizer of the circadian rhythm seen in these peripheral tissues. However, the role of the daily feeding/fasting alternation in setting or altering the circadian rhythm, and the nature of the cues that reset the SCN according to environment sensors other than light sensors raise interesting questions. In addition, the SCN might use cues to override peripheral clocks, to keep control over genes linked to circadian rhythm, thereby allowing the organism to proceed via anticipatory rather than reactive metabolic adjustments. The role of circadian rhythm as imposed by the SCN in energy metabolism is demonstrated by the obesity and metabolic syndrome developed by mice mutant for the transcription factor Clock, a key component of the molecular clock in the SCN (300).

In this respect, recent observations have enlightened our understanding of these regulations. One of these stems from an observation made in nocturnal rodents, such as mice. Restricting their feeding to a short and unusual period of the day (daytime instead of night) results in a progressive phase shift of the circadian rhythm of liver gene expression. Interestingly, this resetting does not alter the activity of the SCN, which continues in its own phase (47, 288). The observed phase shifting is rather

slow, requiring several days for adjustment. In contrast, the reverse shift, i.e., by rescheduling feeding at night, results in a quick resetting. What is the signal that leads to anticipation of food intake with changes in gut motility or in liver gene expression, such as increased expression of Cyp7α hydroxylase and bile acid synthesis? One hypothesis is that the small circadian oscillations in body temperature might be one of these signals (20). Similarly, the redox status that can be sensed by some clock proteins reflects energy metabolism and might participate in the rhythm imposed by fasting-feeding (262, 270). Other blood-borne signals were looked at in light of experiments showing that numerous signaling factors are able to reset circadian rhythm in the ex vivo system represented by cells in culture. Particularly, glucocorticoids for which we have discussed the impact on many aspects of metabolism were good candidates. Their levels are oscillating during the day, influenced by the circadian rhythm of adrenocorticotropic hormone (ACTH) that drives glucocorticoid release. Interestingly, glucocorticoid administration is indeed able to induce a phase shift of the general circadian rhythm, demonstrating that glucocorticoids might participate in the physiological control of the circadian rhythm synchronized by the SCN. However, they are not responsible for, and rather inhibit, the liver phase shift provoked by the challenge of restricted feeding time, since mice without glucocorticoid receptors shift their liver gene expression rhythm faster than wild-type mice.

This short section leads to two main conclusions with respect to metabolism and its regulation. The first is that any in vivo experimentation must take into account feeding behavior as well as light/dark alternation. The second is that while we are used to thinking in terms of metabolic reaction and adjustment, the anticipation conditioned by the circadian rhythm and the ability of the latter to readjust to feeding time introduces a level of complexity that we are not yet able to fully grasp.

and low levels of HDL in the blood. Glycemia levels in the postprandial period or upon fasting are normal at this first stage, reflecting the ability of β-cell insulin secretion to manage high glucose and compensate for nascent insulin resistance. Impaired glucose tolerance, characterized by an increased postprandial glycemia associated with increased insulinemia, signals the second stage with a progression of the disease and an increased risk of macro- and microangiopathy complications (92). The third stage is that of an established type 2 diabetes mellitus (T2DM), which requires pharmacological treatment aimed at controlling the high levels of glycemia, insulin resistance, dyslipidemia, and the vascular complications.

The metabolic syndrome, or syndrome X, corresponds to an aggregation of metabolic disorders clustered around insulin resistance, which does not necessarily evolve towards T2DM (see Appendix I; Ref. 6). The overall defect adds to the insulin resistance and the impaired glucose regulation, a combination of components that comprise altered blood lipid and lipoprotein profiles, hypertension, central obesity, and microalbuminuria. These perturbations lead to pathologies, mainly in the cardiovascular area, such as atherosclerosis, which have an extremely high prevalence in westernized societies where they represent the major cause of death.

For both pathologies, T2DM and the metabolic syndrome, insulin resistance is the core initial defect and is the focus of the following discussion.

B. Insulin Resistance: A Mixed Defect in Glucose and Lipid Metabolism

Glucose homeostasis relies on the concerted production of insulin by the pancreatic β-cells, the uptake of glucose by muscle and liver, and the production of glucose by the liver. Insulin resistance is the inability of the peripheral tissues to respond properly to insulin. Fat cells are the most sensitive to insulin levels, responding by inhibition of lipolysis and stimulation of glucose uptake, followed by liver cells in which hepatic glucose production is inhibited (reviewed in Ref. 194). However, quantitatively speaking, the major intracellular metabolic defect in insulin resistance is an impaired muscle glucose uptake and glycogen synthesis (reviewed in Ref. 229). In insulin resistance, hepatic glucose production and lipolysis from fat cells are unduly maintained, while glucose disposal by muscle is less efficient, provoking an increased secretion of insulin by the pancreatic cells to maintain euglycemia (reviewed in Ref. 264). In parallel, chronic high levels of fatty acids in the β-cell and in blood impair insulin secretion by the pancreatic β-cells and participate in pathogenesis. Obesity, which is responsible for high circulating fatty acid levels, is one major cause of insulin resistance. Accordingly, weight reduction and physical activity of

VI. TRANSCRIPTION FACTORS AND INSULIN RESISTANCE: A MAJOR FOCUS ON PEROXISOME PROLIFERATOR ACTIVATED RECEPTORS

A. Introduction

Our westernized societies are witnessing a frightening increase in metabolic disorders, linked in good part to inappropriate nutritional and life-style habits and favored by genetic predisposition. The current definitions of various terms used for characterizing these metabolic disorders are presented in Appendix I. In many cases, but not all, the pathophysiology progression has the following pattern. Obesity and little physical activity are often the initial triggers of the disorders, characterized by high levels of triglycerides

patients clearly improve insulin sensitivity. However, the nature and the mechanism of the initial perturbation that provokes insulin resistance remain unclear.

Although it remains difficult to identify a single phenomenon at the origin of insulin resistance, the possible intricate causes and diverse consequences of insulin resistance are discussed below. Either as a cause and/or consequence, elevated fatty acid levels appear to be a crucial parameter in the development of insulin resistance. Obesity, even in the absence of other metabolic abnormalities, is accompanied by increased free fatty acid release in the blood, in proportion to the fat mass. These elevated levels of circulating free fatty acids divert triglyceride clearance from the adipose tissue towards nonadipose tissues. Visceral fat (i.e., in the splanchnic area) in particular becomes less sensitive to the antilipolytic action and fatty acid reesterification effects of insulin (308). Once insulin resistance is established in obese patients, lipolysis becomes even more important, leading to an even greater flux of free fatty acids towards the liver. In the liver, the increased efflux of free fatty acids augments VLDL production, reinforcing the pool of blood triglycerides. Furthermore, the liver maintains a high glucose production, aggravating hyperglycemia. In muscle, increased free fatty acids also impair insulin-stimulated glucose transport. The mechanism proposed is that free fatty acids negatively interfere with the insulin signaling cascade, at levels which are not clearly defined but may encompass the phosphorylation of IRS-1, the level of PI3K activity, leading to that of glycogen synthase activity, and finally *Glut 4* gene expression and/or protein translocation (reviewed in Ref. 264).

The diversion of free fatty acids towards nonadipose tissues not only affects liver and muscle metabolism, but may also alter pancreatic β-cell functions. In the Randle hypothesis, a substrate competition occurs in the oxidation pathway between free fatty acids and glucose, to satisfy the energy need of a given tissue. This leads to an impaired utilization of glucose when a high free fatty acid flux leads to increased acetyl CoA in mitochondria (239). Alternatively, it is now suggested that free fatty acids might directly impair insulin signaling.

C. Insulin Resistance: On the Molecular Nature of the Causal Mechanism

The failure of the pancreas to adjust its insulin secretion according to the needs of the organism might lead to a long-term modification of the insulin-sensitive tissue responses and is thought to be one causal mechanism. In particular, the MODY subset of diabetes is connected to this concept. All but one of the factors involved in MODY are transcription factors acting on insulin synthesis and secretion pathways, as has been described in section II B and Figure 3. However, these monogenic determinants of

diabetes do not concern the vast majority of T2DM associated with obesity. Failure of the pancreas may also arise from exposure to high levels of free fatty acids. The actions of an excess of free fatty acids on pancreas function are complex, ranging from regulatory events to lipotoxicity. An acute increase in free fatty acid levels provokes an elevated glucose-stimulated insulin secretion. In obese T2DM patients, insulin secretion is initially high and parallels the increased demand of tissues for an insulin response. Pancreas failure could occur at a later stage, due to exhaustion of the gland. Alternatively, it was shown that chronic high levels of free fatty acids in obese patients inhibit insulin secretion and participate in accumulation in the β-cells of long-chain acyl CoA in the presence of glucose (249, 301). In the model of lipotoxicity, the accumulation of acyl CoA not only alters insulin secretion but also increases β-cell apoptosis, leading to pancreas failure (reviewed in Refs. 289, 302). However, results in experimental models seem to differ between in vitro and in vivo experiments and appear to depend on the species analyzed (rat, mouse, or human islets). The effects also vary depending on whether basal insulin secretion or high glucose-stimulated insulin secretion is being analyzed (reviewed in Ref. 171). Not surprisingly, the role of transcription factors such as SREBPs and PPARs in the accumulation and metabolism of lipids in the pancreatic β-cells and their effects on insulin secretion are presently being analyzed (67, 150, 254, 311), but no working hypothesis can yet be clearly defined. In any case, the pancreas, whose alteration is a critical feature of the progressing disease, does not seem to be the master player in the onset of the T2DM.

At the level of the peripheral cells, mutations in the insulin receptor (IR) gene or in the genes encoding key insulin signaling molecules, such as the transducers IR substrate-1 (IRS1) and IR substrate-2 (IRS2), were examined as a straightforward explanation for insulin resistance. However, polymorphisms in IRS1 or IRS2 are not clearly associated with insulin resistance but may impair β-cell function and insulin secretion (234).

Thus a simple and single causal explanation has to be set aside, and research is now geared to understanding what, in the functioning of the main insulin-sensitive tissues and their cross-talk, initiates and worsens T2DM. Particular attention has been given to muscle, with the hypothesis that the primary defect could be in glucose uptake due to decreased levels of glucose transporters. This hypothesis could not be confirmed so far, even though a decreased expression of Glut4 in skeletal muscle cells may participate in the disorder (229). Along the same lines, PKC-α can be activated by fat metabolites in skeletal muscle where it impairs insulin signaling and glucose transport via decreased tyrosine phosphorylation of IRS-1. Genetic deletion of PKC-α prevents fat-induced defects in insulin signaling and glucose transport in skeletal
etal muscle, pointing to the importance of this pathway in initiating insulin resistance (144).

There is also much interest in the hormone-like substances that might play an important role in the cross-talk between tissues and that have at one point or another been suggested as a possible starting point for the disease. Interestingly, among the molecules described, more than 14 hormone-like substances are peptides secreted by the adipose tissue, identifying it as a major sensor and potential regulator of metabolic status (79). As changing the diet and life habits that primarily affect the adipose mass is a successful method of preventing and improving insulin resistance, some of the adipokines probably participate in the disturbed intertissular communication seen in insulin resistance. The particular physiopathological context of lipodystrophic diabetes highlights the importance of adipokines. Lipoatrophy is characterized by a reduced body-fat mass responsible for a disorder in which insulin resistance and hyperglycemia are associated. In mouse models for lipodystrophy, transplantation of adipose tissue reverses the metabolic abnormalities associated with lipoatrophy (82). A similar benefit is obtained with leptin (217) as well as adiponectin (330). This clearly emphasizes the role of adipokines in regulating insulin sensitivity of peripheral tissues, independent of the ability of the organism to store fat. The nature and mechanisms of the regulatory events taking place in the responsive tissues have yet to be deciphered.

### D. The Thiazolidinediones as a Tool for Understanding the Physiopathology of the Metabolic Syndrome

The discovery that the thiazolidinediones (TZDs), specific activators of PPARγ, modify the insulin-resistance status in humans represents one major step in the search for the molecular mechanisms of the metabolic syndrome. TZDs first decrease plasma-free fatty acid concentrations and fasting hyperglycemia via an insulin-sensitizing effect (137, 266). This class of compounds also has other beneficial effects on T2DM by lowering hypertension if present and improving the overall lipoprotein profile, thereby reducing the risk of atherosclerosis. These benefits, extended to reduced occurrences of heart attack, stroke, and premature death, are now clearly highlighted by the results of the major clinical trial PROactive (www.proactive-results.com). Not surprisingly, however, TZDs enhance adipocyte differentiation, likely explaining the increased weight gain often observed upon its use (84, 98). Other side effects include edema and an idiosyncratic acute hepatotoxicity that seems to be a specific effect of troglitazone, one of the first TZD compounds initially used in clinics. Although it is clear that TZDs are potent and high-affinity ligands for PPARγ, it should be kept in mind that not all TZD effects are mediated by PPAR activation.

---

### 1. Hypothesis for the mechanism involved in PPARγ-mediated improvement of insulin sensitivity

A reasonable hypothesis is that by increasing the ability of the adipose tissue to store the excess of circulating fat, PPARγ diverts the fat that has unduly accumulated in muscle, liver, and pancreas towards fat tissues. It may be particularly efficient as it refurbishes more particularly subcutaneous fat, which is itself more sensitive to insulin than visceral fat (95, 308). This is consistent with the effects of TZD treatment in patients, which does not further increase the deleterious adipogenesis in muscle or the liver, but rather diminishes it. Thus potentially deleterious effects of an increase of the general amount of fat are counterbalanced by a redistribution of these lipids, which prevents excessive fat accumulation in the peripheral organs, including visceral fat, improving their insulin sensitivity (reviewed in Ref. 159).

In addition to fat redistribution, TZDs may also trigger other mechanisms. A direct action of PPARγ activation on the production of adipokines has been investigated, and some data support the hypothesis that adiponectin might be a crucial component connecting PPARγ activation in the adipose tissue and the metabolic response of the peripheral organs (95). Leptin has also been often cited as a potential effector of TZDs. However, consistent clinical data are difficult to assemble, and rodent models have generated some controversies. PPARγ might also directly regulate tumor necrosis factor (TNF)-α expression. A global analysis by DNA microarrays of PPARγ target genes has suggested an interesting mechanism by which PPARγ activation inhibits NF-κB, thereby blocking the TNF-α-mediated inhibition of adipogenesis (260).

One difficulty is revealed by the fact that rodent models of lipodystrophy with associated insulin resistance also benefit from TZD treatment, suggesting the presence of an adipose tissue-independent mechanism. A direct action of TZD acting via PPARγ on the skeletal muscle appears difficult to reconcile with the rather low expression of PPARγ in this tissue. Two reports describe opposing results when TZDs are used in mice selectively lacking PPARγ in muscle, either with persistence of a TZD beneficial response (209) or a loss of TZD response (107) in mutant mice. While these discrepancies are likely due to small but significant differences in the experimental design and methodology, they underline the urgent need for further mechanistic explorations.

Indeed, to better understand these observations and their therapeutic outcome, there is strong hope in learning from selective deletions of PPARγ in various tissues (reviewed in Ref. 148). Muscle-specific deletion of PPARγ is reported to increase adiposity and insulin resistance, suggesting an important intertissular cross-talk, for which the molecules involved remain to be identified (107, 209). Mice in which PPARγ has been specifically deleted in the
liver do not exhibit any phenotype unless they are challenged by an additional mutation with deleterious metabolic effects (ob/ob mice and lipodystrophic A-ZIP/F-1 mice; Refs. 23, 192). This situation likely reflects the very low expression of PPARγ in the normal liver, while it increases and plays a role upon steatosis, as it may occur in the metabolic syndrome (61, 187). Selective deletion of PPARγ in adipose tissue demonstrates its importance for mature adipocyte survival (121). With respect to the metabolic consequences of PPARγ-selective deletion in the adipose tissue, a first report showed that it causes insulin resistance in the adipose tissue and in the liver but not in muscle (103). With the use of a very similar mouse model, a second independent report suggested opposing results, with such mice being protected against high-fat diet-induced obesity and insulin resistance. While the muscle exhibits a diminished glucose uptake, the authors proposed that the protective effect may come from increased glucose utilization by the liver (130). Altogether, these results are difficult to reconcile, but they contribute towards a wealth of knowledge which with more time, experiments, and mouse models, will help in defining the molecular network that controls metabolic homeostasis.

An additional surprise is that when PPARγ activity is decreased via monoallelic disruption of PPARγ, exposure to the agonist TZD does not further enhance insulin sensitivity of the PPARγ +/− mice but has almost the opposite effect, restoring a “wild-type” like phenotype to these mice (198). In this action, TZD might appear as a corrector of insulin sensitivity: it does not affect insulin sensitivity in normal humans and animals, it restores normal insulin sensitivity in human and rodent models with insulin resistance, but it diminishes insulin sensitivity in abnormally sensitive rodents. We have seen above that a TZD treatment, via PPARγ activation, might act by redirecting the triglyceride load from skeletal muscle to adipose tissue, restoring muscle sensitivity to insulin, and thereby improving glucose disposal. PPARγ +/− mice have smaller adipocytes and a smaller fat deposit. Treating these mice with TZD would simply help to restore a “normal” fat mass and would thus place them equal to wild-type mice in the face of a metabolic challenge such as high-fat diet. In contrast, a PPARγ antagonist would inhibit triglyceride accumulation in fat tissue and, as in PPARγ +/− animals, would not allow fat accumulation to occur in ectopic tissues such as muscle, thus preserving insulin sensitivity.

Altogether, these results and pending questions suggest that potent PPARγ agonists might not be the most appropriate treatment of obesity and T2DM. Modulators, which may share antagonist and/or partial agonists properties, might represent a better perspective than thiazolidinediones. However, whereas experiments in mice are promising, the demonstration that these PPARγ modulators can act similarly in adult and obese humans is still lacking.

### 2. The paradox of PPARγ and insulin resistance

One more level of complexity was revealed by the striking observation that PPARγ +/− heterozygous mice, rather than being prone to insulin resistance, are partially protected from high-fat diet-induced or monosodium glutamate-induced weight gain and insulin resistance, compared with their wild-type littermates (154, 199). Lower activation of PPARγ can also be obtained via the use of PPARγ antagonists. Few of these molecules have been identified: GW0072 is a partial agonist in transactivation assays, which inhibits adipocyte differentiation in cell culture (211); BADGE inhibits adipocyte differentiation in cell culture (322) but is highly cytotoxic; PD068235 also blocks adipocyte differentiation but does not revert the phenotype of terminally differentiated adipocytes (26); and LG100641 blocks adipocyte differentiation and stimulates glucose uptake in 3T3-L1 adipocytes (204). However, these inhibitors have not been tested in vivo to verify whether they may prevent obesity and reduce insulin resistance. An in vivo study in mice was performed using SR-202 [dimethyl α-(dimethoxyphosphinyl)-p-chlorobenzyl phosphate], a novel PPARγ-specific antagonist that has no PPARγ agonist activity and partially inhibits adipocyte differentiation induced either by thiazolidinedione or by the combination of dexamethasone, insulin, and IBMX. In vivo, this compound partially blocks PPARγ activity, decreases fat deposits, and increases insulin sensitivity, setting most of the metabolic parameters at the levels of those seen in PPARγ +/− mice (247).

### 3. A possible role of PPARα and PPARβ in the metabolic syndrome

Thanks to its consistent role in stimulating fatty acid oxidation, PPARα could also be an interesting target for controlling the insulin resistance that accompanies dyslipidemia. Whereas fibrates act on lipid profiles mainly through their interaction with PPARα, little evidence for a fibrate action on insulin sensitivity was initially provided. Studies aimed at exploring this possibility demonstrated that the synthetic PPARα ligand Wy-14643, along with fenofibrate and ciprofibrate, are able to improve insulin sensitivity in rodent models of insulin resistance (93, 332). A better consumption of fatty acids by the skeletal muscle, thereby decreasing the overall fat accumulation, is proposed as the main mechanism of PPARα-induced insulin sensitivity. However, in contradiction with the latter observation, PPARα null mice subjected to a high-fat diet or more simply to aging are protected from developing insulin resistance (94).

Notwithstanding these apparent contradictions, a large effort is presently being made by numerous phar-

maceutical companies to test dual agonist compounds, which associate PPARα and PPARγ ligand properties, as a means of controlling some undesired side effects of TZD, which include hypertriglyceridemia, weight gain, and edema. Ragaglitazar, one of the first of these compounds to be tested clinically, seems to be promising, although its development is hampered by the persistence of side effects already mentioned for TZD, e.g., peripheral edema and anemia (285).

A role played by PPARβ in lipid metabolism has been envisioned in the context of obesity, diabetes, and dyslipidemia, based on both the aforementioned effect of a PPARβ ligand on HDL-cholesterol levels in obese monkeys and the decreased adiposity of PPARβ−/− mice (8, 228). However, the tissue-specific gene deletion of PPARβ in the WAT does not alter the fat mass, suggesting a cell-nonautonomous action. It is now reported that PPARβ−/− mice challenged with a high-fat diet are prone to obesity. In addition, an artificially overexpressed and overactive PPARβ in mouse WAT increased fatty acid oxidation and energy dissipation, features that in turn reduce adiposity and improve the lipid profiles (313). This latter observation, while difficult to interpret with respect to physiology, is consistent with a proposed role of PPARβ in muscle fatty acid oxidation (206; reviewed in Ref. 13). Altogether, these observations emphasize the difficulties that lie in addressing the complexity of physiopathology mechanisms that take place in metabolic diseases and the general caution that should be applied in interpreting in vivo observations. It also demonstrates that much work needs to be performed to clarify the benefit that we may gain from targeting PPARβ for diabetes or syndrome X therapeutic strategies.

act. First, the level of expression of a factor might be modulated, often via a known regulator of that gene. For example, glucocorticoids can increase PPARα expression. Understanding the network created by nuclear receptors (see Fig. 11) and their reciprocal interactions on their own gene expression is clearly of importance when establishing the molecular mechanism of action of a ligand, with the prospect of predicting clinical effects. A second general strategy that is commonly used by cells to regulate their metabolic pattern is by acting on the nuclear translocation or exclusion of a transcription factor. This may occur either via dissociating or associating cytoplasmic complexes, or via operating posttranscriptional modifications, such as phosphorylation, which precludes or favors entry of the transcription factor into the nucleus. For example, cytoplasm retention of FOXOs via phosphorylation is suggested to be a mechanism of insulin-mediated gene repression. With respect to drug design strategy, it is somehow difficult to specifically modify the status of a factor of interest, using the physiological means by which protein modification is achieved, since the effect of one single signaling molecule is most often distributed to a full range of factors that affect various pathways. In a third mode of action, the activation of transcription factors in the nucleus relies either on interaction with a ligand, such as for the nuclear receptors, or upon posttranslational modification (phosphorylation, acetylation, or other modifications). Activation implies the formation of specific transcriptional complexes that may vary according to the nature and extent of the modification that occurs. This is particularly explored in the context of nuclear receptors, where the ultimate objective

VII. TRANSCRIPTION FACTORS AS TARGETS FOR THERAPEUTIC APPROACHES OF METABOLIC DISORDERS

In this section, we briefly present how the modulation of transcription factor activities might be used for the treatment of metabolic disorders. The example of nuclear receptors, already discussed above for PPARs, and which are a major focus of pharmaceutical companies at the present time, will be extended to some other members of this family. Other drugs that will be analyzed are those aimed at treating unrelated specific pathologies, but which have serious side effects in metabolic regulation. The glucocorticoid analogs and the protease inhibitors are among the compounds in this category.

A. How to Modulate Transcription Factor Activity

The activity of transcription factors is the result of a number of events on which therapeutic strategies may

FIG. 11. Summary of the network established by the transcription factors involved in metabolic regulation. Each of the transcription factors mentioned in this figure participates in the regulation of at least one aspect of metabolism, often by sensing metabolite levels and adapting the cell response through transcriptional regulation of enzymes belonging to different pathways. In addition, each of them may influence the activity of the others, creating a regulatory network by which homeostasis is achieved. The numbers in parentheses correspond to the bibliographical references giving support to the regulation proposed. This list of references is not exhaustive for space reasons. Various reports show an autoregulation of PPARγ but do not specifically address this issue.

is to generate specific ligands that might trigger a conformation change specific enough to provide selectivity upon the transcriptional complex formed, allowing a specific pattern of gene activation.

B. Nuclear Receptors as Targets for New Therapeutic Approaches

The general interest of pharmaceutical companies in the nuclear receptor family stems from the “druggability” of nuclear receptors. They naturally bind and respond to small lipophilic molecules; the search is for both agonists and antagonists. The notion that modulators differentially affect transcriptionally active complexes, together with the pattern of expression of these receptors and of their coactivator and corepressors, make it possible to develop tissue-specific responses. In the next paragraph, we give a brief description of the present achievements and hopes with respect to nuclear receptor targeting in metabolic diseases, without mentioning PPAR ligands since this has been thoroughly discussed above.

RXR ligands have been considered as a therapeutic approach for metabolic perturbations and have been tested in mice. A heterodimer composed of RXR and a permissive partner such as PPAR, LXR, or FXR can be activated by a RXR ligand, independently of the presence or absence of a ligand for the partner receptor. Rexinoids are particularly attractive for their potential ability to trigger PPAR:RXR activation (220). Indeed, benefits such as potent antidiabetic effects have been reported (203). However, these are accompanied by a massive increase in circulating triglycerides (248), possibly due to the parallel activation of the LXR:RXR pathway. In addition, RXR agonists also interfere with thyroid hormone receptor signaling, provoking a profound hypothyroidism due to a decrease in TSH levels (174). As rexinoids also trigger the LXR:RXR and FXR:RXR pathways, their action on cholesterol metabolism has been evaluated. Oral treatment with rexinoids results in a very efficient inhibition of cholesterol uptake by the gut. This effect was reported to involve at least two mechanisms: 1) the activation of LXR:RXR in the intestine, causing an increased cholesterol efflux in the lumen, and 2) the activation of FXR:RXR in the liver, producing an inhibition of bile acid synthesis, thus reducing micelle formation and cholesterol uptake in the gut (245). In a different context in humans, oral bexarotene (Targretin capsules corresponding to the well-characterized rexinoid LG1069) has been used successfully since the 1980s for the treatment of refractory or persistent early-stage cutaneous T-cell lymphoma. It acts by inhibiting mitogen-induced interleukin-4 production by the peripheral blood cells and inducing apoptosis of malignant T cells. However, studies of phase 2 and 3 clinical trials of cutaneous T-cell lymphoma by bexarotene re-

ported high triglyceridemia, hypothyroidism, but also hypercholesterolemia (60). As these observations helped to illustrate, rexinoid effects might not be as specific as wished for, and therefore, the potential therapeutic use of rexinoids requires a careful evaluation. Alternatively, the search for and development of heterodimer-specific ligands for RXR might allow a better targeting of the pathways of interest (70, 197).

LXR is a metabolic sensor for cellular cholesterol and a major regulator of cholesterol homeostasis, designating it as a pertinent target for drug development (76). More particularly, and as discussed in previous sections, LXR agonists should raise HDL levels, increase reverse cholesterol transport, increase cholesterol elimination via bile acid synthesis, and decrease intestinal cholesterol uptake. Accordingly, the LXR ligand T0901317 increases HDL levels. However, consistent with the role of LXR in lipogenesis, this is accompanied by an increased hepatic production of triglycerides and elevated blood VLDL, together with an accumulation of liver triacylglycerol. The direct effect on glucose tolerance seems to be modest (39). Thus the dual action of LXR on cholesterol disposal and fatty acid synthesis makes the therapeutic potential of LXR agonists rather difficult to predict. The description of the anti-atherosclerotic activities of a new LXR ligand GW3965 suggests that specific modulators might be at hand (133). This clearly needs further evaluation.

FXR is a metabolic sensor for bile acids that mediates their effects on gene expression, particularly by creating a negative-feedback loop controlling their synthesis. Treatments aimed at decreasing the body content of cholesterol should thus antagonize FXR, allowing for an increased production of bile acids from cholesterol in excess. Amazingly, such molecules seem to have been used for ages in Indian traditional medicine. While the gum resin from the *Commiphora mukul* tree was shown to have hypolipidemic activities, it has now been established that its active phytosterol, the guggulsterone, is an effective FXR antagonist (for review, see Ref. 304). Clinical studies have reported a 10–20% reduction in triglyceride levels and a 20–30% reduction in cholesterol levels. Side effects seem to be mild and do not affect basic metabolic pathways. However, 20–30% of patients are nonresponsive to the drug. Finally, it is also important to note that gugulipid interacts with the pregnane X receptor (PXR), raising the concern of drug-drug interference via activation of PXR-mediated increase of the expression of drug metabolizing enzymes. Ironically, this rediscovery underlines the continuing importance of empirical approaches for therapeutic strategies.

In contrast, orphan receptors may represent interesting alternative therapeutic targets. Understanding of their main function is still lacking, often because of the lack of adequate tools for inhibiting or activating them. However, many of them share a ligand-binding domain structure

that suggests the existence of natural and possibly of synthetic ligands that so far remain unidentified. With this in mind, orphan receptors can be considered as an almost untouched reservoir of innovation.

### C. Adverse Effects of Drugs on Energy Metabolism

A number of drugs not initially aimed at treating any metabolic disorder cause severe problems because of important metabolic deregulations, some of which are mediated by transcription factor alterations.

Perhaps the best example is glucocorticoid analogs, mainly used as anti-inflammatory drugs. As already seen, glucocorticoids affect metabolic pathways, particularly those of insulin sensitivity, amino acid degradation, and muscle physiology. Not all mechanisms that cause such disturbances are known, although a direct action of the nuclear glucocorticoid receptor on some target genes is likely to be the main one. Glucocorticoid receptors may also act indirectly by increasing the expression of PPARα. Here again, identifying specific modulators that would provide a better focus for the therapeutic activity in a given tissue or on a given set of target genes represents a major challenge for the future.

An intriguing question concerns the metabolic consequences of the use of protease inhibitors to treat human immunodeficiency virus (HIV) infection. Dyslipidemia occurs in up to 70–80% of the treated patients, often with associated hypertriglyceridemia and hypercholesterolemia. A severe lipodystrophy is also found in many but not all occurrences. The mechanism of this deregulation remains hypothetical thus far. A first hint is the identity, ~60%, of the catalytic domain of the HIV protease with that of the cytoplasmic retinoic acid binding protein type 1. The hypothesis is that the protease inhibitor might erroneously bind to this protease-related domain, affecting the retinoic acid signaling pathway. It also might similarly affect the low-density lipoprotein receptor related protein (LRP), which also encompasses a domain sensitive to protease, resulting in disturbed lipoprotein metabolism. Such perturbations could explain both the alteration in lipid metabolism seen in nontreated HIV patients and the aggravation upon treatment with protease inhibitors (reviewed in Ref. 25). Alternatively, it has been suggested that protease inhibitors might affect the SREBP maturation process, resulting in impaired nuclear accumulation of SREBP-1 (27, 317). A better understanding of the molecular mechanisms at work would clearly be extremely helpful for designing the best therapeutic strategy.

### VIII. CONCLUSIONS

A conclusion for such a long story must be short. Thus let us highlight only one important feature of the

transcriptional regulation of metabolism. Most of the observations gathered for one pathway and its regulation via a given transcription factor show that this regulation not only targets the expression of rate-limiting enzymes but also coordinately regulates the expression of the enzymes working at many intermediary steps of the pathway. The initial concept of regulation operating at the level of the rate-limiting step is losing importance in favor of coordinated regulation. This feature must now be put in parallel with what should also strike us from the present review, i.e., the intense networking between transcription factors by which homeostasis can be achieved (see Fig. 11). Thus, in addition to classical approaches, metabolic explorations must be included in global approaches, such as analyses of the transcriptome, proteome, and metabolome. However, even more than for classical approaches, such experiments must be designed to answer well-defined and physiologically relevant questions. While the first reports on complex metabolic networks drawn from bacteria models are beginning to appear, the realization of such a vision of mammalian metabolic regulation still seems to be some years away.

### ACKNOWLEDGMENTS

We are very grateful to L. Gelman, B. Thorens, E. Bedu, J. Feige, and S. Anghel for careful and critical reading of the manuscript.

Address for reprint requests and other correspondence: B. Desvergne, Center for Integrative Genomics (CIG), Univ. of Lausanne, Genopode Building, CH-1015 Lausanne-Dorigny, Switzerland (e-mail: beatrice.desvergne@unil.ch).

### GRANTS

The work performed in the authors’ laboratories is funded by the Swiss National Science Foundation, the National Centre of Competence in Research “Frontiers in Genetics,” the EUMORPHIA Program from the FP5 of the European Community, and the Canton of Vaud.

### APPENDIX

#### Appendix A: The Nuclear Receptor Family

Nuclear receptors are transcription factors that are characterized by two important properties: 1) they are activated upon binding of specific ligands, and 2) they bind to specific response elements located in the vicinity of the promoter of their target genes. Thus, in a simplified view, the effector function of nuclear receptors in a cell is to adapt the gene expression program according to signals received as specific ligands. An official nomenclature of these receptors across species is now used, organized according to their phylogeny (210; reviewed in Ref. 3).

TRANSCRIPTIONAL REGULATION OF METABOLISM

1. **Nuclear receptors share a common structural and functional organization (Fig. 1A)**

The poorly structured NH₂-terminal domain encompasses a ligand-independent transactivation domain in some of the receptors. The DNA binding domain (DBD) with its two zinc fingers is the hallmark of the nuclear receptor family. The hinge region links the DNA binding domain to the ligand binding domain (LBD). The general fold of the ligand binding domain is structured by 12 α-helices and 3 β-sheets defining the ligand binding pocket (see Fig. 1A, *top left*).

In general, the nuclear receptors bind to DNA in the form of dimers, either homodimers, such as HNF4α or the estrogen receptor, but more often as heterodimers with the receptor for 9-cis-retinoic acid known as RXR (i.e., NR2B according to the unified nomenclature of nuclear hormone receptors; Ref. 210). The DNA response element of nuclear receptors is comprised of two sequence motifs corresponding or closely related to the hexamer AGGTCA. The organization of these two motifs in direct repeats (DR) or palindromic arrays and the length of the spacing between the two hexamers determine the specificity of these response elements towards each receptor dimer (Fig. 1A, *top right*).

The general scheme for transactivation via nuclear receptors is thought to occur in at least two steps. In the absence of ligands, nuclear receptor dimers may bind a corepressor protein that inhibits their transactivation properties. In the presence of ligands, or due to an alternative pathway of activation such as phosphorylation, the corepressor is released and a coactivator is recruited, allowing further interactions with the transcription initiation complex (TIC), eventually resulting in transcription enhancement. Corepressors and coactivators work at least in part by modulating the chromatin status via histone deacetylation and acetylation, respectively. Other modifications such as histone methylation have also been shown (Fig. 1A, *bottom*).

2. **The three functional classes in the nuclear receptor family (Fig. 1B)**

Analyses of the human genome have identified 48 nuclear receptor genes, most of them generating more than one receptor isoform. Based on their ligand binding properties, nuclear receptors can be ordered into the three following classes.

The classic hormone receptors bind molecules such as glucocorticoids, thyroid hormone, retinoic acid, and estrogen, with a high affinity. As key factors of endocrine homeostasis, their activation is associated with many metabolic adjustments (Fig. 1B, *left*).

The orphan receptors possess the structural characteristics of nuclear receptors including a sequence consistent with the presence of a ligand binding domain. However, no ligand has thus far been identified for these receptors. In addition, a tight structure of the 12 helices has been shown in the Nurr1 receptor to preclude the formation of a ligand binding pocket (315). The functions of many orphan receptors remain elusive (Fig. 1B, *right*).

The receptors of the third class are metabolic sensors. This class comprises receptors that bind to a broad range of molecules with, as a corollary, a relatively poor affinity. Rather than responding to hormones secreted by endocrine glands with tight feedback controls, these receptors can bind to molecules that are components of metabolic pathways as substrates, intermediates, or end-products, such as fatty acids, eicosanoids, and oxysterols. These receptors are sensors of the metabolic status, respond to both incoming dietary signals and metabolites generated in the organism, and are responsible for the metabolic adaptation at the cell, organ, and whole organism level. In this class are PPARs and HNF4α, which play a major role in energy metabolism, as well as LXR and FXR, which are closely involved in cholesterol metabolism, together with their common partner RXR. RXR itself is difficult to assign to a distinct class, as it behaves as a classic receptor with respect to 9-cis-retinoic acid. However, the nature of its endogenous ligand is still unclear, and the functions of RXR rather plead for its belonging to the sensor receptor class (see also Appendix F). HNF4 is accompanied by * since its mode of interaction with ligands is disputed (see also Appendix E) (Fig. 1B, *middle*).

**Appendix B: Peroxisome Proliferator Activated Receptors**

PPARs were the first nuclear receptors identified as “sensors” rather than classic hormone receptors. They are nuclear, lipid-activatable molecules that control a variety of genes in several pathways of lipid metabolism (reviewed in Ref. 56).

Three isotopes of PPAR, PPARα (NR1C1), PPARβ (NR1C2) (also called PPARδ, NUCI, and FAAR), and PPARγ (NR1C3), have been cloned in Xenopus, rodents, and humans. Two PPARγ isoforms, PPARγ1 and PPARγ2, are splice variants in their NH₂-terminal domain.

PPARα is highly expressed in tissue with high activity levels of lipid catabolism, e.g., liver, brown adipose tissue, and skeletal and heart muscle. PPARβ is ubiquitously expressed. PPARγ1 is mainly expressed in adipose tissues but is also detected in the colon, spleen, retina, hematopoietic cells, and skeletal muscle. PPARγ2 has been found mainly in the brown and white adipose tissue.

Their modular structure is that of all nuclear receptors. The less conserved NH₂-terminal region bears a ligand-independent activation domain, at least in PPARα and PPARγ. The DNA binding domain is extremely well conserved. The ligand binding pocket of PPARs is much larger than that of the other nuclear receptors and relatively easily accessible (324 and references therein).

PPARs bind to DNA as heterodimers with RXR, on PPAR response elements (PPRE) comprising direct repeats of two hexamers closely related to the sequence AGGTCA and separated by one nucleotide (DR-1 sequence). The five nucleotides that flank the 5′-end of this core sequence are also important for the efficiency of PPARα:RXR binding.

The first molecules to be recognized as PPARα activators, and later characterized as ligands, belong to a group of molecules that induce peroxisome proliferation in rodents, thus explaining the name of peroxisome proliferator activated receptor given to this receptor. This diverse group of substances includes, for example, some plasticizers and herbicides. More interestingly, various fatty acids, more particularly unsaturated fatty acids, and some eicosanoids mainly derived from arachi-

donic acid and linoleic acid, bind to PPARα, -β, and -γ with varying affinities. In addition to being activated by fatty acids, PPARα responds to fibrates that are hypolipidemic drugs, and PPARγ responds to thiazolidinediones that are insulin sensitizers, demonstrating their potential as drug targets.

In the process of transcriptional regulation, ligand-bound PPARs recruit coactivators, most likely organized in large complexes (291). Cofactor recruitment may be PPAR isotype specific and may ensure the specificity of target gene activation. In addition to PPAR ligand binding, PPARs can also be activated by phosphorylation of serines located in the A/B domain, and the PPAR:RXR heterodimer can be activated by RXR ligands. The relevance of these alternate pathways is under study in several laboratories and might lead to interesting new developments.

As can be expected from sensors, PPARs, which recognize and bind a variety of fatty acids, regulate in turn most of the pathways linked to lipid metabolism. Most fascinating is the observation of balanced regulatory actions between fatty acid oxidation in the liver and other organs, via PPARα, and fatty acid storage in the adipose tissue, via PPARγ. In contrast, the role of PPARβ remains elusive, although evidence is emerging for its function in lipid and cholesterol metabolism and transport (reviewed in Ref. 196). However, PPARs are involved in all three main branches of intermediary metabolism, as is detailed in the present review.

## Appendix C: Liver X Receptor

The liver X receptors (LXR; NR1H3) are members of the nuclear receptor family. Their endogenous activators are oxysterols and other derivatives of cholesterol metabolism. As such, they participate in the cholesterol sensing processes and regulate important aspects of cholesterol and fatty acid metabolism (298).

Two isoforms, LXRα and LXRβ, share 77% amino acid identity in their DBD and LBD and are highly conserved between rodents and human.

LXRα is highly expressed in the liver but is also found in kidney, intestine, adipose tissue, and macrophages. LXRβ is expressed ubiquitously.

LXRs heterodimerize with RXR to bind to their DNA response element, formed from a direct repeat of two hexamers related to the sequence AGTTCA, separated by four nucleotides.

Mono-oxidized derivatives of cholesterol are potent LXR ligands. The most potent of these are 22(R)-hydroxycholesterol, 24(S)-hydroxycholesterol, and 24(S),25-epoxycholesterol, which activate both LXRα and LXRβ. Little is known about the sterol hydroxylases that produce these metabolites, but it is assumed that oxysterol concentrations parallel those of cholesterol. Importantly, oxysterols are found at micromolar concentrations in tissues that express high levels of LXRα or LXRβ. Activation of the heterodimer can also be triggered by RXR ligands.

LXRα and LXRβ null mutant mice have been generated and confirm the important role of these receptors, and more particularly that of LXRα in cholesterol homeostasis. These models also pinpoint the role of LXR in fatty acid metabolism.

## Appendix D: Farnesol X Receptor

The farnesol X receptor (FXR, NR1H4) is a member of the nuclear receptor family, which acts as a bile acid sensor and is involved in a negative-feedback regulation to control and moderate excess of bile acid production (reviewed in Ref. 62). FXR was initially cloned as a RXR interacting protein called RIP14.

FXR expression is restricted to adrenal cortex, intestine, colon, kidney, and liver.

FXR forms heterodimers with RXR to bind to DNA response elements consisting of two hexamers (GGGTCA), spaced by one nucleotide, organized in a palindromic configuration (also called inverted repeat-1).

Initially proposed as a receptor of farnesol metabolites, and thus renamed FXR from RIP14, it was shown to bind and be activated by physiological concentrations of free and conjugated bile acid products: chenodeoxycholic acid, lithocholic acid, and deoxycholic acid. These bile acids are the end products of the neutral and acidic bile acid biosynthetic pathway. The heterodimer FXR:RXR can also be activated by RXR ligands.

FXR acts as a bile acid sensor that regulates bile acid metabolism. Because bile acids are an important means for disposal of cholesterol in excess, FXR thereby affects cholesterol metabolism.

## Appendix E: Hepatic Nuclear Factor 4

Hepatocyte nuclear factor 4α (HNF-4α, NR2A1) is a highly conserved member of the nuclear receptor family.

HNF-4α was first identified as a member of a heterogeneous family of transcription factors named the liver-enriched nuclear factor family. At least nine splice variants have been described.

HNF4α is mainly expressed in the liver, intestine, and pancreatic β-cells.

HNF4α mainly binds as homodimer to response elements closely related to DR1 elements, thus resembling PPREs (see Appendix B).

The question of the status of HNF4α as orphan or liganded receptor is still unsolved (286). While a report indicated that fatty acyl CoA thioesters could bind to HNF4α and modulate its activity (105), these compounds were not able to alter HNF4α interaction with cofactors (16). Its transactivation properties, readily present when assessed in transfection assays, also support the proposition that it acts as a constitutively active orphan receptor. These observations may be reconciled by the crystallographic structure of the ligand binding domain, which showed that saturated or monounsaturated C14-C18 fatty acids not only fill the ligand binding pocket but appear to be locked in the protein, suggesting that it plays a structural role rather than acts as an activating signal (320).

Possibilities other than ligand binding for modulating HNF4 activity have thus been proposed. Alternative splicing of the mRNA, phosphorylation of the protein (13 phosphorylation sites exist), and/or inhibitory interaction with phosphorylated FOXOs (108) are among the numerous possible ways of regulating HNF4α activity.

HNF4α affects the expression of genes involved in glucose, fatty acid, and cholesterol metabolisms.

TRANSCRIPTIONAL REGULATION OF METABOLISM

Appendix F: Retinoid X Receptor

Except for HNF4α, the nuclear receptors of the metabolic sensor class are active as heterodimers with RXR. The important part that RXR may play is further emphasized by the fact that RXR is itself a nuclear receptor that can be activated by specific ligands. PPAR:RXR, LXR:RXR, and FXR:RXR are permissive heterodimers within which RXR can bind its own ligand, in the absence of a ligand for its partner, and can thereby activate the transcription of the heterodimer target genes.

There are three isoforms of RXR, α, β, and γ, and several isoforms for each of them (reviewed in Ref. 30).

Each isoform has its specific expression pattern. However, any single tissue contains one or several forms of RXR.

In addition to the various heterodimers for which RXR is an obligatory partner, RXR can form homodimers that bind to DR1 elements (see also Appendix B). The in vivo relevance of these homodimers is still under study.

RXR can be activated by 9-cis-retinoic acid, an isomer of all-trans-retinoic acid. While the occurrence of this molecule in vivo has been questioned, the identification of two enzymes that participate in the isomerization of all-trans-retinoic acid to form 9-cis-retinoic acid lends support for its relevance in the whole organism. However, some doubts linger about the nature of the major natural RXR ligand. An oleic acid molecule was found in the RXR ligand binding pocket in a crystal of RAR:RXR ligand binding domains (17). Whether this indicates that fatty acids might be RXR ligands remains to be confirmed. In support of this notion, RXR is activated by the long-chain polyunsaturated fatty acid docosahexaenoic acid in the adult mouse brain (51).

RXRα null embryos die in utero, whereas the RXRγ and RXRβ null animals do not display any severe phenotype (188, 316). To overcome RXRα embryonic lethality, a mouse allowing tissue-specific invalidation of the RXRα gene has been generated. Deletion of the RXRα gene in the liver allowed the identification of the most affected pathways (309). As expected, many PPARα-mediated functions were altered, and the activity of LXR and FXR were also compromised, suggesting that the absence of RXRα cannot be compensated by RXRβ and RXRγ in the liver.

SREBP-1a compared with that of SREBP-1c. SREBP-1c was initially cloned in rats and called adipocyte determination and differentiation factor-1 (ADD1) (reviewed in Ref. 256).

SREBP-1a has been mainly studied in cell lines, where it is the major form produced, while its expression in animal tissues is relatively weak. In contrast, SREBP-1c is highly expressed in liver and WAT. SREBP-2, also expressed in cell lines, is also predominantly present in the liver and adipose tissue but has a rather weak expression in animal tissues overall.

The three proteins SREBP-1a, SREBP-1c, and SREBP-2 are synthesized as precursors of ~125 kDa, entangled in the ER membrane. Maturation of these proteins requires the activation of the SCAP, a chaperone protein for SREBPs. SCAP acts as a sensor of the content of cholesterol in the ER membrane. In the presence of high levels of cholesterol, SCAP remains anchored in the ER membrane due to its interaction with the polytopic INSIG proteins.

Low membrane cholesterol levels lead to the transport of SCAP/SREBP to the Golgi membrane where activation of the site 1 serine protease results in a first cleavage. A second enzyme, the site 2 metalloproteinase, completes the maturation of SREBPs and releases the 68-kDa NH₂-terminal domain of SREBP from the membrane (reviewed in Ref. 63). This fragment contains a basic helix loop helix (HLH) leucine zipper domain, which functions as a transcription factor upon translocation into the nucleus (see Fig. 7).

The mature forms of SREBPs bind to elements initially characterized as featuring an enhancer sequence called E-box that is recognized by members of the HLH transcription factor family. SREBPs also bind to sites related to the direct repeat TCANCCAC. However, it remains difficult to define a consensus sterol response element (SRE). SREBP-1 and SREBP-2 have preferential target genes, which may be determined, at least in part, by interaction of the mature forms with different cofactors. However, this remains to be demonstrated.

Germline deletion of SREBP-1 (eliminating both SREBP-1a and SREBP-1c) or SREBP-2 leads to partial or fully penetrant embryonic lethality, respectively. In contrast, specific deletion of the SREBP-1c transcript is not lethal, suggesting an important role of SREBP-1a and SREBP-2 in embryonic development.

The SREBP maturation process via membrane cholesterol sensing is consistent with their important role in cholesterol homeostasis. However, and in a schematic view, SREBP-2 is indeed mainly involved in cholesterol metabolism, whereas SREBP-1c has an important implication in fatty acid synthesis (see also Fig. 8). The range of target genes activated by SREBP-1a is wider, encompassing many key genes for both cholesterol and fatty acid synthesis.

Appendix G: Sterol Regulatory Element Binding Proteins

The sterol regulatory element binding proteins (SREBPs) are transcription factors of the helix-loop-helix family. Starting out wrapped in the ER membrane, their activation as transcription factors requires a maturation process tightly controlled by the levels of cholesterol present in the membrane (see Fig. 2). They are thus important regulators of cholesterol metabolism, most notably with respect to cholesterol biosynthesis, but also have an important role in fatty acid metabolism (reviewed in Refs. 63, 114, 218).

SREBPs are encoded by two genes, SREBP-1 and SREBP-2. Alternative promoter usage and alternative splicing of SREBP-1 drive the production of two isoforms, SREBP-1a and SREBP-1c. The 29 additional amino acids present in the SREBP-1a NH₂ terminus are enriched in acidic residues and might be responsible for the higher transcriptional activity of

Appendix H: The Liver-Enriched Transcription Factors

The class of liver-enriched transcription factors regroups very heterogeneous factors initially described for their role in liver development and hepatic cellular functions. However, their expression and subsequent actions are often not restricted to the liver (reviewed in Ref. 271). Among these factors, two main subgroups are involved in metabolic regulations, which are briefly presented below.

504 BEÁTRICE DESVERGNE, LILIANE MICHALIK, AND WALTER WAHLI

1. **Hepatocyte nuclear factors**

   Hepatocyte nuclear factors form a heterogeneous subgroup of transcription factors, each of them being evolutionarily conserved.

   HNF4α belongs to the family of the hormone nuclear receptors and is described in Appendix E.

   HNF1α (or LF-B1) and the related vHNF1 (HNF1β or LF-B3) contain a homeodomain motif through which they bind to DNA. HNF1α is crucial for the liver-specific expression of albumin and has strong implications in normal pancreatic β-cell functions (233).

   HNF3α, -β, and -γ are members of the forkhead family of transcription factors (263), therefore also called FOXA1, -2, and -3, respectively. They are identified by a DNA binding domain structured as a winged helix motif (i.e., an helix-turn-helix motif with an adjacent additional DNA interaction region). Whereas their roles have mainly been explored in the context of development, these factors are also involved in signaling cascades in pancreatic β cells, particularly affecting the regulation of expression of the *insulin* gene.

2. **CCAAT/enhancer-binding proteins**

   C/EBPs are members of the basic leucine zipper family (bZIP) and were predicted to play a major role in energy metabolism (reviewed in Refs. 238, 253).

   Six genes coding for the various isoforms of C/EBP have been identified (renamed α, β, γ, δ, ε, and ζ, according to a systematic nomenclature), some of them coding for several isoforms. C/EBPα, -β, and -δ are the isoforms involved in the transcriptional control of metabolic pathways.

   C/EBPα, -β, and -δ are highly expressed in liver, adipose tissues, lung, and intestine.

   C/EBPs bind DNA via a basic amino acid-rich domain, adjacent to a leucine zipper motif responsible for interaction with the homologous domain of the dimerization partner. The high conservation of this bZIP domain among C/EBPs results in an apparent lack of discrimination between the different isoforms for interaction with C/EBP DNA binding sites and dimerization between the different isoforms.

   The transactivation function of C/EBPs, which are considered as positive constitutive transcriptional regulators, is localized in their NH₂ terminus. C/EBPγ, which lacks the transactivation domain, and C/EBPζ, which has impaired DNA binding properties, can form inactive heterodimers with other isoforms and thus have a dominant negative activity with respect to the α-, β-, δ-, and ε-isoforms. However, the assessment of the physiological importance of these two negative forms needs further investigation.

   C/EBPs and more particularly C/EBPα affect the three main branches of the intermediary metabolism, i.e., glucose, amino acid, and lipid metabolisms.

**Appendix I: Insulin Resistance: Definition and Characteristics**

Diabetes is considered as a major world-wide health problem. Type 1 diabetes is due to the absolute lack of insulin, most often linked to a specific destruction of the β-cells of the pancreatic islets, specialized in insulin production and secretion. They represent 5–10% of all cases. Type 2 diabetes is far more common, and it is estimated that ~25% of the adult citizens in western countries are affected by this syndrome. Several terms are currently used to describe this pathology, and efforts have been made to unify the definition and characterization of the various aspects of the disease. The World Health Organization (WHO) has proposed a series of criteria, which is not easy to evaluate in clinical practice. The National Cholesterol Education Program Expert Panel (NCEP) proposed a more practical approach but does not define the insulin resistance status. Below are some of the definitions proposed by the European group for the study of insulin resistance (EGIR) and the WHO. However, these definitions are still being tested for their relevance in epidemiological studies (6, 155).

Impaired glucose tolerance is assessed by the glucose tolerance test, during which a charge of glucose (75 g) is given orally to a patient and glycemia is followed up to 2 h after the initial charge. A persistent high level of blood glucose (≥7.8 mM) 2 h after glucose intake is a criterion for impaired glucose tolerance.

The diabetes status corresponds to chronic hyperglycemia, with high fasting plasma glucose levels (≥6.1 mM) and impaired glucose tolerance with a glycemia 2 h after glucose intake that is ≥10 mM.

Insulin resistance syndrome (according to EGIR) is defined as hyperinsulinemia associated with two or more of the other components: hyperglycemia, hypertension, dyslipidemia, and central obesity. Insulin resistance is best assessed by euglycemic hyperinsulinemic clamp, which consists of a constant infusion of insulin with a parallel infusion of glucose to ensure euglycemia. The amount of glucose needed to maintain euglycemia under the controlled hyperinsulinemia is used to evaluate the status of insulin sensitivity of the patient. Alternatively, for large epidemiological studies and for practical reasons, EGIR proposes to use hyperinsulinemia, defined as fasting insulin levels in the upper 25%, as a surrogate marker of insulin resistance.

The metabolic syndrome (according to WHO) associates at least insulin resistance and/or impaired glucose tolerance with two or more of the other components: insulin resistance, impaired glucose regulation, hypertension, dyslipidemia (high triglycerides and low HDL cholesterol), central obesity, and microalbuminuria.

**REFERENCES**

1. Amemiya-Kudo M, Shimano H, Yoshikawa T, Yahagi N, Hasty AH, Okazaki H, Tamura Y, Shionoiri F, Iizuka Y, Ohashi K, Osuga J, Harada K, Gotoda T, Sato R, Kimura S, Ishibashi S, and Yamada N. Promoter analysis of the mouse sterol regulatory element-binding protein-1c gene. *J Biol Chem* 275: 31078–31085, 2000.
2. Amri EZ, Bonino F, Ailhaud G, Abumrad NA, and Grimaldi PA. Cloning of a protein that mediates transcriptional effects of fatty acids in preadipocytes. Homology to peroxisome proliferator-activated receptors. *J Biol Chem* 270: 2367–2371, 1995.
3. Aranda A and Pascual A. Nuclear hormone receptors and gene expression. *Physiol Rev* 81: 1269–1304, 2001.
4. Attie AD. Insig: a significant integrator of nutrient and hormonal signals. *J Clin Invest* 113: 1112–1114, 2004.

5. Avisar N, Shiftan L, Ben-Dror I, Havazelet N, and Vardimon L. A silencer element in the regulatory region of glutamine synthetase controls cell type-specific repression of gene induction by glucocorticoids. *J Biol Chem* 274: 11399–11407, 1999.

6. Balkau B, Charles MA, Drivsholm T, Borch-Johnsen K, Wareham N, Yudkin JS, Morris R, Zavaroni I, van Dam R, Feskens E, Gabriel R, Diet M, Nilsson P, and Hedblad B. Frequency of the WHO metabolic syndrome in European cohorts, and an alternative definition of an insulin resistance syndrome. *Diabetes Metab* 28: 364–376, 2002.

7. Balsalobre A, Damiola F, and Schibler U. A serum shock induces circadian gene expression in mammalian tissue culture cells. *Cell* 93: 929–937, 1998.

8. Barak Y, Liao D, He W, Ong ES, Nelson MC, Olefsky JM, Boland R, and Evans RM. Effects of peroxisome proliferator-activated receptor delta on placentation, adiposity, and colorectal cancer. *Proc Natl Acad Sci USA* 99: 303–308, 2002.

9. Barak Y, Nelson MC, Ong ES, Jones YZ, Ruiz-Lozano P, Chien KR, Koder A, and Evans RM. PPAR gamma is required for placental, cardiac, and adipose tissue development. *Mol Cell* 4: 585–595, 1999.

10. Barger PM, Brandt JM, Leone TC, Weinheimer CJ, and Kelly DP. Deactivation of peroxisome proliferator-activated receptor-alpha during cardiac hypertrophic growth. *J Clin Invest* 105: 1723–1730, 2000.

11. Barger PM, Browning AC, Garner AN, and Kelly DP. p38 mitogen-activated protein kinase activates peroxisome proliferator-activated receptor alpha: a potential role in the cardiac metabolic stress response. *J Biol Chem* 276: 44495–44501, 2001.

12. Bastie C, Holst D, Gaillard D, Jehl-Pietri C, and Grimaldi PA. Expression of peroxisome proliferator-activated receptor PPARdelta promotes induction of PPARgamma and adipocyte differentiation in 3T3C2 fibroblasts. *J Biol Chem* 274: 21920–21925, 1999.

13. Bedu E, Wahli W, and Desvergne B. Peroxisome Proliferator-activated receptor βδ as a therapeutic target for metabolic diseases? *Exp Opin Ther Targets*. In press.

14. Berge KE, Tian H, Graf GA, Yu L, Grishin NV, Schultz J, Kwiterovich P, Shan B, Barnes R, and Hobbs HH. Accumulation of dietary cholesterol in sitosterolemia caused by mutations in adjacent ABC transporters. *Science* 290: 1771–1775, 2000.

15. Bocher V, Pineda-Torra I, Fruchart JC, and Staels B. PPARs: transcription factors controlling lipid and lipoprotein metabolism. *Ann NY Acad Sci* 967: 7–18, 2002.

16. Bogan AA, Dallas-Yang Q, Ruse MD Jr, Maeda Y, Jiang G, Nepomuceno L, Scanlan TS, Cohen FE, and Sladek FM. Analysis of protein dimerization and ligand binding of orphan receptor HNF4alpha. *J Mol Biol* 302: 831–851, 2000.

17. Bourguet W, Vivat V, Wurtz JM, Chambon P, Gronemeyer H, and Moras D. Crystal structure of a heterodimeric complex of RAR and RXR ligand-binding domains. *Mol Cell* 5: 289–298, 2000.

18. Braissant O, Foufelle F, Scotto C, Dauca M, and Wahli W. Differential expression of peroxisome proliferator-activated receptors (PPARs): tissue distribution of PPAR-alpha, -beta, and -gamma in the adult rat. *Endocrinology* 137: 354–366, 1996.

19. Brown MS and Goldstein JL. A proteolytic pathway that controls the cholesterol content of membranes, cells, and blood. *Proc Natl Acad Sci USA* 96: 11041–11048, 1999.

20. Brown SA, Zumbrunn G, Fleury-Olela F, Preitner N, and Schibler U. Rhythms of mammalian body temperature can sustain peripheral circadian clocks. *Curr Biol* 12: 1574–1583, 2002.

21. Bruhat A, Averous J, Carraro V, Zhong C, Reimold AM, Kilberg MS, and Fafournoux P. Differences in the molecular mechanisms involved in the transcriptional activation of the CHOP and asparagine synthetase genes in response to amino acid deprivation or activation of the unfolded protein response. *J Biol Chem* 277: 48107–48114, 2002.

22. Bruhat A, Jousse C, Carraro V, Reimold AM, Ferrara M, and Fafournoux P. Amino acids control mammalian gene transcription: activating transcription factor 2 is essential for the amino acid responsiveness of the CHOP promoter. *Mol Cell Biol* 20: 7192–7204, 2000.

23. burGavrilova O, Haluzik M, Matsusue K, Cutson JJ, Johnson L, Dietz KR, Nicol CJ, Vinson C, Gonzalez FJ, and Reitman ML. Liver peroxisome proliferator-activated receptor gamma contributes to hepatic steatosis, triglyceride clearance, and regulation of body fat mass. *J Biol Chem* 278: 34268–34276, 2003.

24. Cagen LM, Deng X, Wilcox HG, Park EA, Raghow R, and Elam MB. Insulin activates the rat sterol-regulatory-element-binding protein 1c (SREBP-1c) promoter through the combinatorial actions of SREBP, LXR, Sp-1 and NF-Y cis-acting elements. *Biochem J* 385: 207–216, 2005.

25. Calza L, Manfredi R, and Chiodo F. Dyslipidaemia associated with antiretroviral therapy in HIV-infected patients. *J Antimicrob Chemother* 53: 10–14, 2004.

26. Camp HS, Chaudhry A, and Leff T. A novel potent antagonist of peroxisome proliferator-activated receptor gamma blocks adipocyte differentiation but does not revert the phenotype of terminally differentiated adipocytes. *Endocrinology* 142: 3207–3213, 2001.

27. Caron M, Auclair M, Vigouroux C, Glorian M, Forest C, and Capeau J. The HIV protease inhibitor indinavir impairs sterol regulatory element-binding protein-1 intranuclear localization, inhibits preadipocyte differentiation, and induces insulin resistance. *Diabetes* 50: 1378–1388, 2001.

28. Cassuto H, Olswang Y, Heinemann S, Sabbagh K, Hanson RW, and Reshef L. The transcriptional regulation of phosphoenolpyruvate carboxykinase gene in the kidney requires the HNF-1 binding site of the gene. *Gene* 318: 177–184, 2003.

29. Cederberg A, Gronning LM, Ahren B, Tasken K, Carlsson P, and Enerback S. FOXC2 is a winged helix gene that counteracts obesity, hypertriglyceridemia, and diet-induced insulin resistance. *Cell* 106: 563–573, 2001.

30. Chambon P. A decade of molecular biology of retinoic acid receptors. *FASEB J* 10: 940–954, 1996.

31. Charron MJ, Katz EB, and Olson AL. GLUT4 gene regulation and manipulation. *J Biol Chem* 274: 3253–3256, 1999.

32. Chawla A, Boisvert WA, Lee CH, Laffitte BA, Barak Y, Joseph SB, Liao D, Nagy L, Edwards PA, Curtiss LK, Evans RM, and Tontonoz P. A PPAR gamma-LXR-ABCA1 pathway in macrophages is involved in cholesterol efflux and atherogenesis. *Mol Cell* 7: 161–171, 2001.

33. Chen G, Liang G, Ou J, Goldstein JL, and Brown MS. Central role for liver X receptor in insulin-mediated activation of Srebp-1c transcription and stimulation of fatty acid synthesis in liver. *Proc Natl Acad Sci USA* 101: 11245–11250, 2004.

34. Chen JY, Levy-Wilson B, Goodart S, and Cooper AD. Mice expressing the human CYP7A1 gene in the mouse CYP7A1 knockout background lack induction of CYP7A1 expression by cholesterol feeding and have increased hypercholesterolemia when fed a high fat diet. *J Biol Chem* 277: 42588–42595, 2002.

35. Cheng L, Ding G, Qin Q, Huang Y, Lewis W, He N, Evans RM, Schneider MD, Brako FA, Xiao Y, Chen YE, and Yang Q. Cardiomyocyte-restricted peroxisome proliferator-activated receptor-delta deletion perturbs myocardial fatty acid oxidation and leads to cardiomyopathy. *Nat Med* 10: 1245–1250, 2004.

36. Cheng L, Ding G, Qin Q, Xiao Y, Woods D, Chen YE, and Yang Q. Peroxisome proliferator-activated receptor delta activates fatty acid oxidation in cultured neonatal and adult cardiomyocytes. *Biochem Biophys Res Commun* 313: 277–286, 2004.

37. Cheng SL, Shao JS, Charlton-Kachigian N, Loewy AP, and Towler DA. MSX2 promotes osteogenesis and suppresses adipogenic differentiation of multipotent mesenchymal progenitors. *J Biol Chem* 278: 45969–45977, 2003.

38. Chevillotte E, Rieuisset J, Roques M, Desage M, and Vidal H. The regulation of uncoupling protein-2 gene expression by omega-6 polyunsaturated fatty acids in human skeletal muscle cells involves multiple pathways, including the nuclear receptor peroxisome proliferator-activated receptor beta. *J Biol Chem* 276: 10853–10860, 2001.

39. Chisholm JW, Hong J, Mills SA, and Lawn RM. The LXR ligand T0901317 induces severe lipogenesis in the db/db diabetic mouse. *J Lipid Res* 44: 2039–2048, 2003.

40. Christoffels VM, Grange T, Kaestner KH, Cole TJ, Darlington GJ, Croniger CM, and Lamers WH. Glucocorticoid receptor, C/EBP, HNF3, and protein kinase A coordinately activate the glu-

cocorticoid response unit of the carbamoylphosphate synthetase I gene. *Mol Cell Biol* 18: 6305–6315, 1998.

41. Christoffels VM, Habets PE, Das AT, Clout DE, van Roon MA, Moorman AF, and Lamers WH. A single regulatory module of the carbamoylphosphate synthetase I gene executes its hepatic program of expression. *J Biol Chem* 275: 40020–40027, 2000.

42. Chung-Bok MI, Vincent N, Jhala U, and Watford M. Rat hepatic glutaminase: identification of the full coding sequence and characterization of a functional promoter. *Biochem J* 324: 193–200, 1997.

43. Claudel T, Leibowitz MD, Fievet C, Tailleux A, Wagner B, Repa JJ, Torpier G, Lobaccaro JM, Paterniti JR, Mangelsdorf DJ, Heyman RA, and Auwerx J. Reduction of atherosclerosis in apolipoprotein E knockout mice by activation of the retinoid X receptor. *Proc Natl Acad Sci USA* 98: 2610–2615, 2001.

44. Croniger CM, Millward C, Yang J, Kawai Y, Arinze IJ, Liu S, Harada-Shiba M, Chakravarty K, Friedman JE, Poli V, and Hanson RW. Mice with a deletion in the gene for CCAAT/enhancer-binding protein beta have an attenuated response to cAMP and impaired carbohydrate metabolism. *J Biol Chem* 276: 629–638, 2001.

45. Crosson SM, Davies GF, and Roesler WJ. Hepatic expression of CCAAT/enhancer binding protein alpha: hormonal and metabolic regulation in rats. *Diabetologia* 40: 1117–1124, 1997.

46. Dalen KT, Ulven SM, Bamberg K, Gustafsson JA, and Nebb HI. Expression of the insulin-responsive glucose transporter GLUT4 in adipocytes is dependent on liver X receptor alpha. *J Biol Chem* 278: 48283–48291, 2003.

47. Damiola F, Le Minh N, Preitner N, Kornmann B, Fleury-Olela F, and Schibler U. Restricted feeding uncouples circadian oscillators in peripheral tissues from the central pacemaker in the suprachiasmatic nucleus. *Genes Dev* 14: 2950–2961, 2000.

48. Daniel PB, Walker WH, and Habener JF. Cyclic AMP signaling and gene regulation. *Annu Rev Nutr* 18: 353–383, 1998.

49. Davis KE, Moldes M, and Farmer SR. The forkhead transcription factor FoxC2 inhibits white adipocyte differentiation. *J Biol Chem* 279: 42453–42461, 2004.

50. DeLuca JG, Doebber TW, Kelly LJ, Kemp RK, Molon-Noblot S, Sahoo SP, Ventre J, Wu MS, Peters JM, Gonzalez FJ, and Moller DE. Evidence for peroxisome proliferator-activated receptor (PPAR)alpha-independent peroxisome proliferation: effects of PPARgamma/delta-specific agonists in PPARalpha-null mice. *Mol Pharmacol* 58: 470–476, 2000.

51. De Luis O, Valero MC, and Jurado LA. WBSCR14, a putative transcription factor gene deleted in Williams-Beuren syndrome: complete characterisation of the human gene and the mouse ortholog. *Eur J Hum Genet* 8: 215–222, 2000.

52. Dentin R, Girard J, and Postic C. Carbohydrate responsive element binding protein (ChREBP) and sterol regulatory element binding protein-1c (SREBP-1c): two key regulators of glucose metabolism and lipid synthesis in liver. *Biochimie* 87: 81–86, 2005.

53. Dentin R, Pegorier JP, Benhamed F, Foufelle F, Ferre P, Fauveau V, Magnuson MA, Girard J, and Postic C. Hepatic glucokinase is required for the synergistic action of ChREBP and SREBP-1c on glycolytic and lipogenic gene expression. *J Biol Chem* 279: 20314–20326, 2004.

54. Desvergne B and Wahli W. Peroxisome proliferator-activated receptors: nuclear control of metabolism. *Endocrine Rev* 20: 649–688, 1999.

55. De Urquiza AM, Liu S, Sjoberg M, Zetterstrom RH, Griffiths W, Sjovall J, and Perlmann T. Docosahexaenoic acid, a ligand for the retinoid X receptor in mouse brain. *Science* 290: 2140–2144, 2000.

56. De Vos A, Heimberg H, Quartier E, Huypens P, Bouwens L, Pipeleers D, and Schuit F. Human and rat beta cells differ in glucose transporter but not in glucokinase gene expression. *J Clin Invest* 96: 2489–2495, 1995.

57. Diaz Guerra MJ, Bergot MO, Martinez A, Cuif MH, Kahn A, and Raymondjean M. Functional characterization of the L-type pyruvate kinase gene glucose response complex. *Mol Cell Biol* 13: 7725–7733, 1993.

58. Dressel U, Allen TL, Pippal JB, Rohde PR, Lau P, and Muscat GE. The peroxisome proliferator-activated receptor beta/delta agonist, GW501516, regulates the expression of genes involved in

lipid catabolism and energy uncoupling in skeletal muscle cells. *Mol Endocrinol* 17: 2477–2493, 2003.

59. Duan LP, Wang HH, and Wang DQ. Cholesterol absorption is mainly regulated by the jejunal and ileal ATP-binding cassette sterol efflux transporters Abcg5 and Abcg8 in mice. *J Lipid Res* 45: 1312–1323, 2004.

60. Duvic M, Martin AG, Kim Y, Olsen E, Wood GS, Crowley CA, and Yocum RC. Phase 2 and 3 clinical trial of oral bexarotene (Targretin capsules) for the treatment of refractory or persistent early-stage cutaneous T-cell lymphoma. *Arch Dermatol* 137: 581–593, 2001.

61. Edvardsson U, Bergstrom M, Alexandersson M, Bamberg K, Ljung B, and Dahllof B. Rosiglitazone (BRL49653), a PPAR-gamma-selective agonist, causes peroxisome proliferator-like liver effects in obese mice. *J Lipid Res* 40: 1177–1184, 1999.

62. Edwards PA, Kast HR, and Anisfeld AM. BAREing it all: the adoption of LXR and FXR and their roles in lipid homeostasis. *J Lipid Res* 43: 2–12, 2002.

63. Edwards PA, Tabor D, Kast HR, and Venkateswaran A. Regulation of gene expression by SREBP and SCAP. *Biochim Biophys Acta* 1529: 103–113, 2000.

64. Ekberg K, Landau BR, Wajngot A, Chandramouli V, Efendic S, Brunengraber H, and Wahren J. Contributions by kidney and liver to glucose production in the postabsorptive state and after 60 h of fasting. *Diabetes* 48: 292–298, 1999.

65. Engelking LJ, Kuriyama H, Hammer RE, Horton JD, Brown MS, Goldstein JL, and Liang G. Overexpression of Insig-1 in the livers of transgenic mice inhibits SREBP processing and reduces insulin-stimulated lipogenesis. *J Clin Invest* 113: 1168–1175, 2004.

66. Escher P, Braissant O, Basu-Modak S, Michalik L, Wahli W, and Desvergne B. Rat PPARs: quantitative analysis in adult rat tissues and regulation in fasting and refeeding. *Endocrinology* 142: 4195–4202, 2001.

67. Eto K, Yamashita T, Matsui J, Terauchi Y, Noda M, and Kadowaki T. Genetic manipulations of fatty acid metabolism in beta-cells are associated with dysregulated insulin secretion. *Diabetes Suppl* 3: S414–S420, 2002.

68. Evans MJ and Scarpulla RC. NRF-1: a trans-activator of nuclear-encoded respiratory genes in animal cells. *Genes Dev* 4: 1023–1034, 1990.

69. Fajas L, Schoonjans K, Gelman L, Kim JB, Najib J, Martin G, Fruchart JC, Briggs M, Spiegelman BM, and Auwerx J. Regulation of peroxisome proliferator-activated receptor gamma expression by adipocyte differentiation and determination factor 1/sterol regulatory element binding protein 1: implications for adipocyte differentiation and metabolism. *Mol Cell Biol* 19: 5495–5503, 1999.

70. Faul MM and Grese TA. Selective RXR modulators for the treatment of type II diabetes. *Curr Opin Drug Discov Dev* 5: 974–985, 2002.

71. Fingar DC, Salama S, Tsou C, Harlow E, and Blenis J. Mammalian cell size is controlled by mTOR and its downstream targets S6K1 and 4EBP1/eIF4E. *Genes Dev* 16: 1472–1487, 2002.

72. Foretz M, Ancellin N, Andreelli F, Saintillan Y, Grondin P, Kahn A, Thorens B, Vaulont S, and Viollet B. Short-term overexpression of a constitutively active form of AMP-activated protein kinase in the liver leads to mild hypoglycemia and fatty liver. *Diabetes* 54: 1331–1339, 2005.

73. Foretz M, Guichard C, Ferre P, and Foufelle F. Sterol regulatory element binding protein-1c is a major mediator of insulin action on the hepatic expression of glucokinase and lipogenesis-related genes. *Proc Natl Acad Sci USA* 96: 12737–12742, 1999.

75. Foretz M, Pacot C, Dugail I, Lemarchand P, Guichard C, Le Liepvre X, Berthelier-Lubrano C, Spiegelman B, Kim JB, Ferre P, and Foufelle F. ADD1/SREBP-1c is required in the activation of hepatic lipogenic gene expression by glucose. *Mol Cell Biol* 19: 3760–3768, 1999.

76. Francis GA, Annicotte JS, and Auwerx J. Liver X receptors: Xcreting Xol to combat atherosclerosis. *Trends Mol Med* 8: 455–458, 2002.

77. Francis GA, Fayard E, Picard F, and Auwerx J. Nuclear receptors and the control of metabolism. *Annu Rev Physiol* 65: 261–311, 2003.

78. Frayn KN, Karpe F, Fielding BA, MacDonald IA, and Coppack SW. Integrative physiology of human adipose tissue. *Int J Obes Relat Metab Disord* 27: 875–888, 2003.

79. Frühbeck G, Gomez-Ambrosi J, Muruzabal FJ, and Burrell MA. The adipocyte: a model for integration of endocrine and metabolic signaling in energy metabolism regulation. *Am J Physiol Endocrinol Metab* 280: E827–E847, 2001.

80. Garofalo RS, Orena SJ, Rafidi K, Torchia AJ, Stock JL, Hildebrandt AL, Coskran T, Black SC, Brees DJ, Wicks JR, McNeish JD, and Coleman KG. Severe diabetes, age-dependent loss of adipose tissue, and mild growth deficiency in mice lacking Akt2/PKB beta. *J Clin Invest* 112: 197–208, 2003.

81. Gautier-Stein A, Mithieux G, and Rajas F. A distal region involving hepatocyte nuclear factor 4alpha and CAAT/enhancer binding protein markedly potentiates the protein kinase A stimulation of the glucose-6-phosphatase promoter. *Mol Endocrinol* 19: 163–174, 2005.

82. Gavriloa O, Marcus-Samuels B, Graham D, Kim JK, Shulman GI, Castle AL, Vinson C, Eckhaus M, and Reitman ML. Surgical implantation of adipose tissue reverses diabetes in lipoatrophic mice. *J Clin Invest* 105: 271–278, 2000.

83. Gerrish K, Van Velkinburgh JC, and Stein R. Conserved transcriptional regulatory domains of the pdx-1 gene. *Mol Endocrinol* 18: 533–548, 2004.

84. Gimble JM, Robinson CEE, Wu X, Kelly KA, Rodriguez BR, Kliewer SA, Lehmann JM, and Morris DC. Peroxisome proliferator-activated receptor γ activation by thiazolidinediones induces adipogenesis in bone marrow stromal cells. *Mol Pharmacol* 50: 1087–1094, 1996.

85. Glass CK and Rosenfeld MG. The coregulator exchange in transcriptional functions of nuclear receptors. *Genes Dev* 14: 121–141, 2000.

86. Goodwin B, Jones SA, Price RR, Watson MA, McKee DD, Moore LB, Galardi C, Wilson JG, Lewis MC, Roth ME, Maloney PR, Willson TM, and Kliewer SA. A regulatory cascade of the nuclear receptors FXR, SHP-1, and LRH-1 represses bile acid biosynthesis. *Mol Cell* 6: 517–526, 2000.

87. Goodwin B, Watson MA, Kim H, Miao J, Kemper JK, and Kliewer SA. Differential regulation of rat and human CYP7A1 by the nuclear oxysterol receptor liver X receptor-alpha. *Mol Endocrinol* 17: 386–394, 2003.

88. Gosmain Y, Dif N, Berbe V, Loizon E, Rieusset J, Vidal H, and Lefai E. Regulation of SREBP-1 expression and transcriptional action on HKII and FAS genes during fasting and refeeding in rat tissues. *J Lipid Res* 46: 697–705, 2005.

89. Gotoh T, Chowdhury S, Takiguchi M, and Mori M. The glucocorticoid-responsive gene cascade. Activation of the rat arginase gene through induction of C/EBPbeta. *J Biol Chem* 272: 3694–3698, 1997.

90. Grefhorst A, Elzinga BM, Voshol PJ, Plosch T, Kok T, Bloks VW, van der Sluijs FH, Havekes LM, Romijn JA, Verkade HJ, and Kuipers F. Stimulation of lipogenesis by pharmacological activation of the liver X receptor leads to production of large, triglyceride-rich very low density lipoprotein particles. *J Biol Chem* 277: 34182–34190, 2002.

91. Grober J, Zaghlini I, Fujii H, Jones SA, Kliewer SA, Willson TM, Ono T, and Besnard P. Identification of a bile acid-responsive element in the human ileal bile acid-binding protein gene. Involvement of the farnesoid X receptor/9-cis-retinoic acid receptor heterodimer. *J Biol Chem* 274: 29749–29754, 1999.

92. Group DS. Glucose tolerance and mortality: comparison of WHO and American Diabetes Association diagnostic criteria. The DECODE study group European Diabetes Epidemiology Group Diabetes Epidemiology: collaborative analysis of diagnostic criteria in Europe. *Lancet* 354: 617–621, 1999.

93. Guerre-Millo M, Gervois P, Raspe E, Madsen L, Poulain P, Derudas B, Herbert JM, Winegar DA, Willson TM, Fruchart JC, Berge RK, and Staels B. Peroxisome proliferator-activated receptor alpha activators improve insulin sensitivity and reduce adiposity. *J Biol Chem* 275: 16638–16642, 2000.

94. Guerre-Millo M, Rouault C, Poulain P, Andre J, Poitout V, Peters JM, Gonzalez FJ, Fruchart JC, Reach G, and Staels B.

PPAR-alpha-null mice are protected from high-fat diet-induced insulin resistance. *Diabetes* 50: 2809–2814, 2001.

95. Gurnell M, Savage DB, Chatterjee VK, and O’Rahilly S. The metabolic syndrome: peroxisome proliferator-activated receptor gamma and its therapeutic modulation. *J Clin Endocrinol Metab* 88: 2412–2421, 2003.

96. Hadden TJ, Ryou C, and Miller RE. Elements in the distal 5′-flanking sequence and the first intron function cooperatively to regulate glutamine synthetase transcription during adipocyte differentiation. *Nucleic Acids Res* 25: 3930–3936, 1997.

97. Hai T and Hartman MG. The molecular biology and nomenclature of the activating transcription factor/cAMP responsive element binding family of transcription factors: activating transcription factor proteins and homeostasis. *Gene* 273: 1–11, 2001.

98. Hallakou S, Doare L, Foufelle F, Kergoat M, Guerremillo M, Berthault MF, Dugail I, Morin J, Auwerx J, and Ferre P. Pioglitazone induces in vivo adipocyte differentiation in the obese Zucker Fa/Fa rat. *Diabetes* 46: 1393–1399, 1997.

99. Hansen JB, Jorgensen C, Petersen RK, Hallenborg P, De Matteis R, Boye HA, Petrovic N, Enerback S, Nedergaard J, Cinti S, te Riele H, and Kristiansen K. Retinoblastoma protein functions as a molecular switch determining white versus brown adipocyte differentiation. *Proc Natl Acad Sci USA* 101: 4112–4117, 2004.

100. Harding HP, Novoa I, Zhang Y, Zeng H, Wek R, Schapira M, and Ron D. Regulated translation initiation controls stress-induced gene expression in mammalian cells. *Mol Cell* 6: 1099–1108, 2000.

101. Harp JB. New insights into inhibitors of adipogenesis. *Curr Opin Lipidol* 15: 303–307, 2004.

102. Harris RA, Bowker-Kinley MM, Huang B, and Wu P. Regulation of the activity of the pyruvate dehydrogenase complex. *Adv Enzyme Regul* 42: 249–259, 2002.

103. He W, Barak Y, Hevener A, Olson P, Liao D, Le J, Nelson M, Ong E, Olefsky JM, and Evans RM. Adipose-specific peroxisome proliferator-activated receptor γ knockout causes insulin resistance in fat and liver but not in muscle. *Proc Natl Acad Sci USA* 100: 15712–15717, 2003.

104. Hegarty BD, Bobard A, Hainault I, Ferre P, Bossard P, and Foufelle F. Distinct roles of insulin and liver X receptor in the induction and cleavage of sterol regulatory element-binding protein-1c. *Proc Natl Acad Sci USA* 102: 791–796, 2005.

105. Hertz R, Magenheim J, Berman I, and Bar-Tana J. Fatty acyl-CoA thioesters are ligands of hepatic nuclear factor-4-alpha. *Nature* 392: 512–516, 1998.

106. Herzig S, Long F, Jhala US, Hedrick S, Quinn R, Bauer A, Rudolph D, Schutz G, Yoon C, Puigserver P, Spiegelman B, and Montminy M. CREB regulates hepatic gluconeogenesis through the coactivator PGC-1. *Nature* 413: 179–183, 2001.

107. Hevener AL, He W, Barak Y, Le J, Bandyopadhyay G, Olson P, Wilkes J, Evans RM, and Olefsky J. Muscle-specific PPARγ deletion causes insulin resistance. *Nat Med* 9: 1491–1497, 2003.

108. Hirota K, Daitoku H, Matsuzaki H, Araya N, Yamagata K, Asada S, Sugaya T, and Fukamizu A. Hepatocyte nuclear factor-4 is a novel downstream target of insulin via FKHR as a signal-regulated transcriptional inhibitor. *J Biol Chem* 278: 13056–13060, 2003.

109. Holm C. Molecular mechanisms regulating hormone-sensitive lipase and lipolysis. *Biochem Soc Trans* 31: 1120–1124, 2003.

110. Holst D, Luquet S, Kristiansen K, and Grimaldi PA. Roles of peroxisome proliferator-activated receptors delta and gamma in myoblast transdifferentiation. *Exp Cell Res* 288: 168–176, 2003.

111. Holst D, Luquet S, Nogueira V, Kristiansen K, Leverve X, and Grimaldi PA. Nutritional regulation and role of peroxisome proliferator-activated receptor delta in fatty acid catabolism in skeletal muscle. *Biochim Biophys Acta* 1633: 43–50, 2003.

112. Hornbuckle LA, Everett CA, Martin CC, Gustavson SS, Svitek CA, Oeser JK, Neal DW, Cherrington AD, and O’Brien RM. Selective stimulation of G-6-Pase catalytic subunit but not G-6-P transporter gene expression by glucagon in vivo and cAMP in situ. *Am J Physiol Endocrinol Metab* 286: E795–E808, 2004.

113. Horowitz JF, Leone TC, Feng W, Kelly DP, and Klein S. Effect of endurance training on lipid metabolism in women: a potential

role for PPARalpha in the metabolic response to training. *Am J Physiol Endocrinol Metab* 279: E348–E355, 2000.

114. Horton JD, Goldstein JL, and Brown MS. SREBPs: activators of the complete program of cholesterol and fatty acid synthesis in the liver. *J Clin Invest* 109: 1125–1131, 2002.

115. Horton JD, Shah NA, Warrington JA, Anderson NN, Park SW, Brown MS, and Goldstein JL. Combined analysis of oligonucleotide microarray data from transgenic and knockout mice identifies direct SREBP target genes. *Proc Natl Acad Sci USA* 100: 12027–12032, 2003.

116. Hua X, Yokoyama C, Wu J, Briggs MR, Brown MS, Goldstein JL, and Wang X. SREBP-2, a second basic-helix-loop-helix-leucine zipper protein that stimulates transcription by binding to a sterol regulatory element. *Proc Natl Acad Sci USA* 90: 11603–11607, 1993.

117. Huang B, Wu P, Bowker-Kinley MM, and Harris RA. Regulation of pyruvate dehydrogenase kinase expression by peroxisome proliferator-activated receptor-alpha ligands, glucocorticoids, and insulin. *Diabetes* 51: 276–283, 2002.

118. Izuka K, Bruick RK, Liang G, Horton JD, and Uyeda K. Deficiency of carbohydrate response element-binding protein (ChREBP) reduces lipogenesis as well as glycolysis. *Proc Natl Acad Sci USA* 101: 7281–7286, 2004.

119. IJpenberg A, Tan NS, Gelman L, Kersten S, Seydoux J, Xu J, Canaple L, Metzger D, Chambon P, Wahli W, and Desvergne B. In vivo RXR homodimers signaling via PPREs. *EMBO J.* 23: 2083–2091, 2004.

120. Imai T, Jiang M, Chambon P, and Metzger D. Impaired adipogenesis and lipolysis in the mouse upon selective ablation of the retinoid X receptor alpha mediated by a tamoxifen-inducible chimeric Cre recombinase (Cre-ERT2) in adipocytes. *Proc Natl Acad Sci USA* 98: 224–228, 2001.

121. Imai T, Takawuka R, Marchand S, Dentz E, Bornert JM, Messadedeq N, Wendling O, Mark M, Desvergne B, Wahli W, Chambon P, and Metzger D. PPAR gamma is required in mature white and brown adipocytes for their survival in the mouse. *Proc Natl Acad Sci USA* 101: 4543–4547, 2004.

122. Inoue Y, Hayhurst GP, Inoue J, Mori M, and Gonzalez FJ. Defective ureagenesis in mice carrying a liver-specific disruption of hepatocyte nuclear factor 4alpha (HNF4alpha). HNF4alpha regulates ornithine transcarbamylase in vivo. *J Biol Chem* 277: 25257–25265, 2002.

123. Ishii S, Izuka K, Miller BC, and Uyeda K. Carbohydrate response element binding protein directly promotes lipogenic enzyme gene transcription. *Proc Natl Acad Sci USA* 101: 15597–15602, 2004.

124. Iynedjian PB, Roth RA, Fleischmann M, and Gjinovci A. Activation of protein kinase B/cAkt in hepatocytes is sufficient for the induction of expression of the gene encoding glucokinase. *Biochem J* 351: 621–627, 2000.

125. Jacinto E and Hall MN. Tor signalling in bugs, brain and brawn. *Nat Rev Mol Cell Biol* 4: 117–126, 2003.

126. Jackson-Hayes L, Song S, Lavrentyev EN, Jansen MS, Hillgartner FB, Tian L, Wood PA, Cook GA, and Park EA. A thyroid hormone response unit formed between the promoter and first intron of the carnitine palmitoyltransferase-Ialpha gene mediates the liver-specific induction by thyroid hormone. *J Biol Chem* 278: 7964–7972, 2003.

127. Janowski BA, Willy PJ, Devi TR, and Mangelsdorf DJ. An oxysterol signalling pathway mediated by the nuclear receptor LXRa. *Nature* 383: 728–731, 1996.

128. Jansen MS, Cook GA, Song S, and Park EA. Thyroid hormone regulates carnitine palmitoyltransferase Ialpha gene expression through elements in the promoter and first intron. *J Biol Chem* 275: 34989–34997, 2000.

129. Jiang ZY, Zhou QL, Coleman KA, Chouinard M, Boese Q, and Czech MP. Insulin signaling through Akt/protein kinase B analyzed by small interfering RNA-mediated gene silencing. *Proc Natl Acad Sci USA* 100: 7569–7574, 2003.

130. Jones JR, Barrick C, Kim KA, Lindner J, Blondeau B, Fujimoto Y, Shiota M, Kesterson RA, Kahn BB, and Magnuson MA. Deletion of PPARgamma in adipose tissues of mice protects against high fat diet-induced obesity and insulin resistance. *Proc Natl Acad Sci USA* 102: 6207–6212, 2005.

131. Jorgensen P, Rupes I, Sharom JR, Schneper L, Broach JR, and Tyers M. A dynamic transcriptional network communicates growth potential to ribosome synthesis and critical cell size. *Genes Dev* 18: 2491–2505, 2004.

132. Joseph SB, Laffitte BA, Patel PH, Watson MA, Matsukuma KE, Walczak R, Collins JL, Osborne TF, and Tontonoz P. Direct and indirect mechanisms for regulation of fatty acid synthase gene expression by liver X receptors. *J Biol Chem* 277: 11019–11025, 2002.

133. Joseph SB, McKilligin E, Pei L, Watson MA, Collins AR, Laf fitte BA, Chen M, Noh G, Goodman J, Hagger GN, Tran J, Tippin TK, Wang X, Lusis AJ, Hsueh WA, Law RE, Collins JL, Willson TM, and Tontonoz P. Synthetic LXR ligand inhibits the development of atherosclerosis in mice. *Proc Natl Acad Sci USA* 99: 7604–7609, 2002.

134. Junger MA, Rintelen F, Stocker H, Wasserman JD, Vegh M, Radimerski T, Greenberg ME, and Hafen E. The *Drosophila* Forkhead transcription factor FOXO mediates the reduction in cell number associated with reduced insulin signaling. *J Biol Chem* 2: 20, 2003.

135. Juvet LK, Andresen SM, Schuster GU, Dalen KT, Tobin KA, Hollung K, Haugen F, Jacinto S, Ulven SM, Bamberg K, Gustafsson JA, and Nebb HI. On the role of liver X receptors in lipid accumulation in adipocytes. *Mol Endocrinol* 17: 172–182, 2003.

136. Kabashima T, Kawaguchi T, Wadzinski BE, and Uyeda K. Xylulose 5-phosphate mediates glucose-induced lipogenesis by xy lulose 5-phosphate-activated protein phosphatase in rat liver. *Proc Natl Acad Sci USA* 100: 5107–5112, 2003.

137. Kahn CR, Chen L, and Cohen SE. Unraveling the mechanism of thiazolidinediones. *J Clin Invest* 106: 1305–1307, 2000.

138. Kast HR, Nguyen CM, Sinal CJ, Jones SA, Laffitte BA, Reue K, Gonzalez FJ, Willson TM, and Edwards PA. Farnesoid X-activated receptor induces apolipoprotein C-II transcription: a molecular mechanism linking plasma triglyceride levels to bile acids. *Mol Endocrinol* 15: 1720–1728, 2001.

139. Kawaguchi T, Osatomi K, Yamashita H, Kabashima T, and Uyeda K. Mechanism for fatty acid “sparing” effect on glucose-induced transcription: regulation of carbohydrate-responsive element-binding protein by AMP-activated protein kinase. *J Biol Chem* 277: 3829–3835, 2002.

140. Kawaguchi T, Takenoshita M, Kabashima T, and Uyeda K. Glucose and cAMP regulate the L-type pyruvate kinase gene by phosphorylation/dephosphorylation of the carbohydrate response element binding protein. *Proc Natl Acad Sci USA* 98: 13710–13715, 2001.

141. Kersten S, Mandard S, Escher P, Gonzalez FJ, Tafuri S, Desvergne B, and Wahli W. The peroxisome proliferator-activated receptor alpha regulates amino acid metabolism. *FASEB J* 15: 1971–1978, 2001.

142. Kersten S, Seydoux J, Peters JM, Gonzalez FJ, Desvergne B, and Wahli W. Peroxisome proliferator-activated receptor α mediates the adaptive response to fasting. *J Clin Invest* 103: 1489–1498, 1999.

143. Kim JB, Spotts GD, Halvorsen YD, Shih HM, Ellenberger T, Towle HC, and Spiegelman BM. Dual DNA binding specificity of ADD1/SREBP1 controlled by a single amino acid in the basic helix-loop-helix domain. *Mol Cell Biol* 15: 2582–2588, 1995.

144. Kim JK, Fillmore JJ, Sunshine MJ, Albrecht B, Higashimori T, Kim DW, Liu ZX, Soos TJ, Cline GW, O’Brien WR, Littman DR, and Shulman GI. PKC-theta knockout mice are protected from fat-induced insulin resistance. *J Clin Invest* 114: 823–827, 2004.

145. Kim SY, Kim HI, Kim TH, Im SS, Park SK, Lee IK, Kim KS, and Ahn YH. SREBP-1c mediates the insulin-dependent hepatic glu-cokinase expression. *J Biol Chem* 279: 30823–30829, 2004.

146. Kimura T, Chowdhury S, Tanaka T, Shimizu A, Iwase K, Oyadomari S, Gotoh T, Matsuzaki H, Mori M, Akira S, and Takiguchi M. CCAAT/enhancer-binding protein beta is required for activation of genes for ornithine cycle enzymes by glucocorticoids and glucagon in primary-cultured hepatocytes. *FEBS Lett* 494: 105–111, 2001.

147. Kimura T, Christoffels VM, Chowdhury S, Iwase K, Matsuzaki H, Mori M, Lamers WH, Darlington GJ, and Takiguchi M. Hypoglycemia-associated hyperammonemia caused by impaired expression of ornithine cycle enzyme genes in C/EBPalpha knock-out mice. *J Biol Chem* 273: 27505–27510, 1998.

148. Kintscher U and Law RE. PPARgamma-mediated insulin sensitization: the importance of fat versus muscle. *Am J Physiol Endocrinol Metab* 288: E287–E291, 2005.

149. Kitamura T, Nakae J, Kitamura Y, Kido Y, Biggs WH III, Wright CV, White MF, Arden KC, and Accili D. The forkhead transcription factor Foxo1 links insulin signaling to Pdx1 regulation of pancreatic beta cell growth. *J Clin Invest* 110: 1839–1847, 2002.

150. Koh EH, Kim MS, Park JY, Kim HS, Youn JY, Park HS, Youn JH, and Lee KU. Peroxisome proliferator-activated receptor (PPAR)-alpha activation prevents diabetes in OLETF rats: comparison with PPAR-gamma activation. *Diabetes* 52: 2331–2337, 2003.

151. Kojima H, Fujimiya M, Matsumura K, Nakahara T, Hara M, and Chan L. Extrapancreatic insulin-producing cells in multiple organs in diabetes. *Proc Natl Acad Sci USA* 101: 2458–2463, 2004.

152. Koo SH, Dutcher AK, and Towle HC. Glucose and insulin function through two distinct transcription factors to stimulate expression of lipogenic enzyme genes in liver. *J Biol Chem* 276: 9437–9445, 2001.

153. Kramer W, Girbig F, Corsiero D, Burger K, Fahrenholz F, Jung C, and Muller G. Intestinal cholesterol absorption: identification of different binding proteins for cholesterol and cholesterol absorption inhibitors in the enterocyte brush border membrane. *Biochim Biophys Acta* 1633: 13–26, 2003.

154. Kubota N, Terauchi Y, Miki H, Tamemoto H, Yamauchi T, Komeda K, Satoh S, Nakano R, Ishii C, Sugiyama T, Eto K, Tsubamoto Y, Okuno A, Murakami K, Sekihara H, Hasegawa G, Naito M, Toyoshima Y, Tanaka S, Shiota K, Kitamura T, Fujita T, Ezaki O, Aizawa S, Nagai R, Tobe K, Kimura S, and Kadowaki T. PPAR gamma mediates high-fat diet-induced adipocyte hypertrophy and insulin resistance. *Mol Cell* 4: 597–609, 1999.

155. Laaksonen DE, Lakka HM, Niskanen LK, Kaplan GA, Salonen JT, and Lakka TA. Metabolic syndrome and development of diabetes mellitus: application and validation of recently suggested definitions of the metabolic syndrome in a prospective cohort study. *Am J Epidemiol* 156: 1070–1077, 2002.

156. Labow BI, Souba WW, and Abcouwer SF. Mechanisms governing the expression of the enzymes of glutamine metabolism—glutaminase and glutamine synthetase. *J Nutr* 131: 2467S–2474S, 2001.

157. Laffitte BA, Chao LC, Li J, Walczak R, Hummasti S, Joseph SB, Castrillo A, Wilpitz DC, Mangelsdorf DJ, Collins JL, Saez E, and Tontonoz P. Activation of liver X receptor improves glucose tolerance through coordinate regulation of glucose metabolism in liver and adipose tissue. *Proc Natl Acad Sci USA* 100: 5419–5424, 2003.

158. Laffitte BA, Joseph SB, Walczak R, Pei L, Wilpitz DC, Collins JL, and Tontonoz P. Autoregulation of the human liver X receptor alpha promoter. *Mol Cell Biol* 21: 7558–7568, 2001.

159. Larsen TM, Toubro S, and Astrup A. PPARgamma agonists in the treatment of type II diabetes: is increased fatness commensurate with long-term efficacy? *Int J Obes Relat Metab Disord* 27: 147–161, 2003.

160. Laterza OF and Curthoys NP. Effect of acidosis on the properties of the glutaminase mRNA pH-response element binding protein. *J Am Soc Nephrol* 11: 1583–1588, 2000.

161. Lee CH and Evans RM. Peroxisome proliferator-activated receptor-gamma in macrophage lipid homeostasis. *Trends Endocrinol Metab* 13: 331–335, 2002.

162. Lee SS, Pineau T, Drago J, Lee EJ, Owens JW, Kroetz DL, Fernandez-Salguero PM, Westphal H, and Gonzalez FJ. Targeted disruption of the α isoform of the peroxisome proliferator-activated receptor gene in mice results in abolition of the pleiotropic effects of peroxisome proliferators. *Mol Cell Biol* 15: 3012–3022, 1995.

163. Lee Y, Yu X, Gonzales F, Mangelsdorf DJ, Wang MY, Richardson C, Witters LA, and Unger RH. PPAR alpha is necessary for the lipopenic action of hyperleptinemia on white adipose and liver tissue. *Proc Natl Acad Sci USA* 99: 11848–11853, 2002.

164. Lee YH, Sauer B, Johnson PF, and Gonzalez FJ. Disruption of the c/ebp alpha gene in adult mouse liver. *Mol Cell Biol* 17: 6014–6022, 1997.

165. Lehman JJ, Barger PM, Kovacs A, Saffitz JE, Medeiros DM, and Kelly DP. Peroxisome proliferator-activated receptor gamma coactivator-1 promotes cardiac mitochondrial biogenesis. *J Clin Invest* 106: 847–856, 2000.

166. Lehman JJ and Kelly DP. Gene regulatory mechanisms governing energy metabolism during cardiac hypertrophic growth. *Heart Fail Rev* 7: 175–185, 2002.

167. Lehmann JM, Kliewer SA, Moore LB, Smith-Oliver TA, Oliver BB, Su JL, Sundseth SS, Winegar DA, Blanchard DE, Spencer TA, and Willson TM. Activation of the nuclear receptor LXR by oxysterols defines a new hormone response pathway. *J Biol Chem* 272: 3137–3140, 1997.

168. Leibowitz MD, Fievet C, Hennuyer N, Peinado-Onsurbe J, Duez H, Bergera J, Cullinan CA, Sparrow CP, Baffic J, Berger GD, Santini C, Marquis RW, Tolman RL, Smith RG, Moller DE, and Auwerx J. Activation of PPARdelta alters lipid metabolism in db/db mice. *FEBS Lett* 473: 333–336, 2000.

169a. Le Lay S, Lefrere I, Trautwein C, Dugail I, and Krief S. Insulin and sterol-regulatory element-binding protein-1c (SREBP-1C) regulation of gene expression in 3T3-L1 adipocytes. Identification of CCAAT/enhancer-binding protein beta as an SREBP-1C target. *J Biol Chem* 277: 35625–35634, 2002.

170. Leone T, Weinheimer C, and Kelly D. A critical role for the peroxisome proliferator-activated receptor α (PPARα) in the cellular fasting response: the PPARα-null mouse as a model of fatty acid oxidation disorders. *Proc Natl Acad Sci USA* 96: 7473–7478, 1999.

171. Lewis GF, Carpentier A, Adeli K, and Giacca A. Disordered fat storage and mobilization in the pathogenesis of insulin resistance and type 2 diabetes. *Endocr Rev* 23: 201–229, 2002.

172. Liang G, Yang J, Horton JD, Hammer RE, Goldstein JL, and Brown MS. Diminished hepatic response to fasting/refeeding and liver X receptor agonists in mice with selective deficiency of sterol regulatory element-binding protein-1c. *J Biol Chem* 277: 9520–9528, 2002.

173. Lin FT and Lane MD. Antisense CCAAT/enhancer-binding protein RNA suppresses coordinate gene expression and triglyceride accumulation during differentiation of 3T3-L1 preadipocytes. *Genes Dev* 6: 533–544, 1992.

174. Liu S, Ogilvie KM, Klausing K, Lawson MA, Jolley D, Li D, Bilakovics J, Pascual B, Hein N, Urcan M, and Leibowitz MD. Mechanism of selective retinoid X receptor agonist-induced hypothyroidism in the rat. *Endocrinology* 143: 2880–2885, 2002.

175. Liu Z, Thompson KS, and Towle HC. Carbohydrate regulation of the rat L-type pyruvate kinase gene requires two nuclear factors: LF-A1 and a member of the c-myc family. *J Biol Chem* 268: 12787–12795, 1993.

176. Lizcano JM and Alessi DR. The insulin signalling pathway. *Curr Biol* 12: R236–R238, 2002.

177. Long X, Lin Y, Ortiz-Vega S, Yonezawa K, and Avruch J. Rheb binds and regulates the mTOR kinase. *Curr Biol* 15: 702–713, 2005.

178. Louet JF, Chatelain F, Decaux JF, Park EA, Kohl C, Pineau T, Girard J, and Pegorier JP. Long-chain fatty acids regulate liver carnitine palmitoyltransferase I gene (L-CPT I) expression through a peroxisome-proliferator-activated receptor alpha (PPARalpha)-independent pathway. *Biochem J* 354: 189–197, 2001.

179. Louet JF, Hayhurst G, Gonzalez FJ, Girard J, and Decaux JF. The coactivator PGC-1 is involved in the regulation of the liver carnitine palmitoyltransferase I gene expression by cAMP in combination with HNF4 alpha and cAMP-response element-binding protein (CREB). *J Biol Chem* 277: 37991–38000, 2002.

180. Louet JF, Le May C, Pegorier JP, Decaux JF, and Girard J. Regulation of liver carnitine palmitoyltransferase I gene expression by hormones and fatty acids. *Biochem Soc Trans* 29: 310–316, 2001.

181. Lu TT, Makishima M, Repa JJ, Schoonjans K, Kerr TA, Auwerx J, and Mangelsdorf DJ. Molecular basis for feedback regulation of bile acid synthesis by nuclear receptors. *Mol Cell* 6: 507–515, 2000.

182. Luquet S, Lopez-Soriano J, Holst D, Fredenrich A, Melki J, Rassoulzadegan M, and Grimaldi PA. Peroxisome proliferator-activated receptor delta controls muscle development and oxidative capability. *FASEB J* 17: 2299–2301, 2003.
183. Ma L, Tsatsos NG, and Towle HC. Direct role of ChREBP.Mlx in regulating hepatic glucose-responsive genes. *J Biol Chem* 280: 12019–12027, 2005.
184. MacDougald OA, Cornelius P, Lin FT, Chen SS, and Lane MD. Glucocorticoids reciprocally regulate expression of the CCAAT/enhancer-binding protein alpha and delta genes in 3T3-L1 adipocytes and white adipose tissue. *J Biol Chem* 269: 19041–19047, 1994.
185. MacFarlane WM, Read ML, Gilligan M, Bujalska I, and Docherty K. Glucose modulates the binding activity of the beta-cell transcription factor IUF1 in a phosphorylation-dependent manner. *Biochem J* 303: 625–631, 1994.
186. Mangelsdorf DJ and Evans RM. The RXR heterodimers and orphan receptors. *Cell* 83: 841–850, 1995.
187. Marchesini G, Marzocchi R, Agostini F, and Bugianesi E. Nonalcoholic fatty liver disease and the metabolic syndrome. *Curr Opin Lipidol* 16: 421–427, 2005.
188. Mark M, Ghyselinck NB, Wendling O, Dupe V, Mascrez B, Kastner P, and Chambon P. A genetic dissection of the retinoid signalling pathway in the mouse. *Proc Nutr Soc* 58: 609–613, 1999.
189. Marshak S, Totary H, Cerasi E, and Melloul D. Purification of the beta-cell glucose-sensitive factor that transactivates the insulin gene differentially in normal and transformed islet cells. *Proc Natl Acad Sci USA* 93: 15057–15062, 1996.
190. Martin DE, Soulard A, and Hall MN. TOR regulates ribosomal protein gene expression via PKA and the Forkhead transcription factor FHL1. *Cell* 119: 969–979, 2004.
191. Matsuda M, Korn BS, Hammer RE, Moon YA, Komuro R, Horton JD, Goldstein JL, Brown MS, and Shimomura I. SREBP cleavage-activating protein (SCAP) is required for increased lipid synthesis in liver induced by cholesterol deprivation and insulin elevation. *Genes Dev* 15: 1206–1216, 2001.
192. Matsusue K, Haluzik M, Lambert G, Yim SH, Gavrilova O, Ward JM, Brewer B Jr, Reitman ML, and Gonzalez FJ. Liver-specific disruption of PPARgamma in leptin-deficient mice improves fatty liver but aggravates diabetic phenotypes. *J Clin Invest* 111: 737–747, 2003.
193. Matsuzaki H, Daitoku H, Hatta M, Tanaka K, and Fukamizu A. Insulin-induced phosphorylation of FKHR (Foxo1) targets to proteasomal degradation. *Proc Natl Acad Sci USA* 100: 11285–11290, 2003.
194. Matthaei S, Stumvoll M, Kellerer M, and Haring HU. Pathophysiology and pharmacological treatment of insulin resistance. *Endocr Rev* 21: 585–618, 2000.
195. Medina MA. Glutamine and cancer. *J Nutr* 131: 2509S–2542S, 2001.
196. Michalik L, Desvergne B, and Wahli W. Peroxisome proliferator-activated receptors beta/delta: emerging roles for a previously neglected third family member. *Curr Opin Lipidol* 14: 129–135, 2003.
197. Michellys PY, Ardecky RJ, Chen JH, Crombie DL, Etgen GJ, Faul MM, Faulkner AL, Grese TA, Heyman RA, Karanewsky DS, Klausing K, Leibowitz MD, Liu S, Mais DA, Mapes CM, Marschke KB, Reifel-Miller A, Ogilvie KM, Rungta D, Thompson AW, Tyhonas JS, and Boehm MF. Novel (2E,4E,6Z)-7-(2-alkoxy-3,5-dialkylbenzene)-3-methylocta-2,4,6-trienoic acid retinoid X receptor modulators are active in models of type 2 diabetes. *J Med Chem* 46: 2683–2696, 2003.
198. Miles PD, Barak Y, Evans RM, and Olefsky JM. Effect of heterozygous PPARgamma deficiency and TZD treatment on insulin resistance associated with age and high-fat feeding. *Am J Physiol Endocrinol Metab* 284: E618–E626, 2003.
199. Miles PD, Barak Y, He W, Evans RM, and Olefsky JM. Improved insulin-sensitivity in mice heterozygous for PPAR-gamma deficiency. *J Clin Invest* 105: 287–292, 2000.
200. Mithieux G, Bady I, Gautier A, Croset M, Rajas F, and Zitoun C. Induction of control genes in intestinal gluconeogenesis is sequential during fasting and maximal in diabetes. *Am J Physiol Endocrinol Metab* 286: E370–E375, 2004.
201. Moon YS, Latasa MJ, Kim KH, Wang D, and Sul HS. Two 5′-regions are required for nutritional and insulin regulation of the fatty-acid synthase promoter in transgenic mice. *J Biol Chem* 275: 10121–10127, 2000.
202. Moore ML, Park EA, and McMillin JB. Upstream stimulatory factor represses the induction of carnitine palmitoyltransferase-Ibeta expression by PGC-1. *J Biol Chem* 278: 17263–17268, 2003.
203. Mukherjee R, Davies PJA, Cromble DL, Bischoff ED, Cesario RM, Jow L, Hamann LG, Boehm MF, Mondon CE, Nadzan AM, Paterniti JR Jr, and Heyman RA. Sensitization of diabetic and obese mice to insulin by retinoid X receptor agonists. *Nature* 386: 407–410, 1997.
204. Mukherjee R, Hoener PA, Jow L, Bilakovics J, Klausing K, Mais DE, Faulkner A, Croston GE, and Paterniti JR Jr. A selective peroxisome proliferator-activated receptor-gamma (PPARgamma) modulator blocks adipocyte differentiation but stimulates glucose uptake in 3T3-L1 adipocytes. *Mol Endocrinol* 14: 1425–1433, 2000.
205. Mukherjee R, Strasser J, Jow L, Hoener P, Paterniti JR Jr, and Heyman RA. RXR agonists activate PPARalpha-inducible genes, lower triglycerides, and raise HDL levels in vivo. *Arterioscler Thromb Vasc Biol* 18: 272–276, 1998.
206. Muoio DM, MacLean PS, Lang DB, Li S, Houmard JA, Way JM, Winegar DA, Corton JC, Dohm GL, and Kraus WE. Fatty acid homeostasis and induction of lipid regulatory genes in skeletal muscles of peroxisome proliferator-activated receptor (PPAR) alpha knock-out mice. Evidence for compensatory regulation by PPAR delta. *J Biol Chem* 277: 26089–26097, 2002.
207. Naar AM, Lemon BD, and Tjian R. Transcriptional coactivator complexes. *Annu Rev Biochem* 70: 475–501, 2001.
208. Newsholme P. Why is L-glutamine metabolism important to cells of the immune system in health, postinjury, surgery or infection? *J Nutr* 131: 2515S–2522S, 2001.
209. Norris AW, Chen L, Fisher SJ, Szanto I, Ristow M, Jozsi AC, Hirshman MF, Rosen ED, Goodyear LJ, Gonzalez FJ, Spiegelman BM, and Kahn CR. Muscle-specific PPARgamma-deficient mice develop increased adiposity and insulin resistance but respond to thiazolidinediones. *J Clin Invest* 112: 608–618, 2003.
210. Nuclear Receptors Nomenclature Committee. A unified nomenclature system for the nuclear receptor superfamily. *Cell* 97: 161–163, 1999.
211. Oberfield JL, Collins JL, Holmes CP, Goreham DM, Cooper JP, Cobb JE, Lenhard JM, Hull-Ryde EA, Mohr CP, Blanchard SG, Parks DJ, Moore LB, Lehmann JM, Plunket K, Miller AB, Milburn MV, Kliewer SA, and Willson TM. A peroxisome proliferator-activated receptor gamma ligand inhibits adipocyte differentiation. *Proc Natl Acad Sci USA* 96: 6102–6106, 1999.
212. O’Brien RM and Granner DK. Regulation of gene expression by insulin. *Physiol Rev* 76: 1109–1161, 1996.
213. O’Brien RM, Streepers RS, Ayala JE, Stadelmaier BT, and Hornbuckle LA. Insulin-regulated gene expression. *Biochem Soc Trans* 29: 552–558, 2001.
214. Ohlsson H, Karlsson K, and Edlund T. IPF1, a homeodomain-containing transactivator of the insulin gene. *EMBO J* 12: 4251–4259, 1993.
215. Ohneda K, Ee H, and German M. Regulation of insulin gene transcription. *Semin Cell Dev Biol* 11: 227–233, 2000.
216. Oliver WR Jr, Shenk JL, Snaith MR, Russell CS, Plunket KD, Bodkin NL, Lewis MC, Winegar DA, Sznaidman ML, Lambert MH, Xu HE, Sternbach DD, Kliewer SA, Hansen BC, and Willson TM. A selective peroxisome proliferator-activated receptor delta agonist promotes reverse cholesterol transport. *Proc Natl Acad Sci USA* 98: 5306–5311, 2001.
217. Oral EA, Simha V, Ruiz E, Andewelt A, Premkumar A, Snell P, Wagner AJ, DePaoli AM, Reitman ML, Taylor SI, Gordon P, and Garg A. Leptin-replacement therapy for lipodystrophy. *N Engl J Med* 346: 570–578, 2002.
218. Osborne TF. Sterol regulatory element-binding proteins (SREBPs): key regulators of nutritional homeostasis and insulin action. *J Biol Chem* 275: 32379–32382, 2000.
219. Ou J, Tu H, Shan B, Luk A, DeBose-Boyd RA, Bashmakov Y, Goldstein JL, and Brown MS. Unsaturated fatty acids inhibit

transcription of the sterol regulatory element-binding protein-1c (SREBP-1c) gene by antagonizing ligand-dependent activation of the LXR. *Proc Natl Acad Sci USA* 98: 6027–6032, 2001.

220. Ouamrane L, Larrieu G, Gauthier B, and Pineau T. RXR activators molecular signalling: involvement of a PPAR alpha-dependent pathway in the liver and kidney, evidence for an alternative pathway in the heart. *Br J Pharmacol* 138: 845–854, 2003.

222. Panda S, Antoch MP, Miller BH, Su AI, Schook AB, Straume M, Schultz PG, Kay SA, Takahashi JS, and Hogenesch JB. Coordinated transcription of key pathways in the mouse by the circadian clock. *Cell* 109: 307–320, 2002.

223. Park EA, Gurney AL, Nizielski SE, Hakimi P, Cao Z, Moorman A, and Hanson RW. Relative roles of CCAAT/enhancer-binding protein beta and cAMP regulatory element-binding protein in controlling transcription of the gene for phosphoenolpyruvate carboxykinase (GTP). *J Biol Chem* 268: 613–619, 1993.

224. Patel DD, Knight BL, Wiggins D, Humphreys SM, and Gibbons GF. Disturbances in the normal regulation of SREBP-sensitive genes in PPAR alpha-deficient mice. *J Lipid Res* 42: 328–337, 2001.

225. Patsouris D, Mandard S, Voshol PJ, Escher P, Tan NS, Havekes LM, Koenig W, Marz W, Tafuri S, Wahli W, Muller M, and Kersten S. PPARalpha governs glycerol metabolism. *J Clin Invest* 114: 94–103, 2004.

226. Peet DJ, Turley SD, Ma W, Janowski BA, Lobaccaro JM, Hammer RE, and Mangelsdorf DJ. Cholesterol and bile acid metabolism are impaired in mice lacking the nuclear oxysterol receptor LXR alpha. *Cell* 93: 693–704, 1998.

227. Peng T, Golub TR, and Sabatini DM. The immunosuppressant rapamycin mimics a starvation-like signal distinct from amino acid and glucose deprivation. *Mol Cell Biol* 22: 5575–5584, 2002.

228. Peters JM, Lee SS, Li W, Ward JM, Gavrilova O, Everett C, Reitman ML, Hudson LD, and Gonzalez FJ. Growth, adipose, brain, and skin alterations resulting from targeted disruption of the mouse peroxisome proliferator-activated receptor beta(delta). *Mol Cell Biol* 20: 5119–5128, 2000.

229. Petersen KF and Shulman GI. Pathogenesis of skeletal muscle insulin resistance in type 2 diabetes mellitus. *Am J Cardiol* 90: 11G–18G, 2002.

230. Petersen OH and Findlay I. Electrophysiology of the pancreas. *Physiol Rev* 67: 1054–1116, 1987.

231. Pineda Torra I, Jamshidi Y, Flavell DM, Fruchart JC, and Staels B. Characterization of the human PPARalpha promoter: identification of a functional nuclear receptor response element. *Mol Endocrinol* 16: 1013–1028, 2002.

232. Plosch T, Kok T, Bloks VW, Smit MJ, Havinga R, Chimini G, Groen AK, and Kuipers F. Increased hepatobiliary and fecal cholesterol excretion upon activation of the liver X receptor is independent of ABCA1. *J Biol Chem* 277: 33870–33877, 2002.

233. Pontoglio M. Hepatocyte nuclear factor 1, a transcription factor at the crossroads of glucose homeostasis. *J Am Soc Nephrol* 11 Suppl 16: S140–S143, 2000.

234. Porzio O, Federici M, Hribal ML, Lauro D, Accili D, Lauro R, Borboni P, and Sesti G. The Gly972→Arg amino acid polymorphism in IRS-1 impairs insulin secretion in pancreatic beta cells. *J Clin Invest* 104: 357–364, 1999.

235. Puig O, Marr MT, Ruhf ML, and Tjian R. Control of cell number by *Drosophila* FOXO: downstream and feedback regulation of the insulin receptor pathway. *Genes Dev* 17: 2006–2020, 2003.

236. Puigserver P, Wu Z, Park CW, Graves R, Wright M, and Spiegelman BM. A cold-inducible coactivator of nuclear receptors linked to adaptive thermogenesis. *Cell* 92: 829–839, 1998.

237. Rafiq I, Kennedy HJ, and Rutter GA. Glucose-dependent translocation of insulin promoter factor-1 (IPF-1) between the nuclear periphery and the nucleoplasm of single MIN6 beta- cells. *J Biol Chem* 273: 23241–23247, 1998.

238. Ramji DP and Foka P. CCAAT/enhancer-binding proteins: structure, function and regulation. *Biochem J* 365: 561–575, 2002.

239. Randle PJ, Garland PB, Hales CN, and Newsholme EA. The glucose fatty-acid cycle. Its role in insulin sensitivity and the metabolic disturbances of diabetes mellitus. *Lancet* 1: 785–789, 1963.

240. Raught B, Gingras AC, and Sonenberg N. The target of rapamycin (TOR) proteins. *Proc Natl Acad Sci USA* 98: 7037–7044, 2001.


241. Reeds PJ. Dispensable and indispensable amino acids for humans. *J Nutr* 130: 1835S–1840S, 2000.

242. Repa JJ, Berge KE, Pomajzl C, Richardson JA, Hobbs H, and Mangelsdorf DJ. Regulation of ATP-binding cassette sterol transporters ABCG5 and ABCG8 by the liver X receptors alpha and beta. *J Biol Chem* 277: 18793–18800, 2002.

243. Repa JJ, Liang G, Ou J, Bashmakov Y, Lobaccaro JM, Shimomura I, Shan B, Brown MS, Goldstein JL, and Mangelsdorf DJ. Regulation of mouse sterol regulatory element-binding protein-1c gene (SREBP-1c) by oxysterol receptors, LXRalpha and LXRbeta. *Genes Dev* 14: 2819–2830, 2000.

244. Repa JJ and Mangelsdorf DJ. Nuclear receptor regulation of cholesterol and bile acid metabolism. *Curr Opin Biotechnol* 10: 557–563, 1999.

245. Repa JJ, Turley SD, Lobaccaro JA, Medina J, Li L, Lustig K, Shan B, Heyman RA, Dietschy JM, and Mangelsdorf DJ. Regulation of absorption and ABC1-mediated efflux of cholesterol by RXR heterodimers. *Science* 289: 1524–1529, 2000.

246. Reppert SM and Weaver DR. Coordination of circadian timing in mammals. *Nature* 418: 935–941, 2002.

247. Rieusset J, Touri F, Michalik L, Escher P, Desvergne B, Niesor E, and Wahli W. A new selective peroxisome proliferator-activated receptor gamma antagonist with antiobesity and antidiabetic activity. *Mol Endocrinol* 16: 2628–2644, 2002.

248. Rizvi NA, Marshall JL, Dahut W, Ness E, Truglia JA, Loewen G, Gill GM, Ulm EH, Geiser R, Jaunakais D, and Hawkins MJ. A Phase I study of LGD1069 in adults with advanced cancer. *Clin Cancer Res* 5: 1658–1664, 1999.

249. Robertson RP, Harmon J, Tran PO, and Poitout V. Beta-cell glucose toxicity, lipotoxicity, and chronic oxidative stress in type 2 diabetes. *Diabetes* 53 Suppl 1: S119–S124, 2004.

250. Robyr D, Wolffe AP, and Wahli W. Nuclear hormone receptor coregulators in action: diversity for shared tasks. *Mol Endocrinol* 14: 329–347, 2000.

251. Rocchi S, Picard F, Vamecq J, Gelman L, Potier N, Zeyer D, Dubuquoy L, Bac P, Champy MF, Plunket KD, Leesnitzer LM, Blanchard SG, Desreumaux P, Moras D, Renaud JP, and Auwerx J. A unique PPARgamma ligand with potent insulin-sensitizing yet weak adipogenic activity. *Mol Cell* 8: 737–747, 2001.

252. Rodrigues CM, Fan G, Ma X, Kren BT, and Steer CJ. A novel role for ursodeoxycholic acid in inhibiting apoptosis by modulating mitochondrial membrane perturbation. *J Clin Invest* 101: 2790–2799, 1998.

253. Roesler WJ. The role of C/EBP in nutrient and hormonal regulation of gene expression. *Annu Rev Nutr* 21: 141–165, 2001.

254. Rosen ED, Kulkarni RN, Sarraf P, Ozcan U, Okada T, Hsu CH, Eisenman D, Magnuson MA, Gonzalez FJ, Kahn CR, and Spiegelman BM. Targeted elimination of peroxisome proliferator-activated receptor gamma in beta cells leads to abnormalities in islet mass without compromising glucose homeostasis. *Mol Cell Biol* 23: 7222–7229, 2003.

255. Rosen ED, Sarraf P, Troy AE, Bradwin G, Moore K, Milstone DS, Spiegelman BM, and Mortensen RM. PPAR gamma is required for the differentiation of adipose tissue in vivo and in vitro. *Mol Cell* 4: 611–617, 1999.

256. Rosen ED and Spiegelman BM. Molecular regulation of adipogenesis. *Annu Rev Cell Dev Biol* 16: 145–171, 2000.

257. Rosen ED and Spiegelman BM. PPARgamma: a nuclear regulator of metabolism, differentiation, and cell growth. *J Biol Chem* 276: 37731–37734, 2001.

258. Rosen ED, Walkey CJ, Puigserver P, and Spiegelman BM. Transcriptional regulation of adipogenesis. *Genes Dev* 14: 1293–1307, 2000.

259. Ross SE, Erickson RL, Gerin I, DeRose PM, Bajnok L, Longo KA, Misek DE, Kuick R, Hanash SM, Atkins KB, Andresen SM, Nebb HI, Madsen L, Kristiansen K, and MacDougald OA. Microarray analyses during adipogenesis: understanding the effects of Wnt signaling on adipogenesis and the roles of liver X receptor alpha in adipocyte metabolism. *Mol Cell Biol* 22: 5989–5999, 2002.

260. Ruan H, Pownall HJ, and Lodish HF. Troglitazone antagonizes tumor necrosis factor-alpha-induced reprogramming of adipocyte gene expression by inhibiting the transcriptional regulatory functions of NF-kappaB. *J Biol Chem* 278: 28181–28192, 2003.

261. Rufo C, Teran-Garcia M, Nakamura MT, Koo SH, Towle HC, and Clarke SD. Involvement of a unique carbohydrate-responsive factor in the glucose regulation of rat liver fatty-acid synthase gene transcription. *J Biol Chem* 276: 21969–21975, 2001.

262. Rutter J, Reick M, Wu LC, and McKnight SL. Regulation of clock and NPAS2 DNA binding by the redox state of NAD cofactors. *Science* 293: 510–514, 2001.

263. Ryffel GU. Mutations in the human genes encoding the transcription factors of the hepatocyte nuclear factor (HNF)1 and HNF4 families: functional and pathological consequences. *J Mol Endocrinol* 27: 11–29, 2001.

264. Saltiel AR. New perspectives into the molecular pathogenesis and treatment of type 2 diabetes. *Cell* 104: 517–529, 2001.

265. Saltiel AR and Kahn CR. Insulin signalling and the regulation of glucose and lipid metabolism. *Nature* 414: 799–806, 2001.

266. Saltiel AR and Olefsky JM. Thiazolidinediones in the treatment of insulin resistance and type II diabetes. *Diabetes* 45: 1661–1669, 1996.

267. Samuelsson L, Stromberg K, Vikman K, Bjursell G, and Enerback S. The CCAAT/enhancer binding protein and its role in adipocyte differentiation: evidence for direct involvement in terminal adipocyte development. *EMBO J* 10: 3787–3793, 1991.

268. Sato R, Inoue J, Kawabe Y, Kodama T, Takano T, and Maeda M. Sterol-dependent transcriptional regulation of sterol regulatory element-binding protein-2. *J Biol Chem* 271: 26461–26464, 1996.

269. Schibler U and Naef F. Cellular oscillators: rhythmic gene expression and metabolism. *Curr Opin Cell Biol* 17: 223–229, 2005.

270. Schibler U and Sassone-Corsi P. A web of circadian pacemakers. *Cell* 111: 919–922, 2002.

271. Schrem H, Klempnauer J, and Borlak J. Liver-enriched transcription factors in liver function and development. Part I: the hepatocyte nuclear factor network and liver-specific gene expression. *Pharmacol Rev* 54: 129–158, 2002.

272. Schultz JR, Tu H, Luk A, Repa JJ, Medina JC, Li L, Schwender S, Wang S, Thoolen M, Mangelsdorf DJ, Lustig KD, and Shan B. Role of LXRs in control of lipogenesis. *Genes Dev* 14: 2831–2838, 2000.

273. Shao J, Sheng H, and DuBois RN. Peroxisome proliferator-activated receptors modulate K-Ras-mediated transformation of intestinal epithelial cells. *Cancer Res* 62: 3282–3288, 2002.

274. Shi XM, Blair HC, Yang X, McDonald JM, and Cao X. Tandem repeat of C/EBP binding sites mediates PPARgamma2 gene transcription in glucocorticoid-induced adipocyte differentiation. *J Cell Biochem* 76: 518–527, 2000.

275. Shi Y, Hon M, and Evans RM. The peroxisome proliferator-activated receptor delta, an integrator of transcriptional repression and nuclear receptor signaling. *Proc Natl Acad Sci USA* 99: 2613–2618, 2002.

276. Shih HM and Towle HC. Definition of the carbohydrate response element of the rat S14 gene. Evidence for a common factor required for carbohydrate regulation of hepatic genes. *J Biol Chem* 267: 13222–13228, 1992.

277. Shimano H, Horton JD, Hammer RE, Shimomura I, Brown MS, and Goldstein JL. Overproduction of cholesterol and fatty acids causes massive liver enlargement in transgenic mice expressing truncated SREBP-1a. *J Clin Invest* 98: 1575–1584, 1996.

278. Shimomura I, Bashmakov Y, Ikemoto S, Horton JD, Brown MS, and Goldstein JL. Insulin selectively increases SREBP-1c mRNA in the livers of rats with streptozotocin-induced diabetes. *Proc Natl Acad Sci USA* 96: 13656–13661, 1999.

279. Shimomura I, Matsuda M, Hammer RE, Bashmakov Y, Brown MS, and Goldstein JL. Decreased IRS-2 and increased SREBP-1c lead to mixed insulin resistance and sensitivity in livers of lipodystrophic and ob/ob mice. *Mol Cell* 6: 77–86, 2000.

280. Shimomura I, Shimano H, Horton JD, Goldstein JL, and Brown MS. Differential expression of exons 1a and 1c in mRNAs for sterol regulatory element binding protein-1 in human and mouse organs and cultured cells. *J Clin Invest* 99: 838–845, 1997.

281. Shimomura I, Shimano H, Korn BS, Bashmakov Y, and Horton JD. Nuclear sterol regulatory element-binding proteins activate genes responsible for the entire program of unsaturated fatty acid biosynthesis in transgenic mouse liver. *J Biol Chem* 273: 35299–35306, 1998.

282. Sinal CJ, Tohkin M, Miyata M, Ward JM, Lambert G, and Gonzalez FJ. Targeted disruption of the nuclear receptor FXR/BAR impairs bile acid and lipid homeostasis. *Cell* 102: 731–744, 2000.

283. Siu F, Bain PJ, LeBlanc-Chaffin R, Chen H, and Kilberg MS. ATF4 is a mediator of the nutrient-sensing response pathway that activates the human asparagine synthetase gene. *J Biol Chem* 277: 24120–24127, 2002.

284. Siu F, Chen C, Zhong C, and Kilberg MS. CCAAT/enhancer-binding protein-beta is a mediator of the nutrient-sensing response pathway that activates the human asparagine synthetase gene. *J Biol Chem* 276: 48100–48107, 2001.

285. Skrumsager BK, Nielsen KK, Muller M, Pabst G, Drake PG, and Edsberg B. Ragaglitazar: the pharmacokinetics, pharmacodynamics, and tolerability of a novel dual PPARalpha and gamma agonist in healthy subjects and patients with type 2 diabetes. *J Clin Pharmacol* 43: 1244–1256, 2003.

286. Sladek F. Desperately seeking ... something. *Mol Cell* 10: 219–221, 2002.

287. Steffensen KR, Schuster GU, Parini P, Holter E, Sadek CM, Cassel T, Eskild W, and Gustafsson JA. Different regulation of the LXRA promoter activity by isoforms of CCAAT/enhancer-binding proteins. *Biochem Biophys Res Commun* 293: 1333–1340, 2002.

288. Stokkan KA, Yamazaki S, Tei H, Sakaki Y, and Menaker M. Entrainment of the circadian clock in the liver by feeding. *Science* 291: 490–493, 2001.

289. Stumvoll M, Goldstein BJ, and van Haeften TW. Type 2 diabetes: principles of pathogenesis and therapy. *Lancet* 365: 1333–1346, 2005.

290. Sugden MC, Bulmer K, Gibbons GF, Knight BL, and Holness MJ. Peroxisome-proliferator-activated receptor-alpha (PPARalpha) deficiency leads to dysregulation of hepatic lipid and carbohydrate metabolism by fatty acids and insulin. *Biochem J* 364: 361–368, 2002.

291. Surapureddi S, Yu S, Bu H, Hashimoto T, Yeldandi AV, Kashireddy P, Cherkaoui-Malki M, Qi C, Zhu YJ, Rao MS, and Reddy JK. Identification of a transcriptionally active peroxisome proliferator-activated receptor alpha-interacting cofactor complex in rat liver and characterization of PRIC285 as a coactivator. *Proc Natl Acad Sci USA* 99: 11836–11841, 2002.

292. Tall AR, Costet P, and Wang N. Regulation and mechanisms of macrophage cholesterol efflux. *J Clin Invest* 110: 899–904, 2002.

293. Tamori Y, Masugi J, Nishino N, and Kasuga M. Role of peroxisome proliferator-activated receptor-gamma in maintenance of the characteristics of mature 3T3-L1 adipocytes. *Diabetes* 51: 2045–2055, 2002.

294. Tanaka T, Yamamoto J, Iwasaki S, Asaba H, Hamura H, Ikeda Y, Watanabe M, Magoori K, Ioka RX, Tachibana K, Watanabe Y, Uchiyama Y, Sumi K, Iguchi H, Ito S, Doi T, Hamakubo T, Naito M, Auwerx J, Yanagisawa M, Kodama T, and Sakai J. Activation of peroxisome proliferator-activated receptor delta induces fatty acid beta-oxidation in skeletal muscle and attenuates metabolic syndrome. *Proc Natl Acad Sci USA* 100: 15924–15929, 2003.

295. Tanaka T, Yoshida N, Kishimoto T, and Akira S. Defective adipocyte differentiation in mice lacking the C/EBPbeta and/or C/EBPdelta gene. *EMBO J* 16: 7432–7443, 1997.

296. Tobin KA, Steineger HH, Alberti S, Spydevold O, Auwerx J, Gustafsson JA, and Nebb HI. Cross-talk between fatty acid and cholesterol metabolism mediated by liver X receptor-alpha. *Mol Endocrinol* 14: 741–752, 2000.

297. Tontonoz P, Kim JB, Graves RA, and Spiegelman BM. ADD1: a novel helix-loop-helix transcription factor associated with adipocyte determination and differentiation. *Mol Cell Biol* 13: 4753–4759, 1993.

298. Tontonoz P and Mangelsdorf DJ. Liver X receptor signaling pathways in cardiovascular disease. *Mol Endocrinol* 17: 985–993, 2003.

299. Tran H, Brunet A, Griffith EC, and Greenberg ME. The many forks in FOXO’s road. *Sci STKE* 2003: RE5, 2003.

300. Turek FW, Joshu C, Kohsaka A, Lin E, Ivanova G, McDearmon E, Laposky A, Losee-Olson S, Easton A, Jensen DR,

Eckel RH, Takahashi JS, and Bass J. Obesity and metabolic syndrome in circadian Clock mutant mice. *Science* 308: 1043–1045, 2005.

301. Unger RH. Lipotoxicity in the pathogenesis of obesity-dependent NIDDM. *Genetic and clinical implications*. *Diabetes* 44: 863–870, 1995.

302. Unger RH. Longevity, lipotoxicity and leptin: the adipocyte defense against feasting and famine. *Biochimie* 87: 57–64, 2005.

303. Urizar NL, Dowhan DH, and Moore DD. The farnesoid X-activated receptor mediates bile acid activation of phospholipid transfer protein gene expression. *J Biol Chem* 275: 39313–39317, 2000.

304. Urizar NL and Moore DD. GUGULIPID: a natural cholesterol-lowering agent. *Annu Rev Nutr* 23: 303–313, 2003.

305. Vaulont S, Vasseur-Cognet M, and Kahn A. Glucose regulation of gene transcription. *J Biol Chem* 275: 31555–31558, 2000.

306. Vidal-Puig A, Jimenez-Linan M, Lowell BB, Hamann A, Hu E, Spiegelman B, Flier JS, and Moller DE. Regulation of PPAR gamma gene expression by nutrition and obesity in rodents. *J Clin Invest* 97: 2553–2561, 1996.

307. Viollet B, Andreelli F, Jorgensen SB, Perrin C, Geloen A, Flamez D, Mu J, Lenzner C, Baud O, Bennoun M, Gomas E, Nicolas G, Wojtaszewski JF, Kahn A, Carling D, Schuit FC, Birnbaum MJ, Richter EA, Burcelin R, and Vaulont S. The AMP-activated protein kinase alpha2 catalytic subunit controls whole-body insulin sensitivity. *J Clin Invest* 111: 91–98, 2003.

308. Wajchenberg BL. Subcutaneous and visceral adipose tissue: their relation to the metabolic syndrome. *Endocr Rev* 21: 697–738, 2000.

309. Wan YJ, An D, Cai Y, Repa JJ, Hung-Po Chen T, Flores M., Postic C, Magnuson MA, Chen J, Chien KR, French S, Mangelsdorf DJ, and Sucov HM. Hepatocyte-specific mutation establishes retinoid X receptor alpha as a heterodimeric integrator of multiple physiological processes in the liver. *Mol Cell Biol* 20: 4436–4444, 2000.

310. Wang H, Hagenfeldt-Johansson K, Otten LA, Gauthier BR, Herrera PL, and Wollheim CB. Experimental models of transcription factor-associated maturity-onset diabetes of the young. *Diabetes* 51 Suppl 3: S333–S342, 2002.

311. Wang H, Maechler P, Antinozzi PA, Herrero L, Hagenfeldt-Johansson KA, Bjorklund A, and Wollheim CB. The transcription factor SREBP-1c is instrumental in the development of beta-cell dysfunction. *J Biol Chem* 278: 16622–16629, 2003.

312. Wang ND, Finegold MJ, Bradley A, Ou CN, Abdelsayed SV, Wilde MD, Taylor LR, Wilson DR, and Darlington GJ. Impaired energy homeostasis in C/EBP alpha knockout mice. *Science* 269: 1108–1112, 1995.

313. Wang YX, Lee CH, Tiep S, Yu RT, Ham J, Kang H, and Evans RM. Peroxisome-proliferator-activated receptor delta activates fat metabolism to prevent obesity. *Cell* 113: 159–170, 2003.

314. Wang YX, Zhang CL, Yu RT, Cho HK, Nelson MC, Bayuga-Ocampo CR, Ham J, Kang H, and Evans RM. Regulation of muscle fiber type and running endurance by PPARdelta. *PLoS Biol* 2: e294, 2004.

315. Wang Z, Benoit G, Liu J, Prasad S, Aarnisalo P, Liu X, Xu H, Walker NP, and Perlmann T. Structure and function of Nurrl identifies a class of ligand-independent nuclear receptors. *Nature* 423: 555–560, 2003.

316. Wendling O, Chambon P, and Mark M. Retinoid X receptors are essential for early mouse development and placentogenesis. *Proc Natl Acad Sci USA* 96: 547–551, 1999.

317. Williams K, Rao YP, Natarajan R, Pandak WM, and Hylemon PB. Indinavir alters sterol and fatty acid homeostatic mechanisms in primary rat hepatocytes by increasing levels of activated sterol regulatory element-binding proteins and decreasing cholesterol 7alpha-hydroxylase mRNA levels. *Biochem Pharmacol* 67: 255–267, 2004.

318. Winder WW and Hardie DG. AMP-activated protein kinase, a metabolic master switch: possible roles in type 2 diabetes. *Am J Physiol Endocrinol Metab* 277: E1–E10, 1999.

319. Winter WE and Silverstein JH. Molecular and genetic bases for maturity onset diabetes of youth. *Curr Opin Pediatr* 12: 388–393, 2000.

320. Wisely GB, Miller AB, Davis RG, Thornquest AD Jr, Johnson R, Spitzer T, Seffler A, Shearer B, Moore JT, Willson TM, and

Williams SP. Hepatocyte nuclear factor 4 is a transcription factor that constitutively binds fatty acids. *Structure* 10: 1225–1234, 2002.

321. Wolfrum C, Shih DQ, Kuwajima S, Norris AW, Kahn CR, and Stoffel M. Role of Foxa-2 in adipocyte metabolism and differentiation. *J Clin Invest* 112: 345–356, 2003.

322. Wright HM, Clish CB, Mikami T, Hauser S, Yanagi K, Hiramatsu R, Serhan CN, and Spiegelman BM. A synthetic antagonist for the peroxisome proliferator-activated receptor gamma inhibits adipocyte differentiation. *J Biol Chem* 275: 1873–1877, 2000.

323. Wu Z, Puigserver P, Andersson U, Zhang C, Adelmant G, Mootha V, Troy A, Cinti S, Lowell B, Scarpulla RC, and Spiegelman BM. Mechanisms controlling mitochondrial biogenesis and respiration through the thermogenic coactivator PGC-1. *Cell* 98: 115–124, 1999.

324. Xu HE, Lambert MH, Montana VG, Plunket KD, Moore LB, Collins JL, Oplinger JA, Kliewer SA, Gampe RT Jr, McKee DD, Moore JT, and Willson TM. Structural determinants of ligand binding selectivity between the peroxisome proliferator-activated receptors. *Proc Natl Acad Sci USA* 98: 13919–13924, 2001.

325. Xu J, Chang V, Joseph SB, Trujillo C, Bassilian S, Saad MF, Lee WN, and Kurland IJ. Peroxisomal proliferator-activated receptor alpha deficiency diminishes insulin-responsiveness of gluconeogenic/glycolytic/pentose gene expression and substrate cycle flux. *Endocrinology* 145: 1087–1095, 2004.

326. Xu J, Teran-Garcia M, Park JH, Nakamura MT, and Clarke SD. Polyunsaturated fatty acids suppress hepatic sterol regulatory element-binding protein-1 expression by accelerating transcript decay. *J Biol Chem* 276: 9800–9807, 2001.

327. Xu J, Xiao G, Trujillo C, Chang V, Blanco L, Joseph SB, Bassilian S, Saad MF, Tontonoz P, Lee WN, and Kurland IJ. Peroxisome proliferator-activated receptor alpha (PPARalpha) influences substrate utilization for hepatic glucose production. *J Biol Chem* 277: 50237–50244, 2002.

328. Yabe D, Komuro R, Liang G, Goldstein JL, and Brown MS. Liver-specific mRNA for Insig-2 down-regulated by insulin: implications for fatty acid synthesis. *Proc Natl Acad Sci USA* 100: 3155–3160, 2003.

329. Yamashita H, Takenoshita M, Sakurai M, Bruick RK, Henzel WJ, Shillinglaw W, Arnot D, and Uyeda K. A glucose-responsive transcription factor that regulates carbohydrate metabolism in the liver. *Proc Natl Acad Sci USA* 98: 9116–9121, 2001.

330. Yamauchi T, Kamon J, Waki H, Terauchi Y, Kubota N, Hara K, Mori Y, Ide T, Murakami K, Tsuboyama-Kasaoka N, Ezaki O, Akanuma Y, Gavrilova O, Vinson C, Reitman ML, Kagechika H, Shudo K, Yoda M, Nakano Y, Tobe K, Nagai R, Kimura S, Tomita M, Froguel P, and Kadowaki T. The fat-derived hormone adiponectin reverses insulin resistance associated with both lipoatrophy and obesity. *Nat Med* 7: 941–946, 2001.

331. Yang T, Espenshade PJ, Wright ME, Yabe D, Gong Y, Aebersold R, Goldstein JL, and Brown MS. Crucial step in cholesterol homeostasis: sterols promote binding of SCAP to INSIG-1, a membrane protein that facilitates retention of SREBPs in ER. *Cell* 110: 489–500, 2002.

332. Ye JM, Doyle PJ, Iglesias MA, Watson DG, Cooney GJ, and Kraegen EW. Peroxisome proliferator-activated receptor (PPAR)-alpha activation lowers muscle lipids and improves insulin sensitivity in high fat-fed rats: comparison with PPAR-gamma activation. *Diabetes* 50: 411–417, 2001.

333. Yechoor VK, Patti ME, Saccone R, and Kahn CR. Coordinated patterns of gene expression for substrate and energy metabolism in skeletal muscle of diabetic mice. *Proc Natl Acad Sci USA* 99: 10587–10592, 2002.

334. Yoon JC, Puigserver P, Chen G, Donovan J, Wu Z, Rhee J, Adelmant G, Stafford J, Kahn CR, Granner DK, Newgard CB, and Spiegelman BM. Control of hepatic gluconeogenesis through the transcriptional coactivator PGC-1. *Nature* 413: 131–138, 2001.

335. Yoshikawa T, Shimano H, Amemiya-Kudo M, Yahagi N, Hasty AH, Matsuzaka T, Okazaki H, Tamura Y, Iizuka Y, Ohashi K, Osuga J, Harada K, Gotoda T, Kimura S, Ishibashi S, and Yamada N. Identification of liver X receptor-retinoid X receptor as an activator of the sterol regulatory element-binding protein 1c gene promoter. *Mol Cell Biol* 21: 2991–3000, 2001.

336. Yoshikawa T, Shimano H, Yahagi N, Ide T, Amemiya-Kudo M, Matsuzaka T, Nakakuki M, Tomita S, Okazaki H, Tamura Y, Izuka Y, Ohashi K, Takahashi A, Sone H, Osuga Ji J, Gotoda T, Ishibashi S, and Yamada N. Polyunsaturated fatty acids suppress sterol regulatory element-binding protein 1c promoter activity by inhibition of liver X receptor (LXR) binding to LXR response elements. *J Biol Chem* 277: 1705–1711, 2002.

337. Young ME, Laws FA, Goodwin GW, and Taegtmeyer H. Reactivation of peroxisome proliferator-activated receptor alpha is associated with contractile dysfunction in hypertrophied rat heart. *J Biol Chem* 276: 44390–44395, 2001.

338. Yu S, Matsusue K, Kashireddy P, Cao WQ, Yeldandi V, Yeldandi AV, Rao MS, Gonzalez FJ, and Reddy JK. Adipocyte-specific gene expression and adipogenic steatosis in the mouse

liver due to peroxisome proliferator-activated receptor gamma (PPARgamma) overexpression. *J Biol Chem* 278: 498–505, 2003.

339. Zanger K, Radovick S, and Wondisford FE. CREB binding protein recruitment to the transcription complex requires growth factor-dependent phosphorylation of its GF box. *Mol Cell* 7: 551–558, 2001.

340. Zannis VI, Chroni A, Kypreos KE, Kan HY, Cesar TB, Zanni EE, and Kardassis D. Probing the pathways of chylomicron and HDL metabolism using adenovirus-mediated gene transfer. *Curr Opin Lipidol* 15: 151–166, 2004.

341. Zhou XY, Shibusawa N, Naik K, Porras D, Temple K, Ou H, Kaihara K, Roe MW, Brady MJ, and Wondisford FE. Insulin regulation of hepatic gluconeogenesis through phosphorylation of CREB-binding protein. *Nat Med* 10: 633–637, 2004.
