
Mol Gen Genet (1991) 231:124–138

**Characterization of a novel Azorhizobium caulinodans ORS571 two-component regulatory system, NtrY/NtrX, involved in nitrogen fixation and metabolism**

K. Pawlowski${}^{1}$, U. Klosse${}^{1}$, and F.J. de Bruijn${}^{1,2,*}$

${}^{1}$ Max-Planck-Institut für Züchtungsforschung, Carl-Von-Linné-Weg 10, W-5000 Köln 30, FRG  
${}^{2}$ MSU-DOE Plant Research Laboratory and Department of Microbiology, Michigan State University, East Lansing, MI 48824, USA  

---

**Summary.** *Azorhizobium caulinodans* ORS571 *nifA* regulation is partially mediated by the nitrogen regulatory gene *ntrC*. However, the residual *nifA* expression in *ntrC* mutant strains is still modulated by the cellular nitrogen and oxygen status. A second *ntrC*-homologous region, linked to *ntrC*, was identified and characterized by site-directed insertion mutagenesis and DNA sequencing. Tn5 insertions in this region cause pleiotropic defects in nitrogen metabolism and affect free-living as well as symbiotic nitrogen fixation. DNA sequencing and complementation studies revealed the existence of a bicistronic operon (*ntrYX*). NtrY is likely to represent the transmembrane ‘sensor’ protein element in a two-component regulatory system. NtrX shares a high degree of homology with NtrC proteins of other organisms and probably constitutes the regulator protein element. The regulation of the *ntrYX* and *ntrC* loci and the effects of *ntrYX*, *ntrY* and *ntrX* mutations on *nifA* expression were examined using β-galactosidase gene fusions. NtrY/NtrX were found to modulate *nifA* expression and *ntrYX* transcription was shown to be partially under the control of NtrC.

**Key words:** *Azorhizobium caulinodans* ORS571 – *nifA* regulation – Nitrogen regulation – Two-component regulatory system – Nitrogen fixation and metabolism

---

**Introduction**

*Azorhizobium caulinodans* ORS571 (Dreyfus et al. 1988; referred to as ORS571 in this report) occupies a unique position among the Rhizobiaceae: it can fix nitrogen in both stem and root nodules induced on its host, the tropical leguminous shrub Sesbania rostrata (Dreyfus and Dommergues 1981) and has the capacity to grow on dinitrogen ($\mathrm{N}_{2}$) in the free-living state, at an unusually high temperature ($37^{\circ} \mathrm{C}$) and $\mathrm{O}_{2}$ concentration (~3%; Dreyfus et al. 1983; Gebhardt et al. 1984; see de Bruijn 1989). It, therefore, constitutes a unique rhizobial strain in which it is possible to compare the regulation of nitrogen fixation (*nif/fix*) and assimilation genes in the free-living and symbiotic states (de Bruijn et al. 1987, 1988, 1990).

Expression of *nif* genes in ORS571 has been shown to be strictly controlled by the *nif*-specific regulatory gene *nifA*, both in the free-living and symbiotic states (Donald et al. 1986; Pawlowski et al. 1987; Ratet et al. 1988b). The regulatory circuitry controlling ORS571 *nifA* expression has been shown to have elements in common with that found in *Klebsiella pneumoniae*, where *nifA* transcription is controlled by the *ntr* (nitrogen regulation) system, consisting of the NtrA, NtrB and NtrC proteins (see Gussin et al. 1986; Magasanik 1988; de Bruijn et al. 1990). NtrB/NtrC represent a two-component regulatory system (see Albright et al. 1989), composed of a cytosolic N-sensor protein (NtrB), which phosphorylates and thereby activates a DNA-binding regulatory protein (NtrC) under N-limiting conditions. NtrA (RpoN) is an alternative sigma factor, initiating transcription from *ntr*-controlled promoters in the presence of the corresponding regulatory DNA-binding proteins, e.g. NtrC or NifA (see Gussin et al. 1986; Magasanik 1988; Thöny and Hennecke 1989; Albright et al. 1989; Buck 1990; de Bruijn et al. 1990). The RNA polymerase – NtrA complex recognizes a characteristic promoter element ($-^{24} \mathrm{GG} /-^{12} \mathrm{GC}$; see Thöny and Hennecke 1989).

A mutation in the ORS571 *ntrC* gene causes a severe reduction in free-living nitrogen fixation and affects *nifA* transcription. In addition, ORS571 *ntrC* mutant strains are drastically altered in their nitrate utilization capacity and symbiotic properties (Pawlowski et al. 1987; de Bruijn et al. 1988; Ratet et al. 1989). The latter feature is peculiar to ORS571, since in *Rhizobium meliloti* and *Bradyrhizobium japonicum*, *ntrC* does not modulate symbiotic behavior (Szeto et al. 1987; Martin et al. 1988; see de Bruijn et al. 1990).

---

*Offprint requests to:* F. de Bruijn, MSU-DOE Plant Research Laboratory, Michigan State University, East Lansing, MI 48824, USA
The R. meliloti *nifA* gene is oxygen-controlled (induced microaerobically) via the two-component regulatory system FixL/FixJ, consisting of a transmembrane O₂ sensor (FixL) and a transcriptional activator, which has been postulated to act independently of NtrA (FixJ; David et al. 1988; Hertig et al. 1989; Gillis-Gonzales et al. 1991; see de Bruijn and Downie 1991). Expression of *nifA* in *B. japonicum* is enhanced microaerobically due to autoactivation of the *fixRnifA* operon by NifA (Thöny et al. 1989). The ORS571 *nifA* promoter is also oxygen-controlled via the *fixLJ* system and NifA-mediated autoregulation has been suggested (de Bruijn et al. 1988; Ratet et al. 1989; Kaminski and Elmerich 1991). Three conserved DNA motifs are present in the ORS571 *nifA* 5′ upstream region, which may be involved in nitrogen and oxygen control (de Bruijn et al. 1988; 1990; Nees et al. 1988; Ratet et al. 1989), namely a putative –24/–12 promoter element (–24GG-N10-GC; see Thöny and Hennecke 1989; presumably involved in *ntr* control), an upstream activating sequence (UAS, GGT-N₁₀-ACA TGT-N10-ACA is the consensus sequence, see Buck 1990), which may interact with NifA and a sequence found in the promoters of *E. coli* genes induced during anaerobiosis by the Fnr regulatory protein (TTGAT-N₄-ATCAA; *fnr* box, see Spiro and Guest 1990).

ORS571 contains several regions homologous to *nifA/ntrC* in its genome, two of which have been characterized (Pawlowski et al. 1987). Since the reduced level of *nifA* expression remains under oxygen and nitrogen control in ORS571 *ntrC* mutants (Ratet et al. 1989), the presence of additional functional *ntrC*-homologous or related regulatory genes, such as those described for *Rhodobacter capsulatus* (Kranz and Haselkorn 1988), has been postulated. Here we report the characterization of an *nifA/ntrC* homologous region (*ntrX*) by DNA sequencing, as well as by Tn5 and miniMu-lac mutagenesis (de Bruijn 1987; Ratet et al. 1988a). *ntrX::Tn5* insertions were found to affect *nifA-lac* expression, nitrogen fixation and assimilation. *ntrX* was found to be part of a bicistronic operon (*ntrYX*), which shares homology with two-component regulatory systems and itself appears to be *ntr*-controlled. Preliminary reports of these results were presented at the 7th and 8th International Nitrogen Fixation Congresses in Cologne, FRG, and Knoxville, USA (de Bruijn et al. 1988; 1990).

### Materials and methods

#### Strains, phages and plasmids
The bacterial strains, phages and plasmids used are listed in Table 1.

#### Media and chemicals
Media and antibiotic concentrations used have been described by Pawlowski et al. (1987). Nitrogen sources were added to LSO medium (Elmerich et al. 1982) to a final concentration of 0.2%.

#### DNA isolation and manipulations
These procedures were carried out as described by Maniatis et al. (1982) and Meade et al. (1982).

DNA sequence analysis. Appropriate restriction fragments were subcloned in the polylinker of M13mp18/19 or pUC18/19 (Yanisch-Perron et al. 1985) and deletion clones were constructed using the exonuclease III/nuclease S1 method described by Henikoff (1984). Sequencing reactions were carried out using the chain termination method (Sanger et al. 1977, 1980) with *TaqI*-polymerase (Promega) according to the instructions of the manufacturer. The *ntrX::MudIIIPR46* junction sequence was determined using the standard M13 primer. DNA sequences and deduced amino acid sequences were analyzed using the software package of the University of Wisconsin Computer Group (Devereux et al. 1984). The ‘codonpreference’ calculation was based on the codon usage found in the ORS571 *nifA* and *nifH* genes (Ratet et al. 1989; Norel and Elmerich 1987). For data bank searches, the protein sequence databases NRBF (National Biomedical Research Foundation, Washington, USA) and Swiss-Prot (EMBL, Heidelberg, FRG) and the nucleic acid sequence databases EMBL data library (EMBL, Heidelberg, FRG) and GenBank (IntelliGenetics, Mountain View, Calif., USA) were used.

*Tn5* mutagenesis, conjugation and gene replacement (homogetization). These experiments were performed as described by de Bruijn (1987).

#### Construction of *Tn5* insertion mutants
The deletion cosmids pLRSC1 Δ3, Δ12, and Δ28 (Fig. 1) were mutagenized with Tn5 and nine new Tn5 insertions within or flanking the *ntrC* and *ntrYX* regions were introduced into the ORS571 chromosome via gene replacement (I34, I284, I289, XY126, YX1215, YX1219, X2816, I2852; Fig. 1). Tn5 insertion I15 was constructed as described for C6 and C7 (Pawlowski et al. 1987).

#### Construction of miniMu-lac gene fusions
Genes were subcloned in pJRD184 and mutagenized in *E. coli* MC4100(Mucts)(MudIIIPR46) as described by Ratet et al. (1988a). After infection of Mu8820(*Muc⁺*), Lac⁺ colonies (blue on medium containing 5-bromo-4-chloro-3-β-D-indolyl-thiogalactoside, X-Gal) were selected for further analysis and the miniMu-lac insertion site was mapped by restriction endonuclease analysis. In order to construct a *ntrC-lac* gene fusion, plasmid pPR66 (Fig. 1) was mutagenized, a Lac⁺ derivative carrying MudIIIPR46 within the *ntrC* locus was selected (pCmM4; Fig. 7) and shown to be unable to complement ORS571C6 for growth on nitrate. An *XhoI-HindIII* fragment of pCmM4, carrying the gene fusion but lacking the S-end of Mu (Fig. 7), was subcloned into pJRD184, yielding pCmMS4, and cointegrated into the chromosome of strains ORS571, YX1219 and X2816 to generate merodiploid strains (ORS571CmMS4, YX1219CmMS4 and X2816CmMS4). To construct a *ntrX-lac* fusion, a MudIIIPR46 insertion in the 3′ end of *ntrX* on pRSX31 (Fig. 1) was selected (pXmM3) and a HindIII-*XhoI* deletion derivative was generated for stable cointegration (pXmMS3; Fig. 7). The DNA sequence of the MudIIIPR46 insertion site was determined
Table 1. Bacterial strains, phages, and plasmids

| Strain | Relevant characteristics$^a$ | Reference |
|--------|-----------------------------|-----------|
| **A. caulinodans** |  |  |
| ORS571 | Wild type; Cb<sup>R</sup> | Dreyfus et al. (1988) |
| ORS571A5, A7 | *nifA*::Tn5; Cb<sup>R</sup>, Km<sup>R</sup> | Pawlowski et al. (1987) |
| ORS571C6, C7 | *ntrC*::Tn5; Cb<sup>R</sup>, Km<sup>R</sup> | Pawlowski et al. (1987) |
| ORS571C46 | *ntrC*::MudIIPR46, phenotype like ORS571C6; Cb<sup>R</sup>, Gm<sup>R</sup> | P. Ratet, unpublished |
| ORS571I15 | Tn5 insertion upstream of *ntrC*; Cb<sup>R</sup>, Km<sup>R</sup> | This work and Pawlowski et al. (1987) |
| ORS571I34, I284, I289 | Tn5 insertions between *ntrC* and *ntrYX*; Cb<sup>R</sup>, Km<sup>R</sup> | This work |
| ORS571YX126 | Tn5 insertion in *ntrY* locus; Cb<sup>R</sup>, Km<sup>R</sup> | This work |
| ORS571YX1215, YX1219 | *ntrY*::Tn5; Cb<sup>R</sup>, Km<sup>R</sup> | This work |
| ORS571X2816 | *ntrX*::Tn5; Cb<sup>R</sup>, Km<sup>R</sup> | This work |
| ORS571Y5 | *ntrY*::Km<sup>R</sup>; Cb<sup>R</sup>, Km<sup>R</sup> | This work |
| ORS571C46Y5 | *ntrY*::Km<sup>R</sup>, *ntrC*::MudIIPR46; Cb<sup>R</sup>, Km<sup>R</sup>, Gm<sup>R</sup> | This work |
| ORS571I2852 | ORF-2::Tn5; Cb<sup>R</sup>, Km<sup>R</sup> | This work |
| ORS571A54 | *nifA*<sup>+</sup> *nifA*::MudIIPR46'; Cb<sup>R</sup>, Gm<sup>R</sup> | Ratet et al. (1989) |
| ORS571CmMS4 | *ntrC*<sup>+</sup> *ntrC*::MudIIPR46'; Cb<sup>R</sup>, Gm<sup>R</sup> | This work |
| YX1219CmMS4 | *ntrC*<sup>+</sup> *ntrC*::MudIIPR46', *ntrY*::Tn5; Cb<sup>R</sup>, Km<sup>R</sup>, Gm<sup>R</sup> | This work |
| X2816CmMS4 | *ntrC*<sup>+</sup> *ntrC*::MudIIPR46', *ntrX*::Tn5; Cb<sup>R</sup>, Km<sup>R</sup>, Gm<sup>R</sup> | This work |
| ORS571XmMS3 | *ntrX*<sup>+</sup> *ntrX*::MudIIPR46'; Cb<sup>R</sup>, Gm<sup>R</sup> | This work |
| ORS571C6XmMS3 | *ntrX*<sup>+</sup> *ntrX*::MudIIPR46', *ntrC*::Tn5; Cb<sup>R</sup>, Km<sup>R</sup>, Gm<sup>R</sup> | This work |
| **E. coli** |  |  |
| HB101 | *recA* strain for cloning experiments; Sm<sup>R</sup> | Boyer and Roulland-Dussoix (1969) |
| TB1 | F' (*traC36*, *proAB*, *lacF*, *lacZΔM15*), Δ(*lac*, *pro*), *supE*, *thi*, *recA*, *srl*::Tn10 (Tc<sup>R</sup>), used for M13 infection and *lacZ* α-complementation | B. Barrell, unpublished |
| MC4100(Mucts) | MC4100(Mucts) lysogen for mini-Mu MudIIPR46 lysate | Ratet et al. (1988a) |
| (MudIIIPR46) |  |  |
| M8820 (Muc<sup>+</sup>) | Recipient strain for mini-Mu transduction | Casadaban (1975) |
| **Plasmid/phage** |  |  |
| pLAFR1 | *cos*, Tra<sup>-</sup>, *mob*<sup>+</sup>, IncP; Tc<sup>R</sup> | Friedman et al. (1982) |
| pRK2013 | *mob*<sup>+</sup>, Tra<sup>+</sup>, IncN; Km<sup>R</sup> | Ditta et al. (1980) |
| pRK290 | *mob*<sup>+</sup>, Tra<sup>-</sup>, IncP; Tc<sup>R</sup> | Ditta et al. (1980) |
| pWB5 | pRK290 derivative containing *nptI* and polylinker; Tc<sup>R</sup>, Km<sup>R</sup> | W. Buikema and F.M. Ausubel, unpublished |
| pACYC184 | *mob*<sup>+</sup>, Tra<sup>-</sup>, IncW; Tc<sup>R</sup>, Cm<sup>R</sup> | Chang and Cohen (1978) |
| pJRD184 | *mob*- , Tra<sup>-</sup>, IncN; Tc<sup>R</sup>, Ap<sup>R</sup> | Heusterspreute et al. (1985) |
| pUC18/19 | *mob*- , IncN; used for cloning and plasmid sequencing; Ap<sup>R</sup> | Yanisch-Perron et al. (1985) |
| M13mp18/19 | used for cloning and single strand DNA sequencing | Yanisch-Perron et al. (1985) |
| pPR66 | 6.8 kb *XbaI-EcoRI* fragment of pLRSC1 cloned in pJRD184; Ap<sup>R</sup> | P. Ratet, unpublished |
| pPR54 | ORS571 *nifA*::MudIIIPR46' fusion cloned in pJRD184 for cointegration into the ORS571 chromosome; Tc<sup>R</sup>, Gm<sup>R</sup> (ORS571) | Ratet et al. (1989) |
| pLRSC1 | pLAFR1 derivative; Tc<sup>R</sup>; ORS571 *ntrC-ntrYX* region | Pawlowski et al. (1987) |
| pLRSC1Δ3, 12, 28 | EcoRI deletion derivatives of pLRSC1 | This work |
| pLRSC1Δ34 | Tn5 insertion in pLRSC1Δ3 | This work |
| pLRSC1Δ126, 1215, 1219 | Tn5 insertions in pLRSC1Δ12 | This work |
| pLRSC1Δ281, 2816 | Tn5 insertions in pLRSC1Δ28 | This work |
| pRSX16 | 3 kb *EcoRI* fragment of pLRSC1 cloned in pACYC184; Tc<sup>R</sup> | This work |
| pRSX16-Km<sup>R</sup> | Fragment carrying the *nptII* gene of Tn5 cloned in the *BglII* site of pRSX16; Tc<sup>R</sup> | This work |
| pWB-Y5 | Insert of pRSX16 containing the *nptII* gene of Tn5 subcloned in pWB5; Km<sup>R</sup>, Tc<sup>R</sup> | This work |
| pRSX31 | 9.4 kb *ClaI-SmaI* fragment of pLRSC1 cloned in pJRD184; Ap<sup>R</sup> | This work |
| pCmMS4 | ORS571 *ntrC*::MudIIIPR46' fusion (pCmM4) subcloned in pJRD184 for cointegration into the ORS571 chromosome; Tc<sup>R</sup>, Cm<sup>R</sup> (*E. coli*), Gm<sup>R</sup> (ORS571) | This work |
| pXmMS3 | ORS571 *ntrX*::MudIIIPR46' fusion (pXmM3) subcloned in pJRD184 for cointegration into the ORS571 chromosome; Tc<sup>R</sup>, Cm<sup>R</sup> (*E. coli*), Gm<sup>R</sup> (ORS571) | This work |

$^a$ The phenotypes of ORS571 mutant strains are listed in Table 2

Fig. 1. Physical and genetic map of the ORS571 ntrBC-ORF-1-ntrYX-ORF2' region.

nifA/ntrC hybridizing regions are shown in black. The start point of the ntrC gene has not been determined by DNA sequencing, but is based on our studies on Tn5 insertion mutants (e.g., C7 and I15) as well as the published lengths of ntrC genes from other rhizobia (Sze- to et al. 1987; Nixon et al. 1986), and is therefore denoted by a dotted line in the expanded map. The position and extent of the putative ntrB gene (ntrB*) are hypothetical and based on partial sequence information (unpublished) and the position of the Tn5 insertion I15. The vertical arrows indicate the positions of Tn5 insertions and the horizontal arrow (C46) the position and orientation of the MudIIIPR46 insertion (Direction and transcription of the lac operon). The restriction enzyme designations are as follows: B, BamHI; Bg, BglII; C, ClaI (C* is only recognized in E. coli dam strains); E, EcoRI; H, HindIII; M, MluI; P, PstI; P2, PvuII; Sp, SphI; S1, SstI; S2, SstII; Xb, XbaI; Xh, XhoI

(see Fig. 2) and the ntrX-lac fusion was cointegrated into strains ORS571 and C6, to yield strains ORS571XmMS3 and C6XmMS3, respectively.

**Construction of ORS571C46.** Cosmid pLRSC1 (Fig. 1) was mutagenized with MudIIIPR46 (Ratet et al. 1988a) and the resulting pLRSC1::MudIIIPR46 plasmids were conjugally transferred to ORS571. Transconjugants were selected on gentamycin (MudIIIPR46 marker). Due to the high degree of instability of pLRSC1::MudIIIPR46 plasmids in ORS571 (P. Ratet, unpublished observation), predominantly integrations of these plasmids into the chromosome were obtained. A double crossover at the resident ntrC locus, resulting in a chromosomal ntrC::MudIIIPR46 insertion mutation, was identified by examining growth on nitrate and ammonium as sole N-sources. Thus strain ORS571C46 was obtained, carrying an MudIIIPR46 insertion immediately adjacent to ntrC::Tn5 C7 (Fig. 1).

**Construction of ORS571Y5 and C46Y5.** The BglII-SmaI fragment of Tn5, containing the neomycin phosphotransferase gene lacking its promoter and part of the bleomycin resistance gene (Beck et al. 1982; Genilloud et al. 1984; see Fig. 1) was inserted into the BglII site of pRSX16. The EcoRI insert of the resulting plasmid, pRSX16-Km, was subcloned in pRK290 to form pRK-Y5, conjugally transferred to ORS571 and used to construct a chromosomal ntrY insertion by gene replacement (ORS571Y5; Fig. 1). The ORS571C46Y5 double mutant was constructed by transferring plasmid pRK-Y5 into strain ORS571C46 and selecting for transconjugants resulting from a gene replacement event.

**Enzyme assays.** Nitrogenase activity was determined using the C₂H₂ (acetylene) reduction assay, as described by Pawlowski et al. (1987). Galactosidase activity of free-living bacteria was estimated on plates containing 5-bromo-4-chloro-3-indoly-β-D-galactoside (X-Gal) or determined by quantitative β-galactosidase assays, as described by Miller (1972) and Pawlowski et al. (1987). In situ staining of nodules for β-galactosidase activity was performed by incubating nodule segments in a 1:3 mixture of X-Gal:Z buffer (Miller 1972).

**Plant experiments.** Nodulation and symbiotic nitrogen fixation were examined as described by Pawlowski et al. (1987).

**Results**

**Identification and sequencing of a second ntrC-homologous region**

The DNA sequence of an ntrC/nifA homologous region on cosmid pLRSC1 (ntrX; Fig. 1; Pawlowski et al. 1987; de Bruijn et al. 1988) was determined and the results are shown in Fig. 2. In the ntrX region, an open reading frame (ORF) was identified, preceded by a second open reading frame, which was designated ntrY. Two additional ORFs were found flanking the ntrYX region, designated ORF-1 and ORF-2 (see Figs. 1 and 2). The 3' region of the ntrC gene was also sequenced. All five ORFs represent protein coding regions with a >90% probability, as determined by analyzing the sequence with the 'testcode' (Fickett 1982) and 'codonpreference' (Gribskov et al. 1984) programs (data not shown).

The ntrY ORF (positions 2745–5060) encodes a 771 amino acid (84.26 kDa) protein. It contains three methionines in its N-terminal region, only one of which is preceded by a putative Shine-Dalgarno sequence (positions 2729–2733; Shine and Dalgarno 1974). The C-terminal region of the NtrY shows significant homology with the conserved C-termini of sensor proteins of two-component regulatory systems (see Fig. 3; Albright et al. 1989). An analysis of the hydropathy plot of NtrY revealed three strongly hydrophobic regions, two of which (Fig. 4, arrows) share homology with transmembrane domains of E. coli and Salmonella typhimurium chemoreceptors.

1 TATTTCTCCGGCTTTGGCGAGGATCTCCGGCCTCCGGGCTCTATCACCGCATCCTGCGG
Ac NtrC Y F S G F G E D L P P P G L Y H R I L R
Rm NtrC H G S G F P N G V P P P G L Y H R I L K
61 GACGTGGAATATCCGCTGCTCTCGGCCGCCCCTCGGAGCCACGGGGGCAACCAGATCAAG
D V E Y P L L S A A L A A T R G N Q I K
E I E I P L L T A A L A A T R G N Q I R
121 GCAGCGGAGCTGCTCGGTCTGAAACGCAAACACCCCTGCGCAAGAAGATTCCGGGACCTCGAC
A A E L L G L N R N T L R K K I R D L D
A A D L L G L N R N T L R K K I R D L D
181 ATCCAGGTGATCCGCACCAGCGCTGAACGGACATCTCCTTGATGGCGTACGGGATGTC
I Q V I R T S R *
I Q V Y R S G G *
241 TGTCACCCCTCCGGTTGTAAGGGGCCTGCATCGACGGCATCGGGGTGATGCGGGCATTCA
301 GGATGTGGTATCAAAATGCCGGGGGCATGCGGGCCTCCGGGGGGAAGGCATGGCGGA
ORF-1
M A D
361 CGAGGCGGAGCAGGCCCCGCTGCGCATACTGGGGGACGTCCCTTCCGGGGCCCATTGAGC
E A E Q A G L R I L G D V L S A A A H S A
421 CGGCGACCTCTCGTGCTCTCTCCGACCCTCCTCGCGGTATCGCTGACACCTTCCATCT
G D L S C L S P D L L A R I A D T F H L
481 CCAGCGGGTTTCGCTTTTCAGGTGCACCGAGGGCAGGGGCATCGCGGGCCACCTG
Q R V S L F Q V H E A E G R G I A A T C
541 CGTGATCGATTGGCGGGCCGGGCTCGCCTTTCCGGCCATGACGGAGGGGTCCCATCC
V I D W R R P G L A F P A M T E G S H P
601 GCCGCTCACGGCGGCGGAGGTGATCCCCCTCTGGCGCAATGGGCGGCGACGGCGCCG
P L T A A G G D P L L A Q W A A R R R R
661 CGGCGAGACGATCGGGCCGGACGGCGATCTACCGGCTATCTGTACGGCTTCTTTTC
G E T I I G R T R D L T G Y L Y G F F S
721 CCACTATGGCGTCGTCAACGTTCCTGACCGAACCGGTCATGGGCGCTGGTGGGGG
H Y G V V T F L T E P V M V H G R W W G
781 CCACTTCTGCGTGGAACGCGGATGCCGAAACATGAGTGGACGGCGGTGAGCGGGCAGGC
H F C V D T P D A E H E W T A V E R Q A
841 CTTCAAGTGCATCGCCGCGGTGCTGGGGGGCTCGGGCCCGCTGGGCAACGAAAGGTCT
F K C I A A V L A G L L A R S G T E G L
901 GGTCAGCGAGGCGCGCCGGGCATGCTGACACCTCCATCGATGCGGTGATCGTGGC
V S E A A R R A M L D T S I D A V I V A
961 CGACGAGGCGGGCCCATCGTGAGTTCAACCACGGGGGAGGCCATCTTGGTCACAC
D E A G A I V E F N H A A A E A I F G H T
1021 GCGCGAGGGGGTGATCGGGCCCGCCATGACCGAGACGATCATCCCCGGCCATTACATCGA
R E G V I G R P M T E T I I P A H Y I D
1081 CCGCCAACGGCCAGGGATTCATGCGCCCATCTGGGACCCGGAGAACACACATCATCGCGC
R H R Q G F M R H L A T G E N H I M R R
1141 GCTCGTCGAGGTGGAGGCGCTGCGGGGATGGCAGCGTCTTCCGGCCGAACCTGACCGT
L V E V E A L R A D G S V F P A E L T V
1201 GAACGAGCATCGCCCGGGTGGGCGCCCTGTTCAAGCGCCTTCGGCAGCATCTCCGA
N E H R A G G R R L F S A F V R D I S D
1261 CCGCATCACGTCCCGCGCGCTGGAGCGGCTGGCGTTCAACGGACATGCAACCCGGGCT
R I T S R R A L E R L A F T D M H T G L
1321 CAGCAACCGCACCCGGGCTGCGGGCTGTGCACCGGACGGCCCGCCCTCCGGCGC
S N R T G L L R L C T G R P T R P S G A
1381 GGTGGTGCTGATCGCTGCGGACCTCGGTGTGGTCAAGACGAGCTTCGGGAGACGTGGGC
V V L M L R D L G V V K T S F G D D W A
1441 GGAGCCCATGATCGTGGAGACCGCCAACCTGCTCAGCCGGATGCTCCCGCAGGGAGCGTG
E P M I V E T A N L L S R M L P Q E A C
1501 CCTGGTGCGACCGGCGAGAGCGAGTTTCAGGGTGCTGACGGGGGAGCGGGCGGGGC
L G R T G E S E F T V V T W Q P G A A A
1561 GGAACTCGCGGAAACGCTCATCGGGGCCCTGCGCTCGGGCATCGAGAGCGGGCGGGGCG
E L A E T L I G R L R S A I E S G G R R
1621 CTTCTATCTTCGGGTGCTCGGGGTGGTGGAGGGGCCGGGGGATGCCACCTATCTCCT
F Y L R V G L G V V E R P G D A T Y L L
1681 GCGGGATGCGGAAATGGCAGCGCGGACTGCGGGGACGGCCACCTGCTGCACTTCGCGGA
R D A E M A A R D C R D G H L L H F A E
1741 ACACATGCGGGGCGCAGCAGCGGCTGGGAAGATGGCCCTCGGGGACGTCAT
H M R A Q H Q R L E L E M A L R D V I
1801 CCAGCGGGGACATCGGGGCTCTCGCTCCATTACCAGCGGGTGGTCAGCGCCCCGGACCGG
Q R R T S A L S L H Y Q P V V S A R T G
1861 CGGGCTTGTGCGTTTCGAGGCTCTGGTGCAGCTTCCGAGAGACGCACGGTCCGGGTCTC
G L V G F E A L V R W Y S E T H G P V S
1921 GCCCGCATTGTTCGTGCTCTGGCGGAAGCGGGAGGGTTCCGGGAGCGGGCTGGGGGGCCTG
P A L F V P L A E A G G F A E R L G A W
1981 GGTCATCGAGACCGCCATCTCCGGCCTGGCGCCGGCTGGAATGTCCGGCGGCGGCGCATGG
V I E T A I S A C A G W N V R R A H G
2041 GCTGGCCCCCTGGCACATCGCCATCAACCTTGGCCACGGAGGTGGTGGCGCCGGACCT
L A P W H I A I N L S A T E V V A P D L
2101 CATCGAGGGGTGCGCCAGACGATGGCCTTTCACGGGCTCCCGCCCCAGTGCCTCTGTTT
I E R V R Q T M A F H G L P P Q C V C F
2161 CGAGCTGACCGAAAGCGCCATCCTGAACCGCCCGGAGATCGCCATCGAGACCTCTCGCG
E L T E S A I L N Q P E I A I E T L S R
2221 CCTGCGCGCCCTTGGCTGCACCACGGCCATCGACGATTCGGCACCGGCTATTCGAGCCT
L R A L G C T T A I D D F G T G Y S S L
2281 CAGCTATCTCCAGGGTCTGCCCATGGAGTGTGAAAGATTGACCGCAGTTTCGTCTCTGA
S Y L Q R L P M D V L K I D R S F V L D
2341 CATGGTGGACAACAGCCGTTCGGGGAGATCGTGGGGTCATGATCGAGATGGCCCAACGG
M V D N S R S R E I V R V M I E M A H G
2401 CCTCGGCATGAGCGTGGTGGGCGGAGGGCGTTGAGACCACGGCGCCTTGCGAGATCCTGCG
L G M S V V A E G V E T T G A L Q I L R
2461 TCAGATGGGCTGCGACCGCGCCAGGGATTCCCTCTTGGCCGTGCCATGCGGGGGGATGT
Q M G C D R A Q G F L F G R A M P G D V
2521 GGCGGGGACCCCTGCGGAGACACTCGCTCCCACGGTTAGGTGTTGGGACCGGGAAACAAT
A G T L P E T L A P T G *
2581 CTGCCGTTGCTGCTGGTGCCTGTCACTTTTGCACCAGTGTCGTCCACTGCATCACTG
2641 CCGAACGACTCAGGGGCTGCGGGCCGACCGCATGGCCCTGACGACGGGCAACAGCGGC
2701 AGGCCCGGACGACCGGCCCCCGCGTGCAGGAGTGCGGTGACACATGACGCAAGCCGGT
Ac NtrY
M T Q A A F
2761 TTGACCAGGCTTCGGACAACGGACCCCATGACGCGTCCGGCTCGTCCCTGGCCCTGTTCG
D Q A S D N G P M T P S G S S F G L F A
2821 CCCGGCAGTGGTGCTGCTCGCCCTCATCTCGGCTCTCGCCACCTTCCCATCCTCATGG
P A V V L L A L I S A L A T F L I L M G
2881 GCCTCACCCCCGTGGTGCCGACCCTCATCGGTGCTCAGCGTGCTGCTGGTGAATGCGG
L T P V V P T H Q V V I S V L L V N A A
2941 CAGCGGTGCTGATCCTCAGCGCCATGGTGGCCGAGATCTGGCGCATCGCCCAAGGCC
A V L I L S A M V G R E I W R I A K A R
3001 GGGCGCGGGGCGGCGGCGCCGGGCTGCGCACATCCGCATTGTGGCCTGTTCGCGGTGG
A R G R A A A R L H I R I V G L F A V V
3061 TCTCCGTGGTGCCGCCATCCTCGTGGCGGTGGCTAGCCTGACGCTCGACCGCGGGC
S V V P A I L V A V V A S L T L D R G L
3121 TCGACCGCTGGTTTCCATGCGCACGGAGACGTGGCGAGTTCCGTCTCCGTGCCC
D R W F S M R T Q E I V A S S V V A Q
3181 AGACCTATGTGCGCGAGCACGCCCTGAACATCCGGGGCGACATCCTCGCCATGAGCGCGG
T Y V R E H A L N I R G D I L A M S A D
3241 ACCTGACGCGGCTGAAGTGGTCTATGAAGGGGACCGCTCGCGCTTCAACCAGATCCTCA
L T R L K S V Y E G D R S F N Q I L T
3301 CTGCGCAGGCGGCGCTGCGGAAACCTGCGGGGCGCCATGCTGATCCGCGGCGACCTCTCGG
A Q A A L R N L P G A M L I R R D L S V
3361 TGGTGGAGCGGGCCAACGTCAACATCGGCGGAATTTCATCGTCCCCGCAAACCTCGCCA
V E R A N V N I G R E F I V P A N L A I
3421 TTGGGGATGCGACGGGGATCAGCCGGGTATCTGCCCAAATGACGGGACTATGTGG
G D A T P D Q P V I Y L P N D A D Y V A
3481 CCGCCGTGGTGCCGCTCAAGGACTATGACGACCTCTATCTACGTGGCCCGCCTCATCG
A V V P L K D Y D D L Y L Y V A R L I D
3541 ATCCGCGGCTCATCGGCTATCTGAAAGACCACGCGAGGAGCGCTGGCCGACTATCGGTGCC
P R V I G Y L K T T Q E T L A D Y R S L
3601 TGGAGGAGCGGCGCTTCGGCGTGCAGGTGGCCCTTCGGCCTCATGTATGGGTCATCACGC
E E R R F G V Q V A F A L M Y A V I T L
3661 TCATCGTGCCTCTCCGCGGCTCTGGGCTCAACTTCTCCAAAGTGGCTGGTGGCGC
I V L L S A V W L G L N F S K W L V A P
3721 CCATCCGCGCCCTCATGTCCGGCAGCJGACCATGTGGCGGAAGGCAATCTCGACGTAOGCG
I R R L M S A A D H V A E G N L D V R V
3781 TGCCCACCTATCGGGCCGAAGGCGATCTCGCCCTCGCCGAGACCTTCAACAAAGATGA
P I Y R A E G D L A S L A E T F N K M T
3841 CCCACGAGCTGCGCAGCGCGGAGGCCATCCTCACGGCCCCGGAACCAGATCGACAGCC
H E L R S Q R E E A I L T A R D Q I D S R
3901 GCCGCGGCTTACGGAAGCCGTGTTTCCGGGTGTCGGCGGGCGTCACTGGCCTCGGATT
R R F T E A V L S G V G A G V I G L D S
3961 CGCAGGAGGCATCACCACCTCTCAACCGGCTCGGGGAGACGGGCGGGCCTTGGGCTGTGGAGG
Q E R I T I L N R S A E R L L G L S E V
4021 TGGAGGCCCCTGCACCGGCATCTCGCCGAGGTTGCGGCGGGCCTCCTGGAAG
E A L H R H L A E V V P E T A G L L E E
4081 AGGCCGAGCATGCGGGGCAGCGTTCAAGGCAACATCACGCTCACCCGCGACAGGGG
A E H A R Q R S V Q G N I T L T R D G R
4141 GTGAGCCGGTCTTCCGGGTCCGGTGCACCACCGAGCAATCGCCCCGAGGCGGCATGGGCT
E R V F A V R V T T E E Q S P E A E H G W
4201 GGGTGGTGACGCTCGACGACATCACCAGCGCCCAGCGCACCTCTGGCCTGGG
V V T L D D I T E L I S A Q R T S A W A

4261 CGGATGTGGCCCCGGCCATCGCCCACGGAGATCAAGAACCCGCTCACC CCCATCCAGCTCT
D V A R R I A H E I K N P L T P I Q L S

4321 CCGCCGAGCGGCTCAAAGCGCAAGTTCGGCCCAGTGACGGCAGGATCGGGAGATCTTCG
A E R L K R K F G R H V T Q D R E I F D

4381 ACCAGTGCACCGACACCATCATCGTCAAGTGGGCGACATCGGCCGGATGGTGGACGAGT
Q C T D T I I R Q V G D I G R M V D E F

4441 TCTCCTCCTTCGCCCCGGATGCCCAAAGCCCGTCGTGGACAGCCAGGACATGTGGGAGATCA
S S F A R M P K P V V D S Q D M S E I I

4501 TCCGCCAGACGGTTT CCTCATGGGGTGGGACATCCCGAGGTGGTTGTTTGATTCGGAGG
R Q T V F L M R V G H P E V V F D S E V

4561 TGCCGCCC GCTATGCCC GCGC GTTCGACCGCCGCTCGTTTCCAAGCCTTAACGAACA
P P A M P A R F D R R L V S Q A L T N I

4621 TCCTCAAAGAACGCTGCGAGGCCATCGAGGCGTTCCGCGGACGTACGGGCAAAGGCC
L K N A A E A I E A V P P D V R G Q G R

4681 GCATCCGCGTCAGCGCCAATCGGGTGGGTGAGGATCTGGTGATCGACATCATCGACAACG
I R V S A N R V G E D L V I D I I D N G

4741 GCACCGG CCTCGCGCAGGAGAGCCGGAACC GTCTTCTGGAACCTATGTGACGAGCGCG
T G L P Q E S R N R L L E P Y V T T R E

4801 AAAAGGGCACGGG CCTCGGCCCTTGGCCATCGTGGGAAAGATCATGGAGGAGCACGGCGGCG
K G T G L G L A I V G K I M E E H G G G

4861 GCATCGAGACTGAACGACGCGCCCCGAGGGGCGCGCGGTGGATCCGGTCTCACCCTCAAGG
I E L N D A P E G R G A W I R L T L K A

4921 CCGAGGGACC GAAAGCGGAACCGATG CCTCAACCAAGGCGACCGGGGCTGCCACGC
E G P K A E P T D A S T K A T G A A T P

4981 CC GCGGCGCCCGGCTTCCGCCATGGCCC GCGATGCGGCTGCCGATTCCGCGCGCGCG
A A P A A S A M A R D A A A D S A A R G

5041 GCAAGAACG AGAGGA CCTGATCCATGGCCC ATGACATCCTGATCGTGATGACGAGCCC G
Ac NtrX K N E R T * M A H D I L I V D D E P D

5101 ACATCAGCGG CCTCGTGGCCGCATCCTGGAGGACGAGGGCTATTCGGCCC CGCACCCGCCC
I S G L V A G I L E D E G Y S A R T A R

5161 GCGAGCCGATGGCGCGCTGGCCGAGATCGCCGCGGCGGAACCTGATCTT CCTCG
D A D G A L A E I A A R R P N L I F L D

5221 ACATCTGGCTGCAGGGCAGCCG CCTCGACGGCCTCGAATTGCTGGACATCATCAAGCGCG
I W L Q G S R L D G L E L L D I I K R E

5281 AGCACC CCGAGGTGCCC GGTGGTGATGATCTCCGGCCA CGGCAACATCGAGACGGGGTGG
H P E V P V V M I S G H G N I E T A V A

5341 CCGCCATCAAGCGCGGGCGCCTACGAGAAAGCCCTTCAATGCCGACCCGCTGG
A I K R G A Y D F I E K P F N A D R L V

5401 TGGTCATCACCGAGCGGGCGCTGGAGACGCTGGGCGCGGAGGTACGGGAGCTGA
V I T E R A L E T L R R R E V R E L K

5461 AGCAGCTCACCCAGCCGCAACCCATGGTGGGCGCTCCAGGTCATCCAGCAGTTGCGCG
Q L T Q P H T M V G R S S V I Q Q L R A

5521 CCACCGTGGACCGAGTCGGCCCCACCAACAGCCGATCCTCATCGTGGCCCCCTCGGGCT
T V D R V G P T N S R I L I V G P S G S


Fig. 2. DNA sequence of the ORS571 ntrC'-ORF-1-ntrYX-ORF-2' region. The 3' end of ntrC extends to position 207, and the amino acids identities between ORS571 NtrC (top amino acid sequence) and R. meliloti NtrC (bottom amino acid sequence; Szeto et al. 1987) are indicated by vertical lines. ORF-1 extends from position 353 to 2560, ntrY from position 2745 to 5060, ntrX from position 5064 to 6427. ORF-2 begins at position 6549. The putative ribo-

ceptor proteins (Krikos et al. 1983; Russo and Koshland 1983; data not shown), suggesting that NtrY may be a transmembrane protein (see Nixon et al. 1986).

The ntrX ORF (position 5064–6428) encodes a 455 amino acid (50.2 kDa) protein, preceded by a putative Shine-Dalgarno sequence (positions 5050–5055). NtrX shows a high degree of homology with NtrC proteins of different organisms (Fig. 5). Therefore the ntrYX loci reveal typical features of a two-component regulatory system, in which NtrY would serve as the sensor and NtrX as the modulator protein. Upstream of ntrY (between ORF-1 and ntrY), two motifs similar to the –24/ –12 consensus sequences found in NtrA-dependent pro- moters (–24GG/GC–12; Thöny and Hennecke 1989) were identified (GG-N10-GC; positions 2663–2676 and

5581 CCGGCAAGGAGCTGACC GCGCATGATCCATGCGG CCTCGCCC GCGCAGGG GCCGT
G K E L T A R M I H A A S A R A Q G P F

5641 TCGTGGTCATCAAACGGCCGCCATCACC CCGAGCGCCTCGAATATGAGCTGTTCGGGC
V V I N A A A A I T P E R L E Y E L F G V

5701 TGGAGGAGGGGGAAGGGCGCGAGCGCCATCGCGGCGCGCTGGAAGAGGCGCACGGCGGCA
E E G E G R E R H R G A L E E A H G G T

5761 CTCTGTTCTTCGACGAGATCGCCGACATGCGCGCGAGACCCAGAAACGGGTGCTGCGGGG
L F L D E I A D M P R E T Q N R V L R V

5821 TGCTGGTGGAGCAGACCTTCAGCCGCATCGGCAGCAGCGAGAAGGTGGCGGTGGACGTGC
L V E Q T F S R I G S S E E K V R V D V R

5881 GCATCATCTCCTCCACCGGGCGTCA CCTGGAGGAAGAGATCGCCGCGGGGCGTTTCCGCG
I I S S T G R H L E E I A A G R F R E

5941 AGGATCTCTACCACCGCTTTCGGTGGTGGCATCCGCGTGCCGCGCTGGCCGAGCGGGC
D L Y H R L S V V P I R V P P L A E R R

6001 GGGAGGACATCCCCGATCTCGGTGATTTCTTCATCGACCTCATCTCTCAGACGACGGGGCC
E D I P D L V D F F I D L I S Q T T G L

6061 TTCAGCGCCGCAAGGTGGGCGAGGATGCCATGGCGTCTCCAGTCCCATGACTGGCCGG
Q R R K V G E D A M A V L Q S H D W P G

6121 GCAACGTGCGGCAGCTGCGCAAACATGTGGAGCCCTGCTGATTCTGCGGCGGGCGGATC
N V R Q L R N N V E R L L I L A G G D P

6181 CGGATGCGGAGGTGACCGCCTCCATGCTGCGGCGGACGTGGGTGCGCTGGTGCCCACCC
D A E V T A S M L P P D V G A L V P T L

6241 TGCCCAACGGCAATGGCGGAGCATCTGATGGGCGCTGCGGTGAGGCCC CGGAAG
P N G N G E H L M G L P L R E A R E V

6301 TGTTCGAGCGCGAAATATCTCGCAGCGCAGATTAACCGCTTTGGCGGCAATATCTCGCGTA
F E R E Y L A A O I N R F G G N I S R T

6361 CGGCGGAATTCGTGCGGCATGGGAACGCTCGGCCCCTGCATCGCAAAGCTGAAGGCGCTCGGGC
A E F V G M E R S A L H R K L K A L G V

6421 TAGGCTGACGGAGCGGGACGGGATAAGGTTCGCCCCGGCGGGTGTTCGAGGCGGCAG
G *

6481 CCGCGCCGGGAAAACGGGATCGGCATTCG GGCCAAGGGCTTGGGGGTGTCAGGGCGGGG

6541 GGGTGGTCATGAAGT CGTGATCTGCGGGGCAGGGCAGGTGGGGTTTCGGCAT TGGCGAGC
ORF-2 M K V V I C G A G Q V G F G I A E R

6601 GCCTCGCCAGCGAGCAGAACGACGTCTCCATCGT CGATGCGGAGCCCCGGCGGATCCAGA
L A S E Q N D V S I V D A S P R R I Q I

6661 TCGCCACC GAGCAGCTCGACGTGCGGGG GTGGGGCCATGGGCTCCCATCGGAGCGTGC
A T D Q L D V R G V V G H G S H P D V L

6721 TGGCCCGTGCCGGCATCGAGCAGGGGACATGCTGATCGGGTCACCCCTG CATGAGCGAAG
A R A G I E Q A D M L I A V T L H D E V

6781 TGAACATGTTGGCCTGCCCAGGTGGGGCCATT CGCTGTTCAACGTGCCGAGCTGCCC GCA T
N M V A C Q V G H S L F N V P T S P A S

6841 CCGCGCCCAGACCTATCTGCGAGCCCCGAATGGCGCAGGGGGA
A P R P I C S P N G A G G

some-binding sites and the probable N-terminal methionines are boxed. Stop codons are indicated by an asterisk. Inverted repeat structures are indicated by horizontal arrows. The putative –24/ –12 promoter elements (ntr boxes) are highlighted in black. The position of the ntrYX-lac fusion is indicated by the horizontal arrow at position 6366

2687–2700). Downstream of ntrX, a palindromic structure, possibly representing a transcription terminal signal, is present (positions 6454–6490). The polypeptide encoded by the 3' part of the ntrC gene shares a high degree of homology with NtrC proteins of other rhizobia, such as R. meliloti (Fig. 2).

The protein encoded by ORF-1 (positions 353–2560; protein 1) shows homology with the product of an open reading frame within the mercury resistance region of plasmid R100 and transposon Tn501 (URF-2; Brown et al. 1986). The physiological significance of URF-2 is unknown, but it is not involved in conferring mercury resistance (Brown et al. 1986). The C-terminal part of the ORF-1 product shows strong homology with the TnpM protein of transposons Tn21 and Tn501, which

EC EnvZ 224 AAGVKQLADDRTLLMAGVSHDLRTPLTRIRLATEMMSEQ.....
EC CpxA 229 VTALERMMTSQORLLSDISHELRTPLTRIQQLCTALL........
Rm FixL 225 ELARLARNEMGEMASTJAHELNQPLSAIANYSHGCTRLLRDMDD
At VirA 455 RLEHAQRLEAVTLAGGIAHEENNILGSILGHAE.LAQNSV....
Bp NtrB 125 QLTHRSAARSVIALAAMLAHEIKNPLSGIRGAAQLLEQQAS....
Ac NtrY 495 ELISAORTSAWADVARRIAHEIKNPLTPIQLSAERLKRKFGRHVT

EC EnvZ ........DGYLAE........SINKDIIECNAT....IE.QFIDYLRTGQEMP
EC CpxA RRSGESKELERIEEAORIDSMLNDLLLVMSRNQQKNALVSETIKANOIW
Rm FixL AVAIRIREALEEVASOSLRAGQTIKHREFVTKGETEKAPEDRKLVEES
At VirA SRTSVTRRYIDYIISSGDRAMLIIDQILLTSRKOER.......MIKPFSVS
Bp NtrB ...SEDRLLTRLICDEADRIIVDRMEVFGDDRPVARP.......PVNCH
Ac NtrY QDREIFDQCTDTIIROVGDIGRMVDERSSFARMPKP.......VDDSQDMS

EC EnvZ MEMADINAVLGEVIAAESGYEREIETAL.YP.....GSLEVKMHPLSIKR
EC CpxA SEVLDNAAFEAEOmgrsLTVNFP..PG.PWPLYGNPNALESALENTVRNA
Rm FixL AALALVG......SREQVRTVFELYLPGAEMVLVDRI.OVOQVLINIMRNA
At VirA ELVTEDAPLLRMALPPNTELSFRF.DQMOSVIEGSPLELOQVLINICKNA
Bp NtrB SVLDHVKRLAQSGFARNVRFIEDYDPSEPVPVLANQ.DQLIQVFINIVKNA
Ac NtrY EIJRQTVFIMRVGHP.EVVFDSEVPPAMPARFDRRLVS...QALTNBKNA

EC EnvZ AVANMVVNAARYGNGWKVSSGTEPNRAW...........FQVEDDGPGJA
EC CpxA LRYSH.T.KIEVGFAVDKD...................GITITTVDDDGPGVS
Rm FixL IEAMRHVDRRELTIRTMBADPGE..............VAVVEDTGGGIP
At VirA SQAMTANG.QIDIIIISQAFLPVKKILAHGVMPPGDY.VLLSISDNGGGIP
Bp NtrB AEAAVADIGTDAEIQLTTAFRPGVRLSVPKKSRVSLPLEFCVKDNGSGVP
Ac NtrY AEATEAVPPDVRGQGRIRVSANRV...........GED..LVIDLIDNGTGLP

EC EnvZ PEQRKHLFQPF..VRGDSARTISGTGLGLAIVQRIVDNHNGMLELGTSER
EC CpxA PEDREQEFRPFYRTDEARDRESGGTGLGLAIVEATAQQHRGWVKAEDSPL
Rm FixL EEVAGOLFKPFVTTKA........SGMGIGLSISKRIVEAHGGEMTVSKNEA
At VirA EAVLPHEEPFFSTRA...RN.GGTGLGLASVHGHTSAFAGYIDVSSTV.
Bp NtrB EDLLPNLFDPFVTTKQ........TGSQLGLALVAKIVGDHGGIIIECESQPR
Ac NtrY QESRNRILLEPYVTT......REK.GTGLGLAIVGKIMEEHGGGIELNDAPE

EC EnvZ G.GLSIRAWLPVPVTR 424
EC CpxA G.GLRIVIWLPLPYKRS 461
Rm FixL GGAT.FRFTLPAYIDE 458
At VirA GHGTRFDIYLPESSKE 696
Bp NtrB KTTERV...LDADVQR 367
Ac NtrY GRGAWRLTLKAEGPK 730

Fig. 3. Comparison of the C-terminal amino acid sequence of NtrY with those of sensor proteins of two-component regulatory systems. *E. coli* EnvZ reacts to the osmolarity of the medium (Comeau et al. 1985); *E. coli* CpxA senses the presence of F⁻ cells and in turn activates ArcA (Albin et al. 1986); *R. meliloti* FixL is a putative oxygen sensor (David et al. 1988); *A. tumefaciens* VirA activates VirG in the presence of acetosyringone or hydroxysyringone (Leroux et al. 1987); *B. parasponiae* NtrB is a nitrogen sensor, whose function is regulated by the GlnB(PII)/GlnD system (Nixon et al. 1986; Magasanik 1988). Identical amino acids at homologous positions are highlighted in black, chemically similar amino acids are stippled and boxed. Chemical similarity was defined according to Gribskov and Burgess (1986)

hydrophobic

![Graph](https://i.imgur.com/placeholder.png)

hydrophilic

Fig. 4. Hydropathy plot of NtrY. The hydropathy profile of NtrY (according to Kyte and Doolittle 1982) is shown and the putative transmembrane regions (see text) are indicated by arrows

plays a role in transposition (Hyde and Tu 1985; data not shown). Tn5 insertions in this locus have a Nif⁺, Fix⁺ and Ntr⁺ phenotype (I34, I284, I289; Fig. 1; Table 2). Therefore, the significance of these homologies and the possible function of the protein encoded by ORF-1 remain unclear. For the protein encoded by ORF-2 (beginning at position 6549), no significant homology to any other protein could be found. This ORF is also preceded by -24/-12 type consensus sequences (e.g. positions 6510–6523; Fig. 2).

Construction of insertion mutations in the ntrC-ntrYX region of cosmid pLRSC1

Ten new Tn5 insertions within or flanking the *ntrC* and *ntrYX* regions were constructed, as described in Materials and methods (I15, I34, I284, I289, YX126, YX1215, YX1219, X2816, I2852; Fig. 1).

Since the DNA sequencing data suggested that *ntrYX* could represent an operon (Fig. 2), Tn5 insertions in *ntrY* were expected to be polar on *ntrX*. Therefore, a

Rm NtrC 1 MTGARILVADDDAAIRTVLNQALSAGYDVRITSNAATLWRWIAAGD
Bp NtrC 1 MPAGSILVADDDTAIRTVLNQALSRAEYVRLTGNAATLWRWVSQGE
Kp NtrC 1 MQRGLAWIVDDDSSTRW.VLERALTGAGLSCCTFESGNEVLDATTKT
Ac NtrX 1 MAHDILIVDDFPDISGVAGTLEDEGYSARTADGALAEIAARR

Rm NtrC GDLVETDVVMPDENA...FDLLPRIKKARPDLVPVMSAQNTFMTAIKAS
Bp NtrC GDLVETDVVMPDENA...FDLLPRIKKMRPNLPVIVMSAQNTFMTAIRPS
Kp NtrC PDVLIISDIRMPC...MDGLALLKQIKORHPMLPVIIMTAHSDIDAAVSAY
Ac NtrX PNIIIFLDIWLQGSR.LDGLELLDIIKREHPEVPVVMSGHNIETAVAAI

Rm NtrC EKGAYDYLPKPFDLTELIIGIIGR...ALAEPK..RRPSKLEDDSQDGMP
Bp NtrC ERGAYSYLPKPFDLKELITIVGRALAEPKERVS....SPADDGEFDSIPL
Kp NtrC QQGAEDYLPKPFDIDEAVALVDRASHYQE....QQOPRNAPINSPTADJ
Ac NtrX KRGAYDFIEKPFNADRLVVITERALETLRLRREVRELKQLT....QPHTM

Rm NtrC VGRSAAMQEIYRVLARLMQTDLTLMIGESGTGKELVARALHDYGKR RNG
Bp NtrC VGRSPAMQEIYRVLARLMQTDLTVMSGESGTGKELVARALHDYGRR RNG
Kp NtrC GERPAMQDVFRTHGRLSSRSSVLINGESGTGKELVAHALHRHS PRAKA
Ac NtrX VGRSSVIOQTRAIVDRVGPTNSRILIVGPSGSGKELTARMTHAASARAQG

Rm NtrC PFVAINMAAIPRDLIESELFGHEKGAAFTGAQTRSHGRFEQAEGGTLFLDE
Bp NtrC PFVANMAAIPRDLIESELFGHERGAFTGANTRASGRFEQAEGGTLFLDE
Kp NtrC PFTAENMAAIPRDLIESELFGHEKGAAFTGANTVRQGRFEQA DGGTLFLDE
Ac NtrX PFVVINA AAITPERBEYEELFGVEEG...ECRERRHRGATEEAHG GTLFLDE

Rm NtrC IGDMPMDAQTRLLRVLQOQGEYTTVGGRTPIRS DVRIVAATNKDLKOSINO
Bp NtrC IGDMPMEAQTRLLRVLQOQGEYTTVGGRTPIKTDVRIVAASNKDLRILLIQO
Kp NtrC IGDMPLEDVQTRLLRVLADGQFYRVGGYAPVKVDVRIIAATHONLELRYQE
Ac NtrX IADM PRETQNRVLRVLVEQTFSRHGSSEKV RV DVRIISS TGRHLEEEIAA

Rm NtrC GLFREDLYYRLNVVPLRIPPLR德拉EDIPDLVRHFYQOAEKE.GLDVKRF
Bp NtrC GLFREDLFFRLNVVPLRVPPLRERIEDPDLIRHFFSLAEK.DGLPPKKL
Kp NtrC GK FREDLHRLNVIRVHLPPLRERREDIPRLARHFLQIAARELGVEAKOL
Ac NtrX GRFREDLYHRLSVVPIRVPPLAERREDIPDLVDFIDLISQTTGLQRRKV

Rm NtrC DQEALEHKAHWPNGVRELENLVRR L...TA.LYPQDVITREIIENELRS
Bp NtrC DAQALERLKQHRWPNGVRELENLARRL...AALYPQDVIT...........
Kp NtrC HPETEMALTRLAWPGNVRQLENTCRWLT.VMA.............AG
Ac NtrX GEDAMA VLOSHDWPGNVRQLRNNVERLLILA GGDPDAEVTASMLPPDVGA

Rm NtrC EIPDSPIEKAAARSGLSISQAVEENMROYFASF GDAIPP............
Bp NtrC ...ASVIDGEL...APPAVTSGST..ATVGVDNLGGAVEAYLSSHFSGFP
Kp NtrC ..QEVLTODLPSELFETAIPDNPTQMLPDSWATLLGOWADRALRSGHQLN
Ac NtrX LVPTLPNGNGGEHLMGLPLREAREVFEREYLA..............

Rm NtrC ........PSGLYDRVLAEMEYPLHLAALTATRGNQIKAADLLGLNRNTLR
Bp NtrC NGVRPPG...LYHRILKETEIPLLTAALAATRGNQIRAADLLGLNRNTLR
Kp NtrC LSEAQP........EMERTLLTTALRHTQGHKQEAARLLGWGRNTLT
Ac NtrX .......................AOINRFCGNISR TAEEVCMERSALH

Rm NtrC KKIRELGVSVYRSLA 482
Bp NtrC KKIRDLDIQVYRSGG 480
Kp NtrC RKFKELGME 470
Ac NtrX RKIKALGVG 436

Fig. 5. Amino acid comparison between NtrX and NtrC proteins. R. meliloti NtrC was described by Szeto et al. (1987), B. parasponiae NtrC by Nixon et al. (1986), K. pneumoniae NtrC by Buikema et al. (1985). The amino acids are labeled as described in the legend to Fig. 3.

non-polar *ntrY* insertion mutant (Y5; Fig. 1) was constructed (see Materials and methods). In order to construct an *ntrCntrY* double mutant, first a MudII PR46 (Ratet et al. 1988a) insertion in *ntrC* was made (ORS571C46; Fig. 1). The phenotype of C46 was found to be the same as that of the previously characterized *ntrC::Tn5* strains C6 and C7 (Pawlowski et al. 1987). To construct the *ntrCntrY* double mutant (ORS571C46Y5), the Y5 insertion was transferred into strain ORS571C46 by gene replacement. The positions of the transposon insertions in these strains were verified by Southern blotting (data not shown).

**Phenotypes of the insertion mutants**

All insertion mutants were analyzed for stem and root nodulation capacity on *S. rostrata* (Nod), for free-living (Nif) and symbiotic nitrogen fixation (Fix), as well as for growth on several nitrogen sources (nitrogen assimilation/regulation; Asm/Ntr phenotype). Their phenotypes were compared to those of previously characterized *ntrC::Tn5* and *nifA::Tn5* mutants (C6, C7; Fig. 1; A5, A7; Pawlowski et al. 1987). The results are summarized in Table 2. ORS571I15, I34, I284, I289 and I2852 showed an essentially Nod⁺, Nif⁺ and Fix⁺ phenotype and exhibited wild-type growth on LSO medium supplemented with glutamine, glutamate, arginine, histidine, leucine, ammonium or nitrate as sole nitrogen (N⁻) sources (Asm⁺, Ntr⁺). These Tn5 insertions therefore do not appear to be located in genes essential for nodulation, nitrogen fixation and metabolism, or their control regions and are therefore labelled “I” (presumably Intergenic; Fig. 1; Table 2).

Strain ORS571YX126 was found to induce stem and

Table 2. Phenotypes of ORS571 Tn5 and MudII PR46 insertion mutants

| Name                | Proposed missing gene product | Phenotype | Nif<sup>a</sup> | Nod Stem | Nod Root | Fix<sup>b</sup> Stem | Fix<sup>b</sup> Root | Ntr<sup>c</sup> |
|---------------------|-------------------------------|-----------|----------------|----------|----------|--------------------|--------------------|----------------|
| ORS571A5, A7        | NifA                          |           | —              | +<sup>d</sup> | +<sup>e</sup> | <1                 | <1             | +              |
| ORS571I15           |                               |           | 100            | +        | +        | 100                | 100              | +              |
| ORS571C6, C7, C46   | NtrC                          |           | 5–15           | +<sup>d</sup> | +<sup>e</sup> | <1                 | del −/+<sup>f</sup> | −/+            |
| ORS571I34, I284, I289 | product of ORF-1             |           | 100            | +        | +        | 100                | 100              | +              |
| ORS571YX126         |                               |           | 100            | +        | +        | 60–80              | 60–80            | +              |
| ORS571YX1215, YX1219 | NtrY, NtrX                    |           | 50–80          | bumps    | +<sup>e</sup> | <1                 | del −/+<sup>f</sup> | +/−            |
| ORS571Y5            | NtrY                          |           | 100            | bumps    | +<sup>e</sup> | <1                 | del −/+<sup>f</sup> | −/+            |
| ORS571C46Y5         | NtrY, NtrC                    |           | 5–15           | bumps    | +<sup>e</sup> | <1                 | del −/+<sup>f</sup> | —              |
| ORS571X2816         | NtrX                          |           | 30–50          | +<sup>d</sup> | +<sup>e</sup> | <1                 | del −/+<sup>f</sup> | −/+            |
| ORS571I2852         | product of ORF-2              |           | ~100%          | +        | +        | 100<sup>g</sup>     | 100               | +              |

<sup>a</sup> Nif phenotype is expressed as the percentage of wild-type acetylene reduction activity  
<sup>b</sup> Fix phenotype is expressed as the percentage of wild-type acetylene reduction activity of four-week-old stem or six-week-old root nodules  
<sup>c</sup> For designation of Ntr phenotype see Table 3  

<sup>d</sup> Small, light green nodules (Pawlowski et al. 1987)  
<sup>e</sup> Hypernodulated (Pawlowski et al. 1987)  
<sup>f</sup> Delayed Fix<sup>−/+</sup> phenotype (Pawlowski et al. 1987)  
<sup>g</sup> Nodulated plants are slightly smaller than those inoculated with ORS571

Fig. 6. Stem nodulation of *S. rostrata* by ORS571, YX1215, YX1219 and X2816. Stem segments of *S. rostrata* plants are shown 7 weeks after infection. For details see text.

Table 3. Ntr phenotypes of $ntrC$, $ntrY$, $ntrYntrC$, $ntrYX$ and $ntrX$ mutant strains

| Strain         | Growth in liquid cultures$^a$ | Growth on plates$^b$ | Ntr phenotype$^c$ |
|----------------|-------------------------------|-----------------------|-------------------|
|                | Glutamine                    | Nitrate               | No N source       | Nitrate            | Ammonium           | Glutamine          |                   |
| ORS571         | $3.09 \pm /-0.29$            | $1.76 \pm /-0.27$     | $0.27 \pm /-0.01$ | Normal             | Normal             | Normal             | +                 |
| ORS571C46      | $3.06 \pm /-0.18$            | $0.34 \pm /-0.02$     | $0.35 \pm /-0.05$ | Slimy              | Normal             | Normal             | $-/+$             |
| ORS571Y5       | $2.97 \pm /-0.20$            | $0.67 \pm /-0.14$     | $0.28 \pm /-0.02$ | Slimy              | Slimy              | Normal             | $-/+$             |
| ORS571C46Y5    | $2.94 \pm /-0.27$            | $0.27 \pm /-0.02$     | $0.27 \pm /-0.02$ | Slimy              | Slimy              | Slimy              | $-$               |
| ORS571X2816    | $2.49 \pm /-0.50$            | $0.98 \pm /-0.14$     | $0.27 \pm /-0.03$ | Slimy              | Slimy              | Slimy$^d$          | $-/+^d$           |
| ORS571YX1219   | $3.19 \pm /-0.24$            | $1.05 \pm /-0.12$     | $0.29 \pm /-0.02$ | Slimy              | Slimy              | Normal$^d$         | $+/-$ $^d$        |

$^a$ 3 ml test cultures were inoculated with washed cells from stationary TY cultures and grown for 3 days at $37^\circ$ C with vigorous shaking. Each number represents the optical density at 600 nm of the culture and constitutes the average from four independent experiments with three cultures each. Standard deviations are given.

$^b$ Normal growth: opaque, white colonies. Slimy growth: translucent colonies producing excess lipopolysaccharides and containing few bacteria.

$^c$ For designation of the Ntr phenotype, growth capacity on nitrate as sole N source on plates and in liquid culture and growth on ammonium as sole N sources on plates were taken into consideration.

$^d$ YX1219 and X2816 could be given the same Ntr designation based on growth on nitrate and ammonium, but X2816 is more strongly impaired in its Ntr phenotype, since it exhibits slimy growth on glutamine as sole N source.

---

sources, allowed the assignment of the different $ntr$ mutants to distinct groups, with increasingly severe Ntr phenotypes ($\text{Ntr}^{+/}$, $\text{Ntr}^{-/+}$, $\text{Ntr}^-$; see Tables 2, 3). Strain YX126 grew like the wild-type strain ORS571 on all N sources tested, except for glutamate ($\text{Ntr}^+$). Strains YX1215, YX1219, Y5, X2816 and C46Y5 exhibited poor growth (formed slimy colonies) on LSO plates supplemented with glutamate, ammonium, nitrate, arginine or leucine (see Table 3). Growth in well aerated liquid cultures with nitrate as sole N source was moderately impaired for YX1215 and YX1219, but substantially impaired for Y5 and X2816 ($\text{Ntr}^{+/}$ and $\text{Ntr}^{-/+}$, respectively; Tables 2 and 3). This effect was not observed in poorly aerated cultures (data not shown). Strains X2816 and C46Y5 exhibited poor growth also on plates with glutamine (Table 3). ORS571C46Y5 showed no growth in minimal medium with nitrate ($\text{Ntr}^-$; Tables 2 and 3). Thus, in ORS571C46Y5 the nitrogen metabolism defects of ORS571C46 and Y5 appear to be combined, suggesting an additive effect of the $ntrC$ and $ntrY$ mutations.

Complementation studies

To examine whether the $ntrYX$ genes form an operon or are transcribed independently, complementation experiments were carried out. Plasmids pLRSC1A28, pLRSC1A281 (carrying a $ntrY::Tn5$ insertion) and pLRSC1A2816 (carrying the $ntrX::Tn5$ insertion X2816; Fig. 1) were introduced into the strain ORS571X2816. Growth of the transconjugants in liquid cultures with nitrate as N source was examined. While pLRSC1A28 could complement the $\text{Ntr}^{-/+}$ phenotype of ORS571X2816, neither pLRSC1A281 nor pLRSC1A2816 were capable of doing so, suggesting a polar effect of the $ntrY::Tn5$ insertion on $ntrX$ expression and organization of the $ntrYX$ loci as an operon.

---

nifA expression in $ntrY$, $ntrX$ and $ntrYX$ mutant strains

To examine the effects of different $ntr$ mutants on $nifA$ expression, a plasmid carrying a $nifA-lac$ fusion (pPR54; Ratet et al. 1989) was introduced into strains ORS571YX126, YX1215, YX1219, Y5 and X2816 and integrated into the chromosomal $nifA$ locus via a single crossover. The resulting strains were examined for $\beta$-galactosidase activity under a variety of physiological conditions. $nifA-lac$ expression in YX1215 and YX1219 was found to be the same as in the wild-type strain, whereas $nifA-lac$ expression in YX126 was slightly increased as compared to the wild type (data not shown). The results for the other strains are summarized in Table 4. The $ntrX$ mutation in X2816 had a similar influence on $nifA$ expression as the $ntrC$ mutation in C6 (60% of wild-type expression without N source; 10% on leucine), whereas the Y5 mutation only reduced $nifA-lac$ expression by approximately 35% relative to the wild type (Table 4), when the cells were grown in the presence of leucine, which has been observed to lead to maximal derepression of the ORS571 $nif/fix$ genes (Ratet et al. 1989). As in the case of the $ntrC$ strain ORS571C6 (Ratet et al. 1989), repression of $nifA$ induction by nitrogen and oxygen was still observed in each mutant strain.

---

Regulation of $ntrC$ and $ntrYX$ expression

To analyze the expression of $ntrC$ and $ntrYX$, respectively, miniMu-lac fusions to these genes were constructed (Fig. 7). The $ntrC-lac$ fusion (CmMS4; Fig. 7A) was found to be highly expressed in media containing glutamine (N-repressing) and nitrate (N-derepressing) conditions (Table 5), and there were no significant differences in $ntrC-lac$ expression levels in strains YX1219 and X2816, as compared to the wild-type (Table 5). The $ntrYX-lac$ fusion (XmMS3; Fig. 7B) was also found to be expressed independently of the N source. However,

Table 4. $nifA$-lac expression in different mutant strains

| Strains          | $\beta$-Galactosidase activity$^a$ |
|------------------|-----------------------------------|
|                  | $3\% O_2^b$                      | $21\% O_2^b$                     |
|                  | $-\mathrm{NH}_4^+$               | $+\mathrm{NH}_4^+$$^c$           | $+\mathrm{Leu}^c$                | $-\mathrm{NH}_4^+$               | $+\mathrm{NH}_4^+$$^c$           | $+\mathrm{Leu}^c$                |
| ORS571A54        | 100                              | 5                                | 174                               | 8                                | $<1$                             | 15                               |
| ORS571C6A54      | 60                               | 2                                | 15–20                             | $<1$                             | $<1$                             | $<1$                             |
| ORS571X2816A54   | 60                               | 2                                | 30                                | $<1$                             | $<1$                             | $<1$                             |
| ORS571Y5A54      | 105                              | 3                                | 114                               | 9                                | $<1$                             | 17                               |

$a$ $\beta$-Galactosidase activity (Miller 1972) is expressed as a percentage of that observed in strain ORS571A54 under normal nitrogen fixation conditions. The percentages shown represent the results of four independent experiments and values did not vary by more than 10%.

$b$ Cultures were incubated under an air/acetylene mixture (nitrogen fixation conditions; Pawlowski et al. 1987) or under air ($21\% O_2$)

$c$ Nitrogen sources [$(\mathrm{NH}_4)_2\mathrm{SO}_4$; leucine] were added to LSO medium to a final concentration of 0.2%

---

Table 5. $ntrC$-lac and $ntrX$-lac expression under different physiological conditions

| Strains          | $\beta$-Galactosidase activity$^a$ |
|------------------|-----------------------------------|
|                  | $3\% O_2^b$                      | $21\% O_2^b$                     |
|                  | $\mathrm{Gln}^c$                 | $\mathrm{NO}_3^-$$^c$            | $\mathrm{Gln}^c$                 | $\mathrm{NO}_3^-$$^c$            |
| ORS571CmMS4      | 560                              | 240                              | 940                               | 310                               |
| ORS571X2816CmMS4 | 620                              | 260                              | 1040                              | 350                               |
| ORS571XmMS3      | 73                               | 68                               | 182                               | 183                               |
| ORS571C6XmMS3    | 135                              | 10                               | 153                               | 9                                 |

$a$ $\beta$-Galactosidase activities are expressed in Miller units (Miller 1972), and represent the average values of three (XmMS3) and two independent experiments (CmMS4), respectively; values for the individual experiments did not vary to more than 13%

$b$ Cultures were incubated under an air/acetylene mixture (nitrogen fixation conditions; Pawlowski et al. 1987) or under air ($21\% O_2$)

$c$ Nitrogen sources (glutamine, $\mathrm{KNO}_3$) were added to LSO medium at a final concentration of 0.2%

---

Fig. 7A, B. Construction of the $ntrC::MudIIIPR46$ fusion CmMS4 and the $ntrX::MudIIIPR46$ fusion XmMS3. A shows the position and orientation of the MudIIIPR46 insertion CmM4 in pPR66. The $ntrC$ hybridizing region is stippled. B shows the position of the MudIIIPR46 insertion XmM3 in pRSX31. The positions of the chromosomal Tn5 insertions are indicated (vertical arrows) as orientation points. The vector sequences (pJRD184) are indicated by cross-hatching, the MudIIIPR46 ends are labeled in black. The extents of the $lac$, $\mathrm{Gm}^R$ and $\mathrm{Cm}^R$ genes are designated by vertical, diagonal and horizontal hatching respectively. The horizontal arrows indicate the orientation of the MudII insertions (direction of transcription of the $lac$ operon). The restriction enzyme abbreviations are described in the legend of Fig. 1

The expression on nitrate was dependent on the presence of an intact $ntrC$ gene, since it was reduced to 5–25% of wild-type levels in ORS571C6XmMS3 (Table 5).

The expression of $ntrC$ and $ntrX$ in the symbiotic state was examined by staining sections of stem nodules induced by ORS571, ORS571CmMS4 and ORS571XmMS3 for $\beta$-galactosidase activity. Strain ORS571A54 (Ratet et al. 1989) was used as a positive control. Staining of the symbiotic zone in ORS571A54 and ORS571CmMS4 was observed at similar intensities, whereas in ORS571XmMS3 induced nodules staining was weaker, although blue color appeared at least 12 h earlier than background plant $\beta$-galactosidase activity in ORS571-induced nodules (data not shown). Therefore, the $ntrYX$ and $ntrC$ loci appear to be expressed during symbiosis, which is consistent with the notion that mutations in these genes affect symbiotic nitrogen fixation (Table 2).

---

Discussion

In this paper we have described the identification and characterization of a novel two-component regulatory system in *A. caulinodans* ORS571 ($NtrY/NtrX$), involved in the regulation of nitrogen fixation and metabolism genes. The $ntrY/ntrX$ genes appear to constitute a bicistronic transcription unit, as observed for most other two-component systems analyzed thus far (see Albright et al. 1989). The $ntrX$ gene was initially identified on the basis of homology with the $ntrC$ and $nifA$ genes of *K. pneumoniae* and found to map approximately 4.5 kb downstream of the ORS571 $ntrC$ locus previously

described (Pawlowski et al. 1987; de Bruijn et al. 1988). DNA sequencing data presented here reveal that the *ntrX* gene product shares extensive amino acid homology with NtrC proteins of *R. meliloti* (Szeto et al. 1987), *B. parasponiae* (Nixon et al. 1986) and *K. pneumoniae* (Buikema et al. 1985), suggesting that it represents a (positive) regulatory protein. The pleiotropic phenotype of *ntrX* mutants with respect to nitrogen fixation and metabolism, as well as the high overall homology to *ntrC*, support its designation as a nitrogen regulation (*ntr*) locus. It is interesting that ORS571 harbors (at least) two different NtrC-like proteins, NtrC and NtrX, both of which contribute to *ntr* control, but in distinct ways. Both *ntrC* and *ntrX* mutants display retarded growth on nitrate, a reduction of *nifA* expression under nitrogen fixation conditions and a severely disturbed symbiotic phenotype (*Fix⁻/⁺* delayed on roots; *Fix⁻* on stems). However, their free-living nitrogen fixation rates differ, since *ntrC* mutants have 5–15% and *ntrX* up to 50% of wild-type activity. In addition, their growth patterns in medium containing different amino acids as N sources are distinct. The biological significance of having two similar positive regulators in ORS571 involved in nitrogen regulation remains to be elucidated, but it may be a reflection of the flexibility needed in an organism which fixes nitrogen in the free-living and symbiotic states, in stem and in root nodules, under greatly varying physiological conditions.

Generally, the genes encoding regulatory proteins of the NtrC family are closely linked to partner gene encoding the sensor (modulator) of the two-component system (see Albright et al. 1989). In the case of the NtrBC systems in enteric bacteria it has been shown that the C-terminal (conserved) part of NtrB phosphorylates and dephosphorylates the N-terminal (conserved) domain of NtrC in response to the cellular N status, thereby activating or deactivating it (Kustu et al. 1989; Magasanik 1988). It is highly likely that the *ntrY* gene of ORS571, found immediately upstream of *ntrX*, corresponds to such a sensor (modulator) protein, especially since NtrY shares extensive homology with other sensor proteins in its C-terminal domain. Whether in fact NtrY phosphorylates NtrX in response to the cellular N status is not known. The putative transmembrane domains of NtrY would suggest that NtrY may be a membrane protein, which would make it an unusual N-sensor (modulator), since NtrB proteins appear to be cytosolic proteins (Magasanik 1988). Indeed NtrY may be involved in sensing the extracellular N concentration.

It is also interesting to note that the Ntr⁻/⁺ phenotype of *ntrY* (or *ntrX*) mutants (impaired growth on nitrate) is only observed on plates and in well aerated cultures, suggesting an effect of the cellular oxygen status. However, the N-terminal domain of NtrY shows no significant homology with the putative oxygen sensor FixL of *R. meliloti* (David et al. 1988).

Upstream of the ORS571 *ntrC* gene, a region showing strong homology with the *B. parasponiae ntrB* gene (Nixon et al. 1986) has been found (K. Pawlowski, U. Klosse and F.J. de Bruijn, unpublished observation). This observation suggests that an *ntrBC* operon exists in

ORS571, as is the case in all other members of the *Rhizobiaceae* examined up to now. It is not known whether NtrB can modulate NtrX in addition to NtrC and/or whether NtrY can modulate NtrC in addition to NtrX. In *E. coli*, such an interaction between two-component systems has been shown to occur, since the transcriptional activator protein ArcA is modified by two different sensor proteins, ArcB and CpxA (Iuchi et al. 1989a, b). A comparison of the phenotypes of *ntrC* or *ntrX* and *ntrYX* mutants, respectively, shows that Ntr and Nif phenotypes are more severely affected by the removal of one activator protein (*ntrC* or *ntrX*), than by the removal of a complete system (*ntrYX*; see Table 2). This observation suggests that when an entire system is removed, the other can partially substitute to yield an intermediate phenotype, while leaving one activator protein together with two distinct (but related) modulators leads to a more severe perturbation of the system and a more extreme phenotype. Thus there may be cross-talk (Ninfa et al. 1988) between the NtrB/NtrC and NtrY/NtrX regulatory systems.

An interaction between the *ntrYX/ntrBC* systems is suggested by the gene fusion experiments, which show that *ntrYX* expression is affected by a *ntrC* mutation when cells are grown on nitrate. All attempts to construct a *ntrCntrX* double mutant have failed thus far (K. Pawlowski, unpublished observations) and therefore it has not been possible to ascertain if any other *ntrC/ntrX*-like genes are involved in nitrogen control.

Both the *ntrC* and *ntrX* loci appear to be involved in controlling *nifA* gene expression, suggesting that the effects of *ntrC* and *ntrX* mutations on free-living and symbiotic nitrogen fixation may be mediated via the *nifA* promoter (see also Ratet et al. 1989; de Bruijn et al. 1990). A typical NtrA-dependent –24/–12 (*ntr*) canonical sequence is present in front of *nifA* (de Bruijn et al. 1988; Ratet et al. 1989) and its importance in *nifA* expression has been suggested (de Bruijn et al. 1990). Whether NtrC and NtrX act in concert with the same sigma factor in activating the *nifA* gene needs to be investigated.

The symbiotic phenotype(s) of *ntrY*, *ntrX* and *ntrYX* mutants are intriguing. *ntrX* mutants resemble *ntrC* mutants in their ability to induce *Fix⁻* nodules on the stems and *Fix⁻/⁺* delayed nodules on the roots of *S. rostrata*. *ntrY* and *ntrYX* mutants, however, only induce slight swellings (bumps) on *S. rostrata* stems at the adventitious root sites, while resembling their *ntrC* (*ntrX*) counterparts on the roots. The phenotype of the ORS571YX126 mutant (60–80% nitrogen fixation activity in nodules), while resembling the wild-type strain in other respects, is also puzzling. Restriction mapping of this insertion mutant suggests that it may be located 5′ of the *ntrY* coding sequences and therefore could represent a promoter mutation with a minor effect on *ntrYX* expression. *ntrC(X)*-mediated *nif/fix* gene regulation in the symbiotic state (nodule) is highly unusual in rhizobia (de Bruijn et al. 1990). It could reflect the need for ORS571 to grow at the expense of N₂ during the infection process. Inability to proliferate, especially in the stem infection site, may delay or even prevent nodule

formation/nitrogen fixation (see de Bruijn 1989; Ratet et al. 1989). This is supported by the observation that ORS571 *glt* (GOGAT) and *glnA* (nitrogen assimilation) mutants also have a Fix⁻ phenotype (Hilger et al. 1987; Donald et al. 1988; de Bruijn et al. 1988, 1990), in contrast to the corresponding *R. meliloti* mutants (Kondorosi et al. 1977; de Bruijn et al. 1989). However, the capacity to fix nitrogen in the free-living state cannot be the sole determinant of the observed deficiency in stem nodulation, since a non-polar *ntrY* mutant has wild-type Nif activity, but is nevertheless seriously disturbed in its symbiotic properties. Furthermore, several Nif⁻ Fix⁻ mutant strains (*nifA*; Donald et al. 1986; Pawlowski et al. 1987; *nifHDK*; Elmerich et al. 1982) are not affected in their stem nodulation ability. Therefore, the stem nodulation deficiency of *ntrY* and *ntrXY* mutant strains must be the result of secondary effects of the *ntr* system, most likely on nitrogen assimilation (metabolism) of bacteria or bacteroids in the (developing) nodule.

By identifying the *ntrXY* two-component system, we have added another level of complexity to the regulatory circuitry controlling the expression of the *A. caulinodans* ORS571 nitrogen fixation and metabolism genes in the free-living and symbiotic states. It is clear that additional ORS571 loci play a role in this process and our present studies are designed to identify them and their target(s) in the promoters of N-regulated genes.

Acknowledgements. We would like to thank M. Schneider and H. Meyer z.A. for skilled technical assistance, as well as S. Rossbach, P. Ratet, U. Hilger and J. Stigter for helpful discussions and critical comments on the manuscript. We also thank Jeff Schell and the Max Planck Gesellschaft for generous support and encouragement, Maxine Lance, Karen Bird and Jan Johnson for help in preparing the manuscript and Kurt Stepnitz, Chris Fetters and Margaret Kalda for preparing the figures. FdB also gratefully acknowledges the support of the US Department of Energy (DE-FG02-90ER20021).

**References**

Albin R, Weber R, Silverman PM (1986) The Cpx proteins of *Escherichia coli* K12. Immunologic detection of the chromosomal *cpxA* gene product. J Biol Chem 261:4698–4705

Albright LM, Huala E, Ausubel FM (1989) Prokaryotic signal transduction mediated by sensor and regulator protein pairs. Annu Rev Genet 23:311–336

Beck F, Ludwig G, Auerswald EA, Reiss B, Schaller H (1982) Nucleotide sequence and exact localization of the neomycin-phosphotransferase gene from transposon Tn5. Gene 19:327–336

Boyer HW, Roulland-Dussoix D (1969) A complementation analysis of the restriction and modification of DNA in *E. coli*. J Mol Biol 41:459–472

Brown NL, Misra TK, Winnie JN, Schmidt A, Seiff M, Silver S (1986) The nucleotide sequence of the mercuric resistance operons of plasmid R100 and transposon Tn501: further evidence for *mer* genes which enhance the activity of the mercuric ion detoxification system. Mol Gen Genet 202:143–151

de Bruijn FJ (1987) Tn5 mutagenesis to map genes. Meth Enzymol 154:175–196

de Bruijn FJ (1989) The unusual symbiosis between the diazotrophic stem-nodulating bacterium *Azorhizobium caulinodans* ORS571 and its host, the tropical legume *Sesbania rostrata*. In: Nester E, Kosuge T (eds) Plant-microbe interactions vol 3. McGraw-Hill, New York, pp 457–493

de Bruijn FJ, Downie JA (1991) Biochemical and molecular studies of symbiotic nitrogen fixation. Curr Opin in Biotechnology 2:184–192

de Bruijn FJ, Pawlowski K, Ratet P, Hilger U, Schell J (1987) The unusual symbiosis between the nitrogen-fixing bacterium ORS571 and its host *Sesbania rostrata*: Regulation of nitrogen fixation and assimilation genes in the free-living versus symbiotic state. In: Verma DPS, Brisson N (eds) Molecular genetics of plant-microbe interactions. Martinus Nijhoff, Dordrecht, pp 266–271

de Bruijn FJ, Pawlowski K, Ratet P, Hilger U, Wong Ch, Meyer zA H, Schell J (1988) Molecular genetics of nitrogen fixation by *Azorhizobium caulinodans* ORS571, the diazotrophic stem nodulating symbiont of *Sesbania rostrata*. In: Bothe H, de Bruijn FJ, Newton WE (eds) Nitrogen fixation: Hundred years after. Gustav Fischer, Stuttgart-New York, pp 351–355

de Bruijn FJ, Rossbach S, Schneider M, Ratet P, Messmer S, Szeto WW, Ausubel FM, Schell J (1989) *Rhizobium meliloti* has three differentially regulated loci involved in glutamine biosynthesis, none of which is essential for symbiotic nitrogen fixation. J Bacteriol 171:1673–1682

de Bruijn FJ, Hilger U, Stigter J, Schneider M, Meyer zA H, Klosse U, Pawlowski K (1990) Regulation of nitrogen fixation and assimilation genes in the free-living versus symbiotic state. In: Gresshoff P, Roth E, Stacey G, Newton W (eds) Nitrogen fixation: achievements and objectives. Chapman and Hall, New York-London, pp 33–44

Buck M (1990) Transcriptional activation of nitrogen fixation genes in *Klebsiella pneumoniae*. In: Gresshoff P, Roth E, Stacey G, Newton W (eds) Nitrogen fixation: achievements and objectives. Chapman and Hall, New-London, pp 451–457

Buikema WJ, Szeto WW, Lemley PV, Orme-Johnson WH, Ausubel FM (1985) Nitrogen fixation-specific regulatory genes of *Klebsiella pneumoniae* and *Rhizobium meliloti* share homology with the general nitrogen regulatory gene *ntrC* of *K. pneumoniae*. Nucleic Acids Res 13:4539–4555

Casadaban MJ (1975) Fusion of the *Escherichia coli lac* genes to the *ara* promoter: A general technique using bacteriophage Mu-1 insertions. Proc Natl Acad Sci USA 72:809–813

Chang ACY, Cohen SN (1978) Construction and characterization of amplifiable multicopy DNA cloning vehicles derived from the P15A cryptic miniplasmids. J Bacteriol 134:1141–1156

Comeau DE, Ikenaka K, Tsung K, Inouye M (1985) Primary characterization of the protein products of the *Escherichia coli ompB* locus: structure and regulation of synthesis of the OmpR and EnvZ protein. J Bacteriol 164:578–584

David M, Daveran M-L, Batut J, Dedieu A, Domergue O, Ghai J, Hertig C, Boistard P, Kahn D (1988) Cascade regulation of *nif* gene expression in *Rhizobium meliloti*. Cell 54:671–683

Devereux J, Haeberli J, Smithies O (1984) A comprehensive set of sequence analysis programs for the VAX. Nucleic Acids Res 12:387–395

Ditta G, Stanfields S, Corbin D, Helinski DR (1980) Broad host range DNA cloning system for gram-negative bacteria: construction of a gene bank of *Rhizobium meliloti*. Proc Natl Acad Sci USA 77:7347–7351

Donald RGK, Nees DW, Raymond CK, Loroch AI, Ludwig RA (1986) Characterization of three genomic loci encoding *Rhizobium* sp. strain ORS571 N₂ fixation genes. J Bacteriol 165:72–81

Donald RGK, LaPointe J, Ludwig RA (1988) Characterization of the *Azorhizobium sesbaniae* genomic locus encoding NADPH-glutamate synthase. J Bacteriol 170:1197–1204

Dreyfus B, Dommergues YR (1981) Nitrogen-fixing nodules in-

produced by Rhizobium on the stem of the tropical legume Sesbania rostrata. FEMS Microbiol Lett 10:313–317

Dreyfus B, Elmerich C, Dommergues YR (1983) Free-living Rhizobium strain able to grow under N₂ as sole nitrogen source. Appl Environ Microbiol 45:711–713

Dreyfus B, Garcia JL, Gillis M (1988) Characterization of Azorhizobium caulinodans gen. nov., sp. nov., a stem-nodulating nitrogen-fixing bacterium isolated from Sesbania rostrata. Int J Bacteriol 38:89–98

Elmerich C, Dreyfus B, Reysset G, Aubert JP (1982) Genetic analysis of nitrogen fixation in a tropical fast-growing Rhizobium. EMBO J 5:441–447

Fickett JW (1982) Recognition of protein coding regions in DNA sequences. Nucleic Acids Res 10:5303–5318

Friedman AM, Long SR, Brown SE, Buikema WJ, Ausubel FM (1982) Construction of a broad host range cosmid cloning vector and its use in the genetic analysis of Rhizobium mutants. Gene 18:289–296

Gebhardt C, Turner GL, Gibson AH, Dreyfus BL, Bergersen FJ (1984) Nitrogen-fixing growth in continuous culture of a strain of Rhizobium sp. isolated from stem nodules of Sesbania rostrata. J Gen Microbiol 130:843–848

Genilloud O, Garrido MC, Moreno F (1984) The transposon Tn5 carries a bleomycin-resistance determinant. Gene 32:225–233

Gillis-Gonzales MA, Ditta GS, Helinski DR (1991) A haemoprotein with kinase activity encoded by the oxygen sensor of Rhizobium meliloti. Nature 15:170–172

Gribskov M, Burgess RR (1986) Sigma factors from E. coli, B. subtilis, phage SP01, and phage T4 are homologous proteins. Nucleic Acids Res 14:6745–6763

Gribskov M, Devereux J, Burgess RR (1984) The codonpreference plot: Graphic analysis of protein coding sequences and prediction of gene expression. Nucleic Acids Res 12:539–549

Gussin GN, Ronson CW, Ausubel FM (1986) Regulation of nitrogen fixation genes. Annu Rev Genet 15:31–49

Henikoff S (1984) Unidirectional digestion with exonuclease III creates target breakpoints for DNA sequencing. Gene 28:351–359

Hertig C, Li RY, Louarn A-M, Garnerone A-M, David M, Batut J, Kahn D, Boistard P (1989) Rhizobium meliloti regulatory gene fixJ activates transcription of R. meliloti nifA and fixK genes in E. coli. J Bacteriol 171:1736–1738

Heusterspreute M, Ha-Thi V, Emery S, Tournis-Gamble S, Kennedy N, Davidson J (1985) Vectors with restriction site banks IV. pJRD184, a 3793-bp plasmid vector having 43 unique cloning sites. Gene 39:299–304

Hilger U, Schell J, de Bruijn FG (1987) Isolation and characterization of Tn5-induced NADPH-glutamate synthase (GOGAT⁻) mutants of Azorhizobium sesbaniae ORS571 and cloning of the corresponding glt locus. Mol Gen Genet 210:195–202

Hyde DR, Tu C-PD (1985) tnpM: A novel regulatory gene that enhances Tn21 transposition and suppresses cointegrate resolution. Cell 42:629–638

Iuchi S, Cameron DC, Lin ECC (1989a) A second global regulator gene arcB mediating repression of enzymes in aerobic pathways of Escherichia coli. J Bacteriol 171:868–873

Iuchi S, Furlong D, Lin ECC (1989b) Differentiation of arcA, arcB, and cpxA mutant phenotypes of Escherichia coli by sex pilus formation and enzyme regulation. J Bacteriol 171:2899–2893

Kaminski PA, Elmerich C (1991) Involvement of fixLJ in the regulation of nitrogen fixation in Azorhizobium caulinodans. Mol Microbiol 5:665–673

Kondorosi A, Svab Z, Kiss GB, Dixon RA (1977) Ammonium assimilation and nitrogen fixation in Rhizobium meliloti. Mol Gen Genet 151:221–226

Kranz RG, Haselkorn R (1988) Ammonia-constitutive nitrogen fixation mutants of Rhodobacter capsulatus. Gene 71:65–74

Krikos A, Mutoh N, Boyd A, Simon MI (1983) Sensory transducers of E. coli are composed of discrete structural and functional domains. Cell 33:615–622

Kustu S, Santero E, Keener J, Popham D, Weiss D (1989) Expression of σ⁵⁴ (ntrA)-dependent genes is probably united by a common mechanism. Microbiol Rev 53:367–376

Kyte J, Doolittle RF (1982) A simple method for displaying the hydrophobic character of a protein. J Mol Biol 157:105–132

Leroux B, Yanofsky MF, Winans SW, Ward JE, Ziegler SF, Nester EW (1987) Characterization of the virA locus of Agrobacterium tumefaciens: A transcriptional regulator and host range determinant. EMBO J 6:849–856

Magasanik B (1988) Reversible phosphorylation of an enhancer binding protein regulates the transcription of bacterial nitrogen utilization genes. Trends Biochem Sci 13:475–479

Maniatis T, Fritsch EF, Sambrook J (1982) Molecular cloning. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York

Martin GB, Chapman KA, Chelm BK (1988) Role of the Bradyrhizobium japonicum ntrC gene product in differential regulation of the glutamine synthetase II gene (glnII). J Bacteriol 170:5452–5459

Meade HM, Long SR, Ruvkun GB, Brown SE, Ausubel FM (1982) Physical and genetic characterization of symbiotic and auxotrophic mutants of Rhizobium meliloti induced by transposon Tn5 mutagenesis. J Bacteriol 149:114–122

Miller JH (1972) Experiments in molecular genetics. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York

Nees DW, Stein PA, Ludwig RA (1988) The Azorhizobium caulinodans nifA gene: Identification of upstream-activating sequences including a new element, the ‘anaerobox’. Nucleic Acids Res 16:9839–9853

Ninfa AJ, Gottlin Ninfa E, Lupas AN, Stock A, Magasanik B, Stock J (1988) Crosstalk between bacterial chemotaxis signal transduction proteins and regulators of transcription of the Ntr regulon: Evidence that nitrogen assimilation and chemotaxis are controlled by a common phosphotransfer mechanism. Proc Natl Acad Sci USA 83:5492–5496

Nixon BT, Ronson CW, Ausubel FM (1986) Two-component regulatory systems responsive to environmental stimuli share strongly conserved domains with the nitrogen assimilation regulatory genes ntrB and ntrC. Proc Natl Acad Sci USA 83:7850–7855

Norel F, Elmerich C (1987) Nucleotide sequence and functional analysis of the two nifH copies of Rhizobium ORS571. J Gen Microbiol 133:1563–1576

Pawłowski K, Ratet P, Schell J, de Bruijn FJ (1987) Cloning and characterization of nifA and ntrC genes of the stem-nodulating bacterium ORS571, the nitrogen-fixing symbiont of Sesbania rostrata: Regulation of nitrogen fixation (nif) genes in the free-living versus symbiotic state. Mol Gen Genet 206:207–219

Ratet P, Schell J, de Bruijn FJ (1988a) MiniMu-lac transposons with broad host-range origins of conjugal transfer and replication designed for gene regulation studies in Rhizobiaceae. Gene 63:41–52

Ratet P, Pawłowski K, Meyer zA H, Schell J, de Bruijn FJ (1988b) Regulation of nitrogen fixation (nif) genes of Azorhizobium caulinodans in culture and in planta. J Plant Physiol 132:405–411

Ratet P, Pawłowski K, Schell J, de Bruijn FJ (1989) The Azorhizobium caulinodans nitrogen-fixation regulatory gene, nifA, is controlled by the cellular nitrogen and oxygen status. Mol Microbiol 3:825–838

Ronson CW, Nixon BT, Albright LM, Ausubel FM (1987) Rhizobium meliloti ntrA (rpoN) gene is required for diverse metabolic functions. J Bacteriol 169:2424–2431

Russo AF, Koshland DE (1983) Separation of signal transduction and adaptation functions of the aspartate receptor in bacterial sensing. Science 220:1016–1020

Sanger F, Nicklen S, Coulson AR (1977) DNA sequencing with chain-terminating inhibitors. Proc Natl Acad Sci USA 74:5463–5467

Sanger F, Coulson AR, Barrell BG, Smith AJH, Roe BA (1980) Cloning in a single-stranded bacteriophage as an aid to rapid DNA sequencing. J Mol Biol 143:161–178

Shine J, Dalgarno L (1974) The 3′ terminal sequence of *Escherichia coli* 16S ribosomal RNA: complementarity to nonsense triplets and ribosome binding sites. Proc Natl Acad Sci USA 71:1342–1346

Spiro S, Guest JR (1990) Fnr and its role in oxygen-regulated gene expression in *Escherichia coli*. FEMS Microbiol Rev 75:399–428

Szeto WW, Nixon T, Ronson CW, Ausubel FM (1987) Identification and characterization of the *Rhizobium meliloti ntrC* gene: *R. meliloti* has separate regulatory pathways for activating nitrogen fixation genes in free-living and symbiotic cells. J Bacteriol 169:1423–1432

Thöny B, Hennecke H (1989) The –24/–12 promoter comes of age. FEMS Microbiol Rev 63:341–358

Thöny B, Anthamatten D, Hennecke H (1989) Dual control of the *Bradyrhizobium japonicum* symbiotic nitrogen fixation regulatory operon *fixRnifA*: Analysis of *cis*- and *trans*-acting elements. J Bacteriol 171:4164–4169

Yanisch-Perron C, Vieira J, Messing J (1985) Improved M13 cloning vectors and host strains: Nucleotide sequence of the M13mp18 and pUC19 vectors. Gene 33:103–110

Communicated by H. Hennecke

---

**Note added in proof:**

After submission of this paper Kaminski et al. (1991, Mol Microbiol 5:1983–1991) have reported the identification of a *fixK/fnr*-like gene in *Azorhizobium caulinodans*, which encodes a positive activator of *nifA* expression. These authors have shown that the *fixLJ* genes control *fixK* expression, presumably in response to the cellular O₂ status and that the *fixK* gene, in turn, controls *nifA* expression. They also report that a *fixL ntrC* double mutant is strictly Nif⁻Fix⁻, suggesting that the *fixLJ/fixK* genes and the *ntrBC/ntrYX* system described here are the primary determinants of *Azorhizobium caulinodans* *nifA* activation under limiting O₂ and nitrogen conditions respectively.
