
Biosynthesis and Assembly  
of Capsular Polysaccharides  
in *Escherichia coli*

Chris Whitfield

Department of Molecular and Cellular Biology, University of Guelph, Guelph,  
Ontario N1G 2W1, Canada; email: cwhitfie@uoguelph.ca

---

**Key Words**

cell-surface biogenesis, *trans*-envelope assembly complex,  
glycosyltransferases, polysaccharide export

---

**Abstract**

Capsules are protective structures on the surfaces of many bacteria. The remarkable structural diversity in capsular polysaccharides is illustrated by almost 80 capsular serotypes in *Escherichia coli*. Despite this variation, the range of strategies used for capsule biosynthesis and assembly is limited, and *E. coli* isolates provide critical prototypes for other bacterial species. Related pathways are also used for synthesis and export of other bacterial glycoconjugates and some enzymes/processes have counterparts in eukaryotes. In gram-negative bacteria, it is proposed that biosynthesis and translocation of capsular polysaccharides to the cell surface are temporally and spatially coupled by multiprotein complexes that span the cell envelope. These systems have an impact on both a general understanding of membrane trafficking in bacteria and on bacterial pathogenesis.

# Contents

INTRODUCTION ..... 40
STRUCTURES AND SURFACE ASSOCIATION OF *ESCHERICHIA COLI* CAPSULES ..... 41
Group 1 and 4 Capsules Are Related to LPS O Antigens ..... 42
Colanic Acid Is Related to Group 1 Capsules ..... 44
Group 2 and 3 Capsules ..... 44
BIOSYNTHESIS AND ASSEMBLY OF CAPSULES BELONGING TO GROUPS 1 AND 4 ..... 45
Genetic Organization of Group 1 *cps* Loci ..... 45
The Genetic Determinants for Group 4 Capsules and Wzy-Dependent O Antigens Are Allelic ..... 47
Regulatory Features Distinguish Expression of Colanic Acid from Group 1 K Antigens ..... 47
Biosynthesis of Group 1 and 4 Capsular Polysaccharides in a Wzy-Dependent Process ..... 47
Regulation of K<sub>LPS</sub> Chain Length by Wzz ..... 49
Wzc Controls High-Level Polymerization of Group 1 Capsular Polysaccharides and Colanic Acid ..... 50
Wza-Dependent Translocation of

Group 1 Capsular Polysaccharides Across the Outer Membrane ..... 51
Assembly of Group 4 Capsules Also Requires Wza, Wzb, and Wzc ..... 52
Wzi Is a Component Unique to Group 1 Capsules ..... 53
BIOSYNTHESIS AND ASSEMBLY OF CAPSULES BELONGING TO GROUPS 2 AND 3 ..... 53
Genetic Organization of Group 2 *kps* Loci ..... 54
Reorganization of the *kps* Locus in Isolates with Group 3 Capsules ..... 55
Chain Elongation of Group 2 Capsular Polymers ..... 55
Initiation Reactions and the Nature of the Endogenous Acceptor ..... 56
ABC Transporter-Dependent Export of Nascent Group 2 Capsular Polysaccharide ..... 57
Additional Proteins Contribute to a Complex that Couples Biosynthesis to Export ..... 59
KpsD and KpsE Mediate Translocation of Group 2 and 3 Capsular Polysaccharides to the Cell Surface ..... 61
CONCLUDING REMARKS ..... 62

# INTRODUCTION

Cell-surface glycoconjugates play critical roles in interactions between bacteria and their immediate environment(s). Given that *Escherichia coli* isolates cause a range of infections and may have to withstand a transition between an animal host and a soil or water environment, it is perhaps not surprising that the surface architectures of these bacteria are diverse. *E. coli* isolates produce two serotype-specific surface polysaccharides: the lipopolysaccharide (LPS) O antigen and capsular polysaccharide K antigen. Variations in structures of these polysaccharides give rise to ~170 different O antigens and ~80 K antigens (1). Other polymers are not serotype specific. For example, most (if not all) isolates produce a polysaccharide known as enterobacterial common antigen (2), and many produce an extracellular polysaccharide called

**K antigen**: a major surface antigen used in *E. coli* serotyping, determined by capsular polysaccharide structure

colanic acid (or M antigen) under specific growth conditions (1) (see below). Two additional exopolymers have been identified more recently in *E. coli* because of their roles in cellular aggregation and the formation of biofilms on abiotic surfaces: the (1-4)-β-glucan bacterial cellulose (3) and a regulated (1-6)-β-GlcNAc polymer similar to a product made by staphylococci (4). The extent of their distribution within the species is unknown.

*E. coli* capsules are surface-enveloping structures comprising high-molecular-weight (capsular) polysaccharides that are firmly attached to the cell (Figure 1). They are well-established virulence factors, often acting by protecting the cell from opsonophagocytosis and complement-mediated killing (reviewed in 2, 5). The 80 different capsular serotypes in *E. coli* were originally divided into groups based on serological properties, and later revisions incorporated genetic and biochemical criteria (reviewed in 1, 5). The classification has since been expanded to four groups (6) (Table 1). *E. coli* group 1 and 4 capsules share a common assembly system, and this is fundamentally different from the one used for group 2 and 3 capsules.

Biosynthesis and assembly of capsular polysaccharides is a complex process. Activated precursors (nucleotide monophospho and diphospho sugars) in the cytoplasm are assembled into the nascent polysaccharide (*M<sub>r</sub>* > 100,000) by enzymes associated with the inner membrane. A dedicated translocation pathway moves nascent polymer through the periplasm and across the outer membrane to the cell surface. Emerging evidence points to the existence of *trans*-envelope assembly complexes that coordinate the biosynthesis of polymer with the export and translocation steps in both space and time. This may provide continuity between the cytoplasm and outer surface of the cell at the site of synthesis, reducing the problems associated with crossing a cell envelope comprising two different membranes, a periplasm, and the peptidoglycan layer (Figure 1). Despite the diversity in bacterial glycoconjugates, bacteria use a lim-

**Figure 1**

Electron micrographs of encapsulated *E. coli*. The schematic on top shows the organization of the cell envelope in gram-negative bacteria. The micrographs show the results of different procedures used to visualize highly hydrated capsule structures. Panel *a* shows serotype K1 (group 2) with the capsule structure preserved (or stabilized) using antibodies specific for the capsule (132). Panels *b* and *c* show serotype K30 (group 1) labeled with cationized ferritin (133) or after freeze substitution (134). Image in panel *c* courtesy of R. Harris and T.J. Beveridge.

**STRUCTURES AND SURFACE ASSOCIATION OF ESCHERICHIA COLI CAPSULES**

The major groups in the early capsule classification systems were distinguished by physical properties, including retention of the masking K antigen after heating cell suspensions (reviewed in 1, 5). The thermostability or thermolability properties of K antigens reflect differences in the means by which the capsule is linked to the cell surface. Despite extensive studies describing the repeat-unit structure of

**Capsule:** the surface layer on many bacteria, usually formed from capsular polysaccharide

| Characteristic | 1 | 2 | 3 | 4 |
|-----------------|---|---|---|---|
| Former K-antigen group | IA | II | I/II or III | IB (O-antigen capsules) |
| Thermostability of K antigen | Yes | No | No | Yes |
| Coexpressed with O serogroups | Limited range (O8, O9, O20, O101) | Many | Many | Often O8, O9 but occasionally none |
| Coexpressed with colanic acid | No | Yes | Yes | Yes |
| Genetic locus | cps near his | kps near serA | kps near serA | Near his |
| Thermoregulated expression | No | Yes | No | No |
| Elevated levels of CMP-Kdo synthetase | No | Yes | No | No |
| Terminal lipid moiety | LPS lipid A core in KLPS; unknown for capsular K antigen | α-glycerophosphate | α-glycerophosphate? | LPS lipid A core in KLPS; unknown for capsular K antigen |
| Polymer chain grows at | Reducing terminus | Nonreducing terminus | Nonreducing terminus? | Reducing terminus |
| Polymerization system | Wzy dependent | Processive glycosyltransferase activity | Processive glycosyltransferase activity? | Wzy dependent |
| PST-1 protein | Wzx | None | None | Wzx |
| ABC transporter | None | KpsMT | KpsMT? | None |
| MPA-1 protein | Wzc | None | None | Wzc |
| MPA-2 protein | None | KpsE | KpsE? | None |
| OMA protein | Wza | KpsD | KpsD? | Wza |
| Model system(s) | Serotype K30 | Serotypes K1, K5 | Serotypes K10, K54 O111 | |
| Similar to capsules in | Klebsiella, Erwinia | Neisseria, Haemophilus | Neisseria, Haemophilus | None known |

Group 1

| K27 | -4)-α-Glc-(1-4)-α-GlcA-(1-3)-α-Fuc-(1- |
| --- | --- |
|  | α-Gal-(1- |
| K29 | -2)-α-Man-(1-3)-β-Glc-(1-3)-β-GlcA-(1-3)-α-Gal-(1- |
|  | HOOC |
|  | H₃C |
| K30 | -2)-α-Man-(1-3)-β-Gal-(1- |
|  | β-GlcA-(1-3)-α-Gal-(1- |

Colanic acid

| OAc |
| --- |
| 2,3 |
| -4)-α-Fuc-(1-3)-β-Glc-(1-3)-β-Fuc-(1- |
| HOOC |
| H₃C |

Group 2

| K1 | -8)-α-Neu5Ac-(2- |
| --- | --- |
|  | 7/9 |
|  | *Ac-O |
| K2a | -4)-α-Gal-(1-2)-Gro-(3-P- |
| K4 | -4)-β-GlcA-(1-3)-β-GlcNAc-(1- |
|  | β-Fru-(1- |
| K5 | -4)-β-GlcA-(1-4)-α-GlcNAc-(1- |
| K12 | -3)-α-Rha-(1-2)-α-Rha-(1-5)-β-Kdo-(2- |
|  | Ac-O |
| K13 | -3)-β-Rib-(1-7)-β-Kdo-(2- |
|  | Ac-O |
| K92 | -8)-α-Neu5Ac-(2-9)-α-Neu5Ac-(2- |
| K100 | -3)-β-Rib-(1-2)-Rit-(5-P- |

Group 4

| O111 | -3)-β-GlcNAc-(1-4)-α-Glc-(1-4)-α-Gal-(1- |
| --- | --- |
|  | α-Col-(1- |
| K40 | -4)-β-GlcA-(1-4)-α-GlcNAc-(1-6)-α-GlcNAc-(1- |
|  | serine HN-OC |

Group 3

| K10 | -3)-α-Rha-(1-3)-β-Qui4NMal-(1- |
| --- | --- |
|  | Ac-O |
| K54 | -4)-β-GlcA-(1-3)-α-Rha-(1- |
|  | threonine HN-OC |
|  | (or serine) |

Figure 2

Repeat-unit structures of representative *E. coli* capsules and colanic acid exopolysaccharide. (Qui4NMal, 4-(2-carboxyacetamido)-4,6-dideoxyglucose; Col, colitose, 3,6-dideoxygalactose.) The asterisk in the K1 structure denotes form-variable (on-off) O-acetylation, and the dashed line in the colanic structure represents nonstoichiometric O-acetylation. The structures have been published elsewhere: K27 (135), K29 (136), K30 (137), K1 (138), K2a (139), K4 (140), K5 (141), K12 (142), K13 (143), K92 (144), K100 (145), K10 (146), K54 (147), O111 (148), K40 (149), and colanic acid (150).

group 1 K<sub>LPS</sub> is limited to a short oligosaccharide containing only one or a few K-antigen repeat units. The capsule evident in electron micrographs (Figure 1) is comprised of high-molecular-weight capsular K antigen, and interactions between the polysaccharide chains on the cell surface create a higher-order structure (7). Although the precise linkage between the capsular K antigen and the cell surface has not been established, LPS is not involved. Furthermore, there is no precursor-product relationship between the K<sub>LPS</sub> and high-molecular-weight capsular forms (8–10); capsule assembly for groups 1 and 4 requires a separate dedicated translocation system that is not used by K<sub>LPS</sub>.

Given the structural similarity between group 4 K<sub>LPS</sub> and LPS O antigens, it is not

surprising that the serology is often confusing. Some isolates produce a group 4 capsule as the only serotype-specific polysaccharide, in which case it is given O-antigen status (examples include O26, O55, O100, O111, O113, and O127) (8, 9, 11). L. Lieve and colleagues first described these as “O-antigen capsules” in their studies of O111 (8). Other group 4 capsules have K-antigen status (e.g., K40) (12) because they (like group 1 K antigens) are found in isolates that coexpress an additional neutral O antigen (i.e., one of O8, O9, O9a, O20, and O101 group). The distribution between the capsular and K<sub>LPS</sub> forms and the involvement of an additional neutral O antigen in some isolates point to a complex interplay in surface polymers that may be critical in pathogenicity.

### Colanic Acid Is Related to Group 1 Capsules

Colanic acid production is widespread in *E. coli* isolates. Its structure resembles group 1 capsules (Figure 2), and they are assembled by essentially identical processes. In contrast to the authentic serotype-specific group 1 capsules, a substantial amount of the colanic acid produced by a culture is secreted into the growth medium as an exopolysaccharide. Whether this is due to physico-chemical properties of the polymer itself or a subtle difference in biosynthesis and assembly has not been established (13). The most obvious feature distinguishing colanic acid from group 1 capsules is the absence of colanic acid production in wild-type isolates grown at 37°C on typical lab media. This is due to complex transcriptional regulation of the colanic acid biosynthesis locus. As might be expected, colanic acid has no known role in virulence, and the biological role for colanic acid lies primarily in the lifestyle of *E. coli* outside the host (reviewed in 14). Colanic acid biosynthesis genes are part of an extensive regulon responding to alterations in (or damage to) cell envelope structure, osmotic shock, and growth on surfaces. This exopolysaccharide is essential for the later stages of *E. coli* K-12 biofilm development on abiotic surfaces and is important for withstanding desiccation.

### Group 2 and 3 Capsules

Group 2 and 3 capsules are found in *E. coli* isolates that cause extraintestinal infections. The structural features and components of the repeat units of group 2 and 3 capsules vary extensively (Figure 2). Some (e.g., K2a and K100) contain phosphate residues in their backbone structures and are reminiscent of gram-positive teichoic acids. Several group 2 capsular polysaccharides resemble vertebrate glycoconjugates. Examples include the K1 antigen [α-(2-8)-linked polysialic acid], K4 (a substituted chondroitin backbone), and K5 (an N-acetylheparosan backbone) (5). These K antigens occur in isolates causing significant extraintestinal infections, and their inability to elicit a strong and protective antibody response limits effective vaccination strategies (reviewed in 5).

In terms of structure and assembly, *E. coli* group 2 and 3 capsules are reminiscent of capsules in *Neisseria meningitidis* and *Haemophilus influenzae*. A unifying structural theme in the related *E. coli* and meningococcal capsules is the presence of diacylglycerophosphate at the reducing terminus of a proportion of the polymer isolated from cultures (15). In the polysialic acid capsules of *E. coli* serotype K92 and meningococcal serotype B, the available evidence indicates that the lipid is linked directly to the reducing terminal Neu5Ac residue via a phosphodiester bridge (15). Unfortunately, an unequivocal resolution of the linkage structure was not possible. The same lipid moiety was later found in several other *E. coli* group 2 and 3 capsules, but in contrast to K92, preliminary evidence was reported for a 3-deoxy-D-*manno*-2-octulosonic acid (Kdo) residue located between the lipid and the reducing terminal sugar of the K12 and K82 polymer chains (16). Again, primary data with a definitive structure for the linkage region have not been published. The proposed Kdo-containing linkage appears throughout the

group 2 capsule literature and is supported by additional biochemical data. However, it is puzzling why the linkage would differ in some serotypes, and the structure(s) of the termini need to be revisited because this uncertainty compromises interpretation of data for critical biosynthetic steps (see below). The lipid terminus is thought to anchor the capsule to the cell surface, but only 20% to 50% of the isolated polymer has the phospholipid substitution (17). This may reflect lability of the phosphodiester linkage, but it raises questions about the extent and integrity of surface association. It has been suggested that nonlipidated polymer may be retained at the surface via ionic and other interactions.

BIOSYNTHESIS AND ASSEMBLY OF CAPSULES BELONGING TO GROUPS 1 AND 4

The sequences of genetic loci for group 1 and 4 capsules have identified important features shared with many LPS O-antigen biosynthesis loci. They all map (entirely or in part) to a polymorphic chromosomal region near the bis (histidine-biosynthesis) operon. The genes contributing to capsule expression in groups 1 and 4 encode several conserved proteins (Table 2), reflecting their common biosynthesis pathways, as well as additional proteins that determine the unique repeat-unit structure of each K antigen. Variations in the organization of the genetic loci, and the distribution of critical genes in more than one chromosomal region, distinguish the capsule biosynthesis systems in groups 1 and 4.

**Genetic Organization of Group 1 cps Loci**

The group 1 capsule biosynthesis locus (cps) comprises two regions (Figure 3) separated by a putative stem-loop transcriptional attenuator (18). The same locus is found in Klebsiella sp., presumably as a result of horizontal gene transfer (19). The 5′ part of the locus contains four conserved genes (wzi, wza, wzb, and wz c) present in all group 1 cps loci. Three of the four gene products (Wza, Wzb, and Wzc) are involved in polymerization control and translocation of the product from the inner membrane to the cell surface. They operate independently of capsule structure. The 3′ region of the locus is serotype specific and encodes enzymes for a Wzy-dependent biosynthesis system. These include enzymes for producing any sugar nucleotide precursors dedicated to capsule synthesis, glycosyltransferases (GTs), and two integral inner membrane proteins (Wzy and Wzx). Although the specific gene products are dictated by the serotype (and repeat-unit structure of the resulting polysaccharide), genes encoding

**Table 2 Conserved proteins involved in the biosynthesis of colanic acid and capsules belonging to groups 1 and 4**

| Protein | Protein family | Location | Function (or putative function) |
|---------|----------------|----------|----------------------------------|
| Wzx     | PST-1          | Inner membrane (integral) | Transfers nascent undecaprenyl diphosphate-linked repeat units across the inner membrane |
| Wzy     |                | Inner membrane (integral) with periplasmic catalytic site | Putative polymerase; assembles undecaprenyl diphosphate-linked polymers using lipid-linked repeat units exported by Wzx |
| Wzc     | MPA-1 (PCP-2a) | Inner membrane (integral) with a large periplasmic domain and cytosolic N and C termini | Participates in high-level polymerization of capsular polysaccharide and forms part of a trans-envelope capsule translocation complex; Wzc activity is determined by cycling of its phosphorylation state via the cytosolic C-terminal tyrosine autokinase domain. |
| Wzb     | PTP            | Cytoplasm | Protein tyrosine phosphatase; dephosphorylates Wzc |
| Wza     | OMA            | Outer membrane | Forms a multimeric putative translocation channel and interacts with the periplasmic domain of Wzc |

GT: glycosyltransferase  
Wzy: an integral inner membrane protein, required for polymerization of und-PP-linked repeat units in Wzy-dependent synthesis  
Wzx: an integral inner membrane protein, required for export of und-PP-linked repeat units in Wzy-dependent synthesis

"22-minute locus"

ymcC  
ymcD | ymcB  
wzaA | ymcA wzb wzc  
wza(etp) (etk)

K40 antigen  
biosynthesis  

orf5  
wzy wzx orf3 orf4 orf6  

hisI*  
O8-antigen biosynthesis  

O8:K40  
(group 4 capsule)  

Colanic acid biosynthesis ($cps_{ca}$)  

wzb  
wza | wcaB  
wcaA | wcaC wzy  
wcaE gmd gmm manB  
wcaF | fcl | wcal manC | wcaJ wcaK | wcaM  
galF  

O-antigen biosynthesis  

gnd ugd wzz hisI  

E. coli K-12 and  
isolates with group 2  
and 3 capsules  

K30 (group 1) capsule ($cps_{K30}$)  

orf2 IS1  
wzi wza | wzc  
wbaP wzy | wcaN | orfZ wbaZ wzx  
orfY wcaO  

IS1 ugd | manC | manB IS1  
O9a-antigen biosynthesis  

O9a:K30  
(group 1 capsule)  

Figure 3

Organization of the genes required for expression of colanic acid and capsules belonging to groups 1 and 4. The central part of the figure shows the conserved chromosomal $bis$-linked region in $E$. coli K-12 and isolates with group 2 and 3 capsules. These isolates are able to produce colanic acid. This region has undergone substantial rearrangement in the group 1 prototype (serotype O9a:K30) with insertion/ replacements introducing genes for expression of both O and K antigens. Group 1-producing isolates are unable to produce colanic acid, and the locus may have been lost during the chromosomal rearrangements. The genes for group 4 capsule biosynthesis are found in a typical O-antigen biosynthesis locus, and the colanic acid locus is intact in isolates producing group 4 capsules. An additional locus at a region corresponding to 22-minutes on the $E$. coli chromosome is essential for group 4 capsule expression and duplicates some genes in the group 1 $cps$ locus. The characteristic Wzx and Wzy genes are highlighted in red, and known genes involved in regulation of high-level polymerization and translocation are in blue. “Housekeeping” genes (green) are identified as reference points.

characteristic Wzx and Wzy homologs define the pathway and are always present.

For all intents and purposes, the 3′ region is identical in gene content to loci found in many bacteria with LPS O antigens synthesized via Wzy-dependent pathways (20). Furthermore, the group 1 $cps$ locus effectively occupies the same location (between $galF$ and $gnd$) on the genome as the LPS O-antigen biosynthesis locus in $E$. coli K-12 and in isolates with group 2, 3, and 4 capsules (Figure 3). It is the presence of the $wzi$, $wza$, $wzb$, and $wzc$ genes (Table 2) that distinguishes the loci for Wzy-dependent group 1 capsules from those for O antigens. These conserved genes form a surface assembly system devoted to capsular K antigen and do not participate in assembly of $K_{LPS}$. An unlinked copy of the $wza$, $wzb$, and $wzc$ genes is found in an operon at a location corresponding to 22 minutes on the $E$. coli K-12 chromosome (Figure 3).

Transcription of the $cps$ locus is driven from a constitutive promoter upstream of $wzi$ (18) and involves an RfaH-dependent antitermination mechanism common to several genetic loci, including many polysaccharide biosynthesis operons (21). RfaH is recruited by an 8-nucleotide $ops$ (operon polarity suppressor) sequence located at the 5′ end of the transcript and interacts with RNA polymerase to favor transcript elongation (reviewed in 22). In the absence of RfaH, transcriptional read-through past the stem-loop transcriptional attenuator in $cps$ is diminished, and capsule production is significantly reduced (18).

The Genetic Determinants for  
Group 4 Capsules and  
Wzy-Dependent O Antigens  
Are Allelic  

Group 4 capsule biosynthesis in serotype K40 (12) (Figure 3) and O111 (23) involves a gene locus indistinguishable from those responsible for expression of many different Wzy-dependent O antigens, and genes encoding Wza, Wzb, and Wzc are absent from the *bis*-linked locus. Instead, these functions are contributed by close homologs in the “22-minute locus” (Figure 3), and all seven genes in this transcriptional unit are required for group 4 capsule assembly in serotype O127 (11). Thus, although the organization of genes required for expression of group 1 and 4 capsules differs, the principal gene products are conserved, and the overall features of the biosynthesis and assembly pathways are predicted to be essentially the same.

Regulatory Features Distinguish  
Expression of Colanic Acid from  
Group 1 K Antigens  

The locus for colanic acid biosynthesis is located upstream of *galF* in *E. coli* K-12 and in isolates with group 2, 3, and 4 capsules (24) (Figure 3). The organization of this locus is, in most respects, identical to the group 1 capsule locus. The 5′ part of the colanic acid locus contains highly conserved homologs of *wza*, *wzb*, and *wzc* (note the absence of *wzi*) and is separated from the biosynthesis region containing *wzx* and *wzy* by a predicted stem-loop transcriptional attenuator. Isolates with group 1 capsules are unable to produce colanic acid (25), and the colanic acid genes may be absent as a result of past genetic rearrangement in these regions. However, colanic acid expression can be induced in isolates with group 2, 3, and 4 capsules (25, 26).

The complex transcriptional regulation of colanic acid production is controlled by the Rcs (regulation of capsule synthesis) proteins. The Rcs system is a complex phosphorelay system that is now known to extend well beyond colanic acid regulation (reviewed in 14). The Rcs transcriptome encodes proteins targeted to the envelope or involved in envelope modifications (such as colanic acid formation). They include cell-envelope proteins induced by shock and osmostress conditions as well as others associated with swarming behavior and biofilm formation. The Rcs system is integrated into other cellular regulatory circuits and may regulate surface remodeling in *E. coli* in response to a change in lifestyle.

Biosynthesis of Group 1 and 4  
Capsular Polysaccharides in a  
Wzy-Dependent Process  

Much of our current knowledge of the Wzy-dependent pathway results from studies on LPS O antigens, particularly those of *Salmonella enterica* (reviewed in 20, 27) (Figure 4).

Group 1 capsular polysaccharides are assembled on a carrier lipid comprising the C55-polyisoprenoid lipid derivative, undecaprenyl phosphate (und-P). The general features of this reaction series, and the identity and involvement of und-P, were first established in the group 1 capsule representative in *Klebsiella* (28). Initiation of the *E. coli* serotype K30 prototype requires the WbaP enzyme and involves the reversible transfer of Gal-1-P from UDP-galactose to und-P (29). WbaP is a member of a family of polyisoprenyl-phosphate hexose-1-phosphate transferases (27) that initiate O antigen and capsule biosynthesis in many bacteria by transfer of Gal-1-P, or Glc-1-P, to und-P (20, 27, 30). In contrast, group 4 capsule initiation involves transfer of GlcNAc-1-P by WecA, a representative of the polyisoprenyl-phosphate N-acetylhexosamine-1-phosphate transferase family that includes both prokaryotic and eukaryotic proteins (reviewed in 31). The *wecA* gene is encoded by the locus for enterobacterial common antigen biosynthesis, and although it was first characterized in this context, WecA is required for biosynthesis

Undecaprenyl  
phosphate (und-P):  
this C55  
polyisoprenoid lipid  
derivative serves as a  
carrier for assembly  
of surface  
polysaccharides in  
*E. coli*

ANRV 277-BI75-03 ARI 3 May 2006 8:39

![Figure 4](#fig4)

A model for biosynthesis and assembly of group 1 and 4 capsules. Beginning at the left, und-PP-linked repeat units are assembled at the interface between the cytoplasm and the inner membrane. Newly synthesized und-PP-linked repeats are then flipped across the membrane in a process requiring Wzx. This provides the substrates for Wzy-dependent polymerization wherein the polymer grows by transfer of the growing chain to the incoming und-PP-linked repeat unit. Continued polymerization requires transphosphorylation of C-terminal tyrosine residues in the Wzc oligomer and dephosphorylation by the Wzb phosphatase. Polymer is translocated by Wza, which likely acts as a channel. Wzi is unique to group 1 capsules and appears to be involved in modulating surface association.

of many *E. coli* O antigens. WecA and WbaP both contain several transmembrane segments, and this may be related to their need to interact with an obligatory lipid acceptor. The putative catalytic site of the *Salmonella* O antigen WbaP and its homologs is located in the C-terminal cytoplasmic domain (32, 33). Although the assignment of WbaP and WecA as initiating transferases is consistent with all available data, neither enzyme has been studied in purified form. Completion of the und-PP-linked repeat unit is catalyzed by a series of peripheral monofunctional GTs that transfer additional glycoses to the lipid intermediate.

Wzy-dependent polymerization was first described in classic experiments involving the assembly of the *Salmonella* O antigens (reviewed in 20, 27). The donors for the polymerization reaction are und-PP-linked repeat units, and polymer elongation involves transfer of the nascent polymer from its und-PP carrier to the new lipid-linked repeat unit, effectively increasing chain length in a block-wise manner by adding new repeat units at the reducing terminus. The available evidence for the *Salmonella* O antigens suggests the same lipid (und-PP) is used throughout, but this has not been confirmed by definitive structural studies for longer-chain intermediates; such analyses are technically challenging owing to the complexity of the molecules and their low abundance. In serotype K30 (group 1) (29) and K40 (group 4) (12), *wzy* mutants lack capsules and add only a single K-antigen repeat unit to the lipid A core in K<sub>LPS</sub>. These results provided the first proof that the capsular and K<sub>LPS</sub> forms of group 1 and 4 K antigens share common repeat-unit donors and polymerization machinery. Wzy is an integral membrane protein containing ~12 transmembrane segments and a large periplasmic loop (34). The catalytic mechanism of Wzy is unknown, and it is important to remember that no Wzy homolog has been purified and studied at a biochemical level to directly confirm the widely assumed “polymerase” activity.

Wzy-dependent polymerization occurs in the periplasm (35), and as a consequence, the assembly pathway is dependent on the export of lipid-linked repeat units across the inner membrane (Figure 4). In O-antigen biosynthesis systems, this process involves the *wzx* gene product (36, 37), and the same is likely true in the capsule biosynthesis systems. Wzx homologs are integral membrane proteins with multiple predicted transmembrane segments, and sequence similarities define a family of putative polysaccharide-specific transport (PST) proteins, designated PST-1 for the capsule-assembly homologs and PST-2 for those involved in O-antigen synthesis (38). However, the sequences do not provide any insight into the biochemical activity of Wzx. Although the Wzx homolog from the related enterobacterial common antigen biosynthesis system is required for transmembrane flipping of a water-soluble isoprenyl-PP-GlcNAc derivative in vesicles (39), it is not certain whether Wzx is the only component required in the process. Indeed, the N-terminal transmembrane domain of WbaP influences export of some *Salmonella* O antigens (32). It has been proposed that Wzx proteins may have specificity for the initial sugar in the lipid-linked repeat unit, perhaps via recognition of WbaP or WecA (40). The interaction may create a scaffold required for forming the lipid-linked repeat unit and then for releasing it to the export pathway (reviewed in 27). Undecaprenol has also been implicated as a scaffold for organizing proteins including GTs (41), and the lipid: protein complexes may alter the biophysical properties of the local membrane environment, perhaps aiding the flipping activity (42). The export process is a complex one, and the system may be correspondingly complicated.

Given that the polymer is apparently elongated in an undecaprenol-linked form, there is a requirement for its release into the translocation pathway once an appropriate chain length has been achieved. This process could be a side reaction of either Wzy or WaaL. Wzy must have the capability to release polymer from the lipid carrier during polymerization. The WaaL protein performs essentially the same cleavage step when it transfers nascent glycans to the lipid A core (e.g., in the formation of K<sub>LPS</sub>) (reviewed in 20, 27). The Wzy and WaaL enzymes share motifs in a periplasmic loop and may share a similar reaction mechanism (43). In either event, the process must be regulated in some manner to ensure the appropriate chain length is produced, and this could be dictated by protein-protein interactions in the context of a larger assembly complex. These processes are currently under investigation.

**Regulation of K<sub>LPS</sub> Chain Length by Wzz**

Polymerization of Wzy-dependent LPS O antigens and K<sub>LPS</sub> is terminated by transfer of the polymer (or oligosaccharide) from the lipid intermediate to lipid A-core acceptor. The reaction is catalyzed by WaaL. The extent of heterogeneity of the O-antigen chain lengths is dictated by the O-antigen chain-length determinant, the Wzz protein (previously designated as Cld or Rol) (reviewed in 20, 44). In the absence of Wzz activity, short unregulated O-antigen chains are formed, rather than a characteristic cluster of modal lengths. The *wzz* gene is located near the *bis* locus in *E. coli* K-12 and isolates with group 2 and 3 capsules (Figure 3). The difference in K<sub>LPS</sub> chain lengths in groups 1 and 4 is simply due to the absence of *wzz* in isolates with group 1 capsules (45, 46). In fact, modality and a longer chain length can be imparted on group 1 K<sub>LPS</sub> by introduction of a heterologous *wzz* gene (45).

Wzz proteins have a characteristic membrane topology and are grouped in a family called polysaccharide copolymerase-1 (PCP-1) (47). These proteins have two transmembrane helices flanking a periplasmic domain. The periplasmic domain is predicted to form coiled-coil structure and is implicated in determining chain-length modality. However, the mechanism of action of Wzz proteins is unknown.

MPA-1 protein  
(aka PCP-2a, e.g.,  
Wzc): it participates  
in synthesis and  
assembly of high-  
molecular-weight  
group 1 and 4  
capsular  
polysaccharides  

**Wzc Controls High-Level  
Polymerization of Group 1 Capsular  
Polysaccharides and Colanic Acid**

The characterization of Wzz has raised the  
question of how the capsular K-antigen poly-  
merization is controlled. Interest in the char-  
acteristic Wzc protein as a candidate chain-  
length regulator for group 1 capsular polysac-  
charides was stimulated by its similarity to  
Wzz in terms of predicted membrane topol-  
ogy (38, 44). *E. coli* K30 mutants with  
*wzc* defects are unable to make detectable  
amounts of capsular K antigen but are still  
able to polymerize K<sub>LPS</sub> (29, 48). Wzc pro-  
teins are grouped in the inner (cytoplasmic)  
membrane-periplasmic auxiliary-1 (MPA-1)  
(38) or PCP-2a (47) families, whose mem-  
bers are involved in a growing number of re-  
lated capsular and exopolysaccharide systems  
in gram-positive and gram-negative bacteria.  
Wzc is distinguished from Wzz by its posses-  

Top          Side         Bottom  
Wzc  
![Image Description](#)  
Wza  
![Image Description](#)  

**Figure 5**  
Surface-rendered three-dimensional structures of Wzc and Wza at 14 Å  
and 15.5 Å, respectively. The structures are derived from single-particle  
analysis of negatively stained samples in cryo-transmission electron  
microscopy (cryo-TEM). Wzc forms a tetramer with C4 rotational  
symmetry (54). The upper domain contains a region of connected density  
~20-Å thick around periphery of the structure. Four unconnected roots  
extend from the upper ring. These contain the N terminus and each can  
accommodate one C-terminal kinase domain based on molecular  
modeling. Wza is an octameric structure that adopts a tetrameric  
symmetry (59). This generates a structure containing two stacked rings  
with the upper one being slightly larger in diameter. A small elliptical  
cavity in the center (~40-Å diameter) is enclosed by four symmetrical  
arms. The cavity is closed at the top and bottom of the structure. The  
vertical bars indicate reference sizes for the inner and outer membranes  
measured in cryo-TEM images of frozen hydrated samples (131).

sion of a C-terminal cytoplasmic domain harbor-  
ing ATP-binding motifs (Walker A and B)  
as well as a tyrosine-rich region (7 of the last  
17 residues in *E. coli* K30). Work in A.J. Coz-  
zone’s laboratory (49) established that Wzc  
homologs are tyrosine autokinases that phos-  
phorylate at multiple residues at the expense  
of ATP and are dephosphorylated by cog-  
nate phosphatases. In *E. coli*, Wzb is the pro-  
tein tyrosine phosphatase (PTP family). Wzc  
phosphorylation involves a transphosphoryla-  
tion process (48, 50). Isolated Wzc shows het-  
erogeneity in phosphorylation (A. Reid and  
C. Whitfield, unpublished results), and muta-  
tional analyses support a model wherein phos-  
phorylation load on the C-terminal tyrosines  
(rather than any specific residue) is important  
for its function in capsule assembly (51). A  
minimum of four modifiable tyrosine residues  
are required in the Wzc C terminus for its  
competence in capsule assembly. The obser-  
vation that *wzb* and *wzc* mutations both result  
in an acapsular phenotype led to the hypoth-  
esis that Wzc function requires its cycling  
between phosphorylated and nonphosphory-  
lated forms (48). The Wzc-Wzb proteins are  
highly conserved in group 1 capsules and  
colanic acid biosynthesis systems. Although  
there is evidence suggesting differential ef-  
fects of phosphorylated Wzc in these systems  
(52), the homologs do function in the same  
manner when examined in the context of the  
group 1 K30 capsule-assembly systems (13).  

Wzc proteins are known to oligomerize in-  
dependent of phosphorylation (51, 53). Using  
cryo-electron microscopy with single-particle  
analysis, a structure of the Wzc tetramer has  
been resolved at 14 Å (54) (Figure 5). The  
structure does not support the proposed role  
of the C-terminal kinase domain in oligomer-  
ization (53), but it is consistent with the  
participation of the periplasmic domain in  
oligomerization of Wzz proteins (55). The  
exact contribution of the putative periplasmic  
coiled-coil motifs to the Wzc oligomerization  
is now being assessed.  

The precise function of Wzc is still un-  
known, as is that of Wzz. One possibility is  
that these proteins really are “copolymerases” and interact with Wzy. However, in preliminary studies, *wzc* mutants also result in dramatic reduction in the initiating Gal-1-P (WbaP) transferase activity (A. Reid and C. Whitfield, unpublished results), and the acapsular phenotype could be due to dramatically reduced flow in the polymer biosynthesis reactions. In this respect, Wzc might interact directly with WbaP to modulate its catalytic activity, or alternatively, Wzc might play a more structural role by coordinating the organization of an efficient multienzyme complex including WbaP. It is tempting to speculate that Wzc and Wzz have similar functions in polymer biosynthesis, but why does Wzc have the additional complexity of the C-terminal autokinase domain? The answer may lie in additional interactions involving Wzc and outer-membrane components in the assembly system (see below).

### Wza-Dependent Translocation of Group 1 Capsular Polysaccharides Across the Outer Membrane

Wza is essential for surface assembly of group 1 capsules (56, 57). The Wza protein is a member of a family of outer-membrane auxiliary (OMA) proteins associated with capsule and exopolysaccharide assembly systems (38). The Wza homologs encoded by the group 1 capsule and colanic acid loci are highly conserved and functionally exchangeable (13).

Wza is a lipoprotein that forms sodium dodecyl sulfate-stable multimeric ring-like structures (56) resembling the “secretins” associated with filamentous phage assembly and protein secretion through type II, III and IV systems in a range of gram-negative bacteria (58). Secretins exist as large channels formed by multimeric complexes of 10 or more monomers, and the structure of Wza in two-dimensional electron crystallography revealed an octameric organization (57). A three-dimensional structure with 15.5-Å resolution was obtained from single-particle analysis and showed an arrangement reflecting a tetrameric symmetry in which the protein complex encloses a central cavity (59) (Figure 5). No channel “openings” are apparent at the presumed external and periplasmic faces, and it is conceivable that isolated Wza oligomers adopt a closed state because large permanently open channels would compromise outer-membrane integrity. The open-closed state could be dictated by essential interactions with additional components in the assembly complex (see below). The best-characterized export channel in *E. coli* is TolC, which is involved in drug efflux and type 1 protein secretion (reviewed in 60). TolC is a trimer with an extensive α-helical periplasmic domain that opens the channel via an iris-like conformational transition. OMA proteins are clearly different in overall structure and, presumably, in function. Crystals of Wza oligomers diffracting to 3 Å have been generated (61), and a high-resolution structure will offer critical insight into capsule biogenesis and may be very informative for some protein secretion systems.

The conclusion that Wza provides the secretin for group 1 capsules is complicated by similar acapsular phenotypes and loss of high-molecular-weight polymerization in both *wza* and *wzc* mutants (56). This suggests some interactions between the outer- and inner-membrane components in a tightly coordinated process coupling biosynthesis to translocation. Strains expressing a nonacylated derivative of Wza are acapsular, but unlike *wza*-null mutants, they accumulate periplasmic polymer (57). The nonacylated Wza localizes to the outer membrane but forms unstable oligomers. These may be recognized by the rest of the complex so biosynthesis is maintained but cannot support translocation and effectively uncouple the coordination. Further evidence for a trans-envelope complex is provided by the interaction of Wzc and Wza identified in cross-linking experiments (57). Phosphorylation of Wzc does not dictate interaction but could potentially modulate conformation of the complex to open or close the

---

**Outer-membrane auxiliary (OMA) protein:** by oligomerization it forms the putative secretin for translocation of polymer

www.annualreviews.org • Capsular Polysaccharides in E. coli   51
ies on sites of capsule translocation carried out by M.E. Bayer. Translocation of group 1 (K29) capsule occurs at a limited number of sites on the cell surface that coincide with domains where the inner and outer membranes come into close apposition (62). The interpretation of these “zones of adhesion” has been controversial because of the need for specific preparation techniques for their visualization (reviewed in 63), but there are increasing examples in the literature of cell envelope-spanning multienzyme complexes for export of proteins and drugs. A coordinated capsule assembly complex (as depicted in Figure 4) would provide a physical and functional connection between the cell surface and the polymerization machinery in the inner membrane and would overcome the practical problem of transferring high-molecular-weight capsular polymers (M<sub>r</sub> > 100,000) to the surface in multiple steps.

### Assembly of Group 4 Capsules also Requires Wza, Wzb, and Wzc

Analysis of Wza, Wzb, and Wzc functions in *E. coli* K30 (group 1) has been complicated by the additional copies of *wza*, *wzb*, and *wzc* in the 22-minute locus (Figure 3). These genes are only functional in certain backgrounds, including EPEC, ETEC, and EHEC isolates (11, 64). The kinase and phosphatase activities of Wzc (also known as Etk) and Wzb (Etp) have been confirmed, and they have been shown to participate with low efficiency in production of K30 capsular polysaccharide and colanic acid (48, 52). The Wza homolog encoded by the 22-minute locus can also function in group 1 capsule assembly (56). Given the overall similarity of group 1 and 4 polymer biosynthesis, it seemed logical that homologs of Wza, Wzb, and Wzc would be involved in group 4 capsule assembly, and this has been confirmed (11). Perhaps more interesting is the finding that the locus contains a single transcriptional unit including the four additional genes (*ymcABCD*) required for group 4 capsule expression. All four are

**TYPE 4 SECRETION SYSTEMS**

Type 4 secretion systems (T4SSs) are involved in DNA transfer by conjugation, DNA uptake, and the transfer of DNA and effector proteins to eukaryotic target cells. In *Agrobacterium* spp., a T4SS is responsible for the delivery of both effector proteins and single-stranded DNA molecules (T-DNA) into plant cells. T4SSs provide an interesting parallel to capsule assembly because both must overcome the challenge of moving a high-molecular-weight anionic polymer across the bacterial cell envelope. In T4SSs, this is achieved by a *trans*-envelope complex comprised of 12 or more different proteins, some present in multiple copies. The complex provides a coordinated conduit from the cytoplasm to the cell exterior. The components include a hetero-oligomeric component associated with the outer membrane that is linked to proteins spanning the periplasm and interacting with those in the inner membrane. The complex also contains at least three ATP-binding proteins required for recruitment of the DNA substrate and its ATP-hydrolysis-dependent transfer to the inner-membrane channel. They may play roles in assembly of a functional complex. The organization of the complex and the protein-protein interactions within it are being dissected by systematic genetic and biochemical approaches, and insight into the DNA translocation pathway is emerging.

translocation channel. The recent isolation of higher-order structures containing Wza and Wzc for cryo-EM structural studies (R.F. Collins, K. Beis, R.C. Ford, J.H. Naismith, and C. Whitfield, unpublished results) will provide a critical next step in resolving the structure and function of the assembly system. There are some interesting parallels in terms of the requirement for ATP-binding proteins in this system and *trans*-envelope type IV secretion systems (T4SS) involved in translocation of DNA and effector proteins (reviewed in 61a). The T4SS seems to involve more components than the group 1 capsule system, but both translocate a hydrophilic polymer via a multiprotein complex that includes an oligomeric outer membrane secretin.

The biochemical and structural evidence for an envelope-spanning multienzyme complex brings into context classic early stud-

predicted to be exported proteins, and YmcA thesis and assembly. It is also not found in
and YmcC may be lipoproteins. Database the colanic acid and group 4 capsule systems.
searches are not informative as to function, Mutants lacking Wzi show a significant re-
although they do identify hypothetical pro- duction in surface-associated capsule and a
teins from polysaccharide systems in other corresponding increase in cell-free polymer.
bacteria. Paralogs of ymcABCD (yjbHGFE) are Although the exact mode of linkage of group
found elsewhere on the E.coli chromosome 1 capsules is still unknown, the mutant phe-
(11). Given the common theme in assembly notype is consistent with Wzi playing a role
of group 1 and 4 capsules, it is interesting that (direct or indirect) in surface attachment of
neither set of paralogs is essential for group the capsule. It is striking that wzi is confined
1K30 capsule formation (A.N. Reid and to those systems wherein the polymer product
C. Whitfield, unpublished data). It is con- is tightly associated with the cell surface in a
ceivable that other (unidentified) genes ful- discrete capsular structure.
fill the same role in group 1 systems; other-
wise this may represent a point of divergence

BIOSYNTHESIS AND ASSEMBLY
OF CAPSULES BELONGING TO
GROUPS 2 AND 3

The biosynthesis of group 2 and 3 capsules
is performed by proteins encoded by kps loci
located near serA. Like the group 1 and 4 cap-
sule loci, group 2 and 3 loci encode several
conserved proteins (Table 3) that serve as di-
agnostic markers for their biosynthesis sys-
tem. The group 2 and 3 loci differ in genetic
organization and by regulatory features that

Wzi Is a Component Unique to
Group 1 Capsules
The Wzi protein is a heat-modifiable
monomeric β-barrel protein that plays a role
in the final stages of capsule assembly (65).
Wzi is the only component of the group 1 cps
locus that is not essential for capsule biosyn-

Table 3 Conserved proteins involved in the biosynthesis of groups 2 and 3 capsules

| Protein | Protein family or homolog | Location | Function |
|---------|---------------------------|----------|----------|
| KpsF    | YrbH                      | Cytoplasm | Homolog of arabinose-5-phosphate epimerase; involved in CMP-Kdo biosynthesis |
| KpsU    | KdsB                      | Cytoplasm | Homolog of CMP-Kdo synthetase; involved in CMP-Kdo biosynthesis |
| KpsC    |                           | Cytoplasm | Precise function not established but essential for capsule export |
| KpsS    |                           | Cytoplasm | Precise function not established but essential for capsule export |
| Wzm     | ABC-A 2 (CPSE) transporter TMD | Inner membrane (integral) | Transmembrane domain component of the ABC transporter; exports nascent polymer across the inner membrane |
| Wzt     | ABC-A 2 (CPSE) transporter NBD | Inner membrane (peripheral) | Nucleotide-binding domain component of the ABC transporter; exports nascent polymer across the inner membrane |
| KpsE    | MPA-2                     | Periplasm (associated with outer face of inner membrane) | Putative membrane-fusion (or adaptor) protein; couples ABC transporter to later translocation steps |
| KpsD    | Putative OMA              | Outer membrane | Candidate for capsular polysaccharide translocation channel; requires KpsE for its proper localization |

www.annualreviews.org · Capsular Polysaccharides in E.coli 53

facilitate the characteristic thermoregulation
of group 2 capsules.

**ABC transporter:** a
transporter
superfamily
containing
representatives that
export group 2 and 3
capsular
polysaccharides

**Genetic Organization of Group 2**
**kps Loci**

In 1981, R.P. Silver and colleagues (66) reported the first cloning and expression of a
capsule gene cluster from any bacterium, with
their studies on *E. coli* group 2 serotype K1
genetics. The chromosomal loci for group
2 capsule expression (designated *kps*) have
a conserved structure comprising three regions (reviewed in 6, 67, 68) (Figure 6).
The serotype-specific central region (region
2) encodes GTs and any specialized sugar
nucleotide synthetases required for capsule
biosynthesis. The size and gene content of region 2 is therefore serotype specific and varies
according to the complexity of the repeat-unit
structure of the polymer formed. The absence
of *wzx* and *wzy* reflects the fundamental differences in the biosynthesis mechanism, compared to group 1 and 4 capsules.
Region 2 is flanked by genes whose products are conserved in different serotypes. They act independently of the structure of the
capsular polysaccharide and are involved in a
range of activities encompassing export and
assembly of the capsule on the cell surface. The completed polymer is exported across the
inner membrane by an ATP-binding-cassette
(ABC) transporter comprised of the region
3 gene products (KpsMT), but there are

**Group 2 capsules**

Conserved *kps* genes
Region 1
Region 3
*kpsF* → *kpsE* → *kpsD* → *kpsU* → *kpsC* → *kpsS*
Region 2
*kpsT* ← *kpsM*

K1
*neuS* ← *neuE* ← *neuC* ← *neuA* ← *neuB* ← *neuD*

K5
*kfID* ← *kfIC* ← *kfIB* ← *kfIA*

K4
*kfoG* ← *kfoF* ← *kfoE* ← *kfoD* ← *IS2* ← *kfoC* ← *kfoB* ← *kfoA*

**Group 3 capsules**

K10
*kpsM** IS3* → *kpsD* → *kpsM* → *kpsT* → *kpsE*
Region 2
*kpsC* → *kpsS*

**Figure 6**

Organization of the genes required for expression of capsules belonging to groups 2 and 3. The group 2
*kps* locus comprises a serotype-specific region 2 flanked by two regions (1 and 3) conserved across group
2 serotypes. Region 3 encodes the ATP-binding-cassette (ABC) transporter, and region 1 gene products
are involved in polymer export and translocation. Region 2 genes encode enzymes for polymer
biosynthesis, and the complexity of this region corresponds to the repeat-unit structures (Figure 2).
Group 3 loci contain region 3 genes and some (but not all) region 1 genes, and the loci show extensive
evidence of rearrangement.
unanswered questions concerning the functions of some of the six proteins encoded by region 1 (KpsFEDUCS).

Transcription from a $\sigma^{70}$-dependent promoter (or promoters) upstream of $kpsF$ yields a polycistronic message that covers the complete region 1 (69, 70). The region 3 promoter generates a large transcript that reads through into region 2 (71) and is dependent on RfaH-mediated antitermination (71), as in the transcription of group 1 capsule loci. The region 1 and 3 promoters are transcriptionally silent at $20^{\circ} \mathrm{C}$ (69, 70, 72). This thermoregulation is a defining feature of group 2 capsules, and although a detailed understanding of the process is not yet available, current information points to a complex and multifactorial system (72).

In at least one case, genes outside $kps$ also influence the repeat-unit structure. The K1 antigen is subject to form variation dictated by the presence or absence of O-acetylation (Figure 2). The O-acetyltransferase (NeuO) is part of a lysogenic bacteriophage-like element that is unlinked to $kps$ (73). Form variation involves the on-off modulation of $neuO$ expression via slipped-strand DNA mispairing.

### Reorganization of the $kps$ Locus in Isolates with Group 3 Capsules

Information for group 3 capsule loci is limited to partial sequences for serotypes K10 and K54 (reviewed in 68). These group 3 clusters have an organization with a central serotype-specific domain flanked by some (but not all) of the characteristic conserved genes from group 2 ($kpsMTEDSC$). However, their positions and relative order differ from group 2, suggesting recombination events have occurred with a locus that is allelic in isolates with group 2 and 3 capsules (74). Direct information for group 3 capsule biosynthesis and assembly is scarce, and the system is largely inferred from conservation of genes encoding critical export and translocation functions and complementation studies exploiting mutants in group 2 systems (74). A striking feature differentiating expression of group 2 and 3 capsules is that the group 3 loci lack the characteristic thermoregulation and are produced at all growth temperatures.

### Chain Elongation of Group 2 Capsular Polymers

Considerable progress has been made in understanding individual group 2 capsule GTs by exploiting the ability of these enzymes to extend exogenous polymeric and oligosaccharide acceptors. GTs expressed in the absence of the remaining capsule biosynthesis machinery and in the absence of initiation show in vitro chain extension activity with an appropriate acceptor. As an added incentive, several representative systems contain GTs with biotechnological relevance. For example, the poly-$\alpha$-2–8-NeuNAc sialyltransferase from $E$. coli K1 has the capacity to generate engineered polysialylgangliosides (75), and the glycosaminoglycan backbones of the K4 and K5 polymers provide templates for chemical and enzymatic modification to generate products of biomedical importance (76).

The polysialyltransferases from serotypes K1 and K92 represent perhaps the best characterized of these GTs. Research over several decades beginning with the work in the laboratories of Roseman and Troy, and extended by Vimr, Vann, and others, has culminated in the identification of NeuS as a processive GT that transfers Neu5Ac residues from CMP-Neu5Ac to the nonreducing terminus of the nascent glycan (reviewed in 77, 78). The poly-$\alpha$-2,8-sialyltransferase (NeuS) from K1 elongates both exogenous and endogenous acceptors in vitro (79, 80). The same is true of the highly conserved K92 NeuS homolog that generates a polymer with alternating $\alpha$-2,8/$\alpha$-2,9 linkages in serotype K92 (81, 82). In a $neuS$-null mutant of $E$. coli K1, the K92 NeuS enzyme forms its cognate serotype-specific product with alternating $\alpha$-2,8/$\alpha$-2,9 linkages (80–82). The collective data indicate that NeuS enzymes are the sole determinants of serotype specificity, and the K92 NeuS enzyme has dual linkage specificity. The highdegree of conservation in NeuS proteins has allowed the generation of informative chimeras to establish regions responsible for linkage specificity (82), but the mechanism of processive transferases is unknown. In fact, in the absence of a solved structure of a representative with defined processive GT activity, even the mechanism for an enzyme as well studied as cellulose synthase is still contentious (reviewed in 83).

The NeuS enzyme from K1 elongates oligosialyl exogenous acceptors (79, 80), with most evidence pointing to a preference for a tetramer or larger (79). Interestingly, soluble oligomers are typically poor acceptors for processive extension by K1 and K92 NeuS enzymes, but sialylgangliosides and acceptors with a terminal hydrophobic aglycone are efficiently elongated (75, 81, 84). The reason for the enhancing effect of the lipid terminus is unknown.

Polymer elongation at the nonreducing terminus is a conserved feature in group 2 capsules, with comparable processes shown for the K4 (85) and K5 (86, 87) polysaccharides. The processive bifunctional chondroitin polymerase (KfoC), from serotype K4 has both β-GalNAc and β-GlcA GT activities, and in contrast to NeuS, it efficiently elongates soluble tetra- and hexasaccharide acceptors to generate a high-molecular-weight product (85, 88). The extension of the K5 oligosaccharide acceptors (87) requires the action of two separate GT activities, but unlike the K4 system, these are found in separate polypeptides. These two K5 GT activities were initially both assigned to KfiC (89), but subsequent studies identify KfiA as the α-GlcNAc GT, with KfiC being the β-GlcA GT (90). In an interesting confirmation of these GT assignments, sequences similar to both KfiA and KfiC are present in the single bifunctional heparosan synthase from *Pasteurella multocida* serotype D (91).

Form variation in O-acetylation of K1 polysialic acid indicates that this modification is not essential for polymerization. The O-acetyltransferase utilizes acetyl coenzyme A as the donor and can modify larger oligosialyl acceptors [(Neu5Ac)₁₄] in vitro (92). The enzyme also modifies polysialic acid in an export-deficient mutant, indicating the reaction occurs in the cytoplasm (73). Along similar lines, chain elongation (and chondroitin formation) by KfoC can occur independent of the addition of side chain fructose residues in K4. Moreover, the observation that fructosylated acceptors cannot serve as exogenous acceptors lends additional support to the conclusion that side chain addition may even occur postpolymerization (85).

The termination of an efficient processive enzyme activity represents an equally interesting unresolved problem. In *E. coli* K1, the majority of the chains fall within a reasonably narrow size range with a maximum chain length of 160–230 residues (93), suggesting an active process in size determination. In some LPS O antigens assembled by processive GTs, chain termination and coupling to an ABC transporter is achieved by enzymes that add novel residues to the nonreducing terminus of the nascent glycan (94). A terminal residue on group 2 capsules would potentially be easily overlooked in structural analyses, but there are no obvious candidates for equivalent terminating enzymes encoded by the *kps* locus. Alternative possibilities for chain termination include loss of affinity of the GT for the polymer beyond a certain chain length, an abortive chain translocation process within the catalytic site similar to that proposed for chain termination in the type 3 polysaccharide synthase from *Streptococcus pneumoniae* (95), or an allosteric effect mediated by other components of the assembly system as proposed elsewhere (78). Regardless, the actual process is expected to have a significant impact on virulence and remains an important area for further study.

## Initiation Reactions and the Nature of the Endogenous Acceptor

Although NeuS enzymes are sufficient for elongation, they are unable to initiate
polysialic acid formation, and de novo synthesis requires other proteins from the *kps* locus (79–82). This indicates a requirement for an “initiate” enzyme, but identification of the players and characterization of the process are currently limited because the nature of the endogenous acceptor for group 2 capsule biosynthesis is equivocal. An early analysis of K1 biosynthesis led to the identification of und-P-Neu5Ac as a potential intermediate, suggesting a process resembling other bacterial glycoconjugates (96). However, lipid-linked polysialic acid has not been identified, and the existing data does not discriminate between a model wherein the nascent chain grows on und-P and one wherein und-P-Neu5Ac serves as a donor to another acceptor, such as diacylglycerophosphate. Superficially, the latter possibility would resemble the extension of lipomannan chains in the gram-positive bacterium, *Micrococcus luteus*, in which und-P-Man is the donor (97). Attempts to demonstrate the involvement of an equivalent und-P-linked intermediate in K5 biosynthesis using several different approaches have proved unsuccessful (86). The data are consistent with this particular system operating without lipid intermediates, although conclusions based on negative results must be interpreted with caution. For example, it is conceivable that all of the in vitro activity involves rapid extension of preexisting acceptors (similar to the majority of the activity seen in K1) (98). Exclusively long lipid-linked chains would not be efficiently extracted with solvent. Further studies are clearly warranted to definitively resolve the exact role (or not) of und-P in group 2 and 3 capsular polysaccharide biosynthesis.

If the endogenous acceptor is not und-P, what other molecules may fulfill this role? In the K1 system, there have also been reports of the involvement of a 20-kDa polypeptide as an endogenous acceptor (99, 100). However, the precise identity of this protein is unknown, as is the role of the protein: Does it provide an acceptor for the growing glycan, or is it an additional intermediate involved in the translocation of the polymer from cytoplasm to cell surface? NeuE and (77) and KpsT (101) have both been considered as candidates for the 20-kDa protein, and both are important for polymer export (see below).

It is also formally possible that the endogenous acceptor is diacylglycerophosphate (or diacylglycerophosphate-Kdo). Several export-deficient mutants in group 2 systems accumulate intracellular polymer with an added lipid terminus (see below). Furthermore, the K5 polymer synthesized in vitro has a reducing terminal residue whose properties are consistent with Kdo (86). Although not definitive, the data certainly suggest that the Kdo residue is present at the onset of chain elongation. Whether the same is true of the diacylglycerophosphate moiety remains to be established. The full range of potential acceptors for bacterial polysaccharides is currently unknown, but there is precedent for initiation of bacterial polysaccharides on a glycerolphospholipid acceptors: The processive streptococcal type 3 capsular polysaccharide synthase uses phosphatidylglycerol as its acceptor in the native streptococcal background and when the synthase is expressed in *E. coli* (102).

Given the overall conservation in the assembly components and processes (to the extent they are understood), completely different endogenous acceptors within representative group 2 systems would be rather surprising. The ambiguity in the existing data serves to underline the need for reinvestigation of the precise structures of the reducing termini of representative capsular polysaccharides (and of polymers accumulating in defined mutants). Until this information is available, the critical chain initiation reaction for group 2 (and 3) capsule biosynthesis cannot be resolved.

### ABC Transporter-Dependent Export of Nascent Group 2 Capsular Polysaccharide

ABC transporters (or traffic ATPases) drive the import or export of substrates at the

Diacylglycerophosphate
Endogenous
acceptor
KpsS & C
KpsF & U
GT(s)
NDP/NMP-glycose
NMP/NDP

KpsM
NDP/NMP-glycose
NMP/NDP

KpsT
ATP
ADP

KpsE oligomer
KpsD oligomer

Outer
membrane

Inner
membrane

Figure 7

A model for biosynthesis and assembly of group 2 capsules. Beginning at the left, polymer formation is initiated on an unknown endogenous acceptor (*open hexagon*) and is extended by processive GTs, adding residues to the nonreducing terminus of the chain. The final product is linked to diacylglycerophosphate (or diacylglycerophosphate-Kdo), but it is unclear whether this is present at initiation or added during, or after (as shown here), polymerization. The polymer is exported via the ABC transporter (KpsM and KpsT). KpsS and C are essential for this process, and KpsF and U also participate, but the details are unknown. The orientation of the polymeric substrate during export has not been established, and biosynthesis and export may be temporally coupled. Translocation across the periplasm and outer membrane requires KpsE and KpsD, which provide putative membrane-fusion (adaptor) protein and OMA protein functions.

expense of ATP hydrolysis. There are 80 members of the ABC transporter superfamily in *E. coli* (reviewed in 103). They are comprised of two domains, a transmembrane domain (TMD) and a nucleotide-binding domain (NBD) that contains the Walker A and B sequences (indicative of ATP-binding), and additional motifs that characterize the ABC transporter superfamily. The two domains can be present as individual polypeptides or may be fused in larger proteins. KpsM and KpsT were initially identified as the components of an ABC transporter on the basis of sequence data, and their identities were then confirmed by biochemical approaches (reviewed in 104, 105). KpsM is the integral inner-membrane TMD component, with six transmembrane helices, and KpsT is the cytoplasmic NBD. These proteins form an ABC transporter classified in the ABC-A2 (106) or CPSE (107) families. The functional group

2 (and 3) capsule transporter is proposed to consist of two subunits each of KpsM and KpsT.

Mutations in *kpsT* result in accumulation of intracellular polymer located (sometimes in patches) at the periphery of the cytoplasm in proximity to the inner membrane (108–110), which is consistent with an inability to complete the export process. An attractive working model for export has been proposed on the basis of studies of a dominant-negative KpsT mutant defective in ATP hydrolysis and analogies to other ABC transporters (104, 105). In this model, KpsT from *E. coli* K1 associates with polysialic acid in the context of the additional capsule biosynthesis machinery (101) and may introduce the polymer into the export channel. KpsT binds ATP, undergoes a conformational change, and interacts with the TMD. Deinsertion requires ATP hydrolysis, but a mutant unable to perform hydrolysis

remains locked into the complex and is nonfunctional (110). In this locked state, the KpsT mutant protein is exposed at the periplasmic face (101). Repetitive cycles of insertion-deinsertion may be necessary to complete the export process. The involvement of the polymeric substrate and other Kps proteins in supporting this process must also be considered.

The nature of the substrate for the ABC transporter has been controversial because of varying reports regarding the lipidation status of the intracellular polymer accumulating in various mutants. In initial studies with the K5 system, the polymer in a region 3 mutant was reported to lack the lipid terminus (108). However, using a different isolation method, the polymer in a K1 *kpsT* mutant was lipidated (70). It has been proposed that the results are very sensitive to the experimental protocols (70). Several other mutants (see below) also accumulate lipid-modified polymer, suggesting that lipidation precedes transport. From the available data, it is apparent that lipidation, by itself, is not sufficient for export via the KpsMT ABC transporter (70). However, the current data does not rule out the modification being essential for export. Another more complex question is the orientation of the exported product. In a coupled biosynthesis and export process, the transporter could potentially export the substrate before elongation is complete, but which end enters the exporter first—the diacylglycerophosphate or the glycan? KpsMT proteins have no specificity for polymer structure, suggesting they recognize a conserved element, assumed to be the lipid modification (as depicted in the model in Figure 7), but this is still conjecture. However, recent studies on the MsbA ABC transporter, which exports LPS lipid A core, have resulted in an attractive model whereby the hydrophilic domains of the export substrate are sequestered within the lumen of the channel, and the hydrophobic acyl chains are effectively excluded and are dragged through the surrounding lipid bilayer (110a).

---

**Additional Proteins Contribute to a Complex that Couples Biosynthesis to Export**

Several mutations result in the accumulation of intracellular polymer within the cytoplasm. The phenotypes of the *kpsMT* mutations are entirely predictable given the function of the ABC transporter, but those involving *kpsFU*, *kpsSC* and *neuE* are less obvious. These mutants accumulate unusual large electron-transparent domains containing aggregates of polymer within the cytoplasm (70, 111); these are often referred to as “lacunae.” The distribution of lacunae within the cell appears to vary in the different mutants, perhaps reflecting defects at different points in the assembly pathway. In serotype K1, the lacunae all appear to contain lipidated polymer (70), and a role can be envisaged for the hydrophobic moiety in overcoming the anionic nature of the polymer and helping form these structures.

KpsF and KpsU are homologs of enzymes involved in synthesis of CMP-Kdo (the activated Kdo donor). Four enzymes (*YrbH*, *KdsA*, *YrbI*, and *KdsB*) have been identified in the synthesis of CMP-Kdo for LPS biosynthesis. KpsF is a homolog of YrbH, the arabinose-5-phosphate epimerase catalyzing the first step in the pathway (112), and was identified through studies focused on KpsF in *N. meningitidis* (113). KpsU is a homolog of the last enzyme in the pathway, CMP-Kdo synthetase (*KdsB*) (114), and explains the observation that *E. coli* isolates with group 2 capsules characteristically have elevated synthetase activity at temperatures permissive for capsule formation (i.e., above 20°C) (5). Given the redundancy in KpsF and KpsU, it is perhaps not surprising that neither KpsF nor KpsU enzymes are essential for encapsulation, although the amount of capsule in the corresponding K1 mutants is substantially reduced (70, 115). The polymer is lipidated and appears to have a size similar to the wild-type product (70). If the requirement for CMP-Kdo is simply related to the reducing terminal

modification, the YrbH and KdsB proteins must provide sufficient CMP-Kdo to sustain some capsule formation and also supply the precursor for the biosynthesis of LPS lipid A core, whose Kdo residues are essential for viability of *E. coli* (reviewed in 20).

A more complex issue is why KpsF and KpsU would have any influence at all on encapsulation in serotype K1 because studies with the closely related K92 and meningococcal serotype B polysialic acids support a direct attachment of the polymer to diacylglycerophosphate, with no Kdo linker (15). Meningococci (unlike *E. coli*) can survive without LPS, allowing the construction of definitive and informative mutants that eliminate all CMP-Kdo synthesis. In the absence of CMP-Kdo, capsule expression is reduced in several different serotypes, regardless of the whether the capsule repeat unit contains Neu5Ac (113). Thus, if Kdo is not present in the linker, it must be involved somewhere in the assembly pathway independent of the structure of the reducing terminal modification. In rationalizing the conflicting structural and biochemical data, Tzeng et al. (113) proposed that the polymer must be removed from the Kdo acceptor prior to its transfer to diacylglycerophosphate. Although there are no data ruling this out, it invokes a rather convoluted and seemingly inefficient assembly scheme for which there are currently no other supporting data. Extended to the *E. coli* systems, it would also require a different mechanism operating in some group 2 capsules depending on the presence (or not) of a reducing terminal Kdo linker. An attractive alternative explanation is that KpsF and KpsU play an additional role independent of their function in CMP-Kdo synthesis (70, 115) (see below).

KpsC and KpsS are cytosolic proteins essential for the appearance of group 2 capsular polymers on the cell surface, and defects lead to the accumulation of intracellular polymer. In initial studies with K5, region 1 deletions (108) and *kpsS* and *kpsC* mutants (111) were proposed to lack lipid modification. This, together with comparable initial data for the

meningococcal KpsSC homologs (LipA and LipB) (116), contributed to speculation that KpsS and C are involved in the addition of the diacylglycerophosphate-Kdo modification (6, 67). However, in the K1 system, *kpsS* and *kpsC* mutants accumulate full-length lipidated intracellular polymer (70). The same result was subsequently obtained with meningococcal *lipAB* mutants, and critically, the structure of the lipid was also definitively determined and shown to be identical to that attached to wild-type polymer (117). Given the high conservation of these homologs (~60% similarity between *E. coli* and meningococci), it seems unlikely that the proteins have different roles in these species. Unless there are cellular components that functionally replace KpsS and KpsC in some (but not all) systems, the majority of available evidence suggests that KpsSC proteins play a role in the export-translocation process that does not involve lipidation per se. Although database searches highlight the wide distribution of KpsS and KpsC homologs in bacteria with group 2-related capsules, they do not provide any clues as to their precise function. Resolution of the exact roles played by KpsS and KpsC is critical for an understanding of the steps involving targeting nascent group 2 (or 3) polymer to the exporter.

In serotype K1 *neuE* mutants, intracellular lacunae are generally located toward one or both poles of the cell (78, 118). NeuE was previously thought to be the initiating GT for polysialic acid synthesis, owing, in part, to its possession of a potential polyisoprenyl-recognition motif in a C-terminal membrane-spanning domain, but this interpretation has to be reevaluated in the light of the intracellular polymer (reviewed in 78, 118). NeuE is the only region 2 gene product that is not essential for polysialic acid biosynthesis.

The properties of the relevant mutants suggest that KpsS, KpsC, KpsF, KpsU, and NeuE may all contribute in some way to a stable enzyme assembly complex, and their absence uncouples the spatial and temporal coupling of biosynthesis and export. In *E. coli* K5,
$kpsC$, $kpsS$, and $kpsU$ mutants all show a considerable reduction in in vitro GT activity in isolated membranes, yet clearly, the GTs are active in vivo (111, 119). Similarly, in serotype K1, the NeuS polysialyltransferase attaches to membranes in the absence of other Kps proteins except KpsS (80), and the level of in vitro endogenous polysialyltransferase activity is reduced when NeuS is expressed without KpsS and KpsT (120). Roberts and coworkers (121) have performed a systematic analysis of individual mutants in K5 and generated data lending strong support to the concept of a coordinated multienzyme complex. KpsCST proteins all independently target to the inner membrane. In contrast, the KfiAC GTs and the precursor-forming KfiD (UDP-glucose dehydrogenase) only target to the inner membrane in the presence of KpsMT and KpsCS. There is also a hierarchy in the assembly of the complex, with KfiA targeting requiring KfiC, but not KfiB, and KfiB requiring both KfiA and KfiC (90). These data infer complex protein-protein interactions between the various components in the assembly complex. The involvement of NeuE in *E. coli* K1 and the absence of $kpsF$ and $kpsU$ in group 3 capsule loci (68) suggest that some of the interactions are unique to a particular system. The cytoplasmic lacunae may therefore represent an aberrant site for “dumping” polymers that could not be exported in an appropriate or timely manner.

### KpsD and KpsE Mediate Translocation of Group 2 and 3 Capsular Polysaccharides to the Cell Surface

Mutations in $kpsD$ (122) and $kpsDE$ (111, 123) result in the accumulation of polymer in the periplasm, indicating that their products operate late in the assembly pathway. The corresponding genes from group 3 complement group 2 mutations (74), indicating a conserved function. In ABC transporter-driven efflux pumps, the transporter in the inner membrane is coupled to an outer membrane via a membrane-fusion protein (MFP) or adaptor protein. For example, the TolC outer-membrane channel protein is recruited and coupled to ABC transporters for RND-type drug efflux and type I protein secretion by the putative MFP proteins, AcrA and HlyD, respectively (reviewed in 60, 124). This generates a functional trans-envelope complex that provides continuity between the cytoplasm and the cell surface. Assembly of a functional TolC-ABC transporter complex involves a series of molecular interactions and conformational changes that, in the case of hemolysin secretion, are influenced by the presence of the export substrate. An MFP protein was proposed for the group 2 capsule system with KpsE [a representative of the inner-membrane-periplasmic auxiliary protein-2 family (MPA-2)] providing the candidate (38). KpsE has a tendency to form dimers and higher-order oligomers (125), as expected from studies of the drug efflux MFPs. The bulk of the protein is located in the periplasm (126), and it associates with the outer face of inner membrane via a C-terminal amphipathic α-helix (125, 127). KpsE may be responsible for bridging the periplasm and maintaining the discrete sites at which nascent capsule is translocated to the surface (128).

It is well accepted that the translocation of group 2 capsular polysaccharide must involve some form of channel to facilitate passage of the polymer across the outer membrane, analogous to the role proposed for the Wza outer-membrane protein in group 1 capsule formation (see above). KpsD has been grouped within the OMA family along with Wza from group 1 and 4 capsule systems on the basis of regions of local sequence similarity (38), although the similarity is low and KpsD (unlike Wza) is not a lipoprotein (56). However, the candidacy of KpsD as the outer-membrane “secretin” was obscured by localization studies that placed KpsD in the periplasm (122). It is now apparent that the location of KpsD is dictated by the genetic background. In the presence of KpsE and the envelope protein,

---

**MFP:**  
membrane-fusion protein (or adaptor) protein family  

**MPA-2 protein:** a putative adaptor protein that links an ABC transporter to an outer-membrane channel  Lpp, KpsD appears in the outer membrane (125). As further support for the candidacy of KpsD as the group 2 OMA protein, it is reported that KpsDE can be exchanged with CpxCD (the MPA-2 and OMA proteins from the capsule locus of *Actinobacillus pleuropneumoniae* 5A) with preservation of capsule assembly (105). CpxD shares significant similarity with Wza and other known OMA proteins (56). The presence of KpsD does not signal a unique situation in *E. coli* because a homolog is found in the capsule cluster of *Campylobacter jejuni*, a locus resembling group 3 capsule gene clusters (e.g., K10) and encoding homologs of KpsMTEDFCS (129).

Thus, it seems likely that KpsD fulfills the functional role of OMA proteins and represents a divergent group in the OMA family. If true, the *kps* locus now appears to encode a dedicated OMA protein and putative MFP, consistent with the gene content for other related capsule biosynthesis loci, and those proteins can operate with heterologous ABC transporters (105). If KpsD is indeed a functional equivalent of Wza, structural comparisons will be both interesting and informative.

### CONCLUDING REMARKS

Significant progress has been made in identifying the components in capsule biosynthesis and assembly. However, unraveling the underlying molecular mechanisms has been difficult because mutations in many of the components have the same capsule-null phenotype. Furthermore, interpretation of defects at a biochemical level is complicated by potential networks of protein-protein interactions within assembly complexes. The existence of a trans-envelope complex would provide a means of coordinating biosynthesis and surface assembly. By creating continuity between the inner membrane and the cell surface, such complexes would overcome the challenges posed by having to cross the periplasm, the peptidoglycan, and the outer membrane. Some of the important individual open questions have been identified above. Clearly, the next step is to determine the structure and function of purified components and resolve the molecular architecture of the complexes. Although biochemical and structural biology methods can be used, the limiting factor is that several of the most important proteins in the systems are integral membrane proteins, for which overexpression has proved difficult, and development of informative activity assays is still a challenge. Current efforts are logically directed at capsule-specific proteins, but capsule assembly occurs within the context of the *E. coli* cell envelope, and it is inevitable that other housekeeping functions will also participate. The recent elegant work of Kahne, Silhavy, and colleagues (130) serves to highlight the relationships and potential interactions between different multiprotein complexes involved in outer-membrane biogenesis. Understanding how the various systems are coordinated to maintain the protective capsule during growth and division of a bacterial cell is a complex and intriguing problem and is of critical importance in virulence.

---

### SUMMARY POINTS

1. *Escherichia coli* capsules are classified into four groups on the basis of genetic and biochemical criteria. The biosynthesis and assembly mechanisms for groups 1 and 4 are closely related and follow Wzy-dependent processes. Group 2 and 3 capsules are both assembled via ABC transporter-dependent pathways.
   
2. Capsule assembly appears to involve a multiprotein complex spanning the cell envelope, and this may facilitate spatial and temporal coupling of polymer biosynthesis, export, and translocation. The complex consists of core export and translocation
machinery, functioning independent of polymer structure, into which is plugged a serotype-specific polymer biosynthesis system.

3. In the Wzy-dependent mechanism, undecaprenol-linked repeat units are used in a polymerization reaction that requires Wzx and Wzy proteins. Newly synthesized lipid-linked repeat units are exported across the inner membrane in a process requiring Wzx. The growing glycan is transferred from its lipid carrier to the newly exported lipid-linked repeat unit, elongating the polymer in a “block-wise” manner by addition of new repeat units at the reducing terminus. The polymerization reaction is dependent on the Wzy protein.

4. In the ABC transporter-dependent mechanism, processive glycosyltransferase activity extends the nascent glycan in the cytoplasm. Chain extension occurs by addition of residues to the nonreducing terminus, but the nature of the endogenous acceptor for glycan growth is controversial. The polymer is transferred across the inner membrane by the ABC transporter composed of KpsM (transmembrane domain) and KpsT (nucleotide-binding domain). Additional proteins appear to be involved in coupling biosynthesis to export in this system, and other proteins may also participate.

5. Translocation of capsular polysaccharide across the outer membrane requires a member of the OMA family. In group 1 and 4 capsules, this protein is Wza. Wza forms a stable octameric complex. KpsD may fulfill this role in group 2 and 3 capsule assembly systems.

6. In group 1 and 4 capsule assembly, Wza is linked to the inner-membrane biosynthesis process by Wzc (an MPA-1 protein). Wzc forms tetramers and undergoes transphosphorylation of several C-terminal tyrosine residues. Phosphorylation of Wzc and its dephosphorylation by the phosphatase Wzb are both essential for group 1 and 4 capsule assembly, suggesting that Wzc must cycle its phosphorylated state.

7. In group 2 (and possibly group 3) capsules, the periplasmic protein KpsE (MPA-2 protein) may facilitate linkage of the ABC transporter and OMA components of the assembly system. KpsE is a putative membrane-fusion protein, or adaptor protein, and is essential for translocation of polymer to the cell surface.

---

**FUTURE ISSUES**

1. What are the mechanisms of action of the characteristic Wzx and Wzy proteins, and how are their activities coordinated in polymerization?

2. What is the nature of the endogenous acceptor for group 2 and 3 capsular polysaccharide biosynthesis, and when is the diacylglycerophosphate anchor added?

3. Do the outer-membrane OMA proteins for groups 1 and 4 and groups 2 and 3 have a conserved function, and what structural features are associated with their coupling to different inner-membrane biosynthesis and export components?

4. If capsule biosynthesis and assembly machinery form a transmembrane complex, as current data suggest, how does this complex cross the stress-bearing peptidoglycan layer without compromising cell wall integrity, and how is its activity coordinated with others involved in membrane biogenesis during cell growth and division?

# ACKNOWLEDGMENTS

The author recognizes the contribution of lab members, past and present, to the research and ideas presented here. Helpful and stimulating discussions with Dr. E.R. Vimr are gratefully acknowledged, as are constructive comments on a draft version provided by Dr. Vimr and Dr. A.N. Reid. Our work in this area is generously supported by the Canadian Institutes of Health Research, and C.W. is a recipient of a Canada Research Chair.

# LITERATURE CITED

1. Ørskov I, Ørskov F, Jann B, Jann K. 1977. *Bacteriol. Rev.* 41:667–710
2. Rick PD, Silver RP. 1996. In *Escherichia coli and Salmonella: Cellular and Molecular Biology*, ed. FC Neidhardt, pp. 104–22. Washington, DC: ASM Press
3. Zogaj X, Nimtz M, Rohde M, Bokranz W, Romling U. 2001. *Mol. Microbiol.* 39:1452–63
4. Wang X, Dubey AK, Suzuki K, Baker CS, Babitzke P, Romeo T. 2005. *Mol. Microbiol.* 56:1648–63
5. Jann K, Jann B. 1997. In *Escherichia coli: Mechanisms of Virulence*, ed. M Sussman, pp. 113–43. Cambridge, UK: Cambridge Univ. Press
6. Whitfield C, Roberts IS. 1999. *Mol. Microbiol.* 31:1307–19
7. Hungerer D, Jann K, Jann B, Ørskov F, Ørskov I. 1967. *Eur. J. Biochem.* 2:115–26
8. Goldman RC, White D, Ørskov F, Ørskov I, Rick PD, et al. 1982. *J. Bacteriol.* 151:1210–21
9. Peterson AA, McGroarty EJ. 1985. *J. Bacteriol.* 162:738–45
10. MacLachlan PR, Keenleyside WJ, Dodgson C, Whitfield C. 1993. *J. Bacteriol.* 175:7515–22
11. Peleg A, Shifrin Y, Ilan O, Nadler-Yona C, Nov S, et al. 2005. *J. Bacteriol.* 187:5259–66
12. Amor PA, Whitfield C. 1997. *Mol. Microbiol.* 26:145–61
13. Reid AN, Whitfield C. 2005. *J. Bacteriol.* 187:5470–81
14. Majdalani N, Gottesman S. 2005. *Annu. Rev. Microbiol.* 59:379–405
15. Gotschlich EC, Fraser BA, Nishimura O, Robbins JB, Liu T-Y. 1981. *J. Biol. Chem.* 256:8915–21
16. Schmidt MA, Jann K. 1982. *FEMS Microbiol. Lett.* 14:69–74
17. Jann B, Jann K. 1990. In *Bacterial Capsules*, ed. K Jann, B Jann, pp. 19–42. Berlin: Springer-Verlag
18. Rahn A, Whitfield C. 2003. *Mol. Microbiol.* 47:1045–60
19. Rahn A, Drummelsmith J, Whitfield C. 1999. *J. Bacteriol.* 181:2307–13
20. Raetz CRH, Whitfield C. 2002. *Annu. Rev. Biochem.* 71:635–700
21. Bailey MJ, Hughes C, Koronakis V. 1997. *Mol. Microbiol.* 26:845–51
22. Artsimovitch I, Landick R. 2002. *Cell* 109:193–203
23. Wang L, Curd H, Qu W, Reeves PR. 1998. *J. Clin. Microbiol.* 36:3182–87
24. Stevenson G, Andrianopoulos K, Hobbs M, Reeves PR. 1996. *J. Bacteriol.* 178:4885–93

25. Jayaratne P, Keenleyside WJ, MacLachlan PR, Dodgson C, Whitfield C. 1993. *J. Bacteriol.* 175:5384–94

26. Keenleyside WJ, Bronner D, Jann K, Jann B, Whitfield C. 1993. *J. Bacteriol.* 175:6725–30

27. Valvano MA. 2003. *Front. Biosci.* 8:S452–71

28. Troy FA, Frerman FE, Heath EC. 1971. *J. Biol. Chem.* 246:118–33

29. Drummelsmith J, Whitfield C. 1999. *Mol. Microbiol.* 31:1321–32

30. Whitfield C, Paiment A. 2003. *Carbohydr. Res.* 338:2491–502

31. Price NP, Momany FA. 2005. *Glycobiology* 15:R29–42

32. Wang L, Liu D, Reeves PR. 1996. *J. Bacteriol.* 178:2598–604

33. Pelosi L, Boumedienne M, Saksouk N, Geiselman J, Geremia RA. 2005. *Biochem. Biophys. Res. Commun.* 327:857–65

34. Daniels C, Vindurampulle C, Morona R. 1998. *Mol. Microbiol.* 28:1211–22

35. McGrath BC, Osborn MJ. 1991. *J. Bacteriol.* 173:649–54

36. Liu D, Cole R, Reeves PR. 1996. *J. Bacteriol.* 178:2102–7

37. Feldman MF, Marolda CL, Monteiro MA, Perry MB, Parodi AJ, Valvano MA. 1999. *J. Biol. Chem.* 274:35129–38

38. Paulsen IT, Beness AM, Saier MHJ. 1997. *Microbiology* 143:2685–99

39. Rick PD, Barr K, Sankaran K, Kajimura J, Rush JS, Waechter CJ. 2003. *J. Biol. Chem.* 278:16534–42

40. Marolda CL, Vicarioli J, Valvano MA. 2004. *Microbiology* 150:4095–105

41. Zhou GP, Troy FA 2nd. 2003. *Glycobiology* 13:51–71

42. Zhou GP, Troy FA 2nd. 2005. *Glycobiology* 15:347–59

43. Schild S, Lamprecht AK, Reidl J. 2005. *J. Biol. Chem.* 280:25936–47

44. Whitfield C, Amor PA, Köplin R. 1997. *Mol. Microbiol.* 23:629–38

45. Dodgson C, Amor P, Whitfield C. 1996. *J. Bacteriol.* 178:1895–902

46. Franco AV, Liu D, Reeves PR. 1996. *J. Bacteriol.* 178:1903–7

47. Morona R, Van den Bosch L, Daniels C. 2000. *Microbiology* 146:1–4

48. Wugeditsch T, Paiment A, Hocking J, Drummelsmith J, Forrester C, Whitfield C. 2001. *J. Biol. Chem.* 276:2361–71

49. Cozzone AJ, Grangeasse C, Doublet P, Duclos B. 2004. *Arch. Microbiol.* 181:171–81

50. Grangeasse C, Doublet P, Cozzone AJ. 2002. *J. Biol. Chem.* 277:7127–35

51. Paiment A, Hocking J, Whitfield C. 2002. *J. Bacteriol.* 184:6437–47

52. Vincent C, Duclos B, Grangeasse C, Vaganay E, Riberty M, et al. 2000. *J. Mol. Biol.* 304:311–21

53. Doublet P, Grangeasse C, Obadia B, Vaganay E, Cozzone AJ. 2002. *J. Biol. Chem.* 277:37339–48

54. Collins RF, Beis K, Clarke BR, Ford RC, Hulley M, et al. 2005. *J. Biol. Chem.* 281:2144–50

55. Daniels C, Morona R. 1999. *Mol. Microbiol.* 34:181–94

56. Drummelsmith J, Whitfield C. 2000. *EMBO J.* 19:57–66

57. Nesper J, Hill CM, Paiment A, Harauz G, Beis K, et al. 2003. *J. Biol. Chem.* 278:49763–72

58. Kostakioti M, Newman CL, Thanassi DG, Stathopoulos C. 2005. *J. Bacteriol.* 187:4306–14

59. Beis K, Collins RF, Ford RC, Kamis AB, Whitfield C, Naismith JH. 2004. *J. Biol. Chem.* 279:28227–32

60. Koronakis V, Eswaran J, Hughes C. 2004. *Annu. Rev. Biochem.* 73:467–89

61. Beis K, Nesper J, Whitfield C, Naismith JH. 2004. *Acta Crystallogr. Sect. D* 60:558–60

61a. Christie PJ, Atmakuri K, Krishnamoorthy V, Jakubowski S, Cascales E. 2005. *Annu. Rev. Microbiol.* 59:451–85

62. Bayer ME, Thurow H. 1977. *J. Bacteriol.* 130:911–36

63. Bayer ME. 1991. *J. Struct. Biol.* 107:268–80

64. Ilan O, Bloch Y, Frankel G, Ullrich H, Geider K, Rosenshine I. 1999. *EMBO J.* 18:3241–48

65. Rahn A, Beis K, Naismith JH, Whitfield C. 2003. *J. Bacteriol.* 185:5882–90

66. Silver RP, Finn CW, Vann WF, Aaronson W, Schneerson R, et al. 1981. *Nature* 289:696–98

67. Roberts IS. 1996. *Annu. Rev. Microbiol.* 50:285–315

68. Barrett B, Ebah L, Roberts IS. 2002. *Curr. Top. Microbiol. Immunol.* 264:137–55

69. Simpson DA, Hammarton TC, Roberts IS. 1996. *J. Bacteriol.* 178:6466–74

70. Gieslewicz M, Vimr E. 1996. *J. Bacteriol.* 178:3212–20

71. Stevens MP, Clarke BR, Roberts IS. 1997. *Mol. Microbiol.* 24:1001–12

72. Rowe S, Hodson N, Griffiths G, Roberts IS. 2000. *J. Bacteriol.* 182:2741–45

73. Deszo EL, Steenbergen SM, Freedberg DI, Vimr ER. 2005. *Proc. Natl. Acad. Sci. USA* 102:5564–69

74. Pearce R, Roberts IS. 1995. *J. Bacteriol.* 177:3992–97

75. Cho JW, Troy FA 2nd. 1994. *Proc. Natl. Acad. Sci. USA* 91:11427–31

76. Lindahl U, Li JP, Kusche-Gullberg M, Salmivirta M, Alaranta S, et al. 2005. *J. Med. Chem.* 48:349–52

77. Troy FA 2nd. 1992. *Glycobiology* 2:5–23

78. Vimr ER, Steenbergen SM. 1993. In *Polysialic Acid from Microbes to Man*, ed. J Roth, U Rutishauser, FA Troy 2nd, pp. 73–91. Basel: Birkhäuser-Verlag

79. Steenbergen SM, Vimr ER. 1990. *Mol. Microbiol.* 4:603–11

80. Steenbergen SM, Wrona TJ, Vimr ER. 1992. *J. Bacteriol.* 174:1099–108

81. McGowen MM, Vionnet J, Vann WF. 2001. *Glycobiology* 11:613–20

82. Steenbergen SM, Vimr ER. 2003. *J. Biol. Chem.* 278:15349–59

83. Charnock SJ, Henrissat B, Davies GJ. 2001. *Plant Physiol.* 125:527–31

84. Shen GJ, Datta AK, Izumi M, Koeller KM, Wong CH. 1999. *J. Biol. Chem.* 274:35139–46

85. Lidholt K, Fjelstad M. 1997. *J. Biol. Chem.* 272:2682–87

86. Finke A, Bronner D, Nikolaev AV, Jann B, Jann K. 1991. *J. Bacteriol.* 173:4088–94

87. Lidholt K, Fjelstad M, Jann K, Lindahl U. 1994. *Carbohydr. Res.* 255:87–101

88. Ninomiya T, Sugiura N, Tawada A, Sugimoto K, Watanabe H, Kimata K. 2002. *J. Biol. Chem.* 277:21567–75

89. Griffiths G, Cook NJ, Gottfridson E, Lind T, Lidholt K, Roberts IS. 1998. *J. Biol. Chem.* 273:11752–57

90. Hodson N, Griffiths G, Cook N, Pourhossein M, Gottfridson E, et al. 2000. *J. Biol. Chem.* 275:27311–15

91. DeAngelis PL, White CL. 2002. *J. Biol. Chem.* 277:7209–13

92. Higa HH, Varki A. 1988. *J. Biol. Chem.* 263:8872–78

93. Pelkonen S, Häyrinen J, Finne J. 1988. *J. Bacteriol.* 170:2646–53

94. Clarke BR, Cuthbertson L, Whitfield C. 2004. *J. Biol. Chem.* 279:35709–18

95. Forsee WT, Cartee RT, Yother J. 2000. *J. Biol. Chem.* 275:25972–78

96. Troy FA, Vijay IK, Tesche N. 1975. *J. Biol. Chem.* 250:156–63

97. Pakkiri LS, Waechter CJ. 2005. *Glycobiology* 15:291–302

98. Rohr TE, Troy FA. 1980. *J. Biol. Chem.* 255:2332–42

99. Rodriguez-Aparicio LB, Reglero A, Ortiz AI, Luengo JM. 1988. *Biochem. J.* 251:589–96

100. Weisgerber C, Troy FA. 1990. *J. Biol. Chem.* 265:1578–87

101. Bliss JM, Silver RP. 1997. *J. Bacteriol.* 179:1400–3

102. Cartee RT, Forsee WT, Yother J. 2005. *J. Bacteriol.* 187:4470–79

103. Davidson AL, Chen J. 2004. *Annu. Rev. Microbiol.* 73:241–68

104. Bliss JM, Silver RP. 1996. *Mol. Microbiol.* 21:221–31

105. Silver RP, Prior K, Nsahlai C, Wright LF. 2001. *Res. Microbiol.* 152:357–64

106. Reizer J, Reizer A, Saier MH Jr. 1992. *Protein Sci.* 1:1326–32

107. Saier MH Jr. 2000. *Mol. Microbiol.* 35:699–710

108. Kröncke K-D, Boulnois G, Roberts I, Bitter-Suermann D, Golecki JR, et al. 1990. *J. Bacteriol.* 172:1085–91

109. Pavelka MS Jr, Hayes SF, Silver RP. 1994. *J. Biol. Chem.* 269:20149–58

110. Bliss JM, Garon CF, Silver RP. 1996. *Glycobiology* 6:445–52

110a. Reyes CL, Chang G. 2005. *Science* 308:1028–31

111. Bronner D, Sieberth V, Pazzani C, Roberts IS, Boulnois GJ, et al. 1993. *J. Bacteriol.* 175:5984–92

112. Meredith TC, Woodard RW. 2003. *J. Biol. Chem.* 278:32771–77

113. Tzeng YL, Datta A, Strole C, Kolli VS, Birck MR, et al. 2002. *J. Biol. Chem.* 277:24103–13

114. Rosenow C, Roberts IS, Jann K. 1995. *FEMS Microbiol. Lett.* 125:159–64

115. Cieslewicz M, Vimr E. 1997. *Mol. Microbiol.* 26:237–49

116. Frosch M, Müller A. 1993. *Mol. Microbiol.* 8:483–93

117. Tzeng YL, Datta AK, Strole CA, Lobritz MA, Carlson RW, Stephens DS. 2005. *Infect. Immun.* 73:1491–505

118. Vimr ER, Kalivoda KA, Deszo EL, Steenbergen SM. 2004. *Microbiol. Mol. Biol. Rev.* 68:132–53

119. Bronner D, Sieberth V, Pazzani C, Smith A, Boulnois G, et al. 1993. *FEMS Microbiol. Lett.* 113:279–84

120. Vimr ER, Aaronson W, Silver RP. 1989. *J. Bacteriol.* 171:1106–17

121. Rigg GP, Barrett B, Roberts IS. 1998. *Microbiology* 144:2905–14

122. Silver RP, Aaronson W, Vann WF. 1987. *J. Bacteriol.* 169:5489–95

123. Pazzani C, Rosenow C, Boulnois GJ, Bronner D, Jann K, Roberts IS. 1993. *J. Bacteriol.* 175:5978–83

124. Eswaran J, Koronakis E, Higgins MK, Hughes C, Koronakis V. 2004. *Curr. Opin. Struct. Biol.* 14:741–47

125. Arrecubieta C, Hammarton TC, Barrett B, Chareonsudjai S, Hodson N, et al. 2001. *J. Biol. Chem.* 276:4245–50

126. Rosenow C, Esumeh F, Roberts IS, Jann K. 1995. *J. Bacteriol.* 177:1137–43

127. Phoenix DA, Brandenburg K, Harris F, Seydel U, Hammerton T, Roberts IS. 2001. *Biochem. Biophys. Res. Commun.* 285:976–80

128. Krönke K-D, Golecki JR, Jann K. 1990. *J. Bacteriol.* 172:3469–72

129. Karlyshev AV, Linton D, Gregson NA, Lastovica AJ, Wren BW. 2000. *Mol. Microbiol.* 35:529–41

130. Wu T, Malinverni J, Ruiz N, Kim S, Silhavy TJ, Kahne D. 2005. *Cell* 121:235–45

131. Matias VR, Al-Amoudi A, Dubochet J, Beveridge TJ. 2003. *J. Bacteriol.* 185:6112–18

132. Whitfield C, Vimr ER, Costerton JW, Troy FA. 1984. *J. Bacteriol.* 159:321–28

133. Reid AN, Whitfield C. 2005. *J. Bacteriol.* 187:5470–81

134. Graham LL, Harris R, Villiger W, Beveridge TJ. 1991. *J. Bacteriol.* 173:1623–33

135. Jann K, Jann B, Schneider KF. 1968. *Eur. J. Biochem.* 5:456–65  
136. Choy Y-M, Fehmel F, Frank N, Stirm S. 1975. *J. Virol.* 16:581–90  
137. Chakraborty AK, Friebolin H, Stirm S. 1980. *J. Bacteriol.* 141:971–72  
138. McGuire EJ, Binkley SB. 1964. *Biochemistry* 3:247–51  
139. Fischer W, Schmidt MA, Jann B, Jann K. 1982. *Biochemistry* 21:1279–84  
140. Rodriguez ML, Jann B, Jann K. 1988. *Eur. J. Biochem.* 177:117–24  
141. Vann WF, Schmidt MA, Jann B, Jann K. 1981. *Eur. J. Biochem.* 116:359–64  
142. Schmidt MA, Jann K. 1983. *Eur. J. Biochem.* 131:509–17  
143. Vann WF, Jann K. 1979. *Infect. Immun.* 25:85–92  
144. Egan W, Liu T-Y, Dorow D, Cohen JS, Robbins JD, et al. 1977. *Biochemistry* 16:3687–92  
145. Rodriguez ML, Jann B, Jann K. 1988. *Carbohydr. Res.* 173:243–53  
146. Sie Barth V, Jann B, Jann K. 1993. *Carbohydr. Res.* 246:219–28  
147. Hoffman P, Jann B, Jann K. 1985. *Carbohydr. Res.* 139:261–71  
148. Kenne L, Lindberg B, Söderstrom E, Bundle DR, Griffith DW. 1983. *Carbohydr. Res.* 111:289–96  
149. Dengler T, Jann B, Jann K. 1986. *Carbohydr. Res.* 150:233–40.  
150. Garegg, PJ, Lindberg B, Onn T, Holme T. 1971. *Acta Chem. Scand.* 25:2103–8  
151. Andreishcheva EN, Vann WF. 2006. *J. Bacteriol.* 188:1786–97  
152. McNulty C, Thompson J, Barrett B, Lord L, Anderson C, Roberts IS. 2006. *Mol. Microbiol.* 59:907–22  

---

### RELATED RESOURCE

Luirink J, von Heijne G, Houben E, de Gier JW. 2005. *Annu. Rev. Microbiol.* 59:329–55  

---

### NOTE ADDED IN PROOF

While this review was in proof, two papers that provide important new insight into group 2 capsule assembly were published (151, 152):

- Work performed in the laboratory of Vann has identified the minimal complement of gene products necessary for de novo synthesis of polysialic acid in *E. coli* K1 (151). Synthesis requires the precursor (CMP-Neu5Ac), together with the polysialyltransferase (NeuS) and the NeuE and KpsC proteins. The additional presence of KpsS dramatically increases the level of polymer made. These observations now set the stage for deciphering the mechanism of polymer initiation in a reaction that minimally involves NeuES and KpsC.

- Research from Roberts' group has identified a *trans*-envelope complex involved in biosynthesis of the K5 capsule (152). This complex contains the K5 GT enzymes, KpsS, the ABC transporter (KpsMT), the periplasmic KpsE protein, and KpsD. KpsD is confirmed as an outer membrane-spanning protein, thus reinforcing its candidacy as an OMA representative. In an unanticipated observation, KpsS and KpsD colocalize to the poles of the cell and are coincident with the export of nascent K5 polymer. This suggests a requirement for diffusion of polymer across the cell surface during encapsulation. While the universality of this concept has yet to be established, the findings raise intriguing questions concerning capsule growth and its integration with the cell cycle.

# Contents

Wanderings of a DNA Enzymologist: From DNA Polymerase to Viral Latency  
**I. Robert Lehman** . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Structure and Mechanism of the Hsp90 Molecular Chaperone Machinery  
*Laurence H. Pearl and Chrisostomos Prodromou* ………………………………………… 271  

Biochemistry of Mammalian Peroxisomes Revisited  
*Ronald J.A. Wanders and Hans R. Waterham* ………………………………………… 295  

Protein Misfolding, Functional Amyloid, and Human Disease  
*Fabrizio Chiti and Christopher M. Dobson* ………………………………………… 333  

Obesity-Related Derangements in Metabolic Regulation  
*Deborah M. Muoio and Christopher B. Newgard* ………………………………………… 367  

Cold-Adapted Enzymes  
*Khawar Sohail Siddiqui and Ricardo Cavicchioli* ………………………………………… 403  

The Biochemistry of Sirtuins  
*Anthony A. Sauve, Cynthia Wolberger, Vern L. Schramm, and Jef D. Boeke* ………………………………………… 435  

Dynamic Filaments of the Bacterial Cytoskeleton  
*Katharine A. Michie and Jan Löwe* ………………………………………… 467  

The Structure and Function of Telomerase Reverse Transcriptase  
*Chantal Autexier and Neal F. Lue* ………………………………………… 493  

Relating Protein Motion to Catalysis  
*Sharon Hammes-Schiffer and Stephen J. Benkovic* ………………………………………… 519  

Animal Cytokinesis: From Parts List to Mechanisms  
*Ulrike S. Eggert, Timothy J. Mitchison, and Christine M. Field* ………………………………………… 543  

Mechanisms of Site-Specific Recombination  
*Nigel D.F. Grindley, Katrine L. Whiteson, and Phoebe A. Rice* ………………………………………… 567  

Axonal Transport and Alzheimer’s Disease  
*Gorazd B. Stokin and Lawrence S.B. Goldstein* ………………………………………… 607  

Asparagine Synthetase Chemotherapy  
*Nigel G.J. Richards and Michael S. Kilberg* ………………………………………… 629  

Domains, Motifs, and Scaffolds: The Role of Modular Interactions in the Evolution and Wiring of Cell Signaling Circuits  
*Roby P. Bhattacharyya, Attila Reményi, Brian J. Yeh, and Wendell A. Lim* ………………………………………… 655  

Ribonucleotide Reductases  
*Pär Nordlund and Peter Reichard* ………………………………………… 681  

Introduction to the Membrane Protein Reviews: The Interplay of Structure, Dynamics, and Environment in Membrane Protein Function  
*Jonathan N. Sachs and Donald M. Engelman* ………………………………………… 707  

Relations Between Structure and Function of the Mitochondrial  
ADP/ATP Carrier  
H. Nury, C. Dahout-Gonzalez, V. Trézéguet, G.J.M. Lauquin,  
G. Brandolin, and E. Pebay-Peyroula ………………………………………… 713  

G Protein–Coupled Receptor Rhodopsin  
Krzysztof Palczewski ……………………………………………………………… 743  

Transmembrane Traffic in the Cytochrome $b_{6} f$ Complex  
William A. Cramer, Huamin Zhang, Jiusheng Yan, Genji Kurisu,  
and Janet L. Smith ……………………………………………………………… 769  

INDEXES  
Subject Index …………………………………………………………………… 791  
Author Index …………………………………………………………………… 825  

ERRATA  

An online log of corrections to *Annual Review of Biochemistry* chapters (if any, 1977 to  
the present) may be found at http://biochem.annualreviews.org/errata.shtml
