
Available online at www.sciencedirect.com

**SCIENCE DIRECT**

Biochemical and Biophysical Research Communications 346 (2006) 583–590

---

**Substrate specificity of Arabidopsis 3-ketoacyl-CoA synthases**

Brenda J. Blacklock ${}^{a,*}$, Jan G. Jaworski ${}^{b}$

${}^{a}$ Department of Chemistry and Chemical Biology, Purdue School of Science, Indiana University-Purdue University Indianapolis (IUPUI), Indianapolis, IN 46202, USA  
${}^{b}$ Donald Danforth Plant Science Center, 975 North Warson Road, St. Louis, MO 63132, USA  

Received 23 May 2006  
Available online 5 June 2006

---

### Abstract

The very long chain fatty acids (VLCFA) incorporated into plant lipids are derived from the iterative addition of C2 units provided by malonyl-CoA to an acyl-CoA by the 3-ketoacyl-CoA synthase (KCS) component of a fatty acid elongase (FAE) complex. Mining of the Arabidopsis genome sequence database revealed 20 genes with homology to seed-specific $ FAE1 \ KCS $. Eight of the 20 putative KCSs were cloned, expressed in yeast, and isolated as $(His)_6$ fusion proteins. Five of the eight ($At1g71160$, $At1g19440$, $At1g07720$, $At5g04530$, and $At4g34250$) had little or no activity with C16 to C20 substrates while three demonstrated activity with C16, C18, and C20 saturated acyl-CoA substrates. $At1g01120 \ KCS$ ($KCS1$) and $At2g26640 \ KCS$ had broad substrate specificities when assayed with saturated and mono-unsaturated C16 to C24 acyl-CoAs while $At4g34510 \ KCS$ was specific for saturated fatty acyl-CoA substrates.

© 2006 Elsevier Inc. All rights reserved.

---

### Keywords: Arabidopsis; 3-Ketoacyl-CoA synthase; Fatty acid elongation; Condensing enzyme

---

Very long chain fatty acids (VLCFA) are incorporated into a variety of plant lipids including seed oil, epicuticular wax, membrane sphingolipids, cutin, suberin, and those lipids found in the extra-cellular pollen coat [1–3]. VLCFA are derived from the sequential elongation of de novo fatty acid biosynthesis products by the action of a fatty acid elongase (FAE) complex consisting of a 3-ketoacyl-CoA synthase (KCS), a 3-ketoacyl-CoA reductase, a dehydrase, and an enoyl-CoA reductase [4–9]. The initial and rate-limiting step in fatty acid elongation; the iterative condensation of acyl-CoA units with malonyl-CoA is catalyzed in the seed by the membrane-bound Fatty Acid Elongase 1 3-ketoacyl-CoA synthase ($FAE1 \ KCS$), an enzyme that was first identified in *Arabidopsis thaliana* [7,10]. Homologues of $FAE1 \ KCS$ have been found in oleaginous plants such as *Brassica napus*, *Brassica juncea*, *Brassica campestris*, *Brassica oleacea*, *Simmondsia chinensis*, *Lesquerella fendleri*, and *Limnanthes alba* and in the green algae, *Dunaliella salina* [11–18]. The chain length of fatty acids incorporated into seed triacylglycerols depends upon the substrate specificity of the $FAE1 \ KCS$ [7,10,16,19].

The variety of VLCFA chain lengths and degree of saturation and the diversity of lipids and tissues in which they are found suggest that many KCS enzymes with differing substrate specificities and expression patterns are involved in fatty acid elongation in the plant. $FAE1 \ KCS$ is specifically expressed in the seed where mono-unsaturated and saturated fatty acids up to C22 are incorporated into storage triacylglycerols [10,18,20,21]. Three KCSs involved in epicuticular wax production in shoots have been described: CER6, KCS1, and FDH [22–27]. CER6 and KCS1 appear to be involved in the production of C26 and longer VLCFAs for epicuticular waxes while FDH has a role in organ development [24–26]. A novel role for KCSs in the high carbon dioxide signaling pathway has recently been described in which mutant $hic$ plants, as well as $cer6$ plants, demonstrated a significant decrease in stomatal density in response to a doubling in $CO_2$ levels [28].

A number of studies of the structure/function basis of KCS catalysis and substrate specificity have been reported. An active site cysteine, predicted by alignment of $FAE1

* Corresponding author. Fax: +1 317 274 4701.  
E-mail address: blacklock@chem.iupui.edu (B.J. Blacklock).

0006-291X/$ - see front matter © 2006 Elsevier Inc. All rights reserved.  
doi:10.1016/j.bbrc.2006.05.162

KCS with the condensing enzyme, resveratrol synthase, was shown to be essential for catalytic activity, including both malonyl-CoA decarboxylation and condensation [16,29]. Other members of the decarboxylating condensing enzyme catalytic triad (His, Asn) were predicted by alignment of FAE1 KCS with 3-ketoacyl-ACP synthases (KASs) and chalcone synthase and appear to play similar roles in FAE1 KCS [30,31]. The difference in catalytic activity of FAE1 KCSs found in high erucic acid and low erucic acid *B. napus* was demonstrated to be the result of a single polymorphism when the ability to produce 20:1 and 22:1 fatty acids was restored to FAE1 KCS from a low erucic acid cultivar by the substitution of Phe282 with the serine residue found in the high erucic acid enzyme [32–34].

We have focused on the structural determinants involved in substrate specificity in FAE1 KCSs. The difference in VLCFA content in *A. thaliana* and *B. napus* seed oil is well known: *B. napus* produces a seed oil with high 22:1 content while *A. thaliana* seed oil has high 20:1 content [35,36]. Through the swapping of domains between *A. thaliana* and *B. napus* FAE1 KCSs, we demonstrated that this diversity in seed oil content stems from differences in substrate specificity of the FAE1 KCS condensing enzymes and that domains that impart chain-length substrate specificity to FAE1 KCSs are found in the amino terminal one-third of the protein excluding the transmembrane domain [19].

Further understanding of the molecular basis for substrate specificity and catalytic activity will help us to more completely comprehend the function of KCSs, decipher the relationship between membrane-bound condensing enzymes and their soluble relatives, such as the KASs and chalcone synthases, and may make possible the engineering of value-added crops producing novel seed oils. A study of the structural determinants involved in substrate specificity and catalytic activity requires the identification of KCSs with an assortment of activities. At least 20 genes in the Arabidopsis genome have homology to FAE1 KCS [37], and the proteins encoded by these genes are expected to have condensing activity with a variety of substrate specificities providing the wide range of VLCFA found in plants. We were particularly interested in identifying KCSs with substrate specificities directed toward saturated VLCFA. These KCSs will be useful in domain-swapping experiments with FAE1 KCS to identify domains/residues responsible for differentiation between saturated and mono-unsaturated fatty acyl-CoA substrates.

Characterization of known and putative membrane-bound KCSs requires a system where the enzymes can be isolated and assayed with a broad range of substrates. Metal-affinity chromatographic isolation of (His)₆KCS fusion proteins has proven useful in providing enzymes for the direct assay of condensation activity with radiolabeled malonyl-CoA and a variety of unlabeled acyl-CoAs [30]. We have directly PCR-cloned a number of putative *A. thaliana* KCS genes chosen for the predicted absence of introns and expressed them as (His)₆ fusion proteins in yeast. (His)₆KCSs solubilized from yeast microsomes and isolated by metal-affinity chromatography were screened for activity with long chain saturated fatty acids. Those enzymes with significant activity were further characterized with a battery of saturated and mono-unsaturated acyl-CoAs and KCSs with a variety of substrate specificities were identified.

## Materials and methods

### Genome sequence analysis
Putative KCS genes were identified through a tBLASTn search of the Arabidopsis genomic sequence database available at the National Center for Biotechnology Information (NCBI) website ([http://www.ncbi.nlm.nih.gov/BLAST/](http://www.ncbi.nlm.nih.gov/BLAST/)) using default parameters [37,38]. Multiple sequence alignment and phylogenetic analysis were performed with the ClustalW algorithm [39,40].

### PCR cloning of KCSs and subcloning into expression vector
Genomic DNA was prepared from *A. thaliana* (ecotype, Columbia) by a standard method. Putative KCS genes were amplified from gDNA by Vent DNA polymerase (New England Biolabs) using primers containing restriction sites convenient for subcloning into a yeast expression vector (pYES2, Invitrogen) engineered to encode a (His)₆GlySer fusion protein (Table 1). When the putative KCS gene contained the restriction sites required for insertion into the expression vector, a sticky end cloning protocol was used where the appropriate restriction site overhang was constructed by annealing two amplification products [41]. PCR products and DNA fragments were purified by agarose gel electrophoresis followed by Gene Clean (Bio 101), ligated into the vector with T4 DNA ligase (Life Technologies, Gibco-BRL) and transformed into TSS-competent XL-1 Blue *Escherichia coli* (Stratagene) by standard techniques [42]. Insert-containing plasmids were sequenced by in-house automated DNA sequencing to ensure sequence fidelity. Plasmids were transformed into competent *Saccharomyces cerevisiae* (InvSc1, Invitrogen) by the lithium acetate method of Geitz et al. [43].

### Expression of KCSs in yeast and preparation of microsomes
Transformed yeast strains were grown overnight with shaking in rich media at 30 °C. The overnight culture was used to inoculate complete minimal media lacking uracil (cm-ura) [44] supplemented with 2% (w/v) galactose to give an initial OD₆₀₀ of 0.01–0.04. The cm-ura + galactose cultures were grown to 1.5–2 OD₆₀₀ units and harvested. Microsomes were prepared as described using ice-cold isolation buffer (IB: 80 mM HEPES-KOH, pH 7.2, 5 mM EGTA, 5 mM EDTA, 10 mM KCl, 320 mM sucrose, and 2 mM dithiothreitol) [45]. Protein concentrations were determined by the method of Bradford using BSA as a standard and adjusted to 2.5 μg/μl by the addition of glycerol to 15% (v/v) and IB [46]. Microsomes were frozen on dry ice and stored at –80 °C.

### Solubilization and isolation of (His)₆-tagged fusion proteins
(His)₆ fusion proteins were solubilized and isolated as described [30]. Microsomes prepared from yeast expressing (His)₆-tagged fusion proteins were solubilized by incubation on ice for 2 h in solubilization buffer (320 mM NaCl, 0.5% Triton X-100 (v/v); 2 μg protein/μl). Insoluble material was then pelleted by ultracentrifugation at 4 °C for 1 h at 235,000g. Supernatants were combined with binding buffer (25 mM sodium phosphate, pH 8.0, 0.5% Triton X-100, 150 mM NaCl, and 10% glycerol (v/v)) at a ratio of 0.43:1 (supernatant:binding buffer) and applied to a 300-μl column of Ni²⁺-pentadentate chelator affinity matrix (Ni-PDC, Affiland, Ans-Liege, Belgium). The column was washed with 1 ml binding buffer, 1 ml wash buffer (25 mM sodium phosphate, pH 8.0, 0.5% Triton X-100, 500 mM NaCl, 10% glycerol, and 20 mM imidazole), and 1 ml binding buffer. Proteins were eluted from the column with 300 μl elution buffer (25 mM sodium phosphate, pH 8.0, 0.5% Triton X-100, 500 mM NaCl, 10% glycerol, and 300 mM imidazole), dithiothreitol added to 2 mM, and samples were frozen on dry ice and stored at –80 °C. The protein concentration of acetone-precipitated eluants was determined by MicroBCA assay (Pierce Chemical Company, Rockford, IL). Mock solubilization

Table 1  
Primers used for PCR cloning of KCS genes from A. thaliana gDNA  

| Primer                     | Sequence                                                                 |
|----------------------------|--------------------------------------------------------------------------|
| At1g01120s@4BamHI          | GGGGATCCGAGAGAAACAAACAGCATTGAG                                          |
| At1g01120a@1563SphI        | GAATTCGCATGCTCATTGCAACAACTTTAACCGG                                      |
| At1g07720s@4BamHI          | GGGGATCCGATCTTCTTGTATGCTCCTC                                             |
| At1g07720a@1416EcoRI       | CCGAATTCTCACATAAATCCTTCGGGCAT                                           |
| At1g19440s@4+1             | CGACGGTGCCGGAGAATCACGA                                                  |
| At1g19440s@4BamHIoh        | GATCCGACGGTGCCGGAGAATCACGA                                              |
| At1g19440a@1531EcoRI       | CCGAATTCCCTAATAACTTAAAGTTACCGG                                          |
| At1g71160s@4BamHI          | GGGGATCCGAATCGAGCTTTCATTCATC                                            |
| At1g71160a@1363EcoRI       | CCGAATTCTCAAGAAAAGAAGCTCTGTCTT                                          |
| At2g26640s@1BamHI          | GGGGATCCATGGATGTAGAGCAAAGAAAC                                           |
| At2g26640a@1508KpnI        | GGGGTACCCTAGATTGTGGAGACCTTGGGG                                          |
| At4g34250s@4BamHI          | GGGGATCCGATTACCCCATGAAGAAGGT                                            |
| At4g34250a@1482SphI        | GAATTCGCATGCTCACTTTTAAATCTATAT                                          |
| At4g34510s@4+1             | CGATGCTAATGGGAGGACCTGTA                                                  |
| At4g34510s@4BamHIoh        | GATCCGATGCTAATGGGAGGACCTGTA                                              |
| At4g34510a@1444EcoRI       | CCGAATTCTCAAAGATCGATCTTAAACCGG                                          |
| At5g04530s@4BamHI          | GGGGATCCGAGCTCTTCTCACTCTCTTC                                            |
| At5g04530a@1395SphI        | GAATTCGCATGCTCAATCAACACGAAACGA                                          |

reactions and Ni–PDC chromatography were carried out with microsomes prepared from induced yeast transformed with the empty expression vector, pYES2.

Condensation assay. Condensation activity of (His)₆KCSs was assayed as described [30, 47]. Briefly, isolated sample (6.25 μg) was incubated with condensation reaction mix (25 μl, 10 mM sodium phosphate, pH 7.2, 0.05% Triton X-100, 15 μM acyl-CoA, and 20 μM) [1-¹⁴C] malonyl-CoA (42 μCi/μmol) at 30°C. The concentrations of the acyl-CoAs were determined by measuring the absorbance of the stock solution at 260 nm and calculating the concentration using the molar extinction coefficient of coenzyme A (16,800 M⁻¹ cm⁻¹). Radiolabeled malonyl-CoA was prepared with isolated pea chloroplasts as described [48]. 3-Ketoacyl-CoA reaction products were reduced to the diols with NaBH₄, extracted into petroleum ether, evaporated under a N₂ stream, and quantified by liquid scintillation analysis [47]. Mock-isolated samples were similarly assayed as negative control samples (pYES2, empty vector control).

Results and discussion

Our goal was to combine the resource of the completely sequenced Arabidopsis genome with biochemical characterization to identify novel KCSs with substrate specificity toward saturated substrates. A protein query: translated database (tBLASTn) search of the Arabidopsis genome sequence available at the NCBI website revealed the presence of 20 homologues of FAE1/KCS [37, 38]. One additional gene with a high degree of homology to FAE1 KCS (At3g10280) was detected which does not possess a conserved active site cysteine and appears to be derived from the duplication of another KCS gene (At2g46720; HIC) [28]. A phylogenetic tree (Fig. 1) was assembled from a multiple sequence alignment by the ClustalW algorithm using default parameters [39, 40].

The genes for a number of the putative KCS homologues were annotated by the genome sequencing project as uninterrupted open-reading frames [37]. Due to ease of cloning, KCS homologues without predicted introns were directly cloned from A. thaliana gDNA (ecotype, Columbia)

Fig. 1. Phylogenetic tree for known (FAE1 KCS, CER6, CER60, KCS1, HIC, and FDH) and putative A. thaliana KCSs. Putative KCSs are named by the loci designation in the Arabidopsis genome. Sequence alignments were assembled by the ClustalW algorithm version 1.83 [38].

bia) by PCR amplification and subcloned into a yeast expression vector engineered to encode a (His)₆ fusion protein. (His)₆KCSs were expressed in S. cerevisiae and isolated from solubilized microsomes. Isolation of (His)₆KCSs

enabled us to directly assay condensation activity with a broad variety of acyl-CoA substrates as the condensation assay was carried out with radiolabeled malonyl-CoA and commercially available unlabeled acyl-CoAs. These assays allowed us to examine the relative substrate specificities of the KCSs without the interference of the yeast enzymes required to complete the elongation cycle in a classical microsomal elongase assay.

Fig. 2 shows the results from a survey of the condensation activity of (His)₆KCSs with saturated C16, C18, and C20 acyl-CoA substrates. These acyl-CoAs were selected for the survey assay because we were primarily interested in identifying KCS enzymes with activity toward saturated acyl substrates for further studies into the domains responsible for substrate specificity. We found (His)₆KCSs with a range of condensation activities toward 16:0, 18:0, and 20:0 acyl-CoA substrates: At4g34510 KCS, At2g26640 KCS, and At1g01120 KCS. The loci, *At1g01120* encodes an enzyme identical to the gene characterized by Todd et al. (KCS1); however, the gene product indicated by the annotated genome has eight additional amino acids at the amino terminus than KCS1 as originally cloned [26]. In this manuscript, *At1g01120* will be referred to as *KCS1* and includes the eight additional amino terminal residues. Comparison of the condensation activity of the truncated (His)₆ fusion protein with the (His)₆ enzyme predicted by the genome

annotation revealed similar substrate specificity (data not shown).

Several putative KCSs did not have activity when assayed as isolated (His)₆ fusion proteins in the initial survey assay (At1g71160 KCS, At1g19440 KCS, At1g07720 KCS, At5g04530 KCS, and At4g34250 KCS). This result may be due to improper folding of the fusion proteins in yeast. However, this seems unlikely as there are many examples of expression of active plant KCSs in yeast (this report [7,19,26,29,30,32,34,49,50]). KCSs are responsible for the condensation activity of elongases involved in producing VLCFA from C20 to C34. We, therefore, expect that many of the KCSs present in the plant will not have activity toward the acyl-CoAs assayed in the survey but will be active toward acyl-CoAs longer than C20. As our goal is to find enzymes with condensation activity toward C16 to C20 saturated fatty acids, we did not pursue further the relative substrate specificities of these putative KCSs.

We characterized the relative substrate specificity of the three KCSs identified in the survey assay and *FAE1* KCS with a broader battery of acyl-CoA substrates (16:0, 16:1, 18:0, 18:1, 20:0, 20:1, 22:0, 22:1, and 24:0). Fig. 3 demonstrates that the four (His)₆KCSs described here have distinct relative substrate specificities. As demonstrated previously, (His)₆*FAE1* KCS is specific for acyl-CoAs of 16 and 18 carbons in length and has activity toward both

![Figure 2](attachment://image.png)

Fig. 2. Condensation assay of isolated (His)₆KCSs. Microsomes prepared from yeast expressing putative KCSs as (His)₆ fusion proteins were solubilized by incubation on ice in 0.32 M NaCl and 0.5% Triton X-100. Insoluble material was pelleted by centrifugation, the supernatant fraction collected, and (His)₆ fusion proteins isolated by Ni²⁺-affinity chromatography as described [30]. Eluted fractions (5 μl) were incubated with [1-¹⁴C]malonyl-CoA (20 μM), 16:0CoA, 18:0CoA or 20:0CoA (15 μM) and 40 mM NaPO₄, pH 7.2, at 30 °C for 18 h. 3-Ketoacyl-CoA condensation products were reduced to the diols, extracted and quantitated by liquid scintillation counting [47]. The activities are expressed as pmol product and are representative of experiments carried out three times. The maximum amount of condensation product (250 pmol) is 50% of the total [1-¹⁴C]malonyl-CoA present in the reaction.

Fig. 3. Condensation assay of isolated (His)₆KCSs that demonstrated activity toward saturated fatty acids with a battery of acyl-CoAs. (A) Isolated (His)₆ fusion proteins were assayed for protein concentration and equal amounts of purified protein were incubated with [1-¹⁴C]malonyl-CoA (20 μM), 16:0CoA, 16:1CoA, 18:0CoA, 18:1CoA, 20:0CoA, 20:1CoA, 22:0CoA, 22:1CoA or 24:0CoA (15 μM) and 40 mM NaPO₄, pH 7.2, at 30 °C for 18 h. 3-Ketoacyl-CoA condensation products were processed as described. Condensation activities are expressed as pmol product/μg protein (mean ± SD for *n* = 3) and are typical of assays carried out several times. (B) Replot of condensation assay of isolated (His)₆KCSs that demonstrated activity toward saturated fatty acids with a battery of acyl-CoAs for enzymes with lower activity, (His)₆At2g26640 KCS and (His)₆At4g34510 KCS.

saturated and mono-unsaturated acyl primers [30]. (His)₆At2g26640 KCS and (His)₆KCS1 were active toward a variety of saturated and mono-unsaturated acyl-CoA substrates. The ability of (His)₆At2g26640 KCS to use both saturated and mono-unsaturated acyl chains C16 to C20 in length demonstrates its broad substrate specificity (Fig. 3B). (His)₆KCS1 (Fig. 3A) was most active with 16:0, 16:1, 18:0, and 20:0 but had very little activity with 18:1 and acyl chains longer than C20. (His)₆At4g34510 KCS (Fig. 3B) was specific for saturated acyl-CoAs up to C22.

The high levels of activity toward the shorter chain acyl-CoAs may be the result of flexible substrate binding sites in some of the KCSs described here. At2g26640 KCS appears to have the broadest substrate specificity with the ability to use both saturated and mono-unsaturated fatty acyl-CoAs from C16 to C20 while At4g34510 KCS is much more restrictive in the level of unsaturation tolerated in its substrates. The low level of activity of KCS1 toward 18:1CoA is interesting: although the binding site of this enzyme is flexible in the chain length of substrate it accepts, it is unable to accept 18:1CoA in particular.

The substrate specificity of (His)₆KCS1, measured here by condensation activity, is similar to that found in an elongation assay in yeast microsomes where KCS1 demonstrated greatest activity toward [¹⁴C-1]18:0CoA and had some activity with 20:0, 22:0, and 24:0 substrates [26]. The substrate specificity of KCS1 as assayed in vitro, however, is different than that predicted by the phenotypic changes in wax composition in a KCS1-tagged plant line

in which KCS1 expression is disrupted where there were large decreases in C26 to C30 wax alcohols and aldehydes and smaller effects on C29 alkanes and ketones [26]. Together, these results suggest that KCS1 catalyzes an early elongation step in the acyl reduction pathway of wax biosynthesis [1,51].

Alignment of the primary sequences of KCSs that demonstrated condensation activity in these experiments revealed 51%, 54%, and 61% sequence identity for KCS1, At2g26640 KCS and At4g34510 KCS, with FAE1 KCS, respectively (Fig. 4) [26]. Examination of the aligned sequences revealed that residues that play a role in KCS catalysis are conserved among the four KCSs presented here. The putative catalytic triad is maintained among the KCSs: Cys 223, His 391, and Asn 424 in FAE1 KCS [29–31]. The residue demonstrated to be responsible for low erucic acid production in B. napus low erucic acid cultivars (Ser 282 in FAE1 KCS) is also conserved [32,34].

Conserved residues important in catalysis are found in stretches of high sequence similarity, while other domains are relatively non-conserved. For example, the amino-terminal domains, including the putative membrane-spanning domains, have a low degree of homology and, while potentially of structural importance, these regions have not been identified to contain key catalytic residues. This is consistent with our demonstration that the amino terminus contains determinants of chain-length substrate specificity since these enzymes differ in the chain length of acyl-CoA primer they prefer [19].

Recent reports of heterologous expression of A. thaliana KCS genes in yeast have included a number of the enzymes studied here [49,50]. In a screen of putative A. thaliana KCSs that are inhibited by K3 and N-class herbicides, Trenkamp et al. expressed a number of KCSs in yeast and assayed elongation activity by examining VLCFA as a % of total cellular fatty acid methyl esters. Of the nine


plaintext
1    MTSVN-----------------------------VKLLY   FAE1 KCS
1    MDVEQKKPL-------------------------IESSDRNLPDFFKKSVKLLKYVKLGY   At2g26640 KCS
1    MDANGGPVQIRTQ--------------------NYSVKLLKYVKLGY   At4g34510 KCS
1    MERTNSIEMDRERLTAEMAFRDSSSAVIRIRRRLPDLLTTSVKLLKYVKLGL   KCS1
11   RY----VLTNFFNLCLFPPLTAFLAGKASRLTINDLHNFLSYLQHNLITVTL   FAE1 KCS
34   HY---LITHGMYLFSLPVLVIAAQISTFSVTDLRSLSWEH--LYNYLISV   At2g26640 KCS
21   HY---LITHFFFKLMFLPLMAVLFMNVSLLSNHLQLYY--YNSTGFIF--F   At4g34510 KCS
51   HNSCNVTTILFLIIPLTGTVLVQLTFDTFSELWSNQAVALDDTATR   KCS1
58   LF---AFTVFGLVLYIVTRPNPVYLVVDYSCYLPPPHLKVSVSCKVMDFIFYQI   FAE1 KCS
79   VVCSMLLVFLMTIYYFMRPRPVYLVNFSCFKPDESRKCTKKIFMDRSKLT   At2g26640 KCS
63   VI---TLAIVGSIVFFMSRRPSIYLLDYSCYLPPSQKVSYQKFMNNSSLI   At4g34510 KCS
101  LTCLVFLSFVLTLYVANRSKPVYLVDFSCKPEDERRKISVDSEFLTMTREEN   KCS1
106  RKADTSSRNVACDDPSSLDFLRKIQERSGLGDETYSPPEGLIHVPPRKTFA   FAE1 KCS
129  GSFTEEN--------LEFQRKILQRSGLGESTYLPEAVLNVPPNPCMK   At2g26640 KCS
111  QDFSETS--------LEFQRKILLRSGLGEETYLPDSIHSIPPRTMA   At4g34510 KCS
151  GSFTDDT--------VQFQQRIISNRAGLGDETYLPRGITSTPPKLNMS   KCS1
156  ASREETEKVIIGALENLFENTKVNPREIGILVVNSSMFNPTTPSLSAMVVN   FAE1 KCS
169  EARKEAETVMFGAIDELLAKTNVNPKDIGILIIVNCSLFNPTTPSLSAMVVN   At2g26640 KCS
151  AAREEEAEQVIFGALDNLFENTKINPREIGVLLVNCSSLFNPTTPSLSAMIVN   At4g34510 KCS
191  EARAEEAEAVMFGALDSLFEKTGIKPAAEVGILIIVNCSSLFNPTTPSLSAMIVN   KCS1
206  TFKLRSNTIKSFNLGGMGCSAGVTATDLAKDLLHVHKNTYALVVSTENITQ   FAE1 KCS
219  HYKLRGNILSYNLGGMGCSAGLISIDLAKHLLHSIPNTYAMVISMENITL   At2g26640 KCS
201  KYKLRGINIKSFNLGGMGCSAGVIAVDLASDMLOIHRTFAVVSTENITQ   At4g34510 KCS
241  HYKMREDIKSYNLGGMGCSAGLISIDLANNLLKANPNSYA VVVSTENITL   KCS1
256  GIYAGENRSMMVSNCLFRVGGAAILLSNKSGDRRRSKYKLVHTVRTHTGA   FAE1 KCS
269  NWYFGNDRSKLVSNCLFRMGGAAILLSNKRWDRRRSKYEELVDTVRTHKGA   At2g26640 KCS
251  NWYFGNKKAMLIPNCLFRRVGGSALLSNKPLDRKRSKYKLVHTVRTHKGS   At4g34510 KCS
291  NWYFGNDRSMLLCNCTIFRMGGAAILLSNRRQDRKKSKYSLVNVRRTHKGS   KCS1
306  DDKSFRCVQEQDEDGKIGVCLSKDIITNVAGTTTLTKNIALGLPILPLSEE   FAE1 KCS
319  DDKCFCGCIQEEDSASKIGVTLSKELMAVAGDALKTNIITTLGPLVVLPTSEE   At2g26640 KCS
301  DENAFNCVYQOECCLKTGVSLSKDLMAIAGEALKTNITSLLGPLVVLPISEE   At4g34510 KCS
341  DDKNYNCVYQKEDERGTTIGVSLARELMSVAGDALKTNIITTLGPMVVLPLSEE   KCS1
356  KFLFFATFVAKKLLKDKIKKHYYVPDFKLAVDHFCAHGGRRAVIDELEKNL   FAE1 KCS
369  QLLFFATLVGKRKLFKMKI-KPYIPDFKLAFEHFCAHGGRRAVIDELEKNL   At2g26640 KCS
351  QJLFFATFVAKRLFNDKKKKPYIPDFKLALDHFCIHAGGRRAVIDELEKSL   At4g34510 KCS
391  QLMFLISLVKKRKMFKLKVK-PYIPDFKLAFEHFCAHGGRRAVIDLEVQKNL   KCS1
406  GLSPIDVEASRSTLHRFGNTSSSSSIWYELAYIEAKGRRMKKGNNKA WQIA LG   FAE1 KCS
418  KLTEWHMEPSRMTLYRFGNTSSSSSLWYELAYS EAKGRIKKKGDR IWQIA FG   At2g26640 KCS
401  KLSPKHVEASRMTLHRFGNTSSSSSIWYELAYTEAKGRRMKGNNRV WQIA FG   At4g34510 KCS
440  DLKDWHMEPSRMTLHRFGNTSSSSSLWYEMAYTEAKGRRVKAGDR LWQIA FG   KCS1
456  SGFKCNSAVVALRNVKASP--QHCIDRYPVKIDSDLSKSKTHVQN   FAE1 KCS
468  SGFKCNSSVWARAVRSVNPKKEK-NPW MDEIHEFPVEV--PKV   At2g26640 KCS
451  SGFKCNSAVVALRNVEPS--VNNPWEHCHIRYPVKIDL   At4g34510 KCS
490  SGFKCNSAVWKALRPVS TEEMTGNAWAGSIDQYPVKV--VQ   KCS1
504  GRS
507  STI
487  
528  

Fig. 4. Primary sequence alignment for FAE1 KCS, KCS1, At2g26640 KCS, and At4g34510 KCS. Sequences were aligned by the ClustalW algorithm (DNAStar) and identical residues are indicated by shaded boxes. Conserved residues in the catalytic triad (Cys, His, and Asn) are indicated by square gray symbols above the aligned sequences.


KCSs examined here, three (FAE1 KCS, KCS1, and At4g34510) were found to be active in the yeast in vivo elongation activity assay reported by Trenkamp et al. and had very similar substrate specificity profiles [50].

A yeast expression library screen demonstrated that three *A. thaliana* KCSs are able to partially rescue the lethal phenotype of *S. cerevisiae elo2Δ/elo3Δ*; a double knockout of 2 *Elo* genes which are involved in supplying VLCFA for sphingolipid production in yeast [49,52]. Interestingly, none of the KCSs examined in our study were identified in the *elo2Δ/elo3Δ* rescue expression library screen. As ELO2 and ELO3 are involved in the elongation of C16 to C26 fatty acids, the substrate specificity of the KCSs examined here suggests that the *A. thaliana* KCS genes would be expected to be able to rescue the yeast ELO double mutant. These KCSs may not have been identified in the *elo2Δ/elo3Δ* rescue screen due to low representation in the yeast expression cDNA library. Quantitative digital gene expression data from the Arabidopsis Massively Parallel Signature Sequencing database indicate that At2g26640, KCS1, and At4g34510 have low representation in the tissues used in the construction of the yeast expression library for the *elo2Δ/elo3Δ* rescue [53–56]. The condensation assays used in this report and by Paul et al. give a more accurate description of substrate specificity of the condensing enzymes as consistent substrate amounts are supplied to the isolated KCSs [49].

We were successful in discovering and characterizing three KCSs through our biochemical genomic approach to cloning new enzymes; the function of many of the other putative KCS enzymes should be accessible with batteries of longer chain fatty acyl-CoAs. At4g34510 KCS is of particular interest as it appears to be specific for saturated acyl groups. This KCS will prove useful in further studies of the domains responsible for substrate specificity in condensing enzymes, in particular, those regions with a role in distinguishing between saturated and unsaturated substrates.

### Acknowledgments

We thank Xiaoyun Deng for technical assistance and Robert Minto for helpful discussions. This work was funded by a grant from Cargill, Inc.

### References

[1] L. Kunst, A.L. Samuels, Biosynthesis and secretion of plant cuticular wax, Prog. Lipid Res. (2003) 51–80.

[2] D. Post-Beittenmiler, The cloned Eceriferum genes of Arabidopsis and the corresponding glossy genes in maize, Plant Physiol. Biochem. (1998) 157–166.

[3] K.M. Schmid, J.B. Ohlrogge, Lipid metabolism in plants, in: D.E. Vance, J.E. Vance (Eds.), New Comprehensive Biochemistry, Biochemistry of Lipids, Lipoproteins and Membranes, vol. 31, Elsevier Science B.V., New York, NY, 1996, pp. 363–389.

[4] F. Beaudoin, K. Gable, O. Sayanova, T. Dunn, J.A. Napier, A *Saccharomyces cerevisiae* gene required for heterologous fatty acid elongase activity encodes a microsomal β-keto-reductase, J. Biol. Chem. (2002) 11481–11488.

[5] F. Domergue, S. Chevalier, A. Créach, C. Cassagne, R. Lessire, Purification of the acyl-CoA elongase complex from developing rapeseed and characterization of the 3-ketoacyl-CoA synthase and the 3-hydroxyacyl-CoA dehydratase, Lipids (2000) 487–494.

[6] J.G. Metz, M.R. Pollard, L. Anderson, T.R. Hayes, M.W. Lassner, Purification of a jojoba embryo fatty acyl-coenzyme A reductase and expression of its cDNA in high erucic acid rapeseed, Plant Physiol. (2000) 635–644.

[7] A.A. Millar, L. Kunst, Very-long-chain fatty acid biosynthesis is controlled through the expression and specificity of the condensing enzyme, Plant J. (1997) 121–131.

[8] X. Xu, C.R. Dietrich, R. Lessire, B.J. Nikolau, P.S. Schnable, The endoplasmic reticulum-associated maize GL8 protein is a component of the acyl-coenzyme A elongase involved in the production of cuticular waxes, Plant Phys. (2002) 924–934.

[9] H. Zheng, O. Rowland, L. Kunst, Disruptions of the Arabidopsis enoyl-CoA reductase gene reveal an essential role for very-long-chain fatty acid synthesis in cell expansion during plant morphogenesis, Plant Cell (2005) 1467–1481.

[10] D.W. James, E. Lim, J. Keller, I. Plöoy, E. Ralston, H.K. Dooner, Directed tagging of the Arabidopsis fatty acid elongation1 (*FAE1*) gene with the maize transposon *Activator*, Plant Cell (1995) 301–319.

[11] M. Azachi, A. Sadka, M. Fisher, P. Goldshlag, I. Gokhman, A. Zamir, Salt induction of fatty acid elongase and membrane lipid modifications in the extreme halotolerant alga *Dunaliella salina.*, Plant Phys. (2002) 1320–1329.

[12] P. Barret, R. Delourme, M. Renard, F. Demergue, R. Lessire, M. Delseny, T.J. Roscoe, A rapeseed *FAE1* gene is linked to the El locus associated with variation in the content of erucic acid, Theor. Appl. Genet. (1998) 177–186.

[13] E.B. Cahoon, E.-F. Marillia, K.L. Stecca, S. Hall, D.C. Taylor, A.J. Kinney, Production of fatty acid components of Meadowfoam oil in somatic soybean embryos, Plant Physiol. (2000) 243–251.

[14] S. Das, T.J. Roscoe, M. Delseny, P.S. Srivastava, M. Lakshmikumaran, Cloning and molecular characterization of the *fatty acid elongase 1 (FAE1)* gene from high and low erucic acid lines of *Brassica campestris* and *Brassica oleracea*, Plant Sci. (2002) 245–250.

[15] M. Fourmann, P. Barret, M. Renard, G. Pelletier, R. Delourme, D. Brunel, The two genes homologous to Arabidopsis *FAE1* co-segregate with the two loci governing erucic acid content in *Brassica napus*, Theor. Appl. Genet. (1998) 852–858.

[16] M.W. Lassner, K. Lardizabal, J.G. Metz, A jojoba β-ketoacyl-CoA synthase cDNA complements the canola fatty acid elongation mutation in transgenic plants, Plant Cell (1996) 281–292.

[17] H. Moon, M.A. Smith, L. Kunst, A condensing enzyme from the seeds of *Lesquerella fendleri* that specifically elongates hydroxy fatty acids, Plant Phys. (2001) 1635–1643.

[18] J. Venkateswari, S. Kanrar, P.B. Kirti, V.G. Malathi, V.L. Chopra, Molecular cloning and characterization of *fatty acid elongation1 (BjFAE1)* gene of *Brassica juncea*, J. Plant Biochem. Biotech. (1999) 53–55.

[19] B.J. Blacklock, J.G. Jaworski, Studies into factors contributing to substrate specificity of membrane-bound 3-ketoacyl-CoA synthases, Eur. J. Biochem. (2002) 4789–4798.

[20] J. Puyaubert, B. Garbay, P. Costaglioli, W. Dieryck, T.J. Roscoe, M. Renard, C. Cassagne, R. Lessire, Acyl-CoA elongase expression during seed development in *Brassica napus*, Biochim. Biophys. Acta (2001) 141–152.

[21] M. Rossak, M. Smith, L. Kunst, Expression of the *FAE1* gene and *FAE1* promoter activity in developing seeds of *Arabidopsis thaliana*, Plant Mol. Biol. (2001) 717–725.

[22] A. Fiebig, J.A. Mayfield, N.L. Miley, S. Chau, R.L. Fischer, D. Preuss, Alterations in *CER6*, a gene identical to *CUTI*, differentially affect long-chain lipid content on the surface of pollen and stems, Plant Cell (2000) 2001–2008.

[23] T.S. Hooker, A.A. Millar, L. Kunst, Significance of the expression of the *CER6* condensing enzyme for cuticular wax production in Arabidopsis, Plant Phys. (2002) 1568–1580.

[24] A.A. Millar, S. Clemens, S. Zachgo, E.M. Giblin, D.C. Taylor, L. Kunst, CUT1, an Arabidopsis gene required for cuticular wax biosynthesis and pollen fertility, encodes a very-long-chain fatty acid condensing enzyme, Plant Cell (1999) 825–838.

[25] R.E. Pruitt, J.-P. Vielle-Calzada, S.E. Plöse, U. Grossniklaus, S.J. Lolle, FIDDLEHEAD, a gene required to suppress epidermal cell interactions in *Arabidopsis*, encodes a putative lipid biosynthetic enzyme, Proc. Natl. Acad. Sci. (2000) 1311–1316.

[26] J. Todd, D. Post-Beittenmiller, J.G. Jaworski, *KCSI* encodes a fatty acid elongase 3-ketoacyl-CoA synthase affecting wax biosynthesis in *Arabidopsis thaliana*, Plant J. (1999) 119–130.

[27] A. Yephremov, E. Wisman, P. Huijser, C. Huijser, K. Wellesen, H. Saedler, Characterization of the FIDDLEHEAD gene of Arabidopsis reveals a link between adhesion response and cell differentiation in the epidermis, Plant Cell (1999) 2187–2201.

[28] J.E. Gray, G.H. Holroyd, F.M. van der Lee, A.R. Bahrami, P.C. Sijmons, F.I. Woodward, W. Schuch, A.M. Hetherington, The HIC signalling pathway links CO₂ perception to stomatal development, Nature (2000) 713–716.

[29] M. Ghanavati, J.G. Jaworski, Active-site residues of a plant membrane-bound fatty acid elongase β-ketoacyl-CoA synthase, FAE1 KCS, Biochim. Biophys. Acta (2001) 77–85.

[30] M. Ghanavati, J. Jaworski, Engineering and mechanistic studies of the Arabidopsis FAE1 β-ketoacyl-CoA synthase, FAE1 KCS, Eur. J. Biochem. (2002) 3531–3539.

[31] R.J. Heath, C.O. Rock, The Claisen condensation in biology, Nat. Prod. Rep. (2002) 581–596.

[32] J. Han, W. Luhs, K. Sonntag, U. Zahringer, D.S. Borchardt, F.P. Wolter, E. Heinz, M. Frentzen, Functional characterization of β-ketoacyl-CoA synthase genes from *Brassica napus* L, Plant Mol. Biol. (2001) 229–239.

[33] T.J. Roscoe, R. Lessire, J. Puyaubert, M. Renard, M. Delseny, Mutations in the *fatty acid elongation 1* gene are associated with a loss of β-ketoacyl-CoA synthase activity in low erucic acid rapeseed, FEBS Lett. (2001) 107–111.

[34] V. Katavic, E. Mietkiewska, D.L. Barton, E.M. Giblin, D.W. Reed, D.C. Taylor, Restoring enzyme activity in nonfunctional low erucic acid *Brassica napus* fatty acid elongase 1 by a single amino acid substitution, Eur. J. Biochem. (2002) 5625–5631.

[35] R.K. Downey, G. Robbelen, Brassica species, in: G. Robbelen, R.K. Downey, A. Ashri (Eds.), Oil Crops of the World, McGraw-Hill Publishing Company, New York, 1989, pp. 339–362.

[36] L. Kunst, D.C. Taylor, E.W. Underhill, Fatty acid elongation in developing seeds of *Arabidopsis thaliana*, Plant Physiol. Biochem. (1992) 425–434.

[37] The Arabidopsis Genome Initiative, Analysis of the genome sequence of the flowering plant *Arabidopsis thaliana*, Nature 408 (2000) 796–815.

[38] S.F. Altschul, T.L. Madden, A.A. Schäffer, J. Zhang, Z. Zhang, W. Miller, D.J. Lipman, Gapped BLAST and PSI-BLAST: a new generation of protein database search programs, Nucleic Acids Res. (1997) 3389–3402.

[39] J.D. Thompson, D.G. Higgins, T.J. Gibson, CLUSTALW: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice, Nucleic Acids Res. (1994) 4673–4680.

[40] J. Felsenstein, PHYLIP (Phylogeny Inference Package) version 3.6 (2005) Distributed by author. Department of Genome Sciences, University of Washington, Seattle.

[41] G. Zeng, Sticky-end PCR: new method for subcloning, Biotechniques (1998) 206–208.

[42] J. Sambrook, E.F. Fritsch, T. Maniatis, Molecular Cloning: A Laboratory Manual, Second ed., Cold Spring Harbor Press, Cold Spring Harbor, 1989.

[43] R.D. Gietz, R.A. Woods, High efficiency transformation in yeast, in: J.A. Johnston (Ed.), Molecular Genetics of Yeast: Practical Approaches, Oxford University Press, Oxford, 1994, pp. 121–134.

[44] F.M. Ausubel, R. Brent, R.E. Kingston, D.D. Moore, J.G. Seidman, J.A. Smith, K. Struhl, Short Protocols in Molecular Biology, John Wiley & Sons, New York, 1992.

[45] T.S. Tillman, R.M. Bell, Mutants of *Saccharomyces cerevisiae* defective in sn-glycerol-3-phosphate acyltransferase, J. Biol. Chem. (1986) 9144–9149.

[46] M.M. Bradford, A rapid and sensitive method of quantitation of microgram quantities of protein utilizing the principle of protein-dye binding, Anal. Biochem. (1976) 248–254.

[47] J.L. Garwin, A.L. Klages, J.E. Cronan, Structural, enzymatic and genetic studies of β-ketoacyl-acyl carrier protein synthases I and II of *Escherichia coli*, J. Biol. Chem. (1980) 11949–11956.

[48] G. Roughan, A semi-preparative enzymic synthesis of malonyl-CoA from \[^{14}C\]acetate and \[^{14}CO_2\]: labeling in the 1, 2 or 3 position, Biochem. J. (1994) 355–358.

[49] S. Paul, K. Gable, F. Beaudoin, E. Cahoon, J. Jaworski, J.A. Napier, T.A. Dunn, Members of the Arabidopsis FAE1-like 3-ketoacyl-CoA synthase gene family substitute for the Elo proteins of *Saccharomyces cerevisiae*, J. Biol. Chem. (2006) 9018–9029.

[50] S. Trenkamp, W. Martin, K. Tietjen, Specific and differential inhibition of very-long-chain fatty acid elongases from *Arabidopsis thaliana* by different herbicides, Proc. Natl. Acad. Sci. USA (2004) 11903–11908.

[51] D. Post-Beittenmiller, Biochemistry and molecular biology of wax production in plants, Annu. Rev. Plant Physiol. Plant Mol. Biol. (1996) 405–430.

[52] C.-S. Oh, D.A. Toke, S. Mandala, C.E. Martin, *ELO2* and *ELO3*, homologues of the *Saccharomyces cerevisiae ELO1* gene, function in fatty acid elongation and are required for sphingolipid formation, J. Biol. Chem. (1997) 17376–17384.

[53] B.C. Meyers, D.K. Lee, T.H. Vu, S.S. Tej, S.B. Edberg, M. Matvienko, L.D. Tindell, Arabidopsis MPSS: an online resource for quantitative expression analysis, Plant Physiol. (2004) 801–813.

[54] B.C. Meyers, D.K. Lee, T.H. Vu, S.S. Tej, S.B. Edberg, M. Matvienko, L.D. Tindell, Arabidopsis Massively Parallel Signature Sequencing Project, <www.dbi.udel.edu/mpss>, 02/06/06.

[55] S. Brenner, M. Johnson, J. Bridgham, G. Golda, D.H. Lloyd, D. Johnson, S. Luo, S. McCurdy, M. Foy, M. Ewan, R. Roth, D. George, S. Eletr, G. Albrecht, E. Vermaas, S.R. Williams, K. Moon, T. Burcham, M. Pallas, R.B. DuBridge, J. Kirchner, K. Fearon, J. Mao, K. Corcoran, Gene expression analysis by massively parallel signature sequencing (MPSS) on microbead arrays, Nat. Biotechnol. (2000) 630–634.

[56] S. Brenner, S.R. Williams, E.H. Vermaas, T. Storck, K. Moon, C. McCollum, J.I. Mao, S. Luo, J.J. Kirchner, S. Eletr, R.B. DuBridge, T. Burcham, G. Albrecht, In vitro cloning of complex mixtures of DNA on microbeads: separation of differentially expressed cDNAs, Proc. Natl. Acad. Sci. USA (2000) 1665–1670.
