
JOURNAL OF BACTERIOLOGY, July 2006, p. 4698–4704
0021-9193/06/$08.00+0 doi:10.1128/JB.00492-06
Copyright © 2006, American Society for Microbiology. All Rights Reserved.

# The Ribulose Monophosphate Pathway Substitutes for the Missing Pentose Phosphate Pathway in the Archaeon Thermococcus kodakaraensis

Izumi Orita,$^{1\dagger}$ Takaaki Sato,$^{2\dagger}$ Hiroya Yurimoto,$^{1}$ Nobuo Kato,$^{1‡}$ Haruyuki Atomi,$^{2}$ Tadayuki Imanaka,$^{2}$ and Yasuyoshi Sakai$^{1*}$

**Division of Applied Life Sciences, Graduate School of Agriculture, Kyoto University, Kitashirakawa-Oiwake, Sakyo-ku, Kyoto 606-8502,**$^{1}$ **and Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Katsura, Nishikyo-ku, Kyoto 615-8510,**$^{2}$ Japan

Received 14 February 2006/Accepted 14 April 2006

The ribulose monophosphate (RuMP) pathway, involving 3-hexulose-6-phosphate synthase (HPS) and 6-phospho-3-hexuloisomerase (PHI), is now recognized as a widespread prokaryotic pathway for formaldehyde fixation and detoxification. Interestingly, HPS and PHI homologs are also found in a variety of archaeal strains, and recent biochemical and genome analyses have raised the possibility that the reverse reaction of formaldehyde fixation, i.e., ribulose 5-phosphate (Ru5P) synthesis from fructose 6-phosphate, may function in the biosynthesis of Ru5P in some archaeal strains whose pentose phosphate pathways are imperfect. In this study, we have taken a genetic approach to address this possibility by using the hyperthermophilic archaeon *Thermococcus kodakaraensis* KOD1. This strain possesses a single open reading frame (TK0475) encoding an HPS- and PHI-fused protein. The recombinant HPS-PHI-fused enzyme exhibited the expected HPS and PHI activities in both directions (formaldehyde fixing and Ru5P synthesizing). The TK0475 deletion mutant Δhps-phi-7A did not exhibit any growth in minimal medium, while growth of the mutant strain could be recovered by the addition of nucleosides to the medium. This auxotrophic phenotype together with the catalytic properties of the HPS-PHI-fused enzyme reveal that HPS and PHI are essential for the biosynthesis of Ru5P, the precursor of nucleotides, showing that the RuMP pathway is the only relevant pathway for Ru5P biosynthesis substituting for the classical pentose phosphate pathway missing in this archaeon.

---

The ribulose monophosphate (RuMP) pathway was originally found in methylotrophic bacteria, which can utilize C₁ compounds as a sole source of carbon and energy (8, 15, 24, 27). In this pathway, formaldehyde is fixed with ribulose 5-phosphate (Ru5P) to form D-arabino-3-hexulose-6-phosphate (Hu6P) by 3-hexulose-6-phosphate synthase (HPS) and then isomerized to fructose 6-phosphate (F6P) by 6-phospho-3-hexuloisomerase (PHI). In bacteria characterized to date, the expression of genes encoding these two enzymes, *hps* and *phi*, respectively, is inducible by formaldehyde (9, 25, 26). Therefore, the RuMP pathway has been thought to be responsible exclusively for formaldehyde assimilation and detoxification. However, recent genome sequence analyses of various microorganisms have revealed that the *hps* and *phi* orthologs coding for these two key enzymes are widely distributed among bacterial and archaeal genomes (8, 9, 12, 21, 25–27). In order to understand the physiological roles of HPS and PHI in archaea, we have characterized the HPS and PHI from a hyperthermophilic and anaerobic archaeon, *Pyrococcus horikoshii* (12), in which the *hps* and *phi* genes are encoded in a single open reading frame (PH1938). Our results demonstrated that the product of PH1938 is a bifunctional fused enzyme harboring both HPS and PHI activities. Interestingly, *P. horikoshii* produced HPS-PHI-fused protein constitutively, i.e., with or without the presence of formaldehyde, suggesting that the RuMP pathway in this organism is involved in a more general metabolic activity.

Both biochemical and genome analyses of archaea have raised the possibility that the RuMP pathway is involved in the synthesis of Ru5P from F6P through the reverse reaction of formaldehyde fixation by HPS and PHI (Fig. 1) (6–8, 19, 27). Genome information suggests that the conventional pentose phosphate pathway (PPP) is incomplete or lacking in several archaea (19). For *Thermococcus zilligii*, the cleavage of hexose phosphate was suggested to be ascribed to the reverse reaction of HPS (20, 23). Subsequently, although the methanogen *Methanocaldococcus jannaschii* possesses a complete set of genes coding for the nonoxidative PPP, Grochowski et al. reported that intermediates of the PPP, erythrose 4-phosphate (E4P), xylulose 5-phosphate (Xu5P), and sedoheptulose 7-phosphate (S7P), could not be detected in this organism (7). Instead, they could detect Hu6P, an intermediate of the RuMP pathway. These studies suggest at least an involvement of the RuMP pathway in Ru5P biosynthesis. However, the physiological significance of the RuMP pathway and the extent of its contribution towards Ru5P synthesis in these archaea are not clear, due to the lack of genetic studies.

In this study, we have used a hyperthermophilic archaeon, *Thermococcus kodakaraensis* KOD1 (2, 10), in order to gain insight into the physiological role of the RuMP pathway in

* Corresponding author. Mailing address: Division of Applied Life Sciences, Graduate School of Agriculture, Kyoto University, Kitashirakawa-Oiwake, Sakyo-ku, Kyoto 606-8502, Japan. Phone: 81-75-753-6385. Fax: 81-75-753-6454. E-mail: ysakai@kais.kyoto-u.ac.jp.
† The first two authors contributed equally to the manuscript.
‡ Present address: Kyotogakuen University, Kameoka, Kyoto 621-8555, Japan.

Vol. 188, 2006                                                                 RIBULOSE MONOPHOSPHATE PATHWAY IN T. KODAKARAENSIS 4699

oxidative PPP

non-oxidative PPP

G6P ——→ 6PGL ——→ 6PG ——→ Ru5P
G6PDH        6PGLase     6PGDH

RPE
Xu5P
S7P
TK
E4P
Aromatic
amino acids
TA
F6P
R5P
GAP
RPI
Nucleotides
HPS
Hu6P
HCHO
PHI

RuMP pathway

FIG. 1. Proposed pentose phosphate synthesis through HPS and PHI in *T. kodakaraensis*. Arrows with broken lines display the reactions catalyzed by enzymes whose orthologs are not apparent on the *T. kodakaraensis* genome. Intermediates: G6P, glucose 6-phosphate; 6PGL, 6-phosphoglucono-δ-lactone; 6PG, 6-phosphogluconate; Ru5P, ribulose 5-phosphate; Xu5P, xylulose 5-phosphate; R5P, ribose 5-phosphate; S7P, sedoheptulose 7-phosphate; GAP, glyceraldehyde 3-phosphate; E4P, erythrose 4-phosphate; F6P, fructose 6-phosphate. Enzymes: G6PDH, glucose-6-phosphate dehydrogenase; 6PGLase, 6-phosphoglucono-δ-lactonase; 6PGDH, 6-phosphogluconate dehydrogenase; RPE, ribulose-5-phosphate-3-epimerase; RPI, ribose 5-phosphate isomerase; TK, transketolase; TA, transaldolase.

Archaea. There are several reasons why this strain was chosen. A gene disruption system based on homologous recombination has been developed for this archaeon (16, 17), a technology possible only with this strain and *Sulfolobus solfataricus* (22) among hyperthermophiles. The complete genome sequence has also been determined and annotated (GenBank accession no. AP006878) (5). The *T. kodakaraensis* genome contains only one putative gene for HPS and PHI, which are fused and comprise a single open reading frame (*hps-phi*) encoding both enzymes on a single polypeptide (TK0475), as in the case of *P. horikoshii* (12). Another major factor was that in addition to the fact that *T. kodakaraensis* did not have putative genes for the oxidative PPP, we could not detect genes coding for two key enzymes in the nonoxidative PPP, i.e., ribulose-5-phosphate-3-epimerase and transaldolase (Fig. 1). Therefore, *T. kodakaraensis* seemed to lack the PPP, which thereby allows a straightforward evaluation of the importance of the RuMP pathway in Ru5P synthesis. We herein provide genetic evidence, by constructing and examining an *hps-phi* deletion mutant of *T. kodakaraensis*, that clearly demonstrates that the *hps-phi* gene indeed functions in the Ru5P biosynthesis in this archaeon. Our present results also enable us to propose an outline of the biosynthesis of pentoses and metabolically related compounds in this archaeon, which can also be expected to apply to other members of Archaea that exhibit phylogenetic patterns of the RuMP pathway and PPP genes that are similar to those for *T. kodakaraensis*.

MATERIALS AND METHODS

Strains, vectors, and culture conditions. The strains and plasmids used in this study are listed in Table 1. *T. kodakaraensis* KOD1 and its derivatives were cultivated under strictly anaerobic conditions at 85°C in a rich growth medium (ASW-YT) or a synthetic medium (ASW-AA) (17). ASW-YT was composed of 0.8× artificial seawater (ASW), 0.5% of yeast extract, 0.5% of tryptone, and 0.2% of elemental sulfur. ASW-AA was composed of 0.8× ASW, 20 amino acids, trace vitamins and minerals, and 0.2% of elemental sulfur. The preparation of the plate medium and cultivation of the cells on it were performed as described previously (17). Individual modifications of the medium for investigation of the auxotrophy of mutant strains and for selection of transformants are described in the respective sections of the text below.

*Escherichia coli* strain DH5α, used for general DNA manipulation, was routinely cultivated at 37°C in Luria-Bertani (LB) medium and supplemented with 50 mg/liter ampicillin when needed. The TK0475 gene was expressed in *E. coli* Rosetta(DE3) (Novagen, Madison, WI) grown in LB medium containing ampicillin (50 mg/liter) and chloramphenicol (34 mg/liter). *E. coli* transformants were grown in LB medium at 37°C, to which 0.5 mM IPTG (isopropyl-β-d-thiogalactopyranoside) was added at mid-log phase, followed by an additional 3 h of growth.

Protein preparation from *T. kodakaraensis*. *T. kodakaraensis* cells were suspended in 20 mM Tris-HCl (pH 7.5) containing 10% glycerol and 1 mM dithiothreitol (buffer A) and were disrupted using a French press (Thermo Spectronic, Rochester, NY). Unbroken cells and cell debris were removed by centrifugation at 5,000 × g for 30 min at 4°C, and the resulting supernatants were used as cell extracts. For Western blotting, the cell extracts were further centrifuged at 150,000 × g for 60 min at 20°C. The resulting supernatant and precipitate were used as the soluble and particulate fractions, respectively.

Enzyme assays. Activity of the combined overall HPS and PHI reaction (HPS/PHI) in the forward direction was determined by measuring the rates of F6P synthesis as described previously (1, 12). Conversion of ribose 5-phosphate (R5P) to Ru5P by ribose 5-phosphate isomerase (RPI) from spinach (Sigma-

TABLE 1. Strains and plasmids used in this study

| Strain or plasmid | Relevant characteristic(s) | Source or reference |
|-------------------|--------------------------|---------------------|
| **Strains**       |                          |                     |
| **Escherichia coli** |                          |                     |
| DH5α              | $F^{-} \phi 80 d l a c Z \Delta M 15 \Delta(l a c Z Y A-a r g F) U 169 d e o R r e c A 1 e n d A 1 h s d R 17(r_{K}^{-} m_{K}^{+}) p h o A$ | Takara Bio |
|                   | $s u p E 44 \lambda^{-} t h i-1 g y r A 96 r e l A 1 F^{-} o m p T h s d S_{B}(r_{B}^{-} m_{B}^{-}) g a l d c m l a c Y I(D E 3)$ |             |
| Rosetta(DE3)      | $F^{-} o m p T h s d S_{B}(r_{B}^{-} m_{B}^{-}) g a l d c m l a c Y I(D E 3) p R A R E^{\delta}(C m^{'})$ | Novagen     |
| Rosetta(DE3)(pEThps-phi) | Rosetta(DE3) harboring pEThps-phi | This study    |
| **Thermococcus kodakaraensis** |                          |                     |
| KOD1              | Wild type                | 2, 10               |
| KW128             | $\Delta p y r F \Delta t r p E::p y r F$ | 16               |
| $\Delta h p s-\phi-7 A$ | $\Delta h p s-\phi::t r p E$ | This study        |
| **Plasmids**      |                          |                     |
| pUC118            | $A m p^{r}$ general cloning vector | Takara Bio |
| pET23a(+)         | Expression vector        | Novagen             |
| pEThps-phi        | pET23a derivative; $h p s-\phi$ | This study        |
| pUMT2             | pUC118 derivative; $t r p E$ marker cassette ($P_{p y r F}::t r p E$) | 18               |
| pUDhps-phi        | pUC118 derivative; $\Delta h p s-\phi::t r p E$ | This study        |

Aldrich, St. Louis, MO) was performed at $30^{\circ} \mathrm{C}$ for $5 \mathrm{~min}$, and the following HPS/PHI reaction was conducted at $80^{\circ} \mathrm{C}$. HPS/PHI activity for the reverse reaction was determined by measuring the rates of formaldehyde production. The reaction mixture (1 ml) contained $50 \mu \mathrm{l}$ of $1 \mathrm{M}$ potassium phosphate buffer (pH 7.5), $50 \mu \mathrm{l}$ of $50 \mathrm{mM} \mathrm{MgCl}_{2}, 50 \mu \mathrm{l}$ of $200 \mathrm{mM} \mathrm{F} 6 \mathrm{P}$, and an appropriate amount of enzyme. HPS/PHI reaction was performed at $80^{\circ} \mathrm{C}$ for $5 \mathrm{~min}$ and was stopped by transfer onto ice. Released formaldehyde was measured using Nash reagent (11). One unit of activity for the reverse reaction was defined as the amount of enzyme that produced $1 \mu \mathrm{mol}$ of formaldehyde per minute. The $K_{m}$ and $V_{\max }$ for the forward reaction were obtained from double reciprocal plots of HPS/PHI activity versus formaldehyde concentration by using a fixed concentration of Ru5P (5 mM) (condition I) and versus Ru5P concentration by using a fixed concentration of formaldehyde (10 mM) (condition II). The $K_{m}$ and $V_{\max }$ for the reverse reaction were obtained from double reciprocal plots of HPS/PHI activities versus F6P concentration (condition III).

**Cloning and expression of TK0475.** The entire TK0475 gene was amplified by PCR using chromosomal DNA of $T$. kodakaraensis KOD1 as the template with the primers $5^{\prime}-G G A A T T C C A T A T G A T A C T C C A G G T C G C T C T-3^{\prime}$ and $5^{\prime}-G G A A T T C T C A C T C A A G G G T C G C G T G C T T C C-3^{\prime}$ (additional NdeI and EcoRI sites are underlined). PCR was performed using Pyrobest DNA polymerase (Takara Bio, Otsu, Japan). The PCR product was digested with NdeI and EcoRI and then ligated into the NdeI/EcoRI site of the T7 expression vector pET23a(+) (Novagen). The resulting plasmid harboring TK0475 was designated pEThps-phi and introduced into $E$. coli Rosetta(DE3) for overexpression. The gene product encoded by TK0475 was designated rHps-Phi (recombinant HPS-PHI-fused enzyme).

**Purification of rHps-Phi.** For purification of rHps-Phi, $E$. coli Rosetta(DE3) (pEThps-phi) cells were suspended in buffer A containing $5 \mathrm{mM} \mathrm{MgCl}_{2}$ and were disrupted by sonication for $10 \mathrm{~min}(19 \mathrm{kHz}$, Insonator model 201M; Kubota, Tokyo, Japan). Unbroken cells and cell debris were removed by centrifugation at $5,000 \times g$ for $30 \mathrm{~min}$ at $4^{\circ} \mathrm{C}$, and the resulting supernatant was used as the cell extract. Cell extracts were further centrifuged at $100,000 \times g$ for $60 \mathrm{~min}$ at $20^{\circ} \mathrm{C}$. Most of the HPS/PHI activity was found in the resulting pellet. The pellet was washed with buffer A containing $5 \mathrm{mM} \mathrm{EDTA}$ and then resuspended in buffer A containing $50 \mathrm{mM} \mathrm{EDTA}$ and stirred for $12 \mathrm{~h}$ at $20^{\circ} \mathrm{C}$. The supernatant obtained by centrifugation at $150,000 \times g$ for $60 \mathrm{~min}$ at $20^{\circ} \mathrm{C}$ was incubated at $80^{\circ} \mathrm{C}$ for $30 \mathrm{~min}$ and centrifuged at $15,000 \times g$ for $15 \mathrm{~min}$ at $20^{\circ} \mathrm{C}$ to remove heat-labile proteins from the $E$. coli strain.

**Protein methods.** Protein concentration was measured using a Bio-Rad protein assay kit (Bio-Rad Laboratories, Tokyo, Japan), with bovine serum albumin as the standard. Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) was performed in a $12 \%$ gel. Western blot analysis was performed using rabbit anti-Hps antibody raised against purified rHps from $P$. horikoshii (12) and Western Lightning chemiluminescent reagent Plus (Perkin Elmer, Boston, MA).

**Construction of the $h p s-\phi$ disruption vector.** A vector for targeted disruption of the $h p s-\phi$ gene in $T$. kodakaraensis through double-crossover homologous recombination was constructed as follows. A DNA fragment containing the target gene together with its flanking regions (1,000 bp each) was amplified from $T$. kodakaraensis KOD1 genomic DNA using the primer set PHPSPHI-R/PHP SPHI-F $(5^{\prime}-G A A A A C C C T A C C G T C C T T C T C A A A G A G G G G G C G A A C-3^{\prime} / 5^{\prime}-A A G G T C T T C A T A C C G C A T A T T G A G G C C A A A A A G G C T C-3^{\prime})$ and KOD Plus (Toyobo, Osaka, Japan) as a DNA polymerase. The amplified DNA fragment was subcloned into pUC118 at the HincII site. An inverse PCR was then carried out to amplify the flanking regions and the pUC118 backbone, thereby removing the coding region of the $h p s-\phi$ gene by using primers PDHPSPHI-R/PDHPSPHI-F $(5^{\prime}-G A C A C G A C C T C C G G A G A T T T T T C A G T T G A A A G G T T T-3^{\prime} / 5^{\prime}-T G A G T G A G G T C G C G A T G T T C T C C T T T A C C C A T A T T-3^{\prime})$. The amplified DNA fragment was ligated with the $t r p E$ marker cassette (1,423 bp) that was excised from pUMT2 (18) by PvuII. The marker cassette was oriented so that it was transcribed in the same direction as that of the target gene. The homologous regions for homologous recombination were sequenced to confirm the absence of unintended mutations. The resulting disruption plasmid was designated pUDhps-phi (6,585 bp, $t r p E$ marker) (Fig. 2).

**Transformation of $T$. kodakaraensis.** Transformation of $T$. kodakaraensis with the $t r p E$ marker was performed as described previously (18). The cells transformed with pUDhps-phi were spread on a selective synthetic Trp$^{-}$ ASW-AA plate medium (ASW-AAW$^{-}$) supplemented with 5 nucleosides, adenosine (A), thymidine (dT), guanosine (G), cytidine (C), and uridine (U) (0.1% each) (ASW-AAW$^{-}+5 \mathrm{NS}$). After cultivation for 5 to 8 days at $85^{\circ} \mathrm{C}$, the transformants grown on the plate medium, tryptophan prototrophs, were isolated and cultivated in ASW-YT medium. A further round of selection on ASW-AAW$^{-}+5 \mathrm{NS}$ medium was performed, and one of the isolated transformants was cultivated in ASW-YT medium and designated the $\Delta h p s-\phi-7 A$ strain.

The genotype of the $\Delta h p s-\phi-7 A$ strain was analyzed by PCR using primer sets that anneal outside the homologous regions (CHDHSPHI-R/CHDHSPHI-F $[5^{\prime}-T C A G C T T C T C C C T C A G C C T C T T T C G A C G T C T T T G A C C T C-3^{\prime} / 5^{\prime}-T C G G A A T G G T G T C C G G A T A A A C G T G G A G A C T T G T G T G G G T A-3^{\prime}]$) and inside the target gene (CHHSPHI-R/CHHSPHI-F $[5^{\prime}-C C C C T C A T C A A A G A A A G A A A G G A A T G A G G G C A G T A G A G C T C A-3^{\prime} / 5^{\prime}-T C C T C G A A A A G A G A T T C A T A G G G G C T A T C C A T T T G T A C T-3^{\prime}]$). The genotype was also confirmed by sequencing of the targeted regions and by Southern hybridization. Southern blot analysis was carried out with $5.0 \mu \mathrm{g}$ of genomic DNA digested with HindIII, and the overall procedures were performed as described previously (18). The probe specific to the $t r p E$ gene was prepared as described previously (18).

**Investigation of growth properties of the $h p s-\phi$ disruptant with various substrates.** We investigated what substrates could support the growth of the $h p s-\phi$ disruptant. We examined three combinations of substrates, A+dT+G+C+U (0.1% each), A+U (0.25% each), $2^{\prime}$-deoxyadenosine (dA)+$2^{\prime}$-deoxyuridine (dU) (0.2% each), and nine individual substrates, A (0.25%), dT (0.5%), G (0.25%), C (0.5%), U (0.5%), inosine (I) (0.5%), GMP (0.5%), R5P (0.1%), and Ru5P (0.1%). Furthermore, 0.5% of sodium pyruvate (a gluconeogenic substrate) and 0.5% of maltodextrin (Nippon starch chemical; Osaka, Japan) (a

Vol. 188, 2006 RIBULOSE MONOPHOSPHATE PATHWAY IN T. KODAKARAENSIS 4701

![Diagram](#)

FIG. 2. Schematic diagram of targeted disruption of *hps-phi* in *T. kodakaraensis* KW128 by use of pUDhps-phi. Relevant regions of the chromosome are illustrated for strains KW128 and Δ*hps-phi-7A*. The gray regions display the homologous regions for recombination. The positions of primer sets used for the analyses of targeted disruption of *hps-phi* (CHDHPSPHI-R/CHDHPSPHI-F and CHHPSPHI-R/CHHPSPHI-F) are indicated by arrows. The thick black bar indicates the region spanned by the *trpE* probe used in Southern blot analyses. Abbreviations: Hd, restriction site of HindIII; Hypo, hypothetical gene; P, putative *pyrF* promoter region.

glycolytic substrate) were also examined as control experiments. The pHs of all media were adjusted within 6.5 to 7.0.

## RESULTS

Expression of the TK0475 gene in *Escherichia coli* and kinetic properties of the produced protein. To clarify whether the TK0475 gene actually encodes an HPS-PHI-fused protein, we heterologously expressed this gene in *E. coli* and characterized the biochemical properties of the purified protein (rHps-Phi). Cell extracts from *E. coli* Rosetta(DE3)(pEThps-phi) exhibited activity of the combined overall HPS and PHI reaction (HPS/PHI) in both forward (formaldehyde-fixing) and reverse (Ru5P-synthesizing) directions, and a major band whose molecular mass (44 kDa) was consistent with the molecular mass of the TK0475 gene product was detected by SDS-PAGE (data not shown). The rHps-Phi was purified 15.1-fold from the cell extract as described in Materials and Methods (Table 2) to apparent homogeneity judged by SDS-PAGE (data not shown) and was found to exhibit both forward and reverse HPS/PHI activities. These results indicated that the TK0475 gene indeed encoded an HPS-PHI-fused bifunctional enzyme.

The kinetic parameters of purified rHps-Phi protein were determined (Table 3). The apparent \( V_{\text{max}} \) of the forward reaction was 134 U/mg under condition I or 132 U/mg under condition II, and that of the reverse reaction was 20.9 U/mg; these results were comparable to results for the purified protein of rHps-Phi from *P. horikoshii* (12). The catalytic efficiency (\( k_{\text{cat}} / K_m \)) of the formaldehyde-fixing reaction was 99.1 or 193 mM⁻¹ s⁻¹ for formaldehyde or Ru5P, respectively, while that of the Ru5P-synthesizing reaction was 21.2 mM⁻¹ s⁻¹. Although the catalytic efficiency of the former reaction was higher than that of the latter reaction in vitro, our results clearly indicate that rHps-Phi can catalyze the synthesis of Ru5P from F6P.

HPS/PHI activity in *T. kodakaraensis* KOD1. We examined HPS/PHI activity for the forward and reverse reactions in cell extract from *T. kodakaraensis* KOD1 cultivated under standard conditions without formaldehyde. The specific activities for the HPS/PHI forward reaction and reverse reaction were found to be 0.160 and 0.043 U per mg protein, respectively. This level of activity for the HPS/PHI forward reaction was comparable to

### TABLE 2. Purification of rHps-Phi from *E. coli* Rosetta(DE3)(pEThps-phi)

| Step               | Activity<sup>a</sup> (U) | Protein (mg) | Sp act (U/mg protein) | Recovery (%) | Purification (fold) |
|--------------------|-------------------------|--------------|------------------------|---------------|---------------------|
| Cell-free extract  | 145                     | 296          | 9.80                   | 100           | 1.0                 |
| Membrane fraction  | 77.0                    | 125          | 15.4                   | 66.4          | 1.57                |
| Washed membrane    | 49.6                    | 42.0         | 29.5                   | 42.8          | 3.01                |
| Solubilization     | 34.0                    | 9.50         | 89.5                   | 29.3          | 9.13                |
| Heat treatment at 80°C | 35.4                  | 6.0          | 148                    | 30.5          | 15.1                |

<sup>a</sup> Activities were HPS/PHI forward activities assayed as described in Materials and Methods.

### TABLE 3. Kinetic parameters of purified rHps-Phi at 80°C

| Condition | Apparent \( K_m \) (mM)<sup>a</sup> | \( V_{\text{max}} \) (U/mg)<sup>a</sup> | \( k_{\text{cat}} \) (s⁻¹) | \( k_{\text{cat}} / K_m \) (mM⁻¹ s⁻¹) |
|-----------|------------------------------------|---------------------------------------|--------------------------|--------------------------------------|
| I         | 0.993 ± 0.285                      | 134 ± 26.1                            | 98.4                     | 99.1                                |
| II        | 0.502 ± 0.004                      | 132 ± 5.77                            | 96.9                     | 193                                 |
| III       | 0.725 ± 0.181                      | 20.9 ± 3.97                           | 15.3                     | 21.2                                |

<sup>a</sup> \( K_m \) and \( V_{\text{max}} \) were obtained by double reciprocal plots of HPS/PHI activity versus concentrations of formaldehyde (condition I), Ru5P (condition II), and F6P (condition III), as described in Materials and Methods. The means ± standard deviations of three independent experiments are presented.

4702 ORITA ET AL.

KOD1 KW128 $\Delta h p s$-phi-7A

CFE S P CFE S P CFE S P

FIG. 3. Western blot analyses of cell extracts of $T$. kodakaraensis strains KOD1, KW128, and $\Delta h p s$-phi-7A. Abbreviations: CFE, cell extract; S, soluble fraction; P, particulate fraction.

the constitutive activity levels observed for $P$. horikoshii OT3 (0.250 U per mg protein) and the induced level for Bacillus subtilis (0.310 U per mg protein) (12, 25). Western blot analysis using anti-HPS antibodies revealed a positive signal corresponding to 44 kDa, thereby confirming the production of this protein in $T$. kodakaraensis (Fig. 3). As the HPS-PHI-fused enzyme is produced even in the absence of formaldehyde, the protein is most likely constitutive, as in the case of the enzyme in $P$. horikoshii (12), supporting a general metabolic role of the enzyme.

Construction of $h p s$-phi deletion mutant. A gene disruption system for $T$. kodakaraensis has been developed and utilized for the analysis of gene function (3, 16–18). Using this system, we aimed to investigate the in vivo function of the $h p s$-phi gene, focusing on whether this gene participates in the biosynthesis of Ru5P, the precursor of nucleotides. We constructed the disruption vector of the $h p s$-phi gene, pUD$h p s$-phi (Fig. 2), and transformed the KW128 strain ($\Delta p y r F t r p E::p y r F$) with the vector as described in Materials and Methods. As the $h p s$-phi deletion mutant was expected to require exogenous nucleotides for growth, the selection medium ASW-AAW$^{-}$ was supplemented with five nucleosides, A, dT, G, C, and U. We applied nucleosides instead of nucleotides, since the phosphate group may have a negative effect on membrane permeability. Tryptophan prototrophs were obtained with a transformation efficiency of approximately $2 \times 10^{1} / \mu g$ DNA, while a control experiment without the exogenous DNA gave no tryptophan prototrophs. One of the isolates was designated strain $\Delta h p s$-phi-7A and examined further.

Confirmation of $h p s$-phi disruption in the isolated mutant strain. The genotype of the $\Delta h p s$-phi-7A strain was confirmed by PCR, sequencing, and Southern blot analyses. Amplification of a DNA fragment with a length corresponding to that of a $\Delta h p s$-phi::$t r p E$ locus formed by double-crossover recombination was observed with the $\Delta h p s$-phi-7A strain (3.6 kbp against 3.4 kbp in KOD1 and KW128) (Fig. 4A). No amplification was observed with the $\Delta h p s$-phi-7A strain by PCR using primer sets that anneal within the target gene (Fig. 4B), indicating complete deletion of the target gene. The replacement of $h p s$-phi by the $t r p E$ marker was also confirmed by sequencing analysis of the targeted regions and Southern blot analysis using the probe specific to the $t r p E$ gene (Fig. 2). As shown in Fig. 4C, wild-type KOD1 showed single signals derived from endogenous $t r p E$ in the $t r p$ operon (3.7 kbp), while no positive signal could be detected in the host strain KW128, with a $\Delta t r p E::p y r F$ genotype. In the $\Delta h p s$-phi-7A strain, the signals could be detected, with the expected mobility corresponding to a $t r p E$ insertion within the target locus (4.9 kbp) (Fig. 4C). We also

FIG. 4. PCR and Southern blot analyses of $T$. kodakaraensis $\Delta h p s$-phi-7A ($\Delta p y r F \Delta t r p E::p y r F \Delta h p s$-phi::$t r p E$). (A) Amplification of the locus containing $h p s$-phi and its homologous regions in strains KOD1 (lane 1), KW128 (lane 2), and $\Delta h p s$-phi-7A (lane 3), using CHDHPS PHI-R/CHDHPSPHI-F as primers. M represents the DNA size marker, HindIII-digested $\lambda$ DNA. (B) Amplification of the region within the $h p s$-phi gene in $T$. kodakaraensis KOD1 (lane 1), KW128 (lane 2), and $\Delta h p s$-phi-7A (lane 3), using CHHSPHI-R/CHHSP HI-F as primers. Primers used for these analyses are displayed in Fig. 2. (C) Southern blot analysis of $T$. kodakaraensis $\Delta h p s$-phi-7A. The probe specific to the $t r p E$ gene was used against genomic DNA of KOD1 (lane 1), KW128 (lane 2), and $\Delta h p s$-phi-7A (lane 3) digested with HindIII. The bars on the left side of the panels indicate the lengths of fragments in the DNA size marker, HindIII-digested $\lambda$ DNA. The region spanned by the probe used for this analysis is indicated in Fig. 2.

confirmed the absence of HPS-PHI proteins in the $\Delta h p s$-phi-7A strain by Western blot analysis (Fig. 3).

Nucleoside requirement for growth of the $h p s$-phi deletion mutant. We investigated whether the $\Delta h p s$-phi-7A strain showed auxotrophy for nucleosides, the precursor of nucleotides. The host strain KW128 and the mutant $\Delta h p s$-phi-7A were cultivated in ASW-AA medium with or without supplementation of five nucleosides, A, dT, G, C, and U. As shown in Fig. 5, the $\Delta h p s$-phi-7A strain did not exhibit any growth in ASW-AA medium, while growth of the same strain could be recovered in a medium supplemented with the five nucleosides (ASW-AA+5NS medium). The host strain could grow in both ASW-AA and ASW-AA+5NS media. This indicates that the disruption of the $h p s$-phi gene resulted in a nucleoside-auxotrophic phenotype in $T$. kodakaraensis. Considering the catalytic function of the rHps-Phi mentioned above and the distribution of these genes (19), we can presume that Ru5P, the precursor of nucleotides and eventually deoxynucleotides, is produced from F6P via the $h p s$-phi gene product in $T$. kodakaraensis. The growth defect of the deletion mutant also indicates that the metabolic pathway in which the $h p s$-phi gene functions is the sole pathway leading to the production of Ru5P and nucleotides.

Growth properties of the $h p s$-phi deletion mutant with various substrates. We examined which substrates could complement the growth defect of the $\Delta h p s$-phi-7A strain. The substrates tested were A+U, dA+dU, A, dT, G, C, U, I, and the nucleoside phosphate GMP. Pyruvate (a gluconeogenic substrate) and maltodextrin (a glycolytic substrate) were also ex-

and eventually deoxynucleotides in *T. kodakaraensis* by generating Ru5P from F6P. Kinetic properties of rHps-Phi have shown that the catalytic efficiency for the formaldehyde-fixing reaction was higher than that for the Ru5P-synthesizing reaction. However, as the RuMP pathway is most likely the sole pathway for Ru5P synthesis, the continuous consumption of Ru5P for nucleotide biosynthesis should drive the reaction in the direction of Ru5P synthesis. The activity level of HPS/PHI for the Ru5P-synthesizing reaction in the cell extracts (0.043 U per mg protein) was much higher than the estimated specific rate for R5P synthesis in archaea (0.0003 U per mg protein) (6), and it is therefore feasible that the HPS-PHI-fused protein is responsible for R5P biosynthesis in this organism.

From the genome sequence data, Ru5P can be expected to be isomerized into R5P by RPI (TK1426) and subsequently converted into phosphoribosyl pyrophosphate (PRPP) by PRPP synthase (TK2235). PRPP can then be utilized for nucleotide biosynthesis via conventional pyrimidine and purine nucleotide biosynthetic pathways, of which members are apparently present in *T. kodakaraensis*. These findings suggest that the wide distribution of HPS and PHI orthologs, coinciding with the absence of pentose phosphate enzyme genes, raises the possibility that this pathway is utilized in a majority of *Archaea*. Furthermore, all *Archaea* whose genomes have been sequenced possess both RPI and PRPP synthase orthologs, except for the archaeal parasite *Nanoarchaeum equitans*. Therefore, it can be presumed that the pathways for nucleotide biosynthesis downstream of Ru5P are generally shared in all three domains of life, *Archaea*, *Bacteria*, and *Eucarya*. *T. kodakaraensis* harbors orthologs of transketolase (N-terminal subunit, TK0270; C-terminal subunit, TK0269) (Fig. 1). The most important function of transketolase in this organism can be expected to be supplying E4P, the precursor of chorismate and eventually aromatic amino acids. However, several archaeal species possess an alternate pathway for aromatic amino acid biosynthesis that does not involve E4P as a precursor. Therefore, transketolase may well have another function. These hypotheses have been described in a previous report (19).

We were surprised to find that single nucleosides (A, G, C,

TABLE 4. Growth properties of *T. kodakaraensis Δhps-phi-7A*

| Medium | % Concn of substrate (total concn [%]) | Result for<sup>a</sup>: |
| --- | --- | --- |
|  |  | KW128 (host strain) | Δhps-phi-7A |
| ASW-AA |  | + | – |
| ASW-AA + A+dT+G+C+U | 0.1 each (0.5) | + | + |
| ASW-AA + A+U | 0.25 each (0.5) | + | + |
| ASW-AA + dA+dU | 0.2 each (0.4) | + | – |
| ASW-AA + A | 0.25 | + | + |
| ASW-AA + dT | 0.5 | + | – |
| ASW-AA + G | 0.25 | + | + |
| ASW-AA + C | 0.5 | + | + |
| ASW-AA + U | 0.5 | + | + |
| ASW-AA + I | 0.5 | + | + |
| ASW-AA + GMP | 0.5 | + | + |
| ASW-AA + pyruvate | 0.5 | + | – |
| ASW-AA + maltodextrin | 0.5 | + | – |

<sup>a</sup> +, growth; –, no growth.

U, and I) could complement the *hps-phi* disruption in *T. kodakaraensis*, indicating that all nucleotides can be generated from any one of these nucleosides. Considering the genome sequence data, PRPP production in media including the individual nucleosides A, G, I, C, and U can be explained as follows: adenosine, guanosine, and inosine can be converted to ribose 1-phosphate (R1P) by purine nucleoside phosphorylase. An enzyme from *Pyrococcus furiosus* (PF0016) has been shown to exhibit activity towards all three purine nucleosides (4), and a closely related homolog (81% identical) is present in *T. kodakaraensis* (TK1895). Cytidine can be converted into uridine by cytidine deaminase (TK1414). Therefore, both cytidine and uridine can provide R1P via uridine phosphorylase (TK1479). The R1P can then be converted into R5P by phosphopentomutase (13) and then flow into the general pathway for nucleotide synthesis.

As HPS and PHI have been found to function in Ru5P synthesis, this also results in the generation of toxic formaldehyde. One of the mechanisms known for formaldehyde detoxification is the system involving formaldehyde-activating enzyme (Fae) (21). Fae catalyzes the condensation of formaldehyde and tetrahydromethanopterin to methylene-tetrahydromethanopterin. The methanogenic archaea *Methanosarcina barkeri* and *Methanosarcina mazei* possess a gene encoding an Fae-HPS-fused enzyme, and Fae might be involved in the detoxification of formaldehyde generated during Ru5P biosynthesis (6). *T. kodakaraensis* does not have an Fae homolog but does harbor a formaldehyde ferredoxin oxidoreductase homolog (14). Further analyses will be necessary to clarify the involvement of ferredoxin oxidoreductase or other enzymes in the detoxification of formaldehyde in archaeal strains that do not contain Fae.

### ACKNOWLEDGMENTS

This research was supported in part by a Grant-in-Aid for Scientific Research (S) 13854008 to Y.S. This paper is supported in part by COE for Microbial-Process Development Pioneering Future Production System (COE program of the Ministry of Education, Culture, Sports, Science and Technology, Japan). A Grant-in-Aid for Scientific Research to T.I. (14103011) and a Grant-in-Aid for JSPS Fellows to T.S. (15005649) are also acknowledged.

### REFERENCES

1. Arfman, N., L. Bystrykh, N. I. Govorukhina, and L. Dijkhuizen. 1990. 3-Hexulose-6-phosphate synthase from thermotolerant methylotroph *Bacillus C1*. Methods Enzymol. 188:391–397.
2. Atomi, H., T. Fukui, T. Kanai, M. Morikawa, and T. Imanaka. 2004. Description of *Thermococcus kodakaraensis* sp. nov., a well studied hyperthermophilic archaeon previously reported as *Pyrococcus* sp. KOD1. Archaea 1:263–267.
3. Atomi, H., R. Matsumi, and T. Imanaka. 2004. Reverse gyrase is not a prerequisite for hyperthermophilic life. J. Bacteriol. 186:4829–4833.
4. Cacciafuoco, G., C. Bertoldo, A. Brio, V. Zappia, and M. Porcelli. 2003. Purification and characterization of 5′-methylthioadenosine phosphorylase from the hyperthermophilic archaeon *Pyrococcus furiosus*: substrate specificity and primary structure analysis. Extremophiles 7:159–168.
5. Fukui, T., H. Atomi, T. Kanai, R. Matsumi, S. Fujiwara, and T. Imanaka. 2005. Complete genome sequence of the hyperthermophilic archaeon *Thermococcus kodakaraensis* KOD1 and comparison with *Pyrococcus* genomes. Genome Res. 15:352–363.
6. Goenrich, M., R. K. Thauer, H. Yurimoto, and N. Kato. 2005. Formaldehyde activating enzyme (Fae) and hexulose-6-phosphate synthase (Hps) in *Methanosarcina barkeri*: a possible function in ribose-5-phosphate biosynthesis. Arch. Microbiol. 184:41–48.
7. Grochowski, L. L., H. Xu, and R. H. White. 2005. Ribose-5-phosphate biosynthesis in *Methanocaldococcus jannaschii* occurs in the absence of a pentose-phosphate pathway. J. Bacteriol. 187:7382–7389.
8. Kato, N., H. Yurimoto, and R. K. Thauer. 2006. The physiological role of the ribulose monophosphate pathway in bacteria and archaea. Biosci. Biotechnol. Biochem. 70:10–21.
9. Mitsui, R., Y. Sakai, H. Yasueda, and N. Kato. 2000. A novel operon encoding formaldehyde fixation: the ribulose monophosphate pathway in the gram-positive facultative methylotrophic bacterium *Mycobacterium gasti* MB19. J. Bacteriol. 182:944–948.
10. Morikawa, M., Y. Izawa, N. Rashid, T. Hoaki, and T. Imanaka. 1994. Purification and characterization of a thermostable thiol protease from a newly isolated hyperthermophilic *Pyrococcus* sp. Appl. Environ. Microbiol. 60:4559–4566.
11. Nash, T. 1953. The colorimetric estimation of formaldehyde by means of the Hantzsch reaction. Biochem. J. 55:416–421.
12. Orita, I., H. Yurimoto, R. Hirai, Y. Kawarabayasi, Y. Sakai, and N. Kato. 2005. The archaeon *Pyrococcus horikoshii* possesses a bifunctional enzyme for formaldehyde fixation via the ribulose monophosphate pathway. J. Bacteriol. 187:3636–3642.
13. Rashid, N., H. Imanaka, T. Fukui, H. Atomi, and T. Imanaka. 2004. Presence of a novel phosphopentomutase and a 2-deoxyribose 5-phosphate aldolase reveals a metabolic link between pentoses and central carbon metabolism in the hyperthermophilic archaeon *Thermococcus kodakaraensis*. J. Bacteriol. 186:4185–4191.
14. Roy, R., S. Mukund, G. J. Schut, D. M. Dunn, R. Weiss, and M. W. W. Adams. 1999. Purification and molecular characterization of the tungsten-containing formaldehyde ferredoxin oxidoreductase from the hyperthermophilic archaeon *Pyrococcus furiosus*: the third of a putative five-member tungstoenzyme family. J. Bacteriol. 181:1171–1180.
15. Sakai, Y., R. Mitsui, Y. Katayama, H. Yanase, and N. Kato. 1999. Organization of the genes involved in the ribulose monophosphate pathway in an obligate methylotrophic bacterium, *Methylomonas aminofaciens* 77a. FEMS Microbiol. Lett. 176:125–130.
16. Sato, T., T. Fukui, H. Atomi, and T. Imanaka. 2005. Improved and versatile transformation system allowing multiple genetic manipulations of the hyperthermophilic archaeon *Thermococcus kodakaraensis*. Appl. Environ. Microbiol. 71:3889–3899.
17. Sato, T., T. Fukui, H. Atomi, and T. Imanaka. 2003. Targeted gene disruption by homologous recombination in the hyperthermophilic archaeon *Thermococcus kodakaraensis* KOD1. J. Bacteriol. 185:210–220.
18. Sato, T., H. Imanaka, N. Rashid, T. Fukui, H. Atomi, and T. Imanaka. 2004. Genetic evidence identifying the true gluconeogenic fructose-1,6-bisphosphatase in *Thermococcus kodakaraensis* and other hyperthermophiles. J. Bacteriol. 186:5799–5807.
19. Soderberg, T. 2005. Biosynthesis of ribose-5-phosphate and erythrose-4-phosphate in archaea: a phylogenetic analysis of archaeal genomes. Archaea 1:347–352.
20. Verhees, C. H., S. W. M. Kengen, J. E. Tuininga, G. J. Schut, M. W. W. Adams, W. M. de Vos, and J. van der Oost. 2003. The unique features of glycolytic pathways in Archaea. Biochem. J. 375:231–246.
21. Vorholz, J. A., C. J. Marx, M. E. Lidstrom, and R. K. Thauer. 2000. Novel formaldehyde-activating enzyme in *Methylobacterium extorquens* AM1 required for growth on methanol. J. Bacteriol. 182:6645–6650.
22. Worthington, P., V. Hoang, F. Perez-Pomares, and P. Blum. 2003. Targeted disruption of the α-amylase gene in the hyperthermophilic archaeon *Sulfolobus solfataricus*. J. Bacteriol. 185:482–488.
23. Xavier, K. B., M. S. Da Costa, and H. Santos. 2000. Demonstration of a novel glycolytic pathway in the hyperthermophilic archaeon *Thermococcus zilligii* by <sup>13</sup>C-labeling experiments and nuclear magnetic resonance analysis. J. Bacteriol. 182:4632–4636.
24. Yanase, H., K. Ikeyama, R. Mitsui, R. Sonmin, K. Kita, Y. Sakai, and N. Kato. 1996. Cloning and sequence analysis of the gene encoding 3-hexulose-6-phosphate synthase from the methylotrophic bacterium, *Methylomonas aminofaciens* 77a, and its expression in *Escherichia coli*. FEMS Microbiol. Lett. 135:201–205.
25. Yasueda, H., Y. Kawahara, and S. Sugimoto. 1999. *Bacillus subtilis yckG* and *yckF* encode two key enzymes of the ribulose monophosphate pathway used by methylotrophs, and *yckH* is required for their expression. J. Bacteriol. 181:7154–7160.
26. Yurimoto, H., R. Hirai, H. Yasueda, R. Mitsui, Y. Sakai, and N. Kato. 2002. The ribulose monophosphate pathway operon encoding formaldehyde fixation in a thermotolerant methylotroph, *Bacillus brevis* S1. FEMS Microbiol. Lett. 214:189–193.
27. Yurimoto, H., N. Kato, and Y. Sakai. 2005. Assimilation, dissimilation, and detoxification of formaldehyde, a central metabolic intermediate of methylotrophic metabolism. Chem. Rec. 5:367–375.
