
REVIEW / SYNTHÈSE

# The ubiquitin–proteasome pathway in viral infections¹

Guang Gao and Honglin Luo

---

**Abstract:** The cellular biological function of the ubiquitin–proteasome pathway as a major intracellular protein degradation pathway, and as an important modulator for the regulation of many fundamental cellular processes has been greatly appreciated over the last decade. The critical role of the ubiquitin–proteasome pathway in viral pathogenesis has become increasingly apparent. Many viruses have been reported to evolve different strategies to utilize the ubiquitin–proteasome pathway for their own benefits. Here, we review the general background and function of the ubiquitin–proteasome pathway, summarize our current understanding of how viruses use this pathway to target cellular proteins, and finally, discuss the roles of this pathway in enteroviral infection, and the potential therapeutic application of proteasome inhibition in myocarditis.

**Key words:** ubiquitin, proteasome, protein degradation, virus.

**Résumé :** La fonction biologique cellulaire de la voie ubiquitine–protéasome comme voie de dégradation majeure des protéines intracellulaires et comme modulateur important de la régulation de multiples processus cellulaires fondamentaux a été largement évaluée au cours de la dernière décennie. Le rôle crucial de la voie ubiquitine–protéasome dans la pathogénèse virale est de plus en plus manifeste. Il appert que de nombreux virus développent des stratégies pour utiliser la voie ubiquitine–protéasome à leur avantage. Dans le présent article, nous examinons le contexte général et la fonction de la voie ubiquitine–protéasome, nous résumons notre compréhension de l’utilisation de cette voie par les virus pour cibler les protéines cellulaires et nous discutons des rôles de cette voie dans l’infection à entérovirus ainsi que de l’application thérapeutique potentielle de l’inhibition des protéasomes dans la myocardite.

**Mots clés :** ubiquitine, protéasome, protéine dégradation, virus.

[Traduit par la Rédaction]

---

## Introduction of the ubiquitin–proteasome pathway

The ubiquitin–proteasome pathway, a major intracellular system for extralysosomal protein degradation, plays an important role in a wide variety of cellular functions, including antigen processing, cell cycle regulation, apoptosis, signal transduction, transcriptional regulation, and DNA repair (Ciechanover 1994; Glickman and Ciechanover 2002). The significance of the discovery of the ubiquitin–proteasome pathway has been recognized by the award of the 2004 Nobel Prize in Chemistry to three distinct scientists.

There are two successive steps involved in protein degradation: (1) ubiquitination, covalent attachment of ubiquitin to the target protein substrate, and (2) proteasome-mediated degradation, degradation of the ubiquitinated protein by the 26S proteasome with the release of recycled ubiquitin (Ciechanover 1994; Glickman and Ciechanover 2002).

Ubiquitin, a small protein composed of 76 amino acids, is highly conserved during evolution. The best known function of ubiquitin in proteolysis is to serve as a signal for the target protein to be recognized and degraded in the proteasome. The ubiquitination step involves three sequential enzymatic reactions. First, ubiquitin is activated in an ATP-dependent manner by a ubiquitin-activating enzyme (E1) to form a thiol ester bond between its C-terminal glycine residue and the active cysteine residue of E1. Then, the activated ubiquitin is transferred to a ubiquitin-conjugating enzyme (E2) through an additional thiol ester linkage. Finally, ubiquitin–protein ligase (E3) transfers the ubiquitin to the target protein by the formation of the covalent isopeptide bond between ubiquitin and the ε-amino group of lysine residues in the substrate protein. After several rounds of ubiquitination, multiple ubiquitin molecules are attached to the substrate. Once a polyubiquitin chain composed of at least four ubiquitins is formed, the substrate is quickly recognized and subsequently degraded by the 26S proteasome, and

---

Received 15 August 2005. Published on the NRC Research Press Web site at http://cjpp.nrc.ca on 2 February 2006.

G. Gao and H. Luo.² Department of Pathology and Laboratory Medicine, The James Hogg iCAPTURE Centre for Cardiovascular and Pulmonary Research, University of British Columbia – St. Paul’s Hospital, 1081 Burrard St, Vancouver, BC V6Z 1Y6, Canada.

¹ This paper is one of a selection of papers published in this Special Issue, entitled Young Investigator’s Forum.

² Corresponding author (e-mail: hluo@mrl.ubc.ca).

Can. J. Physiol. Pharmacol. 84: 5–14 (2006)  
doi: 10.1139/y05-144  
© 2006 NRC Canada

ubiquitin is recycled via the actions of deubiquitinating enzymes (DUBs; Glickman and Ciechanover 2002; Pickart 2001; Weissman 2001). The process of ubiquitination and degradation by proteasome is depicted in Fig. 1.

The 26S proteasome, a large multicatalytic protease presents in both cytosol and nucleus of almost all eukaryotic cells, consists of a central catalytic core, the 20S proteasome molecule, and two 19S regulatory complexes at both ends. The 20S catalytic core is a hollow, barrel-shaped cylinder composed of four stacked rings. The α-rings make up the two outer rings each consists of seven identical α-subunits, whereas the β-rings, each consisting of seven β-subunits, make up the two inner rings. The two outer rings, which are catalytically inactive, modulate the entrance of protein substrates to the inner chamber (Fig. 1). The proteolytic activity of the 20S proteasome resides in the two inner rings, which contain the proteolytic sites formed by N-terminal threonine residues facing the central chamber of the 20S complex. Three distinct proteolytic activities of the 20S proteasome have been reported: trypsin-like, chymotrypsin-like, and caspase-like activities. Two 19S regulatory complexes, attaching to the outer α-rings of the 20S proteasome, play an important role in regulating the function of the 20S complex. The 19S proteasome is involved in ubiquitin recognition, deubiquitination, substrate unwinding, and substrate translocation into the 20S catalytic core, where the degradation is processed in an ATP-dependent manner (Ciechanover 1994; Naujokat and Hoffmann 2002). Besides polyubiquitinated substrates, it was reported that proteasome also degrades certain non-ubiquitinated protein substrates (Murakami et al. 1992).

The protein degradation process is highly specific and regulated. The specificity of proteolysis appears to be achieved primarily at the step of ubiquitination. The structure of the ubiquitin conjugation system is hierarchical: there is only one E1 enzyme in humans, which activates the ubiquitin for all conjugation reactions. Dozens of E2 enzymes have been identified, each of which interacts with one or several E3 enzymes. The number of newly discovered E3 ligases is growing rapidly. Each E3 targets one or a few substrates. Therefore, the substrate specificity of the ubiquitin conjugation system is conferred by the E3s (Glickman and Ciechanover 2002; Pickart 2001; Weissman 2001). E3s associate with both E2 and the target protein to facilitate the transfer of ubiquitin from E2 to the substrate or to the last ubiquitin of a polyubiquitin chain through an isopeptide bond between the ε-amino groups of lysine residues on adjacent ubiquitin molecules. E3s can be divided into three groups according to the proteins with which they interact: the single-subunit really interesting new gene (RING) - finger subfamily (both substrate recognition site and RING domain reside in the same protein); the multi-subunit RING-finger subfamily (protein recognition and RING domain are found in different proteins), and the homologous to E6-associated protein carboxyl terminus (HECT) - domain subfamily E3s (Pickart 2001; Weissman 2001). It has been reported that, in most cases, viruses modulate the ubiquitin-proteasome function at the levels of the ubiquitin-protein ligase, either by encoding an E3 functional domain or by redirecting a pre-existing host E3 ligase to a new target (Banks et al. 2003; Shackelford and Pagano 2004).

Recent studies strongly suggest that protein ubiquitination is controlled by both specific processes of ubiquitination and deubiquitination. Deubiquitinating activity was first found to cleave ubiquitin from histone H2A in 1981 (Andersen et al. 1981). After that, more than 90 DUBs have been identified (Wilkinson 2000; Wing 2003). DUBs hydrolyze the isopeptide bonds between two adjacent ubiquitins or between ubiquitin and the substrate protein. Based on their structure and function, DUBs can be classified into at least four distinct families. Ubiquitin-processing proteases (UBP) and ubiquitin carboxyl-terminal hydrolases (UCH) are two well-characterized classes (Chung and Baek 1999; Wilkinson 2000; Wing 2003). Another two classes, the Jab1/MPN domain-associated metalloisopeptidase (JAMM) group of hydrolases (Verma et al. 2002; Yao and Cohen 2002) and a family of cysteine proteases that contains an ovarian tumor domain (OTU) (Balakirev et al. 2003; Evans et al. 2003), are recently described. The function of these enzymes includes processing ubiquitin gene products, negatively regulating the process of ubiquitination, and recycling free ubiquitin after protein recognition by the proteasome (Wing 2003). Recent studies also suggest a key role of the DUBs in the regulation of numerous pivotal cellular functions, such as cell growth, cell differentiation, endocytosis, oncogenesis, DNA replication, and gene silencing (Wilkinson 2000; Wing 2003).

In addition to the recycling of damaged, misfolded, or unneeded proteins, accumulated evidence suggests that the ubiquitin-proteasome pathway is responsible for the modulation of many critical regulatory proteins that control a wide variety of cellular functions, including cell cycle regulation, apoptosis, antigen processing, signal transduction, transcriptional regulation, DNA repair, and receptor regulation (Glickman and Ciechanover 2002). For example, several cell cycle proteins, including cyclins, cyclin-dependent kinase inhibitors (p21, p27), and tumor suppressors (p53) are all substrates of the ubiquitin-proteasome pathway (DeSalle and Pagano 2001). The NFκB signaling pathway is also a critical target for this pathway (Palombella et al. 1994). The NFκB p50/p65 heterodimer is retained inactive in the cytoplasm through binding to a specific inhibitor protein, IκB. Upon activation, IκB is rapidly and sequentially phosphorylated, ubiquitinated, and degraded by the proteasome. NFκB is released and translocated to the nucleus to participate in transcriptional regulation of multiple genes involved in inflammation. Increasing data have suggested that abnormalities of the ubiquitin-proteasome system contribute to the pathogenesis of many human diseases, including cancer, inflammation, cardiovascular, and neurodegenerative diseases (Golab et al. 2004; Herrmann et al. 2004). Proteasome inhibition has now been recognized as a promising therapeutic option in the treatment of several diseases (Adams 2003; Elliott et al. 2003; Yeh 2002).

Apart from the best known role of polyubiquitination in protein degradation, ubiquitination has been reported to be involved in regulating protein functions. Monoubiquitination of some proteins, such as histones, calmodulin, actin, and some transmembrane proteins, serves as a signal for endocytosis and histone-mediated transcriptional regulation, without targeting for degradation (Hicke 2001; Johnson 2002). Finley and co-workers (1989) showed that covalent attach-

Gao and Luo

Fig. 1. Protein degradation by the ubiquitin–proteasome pathway. (1) Covalent attachment of ubiquitin (Ub) to the target protein substrate. Three enzymatic reactions are involved in this step. Ubiquitin-activating enzyme (E1) activates and binds to the C-terminal glycine residue of ubiquitin through a thiol ester linkage in an ATP-dependent manner. This activated ubiquitin is then transferred to a ubiquitin-conjugating enzyme (E2), again through a thiol ester linkage. Ubiquitin-protein ligase (E3) will subsequently transfer the ubiquitin to the target protein by the formation of the covalent isopeptide bond between ubiquitin and the ε-amino group of lysine residues in the substrate protein. After several cycles of ubiquitination, multiple ubiquitin molecules are conjugated to the substrate. (2) 26S proteasome degradation of the tagged protein with release of recycled ubiquitin by deubiquitinating enzymes (DUBs). The 26S proteasome complex consists of a 20S catalytic core and two 19S regulatory complexes. Polyubiquitinated protein is first recognized, deubiquitinated, and unfolded at 19S proteasome, and then translocated into the 20S catalytic core for degradation in an ATP-dependent manner.

![Diagram](attachment:diagram.png)

ment of ubiquitin to ribosomal proteins is required for efficient ribosome biogenesis. In addition, ubiquitin has been suggested to be a key regulator of eukaryotic mRNA synthesis by regulating RNA polymerase II and related transcription factors (Conaway et al. 2002).

## The ubiquitin–proteasome pathway and viral infection

Host responses to viral infection may reflect a host defense mechanism. However, improper host protein regulation, for instance, aberrant host protein degradation, may further promote viral replication and cause tissue damage. Since the first discovery that human papillomavirus E6 protein targets the cellular tumor suppressor protein p53 for the ubiquitin–proteasome-mediated degradation (Scheffner et al. 1993), studies from many research groups, including those from our laboratory, have shown that various viruses have evolved sophisticated mechanisms to utilize or manipulate the host ubiquitin–proteasome pathway for their own needs. For example, the ubiquitin–proteasome system has been suggested to be required for avoidance of host immune surveillance, for viral maturation and viral progeny release, for efficient viral replication, and for reactivation of virus from latency. Table 1 summarizes the interaction between viruses and the ubiquitin–proteasome system.

### Viral immune evasion

Viruses can persevere inside cells to cause latent or chronic infection of the host, even though the host has developed a sophisticated immune system for eliminating the virus-infected cells. Downregulation of major histocompatibility complex (MHC) class I molecules to reduce antigen presentation becomes a common mechanism used by the virus to avoid immune surveillance (Furman and Ploegh 2002). Viruses have developed different strategies to escape the host immune responses by inhibiting antigen presentation from MHC molecules. US2 and US11, encoded by human cytomegalovirus, induce dislocation of MHC class I from the endoplasmic reticulum to cytoplasm where MHC molecules are polyubiquitinated and rapidly degraded by the proteasome (Kikkert et al. 2001; Shamu et al. 2001). The Epstein–Barr virus encoded nuclear antigen 1 (EBNA1) contains Gly–Ala repeats that interfere with antigen processing and MHC class I - restricted responses by preventing viral protein degradation by the proteasome (Levitskaya et al. 1995; Levitskaya et al. 1997).

Viral proteins containing ubiquitin ligase RING-finger do-

Table 1. Examples of viruses or viral proteins interacting with the ubiquitin-proteasome pathway.

| Virus       | Viral proteins                     | Target proteins                | Actions of the UPP               | Functional effects              | References                          |
|-------------|------------------------------------|-------------------------------|----------------------------------|----------------------------------|--------------------------------------|
| HCM         | US2, US11                         | MHC-1                         | Degradation                      |                                  | Shamu et al. 2001                   |
| EBV         | EBNA 1                            | EBNA                          | ↓ Degradation                    |                                  | Levitskaya et al. 1997              |
| HHV-8       | K3, K5 (PHD-E3)                   | MHC-1B7, ICAM-1               | Degradation                      |                                  | Coscoy et al. 2001                  |
| MHV-68      | MK3 (PHD-E3)                      | MHC-1                         | Degradation                      | Viral immune evasion             | Boname and Stevenson 2001           |
| Myxomavirus | M153R                             | CD4                           | Degradation                      |                                  | Mansouri et al. 2003                |
| HIV-1       | Vpu                               | CD4                           | Degradation                      |                                  | Schubert et al. 1998                |
| Mumps virus | V protein                         | STAT1, STAT2                  | Degradation                      |                                  | Gotoh et al. 2002                   |
| SV          | V protein                         | STAT1                         | Degradation                      |                                  | Gotoh et al. 2002                   |
| HIV-1, HIV-2| Gag                               | —                             | Monoubiquitination               |                                  | Schubert et al. 2000                |
| RSV         | Gag                               | —                             | Monoubiquitination               | Viral budding                    | Strack et al. 2000                  |
| HIV-1       | Tat                               | —                             | Monoubiquitination               |                                  | Bres et al. 2003                    |
| HTLV-1      | Tax                               | —                             | Monoubiquitination               | Viral transcriptional regulation   | Peloponese et al. 2004              |
| CVB 3       | unknown                           | Cyclin D1, p53, β-catenin     | Degradation                      |                                  | Luo et al. 2003b, Yuan et al. 2005  |
| HPV         | E6                                | p53                           | Degradation                      |                                  | Barry et al. 1998                   |
| HTLV-1      | E1B55kE4orf6                      | p53                           | Degradation                      | p53 inactivation, apoptosis suppression | Yew et al. 1992                     |
| HSV         | ICP0                              | p53                           | Deubiquitination                 |                                  | Hagglund and Roizman 2004           |
| Adenovirus  | L323K (DUBs)                      | Unknown                       | Deubiquitination                 |                                  | Balakirev et al. 2002               |
| Coronavirus | PLpro (DUBs)                      | Unknown                       | Deubiquitination                 | Others                           | Sulea et al. 2005                   |

Note: HPV, human papillomavirus; EBV, Epstein-Barr virus; HCMV, human cytomegalovirus; HTLV-1, human T-cell leukemia virus 1; SV, simian virus; HSV, herpes simplex virus; HIV, human immunodeficiency virus type; CVB3, coxsackievirus B3; RSV, Rous Sarcoma virus; UPP, the ubiquitin-proteasome pathway; DUBs, deubiquitinating enzymes; PHD-E3, PHD domain containing ubiquitin ligase; MHC, major histocompatibility complex; ICAM-1, intercellular adhesion molecule 1; STAT, signal transducer and activator of transcription; PLpro, papain-like protease; ICP0, herpes simplex virus infected cell protein 0; “—”, not applicable.

ubiquitinating enzymes (Garnier et al. 1996; Hicke 2001). The WW-domain HECT E3 ligases such as Nedd4 or Nedd4-like proteins have been recently reported to interact with L domain and these functional E3 ligases are required for viral budding (Kikonyogo et al. 2001; Yasuda et al. 2002).

In addition to Nedd4 protein, tumor susceptibility gene 101 (Tsg101) is another binding partner of the Gag protein. Tsg101 is a protein normally involved in vacuolar protein sorting and multivesicular body biogenesis by binding to monoubiquitinated proteins (Katzmann et al. 2001). Recent studies on enveloped viruses have suggested an important role of Tsg101 in viral budding and release. Depletion of Tsg101 by small interfering RNAs significantly reduces viral budding from infected cells (Garrus et al. 2001). Overexpression of the N-terminal UEV domain of Tsg101 inhibits viral progeny release (Demirov et al. 2002).

### Transcriptional regulation of virus

The association between the protein degradation and viral transcriptional regulation has been implicated in recent studies. For example, Zhu et al. (2004) showed that the ubiquitin-mediated proteolysis is required for the transcriptional activation of synthetic herpes simplex viral VP16 transcription activators. Blockage of VP16 degradation severely impairs their transcriptional activities. Furthermore, overexpression of a subunit of the proteasome reduces both the degradation and transcription activation of VP16. However, increasing evidence suggests that ubiquitination process may directly regulate transcriptional activities, independent of proteasome-mediated degradation (Conaway et al. 2002; Muratani and Tansey 2003). The HIV-1 encodes a potent transactivator, Tat, which activates the HIV-1 long terminal repeat by adapting co-activator complexes to the promoter. A recent study demonstrated that ubiquitination of Tat protein positively regulates its transcriptional properties without targeting it for proteasomal degradation (Bres et al. 2003). Another interesting observation was made in the research of human T-cell leukemia virus type 1. This human retrovirus encodes a 40 kDa Tax transcriptional activator that modulates expression of the viral long terminal repeat and transcription of many cellular genes. A recent report revealed that Tax is predominantly monoubiquitinated and its transcriptional function is downregulated through a proteasome-independent mechanism in mammalian cells (Peloponese et al. 2004).

### Apoptosis suppression

Apoptosis or cell death at late viral infection may facilitate virus progeny release. However, premature cell death will decrease the ability of host cells to foster virus replication. Thus, various viruses have evolved different strategies to suppress or delay apoptosis during early viral infection to provide sufficient time for the production of high yields of progeny viruses. The tumor suppressor protein p53 is an important transcription factor that plays a key role in the growth control by modulating processes leading to apoptosis and to DNA replication (Sharpless and DePinho 2002). Additionally, p53 has been reported to interfere with the replication of several viruses, such as HIV-1, simian virus 40,

hepatitis B virus, and herpesvirus (Devireddy and Jones 1999; Duan et al. 1994; Tiemann and Deppert 1994). Thus, p53 appears to be a common barrier to the replication of many different viruses.

The human papillomavirus E6 protein interacts with the cellular ubiquitin ligase E6-associated protein to form a complex, targeting p53 for polyubiquitination and degradation by the proteasome (Barry and McFadden 1998). In adenovirus-infected cells, the level of p53 is also markedly reduced. Two adenovirus gene products, E1B 55K and E4orf6, have been shown to regulate the function of p53 by directing it for degradation. In the absence of either of these proteins, cellular levels of p53 are dramatically increased, suggesting that the formation of a E1B and E4orf6 complex is required for p53 ubiquitination and proteasomal degradation (Yew and Berk 1992). Herpes simplex virus infected cell protein 0 (ICP0) has also been shown to inactivate p53 by a mechanism described below.

Recently, we have shown that expression of p53 was markedly reduced following coxsackievirus infection, and this reduction is abrogated when specific inhibitors of the ubiquitin-proteasome pathway are used (Luo et al. 2003b), suggesting that coxsackievirus may have developed certain mechanisms, like the ubiquitin-proteasome pathway, to inhibit apoptosis or viral replication by inactivating the p53 pathway (see Fig. 2).

### Deubiquitination in viral infection

Although the detailed mechanisms remain to be elucidated, the importance of the deubiquitinating enzymes in the regulation of protein degradation and several fundamental cellular functions has now been appreciated. Recent studies suggest that the deubiquitination process, like its counterpart ubiquitination, can be used or modified by viruses. The herpesvirus-associated ubiquitin-specific protease (HAUSP, also known as ubiquitin-specific protease 7 (USP7)) can suppress cell growth by removing ubiquitin from polyubiquitinated p53 to prevent it from degradation (Li et al. 2002). Herpes simplex virus infected cell protein 0 binds HAUSP to inactivate p53 (Hagglund and Roizman 2004). Epstein-Barr virus nuclear antigen 1 is also reported to interact with HAUSP to regulate its transcriptional activities (Holowaty et al. 2003). Using a chemistry-based functional proteomics approach, it was shown that Epstein-Barr virus infection of B cells leads to the activation of a group of DUBs, which includes UCH-L1, HAUSP/USP7, UCH-L5, USP15, and USP9X (Ovaa et al. 2004).

Virus proteins can encode not only their own ubiquitinating enzymes but also deubiquitinating enzymes. Balakirev et al. (2002) have found that adenovirus infection is accompanied by an increased deubiquitinating activity in infected cells. They further provide evidence that adenovirus protease L3 23K, which is responsible for the cleavage of viral precursor polyproteins, may function as a deubiquitinating enzyme. By function motif search, a most recent publication also revealed that papain-like protease (PLpro), a viral cysteine protease of severe acute respiratory syndrome coronavirus, has a very similar structure to HAUSP, implicating that PLpro might have deubiquitinating activities in addition

Fig. 2. Proposed model on the role of the ubiquitin–proteasome system in coxsackievirus infection. CVB3 infection increases protein ubiquitination, which subsequently promotes host protein degradation by the proteasome. Degradation of intracellular proteins that are against viral infection provides a favorable environment for virus to achieve successful replication. Inhibition of the ubiquitin–proteasome pathway by ubiquitin siRNA, antioxidant PDTC, or proteasome inhibitors reduces CVB3 RNA expression and protein synthesis. CVB3 infection activates ERK signaling pathway and activation of ERK is required for effective CVB3 replication. Proteasome inhibition leads to a loss of ERK phosphorylation, which is correlated with an induction of the MKP-1. Blockade of MKP induction attenuates the loss of ERK phosphorylation, suggesting that inhibition of the ERK signaling pathway contributes, as least in part, to proteasome inhibitor reduction of coxsackievirus replication. Abbreviations: CVB3, coxsackievirus B3; DAF, decay accelerating factor; CAR, coxsackievirus and adenovirus receptor; MKP-1, mitogen-activated protein kinase phosphatase-1; ERK1/2, extracellular signal-regulated kinase 1/2; PDTC, pyrrolidine dithiocarbamate.

![Diagram](attachment://image.png)

to their function in processing viral polyprotein (Sulea et al. 2005).

## The ubiquitin–proteasome pathway in enteroviral infection

Coxsackievirus group B3 (CVB3), an enterovirus of the family Picornaviridae, is the common human pathogen that has been associated with the pathogenesis of myocarditis and idiopathic dilated cardiomyopathy. We have recently demonstrated that the ubiquitin–proteasome pathway plays an important role in CVB3 replication (Luo et al. 2003a; Si et al. 2005). Cell culture experiments using HeLa cells and murine cardiomyocytes showed that inhibition of the ubiquitin–proteasome pathway markedly decreases CVB3 viral RNA and protein levels, and inhibited CVB3 progeny release without direct inhibition of viral protease proteolytic activities. As well, we have shown that there is a reduction of several host proteins during CVB3 infection including cyclin D1, p53, and β-catenin. This reduction is abrogated when specific proteasome inhibitors are used (Luo et al. 2003b; Yuan et al. 2005). Interestingly, pyrrolidine dithiocarbamate (PDTC), an antioxidant, potently reduces CVB3 replication, likely through the inhibition of the ubiquitin–proteasome pathway (Si et al. 2005).

To explore the potential mechanisms by which the ubiquitin–proteasome pathway regulates viral replication, we studied the impact of proteasome inhibitors on mitogen-activated protein kinase phosphatase (MKP-1) protein expression and extracellular signal-regulated kinase (ERK) phosphorylation. We have previously reported that the ERK signaling pathway is activated during CVB3 replication and activation of ERK is required for CVB3 replication and contributes to virus-mediated pathogenesis (Luo et al. 2002). Recent studies have suggested that MKP-1, which dephosphorylates and inactivates ERK signaling, can be regulated post-translationally via the ubiquitin–proteasome pathway. We found that proteasome inhibition leads to a loss of ERK phosphorylation in a dose-dependent manner, which is correlated with an induction of the MKP-1. Blockade of MKP induction, using either MKP-1 anti-sense or MKP-1 short-interfering RNA, attenuated the loss of ERK phosphorylation, suggesting that inhibition of the ERK signaling pathway contributes, as least in part, to proteasome inhibitor reduction of coxsackievirus replication (unpublished data).

Following CVB3 infection, there are no changes in proteasome activity in infected cells. However, we found that CVB3 infection leads to the accumulation of protein–ubiquitin conjugates, suggesting an important role of the increased ubiquitination process in the ubiquitin–proteasome-mediated viral replication or protein degradation. Indeed, gene silencing of ubiquitin using small interfering RNA significantly prevents viral protein synthesis (unpublished data). The underlying mechanism by which CVB3 modulates the ubiquitin conjugation process is currently under investigation.

tion. The role of the ubiquitin–proteasome pathway in CVB3 infestation is summarized in Fig. 2.

Inhibition of proteasome activity provides a potential therapeutic option in the treatment of several diseases through the prevention of cell proliferation and inflammatory responses and the promotion of apoptosis (Schwartz and Ciechanover 1999; Vu and Sakamoto 2000). Of the most promise is the application of proteasome inhibitors in cancer therapy. Indeed, proteasome inhibitor bortezomib has recently been approved by the FDA for the treatment of multiple myeloma (Twombly 2003). In addition to cancer therapy, inhibition of proteasome activity may also represent a novel approach to the treatment of cardiovascular diseases. In animal myocardial ischemia–reperfusion injury model, treatment with proteasome inhibitors reduces inflammatory infiltration, decreases myocardial infarct size, and maintains heart function (Bao et al. 2001; Campbell et al. 1999; Pye et al. 2003). Local application of MG132, a proteasome inhibitor, in a rat balloon injury model significantly reduces neointimal hyperplasia (Meiners et al. 2002). It was also found that proteasome inhibition can repress allograft rejection in mice without apparent side effects at effective doses (Luo et al. 2001). To explore the potential value of proteasome inhibition in the treatment of viral myocarditis, we used a well-established murine myocarditis model. We demonstrated that viral replication and inflammatory responses are attenuated after treatment with a proteasome inhibitor (unpublished data), suggesting that the ubiquitin–proteasome pathway may be an attractive and novel therapeutic target for enteroviral infection and viral myocarditis.

### Summary

Ubiquitin modification and (or) protein degradation by the ubiquitin–proteasome pathway is a mechanism for controlling the function and availability of regulatory proteins in the cell; it also provides a platform for many different viruses to achieve successful viral infection. Virus can utilize this system for degrading those intracellular proteins that are against viral infection or modifying their own function either directly, by encoding their own ubiquitinating/deubiquitinating enzymes, or indirectly, by use of endogenous molecules of the ubiquitin system. A better understanding of the interaction between virus infection and the host ubiquitin–proteasome pathway will provide new insights into viral replication, viral pathogenicity, and may lead to a new avenue for therapeutic intervention for virus-induced diseases.

### Acknowledgements

The research in the author’s laboratory is supported by grants from the Canadian Institutes of Health Research (CIHR), the Hospital for Sick Kids Foundation, and the Canada Foundation for Innovation. GG is a recipient of Doctoral Traineeships from the Michael Smith Foundation for Health Research (MSFHR) and from the Heart and Stroke Foundation of Canada. HL is a New Investigator of the CIH/St. Paul’s Hospital Foundation Award and a Scholar of the MS.

### References

Adams, J. 2003. Potential for proteasome inhibition in the treat-
ment of cancer. Drug Discov. Today, 8: 307–315. doi: 10.1016/S1359-6446(03)02647-3. PMID: 12654543.
Andersen, M.W., Ballal, N.R., Goldknopf, I.L., and Busch, H. 1981. Protein A24 lyase activity in nucleoli of thioacetamide-treated rat liver releases histone 2A and ubiquitin from conjugated protein A24. Biochemistry, 20: 1100–1104. doi: 10.1021/bi00508a009. PMID: 6261785.
Balakirev, M.Y., Jaquinod, M., Haas, A.L., and Chroboczek, J. 2002. Deubiquitinating function of adenovirus proteinase. J. Virol. 76: 6323–6331. doi: 10.1128/JVI.76.12.6323-6331.2002. PMID: 12021365.
Balakirev, M.Y., Tcherniuk, S.O., Jaquinod, M., and Chroboczek, J. 2003. Otubains: a new family of cysteine proteases in the ubiquitin pathway. EMBO Rep. 4: 517–522. doi: 10.1038/sj.embo-rep.embor824. PMID: 12704427.
Banks, L., Pim, D., and Thomas, M. 2003. Viruses and the 26S proteasome: hacking into destruction. Trends Biochem. Sci. 28: 452–459. doi: 10.1016/S0968-0004(03)00141-5. PMID: 12932734.
Bao, J., Sato, K., Li, M., Gao, Y., Abid, R., Aird, W., Simons, M., and Post, M.J. 2001. PR-39 and PR-11 peptides inhibit ischemia-reperfusion injury by blocking proteasome-mediated I kappa B alpha degradation. Am. J. Physiol. Heart Circ. Physiol. 281: H2612–H2618. PMID: 11709430.
Barry, M., and McFadden, G. 1998. Apoptosis regulators from DNA viruses. Curr. Opin. Immunol. 10: 422–430. doi: 10.1016/S0952-7915(98)80116-7. PMID: 9722918.
Boname, J.M., and Stevenson, P.G. 2001. MHC class I ubiquitination by a viral PHD/LAP finger protein. Immunity, 15: 627–636. doi: 10.1016/S1074-7613(01)00213-8. PMID: 11672544.
Bres, V., Kiernan, R.E., Linares, L.K., Chable-Bessia, C., Plechakova, O., Treand, C., et al. 2003. A non-proteolytic role for ubiquitin in Tat-mediated transactivation of the HIV-1 promoter. Nat. Cell Biol. 5: 754–761. PMID: 12883554.
Campbell, B., Adams, J., Shin, Y.K., and Lefer, A.M. 1999. Cardioprotective effects of a novel proteasome inhibitor following ischemia and reperfusion in the isolated perfused rat heart. J. Mol. Cell. Cardiol. 31: 467–476. doi: 10.1006/jmcc.1998.0880. PMID: 10093058.
Chung, C.H., and Baek, S.H. 1999. Deubiquitinating enzymes: their diversity and emerging roles. Biochem. Biophys. Res. Commun. 266: 633–640. doi: 10.1006/bbrc.1999.1880. PMID: 10603300.
Ciechanover, A. 1994. The ubiquitin-proteasome proteolytic pathway. Cell, 79: 13–21. doi: 10.1016/0092-8674(94)90396-4. PMID: 7923371.
Conaway, R.C., Brower, C.S., and Conaway, J.W. 2002. Emerging roles of ubiquitin in transcription regulation. Science, 296: 1254–1258. doi: 10.1126/science.1067466. PMID: 12016299.
Coscoy, L., and Ganem, D. 2000. Kaposi’s sarcoma-associated herpesvirus encodes two proteins that block cell surface display of MHC class I chains by enhancing their endocytosis. Proc. Natl. Acad. Sci. USA, 97: 8051–8056. doi: 10.1073/pnas.140129797. PMID: 10859362.
Coscoy, L., Sanchez, D.J., and Ganem, D. 2001. A novel class of herpesvirus-encoded membrane-bound E3 ubiquitin ligases regulates endocytosis of proteins involved in immune recognition. J. Cell Biol. 155: 1265–1273. doi: 10.1083/jcb.200111010. PMID: 11756476.
Demirov, D.G., Ono, A., Orenstein, J.M., and Freed, E.O. 2002. Overexpression of the N-terminal domain of TSG101 inhibits HIV-1 budding by blocking late domain function. Proc. Natl. Acad. Sci. USA, 99: 955–960. doi: 10.1073/pnas.032511899. PMID: 11805336.
DeSalle, L.M., and Pagano, M. 2001. Regulation of the G1 to S
© 2006 NRC Canada

transition by the ubiquitin pathway. FEBS Lett. 490: 179–189. doi: 10.1016/S0014-5793(01)02121-4. PMID: 11223033.

Devireddy, L.R., and Jones, C.J. 1999. Activation of caspases and p53 by bovine herpesvirus 1 infection results in programmed cell death and efficient virus release. J. Virol. 73: 3778–3788. PMID: 10196272.

Duan, L., Ozaki, I., Oakes, J.W., Taylor, J.P., Khalili, K., and Po- merantz, R.J. 1994. The tumor suppressor protein p53 strongly alters human immunodeficiency virus type 1 replication. J. Virol. 68: 4302–4313. PMID: 8207805.

Elliott, P.J., Zollner, T.M., and Boehncke, W.H. 2003. Proteasome inhibition: a new anti-inflammatory strategy. J. Mol. Med. 81: 235–245. PMID: 12700891.

Evans, P.C., Smith, T.S., Lai, M.J., Williams, M.G., Burke, D.F., Heyninck, K., et al. 2003. A novel type of deubiquitinating enzyme. J. Biol. Chem. 278: 23180–23186. doi: 10.1074/jbc. M301863200. PMID: 12682062.

Finley, D., Bartel, B., and Varshavsky, A. 1989. The tails of ubiquitin precursors are ribosomal proteins whose fusion to ubiquitin facilitates ribosome biogenesis. Nature, 338: 394–401. doi: 10.1038/338394a0. PMID: 2538753.

Furman, M.H., and Ploegh, H.L. 2002. Lessons from viral manipulation of protein disposal pathways. J. Clin. Invest. 110: 875– 879. doi: 10.1172/JCI200216831. PMID: 12370262.

Garnier, L., Wills, J.W., Verderame, M.F., and Sudol, M. 1996. WW domains and retrovirus budding. Nature, 381: 744–745. PMID: 8657277.

Garrus, J.E., von Schwedler, U.K., Pornillos, O.W., Morham, S.G., Zavitz, K.H., Wang, H.E., et al. 2001. Tsg101 and the vacuolar protein sorting pathway are essential for HIV-1 budding. Cell, 107: 55–65. doi: 10.1016/S0092-8674(01)00506-2. PMID: 11595185.

Glickman, M.H., and Ciechanover, A. 2002. The ubiquitin-proteasome proteolytic pathway: destruction for the sake of construction. Physiol. Rev. 82: 373–428. PMID: 11917093.

Golab, J., Bauer, T.M., Daniel, V., and Naujokat, C. 2004. Role of the ubiquitin-proteasome pathway in the diagnosis of human diseases. Clin. Chim. Acta, 340: 27–40. doi: 10.1016/j.cccn.2003. 10.025. PMID: 14734194.

Gotoh, B., Komatsu, T., Takeuchi, K., and Yokoo, J. 2002. Paramyxovirus strategies for evading the interferon response. Rev. Med. Virol. 12: 337–357. doi: 10.1002/rmv.357. PMID: 12410527.

Guerin, J.L., Gelfi, J., Boullier, S., Delverdier, M., Bellanger, F.A., Bertagnoli, S., et al. 2002. Myxoma virus leukemia-associated protein is responsible for major histocompatibility complex class I and Fas-CD95 down-regulation and defines scrapins, a new group of surface cellular receptor abductor proteins. J. Virol. 76: 2912–2923. doi: 10.1128/JVI.76.6.2912-2923.2002. PMID: 11861858.

Hagglund, R., and Roizman, B. 2004. Role of ICP0 in the strategy of conquest of the host cell by herpes simplex virus 1. J. Virol. 78: 2169–2178. doi: 10.1128/JVI.78.5.2169-2178.2004. PMID: 14963113.

Herrmann, J., Ciechanover, A., Lerman, L.O., and Lerman, A. 2004. The ubiquitin-proteasome system in cardiovascular diseases-a hypothesis extended. Cardiovasc. Res. 61: 11–21. doi: 10. 1016/j.cardiores.2003.09.033. PMID: 14732197.

Hicke, L. 2001. Protein regulation by monoubiquitin. Nat. Rev. Mol. Cell Biol. 2: 195–201. doi: 10.1038/35056583. PMID: 11265249.

Holowaty, M.N., Zeghouf, M., Wu, H., Tellam, J., Athanasopoulos, V., Greenblatt, J., and Frappier, L. 2003. Protein profiling with Epstein-Barr nuclear antigen-1 reveals an interaction with the

herpesvirus-associated ubiquitin-specific protease HAUSP/ USP7. J. Biol. Chem. 278: 29987–29994. doi: 10.1074/jbc. M303977200. PMID: 12783858.

Ishido, S., Choi, J.K., Lee, B.S., Wang, C., DeMaria, M., Johnson, R.P., Cohen, G.B., and Jung, J.U. 2000. Inhibition of natural killer cell-mediated cytotoxicity by Kaposi’s sarcoma-associated herpesvirus K5 protein. Immunity, 13: 365–374. doi: 10.1016/ S1074-7613(00)00036-4. PMID: 11021534.

Johnson, E.S. 2002. Ubiquitin branches out. Nat. Cell Biol. 4: E295–E298. doi: 10.1038/ncb1202-e295. PMID: 12461537.

Katzmann, D.J., Babst, M., and Emr, S.D. 2001. Ubiquitin-dependent sorting into the multivesicular body pathway requires the function of a conserved endosomal protein sorting complex, ESCRT-I. Cell, 106: 145–155. doi: 10.1016/S0092-8674(01) 00434-2. PMID: 11511343.

Kikkert, M., Hassink, G., Barel, M., Hirsch, C., van der Wal, F.J., and Wiertz, E. 2001. Ubiquitination is essential for human cytomegalovirus US11-mediated dislocation of MHC class I molecules from the endoplasmic reticulum to the cytosol. Biochem. J. 358: 369–377. doi: 10.1042/0264-6021:3580369. PMID: 11513735.

Kikonyogo, A., Bouamr, F., Vana, M.L., Xiang, Y., Aiyar, A., Carter, C., and Leis, J. 2001. Proteins related to the Nedd4 family of ubiquitin protein ligases interact with the L domain of Rous sarcoma virus and are required for gag budding from cells. Proc. Natl. Acad. Sci. USA, 98: 11199–11204. doi: 10.1073/ pnas.201268998. PMID: 11562473.

Levitskaya, J., Coram, M., Levitsky, V., Imreh, S., Steigerwald-Mullen, P.M., Klein, G., Kurilla, M.G., and Masucci, M.G. 1995. Inhibition of antigen processing by the internal repeat region of the Epstein-Barr virus nuclear antigen-1. Nature, 375: 685–688. doi: 10.1038/375685a0. PMID: 7540727.

Levitskaya, J., Sharipo, A., Leonchiks, A., Ciechanover, A., and Masucci, M.G. 1997. Inhibition of ubiquitin/proteasome-dependent protein degradation by the Gly-Ala repeat domain of the Epstein-Barr virus nuclear antigen 1. Proc. Natl. Acad. Sci. USA, 94: 12616–12621. doi: 10.1073/pnas.94.23.12616. PMID: 9356498.

Li, M., Chen, D., Shiloh, A., Luo, J., Nikolaev, A.Y., Qin, J., and Gu, W. 2002. Deubiquitination of p53 by HAUSP is an important pathway for p53 stabilization. Nature, 416: 648–653. doi: 10.1038/nature737. PMID: 11923872.

Luo, H., Wu, Y., Qi, S., Wan, X., Chen, H., and Wu, J. 2001. A proteasome inhibitor effectively prevents mouse heart allograft rejection. Transplantation, 72: 196–202. PMID: 11477338.

Luo, H., Yanagawa, B., Zhang, J., Luo, Z., Zhang, M., Esfandiarei, M., et al. 2002. Coxsackievirus B3 Replication Is Reduced by Inhibition of the Extracellular Signal-Regulated Kinase (ERK) Signaling Pathway. J. Virol. 76: 3365–3373. doi: 10.1128/JVI. 76.7.3365-3373.2002. PMID: 11884562.

Luo, H., Zhang, J., Cheung, C., Suarez, A., McManus, B.M., and Yang, D. 2003a. Proteasome inhibition reduces coxsackievirus B3 replication in murine cardiomyocytes. Am. J. Pathol. 163: 381–385. PMID: 12875959.

Luo, H., Zhang, J., Dastvan, F., Yanagawa, B., Reidy, M.A., Zhang, H.M., et al. 2003b. Ubiquitin-dependent proteolysis of cyclin D1 is associated with coxsackievirus-induced cell growth arrest. J. Virol. 77: 1–9. PMID: 12477805.

Mansouri, M., Bartee, E., Gouveia, K., Hovey Nerenberg, B.T., Barrett, J., Thomas, L., et al. 2003. The PHD/LAP-domain protein M153R of myxomavirus is a ubiquitin ligase that induces the rapid internalization and lysosomal destruction of CD4. J. Virol. 77: 1427–1440. PMID: 12502858.

Meiners, S., Laule, M., Rother, W., Guenther, C., Prauka, I.,

Gao and Luo

Muschick, P., et al. 2002. Ubiquitin-proteasome pathway as a new target for the prevention of restenosis. Circulation, 105: 483–489. doi: 10.1161/hc0402.102951. PMID: 11815432.

Murakami, Y., Matsufuji, S., Kameji, T., Hayashi, S., Igarashi, K., Tamura, T., Tanaka, K., and Ichihara, A. 1992. Ornithine decarboxylase is degraded by the 26S proteasome without ubiquitination. Nature, 360: 597–599. doi: 10.1038/360597a0. PMID: 1334232.

Muratani, M., and Tansey, W.P. 2003. How the ubiquitin-proteasome system controls transcription. Nat. Rev. Mol. Cell Biol. 4: 192–201. doi: 10.1038/nrm1049. PMID: 12612638.

Naujokat, C., and Hoffmann, S. 2002. Role and function of the 26S proteasome in proliferation and apoptosis. Lab. Invest. 82: 965–980. PMID: 12177235.

O’Shea, J.J., Gadina, M., and Schreiber, R.D. 2002. Cytokine signaling in 2002: new surprises in the Jak/Stat pathway. Cell, 109: Suppl, S121–S131. PMID: 11983158.

Ovaa, H., Kessler, B.M., Rolen, U., Galardy, P.J., Ploegh, H.L., and Masucci, M.G. 2004. Activity-based ubiquitin-specific protease (USP) profiling of virus-infected and malignant human cells. Proc. Natl. Acad. Sci. USA, 101: 2253–2258. doi: 10.1073/pnas.0308411100. PMID: 14982996.

Palombella, V.J., Rando, O.J., Goldberg, A.L., and Maniatis, T. 1994. The ubiquitin-proteasome pathway is required for processing the NF-kappa B1 precursor protein and the activation of NF-kappa B. Cell, 78: 773–785. doi: 10.1016/S0092-8674(94)90482-0. PMID: 8087845.

Patnaik, A., Chau, V., and Wills, J.W. 2000. Ubiquitin is part of the retrovirus budding machinery. Proc. Natl. Acad. Sci. USA, 97: 13069–13074. doi: 10.1073/pnas.97.24.13069. PMID: 11087861.

Peloponese, J.M., Jr., Iha, H., Yedavalli, V.R., Miyazato, A., Li, Y., Haller, K., Benkirane, M., and Jeang, K.T. 2004. Ubiquitination of human T-cell leukemia virus type 1 tax modulates its activity. J. Virol. 78: 11686–11695. doi: 10.1128/JVI.78.21.11686-11695.2004. PMID: 15479810.

Pickart, C.M. 2001. Mechanisms underlying ubiquitination. Annu. Rev. Biochem. 70: 503–533. doi: 10.1146/annurev.biochem.70.1.503. PMID: 11395416.

Pye, J., Ardeishirpour, F., McCain, A., Bellinger, D.A., Merricks, E., Adams, J., et al. 2003. Proteasome inhibition ablates activation of NF-kappa B in myocardial reperfusion and reduces reperfusion injury. Am. J. Physiol. Heart Circ. Physiol. 284: H919–H926. PMID: 12424098.

Scheffner, M., Huibregtse, J.M., Vierstra, R.D., and Howley, P.M. 1993. The HPV-16 E6 and E6-AP complex functions as a ubiquitin-protein ligase in the ubiquitination of p53. Cell, 75: 495–505. doi: 10.1016/0092-8674(93)90384-3. PMID: 8221889.

Schubert, U., Anton, L.C., Bacik, I., Cox, J.H., Bour, S., Bennink, J.R., et al. 1998. CD4 glycoprotein degradation induced by human immunodeficiency virus type 1 Vpu protein requires the function of proteasomes and the ubiquitin-conjugating pathway. J. Virol. 72: 2280–2288. PMID: 9499087.

Schubert, U., Ott, D.E., Chertova, E.N., Welker, R., Tessmer, U., Princiotta, M.F., et al. 2000. Proteasome inhibition interferes with gag polyprotein processing, release, and maturation of HIV-1 and HIV-2. Proc. Natl. Acad. Sci. USA, 97: 13057–13062. doi: 10.1073/pnas.97.24.13057. PMID: 11087859.

Schwartz, A.L., and Ciechanover, A. 1999. The ubiquitin-proteasome pathway and pathogenesis of human diseases. Annu. Rev. Med. 50: 57–74. PMID: 10073263.

Shackelford, J., and Pagano, J.S. 2004. Tumor viruses and cell signaling pathways: deubiquitination versus ubiquitination. Mol. Cell. Biol. 24: 5089–5093. doi: 10.1128/MCB.24.12.5089-5093.2004. PMID: 15169876.


Shamu, C.E., Flierman, D., Ploegh, H.L., Rapoport, T.A., and Chau, V. 2001. Polyubiquitination is required for US11-dependent movement of MHC class I heavy chain from endoplasmic reticulum into cytosol. Mol. Biol. Cell, 12: 2546–2555. PMID: 11514634.

Sharpless, N.E., and DePinho, R.A. 2002. p53: good cop/bad cop. Cell, 110: 9–12. doi: 10.1016/S0092-8674(02)00818-8. PMID: 12150992.

Si, X., McManus, B.M., Zhang, J., Yuan, J., Cheung, C., Esfandiari, M., et al. 2005. Pyrrolidine dithiocarbamate reduces coxsackievirus B3 replication through inhibition of the ubiquitin-proteasome pathway. J. Virol. 79: 8014–8023. doi: 10.1128/JVI.79.13.8014-8023.2005. PMID: 15956547.

Strack, B., Calistri, A., Accola, M.A., Palu, G., and Gottlinger, H.G. 2000. A role for ubiquitin ligase recruitment in retrovirus release. Proc. Natl. Acad. Sci. USA, 97: 13063–13068. doi: 10.1073/pnas.97.24.13063. PMID: 11087860.

Sulea, T., Lindner, H.A., Purisima, E.O., and Menard, R. 2005. Deubiquitination, a new function of the severe acute respiratory syndrome coronavirus papain-like protease? J. Virol. 79: 4550–4551. doi: 10.1128/JVI.79.7.4550-4551.2005. PMID: 15767458.

Tiemann, F., and Deppert, W. 1994. Stabilization of the tumor suppressor p53 during cellular transformation by simian virus 40: influence of viral and cellular factors and biological consequences. J. Virol. 68: 2869–2878. PMID: 8151757.

Twombly, R. 2003. First proteasome inhibitor approved for multiple myeloma. J. Natl. Cancer Inst. 95: 845. PMID: 12813164.

Verma, R., Aravind, L., Oania, R., McDonald, W.H., Yates, J.R., 3rd, Koonin, E.V., and Deshaies, R.J. 2002. Role of Rpn11 metalloprotease in deubiquitination and degradation by the 26S proteasome. Science, 298: 611–615. doi: 10.1126/science.1075898. PMID: 12183636.

Vu, P.K., and Sakamoto, K.M. 2000. Ubiquitin-mediated proteolysis and human disease. Mol. Genet. Metab. 71: 261–266. doi: 10.1006/mgme.2000.3058. PMID: 11001820.

Weissman, A.M. 2001. Themes and variations on ubiquitylation. Nat. Rev. Mol. Cell Biol. 2: 169–178. doi: 10.1038/35056563. PMID: 11265246.

Wilkinson, K.D. 2000. Ubiquitination and deubiquitination: targeting of proteins for degradation by the proteasome. Semin. Cell Dev. Biol. 11: 141–148. doi: 10.1006/scdb.2000.0164. PMID: 10906270.

Wing, S.S. 2003. Deubiquitinating enzymes — the importance of driving in reverse along the ubiquitin-proteasome pathway. Int. J. Biochem. Cell Biol. 35: 590–605. doi: 10.1016/S1357-2725(02)00392-8. PMID: 12672452.

Yao, T., and Cohen, R.E. 2002. A cryptic protease couples deubiquitination and degradation by the proteasome. Nature, 419: 403–407. doi: 10.1038/nature01071. PMID: 12353037.

Yasuda, J., Hunter, E., Nakao, M., and Shida, H. 2002. Functional involvement of a novel Nedd4-like ubiquitin ligase on retrovirus budding. EMBO Rep. 3: 636–640. doi: 10.1093/embo-reports/kvf132. PMID: 12101095.

Yeh, E.T. 2002. Ubiquitin, proteasome, and restenosis: a brave new world for cardiovascular research. Circulation, 105: 408–410. PMID: 11815419.

Yew, P.R., and Berk, A.J. 1992. Inhibition of p53 transactivation required for transformation by adenovirus early 1B protein. Nature, 357: 82–85. doi: 10.1038/357082a0. PMID: 1533443.

Yuan, J., Zhang, J., Wong, B.W., Si, X., Wong, J., Yang, D., and Luo, H. 2005. Inhibition of glycogen synthase kinase 3beta suppresses coxsackievirus-induced cytopathic effect and apoptosis via stabilization of beta-catenin. Cell Death Differ. 12: 1097–1106. doi: 10.1038/sj.cdd.4401652. PMID: 15905881.

Zhu, Q., Yao, J., Wani, G., Chen, J., Wang, Q.E., and Wani, A.A. 2004. The ubiquitin-proteasome pathway is required for the function of the viral VP16 transcriptional activation domain.

FEBS Lett. **556**: 19–25. doi: 10.1016/S0014-5793(03)01347-4. PMID: 14706819.
