
Critical Reviews in Biochemistry and Molecular Biology, 41:105-178, 2006
Copyright © Taylor & Francis Group, LLC
ISSN: 1040-9238 print / 1549-7798 online
DOI: 10.1080/10409230600648736

The General Transcription Machinery
and General Cofactors

Mary C. Thomas and
Cheng-Ming Chiang
Department of Biochemistry
Case Western Reserve
University School of Medicine
Cleveland, OH, USA

ABSTRACT In eukaryotes, the core promoter serves as a platform for the assembly of transcription preinitiation complex (PIC) that includes TFIIA, TFIIB, TFIID, TFIIE, TFIIF, TFIIH, and RNA polymerase II (pol II), which function collectively to specify the transcription start site. PIC formation usually begins with TFIID binding to the TATA box, initiator, and/or downstream promoter element (DPE) found in most core promoters, followed by the entry of other general transcription factors (GTFs) and pol II through either a sequential assembly or a preassembled pol II holoenzyme pathway. Formation of this promoter-bound complex is sufficient for a basal level of transcription. However, for activator-dependent (or regulated) transcription, general cofactors are often required to transmit regulatory signals between gene-specific activators and the general transcription machinery. Three classes of general cofactors, including TBP-associated factors (TAFs), Mediator, and upstream stimulatory activity (USA)-derived positive cofactors (PC1/PARP-1, PC2, PC3/DNA topoisomerase I, and PC4) and negative cofactor 1 (NC1/HMGB1), normally function independently or in combination to fine-tune the promoter activity in a gene-specific or cell-specific manner. In addition, other cofactors, such as TAF1, BTAF1, and negative cofactor 2 (NC2), can also modulate TBP or TFIID binding to the core promoter. In general, these cofactors are capable of repressing basal transcription when activators are absent and stimulating transcription in the presence of activators. Here we review the roles of these cofactors and GTFs, as well as TBP-related factors (TRFs), TAF-containing complexes (TFTC, SAGA, SLIK/SALSA, STAGA, and PRC1) and TAF variants, in pol II-mediated transcription, with emphasis on the events occurring after the chromatin has been remodeled but prior to the formation of the first phosphodiester bond.

KEYWORDS core promoter elements, general transcription machinery, transcription cofactors, preinitiation complex, CTD phosphorylation, pol II ubiquitination, E3 ubiquitin ligase, TAF variants

Address correspondence to
Cheng-Ming Chiang, Department of
Biochemistry, Case Western Reserve
University School of Medicine, 10900
Euclid Avenue, Cleveland, OH
44106-4935.E-mail:cmc23@cwru.edu

INTRODUCTION

Discovery of Eukaryotic RNA Polymerase I, II, III, and IV

The view that genetic information flows from DNA to RNA to protein, known as the Central Dogma (Crick, 1958), has been expanded due to the
discovery of additional pathways to perpetuate genetic information, i.e., from RNA to DNA (Baltimore, 1970; Temin and Mizutani, 1970) and to conformational changes induced by proteinaceous infectious agents (Prusiner et al., 1982). The production of RNA, termed transcription, is a highly coordinated process mediated by RNA polymerase, whose enzymatic activity was first discovered by Weiss and Gladstone in 1959 from rat liver nuclei. This enzyme could synthesize RNA in a DNA-dependent manner, as evidenced by the observation that, upon addition of DNase, incorporation of \([a-32P] CTP\) into RNA was drastically reduced (Weiss and Gladstone, 1959). A similar RNA-synthesizing activity was later identified from Escherichia coli (Hurwitz et al., 1960; Stevens, 1960; Chamberlin and Berg, 1962), thereby establishing a universal role of RNA polymerases in transcribing DNA in both eukaryotes and prokaryotes. To date, four different RNA polymerases have been identified in higher eukaryotes. In contrast, only one RNA polymerase is found in prokaryotes and archaea.

Three eukaryotic RNA polymerases (I, II, and III, or named A, B, and C, respectively) were first identified by Roeder and Rutter, based on the chromatographic fractionation of sea urchin embryo nuclei on a DEAE-Sephadex column. RNA polymerase I came off the column first at the lowest salt concentration, whereas RNA polymerase III eluted at the highest salt concentration (Roeder and Rutter, 1969). Although the existence of three eukaryotic RNA polymerases was established in 1969, their functions remained elusive until Chambon's and Roeder's groups found that the specific activity of each RNA polymerase could be resolved based upon their differential sensitivities to \(\alpha\)-amanitin (Gniazdowski et al., 1970; Kedinger et al., 1970; Weinmann and Roeder, 1974; Weinmann et al., 1974), a drug isolated from the death cap fungus, *Amanita phalloides*, that inhibits 50% activity of RNA polymerase II at low concentrations (0.02 \(\mu g/mL)\) and that of RNA polymerase III at high concentrations (20 \(\mu g/mL\); Weinmann et al., 1974). Using \(\alpha\)-amanitin sensitivity assay with endogenous RNA polymerases present in isolated nuclei, Roeder and Rutter (1970) discovered that RNA polymerase I is primarily involved in transcribing 18S and 28S ribosomal RNAs, while RNA polymerase II transcribes mRNAs, and RNA polymerase III is responsible for synthesis of cellular 5S rRNA, tRNAs, and adenovirus VA RNAs. These results were consistent with the finding that RNA polymerase I

is localized within nucleoli, the sites for rRNA synthesis, whereas RNA polymerase II and III are normally present in the nucleoplasm (Roeder and Rutter, 1970; Zylber and Penman, 1971; Weil and Blatti, 1976).

The fourth RNA polymerase, recently identified in plants to facilitate the production of small interfering RNA (siRNA) involved in RNA-directed DNA methylation and transcriptional silencing and formation of heterochromatin, is also present in the nucleus and is resistant to \(\alpha\)-amanitin (Herr et al., 2005; Kanno et al., 2005; Onodera et al., 2005). Although this plant-specific RNA polymerase IV appears to be nonessential for viability, it exhibits unique properties not shared by other nuclear RNA polymerases. Interestingly, an \(\alpha\)-amanitin-resistant single-polypeptide nuclear RNA polymerase (spRNP-IV), structurally unrelated to the multisubunit plant RNA polymerase IV, has been additionally found in human HeLa cells (Kravchenko et al., 2005). This spRNP-IV enzyme, encoded by an alternative transcript derived from the mitochondrial RNA polymerase gene (*POLRMT*) that also encodes the single-polypeptide mitochondrial RNA polymerase (mtRNP), lacks the N-terminal 262-amino acids encompassing the mitochondrial targeting sequence but contains the same C-terminal 968 amino acid catalytic region of mtRNP. While spRNP-IV appears to be structurally distinct from \(\alpha\)-amanitin-sensitive RNA polymerase II, it represents a second RNA polymerase able to transcribe a subset of mRNA-encoding genes that do not contain the core promoter elements and regulatory sequences commonly found in RNA polymerase II-transcribed genes.

Clearly, the findings that each of these eukaryotic RNA polymerases has unique biochemical properties, as exhibited by their distinct chromatographic elution profile, nuclear localization, and differential sensitivity to \(\alpha\)-amanitin, and that RNA polymerase I has 14 subunits, while RNA polymerase II and III possess 12 and 17 subunits, respectively, suggest that these three classical eukaryotic RNA polymerases and the newly identified plant RNA polymerase IV and human spRNP-IV are unlikely to play a functionally redundant role in the cell. However, these enzymes do share a common property in transcribing a diverse set of DNA sequences, although they lack sequence-specific recognition ability to correctly specify the transcription start site unique to each gene. For site-specific initiation, additional proteins are necessary to form an initiation-competent RNA polymerase complex. Here, we review

M. C. Thomas and C.-M. Chiang
A

Nuclear Extract
↓
P11

(Fraction A) (Fraction B) (Fraction C) (Fraction D)
0.1 M 0.3 M 0.5 M 0.85 M
DE52 DE52 DE52
0.1 M 0.1 M 0.3 M 0.1 M 0.3 M
Mono S Mono S Phenyl Superose
0.1 M 0.3 M 0.5 M 0.1 M 0.3 M 1.4 M 0 M
Mono Q TFIIB TFIIE TFIIF TFIIH
TFIIA

B

Nuclear Pellet → sonication → AS ppt → DE52 [0.5 M] → DEAE-Sephadex A-25 → P11 → pol II
0.3 M 0.1 M 0.35 M 0.35 M
0.065 M 0.035 M

FIGURE 1 Purification scheme for partially purified general transcription factors. Fractionation of HeLa nuclear extract (Panel A) and nuclear pellet (Panel B) by column chromatography and the molar concentrations of KCl used for elutions are indicated in the flow chart, except for the Phenyl Superose column where the molar concentrations of ammonium sulfate are shown. A thick horizontal (Panel A) or vertical (Panel B) line indicates that step elutions are used for protein fractionation, whereas a slant line represents a linear gradient used for fractionation. The purification scheme for pol II, starting from sonication of the nuclear pellet, followed by ammonium sulfate (AS) precipitation is shown in Panel B. (Figures are adapted from Flores et al., 1992 and from Ge et al., 1996)

the functional properties of general transcription factors and general cofactors that help RNA polymerase II engage in the transcriptional process in a site-specific and gene-specific manner, with emphasis on the steps regulating preinitiation complex assembly that is often the focal point for gene regulation.

The General Transcription Machinery

Biochemical evidence of accessory factors necessary for site-specific initiation by RNA polymerase II (hereafter referred to as pol II) became evident when purified pol II, supplemented with crude subcellular fractions, was able to accurately transcribe the native adenovirus DNA template *in vitro* (Weil et al., 1979). Further purification of these subcellular fractions over a Whatman P11 phosphocellulose ion exchange column yielded four enzymatically active fractions (A, B, C and D), which corresponded to the nuclear proteins sequentially eluted by 0.1, 0.3, 0.5 (or 0.6), and 0.85 (or 1.0) M KCl-containing buffer. Components within fractions A,

C, and D were necessary for accurate initiation of transcription by pol II (Matsui et al., 1980; Samuels et al., 1982). The protein factors present in the A and D fractions required for pol II-mediated transcription were named TFIIA and TFIID, respectively (Figure 1A). The C fraction was subsequently fractionated into accessory factors TFIIB, TFIIE, TFIIF, and TFIIH (Sawadogo and Roeder, 1985a; Reinberg and Roeder, 1987; Flores et al., 1989; Flores et al., 1992; Ge et al., 1996). These accessory factors (TFIIA, TFIIB, TFIID, TFIIE, TFIIF, and TFIIH) were collectively defined as general transcription factors (GTFs) using the following nomenclature: TF represents Transcription Factor, the Roman numeral II indicates pol II-driven transcription, and the “letter” generally corresponds to which chromatographic fraction the specific GTF was isolated from (Figure 1A). Nuclear pellets, the residual chromosomal fraction following extraction of soluble nuclear components (*i.e.*, nuclear extract) at 0.42 M KCl-containing buffer, were frequently used for pol II purification as outlined

The General Transcription Machinery and General Cofactors

TABLE 1 Components of the human general transcription machinery

| Factor | Protein composition | Function |
|--------|---------------------|----------|
| TFIIA   | p35(α), p19(β), and p12(γ) | Antirepressor; stabilizes TBP-TATA complex; coactivator |
| TFII B  | p33                 | Start site selection; stabilize TBP-TATA complex; pol II/TFIIF recruitment |
| TFII D  | TBP+TAFs(TAF1-TAF14) | Core promoter-binding factor<br>Coactivator<br>Protein kinase<br>Ubiquitin-activating/conjugating activity<br>Histone acetyltransferase |
| TFII E  | p56(α) and p34(β)    | Recruits TFIIH<br>Facilitates formation of an initiation-competent pol II<br>Involved in promoter clearance |
| TFII F  | RAP30andRAP74       | Binds pol II and facilitates pol II recruitment to the promoter<br>Recruits TFII E and TFII H<br>Functions with TFII B and pol II in start site selection<br>Facilitates pol II promoter escape<br>Enhances the efficiency of pol II elongation |
| TFII H  | P89/XPB, p80/XPD, p62, p52,<br>p44, p40/CDK7, p38/Cyclin H,<br>p34, p32/MAT1, and p8/TFB5 | ATPase activity for transcription initiation and promoter clearance<br>Helicase activity for promoter opening<br>Transcription-coupled nucleotide excision repair<br>Kinase activity for phosphorylating pol II CTD<br>E3 ubiquitin ligase activity |
| pol II  | RP B1-RPB12         | Transcription initiation, elongation, termination<br>Recruitment of mRNA capping enzymes<br>Transcription-coupled recruitment of splicing and 3'end processing factors<br>CTD phosphorylation, glycosylation, and ubiquitination |

(Figure 1B). Accessory factors necessary for site-specific initiation by pol II were similarly purified by different groups from HeLa cells (Gerard et al., 1991), rat liver (Conaway et al., 1991), S. cerevisiae (Sayre et al., 1992), and Drosophila (Parker and Topol, 1984; Heberlein et al., 1985; Price et al., 1987). The protein composition and functional properties of each GTF and pol II are briefly summarized in Table 1 and will be further elaborated in the later sections.

Core Promoter Elements

Studies on eukaryotic promoters have thus far identified seven core promoter elements (Figure 2), which are characteristic DNA sequences required for promoter function and for proper assembly and orientation of the transcription preinitiation complex (PIC). The TATA box is an A/T-rich sequence located approximately 25 to 30 nucleotides upstream of the transcription start site (+1) in humans. It contains a consensus sequence, TATA(A/T)A(A/T)(A/G), whose recognition by the TATA-binding protein (TBP) or the TBP subunit

present in the human TFII D complex, nucleates PIC formation (reviewed by Smale and Kadonaga, 2003). A second core promoter element, the initiator (Inr), contains a pyrimidine-rich sequence, PyPyA+1N(T/A)PyPy, surrounding the transcription start site. The Inr is capable of directing accurate transcription initiation either alone or in conjunction with a TATA box or other core promoter elements. The TAF1/TAF2 components of TFII D have been implicated in Inr recognition, as the Inr sequence was preferentially selected for binding by the immobilized human TAF1 and Drosophila TAF2 dimeric complex in random DNA-binding site selection assays (Chalkley and Verrijzer, 1999). The downstream promoter element (DPE) with a consensus sequence, (A/G)G(A/T)CGTG, represents the third core promoter element, which is located 28 to 34 nucleotides downstream of the transcription start site in many Drosophila TATA-less promoters (Burke and Kadonaga, 1996, 1997; Kutach and Kadonaga, 2000), but is situated between +29 and +35 in the human TATA-less TAF7 promoter (Zhou and Chiang, 2001, 2002). The finding that both Drosophila TAF6 and TAF9 can be cross-linked to the

| Core Promoter Element | Position | Consensus Sequence (5' to 3') | Bound Protein |
|-----------------------|----------|-------------------------------|---------------|
| BRE<sup>u</sup>       | -38 to -32 | (G/C)(G/C)(G/A)CGCC          | TFIIB         |
| TATA                 | -31 to -24 | TATA(A/T)A(A/T)(A/G)        | TBP           |
| BRE<sup>d</sup>       | -23 to -17 | (G/A)T(T/G/A)(T/G)(G/T)(T/G) | TFIIB         |
| Inr                  | -2 to +5   | PyPyAN(T/A)PyPy              | TAF1/TAF2     |
| MTE                  | +18 to +29 | C(G/C)A(A/G)C(G/C)(G/C)AACG(G/C) | n.a.         |
| DPE                  | +28 to +34 | (A/G)G(A/T)CGTG              | TAF6/TAF9     |
| DCE                  | 3 subelements<br>+6 to +11<br>+16 to +21<br>+30 to +34 | core sequence:<br>S<sub>I</sub> CTTC<br>S<sub>II</sub> CTGT<br>S<sub>III</sub> AGC | TAF1         |

FIGURE 2 Recognition of core promoter elements by TFIID and TFIIB. The upper figure depicts the interactions between TFIID and TFIIB with the seven core promoter elements discussed in this review. The table in the lower panel lists the consensus sequence and positions for each of these core promoter elements. n.a., not available.

DPE (Burke and Kadonaga, 1997) and the human TAF6 and TAF9 dimeric complex exhibits DPE-binding specificity in electrophoretic mobility shift assays (Shao et al., 2005) indicates that TFIID is the primary GTF recognizing the DPE. Interestingly, negative cofactor 2 (NC2 or Dr1-Drap1), initially characterized as a TBP-inhibitory factor when assayed with a TATA-containing promoter (Meisterernst and Roeder, 1991; Inostroza et al., 1992; Goppelt et al., 1996; Mermelstein et al., 1996), has also been shown to facilitate transcription from DPE-driven *Drosophila* promoters (Willy et al., 2000). It seems that TFIID and NC2, two of the DPE-targeting factors, may work synergistically through the DPE, although their functional relationship remains to be elucidated.

In addition to the DPE, two other core promoter elements, MTE (motif ten element; Ohler et al., 2002; Lim et al., 2004) and DCE (downstream core element; Lee et al., 2005b), are also situated downstream of the transcription start site. The MTE with a consensus sequence C(G/C)A(A/G)C(G/C)(G/C)AACG(G/C) found between +18 to +29 normally functions in conjunction with the Inr to enhance pol II-mediated transcription. It can also functionally substitute for the loss of the TATA box and/or DPE as well as work synergistically with the TATA box and DPE, in an Inr-dependent manner, to strengthen the promoter activity (Lim et al., 2004). However, the protein factors that act through the MTE have not yet been defined.

In contrast to the sequence continuity seen with other core promoter elements, the DCE contains three discontinuous subelements: S<sub>I</sub>, S<sub>II</sub>, and S<sub>III</sub>, with a core sequence of CTTC, CTGT, and AGC, respectively, spanning from +6 to +34 (Lee et al., 2005b; see Figure 2). The presence of DCE and DPE seems to be mutually exclusive, as indicated by the sequence analysis of human promoter databases (Lee et al., 2005b). Interestingly, the TAF1 component of TFIID is able to contact each subelement as revealed by the crosslinking study, suggesting that TFIID indeed plays a

The General Transcription Machinery and General Cofactors

fundamental role in establishing downstream promoter
contacts first observed by DNase I footprinting studies
(Sawadogo and Roeder, 1985a; Nakajima et al., 1988;
Zhou et al., 1992; Chiang et al., 1993; Purnell et al., 1994).
The sixth and seventh core promoter elements are
recognized by a different GTF, TFIIB, through pro-
moter contacts with DNA sequences flanking the TATA
box, normally following TBP-induced DNA bending.
The upstream TFIIB-recognition element (BREu) with
a consensus sequence, (G/C)(G/C)(G/A)CGCC, is
contacted by the helix-turn-helix (HTH) DNA-binding
motif of TFIIB (Lagrange et al., 1998). This TFIIB-BREu
interaction can occur independently of TBP and helps
orient the directionality of the PIC (Lagrange et al.,
1998; Tsai and Sigler, 2000). In contrast, TFIIB
binding to the downstream TFIIB-recognition element
(BREd) with a consensus (G/A)T(T/G/A)(T/G)(G/T)
(T/G)(T/G) via its recognition loop that includes the
evolutionarily conserved G153 and R154 residues
appears to be TBP-dependent (Deng and Roberts,
2005). Although initially defined in TATA-containing
promoters, both BREu and BREd can also be found
in many TATA-less promoters based on bioinformatics
analysis (Deng and Roberts, 2005; Gershenzon and
Ioshikhes, 2005), suggesting that TFIIB is likely to
enhance TFIID binding to the core promoter region
by providing additional contact points.

The identification of these seven core promoter ele-
ments provides a molecular architecture underlying the
diversity of eukaryotic promoters. With available com-
putational tools, the abundance of each core promoter
element can be calculated in a genome-wide basis. A
previous analysis of the Drosophila promoter database
reveals that 57% of the Drosophila promoters do not
contain a TATA box, whereas the DPE is found in ap-
proximately 40% of the Drosophila promoters (Kutach
and Kadonaga, 2000). A similar inspection of 1031 hu-
man genes with putative transcription start sites indi-
cates that TATA+Inr+, TATA+Inr−, TATA−Inr+, and
TATA−Inr− genes constitute 28%, 4%, 56%, and 12%,
respectively, of the screened human genes (Suzuki et al.,
2001). This analysis suggests that 1/3 of human genes
contain a functional TATA box, whereas the major-
ity (~68%) of human promoters are in fact TATA-
less. Following a more comprehensive promoter analy-
sis of EPD (eukaryotic promoter database) and DBTSS
(database of human transcriptional start sites), it is
found that less than 22% of the human genes contain
a TATA box, and among these TATA-containing pro-

moters, ~62% have an Inr, ~24% include a DPE, and
~12% hold a BREu (Gershenzon and Ioshikhes, 2005).
The same study also indicates that more than 78% of the
human genes are TATA-less, and among these TATA-
deficient promoters, ~45% possess an Inr, ~25% have
a DPE, and ~28% harbor a BREu. While these bioin-
formatics analyses provide an overview of the relative
abundance of each core promoter element present in
a given species, the functional role of these core pro-
moter elements in dictating the promoter strength, the
efficiency of PIC assembly, and as targeting regions for
enhancer activity remain to be investigated. From the
functional studies, it seems that the promoters of hu-
man housekeeping genes, oncogenes, growth factors,
and transcription factors often lack a TATA box (Zhang,
1998; Zhou and Chiang, 2001). Moreover, differential
utilization of alternative promoters that use a distinct
combination of core promoter elements also plays a
critical role in regulating gene expression in a spatial,
temporal or lineage-specific manner (Hansen and Tjian,
1995; Ren and Maniatis, 1998). It is interesting to note
that DPE and TATA motifs not only specify the ini-
tiation site for transcription, but also act as regulatory
elements for enhancer function (Butler and Kadonaga,
2001; Smale and Kadonaga, 2003). Undoubtedly, the
dissection of transcription complex assembly pathways
occurring on the core promoter region that serves as a
converging point for regulatory events is of vital impor-
tance for our understanding of the transcription mech-
anism unique to each gene.


# The Sequential Assembly Pathway

The pathways leading to productive PIC assembly
at the promoter region has been intensively studied for
over a decade. In earlier studies, *in vitro* transcription
experiments performed with chromatographic fractions
derived from HeLa cells suggested that an initiation-
competent PIC could be assembled in a stepwise
manner and was stable, prior to the addition of ribonu-
cleoside triphosphates (NTPs), and refractory to chal-
lenges by competitor DNA (Fire et al., 1984), Sarkosyl
(Hawley and Roeder, 1985), and poly(dI-dC:dI-dC)
(Samuels and Sharp, 1986). There appeared to be
multiple steps leading to formation of a stable
promoter-bound complex, since transcription signals
varied depending on the order that pol II, individual
fractions, template and challenge DNA were added
into the reactions. In later studies, it was found that

M. C. Thomas and C.-M. Chiang

A
The Sequential Assembly Pathway

Core Promoter → TFIID → TFIIA → TFIIB → pol II/TFIIF → TFIIE → TFIIH

B
The RNA Polymerase II Holoenzyme Pathway (Two-Component)

TFIID
TFIIA
Core Promoter
SRBs
SWI/SNF
GCN5
pol II
TFIIE
TFIIB
TFIIF
TFIIH
α₂ββ'
core enzyme
specificity factor
cf.
Prokaryotic RNA Polymerase Holoenzyme

FIGURE 3 Pathways for preinitiation complex assembly. Preinitiation complex (PIC) formation may occur by stepwise recruitment of the general transcription machinery (Panel A, the sequential assembly pathway) or by recruitment of preassembled pol II holoenzyme and TFIID complexes as depicted in the two-component pathway (Panel B). The eukaryotic two-component pathway resembles the prokaryotic RNA polymerase holoenzyme system where a dissociable σ factor directs the entry of the bacterial RNA polymerase core enzyme α₂ββ'.

preincubation of partially purified TFIID with the adenovirus major late promoter template, before inclusion of the remaining transcription components, resulted in a PIC that was resistant to template challenge by a second adenovirus major late promoter-containing DNA (Van Dyke et al., 1989). The hierarchical nature of GTF assembly at the promoter region was further defined by Buratowski and colleagues (1989) using native gel electrophoresis and DNase I footprinting to establish the order of entry and relative position of GTFs at the adenovirus major late promoter, thus suggesting an order-of-addition model for PIC formation. Specifically, TFIID first recognizes the TATA box, followed in a stepwise manner by the entry of TFIIA, TFIIB, pol II and TFIIE (note: TFIIF and TFIIH had yet to be identified). Once all of the GTFs, including TFIIF and TFIIH, were identified and purified to near homogeneity, this stepwise GTF assembly pathway was updated as: TFIID first binds to the promoter

region, followed by the entry of TFIIA and TFIIB that help stabilize promoter-bound TFIID, and then the recruitment of pol II/TFIIF. After formation of a stable TFIID-TFIIA-TFIIB-pol II/TFIIF-promoter complex, TFIIE is then recruited, with the subsequent entry of TFIIH. This stepwise manner of PIC assembly became known as the sequential assembly pathway (Figure 3A).

The Pol II Holoenzyme Pathway

An alternative pathway for PIC formation was uncovered when several laboratories discovered that pol II could be purified as a preassembled holoenzyme complex containing pol II and SRBs (suppressors of RNA polymerase B mutations; Kim et al., 1994; Koleske and Young, 1994), with or without a subset of GTFs; and other proteins involved in chromatin remodeling, DNA repair, and mRNA processing (Ossipow et al., 1995; Cairns et al., 1996b; Chao et al., 1996;

The General Transcription Machinery and General Cofactors
Maldonado *et al.*, 1996; Wilson *et al.*, 1996; Yuryev *et al.*, 1996; McCracken *et al.*, 1997; Nakajima *et al.*, 1997; Cho *et al.*, 1998; Wu and Chiang, 1998; Wu *et al.*, 1999; Liu *et al.*, 2001b). Although the composition of pol II holoenzyme complexes isolated from different laboratories appeared to vary according to the methods of purification and the sources of materials, the human pol II holoenzyme complex isolated in our laboratory contains pol II, TFIIB, TFIIE, TFIIF, TFIIH, GCN 5 histone acetyltransferase, SWI/SNF chromatin remodeling factor, and SRBs, but is devoid of TFIID and TFIIA (Wu and Chiang, 1998; Wu *et al.*, 1999; see Figure 3B). The identification of a TFIID-deficient pol II holoenzyme complex suggests that TFIID, as a core promoter-binding factor, may facilitate the entry of pol II holoenzyme to the promoter region, a scenario analogous to the prokaryotic RNA polymerase system where a dissociable σ factor recruits core RNA polymerase (α₂ββ′) to the promoter region for PIC assembly (Figure 3B). However, whether eukaryotic pol II indeed exists as a holoenzyme complex in the cell or is recruited in a stepwise manner as a separate entity, along with individual GTFs, to the promoter region remains to be elucidated. It is likely that both assembly pathways exist in vivo, and, depending on specific signaling molecules involved and the promoter context, either pathway maybe employed in responding to environmental cues. Indeed, evidence supporting both models has been reported for different regulatory systems (Orphanides *et al.*, 1996; Hampsey, 1998; Parvin and Young, 1998; Lee and Young, 2000; Lemon and Tjian, 2000).

### TFIID

TFIID is one of the first GTF that binds to the core promoter and nucleates PIC assembly through either the sequential assembly pathway or the two-component pol II holoenzyme pathway. It is a multiprotein complex comprising TBP and approximately 13 TBP-associated factors (TAFs), with molecular weights ranging from 250 kDa to 15 kDa (Table 2). That TBP and some TAF components of TFIID bind distinct core promoter elements (see Figure 2) classifies TFIID as a core promoter-binding factor. The TBP subunit of human TFIID contacts the TATA box allowing TFIID to recognize TATA-containing promoters, and the interaction between TAF-Inr, TAF-DPE, and TAF-DCE also confer TFIID the ability to recognize TATA-less promoters. It should be noted that although TFIID

was initially defined as a TATA-binding factor by in vitro DNA-binding assays, yeast and human TFIID seem to act primarily through TATA-less promoters in vivo (Basehoar *et al.*, 2004; Kim *et al.*, 2005a).

#### Identification of TATA-Binding Activity

As described earlier, TFIID was originally identified as a chromatographic fraction necessary to support site-specific transcription by pol II in vitro (Matsui *et al.*, 1980; Samuels *et al.*, 1982). This TFIID chromatographic fraction was relatively crude, and many laboratories sought to isolate the key component(s) in TFIID. Interestingly, TFIID was shown to play a key role in binding the TATA box, initially from Drosophila (Parker and Topol, 1984), then mammals (Sawadogo and Roeder, 1985b) and yeast (Buratowski *et al.*, 1988; Cavallini *et al.*, 1988). Once TFIID bound to the TATA box, it was believed to serve as a scaffold upon which the PIC could assemble (Fire *et al.*, 1984; Workman and Roeder, 1987; Hai *et al.*, 1988; Horikoshi *et al.*, 1988; Moncollin *et al.*, 1992). Eventually a single polypeptide possessing TATA box-binding activity (later termed TBP) was purified (Buratowski *et al.*, 1988; Cavallini *et al.*, 1988; Horikoshi *et al.*, 1989a) and cloned from yeast (Cavallini *et al.*, 1989; Hahn *et al.*, 1989; Horikoshi *et al.*, 1989b; Schmidt *et al.*, 1989; Hoffmann *et al.*, 1990a), Drosophila (Hoey *et al.*, 1990; Muhich *et al.*, 1990), and humans (Kao *et al.*, 1990; Peterson *et al.*, 1990; Hoffmann *et al.*, 1990b). The immediate question that followed was whether this single polypeptide TBP is the functional equivalent of TFIID. In an elegant experiment assaying for transcriptional activation dependent on the transcriptional activator Sp1 (specificity protein 1), it was shown that Sp1 could stimulate transcription from TATA-containing promoters only in the presence of partially purified TFIID, but not with recombinant Drosophila or yeast TBP (Pugh and Tjian, 1990). Furthermore, glycerol gradient sedimentation and immunoprecipitation analyses of partially purified TFIID, which supported activator-dependent transcription, indicated that TFIID was a multiprotein complex with a native size of ~750 kDa, rather than a single polypeptide of 38 kDa (Dynlacht *et al.*, 1991; Pugh and Tjian, 1991; Timmers and Sharp, 1991). Thus, it was proposed that additional proteins (*i.e.*, cofactors) were required to work in conjunction with TBP to potentiate transcriptional activation by Sp1. It was subsequently

M. C. Thomas and C.-M. Chiang
TABLE 2 Nomenclature of TAFs involved in RNA polymerase II-mediated transcription

| New name | H. sapiens | D. melanogaster | S. cerevisiae | S. pombe | C. elegans |
|----------|------------|----------------|--------------|----------|------------|
|          |            |                |              | New name | Previous name |
| TAF 1    | TAF II 250 | TAF II 230     | Taf 145/130  | TAF II 111 | taf-1      | taf-1       |
| TAF 2    | TAF II 150 | TAF II 150     | Taf 150      | T38673*   | taf-2      | taf-2       |
| TAF 3†   | TAF II 140 | TAF II 155     | Taf 47       | TAF II 72 | taf-3      | C11G6.1*    |
| TAF 4†   | TAF II 130/135 | TAF II 110 | Taf 48       | T50183*   | taf-4      | taf-5       |
| TAF4b†   | TAF II 105 |                |              |           |            |             |
| TAF 5    | TAF II 100 | TAF II 80      | Taf 90       | TAF II 73 | taf-5      | taf-4       |
| TAF5b    |            |                |              |           |            |             |
| TAF5L    | PAF65α     | Cannonball     |              |           |            |             |
| TAF 6†   | TAF II 80  | TAF II 60      | Taf 60       | CAA 20756* | taf-6.1    | taf-3.1     |
| TAF6L    | PAF65α     | AAF 52013*     |              |           | taf-6.2    | taf-3.2     |
| TAF 7    | TAF II 55  | AAF 54162*     | Taf 67       | TAF II 62/PTR 6 | taf-7.1    | taf-8.1     |
| TAF7L    | TAF2Q      |                |              |           | taf-7.2    | taf-8.2     |
| TAF 8†   | BAB 71460* | Prodos         | Taf 65       | T40895*   | taf-8      | ZK1320.12*  |
| TAF 9†   | TAF II 32/31 | TAF II 40    | Taf 17       | S62536*   | taf-9      | taf-10      |
| TAF9L    | TAF II 31L |                |              |           |            |             |
| TAF 10†  | TAF II 30  | TAF II 24      | Taf 25       | T39928*   | taf-10     | taf-11      |
| TAF10b   |            | TAF II 16      |              |           |            |             |
| TAF 11†  | TAF II 28  | TAF II 30α     | Taf 40       | CAA 93543* | taf-11.1   | taf-7.1     |
| TAF11L   |            |                |              |           | taf-11.2   | taf-7.2     |
| TAF 12†  | TAF II 20/15 | TAF II 30α    | Taf 61/68    | T37702*   | taf-12     | taf-9       |
| TAF 13†  | TAF II 18  | AAF 53875*     | Taf 19       | CAA 19300* | taf-13     | taf-6       |
| TAF 14   |            |                | Taf 30/ANC 1# |           |            |             |
| TAF 15   | TAF II 68# |                |              |           |            |             |
| BTA F 1  | TAF II 170 | Hel89B         | Mot 1        | T40642*   | bta f-1    | F15D4.1*    |

The TAF nomenclature for C. elegans is indicated on the right two columns, shaded in grey. (Adapted from Tora, 2002).

† TAFs with histone fold domains.

* Accession numbers for TAFs not yet published or biochemically characterized.

# Non-conserved TAFs yet to be found in other species.

consensus sequence TATA(A/T)A(A/T)(A/G), is recognized by the C-terminal region of TBP, which is phylogenetically conserved and is made up of ~180 amino acids (Hernandez, 1993; Nikolov and Burley, 1994; Burley and Roeder, 1996). The crystal structures of yeast TBP in complex with the TATA box of the yeast CYC1 promoter (Kim et al., 1993b) as well as Arabidopsis TBP (Kim et al., 1993a) and human TBP (Nikolov et al., 1996) bound to the TATA sequence of the adenovirus major late promoter, all indicate that the DNA is severely bent upon TBP binding. In the bound and unbound state, TBP resembles a molecular “saddle” with a pair of “stirrups” flanking the DNA-binding surface that helps bend the DNA. This saddle-shaped TBP molecule is composed of four α-helices and ten β-strands (Nikolov et al., 1992), which are organized into a bipartite DNA-binding surface. Each half consists of five antiparallel β-strands located on the concave underside of TBP that straddles the DNA and one α-helix forming the side stirrup. The other α-helix situated on the convex upper side interacts with transcription factors (Kim et al., 1993a; Kim et al., 1993b; Nikolov et al., 1996). This concave surface of TBP, with two pairs of phenylalanine residues (Phe 284/Phe 301 and Phe 193/Phe 210 in humans) situated at its outermost edges, binds the TATA box in the minor groove of the DNA double helix and induces sharp kinks (more than 80°) into the DNA via intercalation of Phe 284/Phe 301 at the 5′ end and Phe 193/Phe 210 at the 3′ end of the TATA box (Juo et al., 1996; Nikolov et al., 1996). Moreover, amino acid residues 190 to 194 and 281 to 285 of human TBP, which make up the two side stirrups, accentuate the DNA bend (Juo et al., 1996; Nikolov et al., 1996). These structural studies, revealing a widened minor groove, and a compressed major groove, are consistent with TBP-TATA electrophoretic mobility shift assays using permuted DNA fragments (Horikoshi et al., 1992).

The N-terminal region of human TBP is highly divergent among species and appears dispensable for TFIID complex assembly (Zhou et al., 1993). Human TBP possesses a contiguous but variable stretch of glutamine residues (starting at residue 58), ranging in length from 29 to 42 between individual alleles, followed by three imperfect Pro-Met-Thr (PMT) repeats located around amino acids 142 to 150 preceding the C-terminal core region of TBP. Expansion of the polyglutamine tract beyond 42 residues is frequently linked to human neurodegenerative diseases, such as spinocerebellar ataxia SCA17 (Nakamura et al., 2001). Although not directly

involved in activator-dependent transcription by pol II (Zhou et al., 1993), this N-terminal region of human TBP is necessary for the recruitment of small nuclear RNA-activating protein complex (SNAPc) to the U6 promoter transcribed by RNA polymerase III (pol III). The N-terminal region also inhibits TATA binding by the C-terminal region of TBP on both pol III-(Mittal and Hernandez, 1997) and pol II-dependent promoters (Zhao and Herr, 2002). Since TBP binding to the TATA box is suggested to be a two-step process involving an initial binding of TBP to the TATA box without bending the DNA and followed by a slow transition into a more stable bent TATA-TBP complex, it is interesting to find that deletion of the glutamine-rich domain and PMT repeats in the N-terminal inhibitory region promotes formation of the stable bent TBP-DNA complex. Moreover, it seems that TFIIB’s ability to enhance TBP binding to the TATA box is likely due to induced stabilization of the bent TATA-TBP complex following TFIIB binding to the solvent-exposed surface on the convex side of the TBP core (Zhao and Herr, 2002).

# Regulation of TBP Promoter Binding Activity by Homodimerization

Although TBP’s specificity for DNA binding is ~10³ weaker than most major groove-binding proteins (Coleman and Pugh, 1995), TBP does exhibit nonspecific DNA-binding activity, sometimes leading to formation of nonproductive transcription complexes on scattered AT-rich sequences. To prevent spurious transcription events initiating at nonpromoter sequence elements, TBP may form a homodimer that is inactive in DNA binding (Figure 4A). In addition, association with TAFs may reduce nonspecific DNA-binding activity of TBP and concurrently increase TFIID specificity at TATA-containing and particularly TATA-less promoters. Formation of TBP homodimers was initially observed in the crystal structures of the C-terminal core of Arabidopsis TBP (Nikolov et al., 1992; Nikolov and Burley, 1994), yeast TBP (Chasman et al., 1993), and human TBP (Nikolov et al., 1996), in the absence of DNA, and further confirmed by biochemical analysis of full-length or the C-terminal core of TBP using gel filtration/glycerol gradient fractionation (Kato et al., 1994), photon correlation spectroscopy (Nikolov et al., 1996) and chemical cross-linking studies (Icard-Liepkalns, 1993; Coleman et al., 1995; Jackson-Fisher et al., 1999). Dimerization of TBP is mediated

M. C. Thomas and C.-M. Chiang

A Inhibition of TBP-TATA Complex Formation

B TAF1 Competition with TFIIA for Overlapping Interacting Surfaces on TBP

C ATP-Dependent Dissociation of a Preformed TBP-TATA Complex

D Prevention of TBP-Mediated Nucleation of PIC Assembly

E Nucleosome-Mediated Occlusion

FIGURE 4 Negative regulation of TBP-TATA complex formation. Multiple mechanisms are employed by transcription regulatory proteins to prevent TBP-TATA complex formation.

through extensive contacts between the concave surfaces of each TBP monomer, thereby masking the DNA-binding domain also located in the concave region. Clearly only the monomeric form of TBP can bind to the DNA, as evidenced by the structural analysis of the TBP-TATA complex (Kim *et al.*, 1993a; Kim *et al.*, 1993b; Nikolov *et al.*, 1996). In yeast, these contacts are located in the deepest part of the concave surface, including amino acid residues N69, V71, V122, T124, N159, V161, V213, and T215, since substitutions of these amino acids individually with a bulky charged arginine amino acid result in destabilization of TBP dimers (Kou *et al.*, 2003) and, as shown with

the V161R mutant, also significantly reduce the half-life of TBP *in vivo* (Jackson-Fisher *et al.*, 1999). The ability of TBP to form dimers also leads to the formation of TFIID homodimers, which can be detected by size exclusion column chromatography and by chemical cross-linking (Taggart and Pugh, 1996).

Although dimerization of the TBP core is clearly observed *in vitro*, dimerization of the full-length TBP is not always detectable under different conditions. By examining self-association of yeast TBP over a wide range of salt (60 mM to 1M KCl), protein concentrations (2.6 μM to 31 μM), and temperature (4°C to 37°C), Daugherty and colleagues (1999, 2000) found by

The General Transcription Machinery and General Cofactors

analytic ultracentrifugation that only monomeric rather
than dimeric TBP is in equilibrium with tetrameric and
octameric forms. Also by sedimentation equilibrium
studies, Campbell and coworkers (2000) observed that
at micromolar protein concentrations (6 μM to 60 μM)
only monomeric yeast TBP was detected when 10%
glycerol that prevents TBP aggregation was included
in the buffer. Moreover, by using chemical crosslink-
ing and gel filtration with 2.6 μM of TBP, Vanathi and
colleagues (2003) noted that the monomeric form of
TBP was favored over the dimeric form when 3 mM of
Mg²⁺ was added in 100 mM to 500 mM KCl-containing
buffer. Clearly, TBP self-association is a dynamic event
influenced by ions, salt, buffer, and protein concentra-
tions, wherein the N-terminal region of TBP somehow
modulates these transitions.

### Inhibition of TBP Promoter Binding
Activity by TAF1

The DNA-binding activity of TBP is likewise sub-
ject to negative regulation by TAF1, the largest subunit
of TFIID (Kokubo et al., 1993). At the N-terminus of
Drosophila TAF1 are two separate regions able to con-
tact TBP: amino acid residues 11 to 77 (TAND1, for
TAF N-terminal domain 1) interacting with the con-
cave underside of TBP to block TATA recognition (Liu
et al., 1998; see Figure 4A) and amino acid residues
82 to 156 (TAND2) binding to the convex surface of
TBP to compete with TFIIA that would otherwise fa-
cilitate TBP-TATA complex formation (Kokubo et al.,
1998; see Figure 4B). Interestingly, the solution struc-
ture of Drosophila TAND1 bound with yeast TBP shows
a remarkable resemblance to the structure of the TBP-
TATA complex (Liu et al., 1998), as TAND1 exhibits an
arch-shaped surface contacting the concave underside
of TBP through both hydrophobic and electrostatic in-
teractions. The negatively charged side chains of TAF1
(Asp 29, Glu 31, Glu 51, Glu 70, and Asp 73) interact with
the conserved lysine and arginine residues of TBP that
also contact the phosphate backbone of the TATA se-
quence. Competition for the same binding surface un-
derlies the mechanism for TAF1-mediated inhibition
of TBP binding to the TATA box (Kokubo et al., 1998),
which may also account for reduced TATA-binding and
transcription activity of TFIID, in comparison with
TBP, in vivo and in vitro (Ozer et al., 1998b; Wu and
Chiang, 1998; Wu et al., 1998). Likewise, TAND2-
mediated inhibition of TFIIA binding to TBP is due

to competition between TAF1 and TFIIA for the same
conserved positive charge residues on the convex sur-
face of TBP (Kokubo et al., 1998; see Figure 4B). This
TAF1-mediated effect on TBP-TFIIA and TBP-TATA in-
teractions appear to be functionally conserved, as the
same observation is also seen with experiments per-
formed with homologous yeast and human proteins
(Kokubo et al., 1998; Ozer et al., 1998b; Banik et al.,
2001).

#### Regulation of TBP-Promoter Complex
Formation by BTAF1

Other than interacting with TAFs as part of TFIID,
TBP also forms a distinct complex involved in pol
II-mediated transcription. This “TFIID-like” complex,
found in the P110.3MKCl (or “B”) fraction, is named
B-TFIID, which is composed of TBP and a unique TBP-
associated factor called BTAF1 (Timmers et al., 1992).
This BTAF1 protein was formerly named TAFII170
and TAF-172 in humans (Timmers et al., 1992; van
der Knapp et al., 1997; Chicca et al., 1998), Mot1 in
yeast (Poon et al., 1994), and 89B helicase in Drosophila
(Goldman-Levi et al., 1996). As observed with TAF1,
human BTAF1 binds the concave and convex surfaces
of TBP likely in a reversible manner (Pereira et al.,
2001). The TBP concave-binding region, located at the
N-terminal amino acid residues 290 to 381 of BTAF1,
not only hinders TBP-DNA complex formation, but
also blocks TAF1 interaction with the concave under-
side of TBP (Pereira et al., 2001; see Figure 4A). While
the N-terminus of BTAF1 interacts with TBP, the C-
terminus possesses enzymatic activity allowing BTAF1
to induce the dissociation of TBP-TATA complexes in
an ATP-dependent manner (Chicca et al., 1998; see
Figure 4C). This ATPase domain, with a conserved sig-
nature DEGH box within the Walker A motif that
classifies BTAF1 as a member of the DNA-dependent
SWI2/SNF2 ATPase family (Pereira et al., 2003), how-
ever, is not necessary for BTAF1 binding to TBP (Auble
et al., 1997; Gumbs et al., 2003). Thus, it is appar-
ent that both the N-terminal TBP interaction domain
and the C-terminal ATPase domain of BTAF1 are
needed to remove TBP from the TATA box. This DNA-
translocating activity also allows BTAF1 to clear TBP
from nonpromoter AT-rich sequences, thereby enhanc-
ing the frequency of TBP binding to functional TATA
elements. The redistribution of TBP to correct pro-
moter regions may partially account for the coactivating

M. C. Thomas and C.-M. Chiang

function of BTAF1 in stimulating both basal and
activator-dependent transcription (Collart, 1996; Li
et al., 1999; Muldrow et al., 1999; Andrau et al., 2002;
Dasgupta et al., 2002; Geisberg et al., 2002). Concurrently, BTAF1 may compete with other TBP-interacting
factors, such as NC2 (see below) and TFIIA, along TBP's
convex surface (Geisberg et al., 2002; Klejman et al.,
2005) to further modulate promoter activity. An additional control mechanism lies in the observation that, in
yeast, Mot1-TBP seems to exist as a transcriptionally inactive promoter-bound complex under normal growth
conditions and is later activated by associating with
other GTFs and pol II following environmental stress
(Geisberg and Struhl, 2004). The dual role of BTAF1 in
gene activation and repression is in agreement with the
genome-wide transcriptional profiling data indicating
that ~10% and ~5% of yeast genes are, respectively,
more than two-fold upregulated and downregulated by
BTAF1 (Dasgupta et al., 2002; Geisberg et al., 2002).
Although BTAF1 recruitment to TATA-containing
promoters seems to be TBP-dependent (Geisberg and
Struhl, 2004), BTAF1 can also stimulate transcription
in a TBP-independent manner by facilitating chromatin
remodeling via the use of its ATPase activity (Topalidou
et al., 2004). This coactivating activity of BTAF1 may
account for the requirement of BTAF1 for transcription
from some yeast TATA-less promoters (Collart,
1996). Nonetheless, whether TBP is somehow involved
in BTAF1 recruitment to promoter regions lacking a
canonical TATA box is yet to be resolved. A recent
finding that BTAF1 is capable of recruiting TBP concave surface mutants defective in TATA binding to promoter fragments containing either wild-type or mutated TATA sequences (Klejman et al., 2005) further
suggests that BTAF1 may alter the DNA-binding specificity of TBP, allowing TBP to associate with TATAless promoters. This intriguing possibility remains to be
explored.

NC2 Regulation of TBP-TATA
Complex Formation

A third TBP-interacting factor able to regulate TBP
binding to the promoter region is negative cofactor 2
(NC2), which is conserved in eukaryotes and is essential
for yeast viability (Inostroza et al., 1992; Goppelt and
Meisterernst, 1996; Gadbois et al., 1997). Human NC2
consists of NC2α (Drap1; 22 kDa) and NC2β (Dr1; 20
kDa) that interact with each other through histone fold

The General Transcription Machinery and General Cofactors

motifs, resembling the dimerization domain found in
histones H2A and H2B (Goppelt et al., 1996; Kamada
et al., 2001b). Both NC2α and NC2β subunits are
required for stable NC2-TBP-TATA complex formation
in electrophoretic mobility shift assays and also for
NC2 to repress transcription from TATA-containing
promoters in reconstituted cell-free transcription
assays (Goppelt et al., 1996). The structural analysis of
NC2-TBP-TATA ternary complex, revealed by X-ray
crystallography at 2.6 Å resolution, shows that the
N-terminal regions of both NC2 subunits bind DNA
on the underside of the preformed TBP-TATA complex
and the C-terminus of NC2β additionally contacts
the convex surface of TBP, hence giving NC2 the appearance of a molecular “clamp” that grips both upper
and lower surfaces of the TBP-TATA complex (Kamada
et al., 2001b; see Figure 4D). As further demonstrated by
electrophoretic mobility shift assays using NC2α mutants with an adenovirus major late TATA-containing
promoter fragment, the C-terminus of NC2α indeed
contributes to TBP-NC2-DNA complex formation
(Gilfillan et al., 2005). Obviously, this molecular clamp
is able to block PIC assembly by inhibiting TFIIA and
TFIIB binding to the upper side of TBP, an observation
consistent with electrophoretic mobility shift assays
and in vitro transcription experiments performed with
recombinant proteins (Inostroza et al., 1992; Kim et al.,
1995; Goppelt et al., 1996). Indeed, some transcription
factors, such as hypoxia-inducible factor 1α (HIF-1α),
are able to inhibit transcription by inducing NC2
activity that, in turn, blocks PIC assembly (Denko
et al., 2003).

In addition to the repressive activity normally seen
with TATA-containing promoters (Meisterernst and
Roeder, 1991; Inostroza et al., 1992), NC2 can also
function as a coactivator in stimulating transcription
from some TATA-less promoters (Lemaire et al., 2000;
Willy et al., 2000). The dual role of NC2 in transcription allows it to regulate expression of ~17% of yeast
genes, among which nearly half is either upregulated
or downregulated (Geisberg et al., 2001). Perhaps one
of the underlying mechanisms for NC2 coactivator
function is that NC2 works in conjunction with TFIID
through the DPE to enhance TATA-less gene transcription (Burke and Kadonaga, 1997; Willy et al., 2000).
The observation that NC2 recruitment co-occurs with
TBP on a number of TATA-containing and TATA-less
human and yeast promoters, as revealed by in vivo chromatin immunoprecipitation (ChIP) assays, suggests that

NC 2 may facilitate PIC formation by modulating TBP accessibility to the promoter region (Geisberg et al., 2001; Cang and Prelich, 2002; Creton et al., 2002; Gilfillan et al., 2005). This finding is consistent with those of in vitro DNA-binding studies showing that the TATA-binding defects of TBP concave mutants can be restored by NC 2, as also observed with BTAF1, in electrophoretic mobility shift assays (Klejman et al., 2005). It is important to note that this NC 2-induced TBP conformational change leading to productive PIC formation is likely distinct from the inhibitory clamp that prevents TBP association with TFIIA and TFIIB. As alluded to earlier in the discussion of BTAF1, NC 2 likely competes with BTAF1 for the overlapping region on the convex surface of TBP, as illustrated by ChIP and in vitro DNA-binding assays showing that NC 2-TBP and BTAF1-TBP complexes cannot be detected simultaneously on the same promoter (Geisberg et al., 2002; Klejman et al., 2005). Alternatively, NC 2 may positively regulate transcription by targeting events downstream of the initiation step (e.g., elongation), since NC 2 association with the hyperphosphorylated form of pol II seems necessary for Gal4-VP16-mediated activation from the TATA-containing HIV-1 promoter (Castano et al., 2000). However, the exact mechanism underlying the coactivating function of NC 2 during the transcription process remains to be elucidated.

In addition to forming a heterodimeric NC 2 complex, NC2α and NC2β are also individually involved in the control of specific gene transcription. It seems that free forms of NC2α and NC2β are mainly found in exponentially growing yeast cells, whereas stable NC2α and NC2β complex is only detected after glucose depletion (Creton et al., 2002). Interestingly, NC2α appears to co-reside with TBP at transcriptionally active promoters and NC2β is mainly associated with TBP-bound promoters at transcriptionally repressed genes (Creton et al., 2002). This is not surprising considering that NC 2 binding to the upper surface of TBP blocks TFIIB association and the available NC2α structure shows only binding to the underside of the TBP-TATA complex presumably without affecting PIC assembly (Kamada et al., 2001b). Unexpectedly, the free entity of human NC2α, but not NC2β, or the NC 2 complex, can interact with BTAF1 and in turn stimulates BTAF1 association with TBP on the DNA, as detected by in vitro DNA-binding assays (Klejman et al., 2004). This finding suggests that NC2α and BTAF1 may bind at the same time to the underside and the upper region of the

TBP-TATA complex, respectively. The functional role of this unusual quaternary complex, however, has not yet been addressed. Obviously, the presence of multiple forms of NC 2-TBP complexes and the observations that NC 2 can bind nonspecifically to DNA and is susceptible to post-translational modification by casein kinase II (CK 2; Goppelt et al., 1996; Creton et al., 2002) provide an additional level of complexity for regulating promoter recognition by TBP and TFIID.

### Positive Regulators that Promote TBP-TATA Complex Formation

For TBP to nucleate the assembly of a functional PIC, it must overcome molecular impediments that prevent productive TBP-TATA complex formation. Some of the impediments, as mentioned in the previous sections, include dimerization of TBP, inhibition of TBP binding to the TATA box exerted by TAF1 and BTAF1, and prevention of TFIIA and TFIIB association with the TBP-TATA complex by NC 2 (see Figure 4A-D), as well as by nucleosomes assembled on the promoter region (Figure 4E). Alleviation of these inhibitory activities can be achieved by transcriptional regulators, which may function at the promoter recognition step by enhancing TBP binding to the TATA box, antagonizing repressor binding for the overlapping surfaces on TBP, stabilizing the bent TATA-TBP complex, or by modifying the chromatin structure surrounding the promoter region.

In general, TFIIA and TFIIB enhance TBP binding to the TATA box and further stabilize the TBP-promoter complex via distinct mechanisms (Figure 5A). TFIIA increases the likelihood of precise promoter recognition by TBP via promoting the dissociation of TBP dimers and thus facilitating the loading of monomeric TBP onto the TATA box (Coleman et al., 1999; see Figure 5B). TFIIA also competes with the inhibitory domain of TAF1 for overlapping binding regions on TBP, hence alleviating TAF1-mediated inhibition of TATA recognition (Kokuba et al., 1998). Furthermore, incorporation of TFIIA into the TBP-TATA complex renders the ternary complex resistant to BTAF1-mediated dissociation of the TBP-TATA complex (Auble and Hahn, 1993). Similarly, TFIIB can enhance TBP binding to the TATA box (Imbalzano et al., 1994b) and also stabilize the bent TBP-TATA complex (Zhao and Herr, 2002), thereby reducing the dissociation rate of TBP from the promoter region (Wolner and Gralla, 2001).

M. C. Thomas and C.-M. Chiang

A Enhancement and Stabilization of TBP Binding to TATA by TFIIA and TFIIB

B TFIIA-Induced Dissociation of TBP Dimers

C Stabilization of TBP-TATA Complex at BRE-Containing Promoters

D Chromatin-Modifying Enzymes Increase the Exposure of the TATA Box for TBP Binding

FIGURE 5 Positive regulation of TBP DNA-binding activity. (A) TBP binding to the TATA box can be stabilized by TFIIA and TFIIB. (B) Inhibition of TBP binding to the TATA box can be alleviated by TFIIA-induced dissociation of TBP dimers. (C) TFIIB binding to BRE may provide additional contact points for further stabilization of TBP-TATA complex formation. (D) Chromatin-modifying enzymes that are recruited by activators may facilitate TBP loading onto the TATA box.

Besides protein-protein interaction observed between TFIIB and TBP, additional DNA contacts provided by TFIIB-BRE<sup>u</sup> and TFIIB-BRE<sup>d</sup> likely contribute to this stabilizing effect (Figure 5C). Clearly, TFIIA and TFIIB help form a stable TBP-TATA complex, yet both protein factors may not efficiently enhance TBP binding to the TATA box when the promoter is in a nucleosome configuration.

Considering that approximately 146 base pairs of DNA are wrapped around each core histone octamer in vivo, the promoter region may be incorporated into a nucleosome, rendering the TATA box inaccessible for TBP binding thus suppressing basal transcription (Imbalzano et al., 1994a; see Figure 4E). Undoubtedly, additional protein factors are needed to alleviate transcriptional repression from nucleosome-embedded promoters. In this regard, chromatin-modifying enzymes play a key role in enhancing TBP access to its recognition sequence by altering chromatin structure surrounding the TATA box (Martinez-Campa et al., 2004; see Figure 5D). Chromatin modifiers may reconfigure nucleosome structure by covalently modifying histones to reduce protein-DNA interactions (Fischle et al., 2003) or by using the energy of ATP hydrolysis to alter histone-DNA contacts in a process known as chromatin remodeling (Narlikar et al., 2002). Since the interactions between the TATA box and nucleosomal core histones are regulated by acetylation of lysine residues at the N-terminal tails of core histones, acetylation of nucleosomal core histones at the promoter region creates a less compact chromatin structure, allowing TBP to bind the TATA box (Sewack et al., 2001). SWI/SNF is an example of an ATP-dependent chromatin remodeler that enhances TBP access to the TATA box situated in a nucleosome (Imbalzano et al., 1994a). The action of SWI/SNF following the acetylation of histones by GCN5 histone acetyltransferase is seen in vivo as well, as exemplified by SWI/SNF-facilitated TBP binding to the β-interferon promoter (Agalioti et al., 2000). Interestingly, loading of TBP onto both TATA-containing and TATA-less promoters can be further enhanced by TBP association with other cellular proteins to form multiprotein complexes, such as TFIID and SAGA (see discussion of TAF-containing

The General Transcription Machinery and General Cofactors

complexes), both of which contain acetyltransferase activity.

### TAF Recognition of Distinct Core Promoter Elements

The association of TBP with the TAF components of TFIID not only helps TBP to overcome chromatin barriers (Wu et al., 1999), but also significantly broadens the scope of promoter recognition by this important core promoter-binding factor. Previous DNase I footprinting assays revealed that TFIID generates an extended protection over the TATA box on the adenovirus major late promoter when compared to that of TBP (Zhou et al., 1992; Chiang et al., 1993), indicating that TAFs may provide additional contacts with core promoter sequences (Oelgeschläger et al., 1996; Verrijzer and Tjian, 1996). As described earlier in the discussion of core promoter elements, TFIID may contact the Inr via TAF1/TAF2, the DPE via TAF6/TAF9, and the DCE via TAF1, highlighting the importance of TFIID in core promoter recognition (see Figure 2). These additional TFIID-DNA contacts are particularly important at promoters lacking TATA sequences, as TAFs are required for transcription from TATA-less promoters (Pugh and Tjian, 1991; Martinez et al., 1994; Orphanides et al., 1996; Smale, 1997; Huisenga and Pugh, 2004). Recent findings that human TAF6 and TAF9 display sequence preference for the DPE present in the human interferon regulatory factor-1 promoter (Shao et al., 2005) and that TAF1 can be crosslinked to individual DCE subelements (Lee et al., 2005b), distinct from the previously identified upstream DNA contacts by TAF4, TAF5, TAF7 and an undefined 30 kDa TAF (Oelgeschläger et al., 1996), in the adenovirus major late promoter further sustain the view that TFIID is a bona fide core promoter-binding factor. The extensive promoter contacts established by TFIID may provide additional stability for TFIID to act as a commitment factor. Indeed, as revealed by ChIP assays performed in HeLa cells, TFIID remains bound to active gene promoters during mitosis (Christova and Oelgeschläger, 2002), even though TFIID is likely inactivated due to mitosis-specific phosphorylation of some TFIID components (Segil et al., 1996). Thus, TFIID may function as a promoter-marking factor for transcriptional memory in order to distinguish active from inactive genes as cells proceed through the cell cycle.

---

### Histone-Like TAFs in TFIID

Many of the TAFs present within TFIID contain histone fold domains that contribute to the recognition of core promoter elements and to the integrity of TFIID complexes. Among many histone fold-containing TAFs identified thus far (*i.e.*, TAF3, TAF4, TAF4b, TAF6, and TAF8-13), four of them (TAF6, TAF9, TAF12, and TAF4/TAF4b) share sequence similarity to core histones H4, H3, H2B, and H2A, respectively (Gangloff et al., 2001). A histone fold motif, consisting of a long α helix (α2) flanked on each side by a random coil loop (L1 or L2) and a short α helix (α1 or α3), is organized in the following order: α1-L1-α2-L2-α3 (Hoffmann et al., 1996; Xie et al., 1996; Birck et al., 1998). This motif present in histone-like TAFs suggests that a TAF9-TAF6 heterotetramer and two of the TAF4b-TAF12 heterodimer may form a histone octamer-like structure (Selleck et al., 2001), similar to H3-H4 tetramers and H2A-H2B dimers found in a nucleosome (Luger et al., 1997). Indeed, initial crystal structures of *Drosophila* TAF9-TAF6 complexes revealed a heterotetramer, resembling the H3-H4 heterotetrameric core of the histone octamer. This finding indicates that TFIID may contain a histone octamer-like substructure (Xie et al., 1996). Further supporting evidence for a histone-like octamer was demonstrated when it was found that yeast TAF9-TAF6-TAF12-TAF4 may reconstitute an octamer in vitro (Selleck et al., 2001). The immediate question that follows is whether the histone-like octamer of TAFs is able to contact DPE in the context of a nucleosome-like structure. In a study investigating protein-protein interactions among human TAF9, TAF6, TAF4b, and TAF12, which contain sequences related to histones H3, H4, H2A, and H2B, respectively, it was found that these TAFs indeed form an octamer-like complex, which enhances both sequence-specific and nonspecific DNA-binding activities of TAF9-TAF6 and TAF4b-TAF12 pairs, respectively (Shao et al., 2005).

In contrast to the interaction studies, which suggest that TFIID may contain a histone-like octamer, low-resolution electron microscopy (EM) studies of human and yeast TFIID complexes do not unveil an octamer-like structure within TFIID (Andel et al., 1999; Brand et al., 1999a; Leurent et al., 2002, 2004). These EM studies revealed TFIID to be a trilobed (termed lobes A, B, and C), horseshoe-shaped structure with TBP sitting in the central cavity, while TFIIA and TFIIB bound to opposite lobes of the horseshoe-shaped structure

M. C. Thomas and C.-M. Chiang
(Andel *et al.*, 1999; Brand *et al.*, 1999a; Leurent *et al.*, 2002). Each of the globular domains found in the TFIID EM structure is almost equivalent in size to the histone octamer, but none are large enough to accommodate all histone motif-containing TAFs (Brand *et al.*, 1999a). Therefore, a histone octamer-like structure containing all the TAFs with histone fold motifs may not exist in TFIID. However, the possibility that fewer TAFs may form a nucleosome-like structure remains to be explored. On the other hand, the histone folds in TAFs are likely to be important for protein-protein interactions for maintaining the structural integrity of TFIID, as pairwise interactions observed between histone-like TAFs can indeed be detected in yeast TFIID (Leurent *et al.*, 2002). From TAF localization studies of yeast TFIID revealed by immuno-EM, it appears that TAF5 plays a central role in connecting the three lobes of TFIID, with each lobe harboring a subset of histone-like TAFs (Leurent *et al.*, 2004). Consistent with the functional studies showing that TAND1 blocks TBP binding to the TATA box (Liu *et al.*, 1998) and TAF7 inhibits the histone acetyltransferase activity of TAF1 (Gegonne *et al.*, 2001), it was found that the N-terminal domain of TAF1 is spanned over TBP in the central lobe, whereas the histone acetyltransferase domain of TAF1 is colocalized with TAF7 in the same side lobe (Leurent *et al.*, 2004). The characterization of TFIID structures undoubtedly will further shed light on the molecular mechanism of TFIID function in PIC assembly.

### Enzymatic Activities of TFIID

The presence of TAF1 in TFIID not only modulates DNA binding by TBP, but also confers on TFIID multiple enzymatic activities to post-translationally modify histones and transcription factors, thus allowing TFIID to serve as a core promoter-binding factor in the context of chromatin and as a coactivator mediating activator response. Human, *Drosophila* and yeast TAF1, a TFIID-specific subunit, is known to function as a histone acetyltransferase (HAT) in acetylating histones H3 and H4 *in vitro* (Mizzen *et al.*, 1996). *Drosophila* TAF1 has also been shown to act as a kinase in phosphorylating histone H2B (Maile *et al.*, 2004) and as a histone-specific ubiquitin-activating/conjugating enzyme in mediating monoubiquitination of linker histone H1 (Pham and Sauer, 2000) both *in vitro* and *in vivo*. These histone-modifying activities indicate that

TFIID may be specifically needed for transcription from nucleosome-embedded promoters. Indeed, only TFIID, rather than TBP, can work in conjunction with a TFIID-deficient pol II holoenzyme complex to facilitate activator-dependent transcription from an *in vitro*-reconstituted chromatin template, suggesting a unique involvement of TAFs in chromatin transcription (Wu *et al.*, 1999). The importance of the acetyltransferase activity of TAF1 for gene transcription has also been demonstrated for MHC class I genes, where suppression of TAF1 enzymatic activity by TAF7, which is a TAF1-interacting protein able to contact multiple transcriptional activators (Chiang and Roeder, 1995; Lavigne *et al.*, 1999), leads to inhibition of transcription (Gegonne *et al.*, 2001). The involvement of TAF1 HAT activity in gene transcription, however, appears to be promoter-specific, as hypoacetylation of H3 due to TAF1 inactivation in the ts13 hamster cell line was only observed at the cyclin D1 promoter, but not the c-fos promoter (Hilton *et al.*, 2005). Similar to some HATs, higher eukaryotic TAF1 contains two tandem bromodomains recognizing acetylated lysine 14 (K14) of histone H3 as well as acetylated K5, K8, K12, and K16 of histone H4 (Jacobson *et al.*, 2000; Kanno *et al.*, 2004). This bromodomain-mediated interaction might enhance TFIID binding to acetylated nucleosomes at promoters previously modified by activator-recruited HATs. The structure of the human TAF1 double bromodomain spanning amino acid residues 1359 to 1638, resolved by X-ray crystallography at 2.1 Å resolution, reveals that two acetylated lysine residues of histone H4, separated by 7 or 8 amino acids (*i.e.*, K5/K12 or K8/K16), are preferentially recognized by the double bromodomain, where the Nε-acetyllysine-binding pocket is situated within the amino acid residues of the two loops connecting the four antiparallel α-helices that constitute the core of each bromodomain (Jacobson *et al.*, 2000).

Other than acting on histone substrates, TAF1, either as a free entity or part of TFIID, can also covalently modify other general transcription factors and cofactors, as demonstrated by acetylation on TFIIEβ (Imhof *et al.*, 1997), phosphorylation on RAP74 (Dikstein *et al.*, 1996), serine 33 of histone H2B (Maile *et al.*, 2004), PC4 (Kershner *et al.*, 1998; Malik *et al.*, 1998), and the β subunit of TFIIA (Solow *et al.*, 2001), and presumably ubiquitination on TAF5 and itself as well (Auty *et al.*, 2004). Interestingly, TAF1-mediated phosphorylation of histone H2B correlates with gene activation

The General Transcription Machinery and General Cofactors(Maile *et al.*, 2004). Thus, the ability of TAF1 to modify distinct histones by acetylation, phosphorylation, or ubiquitination may account for TFIID involvement in chromatin transcription (Wu *et al.*, 1999).

### TFIID as a Coactivator

Characterization of TAFs revealed that many transcriptional activators interact directly with specific TAFs (reviewed by Burley and Roeder, 1996; Verrijzer and Tjian, 1996). For example, the activation domain of Sp1 was found to interact with *Drosophila* TAF4 (Hoey *et al.*, 1993), whereas the DNA-binding domain of Sp1 was shown to contact human TAF7 (Chiang and Roeder, 1995). The finding that distinct domains in an activator are capable of contacting different TAFs suggests that TFIID may modulate activator function through multiple domain interactions (Chiang and Roeder, 1995), presumably by enhancing activator binding to the target site and further stabilizing TFIID-promoter contacts. That human TAF7 is able to interact with different transcriptional activators, including Sp1, YY1, USF, CTF, adenovirus E1A, and HIV-1 Tat proteins through its N-terminal domain, and with TAF1 via its central region indicates that TAF7 indeed acts as a coactivator bridging the activator and the general transcription machinery (Chiang and Roeder, 1995). Additional contacts between TAF7 and other TFIID components, such as TAF5, TAF11, TAF12, and TAF13, have also been demonstrated in human TFIID (Lavigne *et al.*, 1996) and have been verified by immuno-EM structural analysis performed with yeast TFIID (Leurent *et al.*, 2004). These activator-TAF and TAF-TAF interactions imply that activators may function by recruiting TFIID to the promoter in order to nucleate PIC formation (Burley and Roeder, 1996; Verrijzer and Tjian, 1996; Albright and Tjian 2000; Naar *et al.*, 2001). Consistent with its role as a coactivator, only purified TFIID, but not TBP, supports activator-mediated transcription in partially purified cell-free transcription systems (Dynlacht *et al.*, 1991; Chiang *et al.*, 1993). Thus the *in vitro* biochemical concept that TFIID was a universal coactivator required for all gene transcription was formed.

When the role of TAFs was later examined in yeast, it was found that transcription from selective genes *in vivo* could occur in the absence of TAFs (Moqtaderi *et al.*, 1996; Walker *et al.*, 1996). Additional evidence supporting the view that TAFs may not be universally

required for gene function *in vivo* came from studies performed with hamster ts13 cells harboring an amino acid substitution G690D (Hayashida *et al.*, 1994; human equivalent G716D) in TAF1 (Wang and Tjian, 1994; Suzuki-Yagawa *et al.*, 1997) and with chicken DT40 TAF9-conditional knockout cells (Chen and Manley, 2000). Indeed, two distinct classes of yeast genes (TAF-dependent and TAF-independent) have been identified based on their differential requirement for TFIID TAFs (Kuras *et al.*, 2000; Li *et al.*, 2000). Although TAF1 was originally reported to play a limited role in transcription for only a subset (~16%) of yeast genes (Holstege *et al.*, 1998), more refined genome-wide gene profiling studies have indicated a broader involvement of TAF1 for transcription from the majority (~90%) of yeast and human genes whose promoters are predominantly TATA-less (Huisenga and Pugh, 2004; Kim *et al.*, 2005a) and primarily for housekeeping, such as the ribosomal protein-encoding gene promoters (Mencia *et al.*, 2002). This is in agreement with a similar requirement of TAF9 for the majority (~67%) of yeast gene transcription (Apone *et al.*, 1998; Holstege *et al.*, 1998; Michel *et al.*, 1998; Moqtaderi *et al.*, 1998). It should be noted that histone fold-containing TAF9 is present not only in TFIID, but also in several other TAF-containing complexes, including TFTC, SAGA, SLIK, and STAGA (see discussion of TAF-containing complexes). Therefore, inactivation of TAF9 may affect the function of multiple protein complexes.

The finding that TAFs may not be essential *in vivo* for every gene function has also been verified *in vitro* by cell-free transcription studies using highly purified and well-defined transcription systems. In these studies, TBP, in the absence of TAFs but in conjunction with other GTFs, pol II and general cofactor PC4, is able to direct activated transcription from TATA-containing promoters in an activator-specific manner with nucleosome-free DNA templates (Oelgeschläger *et al.*, 1998; Wu and Chiang, 1998; Wu *et al.*, 1998, 1999; Fondell *et al.*, 1999). TBP, free from TAFs of TFIID, is indeed found in yeast (Kuras *et al.*, 2000; Li *et al.*, 2000) and can be biochemically separated from TAFs during *in vitro* fractionation of yeast TFIID (Sanders *et al.*, 2002a). Although TFIID is the predominant form found in mammalian cells, the free form of TBP may transiently exist under some stressed conditions (Wu *et al.*, 1998) and may further associate with other cellular proteins to form distinct TBP complexes. Clearly, activator-regulated steps of transcription complex assembly are likely to be different,

M. C. Thomas and C.-M. Chiangdepending on whether TBP or TFIID is used as the  
TATA-binding factor.

### TAF-Dependent Activation

In TATA-less promoters where TAFs are needed  
for promoter recognition, it is plausible that the  
interactions between activators and TAFs are involved  
in the recruitment of TFIID to the Inr, DCE, and DPE  
promoter elements characterized in higher eukaryotes,  
hence modulating the level of transcription. Indeed,  
specific DPE has been shown to be essential for  
the function of some transcriptional activators in  
*Drosophila* embryos (Butler and Kadonaga, 2001),  
likely reflecting unique contacts between gene-specific  
activators and distinct components of TFIID that  
allosterically modulate TAF6-TAF9 recognition of  
the DPE. In addition, specific TFIID contacts with  
additional promoter sequences can be induced by the  
presence of activators, as illustrated by ATF-induced  
extension of TFIID footprinting to the downstream  
region of the adenovirus E4 promoter (Horikoshi  
*et al.*, 1988). Obviously, activators may facilitate TFIID  
binding to the core promoter region, which is often  
the rate-limiting step for the assembly of a functional  
transcription complex in both TATA-containing and  
TATA-less promoters, as documented by many DNA-  
binding and transcription studies (see Wu and Chiang,  
2001a, and references therein). Normally, TFIID binds  
less efficiently to the TATA box compared to that  
of TBP, due to TAF1-mediated inhibition of TBP-  
promoter contacts. Interestingly, TAFs also contact  
several GTFs, including TFIIA (Yokomori *et al.*, 1993),  
TFIIB (Goodrich *et al.*, 1993), TFIIE (Hisatake *et al.*,  
1995), the RAP74 subunit of TFIIF (Ruppert and Tjian,  
1995), and both the RPB1 and RPB2 subunits of pol II  
(Wu and Chiang, 2001a). These TAF-mediated protein-  
protein interactions facilitate the entry of the remaining  
GTFs and pol II through either the sequential assembly  
pathway or the preassembled pol II holoenzyme  
pathway, thereby shifting the rate-limiting step for  
PIC assembly from the entry of downstream factors to  
promoter recognition by TFIID (Figure 6, top).

### TAF-Independent Activation

The concept that TAFs are not absolutely required  
for pol II-dependent transcription has been supported  
by both yeast genetic analysis (Apone *et al.*, 1996;  
Moqtaderi *et al.*, 1996; Walker *et al.*, 1996) and biochem-  

### The General Transcription Machinery and General Cofactors  
TAF-Dependent Activation: Promoter Recognition

TFIID  
(TAFs)  
Activator  

Inr  
MTE DPE  
DCE  

RNA Polymerase II  
TFIIE  
TFIIB  
TFIIF  
TFIIH  

---

TAF-Independent Activation: Pol II Entry  

TBP  
Activator  

TATA  
Inr  

RNA Polymerase II  
TFIIE  
TFIIB  
TFIIF  
TFIIH  

---

FIGURE 6 TAF-dependent (i.e., TFIID-mediated) vs. TAF-independent (i.e., TBP-mediated) activation. Promoter recognition by TFIID is usually the rate-limiting step (thick arrow; upper panel) facilitated by transcriptional activators. In contrast, when TBP is used as the TATA-binding factor, pol II entry often becomes the rate-limiting step (thick arrow; lower panel) facilitated by activators. For simplicity, a preassembled pol II holoenzyme complex is depicted. (Adapted from Wu and Chiang, 2001a)

activity of TFIIA and TFIIH (Wu et al., 1998), during the activation process. These general cofactors are capable of functionally replacing TAFs in conveying regulatory signals between activators and the general transcription machinery, although the precise mechanisms have not yet been elucidated.

---

**VARIANTS OF TBP AND TAFs**

Some components of TFIID, including TBP and TAFs, have gene paralogs (i.e., related proteins encoded by different genes belonging to the same family) often expressed in a tissue- and development-specific

---

**TBP-Related Factors**

All multicellular organisms contain at least two genes coding for TBP family members that share sequence homology at their C-terminal 180 amino acid core DNA-binding domains. While the first gene encodes

M. C. Thomas and C.-M. Chiang
TBP itself, considered a universal TATA-binding transcription factor present in all eukaryotes and archaea, the second gene encodes TBP-related factor 2 (TRF2), also called TBP-related protein (TRP), TBP-like factor (TLF) or TBP-like protein (TLP). Although TRF2 shares ~60% sequence homology and 41% identity within the C-terminal core with that of TBP, it recognizes a sequence element distinct from the TATA box due to changes in some conserved amino acids, including substitutions of three out of four phenylalanine residues that kink the TATA box (Dantonel et al., 1999; Berk, 2000; Hochheimer and Tjian, 2003). However, amino acid residues important for interactions with TFIIA and TFIIB are mostly unaltered, allowing TRF2 to associate with TFIIA and TFIIB (Maldonado, 1999; Moore et al., 1999; Rabenstein et al., 1999; Teichmann et al., 1999) and thus assembling into a functional PIC able to transcribe some TATA-less promoters via an undefined TRF2-binding element (Ohbayashi et al., 2003; Chong et al., 2005). This property of TRF2 also enables it to function as a repressor, preventing PIC formation initiated by TBP or TFIID on TATA-containing promoters (Moore et al., 1999; Teichmann et al., 1999; Chong et al., 2005), likely due to competition for limiting amounts of TFIIA and TFIIB. Not surprisingly, inactivation of TRF2 in *C. elegans* (Dantonel et al., 2000; Kaltenbach et al., 2000) and *Xenopus laevis* (Veenstra et al., 2000) by RNA interference fails to support embryogenesis, indicating that TRF2 is essential for normal cell development as expected from its dual role in pol II-dependent transcription. In contrast to embryonic lethality seen with *C. elegans* and *X. laevis*, TRF2-null mice are perfectly viable and only exhibit defects in spermiogenesis, suggesting that the biological role of TRF2 is species-specific (Martianov et al., 2001; Zhang et al., 2001).

Other than TRF2, three species-specific TRFs (TRF1, TRF3, and TRF4) have also been identified. TRF1 (Crowley et al., 1993), so far only identified in neuronal and germ cells of *Drosophila* (reviewed in Berk, 2000; Hochheimer and Tjian, 2003), exhibits 63% amino acid sequence identity to TBP at its C-terminal DNA-binding core which maintains most of the conserved residues important for interactions with the TATA box, TFIIA and TFIIB (Rabenstein et al., 1999). It is thus expected that TRF1 binds TFIIA and TFIIB and can partially substitute for TBP in directing pol II-dependent transcription from some TATA-containing promoters in vitro (Hansen et al., 1997). Interestingly, a TRF1 target gene, *Tudor*, whose expression is driven by two tandem promoters containing a TRF1-responsive upstream promoter and a TBP/TFIID-responsive downstream promoter, has been identified in *Drosophila* cells. A TC-rich sequence, located between –22 and –33 relative to the transcription start site of this upstream TRF1-responsive *Tudor* promoter preferentially nucleates TRF1-mediated PIC assembly and transcription. In vivo, promoter selectivity by TRF1 is likely enhanced by some transcriptional activators or by its association with neuron-specific TRF1-associated factors (nTAFs) to form a multiprotein complex, which is distinct from TFIID (Holmes and Tjian, 2000). That TRF1 does not interact with TFIID-specific TAFs provides a rationale why TRF1 and TBP are not interchangeable at respective TRF1-responsive upstream and TBP-responsive downstream *Tudor* promoters (Holmes and Tjian, 2000). While TRF1 clearly plays an important role in pol II-mediated transcription, the majority of cellular TRF1 is in fact associated with the TFIIB-related factor BRF, which is a component of TFIIIB involved in pol III-dependent transcription (Takada et al., 2000). Similar to the pol II system, only addition of the TRF1-BRF complex, but not TBP-BRF, could restore pol III-dependent transcription of the tRNA, 5S RNA and U6 RNA genes in TRF1-depleted extracts (Takada et al., 2000).

TRF3, which shows 93% amino acid sequence identity to TBP at the C-terminal core region and has the same sequence as TBP at all of the conserved residues involved in TATA binding and interactions with TFIIA and TFIIB, is unique to vertebrates, ranging from fish to humans, but is not present in urochordate *Ciona intestinalis* and lower eukaryotes, such as *Drosophila* and *C. elegans* (Persengiev et al., 2003). Unlike TRF1 and TRF2, which are expressed only in selective tissues, TRF3 is ubiquitously expressed in every cell type examined, similar to that of TBP, and is present as a protein complex with a molecular size of approximately 200 kDa. Since TAF1 does not co-fractionate with the TRF3 complex, it is likely that polypeptides constituting the TRF3 complex are different from TAFs defined in TFIID and nTAFs associated with TRF1. The transcriptional properties of TRF3 and its regulated genes remain to be characterized.

TRF4, the only TBP-related protein found in the genomes of the unicellular human parasite *Trypanosoma brucei* (Tb) and several Tb-related family members where DNA sequences corresponding to bona fide TBP cannot be identified, displays only 31% amino acid sequence identity to *Drosophila* TBP at the C-terminal

The General Transcription Machinery and General Cofactorscore domain (Ruan *et al.*, 2004). Since amino acid residues critical for TFIIA and TFIIB interactions, but not the phenylalanine residues involved in TATA recognition, are still conserved in TRF4, it is likely that TRF4 may interact with TFIIA but fails to bind the TATA sequence. Indeed, a six-subunit complex containing the ~30 kDa TRF4 protein, three small nuclear RNA-activating complex (SNAPc) subunits (identified as TbSNAP50, TbSNAP2, TbSNAP3), the 25 kDa Tb ortholog (TbTFIIA-2) of the small TFIIA subunit γ, and a 75 kDa protein exhibiting weak sequence homology to the unprocessed αβ subunits of TFIIA has been isolated. This TRF4 complex is able to recognize the upstream sequence element (USE) of the TATA-less spliced leader (SL) RNA gene and drives transcription of the SL RNA that is critical for mRNA maturation in Trypanosomatid parasites (Schimanski *et al.*, 2005). The finding that TRF4 is universally required for transcription by all three nuclear RNA polymerases suggests that TRF4 may functionally replace the classically defined TBP in these human parasites. However, it is yet to be resolved how transcription can proceed in these organisms lacking the genes coding for the other GTFs, such as TFIIB, TFIIF, and TFIIE.

In contrast to the functionally characterized TRFs present in eukaryotes and in Trypanosomatid parasites described above, very little is known about TBP-related family proteins in archaea despite the fact that multiple genes potentially encoding TBPs are found in several sequenced archaeal genomes. It was reported that the archaeon *Halobacterium* species NRC-1 has six predicted genes (*Ng et al.*, 2000b) that code for TBP family proteins (TBPa, TBPb, TPBc, TBPd, TBPe, and TBPf). Except TBPa, the other five members all have two direct repeats forming a saddle-like structure with two side stirs, as found in the C-terminal region of their eukaryotic counterparts (*Baliga et al.*, 2000). TBPa has only one direct repeat and is thus unlikely to bind the TATA box as a monomer. Similar findings of multiple TBP family genes have also been described in other archaea, including *Methanosarcina acetivorans* (*Galagan et al.*, 2002) and *Haloferax volcanii* (*Thompson et al.*, 1999). The identification of a TBP gene family in archaea, similar to the discovery of multiple σ factors in bacteria (*Borukhov and Nudler*, 2003; *Gruber and Gross*, 2003), raises an interesting possibility that a combination of multiple TBPs and TFBs (the archaeal counterpart of TFIIB) is employed with its single RNA polymerase to generate diverse transcription complexes for recognizing different

---

ent archaeal gene promoters. This is certainly an open area waiting to be explored.

### TAF-Containing Complexes

Similar to TBP that forms SL1, TFIID, TFIIB, and TAC complexes, some TAF components in TFIID are also present in distinct complexes such as TBP-free TAFII-containing complex (TFTC), TFTC-related PCAF/GCN5 complexes, Spt-Ada-Gcn5 acetyltransferase (SAGA), SAGA-like complex (SLIK; also named “SALSA” for SAGA altered, Spt8 absent), Spt3-TAF9-GCN5L (*i.e.*, the long form of GCN5) acetylase (STAGA), and polycomb repressive complex 1 (PRC1). These TBP-lacking TAF-containing complexes are involved in diverse aspects of pol II-dependent transcription. Except PRC1 whose role is mainly involved in gene silencing, the other TAF-containing complexes are mostly implicated in activator-dependent transcription likely due to the HAT activity inherent to each complex. TFTC, originally identified in HeLa cells using a monoclonal antibody against human TAF10 (*Wieczorek et al.*, 1998), contains TAF2, TAF5, TAF5L, TAF6L, TAF7, and some histone fold-containing TAFs, including TAF4, TAF6, TAF9, TAF10, and TAF12. In addition, TFTC has TRRAP (transformation-transactivation domain-associated protein), SAP130 (spliceosome-associated protein 130), GCN5LHAT enzyme, Ada3 adaptor protein, and histone fold-containing Spt3 (*Cavusoglu et al.*, 2003), TAF9L (also called TAF9b; *Frontini et al.*, 2005), and ataxin-7 (*Helm-linger et al.*, 2004). The three-dimensional structures of TFTC and TFIID, both resolved at 35 Å resolution by electron microscopy and single-particle image analysis, resemble a macromolecular clamp consisting of five globular domains (for TFTC) or three lobes (for TFIID) organized around a solvent-accessible groove that may accommodate a DNA duplex (*Andel et al.*, 1999; *Brand et al.*, 1999a). This configuration suggests that TFTC may adapt a DNA-binding conformation similar to that exhibited by TFIID. Indeed, TFTC is able to support basal and Gal4-VP16-mediated transcription *in vitro* from TATA-containing adenovirus major late and rabbit β-globin promoters and also basal transcription from the TATA-less transcription enhancer factor-1 (TEF-1) promoter, presumably via TAF recognition of the core promoter and interaction with other components of the general transcription machinery (*Wieczorek et al.*, 1998). The coactivator function of TFTC is in part mediated

---

M. C. Thomas and C.-M. Chiang

---
by direct protein-protein contacts between activators and multiple subunits of TFTC, including TAF 5, TAF 6, TAF 10, SAP 130, Spt 3, GCN5L, TRRAP, and ataxin-7 (Hardy et al., 2002; Yanagisawa et al., 2002; Helmlinger et al., 2004; Palhan et al., 2005). The TRRAP component of TFTC seems to act as a direct target for several liganded nuclear hormone receptors, including ERα, ERβ, VDR, and PPARγ (Yanagisawa et al., 2002), as observed in the SAGA complex (see below). This activator-TFTC interaction may also facilitate p300-mediated transcription from a Gal4-VP16-dependent chromatin template (Hardy et al., 2002), probably resulting from preferential acetylation of nucleosomal core histone H3 by the GCN5L component of the TFTC complex (Brand et al., 1999b). Besides the HAT component, the presence of many histone fold-containing proteins, including Spt 3, TAF 4, TAF 6, TAF 9, TAF 10, and TAF 12, further contributes to the integrity of the TFTC complex. Recently, ataxin-7, the human homolog of yeast Sgf73 that is an integral component of yeast SAGA and SLIK complexes (see below), has also been shown to be essential for the structural integrity of TFTC and its related STAGA complex (Helmlinger et al., 2004). Both wild-type and mutant ataxin-7, which contains polyglutamine expansion at its N-terminus and is linked to spinocerebellar ataxia type 7 (SCA7) neurodegenerative disorder, can be incorporated into TFTC and STAGA complexes whose structural components are largely unaltered (Helmlinger et al., 2004; McMahon et al., 2005; Palhan et al., 2005). Presumably, incorporation of either form of ataxin-7 will alter the functional properties of these HAT complexes (see below).

Similar to TFTC, the yeast SAGA also lacks TBP but contains some common subunits, such as Tra1 (the yeast homolog of human TRRAP), adaptor protein Ada 3, GCN 5 (short form), TAF 5, and histone fold-containing proteins Spt 3, TAF 6, TAF 9, TAF 10, and TAF 12. The unique components present in SAGA, but not yet documented in TFTC, include Ada 1, Ada 2, Spt 7, Spt 8, Spt 20 (also called Ada 5), Chd 1 (for chromo-ATPase/helicase-DNA binding domain 1), Ubp 8 (for ubiquitin-specific processing protease 8), Sgf 11 (for SAGA-associated factor 11 kDa), Sgf 29, and Sgf 73 (Grant et al., 1998; Sanders et al., 2002b; Henry et al., 2003; Daniel et al., 2004; Powell et al., 2004; Ingvarsdottir et al., 2005; Lee et al., 2005c; McMahon et al., 2005; Pray-Grant et al., 2005). Five major functions attributed to SAGA include: activator interaction, contacting TBP, recognition of methylated

histone H3, HAT activity, and deubiquitinating activity. These SAGA components are organized into five distinct domains with an overall shape analogous to the structure of human TFTC, as resolved by electron microscopy at 31 Å resolution (Wu et al., 2004). The largest subunit of SAGA, Tra1, is located in the outermost globular domain able to contact transcriptional activators, as evidenced by the fluorescence resonance energy transfer (FRET) technique measuring direct in vivo association between yeast Gal4 acidic activator and Tra1 (Bhaumik et al., 2004). This protein-protein interaction was also demonstrated by in vitro crosslinking experiments, in which the Tra1 subunit of SAGA was shown to serve as a direct target for both Gal4 and GCN4 transcriptional activator (Fishburn et al., 2005; Reeves and Hahn, 2005). The illustration of Tra1 as a “true” target for transcriptional activators in *S. cerevisiae* is also consistent with the finding that human TRRAP, which shows 27.3% amino acid identity and 58.9% similarity with its yeast homolog, also interacts with c-Myc and E2F-1 oncoproteins implicated in mammalian transformation and transactivation (McMahon et al., 1998). The c-Myc-recruited TRRAP apparently associates with GCN5HAT activity that is important for c-Myc-induced cellular transformation (McMahon et al., 2000). However, whether recruitment of TRRAP and GCN 5 occurs in the context of human SAGA-like complexes, such as TFTC, PCAF and GCN 5 complexes (Ogryzko et al., 1998), remains to be investigated. Interestingly, approximately 10% of yeast genes, which are mostly driven by TATA-containing promoters and which are stress-induced, appear to be SAGA-dependent (Lee et al., 2000; Huisenga and Pugh, 2004). These TFIID-independent yeast promoters are recognized by the free form of TBP that works in conjunction with SAGA at the targeted promoters via direct interaction between TBP and the Spt 3 and Spt 8 subunits of SAGA (Belotserkovskaya et al., 2000; Larschan and Winston, 2001; Bhaumik and Green, 2002; Warfield et al., 2004). The recruitment of SAGA by transcriptional activators likely enhances TBP binding to the promoter region, as found in the yeast PHO5 promoter (Barbaric et al., 2003), or increases factor accessibility by GCN 5-mediated acetylation of nucleosomal core histones at the promoter region (Grant et al., 1997), thus leading to gene activation. Moreover, it was recently found that the 19S regulatory subcomplex of the 26S proteasome is able to enhance activator-dependent recruitment of the yeast SAGA

The General Transcription Machinery and General Cofactors

complex, apparently in an ATPase-dependent manner,
by stimulating SAGA's nonspecific DNA-binding and
HAT activities (Lee et al., 2005a). These effects may
further contribute to the coactivating function of the
SAGA complex. Interestingly, SAGA can also negatively
regulate basal transcription, in the absence of
activator, by either inhibiting TBP binding to the HIS3
and TRP3TATA boxes (Belotserkovskaya et al., 2000)
or by competing with TFIIA for TBP binding on the
HIS4 promoter (Warfield et al., 2004). This inhibitory
effect of SAGA on TBP binding to the HIS3 and TRP3
genes is also mediated by the Spt3 and Spt8 subunits of
SAGA. Clearly, SAGA plays a dual role in transcription
as typically exemplified by a general cofactor. Other
than serving as a bridging factor for interactions
between activators and TBP or HAT substrates, some
components of the SAGA complex, such as Spt7, Spt20,
Ada1, TAF5, TAF10, TAF12, and Sgf73 are necessary
for the structural integrity of SAGA, as mutations in
these subunits result in disruption of the holo complex
(Grant et al., 1998; Sterner et al., 1999; Durso et al.,
2001; Kirschner et al., 2002; McMahon et al., 2005). In
addition, Sgf73 is important for regulating GCN5HAT
activity in SAGA, as substitution of wild-type yeast protein with a polyglutamine-expanded pathogenic human
Sgf73 mutant impairs GCN5-mediated acetylation of
nucleosomal histones likely by reducing interactions
between Ada2, Ada3, Spt3 and TAF12 with the core
complex without altering the incorporation of GCN5
and other subunits (McMahon et al., 2005). The Ubp8
and Sgf11 subunits of SAGA seem to function together
as nonessential structural components to remove the
monoubiquitin moiety from lysine 123 of histone H2B
(Henry et al., 2003; Daniel et al., 2004; Ingvarsdottir
et al., 2005; Lee et al., 2005c), since deletion of either
the UBP8 or SGF11 gene in yeast results in similar
effects on a common subset of SAGA-regulated genes
(Powell et al., 2004; Ingvarsdottir, 2005) and increased
ubiquitination on histone H2B both in vivo and in vitro
(Ingvardsdottir, 2005; Lee et al., 2005c). At the chromatin level, the recent identification of Chd1 as a
component of SAGA further provides a functional link
between histone methylation and acetylation, given
that Chd1 binding to methylated lysine 4 in histone
H3 apparently enhances GCN5-mediated acetylation
on chromatin (Pray-Grant et al., 2005).
SLIK, with protein composition similar to that
of SAGA, contains a SLIK-specific subunit Rtg2
(retrograde 2) and a C-terminal truncated Spt7 pro-
tein, but lacks Spt8 (Pray-Grant et al., 2002; Sterner
et al., 2002). The remaining polypeptides common to
both SAGA and SLIK complexes include GCN5, Ada1,
Ada2, Ada3, TAF5, histone fold-containing TAF6,
TAF9, TAF10 and TAF12, as well as Tra1, Spt3, Spt20,
Chd1, Ubp8, Sgf11, Sgf29, and Sgf73 (Pray-Grant et al.,
2002; Daniel et al., 2004; McMahon et al., 2005; Pray-
Grant et al., 2005). It should be noted that TAF5 appears to be a major scaffold protein connecting different structural domains in both SAGA and TFIID
complexes (Leurent et al., 2004; Wu et al., 2004) and
likely SLIK as well. In addition, TAF5 can be post-
translationally modified by ubiquitination (Auty et al.,
2004) and sumoylation (Boyer-Guittaut et al., 2005),
suggesting a critical role of this subunit in regulating
the function of TFIID, SAGA, SLIK and other TAF5-
containing complexes. With many common subunits
as found in SAGA, it is not surprising that SLIK can
interact with activators and TBP, and can regulate chromatin transcription by binding methylated histone H3,
acetylating histones, and deubiquitinating histone H2B
(Pray-Grant et al., 2002; Henry et al., 2003; Daniel et al.,
2004; Jazwinski, 2005). As seen with SAGA, the 19S regulatory subcomplex also enhances the affinity of SLIK
for activator-bound DNA (Lee et al., 2005c). However,
there are some functional differences between SLIK
and SAGA, due to a slight variation in their protein
composition. In yeast, deletion of the Rtg2 gene results in the disruption of SLIK but not SAGA, indicating that Rtg2 is indeed a SLIK-specific subunit (Pray-
Grant et al., 2002). Moreover, that Rtg2, rather than
Spt8, is involved in CIT2 gene activation that regulates
biosynthetic and metabolic responses in mitochondria
(Chelstowska and Butow, 1995) suggests that the presence of Rtg2 enables SLIK to regulate a subset of genes
distinct from that of SAGA. This is consistent with the
observation that SLIK does not repress HIS3 basal transcription, presumably due to the absence of Spt8 caused
by truncation of the Spt7 C-terminal region needed for
Spt8 recruitment (Belotserkovskaya et al., 2000; Sterner
et al., 2002; Wu and Winston, 2002).
STAGA, a human counterpart of yeast SAGA initially isolated from HeLa nuclear extracts using polyclonal antibodies against human TAF9 (Martinez et al.,
1998), contains many homolog found in yeast SAGA,
including TRRAP, Ada1, Ada2, Ada3, Spt3, Spt7, and
histone fold-containing TAF9, TAF10, and TAF12.
While additional components of STAGA, TAF5L,
TAF6L, SAP130, ataxin-7 and GCN5L are also present

M. C. Thomas and C.-M. Chiang

in TFTC, the identities of the other subunits (such as STAF 36, STAF 42, STAF 46, STAF 55, STAF 60, and STAF 65γ) remain to be characterized (Martinez et al., 2001; Helmlinger et al., 2004; Palhan et al., 2005). Pertinent to this, whether many of the STAGA components are commonly found in the related human PCAF/GCN 5 complexes (Ogryzko et al., 1998) remains to be examined. The conservation of histone fold-containing Spt 3, TAF 9, TAF 10, and TAF 12 in both SAGA and STAGA suggests that these subunits are likewise important for the structural integrity of these distinct HAT complexes. As seen with yeast SAGA, the existence of TRRAP, GCN5L, Ada 1, Ada 2, Ada 3, and ataxin-7 in STAGA likely accounts for the coactivating activity of STAGA in supporting Gal 4-VP 16-mediated activation from a Gal 4-driven chromatin template (Martinez et al., 2001) and cone-rod homeobox (CRX)-stimulated transcription from the CRX-targeted rhodopsin promoter (Palhan et al., 2005). Clearly, Ada proteins and ataxin-7 facilitate the access of GCN5L to the activator-occupied chromatin template, allowing acetylation of nucleosomal core histones (Balasubramanian et al., 2002; Palhan et al., 2005). That adaptor proteins are additionally required for activator-facilitated chromatin targeting of GCN 5 is further supported by the observation that recombinant GCN 5 protein is unable to potentiate activator-dependent transcription from preassembled chromatin templates (Thomas and Chiang, 2005). From our understanding of the yeast SAGA complex, it is likely that the Spt 3 component of STAGA may contact TBP to facilitate transcription from STAGA-dependent promoters.

Unlike HAT-containing TFTC, SAGA, SLIK, and STAGA complexes involved in gene activation, the TAF-containing Drosophila PRC 1 complex is implicated in repression of homeotic genes that govern body segmentation and the developmental process. This complex, initially isolated from Drosophila embryos using a monoclonal antibody against epitope-tagged polyhomeotic (PH) or posterior sex comb (PSC) protein (Shao et al., 1999), contains approximately 30 subunits, including TAF 1, TAF 4, TAF 5, TAF 6, TAF 9, TAF 11, PH, PSC, PC (polycomb), RING 1, Zeste, HSC 4, SMRTER, Mi-2, Sin3A, Rpd 3, p 55, Sbf 1, DRE 4/Spt 16, p 90, HSC 3, Modulo, Reptin, DNA topoisomerase II, p 110, tubulin, actin, Ribosome RS 2, and Ribosome RL 10 (Saurin et al., 2001). Although this holo complex has histone deacetylase (HDAC) Rpd 3, chromatin remodeling ATPase Mi-2, and other corepressor components (SMRTER, Sin3A, and p 55) likely contributing to repression of transcription and inhibition of chromatin remodeling, it is surprising to see that a PRC 1 core complex (PCC) containing only PH, PSC, PC, and RING 1 is sufficient for transcriptional silencing (King et al., 2002) and for blocking SWI/SNF-mediated mobilization of a nucleosomal array (Francis et al., 2001). This finding, plus the fact that a sequence-specific transcription factor Zeste is present in PRC 1, suggests that PRC 1 may use different mechanisms to target PRC 1-regulated gene transcription, irrespective of the presence or absence of a Zeste-binding element (Mulholland et al., 2003). Given that the identified human PRC 1 complex containing only homologs of the Drosophila core complex (PC, PH, PSC, and RING 1) is able to inhibit SWI/SNF-mediated chromatin remodeling (Levine et al., 2002) and that the other characterized Drosophila PRC 2 (Ng et al., 2000a; Tie et al., 2001) and human PRC 2 (Cao et al., 2002; Kuzmichev et al., 2002), PRC 3 (Kuzmichev et al., 2004), and PRC 4 (Kuzmichev et al., 2005) complexes do not seem to contain TAFs, the functional roles of TAFs and the other subunits constituting PRC 1 await further investigation.

### TAF Variants

Some components of TFIID are present in a substoichiometric ratio relative to the other TAFs. These TAFs, such as TAF4b, TAF5L, and TAF7L, are often found in a tissue-specific manner and likely confers TFIID unique properties functioning in a specialized environment. TAF4b, a paralog of TAF 4 initially identified in TFIID purified from B cells and later found expressed at low levels in every cell types but specifically enriched in the testes and ovary, appears to function in a gonad-specific manner, as knockout of this TAF variant severely affects ovarian development in female mice (Freiman et al., 2001) and also spermatogenesis in male mice (Falender et al., 2005). Likewise, TAF5L and TAF7L, paralogs of TAF 5 and TAF 7, respectively, are implicated in male gametogenesis (Hiller et al., 2001; Pointud et al., 2003). The presence of these tissue-specific TAF variants likely enables TFIID to work in conjunction with germ cell-specific transcription factors, cofactors, or other components of the general transcription machinery, such as TFIIAαβ-like factor (Upadhyaya et al., 1999; Ozer et al., 2000). Related to this, a unique TFIID subunit, TAF 8, is induced during adipocyte differentiation (Guermah et al., 2003). Other than the tissue-specific expression

The General Transcription Machinery and General Cofactors

pattern observed with these TAF variants, functional in-
activation of TAF1 (Hisatake et al., 1993; Ruppert et al., 
1993), which is present in TFIID and PRC1, and of 
TAF10 (Metzger et al., 1999), found in TFTC, STAGA, 
and SAGA-like complexes, causes cell cycle arrest, in-
dicating a general role of selective mammalian TAFs in 
modulating cell growth. Undoubtedly, the presence of 
TAF variants further expands the general properties of 
TFIID to specialized needs in differentiated tissues.

### TFIIA

#### Protein Composition
Human TFIIA is composed of α, β, and γ subunits 
with molecular weights of 35 kDa, 19 kDa, and 12 kDa, 
respectively. These three subunits are encoded by two 
genes: TFIIAαβ and TFIIAγ in higher eukaryotes, and 
TOA1 and TOA2 in yeast. Human TFIIAαβ encodes a 
55-kDa precursor protein that is highly conserved with 
TOA1 in yeast within the N-terminal 54 amino acids 
and the C-terminal 76 amino acids, but is less conserved 
in the central part of TFIIA which has been shown to 
be dispensable for function (Ranish et al., 1992; Kang 
et al., 1995). The smallest subunit of TFIIA, TFIIAγ, 
is homologous to yeast TOA2 (Ozer et al., 1994; Sun 
et al., 1994). In higher eukaryotes, TFIIAαβ is prote-
olytically cleaved into TFIIAα and TFIIAβ subunits 
(DeJong and Roeder, 1993; Ma et al., 1993; Yokomori 
et al., 1993). Originally it was thought that the three in-
dependent subunits (α, β, and γ) constitute TFIIA ac-
tivity, since only the cleavage products were detected 
in cell extracts in association with one another. Re-
cently, intact TFIIAαβ protein was detected in asso-
ciation with TFIIAγ along with TBP in P19 embry-
onal carcinoma cells (Mitsiou and Stunnenberg, 2000, 
2003), and likely in human parasites as a more divergent 
form such as the TRF4/SNAPc/TFIIA complex identi-
fied in *Trypanosoma brucei* (Schimanski et al., 2005). The 
site for proteolytic cleavage within human TFIIAαβ 
was determined by Edman degradation; TFIIAβ was 
found to start at Asp278 (Høiby et al., 2004). Further-
more, cleaved TFIIAα and TFIIAβ were more effi-
ciently degraded than the unprocessed precursor via the 
ubiquitin-proteasome pathway, suggesting that cleavage 
and degradation of TFIIA control the level of TFIIA 
within the cell to perhaps adapt more rapidly to the 
transcriptional needs in responding to environmental 
changes (Høiby et al., 2004). Similar to TAF4b, a cell 
type-specific TFIIAαβ-like factor (ALF) has been iden-

**TFIIA as an Antirepressor, Not a GTF**

The role of TFIIA as a GTF has been controversial. As 
with TFIID, TFIIA was initially identified as a phospho-
cellulose column fraction necessary for pol II-mediated 
transcription *in vitro* (Matsui et al., 1980). Early *in vitro* 
experiments showed that TFIIA was essential for tran-
scription (Reinberg et al., 1987), while later *in vitro* stud-
ies demonstrated that TFIIA was largely dispensable for 
basal level transcription (Van Dyke et al., 1988; Wu et al., 
1998). It has also been suggested that TFIIA stimulates 
both basal and activated transcription *in vitro* two-to-
tenfold, but generally only when TFIID, instead of TBP, 
is used as the promoter-binding factor (Orphanides 
et al., 1996; Hampsey, 1998; Warfield et al., 2004). 
In our laboratory, we found TFIIA was not required 
for either basal or activator-dependent transcription 
in a highly purified transcription system reconstituted 
with recombinant TFIIB, TFIIE, TFIIF, PC4 coactiva-
tor, and epitope-tagged multiprotein complexes (pol II, 
TFIID, and TFIIH), irrespective of whether TFIID or 
recombinant TBP was used in the assay (Wu et al., 1998). 
Nevertheless, TFIIA indeed became essential for tran-
scription in a reconstituted system containing partially 
purified fractions obtained according to the purification 
scheme outlined in Figure 1. Collectively, these studies 
suggest that TFIIA mainly functions as an antirepressor 
to overcome inhibitors present in crude fractions, likely 
by increasing the affinity of TBP or TFIID for DNA 
(Buratowski et al., 1989; Lee et al., 1992; Imbalzano et al., 
1994b; Kang et al., 1995), thus enhancing PIC assembly. 
TFIIA stabilizes TBP-TATA box interactions through 
direct contacts with both TBP (on the surface of TBP 
opposite to the TFIIB-binding side) and the DNA se-
quence immediately upstream of the TATA box (Geiger 
et al., 1996; Lagrange et al., 1996; Oelgeschläger et al., 
1996; Tan et al., 1996). Binding of TFIIA to TBP dimers 
has also been shown to induce TBP monomer forma-
tion and accelerate the kinetics of TBP binding to DNA 
(Coleman et al., 1999). TFIIA can counteract the repres-
sive effects of negative cofactors, such as NC2 (Xie et al., 

M. C. Thomas and C.-M. Chiang

2000), HMGB1 (Ge and Roeder, 1994b), and DNA Topoisomerase I (Merino et al., 1993), as well as the inhibitory activity of TAF1 and BTAF1 on TBP binding to DNA (Auble and Hahn, 1993; Kokubo et al., 1998). Interestingly, TAF1 interaction with TFIIA may modulate TFIIA activity, since it has been shown that TAF1 phosphorylates human TFIIAB on serine residues important for TBP binding and transcription activity (Solow et al., 2001). Additionally, experiments have shown that the TFIIA-TBP-DNA complex may also be regulated by the transcriptional coactivator p300 through acetylation of TFIIA (Mitsiou and Stunnenberg, 2003).

### TFIIA as a Coactivator

In addition to the antirepressing activity, TFIIA can also act as a coactivator to stimulate overall transcription by directly contacting activators, general cofactors, and components of the general transcription machinery. Many studies have demonstrated that TFIIA interacts with activators, such as Gal4-VP16 (Kobayashi et al., 1995; Dion and Coulombe, 2003), Zta (Ozer et al., 1994; Kobayashi et al., 1995), and HTLV-1 Tax (Clemens et al., 1996), or with transcriptional cofactors such as PC4 (Ge and Roeder, 1994a) and HMG2 (Shykind et al., 1995). TFIIA has also been shown to interact with components of TFIID, including TBP (Maldonado et al., 1990; Ranish and Hahn, 1991; Usuda et al., 1991; Cortes et al., 1992; Lee et al., 1992; Yokomori et al., 1993; Sun et al., 1994), TAF1 (Solow et al., 2001), TAF4 (Yokomori et al., 1993), TAF11 (Kraemer et al., 2001; Robinson et al., 2005), and other GTFs, such as TFIIIEα, TFIIIEβ, and RAP74 (Langelier et al., 2001). The observed interaction between TFIIA and TBP-related proteins, including TRF1 (Hansen et al., 1997), TRF2 (Moore et al., 1999; Rabenstein et al., 1999; Teichmann et al., 1999), and TRF4 (Schimanski et al., 2005), further suggests that TFIIA plays a more general role in facilitating PIC assembly mediated by different core promoter-binding factors. In one proposed model of TFIIA function, interaction of TFIIA with both activators and TFIID may stimulate and stabilize TFIID binding to DNA as part of an activator-TFIID-TFIIA-DNA complex, thereby enhancing a rate-limiting step for promoter recognition (Wang et al., 1992; Lieberman and Berk, 1994; Chi et al., 1995; Kobayashi et al., 1995; Ranish et al., 1999; Dion and Coulombe, 2003). Moreover, TFIIA may upregulate PIC formation at a step post TFIID binding to the DNA by stimulating the functions of both TFIIIE

and TFIIF (Langelier et al., 2001). In TAF-independent transcriptional activation experiments using a highly purified transcription system, TFIIA could potentiate TBP-mediated activation, suggesting that TFIIA may function as a coactivator especially in the absence of TAFs (Wu et al., 1998). Several studies using immunodepletion of TFIIA subunits *in vitro* and mutational studies abolishing TFIIA-TBP interactions in yeast have indicated that, similar to TAFs, TFIIA is important for transcription only from a subset of genes and does not seem to be universally required for all gene transcription (Kang et al., 1995; Ozer et al., 1998a; Liu et al., 1999; Stargell et al., 2000).

### TFIIB

#### TFIIB Stabilizes TFIID Promoter Binding

Once TFIID or TBP is bound to the promoter in the absence or presence of TFIIA, TFIIB is the next GTF to enter the PIC assembly pathway. Binding of TFIIB to promoter-bound TBP results in a more stable ternary complex composed of TBP-TFIIB-DNA (Orphanides et al., 1996). Besides stabilizing TFIID/TBP binding to the promoter region, TFIIB plays an important role in recruiting pol II/TFIIF to the ternary complex and in specifying the transcription start site (Orphanides et al., 1996; Hampsey, 1998; Hahn, 2004). In humans, TFIIB has 316 amino acid residues and exists as a single 33-kDa polypeptide (Ha et al., 1991; Malik et al., 1991), which shares sequence homology with a 38-kDa *Drosophila* TFIIB protein and also with a 38-kDa yeast protein encoded by the *SUA7* gene (Pinto et al., 1992; Wampler and Kadonaga, 1992; Yamashita et al., 1992). TFIIB is evolutionarily conserved among various species, and exhibits amino acid and structural conservation with archael TFB (transcription factor B) and eukaryotic pol III accessory factor BRF (TFIIB-related factor) at both the N-terminal pol II/TFIIF-interacting zinc-ribbon domain and the C-terminal DNA-binding domain containing two imperfect direct repeats (Orphanides et al., 1996; Hampsey, 1998, Bell and Jackson, 2001; Hahn, 2004; see Figure 7A). These two functional domains in TFIIB were originally identified through protease digestion, as the N-terminus is rapidly degraded leaving behind a protease-resistant C-terminal “core” that retains the two imperfect direct repeats of TFIIB (Barberis et al., 1993; Malik et al., 1993).

The General Transcription Machinery and General Cofactors

A

TFIIB

Zn²⁺-Ribbon
(C₃H₁)
1 14
B-Finger
42 44
1st Repeat
75
(BH1-BH5)
124
2nd Repeat
200
(BH1'-BH5')
218
316
pol II (RPB1/RPB2)
RAP30
133
171
BREᵈ Contact
251
294
BREᵘ Contact

B

RAP30

σ homology II (162-249)

σ homology I
107
Winged HTH
170 175
243
249
1
98
RAP74
pol II (RPB5)
Cryptic dsDNA Binding
27
152
TFIIF
TFIIB

C-Terminal Region (358-517)

Acidic Region
179
357
Winged HTH
449
517
RAP74
1
139
TAF1
2
172
RAP30
76
136
TFIIAαβ
410
444
TFIIAαβ
363
444
pol II
436
517
FCP1
175
204
homomeric interaction
405
517
homomeric interaction

C

TFIIEα

Leu Repeat
(38-66)
σ Homology
(13-49)
1
121
Zn²⁺-Finger
(C₂C₂)
157
HTH
159
Ala-Rich
182
Ala-Rich
221
240
Acidic
335
349
378
393
439
12
118
TFIIEB
pol II
158
214
TFIIEB
TFIIH

TFIIE

Central Core (66-146)

Ser-Rich
71
Winged HTH
139
Leu Repeat σ3 Homology
145
Basic HLH
163
Basic HL
193
197
238
258
291
TFIIEB
26
Cryptic dsDNA Binding
Enhances TFIIH Kinase
TFIIEα
pol II
RAP30
ssDNA Binding

FIGURE 7 Structural domains and interacting regions on TFIIB, TFIIF, and TFIIIE. Schematic representation of the structural domains is depicted for TFIIB (A), TFIIF (B), and TFIIIE (C). Solid lines below each protein correspond to the amino acid residues within TFIIB, TFIIF, and TFIIIE regions that are shown to interact with other GTFs and pol II or contact DNA. Unless specified, the boundaries of amino acid residues for the solid lines are the same as indicated for the structural domains. It should be noted that, although not shown, RAP30 in panel B also interacts with TFIIEB, whereas TFIIEα and TFIIEB in panel C contact TFIIAγ and TFIIAαβ, respectively, through undefined regions.

M. C. Thomas and C.-M. Chiang
TFIIB-TBP-Promoter Structure

The C-terminal core of TFIIB comprises nearly two-thirds of the protein (amino acids 106 to 316) and contains two imperfect repeats each consisting of 5 α-helices, termed BH1, BH2, BH3, BH4, and BH5 in the first repeat (amino acids 124 to 200) and BH1′, BH2′, BH3′, BH4′, and BH5′ in the second repeat (amino acids 218 to 294; Bagby et al., 1995; Tsai and Sigler, 2000). The C-terminal core of TFIIB, as revealed in the structure of a TATA-TBP-TFIIB ternary complex (Nikolov et al., 1995; Tsai and Sigler, 2000), bound beneath and to one face of the TATA-TBP complex, consistent with hydroxyl-radical footprinting (Lee and Hahn, 1995) and photocrosslinking experiments (Coulombe et al., 1994; Lagrange et al., 1996). Structural evidence also exists for TFIIB competing with NC2 for overlapping binding surface on TBP (Kamada et al., 2001b; see previous NC2 section). The conserved C-terminus of TFIIB interacts with both TBP and DNA, making DNA contacts with sequences flanking the TATA box, in particular the major groove upstream and the minor groove downstream of the TATA box (Nikolov et al., 1995; Tsai and Sigler, 2000). Experiments also indicate that TFIIB alone without TBP can make sequence-specific DNA contacts with the BREu (Lagrange et al., 1998; see also the Core Promoter Elements section) via a nonconserved helix-turn-helix (HTH) motif, which is a trihelical bundle composed of BH3′, BH4′ and BH5′ in the second direct repeat, and further stabilizes the ternary TFIIB-TBP-promoter complex (Evans et al., 2001; Wolner and Gralla, 2001). Recent studies also show that TFIIB can contact the BREd sequence, situated downstream of the adenovirus E4 TATA box, via a separate DNA-binding domain located at a recognition loop linking BH2 and BH3 in the first direct repeat of TFIIB, apparently in a TBP-dependent manner (Fairley et al., 2002; Deng and Roberts, 2005). These TFIIB-DNA interactions can be modulated by transcriptional activators, such as Gal4-VP16, which work in part by disrupting intramolecular interactions between N- and C-terminal domains of TFIIB (Hawkes et al., 2000), thereby exposing the DNA recognition loop of TFIIB for BREd recognition. Concurrently, this activator-induced conformational change may weaken interactions between the HTH DNA-binding domain of TFIIB and the BREu (Evans et al., 2001). Although the role of activator in disrupting TFIIB-BREu interactions seems contradictory with the

previously proposed role of the BREu in stabilizing the TFIIB-TBP-DNA ternary complex, it appears that the BREu may play an inhibitory role in PIC formation or during the transition from the initiation to the elongation stage. Clearly, activators may stimulate transcription by regulating multiple steps. The precise mechanism by which activators, TFIIB, and BRE interactions affect PIC assembly, initiation, and promoter clearance remains to be investigated.

### TFIIB N-Terminal Domain

The TFIIB amino terminus contains a zinc ribbon motif (amino acids 14 to 42 in humans) with a Cys-X2-His-X15-Cys-X2-Cys sequence forming a protease-resistant ZnC3H1 structure comprising of three antiparallel β strands important for the structural integrity of TFIIB (Chen et al., 2000; Ghosh et al., 2004). This zinc ribbon motif interacts with the RPB1 and RPB2 subunits of pol II at the dock domain near the RNA exit channel (Chen and Hahn, 2003; Bushnell et al., 2004) and also with the RAP30 subunit of TFIIF (Ha et al., 1993; Fang and Burton, 1996; see Figure 7A). These interactions facilitate the recruitment of pol II/TFIIF to the TFIID-bound promoter region (Buratowski and Zhou, 1993; Ha et al., 1993; Malik et al., 1993; Yamashita et al., 1993; Fang and Burton, 1996; Bangur et al., 1997). Immediately adjacent to the N-terminal zinc ribbon motif is a highly conserved region called the charged cluster domain (CCD) or the B-finger, spanning amino acids 44 to 75 of human TFIIB (Bushnell et al., 2004; see Figure 7A). It is believed that the B-finger domain acts as a molecular switch to regulate the conformational change of TFIIB, thereby modulating TFIIB function in promoter recognition, start site selection, and transcriptional activation (Pinto et al., 1994; Pardee et al., 1998; Hawkes and Roberts, 1999; Wu and Hampsey, 1999; Hawkes et al., 2000; Faitar et al., 2001; Elsby and Roberts, 2004).

### B-Finger-Induced Conformational Changes

As mentioned previously, TFIIB undergoes a conformational change when it interacts with DNA or activators, and the B-finger plays a vital role in this conformational change (Hawkes et al., 2000; Fairley et al., 2002; Elsby and Roberts, 2004). The activator Gal4-VP16 has been shown to disrupt the intramolecular interaction between the B-finger and the second repeat of the

The General Transcription Machinery and General CofactorsC-terminal domain (Elsby and Roberts, 2004). Evidence with corresponding mutations in the B-finger, which favors N-and C-terminal intramolecular interactions, also shows defects in activator-mediated recruitment and in transcriptional activation *in vivo* and *in vitro* (Hawkes *et al.*, 2000; Glossop *et al.*, 2004; Elsby and Roberts, 2004). These B-finger mutations, however, are competent in PIC assembly, indicating that mutations in the B-finger do not affect TFIIB interactions with TBP/DNA and pol II/TFIIF, mediated individually by the C-terminal core and the zinc ribbon motif. Related studies in yeast TFIIB also suggest that conformational changes in TFIIB regulate the stability of the TFIIB-TBP-promoter ternary complex (Bangur *et al.*, 1999).

### Role of the B-Finger in Start Site Selection

The involvement of yeast TFIIB (Sua7) in start site selection has been observed on *CYC1*, *ADH1*, and many other genes (Pinto *et al.*, 1992; Berroteran *et al.*, 1994; Pardee *et al.*, 1998). Besides its role in mediating activator-dependent transcription, the B-finger also has a critical role in directing accurate initiation of transcription. The functions of the B-finger in transcriptional activation and start site selection are located on different amino acid residues (Hawkes and Roberts, 1999). Similar to the activation-defective B-finger mutants, distinct mutations in the B-finger with aberrant transcriptional start site selection also do not show defects in PIC assembly (Hawkes and Roberts, 1999; Fairley *et al.*, 2002). It has been proposed that these distinct B-finger mutations, which cannot undergo proper conformational changes, alter TFIIB's interaction with the pol II catalytic center, thus shifting the transcription start site. Recent photo crosslinking experiments have shown that TFIIB helps position the path of promoter DNA across the central cleft of pol II (Chen and Hahn, 2003, 2004). Supporting crystallographic data has also indicated that the B-finger of TFIIB forms a finger-like structure projecting into the active center of pol II (Bushnell *et al.*, 2004), at a location close to the RAP74 subunit of TFIIF (Chen and Hahn, 2004). This configuration enables TFIIB to work in conjunction with pol II/TFIIF for start site selection. Although mutations in the B-finger may not be directly involved in PIC formation, conformational changes in TFIIB does play an essential role in transcriptional activation, promoter recognition, and also start site selection.

---

### Role of TFIIB Post PIC Assembly

In addition to its role in facilitating promoter recognition by TBP/TFIID and in recruiting pol II/TFIIF, TFIIB also modulates transcription at a step after PIC assembly. From the structural analysis of the pol II-TFIIB complex, it is obvious that the B-finger of TFIIB, situated in the RNA exit channel, is likely to block the extension of newly synthesized RNA transcripts (Bushnell *et al.*, 2004). Consistent with this prediction, transcript elongation by newly-initiated pol II complexes that retain TFIIB is strongly inhibited beginning at +7. This block is reduced when human TFIIB bearing a mutation in the B-finger (R66L) is substituted for wild-type TFIIB (Pal *et al.*, 2005). This finding is consistent with yeast studies showing that mutations in a B-finger residue (E62G and E62K) exhibit a post PIC assembly transcriptional defect (Cho and Buratowski, 1999; Ranish *et al.*, 1999).

### TFIIF

#### Discovery of TFIIF

TFIIF was not initially identified when nuclear proteins were fractionated by P11 ion-exchange chromatography simply into four (A, B, C, and D) fractions (Matsui *et al.*, 1980). The discovery of TFIIF was made possible only after further purification of the C fraction and following the identification of TFIIE (Flores *et al.*, 1988). Human TFIIF was found to be composed of previously identified RNA polymerase II-associated proteins 30 (RAP30) and 74 (RAP74) isolated from calf thymus extracts as well as from human and mouse cell lines (Sopta *et al.*, 1985; Flores *et al.*, 1988). Further characterization of TFIIF by size exclusion column chromatography suggested that TFIIF is a heterotetramer comprising two subunits each of RAP30 and RAP74 (Flores *et al.*, 1990). The cDNAs for RAP30 and RAP74 had been cloned from humans (Aso *et al.*, 1992; Finkelstein *et al.*, 1992), *Drosophila* (Kephart *et al.*, 1993; Frank *et al.*, 1995; Gong *et al.*, 1995), and yeast (Henry *et al.*, 1994). The yeast TFIIF contains three subunits (Henry *et al.*, 1992; Henry *et al.*, 1994): Tfg1 (105 kDa) and Tfg2 (54 kDa) are essential for yeast viability and are counterparts of human RAP74 and RAP30, respectively; the third subunit, Tfg3 (30 kDa), is non-essential and is identical to yeast TAF14/ANC1, which is also present in yeast TFIID and SWI/SNF chromatin remodeling complexes (Cairns *et al.*, 1996a). Since Tfg3 exists in multiple complexes and is able to interact directly with

M. C. Thomas and C.-M. ChiangTFIIB and TBP (Kimura and Ishihama, 2004), it is probable that Tfg3 may serve as an intermediary factor to facilitate interactions between SWI/SNF and components of the general transcription machinery.

# Structure and Functional Domains of TFIIF

Human RAP30 (249 amino acids, calculated 26 kDa, apparent mass by SDS-PAGE ~30 kDa) shares two regions of sequence homology with bacterial σ factors (Figure 7B): a central domain (σ homology I, residues 107 to 170) that interacts with pol II, and a C-terminal domain (σ homology II, residues 162 to 249) possessing cryptic DNA-binding activity (Sopta et al., 1989; McCracken and Greenblatt, 1991; Garrett et al., 1992; Tan et al., 1994). RAP30 also contains an N-terminal RAP74-interacting domain spanning amino acids 1 to 98 and a TFIIB-interacting region located within amino acids 27 to 152 (Fang and Burton, 1996). Interestingly, human TFIIF prebound to *E. coli* RNA polymerase core enzyme (α₂ββ′) can be displaced by bacterial σ⁷⁰ factor presumably via the same region in σ⁷⁰ that shares homology with RAP30 (McCracken and Greenblatt, 1991). The RPB5 subunit of human pol II has been shown to interact with the central domain of RAP30 spanning amino acids 107 to 170 in vivo and in vitro (Wei et al., 2001). The solution structure of the C-terminal 86 amino acid residues (164 to 249) of RAP30, revealed by multinuclear NMR spectroscopy, shows that the DNA-binding domain of RAP30 belongs to the eukaryotic “winged” HTH family of DNA-binding domains, similar to that found in histone H5 and the hepatocyte nuclear transcription factor HNF-3γ (Groft et al., 1998). This “winged” HTH motif encompasses amino acid residues 175 to 243 of RAP30 (see Figure 7B). Within this region is an α-helix (H1, amino acids 179 to 193) followed by a short strand (S1, residues 196 to 197) of antiparallel sheet leading to the HTH motif, which is composed of an α-helix (H2, amino acids 199 to 205), turn (amino acids 206 to 209), and another α-helix (H3, amino acids 210 to 219). Following the HTH are two antiparallel β-sheets (S2, amino acids 222 to 225; and S3, 231 to 234) that are connected by a short loop of 5 amino acids (residues 226 to 230), forming the “wing” of RAP30 (Groft et al., 1998). The DNA contacts in the winged HTH domain have been mapped to specific amino acid residues in helices H1 (Arg177, Ala178, Lys180-His182) and H2 (Asn198-Lys200), turn (Lys207,

Gln208), helix H3 (Val210, Val211, Glu215, Lys218), and the wing (Lys226, His229, Asn231). It should be noted that RAP30 also interacts with TFIIEβ, although the interaction domain on RAP30 has not yet been described (Okamoto et al., 1998).

The larger subunit of TFIIF, RAP74 (517 amino acids, calculated 58 kDa, apparent mass by SDS-PAGE ~74 kDa), has three functional domains able to interact with RAP30, the TAF1 subunit of TFIID, TFIIAαβ, TFIIB, pol II, and a protein phosphatase FCP1 (Ruppert and Tjian, 1995; Fang and Burton, 1996; Archambault et al., 1998; Okamoto et al., 1998; Langelier et al., 2001; Abbott et al., 2005a; see Figure 7B). The RAP30-interacting region is mapped to the N-terminal globular domain of RAP74 spanning amino acids 1 to 172 (Fang and Burton, 1996). The structure of the RAP30-RAP74 heterodimer from respective protein-protein interaction domains (RAP74, amino acids 2 to 172; and RAP30, residues 2 to 119), resolved at 1.7 Å by X-ray crystallography, reveals a triple-barrel β-structure comprising 16 parallel and antiparallel β-strands and some less localized loops and α-helices (Gaiser et al., 2000). The N-terminal domain of RAP74 that encompasses amino acids 1 to 139 interacts with TAF1 (Ruppert and Tjian, 1995), which can phosphorylate RAP74 through either the N-terminal or the C-terminal kinase domain of TAF1 (Dikstein et al., 1996). This TAF1-mediated interaction and phosphorylation of RAP74 can also be detected in the TFIID complex (Rossignol et al., 1999; Wu and Chiang, 2001a). The TFIIAαβ-interacting regions on RAP74 have been mapped to two independent domains spanning amino acids 76 to 136 and 410 to 444 (Langelier et al., 2001). Clearly, complex formation mediated by the N-terminal domain of RAP74 with RAP30, TFIIAαβ, and TAF1/TFIID is likely important for TFIIF involvement in PIC assembly, initiation and elongation (Lei et al., 1998; Langelier et al., 2001; Funk et al., 2002).

The central region of RAP74 spanning amino acids 179 to 357 is enriched in Glu and Asp acidic amino acid residues (Aso et al., 1992; Finkelstein et al., 1992). This region, significantly lacking hydrophobic residues, is hypothesized to be externally exposed and unstructured within the PIC (Yong et al., 1998) and can be phosphorylated by TAF1 and CK2 (Yonaha et al., 1997; Rossignol et al., 1999). In addition, it was reported that RAP74 has serine/threonine kinase activity capable of autophosphorylating itself at serine 385 and threonine 389, which may down-regulate pol II elongation in “run-off”

The General Transcription Machinery and General Cofactors
transcription assays (Rossignol *et al.*, 1999). The significance of these phosphorylation events in the context of transcriptional complexes has not been deciphered.

Similar to RAP30, RAP74 also contains a cryptic DNA-binding domain belonging to the winged HTH family, in which the structure spanning the C-terminal 69 amino acid residues of RAP74 (amino acids 449 to 517) has been resolved by X-ray crystallography at 1.02 Å resolution (Kamada *et al.*, 2001a). Within this C-terminal region of RAP74, there is an α-helix (H1, amino acids 456 to 465) and a short β-strand (S1, amino acids 467 to 469) leading to the HTH motif, which is composed of a second α-helix (H2, amino acids 470 to 475), a long loop (amino acids 476 to 485), and a third α-helix (H3, amino acids 486 to 500). This HTH motif is followed by two antiparallel β-sheets (S2, amino acids 503 to 507, and S3, residues 510 to 514) connected by a short turn (amino acids 508 to 509) that forms the “wing,” similar to the winged HTH structure described earlier for RAP30. In analogy with the winged HTH domain of RAP30 that exhibits nonspecific DNA-binding activity with preference for double-stranded DNA sequences located between the TATA box and the Inr (Tan *et al.*, 1994; Forget *et al.*, 2004), the winged HTH domain of RAP74 likely contributes to the DNA-binding activity of RAP74 in the PIC (Robert *et al.*, 1998; Forget *et al.*, 2004). Indeed, the presence of the winged HTH domain significantly extends the promoter contacts by the N-terminal region of RAP74 further upstream of the TATA box and downstream of the Inr (Forget *et al.*, 1997; Robert *et al.*, 1998).

Besides binding DNA, the C-terminal region of RAP74 spanning amino acids 358 to 517 can also interact with TFIIB, TFIIAαβ, pol II, and FCP1 (Figure 7B). The interaction with TFIIB has been mapped to amino acids 358 to 517 of RAP74, which competes with TFIIB for overlapping binding surfaces at the N-terminal region of RAP30 (Fang and Burton, 1996). The pol II- and FCP1-interacting regions have been located at amino acids 363 to 444 and 436 to 517 of RAP74, respectively (Fang and Burton, 1996; Archambault *et al.*, 1998; Abbott *et al.*, 2005a), suggesting that pol II and FCP1 may bind simultaneously to non-overlapping surfaces at the C-terminal domain of RAP74 (see Figure 7B). The interaction between RAP74 and FCP1, which removes phosphorylation on serine 2 at the carboxy-terminal domain (CTD) of the pol IIRPB1 subunit to regenerate a hypophosphorylated form of pol II competent for initiation or reinitiation (Cho *et al.*, 2001), is

necessary and sufficient for FCP1 phosphatase activity in vitro (Archambault *et al.*, 1997, 1998) and may account for the requirement of the C-terminal region of RAP74 for multiple rounds of transcription (Lei *et al.*, 1998). Undoubtedly, the FCP1 phosphatase activity is likely modulated by some RAP74-and FCP1-interacting proteins. Indeed, TFIIB can inhibit FCP1 activity by displacing RAP74 from FCP1 via competition with RAP74 for the same binding surface on FCP1 (Chambers *et al.*, 1995; Kobor *et al.*, 2000; Nguyen *et al.*, 2003). In contrast, the FCP1 phosphatase activity is enhanced by CK2-mediated phosphorylation of FCP1 that in turn enhances the interaction between FCP1 and RAP74 (Palancade *et al.*, 2002; Abbott *et al.*, 2005b). Other than these heteromeric protein-protein interactions, RAP74 also shows homomeric interactions with itself, through either the C-terminal region (amino acids 407 to 517) or a linker region (amino acids 172 to 205) situated between the N-terminal RAP30-interacting domain and the central acidic domain (Robert *et al.*, 1998; see Figure 7B). As with other GTFs, TFIIF may be recruited to the promoter through interaction with some transcriptional activators, such as serum response factor (SRF; Joliot *et al.*, 1995), c-Jun/c-Fos (Martin *et al.*, 1996), and androgen receptor (AR; McEwan and Gustafsson, 1997; Reid *et al.*, 2002). The interaction with SRF has been mapped to the central acidic domain of RAP74 (Joliot *et al.*, 1995), whereas the interaction with AR has been mapped to the N-terminal 136 amino acids and C-terminal 155 residues of RAP74 (Reid *et al.*, 2002). These findings indicate that each domain of RAP74 can serve as a target for transcriptional activators, thereby facilitating TFIIF recruitment to the transcriptional complex.

### Multiple Roles of TFIIF

TFIIF plays multiple roles during PIC assembly. First, TFIIF tightly associates with pol II (Sopta *et al.*, 1985; Price *et al.*, 1989). The RAP74 subunit of yeast TFIIF has been shown to contact the dissociable RPB4/RPB7 subunits of yeast pol II, based on structural comparison between pol II-TFIIF and pol II by cryo-electron microscopy (Chung *et al.*, 2003), and to interact with the RPB9 subunit of yeast pol II (Ziegler *et al.*, 2003; Ghazy *et al.*, 2004). This stable TFIIF-associated pol II complex accounts for ~50% of pol II isolated from *Saccharomyces cerevisiae* nuclear extracts and is active in supporting multiple rounds of

M. C. Thomas and C.-M. Chiang

transcription (Rani et al., 2004). Human RAP30 has also been shown to interact with the RPB5 subunit of human pol II (Wei et al., 2001). The interaction between TFIIF and pol II facilitates the recruitment of pol II to the promoter-bound TFIID-TFIIB complex (Flores et al., 1991). Second, TFIIF serves as a stability factor to enhance the affinity of pol II for the TBP-TFIIB-promoter complex by providing additional protein-DNA contact surfaces and likely by inducing changes in DNA topology which causes the promoter to wrap around pol II (Robert et al., 1998). This TFIIF-induced conformational change creates a stable TBP-TFIIB-pol II-TFIIF-promoter DNA complex that may confer resistance to inhibition by transcriptional repressors that target PIC assembly to negatively regulate gene transcription (Hou et al., 2000). In the study of human papillomavirus (HPV) E2-mediated transcriptional repression, we found that E2 could still inhibit HPV transcription when added to a preformed TFIID (or TBP)-TFIIB-pol II-promoter DNA complex; however, E2 failed to inhibit transcription once a TFIID (or TBP)-TFIIB-pol II-TFIIF-DNA complex was formed, presumably a surface targeted by E2 for repression was masked by TFIIF-induced DNA wrapping on this intermediate PIC (Hou et al., 2000). Third, TFIIF is necessary for subsequent recruitment of TFIIE and TFIIH (Orphanides et al., 1996) likely via direct interactions with TFIIE (Maxon et al., 1994). Fourth, TFIIF, together with pol II and TFIIB, plays a role in transcription start site selection (Fairley et al., 2002; Ghazy et al., 2004), due to the close proximity of TFIIB and TFIIF near the active center of pol II (Bushnell et al., 2004; Cramer, 2004; Hahn, 2004; Freire-Picos et al., 2005). Fifth, TFIIF has been implicated in facilitating pol II promoter escape (Yan et al., 1999). Following this transition, TFIIF further enhances the efficiency of pol II elongation (Price et al., 1989; Shilatifard et al., 2003), likely by promoting forward NTP-driven translocation at the active center of pol II (Zhang et al., 2005). When pol II pauses during RNA synthesis, TFIIF is able to work in conjunction with TFIIS, a transcription elongation factor that stimulates pol II cleavage activity on nascent RNA, to overcome this elongation block (Zhang and Burton, 2004). Lastly, TFIIF increases the specificity and efficiency of pol II transcription, similar to bacterial σ factors, by preventing spurious initiation through inhibiting and/or reversing the binding of pol II to non-promoter DNA sequences (Orphanides et al., 1996; Hampsey, 1998). It is clear that TFIIF has multiple roles in the pol II transcription process.

---

CORE POL II

Subunit Composition

Pol II is the key catalytic enzyme in the PIC responsible for transcription of protein-coding genes in eukaryotes. Yeast and human pol II both contain 12 subunits, designated RPB1 to RPB12 by decreasing order of their molecular mass (Young, 1991). In general, the 12 subunits of pol II are highly conserved in sequence, architecture, and function. Indeed, 7 subunits of human pol II can either partially (RPB4, RPB7, and RPB9) or completely (RPB6, RPB8, RPB10, and RPB12) substitute for the function of their yeast counterparts in complementation assays (McKune et al., 1995; Khazak et al., 1998). Of the 12 pol II subunits, 5 (RPB5, RPB6, RPB8, RPB10, and RPB12) are commonly shared among pol I, pol II and pol III (Woychik et al., 1990; Carles et al., 1991; Young, 1991; Hampsey, 1998), whereas 4 subunits (RPB1, RPB2, RPB3, and RPB11) have sequence-homologous counterparts in pol I and pol III. Only RPB4, RPB7, RPB9 and the CTD of RPB1 are unique to pol II. In addition, RPB1, RPB2, RPB3, and RPB6 share similar primary sequences with bacterial RNA polymerase subunits β′, β, α and ω, respectively (Tan et al., 2000; Minakhin et al., 2001; Mitsuzawa and Ishihama, 2004). A prokaryotic α-like sequence also exists in RPB11 (Woychik et al., 1993; Ulmasov et al., 1996). The primary sequence similarity between RPB1 and β′ as well as between RPB2 and β also corresponds to functional similarity: RPB1 and β′ are involved in DNA binding, while RPB2 and β bind nucleotide substrates (Hampsey, 1998). Analogous to their bacterial counterparts, RPB1 and RPB2 are responsible for most of the catalytic activity of polymerase and are essential for phosphodiester bond formation (Hampsey, 1998; Lee and Young, 2000).

---

Structure of Pol II

Recently, there has been a wealth of structural information on prokaryotic and eukaryotic RNA polymerases provided by photocrosslinking, X-ray crystallography, NMR, and cryo-electron microscopy. The structures of a yeast pol II 10-subunit enzyme minus RPB4 and RPB7 in the absence of DNA (Cramer et al., 2000, 2001) and of a complete 12-subunit pol II have recently been resolved by X-ray crystallography (Armache et al., 2003, 2005; Bushnell and Kornberg, 2003). RBP4 and RBP7 were not included in the original crystals

The General Transcription Machinery and General Cofactors
since the heterodimeric RPB4/RPB7 module is found in substoichiometric amounts in pol II and may dissociate from the “core” 10 subunits. Furthermore, the RPB4/RPB7 heterodimer, although required for PIC formation and initiation of transcription, is dispensable for RNA chain elongation (reviewed by Hampsey, 1998; Lee and Young, 2000; Cramer, 2004; Hahn, 2004). The structure of the 12-subunit pol II complex pinpoints the location of RPB4/RPB7 close to the RNA exit channel and also suggests a role of this heterodimer in transcriptional initiation (Armache et al., 2003, 2005; Bushnell and Kornberg, 2003). The CTD (amino acids 1535 to 1733 in yeast), representing an unstructured extension from the catalytic core of pol II, is flexibly linked to a region adjacent to the RNA exit channel via an 80-residue linker and provides a platform for interacting with many proteins involved in 5′ capping, mRNA splicing, termination and 3′-end processing (Meinhart et al., 2005). Comparison of the structures between the 10-subunit free core enzyme and a transcribing elongation complex containing the same core enzyme in complex with 9 base pairs of an RNA-DNA hybrid within a partially unwound DNA duplex has revealed detailed information regarding subunit–subunit and protein-nucleic acid contacts both within and outside the catalytic center of the enzyme (Cramer et al., 2001; Gnatt et al., 2001). Furthermore, contact residues between pol II and distinct domains of TFIIB, based on the structural information, have been defined by photocrosslinking experiments (Chen and Hahn, 2003, 2004). The structures of pol II complexed with part of the zinc ribbon motif and the B-finger of TFIIB (Bushnell et al., 2004), with elongation factor IIS (Kettenberger et al., 2003), and with IIS in the presence of NTPs and a transcription bubble-mimicking DNA-RNA hybrid (Kettenberger et al., 2004) have also been elucidated by X-ray crystallography. Low-resolution cryo-electron microscopy has resolved structures for a pol II-Mediator complex (Davis et al., 2002) and for pol II interaction with TFIIF (Chung et al., 2003). The implications of these structural studies have been the focus of many recent reviews (Woychik and Hampsey, 2002; Asturias, 2004; Cramer, 2004; Hahn, 2004; Boeger et al., 2005).

### CTD Phosphorylation

An essential feature of all pol II complexes involved in transcription and mRNA processing resides in the CTD of RPB1, the largest subunit of pol II. The CTD contains a tandem repeat of a heptapeptide: Tyr-Ser-Pro-Thr-Ser-Pro-Ser (YSPTSPS), which repeats 52 times in humans, 42 times in *Drosophila*, and 26 to 29 times in yeast, depending upon the species (Dahmus, 1995; Hampsey, 1998; Lee and Young, 2000). The CTD is disordered in pol II structures and tends to be degraded by proteases. Depending on the phosphorylation state and the presence or absence of the CTD, three forms of human pol II (II0, IIA, and IIB; Figure 8) can be easily isolated and distinguished (Kershner et al., 1998). The IIA form of pol II contains a hypo- or unphosphorylated CTD normally implicated in PIC assembly and transcription initiation (Lu et al., 1991; Serizawa et al., 1993). The II0 form of pol II, involved in transcript elongation and termination, has a highly phosphorylated CTD with phosphorylation occurring primarily at serine residues 2 and 5 that are subject to a cycle of phosphorylation and dephosphorylation throughout the transcriptional process. The IIB form of pol II, representing a proteolytic derivative, does not have the CTD but remains transcriptionally active for at least the adenovirus major late promoter (Kim and Dahmus, 1989; Buratowski and Sharp, 1990; Kang and Dahmus, 1993; Kershner et al., 1998), the human rep-3b TATA-less promoter (Buermeyer et al., 1992) and some TATA-containing cellular promoters derived from human histone H2B (Buermeyer et al., 1995) and *Drosophila* HSP70 and actin 5C genes (Zehring and Greenleaf, 1990).

Several protein kinases implicated in CTD phosphorylation have been identified in humans, including cyclin-dependent kinase 7 (CDK7) associated with TFIIH (Feaver et al., 1991b), CDK8 found in general cofactor Mediator, and CDK9 present in positive transcription elongation factor b (P-TEFb). The activities of these CTD kinases are regulated by their associated cyclins that form CDK7-cyclin H, CDK8-cyclin C, and CDK9-cyclin T pairs. Similar CTD kinases have been identified in yeast and include Cdk7/Kin28, Cdk8/Srb10, the CTD kinase 1 (CTDK-I), and Sgv1/Bur1 (Prelich, 2002). Both Bur1 and the catalytic subunit of CTDK-I, Ctk1, show sequence homology to mammalian CDK9. Phosphorylation of serine 5 by Cdk7/Kin28 following PIC assembly leads to initiation of transcription (Serizawa et al., 1993) and later recruitment of mRNA-capping enzyme guanylyltransferase (Cho et al., 1997; Komarnitsky et al., 2000; Rodriguez et al., 2000; Schroeder et al., 2000; Pei et al., 2001). Phosphorylation of serine 2 by other

M. C. Thomas and C.-M. Chiang
hRPB9-3 cells

cytoplasmic S100
nuclear extract
nuclear pellet

M2-agarose

low salt
high salt

SRBs
SWI/SNF
GCN5
TFIIE
TFIIB
TFIIF
pol II
pol II holoenzyme

core pol II
pol II
IIA
pol II
IIB
pol II
IIo

FIGURE 8 Purification of human RNA polymerase II complexes. Four different forms of RNA polymerase II (pol II) are purified from a stable cell line (hRPB9-3) conditionally expressing FLAG epitope-tagged human RPB9 (Wu and Chiang, 2001b). Pol II holoenzyme is normally purified from the cytoplasmic S100 or nuclear extract fraction. Similarly, the IIA (i.e., containing hypo- or unphosphorylated CTD) form of core pol II comprised of RPB1 to RPB12 subunits can also be isolated from the same S100 or nuclear extract fraction, but under high salt wash conditions (Kershner et al., 1998). The IIo (i.e., containing hyper-phosphorylated CTD) and the IIB (i.e., CTD-truncated) forms of pol II can be additionally purified from the nuclear pellet. The tail represents the CTD (carboxy-terminal domain) found in the RPB1 subunit of pol II.

CTD kinases, such as CTDK-I or P-TEFb (Cho et al., 2001; Zhou et al., 2000; Shim et al., 2002), results in transcription-coupled recruitment of 3'-end processing factors (Komarnitsky et al., 2000; Ahn et al., 2004). Two other CTD kinases, Cdk8/Srb10 (Hengartner et al., 1998) and c-Abl (Baskaran et al., 1993), have also been implicated in phosphorylation of serine 2 and tyrosine 1, respectively, although the functional effects remain to be defined.

Protein phosphatases that remove the phosphate group on serine 2 or serine 5 have likewise been identified. Yeast Ssu72 (Krishnamurthy et al., 2004), plant Arabidopsis thaliana CTD phosphatase-like proteins AtCPL1 and AtCPL2 (Koiwa et al., 2004), and human small CTD phosphatase 1 (SCP1) protein (Yeo et al., 2003) are able to dephosphorylate serine 5 in vitro, whereas TFIIF-associated CTD phosphatase 1 (FCP1) isolated from yeast (Archambault et al., 1997; Kimura et al., 2002) and humans (Archambault et al., 1998; Cho et al., 1999) is mainly implicated in serine 2 dephosphorylation (Cho et al., 2001; Meinhart et al., 2005). FCP1 interacts with TFIIB (Chambers et al., 1995; Kobor

et al., 2000), the RPB4 subunit of pol II (Kimura et al., 2002), and the RAP74 component of TFIIF (Chambers et al., 1995; Kobor et al., 2000). It has been shown that TFIIF can stimulate FCP1 phosphatase activity and may thus accelerate reinitiation of transcription by enhancing the conversion of pol II from the elongating IIo form back to the initiating IIA form (Chambers et al., 1995; see TFIIF section). Although TFIIB is able to inhibit TFIIF-stimulated FCP1 phosphatase activity, the functional role of this inhibition is still unclear (Chambers et al., 1995). Undoubtedly, the counteracting activity between Ctk1-FCP1 and TFIIH-Ssu72 on CTD phosphorylation must play an important role in initiation, elongation, termination, and transcription-coupled mRNA processing.

### CTD Glycosylation

In addition to phosphorylation, the CTD can also be modified by glycosylation via covalent linkage of a monosaccharide, N-acetylglucosamine (GlcNAc), onto the side chain hydroxyl group of serine or threonine

The General Transcription Machinery and General Cofactors

residues in the heptapeptide repeat (Kelly *et al.*, 1993). The β-O-linked N-acetylglucosamine (O-GlcNAc) is transferred predominantly to the fourth position (usually a threonine in the consensus YSPTSPS repeat, but occasionally a serine in nonconsensus sequences) of the heptapeptide repeat (Kelly *et al.*, 1993) by an O-GlcNAc transferase (OGT), with the reverse deglycosylation reaction carried out by an O-GlcNAc-specific β-N-acetylglucosaminidase (O-GlcNAcase; Iyer *et al.*, 2003 and references therein). The glycosylated Ser/Thr residues can be found in the heptapeptide sequence spread throughout the entire CTD domain (Kelly *et al.*, 1993). Glycosylation is only found in the IIA form of pol II and appears to be mutually exclusive with phosphorylation on the CTD (Comer and Hart, 2001; Iyer *et al.*, 2003). It is likely that glycosylation on Thr/Ser sterically blocks kinase accessibility to the CTD and thus regulates pol II activity during the early stage of the transcriptional process. Alternatively, the extent of glycosylation may alter CTD conformation, thereby modulating protein dynamic interaction on the CTD scaffold. The exact functional role of CTD glycosylation remains to be elucidated.

### Pol II Ubiquitination

Ubiquitination is the third posttranslational modification that has been characterized on pol II. Yet, this important subject has not been extensively covered thus far in any reviews. In general, ubiquitination on pol II occurs at a low level under normal growth condition and can be readily detected when cells are under stress, particularly in response to specific DNA-damaging agents, or when pol II encounters elongation blocks during the transcriptional process independent of DNA damage. Treating cells with UV irradiation or the chemotherapeutic drug, cisplatin, induces DNA damage and causes pol II arrest at sites of DNA lesions (Bregman *et al.*, 1996). Under this circumstance, arrested pol II becomes polyubiquitinated, and the proteins involved in the nucleotide excision repair (NER) pathway are recruited to the damage sites. This transcription-coupled repair (TCR) process correlates with pol II ubiquitination and appears to require transcription-repair coupling factors Cockayne syndrome A and B (CSA and CSB), which are involved in the TCR pathway facilitating removal of DNA lesions preferentially from the transcribed strand of an active gene. Indeed, ubiquitination on the RPB1 subunit of pol II was detected within 15 minutes in UV-irradiated or cisplatin-treated human HeLa cells and persisted for ~8 to 12 hours; in contrast, RPB1 was not ubiquitinated in CSA-or CSB-deficient human fibroblasts derived from patients with Cockayne syndrome who exhibited defects in the TCR pathway (Bregman *et al.*, 1996). This finding suggests that CSA and CSB are essential for pol II ubiquitination in response to UV irradiation and cisplatin treatment. However, when TCR was induced by hydrogen peroxide (H₂O₂), which is a different DNA-damaging agent that triggers the base excision repair (BER) pathway rather than the NER, CSA and CSB were not required for pol II ubiquitination (Inukai *et al.*, 2004), indicating that ubiquitination of pol II can occur via the TCR pathway following treatment of cells with different types of DNA-damaging agents that activate either the NER or the BER pathway.

In yeast, Rad26, which is a homolog of human CSB and a member of the SWI2/SNF2 family of DNA-dependent ATPase, is implicated in TCR and appears to assist pol II in overcoming the elongation block, likely by enhancing an open (*i.e.*, remodeled) chromatin structure at sites of DNA lesions, thereby allowing TCR and resumption of pol II elongation (Lee *et al.*, 2002b; Woudstra *et al.*, 2002; Bucheli and Sweder, 2004). However, when damage is beyond repair, Rad26 then recruits Def1 (degradation factor 1), which in turn interacts with pol II and enhances ubiquitination and subsequent degradation of pol II (Woudstra *et al.*, 2002; Reid and Svejstrup, 2004). In general, DNA damage-induced RPB1 ubiquitination results in pol II degradation by the 26S proteasome (Beaudenon *et al.*, 1999; Inukai *et al.*, 2004), a view consistent with *in vivo* ChIP assays showing that the 26S proteasome can be recruited to stalled pol II at sites of UV-induced DNA damage (Gillette *et al.*, 2004). It should be noted that the association of pol II with only the 19S regulatory subcomplex, but not the 20S catalytic core, of the 26S proteasome appears to enhance pol II elongation during regular transcription, apparently in a nonproteolytic fashion (Ferdous *et al.*, 2001). Clearly, the association of the 19S subcomplex with pol II likely provides a platform for subsequent assembly with the 20S core to form the 26S proteasome leading to pol II degradation, when ubiquitination of pol II occurs at the unrepaired damage sites.

Three enzymes involved in the thioester cascade for pol II ubiquitination have been identified in yeast: E1 ubiquitin-activating enzyme Uba1, E2 ubiquitin-conjugating enzyme Ubc5 (or Ubc4), and E3 ubiquitin

M. C. Thomas and C.-M. Chiang

ligase Rsp 5 (Somesh *et al.*, 2005). Rsp 5 (reversion of Spt phenotype; an extragenic suppressor of mutations in the Spt3 gene) is the only E3 ubiquitin ligase in yeast known to ubiquitinate pol II through interaction with the CTD of RPB1 (Huibregtse *et al.*, 1997; Beaudenon *et al.*, 1999; Lommel *et al.*, 2000). This interaction is mediated through recognition of the PXY motif (X can be any amino acid) present in contiguous heptapeptide repeats (YSPTSPS YSPTSPS) of the CTD by the second or third WW domain of Rsp 5 (Wang *et al.*, 1999; Chang *et al.*, 2000). The WW domain is a conserved 38 to 40 amino acid module containing two conserved tryptophan (W) residues folded into a three-stranded antiparallel β-sheet with a hydrophobic binding pocket for PXY recognition (Macias *et al.*, 2002). Although the CTD is essential for interaction with Rsp 5, ubiquitination of yeast pol II occurs outside the CTD at lysine 695 of RPB1 (Somesh *et al.*, 2005) and also in the RPB7 subunit (Kus *et al.*, 2005). It appears that multiple components of pol II can serve as targets for ubiquitination, leading to pol II degradation induced by different DNA-damaging agents.

In mammalian cells, multiple E3 ubiquitin ligases, including human receptor potentiator factor 1 (hRPF1), von Hippel-Lindau tumor suppressor protein (pVHL) complex, breast cancer gene 1-encoded protein (BRCA1), and the CSA complex, have been implicated in pol II ubiquitination. The WW domain-containing hRPF1 protein, also named hNEDD4 (human neural precursor cell-expressed and developmentally downregulated), is the human homolog of yeast Rsp 5 (Imhof and McDonnell, 1996; Gajewska *et al.*, 2003) and is able to interact with PXY-containing proteins, such as the p45 subunit of NF-E2 transcription factor (Gavva *et al.*, 1997) and the CTD domain of RPB1 (Beaudenon *et al.*, 1999). While the catalytic HECT (homologs to the E6-AP carboxyl terminus) domain of Rsp 5, and likely hRPF1, is essential for pol II ubiquitination, this ubiquitin ligase activity is dispensable for the coactivator function of hRPF1 in stimulating hormone-dependent activation by human glucocorticoid receptor (hGR) and by progesterone receptor isoform B (hPR-B; Imhof and McDonnell, 1996). It is interesting to note that cotransfection of Spt3, which is found in yeast SAGA and SLIK, as well as human TFTC and STAGA complexes (see discussion of TAF-containing complexes), with either Rsp5 or hRPF1, synergistically activate transcription (Imhof and McDonnell, 1996), suggesting that these two E3 ubiquitin ligases may function through

TAF-containing complexes to regulate target gene transcription. The finding that Rsp 5 is also present in some yeast TFIID preparations and that TAF1 and TAF5 can be ubiquitinated (Auty *et al.*, 2004) further indicates that ubiquitination on TAF-containing complexes may play an important role in fine-tuning the activity of these coactivator complexes. This intriguing possibility is yet to be explored.

The pVHL tumor suppressor protein associates with Elongin B, Elongin C, Cullin 2 (Cul2), and RING-box protein Rbx-1 (also called ROC1 for Regulator of Cullins, SAG1 for Sensitive to Apoptosis Gene, Hrt1 for High-level expression reduces transposition by the Retrotransposon Ty3; Kamura *et al.*, 1999; Ohta *et al.*, 1999; Skowyra *et al.*, 1999; Seol *et al.*, 1999; Tan *et al.*, 1999) to form an E3 ubiquitin ligase complex involved in the ubiquitination of RPB1 (Kuznetsova *et al.*, 2003) and RPB7 (Na *et al.*, 2003). In general, a cullin-based E3 ubiquitin ligase contains a substrate recognition protein, adaptors, and an E2-binding module formed by a cullin family member and a RING (Really Interesting New Gene; Saurin *et al.*, 1996) protein possessing two Zn²⁺-binding fingers. This class of RING finger-containing E3 ligases, unlike the HECT E3 ligase family proteins such as Rsp 5 and hRPF1, does not form a covalent thioester bond with ubiquitin and mainly serves as a scaffold bridging the E2 enzyme and the substrate (Pickart, 2004). Together, Cul2 and Rbx1 form a module that interacts with different E2 ubiquitin-conjugating enzymes, including UbcH5a, UbcH5b, and UbcH5c (Jensen *et al.*, 1995; Iwai *et al.*, 1999). UbcH5c and its yeast homologs (*i.e.*, Ubc5/Ubc4) have been shown to work in conjunction with the HECT family E3 ubiquitin ligase Rsp 5 and the E1 enzyme Ubal to ubiquitinate pol II RPB1 (Somesh *et al.*, 2005), suggesting that the UbcH5 family has the capacity to function with both HECT and RING finger E3 ubiquitin ligases. Elongin B and Elongin C, which are also present in the Elongin complex involved in pol II elongation control (Shilatifard *et al.*, 2003), are the adaptors forming a subcomplex that bridges the Cul2/Rbx1 module with the substrate recognition unit pVHL (Petroski and Deshaies, 2005). The pVHL protein with 213 amino acid residues has two distinct domains: an N-terminal domain rich in β sheet spanning amino acids 63 to 154 that mainly interact with substrates destined for ubiquitination, and an α-domain comprising of three α-helices spanning amino acids 155 to 192 that contact Elongin C (Stebbins *et al.*, 1999). The N-terminal
region (amino acids 54 to 113) of pVHL, including  
part of the β-domain, binds RPB7, correlating with  
the finding that naturally occurring pVHL mutations  
(P86H and Y98H) within the β-domain display both  
reduced interaction and ubiquitination of RPB7 (Natal., 2003). Since RPB7 in complex with RPB4  
may form a dissociable module with the 10-subunit  
pol II core, it remains to be investigated whether  
ubiquitination and degradation of RPB7 occurs mainly  
as a free module or in the context of pol II. In contrast,  
the ability of pVHL to bind human RPB1 requires a  
hyperphosphorylated CTD and a hydroxylated proline  
(P1465) in the L(XX)LAP motif (where X represents any  
amino acid; Huang et al., 2002) residing in the linker  
region (Kuznetsova et al., 2003). The requirement of  
an oxidized P1465 for pVHL binding is reminiscent  
of that seen with P564 of HIF-1α in oxygenated cells  
(during normoxia), in which recognition of the hydrox-  
ylated proline (mediated by prolyl-4-hydroxylase) by  
the hydrophobic pocket located at the β-domain of  
pVHL (Hon et al., 2002; Min et al., 2002) eventually  
leads to the ubiquitination and degradation of HIF-1α  
(Jaakkola et al., 2001; Ivan et al., 2001). It is important to  
note that pVHL-mediated degradation of RPB1, which  
occurs in a proline hydroxylation-dependent manner  
in response to UV-induced DNA damage, is distinct  
from that used for degradation of RPB7, which lacks  
the L(XX)LAP motif and is thus likely to be degraded  
through a proline hydroxylation-independent and  
DNA damage-independent pathway.  
BRCA1 is the third mammalian E3 ubiquitin ligase  
implicated in RPB1 ubiquitination and pol II degrada-  
tion upon UV-induced DNA damage (Kleiman et al.,  
2005; Starita et al., 2005). Human BRCA1 has 1863  
amino acid residues and belongs to the RING finger  
E3 ubiquitin ligase family. The E2-binding region of  
BRCA1 lies in the RING motif that encompasses  
N-terminal amino acid residues 23–76 of BRCA1  
(Brzovic et al., 2001; Ruffner et al., 2001). This motif,  
flanked on each side by an α-helix, contains two  
Zn²⁺-binding sites (Cys24, Cys27, Cys44 and Cys47  
for the first finger, and Cys39, His41, Cys61 and Cys64  
for the second finger) organized in the following  
manner (Brzovic et al., 2001): the first Zn²⁺-binding  
loop (residues 23 to 34, a C₂C₂ type finger) is followed  
by two short antiparallel β-strands (residues 35 to 37  
and 42 to 44) and a central α-helix (residues 46 to 53),  
which leads to the second Zn²⁺-binding loop (residues  
54 to 73, a C₃H₁ type finger) and a third short β-strand  

(residues 74 to 76). Part of this RING domain, includ-  
ing two Zn²⁺-binding loops and the central α-helix,  
forms a surface cleft binding UbcH5c (Brzovic et al.,  
2001; Brzovic et al., 2003), while the C-terminal region  
of BRCA1 (residues 1560 to 1863) interacts with the  
RPB2, RPB12, and hyperphosphorylated RPB1 sub-  
units of pol II (Schlegel et al., 2000; Krum et al., 2003).  
The identification of cancer-predisposing mutations  
in human BRCA1 at C24R, C39Y, C61G, and C64Y  
within this RING finger domain further indicates the  
importance of these cysteine residues in maintaining an  
active conformation of BRCA1 (Ruffner et al., 2001).  
Interestingly, human BRCA1-associated RING domain  
1 (BARD1), with 777 amino acid residues, also contains  
a RING finger domain at its N-terminus (residues 49  
to 100) exhibiting a similar feature as that of BRCA1  
but lacking the central α-helix. Thus, the RING  
domain of BARD1 cannot form a surface cleft, as  
seen with BRCA1, for binding UbcH5c (Brzovic et al.,  
2003). However, BARD1, in complex with BRCA1,  
can stimulate BRCA1 function as an E3 ubiquitin  
ligase (Xia et al., 2003). Dimerization of BRCA1 and  
BARD1 is mediated by hydrophobic contacts made  
between the N-terminal α-helices preceding the RING  
domain of each protein (Meza et al., 1999; Morris  
et al., 2002; Brzovic et al., 2003). Together, BRCA1  
and BARD1 form an active E3 ubiquitin ligase able to  
work in conjunction with the E2 ubiquitin-conjugating  
enzyme UbcH5c to promote substrate ubiquitination  
(Hashizume et al., 2001; Brzovic et al., 2003) and likely  
pol II degradation. Indeed, it has been shown that  
hyperphosphorylated RPB1 is more susceptible to UV-  
induced ubiquitination and degradation by cotrans-  
fecting BRCA1 and BARD1 expression plasmids into  
HEK-293T cells (Starita et al., 2005), in agreement with  
the observation that RPB1 stability was increased by  
knocking down both endogenous BRCA1 and BARD1  
in UV-irradiated HeLa cells (Kleiman et al., 2005). These  
experiments further stress the physiological importance  
of BRCA1 and BARD1 in regulating pol II turnover in  
human cells responding to UV-induced DNA damage.  
Another mammalian E3 ubiquitin ligase able to  
regulate pol II ubiquitination and degradation is  
the CSA complex. In addition to functioning as a  
transcription-repair coupling factor in the UV- or  
cisplatin-induced TCR pathway, CSA usually associates  
with other cellular proteins under normal growth condi-  
tion. This CSA complex, first isolated from nuclear ex-  
tracts of FLAG/HA-tagged CSA-expressing HeLa cells  

M. C. Thomas and C.-M. Chiang  (Groisman *et al.*, 2003), consists of Rbx1, Cullin4A (Cul4A), damaged DNA-binding protein 1 (DDB1), and a dissociable module comprising all the 8 subunits of COP9 signalosome (CSN). DDB1 is known to bind DNA lesions and facilitates the recruitment of NER components to the damage sites (Wakasugi *et al.*, 2002), whereas Rbx1 and Cul4A likely form the E2 ubiquitin-conjugating enzyme-binding module. The CSN components negatively regulate the ubiquitin ligase activity of CSA-Rbx1-Cul4A, as the ubiquitin ligase activity of the CSA complex could only be detected after dissociation of the CSN module (Groisman *et al.*, 2003). During purification, it was also found that a stoichiometric amount of hyperphosphorylated RPB1 is present in the CSA complex following UV-induced DNA damage without altering the level of hypophosphorylated RPB1 already associated with the CSA complex prior to irradiation. Interestingly, while the CSN module is not detected in the chromatin-bound CSA complex under normal condition, it becomes associated with the CSA complex after UV irradiation, exhibiting the same kinetics as hyperphosphorylated RPB1 for association and dissociation with the CSA complex. It is likely that the association of CSN with CSA-Rbx1-Cul4A-DDB1 upon UV irradiation may temporarily inhibit pol II ubiquitination, giving TCR components time to repair DNA lesions. If damage cannot be efficiently repaired, hyperphosphorylated pol II is then ubiquitinated and degraded, following the dissociation of the CSN module. This strategy ensures pol II can be readily available to resume transcription after TCR, unless it is otherwise destined for degradation when damage is beyond repair. Whether CSN indeed acts as a molecular switch modulating ubiquitin ligase activity of the CSA complex on stalled pol II remains to be defined.

Ubiquitination of pol II is a prerequisite for proteasome-mediated degradation. In general, degradation by the 26S proteasome requires recognition of substrate-linked polyubiquitin chains formed between the extreme C-terminal Gly76 of a ubiquitin molecule with internal Lys48 (major site) or Lys29 (minor site) of a preceding ubiquitin, although nonproteolytic linkage through the other 5 lysine (K) residues, such as K6, K11, K27, K33, and K63, may also occur (Weissman, 2001). At present, little is known regarding the function of K11-, K27-, and K33-linked polyubiquitination, whereas polyubiquitin-linked K63 chains often leads to nonproteolytic pathways such as endocytosis (Galan

and Haguenauer-Tsapis, 1997), error-free postreplicative DNA repair (Spence *et al.*, 1995; Hofmann and Pickart, 1999; Hoege *et al.*, 2002), activation of the IkB kinase in the NFκB pathway (Deng *et al.*, 2000; Wang *et al.*, 2001), and enhancement of ribosome translational efficiency (Spence *et al.*, 2000). Among the E3 ubiquitin ligases known to ubiquitinate pol II, each ligase is able to assemble a distinct pattern of polyubiquitin chains unique to individual substrates. For examples, Rsp5 shows preference for assembling K48-linked polyubiquitin chains on WW domain-binding protein 2 (WBP2; Kee *et al.*, 2005) and K63-linked polyubiquitin chains on the yeast plasma membrane protein uracil permease (Galan and Haguenauer-Tsapis, 1997); the pVHL complex promotes K48-linked polyubiquitination leading to HIF-1α degradation (Ohh *et al.*, 2000); BRCA1/BARD1 enhances K6-linked polyubiquitination on nucleophosmin (Sato *et al.*, 2004) as well as autoubiquitination on BRCA1 through K6, K29, K48, or K63 linkage (Chen *et al.*, 2002; Wu-Baer *et al.*, 2003; Xia *et al.*, 2003; Nishikawa *et al.*, 2004). Although the specific lysines in polyubiquitinated pol II by these E3 ligases have not yet been characterized, it is likely that some of these modifications may play a degradation-independent role to coordinate pol II activity in TCR or DNA damage-independent pathways. Indeed, it has been found that transcriptional arrest induced by α-amanitin, which blocks nucleoside triphosphates incorporation into the nascent transcript, promotes pol II polyubiquitination at K6, K48, and K63 of ubiquitin, in the absence of DNA damage (Lee and Sharp, 2004; Jung and Lippard, 2005). It will be of interest to investigate how these different E3 ubiquitin ligases regulate both degradation-dependent and degradation-independent activities of pol II via promoting different lysine-linked polyubiquitination.

Unlike glycosylation that is inhibited by CTD phosphorylation, ubiquitination of pol II seems to depend on prior phosphorylation of the CTD, as evidenced by the observation that polyubiquitination of pol II in mammalian cells correlates with enhanced CTD phosphorylation upon DNA damage induced by UV or cisplatin that triggers the NER pathway (Bregman *et al.*, 1996) or by H₂O₂ that induces the BER pathway (Inukai *et al.*, 2004). That hyperphosphorylated pol II provides a signal for polyubiquitination is further supported by the finding that pol II ubiquitination was suppressed when a CTD kinase inhibitor H8, which blocks phosphorylation on serine5byCdk7 and

The General Transcription Machinery and General Cofactors
143
phosphorylation on serine 2 by Cdk9, was included in the ubiquitination reactions with phosphorylation on serine 2 by HeLa nuclear extracts (Mitsui and Sharp, 1999). Indeed, ubiquitination of pol II in vitro and in UV-irradiated mammalian cells by both pVHL and BRCA1/BARD1 RING finger-containing E3 ubiquitin ligase complexes was detected only when serine 5 of the CTD was phosphorylated (Kuznetsova et al., 2003; Starita et al., 2005), consistent with the result showing UV-induced association of hyperphosphorylated RPB1 with the CSA complex (Groisman et al., 2003). Nevertheless, experiments performed in vitro with yeast Rsp5 and human RPF1, both belonging to the HECT E3 ubiquitin ligase family, indicates a phosphorylation-independent interaction and ubiquitination of RPB1 (Huibregtse et al., 1997; Beaudenon et al., 1999), correlating with in vivo studies demonstrating that under UV-induced DNA damage in yeast, serine 5 dephosphorylation by CTD phosphatase Ssu72 makes pol II prone to polyubiquitination and degradation, whereas phosphorylation on serine 5 by Kin28 confers protection to pol II from ubiquitination (Somesh et al., 2005). While the requirement for CTD phosphorylation appears to vary depending on the species (yeast versus mammalian cells) and the type of E3 ubiquitin ligases involved, the CTD undoubtedly plays a central role in coordinating the signaling events induced by different DNA-damaging agents. It is important to note that the particular lysine residues linking polyubiquitin chain formation and the nature of DNA damage-independent transcriptional pausing induced by α-amanitin or during elongation also determine the functional outcome of a stalled pol II complex. To be or not to be degraded will surely be an important issue warranting further investigation for years to come.

### TFIIE

#### Structures and Functional Domains

After formation of a TFIID-TFIIB-pol II/TFIIF-promoter complex, the next step in the sequential PIC assembly pathway is the recruitment of TFIIE and TFIIH. TFIIE consists of two subunits, α and β, which form an α₂β₂ heterotetramer (Ohkuma et al., 1991; Sumimoto et al., 1991; Peterson et al., 1991). In humans, TFIIEα is the larger subunit consisting of 439 amino acids with a molecular weight of 56 kDa, while TFIIEβ is 291 amino acids with an approximate molecular weight of 34 kDa. The N-terminal half of TFIIEα is necessary for interactions with TFIIEβ and pol II via nonoverlapping regions (Figure 7C), for basal transcription, for stimulating TFIIH-mediated CTD phosphorylation, and for the transition from initiation to elongation. The C-terminal region, which appears to be nonessential in yeast, is involved in TFIIH interaction likely to facilitate TFIIH entry into the PIC (Ohkuma et al., 1995; Kuldell and Buratowski, 1997; Okuda et al., 2004). From amino acid sequence analysis, TFIIEα contains multiple structural features (Ohkuma et al., 1995; see Figure 7C), including a bacterial σ factor homology region (residues 13 to 49), a leucine repeat (residues 38 to 66), a zinc-finger motif (residues 121 to 157), an HTH motif (residues 159 to 182), two alanine-rich sequences (residues 221 to 240 and 335 to 349), and a C-terminal region rich in aspartic acids and glutamic acids (residues 378 to 393). The NMR structure of the zinc-finger motif present in a protease-resistant core between amino acids 113 and 174 of human TFIIEα is comprised of one α-helix and five β-strands, which is distinct from conventional zinc finger structures (Okuda et al., 2004). This unusual zinc-finger motif, with a central α-helix (H1) preceded by an antiparallel β-sheet (S1 and S2) and followed by three β-strands (S3–S5), is organized in the following order: β-strand (S1, residues 126 to 129), turn (t1, residues 130 to 133), β-strand (S2, residues 134 to 136), α-helix (H1, residues 138 to 144), β-strand (S3, residues 145 to 146), loop (residues 147 to 150), β-strand (S4, residues 151 to 154), turn (t2, residues 155 to 158), and a fifth β-strand (S5, residues 159 to 164). Binding of the Zn²⁺ ion is coordinated by Cys129, Cys132, Cys154, and Cys157, located adjacent to and within the two turns of this structure. This C₂C₂ zinc-finger is not involved in binding the small subunit of TFIIE, given that mutations introduced into this zinc-finger motif does not impair TFIIEα binding to TFIIEβ based on in vitro GST pull-down assays (Okuda et al., 2004). However, the zinc-finger motif of TFIIEα can enhance TBP binding to adenovirus major late and E4 TATA boxes, as evidenced by DNase I footprinting assays in which only wild-type TFIIEα, but not the C154A mutant, increases TATA recognition by TBP (Yokomori et al., 1998). The structural information for the HTH motif of TFIIEα is derived from its archaeal homolog. The TFE protein from archaeal bacteria with 178 amino acids shows sequence homology to the N-terminal half of TFIIEα (Bell et al., 2001), but lacks the C-terminal domain.

present in TFIIEα found in S. cerevisiae, C. elegans, D. melanogaster, X. laevis, and H. sapiens (Ohkuma et al., 1995; Kuldell and Buratowski, 1997). The winged HTH structure of TFE, resolved by x-ray crystallography at 2.8 Å (Meinhart et al., 2003), is composed of three α-helices and three β-strands in the following order: α1-β1-α2-α3-β2-β3. Similar to the winged HTH motif seen with RAP74 and RAP30, the “wing” of TFE is formed by a loop connecting the antiparallel β2 and β3 strands. However, the TFE wing does not appear to exhibit any DNA-binding activity, as otherwise observed with most proteins possessing this structural motif.

Similar to TFIIEα, human TFIIEβ also contains several functional domains (Okamoto et al., 1998; Okuda et al., 2000): an N-terminal serine-rich region (amino acids 26 to 71) able to enhance TFIIH-mediated phosphorylation of the CTD, a central core domain (residues 66 to 146) exhibiting a winged HTH structure (residues 75 to 139) capable of binding double-stranded DNA, a leucine repeat (residues 145 to 163) with unknown function, a σ3 region (residues 163 to 193) having sequence homology with the bacterial σ factor subdomain 3, and a C-terminal domain containing two basic regions in which the first one (residues 197 to 238) is a basic helix-loop-helix motif contacting TFIIEα and the Drosophila Krüppel transcription factor and the second one (residues 258 to 291) has a basic helix-loop sequence interacting with single-stranded DNA, pol II, TFIIB, the RAP30 subunit of TFIIF and Krüppel (see Figure 7C). The multiple protein-protein and protein-DNA interactions observed with the C-terminal domain of TFIIEβ may account for its involvement in the transition from transcription initiation to elongation by pol II (Watanabe et al., 2003). Since the N-terminal half of TFIIEα is also implicated in promoter clearance, it will be of interest to define whether the TFIIEα N-terminal region and the TFIIEβ C-terminal region function independently or cooperatively in aiding pol II transition from initiation to elongation. With respect to the structural analysis, the winged HTH motif in the central core domain (amino acids 66 to 146) of human TFIIEβ, revealed by NMR, is comprised of three α-helices and an antiparallel β-sheet (Okuda et al., 2000): H1 (residues 75 to 90), S1 (residue 97), H2 (residues 99 to 105), H3 (residues 113 to 120), turn (residues 121 to 125), β-strand (S2, residues 130 to 133), loop (residues 134 to 135), and a third β-strand (S3, residues 136 to 139), similar to other winged HTH motifs with double-stranded DNA-binding activity. The region between residues 79 and 111 has sequence homology with the pol II-binding domain of RAP30, although interaction between this region of TFIIEβ and pol II has yet to be demonstrated. It should also be noted that TFIIEα and TFIIEβ could interact with TFIIAγ and TFIIAαβ, respectively (Yokomori et al., 1998; Langelier et al., 2001; Yamamoto et al., 2001). Without doubt, the respective interaction domains need to be further defined.

### TFIIE Function

As seen with other GTFs, TFIIE may be recruited to the promoter through direct interaction with gene-specific transcriptional activators, such as the Krüppel zinc finger protein (Sauer et al., 1995) and the Antennapedia and Abdominal-B homeodomain proteins (Zhu and Kuziora, 1996). Once recruited, TFIIE interacts directly with both subunits of TFIIF, TFIIB, pol II, promoter DNA, and helps recruit TFIIH (Flores et al., 1989; Maxon et al., 1994; Okamoto et al., 1998; Yokomori et al., 1998; Watanabe et al., 2003; Forget et al., 2004). TFIIE binds to pol II near its active center and to the promoter DNA approximately 10 base pairs upstream from the transcription initiation site, where promoter melting begins (Douziech et al., 2000; Kim et al., 2000; Forget et al., 2004). TFIIE can stimulate the ATPase, CTD kinase, and DNA helicase activities of TFIIH and thus facilitates the formation of an initiation-competent pol II complex (Ohkuma and Roeder, 1994; Serizawa et al., 1994; Ohkuma et al., 1995; Lee and Young, 2000). TFIIE and TFIIH are also essential for promoter melting and the transition from initiation to elongation (Holstege et al., 1996). Consistent with TFIIE’s role in promoter melting is its ability to bind single-stranded DNA (Kuldell and Buratowski, 1997). Interestingly, the requirement of TFIIE and TFIIH for gene transcription also depends on DNA topology and the promoter sequence (Parvin and Sharp, 1993; Goodrich and Tjian, 1994; Wu et al., 1998), in agreement with the observation that TFIIE and TFIIH are not necessary for transcription from premelted promoter templates (Pan and Greenblatt, 1994, Holstege et al., 1996).

### TFIIH

#### Protein Composition

TFIIH is primarily recruited to the promoter region through association with TFIIE. Historically, TFIIH (also called BTF2; Gerard et al., 1991) is a multiprotein

The General Transcription Machinery and General Cofactors
complex consisting of nine subunits: p89/XPB (gene defective in xeroderma pigmentosum patients complementation group B), p80/XPD (gene defective in xeroderma pigmentosum patients complementation group D), p62, p52, p44, p40/CDK7, p38/Cyclin H, p34, and p32/MAT1. TFIIH was initially purified from rat liver (Conaway and Conaway, 1989), HeLa cells (Gerard et al., 1991; Flores et al., 1992), and yeast (Feaver et al., 1991a) by conventional column chromatography, and later by a single-step immun-affinity purification method (Kershner et al., 1998; LeRoy et al., 1998; Winkler et al., 1998). It has three enzymatic activities required for transcription: DNA-dependent ATPase (Conaway and Conaway, 1989; Feaver et al., 1991b; Roy et al., 1994), ATP-dependent helicase (Schaeffer et al., 1993; Serizawa et al., 1993; Drapkin et al., 1994), and CTD kinase (Feaver et al., 1991b; Lu et al., 1992; Serizawa et al., 1992). In addition to the enzymatic activities essential for transcription, some components of TFIIH (e.g., p89/XPB and p80/XPD) are involved in NER DNA damage response. Functionally, TFIIH can be separated into two subcomplexes: a cyclin-activating kinase complex (CAK) and a core complex. The CAK complex, responsible for phosphorylating pol II CTD, is consisted of CDK7, Cyclin H, and MAT1. The core complex contains XPB helicase, p62, p52, p44, and p34. CAK and core TFIIH are linked by the XPD helicase, which is essential for DNA repair activity of TFIIH but serves more or less a structural rather than an enzymatic role in transcription (Rossingnol et al., 1997; Coin et al., 1999). Mutations in either XPB or XPD lead to several human diseases, including xeroderma pigmentosum (XP), trichothiodystrophy (TTD), and Cockayne syndrome (CS) (Lee and Young, 2000; Lehmann, 2001; Zurita and Merino, 2003). Recent studies have identified a tenth subunit of TFIIH, TFB5, in both yeast (Ranish et al., 2004) and humans (Giglia-Mari et al., 2004). This small subunit (~8 kDa) has been implicated in the DNA repair function of TFIIH, as UV-irradiated yeast strains deficient in TFB5 exhibit a lower survival rate when compared with irradiated strains harboring wild-type TFB5 (Ranish et al., 2004). Similarly, human TFB5 is involved in NER DNA damage response, given that cells derived from TTD patients with group A disorder are unable to repair UV-induced DNA lesions (Giglia-Mari et al., 2004). This deficiency correlates with specific mutations in the gene coding for TFB5 and is likely caused by an increased degradation of TFIIH due

to a lack of the TFB5 stabilizing subunit (Giglia-Mari et al., 2004; Coin et al., 2006). From in vitro-reconstituted experiments, human TFB5 has recently been shown to stimulate the XPB ATPase activity of TFIIH and further enhance the opening of damaged DNA facilitated by the XPB and XPD subunits of TFIIH (Coin et al., 2006). It should be mentioned that many yeast core TFIIH components are also implicated in DNA damage response, as mutations in yeast genes encoding TFB1 (human p62; Matsui et al., 1995; Wang et al., 1995), TFB2 (human p52; Feaver et al., 1997), Ssl1 (human p44; Wang et al., 1995), and TFB4 (human p34; Feaver et al., 1999) also exhibit defects in responding to UV irradiation.

# DNA-Dependent ATPase Activity

The ATPase activity of TFIIH is required for transcription initiation and promoter clearance. Although an initial report indicated that TFIIH and ATP hydrolysis are only required for pol II promoter clearance, but not for transcription initiation and formation of the first phosphodiester bond in CpA dinucleotide-primed reactions (Goodrich and Tjian, 1994), subsequent studies found that TFIIH ATPase activity is indeed necessary for stable promoter opening and for first phosphodiester bond formation when natural nucleotide substrates were added with an excess amount of other GTFs, such as TFIIB, TFIIE, and RAP74 (that possess nonspecific DNA-binding activity), to initiate the transcription reactions (Holstege et al., 1996, 1997; Kumar et al., 1998). In general, without TFIIH, pol II tends to stall on the promoter-proximal region, leading to abortive transcription products; the addition of TFIIH in the presence of ATP significantly reduces the amount of the promoter-stalled pol II complex, indicating a direct involvement of TFIIH in promoter clearance (Dvir et al., 1997; Kugel and Goodrich, 1998; Kumar et al., 1998).

# ATP-Dependent Helicase Activity

TFIIH contains two helicases, XPB and XPD which unwind the DNA in a 3′ → 5′ and 5′ → 3′ direction, respectively, making TFIIH a bidirectional DNA helicase (Schaeffer et al., 1994). While XPB 3′ → 5′ helicase activity is critical for both DNA repair and transcription, the XPD 5′ → 3′ helicase activity is only required for DNA repair (Zurita and Merino, 2003). The

M. C. Thomas and C.-M. Chiang
XPB helicase activity is essential for promoter clearance, which typically occurs within formation of the first 20 nucleotides and defines the transition from initiation and elongation. Once promoter clearance has occurred, TFIIH is no longer needed (Goodrich and Tjian, 1994). Consistent with this view, the requirement for TFIIH in transcription may be bypassed via the use of either negatively supercoiled or pre-melted templates (Parvin and Sharp, 1993; Pan and Greenblatt, 1994; Parvin *et al.*, 1994; Tantin and Carey, 1994), suggesting a role of TFIIH also in open complex formation. Nevertheless, it is unclear which activity (helicase or ATPase) of TFIIH is involved in open complex formation. Earlier reports suggested that the XPB helicase activity is needed for promoter opening, as a helicase-impaired human XPB protein (K346R) with a mutation in the ATP-binding domain and a temperature-sensitive yeast strain expressing a helicase-defective mutant of Rad25 (a yeast homolog of human XPB) both exhibited defects, respectively, in melting a number of promoters in vitro and in vivo (Guzmán and Lis, 1999; Tirode *et al.*, 1999). With better defined XPB mutants (Q647A and T478A in the helicase motif) that selectively inactivate helicase activity without impairing its ATPase activity, it was later found that while Q647A and T478A are defective in promoter clearance, they are fully functional in promoter opening (Lin *et al.*, 2005). This result is consistent with site-specific protein-DNA photocrosslinking experiments showing that XPB, in the context of a complete PIC, was not detected at the promoter region undergoing melting (*i.e.*, transcription bubble), which extends initially from -9 to +2 (Holstege *et al.*, 1997), in the adenovirus major late promoter with or without ATP, but rather was found at DNA sequences (+3 to +25) downstream of the transcription bubble through contacts made exclusively with double-stranded DNA (Kim *et al.*, 2000). Thus, it is clear that the ATPase activity is involved in promoter opening and the helicase activity is crucial for promoter clearance. It is interesting to note that promoter clearance constitutes a regulatory step for some transcriptional regulators. The FUSE (Far Upstream Element)-binding protein (FBP) and FBP-interacting repressor (FIR), both playing an important role in modulating cell growth and differentiation in humans via regulating c-Myc gene transcription, can respectively stimulate or inhibit the helicase activity of XPB and thereby regulate transcription at the promoter clearance step (Liu *et al.*, 2000; Liu *et al.*, 2001a).

---

### TFIIH and Nucleotide Excision Repair

NER is a process where damaged DNA is removed and replaced by newly synthesized DNA based on sequence information from the intact template strand. The finding that p89/XPB is identical to ERCC3, a DNA excision repair protein that is mutated in XP patients, led to the hypothesis that transcription might be coupled to DNA repair (Schaeffer *et al.*, 1993). Consistent with a dual role of TFIIH in transcription and DNA repair is the observation that transcriptionally active genes are preferentially repaired (Bohr *et al.*, 1985; Mellon and Hanawalt, 1989). For NER, the combined helicase activities of XPB/ERCC3 and XPD/ERCC2 seem to be required. Experiments have shown that microinjection of TFIIH into human XPD-or XPB-mutant cells led to complementation of the repair-deficient phenotype (van Vuuren *et al.*, 1994). Similarly, yeast cells with mutations in the XPD homolog (Rad3) were rescued by the addition of TFIIH but not by XPD alone, suggesting that the NER pathway requires a functional TFIIH (Wang *et al.*, 1994). Subsequent studies have shown that multiple components in TFIIH are required for DNA repair, including XPB, XPD, p62, p52, and p44 (Drapkin *et al.*, 1994; Humbert *et al.*, 1994; Schaeffer *et al.*, 1994; Wang *et al.*, 1995; Jawhari *et al.*, 2002). The XPB and XPD helicase functions are required for transcription-coupled NER, as defects in helicase activity are linked to human diseases including XP, TTD, and CS. Recent studies have unraveled the mechanism by which the XPB helicase subunit of TFIIH functions in NER and transcription. Experiments showing phosphorylation of the serine 751 residue of XPB leads to inhibition of NER activity, but does not prevent TFIIH from unwinding DNA (Coin *et al.*, 2004). Instead, phosphorylation of XPB serine 751 prevents the 5′ incision triggered by the ERCC1-XPF endonuclease (Coin *et al.*, 2004), providing convincing evidence that a separate but essential role of TFIIH is involved in both transcription and DNA repair.

---

### Ubiquitin Ligase Activity

While XPB and XPD are directly involved in DNA repair, the yeast homolog of human p44 (Ssl1) in the TFIIH core complex has been implicated in DNA damage response likely by modulating the expression of proteins involved in the NER and BER pathways. A ubiquitin ligase activity of Ssl1 has recently been uncovered due to the presence of a RING finger domain.

at its C-terminal region spanning residues 403 to 454, whose corresponding domain is present in human p44 (Takagi et al., 2005). The NMR structure of the p44 RING finger domain (residues 321 to 395) shows two zinc-binding sites coordinated by eight cysteine residues (C345, C348, C360, C363, C368, C371, C382, and C385; Kellenberger et al., 2005). These conserved cysteine residues are also found in the RING finger domain of Ssl1 and are critical for its E3 ubiquitin ligase activity, since mutations in the first two cysteine residues (C403A and C406A) abolished Ssl1 enzymatic activity in an *in vitro* polyubiquitination assays performed in the presence of El and E2 (Ubc4) enzymes and further reduced the yeast survival rate following UV irradiation or methyl methanesulfonate (MMS) treatment (Takagi et al., 2005). The Ssl1 ubiquitin ligase activity can be enhanced by the inclusion of another RING finger-containing protein Tfb4 (the yeast counterpart of human MAT1), similar to BARD1-stimulated BRCA1 E3 ubiquitin ligase activity described earlier (see Pol II Ubiquitination section), or by the presence of other core TFIIH subunits, reminiscent of Rbx1 in the pVHL complex. However, whether TFIIH can polyubiquitinate pol II, in UV-irradiated cells at the sites of DNA lesions, or other protein components involved in transcription and DNA repair pathways remains to be explored.

### TFIIH and CTD Phosphorylation

CDK7 is the kinase responsible for phosphorylating the serine 5 residue of the pol II CTD, whose activity is regulated by cyclin H, MAT1, TFIIIE, Mediator (Svejstrup et al., 1996), XPD (Keriel et al., 2002), and U1 small nuclear RNA (snRNA; O’Gorman et al., 2005). The CDK7-cyclin H-MAT1 CAK complex in the context of TFIIH has higher activity in phosphorylating the CTD compared with the free form of CAK (Yankulov and Bentley, 1997). Phosphorylation of serine 5 leads to the recruitment of 5′ capping enzyme (Cho et al., 1997; Komarnitsky et al., 2000; Rodriguez et al., 2000; Schroeder et al., 2000; Pei et al., 2001) and is implicated in promoter clearance. That CTD phosphorylation regulates the transition from transcription initiation to elongation is supported by observations that pol II enters PIC assembly as the hypophosphorylated IIA form and escapes the promoter as the hyperphosphorylated IIO form (Hampsey, 1998). Besides phosphorylating CTD, TFIIH has been shown to phosphorylate transcriptional activators, such as p53 (Lu et al.,

1997), retinoic acid receptor α (Rochette-Egly et al., 1997), retinoic acid receptor γ (Bastien et al., 2000), Ets-1 (Drane et al., 2004), estrogen receptor α (Chen et al., 2000), and general cofactor PC4 (Kershner et al., 1998). The CTD kinase activity of TFIIH can also be stimulated via interaction with transcriptional activators (Jones, 1997).

#### TFIIH-Activator Interactions

Many activators have been shown to interact with TFIIH including Gal4-VP16, E2F1, Rb, p53, ERα, RARα, RARγ, and androgen receptor (reviewed by Zurita and Merino, 2003). Consistent with TFIIH’s ability to interact with multiple activators is the finding that TFIIH can function as a coactivator in a reconstituted cell-free transcription system (Wu et al., 1998). Perhaps activators work by enhancing the recruitment of TFIIH for PIC assembly or stimulating the enzymatic activities of TFIIH (Liu et al., 2001a; Zurita and Merino, 2003). Conversely, TFIIH may covalently modify amino acid residues critical for activator function. For example, TFIIH has been shown to stimulate the transcriptional activity of the N-terminal activation domain (AF-1) of nuclear receptors RARα1 and RARγ via phosphorylation of specific serine residues in AF-1 (Rochette-Egly et al., 1997; Bastien et al., 2000). The kinase activity of TFIIH, which resides at its CDK7 subunit, appears to be regulated by XPD, as mutations in the XPD subunit impair TFIIH’s ability to phosphorylate RARα (Keriel et al., 2002). Accordingly, XPD within the TFIIH complex can indirectly regulate nuclear receptor phosphorylation and transactivation activity and thus may account for its involvement in the transcriptional process besides its well-documented role in DNA repair. The cyclin H subunit of TFIIH is another target of the kinase module prone to regulation by other regulatory molecules. The U1 snRNA, involved in recognition of the 5′ splice donor site, is able to stimulate TFIIH CTD kinase activity via interaction with cyclin H (O’Gorman et al., 2005), and also enhance first phosphodiester bond formation as well as transcription reinitiation (Kwek et al., 2002). The finding that U1 snRNA can stimulate CDK7 kinase activity (O’Gorman et al., 2005) and 7SK snRNA is inhibitory for CDK9 kinase activity (Nguyen et al., 2001; Yang et al., 2001) further implicates an important role of noncoding RNAs in modulating the transcriptional events through CTD kinases. Undoubtedly, this will be an interesting area for future studies.

FIGURE 9 General cofactors serve as molecular bridges in activator-dependent transcription. General cofactors (TAFs, Mediator, and USA) are required for transducing signals between gene-specific activators and components of the general transcription machinery. An activator normally contains a DNA-binding domain (DBD) contacting specific DNA sequences and an activation domain (AD) interacting with general cofactors or with components of the general transcription machinery. It should be noted that TAFs normally function as an integral part of TFIID, not as a free entity in mammalian cells as drawn here.

---

### GENERAL COFACTORS

Three classes of general cofactors are typically involved in gene activation to facilitate the communication between gene-specific transcription factors and components of the general transcription machinery (Figure 9). These general cofactors include TAFs found in TFIID, Mediator frequently associated with the CTD, and upstream stimulatory activity (USA)-derived positive cofactors (PC1, PC2, PC3, and PC4) and negative cofactor 1 (NC1). Similar to TAFs, some forms of Mediator and USA-derived components are capable of repressing basal transcription when activators are absent, and stimulating transcription in the presence of activators. Although other general cofactors implicated in transcription from higher-order chromatin structures, such as histone-modifying enzymes and chromatin-remodeling complexes, also exist, we will focus our discussion here on Mediator and USA-derived components originally defined in activator-regulated transcription using both genetic systems and nucleosome-free DNA templates.

---

### MEDIATOR COMPLEXES

In addition to TFIID, Mediator represents the second class of general cofactors that transmit the regulatory signals from gene-specific transcription factors to the general transcription machinery (Björklund and Gustafsson, 2005; Kornberg, 2005). Mediator was first identified in yeast and found to consist of approximately 20 polypeptides (Kim *et al.*, 1994), of which 11 are essential for yeast viability (Rgr1, Rox3, Srb4, Srb6, Srb7, Med4, Med6, Med7, Med8, Med10/Nut2, and Med11) (Myers and Kornberg, 2000). While nine of the Mediator components were originally defined by genetic screens as proteins interacting with the CTD of pol II (*i.e.*, Srb2 and Srb4-11 for different dominant suppressors of RNA polymerase B; Nonet and Young, 1989; Thompson *et al.*, 1993), later biochemical purification of Mediator complexes from various species has identified additional conserved as well as species-specific subunits, for which a unified nomenclature has been proposed (Table 3; Bourbon *et al.*, 2004; Björklund and Gustafsson, 2005; Blazek *et al.*, 2005; Conaway *et al.*, 2005; Kim and Lis, 2005; Malik and Roeder, 2005).

---

The General Transcription Machinery and General Cofactors

TABLE 3 Mammalian and yeast Mediator complexes

| New name | S. cerevisiae | TRAP/SMCC (Large) | ARC/DRIP (Large) | CRSP (Small) | PC2 (Small) | Mediator-P.5 (Large) | Mediator-P.85 (Small) |
|----------|---------------|-------------------|------------------|--------------|--------------|-----------------------|------------------------|
| MED 1    | Med 1         | TRAP 220          | ARC/DRIP 205     | CRSP 200     | TRAP 220     | Med 220               | Med 220                |
| MED 2    | Med 2         |                   |                  |              |              |                       |                        |
| MED 3    | Med 3/Pgd1/Hrs1 |                   |                  |              |              |                       |                        |
| MED 4    | Med 4         | TRAP 36           | ARC/DRIP 36      |              | TRAP 36      | Med 36                | Med 36                 |
| MED 5    | Nut1          |                   |                  |              |              |                       |                        |
| MED 6    | Med 6         | hMed 6            | ARC/DRIP 33      |              | hMed 6       | Med 33                | Med 33                 |
| MED 7    | Med 7         | hMed 7            | ARC/DRIP 34      | CRSP 33      | hMed 7       | Med 34                | Med 34                 |
| MED 8    | Med 8         |                   | ARC 32           |              |              | Med 31                | Med 31                 |
| MED 9    | Med 9/Cse2    |                   |                  |              |              |                       |                        |
| MED 10   | Med 10/Nut2   | hNut 2            | hMed 10          |              | hNut 2       |                       |                        |
| MED 11   | Med 11        |                   |                  |              |              |                       |                        |
| MED 12   | Srb8          | TRAP 230          | ARC/DRIP 240     |              |              | Med 230               |                        |
| MED 13   | Srb9/Ssn2     | TRAP 240          | ARC/DRIP 250     |              |              | Med 240               |                        |
| MED 14   | Rgr1          | TRAP 170          | ARC/DRIP 150     | CRSP 150     | TRAP 170     | Med 150               | Med 150                |
| MED 15   | Gal11         |                   | ARC 105          |              | PCQAP        | Med 105               | Med 105                |
| MED 16   | Sin4          | TRAP 95           | DRIP 92          |              | TRAP 95      | Med 95                | Med 95                 |
| MED 17   | Srb4          | TRAP 80           | ARC/DRIP 77      | CRSP 77      | TRAP 80      | Med 78                | Med 78                 |
| MED 18   | Srb5          |                   |                  |              |              |                       |                        |
| MED 19   | Rox3          |                   |                  |              |              |                       |                        |
| MED 20   | Srb2          | hTRFP             |                  |              | hTRFP        |                       |                        |
| MED 21   | Srb7          | hSrb7             | hSrb7            |              |              |                       |                        |
| MED 22   | Srb6          |                   |                  |              |              |                       |                        |
| MED 23   |               | TRAP150β          | ARC/DRIP 130     | CRSP 130     | TRAP150β     |                       |                        |
| MED 24   |               | TRAP 100          | ARC/DRIP 100     | CRSP 100     | TRAP 100     | Med 100               | Med 100                |
| MED 25   |               |                   | ARC 92           |              |              |                       |                        |
| MED 26   |               |                   |                  | CRSP 70      |              | Med 70                |                        |
| MED 27   |               | TRAP 37           |                  | CRSP 34      | TRAP 37      |                       |                        |
| MED 30   |               | TRAP 25           |                  |              |              |                       |                        |
| MED 31   |               | hSoh1             |                  |              | hSoh1        |                       |                        |
| CDK8     | Srb10/Ume5/Ssn3 | hSrb10           |                  | CDK8         |              | CDK8                  |                        |
| CycC     | Srb11/Ume3/Ssn8 | hSrb11           |                  | CycC         |              | CycC                  |                        |

The identified protein composition of human Mediator-P.5 and Mediator-P.85 complexes (grey shaded areas) is described in Wu et al. (2003). Table adapted from Bourbon et al. (2004).

Isolation of Mediator Complexes

Human Mediator, first purified from HeLa cells as a protein complex that associates with the thyroid hormone receptor α (TRα) in a ligand-dependent manner, was able to potentiate TRα-mediated transcription in vitro (Fondell et al., 1996). This TRα-associated protein complex (TRAP) contains many protein subunits subsequently found also present in other coactivator complexes, such as SRB/MED-containing cofactor complex (SMCC; Ito et al., 1999), vitamin D receptor-interacting protein complex (DRIP; Rachez et al., 1998), activator-recruited complex (ARC; Nair et al., 1999), positive factor 2 (PC2; Malik et al., 2000), cofactor re-quired for Sp1 activation (CRSP; Ryu et al., 1999), and negative regulator of activated transcription (NAT; Sun et al., 1998). In humans, at least two forms of Mediator complexes, Mediator-P.5 and Mediator-P.85 isolated individually at 0.5M and 0.85M KCl fractions of the P11 phosphocellulose ion-exchange column, have been identified and demonstrated to enhance activator-dependent and basal transcription, respectively, in a highly purified transcription system reconstituted with recombinant GTFs and FLAG-tagged multiprotein complexes (Wu et al., 2003). Mediator-P.5 represents a class of larger Mediator complexes, including TRAP/SMCC and ARC/DRIP, which contain a dissociable MED12-MED13-CDK8-CycC

module not found in the smaller Mediator complexes, such as Mediator-P.85, CRSP and PC2 (see Table 3). In contrast, the smaller Mediator complex has a unique polypeptide, MED26/CRSP70, normally absent in the larger complex. The rest of Mediator components seem to be commonly shared between large and small Mediator complexes, although the identities of some subunits remain to be characterized. The structures of yeast and murine Mediator as well as human TRAP, ARC and CRSP complexes have been resolved by electron microscopy at 30 to 40 Å (Asturias et al., 1999; Dotson et al., 2000; Taatjes et al., 2002). Comparison of these structures reveals a similarity in the overall organization of Mediator complexes. In general, three visible domains (named “head,” “middle,” and “tail” in yeast Mediator) that may adapt to distinct conformations when in complex with activators or the CTD are distinguishable (Blazek et al., 2005; Chadick and Asturias, 2005; Conaway et al., 2005).

the Rpb1, Rpb3, Rpb6, and Rpb11 subunits of pol II with regions of Mediator extending from the head module to the intersection between middle and tail modules (Davis et al., 2002). These extensive contacts may help Mediator unfold from its compact triangular shape, in which the middle and tail modules are not clearly visible in the absence of pol II, to a more extended conformation when bound to pol II. Despite the extended contacts, the DNA-binding cleft and interaction surfaces for other components of the general transcription machinery are still accessible on pol II (Davis et al., 2002). Since MED1 in the middle module can interact with multiple nuclear receptors (Yuan et al., 1998) and MED14 and MED15 in the tail module can associate with Gal4-VP16 (Lee et al., 1999; Park et al., 2000), it is likely that conformational changes induced upon activator binding to the middle and, especially the tail, modules further contribute to activator-facilitated recruitment of pol II by Mediator (Taatjes et al., 2002).

### Head Module

The head module of Mediator, consisting of MED6, MED8, MED11, MED17, MED18, MED19, MED20, and MED22, forms the base of a roughly triangle-shaped Mediator complex. This triangular complex undergoes a drastic conformational change upon association with pol II, resulting in an arc-shaped structure in which the head module at the leading edge serves as a major docking site for pol II. This structural information is consistent with yeast genetic screens showing direct interaction between pol II and MED17/Srb4, MED18/Srb5, MED20/Srb2 and MED22/Srb6, respectively (Thompson et al., 1993). Moreover, MED17 and MED18 also interact with transcriptional activators p53 and Gal4-VP16, respectively (Ito et al., 1999; Lee et al., 1999), suggesting that activator-induced conformational changes (Taatjes et al., 2002) may further enhance head module-mediated recruitment of pol II.

### MED12-MED13-CDK8-CycC Module

A dissociable module, which contains MED12/Srb8, MED13/Srb9, CDK8/Srb10 and CycC/Srb11 normally found in the large, but not small, Mediator complex (Borggrefe et al., 2002; Samuelsen et al., 2003), seems to contact mainly the middle module, via CDK8 interaction with MED1 and MED4 (Kang et al., 2001), and also the head module, through MED13 interaction with MED17 (Guglielmi et al., 2004). This CDK8 module, which could be isolated as a free entity, is able to phosphorylate serines 2 and 5 of the pol II CTD (Borggrefe et al., 2002). That transcription could be inhibited by CDK8-mediated phosphorylation of the CTD occurring prior to PIC assembly (Hengartner et al., 1998) or through phosphorylation on serines 5 and 304 of the cyclin H subunit of TFIIH by recombinant CDK8-CycC pair or by the NAT complex (Akoulitchev et al., 2000) suggests that the CDK8 module may function as a repression module in the context of Mediator. This view is supported by biochemical evidence showing that the large form of ARC (ARC-L) is transcriptionally inactive (Taatjes et al., 2002) and the coactivating activity of Mediator-P.5 was slightly enhanced when CDK8-CycC was immunodepleted from the large Mediator complex (Wu et al., 2003). Clearly, removal of CDK8 in yeast by nutrient deprivation (Holstege et al., 1998) or in mouse P19 embryonic carcinoma cells following all-trans retinoic acid (tRA) treatment (Pavri et al.,

### Middle and Tail Modules

The middle module of Mediator contains MED1, MED4, MED5, MED7, MED9, MED10, MED21, and MED31, whereas the tail module includes MED2, MED3, MED14, MED15, and MED16 (Boube et al., 2002; Guglielmi et al., 2004). From the structures of yeast Mediator, it is obvious that, besides head module-pol II interaction, additional contacts are made between

### The General Transcription Machinery and General Cofactors
2005) enhances transcription from a subset of cellular genes normally suppressed by CDK 8. In the latter case, it was further demonstrated that dissociation of CDK 8 following tRA treatment converts Mediator from a transcriptionally suppressed state to an activated complex at the tRA-targeted RARβ2 gene promoter, indicating that CDK 8 indeed functions in the context of a repression module *in vivo*.

### Functional Properties of Mediator

In addition to the inhibitory activity conferred by the CDK 8 repression module, Mediator is an authentic coactivator able to stimulate both basal (Mittler *et al.*, 2001; Wu *et al.*, 2003) and activator-dependent transcription (Kim *et al.*, 1994). This stimulating activity of Mediator clearly relies on its ability to serve as a bridging molecule in transducing activation signals typically from activator-tail module to head module-pol II. Although Mediator can be isolated via its direct interactions with activators or with the CTD, direct biochemical evidence demonstrating that Mediator indeed functions by enhancing activator-facilitated entry of pol II to the PIC has only become possible after all the general transcription factors, cofactors, and pol II are available in purified forms devoid of any contaminating activities (Wu *et al.*, 2003). From this *in vitro*-reconstituted transcription system, we learn that the large form of Mediator complexes, such as Mediator-P.5, has intrinsic coactivating activity able to stimulate activator-dependent transcription, whereas the small form of Mediator complexes, such as Mediator-P.85, only enhances basal transcription. Interestingly, the coactivator function of Mediator can occur in the absence of TFIID TAFs, suggesting that Mediator and TAFs may play some redundant roles in the transcriptional process. Indeed, it has been shown that TAFs can also enhance pol II entry to the PIC in the presence of transcriptional activators (Wu and Chiang, 2001a). Besides targeting pol II, Mediator has the ability to enhance TBP binding to the TATA box. This TATA-enhancing activity may help stabilize the promoter-bound scaffold complex, which contains TFIIA, TFIID, TFIIE, TFIIH and Mediator (Yudkovsky *et al.*, 2000), to facilitate reinitiation of transcription from the same promoter.

As found in TFIID, Mediator also exhibits multiple enzymatic activities. The kinase activity of Mediator, inherent to the CDK 8 subunit, can phosphorylate the CTD of pol II (Hengartner *et al.*, 1998; Sun

etal., 1998; Borggrefe *et al.*, 2002), the cyclin H subunit of TFIIH (Akoulitchev *et al.*, 2000), and general cofactor PC 4 (Gu *et al.*, 1999). In addition, yeast Mediator has been reported to exhibit HAT activity, residing in the MED 5/Nut1 subunit, that preferentially acetylates histones H3 and H4 in the context of both free core histones and chromatin (Lorch *et al.*, 2000). However, no HAT activity has been reported in Mediator complexes purified from other species to date. Interestingly, human MED 8 can associate with Elongin B, Elongin C, Cullin 2, and Rbx1 to form an E3 ubiquitin ligase complex able to assemble polyubiquitin chains *in vitro* in conjunction with Uba1 and UbcH5a (Brower *et al.*, 2002), similar to the pVHL complex described earlier that is capable of polyubiquitinating pol II (see Pol II Ubiquitination section). This finding suggests that MED 8 may serve as a substrate recognition component in this MED 8-containing ubiquitin ligase complex and functionally links the ubiquitin machinery to Mediator. It would be interesting to identify the natural substrates ubiquitinated by this MED 8-containing ubiquitin ligase complex. Collectively, these enzymatic activities may contribute to Mediator functions in the transcriptional process, including a possible stimulation of TFIIH kinase activity on the CTD (Kim *et al.*, 1994). Many of these intriguing questions remain to be addressed.

---

### USA-DERIVED COFACTORS

The third class of general cofactors, USA, was initially defined as a crude fraction derived from the P110.85 M KCl fraction of HeLa nuclear extracts able to stimulate activator-dependent transcription (Meisterernst *et al.*, 1991). Upon further fractionation of the USA fraction (Figure 10), several positive cofactors (PC1, PC2, PC3 and PC4) and a negative cofactor (NC1) were identified (Kaiser and Meisterernst, 1996). With ongoing research in the study of USA-derived cofactors, it is remarkable to see that each cofactor has a dual role to function as a coactivator in potentiating activator-dependent transcription and also as a repressor in inhibiting basal transcription when the activator is absent.

#### PC1

PC1 is a nuclear protein that is the functional equivalent of poly(ADP-ribose) polymerase-1 (PARP-1; Meisterernst *et al.*, 1997), an enzyme which is well studied for its role in DNA repair by binding to damaged

M. C. Thomas and C.-M. Chiang
Nuclear Extract
↑
P11

(Fraction A) 0.1M
↓
B-TFIID
(TBP+BTAF1)

(Fraction B) 0.3M
↓
NC2

(Fraction C) 0.5M

(Fraction D) 0.85M
↓
DE52
↓
USA
↓
TFIID

NC1 (HMG1)
PC1 (PARP-1)
PC2 (Mediator)
PC3 (Topo I)
PC4

FIGURE 10 Purification scheme for USA, NC2, TFIID, and B-TFIID. Fractionation of HeLa nuclear extract and the molar concentrations of KCl used for column chromatography are indicated.

DNA and in nucleic acid metabolism by covalently modifying proteins involved in these pathways and also for its involvement in the maintenance of chromatin structure (Lindahl et al., 1995; D'Amours et al., 1999). These properties of PARP-1 explain why TFIIC (i.e., PARP-1) only becomes necessary for site-specific initiation of transcription by pol II when nicked DNA templates were used for in vitro transcription assays, presumably due to suppression of nonspecific initiation from damaged DNA by TFIIC binding to the nicks (Slattery et al., 1983).

Mammalian PARP-1, with a molecular size of approximately 114 kDa, catalyzes the transfer of ADP-ribose units from the donor nicotinamide adenine dinucleotide (NAD⁺) to acceptor proteins, in a process known as poly(ADP-ribosyl)ation (Lindahl et al., 1995; D'Amours et al., 1999; Kraus and Lis, 2003; Kim et al., 2005b). Not only does PARP-1 modify target proteins mediating DNA damage response, nucleic acid metabolism and chromatin dynamics, it also modifies transcription components, such as PC3 (i.e., DNA topoisomerase I), high mobility group protein 1 (HMG1), histones (H1, H2A, H2B, H3, H4 and H5), TBP, pol II, TFIIF (RAP30 and RAP74), p53, YY1, Sp1, and several protein components (TLE1, TopoIIβ, Rad50, nucleolin, and nucleophosmin) in the TLE1 corepressor complex (Rawling and Alvarez-Gonzalez, 1997; Oei et al., 1998; D'Amours et al., 1999; Ju et al., 2004). Moreover, ADP-ribose units can be transferred to PARP-1 itself. Auto(ADP-ribosyl)ation of PARP-1 is mediated through its central domain containing multiple glutamic acid residues, which serve as acceptor sites for poly(ADP-ribosyl)ation. This central domain links the N-terminal zinc-finger DNA-binding domain to the C-terminal NAD⁺-binding catalytic domain (D'Amours et al., 1999; Kraus and Lis, 2003). When disrupted, as observed during apoptosis where PARP-1 is cleaved by caspase death enzymes to release a 24-kDa N-terminal DNA-binding fragment and an 89-kDa C-terminal catalytic fragment, PARP-1 typically loses its enzymatic activity. For years, PARP-1 cleavage has been used as a diagnostic marker for programmed cell death.

Unlike the well-documented roles of PARP-1 in DNA repair and apoptosis, the transcriptional role of PARP-1 has not been extensively studied (Kim et al., 2005b). Obviously, the coactivator function of PARP-1 is mediated by its direct contact with distinct transcriptional activators, including the human T-cell leukemia virus type 1 Tax protein (Anderson et al., 2000), human papillomavirus type 18 E2 (Lee et al., 2002a), B-Myb (Cervellera and Sala, 2000), E2F-1 (Simbulan-Rosenthal et al., 2003), AP2 (Kannan et al., 1999), TEF-1 (Butler and Ordahl, 1999), RARα (Pavri et al., 2005), TRα (Pavri et al., 2005), and NFκB (Hassa et al., 2003), and also with other transcriptional cofactors, such as p300, HDAC1-3, and components (Cdk8 and MED14) of Mediator (Hassa et al., 2005; Pavri et al., 2005). Since the C-terminal domain of PARP-1 is necessary for TEF-1- and NFκB-dependent transcription (Butler and Ordahl, 1999; Hassa et al., 2003), but appears dispensable for Gal4-AH-mediated activation (Meisterernst et al., 1997), it seems that the catalytic activity of PARP-1 is

The General Transcription Machinery and General Cofactors
differentially required for gene activation, depending  
on the specific activators and promoters examined. Not  
surprisingly, the coactivator function of PARP-1 is fur-  
ther modulated by posttranslational modifications, as  
phosphorylation of PARP-1 by CaMKII and acetyla-  
tion of PARP-1 by p300 lead to HES1-and NFkB-  
mediated gene activation, respectively (Ju et al., 2004;  
Hassa et al., 2005). Using a cell-free transcription sys-  
tem performed with a Gal4-driven DNA template, it  
was shown that PARP-1 could stimulate PIC formation  
at a step post TFIID binding to the promoter region  
(Meisterernst et al., 1997). However, the exact step reg-  
ulated by PARP-1 during PIC assembly has not been  
elucidated.  

Other than the coactivator function, PARP-1 also  
possesses repressing activity able to inhibit transcrip-  
tion through different mechanisms. First, PARP-1 can  
be incorporated into chromatin via its N-terminal  
DNA-binding domain to promote formation of a  
highly condensed chromatin structure, thereby inhibi-  
ing activator-dependent transcription (Kim et al., 2004).  
Second, PARP-1 can be incorporated into a corepres-  
sor complex containing many protein components  
susceptible to modification by PARP-1, thus regulating  
the integrity of the corepressor complex on the target  
gene (Ju et al., 2004). Third, PARP-1 is capable of  
inhibiting ligand-dependent transcription by TRα in  
transient reporter gene assays in a catalytic domain-  
dependent manner (Miyamoto et al., 1999), suggesting  
that poly(ADP-ribosyl)ation of critical transcription  
components is likely involved in PARP-1-mediated  
transcriptional repression. This is consistent with the  
observation that poly(ADP-ribosyl)ation of sequence-  
specific DNA-binding proteins, such as p53, YY1, Sp1  
and TBP, prevents binding to their cognate sequences  
(Malanga et al., 1998; Oei et al., 1998; Mendoza-Alvarez  
and Alvarez-Gonzalez, 2001). In the case of p53, the  
sites for poly(ADP-ribosyl)ation have been mapped  
to the DNA-binding and oligomerization domains of  
p53 (Malanga et al., 1998). Interestingly, the enzymatic  
activity of PARP-1 is also critical for derepression to  
occur on some silenced loci, as seen by the association  
of PARP-1 and poly(ADP-ribosyl)ated proteins with  
decondensed chromatin structure at transcriptionally  
induced *Drosophila* polytene chromosome puffs (Tulin  
and Spradling, 2003). Clearly, PARP-1 can function  
as a molecular switch to convert a silenced gene into  
a transcriptionally active state, by first dissociating a  
corepressor complex via poly(ADP-ribosyl)ation (Ju  

---

etal., 2004) or by removing a CDK8 repression module  
from the larger Mediator complex (Pavri et al., 2005),  
and then enhancing activator-facilitated recruitment of  
chromatin-modifying enzymes, such as CBP or p300  
HAT or ATP-dependent chromatin remodelers, to the  
targeted promoters.  

### PC2  

PC2 (also termed USA100; Meisterernst et al., 1991),  
originally isolated from HeLa nuclear extracts as a pro-  
tein complex with a molecular size around 500 kDa  
(Kretzschmar et al., 1994), corresponds to the smaller  
form of human Mediator complexes described earlier  
in the Mediator section. This protein complex can en-  
hance transcription mediated by Gal4-AH and HNF4  
in an *in vitro*-reconstituted transcription system with  
DNA templates (Malik et al., 2000). The coactivator  
function of PC2 seems to require the presence of other  
general cofactors, since PC2 alone only weakly stimu-  
lates activator-dependent transcription and in conjunc-  
tion with TAFs, PC3 and PC4, a synergistic activation  
of Gal4-VP16-mediated transcription recapitulating the  
stimulatory activity of the USA fraction is detected  
(Malik et al., 2000). Likewise, PC2 is able to function  
together with PC3 and PC4 to regulate transcription  
mediated by TRα (Fondell et al., 1999), NFκB, Sp1  
(Guerma et al., 1998), and the B-cell-specific Oct-1-  
OCA-B transcription complex (Luo et al., 1998).  

### PC3  

PC3, functionally equivalent to DNA topoisomerase  
I (Topo I; Kretzschmar et al., 1993; Merino et al., 1993),  
consists of 765 amino acids with a molecular size around  
91 kDa (Leppard and Champoux, 2005). This protein  
is well known for its role in relaxing DNA by tran-  
siently introducing nicks, allowing strand passage and  
religation (Wang, 2002). Topo I has an unstructured N-  
terminal region containing nuclear targeting signals and  
surfaces for interaction with various transcription fac-  
tors, such as p53 (Gobert et al., 1999). This N-terminal  
region, although important for nuclear localization and  
protein-protein interaction, is dispensable for relaxing  
DNA *in vitro* (Stewart et al., 1996). The enzymatic ac-  
tivity of Topo I is regulated posttranslationally by CK2-  
and protein kinase C-mediated phosphorylation, lead-  
ing to increased relaxation activity and in responding to  
mitogenic stimuli (Wang, 1996). When Topo I encoun-  
ters damaged DNA, it is stalled and must be removed  

M. C. Thomas and C.-M. Chiang  
from the lesions in order to prevent formation of irreversible single- or double-strand breaks that impair genomic integrity (Leppard and Champoux, 2005). In this regard, PARP-1 can target Topo I for poly(ADP-ribosyl)ation and facilitate Topo I to remove itself from cleaved DNA and close the resulting gap (Malanga and Althaus, 2004).

Aside from its role in modulating DNA topology, Topo I can function as a transcriptional coactivator by contacting directly with activators, such as Gal4-AH (Kretzschmar et al., 1993), AP2 (Kannan et al., 1999), p53 (Gobert et al., 1999), and c-Jun (Mialon et al., 2005), and also with components of the general transcription machinery, such as TBP (Merino et al., 1993), in order to stimulate transcription. The coactivator function of Topo I is in part due to its ability to enhance TFIID-TFIIA-promoter complex formation (Shykind et al., 1997). In the absence of activator, Topo I functions as a repressor in inhibiting basal transcription. Although the transcription activity of Topo I is separated from its DNA relaxation activity (Merino et al., 1993; Kretzschmar et al., 1993), the precise mechanism by which Topo I regulates transcription remains to be elucidated. This is an area currently underexplored.

### PC4

PC4 is a protein consisting of 127 amino acids with an N-terminal regulatory domain spanning amino acids 1 to 62 and a C-terminal single-stranded DNA-binding and dimerization domain located between amino acids 63 and 127 (Ge and Roeder, 1994a; Kretzschmar et al., 1994; Brandsen et al., 1997). The N-terminal region, important for PC4 interaction with distinct activation domains (Ge and Roeder, 1994a) and binding to double-stranded DNA in a non-sequence-specific manner (Kaiser et al., 1995), contains two serine-enriched acidic (SEAC) domains located respectively at amino acids 9 to 22 and 50 to 61, separated by a lysine-rich region lying between amino acids 23 to 41 (Kaiser et al., 1995). Both SEAC domains are susceptible to phosphorylation by several protein kinases, in particular CK2, which lead to inactivation of the coactivator function of PC4 likely by preventing PC4 interaction with TBP-bound TFIIA on the promoter region (Ge et al., 1994; Kretzschmar et al., 1994) and with transcriptional activators, such as the HIV Tat protein (Holloway et al., 2000). The lysine-rich region, proposed

to bind nonspecific double-stranded DNA (Kaiser et al., 1995), may serve as acetylation sites for p300-enhanced PC4 binding to double-stranded DNA, consistent with the observation that CK2-mediated phosphorylation on the SEAC domains inhibits p300-dependent acetylation on PC4 (Kumar et al., 2001). The C-terminal region, in which the structure has been resolved by X-ray crystallography at 1.74 Å resolution (Brandsen et al., 1997), forms a dimer with each monomer composed of four antiparallel β-strands followed by a kinked α-helix and is able to bind with high affinity (Kd ~ 0.07 nM) two single strands of DNA running in opposite directions, as found in internally melted DNA duplexes. In the full-length protein, this single-stranded DNA-binding region is normally masked by intramolecular interaction with the N-terminal region and only becomes exposed after conformational changes induced, for instance, by CK2-mediated phosphorylation of the SEAC domains (Kaiser et al., 1995). Phosphorylation-induced conformational changes lead to inactivation of PC4 coactivator activity and further inhibition of transcription mediated by the single-stranded DNA-binding activity of PC4 (Werten et al., 1998).

The inhibitory activity of PC4 allows it to function as a repressor in suppressing basal transcription when activators are absent (Malik et al., 1998; Werten et al., 1998; Wu and Chiang, 1998). This inhibition usually occurs prior to PIC assembly in the absence of TAFs (Wu and Chiang, 1998) and can be alleviated by adding increasing amounts of TFIID, TFIIH and pol II holoenzyme in the transcription reaction, correlating with the ability of these multiprotein complexes to phosphorylate PC4 (Kershner et al., 1998; Malik et al., 1998). However, since inactivation of the ATP-binding site of XPB/ERCC3 helicase, but not XPD/ERCC2 helicase or CDK7 kinase, impairs the ability of recombinant TFIIH to overcome PC4-mediated repression (Fukuda et al., 2003), it remains unclear the precise mechanism used by components of the general transcription machinery to antagonize PC4 repressing activity.

The coactivator function of PC4 was evidenced by its ability to substitute for a crude USA fraction in mediating activator-dependent transcription in a cell-free transcription system reconstituted with recombinant GTFs (TFIIB, TBP, TFIIE, and TFIIF) and epitope-tagged multiprotein complexes (TFIID, TFIIH, and pol II; Wu et al., 1998). In this system, where TAFs and Mediator are not essential for activator-dependent transcription, PC4 is the only general cofactor indispensable

The General Transcription Machinery and General Cofactors

for transcriptional activation mediated by Gal4-VP16  
(Wu et al., 1998), and human papillomavirus E2 (Wu  
and Chiang, 2001a; Hou et al., 2002; Wu et al., 2003).  
Not surprisingly, PC4 can interact with both transcriptional activators, such as Gal4-VP16 (Ge and Roeder,  
1994a), BRCA1 (Haile and Parvin, 1999), AP2 (Kannan  
and Tainsky, 1999), HIV Tat (Holloway et al., 2000),  
human papillomavirus E2 (Wu and Chiang, 2001a)  
and p53 (Banerjee et al., 2004), and components of  
the general transcription machinery, such as TFIIA (Ge  
and Roeder, 1994a), TFIIH (Fukuda et al., 2004), and  
pol II (Malik et al., 1998), thereby serving as a bridging  
molecule to facilitate activator-dependent transcription likely through enhancement of PIC assembly on  
the promoter region. In addition, PC4 may promote  
sequence-specific DNA-binding activity of some activators, such as p53 (Banjeree et al., 2004), stimulate promoter escape in a TFIIA-and TAF-dependent manner  
(Fukuda et al., 2004), or enhance pol II elongation by  
modulating TFIIH kinase and FCP1 phosphatase activity on CTD phosphorylation (Calvo and Manley,  
2005). Clearly, PC4 can work in conjunction with other  
general cofactors, such as TAFs (Wu and Chiang, 1998;  
Wu et al., 1998) and Mediator (Fondell et al., 1999; Malik et al., 2000; Wu et al., 2003), to synergistically mediate activator-dependent transcription. Whether phosphorylation of PC4, which accounts for 95% of total  
PC4 in the cell (Ge et al., 1994), by TFIID, TFIIH,  
pol II holoenzyme (Kershner et al., 1998) and Mediator (Gu et al., 1999), plays a role in different steps  
of the transcriptional process remains to be further  
defined.

Besides being a transcriptional coactivator, PC4 has  
also been implicated in other cellular processes, such  
as DNA repair and DNA replication. In the aspect of  
DNA repair, PC4 can prevent mutagenesis arising from  
oxidative DNA damage caused by the interaction of  
reactive oxygen species (ROS) with DNA, depending  
upon its single-stranded DNA-binding activity (Wang  
et al., 2004). The involvement of PC4 in DNA replication appears to be more complicated, as PC4 can interact with replication protein A (RPA) on single-stranded  
DNA and facilitate T-antigen-mediated unwinding of  
DNA containing SV40 origin of replication, while it  
also inhibits RNA primer synthesis and DNA polymerase δ-catalyzed DNA chain elongation (Pan et al.,  
1996). The biological significance of these in vitro reactions performed in the presence of PC4 requires further  
investigations.


NC1

NC1, also known as HMG1 or HMGB1 with a molecular size around 25 kDa, is a member of the highly conserved chromatin-associated proteins that bend DNA and bind preferentially to distorted DNA structures (Bustin, 2001; Thomas and Travers, 2001). HMGB1 is structurally divided into three domains: two homologous DNA-binding HMG-box domains A and B each containing approximately 80 amino acids, and a C-terminal tail containing a stretch of 30 acidic residues. Boxes A and B each forms an “L”-shaped structure with three α-helices constituting a minor-groove DNA-binding domain that preferentially binds distorted DNA, such as four-way junctions, cisplatin-modified DNA and bulged DNA, and induces DNA bending without sequence specificity (Thomas, 2001; Thomas and Travers, 2001). Both domains A and B contain an additional basic extension of amino acid residues that enhance the DNA-binding affinity of the HMG box. The C-terminal acidic tail modulates the DNA-binding activity of HMGB1 and seems to be inhibitory toward HMG box binding to DNA.

The transcriptional role of HMGB1 is similar to other USA-derived cofactors in that it normally functions as a repressor in the absence of an activator, but acts as a coactivator in activator-dependent transcription. The repressing activity of HMGB1 appears to work by promoting the formation of a stable HMGB1-TBP-promoter complex that prevents TFIIB entry (Ge and Roeder, 1994b), as the presence of HMGB1 increases the affinity of TBP for the TATA box by 20-fold (Das and Scovell, 2001). The interaction domains were mapped to the HMG box A of HMGB1 (Sutrias-Grau et al., 1999) and the glutamine-rich region of TBP (Das and Scovell, 2001). TFIIA, as an antirepressor (see TFIIA section), can displace HMGB1 from the ternary complex and overcome HMGB1-mediated inhibition of PIC formation, thereby restoring transcription activity (Ge and Roeder, 1994b). The coactivator function of HMGB1 is attributed to its direct interaction with transcriptional activators, such as p53, steroid hormone receptors, Oct-1, HOX and Rel proteins (Thomas and Travers, 2001; Agresti and Bianchi, 2003), and with components of the general transcription machinery, including TBP (Ge and Roeder, 1994b) and TAF10 (Verrier et al., 1997). Undoubtedly, the architectural role of HMGB1 in bending DNA will further contribute to its coactivator function typically by enhancing


M. C. Thomas and C.-M. Chiang
sequence-specific recognition by these DNA-binding proteins.

# CONCLUDING REMARKS

The finding of multiple core promoter elements (BREu, TATA, BREd, Inr, MTE, DPE and DCE) and core promoter-binding activity exhibited by TFIIB, TBP family members (TBP and TRF1-4) and distinct TAF complexes (TFIID, TFTC and SAGA/STAGA/SLIK) further exemplifies the diversity and specificity of core promoter recognition inherent in the eukaryotic genome. The recent discovery of RNA polymerase IV and spRNAP-IV implicated in transcriptional silencing and mRNA transcription, respectively, also broadens our view of pol II-mediated transcription processes beyond the traditional sets of RNA polymerases and their accessory factors. From the structural and functional analyses of GTFs, pol II, general cofactors, and the built-up transcription complexes, we have come across many unexpected properties of these transcription components. One of the most intriguing features is the uncovering of cryptic DNA-binding activity found in many of these proteins. In the case of TFIIB, activator-induced conformational changes unmask the recognition loop in the first direct repeat of TFIIB for BREd binding and concurrently reduce the affinity of its HTH DNA-binding domain, located in the second direct repeat, for contacting BREu. From the studies of TATA-containing and TATA-less promoters, TBP and some TAFs (TAF1, TAF2, and histone fold-containing TAF6 and TAF9) make sequence-specific contacts with the Inr, DPE, and DCE elements. However, the specific TAF(s) acting through the MTE has yet to be identified. Additional contacts made between GTFs and the promoter have also been observed presumably following TFIIIF-induced DNA wrapping around a partially assembled TBP-TFIIB-pol II/TFIIF complex, due to cryptic DNA-binding activity exhibited by the winged HTH motifs present in RAP30 and RAP74. It is likely that the TAF components of TFIID will play a significant role in stabilizing this complex, if TFIID is substituted for TBP, and that the winged HTH motif of TFIIEβ will further stabilize the final PIC by providing extra DNA contacts. Undoubtedly, the efficiency and the functional property of the assembled PIC will be modulated by the presence of general cofactors, such as PC1, Topo I, PC4, and HMGB1, whose intrinsic DNA-binding activity may help reconfigure the topology and architecture of the promoter and, moreover, the recognition of damaged DNA templates.

The recent elucidation of pol II structures further provides mechanistic insight into the functional role of GTFs during PIC assembly. Besides the protein-DNA contacts contributed by sequence-specific recognition and cryptic DNA-binding activity as outlined above, the interactions amongst GTFs and with pol II also play an important role in stabilizing TBP/TFIID-promoter complex, alleviating repressing activity exerted by TAF1, BTAF1, and NC2, facilitating pol II entry, and forming a proper conformation for start site selection and later transition for pol II from initiation to elongation. The protein-protein contacts between the N-terminal zinc ribbon domain of TFIIB with TFIIIF and pol II help recruit pol II/TFIIF to the promoter. Once assembled, TFIIIF and the B-finger of TFIIB, situated by the catalytic center of pol II, facilitate start site selection. TFIIIF and pol II then aid in the subsequent entry of TFIIIE and TFIIH into the partially assembled PIC complex. The positioning of TFIIIE at the active center of pol II allows TFIIIE to stimulate TFIIH ATPase and DNA helicase activities, which facilitate promoter melting and promoter clearance. TFIIIE also enhances the kinase activity of TFIIH, allowing TFIIH to phosphorylate serine 5 of the unstructured CTD, thereby marking the transition of pol II from initiation to elongation. Further elucidation of the structural mechanism for PIC assembly can be found in several recent reviews. Clearly, future structural studies with the inclusion of additional transcriptional components, such as TAFs, TFIIIE, and TFIIH, into the TBP-TFIIA-TFIIB-pol II-TFIIIF structure, and also with the permeation of small molecule inhibitors, such as DRB and α-amanitin, will provide a more complete understanding of PIC assembly and function.

Another intriguing feature that regulates promoter activity and the functional property of the PIC lies in the ability of these transcriptional components to be posttranslationally modified by, e.g., phosphorylation, acetylation, glycosylation, ubiquitination, and poly(ADP-ribosyl)ation. In fact, many of these modifying activities are inherent to the transcriptional components themselves. TFIID, for example, possesses multiple enzymatic activities capable of phosphorylating H2B, TFIIAB, RAP74, and PC4, acetylating H3, H4, and TFIIIEβ, and ubiquitinating H1 and presumably TAF5 and TAF1 as well. TFIIH, besides its DNA helicase activity, can phosphorylate CTD, PC4 and a

The General Transcription Machinery and General Cofactors
number of transcriptional activators, and may also
exhibit E3 ubiquitin ligase activity. Likewise, Mediator displays kinase, HAT, and likely E3 ubiquitin lig-
ase activity, whereas PARP-1 can introduce poly(ADP-
ribosyl)ation onto TBP, TFIIF, pol II, and other general
cofactors and transcriptional activators. In many cases,
however, the functional consequences of these modifi-
cations and critical substrates remain to be investigated.
Related to this issue, many components of the general
transcription machinery and general cofactors can be
modified by exogenous enzymatic activities. An impor-
tant but underexplored example is pol II ubiquitination,
where pol II becomes polyubiquitinated in response to
DNA-damaging agents or upon transcriptional arrest
by different E3 ubiquitin ligases (Rsp5, hRPFl, pVHL-
Elongin B/C-Cullin 2-Rbx1, BRCA1/BARD1, and the
CSA complex) through polyubiquitin chain linkage via
different lysine residues in the ubiquitin molecule. It
would thus be interesting to see whether a transcription-
ally stalled pol II complex can be polyubiquitinated by
TFIIH and Mediator. This is certainly an exciting area
for future exploration.

In conclusion, the assembly of an initiation-
competent pol II complex is subject to multiple lev-
els of regulation by a diverse set of protein factors and
cofactors. There is no doubt that transcriptional activa-
tors have a significant impact in dictating the functional
property of the PIC, regardless of which pathway (se-
quential assembly vs. pol II holoenzyme) is used for
PIC assembly. The fact that the core promoter itself
can be recognized by TBP, TRFs, TFIID and other
TAF-containing complexes already lends flexibility for
interaction with distinct transcriptional regulators as
well as general cofactors which typically possess dual
activities in repressing basal transcription and enhanc-
ing activator-dependent transcription in response to
environmental cues. While tissue-specific TAFs and
TRFs play an important role in regulating transcription
during development, it remains a mystery what roles
positive cofactors and Mediator play during embryonic
development. Without doubt, the presence of TAF vari-
ants and multiple pathways for regulating PIC assem-
bly provide an additional way to fine-tune the transcrip-
tional events. Moreover, the alternative usage of general
cofactors, such as TAFs, Mediator, and various USA-
derived cofactors, and their functional redundancy and
transcriptional synergy on both TATA-containing and
TATA-less promoters provide an additional level of
complexity in modulating eukaryotic gene transcrip-

tion. Clearly, posttranslational modifications of activa-
tors through phosphorylation, acetylation, ubiquitina-
tion, and poly(ADP-ribosyl)ation by components of the
general transcription machinery and general cofactors,
such as TFIID, TFIIH, and Mediator, as well as by other
cellular proteins functioning in response to extracellu-
lar signaling events will further regulate the functional
property of the transcription complexes assembled on
target genes. For sure, exciting stories will continuously
be unraveled for years to come.

# ACKNOWLEDGMENTS

We are grateful to Drs. Stephen Buratowski, James Goodrich, Samuel Hou, Parminder Kaur, Donal Luse, Michael Meisterernst, Frank Pugh, and Shwu-Yuan Wu for their comments on the manuscript. The research conducted in Dr. Chiang's laboratory is currently sponsored by National Institutes of Health grant CA103867.

# REFERENCES

Abbott, K.L., Archambault, J., Xiao, H., Nguyen, B.D., Roeder, R.G., Greenblatt, J., Omichinski, J.G., and Legault, P. 2005a. Interactions of the HIV-1 Tat and RAP74 proteins with the RNA polymerase II CTD phosphatase FCP1. *Biochemistry* 44: 2716–2731.

Abbott, K.L., Renfrow, M.B., Chalmers, M.J., Nguyen, B.D., Marshall, A.G., Legault, P., and Omichinski, J.G. 2005b. Enhanced binding of RNAPII CTD phosphatase FCP1 to RAP74 following CK2 phosphorylation. *Biochemistry* 44: 2732–2745.

Agalioti, T., Lomvardas, S., Parekh, B., Yie, J., Maniatis, T., and Thanos, D. 2000. Ordered recruitment of chromatin modifying and general transcription factors to the IFN-β promoter. *Cell* 103: 667–678.

Agresti, A. and Bianchi, M.E. 2003. HMGB proteins and gene expression. *Curr Opin Genet Dev* 13: 170–178.

Akoulitchev, S., Chuikov, S., and Reinberg, D. 2000. TFIIH is negatively regulated by cdk8-containing mediator complexes. *Nature* 407: 102–106.

Ahn, S.H., Kim, M., and Buratowski, S. 2004. Phosphorylation of serine 2 within the RNA polymerase II C-terminal domain couples transcription and 3′end processing. *Mol Cell* 13: 67–76.

Albright, S.R. and Tjian, R. 2000. TAFs revisited: more data reveal new twists and confirm old ideas. *Gene* 242: 1–13.

Anderson, M.G., Scoggin, K.E.S., Simbulan-Rosenthal, C.M., and Steadman, J.A. 2000. Identification of poly(ADP-ribose) polymerase as a transcriptional coactivator of the human T-cell leukemia virus type 1 Tax protein. *J Virol* 74: 2169–2177.

Andel, F., III, Ladurner, A.G., Inouye, C., Tjian, R., and Nogales, E. 1999. Three-dimensional structure of the human TFIID-IIA-IIB complex. *Science* 286: 2153–2156.

Andrau, J.-C., Van Oevelen, C.J.C., Van Teeffelen, H.A.A.M., Weil, P.A., Holstege, F.C.P., and Timmers, H.T.M. 2002. Mot1p is essential for TBP recruitment to selected promoters during in vivo gene activation. *EMBO J* 21: 5173–5183.

Apone, L.M., Virbasius, C.M., Reese, J.C., and Green, M.R. 1996. Yeast TAFII90 is required for cell-cycle progression through G2/M but not for general transcription activation. *Genes Dev* 10: 2368–2380.

M. C. Thomas and C.-M. Chiang

Apone, L.M., Virbasius, C.A., Holstege, F.C.P., Wang, J., Young, R.A., and Green, M.R. 1998. Broad, but not universal, transcriptional requirement for yTAFII17, a histone H3-like TAFII present in TFIID and SAGA. Mol Cell 2:653–661.

Archambault, J., Chambers, R.S., Kobor, M.S., Ho, Y., Cartier, M., Bolotin, D., Andrews, B., Kane, C.M., and Greenblatt, J. 1997. An essential component of a C-terminal domain phosphatase that interacts with transcription factor IIF in Saccharomyces cerevisiae. Proc Natl Acad Sci USA 94:14300–14305.

Archambault, J., Pan, G., Dahmus, G.K., Cartier, M., Marshall, N., Zhang, S., Dahmus, M.E., and Greenblatt, J. 1998. FCP1, the RAP74-interacting subunit of a human protein phosphatase that dephosphorylates the carboxyl-terminal domain of RNA polymerase II. J Biol Chem 273:27593–27601.

Armache, K.-J., Kettenberger, H., and Cramer, P. 2003 Architecture of initiation-competent 12-subunit RNA polymerase II. Proc Natl Acad Sci USA 100:6964–6968.

Armache, K.-J., Mitterweger, S., Meinhart, A., and Cramer, P. 2005. Structures of complete RNA polymerase II and its subcomplex, Rpb4/7. J Biol Chem 280:7131–7134.

Aso, T., Vasavada, H.A., Kawaguchi, T., Germino, F.J., Ganguly, S., Kitajima, S., Weissman, S.M., and Yasukochi, Y. 1992. Characterization of cDNA for the large subunit of the transcription initiation factor TFIIF. Nature 355:461–464.

Asturias, F.J., Jiang, Y.W., Myers, L.C., Gustafsson, C.M., and Kornberg, R.D. 1999. Conserved structures of mediator and RNA polymerase II holoenzyme. Science 283:985–987.

Asturias, F.J. 2004. RNA polymerase II structure, and organization of the preinitiation complex. Curr Opin Struct Biol 14:121–129.

Auble, D.T. and Hahn, S. 1993. An ATP-dependent inhibitor of TBP binding to DNA. Genes Dev 7:844–856.

Auble, D.T., Wang, D., Post, K.W., and Hahn, S. 1997. Molecular analysis of the SNF2/SWI2 protein family member MOT1, an ATP-driven enzyme that dissociates TATA-binding protein from DNA. Mol Cell Biol 17:4842–4851.

Auty, R., Steen, H., Myers, L.C., Persinger, J., Bartholomew, B., Gygi, S.P., and Buratowski, S. 2004. Purification of active TFIID from Saccharomyces cerevisiae. Extensive promoter contacts and co-activator function. J Biol Chem 279:49973–49981.

Avigan, M.I., Strober, B., and Levens, D. 1990. A far upstream element stimulates c-myc expression in undifferentiated leukemia cells. J Biol Chem 265:18538–18545.

Bagby, S., Kim, S., Maldonado, E., Tong, K.I., Reinberg, D., and Ikura, M. 1995. Solution structure of the C-terminal core domain of human TFIIB: similarity to cyclin A and interaction with TATA-binding protein. Cell 82:857–867.

Balasubramanian, R., Pray-Grant, M.G., Selleck, W., Grant, P.A., and Tan, S. 2002. Role of the Ada2 and Ada3 transcriptional coactivators in histone acetylation. J Biol Chem 277:7989–7995.

Baliga, N.S., Goo, Y.A., Ng, W.V., Hood, L., Daniels, C.J., and DasSarma, S. 2000. Is gene expression in Halobacterium NRC-1 regulated by multiple TBP and TFB transcription factors? Mol Microbiol 36:1184–1185.

Baltimore, D. 1970. RNA-dependent DNA polymerase in virions of RNA tumour viruses. Nature 226:1209–1211.

Banerjee, S., Kumar, B.R., and Kundu, T.K. 2004. General transcriptional coactivator PC4 activates p53 function. Mol Cell Biol 24:2052–2062.

Bangur, C.S., Pardee, T.S., and Ponticelli, A.S. 1997. Mutational analysis of the D1/E1 core helices and the conserved N-terminal region of yeast transcription factor IIB (TFIIB): identification of an N-terminal mutant that stabilizes TATA-binding protein-TFIIB-DNA complexes. Mol Cell Biol 17:6784–6793.

Bangur, C.S., Faitar, S.L., Folster, J.P., and Ponticelli, A.S. 1999. An interaction between the N-terminal region and the core domain of yeast TFIIB promotes the formation of TATA-binding protein-TFIIB-DNA complexes. J Biol Chem 274:23203–23209.


The General Transcription Machinery and General Cofactors

Banik, U., Beechem, J.M., Klebanow, E., Schroeder, S., and Weil, P.A. 2001. Fluorescence-based analyses of the effects of full-length recombinant TAF130p on the interaction of TATA box-binding protein with TATA box DNA. J Biol Chem 276:49100–49109.

Barbaric, S., Reinke, H., and Hörz, W. 2003. Multiple mechanistically distinct functions of SAGA at the PHO5 promoter. Mol Cell Biol 23:3468–3476.

Barberis, A., Müller, C.W., Harrison, S.C., and Ptashne, M. 1993. Delination of two functional regions of transcription factor TFIIB. Proc Natl Acad Sci USA 90:5628–5632.

Basehoar, A.D., Zanton, S.J., and Pugh, B.F. 2004. Identification and distinct regulation of yeast TATA box-containing genes. Cell 116:699–709.

Baskaran, R., Dahmus, M.E., and Wang, J.Y.J. 1993. Tyrosine phosphorylation of mammalian RNA polymerase II carboxyl-terminal domain. Proc Natl Acad Sci USA 90:11167–11171.

Bastien, J., Adam-Stitah, S., Riedl, T., Egly, J.-M., Chambon, P., and Rochette-Egly, C. 2000. TFIIH interacts with the retinoic acid receptor γ and phosphorylates its AF-1-activating domain through cdk7. J Biol Chem 275:21896–21904.

Beaudenon, S.L., Huacani, M.R., Wang, G., McDonnell, D.P., and Huibregtse, J.M. 1999. Rsp5 ubiquitin-protein ligase mediates DNA damage-induced degradation of the large subunit of RNA polymerase II in Saccharomyces cerevisiae. Mol Cell Biol 19:6972–6979.

Bell, S.D., Brinkman, A.B., van der Oost, J., and Jackson, S.P. 2001. The archaeal TFIIEα homologue facilitates transcription initiation by enhancing TATA-box recognition. EMBO Rep 2:133–138.

Bell, S.D. and Jackson, S.P. 2001. Mechanism and regulation of transcription in archaea. Curr Opin Microbiol 4:208–213.

Belotserkovskaya, R., Sterner, D.E., Deng, M., Sayre, M.H., Lieberman, P.M., and Berger, S.L. 2000. Inhibition of TATA-binding protein function by SAGA subunits Spt3 and Spt8 at Gcn4-activated promoters. Mol Cell Biol 20:634–647.

Berk, A.J. 2000. TBP-like factors come into focus. Cell 103:5–8.

Berroteran, R.W., Ware, D.E., and Hampsey, M. 1994. The sua8 suppressors of Saccharomyces cerevisiae encode replacements of conserved residues within the largest subunit of RNA polymerase II and affect transcription start site selection similarly to sua7 (TFIIB) mutations. Mol Cell Biol 14:226–237.

Bhaumik, S.R. and Green, M.R. 2002. Differential requirement of SAGA components for recruitment of TATA-box-binding protein to promoters in vivo. Mol Cell Biol 22:7365–7371.

Bhaumik, S.R., Raha, T., Aiello, D.P., and Green, M.R. 2004. In vivo target of a transcriptional activator revealed by fluorescence resonance energy transfer. Genes Dev 18:333–343.

Birck, C., Poch, O., Romier, C., Ruff, M., Mengus, G., Lavigne, A.-C., Davidson, I., and Moras, D. 1998. Human TAFII28 and TAFII18 interact through a histone fold encoded by atypical evolutionary conserved motifs also found in the SPT3 family. Cell 94:239–249.

Björklund, S. and Gustafsson, C.M. 2005. The yeast mediator complex and its regulation. Trends Biochem Sci 30:240–244.

Blazek, E., Mittler, G., and Meisterernst, M. 2005. The mediator of RNA polymerase II. Chromosoma 113:399–408.

Boeger, H., Bushnell, D.A., Davis, R., Griesenbeck, J., Lorch, Y., Strattan, J.S., Westover, K.D., and Kornberg, R.D. 2005. Structural basis of eukaryotic gene transcription. FEBS Lett 579:899–903.

Bohr, V.A., Smith, C.A., Okumoto, D.S., and Hanawalt, P.C. 1985. DNA repair in an active gene: removal of pyrimidine dimers from the DHFR gene of CHO cells is much more efficient than in the genome overall. Cell 40:359–369.

Borggreffe, T., Davis, R., Erdjument-Bromage, H., Tempst, P., and Kornberg, R.D. 2002. A complex of the Srb8, -9, -10, and -11 transcriptional regulatory proteins from yeast. J Biol Chem 277:44202–44207.

Borukhov, S. and Nudler, E. 2003. RNA polymerase holoenzyme: structure, function and biological implications. Curr Opin Microbiol 6:93–100.

Boube, M., Joulia, L., Cribbs, D.L., and Bourbon, H.-M. 2002. Evidence for a mediator of RNA polymerase II transcriptional regulation conserved from yeast to man. *Cell* 110:143–151.

Bourbon, H.-M., Aguilera, A., Ansari, A.Z., Asturias, F.J., Berk, A.J., Björklund, S., Blackwell, T.K., Borggrefe, T., Carey, M., Carlson, M., Conaway, J.W., Conaway, R.C., Emmons, S.W., Fondell, J.D., Freedman, L.P., Fukasawa, T., Gustafsson, C.M., Han, M., He, X., Herman, P.K., Hinnebusch, A.G., Holmberg, S., Holstege, F.C.P., Jaehning, J.A., Kim, Y.J., Kuras, L., Leutz, A., Lis, J.T., Meisterernst, M., Näär, A.M., Nasmyth, K., Parvin, J.D., Ptashne, M., Reinberg, D., Ronne, H., Sadowski, I., Sakurai, H., Sipiczki, M., Sternberg, P.W., Stillman, D.J., Strich, R., Struhl, K., Svejstrup, J.Q., Tuck, S., Winston, F., Roeder, R.G., and Kornberg, R.D. 2004. A unified nomenclature for protein subunits of mediator complexes linking transcriptional regulators to RNA polymerase II. *Mol Cell* 14:553–557.

Boyer-Guittaut, M., Birsoy, K., Potel, C., Elliott, G., Jaffray, E., Desterro, J.M., Hay, RT., and Oelgeschläger, T. 2005. SUMO-1 modification of human transcription factor (TF) IID complex subunits. *J Biol Chem* 280:9937–9945.

Brand, M., Leurent, C., Mallouh, V., Tora, L., and Schultz, P. 1999a. Three-dimensional structures of the TAFII-containing complexes TFIID and TFTC. *Science* 286:2151–2153.

Brand, M., Yamamoto, K., Staub, A., and Tora, L. 1999b. Identification of TATA-binding protein-free TAFII-containing complex subunits suggests a role in nucleosome acetylation and signal transduction. *J Biol Chem* 274:18285–18289.

Brandsen, J., Werten, S., van der Vliet, P.C., Meisterernst, M., Kroon, J., and Gros, P. 1997. C-terminal domain of transcription cofactor PC4 reveals dimeric ssDNA binding site. *Nat Struct Biol* 4:900–903.

Bregman, D.B., Halaban, R., van Gool, A.J., Henning, K.A., Friedberg, E.C., and Warren, S.L. 1996. UV-induced ubiquitination of RNA polymerase II: a novel modification deficient in Cockayne syndrome cells. *Proc Natl Acad Sci USA* 93:11586–11590.

Brower, C.S., Sato, S., Tomomori-Sato, C., Kamura, T., Pause, A., Stearman, R., Klausner, R.D., Malik, S., Lane, W.S., Sorokina, I., Roeder, R.G., Conaway, J.W., and Conaway, R.C. 2002. Mammalian mediator subunit mMED8 is an Elongin BC-interacting protein that can assemble with Cul2 and Rbx1 to reconstitute a ubiquitin ligase. *Proc Natl Acad Sci USA* 99:10353–10358.

Brzovic, P.S., Rajagopal, P., Hoyt, D.W., King, M.C., and Klevit, R.E. 2001. Structure of a BRCA1-BARD1 heterodimeric RING-RING complex. *Nat Struct Biol* 8:833–837.

Brzovic, P.S., Keeffe, J.R., Nishikawa, H., Miyamoto, K., Fox, D., III, Fukuda, M., Ohta, T., and Klevit, R. 2003. Binding and recognition in the assembly of an active BRCA1/BARD1 ubiquitin-ligase complex. *Proc Natl Acad Sci USA* 100:5646–5651.

Bucheli, M. and Sweder, K. 2004. In UV-irradiated Saccharomyces cerevisiae, overexpression of Swi2/Snf2 family member Rad26 increases transcription-coupled repair and repair of the non-transcribed strand. *Mol Microbiol* 52:1653–1663.

Buermeyer, A.B., Thompson, N.E., Strasheim, L.A., Burgess, R.R., and Farnham, P.J. 1992. The HIP1 initiator element plays a role in determining the in vitro requirement of the dihydrofolate reductase gene promoter for the C-terminal domain of RNA polymerase II. *Mol Cell Biol* 12:2250–2259.

Buermeyer, A.B., Strasheim, L.A., McMahon, S.L., and Farnham, P.J. 1995. Identification of cis-acting elements that can obviate a requirement for the C-terminal domain of RNA polymerase II. *J Biol Chem* 270:6798–6807.

Buratowski, S., Hahn, S., Sharp, P.A., and Guarente, L. 1988. Function of a yeast TATA element-binding protein in a mammalian transcription system. *Nature* 334:37–42.

Buratowski, S., Hahn, S., Guarente, L., and Sharp, P.A. 1989. Five intermediate complexes in transcription initiation by RNA polymerase II. *Cell* 56:549–561.

Buratowski, S. and Sharp, P.A. 1990. Transcription initiation complexes and upstream activation with RNA polymerase II lacking the C-terminal domain of the largest subunit. *Mol Cell Biol* 10:5562–5564.


Buratowski, S. and Zhou, H. 1993. Functional domains of transcription factor TFIIB. *Proc Natl Acad Sci USA* 90:5633–5637.

Burke, T.W. and Kadonaga, J.T. 1996. Drosophila TFIID binds to a conserved downstream basal promoter element that is present in many TATA-box-deficient promoters. *Genes Dev* 10:711–724.

Burke, T.W. and Kadonaga, J.T. 1997. The downstream core promoter element, DPE, is conserved from Drosophila to humans and is recognized by TAFII60 of Drosophila. *Genes Dev* 11:3020–3031.

Burley, S.K. and Roeder, R.G. 1996. Biochemistry and structural biology of transcription factor IID (TFIID). *Annu Rev Biochem* 65:769–799.

Bushnell, D.A. and Kornberg, R.D. 2003. Complete, 12-subunit RNA polymerase II at 4.1-Å resolution: implications for the initiation of transcription. *Proc Natl Acad Sci USA* 100:6969–6973.

Bushnell, D.A., Westover, K.D., Davis, R.E., and Kornberg, R.D. 2004. Structural basis of transcription: an RNA polymerase II-TFIIB cocrystal at 4.5 angstroms. *Science* 303:983–988.

Bustin, M. 2001. Revised nomenclature for high mobility group (HMG) chromosomal proteins. *Trends Biochem Sci* 26:152–153.

Butler, A.J. and Ordahl, C.P. 1999. Poly(ADP-ribose) polymerase binds with transcription enhancer factor 1 to MCAT1 elements to regulate muscle-specific transcription. *Mol Cell Biol* 19:296–306.

Butler, J.E. and Kadonaga, J.T. 2001. Enhancer-promoter specificity mediated by DPE or TATA core promoter motifs. *Genes Dev* 15:2515–2519.

Cairns, B.R., Henry, N.L., and Kornberg, R.D. 1996a. TFG/TAF30/ANC1, a component of the yeast SWI/SNF complex that is similar to the leukemogenic proteins ENL and AF-9. *Mol Cell Biol* 16:3308–3316.

Cairns, B.R., Lorch, Y., Li, Y., Zhang, M., Lacomis, L., Erdjument-Bromage, H., Tempst, P., Du, J., Laurent, B., and Kornberg, R.D. 1996b. RSC, an essential, abundant chromatin-remodeling complex. *Cell* 87:1249–1260.

Calvo, O. and Manley, J.L. 2005. The transcriptional coactivator PC4/Sub1 has multiple functions in RNA polymerase II transcription. *EMBO J* 24:1009–1020.

Campbell, K.M., Ranallo, R.T., Stargell, L.A., and Lumb, K.J. 2000. Reevaluation of transcriptional regulation by TATA-binding protein oligomerization: predominance of monomers. *Biochemistry* 39:2633–2638.

Cang, Y. and Prelich, G. 2002. Direct stimulation of transcription by negative cofactor 2 (NC2) through TATA-binding protein (TBP). *Proc Natl Acad Sci USA* 99:12727–12732.

Cao, R., Wang, L., Wang, H., Xia, L., Erdjument-Bromage, H., Tempst, P., Jones, R.S., and Zhang, Y. 2002. Role of histone H3 lysine 27 methylation in Polycomb-group silencing. *Science* 298:1039–1043.

Carles, C., Treich, I., Bouet, F., Riva, M., and Sentenac, A. 1991. Two additional common subunits, ABC10α and ABC10β, are shared by yeast RNA polymerases. *J Biol Chem* 266:24092–24096.

Castano, E., Gross, P., Wang, Z., Roeder, R.G., and Oelgeschläger, T. 2000. The C-terminal domain-phosphorylated IIo form of RNA polymerase II is associated with the transcription repressor NC2 (Dr1/DRAP1) and is required for transcription activation in human nuclear extracts. *Proc Natl Acad Sci USA* 97:7184–7189.

Cavallini, B., Huet, J., Plassat, J.-L., Sentenac, A., Egly, J.-M., and Chambon, P. 1988. A yeast activity can substitute for the HeLa cell TATA box factor. *Nature* 334:77–80.

Cavallini, B., Faus, I., Matthes, H., Chipoulet, J.M., Winsor, B., Egly, J.-M., and Chambon, P. 1989. Cloning of the gene encoding the yeast protein BTF1Y, which can substitute for the human TATA box-binding factor. *Proc Natl Acad Sci USA* 86:9803–9807.

Cavusoglu, N., Brand, M., Tora, L., and Dorsselaer, A.V. 2003. Novel subunits of the TATA binding protein free TAFII-containing transcription complex identified by matrix-assisted laser desorption/ionization-time of flight mass spectrometry following one-dimensional gel electrophoresis. *Proteomics* 3:217–223.

Cervellera, M.N. and Sala, A. 2000. Poly(ADP-ribose) polymerase is a B-MYB coactivator. *J Biol Chem* 275:10692–10696.

Chadick, J.Z. and Asturias, F.J. 2005. Structure of eukaryotic Mediator complexes. *Trends Biochem Sci* 30:264–271.

Chalkley, G.E. and Verrijzer, C.P. 1999. DNA binding site selection by RNA polymerase II TAFs: a TAFII250-TAFII150 complex recognizes the initiator. EMBO J 18: 4835–4845.

Chamberlin, M. and Berg, P. 1962. Deoxyribonucleic acid-directed synthesis of ribonucleic acid by an enzyme from Escherichia coli. Proc Natl Acad Sci USA 48: 81–94.

Chambers, R.S., Wang, B.Q., Burton, Z.F., and Dahmus, M.E. 1995. The activity of COOH-terminal domain phosphatase is regulated by a docking site on RNA polymerase II and by the general transcription factors IIF and IIB. J Biol Chem 270: 14962–14969.

Chang, A., Cheang, S., Espanel, X., and Sudol, M. 2000. Rsp5 WW domains interact directly with the carboxyl-terminal domain of RNA polymerase II. J Biol Chem 275: 20562–20571.

Chao, D.M., Gadbois, E.L., Murray, P.J., Anderson, S.F., Sonu, M.S., Parvin, J.D., and Young, R.A. 1996. A mammalian SRB protein associated with an RNA polymerase II holoenzyme. Nature 380: 82–85.

Chasman, D.I., Flaherty, K.M., Sharp, P.A., and Kornberg, R.D. 1993. Crystal structure of yeast TATA-binding protein and model for interaction with DNA. Proc Natl Acad Sci USA 90: 8174–8178.

Chelstowska, A. and Butow, R.A. 1995. RTG genes in yeast that function in communication between mitochondria and the nucleus are also required for expression of genes encoding peroxisomal proteins. J Biol Chem 270: 18141–18146.

Chen, Z. and Manley, J.L. 2000. Robust mRNA transcription in chicken DT40 cells depleted of TAFII31 suggests both functional degeneracy and evolutionary divergence. Mol Cell Biol 20: 5064–5076.

Chen, D., Riedl, T., Washbrook, E., Pace, P.E., Coombes, R.C., Egly, J.-M., and Ali, S. 2000. Activation of estrogen receptor α by S118 phosphorylation involves a ligand-dependent interaction with TFIIH and participation of CDK7. Mol Cell 6: 127–137.

Chen, A., Kleiman, F.E., Manley, J.L., Ouchi, T., and Pan, Z.-Q. 2002. Autoubiquitination of the BRCA1-BARD1 RING ubiquitin ligase. J Biol Chem 277: 22085–22092.

Chen, H.-T. and Hahn, S. 2003. Binding of TFIIIB to RNA polymerase II: mapping the binding site for the TFIIIB zinc ribbon domain within the preinitiation complex. Mol Cell 12: 437–447.

Chen, H.-T. and Hahn, S. 2004. Mapping the location of TFIIIB within the RNA polymerase II transcription preinitiation complex: a model for the structure of the PIC. Cell 119: 169–180.

Cheong, J., Yi, M., Lin, Y., and Murakami, S. 1995. Human RPB5, a subunit shared by eukaryotic nuclear RNA polymerase, binds human hepatitis B virus X protein and may play a role in X transactivation. EMBO J 14: 143–150.

Chi, T., Lieberman, P., Ellwood, K., and Carey, M. 1995. A general mechanism for transcriptional synergy by eukaryotic activators. Nature 377: 254–257.

Chiang, C.-M., Ge, H., Wang, Z., Hoffmann, A., and Roeder, R.G. 1993. Unique TATA-binding protein-containing complexes and cofactors involved in transcription by RNA polymerases II and III. EMBO J 12: 2749–2762.

Chiang, C.-M. and Roeder, R.G. 1995. Cloning of an intrinsic human TFIID subunit that interacts with multiple transcriptional activators. Science 267: 531–536.

Chicca, J.J., II, Auble, D.T., and Pugh, B.F. 1998. Cloning and biochemical characterization of TAF-172, a human homolog of yeast Mot1. Mol Cell Biol 18: 1701–1710.

Cho, E.-J., Takagi, T., Moore, C.R., and Buratowski, S. 1997. mRNA capping enzyme is recruited to the transcription complex by phosphorylation of the RNA polymerase II carboxy-terminal domain. Genes Dev 11: 3319–3326.

Cho, H., Orphanides, G., Sun, X., Yang, X.-J., Ogryzko, V., Lees, E., Nakatani, Y., and Reinberg, D. 1998. A human RNA polymerase II complex containing factors that modify chromatin structure. Mol Cell Biol 18: 5355–5363.

Cho, E.J. and Buratowski, S. 1999. Evidence that transcription factor IIB is required for a post-assembly step in transcription initiation. J Biol Chem 274: 25807–25813.

---

Cho, H., Kim, T.-K., Mancebo, H., Lane, W.S., Flores, O., and Reinberg, D. 1999. A protein phosphatase function to recycle RNA polymerase II. Genes Dev 13: 1540–1552.

Cho, E.-J., Kobor, M.S., Kim, M., Greenblatt, J., and Buratowski, S. 2001. Opposing effects of Ctk1 kinase and Fcp1 phosphatase at Ser 2 of the RNA polymerase II C-terminal domain. Genes Dev 15: 3319–3329.

Chong, J.A., Moran, M.M., Teichmann, M., Kaczmarek, J.S., Roeder, R., and Clapham, D.E. 2005. TATA-Binding Protein (TBP)-Like Factor (TLF) Is a Functional Regulator of Transcription: Reciprocal Regulation of the Neurofibromatosis Type 1 and c-fos Genes by TLF/TRF2 and TBP. Mol Cell Biol 25: 2632–2643.

Christova, R. and Oelgeschläger, R. 2002. Association of human TFIID-promoter complexes with silenced mitotic chromatin in vivo. Nat Cell Biol 4: 79–82.

Chung, W.-H., Craighead, J.L., Chang, W.-H., Ezeokonkwo, C., Bareket-Sarnish, A., Kornberg, R.D., and Asturias, F.J. 2003. RNA polymerase II/TFIIF structure and conserved organization of the initiation complex. Mol Cell 12: 1003–1013.

Clemens, K.E., Piras, G., Radonovich, M.F., Choi, K.S., Duvall, J.F., DeJong, J., Roeder, R., and Brady, J.N. 1996. Interaction of the human T-cell lymphotropic virus type 1 Tax transactivator with transcription factor IIA. Mol Cell Biol 16: 4656–4664.

Coin, F., Bergmann, E., Tremeau-Bravard, A., and Egly, J.-M. 1999. Mutations in XPB and XPD helicases found in xeroderma pigmentosum patients impair the transcription function of TFIIH. EMBO J 18: 1357–1366.

Coin, F., Auriol, J., Tapias, A., Clivio, P., Vermeulen, W., and Egly, J.-M. 2004. Phosphorylation of XPB helicase regulates TFIIH nucleotide excision repair activity. EMBO J 23: 4835–4846.

Coin, F., Proietti De Santis, L., Nardo, T., Zlobinskaya, O., Stefanini, M., and Egly, J.-M. 2006. p8/TTD-A as a repair-specific TFIIH subunit. Mol Cell 21: 215–226.

Coleman, R.A. and Pugh, B.F. 1995. Evidence for functional binding and stable sliding of the TATA binding protein on nonspecific DNA. J Biol Chem 270: 13850–13859.

Coleman, R.A., Taggart, A.K.P., Benjamin, L.R., and Pugh, B.F. 1995. Dimerization of the TATA binding protein. J Biol Chem 270: 13842–13849.

Coleman, R.A., Taggart, A.K.P., Burma, S., Chicca, J.J., II, and Pugh, B.F. 1999. TFIIA regulates TBP and TFIID dimers. Mol Cell 4: 451–457.

Collart, M.A. 1996. The NOT, SPT3, and MOT1 genes functionally interact to regulate transcription at core promoters. Mol Cell Biol 16: 6668–6676.

Comer, F.I. and Hart, G.W. 2001. Reciprocity between O-GlcNAc and O-phosphate on the carboxyl terminal domain of RNA polymerase II. Biochemistry 40: 7845–7852.

Conaway, R.C. and Conaway, J.W. 1989. An RNA polymerase II transcription factor has an associated DNA-dependent ATPase (dATPase) activity strongly stimulated by the TATA region of promoters. Proc Natl Acad Sci USA 86: 7356–7360.

Conaway, J.W., Hanley, J.P., Garrett, K.P., and Conaway, R.C. 1991. Transcription initiated by RNA polymerase II and transcription factors from liver. Structure and action of transcription factors ε and τ. J Biol Chem 266: 7804–7811.

Conaway, R.C., Sato, S., Tomomori-Sato, C., Yao, T., and Conaway, J.W. 2005. The mammalian Mediator complex and its role in transcriptional regulation. Trends Biochem Sci 30: 250–255.

Cortes, P., Flores, O., and Reinberg, D. 1992. Factors involved in specific transcription by mammalian RNA polymerase II: purification and analysis of transcription factor IIA and identification of transcription factor IIJ. Mol Cell Biol 12: 413–421.

Coulombe, B., Li, J., and Greenblatt, J. 1994. Topological localization of the human transcription factors IIA, IIB, TATA box-binding protein, and RNA polymerase II-associated protein 30 on a class II promoter. J Biol Chem 269: 19962–19967.

Cramer, P., Bushnell, D.A., Fu, J., Gnatt, A.L., Maier-Davis, B., Thompson, N.E., Burgess, R.R., Edwards, A.M., David, P.R., and Kornberg, R.D. 2000. Architecture of RNA polymerase II and implications for the transcription mechanism. Science 288:640–649.

Cramer, P., Bushnell, D.A., and Kornberg, R.D. 2001. Structural basis of transcription: RNA polymerase II at 2.8 angstrom resolution. Science 292:1863–1876.

Cramer, P. 2004. RNA polymerase II structure: from core to functional complexes. Curr Opin Genet Dev 14:218–226.

Creton, S., Svejstrup, J.Q., and Collart, M.A. 2002. The NC2 α and β subunits play different roles in vivo. Genes Dev 16:3265–3276.

Crick, F.H.C. 1958. On protein synthesis. Symp Soc Exp Biol 12:138–163.

Crowley, T.E., Hoey, T., Liu, J.K., Jan, Y.N., Jan, L.Y., and Tjian, R. 1993. A new factor related to TATA-binding protein has highly restricted expression patterns in Drosophila. Nature 361:557–561.

D'Amours, D., Desnoyers, S., D'Silva, I., and Poirier, G.G. 1999. Poly(ADP-ribosyl)ation reactions in the regulation of nuclear functions. Biochem J 342:249–268.

Daniel, J.A., Torok, M.S., Sun, Z.W., Schieltz, D., Allis, C.D., Yates, J.R., III, and Grant, P.A. 2004. Deubiquitination of histone H2B by a yeast acetyltransferase complex regulates transcription. J Biol Chem 279:1867–1871.

Dantonel, J.C., Wurtz, J.M., Poch, O., Moras, D., and Tora, L. 1999. The TBP-like factor: an alternative transcription factor in metazoa? Trends Biochem Sci 24:335–339.

Dantonel, J.C., Quintin, S., Lakatos, L., Labouesse, M., and Tora, L. 2000. TBP-like factor is required for embryonic RNA polymerase II transcription in C. elegans. Mol Cell 6:715–722.

Das, D. and Scovell, W.M. 2001. The binding interaction of HMG-1 with the TATA-binding protein/TATA complex. J Biol Chem 276:32597–32605.

Dasgupta, A., Darst, R.P., Martin, K.J., Afshari, C.A., and Auble, D.T. 2002. Mot1 activates and represses transcription by direct, ATPase-dependent mechanisms. Proc Natl Acad Sci USA 99:2666–2671.

Daugherty, M.A., Brenowitz, M., and Fried, M.G. 1999. The TATA-binding protein from Saccharomyces cerevisiae oligomerizes in solution at micromolar concentrations to form tetramers and octamers. J Mol Biol 285:1389–1399.

Daugherty, M.A., Brenowitz, M., and Fried, M.G. 2000. Participation of the amino-terminal domain in the self-association of the full-length yeast TATA binding protein. Biochemistry 39:4869–4880.

Davidson, I. 2003. The genetics of TBP and TBP-related factors. Trends Biochem Sci 28:391–398.

Davis, J.A., Takagi, Y., Kornberg, R.D., and Asturias, F.A. 2002. Structure of the yeast RNA polymerase II holoenzyme: mediator conformation and polymerase interaction. Mol Cell 10:409–415.

Davison, B.L., Egly, J.-M., Mulvihill, E.R., and Chambon, P. 1983. Formation of stable preinitiation complexes between eukaryotic class B transcription factors and promoter sequences. Nature 301:680–686.

DeJong, J. and Roeder, R.G. 1993. A single cDNA, hTFIIA/α, encodes both the p35 and p19 subunits of human TFIIA. Genes Dev 7:2220–2234.

Deng, L., Wang, C., Spencer, E., Yang, L., Braun, A., You, J., Slaughter, C., Pickart, C., and Chen, Z.J. 2000. Activation of the IkB kinase complex by TRAF6 requires a dimeric ubiquitin-conjugating enzyme complex and a unique polyubiquitin chain. Cell 103:351–361.

Deng, W. and Roberts, S.G.E. 2005. A core promoter element downstream of the TATA box that is recognized by TFIIB. Genes Dev 19:2418–2423.

Denko, N., Wernke-Dollries, K., Johnson, A.B., Hammond, E., Chiang, C.-M., and Barton, M.C. 2003. Hypoxia actively represses transcription by inducing negative cofactor 2 (Dr1/DrAP1) and blocking preinitiation complex assembly. J Biol Chem 278:5744–5749.

Dahmus, M.E. 1995. Phosphorylation of the C-terminal domain of RNA polymerase II. Biochim Biophys Acta 1261:171–182.


Dikstein, R., Ruppert, S., and Tjian, R. 1996. TAFII250 is a bipartite protein kinase that phosphorylates the base transcription factor RAP74. Cell 84:781–790.

Dion, V. and Coulombe, B. 2003. Interaction of a DNA-bound transcriptional activator with TBP-TFIIA-TFIIB-Promoter quaternary complex. J Biol Chem 278:11495–11501.

Dotson, M.R., Yuan, C.X., Roeder, R.G., Myers, L.C., Gustafsson, C.M., Jiang, Y.W., Li, Y., Kornberg, R.D., and Asturias, F.J. 2000. Structural organization of yeast and mammalian mediator complexes. Proc Natl Acad Sci USA 97:14307–14310.

Douziech, M., Coin, F., Chipoulet, J.-M., Arai, Y., Ohkuma, Y., Egly, J.-M., and Coulombe, B. 2000. Mechanism of promoter melting by the xeroderma pigmentosum complementation group B helicase of transcription factor IIH revealed by protein-DNA photo-cross-linking. Mol Cell Biol 20:8168–8177.

Drane, P., Compe, E., Catez, P., Chymkowitch, P., and Egly, J.-M. 2004. Selective regulation of vitamin D receptor-responsive genes by TFIID. Mol Cell 16:187–197.

Drapkin, R., Reardon, J.T., Ansari, A., Huang, J.C., Zawel, L., Ahn, K., Sancar, A., and Reinberg, D. 1994. Dual role of TFIID in DNA excision repair and in transcription by RNA polymerase II. Nature 368:769–772.

Durso, R.J., Fisher, A.K., Albright-Frey, T.J., and Reese, J.C. 2001. Analysis of TAF90 mutants displaying allele-specific and broad defects in transcription. Mol Cell Biol 21:7331–7344.

Dvir, A., Conaway, R.C., and Conaway, J.W. 1997. A role for TFIID in controlling the activity of early RNA polymerase II elongation complexes. Proc Natl Acad Sci USA 94:9006–9010.

Dynlacht, B.D., Hoey, T., and Tjian, R. 1991. Isolation of coactivators associated with the TATA-binding protein that mediate transcriptional activation. Cell 66:563–576.

Elsby, L.M. and Roberts, S.G.E. 2004. The role of TFIIB conformation in transcriptional regulation. Biochem Soc Trans 32:1098–1099.

Evans, R., Failey, J.A., and Roberts, S.G.E. 2001. Activator-mediated disruption of sequence-specific DNA contacts by the general transcription factor TFIIB. Genes Dev 15:2945–2949.

Fairley, J.A., Evans, R., Hawkes, N.A., and Roberts, S.G.E. 2002. Core promoter-dependent TFIIB conformation and a role for TFIIB conformation in transcriptional start site selection. Mol Cell Biol 22:6697–6705.

Faitar, S.L., Brodie, S.A., and Ponticelli, A.S. 2001. Promoter-specific shifts in transcription initiation conferred by yeast TFIIB mutations are determined by the sequence in the immediate vicinity of the start sites. Mol Cell Biol 21:4427–4440.

Falender, A.E., Freiman, R.N., Geles, K.G., Lo, K.C., Hwang, K., Lamb, D.J., Morris, P.L., Tjian, R., and Richards, J.S. 2005. Maintenance of spermatogenesis requires TAF4b, a gonad-specific subunit of TFIID. Genes Dev 19:794–803.

Fang, S.M. and Burton, Z.F. 1996. RNA polymerase II-associated protein (RAP) 74 binds transcription factor (TF) IIB and blocks TFIIB-RAP30 binding. J Biol Chem 271:11703–11709.

Feaver, W.J., Gileadi, O., and Kornberg, R.D. 1991a. Purification and characterization of yeast RNA polymerase II transcription factor b. J Biol Chem 266:19000–19005.

Feaver, W.J., Gileadi, O., Li, Y., and Kornberg, R.D. 1991b. CTD kinase associated with yeast RNA polymerase II initiation factor b. Cell 67:1223–1230.

Feaver, W.J., Svejstrup, J.Q., Bardwell, L., Bardwell, A.J., Buratowski, S., Gulyas, K.D., Donahue, T.F., Friedberg, E.C., and Kornberg, R.D. 1993. Dual roles of a multiprotein complex from S. cerevisiae in transcription and DNA repair. Cell 75:1379–1387.

Feaver, W.J., Henry, N.L., Wang, Z., Wu, X., Svejstrup, J.Q., Bushnell, D.A., Friedberg, E.C., and Kornberg, R.D. 1997. Genes for Tfb2, Tfb3, and Tfb4 subunits of yeast transcription/repair factor IIH. Homology to human cyclin-dependent kinase activating kinase and IIH subunits. J Biol Chem 272:19319–19327.

Feaver, W.J., Huang, W., and Friedberg, E.C. 1999. The TFB4 subunit of yeast TFIID is required for both nucleotide excision repair

M. C. Thomas and C.-M. Chiang

and RNA polymerase II transcription. J Biol Chem 274:29564–29567.

Ferdous, A., Gonzalez, F., Sun, L., Kodadek, T., and Johnston, S.A. 2001. The 19S regulatory particle of the proteasome is required for efficient transcription elongation by RNA polymerase II. Mol Cell 7:981–991.

Finkelstein, A., Kostrub, C.F., Li, J., Chavez, D.P., Wang, B.Q., Fang, S.M., Greenblatt, J., and Burton, Z.F. 1992. A cDNA encoding RAP74, a general initiation factor for transcription by RNA polymerase II. Nature 355:464–467.

Fire, A., Samuels, M., and Sharp, P.A. 1984. Interactions between RNA polymerase II, factors, and template leading to accurate transcription. J Biol Chem 259:2509–2516.

Fischle, W., Wang, Y., and Allis, C.D. 2003. Histone and chromatin cross-talk. Curr Opin Cell Biol 15:172–183.

Fishburn, J., Mohibullah, N., and Hahn, S. 2005. Function of a eukaryotic transcription activator during the transcription cycle. Mol Cell 18:369–378.

Flores, O., Maldonado, E., Burton, Z., Greenblatt, J., and Reinberg, D. 1988. Factors involved in specific transcription by mammalian RNA polymerase II. RNA polymerase II-associating protein 30 is an essential component of transcription factor IIF. J Biol Chem 263:10812–10816.

Flores, O., Maldonado, E., and Reinberg, D. 1989. Factors involved in specific transcription by mammalian RNA polymerase II. Factors IIE and IIF independently interact with RNA polymerase II. J Biol Chem 264:8913–8921.

Flores, O., Ha, I., and Reinberg, D. 1990. Factors involved in specific transcription by mammalian RNA polymerase II. Purification and subunit composition of transcription factor IIF. J Biol Chem 265:5629–5634.

Flores, O., Lu, H., Killeen, M., Greenblatt, J., Burton, Z.F., and Reinberg, D. 1991. The small subunit of transcription factor IIF recruits RNA polymerase II into the preinitiation complex. Proc Natl Acad Sci USA 88:9999–10003.

Flores, O., Lu, H., and Reinberg, D. 1992. Factors involved in specific transcription by mammalian RNA polymerase II. Identification and characterization of factor IIH. J Biol Chem 267:2786–2793.

Fondell, J.D., Ge, H., and Roeder, R.G. 1996. Ligand induction of a transcriptionally active thyroid hormone receptor coactivator complex. Proc Natl Acad Sci USA 93:8329–8333.

Fondell, J.D., Guermah, M., Malik, S., and Roeder, R.G. 1999. Thyroid hormone receptor-associated proteins and general positive cofactors mediate thyroid hormone receptor function in the absence of TATA box-binding protein-associated factors of TFIID. Proc Natl Acad Sci USA 96:1959–1964.

Forget, D., Robert, F., Grondin, G., Burton, Z.F., Greenblatt, J., and Coulombe, B. 1997. RAP74 induces promoter contacts by RNA polymerase II upstream and downstream of a DNA bend centered on the TATA box. Proc Natl Acad Sci USA 94:7150–7155.

Forget, D., Langelier, M.-F., Thérien, C., Trinh, V., and Coulombe, B. 2004. Photo-cross-linking of a purified preinitiation complex reveals central roles for the RNA polymerase II mobile clamp and TFIIE in initiation mechanisms. Mol Cell Biol 24:1122–1131.

Francis, N.J., Saurin, A.J., Shao, Z., and Kingston, R.E. 2001. Reconstitution of a functional core polycomb repressive complex. Mol Cell 8:545–556.

Frank, D.J., Tyree, C.M., George, C.P., and Kadonaga, J.T. 1995. Structure and function of the small subunit of TFIIF (RAP30) from *Drosophila melanogaster*. J Biol Chem 270:6292–6297.

Freiman, R.N., Albright, S.R., Zheng, S., Sha, W.C., Hammer, R.E., and Tjian, R. 2001. Requirement of tissue-selective TBP-associated factor TAFII105 in ovarian development. Science 293:2084–2087.

Freire-Picos, M.A., Krishnamurthy, S., Sun, Z.-W., and Hampsey, M. 2005. Evidence that the Tfg1/Tfg2 dimer interface of TFIIF lies near the active center of the RNA polymerase II initiation complex. Nucleic Acids Res 33:5045–5052.


Frontini, M., Soutoglou, E., Argentini, M., Bole-Feysot, C., Jost, B., Scheer, E., and Tora, L. 2005. TAF9b (formerly TAF9L) is a bona fide TAF that has unique and overlapping roles with TAF9. Mol Cell Biol 25:4638–4649.

Fukuda, A., Tokonabe, S., Hamada, M., Matsumoto, M., Tsukui, T., Nogi, Y., and Hisatake, K. 2003. Alleviation of PC4-mediated transcriptional repression by the ERCC3 helicase activity of general transcription factor TFIIH. J Biol Chem 278:14827–14831.

Fukuda, A., Nakadai, T., Shimada, M., Tsukui, T., Matsumoto, M., Nogi, Y., Meisterernst, M., and Hisatake, K. 2004. Transcriptional coactivator PC4 stimulates promoter escape and facilitates transcriptional synergy by GAL4-VP16. Mol Cell Biol 24:6525–6535.

Funk, J.D., Nedialkov, Y.A., Xu, D., and Burton, Z.F. 2002. A key role for the α1 helix of human RAP74 in the initiation and elongation of RNA chains. J Biol Chem 277:46998–47003.

Gadbois, E.L., Chao, D.M., Reese, J.C., Green, M.R., and Young, R.A. 1997. Functional antagonism between RNA polymerase II holoenzyme and global negative regulator NC2 in vivo. Proc Natl Acad Sci USA 94:3145–3150.

Gaiser, F., Tan, S., and Richmond, T.J. 2000. Novel dimerization fold of RAP30/RAP74 in human TFIIF at 1.7 Å resolution. J Mol Biol 302:1119–1127.

Gajewska, B., Shcherbik, N., Oficjalska, D., Haines, D.S., and Zoladek, T. 2003. Functional analysis of the human orthologue of the RSP5-encoded ubiquitin protein ligase, hNedd4, in yeast. Curr Genet 43:1–10.

Galagan, J.E., Nusbaum, C., Roy, A., Endrizzi, M.G., Macdonald, P., FitzHugh, W., Calvo, S., Engels, R., Smirnov, S., Atwood, D., Brown, A., Allen, N., Naylor, J., Stange-Thomann, N., DeArellano, K., Johnson, R., Linton, L., McEwan, P., McKernan, K., Talamas, J., Tirrell, A., Ye, W., Zimmer, A., Barber, R.D., Cann, I., Graham, D.E., Grahame, D.A., Guss, A.M., Hedderich, R., Ingram-Smith, C., Kuettner, H.C., Krzycki, J.A., Leigh, J.A., Li, W., Liu, J., Mukhopadhyay, B., Reeve, J.N., Smith, K., Springer, T.A., Umayam, L.A., White, O., White, R.H., Conway de Macario, E., Ferry, J.G., Jarrell, K.F., Jing, H., Macario, A.J., Paulsen, I., Pritchett, M., Sowers, K.R., Swanson, R.V., Zinder, S.H., Lander, E., Metcalf, W.W., and Birren, B. 2002. The genome of *M. acetivorans* reveals extensive metabolic and physiological diversity. Genome Res 12:532–542.

Galan, J.M. and Haguenauer-Tsapis, R. 1997. Ubiquitin lys63 is involved in ubiquitination of a yeast plasma membrane protein. EMBO J 16:5847–5854.

Gangloff, Y.-G., Romier, C., Thuault, S., Werten, S., and Davidson, I. 2001. The histone fold is a key structural motif of transcription factor TFIID. Trends Biochem Sci 26:250–257.

Garrett, K.P., Serizawa, H., Hanley, J.P., Bradsher, J.N., Tsuboi, A., Arai, N., Yokota, T., Ariai, K., Conaway, R.C., and Conaway, J.W. 1992. The carboxyl terminus of RAP30 is similar in sequence to region 4 of bacterial sigma factors and is require for function. J Biol Chem 267:23942–23949.

Gavva, N.R., Gavva, R., Ermekova, K., Sudol, M., and Shen, C.J. 1997. Interaction of WW domains with hematopoietic transcription factor p45/NF-E2 and RNA polymerase II. J Biol Chem 272:24105–24108.

Ge, H. and Roeder, R.G. 1994a. Purification, cloning, and characterization of a human coactivator, PC4, that mediates transcriptional activation of class II genes. Cell 78:513–523.

Ge, H. and Roeder, R.G. 1994b. The high mobility group protein HMG1 can reversibly inhibit class II gene transcription by interaction with the TATA-binding protein. J Biol Chem 269:17136–17140.

Ge, H., Zhao, Y., Chait, B.T., and Roeder, R.G. 1994. Phosphorylation negatively regulates the function of coactivator PC4. Proc Natl Acad Sci USA 91:12691–12695.

Ge, H., Martinez, E., Chiang, C.-M., and Roeder, R.G. 1996. Activator-dependent transcription by mammalian RNA polymerase II: in vitro reconstitution with general transcription factors and cofactors. Methods Enzymol 274:57–71.

Gegonne, A., Weissman, J.D., and Singer, D.S. 2001. TAFII55 binding to TAFII250 inhibits its acetyltransferase activity. Proc Natl Acad Sci USA 98:12432–12437.

Geiger, J.H., Hahn, S., Lee, S., and Sigler, P.B. 1996. Crystal structure of the yeast TFIIA/TBP/DNA complex. Science 272:830–836.

Geisberg, J.V., Holstege, F.C.P., Young, R.A., and Struhl, K. 2001. Yeast NC2 associates with the RNA polymerase II preinitiation complex and selectively affects transcription in vivo. Mol Cell Biol 21:2736–2742.

Geisberg, J.V., Moqtaderi, Z., Kuras, L., and Struhl, K. 2002. Mot1 associates with transcriptionally active promoters and inhibits the association of NC2 in yeast. Mol Cell Biol 22:8122–8134.

Geisberg, J.V. and Struhl, K. 2004. Cellular stress alters the transcriptional properties of promoter-bound Mot1-TBP complexes. Mol Cell 14:479–489.

Gerard, M., Fischer, L., Moncollin, V., Chipoulet, J.-M., Chambon, P., and Egly, J.-M. 1991. Purification and interaction properties of the human RNA polymerase B(II) general transcription factor BTF2. J Biol Chem 266:20940–20945.

Gershenzon, N.I. and Ioshikhes, I.P. 2005. Synergy of human Pol II core promoter elements revealed by statistical sequence analysis. Bioinformatics 21:1295–1300.

Ghazy, M.A., Brodie, S.A., Ammerman, M.L., Ziegler, L.M., and Ponticelli, A.S. 2004. Amino acid substitutions in yeast TFIIF confer upstream shifts in transcription initiation and altered interaction with RNA polymerase II. Mol Cell Biol 24:10975–10985.

Ghosh, M., Elsby, L.M., Mal, T.K., Gooding, J.M., Roberts, S.G., and Ikura, M. 2004. Probing Zn²⁺-binding effects on the zinc-ribbon domain of human general transcription factor TFIIB. Biochem J 378:317–324.

Giglia-Mari, G., Coin, F., Ranish, J.A., Hoogstraten, D., Theil, A., Wijgers, N., Jaspers, N.G.J., Raams, A., Argentini, M., van der Spek, P.J., Botta, E., Stefanini, M., Egly, J.-M., Aebersold, R., Hoeijmakers, J.H.J., and Vermeulen, W. 2004. A new, tenth subunit of TFIIH is responsible for the DNA repair syndrome trichothiodystrophy group A. Nat Genet 36:714–719.

Gilfillan, S., Stelzer, G., Piaia, E., Hofmann, M.G., and Meisterernst, M. 2005. Efficient binding of NC2·TATA-binding protein to DNA in the absence of TATA. J Biol Chem 280:6222–6230.

Gillette, T.G., Gonzalez, F., Delahodde, A., Johnston, S.A., and Kodadek, T. 2004. Physical and functional association of RNA polymerase II and the proteasome. Proc Natl Acad Sci USA 101:5904–5909.

Glossop, J.A., Dafforn, T.R., and Roberts, S.G.E. 2004. A conformational change in TFIIB is required for activator-mediated assembly of the preinitiation complex. Nucleic Acids Res 32:1829–1835.

Gnatt, A.L., Cramer, P., Fu, J., Bushnell, D.A., and Kornberg, R.D. 2001. Structural basis of transcription: an RNA polymerase II elongation complex at 3.3 Å resolution. Science 292:1876–1882.

Gniazdowski, M., Mandel, J.L. Jr., Gissinger, F., Keding, C., and Chambon, P. 1970. Calf thymus RNA polymerases exhibit template specificity. Biochem Biophys Res Commun 38:1033–1040.

Gobert, C., Skladanowski, A., and Larsen, A.K. 1999. The interaction between p53 and DNA topoisomerase I is regulated differently in cells with wild-type and mutant p53. Proc Natl Acad Sci USA 96:10355–10360.

Goldman-Levi, R., Miller, C., Bogoch, J., and Zak, N.B. 1996. Expanding the Mot1 subfamily: 89B helicase encodes a new Drosophila melanogaster SNF2-related protein which binds to multiple sites on polytene chromosomes. Nucleic Acids Res 24:3121–3128.

Gong, D.-W., Mortin, M.A., Horikoshi, M., and Nakatani, Y. 1995. Molecular cloning of cDNA encoding the small subunit of Drosophila transcription initiation factor TFIIF. Nucleic Acids Res 23:1882–1886.

Goodrich, J.A., Hoey, T., Thut, C.J., Admon, A., and Tjian, R. 1993. Drosophila TAFII40 interacts with both a VP16 activation domain and the basal transcription factor TFIIB. Cell 75:519–530.

Goodrich, J.A. and Tjian, R. 1994. Transcription factors IIE and IIH and ATP hydrolysis direct promoter clearance by RNA polymerase II. Cell 77:145–156.


Goppelt, A. and Meisterernst, M. 1996. Characterization of the basal inhibitor of class II transcription NC2 from Saccharomyces cerevisiae. Nucleic Acids Res 24:4450–4455.

Goppelt, A., Stelzer, G., Lottspeich, F., and Meisterernst, M. 1996. A mechanism for repression of class II gene transcription through specific binding of NC2 to TBP-promoter complexes via heterodimeric histone fold domains. EMBO J 15:3105–3116.

Grant, P.A., Duggan, L., Cote, J., Roberts, S.M., Brownell, J.E., Candau, R., Ohba, R., Owen-Hughes, T., Allis, C.D., Winston, F., Berger, S.L., and Workman, J.L. 1997. Yeast Gcn5 functions in two multisubunit complexes to acetylate nucleosomal histones: characterization of an Ada complex and the SAGA (Spt/Ada) complex. Genes Dev 11:1640–1650.

Grant, P.A., Schieltz, D., Pray-Grant, M.G., Steger, D.J., Reese, J.C., Yates, J.R., III, and Workman, J.L. 1998. A subset of TAFIIs are integral components of the SAGA complex required for nucleosome acetylation and transcriptional stimulation. Cell 94:45–53.

Green, M.R. 2000. TBP-associated factors (TAFIIs): multiple, selective transcriptional mediators in common complexes. Trends Biochem Sci 25:59–63.

Groft, C.M., Uljon, S.N., Wang, R., and Werner, M.H. 1998. Structural homology between the Rap30 DNA-binding domain and linker histone H5: implications for preinitiation complex assembly. Proc Natl Acad Sci USA 95:9117–9122.

Groisman, R., Polanowska, J., Kuraoka, I., Sawada, J., Saijo, M., Drapkin, R., Kisselev, A.F., Tanaka, K., and Nakatani, Y. 2003. The ubiquitin ligase activity in the DDB2 and CSA complexes is differentially regulated by the COP9 signalosome in response to DNA damage. Cell 113:357–367.

Gruber, T.M., and Gross, C.A. 2003. Multiple sigma subunits and the partitioning of bacterial transcription space. Annu Rev Microbiol 57:441–466.

Gu, W., Malik, S., Ito, M., Yuan, C.X., Fondell, J.D., Zhang, X., Martinez, E., Qin, J., and Roeder, R.G. 1999. A novel human SRB/MED-containing cofactor complex SMCC, involved in transcriptional regulation. Mol Cell 397–108.

Guermah, M., Malik, S., and Roeder, R.G. 1998. Involvement of TFIID and USA components in transcriptional activation of the human immunodeficiency virus promoter by NF-κB and Sp1. Mol Cell Biol 18:3234–3244.

Guermah, M., Ge, K., Chiang, C.-M., and Roeder, R.G. 2003. The TBN protein, which is essential for early embryonic mouse development, is an inducible TAFII implicated in adipogenesis. Mol Cell 12:991–1001.

Guglielmi, B., van Berkum, N.L., Klapholz, B., Bijma, T., Boube, M., Boschiero, C., Bourbon, H.M., Holstege, F.C.P., and Werner, M. 2004. A high resolution protein interaction map of the yeast Mediator complex. Nucleic Acids Res 32:5379–5391.

Gumbs, O.H., Campbell, A.M., and Weil, P.A. 2003. High-affinity DNA binding by a Mot1p-TBP complex: implications for TAF-independent transcription. EMBO J 22:3131–3141.

Guzmán, E. and Lis, J.T. 1999. Transcription factor TFIIH is required for promoter melting in vivo. Mol Cell Biol 19:5652–5658.

Ha, I., Lane, W.S., and Reinberg, D. 1991. Cloning of a human gene encoding the general transcription initiation factor IIB. Nature 352:689–695.

Ha, I., Roberts, S., Maldonado, E., Sun, X., Kim, L.U., Green, M., and Reinberg, D. 1993. Multiple functional domains of human transcription factor IIB: distinct interactions with two general transcription factors and RNA polymerase II. Genes Dev 7:1021–1032.

Hahn, S., Buratowski, S., Sharp, P.A., and Guarente, L. 1989. Isolation of the gene encoding the yeast TATA binding protein TFIID: a gene identical to the SPT15 suppressor of Ty element insertions. Cell 58:1173–1181.

Hahn, S. 1998. The role of TAFs in RNA polymerase II transcription. Cell 95:579–582.

Hahn, S. 2004. Structure and mechanism of the RNA polymerase II transcription machinery. Nat Struct Mol Biol 11:394–403.

Hai, T., Horikoshi, M., Roeder, R.G., and Green, M.R. 1988. Analysis of the role of the transcription factor ATF in the assembly of a functional preinitiation complex. *Cell* 54:1043–1051.

Haile, D.T. and Parvin, J.D. 1999. Activation of transcription *in vitro* by the BRCA1 carboxyl-terminal domain. *J Biol Chem* 274:2113–2117.

Hampsey, M. 1998. Molecular genetics of the RNA polymerase II general transcriptional machinery. *Microbiol Mol Biol Rev* 62:465–503.

Han, S., Xie, W., Hammes, S.R., and DeJong, J. 2003. Expression of the germ cell-specific transcription factor ALF in Xenopus oocytes compensates for translational inactivation of the somatic factor TFIIA. *J Biol Chem* 278:45586–45593.

Hansen, S.K. and Tjian, R. 1995. TAFs and TFIIA mediate differential utilization of the tandem Adh promoters. *Cell* 82:565–575.

Hansen, S.K., Takada, S., Jacobson, R.H., Lis, J.T., and Tjian, R. 1997. Transcription properties of a cell type-specific TATA-binding protein, TRF. *Cell* 91:71–83.

Hardy, S., Brand, M., Mittler, G., Yanagisawa, J., Kato, S., Meisterernst, M., and Tora, L. 2002. TATA-binding protein-free TAF-containing complex (TFTC) and p300 are both required for efficient transcriptional activation. *J Biol Chem* 277:32875–32882.

Hashizume, R., Fukuda, M., Maeda, I., Nishikawa, H., Oyake, D., Yabuki, Y., Ogata, H., and Ohta, T. 2001. The RING heterodimer BRCA1-BARD1 is a ubiquitin ligase inactivated by a breast cancer-derived mutation. *J Biol Chem* 276:14537–14540.

Hassa, P.O., Buerki, C., Lombardi, C., Imhof, R., and Hottiger, M.O. 2003. Transcriptional coactivation of nuclear factor-κB-dependent gene expression by p300 is regulated by poly(ADP)-ribose polymerase-1. *J Biol Chem* 278:45145–45153.

Hassa, P.O., Haenni, S.S., Buerki, C., Meier, N.I., Lane, W.S., Owen, H., Gersbach, M., Imhof, R., and Hottiger, M.O. 2005. Acetylation of PARP-1 by p300/CBP regulates coactivation of NF-κB-dependent transcription. *J Biol Chem* 280:40450–40464.

Hawkes, N.A. and Roberts, S.G.E. 1999. The role of human TFIIB in transcription start site selection *in vitro* and *in vivo*. *J Biol Chem* 274:14337–14343.

Hawkes, N.A., Evans, R., and Roberts, S.G.E. 2000. The conformation of the transcription factor TFIIB modulates the response to transcriptional activators *in vivo*. *Curr Biol* 10:273–276.

Hawley, D.K. and Roeder, R.G. 1985. Separation and partial characterization of three functional steps in transcription initiation by human RNA polymerase II. *J Biol Chem* 260:8163–8172.

Hayashida, T., Sekiguchi, T., Noguchi, E., Sunamoto, H., Ohba, T., and Nishimoto, T. 1994. The CCG1/TFIID250 gene is mutated in thermosensitive G1 mutants of the BHK21 cell line derived from golden hamster. *Gene* 141:267–270.

Heberlein, U., England, B., and Tjian, R. 1985. Characterization of Drosophila transcription factors that activate the tandem promoters of the alcohol dehydrogenase gene. *Cell* 41:965–977.

Helmlinger, D., Hardy, S., Sasorith, S., Klein, F., Robert, F., Weber, C., Miquet, L., Potier, N., Van-Dorsselaer, A., Wurtz, J.M., Mandel, J.L., Tora, L., and Devys, D. 2004. Ataxin-7 is a subunit of GCN5 histone acetyltransferase-containing complexes. *Hum Mol Genet* 13:1257–1265.

Hengartner, C.J., Myer, V.E., Liao, S.-M., Wilson C.J., Koh, S.S., and Young, R.A. 1998. Temporal regulation of RNA polymerase II by Srb10 and Kin28 cyclin-dependent kinases. *Mol Cell* 2:43–53.

Henry, N.L., Sayre, M.H., and Kornberg, R.D. 1992. Purification and characterization of yeast RNA polymerase II general initiation factor g. *J Biol Chem* 267:23388–23392.

Henry, N.L., Campbell, A.M., Feaver, W.J., Poon, D., Weil, P.A., and Kornberg, R.D. 1994. TFIIF-TAF-RNA polymerase II connection. *Genes Dev.* 8:2868–2878.

Henry, K.W., Wyce, A., Lo, W.S., Duggan, L.J., Emre, N.C., Kao, C.F., Pillus, L., Shilatifard, A., Osley, M.A., and Berger, S.L. 2003. Transcriptional activation via sequential histone H2B ubiquitylation and deubiquitylation, mediated by SAGA-associated Ubp8. *Genes Dev* 17:2648–2663.

---

Hernandez, N. 1993. TBP, a universal eukaryotic transcription factor? *Genes Dev* 7:1291–1308.

Herr, A.J., Jensen, M.B., Dalmay, T., and Baulcombe, D.C. 2005. RNA polymerase IV directs silencing of endogenous DNA. *Science* 308:118–120.

Hiller, M.A., Lin, T.Y., Wood, C., and Fuller, M.T. 2001. Developmental regulation of transcription by a tissue-specific TAF homolog. *Genes Dev.* 15:1021–1030.

Hilton, T.L., Li, Y., Dunphy, E.L., and Wang, E.H. 2005. TAF1 histone acetyltransferase activity in Sp1 activation of the cyclin D1 promoter. *Mol Cell Biol* 25:4321–4332.

Hisatake, K., Hasegawa, S., Takada, R., Nakatani, Y., Horikoshi, M., and Roeder, R.G. 1993. The p250 subunit of native TATA box-binding factor TFIID is the cell-cycle regulatory protein CCG1. *Nature* 362:179–181.

Hisatake, K., Ohta, T., Takada, R., Guermah, M., Horikoshi, M., Nakatani, Y., and Roeder, R.G. 1995. Evolutionary conservation of human TATA-binding-polypeptide-associated factors TAFII31 and TAFII80 and interactions of TAFII80 with other TAFs and with general transcription factors. *Proc Natl Acad Sci USA* 92:8195–8199.

Hochheimer, A. and Tjian, R. 2003. Diversified transcription initiation complexes expand promoter selectivity and tissue-specific gene expression. *Genes Dev* 17:1309–1320.

Hoege, C., Pfander, B., Moldovan, G.L., Pyrowolakis, G., and Jentsch, S. 2002. RAD6-dependent DNA repair is linked to modification of PCNA by ubiquitin and SUMO. *Nature* 419:135–141.

Hoey, T., Dynlacht, B.D., Peterson, M.G., Pugh, B.F., and Tjian, R. 1990. Isolation and characterization of the Drosophila gene encoding the TATA box binding protein, TFIID. *Cell* 61:1179–1186.

Hoey, T., Weinzierl, R.O., Gill, G., Chen, J.-L., Dynlacht, B.D., and Tjian, R. 1993. Molecular cloning and functional analysis of Drosophila TAF110 reveal properties expected of coactivators. *Cell* 72:247–260.

Hoffmann, A., Horikoshi, M., Wang, C.K., Schroeder, S., Weil, P.A., and Roeder, R.G. 1990a. Cloning of the Schizosaccharomyces pombe TFIID gene reveals a strong conservation of functional domains present in Saccharomyces cerevisiae TFIID. *Genes Dev* 4:1141–1148.

Hoffman, A., Sinn, E., Yamamoto, T., Wang, J., Roy, A., Horikoshi, M., and Roeder, R.G. 1990b. Highly conserved core domain and unique N terminus with presumptive regulatory motifs in a human TATA factor (TFIID). *Nature* 346:387–390.

Hoffmann, A., Chiang, C.-M., Oelgeschläger, T., Xie, X., Burley, S.K., Nakatani, Y., and Roeder, R.G. 1996. A histone octamer-like structure within TFIID. *Nature* 380:356–359.

Hofmann, R.M. and Pickart, C.M. 1999. Noncanonical MMS2-encoded ubiquitin-conjugating enzyme functions in assembly of novel polyubiquitin chains for DNA repair. *Cell* 96:645–653.

Høiby, T., Mitsiou, D.J., Zhou, H., Erdjument-Bromage, H., Tempst, P., and Stunnenberg, H.G. 2004. Cleavage and proteasome-mediated degradation of the basal transcription factor TFIIA. *EMBO J* 23:3083–3091.

Holloway, A.F., Occhiodoro, F., Mittler, G., Meisterernst, M., and Shannon, M.F. 2000. Functional interaction between the HIV transactivator Tat and the transcriptional coactivator PC4 in T cells. *J Biol Chem* 275:21668–21677.

Holmes, M.C. and Tjian, R. 2000. Promoter-selective properties of the TBP-related factor TRF1. *Science* 288:867–870.

Holstege, F.C.P., van der Vliet, P.C., and Timmers H.T.M. 1996. Opening of an RNA polymerase II promoter occurs in two distinct steps and requires the basal transcription factors IIe and IIH. *EMBO J* 15:1666–1677.

Holstege, F.C.P., Fiedler, U., and Timmers, H.T.M. 1997. Three transitions in the RNA polymerase II transcription complex during initiation. *EMBO J* 16:7468–7480.

Holstege, F.C.P., Jennings, E.G., Wyrick, J.J., Lee, T.I., Hengartner, C.J., Green, M.R., Golub, T.R., Lander, E.S., and Young, R.A. 1998. Dissecting the regulatory circuitry of a eukaryotic genome. *Cell* 95:717–728.

Hon, W.C., Wilson, M.I., Harlos, K., Claridge, T.D., Schofield, C.J., Pugh., C.W., Maxwell, P.H., Ratcliffe, P.J., Stuart, D.I., and Jones, E.Y. 2002. Structural basis for the recognition of hydroxyproline in HIF-1α by pVHL. *Nature* 417:975–978.

Horikoshi, M., Hai, T., Lin, Y.-S., Green, M.R., and Roeder, R.G. 1988. Transcription factor ATF interacts with the TATA factor to facilitate establishment of a preinitiation complex. *Cell* 54:1033–1042.

Horikoshi, M., Wang, C.K., Fujii, H., Cromlish, J.A., Weil, P.A., and Roeder, R.G. 1989a. Cloning and structure of a yeast gene encoding a general transcription initiation factor TFIID that binds to the TATA box. *Nature* 341:299–303.

Horikoshi, M., Wang, C.K., Fujii, H., Cromlish, J.A., Weil, P.A., and Roeder, R.G. 1989b. Purification of a yeast TATA box-binding protein that exhibits human transcription factor IID activity. *Proc Natl Acad Sci USA* 86:4843–4847.

Horikoshi, M., Bertuccioli, C., Takada, R., Wang, J., Yamamoto, T., and Roeder, R.G. 1992. Transcription factor TFIID induces DNA bending upon binding to the TATA element. *Proc Natl Acad Sci USA* 89:1060–1064.

Hou, S.Y., Wu, S.-Y., Zhou, T., Thomas, M.C., and Chiang, C.-M. 2000. Alleviation of human papillomavirus E2-mediated transcriptional repression via formation of a TATA binding protein (or TFIID)-TFIIB-RNA polymerase II-TFIIIF preinitiation complex. *Mol Cell Biol* 20:113–125.

Hou, S.Y., Wu, S.-Y., and Chiang, C.-M. 2002. Transcriptional activity among high and low risk human papillomavirus E2 proteins correlates with E2 DNA binding. *J Biol Chem* 277:45619–45629.

Huang, J., Zhao, Q., Mooney, S.M., and Lee, F.S. 2002. Sequence determinants in hypoxia-inducible factor-1α for hydroxylation by the prolyl hydroxylases PHD1, PHD2, and PHD3. *J Biol Chem* 277:39792–39800.

Huibregtse, J.M., Yang, J.C., and Beaudenon, S.L. 1997. The large subunit of RNA polymerase II is a substrate of the Rsp5 ubiquitin-protein ligase. *Proc Natl Acad Sci USA* 94:3656–3661.

Huisenga K.L., and Pugh, B.F. 2004. A genome-wide housekeeping role for TFIID and a highly regulated stress-related role for SAGA in Saccharomyces cerevisiae. *Mol Cell* 13:573–585.

Humbert, S., van Vuuren, H., Lutz, Y., Hoeijmakers, J.H.J., Egly, J.-M., and Moncollin, V. 1994. p44 and p34 subunits of the Btf2/TFIIH transcription factor have homologies with Ssl1, a yeast protein involved in DNA repair. *EMBO J* 13:2393–2398.

Hurwitz, J., Bresler, A., and Diringer, R. 1960. The enzymic incorporation of ribonucleotides into polyribonucleotides and the effect of DNA. *Biochem Biophys Res Commun* 3:15–18.

Icard-Liepkalns, C. 1993. Binding activity of the human transcription factor TFIID. *Biochem Biophys Res Commun* 193:453–459.

Imbalzano, A.N., Kwon, H., Green, M.R., and Kingston, R.E. 1994a. Facilitated binding of TATA-binding protein to nucleosomal DNA. *Nature* 370:481–485.

Imbalzano, A.N., Zaret, K.S., and Kingston, R.E. 1994b. Transcription factor (TF) IIB and TFIIA can independently increase the affinity of the TATA-binding protein for DNA. *J Biol Chem* 269:8280–8286.

Imhof, M.O., and McDonnell, D.P. 1996. Yeast RSP5 and its human homolog hRPF1 potentiate hormone-dependent activation of transcription by human progesterone and glucocorticoid receptors. *Mol Cell Biol* 16:2594–2605.

Imhof, A., Yang, X.J., Ogryzko, V.V., Nakatani, Y., Wolffe, A.P., and Ge, H. 1997. Acetylation of general transcription factors by histone acetyltransferases. *Curr Biol* 7:689–692.

Ingvarsdottir, K., Krogan, N.J., Emre, N.C., Wyce, A., Thompson, N.J., Emili, A., Hughes, T.R., Greenblatt, J.F., and Berger, S.L. 2005. H2B ubiquitin protease Ubp8 and Sgf11 constitute a discrete functional module within the Saccharomyces cerevisiae SAGA complex. *Mol Cell Biol* 25:1162–1172.

Inostroza, J.A., Mermelstein, F.H., Ha, I., Lane, W.S., and Reinberg, D. 1992. Dr1, a TATA-binding protein-associated phosphoprotein and inhibitor of class II gene transcription. *Cell* 70:477–489.

Inukai, N., Yamaguchi, Y., Kuraoka, I., Yamada, T., Kamijo, S., Kato, J., Tanaka, K., and Handa, H. 2004. A novel hydrogen peroxide-

induced phosphorylation and ubiquitination pathway leading to RNA polymerase II proteolysis. *J Biol Chem* 279:8190–8195.

Ito, M., Yuan, C.-X., Malik, S., Gu, W., Fondell, J.D., Yamamura, S., Fu, Z.-Y., Zhang, X., Qin, J., and Roeder, R.G. 1999. Identity between TRAP and SMCC complexes indicates novel pathways for the function of nuclear receptors and diverse mammalian activators. *Mol Cell* 3:361–370.

Ito, M., Okano, H.J., Darnell, R.B., and Roeder, R.G. 2002. The TRAP100 component of the TRAP/Mediator complex is essential in broad transcriptional events and development. *EMBO J* 21:3464–3475.

Iwai, K., Yamanaka, K., Kamura, T., Minato, N., Conaway, R.C., Conaway, J.W., Klausner, R.D., and Pause, A. 1999. Identification of the von Hippel-Lindau tumor-suppressor protein as part of an active E3 ubiquitin ligase complex. *Proc Natl Acad Sci USA* 96:12436–12441.

Ivan, M., Kondo, K., Yang, H., Kim, W., Valiando, J., Ohh, M., Salic, A., Asara, J.M., Lane, W.S., and Kaelin, W.G. Jr. 2001. HIF-α targeted for VHL-mediated destruction by proline hydroxylation: implications for O₂ sensing. *Science* 292:464–468.

Iyer, S.P., Akimoto, Y., and Hart, G.W. 2003. Identification and cloning of a novel family of coiled-coil domain proteins that interact with O-GlcNAc transferase. *J Biol Chem* 278:5399–5409.

Jaakkola, P., Mole, D.R., Tian, Y.M., Wilson, M.I., Gielbert, J., Gaskell, S.J., Kriegsheim, Av, Hebestreit, H.F., Mukherji, M., Schofield, C.J., Maxwell, P.H., Pugh, C.W., and Ratcliffe, P.J. 2001. Targeting of HIF-α to the von Hippel-Lindau ubiquitylation complex by O₂-regulated prolyl hydroxylation. *Science* 292:468–472.

Jackson-Fisher, A.J., Chitikila, C., Mitra, M., and Pugh, B.F. 1999. A role for TBP dimerization in preventing unregulated gene expression. *Mol Cell* 3:717–727.

Jacobson, R.H., Ladurner, A.G., King, D.S., and Tjian, R. 2000. Structure and function of a human TAFII250 double bromodomain module. *Science* 288:1422–1425.

Jazwinski, S.M. 2005. The retrograde response links metabolism with stress responses, chromatin-dependent gene activation, and genome stability in yeast aging. *Gene* 354:22–27.

Jawahri, A., Lainé, J.-P., Dubaele, S., Lamour, V., Poterszman, A., Coin, F., Moras, D., and Egly, J.-M. 2002. p52 mediates XPB function within the transcription/repair factor TFIIH. *J Biol Chem* 277:31761–31767.

Jensen, J.P., Bates, P.W., Yang, M., Vierstra, R.D., and Weissman, A.M. 1995. Identification of a family of closely related human ubiquitin conjugating enzymes. *J Biol Chem* 270:30408–30414.

Johnson, E.S., Ma, P.C., Ota, I.M., and Varshavsky, A. 1995. A proteolytic pathway that recognizes ubiquitin as a degradation signal. *J Biol Chem* 270:17442–17456.

Joliot, V., Demma, M., and Prywes, R. 1995. Interaction with RAP74 subunit of TFIIIF is required for transcriptional activation by serum response factor. *Nature* 373:632–635.

Jones, K.A. 1997. Taking a new TAK on tat transactivation. *Genes Dev* 11:2593–2599.

Ju, B.-G., Solum, D., Song, E.J., Lee, K.-J., Rose, D.W., Glass, C.K., and Rosenfeld, M.G. 2004. Activating the PARP-1 sensor component of the groucho/TLE1 corepressor complex mediates a CaMKinase IIδ-dependent neurogenic gene activation pathway. *Cell* 119:815–829.

Jung, Y. and Lippard, S.J. 2005. RNA polymerase II blockage by cisplatin-damaged DNA: stability and polyubiquitylation of stalled polymerase. *J Biol Chem* 281:1361–1370.

Juo, Z.S., Chiu, T.K., Leiberman, P.M., Baikalov, I., Berk, A.J., and Dickerson, R.E. 1996. How proteins recognize the TATA box. *J Mol Biol* 261:239–254.

Kaiser, K., Stelzer, G., and Meisterernst, M. 1995. The coactivator p15 (PC4) initiates transcriptional activation during TFIIA-TFIID-promoter complex formation. *EMBO J* 14:3520–3527.

Kaiser, K. and Meisterernst, M. 1996. The human general co-factors. *Trends Biochem Sci* 21:342–345.

Kaltenbach, L., Horner, M.A., Rothman, J.H., and Mango, S.E. 2000. The TBP-like factor CeTLF is required to activate RNA polymerase II transcription during C. elegans embryogenesis. *Mol Cell* 6:705–713.

Kamada, K., De Angelis, J., Roeder, R.G., and Burley, S.K. 2001a. Crystal structure of the C-terminal domain of the RAP74 subunit of human transcription factor IIF. *Proc Natl Acad Sci USA* 98:3115–3120.

Kamada, K., Shu, F., Chen, H., Malik, S., Stelzer, G., Roeder, R.G., Meisterernst, M., and Burley, S.K. 2001b. Crystal structure of negative cofactor 2 recognizing the TBP-DNA transcription complex. *Cell* 106:71–81.

Kamura, T., Koepp, D.M., Conrad, M.N., Skowyra, D., Moreland, R.J., Iliopoulos, O., Lane, W.S., Kaelin, W.G., Jr., Elledge, S.J., Conaway, R.C., Harper, J.W., and Conaway, J.W. 1999. Rbx1, a component of the VHL tumor suppressor complex and SCF ubiquitin ligase. *Science* 284:657–661.

Kang, M.E. and Dahmus, M.E. 1993. RNA polymerases II A and II O have distinct roles during transcription from the TATA-less murine dihydrofolate reductase promoter. *J Biol Chem* 268:25033–25040.

Kang, J.J., Auble, D.T., Ranish, J.A., and Hahn, S. 1995. Analysis of yeast transcription factor TFIIA: distinct functional regions and a polymerase II-specific role in basal and activated transcription. *Mol Cell Biol* 15:1234–1243.

Kang, J.S., Kim, S.H., Hwang, M.S., Han, S.J., Lee, Y.C., and Kim, Y.-J. 2001. The structural and functional organization of the yeast mediator complex. *J Biol Chem* 276:42003–42010.

Kannan, P. and Tainsky, M.A. 1999. Coactivator PC4 mediates AP-2 transcriptional activity and suppresses ras-induced transformation dependent on AP-2 transcriptional interference. *Mol Cell Biol* 19:899–908.

Kannan, P., Yu, Y., Wankhade, S., and Tainsky, M.A. 1999. PolyADP-ribose polymerase is a coactivator for AP-2-mediated transcriptional activation. *Nucleic Acids Res* 27:866–874.

Kanno, T., Kanno, Y., Siegel, R.M., Jang, M.K., Lenardo, M.J., and Ozato, K. 2004. Selective recognition of acetylated histones by bromodomain proteins visualized in living cells. *Mol Cell* 13:33–43.

Kanno, T., Huettel, B., Mette, M.F., Aufsatz, W., Jaligot, E., Daxinger, L., Kreil, D.P., Matzke, M., and Matzke, A.J.M. 2005. Atypical RNA polymerase subunits required for RNA-directed DNA methylation. *Nature Genetics* 37:761–765.

Kao, C.C., Lieberman, P.M., Schmidt, M.C., Zhou, Q., Pei, R., and Berk, A.J. 1990. Cloning of a transcriptionally active human TATA binding factor. *Science* 248:1646–1650.

Kato, K., Makino, Y., Kishimoto, T., Yamauchi, J., Kato, S., Muramatsu, M., and Tamura, T. 1994. Multimerization of the mouse TATA-binding protein (TBP) driven by its C-terminal conserved domain. *Nucleic Acids Res* 22:1179–1185.

Kedinger, C., Gniazdowski, M., Mandel, J.L. Jr, Gissinger, F., and Chambon, P. 1970. Alpha-amanitin: a specific inhibitor of one of two DNA-pendent RNA polymerase activities from calf thymus. *Biochem Biophys Res Commun* 38:165–171.

Kedinger, C., Nuret, P., and Chambon, P. 1971. Structural evidence for two α-amanitin sensitive RNA polymerases in calf thymus. *FEBS Lett* 15:169–174.

Kee, Y., Lyon, N., and Huibregtse, J.M. 2005. The Rsp5 ubiquitin ligase is coupled to and antagonized by the Ubp2 deubiquitinating enzyme. *EMBO J* 24:2414–2424.

Kellenberger, E., Dominguez, C., Fribourg, S., Wasielewski, E., Moras, D., Poterszman, A., Boelens, R., and Kieffer, B. 2005. Solution structure of the C-terminal domain of TFIIH P44 subunit reveals a novel type of C4C4 ring domain involved in protein-protein interactions. *J Biol Chem* 280:20785–20792.

Kelly, W.G., Dahmus, M.E., and Hart, G.W. 1993. RNA polymerase II is a glycoprotein. Modification of the COOH-terminal domain by O-GlcNAc. *J Biol Chem* 268:10416–10424.

Kephart, D.D., Price, M.P., Burton, Z.F., Finkelstein, A., Greenblatt, J., and Price, D.H. 1993. Cloning of a *Drosophila* cDNA with sequence similarity to human transcription factor RAP74. *Nucleic Acids Res* 21:1319.

Keriel, A., Stary, A., Sarasin, A., Rochette-Egly, C., and Egly, J.-M. 2002. XPD mutations prevent TFIIE-dependent transactivation by nuclear receptors and phosphorylation of RARα. *Cell* 109:125–135.

The General Transcription Machinery and General Cofactors

Kershner, E., Wu, S.-Y., and Chiang, C.-M. 1998. Immunoaffinity purification and functional characterization of human transcription factor IIH and RNA polymerase II from clonal cell lines that conditionally express epitope-tagged subunits of the multiprotein complexes. *J Biol Chem* 273:34444–34453.

Kettenberger, H., Armache, K.-J., and Cramer, P. 2003. Architecture of the RNA polymerase II-TFIIS complex and implications for mRNA cleavage. *Cell* 114:347–357.

Kettenberger, H., Armache, K.-J., and Cramer, P. 2004. Complete RNA polymerase II elongation complex structure and its interactions with NTP and TFIIS. *Mol Cell* 16:955–965.

Khazak, V., Estojak, J., Cho, H., Majors, J., Sonoda, G., Testa, J.R., and Golemis, E.A. 1998. Analysis of the interaction of the novel RNA polymerase II (pol II) subunit hsRPB4 with its partner hsRPB7 and with pol II. *Mol Cell Biol* 18:1935–1945.

Kim, W.-Y. and Dahmus, M.E. 1989. The major late promoter of adenovirus-2 is accurately transcribed by RNA polymerase II O, II A, and II B. *J Biol Chem* 264:3169–3176.

Kim, J.L., Nikolov, D.B., and Burley, S.K. 1993a. Co-crystal structure of TBP recognizing the minor groove of a TATA element. *Nature* 365:520–527.

Kim, Y., Geiger, J.H., Hahn, S., and Sigler, P.B. 1993b. Crystal structure of a yeast TBP/TATA-box complex. *Nature* 365:512–520.

Kim, J.L. and Burley, S.K. 1994. 1.9 Å resolution refined structure of TBP recognizing the minor groove of TATAAAAG. *Nat Struct Biol* 1:638–653.

Kim, Y.-J., Björklund, S., Li, Y., Sayre, M.H., and Kornberg, R.D. 1994. A multiprotein mediator of transcriptional activation and its interaction with the C-terminal repeat domain of RNA polymerase II. *Cell* 77:599–608.

Kim, T.K., Zhao, Y., Ge, H., Bernstein, R., and Roeder, R.G. 1995. TATA-binding protein residues implicated in a functional interplay between negative cofactor NC2 (Dr1) and general factors TFIIA and TFIIIB. *J Biol Chem* 270:10976–10981.

Kim, T.K., Ebright, R.H., and Reinberg, D. 2000. Mechanism of ATP-dependent promoter melting by transcription factor IIH. *Science* 288:1418–1422.

Kim, M.Y., Mauro, S., Gevry, N., Lis, J.T., and Kraus, W.L. 2004. NAD⁺-dependent modulation of chromatin structure and transcription by nucleosome binding properties of PARP-1. *Cell* 119:803–814.

Kim, T.H., Barrera, L.O., Zheng, M., Qu, C., Singer, M.A., Richmond, T.A., Wu, Y., Green, R.D., and Ren, B. 2005a. A high-resolution map of active promoters in the human genome. *Nature* 436:876–880.

Kim, Y.J. and Lis, J.T. 2005b. Interactions between subunits of *Drosophila* Mediator and activator proteins. *Trends Biochem Sci* 30:245–249.

Kim, M.Y., Zhang, T., and Kraus, W.L. 2005b. Poly(ADP-ribosyl)ation by PARP-1: 'PAR-laying' NAD⁺ into a nuclear signal. *Genes Dev* 19:1951–1967.

Kimura, M., Suzuki, H., and Ishihama, A. 2002. Formation of a carboxy-terminal domain phosphatase (FCP1)/TFIIF/RNA polymerase II (pol II) complex in *Schizosaccharomyces pombe* involves direct interaction between Fcp1 and the Rpb4 subunit of pol II. *Mol Cell Biol* 22:1577–1588.

Kimura, M. and Ishihama, A. 2004. Tfg3, a subunit of the general transcription factor TFIIF in *Schizosaccharomyces pombe*, functions under stress conditions. *Nucleic Acids Res* 32:6706–6715.

King, I.F., Francis, N.J., and Kingston, R.E. 2002. Native and recombinant polycomb group complexes establish a selective block to template accessibility to repress transcription in vitro. *Mol Cell Biol* 22:7919–7928.

Kirschner, D.B., vom Baur, E., Thibault, C., Sanders, S.L., Gangloff, Y.G., Davidson, I., Weil, P.A., and Tora, L. 2002. Distinct mutations in yeast TAFII25 differentially affect the composition of TFIID and SAGA complexes as well as global gene expression patterns. *Mol Cell Biol* 22:3178–3193.

Klejman, M.P., Pereira, L.A., van Zeeburg, H.J., Gilfillan, S., Meisterernst, M., and Timmers, H.T.M. 2004. NC2α interacts with BTAF1 and stimulates its ATP-dependent association with TATA-binding protein. *Mol Cell Biol* 24:10072–10082.

Klejman, M.P., Zhao, X., van Schaik, F.M.A., Herr, W., and Timmers, H.T.M. 2005. Mutational analysis of BTAF1-TBP interaction: BTAF1 can rescue DNA-binding defective TBP mutants. *Nucleic Acids Res* 33:5426–5436.

Kleiman, F.E., Wu-Baer, F., Fonseca, D., Kaneko, S., Baer, R., and Manley, J.L. 2005. BRCA1/BARD1 inhibition of mRNA 3′ processing involves targeted degradation of RNA polymerase II. *Genes Dev* 19:1227–1237.

Kobayashi, N., Boyer, T.G., and Berk, A.J. 1995. A class of activation domains interacts directly with TFIIA and stimulates TFIIA-TFIID-promoter complex assembly. *Mol Cell Biol* 15:6465–6473.

Kobor, M.S., Simon, L.D., Omichinski, J., Zhong, G., Archambault, J., and Greenblatt, J. 2000. A motif shared by TFIIF and TFIIB mediates their interaction with the RNA polymerase II carboxy-terminal domain phosphatase Fcp1p in Saccharomyces cerevisiae. *Mol Cell Biol* 20:7438–7449.

Koiwa, H., Hausmann, S., Bang, W.Y., Ueda, A., Kondo, N., Hiraguri, A., Fukuhara, T., Bahk, J.D., Yun, D.-J., Bressan, R.A., Hasegawa, P.M., and Shuman, S. 2004. Arabidopsis C-terminal domain phosphatase-like 1 and 2 are essential Ser-5-specific C-terminal domain phosphatases. *Proc Natl Acad Sci USA* 101:14539–14544.

Kokubo, T., Gong, D.W., Yamashita, S., Horikoshi, M., Roeder, R.G., and Nakatani, Y. 1993. Drosophila 230-kD TFIID subunit, a functional homolog of the human cell cycle gene product, negatively regulates DNA binding of the TATA box-binding subunit of TFIID. *Genes Dev* 7:1033–1046.

Kokubo, T., Swanson, M.J., Nishikawa, J.-I., Hinnebusch, A.G., and Nakatani, Y. 1998. The yeast TAF145 inhibitory domain and TFIIA competitively bind to TATA-binding protein. *Mol Cell Biol* 18:1003–1012.

Koleske, A.J. and Young, R.A. 1994. An RNA polymerase II holoenzyme responsive to activators. *Nature* 368:466–469.

Komarnitsky, P., Cho, E.-J., and Buratowski, S. 2000. Different phosphorylated forms of RNA polymerase II and associated mRNA processing factors during transcription. *Genes Dev* 14:2452–2460.

Kornberg, R.D. 2005. Mediator and the mechanism of transcriptional activation. *Trends Biochem Sci* 30:235–239.

Kou, H., Irvin, J.D., Huisenga, K.L., Mitra, M., and Pugh, B.F. 2003. Structural and functional analysis of mutations along the crystallographic dimer interface of the yeast TATA binding protein. *Mol Cell Biol* 23:3186–3201.

Kraemer, S.M., Ranallo, R.T., Ogg, R.C., and Stargell, L.A. 2001. TFIIA interacts with TFIID via association with TATA-binding protein and TAF40. *Mol Cell Biol* 21:1737–1746.

Kraus, W.L. and Lis, J.T. 2003. PARP goes transcription. *Cell* 113:677–683.

Kravchenko, J.E., Rogozin, I.B., Koonin, E.V., and Chumakov, P.M. 2005. Transcription of mammalian messenger RNAs by a nuclear RNA polymerase of mitochondrial origin. *Nature* 436:735–739.

Kretzschmar, M., Meisterernst, M., and Roeder, R.G. 1993. Identification of human DNA topoisomerase I as a cofactor for activator-dependent transcription by RNA polymerase II. *Proc Natl Acad Sci USA* 90:11508–11512.

Kretzschmar, M., Kaiser, K., Lottspeich, F., and Meisterernst, M. 1994. A novel mediator of class II gene transcription with homology to viral immediate-early transcriptional regulators. *Cell* 78:525–534.

Krishnamurthy, S., He, X., Reyes-Reyes, M., Moore, C., and Hampsey, M. 2004. Ssu72 is an RNA polymerase II CTD phosphatase. *Mol Cell* 14:387–394.

Krum, S.A., Miranda, G.A., Lin, C., and Lane, T.F. 2003. BRCA1 associates with processive RNA polymerase II. *J Biol Chem* 278:52012–52020.

Kugel, J.F. and Goodrich, J.A. 1998. Promoter escape limits the rate of RNA polymerase II transcription and is enhanced by TFIIE, TFIH, and ATP on negatively supercoiled DNA. *Proc Natl Acad Sci USA* 95:9232–9237.

Kuldell, N.H. and Buratowski, S. 1997. Genetic analysis of the large subunit of yeast transcription factor IIE reveals two regions with distinct functions. *Mol Cell Biol* 17:5288–5298.


Kumar, K.P., Akoulitchev, S., and Reinberg, D. 1998. Promoter-proximal stalling results from the inability to recruit transcription factor IIH to the transcription complex and is a regulated event. *Proc Natl Acad Sci USA* 95:9767–9772.

Kumar, B.R.P., Swaminathan, V., Banerjee, S., and Kundu, T. 2001. p300-mediated acetylation of human transcriptional coactivator PC4 is inhibited by phosphorylation. *J Biol Chem* 276:16804–16809.

Kuras, L., Kosa, P., Mencia, M., and Struhl, K. 2000 TAF-containing and TAF-independent forms of transcriptionally active TBP in vivo. *Science* 288:1244–1248.

Kus, B., Gajadhar, A., Stanger, K., Cho, R., Sun, W., Rouleau, N., Lee, T., Chan, D., Wolting, C., Edwards, A., Bosse, R., and Rotin, D. 2005. A high throughput screen to identify substrates for the ubiquitin ligase Rsp5. *J Biol Chem* 280:29470–29478.

Kutach, A. and Kadonaga, J.T. 2000. The downstream promoter element DPE appears to be as widely used as the TATA box in Drosophila core promoters. *Mol Cell Biol* 20:4754–4764.

Kuzmichev, A., Nishioka, K., Erdjument-Bromage, H., Tempst, P., and Reinberg, D. 2002. Histone methyltransferase activity associated with a human multiprotein complex containing the Enhancer of Zeste protein. *Genes Dev* 16:2893–2905.

Kuzmichev, A., Jenuwein, T., Tempst, P., and Reinberg, D. 2004. Different EZH2-containing complexes target methylation of histone H1 or nucleosomal histone H3. *Mol Cell* 14:183–193.

Kuzmichev, A., Margueron, R., Vaquero, A., Preissner, T.S., Scher, M., Kir- mizis, A., Ouyang, X., Brockdorff, N., Abate-Shen, C., Farnham, P., and Reinberg, D. 2005. Composition and histone substrates of polycomb repressive group complexes change during cellular differentiation. *Proc Natl Acad Sci USA* 102:1859–1864.

Kuznetsova, A.V., Meller, J., Schnell, P.O., Nash, J.A., Ignacak, M.L., Sanchez, Y., Conaway, J.W., Conaway, R.C., and Czyzyk-Krzeska, M.F. 2003. von Hippel-Lindau protein binds hyperphosphorylated large subunit of RNA polymerase II through a proline hydroxylation motif and targets it for ubiquitination. *Proc Natl Acad Sci USA* 100:2706–2711.

Kwek, K.Y., Murphy, S., Furger, A., Thomas, B., O’Gorman, W., Kimura, H., Proudfoot, N.J., and Akoulitchev, A. 2002. U1 snRNA associates with TFIIH and regulates transcriptional initiation. *Nat Struct Biol* 9:800–805.

Lagrange, T., Kim, T.K., Orphanides, G., Ebright, Y.W., Ebright, R.H., and Reinberg, D. 1996. High-resolution mapping of nucleoprotein complexes by site-specific protein-DNA photocrosslinking: organization of the human TBP-TFIIA-TFIIIB-DNA quaternary complex. *Proc Natl Acad Sci USA* 93:10620–10625.

Lagrange, T., Kapanidis, A.N., Tang, H., Reinberg, D., and Ebright, R.H. 1998. New core promoter element in RNA polymerase II-dependent transcription: sequence-specific DNA binding by transcription factor IIB. *Genes Dev* 12:34–44.

Langelier, M.-F., Forget, D., Rojas, A., Porlier, Y., Burton, Z.F., and Coulombe, B. 2001. Structural and functional interactions of transcription factor (TF) IIA with TFIIE and TFIIF in transcription initiation by RNA polymerase II. *J Biol Chem* 276:38652–38657.

Larschan, E. and Winston, F. 2001. The S. cerevisiae SAGA complex functions in vivo as a coactivator for transcriptional activation by Gal4. *Genes Dev* 15:1946–1956.

Lavigne, A.C., Mengus, G., May, M., Dubrovskaya, V., Tora, L., Chambon, P., and Davidson, I. 1996. Multiple interactions between TAFII55 and other TFIID subunits. Requirements for the formation of stable ternary complexes between hTAFII55 and the TATA-binding protein. *J Biol Chem* 271:19774–19780.

Lavigne, A.C., Mengus, G., Gangloff, Y.-G., Wurtz, J.-M., and Davidson, I. 1999. Human TAFII55 interacts with the vitamin D3 and thyroid hormone receptors and with derivatives of the retinoid X receptor that have altered transactivation properties. *Mol Cell Biol* 19:5486–5494.

Lee, D.K., DeJong, J., Hashimoto, S., Horikoshi, M., and Roeder, R.G. 1992. TFIIA induces conformational changes in TFIID via interactions with the basic repeat. *Mol Cell Biol* 12:5189–5196.
Lee, S. and Hahn, S. 1995. Model for binding of transcription factor TFIIB to the TBP-DNA complex. *Nature* 376:609–612.

Lee, Y.C., Park, J.M., Min, S., Han, S.J., and Kim, Y.-J. 1999. An activator binding module of yeast RNA polymerase II holoenzyme. *Mol Cell Biol* 19:2967–2976.

Lee, T.I. and Young, R.A. 2000. Transcription of eukaryotic protein-coding genes. *Annu Rev Genet* 34:77–137.

Lee, T.I., Causton, H.C., Holstege, F.C.P., Shen, W.C., Hannett, N., Jennings, E.G., Winston, F., Green, M.R., and Young, R.A. 2000. Redundant roles for the TFIID and SAGA complexes in global transcription. *Nature* 405:701–704.

Lee, D., Kim, J.W., Kim, K., Joe1, C.O., Schreiber, V., MéniSSier-de Murcia, J., and Choe, J. 2002a. Functional interaction between human papillomavirus type 18E2 and poly(ADP-ribose) polymerase 1. *Oncogene* 21:5877–5885.

Lee, K.B., Wang, D., Lippard, S.J., and Sharp, P.A. 2002b. Transcription-coupled and DNA damage-dependent ubiquitination of RNA polymerase II in vitro. *Proc Natl Acad Sci USA* 99:4239–4244.

Lee, K.B. and Sharp, P.A. 2004. Transcription-dependent polyubiquitination of RNA polymerase II requires lysine 63 of ubiquitin. *Biochemistry* 43:15223–15229.

Lee, D., Ezhkova, E., Li, B., Pattenden, S.G., Tansey, W.P., and Workman, J.L. 2005a. The Proteasome regulatory particle alters the SAGA coactivator to enhance its interactions with transcriptional activators. *Cell* 123:423–436.

Lee, D.-H., Gershenson, N., Gupta, M., Ioshikhes, I.P., Reinberg, D., and Lewis, B.A. 2005b. Functional characterization of core promoter elements: the downstream core element is recognized by TAF1. *Mol Cell Biol* 25:9674–9686.

Lee, K.K., Florens, L., Swanson, S.K., Washburn, M.P., and Workman, J.L. 2005c. The deubiquitylation activity of Ubp8 is dependent upon Sgf11 and its association with the SAGA complex. *Mol Cell Biol* 25:1173–1182.

Lehmann, A.R. 2001. The xeroderma pigmentosum group D (XPD) gene: one gene, two functions, three diseases. *Genes Dev* 15:15–23.

Lei, L., Ren, D., Finkelstein, A., and Burton, Z.F. 1998. Functions of the N- and C-terminal domains of human RAP74 in transcription initiation, elongation, and recycling of RNA polymerase II. *Mol Cell Biol* 18:2130–2142.

Lemaire, M., Xie, J., Meisterernst, M., and Collart, M.A. 2000. The NC2 repressor is dispensable in yeast mutated for the Sin4p component of the holoenzyme and plays roles similar to Mot1p in vivo. *Mol Microbiol* 36:163–173.

Lemon, B. and Tjian, R. 2000. Orchestrated response: a symphony of transcription factors for gene control. *Genes Dev* 14:2551–2569.

Leppard, J.B., and Champoux, J.J. 2005. Human DNA topoisomerase I: relaxation, roles, and damage control. *Chromosoma* 114:75–85.

LeRoy, G., Drapkin, R., Weis, L., and Reinberg, D. 1998. Immunoaffinity purification of the human multisubunit transcription factor IIH. *J Biol Chem* 273:7134–7140.

Leurent, C., Sanders, S., Ruhlmann, C., Mallouh, V., Weil, P.A., Kirschner, D.B., Tora, L., and Schultz, P. 2002. Mapping histone fold TAFs within yeast TFIID. *EMBO J* 21:3424–3433.

Leurent, C., Sanders, S.L., Demény, M.A., Garbett, K.A., Ruhlmann, C., Weil, P.A., Tora, L., and Schultz, P. 2004. Mapping key functional sites within yeast TFIID. *EMBO J* 23:719–727.

Levine, S.S., Weiss, A., Erdjument-Bromage, H., Shao, Z., Tempst, P., and Kingston, R.E. 2002. The core of the polycomb repressive complex is compositionally and functionally conserved in flies and humans. *Mol Cell Biol* 22:6070–6078.

Li, X.-Y., Virbasius, A., Zhu, X., and Green, M.R. 1999. Enhancement of TBP binding by activators and general transcription factors. *Nature* 399:605–609.

Li, X.-Y., Bhaumik, S.R., and Green, M.R. 2000. Distinct classes of yeast promoters revealed by differential TAF recruitment. *Science* 288:1242–1244.

Lieberman, P.M. and Berk, A.J. 1994. A mechanism for TAFs in transcriptional activation: activation domain enhancement of TFIID–

TFIIA-promoter DNA complex formation. *Genes Dev* 9:995–1006.

Lim, C.Y., Santoso, B., Boulay, T., Dong, E., Ohler, U., and Kadonaga, J.T. 2004. The MTE, a new core promoter element for transcription by RNA polymerase II. *Genes Dev* 18:1606–1617.

Lin, Y.C., Choi, W.S., and Gralla, J.D. 2005. TFIIH XPB mutants suggest a unified bacterial-like mechanism for promoter opening but not escape. *Nat Struct Mol Biol* 12:603–607.

Lindahl, T., Satoh, M.S., Poirier, G.G., and Klungland, A. 1995. Post-translational modification of poly(ADP-ribose) polymerase induced by DNA strand breaks. *Trends Biochem Sci* 20:405–411.

Liu, D., Ishima, R., Tong, K.I., Bagby, S., Kokubo, T., Muhandiram, D.R., Kay, L.E., Nakatani, Y., and Ikura, M. 1998. Solution structure of a TBP-TAFII230 complex: protein mimicry of the minor groove surface of the TATA box unwound by TBP. *Cell* 94:573–583.

Liu, Q., Gabriel, S.E., Roinick, K.L., Ward, R.D., and Arndt, K.M. 1999. Analysis of TFIIA function in vivo: Evidence for a role in TATA-binding protein recruitment and gene-specific activation. *Mol Cell Biol* 19:8673–8685.

Liu, J., He, L., Collins, I., Ge, H., Libutti, D., Li, J., Egly, J.-M., and Levens, D. 2000. The FBP interacting repressor targets TFIIH to inhibit activated transcription. *Mol Cell* 5:331–341.

Liu, J., Akoulitchev, S., Weber, A., Ge, H., Chuikov, S., Libutti, D., Wang, X.W., Conaway, J.W., Harris, C.C., Conaway, R.C., Reinberg, D., and Levens, D. 2001a. Defective interplay of activators and repressors with TFIIH in xeroderma pigmentosum. *Cell* 104:353–363.

Liu, Y., Ranish, J.A., Aebersold, R., and Hahn, S. 2001b. Yeast nuclear extract contains two major forms of RNA polymerase II mediator complexes. *JBiolChem* 276:7169–7175.

Lommel, L., Bucheli, M.E., and Sweder, K.S. 2000. Transcription-coupled repair in yeast is independent from ubiquitylation of RNA pol II: implications for Cockayne's syndrome. *Proc Natl Acad Sci USA* 97:9088–9092.

Lorch, Y., Beve, J., Gustafsson, C.M., Myers, L.C., and Kornberg, R.D. 2000. Mediator-nucleosome interaction. *Mol Cell* 6:197–201.

Lu, H., Flores, O., Weinmann, R., and Reinberg, D. 1991. The nonphosphorylated form of RNA polymerase II preferentially associates with the preinitiation complex. *Proc Natl Acad Sci USA* 88:10004–10008.

Lu, H., Zawel, L., Fisher, L., Egly, J.-M., and Reinberg, D. 1992. Human general transcription factor IIH phosphorylates the C-terminal domain of RNA polymerase II. *Nature* 358:641–645.

Lu, H., Fisher, R.P., Bailey, P., and Levine, A.J. 1997. The CDK7-cyclin H-p36 complex of transcription factor IIH phosphorylates p53, enhancing its sequence-specific DNA binding activity in vitro. *Mol Cell Biol* 17:5923–5934.

Luger, K., Mader, A.W., Richmond, R.K., Sargent, D.F., and Richmond, T.J. 1997. Crystal structure of the nucleosome core particle at 2.8 Å resolution. *Nature* 389:251–260.

Luo, Y., Ge, H., Stevens, S., Xiao, H., and Roeder, R.G. 1998. Coactivation by OCA-B: definition of critical regions and synergism with general cofactors. *Mol Cell Biol* 18:3803–3810.

Ma, D., Watanabe, H., Mermelstein, F., Admon, A., Oguri, K., Sun, X., Wada, T., Imai, T., Shiroya, T., Reinberg, D., and Handa, H. 1993. Isolation of a cDNA encoding the largest subunit of TFIIA reveals functions important for activated transcription. *Genes Dev* 7:2246–2257.

Macias, M.J., Wiesner, S., and Sudol, M. 2002. WW and SH3 domains, two different scaffolds to recognize proline-rich ligands. *FEBS Lett* 513:30–37.

Maile, T., Kwoczynski, S., Katzenberger, R.J., Wassarman, D.A., and Sauer, F. 2004. TAF1 activates transcription by phosphorylation of serine 33 in histone H2B. *Science* 304:1010–1014.

Malanga, M., Pleschke, J.M., Kleczkowska, H.E., and Althaus, F.R. 1998. Poly(ADP-ribose) binds to specific domains of p53 and alters its DNA binding functions. *J Biol Chem* 273:11839–11843.

Malanga, M. and Althaus, F.R. 2004. Poly(ADP-ribose) reactivates stalled DNA topoisomerase I and induces DNA strand break resealing. *J Biol Chem* 279:5244–5248.

The General Transcription Machinery and General Cofactors

169Maldonado, E., Ha, I., Cortes, P., Weis, L., and Reinberg, D. 1990. Factors involved in specific transcription by mammalian RNA polymerase II: role of transcription factors IIA, IID, and IIB during formation of a transcription-competent complex. *Mol Cell Biol* 10:6335–6347.

Maldonado, E., Shiekhattar, R., Sheldon, M., Cho, H., Drapkin, R., Rickert, P., Lees, E., Anderson, C.W., Linn, S., and Reinberg, D. 1996. A human RNA polymerase II complex associated with SRB and DNA-repair proteins. *Nature* 381:86–89.

Maldonado, E. 1999. Transcriptional functions of a new mammalian TATA-binding protein-related factor. *J Biol Chem* 274:12963–12966.

Malik, S., Hisatake, K., Sumimoto, H., Horikoshi, M., and Roeder, R.G. 1991. Sequence of general transcription factor TFIIIB and relationships to other initiation factors. *Proc Natl Acad Sci USA* 88:9553–9557.

Malik, S., Lee, D.K., and Roeder, R.G. 1993. Potential RNA polymerase II-induced interactions of transcription factor TFIIIB. *Mol Cell Biol* 13:6253–6259.

Malik, S., Guermah, M., and Roeder, R.G. 1998. A dynamic model for PC4 coactivator function in RNA polymerase II transcription. *Proc Natl Acad Sci USA* 95:2192–2197.

Malik, S., Gu, W., Wu, W., Qin, J., and Roeder, R.G. 2000. The USA-derived transcriptional coactivator PC2 is a submodule of TRAP/SMCC and acts synergistically with other PCs. *Mol Cell* 5:753–760.

Malik, S. and Roeder, R.G. 2005. Dynamic regulation of pol II transcription by the mammalian Mediator complex. *Trends Biochem Sci* 30:256–263.

Martianov, I., Fimia, G.-M., Dierich, A., Parvinen, M., Sassone-Corsi, P., and Davidson, I. 2001. Late arrest of spermiogenesis and germ cell apoptosis in mice lacking the TBP-like TLF/TRF2 gene. *Mol Cell* 7:509–515.

Martin, M.L., Lieberman, P.M., and Curran, T. 1996. Fos-Jun dimerization promotes interaction of the basic region with TFIIE-34 and TFIIF. *Mol Cell Biol* 16:2110–2118.

Martinez-Campa, C., Politis, P., Moreau, J.-L., Kent, N., Goodall, J., Melior, J., and Goding, C.R. 2004. Precise nucleosome positioning and the TATA box dictate requirements for the histone H4 tail and the bromodomain factor Bdf1. *Mol Cell* 15:69–81.

Martinez, E., Chiang, C.-M., and Roeder, R.G. 1994. TATA-binding protein-associated factor(s) in TFIID function through the initiator to direct basal transcription from a TATA-less class II promoter. *EMBO J* 13:3115–3126.

Martinez, E., Kundu, T.K., Fu, J., and Roeder, R.G. 1998. A human SPT3-TAFII 31-GCN 5-acetylase complex distinct from transcription factor IID. *J Biol Chem* 273:23781–23785.

Martinez, E., Palhan, V.B., Tjernberg, A., Lymar, E.S., Gamper, A.M., Kundu, T.K., Chait, B.T., and Roeder, R.G. 2001. Human STAGA complex is a chromatin-acetylating transcription coactivator that interacts with pre-mRNA splicing and DNA damage-binding factors in vivo. *Mol Cell Biol* 21:6782–6795.

Matsui, T., Segall, J., Weil, P.A., and Roeder, RG. 1980. Multiple factors required for accurate initiation of transcription by purified RNA polymerase II. *J Biol Chem* 255:11992–11996.

Matsui, P., DePaulo, J., and Buratowski, S. 1995. An interaction between the Tfb1 and Ssl1 subunits of yeast TFIIH correlates with DNA repair activity. *Nucleic Acids Res* 23:767–772.

Maxon, M.E., Goodrich, J.A., and Tjian, R. 1994. Transcription factor IIE binds preferentially to RNA polymerase IIa and recruits TFIIH: a model for promoter clearance. *Genes Dev* 8:515–524.

McCracken, S. and Greenblatt, J. 1991. Related RNA polymerase-binding regions in human RAP 30/74 and *Escherichia coli σ* 70. *Science* 253:900–902.

McCracken, S., Fong, N., Yankulov, K., Ballantyne, S., Pan, G., Greenblatt, J., Patterson, S.D., Wickens, M., and Bentley, D.L. 1997. The C-terminal domain of RNA polymerase II couples mRNA processing to transcription. *Nature* 385:357–361.

McEwan, I.J. and Gustafsson, J. 1997. Interaction of the human androgen receptor transactivation function with the general transcription factor TFIIF. *Proc Natl Acad Sci USA* 94:8485–8490.

---

McKune, K., Moore, P.A., Hull, M.W., and Woychik, N.A. 1995. Six human RNA polymerase subunits functionally substitute for their yeast counterparts. *Mol Cell Biol* 15:6895–6900.

McMahon, S.B., Van Buskirk, H.A., Dugan, K.A., Copeland, T.D., and Cole, M.D. 1998. The novel ATM-related protein TRRAP is an essential cofactor for the c-Myc and E2F oncoproteins. *Cell* 94:363–74.

McMahon, S.B., Wood, M.A., and Cole, M.D. 2000. The essential cofactor TRRAP recruits the histone acetyltransferase hGCN5 to c-Myc. *Mol Cell Biol* 20:556–562.

McMahon, S.J., Pray-Grant, M.G., Schieltz, D., Yates, J.R., III., and Grant, P.A. 2005. Polyglutamine-expanded spinocerebellar ataxia-7 protein disrupts normal SAGA and SLIK histone acetyltransferase activity. *Proc Natl Acad Sci USA* 102:8478–8482.

Meinhart, A., Blobel, J., and Cramer, P. 2003. An extended winged helix domain in general transcription factor E/TFIIEα. *J Biol Chem* 278:48267–48274.

Meinhart, A., Kamenski, T., Hoeppner, S., Baumli, S., and Cramer, P. 2005. A structural perspective of CTD function. *Genes Dev* 19:1401–1415.

Meisterernst, M. and Roeder, R.G. 1991. Family of proteins that interact with TFIID and regulate promoter activity. *Cell* 67:557–567.

Meisterernst, M., Roy, A.L., Lieu, H.M., and Roeder, R.G. 1991. Activation of class II gene transcription by regulatory factors is potentiated by a novel activity. *Cell* 66:981–993.

Meisterernst, M., Stelzer, G., and Roeder, R.G. 1997. Poly(ADP-ribose) polymerase enhances activator-dependent transcription *in vitro*. *Proc Natl Acad Sci USA* 94:2261–2265.

Mellon, I. and Hanawalt, P.C. 1989. Induction of the *Escherichia coli* lactose operon selectively increases repair of its transcribed DNA strand. *Nature* 342:95–98.

Mencia, M., Moqtaderi, Z., Geisberg, J.V., Kuras, L., and Struhl, K. 2002. Activator-specific recruitment of TFIID and regulation of ribosomal protein genes in yeast. *Mol Cell* 9:823–833.

Mendoza-Alvarez, H., and Alvarez-Gonzalez, R. 2001. Regulation of p53 sequence-specific DNA-binding by covalent poly(ADP-ribosyl)ation. *J Biol Chem* 276:36425–36430.

Merika, M. and Thanos, D. 2001. Enhanceosomes. *Curr Opin Genet Dev* 11:205–208.

Merino, A., Madden, K.R., Lane, W.S., Champoux, J.J., and Reinberg, D. 1993. DNA topoisomerase I is involved in both repression and activation of transcription. *Nature* 365:227–232.

Mermelstein, F., Yeung, K., Cao, J., Inostroza, J.A., Erdjument-Bromage, H., Eagelson, K., Landsman, D., Levitt, P., Tempst, P., and Reinberg, D. 1996. Requirement of a corepressor for Dr1-mediated repression of transcription. *Genes Dev* 10:1033–1048.

Metzger, D., Scheer, E., Soldatov, A., and Tora, L. 1999. Mammalian TAFII 30 is required for cell cycle progression and specific cellular differentiation programmes. *EMBO J* 17:4823–4834.

Meza, J.E., Brzovic, P.S., King, M.C., and Klevit, R.E. 1999. Mapping the functional domains of BRCA1. Interaction of the ring finger domains of BRCA1 and BARD1. *J Biol Chem* 274:5659–5665.

Mialon, A., Sankinen, M., Soderstrom, H., Juntila, T.T., Holmstrom, T., Koivusalo, R., Papageorgiou, A.C., Johnson, R.S., Hietanen, S., Elenius, K., and Westermarck, J. 2005. DNA topoisomerase I is a cofactor for c-Jun in the regulation of epidermal growth factor receptor expression and cancer cell proliferation. *Mol Cell Biol* 25:5040–5051.

Michel, B., Komarnitsky, P., and Buratowski, S. 1998. Histone-like TAFs are essential for transcription *in vivo*. *Mol Cell* 2:663–673.

Min, J.H., Yang, H., Ivan, M., Gertler, F., Kaelin, W.G., Jr., and Pavletich, N.P. 2002. Structure of an HIF-1α-pVHL complex: hydroxyproline recognition in signaling. *Science* 296:1886–1889.

Minakhin, L., Bhagat, S., Brunning, A., Campbell, E.A., Darst, S.A., Ebright, R.H., and Severinov, K. 2001. Bacterial RNA polymerase subunit ω and eukaryotic RNA polymerase subunit RPB6 are sequence, structural, and functional homologs and promote RNA polymerase assembly. *Proc Natl Acad Sci USA* 98:892–897.

---

**M. C. Thomas and C.-M. Chiang**
Mitsiou, D.J. and Stunnenberg, H.G. 2000. TAC, a TBP-sans-TAFs complex containing the unprocessed TFIIAαβ precursor and the TFIIAy subunit. *Mol Cell* 6:527–537.

Mitsiou, D.J. and Stunnenberg, H.G. 2003. p300 is involved in formation of the TBP-TFIIA-containing basal transcription complex, TAC. *EMBO J* 22:4501–4511.

Mitsui, A. and Sharp, P.A. 1999. Ubiquitination of RNA polymerase II large subunit signaled by phosphorylation of carboxyl-terminal domain. *Proc Natl Acad Sci USA* 96:6054–6059.

Mitsuzawa, H. and Ishihama, A. 2004. RNA polymerase II transcription apparatus in Schizosaccharomyces pombe. *Curr Genet* 44:287–294.

Mittal, V. and Hernandez, N. 1997. Role for the amino-terminal region of human TBP in U6 snRNA transcription. *Science* 275:1136–1140.

Mittler, G., Kremmer, E., Timmers, H.T., and Meisterernst, M. 2001. Novel critical role of a human Mediator complex for basal RNA polymerase II transcription. *EMBO Rep* 2:808–813.

Mittler, G., Stuhler, T., Santolin, L., Uhlmann, T., Kremmer, E., Lottspeich, F., Berti, L., and Meisterernst, M. 2003. A novel docking site on Mediator is critical for activation by VP16 in mammalian cells. *EMBO J* 22:6494–6504.

Miyamoto, T., Kakizawa, T. and Hashizume, K. 1999. Inhibition of nuclear receptor signalling by poly(ADP-ribose) polymerase. *Mol Cell Biol* 19:2644–2649.

Mizzen, C.A., Yang, X.-J., Kokubo, T., Brownell, J.E., Bannister, A.J., Owen-Hughes, T., Workman, J., Wang, L., Berger, S.L., Kouzarides, T., Nakatani, Y., and Allis, C.D. 1996. The TAFII250 subunit of TFIID has histone acetyltransferase activity. *Cell* 87:1261–1270.

Moncollin, V., Fischer, L., Cavallini, B., Egly, J.-M., and Chambon, P. 1992. Class II (B) general transcription factor (TFIIB) that binds to the template-committed preinitiation complex is different from general transcription factor BTF3. *Proc Natl Acad Sci USA* 89:397–401.

Moore, P.A., Ozer, J., Salunek, M., Jan, G., Zerby, D., Campbell, S., and Lieberman, P.M. 1999. A human TATA binding protein-related protein with altered DNA binding specificity inhibits transcription from multiple promoters and activators. *Mol Cell Biol* 19:7610–7620.

Moqtaderi, Z., Bai, Y., Poon, D., Weil, P.A., and Struhl, K. 1996. TBP-associated factors are not generally required for transcriptional activation in yeast. *Nature* 383:188–191.

Moqtaderi, Z., Keaveney, M., and Struhl, K. 1998. The histone H3-like TAF is broadly required for transcription in yeast. *Mol Cell* 2:675–682.

Morris, J.R., Keep, N.H., and Solomon, E. 2002. Identification of residues required for the interaction of BARD1 with BRCA1. *J Biol Chem* 277:9382–9386.

Muhich, M.L., Iida, C.T., Horikoshi, M., Roeder, R.G., and Parker, C.S. 1990. cDNA clone encoding *Drosophila* transcription factor TFIID. *Proc Natl Acad Sci USA* 87:9148–9152.

Muldrow, T.A., Campbell, A.M., Weil, P.A., and Auble, D.T. 1999. MOT1 can activate basal transcription in vitro by regulating the distribution of TATA binding protein between promoter and nonpromoter sites. *Mol Cell Biol* 19:2835–2845.

Mulholland, N.M., King, I.F., and Kingston, R.E. 2003. Regulation of Polycomb group complexes by the sequence-specific DNA binding proteins Zeste and GAGA. *Genes Dev* 17:2741–2746.

Myers, L.C. and Kornberg, R.D. 2000. Mediator of transcriptional regulation. *Annu Rev Biochem* 69:729–749.

Na, X., Duan, H.O., Messing, E.M., Schoen, S.R., Ryan, C.K., di Sant’Agnese, P.A., Golemis, E.A., and Wu, G. 2003. Identification of the RNA polymerase II subunit hsRPB7 as a novel target of the von Hippel-Lindau protein. *EMBO J* 22:4249–4259.

Näär, A.M., Beaurang, P.A., Zhou, S., Abraham, S., Solomon, W., and Tjian, R. 1999. Composite co-activator ARC mediates chromatin-directed transcriptional activation. *Nature* 398:828–832.

Näär, A.M., Lemon, B.D., and Tjian, R. 2001. Transcriptional coactivator complexes. *Annu Rev Biochem* 70:475–501.

Nakajima, T., Uchida, C., Anderson, S. F., Parvin, J.D., and Montminy, M. 1997. Analysis of a cAMP-responsive activator reveals a two-component mechanism for transcriptional induction via signal-dependent factors. *Genes Dev* 11:738–747.

Nakajima, N., Horikoshi, M., and Roeder, R.G. 1988. Factors involved in specific transcription by mammalian RNA polymerase II: purification, genetic specificity, and TATA box-promoter interactions of TFIID. *Mol Cell Biol* 8:4028–4040.

Nakamura, K., Jeong, S.-Y., Uchihara, T., Anno, M., Nagashima, K., Nagashima, T., Ikeda, S., Tsuji, S., and Kanazawa, I. 2001. SCA17, a novel autosomal dominant cerebellar ataxia caused by an expanded polyglutamine in TATA-binding protein. *Hum Mol Genet* 10:1441–1448.

Narlikar, G.J., Fan, H.Y., and Kingston, R.E. 2002. Cooperation between complexes that regulate chromatin structure and transcription. *Cell* 108:475–487.

Nekrasov, M., Wild, B., and Müller, J. 2005. Nucleosome binding and histone methyltransferase activity of *Drosophila* PRC2. *EMBO Rep* 6:348–353.

Ng, J., Hart, C.M., Morgan, K., and Simon, J.A. 2000a. A *Drosophila* ESC-E(Z) protein complex is distinct from other polycomb group complexes and contains covalently modified ESC. *Mol Cell Biol* 20:3069–3078.

Ng, W.V., Kennedy, S.P., Mahairas, G.G., Berquist, B., Pan, M., Shukla, H.D., Lasky, S.R., Baliga, N.S., Thorsson, V., Sbrogna, J., Swartzell, S., Weir, D., Hall, J., Dahl, T.A., Welter, R., Goo, Y.A., Leithauser, B., Keller, K., Cruz, R., Danson, M.J., Hough, D.W., Maddocks, D.G., Jablonski, P.E., Krebs, M.P., Angevine, C.M., Dale, H., Isenbarger, T.A., Peck, R.F., Pohl Schroder, M., Spudich, J.L., Jung, K.W., Alam, M., Freitas, T., Hou, S., Daniels, C.J., Dennis, P.P., Omer, A.D., Ebhardt, H., Lowe, T.M., Liang, P., Riley, M., Hood, L., and DasSarma, S. 2000b. Genome sequence of *Halobacterium* species NRC-1. *Proc Natl Acad Sci USA* 97:12176–12181.

Nguyen, V.T., Kiss, T., Michels, A.A., and Bensaude, O. 2001. 7SK small nuclear RNA binds to and inhibits the activity of CDK9/cyclin T complexes. *Nature* 414:322–325.

Nguyen, B.D., Chen, H.T., Kobor, M.S., Greenblatt, J., Legault, P., and Omichinski, J.G. 2003. Solution structure of the carboxyl-terminal domain of RAP74 and NMR characterization of the FCP1-binding sites of RAP74 and human TFIIB. *Biochemistry* 42:1460–1469.

Nikolov, D.B., Hu, S.-H., Lin, J., Gasch, A., Hoffman, A., Horikoshi, M., Chua, N.-H., Roeder, R.G., and Burley, S.K. 1992. Crystal structure of TFIID TATA-box binding protein. *Nature* 360:40–46.

Nikolov, D.B., and Burley S.K. 1994. 2.1 Å resolution refined structure of a TATA box-binding protein (TBP). *Nat Struct Biol* 1:621–637.

Nikolov, D.B., Chen, H., Halay, E.D., Usheva, A.A., Hisatake, K., Lee, D.K., Roeder, R.G., and Burley, S.K. 1995. Crystal structure of a TFIIB-TBP-TATA-element ternary complex. *Nature* 377:119–128.

Nikolov, D.B., Chen, H., Halay, E.D., Hoffman, A., Roeder, R.G., and Burley, S.K. 1996. Crystal structure of a human TATA box-binding protein/TATA element complex. *Proc Natl Acad Sci USA* 93:4862–4867.

Nishikawa, H., Ooka, S., Sato, K., Arima, K., Okamoto, J., Klevit, R.E., Fukuda, M., and Ohta, T. 2004. Mass spectrometric and mutational analyses reveal Lys-6-linked polyubiquitin chains catalyzed by BRCA1-BARD1 ubiquitin ligase. *J Biol Chem* 279:3916–3924.

Nonet, M.L. and Young, R.A. 1989. Intragenic and extragenic suppressors of mutations in the heptapeptide repeat domain of *Saccharomyces cerevisiae* RNA polymerase II. *Genetics* 123:715–724.

Oei, S.L., Griesenbeck, J., Schweiger, M., and Ziegler, M. 1998. Regulation of RNA polymerase II-dependent transcription by poly(ADP-ribosyl)ation of transcription factors. *J Biol Chem* 273:31644–31647.

Oelgeschläger, T., Chiang, C.-M., and Roeder, R.G. 1996. Topology and reorganization of a human TFIID-promoter complex. *Nature* 382:735–738.

Oelgeschläger, T., Tao, Y., Kang, Y.K., and Roeder, R.G. 1998. Transcription activation via enhanced preinitiation complex assembly in a human cell-free system lacking TAFIIs. *Mol Cell* 1:925–931.

O'Gorman, W., Thomas, B., Kwek, K.Y., Furger, A., and Akoulitchev, A. 2005. Analysis of U1 small nuclear RNA interaction with cyclin H. J Biol Chem 280:36920–36925.

Ogryzko, V.V., Kotani, T., Zhang, X., Schiltz, R.L., Howard, T., Yang, X.J., Howard, B.H., Qin, J., and Nakatani, Y. 1998. Histone-like TAFs within the PCAF histone acetylase complex. Cell 94:35–44.

Ohbayashi, T., Shimada, M., Nakadai, T., Wada, T., Handa, H., and Tamura, T. 2003. Vertebrate TBP-like protein (TLP/TRF2/TLF) stimulates TATA-less terminal deoxynucleotidyl transferase promoters in a transient reporter assay, and TFIIA-binding capacity of TLP is required for this function. Nucleic Acids Res 31:2127–2133.

Ohh, M., Park, C.W., Ivan, M., Hoffman, M.A., Kim, T.Y., Huang, L.E., Pavletich, N., Chau, V., and Kaelin, W.G. 2000. Ubiquitination of hypoxia-inducible factor requires direct binding to the β-domain of the von Hippel-Lindau protein. Nat Cell Biol 2:423–427.

Ohkuma, Y., Sumimoto, H., Hoffmann, A., Shimasaki, S., Horikoshi, M., and Roeder, R.G. 1991. Structural motifs and potential σ homologies in the large subunit of human general transcription factor TFIIIE. Nature 354:398–401.

Ohkuma, Y. and Roeder, R.G. 1994. Regulation of TFIIH ATPase and kinase activities by TFIIIE during active initiation complex formation. Nature 368:160–163.

Ohkuma, Y., Hashimoto, S., Wang, C.K., Horikoshi, M., and Roeder, R.G. 1995. Analysis of the role of TFIIIE in basal transcription and TFIIH-mediated carboxy-terminal domain phosphorylation through structure-function studies of TFIIIE-α. Mol Cell Biol 15:4856–4866.

Ohler, U., Liao, G.-C., Niemann, H., and Rubin, G.M. 2002. Computational analysis of core promoters in the Drosophila genome. Genome Biology 3:research0087.1–0087.12.

Ohta, T., Michel, J.J., Schottelius, A.J., and Xiong, Y. 1999. ROC1, a homolog of APC11, represents a family of cullin partners with an associated ubiquitin ligase activity. Mol Cell 3:535–541.

Okamoto, T., Yamamoto, S., Watanabe, Y., Ohta, T., Hanaoka, F., Roeder, R.G., and Ohkuma, Y. 1998. Analysis of the role of TFIIIE in transcriptional regulation through structure-function studies of the TFIIEB subunit. J Biol Chem 273:19866–19876.

Okuda, M., Watanabe, Y., Okamura, H., Hanaoka, F., Ohkuma, Y., and Nishimura, Y. 2000. Structure of the central core domain of TFIIEB with a novel double-stranded DNA-binding surface. EMBO J 19:1346–1356.

Okuda, M., Tanaka, A., Arai, Y., Satoh, M., Okamura, H., Nagadoi, A., Hanaoka, F., Ohkuma, Y., and Nishimura, Y. 2004. A novel zinc finger structure in the large subunit of human general transcription factor TFIIIE. J Biol Chem 279:51395–51403.

Onodera, Y., Haag, J.R., Ream, T., Nunes, P.C., Pontes, O., and Pikaard, C.S. 2005. Plant nuclear RNA polymerase IV mediates siRNA and DNA methylation-dependent heterochromatin formation. Cell 120:613–622.

Orphanides, G., Lagrange, T., and Reinberg, D. 1996. The general transcription factors of RNA polymerase II. Genes Dev 10:2657–2683.

Ossipow, V., Tassan, J.-P., Nigg, E.A., and Schibler, U. 1995. A mammalian RNA polymerase II holoenzyme containing all components required for promoter-specific transcription initiation. Cell 83:137–146.

Ozer, J., Moore, P.A., Bolden, A.H., Lee, A., Rosen, C.A., and Lieberman, P.M. 1994. Molecular cloning of the small γ subunit of human TFIIA reveals functions critical for activated transcription. Genes Dev 8:2324–2335.

Ozer, J., Lezina, L.E., Ewing, J., Audi, S., and Lieberman, P.M. 1998a. Association of transcription factor IIA with TBP is required for transcriptional activation of a subset of promoters and cell cycle progression in S. cerevisiae. Mol Cell Biol 18:2559–2570.

Ozer, J., Mitsouras, K., Zerby, D., Carey, M., and Lieberman, P.M. 1998b. Transcription factor IIA derepresses TATA binding protein (TBP)-associated factor inhibition of TBP-DNA binding. J Biol Chem 273:14293–14300.

Ozer, J., Moore, P.A., and Lieberman, P.M. 2000. A testis-specific transcription factor IIA (TFIIAt) stimulates TATA-binding protein

DNA binding and transcription activation. J Biol Chem 275:122–128.

Pal, M., Ponticelli, A.S., and Luse, D.S. 2005. The role of the transcription bubble and TFIIIB in promoter clearance by RNA polymerase II. Mol Cell 19:101–110.

Palancade, B., Dubois, M.F., Bensaude, O. 2002. FCP1 phosphorylation by casein kinase 2 enhances binding to TFIIIF and RNA polymerase II carboxyl-terminal domain phosphatase activity. J Biol Chem 277:36061–36067.

Palhan, V.B., Chen, S., Peng, G.H., Tjernberg, A., Gamper, A.M., Fan, Y., Chait, B.T., La Spada, A.R., and Roeder, R.G. 2005. Polyglutamine-expanded ataxin-7 inhibits STAGA histone acetyltransferase activity to produce retinal degeneration. Proc Natl Acad Sci USA 102:8472–8477.

Pan, G. and Greenblatt, J. 1994. Initiation of transcription by RNA polymerase II is limited by melting of the promoter DNA in the region immediately upstream of the initiation site. J Biol Chem 269:30101–30104.

Pan, Z.-Q., Ge, H., Amin, A.A., and Hurwitz, J. 1996. Transcription-positive cofactor 4 forms complexes with HSSB (RPA) on single-stranded DNA and influences HSSB-dependent enzymatic synthesis of simian virus 40 DNA. J Biol Chem 271:22111–22116.

Pardee, T.S., Bangur, C.S., and Ponticelli, A.S. 1998. The N-terminal region of yeast TFIIIB contains two adjacent functional domains involved in stable RNA polymerase II binding and transcription start site selection. J Biol Chem 273:17859–17864.

Park, J.M., Kim, H.S., Han, S.J., Hwang, M.S., Lee, Y.C., and Kim, Y.-J. 2000. In vivo requirement of activator-specific binding targets of mediator. Mol Cell Biol 20:8709–8719.

Parker, C.S. and Topol, J. 1984. A Drosophila RNA polymerase II transcription factor contains a promoter-region-specific DNA-binding activity. Cell 36:357–369.

Parvin, J.D. and Sharp, P.A. 1993. DNA topology and a minimal set of basal factors for transcription by RNA polymerase II. Cell 73:533–540.

Parvin, J.D., Shykind, B.M., Meyers, R.E., Kim, J., and Sharp, P.A. 1994. Multiple sets of basal factors initiate transcription by RNA polymerase II. J Biol Chem 269:18414–18421.

Parvin, J.D. and Young, R.A. 1998. Regulatory targets in the RNA polymerase II holoenzyme. Curr Opin Genet Dev 8:565–570.

Pavri, R., Lewis, B., Kim, T.K., Dilworth, F.J., Erdjument-Bromage, H., Tempst, P., de Murcia, G., Evans, R., Chambon, P., and Reinberg, D. 2005. PARP-1 determines specificity in a retinoid signaling pathway via direct modulation of Mediator. Mol Cell 18:83–96.

Pei, Y., Hausmann, S., Ho, C.K., Schwer, B., and Shuman, S. 2001. The length, phosphorylation state, and primary structure of the RNA polymerase II carboxy-terminal domain dictate interactions with mRNA capping enzymes. J Biol Chem 276:28075–28082.

Pereira, L.A., van der Knaap, J.A., van den Boom, V., van den Heuvel, F.A.J., and Timmers, H.T.M. 2001. TAFII 170 interacts with the concave surface of TATA-binding protein to inhibit its DNA binding activity. Mol Cell Biol 21:7523–7534.

Pereira, L.A., Klejman, M.P., and Timmers, H.T.M. 2003. Roles for B-TAF1 and Mot1p in dynamics of TATA-binding protein and regulation of RNA polymerase II transcription. Gene 315:1–13.

Perez-Howard, G.M., Weil, P.A., and Beechem, J.M. 1995. Yeast TATA binding protein interaction with DNA: fluorescence determination of oligomeric state, equilibrium binding, on-rate, and dissociation kinetics. Biochemistry 34:8005–8017.

Persengiev, S.P., Zhu, X., Dixit, B.L., Maston, G.A., Kittler, E.L., and Green, M.R. 2003. TRF3, a TATA-box-binding protein-related factor, is vertebrate-specific and widely expressed. Proc Natl Acad Sci USA 100:14887–14891.

Peterson, M.G., Tanese, N., Pugh, B.F., and Tjian, R. 1990. Functional domains and upstream activation properties of cloned human TATA binding protein. Science 248:1625–1630.

Peterson, M.G., Inostroza, J., Maxon, M.E., Flores, O., Admon, A., Reinberg, D., and Tjian, R. 1991. Structure and functional properties of human general transcription factor IIE. Nature 354:369–373.

Petroski, M.D. and Deshaies, R.J. 2005. Function and regulation of cullin-RING ubiquitin ligases. *Nat Rev Mol Cell Biol* 6:9–20.

Pham, A.-D. and Sauer, F. 2000. Ubiquitin-activating/conjugating activity of TAFII250, a mediator of activation of gene expression in *Drosophila*. *Science* 289:2357–2360.

Pickart, C.M. 2004. Back to the future with ubiquitin. *Cell* 116:181–190.

Pinto, I., Ware, D.E., and Hampsey, M. 1992. The yeast SUA7 gene encodes a homolog of human transcription factor TFIIB and is required for normal start site selection *in vivo*. *Cell* 68:977–988.

Pinto, I., Wu, W.-H., Na, J.G., and Hampsey, M. 1994. Characterization of sua7 mutations defines a domain of TFIIB involved in transcription start site selection in yeast. *J Biol Chem* 269:30569–30573.

Pointud, J.-C., Mengus, G., Brancorsini, S., Monaco, L., Parvinen, M., Sassone-Corsi, P., and Davidson, I. 2003. The intracellular localization of TAF7L, a paralogue of transcription factor TFIID subunit TAF7, is developmentally regulated during male germ-cell differentiation. *J Cell Science* 116:1847–1858.

Poon, D. and Weil, P.A. 1993. Immunopurification of yeast TATA-binding protein and associated factors. Presence of transcription factor IIIB transcriptional activity. *J Biol Chem* 268:15325–15328.

Poon, D., Campbell, A.M., Bai, Y., and Weil, P.A. 1994. Yeast Taf170 is encoded by MOT1 and exists in a TATA box-binding protein (TBP)-associated factor complex distinct from transcription factor IID. *J Biol Chem* 269:23135–23140.

Powell, D.W., Weaver, C.M., Jennings, J.L., McAfee, K.J., He, Y., Weil, P.A., and Link, A.J. 2004. Cluster analysis of mass spectrometry data reveals a novel component of SAGA. *Mol Cell Biol* 24:7249–7259.

Pray-Grant, M.G., Schieltz, D., McMahon, S.J., Wood, J.M., Kennedy, E.L., Cook, R.G., Workman, J.L., Yates, J.R., III, and Grant, P.A. 2002. The novel SLIK histone acetyltransferase complex functions in the yeast retrograde response pathway. *Mol Cell Biol* 22:8774–8786.

Pray-Grant, M.G., Daniel, J.A., Schieltz, D., Yates, J.R., III, and Grant, P.A. 2005. Chd1 chromodomain links histone H3 methylation with SAGA- and SLIK-dependent acetylation. *Nature* 433:434–438.

Prelich, G. 2002. RNA polymerase II carboxy-terminal domain kinases: emerging clues to their function. *Eukaryot Cell* 1:153–162.

Price, D.H., Sluder, A.E., and Greenleaf, A.L. 1987. Fractionation of transcription factors for RNA polymerase II from *Drosophila* Kc cell nuclear extracts. *J Biol Chem* 262:3244–3255.

Price, D.H., Sluder, A.E., and Greenleaf, A.L. 1989. Dynamic interaction between a *Drosophila* transcription factor and RNA polymerase II. *Mol Cell Biol* 9:1465–1475.

Prusiner, S.B. 1982. Novel proteinaceous infectious particles cause scrapie. *Science* 216:136–144.

Pugh, B.F. and Tjian, R. 1990. Mechanism of transcriptional activation by Sp1: evidence for coactivators. *Cell* 61:1187–1197.

Pugh, B.F. and Tjian, R. 1991. Transcription from a TATA-less promoter requires a multisubunit TFIID complex. *Genes Dev* 5:1935–1945.

Purnell, B.A., Emanuel, P.A., and Gilmour, D.S. 1994. TFIID sequence recognition of the initiator and sequences farther downstream in *Drosophila* class II genes. *Genes Dev* 8:830–842.

Rabenstein, M.D., Zhou, S., Lis, J.T., and Tjian, R. 1999. TATA box-binding protein (TBP)-related factor 2 (TRF2), a third member of the TBP family. *Proc Natl Acad Sci USA* 96:4791–4796.

Rachez, C., Suldan, Z., Ward, J., Chang, C.P., Burakov, D., Erdjument-Bromage, H., Tempst, P., and Freedman, L.P. 1998. A novel protein complex that interacts with the vitamin D3 receptor in a ligand-dependent manner and enhances VDR transactivation in a cell-free system. *Genes Dev* 12:1787–1800.

Rani, P.G., Ranish, J.A., and Hahn, S. 2004. RNA polymerase II (Pol II) -TFIIF and Pol II-mediator complexes: the major stable Pol II complexes and their activity in transcription initiation and reinitiation. *Mol Cell Biol* 24:1709–1720.

Ranish, J.A. and Hahn, S. 1991. The yeast general transcription factor TFIIA is composed of two polypeptide subunits. *J Biol Chem* 266:19320–19327.

Ranish, J.A., Lane, W.S., and Hahn, S. 1992. Isolation of two genes that encode subunits of the yeast transcription factor II A. *Science* 255:1127–1229.

Ranish, J.A., Yudkovsky, N., and Hahn, S. 1999. Intermediates in formation and activity of the RNA polymerase II preinitiation complex: holoenzyme recruitment and a postrecruitment role for the TATA box and TFIIB. *Genes Dev* 13:49–63.

Ranish, J.A., Hahn, S., Lu, Y., Yi, E.C., Li, X.-J., Eng, J., and Aebersold, R. 2004. Identification of TFB5, a new component of general transcription and DNA repair factor IIH. *Nat Genet* 36:707–713.

Ratner, J.N., Balasubramanian, B., Corden, J., Warren, S.L., and Bregman, D.B. 1998. Ultraviolet radiation-induced ubiquitination and proteasomal degradation of the large subunit of RNA polymerase II. Implications for transcription-coupled DNA repair. *J Biol Chem* 273:5184–5189.

Rawling, J.M. and Alvarez-Gonzalez, R. 1997. TFIIF, a basal eukaryotic transcription factor, is a substrate for poly(ADP-ribosyl)ation. *Biochem J* 324:249–253.

Reese, J.C., Apone, L., Walker, S.S., Griffin, L.A., and Green, M.R. 1994. Yeast TAFIIs in a multisubunit complex required for activated transcription. *Nature* 371:523–527.

Reeves, W.M. and Hahn, S. 2005. Targets of the Gal4 transcription activator in functional transcription complexes. *Mol Cell Biol* 25:9092–9102.

Reid, J., Murray, I., Watt, K., Betney, R., and McEwan, I.J. 2002. The androgen receptor interacts with multiple regions of the large subunit of general transcription factor TFIIF. *J Biol Chem* 277:41247–41253.

Reid, J. and Svejstrup, J.Q. 2004. DNA damage-induced Def1-RNA polymerase II interaction and Def1 requirement for polymerase ubiquitylation *in vitro*. *J Biol Chem* 279:29875–29878.

Reinberg, D. and Roeder, R.G. 1987. Factors involved in specific transcription by mammalian RNA polymerase II. Purification and functional analysis of initiation factors IIB and IIE. *J Biol Chem* 262:3310–3321.

Reinberg, D., Horikoshi, M., and Roeder, R.G. 1987. Factors involved in specific transcription in mammalian RNA polymerase II. Functional analysis of initiation factors IIA and IID and identification of a new factor operating at sequences downstream of the initiation site. *J Biol Chem* 262:3322–3330.

Ren, B. and Maniatis, T. 1998. Regulation of *Drosophila* Adh promoter switching by an initiator-targeted repression mechanism. *EMBO J* 17:1076–1086.

Robert, F., Douziech, M., Forget, D., Egly, J.-M., Greenblatt, J., Burton, Z.F., and Coulombe, B. 1998. Wrapping of promoter DNA around the RNA polymerase II initiation complex induced by TFIIF. *Mol Cell* 2:341–351.

Robinson, M.M., Yatherajam, G., Ranallo, R.T., Bric, A., Paule, M.R., and Stargell, L.A. 2005. Mapping and functional characterization of the TAF11 interaction with TFIIA. *Mol Cell Biol* 25:945–957.

Rochette-Egly, C., Adam, S., Rossignol, M., Egly, J.-M., and Chambon, P. 1997. Stimulation of RARα activation function AF-1 through binding to the general transcription factor TFIIH and phosphorylation by CDK7. *Cell* 90:97–107.

Rodriguez, C.R., Cho, E.-J., Keogh, M.-C., Moore, C.L., Greenleaf, A.L., and Buratowski, S. 2000. Kin28, the TFIIH-associated carboxy-terminal domain kinase, facilitates the recruitment of mRNA processing machinery to RNA polymerase II. *Mol Cell Biol* 20:104–112.

Roeder, R.G. and Rutter, W.J. 1969. Multiple forms of DNA-dependent RNA polymerase in eukaryotic organisms. *Nature* 224:234–237.

Roeder, R.G. and Rutter, W.J. 1970. Specific nuclear and nucleoplasmic RNA polymerases. *Proc Natl Acad Sci USA* 65:675–682.

Roeder, R.G. 2005. Transcriptional regulation and the role of diverse coactivators in animal cells. *FEBS Lett* 579:909–915.

Rossignol, M., Kolb-Cheynel, I., and Egly, J.-M. 1997. Substrate specificity of the cdk-activating kinase (CAK) is altered upon association with TFIIH. *EMBO J* 16:1628–1637.

Rossignol, M., Keriel, A., Staub, A., and Egly, J.-M. 1999. Kinase activity and phosphorylation of the largest subunit of TFIIF transcription factor. *J Biol Chem* 274:22387–22392.

Roy, R., Schaeffer, L., Humbert, S., Vermeulen, W., Weeda, G., and Egly, J.-M. 1994. The DNA-dependent ATPase activity associated with the class II basic transcription factor BTF2/TFIIF. *J Biol Chem* 269: 9826–9832.

Ruan, J.P., Arhin, G.K., Ullu, E., and Tschudi, C. 2004. Functional characterization of a Trypanosoma brucei TATA-binding protein-related factor points to a universal regulator of transcription in trypanosomes. *Mol Cell Biol* 24: 9610–9618.

Ruffner, H., Joazeiro, C.A., Hemmati, D., Hunter, T., and Verma, I.M. 2001. Cancer-predisposing mutations within the RING domain of BRCA1: loss of ubiquitin protein ligase activity and protection from radiation hypersensitivity. *Proc Natl Acad Sci USA* 98: 5134–5139.

Ruppert, S., Wang, E.H., and Tjian, R. 1993. Cloning and expression of human TAFII 250: a TBP-associated factor implicated in cell-cycle regulation. *Nature* 362: 175–179.

Ruppert, S. and Tjian, R. 1995. Human TAFII 250 interacts with RAP74: implications for RNA polymerase II initiation. *Genes Dev* 9: 2747–2755.

Ryu, S., Zhou, S., Ladurner, A.G., and Tjian, R. 1999. The transcriptional cofactor complex CRSP is required for activity of the enhancer-binding protein Sp1. *Nature* 397: 446–450.

Samuels, M., Fire, A., and Sharp, P.A. 1982. Separation and characterization of factors mediating accurate transcription by RNA polymerase II. *J Biol Chem* 257: 14419–14427.

Samuels, M. and Sharp, P.A. 1986. Purification and characterization of a specific RNA polymerase II transcription factor. *J Biol Chem* 261: 2003–2013.

Samuelsen, C.O., Baraznenok, V., Khorosjutina, O., Spahr, H., Kieselbach, T., Holmberg, S., and Gustafsson, C.M. 2003. TRAP230/ARC240 and TRAP240/ARC250 Mediator subunits are functionally conserved through evolution. *Proc Natl Acad Sci USA* 100: 6422–6427.

Sanders, S.L., Garbett, K.A., and Weil, P.A. 2002a. Molecular characterization of Saccharomyces cerevisiae TFIID. *Mol Cell Biol* 22: 6000–6013.

Sanders, S.L., Jennings, J., Canutescu, A., Link, A.J., and Weil, P.A. 2002b. Proteomics of the eukaryotic transcription machinery: identification of proteins associated with components of yeast TFIID by multidimensional mass spectrometry. *Mol Cell Biol* 22: 4723–4738.

Sato, K., Hayami, R., Wu, W., Nishikawa, T., Nishikawa, H., Okuda, Y., Ogata, H., Fukuda, M., and Ohta, T. 2004. Nucleophosmin/B23 is a candidate substrate for the BRCA1-BARD1 ubiquitin ligase. *J Biol Chem* 279: 30919–30922.

Sauer, F., Fondell, J.D., Ohkuma, Y., Roeder, R.G., and Jackle, H. 1995. Control of transcription by Krüppel through interactions with TFIIB and TFIIIEβ. *Nature* 375: 162–164.

Saurin, A.J., Borden, K.L., Boddy, M.N., and Freemont, P.S. 1996. Does this have a familiar RING? *Trends Biochem Sci* 21: 208–214.

Saurin, A.J., Shao, Z., Erdjument-Bromage, H., Tempst, P., and Kingston, R.E. 2001. A Drosophila polycomb group complex includes Zeste and dTAFII proteins. *Nature* 412: 655–660.

Sawadogo, M. and Roeder, R.G. 1985a. Factors involved in specific transcription by human RNA polymerase II: analysis by a rapid and quantitative in vitro assay. *Proc Natl Acad Sci USA* 82: 4394–4398.

Sawadogo, M. and Roeder, R.G. 1985b. Interaction of a gene-specific transcription factor with the adenovirus major late promoter upstream of the TATA box region. *Cell* 43: 165–175.

Sayre, M.H., Tschochner, H., and Kornberg, R.D. 1992. Reconstitution of transcription with five purified initiation factors and RNA polymerase II from Saccharomyces cerevisiae. *J Biol Chem* 267: 23376–23382.

Schaeffer, L., Roy, R., Humbert, S., Moncollin, V., Vermeulen, W., Hoeijmakers, J.H.J., Chambon, P., and Egly, J.-M. 1993. DNA repair helicase: a component of BTF2 (TFIIF) basic transcription factor. *Science* 260: 58–63.

Schaeffer, L., Moncollin, V., Roy, R., Staub, A., Mezzina, M., Sarasin, A., Weeda, G., Hoeijmakers, J.H.J., and Egly, J.-M. 1994. The ERCC2/DNA repair protein is associated with the class II BTF2/TFIIF transcription factor. *EMBO J* 13: 2388–2392.


Schlegel, B.P., Green, V.J., Ladias, J.A., and Parvin, J.D. 2000. BRCA1 interaction with RNA polymerase II reveals a role for hRPB2 and hRPB10α in activated transcription. *Proc Natl Acad Sci USA* 97: 3148–3153.

Schroeder, S.C., Schwer, B., Shuman, S., and Bentley, D. 2000. Dynamic association of capping enzymes with transcribing RNA polymerase II. *Genes Dev* 14: 4435–2440.

Schimanski, B., Nguyen, T.N., and Günzl, A. 2005. Characterization of a multisubunit transcription factor complex essential for spliced-leader RNA gene transcription in Trypanosoma brucei. *Mol Cell Biol* 25: 7303–7313.

Schmidt, M.C., Kao, C.C., Pei, R., and Berk, A.J. 1989. Yeast TATA-box transcription factor gene. *Proc Natl Acad Sci USA* 86: 7785–7789.

Segil, N., Guermah, M., Hoffmann, A., Roeder, R.G., and Heintz, N. 1996. Mitotic regulation of TFIID: inhibition of activator-dependent transcription and changes in subcellular localization. *Genes Dev* 10: 2389–2400.

Selleck, W., Howley, R., Fang, Q., Podolny, V., Fried, M.G., Buratowski, S., and Tan, S. 2001. A histone fold TAF octamer within the yeast TFIID transcriptional coactivator. *Nat Struct Biol* 8: 695–700.

Seol, J.H., Feldman, R.M., Zachariae, W., Shevchenko, A., Correll, C.C., Lyapina, S., Chi, Y., Galova, M., Claypool, J., Sandmeyer, S., Nasmyth, K., Deshaies, R.J., and Shevchenko, A., and Deshaies, R.J. 1999. Cdc53/cullin and the essential Hrt1 RING-H2 subunit of SCF define a ubiquitin ligase module that activates the E2 enzyme Cdc34. *Genes Dev* 13: 1614–1626.

Serizawa, H., Conaway, R.C., and Conaway, J.W. 1992. A carboxyl-terminal-domain kinase associated with RNA polymerase II transcription factor δ from rat liver. *Proc Natl Acad Sci USA* 89: 7476–7480.

Serizawa, H., Conaway, J.W., and Conaway, R.C. 1993. Phosphorylation of C-terminal domain of RNA polymerase II is not required in basal transcription. *Nature* 363: 371–374.

Serizawa, H., Conaway, J.W., and Conaway, R.C. 1994. An oligomeric form of the large subunit of transcription factor (TF) IIE activates phosphorylation of the RNA polymerase II carboxyl-terminal domain by TFIIF. *J Biol Chem* 269: 20750–20756.

Sewack, G.F., Ellis, T.W., and Hansen, U. 2001. Binding of TATA binding protein to a naturally positioned nucleosome is facilitated by histone acetylation. *Mol Cell Biol* 21: 1404–1415.

Shao, Z., Raible, F., Mollaaghababa, R., Guyon, J.R., Wu, C.T., Bender, W., and Kingston, R.E. 1999. Stabilization of chromatin structure by PRC1, a Polycomb complex. *Cell* 98: 37–46.

Shao, H., Revach, M., Moshonov, S., Tzuman, Y., Gazit, K., Albeck, S., Unger, T., and Dikstein, R. 2005. Core promoter binding by histone-like TAF complexes. *Mol Cell Biol* 25: 206–219.

Shilatifard, A., Conaway, R.C., and Conaway, J.W. 2003. The RNA polymerase II elongation complex. *Annu Rev Biochem* 72: 693–715.

Shim, E.Y., Walker, A.K., Shi, Y., and Blackwell, T.K. 2002. CDK-9/cyclin T (P-TEFb) is required in two postinitiation pathways for transcription in the C. elegans embryo. *Genes Dev* 16: 2135–2146.

Shykind, B.M., Kim, J., and Sharp, P.A. 1995. Activation of the TFIID-TFIIA complex with HMG-2. *Genes Dev* 9: 1354–1365.

Shykind, B.M., Kim, J., Stewart, L., Champoux, J.J., and Sharp, P.A. 1997. Topoisomerase I enhances TFIID-TFIIA complex assembly during activation of transcription. *Genes Dev* 11: 397–407.

Skowyra, D., Koepp, D.M., Kamura, T., Conrad, M.N., Conaway, R.C., Conaway, J.W., Elledge, S.J., and Harper, J.W. 1999. Reconstitution of G1 cyclin ubiquitination with complexes containing SCFGrr1 and Rbx1. *Science* 284: 662–665.

Simbulan-Rosenthal, C.M., Rosenthal, D.S., Luo, R., Samara, R., Espinoza, L.A., Hassa, P.O., Hottiger, M.O., and Smulson, M.E. 2003. PARP-1 binds E2F-1 independently of its DNA binding and catalytic domains, and acts as a novel coactivator of E2F-1-mediated transcription during re-entry of quiescent cells into S phase. *Oncogene* 22: 8460–8471.

Slattery, E., Dignam, J.D., Matsui, T., and Roeder, R.G. 1983. Purification and analysis of a factor which suppresses nick-induced

M. C. Thomas and C.-M. Chiang

transcription by RNA polymerase II and its identity with Poly(ADP-ribosyl) polymerase. J Biol Chem 258:5955–5959.

Smale, S.T. 1997. Transcription initiation from TATA-less promoters within eukaryotic protein-coding genes. Biochim Biophys Acta 1351:73–88.

Smale, S.T. and Kadonaga, J.T. 2003. The RNA polymerase II core promoter. Annu Rev Biochem 72:449–479.

Solow, S., Salunek, M., Ryan, R., and Lieberman, P.M. 2001. TAFII250 phosphorylates human transcription factor IIA on serine residues important for TBP binding and transcription activity. J Biol Chem 276:15886–15892.

Somesh, B.P., Reid, J., Liu, W.F., Sogaard, T.M., Erdjument-Bromage, H., Tempst, P., and Svejstrup, J.Q. 2005. Multiple mechanisms confining RNA polymerase II ubiquitylation to polymerases undergoing transcriptional arrest. Cell 121:913–923.

Sopta, M., Carthew, R.W., and Greenblatt, J. 1985. Isolation of three proteins that bind to mammalian RNA polymerase II. J Biol Chem 260:10353–10360.

Sopta, M., Burton, Z.F., and Greenblatt, J. 1989. Structure and associated DNA-helicase activity of a general transcription initiation factor that binds to RNA polymerase II. Nature 341:410–414.

Spence, J., Sadis, S., Haas, A.L., and Finley, D. 1995. A ubiquitin mutant with specific defects in DNA repair and multiubiquitination. Mol Cell Biol 15:1265–1273.

Spence, J., Gali, R.R., Dittmar, G., Sherman, F., Karin, M., and Finley, D. 2000. Cell cycle-regulated modification of the ribosome by a variant multiubiquitin chain. Cell 102:67–76.

Stargell, L.A., Moqtaderi, Z., Dorris, D.R., Ogg, R.C., and Struhl, K. 2000. TFIIA has activator-dependent and core promoter functions in vivo. J Biol Chem 275:12374–12380.

Starita, L.M., Horwitz, A.A., Keogh, M.C., Ishioka, C., Parvin, J.D., and Chiba, N. 2005. BRCA1/BARD1 ubiquitinate phosphorylated RNA polymerase II. J Biol Chem 280:24498–24505.

Stebbins, C.E., Kaelin, W.G., Jr., and Pavletich, N.P. 1999. Structure of the VHL-ElonginC-ElonginB complex: implications for VHL tumor suppressor function. Science 284:455–461.

Sterner, D.E., Grant, P.A., Roberts, S.M., Duggan, L.J., Belotserkovskaya, R., Pacella, L.A., Winston, F., Workman, J.L., and Berger, S.L. 1999. Functional organization of the yeast SAGA complex: distinct components involved in structural integrity, nucleosome acetylation, and TATA-binding protein interaction. Mol Cell Biol 19:86–98.

Sterner, D.E., Belotserkovskaya, R., and Berger, S.L. 2002. SALSA, a variant of yeast SAGA, contains truncated Spt7, which correlates with activated transcription. Proc Natl Acad Sci USA 99:11622–11627.

Stevens, A. 1960. Incorporation of the adenine ribonucleotide into RNA by cell fractions from E. coli B. Biochem Biophys Res Commun 3:92–96.

Stewart, L., Ireton, G.C., and Champoux, J.J. 1996. The domain organization of human topoisomerase I. J Biol Chem 271:7602–7608.

Sumimoto, H., Ohkuma, Y., Sinn, E., Kato, H., Shimasaki, S., Horikoshi, M., and Roeder, R.G. 1991. Conserved sequence motifs in the small subunit of human general transcription factor TFIIE. Nature 354:401–404.

Sun, X., Ma, D., Sheldon, M., Yeung, K., and Reinberg, D. 1994. Reconstitution of human TFIIA activity from recombinant polypeptides: a role in TFIID-mediated transcription. Genes Dev 8:2336–2348.

Sun, X., Zhang, Y., Cho, H., Rickert, P., Lees, E., Lane, W., and Reinberg, D. 1998. NAT, a human complex containing Srb polypeptides that functions as a negative regulator of activated transcription. Mol Cell 2:213–222.

Sutrias-Grau, M., Bianchi, M.E., and Bernues, J. 1999. High mobility group protein 1 interacts specifically with the core domain of human TATA box-binding protein and interferes with transcription factor IIB within the preinitiation complex. J Biol Chem 274:1628–1634.

Suzuki-Yagawa, Y., Guermah, M., and Roeder, R.G. 1997. The ts13 mutation in the TAFII250 subunit (CCG1) of TFIID directly affects transcription of D-type cyclin genes in cells arrested in G1 at the nonpermissive temperature. Mol Cell Biol 17:3284–3294.

---

Suzuki, Y., Tsunoda, T., Sese, J., Taira, H., Mizushima-Sugano, J., Hata, H., Ota, T., Isogai, T., Tanaka, T., Nakamura, Y., Suyama, A., Sakaki, Y., Morishita, S., Okubo, K., and Sugano, S. 2001. Identification and characterization of the potential promoter regions of 1031 kinds of human genes. Genome Res 11:677–684.

Svejstrup, J.Q., Vichi, P., and Egly, J.-M. 1996. The multiple roles of transcription/repair factor TFIIH. Trends Biochem Sci 21:346–350.

Taatjes, D.J., Näär, A.M., Andel, F., III, Nogales, E., and Tjian, R. 2002. Structure, function, and activator-induced conformations of the CRSP coactivator. Science 295:1058–1062.

Taggart, A.K.P. and Pugh, B.F. 1996. Dimerization of TFIID when not bound to DNA. Science 272:1331–1333.

Takada, S., Lis, J.T., Zhou, S., and Tjian, R. 2000. A TRF1:BRF complex directs Drosophila RNA polymerase III transcription. Cell 101:459–469.

Takagi, Y., Masuda, C.A., Chang, W.H., Komori, H., Wang, D., Hunter, T., Joazeiro, C.A., and Kornberg, R.D. 2005. Ubiquitin ligase activity of TFIIH and the transcriptional response to DNA damage. Mol Cell 18:237–243.

Tan, S., Garrett, K.P., Conaway, R.C., and Conaway, J.W. 1994. Cryptic DNA-binding domain in the C terminus of RNA polymerase II general transcription factor RAP30. Proc Natl Acad Sci USA 91:9808–9812.

Tan, S., Hunziker, Y., Sargent, D.F., and Richmond, T.J. 1996. Crystal structure of a yeast TFIIA/TBP/DNA complex. Nature 381:127–134.

Tan, P., Fuchs, S.Y., Chen, A., Wu, K., Gomez, C., Ronai, Z., and Pan, Z.-Q. 1999. Recruitment of a ROC1-CUL1 ubiquitin ligase by Skp1 and HOS to catalyze the ubiquitination of IkBa. Mol Cell 3:527–533.

Tan, Q., Linask, K.L., Ebright, R.H., and Woychik, N.A. 2000. Activation mutants in yeast RNA polymerase II subunit RPB3 provide evidence for a structurally conserved surface required for activation in eukaryotes and bacteria. Genes Dev 14:339–348.

Tanese, N., Pugh, B.F., and Tjian, R. 1991. Coactivators for a proline-rich activator purified from the multisubunit human TFIID complex. Genes Dev 5:2212–2224.

Tantin, D. and Carey, M. 1994. A heteroduplex template circumvents the energetic requirement for ATP during activated transcription by RNA polymerase II. J Biol Chem 269:17397–17400.

Teichmann, M., Wang, Z., Martinez, E., Tjernberg, A., Zhang, D., Vollmer, F., Chait, B.T., and Roeder, R.G. 1999. Human TATA-binding protein-related factor-2 (hTRF2) stably associates with hTFIIA in HeLa cells. Proc Natl Acad Sci USA 96:13720–13725.

Temin, H.M. and Mizutani, S. 1970. RNA-dependent DNA polymerase in virions of Rous sarcoma virus. Nature 226:1211–1213.

Thomas, J.O. 2001. HMG1 and 2: architectural DNA-binding proteins. Biochem Soc Trans 29:395–401.

Thomas, J.O. and Travers, A.A. 2001. HMG1 and 2, and related 'architectural' DNA-binding proteins. Trends Biochem Sci 26:167–174.

Thomas, M.C. and Chiang, C.-M. 2005. E6 oncoprotein represses p53-dependent gene activation via inhibition of protein acetylation independently of inducing p53 degradation. Mol Cell 17:251–264.

Thompson, C.M., Koleske, A.J., Chao, D.M. and Young, R.A. 1993. A multisubunit complex associated with the RNA polymerase II CTD and TATA-binding protein in yeast. Cell 73:1361–1375.

Thompson, D.K., Palmer, J.R., and Daniels, C.J. 1999. Expression and heat-responsive regulation of a TFIIB homologue from the archaeon Haloferax volcanii. Mol Microbiol 33:1081–1092.

Tie, F., Furuyama, T., Prasad-Sinha, J., Jane, E., and Harte, P.J. 2001. The Drosophila Polycomb Group proteins ESC and E(Z) are present in a complex containing the histone-binding protein p55 and the histone deacetylase RPD3. Development 128:275–286.

Timmers, H.T.M., Meyers, R.E., and Sharp, P.A. 1992. Composition of transcription factor B-TFIID. Proc Natl Acad Sci USA 89:8140–8144.

Timmers, H.T.M. and Sharp, P.A. 1991. The mammalian TFIID protein is present in two functionally distinct complexes. Genes Dev 5:946–1956.

Tirode, F., Busso, D., Coin, F., and Egly, J.-M. 1999. Reconstruction of the transcription factor TFIIH: assignment of functions for the three enzymatic subunits, XPB, XPD, and cdk7. Mol Cell 3:87–95.

Topalidou, I., Papamichos-Chronakis, M., Thireos, G., and Tzamarias, D. 2004. Spt3 and Mot1 cooperate in nucleosome remodeling independently of TBP recruitment. *EMBO J* 23: 1943–1948.

Tora, L. 2002. A unified nomenclature for TATA box binding protein (TBP)-associated factors (TAFs) involved in RNA polymerase II transcription. *Genes Dev* 16: 673–675.

Tsai, F.T.F. and Sigler, P.B. 2000. Structural basis of preinitiation complex assembly on human pol II promoters. *EMBO J* 19: 5–36.

Tulin, A. and Spradling, A. 2003. Chromatin loosening by poly(ADP)-ribose polymerase at *Drosophila* puff loci. *Science* 299: 560–562.

Ulmasov, T., Larkin, R.M., and Guilfoyle, T.J. 1996. Association between 36- and 13.6-kDa α-like subunits of Arabidopsis thaliana RNA polymerase II. *J Biol Chem* 271: 5085–5094.

Upadhyaya, A.B., Lee, S.H., and DeJong, J. 1999. Identification of a general transcription factor TFIIAα/β homolog selectively expressed in testis. *J Biol Chem* 274: 18040–18048.

Upadhyaya, A.B., Khan, M., Mou, T.-C., Junker, M., Gray, D.M., and DeJong, J. 2002. The germ cell-specific transcription factor ALF. Structural properties and stabilization of the TATA-binding protein (TBP)-DNA complex. *J Biol Chem* 277: 34208–34216.

Usuda, Y., Kubota, A., Berk, A.J., and Handa, H. 1991. Affinity purification of transcription factor IIA from HeLa cell nuclear extracts. *EMBO J* 10: 2305–2310.

Vanathi, P., Mishra, A.K., and Bhargava, P. 2003. Regulation of activity of the yeast TATA-binding protein through intra-molecular interactions. *J Biosci* 28: 413–421.

van der Knaap J.A., Borst, J.W., van der Vliet, P.C., Gentz, R., and Timmers H.T.M. 1997. Cloning of the cDNA for the TATA-binding protein-associated factor I 170 subunit of transcription factor B-TFIID reveals homology to global transcription regulators in yeast and *Drosophila*. *Proc Natl Acad Sci USA* 94: 11827–11832.

Van Dyke, M.W., Roeder, R.G., and Sawadogo, M. 1988. Physical analysis of transcriptional preinitiation complex assembly on a class II gene promoter. *Science* 241: 1335–1338.

Van Dyke, M.W., Sawadogo, M., and Roeder, R.G. 1989. Stability of transcription complexes on class II genes. *Mol Cell Biol* 9: 342–344.

van Vuuren, A.J., Vermeulen, W., Ma, L., Weeda, G., Appeldoorn, E., Jaspers, N.G., van der Eb, A.J., Bootsma, D., Hoeijmakers, J.H.J., and Humbert, S. 1994. Correction of xeroderma pigmentosum repair defect by basal transcription factor BTF2 (TFIIH). *EMBO J* 13: 1645–1653.

Veenstra, G.J., Weeks, D.L., and Wolffe, A.P. 2000. Distinct roles for TBP and TBP-like factor in early embryonic gene transcription in Xenopus. *Science* 290: 2312–2315.

Verrier, C.S., Roodi, N., Yee, C.J., Bailey, L.R., Jensen, R.A., Bustin, M., and Parl, F.F. 1997. High-mobility group (HMG) protein and TATA-binding protein-associated factor TAFII 30 affect estrogen receptor-mediated transcriptional activation. *Mol Endocrinol* 11: 1009–1019.

Verrijzer, C.P. and Tjian, R. 1996. TAFs mediate transcriptional activation and promoter selectivity. *Trends Biochem Sci* 21: 338–342.

Wakasugi, M., Kawashima, A., Morioka, H., Linn, S., Sancar, A., Mori, T., Nikaido, O., and Matsunaga, T. 2002. DDB accumulates at DNA damage sites immediately after UV irradiation and directly stimulates nucleotide excision repair. *J Biol Chem* 277: 1637–1640.

Walker, S.S., Reese, J.C., Apone, L.M., and Green, M.R. 1996. Transcription activation in cells lacking TAFIIs. *Nature* 383: 185–188.

Wampler, S.L. and Kadonaga, J.T. 1992. Functional analysis of *Drosophila* transcription factor IIB. *Genes Dev* 6: 1542–1552.

Wang, W., Gralla, J.D., and Carey, M. 1992. The acidic activator GAL4-AH can stimulate polymerase II transcription by promoting assembly of a closed complex requiring TFIID and TFIIA. *Genes Dev* 6: 1716–1727.

Wang, Z., Svejstrup, J.Q., Feaver, W.J., Wu, X., Kornberg, R.D., and Friedberg, E.C. 1994. Transcription factor b (TFIIH) is required during nucleotide-excision repair in yeast. *Nature* 368: 74–76.

Wang, E.H. and Tjian, R. 1994. Promoter-selective transcriptional defect in cell cycle mutant ts13 rescued by hTAFII 250. *Science* 263: 811–814.


Wang, Z., Buratowski, S., Svejstrup, J.Q., Feaver, W.J., Wu, X., Kornberg, R.D., Donahue, T.F., and Friedberg, E.C. 1995. The yeast TFB1 and SSL1 genes, which encode subunits of transcription factor IIH, are required for nucleotide excision repair and RNA polymerase II transcription. *Mol Cell Biol* 15: 2288–2293.

Wang, J.C. 1996. DNA topoisomerases. *Annu Rev Biochem* 65: 635–692.

Wang, G., Yang, J., and Huibregtse, J.M. 1999. Functional domains of the Rsp5 ubiquitin-protein ligase. *Mol Cell Biol* 19: 342–352.

Wang, C., Deng, L., Hong, M., Akkaraju, G.R., Inoue, J., and Chen, Z.J. 2000. TAK1 is a ubiquitin-dependent kinase of MKK and IKK. *Nature* 412: 346–351.

Wang, J.C. 2002. Cellular roles of DNA topoisomerases: a molecular perspective. *Nature Rev Mol Cell Biol* 3: 430–440.

Wang, J.Y., Sarker, A.H., Cooper, P.K., and Volkert, M.R. 2004. The single-strand DNA binding activity of human PC4 prevents mutagenesis and killing by oxidative DNA damage. *Mol Cell Biol* 24: 6084–6093.

Wang, G., Balamotis, M.A., Stevens, J.L., Yamaguchi, Y., Handa, H., and Berk, A.J. 2005. Mediator requirement for both recruitment and postrecruitment steps in transcription initiation. *Mol Cell* 17: 683–694.

Warfield, L., Ranish, J.A., and Hahn, S. 2004. Positive and negative functions of the SAGA complex mediated through interaction of Spt8 with TBP and the N-terminal domain of TFIIA. *Genes Dev* 18: 1022–1034.

Watanabe, T., Hayashi, K., Tanaka, A., Furumoto, T., Hanaoka, F., and Ohkuma, Y. 2003. The carboxy terminus of the small subunit of TFIIE regulates the transition from transcription initiation to elongation by RNA polymerase II. *Mol Cell Biol* 23: 2914–2926.

Wei, W., Dorjsuren, D., Lin, Y., Qin, W., Nomura, T., Hayashi, N., and Murakami, S. 2000. Direct interaction between subunit RAP30 of the transcription factor IIF (TFIIF) and RNA polymerase subunit 5, which contributes to the association between TFIIF and RNA polymerase II. *J Biol Chem* 276: 12266–12273.

Weil, P.A. and Blatti, S.P. 1976. HeLa cell deoxyribonucleic acid dependent RNA polymerases: function and properties of the class III enzymes. *Biochemistry* 15: 1500–1509.

Weil, P.A., Luse, D.S., Segall, J., and Roeder, R.G. 1979. Selective and accurate initiation of transcription at the Ad2 major late promoter in a soluble system dependent on purified RNA polymerase II and DNA. *Cell* 18: 469–484.

Weinmann, R., Raskas, H.J., and Roeder, R.G. 1974. Role of DNA-dependent RNA polymerases II and III in transcription of the adenovirus genome late in productive infection. *Proc Natl Acad Sci USA* 71: 3426–3439.

Weinmann, R. and Roeder, R.G. 1974. Role of DNA-dependent RNA polymerase III in the transcription of the tRNA and 5S RNA genes. *Proc Natl Acad Sci USA* 71: 1790–1794.

Weiss, S. and Gladstone, L. 1959. A mammalian system for the incorporation of cytidine triphosphate into ribonucleic acid. *J Am Chem Soc* 81: 4118–4119.

Weissman, A.M. 2000. Themes and variations on ubiquitylation. *Nat Rev Mol Cell Biol* 2: 169–178.

Werten, S., Stelzer, G., Goppelt, A., Langen, F.M., Gros, P., Timmers, H.T.M., Van der Vliet, P.C., and Meisterernst M. 1998. Interaction of PC4 with melted DNA inhibits transcription. *EMBO J* 17, 5103–5111.

Wieczorek, E., Brand, M., Jacq, X., and Tora, L. 1998. Function of TAFII-containing complex without TBP in transcription by RNA polymerase II. *Nature* 393: 187–191.

Willy, P.J., Kobayashi, R., and Kadonaga, J.T. 2000. A basal transcription factor that activates or represses transcription. *Science* 290: 982–985.

Wilson, C.J., Chao, D.M., Imbalzano, A.N., Schnitzler, G.R., Kingston, R.E., and Young, R.A. 1996. RNA polymerase II holoenzyme contains SWI/SNF regulators involved in chromatin remodeling. *Cell* 84: 235–244.

Winkler, G.S., Vermeulen, W., Coin, F., Egly, J.-M., Hoeijmakers, J.H., and Weeda, G. 1998. Affinity purification of human DNA

M. C. Thomas and C.-M. Chiang

repair/transcription factor TFIH using epitope-tagged xeroderma pigmentosum B protein. J Biol Chem 273:1092–1098.

Wolner, B.S. and Gralla, J.D. 2000. TATA-flanking sequences influence the rate and stability of TATA-binding protein and TFIIB binding. J Biol Chem 276:6260–6266.

Workman, J.L. and Roeder, R.G. 1987. Binding of transcription factor TFIID to the major late promoter during in vitro nucleosome assembly potentiates subsequent initiation by RNA polymerase II. Cell 51:613–622.

Wösten, M.M. 1998. Eubacterial sigma-factors. FEMS Microbiol Rev 22:127–150.

Woudstra, E.C., Gilbert, C., Fellows, J., Jansen, L., Brouwer, J., Erdjument-Bromage, H., Tempst, P., and Svejstrup, J.Q. 2002. A Rad26-Def1 complex coordinates repair and RNA pol II proteolysis in response to DNA damage. Nature 415:929–933.

Woychik, N.A., Liao, S.M., Kolodziej, P.A., and Young, R.A. 1990. Subunits shared by eukaryotic nuclear RNA polymerases. Genes Dev 4:313–323.

Woychik, N.A., McKune, K., Lane, W.S., and Young, R.A. 1993. Yeast RNA polymerase II subunit RPB11 is related to a subunit shared by RNA polymerase I and III. Gene Expr 3:77–82.

Woychik, N.A. and Hampsey, M. 2002. The RNA polymerase II machinery: structure illuminates function. Cell 108:453–463.

Wu-Baer, F., Lagrazon, K., Yuan, W., and Baer, R. 2003. The BRCA1/BARD1 heterodimer assembles polyubiquitin chains through an unconventional linkage involving lysine residue K6 of ubiquitin. J Biol Chem 278:34743–34746.

Wu, S.-Y. and Chiang, C.-M. 1998. Properties of PC4 and an RNA polymerase II complex in directing activated and basal transcription in vitro. J Biol Chem 273:12492–12498.

Wu, S.-Y., Kershner, E., and Chiang, C.-M. 1998. TAFII-independent activation mediated by human TBP in the presence of the positive cofactor PC4. EMBO J 17:4478–4490.

Wu, W.-H. and Hampsey, M. 1999. An activation-specific role for transcription factor TFIIB in vivo. Proc Natl Acad Sci USA 96:2764–2769.

Wu, S.-Y., Thomas, M.C., Hou, S.Y., Likhite, V., and Chiang, C.-M. 1999. Isolation of mouse TFIID and functional characterization of TBP and TFIID in mediating estrogen receptor and chromatin transcription. J Biol Chem 274:23480–23490.

Wu, S.-Y. and Chiang, C.-M. 2001a. TATA-binding protein-associated factors enhance the recruitment of RNA polymerase II by transcriptional activators. J Biol Chem 276:34235–34243.

Wu, S.-Y. and Chiang, C.-M. 2001b. Expression and purification of epitope-tagged multisubunit protein complexes from mammalian cells. Current Protocols in Molecular Biology, Unit 16.22.1–16.22.17.

Wu, P.Y. and Winston, F. 2002. Analysis of Spt7 function in the Saccharomyces cerevisiae SAGA coactivator complex. Mol Cell Biol 22:5367–5379.

Wu, S.-Y., Zhou, T., and Chiang, C.-M. 2003. Human Mediator enhances activator-facilitated recruitment of RNA polymerase II and promoter recognition by TATA-binding protein (TBP) independently of TBP-associated factors. Mol Cell Biol 23:6229–6242.

Wu, P.-Y.J., Ruhlmann, C., Winston, F., and Schultz, P. 2004. Molecular architecture of the S. cerevisiae SAGA complex. Mol Cell 15:199–208.

Xia, Y., Pao, G.M., Chen, H.W., Verma, I.M., and Hunter, T. 2003. Enhancement of BRCA1 E3 ubiquitin ligase activity through direct interaction with the BARD1 protein. J Biol Chem 278:5255–5263.

Xie, X., Kokubo, T., Cohen, S.L., Mirza, U.A., Hoffmann, A., Chait, B.T., Roeder, R.G., Nakatani, Y., and Burley, S.K. 1996. Structural similarity between TAFs and the heterotetrameric core of the histone octamer. Nature 380:316–322.

Xie, J., Collart, M., Lemaire, M., Stelzer, G., and Meisterernst, M. 2000. A single point mutation in TFIIA suppresses NC2 requirement in vivo. EMBO J 19:672–682.

Yamamoto, S., Watanabe, Y., van der Spek, P.J., Watanabe, T., Fujimoto, H., Hanaoka, F., and Ohkuma, Y. 2000. Studies of nematode TFIIIE

function reveal a link between Ser-5 phosphorylation of RNA polymerase II and the transition from transcription initiation to elongation. Mol Cell Biol 21:1–15.

Yamashita, S., Wada, K., Horikoshi, M., Gong, D.-W., Kokubo, T., Hisatake, K., Yokotani, N., Malik, S., Roeder, R.G., and Nakatani, Y. 1992. Isolation and characterization of a cDNA encoding Drosophila transcription factor TFIIB. Proc Natl Acad Sci USA 89:2839–2843.

Yamashita, S., Hisatake, K., Kokubo, T., Doi, K., Roeder, R.G., Horikoshi, M., and Nakatani, Y. 1993. Transcription factor TFIIB sites important for interaction with promoter-bound TFIID. Science 261:463–466.

Yan, Q., Moreland, R.J., Conaway, J.W., and Conaway, R.C. 1999. Dual roles for transcription factor IIF in promoter escape by RNA polymerase II. J Biol Chem 274:35668–35675.

Yanagisawa, J., Kitagawa, H., Yanagida, M., Wada, O., Ogawa, S., Nakagomi, M., Oishi, H., Yamamoto, Y., Nagasawa, H., McMahon, S.B., Cole, M.D., Tora, L., Takahashi, N., and Kato, S. 2002. Nuclear receptor function requires a TFTC-type histone acetyltransferase complex. Mol Cell 9:553–562.

Yang, Z., Zhu, Q., Luo, K., and Zhou, Q. 2000. The 7SK small nuclear RNA inhibits the CDK9/cyclin T1 kinase to control transcription. Nature 414:317–322.

Yang, F., DeBeaumont, R., Zhou, S., and Näär, A.M. 2004. The activator-recruited cofactor/Mediator coactivator subunit ARC92 is a functionally important target of the VP16 transcriptional activator. Proc Natl Acad Sci USA 101:2339–2344.

Yankulov, K.Y. and Bentley, D.L. 1997. Regulation of CDK7 substrate specificity by MAT1 and TFIIH. EMBO J 16:1638–1646.

Yeo, M., Lin, P.S., Dahmus, M.E., and Gill, G.N. 2003. A novel RNA polymerase II C-terminal domain phosphatase that preferentially dephosphorylates serine 5. J Biol Chem 278:26078–26085.

Yokomori, K., Admon, A., Goodrich, J.A., Chen, J.-L., and Tjian, R. 1993. Drosophila TFIIA-L is processed into two subunits that are associated with the TBP/TAF complex. Genes Dev 7:2235–2245.

Yokomori, K., Verrijzer, C.P., and Tjian, R. 1998. An interplay between TATA box-binding protein and transcription factors IIE and IIA modulates DNA binding and transcription. Proc Natl Acad Sci USA 95:722–6727.

Yonaha, M., Tsuchiya, T., and Yasukochi, Y. 1997. Cell-cycle-dependent phosphorylation of the basal transcription factor RAP74. FEBS Lett 410:477–480.

Yong, C., Mitsuyasu, H., Chun, Z., Oshiro, S., Hamasaki, N., and Kitajima, S. 1998. Structure of the human transcription factor TFIIIF revealed by limited proteolysis with trypsin. FEBS Lett 435:191–194.

Young, R.A. 1991. RNA polymerase II. Annu Rev Biochem 60:689–715.

Yudkovsky, N., Ranish, J.A., and Hahn, S. 2000. A transcription reinitiation intermediate that is stabilized by activator. Nature 408:225–229.

Yuan, C.-X., Ito, M., Fondell, J.D., Fu, Z.-Y., and Roeder, R.G. 1998. The TRAP220 component of a thyroid hormone receptor-associated protein (TRAP) coactivator complex interacts directly with nuclear receptors in a ligand-dependent fashion. Proc Natl Acad Sci USA 95:7939–7944.

Yuryev, A., Patturajan, M., Litingtung, Y., Joshi, R.V., Gentile, C., Gebara, M., and Corden, J.L. 1996. The C-terminal domain of the largest subunit of RNA polymerase II interacts with a novel set of serine/arginine-rich proteins. Proc Natl Acad Sci USA 93:6975–6980.

Zanton, S.J. and Pugh B.F. 2004. Changes in genomewide occupancy of core transcriptional regulators during heat stress. Proc Natl Acad Sci USA 101:16843–16848.

Zehring, W.A. and Greenleaf, A.L. 1990. The carboxyl-terminal repeat domain of RNA polymerase II is not required for transcription factor Sp1 to function in vitro. J Biol Chem 265:8351–8353.

Zhang, M.Q. 1998. Identification of human gene core promoters in silico. Genome Res 8:319–326.

Zhang, J. and Fondell, J.D. 1999. Identification of mouse TRAP100: a transcriptional coregulatory factor for thyroid hormone and vitamin D receptors. Mol Endocrinol 13:1130–1140.

Zhang, D., Penttila, T.-L., Morris, P.L., Teichmann, M., and Roeder, R.G. 2000. Spermiogenesis deficiency in mice lacking the Trf2 gene. Science 292:1153–1155.

Zhang, C. and Burton, Z.F. 2004. Transcription factors IIF and IIS and nucleoside triphosphate substrates as dynamic probes of the human RNA polymerase II mechanism. J Mol Biol 342:1085–1099.

Zhang, C., Zobeck, K.L., and Burton, Z.F. 2005. Human RNA polymerase II elongation in slow motion: role of the TFIIF RAP74 α1 helix in nucleoside triphosphate-driven translocation. Mol Cell Biol 25:3583–3595.

Zhao, X. and Herr, W. 2002. A regulated two-step mechanism of TBP binding to DNA: a solvent-exposed surface of TBP inhibits TATA box recognition. Cell 108:615–627.

Zhou, Q., Lieberman, P.M., Boyer, T.G., and Berk, A.J. 1992. Holo-TFIID supports transcriptional stimulation by diverse activators and from a TATA-less promoter. Genes Dev 6:1964–1974.

Zhou, Q., Boyer, T.G., and Berk, A.J. 1993. Factors (TAFs) required for activated transcription interact with TATA box-binding protein conserved core domain. Genes Dev 7:180–187.

Zhou, M., Halanski, M.A., Radonovich, M.F., Kashanchi, F., Peng, J., Price, D.H., and Brady, J.N. 2000. Tat modifies the activity of CDK9 to phosphorylate serine 5 of the RNA polymerase II carboxyl-terminal domain during human immunodeficiency virus type 1 transcription. Mol Cell Biol 20:5077–5086.

Zhou, T. and Chiang, C.-M. 2000. The intronless and TATA-less human TAFII55 gene contains a functional initiator and a downstream promoter element. J Biol Chem 276:25503–25511.

Zhou, T. and Chiang, C.-M. 2002. Sp1 and AP2 regulate but do not constitute TATA-less human TAFII55 core promoter activity. Nucleic Acids Res 30:4145–4157.

Zhu, A. and Kuziora, M.A. 1996. Homeodomain interaction with the β subunit of the general transcription factor TFIIE. J Biol Chem 271:20993–20996.

Ziegler, L.M., Khaperskyy, D.A., Ammerman, M.L., and Ponticelli, A.S. 2003. Yeast RNA polymerase II lacking the Rpb9 subunit is impaired for interaction with transcription factor IIF. J Biol Chem 278:48950–48956.

Zurita, M. and Merino, C. 2003. The transcriptional complexity of the TFIIH complex. Trends Genet 19:578–584.

Zylber, E.A. and Penman, S. 1971. Products of RNA polymerases in HeLa cell nuclei. Proc Natl Acad Sci USA 68:2861–2865.
