
THE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 281, NO. 40, pp. 29633–29640, October 6, 2006  
© 2006 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in the U.S.A.

# Substrate Specificity of the Adenylation Enzyme SgcC1 Involved in the Biosynthesis of the Enediyne Antitumor Antibiotic C-1027*

Received for publication, June 20, 2006, and in revised form, August 1, 2006 Published, JBC Papers in Press, August 3, 2006, DOI 10.1074/jbc.M605887200

Steven G. Van Lanen ${}^{1}$, Shuangjun Lin ${}^{2}$, Pieter C. Dorrestein ${}^{2}$, Neil L. Kelleher ${}^{S}$, and Ben Shen ${}^{\dagger||3}$

From the ${}^{\dagger}$Division of Pharmaceutical Sciences, ${}^{9}$University of Wisconsin National Cooperative Drug Discovery Group, ${}^{||}$Department of Chemistry, University of Wisconsin, Madison, Wisconsin 53705 and ${}^{S}$Department of Chemistry, University of Illinois, Urbana, Illinois 61801

---

C-1027 is an enediyne antitumor antibiotic composed of a chromophore with four distinct chemical moieties, including an $(S)$-3-chloro-4,5-dihydroxy-$\beta$-phenylalanine moiety that is derived from $L$-$\alpha$-tyrosine. SgcC4, a novel aminomutase requiring no added co-factor that catalyzes the formation of the first intermediate $(S)$-$\beta$-tyrosine and subsequently SgcC1 homologous to adenylation domains of nonribosomal peptide synthetases, was identified as specific for the SgcC4 product and did not recognize any $\alpha$-amino acids. To definitively establish the substrate for SgcC1, a full kinetic characterization of the enzyme was performed using amino acid-dependent ATP-${}^{[32P]}$PP${}_{i}$ exchange assay to monitor amino acid activation and electrospray ionization-Fourier transform mass spectroscopy to follow the loading of the activated $\beta$-amino acid substrate to the peptidyl carrier protein SgcC2. The data establish $(S)$-$\beta$-tyrosine as the preferred substrate, although SgcC1 shows promiscuous activity toward aromatic $\beta$-amino acids such as $\beta$-phenylalanine, 3-chloro-$\beta$-tyrosine, and 3-hydroxy-$\beta$-tyrosine, but all were <50-fold efficient. A putative active site mutant P571A adjacent to the invariant aspartic acid residue of all $\alpha$-amino acid-specific adenylation domains known to date was prepared as a preliminary attempt to probe the substrate specificity of SgcC1; however, the mutation resulted in a loss of activity with all substrates except $(S)$-$\beta$-tyrosine, which was 142-fold less efficient relative to the wild-type enzyme. In total, SgcC1 is now confirmed to catalyze the second step in the biosynthesis of the $(S)$-3-chloro-4,5-dihydroxy-$\beta$-phenylalanine moiety of C-1027, presenting downstream enzymes with an $(S)$-$\beta$-tyrosyl-$S$-SgcC2 thioester substrate, and represents the first $\beta$-amino acid-specific adenylation enzyme characterized biochemically.

C-1027 is an enediyne antitumor antibiotic isolated from the fermentation broth of *Streptomyces globisporus* (1, 2). It is produced as a chromoprotein complex consisting of a binding protein (annotated as CagA) and the reactive C-1027 chromophore containing a conjugated enediyne harbored within a nine-membered cyclic ring (see Fig. 1). Upon release from CagA, the enediyne core of the C-1027 chromophore readily undergoes a Bergman cycloaromatization to yield a transient biradical species capable of extracting hydrogen atoms from DNA, which in the presence of molecular oxygen can ultimately lead to single- and double-stranded DNA breaks (3). C-1027 and the entire enediyne family share this mode of action, and as a family, their potent cytotoxicity rivals that of any previously discovered natural product (4).

The biosynthetic gene cluster for C-1027 was previously cloned and sequenced, and analysis of the open reading frames (ORFs) ${}^{4}$ provided genetic evidence to adequately propose a mechanism of biosynthesis, transport, resistance, and regulation of C-1027 (5). Comparison of the gene cluster to other members of the enediyne family, including calicheamicin (6) and neocarzinostatin (7), revealed a unified biosynthetic approach among the enediynes and supported a convergent biosynthesis of C-1027 from four components, a deoxy aminosugar, benzoxazoline, $\beta$-amino acid, and enediyne core starting from glucose-1-phosphate, chorismic acid, $L$-$\alpha$-tyrosine, and acyl-CoAs, respectively (Fig. 1). The pathway to generating the C-1027 deoxy aminosugar was initially targeted for biochemical analysis, and the first reaction catalyzed by SgcA1 has been characterized as a $\alpha$-$D$-glucopyranosyl-1-phosphate thymidylyltransferase (8). Of the remaining moieties, only the initial step for the $\beta$-amino acid moiety $(S)$-3-chloro-4,5-dihydroxy-$\beta$-phenylalanine has been identified, a novel aminomutase reaction catalyzed by SgcC4 converting $L$-$\alpha$-tyrosine to $(S)$-$\beta$-tyrosine (9).

Among the 56 ORFs identified within the boundaries of the C-1027 biosynthetic gene cluster, three ORFs exhibit high homology to nonribosomal peptide synthetase (NRPS) domains. NRPSs are typically large modular proteins that catalyze the synthesis of a wide range of biologically active peptides

---

* This work is supported in part by National Institutes of Health Grants GM067725 (to N.L.K.) and CA78747 (to B.S.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
1 Recipient of a National Institutes of Health postdoctoral fellowship (CA1059845).
2 Recipient of a National Institutes of Health postdoctoral fellowship (GM073323).
3 Recipient of a National Institutes of Health Independent Scientist Award AI51687. To whom correspondence should be addressed: Div. of Pharmaceutical Sciences, School of Pharmacy, University of Wisconsin-Madison, 777 Highland Ave., Madison, WI 53705. Tel.: 608-263-2673; Fax: 608-262-5245; E-mail: bshen@pharmacy.wisc.edu.

4 The abbreviations used are: ORF, open reading frame; NRPS, nonribosomal peptide synthetase; PCP, peptidyl carrier protein; ESI-MS, electrospray ionization-mass spectrometry; ESI-FTMS, electrospray ionization-Fourier transform mass spectrometry; PheA, the excised $L$-Phe-specific adenylation domain of GrsA; ACES, 2-[(2-amino-2-oxoethyl)amino]ethanesulfonic acid.

The SgcC1 (S)-β-Tyrosine-specific Adenylation Enzyme

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

FIGURE 1. Structure of C-1027 and a convergent model for C-1027 biosynthesis from primary metabolites.

(10). A minimal NRPS module consists of three domains, an adenylation domain that sequentially selects, activates, and loads an amino acid (or carboxylic acid) to the 4′-phosphopantetheine prosthetic group of a peptidyl carrier protein (PCP) domain and a condensation domain that catalyzes the formation of an amide bond. In most examples, a single NRPS protein contains multiple modules, and the primary sequence of the resulting peptide product is correlated to the linear architecture of the modules (hence the so-called “thiotemplate colinearity” rule). Extra domains embedded in the multimodular NRPS proteins, such as epimerization, methyltransferase, or oxidation domains, can be present to afford final peptides with enriched chemistry. As opposed to large multimodular proteins, singular domains or didomains of NRPS as individual ORFs have also been recently characterized, and this phenomenon has facilitated research toward understanding domain-domain recognition and interaction in NRPS and the molecular mechanism of NRPS-catalyzed nonribosomal peptide biosynthesis.

Within the C-1027 biosynthetic gene cluster, two ORFs upstream and in opposite orientation to *sgcC4* were found to encode proteins with sequence similarities to condensation (SgcC5) and PCP (SgcC2) domains (Fig. 2A). An ORF located adjacent to and downstream of *sgcC4* was also identified and encodes an 881-amino-acid protein, SgcC1, having sequence homology to adenylation domains. Although SgcC1 has closest homology to adenylation domains that activate L-α-tyrosine, preliminary characterization has revealed that SgcC1 recognizes (S)-β-tyrosine as opposed to any of the standard α-amino acids, including either isomer of α-tyrosine or α-phenylalanine or analogs (11). Although these results support the findings that SgcC4 is the initial enzyme in the pathway (catalyzing the stereospecific formation of (S)-β-tyrosine from L-α-tyrosine) and SgcC1 catalyzes the subsequent step (Fig. 2B), they contradicted the prediction on the basis of the so-called “nonribosomal codes” (12, 13) that SgcC1 would be an α-amino acid-specific adenylation domain (11).

As a follow-up to the initial characterization of SgcC1, we now provide definitive experimental data to assign the substrate specificity for SgcC1. Using amino acid-dependent ATP-[³²P]PPᵢ exchange assays and electrospray ionization-Fourier transform mass spectrometry (ESI-FTMS), chlorinated and hydroxylated tyrosine or phenylalanine analogs were tested revealing that (S)-β-tyrosine is the preferred substrate for SgcC1. The results unambiguously establish that SgcC1, following the SgcC4 aminomutase reaction, catalyzes the second step in the biosynthesis of the (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety from L-α-tyrosine by forming the (S)-β-tyrosyl-S-SgcC2 intermediate. Subsequent steps involve halogenation (SgcC3), hydroxylation (SgcC), and incorporation of the fully modified β-amino acid unit into the enediyne core (SgcC5), although the precise timing of the last steps awaits further validation (Fig. 2B). SgcC1 as a naturally occurring, discrete protein could therefore provide a relatively simple platform to address the inherent amino acid specificity-conferring elements of adenylation domains to β-amino acids, of which the current nonribosomal codes were based solely on a single structure of the excised, L-α-phenylalanine-specific adenyla-

The SgcC1 (S)-β-Tyrosine-specific Adenylation Enzyme

A.
![Diagram](https://i.imgur.com/yourimageurl.png)

B.
![Chemical Structures](https://i.imgur.com/yourimageurl2.png)

L-α-tyrosine

FIGURE 2. Biosynthesis of (S)-3-chloro-4,5-dihydroxy-β-phenylalanine. Shown is a subcluster of genes within the C-1027 biosynthetic gene cluster that encodes a minimal NRPS module consisting of three discrete proteins (SgcC1, SgcC2, and SgcC3) (A) and a proposed pathway for biosynthesis of the (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety of C-1027 (B). SgcC4, L-α-tyrosine aminomutase; SgcC1, (S)-β-tyrosine adenylation enzyme; SgcC2, peptidyl carrier protein; SgcC3, halogenase; SgcC, hydroxylase; SgcC5, condensation enzyme.

tion domain of GrsA (PheA) (12, 13). As an initial attempt to determine the structural elements of SgcC1 necessary for proper β-amino acid selection, truncated protein and a single point mutation of SgcC1 were generated and the results reported.

# EXPERIMENTAL PROCEDURES

**Chemicals and Instrumentation—**If not mentioned, chemicals and instruments used were identical to that previously reported (11). Cinnamic acid, *p*-hydroxycinnamic acid, and 3-amino-3-phenylpropionic acid (β-phenylalanine) were from Fisher-Acros (Pittsburgh, PA). (*R*)-3-Amino-(4-hydroxy-phenyl)-propionic acid ((*R*)-β-tyrosine) was from PepTech Corporation (Burlington, MA). Restriction enzymes were from New England Biolabs (Beverly, MA) or Invitrogen, and expression vectors were from Novagen (Madison, WI). Unless explicitly stated, the compounds used were racemic mixtures. Protein analysis was performed with position-specific iterated-BLAST and BL2SEQ using the San Diego Supercomputer Center Biology WorkBench, version 3.2, and the adenylation domain active sites were extracted using the NRPS Predictor software from Universität Tübingen.

**Synthesis of Racemic 3-Chloro-β-tyrosine—**Synthesis of 3-chloro-β-tyrosine was achieved by following the method reported by Weaver and co-worker (14). 3-Chloro-benzaldehyde (76 mg) was refluxed with 1 equivalent of malonic acid (51 mg) and 2 equivalents of ammonium acetate (76 mg) in ethanol (5 ml) for 2 days. The reaction mixture was adjusted to an approximate pH of 4 with 1 N HCl and separated by a strong acid cation exchange column (Dowex® 50WX8). The product, 3-chloro-β-tyrosine, was eluted with 1% ammonium hydroxide. The residue was dissolved in distilled water after the solvent was removed through evaporation under reduced pressure and further purified on C-18 reverse phase column (Alltima 10 × 250 mm, 5 μm; Grace Davison Discovery Sciences). ${}^{1}$H NMR assignments in D$_{2}$O containing CF$_{3}$CO$_{2}$D at 400 MHz are as follows (where d means doublet, dd means doublet of doublets, m means multiplet, and t means triplet): δ 7.08 (d, $J = 2.4$ Hz, 1H), 6.87 (dd, $J = 8.8$, 2.4 Hz, 1H), 6.65 (d, $J = 8.4$ Hz, 1H), 4.30 (t, $J = 7.2$ Hz, Hβ, 1H), 2.77 (dd, $J = 17.2$, 7.6 Hz, Hα, 1H), and 2.66 (dd, $J = 17.2$, 6.4 Hz, Hα, 1H). ${}^{13}$C NMR assignments in D$_{2}$O at 100 MHz are as follows: δ 173.1 (C=O), 152.8 (ArC), 129.2 (ArC), 128.0 (ArC), 127.1 (ArC), 120.7 (ArC), 117.6 (ArC), 50.7 (Cβ), and 37.5 (Ca). ESI-MS gave $m/z$: [M + H]$^{+}$, 216.0; [M − H]$^{-}$, 213.9; calculated 215.0 for C$_{9}$H$_{10}$NO$_{3}$Cl.

**Synthesis of Racemic 3-Hydroxy-β-tyrosine—**3-Hydroxy-β-tyrosine was obtained by the same procedure as that used for the synthesis of 3-chloro-β-tyrosine beginning from 3-hydroxy-benzaldehyde. ${}^{1}$H NMR assignments in D$_{2}$O at 400 MHz are as follows: δ 6.74–6.85 (m, 3H), 4.42 (t, $J = 6.8$ Hz, Hβ, 1H), 2.79 (dd, $J = 15.6$, 8.0 Hz, Hα, 1H), and 2.69 (dd, $J = 16.0$, 6.4 Hz, Hα, 1H). ${}^{13}$C NMR assignments in D$_{2}$O at 125 MHz are as follows: δ 40.9 (Ca), 52.7 (Cβ), 115.0 (ArC), 116.6 (ArC), 119.4 (ArC), 128.7 (ArC), 145.6 (ArC), 145.5 (ArC), and 177.9 (C=O). ESI-MS gave $m/z$: [M + H]$^{+}$, 198.0; [M − H]$^{-}$, 195.9; calculated 197.0 for C$_{9}$H$_{11}$NO$_{4}$.

**Synthesis of Racemic 3-Chloro-5-hydroxy-β-tyrosine—**3-Chloro-5-hydroxy-β-tyrosine was synthesized by the same procedure as that used for the synthesis of 3-chloro-β-tyrosine beginning from 3-chloro-4,5-dihydroxybenzaldehyde, which was prepared from 3-chloro-4-hydroxyl-5-methoxybenzaldehyde following the method reported by Perchellet and co-workers (15). ${}^{1}$H NMR assignments in D$_{2}$O at 500 MHz are as follows: δ 7.04 (d, $J = 2.5$ Hz, 1H), 6.92 (d, $J = 2.5$ Hz, 1H), 4.60 (t, $J = 7.5$ Hz, 1H), 3.04 (dd, $J = 17.0$, 8.0 Hz, 1H), and 2.95 (dd, $J = 17.0$, 6.5 Hz, 1H). ${}^{13}$C NMR assignments in D$_{2}$O at 125 MHz are as follows: δ 38.8 (Ca), 51.6 (Cβ), 113.6, 120.2, 121.8, 128.6, 141.8, 146.1 (ArC), and 175.1 (C=O). ESI-MS gave $m/z$: [M + H]$^{+}$, 232.1; calculated 231.0 for C$_{9}$H$_{10}$NO$_{4}$Cl.

**DNA Manipulations—**Cloning and construction of the *sgcC1* (pBS1033) and *sgcC2* (pBS1034) expression vectors were previously described (11). A truncated form of SgcC1 was prepared by PCR amplification using pBS1005 (5) as a template and a forward primer of 5′-GGGAATTCCATATGGGCGCTCTGCCGCTGGAC-3′ (NdeI site underlined) and a reverse primer of 5′-GGCAAGCTTGCAGGGGTGAGCCGGGAGCG-3′ (HindIII site underlined). The amplified 1629-base-pair fragment was cloned into the same sites of pET29a to yield pBS1037. After confirmation of the DNA sequence fidelity, the NdeI/HindIII fragment was isolated and cloned into pET28a to yield pBS1038. Although the former construct produces SgcC1 as a C-terminal His$_{6}$-tagged protein, the latter construct produces SgcC1 as a C- and N-terminal His$_{6}$-tagged protein, both

The SgcC1 (S)-β-Tyrosine-specific Adenylation Enzyme

of which have the 338 amino acids at the N terminus of SgcC1 deleted.

A P571A point mutation of SgcC1 was generated by PCR amplification of the template pBS1033 (11) using the Expand long template PCR system (Roche Applied Science). Reactions were performed using the manufacturer’s provided Buffer 2 with 5% Me₂SO, primers of 5′-GTCTCCCCGGAGCACGA-CGCGGCGCTGGCCGAGGTC-3′ and the reverse complement (with the Ala codon underlined), and a PCR program consisting of an initial hold at 94 °C for 2 min followed by 20 cycles of 94 °C for 10 s, 56 °C for 30 s, and 68 °C for 7 min. The template DNA was digested with 10 units of DpnI for 1 h at 37 °C followed by heating to 90 °C for 5 min and cooling at room temperature before transformation. The introduction of the correct point mutation and the fidelity of the entire gene including 250 bp upstream and downstream were confirmed by DNA sequencing to yield pBS1039.

**Amino Acid-dependent ATP-[³²P]PPᵢ Exchange Assays—Assessment of the SgcC1 adenylation enzyme substrate specificity** was performed as described previously (11). A pH profile for SgcC1 was generated using a three-buffer system of 52/52/100 mM Tris-ACES-ethanolamine as described previously (16). Assays were run at 30 °C in 1× buffer with 5 mM MgCl₂, 0.1 mM EDTA, 5 mM ATP, 0.9 μM [³²P]PPᵢ, 1 mM (S)-β-tyrosine, and 10 nM SgcC1 under initial velocity conditions (<10% isotopic conversion into ATP). Each data point represents a minimum of four duplicate end-point assays. Because the pH profile was utilized only for determining the conditions for optimal activity, the data were fitted to a smooth curve in contrast to a specific rate equation, which would warrant further kinetic analysis at varied pH.

The steady-state kinetic parameters for SgcC1 were determined with activity assays carried out at 30 °C in 100 mM Tris-HCl (pH 9.0), 5 mM MgCl₂, 0.1 mM EDTA, 5 mM ATP, 1.0 μM [³²P]PPᵢ, and varied co-substrate as follows: 25–1600 μM for 3-chloro-β-tyrosine, 3-hydroxy-β-tyrosine, and β-phenylalanine, 5–370 μM for (R)-β-tyrosine, and 0.5–200 μM for (S)-β-tyrosine. Enzyme concentration for each substrate ranged from 25 to 70 nM to maintain initial velocity conditions. Single time points were analyzed between 1 and 5 min; each data point represents a minimum of four duplicates with a S.D. of <10%. Kinetic constants were extracted by nonlinear regression analysis using Kaleidagraph software (Adelbeck Software, Reading, PA).

**Activity of SgcC1 with SgcC2—Preparation of recombinant SgcC1 and SgcC2 proteins**, including overproduction in *Escherichia coli*, purification, and *in vitro* 4′-phosphopantetheinylation of apoSgcC2, were as described previously (11). Acylation with (S)-β-tyrosine was analyzed as described previously (11). Briefly, a quenched sample was loaded onto a Jupiter C4 column (Phenomenex, La Jolla, CA) to purify and desalt SgcC2 species. Mass spectrometric analysis was performed with a custom 8.5-T ESI-FTMS spectrometer equipped with a front-end quadrupole. The sample was introduced using a Nanomate 100 for automated nanospray (Advion Biosciences, Ithaca, NY) and typically 500 ms of ion accumulation/scan was used. The instrument was externally calibrated using ubiquitin (Sigma).

---

| **amino acid** |
| --- |
| **3₃₅** | **3₃₆** | **3₃₇** | **3₃₈** | **3₃₉** | **3₄₀** | **3₄₁** | **3₄₂** | **3₄₃** | **3₄₄** |
| PheA (L-α-Phe) | D | A | W | T | I | A | A | V | C | K |
| SimH (L-α-Tyr) | D | A | T | V | V | A | A | V | C | K |
| NovH (L-α-Tyr) | D | A | L | T | T | G | E | V | V | K |
| TycC (L-α-Tyr) | D | A | S | T | V | A | A | V | C | K |
| BacC (L-α-Tyr) | D | G | T | L | T | A | E | V | A | K |
| SgcC1 (β-Tyr) | D | P | A | Q | L | M | L | I | A | K |
| MdpC1 (β-Tyr) | D | P | C | Q | V | M | V | I | A | K |
| AcvA (Aad) | D | P | R | H | F | V | M | R | A | K |
| MycA (L-α-Glu) | D | P | R | H | S | G | V | V | G | K |
| Tcp12 (dhpg) | D | P | Y | H | G | G | T | L | C | K |

FIGURE 3. Alignment of the nonribosomal codes for SgcC1 with L-α-phenylalanine-specific adenylation domain PheA (P0C061); L-α-tyrosine-specific adenylation domains exemplified by SimH (AAL15600), NovH (AAF67501), TycC (O30409), and BacC (AAC06348); (S)-β-Tyr-specific adenylation domains SgcC1 (AAL06681) and MdpC1;⁶ L-α-amino adipic acid-specific adenylation domains exemplified by AcvA (P07702); L-α-glutamic acid-specific adenylation domains that activate the δ-carboxylic acid exemplified by MycA (Q9RNB3); and dihydroxyphenylglycine-specific adenylation domains exemplified by Tcp12 (Q70AZ6). Given in parentheses in the figure legend are GenBank™ accession numbers. The “invariable” Asp-235 and Lys-517 (PheA numbering) that interact with the α-amino and carboxylate groups, thereby locking the orientation of the side chain of the α-amino acid in the substrate binding pocket of adenylation domains, are highlighted in bold, and the residues subjected to point mutation are boxed. Aad, L-α-amino adipic acid; dhpg, dihydroxyphenylglycine.

---

## RESULTS

### Organization of NRPS within C-1027 Biosynthetic Gene Cluster—The gene cluster for C-1027 biosynthesis contains three ORFs with sequence homology to condensation (SgcC5), adenylation (SgcC1), and PCP (SgcC2) domains found in modular NRPSs. The ORFs are located in close proximity, interrupted only by *sgcC4*, of which the gene product has already been characterized and was identified to catalyze the initial step in the biosynthesis of the β-amino acid moiety of C-1027 by forming (S)-β-tyrosine (Fig. 2A). Therefore, these three ORFs, which constitute a minimal NRPS module, were hypothesized to process the SgcC4 product to maturation and its subsequent incorporation into the C-1027 chromophore (Fig. 2B).

### Substrate Specificity of SgcC1 Using Amino Acid-dependent ATP-PPᵢ Exchange Assay—SgcC1 consists of 881 amino acids, of which ~550 residues of the C terminus are similar to AMP-forming adenylation domains of NRPS. SgcC1 contains the 10 core regions that are conserved in all NRPS adenylation domains (10) and overall has closest homology to TycC3 (38/55% identity/similarity from BL2SEQ analysis), NovH (35/46), and SimH (33/43), all L-α-tyrosine-activating enzymes in tyrocidine, novobiocin, and simocyclinone biosynthesis, respectively (Fig. 3) (17–19). However, SgcC1 was previously shown to activate (S)-β-tyrosine but not α-amino acids (11).

Here, tyrosine analogs that could be potentially formed during the biosynthesis of the (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety were tested to ascertain the timing of the enzymatic step as well as to analyze the general substrate specificity of SgcC1. The data clearly establish that only β-phenyl-

The SgcC1 (S)-β-Tyrosine-specific Adenylation Enzyme

D.

C.

B.

A.

Relative Activity

1.0

0.8

0.6

0.4

0.2

D-α-alanine  
L-α-phenylalanine  
L-α-tyrosine  
cinnamic acid  
p-hydroxycinnamic acid  
3-hydroxy-L-α-tyrosine  
3-chloro-β-tyrosine  
(S)-β-tyrosine  
(R)-β-tyrosine  
3-hydroxy-β-tyrosine  
3-chloro-5-hydroxy-β-tyrosine  

FIGURE 4. Substrate specificity of SgcC1. The data are grouped into four different categories: α-amino acids where \( R_1 = \) respective side chain (A); products of aromatic α-amino acids upon subjecting to ammonia lyase chemistry, cinnamic acid (\( R_2 = \mathrm{H} \)) and p-hydroxycinnamic acid (\( R_2 = \mathrm{OH} \)) (B); α-tyrosine analogs that could be viewed conceptually as possible biosynthetic intermediates, 3-chloro-α-tyrosine (\( R_3 = \mathrm{Cl} \)) and 3-hydroxy-α-tyrosine (\( R_3 = \mathrm{OH} \)) (C); and β-tyrosine analogs that could be viewed conceptually as possible biosynthetic intermediates, β-phenylalanine (\( R_4 = \mathrm{H}, R_5 = \mathrm{H}, R_6 = \mathrm{H} \)), (S)-β-tyrosine and (R)-β-tyrosine (\( R_4 = \mathrm{OH}, R_5 = \mathrm{H}, R_6 = \mathrm{H} \)), 3-hydroxy-β-tyrosine (\( R_4 = \mathrm{OH}, R_5 = \mathrm{OH}, R_6 = \mathrm{H} \)), 3-chloro-β-tyrosine (\( R_4 = \mathrm{OH}, R_5 = \mathrm{Cl}, R_6 = \mathrm{H} \)), and 3-chloro-5-hydroxy-β-tyrosine (\( R_4 = \mathrm{OH}, R_5 = \mathrm{Cl}, R_6 = \mathrm{OH} \)) (D).

Relative Activity

1

0.5

0

4   5   6   7   8   9   10   11

pH

FIGURE 5. Effect of pH on the relative activity of SgcC1 as an adenylation enzyme as determined with (S)-β-tyrosine as a substrate by the ATP-[³²P]PPᵢ exchange assay.

alanine or β-tyrosine analogs are recognized by SgcC1 (Fig. 4). Among the possible substrates that had little or no activity were cinnamic acid and p-hydroxycinnamic acid, products of ammo-

nia lyase chemistry of the corresponding α-amino acids (20), and 3-chloro-α-tyrosine or 3-hydroxy-α-tyrosine, possible intermediates assuming halogenation or hydroxylation, respectively, were the initial steps in biosynthesis of the β-amino acid moiety. The highest activity was observed with (S)-β-tyrosine followed closely by the R enantiomer (95% relative to (S)-β-tyrosine). Other substrates that were recognized include β-phenylalanine (75%), 3-hydroxy-β-tyrosine (62%), 3-chloro-β-tyrosine (51%), and 3-chloro-5-hydroxy-β-tyrosine (9%).

**pH Profile of SgcC1—** The optimal pH for SgcC1 activity was analyzed using amino acid-dependent ATP-PPᵢ exchange assay and (S)-β-tyrosine as the substrate. A three-buffer system was employed to maintain a constant ionic strength (16). SgcC1 was active between pH 5 and 11, and the pH profile exhibited two maxima at pH values 7.5 and 9.0 (Fig. 5). When 100 mM Tris-HCl was substituted, no change in activity was observed (data not shown). Because SgcC4 had a pH optimum of ~9.0, further experiments were performed in 100 mM Tris-HCl at pH 9.0.

**Single-substrate Kinetics of SgcC1—** Kinetic parameters were determined using the five substrates that had significant activity when tested with SgcC1. All of the substrates displayed typical Michaelis-Menten kinetics in the presence of saturating ATP, as exemplified by (S)- or (R)-β-tyrosine (Fig. 6). The best substrate, (S)-β-tyrosine, had a \( k_{\text{cat}} \) of 2.2 ± 0.5 s⁻¹ and \( K_m \) of 3.2 ± 0.6 μM, whereas the other substrates had a similar but slightly lower \( k_{\text{cat}} \) and significantly increased \( K_m \) (Table 1). Overall, SgcC1 catalysis was >50-fold more efficient with (S)-β-tyrosine than any substrate analog tested, excluding the R-enantiomer, which had a 25-fold lower efficiency.

**SgcC1-catalyzed Loading of SgcC2—** The second half-reaction of SgcC1 was monitored by ESI-FTMS following the mass shift observed upon loading the activated β-tyrosine onto holo-SgcC2 PCP to generate a β-tyrosyl-S-SgcC2 thioester. With either (S)- or (R)-β-tyrosine as the substrate, the appropriate mass shift of +162.6 was observed (11). Time course analysis with (S)-β-tyrosine resulted in a rate constant of 2.1 s⁻¹ and with a (R)-β-tyrosine rate constant of 1.9 s⁻¹, both of which are nearly identical to the \( k_{\text{cat}} \) calculated from the first half-reaction as determined by the ATP-[³²P]PPᵢ exchange assays (Table 1).

The SgcC1 (S)-β-Tyrosine-specific Adenylation Enzyme

**FIGURE 6.** Kinetic characterization of SgcC1 as an adenylation enzyme with (S)-β-tyrosine (A) and (R)-β-tyrosine (B) as a substrate by plotting the initial velocities of ATP-[³²P]PP₁ exchange as a function of β-tyrosine concentration.

**TABLE 1**

Michaelis-Menten constants for SgcC1 as an adenylation enzyme toward selected β-amino acids

| Substrate    | $K_m$       | $k_{cat}$      | $k_{cat}/K_m$   | Relative efficiency |
|--------------|-------------|----------------|------------------|---------------------|
|              | $\mu M$     | $s^{-1}$       | $\mu M^{-1}s^{-1}$ |                     |
| (S)-β-Tyr    | 3.2 ± 0.6   | 2.2 ± 0.5      | 0.69             | 1                   |
| (R)-β-Tyr    | 53 ± 12     | 1.5 ± 0.3      | 0.028            | 0.04                |
| 3-OH-β-Tyr   | 81 ± 16     | 1.2 ± 0.1      | 0.015            | 0.02                |
| β-Phe        | $1.9 \times 10^2 \pm 40$ | 1.4 ± 0.4      | 0.0075             | 0.01                |
| 3-Cl-β-Tyr   | $3.0 \times 10^2 \pm 50$ | 1.6 ± 0.3      | 0.0053             | 0.008               |

**SgcC1 Mutation Analysis—**Adenylation domains have been proposed to extend ~550 amino acids within an NRPS module (10). Therefore, the wild-type SgcC1 protein (881-amino-acid residues) was truncated by deleting 338 amino acids from its N terminus to encompass only the putative AMP-forming domain (C terminus). Of the constructs prepared, only the dual His₆-tagged version was slightly soluble, but no activity was observed with this construct.

A P571A mutant of SgcC1 was prepared to examine possible changes in substrate specificity and/or catalysis. This residue neighbors the invariant aspartate residue Asp-570, the carboxylate side chain of which interacts with the α-amino group to lock the orientation of the L-α-amino acid into the substrate binding pocket (Fig. 3). The mutation abolished activity with all substrates except (S)-β-tyrosine. Kinetic analysis for SgcC1(P571A) yielded a $K_m$ of 13 ± 4 μM and a $k_{cat}$ of 0.07 ± 0.02 s⁻¹, a 142-fold loss in catalytic efficiency when compared with the wild-type SgcC1 (Table 2).

**TABLE 2**

Michaelis-Menten constants for SgcC1(P571A) as an adenylation enzyme toward (S)-β-tyrosine and comparison with that of the wild-type SgcC1 enzyme

| Enzyme         | $K_m$       | $k_{cat}$      | $k_{cat}/K_m$   | Relative efficiency |
|----------------|-------------|----------------|------------------|---------------------|
|                | $\mu M$     | $s^{-1}$       | $\mu M^{-1}s^{-1}$ |                     |
| SgcCl (wild-type) | 3.2 ± 0.6   | 2.2 ± 0.5      | 0.69             | 1                   |
| SgcCl (P576A)  | 13 ± 4      | $7.0 \times 10^{-2} \pm 0.02$ | 0.0054             | 0.007               |

**DISCUSSION**

Adenylation domains of NRPS catalyze the activation, at the expense of ATP, and subsequent loading of the activated amino acids or carboxylic acids to a PCP domain to form an aminoacyl-S-PCP thioester. SgcC1 has significant sequence homology to numerous adenylation domains of NRPS, and as a result, has been hypothesized to be involved in the formation, activation, and incorporation of the (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety into C-1027, although the precise timing of the steps cannot be predicted *a priori* (Fig. 2). Sequence analysis predicted L-α-tyrosine as the probable substrate for SgcC1, as deduced from its closest sequence identity to domains that activate L-α-tyrosine, such as TycC3, NovH, and SimH, and comparisons to the nonribosomal codes of NRPS suggested that L-α-tyrosine was the most likely substrate for SgcC1 (Fig. 3) (12, 13). However, our previous results with SgcC4 preferring L-α-tyrosine were inconsistent with this proposal (11), and therefore, a full biochemical characterization of SgcC1 was undertaken.

As previously reported, when L-α-tyrosine was tested as a substrate, SgcC1 was only active in the presence of SgcC4, which converts L-α-tyrosine to (S)-β-tyrosine *in situ* (11). In contrast, authentic (S)-β-tyrosine, the established product of SgcC4 (9), was recognized and loaded to SgcC2 to form an (S)-β-aminoacyl-S-SgcC2 thioester intermediate. When a full spectrum of substrates was tested, including substituted α- and β-phenylalanine analogs that could hypothetically be generated during biosynthesis of the (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety of C-1027, only β-amino acids were activated, all with relatively high rates under the conditions employed, prompting a thorough kinetic analysis. Although the five aromatic β-amino acids examined were all recognized and activated by SgcC1, (S)-β-tyrosine was clearly the preferred substrate by >50-fold. The enantiomer (R)-β-tyrosine is also a substrate for SgcC1 but is 25-fold less efficient, and this nonspecific activity is likely to be further compensated by the stereospecificities of SgcC4 and downstream enzymes such as SgcC5. Interestingly, condensation domains have recently been shown to contribute to substrate specificity by discriminating between isomers (21).

Rational engineering of the adenylation domain specificity without adverse effect on the structural integrity of the NRPS biosynthetic machinery could be viewed as one of the most powerful combinatorial biosynthesis strategies for natural product diversity. Central to this strategy is the development of the general rule, known as nonribosomal codes, for substrate recognition in the adenylation domain of an NRPS, which has provided a molecular basis for engineered biosynthesis of novel peptides (12, 13, 22). Based on the structure of PheA (23), 10

amino acid residues that line the amino acid binding pocket were identified as the general specificity-conferring code of adenylatation domains for NRPSs (Fig. 3). According to this model, the Asp residue at position 235 and the Lys residue at 517 interact with the α-amino and the carboxylate groups, respectively, to lock orientation of the L-α-amino acid upon activation. This configuration projects the side chain of the amino acid into the binding pocket, comprising the 10 amino acid residues within a radius of ~5.5 Å, which in the adenylatation domain could then be used to predict the nature of the side chain, and thereby, the amino acid specificity. This model is based on the structural conservation of the binding pocket as reflected by the relatively high sequence similarity of adenylatation domains and has been supported by predicting the adenylatation domain substrate specificity according to the nonribosomal code followed by experimental verification (12, 13) and re-engineering to alter adenylatation domain specificity followed by isolation of the resultant products with the targeted amino acid incorporated (22).

An important feature of NRPSs is their ability to incorporate nonproteinogenic amino acids, such as β-amino acids, and their presence in the resultant metabolites contributes critically to their biological activities and therapeutic value. Because the current model is based on the structure of PheA, the prediction model is severely limited to L-α-amino acids and cannot be used to predict adenylatation domain specificity for β-amino acids or several nonproteinogenic substrates, given the key interaction between the α-amino group and Asp-235, which is fundamental to this model, as this interaction locks the configuration of the amino acid. In β-amino acids, this fundamental interaction is no longer applicable, and thus, the predictive model breaks down. The β-amino acid specificity of SgcC1 reported here unveiled the limitation of the current model, providing an outstanding opportunity to decipher the nonribosomal codes for β-amino acids.

Of the common amino acids, Pro has the uncanny ability to modulate stability, structure, and function of a given protein (24). The presence of Pro generates kinks in α-helix structure, and in particular, the torsional angle inherent in Pro is highly coupled to the preceding residue in the case of SgcC1, the invariant Asp residue, the carboxylate side chain of which is critical in developing the current nonribosomal codes for α-amino acids (12, 13). A survey of L-tyrosine or L-phenylalanine-specific adenylatation domains showed that an Ala (or Gly in a couple of cases) occupies this position, and 32 of 1250+ adenylatation domains (25) contain a Pro (Fig. 3). Of the 32, 17 have known specificity,⁵ and in all of these cases, only three substrates are utilized, none of which are proteinogenic α-amino acids or utilized to form an amide bond following the classical NRPS paradigm. AcvA homologs activate the δ-carboxylic acid of L-α-aminoadipic acid for penicillin biosynthesis (26), MycE homologs activate the δ-carboxylic acid of L-α-glutamic acid

⁵ Known specificity refers to an adenylatation domain “based on experimental evidence by an ATP-PPi-exchange reaction or, when the specificity was inferred by the co-linearity rule, based on the ordered composition of the peptide product, and that the inference was confirmed by an unambiguous match with a known ‘specificity-conferring code’ of another adenylatation domain” (25).

for microcystin biosynthesis (27), and CepC homologs activate dihydroxyphenylglycine in vancomycin and teicoplanin biosynthesis (28) (Fig. 3). In addition, the enediyne maduropeptin contains a similar aromatic moiety to C-1027 and likewise the gene cluster has the entire set of SgcC-C5 homologs. Its adenylatation domain (MdpC1) is proposed to catalyze the identical reaction to SgcC1 or minimally is unable to process α-amino acids and also contains a Pro adjacent to the invariant Asp.⁶ Assuming a common fold and mechanism for all adenylatation domains, which has been observed from structural analysis with three sequence-unrelated enzymes, firefly luciferase (29), PheA (23), and DhbA (30), P571 of SgcC1 was initially hypothesized to aid in repositioning the preceding Asp-570 to appropriately interact with the β-amino group, which could be envisaged to produce a similar effect for the other respective adenylatation domains. However, tests of the P571A mutant showed that activity was completely abolished, except with (S)-β-tyrosine, and in this case with a 142-fold drop in catalytic efficiency mostly manifested from a decrease in $k_{\text{cat}}$ (Table 2). Structural analysis of SgcC1, which is currently in progress, will undoubtedly reveal the particular features of SgcC1 that modulate catalysis and specificity for a β-amino acid, and subsequent structural comparisons to other adenylatation domains will ultimately enable us to expand the nonribosomal codes of NRPS to cover nonstandard amino acids such as β-amino acids.

The C-1027 gene cluster features many unusual features, including the occurrence of three ORFs that constitute a minimal NRPS module. We foresee this as an excellent opportunity to study the features inherent in NRPS domains responsible for high fidelity molecular recognition and catalysis while maintaining the native protein structural integrity (*i.e.* naturally occurring discrete proteins *versus* artificially excised domains out of the context of the native, giant multidomain NRPS enzymes). The fact that SgcC1 is rather promiscuous for β-aromatic amino acids and that genetic engineering has proven that downstream processes are quite forgiving (5, 11) also presents an opportunity to produce novel C-1027 analogs by precursor-directed biosynthesis approach.

Acknowledgments—We thank Dr. Y. Li, Institute of Medicinal Biotechnology, Chinese Academy of Medical Sciences, Beijing, China for the wild-type *S. globisporus* strain, and the Analytical Instrumentation Center of the School of Pharmacy, University of Wisconsin-Madison for support in obtaining MS and NMR data.

---

**REFERENCES**

1. Hu, J., Xue, Y. C., Xie, M. Y., Zhang, R., Otani, T., Minami, Y., Yamada, Y., and Marunaka, T. (1988) *J. Antibiot.* (Tokyo) **41**, 1575–1579
2. Otani, T., Minami, Y., Marunaka, T., Zhang, R., and Xie, M. Y. (1988) *J. Antibiot.* (Tokyo) **41**, 1580–1585
3. Sugimoto, Y., Otani, T., Oie, S., Wierzba, K., and Yamada, Y. (1990) *J. Antibiot.* (Tokyo) **43**, 417–421
4. Wang, X., and Xie, H. (1999) *Drugs Future* **24**, 847–852
5. Liu, W., Chrisenson, S. D., Standage, S., and Shen, B. (2002) *Science* **297**, 1170–1173
6. Ahlert, J., Shepard, E., Lomovskaya, N., Zazopoulos, E., Staffa, A., Bachmann, B. O., Huang, K., Fonstein, L., Czisny, A., Whitwam, R. E., Farnet,
⁶ B. Shen, unpublished data.

The SgcC1 (S)-β-Tyrosine-specific Adenylation Enzyme

7. Liu, W., Nonaka, K., Nie, L., Zhang, J., Christenson, S. D., Bae, J., Van Lanen, S. G., Zazopoulos, E., Farnet, C. M., Yang, C. F., and Shen, B. (2005) Chem. Biol. 12, 1–10
8. Murrell, J. M., Liu, W., and Shen, B. (2004) J. Nat. Prod. 67, 206–213
9. Christenson, S. D., Wu, W., Spies, M. A., Shen, B., and Toney, M. D. (2003) Biochemistry 42, 12708–12718
10. Schwarzer, D., Finking, R., and Marahiel, M. A. (2003) Nat. Prod. Rep. 20, 275–287
11. Van Lanen, S. G., Dorrestein, P. C., Christenson, S. D., Liu, W., Ju, J., Kelleher, N. L., and Shen, B. (2005) J. Am. Chem. Soc. 127, 11594–11595
12. Stachelhaus, T., Mootz, H. D., and Marahiel, M. A. (1999) Chem. Biol. 6, 493–505
13. Challis, G. L., Ravel, J., and Townsend, C. A. (2000) Chem. Biol. 7, 211–224
14. Tan, C. Y. K., and Weaver, D. F. (2002) Tetrahedron 58, 7449–7461
15. Hua, D. H., Huang, X., Chen, Y., Battina, S. K., Tamura, M. S., Noh, K., Koo, S. I., Namatame, I., Tomoda, H., Perchellet, E. M., and Perchellet, J. P. (2004) J. Org. Chem. 69, 6065–6078
16. Ellis, K. J., and Morrison, J. F. (1982) Methods Enzymol. 87, 405–426
17. Mootz, H. D., and Marahiel, M. A. (1997) J. Bacteriol. 179, 6843–6850
18. Chen, H., and Walsh, C. T. (2001) Chem. Biol. 8, 301–312
19. Galm, U., Schimana, J., Fiedler, H., Schmidt, J., Li, S., and Heide, L. (2002) Arch. Microbiol. 178, 102–114
20. Poppe, L. (2001) Curr. Opin. Struct. Biol. 4, 902–911
21. Belshaw, P. J., Walsh, C. T., and Stachelhaus, T. (1999) Science 284, 486–489
22. Stachelhaus, T., Schneider, A., and Marahiel, M. A. (1995) Science 269, 69–72
23. Conti, E., Stachelhaus, T., Marahiel, M. A., and Brick, P. (1997) EMBO J. 16, 4174–4183
24. Reiersen, H., and Rees, A. R. (2001) Trends Biochem. Sci. 26, 679–683
25. Rausch, C., Weber, T., Kohlbacher, O., Wohlleben, W., and Huson, D. H. (2005) Nucleic Acids Res. 33, 5799–5808
26. Zhang, J., Wolfe, S., and Demain, A. L. (1992) Biochem. J. 283, 691–698
27. Tillett, D., Dittmann, E., Erhard, M., von Döhlen, H., Börner, T., and Neilan, B. A. (2000) Chem. Biol. 7, 753–764
28. van Wageningen, A. M. A., Kirkpatrick, P. N., Williams, D. H., Harris, B. R., Kershaw, J. K., Lennard, N. J., Jones, M., Jones, S. J. M., and Solenberg, P. J. (1998) Chem. Biol. 5, 155–162
29. Conti, E., Franks, N. P., and Brick, P. (1996) Structure (Lond.) 4, 287–298
30. May, J. J., Kebler, N., Marahiel, M. A., and Stubbs, M. T. (2000) Proc. Natl. Acad. Sci. U. S. A. 97, 2509–2514

Enzyme Catalysis and Regulation:
Substrate Specificity of the Adenylation
Enzyme SgcC1 Involved in the Biosynthesis
of the Enediyne Antitumor Antibiotic
C-1027

Steven G. Van Lanen, Shuangjun Lin, Pieter
C. Dorrestein, Neil L. Kelleher and Ben Shen
J. Biol. Chem. 2006, 281:29633-29640.
doi: 10.1074/jbc.M605887200 originally published online August 3, 2006

Access the most updated version of this article at doi: [10.1074/jbc.M605887200](http://dx.doi.org/10.1074/jbc.M605887200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](http://www.jbc.org/site/misc/affinity-sites.xhtml).

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 30 references, 8 of which can be accessed free at
[http://www.jbc.org/content/281/40/29633.full.html#ref-list-1](http://www.jbc.org/content/281/40/29633.full.html#ref-list-1)
