
# Molecular and functional analysis of nicotinate catabolism in *Eubacterium barkeri*

Ashraf Alhapel, Daniel J. Darley, Nadine Wagener, Elke Eckel, Nora Elsner, and Antonio J. Pierik*

Laboratorium für Mikrobielle Biochemie, Fachbereich Biologie, Philipps Universität, D-35032 Marburg, Germany

Edited by Perry A. Frey, University of Wisconsin, Madison, WI, and approved June 29, 2006 (received for review March 1, 2006)

The anaerobic soil bacterium *Eubacterium barkeri* catabolizes nicotinate to pyruvate and propionate via a unique fermentation. A full molecular characterization of nicotinate fermentation in this organism was accomplished by the following results: (*i*) A 23.2-kb DNA segment with a gene cluster encoding all nine enzymes was cloned and sequenced, (*ii*) two chiral intermediates were discovered, and (*iii*) three enzymes were found, completing the hitherto unknown part of the pathway. Nicotinate dehydrogenase, a (non-selenocysteine) selenium-containing four-subunit enzyme, is encoded by *ndhF* (FAD subunit), *ndhS* (2 x [2Fe-2S] subunit), and by the *ndhL/ndhM* genes. In contrast to all enzymes of the xanthine dehydrogenase family, the latter two encode a two-subunit molybdopterin protein. The 6-hydroxynicotinate reductase, catalyzing reduction of 6-hydroxynicotinate to 1,4,5,6-tetrahydro-6-oxonicotinate, was purified and shown to contain a covalently bound flavin cofactor, one [2Fe-2S]$^{2+/1+}$ and two [4Fe-4S]$^{2+/1+}$ clusters. Enamidase, a bifunctional Fe-Zn enzyme belonging to the amidohydrolase family, mediates hydrolysis of 1,4,5,6-tetrahydro-6-oxonicotinate to ammonia and (S)-2-formylglutamate. NADH-dependent reduction of the latter to (S)-2-(hydroxymethyl)glutarate is catalyzed by a member of the 3-hydroxyisobutyrate/phosphogluconate dehydrogenase family. A [4Fe-4S]-containing serine dehydratase-like enzyme is predicted to form 2-methylene-glutarate. After the action of the coenzyme B$_{12}$-dependent 2-methyleneglutamate mutase and 3-methylitaconate isomerase, an aconitase and isocitrate lyase family pair of enzymes, (2R,3S)-dimethylmalate dehydratase and lyase, completes the pathway. Genes corresponding to the first three enzymes of the *E. barkeri* nicotinate catabolism were identified in nine Proteobacteria.

Nicotinate (niacin, vitamin B3) is an important constituent of all living cells in the form of nicotinamide adenine dinucleotide (phosphate). Cells contain NAD(P) concentrations of 0.1–1 mM (1), which supply nicotinate as a nitrogen, carbon, and energy source to a diverse set of dedicated nicotinate-catabolizing microorganisms (2). Nicotinate catabolism in all organisms starts with hydroxylation to 6-hydroxynicotinate by the well characterized and industrially used enzyme nicotinate dehydrogenase (3). Further catabolism depends on the availability of oxygen in the environment. In several aerobic organisms, such as Pseudomonads, 6-hydroxynicotinate is oxidatively decarboxylated to 2,5-dihydroxypyridine (4) or, in the unique case of *Bacillus niacini*, subjected to a second hydroxylation yielding 2,6-dihydroxynicotinate (5). Under microaerobic (6) or fermentative conditions (7), ferredoxin-dependent reduction to 1,4,5,6-tetrahydro-6-oxonicotinate (THON) is observed.

Work by Harary (8) and Stadtman (9) identified an anaerobic soil bacterium now called *Eubacterium barkeri* (order Clostridiales) that fermented nicotinate according to the following equation:

Nicotinate + 4 H$_2$O → Propionate  
+ Acetate + CO$_2$ + NH$_4^+$ (1 ATP/nicotinate)

Cell extracts incubated with radioactively labeled nicotinate allowed a number of unusual intermediates to be identified (10, 11), and it became clear that the pathway was remarkably complex (see Fig. 1). Based on the identified intermediates, several anticipated enzymes were purified and characterized: nicotinate dehydrogenase (12), 6-hydroxynicotinate reductase (7), 2-methyleneglutamate mutase, and 3-methylitaconate isomerase (13, 14). These findings outlined the nicotinate fermentation pathway and placed the identified intermediates in an enzymatic framework.

The nicotinate dehydrogenase contains [2Fe-2S] clusters (15), FAD and molybdopterin cytosine dinucleotide (16), and has an unusual subunit composition [50, 37, 33, and 23 kDa (17)]. It has labile (nonselenocysteine) selenium (18) also identified in purine dehydrogenase from *Clostridium purinolyticum* and xanthine dehydrogenases from *C. purinolyticum* (19), *Clostridium acidurici* (20), and *E. barkeri* (21). The selenium coordinates molybdenum (15) and is thought to be a selenido equivalent of the cyanolyzable sulfido-ligand (22) in the xanthine dehydrogenase family of enzymes. Studies in Marburg (23, 24) focused on the adenosylcobalamin-dependent carbon skeleton-rearranging enzyme 2-methyleneglutamate mutase and 3-methylitaconate isomerase. Genes encoding these two enzymes were cloned from a 3.7-kbp PstI-DNA fragment (24). The last two steps of the pathway have been characterized through partial purification of a labile (2R,3S)-dimethylmalate dehydratase and (2R,3S)-dimethylmalate lyase, and the stereochemical course was determined (25–28).

Despite the work described earlier, our understanding of nicotinate fermentation is still incomplete. Previously, 6-hydroxynicotinate reductase was reported to be an [Fe-S] protein, but no molecular characterization was performed. Although enzyme-catalyzed THON hydrolysis was detected (29), conversion of THON to 2-methyleneglutamate was not investigated. Here we report a full characterization of 6-hydroxynicotinate reductase and identify two nicotinate fermentation enzymes: a bifunctional hydrolase that converts THON to 2-formylglutamate (called enamidase) and 2-(hydroxymethyl)glutarate dehydrogenase. Evidence is presented for the intermediacy of chiral 2-formylglutamate and 2-(hydroxymethyl)glutarate. The nucleotide sequence of a 23.2-kbp chromosomal DNA fragment of *E. barkeri* harboring all genes for the nicotinate fermentation enzymes has been determined. Gene clusters associated with nicotinate catabolism in other bacteria were identified with database searches.

## Results and Discussion

### The *E. barkeri* Nicotinate Gene Cluster

Chromosomal DNA fragments of *E. barkeri* were cloned by using λ-ZAP-Express phage libraries (30) and Southern blot hybridization with digoxygenin.

---

Conflict of interest statement: No conflicts declared.

This paper was submitted directly (Track II) to the PNAS office.

Abbreviations: NCP, nicotinate-catabolizing Proteobacteria; THON, 1,4,5,6-tetrahydro-6-oxonicotinate.

Data deposition: The sequence reported in this paper has been deposited in the GenBank database (accession no. DQ310789).

*To whom correspondence should be addressed. E-mail: pierik@staff.uni-marburg.de.*

© 2006 by The National Academy of Sciences of the USA

nicotinate 6-hydroxynicotinate tautomers 1,4,5,6-tetrahydro-6-oxonicotinate (S)-2-formylglutamate (S)-2-(hydroxymethyl) glutarate

![image](https://i.imgur.com/your_image_url.png)

Fig. 1. Nicotinate fermentation in *E. barkeri*. ①, nicotinate dehydrogenase; ②, 6-hydroxynicotinate reductase; ③, enamidase; ④, 2-(hydroxymethyl)glutarate dehydrogenase; ⑤, 2-(hydroxymethyl)glutarate dehydratase; ⑥, 2-methyleneglutamate mutase; ⑦, (*R*)-3-methylitaconate isomerase; ⑧, (*2R*,*3S*)-dimethylmalate dehydratase; ⑨, (*2R*,*3S*)-dimethylmalate lyase.

labeled probes derived from the known PstI fragment (24) (Fig. 2A). In conjunction with direct genomic sequencing (31) and SeeGene DNA walking (32), a contig of 23,202 bp (52.8% GC) was assembled. Identification of genes and startcodons used for their translational initiation was unambiguous: Near consensus GGAGG Shine–Dalgarno sequences were present at 7 ± 3 nucleotides from the startcodons (18 × ATG, 2 × GTG, and 1 × TTG). Predicted and experimental N-terminal amino acid sequences of 6-hydroxynicotinate reductase and enamidase reported here were in full agreement, as were those of the nicotinate dehydrogenase subunits (17), 2-methyleneglutamate mutase and methylitaconate isomerase (24).

An overview of the *E. barkeri* nicotinate fermentation gene cluster is shown in Fig. 2A. The central region harbors 17 convergently transcribed genes (*hnr* to *dmdB*, nucleotides 3,069 to 21,980), which are overlapping or have short intergenic regions, typical for gene clusters associated with bacterial catabolic pathways. Ten genes encode seven structural enzymes of nicotinate fermentation. Three genes can be assigned to two structural enzymes based on amino acid sequence identity with enzyme classes of known function. One gene encodes a postulated 2-methyleneglutamate mutase repair enzyme (*mgmL*), and two genes are tightly linked to *hnr* genes in nicotinate gene clusters of Proteobacteria (*orfC* and *orfE*; see Fig. 2). The function of *orfD*, encoding a putative transmembrane protein, is unclear. Downstream of *dmdB*, two further genes (*orfFG*; data not shown) with an unknown relation to nicotinate fermentation are present. Upstream, the nicotinate catabolon is flanked by a divergently transcribed gene encoding a LysR-type regulator (designated as *nicR*) and two genes of unclear association (*orfAB*; data not shown). Such LysR-type regulators have been observed downstream of numerous gene clusters associated with degradation of aromatic compounds (33). The chemical inducer could be 6-hydroxynicotinate, which is known to accumulate early in the growth phase (3), similar to transcriptional activation by pathway intermediates in aromatic degradation.

**Nicotinate Dehydrogenase.** For the first time to our knowledge, the complete primary sequence of a nicotinate dehydrogenase has been determined. The *ndhF*, *ndhS*, *ndhL*, and *ndhM* genes encode the 33-, 23-, 50-, and 37-kDa subunits of the *E. barkeri* nicotinate dehydrogenase based on the known N-terminal sequences (17). In agreement with the presence of FAD and two [2Fe-2S] clusters (16, 17), high sequence identities of NdhS and NdhF were found with the 2×[2Fe-2S]- and FAD-containing subunits/domains of xanthine dehydrogenases, respectively. NdhF lacks the insert with [4Fe-4S] cluster coordinating cysteines observed in 4-hydroxybenzoyl-CoA reductase (34). The 17-bp overlapping *ndhL* and *ndhM* genes formed two separate transcriptional units in different frames, with *ndhM* preceded by a Shine–Dalgarno sequence. *NdhL* is terminated by a TAA rather than a potentially selenocysteine-encoding TGA codon. These two subunits correspond to the two molybdopterin domains of the ≈85-kDa subunit of xanthine dehydrogenase-like enzymes (34, 35). Only three other two-subunit proteins of this type could be identified by literature and database searches with NdhLM: the (nonselenocysteine) selenium-containing purine dehydrogenase from *C. purinolyticum* (54 and 42 kDa) (19) and the xanthine dehydrogenase-like proteins both from *Mesorhizobium loti* (*mlr1703/mlr1704*) and encoded by an environmental sequence (AACY01708552). A two-subunit nature is not characteristic for this special class of enzymes because (nonseleno-

![image](https://i.imgur.com/your_image_url.png)

Fig. 2. Nicotinate catabolism gene clusters. (A) The nicotinate fermentation gene cluster of *E. barkeri* with the PstI fragment (24), indicated by a dashed line and BamHI, BglII, EcoRI, and PstI restriction sites (B, Bg, E, and P), is shown. Genes associated with conversion of 2-formylglutamate to propionate and pyruvate are in gray. (B) Gene clusters associated with nicotinate catabolism via THON (descriptions and accession codes in Supporting Text). (C) Key: Genes with unclear association to nicotinate catabolism are colorless.

acid sequence identity of 35–38% and revealed that the *E. barkeri* 6-hydroxynicotinate reductase is composed of four parts. It has an N-terminal extension (amino acids 1–53) with two CXXCXX-CXXXC sequence motifs typical for ferredoxins binding two \[4Fe-4S\]$^{2+/1+}$ clusters (absent in the homologs) and a CXXCPXXCX$_7$GACXRY motif (amino acids 54–103) present in the N-terminal part of the homologs. These regions are linked by an unconserved section (amino acids 104–118) to a main domain (amino acids 119–499) exhibiting 41–45% sequence identity with the homologs. This modularity is reminiscent of hydrogenase in which an N-terminal ferredoxin-like domain functions as electron donor/acceptor, and a median cluster wires electrons to the active site at the heart of the protein. Based on EPR spectroscopic evidence for a \[2Fe-2S\] cluster magnetically coupled to another \[Fe-S\] cluster, we propose that the CXXCPXXCX$_7$GACXRY motif forms the binding site for the \[2Fe-2S\] cluster, which is at a distance of $<$12 Å from two ferredoxin-like \[4Fe-4S\]$^{2+/1+}$ clusters at the N-terminus. This arrangement would define an electron flow from the physiological electron donor ferredoxin, reduced by pyruvate:ferredoxin oxidoreductase, to the N-terminal 2 × \[4Fe-4S\]$^{2+/1+}$ clusters followed by single electron transfers via the median \[2Fe-2S\] cluster to the covalently bound flavin in the active site. Heterologous expression in *Escherichia coli* has not been successful and might require coexpression of *orfC* and/or *orfE*, which lie immediately downstream of the *hnr* homologous genes.

Enamidase, a Bifunctional Enzyme Belonging to the Amidohydrolase Family. Previous studies indicated that nicotinate-grown *E. barkeri* extracts catalyzed the hydrolysis of THON to ammonia and a labile compound tentatively identified as 2-formylglutarate (29). This compound was transparent at the absorbance maximum of THON ($\varepsilon_{273~\text{nm}} = 11.2~\text{mM}^{-1}\text{cm}^{-1}$) and yielded the 2,4-dinitrophenylhydrazone of glutaric semialdehyde, thought to result from decarboxylation. Based on these observations, we decided to isolate the hydrolytic enzyme that will be called enamidase. This name reflects the enamide moiety of THON subject to amidase action.

Instead of discontinuous assays using 2,4-dinitrophenylhydrazine, a continuous UV-spectrophotometric method was developed. Hydrolysis of THON was followed in cuvettes with a pathlength of 1–2 mm at 307 nm ($\varepsilon = 1.1~\text{mM}^{-1}\text{cm}^{-1}$). THON was obtained either by enzyme-catalyzed reduction of 6-hydroxynicotinate or chemical synthesis by means of a modified literature procedure (29). Both methods gave identical material as judged by activity measurement, UV-, $^{13}$C- and $^1$H-NMR spectroscopy (data not shown). A four-step purification of enamidase from cell extracts of *E. barkeri* grown on nicotinate gave homogeneous preparations exhibiting a single 40-kDa band on SDS/PAGE (Fig. 4A). Gel filtration showed that the protein is homotetrameric. Enamidase contains 1.0 Fe and ≈0.6 Zn per subunit and exhibits weak visible absorbance bands from ferric iron that bleach upon dithionite reduction and slightly increase upon ferricyanide oxidation. The latter treatments did not significantly change the kinetic parameters of enamidase (57 units/mg with a $K_\text{m}$ for THON of 5 mM). Purified N-terminally Streptagged enamidase had identical properties.

$^1$H-NMR of crude reaction mixtures obtained upon incubation of THON with enamidase showed aldehyde peaks at 9.3 and 9.7 ppm. However, isolation and purification of 2-formylglutarate proved impossible because of decarboxylation. Enamidase not only catalyzed THON hydrolysis but also the reverse reaction after shifting the equilibrium by $\text{NH}_4^+$ addition (Fig. 4B). The UV-difference spectrum of the compound formed was identical to THON. This reverse reaction was not spontaneous, because no THON formation was detected in the absence of enamidase (data not shown).

The N-terminal sequence of the native enamidase and those

Alhapel et al.
PNAS | August 15, 2006 | vol. 103 | no. 33 | 12343
(data not shown). The $K_{\mathrm{m}}$ for racemic 2-(hydroxymethyl)glutarate was 1.1 mM (at 5 mM NAD$^{+}$) and that for NAD$^{+}$ was 0.1 mM [at 10 mM racemic 2-(hydroxymethyl)glutarate]. Because no activity was observed with NADP$^{+}$, it is unlikely that the NADPH produced by nicotinate dehydrogenase is the substrate for the Hgd-catalyzed reduction of 2-formylglutarate. An NADPH:NAD$^{+}$ transhydrogenase could supply an additional source of energy via the difference in electrochemical potentials of nicotinate/6-hydroxynicotinate [$<-380 \mathrm{mV}(12)$] vs. 2-(hydroxymethyl)-/2-formyl-glutarate couples (estimated to be $-200 \mathrm{mV}$).

Upstream of $n d h F S L M$, a gene was identified which encoded a protein (30,874 Da) with 37% and 38% amino acid sequence identity to *Pseudomonas aeruginosa* 3-hydroxyisobutyrate dehydrogenase and *Escherichia coli* tartrate reductase, respectively. Comparison of 2-(hydroxymethyl)glutarate with substrates of these and other enzymes of the 3-hydroxyisobutyrate/phosphogluconate dehydrogenase family (39) revealed a common substituted 3-hydroxypropionate moiety. With the anticipation that Hgd was encoded by this gene, we purified the corresponding N-terminally Streptagged protein after heterologous expression in *Escherichia coli*. The tetrameric protein ($4 \times 32.5 \mathrm{kDa}$, including tag) had a specific activity and $K_{\mathrm{m}}$ values similar to those of partially purified wild-type enzyme.

### Conversion of 2-(Hydroxymethyl)glutarate to 2-Methyleneglutarate

Because the substrates for Hgd and 2-methyleneglutarate mutase are free carboxylates, either transient formation of CoA-esters would have to occur or a [4Fe-4S]-cluster containing dehydratase could be involved. A gene in the 23.2-kb DNA fragment encodes a protein of 471 amino acids with similarity to both $\alpha$- and $\beta$-subunits of labile [4Fe-4S]-containing bacterial serine dehydratases (40). The N-terminal amino acid sequence showed 31% identity with the *Lactobacillus johnsonii* serine dehydratase $\alpha$-subunit (LJ1328) and the C-terminal part had 28% identity with the corresponding $\beta$-subunit (LJ1329). Three conserved cysteines (171, 214, and 224) match those proposed to coordinate the [4Fe-4S] cluster in the serine dehydratase $\alpha$-subunit (40). The presence of HOCH$_{2}$-CH-CO$_{2}^{-}$ substructures both in 2-(hydroxymethyl)glutarate and L-serine lends support to the argument that this gene (*hmd*; Fig. 2A) encodes 2-(hydroxymethyl)glutarate dehydratase. Purification of N-terminally Streptagged Hmd in an active form has not thus far been successful, presumably because of the same [4Fe-4S] cluster lability known for L-serine dehydratase (40).

### Chirality of 2-Formylglutarate and 2-(Hydroxymethyl)glutarate

Previous studies on *E. barkeri* nicotinate fermentation identified $(R)$-3-methylitaconate (23) and $(2 R, 3 S)$-dimethylmalate (26) as chiral intermediates, but chirality of 2-formyl- or 2-(hydroxymethyl)-glutarate has not been considered. Because an enantioselective synthesis of the latter substrate has not yet been accomplished, serine and 3-hydroxyisobutyrate enantiomers were used to determine the stereoselectivity of Hgd. The dehydrogenase catalyzed oxidation of $(S)$-serine (0.3 units/mg) and $(S)$-3-hydroxyisobutyrate (0.1 units/mg) but not the corresponding $(R)$-enantiomers (<0.005 units/mg). With the assumption that the HOCH$_{2}$-CH-CO$_{2}^{-}$ moieties of the probes and

Fig. 5. Proposed intermediates of the enamidase catalyzed THON hydrolysis.
Fig. 6. Dehydratase and lyase pairs acting on the (2R,3S)-3-substituted malate moiety.

2-(hydroxymethyl)glutarate are bound in the active site in a comparable manner; it is reasonable to conclude that (S)-2-(hydroxymethyl)glutarate and thus (S)-2-formylglutarate are the physiological intermediates.

The stereospecificity of Hgd was used to determine the chirality of the 2-formylglutarate formed by enamide action. Spontaneous hydrolysis of 2-(enamine)glutarate would lead to racemization. If, however, 2-(enamine)glutarate would not be released from the binuclear center of enamide, a proton could stereospecifically be added to C-2, forming a chiral iminium intermediate (Fig. 5). After hydration to 2-(carbinolamine)glutarate, loss of ammonia would yield (S)-2-formylglutarate, the substrate for Hgd. When THON was incubated with enamide initially 0.98 equivalents of (S)-2-formylglutarate were recovered per mole of THON hydrolyzed, which progressively decreased to 0.74 over 80 min (Fig. 4C). This loss probably results from decarboxylation of (S)-2-formylglutarate and racemization. We note that the (S)-2-formylglutarate determination was carried out quickly (20 s) to avoid overestimation of (S)-2-formylglutarate by racemization during the Hgd assay. After removal of enamide by membrane filtration, (S)-2-formylglutarate solutions could be used to determine a \( K_m \) of 0.061 mM (at 0.25 mM NADH) and 0.016 mM for NADH [at 1.2 mM (S)-2-formylglutarate].

(2R,3S)-Dimethylmalate Dehydratase and Lyase. Partial enrichment of the Fe²⁺-dependent and oxygen-sensitive (2R,3S)-dimethylmalate dehydratase from *E. barkeri* has been reported (28). Analysis of the cloned *E. barkeri* sequence identified two genes (*dmdAB*) encoding proteins with amino acid sequence identities of 70% to archaeal proteins of the aconitase family (41) and 30% with biochemically characterized eubacterial isopropylmalate isomerase subunits. Inclusion of (2R,3S)-dimethylmalate dehydratase in the aconitase family is further supported by the presence of conserved cysteine residues at position 301, 361, and 364 in DmdA, typical for [4Fe-4S] cluster coordination in other members.

(2R,3S)-Dimethylmalate lyase is encoded by the *dml* gene that has been expressed in *Escherichia coli*. The purified enzyme is a homotetramer (4 × 31.4 kDa) and has the same Mg²⁺ dependence and catalytic properties as from wild type (25). A primary sequence identity of 36% with *Escherichia coli* 2-methylisocitrate lyase and the conserved KKCGH active site motif (42) defined Dml as a member of the isocitrate lyase family.

Dehydratases and lyases acting on (2R,3S)-3-substituted malate substrates (Fig. 6) can be found in many pathways: (methyl)citrate and glyoxylate cycle, lysine biosynthesis via homoaconitate, coenzyme B biosynthesis, and, as shown here, in nicotinate fermentation. Primary sequence and structural conservation in the aconitase and isocitrate lyase families seem to be inherently linked to stereoselectivity.

Nicotinate Catabolism in Other Organisms. Database searches identified nine Proteobacteria with gene clusters encoding proteins with amino acid sequence identities of 40–46, 17–38, 40–43, and 42–47% to *E. barkeri*, NdhS, NdhLM, Hnr, and Ena, respectively

(see Fig. 2B). These observations led us to postulate that these organisms are nicotinate-catabolizing Proteobacteria (NCP), which have the first three enzymes in common with *E. barkeri*. This assumption is supported by preliminary data, which show that the heterologously expressed *Bradyrhizobium japonicum* ena homolog has kinetic parameters similar to that of *E. barkeri* enamide. The α-Proteobacterium *Azorhizobium caulinodans* is known to catabolize nicotinate via THON (6, 43) and can therefore be expected to harbor genes similar to those of the nine NCPs. In *M. loti*, THON hydrolysis might be carried out by an evolutionary divergent type of enamide, because no Ena homolog is encoded on the genome. Further catabolism of 2-formylglutarate in the nine NCPs probably proceeds via the glutarate/glutaryl-CoA pathway identified in *A. caulinodans* (43) and thus explains the absence of *hgd*, *hmd*, *mgm*, *mii*, *dmdAB*, and *dml* homologs.

The gene clusters of eight NCPs contained ABC-transporter encoding genes (Fig. 2B). These genes are presumably involved in nicotinate transport as the *B. japonicum* periplasmic binding protein has submicromolar affinity for nicotinate (data not shown). Five NCPs lacked flavin-binding NdhF homologs but had C-terminal extensions to the NdhLM homologs with CXXCH cytochrome *c* binding motifs (three each). Instead of NAD(P)⁺, these nicotinate dehydrogenases might link to the respiratory chain, as in *A. caulinodans* (43). Gene clusters containing nicotinate dehydrogenases with C-terminal tricytochrome *c* extensions but without *hnr* and *ena* were found in Pseudomonads (data not shown). This finding unravels gene clusters associated with nicotinate catabolism via 2,5-dihydroxypyridine, because 6-hydroxynicotinate monooxygenase (4) and maleate cis-trans-isomerase homologous genes were found adjacent.

Concluding Remarks. Fifty years after the discovery of a soil bacterium fermenting nicotinate (8), the plethora of enzymes and cofactors involved in the pathway can now be understood at a molecular level. Two nonmetalloenzymes and seven metalloenzymes containing molybdopterin, selenium, FAD, [2Fe-2S], covalently bound flavin, [4Fe-4S], binuclear Fe-Zn, adenosylcobalamin, and Mg²⁺ catalyze nine highly unique reactions. Structural characterization is in progress and will extend our knowledge on the enzymes at the atomic level.

Materials and Methods

Strains and Cell Growth. *E. barkeri* (DSMZ 1223) was grown under anaerobic conditions at 32°C with nicotinate medium in a 100-l fermentor (24).

Molecular Biology. λ-ZAP-Express phage libraries (30) were prepared by ligation of EcoRI or BamHI/BglII-digested *E. barkeri* DNA into EcoRI or BamHI cut λ-DNA according to manufacturer’s protocol (Stratagene, Amsterdam, The Netherlands). Phage plaques were screened with Southern blot hybridization with digoxigenin-labeled probes based on the known PstI fragment (24), already cloned DNA fragments, or PCR fragments obtained with

primers derived from ≈400 bp reads of direct genomic sequencing at GATC (31). pBK-CMV plasmids were generated by *in vivo* excision and circularization of λ-phage DNA of positive plaques (30). Thus, two EcoRI (1.5 and 4.9 kbp), a 5.4-kbp BglII/BamHI fragment, a 4.2-kbp BglII fragment, and a 2.3-kbp EcoRI fragment were cloned. The extreme 5′ and 3′ ends of the contig were obtained by TOPO TA-cloning (Invitrogen, Karlsruhe, Germany) of PCR fragments. Primers for *Pfu*Hotstart amplification were derived from direct genomic sequencing data and nested PCR by using SeeGene DNA walking with gene specific and annealing control primers (32). Details on cloning and sequencing can be found in Supporting Text, which is published as supporting information on the PNAS web site.

**Substrates.** THON was synthesized from coumalic acid (29) omitting UV-induced (*E/Z*) isomerization of dimethyl-2-aminomethylenegluarate. Synthesis of 2-(hydroxymethyl)glutarate was by saponification (0.1 M NaOH in ethanol for 10 h at room temperature) of NaBH₄-reduced dimethyl 2-formylglutarate obtained by condensation of ethylformate and NaH-treated dimethylglutarate in ether (0°C for 4 h, then room temperature for 10 h). Preparation of 2-formylglutarate was by incubation of 10 mM THON in 20 mM KPᵢ at pH 7.4 with 40 units enamidase at 23°C for 20 min. After Centricon (Millipore, Schwalbach, Germany) YM10 membrane filtration, such preparations contained ≈70 mol % enzymatically active 2-formylglutarate (i.e., NADH consumption in 20 s with 1 unit 2-(hydroxymethyl)glutarate dehydrogenase in 50 mM KPᵢ, pH 7.4) and a residual 20 mol % THON as judged by UV spectroscopy. The (2R,3S)-dimethylmalate was prepared according to ref. 26.

**Activity Measurements.** The 6-hydroxynicotinate reductase was assayed according to ref. 7. Enamidase activity was measured by the decrease of THON absorbance in 50 mM CHES/NaOH, pH 9.5 (Δε₃₀₇ₙₘ = −1.1 mM⁻¹cm⁻¹). NADH production at 340 nm determined 2-(hydroxymethyl)glutarate dehydrogenase activity by using racemic 2-(hydroxymethyl)glutarate in 100 mM glycine/NaOH at pH 9.2. The (2R,3S)-dimethylmalate lyase activity was measured as in ref. 25.

**Purification of Enzymes.** The 6-hydroxynicotinate reductase was isolated from soluble protein extracts of nicotinate-grown *E. barkeri* cells by FPLC purification by using Source 15Q, ceramic hydroxyapatite, and MonoQ columns in an anaerobic glovebox. Enamidase was purified aerobically with Source Phe substituting the second column and an additional MonoQ step. The (2R,3S)-dimethylmalate lyase was isolated from *Escherichia coli* BL21 transformed with pBluescript SK+ containing a KpnI-EcoRI fragment with *E. barkeri dml*. After heat-treatment of soluble protein, purification was effected by fractionated (NH₄)₂SO₄ precipitation, Source Phe, and Source 15Q FPLC purification. Detailed purification protocols can be found in Supporting Text.

**Expression and Purification of N-Terminally Streptagged Proteins.** *Hnr*, *hmd* and *hgd* genes were PCR amplified with *Pfu*Ultra (Stratagene) by using primers for BsaI ligation into pPR-IBA2 as suggested by Primer D’signer (IBA GmbH; Göttingen, Germany). For *ena*, Cfr42I and BamHI restriction sites were used. *Escherichia coli* BL21(DE3) expression at 30°C and Streptactin II affinity chromatography was according to IBA specifications.

**Other Biochemical Techniques.** Protein, Fe, S²⁻ and flavin determination, gel-filtration, SDS/PAGE, MALDI-TOF MS, UV-visible, and EPR spectroscopy were performed as in ref. 44. Zinc was determined as in ref. 45. D. Linder (Giessen, Germany) N-terminally sequenced 6-hydroxynicotinate reductase blotted on PVDF. Purified enamidase was carboxymethylated and subjected to C4-RP HPLC and N-terminally sequenced (G. Mersmann; Münster, Germany), as well as two tryptic peptides (C18-RP HPLC, 0.1% TFA in water, 0–100% acetonitrile gradient).

We thank Dr. T. Selmer for peptide analysis; Prof. R. K. Thauer for access to MALDI-TOF mass and EPR spectrometers; Profs. W. Buckel and B. T. Golding for discussion and support; and Drs. A. Fackelmayer (Genomic Analysis and Technology Core, Konstanz, Germany), O. Knobloch (Seqlab, Göttingen, Germany), and S. Zauner for help with sequencing. This work was supported by Deutsche Forschungsgemeinschaft via the Graduiertenkolleg “Protein Function at the Atomic Level” (to A.A.).

1. London, J. & Knight, M. (1966) *J. Gen. Microbiol.* **44**, 241–254.
2. Kaiser, J.-P., Feng, Y. & Bollag, J.-M. (1996) *Microbiol. Rev.* **60**, 483–498.
3. Andresen, J. R. & Fetzner, S. (2002) *Met. Ions Biol. Syst.* **39**, 405–430.
4. Nakano, H., Wieser, M., Hurh, B., Kawai, T., Yoshida, T., Yamane, T. & Nagasawa, T. (1999) *Eur. J. Biochem.* **260**, 120–126.
5. Nagel, M. & Andresen, J. R. (1990) *Arch. Microbiol.* **154**, 605–613.
6. Kitts, C. L., Schaechter, L. E., Rabin, R. S. & Ludwig, R. A. (1989) *J. Bacteriol.* **171**, 3406–3411.
7. Holcenberg, J. S. & Tsai, L. (1969) *J. Biol. Chem.* **244**, 1204–1211.
8. Harary, I. (1956) *Nature* **177**, 328–329.
9. Stadtman, E. R., Stadtman, T. C., Pastan, I. & Smith, L. D. (1972) *J. Bacteriol.* **110**, 758–760.
10. Pastan, I., Tsai, L. & Stadtman, E. R. (1964) *J. Biol. Chem.* **239**, 902–906.
11. Tsai, L., Pastan, I. & Stadtman, E. R. (1966) *J. Biol. Chem.* **241**, 1807–1813.
12. Holcenberg, J. S. & Stadtman, E. R. (1969) *J. Biol. Chem.* **244**, 1194–1203.
13. Kung, H. F., Cederbaum, S., Tsai, L. & Stadtman, T. C. (1970) *Proc. Natl. Acad. Sci. USA* **65**, 978–984.
14. Kung, H. F. & Stadtman, T. C. (1971) *J. Biol. Chem.* **246**, 3378–3388.
15. Gladyshev, V. N., Khangulov, S. V. & Stadtman, T. C. (1994) *Proc. Natl. Acad. Sci. USA* **91**, 232–236.
16. Gladyshev, V. N. & Lecchi, P. (1996) *Biofactors* **5**, 93–97.
17. Gladyshev, V. N., Khangulov, S. V. & Stadtman, T. C. (1996) *Biochemistry* **35**, 212–223.
18. Dilworth, G. L. (1982) *Arch. Biochem. Biophys.* **219**, 30–38.
19. Self, W. T. & Stadtman, T. C. (2000) *Proc. Natl. Acad. Sci. USA* **97**, 7208–7213.
20. Wagner, R., Cammack, R. & Andresen, J. R. (1984) *Biochim. Biophys. Acta* **791**, 63–74.
21. Schräder, T., Rienhöfer, A. & Andresen, J. R. (1999) *Eur. J. Biochem.* **264**, 862–871.
22. Huber, R., Hof, P., Duarte, R. O., Moura, J. J. G., Moura, I., Liu, M.-Y., LeGall, J., Hille, R., Archer, M. & Romão, M. J. (1996) *Proc. Natl. Acad. Sci. USA* **93**, 8846–8851.
23. Hartrampf, G. & Buckel, W. (1986) *Eur. J. Biochem.* **156**, 301–304.
24. Beatrix, B., Zelder, O., Linder, D. & Buckel, W. (1994) *Eur. J. Biochem.* **221**, 101–109.
25. Pirzer, P., Lill, U. & Eggerer, H. (1979) *Hoppe-Seyler’s Z. Physiol. Chem.* **360**, 1693–1702.
26. Lill, U., Pirzer, P., Kukla, D., Huber, R. & Eggerer, H. (1980) *Hoppe-Seyler’s Z. Physiol. Chem.* **361**, 875–884.
27. Löhllein, G. & Eggerer, H. (1982) *Hoppe-Seyler’s Z. Physiol. Chem.* **363**, 1103–1109.
28. Kollmann-Koch, A. & Eggerer, H. (1984) *Hoppe-Seyler’s Z. Physiol. Chem.* **365**, 847–857.
29. Tsai, L. & Stadtman, E. R. (1971) *Methods Enzymol.* **18B**, 233–249.
30. Short, J. M., Fernandez, J. M., Sorge, J. A. & Huse, W. D. (1988) *Nucleic Acids Res.* **16**, 7583–7600.
31. Heiner, C. R., Hunkapiller, K. L., Chen, S. M., Glass, J. I. & Chen, E. Y. (1998) *Genome Res.* **8**, 557–561.
32. Hwang, I. T., Kim, Y. J., Kim, S. H., Kwak, C. I., Gu, Y. Y. & Chun, J. Y. (2003) *BioTechniques* **35**, 1180–1184.
33. Tropel, D. & van der Meer, J. R. (2004) *Microbiol. Mol. Biol. Rev.* **68**, 474–500.
34. Unciuleac, M., Warkentin, E., Page, C. C., Boll, M. & Ermler, U. (2004) *Structure (London)* **12**, 2249–2256.
35. Bonin, I., Martins, B. M., Purvanov, V., Fetzner, S., Huber, R. & Dobbek, H. (2004) *Structure (London)* **12**, 1425–1435.
36. Holm, L. & Sander, C. (1997) *Proteins Struct. Funct. Genet.* **28**, 72–82.
37. Seibert, C. M. & Raushel, F. M. (2005) *Biochemistry* **44**, 6383–6391.
38. Ireton, G. C., McDermott, G., Black, M. E. & Stoddard, B. L. (2002) *J. Mol. Biol.* **315**, 687–697.
39. Hawes, J. W., Harper, E. T., Crabb, D. W. & Harris, R. A. (1996) *FEBS Lett.* **389**, 263–267.
40. Hofmeister, A. E. M., Textor, S. & Buckel, W. (1997) *J. Bacteriol.* **179**, 4937–4941.
41. Irvin, S. D. & Bhattacharjee, J. K. (1998) *J. Mol. Evol.* **46**, 401–408.
42. Grimm, C., Evers, A., Brock, M., Maerker, C., Klebe, G., Buckel, W. & Reuter, K. (2003) *J. Mol. Biol.* **328**, 609–621.
43. Kitts, C. L., Lapointe, J. P., Lam, V. T. & Ludwig, R. A. (1992) *J. Bacteriol.* **174**, 7791–7797.
44. Dickert, S., Pierik, A. J. & Buckel, W. (2002) *Mol. Microbiol.* **44**, 49–60.
45. Hunt, J. B., Neece, S. H., Schachman, H. K. & Ginsburg, A. (1984) *J. Biol. Chem.* **259**, 14793–14803.
