
Available online at www.sciencedirect.com

Journal of Structural Biology 156 (2006) 72–83

Review

Proteasomes and their associated ATPases: A destructive combination

David M. Smith${}^{a}$, Nadia Benaroudj${}^{b}$, Alfred Goldberg${}^{a,*}$

${}^{a}$ Harvard Medical School, Department of Cell Biology, 240 Longwood Avenue, Boston, MA 02115, USA  
${}^{b}$ Pasteur Institute, Unit of Bacterial Genetics and Differentiation, 25-28 rue du Dr. Roux, 75724 Paris Cedex 15, France  

Received 24 February 2006; received in revised form 19 April 2006; accepted 19 April 2006  
Available online 8 May 2006

---

### Abstract

Protein degradation by 20S proteasomes in vivo requires ATP hydrolysis by associated hexameric AAA ATPase complexes such as PAN in archaea and the homologous ATPases in the eukaryotic 26S proteasome. This review discusses recent insights into their multistep mechanisms and the roles of ATP. We have focused on the PAN complex, which offers many advantages for mechanistic and structural studies over the more complex 26S proteasome. By single-particle EM, PAN resembles a “top-hat” capping the ends of the 20S proteasome and resembles densities in the base of the 19S regulatory complex. The binding of ATP promotes formation of the PAN–20S complex, which induces opening of a gate for substrate entry into the 20S. PAN’s C-termini, containing a conserved motif, docks into pockets in the 20S’s α ring and causes gate opening. Surprisingly, once substrates are unfolded, their translocation into the 20S requires ATP binding but not hydrolysis and can occur by facilitated diffusion through the ATPase in its ATP-bound form. ATP therefore serves multiple functions in proteolysis and the only step that absolutely requires ATP hydrolysis is the unfolding of globular proteins. The 26S proteasome appears to function by similar mechanisms.

© 2006 Elsevier Inc. All rights reserved.

Keywords: Proteasome; ATPase; PAN; Proteasome activating nucleotidase; 20S; 26S; RPT; 19S; HbYX; Gate; YDR; ATP; ATPγS; Translocation; Protein; Brownian ratchet; Brownian; Proteasomal ATPases; Archaea; EM; Electron microscopy; LFP; Degradation; Protein degradation; Turnover

---

### 1. Introduction

A fundamental feature of protein breakdown in eukaryotic and prokaryotic cells is its requirement for ATP ([Goldberg and St. John, 1976](#)). Much of our current knowledge about intracellular proteolysis came from studies seeking to understand the biochemical basis of this surprising requirement ([Ciechanover, 2005](#); [Goldberg, 2005](#)). The key early developments were the discovery of a soluble (nonlysosomal) ATP-dependent proteolytic system in reticulocytes ([Etlinger and Goldberg, 1977](#)) followed by the establishment of similar energy-dependent proteolytic systems in extracts of *Escherichia coli* ([Murakami et al., 1979](#)). Analysis of these bacterial systems led to the discovery of large ATP-dependent proteolytic complexes that degrade proteins and ATP in linked processes ([Chung and Goldberg, 1981](#); [Gottesman, 1996](#)).

In eukaryotes, ATP is required both for ubiquitin conjugation to substrates and for the function of the 26S proteasome, the ATP-dependent complex that catalyzes the breakdown of ubiquitinated and certain non-ubiquitinated polypeptides ([Ciechanover, 2005](#); [Goldberg, 2005](#); [Voges et al., 1999](#)). The discovery of the first ATP-dependent protease in bacteria (*lon/La*) ([Chung and Goldberg, 1981](#)) was made about the same time as the classic discovery of the role of ubiquitin in protein breakdown in the reticulocyte system by Hershko, Ciechanover, and Rose ([Ciechanover, 2005](#); [Glickman and Ciechanover, 2002](#)). The energy requirement for ubiquitin conjugation in eukaryotes was thought to explain the ATP requirements for intracellular proteolysis in eukaryotes. Thus, initially it was believed that there are two very different explanations for the ATP requirements for proteolysis in prokaryotes and eukaryotes. However, after further study, it became clear that after

* Corresponding author.
E-mail address: alfred_goldberg@hms.harvard.edu (A. Goldberg).

1047-8477/$ - see front matter © 2006 Elsevier Inc. All rights reserved.  
doi:10.1016/j.jsb.2006.04.012
ubiquitination, ATP was still required for breakdown of the protein (Tanaka et al., 1983), and by the late 1980s, the 26S proteasome was identified as the ATP-dependent proteolytic complex that degrades ubiquitinated proteins (Hough et al., 1987; Waxman et al., 1987).

As interest in the eukaryotic 20S proteasome developed, archaea were found to contain a simpler, but structurally markedly similar proteolytic complex in *Thermoplasma acidophilum* (Baumeister et al., 1998; Dahlmann et al., 1989; Voges et al., 1999). Further work also uncovered the existence of an ATPase complex, PAN, which functions together with the archaeal 20S proteasome (Benaroudj and Goldberg, 2000; Smith et al., 2005; Zwickl et al., 1999). Thus, protein breakdown in archaea, bacteria and eukaryotes is catalyzed by large proteolytic complexes that hydrolyze ATP and protein in linked reactions. Interestingly, PAN is not found in all archaea (e.g., *T. acidophilum*). However, it appears likely that all archaea contain ATPase ring complexes of the AAA family that may also function in protein degradation by the proteasome. For example, VAT, which is found in *T. acidophilum*, seems likely to function in substrate recognition, unfolding, and translocation of substrates into the 20S proteasome (Gerega et al., 2005).

### 1.1. The 26S proteasome

The ATP-dependent 26S proteasome is composed of one or two 19S regulatory complexes and the central 20S particle (Voges et al., 1999; Zwickl et al., 1999), which is a hollow cylinder, within which proteolysis occurs. The two outer α rings and two inner β rings of the 20S particles are each composed of seven distinct but homologous subunits. In eukaryotes, three of the β subunits contain proteolytic sites, which are sequestered in the hollow interior of the 20S particle (Groll et al., 1997). Substrates enter the 20S through a narrow channel formed by the α subunits, whose N-termini, depending on their conformation, can either obstruct or allow substrate entry and thus function as a gate (Groll et al., 2000; Groll and Huber, 2003). This entry channel is narrow and only permits passage of unfolded, linearized polypeptides (Groll et al., 1997). The 19S regulatory complex is composed of two subcomplexes, the lid, which seems to bind and disassemble the ubiquitin-conjugated substrate, and the base, which contains six homologous ATPase subunits (termed Rpt1–6 in yeast) plus two non-ATPases, Rpn 1 and 2 (Voges et al., 1999). These ATPases are members of the AAA family of ATPases (Patel and Latterich, 1998). For a globular protein to be degraded, it must associate with the 19S ATPases and undergo ATP-dependent unfolding followed by translocation into the 20S particle, which requires opening of the gate in the α ring (Kohler et al., 2001). Each of these steps is regulated in some way by the ATPase complex.

### 1.2. AAA ATPases and proteolysis

The ATPase complexes that regulate protein degradation in eukaryotes, bacteria and archaea are all members of the AAA⁺ (ATPases Associated with various cellular Activities) ATPase superfamily (for review see Ogura and Tanaka, 2003). The AAA⁺ family of ATPases are found in all living organisms and in all cell compartments, where they participate in a variety of essential cellular processes such as mitosis, protein folding and translocation, DNA replication and repair, membrane fusion and proteolysis. They are characterized by the presence of one or two conserved ATP-binding domains (200–250 residues), called the AAA motif, consisting of a Walker A and a Walker B motif (Confalonieri and Duguet, 1995). The eukaryotic and archaeal (PAN) proteasomal ATPases belong to a subfamily of AAA⁺ ATPases (AAA family) that contains an additional motif called the second region of homology (SRH) (Lupas and Martin, 2002). Despite the large variety of cellular processes in which AAA⁺ ATPases participate, they have some common features. A recurrent structural feature of most AAA⁺ ATPases is their assembly into oligomeric (generally hexameric) ring-shaped structures with a central pore. In addition, most appear to be involved in protein folding or unfolding, and assembly or disassembly of protein complexes through nucleotide-dependent conformational changes. Thus, recent insights into the functioning of the archaeal PAN complex and the 19S proteasomal regulatory ATPase may illuminate the functioning of these other AAA Family members (and vise versa).

Though bacteria do not contain 20S proteasomes, like those in eukaryotes, they do contain several large compartmentalized protease complexes that associate with AAA ATPase complexes such as HslUV and ClpAP. HslV is a two-ring peptidase complex which shares homology with the beta subunits of the 20S proteasome (Bochtler et al., 2000), and forms a six-membered ring (Rohrwild et al., 1997) rather than the seven-membered ring, which is characteristic of the 20S proteasome. HslU, the ATPase complex, associates with HslV to stimulate protein degradation, and is homologous to PAN. X-ray diffraction studies established that HslU induces conformational changes in the peptidase active site of HslV upon association and increases the pore size of HslV. Thus, HslU increases the peptidase activity of HslV by allosteric activation and probably also by promoting substrate unfolding for peptide entry (Huang and Goldberg, 1997; Sousa et al., 2000; Wang et al., 2001; Yoo et al., 1997). Facilitating peptide entry thus appears to be a common property shared by HslU, PAN, and the 19S ATPases, although HslV does not contain an outer α ring or gating termini like those in the 20S proteasome.

## 2. The function of the 26S ATPases

Studying the ATP-dependent processes and the mechanisms of protein breakdown within the 26S proteasome has proven difficult because of its structural complexity, multiple enzymatic activities and ubiquitin requirement. Nevertheless, several important discoveries about these ATPases have been made using genetic tools in yeast. Through
systematic mutagenesis of the ATP binding sites in each of
the six different ATPases, Finley and co-workers showed
that these 19S ATPases (Rpt1–6) perform distinct functions
in protein degradation (Rubin et al., 1998). For example,
mutations in the ATP-binding site of Rpt2 severely impair
global protein degradation and cause a G2 cell cycle arrest,
while the corresponding mutation in Rpt1 does not influence
global protein breakdown but caused a G1 cell cycle
arrest (Braun et al., 1999). This same type of mutation in
the other ATPase subunits caused varying degrees of
growth defects in yeast when challenged with heat shock or
canavinine, which promote the accumulation of large
amounts of misfolded proteins. These findings argue that
different ATPases aid in the recognition and degradation of
different subsets of proteins. Interestingly, the Rpt2 mutant
was found to be the most influential for cell growth. Subsequent work showed that an intact ATP binding domain in
Rpt2 was necessary to induce gate opening in the 20S
(Kohler et al., 2001) and that the α3ΔN 20S truncation
mutant, which cannot form a closed gate, could rescue
yeast cells containing the Rpt2 mutant from their growth
defect. It is not clear however whether or not Rpt2 is the
only ATPase that can induce gate opening. Although,
genetic analysis has been informative, further biochemical
studies of the yeast 26S proteasome are necessary to better
understand its molecular mechanisms and such studies
have proven quite challenging. For example, though Braun
et al. and DeMartino et al. have demonstrated that the
base of 19S contains some chaperone-like properties
(Braun et al., 1999; Liu et al., 2003) no one has yet been able
to directly demonstrate unfolding of globular proteins by
purified 26S proteasomes, even though such an activity is
necessary for globular protein to enter the 20S particle (see
below). However, many of the 26S proteasomes properties
are very similar to those found for the homologous ATPase
proteasome complex found in archaea, which has proven to
be an excellent model system to elucidate the roles of ATP
in proteasome function.

ATPases and shares 41–45% similarity with all six of them.
In fact, PAN shows greater sequence similarity to several of
the 26S ATPases than these ATPases show with one
another! Based on its homology to other AAA complexes,
PAN is presumably a hexameric ring complex which possesses a predicted coiled-coil motif at its N-termini, as do
the 26S ATPases (Zwickl et al., 1999). In contrast to prote-
olysis in eukaryotes, degradation of proteins in archaea
occurs without ubiquitin conjugation or any similar ATP-
dependent modification of substrate (Zwickl et al., 1999).
Nevertheless, protein degradation by PAN and archaeal
20S proteasomes still seems to involve substrate recogni-
tion, ATP-dependent unfolding, translocation, and opening
of the gated channel in the proteasome (Benaroudj et al.,
2003; Ogura and Tanaka, 2003; Smith et al., 2005).

4. Structure and association of PAN with the 20S particle

Although the association of an ATPase chaperone-like
complex with a proteolytic particle appears to be a com-
mon feature of several ATP-dependent systems for intracel-
lular protein degradation (the 26S proteasome and the
bacterial ClpAP, ClpXP and HslUV complexes), an associ-
ation between PAN and the 20S particle was difficult to
observe by typical biochemical approaches, even when
PAN and the 20S came from the same species. However,
when PAN is mixed with archaeal 20S proteasomes and
ATP, it could stimulate the degradation of proteins that
lack tight tertiary structures as well as stable globular
proteins (Benaroudj and Goldberg, 2000; Navon and
Goldberg, 2001; Smith et al., 2005).

Our recent studies using surface plasmon resonance
(SPR) and electron microscopy have directly demon-
strated that PAN does indeed associate with the 20S. This
association requires binding but not hydrolysis of ATP, as
determined by use of the non-hydrolyzable analogs of
ATP: ATPγS, and AMPPNP. Moreover, ADP cannot
support PAN–20S association and even competes for
binding of ATPγS as was found in the bacterial ATP-
dependent protease complexes ClpAP (Maurizi et al.,
1998) and HslUV (Huang and Goldberg, 1997; Yoo et al.,
1997). While this association of PAN with the 20S was
readily observed in the presence of ATPγS, it is difficult to
demonstrate with ATP, presumably because ATP hydro-
lysis to ADP (which must be occurring continuously in
the PAN complex) weakens this association, since ADP
inhibits complex formation. Yoo et al. (1997) reached the
same conclusion regarding the HslUV complex. Accord-
ingly, we found by SPR that the PAN–20S complex (once
formed in the presence of ATPγS) dissociates five times
faster with ATP present than with ATPγS (Smith et al.,
2005). Although the PAN–20S complex is short-lived in
the presence of ATP, it clearly is sufficiently stable to
allow substrate translocation. It is also possible that sub-
strate binding to PAN increases its affinity for the 20S
complex, just as it enhances its ATPase activity (Ben-
aroudj et al., 2003; Smith et al., 2005).

3. The PAN ATPase complex from archaea

The first complete genome sequence in the domain of
archaea was from Methanococcus jannaschii. This sequenc-
ing revealed a gene (S4) which was highly homologous to
the genes encoding the 19S ATPases (Bult et al., 1996). To
test if this gene product might regulate the 20S proteasome,
the S4 gene was expressed in E. coli, and the 50kDa prod-
uct, named PAN (proteasome-activating nucleotidase), was
purified and characterized by Zwickl et al. (1999). PAN’s
sequence contains several hallmarks of the AAA ATPases
family: a single AAA domain, one P-loop motif (which
includes the Walker A and B motifs), and a second region
of homology (SRH motif) at its C-terminus. When mixed
with the archaeal 20S particle in the presence of ATP, PAN
stimulated the degradation of unfolded proteins as well as
globular ones (Benaroudj et al., 2001; Zwickl et al., 1999).
PAN is the closest known homolog of the eukaryotic 26S

The EM images of the PAN–20S complex provided the first structural information on the proteasomal ATPase PAN and also helped clarify the prior EM structures of the 26S proteasome (Fig. 1). In addition to the expected large inner ring, PAN contains what looks like a smaller “outer ring”, and thus resembles a “top hat” that caps either or both ends of the 20S particles. A very similar “top hat” structure is also evident in EM images of 26S proteasomes from several species. Thus, the PAN–20S complex closely resembled the 26S complex less its lid (Fig. 1). This apparent secondary “outer ring” in the base of the 19S cap had not been thought to be part of the ATPases and had been proposed to correspond to Rpn1 and 2 (Kajava, 2002). Based on its close similarity to PAN, we proposed that this “outer-ring” density is part of the ring of ATPases Rpt1–6. As mentioned, the N-terminal regions of these six ATPases, like that of PAN, are predicted to adopt a coiled-coil fold (Gorbea et al., 1999; Zwickl et al., 1999), which may mediate the binding of protein substrates (Wang et al., 1996). A similar coiled-coil region also exists in the homologous ATPase HslU, which forms a structure similar to PAN when complexed with HsIV (Rohrwild et al., 1997). The coiled-coil region is found in the outer ring of HslU, and is essential for ATP-dependent substrate degradation (Song et al., 2000). Most likely, these outer rings in PAN and the 19S also correspond to the coiled-coil domain and are important for substrate recognition, especially since they form the entrance of the axial translocation channel that leads to the 20S active sites.

In addition, the images of the 26S also display a weaker density bound to this “top hat” structure on the opposite side to where the 19S lid joins the base (Fig. 1, see *). It had been speculated that this density corresponds to a substrate (Walz et al., 1998), but this seems unlikely because this density is seen consistently in proteasomes purified by different approaches from several species. The two non-ATPase base subunits, Rpn1 and 2, both associate with the ATPases, apparently at opposite flanks of the ATPase ring, and Rpn1 is known to contact Rpn10 at the joint where the base and lid intersect (Ferrell et al., 2000). By exclusion, we therefore proposed that this weaker density probably corresponds to Rpn2.

These studies uncovered another interesting feature of the PAN–20S complex; we observed significant class average variations caused by a deviation in PAN’s inclination with respect to the 20S. Thus, the PAN ATPase ring seems to “wag” in relation to the 20S particle, and a similar class average variation has been found for the 26S proteasomes by Baumeister and colleagues (Walz et al., 1998). This “wagging” suggests that PAN’s contact with the 20S is not static and may occur at local regions at the interface of the two complexes. Since this apparent “wagging” is conserved between archaeal and eukaryotic proteasomes, this motion may be functionally significant. It is likely that the ATPase subunits tightly associated with the 20S are those with ATP bound, while the others have ADP or no nucleotide bound since ATP binding, but not ADP, promotes association with the 20S. Such a mechanism could generate a “wagging” motion if ATP hydrolysis (and then ATP-ADP exchange) occurs randomly or sequentially around the ATPase ring rather than in a concerted manner.

5. PAN regulates gate opening

Because of the tight interaction between the 20S proteasomes α and β subunits, substrates can enter only through the 20S pore at either end of the particle (Baumeister et al., 1998). The elegant X-ray analysis of M. Groll et al. showed that this channel is gated by the N-termini of the α subunits (Groll et al., 1997). These N-termini in eukaryotic proteasomes can assume either of two ordered structures, an open conformation and a closed one, both of which require the YDR motif for stabilization (Forster et al., 2003; Groll et al., 1997, 2000; Groll and Huber, 2003). As discussed above, at least one ATPase subunit, Rpt2, is able to switch the gate from the closed to open state (Kohler et al., 2001; Rubin et al., 1998). In addition, gate opening also occurs in an ATP-independent manner when the 11S regulator, PA28 (or its homolog PA26) associates with the 20S particle (Forster et al., 2005, 2003; Whitby et al., 2000). The conserved YDR motif is essential for gating in eukaryotic proteasomes, but it is also present in the N-termini of the α subunits of many archaeal strains (Groll and Huber, 2003), even though initial reports have concluded that archaeal proteasomes lack such a functional gate. Under certain crystallization conditions the archaeal N-terminal residues also form an ordered open gate structure that is dependent on the YDR motif (Groll et al., 2003) and is congruent with the eukaryotic open structure (Forster et al., 2003). However, a closed gate structure, like that in eukaryotic proteasomes, has not been observed in archaeal proteasomes, and Forster et al. (2003) concluded that the archaeal 20S N-termini are not able to form such a closed gate because they

lack the sequence asymmetry that is necessary for the closed gate conformation found in eukaryotic proteasomes. In fact, the absence of a regulated gating mechanism for archaeal 20S proteasomes had been assumed to represent a major difference between archaeal and eukaryotic proteasomes (Groll et al., 2003; Groll and Huber, 2003). Nevertheless, archaeal proteasomes could have a symmetric closed gate structure that is unique to archaeal 20S particles and that is easily disrupted by crystallization or conditions required for crystallization.

Several other observations have also suggested that archaeal 20S proteasomes lack a functional gate in the α ring that prevents substrate entry (Voges et al., 1999). For example, archaeal 20S proteasomes (unlike eukaryotic particles) alone rapidly cleave tri- or tetra-peptide substrates, and PAN and ATP do not stimulate their hydrolysis (Zwickl et al., 1999). Also, deletion of residues 2–12 in the α subunits, which are homologous to the residues forming the gate in eukaryotic proteasomes, does not enhance the degradation of these short peptides (Benaroudj et al., 2003). However, after deletion of these gating residues, a central pore in the α rings becomes evident by EM, and the proteasome shows a dramatic increase in its capacity to degrade unfolded proteins (e.g., β-casein) (Benaroudj et al., 2003). Using fluorescently quenched peptides of different lengths we found that the gating residues in the archaeal α ring can act as a barrier to entry of peptides as small as seven residues (Smith et al., 2005). When these N-termini are deleted, these heptapeptides enter readily and are degraded (as occurs in the yeast 20S with tetrapeptide substrates when the α3-N-terminus is deleted) (Groll et al., 2000; Smith et al., 2005). However, the configuration of the closed archaeal gate clearly differs from that in eukaryotes since it contains an opening large enough for four-residue peptides to traverse readily. Whether the closed form of the archaeal gate conforms to a strict conformation or whether the random occupation of the α N-termini in the translocation channel is sufficient to block peptide entry is unknown, either way the archaeal proteasome certainly does contain a functional gate, with both open and closed states.

Since gate opening could now be directly monitored by use of these new fluorescent peptides, we tested if PAN was able to induce gate opening. PAN with ATP or ATPγS present was found to stimulate hydrolysis of a nine-residue peptide (LFP) by the 20S particle (Fig. 2) (Smith et al., 2005). The 20S therefore contains a functional gate that exists in both open and closed states, and PAN in the ATP bound state stimulates opening of this gate. Thus, ATP binding to PAN supports both formation of PAN–20S complex and stimulates peptide entry through gate opening. Several of our observations strongly suggest that gate opening occurs upon the association of PAN with the 20S particle. For example, gate opening occurred under the exact conditions where PAN was found complexed with the proteasome (i.e., with ATP, AMPPNP and ATPγS present, but not with ADP), and gate opening and complex formation show the same nucleotide concentration dependencies

![Figure 2](#fig2)

Fig. 2. ATP binding to PAN stimulates LFP hydrolysis by 20S proteasomes. Both ATP and non-hydrolyzable analogs of ATP (1 mM) stimulate LFP degradation by the PAN–20S complex. All data points contain PAN and the 20S and are normalized to control (no nucleotide).

(D.M. Smith, G. Kafri, and A. Goldberg unpublished observations). The mechanism of this activation of 20S function by PAN and ATP, like the 26S proteasome, is quite different from the activation of the bacterial ATP-dependent proteases, such as HslUV and ClpA(X)P, where ATP binding induces an allosteric activation of their peptidase sites (Kim et al., 2001; Smith et al., 2005). By contrast, activation of the 20S particle by PAN occurs by induction of gate opening without any alteration in the activity of the peptidase sites, because PAN with ATP could not stimulate the degradation of tri or tetra peptide substrates which freely enter the archaeal 20S particle. Furthermore, PAN is not able to stimulate gateless proteasomes to degrade the nine-residue peptide LFP. Thus, upon binding of ATP, PAN associates with the 20S and induces gate-opening a key step in the activation of proteasome function.

### 6. The mechanism of the PAN–20S association and gate opening

These studies have established that ATP-binding to PAN is essential for its association with the 20S particle and for triggering gate opening in the 20S (Smith et al., 2005). A very different (non-homologous) type of proteasomal regulator, the 11S, PA28 (REG) complex and its invertebrate homolog, PA26, also bind to the α-ring (independent of ATP) and induce gate opening. However, these complexes stimulate peptide but not protein entry. Their association with the 20S requires their extreme C-termini (Ma et al., 1993), and using X-ray crystallography, Hill and co-workers demonstrated that these C-termini dock into pores between the adjacent α subunits (Forster et al., 2005; Whitby et al., 2000). Opening of the gate in the α ring also required a distinct activation loop in the PA26 subunits that interacts with the base of the N-terminal gating residues of the 20S particle to stabilize the open-gate conformation (Whitby et al., 2000). Though PA28 shows no sequence homology with the proteasomal ATPases and has quite different biochemical actions, this mechanism of gate

opening initially suggested an attractive model for how proteasomal ATPases may induce gate opening.

While investigating this hypothesis we noted that four of the eukaryotic ATPases, as well as PAN, contain a three-residue conserved C-terminal motif composed of a hydrophobic-tyrosine-X sequence (HbYX). We therefore tested whether this C-termini motif might be important for association with the 20S. Altering the HbYX residues in PAN by mutagenesis or removing the C-terminal residue prevented its association with the 20S complex and simultaneously blocked opening of the gated substrate-entry channel (unpublished data). In addition, we found that these C-terminal residues bind to pockets between the α subunits of the 20S, much like the C-termini of PA28/PA26. Surprisingly, small peptides that correspond to PAN's C-terminus contain this HbYX motif and can by themselves bind to these intersubunit pockets in the 20S and trigger gate-opening. In fact, they can even induce gate opening in eukaryotic 20S particles whose seven α subunits are distinct but homologous to each other (unpublished data). We thus were able to show that: (1) Upon ATP-binding to these ATPases, their conserved C-terminal residues dock into pockets in the α ring and act as “fasteners” to link the ATPase and 20S complex and (2) they function like a “key-in-a-lock” to open the gate that limits substrate entry into the

20S particle (Fig. 3A). Because the seven-residue peptide from PAN’s C-terminus can bind and by itself trigger gate opening in both archaeal and mammalian proteasomes, the gate-opening reactions do not require other parts of the ATPase molecule (Fig. 3B). Specifically, it does not require an activation domain like that required for gate opening by PA28/PA26. In other words, their mechanisms of gate opening differ in important ways. Therefore, this ATPase-20S interaction appears to be a universal physiological mechanism that regulates protein degradation by the proteasome.

Although the 19S and 20S dissociate slowly in the absence of a nucleotide and reassociate when ATP is added (Coux et al., 1996; Voges et al., 1999), the 26S proteasome is quite stable in the presence of ATP, unlike the PAN–20S complex. With the isolated eukaryotic 26S proteasome, ADP prevented gate opening, and non-hydrolyzable ATP analogs stimulated this process even better than ATP, just as with the PAN–20S complex (Smith et al., 2005). Thus, gate opening in the eukaryotic 26S complex shows similar nucleotide dependence as does the PAN–20S complex. Since the removal of the N-terminus of only the α-3 subunit in yeast 20S causes destabilization of the closed gate (Groll et al., 2000; Smith et al., 2005), it seems likely that ATP binding to the ATPase subunit adjacent to the α-3 subunit


Fig. 3. Model’s depicting the association of PAN with the α ring of the 20S proteasome and how PAN’s C-termini induce gate opening. (A) The C-termini (yellow) of PAN (orange) dock into the intersubunit pores in the top of the 20S. The HbYX motif in PAN’s C-termini is colored red. When PAN associates with the 20S proteasome (upon binding ATP) the translocation channel gate in the 20S is opened. (B) Seven-residue peptides that correspond to PAN’s C-termini dock to the intersubunit pockets in the α ring and induce gate opening by themselves.

can induce gate opening in all the N-termini. Possibly, binding of ATP to any of the 19S ATPases or PAN subunits may induce the open-gate conformation in the adjacent α subunit. This mechanism would suggest that each of the α subunit’s N-terminus in the 20S could be cycling through open and closed states, as different ATPase subunits bind ATP and hydrolyze it to ADP. However, thus far only Rpt2 has been shown to induce gate opening in the 26S complex (Kohler et al., 2001) and it contains the gate-opening HbYX motif on its C-terminus.

7. The energy requirements for protein unfolding and translocation

It has been clear since the early seventies that protein degradation in prokaryotes and eukaryotes requires ATP (Ciechanover, 2005; Goldberg, 2005; Goldberg and St. John, 1976). However, it is still unclear which of the multiple steps in the process of protein degradation requires ATP and how nucleotide binding or hydrolysis enhance these steps. In particular, determining the mechanisms whereby the 19S regulatory particle unfolds substrates and facilitates their entry into the 20S proteolytic core particle remain important mysteries. Investigating these processes in the 26S proteasome is very difficult due to the requirement for ubiquitination of substrates and the instability and complexity of the 19S particle, which contains at least 17 different subunits. To investigate these questions in recent years, we have focused on the much simpler PAN–20S complex.

Like the several AAA ATPases that promote protein degradation in *E. coli*, PAN’s ATPase activity is stimulated 2–5 fold by protein substrates such as the globular GFPssrA, the loosely folded casein, and even by the 11 residues ssrA recognition peptide (Benaroudj et al., 2003). In bacteria, this 11 residue ssrA sequence is incorporated into nascent polypeptide chains when the ribosome stalls due to the lack of aminoacyl transfer RNA. In *E. coli*, this ssrA “tag” then targets the protein to degradation by several ATP-dependent proteases (Gottesman et al., 1998). Interestingly, although no such modification of proteins has been found in archaea, PAN also recognizes this peptide and its fusion to GFP targets it to PAN which can unfold GFP and translocate it into the 20S particle for degradation in the presence of ATP. Since the 11 residue peptide of ssrA itself can stimulate PAN’s ATPase activity, ATP hydrolysis is thus activated merely by substrate binding (Benaroudj et al., 2003).

Prior studies of PAN (Benaroudj and Goldberg, 2000; Benaroudj et al., 2003), and of the *E. coli* ATP-dependent proteases ClpAP (Weber-Ban et al., 1999) and ClpXP (Kenniston et al., 2003) demonstrated that unfolding of globular substrates (e.g., GFPssrA) requires ATP hydrolysis. As expected, PAN-catalyzed unfolding of GFPssrA is essential for its degradation by 20S proteasomes since this globular protein is too large to pass through the narrow opening in the α ring, which only admits unfolded polypeptides (Benaroudj and Goldberg, 2000). These findings provided the first evidence that proteasomal ATPases have an unfoldase activity (Benaroudj et al., 2001). Surprisingly, to date no similar unfoldase activity has been demonstrated for purified 26S particles. Nevertheless, such an activity is certain to exist in the 26S proteasome (Murakami et al., 2000), since many GFP fusion proteins, when expressed in eukaryotic cells, are rapidly degraded by the proteasome, and clearly such an activity is required for their translocation into the 20S particle. As noted above, the base of the 19S particles, presumably though its ATPases, does have certain chaperone-like activities; e.g., it can reduce protein aggregation, promote refolding of denatured proteins (Braun et al., 1999; Strickland et al., 2000), remodel certain misfolded substrates and expose buried hydrophobic sequences in the polyubiquitinated DHFR (although, this remodeling activity does not seem to require ATP binding or hydrolysis) (Liu et al., 2003). Therefore, as yet no direct linkage of these chaperone-like activities to protein breakdown, by the 26S proteasome, has been established, such a role is very likely in both proteolysis and in the functioning of the 19S particle in regulating gene transcription. By contrast for the archaeal ATPase complex, the role of ATP binding and hydrolysis in substrate binding, unfolding and translocation into the 20S proteasome, are now understood in considerable detail.

7.1. The nucleotide requirement for substrate translocation

Protein translocation through a small pore is a critical step in many cellular processes. “Active” mechanisms, for protein degradation, that utilize ATP hydrolysis to pull or push a protein in one direction through the pore in the ATPase complexes, have been proposed for the 26S proteasome (Matouschek, 2003) and bacterial ATP-dependent Clp proteases (Sauer et al., 2004). Because the 26S proteasome seems to unfold globular substrates by sequential unraveling, it has been suggested that protein unfolding and translocation are linked processes in which the ATPases pull an unstructured region through the pore in the ATPase into the 20S particle, thus causing unfolding of upstream regions (Lee et al., 2001; Matouschek, 2003; Prakash et al., 2004). Such a mechanism requires concomitant ATP hydrolysis and implies that unfolding cannot occur without translocation. Although widely accepted, several key assumptions of this attractive model do not seem valid for the proteasome.

Although ATP hydrolysis is necessary for degradation of globular proteins, we found that if a globular protein is first denatured, then ATP is no longer necessary for translocation and degradation by the PAN–20S complex. For example, when GFPssrA is given to the PAN–20S complex in the presence of ATPγS, no degradation occurs, but if GFPssrA (or ovalbumin) is first chemically denatured then it is rapidly degraded by the PAN–20S complex (Fig. 4) (Smith et al., 2005). Thus, translocation by PAN can occur by purely “passive diffusion” once the substrate has been

Fig. 4. ATP hydrolysis is not required for translocation of unfolded proteins. (A) Both ATP and ATPγS are able to support degradation of denatured ovalbumin but not native ovalbumin (2.5 μM). (B) Degradation of native GFPssrA requires ATP hydrolysis, but degradation of acid-denatured GFPssrA only requires ATP binding (ATPγS). One μM native or acid-denatured (AD) GFPssrA was incubated with 1 μg of 20S, 4 μg PAN with the nucleotides (1 mM) in a 0.1 ml reaction volume for 15 min. Degradation in both experiments was assayed with fluorescamine.

unfolded. When translocation proceeds by this “passive” mechanism, unfolding must occur through another mechanism requiring ATP hydrolysis. Also in earlier experiments, we demonstrated that unfolding of GFP occurs even when translocation of GFPssrA through the ATPase ring was blocked by derivatization with a large globular domain (Navon and Goldberg, 2001; Smith et al., 2005). These experiments imply that unfolding can occur in the absence of translocation, by a mechanism other than ATP hydrolysis-driven translocation. Thus, two very different types of experiments demonstrate that unfolding and translocation can be dissociated from one another: (1) the process of unfolding requires ATP hydrolysis but does not require substrate translocation, (2) translocation requires unfolding, but does not require ATP hydrolysis.

### 7.2. Brownian ratchet mechanism for translocation

Since protein translocation into the 20S particle does not require ATP hydrolysis it appears to be a “passive” process in which PAN (or the 19S ATPases) in the ATP-bound state facilitates the diffusion of the bound unfolded protein into the 20S particle. Such a process can be unidirectional if there is a mechanism that prevents retrograde movement of the substrate out of the proteasome, such as a Brownian ratchet (Glick, 1995; Pfanner and Meijer, 1995). A growing body of evidence has implicated “Brownian ratchets” in many unidirectional translocation processes initially thought to be “active” (i.e., linked to ATP hydrolysis) including Transcription Elongation (Bar-Nahum et al., 2005), movement of collagenase (Saffarian et al., 2004), kinesin translocation (Nishiyama et al., 2002), and transport across the ER and mitochondrial membranes (Matlack et al., 1999), for example where the Hsp70 homolog BIP in the ER binds the protein being translocated, blocking its retrograde movement, and then dissociates in an ATP-hydrolysis driven cycle.

D.M. Smith et al. / Journal of Structural Biology 156 (2006) 72–83

Passive translocation model

Antichamber  
Central chamber  
Antichamber  

Substrate binding  
to first site.  

Upstream regions  
binding to other sites.  

Hydrolysis at first site  
while second site holds.  

Fig. 5. Model depicting one possible “Brownian ratchet” mechanism for passive substrate translocation, whereby diffusion is biased inward (unidirectional) due to the architecture of the 20S proteasome and the simultaneous binding of different sections of a linear polypeptide to multiple sites.

structural and catalytic properties, a linear polypeptide could be bound to multiple active sites at any one moment, and while certain segments are being cleaved, upstream regions could bind covalently or non-covalently to other sites in the 20S preventing backward diffusion of the protein (Fig. 5). This model would also allow for those special cases where partial degradation of one terminus occurs (e.g., NFκB) while sparing upstream structured regions (Rape and Jentsch, 2002), since such untranslocatable structured regions upstream would overcome the inward bias after distal regions had been degraded, thus permitting diffusion outward.

These findings also indicate that after a polypeptide binds to PAN or the 19S ATPases and is unfolded, its diffusion potential provides sufficient driving force for translocation into the 20S. Protein substrates bind preferentially to PAN in its ATP-bound form (Benaroudj and Goldberg, 2000), the same form that facilitates passive diffusion into the 20S. The translocation process thus resembles a type of “carrier-mediated” diffusion, in which the ATPase in its ATP-bound form binds the linearized polypeptide so as to increase the local concentration of the substrate at the mouth of the translocation channel, and thus to facilitate its diffusion through the ATPase ring and the opened gate in the 20S. This translocation process involves the dynamic interaction of the substrate, ATPase and open-gated 20S particle, and it appears to be the rate-limiting step in the degradation of many (perhaps most) proteins by the PAN-20S complex (Benaroudj et al., 2003).

### 7.3. Direction of substrate translocation

To reach the active sites within the 20S particle, substrates have to traverse the narrow axial pore formed by the α ring, in a linear form, presumably after passing through the pore in the ATPase ring of PAN or the base of the 19S complex. This model raises the question of whether polypeptide chains have a preferred or exclusive orientation to translocate into the 20S proteasome. To determine whether a polypeptide chain enters the 20S particle by its N- or C-terminus or by an internal loop, we attached bulky moieties to the terminus of protein substrates that can prevent the translocation of modified terminus through the pore of PAN and the 20S (Navon and Goldberg, 2001). Surprisingly, GFPssrA was found to be translocated exclusively in a C-to-N orientation (with C-terminus entering first). However, different substrates were found to exhibit different translocation directionalties. While some proteins (Maltose Binding Protein and GFPssrA) are transported into the proteasome by their C-terminus, others (Casein) are exclusively translocated from their N-terminus, and some (Calmodulin) could be translocated starting with either terminus.

PAN  
-Complex formation  
-20S gate opening  

20S  
Binding  
(ATPγs)  

ATP  
Protein  
binding  

ATP  
ADP  
Pi  

-Unfolding  
-Translocation  

-Binding  
(ATPγs)  

-ATPase Activation  

Fig. 6. The multiple ATP-dependent steps in protein degradation and the requirement for nucleotide binding and hydrolysis (see text for details).

minus (or by an internal loop). By contrast, the isolated 20S showed no such directional preference. Thus, the orientation of entry seems to be a property of the substrate’s termini and its interaction with the ATPase. In related experiments with eukaryotic 26S proteasomes, it has been observed that different substrates appear to enter by different extremities (Prakash et al., 2004; Zhang and Coffino, 2004; Navon et al., 2006, in preparation), and some even by an internal loop (Liu et al., 2003). However, these observations were made with crude cell lysates, or with pure 26S proteasomes and substrates that do not require ubiquitin conjugation such as casein (Navon et al., 2006, in preparation), p21<sup>cip1</sup> and α-synuclein (Liu et al., 2003). The influence of ubiquitination on the recognition of substrate by the ATPases as well as on the unfolding process are likely to strongly influence the directionality of substrate translocation.

## 8. Conclusion

A more complete understanding of the molecular mechanisms involved in protein degradation by the proteasome will be aided by detailed structural information about the ATP and ADP-bound forms of ATPase complex, as well as the delineation of its ATP hydrolysis cycle. Presumably X-ray crystallography will first be achieved for PAN, whose many benefits for study have been summarized here. Already however, study of the PAN:20S complex has allowed us to learn much concerning the multiple steps in this process and about the multiple roles of ATP in protein degradation. Our present understanding of this process is illustrated by the reaction scheme in Fig. 6. (i) Nucleotide binding to PAN promotes the association between the ATPase ring and the 20S complex, (ii) complex formation triggers gate-opening in the α ring, (iii) the binding of the protein substrate induces a conformational change in PAN that activates ATP hydrolysis, (iv) repeated cycles of ATP hydrolysis catalyze unfolding of globular proteins, (v) the unfolded polypeptide can diffuse through the ATPase ring (in its ATP-bound form) and the open gates in the α ring, (vi) the polypeptide in the central chamber of the 20S particle is processively degraded to small peptides.

These findings raise several important unanswered questions regarding the mechanism of proteasomal protein degradation that is regulated by ATPase complexes. (1) How does ATP binding promote a conformational change in the ATPase ring that allows association with the 20S proteasome, (2) how does PAN open the gate in the 20S proteasome, (3) are there factors that increase the affinity of PAN for the 20S, (4) how do these ATPases mechanistically cause unfolding of globular proteins and how is ssrA involved, (5) does ubiquitin perform the same role in the eukaryotic 26S particle as ssrA does for the PAN–20S complex, (6) how are these steps in eukaryotes integrated with the binding and disassembly of the polyubiquitin chain?

### References

Akopian, T.N., Kisselev, A.F., Goldberg, A.L., 1997. Processive degradation of proteins and other catalytic properties of the proteasome from *Thermoplasma acidophilum*. J. Biol. Chem. 272, 1791–1798.

Bajorek, M., Finley, D., Glickman, M.H., 2003. Proteasome disassembly and downregulation is correlated with viability during stationary phase. Curr. Biol. 13, 1140–1144.

Bar-Nahum, G., Epshtein, V., Ruckenstein, A.E., Rafikov, R., Mustaev, A., Nudler, E., 2005. A ratchet mechanism of transcription elongation and its control. Cell 120, 183–193.

Baumeister, W., Walz, J., Zuhl, F., Seemuller, E., 1998. The proteasome: paradigm of a self-compartmentalizing protease. Cell 92, 367–380.

Benaroudj, N., Goldberg, A.L., 2000. PAN, the proteasome activating nucleotidase from archaeabacteria, is a molecular chaperone which unfolds protein substrate. Nat. Cell Biol. 2, 833–839.

Benaroudj, N., Tarcsa, E., Cascio, P., Goldberg, A.L., 2001. The unfolding of substrates and ubiquitin-independent protein degradation by proteasomes. Biochimie 83, 311–318.

Benaroudj, N., Zwickl, P., Seemuller, E., Baumeister, W., Goldberg, A.L., 2003. ATP hydrolysis by the proteasome regulatory complex PAN serves multiple functions in protein degradation. Mol. Cell 11, 69–78.

Bochtler, M., Hartmann, C., Song, H.K., Bourenkov, G.P., Bartunik, H.D., Huber, R., 2000. The structures of HsIU and the ATP-dependent protease HsIU-HsIV. Nature 403, 800–805.

Braun, B.C., Glickman, M., Kraft, R., Dahlmann, B., Kloetzel, P.M., Finley, D., Schmidt, M., 1999. The base of the proteasome regulatory particle exhibits chaperone-like activity. Nat. Cell Biol. 1, 221–226.

Bult, C.J., White, O., Olsen, G.J., Zhou, L., Fleischmann, R.D., Sutton, G.G., Blake, J.A., FitzGerald, L.M., Clayton, R.A., Gocayne, J.D., et al., 1996. Complete genome sequence of the methanogenic archaeon, *Methanococcus jannaschii*. Science 273, 1058–1073.

Cascio, P., Call, M., Petre, B.M., Walz, T., Goldberg, A.L., 2002. Properties of the hybrid form of the 26S proteasome containing both 19S and PA28 complexes. EMBO J. 21, 2636–2645.

Chung, C.H., Goldberg, A.L., 1981. The product of the lon (capR) gene in *Escherichia coli* is the ATP-dependent protease, protease La. Proc. Natl. Acad. Sci. USA 78, 4931–4935.

Ciechanover, A., 2005. Proteolysis: from the lysosome to ubiquitin and the proteasome. Nat. Rev. Mol. Cell Biol. 6, 79–87.

Confalonieri, F., Duguet, M., 1995. A 200-amino acid ATPase module in search of a basic function. Bioessays 17, 639–650.

Coux, O., Tanaka, K., Goldberg, A.L., 1996. Structure and functions of the 20S and 26S proteasomes. Annu. Rev. Biochem. 65, 801–847.

Dahlmann, B., Kopp, F., Kuehn, L., Niedel, B., Pfeifer, G., Hegerl, R., Baumeister, W., 1989. The multicatalytic proteinase (prosome) is ubiquitous from eukaryotes to archaebacteria. FEBS Lett. 251, 125–131.

Etlinger, J.D., Goldberg, A.L., 1977. A soluble ATP-dependent proteolytic system responsible for the degradation of abnormal proteins in reticulocytes. Proc. Natl. Acad. Sci. USA 74, 54–58.

Ferrell, K., Wilkinson, C.R.M., Dubiel, W., Gordon, C., 2000. Regulatory subunit interactions of the 26S proteasome, a complex problem [Review]. Trends Biochem. Sci. 25, 83–88.

Forster, A., Masters, E.I., Whitby, F.G., Robinson, H., Hill, C.P., 2005. The 1.9 Å structure of a proteasome-11S activator complex and implications for proteasome-PAN/PA700 interactions. Mol. Cell 18, 589–599.

Forster, A., Whitby, F.G., Hill, C.P., 2003. The pore of activated 20S proteasomes has an ordered 7-fold symmetric conformation. EMBO J. 22, 4356–4364.

Gerega, A., Rockel, B., Peters, J., Tamura, T., Baumeister, W., Zwickl, P., 2005. VAT, the thermoplasma homolog of mammalian p97/VCP, is an N domain-regulated protein unfoldase. J. Biol. Chem. 280, 42856–42862.

Glick, B.S., 1995. Can Hsp70 proteins act as force-generating motors? Cell 80, 11–14.

Glickman, M.H., Ciechanover, A., 2002. The ubiquitin-proteasome proteolytic pathway: destruction for the sake of construction. Physiol. Rev. 82, 373–428.

Goldberg, A.L., 2005. Nobel committee tags ubiquitin for distinction. Neuron 45, 339–344.

Goldberg, A.L., St. John, A.C., 1976. Intracellular protein degradation in mammalian and bacterial cells: part 2. Annu. Rev. Biochem. 45, 747–803.

Gorbea, C., Taillandier, D., Rechsteiner, M., 1999. Assembly of the regulatory complex of the 26S proteasome. Mol. Biol. Rep. 26, 15–19.

Gottesman, S., 1996. Proteases and their targets in *Escherichia coli*. Annu. Rev. Genet. 30, 465–506.

Gottesman, S., Roche, E., Zhou, Y., Sauer, R.T., 1998. The ClpXP and ClpAP proteases degrade proteins with carboxy-terminal peptide tails added by the SsrA-tagging system. Genes Dev. 12, 1338–1347.

Groll, M., Bajorek, M., Kohler, A., Moroder, L., Rubin, D.M., Huber, R., Glickman, M.H., Finley, D., 2000. A gated channel into the proteasome core particle. Nat. Struct. Biol. 7, 1062–1067.

Groll, M., Brandstetter, H., Bartunik, H., Bourenkow, G., Huber, R., 2003. Investigations on the maturation and regulation of archaeal proteasomes. J. Mol. Biol. 327, 75–83.

Groll, M., Ditzel, L., Lowe, J., Stock, D., Bochtler, M., Bartunik, H.D., Huber, R., 1997. Structure Of 20S proteasome from yeast at 2.4 Ångstrom resolution. Nature 386, 463–471.

Groll, M., Huber, R., 2003. Substrate access and processing by the 20S proteasome core particle. Int. J. Biochem. Cell Biol. 35, 606–616.

Hough, R., Pratt, G., Rechsteiner, M., 1987. Purification of two high molecular weight proteases from rabbit reticulocyte lysate. J. Biol. Chem. 262, 8303–8313.

Huang, H., Goldberg, A.L., 1997. Proteolytic activity of the ATP-dependent protease HslVU can be uncoupled from ATP hydrolysis. J. Biol. Chem. 272, 21364–21372.

Kajava, A.V., 2002. What curves alpha-solenoids? Evidence for an alpha-helical toroid structure of Rpn1 and Rpn2 proteins of the 26S proteasome. J. Biol. Chem. 277, 49791–49798.

Kenniston, J.A., Baker, T.A., Fernandez, J.M., Sauer, R.T., 2003. Linkage between ATP consumption and mechanical unfolding during the protein processing reactions of an AAA+ degradation machine. Cell 114, 511–520.

Kim, Y.I., Levchenko, I., Fraczkowska, K., Woodruff, R.V., Sauer, R.T., Baker, T.A., 2001. Molecular determinants of complex formation between Clp/Hsp100 ATPases and the ClpP peptidase. Nat. Struct. Biol. 8, 230–233.

Kisselev, A.F., Akopian, T.N., Goldberg, A.L., 1998. Range of sizes of peptide products generated during degradation of different proteins by archaeal proteasomes. J. Biol. Chem. 273, 1982–1989.

Kisselev, A.F., Songyang, Z., Goldberg, A.L., 2000. Why does threonine, and not serine, function as the active site nucleophile in proteasomes. J. Biol. Chem. 275, 14831–14837.

Kohler, A., Cascio, P., Leggett, D.S., Woo, K.M., Goldberg, A.L., Finley, D., 2001. The axial channel of the proteasome core particle is gated by the Rpt2 ATPase and controls both substrate entry and product release. Mol. Cell 7, 1143–1152.

Lee, C., Schwartz, M.P., Prakash, S., Iwakura, M., Matouschek, A., 2001. ATP-dependent proteases degrade their substrates by processively unraveling them from the degradation signal. Mol. Cell 7, 627–637.

Liu, C.W., Corboy, M.J., DeMartino, G.N., Thomas, P.J., 2003. Endoproteolytic activity of the proteasome. Science 299, 408–411.

Lowe, J., Stock, D., Jap, B., Zwickl, P., Baumeister, W., Huber, R., 1995. Crystal structure of the 20S proteasome from the archaeon *T. acidophilum* at 3.4 Å resolution. Science 268, 533–539.

Lupas, A.N., Martin, J., 2002. AAA proteins. Curr. Opin. Struct. Biol. 12, 746–753.

Ma, C.P., Willy, P.J., Slaughter, C.A., DeMartino, G.N., 1993. PA28, an activator of the 20S proteasome, is inactivated by proteolytic modification at its carboxyl terminus. J. Biol. Chem. 268, 22514–22519.

Matlack, K.E., Misselwitz, B., Plath, K., Rapoport, T.A., 1999. BiP acts as a molecular ratchet during posttranslational transport of prepro-alpha factor across the ER membrane. Cell 97, 553–564.

Matouschek, A., 2003. Protein unfolding—an important process in vivo? Curr. Opin. Struct. Biol. 13, 98–109.

Maurizi, M.R., Singh, S.K., Thompson, M.W., Kessel, M., Ginsburg, A., 1998. Molecular properties of ClpAP protease of *Escherichia coli*: ATP-dependent association of ClpA and clpP. Biochemistry 37, 7778–7786.

Murakami, K., Voellmy, R., Goldberg, A.L., 1979. Protein degradation is stimulated by ATP in extracts of *Escherichia coli*. J. Biol. Chem. 254, 8194–8200.

Murakami, Y., Matsufuji, S., Hayashi, S., Tanahashi, N., Tanaka, K., 2000. Degradation of ornithine decarboxylase by the 26S proteasome. Biochem. Biophys. Res. Commun. 267, 1–6.

Navon, A., Goldberg, A.L., 2001. Proteins are unfolded on the surface of the ATPase ring before transport into the proteasome. Mol. Cell 8, 1339–1349.

Navon, A., Goodman, A., Goldberg, A.L., 2006. The terminal sequences of a protein determine the direction of translocation into the proteasome and whether a distant globular domain is degraded (Manuscript in preparation).

Nishiyama, M., Higuchi, H., Yanagida, T., 2002. Chemomechanical coupling of the forward and backward steps of single kinesin molecules. Nat. Cell Biol. 4, 790–797.

Ogura, T., Tanaka, K., 2003. Dissecting various ATP-dependent steps involved in proteasomal degradation. Mol. Cell 11, 3–5.

Patel, S., Latterich, M., 1998. The AAA team: related ATPases with diverse functions. Trends Cell Biol. 8, 65–71.

Pfanner, N., Meijer, M., 1995. Protein sorting. Pulling in the proteins. Curr. Biol. 5, 132–135.

Prakash, S., Tian, L., Ratliff, K.S., Lehotzky, R.E., Matouschek, A., 2004. An unstructured initiation site is required for efficient proteasome-mediated degradation. Nat. Struct. Mol. Biol. 11, 830–837.

Rape, M., Jentsch, S., 2002. Taking a bite: proteasomal protein processing. Nat. Cell Biol. 4, E113–E116.

Rohrwild, M., Pfeifer, G., Santarius, U., Muller, S.A., Huang, H.C., Engel, A., Baumeister, W., Goldberg, A.L., 1997. The ATP-dependent HslVU protease from *Escherichia coli* is a four-ring structure resembling the proteasome. Nat. Struct. Biol. 4, 133–139.

Rubin, D.M., Glickman, M.H., Larsen, C.N., Dhruvakumar, S., Finley, D., 1998. Active site mutants in the six regulatory particle ATPases reveal multiple roles for ATP in the proteasome. EMBO J. 17, 4909–4919.

Saffarian, S., Collier, I.E., Marmer, B.L., Elson, E.L., Goldberg, G., 2004. Interstitial collagenase is a Brownian ratchet driven by proteolysis of collagen. Science 306, 108–111.

Sauer, R.T., Bolon, D.N., Burton, B.M., Burton, R.E., Flynn, J.M., Grant, R.A., Hersch, G.L., Joshi, S.A., Kenniston, J.A., Levchenko, I., et al., 2004. Sculpting the proteome with AAA(+) proteases and disassembly machines. Cell 119, 9–18.

Smith, D.M., Kafri, G., Cheng, Y., Ng, D., Walz, T., Goldberg, A.L., 2005. ATP binding to PAN or the 26S ATPases causes association with the 20S proteasome, gate opening, and translocation of unfolded proteins. Mol. Cell 20, 687–698.

Song, H.K., Hartmann, C., Ramachandran, R., Bochtler, M., Behrendt, R., Moroder, L., Huber, R., 2000. Mutational studies on HslU and its docking mode with HslV. Proc. Natl. Acad. Sci. USA 97, 14103–14108.

Sousa, M.C., Trame, C.B., Tsuruta, H., Wilbanks, S.M., Reddy, V.S., McKay, D.B., 2000. Crystal and solution structures of an HslUV protease-chaperone complex. Cell 103, 633–643.

Strickland, E., Hakala, K., Thomas, P.J., DeMartino, G.N., 2000. Recognition of misfolding proteins by PA700, the regulatory subcomplex of the 26S proteasome. J. Biol. Chem. 275, 5565–5572.

Tanaka, K., Waxman, L., Goldberg, A.L., 1983. ATP serves two distinct roles in protein degradation in reticulocytes, one requiring and one independent of ubiquitin. J. Cell Biol. 96, 1580–1585.

Vigneron, N., Stroobant, V., Chapiro, J., Ooms, A., Degiovanni, G., Morel, S., vanderBruggen, P., Boon, T., VandenEynde, B.J., 2004. An antigenic peptide produced by peptide splicing in the proteasome. Science 304, 587–590.

Voges, D., Zwickl, P., Baumeister, W., 1999. The 26S proteasome: a molecular machine designed for controlled proteolysis. Annu. Rev. Biochem. 68, 1015–1068.

Walz, J., Erdmann, A., Kania, M., Typke, D., Koster, A.J., Baumeister, W., 1998. 26S proteasome structure revealed by three-dimensional electron microscopy. J. Struct. Biol. 121, 19–29.

Wang, W.L., Chevray, P.M., Nathans, D., 1996. Mammalian Sug1 and C-Fos in the nuclear 26s proteasome. Proc. Natl. Acad. Sci. USA 93, 8236–8240.

Wang, J., Song, J.J., Seong, I.S., Franklin, M.C., Kamtekar, S., Eom, S.H., Chung, C.H., 2001. Nucleotide-dependent conformational changes in a protease-associated ATPase HsIU. Structure 9, 1107–1116.

Waxman, L., Fagan, J.M., Goldberg, A.L., 1987. Demonstration of two distinct high molecular weight proteases in rabbit reticulocytes, one of which degrades ubiquitin conjugates. J. Biol. Chem. 262, 2451–2457.

Weber-Ban, E.U., Reid, B.G., Miranker, A.D., Horwich, A.L., 1999. Global unfolding of a substrate protein by the Hsp100 chaperone ClpA. Nature 401, 90–93.

Whitby, F.G., Masters, E.I., Kramer, L., Knowlton, J.R., Yao, Y., Wang, C.C., Hill, C.P., 2000. Structural basis for the activation of 20S proteasomes by 11S regulators. Nature 408, 115–120.

Yoo, S.J., Seol, J.H., Seong, I.S., Kang, M.S., Chung, C.H., 1997. ATP binding, but not its hydrolysis, is required for assembly and proteolytic activity of the HslVU protease in *Escherichia coli*. Biochem. Biophys. Res. Commun. 238, 581–585.

Zhang, M., Coffino, P., 2004. Repeat sequence of Epstein–Barr virus-encoded nuclear antigen 1 protein interrupts proteasome substrate processing. J. Biol. Chem. 279, 8635–8641.

Zwickl, P., Ng, D., Woo, K.M., Klenk, H.P., Goldberg, A.L., 1999. An archaeobacterial ATPase, homologous to ATPases in the eukaryotic 26S proteasome, activates protein breakdown by 20S proteasomes. J. Biol. Chem. 274, 26008–26014.
