
# REVIEWS

---

## IRFs: master regulators of signalling by Toll-like receptors and cytosolic pattern-recognition receptors

### Kenya Honda and Tadatsugu Taniguchi

**Abstract** | The interferon-regulatory factor (IRF) family of transcription factors was initially found to be involved in the induction of genes that encode type I interferons. IRFs have now been shown to have functionally diverse roles in the regulation of the immune system. Recently, the crucial involvement of IRFs in innate and adaptive immune responses has been gaining much attention, particularly with the discovery of their role in immunoregulation by Toll-like receptors and other pattern-recognition receptors.

---

### Helix–turn–helix motif
A structural motif that can bind DNA. It comprises two α-helices joined by a short strand of amino acids, and it is found in many proteins that regulate gene expression.

### IFN-stimulated response element (ISRE)
A common DNA motif that is found in the promoters of genes that are regulated by type I interferons (IFNs). It is bound by IFN-regulatory factors (IRFs) and was initially known as the IRF enhancer (IRF-E). The consensus sequence is GAAANNGAAAG/CT/C, where N denotes any nucleotide.

### Type I IFNs
A family of cytokines that includes interferon-α (IFNα; which is encoded by 13 functional genes in humans and 14 in mice) and IFNβ.

---

Prompt and regulated cellular responses are central to immunity. These responses are coordinated by intricate gene-regulatory networks, which ensure rapid alterations in gene-expression programmes. Such regulatory networks are controlled by the functional diversity of the transcription factors that are present in a stimulated cell, and this diversity is influenced by the post-translational modification of these transcription factors and/or by the interaction of these transcription factors with other transcription factors¹. In this context, the prototypical transcription factor that has been extensively studied is nuclear factor-κB (NF-κB), and the importance of NF-κB in the gene-regulatory networks in immune responses, including the induction of the genes that encode interferon-β (IFNβ) and pro-inflammatory cytokines, is well documented²,³. More recently, members of the IFN-regulatory factor (IRF) family of transcription factors have been shown to have markedly diverse roles in the gene-regulatory networks in the immune system.

The mammalian IRF family comprises nine members: **IRF1**, **IRF2**, **IRF3**, **IRF4** (also known as LSIRF, PIP or ICSAT), **IRF5**, **IRF6**, **IRF7**, **IRF8** (also known as ICSBP) and **IRF9** (also known as ISGF3γ)⁴⁵. Each IRF contains a well-conserved DNA-binding domain of ~120 amino acids; this is located at the amino terminus and forms a helix–turn–helix motif. This region recognizes a consensus DNA sequence that is known as the IFN-stimulated response element (ISRE; also known as IRF-E)⁴ because it was first identified in the promoters of genes that are induced by signalling mediated by type I IFNs (that is, IFNα and IFNβ)⁶⁻⁸. ISREs are also found in the promoters of the genes that encode the type I IFNs themselves, as well as in the promoters of many other genes that are involved in immunity and oncogenesis. By contrast, the carboxy-terminal regions of IRFs, except IRF1 and IRF2, show homology to the C-terminal domains of the SMAD (mothers against decapentaplegic homologue) family of transcription factors, which mediates the response to the cytokine transforming growth factor-β. These C-terminal regions might be regulatory regions, containing protein-binding domains⁹. Gene-disruption studies of most of the genes that encode IRFs have been carried out, and these studies, which are described elsewhere⁴⁵, show that IRFs have distinct roles in the development and function of immune cells (TABLE 1).

IRFs gained much attention as essential regulators of the activation of immune cells with the discovery of pattern-recognition receptors (PRRs), which seem to function as a ‘platform’ that links innate and adaptive immune responses. PRRs recognize pathogen-associated molecular patterns (PAMPs), such as lipopolysaccharide (LPS) and viral nucleic acids, and the engagement of PRRs activates particular gene-induction programmes in the host¹⁰. The differences in the programmes that are elicited by signalling through distinct PRRs might be associated with the type and the location of the invading pathogen, and could therefore lead to appropriate ‘tuning’ of the immune response for clearance of the specific pathogen. Two classes of PRR have been defined: the transmembrane PRRs, namely Toll-like receptors (TLRs); and the cytosolic PRRs, which include the retinoic-acid-inducible gene I (RIG-I) family (comprising melanoma-differentiation-associated gene 5 (MDA5) and RIG-I), IFN-inducible double-stranded RNA (dsRNA)-dependent protein kinase (PKR), and nucleotide-binding oligomerization domain (NOD) proteins¹⁰,¹¹. Our understanding of how infection elicits immune responses by activating these PRRs and of

---

Department of Immunology,  
Graduate School of Medicine  
and Faculty of Medicine,  
University of Tokyo,  
Hongo 7-3-1, Bunkyo-ku,  
Tokyo 113-0033, Japan.  
Correspondence to T.T.  
e-mail: tada@m.u-tokyo.ac.jp  
doi:10.1038/nri1900

---

© 2006 Nature Publishing Group

Table 1 | Interferon-regulatory-factor-family members and their roles in immune responses

| IRF | Expression | Target gene | Main immune function of target gene | Phenotype of gene-knockout mice | References |
| --- | --- | --- | --- | --- | --- |
| IRF1 | Constitutive, and inducible by IFNγ | NOS2, GBP1 and gp91<sup>PHOX</sup> | Promote antibacterial and antiviral innate immunity | Susceptibility to infection with the intracellular bacteria M. bovis and B. abortus; reduced resistance to infection with the virus EMCV | 61,115,116 |
|  |  | IL12 | Promotes T<sub>H</sub>1-cell responses | Defective T<sub>H</sub>1-cell responses; susceptibility to infection with the parasite L. major and the intracellular bacterium L. monocytogenes | 117 |
|  |  | IL15 | Controls haematopoietic-cell development | Defective NK-cell, NKT-cell and IEL development | 118 |
|  |  | TAP1 and LMP2 | Provide MHC class I molecules with short peptides | Reduced CD8<sup>+</sup>T-cell numbers | 119 |
|  |  | CDKN1A | Controls the cell-cycle and apoptosis | Reduced DNA-damage-induced cell-cycle arrest | 120 |
|  |  | Caspase-1 | Controls apoptosis and processes IL-1β | Reduced DNA-damage-induced apoptosis of mitogen-activated mature T cells | 121 |
| IRF2 | Constitutive | IFN-inducible genes (such as OAS, PKR and IRF7)* | Attenuate type I IFN responses | Severe skin inflammation mediated by CD8<sup>+</sup>T cells; defective CD4<sup>+</sup>DC development | 122,123 |
|  |  | IL12 | Promotes T<sub>H</sub>1-cell responses | Susceptibility to infection with the parasite L. major | 124 |
| IRF3 | Constitutive | IFNB, IFNA4, IFIT1, CXCL9, CXCL10 and CCL5 | Promote antibacterial and antiviral innate immunity | Reduced resistance to infection with the virus EMCV; increased ability to clear infection with the intracellular bacterium L. monocytogenes; resistance to LPS-induced endotoxic shock | 27,50,57,60 |
| IRF4 | Constitutive, and inducible by TLR ligation | IL4 and GATA3 | Controls T-cell function | Defective T<sub>H</sub>2-cell differentiation | 125,126 |
|  |  | IGK and CD23 | Control B-cell development and function | Defective mature B-cell function; defective immunoglobulin production | 127,128 |
|  |  | ND | Controls DC development | Defective CD4<sup>+</sup>DC development | 129 |
| IRF5 | Constitutive, and inducible by type I IFNs and by TLR ligation | IL6, IL12, TNF, NFKBIZ and CXCL2 | Promote inflammation | Resistance to LPS-or CpG-containing-DNA-induced shock | 18 |
| IRF6 | ND | ND | ND | ND | — |
| IRF7 | Constitutive, and inducible by type I IFNs | IFNB and IFNA | Promote antiviral innate immunity | Susceptibility to infection with the viruses EMCV, VSV and HSV; reduced efficiency of CD8<sup>+</sup>T-cell cross-priming | 28 |
| IRF8 | Constitutive, and inducible by IFNγ | IL12 and IL18 | Promote T<sub>H</sub>1-cell responses | Defective T<sub>H</sub>1-cell differentiation; susceptibility to infection with the intracellular bacterium L. monocytogenes and the parasites L. major and T. gondii | 130–132 |
|  |  | ND | Controls haematopoietic-cell development | Development of chronic-myeloid-leukaemia-like disease; defective plasmacytoid-DC and CD8<sup>+</sup>DC development | 133,134 |
| IRF9 | Constitutive | IFN-inducible genes (such as OAS, PKR and IRF7) | Mediate type I IFN responses | Susceptibility to infection with the viruses EMCV, VSV and HSV | 135 |

*Interferon (IFN)-inducible genes are negatively regulated by IFN-regulatory factor 2 (IRF2). B. abortus, Brucella abortus; CCL, CC-chemokine ligand; CD23, also known as low-affinity receptor for IgE (FcεRII); CDKN1A, cyclin-dependent kinase inhibitor 1A (also known as WAF1 or p21); CXCL, CXC-chemokine ligand; DC, dendritic cell; EMCV, encephalomyocarditis virus; GATA3, GATA-binding protein 3; GBP1, guanylate-binding protein 1; gp91<sup>PHOX</sup>, glycoprotein 91 (phagocyte oxidase); HSV, herpes simplex virus; IEL, intraepithelial lymphocyte; IFIT1, IFN-induced protein with tetratricopeptide repeats 1; IGK, immunoglobulin κ light chain; IL, interleukin; L. major, Leishmania major; L. monocytogenes, Listeria monocytogenes; LMP2, low-molecular-mass protein 2; LPS, lipopolysaccharide; M. bovis, Mycobacterium bovis; ND, not determined; NFKBIZ, inhibitor of nuclear factor-κB (IκB)-ζ; NK, natural killer; NKT, natural killer T; NOS2, nitric-oxide synthase 2; OAS, 2′,5′-oligoadenylate synthetase; PKR, IFN-inducible double-stranded-RNA-dependent protein kinase; TAP1, transporter associated with antigen processing 1; T. gondii, Toxoplasma gondii; T<sub>H</sub>, T helper; TLR, Toll-like receptor; TNF, tumour-necrosis factor; VSV, vesicular stomatitis virus.

how such innate immune responses are linked to adaptive
immune responses has progressed remarkably in the past
few years. Depending on the nature of the pathogen and
the cell type, these recognition systems are differentially
activated, thereby eliciting distinct immune responses.
As we outline in this Review, many IRFs function as
important mediators of these responses.

### IRFs in cytosolic PRR signalling

The mechanisms that underlie the induction of type I
IFN genes have been extensively studied in the context
of immunity to viruses. It has long been known that most viruses — for example, Newcastle disease virus
(NDV; a negative-sense single-stranded RNA (ssRNA)
virus), vesicular stomatitis virus (VSV; a negative-sense
ssRNA virus) and encephalomyocarditis virus (EMCV;
a positive-sense ssRNA virus) — commonly elicit the
induction of type I IFN genes in most cell types. Recent
studies have revealed that this long-discovered pathway
of type I IFN gene induction is initiated by the activation
of cytosolic PRRs<sup>12,13</sup>. Therefore, it can be called the classical pathway or the cytosolic pathway, in contrast
to the recently discovered TLR pathway of type I IFN
gene induction, which operates in distinct cell types
(described later). As depicted in FIG. 1, the mechanisms
that underlie the cytosolic pathway are becoming clearer,
and in this section, we describe these mechanisms,
focusing on the roles of IRFs.

#### Pathogen-associated molecular patterns (PAMPs)

Molecular patterns that are found in pathogens but not in mammalian cells. Examples include terminally mannosylated and polymannosylated compounds (which bind the mannose receptor) and various microbial components, such as bacterial lipopolysaccharide, hypomethylated DNA, flagellin and double-stranded RNA (all of which bind Toll-like receptors).

#### Nucleotide-binding oligomerization-domain proteins (NOD proteins)

Members of a family that includes the apoptosis regulator APAF1 (apoptotic-protease-activating factor 1), mammalian NOD-LRR proteins (also known as NACHT-LRR proteins or CATERPILLERs) and plant disease-resistance gene products. Several NOD proteins have been implicated in the induction of nuclear factor-κB activity and in the activation of caspases.

#### Virus-responsive elements (VREs)

The promoter of the gene that encodes interferon-β (IFNβ) contains at least four regulatory *cis* elements — positive-regulatory domain I (PRDI), PRDII, PRDIII and PRDIV — which are involved in virus-mediated gene induction.

#### IFN-regulatory factors (IRFs)

bind PRDI and PRDIII (which are IFN-stimulated response elements, ISREs), whereas nuclear factor-κB and activator protein 1 (AP1) bind PRDII and PRDIV, respectively. By contrast, the promoters of the genes that encode IFNα contain only IRF-binding elements, and these are known as PRDI-like and PRDIII-like elements (PRD-LEs).

(CREB)-binding protein) or p300 to form a holocomplex
in the nucleus<sup>19–23</sup>. The holocomplex then binds its target
DNA sequence (in type I IFN genes and in certain chemokine genes) (TABLE 1) and presumably alters the local chromatin structure through the histone-acetyltransferase activity of the co-activators. In addition, other transcription factors are recruited to form the enhanceosome (discussed later); consequently, efficient transcription of target genes is initiated.

Unlike IRF3, IRF7 is expressed in small amounts in most cells and is strongly induced by type-I-IFN-mediated signalling. The binding of type I IFNs to the type I IFN receptor results in the activation of a heterotrimeric transcriptional activator known as IFN-stimulated gene factor 3 (ISGF3), which consists of IRF9 and signal transducer and activator of transcription 1 (STAT1) and STAT2, and is responsible for the induction of the IRF7 gene<sup>24,25</sup> (FIG. 2). Similar to IRF3, IRF7 resides in the cytosol and, following viral infection, undergoes serine phosphorylation of its C-terminal region, allowing it to dimerize and translocate to the nucleus. IRF7 can form a homodimer or a heterodimer with IRF3, and these dimers have differential effects on the expression of the type I IFN gene-family members<sup>25,26</sup>. IRF3 is a potent activator of the IFNB gene but not the IFNA genes, except the IFNA4 gene, whereas IRF7 efficiently activates both IFNA and IFNB genes<sup>24,25,27</sup>.

These findings led to the idea that IRF3 is mainly responsible for the initial induction of the IFNB gene, whereas IRF7, the upregulation of which is mediated by type I IFNs themselves, is involved in the late phase of type I IFN gene induction. That is, the induction of the IFNB gene that is mediated by IRF3 initiates a two-step induction of type I IFN genes by a positive-feedback loop (in which type-I-IFN-dependent IRF7 induction ensures the second step), thereby allowing the efficient production of type I IFNs during viral infection. This model of type I IFN production was supported by several lines of evidence; however, the subsequent generation of *Irf7*<sup>−/−</sup> mice allowed a more rigorous assessment of the model. In MEFs from *Irf7*<sup>−/−</sup> mice, the induction of type I IFN genes by ssRNA viruses (VSV and EMCV) is severely impaired<sup>28</sup>, indicating that IRF7 is crucial for the cytosolic pathway of type I IFN gene induction by these viruses and that the contribution of IRF3 is minor in the absence of IRF7. Therefore, a revised positive-feedback model was proposed, in which IRF7, expressed in small amounts, seems to be crucial for activating the initial phase of type I IFN gene induction (FIG. 2). So the homodimer of IRF7 or the heterodimer of IRF7 and IRF3, rather than the homodimer of IRF3, might be more important for the cytosolic pathway of type I IFN gene induction by viruses. It is probable that the IRF3 homodimer participates in the induction of other genes, such as the gene that encodes the chemokine CXC-chemokine ligand 10 (CXCL10)<sup>29</sup>. After the initial activation of type I IFN genes is achieved by IRF7 (and to a lesser extent IRF3), the positive-feedback regulation comes into effect, and the participation of type-I-IFN-induced IRF7 is crucial for the induction of type I IFN genes to occur in full (FIG. 2).

Plasma membrane

RNA  
virus  

Intracellular  
bacterium  

DNA  
virus  

Phagosome  

Recognition of  
dsRNA or dsDNA  

dsDNA  

Cleavage  

HCV NS3–NS4A  

Mitochondrion  

IKK degradation and  
activation of NF-κB  

NF-κB  

p50 p65  

p50 p65  

Pro-inflammatory  
cytokines  

Type I  
IFNs  

Polyubiquitylation  
and degradation  

Mouse embryonic  
fibroblasts  
(MEFs). A well-defined cell type  
that has been widely used to  
identify the consequences of  
ablation or ectopic expression  
of a gene of interest. In  
addition, MEFs are known to  
allow infection with various  
viruses and to express type I  
IFN genes effectively. They  
therefore provide a simple  
model for the study of innate  
immunity to viral infections.  

Latent form  
A protein that is inactive in  
the absence of additional  
modification(s), such as  
phosphorylation or  
ubiquitylation.  

Holocomplex  
A complex that consists of  
subunits, each of which cannot  
carry out a reaction by itself  
but can carry out the reaction  
as a complex.  

Histone acetyltransferase  
A protein that acetylates core  
histones, resulting in important  
regulatory effects on chromatin  
structure and assembly, and on  
gene transcription.  

Figure 1 | Activation of interferon-regulatory factors by cytosolic pattern-recognition receptors. The presence  
of double-stranded RNA (dsRNA) or double-stranded DNA (dsDNA) in the cytosol — following infection with viruses  
or intracellular bacteria — triggers innate immune responses through a specific cytosolic pattern-recognition system.  
The interaction of dsRNA — a replication intermediate of RNA viruses — with the RNA-helicase domain of the pattern-  
recognition receptors RIG-I (retinoic-acid-inducible gene I) or MDA5 (melanoma-differentiation-associated gene 5)  
induces interaction between the caspase-recruitment domain (CARD) of RIG-I or MDA5 and the CARD-like domain  
of their adaptor protein, which is located on the mitochondrial membrane. (The adaptor has four alternative names —  
CARDIF (CARD adaptor inducing interferon-β, IFNβ), IPS1 (IFNB-promoter stimulator 1), MAVS (mitochondrial antiviral  
signalling protein) and VISA (virus-induced signalling adaptor) — and is denoted here as CARDIF/IPS1/MAVS/VISA.)  
This receptor-adaptor interaction results in the activation of TBK1 (TANK (tumour-necrosis-factor-receptor-associated  
factor (TRAF)-family-member-associated nuclear factor-κB (NF-κB) activator)-binding kinase 1). Activated TBK1  
induces the phosphorylation of IFN-regulatory factor 3 (IRF3) and IRF7 on specific serine residues, resulting in their  
homo- or heterodimerization. These dimers then translocate to the nucleus and activate the transcription of type I IFN  
genes (that is, the IFNA and IFNB genes). Cyclophilin B (CYPB) and TRAF3 positively regulate the activation of TBK1-  
activated IRFs, whereas SIKE (suppressor of IkB (inhibitor of NF-κB)-kinase ε) might function as a negative regulator.  
The expression of IRF7, RIG-I and MDA5 is induced by type-I-IFN-mediated signalling, a process that is essential for the  
amplification of this signalling (FIG. 2). PIN1 (cis-trans peptidylprolyl isomerase, NIMA-interacting 1) interacts with  
the phosphorylated form of IRF3 in the nucleus and facilitates polyubiquitin conjugation and proteasome-dependent  
degradation of IRF3, thereby terminating the function of IRF3. IRF7 might be regulated in a similar manner. FADD  
(FAS-associated via death domain), RIP1 (receptor-interacting protein 1) and TRAF6 have been shown to interact with  
CARDIF/IPS1/MAVS/VISA and are implicated in the NF-κB-activation pathway, which results in the induction of pro-  
inflammatory cytokine genes, such as the gene that encodes interleukin-6. LGP2 functions as a negative regulator of  
RIG-I and MDA5. In addition, the hepatitis C virus (HCV) protein NS3–NS4A is an inhibitor of these IRF- and NF-κB-  
signalling pathways, because it cleaves the transmembrane domain of CARDIF/IPS1/MAVS/VISA, thereby releasing  
the protein from the mitochondrial membrane and blocking downstream signalling. After infection with intracellular  
bacteria such as *Listeria monocytogenes* or with DNA viruses, dsDNA might also activate a similar pathway by  
interaction with an as-yet-unidentified receptor and adaptor system. IKK, IkB kinase.

REVIEW S

**Immunophilin family**
A family of *cis–trans* peptidylprolyl isomerases that includes cyclophilins and FK506-binding proteins (FKBPs). These proteins were originally discovered as cellular receptors for immunosuppressive drugs, including cyclosporin A and FK506. The complexes that form between immunophils and their cognate ligands are the functional modules for immunosuppression. Immunophils are now known to function at the crossroads of protein folding and trafficking, and signal transduction.

**IRF kinases and other molecules involved in regulating the function of IRF3 and IRF7.** The virus-activated protein kinases that phosphorylate and thereby activate IRF3 and IRF7 have been identified to be two serine/threonine kinases: TBK1 (TANK (tumour-necrosis factor (TNF)-receptor-associated factor (TRAF)-family-member-associated NF-κB activator)-binding kinase 1; also known as T2K and NAK); and IKKε (inhibitor of NF-κB (IkB) kinase ε; also known as IKKi)³⁰,³¹. Although both TBK1 and IKKε phosphorylate serine residues in the C-terminal (regulatory) region of IRF3 and IRF7 *in vitro*, gene-targeting studies have shown that TBK1 is the main contributor to type I IFN gene induction in virus-infected MEFs (FIG. 1). As discussed later, TBK1 also functions downstream of TLR3 and TLR4 to activate IRF3, whereas IRF7 is activated by other protein kinases during TLR7 and TLR9 signalling³²⁻³⁵.

Cyclophilin B (CYPB), a member of the immunophilin family of *cis–trans* peptidylprolyl isomerases, has also been shown to be involved in virus-mediated phosphorylation of IRF3 (REF. 36). CYPB interacts with IRF3 immediately after viral infection and assists TBK1-mediated phosphorylation and dimerization of IRF3 (FIG. 1). Another peptidylprolyl isomerase, PIN1 (*cis–trans* peptidylprolyl isomerase, NIMA-interacting 1), interacts with the serine-phosphorylated form of IRF3 in the nucleus and facilitates the conjugation of polyubiquitin (presumably Lys48 linked) and the proteasome-dependent degradation of IRF3, thereby terminating the action of IRF3 (REF. 37) (FIG. 1). In addition, the recently identified molecule SIKE (suppressor of IKKε) interacts with both IKKε and TBK1, and might inhibit the activation of IRF3 after viral infection³⁸ (FIG. 1). Although the precise function of CYPB, PIN1 and SIKE remains unclear, it is

---

**a Early phase**

Plasma membrane  
![Diagram](#)

**b Later phase**

![Diagram](#)

Figure 2 | Interferon-regulatory factors in positive-feedback regulation of type I interferon genes. **a** | In the early phase of infection with an RNA or DNA virus, interferon (IFN)-regulatory factor 3 (IRF3) and IRF7 are phosphorylated on specific serine residues, resulting in the homodimerization or heterodimerization of IRF3 and IRF7. These dimers then translocate to the nucleus and induce the expression of chemokines (including CXC-chemokine ligand 10, CXCL10), as well as small amounts of IFNβ (and IFNα). IRF7 has a pivotal role in the induction of type I IFN genes (the IFNB gene and the IFNA genes). IRF3, which also contributes to the induction of type I IFN genes (albeit less than IRF7), contributes to the induction of chemokine genes, such as the gene encoding CXCL10. **b** | Secreted IFNs then bind and activate the type I IFN receptor (a heterodimer of IFNAR1 and IFNAR2) in an autocrine or paracrine manner. This interaction leads to the activation of IFN-stimulated gene factor 3 (ISGF3; a heterotrimer of signal transducer and activator of transcription 1 (STAT1), STAT2 and IRF9), which translocates to the nucleus and induces the transcription of the IRF7 gene. Activation of newly synthesized IRF7, following recognition of viral nucleic acids by pattern recognition receptors (PRRs), leads to the expression of large amounts of IFNβ and many of the IFNα proteins. Therefore, a positive-feedback loop (that is, type-I-IFN-dependent type I IFN gene induction) is operational. Expression of the cytosolic PRRs RIG-I (retinoic-acid-inducible gene I) and MDA5 (melanoma-differentiation-associated gene 5) is also induced by type I IFNs (not shown), thereby contributing to the positive-feedback regulation.
RNA-helicase domain  
A protein domain that is found  
in many RNA-binding proteins  
that are required for mRNA  
synthesis, pre-mRNA splicing,  
ribosome biogenesis and RNA  
decay. This domain can unwind  
double-stranded RNA using  
energy derived from the  
hydrolysis of ATP.

Caspase-recruitment  
domain  
(CARD). A protein domain that  
is found in certain initiator  
caspases (for example,  
mammalian caspase-9) and  
their adaptor proteins (for  
example, apoptotic-protease-  
activating factor 1, APAF1).  
This domain mediates protein-  
protein interaction.

apparent that there might be several molecules involved  
in balancing the functions of IRF3 and IRF7 (FIG. 1).

**Cytosolic PRRs and their adaptor molecules for the  
induction of type I IFN genes.** The cytosolic pathway of  
TBK1-mediated IRF3 and IRF7 activation, which leads to  
type I IFN gene induction is initiated by the intracellular  
recognition of viral PAMPs. The best-characterized PAMP  
is dsRNA, which is produced by many viruses during  
replication. The first sensor of intracellular dsRNA to be  
identified was PKR, the catalytic activity of which is stimu-  
lated by binding dsRNA. However, although PKR con-  
tributes to type I IFN production in response to poly(I:C)  
(polyinosinic-polycytidylic acid), which is a synthetic  
analogue of viral dsRNA, gene-targeting studies in mice  
have shown that PKR is not required for type I IFN gene  
induction in response to viral infection³⁹. Recently, the  
molecules RIG-I and MDA5 have been identified as cyto-  
solic receptors for intracellular viral RNA and synthetic  
dsRNA, and these molecules mediate TLR-independent  
induction of type I IFN genes¹²,¹³,⁴⁰,⁴¹ (FIG. 1).

RIG-I and MDA5 both have a C-terminal DEXD/H-  
box-containing RNA-helicase domain, as well as two  
N-terminal caspase-recruitment domains (CARDs)¹²,¹³.  
The interactions of the RNA-helicase domain of RIG-I  
and MDA5 with viral RNA or synthetic dsRNA induce  
conformational changes and promote downstream sig-  
nalling cascades, leading to the activation of IRF3, IRF7  
and NF-κB. Definitive evidence for the essential role of  
RIG-I and MDA5 in type I IFN gene induction by RNA  
viruses has been obtained by generating mice that are  
deficient in RIG-I or MDA5 (REFS 40,41). RIG-I is essen-  
tial for the activation of IRF3 and NF-κB and for the  
induction of type I IFN genes in response to infection  
with negative-sense ssRNA viruses such as NDV, VSV,  
Sendai virus and influenza virus, as well as positive-  
sense ssRNA viruses such as Japanese encephalitis  
virus. By contrast, MDA5 is crucial for the production  
of type I IFNs in response to infection with positive-  
sense ssRNA viruses such as EMCV, Theiler’s murine  
encephalomyelitis virus and Mengo virus⁴⁰,⁴¹ (TABLE 2).  
Therefore, RIG-I and MDA5 cooperate to ensure an

---

**Table 2 | Pattern-recognition receptors and immunostimulatory motifs in nucleic acids**

| PRR   | Adaptor          | Localization* | IRF activated† | Pathogens or pathogenic form of host nucleic acid                                                                 | Immunostimulatory motif                                                                                          | Refs     |
|-------|------------------|---------------|----------------|-------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|----------|
| RIG-I | CARDIF/IPS1/MAVS/VISA§ | Cytosol      | IRF3, IRF7    | • VSV, NDV, Sendai virus and influenza virus (negative-sense ssRNA viruses)                                      | • Long dsRNA (sequence independent)                                                                             | 12,13,40 |
|       |                  |               |                | • JEV (positive-sense ssRNA virus)                                                                                 |                                                                                                               |          |
| MDA5  | CARDIF/IPS1/MAVS/VISA§ | Cytosol      | IRF3, IRF7, (IRF1) | • EMCV, Theiler’s murine encephalomyelitis virus and Mengo virus (positive-sense ssRNA viruses)                 | • Long dsRNA (sequence independent)                                                                             | 13,41    |
| ND    | ND               | Cytosol       | IRF3          | • *Listeria monocytogenes* and *Legionella pneumophila*                                                          | • B-form dsDNA of >25 base pairs (sequence independent)                                                         | 51,52    |
|       |                  |               |                | • dsDNA viruses (possibly)                                                                                        |                                                                                                               |          |
| TLR3  | TRIF             | Cell surface (on epithelial cells) or endosomes | IRF3, (IRF7) | • Reovirus (dsRNA virus)                                                                                       | • Long dsRNA (sequence independent)                                                                             | 62–64,   |
|       |                  |               |                | • WNV (positive-sense ssRNA virus)                                                                                | • ssRNA without m⁶A or s²U modification                                                                         | 66,67,   |
|       |                  |               |                | • RSV (negative-sense ssRNA virus)                                                                                |                                                                                                               | 103,136  |
|       |                  |               |                | • MCMV (dsDNA virus)                                                                                             |                                                                                                               |          |
|       |                  |               |                | • *Leishmania donovani* and *Schistosoma mansoni*                                                                 |                                                                                                               |          |
|       |                  |               |                | • Virus-infected dying cells                                                                                     |                                                                                                               |          |
| TLR4  | MyD88, TIRAP, TRAM, TRIF | Cell surface | IRF3, (IRF7), IRF5, IRF8 | • Gram-negative bacteria                                                                                         | • Lipopolysaccharide                                                                                             | 10,54    |
|       |                  |               |                | • RSV (negative-sense ssRNA virus)                                                                                | • F protein (of RSV)                                                                                            |          |
| TLR7  | MyD88            | Endosome      | IRF5, IRF7, IRF4 | • VSV and influenza virus (negative-sense ssRNA viruses)                                                          | • U-and G-rich ssRNA without pseudouridine, m⁶A, m⁵C or s²U modifications                                       | 80,85,103, 106,107, 137–139 |
|       |                  |               |                | • Immune complexes of snRNP and IgG                                                                               | • G analogues                                                                                                   |          |
|       |                  |               |                |                                                                                                                 | • Short dsRNA that is U and G rich or contains GUCCUUCAA or UGUGU                                              |          |
| TLR9  | MyD88            | Endosome      | IRF5, IRF7, IRF8, IRF4 | • HSV and MCMV (dsDNA viruses)                                                                                  | • CpG-containing DNA                                                                                           | 83,84,   |
|       |                  |               |                | • Immune complexes of host DNA and IgG                                                                            |                                                                                                               | 108–110, 140 |

*For the stimulation of cytosolic receptors *in vitro*, transfection reagents need to be used so that nucleic acids enter the cells. Also, for *in vitro* experiments, the cationic lipid DOTAP can be used to increase retention of ligands in the the endosome. †Interferon-regulatory factors (IRFs) listed in parentheses are less efficiently activated. IRF4 is a negative regulator. §CARDIF/IPS1/MAVS/VISA is used to denote the adaptor for RIG-I (retinoic-acid-inducible gene I) and MDA5 (melanoma-differentiation-associated gene 5). The adaptor has four alternative names: CARDIF (CARD adaptor inducing interferon-β, IFNβ), IPS1 (IFNB-promoter stimulator 1), MAVS (mitochondrial antiviral signalling protein) and VISA (virus-induced signalling adaptor). dsDNA, double-stranded DNA; dsRNA, double-stranded RNA; EMCV, encephalomyocarditis virus; F protein, fusion protein; HSV, herpes simplex virus; JEV, Japanese encephalitis virus; m⁶A, N⁶-methyladenosine; m⁵C, 5-methylcytidine; MCMV, murine cytomegalovirus; MyD88, myeloid differentiation primary-response protein 88; ND, not determined; NDV, Newcastle disease virus; PRR, pattern-recognition receptor; RSV, respiratory syncytial virus; snRNP, small nuclear ribonucleoprotein; ssRNA, single-stranded RNA; s²U, 2-thiouridine; TIRAP, Toll/interleukin-1 receptor (TIR)-domain-containing adaptor protein; TLR, Toll-like receptor; TRAM, TRIF-related adaptor molecule; TRIF, Toll/interleukin-1 receptor (TIR)-domain-containing adaptor protein inducing IFNβ; VSV, vesicular stomatitis virus; WNV, West Nile virus.

NATURE REVIEWS | IMMUNOLOGY  
© 2006 Nature Publishing Group  
VOLUME 6 | SEPTEMBER 2006 | 649
effective antiviral innate immune response to both negative-sense and positive-sense ssRNA viruses.

In addition to RIG-I and MDA5, another DEXD/H-box-containing RNA helicase — LGP2 — has been identified<sup>13</sup>. Although LGP2 has a DEXD/H-box-containing RNA-helicase domain, it lacks CARDs and other signalling domains<sup>13</sup>, so it might negatively regulate the RIG-I- and MDA5-signalling pathways by competing with these molecules for engagement of viral RNA.

The sensing of incoming viral RNA by RIG-I or MDA5 is connected to downstream signalling pathways and gene activation by an adaptor molecule. This molecule is known as IPS1 (IFNB-promoter stimulator 1)<sup>42</sup>, CARDIF (CARD adaptor inducing IFNβ)<sup>43</sup>, VISA (virus-induced signalling adaptor)<sup>44</sup> and MAVS (mitochondrial antiviral signalling protein)<sup>45</sup> and is denoted CARDIF/IPS1/MAVS/VISA here. CARDIF/IPS1/MAVS/VISA consists of two domains: an N-terminal CARD-like domain, which directly associates with the CARD of RIG-I and MDA5; and a C-terminal effector domain, which recruits signalling proteins such as the adaptor FADD (FAS-associated via death domain), the protein kinase RIP1 (receptor-interacting protein 1) and TRAF6 (REFS 42,44,45). FADD, RIP1 and TRAF6 are implicated in the NF-κB-activation pathway (presumably through the IKK complex) and therefore in the induction of pro-inflammatory cytokine genes, such as the gene that encodes interleukin-6 (IL-6) (FIG. 1). CARDIF/IPS1/MAVS/VISA also mediates the activation of TBK1 and the subsequent phosphorylation of IRF3 and IRF7. Moreover, RIG-I- or MDA5-mediated type I IFN gene induction following viral infection has been shown to be severely impaired in MEFs and macrophages from mice deficient in CARDIF/IPS1/MAVS/VISA<sup>46,47</sup>. However, it is unclear how CARDIF/IPS1/MAVS/VISA activates TBK1, because these two proteins are not found to be associated in immunoprecipitation assays<sup>42,43</sup>. It is interesting to note that MEFs from TRAF3-deficient mice are deficient in type I IFN gene induction by VSV. Given that TRAF3 interacts with TBK1 (REF. 48), TRAF3 might provide a link between IPS1 and TBK1 (FIG. 1).

with *L. monocytogenes*<sup>49,50</sup>. Furthermore, a mutant bacterium lacking listeriolysin O — which is required for the escape of *L. monocytogenes* from the phagosome (where the bacterium initially resides after entering the cell) into the cytosol — does not result in *Ifnb* gene induction. NOD1 and/or NOD2 — cytosolic proteins that bind components of bacterial peptidoglycan and initiate signal transduction through their CARD — have been proposed as candidates for initiating cytosolic signalling in response to *L. monocytogenes* infection. However, these proteins are not required for *Ifnb* gene induction in infected macrophages<sup>49</sup>, and such signalling is also independent of TLRs. More recently, it was shown that *L. monocytogenes* extracts that had been treated with DNase were less able to induce IFNβ production than untreated extracts. By contrast, extracts that had been treated with RNase induced a similar response to that elicited by untreated extracts<sup>51</sup>. *Legionella pneumophila*, a vacuolar bacterial pathogen that injects DNA into the cytosol of the host cell during infection, also activates the TLR- and NOD-independent pathway of *Ifnb* gene induction<sup>51</sup>. These findings indicate that recognition of bacterial DNA has a role in IFNB gene induction<sup>51</sup>. Accordingly, transfection of cells with dsDNA derived from either a pathogen or the host has been shown to induce expression of type I IFN genes, as well as many IFN-inducible genes, by a TLR-independent pathway<sup>51,52</sup>. This pathway requires TBK1, as well as IRF3 (and presumably IRF7)<sup>51</sup> (FIG. 1). It has also been shown that the B form of DNA (which has a right-handed double-helix structure) has a greater stimulatory activity than the Z form of DNA (which has a left-handed double-helix structure)<sup>52</sup>, indicating that recognition of cytosolic dsDNA depends on structure (TABLE 2). Taken together, these results indicate that there is a pathway(s) for the recognition of cytosolic DNA by an as-yet-unidentified receptor(s) and that this pathway results in induction of type I IFN genes (FIG. 1).

**IRFs in TLR signalling**

The mammalian TLR family, the original member of which is TLR4 (REF. 53), consists of germline-encoded transmembrane receptors (encoded by 10 genes in humans and 12 genes in mice). Each TLR recognizes various PAMPs derived from bacteria, viruses, fungi and/or protozoa<sup>10,11</sup>. All TLRs contain an intracellular Toll/IL-1 receptor (TIR) domain, which transmits downstream signals by recruiting one or more TIR-containing adaptor proteins. These adaptors include MyD88 (myeloid differentiation primary-response protein 88), TIRAP (TIR-domain-containing adaptor protein; also known as MAL), TRIF (TIR-domain-containing adaptor protein inducing IFNβ; also known as TICAM1) and TRAM (TRIF-related adaptor molecule; also known as TICAM2)<sup>10</sup>. Signalling through TLRs can be broadly categorized into two pathways: the MyD88-dependent pathway and the MyD88-independent pathway. All TLRs, except TLR3, activate the MyD88-dependent pathway. By contrast, TLR3 and TLR4 activate the MyD88-independent pathway. Both pathways are linked to the activation of two important downstream pathways:

**Other cytosolic initiators of IRF-mediated induction of type I IFN genes.** In addition to the RIG-I and MDA5 pathways, there is evidence of other cytosolic pathways of type I IFN gene induction. For example, DNA viruses, such as herpes simplex virus (HSV; a double-stranded DNA (dsDNA) virus), induce expression of type I IFN genes by a TLR-independent, IRF3- and IRF7-dependent pathway<sup>28</sup>. Therefore, it seems that, similar to responses to RNA viruses, type I IFN production can also be elicited by a cytosolic surveillance system for DNA viruses, and one can envisage that this involves a RIG-I- and MDA5-like molecule (FIG. 1).

Similar to viruses, intracellular bacteria, such as *Listeria monocytogenes*, have a cytosolic phase in their life cycle. The cytosolic recognition of *L. monocytogenes* results in *Ifnb* gene induction through the TBK1–IRF3-signalling pathway, because macrophages from *Tbk1*<sup>-/-</sup> or *Irfl*<sup>-/-</sup> mice do not produce IFNβ when infected

**B form of DNA (B-DNA).** DNA with a right-handed double helix. This is the conformation that is normally seen in solution and is thought to be the conformation of most DNA in vivo. It also formed the basis of the model described by James Watson and Francis Crick.

**Z form of DNA (Z-DNA).** DNA with a left-handed double helix. This conformation occurs as a consequence of methylation. It is found mainly in genes that are undergoing transcription. It is present only transiently, because the cessation of transcription results in rapid conversion of Z-DNA to the B form of DNA (the normal conformation), through the activity of topoisomerases.
the NF-κB pathway, and the mitogen-activated protein kinase (MAPK) pathway. There is now an increasing amount of evidence that several IRF-family members are also activated by the MyD88-dependent and MyD88-independent pathways and contribute to the specific gene-expression programmes induced by TLRs. In this section, we summarize recent findings relating to the activation mechanisms and functions of IRFs in TLR signalling.

**IRF3 and IRF7: activation by ligation of TLR4 and TLR3.** The recognition of PAMPs by TLR4 and TLR3 leads to the recruitment of overlapping sets of signalling molecules. The contribution of IRFs to the response initiated by ligation of these receptors has been extensively studied and is discussed in this section.

TLR4 is a cell-surface receptor that is activated by LPS, from bacteria, as well as by some viral components, such as the fusion (F) protein of respiratory syncytial virus (RSV)⁵⁴. Signalling through TLR4 results in induction of the IFNB gene but not the IFNA genes⁵⁵⁻⁵⁷. Although it is not clear whether TLR4 is involved in antiviral responses, TLR4-mediated IFNB gene induction is essential for the maturation of dendritic cells (DCs) and for the subsequent initiation of adaptive immune responses⁵⁸.

TLR4 signals through two pathways: the MyD88–TIRAP and the TRAM–TRIF pathways¹⁰ (FIG. 3a). Induction of the IFNB gene by signalling through TLR4 is mostly, if not entirely, MyD88 independent and is TRAM–TRIF dependent. By contrast, the TLR4-mediated induction of pro-inflammatory cytokine genes (such as TNF and IL6) depends on both the MyD88–TIRAP and the TRAM–TRIF pathways¹⁰. For production of IFNβ, the TRAM–TRIF pathway involves TBK1 downstream, because *Ifnb* gene induction is abolished in *Tbk1*⁻/⁻ MEFs and macrophages that have been stimulated with LPS³²,³³. In addition, IRF3, rather than IRF7, is essential for this pathway of *Ifnb* gene induction (FIG. 3a), because induction of the *Ifnb* gene in response to LPS is abolished in DCs from *Irf3*⁻/⁻ mice⁵⁷,⁵⁹ but is almost normal in DCs from *Irf7*⁻/⁻ mice²⁸. Furthermore, DCs and macrophages from *Irf3*⁻/⁻ mice show a defect in the induction of other genes, such as those that encode the chemokines CXCL9 and CXCL10 (REFS 57,60). So IFNB gene induction in response to TLR4 ligation is usually mediated by an IRF3 homodimer that has been activated by TBK1, and this mechanism is similar to that of the RIG-I or MDA5 pathway of IFNB gene induction. Interestingly, however, when the expression of IRF7 is upregulated by pretreatment with recombinant IFNβ, exposure to LPS elicits *Ifnb* mRNA expression in DCs from *Irf3*⁻/⁻ mice⁵⁷. Therefore, under certain conditions, IRF7 might also participate in TLR4 signalling, presumably also being activated by TBK1. In addition to IRF3 and IRF7, induction of the nitric-oxide synthase 2 (*Nos2*) gene, an event that is crucial for antibacterial responses, depends on IRF1 in LPS- and IFNγ (type II IFN)-stimulated macrophages⁶¹.

TLR3 is expressed at the cell surface or in endosomes, depending on the cell type. It recognizes the synthetic dsRNA analogue poly(I:C), and it probably recognizes viral dsRNA, either derived from dsRNA viruses such as reovirus or derived from the replication intermediates of ssRNA viruses such as West Nile virus and RSV⁶²⁻⁶⁴ (TABLE 2). In addition, TLR3 has been reported to be involved in defence against infection with murine cytomegalovirus (a dsDNA virus), as well as the parasites *Leishmania donovani* and *Schistosoma mansoni*⁶⁵⁻⁶⁷, indicating that structures other than dsRNA might also be recognized by TLR3. Similar to TLR4, the activation of TLR3 can induce type I IFN expression by a MyD88-independent, TRIF- and TBK1-dependent signalling pathway (but, in the case of TLR3, TRAM is not involved). Indeed, the induction of type I IFN genes by poly(I:C) is severely impaired in macrophages from *Trif*⁻/⁻ or *Tbk1*⁻/⁻ mice³²,³³,⁶⁸. IRF3 has an essential role in this induction. However, type I IFN mRNA is still expressed after stimulation of DCs from *Irf3*⁻/⁻ mice with poly(I:C), albeit in lower amounts, and this residual induction is abolished in DCs that lack both IRF3 and IRF7 (K.H., unpublished observations). Therefore, IRF7 is also required for complete induction of type I IFN genes during TRIF-mediated signalling. In addition, it has been reported that recognition of dsRNA by TLR3 results in the recruitment of phosphatidylinositol 3-kinase (PI3K) to the receptor⁶⁹, and it is thought that PI3K activity might be required for the full activation of IRF3 during TLR3 signalling.

There are certain caveats associated with interpreting experiments in which poly(I:C) is used as a TLR3 ligand, because poly(I:C) can be recognized by other sensors, including RIG-I, MDA5 and PKR. Indeed, type I IFN gene induction in response to poly(I:C) is still observed in cells from *Tlr3*⁻/⁻ mice³⁹. So careful studies are required to dissociate the TLR3-signalling pathway from the RIG-I-, MDA5- and PKR-signalling pathway.

**Linking TLR4- and TLR3-mediated TRIF-dependent signalling with IRF activation.** Several molecules have been shown to interact with TRIF and have been implicated in the subsequent activation of TBK1 and IRF3 (FIG. 3a). Initially, it was shown that TRAF6 interacts, through its TRAF domain, with the N-terminal region of TRIF⁷⁰. However, a study using *Traf6*⁻/⁻ mice indicates that TRAF6 is dispensable for signalling by the TRIF-dependent pathways of TLR3 and TLR4 (REF. 71). Recently, TRAF3, instead of TRAF6, has been shown to form a complex with TRIF, as well as TBK1, and to positively regulate TRIF-dependent IFNB gene induction⁶⁸,⁷². Other TRAF-family members — TRAF1 and TRAF4 — have also been implicated in the TRIF-dependent IFNB gene induction pathway; however, these TRAFs were found to be negative regulators⁷³,⁷⁴. It has also been reported that NAP1 (NAK (TBK1)-associated protein 1) forms a complex with TRIF and is required for inducing the oligomerization and activation of TBK1 (REF. 75). Collectively, these reports indicate that TRAF3 and NAP1 form a complex with TRIF and cooperate to activate TBK1 and, consequently, IRF3 (FIG. 3a).
REVIEWS

IRF7: activation by ligation of the TLR9 subfamily of TLRs. Recently, much attention has been focused on the large amounts of type I IFNs that are produced by plasmacytoid DCs (pDCs)⁷⁶⁻⁷⁸. In contrast to conventional DCs and MEFs, pDCs detect viruses by using TLRs instead of RIG-I or MDA5 (REFS 78–81). More specifically, pDCs use members of the TLR9 subfamily of TLRs (that is, TLR7 and TLR9), which are located in the endosome. TLR9 recognizes unmethylated CpG motifs, a large number of which are present in the

Figure 3 | Toll-like-receptor-mediated pathways for activation of interferon-regulatory factors. Several interferon (IFN)-regulatory factor (IRF)-family members are activated by the MyD88 (myeloid differentiation primary-response protein 88)-dependent and/or -independent pathways of Toll-like receptor (TLR) signalling. These IRFs contribute to the specific gene-expression programme that is induced by TLR4 (which is localized at the plasma membrane) (**a**) and by TLR7 (not shown) and/or TLR9 (which are localized in the endosome) (**b–d**). **a** | TLR4 signals through at least four adaptors: MyD88, TIRAP (Toll/interleukin-1 receptor (TIR)-domain-containing adaptor protein inducing IFNβ) and TRAM (TRIF-related adaptor molecule). TRAM and TRIF mediate the activation of IRF3. TRIF associates with TBK1 (TANK (tumour-necrosis-factor-receptor-associated factor (TRAF)-family-member-associated nuclear factor-κB (NF-κB) activator)-binding kinase 1; also known as NAK) through NAP1 (NAK-associated protein 1) and TRAF3. TBK1 then phosphorylates IRF3 (and IRF7 if it is present). Phosphorylated IRF3 forms homodimers and translocates to the nucleus, where it induces the *IFNB* gene and other genes, such as the gene encoding the chemokine CXC-chemokine ligand 10 (CXCL10). (TLR3 uses similar signalling pathways, which involve the activation of IRF3 in a TRIF-dependent manner; however, these pathways are not as well defined and are therefore not depicted here). **b** | IRF7 binds MyD88, which is localized on endosomes, and regulates the type I IFN gene-expression programme. After ligation of TLR9 with DNA containing unmethylated CpG motifs, the IRF7 bound to MyD88 is activated by a protein-kinase cascade that involves IRAK4 (interleukin-1-receptor-associated kinase 4), IRAK1 and IKKα (inhibitor of NF-κB (IκB) kinase α). Similar activation of IRF7 also occurs following the activation of TLR7 with single-stranded RNA (not shown). Although the exact function of the other molecules that are depicted here is not known, TRAF6 and TRAF3 have been shown to interact with IRF7, and all of the molecules shown are known to be essential to this pathway, as shown by gene-knockout studies. **c** | IRF5 interacts with, and is activated by, MyD88 and TRAF6, by an as-yet-unknown mechanism. Activated IRF5 dimerizes and translocates to the nucleus, where it activates the transcription of pro-inflammatory cytokine genes, presumably in cooperation with NF-κB. It remains to be determined whether IRF5 needs to interact directly with DNA (as is depicted here) for its function. IRF8 is also involved in TLR9 signalling, either through directly activating transcription of pro-inflammatory cytokine genes (not shown) or through activating TRAF6. **d** | IRF4, the transcription of which is induced by TLR activation, binds MyD88 in a region that overlaps the IRF5-binding region, thereby inhibiting the binding of IRF5 to MyD88 and attenuating the MyD88-dependent activation of IRF5. ISRE, IFN-stimulated response element; LPS, lipopolysaccharide.

genome of dsDNA viruses such as HSV and murine cytomegalovirus, and this leads to robust type I IFN production by pDCs<sup>82–84</sup> (TABLE 2). Similarly, signalling through TLR7 is essential for type I IFN gene induction in response to infection with influenza virus or VSV, the genomes of which consist of ssRNA<sup>80,85,86</sup>. In contrast to TLR3- or TLR4-mediated type I IFN gene induction, which depends on TRIF, the TLR9-subfamily members exclusively use MyD88 as the signalling adaptor<sup>10</sup>.

Interestingly, MyD88 interacts directly with IRF7 (not with IRF3) at the endosomal compartment<sup>87–89</sup> (FIG. 3b). IRF7 also interacts with TRAF6, another adaptor molecule that functions downstream of MyD88, and after ligation of TLR9, IRF7 is activated in a MyD88- and TRAF6-dependent manner. Splenic pDCs from *Irf7*<sup>−/−</sup> mice have a marked defect in type I IFN gene induction following infection with viruses (the dsDNA virus HSV or the ssRNA virus VSV) or exposure to synthetic TLR ligands (CpG-A<sup>90,91</sup> or ssRNA<sup>80,85</sup>). By contrast, such induction is normal in pDCs deficient in IRF1, IRF3 or IRF5 (REF. 28). Therefore, robust type I IFN gene induction in pDCs requires IRF7.

Mutation studies of MyD88 revealed that its death domain interacts with IRF7 (REF. 87). The death domain also interacts with the IL-1-receptor-associated kinase (IRAK) family of serine/threonine kinases, which transduce the signal between MyD88 and TRAF6, indicating that IRAKs might be involved in the IRF7-signalling pathway. Indeed, pDCs from *Irak1*<sup>−/−</sup> or *Irak4*<sup>−/−</sup> mice were found to have a defect in IFNα production in response to activation of TLR9-subfamily members<sup>34,87</sup>. In light of the recent report showing that IKKα is also essential for activation of IRF7 (REF. 35), it is probable that the IRAK4–IRAK1–IKKα protein-kinase cascade, which is known to be involved in NF-κB activation<sup>10</sup>, also leads to the activation of IRF7. In addition, TRAF3 and osteopontin are known to be important members of this pathway, but their exact functions have not been determined<sup>48,92</sup> (FIG. 3b).

**Death domain**

A protein domain that is found in many proteins that are involved in signalling and apoptosis. This domain mediates protein–protein interaction.

**Endotoxic shock**

A clinical condition that is induced by hyperreactivity of the innate immune system to bacterial lipopolysaccharide (LPS). It is mediated by the inflammatory cytokines interleukin-1 and tumour-necrosis factor, both of which are produced in high amounts following sustained activation of Toll-like receptor 4 by LPS.

**CpG-B**

Also known as K-type CpG. Synthetic oligodeoxynucleotides that contain a CpG motif(s) on a phosphorothioate backbone. Analogous to bacterial infection, CpG-B triggers the differentiation of both plasmacytoid and conventional DCs, as well as the proliferation and activation of B cells. However, identical sequences have not been found in the genomes of bacteria.

Similarly, in macrophages, which cannot mount robust type I IFN gene induction, CpG-A does not colocalize with transfected MyD88 or IRF7 but localizes in lysosomes<sup>89</sup>. Therefore, the retention of CpG-A in the endosome of pDCs allows for spatiotemporal regulation of TLR9 signalling, a crucial step in type I IFN gene induction mediated by the MyD88–IRF7-signalling pathway. Indeed, manipulating the transport of CpG-A to the endosome of conventional DCs and macrophages was found to result in the production of IFNα in similar amounts to that produced by pDCs through activation of the TLR9–MyD88–IRF7-signalling pathway<sup>89</sup>. Therefore, pDCs seem to use a unique mechanism to retain TLR9-bound CpG-A in the endosome, and prolonged signalling might allow the phosphorylation of *de novo*-synthesized IRF7 and the activation of the positive-feedback loop, thereby eliciting robust type I IFN production.

It has been shown that viruses such as VSV and HSV tend to reside in an endosomal compartment after infection<sup>94,95</sup>, and there is evidence that this localization is important for type I IFN gene induction<sup>83</sup>. Although the mechanisms that control endosomal trafficking in CpG-A- or virus-stimulated pDCs remain unknown, one could envisage a molecule(s) that is selectively expressed by pDCs and assists the endosomal longevity of TLR ligands. Further studies are required to shed light on the idea that the subcellular localization of ligand-bound TLRs and adaptors, as well as the longevity of signalling, could markedly affect the outcome of TLR signalling.

**IRF5.** In addition to IRF7, IRF5 also interacts directly with MyD88 and TRAF6 (REF. 18) (FIG. 3c). Unlike IRF7, which binds the death domain of MyD88, IRF5 interacts with the central region (known as the intermediary domain) and part of the TIR domain of MyD88 (REF. 96). Activation of the MyD88-dependent pathway by a TLR7 or TLR9 ligand leads to translocation of IRF5 to the nucleus<sup>18,97</sup>. That IRF5 has a crucial function in TLR signalling is underscored by the observation that induction of pro-inflammatory cytokine genes (such as those encoding TNF, IL-6 and IL-12p40) by ligands of TLR4, TLR5, TLR7 and TLR9 is severely impaired in macrophages and conventional DCs from *Irf5*<sup>−/−</sup> mice<sup>18</sup>. Following ligation of TLR9, the expression of other genes involved in immune responses, such as those encoding CXCL2 and IkBζ, also depends on IRF5 (REFS 18,96). Consistent with this, *Irf5*<sup>−/−</sup> mice show increased resistance to endotoxic shock following exposure to CpG-B or LPS, as well as impaired induction of pro-inflammatory cytokine genes<sup>18</sup>. Collectively, these observations indicate that TLR activation commonly induces the formation of MyD88–IRF5–TRAF6 complexes and results in the nuclear translocation of IRF5 (probably following its phosphorylation), leading to the production of pro-inflammatory cytokines (FIG. 3c). Following activation of TLR9, IRF5 has been shown to bind the ISRE in the promoter of the gene that encodes IL-12p40 (REF. 18); however, the way in which IRF5 participates in the induction of other target genes remains to be clarified.

**Spatiotemporal regulation of the MyD88–IRF7-signalling pathway in pDCs.** Although the importance of IRF7 in MyD88-dependent type I IFN gene induction is clear, it is not fully understood why pDCs — but not other cell types, such as conventional DCs — produce large amounts of type I IFNs. It was initially assumed that IRF7 is constitutively expressed in greater amounts in pDCs than in conventional DCs; however, this does not fully account for the robust type I IFN gene induction. For example, in conventional DCs from *Irf2*<sup>−/−</sup> mice, the expression of IRF7 is upregulated, but these conventional DCs remain poor at inducing type I IFN genes after activation of TLR9 (REF. 93). Instead, one mechanism that probably accounts for the specific ability of pDCs to produce type I IFNs is the spatiotemporal regulation of TLR7 and TLR9 signalling.

In pDCs, CpG-A — a TLR9 ligand that induces type I IFN genes— localizes for a long period in the endosomal compartment, where the MyD88–IRF7 complex also preferentially localizes. By contrast, in conventional DCs, CpG-A localizes in lysosomes.

REVIEWS

Table 3 | Interaction of interferon-regulatory factors with other factors

| IRF-interacting molecule | Interacting IRF | Function of interaction | Refs |
|--------------------------|----------------|------------------------|------|
| Casein kinase 2         | IRF1           | Phosphorylates IRF1    | 141  |
| Nucleophosmin           | IRF1           | Suppresses IRF1 function | 142  |
| p53                     | IRF1           | Induces CDKN1A gene   | 120  |
| CBP, p300               | IRF1, IRF3     | Acetylates IRF1 and IRF3 (as well as histones) | 15,20–23, 143 |
| CYPB                    | IRF3           | Assists IRF3 phosphorylation | 36   |
| GRIP1                   | IRF3           | Activates IRF3 (together with CBP or p300) | 144  |
| p65 subunit of NF-κB    | IRF3           | Activates IRF3 (together with CBP or p300) | 60,101,102 |
| PIN1                    | IRF3           | Facilitates proteasome-dependent degradation of phosphorylated IRF3 | 37   |
| TBK1, IKKε              | IRF3, IRF7     | Phosphorylates IRF3 and IRF7 | 30,31 |
| FKBP52                  | IRF4           | Inhibits IRF4–PU.1 interaction | 145  |
| NFATc                   | IRF4           | Induces IL4 gene      | 125  |
| STAT6                   | IRF4           | Induces CD23 gene     | 100  |
| BCL-6                   | IRF4           | Represses CD23 gene induction | 100  |
| PU.1                    | IRF4           | Induces IGK gene      | 100  |
|                         | IRF8           | Induces IL12 and IL18 genes | 146  |
| MyD88                   | IRF4           | Inhibits MyD88–IRF5 interaction | 96   |
|                         | IRF5, IRF7     | Activates IRF5 and IRF7 | 18,87 |
| TRAF6                   | IRF5, IRF7     | Activates IRF5 and IRF7 | 18,87 |
| IKKα                    | IRF7           | Phosphorylates IRF7    | 35   |
| SMAD3                   | IRF7           | Induces IFNA and IFNB genes | 147  |
| STAT1, STAT2            | IRF9           | Induces IFN-inducible genes | 148  |

BCL-6, B-cell lymphoma 6; CBP, cyclic-AMP-responsive-element-binding protein (CREB)-binding protein; CD23, also known as low-affinity receptor for IgE (FcεRII); CDKN1A, cyclin-dependent kinase inhibitor 1A (also known as WAF1 or p21); CYPB, cyclophilin B; FKBP52, FK506-binding protein 52 (an immunophilin); GRIP1, glucocorticoid-receptor-interacting protein 1 (also known as NCOA2); IFN, interferon; IGK, immunoglobulin κ light chain; IKK, inhibitor-of-NF-κB kinase; IL, interleukin; IRAK1, IL-1-receptor-associated kinase 1; IRF, IFN-regulatory factor; MyD88, myeloid differentiation primary-response protein 88; NFATc, nuclear factor of activated T cells, cytoplasmic; NF-κB, nuclear factor-κB; p53, tumour-suppressor protein p53; p300, co-activator p300; PIN1, cis-trans peptidylprolyl isomerase, NIMA-interacting 1; PU.1, transcription factor PU.1; SMAD3, mothers against decapentaplegic homologue 3; STAT, signal transducer and activator of transcription; TBK1, TANK (TRAF-family-member-associated NF-κB activator)-binding kinase 1; TRAF6, tumour-necrosis-factor-receptor-associated factor 6.

the selective positioning of specific molecules in discrete subcellular compartments, and the physical inclusion or exclusion of signalling molecules from particular sites in the cell might control the magnitude, duration and type of downstream signalling pathway engaged.

Other IRFs. Studies of other IRF-family members have also been carried out. IRF4 has been shown to form a complex with MyD88 (REF. 96) (FIG. 3d). It binds the central region of MyD88, as does IRF5 (REF. 96), indicating that IRF4 competes with IRF5, but not IRF7, for binding MyD88. Indeed, following TLR ligation, the induction of pro-inflammatory cytokine genes, which depends on IRF5, is markedly increased in peritoneal macrophages from *Ir*4−/− mice; this indicates that the interaction of IRF4 with MyD88 inhibits the binding of IRF5 to MyD88, attenuating the MyD88-dependent activation of IRF5 (FIG. 3d). Because the expression of IRF4 is restricted to cells of the immune system — particularly B cells, T cells and macrophages<sup>5</sup> — IRF4 might selectively control the expression of subsets of MyD88-dependent genes in a cell-type-specific manner.

IRF8, another immune-cell-specific member of the IRF family, is also involved in the TLR9–MyD88 pathway (FIG. 3c). IRF8 increases signalling by interacting with, and activating, TRAF6 (REF. 98). In response to ligation of TLR4 or TLR9, macrophages and conventional DCs from *Ir*f8−/− mice fail to induce the genes that encode the pro-inflammatory cytokines TNF and IL-6 (REFS 98,99). This defect is attributed to the inability of *Ir*f8−/− cells to activate NF-κB and/or MAPKs in response to TLR activation. The participation of IRF8 in the MyD88-dependent pathway might add to the diversity of the signalling cascade downstream of TLRs and thereby enable the cell-type-specific and signal-specific activation of target genes.

In addition to these MyD88-interacting IRFs, we recently found that IRF1 and IRF6 also interact with MyD88 (K.H., unpublished observations); therefore, these IRFs might also participate in signalling through TLRs.

Interaction of IRFs with other factors

The cooperation between IRFs, and between IRFs and members of unrelated families of transcription factors, is thought to be an important mechanism by which IRFs control the specificity and magnitude of a transcriptional event. There is ample evidence for such cooperative interactions of IRFs (TABLE 3). For example, IRF4 and IRF8 can interact with the ETS-family transcription factor PU.1, and this complex binds the ETS/ISRE composite DNA motif<sup>5,100</sup>. IRF4 and IRF8 can function as transcriptional repressors when bound to an ISRE motif but as transcriptional activators when bound to an ETS/ISRE composite DNA motif<sup>5,100</sup>.

It has long been known that members of the NF-κB and IRF families cooperatively control the transcription of several cytokine genes. The most extensively studied genes have been those associated with IFNB gene induction. Activation of transcription of the IFNB gene requires the coordinated binding of the transcription factors NF-κB, AP1 (a complex of activating

transcription factor 2 (ATF2) and JUN) and homodimers or heterodimers of IRF3 and IRF7, together with the binding of the co-activators CBP or p300 (REFS 2,4,15) (FIG. 4a). Assembly of these transcription factors on the IFNB gene promoter results in cooperative

**Figure 4 | Interactions between interferon-regulatory factor 3 and nuclear factor-κB.** Interferon (IFN)-regulatory factor 3 (IRF3) and the nuclear factor-κB (NF-κB)-family member p65 cooperatively control the transcription of several cytokine genes. Induction of the IFNB gene in response to activation of cytosolic or transmembrane pattern-recognition receptors, and the role of IRF3 and NF-κB, is depicted (a). In addition, models of the way in which different types of interaction between IRF3 and p65 might differentially regulate the MyD88 (myeloid differentiation primary-response protein 88)-dependent gene-expression programme are depicted (b–e). **a** | The gene that encodes IFNβ contains at least four positive-regulatory *cis* elements. One NF-κB complex (consisting of the subunits p50 and p65) and two IRF dimers (IRF3 homodimers, IRF7 homodimers or IRF3–IRF7 heterodimers) independently bind their respective binding elements (the NF-κB-binding site and the IFN-stimulated response element, ISRE). As an example, IRF3–IRF7 heterodimers are shown. These transcription-factor complexes cooperatively interact with each other, together with the co-activators CBP (cyclic-AMP-responsive-element-binding protein (CREB)-binding protein) or p300, to form the enhanceosome. (For clarity, the interaction of CBP or p300 with these transcription factors is indicated in a simplified manner.) Assembly of the enhanceosome allows robust induction of the IFNB gene. **b** | IRF3 and p65 can directly interact. Following ligation of Toll-like receptor 4 (TLR4), the IRF3-mediated activation of ISRE-containing genes, such as those encoding CXCL10 (CXC-chemokine ligand 10) and IFIT1 (IFN-induced protein with tetratricopeptide repeats 1), requires p65 as a co-activator, presumably for the recruitment of CBP or p300. **c** | Following ligation of TLR4, the induction of the genes encoding CXCL9 and CLIC4 (chloride intracellular channel 4), both of which contain the NF-κB-binding site but no IRF-binding site (ISRE), requires IRF3 functioning as a co-activator. **d,e** | Ligand-bound glucocorticoid receptor (GR) binds p65 in a region that overlaps with the IRF3-binding region, thereby inhibiting the binding of IRF3 to p65 and attenuating the gene-expression programmes that require IRF3. All of these models need further examination. At present, it is unknown whether IRF3 interacts with other members of the NF-κB family. AP1, activator protein 1; ATF2, activating transcription factor 2.

physical interactions between these transcription factors. The resultant DNA-bound complex (known as the enhanceosome) is more stable and effective at inducing transcription than any of its individual components bound independently to the promoter¹⁵.

The classical interaction between IRFs and NF-κB occurs through independent binding of transcription factors to adjacent sites (FIG. 4a). By contrast, much evidence indicates that IRF3 can form a complex directly with the NF-κB subunit p65 (also known as REL-A), through binding the REL-homology domain (RHD) of p65. In this way, IRF3 can function as a promoter-specific and signal-specific cofactor to activate transcription of a set of NF-κB-dependent genes, without directly binding to an ISRE⁶⁰,¹⁰¹,¹⁰² (FIG. 4c). Conversely, MEFs deficient in p65 show markedly impaired induction of ISRE-containing genes in response to TLR4 ligation, indicating that IRF3–p65 complexes (with IRF3 binding the ISRE) support full induction of these genes⁶⁰,¹⁰² (FIG. 4b). But p65 is not required for IRF3 activity in response to ligation of a different TLR, TLR3 (REF. 102). Therefore, depending on the nature of signalling, either IRF3 functions as an essential co-activator of p65 for the transcription of NF-κB-dependent genes, or p65 functions as a cofactor of IRF3 for the transcription of IRF-dependent genes.

The importance of the cooperation between NF-κB and IRF3 was also confirmed by a recent study showing that the glucocorticoid-receptor-mediated repression of TLR-dependent gene-expression programmes operates through IRF3 or NF-κB⁶⁰ (FIG. 4d,e). The glucocorticoid receptor represses transcriptional responses to diverse pro-inflammatory signalling pathways and exerts anti-inflammatory effects in various diseases. Recent studies revealed that the ligand-bound glucocorticoid receptor prevents interactions between p65 and IRF3 by binding the RHD of p65 and therefore competing with IRF3 for binding to this site. Consequently, the glucocorticoid receptor represses a large set of inflammatory-response genes⁶⁰. This *trans*-repression mechanism operates on the MyD88-dependent pathway. So the signal-specific interaction of IRF3 with p65 might differentially regulate the gene-expression programme in response to various pathogens. It is probable that additional interactions of IRFs with co-activators, and possibly co-repressors, will be identified in future studies, and these efforts should lead to a more complete understanding of the mechanisms that underlie the regulation and function of IRFs.

## Implications for human diseases

Given the diverse and potent effects of IRFs on the immune system, it is not surprising that these transcription factors have a pivotal role in several autoimmune diseases. In particular, the aberrant activation of the IRF-type-I-IFN system by self nucleic acids could be a pathogenic mechanism in certain autoimmune diseases.

To avoid autoimmunity, ‘danger’ motifs that are present in pathogen-specific nucleic acids are either under-represented or modified in the nucleic acids of mammalian cells (TABLE 2). There is evidence that specific

REVIEW S

modifications of mammalian RNA, including the poly(A) tail of mRNA, might be crucial for silencing the immunostimulatory activity of self RNA ${ }^{103,104}$. In addition, vertebrate DNA is hypermethylated at CpG motifs ${ }^{105}$, unlike bacterial and viral DNA, which is hypomethylated.

Recent studies, however, point to self DNA or self RNA as activators of autoimmunity. Systemic lupus erythematosus (SLE) is a typical example of this. It is characterized by the production of autoantibodies specific for certain nucleic-acid-containing macromolecules, such as small nuclear ribonucleoproteins (snRNPs) and chromatin ${ }^{106-110}$. Autoantibody-containing sera from patients with SLE have been shown to form immune complexes with snRNPs, which contain ssRNA, and to activate TLR7 and/or TLR8 efficiently, thereby inducing the production of type I IFNs by pDCs ${ }^{106,107}$. Immune complexes that contain DNA derived from apoptotic cells also elicit the production of large amounts of type I IFNs by pDCs and might contribute to the development of SLE ${ }^{109,111}$. DNA-containing immune complexes also induce the production of TNF by conventional DCs and the proliferation of B cells, at least in part through TLR9 (REFS 108, 110). Therefore, these reports indicate that both self RNA and self DNA are potential ligands for TLR7, TLR8 and TLR9 and that they contribute to the development of SLE.

Notably, self nucleic acids are involved in pathogenesis only when in complex with autoantibodies. Several studies have shown that the method of delivery of nucleic acids is crucial for their immunostimulatory activity ${ }^{89,112}$. For example, complexes that consist of mammalian DNA and liposomes elicit inflammatory responses when introduced into macrophages, and these responses depend on acidification of the endosome ${ }^{112}$. In patients with SLE, serum autoantibodies presumably bind the Fc receptor or rheumatoid factor (which recognizes self IgG) at the surface of B cells, pDCs or conventional DCs, and this interaction facilitates the internalization and retention of self nucleic acids in endosomal vesicles, where MyD88 is colocalized with IRF7 and/or IRF5 (REFS 106, 109, 113). Interestingly, a polymorphism in the IRF5 gene is associated with SLE ${ }^{114}$. Therefore, these reports indicate that the activation of IRF7 and/or IRF5 by ‘mislocated’ self nucleic acids might be responsible for the pathogenesis of autoimmune diseases.

### Future perspectives

In recent years, there has been considerable progress in our understanding of the roles of the IRF family of transcription factors, together with other transcription factors (including NF-$\kappa$B), in the regulation of immune responses mediated by cytosolic and transmembrane PRRs. In particular, more recent studies have revealed the role of IRFs in the MyD88-dependent pathway and have implicated IRFs in the pathogenesis of autoimmune diseases. It will be interesting to study further the way in which MyD88, which seems to interact with numerous signalling molecules and transcription factors, leads to divergent pathways, allowing the differential regulation of gene-regulatory networks. In this context, we have given the name ‘cytoplasmic transductional-transcriptional processor’ (CTTP) to the multimolecular complex that consists of MyD88 and the IRFs, in analogy to computing terminology ${ }^{87}$: CTTP might provide a point in the cytoplasm for an organized array of signalling molecules and transcription factors that dynamically determines the specificity, strength and longevity of the input signal and translates this into the output signal (that is, transcriptional events).

A more detailed understanding of the signalling pathways that turn on and turn off the IRF system is important for the development of new therapies for infectious and autoimmune diseases. It should also be noted that IRFs have roles in other aspects of host defence ${ }^{4,6}$ (such as the regulation of oncogenesis), and the study of these areas is emerging as an interesting area of research.

---

**References**

1. Tjian, R. & Maniatis, T. Transcriptional activation: a complex puzzle with few easy pieces. *Cell* **77**, 5–8 (1994).
2. Lenardo, M. J., Fan, C. M., Maniatis, T. & Baltimore, D. The involvement of NF-$\kappa$B in $\beta$-interferon gene regulation reveals its role as widely inducible mediator of signal transduction. *Cell* **57**, 287–294 (1989).
3. Li, Q. & Verma, I. M. NF-$\kappa$B regulation in the immune system. *Nature Rev. Immunol.* **2**, 725–734 (2002).
4. Taniguchi, T., Ogasawara, K., Takaoka, A. & Tanaka, N. IRF family of transcription factors as regulators of host defense. *Annu. Rev. Immunol.* **19**, 623–655 (2001).
5. Lohoff, M. & Mak, T. W. Roles of interferon-regulatory factors in T-helper-cell differentiation. *Nature Rev. Immunol.* **5**, 125–135 (2005).
6. Taniguchi, T. & Takaoka, A. A weak signal for strong responses: interferon-$\alpha / \beta$ revisited. *Nature Rev. Mol. Cell Biol.* **2**, 378–386 (2001).
7. Platanias, L. C. Mechanisms of type-I- and type-II-interferon-mediated signalling. *Nature Rev. Immunol.* **5**, 375–386 (2005).
8. Decker, T., Muller, M. & Stockinger, S. The Yin and Yang of type I interferon activity in bacterial infection. *Nature Rev. Immunol.* **5**, 675–687 (2005).
9. Eroshkin, A. & Mushegian, A. Conserved transactivation domain shared by interferon regulatory factors and Smad morphogens. *J. Mol. Med.* **77**, 403–405 (1999).
10. Akira, S., Uematsu, S. & Takeuchi, O. Pathogen recognition and innate immunity. *Cell* **124**, 783–801 (2006).
11. Janeway, C. A. Jr. & Medzhitov, R. Innate immune recognition. *Annu. Rev. Immunol.* **20**, 197–216 (2002).
12. Yoneyama, M. *et al.* The RNA helicase RIG-I has an essential function in double-stranded RNA-induced innate antiviral responses. *Nature Immunol.* **5**, 730–737 (2004).
13. Yoneyama, M. *et al.* Shared and unique functions of the DExD/H-box helicases RIG-I, MDA5, and LGP2 in antiviral innate immunity. *J. Immunol.* **175**, 2851–2858 (2005).
    References 12 and 13 were the first reports of the cytosolic PRR molecules involved in type I IFN gene induction by viruses or dsRNA.
14. Mamane, Y. *et al.* Interferon regulatory factors: the next generation. *Gene* **237**, 1–14 (1999).
15. Wathelet, M. G. *et al.* Virus infection induces the assembly of coordinately activated transcription factors on the IFN-$\beta$ enhancer *in vivo*. *Mol. Cell* **1**, 507–518 (1998).
16. Miyamoto, M. *et al.* Regulated expression of a gene encoding a nuclear factor, IRF-1, that specifically binds to IFN-$\beta$ gene regulatory elements. *Cell* **54**, 903–913 (1988).
    This paper reports the discovery of the first IRF-family member, IRF1.
17. Matsuyama, T. *et al.* Targeted disruption of IRF-1 or IRF-2 results in abnormal type I IFN gene induction and aberrant lymphocyte development. *Cell* **75**, 83–97 (1993).
18. Takaoka, A. *et al.* Integral role of IRF-5 in the gene induction programme activated by Toll-like receptors. *Nature* **434**, 243–249 (2005).
    This paper was the first report to show that IRF5 is essential for TLR-mediated induction of pro-inflammatory cytokine genes.
19. Weaver, B. K., Kumar, K. P. & Reich, N. C. Interferon regulatory factor 3 and CREB-binding protein/p300 are subunits of double-stranded RNA-activated transcription factor DRAF1. *Mol. Cell. Biol.* **18**, 1359–1368 (1998).
20. Lin, R., Heylbroeck, C., Pitha, P. M. & Hiscott, J. Virus-dependent phosphorylation of the IRF-3 transcription factor regulates nuclear translocation, transactivation potential, and proteasome-mediated degradation. *Mol. Cell. Biol.* **18**, 2986–2996 (1998).
21. Yoneyama, M. *et al.* Direct triggering of the type I interferon system by virus infection: activation of a transcription factor complex containing IRF-3 and CBP/p300. *EMBO J.* **17**, 1087–1095 (1998).
22. Sato, M., Tanaka, N., Hata, N., Oda, E. & Taniguchi, T. Involvement of the IRF family transcription factor IRF-3 in virus-induced activation of the IFN-$\beta$ gene. *FEBS Lett.* **425**, 112–116 (1998).
    References 19–22 provide evidence for the phosphorylation-dependent activation of IRF3 during type I IFN gene induction.

23. Suhara, W. *et al.* Analyses of virus-induced homomeric and heteromeric protein associations between IRF-3 and coactivator CBP/p300. *J. Biochem. (Tokyo)* **128**, 301–307 (2000).

24. Sato, M. *et al.* Positive feedback regulation of type I IFN genes by the IFN-inducible transcription factor IRF-7. *FEBS Lett.* **441**, 106–110 (1998).

25. Marie, I., Durbin, J. E. & Levy, D. E. Differential viral induction of distinct interferon-α genes by positive feedback through interferon regulatory factor-7. *EMBO J.* **17**, 6660–6669 (1998).

References 24 and 25 describe the positive-feedback regulation of type I IFN gene induction, which involves the expression and activation of IRF7.

26. Lin, R., Mamane, Y. & Hiscott, J. Multiple regulatory domains control IRF-7 activity in response to virus infection. *J. Biol. Chem.* **275**, 34320–34327 (2000).

27. Sato, M. *et al.* Distinct and essential roles of transcription factors IRF-3 and IRF-7 in response to viruses for IFN-α/β gene induction. *Immunity* **13**, 539–548 (2000).

28. Honda, K. *et al.* IRF-7 is the master regulator of type-I interferon-dependent immune responses. *Nature* **434**, 772–777 (2005).

References 27 and 28 are gene-targeting studies of IRF3 and IRF7, respectively. Reference 28 provides definitive evidence that IRF7 is the main regulator of both cytosolic PRR- and TLR-mediated type I IFN gene induction.

29. Nakaya, T. *et al.* Gene induction pathways mediated by distinct IRFs during viral infection. *Biochem. Biophys. Res. Commun.* **283**, 1150–1156 (2001).

30. Sharma, S. *et al.* Triggering the interferon antiviral response through an IKK-related pathway. *Science* **300**, 1148–1151 (2003).

31. Fitzgerald, K. A. *et al.* IKKε and TBK1 are essential components of the IRF3 signaling pathway. *Nature Immunol.* **4**, 491–496 (2003).

References 30 and 31 were the first reports to identify that TBK1 and IKKε are the protein kinases that activate IRF3 and IRF7.

32. Hemmi, H. *et al.* The roles of two IkB kinase-related kinases in lipopolysaccharide and double stranded RNA signaling and viral infection. *J. Exp. Med.* **199**, 1641–1650 (2004).

33. Perry, A. K., Chow, E. K., Goodnough, J. B., Yeh, W. C. & Cheng, G. Differential requirement for TANK-binding kinase-1 in type I interferon responses to Toll-like receptor activation and viral infection. *J. Exp. Med.* **199**, 1651–1658 (2004).

34. Uematsu, S. *et al.* Interleukin-1 receptor-associated kinase-1 plays an essential role for Toll-like receptor (TLR)7- and TLR9-mediated interferon-α induction. *J. Exp. Med.* **201**, 915–923 (2005).

35. Hoshino, K. *et al.* IkB kinase-α is critical for interferon-α production induced by Toll-like receptors 7 and 9. *Nature* **440**, 949–953 (2006).

References 34 and 35 show that IRAK1 and IKKα are involved in the activation of IRF7.

36. Obata, Y. *et al.* Role of cyclophilin B in activation of interferon regulatory factor-3. *J. Biol. Chem.* **280**, 18355–18360 (2005).

37. Saitoh, T. *et al.* Negative regulation of interferon-regulatory factor 3-dependent innate antiviral response by the prolyl isomerase Pin1. *Nature Immunol.* **7**, 598–605 (2006).

38. Huang, J. *et al.* SIKE is an IKKε/TBK1-associated suppressor of TLR3- and virus-triggered IRF-3 activation pathways. *EMBO J.* **24**, 4018–4028 (2005).

39. Honda, K. *et al.* Selective contribution of IFN-α/β signaling to the maturation of dendritic cells induced by double-stranded RNA or viral infection. *Proc. Natl Acad. Sci. USA* **100**, 10872–10877 (2003).

40. Kato, H. *et al.* Cell type-specific involvement of RIG-I in antiviral response. *Immunity* **23**, 19–28 (2005).

41. Kato, H. *et al.* Differential roles of MDA5 and RIG-I helicases in the recognition of RNA viruses. *Nature* **441**, 101–105 (2006).

42. Kawai, T. *et al.* IPS-1, an adaptor triggering RIG-I- and Mda5-mediated type I interferon induction. *Nature Immunol.* **6**, 981–988 (2005).

43. Meylan, E. *et al.* Cardif is an adaptor protein in the RIG-I antiviral pathway and is targeted by hepatitis C virus. *Nature* **437**, 1167–1172 (2005).

44. Xu, L. G. *et al.* VISA is an adapter protein required for virus-triggered IFN-β signaling. *Mol. Cell* **19**, 727–740 (2005).

45. Seth, R. B., Sun, L., Ea, C. K. & Chen, Z. J. Identification and characterization of MAVS, a mitochondrial antiviral signaling protein that activates NF-κB and IRF 3. *Cell* **122**, 669–682 (2005).

46. Sun, Q. *et al.* The specific and essential role of MAVS in antiviral innate immune responses. *Immunity* **24**, 633–642 (2006).

47. Kumar, H. *et al.* Essential role of IPS-1 in innate immune responses against RNA viruses. *J. Exp. Med.* **203**, 1795–1803 (2006).

References 46 and 47 describe gene-targeting studies of CARDIF/IPS1/MAVS/VISA and show that this adaptor has an essential role in the cytosolic pathway of type I IFN gene induction.

48. Oganesyan, G. *et al.* Critical role of TRAF3 in the Toll-like receptor-dependent and -independent antiviral response. *Nature* **439**, 208–211 (2006).

49. Stockinger, S. *et al.* IFN regulatory factor 3-dependent induction of type I IFNs by intracellular bacteria is mediated by a TLR- and Nod2-independent mechanism. *J. Immunol.* **173**, 7416–7425 (2004).

50. O'Connell, R. M. *et al.* Type I interferon production enhances susceptibility to *Listeria monocytogenes* infection. *J. Exp. Med.* **200**, 437–445 (2004).

51. Stetson, D. B. & Medzhitov, R. Recognition of cytosolic DNA activates an IRF3-dependent innate immune response. *Immunity* **24**, 93–103 (2006).

52. Ishii, K. J. *et al.* A Toll-like receptor-independent antiviral response induced by double-stranded B-form DNA. *Nature Immunol.* **7**, 40–48 (2006).

53. Medzhitov, R., Preston-Hurlburt, P. & Janeway, C. A. Jr. A human homologue of the *Drosophila* Toll protein signals activation of adaptive immunity. *Nature* **388**, 394–397 (1997).

54. Kurt-Jones, E. A. *et al.* Pattern recognition receptors TLR4 and CD14 mediate response to respiratory syncytial virus. *Nature Immunol.* **1**, 398–401 (2000).

55. Hoshino, K., Kaisho, T., Iwabe, T., Takeuchi, O. & Akira, S. Differential involvement of IFN-β in Toll-like receptor-stimulated dendritic cell activation. *Int. Immunol.* **14**, 1225–1231 (2002).

56. Kawai, T. *et al.* Lipopolysaccharide stimulates the MyD88-independent pathway and results in activation of IFN-regulatory factor 3 and the expression of a subset of lipopolysaccharide-inducible genes. *J. Immunol.* **167**, 5887–5894 (2001).

This paper provides evidence that there is a MyD88-independent pathway for activation of IRF3.

57. Sakaguchi, S. *et al.* Essential role of IRF-3 in lipopolysaccharide-induced interferon-β gene expression and endotoxin shock. *Biochem. Biophys. Res. Commun.* **306**, 860–866 (2003).

58. Hoebe, K. *et al.* Upregulation of costimulatory molecules induced by lipopolysaccharide and double-stranded RNA occurs by Trif-dependent and Trif-independent pathways. *Nature Immunol.* **4**, 1223–1229 (2003).

59. Werner, S. L., Barken, D. & Hoffmann, A. Stimulus specificity of gene expression programs determined by temporal control of IKK activity. *Science* **309**, 1857–1861 (2005).

60. Ogawa, S. *et al.* Molecular determinants of crosstalk between nuclear receptors and Toll-like receptors. *Cell* **122**, 707–721 (2005).

Reference 60, together with references 101 and 102, provides evidence for the association of IRF3 with the NF-κB component p65.

61. Kamijo, R. *et al.* Requirement for transcription factor IRF-1 in NO synthase induction in macrophages. *Science* **263**, 1612–1615 (1994).

62. Alexopoulou, L., Holt, A. C., Medzhitov, R. & Flavell, R. A. Recognition of double-stranded RNA and activation of NF-κB by Toll-like receptor 3. *Nature* **413**, 732–738 (2001).

63. Wang, T. *et al.* Toll-like receptor 3 mediates West Nile virus entry into the brain causing lethal encephalitis. *Nature Med.* **10**, 1366–1373 (2004).

References 62 and 63 are the initial reports on the role of TLR3 in the recognition of virus-derived and synthetic dsRNA.

64. Rudd, B. D. *et al.* Deletion of TLR3 alters the pulmonary immune environment and mucus production during respiratory syncytial virus infection. *J. Immunol.* **176**, 1937–1942 (2006).

65. Tabetá, K. *et al.* Toll-like receptors 9 and 3 as essential components of innate immune defense against mouse cytomegalovirus infection. *Proc. Natl Acad. Sci. USA* **101**, 3516–3521 (2004).

66. Flandin, J. F., Chano, F. & Descoteaux, A. RNA interference reveals a role for TLR2 and TLR3 in the recognition of *Leishmania donovani* promastigotes by interferon-γ-primed macrophages. *Eur. J. Immunol.* **36**, 411–420 (2006).

67. Aksoy, E. *et al.* Double-stranded RNAs from the helminth parasite *Schistosoma* activate TLR3 in dendritic cells. *J. Biol. Chem.* **280**, 277–283 (2005).

68. Yamamoto, M. *et al.* Role of adaptor TRIF in the MyD88-independent Toll-like receptor signaling pathway. *Science* **301**, 640–643 (2003).

69. Sarkar, S. N. *et al.* Novel roles of TLR3 tyrosine phosphorylation and PI3 kinase in double-stranded RNA signaling. *Nature Struct. Mol. Biol.* **11**, 1060–1067 (2004).

70. Sato, S. *et al.* Toll/IL-1 receptor domain-containing adaptor inducing IFN-β (TRIF) associates with TNF receptor-associated factor 6 and TANK-binding kinase 1, and activates two distinct transcription factors, NF-κB and IFN-regulatory factor-3, in the Toll-like receptor signaling. *J. Immunol.* **171**, 4304–4310 (2003).

71. Gohda, J., Matsumura, T. & Inoue, J. TNFR-associated factor (TRAF) 6 is essential for MyD88-dependent pathway but not Toll/IL-1 receptor domain-containing adaptor-inducing IFN-β (TRIF)-dependent pathway in TLR signaling. *J. Immunol.* **173**, 2913–2917 (2004).

72. Hacker, H. *et al.* Specificity in Toll-like receptor signaling through distinct effector functions of TRAF3 and TRAF6. *Nature* **439**, 204–207 (2006).

73. Takeshita, F. *et al.* TRAF4 acts as a silencer in TLR-mediated signaling through the association with TRAF6 and TRIF. *Eur. J. Immunol.* **35**, 2477–2485 (2005).

74. Su, X. *et al.* TNF receptor-associated factor-1 (TRAF1) negatively regulates Toll/IL-1 receptor domain-containing adaptor inducing IFN-β (TRIF)-mediated signaling. *Eur. J. Immunol.* **36**, 199–206 (2006).

75. Sasai, M. *et al.* NF-κB-activating kinase-associated protein 1 participates in TLR3/Toll-IL-1 homology domain-containing adapter molecule-1-mediated IFN regulatory factor 3 activation. *J. Immunol.* **174**, 27–30 (2005).

76. Nakano, H., Yanagita, M. & Gunn, M. D. CD11c⁺B220⁺Gr-1⁺ cells in mouse lymph nodes and spleen display characteristics of plasmacytoid dendritic cells. *J. Exp. Med.* **194**, 1171–1178 (2001).

77. Asselin-Paturel, C. *et al.* Mouse type I IFN-producing cells are immature APCs with plasmacytoid morphology. *Nature Immunol.* **2**, 1144–1150 (2001).

78. Colonna, M., Trinchieri, G. & Liu, Y. J. Plasmacytoid dendritic cells in immunity. *Nature Immunol.* **5**, 1219–1226 (2004).

79. Wagner, H. The immunobiology of the TLR9 subfamily. *Trends Immunol.* **25**, 381–386 (2004).

80. Diebold, S. S., Kaisho, T., Hemmi, H., Akira, S. & Reis e Sousa, C. Innate antiviral responses by means of TLR7-mediated recognition of single-stranded RNA. *Science* **303**, 1529–1531 (2004).

81. Hemmi, H., Kaisho, T., Takeda, K. & Akira, S. The roles of Toll-like receptor 9, MyD88, and DNA-dependent protein kinase catalytic subunit in the effects of two distinct CpG DNAs on dendritic cell subsets. *J. Immunol.* **170**, 3059–3064 (2003).

82. Krug, A. *et al.* Herpes simplex virus type 1 activates murine natural interferon-producing cells through Toll-like receptor 9. *Blood* **103**, 1433–1437 (2004).

83. Lund, J., Sato, A., Akira, S., Medzhitov, R. & Iwasaki, A. Toll-like receptor 9-mediated recognition of herpes simplex virus-2 by plasmacytoid dendritic cells. *J. Exp. Med.* **198**, 513–520 (2003).

84. Krug, A. *et al.* TLR9-dependent recognition of MCMV by IPC and DC generates coordinated cytokine responses that activate antiviral NK cell function. *Immunity* **21**, 107–119 (2004).

85. Heil, F. *et al.* Species-specific recognition of single-stranded RNA via Toll-like receptor 7 and 8. *Science* **303**, 1526–1529 (2004).

86. Lund, J. M. *et al.* Recognition of single-stranded RNA viruses by Toll-like receptor 7. *Proc. Natl Acad. Sci. USA* **101**, 5598–5603 (2004).

87. Honda, K. *et al.* Role of a transduction-transcriptional processor complex involving MyD88 and IRF-7 in Toll-like receptor signaling. *Proc. Natl Acad. Sci. USA* **101**, 15416–15421 (2004).

88. Kawai, T. *et al.* Interferon-α induction through Toll-like receptors involves a direct interaction of IRF7 with MyD88 and TRAF6. *Nature Immunol.* **5**, 1061–1068 (2004).

References 87 and 88 report the evidence for a direct interaction between MyD88 and IRF7.

REVIEW S

89. Honda, K. *et al.* Spatiotemporal regulation of MyD88–IRF-7 signalling for robust type-I interferon induction. *Nature* **434**, 1035–1040 (2005). This paper shows the importance of endosomal MyD88–IRF7 signalling for robust induction of type I IFN genes in pDCs.

90. Verthelyi, D., Ishii, K. J., Gursel, M., Takeshita, F. & Klinman, D. M. Human peripheral blood cells differentially recognize and respond to two distinct CpG motifs. *J. Immunol.* **166**, 2372–2377 (2001).

91. Klinman, D. M. Immunotherapeutic uses of CpG oligodeoxynucleotides. *Nature Rev. Immunol.* **4**, 249–258 (2004).

92. Shinohara, M. L. *et al.* Osteopontin expression is essential for interferon-α production by plasmacytoid dendritic cells. *Nature Immunol.* **7**, 498–506 (2006).

93. Yanai, H. *et al.* IRF family transcription factors in type I interferon induction. *Int. Congr. Ser.* **1285**, 104–113 (2005).

94. Durrer, P., Gaudin, Y., Ruigrok, R. W., Graf, R. & Brunner, J. Photolabeling identifies a putative fusion domain in the envelope glycoprotein of rabies and vesicular stomatitis viruses. *J. Biol. Chem.* **270**, 17575–17581 (1995).

95. Brunetti, C. R., Dingwell, K. S., Wale, C., Graham, F. L. & Johnson, D. C. Herpes simplex virus gD and virions accumulate in endosomes by mannose 6-phosphate-dependent and -independent mechanisms. *J. Virol.* **72**, 3330–3339 (1998).

96. Negishi, H. *et al.* Negative regulation of Toll-like receptor signaling by IRF-4. *Proc. Natl Acad. Sci. USA* **102**, 15989–15994 (2005).

97. Schoenemeyer, A. *et al.* The interferon regulatory factor, IRF5, is a central mediator of Toll-like receptor 7 signaling. *J. Biol. Chem.* **280**, 17005–17012 (2005).

98. Zhao, J. *et al.* IRF-8/interferon (IFN) consensus sequence-binding protein is involved in Toll-like receptor (TLR) signaling and contributes to the cross-talk between TLR and IFN-γ signaling pathways. *J. Biol. Chem.* **281**, 10073–10080 (2006).

99. Tsujimura, H. *et al.* Toll-like receptor 9 signaling activates NF-κB through IFN regulatory factor-8/IFN consensus sequence binding protein in dendritic cells. *J. Immunol.* **172**, 6820–6827 (2004).

100. Marecki, S. & Fenton, M. J. The role of IRF-4 in transcriptional regulation. *J. Interferon Cytokine Res.* **22**, 121–133 (2002).

101. Leung, T. H., Hoffmann, A. & Baltimore, D. One nucleotide in a κB site can determine cofactor specificity for NF-κB dimers. *Cell* **118**, 453–464 (2004).

102. Wietek, C., Miggin, S. M., Jefferies, C. A. & O’Neill, L. A. Interferon regulatory factor-3-mediated activation of the interferon-sensitive response element by Toll-like receptor (TLR) 4 but not TLR3 requires the p65 subunit of NF-κB. *J. Biol. Chem.* **278**, 50923–50931 (2003).

103. Kariko, K., Buckstein, M., Ni, H. & Weissman, D. Suppression of RNA recognition by Toll-like receptors: the impact of nucleoside modification and the evolutionary origin of RNA. *Immunity* **23**, 165–175 (2005).

104. Koski, G. K. *et al.* Innate immune system discriminates between RNA containing bacterial versus eukaryotic structural features that prime for high-level IL-12 secretion by dendritic cells. *J. Immunol.* **172**, 3989–3993 (2004).

105. Krieg, A. M. CpG motifs in bacterial DNA and their immune effects. *Annu. Rev. Immunol.* **20**, 709–760 (2002).

106. Vollmer, J. *et al.* Immune stimulation mediated by autoantigen binding sites within small nuclear RNAs involves Toll-like receptors 7 and 8. *J. Exp. Med.* **202**, 1575–1585 (2005).

107. Lau, C. M. *et al.* RNA-associated autoantigens activate B cells by combined B cell antigen receptor/ Toll-like receptor 7 engagement. *J. Exp. Med.* **202**, 1171–1177 (2005).

108. Leadbetter, E. A. *et al.* Chromatin–IgG complexes activate B cells by dual engagement of IgM and Toll-like receptors. *Nature* **416**, 603–607 (2002).

109. Ronnblom, L. & Alm, G. V. Systemic lupus erythematosus and the type I interferon system. *Arthritis Res. Ther.* **5**, 68–75 (2003).

110. Boule, M. W. *et al.* Toll-like receptor 9-dependent and -independent dendritic cell activation by chromatin–immunoglobulin G complexes. *J. Exp. Med.* **199**, 1631–1640 (2004).

111. Blanco, P., Palucka, A. K., Gill, M., Pascual, V. & Banchereau, J. Induction of dendritic cell differentiation by IFN-α in systemic lupus erythematosus. *Science* **294**, 1540–1543 (2001).

112. Yasuda, K. *et al.* Endosomal translocation of vertebrate DNA activates dendritic cells via TLR9-dependent and -independent pathways. *J. Immunol.* **174**, 6129–6136 (2005).

113. Wagner, H., Heit, A., Schmitz, F. & Bauer, S. Targeting split vaccines to the endosome improves vaccination. *Curr. Opin. Biotechnol.* **15**, 538–542 (2004).

114. Graham, R. R. *et al.* A common haplotype of interferon regulatory factor 5 (IRF5) regulates splicing and expression and is associated with increased risk of systemic lupus erythematosus. *Nature Genet.* **38**, 550–555 (2006). This paper reports that variants of IRF5 can confer risk of SLE.

115. Kimura, T. *et al.* Involvement of the IRF-1 transcription factor in antiviral responses to interferons. *Science* **264**, 1921–1924 (1994).

116. Ko, J., Gendron-Fitzpatrick, A. & Splitter, G. A. Susceptibility of IFN regulatory factor-1 and IFN consensus sequence binding protein-deficient mice to brucellosis. *J. Immunol.* **168**, 2433–2440 (2002).

117. Taki, S. *et al.* Multistage regulation of T~H~1-type immune responses by the transcription factor IRF-1. *Immunity* **6**, 673–679 (1997).

118. Ogasawara, K. *et al.* Requirement for IRF-1 in the microenvironment supporting development of natural killer cells. *Nature* **391**, 700–703 (1998).

119. Penninger, J. M. *et al.* The interferon regulatory transcription factor IRF-1 controls positive and negative selection of CD8^+^ thymocytes. *Immunity* **7**, 243–254 (1997).

120. Tanaka, N. *et al.* Cooperation of the tumour suppressors IRF-1 and p53 in response to DNA damage. *Nature* **382**, 816–818 (1996).

121. Tamura, T. *et al.* An IRF-1-dependent pathway of DNA damage-induced apoptosis in mitogen-activated T lymphocytes. *Nature* **376**, 596–599 (1995).

122. Hida, S. *et al.* CD8^+^ T cell-mediated skin disease in mice lacking IRF-2, the transcriptional attenuator of interferon-α/β signaling. *Immunity* **13**, 643–655 (2000).

123. Honda, K., Mizutani, T. & Taniguchi, T. Negative regulation of IFN-α/β signaling by IFN regulatory factor 2 for homeostatic development of dendritic cells. *Proc. Natl Acad. Sci. USA* **101**, 2416–2421 (2004).

124. Lohoff, M. *et al.* Deficiency in the transcription factor interferon regulatory factor (IRF)-2 leads to severely compromised development of natural killer and T helper type 1 cells. *J. Exp. Med.* **192**, 325–336 (2000).

125. Rengarajan, J. *et al.* Interferon regulatory factor 4 (IRF4) interacts with NFATc2 to modulate interleukin 4 gene expression. *J. Exp. Med.* **195**, 1003–1012 (2002).

126. Lohoff, M. *et al.* Dysregulated T helper cell differentiation in the absence of interferon regulatory factor 4. *Proc. Natl Acad. Sci. USA* **99**, 11808–11812 (2002).

127. Mittrucker, H. W. *et al.* Requirement for the transcription factor LSIRF/IRF4 for mature B and T lymphocyte function. *Science* **275**, 540–543 (1997).

128. Lu, R., Medina, K. L., Lancki, D. W. & Singh, H. IRF-4, 8 orchestrate the pre-B-to-B transition in lymphocyte development. *Genes Dev.* **17**, 1703–1708 (2003).

129. Tamura, T. *et al.* IFN regulatory factor-4 and -8 govern dendritic cell subset development and their functional diversity. *J. Immunol.* **174**, 2573–2581 (2005).

130. Fehr, T. *et al.* Crucial role of interferon consensus sequence binding protein, but neither of interferon regulatory factor 1 nor of nitric oxide synthesis for protection against murine listeriosis. *J. Exp. Med.* **185**, 921–931 (1997).

131. Giese, N. A. *et al.* Interferon (IFN) consensus sequence-binding protein, a transcription factor of the IFN regulatory factor family, regulates immune responses *in vivo* through control of interleukin 12 expression. *J. Exp. Med.* **186**, 1535–1546 (1997).

132. Scharton-Kersten, T., Contursi, C., Masumi, A., Sher, A. & Ozato, K. Interferon consensus sequence binding protein-deficient mice display impaired resistance to intracellular infection due to a primary defect in interleukin 12 p40 induction. *J. Exp. Med.* **186**, 1523–1534 (1997).

133. Holtschke, T. *et al.* Immunodeficiency and chronic myelogenous leukemia-like syndrome in mice with a targeted mutation of the ICSBP gene. *Cell* **87**, 307–317 (1996).

134. Schiavoni, G. *et al.* ICSBP is essential for the development of mouse type I interferon-producing cells and for the generation and activation of CD8α^+^ dendritic cells. *J. Exp. Med.* **196**, 1415–1425 (2002).

135. Harada, H. *et al.* Regulation of IFN-α/β genes: evidence for a dual function of the transcription factor complex ISGF3 in the production and action of IFN-α/β. *Genes Cells* **1**, 995–1005 (1996).

136. Schulz, O. *et al.* Toll-like receptor 3 promotes cross-priming to virus-infected cells. *Nature* **433**, 887–892 (2005).

137. Lee, J. *et al.* Molecular basis for the immunostimulatory activity of guanine nucleoside analogs: activation of Toll-like receptor 7. *Proc. Natl Acad. Sci. USA* **100**, 6646–6651 (2003).

138. Hornung, V. *et al.* Sequence-specific potent induction of IFN-α by short interfering RNA in plasmacytoid dendritic cells through TLR7. *Nature Med.* **11**, 263–270 (2005).

139. Judge, A. D. *et al.* Sequence-dependent stimulation of the mammalian innate immune response by synthetic siRNA. *Nature Biotechnol.* **23**, 457–462 (2005).

140. Hemmi, H. *et al.* A Toll-like receptor recognizes bacterial DNA. *Nature* **408**, 740–745 (2000). This paper reports on the discovery and function of TLR9.

141. Lin, R. & Hiscott, J. A role for casein kinase II phosphorylation in the regulation of IRF-1 transcriptional activity. *Mol. Cell. Biochem.* **191**, 169–180 (1999).

142. Kondo, T. *et al.* Identification and characterization of nucleophosmin/B23/numatrin which binds the anti-oncogenic transcription factor IRF-1 and manifests oncogenic activity. *Oncogene* **15**, 1275–1281 (1997).

143. Dornan, D. *et al.* Interferon regulatory factor 1 binding to p300 stimulates DNA-dependent acetylation of p53. *Mol. Cell. Biol.* **24**, 10083–10098 (2004).

144. Reily, M. M., Pantoja, C., Hu, X., Chinenov, Y. & Rogatsky, I. The GRIP1:IRF3 interaction as a target for glucocorticoid receptor-mediated immunosuppression. *EMBO J.* **25**, 108–117 (2006).

145. Mamane, Y., Sharma, S., Petropoulos, L., Lin, R. & Hiscott, J. Posttranslational regulation of IRF-4 activity by the immunophilin FKBP52. *Immunity* **12**, 129–140 (2000).

146. Kim, Y. M. *et al.* Roles of IFN consensus sequence binding protein and PU.1 in regulating IL-18 gene expression. *J. Immunol.* **163**, 2000–2007 (1999).

147. Qing, J. *et al.* Transforming growth factor β/Smad3 signaling regulates IRF-7 function and transcriptional activation of the β interferon promoter. *Mol. Cell. Biol.* **24**, 1411–1425 (2004).

148. Darnell, J. E. Jr., Kerr, I. M. & Stark, G. R. Jak–STAT pathways and transcriptional activation in response to IFNs and other extracellular signaling proteins. *Science* **264**, 1415–1421 (1994).


Acknowledgements  
We thank E. Barsoumian, A. Takaoka, Y. Ohba and H. Yanai for valuable discussion and advice. This work was supported by Kakenhi (Grants-in-Aid for Scientific Research) on the Priority Area ‘Integrative Research Toward the Conquest of Cancer’, from the Ministry of Education, Culture, Sports, Science and Technology (Japan).

Competing interests statement  
The authors declare no competing financial interests.


DATABASES  
The following terms in this article are linked online to: Entrez Gene: http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gene IRF1 | IRF2 | IRF3 | IRF4 | IRF5 | IRF6 | IRF7 | IRF8 | IRF9


FURTHER INFORMATION  
Tadatsugu Taniguchi’s homepage: http://www.immunol.m.u-tokyo.ac.jp/english Access to this links box is available online.
