
MICROBIOLOGY AND MOLECULAR BIOLOGY REVIEWS, Sept. 2006, p. 704–728
1092-2172/06/$08.00+0 doi:10.1128/MMBR.00004-06
Copyright © 2006, American Society for Microbiology. All Rights Reserved.

# Genetics of *Streptomyces rimosus*, the Oxytetracycline Producer

Hrvoje Petković,¹ John Cullum,² Daslav Hranueli,³ Iain S. Hunter,⁴ Nataša Perić-Concha,⁵ Jasenka Pigac,⁶ Arinthip Thamchaipenet,⁷ Dušica Vujaklija,⁶ and Paul F. Long⁵*

Biotechnical Faculty, University of Ljubljana, Ljubljana, Slovenia¹; Department of Genetics, University of Kaiserslautern, Kaiserslautern, Germany²; Faculty of Food Technology and Biotechnology, University of Zagreb, Zagreb, Croatia³; Royal College, University of Strathclyde, Glasgow, Scotland⁴; School of Pharmacy, University of London, London, England⁵; Department of Molecular Biology, Ruđer Bošković Institute, Zagreb, Croatia⁶; and Faculty of Science, Kasetsart University, Bangkok, Thailand⁷

## INTRODUCTION ..... 704
## BACKGROUND ..... 706
## CLONING AND MOLECULAR ARCHITECTURE OF POLYKETIDE GENE CLUSTERS ..... 707
- Cloning of Tetracycline Resistance Genes ..... 707
- Cloning of Genes Involved in OTC Biosynthesis ..... 707
- Gene Disruptions in the Rimocidin Biosynthetic Gene Cluster ..... 710
## S. RIMOSUS GENOME ..... 710
- Chromosome ..... 710
- Plasmids ..... 711
- Prophages ..... 712
- Genetic Instability ..... 713
## tRNA, rRNA, AND GENE EXPRESSION IN S. RIMOSUS ..... 714
## SYSTEMS FOR PROMOTING GENE EXCHANGE AND CHROMOSOMAL LINKAGE MAPPING ..... 717
- Systems Promoting Gene Exchange ..... 717
- Chromosomal Linkage Mapping ..... 717
- S. *rimosus* as a Host-Vector System ..... 718
- Introduction of DNA into S. *rimosus* ..... 718
- Vectors for S. *rimosus* ..... 719
## MUTATION AND SELECTION ..... 720
- Random Mutagenesis ..... 720
- Localized Mutagenesis ..... 720
- Mutations affecting regulatory antibiotic genes ..... 721
- Other mutations affecting antibiotic yield ..... 721
- Screening and Selection ..... 722
## FUTURE PROSPECTS ..... 723
## ACKNOWLEDGMENTS ..... 723
## REFERENCES ..... 723

---

### INTRODUCTION

The need to develop new antibiotics to fight pan-drug-resistant bacteria presents significant challenges and opportunities for the biotechnology and pharmaceutical industries. Antibiotics are commercially among the largest group of pharmaceutical products, with an estimated global market value expected to exceed $8.5 billion per annum within the next 5 years. Since the discovery of penicillin more than 70 years ago, a conservative estimate of compounds exhibiting antibiotic activity described in the scientific literature is >20,000, yet only a small fraction have been developed for human, agricultural, or veterinary use. The great majority of these compounds are synthesized by different *Streptomyces* species (27, 232). Some of the most clinically and commercially significant natural *Streptomyces* products belong to the tetracycline (TC) class of antibiotics, which have been used to treat infections caused by both gram-positive and -negative bacteria as well as infections caused by intracellular pathogens, including mycoplasmas, chlamydiae, and rickettsiae (45, 84). It is not surprising that since the introduction of these broad-spectrum antibiotics over 5 decades ago, some of the most intensively studied species, from a genetic point of view, among industrially important *Streptomyces* species have been the tetracycline producers. Béhal and Hunter (23) estimated in 1995 that the global production of tetracyclines was in excess of 5,000,000 kg, and there is no evidence that this has diminished in recent years. Indeed, the bulk usage of tetracyclines is likely to have increased over the last 10 years, mainly due to their use in aquaculture. Although resistance to existing tetracyclines has meant that they can no longer be used effectively for many clinical infections, there are still some important high-risk infections for which tetracyclines are the drugs of choice. Among them are infection with the malaria parasite, *Plasmodium falciparum* (164); Lyme disease, for which the semisynthetic drug doxycycline is the agent of choice for early diagnosis when the symptoms do not involve the central nervous system (82); and

* Corresponding author. Mailing address: The School of Pharmacy, University of London, 29/39 Brunswick Square, London WC1N 1AX, United Kingdom. Phone: 44-20-77535868. Fax: 44-20-77535868. E-mail: paul.long@ams1.ulsop.ac.uk.

rickettsial tick-borne diseases, where doxycycline is again the agent of choice (216). Tetracycline is used in the eradication of infection by *Helicobacter pylori*, the causative agent of most duodenal ulcers. Formulated with metronidazole and bismuth, it eradicates the infection in around 90% of cases, and new formulations of this triple therapy are being developed (79). Emerging infections, such as infections with community-acquired methicillin-resistant *Staphylococcus aureus*, can be treated effectively with tetracyclines (135). Should anthrax ever reemerge, or more likely be used as an infective agent in bioterrorism, doxycycline (a tetracycline derivative) or ciprofloxacin will be the drug of choice (152). Tetracyclines have an increasing role in the treatment of noninfective disease. Their roles as inhibitors of angiogenesis (130) and inhibitors of metalloproteinases have been studied intensely (e.g., see reference 131). These effects are not due to antibiotic action, since tetracycline derivatives which are structurally related but do not possess any anti-infective activity also act in this way (120). Tetracyclines (or derivatives such as anhydrotetracycline) are now used as inducers in controllable eukaryotic expression systems. These systems, adapted from the mechanism of regulation of bacterial tetracycline resistance from transposon Tn10, use a modified bacterial *tet* repressor that is fused to the activating domain of virion protein 16 of herpes simplex virus to generate a tetracycline-controlled transactivator that can be expressed constitutively in HeLa cells (77). These systems have been adapted for a wide range of applications, not least of which are strategies for gene therapy (78). Oxytetracycline (OTC) finds particularly heavy use in aquaculture, where 500-kg doses may be used in one campaign (124). It continues to be approved within the European Union and elsewhere for fish health care (12), despite some evidence that tetracycline-resistant infections are becoming more prevalent.

Perhaps the most significant development in the tetracycline field in recent years has been the introduction of glycyl-glycyl-9-substituted derivatives (214)—the so-called glycyclines—that are now marketed under the trademark Tigecycline. This family of new compounds is active against a number of clinical infections that had become resistant to conventional tetracycline therapy. Among these are multiply resistant *Staphylococcus* spp. (including vancomycin-insensitive strains), *Streptococcus pneumoniae*, *Enterococcus* spp. (including vancomycin-resistant strains), and some extended-spectrum-β-lactamase-producing isolates of the *Enterobacteriaceae* (68). Significantly, Tigecycline was the only anti-infective agent to receive FDA approval in the first half of 2005 (235).

The primary tetracyclines, namely, TC, OTC, chlortetracycline (CTC), and 6-demethylchlortetracycline (Fig. 1A), are produced during submerged fermentations of various *Streptomyces* species isolated from soils during the course of natural product screening programs by industrial as well as academic research groups. The majority of commercially significant tetracyclines are produced by strains of *Streptomyces aureofaciens* and *Streptomyces rimosus*, although other species have also been employed (160). Because of the great economic value of tetracycline antibiotics, the results of strain development programs have not been disseminated widely in the open scientific literature. However, procedures developed for the genetic manipulation of *Streptomyces* species producing tetracyclines have been published. The number of publications concerning *S.*

aureofaciens* genetics is not significant. For this species, scientists have been more interested in elucidating the biochemical pathway leading to TC biosynthesis (see references 21 and 103 and references therein). On the other hand, >100 publications have appeared describing the genetic manipulation of OTC-producing *S. rimosus* strains, affording this species one of the best-developed genetic systems of any streptomycete, second only to *Streptomyces coelicolor*. Unlike the genetics of *S. coelicolor*, which has been reviewed extensively (97), there is a paucity of curated information for *S. rimosus*. Accordingly, the present review concentrates on *S. rimosus* genetics, paying particular attention to the application of modern molecular approaches that will be invaluable for the development of new tetracyclines by combinatorial biosynthesis and novel recombinant approaches (109, 122). It should also be noted, however, that *S. rimosus* has been known for some time to be a producer of other biologically active secondary metabolites, such as the polyene antifungal agent rimocidin (51) (Fig. 1B).

Much of what is currently known about *S. rimosus* genetics comes from results published by three groups in Russia at the Institute of Genetics and Selection of Industrial Microorganisms and the All-Union Research Institute of Antibiotics, Moscow; from Pfizer Central Research in the United Kingdom, in collaboration with scientists from the John Innes Centre and the University of Glasgow; and from the PLIVA Research Institute in Croatia, together with scientists from the University of Zagreb and the Ruđer Bošković Institute. The majority of the important contributions to this field have been published in the English language literature, but a considerable number have also appeared elsewhere in Russian or Croatian. Among the latter, only those deemed most relevant have been translated for inclusion in this perspective. To complicate an objective review of the literature even further, these groups have used four strains of *S. rimosus*, but with several different designations for each.

During the late 1950s and until the early 1970s, the majority of published work performed in Russia was done with *S. rimosus* mutants derived from a strain previously called *Actinomyces rimosus* LS-T118 (155). More recently, *S. rimosus* strain 183 has been used to study genetic instability (see reference 210 and references therein). Since the early 1960s, the groups working in Croatia have been generating mutants isolated from two independent *S. rimosus* strains. These strains are ATCC 10970 (NRRL 2234) from the American Type Culture Collection, whose name has been abbreviated to R7 in the majority of publications, and *S. rimosus* R6, which is a soil isolate from the Faculty of Food Technology and Biotechnology, University of Zagreb, used for the development of mutants for the commercial production of OTC by PLIVA (Fig. 2). The linkage maps of *S. rimosus* R6 and R7 are almost identical (10). During the 1970s, the group from Pfizer Central Research, in collaboration with scientists from the John Innes Centre in the United Kingdom (71) and, more recently, scientists from the Institute of Genetics, University of Glasgow, published work using mutants derived from the prototrophic strain M4018 (37), employed commercially for the production of OTC by Pfizer. In the scientific literature, these bacteria have come to be known as the Russian (LS-T118), Zagreb (R6), and Pfizer (M4018) strains (94, 117). Other strains have also been used sporadically as donors or recipients of DNA (e.g.,

PETKOVIĆ ET AL.
MICROBIOL. MOL. BIOL. REV.

A

![image](https://i.imgur.com/1234567.png)

Tetracycline

![image](https://i.imgur.com/8901234.png)

Oxytetracycline

![image](https://i.imgur.com/5678901.png)

Chlortetracycline

![image](https://i.imgur.com/2345678.png)

6-demethylchlortetracycline

B

![image](https://i.imgur.com/9012345.png)

Rimocidin

FIG. 1. Chemical structures of primary tetracyclines (A) and of rimocidin (B).

During the early 1970s, circular chromosomal linkage maps of three *S. rimosus* strains were constructed. After a preliminary report at the First International Symposium on the Genetics of Industrial Microorganisms in 1970 (7), these maps were independently published by Friend and Hopwood (71) and by Alačević and collaborators (10). Data from these maps were combined to also include the positions of markers at new loci (9). Meanwhile, optimal conditions for mutagenesis with *N*-methyl-*N′*-nitro-*N*-nitrosoguanidine (MNNG; referred to as nitrosoguanidine [NTG] in some of the cited references) and UV irradiation were also developed (8, 39, 52, 53). At about the same time, the first estimate of the size of the *S. rimosus* genome was published (24). In the early 1970s, studies following three avenues of research were initiated. These were the biochemical characterization of non-OTC-producing mutants and mapping of the OTC genes (180, 200), characterization and persistence of *S. rimosus* phages (104, 107, 193, 225), and genetic and physical characterization of circular and linear *S. rimosus* plasmids (31, 41, 72, 80, 201, 212).

BS-21, 907, 183, RCC, and PG3) (41, 155, 188, 198, 212), but no more data on their genetics have been published to date.

## BACKGROUND

The first report of recombination in *S. rimosus* was published by Alikhanian and Mindlin in 1957 (13). From this time and until the late 1960s, almost all research on *S. rimosus* genetics concentrated on the following four areas: (i) isolation of nutritional and non-OTC-producing mutants, (ii) classification of the latter using the principle of cosynthesis on solid media, (iii) development of mapping procedures by selective and heteroclone analysis, and (iv) establishment of linkage between mutations by conjugation (6, 14, 15, 17, 32, 33, 54, 67, 155, 156, 157, 159). During this initial period, interspecific recombination among tetracycline-producing streptomycetes, as well as recombination with some more distantly related species, was also reported (5, 11, 187). All early reports of interspecific recombination must be treated with caution, because morphological variability and genetic instability can lead to the appearance of apparent recombinants. At the time of the early reports, DNA methods were not available to prove the presence of sequences from both parents.

The *otrB* gene encodes an integral membrane protein that is responsible for efflux of OTC from the cell (151, 167, 198). A third OTC resistance gene, *otrC*, has been cloned from *S*. *rimosus*, but the mechanism of resistance encoded by this gene has yet to be elucidated (I. S. Hunter, unpublished data).

Transcriptional analysis of *otrA* revealed two promoters, *otrAp*₁ and *otcCp*₁. During exponential growth, the *otrA* gene is transcribed as a single cistron from *otrAp*₁. At the beginning of the stationary phase of growth and during OTC biosynthesis, the *otrAp*₁ promoter is silent, and *otrA* is transcribed as the 3′ gene of a polycistronic mRNA originating from the *otcCp*₁ promoter (60, 61, 149). This arrangement makes good teleological sense, as the *otrA* gene is always expressed. During exponential growth, the cells would otherwise be susceptible to the action of neighboring cells that are making OTC, whereas during the OTC production phase the continued expression of *otrA* is ensured, in concert with that of the other production genes. MacGregor-Pryde (137) identified a gene upstream of *otrB*, *otrR*, whose product is a putative repressor. Thus, the overall topology of the *otrB-otrR* region mirrors that found in Tn10, where expression of the tetracycline efflux pump is controlled by a divergent repressor.

### Cloning of Genes Involved in OTC Biosynthesis

The major steps in the biosynthesis of tetracycline antibiotics were elucidated in the late 1960s from studies of the CTC producer, *S*. *aureofaciens* (145, 146). OTC is an aromatic polyketide antibiotic composed of a C₁₉ backbone formed after the sequential condensation of an aminated starter unit, derived from malonyl-coenzyme A (malonyl-CoA), to eight acetate extender units derived from malonyl-CoA (218, 219). The tetracyclic backbone is assembled by a so-called type II polyketide synthase (PKS) (96), which is a multienzyme complex that catalyzes the iterative assembly of the backbone by a mechanism analogous to bacterial fatty acid biosynthesis. This is then followed by cyclization and modification of the nascent polyketide to give the final OTC product. Molecular genetic analysis of OTC biosynthesis in two *S*. *rimosus* strains (M4018 and R6) revealed that the pathway genes are clustered together, covering approximately 30 kb on the chromosome, and are flanked by the resistance genes *otrA* and *otrB* (29, 35, 112). Genetic mapping and cross-feeding studies of the *otc* mutants initially suggested that there were at least four complementation groups (200). Subsequently, DNA sequencing of the biosynthetic cluster revealed 23 open reading frames (116) (Fig. 3).

The gene *otcY1-1* encodes a ketoacyl-ACP synthase, *otcY1-2* encodes the second component of this heterodimeric condensing enzyme, and *otcY1-3* encodes an acyl carrier protein. Although acetate is commonly the starter unit for polyketide biosynthesis catalyzed by type II PKSs, OTC is somewhat unusual in that it has a carboxamido moiety at the starter position. This is assumed to be derived from malonyl-CoA, which is believed to be incorporated preferentially in the presence of *otcY2-3*, encoding a CoA ligase homologue that would attach a CoA group to malonamate, and *OtcX3*, a putative acyltransferase that would load the malonyl-CoA onto the PKS. The malonamate would be supplied through the action of *OtcY1-4*, which is an amidotransferase. Zhang and collaborators

FIG. 2. Colony morphology of two *Streptomyces rimosus* strains. (Right) *S*. *rimosus* ATCC 10970 (NRRL 2234), abbreviated to strain R7. (Left) *S*. *rimosus* strain R6, also known as the Zagreb strain, isolated from soil by the Faculty of Food Science and Biotechnology, University of Zagreb. Note the differences in sporulation, an important determinant in developing fermentation parameters and systems for gene exchange.

rial biosynthesis, with potentially novel biological activities which are difficult to produce by traditional synthetic routes (125, 148, 175, 177).

## CLONING AND MOLECULAR ARCHITECTURE OF POLYKETIDE GENE CLUSTERS

The molecular genetics of OTC biosynthesis has been studied intensively by Butler, Binnie, and their collaborators (29, 35). The entire gene cluster for OTC biosynthesis has been cloned and sequenced from *S*. *rimosus* strain 15883 (116), while the *otc* biosynthetic genes from strain ATCC 10970 were also sequenced recently (239) (GenBank accession number DQ143963). Manipulations of *otc* biosynthetic genes have led to the biosynthesis of diverse “unnatural” natural products (73, 127, 175, 177, 239), revealing the potential for manipulation of this type II polyketide system.

### Cloning of Tetracycline Resistance Genes

OTC is a bacteriostatic antibiotic that inhibits bacterial growth by reversibly binding to the 30S ribosomal subunit, preventing formation of the amino-acyl tRNA–ribosome complex. Three OTC (*otr*) genes encoding resistant determinants, designated *otrA*, *otrB*, and *otrC*, have been isolated from *S*. *rimosus*. OtrA protects the ribosome from translational arrest by tetracyclines (61, 167). Paralogues are present in other streptomycetes, e.g., *Streptomyces lividans* (58), and in tetracycline-resistant clinical infections (encoded by *tetO* and *tetM*). These proteins are similar to elongation factors and act by releasing tetracyclines from their inhibitory site on the ribosome (46).

708 PETKOVIĆ ET AL.

MICROBIOL. MOL. BIOL. REV.

![Diagram](#)

FIG. 3. Architecture of the oxytetracycline gene cluster. The four regions \( otcZ \), \( otcX \), \( otcD \), and \( otcY \) correspond to loci originally characterized by genetic analysis and cross-feeding studies of blocked mutants. The open boxes show the positions of identified genes in the cluster, with their orientations marked by arrows. (Modified from reference 111 with permission of the publisher.)

tors (239) recently proposed a similar but subtly different route that involves direct amidation of malonyl-CoA by OtcY1-4. This work was based on heterologous expression of \( otc \) genes in \( S. coelicolor \). They were unable to detect any enhanced incorporation of an amidated starter unit when OtcX3 (the acyltransferase) was coexpressed, but they did not examine OtcY2-3. At present, the reasoning for the involvement of OtcY2-3 is teleological—most type II PKS clusters have no CoA ligase, and hence the former scheme incorporates this function on that basis.

Disruptions of \( otcD1 \) (encoding an aromatase/cyclase) resulted in four novel carboxamido-derived polyketides with shorter chain lengths (177). This provided strong evidence that the carboxamido group was present from the start, rather than the nascent polyketide backbone being amidated later in the biosynthetic scheme. The caboxamido-derived decaketide isolated by Zhang and collaborators (239) by heterologous expression of \( otcY1-4 \) reinforces the view that the caboxamido group is present from the outset.

When the nascent polyketide chain is completed, it is subsequently reduced and folded into a tetracyclic structure by the action of the gene products of \( otcY2-1 \) (a ketoreductase) and \( otcD1 \) (cyclase/aromatase). The folded polyketides then act as substrates for tailoring enzymes (Fig. 4). These include methylation at C-6 to form the first enzyme-free intermediate, 6-methylpretetramid (6-MPT) (145). There are three methylation steps in OTC biosynthesis, including C-6 methylation and N-dimethylation of the amino group at C-4. Two gene products, OtcZ and OtcY2-5, are methyltransferase homologues. By studying \( otc \) mutants, Rhodes and collaborators (200) initially proposed that \( otcZ \) catalyzes C-6 methylation, whereas the \( otcD \) region could be involved in the dimethylation step to form 4-aminodimethylaminoanhydrotetracycline (4-amino-ATC). However, bioinformatic analyses of the DNA sequence for the \( otc \) cluster deduced that the second methyltransferase was encoded by \( otcY2-5 \).

The predicted amino acid sequence of OtcZ (150) revealed that this protein is similar to bovine hydroxyindole \( O \)-methyltransferase (119) and to the carboxy-terminal domain of the TcmN protein (a putative \( O \)-methyltransferase) in *Streptomyces glaucescens* (215). Since the cosynthesis pattern of the \( otcZ(151) \) mutant indicated that it was blocked before C-4

dimethylation of 4-ketodimethylaminoanhydrotetracycline (4-keto-ATC) and after 4-amino-ATC was formed (200), it had been postulated that OtcZ was involved in 6-methylation of pretetramid (35). However, an amino acid sequence comparison with OtcZ showed significant homology to the C-4 \( N \)-methylase (\( ctc-ORF2 \)) of the \( S. aureofaciens \) chlortetracycline gene cluster (202). This gene product is involved in the dimethylation of 4-amino-ATC. There is also high sequence similarity of \( otcZ \) to the \( cts-6 \) gene, which was isolated by complementation of a mutant which failed to methylate pretetramid at the C-6 position during chlortetracycline biosynthesis in \( S. aureofaciens \) (48). It can be concluded, therefore, that the \( otcD \) region was misassigned in the mutagenesis study of Rhodes and collaborators (200).

Conversion of 6-MPT to 4-hydroxy-6-methylpretetramid (4-hydroxy-6-MPT) occurs through the activity of a specific hydroxylase. In the original mutagenesis studies performed by Rhodes and collaborators at Pfizer, the \( otcX \) locus was assigned to this biochemical step (200). OtcX1 has striking similarity to the \( S. coelicolor \) ActVA-ORF2 hydroxylase from the actinorhodin cluster (38, 150). It is likely that OtcX1 is an ancillary protein in the hydroxylation reaction rather than the catalyst per se. Several other genes for hydroxylase-like proteins have been identified in the \( otc \) gene cluster. These include \( otcC \) (6-hydroxylase), \( otcD2 \), and \( otcY1-5 \). It has yet to be established which gene products hydroxylate the substrate at the C-4 position at this point of the biosynthetic scheme and which add a hydroxyl group later at the C-5 position.

Reduction of the hydroxyl group at the C-4 position of 4-hydroxy-6-MPT is controlled by a gene designated \( otcD3 \), whose product shows homology to ketoreductases. Amination at C-4 results in the formation of 4-amino-anhydrotetracycline. This step is likely catalyzed by the gene product of \( otcX2 \), which has high similarity with aminotransferases. Conversion of 4-amino-ATC to ATC occurs by the introduction of two methyl groups at C-4, most likely by the product of \( otcZ \), as discussed above.

ATC oxygenase catalyzes the hydroxylation of ATC at C-6 in the presence of oxygen and NADPH and results in the formation of 5a,11a-dehydrotetracycline (5-DHTC). The 52-kDa subunit of this enzyme has been purified, and the N-terminal amino acid sequence has been obtained (35). These data were

FIG. 4. Proposed gene functions in the biosynthesis of oxytetracycline. Nonaketamide (nascent linear polyketide chain), pretetramid, and the intermediates 6-MPT, 4-hydroxy-6-MPT, 4-keto-ATC, ATC, 5-DHTC, and DHOTC, as well as OTC, are shown. Nonaketamide carbon atoms are numbered starting from the enzyme (-S-E), while carbon atoms from the first tetracyclic structure (pretetramid) are numbered according to IUPAC nomenclature.

used to design an oligonucleotide probe to isolate the gene from an *S. rimosus* cosmid library (29). The oligonucleotide hybridized to the DNA region between *otcZ* and *otcX*, locating *otcC*. The deduced DNA sequence indicates that OtcC belongs to the family of bacterial flavin-type hydroxylases. The *otcC* promoter is located in the region between *otcC* and *otcX-ORF1*. The *otcC* transcript appears to be polycistronic and includes the *otrA* gene. Recently, a recombinant strain of *S. rimosus* that was disrupted in the genomic copy of *otcC* was found to synthesize a novel C<sub>17</sub> polyketide. This result indicated that the absence of the *otcC* gene product significantly affects the ability of the OTC minimal PKS to synthesize a polyketide product of the correct chain length, indicating that OtcC is an essential partner in the quaternary structure of the synthase complex (175).

A hydroxyl group is then added at C-5 of 5-DHTC to form 5a,11a-dehydrooxytetracycline (DHOTC). Whether the product of the *otcY1-5* or *otcD2* gene is involved is discussed above. The last step in OTC biosynthesis involves a dehydrogenase. The enzyme that converts 5-DHTC to TC in *S. aureofaciens* has been purified (165), and OtcY2-4 was assigned to this function on the basis of its amino-terminal sequence (217). After OTC has been formed, it is exported from the cell by OtrB. Located between the *otcY1* and *otrB* genes, *otrR* encodes a transcriptional regulator that does not resemble typical tetracycline repressors (190) but shows some similarity to so-called multidrug repressors from *Escherichia coli*. The juxtaposition and divergent transcription of *otrR* and *otrB* indicate that OtrR may be the repressor for *otrB*, but this notion remains to be tested.

### Gene Disruptions in the Rimocidin Biosynthetic Gene Cluster

Rimocidin is a polyene antifungal antibiotic produced by *S. rimosus* (51). The partial structure of rimocidin was elucidated by Cope and colleagues (47) and by Pandey and Rinehart (170), although absolute stereochemical assignments of the 28-membered aglycone rimocidinolide came later (209). Recently, rimocidin production by *Streptomyces diastaticus* var. 108 was reported; this strain also produces OTC (174).

The molecular genetics of rimocidin biosynthesis has been investigated independently with both *S. rimosus* R7 (A. Thamchaipenet, unpublished data) and *S. diastaticus* var. 108 (203), with some sequence data already having been deposited (GenBank accession no. AY423269, AY701054, AY442225, and DQ174320). Disruption of rimocidin synthase (RMS) resulted in nonproducing mutants in both *S. rimosus* R7 (Thamchaipenet, unpublished data) and *S. diastaticus* var. 108 (203), validating that the RMS genes are involved in rimocidin biosynthesis.

Deduced polypeptide analysis of the rimocidin gene cluster indicated that it is synthesized by a modular type I PKS system (59), unlike OTC, which is synthesized by a type II PKS. The domains of the rimocidin synthase of *S. rimosus* strain R7 (Thamchaipenet, unpublished data) (Fig. 5) are organized into 14 modules in a linear manner, similar to that of other type I PKSs, and they share common features with other polyene antibiotic gene clusters, as previously described (18). The chemical structure of rimocidin indicates that the starter unit is

---

**FIG. 5. Proposed architecture of rimocidin-producing modular (type I) polyketide synthase. Modules are numbered according to the order of chain elongation. KS, ketosynthase; AT, acyltransferase; KR, ketoreductase; DH, dehydratase; ER, enoyl reductase; ACP, acyl carrier protein.**

butyryl-CoA, which is likely formed by the condensation of two acetyl-CoA units catalyzed by crotonyl-CoA reductase. If acetyl-CoA is incorporated, then the homologous compound CE108 is formed (203).

#### S. RIMOSUS GENOME

The first two *Streptomyces* genomes that were completely sequenced were those of *S. coelicolor* A3(2) (26) and *Streptomyces avermitilis* (118). They possess linear chromosomes of 8.7 Mb and 9.0 Mb, respectively, and have G+C contents of 72.1% and 70.7%, respectively. There are about 7,600 protein-coding genes in each species, and at least 30% of the genes in one species do not have a homologue in the other species. In general, the conserved genes are concentrated in the central area of the chromosomes, and the end regions of the chromosomes are less conserved.

##### Chromosome

The first information about the *S. rimosus* genome was obtained using DNA renaturation kinetics (24) and compared *S. coelicolor* A3(2) and *S. rimosus* R7 (ATCC 10970). These two species did not differ significantly in G+C content (73.0 and 71.4 mol% G+C, respectively) or genome size (10.8 Mb and 10.4 Mb, respectively), as calculated by DNA reassociation kinetics. All renaturation analyses of *Streptomyces* species have shown genome sizes considerably larger than that of *E. coli*, although there have been some disagreements depending on

the strains used, with estimates varying between 1.5- and 3-fold larger than the size of *E. coli*. The very high G+C contents of these species might be expected to lead to underestimations of genome size because of faster reannealing than that for lower-G+C-content DNA of the same complexity or because of the presence of repetitive DNA, which in several species seems to account for up to 10% of the genome. However, neither *S. rimosus* R7 (24) nor *S. rimosus* NRRL 2455 (221) showed any detectable repetitive DNA in renaturation analysis.

Most recent work on genome size and structure has used *S. rimosus* R6 (the Zagreb strain). Pulsed-field gel electrophoresis (PFGE) was used to construct a restriction map of the chromosome of strain R6-501 (Fig. 6) for the rarely cutting enzymes AseI and DraI (172). The map is linear, like those of other *Streptomyces* species. The total chromosome size was estimated to be 8 Mb, similar to that of *S. coelicolor* A3(2). This would correspond to a true size of about 8.6 Mb (81) because the *Saccharomyces cerevisiae* chromosome markers used underestimate the sizes of high-G+C-content fragments. The ends of the chromosome are inverted repeats of about 550 kb. The chromosome end (GenBank accession number AY043328) resembles that of other *Streptomyces* species (113), with several inverted repeats in a region of about 200 bp. The *dnaA* gene is located in the AseI C2 fragment, which means that the replication origin is at least 0.5 Mb from the center of the chromosome and is not as centrally located as that in *S. coelicolor* A3(2). However, the *dnaA* gene of *S. avermitilis* is also displaced about 0.8 Mb from the center of the chromosome. Only a small number of other genes (*recA*, rRNA operons, and *att*-pSAM2) have been localized in the chromosome of *S. rimosus*. It is interesting that their positions differ significantly from those of their homologues in *S. coelicolor* A3(2). However, too few data are available to conclude that *S. rimosus* possesses a chromosomal organization radically different from that of the two sequenced species, *S. coelicolor* A3(2) and *S. avermitilis*, which show a high degree of conservation of gene order (118).

AseI and DraI digests of DNA from strain *S. rimosus* R7 were very similar to those of DNA from R6 (172). The major difference was that there were additional sequences at one chromosome end of R7 instead of the long inverted repeat in R6 (171). This suggests that the ancestor of strain R6 was very similar to R7 and that the long terminal inverted repeats were created by a relatively recent recombination event. Such events have been observed in *Streptomyces ambofaciens* (69, 234).

### Plasmids

Plasmids are common in *Streptomyces* species, and a wide range of sizes and copy numbers have been observed. Both circular and linear plasmids are known. Apart from some cases of antibiotic biosynthesis gene clusters being carried on plasmids (161), there are relatively few well-documented phenotypic properties encoded by plasmids (25). Many plasmids are conjugative. Often, conjugation on solid medium is accompanied by the phenomenon of lethal zygosis and by pock formation, where areas of mycelium that are acting as recipients in conjugation show delayed sporulation and can be recognized on agar medium (see reference 101 and references therein). The first physical evidence of a naturally occurring plasmid

FIG. 6. Restriction map of the chromosome of *Streptomyces rimosus* R6-501 for the enzymes AseI (outer arc) and DraI (inner arc). The terminal inverted repeats are drawn as a stem structure. The numbers of the linking clones are indicated adjacent to the corresponding restriction sites; the missing DraI linking clone is indicated by an asterisk. The cosmid clones carrying the ends of the terminal inverted repeats (C-136 and C-123) are also indicated. The OTC cluster and the *attB*-pSAM2 gene have been localized precisely. The other markers have been localized only to particular AseI and DraI fragments. (Reprinted from reference 172 with permission of the publisher.)

in *S. rimosus* (*Actinomyces rimosus* 907) was published by Stepnov and collaborators (212). A circular DNA molecule of 55 kb was detected in crude DNA preparations. Total DNA was analyzed electrophoretically in an agarose gel, followed by electron microscopic analysis of DNA isolated from the gel. Although attempts to isolate covalently closed circular DNA from *S. rimosus* RCC by CsCl-ethidium bromide density gradient centrifugation were unsuccessful, Chardon-Loriaux and collaborators (41) succeeded in detecting and purifying extrachromosomal DNA by agarose gel electrophoresis and electroelution. Electron microscopy did not show any circular molecules. This was in agreement with the results of restriction mapping showing a linear plasmid DNA molecule of about 43 kb. The plasmid, named pSRM, exhibited the lethal zygosis phenomenon.

Most of the work on *S. rimosus* plasmids has used *S. rimosus* R6. R6-501, the strain used to construct the chromosomal restriction map, did not carry any detectable circular plasmids but had a 387-kb linear plasmid, pPZG101 (81). The pPZG101 plasmid has a unique central region of about 30 kb flanked by inverted repeats of about 180 kb (173). Strain R6-65, which is an ancestor of R6-501, carries a smaller linear plasmid of about 310 kb, pPZG102, which does not have extensive inverted repeats. One end of pPZG102 seems identical to the inverted repeat of pPZG101. This suggests that pPZG101 was derived

from pPZG102 by a recombination event between copies of the plasmid in inverted orientation to generate the long inverted repeats. Curing of pPZG101 has been observed. When 17 auxotrophic mutant strains were analyzed, 5 no longer carried free plasmid and 2 showed no hybridization to pPZG101. The other three strains had integrated parts of the plasmid into the chromosome, indicating that there are frequent interactions between the plasmid and the chromosome. During the study of genetic instability in *S. rimosus* R6 (see below), a mutant was isolated that overproduced oxytetracycline. This mutant (MV17) proved to carry a 1-Mb linear plasmid, pPZG103, derived from pPZG101 and carrying the *otc* cluster (173). The chromosome had a parental structure, so there was an increased copy number of the *otc* cluster due to the extra copy on the plasmid. pPZG103 arose from a crossover between pPZG101 and the chromosome, such that about 200 kb at one end is derived from pPZG101 and the other 800 kb is the chromosomal terminal sequence. A second mutant derived in the study (MV25) proved to have replaced a chromosome end with one end of plasmid pPZG101; the details of this mutant are discussed below.

Classical genetic analysis by the Pfizer group with strain M4018 (derived from *S. rimosus* R7) defined plasmid SRP2 and found a derivative, SRP2′, which carries the *otc* cluster. PFGE analysis showed that *S. rimosus* R7 carries a 310-kb plasmid with a restriction pattern very similar to that of pPZG102 from *S. rimosus* R6-65. The SRP2′-carrying strain had replaced the original linear plasmid of about 310 kb with a much larger one, of about 950 kb, suggesting that an event similar to that observed for *S. rimosus* R6 had occurred (81).

### Prophages

For more than 50 years, it has been known that phages can attack streptomycetes, but they did not attract much attention until their undesirable interference with industrial fermentations was recognized (134, 233). Although problems in fermentations tend to be associated with virulent phages, the study of temperate phages is also important for industrial strains. On the one hand, the lysogenic status of a strain might affect its susceptibility to infections during fermentation, and on the other hand, the presence of a prophage might affect antibiotic yields. Although phage-like particles in lysogenic cultures may be characterized by electron microscopy, an effective study of lysogenic phages is dependent on a suitable host strain for growth and plaque formation. Lysogens show superinfection immunity, so a strain without prophage is necessary for plaque formation. In some cases, such as ΦC31 of *S. lividans* (134), the phage has a broad host range and can be grown on many species of *Streptomyces*. If the phage has a narrow host range, it will probably be necessary to isolate a cured derivative of the host strain, which usually involves a good portion of serendipity.

Hranueli and collaborators (104, 106, 107) detected and isolated free phage particles from a liquid culture of *S. rimosus* R7. The phage, designated RP2, appeared to be a typical temperate DNA phage producing turbid plaques in lawns of sensitive *S. rimosus* R6 cells. The actinophage RP2 has a very narrow host range restricted to *S. rimosus* strains. Later work (193) showed that *S. rimosus* R6 is usually lysogenic for RP2 but that the strain used in these experiments had been spontaneously cured of its prophage. Its tadpole-shaped morphology and double-stranded DNA content place RP2 in group B1 (Fig. 7A) of the bacteriophage classification system (1). Although the latent period of actinophage-actinomycete systems is usually long, RP2—with its 6-h latent period—is the slowest-multiplying actinophage described so far (see Table 1 in reference 134). The lysogenic nature of RP2 was established on the basis of the following criteria: (i) a low spontaneous lysis frequency of \(2 \times 10^{-6}\) per cell, (ii) resistance to curing with actinophage-specific antiserum, (iii) a low spontaneous curing frequency of <0.05%, and (iv) immunity to superinfection with the homologous phage. Treatment with UV light and other agents did not lead to induction of the prophage. Clear plaque mutants of RP2, which failed to lysogenize sensitive cultures, arose at a frequency of about \(2 \times 10^{-5}\) per phage particle (104, 106, 107). However, no virulent mutants that could infect lysogenic strains were observed, probably because multiple mutations would be necessary. After mutagenic treatment of a lysogenic strain with MNNG, two classes of mutants were isolated that no longer produced actinophage. Mutants belonging to the first class were phage sensitive and probably cured of the prophage, whereas those of the second class retained immunity to superinfection and presumably carried defective RP2 prophage. Lysogeny with RP2 did not affect OTC production.

A second prophage, RP3 (Fig. 7B), was found fortuitously during transfection experiments with temperate *Streptomyces* phages. *S. rimosus* R6 protoplasts were used as recipients for phage DNA, and protoplasts of the auxotrophic mutant strain *S. rimosus* R6-554 were used to form a lawn of indicator cells to yield plaques arising from transfection. There was a large excess of plaque formation (193), which turned out to be due to phage RP3 being released from the lysogenic recipient strain. Strain R6-554 had been cured of RP3, presumably during the mutagenic treatment. Otherwise, RP3 was present as an integrated prophage in the chromosomes of both *S. rimosus* R6 and *S. rimosus* R7. RP3 showed typical properties of a lysogenic phage, including (i) production of turbid plaques on a lawn of sensitive *S. rimosus* R6-554 cells, (ii) lysogeny of sensitive cells, (iii) spontaneous induction of the prophage in lysogenized cells, and (iv) immunity of lysogenic cells to superinfection with the homologous phage. The actinophages RP2 and RP3 are heteroimmune. A defective prophage was also described for *S. rimosus* ATCC 10970 (226). However, if it was a narrow-host-range phage such as RP2 and RP3, its apparently defective nature might indicate only the absence of an appropriate host strain.

The RP2 and RP3 phages contain linear double-stranded DNA molecules of 64.7 kb and 62.4 kb, respectively, which have been mapped by restriction enzyme digestion (193). They have a G+C content of about 70%, which is indistinguishable from that of the chromosome. The DNA molecules of both phages have cohesive ends, suggesting site-specific staggered cutting of concatameric DNA resulting from rolling circle replication (42). The two phages show hardly any cross-hybridization, implying that they are not closely related. Both phages integrate into the chromosome by using specific attachment sites that have been located in the restriction maps of the phages. The positions of the integrated prophages were also

Vol. 70, 2006

A
B

FIG. 7. Electron micrographs of RP2 (A) and RP3 (B) particles negatively stained with potassium phosphotungstate. Magnification, ×75,000. Bars, 100 nm. (Reprinted from reference 104 [panel A] and reference 193 [panel B] with permission of the publisher.)

localized on the chromosomal restriction map (172). DNA from RP3 was used to generate ladders of concatemers suitable for use as high-G+C-content markers for PFGE (81).

### Genetic Instability

Genetic instability is common in Streptomyces species (227, 234). The instability affects only certain genes, and the genes affected differ between species. Characteristics that are commonly affected include antibiotic production and resistance, sporulation, pigmentation, and colony morphology. Genetic instability has been studied in detail for the oxytetracycline production strain S. rimosus R6-500 (80). When mycelial fragments were plated, 1 to 3% of colonies showed variant morphology. However, when spores were plated, there was extreme instability, with up to 80% of colonies being variants. The variants were very heterogeneous, being affected to various degrees in sporulation, pigmentation, colony morphology, oxytetracycline production, and oxytetracycline resistance. The parent strain does not sporulate very prolifically, and colonies have a characteristic Kuglof morphology and produce a dark brown pigment. Many variants show little or no sporulation, and the Kuglof colony morphology is also often lost. Oxytetracycline production levels correlate well with pigmentation. Many variants have a greatly reduced production of oxytetracycline and pigment. The most useful character for classifying variants proved to be resistance to oxytetracycline: class I variants have unchanged resistance levels compared to the parent strain, class II variants are sensitive to oxytetracycline, and class III variants have more resistance than the parent strain. Class I variants account for about 99% of the total and are very heterogeneous. Most do not sporulate, and many have low production levels. These variants can cause problems during commercial production because they can outgrow the parent (80, 110). Representative class I variants were analyzed by PFGE, but no changes were found. Class II mutants are sensitive to oxytetracycline. They account for about 1% of variants and have uniform properties, being nonpigmented and showing reduced sporulation. PFGE analysis (80, 172) showed that they have a deletion of one chromosome end, including the *otc* gene cluster, which lies about 600 kb from the end (Fig. 6). The sensitivity to oxytetracycline was explained by the fact that resistance genes flank the *otc* cluster (116). Initial attempts to define the extent of the deletion in class II mutants and to explain the origin of the 340-kb AseI junction fragment present in class II mutants were unsuccessful. However, further analysis suggested that class II mutants had undergone a fusion of two inverted copies of the chromosome, with deletion of terminal sequences, including the *otc* cluster (Fig. 8) (55). Similar mutants have been characterized for *S.* *ambofaciens* (234). Chromosome behavior and genetic structure suggest that this could represent an earlier state in the evolution of diploidy, perhaps found in the ancestors of modern *Streptomyces* spp. This departs from most theories of the origin of eukaryosis that suggest that such chromosome behavior occurred first in the haploid condition, followed by diploidy; clearly, this is an issue that warrants closer scrutiny (109).

Careful analysis of class II mutants revealed some more puzzling properties. Initial measurements indicated that, as expected, deletion of the *otc* cluster resulted in a loss of OTC production (80). However, more-sensitive methods showed that there was still a residual production of about 0.1 μg/ml, compared to 10 mg/ml in the parent strain (112). Furthermore, there appeared to be rare reversion events that restored OTC

resistance and production. PFGE analysis of the revertants (55) showed that some of them had a chromosome structure like that of the parent. Thus, although unlikely, contamination with the parent strain could not be excluded. The other revertants showed a novel PFGE pattern (although they were clearly *S. rimosus* strains) which could not be explained by contamination. It was suggested that the “deleted” sequences might still be present at a low copy number in the mycelia of class II variants, and it was possible to detect the *otcC* gene at a copy number of about 0.1% using real-time PCR. Similar low-copy-number retention of “deleted” terminal sequences was also observed in *S. lividans* 66. It seems surprising that such low-copy-number sequences are stably retained during strain propagation and storage. In the case of *S. lividans*, the strains still sporulated, albeit with the production of fewer spores, and it was shown that the “deleted” sequences were present at a much higher copy number in DNA preparations from spores than in mycelial DNA preparations. It was therefore suggested that the terminal sequences might be segregated preferentially into aerial mycelia and spores (55).

Class III mutants show increased oxytetracycline resistance and production and form a phenotypically homogeneous class. They were analyzed by PFGE, and most showed identical restriction patterns. A 200-kb DNA fragment including the *otc* cluster was amplified in three or four tandem copies, and the distal chromosomal sequences had been lost. This structure resembles those seen in other species, such as *S. lividans* 66 (192), with reiteration of a DNA fragment and loss of the distal sequences.

However, in most other cases, the size of the reiterated sequence is much smaller (e.g., 5.7 kb in *S. lividans*) and the degree of amplification is much higher (e.g., >100 copies). The increased copy number of the *otc* cluster accounts for the increased resistance and production. The chromosomal sequences distal to the amplified sequence have been lost, and the structure of the chromosome end in class III mutants is not clear. It is possible that dynamic amplification compensates for the loss of sequences upon chromosomal replication. In one case (mutant MV25), chromosomal amplification was accompanied by recombination with the linear plasmid pPZG101 such that a plasmid end had replaced the chromosome end distal to the reiterated *otc* region. In MV25, the number of reiterated copies of the *otc* region differed between members of the population, and there were also members of the population that had lost all copies of the OTC region, resulting in a sensitive nonproducing phenotype. As mentioned above, there was another class III mutant (MV17), which carried an extra copy of the *otc* cluster on the linear plasmid. The use of class III strains to achieve increased production might seem attractive, but they are subject to an increased frequency of genetic instability to produce class I variants, which eliminates the advantages of increased production.

Many streptomycetes that are currently used for industrial-scale antibiotic production have probably acquired amplified DNA sequences in their genomes as a result of repeated cycles of strain selection. In many instances, fortuitously amplified sequences will be present, but in some cases the DNA amplification may play a direct role in hyperproduction of a particular product. The role of amplification in hyperproduction of an alpha-amylase inhibitor by a strain of *Streptomyces tendae*

has been well established, and it may also account for oleandomycin hyperproduction by a strain of *Streptomyces antibioticus* (see reference 238 and references therein). However, genetic instability can also be a serious problem in industrial production processes. It is well known that industrial strains are susceptible to degeneration (i.e., an irreversible loss of favored characteristics), which is undoubtedly—at least in part—a phenotypic consequence of genetic instability. For example, among others, *S. aureofaciens* (produces CTC) and *S. rimosus* (produces OTC) are commercially important species that are subject to genetic instability (95, 168). It is not surprising, therefore, that the molecular mechanisms responsible for such genetic instabilities have attracted considerable attention.

In their studies, Danilenko and collaborators (50, 66, 188) have shown that a kanamycin resistance (*Km<sup>r</sup>*) gene of *S. rimosus* ATCC 10970 is located on a 15.6-kb amplified unit of DNA (AUD). In *S. rimosus* ATCC 10970 variants (e.g., P3), this AUD could undergo up to 300-fold amplification after serial subculturing on media containing increasing concentrations of the antibiotic, resulting in increased activity of the encoded aminoglycoside phosphotransferase enzyme. The *Km<sup>r</sup>* gene was cloned as a 9.5-kb PstI fragment from a 15.6-kb AUD into *S. lividans*, using the plasmid vector SLP1.2. The recombinant plasmid also underwent amplification in response to the same selection pressure in *S. lividans*. However, it was not completely clear whether the plasmids studied integrated into the chromosome of *S. lividans*, because the SLP1.2 vector on which they were based apparently had lost the sequences that normally promote integration in this organism (169). The *Km<sup>r</sup>* fragment from *S. rimosus* P3 might conceivably promote chromosomal integration of the plasmids, or alternatively, they may simply exist as very large plasmid multimers that could have escaped detection because of their sensitivity to shearing during DNA preparation procedures. Regarding the mechanism, this approach could prove useful for constructing stable highly expressing clones in which foreign DNA fragments could be cloned and coamplified together with plasmids derived in such a way (49, 211). The activity of the *Km<sup>r</sup>* enzyme is controlled by at least two different serine/threonine protein kinases (66), but it is unknown whether changes in the expression of protein kinases play a role in genetic instability.

### tRNA, rRNA, AND GENE EXPRESSION IN *S. RIMOSUS*

The first report on the initiator tRNA genes from *S. rimosus* appeared in 1987 (74), describing two genes that encode tRNA<sup>fMet</sup>, with each differing from the previously published *S. griseus* tRNA<sup>fMet</sup> gene (128) by one nucleotide. One of these had an invalid G<sub>53</sub>-T<sub>61</sub> base pairing, which raises the question of whether it could in fact be a pseudogene. The other gene possessed an unusual G<sub>30</sub>-T<sub>40</sub> base pairing. The presence of this C-U mismatch in the first base pair of the aminoacyl stem and the absence of a CCA end were found in mature *S. rimosus* initiator tRNA, indicating that this pairing might be a general characteristic of *Streptomyces*. This idea was further supported by Plohl and Gamulin (185), who reported the cloning of an *S. rimosus* DNA fragment containing five tRNA genes lacking CCA termini. Two tRNA<sup>Gln</sup> (CUG) genes differing by 1 bp in the

A

(I)

(II)

(III)
500 kb

B

AseI

XbaI

R6
MV15

795 kb
OTC cluster

New band
340 kb

New band
610 kb

415 kb
OTC cluster

R6
MV15

New band
930 kb

FIG. 8. Model to explain the structure of inverted chromosomes in deletion mutants of *Streptomyces rimosus*. (A) Two copies of the chromosome undergo recombination in inverse orientation (I), which generates a double chromosome with two origins of replication (II) and a linear molecule that carries two copies of the OTC gene cluster but no known origin of replication (III). (B) PFGE of AseI and XbaI digests of *S. rimosus* strains. Lanes 1, parental strain *S. rimosus* R6-500; lanes 2, MV15. (Modified from reference 55 with permission of the publisher.)

aminoacyl stem and three identical tRNA<sup>Glu</sup> genes were arranged in the order Gln1-Glu1-Glu2-Gln2-Glu3 and separated by short, nonhomologous intergenic regions. Interestingly, only one of the *S.* *rimonosus* tRNA<sup>fMet</sup> genes carried a potential promoter sequence (TTGCGC-18 bp-TAGACT) at the 5′-flanking region (185). Southern hybridization analyses of *S.* *rimonosus* genomic DNA with <sup>32</sup>P-labeled total tRNAs or DNA fragments containing Gln-Glu have shown that the cluster organization of *S.* *rimonosus* tRNA genes is not typical for gram-positive bacteria (185), with the cluster of five Glu-Gln genes in *S.* *rimonosus* representing one of the largest clusters of tRNA genes. Analysis of the association between tRNA genes and rRNA genes in *Streptomyces* revealed a striking paradigm, with no tRNAs so far found to be associated with any rRNA operons. Sedlmeier and collaborators (204) proposed that the lack of tRNA genes with rRNA genes might be characteristic for all actinomycetes. Based on the first reports that initiator tRNAs from *Streptomyces griseus* and *S.* *rimonosus* lack CCA ends (74, 128) and on later publications indicating that potential tRNA<sup>Pro</sup>, tRNA<sup>Thr</sup>, and tRNA<sup>Tyr</sup> genes associated with site-specific recombination in *Streptomyces* also lack CCA termini (132, 143, 197), Plohl and Gamulin (185) predicted that most, if not all, tRNA genes in *S.* *rimonosus* do not encode a CCA terminus. At present, this prediction is still valid and in agreement with current analyses of the tRNA genes based on genome sequencing data.

A study of rRNA operons in *S.* *rimonosus* was initiated by Plohl and Gamulin (186). Six rRNA operons were defined by Southern blot hybridization, with the rRNA genes in the operons separated by short intergenic regions and organized in the order 16S-23S-5S. Again, the tRNA genes were not found in association with any rRNA operons. The sequence of the 3′ end of the rRNA operon (*rrnF*) with the whole 5S rRNA gene was characterized (186). The gene is 120 bp long and differs at two positions from the *S.* *ambofaciens* 5S rRNA gene (176). The 3′ end (~450 bp) of the 23S rRNA gene and a spacer between the 23S and 5S rRNA genes are located upstream of the 5S rRNA gene on the same cloned fragment. This DNA fragment also shows high homology with the *S.* *ambofaciens* 23S rRNA gene. Remarkable sequence homology was also found with the *S.* *ambofaciens* *rrnD* operon in the 3′-noncoding regions, including the first termination signal. The *S.* *rimonosus* *rrnF* operon contains a second putative terminator which is absent from the *S.* *ambofaciens* *rrnD* operon. The G+C content was 60%, which is much lower than the average, and the 23S-5S rRNA gene spacer was only 48.6% G+C. The whole *S.* *rimonosus* operon was then sequenced and analyzed by Pujić and collaborators (189). The predicted order of the genes is 16S-23S-5S, and the genes are 1,529, 3,121, and 120 nucleotides long, respectively. Homology between rRNA genes from other streptomycetes is in the range of 95% for all three genes and at least 70% for known intergenic regions of *Streptomyces* rRNA operons.

Evidence for the distribution of the rRNA operons and one tRNA gene was obtained following physical mapping of these genes. In *S.* *rimonosus*, the *rrn* operons are located in the central region of the chromosome, with no operon located within 2,755 kb and 3,095 kb of the respective ends of the arms (172). Conversely, in *S.* *coelicolor* A3(2), ribosomal operons *rrnC* and *rrnE* are located about 1.4 Mb and 2.1 Mb from the chromosome ends (196). Physical mapping showed that all rRNA genes could be located on four Asel and DraI fragments (172). In *S.* *rimonosus*, the tRNA<sup>Pro</sup> attachment site for pSAM2 (143) is located 1.8 Mb from the chromosome end, while in *S.* *coelicolor* A3(2) it is near the center of the chromosome (172). In general, comparison of these and other gene markers with the map of *S.* *coelicolor* A3(2) has suggested clear differences in genome organization between the two species (172), a point worthy of further investigation, possibly through whole-genome sequencing efforts with *S.* *rimonosus* in the future.

Regulatory sequences for transcription in *S.* *rimonosus* have been poorly investigated. The first extensive analysis of the DNA fragments associated with apparent *Streptomyces* transcriptional start sites (213) showed a wide range of sequence diversity. Only about 20% appeared to belong to a group similar to that recognized by eubacterial RNA polymerases containing sigma 70-like subunits. These promoters, designated SEPs (*Streptomyces-E. coli*-type promoters), contain –35 and –10 regions with a spacing of 16 to 18 bp between the two sequences (121) and are typically associated with housekeeping genes (44). In *S.* *rimonosus*, there are well-studied promoters for the *otcC* and *otcX* genes associated with OTC biosynthesis (149). Both promoters contain sequences that are similar to the consensus –10 and –35 regions of the major eubacterial and SEP-like promoters.

The expression of seven previously cloned tRNA genes from *S.* *rimonosus* (74, 185) was studied by deletion experiments and Northern blot hybridization. It was demonstrated that the clusters encoding tRNA<sup>Gln</sup> and tRNA<sup>Glu</sup> were transcriptionally active in homologous systems and an *E. coli* heterologous system. All genes in the cluster were cotranscribed as one transcriptional unit from the same promoter, located 140 to 65 bp upstream of the first gene. The sequence TTGGAC-17 bp-TAATGT, resembling an SEP, was also located in this region. Two tRNA<sup>fMet</sup> genes from *S.* *rimonosus*, however, were found to be transcriptionally active only in a homologous system, indicating that they have significantly different promoters which do not function in *E. coli* (65).

One of the six *S.* *rimonosus* ribosomal operons, *rrnF*, has been completely sequenced and analyzed in detail (189). Only one putative promoter for this operon (P4) was identified by sequence similarity in a DNA region upstream from the start of the 16S rRNA gene. The same –10 box (TAGAGT) was found in numerous promoters of rRNA operons in different *Streptomyces* spp. (127, 176, 224, 237), while the –35 regions varied considerably. A 21-bp sequence defined as an rRNA processing site was found downstream from the P4 promoter of *S.* *rimonosus* *rrnF*. This sequence is also found in front of all *Streptomyces* rRNA operons studied to date (176, 231). Two more DNA fragments corresponding to upstream regions of two *S.* *rimonosus* 16S rRNA operons were also cloned and analyzed (189). In the first sequence (type I), four putative promoters (P1 to P4) were identified. The second DNA fragment (type II) seemed to be a deletion derivative of the type I fragment and lacked the DNA region carrying the P2 and P3 promoters. Aside from the internal deletion, the upstream regions of the fragments are highly homologous to the upstream region of the previously described *rrnF*.

operon, starting from the P4 promoter. The putative $-10$ boxes of the P1 to P4 promoters of S. rimosus rRNA operons were compared with all experimentally identified or proposed promoters of Streptomyces rRNA operons. The results indicated that these sequences are conserved among all the species examined to date. In the case of the P2 and P4 promoters, sequence identity extended for several nucleotides on both sides of the $-10$ box. Based on the results of hybridization experiments (189), a general scheme was proposed for the transcription of all six rRNA operons in S. rimosus, with three rRNA operons under the control of four promoters (P1 to P4), two operons expressed from two promoters (P1 and P4), and only the $rrnF$ operon expressed from one promoter, P4, which is the most proximal promoter common to all six S. rimosus rRNA operons.

Two transcription start sites were identified during study of the transcription of the S. rimosus $recA$ gene by primer extension analysis (2). The longer of the two transcripts is initiated from a distal SEP-like promoter (TTGACA-18 bp-TCTTAT) that contains a Cheo box-like sequence (GAAC-$N_4$-ATTC). The major transcript is initiated at position $-36$ and increases significantly in response to DNA damage. Ahel and collaborators (2) proposed a $-35$ box (TTGTCA) and $-10$ box (TAGCGT) that are only 11 bp apart. They noticed that regulatory sequences of this putative promoter and a part of the spacer region are almost identical to those of the DNA damage-inducible promoter of the Mycobacterium tuberculosis $recA$ gene (162), which has been well characterized (76). This sequence is perfectly conserved in all four $recA$ genes analyzed from Streptomyces to date (4, 153, 166, 236). A recent analysis of S. rimosus showed a significant increase in transcription from the proximal promoter of the $recA$ gene in the presence of a RecA protein that lacks 21 amino acids from the C terminus. This up-regulation, albeit less dramatic, was also observed in S. lividans, suggesting the presence of a similar regulation mechanism for the expression of $recA$ in other streptomycetes (3). Inspection of the S. coelicolor genome sequence identified the presence of this type of promoter in the upstream regions of many potentially UV-inducible genes and some other genes/open reading frames. A consensus sequence for this proposed Streptomyces promoter is TTGTCAGTGCC-$N_6$-TAGGGT.

## SYSTEMS FOR PROMOTING GENE EXCHANGE AND CHROMOSOMAL LINKAGE MAPPING

Systems for the transfer of genetic material are a prerequisite for any form of in vivo or in vitro genetic recombination. These systems allow genetic analysis, such as determinations of the locations of structural genes or regulatory regions on a chromosome, extrachromosomal genetic elements, or a fragment of chromosomal or extrachromosomal DNA. Such experimental approaches can then be used for genetic or physical mapping of mutant sites within genes or for carrying out dominance or complementation tests, which are powerful indicators of the nature of genetic controls. Moreover, these systems can also be used in strain-breeding programs, followed by screening and/or selection. Systems promoting gene exchange in Streptomyces species have been well reviewed in recent years (97).

### Systems Promoting Gene Exchange

The natural transfer of genetic material by conjugation in Streptomyces was first demonstrated independently by Sermoni and Spada-Sermoni (207) and by Hopwood (87), with two different strains of S. coelicolor. Chromosome mobilization during conjugation has been demonstrated for more than 20 Streptomyces species (199). In S. coelicolor A3(2), nearly all conjugation can be ascribed to the linear plasmid SCP1 and the circular plasmid SCP2. More than 10 plasmids have now been shown to promote chromosomal recombination in various species, with the majority of these existing as covalently closed circular DNA (101). However, only in S. lividans 66, S. coelicolor A3(2), Saccharopolyspora erythraea (57), and S. rimosus M4018 could the recombination observed upon mating be attributed to a particular fertility plasmid present in one or both parents. A self-transmissible plasmid (SRP1) was shown to determine most of the genetic recombination which occurs between marked derivatives of the Pfizer strain (72). In contrast to the SCP1 and SCP2 plasmids of S. coelicolor, which have very high transfer rates in conjugation experiments (see reference 101 and references therein), SRP1 has a very low transfer rate due to a second plasmid, SRP2. No DNA species corresponding to SRP1 have been detected in S. rimosus strains. An SRP2′ plasmid carrying a chromosomal OTC resistance gene was reported for the Pfizer strain (201). Analysis of this strain showed that SRP2′ was derived from a 310-kb linear plasmid, which presumably corresponds to SRP2 (81).

The crossing techniques used to establish conjugation in S. rimosus were adapted from those developed for S. coelicolor A3(2) and were described in detail by Hopwood (92) and Sermoni (205). These methods use spore suspensions, but the Zagreb strain (S. rimosus R6) does not sporulate well. This led to the use of modified methods based on mycelia, including a mixed-culture method (6, 10, 13, 16, 71, 180, 200) and matings on cellophane membranes (6, 10).

Based on protoplast fusion techniques developed for S. coelicolor (100), chromosomal linkage mapping for S. rimosus has also been possible. The analysis of linkage after protoplast fusion is essentially identical to that of linkage after conjugation, but treatment of S. rimosus protoplasts with 40% (wt/vol) polyethylene glycol 1550 for 30 min increases the frequency of recombination 2 to 3 orders of magnitude compared to that for conjugation, and therefore recombination is not so dependent on the plasmid compositions of the two parent strains (108).

### Chromosomal Linkage Mapping

The development of genetic analysis of the model actinomycete S. coelicolor A3(2) has been well recorded in a number of papers (88–93, 99, 206) and has now been applied to many Streptomyces strains of industrial interest. The analysis of genetic crosses in Streptomyces species differs fundamentally from that of the Hfr system in E. coli. In E. coli, there is usually transfer only of short segments of the chromosome, with a high recombination rate. There is therefore no detectable linkage between most gene pairs, and methods such as measuring the time of entry of markers in interrupted matings must be used. In contrast, Streptomyces strains transfer large DNA segments and have a low recombination rate, allowing detection of link-

ages between markers that are half a chromosome apart. Genetic maps of Streptomyces chromosomes are circular, based upon the generation of recombinants arising from an even number of crossover events. Usually, recombinants are selected with two of the markers in the cross and are tested for the other markers. In order to locate unknown markers in the map, the number of crossovers needed to generate each recombinant class observed can be calculated under the assumption that the unknown markers are located in each of the intervals between known markers. It was found that the correct interval requires the assumption of a small number of higher-order crossover events (fourfold, sixfold, etc.), whereas other intervals would require many more such events. This process is tedious and error prone, so Alačević and collaborators (9) introduced a computer-assisted approach to the processing of data. The mapping method gives information about gene order in the chromosome but does not give genetic distances.

These analysis methods were used to construct a genetic map of Pfizer strain M4018 for 24 genetic markers (71). When crosses were carried out, there was frequent recovery of apparent recombinants that segregated parental genotypes; these were deduced to be heterokaryons. This phenomenon was not observed with *S. coelicolor* A3(2).

A genetic linkage map was also constructed for *S. rimosus* ATCC 10970 (strain R7) (6, 10). For this strain, many of the recombinants were heteroclones. Heteroclones, which are also observed in *S. coelicolor* A3(2), are relatively stable partial diploids that eventually segregate haploid progeny. The frequency of inclusion of any locus in the diploid region depends on its location relative to the selected markers (102), which can be used for construction of a map. Analysis of the frequencies of different classes of haploid segregants from heteroclones allows the estimation of linkage distances between genes. Similar to the case for eukaryotic genetic maps based on meiosis, the unit of distance is the centimorgan (cM), i.e., the percentage of progeny that have undergone a crossover in the region between two genes. This analysis generated a circular map (Fig. 9) with a length of 130 to 150 cM for *S. rimosus* R7. The map substantially agrees with that of *S. rimosus* M4018. However, without a biochemical characterization of most markers used, it could not be assumed that the same genes were always being compared in the two strains.

Protoplast fusion was also used for mapping of *S. rimosus* R7 (108). A map was constructed for six loci by minimizing the multiple crossover classes. This map was in perfect agreement with the results from conjugation experiments.

The chromosomal linkage maps for *S. rimosus*, like those for other Streptomyces species, are indisputably circular, whereas the chromosomes are clearly linear DNA molecules. The linkage across the chromosome ends would be explained most easily by association of the terminal proteins, but there is no experimental evidence available. In crosses between the two closely related strains *S. coelicolor* A3(2) and *S. lividans*, it was shown that nearly all recombinants carried two chromosome ends derived from one parent (230).

Mutants blocked in OTC production were isolated from *S. rimosus* LS-T 118 (32), *S. rimosus* R6 (180), and *S. rimosus* M4018 (200). The mutants were placed into cosynthesis groups. The seven mutants of *S. rimosus* R6, which all belonged to different cosynthesis groups, had mutations that

FIG. 9. Comparison between genetic linkage maps of Streptomyces *rimosus* and other Streptomyces species. Combined genetic maps of *S. rimosus* R6 and R7 (outer circle) and *S. coelicolor* A3(2) (middle circle) and part of the map of *S. bikiniensis* subsp. *zorbonensis* (inner circle) are shown. (Modified from reference 180 with permission of the publisher.)

mapped to one chromosomal region in the chromosomal genetic map (Fig. 9), whereas the other two groups had mutants blocked in two different chromosomal regions. A single 34-kb DNA segment allowed heterologous expression of OTC biosynthesis in *S. lividans* (29), showing the presence of a single OTC cluster. Cross-feeding experiments (200) showed that some of the mutants in the second region (*csf* mutants) were defective in the synthesis of an essential flavin cofactor, cosynthesis factor 1 (146, 147, 154).

*S. rimosus* as a Host-Vector System

Introduction of DNA into *S. rimosus*. Protoplast transformation using polyethylene glycol was the first method described for introducing DNA into Streptomyces strains (28) and is still the most widely used method. The optimal conditions are strain specific, and transformation systems have been developed for the Zagreb and Pfizer *S. rimosus* strains (108, 114, 183). A comparative study of protoplast preparation and regeneration in *S. rimosus* (Fig. 10) and *S. lividans* (105) examined the morphology of protoplasts and showed that with the same amount of mycelium, one can prepare 2 to 3 times more protoplasts from *S. rimosus* than from *S. lividans*, and that the efficiency of regeneration of *S. rimosus* protoplasts is some 10 to 20 times greater than that obtained with *S. lividans*.

The Pfizer strain showed a restriction barrier (114) against DNA derived from *E. coli* that caused a transformation fre-

FIG. 10. Electron photomicrographs of ultrathin sections of *Streptomyces rimosus* R7 hyphae (A) and protoplasts (B). CM, cytoplasmic membrane; CW, cell wall; NE, nuclear equivalent. Bars, 500 nm. (Reprinted from reference 183.)

quency \(10^3\) to \(10^4\) lower than that for DNA derived from *S. rimosus*. A restriction-deficient mutant was isolated (115) which allowed efficient transformation. It was subsequently shown that the Pfizer strain possesses a methyl-dependent restriction system (Hunter, unpublished data), so efficient transformation can be achieved by using DNA from an *E. coli dcm* mutant (138). *S. rimosus* R6 did not show any restriction against DNA derived from *E. coli*.

An electroporation procedure was developed for *S. rimosus* mycelium (181) which gave efficiencies similar to those observed for protoplast transformation \(\left(10^{6}\right.\) transformants/µg plasmid DNA). Although efficient electroporation has been reported for some other *Streptomyces* species (144), the method does not give good results for many strains; it is likely

that the dispersed growth of *S. rimosus* strains makes them particularly amenable to this convenient method of introducing DNA, which avoids the complications of protoplast formation and regeneration.

A system was developed for conjugational transfer from *E. coli* to *Streptomyces* species (142), and efficient transfer to *Streptomyces* species with methyl-dependent restriction systems was also reported (70). This method has been adapted successfully for *S. rimosus*. Because of poor sporulation, it was necessary to preincubate mycelia at \(30^\circ \mathrm{C}\) for 24 h in order to achieve a high efficiency of conjugation (Thamchaipenet, unpublished data).

Vectors for *S. rimosus*. *S. rimosus* strains can be transformed using broad-host-range plasmid vectors based on pIJ101 (126)

and SCP2* (136). The plasmid vector pIJ303, a thiostrepton- resistant derivative of pIJ101, transformed S. *rimonosus* R7. Plasmid pIJ101 is a conjugative plasmid, and therefore transformants have a pock phenotype akin to lethal zygosis (126) due to growth retardation as the plasmid is transferred to the plasmid-free cells in the untransformed lawn. However, no pock phenotype was observed with such pIJ303 transformants, but isolation of the plasmid from these recombinants and retransformation into *S. lividans* showed that the plasmid was still capable of conferring this phenotype. When pIJ303 was introduced into the Pfizer M4018 lineage, only deletion derivatives of the plasmid were obtained. These were linearly permuted, but all had lost the BglIII site of pIJ303. Three different deletion versions were analyzed in more detail. The plasmid vector pPFZ12 (37) was selected as the progenitor vector for plasmid development at Pfizer, based on its relatively small size (6.2 kb) and the fact that it was nonconjugative, increasing the safety factor of recombinants that might be grown at fermentation plant scale.

The bifunctional cosmid pPZ74 contains a pIJ101-derived replication origin for *Streptomyces* and a colE1-derived origin for *E. coli* as well as cos sites from phage λ for in vitro packaging (40). Cosmid pPZ74 confers resistance to tetracycline and chloramphenicol in *E. coli* and resistance to thiostrepton in *Streptomyces*. The cosmid was used to construct a genomic bank of the Pfizer strain. Colony hybridization of a primary bank of *E. coli* with two distinct DNA probes showed positive hybridization at a frequency consistent with a well-represented gene library. The clones showed the same colony morphology as the parental strains when retransformed into *Streptomyces*. This bifunctional cosmid allows efficient cloning of streptomycete genomes in *E. coli* and gives the ability to perform colony hybridization, rapid chromosome walking, and transformation of selected clones directly back into *Streptomyces* (40). It was used (under the slightly different name of pPFZ74) for the isolation of *S. rimosus* DNA sequences flanking OTC resistance determinants (37).

The *E. coli*-Streptomyces shuttle vector pZG1 was constructed from the pIJ101 derivative pIJ350 and from the colE1 derivative pBR322 (75, 182). However, this vector was rather unstable in *S. rimosus*, giving rise to many deletion derivatives. The vector is stable in *E. coli*. Plasmid pIJ101 derivatives replicate by a rolling circle mechanism, and some of them, including pZG1, lack the second-strand synthesis site (*sti*) (56) and thus accumulate single-stranded DNA, which might be expected to promote structural instability (184). However, two further shuttle vectors that did contain a *sti* site were also unstable in *S. rimosus*, so the reasons for the instability remain unknown (179).

Phage vectors have not been used extensively in *S. rimosus*. The most developed phage vectors for *Streptomyces* are based on the broad-host-range phage ΦC31 (43). When such a vector was tested on *S. rimosus*, it did not give rise to visible plaques (D. Hranueli, unpublished data). Later work showed that by increasing the concentration of Ca²⁺ ions in the medium, it was possible to generate plaques (Hunter, unpublished data). A transfection procedure based on protoplast transformation was developed for *S. rimosus* (208), and successful lysogenization was achieved with the phages R4, SH10, and RP2. Phages R4 and SH10 are broad-host-range phages that have been used

for vector construction. Phage RP2 is a narrow-host-range phage isolated from *S. rimosus*. It was shown that deletions could be isolated in RP2, thus opening the way for future vector construction (Hranueli, unpublished data).

Although not much cloning work has been carried out with *S. rimosus*, it is clear that many standard *Streptomyces* vectors will function in this species. It is possible to achieve high transformation frequencies both by using protoplasts and, unusually for *Streptomyces* strains, by the more convenient method of electroporation. The dispersed growth and good fermentation properties of *S. rimosus* strains make these bacteria attractive as cloning hosts for the expression of heterologous genes.

# MUTATION AND SELECTION

Mutation and selection are essential parts of strain development to improve product yields by more efficient conversion of substrates during microbial fermentations. While much has been written about *S. rimosus* genetics over the past 50 years, a significant proportion of the most important research in mutation and selection remains the intellectual property of the various industrial concerns.

## Random Mutagenesis

Empirical application of random mutagenesis has been employed for mutant isolation from the very beginning of genetic studies on *S. rimosus* (14, 15, 156). More-detailed studies determining the optimal conditions required for the use of MNNG (referred to as NTG in some of the cited references) and UV irradiation were published by the early 1970s (52, 53). Delić and coauthors (53) showed that similar conditions, which were defined as optimal for mutagenesis in *S. coelicolor* A3(2), were also found to be effective for auxotroph isolation from the OTC-producing *S. rimosus* strain R7 (ATCC 10970). Incubating spores of this strain for 1 h in the presence of 3 mg/ml MNNG at pH 9 recovered >22% of auxotrophic mutants among survivors. While UV irradiation is a fairly potent mutagen for many microorganisms, it is only poorly mutagenic for *S. rimosus*. When spores of *S. rimosus* strain R7 were exposed to 300 J/m² (defined as optimal conditions for UV mutagenesis in *S. rimosus*), only 1.24% of the survivors were auxotrophs, even if photoreactivation, which occurs in this strain, was prevented by performing the mutagenic treatment and subsequent incubation of mutagenized cells in the dark (52). Comparative studies of the maximum frequencies of mutations induced by various mutagens in the tylosin producer *Streptomyces fradiae* helped to confirm these early findings (19, 20). These procedures were later used successfully in genetic studies of *S. rimosus* for the isolation of auxotrophs (39, 71) and non-OTC-producing mutants (180, 200) as well as mutants of other types (107, 225).

## Localized Mutagenesis

Several possibilities for localized mutagenesis in *Streptomyces* have been used or suggested. These include (i) mutagenic treatment of synchronized cultures with MNNG, (ii) comutation with MNNG, (iii) treatment of temperate transducing

phages with hydroxylamine, (iv) mutagenesis of protoplasts followed by fusion to untreated protoplasts carrying a counterselectable marker, (v) in vitro mutagenesis of naked chromosomal DNA followed by liposome-mediated transformation of chromosomal DNA into protoplasts, (vi) mutagenesis by insertion of transposons or insertion sequences into target genes, (vii) mutational cloning, and (viii) site-specific in vitro mutagenesis of cloned fragments of DNA followed by reintro- duction of the DNA into protoplasts and recombination with homologous resident sequences (see reference 98 and references therein). The only procedure for localized mutagenesis applied to *S. rimosus* has been comutation with MNNG. Randazzo and coauthors (191) used selection of revertants of a gene for histidinol dehydrogenase (*hisA* gene) originating from the *S. coelicolor* A3(2) *his* operon after treatment with MNNG. This produced populations containing >6% mutants with mutations within the operon and about 4% mutants with mutations in the adjacent loci. A very similar pattern of mutation was observed when *S. rimosus* R7 was used (39). By reverting the mutation in the *hisA2* gene with MNNG, using a selective medium designed for *S. coelicolor*, the occurrence of comutations in the *his* cluster around *hisA*, as well as comutations with *hisA* of the *amm*, *met*, *arg*, and *trp* genes, was observed. Therefore, the comutation procedure proved to be a promising shortcut for the isolation and preliminary localization of new genes with respect to reference markers as well as for checking the positions of genes that were already located. Furthermore, at that time, it was the most effective procedure for determining the degree of homology between similar species. Comutants of *S. rimosus* R6 (the Zagreb strain) that appeared among revertants of the *pdx-10* locus (*phe* and *ilv*) (8) were of special interest. Firstly, genes in these classes had not been isolated previously from *S. rimosus*. Secondly, they are located in a poorly mapped region, and most importantly, they may be situated near the *otcA*, -B, -C, and -D loci, opening the possibility of isolating comutants with mutations in the *otc* region (180). Although the comutation phenomenon has been used for the isolation of genes involved in antibiotic production in *Streptomyces bikiniensis* (85), further applications of this procedure for localized mutagenesis in *S. rimosus* genetic studies or strain development programs have not been published.

Mutations affecting regulatory antibiotic genes. The concept that control mechanisms can be bypassed through mutations can be exploited in strain development programs (140). Martin and collaborators (141) devised a method for visualizing antibiotic production on solid media in the presence of phosphate. Mutants of *S. griseus* altered in phosphate regulation were isolated so that phosphate no longer inhibited the biosynthesis of candicidin. Some of these mutants produced more antibiotic than the wild type in both phosphate-supplemented and unsupplemented production media. These mutants were not altered in the permeation of phosphate but rather appeared to be deregulated in phosphate control of candicidin synthesis. Phosphate-deregulated mutants may have considerable industrial significance, permitting antibiotic fermentations to be performed in complex media with high phosphate contents (141). Biosynthesis of OTC is regulated by the concentration of phosphate in the growth medium (22), and industrial production is carried out at growth-limiting concentrations of inorganic phosphate (140). Apart from a very early report (16) of random mutagenesis by UV irradiation leading to the isolation of *S. rimosus* LS-T293 mutants capable of tolerating high concentrations of inorganic phosphate, there are no published data employing this or any other method for the isolation, characterization, and commercial application of phosphate-deregulated mutants in tetracycline-producing microorganisms, including *S. rimosus*.

Other mutations affecting antibiotic yield. Apart from mutations in regulatory genes that could affect antibiotic production in a direct way, mutations that influence production indirectly can also be uncovered. However, the isolation of mutants with the ability to use cheaper raw materials with superior biotechnological properties, such as a lower level of foaming, improved filtration, or the ability to produce higher yields under particular fermentation conditions of temperature or oxygen tension, has not been widely described (although undoubtedly such mutants have been isolated and used in industry). The only published example is the construction of the *S. rimosus* LS-T hybrid (type I) by genetic recombination (158). The Russian group succeeded, after screening prototrophic recombinants of a genetic cross, to recognize a recombinant (LS-T hybrid) that displayed no foaming of the fermentation medium. This strain also produced an elevated level of OTC, but it was not resolved whether the increase in antibiotic level was due to the reduced foaming or caused by a separate recombination event.

Strains having other indirect superior characteristics, e.g., resistance to various actinophages, have also been isolated (106, 107, 225). For more than 50 years, it has been known that actinophages can attack streptomycetes, but these viruses did not attract much attention until their undesirable interference with industrial fermentations was recognized. Efforts have been made to select antibiotic-producing mutants which are resistant to infection. Mutants possessing a partial degree of resistance to actinophages often appear. Such strains, when plated with a given phage preparation, give rise to much smaller numbers of plaques than the original, sensitive organism. These strains have been designated semiresistant (233). Vešligaj and collaborators (225) described mutants of *S. rimosus* R6 that were semiresistant to a virulent actinophage. To isolate *S. rimosus* mutants capable of overcoming actinophage infections, the virulent actinophage RP1 (Fig. 11) was obtained and partially characterized.

Actinophage RP1 has a very narrow host range restricted to *S. rimosus* strains. One-step growth experiments with RP1 showed a 2-hour minimum latent period and a 1.5-hour rise period. The average burst size was about 50 actinophage particles. Divalent cations are necessary for attachment, and the actinophage forms large, regular, clear plaques. RP1-resistant mutants were isolated, but it appeared that enrichment by selective pressure was not very efficient in obtaining fully resistant mutants. After a few subcultures in RP1-containing liquid medium, the population predominantly consisted of phenotypically resistant cells, but the majority could be made to revert to the sensitive state by restreaking of mycelial fragments or by growing the cells in the presence of RP1-specific antiserum. Therefore, a pseudolysogenic condition was established. However, 0.1% of potentially resistant isolates, when tested in RP1-containing liquid culture, exhibited the following characteristics: (i) there was no loss of viable cells upon infection.

FIG. 11. (A) Morphology of RP1 plaques on (a) *Streptomyces rimosus* R6 after overnight incubation and (b) *Streptomyces rimosus* R6Φ'10 after incubation for 4 days. Magnification, ×0.5. (B) Electron micrograph of actinophage RP1 negatively stained with potassium phosphotungstate. Bar, 100 nm. (Reprinted from reference 225.)

tion with RP1, (ii) resistant cells supported almost no growth of the actinophage, and (iii) the relative OTC titer remained the same, irrespective of the presence of the actinophage. In semiresistant strains, the propagation of RP1 was characterized by approximately 2.5-fold-longer latent and rise periods, but there was no significant reduction in burst size. The longer generation time for the phage changed the balance between growth of the host and killing by the phage. Semiresistant mutants of *S.* *rimosus* R6 retained the ability to produce the same yield of OTC even in the presence of the actinophage and proved useful in the industrial production of the antibiotic (225). All attempts to isolate fully resistant mutants have been unsuccessful. Inefficient selective pressure and the relatively frequent appearance of semiresistant strains could explain the difficulties in detecting fully resistant mutants. Putative temperate actinophages have also been studied. For example, lysis of *S.* *rimosus* cultures by two lysogenic strains (560 and IK) was observed in several cases under industrial conditions (195).

However, lysis was not induced by a virulent mutant of a temperate actinophage but by some other lytic factor which could be separated by differential centrifugation and seemed to consist of at least two enzymes, a lytic and a proteolytic enzyme (194). Although *S.* *rimosus* R6 is lysogenic for two prophages, RP2 and RP3 (104, 193), this does not seem to cause any problems in antibiotic production.

### Screening and Selection

It is possible to screen for mutants of *S.* *rimosus* affected in OTC production by using an agar-based diffusion bioassay method (178, 180, 200). Although this approach is effective for the isolation of nonproducing mutants, there are problems in isolating highly producing mutants, as there is a poor correlation between the sizes of zones of inhibition on agar and the yields of antibiotic in liquid culture. Dulaney (63) obtained overproducing mutants among apparent revertants of nonpro-

ducing mutants. Mathematical modeling of colony growth and OTC production was used to try to improve the results from the agar diffusion method (34). Although this helped in the design of the agar-based screening, it did not overcome the basic problem of poor correlation with results in liquid culture. For many years, shake flask screens have been the main method employed by the fermentation industry to isolate improved mutants, and they are still in widespread use today. However, detailed descriptions of shake flask screening of *S. rimosus* mutants or recombinants have not been published.

# FUTURE PROSPECTS

*S. rimosus* is among the best-characterized industrial streptomycetes, with the other contender being the streptomycin producer *S. griseus* (e.g., see references 133 and 220). As such, *S. rimosus* will continue to be researched to meet the substantial demand for oxytetracycline to be used as an antibiotic or to supply oxytetracyline base, which is modified chemically into semisynthetic derivative structures. It is one of the few streptomycetes that grows rapidly, in a highly dispersed form, in fermentors, with cell yields over 50 g/liter. A chemically defined growth and production medium simplifies physiological investigation.

The genetics of *S. rimosus* is the best developed of all industrial streptomycetes. Since the 1970s, the understanding of *S. rimosus* genetics has closely paralleled the groundbreaking work undertaken with *S. coelicolor*. To date, the genetic linkage map of *S. rimosus* is second only to that of *S. coelicolor* (9, 10, 71, 200). The completion of the genome sequence for *S. coelicolor* (25) has promoted a wealth of postgenomic tools for *S. coelicolor*, including PCR-targeted mutagenesis (83), whole-genome mutagenesis (30), and transfer of streptomycete-derived, plasmid-encoded genes (or modified genes) directly from *E. coli*, using an interspecific conjugal transfer system based on the *oriT* plasmid transfer system (70). All of these technologies have now been adapted for use with *S. rimosus*. Other postgenomic tools, such as microarrays and protein mapping (both of which exist for *S. coelicolor*) (86, 123) must await the completion of the genome sequence of *S. rimosus*. Recent changes in the methodology of genome sequencing involving nanotechnology (139) bring such projects down to a scale that can be accomplished well within a week, with an associated 100- to 200-fold reduction in the cost of derivation of the primary data. Advances in programming that allow more automated and precise annotation of genome sequences will further reduce the time and cost of obtaining a fully annotated genome sequence for *S. rimosus*. The realistic prospect must be that it will be possible to obtain the genome sequence of *S. rimosus* in the near term.

Three important opportunities will arise from the genomic data. Firstly, *S. rimosus* is taxonomically distant (129) from *S. coelicolor*, *S. avermitilis*, and *Streptomyces scabies*, whose genome sequences are available at present. It also falls in a different cluster group from *Streptomyces venezuelae*, whose genome sequence has been obtained (at the Diversa biotechnology company) but is not yet in the public database. All streptomycetes will have synteny of genome sequences with each other, but the differences will be most enlightening in terms of understanding the diversity within the genus. *S. rimosus* lies within a cluster grouping that has no genome sequencing activity at present, reinforcing the view that this information is critical to a better understanding of the relationships between and across members of the genus. We already know that homologs of some genes, particularly the developmental genes (*bld* and *whi*) identified for *S. coelicolor*, may have subtly different roles in other species. Thus, gene knockouts of homologs in different species have different phenotypes. Pursuit of this important perspective requires both the genome sequence and advanced genetic tools.

Secondly, *S. rimosus* provides an unparalleled resource in terms of the genealogy of improved OTC production strains. Industrial mutation programs have resulted in an increase in production from that of the initial soil isolate (producing some tens of mg/liter of OTC) to that of advanced strains, which produce yields approaching 100 g/liter of the antibiotic. Now that whole-genome sequencing of members of this lineage is realistic, it will be possible to define the changes in genome architecture as well as single-nucleotide polymorphic changes which were responsible for the dramatic increase in production.

Thirdly, the integrated genomic data provide the platform to undertake further manipulation of the *S. rimosus* chromosome, informed by observations from other species which increase antibiotic productivity, in a well-defined way. Previously, such advances had to be identified by random or directed screening (both very labor-intensive), but they will now be possible using defined genetic changes based on site-directed mutagenesis and allelic replacement. In addition, this precise manipulation of antibiotic production genes will provide the opportunity to derive novel bioactive compounds based on the tetracyclic nucleus, which could herald a new generation of tetracycline-based therapeutics.

## ACKNOWLEDGMENTS

We thank Marija Alačević for her role over many years in stimulating the genetics of *S. rimosus*. Apart from her direct scientific contribution, she was responsible for the undergraduate studies of five of the authors and supervised the doctoral theses of two. She organized a number of key scientific meetings and forged international links, without which this review could not have been written.

## REFERENCES

1. Ackermann, H.-W., and M. S. DuBow. 1987. Bacteriophage taxonomy, p. 13–28. *In* Viruses of prokaryotes, vol. 1. General properties of bacteriophages. CRC Press, Boca Raton, Fla.
2. Ahel, I., D. Vujaklija, A. Mikoc, and V. Gamulin. 2002. Transcriptional analysis of the *recA* gene in *Streptomyces rimosus*: identification of the new type of promoter. FEMS Microbiol. Lett. **209**:133–137.
3. Ahel, I., A. Mikoc, and V. Gamulin. 2005. *recA* gene expression in a streptomycete is mediated by the unusual C-terminus of RecA protein. FEMS Microbiol. Lett. **248**:119–124.
4. Aigle, B., A. C. Holl, J. F. Angulo, P. Leblond, and B. Decaris. 1997. Characterization of two *Streptomyces ambofaciens recA* mutants: identification of the RecA protein by immunoblotting. FEMS Microbiol. Lett. **149**:181–187. (Erratum, **155**:131–132.)
5. Alačević, M. 1963. Interspecific recombination in *Streptomyces*. Nature **197**:1323.
6. Alačević, M. 1969. Recombination in *Streptomyces* producing tetracyclines, p. 137–145. *In* G. Sermonti and M. Alačević (ed.), Proceedings of the International Symposium on the Genetics and Breeding of *Streptomyces*, Dubrovnik, Yugoslavia, 1968. Yugoslav Academy of Sciences and Arts, Zagreb, Yugoslavia.
7. Alačević, M. 1973. Genetics of tetracycline-producing streptomycetes, p. 59–70. *In* Z. Vaněk, Z. Hošťálek, and J. Cudlin (ed.), Proceedings of the First International Symposium on the Genetics of Industrial Microorganisms, Prague, Czechoslovakia, 1970. Academia, Prague, Czechoslovakia.

8. Alačević, M. 1976. Recent advances in *Streptomyces rimosus* genetics, p. 513–519. In K. D. MacDonald (ed.), Proceedings of the Second International Symposium on the Genetics of Industrial Microorganisms, Sheffield, United Kingdom, 1974. Academic Press, London, United Kingdom.

9. Alačević, M., J. Pigac, and M. Vešligaj. 1978. Different approaches to the gene mapping in *Streptomyces rimosus*, p. 65–70. In E. Freerksen, I. Tarnok, and J. H. Thumim (ed.), Proceedings of the Fourth International Colloquium on the Genetics of the Actinomycetales, Borstel, F.R.G., 1976. Gustav Fischer Verlag, Stuttgart, Germany.

10. Alačević, M., M. Strašek-Vešligaj, and G. Sermonti. 1973. The circular linkage map of *Streptomyces rimosus*. J. Gen. Microbiol. 77:173–185.

11. Alačević, M., D. Vlašić, and I. Spada-Sermonti. 1966. Interspecific recombination among antibiotic-producing *Streptomycetes*, p. 720–722. In M. Herold and Z. Gabriel (ed.), Proceedings of the Congress on Antibiotics. Antibiotics—advances in research, production and clinical use. Czechoslovak Medical Press, Prague, Czechoslovakia.

12. Alcaide, E., M. D. Blasco, and C. Esteve. 2005. Occurrence of drug-resistant bacteria in two European eel farms. Appl. Environ. Microbiol. 71:3348–3350.

13. Alikhanian, S. I., and S. Z. Mindlin. 1957. Recombinations in *Streptomyces rimosus*. Nature 180:1207–1209.

14. Alikhanian, S. I., S. U. Goldat, and A. F. Teteryatnik. 1957. Mutagenic effect of combined action of ethylenimine and ultraviolet rays on actinomycetes. Dokl. Akad. Nauk SSSR 115:1015–1017. (In Russian.)

15. Alikhanian, S. I., S. Z. Mindlin, S. U. Goldat, and A. V. Vladimirov. 1959. Genetics of organisms producing tetracyclines. Ann. N. Y. Acad. Sci. 81:914–949.

16. Alikhanian, S. I., S. Z. Mindlin, N. V. Orlova, and T. P. Verkhovtzeva. 1959. Hereditary changes of some physiological properties in *Streptomyces rimosus*. Appl. Microbiol. 7:141–144.

17. Alikhanian, S. I., N. V. Orlova, S. Z. Mindlin, and Z. M. Zaytzeva. 1961. Genetic control of oxytetracycline biosynthesis. Nature 189:939–940.

18. Apracio, J. F., P. Caffrey, J. A. Gil, and S. B. Zotchev. 2003. Polyene antibiotic biosynthesis gene clusters. Appl. Microbiol. Biotechnol. 61:179–188.

19. Baltz, R. H. 1986. Mutation in *Streptomyces*, p. 61–94. In S. W. Queener and L. E. Day (ed.), The bacteria, vol. 9. Antibiotic-producing *Streptomyces*. Academic Press, London, United Kingdom.

20. Baltz, R. H. 1986. Mutagenesis in *Streptomyces* spp., p. 184–190. In A. L. Demain and N. A. Solomon (ed.), Manual of industrial microbiology and biotechnology. American Society for Microbiology, Washington, D.C.

21. Bêhal, V. 1987. The tetracycline fermentation and its regulation. Crit. Rev. Biotechnol. 5:275–318.

22. Bêhal, V., J. Prusakova-Gregrova, and Z. Hoštálek. 1982. Effect of inorganic phosphate and bezylthiocyanate on activity of anhydrotetracycline oxygenase in *Streptomyces aureofaciens*. Folia Microbiol. 27:102–106.

23. Bêhal, V., and I. S. Hunter. 1995. Tetracyclines, p. 359–384. In L. C. Vining and C. Stuttard (ed.), Genetics and biochemistry of antibiotic production. Butterworth-Heinemann, Boston, Mass.

24. Benigni, R., P. A. Petrov, and A. Carere. 1975. Estimate of the genome size by renaturation studies in *Streptomyces*. Appl. Microbiol. 30:324–326.

25. Bentley, S. D., S. Brown, L. D. Murphy, D. E. Harris, M. A. Quail, J. Parkhill, B. G. Barrell, J. R. McCormick, R. I. Santamaria, R. Losick, M. Yamasaki, H. Kinashi, C. W. Chen, G. Chandra, D. Jakimowicz, H. M. Kieser, T. Kieser, and K. F. Chater. 2004. SCP1, a 356,023 bp linear plasmid adapted to the ecology and developmental biology of its host, *Streptomyces coelicolor* A3(2). Mol. Microbiol. 51:1615–1628.

26. Bentley, S. D., K. F. Chater, A. M. Cerdeno-Tarraga, G. L. Challis, N. R. Thomson, K. D. James, D. E. Harris, M. A. Quail, H. Kieser, D. Harper, A. Bateman, S. Brown, G. Chandra, C. W. Chen, M. Collins, A. Cronin, A. Fraser, A. Goble, J. Hidalgo, T. Hornsby, S. Howarth, C. H. Huang, T. Kieser, L. Larke, L. Murphy, K. Oliver, S. O’Neil, E. Rabbinowitsch, M. A. Rajandream, K. Rutherford, S. Rutter, K. Seeger, D. Saunders, S. Sharp, R. Squares, S. Squares, K. Taylor, T. Warren, A. Wietzorrek, J. Woodward, B. G. Barrell, J. Parkhill, and D. A. Hopwood. 2002. Complete genome sequence of the model actinomycete *Streptomyces coelicolor* A3(2). Nature 417:141–147.

27. Bérdy, J. 2005. Bioactive microbial metabolites. J. Antibiot. (Tokyo) 58:1–26.

28. Bibb, M. J., J. M. Ward, and D. A. Hopwood. 1978. Transformation of plasmid DNA into *Streptomyces* at high frequency. Nature 274:398–400.

29. Binnie, C., M. Warren, and M. J. Butler. 1989. Cloning and heterologous expression in *Streptomyces lividans* of *Streptomyces rimosus* genes involved in oxytetracycline biosynthesis. J. Bacteriol. 171:887–895.

30. Bishop, A., S. Fielding, P. Dyson, and P. Herron. 2004. Systematic insertional mutagenesis of a streptomycete genome: a link between osmoadaptation and antibiotic production. Genome Res. 14:893–900.

31. Borisoglebskaya, A. N., A. N. Perebityuk, and A. M. Boronin. 1979. Study of resistance of *Actinomyces rimosus* to oxytetracycline. Antibiotiki 24:883–888. (In Russian.)

32. Boronin, A. M., and S. Z. Mindlin. 1970. A study of complementation and cosynthesis in the mutants of *Actinomyces rimosus* with disturbances of the antibiotic production. Genetika 6:72–77. (In Russian.)

33. Boronin, A. M., and S. Z. Mindlin. 1971. Genetical analysis of *Actinomyces rimosus* mutants with impaired synthesis of the antibiotic. Genetika 7:125–131. (In Russian.)

34. Bošnjak, M., J. Pigac, R. Valinger, M. Vampola, and M. Vešligaj. 1990. Kinetics of product formation in microbial colonies, p. 345–358. In A. Blačej and A. Ottová (ed.), Interbiotech ’89—mathematical modeling in biotechnology. Proceedings of the International Symposium on Biotechnology, Bratislava, Czechoslovakia, 1989. Elsevier, Amsterdam, The Netherlands.

35. Butler, M. J., and B. N. Gedge. 1989. Purification of anhydrotetracycline oxygenase from *Streptomyces rimosus* using fast protein liquid chromatography. Biotechnol. Tech. 4:235–238.

36. Butler, M. J., C. Binnie, I. S. Hunter, D. A. Sudgen, and M. Warren. 1990. Genetic manipulation of the oxytetracycline biosynthesis pathway genes. Dev. Ind. Microbiol. 31:41–50.

37. Butler, M. J., E. J. Friend, I. S. Hunter, F. S. Kaczmarek, D. A. Sudgen, and M. Warren. 1989. Molecular cloning of resistance gene and architecture of a linked gene cluster involved in biosynthesis of oxytetracycline by *Streptomyces rimosus*. Mol. Gen. Genet. 215:231–238.

38. Caballero, J. L., E. Martinez, F. Malpartida, and D. A. Hopwood. 1991. Organization and functions of the *actVA4* region of the actinorhodin biosynthesis gene cluster of *Streptomyces coelicolor*. Mol. Gen. Genet. 230:401–412.

39. Carere, A., and R. Randazzo. 1976. Co-mutation in *Streptomyces*, p. 573–581. In K. D. MacDonald (ed.), Proceedings of the Second International Symposium on the Genetics of Industrial Microorganisms, Sheffield, United Kingdom, 1974. Academic Press, London, United Kingdom.

40. Chambers, A. E., and I. S. Hunter. 1984. Construction and use of a bifunctional streptomycete cosmid. Biochem. Soc. Trans. 12:644–645.

41. Chardon-Loriaux, I., M. Charpentier, and F. Percheron. 1986. Isolation and characterization of a linear plasmid from *Streptomyces rimosus*. FEMS Microbiol. Lett. 35:151–155.

42. Chater, K. F. 1980. Actinophage DNA. Dev. Ind. Microbiol. 21:65–74.

43. Chater, K. F. 1986. *Streptomyces* phages and their applications to *Streptomyces* genetics, p. 119–158. In S. W. Queener and L. E. Day (ed.), The bacteria, vol. 9. Antibiotic-producing *Streptomyces*. Academic Press, London, United Kingdom.

44. Chater, K. F., and D. A. Hopwood. 1989. Antibiotic biosynthesis in *Streptomyces*, p. 129–150. In D. A. Hopwood and K. F. Chater (ed.), Genetics of bacterial diversity. Academic Press, London, United Kingdom.

45. Chopra, I., P. M. Hawkey, and M. Hinton. 1992. Tetracyclines, molecular and clinical aspects. J. Antibi. Chemother. 29:245–277.

46. Connell, S. R., D. M. Tracz, K. H. Nierhaus, and D. E. Taylor. 2003. Ribosomal protection proteins and their mechanism of tetracycline resistance. Antimicrob. Agents Chemother. 47:3675–3681.

47. Cope, A. C., E. P. Burrows, M. E. Derieg, S. Moon, and W.-D. Wirth. 1965. Rimocidin. I. Carbon skeleton, partial structure, and absolute configuration at C-27. J. Am. Chem. Soc. 87:5452–5460.

48. Dairi, T., T. Nakano, K. Aisaka, R. Katsumata, and M. Hasegawa. 1995. Cloning and nucleotide sequence of the gene responsible for chlorination of tetracycline. Biosci. Biochem. 59:1099–1106.

49. Danilenko, V. N., L. I. Starodubtseva, and S. M. Navashin. 1985. Cloning and amplification of DNA of actinomycetes: systems and strategy of origination of strains—superproducers of antibiotics. Biotechnologiya 2:62–67. (In Russian.)

50. Danilenko, V. N., Y. A. Potekhin, I. V. Biryukova, and S. M. Navashin. 1984. Cloning of actinomycetes DNA: construction of vector systems. Antibiotiki 8:563–572. (In Russian.)

51. Davisson, J. W., F. W. Tanner, Jr., A. C. Finlay, and I. A. Solomon. 1951. Rimocidin, a new antibiotic. Antibiot. Chemother. 1:289–290.

52. Delić, V. 1970. Ph.D. thesis. University of Zagreb, Zagreb, Yugoslavia.

53. Delić, V., D. A. Hopwood, and E. J. Friend. 1970. Mutagenesis by *N*-methyl-*N*′-nitro-*N*-nitrosoguanidine (NTG) in *Streptomyces coelicolor*. Mutat. Res. 9:167–182.

54. Delić, V., J. Pigac, and G. Sermonti. 1969. Detection and study of cosynthesis of tetracycline antibiotics by an agar method. J. Gen. Microbiol. 55:103–108.

55. Denapaite, D., A. Paravić Radičević, B. Čajavec, I. S. Hunter, D. Hranueli, and J. Cullum. 2005. Persistence of the chromosome end regions at low copy number in mutant strains of *Streptomyces rimosus* and *S. lividans*. Food Technol. Biotechnol. 43:9–17.

56. Deng, Z. X., T. Kieser, and D. A. Hopwood. 1988. “Strong incompatibility” between derivatives of the *Streptomyces* multi-copy plasmid pIJ101. Mol. Gen. Genet. 214:286–294.

57. DeWitt, J. P. 1985. Evidence for a sex factor in *Streptomyces erythreus*. J. Bacteriol. 164:969–971.

58. Dittrich, W., and H. Schrempp. 1992. The unstable tetracycline resistance gene of *Streptomyces lividans* 1326 encodes a putative protein with similarities to translational elongation factors and Tet (M) and Tet (O) proteins. Antimicrob. Agents Chemother. 36:1119–1124.

59. Donadio, S., M. J. Staver, J. B. McAlpine, S. J. Swanson, and L. Katz. 1991. Modular organization of genes required for complex polyketide biosynthesis. Science 252:675–679.

60. Doyle, D., M. J. Butler, and I. S. Hunter. 1988. Molecular analysis of the oxytetracycline resistance gene, otrA, from *Streptomyces rimosus*. Heredity **61**:305.

61. Doyle, D., K. J. McDowall, M. J. Butler, and I. S. Hunter. 1991. Characterization of an oxytetracycline resistance gene, otrA, of *Streptomyces rimosus*: nucleotide sequence, deduced function and promoter analysis. Mol. Microbiol. **5**:2923–2933.

62. Dudnik, Y. V., and L. I. Kozmyan. 1987. Actinomycete protoplast fusion in obtaining antibiotic-producing recombinant strains. Antibiot. Med. Biotechnol. **32**:86–89. (In Russian.)

63. Dulaney, E. L. 1969. Variation in selected populations of tetracycline producers, p. 93–120. In G. Sermonti and M. Alačević (ed.), Proceedings of the International Symposium on the Genetics and Breeding of Streptomycetes, Dubrovnik, Yugoslavia, 1986. Yugoslav Academy of Sciences and Arts, Zagreb, Yugoslavia.

64. Durajlija, S., J. Pigac, and V. Gamulin. 1991. Construction of two stable bifunctional plasmids for *Streptomyces* spp. and *Escherichia coli*. FEMS Microbiol. Lett. **83**:317–321.

65. Durajlija Žinić, S., M. Plohl, and V. Gamulin. 2000. The *in vivo* expression of *Streptomyces rimosus* tRNA genes. Food Technol. Biotechnol. **38**:161–166.

66. Elizarov, S. M., O. V. Sergienko, I. A. Sizova, and V. N. Danilenko. 2005. Dependence of aminoglycoside 3′-phosphotransferase VIII activity on serine/threonine protein kinases in *Streptomyces rimosus*. Mol. Biol. (Moscow) **39**:255–263. (In Russian.)

67. Erokhina, L. I., and S. I. Alikhanian. 1966. Obtaining mutants of *Actinomyces rimosus*, which synthesize substances different from tetracycline, p. 689–701. In M. Herold and Z. Gabriel (ed.), Proceedings of the Congress on Antibiotics. Antibiotics—advances in research, production and clinical use. Czechoslovak Medical Press, Prague, Czechoslovakia.

68. Felmingham, D. 2005. Tigecyclin—the first glycylcycline to undergo clinical development: an overview of *in vitro* activity compared to tetracycline. J. Chemother. **17**(Suppl. 1):5–11.

69. Fischer, G., T. Wenner, B. Decaris, and P. Leblond. 1998. Chromosomal arm replacement generates a high level of intraspecific polymorphism in the terminal inverted repeats of the linear chromosomal DNA of *Streptomyces ambofaciens*. Proc. Natl. Acad. Sci. USA **95**:14296–14301.

70. Flett, F., V. Mersinians, and C. P. Smith. 1997. High efficiency intergeneric conjugal transfer of plasmid DNA from *Escherichia coli* to methyl DNA-restricting streptomycetes. FEMS Microbiol. Lett. **155**:223–229.

71. Friend, E. J., and D. A. Hopwood. 1971. The linkage map of *Streptomyces rimosus*. J. Gen. Microbiol. **68**:187–197.

72. Friend, E. J., M. Warren, and D. A. Hopwood. 1978. Genetic evidence for a plasmid controlling fertility in an industrial strain of *Streptomyces rimosus*. J. Gen. Microbiol. **106**:201–206.

73. Fu, H., S. Ebert-Khosla, D. A. Hopwood, and C. Khosla. 1994. Relaxed specificity of the oxytetracycline polyketide synthase for an acetate primer in the absence of a malonyl primer. J. Am. Chem. Soc. **116**:6443–6444.

74. Gamulin, V., and D. Soll. 1987. The initiator tRNA genes from *Streptomyces rimosus*. Nucleic Acids Res. **15**:6747.

75. Gamulin, V., D. Vujaklija, and J. Pigac. 1986. New bifunctional vector pZGl suitable for gene cloning in the *Escherichia coli-Streptomyces rimosus* system, p. 52–61. In Proceedings of the First Yugoslav Conference on the Genetic and Biochemical Engineering in Biotechnology, Beograd, Yugoslavia, 1984. Narodna Tehnika: The Association of Yugoslav Organizations for Technological Affairs, Beograd, Yugoslavia. (In Croatian.)

76. Gopaul, K. K., P. C. Brooks, J.-F. Prost, and E. O. Davis. 2003. Characterization of the two *Mycobacterium tuberculosis recA* promoters. J. Bacteriol. **185**:6005–6015.

77. Gossen, M., and H. Bujard. 1992. Tight control of gene expression in mammalian cells by tetracycline-responsive promoters. Proc. Natl. Acad. Sci. USA **89**:5547–5551.

78. Goverdhana, S., M. Puntel, W. Xiong, J. M. Zirger, C. Barcia, J. F. Curtin, E. B. Soffer, S. Mondkar, G. D. King, J. Hu, S. A. Sciascia, M. Candolfi, D. S. Greengold, P. R. Lowenstein, and M. G. Castro. 2005. Regulatable gene expression systems for gene therapy applications: progress and future challenges. Mol. Ther. **12**:189–211.

79. Graham, D. Y., A. R. Opekun, G. Belson, H. M. El-Zimaity, and M. R. Carlson. 2005. Novel bismuth-metronidazole-tetracycline triple-layer tablet for treatment of *Helicobacter pylori*. Aliment. Pharmacol. Ther. **15**:165–168.

80. Gravius, B., T. Bezmalinović, D. Hranueli, and J. Cullum. 1993. Genetic instability and strain degeneration in *Streptomyces rimosus*. Appl. Environ. Microbiol. **59**:2220–2228.

81. Gravius, B., D. Glockler, J. Pigac, K. Pandža, D. Hranueli, and J. Cullum. 1994. The 387 kb linear plasmid pPZG101 of *Streptomyces rimosus* and its interactions with the chromosome. Microbiology **140**:2271–2277.

82. Gunther, G., and M. Haglund. 2005. Tick-borne encephalopathies: epidemiology, diagnosis, treatment and prevention. CNS Drugs **19**:1009–1032.

83. Gust, B., G. L. Challis, K. Fowler, T. Kieser, and K. F. Chater. 2003. PCR-targeted *Streptomyces* gene replacement identifies a protein domain needed for biosynthesis of the sesquiterpene soil odor geosmin. Proc. Natl. Acad. Sci. USA **100**:1541–1546.

84. Hacking, A. J. 1986. Economic aspects of biotechnology. Cambridge University Press, Cambridge, United Kingdom.

85. Hanssen, R., and R. Kirby. 1983. The induction by *N*-methyl-*N*′-nitro-*N*-nitrosoguanidine of multiple closely linked mutations in *Streptomyces bikiniensis* ISP5235 affecting streptomycin resistance and streptomycin biosynthesis. FEMS Microbiol. Lett. **17**:317–320.

86. Hesketh, A. R., G. Chandra, A. D. Shaw, J. J. Rowland, D. B. Kell, M. J. Bibb, and K. F. Chater. 2002. Primary and secondary metabolism, and post-translational protein modifications, as portrayed by proteomic analysis of *Streptomyces coelicolor*. Mol. Microbiol. **46**:917–932.

87. Hopwood, D. A. 1957. Genetic recombination in *Streptomyces coelicolor*. J. Gen. Microbiol. **16**:ii–iii.

88. Hopwood, D. A. 1959. Linkage and the mechanism of recombination in *Streptomyces coelicolor*. Ann. N. Y. Acad. Sci. **81**:881–898.

89. Hopwood, D. A. 1965. New data on the linkage map of *Streptomyces coelicolor*. Genet. Res. **6**:248–262.

90. Hopwood, D. A. 1965. A circular linkage map in the actinomycete *Streptomyces coelicolor*. J. Mol. Biol. **12**:514–516.

91. Hopwood, D. A. 1966. Lack of constant genome ends in *Streptomyces coelicolor*. Genetics **54**:1177–1186.

92. Hopwood, D. A. 1967. Genetic analysis and genome structure in *Streptomyces coelicolor*. Bacteriol. Rev. **31**:373–403.

93. Hopwood, D. A. 1969. Genome topology and mapping in *Streptomyces coelicolor*, p. 5–18. In G. Sermonti and M. Alačević (ed.), Proceedings of the International Symposium on the Genetics and Breeding of Streptomycetes, Dubrovnik, Yugoslavia, 1968. Yugoslav Academy of Sciences and Arts, Zagreb, Yugoslavia.

94. Hopwood, D. A. 1978. Extrachromosomally determined antibiotic production. Annu. Rev. Microbiol. **32**:373–392.

95. Hopwood, D. A. 1983. Actinomycete genetics and antibiotic production, p. 1–23. In L. C. Vining (ed.), Biochemistry and genetic regulation of commercially important antibiotics. Addison-Wesley, Reading, Mass.

96. Hopwood, D. A. 1997. Genetic contribution to understanding polyketide synthases. Chem. Rev. **97**:2465–2497.

97. Hopwood, D. A. 2003. *Streptomyces* genes: from Waksman to Sanger. J. Ind. Microbiol. Biotechnol. **30**:468–471.

98. Hopwood, D. A., and K. F. Chater. 1984. Streptomycetes, p. 7–42. In C. Ball (ed.), Genetics and breeding of industrial microorganisms. CRC Press, Boca Raton, Fla.

99. Hopwood, D. A., and G. Sermonti. 1962. The genetics of *Streptomyces coelicolor*. Adv. Genet. **11**:273–342.

100. Hopwood, D. A., and H. M. Wright. 1978. Bacterial protoplast fusion: recombination in fused protoplasts of *Streptomyces coelicolor*. Mol. Gen. Genet. **162**:307–317.

101. Hopwood, D. A., F. Malpartida, and K. F. Chater. 1986. Gene cloning to analyse the organization and expression of antibiotic biosynthesis genes in *Streptomyces*, p. 22–33. In H. Kleinkauf, H. Van Döhlen, H. Dornauer, and G. Nesemann (ed.), Regulation of secondary metabolite formation. VCH, Weinheim, Germany.

102. Hopwood, D. A., G. Sermonti, and I. Spada-Sermonti. 1963. Heterozygous clones in *Streptomyces coelicolor*. J. Gen. Microbiol. **20**:249–260.

103. Hoštálek, Z., and Z. Vanek. 1986. Tetracyclines, p. 393–429. In H.-J. Rehm and G. Reed (ed.), Biotechnology, vol. 4. VCH Verlagsgesellschaft, Weinheim, Germany.

104. Hranueli, D., J. Pigac, and M. Vešligaj. 1979. Characterization and persistence of actinophage RP2 isolated from *Streptomyces rimosus* ATCC 10970. J. Gen. Microbiol. **114**:295–303.

105. Hranueli, D., T. Smokvina, and M. Alačević. 1986. A comparative study of protoplast preparation and regeneration in *Streptomyces rimosus* and *Streptomyces lividan*. Microbiologica **9**:387–392.

106. Hranueli, D., M. Filipović, J. Pigac, and M. Vešligaj. 1981. Properties of defective lysogenic variants of *Streptomyces rimosus*, p. 469–470. In K. P. Schaal and G. Pulverer (ed.), Actinomycetes. Proceedings of the Fourth International Symposium on Actinomycete Biology, Cologne, Germany, 1979. Gustav Fischer Berlag, Stuttgart, Germany.

107. Hranueli, D., M. Filipović, J. Pigac, and M. Vešligaj. 1983. The interference of prophage with the biosynthesis of antibiotic. Kem. Ind. (Zagreb) **32**:401–404. (In Croatian.)

108. Hranueli, D., J. Pigac, T. Smokvina, and M. Alačević. 1983. Genetic interactions in *Streptomyces rimosus* mediated by conjugation and by protoplast fusion. J. Gen. Microbiol. **129**:1415–1422.

109. Hranueli, D., J. Cullum, B. Basrak, P. Goldstein, and P. F. Long. 2005. Plasticity of the *Streptomyces* genome-evolution and engineering of new antibiotics. Curr. Med. Chem. **12**:1697–1704.

110. Hranueli, D., B. Gravius, T. Bezmalinović, A. Eichner, and J. Cullum. 1995. ‘Strain degeneration’ in industrial streptomycetes. Food Technol. Biotechnol. **33**:51–54.

111. Hranueli, D., N. Perić, B. Borovička, S. Bogdan, J. Cullum, P. G. Waterman, and I. Hunter. 2001. Molecular biology of polyketide biosynthesis. Food Technol. Biotechnol. **39**:203–213.

112. Hranueli, D., N. Perić, H. Petković, G. Biuković, Z. Toman, J. Pigac, B. Borovička, A. Bago, I. Crnolatac, T. Maršić, L. Zhou, S. Matošić, P. G.

Waterman, J. Cullum, and I. S. Hunter. 1999. Novel hybrid polyketide compounds produced by genetic engineering of the oxytetracycline biosynthetic pathway. Food Technol. Biotechnol. 37:117–125.

113. Huang, C.-H., Y. S. Lin, Y. L. Yang, S. W. Huang, and C. W. Chen. 1998. The telomeres of *Streptomyces* chromosomes contain conserved palindromic sequences with potential to form complex secondary structures. Mol. Microbiol. 28:905–916.

114. Hunter, I. S. 1985. Gene cloning in *Streptomyces*, p. 19–44. *In* D. M. Glover (ed.), DNA cloning, vol. 2. A practical approach. IRL Press, Oxford, United Kingdom.

115. Hunter, I. S., and E. J. Friend. 1984. “Restriction-deficient” mutants of industrial *Streptomyces*. Biochem. Soc. Trans. 12:643–644.

116. Hunter, I. S., and R. A. Hill. 1997. Tetracyclines, p. 659–682. *In* W. R. Strohl (ed.), Biotechnology of antibiotics, 2nd ed. Marcel Dekker, New York, N.Y.

117. Hütter, R., and T. Eckhardt. 1988. Genetic manipulation, p. 89–184. *In* M. Goodfellow, S. T. Williams, and M. Mordarski (ed.), Actinomycetes in biotechnology. Academic Press, San Diego, Calif.

118. Ikeda, H., J. Ishikawa, A. Hanamoto, M. Shinose, H. Kikuchi, T. Shiba, Y. Sakaki, M. Hattori, and S. Omura. 2003. Complete genome sequence and comparative analysis of the industrial microorganism *Streptomyces avermitilis*. Nat. Biotechnol. 21:526–531.

119. Ishida, I., M. Obinata, and T. Deguchi. 1987. Molecular cloning and nucleotide sequence of cDNA encoding hydroxyindole O-methyltransferase of bovine pineal glands. J. Biol. Chem. 262:2895–2899.

120. Islam, M. M., C. D. Franco, D. W. Courtman, and M. P. Bendeck. 2003. A nonantibiotic chemically modified tetracycline (CMT-3) inhibits intimal thickening. Am. J. Pathol. 163:1557–1566.

121. Jaurin, B., and S. N. Cohen. 1985. *Streptomyces* contain *Escherichia coli*-type A + T-rich promoters having novel structural features. Gene 39:191–201.

122. Kantola, J., T. Kunnari, P. Mantsala, and K. Ylihonkka. 2003. Expanding the scope of aromatic polyketides by combinatorial biosynthesis. Comb. Chem. High Throughput Screen. 6:501–512.

123. Karoonuthaisiri, N., D. Weaver, J. Huang, S. N. Cohen, and C. M. Kao. 2005. Regional organization of gene expression in *Streptomyces coelicolor*. Gene 353:53–66.

124. Kerry, J., M. Hiney, R. Coyne, S. Nicgabhainn, D. Gilroy, D. Cazabon, and P. Smith. 1995. Fish feed as a source of oxytetracycline-resistant bacteria in the sediments under fish farms. Aquaculture 131:101–113.

125. Khosla, C., and Y. Tang. 2005. Chemistry. A new route to designer antibiotics. Science 308:367–368.

126. Kieser, T., D. A. Hopwood, H. M. Wright, and C. J. Thompson. 1982. pIJ101, a multi-copy broad host-range *Streptomyces* plasmid: functional analysis and development of DNA cloning vectors. Mol. Gen. Genet. 185:223–238.

127. Kim, E. S., H. Kim, S. Hong, K. H. Kang, E. Kho, and Y. Park. 1993. Gene organization and primary structure of a ribosomal RNA gene cluster from *Streptomyces griseus* subsp. *griseus*. Gene 132:21–31.

128. Kuchino, Y., I. Yamamoto, and S. Nishimura. 1982. Nucleotide sequence of *Streptomyces griseus* initiator tRNA. Nucleic Acids Res. 10:6671–6674.

129. Lanoot, B., M. Vancanneyt, B. Hoste, K. Vandemeulebroecke, M. C. Cnockaert, P. Dawyndt, Z. Liu, Y. Huang, and J. Swings. 2005. Grouping of streptomycetes using 16S-ITS RFLP fingerprinting. Res. Microbiol. 156:755–762.

130. Lee, C. Z., B. Xu, T. Hashimoto, C. E. McCulloch, G.-Y. Yang, and W. L. Young. 2004. Doxycycline suppresses cerebral matrix metalloproteinase-9 and angiogenesis induced by focal hyperstimulation of vascular endothelial growth factor in a mouse model of stroke. Stroke 35:1715–1719.

131. Lee, C. Z., J. S. Yao, Y. Huang, W. Zhai, W. Liu, B. J. Guglielmo, E. Lin, G. Y. Yang, and W. L. Young. 2006. Dose-response effect of tetracyclines on cerebral matrix metalloproteinase-9 after vascular endothelial growth factor hyperstimulation. J. Cereb. Blood Flow Metab. [Epub ahead of print.]

132. Lee, S. C., C. A. Omer, M. A. Brasch, and S. N. Cohen. 1988. Analysis of recombination occurring at SLP1 att sites. J. Bacteriol. 170:5806–5813.

133. Lezhava, A., T. Mizukami, T. Kajitani, D. Kameoka, M. Redenbach, H. Shinkawa, O. Nimi, and H. Kinashi. 1995. Physical map of the linear chromosome of *Streptomyces griseus*. J. Bacteriol. 177:6492–6498.

134. Lomovskaya, N. D., K. F. Chater, and N. M. Mkrtumian. 1980. Genetics and molecular biology of *Streptomyces* bacteriophages. Microbiol. Rev. 44:206–229.

135. Lu, D., and P. Holtom. 2005. Community-acquired methicillin-resistant *Staphylococcus aureus*, a new player in sports medicine. Curr. Sports Med. Rep. 4:265–270.

136. Lydiate, D. J., F. Malpartida, and D. A. Hopwood. 1985. The *Streptomyces* plasmid SCP2*: its functional analysis and development into useful cloning vectors. Gene 35:223–235.

137. Macgregor-Pryde, S. E. 1995. Ph.D. thesis. The University of Glasgow, Glasgow, United Kingdom.

138. MacNeil, D. J. 1988. Characterization of a unique methyl-specific restriction system in *Streptomyces avermitilis*. J. Bacteriol. 170:5607–5612.

139. Margulies, M., M. Egholm, W. E. Altman, S. Attiya, J. S. Bader, L. A. Bemben, J. Berka, M. S. Braverman, Y. J. Chen, Z. Chen, S. B. Dewell, L.


Du, J. M. Fierro, X. V. Gomes, B. C. Godwin, W. He, S. Helgesen, C. H. Ho, G. P. Irzyk, S. C. Jando, M. L. Alenquer, T. P. Jarvie, K. B. Jirage, J. B. Kim, J. R. Knight, J. R. Lanza, J. H. Leamon, S. M. Lefkowitz, M. Lei, J. Li, K. L. Lohman, H. Lu, V. B. Makhijani, K. E. McDade, M. P. McKenna, E. W. Myers, E. Nickerson, J. R. Nobile, R. Plant, B. P. Puc, M. T. Ronan, G. T. Roth, G. J. Sarkis, J. F. Simons, J. W. Simpson, M. Srinivasan, K. R. Tartaro, A. Tomasz, K. A. Vogt, G. A. Volkmer, S. H. Wang, Y. Wang, M. P. Weiner, P. Yu, R. F. Begley, and J. M. Rothberg. 2005. Genome sequencing in microfabricated high-density picolitre reactors. Nature 437:376–380.

140. Martin, J. F., and A. L. Demian. 1980. Control of antibiotic biosynthesis. Microbiol. Rev. 44:230–251.

141. Martin, J. F., G. Naharro, P. Liras, and J. R. Villanueva. 1979. Isolation of mutants deregulated in phosphate control of candicin biosynthesis. J. Antibiot. 32:600–606.

142. Mazodier, P., R. Petter, and C. Thompson. 1989. Intergeneric conjugation between *Escherichia coli* and *Streptomyces* species. J. Bacteriol. 171:3583–3585.

143. Mazodier, P., C. Thompson, and F. Bocard. 1990. The chromosomal integration site of the *Streptomyces* element pSAM2 overlaps a putative tRNA gene conserved among actinomycetes. Mol. Gen. Genet. 222:431–434.

144. Mazy-Servais, C., D. Baczkowski, and J. Dusart. 1997. Electroporation of intact cells of *Streptomyces parvulus* and *Streptomyces vinaceus*. FEMS Microbiol. Lett. 151:135–138.

145. McCormick, J. R. D. 1967. Tetracyclines, p. 113–122. *In* D. Gottlieb and P. D. Shaw (ed.), Antibiotics, vol. 2. Biosynthesis. Springer-Verlag, Berlin, Germany.

146. McCormick, J. R. D. 1969. Point blocked mutants and the biogenesis of tetracyclines, p. 163–176. *In* G. Sermonti and M. Alačević (ed.), Proceedings of the International Symposium on the Genetics and Breeding of Streptomycetes, Dubrovnik, Yugoslavia, 1968. Yugoslav Academy of Sciences and Arts, Zagreb, Yugoslavia.

147. McCormick, J. R. D., and G. O. Morton. 1982. Identity of cosynthetic factor 1 of *Streptomyces aureofaciens* and fragment FO from coenzyme F₄₂₀ of Methanobacterium sp. J. Am. Chem. Soc. 104:4014–4015.

148. McDaniel, R., S. Ebert-Khosla, D. A. Hopwood, and C. Khosla. 1995. Rational design of aromatic polyketide natural products by recombinant assembly of enzymatic subunits. Nature 375:549–554.

149. McDowall, K. J., A. Thamchaipenet, and I. S. Hunter. 1999. Oxytetracycline production by *Streptomyces rimosus* is controlled at the level of transcription from promoters overlapped by tandem repeats similar to those of the DNA-binding sites of *Escherichia coli* PhoB. J. Bacteriol. 181:3025–3032.

150. McDowall, K. J., D. Doyle, M. J. Butler, C. Binnie, M. Warren, and I. S. Hunter. 1991. Molecular genetics of oxytetracycline production by *Streptomyces rimosus*, p. 105–116. *In* S. Baumberg, H. Krügel, and D. Noack (ed.), Proceedings of the International Symposium on the Genetics and Product Formation in Streptomycetes, Erfurt, Germany, 1990. Plenum Press, New York, N.Y.

151. McMurry, L. M., and S. B. Levy. 1998. Revised sequence of OtrB (*tet347*) tetracycline efflux protein from *Streptomyces rimosus*. Antimicrob. Agents Chemother. 42:3050.

152. M’ikanatha, N. M., K. G. Julian, A. R. Kunselman, R. C. Aber, J. T. Rankin, and E. Lautenbach. 2005. Patients’ request for and emergency physicians’ prescription of antimicrobial prophylaxis for anthrax during the 2001 bioterrorism-related outbreak. BMC Public Health 5:2.

153. Mikoc, A., D. Vujaklija, and V. Gamulin. 1997. The *recA* gene from *Streptomyces rimosus* R6: sequence and expression in *Escherichia coli*. Res. Microbiol. 148:397–403.

154. Miller, P. A., N. O. Sjolander, S. Nalesynk, N. Arnold, S. Johnson, A. P. Doerschuk, and J. R. D. McCormick. 1960. Cosynthetic factor 1, a factor in hydrogen-transfer in *Streptomyces aureofaciens*. J. Am. Chem. Soc. 82:5002–5003.

155. Mindlin, S. Z. 1969. Genetic recombination in Actinomycetes breeding, p. 147–159. *In* G. Sermonti and M. Alačević (ed.), Proceedings of International Symposium on Genetics and Breeding of Streptomycetes, Dubrovnik, Yugoslavia, 1968. Yugoslav Academy of Sciences and Arts, Zagreb, Yugoslavia.

156. Mindlin, S. Z., and S. I. Alikhanian. 1958. A study of UVR-induced variation and selection of *Actinomyces rimosus* (oxytetracycline producer). Antibiotiki 3:18–21. (In Russian.)

157. Mindlin, S. Z., Z. M. Zaytzeva, and T. A. Shishkina. 1968. Genetic and physiological studies of inactive mutants of *Actinomyces rimosus*, the producer of oxytetracycline. Genetika 3:126–135. (In Russian.)

158. Mindlin, S. Z., S. I. Alikhanian, A. V. Vladimirov, and G. R. Mikhailova. 1961. A new hybrid strain of an oxytetracycline-producing organism, *Streptomyces rimosus*. Appl. Microbiol. 9:349–353.

159. Mindlin, S. Z., Z. M. Zaytzeva, A. B. Germanov, and T. A. Shishkina. 1966. The genetic analysis of “inactive” mutants of *Actinomyces rimosus*, p. 695–697. *In* M. Herold and Z. Gabriel (ed.), Proceedings of the Congress on Antibiotics. Antibiotics—advances in research, production and clinical use. Czechoslovak Medical Press, Prague, Czechoslovakia.

160. Mitscher, L. A. 1978. The chemistry of the tetracycline antibiotic. Marcel Dekker, New York, N.Y.

161. Mochizuki, S., K. Hiratsu, M. Suwa, T. Ishii, F. Sugino, K. Yamada, and H. Kinashi. 2003. The large linear plasmid pSLA2-L of *Streptomyces rochei* has an unusually condensed gene organization for secondary metabolism. Mol. Microbiol. **48**:1501–1510.

162. Movahedzadeh, F., M. J. Colston, and E. O. Davis. 1997. Determination of DNA sequences required for regulated *Mycobacterium tuberculosis* RecA expression in response to DNA-damaging agents suggests that two modes of regulation exist. J. Bacteriol. **179**:3509–3518.

163. Naidenova, M., and H. I. Krustev. 1985. Electron microscopic investigations of fusion between protoplasts of *Streptomyces rimosus* and *Streptomyces aureofaciens*. Dokl. Bolg. Akad. Nauk **38**:623–626.

164. Newton, P. N., J. F. Chaulet, A. Brockman, W. Chierakul, A. Dondorp, R. Ruangveerayuth, S. Looareesuwan, C. Mounier, and N. J. White. 2005. Pharmacokinetics of oral doxycycline during combination treatment of severe falciparum malaria. Antimicrob. Agents Chemother. **49**:1622–1625.

165. Novotna, J., J. Neuzil, I. Vančurova, V. Běhal, and Z. Hostalek. 1990. Enzymes and coenzymes of the terminal part of the antibiotic pathway in *Streptomyces* producing tetracyclines, p. 202–219. In S. Baumberg, H. Krügel, and D. Noack (ed.), Proceedings of the International Symposium on the Genetics and Product Formation in Streptomycetes, Erfurt, Germany, 1990. Plenum Press, New York, N.Y.

166. Nubaumer, B., and W. Wohlleben. 1994. Identification, isolation and sequencing of the *recA* gene of *Streptomyces lividanis* TK24. FEMS Microbiol. Lett. **118**:57–63.

167. Ohnuki, T., T. Katoh, T. Imanaka, and S. Aiba. 1985. Molecular cloning of tetracycline resistance genes from *Streptomyces rimosus* in *Streptomyces griseus* and characterization of the cloned genes. J. Bacteriol. **161**:1010–1016.

168. Okanishi, M. 1980. Plasmid involvement in antibiotic production, p. 29–46. In K. Sakaguchi and M. Okanishi (ed.), Molecular breeding and genetics of applied microorganisms. Kodansha, Tokyo, Japan.

169. Omer, C. A., and S. N. Cohen. 1984. Plasmid formation in *Streptomyces*: excision and integration of the SLP1 replicon at a specific chromosomal site. Mol. Gen. Genet. **196**:429–438.

170. Pandey, R. C., and K. L. Rinehart, Jr. 1977. Polyene antibiotics. VIII. The structure of rimocidin. J. Antibiot. **30**:146–157.

171. Pandza, K. 1996. Ph.D. thesis. The University of Kaiserslautern, Kaiserslautern, Germany.

172. Pandza, K., G. Pfalzer, J. Cullum, and D. Hranueli. 1997. Physical mapping shows that the unstable oxytetracycline gene cluster of *Streptomyces rimosus* lies close to one end of the linear chromosome. Microbiology **143**:1493–1501.

173. Pandza, S., G. Biuković, A. Paravić, J. Cullum, and D. Hranueli. 1998. Recombination between the linear plasmid pPZG101 and the linear chromosome of *Streptomyces rimosus* can lead to exchange of ends. Mol. Microbiol. **28**:1165–1176.

174. Perez-Zuniga, F. J., E. M. Seco, T. Cuesta, F. Degenhardt, J. Rohr, C. Vallin, Y. Iznaga, M. E. Perez, L. Gonzalez, and F. Malpartida. 2004. CE-108, a new macrolide tetraene antibiotic. J. Antibiot. **57**:197–204.

175. Perić-Concha, N., B. Borovička, P. F. Long, D. Hranueli, P. G. Waterman, and I. S. Hunter. 2005. Ablation of the *otcC* gene encoding a post-polyketide hydroxylase from the oxytetracyline biosynthetic pathway in *Streptomyces rimosus* results in novel polyketides with altered chain length. J. Biol. Chem. **280**:37455–37460.

176. Pernodet, J. L., F. Bocard, M. T. Alegre, J. Gagnat, and M. Guérineau. 1989. Organization and nucleotide sequence analysis of a ribosomal RNA gene cluster from *Streptomyces ambofaciens*. Gene **79**:33–46.

177. Petković, H., A. Thamchaipenet, L.-H. Zhou, D. Hranueli, P. Raspor, P. G. Waterman, and I. S. Hunter. 1999. Disruption of the aromatase/cyclase from the oxytetracycline gene cluster of *Streptomyces rimosus* results in production of novel polyketides with shorter chain lengths. J. Biol. Chem. **274**:32829–32834.

178. Pigac, J. 1975. Ph.D. thesis. University of Zagreb, Zagreb, Yugoslavia.

179. Pigac, J. 1991. Structural instability of bifunctional vectors in *Streptomyces*, p. 287–294. In S. Baumberg, H. Krügel, and D. Noack (ed.), Proceedings of the International Symposium on the Genetics and Product Formation in Streptomycetes, Erfurt, Germany, 1990. Plenum Press, New York, N.Y.

180. Pigac, J., and M. Alačević. 1979. Mapping of oxytetracycline genes in *Streptomyces rimosus*. Period. Biol. **81**:575–582.

181. Pigac, J., and H. Schrempf. 1995. A simple and rapid method of transforming of *Streptomyces rimosus* R6 and other *Streptomyces* by electroporation. Appl. Environ. Microbiol. **61**:352–356.

182. Pigac, J., D. Vujaklija, and V. Gamulin. 1986. Structural segregation of a bifunctional vector pZG1 in *Streptomyces lividanis* and *Streptomyces rimosus*, p. 103–105. In G. Szabó, S. Biró, and M. Goodfellow (ed.), Proceedings of the Sixth International Symposium on Actinomycetes: biological, biochemical and biomedical aspects of actinomycetes, part A, Debrecen, Hungary, 1985. Akadémiai Kiadó, Budapest, Hungary.

183. Pigac, J., D. Hranueli, T. Smokvina, and M. Alačević. 1982. Optimal cultural and physiological conditions for handling *Streptomyces rimosus* protoplasts. Appl. Environ. Microbiol. **44**:1178–1186.

184. Pigac, J., D. Vujaklija, Z. Toman, V. Gamulin, and H. Schrempf. 1988.

Structural instability of a bifunctional plasmid pZG1 and single-stranded DNA formation in *Streptomyces*. Plasmid **19**:222–230.

185. Plohl, M., and V. Gamulin. 1990. Five transfer RNA genes lacking CCA termini are clustered in the chromosome of *Streptomyces rimosus*. Mol. Gen. Genet. **222**:129–134.

186. Plohl, M., and V. Gamulin. 1991. Sequence of the 5S rRNA gene organization of ribosomal RNA operons in *Streptomyces rimosus*. FEMS Microbiol. Lett. **77**:139–144.

187. Polsinelli, M., and M. Beretta. 1966. Genetic recombination in crosses between *Streptomyces aureofaciens* and *Streptomyces rimosus*. J. Bacteriol. **91**:63–68.

188. Potekhin, Y. A., and V. N. Danilenko. 1985. Kanamycin resistance determinant in *Streptomyces rimosus*: capacity for amplification in chromosome and for reversible genetic instability. Mol. Biol. (Moscow) **19**:805–817. (In Russian.)

189. Pujić, P., S. Durajlija-Žinić, S. Pandža, A. Mikoč, M. Plohl, and V. Gamulin. 2001. Ribosomal RNA operons in *Streptomyces rimosus*: sequence of the *rrnF* and comparative analysis of *rrn* promoter regions. Food Technol. Biotechnol. **39**:77–81.

190. Ramos, J. L., M. Martinez-Bueno, A. J. Molina-Henares, W. Terán, K. Watanabe, X. Zhang, M. T. Gallegos, R. Brennan, and R. Tobes. 2005. The TetR family of transcriptional repressors. Microbiol. Mol. Biol. Rev. **69**:326–356.

191. Randazzo, R., G. Sermenti, A. Carere, and M. Bignami. 1973. Comutation in *Streptomyces*. J. Bacteriol. **113**:500–501.

192. Rauland, U., I. Glockner, M. Redenbach, and J. Cullum. 1995. DNA amplifications and deletions in *Streptomyces lividanis* 66 and the loss of one end of the linear chromosome. Mol. Gen. Genet. **246**:37–44.

193. Rausch, H., M. Vešligaj, D. Počta, G. Biuković, J. Pigac, J. Cullum, H. Schmieger, and D. Hranueli. 1993. The temperate phages RP2 and RP3 of *Streptomyces rimosus*. J. Gen. Microbiol. **139**:2517–2524.

194. Rautenshtein, Y. I., V. V. Zhunaeva, Y. M. Khokhlova, N. G. Nesterova, and L. N. Moskalenko. 1975. Factors inducing lysis of *Actinomyces rimosus* producing oxytetracycline. Mikrobiologiya **44**:899–904. (In Russian.)

195. Rautenshtein, Y. I., T. G. Borisova, S. Y. Goldat, V. V. Zhunaeva, A. V. Vladimirov, L. N. Moskalenko, L. S. Terebkova, L. S. Isaikina, and N. G. Misiureva. 1972. Lysis in the culture of *Actinomyces rimosus*. Mikrobiologiya **41**:1055–1063. (In Russian.)

196. Redenbach, M., H. M. Kieser, D. Denapaite, A. Eichner, J. Cullum, H. Kinashi, and D. A. Hopwood. 1996. A set of ordered cosmids and a detailed genetic and physical map for the 8 Mb *Streptomyces coelicolor* A3(2) chromosome. Mol. Microbiol. **21**:77–96.

197. Reiter, W. D., P. Palm, and S. Yeats. 1989. Transfer RNA genes frequently serve as integration sites for prokaryotic genetic elements. Nucleic Acids Res. **17**:1907–1914.

198. Reynes, J. P., T. Calmels, D. Drocourt, and G. Tiraby. 1988. Cloning, expression in *Escherichia coli* and nucleotide sequence of a tetracycline-resistance gene from *Streptomyces rimosus*. J. Gen. Microbiol. **134**:585–598.

199. Rhodes, P. M. 1986. Genetic recombination and maps for *Streptomyces*, p. 27–60. In S. W. Queener and L. E. Day (ed.), The bacteria, vol. 9. Antibiotic-producing Streptomycetes. Academic Press, Orlando, Fla.

200. Rhodes, P. M., N. Winskill, E. J. Friend, and M. Warren. 1981. Biochemical and genetic characterization of *Streptomyces rimosus* mutants impaired in oxytetracycline biosynthesis. J. Gen. Microbiol. **124**:329–338.

201. Rhodes, P. M., I. S. Hunter, E. J. Friend, and M. Warren. 1984. Recombinant DNA methods for the oxytetracycline producer *Streptomyces rimosus*. Biochem. Soc. Trans. **12**:586–587.

202. Ryan, M. J., J. A. Lotvin, N. K. Strathy, and S. E. Fantini. December 1996. Cloning of the biosynthetic pathway for chlorotetracycline and tetracycline formation and cosmids useful therein. U.S. patent 5589385-A 1.

203. Seco, E. M., F. J. Perez-Zuniga, M. S. Rolon, and F. Malpartida. 2004. Starter unit choice determines the production of two tetraene macrolides, rimocidin and CE-108, in *Streptomyces diastaticus* var. 108. Chem. Biol. **11**:357–366.

204. Sedlmeier, R., T. Werner, H. M. Kieser, D. A. Hopwood, and H. Schmieger. 1994. tRNA genes of *Streptomyces lividanis*: new sequences and comparison of structure and organization with those of other bacteria. J. Bacteriol. **176**:5550–5553.

205. Sermenti, G. 1969. Fertility system and genome transfer in *Streptomyces coelicolor*, p. 21–26. In G. Sermenti and M. Alačević (ed.), Proceedings of the International Symposium on the Genetics and Breeding of Streptomycetes, Dubrovnik, Yugoslavia, 1968. Yugoslav Academy of Sciences and Arts, Zagreb, Yugoslavia.

206. Sermenti, G., and D. A. Hopwood. 1964. Genetic recombination in Streptomycetes, p. 223–251. In I. C. Gunzalus and R. Y. Stanier (ed.), The bacteria, vol. 5. Academic Press, New York, N.Y.

207. Sermenti, G., and I. Spada-Sermenti. 1960. Genetic recombination in Streptomycetes. Nature **176**:121.

208. Smokvina, T. 1984. M.S. thesis. University of Zagreb, Zagreb, Yugoslavia.

209. Sowinski, P., J. Pawlak, E. Borowski, and P. J. Gariboldi. 1995. Stereostructure of rimocidin. J. Antibiot. **48**:1288–1291.

210. Starodubtseva, L. I., and V. N. Danilenko. 1989. Location of determinants

of stability to neomycin and kanamycin as well as DNA segments, responsible for amplification in *Streptomyces* plasmids pSU3 and pSU10. Mol. Biol. (Moscow) 23:193–203. (In Russian.)

211. Starodubtseva, L. I., A. S. Taisova, and V. N. Danilenko. 1985. Study on amplification of kanamycin resistance determinant (*Kan*') in constructed hybrid plasmids of *Streptomyces lividans*. Antibiotiki 30:565–572. (In Russian.)

212. Stepnov, V. P., M. M. Garaev, A. R. Fedotov, and E. I. Golub. 1978. Plasmids in actinomycetes producing oxytetracycline and neomycin. Antibiotiki 23:892–895. (In Russian.)

213. Strohl, W. R. 1992. Compilation and analysis of DNA sequences associated with apparent streptomycete promoters. Nucleic Acids Res. 20:961–974.

214. Sum, P. E., V. J. Lee, R. T. Testa, J. J. Hlavka, G. A. Ellestad, J. D. Bloom, Y. Gluzman, and F. P. Tally. 1994. Glycylcyclines. 1. A new generation of potent antibacterial agents through modification of 9-aminotetracyclines. J. Med. Chem. 37:184–188.

215. Summers, R. G., E. Wendt-Pienkowski, H. Motamedi, and C. R. Hutchinson. 1992. Nucleotide sequence of the *tcmII-tcmIV* region of the tetracenomycin C biosynthetic gene cluster of *Streptomyces glaucescens* and evidence that the *tcmN* gene encodes a multifunctional cyclase-dehydratase-O-methyltransferase. J. Bacteriol. 174:1810–1820.

216. Supputamongkol, Y., K. Niwattayakul, C. Suttinont, K. Losuwanaluk, R. Limpaiboon, W. Chierakul, V. Wuthiekanun, S. Triengrim, M. Chenchittikul, and N. J. White. 2004. An open, randomized, controlled trial of penicillin, doxycycline, and cefotaxime for patients with severe leptospirosis. Clin. Infect. Dis. 15:1417–1424.

217. Thamchaipenet, A. 1994. Ph.D. thesis. University of Glasgow, Glasgow, United Kingdom.

218. Thomas, R., and D. J. Williams. 1983. Oxytetracycline biosynthesis: mode of incorporation of \[^{13}C\]- and \[^{1,2-13}C_2\]-acetate. J. Chem. Soc. Chem. Commun. 12:128–130.

219. Thomas, R., and D. J. Williams. 1983. Oxytetracycline biosynthesis: origin of the carboxamide substituent. J. Chem. Soc. Chem. Commun. 12:677–679.

220. Uchida, T., N. Ishihara, H. Zenitani, K. Hiratsu, and H. Kinashi. 2004. Circularized chromosome with a large palindromic structure in *Streptomyces griseus* mutants. J. Bacteriol. 186:3313–3320.

221. Usdin, K., K. Gertsch, and R. Kirby. 1984. Evidence for the wide distribution of repetitive DNA sequences in the genus *Streptomyces*. J. Mol. Evol. 20:25–30.

222. Vallin, C., R. Rodriquez, A. Ramos, E. Alonso, and S. Biro. 1986. Increased oxytetracycline production in *Streptomyces rimosus* after protoplast fusion. Biotechnol. Lett. 8:343–344.

223. Vallin, C., A. Ramos, R. Rodriquez, D. Perez-Velazco, and E. Alonso. 1988. Use of protoplast fusion in the isolation of high antibiotic-producing *Streptomyces* strains. Interferon Biotechnol. 5:288–290. (In Spanish.)

224. van Wezel, G. P., I. M. Krab, S. Douthwaite, M. J. Bibb, E. Vijgenboom, and L. Bosch. 1994. Transcription analysis of the *Streptomyces coelicolor* A3(2) *rrnA* operon. Microbiology 140:3357–3365.

225. Vešligaj, M., M. Filipović, J. Pigac, and D. Hranueli. 1981. Isolation of *Streptomyces rimosus* mutants with reduced actinophage susceptibility. Appl. Environ. Microbiol. 41:986–991.

226. Voeykova, T. A., N. M. Mkrtumian, L. K. Emelyanova, and N. D. Lomovskaya. 1988. Relationship between actinophages and industrial strains of actinomycetes. Biotechnologiya 4:176–182. (In Russian.)

227. Volff, J. N., and J. Altenbuchner. 2000. A new beginning with new ends: linearisation of circular chromosomes during bacterial evolution. FEMS Microbiol. Lett. 186:143–150.

228. Vyskocil, P., L. Homolka, and P. Pilat. 1987. Use of protoplasts in the improvement of filamentous microorganisms. I. Action of *N*-methyl-*N'*-nitro-*N*-nitrosoguanidine on protoplasts of streptomycetes. Biotechnol. Tech. 1:25–30.

229. Wang, J., X. Kong, X. Zhang, J. Cheng, N. Li, and Y. Zhou. 1989. The mutation of protoplasts and spores of *Streptomyces rimosus* induced by UV light and sodium bisulfite. Kangshengsu 14:86–88. (In Chinese.)

230. Wang, S.-J., H.-M. Chang, Y.-S. Lin, C.-H. Huang, and C. W. Chen. 1999. Streptomyces genomes: circular genetic maps from the linear chromosomes. Microbiology 145:2209–2220.

231. Wang, Y., Z. Zhang, and N. Ramanan. 1997. The actinomycete *Thermo-bispora bispora* contains two distinct types of transcriptionally active 16S rRNA genes. J. Bacteriol. 179:3270–3276.

232. Watve, M. G., R. Tickoo, M. M. Jog, and B. D. Bhole. 2001. How many antibiotics are produced by the genus *Streptomyces*? Arch. Microbiol. 176: 386–390.

233. Welsch, M. 1969. Biology of actinophages, p. 43–62. In G. Sermonti and M. Alačević (ed.), Proceedings of International Symposium on Genetics and Breeding of *Streptomyces*, Dubrovnik, Yugoslavia, 1968. Yugoslav Academy of Sciences and Arts, Zagreb, Yugoslavia.

234. Wenner, T., V. Roth, G. Fischer, C. Fourrier, B. Aigle, B. Decaris, and P. Leblond. 2003. End-to-end fusion of linear deleted chromosomes initiates a cycle of genome instability in *Streptomyces ambofaciens*. Mol. Microbiol. 50:411–425.

235. Wenzel, R., G. Bate, and P. Kirkpatrick. 2005. Tigecycline. Nat. Rev. Drug Discov. 4:809–810.

236. Yao, W., and L. C. Vining. 1994. Cloning and sequence analysis of a *recA*-like gene from *Streptomyces venezuelae* ISP5230. FEMS Microbiol. Lett. 118:51–56.

237. Yap, W. H., and Y. Wang. 1999. Molecular cloning and comparative sequence analyses of rRNA operons in *Streptomyces nodosus* ATCC 14899. Gene 232:77–85.

238. Young, M., and D. Hranueli. 1988. Chromosomal gene amplification in gram-positive bacteria, p. 157–200. In J. A. Thomson (ed.), Recombinant DNA and bacterial fermentation. CRC Press, Boca Raton, Fla.

239. Zhang, W., B. D. Ames, and Y. Tang. 2006. Engineered biosynthesis of a novel amidated polyketide, using the malonyl-specific initiation module from the oxytetracycline polyketide synthase. Appl. Environ. Microbiol. 72:2573–2580.
