
Molecular Microbiology (2006) 61(6), 1372–1379

doi:10.1111/j.1365-2958.2006.05331.x
First published online 10 August 2006

# MicroReview

## The exosome: a macromolecular cage for controlled RNA degradation

Katharina Büttner, Katja Wenig and Karl-Peter Hopfner*

Gene Center and Department of Chemistry and Biochemistry, University of Munich, Feodor-Lynen-Str. 25, 81377 Munich, Germany.

### Summary

The exosome, a large multisubunit complex with exoribonucleic activity, emerges as the central 3′ RNA degradation and processing factor in eukaryotes and archaea. But how are the many RNA substrates of the exosome degraded in a processive, yet controlled manner? Recent functional and structural progress shows that the exosome is a macromolecular cage, where the nuclease active sites are situated in a central processing chamber. A narrow entry pore controls access to the active sites in the processing chamber and prevents uncontrolled RNA decay. The emerging mechanism of exosome function suggests a strikingly parallel architectural concept to protein degradation by proteasomes.

### Introduction

The correct processing, quality control and turnover of cellular RNA molecules is critical to many aspects in the expression of genetic information. One of the key enzymes involved in these processes in eukaryotic and archaeal cells is the exosome. The exosome was originally identified as an essential factor in the maturation of ribosomal 5.8S rRNA in yeast and shown to be composed of a set of demonstrated or predicted exoribonucleases (Mitchell et al., 1997). Subsequent studies extended the role of the eukaryotic exosome from 5.8S rRNA maturation to numerous functions in RNA metabolism, like 3′ end processing and 3′→5′ degradation of nuclear snRNA and snoRNA (Allmang et al., 1999a; van Hoof et al., 2000a; Mitchell and Tollervey, 2003), quality control of (pre)mRNA and tRNAi(Met) (de la Cruz et al., 1998; Allmang

Accepted 17 July, 2006. *For correspondence. E-mail hopfner@imb.uni-muenchen.de; Tel. (+49) 89 2180 76953; Fax (+49) 89 2180 76999.

© 2006 The Authors
Journal compilation © 2006 Blackwell Publishing Ltd

et al., 1999a; Hilleren et al., 2001; Suzuki et al., 2001; Torchet et al., 2002; Mitchell and Tollervey, 2003; Kadaba et al., 2004; Rajmakers et al., 2004) and normal turnover of cytosolic mRNA (Anderson and Parker, 1998; Mitchell and Tollervey, 2001). In addition, recent results revealed an involvement of the exosome in the degradation of cryptic nuclear polymerase II transcripts and RNA fragments arising during RNA interference processes (Milligan et al., 2005; Orban and Izaurralde, 2005; Wyers et al., 2005).

The widespread function of the exosome in RNA metabolism virtually required the ability to process and/or degrade many different RNA molecules independently of their sequence and structure. This requirement now poses interesting questions about how exosome activity is regulated: How is the exosome prevented from randomly degrading cellular RNA? What mechanisms stimulate efficient and processive degradation of only substrate RNA? How does it distinguish substrate RNA from non-substrate RNA? Why are some exosome substrates processed only at the 3′ end (e.g. 7S to 5.8S rRNA), while many others are fully degraded? We summarize the progress on these aspects based on recent functional and structural analysis of exosomes.

### Composition and conservation of exosomes

The Saccharomyces cerevisiae core exosome consists of six polypeptides with similarity to the phosphate-dependent 3′→5′ exoribonuclease RNase PH (Rrp41, Rrp42, Rrp43, Rrp45, Rrp46, Mtr3), three (protein K homology) KH and/or (ribosomal protein S1 homology) S1 domain-containing RNA binding proteins (Rrp4, Rrp40, Csl4) (Table 1) and an RNase R homologue (Rrp44/Dis3) (Mitchell et al., 1997). It is found in the nucleus and cytoplasm. The nuclear isoform contains an additional subunit, the RNase D homologue Rrp6 (Mitchell et al., 1997; Allmang et al., 1999a). The human counterpart was identified soon after characterization of the yeast exosome. Its components turned out to be part of the previously described polymyositis/scleroderma (PM/Scl) autoantigenic multiprotein complex (Allmang et al., 1999b). The human exosome also contains six

Table 1. Components of the yeast and archaeal exosome.

| Yeast         | Archaea       | Domains          | Activity                                                                 |
|---------------|---------------|------------------|-------------------------------------------------------------------------|
| Core proteins |               |                  |                                                                         |
| Rrp41p        | aRrp41        | RNase PH         | Eukaryotes: 3′-5′ phosphorolytic exoribonuclease<sup>a</sup>              |
|               |               |                  | Archaea: 3′-5′ phosphorolytic exoribonuclease and adenylase<sup>b</sup>    |
| Rrp42p        | aRrp42        | RNase PH         |                                                                         |
| Rrp43p        | aRrp42        | RNase PH         |                                                                         |
| Rrp45p        | aRrp42        | RNase PH         |                                                                         |
| Rrp46p        | aRrp41        | RNase PH         | Archaea: 3′-5′ phosphorolytic exoribonuclease and adenylase<sup>b</sup>    |
| Mtr3p         | aRrp41        | RNase PH         | Archaea: 3′-5′ phosphorolytic exoribonuclease and adenylase<sup>b</sup>    |
| Csl4p         | aCsl4         | NT/S1/ZnR<sup>c</sup> | RNA binding<sup>d</sup>                                              |
| Rrp4p         | aRrp4         | NT/S1/KH         | Eukaryotes: 3′-5′ hydrolytic exoribonuclease<sup>e</sup>                 |
|               |               |                  | RNA binding<sup>f</sup>                                                |
| Rrp40p        | aRrp4         | NT/S1/KH         | RNA binding<sup>d</sup>                                                |
| Additional components |           |                  |                                                                         |
| Rrp6p         |               | RNase D          | 3′-5′ hydrolytic exoribonuclease<sup>g</sup>                           |
| Rrp44p        |               | RNase R          | 3′-5′ hydrolytic exoribonuclease<sup>a</sup>                            |
|               | aDnaG         | Toprim domain     | unknown<sup>h</sup>                                                    |

a. Mitchell *et al.* (1997).  
b. Buttner *et al.* (2005); Lorentzen *et al.* (2005).  
c. Eukaryotic Csl4 does not have the zinc co-ordinating cysteine residues of the aCsl4.  
d. Predicted from the domain architecture.  
e. Mitchell *et al.* (1997); Chekanova *et al.* (2002).  
f. Chekanova *et al.* (2002); Buttner *et al.* (2005).  
g. Burkard and Butler (2000).  
h. Eubacterial DnaG synthesizes short RNA oligonucleotides on a ssDNA template (primase).

RNase-PH type domains along with three KH and/or S1 domain-containing subunits. Similar exosomes were later described in other eukaryotes like the fruit fly (*Drosophila melanogaster*), plants (*Arabidopsis thaliana*) and protozoan parasites (*Trypanosoma brucei*) (Estevez *et al.*, 2001; Andrulis *et al.*, 2002; Chekanova *et al.*, 2002).

The archaeal exosome complex was predicted by Koonin and coworkers based on sequence analysis of archaeal genomes, which contain homologues of yeast Rrp4, Rrp41, Rrp42 and Csl4 (Koonin *et al.*, 2001). Affinity purification identified an exosome-like complexes in *Sulfolobus solfataricus* (Evguenieva-Hackenberg *et al.*, 2003). Proteomic analysis indicated that exosome subunits are expressed also in other archaea (Goodchild *et al.*, 2004; Farhoud *et al.*, 2005). Some archaea do not possess detectable exosome sequence homologues (*Methanocaldococcus jannaschii* and *Halobacterium sp.*, Koonin *et al.*, 2001), and RNA degradation in these organisms is not well understood. However, the exosome appears to be a conserved component of eukaryotes and most archaea.

Archaeal exosomes share high similarity to their eukaryotic counterparts but exhibit a simplified architecture (Table 1). The core complex consists of only two RNase PH-like proteins [archaeal (a)Rrp41 and aRrp42] and only two putative RNA binding proteins with either S1 and KH (aRrp4 subunit) or S1 and Zn-Ribbon (ZnR) domains (aCsl4 subunit) (Koonin *et al.*, 2001; Evguenieva-Hackenberg *et al.*, 2003). These proteins are present in multiple copies in the exosome complex, and

the overall number of RNase-PH homologue subunits and KH and/or S1 domain containing subunits appears to be evolutionary conserved (Evguenieva-Hackenberg *et al.*, 2003; Buttner *et al.*, 2005; Lorentzen *et al.*, 2005; Ramos *et al.*, 2006).

### Structural analysis of the exosome: common features in RNA degrading enzymes

Identification of the exosome led to the discovery that exosome subunits share sequence similarity with bacterial RNase-PH and polynucleotide phosphorylase (PNPase). PNPase is a central enzyme in eubacterial and organelle RNA 3′ degradation (see, e.g. Symmons *et al.*, 2002, and references). PNPase contains two RNase-PH domains, a KH domain and a S1 domain. Three PNPase monomers assemble into a ring-like structure, resulting in six RNase-PH type domains, topped by three S1 and KH domains (Symmons *et al.*, 2000). Association of the six RNase-PH type subunits of the eukaryotic exosome into a PNPase-like structure was proposed based on two-hybrid studies of protein–protein interactions of exosome subunits and electron microscopy of yeast core exosome particles (Aloy *et al.*, 2002; Rajmakers *et al.*, 2002). Recently, atomic resolution crystal structures of the complete archaeal exosome from *Archaeoglobus fulgidus* (Buttner *et al.*, 2005) and of the RNase PH ring from the *S. solfataricus* exosome in the absence and in complex with short RNA oligonucleotides (Lorentzen *et al.*, 2005; Lorentzen and

© 2006 The Authors  
Journal compilation © 2006 Blackwell Publishing Ltd, *Molecular Microbiology*, **61**, 1372–1379

1374 K. Büttner, K. Wenig and K.-P. Hopfner

![Image Description](image1.png)

Fig. 1. Structural features of the archaeal exosome.

a. Ribbon model of the nine subunit *Archaeoglobus fulgidus* exosome (left) and the structurally and architecturally related bacterial PNase trimer (right) (Symmons et al., 2000). The exosome consists of a hetero-hexameric ring of alternating Rrp41 (blue) and Rrp42 (green) subunits, topped by a trimer of Rrp4 subunits (orange). At the centre of the Rrp4 trimer is an opening (S1 pore) that connects the putative RNA binding surface of the S1 and KH domains with the internal processing chamber. The PNase complex is a homotrimer, where the protomers contain both RNase-PH (blue/green) and S1/KH (orange) domains in a single polypeptide chain.

b. Surface model, sliced at the centre of the archaeal exosome, using the colour code and annotation of 1a). Three active sites are buried in the processing chamber (only two are displayed). RNA, binding at the S1 domains, likely enters the processing chamber via narrow neck regions that restrict entry to single-stranded unfolded RNA.

c. Close up view, in stick representation, of a short oligo(A) bound at the active site of the *S. solfataricus* exosome (only selected protein residues are displayed).

The interaction of RNA with active site residues is mainly mediated by phosphate backbone and sugar moieties, consistent with sequence-independent RNA degradation. A chloride ion occupies the putative position of the attacking phosphate.

Conti, 2005) revealed a detailed image of the architecture of archaeal exosomes.

The archaeal exosome is a globular, double-donut-like structure with a central hole (Fig. 1). One ring of the double-donut is formed by the six RNase-PH type subunits. In archaea, three aRrp41 subunits alternate with three aRrp42 subunits (Lorentzen et al., 2005). The aRrp42:aRrp42 ring binds three copies of aRrp4 and/or aCsl4 subunits, which together form the second ring (Buttner et al., 2005). aCsl4 and aRrp4 both consist of three structural domains. An N-terminal (NT) domain anchors aRrp4 and aCsl4 onto the aRrp41:aRrp42 ring. The S1 domains are situated in the centre of the aRrp4/aCsl4 ring and frame a pore that is proposed to be the entry port for RNA. The KH domains of aRrp4 and zinc-ribbon (ZnR) domains of aCsl4 are situated on the periphery of the aRrp4/aCsl4 ring (Buttner et al., 2005). As homologues of KH, S1 and ZnR domains are typically involved in nucleic acid binding, this KH/S1/ZnR array might form a large RNA recruitment surface.

The archaeal exosome structure now reveals striking similarities to PNase (Fig. 1, Symmons et al., 2000). In fact, the aRrp41:aRrp42 dimer of the archaeal exosome is structurally equivalent to the tandem RNase-PH domains of PNases (Symmons et al., 2000; Buttner et al., 2005). In addition to the RNase-PH domains, the location of S1 and KH domains of PNase are similar (but not identical) to the location of S1 and KH domains of Rrp4 bound at the aRrp41:aRrp42 ring. This overall equivalence suggests a common evolutionary basis of RNA degradation by archaeal/eukaryotic exosomes and the bacterial/organelle PNases. In addition, it raises the possibility

© 2006 The Authors  
Journal compilation © 2006 Blackwell Publishing Ltd, *Molecular Microbiology*, **61**, 1372–1379
that the mechanism of 3′ RNA decay by phosphorolytic degradation complexes is more or less conserved across the kingdoms of life.

### Sequence-unspecific RNA degradation and nuclease mechanism

The phosphorolytic exonuclease active sites of the archaeal exosome are situated in an internal ‘processing chamber’ of the aRrp41:aRrp42 ring. RNase-PH type RNases act by adding inorganic phosphate across the 5′-3′ phosphoester bond, releasing a nucleotide diphosphate ([pN]n + Pi→[pN]n−1 + NDP). Structure-derived mutagenesis studies and crystallographic visualization of RNA molecules and phosphate mimics showed that the archaeal exosome contains three phosphorolytic active sites near the interface of aRrp41 and aRrp42 (Buttner et al., 2005; Lorentzen et al., 2005; Lorentzen and Conti, 2005). The RNA substrates are recognized mainly via the phosphate and sugar moieties and bound by a set of arginine side chains at the interface of aRrp41 and aRrp42 (Lorentzen and Conti, 2005) (Fig. 1). The discrimination of RNA versus DNA occurs by interaction with the ribose moieties of the first and fourth nucleotides (termed N1 and N4). Cleavage of the phosphodiester bond between the ultimate and penultimate nucleotide is in part catalysed by positioning an inorganic phosphate in close proximity to the phosphate moiety of the 3′ end nucleotide (N1). The lack of specific interaction with the nucleotide bases is consistent with a sequence unspecific degradation activity.

In archaeal exosomes, the RNA has a choice of three active sites (Fig. 1). Such a high concentration of active sites might favour the high processivity of these enzymes. At present, it is not clear whether RNA is processively degraded in one active site, or alternates between active sites. These alternatives might be important, because it is not clear whether eukaryotic exosomes also contain three phosphorolytic active sites. Eukaryotic RNase-PH subunits can be grouped into aRrp41-like (Rrp41, Mtr3, Rrp46) and aRrp42-like (Rrp42, Rrp43, Rrp45) subunits on the basis of sequence comparison, suggesting that eukaryotic exosomes are also formed by alternating Rrp41-like and Rrp42-like subunits (Koonin et al., 2001; Lorentzen et al., 2005). However, the precise arrangement of eukaryotic subunits is currently unclear. The phosphorolytic active site residues of the archaeal exosome are conserved in RNase-PH and the first RNase-PH domain of PNPs, but, surprisingly, are not conserved in eukaryotic exosome subunits (Lorentzen and Conti, 2005). At present, only recombinant eukaryotic Rrp41 has been shown to display phosphorolytic activity in vitro (Mitchell et al., 1997; Chekanova et al., 2000). In addition to the RNase-PH-like active sites, eukaryotic

exosomes contain other hydrolytic ribonucleolytic Rrp44 and Rrp6 subunits. Eukaryotic Rrp4 was reported to have nuclease activity (Mitchell et al., 1997; Chekanova et al., 2002; Buttner et al., 2005), although the nature of this activity is unclear and not evident from the biochemical features and structure of archaeal Rrp4 (Mitchell et al., 1997; Chekanova et al., 2002; Buttner et al., 2005).

Thus, one of the key open questions in eukaryotes is the nature and number of different copies of the phosphorolytic nuclease in the eukaryotic exosome and its mechanistic interplay with the additional hydrolytic nucleases. For instance, do these enzymes act cooperatively on the same RNA molecules, or are they required to degrade different types of RNA? Insights into the various nuclease activities of the eukaryotic exosome will require detailed mutagenesis and biochemical and/or structural analysis.

#### RNA recruitment and regulation of substrate entry into the exosome

If the RNA is degraded in a sequence-unspecific manner, how does the exosome prevent uncontrolled RNA decay? A large putative RNA binding platform, formed by Rrp4 and Csl4 subunits, spreads over the ‘top’ side of the archaeal exosome (Fig. 1). In yeast Csl4, a separation of function mutation (Ski4-1) affected mRNA decay but not rRNA processing (van Hoof et al., 2000b). This might point to a role of these subunits in RNA substrate recognition and distinction. In support of this, both S1 and KH domains of eubacterial PNPase are required for efficient RNA degradation (Stickney et al., 2005) and the S1 domain of PNPase contains a high affinity poly(A) binding site (Yehudai-Resheff et al., 2003). Small angle X-ray scattering studies of the Pyrococcus furiosus exosome suggest that the S1 and KH domains are quite dynamic in solution, raising the possibility that S1 and KH domains act together in substrate recognition (Ramos et al., 2006).

But how does the RNA get to the active sites if it is bound at the S1 and/or KH domains? The structure of the archaeal exosome showed that the RNase-PH type active sites are buried in a cage-like structure (Fig. 1 Buttner et al., 2005; Lorentzen and Conti, 2005). A pore connects the putative RNA binding S1 and KH domains with the central processing chamber. This pore is only 8–10 Å wide, a feature that suggests that only fully unfolded RNA can enter the processing chamber. As RNA molecules in a cellular environment are typically bound to proteins or form secondary structures, this narrow entry pore could be important to prevent uncontrolled decay of cellular RNA. In support of this, the yeast exosome is more or less inactive on long RNA substrates in vitro and requires coactivators (see below). Degradation of an artificial RNA

© 2006 The Authors  
Journal compilation © 2006 Blackwell Publishing Ltd, *Molecular Microbiology*, **61**, 1372–1379
substrate by the archaeal exosome stalls eight to nine nucleotides before an engineered stem loop structure (Lorentzen and Conti, 2005), which is the right distance to bridge the surface outside of the entry pore and the active sites. Thus, the architecture of the exosome provides a first basis for understanding controlled RNA degradation. At the same time, this restriction implies the presence of additional cellular factors that supply the exosome with suitable substrate RNA.

**Coactivator complexes**

Several coactivators of the exosome have been identified in Saccharomyces. The cytoplasmic activity of the yeast exosome depends on the presence of the SKI complex *in vivo* (Anderson and Parker, 1998; Brown *et al.*, 2000; Araki *et al.*, 2001). Mutations in SKI2, SKI3 and SKI8 inhibit 3′→5′ decay of mRNA and the yeast SKI complex interacts with the exosome via Ski7p, a GTP-binding protein (van Hoof *et al.*, 2000b; Araki *et al.*, 2001). The degradation of several nuclear RNAs requires the Trf4p-Air1/2p-Mtr4p and/or Trf5p-Air1p-Mtr4p complexes (TRAMP) (Haracska *et al.*, 2005; LaCava *et al.*, 2005; Vanacova *et al.*, 2005; Wyers *et al.*, 2005; Houseley and Tollervey, 2006). TRAMP stimulates processive RNA degradation of the yeast core exosome *in vitro* and contains adenylatation (Trf4/5) and putative RNA binding activities(Air2), as well as a putative RNA helicase homologue (Mtr4). But what could be the function of these coactivators? Ski2p and Mtr4p, members of the SKI and TRAMP complex, are superfamily II helicases. Thus, these coactivator complexes could unwind secondary structures of mRNA before and/or during degradation by the core exosome. It will therefore be important to test if these proteins directly provide unfolded RNA for degradation by the exosome. These coactivator complexes could also specifically target RNA molecules and recruit them to the exosome. For instance, Ski7p is required to degrade mRNAs that do not contain a stop codon and is sequence related to translation factors eRF3 (van Hoof *et al.*, 2002). Since Ski7p binds to the exosome, recognition of both the stalled ribosome and the exosome would be an elegant mechanism to target mRNA directly for decay. Along this line, recent studies indicate that Nrd1, complexed with Nab3/Sen1/CBP, can target the nuclear exosome to (pre) mRNA and that Nrd1 activates the exosome exonuclease activity *in vitro* (Arigo *et al.*, 2006; Houalla *et al.*, 2006; Vasiljeva and Buratowski, 2006).

The possible presence of coactivators needs to be investigated in archaea. Two archaeal exosome complexes copurified with an additional protein that shows sequence homology to eubacterial DNA primase DnaG (Table 1) (Evguenieva-Hackenberg *et al.*, 2003; Farhoud

et al., 2005). The function of this eubacterial DnaG primase homologue (archaea also possess homologues of the eukaryotic primase) and its possible contribution to the function of the archaeal exosome are unclear. Although archaea contain a Ski2 helicase homologue (Koonin *et al.*, 2001), it probably acts on replication forks rather than on RNA (Guy and Bolt, 2005). Thus, it will be important to evaluate further the coactivating factors in archaea.

**Macromolecular cages for controlled degradation of biopolymers**

The requirement of processive and sequence-independent, yet controlled and regulated degradation of biopolymers is not unique to RNA. In fact, a similar requirement can be formulated for proteins. Proteins are degraded in eukaryotes and archaea by the proteasome, or in eubacteria by the related HslU and ClpP proteases. The proteasome, HslU and ClpP are large multisubunit assemblies in which the proteolytic active sites are situated inside a large processing chamber (Bochtler *et al.*, 1999). The sites of proteolytic activities are accessible from two entry pores that are too narrow for folded proteins to enter (Fig. 2). ATP-dependent regulatory 19S complexes of the 20S proteasome bind to the outside of these entry pores, forming the 26S proteasome. These regulatory caps use ATP-hydrolysis to unfold and feed polypeptides into the processing chambers (Voges *et al.*, 1999). In the case of the eukaryotic proteasome, poly-ubiquitylated proteins are specifically recognized by the 19S regulatory subunits, leading to degradation of these marked substrate proteins.

Although the mechanism and regulation of RNA degradation by the exosome are less well understood than the regulation of protein degradation by the proteasome, strikingly parallel principles seem to emerge (Fig. 2). In fact, the architectural similarity of exosomes and proteasomes were suggested early on (van Hoof and Parker, 1999) and observed and highlighted recently (Buttner *et al.*, 2005; Lorentzen and Conti, 2005; Lorentzen and Conti, 2006). First, a major regulatory mechanism seems to be the limitation of access to unfolded substrates, which prevents uncontrolled substrate degradation. Second, biopolymer degradation occurs in a chamber that is shielded from the outside and contains a multitude of active sites, at least in the case of the archaeal exosome. This multiplicity could be important for processivity, due to a high local concentration of active sites in conjunction with limited dissociation/escape possibilities for the substrate. Third, access and/or targeting of substrates is controlled by regulatory subunits/coactivators and possibly by energy consumption. Thus, although there is no structural equivalency between exosome and proteasome

The exosome: a macromolecular cage 1377

Fig. 2. Comparison of the proposed RNA degradation by exosomes with protein degradation by proteasomes. Both enzyme assemblies degrade biopolymers in a protected internal chamber, accessible only through narrow entry pores. Substrates probably need to be unfolded using energy by coactivator/regulatory caps for processive degradation. AS, active sites; S1, S1 domain.

components, nature came up with surprisingly similar principles in the controlled degradation of RNA and polypeptide biopolymers.

**Future directions and outlook**

While the structural data suggest an interesting analogy between controlled RNA and protein degradation by self-compartmentalized (i.e. the active sites are situated in a proteinaceous nanocompartment formed by the enzyme assembly) exonucleases and proteases, many exciting questions need to be addressed. For instance, what is the role of S1, KH and ZnR domains in RNA recognition? What is the functional and biochemical difference between Rrp4, Rrp40 and Csl4 type RNA binding subunits? One possibility is that these subunits recognize different types of RNA molecules (e.g. mRNA or rRNA). Is there a mechanistic interplay between the RNase-PH active sites in the exosome centre and additional associated RNase subunits? How does the exosome communicate with coactivator complexes such as the nuclear TRAMP complexes and the cytoplasmic SKI complex? What is the biochemical basis for the activation of the exosome by these coactivators? These complexes could provide and target unfolded RNA for the core exosome or might modulate the activity of the core exosomes by conformational control. Finally, it remains unclear if and how substrate RNA is specifically targeted for degradation by the exosome. In other words, is there a functional equivalent to the targeting of proteins by poly-ubiquitylation?

One possibility for a general ‘degradation signal’ is a 3′ adenylated tail, observed on many exosome substrates. The nuclear TRAMP complexes polyadenylate nuclear RNA prior to degradation by the exosome, and *rrp6Δ* yeast strains accumulate adenylated small nuclear RNAs and rRNA (Kuai et al., 2004; LaCava et al., 2005). Some cotranscriptionally cleaved (pre) mRNAs also contain short adenylated ends that become more abundant upon PM/Scl-100 depletion (West et al., 2006). Oligo(A) is a good substrate for the archaeal exosome in vitro, without the necessity for coactivators (Buttner et al., 2005; Lorentzen et al., 2005). Interestingly, while archaea do not possess homologues of known adenyl-ation factors, the *S. solfataricus* exosome itself catalyses the polyadenylation of RNA in vivo (Portnoy et al., 2005) and the archaeal exosome can processively polyadenyl-ate RNA in vitro (Lorentzen et al., 2005). In eubacteria, PNPase can polyadenylate RNA (Grunberg-Manago and Ochoa, 1955) and 3′ adenylated tails are a general deg-radation signal (Dreyfus and Regnier, 2002). Thus, in the light of this emerging evolutionary conserved role of aden-ylation in RNA decay, it will be important to test a more direct role of adenylated 3′ tails in the degradation of RNA by the eukaryotic exosome. The biochemical function for such adenylate tails in degradation of RNA by the exo-somes is currently unclear. For instance, it could be specifically recognized by exosome subunits and/or provide a conveniently unfolded 3′ end for entry into the processing chamber. This and the many other questions lead us to anticipate an exciting future for exosome research

© 2006 The Authors  
Journal compilation © 2006 Blackwell Publishing Ltd, *Molecular Microbiology*, **61**, 1372–1379

1378 K. Büttner, K. Wenig and K.-P. Hopfner

through combined genetics, cell biology, biochemistry and structural biology approaches.

Acknowledgements

We apologize to all colleagues whose key contributions could not be cited due to space limitations and focus. We thank Annette Karcher for suggestions on the manuscript. Work in K.P.H.’s laboratory is supported by the German Research Council (DFG).

References

Allmang, C., Kufel, J., Chanfreau, G., Mitchell, P., Petfalski, E., and Tollervey, D. (1999a) Functions of the exosome in rRNA, snoRNA and snRNA synthesis. EMBO J 18: 5399–5410.

Allmang, C., Petfalski, E., Podtelejnikov, A., Mann, M., Tollervey, D., and Mitchell, P. (1999b) The yeast exosome and human PM-Scl are related complexes of 3′→5′ exonucleases. Genes Dev 13: 2148–2158.

Aloy, P., Ciccarelli, F.D., Leutwein, C., Gavin, A.C., Superti-Furga, G., Bork, P., et al. (2002) A complex prediction: three-dimensional model of the yeast exosome. EMBO Rep 3: 628–635.

Anderson, J.S., and Parker, R.P. (1998) The 3′ to 5′ degradation of yeast mRNAs is a general mechanism for mRNA turnover that requires the SKI2 DEVH box protein and 3′ to 5′ exonucleases of the exosome complex. EMBO J 17: 1497–1506.

Andrulis, E.D., Werner, J., Nazarian, A., Erdjument-Bromage, H., Tempst, P., and Lis, J.T. (2002) The RNA processing exosome is linked to elongating RNA polymerase II in Drosophila. Nature 420: 837–841.

Araki, Y., Takahashi, S., Kobayashi, T., Kajiho, H., Hoshino, S., and Katada, T. (2001) Ski7p G protein interacts with the exosome and the Ski complex for 3′ to 5′ mRNA decay in yeast. EMBO J 20: 4684–4693.

Arigo, J.T., Carroll, K.L., Ames, J.M., and Corden, J.L. (2006) Regulation of yeast NRD1 expression by premature transcription termination. Mol Cell 21: 641–651.

Bochtler, M., Ditzel, L., Groll, M., Hartmann, C., and Huber, R. (1999) The proteasome. Annu Rev Biophys Biomol Struct 28: 295–317.

Brown, J.T., Bai, X., and Johnson, A.W. (2000) The yeast antiviral proteins Ski2p, Ski3p, and Ski8p exist as a complex in vivo. RNA 6: 449–457.

Burkard, K.T., and Butler, J.S. (2000) A nuclear 3′-5′ exonuclease involved in mRNA degradation interacts with Poly (A) polymerase and the hnRNA protein Npl3p. Mol Cell Biol 20: 604–616.

Buttner, K., Wenig, K., and Hopfner, K.P. (2005) Structural framework for the mechanism of archaeal exosomes in RNA processing. Mol Cell 20: 461–471.

Chekanova, J.A., Shaw, R.J., Wills, M.A., and Belostotsky, D.A. (2000) Poly (A) tail-dependent exonuclease AtRrp41p from Arabidopsis thaliana rescues 5.8 S rRNA processing and mRNA decay defects of the yeast ski6 mutant and is found in an exosome-sized complex in plant and yeast cells. J Biol Chem 275: 33158–33166.

Chekanova, J.A., Dutko, J.A., Mian, I.S., and Belostotsky, D.A. (2002) Arabidopsis thaliana exosome subunit AtRrp4p is a hydrolytic 3′→5′ exonuclease containing S1 and KH RNA-binding domains. Nucleic Acids Res 30: 695–700.

de la Cruz, J., Kressler, D., Tollervey, D., and Linder, P. (1998) Dob1p (Mtr4p) is a putative ATP-dependent RNA helicase required for the 3′ end formation of 5.8S rRNA in Saccharomyces cerevisiae. EMBO J 17: 1128–1140.

Dreyfus, M., and Regnier, P. (2002) The poly (A) tail of mRNAs: bodyguard in eukaryotes, scavenger in bacteria. Cell 111: 611–613.

Estevez, A.M., Kempf, T., and Clayton, C. (2001) The exosome of Trypanosoma brucei. EMBO J 20: 3831–3839.

Evguenieva-Hackenberg, E., Walter, P., Hochleitner, E., Lottspeich, F., and Klug, G. (2003) An exosome-like complex in Sulfolobus solfataricus. EMBO Rep 4: 889–893.

Farhoud, M.H., Wessels, H.J., Steenbakkers, P.J., Mattijssen, S., Wevers, R.A., van Engelen, B.G., et al. (2005) Protein complexes in the archaeon Methanothermobacter thermautotrophicus analyzed by blue native/SDS-PAGE and mass spectrometry. Mol Cell Proteomics 4: 1653–1663.

Goodchild, A., Rafferty, M., Saunders, N.F., Guilhaus, M., and Cavicchioli, R. (2004) Biology of the cold adapted archaeon, Methanococcoides burtonii determined by proteomics using liquid chromatography-tandem mass spectrometry. J Proteome Res 3: 1164–1176.

Grunberg-Manago, M., and Ochoa, S. (1955) Enzymatic synthesis and breakdown of polynucleotides; polynucleotide phosphorylase. J Am Chem Soc 77: 3165–3166.

Guy, C.P., and Bolt, E.L. (2005) Archaeal Hel308 helicase targets replication forks in vivo and in vitro and unwinds lagging strands. Nucleic Acids Res 33: 3678–3690.

Haracska, L., Johnson, R.E., Prakash, L., and Prakash, S. (2005) Trf4 and Trf5 proteins of Saccharomyces cerevisiae exhibit poly (A) RNA polymerase activity but no DNA polymerase activity. Mol Cell Biol 25: 10183–10189.

Hilleren, P., McCarthy, T., Rosbash, M., Parker, R., and Jensen, T.H. (2001) Quality control of mRNA 3′-end processing is linked to the nuclear exosome. Nature 413: 538–542.

van Hoof, A., and Parker, R. (1999) The exosome: a proteasome for RNA? Cell 99: 347–350.

van Hoof, A., Lennertz, P., and Parker, R. (2000a) Yeast exosome mutants accumulate 3′-extended polyadenylated forms of U4 small nuclear RNA and small nucleolar RNAs. Mol Cell Biol 20: 441–452.

van Hoof, A., Staples, R.R., Baker, R.E., and Parker, R. (2000b) Function of the ski4p (Csl4p) and Ski7p proteins in 3′-to-5′ degradation of mRNA. Mol Cell Biol 20: 8230–8243.

van Hoof, A., Frischmeyer, P.A., Dietz, H.C., and Parker, R. (2002) Exosome-mediated recognition and degradation of mRNAs lacking a termination codon. Science 295: 2262–2264.

Houalla, R., Devaux, F., Fatica, A., Kufel, J., Barrass, D., Torchet, C., and Tollervey, D. (2006) Microarray detection of novel nuclear RNA substrates for the exosome. Yeast 23: 439–454.

Houseley, J., and Tollervey, D. (2006) Yeast Trf5p is a nuclear poly (A) polymerase. *EMBO Rep* 7: 205–211.

Kadaba, S., Krueger, A., Trice, T., Krecic, A.M., Hinnebusch, A.G., and Anderson, J. (2004) Nuclear surveillance and degradation of hypomodified initiator tRNAMet in S. cerevisiae. *Genes Dev* 18: 1227–1240.

Koonin, E.V., Wolf, Y.I., and Aravind, L. (2001) Prediction of the archaeal exosome and its connections with the proteasome and the translation and transcription machineries by a comparative-genomic approach. *Genome Res* 11: 240–252.

Kuai, L., Fang, F., Butler, J.S., and Sherman, F. (2004) Polyadenylation of rRNA in Saccharomyces cerevisiae. *Proc Natl Acad Sci USA* 101: 8581–8586.

LaCava, J., Houseley, J., Saveanu, C., Petfalski, E., Thompson, E., Jacquier, A., and Tollervey, D. (2005) RNA degradation by the exosome is promoted by a nuclear polyadenylation complex. *Cell* 121: 713–724.

Lorentzen, E., and Conti, E. (2005) Structural basis of 3′ end RNA recognition and exoribonucleolytic cleavage by an exosome RNase PH core. *Mol Cell* 20: 473–481.

Lorentzen, E., and Conti, E. (2006) The exosome and the proteasome: nano-compartments for degradation. *Cell* 125: 651–654.

Lorentzen, E., Walter, P., Fribourg, S., Evguenievna-Hackenberg, E., Klug, G., and Conti, E. (2005) The archaeal exosome core is a hexameric ring structure with three catalytic subunits. *Nat Struct Mol Biol* 12: 575–581.

Milligan, L., Torchet, C., Allmang, C., Shipman, T., and Tollervey, D. (2005) A nuclear surveillance pathway for mRNAs with defective polyadenylation. *Mol Cell Biol* 25: 9996–10004.

Mitchell, P., and Tollervey, D. (2001) mRNA turnover. *Curr Opin Cell Biol* 13: 320–325.

Mitchell, P., and Tollervey, D. (2003) An NMD pathway in yeast involving accelerated deadenylation and exosome-mediated 3′→5′ degradation. *Mol Cell* 11: 1405–1413.

Mitchell, P., Petfalski, E., Shevchenko, A., Mann, M., and Tollervey, D. (1997) The exosome: a conserved eukaryotic RNA processing complex containing multiple 3′→5′ exoribonucleases. *Cell* 91: 457–466.

Orban, T.I., and Izaurralde, E. (2005) Decay of mRNAs targeted by RISC requires XRN1, the Ski complex, and the exosome. *RNA* 11: 459–469.

Portnoy, V., Evguenievna-Hackenberg, E., Klein, F., Walter, P., Lorentzen, E., Klug, G., and Schuster, G. (2005) RNA polyadenylation in Archaea: not observed in Haloferax while the exosome polynucleotidylates RNA in Sulfolobus. *EMBO Rep* 6: 1188–1193.

Rajmakers, R., Egberts, W.V., van Venrooij, W.J., and Pruijn, G.J. (2002) Protein–protein interactions between human exosome components support the assembly of RNase

PH-type subunits into a six-membered PNPase-like ring. *J Mol Biol* 323: 653–663.

Rajmakers, R., Schilders, G., and Pruijn, G.J. (2004) The exosome, a molecular machine for controlled RNA degradation in both nucleus and cytoplasm. *Eur J Cell Biol* 83: 175–183.

Ramos, C.R., Oliveira, C.L., Torriani, I.L., and Oliveira, C.C. (2006) The Pyrococcus exosome complex: structural and functional characterization. *J Biol Chem* 281: 6751–6759.

Stickney, L.M., Hankins, J.S., Miao, X., and Mackie, G.A. (2005) Function of the conserved S1 and KH domains in polynucleotide phosphorylase. *J Bacteriol* 187: 7214–7221.

Suzuki, N., Noguchi, E., Nakashima, N., Oki, M., Ohba, T., Tartakoff, A., et al. (2001) The Saccharomyces cerevisiae small GTPase, Gsp1p/Ran, is involved in 3′ processing of 7S-to-5.8S rRNA and in degradation of the excised 5′-A0 fragment of 35S pre-rRNA, both of which are carried out by the exosome. *Genetics* 158: 613–625.

Symmons, M.F., Jones, G.H., and Luisi, B.F. (2000) A duplicated fold is the structural basis for polynucleotide phosphorylase catalytic activity, processivity, and regulation. *Struct Fold Des* 8: 1215–1226.

Symmons, M.F., Williams, M.G., Luisi, B.F., Jones, G.H., and Carpousis, A.J. (2002) Running rings around RNA: a superfamily of phosphate-dependent RNases. *Trends Biochem Sci* 27: 11–18.

Torchet, C., Bousquet-Antonelli, C., Milligan, L., Thompson, E., Kufel, J., and Tollervey, D. (2002) Processing of 3′-extended read-through transcripts by the exosome can generate functional mRNAs. *Mol Cell* 9: 1285–1296.

Vanacova, S., Wolf, J., Martin, G., Blank, D., Dettwiler, S., Friedlein, A., et al. (2005) A new yeast poly (A) polymerase complex involved in RNA quality control. *PLoS Biol* 3: e189.

Vasiljeva, L., and Buratowski, S. (2006) Nrd1 interacts with the nuclear exosome for 3′ processing of RNA polymerase II transcripts. *Mol Cell* 21: 239–248.

Voges, D., Zwickl, P., and Baumeister, W. (1999) The 26S proteasome: a molecular machine designed for controlled proteolysis. *Annu Rev Biochem* 68: 1015–1068.

West, S., Gromak, N., Norbury, C.J., and Proudfoot, N.J. (2006) Adenylation and exosome-mediated degradation of cotranscriptionally cleaved pre-messenger RNA in human cells. *Mol Cell* 21: 437–443.

Wyers, F., Rougemaille, M., Badis, G., Rousselle, J.C., Dufour, M.E., Boulay, J., et al. (2005) Cryptic pol II transcripts are degraded by a nuclear quality control pathway involving a new poly (A) polymerase. *Cell* 121: 725–737.

Yehudai-Resheff, S., Portnoy, V., Yogev, S., Adir, N., and Schuster, G. (2003) Domain analysis of the chloroplast polynucleotide phosphorylase reveals discrete functions in RNA degradation, polyadenylation, and sequence homology with exosome proteins. *Plant Cell* 15: 2003–2019.
