
ARTICLES

FOXO4 transcriptional activity is regulated by monoubiquitination and USP7/HAUSP

Armando van der Horst¹, Alida M.M. de Vries-Smits¹, Arjan B. Brenkman¹, Miranda H. van Triest¹, Niels van den Broek¹, Frédéric Colland², Madelon M. Maurice³ and Boudewijn M.T. Burgering¹,⁴

FOXO (Forkhead box O) transcription factors are important regulators of cellular metabolism, cell-cycle progression and cell death. FOXO activity is regulated by multiple post-translational modifications, including phosphorylation, acetylation and polyubiquitination. Here, we show that FOXO becomes monoubiquitinated in response to increased cellular oxidative stress, resulting in its re-localization to the nucleus and an increase in its transcriptional activity. Deubiquitination of FOXO requires the deubiquitinating enzyme USP7/HAUSP (herpesvirus-associated ubiquitin-specific protease), which interacts with and deubiquitates FOXO in response to oxidative stress. Oxidative stress-induced ubiquitination and deubiquitination by USP7 do not influence FOXO protein half-life. However, USP7 does negatively regulate FOXO transcriptional activity towards endogenous promoters. Our results demonstrate a novel mechanism of FOXO regulation and indicate that USP7 has an important role in regulating FOXO-mediated stress responses.

The FOXO subfamily of forkhead transcription factors consists of four members — FOXO1, FOXO3, FOXO4 and FOXO6 (ref. 1). These proteins are involved in a number of important biological processes, such as control of cellular metabolism, cell-cycle regulation, apoptosis and regulation of the stress response²,³ (reviewed in ref. 4). Initially, it was shown that the FOXO factors are regulated by protein kinase B (PKB/c-Akt)-mediated phosphorylation and that this phosphorylation results in their nuclear exclusion⁵,⁶. Regulation of FOXO by PKB is conserved through evolution as DAF-16, the FOXO homologue in the nematode *Caenorhabditis elegans*, is regulated in a similar manner. Recently, other signalling pathways have been described to regulate FOXO (reviewed in ref. 7), but it remains to be established how conserved these non-PKB regulatory pathways are. More recently, we showed that a pathway that is sensitive to cellular oxidative stress, involving the small GTPase Ral and the stress kinase JNK, positively regulates the transcriptional activity of FOXO4 by phosphorylation⁸,⁹. Interestingly, like PKB signalling towards FOXO, regulation by JNK¹⁰,¹¹ and possibly Ral¹² seems to be conserved throughout evolution.

As well as phosphorylation, FOXO transcription factors can also be post-translationally regulated by acetylation and polyubiquitination. FOXO acetylation occurs by p300, CBP (CREB-binding protein) and P/CAF (p300/CBP-associated factor) and deacetylation by histone deacetylases and Sir2 (silent information regulator 2). However, the consequence of deacetylation on FOXO activity is still a matter of debate¹³–¹⁸. Polyubiquitination of FOXOs by ubiquitin chains that are linked by Lys 48 of ubiquitin can be induced by growth-factor signalling and this is probably mediated by Skp2 (refs 19–22), which is an F-box protein that is present in so-called SCF-complexes (Skp1, Cullin-1, F-box protein) of ubiquitin ligases. However, FOXOs are relatively stable proteins (with a half-life of approximately 9–10 h), even when the cells are cultured in the presence of serum. Thus, a role for Lys 48-mediated polyubiquitination on FOXO in normal signalling is, at present, unclear²⁰,²¹.

**RESULTS**

Hydrogen peroxide induces monoubiquitination of FOXO4

We set out to study the ubiquitination of FOXO in more detail. To this end, HA–FOXO4 and His–ubiquitin were coexpressed in HEK293T cells and cells were left untreated or treated with hydrogen peroxide, a known substrate-dependent inducer of mono- and/or polyubiquitination²³,²⁴. Subsequently, cells were lysed in a denaturing buffer containing 8 M urea to remove any ubiquitinated proteins bound to HA–FOXO, and ubiquitinated proteins were precipitated using Ni–NTA beads²⁵. Western blot detection using an anti-HA antibody indicated that FOXO4 was ubiquitinated (Fig. 1a, left panel). To verify that the bands recognized by the anti-HA antibody represent ubiquitinated HA–FOXO4, we performed the same experiment, but probed with anti-FOXO4 instead of anti-HA antibody (Fig. 1a, right panel). Both approaches resulted in the same pattern of ubiquitinated bands, indicating that these indeed represent ubiquitinated HA–FOXO4. Notably, we observed two prominent bands,

---

¹ Department of Physiological Chemistry, Centre for Biomedical Genetics, University Medical Center Utrecht, 3584 CG Utrecht, The Netherlands.  
² Hybrigenics SA, 3-5 Impasse Reille, 75014 Paris, France.  
³ Department of Cell Biology, University Medical Center Utrecht, 3584 CX Utrecht, The Netherlands.  
⁴ Correspondence should be addressed to B.M.T.B. (e-mail: B.M.T.Burgering@med.uu.nl)

Received 20 April 2006; accepted 21 June 2006; published online 10 September 2006; DOI: 10.1038/ncb1469

1064 | NATURE CELL BIOLOGY | VOLUME 8 | NUMBER 10 | OCTOBER 2006

©2006 Nature Publishing Group

a
Pulldown: Ni–NTA
WB: anti-HA

Ub-FOXO4→

FOXO4→

His-ubi:
HA-FOXO4:
H₂O₂:

b
Ni–NTA
anti-FOXO4
M<sub>r</sub>(K)
117
92
66

Ub-FOXO4:
2×mono⇒
mono⇒

FOXO4→

His-ubi:
HA-FOXO4:
H₂O₂:

c
Pulldown: Ni–NTA
WB: anti-HA
M<sub>r</sub>(K)
117
92
66

Ub-FOXO4→

HA-FOXO4: + + + +
TSA+Nam: - + + +
H₂O₂: - + - +

d
Pulldown: Ni–NTA
WB: anti-HA M<sub>r</sub>(K)
117
92
66

Ub-FOXO4→

HA-FOXO4: WTWT2R2R
H₂O₂: - + - +

e
Pulldown: Ni–NTA
WB: anti-HA

Ub-FOXO4→

FOXO4→

His-ubi:
HA-FOXO4:
H₂O₂ (µM):
Time (min):

f
WB: anti-His
WB: anti-FOXO4

IP: ur FOXO4
ur FOXO4

His-ubi: + + + + +
H₂O₂ (min): 30 - 15 30 60
30 - 15 30 60


Figure 1 Hydrogen peroxide induces monoubiquitination of FOXO4.
(a) Ubiquitinated proteins were precipitated using Ni–NTA agarose from HEK293T cells transfected with HA–FOXO4 and wild-type His-tagged ubiquitin after treatment with 200 µM hydrogen peroxide for 30 min. Western blots (WBs) were analysed for the presence of HA–FOXO4, using anti-HA (left) and anti-FOXO4 (right) antibodies.
(b) The experiment was performed as in a, left panel, using both wild-type and mutant ubiquitin in which all seven lysine residues have been mutated to arginines. Note that His–ubiquitin-K7R migrates faster than wild-type His–ubiquitin.
(c) As in a, cells were pretreated with TSA plus Nam for 90 min before the treatment with 200 µM hydrogen peroxide for 30 min. Western blots were analysed for the presence of HA–FOXO4 using anti-HA antibody.
(d) Ubiquitinated proteins were precipitated from HEK293T cells that had been transfected with HA–FOXO4 or HA–FOXO4-K199,211R plus wild-type His–ubiquitin and treated with 200 µM hydrogen peroxide for 30 min. Western blots were analysed for the presence of HA–FOXO4 using anti-HA antibody.
(e) As in a; cells were treated with hydrogen peroxide at the indicated concentrations and for the indicated times. Western blots were analysed for the presence of HA–FOXO4 using anti-HA antibody.
(f) Endogenous FOXO4 was immunoprecipitated from HEK293T cells that had been transfected with wild-type His–ubiquitin; an unrelated (ur) antibody was used as a control. Samples were analysed for the presence of monoubiquitinated FOXO4 using anti-His (left) and anti-FOXO4 (right) antibodies. Full scans of the blots shown in a and f can be found in the Supplementary Information, Fig. S5.

of which the relative molecular mass of the lowest band corresponded to the molecular mass of FOXO4 plus ~8,000, indicating that FOXO4 can be monoubiquitinated at one or multiple sites. To further validate the finding that FOXO4 is monoubiquitinated, we used a His–ubiquitin construct in which all seven lysine residues of ubiquitin were mutated to arginines (K7R). Ni–NTA pulldown using this modified His–ubiquitin resulted in a similar pattern of FOXO4 ubiquitination (Fig. 1b), supporting our conclusion that FOXO4 can be monoubiquitinated at multiple sites.

Ubiquitination, as well as acetylation, occurs on lysine residues. Interestingly, in the case of FOXO, both ubiquitination and acetylation are induced by oxidative stress (this study and refs 15, 16, 26). Previously, we showed that treatment of cells with inhibitors of deacetylases (trichostatin A, TSA; and nicotinamide, Nam) results in increased acetylation of FOXO4 (ref. 15). Therefore, we set out to determine whether TSA and Nam affect the monoubiquitination of FOXO4 as well. Indeed, we observed a reduction of hydrogen-peroxide-induced monoubiquitination of HA–FOXO4 by TSA and Nam (Fig. 1c). This indicates that, at least in part, the same lysine residues of HA–FOXO4 are used for acetylation and ubiquitination. Several acetylated residues have been described for the various FOXOs, some of which are conserved among all members, including Lys 199 and Lys 211 of FOXO4 (see, for a review, ref. 27). Indeed, an acetylation site mutant of lysine residues Lys 199 and

Lys 211 of FOXO4 displayed reduced monoubiquitination (Fig. 1d), albeit the level of reduction seemed to be variable between experiments.

Next, we examined the dose-response and kinetics of hydrogen-peroxide-induced monoubiquitination of FOXO4 (Fig. 1e). Monoubiquitination of HA–FOXO4 was induced by hydrogen peroxide in a concentration- and time-dependent manner and could be observed at concentrations as low as 50 µM and as early as after 5 min of treatment. Moreover, monoubiquitination of HA–FOXO4 is transient, with maximal induction occurring between 15 and 30 min (Fig. 1e, also see Fig. 4a). Importantly, endogenous FOXO4 was monoubiquitinated with similar kinetics (Fig. 1f). This is in sharp contrast to the growth-factor-induced polyubiquitination of FOXO, which was visible only after 12 h of insulin treatment (ref. 20, and data not shown). As well as hydrogen peroxide, we also tested several other general stressors (for example, anisomycin, tunicamycin and ultraviolet light). However, none of these treatments induced significant monoubiquitination of HA–FOXO4 compared with hydrogen peroxide (data not shown), indicating that the observed monoubiquitination of HA–FOXO4 specifically follows the generation of cellular oxidative stress produced by hydrogen peroxide. Furthermore, monoubiquitination of HA–FOXO3 was also induced by hydrogen peroxide, indicating that this may be a general regulatory mechanism for the O class of Forkhead transcription factors (data not shown).

ARTICLES

a

![Diagram](attachment:diagram_a.png)

b

| WT FOXO4 | Ubi-FOXO4 |
| --- | --- |
| Cy3 | Cy3 |
| DAPI | DAPI |

c

![Bar Chart](attachment:bar_chart_c.png)

Figure 2 Effects of monoubiquitination on FOXO4. (a) Ubiquitin-mediated fluorescence complementation of YN–ubiquitin (Ub) and YC–FOXO4 cotransfected in A14 cells. The principal of the method is shown schematically (top panel). Cells were fixed and analysed using a confocal microscope (bottom panel). Green, complemented YFP; blue, DAPI counterstaining to visualise nuclei. No background was observed when either construct was expressed alone (bottom left panel). We observed a 0.4:0.6 ratio between the number of cells displaying nuclear and perinuclear staining, respectively (bottom right panel). Scale bar, 10 μm. (b) A14 cells expressing wild-type (WT) FOXO4 or ubi–FOXO4 (a ubiquitin–FOXO4 fusion construct) were stained using anti-FOXO4 antibody. (Top panel) Staining was analysed using a confocal microscope.

Monoubiquitination of FOXO4 induces nuclear localization and increased transcriptional activity

Polyubiquitination of proteins generally results in proteasome-mediated protein degradation, whereas monoubiquitination may mediate other signalling functions, such as protein trafficking and DNA repair (reviewed in refs 28, 29). Previously, we and others have shown that the treatment of cells with hydrogen peroxide results in FOXO translocation to the nucleus⁹,¹⁶. Therefore, we examined the localization of ubiquitinated FOXO4 using two different approaches. First, we used ubiquitin-mediated fluorescence complementation, as described by Fang and Kerppola³⁰. A14 cells were transfected with the amino-terminal part

Red, anti-rabbit-Cy3; blue, DAPI counterstaining. (Bottom panel) Fifty cells per experiment were scored for FOXO4 localization. Representative examples for scoring are shown in Supplementary Information, Fig. S6. Data presented are the mean ± s.e.m. of two independent experiments. Scale bar, 10 μm. (c) HEK293T cells were transfected with a FOXO-responsive 6×DBE-luciferase plasmid, together with the indicated FOXO constructs. Cells were left untreated or were treated with insulin and epidermal growth factor for 16 h. Wild-type FOXO4 and ubi-FOXO4 expression levels in whole-cell extracts determined using an anti-FOXO4 antibody are shown below the corresponding bars. Representative data, expressed as 6×DBE-luciferase divided by Renilla luciferase counts, from three experiments are shown as mean ± s.d. of triplicates.

of YFP (aa 1–172) tagged to ubiquitin (YN–ubiquitin) and the carboxy-terminal portion of YFP (aa 159–238) attached to FOXO4 (YC–FOXO4). Whereas transfection of either construct did not result in a specific YFP signal (Fig. 2a, left), YFP complementation was observed either in the nucleus or in a perinuclear region following co-transfection of the constructs (Fig. 2a, top right and bottom right, respectively). Second, to further examine the role of monoubiquitination, we used an artificial fusion construct of ubiquitin fused to the N terminus of FOXO4 (ubi–FOXO4). Previously, a similar strategy revealed the role of epidermal growth factor (EGF)-receptor monoubiquitination in EGF-receptor trafficking³¹. Consistent with the YFP complementation observed within

a
USP7 | IP: IgG  
WB: anti-USP7  
USP7 | WB: USP7  
FOXO4 | IP: IgG  
WB: anti-FOXO4  
TAP–FOXO4: – +  

b
293T  
FOXO4  
$M_{r}$(K)  
117  
66  
Myc–USP7: – + +  
HA–FOXO: + + +  
H₂O₂: – – +  

FOXO3  
117  
92  
– + +  
+ + +  
– – +  

H1299  
FOXO4  
117  
66  
IP: anti-HA  
WB: anti-Myc  
IP: anti-HA  
WB: anti-HA  
– + +  
+ + +  
– – +  

c
Pulldown: Ni–NTA  
WB: anti-HA  
$M_{r}$(K)  
117  
92  
66  
Ub–FOXO4 →  
His–ubi: – – + + + +  
HA–FOXO4: + + + + + +  
Myc–USP7: – – – + +  
H₂O₂: – + – + +  

d
Pulldown: Ni–NTA  
WB: anti-HA  
$M_{r}$(K)  
117  
92  
66  
Ub–FOXO4 →  
His–ubi: + + + + +  
HA–FOXO4: – + + + +  
Myc–USP7: – – + +  
H₂O₂: + – + – +  

e
Pulldown: Ni–NTA  
WB: anti-HA  
HA–FOXO4: + + +  
H₂O₂: – + +  
USP7: – – +  

Figure 3 USP7 interacts with and deubiquitates FOXO4. (a) Nuclei from C2C12 cells stably expressing TAP–FOXO4 were purified and TAP–FOXO4 was immunoprecipitated. Immunoprecipitation (IP) samples were analysed by western blotting using anti-USP7 and anti-FOXO4 antibodies. (b) HA–FOXO4 or HA–FOXO3 was immunoprecipitated from HEK293T (left and middle panels) or H1299 (p53 null, right panel) cells treated with 200 μM hydrogen peroxide for 30 min. Western blots were probed for USP7 and the indicated FOXO protein. (c, d) Ubiquitinated proteins were precipitated the nucleus, we observed significantly increased nuclear localization of ubi–FOXO4 compared with wild-type FOXO4 (Fig. 2b). Taken together, these data indicate that monoubiquitination probably results in increased FOXO4 nuclear localization.

Increased nuclear localization of FOXO may result in increased FOXO transcriptional activity. Therefore, we tested the transcriptional activity of the ubi–FOXO4 fusion construct on a FOXO-responsive reporter construct bearing six canonical FOXO-binding elements (6×DBE-luc³²). Indeed, when corrected for expression levels, the ubi–FOXO4 construct showed approximately tenfold higher activity than wild-type FOXO4 (Fig. 2c). Notably, fusing moieties, such as GFP or a tandem affinity purification (TAP) tag, to the N terminus of FOXO4 does not change its activity (data not shown), indicating that the observed transcriptional increase is specific for the fused ubiquitin moiety. Furthermore, the ubi–FOXO4 fusion can still be regulated by PKB signalling (Fig. 2c), which is compatible with previous results that suggested competition between insulin and oxidative stress signalling with respect to the regulation of FOXO4 activity⁹,¹⁶.

As the HA–FOXO4-K199,211R mutant showed diminished monoubiquitination compared with wild-type FOXO4 following treatment of cells with hydrogen peroxide (see Fig. 1d), we determined the transcriptional activity of this mutant. The activity of FOXO4-K199,211R did not substantially differ from wild-type FOXO4 (data not shown). Consistent with this, the localization of the HA–FOXO4-K199,211R mutant was also similar to that of wild-type protein (see Supplementary Information,

from HEK293T (c) or H1299 (d) cells treated with hydrogen peroxide (as in Fig. 1a) in the absence or presence of wild-type USP7. Samples were analysed for HA–FOXO4 levels. (e) Monoubiquitinated FOXO4 was isolated from HEK293T cells treated with hydrogen peroxide for 15 min, as described for Fig. 1. Samples were dialysed against reaction buffer, purified FLAG–USP7 was added and reactions were incubated at 30 °C for 1 h. Reactions were terminated using 5× Laemmlil sample buffer and samples were analysed by western blotting.

Fig. S1). Previously, we and others showed that acetylation, in contrast to monoubiquitination, reduces FOXO4 transcriptional activity¹⁴–¹⁶,¹⁸,²⁶. We therefore conclude that the opposing effects of acetylation and monoubiquitination cancel each other out for this mutant.

USP7/HAUSP interacts with and deubiquitates FOXO4

Ubiquitination of proteins is counteracted by deubiquitinating enzymes (DUBs), which remove ubiquitin from proteins. An independent yeast two-hybrid screen with a C-terminal fragment (amino acids 884–1065) of the DUB USP7/HAUSP (herpesvirus-associated ubiquitin-specific protease) was performed using a highly complex placental cDNA library that has been previously described³³. This screen identified FOXO4 and, more precisely, the C-terminal part of FOXO4 (amino acids 360–501), as interacting with USP7. To confirm binding of USP7 to FOXO4, we first tagged FOXO4 with a TAP-tag, and isolated cell lines that were derived from the parental C2C12 line, which stably express TAP–FOXO4 at near-endogenous levels (A.B. Brenkman *et al.*, manuscript in preparation). Purification of TAP–FOXO4 from these cells showed binding of endogenous USP7 to TAP–FOXO4 (Fig. 3a). Second, we analysed the interaction between FOXO4 and USP7 in more detail using exogenously expressed proteins. Indeed, HA–FOXO4 and myc–USP7 interacted and this interaction was further enhanced by treatment of cells with hydrogen peroxide (Fig. 3b, left). Moreover, FOXO3 could also bind to USP7, indicating that, within the FOXO class of transcription factors, this interaction is not specific to FOXO4 (Fig. 3b, middle). Recently, USP7 has

ARTICLES

a  
Pulldown: Ni–NTA  
WB: anti-HA  

Ub–FOXO4 →  

HA–FOXO4: + – + + + + +  
His–ubi: – + + + + + +  
H₂O₂(min): – – 5 15 30 60 120  

b  
$M_{r}$(K)  
117 –  
117 –  
92 –  
66 –  

IP: anti-HA  
WB: anti-Myc  
WB: anti-Myc  
WB: anti-HA  

Myc–USP7: – + + + + + + +  
HA–FOXO4: – – + + + + + +  
H₂O₂(min): – – – 5 15 30 60 120  

c  
$M_{r}$(K)  
117 –  
117 –  
117 –  

IP anti-USP7  
IP anti-Myc  

WB: anti-HA  
WB: anti-USP7  
WB: anti-HA  

Myc–USP7: – – – WT WT WT CS CS CS  
HA–Ub-probe: – + + – + + + +  
H₂O₂(min): – – – 15 30 – – 15 30  

Figure 4 FOXO4 monoubiquitination precedes USP7 binding to FOXO4. (a) Ubiquitinated proteins were precipitated using Ni–NTA agarose from HEK293T cells that had been transfected with HA–FOXO4 and wild-type His-tagged ubiquitin after treatment with 200 μM hydrogen peroxide for the indicated times (as in Fig. 1a). Western blots were analysed for the presence of HA–FOXO4 using anti-HA antibody. (b) HA–FOXO4 was immunoprecipitated from HEK293T cells treated with 200 μM hydrogen peroxide for the indicated times. Western blots were probed for USP7 and FOXO4. (c) Either endogenous or exogenous USP7 was immunoprecipitated (IP) from HEK293T cells treated with 200 μM hydrogen peroxide for the indicated times. Immunoprecipitates were incubated with an HA–ubiquitin probe for 1 h at 37 °C. Active USP7 was visualized with anti-HA antibody (top panel), whereas the total amount of immunoprecipitated USP7 was detected with anti-USP7 antibody. The lanes to which probe was added clearly show a shift of the immunoprecipitated USP7 (middle panel). Bottom: §, active USP7 levels in total lysate (TL) samples; *, an unidentified DUB that is not sensitive to hydrogen peroxide; #, an unidentified DUB that is sensitive to hydrogen peroxide. Full scans of the blots shown in a and b can be found in the Supplementary Information, Fig. S5.

USP7 for 1 h³⁷. In addition, in this cell-free assay, USP7 removed the monoubiquitin moiety from FOXO4 (Fig. 3e), indicating that USP7 directly deubiquitates FOXO4.

The family of DUB proteins consists of approximately 60 members³⁸. It has been suggested that the DUB USP14 lacks activity towards polyubiquitin chains and, therefore, may be a specific DUB for monoubiquitinated proteins³⁹. Therefore, to test the specificity of USP7 as a DUB for monoubiquitinated FOXO4, we analysed the ability of HA–USP14 to deubiquitate FLAG–FOXO4 (see Supplementary Information, Fig. S2a). HA–USP14 displayed no activity towards FLAG–FOXO4 and, in agreement with this, we did not observe binding of HA–USP14 to FLAG–FOXO4 (see Supplementary Information, Fig. S2b), indicating that the interaction of FOXO4 with USP7 and the deubiquitination of FOXO4 by USP7 are specific.

Both monoubiquitination of FOXO and USP7 binding to FOXO, resulting in deubiquitination of FOXO, are induced by oxidative stress. To determine whether oxidative stress acts on FOXO through the inhibition of USP7 activity, we analysed the kinetics of monoubiquitination of FOXO4 and USP7 binding to FOXO4 following treatment of cells with hydrogen peroxide (Fig. 4a, b). Importantly, we observed that the kinetics

ARTICLES

Figure 5 USP7 regulates FOXO transcriptional activity. (a) Luciferase assay using HEK293T cells expressing a FOXO-responsive luciferase-reporter, Tk-Renilla, HA-FOXO4 and wild-type USP7 or USP7-CS. Representative data are shown as mean ± s.d. of triplicates (n = 4). Asterisk indicates P < 0.001 (Student's t-test). (b) Western blot analysis for p27<sup>Kip1</sup> expression of puromycin-selected HEK293T cells transfected with HA-FOXO4 and a non-targeting siRNA oligonucleotide or one of two independent siRNAs against USP7. (c) Western blot analysis for p27<sup>Kip1</sup> expression of HEK293T cells transfected with siRNAs against USP7 and siRNA targeting all three human FOXO homologues. A non-targeting siRNA oligonucleotide was used as a control. The relative level of expression of p27<sup>Kip1</sup>, as determined with the Odyssey Infra-red imaging system, is indicated above the blot. (d) HEK293T cells were transfected with either a non-targeting siRNA of USP7 binding to FOXO4 induced after hydrogen peroxide treatment are slower than the kinetics of monoubiquitination of FOXO4. Whereas monoubiquitination increases within 5 min, USP7 binding increases at around 15 min. This is followed by a decline in monoubiquitination, which is visible after 30–60 min, after which USP7 binding also declines. Thus, these data indicate that the monoubiquitination of FOXO4 is not induced through inhibition of USP7 by enhanced oxidative stress. To further support this, we determined USP7 activity following treatment of cells with hydrogen peroxide using a previously described USP activity assay based on an HA-tagged ubiquitin probe fused to a C-terminal electrophilic trap<sup>40</sup>. Consistent with the kinetics, hydrogen peroxide did not affect USP7 activity (Fig. 4c). Together, these data show that enhanced monoubiquitination of FOXO4 facilitates increased USP7 binding and, consequently, deubiquitination. Furthermore, they indicate that induced binding and/or activity of an E3 ligase accounts for the induction of FOXO4 monoubiquitination by oxidative stress.

USP7 inhibits FOXO transcriptional activity

Based on our observation that monoubiquitination induces nuclear localization and increased FOXO transcriptional activity, we analysed the effect of USP7 on FOXO transcriptional activity. Overexpression of wild-type USP7, but not that of the catalytically inactive mutant, inhibited FOXO

oligonucleotide or a combination of two siRNAs against USP7, together with HA-FOXO4, His-ubiquitin and Myc-USP7, as indicated. Cells were treated with 50 μM hydrogen peroxide for 15 min. Experiments were performed as described in Fig. 1a. In addition, western blot analysis for p27<sup>Kip1</sup> was performed. (e) U2OS cells expressing HA-FOXO4 were stained using anti-HA and anti-USP7 antibodies. (Left panel) Staining was analysed using an immunofluorescence microscope. Red, USP7 (anti-rabbit-Alexa 568); green, FOXO4 (anti-mouse-Alexa 488); blue, DAPI counterstaining. (Right panel) Twenty-five cells per experiment were scored for FOXO4 localization. Representative examples for scoring are shown in Supplementary Information, Fig. S6. Data presented are the mean ± s.e.m. of two independent experiments. Scale bar, 10 μm. Full scans of the blots shown in c can be found in the Supplementary Information, Fig. S5.


This Markdown output captures the OCR content from the provided image without any additional text.
ARTICLES

1
Oxidative stress

FOXO
E3
Ub
FOXO
Ub
FOXO

Cytoplasm
Nucleus

USP7
E3
FOXO
CBP/p300
Ac
FOXO
Ac
Sir2

2
Oxidative stress

Figure 6 A model illustrating how oxidative stress regulates FOXO transcriptional activity through ubiquitination and acetylation. Following the induction of cellular oxidative stress, FOXO becomes rapidly monoubiquitinated and activated, possibly through the activation of an E3 ligase. Monoubiquitination either (1) occurs in the cytoplasm and results in nuclear translocation or (2) takes place in the nucleus and causes nuclear retention. Subsequently, USP7 deubiquitinates FOXO, which by itself inhibits FOXO activity by nuclear exclusion. However, FOXO is then available for another round of activation by ubiquitination. Alternatively, CREB-binding protein (CBP)-mediated acetylation of FOXO follows deubiquitination and results in the inhibition of FOXO. This type of inactivation may be long-lasting, as opposed to deubiquitination and nuclear exit only. Finally, monoubiquitination may also be terminated by polyubiquitination and the subsequent degradation of FOXO. USP7, CBP/p300 and Sir2 are all localized within the nucleus (data not shown), therefore deubiquitination and deacetylation occur within the nucleus. Ac, acetyl moiety; Ub, ubiquitin moiety.

transcriptional activity, as measured in a reporter assay using the 6×D BE-luc (Fig. 5a). To assess the involvement of endogenous USP7 in the control of FOXO, we used small interfering RNA (siRNA) against endogenous USP7. Compared with a non-targeting siRNA oligonucleotide, two independent siRNAs against USP7 induced expression of p27kip1 (Fig. 5b, c, d, and see Supplementary Information, Fig. S3) and the retinoblastoma-like protein p130 (see Supplementary Information, Fig. S3), both of which are well-established FOXO transcriptional target genes41,42. This is consistent with the observed inhibition of FOXO transcriptional activity by overexpressed USP7. However, to confirm that the increase in p27kip1 expression was dependent on the presence of endogenous FOXO, we cosilenced FOXOs by siRNA. siRNA against endogenous FOXO abolished the increase in p27kip1 expression that was induced by USP7 knockdown (Fig. 5c), indicating that, indeed, the increase in p27kip1 expression induced by siRNA against USP7 is mediated by FOXO. In addition, increased p27kip expression following USP7 knockdown correlated with increased monoubiquitination of HA–FOXO4, as siRNA against USP7 compared with control increased monoubiquitination of HA–FOXO4 (Fig. 5d, and see Supplementary Information, Fig. S3). Importantly, add-back of USP7 reversed the effect of USP7 knockdown both on FOXO4 monoubiquitination and p27kip1 expression (Fig. 5d), indicating that the effects that we observed with USP7 siRNA are specific for USP7. Moreover, knockdown of USP7 also increased the nuclear localization of FOXO4 (Fig. 5e). This is consistent with the observed nuclear localization of the monoubiquitinated FOXO4 (see Supplementary Information, Fig. S1). Taken together, these data demonstrate that USP7 functions to inhibit FOXO4 activity by removing its monoubiquitin moiety and, thereby, changing FOXO4 localization from the nucleus to the cytosol.

Finally, we tested whether the observed ubiquitination is involved in the regulation of FOXO protein stability (see Supplementary Information, Fig. S4a, b). Under normal conditions, we determined a half-life of FOXO4 of approximately 9–10 h, in agreement with results obtained by Matsuzaki et al.20. Treatment of cells with hydrogen peroxide or co-expression of USP7 did not significantly change the HA–FOXO4 half-life. In this experiment, a puromycin-resistance marker was co-expressed and transfected cells were selected by adding puromycin to the culture medium. This enabled us to not only determine the half-life of the transfected HA–FOXO4, but also that of endogenous FOXO3. Again, expression of USP7 and/or treatment of cells with hydrogen peroxide did not considerably change FOXO3 half-life. In agreement with this, we noted that knockdown of USP7 did not increase FOXO3 levels either (Fig. 5c). Taken together, these data show that, over the time course studied, hydrogen-peroxide-induced ubiquitination and USP7-mediated deubiquitination have no major effect on FOXO protein stability. This reinforces our observation that FOXO4 activity under these conditions is regulated by monoubiquitination.

DISCUSSION

Here, we report the monoubiquitination of FOXO4 following increased cellular oxidative stress due to treatment of cells with hydrogen peroxide. FOXO4 binds to the deubiquitinating enzyme USP7/HAUSP and this binding is increased by cellular oxidative stress. Binding of USP7 results in removal of the ubiquitin moiety on FOXO4 that is induced by hydrogen peroxide treatment. Notably, following induction of oxidative stress, enhanced monoubiquitination of FOXO4 precedes induction of binding of USP7 to FOXO4 and oxidative stress does not inhibit USP7 activity. Furthermore, we show that monoubiquitinated FOXO4 is localized within the nucleus and displays increased transcriptional activity. Overexpression of USP7 and loss of USP7 expression inhibit and stimulate FOXO4 activity, respectively. In addition, for FOXO3 we observed monoubiquitination and increased USP7 binding following induction of cellular oxidative stress, which indicates that

monoubiquitination is a general regulatory mechanism for FOXOs. As FOXO is known to be regulated by PI(3)K (phosphoinositide-3 kinase)/PKB signalling, our results indicate that targeting USP7 by small-molecule inhibitors could be an alternative way to activate FOXO and, therefore, to negatively regulate the oncogenic PI(3)K/PKB signalling pathway, which may be of therapeutic potential in cancer.

Importantly, we have previously observed that acetylation of FOXO4 following hydrogen peroxide treatment occurs with relatively slow kinetics (>60 min), whereas here we show that monoubiquitination occurs rapidly (<5 min) and transiently. This suggests a model of stress regulation of FOXO4 in which the initial activation of FOXO4 through monoubiquitination can be terminated by USP7-mediated deubiquitination and subsequent CBP/p300-mediated acetylation to prevent further monoubiquitination (Fig. 6).

Ubiquitination of FOXOs has been observed in several studies. However, these studies all relate to polyubiquitination^{19–21}. Initially, it was shown that PKB substrates in general, including FOXOs, are targeted for polyubiquitination by growth factors through PI(3)K/PKB signalling^{20,21}. However, in these studies, the kinetics of polyubiquitination (detectable after ~12 h), as well as of protein degradation, are relatively slow (half-life of ~9 h). In chicken embryo fibroblasts (CEFs) that have been transformed through overexpression of activated forms of PI(3)K and/or PKB, platelet-derived growth factor treatment results in a strong reduction in FOXO half-life^{22}. This is probably due to increased expression of the F-box protein Skp2 in these transformed CEFs, as it has been shown that overexpression of Skp2 enhances FOXO polyubiquitination^{19} and that loss of PTEN, and thus activation of PKB, enhances the expression of Skp2 (ref. 43). These data indicate that enhanced degradation of PKB targets such as FOXO is a gain of function for cells that have been transformed through the PI(3)K pathway and are consistent with the observation that forced FOXO activation inhibits cellular transformation of these cells^{41}. In normal, non-transformed cells, the kinetics of FOXO polyubiquitination are clearly different from the kinetics of ubiquitination that we observe here, which supports the notion that, under non-transformed conditions, cellular oxidative stress induces the monoubiquitination of FOXO4 and activates FOXO4. This, in turn, would enable cells to arrest in the G1 phase of the cell cycle to prevent further oxidative damage before resuming the cycle.

Recently, it has become clear that remarkable similarities exist between p53 and FOXO, not only in their shared ability to induce cell-cycle arrest and apoptosis, but apparently also in their mode of regulation. Previously, we and others showed that, similar to p53, FOXOs are regulated by acetylation by p300/CBP and deacetylation by Sir2 (refs 15–18, 26, 44, 45). Here, we extend this similarity by showing regulation of FOXO by monoubiquitination and USP7. We note that the commonality between p53 and FOXO in their cellular response, as well as in their shared regulatory network, may have important implications for our understanding of ageing in higher eukaryotes, as this process is likely to involve both the FOXO and the p53 pathway.

---

### METHODS

#### Cell culture, transfection and treatment.
HEK293T, A14 (human insulin receptor overexpressing mouse NIH3T3 cells^{46}), H1299 and C2C12 cells were maintained in DMEM supplemented with glutamine, penicillin/streptomycin and 10% FBS. In all experiments, cells were cultured in the presence of 10% FBS. HEK293T cells were transiently transfected using FuGENE6 reagent according to the manufacturer’s instructions (Roche, Basel, Switzerland). A14 cells were transfected using the calcium phosphate method. H1299 cells were transfected using PEI (polyethylenimine; 16 μg μl⁻¹; Polysciences, Warrington, PA) in a 2:1 ratio to the amount of DNA. Total amounts of transfected DNA were equalized using pBluescript KSII+. siRNA oligonucleotides were transfected using Xtremegene, according to the manufacturer’s instructions (Roche).

Cycloheximide (Sigma, St Louis, MO), puromycin, insulin, EGF, hydrogen peroxide (H₂O₂; Merck, Whitehouse Station, NJ), TSA and Nam were added at 10 μg ml⁻¹, 2 μg ml⁻¹, 1 μg ml⁻¹, 20 ng ml⁻¹, 50–200 μM, 5 μM and 5 mM, respectively, as indicated.

#### Plasmids, oligonucleotides and recombinant proteins.
pMT2-HA-FOXO4 (ref. 6), GLOFLAG3-FLAG-FOXO4 (ref. 15), pBabe-puro^{47} and 6×DBE^{32} have been described previously. pMT2-HA-ubiquitin-FOXO4 was made by inserting a *SalI* (5′-ACGCGTCGACAATGCAAATTCTCGTCAAAAAC-3′, 5′-ACGCGTCGACACTAGCGAATTCTGACCAC-3′)-digested polymerase chain reaction product (YN-ubiquitin, kindly provided by T. Kerppola, University of Michigan, Ann Arbor, MI, was used as a template) into *SalI*-digested pMT2-HA-FOXO4. YC-FOXO4 was created by ligating an *XhoI*/Klenow-blunted *SalI* fragment from pMT2-HA-FOXO4 into *XhoI*/Klenow-blunted *HindIII* pcDNA-YFP[159–238], kindly provided by J. Rodriguez (VU University Medical Center, Amsterdam, The Netherlands). pBabe-TAP-FOXO4 was created using Gateway Technology (Invitrogen, Carlsbad, CA) and full-length FOXO4 cDNA. HA-FOXO4-K199, 211R was created by site-directed mutagenesis (K199R forward: 5′-GATAGCAGCAGCCGGCTGCTCCGGGGCC-3′, K199R reverse: 5′-GGCCCCGGAGCAGCCGGCTGCTATC-3′, K211R forward: 5′-GCCCAAAAGAAGCGACCATCTGTGCTGC-3′, K211R reverse: 5′-GCAGCACAGATGGTCGCTTCTGGGGGC-3′). The resulting plasmids were verified by automated sequencing.

GloMyc-USP7 and GloMyc-USP7-C223S (USP7-CS) have been described previously^{36}. HA-USP14 and His-ubiquitin were kind gifts from A. Borodovsky (Biogen Idec, Cambridge, MA) and D. Bohmann (University of Rochester Medical Center, Rochester, NY), respectively. pRL-TK (*Tk-Renilla luciferase*) was purchased from Promega (Madison, WI).

Non-targeting siRNA duplex (siCONTROL), siRNA against panFOXO (sense sequence: 5′-AAGGAUAAGGGCGACAGACC-dTdT-3′), and duplexes to downregulate USP7 expression (duplex 2, sense sequence: 5′-ACCUUGGACAAUUUCCUdTdT-3′; duplex 3, sense sequence: 5′-AGUCGUUCAGUCGUAUdTdT-3′) were purchased from Dharmacon RNA Technologies (Lafayette, CO). Purified Flag-USP7 was kindly provided by B. Kumar and J. van der Knaap^{37} (Erasmus University Medical Center, Rotterdam, The Netherlands).

#### Antibodies.
Monoclonal 12CA5 and 9E10 antibodies were produced using hybridoma cell lines and purification on protein-A-agarose columns. Monoclonal antibodies recognizing the Flag-M2 epitope, p27^kip1^, the penta-His epitope, anti-tubulin and glyceraldehyde-3-phosphate dehydrogenase were obtained from Sigma, Transduction Laboratories (Franklin Lakes, NJ), Qiagen (Hilden, Germany), Calbiochem (San Diego, CA) and Chemicon (Temecula, CA), respectively. Polyclonal antibodies recognizing FOXO3 (sc-11351), FOXO4 (sc-5221), Rheb (sc-6342), p130 and USP7 (BL851) were purchased from Santa Cruz Biotechnology (4x) (Santa Cruz, CA) and Bethyl Laboratories (Montgomery, TX), respectively. The polyclonal antibody recognizing FOXO4 (ref. 41) has been described previously.

#### Western blot analysis.
Protein samples were separated by SDS-PAGE and transferred to PVDF membrane (Immobilon; Millipore, Billerica, MA). Western blot analysis was performed under standard conditions using the indicated antibodies. For quantification of protein levels, membranes were probed with fluorophore-conjugated antibodies and analysed using the Odyssey Infra-red imaging system and software according to the manufacturer (Li-Cor, Lincoln, NE).

#### FOXO activity.
To determine the expression of endogenous p27^kip1^, HEK293T cells were transfected with HA-FOXO4 and pBabe-puro using FuGENE and/or with siRNA oligonucleotides. Twenty-four hours before harvest, 2 μg ml⁻¹ puromycin was added to the cultures to select for transfected cells. Cells were lysed in RIPA lysis buffer, equalized using Bradford reagent and 5× Laemmli sample buffer was added to a final concentration of 1×. Samples were analysed
by western blot analysis. Alternatively, cells were lysed in urea lysis buffer (8 M urea, 100 mM Na₂HPO₄/NaH₂PO₄ at pH 8.0, 10 mM Tris-HCl at pH 8.0, 0.2% Triton-X-100, 5 mM N-ethylmaleimide) and expression levels of p27kip1 and p130 were determined.

For luciferase assays, HEK293T cells transiently transfected with a luciferase reporter construct bearing six canonical FOXO binding elements (6×DBE-luc) were transfected with the indicated constructs. Luciferase counts were normalized using Tk-Renilla luciferase. Samples were analysed according to the manufacturer (Promega). Representative data are shown as mean ± s.d. of triplicates. If indicated, a Student’s *t*-test was performed using GraphPad statistical analysis.

### Co-immunoprecipitation.
HEK293T or H1299 cells were transfected with the indicated constructs. Cells were left untreated or were treated with hydrogen peroxide for the indicated times. Forty hours post-transfection, cells were lysed. FOXO proteins were immunoprecipitated from whole-cell lysates (50 mM Tris-HCl at pH 7.5, 1% Triton-X-100, 1.5 mM MgCl₂, 5 mM EDTA, 100 mM NaCl, protease inhibitors). TAP–FOXO4 was immunoprecipitated from isolated nuclei (isolated using a standard nuclear–cytosolic fractionation method) from C2C12 cells that were stably expressing TAP–FOXO4 at near-endogenous levels, using IgG sepharose beads. Immunoprecipitation samples were analysed by western blot analysis.

### Ubiquitination.
HEK293T or H1299 cells were transfected with the indicated constructs and/or siRNA oligonucleotides. Cells were treated with hydrogen peroxide using the indicated concentrations and for the indicated times. Forty hours post-transfection, cells were lysed in urea lysis buffer. Ubiquitinated proteins were precipitated from urea lysates using Ni–NTA agarose beads, essentially as described previously²⁵, and analysed by western blotting for the presence of FOXO4. For *in vitro* deubiquitination of FOXO4 by USP7, pull-down samples were dialysed using reaction buffer (60 mM HEPES at pH 7.5, 5 mM MgCl₂, 4% glycerol) and purified USP7 was added. Reactions were incubated at 30 °C for 1 h and terminated by the addition of 5× LaemmlI sample buffer. Alternatively, cells were lysed in RIPA lysis buffer, FOXO4 was immunoprecipitated using a combination of N19 α-FOXO4 antibody and home-made polyclonal α-FOXO4 antibody. α-Rheb antibody was used as an unrelated control. Immunoprecipitates were analysed for the presence of ubiquitin using an anti-penta-His antibody.

### Activity assay for USP7.
To measure the activity of USP7, an HA-tagged ubiquitin with a C-terminal electrophilic trap (vinyl methyl ester), synthesized by an intein-based method as described previously⁴⁰ and a kind gift from Dr H. Ovaa, was used. Endogenous and exogenous USP7 was immunoprecipitated from HEK293T cells treated with 200 μM hydrogen peroxide for the indicated times. Subsequently, the immunoprecipitated USP7 was incubated with the HA-tagged ubiquitin probe for 1 h at 37 °C in 50 mM Tris at pH 7.4, 150 mM NaCl, 0.1% Triton-X-100 and 5 mM EDTA. The reaction was terminated by adding 5× LaemmlI sample buffer. Samples were analysed by western blotting using the indicated antibodies.

### FOXO half-life analysis.
HEK293T were transfected with pBabe-puro plasmid and the indicated constructs. Twenty-four hours post-transfection, puromycin and 50 μM hydrogen peroxide were added for 24 h. Cells were treated with cycloheximide for 8, 12, 16 or 20 h and lysed in RIPA lysis buffer. Protein levels were equalized using Bradford reagent. Samples were analysed by western blotting. Relative FOXO4 and FOXO3 protein expression levels were calculated.

### Ubiquitin-mediated fluorescence complementation.
Ubiquitin-mediated fluorescence complementation was essentially performed as described previously³⁰. A14 cells were transfected with YC–FOXO4 and YN–ubiquitin. Forty hours post-transfection, cells were fixed using 4% paraformaldehyde and nuclei were stained using DAPI. Fluorescence was visualized using a Zeiss LSM 510 confocal microscope.

### Immunofluorescence.
A14 cells were transfected with HA–FOXO4, HA–ubiquitin–FOXO4 or HA–FOXO4-K199,211R. Forty hours post-transfection, cells were fixed in 4% paraformaldehyde. The different FOXO4 proteins were stained using anti-FOXO4 antibody and anti-rabbit-Cy3 (Fig. 4b) or anti-HA antibody and α-mouse-Alexa488 (see Supplementary Information, Fig. S1). Nuclei were visualized using DAPI. Fluorescence was captured using a Zeiss LSM 510 confocal microscope

(Fig. 4b) or a Zeiss Axioskop2 (see Supplementary Information, Fig. S1). A total of 50 (Fig. 2b) or 25 (see Supplementary Information, Fig. S1) cells per experiment were scored for FOXO4 localization (nuclear, cytosolic or both). Data presented are the mean ± s.e.m. of two independent experiments.

U2OS cells were transfected with HA–FOXO4 and either control siRNA or siRNA against USP7. Forty hours post-transfection, cells were fixed in 4% paraformaldehyde. Cells were stained using anti-HA antibody in combination with anti-mouse-Alexa488 and anti-USP7 plus anti-rabbit-Alexa568. Nuclei were visualised using DAPI. Fluorescence was captured using a Zeiss Axioskop2. Twenty-five cells per experiment were scored for FOXO4 localization (nuclear, cytosolic or both). Data presented are the mean ± s .e.m. of two independent experiments.

**Note:** Supplementary Information is available on the Nature Cell Biology website.

### ACKNOWLEDGEMENTS
We thank J.L. Bos and L. Price for critically reading the manuscript; the members of the Bos and Burgering labs for helpful discussions; T. Kerppola., J. Rodriguez, A. Borodovsky and D. Bohmann for kindly providing DNA constructs; and B. Kumar and J. van der Knaap for supplying purified USP7. The authors also acknowledge H. Ovaa for providing the HA–ubiquitin probe and Hybrigenics staff for their contributions. This work was supported by grants from the Dutch Cancer Society (KWF, UU-2001-2438) to A.v.d.H.; from The Netherlands organization for scientific research (NWO, Zon-MW 906-02-041) to M.M.M.; from the Cancer Genomics center to N.v.d.B.; and from the European Commission Transfog Consortium (LSHC-CT-2004-503438) to A.B.B.

### AUTHOR CONTRIBUTIONS
A.v.d.H., A.M.M.d.V-S. and B.M.T.B. conceived and designed the experiments. A.v.d.H., A.M.M.d.V-S., A.B.B., M.H.v.T. and N.v.d.B performed the experiments. F.C. and M.M.M. contributed essential materials. A.v.d.H., A.M.M.d.V-S. and B.M.T.B analysed the data. A.v.d.H. and B.M.T.B wrote the paper.

### COMPETING FINANCIAL INTERESTS
The authors declare that they have no competing financial interests.

Published online at http://www.nature.com/naturecellbiology/
Reprints and permissions information is available online at http://npg.naturecom/reprintsandpermissions/

1. Katoh, M. Human FOX gene family (Review). *Int. J. Oncol.* **25**, 1495–1500 (2004).
2. Tran, H. *et al.* DNA repair pathway stimulated by the forkhead transcription factor FOXO3a through the Gadd45 protein. *Science* **296**, 530–534 (2002).
3. Kops, G. J. *et al.* Forkhead transcription factor FOXO3a protects quiescent cells from oxidative stress. *Nature* **419**, 316–321 (2002).
4. Burgering, B. M. & Kops, G. J. Cell cycle and death control: long live Forkheads. *Trends Biochem. Sci.* **27**, 352–360 (2002).
5. Brunet, A. *et al.* Akt promotes cell survival by phosphorylating and inhibiting a Forkhead transcription factor. *Cell* **96**, 857–868 (1999).
6. Kops, G. J. *et al.* Direct control of the Forkhead transcription factor AFX by protein kinase B. *Nature* **398**, 630–634 (1999).
7. Tran, H., Brunet, A., Griffith, E. C. & Greenberg, M. E. The many forks in FOXO’s road. *Sci STKE* **2003**, RE5 (2003).
8. De Ruiter, N. D., Burgering, B. M. & Bos, J. L. Regulation of the Forkhead transcription factor AFX by Ral-dependent phosphorylation of threonines 447 and 451. *Mol. Cell. Biol.* **21**, 8225–8235 (2001).
9. Essers, M. A. *et al.* FOXO transcription factor activation by oxidative stress mediated by the small GTPase Ral and JNK. *EMBO J.* **23**, 4802–4812 (2004).
10. Wang, M. C., Bohmann, D. & Jasper, H. JNK extends life span and limits growth by antagonizing cellular and organism-wide responses to insulin signaling. *Cell/* **121**, 115–125 (2005).
11. Oh, S. W. *et al.* JNK regulates lifespan in *Caenorhabditis elegans* by modulating nuclear translocation of forkhead transcription factor/DAF-16. *Proc. Natl Acad. Sci. USA* **102**, 4494–4499 (2005).
12. Nanji, M., Hopper, N. A. & Gems, D. LET-60 RAS modulates effects of insulin/IGF-1 signaling on development and aging in *Caenorhabditis elegans*. *Aging Cell* **4**, 235–245 (2005).
13. Mahmud, D. L. *et al.* Phosphorylation of forkhead transcription factors by erythropoietin and stem cell factor prevents acetylation and their interaction with coactivator p300 in erythroid progenitor cells. *Oncogene* **21**, 1556–1562 (2002).
14. Fukuoka, M. *et al.* Negative regulation of forkhead transcription factor AFX (Foxo4) by CBP-induced acetylation. *Int. J. Mol. Med.* **12**, 503–508 (2003).
15. van der Horst, A. *et al.* FOXO4 is acetylated upon peroxide stress and deacetylated by the longevity protein hSir2(SIRT1). *J. Biol. Chem.* **279**, 28873–28879 (2004).
16. Brunet, A. *et al.* Stress-dependent regulation of FOXO transcription factors by the SIRT1 deacetylase. *Science* **303**, 2011–2015 (2004).
17. Motta, M. C. *et al.* Mammalian SIRT1 represses forkhead transcription factors. *Cell/* **116**, 551–563 (2004).
18. Daitoku, H. *et al.* Silent information regulator 2 potentiates Foxo1-mediated transcription through its deacetylase activity. *Proc. Natl Acad. Sci. USA* **101**, 10042–10047 (2004).

19. Huang, H. *et al.* Skp2 inhibits FOXO1 in tumor suppression through ubiquitin-mediated degradation. *Proc. Natl Acad. Sci. USA* **102**, 1649–1654 (2005).

20. Matsuzaki, H., Daitoku, H., Hatta, M., Tanaka, K. & Fukamizu, A. Insulin-induced phosphorylation of FKHR (Foxo1) targets to proteasomal degradation. *Proc. Natl Acad. Sci. USA* **100**, 11285–11290 (2003).

21. Plas, D. R. & Thompson, C. B. Akt activation promotes degradation of tuberin and FOXO3a via the proteasome. *J. Biol. Chem.* **278**, 12361–12366 (2003).

22. Aoki, M., Jiang, H. & Vogt, P. K. Proteasomal degradation of the FoxO1 transcriptional regulator in cells transformed by the P3k and Akt oncoproteins. *Proc. Natl Acad. Sci. USA* **101**, 13613–13617 (2004).

23. Onishi, Y. *et al.* Identification of mono-ubiquitinated LDH-A in skeletal muscle cells exposed to oxidative stress. *Biochem. Biophys. Res. Commun.* **336**, 799–806 (2005).

24. Cao, C. *et al.* Ubiquitination and degradation of the Arg tyrosine kinase is regulated by oxidative stress. *Oncogene* **24**, 2433–2440 (2005).

25. Peng, J. *et al.* A proteomics approach to understanding protein ubiquitination. *Nature Biotechnol.* **21**, 921–926 (2003).

26. Kobayashi, Y. *et al.* SIRT1 is critical regulator of FOXO-mediated transcription in response to oxidative stress. *Int. J. Mol. Med.* **16**, 237–243 (2005).

27. Greer, E. L. & Brunet, A. FOXO transcription factors at the interface between longevity and tumor suppression. *Oncogene* **24**, 7410–7425 (2005).

28. Sigismund, S., Polo, S. & Di Fiore, P. P. Signaling through monoubiquitination. *Curr. Top. Microbiol. Immunol.* **286**, 149–185 (2004).

29. Dikic, I. Mechanisms controlling EGF receptor endocytosis and degradation. *Biochem. Soc. Trans.* **31**, 1178–1181 (2003).

30. Fang, D. & Kerppola, T. K. Ubiquitin-mediated fluorescence complementation reveals that Jun ubiquitinated by Itch/AIP4 is localized to lysosomes. *Proc. Natl Acad. Sci. USA* **101**, 14782–14787 (2004).

31. Haglund, K. *et al.* Multiple monoubiquitination of RTKs is sufficient for their endocytosis and degradation. *Nature Cell Biol.* **5**, 461–466 (2003).

32. Furuyama, T., Nakazawa, T., Nakano, I. & Mori, N. Identification of the differential distribution patterns of mRNAs and consensus binding sequences for mouse DAF-16 homologues. *Biochem. J.* **349**, 629–634 (2000).

33. Colland, F. *et al.* Functional proteomics mapping of a human signaling pathway. *Genome Res.* **14**, 1324–1332 (2004).

34. Li, M. *et al.* Deubiquitination of p53 by HAUSP is an important pathway for p53 stabilization. *Nature* **416**, 648–653 (2002).

35. Li, M., Brooks, C. L., Kon, N. & Gu, W. A dynamic role of HAUSP in the p53-Mdm2 pathway. *Mol. Cell* **13**, 879–886 (2004).

36. Meulmeester, E. *et al.* Loss of HAUSP-mediated deubiquitination contributes to DNA damage-induced destabilization of Hdmx and Hdm2. *Mol. Cell* **18**, 565–576 (2005).

37. van der Knaap, J. A. *et al.* GMP synthetase stimulates histone H2B deubiquitylation by the epigenetic silencer USP7. *Mol. Cell* **17**, 695–707 (2005).

38. Amerik, A. Y. & Hochstrasser, M. Mechanism and function of deubiquitinating enzymes. *Biochim. Biophys. Acta* **1695**, 189–207 (2004).

39. Yin, L., Krantz, B., Russell, N. S., Deshpande, S. & Wilkinson, K. D. Nonhydrolyzable diubiquitin analogues are inhibitors of ubiquitin conjugation and deconjugation. *Biochemistry* **39**, 10001–10010 (2000).

40. Borodovsky, A. *et al.* Chemistry-based functional proteomics reveals novel members of the deubiquitinating enzyme family. *Chem. Biol.* **9**, 1149–1159 (2002).

41. Medema, R. H., Kops, G. J., Bos, J. L. & Burgering, B. M. AFX-like Forkhead transcription factors mediate cell-cycle regulation by Ras and PKB through p27kip1. *Nature* **404**, 782–787 (2000).

42. Kops, G. J. *et al.* Control of cell cycle exit and entry by protein kinase B-regulated forkhead transcription factors. *Mol. Cell. Biol.* **22**, 2025–2036 (2002).

43. Mamillapalli, R. *et al.* PTEN regulates the ubiquitin-dependent degradation of the CDK inhibitor p27(KIP1) through the ubiquitin E3 ligase SCF(SKP2). *Curr. Biol.* **11**, 263–267 (2001).

44. Luo, J. *et al.* Negative control of p53 by Sir2α promotes cell survival under stress. *Cell* **107**, 137–148 (2001).

45. Vaziri, H. *et al.* hSIR2(SIRT1) functions as an NAD-dependent p53 deacetylase. *Cell* **107**, 149–159 (2001).

46. Burgering, B. M. & Coffer, P. J. Protein kinase B (c-Akt) in phosphatidylinositol-3-OH kinase signal transduction. *Nature* **376**, 599–602 (1995).

47. Medema, R. H., Herrera, R. E., Lam, F. & Weinberg, R. A. Growth suppression by p16ink4 requires functional retinoblastoma protein. *Proc. Natl Acad. Sci. USA* **92**, 6289–6293 (1995).

Van der Horst et al., Figure S1

- -H₂O₂
- +H₂O₂

| wt FOXO4 |
| --- |
| Alexa488 |

| FOXO4-K199,211R |
| --- |
| Alexa488 |

FOXO4 localisation

![Bar Chart](#)

Figure S1. Localisation of FOXO4-K199,211R is similar to localisation of wild-type FOXO4. A14 cells expressing wild-type HA-FOXO4 or HA-FOXO4-K199,211R were treated with 200 μM hydrogen peroxide for 30 minutes and stained using α-HA antibody. (top) Staining was analysed using an immunofluorescence microscope. Green, α-mouse-Alexa488; blue, DAPI counterstaining. (bottom) Twenty-five cells per experiment were scored for FOXO4 localisation (c=cytosol, n=nucleus). Representative examples for scoring are shown in Supplementary Information, Fig. S5. Data presented are the mean ± s.d. of two independent experiments. Bar, 10 μm.

© 2006 Nature Publishing Group

Van der Horst et al., Figure S2

a
myc-USP7 HA-USP14
Ub-FOXO4 →
USP7 →
FOXO4 →
USP14 →
His-ubi: + - + + + + + + +
FLAG-FOXO4: - + + + + + + + +
myc-USP7/HA-USP14: - - - wt wt CS CS wt wt
H₂O₂: + + - + - + + - +

b
myc-USP7 HA-USP14
IP: myc FLAG HA FLAG
USP7 →
USP14 Ig HC →
USP7 →
FOXO4 →
USP14 →
myc-USP7/HA-USP14: wt - wt wt CS CS wt - wt
FLAG-FOXO4: - + + + + - + + +
H₂O₂: + + - + - + + - +

Figure S2. Mutant USP7 and USP14 do not deubiquitinate FOXO4. (a) To show dependence on enzymatic activity and specificity of USP7, respectively myc-USP7-CS and HA-USP14 were used, as in Fig. 3c. (b) Co-immunoprecipitation of FOXO4 and USP7 or USP14 in HEK293T cells treated with hydrogen peroxide (as in Fig. 3b).

Van der Horst et al., Figure S3

Ub-FOXO4 → 
Pulldown: Ni-NTA  
WB: α-HA  

WB: α-HA  
WB: α-USP7  
WB: α-p27kip1  
WB: α-p130  
WB: α-tubulin  

HA-FOXO: + + + +  
His-ubi: + + + +  
RNAi: ctrl USP7 ctrl USP7  
H₂O₂: - - + +

Figure S3. Knockdown of USP7 increases FOXO4 monoubiquitination and FOXO4 activity. HEK293T cells were transfected with HA-FOXO4, his-ubiquitin and either a non-targeting siRNA oligo or a combination of two siRNAs against USP7. Cells were treated with 50 μM hydrogen peroxide for 30 minutes. Experiments were performed as described in Figure 1a. In addition, Western blot analysis for p27kip1, p130, USP7 and tubulin expression was performed.

© 2006 Nature Publishing Group

Van der Horst et al., Figure S4

a

| FOXO4 → |  | WB: α-HA |
| --- | --- | --- |
| FOXO3 → |  | WB: α-FOXO3 |
| p27<sup>kip1</sup> → |  | WB: α-p27<sup>kip1</sup> |
| USP7 → |  | WB: α-myc |

| HA-FOXO4: | - | + | + | + | + | + | + | + | - | + | + | + | + | + | + | + | + |
| myc-USP7: | - | - | - | - | - | + | + | + | + | - | - | - | - | + | + | + | + |
| CHX (h): | 0 | 0 | 8 | 12 | 16 | 20 | 0 | 8 | 12 | 16 | 20 | 0 | 0 | 8 | 12 | 16 | 20 |
| H₂O₂: | - | - | - | - | - | - | - | - | - | - | + | + | + | + | + | + | + |

b

**FOXO4 half-life**

![Graph](#)

**FOXO3 half-life**

![Graph](#)

**Legend:**
- ○: mock
- ●: USP7
- □: mock + H₂O₂
- ■: USP7 + H₂O₂

Figure S4. Half-life analysis of FOXO in HEK293T cells overexpressing HA-FOXO4 and myc-USP7. (a) Puromycin-selected cells were treated with 50 μM hydrogen peroxide for 24 hours and with cycloheximide (CHX) for the indicated times. Protein levels were equalized and samples were analyzed using the indicated antibodies. (b) Using the Odyssey Infrared imaging system, relative expression levels were calculated and displayed in a graph.

Van der Horst et al., Figure S5

Original figures

ECL
Ni-NTA pulldown
Total lysate
M<sub>r</sub>(K)
ECL
117
92
Ub-FOXO4
FOXO4
54
1 2 3 4
Fig. 1a, right panel, lanes 1-4
1 2 3 4 5
Fig. 1f, right panel, lanes 1-5

ECL
Ni-NTA pulldown
M<sub>r</sub>(K)
Total lysate
117
92
Ub-FOXO4
66
1 2 3 4 5 7 8
1 2 3 4 5 6 7 8
Fig. 4a, lanes 1-8

ECL
co-IP
M<sub>r</sub>(K)
Total lysate
USP7
117
92
66
117
92
66
1 2 3 4 5 6 7 8 9
1 2 3 4 5 6 7 8 9
Fig. 4b, lanes 1-9

Odyssey
p27<sup>kip1</sup>
54
38
33
1 2 3 4 5 6
Fig. 5c, lanes 1-6

Odyssey
FOXO3
MW marker
↓
66
1 2 3 4 5 6
205
117
92
66

ECL
USP7
tubulin
117
92
66
54
38
1 2 3 4 5 6
© 2006 Nature Publishing Group

Van der Horst et al., Figure S4

a  
![Image Description](image1)  

| HA-FOXO4: | - | + | + | + | + | + | + | + | - | + | + | + | + | + | + | + | + |
| myc-USP7: | - | - | - | - | - | + | + | + | - | - | - | - | + | + | + | + | + |
| CHX (h):   | 0 | 0 | 8 | 12 | 16 | 20 | 0 | 8 | 12 | 16 | 20 | 0 | 8 | 12 | 16 | 20 |
| H₂O₂:      | - | - | - | - | - | - | - | - | - | + | + | + | + | + | + | + | + |

b  
![Graph Description](image2)  

Figure S4. Half-life analysis of FOXO in HEK293T cells overexpressing HA-FOXO4 and myc-USP7. (a) Puromycin-selected cells were treated with 50 μM hydrogen peroxide for 24 hours and with cycloheximide (CHX) for the indicated times. Protein levels were equalized and samples were analyzed using the indicated antibodies. (b) Using the Odyssey Infrared imaging system, relative expression levels were calculated and displayed in a graph.

---

Van der Horst et al., Figure S6  

cytosol          cytosol + nucleus         nucleus  
![Image Description](image3)  

Figure S6. Representative examples of FOXO4 localization for scoring the experiments shown in Figs. 2b, 5e, and S2. Bar, 10 μm.
