
FEBS Letters 580 (2006) 5666–5670

# Identification of cDNAs encoding pterocarpan reductase involved in isoflavan phytoalexin biosynthesis in *Lotus japonicus* by EST mining

**Tomoyoshi Akashi**, Shoko Koshimizu, Toshio Aoki, Shin-ichi Ayabe

*Department of Applied Biological Sciences, Nihon University, 1866 Kameino, Fujisawa, Kanagawa 252-8510, Japan*

Received 26 July 2006; revised 5 September 2006; accepted 8 September 2006

Available online 18 September 2006

Edited by Ulf-Ingo Flügge

---

### Abstract
Isoflavans and pterocarpans are the major biosynthetically connected phytoalexins in legumes. A search of the expressed sequence tag library of a model legume *Lotus japonicus*, which produces an (−)-isoflavan, for homologs of phenylcoumaran benzyl ether reductase catalyzing the reductive cleavage of dihydrofurans, yielded seven full-length cDNAs, and the encoded proteins were analyzed in vitro. Four of them cleaved the dihydrofuran of a pterocarpan medicarpin to yield an isoflavan (−)-vestitol and were designated pterocarpan reductase (PTR). Two PTRs displayed enantiospecificity to (−)-medicarpin, representing genuine *L. japonicus* PTRs, while the other two lacked enantiospecificity and were presumed to be evolutionarily primitive types.

© 2006 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.

**Keywords:** Isoflavonoid biosynthesis; Leguminosae; Phytoalexin; Pterocarpan reductase; Short-chain dehydrogenase/reductase; *Lotus japonicus*

---

### 1. Introduction

Natural isoflavonoids comprise a variety of skeletons within the basic 3-phenylchroman structure [1]. They are mainly distributed to leguminous plants and play ecophysiological roles as defensive materials against other organisms from animals to microbes and also as symbiotic signals to rhizobial bacteria to form nitrogen-fixing root nodules [2,3]. Isoflavonoids show estrogenic activity in animals because of their structural resemblance to partially aromatized steroids, and appropriate intake of isoflavonoids in the diet is beneficial to human health [2]. Isoflavonoids possessing pterocarpan and isoflavan skeletons (see Fig. 1) are the most frequently found phytoalexins of leguminous plants, and they are involved in defense responses against phytopathogenic microorganisms [2]. The biosyntheses of these two classes of isoflavonoids are believed to be closely related, because most, if not all, isoflavans bear a hydroxyl at C-2′ in the B-ring that can participate in an ether linkage to C-4 to produce the pterocarpan skeleton. In fact, interconversion of the pterocarpan medicarpin and the isoflavan vestitol has been demonstrated experimentally by feeding ^14^C-labeled precursors to CuCl₂-treated alfalfa seedlings [4]. The biosynthetic pathway leading to medicarpin from phenylpropanoid- and acetate-derived CoAs has been established, and almost all the cDNAs encoding enzymes in the pathway have been characterized [2,5–7]. However, the mechanism of the formation of isoflavans and the enzymes involved has not been clarified until now.

As the identification of the genes of enzymes involved in plant secondary metabolism has progressed, it has become clearer that the enzymes are often encoded by multigene families that evolved by gene duplication and mutation [8,9]. A typical case is represented by the members of the short-chain dehydrogenase/reductase (SDR) family [10], which catalyze multiple steps in the biosynthesis of flavonoids (dihydroflavonol 4-reductase, leucoanthocyanidin reductase, and anthocyanidin reductase) [11] and isoflavonoids [isoflavone reductase (IFR) and vestitone reductase (VR)] [2]. The enzymes of this family show further diverse functions in the biosynthesis of lignans [phenylcoumaran benzyl ether reductase (PCBER) and pinoresinol–lariciresinol reductase (PLR)] [12] and phenylpropenes (eugenol synthase and isoeugenol synthase) [13]. PCBER is ubiquitous in vascular plants and has been proposed to be an ancestor of IFR and PLR [12,14].

We have so far identified cDNAs encoding enzymes of the isoflavonoid pathway based on the known catalytic mechanism of P450s [15] and also by assaying the activity in the fractionated cDNA expression library [6,7]. In this study, we hypothesized that the isoflavan skeleton is synthesized from a pterocarpan by an analogous reaction to the cleavage of the dihydrofuran ring of lignans catalyzed by PCBER (Fig. 1). We here present the cloning of cDNAs encoding the enzyme [pterocarpan reductase (PTR)] responsible for isoflavan production (Fig. 1A) through a new approach, i.e., mining the expressed sequence tag (EST) database of a model legume, *Lotus japonicus*, which produces vestitol, for homologs of the enzymes that perform the same type of reaction as isoflavan production in the SDR family.

---

### 2. Materials and methods

#### 2.1. Chemicals
(±)-Medicarpin, (±)-vestitol, and 2′-hydroxyformononetin were obtained from Plantech (Reading, UK). (−)-Medicarpin and (−)-vestitol were isolated from *Glycyrrhiza echinata* cells [16] and *L. japonicus*

---

*Corresponding author. Fax: +81 466 84 3353.*

*E-mail address:* akashi@brs.nihon-u.ac.jp (T. Akashi).

**Abbreviations:** DMID, 7,2′-dihydroxy-4′-methoxyisoflavanol dehydratase; EST, expressed sequence tag; GSH, reduced glutathione; IFR, isoflavone reductase; NMR, nuclear magnetic resonance; PCBER, phenylcoumaran benzyl ether reductase; PLR, pinoresinol–lariciresinol reductase; PTR, pterocarpan reductase; Rt, retention time; SDR, short-chain dehydrogenase/reductase; VR, vestitone reductase

0014-5793/$32.00 © 2006 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.

doi:10.1016/j.febslet.2006.09.016

T. Akashi et al. / FEBS Letters 580 (2006) 5666–5670

A

4-Coumaroyl-CoA  
3 x malonyl-CoA  

$\longrightarrow$

HO  
$\begin{array}{c} \text { O } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { HO } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { OCH }_{3} \\ \text { | } \\ \text { C } \end{array}$  

2'-Hydroxyformononetin  

$\downarrow$  

IFR  
(NADPH)  

HO  
$\begin{array}{c} \text { O } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { HO } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { OCH }_{3} \\ \text { | } \\ \text { C } \end{array}$  

Vestitone  

$\longleftarrow$  

VR  
(NADPH)  

HO  
$\begin{array}{c} \text { O } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { HO } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { OCH }_{3} \\ \text { | } \\ \text { C } \end{array}$  

7,2'-Dihydroxy-4'-  
methoxyisoflavanol (DMI)  

$\downarrow$  

DMID  

HO  
$\begin{array}{c} \text { O } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { HO } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { OCH }_{3} \\ \text { | } \\ \text { C } \end{array}$  

(-)-Medicarpin  
(PTEROCARPAN)  

$\longrightarrow$  

PTR  
(NADPH)  

HO  
$\begin{array}{c} \text { O } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { HO } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { OCH }_{3} \\ \text { | } \\ \text { C } \end{array}$  

(-)-Vestitol  
(ISOFLAVAN)  

B

OH  
$\begin{array}{c} \text { O } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { OH } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { OMe } \\ \text { | } \\ \text { C } \end{array}$  

(+)-Dehydrodiconiferyl alcohol  

$\longrightarrow$  

PCBER  
(NADPH)  

OH  
$\begin{array}{c} \text { O } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { OH } \\ \text { | } \\ \text { C } \end{array}$  
$\begin{array}{c} \text { OMe } \\ \text { | } \\ \text { C } \end{array}$  

(+)-Isodihydrodehydrodiconiferyl alcohol  

Fig. 1. Biosynthesis of isoflavonoids (A) and the reaction catalyzed by PCBER (B). In this study, cDNAs encoding PTR (box) were identified. A total of seven enzymes (chalcone synthase, chalcone polyketide reductase, chalcone isomerase, 2-hydroxyisoflavanone synthase, 2,7,4'-trihydroxyisoflavanone 4'-O-methyltransferase, 2-hydroxyisoflavanone dehydratase, and isoflavone 2'-hydroxylase) participate in the biosynthesis of 2'-hydroxyformononetin from two CoA-type precursors. Abbreviations used are: DMID, 7,2'-dihydroxy-4'-methoxyisoflavanol dehydratase; IFR, isoflavone reductase; PCBER, phenylcoumaran benzyl ether reductase; PTR, pterocarpan reductase; VR, vestitone reductase.

seedlings [17], respectively. (+)-Pinoresinol and (+)-dehydrodiconiferyl alcohol were synthesized by phenol coupling with coniferyl alcohol [12].

### 2.2. L. japonicus EST database search, cDNA sequencing, and heterologous expression of proteins

A *L. japonicus* EST database ([http://www.kazusa.or.jp/en/plant/lotus/EST](http://www.kazusa.or.jp/en/plant/lotus/EST)) was searched for cDNAs homologous to *Populus trichocarpa* PCBER (AJ005803). EST clones obtained from the Kazusa DNA Research Institute (Kisarazu, Japan) were sequenced. Clone IDs of the Kazusa DNA Research Institute (given in parentheses) and accession numbers of the complete cDNA sequences deposited in GenBank databases are: **AB265589** (MWM249g01) for *PTR1*, **AB265590** (MWM058d07) for *PTR2*, **AB265591** (SPDL089e08) for *PTR3*, **AB265592** (MPD075a04) for *PTR4*, **AB265593** (MWM134b04) for *R5*, **AB265594** (MWM092h09) for *R6* and **AB265595** (MWM027h09) for *R7*.

The coding regions of *PTRs* were amplified by PCR with KOD polymerase (Toyobo, Tokyo, Japan) and cDNA clones as the templates, using specific primers (*Supplementary methods and data*). The *NdeI-XhoI* fragments of the PCR products from *PTR1* and *PTR2* were subcloned into corresponding sites of pET28a (Novagen). The *BamHI-SalI* fragment obtained from *R7* was subcloned into the corresponding sites of pCDFDuet-1 (Novagen). The PCR products from *PTR3, PTR4, R5,* and *R6* were cloned into pET46 Ek/LIC (Novagen) according to the manufacturer's instructions. Recombinant PTR proteins expressed in *Escherichia coli* Rosetta2(DE3) (Novagen) were purified as described [6].

### 2.3. Preparation of cell-free extracts of *L. japonicus*

Seedlings of *L. japonicus* Gifu B-129 were grown from the seeds that are available from the National BioResource Project Office ([http://www.shigen.nig.ac.jp/legume/legumebase/index.jsp](http://www.shigen.nig.ac.jp/legume/legumebase/index.jsp)) and elicited by 10 mM reduced glutathione (GSH) for 10 h as described [17]. Filtered homogenates of the seedlings (1 g fresh weight) in 2 ml of 100 mM potassium phosphate buffer (pH 7.5) supplemented with 10% sucrose were centrifuged (10000 × g, 10 min) and then treated with Dowex 1-X2 (1 g) for 20 min. The solution obtained by filtration was used as the crude enzyme (300 μg protein/ml).

### 2.4. Assays

The buffer used was 100 mM potassium phosphate buffer (pH 7.5) supplemented with 10% sucrose. (-)-Medicarpin (20 nmol) was incubated with the enzyme preparations in the presence of 1 mM NADPH in the total volume of 100 μl at 30 °C for 10 min. To determine the specific activity, the purified recombinant PTR proteins (PTR1, 100 ng; PTR2, 50 ng; PTR3, 12.3 μg; PTR4, 13.4 μg) and the crude extracts of *L. japonicus* (100 μg protein) were used. To analyze the stereospecificity of the PTR reaction, (±)-medicarpin (10 nmol) and purified recombinant PTR proteins (PTR1, 100 ng; PTR2, 50 ng; PTR3, 30 μg; PTR4, 30 μg) or the crude extracts of *L. japonicus* (100 μg protein) were incubated with NADPH for 2 h. The ethyl acetate extracts of the reaction mixtures were analyzed by HPLC on a TSK-Gel ODS-80TM column (4.6 × 150 mm; Tosoh, Tokyo, Japan) with 55% methanol in water at a flow rate of 0.8 ml/min at 40 °C or a Chiral RU-2 column (4.6 × 150 mm; Shiseido Fine Chemicals, Tokyo, Japan) with 50% acetonitrile in water at a flow rate of 0.25 ml/min at 40 °C. The purified recombinant PTR proteins (PTR1, 1 μg; PTR2, 1 μg; PTR3, 100 μg; PTR4, 100 μg) or the crude extracts of *L. japonicus* (300 μg protein) were also incubated with 20 nmol of (±)-pinoresinol or (±)-dehydrolconiferyl alcohol and 1 mM NADPH (for substrate specificity), or 10 nmol (±)-vestitol and 1 mM NADP⁺ (for reverse reaction of PTR), at 30 °C for 90 min. The ethyl acetate extracts of the reaction mixtures were analyzed by HPLC on a TSK-Gel ODS-80TM column as described above.

### 2.5. Phylogenetic analysis

The amino acid sequences were analyzed using the ClustalW program [18] of the DNA Data Bank of Japan (Shizuoka, Japan), and a neighbor-joining tree was produced from the results of 1000 bootstrap replicates. The tree was displayed by NJPlot software [19].

## 3. Results and discussion

### 3.1. EST database search and cloning of *L. japonicus* PTR cDNAs

PCBER catalyzes the cleavage of the ether linkage of the dihydrofuran ring of (±)-dehydrodiconiferyl alcohol (*Fig.* 1B). The isoflavan skeleton can be produced from a pterocarpan by the same reaction mechanism (*Fig.* 1A). Seven cDNA sequences with >40% identity at the amino acid level with *P. trichocarpa* PCBER were then selected from the *L. japonicus* EST library [20]. All these clones were presumed to contain full open reading frames because they exhibited the predicted initiation codons. Therefore, they were completely sequenced, and the activities were tested in the recombinant *E. coli* systems as described below to identify four PTRs, PTR1 to PTR4. The other three, R5 to R7, did not show PTR activity, but the amino acid sequences of R5 and R6 were about 70% identical to *Linum usitatissimum* PLR [21] and the amino acid sequence of R7 was 99% identical to the previously reported LjIFR [17] (*Supplementary Table 1*). The identities of the amino acid sequences among PTR1–PTR4 are 60–67%. PTR3 (65%) and PTR4 (72%) had slightly higher identity to *Populus trichocarpa* PCBER than PTR1 and PTR2 (about 60%). PTR1 and PTR2 sequences were found in the two-week-old whole plant and

the root EST libraries and PTR3 and PTR4 in the flower and the pod EST libraries (Supplementary Table 1).

### 3.2. Biochemical characterization of PTR

Recombinant PCBER-like proteins with six His residues at the N-terminal were expressed in *E. coli*, and PTR activity was assayed using (-)-medicarpin and NADPH. A peak of vestitol [retention time (Rt), 12.5 min] appeared on the HPLC of extracts from assays with the 10000×g supernatant of *E. coli* expressing PTR1–PTR4 (Supplementary Fig. 1). The chemical structure of the reaction product was unambiguously identified as (-)-vestitol by electron ionization mass spectrometry and ¹H-nuclear magnetic resonance (NMR) spectra (Supplementary methods and data) and HPLC on a chiral separation column. (+)-Dehydrosiconiferyl alcohol (PCBER substrate), (+)-pinoresinol (PLR substrate), and 2′-hydroxyformononetin (IFR substrate) were not accepted as substrates for PTR1–PTR4. The conversion of vestitol to medicarpin in the presence of NADP⁺, the reverse reaction of PTR, was not detected either under the assay condition described in Section 2.

The specific activity of each PTR isoform toward (-)-medicarpin was compared using the purified recombinant proteins. PTR2 showed the highest specific activity (67 nkatal/mg protein; relative activity, 100%), and the activity of PTR1 was 34% of that of PTR2. PTR3 and PTR4 showed 0.1% and 0.2% of the activity of PTR2, respectively. The stereospecificity of the PTR reactions was further explored by HPLC on a chiral separation column that clearly distinguishes (+)- and (-)-medicarpin (Rts, 18.5 and 23.5 min, respectively) and (+)- and (-)-vestitol (Rts, 17.0 and 25.5 min, respectively) (Fig. 2). When racemic medicarpin was reacted with the purified recombinant PTR1 and PTR2 proteins, (-)-medicarpin was consumed to produce (-)-vestitol, while (+)-medicarpin remained unconsumed (Fig. 2A). In contrast, both (-)- and (+)-vestitol were produced from (-)- and (+)-medicarpin in reactions with PTR3 and PTR4. It is thus concluded that PTR1 and PTR2 possess high specific activity with enantiospecificity toward the (-)-isomer, in contrast to PTR3 and PTR4, which have low activity without enantiospecificity.

### 3.3. PTR activity in GSH-treated L. japonicus seedlings

The reactivity of plant tissues with the pterocarpan substrate was examined. Cell-free extracts of GSH-treated *L. japonicus* seedlings exhibited PTR activity (65.5 pkatal/mg protein) and used only (-)-medicarpin as the substrate (Fig. 2B). This stereospecificity of the reaction is consistent with the accumulation of (-)-vestitol in GSH-treated *L. japonicus* seedlings (Fig. 2C). PTR1 and PTR2, both of which have the same enantiospecificity as the cell-free extracts, are therefore considered to be responsible for the production of (-)-vestitol in *L. japonicus*. Higher transcript levels of *PTR1* and *PTR2* than *PTR3* and *PTR4* in (-)-vestitol-producing *L. japonicus* seedlings (Shimada N. et al. unpublished) also support the major roles of PTR1 and PTR2 in the vestitol production in *L. japonicus*.

### 3.4. Molecular evolution and reaction mechanism of PTR

The phylogenetic tree composed of PTR, PCBER, and two other SDR family enzymes, IFR and PLR, in dicotyledonous plants shows that PTR and PCBER belong to the same branch, which is different from that of IFR and PLR, suggesting a close evolutionary relationship between PTR and PCBER (Fig. 3). It is likely that duplication of an ancestral PCBER gene has produced redundant genes, from which new genes encoding enzymes with the function of reducing the pterocarpan skeleton to yield isoflavans emerged in leguminous plants. A majority of

---

**Fig. 2. Chiral separation of enantiomers of medicarpin and vestitol for analysis of stereoselectivity of the reactions by the recombinant PTR isozymes (A) and with the cell-free extracts of *L. japonicus* seedlings (B), and of chirality of the vestitol contained in the seedlings (C). The chart on the far left in (A) shows the substrate before the reaction. Vestitol was recovered from the methanol extract of seedlings treated with GSH for 20 h by silica-gel TLC and analyzed (C). (-)-M, (-)-medicarpin; (+)-M, (+)-medicarpin; (-)-V, (-)-vestitol; (+)-V, (+)-vestitol.**

T. Akashi et al. / FEBS Letters 580 (2006) 5666–5670

![Diagram]

Fig. 3. The phylogenetic relationship of *L. japonicus* PTRs and related proteins of dicotyledonous plants. The parentheses indicate the plant species and GenBank accession number. Leucoanthocyanidin reductase (LAR) served as an outgroup. (+)-PLR and (-)-PLR are pinoresinol-lariciresinol reductase proteins that have enantiospecificity toward (+)- and (-)-pinoresinol, respectively.

leguminous plants including *L. japonicus* accumulate only (-)-enantiomers of pterocarpan and isoflavan phytoalexins like (-)-vestitol and (-)-medicarpin. It is tempting to assume that the more primitive type of PTRs are PTR3 and PTR4 both lacking enantiospecificity and having a little higher identity than PTR1 and PTR2 to PCBER that also lacks enantiospecificity to dehydrodiconiferyl alcohol substrate [12]. PTRs that perform stereoselective reaction like PTR1 and PTR2 may have subsequently emerged in plants that only produce (-)-pterocarpan and become more specialized for the synthesis of the phytoalexin with (-)-chirality. The mechanism for the acquisition of enantiospecificity awaits more detailed comparative studies on the kinetics of the reactions combined with the active site structures of enzyme proteins. In this respect, it may be noteworthy that the three-dimensional active site structures of PCBER and PLR [14] have partially explained the lack or presence of enantiospecificity toward the respective substrates. Now that the amino acid sequences of both enantiospecific and non-specific PTRs are available, clearer aspects about the factors governing the stereochemical course of the reaction is expected in the near future.

The strategy taken in this study, i.e., mining the EST library for homologs of known enzymes within the SDR family, resulted in the cloning of cDNAs encoding a new IFR (R7) and PLR-like proteins (R5 and R6), in addition to PTRs. The catalytic functions of R5 and R6 are now being examined, and future studies on lignan components and their biosynthetic controls in *L. japonicus* will be interesting. Also, it is surprising that no PCBER activity was displayed by PTR1-PTR4 and that no other PCBERs exist in the *L. japonicus* EST databases.

Now, the biosynthetic course of isoflavan skeleton and the enzymes involved are clarified at molecular levels. Undetectable levels of reverse reaction by recombinant PTRs in vitro apparently contrast to the reported interconversion of radiolabeled medicarpin and vestitol in alfalfa [4]. In fact, our preliminary experiments showed a weak activity of the conversion of vestitol to medicarpin in cell-free extracts of GSH-treated *L. japonicus* seedlings (0.5 pkatal/mg protein, 0.8% of PTR activity) as well as in 5-day-old alfalfa seedlings and elicited *G. echinata* cells (data not shown). However, medicarpin production from vestitol in these extracts was always accompanied by the formation of a large quantity of unknown compound, implicating

![Chemical Reaction Scheme]

Fig. 4. Putative reaction scheme of PTR.

that this is not a simple PTR reverse reaction but the reaction is mediated by another enzyme than PTR.

The identification of all cDNAs/genes encoding enzymes involved in the isoflavan pathway from two CoA-type precursors is now complete (see Fig. 1A), except for 7,2′-dihydroxy-4′-methoxyisoflavanol dehydratase (DMID) which is only biochemically characterized [22]. Regarding the reaction mechanism for PTR, a quinone methide intermediate is postulated for the PCBER reaction [12], and a similar quinoid intermediate is feasible for PTR (Fig. 4). The intermediate is also a possible intermediate of the DMID reaction, which produces the pterocarpan skeleton [23]. It is intriguing that DMID interacts physically with VR, which catalyzes the preceding reaction [23]. Interaction of enzyme proteins including PTR could be an important factor controlling the product distribution in the later steps of isoflavonoid biosynthesis.

**Acknowledgments:** This work was supported by a Grant-in-Aid for Scientific Research (C) (No. 15510183) from the Japan Society for the Promotion of Science, and also funded by the 21st Century Center of Excellence (COE) Program of Ministry of Education, Culture, Sports, Science and Technology of Japan.

## Appendix A. Supplementary data

Supplementary data associated with this article can be found, in the online version, at doi:10.1016/j.febslet.2006.09.016.

## References

[1] Dewick, P.M. (1993) Isoflavonoids in: The Flavonoids. Advances in Research since 1986 (Harborne, J.B., Ed.), pp. 117–238, Chapman & Hall, London.

[2] Dixon, R.A. (1999) Isoflavonoids: biochemistry, molecular biology, and biological functions in: Comprehensive Natural Products Chemistry (Sankawa, U., Ed.), Polyketides and other Secondary Metabolites Including Fatty Acids and their Derivatives, Vol. 1, pp. 773–823, Elsevier, Oxford.

[3] Aoki, T., Akashi, T. and Ayabe, S. (2000) Flavonoids of leguminous plants: structure, biological activity, and biosynthesis. J. Plant Res. 113, 475–488.

[4] Dewick, P. and Martin, M. (1979) Biosynthesis of pterocarpan and isoflavan phytoalexins in *Medicago sativa*: the biochemical interconversion of pterocarpans and 2′-hydroxyisoflavans. Phytochemistry 18, 591–596.

[5] Humphreys, J.M. and Chapple, C. (2000) Molecular ‘pharming’ with plant P450s. Trends Plant Sci. 5, 271–272.

[6] Akashi, T., Sawada, Y., Shimada, N., Sakurai, N., Aoki, T. and Ayabe, S. (2003) cDNA cloning and biochemical characterization of S-adenosyl-L-methionine: 2,7,4′-trihydroxyisoflavanone 4′-O-methyltransferase, a critical enzyme of the legume isoflavonoid phytoalexin pathway. Plant Cell Physiol. 44, 103–112.

[7] Akashi, T., Aoki, T. and Ayabe, S. (2005) Molecular and biochemical characterization of 2-hydroxyisoflavanone dehydratase. Involvement of carboxylesterase-like proteins in leguminous isoflavone biosynthesis. Plant Physiol. 137, 882–891.

[8] Pichersky, E. and Gang, D.R. (2000) Genetics and biochemistry of secondary metabolites in plants: an evolutionary perspective. Trends Plant Sci. 5, 439–445.

[9] Ober, D. (2005) Seeing double: gene duplication and diversification in plant secondary metabolism. Trends Plant Sci. 10, 444–449.

[10] Jornvall, H., Persson, B., Krook, M., Atrian, S., Gonzalez-Duarte, R., Jeffery, J. and Ghosh, D. (1995) Short-chain dehydrogenases/reductases (SDR). Biochemistry 34, 6003–6013.

[11] Xie, D.Y. and Dixon, R.A. (2005) Proanthocyanidin biosynthesis – still more questions than answers? Phytochemistry 66, 2127–2144.

[12] Gang, D.R., Kasahara, H., Xia, Z.Q., Vander Mijnsbrugge, K., Bauw, G., Boerjan, W., Van Montagu, M., Davin, L.B. and Lewis, N.G. (1999) Evolution of plant defense mechanisms. Relationships of phenylcoumaran benzyl ether reductases to pinoresinol-lariciresinol and isoflavone reductases. J. Biol. Chem. 274, 7516–7527.

[13] Koeduka, T., Fridman, E., Gang, D.R., Vassao, D.G., Jackson, B.L., Kish, C.M., Orlova, I., Spassova, S.M., Lewis, N.G., Noel, J.P., Baiga, T.J., Dudareva, N. and Pichersky, E. (2006) Eugenol and isoeugenol, characteristic aromatic constituents of spices, are biosynthesized via reduction of a coniferyl alcohol ester. Proc. Natl. Acad. Sci. USA 103, 10128–10133.

[14] Min, T., Kasahara, H., Bedgar, D.L., Youn, B., Lawrence, P.K., Gang, D.R., Halls, S.C., Park, H., Hilsenbeck, J.L., Davin, L.B., Lewis, N.G. and Kang, C. (2003) Crystal structures of pinoresinol-lariciresinol and phenylcoumaran benzyl ether reductases and their relationship to isoflavone reductases. J. Biol. Chem. 278, 50714–50723.

[15] Ayabe, S., Akashi, T. and Aoki, T. (2002) Cloning of cDNAs encoding P450s in the flavonoid/isoflavonoid pathway from elicited leguminous cell cultures. Methods Enzymol. 357, 360–369.

[16] Nakamura, K., Akashi, T., Aoki, T., Kawaguchi, K. and Ayabe, S. (1999) Induction of isoflavonoid and retrochalcone branches of the flavonoid pathway in cultured *Glycyrrhiza echinata* cells treated with yeast extract. Biosci. Biotechnol. Biochem. 63, 1618–1620.

[17] Shimada, N., Akashi, T., Aoki, T. and Ayabe, S. (2000) Induction of isoflavonoid pathway in the model legume *Lotus japonicus*: molecular characterization of enzymes involved in phytoalexin biosynthesis. Plant Sci. 160, 37–47.

[18] Thompson, J.D., Higgins, D.G. and Gibson, T.J. (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res. 22, 4673–4680.

[19] Perrière, G. and Gouy, M. (1996) WWW-query: an on-line retrieval system for biological sequence banks. Biochimie 78, 364–369.

[20] Asamizu, E., Nakamura, Y., Sato, S. and Tabata, S. (2000) Generation of 7137 non-redundant expressed sequence tags from a legume, *Lotus japonicus*. DNA Res. 7, 127–130.

[21] von Heimendahl, C.B., Schafer, K.M., Eklund, P., Sjoholm, R., Schmidt, T.J. and Fuss, E. (2005) Pinoresinol-lariciresinol reductases with different stereospecificity from *Linum album* and *Linum usitatissimum*. Phytochemistry 66, 1254–1263.

[22] Guo, L., Dixon, R.A. and Paiva, N.L. (1994) Conversion of vestitone to medicarpin in alfalfa (*Medicago sativa* L.) is catalyzed by two independent enzymes. Identification, purification, and characterization of vestitone reductase and 7,2′-dihydroxy-4′-methoxyisoflavanol dehydratase. J. Biol. Chem. 269, 22372–22378.

[23] Guo, L., Dixon, R.A. and Paiva, N.L. (1994) The ‘pterocarpan synthase’ of alfalfa: association and co-induction of vestitone reductase and 7,2′-dihydroxy-4′-methoxy-isoflavanol (DMI) dehydratase, the two final enzymes in medicarpin biosynthesis. FEBS Lett. 356, 221–225.
