
Appl Microbiol Biotechnol (2006) 73:1–14  
DOI 10.1007/s00253-006-0511-6  

MINI-REVIEW  

**The aureolic acid family of antitumor compounds: structure, mode of action, biosynthesis, and novel derivatives**

Felipe Lombó · Nuria Menéndez · José A. Salas · Carmen Méndez  

Received: 5 April 2006 / Revised: 15 May 2006 / Accepted: 16 May 2006 / Published online: 30 September 2006  
© Springer-Verlag 2006  

Abstract Members of the aureolic acid family are tricyclic polyketides with antitumor activity which are produced by different streptomycete species. These members are glycosylated compounds with two oligosaccharide chains of variable sugar length. They interact with the DNA minor groove in high-GC-content regions in a nonintercalative way and with a requirement for magnesium ions. Mithramycin and chromomycins are the most representative members of the family, mithramycin being used as a chemotherapeutic agent for the treatment of several cancer diseases. For chromomycin and durhamycin A, antiviral activity has also been reported. The biosynthesis gene clusters for mithramycin and chromomycin A₃ have been studied in detail by gene sequencing, insertional inactivation, and gene expression. Most of the biosynthetic intermediates in these pathways have been isolated and characterized. Some of these compounds showed an increase in antitumor activity in comparison with the parent compounds. A common step in the biosynthesis of all members of the family is the formation of the tetracyclic intermediate premithramycinone. Further biosynthetic steps (glycosylation, methylations, acylations) proceed through tetracyclic intermediates which are finally converted into tricyclic compounds by the action of a monooxygenase, a key event for the biological activity. Heterologous expression of biosynthetic genes from other aromatic polyketide pathways in the mithramycin producer (or some mutants) led to the isolation of novel hybrid compounds.

Keywords Streptomyces · Polyketides · Actinomycetes · Glycosylation

The aureolic acid family: isolation and structural characteristics

The first member of this family of compounds, mithramycin, was described in the 1950s and was also known as aureolic acid, plicamycin, antibiotic LA-7017, and PA-144 as a consequence of its isolation by different groups (Grundy et al. 1953; Sensi et al. 1958; Rao et al. 1962). It is produced by several actinomycetes, like *Streptomyces argillaceus* American Type Culture Collection (ATCC) 12956, *Streptomyces plicatus* ATCC 12957, *Streptomyces tanashiensis* ATCC 31053, and *Streptomyces atroolivaceus* ATCC 27627. The family also includes chromomycins (Sato et al. 1960), produced by *Streptomyces griseus* subsp. *griseus* ATCC 13273 and *Streptomyces cavourensis* ATCC 27732; olivomycins (Brazhnikova et al. 1962), produced by *Streptoverticillum cinnamonaeum*; chromocyclomycin (Blumauerova et al. 1976), produced by *S. atroolivaceus* UCH9 (Ogawa et al. 1998), produced by *Streptomyces* sp.; and durhamycin A (Jayasurya et al. 2002), produced by *Actinoplanes durhamensis* (Fig. 1).

All these compounds are glycosylated aromatic polyketides with an intense yellow color and fluorescence under UV light, which is responsible for the name of the family. With the exception of chromocyclomycin, which is a tetracyclic compound, the aglycons of this family show a tricyclic ring system fused to an unique dihydroxy-

Mithramycin

UCH9

Chromomycin A3

Durhamycin A

Olivomycin A

Chromocyclomycin

Fig. 1 Chemical structures of members of the aureolic acid family

methoxy-oxo-pentyl aliphatic side chain attached at C-3. In some cases a small alkyl residue (methyl, isobutyl) is attached at position C-7. Some initial suggestions postulating the involvement of two different polyketide chains to form these aglycon systems have been finally ruled out, and nowadays the involvement of only one polyketide synthase (PKS) and one polyketide chain has been unequivocally established (see below).

In all members of the family, two oligosaccharide chains are bound to the aromatic polyketide moiety. In the case of mithramycin, chromocyclomycin, chromomycins, and olivomycins, these chains contain two and three deoxysugars. There were several structural disagreements regarding the positions and linkages of the deoxysugars in the case of mithramycin. Its correct structure was first introduced in a DNA interaction paper (Sastry and Patel 1993) and later confirmed by NMR and mass spectrometry (Wohlert et al. 1999). UCH9 and durhamycin contain a tetrasaccharide and a mono- or disaccharide, respectively. All sugars belong to the 2,6-dideoxysugar family and they comprise different combinations of D-olivose, D-oliose, D-mycarose, L-chromose B, and O-methylated or O-acetylated derivatives. These deoxysugars are connected via β-(1,3) glycosidic bonds.

### Biological activities and mode of action

Initially, the members of this family of natural products were isolated due to their antibiotic activity against Gram-positive bacteria. However, they are not active against Gram-negative bacteria due to permeability problems. Their main pharmacological interest resides in their antitumor activity.

The members of this family interact with the DNA helix minor groove in regions with high GC content and in a nonintercalative way (Waring 1981; Katahira et al. 1998). This binding is carried out by complexes of dimers together with a Mg²⁺ ion. During these interactions, several H-bonds are created among the aglycon hydroxyl groups and the guanine amino protons (Sastry and Patel 1993). The deoxysugars are necessary for stabilizing this complex with
the DNA (Sastry et al. 1995; Keniry et al. 2000), and its structure influences the sequence specificity. Consequently, the acetyl and methyl groups in the chromomycin deoxy-sugars make these oligosaccharides less flexible, which induces higher DNA-sequence specificity than in the case of mithramycin, and a more stable minor groove binding (Majee et al. 1997; Chakrabarti et al. 2000–2001).

The interaction with the double helix causes a DNA-dependent inhibition on RNA synthesis, which gives this family of compounds a strong antitumor activity against a variety of cancer cell lines (Wakisaka et al. 1963; Ward et al. 1965; Ogawa et al. 1998). Based on this antitumor activity, mithramycin has found clinical application in the treatment of some cancers, such as testicular carcinoma (Du Priest and Fletcher 1973).

The specificity for GC-rich regions along the DNA makes these compounds good inhibitors of specific promoter regions, preventing the binding of regulatory proteins. This effect has been described for the *c-myc* and *c-Ha-ras* (Campbell et al. 1994), *c-myb* (Vigneswaran et al. 2001), and *MDR1* genes (Tagashira et al. 2000). Mithramycin binds at the C-fos-depending Sp1 regulatory regions, and therefore, it prevents transcription due to this transcriptional factor, generating a global inhibition mechanism (Ryuto et al. 1996).

Mithramycin also inhibits calcium resorption in osteoclasts, and it has been used for treating cancer-associated hypercalcemia processes (Hall et al. 1993). This effect is based on transcription regulation in these cells. The exact mechanism of action involves binding of the drug to the promoter regions of a gene, which is necessary for osteoclasts promotion, the *c-src* gene, abolishing the binding of Sp1 transcription factors (Remsing et al. 2003a).

Antiviral activity has also been described for some members of the family, as the inhibitory effect of durhamycin A on HIV Tat replication protein (Jayasuriya et al. 2002). Chromomycin also causes inhibition on the binding of the transcription factor Sp1 to its target sequences in the HIV-1 long terminal repeat regions, thus abolishing the activation of the HIV-1 provirus (Bianchi et al. 1997).

Some aureolic acids have been shown to prevent resistance to other antitumor agents by a number of mechanisms, including the down regulation of proteins such as MDR1 (Mir et al. 2003; Tagashira et al. 2000). Chromomycin and mithramycin are also potent inhibitors of neuronal apoptosis (Chatterjee et al. 2001). These two compounds also bind, in a Mg²⁺-independent manner, to the erythrocyte cytoskeletal protein spectrin with affinity constants comparable to those for the association of spectrin with other cytoskeletal proteins like F-actin or ankyrin (Majee and Chakrabarti 1995; Majee et al. 1999).

Aureolic acid biosynthesis gene clusters

Currently, two aureolic acid biosynthesis gene clusters have been isolated and characterized: those involved in the biosynthesis of mithramycin and chromomycin A₃. Both gene clusters have been sequenced and most of the genes have been mutated by insertional inactivation, and the accumulated intermediates (or shunt products) in these mutant strains have been purified and subjected to structural elucidation. Despite the high structural similarity between mithramycin and chromomycin A₃, the genetic organization of both gene clusters is highly different (Fig. 2), which favors the hypothesis of convergent evolution for the generation in both antitumor compounds, instead of divergent evolution from a common ancestor (Menéndez et al. 2004a).

The putative borders of the chromomycin cluster are genes coding for a cyclase and an aromatase. In the case of mithramycin, genes encoding regulatory and resistance functions would be the ends of the cluster. One surprising characteristic of the mithramycin gene cluster is the presence of a perfectly 241-bp repeated sequence at each end of the cluster. This could have implications in how the producer strain, *S. argillaceus*, acquired the gene cluster through evolution. This cluster could have been part of an extrachromosomal element in which a copy of this repeated sequence would be present. Through Campbell-type recombination to an identical repeated sequence that would have been present in the chromosome of an ancestor nonmithramycin-producing strain of *S. argillaceus*, the extrachromosomal element could have been incorporated into the *S. argillaceus* chromosome, and as a consequence, two repeated sequences are now flanking the mithramycin cluster (Lombó et al. 1999).

Formation of premithramycinone: a pivotal tetracyclic intermediate

Several studies have tried to elucidate the biosynthetic origin of the 20-carbon polyketide chain of this family of compounds. The initial incorporation experiments using ¹³C- and ¹⁸O-labeled acetate assumed that this polyketide backbone was the result of two, or maybe three, independent polyketide chains (Montanari and Rosazza 1990). Another possible explanation, based on incorporation studies using ¹⁸O₂, could be the putative presence of a tetracyclic intermediate which would be later cleaved at the fourth ring by an oxygenase, giving rise to the final aglycon (Rohr 1992). This was finally found to be the correct one and it was demonstrated in three different ways. In the first approach, the mithramycin PKS genes which determine the length of the nascent polyketide, the β-ketoacyl synthase alpha (*mtmP*) and β-ketoacyl synthase beta (*mtmK*), were
Fig. 2 Organization of the biosynthesis gene clusters for mithramycin (mtm, mtr) and chromomycin A3 (cmm, cmr). Accession numbers for the entire clusters are AJ578458 for chromomycin and X89899 for mithramycin

![Diagram](https://i.imgur.com/1kb.png)

amplified and expressed together in *Streptomyces coelicolor* CH999 with the actinorhodin acyl carrier protein and ketoreductase, giving rise to RM20b,c, which is a 20-carbon polyketide compound (Blanco et al. 1996). This implies that the product generated by the mithramycin minimal PKS is the result of the condensation of ten acetate subunits [one acetyl-coenzyme A (CoA) and nine malonyl-CoAs]. Second, a *Streptomyces lividans* clone expressing a fragment from the mithramycin cluster, which codes for the MtmX cyclase, the MtmP and MtmK β-ketoacyl synthases, the MtmS acyl carrier protein, and the MtmT1 ketoreductase, generated SEK15, another decaketide compound (Künzel et al. 1997). Finally, a mutant in the mithramycin producer organism affected in early glycosylation steps generated a tetracyclic intermediate, premithramycinone (PMC; Fig. 3) (Lombó et al. 1997; Rohr et al. 1998).

The first known intermediate for which evidence exists is a tricyclic anthrone (Lombó et al. 2000). This is the result of the joint action of the minimal PKS genes (*mtmPKS*), generating a linear decaketide (Blanco et al. 1996) which is aromatized in the first two rings by the MtmQ aromatase and cyclized at the third ring by the MtmY cyclase (Lombó et al. 1996). Then, the action of the MtmOII oxygenase introduces two hydroxyl groups (Prado et al. 1999a) and the MtmX cyclase carries out the cyclization at the fourth ring of this anthrone (Künzel et al. 1997). This generates the first tetracyclic intermediate, 4-demethylpremithramycinone (4-DMPC) (Lozano et al. 2000). This compound is methylated by the MtmMI O-methyltransferase, giving rise to PMC, which is the last nonglycosylated compound in the biosynthetic pathway (Fig. 3) (Lombó et al. 1997). Equivalent genes have been described in the chromomycin cluster (Table 1). PMC plays a pivotal role in the biosynthesis of all aureolic acid compounds. In spite of structural differences in late biosynthesis steps between members of the group, the early stages are the same, leading to the formation of this key intermediate PMC.

Glycosylated tetracyclic intermediates

Mithramycin and chromomycin A₃ share D-olivose and D-oliose, and differ in D-mycarose (in mithramycin) and L-chromose B (in chromomycin A₃). Biosynthesis of all deoxysugars involves four common steps (Fig. 4): (1) activation of D-glucose to thymidine diphosphate (dTDP)-D-glucose by the nucleotidyldiphosphate (NDP)-glucose synthases MtmD or CmmD; (2) the 4,6-dehydration to dTDP-4-keto-6-deoxy-D-glucose by the NDP-4,6-dehydratases MtmE or CmmE; (3) the 2,3-dehydration by the NDP-2,3-dehydratases MtmV or CmmV; and (4) the 3-ketoreduction of this intermediate generating dTDP-4-keto-2,6-dideoxy-D-glucose. This last enzymatic activity is represented in the chromomycin cluster by CmmW, but a gene encoding this function is not clearly present in the mithramycin cluster (see below) (Lombó et al. 1997; González et al. 2001; Menéndez et al. 2004a). From dTDP-4-keto-2,6-dideoxy-D-glucose, dTDP-D-olivose is generated by a 4-ketoreductase, which renders an equatorial hydroxyl group, CmmUII in the case of chromomycin. An equivalent enzyme has not been found in the mithramycin cluster, although MtmC could fulfill this role because mutants in MtmC produce a compound with a 4-keto-D-olivose (Remsing et al. 2002). dTDP-D-olivose is produced by a 4-ketoreductase, which renders a final axial hydroxyl group. In the mithramycin pathway, this should be MtmU, a protein with high similarity to 3-ketoreductases. Inactivation of this gene abolishes D-olivose biosynthesis in *S. argillaceus* (González et al. 2001). CmmUII is probably the equivalent gene in the chromomycin pathway.

Formation of D-mycarose and L-chromose B requires the C-3 methylation of the common intermediate dTDP-4-

1 acetate  
9 malonate  

MtmPKS  
CmmPKS  

MtmXY Q  
CmmXY Q  

MtmOII  
CmmOII  

4-DMPC  

MtmMI  
CmmMI  

PCA2  

MtmGV  
CmmGV  

CmmGIV  
CmmMII  

PMC  

PMCA2  

MtmGIII  

PMA1  

MtmGIV  
MtmMII  

PCA3  

CmmGII  

Prechromomycin B  

CmmONV  
CmmWl  

CmmMIII  
CmmA  

CHROMOMYCIN A₈  

Fig. 3 Proposed pathways for deoxysugars biosynthesis

keto-D-olivose by MtmC or CmmC, generating dTDP-4-  
keto-D-mycarose (González et al. 2001; Remsing et al.  
2002; Lombó et al. 2004; Menéndez et al. 2004a). In the  
case of dTDP-D-mycarose, the final 4-ketoreduction of this  
intermediate is carried out by MtmTIII (Remsing et al.  
2002). In the case of dTDP-L-chromose B, CmmF would  
produce a 5-epimerization on dTDP-4-keto-D-mycarose,  
and finally, the 4-ketoreductase CmmUIII would reduce the

Table 1 Protein comparison between chromomycin and mithramycin homologs

| Chromomycin pathway | Mithramycin pathway | Proposed function | % Identity/similarity |
|---------------------|---------------------|--------------------|-----------------------|
| CmmA                | –                   | O-acyltransferase  |                       |
| –                   | MtmA                | AdoMet synthetase/MTHF reductase |                       |
| CmmC                | MtmC                | NDP-C-methyltransferase | 39.2/52.6           |
| CmmD                | MtmD                | NDP-glucose synthase | 52.4/70.2            |
| CmmE                | MtmE                | NDP-4, 6-dehydratase | 61.2/71.1            |
| CmmF                | –                   | NDP-5-epimerase    |                       |
| CmmGI               | MtmGI               | Glycosyltransferase | 39.4/49.5            |
| CmmGII              | MtmGII              | Glycosyltransferase | 43.4/63.8            |
| CmmGIII             | MtmGIII             | Glycosyltransferase | 48.9/68.5            |
| CmmGIV              | MtmGIV              | Glycosyltransferase | 51.6/67.9            |
| –                   | MtmH                | Adenosylhomocysteinase |                       |
| CmmK                | MtmK                | β-Ketosynthase     | 62.6/73.2            |
| CmmLI               | –                   | Acyl-CoA ligase   |                       |
| CmmLII              | MtmL                | Acyl-CoA ligase   | 54.8/68.1            |
| CmmMI               | MtmMI               | O-methyltransferase | 49.0/66.7            |
| CmmMII              | MtmMII              | C-methyltransferase | 52.5/73.0            |
| CmmMIII             | –                   | O-methyltransferase |                       |
| CmmOI               | MtmOI               | Oxygenase         | 57.7/75.1            |
| CmmOII              | MtmOII              | Oxygenase         | 57.9/72.9            |
| –                   | MtmOIII             | Oxygenase         |                       |
| CmmOIV              | MtmOIV              | Oxygenase         | 58.7/78.3            |
| CmmP                | MtmP                | α-Ketosynthase    | 71.8/82.4            |
| CmmQ                | MtmQ                | Aromatase         | 50.5/66.8            |
| CmmRI               | MtmR                | Transcriptional activator | 37.6/59.1          |
| CmmRII              | MtrY                | Regulator         | 25.4/30.1            |
| CmmS                | MtmS                | Acyl carrier protein | 48.5/65.2           |
| CmmTI               | MtmTI               | Ketoreductase     | 54.2/68.3            |
| CmmTII              | MtmTII              | Ketoreductase     | 61.7/75.9            |
| –                   | MtmTIII             | NDP-4-ketoreductase |                       |
| CmmUI               | –                   | NDP-4-ketoreductase |                       |
| CmmUII              | –                   | NDP-4-ketoreductase |                       |
| CmmUIII             | –                   | NDP-4-ketoreductase |                       |
| CmmV                | MtmV                | NDP-2, 3-dehydratase | 49.8/62.1            |
| CmmW                | MtmU                | NDP-3-ketoreductase | 48.9/59.0            |
| CmmWI               | MtmW                | Ketoreductase     | 63.5/75.4            |
| CmmX                | MtmX                | Cyclase           | 50.2/62.8            |
| CmmY                | MtmY                | Cyclase           | 65.4/77.4            |
| –                   | MtmZ                | Thioesterase      |                       |
| CmrA                | MtrA                | ATP-binding protein | 54.2/70.1            |
| CmrB                | MtrB                | Membrane protein  | 47.9/71.8            |
| CmrX                | MtrX                | UV-repair system  | 55.8/67.6            |

MTHF methenyltetrahydrofolate synthetase

4-keto group, generating dTDP-L-chromose B (Menéndez et al. 2004a).

Incorporation of the five deoxysugars to the corresponding intermediates is catalyzed by specific glyco-syltransferases (Fig. 3). Glycosyltransferases MtmGIV or CmmGIV attach the first sugar to the tetracyclic compound PMC, rendering premithramycin A1 (Blanco et al. 2000; González et al. 2001; Menéndez et al. 2006). MtmGIII and CmmGIII transfer the second deoxysugar to this intermediate, elongating the monosaccharide chain with a D-oliose or a D-olivose, respectively. This generates premithramycin

A2 or prechromomycin A2 (Blanco et al. 2000; Menéndez et al. 2006). The third deoxysugar of the trisaccharide is a branched chain sugar which is most probably connected by the action of MtmGIV or CmmGIV. In the case of mithramycin, the glycosyltransferase would rather incorporate 4-keto-D-mycarose, which will be reduced afterwards by the action of the MtmTIII ketoreductase, generating the D-mycarose moiety present in 9-demethyl-premithramycin A3 (Remsing et al. 2002). In the case of chromomycin, the glycosyltransferase would transfer L-chromose B, rendering 9-demethyl-prechromomycin A3 (Menéndez et al. 2006).

Glucose-1-P

MtmD
CmmD

MtmE
CmmE

O
H₃C
HO
OH
ONDP

MtmV
CmmV

Mtm?
CmmW

O
H₃C
HO
OH
ONDP

MtmC
CmmC

O
H₃C
OH
ONDP

MtmTIII

H₃C
HO
OH
ONDP

dTDP-D-Mycarose

MtmC?
CmmUI

MtmU
CmmUII

HO
H₃C
HO
ONDP

OH
H₃C
HO
ONDP

CmmF

H₃C
O
HO
ONDP

CmmUIII

HO
H₃C
HO
ONDP

dTDP-L-Chromose B

dTDP-D-Olivose
dTDP-D-Oliose

Fig. 4 Proposed pathways for mithramycin and chromomycin A3 biosynthesis

The last two deoxysugars incorporated during the biosynthesis form part of the disaccharide moiety. In the mithramycin pathway, initial experiments showed the transfer by the glycosyltransferases MtmGI and MtmGII of both deoxysugars to the corresponding intermediate as a disaccharide (Fernández et al. 1998). However, recent experiments have shown that they are incorporated sequentially, the first one by MtmGI and the second by MtmGII (Nur-e-Alam et al. 2005). In the case of chromomycin, CmmGII and CmmGI transfer the first and the second D-oliose moieties, respectively, in a sequential manner (Menéndez et al. 2006). These last glycosylation steps generate the tetracyclic compounds premithramycin B and prechromomycin B, respectively.

Conversion of tetracyclic intermediates into active tricyclic compounds

As it has been previously mentioned, at the early stages the biosynthesis of aureolic acid compounds proceeds through tetracyclic tetracycline-like intermediates. However, the final products of the biosynthesis (with the only exception of chromocyclomycin) have a tricyclic aromatic structure with a pentyl side chain attached at position C3. This conversion from tetracyclic into tricyclic compounds takes place through the oxidative cleavage of the fourth ring (Fig. 3), and it represents a key step to convert relatively inactive intermediates into the final bioactive compounds. This requires the action of a monooxygenase: the products of two orthologous genes were identified in the mithramycin and chromomycin clusters, named *mtmOIV* and *cmnOIV*, respectively (Prado et al. 1999b; Menéndez et al. 2004a). Experimental evidence of their involvement in the breakage of the fourth ring was obtained through insertional inactivation of MtmOIV and the isolation of premithramycin B, a fully glycosylated tetracyclic intermediate (Prado et al. 1999b). In vivo and in vitro activity of the isolated compound was assayed and compared to the parental drug mithramycin. Interestingly, premithramycin B did not show detectable antibiotic or antitumor activity, thus suggesting that the opening of the fourth ring becomes essential to generate biologically active compounds. Biochemical confirmation of the role assigned to MtmOIV came from the purification of this protein and the characterization of the enzymatic reaction catalyzed by this enzyme (Rodriguez et al. 2003; Gibson et al. 2005). In addition, MtmOIV has also been crystallized (Wang et al. 2005). From incorporation experiments with isotope-labeled precursors, it was suggested that this key oxidative cleavage proceeds through a Baeyer–Villiger oxidation mechanism (Rohr et al. 1999). The oxygenase inserts an oxygen and opens the resulting lactone. In addition, the loss of one carbon unit, presumably as CO₂, must occur, and this must take place spontaneously (Rohr et al. 1999; Prado et al. 1999b). As a result of this reaction, a 4′-keto-mithramycin is generated, which is further reduced to form the 4′-hydroxyl group of the thereby formed pentyl side chain. This biosynthetic step is catalyzed in mithramycin and chromomycin pathways by MtmW and CmmWI, respectively. These enzymes exhibit similarity to the aldo/
keto reductase protein family. Insertional inactivation of the corresponding genes (mtmW and cmmWI) leads to the generation of mutants that accumulated new derivatives differing in the length and structure of their side chains (Remsing et al. 2003b; Menéndez et al. 2004a). The major compounds, mithramycin SK and chromomycin SK, show high antitumor activity, with mithramycin SK showing values up to nine times higher than those of its parent compound, mithramycin. The rest of the aureolic acid analogs obtained by inactivating the final ketoreduction step (named chromomycin SA, mithramycin SA, and chromomycin SDK) also exhibit high antitumor activity values. These results clearly showed that the 3-aliphatic side chain could be an interesting target to modify to generate novel aureolic acid compounds with potential therapeutic applications.

**Tailoring modifications**

During aureolic acid biosynthesis, several tailoring modifications (methylations and acetylations) take place at different steps. Some of them affect the chromophoric structure while others involve modification of the deoxysugar moieties (Fig. 3). Two methylation events affecting the chromophoric structures take place. The first one is the methylation of 4-DMPC, rendering PMC. In the mithramycin pathway, it was demonstrated that this methylation event is catalyzed by MtmMI, a S-adenosyl-L-methionine-dependent methyltransferase (SAM)-dependent methyltransferase. Insertional inactivation of the mtmMI gene leads to the isolation of 4-DMPC, and further in vitro assays performed with cell-free extracts of a recombinant strain expressing MtmMI showed that this enzyme is responsible for the addition of the methyl group to the substrate 4-DMPC to render PMC (Lozano et al. 2000). It is important to point out that this O-methylation step is essential to continue with the biosynthesis process because the absence of this methyl group completely abolishes the subsequent biosynthetic steps that render the final drug. An equivalent gene was also found in the chromomycin cluster (cmmMI).

A second methylation event that modifies the architecture of the polyketide moiety has also been characterized in the mithramycin biosynthetic pathway. This modification affects the C7 position of the aglycon (C9 in tetracyclic intermediates). The presence of this group is not common to all the aureolic acid members. It does not exist in olivomycin, and it is substituted by an isobutyl in the case of UCH9 and durhamycin. A methyltransferase gene, mtmMII, was identified in the mithramycin cluster to be responsible for this biosynthetic step. Insertional inactivation of mtmMII yielded a mutant that accumulated several compounds lacking the corresponding methyl group. One of these compounds, 9-demethyl-premithramycin A3, is a

tetracyclic intermediate partially glycosylated with only the trisaccharide chain. Subsequent in vitro methylation assays demonstrated that this compound was the natural substrate of the MtmMII enzyme (Lozano et al. 2000). However, the mtmMII-minus mutant also accumulated as the major product 7-demethyl-mithramycin (7-DMTM), a final side compound that only differs from the original drug in the absence of the C7 methyl group. Interestingly, this novel derivative lacks significant biological activity when tested against several tumor cell lines. According to these data, the C-methylation event catalyzed by MtmMII is not essential to complete mithramycin biosynthesis because the rest of the biosynthetic steps can take place over intermediates lacking this methyl group. In contrast, the biological activity of the molecule was revealed to be strictly dependent on the presence of this methyl group because 7-DMTM did not show any activity. Analysis of the interactions between mithramycin, 7-DMTM, and DNA revealed that the presence of the methyl group at C7 increases drug–DNA binding affinity by more than 60 times (Rodriguez et al. 2004). Within the aureolic acid group members, it seems that there is a direct close relationship between the size of the substituent chemical group at C7 position and the length of the C2-oligosaccharide present in the corresponding molecule. This correlation suggests that the C7 group plays an important role in facilitating an optimal interaction of third sugar in the trisaccharide chain along the minor groove of DNA, which is essential for high affinity binding and sequence selectivity (Rodriguez et al. 2004). A homologous counterpart of mtmMII, named cmmMII, was also found in the chromomycin gene cluster.

In addition to the methylations on the polyketide structure, chromomycin (and olivomycin and durhamycin) also undergo several tailoring modifications of the deoxysugar moieties, one methylation and two acetylations, which take place as final steps in the biosynthesis (Menéndez et al. 2004b). The O-methylation at the C4 position of D-oliose (second sugar in disaccharide) is catalyzed by the cmmMIII gene product. Inactivation of this gene generated a mutant that accumulated three unmethylated derivatives containing all deoxysugars but differing in the acylation pattern. The major product, demethyl-chromomycin A₃, was found to be a substrate of the methyltransferase, as determined by its bioconversion into chromomycin A₃ after feeding this compound to a Staphylococcus albus strain expressing the cmmMIII gene. Bioactivity testing of these unmethylated compounds revealed that they have potent antibiotic and antitumor activity, similar to the parental drug, thus indicating that the presence of this methoxy group is not crucial to confer high biological activity to these compounds. In the case of the acyl substituents that decorate two of the chromomycindeoxysugars (sugars A and E), it has been shown that a single enzyme (CmmA) is responsible for both acetyl transfer reactions (Menéndez et al. 2004b). Insertional inactivation of the *cmmA* gene resulted in a mutant accumulating a dideacetylated chromomycin derivative. Bioconversion experiments showed that the dideacetylated compound was converted into chromomycin in a two-step reaction through a 4E-monoacetylated intermediate. Interestingly, CmmA was predicted to be located in the cell membrane, a hypothesis finally proven by performing in vitro assays using cell membrane fractions. This fact strongly suggests that these two acetylation reactions constitute the final steps in chromomycin biosynthesis, probably associated with the transport of the drug across the cell membrane. It has to be pointed out that the dideacetylated intermediate shows a very important decrease in the biological activity compared to chromomycin A₃. According to these data, the final acetylations could constitute a very important event for self-protection of the producer organism because all biosyntheses proceed through relatively inactive compounds, only generating a fully active drug when the compound is being exported out of the cell.

### Regulation and resistance

The aureolic acid biosynthetic clusters contain pathway-specific regulatory genes that control the expression of the structural genes. Thus, in mithramycin and chromomycin pathways, two genes have been identified (*mtmR* and *cmmR*, respectively) coding for proteins similar to members of the transcriptional activators Streptomyces antibiotic regulatory protein family. The generation of a mutant lacking the *mtmR* gene in *S. argillaceus* completely abolished the production of mithramycin or any other intermediate. Experimental evidence supporting the role of a positive regulator for MtmR came from its expression using a high-copy-number vector in *S. argillaceus*, which caused a 16-fold increase in mithramycin production. In addition, *mtmR* was able to complement a mutation in the actinorhodin-specific activator *actII-orfIV* gene, and it also activated actinorhodin biosynthesis in *S. lividans* (Lombó et al. 1999).

In addition, chromomycin and mithramycin pathways also contain other regulatory elements. Within the chromomycin gene cluster, a gene coding for a putative regulator has been identified, named *cmmRII*. The deduced protein contains a PadR domain, which is characteristic for negative regulators. Insertional inactivation of *cmmRII* generated a strain in which production started earlier, and the yields were higher than in the wild-type strain. This result suggests that CmmRII acts as a transcriptional repressor in chromomycin biosynthesis. CmmRII is located immediately upstream of two genes, *cmrA* and *cmrB*, which code for proteins showing similarity to members of the ATP-binding cassette (ABC) transporter superfamily, particularly to Streptomyces systems involved in antibiotic efflux. Noticeably, in the mithramycin gene cluster, there is a gene, *mtrY*, also located immediately upstream of two genes (*mtrA* and *mtrB*) also encoding for an ABC transporter. The product of *mtrY* shows some similarity to *cmmRII* product (25.4% identity). However, in contrast to the *cmmRII*-minus mutant, the inactivation of *mtrY* renders a mutant showing lower levels of production, which suggests that this protein could behave like a positive regulator in mithramycin biosynthesis (Garcia-Bernardo et al. 2000).

MtrA and MtrB constitute the main resistant determinant to mithramycin, as it has been shown that they are capable of conferring high levels of resistance to this drug (>100 μg/ml) when expressed in *S. albus* (Fernández et al. 1996). Additionally, a second resistant determinant has been identified in the mithramycin cluster. This determinant is encoded by *mtrX*, a gene of which deduced product resembles UvrA-like proteins from excision systems involved in DNA repair. MtrX confers only low levels of resistance to mithramycin, and it has been proposed that it could play a role in repairing minor DNA damages caused by mithramycin molecules that are not removed from the cytoplasm by the ABC transporter (Garcia-Bernardo et al. 2000). In the chromomycin cluster, in addition to the ABC transporter genes *cmrA* and *cmrB*, an equivalent gene to *mtrX* (named *cmrX*) has been identified. However, analysis of the resistance mechanisms shows significant differences compared to mithramycin: (1) The expression of the chromomycin transporter in the producer organism is inducible by the final product. (2) High level of resistance to chromomycin is only obtained by coexpressing the ABC transporter and the *cmrX* genes. In this case, CmrX plays a more important role in resistance, probably acting in a cooperative way with the transporter. (3) As it has been previously mentioned, it seems that, in the chromomycin pathway, a deacetylated intermediate is exported instead of the final drug. This is supported by the transmembrane location of the acetyltransferase CmmA (Menéndez et al. 2004b). It has been shown that the transport system formed by CmrA and CmrB only confers low levels of resistance to chromomycin (<20 μg/ml), but it efficiently exports the deacetylated intermediate. The secretion of a relative inactive compound and its activation out of the cell can represent an evolutionary advantage to the producer strain to survive during antibiotic biosynthesis.

In the case of the mithramycin cluster, two additional genes have been found (*mtmA* and *mtmH*) that apparently are involved in the regulation of mithramycin biosynthesis. These genes code for proteins which seem to be part of an
activated methyl cycle. MtmA is a fusion protein of an incomplete S-adenosylmethionine synthetase and a methylenetetrahydrofolate reductase, and MtmH resembles adenosylhomocysteines. Based on these similarities, MtmA and MtmH were initially proposed as candidates to generate SAM, the cofactor used by the methyltransferases MtmMI and MtmMII. However, the corresponding *mtmAH-minus* mutant was not affected in the methylation profile, and, surprisingly, it produces mithramycin at higher yields. According to these data, *mtmA* and/or *mtmH* could play a role in the negative control of mithramycin biosynthesis.

### Generation of hybrid compounds

The knowledge generated within the mithramycin biosynthetic pathway has allowed the design of combinatorial biosynthesis experiments to generate novel hybrid compounds. In some cases, mithramycin genes contributed to modifying other aromatic polyketide routes, whereas in others, genes from different biosynthesis gene clusters were introduced into selected *S. argillaceus* hosts, altering the mithramycin pathway and therefore leading to the formation of new derivatives.

Introduction of genes encoding the mithramycin minimal PKS (*mtmPKS*), along with the putative cyclase *mtmX* and the ketoreductase *mtmTI*, into the producer strain of tetracenomycin C, *Streptomyces glaucescens* Tü49, was expected to modify some of the tetracyclic tetracenomycin C intermediates. In fact, a new hybrid compound, tetracenomycin M (Fig. 5a) was generated (Künzel et al. 1997). Its structure revealed that this compound had suffered a fourth ring closure involving an intramolecular aldol addition, quite different from the typical intramolecular aldol condensation of tetracenomycins (Shen and Hutchinson 1993a). Formation of tetracenomycin M threw light on the function of MtmX, which was proposed to be the mithramycin fourth ring cyclase responsible for this aldol addition (Künzel et al. 1997).

Some tailoring enzymes as oxygenases have enough substrate flexibility to be used as appropriate tools in combinatorial biosynthesis. The four-ring intermediate

![Figure 5](https://i.imgur.com/yourimageurl.png)

**Fig. 5** Novel antitumor compounds generated by combinatorial biosynthesis. **a** Tetracenomycin M, produced by expressing the *mtmPKS* (minimal PKS) and the *mtmX* cyclase and *mtmTI* ketoreductase in the tetracenomycin C producer *S. glaucescens* Tü49. **b** PMC H, produced by expressing the *tcmH* monooxygenase from the tetracenomycin cluster in *S. argillaceus* M7D1. **c** Formation of glycosylated premithramycin derivatives by expressing the nogalamycin minimal PKS genes (*snoABC*), the *snoaD* ketoreductase, and the *snoaE* aromatase in *S. argillaceus*. **d** Formation of glycosylated PMCs by expressing *urdGT2* (glycosyltransferase from the urdamycin cluster) alone or together with *lanGT1* (glycosyltransferase from the landomycin cluster) in *S. argillaceus* M3G4. **e** Formation of novel glycosylated tetracenomycins by expressing *cos16F4* in *S. argillaceus*

tetracenomycin F1 is the substrate for the monooxygenase TcmH during tetracenomycin C biosynthesis (Shen and Hutchinson 1993b). This compound shows some structural similarity to PMC, a tetracyclic biosynthetic intermediate in the mithramycin pathway. Based on this, an experiment was rationally designed to try to modify PMC with TcmH monooxygenase. The *tcmH* gene was expressed in the mutant strain *S. argillaceus* M7D1, which accumulates PMC, resulting in the formation of a new hybrid antitumor compound, PMC H (Fig. 5b; Lombó et al. 2000). This new compound resulted from the spontaneous cyclization of an anthraquinone, derived from an early and unstable tricyclic anthrone PMC intermediate, which is oxygenated at the second ring by TcmH, generating a quinone system.

By altering the polyketide structure, it is possible to modify the glycosylation pattern. In this way, by expressing several genes (PKS, aromatase, ketoreductase) involved in the biosynthesis of the polyketide moiety of nogalamycin in *S. argillaceus*, three new glycosylated hybrid compounds were generated (Fig. 5c; Kantola et al. 2000). All these compounds share the premithamycinone aglycon lacking a hydroxyl group at C-8, resulting from the action of the nogalamycin ketoreductase and aromatase together with the mithramycin aglycon genes. This new aglycon gets glycosylated at position 12a (as in wild-type *S. argillaceus*) with oligosaccharides of different lengths containing the natural mithramycin deoxysugars: D-olivosyl, D-olivosyl-D-oliosyl, and D-olivosyl-D-oliosyl-D-mycarosyl. Further glycosylation toward a fully glycosylated mithramycin was impossible because the elimination of the C-8 hydroxyl group in this aglycon by the action of the nogalamycin ketoreductase/aromatase abolished the disaccharide binding position (Kunnari et al. 2002).

Making use of the known substrate flexibility of some glycosyltransferases, it is possible to alter the glycosylation profile of bioactive compounds. UrdGT2 is a flexible glycosyltransferase responsible for the C-binding of a D-olivose moiety to the urdamycin aglycon (Faust et al. 2000). Precursors of this tetracyclic angular aglycon share some common features with the linear mithramycin intermediate PMC. A plasmid containing the *urdGT2* gene was introduced into a *S. argillaceus* mutant lacking all the mithramycin glycosyltransferases (Prado et al. 1999b), and also in mutant M3G4, lacking only the first PMC glycosyltransferase, *mtmGIV* (Blanco et al. 2000). This recombinant strain produced two new hybrid compounds which contained either D-olivose or D-mycarose attached to the C-9 position of PMC (or its 4-demethyl precursor) by a C—C bond (Fig. 5d; Trefzer et al. 2002). This demonstrated that UrdGT2 was flexible enough to attach D-mycarose, a branched-chain deoxysugar moiety very different with respect to its natural substrate, D-olivose. Furthermore, UrdGT2 was able to recognize a linear polyketide as

aglycon, although its natural substrate is an angucycline (Künzel et al. 1999). One of these two new compounds, 9-C-olivosyl-PMC, was further modified by expressing the landomycin glycosyltransferase gene *lanGT1* together with *urdGT2*, into these two *S. argillaceus* mutants. LanGT1 is responsible for the attachment of the second D-olivose moiety of the trisaccharide during landomycin biosynthesis (Trefzer et al. 2001). As a consequence of this experiment, a novel hybrid compound was generated, in which the PMC aglycon contained, attached at position C-9, a diolivosyl moiety (Fig. 5d; Trefzer et al. 2002).

A last example of the possibilities of combinatorial biosynthesis consists in the use of the mithramycin deoxy-sugar biosynthetic machinery and the flexible glycosyltransferase ElmGT of the elloramycin cluster to modify the glycosylation pattern of this antitumor compound. Cosmid 16F4 (Decker et al. 1995) is a cosmid containing the genes for the generation of the elloramycin aglycon (8-demethyl-tetracenomycin C) and the glycosyltransferase ElmGT which attaches L-rhamnose to this aglycon. Expression of cos16F4 in the mithramycin producer led to the generation of three new hybrid compounds, 8-demethyl-8-β-D-olivosyl-tetracenomycin C, 8-demethyl-8-β-D-mycarosyl-tetracenomycin C, and 8-demethyl-8-β-D-diolivosyl-tetracenomycin C (Fig. 5e; Wohlert et al. 1998; Blanco et al. 2001). In this case, the elloramycin aglycon gets glycosylated at its natural position, but by three different sugar moieties, including a disaccharide, which are synthesized by the mithramycin producer.

Acknowledgements The authors wish to thank all the people in their laboratory involved in research on aureolic acid drugs. Research at the authors’ laboratory has been supported by grants from the Spanish Ministry of Education and Science (BMC2002-03599 and BIO2005-04115 to C. Méndez) and by a grant from the Plan Regional de Investigación del Principado de Asturias (GE-MEDO1-05 to J.A. Salas). We thank Obra Social Cajastur for financial support to F. Lombó.

References

Bianchi N, Rutigliano C, Passadore M, Tomassetti M, Pippo L, Mischiati C, Feriotto G, Gambari R (1997) Targeting of the HIV-1 long terminal repeat with chromomycin potentiates the inhibitory effects of a triplex-forming oligonucleotide on Spl-DNA interactions and in vitro transcription. Biochem J 326:919–927

Blanco G, Fu H, Méndez C, Khosla C, Salas JA (1996) Deciphering the biosynthetic origin of the aglycone of the aureolic acid group of anti-tumor agents. Chem Biol 3:193–196

Blanco G, Fernández E, Fernández MJ, Braña AF, Weissbach U, Künzel E, Rohr J, Méndez C, Salas JA (2000) Characterization of two glycosyltransferases involved in early glycosylation steps during biosynthesis of the antitumor polyketide mithramycin by *Streptomyces argillaceus*. Mol Gen Genet 262:991–1000

Blanco G, Patallo EP, Braña AF, Trefzer A, Bechthold A, Rohr J, Méndez C, Salas JA (2001) Identification of a sugar

flexible glycosyltransferase from *Streptomyces olivaceus*, the producer of the antitumor polyketide elloramycin. Chem Biol 8:253–263

Blumauerova M, Lipavska H, Stajner K, Vanek Z (1976) The study of variability and strain selection in *Streptomyces atroolivaceus*. III. Isolation and preliminary characteristics of mutants impaired in the biosynthesis of mithramycin. Folia Microbiol (Praha) 21:285–293

Brazhnikova MG, Krugliak EB, Kovsharova IN, Konstantinova NV, Proshliakova VV (1962) Isolation, purification and study on some physico-chemical properties of a new antibiotic olivomycin. Antibiotiki 7:39–44

Campbell VW, Davin D, Thomas S, Jones D, Roesel J, Tran-Patterson R, Mayfield CA, Rodu B, Miller DM, Hiramoto RA (1994) The G-C specific DNA binding drug, mithramycin, selectively inhibits transcription of the C-MYC and C-HA-RAS genes in regenerating liver. Am J Med Sci 307:167–172

Chakrabarti S, Bhattacharyya D, Dasgupta D (2000–2001) Structural basis of DNA recognition by anticancer antibiotics, chromomycin A(3), and mithramycin: roles of minor groove width and ligand flexibility. Biopolymers 56:85–95

Chatterjee S, Zaman K, Ryu H, Conforto A, Ratan RR (2001) Sequence-selective DNA binding drugs mithramycin A and chromomycin A3 are potent inhibitors of neuronal apoptosis induced by oxidative stress and DNA damage in cortical neurons. Ann Neurol 49:345–354

Decker H, Rohr J, Motamedi H, Zahner H, Hutchinson CR (1995) Identification of *Streptomyces olivaceus* Tu 2353 genes involved in the production of the polyketide elloramycin. Gene 166: 121–126

Du Priest RW Jr, Fletcher WS (1973) Chemotherapy of testicular germinal tumors. Oncology 28:147–163

Faust B, Hoffmeister D, Weitnauer G, Westrich L, Haag S, Schneider P, Decker H, Kunzel E, Rohr J, Bechthold A (2000) Two new tailoring enzymes, a glycosyltransferase and an oxygenase, involved in biosynthesis of the angucycline antibiotic urdamycin A in *Streptomyces fradiae* Tu 2717. Microbiology 146:147–154

Fernández E, Lombó F, Méndez C, Salas JA (1996) An ABC transporter is essential for resistance to the antitumor agent mithramycin in the producer *Streptomyces argillaceus*. Mol Gen Genet 251:692–698

Fernández E, Weissbach U, Sánchez-Reillo C, Braña AF, Méndez C, Rohr J, Salas JA (1998) Identification of two genes from *Streptomyces argillaceus* encoding glycosyltransferase involved in transfer of a disaccharide during biosynthesis of the antitumor drug mithramycin. J Bacteriol 180:4929–4937

Garcia-Bernardo J, Braña AF, Méndez C, Salas JA (2000) Insertional inactivation of *mtrX* and *mtrY* genes from the mithramycin gene cluster affects production and growth of the producer organism *Streptomyces argillaceus*. FEMS Microbiol Lett 186:61–65

Gibson M, Nur-e-alam M, Lipata F, Oliveira MA, Rohr J (2005) Characterization of kinetics and products of the Baeyer-Villiger oxygenase MtmOIV, the key enzyme of the biosynthetic pathway toward the natural product anticancer drug mithramycin from *Streptomyces argillaceus*. J Am Chem Soc 127:17594–17595

González A, Remsing LL, Lombó F, Fernández MJ, Prado L, Braña AF, Kunzel E, Rohr J, Méndez C, Salas JA (2001) The *mtmVUC* genes of the mithramycin gene cluster in *Streptomyces argillaceus* are involved in the biosynthesis of the sugar moieties. Mol Gen Genet 264:827–835

Grundy WE, Goldstein AW, Rickher JC, Hanes ME, Warren HB, Sylvester JC (1953) Aureolic acid, a new antibiotic. I. Microbiological studies. Antimicrob Chemother 3:1215–1221

Hall TJ, Schaeublin M, Chambers TJ (1993) The majority of osteoclasts require mRNA and protein synthesis for bone resorption in vitro. Biochem Biophys Res Commun 195:1245–1253

Jayasuriya H, Lingham RB, Graham P, Quamina D, Herranz L, Genilloud O, Gagliardi M, Danzeisen R, Tomassini JE, Zink DL, Guan Z, Singh SB (2002) Durhamycin A, a potent inhibitor of HIV Tat transactivation. J Nat Prod 65:1091–1095

Kantola J, Kunnari T, Hautala A, Hakala J, Ylihonko K, Mantsala P (2000) Elucidation of anthracyclinone biosynthesis by stepwise cloning of genes for anthracyclines from three different *Streptomyces* spp. Microbiology 146:155–163

Katahira R, Katahira M, Yamashita Y, Ogawa H, Kyogoku Y, Yoshida M (1998) Solution structure of the novel antitumor drug UCH9 complexed with d(TTGGCCAA)2 as determined by NMR. Nucleic Acids Res 26:744–755

Keniry MA, Owen EA, Shafer RH (2000) The three-dimensional structure of the 4:1 mithramycin:d(ACCCGGGT)(2) complex: evidence for an interaction between the E saccharides. Biopolymers 54:104–114

Kunnari T, Klika KD, Blanco G, Méndez C, Mäntsälä P, Hakala J, Sillanpää R, Tähtinen P, Salas JA, Ylihonko K (2002) Hybrid compounds generated by the introduction of a nogalamycin-producing plasmid into *Streptomyces argillaceus*. J Chem Soc Perkin Trans 1:1818–1825

Künzel E, Wohlert SE, Beninga C, Haag S, Decker H, Hutchinson CR, Blanco G, Mendez C, Salas JA, Rohr J (1997) Tetracenomycin M, a novel genetically engineered tetracenomycin resulting from a combination of mithramycin and tetracenomycin biosynthetic genes. Chem Eur J 3:1675–1678

Künzel E, Faust B, Oelkers C, Weissbach U, Bearden DW, Weitnauer G, Westrich L, Bechthold A, Rohr J (1999) Inactivation of the *urdGT2* gene, which encodes a glycosyltransferase responsible for the C-glycosylation of activated D-olivose, leads to formation of the novel urdamycins I, J, and K. J Am Chem Soc 121:11058–11062

Lombó F, Blanco G, Fernández E, Méndez C, Salas JA (1996) Characterization of *Streptomyces argillaceus* genes encoding a polyketide synthase involved in the biosynthesis of the antitumor mithramycin. Gene 172:87–91

Lombó F, Siems K, Braña AF, Méndez C, Bindseil K, Salas JA (1997) Cloning and insertional inactivation of *Streptomyces argillaceus* genes involved in the earliest steps of biosynthesis of the sugar moieties of the antitumor polyketide mithramycin. J Bacteriol 179:3354–3357

Lombó F, Braña AF, Méndez C, Salas JA (1999) The mithramycin gene cluster of *Streptomyces argillaceus* contains a positive regulatory gene and two repeated DNA sequences that are located at both ends of the cluster. J Bacteriol 181:642–647

Lombó F, Kunzel E, Prado L, Braña AF, Bindseil KU, Frevert J, Bearden D, Méndez C, Salas JA, Rohr J (2000) The novel hybrid antitumor compound premithramycinone H provides indirect evidence for a tricyclic intermediate of the biosynthesis of the aureolic acid antibiotic mithramycin. Angew Chem Int Ed Engl 39:796–799

Lombó F, Gibson M, Greenwell L, Braña AF, Rohr J, Salas JA, Méndez C (2004) Engineering biosynthetic pathways for deoxy-sugars: branched-chain sugar pathways and derivatives from the antitumor tetracenomycin. Chem Biol 11:1709–1718

Lozano MJ, Remsing LL, Quirós LM, Braña AF, Fernández E, Sánchez C, Méndez C, Rohr J, Salas JA (2000) Characterization of two polyketide methyltransferases involved in the biosynthesis of the antitumor drug mithramycin by *Streptomyces argillaceus*. J Biol Chem 275:3065–3074

Majee S, Chakrabarti A (1995) A DNA-binding antitumor antibiotic binds to spectrin. Biochem Biophys Res Commun 212:428–432

Majee S, Sen R, Guha S, Bhattacharyya D, Dasgupta D (1997) Differential interactions of the Mg²⁺ complexes of chromomycin

A₃ and mithramycin with poly(dG-dC) x poly(dC-dG) and poly (dG) x poly(dC). Biochemistry 36:2291–2299

Majee S, Dasgupta D, Chakrabarti A (1999) Interaction of the DNA- binding antitumor antibiotics, chromomycin and mithramycin with erythroid spectrin. Eur J Biochem 260:619–626

Menéndez N, Nur-e-Alam M, Braña AF, Rohr J, Salas JA, Méndez C (2004a) Biosynthesis of the antitumor chromomy- cin A₃ in Streptomyces griseus: analysis of the gene cluster and rational design of novel chromomycin analogs. Chem Biol 11:21–32

Menéndez N, Nur-e-Alam M, Braña AF, Rohr J, Salas JA, Méndez C (2004b) Tailoring modification of deoxysugars during biosyn- thesis of the antitumour drug chromomycin A3 by Streptomyces griseus ssp. griseus. Mol Microbiol 53:903–915

Menéndez N, Nur-e-Alam M, Fischer C, Braña AF, Salas JA, Rohr J, Méndez C (2006) Deoxysugar transfer during chromomycin A₃ biosynthesis in Streptomycetes griseus subsp. griseus: new deriv- atives with antitumor activity. Appl Environ Microbiol 72: 167–177

Mir MA, Majee S, Das S, Dasgupta D (2003) Association of chromatin with anticancer antibiotics, mithramycin and chromo- mycin A3. Bioorg Med Chem 11:2791–2801

Montanari A, Rosazza JP (1990) Biogenesis of chromomycin A₃ by Streptomyces griseus. J Antibiot (Tokyo) 43:883–889

Nur-e-Alam M, Méndez C, Salas JA, Rohr J (2005) Elucidation of the glycosylation sequence of mithramycin biosynthesis: isola- tion of 3³-deolivosylpremithramycin B and its conversion to premithramycin B by glycosyltransferase MtmGII. Chembiochem 6:632–636

Ogawa H, Yamashita Y, Katahira R, Chiba S, Iwasaki T, Ashizawa T, Nakano H (1998) UCH9, a new antitumor antibiotic produced by Streptomyces: I. Producing organism, fermentation, isolation and biological activities. J Antibiot (Tokyo) 5:261–266

Prado L, Lombó F, Braña AF, Méndez C, Rohr J, Salas JA (1999a) Analysis of two chromosomal regions adjacent to genes for a type II polyketide synthase involved in the biosynthesis of the antitumor polyketide mithramycin in Streptomyces argillaceus. Mol Gen Genet 261:216–225

Prado L, Fernández E, Weissbach U, Blanco G, Quirós LM, Braña AF, Méndez C, Rohr J, Salas JA (1999b) Oxidative cleavage of premithramycin B is one of the last steps in the biosynthesis of the antitumor drug mithramycin. Chem Biol 6:19–30

Rao KV, Cullen WP, Sabin BA (1962) A new antibiotic with antitumor properties. Antibiot Chemother 12:182–186

Remsing LL, Garcia-Bernardo J, González A, Kunzel E, Rix U, Braña AF, Bearden DW, Méndez C, Salas JA, Rohr J (2002) Ketopremitramycins and ketomithramycins, four new aureolic acid-type compounds obtained upon inactivation of two genes involved in the biosynthesis of the deoxysugar moieties of the antitumor drug mithramycin by Streptomyces argillaceus, reveal novel insights into post-PKS tailoring steps of the mithramycin biosynthetic pathway. J Am Chem Soc 124:1606–1614

Remsing LL, Bahadori HR, Carbone GM, McGuifie EM, Catapano CV, Rohr J (2003a) Inhibition of c-src transcription by mithramycin: structure–activity relationships of biosynthetically produced mithramycin analogues using the c-src promoter as target. Biochemistry 42:8313–8324

Remsing LL, González AM, Nur-e-Alam M, Fernández-Lozano MJ, Braña AF, Rix U, Oliveira MA, Méndez C, Salas JA, Rohr J (2003b) Mithramycin SK, a novel antitumor drug with improved therapeutic index, mithramycin SA, and demycarosyl-mithramy- cin SK: three new products generated in the mithramycin producer Streptomyces argillaceus through combinatorial bio- synthesis. J Am Chem Soc 125:5745–5753

Rodriguez D, Quirós LM, Braña AF, Salas JA (2003) Purification and characterization of a monooxygenase involved in the biosynthetic

pathway of the antitumor drug mithramycin. J Bacteriol 185:3962–3965

Rodriguez D, Quirós LM, Salas JA (2004) MtmMII-mediated C- methylation during biosynthesis of the antitumor drug mithra- mycin is essential for biological activity and DNA–drug interaction. J Biol Chem 279:8149–8158

Rohr J (1992) Comparison of multicyclic polyketides by folding analysis. A novel approach to recognize biosynthetic and/or evolutionary interrelationships of the natural products or inter- mediates, and its exemplification on hepta-, octa- and decake- tides. J Org Chem 57:5217–5223

Rohr J, Weissbach U, Beninga C, Künzel E, Siems K, Bindseil KU, Lombó F, Prado L, Braña AF, Méndez C, Salas JA (1998) The structures of premithramycinone and demethylpremithramyci- none, plausible early intermediates of the aureolic acid group antibiotic mithramycin. Chem Commun 437–438

Rohr J, Méndez C, Salas JA (1999) The biosynthesis of aureolic acid group antibiotics. Bioorg Chem 27:41–54

Ryuto M, Ono M, Izumi H, Yoshida S, Weich HA, Kohno K, Kuwano M (1996) Induction of vascular endothelial growth factor by tumor necrosis factor alpha in human glioma cells. Possible roles of SP-1. J Biol Chem 271:28220–28228

Sastry M, Patel DJ (1993) Solution structure of the mithramycin dimer–DNA complex. Biochemistry 32:6588–6604

Sastry M, Fiala R, Patel DJ (1995) Solution structure of mithramycin dimers bound to partially overlapping sites on DNA. J Mol Biol 251:674–689

Sato K, Okamura N, Utagawa K, Ito Y, Watanabe M (1960) Studies on the antitumor activity of chromomycin A₃. Sci Rep Res Inst Tohoku Univ 9:224–232

Sensi P, Greco AM, Pagani H (1958) Isolation and properties of a new antibiotic LA-7017. Antibiort Chemother 8:241–244

Shen B, Hutchinson CR (1993a) Tetracenomycin F2 cyclase: intramolecular aldol condensation in the biosynthesis of tetrace- nomycin C in Streptomyces glaucescens. Biochemistry 32:11149–11154

Shen B, Hutchinson CR (1993b) Tetracenomycin Fl monooxygenase: oxidation of a naphthacene to a naphthacenequinone in the biosynthesis of tetracenomycin C in Streptomyces glaucescens. Biochemistry 32:6656–6663

Tagashira M, Kitagawa T, Isonishi S, Okamoto A, Ochiai K, Ohtake Y (2000) Mithramycin represses MDR1 gene expres- sion in vitro, modulating multidrug resistance. Biol Pharm Bull 23:926–929

Trefzer A, Fischer C, Stockert S, Westrich L, Kunzel E, Girreser U, Rohr J, Bechthold A (2001) Elucidation of the function of two glycosyltransferase genes (lanGTI and lanGT4) involved in landomycin biosynthesis and generation of new oligosaccharide antibiotics. Chem Biol 8:1239–1252

Trefzer A, Blanco G, Remsing L, Kunzel E, Rix U, Lipata F, Braña AF, Méndez C, Rohr J, Bechthold A, Salas JA (2002) Rationally designed glycosylated premithramycins: hybrid aromatic poly- ketides using genes from three different biosynthetic pathways. J Am Chem Soc 124:6056–6062

Vigneswaran N, Thayaparan J, Knops J, Trent J, Potaman V, Miller DM, Zacharias W (2001) Intra- and intermolecular triplex DNA formation in the murine c-myb proto-oncogene promoter are inhibited by mithramycin. Biol Chem 382:329–342

Wakisaka G, Uchino H, Nakamura T, Sotovayashi H, Shirakawa S, Adachi A, Sakurai M (1963) Selective incells by chromomycin A₃. Nature 198:385–386

Wang C, Gibson M, Rohr J, Oliveira MA (2005) Crystallization and X-ray diffraction properties of Baeyer–Villiger monooxygenase MtmOIV from the mithramycin biosynthetic pathway in Strep- tomyces argillaceus. Acta Crystallogr Sect F Struct Biol Cryst Commun 61:1023–1026

Ward DC, Reich E, Goldberg IH (1965) Base specificity in the interaction of polynucleotides with antibiotic drugs. Science 149:1259–1263

Waring MJ (1981) DNA modification and cancer. Annu Rev Biochem 50:159–192

Wohlert SE, Blanco G, Lombó F, Fernández E, Braña AF, Reich S, Udvarnoki G, Méndez C, Decker H, Frevert J, Salas JA, Rohr J

(1998) Novel hybrid tetracenomycins through combinatorial biosynthesis using a glycosyltransferase encoded by the *elm* genes in cosmid 16F4 and which shows a broad sugar substrate specificity. J Am Chem Soc 120:10596–10601

Wohlert SE, Kunzel E, Machinek R, Méndez C, Salas JA, Rohr J (1999) The structure of mithramycin reinvestigated. J Nat Prod 62:119–121
