
REVIEW

# Regulation and Function of IKK and IKK-Related Kinases

Hans Häcker${}^{1*}$ and Michael Karin${}^{2}$

(Published 17 October 2006)

Members of the nuclear factor kappa B (NF-$\kappa$B) family of dimeric transcription factors (TFs) regulate expression of a large number of genes involved in immune responses, inflammation, cell survival, and cancer. NF-$\kappa$B TFs are rapidly activated in response to various stimuli, including cytokines, infectious agents, and radiation-induced DNA double-strand breaks. In nonstimulated cells, some NF-$\kappa$B TFs are bound to inhibitory IkB proteins and are thereby sequestered in the cytoplasm. Activation leads to phosphorylation of IkB proteins and their subsequent recognition by ubiquitinating enzymes. The resulting proteasomal degradation of IkB proteins liberates NF-$\kappa$B-bound NF-$\kappa$B TFs, which translocate to the nucleus to drive expression of target genes. Two protein kinases with a high degree of sequence similarity, IKK$\alpha$ and IKK$\beta$, mediate phosphorylation of IkB proteins and represent a convergence point for most signal transduction pathways leading to NF-$\kappa$B activation. Most of the IKK$\alpha$ and IKK$\beta$ molecules in the cell are part of IKK complexes that also contain a regulatory subunit called IKK$\gamma$ or NEMO. Despite extensive sequence similarity, IKK$\alpha$ and IKK$\beta$ have largely distinct functions, due to their different substrate specificities and modes of regulation. IKK$\beta$ (and IKK$\gamma$) are essential for rapid NF-$\kappa$B activation by proinflammatory signaling cascades, such as those triggered by tumor necrosis factor $\alpha$ (TNF$\alpha$) or lipopolysaccharide (LPS). In contrast, IKK$\alpha$ functions in the activation of a specific form of NF-$\kappa$B in response to a subset of TNF family members and may also serve to attenuate IKK$\beta$-driven NF-$\kappa$B activation. Moreover, IKK$\alpha$ is involved in keratinocyte differentiation, but this function is independent of its kinase activity. Several years ago, two protein kinases, one called IKK$\varepsilon$ or IKK-i and one variously named TBK1 (TANK-binding kinase), NAK (NF-$\kappa$B-activated kinase), or T2K (TRAF2-associated kinase), were identified that exhibit structural similarity to IKK$\alpha$ and IKK$\beta$. These protein kinases are important for the activation of interferon response factor 3 (IRF3) and IRF7, TFs that play key roles in the induction of type I interferon (IFN-I). Together, the IKKs and IKK-related kinases are instrumental for activation of the host defense system. This Review focuses on the functions of IKK and IKK-related kinases and the molecular mechanisms that regulate their activities.

## Introduction

NF-$\kappa$B transcription factors (TFs) are rapidly activated in response to various stimuli, allowing quick activation of target genes that encode cytokines [for example, interleukin-1 (IL-1), IL-12, IL-2, and interferon-$\beta$ (IFN-$\beta$)], membrane proteins [for example, major histocompatibility complex classes I and II, ICAM-1 (intracellular adhesion molecule 1), and E-selectin], TFs [for example, c-Myc and IRF4 (interferon-regulatory factor)], and inhibitors of apoptosis [for example, c-FLIP (cellular FLICE-like inhibitory protein) and Bcl-X$_L$]. This rapid response system depends on the sequestration of NF-$\kappa$B dimers in the cytoplasm through interaction with inhibitory IkB proteins. Cell stimulation leads to IkB phosphorylation, thereby creating a recognition signal for ubiquitinating enzymes, which mark the IkBs for rapid proteasomal degradation ($I$). Degradation of IkBs liberates NF-$\kappa$B dimers to translocate to the nucleus and activate transcription of target genes. Phosphorylation is accomplished by protein kinases, whose activity is tightly controlled and represents the primary mode of NF-$\kappa$B regulation. Two protein kinases with high sequence similarity, IkB kinase alpha (IKK$\alpha$) and IKK$\beta$, are the most important IkB kinases (2). Both kinases were purified and cloned on the basis of their ability to phosphorylate IkB proteins in response to cell stimulation with tumor necrosis factor $\alpha$ (TNF$\alpha$) and are present in cells as part of a high molecular weight complex that also contains a regulatory subunit termed IKK$\gamma$ or NEMO (NF-$\kappa$B essential modulator) (3–7). This basic trimolecular complex, which may contain an additional substrate-targeting subunit named ELKS (8), is referred to as the IKK complex. Despite their similarities and presence in a common protein complex, IKK$\alpha$ and IKK$\beta$ have largely nonoverlapping functions, due to different substrate specificities and other factors (see below). IKK$\beta$ is the major IKK catalytic subunit for NF-$\kappa$B activation by pro-inflammatory stimuli, such as TNF$\alpha$, IL-1, and “Toll-like” receptor (TLR) agonists, such as lipopolysaccharide (LPS). In contrast, kinase activity of IKK$\alpha$ is primarily required for activation of a specific type of NF-$\kappa$B dimer (p52:RelB; see below) in response to a subset of TNF family members, such as BAFF (B cell–activating factor), CD40-ligand, and lymphotoxin $\alpha$ (LT$\alpha$)–LT$\beta$ heterotrimers. Whereas the IKK$\beta$-dependent pathway is essential for activation of innate immunity, the IKK$\alpha$-dependent pathway is more important for regulation of adaptive immunity and lymphoid organogenesis (9). IKK$\alpha$ may also attenuate signaling through the IKK$\beta$-dependent pathway and has a nonrelated, but critical, function in keratinocyte differentiation that does not depend on its catalytic activity and does not involve NF-$\kappa$B TFs. This function of IKK$\alpha$ will not be discussed further in this review.

Two other protein kinases have been identified with sequence similarity to IKK$\alpha$ and IKK$\beta$, the so-called IKK-related kinases, IKK$\varepsilon$ (also called IKK-i) (10, 11) and TBK1 [also called NAK (NF-$\kappa$B–activated kinase) or T2K (TRAF2-associated kinase).

---

1 Department of Infectious Diseases, St. Jude Children’s Research Hospital, 332 North Lauderdale Street, Memphis, TN 38105, USA.
2 Laboratory of Gene Regulation and Signal Transduction, Department of Pharmacology, School of Medicine, University of California, San Diego, 9500 Gilman Drive, La Jolla, CA 92093, USA.
*Corresponding author. E-mail, hans.haecker@stjude.org

Science’s stke www.stke.org/cgi/content/full/2006/357/re13 Page 1

ated kinase)] (12–14). Although these kinases were implicated in the regulation of NF-κB activity (a function that they indeed may exert), it is now clear that the IKK-related kinases also activate two other transcription factors, IRF3 and IRF7 (15–18), which are critical for expression of type I IFN genes. Thus, the IKK-related kinases are important mediators of antiviral responses and, together with the original IKKs, coordinate and marshal the host defense response. Since the discovery of the IKKs and IKK-related kinases nearly 10 years ago, a large body of experimental data has accumulated, illuminating the importance of these kinases in various physiological and pathophysiological functions. These include the host defense response, inflammation, cell survival, cancer, metabolic regulation, and neuroadaptation. Various activators use distinct signal transduction pathways to regulate the activity of IKKs and IKK-related kinases. This review focuses on the molecular mechanisms that regulate IKKs and IKK-related kinases and their host defense-related functions.

### Background

The term “NF-κB” refers to a heterogeneous collection of dimeric TFs composed of members of the NF-κB/Rel family. The mammalian genome encodes five NF-κB subunits: NF-κB1 (p50 and its precursor p105), NF-κB2 (p52 and its precursor p100), c-Rel, RelA (p65), and RelB (19). These proteins are characterized by a highly conserved Rel homology domain (RHD), which mediates dimerization, DNA binding, and interaction with IkB proteins. Three of the NF-κB proteins—RelA, c-Rel, and RelB—are synthesized in their mature form and are held in the cytoplasm by IkB proteins, whose major structural features are six or seven ankyrin repeats. These repeats mediate binding to the RHD and masking of the nuclear localization sequences (NLSs) in the C terminus of NF-κB TFs. By contrast, NF-κB1 and NF-κB2 are synthesized as large precursors, p105 and p100, respectively, containing an N-terminal RHD and a C-terminal IkB-like region with multiple ankyrin repeats. Limited proteolysis of the C terminus of these precursors is required to yield the mature p50 and p52 subunits, consisting of the N-terminal RHD (20–22). Processing of p105 appears to be mainly a constitutive event, providing a pool of mature p50 subunits, which dimerize with other Rel proteins (23). In contrast, p100 processing is tightly regulated through activation of a limited set of TNF receptor (TNFR) family members (see below). While RelA, c-Rel, and RelB contain transactivation domains, which are required to drive gene transcription, p50 and p52 do not contain transactivation domains and serve primarily as dimerization and DNA-binding partners (19). Targeted disruption of all NF-κB genes has been accomplished in mice (24, 25), revealing a certain degree of redundancy among some of the family members, which is explained by their ability to form different homo- and heterodimers that can recognize a common DNA sequence motif—the κB site (26). However, there are also clear examples for specific and nonredundant roles of individual Rel proteins. For instance, RelA-deficient mice die during embryogenesis because of massive, TNFα-induced apoptosis in the liver, demonstrating an essential, anti-apoptotic role of RelA (27). Other examples are expression of the gene encoding the p40 subunit of IL-12, which requires p50:c-Rel heterodimers (28), or expression of certain chemokines involved in lymphoid organogenesis, which requires p52:RelB dimers (9). RelB, in contrast to other Rel family members, preferentially interacts

with p100 (29), whose processing is controlled by a limited number of receptor pathways (see below). The phenotype of individual Rel-knockout mice has been reviewed in more detail elsewhere (9, 25).

Activation of all NF-κB dimers except p52:RelB is tightly controlled by IkB proteins, which mask the NLS present near the C terminus of the RHD, thereby preventing nuclear entry and DNA binding. The IkB group consists of four main members: IkBα, IkBβ, IkBε, and IkBNS (19, 30, 31). There are other IkB-like proteins that contain ankyrin repeats, such as BCL3 and IkBζ. However, these proteins seem to function not as NF-κB inhibitors, but rather as coactivators that specifically interact with p50 and p52 homodimers (BCL3) (32, 33), or promote the activation of a subset of NF-κB target genes, such as the IL-6 gene (IkBζ) (34). The most critical regulatory event in NF-κB activation is site-specific phosphorylation of IkB (α, β, and ε) by the IKK complex, which leads to polyubiquitination and proteasomal degradation (1). Nevertheless, this is not the only mechanism that regulates NF-κB activity. IkB proteins are also regulated through NF-κB-dependent de novo synthesis, thereby allowing feedback regulation and termination of the NF-κB response (35, 36). This seems to be the primary mode of regulation for IkBNS, which is transcribed in response to LPS and IL-10 to control specific NF-κB responses (31, 37). Because this type of NF-κB regulation does not directly depend on IKK, it will not be discussed further. Other regulatory mechanisms that depend on IKK-mediated phosphorylation of NF-κB proteins will be discussed below.

Most of the knowledge of IKK-dependent IkB degradation is based on studies of IkBα. Phosphorylation of two conserved serine (S) residues in its N terminus, S32 and S36, leads to recognition by the β-TrCP F-box-containing component of a Skp1-Cullin-F-box (SCF)-type E3 ubiquitin-protein ligase complex, called SCF<sup>βTrCP</sup>, resulting in polyubiquitination and degradation of IkBα by the 26S proteasome (1). IkBβ and IkBε are also regulated by N-terminal, site-specific phosphorylation, but the kinetics of their phosphorylation and degradation are much slower than that of IkBα and may reflect different substrate specificities of the IKK complex (38). In contrast to IkBα, IkBβ, and IkBε, which are phosphorylated by activated IKKβ within the trimolecular IKK complex, p100 is inducibly phosphorylated by IKKα, and this also leads to SCF<sup>βTrCP</sup> recruitment and polyubiquitination (39). Proteolytic degradation of p100 is limited to its C-terminal IkB-like portion, thereby generating mature p52, which (primarily) enters the nucleus as a dimer with RelB (22, 29). p105 is also processed by limited proteolysis to generate p50, but in contrast to that of p100, the processing of p105 seems to be a constitutive process, suggested to be independent of ubiquitination, at least under certain conditions (40). IKKβ-dependent phosphorylation and degradation of the p105 C-terminal IkB-like domain has also been described (41), but the importance of this putative pathway remains to be determined.

### The IKK Complex and Its Mechanism of Activation

Whereas IKKβ is most important for rapid degradation of NF-κB-bound IkBs, IKKα controls processing of p100, leading to activation of p52:RelB dimers, a response that is substantially slower than the activation of IkB-bound dimers (9). Whereas IKKβ-dependent degradation of IkBα occurs within minutes, IKKα-dependent processing of p100 requires several hours.

The IKKβ-dependent pathway leading to rapid degradation of IkBα, IkBβ, and IkBε is commonly referred to as the canonical, or classical, NF-κB pathway, whereas the more recently discovered, IKKα-dependent pathway leading to processing of p100 and activation of p52:RelB is referred to as the alternative NF-κB pathway (42). Nevertheless, both IKKα and IKKβ were identified by biochemical means as constituents of a 700- to 900-kD protein complex that exhibits TNFα-induced IkBα-specific kinase activity (3, 4). IKKα was also identified in a two-hybrid screen as a protein that interacts with the NF-κB-inducing kinase (NIK), named after its ability to increase transcription of the NF-κB gene when overexpressed in cells (43). IKKα and IKKβ are similar in structure (50% sequence identity, 70% protein similarity) and contain N-terminal protein kinase domains and more C-terminally located leucine zipper (LZ) and helix-loop-helix (HLH) motifs (Fig. 1A). Both proteins exhibit kinase activity toward IkBα. The third component of this complex is a 48-kD regulatory subunit, named IKKγ (6) [and given the alternative names NEMO (44), IKKAP1 (45), or FIP-3 (46)]. IKKγ was identified biochemically as a component of the IKKα/β-containing complex and by genetic complementation of a cell line nonresponsive to NF-κB-activating stimuli (6, 44). The precise mechanistic role of IKKγ is still enigmatic. Studies with gene-deficient cells or mice indicate that IKKγ is essential for activation of the classical NF-κB pathway and is required for formation of the large IKK complex, described above. IKKγ is essential for connecting the bound catalytic subunits to upstream activation in the classical NF-κB pathway, but it is not required for activation of the IKKα-dependent alternative pathway (47). Structure-prediction algorithms predict that IKKγ is primarily a helical protein with large stretches of coiled-coil structure, including a LZ at its C terminus (Fig. 1A). A major handicap in understanding its molecular function is the absence of structural data for any component of the IKK complex. Thus, most of our knowledge is based on mutational analysis, reconstitution experiments, and gene-targeting studies.

When isolated under stringent conditions, the IKK complex has an apparent molecular size of 700 to 900 kD and consists of IKKα, IKKβ, and IKKγ (6). The sum of the molecular masses of these subunits is 220 kD, suggesting that either other components are present or that the large IKK complex is oligomeric. Indeed, there is ample evidence that IKKα and IKKβ form dimers via their LZ motif (5, 48). When overexpressed in human embryonic kidney 293 (HEK293) cells or analyzed in IKKγ-deficient cells, IKKα/β dimers elute with an apparent size of 300 kD, implying an elongated shape. These dimers are further assembled to a complex of still larger apparent size, which contains two IKKα-IKKβ dimers, bound to IKKγ (44). Overexpression of IKKα and IKKβ together with IKKγ, either in mammalian or yeast cells, results in the formation of a catalytically active complex comparable in size to the large IKK complex, yet the activity of this complex is low and its full activation depends on further modification, most likely phosphorylation (49).

Consistent with its role in the assembly of a high-order IKK complex, IKKγ itself forms multimers, either trimers (50) or tetramers (51). Because the large IKK complex contains two IKKα/β dimers, the IKKγ oligomer within this complex is most likely a tetramer. IKKγ oligomerization depends on its C-terminal coiled-coil (CC) and LZ motifs. Binding of IKKβ (and probably that of IKKα) is conferred by a C-terminal decapeptide motif that recognizes residues located in the N-terminal CC motif of IKKγ (6, 52). This decapeptide motif was named NEMO-binding domain (NBD) and was shown to be required for IKK assembly (53). Accordingly, a cell-permeable NBD peptide prevents IKK activation (53). These data and IKKγ deletion and loss-of-function mutations illustrate that IKK activation is absolutely dependent on IKKγ. Most likely, IKKγ

A

| Kinase domain | L | H | N |
| --- | --- | --- | --- |
|  | Z | H | B |
|  |  | D |  |

| Kinase domain | L | H | N |
| --- | --- | --- | --- |
|  | Z | H | B |
|  |  | D |  |

|  | C | C | L | Z |
| --- | --- | --- | --- | --- |
|  |  |  | F |  |

B

![Diagram](https://i.imgur.com/placeholder.png)

Very low  
activity  

Inactive  

High  
activity  

Low  
activity  

Fig. 1. IKK structure and activation. (A) Schematic diagram showing the principal IKK subunits and their structural and functional motifs. LZ, leucine zipper; HLH, helix-loop-helix; CC, coiled coil; ZF, zinc finger; NBD, NEMO-binding domain. (B) A model for IKK regulation through autophosphorylation. The IKK catalytic subunits are dimerized through their LZ motifs. The HLH motif interacts with the kinase domain. Phosphorylation of two critical serines in the activation loop of IKK activates the kinase. Full kinase activity is accompanied by no or a low amount of phosphorylation at the C terminus. However, once activated, in addition to phosphorylating its substrates, IKK also phosphorylates its own C terminus, thereby inducing a conformational change, which weakens the interaction between the HLH motif and the kinase domain, resulting in decreased kinase activity. In this state, IKK is more susceptible to further dephosphorylation and inactivation by phosphatase action.

brings two IKKα/β dimers together, such that they activate each other by transautophosphorylation (2).

More recent experiments have suggested the interaction of the core-IKK complex with additional subunits. One putative subunit is Hsp90 (54), but Hsp90 also interacts with many other protein kinases and seems to serve as a general chaperone that enhances kinase-domain folding (55). Another protein that interacts with IKK and regulates NF-κB activity is ELKS (8), a protein that also functions in protein trafficking [for example, insulin exocytosis in pancreatic β cells (56)]. Immunodepletion experiments implicated ELKS as a stoichiometric component of IKK. RNAi-mediated suppression of ELKS reduced activation of IKK and of NF-κB in response to TNFα and IL-1. Moreover, ELKS coimmunoprecipitated with IkBa, implying that it might contribute to the formation of the IKK:IkBα:NF-κB complex (8). Nonetheless, the role of ELKS in IKK and NF-κB activation needs to be further confirmed through generation of a knockout mouse strain.

There is considerable evidence that both IKKα and IKKβ need to be phosphorylated to become activated. Purified IKK is inactivated upon incubation with protein phosphatase 2A (PP2A), whereas treatment of cells with the PP2A inhibitor okadaic acid results in IKK activation (3). Like those of other protein kinases, the kinase domains of IKKα and IKKβ contain an activation loop. The activation loop is subject to phosphorylation at two serines, resulting in a conformational change leading to kinase activation (4, 57, 58) (Fig. 1B). Replacement of these serines in IKKβ (S177/S181) with alanines [IKKβ(AA)] prevents kinase activation, whereas replacement with phosphomimetic glutamates [IKKβ(EE)] results in a constitutively active kinase (57). Both serines are phosphorylated in vivo in response to pro-inflammatory stimuli (57). The activation loop of IKKα is identical in sequence to that of IKKβ and also becomes phosphorylated at the corresponding serines (S176/S180) during cell stimulation (57). However, IKKα phosphorylation is not critical for activation of the classical IKK complex or NF-κB activation by most pro-inflammatory stimuli (59, 60). Nonetheless, IKKα phosphorylation is required for activation of NF-κB in mammary epithelial cells in response to RANK (receptor activator of NF-κB) ligand (61) and for activation of the alternative NF-κB pathway (see below) (42).

Another region involved in regulation of IKK activity is the HLH motif. Mutations within this motif decrease IKK activity without affecting complex assembly in transiently transfected cells or in vitro (5, 48). Mutants of IKKβ lacking the HLH motif and the NBD are devoid of kinase activity (57), as expected, but are reactivated upon coexpression of a C-terminal HLH-and NBD-containing fragment (57). The HLH motif physically interacts with the kinase domain and seems to serve as an endogenous activator of IKK in a manner that resembles activation of cyclin-dependent kinases (CDKs) by their cyclin subunits (62). C-terminal of the HLH motif, just before the NBD, IKKα and IKKβ contain a stretch of serines, which are heavily phosphorylated during IKK activation (57). Phosphorylation at these sites requires IKK activity, but in contrast to phosphorylation of the activation loop, phosphorylation of the C-terminal cluster has a negative autoregulatory function (57). Replacement of 10 of the C-terminal serines with alanines prolongs TNFα-induced IKKβ activity, whereas replacement of these serines with phosphomimetic glutamate residues reduces IKK activity (57). Given the transient nature of IKK activation, this

autophosphorylation event may contribute to termination of IKK activity (Fig. 1B). Termination of kinase activity may also depend on protein phosphatase 2Cβ, which was found to associate with IKKβ when overexpressed in HEK293 cells and whose suppression prolongs TNFα-induced IKK activity (63). Whether PP2Cβ dephosphorylates the activation-loop serines of IKKβ needs to be determined.

The alternative NF-κB pathway does not depend on IKKβ and IKKγ and is activated by a subset of TNFR family members, such as BAFF-R and CD40 on B cells and LTβR on splenic stromal cells (9). There is still much less information on the molecular events involved in IKKα-dependent processing of p100 compared to our understanding of IKKβ-dependent IkB phosphorylation, because most of our knowledge on the alternative pathway is based on genetic rather than biochemical analysis. IKKα-deficient cells or cells expressing an IKKα(AA) mutant, which cannot be activated by upstream kinases, are deficient in p100 processing (42). Conversely, overexpression of a constitutively active IKKα(EE) mutant induces p100 processing in HEK293 cells (42). A C-terminal serine residue in p100, which functionally corresponds to S32 and S36 in IkBα, is required for IKKα-dependent processing and is phosphorylated by purified IKKα in vitro (42). However, stimulation-dependent activation of IKKα dimers has not clearly been demonstrated. Whether this is due to a low level of IKKα activation (reflected by the slow kinetics of p100 processing) or the loss of other critical factors during immunopurification of IKKα (for instance, the upstream kinase NIK) is unknown. It is quite possible that IKKα dimers interact with NIK to form a high-order complex. The inability to show regulated activation of IKKα dimers also explains why it has been difficult to conclusively demonstrate that IKKα directly phosphorylates p100 in vivo. Nevertheless, it is clear that the alternative IKKα-dependent pathway is fully operational in the absence of IKKγ or IKKβ (47), indicating that the classical trimolecular IKK complex is not essential and in fact is incapable of promoting p100 processing.

Nonetheless, there are situations in which catalytically active IKKα as part of the classical IKKγ-containing complex is more critical than is IKKβ for inducible IkBα phosphorylation and degradation. For instance, NF-κB activation in mammary epithelial cells in response to RANK ligand (RANK-L) requires the kinase activity of IKKα but not that of IKKβ (61). This activity, which can be inhibited by a nonphosphorylatable IkBα mutant, triggers a classical NF-κB activation pathway that drives cyclin D1 expression. IKKα is also required for activation of classical NF-κB signaling by TNFR family members CD27 and CD40 in a human B lymphoma cell line (64). These results, however, need to be confirmed in primary, gene-targeted B cells. Nevertheless, it appears that at least in some pathways of IKK activation, the activating signal is “channeled” to IkB through NIK and IKKα. The factors that dictate the subunit specificity for each stimulus are currently unknown.

Two other kinases, Akt1 and Akt2, have also been implicated in the regulation of p100 processing (65). Overexpression of a constitutively active form of Akt in HEK293 cells leads to a slight increase in p52 production, and mouse embryonic fibroblasts (MEFs) deficient in Akt1 and Akt2 exhibited a defect in (primarily) an ill-defined “constitutive” form of p100 processing. Although these observations might point to an important role of Akt in NIK- or IKKα-dependent gene regulation, it

REVIEW

A The classical NF-κB pathway
B The alternative NF-κB pathway

Fig. 2. Activation of NF-κB signaling by TNF receptors. (A) Activation of the classical pathway via TNFR1. TNFR1 engagement leads to recruitment of the adaptor molecule TRADD, which subsequently recruits the signaling proteins TRAF2 (and TRAF5) and RIP1 to form complex 1. TRAF2 mediates recruitment of the IKK complex to complex 1 through interaction with its catalytic subunits. TRAF2 also promotes K63-linked ubiquitination of RIP1 and possibly of itself. Ubiquitinated RIP1 binds to IKKγ and stabilizes the interaction of IKK with complex 1 and may even alter the conformation of the IKK complex. TAB2 and TAB3 interact with TRAF2 and TAK1 and this may result in the activation of TAK1. This also places TAK1 near other components of the signaling complex, including IKKβ, which may be directly phosphorylated by TAK1 on its activation loop. How MEKK3 is brought into proximity of this signaling complex is less clear, but this may involve interaction with RIP1, rather than TAB2, TAB3, TRAF2, or TRAF5. Active IKK phosphorylates IκBs on conserved serines, leading to their recognition by the E3 ubiquitin ligase SCF<sup>βTrCP</sup>. Polyubiquitination of IκBs leads to their degradation by the 26S proteasome and liberation of bound NF-κB dimers, which translocate to the nucleus to drive gene transcription. (B) Activation of the alternative NF-κB pathway by BAFF. In nonstimulated cells, the kinase NIK is constitutively bound to TRAF3 in the cytoplasm. This results in NIK ubiquitination and rapid turnover. BAFF-R engagement leads to the recruitment and sequestration of TRAF3, which results in NIK stabilization and increased expression, leading to its activation by autophosphorylation. NIK forms a complex with p100 and IKKα. NIK directly phosphorylates IKKα at its activation loop, resulting in its activation and IKKα-dependent phosphorylation of p100. Phosphorylated p100 is recognized by the E3 ubiquitin ligase SCF<sup>βTrCP</sup>, leading to its polyubiquitination and partial proteolytic degradation of its C-terminal ankyrin-repeat region by the 26S proteasome. Free p52:RelB dimers translocate to the nucleus and drive gene transcription.

Science’s stke
www.stke.org/cgi/content/full/2006/357/re13
Page 5

should be noted that different sources of immortalized MEFs exhibit substantially different amounts of constitutive p100 processing. Also, the factors involved in regulation of this form of constitutive p100 processing in vitro are entirely undefined. It will therefore be important to confirm these data in primary cells (for example, B cells or stromal cells) and also to further define the physiological importance of this constitutive form of p100 processing.

**Signaling Pathways Leading to IKK Activation**

As discussed above, it is likely that activation of IKK involves trans-autophosphorylation by its catalytic subunits IKKα and IKKβ (Fig. 1B). However, other molecular mechanisms have also been proposed to regulate this initiating event. Yet, for the time being, the molecular details of IKK activation are not clear. Generally, three mechanisms can be envisaged: (i) direct phosphorylation of one of the IKK catalytic subunits at the activation loop; (ii) IKK multimerization, resulting in trans-autophosphorylation; and (iii) a conformational change induced by a posttranslational modification other than phosphorylation or through protein-protein interactions. These mechanisms are not mutually exclusive. For instance, phosphorylation at the activation loop of either IKKα or IKKβ can be brought about by an upstream kinase (IKK-K) or could be due to autophosphorylation through induced proximity of IKKα-IKKβ dimers. The latter could be mediated through interaction with multimerized receptors or docking proteins or could be induced by posttranslational modifications of IKKγ. We will discuss the different mechanisms proposed to explain IKK activation in relation to the major families of receptors that trigger NF-κB signaling.

**Activation of NF-κB Signaling by Members of the TNFR Family**

TNFR family members activate either the classical pathway, or both the classical and alternative NF-κB pathway. As discussed above, activation of the classical pathway is mostly dependent on IKKβ, whereas activation of the alternative pathway strictly depends on IKKα.

Most of our knowledge on activation of the classical pathway comes from studies of the major TNFα receptor, TNFR1, which will be the focus of our discussion. TNFR family members lack enzymatic activity and rely on recruitment of intracellular adaptors and signaling molecules to initiate signal transduction (Fig. 2A). All family members interact directly or indirectly with TNFR-associated factors (TRAFs), which are critical mediators of NF-κB activation (66). On the basis of their eponymous C-terminal TRAF domain, six members of this family were identified. Apart from TRAF1, all TRAFs contain N-terminal zinc-binding motifs, including a RING finger, which together constitute the TRAF effector domain. The C-terminal TRAF domain is further divided into an N-terminal CC region and the TRAF-C domain, the latter conferring binding to upstream molecules (67). In some cases, the TRAF-C domain directly interacts with the intracellular signaling domain of the respective TNFR (for example, CD40), whereas in other cases, receptor-induced TRAF recruitment depends on additional adaptors (such as TRADD in the case of TNFR1) (68). In addition to TRAFs—specifically, TRAF2 and TRAF5—TRADD recruits also another molecule named RIP1 (receptor interacting protein 1) (69). RIP1 contains multiple structural motifs including a death domain (DD), which mediates homotypic interaction with TRADD, a so-called RHIM (RIP homotypic interaction motif), and a protein kinase domain. On the basis of sequence similarities in the kinase domain, three other RIP family members were cloned, and three more were found in database searches (70). However, so far, only RIP1 is known to be required for TNFR-dependent NF-κB activation (71, 72). When overexpressed, all of these molecules—TRADD, RIP1, TRAF2, and TRAF5—cause activation of NF-κB. RIP1-knockout cells fail to activate IKK [and JNK1 (c-Jun N-terminal kinase 1) or JNK2] in response to TNFα, whereas TRAF2-deficient cells show reduced activation of IKK and almost no activation of JNK1 or JNK2 (72, 73). Residual NF-κB activation in TRAF2-knockout cells might be due to compensation by TRAF5, because cells lacking TRAF2 and TRAF5 show essentially no TNFα-dependent IKK activity (74). The kinase activity of RIP1 is dispensable for activation of NF-κB (and JNK1 or JNK2), because reconstitution of RIP1-deficient cells with a catalytically inactive RIP1 mutant confers full responsiveness (71). Therefore, identification of the above-mentioned proteins, which do not display intrinsic catalytic activities, does not offer an immediate solution to the mechanism of IKK activation. TRAF2 has been shown to recruit the IKK complex to the activated TNFR1, through interaction with the LZ motifs of IKKα and IKKβ (75). This initial recruitment of IKK does not seem to depend on RIP1. However, it triggers RIP1-dependent binding of IKKγ and activation of the IKK complex (76, 77). Because RIP1 is ubiquitinated during TNFR1 activation and TRAF2 seems to be critically involved in this process (78), it is possible that TRAF2-dependent ubiquitination of RIP1 is involved in the formation of a stable supramolecular complex between TRAF2, RIP1, and IKK. Nonetheless, as a RING-finger protein, TRAF2 (like other TRAFs) is devoid of intrinsic E3 ubiquitin ligase activity and rather serves as a scaffold that recruits an E2 ubiquitin-conjugating enzyme to its target (79).

Although induced proximity of IKK catalytic subunits within this signaling complex might explain TNFR1-mediated IKK activation, there are two other molecules—the kinases MEKK3 and TAK1—that are implicated in this response. Both MEKK3 and TAK1 can induce NF-κB activation upon overexpression, but the response to TAK1 requires coexpression of the additional proteins TAB1, TAB2, or TAB3 (80–82). Data from gene-disrupted cells have been published for MEKK3 and for TAK1 (83–86). Mice deficient in either gene die during embryogenesis; MEKK3−/− embryos die around embryonic day 11 (E11), exhibiting defects in blood vessel development (87), whereas TAK1−/− embryos die around E10, exhibiting defective formation of the head fold and neural tube (86). Unfortunately, these distinct phenotypes do not illuminate the roles of either kinase in NF-κB signaling and suggest additional functions mediated via other pathways. Nonetheless, MEKK3-deficient fibroblasts exhibit defects in IL-1 and TNFα-induced activation of IKK (and of JNK1 or JNK2), a phenotype similar to that of RIP1-deficient cells (71, 72, 83). MEKK3 interacts with RIP1 and, when overexpressed, it can activate NF-κB and JNK in RIP1-and TRAF2-deficient cells, suggesting that it operates downstream of TRAF2 and RIP1 (83). When RIP1-deficient MEFs were reconstituted with a MEKK3 fusion protein containing MEKK3 and the DD of RIP1, TNFα-induced NF-κB activation was restored, indicating that RIP1 might serve as an adaptor protein that recruits MEKK3 into a TNFR1-induced signaling

Science’s stke www.stke.org/cgi/content/full/2006/357/re13 Page 6

complex (88). The NF-κB–inducing activity of the MEKK3-RIP1 fusion protein requires the intact ATP-binding pocket of MEKK3, implying that unlike that of RIP1, the catalytic activity of MEKK3 is directly involved in IKK activation (88). Although it is formally possible that binding and oligomerization of the DD of RIP1 fused to MEKK3 results in the nonphysiological activation of MEKK3 through TNFR1, an attractive scenario would be that TRADD oligomerization induces recruitment of TRAF2 or TRAF5, or both, and of RIP1 through direct protein-protein interactions. TRAF2 and TRAF5 may recruit the IKK complex through binding to its IKKα/β subunits, and this complex could be further stabilized by an interaction between RIP1 and IKKγ. RIP1 also serves as an adaptor that brings MEKK3 into close proximity with the IKK complex, resulting in activation of the latter through direct phosphorylation of its catalytic subunits (Fig. 2A). It needs to be stressed, however, that direct IKKα/β phosphorylation by MEKK3 has not been demonstrated, and it still has to be established whether MEKK3 acts as an IKK kinase in vivo.

TAK1 was identified as a kinase involved in TGFβ signaling (89), but it was later found to be activated in response to other stimuli, including TNFα and IL-1 (90, 91). TAK1 was also identified as a component of an IKK activating complex, coeluting with IKK-inducing activity in an in vitro, cell-free system (81). In contrast to MEKK3, TAK1 does not activate NF-κB when overexpressed in cells but rather depends on coexpression of other molecules, such as TAB1, TAB2, or TAB3 (81, 82, 92). However, the analysis of TAB1- or TAB2-knockout mice failed to reveal a role for these molecules in TNFα- or IL-1–induced activation of IKK (86). Instead, TAB1 appears to be involved in TGFβ signaling (93). However, RNA interference experiments have shown that suppression of TAB2 or the related molecule TAB3 had no effect on NF-κB activation, but the simultaneous suppression of both molecules resulted in a pronounced signaling defect (82). Given the structural and functional similarities between these proteins, it is possible that TAB2 and TAB3 compensate for each other. Both proteins are recruited to TAK1 during TNFα and IL-1 signaling, and both also interact with TRAF2 and TRAF6, which are involved in TNFα and IL-1 signal transduction, respectively (82). The C termini of TAB2 and TAB3, which are required for their function, contain a Zn-finger motif, typical of ubiquitin-binding proteins (94). As mentioned above, the TRAF proteins contain RING-finger domains, which are proposed to be involved in the generation of lysine-63 (K63)–linked polyubiquitin chains (95). Moreover, activation of cells by TNFα and IL-1 induces TRAF2-dependent ubiquitination of RIP1 (76, 96).

Further support for atypical (K63-mediated) ubiquitination as an important signaling event comes from the observation that an E2 complex containing UBC13 and the UBC-like protein UEV1A activates IKK in cytoplasmic extracts and that expression of dominant negative mutants or inhibition of UBC13 interferes with IKK activation in cells (95, 97). It should be noted, though, that the complete knockout of UBC13 in mice did not affect NF-κB activation by TNFα or other stimuli (98). It is possible, however, that another E2 may substitute for UBC13. An interesting model encompassing these data would be that TNFR1 activation leads through TRADD oligomerization to recruitment (and oligomerization) of TRAF2 or TRAF5, which initiate ubiquitination events, catalyzed by a yet-to-be identified E2 (Fig. 2A). Newly formed K63-mediated polyubiquitin

chains (or some other type of a branched polyubiquitin) on either RIP1 or the TRAF proteins themselves may then serve as docking sites for TAB2 and TAB3, which recruit TAK1 to the TNFR1 signaling complex through protein-protein interactions. Binding and possibly oligomerization of TAK1 through TAB2 and TAB3 may trigger the catalytic activity of TAK1 (94, 99). Concomitant recruitment of the IKK complex via TRAF2, TRAF5, and RIP1, possibly also ubiquitination-dependent, would bring TAK1 and IKK into close proximity, leading to phosphorylation of IKKα/β and its activation (Fig. 2A). This model is also consistent with other data showing that TAK1, immunopurified from HeLa cells and incubated with other proteins (including UBC13, UEV1A, TRAF6, and ubiquitin), leads to specific phosphorylation of IKKβ at its activation loop, supporting a role for TAK1 as an IKK-K (81).

Moreover, ubiquitin-dependent signal transduction may provide a unifying concept for TNFRI and IL-1R signaling, because TRAF6, implicated as an E3 ubiquitin ligase in the IL-1R pathway, may function similarly to TRAF2 and TRAF5 in the TNFR1 pathway (see below). However, so far there are many aspects of this model that are not supported by solid experimental data. Specific ubiquitination sites of proposed TRAF substrates, such as RIP1, have not unequivocally been defined by biochemical means. Even in the case of the site-specific mutation K377R in RIP1, which leads to a defect in IKKγ recruitment to the TNFRI complex and a consecutive reduction in NF-κB–dependent gene transcription (77), it is still unclear whether this specific lysine is ubiquitinated during physiological cell stimulation with TNFα or, alternatively, whether this mutation interferes with other unrelated functions of RIP1.

Furthermore, the formation of K63-mediated polyubiquitin chains on any protein involved in NF-κB signaling has not been demonstrated by either mass spectrometry or specific antibodies, and their signaling function is essentially a matter of conjecture that rests on the use of different ubiquitin mutants without direct biochemical support. As mentioned above, UBC13-deficient cells display no defect in NF-κB activation in response to different stimuli (98). Although this could be explained by the action of another E2, which substitutes for UBC13, it also could be due to a new type of branched ubiquitination that does not trigger proteasomal degradation. Moreover, reconstitution of TRAF6-deficient cells with a TRAF6 mutant lacking the signature motif of E3 RING-finger ligases—that is, the RING finger itself—completely restored IL-1–induced activation of NF-κB and of JNK1 and JNK2 activation in vivo (100). Other nonrelated effector functions did depend on an intact TRAF6 RING finger (100), implying a more restricted and specific function of this domain. It is therefore difficult to say how important the proposed E3 ligase activity of TRAF proteins is in vivo and whether it participates in some pathways and not in others. Only in vivo mapping of ubiquitination and phosphorylation sites, mass spectrometric identification of polyubiquitin chains, detailed analysis of reconstitution experiments with appropriate mutant proteins, and determination of binding affinities and interactions between endogenous proteins will resolve these issues. Furthermore, it is currently unknown whether MEKK3 and TAK1 act in parallel or serially, although, based on the incomplete defects in signaling observed in the absence of either molecule, one can speculate that TAK1 and MEKK3 may act in parallel. Obviously, the role of these molecules in NF-κB signaling is not as clear as the role of IKK itself.

The alternative NF-κB signaling pathway is activated by a rather limited number of TNFR family members, including BAFF-R (B cell–activating factor receptor) and CD40 (on B cells), RANK (on osteoclasts), and LTβR (on stromal cells) (47, 101–104). A hallmark of the alternative NF-κB pathway is inducible p100 processing, leading to liberation of the mature transcription factor p52 already in complex with RelB (Fig. 2B). This event explains the distinct biological function of the alternative pathway, one not provided by the classical pathway. Like phosphorylated IkBα, phosphorylated p100 is recognized by the E3 ligase SCF<sup>βTrCP</sup> and is targeted for proteasome-dependent proteolysis (39). The exact reason why the proteolysis of p100, in contrast to that of IkB, is partial is not fully clear, but it may have to do with the presence of stop signals that impede proteasome action (105). The alternative NF-κB pathway was discovered through the observation that B cells from IKKα-deficient or IKKα(AA) mutant mice exhibit defective processing of p100, suggesting that B cells are exposed to a physiological ligand that induces p100 processing in an IKKα-dependent pathway (42). This ligand and receptor pair was identified as BAFF and BAFF-R (101).

Two kinases are critically involved in activation of the alternative NF-κB pathway: NIK and IKKα (42, 106). Accordingly, mice carrying an inactivating mutation in the NIK gene, the so-called alymphoplasia (aly) mice or NIK<sup>-/-</sup> mice, have defects in BAFF- and LTβR-induced processing of p100 that are similar to those of IKKα(AA) mice (47, 102). Both IKKβ and IKKγ are dispensable for this pathway (47). There is ample evidence that the catalytic activity of both NIK and IKKα is required for inducible p100 processing. Overexpression of NIK or the constitutive active IKKα(EE) variant induces processing of p100 through phosphorylation of specific serine residues within the C-terminal domain of the latter, thereby creating an SCF<sup>βTrCP</sup> binding site (42, 106). The response to NIK depends on the presence of catalytically active IKKα, and in vitro, NIK is a potent IKKα-activating kinase (107). The IKKα(AA) mutant is not phosphorylated by NIK and fails to induce LTβR- or BAFF-R–dependent processing of p100 (9, 47). Moreover, IKKα has been cloned as a NIK-binding protein, demonstrating direct interaction between the two (43). Thus, the alternative pathway seems to function as a typical

phosphorylation-dependent kinase cascade (Fig. 2B).

However, there are still open questions regarding this pathway, as well. One is the remarkably slow kinetics of p100 processing, requiring several hours, in contrast to that of the classical NF-κB pathway, in which IkBα degradation occurs within minutes. Also, it is not clear why it is so difficult to demonstrate increased IKKα kinase activity in response to either BAFF-R, CD40, RANK, or LTβR engagement (108). It is possible that these problems are related and reflect a low affinity of IKKα for its substrate, p100, in comparison to that of the trimeric IKK complex for IkBα. By itself, IKKα binds only weakly to p100, and the interaction between the two is enhanced in the presence of NIK, implicating at least two functions for NIK—one as an IKKα kinase and the other as an adaptor that facilitates docking of IKKα to its substrate (109, 110). It is well established that protein kinases need to physically dock onto their substrates (110), and in the case of the large IKK complex, such a function may be mediated by the ELKS subunit (8).

NIK activity is regulated by TRAF proteins, particularly TRAF3, and is thought to involve ubiquitination (111). NIK seems to be constitutively bound by TRAF3, resulting in its constitutive ubiquitination and rapid turnover (111). Receptor-induced degradation or sequestration of TRAF3 by the oligomerized receptor precedes processing of p100, presumably leading to increased steady-state amounts of NIK, once it no longer interacts with TRAF3. Accordingly, decreased concentrations of TRAF3 lead to constitutive p100 processing (111). TRAF2-deficient B cells also exhibit constitutive processing of p100, indicating a similar negative regulatory role for TRAF2 (112). Whether TRAF2-dependent activation of the alternative NF-κB pathway also depends on NIK is not known. Also, it is unclear whether activation of NIK itself requires other upstream kinases. Moderate overexpression of NIK is sufficient for signaling, suggesting that it is readily activated by autophosphorylation. Thus, it is possible that reduced NIK degradation after sequestration of TRAF3 to the activated receptor is sufficient to increase NIK activity, leading to activation of IKKα. Together with NIK’s ability to stabilize the interaction of IKKα with p100, this would ultimately lead to p100 processing and activation of p52:RelB (Fig. 2B).

REVIEW

Activation of the NF-κB Pathway by Members of the IL-1R and Toll-Like Receptor (TLR) Family

Members of the IL-1R family, such as IL-1R and IL-18R, and members of the TLR receptor family, such as TLR4, whose activation is triggered by LPS, are also potent activators of the classical NF-κB pathway. The cytoplasmic portions of both IL-1R– and TLR-family members share a common structural motif, the so-called TLR and IL-1R (TIR) homology domain at their cytoplasmic portion (113). Like TNFRs, TIR-containing receptors do not have catalytic activity and use intracellular adaptors and signal-transducing molecules to activate effector pathways (114) (Fig. 3). Homotypic TIR-TIR interactions with a limited set of TIR-containing adaptors explain why more than 15 different receptors trigger only a small number of signaling pathways (114). Genetic evidence and limited biochemical analyses revealed two adaptor molecules, MyD88 (myeloid differentiation primary response gene 88) and TRIF [TIR domain–containing adaptor inducing IFN-β (also called TICAM1)], that define IL-1R and TLR signaling (Fig. 3). For some receptors, such as TLR9 and TLR3, direct binding of the respective adaptor molecule to the cytoplasmic part of the receptor has been demonstrated. Other receptors, such as TLR2 and TLR4, seem to use additional, intermediate TIR-containing adaptor molecules, such as TIRAP [Toll–interleukin-1 receptor (TIR) domain–containing adaptor protein (also called Mal)] and TRAM [TRIF-related adaptor molecule (also called TICAM-2)] (115–118) to engage either MyD88 or TRIF. Some receptors, such as TLR3 or TLR9, signal exclusively through either TRIF or MyD88, respectively, whereas TLR4 uses both (114). Most receptors in these groups, including IL-1R, IL18R, and TLRs 1, 2, 5, 6, 7, 8, and 9, seem to signal primarily through MyD88. MyD88-dependent activation of NF-κB also depends on a TRAF molecule, in this case TRAF6, which is also involved in CD40 signaling (119, 120). TIR-mediated signaling also involves additional molecules that belong to the IRAK (IL-1R associated kinase) family (113). Two of the four IRAKs, IRAK1 and IRAK4, are involved in IL-1R– and TLR-induced activation of NF-κB (121–124). Data from knockout fibroblasts indicate that both IRAK1 and IRAK4 are required for IL-1–dependent activation of NF-κB and JNK (122, 124). However, kinase activity is not required for these effects of IRAK1 and is only partially required for those of IRAK4, at least in human cells (125–128). The IRAKs are required for recruitment of TRAF6 to MyD88, possibly through engagement of other molecules, such as TIFA [TRAF-interacting protein with a forkhead-associated (FHA) domain] (129).

Although it cannot be excluded that IRAKs have functions other than TRAF6 recruitment, direct, IRAK-independent recruitment of TRAF6 to CD40 or artificial dimerization or oligomerization of TRAF6 can initiate activation of NF-κB (67, 120). Moreover, dimerization of MyD88, but not of TRAF6, leads to activation of IRAK1 (130), further supporting the notion that TRAF6 acts downstream of IRAK1 and IRAK4. TRAF6 is structurally similar to other TRAF proteins, especially TRAF2 or TRAF5, and thus may function in a similar manner (66). Indeed, TRAF6 interacts with TAB1, TAB2, and TAB3 (81, 82), and both TAK1- and MEKK3-deficient fibroblasts exhibit defective NF-κB activation in response to IL-1 (83, 85). However, RIP1-(and RIP2- or RIP3-) deficient cells exhibit normal IL-1–induced activation of NF-κB (131). Whether IRAKs function similarly to RIP1 is not clear, although both

IRAKs and RIP1 contain similar structural features (death and kinase domains) (70). Also, similar to RIP1, ubiquitination (and degradation) of IRAK1 follows its activation, although so far this has only been implicated in regulation of the protein level of IRAK1, not as an intrinsic signaling mechanism (132). In support of a role for TAK1 in TIR signaling, TAK1-deficient B cells exhibit defects in NF-κB activation in response to the engagement of TLR9, which signals through MyD88 (85, 133). Signals initiated by LPS and poly(I:C) (polyinosinic-polycytidylic acid, used as an analog of viral double-stranded DNA), which depend either partially (LPS) or completely [poly(I:C)] on TRIF, are also ineffective in the activation of NF-κB in either TAK1-deficient B cells or IRAK4-deficient macrophages (85, 124). However, there is one important difference between MyD88- and TRIF-dependent NF-κB activation: TRAF6 is only essential for signaling by MyD88, but not for signaling through TRIF, at least in bone marrow–derived macrophages (130, 134). So far, there are no data that implicate other known TRAF proteins in TIR-mediated activation of NF-κB. Although TRAF3, which also contains a RING-finger motif, is recruited into MyD88- and TRIF-dependent signaling complexes, it is not involved in IKK activation (130). Instead, TRAF3 is required for recruitment of the IKK-related kinase TBK1 (and probably also for recruitment of IKKε) into the TIR signaling complex, thereby controlling the TLR-mediated expression of IFN-I (see below). The C terminus of TRIF binds to RIP1 through the RHIM domain, and RIP1-deficient fibroblasts exhibit reduced NF-κB activation when stimulated with poly(I:C) (72, 135). However, at least in fibroblasts, TRIF-dependent activation of NF-κB requires autocrine production of TNFα (136), possibly through activation of IRF3, which may explain the involvement of RIP1. More detailed analysis of primary gene-deficient cells will be required to resolve these questions. Taken together, although the past few years have witnessed a substantial increase in our understanding of microbial recognition by TLR family members and TIR-mediated signaling, not much information has been gathered about the molecular details of the mechanism of IKK activation that would extend our knowledge beyond what has been learned from studies of TNFR signaling.

Activation of the NF-κB Pathway by Antigen Receptors

During the past few years, our understanding of T cell receptor (TCR) and B cell receptor (BCR) signaling has progressed remarkably through identification of several molecules that play critical roles in NF-κB activation (Fig. 4) (137). In contrast to the receptors mentioned above, TCR and BCR are directly associated with cytoplasmic protein kinases and initiate signal transduction through tyrosine-phosphorylation events, which ultimately lead to recruitment and activation of protein kinase C isozyymes (PKCθ for TCR and PKCβ for BCR). TCR-mediated activation of PKCθ depends on PDK1 (3-phosphoinositide-dependent kinase–1), which may directly phosphorylate PKCθ at its activation loop (138).

PKC-mediated IKK activation requires several additional molecules, i.e., CARMA1 [CARD-containing MAGUK protein (also called CARD11)], BCL10, and MALT1(mucosa-associated lymphoid tissue lymphoma translocation gene 1), whose function has been substantiated by gene disruption and forward genetic screens (137) (Fig. 4). Studies of mutant cells, as well as transient transfection experiments, show that CARMA1, which appears to be constitutively localized at the plasma mem-

(K399R) mutant, NF-κB activation in response to phorbol ester and ionomycin was only partially reduced (97), suggesting that either other ubiquitination sites can compensate for the loss of K399 or that IKKγ ubiquitination may only have an auxiliary role in IKK activation.

TRAF6 and TRAF2, whose roles were thought to be limited to IL-1R and TNFR signaling, were recently suggested to be also involved in TCR-mediated activation of IKK (145). Inhibition of TRAF6 reduced TCR-induced activation of NF-κB, and purified MALT1 was found to bind and oligomerize TRAF6, stimulating E3 ligase activity toward the Zn finger of IKKγ (145). TRAF6-autoubiquitination was suggested to lead to activation of TAK1, which then may phosphorylate IKKβ. Again, many steps in this complicated scenario need to be confirmed in vivo and backed up by reconstitution experiments with proteins that have specific substitutions at phosphorylation or ubiquitination sites. Nevertheless, TAK1-deficient CD4 single-positive thymocytes were shown to be defective in TCR-mediated NF-κB (and JNK) activation, implying that TAK1 is critically involved in NF-κB activation in mature thymocytes (146). Peripheral CD4-positive TAK1−/−T cells showed no reduction in NF-κB activation, at least in response to phorbol 12-myristate 13-acetate (PMA) and ionomycin stimulation, but they were still clearly defective in JNK activation. Although the mode of stimulation (TCR and CD28 cross-linking versus PMA and ionomycin treatment) is not directly comparable, the results point toward the possibility that TAK1 is an essential component of TCR-mediated NF-κB activation in some T cell populations such as mature thymocytes, whereas in other cell populations, such as effector T cells, its function is replaced by an ill-defined, yet TAK1-independent, pathway. In accordance with such a cell differentiation-dependent role of TAK1 in T cells, two other reports investigating the role of TAK1 in different types of B cells led to remarkably different results. A TAK1-deficient B cell line (DT40) was clearly defective in BCR-mediated NF-κB activation (147). In this report, TAK1 (and IKK) recruitment correlated with PKCβ-dependent phosphorylation of CARMA1, indicating that CARMA1 might serve as a platform to bring the upstream kinase TAK1 into close proximity with its substrate IKKβ, allowing efficient substrate phosphorylation. Whether this co-recruitment of TAK1 and IKKβ is due to direct interaction with CARMA1 has yet to be demonstrated. Also, the relation between TAK1, BCL10, and MALT1 needs to be defined. Surprisingly, another group investigating conditional TAK1-deficient, primary B cells did not observe a notable defect in BCR-dependent activation of NF-κB (85), although BCR-induced activation of JNK was severely impaired, demonstrating efficient deletion of TAK1 in these cells. The discrepancy between these results is clear, but difficult to explain. Akin to the apparently differential role of TAK1 in thymocytes and peripheral T cells, it may be due to the different cell types studied. If so, it is possible that alternative TAK1-dependent and TAK1-independent IKK activation pathways exist also in B cells, which may be dependent on a particular B cell differentiation status and be able to compensate for each other.

Another twist in TCR- and BCR-induced NF-κB activation came from the observation that cells with a loss-of-function mutation in the caspase 8 gene or a caspase 8 deficiency exhibit substantial defects in TCR- and BCR-induced activation of NF-

REVIEW

Cytoplasm

Nucleus

DNA strand break

Fig. 5. NF-κB activation by DNA double-strand breaks (DSBs). Free IKKγ, not attached to IKKα/β, is modified by SUMO attachment in the cytoplasm, and this targets it to the nucleus. In the nucleus, formation of DSBs leads to activation of ATM, which in turn leads to ubiquitination of nuclear IKKγ, possibly through phosphorylation of IKKγ, resulting in replacement of SUMO by ubiquitin chains. The molecular function of PIDD is unclear, but somehow it enhances IKKγ sumoylation and it interacts with RIP1. Ubiquitinated IKKγ is transported to the cytoplasm, where it associates with IKKα and IKKβ to form an active IKK complex. The molecular function of RIP1 in this pathway is not very clear either. It may interact with PIDD and IKKγ in the nucleus, supporting phosphorylation and ubiquitination of the latter or somehow promote IKK activation in the cytoplasm through a kinase-independent mechanism.

kB, but show normal responses to TNFα (148, 149). This is surprising because so far, caspase 8 has only been implicated in FADD (Fas-associated death domain protein)-dependent regulation of cell death (150), and NF-κB activation leads to inhibition of caspase 8 activation by increasing transcription of the c-FLIP gene (151). Stimulation of T cell lines with a TCR agonist induced recruitment of IKKα and IKKβ to BCL10 and MALT in a caspase 8-dependent manner (149). Other signaling pathways, such as mitogen-activated protein kinase (MAPK) cascades, were not affected by loss of caspase 8, implicating a rather specific role in IKK activation. The point of divergence of these pathways from the CARMA1-BCL10-MALT1 module is not entirely clear, and presently it is difficult to place caspase

8 into that pathway. The observation that caspase 8 activity is required for IKK activation is even more puzzling because no obligatory proteolytic step upstream of IKK has been described so far. However, it should be noted that one of the NF-κB activation pathways in *Drosophila*, the so-called IMD pathway, also depends on a caspase (DREDD) for activation (152, 153). The molecular function of DREDD in this pathway is not clear. Given the observation that caspase activity is required for NF-κB activation, it will be very interesting to see whether it acts on one of the molecules already known to be involved in antigen receptor signaling. Taken together, these observations indicate that our understanding of the molecular events leading to IKK activation is far from complete.

# Activation of the NF-κB Pathway by DNA Damage

Stimuli that induce double-stranded breaks (DSBs) in DNA, such as ionizing radiation or topoisomerase inhibitors such as etoposide (VP16), also activate the classical NF-κB pathway in an IKKγ-dependent manner (154, 155). Genetic evidence shows that the kinase ATM (ataxia telangiectasia mutated) is required for this response, although the mechanism by which it activates NF-κB has been enigmatic (156, 157). Recently, a surprising sequence of events was suggested to lead from ATM to IKK activation (Fig. 5) (158). It was observed that IKKγ, probably in a form free of IKKα or IKKβ, is constantly modified by the attachment of SUMO (small ubiquitin-related modifier). This leads to nuclear translocation of a small fraction of the total IKKγ pool (Fig. 5). IKKγ sumoylation requires its C-terminal Zn finger, although the basis for this requirement is unclear. The sumoylation sites, K277 and K309, that were identified by mutational analysis are not located in this region. Inducible IKKγ sumoylation can be mimicked by fusion of SUMO to the N terminus of IKKγ, thereby bypassing the need for the Zn finger (158). In either case, sumoylation of IKKγ induces its nuclear translocation and is important for IKK activation in response to DSBs, but not LPS. Sumoylation of IKKγ is increased in cells overexpressing PIDD (p53-induced protein with a death domain), a p53-inducible protein that was originally described as a proapoptotic molecule (159, 160). PIDD overexpression slightly enhanced DSB-induced IkBa phosphorylation, and the reverse was seen after PIDD suppression, indicating that PIDD is another player in DSB-induced NF-κB activation (160). However, it is not clear how PIDD stimulates IKKγ sumoylation.

IKKγ is ubiquitinated in response to DSB subsequent to its sumoylation, probably at the same lysine residues as those used for SUMO attachment (158). That is, there is a replacement of IKKγ-attached SUMO with polyubiquitin chains. It was thus proposed that SUMO-dependent nuclear translocation places IKKγ next to nuclear ATM, which upon activation by DSBs, phosphorylates IKKγ and triggers its ubiquitination and nuclear export, ultimately resulting in the binding of modified IKKγ to IKKα and IKKβ in the cytoplasm and IKK activation (Fig. 5). Many aspects of this model remain to be confirmed and characterized: The ATM-phosphorylation sites in IKKγ need to be mapped and the ubiquitinating enzymes remain to be identified and characterized. Also, the nuclear export mechanism, which seems to depend on ubiquitination and overrules SUMO-dependent nuclear localization, needs to be identified. It will also be important to see whether ubiquitination of IKKγ is sufficient for activation of IKK, or whether ubiquitination is only required for nuclear export. DSB-mediated activation of NF-κB also depends on RIP1 (161). In this case, autocrine TNFα-dependent NF-κB activation was excluded. Moreover, induction of DSBs with adriamycin induced interaction of RIP1 with IKKγ, which was dependent on ATM activity, implying that RIP1 acts downstream of ATM. As in the case of TNFR1 signaling, the kinase activity of RIP1 is dispensable for DSB-mediated NF-κB activation. Ubiquitination of IKKγ might control its interaction with RIP1, which may serve as a scaffold protein that eventually leads to activation of IKK (Fig. 5). Several aspects of this model are speculative, and the exact mode of interaction of RIP1 and IKKγ and their relation to IKK activation need to be investigated.


## IKK-Related Kinases

Two other protein kinases, that is, IKKε (also called IKK-i) and TBK1 (also called NAK or T2K), have been cloned and found to exhibit sequence similarity to IKKα and IKKβ. IKKε was cloned as a gene that underwent LPS-induced transcription (thus “i” for inducible) and was also identified through a database search for proteins with similarity to IKKα and IKKβ (10, 11). Like IKKα and IKKβ, IKKε contains an N-terminal kinase domain and a C-terminal LZ and a potential HLH motif. The kinase domain shows 33% and 30% sequence identity to IKKα and IKKβ, respectively, and overall 27% identity when compared to full-length IKKα and IKKβ. TBK1, which exhibits 64% homology to IKKε, was first cloned because of its interaction with TANK [TRAF-associated NF-κB activator, also called i-TRAF (TRAF-interacting protein)] (12), a protein that was itself identified by its interaction with TRAF proteins (162, 163). TBK1 was also cloned through application of degenerate primers based on sequences common to IKKα and IKKβ and named NAK (NF-κB–activating kinase) and was also cloned as part of a TRAF2-binding complex that was identified by mass spectrometry and named T2K (13, 14).

In contrast to IKKα and IKKβ, which are very widely expressed, IKKε and TBK1 exhibit a more restricted expression pattern (10, 17). IKKε is preferentially expressed in T cells and peripheral blood cells. TBK1 (but not IKKε) is constitutively expressed in embryonic fibroblasts (164). Furthermore, the abundance of both kinases was demonstrated to increase upon cell stimulation with LPS or TNFα, and for IKKε this up-regulation was shown to depend on NF-κB activity (10, 17, 165). Despite the structural similarities of IKKε and TBK1 to IKKα and IKKβ, neither IKKε nor TBK1 is part of the classical IKKγ-containing IKK complex (10, 11, 166). Nevertheless, it was initially suggested that both kinases also control NF-κB activity. When overexpressed in HEK293 cells, IKKε and TBK1 activate NF-κB and are able to induce phosphorylation of IkBα (10, 11, 12). However, in contrast to IKKβ, IKKε and TBK1 phosphorylate only one of the two IkBα phosphoacceptor sites and thus fail to generate a proper SCF<sup>β-TrCP</sup> recognition site (167). These findings and the observation that TBK1-induced NF-κB activity was reduced in IKKβ-deficient cells led to the suggestion that TBK1 might be an upstream kinase, possibly phosphorylating IKKβ itself (14). However, a more likely scenario is that TBK1 can enhance NF-κB transcriptional activity through phosphorylation of RelA, whose nuclear translocation is IKKβ-dependent (166). Information about the activation of TBK1 by extracellular stimuli is rather limited. Besides viral infection (see below), only PMA and PDGF (platelet-derived growth factor), which activate protein kinase C, and CpG-DNA, which signals via TLR9, were directly shown to activate TBK1 and IKKε (11, 14, 130). Although these results and results obtained with dominant negative molecules indicate a more restricted role of IKKε and TBK1 in NF-κB activation, results from TBK1-deficient mice strongly support the idea that TBK1 is also involved in TNFR1 signaling to NF-κB. TBK1-deficient mice die at E14.5 of massive liver degeneration and apoptosis (13), a phenotype that is very similar to that of IKKγ-, IKKβ-, and RelA-deficient mice (27, 168–171). Furthermore, as with these mutants, the lethality associated with TBK1 deficiency is prevented by disruption of TNFR1, suggesting that TBK1 may be required for TNFα-dependent activation of NF-κB and may thus serve an essential role in prevention of TNFR1-induced

apoptosis of the liver (13). However, more detailed analysis of TBK1-deficient MEFs did not reveal an appreciable decrease in TNFα-induced NF-κB DNA binding activity, but it did uncover a defect in NF-κB transcriptional activity (13). Further support for these observations was provided by identification of NAP1 (NAK-associated protein 1) as a TBK1-interacting protein (166). NAP1, which exhibits a notable level of similarity to TANK, activates TBK1 and enhances its ability to phosphorylate the transactivation domain of RelA, suggesting a mechanism for TBK1-enhanced NF-κB transcriptional activity (166). Indeed, the inhibition of NAP1 decreased TNFα-dependent NF-κB transcriptional activity and increased cellular sensitivity to TNFα-induced apoptosis (166). However, a defect in NF-κB–dependent gene transcription was not confirmed in more recent studies with MEFs from TBK1 and IKKε single- and double-knockout mice (17, 18). Thus, the role of TBK1 and IKKε in NF-κB activation is still not fully understood. Given that TBK1 is also involved in the activation of two other transcription factors (see below), one of which, IRF3, is required for the activation of certain NF-κB target genes (136), it is possible that TBK1-dependent activation of NF-κB target genes in certain cells is mediated through IRFs. However, it should be noted that neither IRF3- nor IRF7-deficient mice exhibit embryonic lethality (172, 173).

Experiments have firmly established a role for TBK1 and IKKε in activation of the IFN-I response through phosphorylation and activation of IRF3 and IRF7 (15–18). The type I interferons include IFN-β and several IFN-α isotypes and are cytokines that signal through a common receptor, the IFN-I receptor (IFNR1) to activate antiviral innate immunity (174). Viral infection or engagement of certain TLRs leads to phosphorylation of IRF3 and IRF7 at defined C-terminal serine residues, causing their dimerization and eventual activation of IRF target genes (172, 175–178). Small amounts of initially produced IFN-I up-regulate the expression of secondary IFN-I genes and IFN-I–responsive genes, including IRF7, by virtue of a feed-forward loop, thereby leading to a robust and measurable IFN-I response (172, 175). TBK1 and IKKε are required for phosphorylation of IRF3 and IRF7 and activation of the IFN-I response (17, 18). As mentioned above, fibroblasts express TBK1, but not IKKε, and stimulation of TBK1-deficient fibroblasts with poly(I:C) results in a reduced and delayed IFN-I response, which is completely abrogated in TBK1-IKKε double-knockout cells (17). Accordingly, Sendai virus-infected TBK1⁻/⁻ cells exhibit impaired IRF3 nuclear translocation and reduced transcription of the IFNβ gene, which are partially restored by reconstitution with exogenous IKKε (164). These data indicate that both the upstream activation mechanisms and the downstream targets for both kinases are highly similar or identical. Indeed, in vitro phosphorylation experiments suggest that TBK1 and IKKε are the direct IRF3 and IRF7 kinases that phosphorylate these TFs on sites shown by mutational analysis to be required for their activation (15, 18, 179–181). Nonetheless, the involvement of other protein kinases in IRF activation cannot be fully excluded.

As in the case of IKK, the upstream events leading to activation of TBK1 and IKK are only partially understood. Studies have revealed several distinct modes of virus recognition that lead to IFN-I production (Fig. 6): (i) TLR-dependent mechanisms, mediated via MyD88- or TRIF-dependent signal transduction pathways. (ii) TLR-independent mechanisms mediated by the RNA helicases RIG-I (retinoic acid-inducible gene I) and Mda5 (melanoma differentiation-associated gene 5), both of which can directly recognize different types of virus-derived RNA (89, 182, 183). RIG-I–dependent signaling relies on a recently identified adaptor protein called IPS-1 (interferon β promoter stimulator 1), MAVS (mitochondrial antiviral

Fig. 6. Activation of IKK-related kinases and expression of IFN-I through TLR-dependent and -independent pathways. TLR engagement leads to recruitment and oligomerization of the adaptor molecules MyD88 and TRIF. Oligomerization of these TIR domain adaptors leads to activation of IRAK family members and recruitment of other molecules including TRAF3 and IRF7 (which interacts with MyD88) or TRAF3 and IRF3 (which interacts with TRIF). TRAF3 is required for assembly of a signaling complex containing IRF3, IRF7, and TBK1 or IKKε, allowing efficient phosphorylation of IRF3 and IRF7. NAP1 is required for TRIF-dependent TBK1 activation. The related protein TANK may serve a function similar to that of NAP1, and both of them interact with TRAF3 and IKK-related kinases. Activated IRFs translocate to the nucleus to drive IFN-I gene induction. Full activation of the IFN-I response depends on concomitant activation of NF-κB, which relies on TRAF6 (in the case of MyD88-dependent signaling) or RIP1 (in the case of TRIF-dependent signaling). Virus-derived single-stranded RNA (ssRNA) is bound by RIG-I or Mda5, possibly leading to a conformational change of these molecules, allowing homotypic (CARD-CARD) interaction of RIG-I or Mda5 with mitochondria-anchored IPS-1. IPS-1 activates TBK1 and IKKε by unknown means, probably also involving TRAF3, resulting in phosphorylation and activation of IRF3 and IRF7. IPS-1 also activates NF-κB through FADD and RIP1. IFN-I up-regulates IFN-I–dependent genes, including IRF-7, thereby amplifying the IFN-I response.

signaling), VISA (virus-induced signaling adaptor), or Cardif (CARD adaptor inducing IFN-β), which seems to act downstream of RIG-I but upstream of both IKK and IKK-related kinases (184–187). Although there is some inconsistency between the different reports regarding the relations between TRIF and IPS-1, most of the data indicate that the TLR- and the RIG-I-mediated pathways are independent, at least at the level of the adaptor molecules they use. The adaptor molecules FADD and RIP1 were also proposed to transduce RIG-I– and IPS-1–dependent signals to NF-κB (184), suggesting that the molecular mechanism of IKK activation is similar to the one used by members of the TNFR family—for instance, FAS, which uses FADD and RIP1 to activate NF-κB (188). Consistent with this suggestion, both FADD- and RIP1-deficient cells show a modest increase in sensitivity to viral infection, accompanied by a small reduction in IFN-I production (189), whose full transcriptional activation requires both IRF and NF-κB activity (189, 190). It is rather unlikely that FADD and RIP1 are also directly involved in IRF activation because overexpression of neither molecule enhances IRF transcriptional activity.

How IPS-1 leads to activation of TBK1 and IKKε is unclear. Interaction of IPS-1 with TBK1 in HEK293 cells has been described in one publication (186), but not confirmed (184), implying either low affinity or indirect interaction between the two. MyD88 and TRIF interact directly with either IRF7 or both IRF3 and IRF7, respectively (191–193). However, how this interaction promotes IRF activation is unclear. TRAF3, whose molecular function in TLR signaling was previously unknown, was identified by mass spectrometry as part of TIR-associated signaling complexes and shown to interact with both MyD88 and TRIF upon stimulation of cells with different TLR agonists or IL-1 (130). Importantly, TRAF3-deficient macrophages and dendritic cells (DCs) exhibited a selective defect in the activation of transcription of the IFN-I and IL-10 genes in response to TLR engagement or viral infection, accompanied by increased production of proinflammatory mediators (130, 194). Moreover, TBK1 recruitment to activated MyD88 is reduced in TRAF3-deficient cells, indicating that TRAF3 is important for recruiting TBK1 into TIR-assembled signaling complexes, which may also contain IRF3 or IRF7, or both, thereby enabling more efficient phosphorylation of IRF proteins (130). Accordingly, overexpression of MyD88 and TRIF in HEK293 cells led to coprecipitation of these adaptors and TRAF3, and TRAF3 overexpression led to coprecipitation of TRAF3 with TBK1 and IKKε (194). However, it should be noted that DCs from TBK1-TNFR1 double-knockout mice do not exhibit a defect in CpG-DNA–induced expression of IFN-I genes, indicating that IKKε might be able to compensate for TBK1 (191). IRAK1-deficient cells also exhibit a defect in CpG-DNA–induced production of IFN-I and IL-10 production (195, 196). However, when overexpressed in HEK293 cells and in contrast to TBK1 and IKKε, IRAK1 does not induce IRF3- and IRF7-dependent gene transcription, indicating that IRAK1 is involved upstream of TBK1 and IKKε and is not a direct IRF7 kinase. This interpretation is in good agreement with the analysis of TLR signaling to NF-κB described above, which showed that IRAK1 acts upstream of TRAF6 in the pathway leading to IKK activation (122, 130, 197). Moreover, IRAK1 can activate the IRF-inducible IFNα4 promoter when coexpressed with TRAF3, but has essentially no effect on its own (194).

IKKα is also required for MyD88-dependent production of IFN-I (198). Although IKKα is controlled by TRAF3 (and NIK) in the alternative NF-κB pathway (see above) and therefore might be expected to be connected in other pathways as well, it appears that TRAF3 and IKKα play distinct roles in TLR-mediated production of IFN-I. Although TRAF3 is involved in both the MyD88-dependent and TRIF-dependent pathways, the function of IKKα seems to be restricted to the MyD88-dependent pathway, activated through TLR9 and TLR7 (130, 194, 198). Moreover, IKKα-deficient cells exhibit not only a defect in TLR9- and TLR7-induced IFN-I production, but also a reduced production of other TLR-induced cytokines, such as IL-12p40. This is in marked contrast to what is seen in TRAF3-deficient cells (see above). The molecular mechanism of IKKα-dependent regulation of IFN-I is not entirely clear. IKKα-deficient FLT3L-induced DCs exhibit a defect in TLR7-triggered ISRE (IFN-stimulated response element)–binding activity, which points toward the possibility that IKKα acts as an IRF7 kinase, controlling its phosphorylation, nuclear translocation, and consequent ISRE-dependent transcription. However, several observations cast doubt on this scenario: In marked contrast to the established IRF kinases TBK1 and IKKε, IKKα induces only a subtle activation of IRF7-dependent transcriptional activity when overexpressed in HEK293 cells. Also, IRF7 appears to be a rather poor substrate for IKKα, as measured by in vitro kinase assays, at least when compared to TBK1 or when compared to the activity of IKKα toward IkBa (3, 130, 198). Moreover, IKKα-deficient FLT3L-induced DCs also show a reduction in TLR7-dependent NF-κB DNA binding activity (198), which points toward the possibility that IKKα performs as a “classic” IkB kinase. Reduced NF-κB activity might then lead to reduced production of IFN-I– and IFN-I–dependent genes, such as IRF7, as well as to reduced transcriptional activation of IL-12p40, whose promoter is controlled by NF-κB (28). It is clear that more detailed analysis of IKKα-deficient cells and mapping of IKKα-dependent IRF7 phosphorylation sites will be required to resolve these issues.

As mentioned above, TBK1 interacts with both TANK and NAP1 (166). For TANK, no data have been published with respect to IFN-I activation, but NAP1 was found to coprecipitate with TRIF when overexpressed in HEK293 cells and its suppression reduces TRIF-dependent IFN-I gene expression (199). Although the exact details of the interactions between these proteins remain to be investigated, it appears that TRIF marshals a complex including TRAF3, NAP1, TBK1 (or IKKε), and IRF3, ultimately resulting in IRF3 phosphorylation, nuclear translocation, and IFN-I gene expression. Whether NAP1 is also involved in other IFN-I–inducing pathways needs to be investigated. The role of TRAF3 in activation of the IFN-I response is independent of its role in the alternative NF-κB pathway (see above). TRAF3–NF-κB2 double-knockout cells exhibit the same phenotype as TRAF3 single-knockout cells with respect to IFN-I production, implying that processing of p100 to NF-κB2 is not directly involved in activation of the IFN-I response (194). It is also unlikely that TRAF3 regulates the stability of TBK1 and IKKε, because the amounts of TRAF3 protein do not change during TLR signaling and those of TBK1 and IKKε are comparable in wild-type and TRAF3-deficient cells. A schematic diagram of the signaling pathways that lead to IRF3 and IRF7 activation is shown in Fig. 6.

Summary
Substantial progress has been made in our understanding of the physiological functions of IKK and its role in NF-κB activation. It is now clear that IKK activation is the critical step in all major pathways leading to NF-κB activation. Many new molecules that are involved in IKK activation in response to occupancy of different receptors have been identified, and their biological roles have been confirmed in gene-deficient mice or even in human disease. Identification of new players in these pathways and characterization of their biological function has further substantiated the overall importance of IKK and NF-κB in the control of immune responses, inflammation, and cancer development. Similarly, recent years have seen an explosive growth in our understanding of the IFN-I response, firmly establishing a critical role of the IKK-related kinases TBK1 and IKKε in the activation of IRF3 and IRF7. Given that NF-κB and IRF3 and IRF7 coactivate a common class of target genes, such an interaction provides a way for integrating both pathways. Yet the exact molecular mechanisms responsible for activation of IKK and its relatives are still surprisingly obscure. Therefore, it is currently not possible to conclude that all participants in the activation of IKK and IKK-related kinases have already been described and just need to be placed into the proper spots in this extensive jigsaw puzzle. The full characterization of the biochemical events that lead to the activation of these kinases is still a major challenge, but accomplishment of this goal promises to provide us with new and improved ways to treat and modulate the outcome of human diseases associated with deregulation or persistent activation of these pathways.

References
1. M. Karin, Y. Ben-Neriah, Phosphorylation meets ubiquitination: The control of NF-[kappa]B activity. *Annu. Rev. Immunol.* 18, 621–663 (2000).
2. D. M. Rothwarf, M. Karin, The NF-kappa B activation pathway: A paradigm in information transfer from membrane to nucleus. *Sci. STKE* 1999, RE1 (1999).
3. J. A. DiDonato, M. Hayakawa, D. M. Rothwarf, E. Zandi, M. Karin, A cytokine-responsive IkappaB kinase that activates the transcription factor NF-kappaB. *Nature* 388, 548–554 (1997).
4. F. Mercurio, H. Zhu, B. W. Murray, A. Shevchenko, B. L. Bennett, J. Li, D. B. Young, M. Barbosa, M. Mann, A. Manning, A. Rao, IKK-1 and IKK-2: Cytokine-activated IkappaB kinases essential for NF-kappaB activation. *Science* 278, 860–866 (1997).
5. E. Zandi, D. M. Rothwarf, M. Delhase, M. Hayakawa, M. Karin, The IkappaB kinase complex (IKK) contains two kinase subunits, IKKalpha and IKKbeta, necessary for IkappaB phosphorylation and NF-kappaB activation. *Cell* 91, 243–252 (1997).
6. D. M. Rothwarf, E. Zandi, G. Natoli, M. Karin, IKK-gamma is an essential regulatory subunit of the IkappaB kinase complex. *Nature* 395, 297–300 (1998).
7. Z. J. Chen, L. Parent, T. Maniatis, Site-specific phosphorylation of IkappaBalpha by a novel ubiquitination-dependent protein kinase activity. *Cell* 84, 853–862 (1996).
8. J. L. Ducut Sigala, V. Bottero, D. B. Young, A. Shevchenko, F. Mercurio, I. M. Verma, Activation of transcription factor NF-kappaB requires ELKS, an IkappaB kinase regulatory subunit. *Science* 304, 1963–1967 (2004).
9. G. Bonizzi, M. Karin, The two NF-kappaB activation pathways and their role in innate and adaptive immunity. *Trends Immunol.* 25, 280–288 (2004).
10. T. Shimada, T. Kawai, K. Takeda, M. Matsumoto, J. Inoue, Y. Tatsumi, A. Kanamaru, S. Akira, IKK-i, a novel lipopolysaccharide-inducible kinase that is related to IkappaB kinases. *Int. Immunol.* 11, 1357–1362 (1999).
11. R. T. Peters, S. M. Liao, T. Maniatis, IKKepsilon is part of a novel PMA-inducible IkappaB kinase complex. *Mol. Cell* 5, 513–522 (2000).
12. J. L. Pomerantz, D. Baltimore, NF-kappaB activation by a signaling complex containing TRAF2, TANK and TBK1, a novel IKK-related kinase. *EMBO J.* 18, 6694–6704 (1999).

13. M. Bonnard, C. Mirtsos, S. Suzuki, K. Graham, J. Huang, M. Ng, A. Itie, A. Wakeham, A. Shahinian, W. J. Henzel, A. J. Elia, W. Shillinglaw, T. W. Mak, Z. Cao, W. C. Yeh, Deficiency of T2K leads to apoptotic liver degeneration and impaired NF-kappaB-dependent gene transcription. *EMBO J.* 19, 4976–4985 (2000).
14. Y. Tojima, A. Fujimoto, M. Delhase, Y. Chen, S. Hatakeyama, K. Nakayama, Y. Kaneko, Y. Nimura, N. Motoyama, K. Ikeda, M. Karin, M. Nakanishi, NAK is an IkappaB kinase-activating kinase. *Nature* 404, 778–782 (2000).
15. S. Sharma, B. R. tenOever, N. Grandvaux, G. P. Zhou, R. Lin, J. Hiscott, Triggering the interferon antiviral response through an IKK-related pathway. *Science* 300, 1148–1151 (2003).
16. K. A. Fitzgerald, S. M. McWhirter, K. L. Faia, D. C. Rowe, E. Latz, D. T. Golenbock, A. J. Coyle, S. M. Liao, T. Maniatis, IKKepsilon and TBK1 are essential components of the IRF3 signaling pathway. *Nat. Immunol.* 4, 491–496 (2003).
17. H. Hemmi, O. Takeuchi, S. Sato, M. Yamamoto, T. Kaisho, H. Sanjo, T. Kawai, K. Hoshino, K. Takeda, S. Akira, The roles of two IkappaB kinase-related kinases in lipopolysaccharide and double stranded RNA signaling and viral infection. *J. Exp. Med.* 199, 1641–1650 (2004).
18. S. M. McWhirter, K. A. Fitzgerald, J. Rosains, D. C. Rowe, D. T. Golenbock, T. Maniatis, IFN-regulatory factor 3-dependent gene expression is defective in Tbk1-deficient mouse embryonic fibroblasts. *Proc. Natl. Acad. Sci. U.S.A.* 101, 233–238 (2004).
19. M. S. Hayden, S. Ghosh, Signaling to NF-kappaB. *Genes Dev.* 18, 2195–2224 (2004).
20. C. M. Fan, T. Maniatis, Generation of p50 subunit of NF-kappa B by processing of p105 through an ATP-dependent pathway. *Nature* 354, 395–398 (1991).
21. V. J. Palombella, O. J. Rando, A. L. Goldberg, T. Maniatis, The ubiquitin-proteasome pathway is required for processing the NF-kappa B1 precursor protein and the activation of NF-kappa B. *Cell* 78, 773–785 (1994).
22. J. C. Betts, G. J. Nabel, Differential regulation of NF-kappaB2(p100) processing and control by amino-terminal sequences. *Mol. Cell. Biol.* 16, 6363–6371 (1996).
23. L. Cohen, J. Hiscott, Heterodimerization and transcriptional activation in vitro by NF-kappa B proteins. *J. Cell. Physiol.* 152, 10–18 (1992).
24. J. Kucharczak, M. J. Simmons, Y. Fan, C. Gelinas, To be, or not to be: NF-kappaB is the answer—role of Rel/NF-kappaB in the regulation of apoptosis. *Oncogene* 22, 8961–8982 (2003).
25. S. Gerondakis, M. Grossmann, Y. Nakamura, T. Pohl, R. Grumont, Genetic approaches in mice to understand Rel/NF-kappaB and IkappaB function: Transgenics and knockouts. *Oncogene* 18, 6888–6895 (1999).
26. L. M. Staudt, H. Singh, R. Sen, T. Wirth, P. A. Sharp, D. Baltimore, A lymphoid-specific protein binding to the octamer motif of immunoglobulin genes. *Nature* 323, 640–643 (1986).
27. A. A. Beg, W. C. Sha, R. T. Bronson, S. Ghosh, D. Baltimore, Embryonic lethality and liver degeneration in mice lacking the RelA component of NF-kappa B. *Nature* 376, 167–170 (1995).
28. T. L. Murphy, M. G. Cleveland, P. Kulesza, J. Magram, K. M. Murphy, Regulation of interleukin 12 p40 expression through an NF-kappa B half-site. *Mol. Cell. Biol.* 15, 5258–5267 (1995).
29. N. J. Solan, H. Miyoshi, E. M. Carmona, G. D. Bren, C. V. Paya, RelB cellular regulation and transcriptional activity are regulated by p100. *J. Biol. Chem.* 277, 1405–1418 (2002).
30. E. Fiorini, I. Schmitz, W. E. Marissen, S. L. Osborn, M. Touma, T. Sasada, P. A. Reche, E. V. Tibaldi, R. E. Hussey, A. M. Kruisbeek, E. L. Reinherz, L. K. Clayton, Peptide-induced negative selection of thymocytes activates transcription of an NF-kappa B inhibitor. *Mol. Cell* 9, 637–648 (2002).
31. H. Kuwata, M. Matsumoto, K. Atarashi, H. Morishita, T. Hirotani, R. Koga, K. Takeda, IkappaBNS inhibits induction of a subset of Toll-like receptor-dependent genes and limits inflammation. *Immunity* 24, 41–51 (2006).
32. T. Fujita, G. P. Nolan, H. C. Liou, M. L. Scott, D. Baltimore, The candidate proto-oncogene bcl-3 encodes a transcriptional coactivator that activates through NF-kappa B p50 homodimers. *Genes Dev.* 7, 1354–1363 (1993).
33. V. Bours, G. Franzoso, V. Azarenko, S. Park, T. Kanno, K. Brown, U. Siebenlist, The oncoprotein Bcl-3 directly transactivates through kappa B motifs via association with DNA-binding p50B homodimers. *Cell* 72, 729–739 (1993).
34. M. Yamamoto, S. Yamazaki, S. Uematsu, S. Sato, H. Hemmi, K. Hoshino, T. Kaisho, H. Kuwata, O. Takeuchi, K. Takeshige, T. Saitoh, S. Yamaoka, N. Yamamoto, S. Yamamoto, T. Muta, K. Takeda, S. Akira, Regulation of Toll/IL-1-receptor-mediated gene expression by the inducible nuclear protein IkappaBzeta. *Nature* 430, 218–222 (2004).

35. K. Brown, S. Park, T. Kanno, G. Franzoso, U. Siebenlist, Mutual regulation of the transcriptional activator NF-kappa B and its inhibitor, I kappa B-alpha. *Proc. Natl. Acad. Sci. U.S.A.* **90**, 2532–2536 (1993).

36. S. C. Sun, P. A. Ganchi, D. W. Ballard, W. C. Greene, NF-kappa B controls expression of inhibitor I kappa B alpha: Evidence for an inducible autoregulatory pathway. *Science* **259**, 1912–1915 (1993).

37. T. Hirotani, P. Y. Lee, H. Kuwata, M. Yamamoto, M. Matsumoto, I. Kawase, S. Akira, K. Takeda, The nuclear IkappaB protein IkappaBNS selectively inhibits lipopolysaccharide-induced IL-6 production in macrophages of the colonic lamina propria. *J. Immunol.* **174**, 3650–3657 (2005).

38. M. Shirane, S. Hatakeyama, K. Hattori, K. Nakayama, K. Nakayama, Common pathway for the ubiquitination of IkappaBalpha, IkappaBbeta, and IkappaBepsilon mediated by the F-box protein FWD1. *J. Biol. Chem.* **274**, 28169–28174 (1999).

39. R. E. Amir, H. Haecker, M. Karin, A. Ciechanover, Mechanism of processing of the NF-kappa B2 p100 precursor: Identification of the specific polyubiquitin chain-anchoring lysine residue and analysis of the role of NEDD8-modification on the SCF(beta-TrCP) ubiquitin ligase. *Oncogene* **23**, 2540–2547 (2004).

40. A. K. Moorthy, O. V. Savinova, J. Q. Ho, V. Y. Wang, D. Vu, G. Ghosh, The 20S proteasome processes NF-kappaB1 p105 into p50 in a translation-independent manner. *EMBO J.* **25**, 1945–1956 (2006).

41. V. Heissmeyer, D. Krappmann, E. N. Hatada, C. Scheidereit, Shared pathways of IkappaB kinase-induced SCF(betaTrCP)-mediated ubiquitination and degradation for the NF-kappaB precursor p105 and IkappaBalpha. *Mol. Cell. Biol.* **21**, 1024–1035 (2001).

42. U. Senftleben, Y. Cao, G. Xiao, F. R. Greten, G. Krahn, G. Bonizzi, Y. Chen, Y. Hu, A. Fong, S. C. Sun, M. Karin, Activation by IKKalpha of a second, evolutionary conserved, NF-kappa B signaling pathway. *Science* **293**, 1495–1499 (2001).

43. C. H. Regnier, H. Y. Song, X. Gao, D. V. Goeddel, Z. Cao, M. Rothe, Identification and characterization of an IkappaB kinase. *Cell* **90**, 373–383 (1997).

44. S. Yamaoka, G. Courtois, C. Bessia, S. T. Whiteside, R. Weil, F. Agou, H. E. Kirk, R. J. Kay, A. Israel, Complementation cloning of NEMO, a component of the IkappaB kinase complex essential for NF-kappaB activation. *Cell* **93**, 1231–1240 (1998).

45. F. Mercurio, B. W. Murray, A. Shevchenko, B. L. Bennett, D. B. Young, J. W. Li, G. Pascual, A. Motiwala, H. Zhu, M. Mann, A. M. Manning, IkappaB kinase (IKK)-associated protein 1, a common component of the heterogeneous IKK complex. *Mol. Cell. Biol.* **19**, 1526–1538 (1999).

46. Y. Li, J. Kang, J. Friedman, L. Tarassishin, J. Ye, A. Kovalenko, D. Wallach, M. S. Horwitz, Identification of a cell protein (FIP-3) as a modulator of NF-kappaB activity and as a target of an adenovirus inhibitor of tumor necrosis factor alpha-induced apoptosis. *Proc. Natl. Acad. Sci. U.S.A.* **96**, 1042–1047 (1999).

47. E. Dejardin, N. M. Droin, M. Delhase, E. Haas, Y. Cao, C. Makris, Z. W. Li, M. Karin, C. F. Ware, D. R. Green, The lymphotoxin-beta receptor induces different patterns of gene expression via two NF-kappaB pathways. *Immunity* **17**, 525–535 (2002).

48. E. Zandi, Y. Chen, M. Karin, Direct phosphorylation of IkappaB by IKKalpha and IKKbeta: Discrimination between free and NF-kappaB-bound substrate. *Science* **281**, 1360–1363 (1998).

49. B. S. Miller, E. Zandi, Complete reconstitution of human IkappaB kinase (IKK) complex in yeast. Assessment of its stoichiometry and the role of IKKgamma on the complex activity in the absence of stimulation. *J. Biol. Chem.* **276**, 36320–36326 (2001).

50. F. Agou, F. Ye, S. Goffinont, G. Courtois, S. Yamaoka, A. Israel, M. Veron, NEMO trimerizes through its coiled-coil C-terminal domain. *J. Biol. Chem.* **277**, 17464–17475 (2002).

51. S. Tegethoff, J. Behlke, C. Scheidereit, Tetrameric oligomerization of IkappaB kinase gamma (IKKgamma) is obligatory for IKK complex activity and NF-kappaB activation. *Mol. Cell. Biol.* **23**, 2029–2041 (2003).

52. M. J. May, R. B. Marienfeld, S. Ghosh, Characterization of the IkappaB-kinase NEMO binding domain. *J. Biol. Chem.* **277**, 45992–46000 (2002).

53. M. J. May, F. D'Acquisto, L. A. Madge, J. Glockner, J. S. Pober, S. Ghosh, Selective inhibition of NF-kappaB activation by a peptide that blocks the interaction of NEMO with the IkappaB kinase complex. *Science* **289**, 1550–1554 (2000).

54. G. Chen, P. Cao, D. V. Goeddel, TNF-induced recruitment and activation of the IKK complex require Cdc37 and Hsp90. *Mol. Cell* **9**, 401–410 (2002).

55. K. Richter, J. Buchner, Hsp90: Chaperoning signal transduction. *J. Cell. Physiol.* **188**, 281–290 (2001).

56. M. Ohara-Imaizumi, T. Ohtsuka, S. Matsushima, Y. Akimoto, C. Nishiwaki, Y. Nakamichi, T. Kikuta, S. Nagai, H. Kawakami, T. Watanabe, S. Nagamatsu, ELKS, a protein structurally related to the active zone-associated protein CAST, is expressed in pancreatic beta cells and functions in insulin exocytosis: Interaction of ELKS with exocytic machinery analyzed by total internal reflection fluorescence microscopy. *Mol. Biol. Cell* **16**, 3289–3300 (2005).

57. M. Delhase, M. Hayakawa, Y. Chen, M. Karin, Positive and negative regulation of IkappaB kinase activity through IKKbeta subunit phosphorylation. *Science* **284**, 309–313 (1999).

58. L. N. Johnson, M. E. Noble, D. J. Owen, Active and inactive protein kinases: Structural basis for regulation. *Cell* **85**, 149–158 (1996).

59. Y. Hu, V. Baud, M. Delhase, P. Zhang, T. Deerinck, M. Ellisman, R. Johnson, M. Karin, Abnormal morphogenesis but intact IKK activation in mice lacking the IKKalpha subunit of IkappaB kinase. *Science* **284**, 316–320 (1999).

60. K. Takeda, O. Takeuchi, T. Tsujimura, S. Itami, O. Adachi, T. Kawai, H. Sanjo, K. Yoshikawa, N. Terada, S. Akira, Limb and skin abnormalities in mice lacking IKKalpha. *Science* **284**, 313–316 (1999).

61. Y. Cao, G. Bonizzi, T. N. Seagroves, F. R. Greten, R. Johnson, E. V. Schmidt, M. Karin, IKKalpha provides an essential link between RANK signaling and cyclin D1 expression during mammary gland development. *Cell* **107**, 763–775 (2001).

62. D. O. Morgan, Principles of CDK regulation. *Nature* **374**, 131–134 (1995).

63. S. Prajapati, U. Verma, Y. Yamamoto, Y. T. Kwak, R. B. Gaynor, Protein phosphatase 2Cbeta association with the IkappaB kinase complex is involved in regulating NF-kappaB activity. *J. Biol. Chem.* **279**, 1739–1746 (2004).

64. P. Ramakrishnan, W. Wang, D. Wallach, Receptor-specific signaling for both the alternative and the canonical NF-kappaB activation pathways by NF-kappaB-inducing kinase. *Immunity* **21**, 477–489 (2004).

65. J. A. Gustin, C. K. Korgaonkar, R. Pincheira, Q. Li, D. B. Donner, Akt regulates basal and induced processing of NF-kappaB2 (p100) to p52. *J. Biol. Chem.* **281**, 16473–16481 (2006).

66. P. W. Dempsey, S. E. Doyle, J. Q. He, G. Cheng, The signaling adapters and pathways activated by TNF superfamily. *Cytokine Growth Factor Rev.* **14**, 193–209 (2003).

67. V. Baud, Z. G. Liu, B. Bennett, N. Suzuki, Y. Xia, M. Karin, Signaling by proinflammatory cytokines: Oligomerization of TRAF2 and TRAF6 is sufficient for JNK and IKK activation and target gene induction via an amino-terminal effector domain. *Genes Dev.* **13**, 1297–1308 (1999).

68. H. Hsu, J. Xiong, D. V. Goeddel, The TNF receptor 1-associated protein TRADD signals cell death and NF-kappa B activation. *Cell* **81**, 495–504 (1995).

69. H. Hsu, J. Huang, H. B. Shu, V. Baichwal, D. V. Goeddel, TNF-dependent recruitment of the protein kinase RIP to the TNF receptor-1 signaling complex. *Immunity* **4**, 387–396 (1996).

70. E. Meylan, J. Tschopp, The RIP kinases: Crucial integrators of cellular stress. *Trends Biochem. Sci.* **30**, 151–159 (2005).

71. A. T. Ting, F. X. Pimentel-Muinos, B. Seed, RIP mediates tumor necrosis factor receptor 1 activation of NF-kappaB but not Fas/APO-1-initiated apoptosis. *EMBO J.* **15**, 6189–6196 (1996).

72. E. Meylan, K. Burns, K. Hofmann, V. Blancheteau, F. Martinon, M. Kelliher, J. Tschopp, RIP1 is an essential mediator of Toll-like receptor 3-induced NF-kappa B activation. *Nat. Immunol.* **5**, 503–507 (2004).

73. W. C. Yeh, A. Shahinian, D. Speiser, J. Kraanus, F. Billia, A. Wakeham, J. L. de la Pompa, D. Ferrick, B. Hum, N. Iscove, P. Ohashi, M. Rothe, D. V. Goeddel, T. W. Mak, Early lethality, functional NF-kappaB activation, and increased sensitivity to TNF-induced cell death in TRAF2-deficient mice. *Immunity* **7**, 715–725 (1997).

74. K. Tada, T. Okazaki, S. Sakon, T. Kobara, K. Kurosawa, S. Yamaoka, H. Hashimoto, T. W. Mak, H. Yagita, K. Okumura, W. C. Yeh, H. Nakano, Critical roles of TRAF2 and TRAF5 in tumor necrosis factor-induced NF-kappa B activation and protection from cell death. *J. Biol. Chem.* **276**, 36530–36534 (2001).

75. A. Devin, Y. Lin, S. Yamaoka, Z. Li, M. Karin, Z. Liu, The alpha and beta subunits of IkappaB kinase (IKK) mediate TRAF2-dependent IKK recruitment to tumor necrosis factor (TNF) receptor 1 in response to TNF. *Mol. Cell. Biol.* **21**, 3986–3994 (2001).

76. S. Q. Zhang, A. Kovalenko, G. Cantarella, D. Wallach, Recruitment of the IKK signalosome to the p55 TNF receptor: RIP and A20 bind to NEMO (IKKgamma) upon receptor stimulation. *Immunity* **12**, 301–311 (2000).

77. C. K. Ea, L. Deng, Z. P. Xia, G. Pineda, Z. J. Chen, Activation of IKK by TNFalpha requires site-specific ubiquitination of RIP1 and polyubiquitin binding by NEMO. *Mol. Cell* **22**, 245–257 (2006).

78. T. H. Lee, J. Shank, N. Cusson, M. A. Kelliher, The kinase activity of Rip1 is not required for tumor necrosis factor-alpha-induced IkappaB kinase or p38 MAP kinase activation or for the ubiquitination of Rip1 by Traf2. *J. Biol. Chem.* **279**, 33185–33191 (2004).

79. M. Gao, M. Karin, Regulating the regulators: Control of protein ubiquitination and ubiquitin-like modifications by extracellular stimuli. *Mol. Cell* **19**, 581–593 (2005).

80. Q. Zhao, F. S. Lee, Mitogen-activated protein kinase/ERK kinase kinases 2 and 3 activate nuclear factor-kappaB through IkappaB kinase-alpha and IkappaB kinase-beta. J. Biol. Chem. 274, 8355–8358 (1999).
81. C. Wang, L. Deng, M. Hong, G. R. Akkaraju, J. Inoue, Z. J. Chen, TAK1 is a ubiquitin-dependent kinase of MKK and IKK. Nature 412, 346–351 (2001).
82. T. Ishitani, G. Takaesu, J. Ninomiya-Tsuji, H. Shibuya, R. B. Gaynor, K. Matsumoto, Role of the TAB2-related protein TAB3 in IL-1 and TNF signaling. EMBO J. 22, 6277–6288 (2003).
83. J. Yang, Y. Lin, Z. Guo, J. Cheng, J. Huang, L. Deng, W. Liao, Z. Chen, Z. Liu, B. Su, The essential role of MEKK3 in TNF-induced NF-kappaB activation. Nat. Immunol. 2, 620–624 (2001).
84. Q. Huang, J. Yang, Y. Lin, C. Walker, J. Cheng, Z. G. Liu, B. Su, Differential regulation of interleukin 1 receptor and Toll-like receptor signaling by MEKK3. Nat. Immunol. 5, 98–103 (2004).
85. S. Sato, H. Sanjo, K. Takeda, J. Ninomiya-Tsuji, M. Yamamoto, T. Kawai, K. Matsumoto, O. Takeuchi, S. Akira, Essential function for the kinase TAK1 in innate and adaptive immune responses. Nat. Immunol. 6, 1087–1095 (2005).
86. J. H. Shim, C. Xiao, A. E. Paschal, S. T. Bailey, P. Rao, M. S. Hayden, K. Y. Lee, C. Bussey, M. Steckel, N. Tanaka, G. Yamada, S. Akira, K. Matsumoto, S. Ghosh, TAK1, but not TAB1 or TAB2, plays an essential role in multiple signaling pathways in vivo. Genes Dev. 19, 2668–2681 (2005).
87. J. Yang, M. Boerm, M. McCarty, C. Bucana, I. J. Fidler, Y. Zhuang, B. Su, Mekk3 is essential for early embryonic cardiovascular development. Nat. Genet. 24, 309–313 (2000).
88. M. Blonska, Y. You, R. Geleziunas, X. Lin, Restoration of NF-kappaB activation by tumor necrosis factor alpha receptor complex-targeted MEKK3 in receptor-interacting protein-deficient cells. Mol. Cell. Biol. 24, 10757–10765 (2004).
89. H. Kato, O. Takeuchi, S. Sato, M. Yoneyama, M. Yamamoto, K. Matsui, S. Uematsu, A. Jung, T. Kawai, K. J. Ishii, O. Yamaguchi, K. Otsu, T. Tsujimura, C. S. Koh, C. Reis e Sousa, Y. Matsuura, T. Fujita, S. Akira, Differential roles of MDA5 and RIG-I helicases in the recognition of RNA viruses. Nature 441, 101–105 (2006).
90. H. Sakurai, N. Shigemori, K. Hasegawa, T. Sugita, TGF-beta-activated kinase 1 stimulates NF-kappa B activation by an NF-kappa B-inducing kinase-independent mechanism. Biochem. Biophys. Res. Commun. 243, 545–549 (1998).
91. J. Ninomiya-Tsuji, K. Kishimoto, A. Hiyama, J. Inoue, Z. Cao, K. Matsumoto, The kinase TAK1 can activate the NIK-I kappaB as well as the MAP kinase cascade in the IL-1 signalling pathway. Nature 398, 252–256 (1999).
92. P. C. Cheung, A. R. Nebreda, P. Cohen, TAB3, a new binding partner of the protein kinase TAK1. Biochem. J. 378, 27–34 (2004).
93. Y. Komatsu, H. Shibuya, N. Takeda, J. Ninomiya-Tsuji, T. Yasui, K. Miyado, T. Sekimoto, N. Ueno, K. Matsumoto, G. Yamada, Targeted disruption of the Tab1 gene causes embryonic lethality and defects in cardiovascular and lung morphogenesis. Mech. Dev. 119, 239–249 (2002).
94. A. Kanayama, R. B. Seth, L. Sun, C. K. Ea, M. Hong, A. Shaito, Y. H. Chiu, L. Deng, Z. J. Chen, TAB2 and TAB3 activate the NF-kappaB pathway through binding to polyubiquitin chains. Mol. Cell 15, 535–548 (2004).
95. L. Deng, C. Wang, E. Spencer, L. Yang, A. Braun, J. You, C. Slaughter, C. Pickart, Z. J. Chen, Activation of the IkappaB kinase complex by TRAF6 requires a dimeric ubiquitin-conjugating enzyme complex and a unique polyubiquitin chain. Cell 103, 351–361 (2000).
96. D. F. Legler, O. Micheau, M. A. Doucey, J. Tschopp, C. Bron, Recruitment of TNF receptor 1 to lipid rafts is essential for TNFalpha-mediated NF-kappaB activation. Immunity 18, 655–664 (2003).
97. H. Zhou, I. Wertz, K. O'Rourke, M. Ultsch, S. Seshagiri, M. Eby, W. Xiao, V. M. Dixit, Bcl10 activates the NF-kappaB pathway through ubiquitination of NEMO. Nature 427, 167–171 (2004).
98. M. Yamamoto, T. Okamoto, K. Takeda, S. Sato, H. Sanjo, S. Uematsu, T. Saitoh, N. Yamamoto, H. Sakurai, K. J. Ishii, S. Yamaoka, T. Kawai, Y. Matsuura, O. Takeuchi, S. Akira, Key function for the Ubc13 E2 ubiquitin-conjugating enzyme in immune receptor signaling. Nat. Immunol. 7, 962–970 (2006).
99. Z. J. Chen, Ubiquitin signalling in the NF-kappaB pathway. Nat. Cell Biol. 7, 758–765 (2005).
100. N. Kobayashi, Y. Kadono, A. Naito, K. Matsumoto, T. Yamamoto, S. Tanaka, J. Inoue, Segregation of TRAF6-mediated signaling pathways clarifies its role in osteoclastogenesis. EMBO J. 20, 1271–1280 (2001).
101. N. Kayagaki, M. Yan, D. Seshasayee, H. Wang, W. Lee, D. M. French, I. S. Grewal, A. G. Cochran, N. C. Gordon, J. Yin, M. A. Starovasnik, V. M. Dixit, BAFF/BLyS receptor 3 binds the B cell survival factor BAFF ligand through a discrete surface loop and promotes processing of NF-kappaB2. Immunity 17, 515–524 (2002).

102. E. Claudio, K. Brown, S. Park, H. Wang, U. Siebenlist, BAFF-induced NEMO-independent processing of NF-kappa B2 in maturing B cells. Nat. Immunol. 3, 958–965 (2002).
103. H. J. Coope, P. G. Atkinson, B. Huhse, M. Belich, J. Janzen, M. J. Holman, G. G. Klaus, L. H. Johnston, S. C. Ley, CD40 regulates the processing of NF-kappaB2 p100 to p52. EMBO J. 21, 5375–5385 (2002).
104. D. V. Novack, L. Yin, A. Hagen-Stapleton, R. D. Schreiber, D. V. Goeddel, F. P. Ross, S. L. Teitelbaum, The IkappaB function of NF-kappaB2 p100 controls stimulated osteoclastogenesis. J. Exp. Med. 198, 771–781 (2003).
105. L. Lin, S. Ghosh, A glycine-rich region in NF-kappaB p105 functions as a processing signal for the generation of the p50 subunit. Mol. Cell. Biol. 16, 2248–2254 (1996).
106. G. Xiao, E. W. Harhaj, S. C. Sun, NF-kappaB-inducing kinase regulates the processing of NF-kappaB2 p100. Mol. Cell 7, 401–409 (2001).
107. L. Ling, Z. Cao, D. V. Goeddel, NF-kappaB-inducing kinase activates IKK-alpha by phosphorylation of Ser-176. Proc. Natl. Acad. Sci. U.S.A. 95, 3792–3797 (1998).
108. H. Häcker, M. Karin, unpublished observation.
109. G. Xiao, A. Fong, S. C. Sun, Induction of p100 processing by NF-kappaB-inducing kinase involves docking IkappaB kinase alpha (IKKalpha) to p100 and IKKalpha-mediated phosphorylation. J. Biol. Chem. 279, 30099–30105 (2004).
110. T. Kallunki, T. Deng, M. Hibi, M. Karin, c-Jun can recruit JNK to phosphorylate dimerization partners via specific docking interactions. Cell 87, 929–939 (1996).
111. G. Liao, M. Zhang, E. W. Harhaj, S. C. Sun, Regulation of the NF-kappaB-inducing kinase by tumor necrosis factor receptor-associated factor 3-induced degradation. J. Biol. Chem. 279, 26243–26250 (2004).
112. A. P. Grech, M. Amesbury, T. Chan, S. Gardam, A. Basten, R. Brink, TRAF2 differentially regulates the canonical and noncanonical pathways of NF-kappaB activation in mature B cells. Immunity 21, 629–642 (2004).
113. M. U. Martin, H. Wesche, Summary and comparison of the signaling mechanisms of the Toll/interleukin-1 receptor family. Biochim. Biophys. Acta 1592, 265–280 (2002).
114. M. Yamamoto, S. Akira, TIR domain-containing adaptors regulate TLR signaling pathways. Adv. Exp. Med. Biol. 560, 1–9 (2005).
115. K. A. Fitzgerald, E. M. Palsson-McDermott, A. G. Bowie, C. A. Jefferies, A. S. Mansell, G. Brady, E. Brint, A. Dunne, P. Gray, M. T. Harte, D. McMurray, D. E. Smith, J. E. Sims, T. A. Bird, L. A. O'Neill, Mal (MyD88-adaptor-like) is required for Toll-like receptor-4 signal transduction. Nature 413, 78–83 (2001).
116. T. Horng, G. M. Barton, R. A. Flavell, R. Medzhitov, The adaptor molecule TIRAP provides signalling specificity for Toll-like receptors. Nature 420, 329–333 (2002).
117. M. Yamamoto, S. Sato, H. Hemmi, S. Uematsu, K. Hoshino, T. Kaisho, O. Takeuchi, K. Takeda, S. Akira, TRAM is specifically involved in the Toll-like receptor 4-mediated MyD88-independent signaling pathway. Nat. Immunol. 4, 1144–1150 (2003).
118. H. Oshiumi, M. Sasai, K. Shida, T. Fujita, M. Matsumoto, T. Seya, TIR-containing adapter molecule (TICAM)-2, a bridging adapter recruiting to toll-like receptor 4 TICAM-1 that induces interferon-beta. J. Biol. Chem. 278, 49751–49762 (2003).
119. Z. Cao, J. Xiong, M. Takeuchi, T. Kurama, D. V. Goeddel, TRAF6 is a signal transducer for interleukin-1. Nature 383, 443–446 (1996).
120. T. Ishida, S. Mizushima, S. Azuma, N. Kobayashi, T. Tojo, K. Suzuki, S. Aizawa, T. Watanabe, G. Mosialos, E. Kieff, T. Yamamoto, J. Inoue, Identification of TRAF6, a novel tumor necrosis factor receptor-associated factor protein that mediates signaling from an amino-terminal domain of the CD40 cytoplasmic region. J. Biol. Chem. 271, 28745–28748 (1996).
121. Z. Cao, W. J. Henzel, X. Gao, IRAK: A kinase associated with the interleukin-1 receptor. Science 271, 1128–1131 (1996).
122. J. A. Thomas, J. L. Allen, M. Tsen, T. Dubnicoff, J. Danao, X. C. Liao, Z. Cao, S. A. Wasserman, Impaired cytokine signaling in mice lacking the IL-1 receptor-associated kinase. J. Immunol. 163, 978–984 (1999).
123. S. Li, A. Strelow, E. J. Fontana, H. Wesche, IRAK-4: A novel member of the IRAK family with the properties of an IRAK-kinase. Proc. Natl. Acad. Sci. U.S.A. 99, 5567–5572 (2002).
124. N. Suzuki, S. Suzuki, G. S. Duncan, D. G. Millar, T. Wada, C. Mirtsos, H. Takada, A. Wakeham, A. Itie, S. Li, J. M. Penninger, H. Wesche, P. S. Ohashi, T. W. Mak, W. C. Yeh, Severe impairment of interleukin-1 and Toll-like receptor signaling in mice lacking IRAK-4. Nature 416, 750–756 (2002).
125. X. Li, M. Commane, C. Burns, K. Vithalani, Z. Cao, G. R. Stark, Mutant cells that do not respond to interleukin-1 (IL-1) reveal a novel role for IL-1 receptor-associated kinase. Mol. Cell. Biol. 19, 4643–4652 (1999).
126. J. Knop, M. U. Martin, Effects of IL-1 receptor-associated kinase (IRAK) expression on IL-1 signaling are independent of its kinase activity. FEBS Lett. 448, 81–85 (1999).

127. E. Lye, C. Mirtsos, N. Suzuki, S. Suzuki, W. C. Yeh, The role of interleukin 1 receptor-associated kinase-4 (IRAK-4) kinase activity in IRAK-4-mediated signaling. J. Biol. Chem. 279, 40653–40658 (2004).
128. J. Qin, Z. Jiang, Y. Qian, J. L. Casanova, X. Li, IRAK4 kinase activity is redundant for interleukin-1 (IL-1) receptor-associated kinase phosphorylation and IL-1 responsiveness. J. Biol. Chem. 279, 26748–26753 (2004).
129. H. Takatsuna, H. Kato, J. Gohda, T. Akiyama, A. Moriya, Y. Okamoto, Y. Yamagata, M. Otsuka, K. Umezawa, K. Semba, J. Inoue, Identification of TIFA as an adapter protein that links tumor necrosis factor receptor-associated factor 6 (TRAF6) to interleukin-1 (IL-1) receptor-associated kinase-1 (IRAK-1) in IL-1 receptor signaling. J. Biol. Chem. 278, 12144–12150 (2003).
130. H. Häcker, V. Redecke, B. Blagoev, I. Kratchmarova, L. C. Hsu, G. G. Wang, M. P. Kamps, E. Raz, H. Wagner, G. Häcker, M. Mann, M. Karin, Specificity in Toll-like receptor signaling through distinct effector functions of TRAF3 and TRAF6. Nature 439, 204–207 (2006).
131. A. Devin, A. Cook, Y. Lin, Y. Rodriguez, M. Kelliher, Z. Liu, The distinct roles of TRAF2 and RIP in IKK activation by TNF-R1: TRAF2 recruits IKK to TNF-R1 while RIP mediates IKK activation. Immunity 12, 419–429 (2000).
132. T. T. Yamin, D. K. Miller, The interleukin-1 receptor-associated kinase is degraded by proteasomes following its phosphorylation. J. Biol. Chem. 272, 21540–21547 (1997).
133. H. Häcker, R. M. Vabulas, O. Takeuchi, K. Hoshino, S. Akira, H. Wagner, Immune cell activation by bacterial CpG-DNA through myeloid differentiation marker 88 and tumor necrosis factor receptor-associated factor (TRAF)6. J. Exp. Med. 192, 595–600 (2000).
134. J. Gohda, T. Matsumura, J. Inoue, Cutting edge: TNFR-associated factor (TRAF) 6 is essential for MyD88-dependent pathway but not toll/IL-1 receptor domain-containing adaptor-inducing IFN-beta (TRIF)-dependent pathway in TLR signaling. J. Immunol. 173, 2913–2917 (2004).
135. N. Cusson-Hermance, S. Khurana, T. H. Lee, K. A. Fitzgerald, M. A. Kelliher, Rip1 mediates the Trif-dependent toll-like receptor 3- and 4-induced NF-kappaB activation but does not contribute to interferon regulatory factor 3 activation. J. Biol. Chem. 280, 36560–36566 (2005).
136. M. W. Covert, T. H. Leung, J. E. Gaston, D. Baltimore, Achieving stability of lipopolysaccharide-induced NF-kappaB activation. Science 309, 1854–1857 (2005).
137. M. Thome, CARMA1, BCL-10 and MALT1 in lymphocyte development and activation. Nat. Rev. Immunol. 4, 348–359 (2004).
138. K. Y. Lee, F. D'Acquisto, M. S. Hayden, J. H. Shim, S. Ghosh, PDK1 nucleates T cell receptor-induced signaling complex for NF-kappaB activation. Science 308, 114–118 (2005).
139. O. Gaide, B. Favier, D. F. Legler, D. Bonnet, B. Brissoni, S. Valitutti, C. Bron, J. Tschopp, M. Thome, CARMA1 is a critical lipid raft-associated regulator of TCR-induced NF-kappa B activation. Nat. Immunol. 3, 836–843 (2002).
140. D. Wang, R. Matsumoto, Y. You, T. Che, X. Y. Lin, S. L. Gaffen, X. Lin, CD3/CD28 costimulation-induced NF-kappaB activation is mediated by recruitment of protein kinase C-theta, Bcl10, and IkappaB kinase beta to the immunological synapse through CARMA1. Mol. Cell. Biol. 24, 164–171 (2004).
141. N. S. C. van Oers, Z. J. Chen, Cell biology. Kinasing and clipping down the NF-kappa B trail. Science 308, 65–66 (2005).
142. K. Sommer, B. Guo, J. L. Pomerantz, A. D. Bandaranayake, M. E. Moreno-Garcia, Y. L. Ovechkina, D. J. Rawlings, Phosphorylation of the CARMA1 linker controls NF-kappaB activation. Immunity 23, 561–574 (2005).
143. R. Matsumoto, D. Wang, M. Blonska, H. Li, M. Kobayashi, B. Pappu, Y. Chen, D. Wang, X. Lin, Phosphorylation of CARMA1 plays a critical role in T cell receptor-mediated NF-kappaB activation. Immunity 23, 575–585 (2005).
144. P. C. Lucas, M. Yonezumi, N. Inohara, L. M. McAllister-Lucas, M. E. Abazeed, F. F. Chen, S. Yamaoka, M. Seto, G. Nunez, Bcl10 and MALT1, independent targets of chromosomal translocation in malt lymphoma, cooperate in a novel NF-kappa B signaling pathway. J. Biol. Chem. 276, 19012–19019 (2001).
145. L. Sun, L. Deng, C. K. Ea, Z. P. Xia, Z. J. Chen, The TRAF6 ubiquitin ligase and TAK1 kinase mediate IKK activation by BCL10 and MALT1 in T lymphocytes. Mol. Cell 14, 289–301 (2004).
146. Y. Y. Wan, H. Chi, M. Xie, M. D. Schneider, R. A. Flavell, The kinase TAK1 integrates antigen and cytokine receptor signaling for T cell development, survival and function. Nat. Immunol. 7, 851–858 (2006).
147. H. Shinohara, T. Yasuda, Y. Aiba, H. Sanjo, M. Hamadate, H. Watarai, H. Sakurai, T. Kurosaki, PKC beta regulates BCR-mediated IKK activation by facilitating the interaction between TAK1 and CARMA1. J. Exp. Med. 202, 1423–1431 (2005).
148. H. J. Chun, L. Zheng, M. Ahmad, J. Wang, C. K. Speirs, R. M. Siegel, J. K. Dale, J. Puck, J. Davis, C. G. Hall, S. Skoda-Smith, T. P. Atkinson, S.

E. Straus, M. J. Lenardo, Pleiotropic defects in lymphocyte activation caused by caspase-8 mutations lead to human immunodeficiency. Nature 419, 395–399 (2002).
149. H. Su, N. Bidere, L. Zheng, A. Cubre, K. Sakai, J. Dale, L. Salmena, R. Hakem, S. Straus, M. Lenardo, Requirement for caspase-8 in NF-kappaB activation by antigen receptor. Science 307, 1465–1468 (2005).
150. M. Muzio, A. M. Chinnaiyan, F. C. Kischkel, K. O'Rourke, A. Shevchenko, J. Ni, C. Scaffidi, J. D. Bretz, M. Zhang, R. Gentz, M. Mann, P. H. Krammer, M. E. Peter, V. M. Dixit, FLICE, a novel FADD-homologous ICE/CED-3-like protease, is recruited to the CD95 (Fas/APO-1) death-inducing signaling complex. Cell 85, 817–827 (1996).
151. W. C. Yeh, A. Itie, A. J. Elia, M. Ng, H. B. Shu, A. Wakeham, C. Mirtsos, N. Suzuki, M. Bonnard, D. V. Goeddel, T. W. Mak, Requirement for Casper (c-FLIP) in regulation of death receptor-induced apoptosis and embryonic development. Immunity 12, 633–642 (2000).
152. R. S. Khush, F. Leulier, B. Lemaitre, Drosophila immunity: Two paths to NF-kappaB. Trends Immunol. 22, 260–264 (2001).
153. S. Stoven, I. Ando, L. Kadalayil, Y. Engstrom, D. Hultmark, Activation of the Drosophila NF-kappaB factor Relish by rapid endoproteolytic cleavage. EMBO Rep. 1, 347–352 (2000).
154. N. Li, M. Karin, Ionizing radiation and short wavelength UV activate NF-kappaB through two distinct mechanisms. Proc. Natl. Acad. Sci. U.S.A. 95, 13012–13017 (1998).
155. T. T. Huang, S. L. Feinberg, S. Suryanarayanan, S. Miyamoto, The zinc finger domain of NEMO is selectively required for NF-kappa B activation by UV radiation and topoisomerase inhibitors. Mol. Cell. Biol. 22, 5813–5825 (2002).
156. S. J. Lee, A. Dmitchev, M. F. Lavin, A. Dritschilo, M. Jung, A novel ionizing radiation-induced signaling pathway that activates the transcription factor NF-kappaB. Oncogene 17, 1821–1826 (1998).
157. B. Piret, S. Schoonbroodt, J. Piette, The ATM protein is required for sustained activation of NF-kappaB following DNA damage. Oncogene 18, 2261–2271 (1999).
158. T. T. Huang, S. M. Wuerzberger-Davis, Z. H. Wu, S. Miyamoto, Sequential modification of NEMO/IKKgamma by SUMO-1 and ubiquitin mediates NF-kappaB activation by genotoxic stress. Cell 115, 565–576 (2003).
159. A. Tinel, J. Tschopp, The PIDDosome, a protein complex implicated in activation of caspase-2 in response to genotoxic stress. Science 304, 843–846 (2004).
160. S. Janssens, A. Tinel, S. Lippens, J. Tschopp, PIDD mediates NF-kappaB activation in response to DNA damage. Cell 123, 1079–1092 (2005).
161. G. M. Hur, J. Lewis, Q. Yang, Y. Lin, H. Nakano, S. Nedospasov, Z. G. Liu, The death domain kinase RIP has an essential role in DNA damage-induced NF-kappa B activation. Genes Dev. 17, 873–882 (2003).
162. G. Cheng, D. Baltimore, TANK, a co-inducer with TRAF2 of TNF- and CD 40L-mediated NF-kappaB activation. Genes Dev. 10, 963–973 (1996).
163. M. Rothe, J. Xiong, H. B. Shu, K. Williamson, A. Goddard, D. V. Goeddel, I-TRAF is a novel TRAF-interacting protein that regulates TRAF-mediated signal transduction. Proc. Natl. Acad. Sci. U.S.A. 93, 8241–8246 (1996).
164. A. K. Perry, E. K. Chow, J. B. Goodnough, W. C. Yeh, G. Cheng, Differential requirement for TANK-binding kinase-1 in type I interferon responses to toll-like receptor activation and viral infection. J. Exp. Med. 199, 1651–1658 (2004).
165. V. V. Kravchenko, J. C. Mathison, K. Schwamborn, F. Mercurio, R. J. Ulevitch, IKKi/IKKepsilon plays a key role in integrating signals induced by pro-inflammatory stimuli. J. Biol. Chem. 278, 26612–26619 (2003).
166. F. Fujita, Y. Taniguchi, T. Kato, Y. Narita, A. Furuya, T. Ogawa, H. Sakurai, T. Joh, M. Itoh, M. Delhase, M. Karin, M. Nakanishi, Identification of NAP1, a regulatory subunit of IkappaB kinase-related kinases that potentiates NF-kappaB signaling. Mol. Cell. Biol. 23, 7780–7793 (2003).
167. K. Brown, S. Gerstberger, L. Carlson, G. Franzoso, U. Siebenlist, Control of I kappa B-alpha proteolysis by site-specific, signal-induced phosphorylation. Science 267, 1485–1488 (1995).
168. D. Rudolph, W. C. Yeh, A. Wakeham, B. Rudolph, D. Nallainathan, J. Potter, A. J. Elia, T. W. Mak, Severe liver degeneration and lack of NF-kappaB activation in NEMO/IKKgamma-deficient mice. Genes Dev. 14, 854–862 (2000).
169. C. Makris, V. L. Godfrey, G. Krahn-Senftleben, T. Takahashi, J. L. Roberts, T. Schwarz, L. Feng, R. S. Johnson, M. Karin, Female mice heterozygous for IKK gamma/NEMO deficiencies develop a dermatopathy similar to the human X-linked disorder incontinentia pigmenti. Mol. Cell 5, 969–979 (2000).
170. Q. Li, D. Van Antwerp, F. Mercurio, K. F. Lee, I. M. Verma, Severe liver degeneration in mice lacking the IkappaB kinase 2 gene. Science 284, 321–325 (1999).

171. Z. W. Li, W. Chu, Y. Hu, M. Delhase, T. Deerinck, M. Ellisman, R. Johnson, M. Karin, The IKKbeta subunit of IkappaB kinase (IKK) is essential for nuclear factor kappaB activation and prevention of apoptosis. *J. Exp. Med.* **189**, 1839–1845 (1999).
172. M. Sato, H. Suemori, N. Hata, M. Asagiri, K. Ogasawara, K. Nakao, T. Nakaya, M. Katsuki, S. Noguchi, N. Tanaka, T. Taniguchi, Distinct and essential roles of transcription factors IRF-3 and IRF-7 in response to viruses for IFN-alpha/beta gene induction. *Immunity* **13**, 539–548 (2000).
173. K. Honda, H. Yanai, H. Negishi, M. Asagiri, M. Sato, T. Mizutani, N. Shimada, Y. Ohba, A. Takaoka, N. Yoshida, T. Taniguchi, IRF-7 is the master regulator of type-I interferon-dependent immune responses. *Nature* **434**, 772–777 (2005).
174. D. Novick, B. Cohen, M. Rubinstein, The human interferon alpha/beta receptor: Characterization and molecular cloning. *Cell* **77**, 391–400 (1994).
175. I. Marie, J. E. Durbin, D. E. Levy, Differential viral induction of distinct interferon-alpha genes by positive feedback through interferon regulatory factor-7. *EMBO J.* **17**, 6660–6669 (1998).
176. R. Lin, C. Heylbroeck, P. M. Pitha, J. Hiscott, Virus-dependent phosphorylation of the IRF-3 transcription factor regulates nuclear translocation, transactivation potential, and proteasome-mediated degradation. *Mol. Cell. Biol.* **18**, 2986–2996 (1998).
177. M. Yoneyama, W. Suhara, Y. Fukuhara, M. Fukuda, E. Nishida, T. Fujita, Direct triggering of the type I interferon system by virus infection: Activation of a transcription factor complex containing IRF-3 and CBP/p300. *EMBO J.* **17**, 1087–1095 (1998).
178. B. K. Weaver, K. P. Kumar, N. C. Reich, Interferon regulatory factor 3 and CREB-binding protein/p300 are subunits of double-stranded RNA-activated transcription factor DRAF1. *Mol. Cell. Biol.* **18**, 1359–1368 (1998).
179. A. Caillaud, A. G. Hovanessian, D. E. Levy, I. J. Marie, Regulatory serine residues mediate phosphorylation-dependent and phosphorylation-independent activation of interferon regulatory factor 7. *J. Biol. Chem.* **280**, 17671–17677 (2005).
180. B. R. tenOever, S. Sharma, W. Zou, Q. Sun, N. Grandvaux, I. Julkunen, H. Hemmi, M. Yamamoto, S. Akira, W. C. Yeh, R. Lin, J. Hiscott, Activation of TBK1 and IKKepsilon kinases by vesicular stomatitis virus infection and the role of viral ribonucleoprotein in the development of interferon antiviral immunity. *J. Virol.* **78**, 10636–10649 (2004).
181. M. Mori, M. Yoneyama, T. Ito, K. Takahashi, F. Inagaki, T. Fujita, Identification of Ser-386 of interferon regulatory factor 3 as critical target for inducible phosphorylation that determines activation. *J. Biol. Chem.* **279**, 9698–9702 (2004).
182. D. C. Kang, R. V. Gopalkrishnan, Q. Wu, E. Jankowsky, A. M. Pyle, P. B. Fisher, mda-5: An interferon-inducible putative RNA helicase with double-stranded RNA-dependent ATPase activity and melanoma growth-suppressive properties. *Proc. Natl. Acad. Sci. U.S.A.* **99**, 637–642 (2002).
183. M. Yoneyama, M. Kikuchi, T. Natsukawa, N. Shinobu, T. Imazumi, M. Miyagishi, K. Taira, S. Akira, T. Fujita, The RNA helicase RIG-I has an essential function in double-stranded RNA-induced innate antiviral responses. *Nat. Immunol.* **5**, 730–737 (2004).
184. T. Kawai, K. Takahashi, S. Sato, C. Coban, H. Kumar, H. Kato, K. J. Ishii, O. Takeuchi, S. Akira, IPS-1, an adaptor triggering RIG-I- and Mda5-mediated type I interferon induction. *Nat. Immunol.* **6**, 981–988 (2005).
185. R. B. Seth, L. Sun, C. K. Ea, Z. J. Chen, Identification and characterization of MAVS, a mitochondrial antiviral signaling protein that activates

NF-kappaB and IRF 3. *Cell* **122**, 669–682 (2005).
186. L. G. Xu, Y. Y. Wang, K. J. Han, L. Y. Li, Z. Zhai, H. B. Shu, VISA is an adapter protein required for virus-triggered IFN-beta signaling. *Mol. Cell* **19**, 727–740 (2005).
187. E. Meylan, J. Curran, K. Hofmann, D. Moradpour, M. Binder, R. Bartenschlager, J. Tschopp, Cardif is an adaptor protein in the RIG-I antiviral pathway and is targeted by hepatitis C virus. *Nature* **437**, 1167–1172 (2005).
188. S. Kreuz, D. Siegmund, J. J. Rumpf, D. Samel, M. Leverkus, O. Janssen, G. Hacker, O. Dittrich-Breiholz, M. Kracht, P. Scheurich, H. Wajant, NFkappaB activation by Fas is mediated through FADD, caspase-8, and RIP and is inhibited by FLIP. *J. Cell Biol.* **166**, 369–380 (2004).
189. S. Balachandran, E. Thomas, G. N. Barber, A FADD-dependent innate immune mechanism in mammalian cells. *Nature* **432**, 401–405 (2004).
190. W. M. Chu, D. Ostertag, Z. W. Li, L. Chang, Y. Chen, Y. Hu, B. Williams, J. Perrault, M. Karin, JNK2 and IKKbeta are required for activating the innate response to viral infection. *Immunity* **11**, 721–731 (1999).
191. T. Kawai, S. Sato, K. J. Ishii, C. Coban, H. Hemmi, M. Yamamoto, K. Terai, M. Matsuda, J. Inoue, S. Uematsu, O. Takeuchi, S. Akira, Interferon-alpha induction through Toll-like receptors involves a direct interaction of IRF7 with MyD88 and TRAF6. *Nat. Immunol.* **5**, 1061–1068 (2004).
192. M. Yamamoto, S. Sato, K. Mori, K. Hoshino, O. Takeuchi, K. Takeda, S. Akira, Cutting edge: A novel Toll/IL-1 receptor domain-containing adapter that preferentially activates the IFN-beta promoter in the Toll-like receptor signaling. *J. Immunol.* **169**, 6668–6672 (2002).
193. K. J. Han, X. Su, L. G. Xu, L. H. Bin, J. Zhang, H. B. Shu, Mechanisms of the TRIF-induced interferon-stimulated response element and NF-kappaB activation and apoptosis pathways. *J. Biol. Chem.* **279**, 15652–15661 (2004).
194. G. Oganesyan, S. K. Saha, B. Guo, J. Q. He, A. Shahangian, B. Zarnegar, A. Perry, G. Cheng, Critical role of TRAF3 in the Toll-like receptor-dependent and -independent antiviral response. *Nature* **439**, 208–211 (2006).
195. S. Uematsu, S. Sato, M. Yamamoto, T. Hirotani, H. Kato, F. Takeshita, M. Matsuda, C. Coban, K. J. Ishii, T. Kawai, O. Takeuchi, S. Akira, Interleukin-1 receptor-associated kinase-1 plays an essential role for Toll-like receptor (TLR)7- and TLR9-mediated interferon-{alpha} induction. *J. Exp. Med.* **201**, 915–923 (2005).
196. Y. Huang, T. Li, D. C. Sane, L. Li, IRAK1 serves as a novel regulator essential for lipopolysaccharide-induced interleukin-10 gene expression. *J. Biol. Chem.* **279**, 51697–51703 (2004).
197. J. L. Swantek, M. F. Tsen, M. H. Cobb, J. A. Thomas, IL-1 receptor-associated kinase modulates host responsiveness to endotoxin. *J. Immunol.* **164**, 4301–4306 (2000).
198. K. Hoshino, T. Sugiyama, M. Matsumoto, T. Tanaka, M. Saito, H. Hemmi, O. Ohara, S. Akira, T. Kaisho, IkappaB kinase-alpha is critical for interferon-alpha production induced by Toll-like receptors 7 and 9. *Nature* **440**, 949–953 (2006).
199. M. Sasai, H. Oshiumi, M. Matsumoto, N. Inoue, F. Fujita, M. Nakanishi, T. Seya, Cutting Edge: NF-kappaB-activating kinase-associated protein 1 participates in TLR3/Toll-IL-1 homology domain-containing adapter molecule-1-mediated IFN regulatory factor 3 activation. *J. Immunol.* **174**, 27–30 (2005).

Citation: H. Häcker, M. Karin, Regulation and function of IKK and IKK-related kinases. *Sci. STKE* **2006**, re13 (2006).

Regulation and Function of IKK and IKK-Related Kinases  
Hans Häcker and Michael Karin (October 10, 2006)  
*Science Signaling* 2006 (357), re13. [doi: 10.1126/stke.3572006re13]

The following resources related to this article are available online at http://stke.sciencemag.org.  
This information is current as of November 24, 2016.

**Article Tools**  
Visit the online version of this article to access the personalization and article tools:  
http://stke.sciencemag.org/content/2006/357/re13

**Related Content**  
The editors suggest related resources on *Science*'s sites:  
http://stke.sciencemag.org/content/sigtrans/1/30/pe36.full  
http://stke.sciencemag.org/content/sigtrans/3/128/ec195.abstract  
http://stke.sciencemag.org/content/sigtrans/3/141/pe34.full  
http://stke.sciencemag.org/content/sigtrans/2/79/pe41.full  
http://stke.sciencemag.org/content/sigtrans/2007/401/tw310.abstract  
http://stke.sciencemag.org/content/sigtrans/2/61/ec91.abstract  
http://stke.sciencemag.org/content/sigtrans/2/76/ec207.abstract  
http://stke.sciencemag.org/content/sigtrans/2007/415/tw436.abstract  
http://stke.sciencemag.org/content/sigtrans/2007/368/tw12.abstract  
http://stke.sciencemag.org/content/sigtrans/2007/379/tw102.abstract  
http://stke.sciencemag.org/content/sigtrans/5/218/ec100.abstract  
http://stke.sciencemag.org/content/sigtrans/5/251/ec300.abstract  
http://stke.sciencemag.org/content/sigtrans/5/234/ec198.abstract  
http://stke.sciencemag.org/content/sigtrans/3/138/pe29.full  
http://stke.sciencemag.org/content/sigtrans/5/248/ec277.abstract  
http://stke.sciencemag.org/content/sigtrans/4/156/pe2.full  
http://stke.sciencemag.org/content/sigtrans/2/98/ec378.abstract  
http://stke.sciencemag.org/content/sigtrans/2/57/ec51.abstract  
http://stke.sciencemag.org/content/sigtrans/7/356/ec349.abstract  

**References**  
This article cites 198 articles, 97 of which you can access for free at:  
http://stke.sciencemag.org/content/2006/357/re13#BIBL  

**Permissions**  
Obtain information about reproducing this article:  
http://www.sciencemag.org/about/permissions.dtl  

Science Signaling (ISSN 1937-9145) is published weekly, except the last December, by the American Association for the Advancement of Science, 1200 New York Avenue, NW, Washington, DC 20005. Copyright 2016 by the American Association for the Advancement of Science; all rights reserved.
