
Available online at www.sciencedirect.com

ScienceDirect

Gene 386 (2007) 123–130

---

**Functional analysis of DesVIII homologues involved in glycosylation of macrolide antibiotics by interspecies complementation**

Jay Sung Joong Hong ${}^{a,1}$, Su Jin Park ${}^{b,1}$, Niranjan Parajuli ${}^{b}$, Sung Ryul Park ${}^{b}$, Hwa Soo Koh ${}^{b}$, Won Seok Jung ${}^{a}$, Cha Yong Choi ${}^{a}$, Yeo Joon Yoon ${}^{b,*}$

${}^{a}$ Interdisciplinary Program of Biochemical Engineering and Biotechnology, Seoul National University, San 56-1, Shilim-dong, Gwanak-gu, Seoul 151-742, Republic of Korea  
${}^{b}$ Division of Nano Sciences and Department of Chemistry, Ewha Womans University, 11-1, Daehyun-dong, Seodaemun-gu, Seoul 120-750, Republic of Korea  

Received 3 June 2006; received in revised form 7 August 2006; accepted 21 August 2006  
Available online 12 September 2006  
Received by A.J. van Wijnen  

---

### Abstract

The DesVIII is an auxiliary protein which enhances the transfer of TDP-D-desosamine catalyzed by DesVII glycosyltransferase in the biosynthesis of macrolide antibiotics, neomethymycin, methymycin and pikromycin, in *Streptomyces venezuelae* ATCC 15439. Homologues of the *desVIII* gene are present in a number of aminosugar-containing antibiotic biosynthetic gene clusters including *eryCII* from the erythromycin producer *Saccharopolyspora erythraea*, *oleP1* from the oleandomycin producer *Streptomyces antibioticus*, *dnrQ* from the doxorubicin producer *Streptomyces peucetius*, and *tylMIII* from the tylosin producer *Streptomyces fradiae*. In order to gain further insight into the function of these DesVIII homologues, interspecies complementation experiments were carried out by expressing each gene in a *desVIII* deletion mutant strain of *S. venezuelae*. Complementation by expressing EryCII, OleP1, and DnrQ in this mutant strain restored the production of glycosylated macrolides to an approximate level of 66%, 26% and 26%, respectively, compared to self-complementation by DesVIII. However, expression of TylMIII did not restore the antibiotic production. These results suggest that the DesVIII homologues (except for TylMIII) can functionally replace the native DesVIII for glycosylation to proceed *in vivo* and their functions are similar in acting as glycosyltransferase auxiliary proteins. The requirement of glycosyltransferase auxiliary protein seems to be more widespread in polyketide biosynthetic pathways than previously known.

© 2006 Elsevier B.V. All rights reserved.

#### Keywords: Polyketide; Glycosyltransferase; Auxiliary protein; Streptomyces venezuelae

---

### 1. Introduction

Deoxysugars are essential components of many biologically active secondary metabolites ([Weymouth-Wilson, 1997](https://doi.org/10.1016/S0006-2952(97)00002-0)), including the macrolide antibiotics neomethymycin (6), methymycin (7) and pikromycin (8) produced by *Streptomyces venezuelae* ([Xue et al., 1998](https://doi.org/10.1016/S0006-2952(97)00002-0)). The importance of the role played by deoxysugars has been well recognized. Cloning, characterization and functional studies

---

**Abbreviations:** ESI, electrospray ionization; kb, kilobase; LC, Liquid chromatography; MS, Mass spectrometry; PCR, polymerase chain reaction; TDP, thymidine diphosphate.

* Corresponding author. Tel.: +82 2 3277 4082; fax: +82 2 3277 3419.
E-mail address: joonyoon@ewha.ac.kr (Y.J. Yoon).

${}^{1}$ These authors contributed equally to this work.

0378-1119/$ - see front matter © 2006 Elsevier B.V. All rights reserved.

doi:[10.1016/j.gene.2006.08.021](https://doi.org/10.1016/j.gene.2006.08.021)

respectively. Compounds 4 and 5 are further modified by the PikC hydroxylase to 6, 7 and 8, respectively (Fig. 1). For glycosylation to proceed efficiently via DesVII, the presence of an auxiliary protein DesVIII was found to be a necessary requirement although its actual role in catalysis was unclear. We have also previously shown that DesVIII is required as a ‘helper’ protein for the *in vivo* glycosyltransfer of exogenous deoxysugars, namely TDP-D-quinovose and TDP-D-olivose via DesVII (Hong et al., 2004, 2005).

Since the discovery of DesVIII protein’s involvement in glycosylation of macrolide antibiotics, other pairs of glycosyltransferase and auxiliary protein were discovered. Recently, TylMII/TylMIII from the tylosin pathway of *Streptomyces fradiae* and MycB/MydC from the mycinamicin pathway of *Micromonospora griseorubida* were identified as glycosyltransferase/auxiliary protein pairs involved in glycosylation (Melançon et al., 2004). Likewise, AknS glycosyltransferase required activation by a partner protein, AknT, for transferring TDP-L-2-deoxyfucose to the aklavinone aglycon in the biosynthesis of anthracycline compound, aclacinomycin A (Lu et al., 2005). The activity of the glycosyltransferase EryCIII was reported to be greatly enhanced through activation by EryCII or AknT *in vitro* for attachment of TDP-D-desosamine in the biosynthesis of the macrolide, erythromycin D, however, once EryCIII was fully ‘activated’ it no longer required its protein partner (Yuan et al., 2005). Although the mechanistic role of auxiliary protein in activating the glycosyltransferase remains unclear, these studies have shown that the glycosyltransferase/auxiliary protein pair is critical for the efficient transfer of deoxysugar in the biosynthesis of macrolide and anthracycline compounds.

It is important to identify glycosyltransferase auxiliary proteins and understand their role in the glycosyltransfer process in order to enhance the efficiency of glycosylation in the biosynthesis of macrolide antibiotics. In this study, the pairing of glycosyltransferase with various noncognate auxiliary proteins was investigated to functionally identify auxiliary proteins and examine their effect on glycosylation of macrolide antibiotics *in vivo*. The DesVII glycosyltransferase was coupled with proteins homologous to DesVIII, including EryCII, TylMIII, OleP1, and DnrQ by expressing each gene in a *desVIII* deletion mutant strain of *S. venezuelae*.

## 2. Materials and methods

### 2.1. Bacterial strains, culture conditions and DNA manipulation

*S. venezuelae* ATCC 15439 was used for the construction of mutant strains of *S. venezuelae*. The wild-type and mutant strains were cultured in SCM liquid media (per liter: 15 g of soluble starch, 20 g of soytone, 0.1 g of CaCl₂·2H₂O, 1.5 g of yeast extract, 8.4 g of MOPS, pH adjusted to 7.2 before autoclaving) under appropriate antibiotic selection as described previously (Yoon et al., 2002). *Escherichia coli* DH10B was used throughout the study as a cloning host for DNA manipulation. The routine manipulation of DNA in *E. coli* was performed according to standard procedures (Sambrook and Russell, 2001) in Litmus28 (New England Biolabs) and pGEM-3Zf(+) (Promega). Southern blot hybridization followed the protocols recommended by the DIG labeling and detection kit (Roche).

### 2.2. Construction of *S. venezuelae desVIII* deletion mutant

The *desVIII* gene is located upstream of the *desVII–desVI–desR* genes within the desosamine biosynthetic cluster (GenBank accession number AF079762) and an in-frame deletion was introduced to avoid any polar effect on the downstream genes. A gene deletion plasmid, pYJ201, derived from pKC1139 (Bierman et al., 1992) that contains an apramycin resistant marker, was constructed for deletion of the *desVIII* gene. The *desVIII*-flanking DNA fragments used for the construction of pYJ201 were PCR amplified from the genomic DNA of *S. venezuelae*. The following

![Figure 1](https://i.imgur.com/yourimage.png)

Fig. 1. Glycosylation by DesVII/DesVIII in the biosynthesis of macrolide antibiotics: neomethymycin (6), methymycin (7) and pikromycin (8).

Table 1  
Genes encoding the auxiliary protein used for the construction of complementation plasmids  

| Gene    | Microorganism       | Antibiotic      | Sugar pathway     | References                                                                 |
|---------|---------------------|-----------------|-------------------|---------------------------------------------------------------------------|
| desVIII | S. venezuelae       | Pikromycin      | D-desosamine      | Xue et al. (1998), Borisova et al. (2004)                                   |
| eryCII  | S. erythraea        | Erythromycin    | D-desosamine      | Summers et al. (1997), Yuan et al. (2005)                                  |
| olePI   | S. antibioticus     | Oleandomycin    | D-desosamine      | Olano et al. (1998), Doumith et al. (1999)                                 |
| dnrQ    | S. peucetius        | Daunorubicin    | L-daunosamine     | Otten et al. (1995)                                                       |
| tylMIII | S. fradiae         | Tylosin         | D-mycaminose      | Gandecha et al. (1997), Melançon et al. (2004)                             |

nucleotides with engineered restriction sites (in bolds) and natural restriction sites (in italics) were used for cloning: a HindIII/BsiWI fragment containing the 3'-region of pikAV and 5'-region of desVIII, forward 5'-GACGCAAGCTTGGGCATCGAGGG GGCGGGCAAG-3' and reverse 5'-GGTGGCGTACGGGGTCG CCGTTCGCGGGCGTGAT-3'; and a BsiWI/EcoRI fragment containing the 3'-region of desVIII and 5'-region of desVII, forward 5'-CGGTCCCGTACGGCCCCGGTGCGTCCCT GGTCCG-3' and reverse 5'-GCTCAGAATTCCAGTCCGGC ACGACCGACGTGC-3'. The PCR-generated DNA fragments were simultaneously cloned into pKC1139 to generate plasmid pYJ201 for the desVIII deletion experiment. Plasmid pYJ201 was introduced into protoplasts of wild-type S. venezuelae (Keiser et al., 2000) for deletion by a replicative plasmid-mediated homologous recombination as described previously (Yoon et al., 2002). Double crossover mutants were selected by screening for the apramycin-sensitive phenotype and confirming the mutant's genotype by southern hybridization. In the resulting strain, a 912 bp fragment between the two BsiWI sites within desVIII (nucleotides 139 to 1051 downstream of the start codon of des-VIII) was deleted. This strain was designated YJ041.

2.3. Construction of expression plasmids and mutant strains

The expression plasmids used in this study were derivatives of pSE34 (Yoon et al., 2002), which is a shuttle vector with an E. coli ColE1 origin of replication and a high-copy pIJ101 origin (Lydiate et al., 1985). In each vector, the cloned genes were expressed under the control of the upstream ermE* promoter (Vara et al., 1989).

PCR was performed with Pfu polymerase (Fermentas) under the recommended conditions. The genes including desVIII, eryCII, tylMIII, olePI, and dnrQ were PCR amplified from S. venezuelae, Saccharopolyspora erythraea, S. fradiae, Streptomyces antibioticus, and Streptomyces peucetius genomic DNA, respectively. Genes used for the construction of complementation plasmids and the corresponding primers used for PCR amplification are described in Tables 1 and 2, respectively. PCR amplified desVIII gene was subcloned as a XbaI fragment into Litmus 28, sequenced to confirm the correct nucleotide sequence and ligated into the expression vector, pSE34, yielding plasmid pYJ194. PCR amplified eryCII, tylMIII, olePI and dnrQ genes were each subcloned as a BamHI/XbaI fragment into Litmus 28, confirmed for the correct sequence and then ligated into the expression vector, pSE34, yielding plasmid pYJ195, 196, 197 and 198, respectively. The strain YJ041 was transformed by each plasmid pYJ194, 195, 196, 197 or 198, separately.

The downstream genes of desVIII including desVII, desVI and desR were cloned into the integrative plasmid pSET152 (Bierman et al., 1992) derivative containing an ermE* promoter. The desVII and desVI genes were each PCR amplified, digested by appropriate restriction enzymes and subcloned in series into Litmus 28. The desR gene was PCR amplified and subcloned into pGEM-3Zf(+) as a BamHI/SphI fragment which was then digested by BamHI/HindIII for cloning into the Litmus 28 containing the desVII and desVI genes. An XbaI/BglII-digested fragment including these three genes was digested from Litmus 28 and ligated into XbaI/BamHI-digested pSET152 derivative containing an ermE* promoter and designated

Table 2  
PCR primers used in amplification of genes  

| Gene    | Primer sequence (5'-3') (restriction site underlined)                                                                 | Portion of gene | Restriction site |
|---------|----------------------------------------------------------------------------------------------------------------------|-----------------|------------------|
| desVIII | CGTTCTAGACCCGACCAGCCTATCGAAGGAGC                                                                                   | 5'              | XbaI             |
|         | CGTTCTAGAGGTCCGAGCCGTCCGGTCCCG                                                                                     | 3'              | XbaI             |
| eryCII  | GCACAGGATCCCCCTGGGCTGGGCGGGGAATTC                                                                                  | 5'              | BamHI            |
|         | CGCGCTCTAGAGAATCCTCAGAGCTCGACGGGG                                                                                   | 3'              | XbaI             |
| tylMIII | ACCGTGGATCCCCGGGGGGAGAGAGGAGAGGCC                                                                                  | 5'              | BamHI            |
|         | CGGCGTCTAGACAGTGCCTTCTCACTCGGGGA                                                                                   | 3'              | XbaI             |
| olePI   | CGCGGGGATCCCAGCTCGAACAGAGCGAACGGC                                                                                  | 5'              | BamHI            |
|         | AGCGGCTCTAGATTCGACGGCATGCTCATGCGGGC                                                                                 | 3'              | XbaI             |
| dnrQ    | CCCGGGGATCCACAGTGAGGAGCACGACGAT                                                                                    | 5'              | BamHI            |
|         | GCACCTCTAGAGGGTCACCCCTCCCTGGGTCACT                                                                                 | 3'              | XbaI             |
| desVII  | CGGCTCAGATCTGCCCCCATCCCGCCCCCTCCACC                                                                                | 5'              | BglII            |
|         | ATACAGGAATTCGGGCCTGGGGCGAGGGGTGCGG                                                                                  | 3'              | EcoRI            |
| desVI   | GGCCGAATTCCTGTATCTGCGCCGGGGGACGCC                                                                                  | 5'              | EcoRI            |
|         | TGCACCCGGGATCCTCCCGCCCCGGGGGGGTCTTG                                                                                | 3'              | BamHI            |
| desR    | GGAGGATCCGGGTGCACCAAGCAAAGAGAGAGA                                                                                  | 5'              | BamHI            |
|         | GGCGCATGCTCACCAAGACGTTGACC GTGGCGCT                                                                                | 3'              | SphI             |

Table 3  
Strains and plasmids used in this study  

| Strain or plasmid | Description | Source or reference |
|-------------------|-------------|---------------------|
| S. venezuelae      |             |                     |
| Wild-type (ATCC 15439) | Wild-type producer for (neo)methymycin and pikromycin | Xue et al. (1998) |
| YJ041             | In-frame gene deletion of sequence encoding DesVIII (see Fig. 2) | This work |
| YJ051             | YJ041 with chromosomal integration of expression plasmid pYJ238 | This work |
| Plasmids          |             |                     |
| pKC1139           | E. coli–Streptomyces shuttle plasmid vector | Bierman et al. (1992) |
| pYJ201            | pKC1139 derivative containing two 1kb homologous flanking regions of desVIII gene | This work |
| pSE34             | E. coli–Streptomyces shuttle plasmid vector | Yoon et al. (2002) |
| pYJ194            | pSE34 derivative for expression of desVIII gene | This work |
| pYJ195            | pSE34 derivative for expression of eryCII gene | This work |
| pYJ196            | pSE34 derivative for expression of tylMIII gene | This work |
| pYJ197            | pSE34 derivative for expression of oleP1 gene | This work |
| pYJ198            | pSE34 derivative for expression of dnrQ gene | This work |
| pSET152           | E. coli–Streptomyces shuttle plasmid vector | Bierman et al. (1992) |
| pYJ238            | pSET152 derivative for expression of desVII, desVI and desR genes | This work |

pYJ238. The plasmid pYJ238 was integrated into the genomic DNA of strain YJ041 to produce the mutant strain YJ051. This strain YJ051 was transformed by each plasmid pYJ194, 195, 196, 197 or 198, separately. The strains used in this study are summarized in Table 3.

### 2.4. Production and analysis of macrolides

Wild-type S. venezuelae and mutant strains were cultured in SCM liquid medium at 30 °C for 36 h under appropriate antibiotic selection. Metabolites were extracted as described previously (Xue et al., 1998). The products were analyzed by liquid chromatography coupled with electrospray ionization-mass spectrometry (LC–ESI-MS). LC was performed with a Waters (Milford, MA) Model 2690 separations module, using a 25 cm × 4.6 mm Watchers 120 ODS-BP (5 μm) column (Daiso Watchers, Osaka, Japan) with 20–80% acetonitrile in water gradient containing 10 mM of ammonium acetate at a flow rate of 1 ml/min. The effluent was split 1:5 before ESI interface, with approximately 200 μl/min routed to the Micromass (Beverly, MA) Quattro LC triple tandem quadrupole mass spectrometer. Mass spectrometric data was acquired in the

![Figure 2](#fig2)

Fig. 2. Construction of desVIII deletion mutant of S. venezuelae. (A) Construction of desVIII-deleted YJ041 mutant via homologous recombination of deletion plasmid pYJ201 and the chromosome of wild-type S. venezuelae. The endonuclease restriction sites are abbreviated as follows: B, BsiWI; S, SacI; E, EcoRI. (B) Southern blot analysis of SacI-digested genomic DNA of S. venezuelae wild-type and YJ041 mutant. Lane 1 (wild-type), 2 (YJ041) were probed by a 1.1 kb BsiWI–EcoRI fragment containing 3′-region of desVIII and 5′-region of desVII.

Fig. 3. Phylogenetic analysis of DesVIII homologues. DesVIII (AAC68676) from *S. venezuelae*, NbmC (BAC57038) from *S. narbonensis*, MydC (AB089954.1) from *Micromonospora griseorubida*, DnrQ (AAD15266) from *S. peucetius*, AknT (AAF73456) from *S. galilaeus*, CosT (ABC00727) from *S. olindensis*, TylMIII (CAA57471) from *S. fradiae*, GerY (ABB52546) from *S. sp.* KCTC 0041BP, ChmCII (AAS79464) from *S. bikiniensis*, RdmG (AAL24449) from *S. purpurascens*, EryCII (CAA74711) from *S. erythraea*, MegCII (AAG13920) from *M. megalomicea* subsp. *nigra*, MegDVI (AAG13907) from *M. megalomicea* subsp. *nigra*, ORF32 (NP_851454) from *S. rochei*, and OleP1 (CAA05640) from *S. antibioticus*.

positive-ion mode while scanning for the specific parent mass to be fragmented. The relative amount of each compound produced was compared by the peak intensity from the LC–MS spectrum. The production level of the total glycosylated compounds (4, 5, 6, 7, and 8) were averaged from five separate cultivations and extractions.

### 3. Results and discussion

#### 3.1. Construction of desVIII gene deletion mutant of S. venezuelae and self-complementation

In order to create a deletion strain for investigating the function of DesVIII homologues, the *desVIII* gene was deleted in-frame from the chromosome of *S. venezuelae* using the plasmid pYJ201 by homologous recombination (Fig. 2A). Southern hybridization analysis of the genomic DNA from several recombinant clones was performed to identify the mutant strain with the desired double crossover recombination. The DIG-labeled 1.1 kb *BsiWI-EcoRI* DNA fragment containing the 3′-region of *desVIII* and the 5′-region of *desVII* was used as a probe. When *SacI*-digested genomic DNA was analyzed, a 1.9 kb DNA fragment was detected (Fig. 2B), as expected, confirming the deletion of a 0.9 kb *BsiWI* internal fragment from the *desVIII* gene. The resulting mutant strain, designated YJ041, was cultured and the culture extract was analyzed by LC–MS. The production of aglycons, 2 and 3, were detected corresponding to *m/z* 297 and 353 for H⁺ ionized forms, respectively (data not shown). Less than 1% of the total glycosylated compounds (4, 5, 6, 7, and 8) were produced from this strain compared to the wild-type strain.

Self-complementation of YJ041 mutant by transformation with pYJ194 expressing the DesVIII protein (YJ041/pYJ194) restored the production of glycosylated compounds, however the total amount of glycosylated compounds was significantly reduced to approximately 5% when compared to that of the wild-type strain (data not shown). Although an in-frame deletion was introduced in the deletion of the *desVIII* gene from the chromosome, a polar effect seems to have influenced the expression of the downstream genes. Therefore the downstream

genes of *desVIII* including *desVII*, *desVI*, and *desR* were cloned into a separate plasmid pYJ238 under the control of an *ermE* promoter for integration into the chromosomal DNA of YJ041 yielding the strain YJ051. This strain was cultured and less than 1% of the glycosylated compounds were detected from the culture broth (Fig. 4A). Expression of DesVIII in the YJ051/pYJ194 strain restored the total production of glycosylated compounds up to approximately 60% (Fig. 4B) when compared to the wild-type strain and this strain was used as a control strain for comparison of relative yield of glycosylated compounds in complementation experiments with other *desVIII* homologous genes.

#### 3.2. Sequence comparison of DesVIII and homologues

A search for proteins with amino acid sequence homology to DesVIII using BLAST displayed similarity to a number of P450 enzymes, but in our previous report we have ruled out the function of DesVIII as a P450 enzyme since DesVIII lacks the rigorously conserved cysteine residue normally found at the haem iron binding site (Hong et al., 2005). Other matches include putative deoxyhexose isomerases and the *desVIII* gene product itself has been labeled as a putative tautomerase in the NCBI database. It should be pointed out that the putative functions of DesVIII and homologues may require revision since the discovery of their functions *in vitro* as glycosyltransferase activating proteins. Fifteen matching proteins, encoded from macrolide and anthracycline antibiotic biosynthetic gene clusters, were found in the databank and a phylogenetic analysis using the clustalW software (Thompson et al., 1994) of the related proteins is shown in Fig. 3. It is likely that these proteins may possibly act as glycosyltransferase activating proteins and further investigation is required to provide experimental evidence elucidating their functions. Comparison of the amino acid sequence of DesVIII to the sequence of EryCII, TylMIII, OleP1 and DnrQ proteins produced sequence identities of 34%, 36%, 32% and 38%, respectively. A certain degree of sequence similarity between these homologues suggests that these enzymes may play an identical role in the biosynthetic pathway of the respective antibiotic compounds. Interestingly, all of these genes are located

Table 4  
Relative yield of glycosylated compounds  

| Strain name (plasmid expressed gene) | Relative yield of total glycosylated compounds<sup>a</sup> (neomethymycin, methymycin, pikromycin, YC-17, and narbomycin) |
|-------------------------------------|-----------------------------------------------------------------------------------------------|
| YJ051                              | <1%                                                                                           |
| YJ051/pYJ194 (desVIII)             | 100%                                                                                          |
| YJ051/pYJ195 (eryCII)              | 66%                                                                                           |
| YJ051/pYJ197 (oleP1)               | 26%                                                                                           |
| YJ051/pYJ198 (dnrQ)                | 26%                                                                                           |
| YJ051/pYJ196 (tylMIII)             | <1%                                                                                           |

<sup>a</sup> The total yield of glycosylated compounds from the extract of YJ051/ pYJ194 was taken as 100%.

and DnrQ in the mutant strain YJ051. The results of the complementation analysis are summarized in Table 4.

Strain YJ051/pYJ195 expressing EryCII produced a total amount of glycosylated compounds (4, 5, 6, 7, and 8) at a level of approximately 66% relative to the control strain YJ051/ pYJ194 which was the highest yield among the DesVIII homologues tested (Fig. 4C). From the result of this complementation test we concluded that EryCII can functionally replace DesVIII in vivo implying that both proteins serve the same function in acting as glycosyltransferase activating proteins. However, the ability of EryCII in activating the DesVII glycosyltransferase seems to be less efficient than the native DesVIII protein. In a previous study both EryCII and AknT were able to activate the EryCIII glycosyltransferase in vitro but a larger amount of AknT than EryCII was required to achieve the same level of activation (Yuan et al., 2005). It is likely that a greater amount of the EryCII than DesVIII protein may also be required to achieve the same level of activation of the DesVII glycosyltransferase in vivo.

Expression of OleP1 in the strain YJ051/pYJ197 restored the level of glycosylated compounds to approximately 26% compared to the control strain which suggests that OleP1 is also a glycosyltransferase auxiliary protein (Fig. 4D). A previous study

Fig. 4. LC–MS analysis of metabolites produced from S. venezuelae mutant strains. LC–MS selected for *m/z* = 454, 510, 470, 470 and 526, corresponding to the protonated forms of YC-17 (4), narbomycin (5), neomethymycin (6), methymycin (7), and pikromycin (8), respectively. (A) YJ051/pSE34, (B) YJ051/pYJ194, (C) YJ051/pYJ195, (D) YJ051/pYJ197, (E) YJ051/pYJ198, and (F) YJ051/pYJ196.

reported that *oleP1* gene introduced into an *eryCII* deletion mutant of *S. erythraea* restored the production of erythromycin A and concluded that the function of OleP1 is identical to EryCII (Doumith et al., 1999). The functions of EryCII, OleP1 and DesVIII are all identical, as shown in this study, which supports the results of a previous report. Although the native counterpart glycosyltransferase for the auxiliary proteins DesVIII, EryCII and OleP1 are all desosamine-transferring enzymes similar to DesVII, the production levels of glycosylated compounds produced from complementation by DesVIII, EryCII and OleP1 differed from each other which suggests that the auxiliary protein may not act on the deoxysugar itself but rather binds to the glycosyltransferase to induce a conformational change from an inactive to an active form as proposed in previous studies (Lu et al., 2005; Borisova et al., 2006).

Complementation by DnrQ in the strain YJ051/pYJ198 restored the level of glycosylated compounds to approximately 26% compared to the control strain (Fig. 4E). The successful complementation of DnrQ to functionally replace DesVIII is intriguing since DnrQ is naturally involved in the biosynthesis of an anthracyclin antibiotic (doxorubicin) whereas the other DesVIII homologues are involved in macrolide antibiotic biosynthetic pathways. A similar result was reported for AknT which was shown to be required for AknS glycosyltransferase activity in another anthracycline (aclaricinomycin A) biosynthetic pathway (Lu et al., 2005) and also function as a glycosyltransferase-activator protein for EryCIII glycosyltransferase (Yuan et al., 2005). Although an earlier work reported that DnrQ was required for the efficient bioconversion of doxorubicin aglycone (*ε*-rhodomycinone) to the glycosylated product (rhodomycin B) in the doxorubicin biosynthetic pathway from *S. peucetius* (Olano et al., 1999), the exact function of DnrQ was unknown. This is the first report proposing the function of DnrQ to be a glycosyltransferase activator. The requirement of an auxiliary protein in glycosylation may not be restricted to macrolide antibiotics and further studies are necessary to investigate their involvement in other polyketide biosynthetic pathways.

Among the auxiliary proteins investigated in this study, only TylMIII did not complement DesVIII and the production of glycosylated compounds was not restored (Fig. 4F). The role of TylMIII has been shown in a previous study to be required for the glycosylation of tylactone via the glycosyltransferase TylMII (Melançon et al., 2004). In a separate study, we carried out feeding experiments in which tylactone was fed to a mutant strain of *S. venezuelae*, expressing the same *tylMIII* gene used in this study, along with *tylMII* and deoxysugar biosynthetic genes, and verified the production of a novel glycosylated derivative of tylactone (unpublished results). Therefore we ruled out the possibility that failure to restore the glycosylated compounds in the YJ051/pYJ196 strain was due to no expression or other truncated TylMIII being expressed. TylMIII may be more specific towards the partner TylMII glycosyltransferase and unable to activate the noncognate DesVII glycosyltransferase. The complementation of DesVIII by the homologues EryCII, OleP1 and DnrQ but not by TylMIII, despite a high sequence identity of TylMIII to DesVII, demonstrates that the pairing of the glycosyltransferase with a

specific auxiliary protein is necessary for glycosylation to proceed efficiently and the proper pairing cannot be predicted by sequence homology.

### 4. Conclusions

The present study demonstrates that the homologues of DesVIII from other antibiotic biosynthetic gene clusters were able to functionally replace the role of DesVIII, providing evidence that these homologues may serve analogous functions as glycosyltransferase activators in their respective pathways. The requirement of auxiliary protein in glycosylation may be more widespread in secondary metabolite biosynthetic pathways than previously known. However, the production levels of glycosylated compounds produced from the help of exogenous auxiliary proteins were lower than that of the self-complemented strain which shows that changing the native partner of glycosyltransferase with a heterologous activator protein affects the efficiency of glycosylation. Further investigation is required to identify other existing glycosyltransferase auxiliary proteins and elucidate the exact mechanism by which these DesVIII homologues activate the glycosyltransferase, which should provide valuable information for the generation of novel glycosylated macrolide antibiotics through combinatorial biosynthesis.

### Acknowledgement

This work was supported by the Korean Research Foundation Grant (KRF-2004-041-D00251). The authors thank the Ministry of Education for the Brain Korea 21 fellowship.

### References

Bierman, M., Logan, R., O’Brien, K., Seno, E.T., Rao, R.N., Schoner, B.E., 1992. Plasmid cloning vectors for the conjugal transfer of DNA from *Escherichia coli* to *Streptomyces* spp. Gene 116, 43–49.

Borisova, S.A., Zhao, L., Melançon III, C.E., Kao, C.L., Liu, H.W., 2004. Characterization of the glycosyltransferase activity of desVII: analysis of and implications for the biosynthesis of macrolide antibiotics. J. Am. Chem. Soc. 126, 6534–6535.

Borisova, S.A., et al., 2006. Substrate specificity of the macrolide-glycosylating enzyme pair DesVII/DesVIII: opportunities, limitations, and mechanistic hypotheses. Angew. Chem., Int. Ed. Engl. 45, 2748–2753.

Doumith, M., Legrand, R., Lang, C., Salas, J.A., Raynal, M.C., 1999. Interspecies complementation in *Saccharopolyspora erythraea*: elucidation of the function of *oleP1, oleG1* and *oleG2* from the oleandomycin biosynthetic gene cluster of *Streptomyces antibioticus* and generation of new erythromycin derivatives. Mol. Microbiol. 34, 1039–1048.

Gandecha, A.R., Large, S.L., Cundliffe, E., 1997. Analysis of four tylosin biosynthetic genes from the *tylLM* region of the *Streptomyces fradiae* genome. Gene 184, 197–203.

Hong, J.S.J., Park, S.H., Choi, C.Y., Sohng, J.K., Yoon, Y.J., 2004. New olivosyl derivatives of methymycin/pikromycin from an engineered strain of *Streptomyces venezuelae*. FEMS Microbiol. Lett. 238, 391–399.

Hong, J.S.J., et al., 2005. The role of a second protein (DesVIII) in glycosylation for the biosynthesis of hybrid macrolide antibiotics in *Streptomyces venezuelae*. J. Microbiol. Biotechnol. 15 (3), 640–645.

Keiser, T., Bibb, M.J., Buttner, M.J., Chater, K.F., Hopwood, D.A., 2000. Practical *Streptomyces* Genetics. John Innes Foundation, Norwich, United Kingdom.

Lu, W., et al., 2005. AknT is an activating protein for the glycosyltransferase AknS in L-aminodeoxysugar transfer to the aglycone of aclacinomycin A. Chem. Biol. 12, 527–534.

Lydiate, D.J., Malpartida, F., Hopwood, D.A., 1985. The Streptomyces plasmid SCP2*: its functional analysis and development into useful cloning vectors. Gene 35, 223–235.

Melançon III, C.E., Takahashi, H., Liu, H.W., 2004. Characterization of *tylM3/ tylM2* and *mydC/mycB* pairs required for efficient glycosyltransfer in macrolide antibiotic biosynthesis. J. Am. Chem. Soc. 126, 16726–16727.

Olano, C., Rodriguez, A.M., Michel, J.M., Mendez, C., Raynal, M.C., Salas, J.A., 1998. Analysis of a *Streptomyces antibioticus* chromosomal region involved in oleandomycin biosynthesis, which encodes two glycosyltransferases responsible for glycosylation of the macrolactone ring. Mol. Gen. Genet. 259, 299–308.

Olano, C., Lomovskaya, N., Fonstein, L., Roll, J.T., Hutchinson, C.R., 1999. A two-plasmid system for the glycosylation of polyketide antibiotics: bioconversion of epsilon-rhodomycinone to rhodomycin D. Chem. Biol. 6, 845–855.

Otten, S.L., Liu, X., Ferguson, J., Hutchinson, C.R., 1995. Cloning and characterization of the *Streptomyces peucetius dnrQS* genes encoding a daunosamine biosynthesis enzyme and a glycosyl transferase involved in daunorubicin biosynthesis. J. Bacteriol. 177, 6688–6692.

Sambrook, J., Russell, D.W., 2001. Molecular Cloning: a Laboratory Manual, 3rd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

Summers, R.G., Donadio, S., Staver, M.J., Wendt-Pienkowski, E., Hutchinson, C.R., Katz, L., 1997. Sequencing and mutagenesis of genes from the erythromycin biosynthetic gene cluster of *Saccharopolyspora erythrea* that are involved in L-mycarose and D-desosamine production. Microbiology 143, 3251–3262.

Thompson, J.D., Higgins, D.G., Gibson, T.J., 1994. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res. 22, 4673–4680.

Vara, J., Lewandowska-Skarbek, M., Wang, Y.G., Donadio, S., Hutchinson, C.R., 1989. Cloning of genes governing the deoxysugar portion of the erythromycin biosynthesis pathway in *Saccharopolyspora erythraea* (*Streptomyces erythreus*). J. Bacteriol. 171, 5872–5881.

Weymouth-Wilson, A.C., 1997. The role of carbohydrates in biologically active natural products. Nat. Prod. Rep. 14, 99–110.

Xue, Y., Zhao, L., Liu, H.W., Sherman, H.W., 1998. A gene cluster for macrolide antibiotic biosynthesis in *Streptomyces venezuelae*: architecture of metabolic diversity. Proc. Natl. Acad. Sci. U. S. A. 95, 12111–12116.

Yoon, Y.J., Beck, B.J., Kim, B.S., Kang, H.Y., Reynolds, K.A., Sherman, D.H., 2002. Generation of multiple bioactive macrolides by hybrid modular polyketide synthases in *Streptomyces venezuelae*. Chem. Biol. 9, 203–214.

Yuan, Y., Chung, H.S., Leimkuhler, C., Walsh, C.T., Kahne, D., Walker, S., 2005. In vitro reconstitution of EryCIII activity for the preparation of unnatural macrolides. J. Am. Chem. Soc. 127, 14128–14129.
