
Vol 444 | 16 November 2006 | doi:10.1038/nature05286

**REVIEWS**

# The plant immune system

Jonathan D. G. Jones¹ & Jeffery L. Dangl²

Many plant-associated microbes are pathogens that impair plant growth and reproduction. Plants respond to infection using a two-branched innate immune system. The first branch recognizes and responds to molecules common to many classes of microbes, including non-pathogens. The second responds to pathogen virulence factors, either directly or through their effects on host targets. These plant immune systems, and the pathogen molecules to which they respond, provide extraordinary insights into molecular recognition, cell biology and evolution across biological kingdoms. A detailed understanding of plant immune function will underpin crop improvement for food, fibre and biofuels production.

## Introduction

Plant pathogens use diverse life strategies. Pathogenic bacteria proliferate in intercellular spaces (the apoplast) after entering through gas or water pores (stomata and hydathodes, respectively), or gain access via wounds. Nematodes and aphids feed by inserting a stylet directly into a plant cell. Fungi can directly enter plant epidermal cells, or extend hyphae on top of, between, or through plant cells. Pathogenic and symbiotic fungi and oomycetes can invaginate feeding structures (haustoria), into the host cell plasma membrane. Haustorial plasma membranes, the extracellular matrix, and host plasma membranes form an intimate interface at which the outcome of the interaction is determined. These diverse pathogen classes all deliver effector molecules (virulence factors) into the plant cell to enhance microbial fitness.

Plants, unlike mammals, lack mobile defender cells and a somatic adaptive immune system. Instead, they rely on the innate immunity of each cell and on systemic signals emanating from infection sites¹⁻³. We previously reviewed disease resistance (R) protein diversity, polymorphism at R loci in wild plants and lack thereof in crops, and the suite of cellular responses that follow R protein activation¹. We hypothesized that many plant R proteins might be activated indirectly by pathogen-encoded effectors, and not by direct recognition. This ‘guard hypothesis’ implies that R proteins indirectly recognize pathogen effectors by monitoring the integrity of host cellular targets of effector action¹⁴. The concept that R proteins recognize ‘pathogen-induced modified self’ is similar to the recognition of ‘modified self’ in ‘danger signal’ models of the mammalian immune system⁵.

It is now clear that there are, in essence, two branches of the plant immune system. One uses transmembrane pattern recognition receptors (PRRs) that respond to slowly evolving microbial- or pathogen-associated molecular patterns (MAMPS or PAMPs), such as flagellin⁶. The second acts largely inside the cell, using the polymorphic NB-LRR protein products encoded by most R genes¹. They are named after their characteristic nucleotide binding (NB) and leucine rich repeat (LRR) domains. NB-LRR proteins are broadly related to animal CATERPILLER/NOD/NLR proteins⁷ and STAND ATPases⁸. Pathogen effectors from diverse kingdoms are recognized by NB-LRR proteins, and activate similar defence responses. NB-LRR-mediated disease resistance is effective against pathogens that can grow only on living host tissue (obligate biotrophs), or hemibiotrophic pathogens, but not against pathogens that kill host tissue during colonization (necrotrophs)⁹.

Our current view of the plant immune system can be represented as a four phased ‘zigzag’ model (Fig. 1), in which we introduce several important abbreviations. In phase 1, PAMPs (or MAMPs) are recognized by PRRs, resulting in PAMP-triggered immunity (PTI) that can halt further colonization. In phase 2, successful pathogens deploy effectors that contribute to pathogen virulence. Effectors can interfere with PTI. This results in effector-triggered susceptibility (ETS). In phase 3, a given effector is ‘specifically recognized’ by one of the NB-LRR proteins, resulting in effector-triggered immunity (ETI). Recognition is either indirect, or through direct NB-LRR recognition of an effector. ETI is an accelerated and amplified PTI response, resulting in disease resistance and, usually, a hypersensitive cell death response (HR) at the infection site. In phase 4, natural selection drives pathogens to avoid ETI either by shedding or diversifying the recognized effector gene, or by acquiring additional effectors that suppress ETI. Natural selection results in new R specificities so that

![Figure 1](https://example.com/figure1.png)

*Figure 1 | A zigzag model illustrates the quantitative output of the plant immune system. In this scheme, the ultimate amplitude of disease resistance or susceptibility is proportional to [PTI – ETS + ETI]. In phase 1, plants detect microbial/pathogen-associated molecular patterns (MAMPs/PAMPs, red diamonds) via PRRs to trigger PAMP-triggered immunity (PTI). In phase 2, successful pathogens deliver effectors that interfere with PTI, or otherwise enable pathogen nutrition and dispersal, resulting in effector-triggered susceptibility (ETS). In phase 3, one effector (indicated in red) is recognized by an NB-LRR protein, activating effector-triggered immunity (ETI), an amplified version of PTI that often passes a threshold for induction of hypersensitive cell death (HR). In phase 4, pathogen isolates are selected that have lost the red effector, and perhaps gained new effectors through horizontal gene flow (in blue)—these can help pathogens to suppress ETI. Selection favours new plant NB-LRR alleles that can recognize one of the newly acquired effectors, resulting again in ETI.*

---

¹The Sainsbury Laboratory, John Innes Centre, Norwich Research Park, Colney, Norwich NR4 7UH, UK.  
²Department of Biology, Department of Microbiology and Immunology, Curriculum in Genetics, and Carolina Center for Genome Sciences, CB-3280, University of North Carolina, Chapel Hill, North Carolina 27599, USA.

**REVIEWS**

ETI can be triggered again. Below, we review each phase in turn, we update the experimental validation of the ‘guard hypothesis’, and we consider future challenges in understanding and manipulating the plant immune system. We will not discuss the small RNA-based plant immune system active against viruses¹⁰ or the active response of plants to herbivores¹¹.

### Microbial patterns and plant pattern recognition

We define basal disease resistance as that activated by virulent pathogens on susceptible hosts. Thus, basal disease resistance is, at first glance, PTI minus the effects of ETS; however, there is also likely to be weak ETI triggered by weak recognition of effectors, as detailed below. Hence, the most accurate definition of basal defence would be ‘PTI plus weak ETI, minus ETS’. The archetypal elicitor of PTI is bacterial flagellin, which triggers defence responses in various plants¹². Flagellum-based motility is important for bacterial pathogenicity in plants⁶. A synthetic 22-amino-acid peptide (flg22) from a conserved flagellin domain is sufficient to induce many cellular responses¹³ including the rapid (<1 h) transcriptional induction of at least 1,100 *Arabidopsis thaliana* (hereafter *Arabidopsis*) genes¹⁴. A genetic screen using flg22 defined the *Arabidopsis* LRR-receptor kinase FLS2, which binds flg22 (ref. 15). FLS2 and mammalian TLR5 recognize different flagellin domains⁶. FLS2 is internalized following stimulation by a receptor-mediated endocytic process that presumably has regulatory functions¹⁶. *fls2* mutants exhibit enhanced sensitivity to spray application of pathogenic *Pseudomonas syringae* pv. *tomato* DC3000 (*Pto* DC3000), but not to syringe infiltration into the leaf apoplast¹⁴,¹⁷, suggesting that FLS2 acts early against pathogen invasion.

Bacterial cold shock proteins and elongation factor Tu (EF-Tu) activate similar defence responses to flg22 (refs 18–20). Ef-Tu is recognized by an *Arabidopsis* LRR-kinase called EFR (ref. 20). *efr* mutants support higher levels of transient transformation with *Agrobacterium*, suggesting that PTI might normally limit *Agrobacterium* pathogenicity. Treatment with a conserved EF-Tu peptide induces expression of a gene set nearly identical to that induced by flg22 (ref.20). Conversely, *EFR* transcription is induced by flg22. Hence, the responses to MAMPs/PAMPs converge on a limited number of signalling pathways and lead to a common set of outputs that comprise PTI. Remarkably, mutations in genes required for NB-LRR function have no effect on early responses to flg22 (ref. 14). Thus, NB-LRR-dependent signalling and MAMP/PAMP-mediated signalling require partially distinct components.

Molecules that induce PTI are not easily discarded by the microbes that express them. Yet flagellin from various *Xanthomonas campestris* pv. *campestris* strains is variably effective in triggering FLS2-mediated PTI in *Arabidopsis*, and flagellin from *Agrobacterium tumefaciens* or *Sinorhizobium meliloti* is less active than that from *P. syringae*¹³. Ef-Tu from *Pto* DC3000 is much less active in eliciting PTI in *Arabidopsis* than is Ef-Tu from *Agrobacterium*¹⁹. Limited variation also exists in PAMP responsiveness within a plant species. *Arabidopsis* accession Ws-0 carries a point mutation in *FLS2*, rendering it non-responsive to flg22 (ref.12). In fact, individual plant species recognize only a subset of potential PAMPs (ref. 6). Neither PAMPs nor PRRs are invariant, and each can be subject to natural selection.

Additional MAMPs/PAMPs and corresponding PRRs must exist, since *Agrobacterium* extracts elicit PTI on an *fls2 efr-1* double mutant (ref. 20). Other LRR kinases may encode additional PRRs whose transcription is stimulated by engagement of related PRRs. There are over 200 LRR-kinases in the *Arabidopsis* Col-0 genome²¹; 28 of these are induced within 30 min of flg22 treatment¹⁴. FLS2 and EFR are members of an atypical kinase family that might have a plant immune system specific function²². There are also 56 *Arabidopsis* receptor-like proteins (RLPs) that encode type I transmembrane proteins with LRR ectodomains, but no intracellular kinase domains²³. MAMP/PAMP elicitation might ‘prime’ further defence responses by elevating responsiveness to other microbial patterns¹⁴.


---

**Successful pathogens suppress PTI**

What does a would-be pathogen, using its collection of effectors, need to achieve? Some effectors may serve structural roles, for example, in the extrahaustorial matrix that forms during fungal and oomycete infection²⁴. Others may promote nutrient leakage or pathogen dispersal²⁵. Many are likely to contribute to suppression of one or more components of PTI or ETI. The extent to which ETI and PTI involve distinct mechanisms is still an open question, and some effectors may target ETI rather than PTI, or vice versa (Fig. 1).

Plant pathogenic bacteria deliver 15–30 effectors per strain into host cells using type III secretion systems (TTSS). Bacterial effectors contribute to pathogen virulence, often by mimicking or inhibiting eukaryotic cellular functions²⁶–²⁸. A pathogenic *P. syringae* strain mutated in the TTSS, and unable to deliver any type III effectors, triggers a faster and stronger transcriptional re-programming in bean than does the isogenic wild-type strain²⁹. This strain, representing the sum of all bacterial MAMPs/PAMPs, induces transcription of essentially the same genes as flg22 (refs 30–32). Hence, the type III effectors from any successful bacterial pathogen dampen PTI sufficiently to allow successful colonization³³ (Fig. 1).

Excellent reviews discuss cellular processes targeted by bacterial type III effectors²⁶–²⁸,³⁴; we highlight only new examples. The *P. syringae* HopM effector targets at least one ARF-GEF protein likely to be involved in host cell vesicle transport³⁵. HopM functions redundantly with the unrelated effector AvrE in *P. syringae* virulence³⁶ suggesting that manipulation of host vesicle transport is important for successful bacterial colonization. AvrPto and AvrPtoB are unrelated type III effectors that may contribute to virulence by inhibiting early steps in PTI, upstream of MAPKKK (ref. 37). Like other type III effectors, AvrPtoB is a bipartite protein. The amino terminus contributes to virulence; the carboxy terminus may have a function in blocking host cell death³⁸,³⁹. A domain from the AvrPtoB C terminus folds into an active E3 ligase, suggesting that its function involves host protein degradation⁴⁰. The *Yersinia* effector YopJ, a member of the AvrRxv family of effectors from phytopathogenic bacteria, inhibits MAP kinase cascades by acetylation of phosphorylation-regulated residues on a MEK protein⁴¹. Many additional bacterial type III effectors protein families have been identified²⁶–²⁸; their targets and functions await definition.

Effectors from plant pathogens that are eukaryotic are poorly understood. Fungal and oomycete effectors can act either in the extracellular matrix or inside the host cell. For example, the tomato RLPs, Cf-2, Cf-4, Cf-5 and Cf-9 respond specifically to extracellular effectors produced by *Cladosporium fulvum*⁴². Other fungal and oomycete effectors probably act inside the host cell; they are recognized by NB-LRR proteins. For example, the gene encoding the oomycete effector Atr13 from *Hyaloperonospora parasitica* exhibits extensive allelic diversity between *H. parasitica* strains matched by diversity at the corresponding *Arabidopsis* RPP13 NB-LRR locus⁴³. Diversity is also observed across *H. parasitica* Atr1 and *Arabidopsis* RPP1 alleles⁴⁴. Atr1 and Atr13 carry signal peptides for secretion from *H. parasitica*. They share with each other, and with the *Phytophthora infestans* Avr3a protein, an RxLR motif, that enables import of *Plasmodium* effectors into mammalian host cells⁴⁵. This is consistent with the taxonomic proximity of oomycetes and *Plasmodium*. Races of the flax rust fungus *Melampsora lini* express Avr genes recognized by specific alleles of the flax L, M and P NB-LRR proteins. These haustorial proteins carry signal peptides for fungal export and can function inside the plant cell⁴⁶,⁴⁷. How they are taken up by the host cell is unknown. However, the barley powdery mildew (*Blumeria graminis* f.sp. *hordei*) Avrk and Avra10 proteins, recognized by the NB-LRR barley genes *Mlk* and *Mla10*, contain neither obvious signal peptides nor RxLR motifs, yet are members of large gene families in *Blumeria* and *Erysiphe* species⁴⁸. How these oomycete and fungal effectors are delivered to the host cell and contribute to pathogen virulence is unknown.

Pathogens produce small molecule effectors that mimic plant hormones. Some *P. syringae* strains make coronatine, a jasmonic acid mimic that suppresses salicylic-acid-mediated defence to biotrophic pathogens ${ }^{49,50}$ and induces stomatal opening, helping pathogenic bacteria gain access to the apoplast ${ }^{51}$. PTI involves repression of auxin responses, mediated in part by a micro-RNA that is also induced during abscisic-acid-mediated stress responses ${ }^{52}$. Gibberellin is produced by the fungal pathogen *Gibberella fujikuroi* leading to ‘foolish seedling’ syndrome, and cytokinin produced by many pathogens can promote pathogen success through retardation of senescence in infected leaf tissue. The interplay between PTI and normal hormone signalling, and pathogen mimics that influence it, is just beginning to be unravelled.

**Indirect and direct host recognition of pathogen effectors**

Effectors that enable pathogens to overcome PTI are recognized by specific disease resistance (*R*) genes. Most *R* genes encode NB-LRR proteins; there are $\sim 125$ in the *Arabidopsis* Col-0 genome. If one effector is recognized by a corresponding NB-LRR protein, ETI ensues. The recognized effector is termed an avirulence (Avr) protein. ETI is a faster and stronger version of PTI ${ }^{30-32}$ that often culminates in HR ${ }^{53}$ (Fig. 1). HR typically does not extend beyond the infected cell: it may retard pathogen growth in some interactions, particularly those involving haustorial parasites, but is not always observed, nor required, for ETI. It is unclear what actually stops pathogen growth in most cases.

Very little is known about the signalling events required to activate NB-LRR-mediated ETI. NB-LRR proteins are probably folded in a signal competent state by cytosolic heat shock protein 90 and other receptor co-chaperones ${ }^{54,55}$. The LRRs seem to act as negative regulators that block inappropriate NB activation. NB-LRR activation involves intra- and intermolecular conformational changes and may resemble the induced proximity mechanism by which the related animal Apaf-1 protein activates programmed cell death ${ }^{56}$.

NB-LRR activation results in a network of cross-talk between response pathways deployed, in part, to differentiate biotrophic from necrotrophic pathogen attack ${ }^{9}$. This is maintained by the balance between salicylic acid, a local and systemic signal for resistance against many biotrophs, and the combination of jasmonic acid and ethylene accumulation as signals that promote defence against necrotrophs ${ }^{9}$. Additional plant hormones are likely to alter the salicylic-acid–jasmonic-acid/ethylene signalling balance. *Arabidopsis* mutants defective in salicylic acid biosynthesis or responsiveness are compromised in both basal defence and systemic acquired resistance (SAR) ${ }^{57}$. NB-LRR activation induces differential salicylic-acid- and ROS-dependent responses at and surrounding infection sites, and systemically ${ }^{58}$. The NADPH-oxidase-dependent oxidative burst that accompanies ETI represses salicylic acid-dependent cell death spread in cells surrounding infection sites ${ }^{59}$. Local and systemic changes in gene expression are mediated largely by transcription factors of the WRKY and TGA families ${ }^{60}$.

Several NB-LRR proteins recognize type III effectors indirectly, by detecting products of their action on host targets, consistent with the ‘guard hypothesis’ ${ }^{1}$. The key tenets of this hypothesis are that: (1) an effector acting as a virulence factor has a target(s) in the host; (2) by manipulating or altering this target(s) the effector contributes to pathogen success in susceptible host genotypes; and (3) effector perturbation of a host target generates a ‘pathogen-induced modified-self’ molecular pattern, which activates the corresponding NB-LRR protein, leading to ETI. Three important consequences of this model, now supported by experimental evidence, are that: (1) multiple effectors could evolve independently to manipulate the same host target, (2) this could drive the evolution of more than one NB-LRR protein associated with a target of multiple effectors, and (3) these NB-LRRs would be activated by recognition of different modified-self patterns produced on the same target by the action of the effectors in (1).

RIN4, a 211-amino-acid, acylated ${ }^{61}$ and plasma-membrane-associated protein, is an archetypal example of a host target of type III effectors that is guarded by NB-LRR proteins (Fig. 2). It is manipulated by three different bacterial effectors, and associates *in vivo* with two *Arabidopsis* NB-LRR proteins (Fig. 2a and 2b). Two unrelated type III effectors, AvrRpm1 and AvrB, interact with and induce phosphorylation of RIN4 (ref. 62). This RIN4 modification is predicted to activate the RPM1 NB-LRR protein. A third effector, AvrRpt2 is a cysteine protease ${ }^{63}$, activated inside the host cell ${ }^{64}$, that eliminates RIN4 by cleaving it at two sites ${ }^{61,65}$. Cleavage of RIN4 activates the RPS2 NB-LRR protein ${ }^{66,67}$. Activation of both RPM1 and RPS2 requires the GPI-anchored NDR1 protein, and RIN4 interacts with NDR1 ${ }^{68}$.

If RIN4 was the only target for these three effectors, then its elimination would abolish their ability to add virulence to a weakly pathogenic strain. However, elimination of RIN4 demonstrated that it is not the only host target for AvrRpm1 or AvrRpt2 in susceptible (*rin4 rpm1 rps2*) plants ${ }^{69}$. Additionally, AvrRpt2 can cleave *in vitro* several *Arabidopsis* proteins that contain its consensus cleavage site ${ }^{65}$. Hence, any effector’s contribution to virulence might involve manipulation of several host targets, and the generation of several modified-self molecules. However, the perturbation of only one target is sufficient for NB-LRR activation. RIN4 negatively regulates RPS2 and RPM1 (and only these two NB-LRR proteins) ${ }^{69,70}$. But what is the function of RIN4 in the absence of RPS2 and RPM1? In *rpm1 rps2* plants, AvrRpt2 or AvrRpm1 (and possibly other effectors) manipulate RIN4 (and possibly associated proteins or other targets) in order to suppress PTI ${ }^{71}$. Thus, plants use NB-LRR proteins to guard against pathogens that deploy effectors to inhibit PAMP-signalling. Additional examples of indirect recognition are detailed in Fig. 2; these include both intra- and extra-cellular recognition of pathogen-induced modified self.

Not all NB-LRR recognition is indirect, and there are three examples of direct Avr-NB-LRR interaction ${ }^{72-74}$. The flax *L* locus alleles encode NB-LRR proteins that interact in yeast with the corresponding AvrL proteins, providing the first evidence that effector diversity determining NB-LRR recognition can be correlated perfectly with effector–NB-LRR-protein interaction ${ }^{72}$. Both *L* and AvrL proteins are under diversifying selection, arguing for a direct evolutionary arms race. The allelic diversity of other fungal and oomycete pathogen effectors, and of their corresponding host NB-LRR proteins as described above, also suggests direct interaction, though this remains to be demonstrated.

The evolutionary radiation of several hundred thousand angiosperm plant species $\sim 140-180$ million years ago was probably accompanied by many independent cases of pathogen co-evolution, particularly of host-adapted obligate biotrophs. Most plants resist infection by most pathogens; they are said to be ‘non-hosts’. This non-host resistance could be mediated by at least two mechanisms. First, a pathogen’s effectors could be ineffective on a potential new, but evolutionarily divergent, host, resulting in little or no suppression of PTI, and failure of pathogen growth. Alternatively, one or more of the effector complement of the would-be pathogen could be recognized by the NB-LRR repertoire of plants other than its co-adapted host, resulting in ETI. These two scenarios predict different outcomes with respect to the timing and amplitude of the response they would trigger, and they also give rise to different evolutionary pressures on both host and pathogen.

Non-host resistance in *Arabidopsis* against the non-adapted barley pathogen, *B. graminis* f. sp. *hordei* (Bgh) normally involves the rapid production of cell wall appositions (physical barriers) and antimicrobial metabolites at the site of pathogen entry, but no HR. *Arabidopsis* penetration (*pen*) mutants are partially compromised in this response. PEN2 is a peroxisomal glucosyl hydrolase ${ }^{75}$, and PEN3 encodes a plasma membrane ABC transporter ${ }^{76}$. PEN2 and PEN3 are both recruited to attempted fungal entry sites, apparently to mediate the polarized delivery of a toxin to the apoplast ${ }^{75,76}$. The

REVIEWs

**Pseudomonas syringae**

a

Susceptible host  
NDR1  

Resistant host  

No RPM1  
(rpm1 plants)  

Increased virulence owing  
to AvrRpm1 activity on  
RIN4 and other targets  

b

Susceptible host  
NDR1  

Resistant host  

No RPS2  
(rps2 plants)  

Increased virulence owing  
to AvrRpt2 activity on  
RIN4 and other targets  

c

Susceptible host  

Resistant host  

No RPS5  
(rps5 plants)  

Increased virulence owing  
to AvrPphB activity  

d

Susceptible host  

Resistant host  

No PRF or PTO  
(prf or pto plants)  

Increased virulence owing  
to AvrPto and AvrPtoB activity  
on unknown targets  

e

Susceptible host  

Inhibition of Rcr3 protease  
activity by the Avr2 protease  
inhibitor contributes to disease  

Activation of Cf-2-dependent  
defence response  

actin cytoskeleton probably contributes to this response<sup>77</sup>, perhaps as  
a track for PEN2-containing peroxisomes and/or vesicles. This pre-  
invasion non-host resistance is genetically separable from a post-  
invasion mechanism that requires additional elements that regulate  
both PTI and ETI<sup>78</sup>. Elimination of both PEN2 and PTI/ETI signal-  
ling transforms *Arabidopsis* into a host for an evolutionarily non-  
adapted fungal pathogen<sup>75</sup>. This suggests that non-host resistance  
comprises mechanistically distinct layers of resistance.

The PEN1 syntaxin acts in a different pre-invasion non-host res-  
istance pathway. PEN1 is likely to be part of a ternary SNARE com-  
plex that secretes vesicle cargo to the site of attempted fungal  
invasion, contributing to formation of cell wall appositions<sup>79–81</sup>. Specific seven-transmembrane MLO (mildew resistance locus O)  
family members negatively regulate PEN1-dependent secretion at  
sites of attempted pathogen ingress<sup>79,80</sup>. Recessive *mlo* mutations in  
either *Arabidopsis* or barley result in resistance to the respective co-  
evolved powdery mildew pathogens<sup>82</sup>. Hence, in both *Arabidopsis*  
and barley, these fungi might suppress PEN1-mediated disease res-  
istance by activation of MLO. This remarkable set of findings implies  
that a common host cell entry mechanism evolved in powdery mil-  
dew fungi at or before the monocot-dicot divergence. *PEN2* and  
*PEN3* genes are induced by flg22, indicating that they might be  
involved in PTI.

Non-host resistance can also be mediated by parallel ETI res-  
ponses. For example, four bacterial effectors from a tomato pathogen  

---

Figure 2 | Plant immune system activation by pathogen effectors that  
generate modified self molecular patterns. **a**, *Arabidopsis* RPM1 is a  
peripheral plasma membrane NB-LRR protein. It is activated by either the  
AvrRpm1 or the AvrB effector proteins. AvrRpm1 enhances the virulence of  
some *P. syringae* strains on *Arabidopsis* as does AvrB on soybeans. AvrRpm1  
and AvrB are modified by eukaryote-specific acylation once delivered into  
the cell by the type III secretion system (red syringe) and are thus targeted to  
the plasma membrane. The biochemical functions of AvrRpm1 and AvrB are  
unknown, although they target RIN4, which becomes phosphorylated (+P),  
and activate RPM1, as detailed in the text. In the absence of RPM1, AvrRpm1  
and AvrB presumably act on RIN4 and other targets to contribute to  
virulence. Light blue eggs in this and subsequent panels represent as yet  
unknown proteins. **b**, RPS2 is an NB-LRR protein that resides at the plasma  
membrane. It is activated by the AvrRpt2 cysteine protease type III effector  
from *P. syringae*. Auto-processing of AvrRpt2 by a host cyclophilin reveals a  
consensus, but unconfirmed, myristoylation site at the new amino terminus,  
suggesting that it might also be localized to the host plasma membrane.  
AvrRpt2 is the third effector that targets RIN4. Cleavage of RIN4 by AvrRpt2  
leads to RPS2-mediated ETI. In the absence of RPS2, AvrRpt2 presumably  
cleaves RIN4 and other targets as part of its virulence function. **c**, RPS5 is an  
*Arabidopsis* NB-LRR protein localized to a membrane fraction, probably via  
acylation. RPS5 is NDR1-independent. It is activated by the AvrPphB  
cysteine protease effector from *P. syringae*<sup>100</sup>. AvrPphB is cleaved, acylated  
and delivered to the host plasma membrane. Activated AvrPphB cleaves the  
*Arabidopsis* PBS1 serine-threonine protein kinase, leading to RPS5  
activation. The catalytic activity of cleaved PBS1 is required for RPS5  
activation, suggesting that this ‘modified-self’ fragment retains its enzymatic  
activity as part of the RPS5 activation mechanism<sup>100</sup>. To date, no function  
has been ascribed to PBS1 in the absence of RPS5. **d**, Pto is a tomato serine-  
threonine protein kinase. Pto is polymorphic and hence satisfies the genetic  
criteria for the definition of a disease resistance protein. Pto activity requires  
the NB-LRR protein Prf, and the proteins form a molecular complex<sup>101</sup>. Prf is  
monomorphic, at least in the tomato species analysed to date. Pto is the  
direct target of two unrelated *P. syringae* effectors, AvrPto and AvrPtoB,  
each of which contributes to pathogen virulence in *pto* mutants<sup>102</sup>. It is thus  
likely that Prf guards Pto (refs 101, 103). The Pto kinase is apparently not  
required for PTI, though there may be redundancy in its function because it  
is a member of a gene family. **e**, The transmembrane RLP Cf-2 guards the  
extracellular cysteine protease Rcr3. Cf-2 recognizes the *C. fulvum*  
extracellular effector Avr2, which encodes a cysteine protease inhibitor. Avr2  
binds and inhibits the tomato Rcr3 cysteine protease. Mutations in Rcr3  
result in the specific loss of Cf-2-dependent recognition of Avr2. Hence, Cf-2  
seems to monitor the state of Rcr3, and activates defence if Rcr3 is inhibited  
by Avr2 (ref. 104).
unable to colonize soybean can each trigger specific soybean R genes when delivered from a soybean pathogen⁸³. Deletion of these effector genes from the tomato pathogen diminishes its virulence on tomato, but does not allow it to colonize soybean⁸⁴. Hence, there might be other factors lacking in this strain that are required to colonize soybean. Also, a widely distributed, monomorphic effector acting as an avirulence protein is sufficient to render *Magnaporthe oryzae* strains unable to colonize rice. Its presence in over 50 strains that successfully colonize perennial ryegrass suggests a virulence function⁸⁵. Finally, *Arabidopsis* non-host resistance to *Leptosphaeria maculans*, a fungal pathogen of *Brassica*, is actually mediated by unlinked NB-LRR proteins present in each parent of a cross between two accessions⁸⁶. Hence, cryptic NB-LRR mediated responses acting in parallel can limit pathogen host range.

### Pathogens dodge host surveillance

The effectiveness of ETI selects for microbial variants that can avoid NB-LRR-mediated recognition of a particular effector (Fig. 3). Effector allele frequencies are likely to be influenced by their mode of action. The diversity of both flax rust AvrL alleles and oomycete Atr13 and Atr1 alleles suggests one means of effector evolution. These proteins are likely to interact directly *in planta* with proteins encoded by alleles of the flax *L* and *Arabidopsis RPP1* and *RPP13* loci, respectively. The high level of diversifying selection among these effector alleles is presumably selected by host recognition, and hence acts on effector residues that are probably not required for effector function.

In contrast, effectors providing biochemical functions that generate modifications of host targets are likely to be under purifying selection⁸⁷. NB-LRR activation via recognition of pathogen-induced modified self provides a mechanism for host perception of multiple effectors evolved to compromise the same host target (Fig. 2). For selection to generate an effector that escapes ETI, the effector is likely to lose its nominal function. The simplest pathogen response to host recognition is to jettison the detected effector gene, provided the population’s effector repertoire can cover the potential loss of fitness.

#### Figure 3 | Co-evolution of host R genes and the pathogen effector complement

A pathogen carries an effector gene (*E1*) that is recognized by a rare *R1* allele (top). This results in selection for an elevated frequency of *R1* in the population. Pathogens in which the effector is mutated are then selected, because they can grow on *R1*-containing plants (right). *R1* effectiveness erodes, and, because at least some *R* genes have associated fitness costs⁹⁴, plants carrying *R1* can have reduced fitness (bottom), resulting in reduced *R1* frequencies. The pathogen population will still contain individuals with *E1*. In the absence of *R1*, *E1* will confer increased fitness, and its frequency in the population will increase (left). This will lead to resumption of selection for *R1* (top). In populations of plants and pathogens, this cycle is continuously turning, with scores of effectors and many alleles at various *R* loci in play.

on susceptible hosts. In fact, effector genes are often associated with mobile genetic elements or telomeres and are commonly observed as presence/absence polymorphisms across bacterial and fungal strains. Indirect recognition of effector action, via recognition of pathogen-induced modified self, is likely to enable relatively stable, durable and evolutionarily economic protection of the set of cellular machinery targeted by pathogen effectors.

ETI can also be overcome through evolution of pathogen effectors that suppress it directly (Fig. 1). For example, in *P. syringae* pv. *phaseolicola*, the AvrPphC effector suppresses ETI triggered by the AvrPphF effector in some cultivars of bean, whereas, as its name implies, AvrPphC itself can condition avirulence on different bean cultivars⁸⁸. Other cases of bacterial effectors acting to dampen or inhibit ETI have been observed³⁸. Genetic analysis in flax rust revealed so-called inhibitor genes that function to suppress ETI triggered by other avirulence genes⁸⁹. Hence, it seems likely that some effectors suppress the ETI triggered by other effectors.

Microbial evolution in response to ETI may result in two extremes of NB-LRR evolution. NB-LRR gene homologues in diverse *Arabidopsis* accessions accumulate evolutionary novelty at different rates at different loci⁹⁰. Some NB-LRR genes are not prone to duplication, and are evolving relatively slowly. Their products are perhaps stably associated with a host protein whose integrity they monitor, retarding diversification. Others are evolving more rapidly and may interact directly with rapidly evolving effectors⁹¹,⁹². What might drive these evolutionary modes (Fig. 3)? In pathogen populations, the frequency of an effector gene will be enhanced by its ability to promote virulence, and reduced by host recognition. For example, in the flax/flax rust system, avirulence gene frequency in the rust is elevated on plant populations with a lower abundance of the corresponding *R* genes, consistent with *avr* genes increasing pathogen fitness⁹³. Hence, natural selection should maintain effector function in the absence of recognition. But effector function has a cost that is dependent on the frequency of the corresponding *R* gene. And *R* genes may exact a fitness cost in the host⁹⁴. Thus, if effector frequency drops in a pathogen population, hosts might be selected for loss of the corresponding *R* allele, and the frequency-dependent cycle would continue (Fig. 3).

### Challenges and opportunities for the future

We need to define the repertoire and modes of action of effectors from pathogens with diverse life histories. This will help to define the comprehensive set of host targets, as well as the evolutionary pressures acting on both hosts and pathogens. For example, if the majority of bacterial effectors evolve under purifying selection to maintain intrinsic function⁸⁷, then a population-wide set of unrelated microbial effectors might converge onto a limited set of NB-LRR-associated host targets. These stable associations of NB-LRR proteins and the host proteins whose integrity they monitor are presumably being challenged by newly evolved, or newly acquired, effectors that can still surreptitiously manipulate the target in the service of virulence.

We need to understand the haustorial interface. Rewiring of host and microbe vesicle traffic will probably underpin haustorial differentiation. We do not yet know whether the extrahaustorial membrane is derived from the host plasma membrane or is a newly synthesized, novel host membrane. High-throughput sequencing renders it feasible to index the gene complements of obligate biotrophs like powdery and downy mildews, and rusts. Genomics can be also be used to identify the genes expressed by the pathogen over a time course of infection. The presence of a signal peptide and (in the case of oomycetes) an RxLR motif, can then be used to computationally identify the complement of effector candidates. With the development of appropriate high-throughput delivery systems, it will be possible to investigate their functions, and their ability to impinge on PTI and/or ETI, on both host plants and other plant species.

Do the transcriptional controls of PTI and ETI, which culminate in similar outputs, overlap? Several effectors can be nuclear localized⁹⁵,⁹⁶.
The RRS1 R protein is perhaps a Rosetta Stone chimera of NB-LRR and WRKY transcription factor<sup>74</sup>. A TATA-binding protein accessory factor, AtTIP49a, interacts with RPM1 and other NB-LRR proteins, and is a negative regulator of defence<sup>97</sup>. Two nucleoporins and importins are required for the output response of an ectopically activated NB-LRR protein<sup>98,99</sup>. Notably, the prototypic animal CATERPILLAR protein is CIITA, a transcriptional co-activator of MHC class II response to viral infection that is, in turn, the target of viral proteins that aim to shut it down<sup>7</sup>. Whether NB-LRR proteins are transcriptional co-regulators is at present unknown.

We need to know what causes pathogen growth arrest. Because plants are sessile, they must continuously integrate both biotic and abiotic signals from the environment. Plants lack circulating cells, so these responses also need to be partitioned both locally over several cell diameters and systemically over metres. Understanding the spatial interplay of PTI, ETI, and plant hormone and abiotic stress-signalling systems is in its infancy.

Finally, we need to understand the population biology of pathogen effectors, and their co-evolving host NB-LRR genes. Knowledge of their allele frequencies and their spatial distribution in wild ecosystems should tell us more about the evolution of this fascinating ancient immune system and how we might deploy it more effectively to control disease.

1. Dangl, J. L. & Jones, J. D. G. Plant pathogens and integrated defence responses to infection. *Nature* 411, 826–833 (2001).
2. Ausubel, F. M. Are innate immune signaling pathways in plants and animals conserved? *Nature Immunol.* 6, 973–979 (2005).
3. Chisholm, S. T., Coaker, G., Day, B. & Staskawicz, B. J. Host–microbe interactions: shaping the evolution of the plant immune response. *Cell* 124, 803–814 (2006).
4. van der Biezen, E. A. & Jones, J. D. G. Plant disease resistance proteins and the gene-for-gene concept. *Trends Biochem. Sci.* 23, 454–456 (1998).
5. Matzinger, P. The danger model: a renewed sense of self. *Science* 296, 301–305 (2002).
6. Zipfel, C. & Felix, G. Plants and animals: a different taste for microbes? *Curr. Opin. Plant Biol.* 8, 353–360 (2005).
7. Ting, J. P. & Davis, B. K. CATERPILLER: a novel gene family important in immunity, cell death, and diseases. *Annu. Rev. Immunol.* 23, 387–414 (2005).
8. Leipe, D. D., Koonin, E. V. & Aravind, L. STAND, a class of P-loop NTPases including animal and plant regulators of programmed cell death: multiple, complex domain architectures, unusual phyletic patterns, and evolution by horizontal gene transfer. *J. Mol. Biol.* 343, 1–28 (2004).
9. Glazebrook, J. Contrasting mechanisms of defense against biotrophic and necrotrophic pathogens. *Annu. Rev. Phytopathol.* 43, 205–227 (2005).
10. Voinnet, O. Induction and suppression of RNA silencing: insights from viral infections. *Nature Rev. Genet.* 6, 206–220 (2005).
11. Kessler, A. & Baldwin, I. T. Plant responses to insect herbivory: the emerging molecular analysis. *Annu. Rev. Plant Biol.* 53, 299–328 (2002).
12. Gomez-Gomez, L. & Boller, T. Flagellin perception: a paradigm for innate immunity. *Trends Plant Sci.* 7, 251–256 (2002).
13. Felix, G., Duran, J. D., Volko, S. & Boller, T. Plants have a sensitive perception system for the most conserved domain of bacterial flagellin. *Plant J.* 18, 265–276 (1999).
14. Zipfel, C. *et al.* Bacterial disease resistance in Arabidopsis through flagellin perception. *Nature* 428, 764–767 (2004).
15. Chinchilla, D., Bauer, Z., Regenass, M., Boller, T. & Felix, G. The Arabidopsis receptor kinase FLS2 binds flg22 and determines the specificity of flagellin perception. *Plant Cell* 18, 465–476 (2006).
16. Robatzek, S., Chinchilla, D. & Boller, T. Ligand-induced endocytosis of the pattern recognition receptor FLS2 in Arabidopsis. *Genes Dev.* 20, 537–542 (2006).
17. Sun, W., Dunning, F. M., Pfund, C., Weingarten, R. & Bent, A. F. Within-species flagellin polymorphism in Xanthomonas campestris pv campestris and its impact on elicitation of Arabidopsis FLAGELLIN SENSING2-dependent defenses. *Plant Cell* 18, 764–779, doi:10.1105/tpc.105.037648 (2006).
18. Felix, G. & Boller, T. Molecular sensing of bacteria in plants. The highly conserved RNA-binding motif RNP-1 of bacterial cold shock proteins is recognized as an elicitor signal in tobacco. *J. Biol. Chem.* 278, 6201–6208 (2003).
19. Kunze, G. *et al.* The N terminus of bacterial elongation factor Tu elicits innate immunity in Arabidopsis plants. *Plant Cell* 16, 3496–3507 (2004).
20. Zipfel, C. *et al.* Perception of the bacterial PAMP EF-Tu by the receptor EFR restricts Agrobacterium-mediated transformation. *Cell* 125, 749–760 (2006).
21. Shiu, S. H. & Bleecker, A. B. Expansion of the receptor-like kinase/Pelle gene family and receptor-like proteins in Arabidopsis. *Plant Physiol.* 132, 530–543 (2003).
22. Dardick, C. & Ronald, P. Plant and animal pathogen recognition receptors signal through non-RD kinases. *PLoS Pathog.* 2, e2 (2006).

23. Fritz-Laylin, L. K., Krishnamurthy, N., Tor, M., Sjolander, K. V. & Jones, J. D. Phylogenomic analysis of the receptor-like proteins of rice and Arabidopsis. *Plant Physiol.* 138, 611–623 (2005).
24. Schulze-Lefert, P. & Panstruga, R. Establishment of biotrophy by parasitic fungi and reprogramming of host cells for disease resistance. *Annu. Rev. Phytopathol.* 41, 641–667 (2003).
25. Badel, J. L., Charkowski, A. O., Deng, W. L. & Collmer, A. A gene in the *Pseudomonas syringae* pv. tomato Hrp pathogenicity island conserved effector locus, *hopPtoA1*, contributes to efficient formation of bacterial colonies in planta and is duplicated elsewhere in the genome. *Mol. Plant Microbe Interact.* 15, 1014–1024 (2002).
26. Grant, S. R., Fisher, E. J., Chang, J. H., Mole, B. M. & Dangl, J. L. Subterfuge and manipulation: type III effector proteins of phytopathogenic bacteria. *Annu. Rev. Microbiol.* 60, 425–449, doi:10.1146/annurev.micro.60.080805.142251 (2006).
27. Abramovitch, R. B., Anderson, J. C. & Martin, G. B. Bacterial elicitation and evasion of plant innate immunity. *Nature Rev. Mol. Cell Biol.* 7, 601–611 (2006).
28. Mudgett, M. B. New insights to the function of phytopathogenic bacterial type III effectors in plants. *Annu. Rev. Plant Biol.* 56, 509–531 (2005).
29. Jakobek, J. L., Smith, J. A. & Lindgren, P. B. Suppression of bean defense responses by *Pseudomonas syringae*. *Plant Cell* 5, 57–63 (1993).
30. Thilmony, R., Underwood, W. & He, S. Y. Genome-wide transcriptional analysis of the *Arabidopsis thaliana* interaction with the plant pathogen *Pseudomonas syringae* pv. tomato DC3000 and the human pathogen *Escherichia coli* O157:H7. *Plant J.* 46, 34–53 (2006).
31. Tao, Y. *et al.* Quantitative nature of *Arabidopsis* responses during compatible and incompatible interactions with the bacterial pathogen *Pseudomonas syringae*. *Plant Cell* 15, 317–330 (2003).
32. Truman, W., de Zabala, M. T. & Grant, M. Type III effectors orchestrate a complex interplay between transcriptional networks to modify basal defence responses during pathogenesis and resistance. *Plant J.* 46, 14–33 (2006).
33. Nomura, K., Melotto, M. & He, S. Y. Suppression of host defense in compatible plant–*Pseudomonas syringae* interactions. *Curr. Opin. Plant Biol.* 8, 361–368 (2005).
34. Desveaux, D., Singer, A. U. & Dangl, J. L. Type III effector proteins: doppelgangers of bacterial virulence. *Curr. Opin. Plant Biol.* 9, 376–382 (2006).
35. Nomura, K. *et al.* A bacterial virulence protein suppresses host innate immunity to cause plant disease. *Science* 313, 220–223, doi:10.1126/science.1129523 (2006).
36. DebRoy S., Thilmony, R., Kwack, Y-B., Nomura, K. & He, S. Y. A family of conserved bacterial effectors inhibits salicylic acid-mediated basal immunity and promotes disease necrosis in plants. *Proc. Natl Acad. Sci. USA* 101, 9927–9932 (2004).
37. He, P. *et al.* Specific bacterial suppressors of MAMP signaling upstream of MAPKKK in *Arabidopsis* innate immunity. *Cell* 125, 563–575 (2006).
38. Abramovitch, R. B., Kim, Y. J., Chen, S., Dickman, M. B. & Martin, G. B. *Pseudomonas* type III effector AvrPtoB induces plant disease susceptibility by inhibition of host programmed cell death. *EMBO J.* 22, 60–69 (2003).
39. de Torres, M. *et al.* *Pseudomonas syringae* effector AvrPtoB suppresses basal defence in *Arabidopsis*. *Plant J.* 47, 368–382, 10.1111/j.1365–313X.2006.02798.x (2006).
40. Janjusevic, R., Abramovitch, R. B., Martin, G. B. & Stebbins, C. E. A bacterial inhibitor of host programmed cell death defenses is an E3 ubiquitin ligase. *Science* 311, 222–226 (2006).
41. Mukherjee, S. *et al.* *Yersinia* YopJ acetylates and inhibits kinase activation by blocking phosphorylation. *Science* 312, 1211–1214 (2006).
42. Rivas, S. & Thomas, C. M. Molecular interactions between tomato and the leaf mold pathogen *Cladosporium fulvum*. *Annu. Rev. Phytopathol.* 43, 395–436 (2005).
43. Allen, R. L. *et al.* Host–parasite coevolutionary conflict between *Arabidopsis* and downy mildew. *Science* 306, 1957–1960 (2004).
44. Rehmany, A. P. *et al.* Differential recognition of highly divergent downy mildew avirulence gene alleles by *RPP1* resistance genes from two *Arabidopsis* lines. *Plant Cell* 17, 1839–1850 (2005).
45. Bhattacharjee, S. *et al.* The malarial host-targeting signal is conserved in the Irish potato famine pathogen. *PLoS Pathog.* 2, e50 (2006).
46. Dodds, P. N., Lawrence, G. J., Catanzariti, A. M., Ayliffe, M. A. & Ellis, J. G. The *Melampsora lini* AvrL567 avirulence genes are expressed in haustoria and their products are recognized inside plant cells. *Plant Cell* 16, 755–768 (2004).
47. Catanzariti, A. M., Dodds, P. N., Lawrence, G. J., Ayliffe, M. A. & Ellis, J. G. Haustorially expressed secreted proteins from flax rust are highly enriched for avirulence elicitors. *Plant Cell* 18, 243–256 (2006).
49. Zhao, Y. *et al.* Virulence systems of *Pseudomonas syringae* pv. tomato promote bacterial speck disease in tomato by targeting the jasmonate signaling pathway. *Plant J.* 36, 485–499 (2003).
50. Brooks, D. M., Bender, C. L. & Kunkel, B. N. The *Pseudomonas syringae* phytotoxin coronatine promotes virulence by overcoming salicylic acid-dependent defences in *Arabidopsis thaliana*. *Mol. Plant Pathol.* 6, 629–640 (2005).
51. Melotto, M., Underwood, W., Koczan, J., Nomura, K. & He, S. The innate immune function of plant stomata against bacterial invasion. *Cell* 126, 969–980 (2006).
52. Navarro, L. *et al.* A plant miRNA contributes to antibacterial resistance by repressing auxin signaling. *Science* 312, 436–439 (2006).
53. Greenberg, J. T. & Yao, N. The role and regulation of programmed cell death in plant–pathogen interactions. *Cell. Microbiol.* 6, 201–211 (2004).
54. Schulze-Lefert, P. Plant immunity: the origami of receptor activation. *Curr. Biol.* 14, R22–R24 (2004).

NATURE|Vol 444|16 November 2006

55. Holt, B. F. III, Belkhadir, Y. & Dangl, J. L. Antagonistic control of disease resistance protein stability in the plant immune system. *Science* **309**, 929–932 (2005).

56. Takken, F. L., Albrecht, M. & Tameling, W. I. Resistance proteins: molecular switches of plant defence. *Curr. Opin. Plant Biol.* **9**, 383–390 (2006).

57. Durrant, W. E. & Dong, X. Systemic acquired resistance. *Annu. Rev. Phytopathol.* **42**, 185–209 (2004).

58. Dorey, S. *et al.* Spatial and temporal induction of cell death, defense genes, and accumulation of salicylic acid in tobacco leaves reacting hypersensitively to a fungal glycoprotein. *Mol. Plant Microbe Interact.* **10**, 646–655 (1997).

59. Torres, M. A., Jones, J. D. & Dangl, J. L. Pathogen-induced, NADPH oxidase-derived reactive oxygen intermediates suppress spread of cell death in *Arabidopsis thaliana*. *Nature Genet.* **37**, 1130–1134 (2005).

60. Eulgem, T. Regulation of the *Arabidopsis* defense transcriptome. *Trends Plant Sci.* **10**, 71–78 (2005).

61. Kim, H. S. *et al.* The *Pseudomonas syringae* effector AvrRpt2 cleaves its C-terminally acylated target, RIN4, from *Arabidopsis* membranes to block RPM1 activation. *Proc. Natl Acad. Sci. USA* **102**, 6496–6501 (2005).

62. Mackey, D., Holt, B. F. III, Wiig, A. & Dangl, J. L. RIN4 interacts with *Pseudomonas syringae* type III effector molecules and is required for RPM1-mediated disease resistance in *Arabidopsis*. *Cell* **108**, 743–754 (2002).

63. Axtell, M. J., Chisholm, S. T., Dahlbeck, D. & Staskawicz, B. J. Genetic and molecular evidence that the *Pseudomonas syringae* type III effector protein AvrRpt2 is a cysteine protease. *Mol. Microbiol.* **49**, 1537–1546 (2003).

64. Coaker, G., Falick, A. & Staskawicz, B. Activation of a phytopathogenic bacterial effector protein by a eukaryotic cyclophilin. *Science* **308**, 548–550 (2005).

65. Chisholm, S. T. *et al.* Molecular characterization of proteolytic cleavage sites of the *Pseudomonas syringae* effector AvrRpt2. *Proc. Natl Acad. Sci. USA* **102**, 2087–2092 (2005).

66. Mackey, D., Belkhadir, Y., Alonso, J. M., Ecker, J. R. & Dangl, J. L. *Arabidopsis* RIN4 is a target of the type III virulence effector AvrRpt2 and modulates RPS2-mediated resistance. *Cell* **112**, 379–389 (2003).

67. Axtell, M. J. & Staskawicz, B. J. Initiation of RPS2-specified disease resistance in *Arabidopsis* is coupled to the AvrRpt2-directed elimination of RIN4. *Cell* **112**, 369–377 (2003).

68. Day, B., Dahlbeck, D. & Staskawicz, B. J. NDR1 interaction with RIN4 mediates the differential activation of multiple disease resistance pathways. *Plant Cell* (September 29, doi:10.1105/tpc.106.044693 2006).

69. Belkhadir, Y., Nimchonok, Z., Hubert, D. A., Mackey, D. & Dangl, J. L. *Arabidopsis* RIN4 negatively regulates disease resistance mediated by RPS2 and RPM1 downstream or independent of the NDR1 signal modulator, and is not required for the virulence functions of bacterial type III effectors AvrRpt2 or AvrRpm1. *Plant Cell* **16**, 2822–2835 (2004).

70. Day, B. *et al.* Molecular basis for the RIN4 negative regulation of RPS2 disease resistance. *Plant Cell* **17**, 1292–1305 (2005).

72. Dodds, P. N. *et al.* Direct protein interaction underlies gene-for-gene specificity and coevolution of the flax resistance genes and flax rust avirulence genes. *Proc. Natl Acad. Sci. USA* **103**, 8888–8893 (2006).

73. Jia, Y., McAdams, S. A., Bryan, G. T., Hershey, H. P. & Valent, B. Direct interaction of resistance gene and avirulence gene products confers rice blast resistance. *EMBO J.* **19**, 4004–4014 (2000).

74. Deslandes, L. *et al.* Physical interaction between RRS1-R, a protein conferring resistance to bacterial wilt, and PopP2, a type III effector targeted to the plant nucleus. *Proc. Natl Acad. Sci. USA* **100**, 8024–8029 (June 3, doi:10.1073/pnas.1230660100 2003).

75. Lipka, V. *et al.* Pre- and postinvasion defenses both contribute to nonhost resistance in *Arabidopsis*. *Science* **310**, 1180–1183, doi:10.1126/science.1119409 (2005).

76. Stein, M. *et al.* *Arabidopsis* PEN3/PDR8, an ATP binding cassette transporter, contributes to nonhost resistance to inappropriate pathogens that enter by direct penetration. *Plant Cell* **18**, 731–746, doi:10.1105/tpc.105.038372 (2006).

77. Yun, B. W. *et al.* Loss of actin cytoskeletal function and EDS1 activity, in combination, severely compromises non-host resistance in *Arabidopsis* against wheat powdery mildew. *Plant J.* **34**, 768–777 (2003).

78. Feys, B. J. *et al.* *Arabidopsis* SENESCENCE-ASSOCIATED GENE101 stabilizes and signals within an ENHANCED DISEASE SUSCEPTIBILITY1 complex in plant innate immunity. *Plant Cell* **17**, 2601–2613 (2005).

79. Collins, N. C. *et al.* SNARE-protein-mediated disease resistance at the plant cell wall. *Nature* **425**, 973–977 (2003).

80. Assaad, F. F. *et al.* The PEN1 syntaxin defines a novel cellular compartment upon fungal attack and is required for the timely assembly of papillae. *Mol. Biol. Cell* **15**, 5118–5129 (2004).

81. Bhat, R. A., Miklis, M., Schmelzer, E., Schulze-Lefert, P. & Panstruga, R. Recruitment and interaction dynamics of plant penetration resistance components in a plasma membrane microdomain. *Proc. Natl Acad. Sci. USA* **102**, 3135–3140 (2005).


82. Consonni, C. *et al.* Conserved requirement for a plant host cell protein in powdery mildew pathogenesis. *Nature Genet.* **38**, 716–720 (2006).

83. Kobayashi, D. Y., Tamaki, S. J. & Keen, N. T. Cloned avirulence genes from the tomato pathogen *Pseudomonas syringae pv. tomato* confer cultivar specificity on soybean. *Proc. Natl Acad. Sci. USA* **86**, 157–161 (1989).

84. Lorang, J. M., Shen, H., Kobayashi, D., Cooksey, D. & Keen, N. T. *avrA* and *avrE* in *Pseudomonas syringae pv. tomato* PT23 play a role in virulence on tomato plants. *Mol. Plant Microbe Interact.* **7**, 208–215 (1994).

85. Peyyala, R. & Farman, M. L. *Magnaporthe oryzae* isolates causing gray leaf spot of perennial ryegrass possess a functional copy of the AVR1-CO39 avirulence gene. *Mol. Plant Pathol.* **7**, 157–165 (2006).

86. Staal, J., Kaliff, M., Bohman, S. & Dixelius, C. Transgressive segregation reveals two *Arabidopsis* TIR-NB-LRR resistance genes effective against *Leptosphaeria maculans*, causal agent of blackleg disease. *Plant J.* **46**, 218–230 (2006).

87. Rohmer, L., Guttman, D. S. & Dangl, J. L. Diverse evolutionary mechanisms shape the type III effector virulence factor repertoire in the plant pathogen *Pseudomonas syringae*. *Genetics* **167**, 1341–1360 (2004).

88. Tsiamis, G. *et al.* Cultivar-specific avirulence and virulence functions assigned to *avrPphF* in *Pseudomonas syringae pv. phaseolicola*, the cause of bean halo-blight disease. *EMBO J.* **19**, 3204–3214 (2000).

89. Lawrence, G. L., Mayo, G. M. E. & Shepherd, K. W. Interactions between genes controlling pathogenicity in the flax rust fungus. *Phytopathol.* **71**, 12–19 (1981).

90. Bakker, E. G., Toomajian, C., Kreitman, M. & Bergelson, J. A. Genome-wide survey of *R* gene polymorphisms in *Arabidopsis*. *Plant Cell* **18** 1803–1818 (2006).

91. Kuang, H., Woo, S. S., Meyers, B. C., Nevo, E. & Michelmore, R. W. Multiple genetic processes result in heterogeneous rates of evolution within the major cluster disease resistance genes in lettuce. *Plant Cell* **16**, 2870–2894 (2004).

92. Van der Hoorn, R. A., De Wit, P. J. & Joosten, M. H. Balancing selection favors guarding resistance proteins. *Trends Plant Sci.* **7**, 67–71 (2002).

93. Thrall, P. H. & Burdon, J. J. Evolution of virulence in a plant host-pathogen metapopulation. *Science* **299**, 1735–1737 (2003).

94. Tian, D., Traw, M. B., Chen, J. Q., Kreitman, M. & Bergelson, J. Fitness costs of *R*-gene-mediated resistance in *Arabidopsis thaliana*. *Nature* **423**, 74–77 (2003).

95. Kemen, E. *et al.* Identification of a protein from rust fungi transferred from haustoria into infected plant cells. *Mol. Plant Microbe Interact.* **18**, 1130–1139 (2005).

96. Szurek, B., Marois, E., Bonas, U. & Van den Ackervecken, G. Eukaryotic features of the *Xanthomonas* type III effector AvrBs3: protein domains involved in transcriptional activation and the interaction with nuclear importin receptors from pepper. *Plant J.* **26**, 523–534 (2001).

97. Holt, B. F. III. *et al.* An evolutionarily conserved mediator of plant disease resistance gene function is required for normal *Arabidopsis* development. *Dev. Cell* **2**, 807–817 (2002).

98. Palma, K., Zhang, Y. & Li, X. An importin α homolog, MOS6, plays an important role in plant innate immunity. *Curr. Biol.* **15**, 1129–1135 (2005).

99. Zhang, Y. & Li, X. A putative nucleoporin 96 is required for both basal defense and constitutive resistance responses mediated by *suppressor of npr1-1, constitutive 1*. *Plant Cell* **17**, 1306–1316, doi: 10.1105/tpc.104.029926 (2005).

100. Shao, F. *et al.* Cleavage of *Arabidopsis* PBS1 by a bacterial type III effector. *Science* **301**, 1230–1233 (2003).

101. Mucyn, T. S. *et al.* The NB-ARC-LRR protein Prf interacts with Pto kinase *in vivo* to regulate specific plant immunity. *Plant Cell* (October 6, doi:10.1105/tpc.106.044016 2006).

102. Abramovitch, R. B. & Martin, G. B. AvrPtoB: a bacterial type III effector that both elicits and suppresses programmed cell death associated with plant immunity. *FEMS Microbiol. Lett.* **245**, 1–8 (2005).

103. Wu, A. J., Andriotis, V. M., Durrant, M. C. & Rathjen, J. P. A patch of surface-exposed residues mediates negative regulation of immune signaling by tomato Pto kinase. *Plant Cell* **16**, 2809–2821 (2004).

104. Rooney, H. C. *et al.* *Cladosporium* Avr2 inhibits tomato Rcr3 protease required for Cf-2-dependent disease resistance. *Science* **308**, 1783–1786 (2005).

Acknowledgements We thank S. Grant, D. Hubert, J. McDowell, M. Nishimura, R. Panstruga, J. Rathjen, P. Schulze-Lefert, S. Somerville and C. Zipfel for their critical comments, and our various colleagues for permission to cite their unpublished results. We apologize to the many colleagues whose work could not be discussed owing to strict page limits. Our labs are supported by grants from the NIH, NSF and DOE (JLD) and from the Gatsby Charitable Trust, the BBSRC and the EU (JDGJ).

Author Contributions JDGJ and JLD conceived and wrote the review together.

Author Information Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Correspondence should be addressed to J.J. (jonathan.jones@tsl.ac.uk) or J.L.D. (dangl@email.unc.edu).
