
REVIEW

The biosynthesis and regulation of bacterial prodiginines

Neil R. Williamson*§, Peter C. Fineran*§, Finian J. Leeper† and George P. C. Salmond*

Abstract | The red-pigmented prodiginines are bioactive secondary metabolites produced by both Gram-negative and Gram-positive bacteria. Recently, these tripyrrole molecules have received renewed attention owing to reported immunosuppressive and anticancer properties. The enzymes involved in the biosynthetic pathways for the production of two of these molecules, prodigiosin and undecylprodigiosin, are now known. However, the biochemistry of some of the reactions is still poorly understood. The physiology and regulation of prodigine production in Serratia and Streptomyces are now well understood, although the biological role of these pigments in the producer organisms remains unclear. However, research into the biology of pigment production will stimulate interest in the bioengineering of strains to synthesize useful prodigine derivatives.

Prodiginines are a family of tripyrrole red pigments which are attracting increasing interest because of their immunosuppressive and anticancer activities (BOX 1). Prodiginines have also been shown to have antifungal, antibacterial, antiprotozoal and antimalarial activities¹². The prodiginines are produced by Serratia spp., actinomycetes (for example, *Streptomyces coelicolor* A3(2)) and various marine bacteria, including *Hahella chejuensis* KCTC 2396 and *Pseudoalteromonas denitrificans*³⁴ (BOX 2).

Many strains of *Serratia marcescens* synthesize prodigiosin (2-methyl-3-pentyl-6-methoxyprodigiosine), and production of the pigment by this bacterium has a long and colourful history. For example, prodigiosin-producing *S*. *marcescens* has been used as a biological marker in germ warfare dispersal modelling and has been proposed to be responsible for the ‘miraculous’ phenomenon of ‘bleeding’ bread in religious ceremonies (TIMELINE). Also, prior to the development of synthetic colour chemistry, prodigiosin was produced commercially, albeit for a short time, for the dyeing of silk and wool⁵. The linear tripyrrole structure of prodigiosin was elucidated in the 1960s by partial and complete synthesis⁶⁷, and it is a classical secondary metabolite, only appearing in the later stages of bacterial growth⁸. Prodiginines have been examined as model bacterial secondary metabolites owing to their visible colour and the genetically amenable nature of some producer strains of *Serratia* and *Streptomyces* species.

There are four broad structural classes of prodiginines⁵⁹. The first group consists of prodigiosins with only straight chain alkyl substituents, such as prodigiosin and undecylprodigiosin. The other three groups contain cyclic derivatives of prodiginines. For example, the Gram-positive soil bacterium, *S*. *coelicolor* A3(2), produces a mixture of prodiginines, with a 2:1 ratio of undecylprodigiosin and butyl-meta-cycloheptylprodigine¹⁰.

As with many secondary metabolites, the true physiological role of prodiginines in the producer organisms is still debated. Furthermore, the precise cellular location of prodigiosin is unclear, because it has been detected in intracellular and extracellular fractions¹¹. However, early studies indicated that prodigiosin, a highly hydrophobic molecule, is possibly associated with the cell envelope²,¹² and, in particular, is present in a complex with a 100-kDa protein in the inner membrane¹³. In addition to their proposed antibacterial, antifungal or trypanolytic activities, prodiginines have been associated with surface adherence and enhancing bacterial dispersal²,¹⁴–¹⁶. It was also suggested that the biosynthesis of prodigiosins might function as a ‘metabolic sink’, by consuming the overflow of NAD(P)H¹⁷ or proline¹⁸ from primary metabolism. Unfortunately, most of the earlier studies assigning a function to prodigiosin were based on a comparison of non-pigmented with pigmented isolates (instead of isogenic strains) or have studied the effects of purified prodiginines. Therefore, other factors that are different between the strains, or the impact of non-physiological concentrations of the pigment, might have influenced the phenotypes that were ascribed to the prodiginines. With genetically defined prodigiosin-deficient mutants,

*Department of Biochemistry, Tennis Court Road, University of Cambridge, Cambridge, CB2 1QW, UK.
†Department of Chemistry, University of Cambridge, Lensfield Road, Cambridge, CB2 1EW, UK.
§These authors contributed equally to the review.
Correspondence to G.P.C.S. e-mail: gpcs@mole.bio.cam.ac.uk
doi:10.1038/nrmicro1531

NATURE REVIEWS | MICROBIOLOGY
© 2006 Nature Publishing Group
VOLUME 4 | DECEMBER 2006 | 887

REVIEWS

Box 1 | Anticancer and immunosuppressant agents

The potential of prodiginines as anticancer agents and immunosuppressants has led to an expansion of research in this area. Prodiginines are effective as immunomodulators at non-cytotoxic concentrations, whereas higher levels lead to anticancer activities by apoptosis, with little toxicity to normal cells (reviewed in REF. 96).

Prodigine derivatives display immunosuppressive effects on T-cells by blocking IL-2-dependent proliferation through downregulation of the IL-2Rα receptor<sup>97–100</sup>. In vivo, prodigiosin delayed the onset and progression of autoimmune diabetes and delayed arthritis progression in mouse models<sup>98,101</sup>. Furthermore, prodiginines improved animal survival in both rat heart transplant<sup>102</sup> and in mouse graft-versus-host disease<sup>101</sup> models.

The anticancer effect of prodiginines has been observed against numerous cancer cell lines, including liver, spleen, haematopoietic, colon, gastric, lung<sup>103</sup> and breast<sup>104</sup>, in primary cultures of chronic lymphocytic leukaemia and in a liver cancer xenograft in mice (REF. 96 and references therein). Recently, prodigiosin was demonstrated to efficiently inhibit tumour metastasis and migration in vitro and in vivo<sup>105</sup>.

Interestingly, there are multiple modes of action of prodiginines, and despite extensive efforts, the contribution of each of these mechanisms to the observed cellular activities of these molecules is unclear. Known activities of prodiginines include:

- Prodigines can reversibly disrupt the pH gradient between various cellular compartments (generated by ATPases) by functioning as H⁺/Cl⁻ symporters, resulting in the acidification of the cytoplasm and in some cases apoptosis (reviewed in REFS 96,106).
- Prodigines can intercalate DNA and bind Cu(II) to facilitate oxidative DNA cleavage (reviewed in REF. 106). An interesting recent finding demonstrated that prodigiosin stabilized topoisomerase I and II-DNA complexes and caused DNA fragmentation<sup>107</sup>.
- Cell-cycle arrest is induced by prodiginines at either late G₁ or the G₁-S transition, depending on the cell type and prodigine structure. The induction of cyclin-dependent kinase inhibitor p27, cyclin E, cyclin D, cyclin-dependent kinase-2 and cyclin-dependent kinase-4 and the phosphorylation of retinoblastoma protein was inhibited by undecylprodigiosin, leading to cell cycle arrest (REF. 96 and references therein).
- Prodigines can activate caspase induction in both mitochondrial-dependent (cytochrome c release) and mitochondrial-independent pathways and can also induce apoptosis independently of caspases through the mitochondrial apoptosis-inducing factor (AIF) route (REF. 96,103,104).
- Various mitogen-activated protein kinase (MAPK) pathways including extracellular signal-regulated kinase (ERK), stress-activated protein kinase (SAPK)/c-jun N-terminal kinase (JNK) and p38-MAPK are affected by prodiginines (REF. 96 and references therein). Recently, protein phosphatase inhibitory effects were ascribed to prodiginines<sup>108</sup>, pointing to a possible mechanism of interference with signal-transduction pathways.

It is apparent that these bacterial secondary metabolites hold promise for development as immunomodulatory and anticancer drugs. However, an improved understanding of their modes of action and further development of derivatives with improved pharmacological properties and reduced cytotoxicities is essential.

we are now in a good position to re-address the fundamental question of the role(s) of these pigments in the producing organisms.

This Review will focus on the regulation and biosynthesis of prodiginines in Gram-negative Serratia spp. and Gram-positive Streptomyces spp., highlighting the significant recent advances in these areas. This increased understanding has the potential to enable the design of bacterial systems for the enhanced production of a range of related bioactive compounds.

### Biosynthesis of the prodiginines

Early studies showed that the biosynthesis of prodigiosin proceeds by a bifurcated pathway culminating in the enzymic condensation of the terminal products of the two pathways, 2-methyl-3-n-amyl-pyrrole (MAP) and 4-methoxy-2,2′-bipyrrrole-5-carbaldehyde (MBC) to form prodigiosin<sup>2,19–21</sup>. It should be noted that MAP should actually be defined as 2-methyl-3-pentyl-pyrrole. However, historically n-amyl has been used and we have used this definition throughout the Review to be consistent with the literature. In S. coelicolor A3(2), it was shown that at least 18 closely linked genes were required for the biosynthesis of undecylprodigiosin<sup>22–27</sup>. Subsequently, the entire undecylprodigiosin biosynthetic (red) cluster was cloned and expressed in the heterologous host,

Streptomyces parvulus<sup>26</sup>. In addition, the prodigiosin biosynthetic gene (pig) clusters of Serratia sp. ATCC 39006 (Serratia 39006) and S. marcescens ATCC 274 (Sma 274) were cloned and expressed in the heterologous hosts Erwinia carotovora subsp. carotovora and *Escherichia coli*, respectively<sup>28,29</sup>. Sequencing of these three gene clusters identified genes that are common to all three clusters and genes that are specific to either the Serratia pig clusters or the Streptomyces red cluster<sup>28–30</sup>. The following sections will discuss our current understanding of the biosynthesis of the prodiginines by Serratia spp. and Streptomyces spp. Chemical strategies for the synthesis of prodiginines have been reviewed recently and will not be covered here<sup>31</sup>.

#### Gene organization of the prodigine biosynthetic clusters

Four prodigine biosynthetic gene clusters have now been identified, sequenced and expressed in heterologous hosts<sup>4,26,29,30</sup>. Genome sequencing of the marine γ-proteobacterium H. chejuensis KCTC 2396 identified the fourth prodigiosin biosynthetic gene cluster, in which the gene content and organization closely resembles that of the Serratia pig clusters<sup>4</sup> (FIG. 1). The red cluster of S. coelicolor A3(2) is the largest, consisting of 23 genes, two of which have a regulatory role (*redD* and *redZ*, discussed later). The red cluster is arranged as four

Box 2 | Examples of prodiginines and the producing bacteria

There are four broad structural classes of prodiginines ${ }^{5,9}$. Prodigiosin (methyl group at C-2 and a pentyl group at C-3 of pyrrole C) and undecylprodigiosin (undecyl side chain at position C-2) are linear tripyrroles. Butyl-meta-cycloheptylprodigine has a ring formed between positions 2 and 4 of pyrrole ring C. Cycloprodigiosin has a ring between positions 3 and 4 of pyrrole ring C and a methyl group at position C-2. Cyclononylprodigiosin is a macrocyclic prodigine with ring formation between position 2 of pyrrole C and position 10 of pyrrole A ${ }^{2,5}$.

| Prodiginine                  | Group | Source                                                                                   | Refs   |
|-----------------------------|-------|------------------------------------------------------------------------------------------|--------|
| Prodigiosin                 | 1     | Serratia 39006, Serratia marcescens, Serratia plymuthica, Hahella chejuensis, Pseudomonas magnesiourba, Vibrio psychroerythreus | 2,4,11, 101 |
| Undecylprodigiosin          | 1     | Streptomyces longisporus ruber, Streptoverticillium rubireticuli, Actinomadura madurae, Streptomyces coelicolor A3(2), Saccharopolyspora sp. nov. | 2,10, 102 |
| Butyl-meta-cyclo-<br>heptylprodigine | 2     | Saccharopolyspora sp. nov., Streptomyces coelicolor A3(2)                                      | 102    |
| Cycloprodigiosin            | 3     | Vibrio gazogenes, Alteromonas rubra, Pseudoalteromonas denitrificans                              | 3,5    |
| Cyclononylprodigiosin       | 4     | Actinomadura pelletieri, Actinomadura madurae                                               | 5      |

and the amino-terminal 146 amino-acid transmembrane domain of PigB. The adjacent gene, HCH_06024, encodes a protein with homology to the amine oxidase carboxy-terminal domain of PigB. Therefore, Serratia contains a multidomain protein, which in $H$. chejuensis is encoded by two separate genes (FIG. 1).

Each biosynthetic pathway contains proteins with similarity to other natural product syntheses, such as the adenylation domains and peptidyl carrier protein (PCP) domains of non-ribosomal peptide synthases (NRPS), and acyl carrier proteins (ACP) and ketosynthase domains of polyketide synthases (PKS). In the Serratia and $H$. chejuensis pig clusters, a degree of genomic modularity exists. The genes $pigB-pigE$ encode proteins sufficient for the biosynthesis of MAP and condensation with MBC to form prodigiosin ${ }^{11}$. Interestingly, prior to sequencing of the Serratia pig clusters, a $S$. marcescens genomic clone was isolated that was capable of the biosynthesis of MAP and condensation with MBC to form prodigiosin. This indicated that the genes responsible for the biosynthesis of MAP and the condensing enzyme were closely linked ${ }^{33}$. The remainder of the Serratia pig cluster, $pigF-pigN$, encodes proteins that are either directly or indirectly involved in the biosynthesis of MBC.

### Biosynthesis of MBC

Recently, every biosynthetic gene of the Serratia 39006 pig cluster was inactivated and the biosynthetic intermediates that accumulated in each mutant were analysed by liquid chromatography-mass spectrometry (LC-MS), cross-feeding (FIG. 2) and genetic complementation ${ }^{11}$. These analyses allowed specific Pig enzymes to be assigned to every step of the two convergent pathways in the biosynthesis of MAP, MBC and condensation to form prodigiosin (FIG. 3). A common pathway for the biosynthesis of MBC is proposed for Serratia spp., S. coelicolor A3(2) and $H$. chejuensis (FIGS 1, 3), because all of the Serratia proteins known to be involved in the biosynthesis of MBC have homologues in $H$. chejuensis and S. coelicolor A3(2). Furthermore, the same early precursors, proline, acetate, serine and S-adenosylmethionine are incorporated into the bipyrrrole in both S. coelicolor A3(2) and S. marcescens ${ }^{34,35}$ (FIG. 3).

The first steps in the biosynthesis of MBC have been shown to involve the incorporation of proline to form the pyrrole ring through a pyrrolyl-2-carboxyl-S-PCP intermediate catalysed by RedM, RedO and RedW ${ }^{11,29,30,36}$. Therefore, RedM, which shows sequence identity to the adenylation domains of modular NRPSs, activates L-proline using ATP and then transfers the L-prolyl group to the thiol group of the phosphopantetheine moiety of the PCP (RedO). RedW/PigA then oxidize the L-prolyl-S-PCP to pyrrolyl-2-carboxyl-S-PCP (FIG. 3). This mechanism for the incorporation of proline into a pyrrole is common to many pyrrole-containing compounds, including undecylprodigiosin ${ }^{30,36}$, prodigiosin ${ }^{11,29}$, pyoluteorin ${ }^{36}$, coumermycin A1, novobiocin and chlorobiocin ${ }^{37}$ (recently reviewed in REF. 38). 4'-Phosphopantetheinyltransferases (PPTases) catalyse the addition of a 4'-phosphopantetheinyl moiety to the conserved

transcriptional units whereas the other three clusters are arranged unidirectionally. The Serratia 39006 pig cluster is transcribed as a polycistronic mRNA from a promoter upstream of $pigA^{32}$. Interestingly, the Serratia 39006 pig operon contains an additional gene ($pigO$) at the 3' end of the cluster, with no homologue in the other prodiginine clusters. A $pigO$ mutant of Serratia 39006 still produces prodigiosin ${ }^{29}$. Another interesting difference is that, in $H$. chejuensis, the HCH_06023 gene encodes a 137 amino-acid protein with sequence similarity to RedS

REVIEW S

**Timeline | History of prodigiosine pigments**

| (501–600 bc) Pythagoras discovers 'bloody' coloration on food<sup>122</sup> | Miracle of Bolsena, Italy. 'Blood' on Eucharistic bread<sup>123</sup> | Red pigment of Bacillus prodigious (S. marcescens) investigated and named prodigiosin<sup>9</sup> | (1950–66) US army germ-warfare trials used S. marcescens in New York subways and the San Francisco Bay Area<sup>122</sup> | Early isotopic tracer studies used to investigate the biosynthesis of prodigiosin<sup>124</sup> | Confirmation of bifurcated pathway of MBC and MAP to make prodigiosin<sup>20</sup> | Red pigment in Streptomyces coelicolor A3(2) determined as undecylprodigiosin and related compounds<sup>10</sup> |
| --- | --- | --- | --- | --- | --- | --- |

| 501 bc | 332 bc | 1264 | 1817 | 1902 | 1906 | 1950 | 1960 | 1966 | 1973 | 1985 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Siege of Tyre. 'Bloody' bread predicts victory for Alexander the Great<sup>122</sup> | Serratia marcescens named after Italian physicist Serafino Serrati<sup>122</sup> | S. marcescens used for transmission experiments in UK House of Commons<sup>122</sup> | (1960–62) Structure of prodigiosin confirmed by chemical synthesis<sup>6,7</sup> | Prodigiosin precursors identified by incorporation studies<sup>34</sup> |

serine residue of the PCP, converting the enzyme from its inactive (apo) to its active (holo) form. All four clusters contain genes predicted to encode PPTases (PigL, RedU and HCH_06035), which might phosphopantetheinylate the relevant ACP/PCP domain. In Sma 274, a second PPTase involved in the biosynthesis of prodigiosin, PspW, has been identified. Mutants defective in *pspW*, although unable to produce prodigiosin and the surfactant serrawettin, were still capable of making MAP and condensing it with MBC to form prodigiosin<sup>39</sup>. In *Serratia*, it seems that two PPTases are probably required for the biosynthesis of MBC, one to activate the PCP domain of PigG and the second to phosphopantetheinylate the ACP domains of PigH. Recently, it has been shown, by deletion of the gene, that RedU is required for activation of the ACP RedO but not for the activation of the other ACP/PCP domains<sup>40</sup>.

The next step involves a polyketide-type chain extension of the pyrrolyl-2-carboxyl-S-PCP intermediate. We have proposed that the pyrrole-2-carboxyl unit is transferred from PigG/RedO to the active site cysteine of PigJ/RedX, followed by the transfer of a malonyl group

---

**Serratia sp. ATCC 39006**

orfy → pigA → pigB → pigC → pigD → pigE → pigF → pigG → pigH → pigI → pigJ → pigK → pigL → pigM → pigN → pigO → orfZ

---

**Serratia marcescens ATCC 274**

cueR → pigA → pigB → pigC → pigD → pigE → pigF → pigG → pigH → pigI → pigJ → pigK → pigL → pigM → pigN → copA

---

**Hahella chejuensis KCTC 2396**

06021 → 06023 → 06024 → 06026 → 06027 → 06028 → 06029 → 06030 → 06031 → 06032 → 06033 → 06034 → 06035 → 06036 → 06037

---

**Streptomyces coelicolor A3(2)**

redD → redX → redW → redY → redZ → redV → redU → redT → redS → redQ → redO → redN → redM → redL → redK → redJ → redI → redH → redG → redF → redE

---

**Figure 1 | Comparison of prodigiosine biosynthetic gene clusters.** The genetic organization of the four sequenced prodigiosine biosynthetic gene clusters of *Serratia* 39006, *Sma* 274, *Hahella chejuensis* and *Streptomyces coelicolor* A3(2). Dark blue block arrows show genes that encode proteins required for the biosynthesis of MBC. Light blue block arrows show genes that encode proteins required for the biosynthesis of the monopyrroles. The gene encoding the condensing enzyme is *red* and *redG*, which encodes the enzyme proposed to cyclize undecylprodigiosin to butyl-meta-cycloheptylprodigiosine, is coloured purple. Genes that are not involved in the biosynthesis of prodiginines are yellow. The regulatory genes *redD* and *redZ* are shown in green. *HCH_06023*, *redS* and the amino terminus of *pigB* are shaded grey to show sequence similarity. The black lines link the *H. chejuensis* and *Serratia* genes with their respective *red* homologues. Only the carboxy-terminal aminotransferase domain of RedL shows sequence similarity with PigE. The *red* cluster is organized as four transcriptional units shown by the black arrows at the bottom.

Lead compound, prodigiosine derivative (PNU156804), developed with improved immunosuppressive properties<sup>97</sup>

The red gene cluster was sequenced and a biosynthetic mechanism proposed<sup>30</sup>

Prodigiosin in pre-clinical trials for the treatment of pancreatic cancer (see AIDA website in Further information)

GX15-070, a synthetic prodigiosin derivative, in phase I/II clinical trials as an anticancer treatment<sup>125</sup>

Undecylprodigiosin (red) gene cluster from S. coelicolor A3(2) cloned<sup>26</sup>

Prodigiosin (pig) gene cluster cloned from Serratia 39006 (Ref. 28)

(2004-05) The pig genes were sequenced and biosynthesis of prodigiosin determined<sup>11,29</sup>

---

can accept an exogenous pyrrolyl-2-carboxyl thioester, bypassing RedM and RedW. Inactivation of *pigM* results in the accumulation of HBM, indicating that PigM is involved in the oxidation of the alcohol group of HBM to give 4-hydroxy-2,2′-bipyrrole-5-carbaldehyde (HBC)<sup>11</sup> (FIG. 3). Finally, the hydroxyl group of HBC is methylated by PigF/PigN resulting in MBC formation. PigF/RedI are predicted to be S-adenosylmethionine-dependent methyltransferases. Norprodigiosin, a product of the condensation of MAP with HBC, accumulates in a *pigN* mutant, indicating a role for PigN in the methylation step. However, the precise role of PigN/RedF in facilitating the methylation of HBC is currently unknown. The predicted requirement of two proteins for the methylation step supports the work of Feiltson and co-workers who showed that the gene products of two classes of *red* mutation (*redE* and *redF*) were necessary for a functional O-methyltransferase complex which was probably bound to the cell membrane<sup>23,24</sup>. Currently, no role is assigned to PigK/RedY. A Δ*pigK* mutant produces prodigiosin but can also cross-feed mutants affected in the early steps of MBC biosynthesis. This indicates that PigK might function in the biosynthesis of MBC.

**Biosynthesis of the monopyrroles**

In contrast to the biosynthesis of MBC, in which the assembly pathway is highly conserved between prodigiosin and undecylprodigiosin, the biosynthetic routes to the monopyrroles are completely different. In fact, none of the proteins involved in the biosynthesis of MAP in *Serratia* have close homologues in the *red* cluster (FIG. 1). This observation might be explained by the need to incorporate alternative starter units or to construct slightly different monopyrroles.

**Biosynthesis of MAP.** Recently, a novel pathway for the biosynthesis of MAP, requiring only three of the *Serratia* Pig enzymes, *PigB*, *PigD* and *PigE*, was determined<sup>11</sup> (FIG. 3). Proteins showing sequence similarity with PigB, PigD and PigE are present in *H. chejuensis*. Therefore,

![Figure 2](https://i.imgur.com/yourimageurl.png)

Figure 2 | Examples of cross-feeding. The first panel (from left to right) shows reciprocal cross-feeding between strains blocked in the biosynthesis of 4-methoxy-2,2′-bipyrrole-5-carbaldehyde (MBC) and 2-methyl-3-n-amyl-pyrrole (MAP). The second panel shows that the terminal products (MAP and MBC) accumulating in the Δ*pigC* mutant (*pigC* encodes the condensing enzyme PigC) can cross-feed mutants in both the MAP and MBC pathways. *Escherichia coli* ER2566 pNRW55 (pGEX-2T-*pigC*) is streaked on the left and can condense MAP and MBC to form prodigiosin. The final panel shows that a Δ*pigB* mutant, blocked later in the biosynthesis of MAP, can cross-feed a Δ*pigD* mutant blocked earlier in the biosynthesis of the monopyrrole.

REVIEW S

Biosynthesis of MAP in Serratia sp.

![Diagram](attachment:diagram1.png)

Common pathway to the biosynthesis of MBC

![Diagram](attachment:diagram2.png)

Biosynthesis of 2-undecylpyrrole in S. coelicolor A3(2)

![Diagram](attachment:diagram3.png)

Figure 3 | The bifurcated biosynthetic pathway of the prodiginines. The common pathway to the biosynthesis of 4-methoxy-2,2′-bipyrrole-5-carbaldehyde (MBC) in *Serratia* sp., *Hahella chejuensis* and *Streptomyces coelicolor* A3(2) and the different routes to the biosynthesis of the monopyrroles are shown. Terminal products are shown in bold type. AdoMet, S-adenosylmethionine; HBC, 4-hydroxy-2,2′-bipyrrole-5-carbaldehyde; HBM, 4-hydroxy-2,2′-bipyrrole-5-methanol; MAP, 2-methyl-3-n-amyl-pyrrole; PLP, pyridoxal phosphate.

it is predicted that the biosynthesis of the monopyrrole in *H. chejuensis* is analogous to the biosynthesis of MAP in *Serratia* spp. The initial precursor for the biosynthesis of MAP, 2-octenal, might be generated by fatty acid biosynthesis enzymes or might be derived from the autoxidation of unsaturated fatty acids. PigD has homology with thiamine pyrophosphate (TPP)-requiring enzymes, and is proposed to catalyse the transfer of a two-carbon fragment from pyruvate (with the loss of CO₂) to 2-octenal, yielding 3-acetyloctanal¹¹. Interestingly, Goldschmidt and Williams (1968) showed that the addition of thiamine (the precursor of TPP) induces monopyrrole biosynthesis²¹. Therefore, it seems plausible that this stimulatory effect of thiamine results from the TPP requirement of PigD. Next, PigE transaminates the aldehyde group of 3-acetyloctanal, resulting in an aminoketone, which spontaneously cyclizes to form H₂MAP¹¹. The final step in the biosynthesis of MAP is catalysed by PigB, which is predicted to contain three transmembrane domains anchoring the enzyme in the inner membrane. PigB catalyses the 2-electron oxidation of H₂MAP to MAP. Interestingly, the terminal enzyme in the biosynthesis of MBC, **PigN**, is also predicted to be localized in the inner membrane¹¹. Therefore, it seems likely that the terminal steps, at least in precursor assembly through the parallel pathways, sequester MAP and MBC to the cytoplasmic membrane.

**Biosynthesis of 2-undecylpyrrole.** Much of our current understanding of the biosynthesis of 2-undecylpyrrole is based on the predicted functions of the Red proteins¹¹,³⁰. It has been shown that, although not essential, RedP is important for the biosynthesis of undecylprodigiosin⁴². A *redP* deletion mutant produced reduced levels of undecylprodigiosin, which could be restored by supplying a copy of FabH (KASIII) in
(trans). Interestingly, the Δ*redP* mutant produced two new prodiginines, 10-methylundecylprodigine and 11-methyldodecylprodigine, consistent with 3-methylbutyryl-CoA and isobutyryl CoA as new starter units<sup>42</sup>. RedP is proposed to condense an acetyl-CoA starter unit with a malonyl-RedQ (ACP), which would be reduced by type II fatty acid biosynthesis enzymes to give butyryl-RedQ. RedR catalyses four subsequent elongation steps with malonyl-RedQ. The resulting dodecyl group is then transferred to the ACP domain of RedL, in which a ketosynthase reaction with a malonyl group attached to the second ACP of RedL would generate a β-ketomyristoyl thioester. The pyridoxal phosphate (PLP) domain of RedL would condense glycine with the bound intermediate, releasing CO₂. Release of this intermediate and cyclization would give 4-keto-2-undecyl-4,5-dihydropyrrole<sup>11</sup> (FIG. 3). The final step in the biosynthesis of 2-undecylpyrrole is the reduction and dehydration of 4-keto-2-undecyl-4, 5-dihydropyrrole by RedK.

### Monopyrrole and MBC condensation

The final step in the biosynthesis of prodigiosin and undecylprodigiosin is the condensation of the monopyrroles with MBC to form the tripyrrole pigments. This condensation reaction can occur chemically at low pH<sup>31</sup>. However, in bacteria, the condensation of the monopyrrole and MBC is catalysed by *PigC* and its homologues<sup>11</sup> (FIGS 2, 3). PigC and its homologues, RedH and HCH_06026 have N- and C-terminal domains with sequence similarity to the ATP-binding domain and phosphoryl transfer domains of a pyruvate phosphate dikinase<sup>11,29,30</sup>. All four proteins share a central domain containing the core catalytic region predicted to bind MBC and the corresponding monopyrrole to catalyse the formation of the prodigidine. The condensing enzyme has been shown to have relaxed specificity, being able to incorporate an analogue of MAP, 2,4-dimethyl-3-ethylpyrrole (DEP) and HBC to form 2,4-dimethyl-3-ethyl-prodigine and norprodigiosin, respectively<sup>11,19</sup>. Also the analogue of MBC with a phenyl ring in place of pyrrole ring A is reported to restore pigment production to undecylprodigiosin biosynthetic mutants of *S.* coelicolor A3(2) (REF. 10). The relaxed specificity of PigC might have biotechnological uses. Following the condensation reaction, the oxidative cyclization of undecylprodigiosin to butyl-meta-cycloheptylprodigiosin in *S.* coelicolor A3(2) is predicted to be catalysed by RedG<sup>30</sup>.

### Regulation of prodigidine biosynthesis

The complex and exquisite regulation of prodigidine synthesis integrates both physiological and environmental cues. Here, we outline some of the mechanisms that underpin the coordination of pigment production.

#### Small molecules

Both *Serratia* and *Streptomyces* spp. produce several small-molecule signals that have a role in the regulation of prodigidine synthesis.

#### Extracellular signalling (quorum sensing)

A significant input into prodigidine production is quorum sensing (QS); a mechanism used to coordinate bacterial behaviour in response to population density<sup>43</sup>. Chemical signals produced by the bacteria accumulate as cell density increases, until a threshold concentration is reached, at which time these molecules elicit changes in target gene expression.

In *Serratia* 39006, *smaI* and *smaR* (secondary metabolite activator) constitute the *N*-acyl-homoserine-lactone (*N*-AHL) QS locus<sup>28</sup>. SmaI synthesizes two signalling molecules, *N*-butanoyl-L-homoserine lactone (C4-HSL) and *N*-hexanoyl-L-homoserine lactone (C6-HSL)<sup>28</sup>. At low cell densities, SmaR represses the transcription of the *pig* cluster, at least in part by repressing the expression of the pigment regulators *pigR*, *rap* and *pigQ*. As the cell density increases, levels of C4-HSL/C6-HSL accumulate and inhibit the DNA binding activity of SmaR (derepression), resulting in the production of PigQ, PigR and Rap, and the activation of pigment biosynthesis<sup>32,44</sup> (FIG. 4). In *S.* marcescens SS-1 (*Sma* SS-1), the biosynthesis of prodigiosin is also dependent on an *N*-AHL QS system, SpnIR<sup>45</sup>. In a mechanism similar to that used by *Serratia* 39006, SpnR represses prodigiosin production in the absence of *N*-AHLs (*N*-3-oxo-hexanoyl-L-homoserine lactone (3-oxo-C6-HSL) and C6-HSL), which are produced by SpnI<sup>45</sup>. A proposed interspecies QS signalling system, AI-2 (LuxS)<sup>46</sup>, also regulates prodigiosin biosynthesis in some *Serratia* strains<sup>47</sup>. In *Sma* 274, LuxS synthesizes the AI-2 signal, which is required for maximum pigmentation<sup>47</sup>.

Many streptomycetes produce other signalling molecules (*γ*-butyrolactones) that accumulate late in growth and control secondary metabolism<sup>48</sup>. *S.* coelicolor A3(2) requires *scbA* and *scbR* for the production of several *γ*-butyrolactones, and the addition of one of these, SCB1 ((2*R*,3*R*,1′*R*)-2-(1′-hydroxy-6′-methylheptyl)-3-hydroxymethyl-butanolide), induced precocious undecylprodigiosin synthesis<sup>49,50</sup>. Intriguingly, deletion of *scbA* and *scbR* results in elevated and reduced undecylprodigiosin synthesis, respectively<sup>50</sup>. *ScbR* is one of four putative *γ*-butyrolactone-binding proteins in *S.* coelicolor A3(2), including *CprA* (coelicolor pigment regulator). It is unclear how ScbR regulates undecylprodigiosin synthesis. However, ScbR is a DNA binding protein, the binding capacity of which is decreased significantly in the presence of SCB1 (REF. 50). CprA activates undecylprodigiosin production but this might be the result of limited precursor availability caused by the overproduction of actinorhodin in the *cprA* mutant<sup>51</sup>.

#### ppGpp and the stringent response

In *S.* coelicolor A3(2), the stringent response, requiring the *relA* gene and its product guanosine tetraphosphate (ppGpp), are involved in the growth-rate control of undecylprodigiosin biosynthesis. *S.* coelicolor A3(2) starved for nitrogen requires RelA for the synthesis of ppGpp and the subsequent expression of the pathway-specific regulator *redD* (discussed below) and undecylprodigiosin production<sup>52,53</sup>. The conditional requirement for *relA* can be bypassed by Pᵢ limitation or by overexpression of *rshA* (RelA/SpoT homologue)<sup>54</sup>. However, phosphate and RshA mediate the restoration of pigment in *relA* mutants without the restoration of ppGpp levels<sup>54</sup>.
Mutations predicted to inhibit RelA-ribosome interactions abolish both ppGpp and undecylprodigiosin synthesis^{55–57}. ppGpp exerts an effect on transcription by binding to RNAP^{58}. In support of the role of ppGpp/RelA in secondary metabolism, mutations in the principal sigma factor (σ^HrdB) or in *rpoB* (RNA polymerase β-subunit) result in decreased and increased undecylprodigiosin production, respectively. The effect of the σ^HrdB mutation on pigment production is through modulation of *redD* transcription and might function by preventing the interaction of ppGpp-bound RNAP with σ^HrdB (REF. 59). Furthermore, it is proposed that the mutations in *rpoB* mimic the ppGpp-bound form of RNAP, resulting in elevated levels of the regulators, RedZ and RedD^{60,61}.

### Transcriptional regulators

Production of undecylprodigiosin in S. coelicolor A3(2) is under positive control by the pathway-specific activator RedD, a founding member of the SARP (streptomyces antibiotic regulatory protein) family of DNA-binding proteins^{62,63}. RedZ, another pathway specific-response regulator, activates the transcription of *redD*^{64} presumably by binding DNA without the requirement for phosphorylation^{65}. RedZ negatively autoregulates^{64} and is also under translational control by BldA which codes for a tRNA for a rare leucine codon (UUA)^{64,66}. In addition to the pathway-specific activators, there are numerous other regulatory inputs into prodiginine synthesis in S. coelicolor A3(2) (FIG. 5).

Serratia 39006 does not have pathway-specific activators comparable to RedD or RedZ. However, there are multiple transcriptional inputs into prodigiosine production, including PigP, a major regulatory protein. PigP contains a helix-turn-helix (HTH) domain and activates prodigiosin production by modulating the expression of six other prodigiosin regulators (*pigQ*, *pigR*, *pigS*, *pigV*, *pigX* and *rap*) and *pigA-O* transcription^{44}. Three of these genes are under the joint control of PigP and the QS system (*pigQ*, *pigR* and *rap*) and therefore represent ‘nodes’ in which multiple signals might be integrated and channelled into secondary metabolism (FIG. 4).

Rap (regulator of antibiotic and pigment) contains a predicted winged helix (WH) HTH DNA-binding motif and activates the transcription of *pigA-O* and therefore prodigiosin biosynthesis^{28,32}. Potentially, Rap promotes transcription by a similar mechanism to its homologue RovA (regulator of virulence) from Yersinia spp. by contacting RNA polymerase (RNAP) and/or by antirepression of the global repressor H-NS (heat-stable nucleoid-structuring protein; REF. 67 and references therein).

PigR, a predicted class IV adenylate cyclase in Serratia 39006, activates prodigiosin production, presumably through the synthesis of cyclic adenosine 3′, 5′-monophosphate (cAMP)^{44}. In *E. coli*, cAMP, in a complex with the cAMP receptor protein (CRP), exerts its control over a large gene set^{68}. In fact, the *pigA* promoter contains a putative CRP-binding site, indicating that the cAMP-CRP complex, might regulate prodigiosin production in Serratia^{69}.

![Diagram](attachment:diagram.png)

**Figure 4 | Summary of the genetic regulation of prodigiosin production in Serratia species.** Summary of the genetic regulation of prodigiosin (PIG) production (from the *pigA-O* biosynthetic cluster) in Serratia 39006 and Sma 274. Activation is represented by triangular arrow-heads, repression is shown by flat arrow-heads. Details of individual pathways are described in the text.

Figure 5 | Summary of the genetic regulation of undecylprodigiosin production in Streptomyces coelicolor A3(2). Activation is represented by triangular arrow-heads, repression is shown by flat arrow-heads. Details of individual pathways are described in the text except for AbsB, a homologue of Escherichia coli RNase III<sup>82,111,112</sup>, BldC, a DNA-binding protein<sup>113</sup>, σ<sup>B</sup>, an alternative sigma factor involved in osmoprotection<sup>114</sup>, SpaA, a predicted protein involved in degradation of homoserine lactones<sup>115</sup>, AmfC, important for the timing of undecylprodigiosin production<sup>116</sup>, BrgA (benzamide-derivative resistance gene) involved in ADP-ribosylation<sup>117</sup>, AbaA (antibiotic biosynthesis activator)<sup>118</sup>, AbaB<sup>119</sup>, PtpA, (a phosphotyrosine protein phosphatase)<sup>120</sup> and cAMP synthesized by the adenylate cyclase Cya<sup>121</sup>. The role of the PhoR/P, Ppk and AbaB systems in S. coelicolor is inferred based on results obtained in Streptomyces lividans<sup>86,88,119</sup>.

The intracellular secondary messenger, bis-(3′-5′)-cyclic dimeric guanosine monophosphate (c-di-GMP), controls numerous bacterial phenotypes<sup>70</sup>. In Serratia 39006, PigX represses the transcription of *pigA-O* and prodigiosin biosynthesis<sup>44</sup>. PigX is a predicted inner-membrane signalling protein with GGDEF and EAL domains, which might possess c-di-GMP-specific diguanylate cyclase and phosphodiesterase activities respectively. Therefore, it is hypothesized that intracellular c-di-GMP levels modulate the production of prodigiosin. The mechanism used by another predicted inner membrane regulatory protein, PigV, to activate prodigiosin production is unknown but it is plausible that it is involved in sensing one or more environmental signals<sup>44</sup>.

In Serratia 39006, PigS, a predicted DNA binding protein of the ArsR/SmtB-family, positively controls levels of the *pigA-O* transcript<sup>44</sup>. In Serratia spp., LysR-type transcriptional regulators repress pigmentation. These include PigU from Serratia 39006 (REF. 71) and HexS from Sma 274, both of which are homologues of HexA (hyperproduction of exoenzymes) from *E. carotovora*. Furthermore, HexS directly binds to the *pigA* promoter and presumably inhibits transcription of the biosynthetic genes<sup>72</sup>.

Signal transduction and two-component systems

A regulatory theme common to both Serratia spp. and Streptomyces spp. is the use of two-component and signal-transduction systems to relay extracellular environmental cues into intracellular prodiginine production. GacA/S two-component phosphorelay systems are widespread in Gram-negative bacteria and control a range of phenotypes<sup>73</sup>. In Serratia 39006 the sensor kinase PigW (GacS homologue), is predicted to respond to an unknown physiological signal(s), resulting in the phosphorylation of the response regulator PigQ (GacA homologue). Phosphorylated PigQ then activates prodigiosin production by promoting the transcription of *pigA-O*<sup>44</sup>. Commonly, GacA-type proteins function by activating untranslated RNA molecules<sup>74</sup> that antagonize (by sequestration) homologues of the post-transcriptional regulator CsrA (carbon-storage regulator) from *E. coli*<sup>75</sup>. CsrA homologues are small proteins that bind to mRNAs, alter transcript stability and affect translation. In *Sma CH-1* and *Sma SS-1* the CsrA homologue, RsmA (repressor of secondary metabolism) represses prodigiosin production<sup>76</sup>.

At least three two-component systems and a eukaryotic-like signal-transduction system regulate undecylprodigiosin production in streptomycetes. AfsQ1/Q2 (A-factor synthesis)<sup>77</sup> and EcrA1/A2 (expression coordinated with *red*)<sup>78,79</sup> activate undecylprodigiosin biosynthesis by unknown mechanisms. The *absA1/A2* genes are present in the biosynthetic cluster for another antibiotic in *S. coelicolor* A3(2), CDA (calcium-dependent antibiotic)<sup>80</sup>. The sensor

REVIEW S

kinase, AbsA1, has kinase and phosphatase activities towards the response regulator, AbsA2 (REF. 81). Phosphorylated AbsA2 represses undecylprodigiosin production by affecting the expression of *redD*<sup>82,83</sup>.

The eukaryotic-like signal transduction system (reviewed in REF. 84) contains at least three membrane-associated serine/threonine kinases (AfsK, PkaG and AfsL), which autophosphorylate in response to unknown signals and phosphorylate AfsR<sup>85</sup>. The phosphotransfer from AfsK to AfsR is inhibited by the AfsK-binding protein, KbpA. Phosphorylated AfsR binds the *afsS* promoter and activates transcription by a DNA bending mechanism. AfsR possesses an ATPase domain that might assist in the conversion of a closed to an open promoter complex, leading to transcriptional activation. AfsS is similar to σ factors and activates expression of *redD* which enhances the production of undecylprodigiosin. In addition, AfsR controls the production of γ-butyrolactones.

**Environmental cues**

Like many secondary metabolites, the biosynthesis of prodiginines is dependent on growth phase, with maximal expression occurring under conditions of nutrient deprivation or stress, such as when cells enter stationary phase. Prodiginine synthesis responds to multiple environmental and physiological cues, which can include temperature, oxygen supply, pH, light and ionic strength<sup>2,5</sup>. In addition to variations in carbon and nitrogen source, the availability of inorganic phosphate (*P*<sub>i</sub>), salt, detergent and various cations and anions all affect prodiginine biosynthesis<sup>2,5</sup>. It is clear that bacteria synthesize prodiginines only when conditions are appropriate, but why is the production of these secondary metabolites so exquisitely environmentally sensitive? The molecular mechanisms underpinning a few of these associations have been elucidated and are outlined below.

**Genetic mechanisms of environmental sensing.** In *Serratia* 39006, *P*<sub>i</sub> availability, through the Pst transporter and the PhoB/R two-component system, regulates prodigiosin in both QS-independent and QS-dependent pathways<sup>32</sup>. Under conditions of limited *P*<sub>i</sub>, PhoR phosphorylates PhoB in response to signals relayed by the PstSCABPhoU–*P*<sub>i</sub> transport complex. Phosphorylated PhoB then activates the transcription of *smaI* and *pigA–O*, presumably by binding to Pho-boxes in the promoters. In an interesting parallel, the synthesis of undecylprodigiosin is repressed by phosphate, and the PhoR/P two-component system in *Streptomyces lividans*<sup>86,87</sup>. However, in *phoRP* mutants, intracellular *P*<sub>i</sub> is reduced and *P*<sub>i</sub> still represses pigment production, which might indicate that internal *P*<sub>i</sub> affects undecylprodigiosin biosynthesis<sup>86</sup>. Indeed, inactivation of polyphosphate kinase (*ppk*), which would lead to reduced internal *P*<sub>i</sub> under conditions of limited phosphate, results in increased antibiotic production and *redD* transcription<sup>88</sup>. NaCl also dramatically increases the transcript levels of *redD* and undecylprodigiosin production<sup>89</sup>.

Another environmental ‘sensor’ in *Serratia* 39006 is PigT, a GntR homologue that activates prodigiosin production by binding a PigT-box in the *pigA* promoter. In the presence of gluconate the DNA binding activity of PigT is thought to be alleviated, resulting in a loss of *pigA–O* activation and the biosynthesis of pigment<sup>69</sup>.

It is apparent that in the Gram-positive and Gram-negative prodiginine producers studied, numerous physiological inputs are ‘sensed’ and complex genetic networks are used to integrate these signals into the biosynthesis of these bioactive secondary metabolites (FIGS 4,5). The conservation of the Pho systems seems to have resulted in similar systems that mediate *P*<sub>i</sub> repression of prodigine in both *Serratia* spp. and *Streptomyces* spp.<sup>32,86</sup>. Conversely, there is also considerable variation in the modes of prodigine control, which indicates there might be selection of both independent, and convergent (for example, alternative extracellular signalling systems), regulatory scenarios that enable the production of these bioactive secondary metabolites under the appropriate environmental conditions. It is important to stress that regulation of prodigine production has been studied in just a few strains of *Serratia* and *Streptomyces* species. Therefore, it is possible that there might be significant regulatory variation between strains in the same genera — as already demonstrated by comparing *Serratia* 39006, *Sma* SS-1 and *Sma* 274.

**Horizontal gene transfer and prodiginines**

In *S.* *marcescens*, the *pig* cluster is always flanked by homologues of *cueR* and *copA*. When the *pig* cluster is absent, *cueR–copA* are adjacent and divergently transcribed<sup>29</sup>. CueR is the transcriptional regulator of CopA, a copper-transporting P-type ATPase efflux pump involved in copper homeostasis<sup>90,91</sup>. This observation indicated not only that there is a link between copper homeostasis and prodigiosin production but that the *pig* cluster might have been acquired by horizontal gene transfer into the *cueR/copA* locus in some strains of *S.* *marcescens*<sup>29</sup>. Indeed, in *Sma* 274, copper repressed prodigiosin production and a *copA* mutant overproduced prodigiosin in the presence of copper<sup>91</sup>. Furthermore, we recently demonstrated a phage-mediated horizontal gene transfer event of the complete *pig* cluster into the chromosome of a non-pigmented clinical strain (*S.* *marcescens* strain 12 (*Sma* 12)), which had an intact *cueR/copA* genetic locus. Surprisingly, not only was *Sma* 12 capable of expressing prodigiosin from the ‘foreign’ operon, but its regulation was brought under the control of the native QS system<sup>92</sup>. We then used a bacteriophage to transfer the *smaIR* locus from *Sma* 12 into the chromosome of *Sma* 274 which lacks *SmaIR* homologues. Acquisition of *smaIR* brought the biosynthesis of prodigiosin in *Sma* 274 under the control of the foreign QS locus. This showed that QS signalling modules can be horizontally mobilized between strains by transduction<sup>92</sup>. In addition some *Serratia* spp. have QS loci flanked by remnants of transposable elements<sup>28,45</sup>. In *Sma* SS-1, *spnIR* is part of a functional transposon, named Tn*TIR*<sup>93</sup>, which could transpose (by plasmid intermediate / conjugation) into the *luxIR*-negative *S.* *marcescens* CH-1 (*Sma* CH-1).

Therefore, regulatory and biosynthetic modules involved in prodigiosin production can be horizontally transferred and immediately incorporated into the existing genetic circuitry of the recipient cell, demonstrating the adaptive plasticity of bacterial secondary metabolism.

**Conclusions and future directions**

As is the case for most secondary metabolites, the true physiological and ecological functions of prodiginines are unknown. Nevertheless, the prodiginines hold some promise as new immunosuppressant and anticancer agents (BOX 1). A major hurdle in the field of secondary metabolism is the experimental determination of the physiological functions of these compounds. A recent model to explain the evolution of secondary metabolism indicates that, similar to combinatorial chemistry, organisms must generate molecular diversity in order to evolve compounds with specific biomolecular activity<sup>94</sup>. The presence of secondary metabolism *per se* might increase opportunities for enhanced fitness, although many compounds that are synthesized might not actually benefit the producers. Nevertheless, the retention of biosynthetic clusters that encode enzymes with relaxed specificity, capable of generating chemical diversity (biosynthetic plasticity), can allow evolution under environmental pressures<sup>94,95</sup>. Some of the prodigine biosynthetic enzymes show flexibility in substrate use and product formation. It has recently been shown that replacing RedP with FabH generated two new prodigine derivatives<sup>42</sup>. Furthermore, the condensing enzyme is known to be promiscuous, generating novel prodiginines by incorporating analogues of MAP and MBC<sup>11,19</sup>. There is now a good understanding of how each pyrrole is constructed and how the unusual pyrryldipyrrromethene structure of prodigiosin is assembled. We need a better understanding of the catalytic mechanisms and substrate ranges of the novel prodigine biosynthetic enzymes, and an increased knowledge of the complex regulation systems involved. This might enable biotechnological exploitation to generate derivatives with biological or synthetic uses, and might also help to explain the factors impinging on the evolution of this group of secondary metabolites.

In this Review, we have highlighted the significant recent advances that have increased our understanding of more than thirty genes involved in the biosynthesis and regulation of the prodigine secondary metabolites in both Gram-negative *Serratia* spp. and Gram-positive streptomycetes. With such a genetic and physiological investment in complex regulation and biosynthesis systems, we presume that these tripyrrole pigments provide an ecological advantage to the producer hosts. Unfortunately, the physiological role of these bioactive molecules still remains elusive, at least to scientists!

---

1. Demain, A. L. in *Fifty Years of Antimicrobials: Past Perspectives and Future Trends* (eds Hunter, P. A., Darby, G. K. & Russell, N. J.) 205–228 (Society for General Microbiology, Cambridge, 1995).
2. Williams, R. P. & Quadri, S. M. in *The Genus Serratia* (eds Von Graevenitz, A. & Rubin, S. J.) 31–75 (Boca Raton: CRC Press Inc, 1980).
3. Kawauchi, K. *et al.* A possible immunosuppressant, cycloprodigiosin hydrochloride, obtained from *Pseudoalteromonas denitrificans*. *Biochem. Biophys. Res. Commun.* **237**, 543–547 (1997).
4. Jeong, H. *et al.* Genomic blueprint of *Hahella chejuensis*, a marine microbe producing an algicidal agent. *Nucleic Acids Res.* **33**, 7066–7073 (2005).
5. Bennett, J. W. & Bentley, R. Seeing red: the story of prodigiosin. *Adv. Appl. Microbiol.* **47**, 1–32 (2000).
6. Wasserman, H. H., McKeon, J. E. & Smith, L. Prodigiosin. Structure and partial synthesis. *J. Am. Chem. Soc.* **82**, 506–507 (1960).
7. Rapoport, H. & Holden, H. G. The synthesis of prodigiosin. *J. Am. Chem. Soc.* **84**, 635–642 (1962).
8. Williams, R. P. Biosynthesis of prodigiosin, a secondary metabolite of *Serratia marcescens*. *Appl. Microbiol.* **25**, 396–402 (1973). This paper demonstrated that prodigiosin was a secondary metabolite and was one of many important papers on the biosynthesis of prodigiosin by this author.
9. Gerber, N. N. Prodigiosin-like pigments. *CRC Crit. Rev. Microbiol.* **3**, 469–485 (1975).
10. Tsao, S. W., Rudd, B. A., He, X. G., Chang, C. J. & Floss, H. G. Identification of a red pigment from *Streptomyces coelicolor* A3(2) as a mixture of prodigiosin derivatives. *J. Antibiot. (Tokyo)* **38**, 128–131 (1985).
11. Williamson, N. R. *et al.* Biosynthesis of the red antibiotic, prodigiosin, in *Serratia*: identification of a novel 2-methyl-3-n-amyl-pyrrole (MAP) assembly pathway, definition of the terminal condensing enzyme, and implications for undecylprodigiosin biosynthesis in *Streptomyces*. *Mol. Microbiol.* **56**, 971–989 (2005). This paper described a detailed genetic investigation into the biosynthesis of prodigiosin. Based on LC-MS analyses and cross-feeding of the accumulating intermediates, revised pathways for the biosynthesis of prodigiosin and undecylprodigiosin were proposed.
12. Purkayastha, M. & Williams, R. P. Association of pigment with the cell envelope of *Serratia marcescens* (*Chromobacterium prodigiosum*). *Nature* **187**, 349–350 (1960).
13. Kobayashi, N. & Ichikawa, Y. A protein associated with prodigiosin formation in *Serratia marcescens*. *Microbiol. Immunol.* **33**, 257–263 (1989).
14. Syzdek, L. D. Influence of *Serratia marcescens* pigmentation on cell concentrations in aerosols produced by bursting bubbles. *Appl. Environ. Microbiol.* **49**, 173–178 (1985).
15. Burger, S. R. & Bennett, J. W. Droplet enrichment factors of pigmented and nonpigmented *Serratia marcescens*: possible selective function for prodigiosin. *Appl. Environ. Microbiol.* **50**, 487–490 (1985).
16. Azambuja, P., Feder, D. & Garcia, E. S. Isolation of *Serratia marcescens* in the midgut of *Rhodnius prolixus*: impact on the establishment of the parasite *Trypanosoma cruzi* in the vector. *Exp. Parasitol.* **107**, 89–96 (2004).
17. Trutko, S. & Akimenko, V. The role of prodigiosin biosynthesis in the regulation of oxidative metabolism of the producer *Serratia marcescens*. *Mikrobiologiia* **58**, 723–729 (1989).
18. Hood, D. W., Heidstra, R., Swoboda, U. K. & Hodgson, D. A. Molecular genetic analysis of proline and tryptophan biosynthesis in *Streptomyces coelicolor* A3(2): interaction between primary and secondary metabolism — a review. *Gene* **115**, 5–12 (1992).
19. Williams, R. P., Goldschmidt, M. E. & Gott, C. L. Inhibition by temperature of the terminal step in biosynthesis of prodigiosin. *Biochem. Biophys. Res. Commun.* **19**, 177–181 (1965).
20. Morrison, D. A. Prodigiosin synthesis in mutants of *Serratia marcescens*. *J. Bacteriol.* **131**, 1599–1604 (1966).
21. Goldschmidt, M. C. & Williams, R. P. Thiamine-induced formation of the monopyrrole moiety of prodigiosin. *J. Bacteriol.* **116**, 609–616 (1968).
22. Feitelson, J. S., Sinha, A. M. & Coco, E. A. Molecular genetics of Red biosynthesis in *Streptomyces*. *J. Nat. Prod.* **49**, 988–994 (1986).
23. Feitelson, J. S., Malpartida, F. & Hopwood, D. A. Genetic and biochemical characterization of the red gene cluster of *Streptomyces coelicolor* A3(2). *J. Gen. Microbiol.* **131**, 2431–2441 (1985).
24. Feitelson, J. S. & Hopwood, D. A. Cloning of a *Streptomyces* gene for an O-methyltransferase involved in antibiotic biosynthesis. *Mol. Gen. Genet.* **190**, 394–398 (1983).
25. Rudd, B. A. & Hopwood, D. A. A pigmented mycelial antibiotic in *Streptomyces coelicolor*: control by a chromosomal gene cluster. *J. Gen. Microbiol.* **119**, 333–340 (1980).
26. Malpartida, F., Niemi, J., Navarrete, R. & Hopwood, D. A. Cloning and expression in a heterologous host of the complete set of genes for biosynthesis of the *Streptomyces coelicolor* antibiotic undecylprodigiosin. *Gene* **93**, 91–99 (1990).
27. Coco, E. A., Narva, K. E. & Feitelson, J. S. New classes of *Streptomyces coelicolor* A3(2) mutants blocked in undecylprodigiosin (Red) biosynthesis. *Mol. Gen. Genet.* **227**, 28–32 (1991).
28. Thomson, N. R., Crow, M. A. A., McGowan, S. J., Cox, A. & Salmond, G. P. Biosynthesis of carbapenem antibiotic and prodigiosin pigment in *Serratia* is under quorum sensing control. *Mol. Microbiol.* **36**, 539–556 (2000). This paper reported the cloning and heterologous expression of the *Serratia* 39006 prodigiosin biosynthetic genes and revealed their regulation by an *N*-AHL quorum sensing system.
29. Harris, A. K. *et al.* The *Serratia* gene cluster encoding biosynthesis of the red antibiotic, prodigiosin, shows species- and strain-dependent genome context variation. *Microbiology* **150**, 3547–3560 (2004). This paper described the sequencing of two *Serratia* prodigiosin biosynthetic gene clusters.
30. Cerdeño, A. M., Bibb, M. J. & Challis, G. L. Analysis of the prodigine biosynthesis gene cluster of *Streptomyces coelicolor* A3(2): new mechanisms for chain initiation and termination in modular multienzymes. *Chem. Biol.* **8**, 817–829 (2001). This paper described a detailed sequence analysis of the undecylprodigiosin (red) biosynthetic gene cluster.
31. Fürstner, A. Chemistry and biology of roseophilin and the prodigiosin alkaloids: a survey of the last 2500 years. *Angew. Chem. Int. Ed.* **42**, 3582–3603 (2003).

NATURE REVIEWS | MICROBIOLOGY

© 2006 Nature Publishing Group
VOLUME 4 | DECEMBER 2006 | 897

REVIEW S

32. Slater, H., Crow, M., Everson, L. & Salmond, G. P. Phosphate availability regulates biosynthesis of two antibiotics, prodigiosin and carbapenem, in *Serratia* via both quorum-sensing-dependent and -independent pathways. *Mol. Microbiol.* **47**, 303–320 (2003).

33. Dauenhauer, S. A., Hull, R. A. & Williams, R. P. Cloning and expression in *Escherichia coli* of *Serratia marcescens* genes encoding prodigiosin biosynthesis. *J. Bacteriol.* **158**, 1128–1132 (1984).

34. Wasserman, H. H. *et al.* Biosynthesis of prodigiosin. Incorporation patterns of C-labeled alanine, proline, glycine, and serine elucidated by fourier transform nuclear magnetic resonance. *J. Am. Chem. Soc.* **95**, 6874–6875 (1973).

35. Qadri, S. M. & Williams, R. P. Role of methionine in biosynthesis of prodigiosin by *Serratia marcescens*. *J. Bacteriol.* **116**, 1191–1198 (1973).

36. Thomas, M. G., Burkart, M. D. & Walsh, C. T. Conversion of L-proline to pyrrolyl-2-carboxyl-S-PCP during undecylprodigiosin and pyoluteorin biosynthesis. *Chem. Biol.* **9**, 171–184 (2002). This paper proved biochemically that the incorporation of L-proline to pyrrolyl-2-carboxyl-S-PCP requires 3 Red proteins (RedO, RedM & RedW). Therefore, this paper described a novel mechanism in pyrrole biosynthesis for the incorporation of L-proline.

37. Xu, H., Wang, Z. X., Schmidt, J., Heide, L. & Li, S. M. Genetic analysis of the biosynthesis of the pyrrole and carbamoyl moieties of coumermycin A1 and novobiocin. *Mol. Genet. Genomics* **268**, 387–396 (2002).

38. Walsh, C. T., Garneau-Tsodikova, S. & Howard-Jones, A. R. Biological formation of pyroles: Nature’s logic and enzymatic machinery. *Nat. Prod. Rep.* **23**, 517–531 (2006).

39. Sunaga, S., Li, H., Sato, Y., Nakagawa, Y. & Matsuyama, T. Identification and characterization of the *pswP* gene required for the parallel production of prodigiosin and serrawettin W1 in *Serratia marcescens*. *Microbiol. Immunol.* **48**, 723–728 (2004).

40. Stanley, A. E., Walton, L. J., Kourid Zerikly, M., Corre, C. & Challis, G. L. Elucidation of the *Streptomyces coelicolor* pathway to 4-methoxy-2, 2-bipyrrole-5-carboxaldehyde, an intermediate in prodiginine biosynthesis. *Chem. Commun.* 3981–3983 (2006). This paper demonstrated the involvement of RedM, RedW, RedX and RedN in the biosynthesis of MBC in *S. coelicolor* confirming the previously proposed MBC biosynthesis pathway.

41. Garneau-Tsodikova, S., Dorrestein, P. C., Kelleher, N. L. & Walsh, C. T. Protein assembly line components in prodigiosin biosynthesis: Characterization of PigA, G, H, I, J. *Am. Chem. Soc.* **128**, 12600–12601 (2006). *In vitro* characterization of PigA, PigG, PigH, PigI and PigJ confirming their proposed roles in the biosynthesis of MBC in *Serratia*.

42. Mo, S., Kim, B. S. & Reynolds, K. A. Production of branched-chain alkylprodigines in *S. coelicolor* by replacement of the 3-ketoacyl ACP synthase III initiation enzyme, RedP. *Chem. Biol.* **12**, 191–200 (2005).

43. Whitehead, N. A., Barnard, A. M., Slater, H., Simpson, N. J. & Salmond, G. P. Quorum-sensing in Gram-negative bacteria. *FEMS Microbiol. Rev.* **25**, 365–404 (2001).

44. Fineran, P. C., Slater, H., Everson, L., Hughes, K. & Salmond, G. P. Biosynthesis of tripyrrole and β-lactam secondary metabolites in *Serratia*: integration of quorum sensing with multiple new regulatory components in the control of prodigiosin and carbapenem antibiotic production. *Mol. Microbiol.* **56**, 1495–1517 (2005). This paper described a complex regulatory network, including quorum sensing, involved in the control of prodigiosin biosynthesis in *Serratia* 39006.

45. Horng, Y. T. *et al.* The LuxR family protein SpnR functions as a negative regulator of N-acylhomoserine lactone-dependent quorum sensing in *Serratia marcescens*. *Mol. Microbiol.* **45**, 1655–1671 (2002).

46. Vendeville, A., Winzer, K., Heurlier, K., Tang, C. M. & Hardie, K. R. Making ‘sense’ of metabolism: autoinducer-2, LuxS and pathogenic bacteria. *Nature Rev. Microbiol.* **3**, 383–396 (2005).

47. Coulthurst, S. J., Kurz, C. L. & Salmond, G. P. *luxS* mutants of *Serratia* defective in autoinducer-2-dependent ‘quorum sensing’ show strain-dependent impacts on virulence and production of carbapenem and prodigiosin. *Microbiology* **150**, 1901–1910 (2004).

48. Horinouchi, S. & Beppu, T. A-factor as a microbial hormone that controls cellular differentiation and secondary metabolism in *Streptomyces griseus*. *Mol. Microbiol.* **12**, 859–864 (1994).

49. Takano, E. *et al.* Purification and structural determination of SCB1, a γ-butyrolactone that elicits antibiotic production in *Streptomyces coelicolor* A3(2). *J. Biol. Chem.* **275**, 11010–11016 (2000).

50. Takano, E., Chakraburty, R., Nihira, T., Yamada, Y. & Bibb, M. J. A complex role for the γ-butyrolactone SCB1 in regulating antibiotic production in *Streptomyces coelicolor* A3(2). *Mol. Microbiol.* **41**, 1015–1028 (2001). This paper reported a γ-butyrolactone signalling system required for the production of undecylprodigiosin in *S. coelicolor* A3(2).

51. Onaka, H., Nakagawa, T. & Horinouchi, S. Involvement of two A-factor receptor homologues in *Streptomyces coelicolor* A3(2) in the regulation of secondary metabolism and morphogenesis. *Mol. Microbiol.* **28**, 743–753 (1998).

52. Chakraburty, R. & Bibb, M. The ppGpp synthetase gene (*relA*) of *Streptomyces coelicolor* A3(2) plays a conditional role in antibiotic production and morphological differentiation. *J. Bacteriol.* **179**, 5854–5861 (1997).

53. Martinez-Costa, O. H. *et al.* A *relA/spoT* homologous gene from *Streptomyces coelicolor* A3(2) controls antibiotic biosynthetic genes. *J. Biol. Chem.* **271**, 10627–10634 (1996).

54. Sun, J., Hesketh, A. & Bibb, M. Functional analysis of *relA* and *rshA*, two *relA/spoT* homologues of *Streptomyces coelicolor* A3(2). *J. Bacteriol.* **183**, 3488–3498 (2001).

55. Ochi, K., Zhang, D., Kawamoto, S. & Hesketh, A. Molecular and functional analysis of the ribosomal L11 and S12 protein genes (*rplK* and *rpsL*) of *Streptomyces coelicolor* A3(2). *Mol. Gen. Genet.* **256**, 488–498 (1997).

56. Ochi, K. A relaxed (*rel*) mutant of *Streptomyces coelicolor* A3(2) with a missing ribosomal protein lacks the ability to accumulate ppGpp, A-factor and prodigiosin. *J. Gen. Microbiol.* **136**, 2405–2412 (1990).

57. Shima, J., Hesketh, A., Okamoto, S., Kawamoto, S. & Ochi, K. Induction of actinorhodin production by *rpsL* (encoding ribosomal protein S12) mutations that confer streptomycin resistance in *Streptomyces lividans* and *Streptomyces coelicolor* A3(2). *J. Bacteriol.* **178**, 7276–7284 (1996).

58. Magnusson, L. U., Farewell, A. & Nyström, T. ppGpp: a global regulator in *Escherichia coli*. *Trends Microbiol.* **13**, 236–242 (2005).

59. Aigle, B., Wietzorrek, A., Takano, E. & Bibb, M. J. A single amino acid substitution in region 1.2 of the principal sigma factor of *Streptomyces coelicolor* A3(2) results in pleiotropic loss of antibiotic production. *Mol. Microbiol.* **37**, 995–1004 (2000).

60. Hu, H., Zhang, Q. & Ochi, K. Activation of antibiotic biosynthesis by specified mutations in the *rpoB* gene (encoding the RNA polymerase beta subunit) of *Streptomyces lividans*. *J. Bacteriol.* **184**, 3984–3991 (2002).

61. Bibb, M. J. Regulation of secondary metabolism in streptomycetes. *Curr. Opin. Microbiol.* **8**, 208–215 (2005).

62. Takano, E. *et al.* Transcriptional regulation of the *redD* transcriptional activator gene accounts for growth-phase-dependent production of the antibiotic undecylprodigiosin in *Streptomyces coelicolor* A3(2). *Mol. Microbiol.* **6**, 2797–2804 (1992).

63. Narva, K. E. & Feitelson, J. S. Nucleotide sequence and transcriptional analysis of the *redD* locus of *Streptomyces coelicolor* A3(2). *J. Bacteriol.* **172**, 326–333 (1990).

64. White, J. & Bibb, M. *bldA* dependence of undecylprodigiosin production in *Streptomyces coelicolor* A3(2) involves a pathway-specific regulatory cascade. *J. Bacteriol.* **179**, 627–633 (1997).

65. Guthrie, E. P. *et al.* A response-regulator-like activator of antibiotic synthesis from *Streptomyces coelicolor* A3(2) with an amino-terminal domain that lacks a phosphorylation pocket. *Microbiology* **144**, 727–738 (1998).

66. Guthrie, E. P. & Chater, K. F. The level of a transcript required for production of a *Streptomyces coelicolor* antibiotic is conditionally dependent on a tRNA gene. *J. Bacteriol.* **172**, 6189–6193 (1990).

67. Ellison, D. W. & Miller, V. L. Regulation of virulence by members of the MarR/SlyA family. *Curr. Opin. Microbiol.* **9**, 153–159 (2006).

68. Botsford, J. L. & Harman, J. G. Cyclic AMP in prokaryotes. *Microbiol. Rev.* **56**, 100–122 (1992).

69. Fineran, P. C., Everson, L., Slater, H. & Salmond, G. P. A GntR family transcriptional regulator (PigT) controls gluconate-mediated repression and defines a new, independent pathway for regulation of the tripyrrole antibiotic, prodigiosin, in *Serratia*. *Microbiology* **151**, 3833–3845 (2005).

70. Romling, U., Gomelsky, M. & Galperin, M. Y. C-di-GMP: the dawning of a novel bacterial signalling system. *Mol. Microbiol.* **57**, 629–639 (2005).

71. Harris, S. J., Shih, Y. L., Bentley, S. D. & Salmond, G. P. The hexA gene of *Erwinia carotovora* encodes a LysR homologue and regulates motility and the expression of multiple virulence determinants. *Mol. Microbiol.* **28**, 705–717 (1998).

72. Tanikawa, T., Nakagawa, Y. & Matsuyama, T. Transcriptional downregulator HexS controlling prodigiosin and serrawettin W1 biosynthesis in *Serratia marcescens*. *Microbiol. Immunol.* **50**, 587–596 (2006).

73. Heeb, S. & Haas, D. Regulatory roles of the GacS/GacA two-component system in plant-associated and other Gram-negative bacteria. *Mol. Plant Microbe Interact.* **14**, 1351–1363 (2001).

74. Suzuki, K. *et al.* Regulatory circuitry of the CsrA/CsrB and BarA/UvrY systems of *Escherichia coli*. *J. Bacteriol.* **184**, 5130–5140 (2002).

75. Romeo, T. Global regulation by the small RNA-binding protein CsrA and the non-coding RNA molecule CsrB. *Mol. Microbiol.* **29**, 1321–1330 (1998).

76. Ang, S. *et al.* The role of RsmA in the regulation of swarming motility in *Serratia marcescens*. *J. Biomed. Sci.* **8**, 160–169 (2001).

77. Ishizuka, H., Horinouchi, S., Kieser, H. M., Hopwood, D. A. & Beppu, T. A putative two-component regulatory system involved in secondary metabolism in *Streptomyces* spp. *J. Bacteriol.* **174**, 7585–7594 (1992).

78. Li, Y. Q., Chen, P. L., Chen, S. F., Wu, D. & Zheng, J. A pair of two-component regulatory genes *ecrA1/A2* in *S. coelicolor*. *J. Zhejiang Univ. Sci.* **5**, 173–179 (2004).

79. Huang, J., Lih, C. J., Pan, K. H. & Cohen, S. N. Global analysis of growth phase responsive gene expression and regulation of antibiotic biosynthetic pathways in *Streptomyces coelicolor* using DNA microarrays. *Genes Dev.* **15**, 3183–3192 (2001). This paper discusses the global transcriptomic analysis of antibiotic biosynthesis in *S. coelicolor*. Biosynthetic loci were confirmed by cluster analysis and both previously characterized and novel undecylprodigiosin regulators were identified.

80. Anderson, T. B., Brian, P. & Champness, W. C. Genetic and transcriptional analysis of *absA*, an antibiotic gene cluster-linked two-component system that regulates multiple antibiotics in *Streptomyces coelicolor*. *Mol. Microbiol.* **39**, 553–566 (2001).

81. Sheeler, N. L., MacMillan, S. V. & Nodwell, J. R. Biochemical activities of the *absA* two-component system of *Streptomyces coelicolor*. *J. Bacteriol.* **187**, 687–696 (2005).

82. Aceti, D. J. & Champness, W. C. Transcriptional regulation of *Streptomyces coelicolor* pathway-specific antibiotic regulators by the *absA* and *absB* loci. *J. Bacteriol.* **180**, 3100–3106 (1998).

83. Adamidis, T., Riggle, P. & Champness, W. Mutations in a new *Streptomyces coelicolor* locus which globally block antibiotic biosynthesis but not sporulation. *J. Bacteriol.* **172**, 2962–2969 (1990).

84. Horinouchi, S. AfsR as an integrator of signals that are sensed by multiple serine/threonine kinases in *Streptomyces coelicolor* A3(2). *J. Ind. Microbiol. Biotechnol.* **30**, 462–467 (2003). This review summarized research on the *S. coelicolor* AfsR undecylprodigiosin regulator. This system was the first example of serine/threonine kinases in bacteria.

85. Sawai, R., Suzuki, A., Takano, Y., Lee, P. C. & Horinouchi, S. Phosphorylation of AfsR by multiple serine/threonine kinases in *Streptomyces coelicolor* A3(2). *Gene* **334**, 53–61 (2004).

86. Sola-Landa, A., Moura, R. S. & Martin, J. F. The two-component PhoR-PhoP system controls both primary metabolism and secondary metabolite biosynthesis in *Streptomyces lividans*. *Proc. Natl Acad. Sci. USA* **100**, 6133–6138 (2003).

87. Martin, J. F. Phosphate control of the biosynthesis of antibiotics and other secondary metabolites is mediated by the PhoR-PhoP system: an unfinished story. *J. Bacteriol.* **186**, 5197–5201 (2004).

88. Chouayekh, H. & Virole, M. J. The polyphosphate kinase plays a negative role in the control of antibiotic production in *Streptomyces lividans*. *Mol. Microbiol.* **43**, 919–930 (2002).

89. Sevcikova, B. & Kormanec, J. Differential production of two antibiotics of *Streptomyces coelicolor* A3(2), actinorhodin and undecylprodigiosin, upon salt stress conditions. *Arch. Microbiol.* **181**, 384–389 (2004).

90. Stoyanov, J. V., Hobman, J. L. & Brown, N. L. CueR (YbbI) of *Escherichia coli* is a MerR family regulator controlling expression of the copper exporter CopA. *Mol. Microbiol.* **39**, 502–511 (2001).

91. Williamson, N. R., Simonsen, H. T., Harris, A. K., Leeper, F. J. & Salmond, G. P. Disruption of the copper efflux pump (CopA) of *Serratia marcescens* ATCC 274 pleiotropically affects copper sensitivity and production of the tripyrrole secondary metabolite, prodigiosin. *J. Ind. Microbiol. Biotechnol.* **33**, 151–158 (2006).

92. Coulthurst, S. J., Williamson, N. R., Harris, A. K., Spring, D. R. & Salmond, G. P. Metabolic and regulatory engineering of *Serratia marcescens*: mimicking phage-mediated horizontal acquisition of antibiotic biosynthesis and quorum-sensing capacities. *Microbiology* **152**, 1899–1911 (2006). The horizontally mobile modular nature of the prodigiosin biosynthetic loci and their quorum sensing regulators was demonstrated in *S. marcescens*, highlighting the adaptive flexibility of bacterial secondary metabolism.

93. Wei, J. R. *et al.* A mobile quorum-sensing system in *Serratia marcescens*. *J. Bacteriol.* **188**, 1518–1525 (2006).

94. Firn, R. D. & Jones, C. G. The evolution of secondary metabolism — a unifying model. *Mol. Microbiol.* **37**, 989–994 (2000).

95. Demain, A. L. Pharmacologically active secondary metabolites of microorganisms. *Appl. Microbiol. Biotechnol.* **52**, 455–463 (1999).

96. Pérez-Tomás, R., Montaner, B., Llagostera, E. & Soto-Cerrato, V. The prodigiosins, proapoptotic drugs with anticancer properties. *Biochem. Pharmacol.* **66**, 1447–1452 (2003).

This review provided an overview of a large body of research on the pro-apoptotic and anticancer properties of prodigiosins.

97. Mortellaro, A. *et al.* New immunosuppressive drug PNU156804 blocks IL-2-dependent proliferation and NF-kappa B and AP-1 activation. *J. Immunol.* **162**, 7102–7109 (1999).

98. Han, S. B. *et al.* Prodigiosin blocks T cell activation by inhibiting interleukin-2Ralpha expression and delays progression of autoimmune diabetes and collagen-induced arthritis. *J. Pharmacol. Exp. Ther.* **299**, 415–425 (2001).

99. D’Alessio, R. *et al.* Synthesis and immunosuppressive activity of novel prodigiosin derivatives. *J. Med. Chem.* **43**, 2557–2565 (2000).

100. Han, S. B. *et al.* T-cell specific immunosuppression by prodigiosin isolated from *Serratia marcescens*. *Int. J. Immunopharmacol.* **20**, 1–13 (1998).

101. Han, S. B. *et al.* Effective prevention of lethal acute graft-versus-host disease by combined immunosuppressive therapy with prodigiosin and cyclosporine A. *Biochem. Pharmacol.* **70**, 1518–1526 (2005).

102. Stepkowski, S. M. *et al.* Selective inhibitor of Janus tyrosine kinase 3, PNU156804, prolongs allograft survival and acts synergistically with cyclosporine but additively with rapamycin. *Blood* **99**, 680–689 (2002).

103. Llagostera, E., Soto-Cerrato, V., Montaner, B. & Pérez-Tomás, R. Prodigiosin induces apoptosis by acting on mitochondria in human lung cancer cells. *Ann. NY Acad. Sci.* **1010**, 178–181 (2003).

104. Soto-Cerrato, V., Llagostera, E., Montaner, B., Scheffer, G. L. & Pérez-Tomás, R. Mitochondria-mediated apoptosis operating irrespective of multidrug resistance in breast cancer cells by the anticancer agent prodigiosin. *Biochem. Pharmacol.* **68**, 1345–1352 (2004).

105. Zhang, J., Shen, Y., Liu, J. & Wei, D. Antimetastatic effect of prodigiosin through inhibition of tumor invasion. *Biochem. Pharmacol.* **69**, 407–414 (2005).

106. Manderville, R. A. Synthesis, proton-affinity and anti-cancer properties of the prodigiosin-group natural products. *Curr. Med. Chem. Anti-Canc. Agents.* **1**, 195–218 (2001).

107. Montaner, B. *et al.* DNA interaction and dual topoisomerase I and II inhibition properties of the anti-tumor drug prodigiosin. *Toxicol. Sci.* **85**, 870–879 (2005).

108. Fürstner, A., Reinecke, K., Prinz, H. & Waldmann, H. The core structures of roseophilin and the prodigiosin alkaloids define a new class of protein tyrosine phosphatase inhibitors. *Chembiochem.* **5**, 1575–1579 (2004).

109. Berg, G. Diversity of antifungal and plant-associated *Serratia plymuthica* strains. *J. Appl. Microbiol.* **88**, 952–960 (2000).

110. Liu, R., Cui, C. B., Duan, L., Gu, Q. Q. & Zhu, W. M. Potent *in vitro* anticancer activity of metacycloprodigiosin and undecylprodigiosin from a sponge-derived actinomycete *Saccharopolyspora* sp. nov. *Arch. Pharm. Res.* **28**, 1341–1344 (2005).

111. Adamidis, T. & Champness, W. Genetic analysis of *absB*, a *Streptomyces coelicolor* locus involved in global antibiotic regulation. *J. Bacteriol.* **174**, 4622–4628 (1992).

112. Champness, W., Riggle, P., Adamidis, T. & Vandervere, P. Identification of *Streptomyces coelicolor* genes involved in regulation of antibiotic synthesis. *Gene* **115**, 55–60 (1992).

113. Hunt, A. C., Servin-Gonzalez, L., Kelemen, G. H. & Buttner, M. J. The *bldC* developmental locus of *Streptomyces coelicolor* encodes a member of a family of small DNA-binding proteins related to the DNA-binding domains of the MerR family. *J. Bacteriol.* **187**, 716–728 (2005).

114. Cho, Y. H., Lee, E. J., Ahn, B. E. & Roe, J. H. SigB, an RNA polymerase sigma factor required for osmoprotection and proper differentiation of *Streptomyces coelicolor*. *Mol. Microbiol.* **42**, 205–214 (2001).

115. Schneider, D., Bruton, C. J. & Chater, K. F. Characterization of *spaA*, a *Streptomyces coelicolor* gene homologous to a gene involved in sensing starvation in *Escherichia coli*. *Gene* **177**, 243–251 (1996).

116. Yonekawa, T., Ohnishi, Y. & Horinouchi, S. Involvement of *amfC* in physiological and morphological development in *Streptomyces coelicolor* A3(2). *Microbiology* **145**, 2273–2280 (1999).

117. Shima, J., Penyige, A. & Ochi, K. Changes in patterns of ADP-ribosylated proteins during differentiation of *Streptomyces coelicolor* A3(2) and its development mutants. *J. Bacteriol.* **178**, 3785–3790 (1996).

118. Fernandez-Moreno, M. A. *et al.* *abaA*, a new pleiotropic regulatory locus for antibiotic production in *Streptomyces coelicolor*. *J. Bacteriol.* **174**, 2958–2967 (1992).

119. Scheu, A. K., Martinez, E., Soliveri, J. & Malpartida, F. *abaB*, a putative regulator for secondary metabolism in *Streptomyces*. *FEMS Microbiol. Lett.* **147**, 29–36 (1997).

120. Umeyama, T., Tanabe, Y., Aigle, B. D. & Horinouchi, S. Expression of the *Streptomyces coelicolor* A3(2) *ptpA* gene encoding a phosphotyrosine protein phosphatase leads to overproduction of secondary metabolites in *S. lividans*. *FEMS Microbiol. Lett.* **144**, 177–184 (1996).

121. Süsstrunk, U., Pidoux, J., Taubert, S., Ullmann, A. & Thompson, C. J. Pleiotropic effects of cAMP on germination, antibiotic biosynthesis and morphological development in *Streptomyces coelicolor*. *Mol. Microbiol.* **30**, 33–46 (1998).

122. Yu, V. L. *Serratia marcescens*: historical perspective and clinical review. *N. Engl. J. Med.* **300**, 887–893 (1979).

123. Cullen, J. C. in ASM News 187–191 (1994).

124. Hubbard, R. & Rimington, C. The biosynthesis of prodigiosin, the tripyrrolylmethene pigment from *Bacillus prodigiosus* (*Serratia marcescens*). *Biochem. J.* **46**, 220–225 (1950).

125. Dairi, K., Tripathy, S., Attardo, G. & Lavallee, J. F. Two-step synthesis of the bipyrrrole precursor of prodigiosins. *Tetrahedron Lett.* **47**, 2605–2606 (2006).

**Acknowledgements**

Work in the authors’ laboratories is generously funded by the Biotechnology and Biological Sciences Research Council, UK. P.F.’s Ph.D. research on prodigiosin was supported by a Bright Futures Top Achiever Doctoral Scholarship from the tertiary education commission of New Zealand.

**Competing interests statement**

The authors declare no competing financial interests.

**DATABASES**

The following terms in this article are linked online to:

Entrez Genome Project: http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=genomeprj

Escherichia coli | *Hahella chejuensis* KCTC 2396 | *Streptomyces coelicolor* A3(2)

UniProtKB: http://ca.expasy.org/sprot CprA|PigA|PigB|PigC|PigD|PigE|PigN|RedD|RedZ|ScbR

**FURTHER INFORMATION**

George P. C. Salmond’s homepage: http://www.bio.cam.ac.uk/~salmond

Aida pharmaceuticals: http://aidapharma.server286.com/index.html

Access to this links box is available online.
