
# The receptors and cells for mammalian taste

Jayaram Chandrashekar¹, Mark A. Hoon², Nicholas J. P. Ryba² & Charles S. Zuker¹

The emerging picture of taste coding at the periphery is one of elegant simplicity. Contrary to what was generally believed, it is now clear that distinct cell types expressing unique receptors are tuned to detect each of the five basic tastes: sweet, sour, bitter, salty and umami. Importantly, receptor cells for each taste quality function as dedicated sensors wired to elicit stereotypic responses.

Our sensory systems are responsible for generating an internal representation of the outside world, including its chemical (taste and olfaction) and physical (mechanical, sound, vision and temperature) features. In this review we examine recent advances in our understanding of the biology of taste, focusing on receptors, cells and the logic of taste coding at the periphery.

Taste is in charge of evaluating the nutritious content of food and preventing the ingestion of toxic substances. Sweet taste permits the identification of energy-rich nutrients, umami allows the recognition of amino acids, salt taste ensures the proper dietary electrolyte balance, and sour and bitter warn against the intake of potentially noxious and/or poisonous chemicals. In humans, taste has the additional value of contributing to the overall pleasure and enjoyment of a meal. Surprisingly, although we can taste a vast array of chemical entities, it is now generally accepted that, qualitatively, they evoke few distinct taste sensations: sweet, bitter, sour, salty and savoury (or umami). Although this repertoire may seem modest, it has satisfactorily accommodated the evolutionary need for an effective and reliable platform to help recognize and distinguish key dietary components.

The anatomical substrates and units of taste detection are taste-receptor cells (TRCs; Fig. 1). TRCs are assembled into taste buds, which are distributed across different papillae of the tongue and palate epithelium. How are the different tastes detected, and how is taste quality represented? In the simplest scenario, sweet, bitter, sour, salty and umami tastants would each be recognized by different cells expressing specialized receptors. Coding at the periphery could then rely on straightforward labelled lines (that is, independent sweet, bitter, sour, salty and umami signals) to transform tastant quality into neural signals (Fig. 2a). In an alternative view, and the prevailing model for the past two decades¹⁻³, TRCs were proposed to be broadly tuned across taste modalities (Fig. 2b, c). In this case, it would be expected that individual TRCs would express different families of taste receptors, and that tastant recognition would result from decoding of the combined activity of various classes of such broadly tuned TRCs (the ‘across-fibre pattern’ of coding)⁴⁵. The recent identification of cells and receptors mediating sweet, bitter, umami and sour taste (Figs 3, 4; Table 1) has generated powerful molecular tools that can be used to devise rigorous tests to distinguish between these models and establish the basis of taste coding at the periphery.

**Figure 1 | Taste-receptor cells, buds and papillae.**  
a, Taste buds (left) are composed of 50–150 TRCs (depending on the species), distributed across different papillae. Circumvallate papillae are found at the very back of the tongue and contain hundreds (mice) to thousands (human) of taste buds. Foliate papillae are present at the posterior lateral edge of the tongue and contain a dozen to hundreds of taste buds. Fungiform papillae contain one or a few taste buds and are found in the anterior two-thirds of the tongue. TRCs project microvilli to the apical surface of the taste bud, where they form the ‘taste pore’; this is the site of interaction with tastants. b, Recent molecular and functional data have revealed that, contrary to popular belief, there is no tongue ‘map’: responsiveness to the five basic modalities — bitter, sour, sweet, salty and umami — is present in all areas of the tongue⁶,⁸,⁹,³²,⁷⁸.

---

¹Howard Hughes Medical Institute and Departments of Neurobiology and Neurosciences, University of California at San Diego, La Jolla, California 92093-0649, USA.  
²National Institute of Dental and Craniofacial Research, National Institutes of Health, Bethesda, Maryland 20892, USA.

NATURE|Vol 444|16 November 2006

**Labelled-line model**  
Across-fibre models  

Figure 2 | Encoding of taste qualities at the periphery. There are two opposing views of how taste qualities are encoded in the periphery. **a**, In the labelled-line model, receptor cells are tuned to respond to single taste modalities — sweet, bitter, sour, salty or umami — and are innervated by individually tuned nerve fibres. In this case, each taste quality is specified by the activity of non-overlapping cells and fibres. **b, c**, Two contrasting models of what is known as the ‘across-fibre pattern’. This states that either individual TRCs are tuned to multiple taste qualities (indicated by various tones of grey and multicoloured stippled nuclei), and consequently the same afferent fibre carries information for more than one taste modality (**b**), or that TRCs are still tuned to single taste qualities but the same afferent fibre carries information for more than one taste modality (**c**). In these two models, the specification of any one taste quality is embedded in a complex pattern of activity across various lines. Recent molecular and functional studies in mice have demonstrated that different TRCs define the different taste modalities, and that activation of a single type of TRC is sufficient to encode taste quality, strongly supporting the labelled-line model.

### Sweet taste

The sweetness of sugar and the pleasure it evokes are so familiar to us that they almost seem to be physical properties of sucrose rather than a representation of neuronal firing in the brain. This tight relationship between sensory quality, positive hedonic value and behavioural acceptance richly illustrates how sweet taste detection and perception evolved to help with the recognition of the most basic and fundamental sources of metabolic energy.

The attractive taste modalities, sweet and umami, are mediated by a small family of three G-protein-coupled receptors (GPCRs) — T1R1, T1R2 and T1R3 — that is distantly related to metabotropic glutamate, pheromone, extracellular-calcium sensing and γ-aminobutyric-acid type B receptors⁶⁻¹⁵. These GPCRs assemble into either homodimeric or heterodimeric receptor complexes¹⁶, and are characterized by the presence of long amino-terminal extracellular domains that are believed to mediate ligand recognition and binding¹⁷.

The critical role of T1Rs in sweet taste detection and perception emerged from an ensemble of studies, including the characterization of T1R expression profiles, the analysis of naturally occurring sweet receptor mutants (and the identification of species-specific differences in sweet taste preferences), functional experiments in cell-based assays, and the generation of genetically modified mouse lines.

T1Rs are expressed in subsets of TRCs, and their expression pattern defines three cell types: TRCs co-expressing T1R1 and T1R3 (T1R1+3 cells), TRCs co-expressing T1R2 and T1R3 (T1R2+3 cells) and TRCs containing T1R3 alone⁸. What do these cells do? More than 30 years ago, genetic studies of sweet taste in mice identified a single principal locus that influences responses to several sweet substances¹⁸,¹⁹. This locus, known as *Sac*, determines threshold differences in the ability of some strains to distinguish sucrose- and saccharin-containing solutions from water¹⁹. The *Sac* locus was recently shown by linkage analysis⁸,¹¹⁻¹⁴,²⁰ and genetic rescue⁸ to encode T1R3, thus implicating a member of the *Tlr* gene family in sweet taste detection. Indeed, functional expression studies in heterologous cells revealed that T1R3 combines with T1R2 (T1R2+3) to form a sweet taste receptor that responds to all classes of sweet tastants, including natural sugars, artificial sweeteners, D-amino acids and intensely sweet proteins⁸,¹⁰. These results validated the T1R2+3 heteromer as a sweet receptor, and suggested that T1R2+3 cells are the sweet-sensing TRCs (see below).

Humans and mice show some prominent differences in their ability to taste certain artificial sweeteners and intensely sweet proteins — for example, mice cannot taste aspartame or monellin. Notably, introduction of the human T1R2 receptor into mice significantly changes their sweet taste preferences to a human-like response profile¹⁵, proving that species differences in sweet taste sensitivity and selectivity are a direct reflection of T1R-sequence variation between species. How does a single receptor complex respond to such a wide range of sweet-tasting compounds, from simple six-carbon sugars to guanidinoacetic acids and even large peptides²¹ and polypeptides? Recently, biochemical studies of human, rodent and chimaeric human–rodent T1R2+3 receptors have shown that diverse classes of sweet-receptor ligands actually require different domains of the receptor complex for recognition²²⁻²⁴, thus providing a simple solution to this puzzle. Together, these genetic, functional and biochemical studies have amply validated the role of the T1R2 and T1R3 subunits in sweet-tastant recognition, and demonstrated the importance of heteromerization in receptor function.

Definitive proof that T1R2+3 is the principal mammalian sweet taste receptor was obtained from studies of *T1r2-* and *T1r3*-knockout mice¹⁵,²⁵ (Fig. 3). Homozygous mutants for either receptor subunit show a devastating loss of sweet taste — all behavioural and electrophysiological responses to artificial sweeteners, D-amino acids and low to moderately high concentrations (up to 300 mM) of natural sugars are abolished¹⁵,²⁵. However, these animals retain very small, albeit measurable, responses to very high concentrations of sugars. Importantly, a *T1r2;T1r3* double knockout completely eliminated these remaining sweet responses¹⁵, unequivocally demonstrating the essential role of T1Rs in all sweet taste detection and perception. Unexpected corroboration of the fundamental requirement of T1Rs for sweet taste came from the recent discovery that cats (all felidae from the common house kitten to the tiger) carry a naturally occurring deletion in their *T1r2* gene²⁶, providing a molecular explanation to the striking, and long-standing, observation that cats do not respond to sweets.

### Umami taste

Most mammals are robustly attracted to the taste of a broad range of L-amino acids¹⁵,²⁷⁻²⁹. In humans, however, just two amino acids — monosodium glutamate (MSG) and aspartate — evoke the unique savory sensation known as umami (whose Japanese characters can be

glutamate and L-amino acids¹⁵ (but see also ref. 25). Together, these results firmly established the T1R1+3 heteromeric GPCR complex as the mammalian umami taste receptor and provided a striking example of heteromeric GPCRs radically altering their selectivity according to a combinatorial arrangement of subunits (sweet T1R2+3 versus umami T1R1+3). They also revealed that sweet and amino-acid (umami) taste — two chemosensory inputs that trigger behavioural attraction — share a common receptor repertoire and evolutionary origin.

### Bitter taste

In contrast to sweet and umami taste, which evolved to recognize a limited subset of nutrients, bitter taste has the onerous task of preventing the ingestion of a large number of structurally distinct toxic compounds. Remarkably, despite the vastness of this repertoire, these compounds all evoke such a similar sensation that we simply know them as ‘bitter’. These observations suggest that bitter taste receptors are probably encoded by a large family of genes, and that the bitter sensation evolved to allow recognition of a wide range of chemicals, but not necessarily to distinguish between them.

Bitter taste is mediated by a family of ~30 highly divergent GPCRs (the T2Rs)³²,³³. T2R genes are selectively expressed in subsets of TRCs distinct from those containing sweet and umami receptors³², and are clustered in regions of the genome genetically linked to bitter taste in humans and mice³²–³⁵. A large number of T2Rs have been shown to function as bitter taste receptors in heterologous expression assays³⁶–⁴⁰, and several have distinctive polymorphisms that are associated with significant variations in sensitivity to selective bitter tastants in mice³⁶, chimpanzees⁴¹ and humans⁴².

Proof that T2Rs are necessary and sufficient for bitter taste came from knockout and misexpression studies in mice. On the one hand, animals lacking a specific T2R (for example, T2R5, the candidate cycloheximide receptor), exhibited a marked and selective loss of their ability to taste the cognate bitter compound⁴³ (Fig. 3). On the other hand, mice engineered to express the human candidate receptors for PTC (phenylthiocarbamide) and salicin, two bitter substances that mice do not normally respond to, became vigorously averse to these two chemicals⁴³. These results demonstrated that T2Rs are necessary and sufficient for selective responses to bitter tastants, and validated T2Rs and T2R-expressing cells as the *in vivo* mediators of bitter taste detection and perception. In addition, the fact that the bitter taste responses of mice can be humanized by introducing human taste receptors illustrated an important feature of T2Rs and bitter taste: selectivity and sensitivity differences to bitter compounds between species is likely to be a reflection of sequence differences in their respective T2R repertoires⁴⁴,⁴⁵.

A remarkable feature of bitter-receptor biology was exposed by the discovery that most, if not all, T2Rs are expressed in the same TRCs³². This implied that individual T2R-expressing cells may function as broadly tuned sensors for all bitter chemicals but might have very limited, if any, discrimination. In fact, it would not be unreasonable to imagine that although animals must be able to detect many bitter compounds, they have no need to distinguish between them qualitatively. Indeed, recent studies in mice have confirmed that T2R-expressing cells operate as universal bitter sensors⁴³,⁴⁶, and that, although mice and rats can distinguish differences in intensity between bitter tastants, they are incapable of discriminating between them⁴⁷. Of course, it would be unreasonable to expect that different bitter TRCs express the same T2R proteins at identical levels, and thus individual bitter-sensing cells can be predicted to vary in their sensitivity to bitter tastants but still be able to respond to the full repertoire.

### Signalling downstream of T1Rs and T2Rs

Signalling cascades downstream of taste receptors have been the subject of intense speculation over the years, with most models hypothesizing a surprising diversity of pathways and strategies⁴⁸–⁵⁰. This proposed complexity contrasted sharply with the demonstrated simplicity of the signalling pathways of other senses, such as olfaction, in which hundreds of distinct receptors share an identical transduction cascade⁵¹.
Sure enough, recent results have demonstrated that the receptors for sweet, bitter and umami taste, although expressed in separate subsets of cells<sup>8</sup>, all signal through a common pathway to transduce tastant recognition into cell activation<sup>46</sup> (but see also ref. 52).

The current data suggest that tastant binding to T1Rs or T2Rs activates the heterotrimeric G proteins gustducin<sup>53</sup> or Ga<sub>i2</sub> (ref. 54) leading to the release of the Gβγ subunits<sup>46,55</sup> and the subsequent stimulation of phospholipase Cβ2 (PLC-β2)<sup>46,56</sup>. Activation of PLC-β2 hydrolyses phosphatidylinositol-4,5-bisphosphate to produce the two intracellular messengers inositol-1,4,5-trisphosphate and diacylglycerol, and ultimately leads to the gating of the taste-transduction channel (the transient receptor potential (TRP) protein TRPM5)<sup>46,57</sup>. As expected from this model, mouse knockouts of gustducin<sup>58,59</sup>, PLC-β2 (refs 46, 60) or TRPM5 (refs 46, 52) have major deficits in sweet, umami and bitter tastes (Fig. 3). Importantly, salty and sour tastes remain unimpaired in all cases<sup>46</sup>, demonstrating that these two modalities use a different signalling pathway and operate independently of sweet, umami and bitter tastes.

Are other pathways important in the detection and perception of sweet, bitter and umami tastes? We would expect signalling molecules representing many different transduction cascades to be present in most types of cell, including TRCs<sup>56,61–66</sup>. However, their mere presence does not imply that they must be involved in taste transduction<sup>50</sup>. Comprehensive physiological and genetic studies will be required to assess what role, if any, they have in taste signalling. It would be interesting if second messengers from a number of pathways modulate taste signals, both at the receptor and downstream levels, and thus provide a platform to shape taste responses as a function of various cues and cellular states. Intriguingly, activation of the TRPM5 transduction channel was recently shown to be strongly temperature dependent<sup>67</sup>, at a range within the normal function of TRCs (15–35 °C). Talavera and colleagues<sup>67</sup> proposed that this property of the channel may underlie some of the effects of temperature on taste detection and, ultimately, perception<sup>67</sup>. It would be rewarding to engineer animals expressing TRPM5 channels with modified temperature profiles and determine the behavioural and physiological impact of such changes on taste responses.

### Salt and sour tastes

A number of studies have suggested that salty and sour tastants modulate taste-cell function by direct entry of Na<sup>+</sup> and H<sup>+</sup> through specialized membrane channels on the apical surface of the cell. In the case of salt, TRC activation is believed to be mediated at least in part by the entry of Na<sup>+</sup> through amiloride-sensitive Na<sup>+</sup> channels<sup>68,69</sup>. However, the identity of the salt ‘receptor’ remains speculative and highly controversial<sup>68,70</sup>.

A broad range of cell types, receptors and mechanisms have been proposed to be responsible for sour taste. These include the activation of hyperpolarization-activated cyclic-nucleotide-gated (HCN) channels<sup>71</sup>, acid-sensing ion channels (ASICs)<sup>72</sup>, potassium (K2P) channels<sup>73,74</sup> and H<sup>+</sup>-gated calcium channels<sup>75</sup>, as well as the involvement of Na<sup>+</sup>/H<sup>+</sup> exchangers<sup>76</sup> and acid inactivation of K<sup>+</sup> channels<sup>77</sup>. However, recent genetic and functional studies have greatly simplified the quest for the sour receptor by demonstrating that a member of the TRP ion-channel family, PKD2L1, demarcates sour-sensing TRCs<sup>78</sup>. PKD2L1 is selectively expressed in a population of TRCs distinct from those mediating sweet, umami and bitter tastes<sup>78,79</sup>, further substantiating the cellular segregation of taste modalities at the periphery.

Proof that PKD2L1-expressing cells function as the acid receptors in the taste system came from conclusive genetic-ablation experiments. The targeting of diphtheria toxin to PKD2L1-expressing cells of the tongue produced animals with a specific and total loss of sour taste<sup>78</sup> (Fig. 3). These results validated PKD2L1 TRCs as the sole acid-sensing cells and implicated the PKD2L1 ion channel as the candidate component of the sour taste (pH) receptor<sup>78,80</sup>. The further demonstration that these sour-deficient mice have normal salt responses indicates that salt taste also must be mediated by an independent population of TRCs (see Fig. 3 and below).

Although sweet, umami and bitter sensing are primarily required in the taste system, acid sensing is also important in a number of other processes, including the monitoring of CO<sub>2</sub> levels in the blood<sup>81</sup> and the internal state of the cerebrospinal fluid and brain<sup>82</sup>. Consequently, it may be predicted that PKD2L1 might also function in other physiological settings. Indeed, Huang and colleagues<sup>78</sup> showed that PKD2L1 is expressed in a selective population of neurons contacting the central canal of the spinal cord that fire in response to minor changes in proton concentration. These results suggest that these neurons function as sentinels of cerebrospinal and ventricular pH, and bring forth a surprising unity in the cellular basis of pH sensing in very different physiological systems.

#### Taste coding at the periphery

Several electrophysiological and calcium-imaging-based studies in rats and mice have reported that individual TRCs are tuned to various taste modalities<sup>4,83–85</sup> and have proposed that encoding of taste quality at the periphery must use an across-fibre pattern of activity (Fig. 2). However, the discovery that sweet, umami, bitter and sour (and, by extrapolation, salt) taste cells are segregated into non-overlapping populations expressing distinct receptors (Fig. 4) demands a revision of this model. We review three lines of investigation demonstrating that the TRCs defined by T1Rs, T2Rs and PKD2L1 function as highly dedicated sensors for sweet, umami, bitter and sour tastes, strongly arguing, instead, for a labelled-line model of coding across all taste modalities.

---

**Figure 4 | Summary of receptors for umami, sweet, bitter and sour tastes.** Schematic representation of taste receptors (and candidate receptors) mediating four of the five basic taste modalities. Although not indicated in the figure, responses to high concentrations of sugars, but not other sweet tastants, are also detected by T1R3 alone<sup>15</sup>. The grey T2R receptor is designed to illustrate the possibility that T2Rs, much like T1Rs, may function as heteromeric complexes. Similarly, the grey receptor next to PKD2L1 depicts a PKD1-family member as a candidate partner<sup>78–80</sup>.

| GPCRs                                                                                          | TRP channel                                                                 |
|------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
| **Umami**                                                                                      | **Candidate sour**                                                          |
| ![Umami Receptor](#)                                                                          | ![PKD2L1 Channel](#)                                                        |
| **T1R1+T1R3**                                                                                 | **PKD2L1**                                                                  |
| **Sweet**                                                                                      | **Bitter**                                                                  |
| ![Sweet Receptor](#)                                                                          | ![Bitter Receptor](#)                                                       |
| **T1R2+T1R3**                                                                                 | **~30 T2Rs**                                                                |

©2006 Nature Publishing Group
Table 1 | Tastant selectivity of candidate mammalian taste receptors

| Tastant quality | Receptor(s) | Class of tastant | Examples of tastants |
|-----------------|------------|------------------|----------------------|
| Umami           | T1R1+T1R3  | Amino acids      | L-Glutamate, L-AP4, glycine*, L-amino acids* |
|                 |            | Nucleotide enhancers | IMP, GMP, AMP |
| Sweet           | T1R2+T1R3  | Sugar†          | Sucrose, fructose, glucose, maltose |
|                 |            | Artificial sweeteners | Saccharin, acesulfame-K, cyclamate‡, aspartame‡ |
|                 |            | D-amino acids    | D-Phenylalanine, D-alanine, D-serine (also some selective L-amino acids) |
|                 |            | Sweet proteins‡  | Monellin, thaumatin, curculin |
| Bitter§         | T2R5¶      |                  | Cycloheximide |
|                 | T2R8¶, T2R4, T2R44 | Denatonium |
|                 | T2R16      |                  | Salicin‡ |
|                 | T2R38      |                  | PTC‡ |
|                 | T2R43, T2R44 |                  | Saccharin |
| Not known       |            | Other toxic/noxious compounds | Quinine, strychnine, atropine |
| Sour            | PKD2L1     | Acids           | Citric acid, tartaric acid, acetic acid, hydrochloric acid |

* Preferentially activates mouse but not human receptors.
† High concentrations of sugars, but not other sweet tastants, can also be detected by T1R3 alone¹⁵.
‡ Activates human but not mouse receptors and does not elicit behavioural responses in wild-type mice.
§ About 30 T2Rs are involved in bitter-tastant recognition.
¶ Mouse T2Rs; all others shown are human. There are 25 human and 35 mouse T2R bitter-taste receptors. For illustrative purposes we have included receptor-ligand matches for a number of de-orphaned T2Rs (for example, mouse T2R5 is the receptor for the protein synthesis inhibitor toxin cycloheximide).

---

aversion mediated by sweet- or bitter-sensing cells, thus strongly substantiating a labelled-line model of taste coding at the periphery. A final corollary to these findings is that expression of a sweet receptor in bitter cells should trigger behavioural aversion to sweet tastants, whereas expression of a bitter receptor in sweet cells should result in attraction to the bitter compound. Indeed, Mueller³³ engineered animals expressing a bitter receptor in sweet cells (Fig. 5) and these mice showed strong attraction to the cognate bitter compounds. Thus, the ‘taste’ of a sweet or bitter compound (in other words, the perception of sweet or bitter) is a reflection of the selective activation of T1R- versus T2R-expressing cells, rather than a property of the receptors or even of the tastant molecules.

### Bitter, sweet and sour

Using a conceptually complementary approach to the functional rescue of subsets of TRCs described above, Huang and colleagues⁷⁸ eliminated entire populations of TRCs by genetically targeting expression of diphtheria toxin (DTA) to defined subsets of taste cells. Remarkably, animals expressing DTA in T1R2-, T2R- or PKD2L1-expressing cells showed extraordinarily specific taste deficits, with each exhibiting a marked loss of only a single taste quality (sweet, bitter and sour, respectively). Taken together, these studies reveal three fundamental features of taste coding at the periphery. First, they prove the functional segregation of individual taste modalities at the cellular level (as proposed in the original receptor expression studies⁸,⁴⁶,⁷⁸). Second, they show the absolute requirement of T1R2-, T2R- and PKD2L1-cells for sweet, bitter and sour taste. Finally, they demonstrate that animals recognize and respond to taste cues (that is, encode and decode signals) without the need for combinatorial patterns of activity at the periphery (across-fibre models).

### The exciting journey from detection to perception

The discovery that individual taste modalities are encoded by different TRCs should make it possible to mark the connectivity pathway for each taste quality individually. As a result, it should be possible to trace defined lines of information from the tongue to the cortex to understand not only where these signals go, but where and how they combine in the circuitry to choreograph taste and flavour.

Two recent reports have provided promising avenues to explore the connectivity between TRCs and central neuronal stations. In the

![Bitter receptor](image.png)

Figure 5 | Behavioural attraction and aversion are mediated by dedicated taste-receptor cells. Targeted expression of a novel bitter receptor to bitter (T2R-expressing) cells results in dose-dependent aversion to the specific bitter tastant (open blue squares). In marked contrast, directing expression of the same receptor to sweet cells produces animals that are strongly attracted to this bitter tastant (filled red circles). Control animals lacking the receptor (filled grey circles) are indifferent to the tastant.

first, Finger and colleagues<sup>87</sup> showed that all sweet, bitter, sour, salty and umami nerve responses are lost in *P2x2;P2x3* (purinergic receptor) double-knockout mice, implicating the purinergic agonist ATP as a potential neurotransmitter in taste<sup>87</sup>. The availability of these taste-blind mice may now provide an experimental platform to engineer animals with function in defined sets of fibres, and therefore track the response of selective ganglion neurons. In the second, Sugita and Shiba<sup>88</sup> used a genetically encoded fluorescent transneuronal tracer to help reveal the circuitry linking TRCs to the brain<sup>88</sup>. Interpretation of the results of this study was complicated by several technical difficulties, including poor transmission of the tracer, the use of a single label for both sweet and bitter pathways (thus necessitating comparison between animals), and the lack of anatomical correlates for many of the labelled neurons. However, this type of approach<sup>89,90</sup> will be valuable in helping decipher the neural wiring for sweet, umami, bitter, sour and salty tastes.

Everyone appreciates that taste perception varies according to context. For example, the addition of sugar to lemon juice masks its sourness without affecting its acidity. More importantly, our perception of taste is significantly extended by other inputs — such as olfactory, visual and somatosensory — as well as prior experience, satiety and hunger<sup>91</sup>. This indicates that combination and comparison across taste qualities, together with information from other sensory systems, must ultimately converge to orchestrate the final percept in the brain. Although electrophysiological studies of the response profiles of brainstem, thalamic or cortical taste neurons are providing important insight into the basic properties of the central taste circuitry<sup>92,93</sup>, the inherent technical difficulties in obtaining such data (often resulting in small sample sizes), have so far prevented the formulation of a true consensus view<sup>94</sup>. We expect that molecular genetic and physiological approaches using novel reporters and genetically encoded activators and inhibitors of neuronal activity, combined with functional imaging at single-cell resolution<sup>95–100</sup>, will be invaluable in helping to decipher how information flows from the tongue to sensory integration centres in the brain, ultimately, to dictate behaviour.


1. Smith, D. V. & St John, S. J. Neural coding of gustatory information. *Curr. Opin. Neurobiol.* **9**, 427-435 (1999).
2. Erickson, R. P., Covey, E. & Doetsch, G. S. Neuron and stimulus typologies in the rat gustatory system. *Brain Res.* **196**, 513–519 (1980).
3. Erickson, R. P. The evolution of neural coding ideas in the chemical senses. *Physiol. Behav.* **69**, 3-13 (2000).
4. Caicedo, A., Kim, K. N. & Roper, S. D. Individual mouse taste cells respond to multiple chemical stimuli. *J. Physiol. (Lond.)* **544**, 501-509 (2002).
5. Smith, D. V., John, S. J. & Boughter, J. D. Neuronal cell types and taste quality coding. *Physiol. Behav.* **69**, 77-85 (2000).
6. Hoon, M. A. *et al.* Putative mammalian taste receptors: a class of taste-specific GPCRs with distinct topographic selectivity. *Cell* **96**, 541-551 (1999).
7. Bachmanov, A. A. *et al.* Positional cloning of the mouse saccharin preference (*Sac*) locus. *Chem. Senses* **26**, 925-933 (2001).
8. Nelson, G. *et al.* Mammalian sweet taste receptors. *Cell* **106**, 381-390 (2001).
9. Nelson, G. *et al.* An amino-acid taste receptor. *Nature* **416**, 199-202 (2002).
10. Li, X. *et al.* Human receptors for sweet and umami taste. *Proc. Natl Acad. Sci. USA* **99**, 4692-4696 (2002).
11. Kitagawa, M., Kusakabe, Y., Miura, H., Ninomiya, Y. & Hino, A. Molecular genetic identification of a candidate receptor gene for sweet taste. *Biochem. Biophys. Res. Commun.* **283**, 236-242 (2001).
12. Max, M. *et al.* Tas1r3, encoding a new candidate taste receptor, is allelic to the sweet responsiveness locus *Sac*. *Nature Genet.* **28**, 58-63 (2001).
13. Montmayeur, J. P., Liberles, S. D., Matsunami, H. & Buck, L. B. A candidate taste receptor gene near a sweet taste locus. *Nature Neurosci.* **4**, 492-498 (2001).
14. Sainz, E., Korley, J. N., Battey, J. F. & Sullivan, S. L. Identification of a novel member of the T1R family of putative taste receptors. *J. Neurochem.* **77**, 896-903 (2001).
15. Zhao, G. Q. *et al.* The receptors for mammalian sweet and umami taste. *Cell* **115**, 255-266 (2003).
16. Pin, J. P. & Acher, F. The metabotropic glutamate receptors: structure, activation mechanism and pharmacology. *Curr. Drug Targets CNS Neurol. Disord.* **1**, 297-317 (2002).
17. Kunishima, N. *et al.* Structural basis of glutamate recognition by a dimeric metabotropic glutamate receptor. *Nature* **407**, 971-977 (2000).
18. Fuller, J. L. Single-locus control of saccharin preference in mice. *J. Hered.* **65**, 33-36 (1974).
19. Lush, I. E. The genetics of tasting in mice. VI. Saccharin, acesulfame, dulcin and sucrose. *Genet. Res.* **53**, 95-99 (1989).
20. Li, X. *et al.* High-resolution genetic mapping of the saccharin preference locus (*Sac*) and the putative sweet taste receptor (*T1R1*) gene (*Gpr70*) to mouse distal Chromosome 4. *Mamm. Genome* **12**, 13-16 (2001).
21. Danilova, V., Hellekant, G., Tinti, J. M. & Nofre, C. Gustatory responses of the hamster *Mesocricetus auratus* to various compounds considered sweet by humans. *J. Neurophysiol.* **80**, 2102-2112 (1998).
22. Xu, H. *et al.* Different functional roles of T1R subunits in the heteromeric taste receptors. *Proc. Natl Acad. Sci. USA* **101**, 14258-14263 (2004).
23. Jiang, P. *et al.* Molecular mechanisms of sweet receptor function. *Chem. Senses* **30** (Suppl. 1), i17-i18 (2005).
24. Jiang, P. *et al.* The cysteine-rich region of T1R3 determines responses to intensely sweet proteins. *J. Biol. Chem.* **279**, 45068-45075 (2004).
25. Damak, S. *et al.* Detection of sweet and umami taste in the absence of taste receptor T1r3. *Science* **301**, 850-853 (2003).
26. Li, X. *et al.* Pseudogenization of a sweet-receptor gene accounts for cats' indifference toward sugar. *PLoS Genet.* **1**, 27-35 (2005).
27. Iwasaki, K., Kasahara, T. & Sato, M. Gustatory effectiveness of amino acids in mice: behavioral and neurophysiological studies. *Physiol. Behav.* **34**, 531-542 (1985).
28. Iwasaki, K. & Sato, M. A. Taste preferences for amino acids in the house musk shrew, *Suncus murinus*. *Physiol. Behav.* **28**, 829-833 (1982).
29. Pritchard, T. C. & Scott, T. R. Amino acids as taste stimuli. I. Neural and behavioral attributes. *Brain Res.* **253**, 81-92 (1982).
30. Ikeda, K. New seasonings. *Chem. Senses* **27**, 847-849 (2002).
31. Yamaguchi, S. The synergistic taste effect of monosodium glutamate and disodium 5'-inosinate. *J. Food Sci.* **32**, 473-478 (1967).
32. Adler, E. *et al.* A novel family of mammalian taste receptors. *Cell* **100**, 693-702 (2000).
33. Matsunami, H., Montmayeur, J. P. & Buck, L. B. A family of candidate taste receptors in human and mouse. *Nature* **404**, 601-604 (2000).
34. Lush, I. E. & Holland, G. The genetics of tasting in mice. V. Glycine and cycloheximide. *Genet. Res.* **52**, 207-212 (1988).
35. Reed, D. R. *et al.* Localization of a gene for bitter-taste perception to human chromosome 5p15. *Am. J. Hum. Genet.* **64**, 1478-1480 (1999).
36. Chandrashekar, J. *et al.* T2Rs function as bitter taste receptors. *Cell* **100**, 703-711 (2000).
37. Bufe, B., Hofmann, T., Krautwurst, D., Raguse, J. D. & Meyerhof, W. The human TAS2R16 receptor mediates bitter taste in response to β-glucopyranosides. *Nature Genet.* **32**, 397-401 (2002).
38. Pronin, A. N., Tang, H., Connor, J. & Keung, W. Identification of ligands for two human bitter T2R receptors. *Chem. Senses* **29**, 583-593 (2004).
39. Kuhn, C. *et al.* Bitter taste receptors for saccharin and acesulfame K. *J. Neurosci.* **24**, 10260-10265 (2004).
40. Behrens, M. *et al.* The human taste receptor hTAS2R14 responds to a variety of different bitter compounds. *Biochem. Biophys. Res. Commun.* **319**, 479-485 (2004).
41. Wooding, S. *et al.* Independent evolution of bitter-taste sensitivity in humans and chimpanzees. *Nature* **440**, 930-934 (2006).
42. Kim, U. K. *et al.* Positional cloning of the human quantitative trait locus underlying taste sensitivity to phenylthiocarbamide. *Science* **299**, 1221-1225 (2003).
43. Mueller, K. L. *et al.* The receptors and coding logic for bitter taste. *Nature* **434**, 225-229 (2005).
44. Shi, P. & Zhang, J. Contrasting modes of evolution between vertebrate sweet/umami receptor genes and bitter receptor genes. *Mol. Biol. Evol.* **23**, 292-300 (2006).
45. Go, Y., Satta, Y., Takenaka, O. & Takahata, N. Lineage-specific loss of function of bitter taste receptor genes in humans and nonhuman primates. *Genetics* **170**, 313-326 (2005).
46. Zhang, Y. *et al.* Coding of sweet, bitter, and umami tastes: different receptor cells sharing similar signaling pathways. *Cell* **112**, 293-301 (2003).
47. Spector, A. C. & Kopka, S. L. Rats fail to discriminate quinine from denatonium: implications for the neural coding of bitter-tasting compounds. *J. Neurosci.* **22**, 1937-1941 (2002).
48. Chaudhari, N. & Roper, S. D. Molecular and physiological evidence for glutamate (umami) taste transduction via a G protein-coupled receptor. *Ann. NY Acad. Sci.* **855**, 398-406 (1998).
49. Kinnamon, S. C. A plethora of taste receptors. *Neuron* **25**, 507-510 (2000).
50. Smith, D. V. & Margolskee, R. F. Making sense of taste. *Sci. Am.* **284**, 32-39 (2001).
51. Brunet, L. J., Gold, G. H. & Ngai, J. General anosmia caused by a targeted disruption of the mouse olfactory cyclic nucleotide-gated cation channel. *Neuron* **17**, 681-693 (1996).
52. Damak, S. *et al.* Trpm5 null mice respond to bitter, sweet, and umami compounds. *Chem. Senses* **31**, 253-264 (2006).
53. McLaughlin, S. K., McKinnon, P. J. & Margolskee, R. F. Gustducin is a taste-cell-specific G protein closely related to the transducins. *Nature* **357**, 563-569 (1992).
54. Kusakabe, Y. *et al.* Comprehensive study on G protein α-subunits in taste bud cells, with special reference to the occurrence of Gα2 as a major Ga species. *Chem. Senses* **25**, 525-531 (2000).
55. Huang, L. *et al.* Gy13 colocalizes with gustducin in taste receptor cells and mediates IP3 responses to bitter denatonium. *Nature Neurosci.* **2**, 1055-1062 (1999).
56. Rossler, P., Kroner, C., Freitag, J., Noe, J. & Breer, H. Identification of a phospholipase C β subtype in rat taste cells. *Eur. J. Cell Biol.* **77**, 253-261 (1998).
57. Perez, C. A. *et al.* A transient receptor potential channel expressed in taste receptor cells. *Nature Neurosci.* **5**, 1169-1176 (2002).
58. Wong, G. T., Gannon, K. S. & Margolskee, R. F. Transduction of bitter and sweet taste by gustducin. *Nature* **381**, 796-800 (1996).
59. Ruiz, C. J., Wray, K., Delay, E., Margolskee, R. F. & Kinnamon, S. C. Behavioral evidence for a role of α-gustducin in glutamate taste. *Chem. Senses* **28**, 573-579 (2003).
60. Dotson, C. D., Roper, S. D. & Spector, A. C. PLCβ2-independent behavioral avoidance of prototypical bitter-tasting ligands. *Chem. Senses* **30**, 593-600 (2005).
61. Varkevisser, B. & Kinnamon, S. C. Sweet taste transduction in hamster: role of protein kinases. *J. Neurophysiol.* **83**, 2526-2532 (2000).
62. Rosenzweig, S., Yan, W., Dasso, M. & Spielman, A. I. Possible novel mechanism for bitter taste mediated through cGMP. *J. Neurophysiol.* **81**, 1661-1665 (1999).
63. Bernhardt, S. J., Naim, M., Zehavi, U. & Lindemann, B. Changes in IP3 and cytosolic Ca²⁺ in response to sugars and non-sugar sweeteners in transduction of sweet taste in the rat. *J. Physiol. (Lond.)* **490**, 325-336 (1996).
64. Striem, B. J., Pace, U., Zehavi, U., Naim, M. & Lancet, D. Sweet tastants stimulate adenylate cyclase coupled to GTP-binding protein in rat tongue membranes. *Biochem. J.* **260**, 121-126 (1989).
65. Gilbertson, T. A. & Boughter, J. D. Taste transduction: appetizing times in gustation. *Neuroreport* **14**, 905-911 (2003).

66. Avenet, P., Hofmann, F. & Lindemann, B. Transduction in taste receptor cells requires cAMP-dependent protein kinase. *Nature* **331**, 351–354 (1988).

67. Talavera, K. *et al.* Heat activation of TRPM5 underlies thermal sensitivity of sweet taste. *Nature* **438**, 1022–1025 (2005).

68. Heck, G. L., Mierson, S. & DeSimone, J. A. Salt taste transduction occurs through an amiloride-sensitive sodium transport pathway. *Science* **223**, 403–405 (1984).

69. Avenet, P. & Lindemann, B. Amiloride-blockable sodium currents in isolated taste receptor cells. *J. Membr. Biol.* **105**, 245–255 (1988).

70. Lyall, V. *et al.* The mammalian amiloride-insensitive non-specific salt taste receptor is a vanilloid receptor-1 variant. *J. Physiol. (Lond.)* **558**, 147–159 (2004).

71. Stevens, D. R. *et al.* Hyperpolarization-activated channels HCN1 and HCN4 mediate responses to sour stimuli. *Nature* **413**, 631–635 (2001).

72. Ugawa, S. *et al.* Receptor that leaves a sour taste in the mouth. *Nature* **395**, 555–556 (1998).

73. Lin, W., Burks, C. A., Hansen, D. R., Kinnamon, S. C. & Gilbertson, T. A. Taste receptor cells express pH-sensitive leak K⁺ channels. *J. Neurophysiol.* **82**, 2909–2919 (2004).

74. Richter, T. A., Dvoryanchikov, G. A., Chaudhari, N. & Roper, S. D. Acid-sensitive two-pore domain potassium (K2P) channels in mouse taste buds. *J. Neurophysiol.* **92**, 1928–1936 (2004).

75. Waldmann, R., Champigny, G., Bassilana, F., Heurteaux, C. & Lazdunski, M. A proton-gated cation channel involved in acid-sensing. *Nature* **386**, 173–177 (1997).

76. Lyall, V. *et al.* Basolateral Na⁺-H⁺ exchanger-1 in rat taste receptor cells is involved in neural adaptation to acidic stimuli. *J. Physiol. (Lond.)* **556**, 159–173 (2004).

77. Cummings, T. A. & Kinnamon, S. C. Apical K⁺ channels in *Necturus* taste cells. Modulation by intracellular factors and taste stimuli. *J. Gen. Physiol.* **99**, 591–613 (1992).

78. Huang, A. L. *et al.* The cells and logic for mammalian sour taste detection. *Nature* **442**, 934–938 (2006).

79. Lopezjimenez, N. D. *et al.* Two members of the TRPP family of ion channels, Pkd1l3 and Pkd2l1, are co-expressed in a subset of taste receptor cells. *J. Neurochem.* **98**, 68–77 (2006).

80. Ishimaru, Y. *et al.* Transient receptor potential family members PKD1L3 and PKD2L1 form a candidate sour taste receptor. *Proc. Natl Acad. Sci. USA* **103**, 12569–12574 (2006).

81. Lahiri, S. & Forster, R. E. CO₂/H⁺ sensing: peripheral and central chemoreception. *Int. J. Biochem. Cell Biol.* **35**, 1413–1435 (2003).

82. Vigh, B. *et al.* The system of cerebrospinal fluid-contacting neurons. Its supposed role in the nonsynaptic signal transmission of the brain. *Histol. Histopathol.* **19**, 607–628 (2004).

83. Gilbertson, T. A., Boughter, J. D., Zhang, H. & Smith, D. V. Distribution of gustatory sensitivities in rat taste cells: whole-cell responses to apical chemical stimulation. *J. Neurosci.* **21**, 4931–4941 (2001).

84. Sato, T. & Beidler, L. M. Broad tuning of rat taste cells for four basic taste stimuli. *Chem. Senses* **22**, 287–293 (1997).

85. Richter, T. A., Caicedo, A. & Roper, S. D. Sour taste stimuli evoke Ca²⁺ and pH responses in mouse taste cells. *J. Physiol. (Lond.)* **547**, 475–483 (2003).

86. Redfern, C. H. *et al.* Conditional expression and signaling of a specifically designed Gi-coupled receptor in transgenic mice. *Nature Biotechnol.* **17**, 165–169 (1999).

87. Finger, T. E. *et al.* ATP signaling is crucial for communication from taste buds to gustatory nerves. *Science* **310**, 1495–1499 (2005).

88. Sugita, M. & Shiba, Y. Genetic tracing shows segregation of taste neuronal circuits for bitter and sweet. *Science* **309**, 781–785 (2005).

89. Zou, Z., Horowitz, L. F., Montmayeur, J. P., Snapper, S. & Buck, L. B. Genetic tracing reveals a stereotyped sensory map in the olfactory cortex. *Nature* **414**, 173–179 (2001).

90. Kuze, B., Matsuyama, K., Matsui, T., Miyata, H. & Mori, S. Segment-specific branching patterns of single vestibulospinal tract axons arising from the lateral vestibular nucleus in the cat: A PHA-L tracing study. *J. Comp. Neurol.* **414**, 80–96 (1999).

91. Rolls, E. T. *The Brain and Emotion* (Oxford Univ. Press, USA, 2000).

92. Katz, D. B., Simon, S. A. & Nicolelis, M. A. Dynamic and multimodal responses of gustatory cortical neurons in awake rats. *J. Neurosci.* **21**, 4478–4489 (2001).

93. Di Lorenzo, P. M. The neural code for taste in the brain stem: response profiles. *Physiol. Behav.* **69**, 87–96 (2000).

94. Spector, A. C. & Travers, S. P. The representation of taste quality in the mammalian nervous system. *Behav. Cogn. Neurosci. Rev.* **4**, 143–191 (2005).

95. Zhang, J., Campbell, R. E., Ting, A. Y. & Tsien, R. Y. Creating new fluorescent probes for cell biology. *Nature Rev. Mol. Cell Biol.* **3**, 906–918 (2002).

96. Choi, G. B. *et al.* Lhx6 delineates a pathway mediating innate reproductive behaviors from the amygdala to the hypothalamus. *Neuron* **46**, 647–660 (2005).

97. Miesenböck, G. & Kevrekidis, I. G. Optical imaging and control of genetically designated neurons in functioning circuits. *Annu. Rev. Neurosci.* **28**, 533–563 (2005).

98. Gosgnach, S. *et al.* V1 spinal neurons regulate the speed of vertebrate locomotor outputs. *Nature* **440**, 215–219 (2006).

99. Gogos, J. A., Osborne, J., Nemes, A., Mendelsohn, M. & Axel, R. Genetic ablation and restoration of the olfactory topographic map. *Cell* **103**, 609–620 (2000).

100. Brecht, M. *et al.* Novel approaches to monitor and manipulate single neurons *in vivo*. *J. Neurosci.* **24**, 9223–9227 (2004).

**Acknowledgements** We thank a group of extraordinary students, postdoctoral fellows and research technicians in our laboratories, who joined us on this wonderful journey of mammalian taste research beginning in the fall of 1997. N.J.P.R. is an investigator in the Intramural program at the NIH, NIDCR. C.S.Z. is an investigator of the Howard Hughes Medical Institute.

**Author Information** Reprints and permissions information is available at npg.nature.com/reprintsandpermissions. The authors declare no competing financial interests. Correspondence should be addressed to C.S.Z. or N.J.P.R. (czuker@ucsd.edu; nick.ryba@nih.gov).
