
Functional Analysis of Arabidopsis thaliana RHM2/MUM4,  
a Multidomain Protein Involved in UDP-<sub>D</sub>-glucose  
to UDP-<sub>L</sub>-rhamnose Conversion*

Received for publication, October 31, 2006, and in revised form, December 26, 2006 Published, JBC Papers in Press, December 26, 2006, DOI 10.1074/jbc.M610196200

Takuji Oka†, Tadashi Nemoto§, and Yoshifumi Jigami‡1  
From the †Research Center for Glycoscience and the §Biological Information Research Center, National Institute of Advanced Industrial Science and Technology (AIST), Tsukuba, Ibaraki 305-8566, Japan

UDP-<sub>L</sub>-rhamnose is required for the biosynthesis of cell wall rhamnogalacturonan-I, rhamnogalacturonan-II, and natural compounds in plants. It has been suggested that the *RHM2/MUM4* gene is involved in conversion of UDP-<sub>D</sub>-glucose to UDP-<sub>L</sub>-rhamnose on the basis of its effect on rhamnogalacturonan-I-directed development in *Arabidopsis thaliana*. *RHM2/MUM4*-related genes, *RHM1* and *RHM3*, can be found in the *A. thaliana* genome. Here we present direct evidence that all three RHM proteins have UDP-<sub>D</sub>-glucose 4,6-dehydratase, UDP-4-keto-6-deoxy-<sub>D</sub>-glucose 3,5-epimerase, and UDP-4-keto-<sub>L</sub>-rhamnose 4-keto-reductase activities in the cytoplasm when expressed in the yeast *Saccharomyces cerevisiae*. Functional domain analysis revealed that the N-terminal region of RHM2 (RHM2-N; amino acids 1–370) has the first activity and the C-terminal region of RHM2 (RHM2-C; amino acids 371–667) has the two following activities. This suggests that RHM2 converts UDP-<sub>D</sub>-glucose to UDP-<sub>L</sub>-rhamnose via an UDP-4-keto-6-deoxy-<sub>D</sub>-glucose intermediate. Site-directed mutagenesis of RHM2 revealed that mucilage defects in *MUM4-1* and *MUM4-2* mutant seeds of *A. thaliana* are caused by abolishment of RHM2 enzymatic activity in the mutant strains and furthermore, that the GXXGXX(G/A) and YXXXK motifs are important for enzymatic activity. Moreover, a kinetic analysis of purified His<sub>6</sub>-tagged RHM2-N protein revealed 5.9-fold higher affinity of RHM2 for UDP-<sub>D</sub>-glucose than for dTDP-<sub>D</sub>-glucose, the preferred substrate for dTDP-<sub>D</sub>-glucose 4,6-dehydratase from bacteria. RHM2-N activity is strongly inhibited by UDP-<sub>L</sub>-rhamnose, UDP-<sub>D</sub>-xylose, and UDP but not by other sugar nucleotides, suggesting that RHM2 maintains cytoplasmic levels of UDP-<sub>D</sub>-glucose and UDP-<sub>L</sub>-rhamnose via feedback inhibition by UDP-<sub>L</sub>-rhamnose and UDP-<sub>D</sub>-xylose.

ronan-I (RG-I) and rhamnogalacturonan-II (RG-II), and various L-rhamnose-containing natural compounds (flavonoids, terpenoids, and saponins) (1, 2). Three major pectic polysaccharides (homogalacturonan, RG-I, and RG-II) are present in the primary cell walls of plants (1). RG-I is a polymer of more than 100 individual α1,4-linked disaccharide units that are themselves composed of L-rhamnose and D-galacturonic acid (1). RG-II is a structurally complex polysaccharide that exists in primary walls as a dimer covalently cross-linked by a borate diester bond (1, 3). These pectic polysaccharides play a major role in the development and growth of all vascular plants (1). In bacteria, cell surface polysaccharides also contain L-rhamnose and are essential for survival and interaction between bacteria (4, 5). The rhamnose-containing polysaccharides of bacteria are synthesized from dTDP-rhamnose (dTDP-Rha) (4, 5). In the last decade, much effort has been directed at studying the dTDP-Rha synthetic pathway (e.g. gene structures, enzymatic properties, and their functions in cell wall synthesis) (4). A dTDP-Rha biosynthetic gene cluster consisting of the *rmlB* (or *rfbB*), *rmlC* (or *rfbC*), and *rmlD* (or *rfbD*) genes is responsible for biosynthesis of dTDP-Rha from dTDP-glucose (dTDP-Glc) in bacteria, *Escherichia coli*, *Mycobacterium tuberculosis*, and *Salmonella enterica* serovar Typhimurium (Fig. 1) (4, 5). The *rmlB*, *rmlC*, and *rmlD* genes encode dTDP-Glc 4,6-dehydratase (EC 4.2.1.46), dTDP-4-keto-6-deoxy-D-glucose (dTDP-4K6DG) 3,5-epimerase (EC 5.1.3.13), and dTDP-4-keto-L-rhamnose (dTDP-4KR) 4-keto-reductase (EC 1.1.1.133), respectively (5).

In plants, UDP-Rha is synthesized from UDP-D-glucose (UDP-Glc) via an analogous enzymatic pathway (6) (Fig. 1). It is known that UDP-Rha is synthesized from UDP-Glc by UDP-Glc 4,6-dehydratase (EC 4.2.1.76), UDP-4-keto-6-deoxy-D-glucose (UDP-4K6DG) 3,5-epimerase, and UDP-4-keto-L-rhamnose (UDP-4KR) 4-keto-reductase (6). RHM1, RHM2/MUM4, RHM3, and UER1 proteins of *Arabidopsis thaliana* are putative plant orthologues of the dTDP-Rha biosynthetic enzymes in bacteria (Figs. 2A and 3A). It has been reported that the UER1 protein (7), also called NRS/ER (8), has both dTDP/UDP-4K6DG 3,5-epimerase and dTDP/UDP-4KG 4-keto-reductase activities, forming dTDP/UDP-Rha from dTDP/UDP-4K6DG. The N-terminal and C-terminal regions of RHM proteins are

lacturonan-II; dTDP-4K6DG, dTDP-4-keto-6-deoxy-D-glucose; dTDP-4KR, dTDP-4-keto-L-rhamnose; UDP-Xyl, UDP-D-xylose; MOPS, 4-morpholinepropanesulfonic acid; MES, 4-morpholineethanesulfonic acid; DTT, dithiothreitol; Ni-NTA, nickel-nitrilotriacetic acid.

* This work was supported by grants from the New Energy and Industrial Technology Development Organization of Japan (NEDO). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked "advertisement" in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
1 To whom correspondence should be addressed. Tel.: 81-29-861-6160; Fax: 81-29-861-6161; E-mail: jigami.yoshi@aist.go.jp.
2 The abbreviations used are: UDP-Rha, UDP-L-rhamnose; dTDP, deoxythymidine 5′-diphosphate; HPLC, high performance liquid chromatography; ESI/MS, electrospray ionization-mass spectrometry; COSY, correlated spectroscopy; TOCSY, total correlation spectroscopy; NOESY, nuclear Overhauser effect spectroscopy; RG-I, rhamnogalacturonan-I; RG-II, rhamnoga-

Characterization of Plant UDP- _L_ -rhamnose Synthase in A. thaliana

dTDP-glucose
![Chemical Structure](https://i.imgur.com/1234567.png)
dTDP

NAD⁺
↓
NADH

4,6-dehydratase

(RmlB)

dTDP-4-keto-6-deoxy-glucose
![Chemical Structure](https://i.imgur.com/8901234.png)
dTDP

3,5-epimerase

(RmlC)

dTDP-4-keto-rhamnose
![Chemical Structure](https://i.imgur.com/5678901.png)
dTDP

NADPH
↓
NADP⁺

4-keto-reductase

(RmlD)

dTDP-rhamnose
![Chemical Structure](https://i.imgur.com/2345678.png)
dTDP

Cell wall polysaccharide

UDP-glucose
![Chemical Structure](https://i.imgur.com/9012345.png)
UDP

NAD⁺
↓
NADH

4,6-dehydratase

UDP-4-keto-6-deoxy-glucose
![Chemical Structure](https://i.imgur.com/6789012.png)
UDP

3,5-epimerase

(UER1)

UDP-4-keto-rhamnose
![Chemical Structure](https://i.imgur.com/3456789.png)
UDP

NADPH
↓
NADP⁺

4-keto-reductase

(UER1)

UDP-rhamnose
![Chemical Structure](https://i.imgur.com/0123456.png)
UDP

Rhamnogalacturonan I, rhamnogalacturonan II and L-rhamnose-containing natural compounds

**Bacteria**  
**Plant**

FIGURE 1. Schematic representation of the dTDP-Rha and UDP-Rha biosynthetic pathways in bacteria and plants, respectively. In bacteria, dTDP-Rha is produced from dTDP-Glc by dTDP-Glc 4,6-dehydratase (RmlB), dTDP-4K6DG 3,5-epimerase (RmlC), and dTDP-4KR 4-keto-reductase (RmlD) activities via dTDP-4K6DG and dTDP-4KR. In plants, UDP-Rha is produced from UDP-Glc by UDP-Glc 4,6-dehydratase, UDP-4K6DG 3,5-epimerase, and UDP-4KR 4-keto-reductase activities via UDP-4K6DG and UDP-4KR. Watt et al. (8) reported that UER1 encodes both dTDP/UDP-4K6DG 3,5-epimerase and dTDP/UDP-4KR 4-keto-reductase. UDP-Rha is further utilized to synthesize cell wall polysaccharides and L-rhamnose-containing natural compounds.

similar to bacterial RmlB and RmlD proteins, respectively. On the basis of bioinformatics analysis, Reiter and Vanzin (9) reported that RHM1, RHM2/MUM4, and RHM3 proteins are putative nucleotide sugar interconversion enzymes in A. thaliana. It has also been reported that in the mucilage of seeds of MUM4 (MUCILAGE-MODIFIED4)-1, MUM4-2 mutants and rhm2 T-DNA insertion mutants, RG-I is found at lower levels and is of a lower molecular weight than in the mucilage of normal seeds (10, 11), suggesting that RHM2 is involved in the synthesis of UDP-Rha. However, no biochemical evidence for the postulated function of these enzymes has been described. On the other hand, a mutation of *rhm1* was isolated as an extragenic suppressor of a mutation in the gene that encodes LRR-extensin1 (LRX1), a typical A. thaliana cell wall-glycosylated protein (12). Although using recombinant RHM1, B. Link and W. D. Reiter (12) reportedly found that the biochemical activity of RHM1 includes in vitro conversion of UDP-Glc to UDP-Rha,

the finding was presented as unpublished data and to the best of our knowledge, a characterization of RHM1 has not yet been published.

In this study, we expressed *RHM1*, *RHM2*, and *RHM3* in protease-deficient yeast cells, and found that the RHM1, RHM2/MUM4, and RHM3 proteins are trifunctional enzymes with UDP-Glc 4,6-dehydratase, UDP-4K6DG 3,5-epimerase, and UDP-4KR 4-keto-reductase activities. Analysis of both enzymatic and physicochemical properties of specific protein fragments revealed that the N-terminal region of RHM2/MUM4 (RHM2-N: 1–370 amino acids) encodes UDP-Glc 4,6-dehydratase and the C-terminal region of RHM2/MUM4 (RHM2-C: 371–667 amino acids) encodes both UDP-4K6DG 3,5-epimerase and UDP-4KR 4-keto-reductase, indicating that the genes are responsible for the synthesis of UDP-Rha from UDP-Glc in plants. In addition, RHM2-N activity is strongly inhibited by UDP-Rha, UDP- _D_ -xylose (UDP-Xyl), and UDP but

not by other sugar nucleotides and D-glucose, suggesting that RHM2/MUM4 maintains cytoplasmic levels of UDP-Glc and UDP-Rha via feedback inhibition by UDP-Rha and UDP-Xyl. Finally, we discuss the implications of our results on the regulatory mechanism of UDP-sugar metabolism in plants.

**EXPERIMENTAL PROCEDURES**

Microorganisms and Growth Conditions—Yeast strains used in this study were as follows (13): BY4741 (MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0), Δpep4 (as BY4741, pep4::kanMX4), Δprb1 (as BY4741, prb1::kanMX4), Δyps1 (As BY4741, yps1::kanMX4), Δyps2 (as BY4741, yps2::kanMX4), and Δyps3 (as BY4741, yps3::kanMX4). Strains were grown in a synthetic minimal (SD) medium containing 0.67% yeast nitrogen base without amino acids (Difco, Detroit, MI) and 0.5% D-glucose (WAKO, Osaka, Japan), supplemented with an amino acids mixture (14). Cell growth in liquid culture was done by inoculating 0.1 A₆₀₀ of cells (1.0 A₆₀₀ = 1 × 10⁷ cells) into 5 ml of medium in a conical tube. The tubes were shaken at 120 rpm at 30°C. Standard transformation procedures for Saccharomyces cerevisiae were used (15).

Source of Sugar Nucleotides—UDP-D-galactose or UDP-Gal, UDP-N-acetyl-D-galactosamine or UDP-GalNAc, UDP-N-acetyl-D-glucosamine or UDP-GlcNAc, UDP-D-glucuronic acid or UDP-GlcA, GDP-D-mannose or GDP-Man, GDP-L-fucose or GDP-Fuc, CMP-D-sialic acid or CMP-Sia, UDP-L-arabinose or UDP-Ara, and UDP-Rha, and UDP-Xyl were used. The above reagents were from Sigma with the following exceptions. UDP-Ara and UDP-Xyl were from CarboSource Services (Athens, GA). UDP-Rha was synthesized using a cytoplasmic fraction from recombinant Δprb1 yeast cells expressing RHM2/MUM4 (crude enzyme fraction) and purified by HPLC using a Develosil RPAQUEOUS column (250 × 4.6 mm, Nomura Chemical Co., Ltd, Seto, Japan).

Construction of His₆-tagged RHM1, RHM2/MUM4, and RHM3—All PCR were done using Phusion High Fidelity DNA Polymerase (Daiichi Pure Chemicals, Tokyo, Japan). Plasmids useful for expression of RHM1 (TAIR locus number At1g78570), RHM2/MUM4 (TAIR locus number At1g53500), and RHM3 (TAIR locus number At3g14790) were constructed as follows. The A. thaliana RHM1 and RHM2/MUM4 genes were amplified by PCR using A. thaliana λ cDNA Library (Stratagene, La Jolla, CA) as a template for the primers RHM1–6xHIS-SacI-F (5′-AAGAGCTCATGCATCACCATCACATGGCTTCGTACACTCCAAAGAAC-3′) and RHM1-R-KpnI (5′-AAAGGTACCTCAGGTTTCTTGGCCC GTATG-3′), or RHM2-6xHIS-EcoRI-F (5′-AGAATTCATGCATCACCATCACATGGCATCACATGGATATAAGCCAAAAGAAC-3′) and RHM2-SalI-R (5′-AAAAAGTCGACTTAGGTTCTTGT TTTGGTCAAAGAC-3′), respectively. Underlining indicates restriction sites (SacI, KpnI, EcoRI, and SalI, respectively). RHM3 was amplified by PCR using a RAFL clone (16, 17) from RIKEN GSC (Resource number pda08705) as template for primers RHM3–6xHIS-EcoRI-F (5′-AGAATTCATGCATCACCATCACATGGCTACATATAAGCCTAAGAACATCCTC-3′) and RHM3-SalI-R (5′-AAAAAGTCGACTTACGTTCTTGT TTAGG

TTCGAAGACG-3′). Underlining indicates restriction sites (EcoRI and SalI, respectively). The amplified DNA fragments were digested with the corresponding restriction enzymes and inserted into corresponding sites in YEp352GAPII (18) to yield pTO-6xHIS-RHM1, pTO-6xHIS-RHM2, and pTO-6xHIS-RHM3.

Construction of His₆-tagged RHM2-N, RHM2-C, and UER1—Plasmids for expression of RHM2-N, RHM2-C, and UER1 were constructed as follows. RHM2-N and RHM2-C were amplified by PCR using pTO-6xHIS-RHM2 as a template for primers RHM2–6xHIS-EcoRI-F and RHM2-N-SalI-R (5′-AAAAAGTCGACTTATACAACC GTAATGTCTGGAC-3′), RHM2-C-6xHIS-EcoRI-F (5′-AGAATTCATGCA TCACCATCACACACCTAAGAATGGTGATTC TGGTG-3′), and RHM2-SalI-R, respectively. Underlining indicates restriction sites (SalI and EcoRI, respectively). UER1 (TAIR Locus number At1g63000) was amplified by PCR using an A. thaliana λ cDNA Library (Stratagene) as template for primers UER1–6xHIS-EcoRI-F (5′-AGAATT CATGCATCACCATCACATGGTTGCAGACG CAAACGGTTC-3′) and UER1-SalI-R (5′-AAAAAGTCG ACTCAAGCTTTAACTTCAGTCTTCTTGGGGC-3′). Underlining indicates restriction sites (EcoRI and SalI, respectively). The amplified DNA fragments were digested with EcoRI and SalI restriction enzymes, and inserted into the EcoRI and SalI sites in YEp352GAPII to yield pTO- 6xHIS-RHM2-N, YEp352-6xHIS-RHM2-C, and YEp352- 6xHIS-UER1. The BamHI fragment that included His₆- tagged RHM2-C or His₆-tagged UER1 gene expression cassettes from YEp352-6xHIS-RHM2-C or YEp352-6xHIS- UER1 was inserted into the BamHI site of YEp351 (19) to yield pTO-6xHIS-RHM2-C and pTO-6xHIS-UER1.

Construction of RHM2-C without a His₆ Tag—Plasmids for expression of untagged RHM2-C were constructed as follows. RHM2-C was amplified by PCR from pTO-6xHIS-RHM2 with primers RHM2-C-EcoRI-F (5′-AGAATTCATGACAC CTAAAGATGGTGATTCTGGTG-3′) and RHM2-SalI-R. Underlining indicates the EcoRI restriction sites. The amplified DNA fragment was digested with EcoRI and SalI restriction enzymes and inserted into the EcoRI and SalI sites of YEp352GAPII to yield YEp352-RHM2-C. A BamHI fragment that includes the RHM2-C gene expression cassette from YEp352-RHM2-C was inserted into the BamHI site of the YEp351 to yield pTO-RHM2-C.

Construction of RHM2-G18A, RHM2-K36A, RHM2-D96A, RHM2-K165A, RHM2-G193R, RHM2-G392A, RHM2-K413A, and RHM2-K518A—The following single point mutations in the Rossmann motif (GXXGXX(G/A)) (20) and YXXXK motif (21) of the RHM2 protein were introduced via site-directed mutagenesis: G18A, K36A, D96N, K165A, G193R, G392A, K413A, and K518A. Mutagenesis was performed with pTO-6xHIS-RHM2 as a template using primers RHM2- G18A-F (5′-CATTACTGGAGCTGCTGCATTTATTGCTT CTCATG-3′) and RHM2-G18A-R (5′-CATGAGAAGCAAT AAATGCAGCAGCTCCAGTAATG-3′), RHM2-K36A-F (5′- GTAACTATCCTGATTACGCGATCGTTGTTCTTGGAC TAATCAGGATAGTTAC-3′), RHM2-D96N-F (5′-GCTG-

Characterization of Plant UDP- _L_ -rhamnose Synthase in A. thaliana

CTCAAACATGTTAATAACTCTTTTGGTAATAGC- buffer (pH 7.0) at a flow rate of 0.7 ml/min. The retention times for UDP-Glc and UDP-Rha were 12.2 and 13.2 min, respectively, under the assay conditions. UDP-4K6DG was observed as a broad peak at the retention time from ranging 12.5 to 20 min. Alternatively, products were analyzed by HPLC with a CarboPac PA1 anion-exchange column (250 × 4.0 mm, Dionex Corp., Sunnyvale, CA) according to the method described by Kang et al. (23) with a slight modification. After sample injection, the column was equilibrated with solvent A (20 mM K₂HPO₄-KHPO₄, pH 7.5) at a flow rate of 0.7 ml/min for 5 min, and analyzed isocratically with solvent B (200 mM K₂HPO₄-KHPO₄, pH 7.5) at a flow rate of 0.7 ml/min for 30 min. The retention times for UDP-Glc, UDP-4K6DG, and UDP-Rha were 17.5, 18.3, and 16.2 min, respectively. After either method of separation, UDP-sugars were detected by UV₂₆₀ absorbance.

Mass Spectrometry—The HPLC-purified enzymatic products from UDP-Glc of RHM2 and RHM2-N were lyophilized and suspended with 1 mM ammonium acetate buffer (pH 10.0). The fractions were analyzed by electrospray ionization mass spectrometry (ESI-MS). Mass spectra were acquired on an Esquire 3000-plus instrument (Bruker Daltonik GmbH, Bremen, Germany) in the negative-ion mode. Conditions for ESI-MS were as follows: nebulizer flow 10 p.s.i., nozzle temperature 300 °C, drying gas (N₂) flow 5.0 liters/min. Negative ion spectra were generated by scanning the range *m/z* 500–570.

NMR Analysis—Products were analyzed by proton nuclear magnetic resonance (¹H NMR) spectroscopy. All proton spectra were acquired on a Bruker DMX-500 spectrometer (Bruker Daltonik GmbH) equipped with a 5-mm TXI probe, operating at 500.13 MHz and 25.0 °C. The HPLC-purified products, UDP-Rha (~250 μg) and UDP-4K6DG (~100 μg), were lyophilized. Subsequently, purified material was dissolved in 500 μl of 99.97% D₂O. The pH (pD) of samples were uncorrected. For each sample, water-suppressed high-resolution one-dimensional, two-dimensional water-suppressed COSY (correlated spectroscopy), TOCSY (total correlation spectroscopy), and NOESY (nuclear Overhauser effect spectroscopy) experiments were performed. Data points of 32 k were given to one-dimensional spectra and 2024 (*t₂*: complex) × 256 (*t₁*: real) data points were given to all two-dimensional spectra. The spectral width was set to 5.00000 × 10³ Hz and phase-sensitive detection in the *t₁* domain was obtained in the States-TPPI (time proportional phase incrementation frequency discrimination (States method)) mode. A total of 16 or 32 scans for *t₁* increments were accumulated. For TOCSY, MLEV-17 (Malcom Levitt’s decoupling sequence) spin lock pulses (25–100 ms) were applied. For NOESY, 75–500 ms was selected for NOE mixing. Solvent suppression for all NMR measurements was achieved by weak continuous wave presaturation. All data were processed using the XWINNMR3.5 software package (Bruker Daltonik GmbH) to obtain the final 64 k (one-dimensional) or 2048 × 1024 (two-dimensional) data.

Sequence Analysis—To analyze the nucleotide and amino acid sequences, GENETYX-MAC version 12.0.0 (Genetyx Co., Tokyo, Japan) was used according to the supplier’s protocol.

Preparation of a Crude Enzyme Fraction—Yeast cells were grown in SD medium at 30 °C for 24 h. Cells were harvested, resuspended in 10 mM Tris-HCl buffer (pH 7.5) with the Complete EDTA-free protease inhibitor (1 tablet of Complete/50 ml; Roche Applied Science), and lysed with glass beads. Crude membranes were removed by centrifugation at 14,000 × g and the supernatants were used as crude enzyme.

UDP-Rha Synthesis Assay—An in vitro assay for UDP-Rha synthase was performed as follows. The reaction mixture contained 3 mM UDP-Glc, 3 mM NAD⁺, and 3 mM NADPH (Sigma), Complete EDTA-free protease inhibitor (1 tablet of Complete/50 ml; Roche), 250 mM MOPS-NaOH buffer (pH 7.5), and *S. cerevisiae* crude enzyme (*A*₆₀₀ = 5.0–7.5) in a total volume of 100 μl. The mixtures were incubated at 30 °C for 60 min and the reaction was stopped by vortex mixing with 100 μl of ice-cold phenol/chloroform/isoamyl alcohol (25:24:1). The supernatants were analyzed by HPLC (below).

HPLC Analysis—The products were analyzed by HPLC using a Develosil RPAQUEOUS reverse phase column (250 × 4.6 mm, Nomura Chemical Co.) as previously described (22). The column was equilibrated with 20 mM triethylamine acetate

Purification of RHM2-N from Yeast Cells—The His₆-tagged RHM2-N protein was purified from yeast cell extracts using Ni-NTA-agarose (Qiagen). All steps were performed at 4 °C

Characterization of Plant UDP- _L_ -rhamnose Synthase in A. thaliana

A
N-terminal
C-terminal
(UDP-glucose 4,6-dehydratase domain)
(UDP-4-keto-6-deoxy-glucose 3,5-epimerase and
UDP-4-keto-rhamnose 4-keto-reductase domain)

GAP-P
6 x HIS tag
RHM1
1 aa
669 aa

GAP-P
6 x HIS tag
RHM2
1 aa
667 aa

GAP-P
6 x HIS tag
RHM3
1 aa
664 aa

B
1 2 3 4 5 6 7
75 kDa
42 kDa
RHM1

C
1 2 3 4
75 kDa
Pgk1p
45 kDa

D
Intensity A260
(1)
UDP-Glc
BY4741 Δprb1
(2)
UDP-Glc
peak A (product A)
(3)
UDP-Glc
peak A (product A)
(4)
UDP-Glc
peak A (product A)
10 15 20 (min)

buffer A (10 mM Tris-HCl buffer, pH 8.0, 5 mM DTT from WAKO, 1 mM EDTA from Sigma, and complete protease inhibitor mixture from Roche), and then lysed with glass beads. Cell debris was removed by centrifugation at 15,000 × g for 15 min and then 4 ml of Ni-NTA-agarose was added to the supernatant. The supernatant-agarose mixture was gently shaken for 1 h. The Ni-NTA-agarose was collected by centrifugation at 3,000 × g for 10 min, and washed for five times with 50 ml of buffer B (10 mM Tris-HCl buffer, pH 8.0, 5 mM DTT, 1 mM EDTA, 150 mM NaCl from WAKO, and Complete protease inhibitor mixture). The enzyme was then eluted twice with 5 ml of buffer A containing 500 mM imidazole (WAKO). The eluted enzyme was dialyzed in buffer A overnight. Finally, ~4.5 mg of purified His<sub>6</sub>-tagged RHM2-N enzyme sample was obtained. The purified enzyme was analyzed by SDS-PAGE. Protein concentrations were determined using the bicinchoninic acid protein assay reagent (Pierce) with bovine serum albumin as a standard.

Test of Putative Inhibitors of UDP-Glc 4,6-Dehydratase—UDP-Glc 4,6-dehydratase activities were measured as follows. Reaction mixtures contained MOPS-NaOH (250 mM, pH 7.5), NAD<sup>+</sup> (1.0 mM), DTT (5 mM), EDTA (2.5 mM), UDP-Glc (1.0 mM), and purified His<sub>6</sub>-tagged RHM2-N in a total volume of 100 μl. The UDP-Glc 4,6-dehydratase assay was performed in the presence of various inhibitors. The purified His<sub>6</sub>-tagged RHM2-N (1.92 μg) was incubated at 30 °C for 60 min. The reaction was stopped by heat treatment (99 °C) for 10 min. The supernatants were analyzed by HPLC using an anion-exchange column (CarboPac PA1). UDP-sugars were detected by UV<sub>260</sub> absorbance. Three independent repetitions of each experiment were performed.

Effects of Temperature and pH on

Enzymatic Activity of His<sub>6</sub>-tagged RHM2-N—Optimal conditions for UDP-4,6-dehydratase activity were determined as follows. To determine the optimal temperature for His<sub>6</sub>-tagged

FIGURE 2. Functional analyses of RHM1, RHM2/MUM4, and RHM3 proteins. A, schematic representation of RHM1 (amino acids 1–669), RHM2/MUM4 (amino acids 1–667), and RHM3 (amino acids 1–664). B, expression of RHM2/MUM4 in various protease-deficient yeast cells. RHM2/MUM4 is detected as a 75-kDa protein band and the major degraded RHM2/MUM4 protein as a 42-kDa protein using an anti-penta-HIS antibody. Cell lysate from 0.25 A<sub>600</sub> of yeast cells was loaded on each lane (lanes 1–7). Pgk1p was used as a loading control for cytosolic proteins. Lane 1, control BY4741 cells without plasmid; lane 2, BY4741; lane 3, BY4741Δprb1; lane 4, BY4741Δpep4; lane 5, BY4741Δyps1; lane 6, BY4741Δyps2; and lane 7, BY4741Δyps3. Except for the control strain (lane 1), each strain harbored the His<sub>6</sub>-tagged RHM2 expression construct. C, immunoblotting analysis of RHM1, RHM2/MUM4, and RHM3. The presence of RHM1, RHM2/MUM4, or RHM3 proteins was detected as a 75-kDa protein using an anti-penta-HIS antibody. Cell lysate from 0.25 A<sub>600</sub> of yeast cells was loaded on each lane (lanes 1–4). Pgk1p was used as a loading control for cytosolic proteins. Lanes 1, control BY4741Δprb1 cells without plasmid. Lanes 2–4, BY4741Δprb1 cells with the expression plasmid for His<sub>6</sub>-tagged RHM1 (lane 2), His<sub>6</sub>-tagged RHM2 (lane 3), or His<sub>6</sub>-tagged RHM3 (lane 4). D, in vitro analysis of UDP-Rha synthase activity. Enzyme activities were assayed as described under “Experimental Procedures.” Cell lysate from 5.0 A<sub>600</sub> of yeast cells was used as a source of crude enzyme. Panel 1, control BY4741Δprb1 cells without plasmid; panels 2–4, BY4741Δprb1 cells with the expression plasmid for His<sub>6</sub>-tagged RHM1, His<sub>6</sub>-tagged RHM2, and His<sub>6</sub>-tagged RHM3, respectively. Representative results from three independent experiments are shown.

unless otherwise stated. To prepare the cell extracts, yeast cells were grown in 6 liters of SD medium at 30 °C for 24 h. Cells (A<sub>600</sub> = 3600) were harvested, resuspended in ~200 ml of

FEBRUARY 23, 2007 • VOLUME 282 • NUMBER 8
JOURNAL OF BIOLOGICAL CHEMISTRY 5393

Characterization of Plant UDP- _L_ -rhamnose Synthase in A. thaliana

RHM2-N, the purified protein (0.96 μg) was incubated at a series of temperatures (20, 25, 30, 35, 40, and 45 °C) for 60 min and the reaction was stopped by heat treatment (99 °C) for 10 min. The optimal pH for His<sub>6</sub>-tagged RHM2-N was determined as follows. Three kinds of pH buffer solutions, 250 mM MES-NaOH (pH 5.5, 6.0, and 7.0), 250 mM MOPS-NaOH (pH 6.5, 7.0, 7.5, and 8.0), and 250 mM Tris-HCl (pH 7.5, 8.0, and 9.0) were tested. The purified His<sub>6</sub>-tagged RHM2-N (0.96 μg) was used for reaction at 30 °C for 60 min and the reactions were stopped by heat treatment (99 °C) for 10 min. Enzymatic activity was evaluated by determining the area of the products via HPLC equipped with an anion-exchange column (CarboPac PA1). UDP-sugars were detected by UV<sub>260</sub> absorbance. Three independent repetitions of each experiment were performed.

**Kinetic Studies—**The level of dTDP/UDP-Glc 4,6-dehydratase activity was estimated by determining the amount of dTDP/UDP-4K6DG. Reaction mixtures contained MOPS-NaOH (250 mM, pH 7.5), NAD<sup>+</sup> (2.0 mM), DTT (5 mM), EDTA (2.5 mM), UDP-Glc or dTDP-Glc, and purified His<sub>6</sub>-tagged RHM2-N in a total volume of 100 μl. Variations in the reaction mixtures are noted in the text. Reactions were started by addition of UDP-Glc or dTDP-Glc; then, 0.96 or 9.94 μg of the purified His<sub>6</sub>-tagged RHM2-N was used in assays for production of UDP-Glc or dTDP-Glc, respectively. Assay mixtures were incubated for 30 min and reactions were stopped by heat treatment (99 °C) for 10 min. Enzymatic activity was evaluated by determining the area of the products via HPLC equipped with an anion-exchange column (CarboPac PA1). UDP-sugars were detected by UV<sub>260</sub> absorbance, and sugar nucleotide levels were compared with dTDP-Glc or UDP-Glc as a standard. Linearity (*r*² = 0.999) was maintained between 10 μM and 10 mM of dTDP-Glc and UDP-Glc. The *K*<sub>m</sub> and *V*<sub>max</sub> values were determined using the Michaelis-Menten equation using the averages of three independent experiments.

**RESULTS**

**Cloning and Expression of RHM1, RHM2/MUM4, and RHM3—**The A. thaliana RHM1, RHM2/MUM4, and RHM3 genes were amplified by PCR from an A. thaliana cDNA library or RAFL clones. The RHM1, RHM2/MUM4, and RHM3 proteins showed 85.6–91.6% sequence identity to one another. Putative orthologues were also identified in other plant genomes (rice and maize). Interestingly, invertebrate *Caenorhabditis elegans* has a putative homolog of RHM2/MUM4 with 29.3% identity (NCBI accession number AAB00715); however, the function of the gene is unclear. The N-terminal regions of RHM1, RHM2/MUM4, and RHM3 are 38.9–39.5% identical at the amino acid level to the RmlB protein, a dTDP-Glc 4,6-dehydratase of *E. coli* K12 (NCBI accession number AAB88398). The C termini of RHM1, RHM2/MUM4, and RHM3 have some similarity to Rm1D (21.0–25.2% identity), which is a dTDP-4KR 4-keto-reductase of *E. coli* K12 (NCBI accession number AAB88399) and a high degree of similarity (78.6–82.9%) to A. thaliana UER1, which has both dTDP/UDP-4K6DG 3,5-epimerase and dTDP/UDP-4KR 4-keto-reductase activities, suggesting that the N-terminal and C-terminal regions of RHM proteins are functionally distinct. Thus we define the N-terminal region of RHM2 (amino acids 1–370) as RHM2-N and the

TABLE 1  
Proton chemical shifts and coupling constants of UDP-Rha synthesized from UDP-Glc by RHM2 protein

| Proton | Uracil   | Ribose    | UDP-β- _L_ -rhamnose | L-Rhamnose coupling constants |
|--------|----------|-----------|-----------------------|-------------------------------|
|        | ppm      |           |                       |                             |
| H1     | 5.96     | 5.20      |                       | *J*<sub>1",P</sub> 8.70 *J*<sub>1",2"</sub> <1 |
| H2     | 4.35     | 4.07      |                       | *J*<sub>2",3"</sub> 3.20     |
| H3     | 4.26     | 3.63      |                       | *J*<sub>3",4"</sub> 9.31     |
| H4     | 4.34     | 3.36      |                       | *J*<sub>4",5"</sub> 9.61     |
| H5     | 5.95     | 4.20      |                       | *J*<sub>5",6"</sub> 6.10     |
| H6     | 7.94     | 1.29      |                       |                             |

<sup>a</sup> Assignments were confirmed by coupling constant analysis with high-resolution one-dimensional spectrum.

C-terminal region (amino acids 371–667) as RHM2-C. Both RHM2-N and RHM2-C belong to the sugar nucleotide modifying subfamily of short chain dehydrogenase/reductase enzymes, which include a putative NAD(P)(H) binding motif (GXXGXX(G/A)) (20) and a conserved catalytic triad (YXXXK) (21). The hydropathy profiles suggest that RHM1, RHM2, and RHM3 are cytoplasmic proteins that lack transmembrane regions (data not shown).

To examine the functions of RHM proteins, we attempted to use the budding yeast *S. cerevisiae* to produce the proteins, as it is well known that many recombinant proteins from plants can be efficiently expressed and produced in yeast cells, probably due to similar codon usage (24) and biogenesis of similar subcellular organelles such as the cell wall. We placed the *RHM1*, *RHM2/MUM4*, and *RHM3* genes under the control of the *S. cerevisiae* constitutive *TDH3 (GAP)* promoter (pTO-6xHIS-RHM1, pTO-6xHIS-RHM2, and pTO-6xHIS-RHM3 plasmids; Fig. 2A). When *RHM2/MUM4* was expressed in the *S. cerevisiae* parental strain BY4741, we were initially able to detect a 42-kDa major band and several smaller, putative degradation products using an anti-Penta-HIS antibody (Fig. 2B, lane 2). Because the deduced molecular mass of RHM2/MUM4 protein is ~75 kDa, even the largest observable band (the 42-kDa band) is presumed to be a degradation product of the intact RHM2/MUM4 protein. To avoid protein degradation, the *RHM2/MUM4* gene was next expressed in various protease null mutants of *S. cerevisiae*. The estimated molecular size of RHM2/MUM4 protein was the expected size for the intact protein in the Δ*prb1*, Δ*pep4*, and Δ*yps3* strains (Fig. 2B, lanes 3, 4, and 7) but not in the Δ*yps1* or Δ*yps2* strains (Fig. 2B, lanes 5 and 6). The 42-kDa band and several other degraded bands disappeared to the greatest extent in the Δ*prb1* strain, indicating that among the strains we tested, Δ*prb1* is the most suitable for expression and isolation of intact RHM proteins (Fig. 2B, lane 3). Thus, we used the Δ*prb1* strain as a host for protein expression in subsequent assays.

To examine the enzymatic properties of RHM proteins, we introduced an appropriate set of expression vectors into Δ*prb1* cells and protein expression was confirmed using antibodies that recognize a penta-His epitope tag present on the heterologously expressed proteins, which could be isolated from the cytoplasmic fraction of cell lysates. Expression of *RHM1*, *RHM2/MUM4*, and *RHM3* genes in Δ*prb1* cells resulted in production of a major protein corresponding to 75 kDa without detectable degradation products (Fig. 2C, upper panel, lanes

Characterization of Plant UDP- _L_ -rhamnose Synthase in A. thaliana

Functional Analysis of RHM1, RHM2, and RHM3 Proteins—Next, we measured enzymatic activities of the RHMs using the cytoplasmic fraction of recombinant yeast cells expressing RHM1, RHM2/MUM4, or RHM3 as a source of crude enzyme. UDP-Rha synthetic activity was assayed using 3 mM UDP-Glc as a substrate and 3 mM NAD<sup>+</sup> and 3 mM NADPH as co-factors. The cytoplasmic fraction of Δprb1 cells without any RHM genes showed no conversion of UDP-Glc to UDP-Rha (Fig. 2D, panel 1) but in contrast, cytoplasmic fractions from cells expressing RHM1, RHM2/MUM4, or RHM3 had a new peak (defined as peak A) at 13.2 min (Fig. 2D, panels 2–4), suggesting that peak A is an enzymatic reaction product (defined as product A; Fig. 2D, panels 2–4). To confirm that product A corresponds to UDP-Rha, we collected the peaks for both the substrate (UDP-Glc) and product A from the RHM2 sample and analyzed their molecular weights by ESI-MS. Two peaks, *m/z* 564.8 and 548.8, were detected in the purified sugar nucleotide fraction from the RHM2/MUM4 gene-expressing cells (Fig. 4A, middle panel). The 564.8 peak was identical to that of authentic UDP-Glc from a Δprb1 sample (Fig. 4A, upper panel) and the 548.8 peak is consistent with the theoretical molecular weight of UDP-Rha, suggesting that product A is UDP-Rha.

To further test the chemical structure, we collected ~250 μg of product A using HPLC with a Develosil RPAQUEOUS C30 column from RHM2 (Fig. 2D, panel 3) and analyzed the sample by <sup>1</sup>H NMR. Spectral assignments were done by cross-checking high-resolution one-dimensional, two-dimensional COSY, TOCSY, and NOESY spectra with various mixing times. Primarily, a mixing time of 80 ms for TOCSY spectra were analyzed for product A. Assignments in TOCSY spectra for product A are shown in Fig. 4B, panel 1. The chemical structures and proton positions of UDP-Rha are also shown in Fig. 4B, panel 3. All assignments were confirmed by chemical shift and corresponding coupling constant analysis with high-resolution one-dimensional spectrum

FIGURE 3. Functional domain analysis of RHM2 protein. A, schematic representations of RHM2, RHM2-N, RHM2-C, and UER1 proteins fused to a His<sub>6</sub> tag. RHM2, full-length RHM2 (1–667 amino acids). RHM2-N, N-terminal region of RHM2 (1–370 amino acids). RHM2-C, C-terminal region of RHM2 (371–667 amino acids). UER1, full-length UER1 (1–301 amino acids). Black box, His<sub>6</sub> tag. B, immunoblotting analysis of RHM2-N, RHM2-C, and UER1 proteins. The presence of RHM2-N, RHM2-C, or UER1 was detected with an anti-penta-HIS antibody. Cell lysate from 0.25 A<sub>600</sub> of yeast cells was loaded on each lane (lanes 1–6). Pkg1p was used as a loading control for cytosolic proteins. RHM2-N was detectable as a 42-kDa protein, and RHM2-C and UER1 proteins were detected at 35 kDa. Lane 1, control BY4741Δprb1 cells without plasmid; lanes 2–6, BY4741Δprb1 cells harboring expression plasmid for His<sub>6</sub>-tagged RHM2-N (lane 2), His<sub>6</sub>-tagged RHM2-C (lane 3), both His<sub>6</sub>-tagged RHM2-N and His<sub>6</sub>-tagged RHM2-C (lane 4), His<sub>6</sub>-tagged UER1 (lane 5), or both His<sub>6</sub>-tagged RHM2-N and His<sub>6</sub>-tagged UER1 (lane 6). C, in vitro analysis of UDP-Rha synthase activity. Enzyme activities were assayed as described under “Experimental Procedures.” Cell lysate from 7.5 A<sub>600</sub> of yeast cells was used as a source of crude enzyme. Panel 1, control BY4741Δprb1 cells without plasmid. Panels 2–7, BY4741Δprb1 cells harboring expression plasmid for His<sub>6</sub>-tagged RHM2 (panel 2), His<sub>6</sub>-tagged RHM2-N (panel 3), His<sub>6</sub>-tagged RHM2-C (panel 4), both His<sub>6</sub>-tagged RHM2-N and RHM2-C (panel 5), His<sub>6</sub>-tagged UER1 (panel 6) or both His<sub>6</sub>-tagged RHM2-N and UER1 (panel 7). Representative results from three independent experiments are shown.

2–4). Pkg1p (3-phosphoglycerokinase) was used as a loading control for cytosolic proteins (Fig. 2C, bottom panel), and the relative amount of RHM2/MUM4 was greater than that of RHM1 and RHM3 proteins, suggesting that the RHM2/MUM4 protein is more stable than RHM1 or RHM3 in yeast cells (Fig. 2C, upper panel, lanes 2–4).

Characterization of Plant UDP-β-L-rhamnose Synthase in A. thaliana

A

BY4741  
Δprb1  

RHM2  
(product A)  

RHM2-N  
(product B)  

B

(1) TOCSY (product A)

(2) NOESY (product A)

(3) UDP-β-L-rhamnose (UDP-Rha)

C

(1) TOCSY (product B)

(2) UDP-4-keto-6-deoxy-α-D-glucose (UDP-4K6DG)

FIGURE 4. Molecular identification of the enzymatic reaction products from UDP-Glc using RHM2 and RHM2-N. A, ESI-MS analyses of the products from UDP-Glc using yeast cell extracts of BY4741Δprb1 cells, BY4741Δprb1 cells harboring expression plasmid for His₆-tagged whole RHM2/MUM4 protein (RHM2), and BY4741Δprb1 cells harboring expression plasmid for His₆-tagged RHM2-N (RHM2-N). Nucleotide sugars used as a substrate and found as products were purified by HPLC equipped with a C30 column (RPAQUEOUS). The mass spectra are shown for the control strain BY4741Δprb1 cells without plasmid (upper panel), with the His₆-tagged RHM2 expression plasmid (middle panel), or with the His₆-tagged RHM2-N expression plasmid (bottom panel). The mass units for UDP-Glc, product A, and product B, are shown (564.8, 548.8, and 546.8, respectively). B, structural identification of product A from UDP-Glc by RHM2/MUM4 protein and the chemical structure of UDP-Rha. ¹H NMR was performed as described under “Experimental Procedures.” Panel 1, TOCSY spectra of product A from UDP-Glc using yeast cell extracts of BY4741Δprb1 cells with the His₆-tagged RHM2 (RHM2) expression plasmid; panel 2, NOE spectra of product A from UDP-Glc using yeast cell extracts of BY4741Δprb1 cells with the His₆-tagged RHM2 (RHM2) expression plasmid; panel 3, the chemical structure and proton position of UDP-Rha. C, structural identification of product B from UDP-Glc by RHM2-N protein and the chemical structure of UDP-4K6DG. Panel 1, TOCSY spectra of product B from UDP-Glc using yeast cell extracts of BY4741Δprb1 cells with the His₆-tagged RHM2-N (RHM2-N) expression plasmid; panel 2, the chemical structure and proton position of UDP-4K6DG.

of product A are summarized in Table 1. The relative configuration of UDP-Rha based on NOE signals (mixing time 500 ms) is shown in Fig. 4B, panel 2. Signals for product A were in good agreement with the reported chemical shift values and coupling constants (8) of UDP-Rha (Table 1). In addition, NOE analysis revealed that H1, H2, H3, and H5 positions of the product were on the same side of a hexose ring (Fig. 4B, panel 2). Therefore, the chemical structure of product A was identified as UDP-Rha, strongly suggesting that RHM1, RHM2/MUM4, and RHM3 proteins can convert UDP-Glc to UDP-Rha.

**Analysis of Functional Domains in the RHM2/MUM4 Protein—** Next, we analyzed the function of the N- and C-terminal domains of RHM2/MUM4. The results of sequence comparisons suggest that RHM2-N and RHM2-C have different enzymatic functions, as RHM2-N is similar to bacterial RmlB, whereas RHM2-C is similar to bacterial RmlD and plant UER1 (8–11). To elucidate the function of each domain of RHM2, we constructed expression vectors for the RHM2-N, RHM2-C, and UER1 genes under the constitutive S. cerevisiae TDH3 (GAP) promoter using plasmids pTOP-6xHIS-RHM2-N, pTO-6xHIS-RHM2-C, and pTO-6xHIS-UER1 (Fig. 3A). The expression vectors were introduced into Δprb1 cells and the presence of heterologous proteins in the cytoplasmic fraction of cell lysates was confirmed by immunoblot analysis using anti-penta-His epitope tag antibodies as described under “Experimental Procedures” (Fig. 3B). The RHM2-N construct resulted in production of a 42-kDa protein (Fig. 3B, lane 2), and the RHM2-C and UER1 constructs resulted in production of 35-kDa proteins (Fig. 3B, lanes 3 and 5). When Δprb1 cells co-expressed RHM2-N and RHM2-C or RHM2-N and UER1, proteins of 42 and 35 kDa in size were detected (Fig. 3B, lanes 4 and 6).

We next examined the enzymatic activities of RHM2-N and RHM2-C, using the cytoplasmic fraction of cells expressing RHM2-N, RHM2-C, or UER1 genes as a source of crude enzyme. UDP-Rha synthetic activity was assayed with 3 mM UDP-Glc as a substrate and 3 mM NAD⁺ and 3 mM NADPH as cofactors. The cytoplasmic fraction of Δprb1 control cells, RHM2-C and UER1 expressing cells showed no conversion of UDP-Glc (Fig. 3C, panels 1, 4, and 6). In contrast, in RHM2-N-expressing cells, a new peak was observed (defined as peak B). The peak was broad and eluted with a retention time ranging from 12.5 to 20 min (Fig. 3C, panel 3), suggesting that peak B is an enzymatic reaction product of RHM2-N (defined as product B; Fig. 3C, panel 3). It is likely that increased hydrophobicity of product B via 4-keto group formation affects the interaction of product B with the column, resulting in the broad peak we observed in the chromatogram. In addition, cells co-expressing either RHM2-N and RHM2-C, or RHM2-N and UER1, showed conversion of product B to UDP-Rha (Fig. 3C, panels 5 and 7), indicating that RHM2-N and RHM2-C are functionally distinct and that RHM2-C has the same enzymatic activity as UER1 (Fig. 3C, panels 5 and 7).

To better understand the structure of product B, we collected the unmodified substrate (UDP-Glc) from a Δprb1 sample and product B from a RHM2-N sample, and analyzed their molecular weights by ESI-MS. Two peaks (*m/z* 564.8 and 546.8) were detected in the purified sugar nucleotide fraction from the

---

**TABLE 2**

Proton chemical shifts and coupling constants of UDP-4K6DG synthesized from UDP-Glc by RHM2-N proteins

| Proton | Chemical shifts, δ | D-4-Keto-6-deoxy-Glc |
|--------|---------------------|----------------------|
|        | Uracil              | Ribose               | UDP-4-keto-α-D-6-deoxy-Glc | coupling constants |
|        | ppm                 |                      |                          |                    |
| H1     | 5.96                | 5.53                 |                          | *Hz*<sup>a</sup>   |
| H2     | 4.35                | 3.60                 |                          | *J*<sub>1′,P</sub> 6.10 *J*<sub>1′,2′</sub> 3.20 |
| H3     | 4.40                | 3.76                 |                          | *J*<sub>2′,3′</sub> 9.92 |
| H4     | 4.25                |                      |                          |                    |
| H5     | 5.95                | 4.20                 | 4.08                     | *J*<sub>5′,6′</sub> 7.02 |
| H6     | 7.96                |                      | 1.20                     |                    |

<sup>a</sup> Assignments were confirmed by coupling constant analysis with high-resolution one-dimensional spectrum.

RHM2-N sample (Fig. 4A). The 564.8 peak has the same molecular weight as authentic UDP-Glc from the Δprb1 sample (Fig. 4A, upper panel) and the 546.8 peak is consistent with the theoretical molecular weight of UDP-4K6DG, suggesting that the product B is UDP-4K6DG (Fig. 4A, bottom panel).

To determine the chemical structure of the reaction product, we collected ~100 μg of product B from the RHM2-N sample and analyzed the product via <sup>1</sup>H NMR using the same methods applied for analysis of UDP-Rha (Fig. 4C and Table 2). Primarily, a mixing time of 80 ms TOCSY spectra was used to analyze product B proton signals, along with a chemical shift analysis. TOCSY assignments for product B are shown in Fig. 4C, panel 1. The chemical structures and proton positions of product B are shown in Fig. 4C, panel 2. Chemical shifts were tentatively determined on the basis of UDP-Rha signals (uracil H5 = 5.95). The chemical shifts and corresponding coupling constants are summarized in Table 2. In the 4-keto-6-deoxy-D-glucose spectrum, the lack of a H4 proton signal and a loss of scalar connectivity were observed (Fig. 4C, panel 1, and Table 2). Chemical shift changes occurred at positions H3 and H5 in the hexose ring originating from UDP-Rha, likely due to substitution of a carbonyl group at the fourth position. This strongly suggests that the chemical structure of the product is UDP-4K6DG. The coupling constants also support the orientations of H1, H2, and H3 as equatorial, axial, and axial, respectively, indicating that H3 is inverted in comparison with the direction of H3 of UDP-Rha. These results also support the idea that the sugar nucleotide is UDP-4K6DG, providing additional evidence that RHM2-N encodes a UDP-Glc 4,6-dehydratase that converts UDP-Glc to UDP-4K6DG. Furthermore, the data suggest that RHM2-C encodes both a UDP-4K6DG 3,5-epimerase and a UDP-4KR 4-keto-reductase, which together convert UDP-4K6DG to UDP-Rha.

**Determination of Essential Amino Acid Residues in the RHM2/MUM4 Protein—** We next asked if single amino acid replacements can affect the enzymatic activities of the RHM2/MUM4 protein. The positions of mutations in the RHM2/MUM4 protein are shown in Fig. 5A. Western et al. (11) reported that MUM4-1 and MUM4-2 contain a single amino acid replacement of Asp<sup>96</sup> to Asn (D96N) and Gly<sup>193</sup> to Arg (G193R), respectively. These mutations cause a decrease in seed coat mucilage and incomplete cytoplasmic rearrangement (11). However, it has not been clear if these phenotypes are caused by a reduction of RHM2/MUM4 protein stability or by a change in RHM2/MUM4 enzymatic activity. To ask if muta-

Characterization of Plant UDP-L-rhamnose Synthase in A. thaliana

A
RHM2
Gxx18GxxA
N
96D
R
193G
A
Gxx392GxxG
RHM2-N domain (1 - 370 aa)
RHM2-C domain (371 - 667 aa)
Yxxx36K
Yxxx165K
Yxxx413K
Yxxx518K
A
A
A
A

B
1 2 3 4 5 6 7 8 9 10
RHM2
Pgk1p
75 kDa
45 kDa

C
(1)
BY4741Δprbl
UDP-Glc
RHM2
UDP-Rha
UDP-4K6DG
Intensity A260
12 16 20 12 16 20 (min)

(2)
RHM2 (D96N)
UDP-Glc
RHM2 (G193R)
UDP-Glc
Intensity A260
12 16 20 12 16 20 (min)

(3)
RHM2 (G18A)
UDP-Glc
RHM2 (K36A)
UDP-Glc
RHM2 (K165A)
UDP-Glc
UDP-Rha
UDP-4K6DG
Intensity A260
12 16 20 12 16 20 12 16 20 (min)

(4)
RHM2 (G392A)
UDP-Glc
UDP-4K6DG
RHM2 (K413A)
UDP-Rha
UDP-Glc
UDP-4K6DG
RHM2 (K518A)
UDP-Rha
UDP-Glc
UDP-4K6DG
Intensity A260
12 16 20 12 16 20 12 16 20 (min)

5398 JOURNAL OF BIOLOGICAL CHEMISTRY
VOLUME 282 • NUMBER 8 • FEBRUARY 23, 2007

Characterization of Plant UDP-L-rhamnose Synthase in A. thaliana

tions of RHM2/MUM4 influence UDP-Rha synthase activity, we first constructed plasmids for expression of D96N and G193R mutant versions of RHM2/MUM4. Comparison of RHM2-N and RHM2-C sequences revealed that both domains belong to the short chain dehydrogenase reductase family, have a Rossmann fold (GXXGXX(G/A)) for NAD(P)(H) binding (20) in their N-terminal regions, and have two sets of conserved YXXXK motifs, a motif implicated as the catalytic site in short chain dehydrogenase/reductase family proteins (21) (Fig. 5A). To confirm the importance of these amino acid residues, we also constructed several point mutation versions of RHM2/ MUM4, including G18A, K36A, K165A, G392A, K413A, and K518A (Fig. 5A). All eight mutant constructs were expressed in Δprb1 cells and detected with anti-penta-HIS. The Pgk1p signal served as a loading control for cytosolic proteins. The amount of G193R(MUM4-2) mutant protein present in the extract was significantly lower than that of wild-type RHM2 (Fig. 5B, lanes 2 and 7), and the G18A and G392A mutants, which affect the motif for NAD(P)(H) binding in RHM2-N and RHM2-C, respectively, were also very low or absent (Fig. 5B, lanes 3 and 8), suggesting that the mutant proteins are significantly less stable than wild-type RHM2 (Fig. 5B, lane 2). For the remaining mutant constructs we tested (K36A, D96N or MUM4-1, K165A, K413A, and K518A), no remarkable decrease in protein levels were observed (Fig. 5B, lanes 2, 4–6, 9, and 10).

Next, we measured UDP-Rha synthase activity of the mutant proteins using equal amounts of cytosolic fractions as a source of crude enzyme. D96N (MUM4-1) and G193R (MUM4-2) completely abolish UDP-Glc 4,6-dehydratase activity (Fig. 5C, panel 2), providing strong evidence that the MUM4-1 and MUM4-2 phenotypes in A. thaliana (11) are due to abolishment of UDP-Rha synthesis activity. Interestingly, the G193A (MUM4-2) mutation affected not only enzyme activity but also stability of the protein. G18A and K165A also completely abolish UDP-Glc 4,6-dehydratase activity, indicating that Gly<sup>18</sup> and Lys<sup>165</sup> are essential for catalytic conversion of UDP-Glc to UDP-4K6DG (Fig. 5C, panel 3). Activity of the K36A mutant protein was lower than that of the wild-type protein, suggesting that Lys<sup>36</sup> is also important for robust UDP-Glc 4,6-dehydratase activity (Fig. 5C, panel 3). Because G18A, K36A, D96N, K165A, and G193R are located within the RHM2-N domain, these mutations almost certainly exert their influence on the UDP-Glc 4,6-dehydratase activity of RHM2/MUM4. Indeed, when UDP-4K6DG 3,5-epimerase and UDP-4KR 4-keto-reductase activity were assayed using cytoplasmic fractions from strains expressing these mutant proteins, the levels of these activities were indistinguishable from those of wild-type RHM2/MUM4 (data not shown). On the other hand, G392A, K413A, and K518A mutants are able to convert UDP-Glc to UDP-4K6DG (Fig. 5C, panel 4). Because G392A, K413A, and K518A are located in the RHM2-C domain, it is reasonable to suppose that these mutations do not affect UDP-Glc 4,6-dehydratase activity. Taken together, these results provide additional evidence that the RHM2-N and RHM2-C domains of RHM2/MUM4 can function independently.

Different changes in the RHM2 C-terminal region had different effects on activity. The G392A mutation completely abolished RHM2-C activities (Fig. 5C, panel 4), whereas the K413A mutation had only a modest effect on RHM2-C activity and K518A caused a decrease in RHM2-C activity as compared with wild-type RHM2 (Fig. 5C, panel 4). Based on these results, we concluded that alteration of the NAD(P)(H) binding motif in either domain (G18A and G392A) abolishes enzymatic activity of that domain (Fig. 5C, left panels of 3 and 4) and moreover, causes a decrease in the overstabiltiy of the protein (Fig. 5B, lanes 3 and 8). These data suggest that binding of a co-factor is important not only for activity but also for the stability of RHM2 and furthermore, that K36A, K165A, and K518A have deleterious effects on the activity of the corresponding domains they affect. Despite the fact that the residue is conserved among RHM1, RHM2/MUM4, and RHM3 proteins, the K413A mutation did not affect the activity of RHM2/MUM4. The Lys<sup>413</sup> residue is conserved among RHM1, RHM2/MUM4, and RHM3 but not in UER1 proteins, and it seems likely that Lys<sup>413</sup> has been replaced by Ser in UER1. We were able to confirm a report by Watt et al. (8) indicating that the UER1 protein has UDP-4K6DG 3,5-epimerase and UDP-4KR 4-keto-reductase activities (Fig. 3C, panel 7), providing evidence that Lys<sup>413</sup> is not important for RHM2/MUM4 activity.

Kinetic Analysis of the RHM2-N Protein Fragment—Next, we focused on the function of the RHM2-N protein fragment. Overexpression of RHM2-N had a negative effect on yeast cell growth and protein expression (Fig. 3B, lane 2); however, these defects can be overcome by co-expression of the RHM2-C gene fragment or of the UER1 gene (Fig. 3B, lanes 4 and 6). Thus, we expressed RHM2-N with a His<sub>6</sub> tag and RHM2-C without a His<sub>6</sub> tag in Δprb1 cells and then purified RHM2-N protein using Ni-NTA-agarose. Using this approach, we were able to obtain ~4.5 mg of the recombinant protein from a 6-liter culture. A single 42-kDa band was observed with Coomassie Brilliant Blue (Fig. 6A).

FIGURE 5. Site-directed mutagenic analysis of the RHM2 protein. A, the location of single amino acid changes in the RHM2 proteins that were tested. B, immunoblot analysis of mutant RHM2 proteins. Mutant RHM2 proteins were detected as a 75-kDa protein with anti-penta-HIS. Cell lysate from 0.25 A<sub>600</sub> of yeast cells was loaded on each lane (lanes 1–10). Pgk1p was used as a loading control for cytosolic proteins. Lane 1, control BY4741Δprb1 cells; lane 2, BY4741Δprb1 cells with the wild-type His<sub>6</sub>-tagged RHM2 expression plasmid. Lanes 3–10, BY4741Δprb1 cells with the expression plasmids for His<sub>6</sub>-tagged RHM2 (G18A) (lane 3), His<sub>6</sub>-tagged RHM2 (K36A) (lane 4), His<sub>6</sub>-tagged RHM2 (D96N) (lane 5), His<sub>6</sub>-tagged RHM2 (K165A) (lane 6), His<sub>6</sub>-tagged RHM2 (G193R) (lane 7), His<sub>6</sub>-tagged RHM2 (G392A) (lane 8), His<sub>6</sub>-tagged RHM2 (K413A) (lane 9), and His<sub>6</sub>-tagged RHM2 (K518A) (lane 10). C, in vitro analysis of UDP-Rha synthase activity. Enzyme activities were assayed as described under “Experimental Procedures.” 7.5 A<sub>600</sub> of yeast cell lysates was used as crude enzyme. Panel 1, BY4741Δprb1 cells without plasmid as a negative control (left panel) and BY4741Δprb1 cells with the wild type His<sub>6</sub>-tagged RHM2 expression plasmid (right panel). Panel 2, the MUM4-1 and MUM4-2 mutations. BY4741Δprb1 cells harboring expression plasmid for His<sub>6</sub>-tagged RHM2 (D96N; left panel) and BY4741Δprb1 cells harboring expression plasmid for His<sub>6</sub>-tagged RHM2 (G193R; right panel). Panel 3, mutations affecting the RHM2-N domain. BY4741Δprb1 cells with the mutations G18A (left panel), K36A (middle panel), or K165A (right panel). Panel 4, mutations affecting the RHM2-C domain. BY4741Δprb1 cells harboring expression plasmids for His<sub>6</sub>-tagged RHM2 with mutations G392A (left panel), K413A (middle panel), or K518A (right panel). Representative results from three independent experiments are shown.

Characterization of Plant UDP- _L_ -rhamnose Synthase in A. thaliana

A

B

C

was fairly stable over the pH range 7.0 to 8.0 (Fig. 6B). Enzyme activity was also assayed at different temperatures. The optimal temperature for activity was 35 °C and the data suggest that the enzyme is relatively active in the temperature range 30 to 40 °C (Fig. 6C).

To address substrate specificity and turnover of RHM2, we compared the \( K_m \) and \( k_{cat} \) of UDP-Glc and dTDP-Glc in the presence of the protein. Substrate saturation kinetics in the presence of His₆-tagged RHM2-N were determined using the UDP-Glc 4,6-dehydratase assay conditions, in which the concentrations of UDP-Glc (linearity \( r^2 = 0.995 \)) and dTDP-Glc (linearity \( r^2 = 0.999 \)) are between 30 and 1000 or 100 and 1000 μM, respectively. Kinetics curves were drawn using the Lineweaver-Burk triple-reciprocal plot method. \( K_m \) values were also determined and the apparent \( K_m \) values of UDP-Glc and dTDP-Glc were 116 and 680 μM, respectively (Table 3). Moreover, the \( k_{cat} \) values for UDP-Glc and dTDP-Glc were \( 2.86 \times 10^3 \) and \( 1.73 \times 10^2 \, s^{-1} \), respectively (Table 3). These properties are opposite those of bacterial RmlB, suggesting that unlike Rm1B, RHM2 prefers UDP-Glc to dTDP-Glc as a substrate.

FIGURE 6. Purification and analysis of His₆-tagged RHM2-N protein. A, expression and purification of recombinant His₆-tagged RHM2-N. The proteins were extracted from the recombinant yeast cells and separated by 4–20% SDS-PAGE, and the gel was stained with Coomassie Brilliant Blue. Lane 1, crude protein fraction from the recombinant yeast cells; lane 2, purified protein fraction (42 kDa as His₆-tagged RHM2-N, arrow); lane 3, protein molecular markers shown in the right side in numbers (kDa). B, effects of pH on enzymatic activity of His₆-tagged RHM2-N. 100% corresponds to the incorporation of \( 4.4 \times 10^{-2} \) unit (nmol/min/μg) at the condition of pH 7.5 (MOPS-NaOH). C, effects of temperature on enzymatic activity of His₆-tagged RHM2-N. 100% corresponds to the incorporation of \( 5.2 \times 10^{-2} \) unit (nmol/min/μg) at 35 °C.

TABLE 3  
Kinetic analyses of RHM2-N protein for UDP-Glc and dTDP-Glc  

| RHM-N (A. thaliana) | UDP-glucose | dTDP-glucose |
|---------------------|--------------|---------------|
| Specific activity   | \( 1.68 \times 10^{-1} \) | \( 1.61 \times 10^{-2} \) |
| (nmol/min/μg of protein) | | |
| \( K_m \) (μM)      | 116          | 680           |
| \( V_{max} \) (M)   | \( 6.01 \times 10^{-5} \) | \( 7.26 \times 10^{-5} \) |
| \( K_{cat} \) (s⁻¹) | \( 2.86 \times 10^3 \)    | \( 1.73 \times 10^2 \)     |

Inhibition of RHM2-N—We next asked if the activity of RHM2-N is inhibited by various sugar-nucleotides and related chemicals (Table 4). The activity of the purified His₆-tagged RHM2-N protein was inhibited by UDP-Rha, UDP-Xyl, UDP, and UMP. UDP-Rha is a final product of UDP-Glc conversion by RHM2 enzymatic reaction, and when 1 mM UDP-Rha was used as an inhibitor, enzyme activity was dramatically inhibited (18.1% of the value for the no inhibitor control; Table 4). Interestingly, 1 mM UDP-Xyl was also able to inhibit activity to a comparable level (20.9% of the value for the no inhibitor control; Table 4). However, 0.1 mM UDP-Rha is more efficient than 0.1 mM UDP-Xyl (80.4 versus 92.3% of no inhibitor control; Table 4). RHM2-N activity was not affected by other sugar-nucleotides we tested; namely, 1 mM of UDP-Ara, UDP-GalNAc, UDP-GlcNAc, UDP-GlcA, GDP-Man, GDP-Fuc, or CMP-Sia. These results lead us to conclude that RHM2-N activity is regulated by the concentration of UDP-Rha and UDP-Xyl in the cytoplasmic compartment. Some types of nucleotides (namely, UDP and UMP, tested at 1 mM levels) also inhibited the activity of RHM2-N but the same amounts of UTP or D-glucose did not (Table 4). This suggests that UDP and UMP can bind to the UDP-binding site and thus inhibit RHM2-N function, perhaps because these compounds are

For S. enterica RmlB, RmlC, and RmlD, enzymatic activity toward UDP-Glc was lower than 1% of the activity toward dTDP-Glc under the same assay conditions (25), indicating that bacterial Rml proteins preferentially convert dTDP-Glc to dTDP-4K6DG and not UDP-Glc to UDP-4K6DG (25). However, because the properties and substrate specificity of 4,6-dehydratase of RHM2 are poorly understood, we determined the optimal pH, temperature, and kinetic parameters for recombinant RHM2-N. To do this, we first monitored enzyme activity over a range of pH values. The highest level of enzyme activity was observed in 250 mM MOPS-NaOH buffer, pH 7.5. In 250 mM Tris-HCl buffer, pH 7.5, the relative activity was lower than that observed for MOPS-NaOH buffer at the same pH. Enzyme activity was reduced when the pH was shifted below or above the optimal level of pH 7.5; however, activity

Characterization of Plant UDP-L-rhamnose Synthase in A. thaliana

smaller than UDP-Glc; whereas UTP, which is larger than UDP, did not inhibit the activity of RHM2-N.

DISCUSSION

In this study, we used a yeast expression system to provide biochemical evidence that *RHM1*, *RHM2/MUM4*, and *RHM3* encode proteins that can synthesize UDP-Rha from UDP-Glc. This is the first report of genes that function in the *de novo* UDP-Rha synthetic pathway.

We succeeded in expressing functional, full-length RHM proteins using protease-deficient yeast mutants (*Δpep4* and *Δprb1*) and showed that RHM1, RHM2, and RHM3 act as a 75-kDa protein with three distinct enzymatic activities that together convert UDP-Glc to UDP-Rha. Pep4p and Prblp are well-studied vacuolar proteases involved in maturation and activation of other intracellular proteases, such that levels of intracellular protease activities are lower in these mutants than in wild-type cells (26). However, it has been unclear if the cytosolic RHM proteins are transported to the vacuole and then degraded by various vacuolar proteases or if overexpression of RHM proteins under the *TDH3* (*GAP*) promoter leads to degradation by vacuolar-derived proteases during cell lysate preparation. UDP-Rha is utilized by several glycosyltransferases to synthesize cell wall polysaccharides, which consist of RG-I, RG-II (1), and various L-rhamnose-containing natural compounds in plants (2), suggesting that UDP-Rha is essential to plant cells. However, little is known about the responsible glycosyltransferases, as UDP-Rha is a novel sugar nucleotide that is not commercially available. Recently, bioinformatics-based analysis of several genome sequences revealed the presence of many glycosyltransferase-like genes in the genomes of diverse species, including *A. thaliana* (27). It should be possible to identify genes that encode proteins with rhamnosyltransferase activity from among these candidate genes whenever UDP-Rha can easily be obtained. Thus, our yeast system will be useful to produce UDP-Rha that can be used to address the function of putative rhamnosyltransferases and to explore the molecular mechanisms of the synthesis of plant cell wall polysaccharides containing L-rhamnose in future.

We also provide evidence that the N-terminal and C-terminal regions of RHM2/MUM4 are functionally independent. The N-terminal domain of RHM2/MUM4 (RHM2-N; amino acids 1–370) acts as a UDP-Glc 4,6-dehydratase and the C-terminal domain of RHM2/MUM4 (RHM2-C; amino acids 371–667) acts as a bi-functional domain, consisting of UDP-4K6DG 3,5-epimerase and UDP-4KR 4-keto-reductase activities. It is interesting to ask why the two domains were fused in plant RHM proteins. Notably, the enzymatic reactions performed by the RHMs are similar to conversion of GDP-Man to GDP-Fuc by GMD2/MUR1 and GER1/AtFx, as GDP-Fuc is synthesized from GDP-Man by GDP-Man 4,6-dehydratase, GDP-4-keto-6-deoxy-D-mannose 3,5-epimerase, and GDP-4-keto-L-fucose 4-keto-reductase (28, 29). Whereas these genes are present as independent open reading frames in *A. thaliana*, RHM2/MUM4 is a fused single open reading frame consisting of 4,6-hydratase domain and 3,5-epimerase/4-keto-reductase domains. We previously reported that GMD2/MUR1 and GER1/AtFx proteins interact to stabilize enzymatic activity (18). Therefore, it is possible that the two domains of RHM2 pro-

TABLE 4  
Inhibitory effect of compounds on UDP-Glc 4,6-dehydratase (RHM2-N protein)

Values are percentages of UDP-Glc 4,6-dehydratase activity from $1.86 \times 10^{-1}$ (nmol/min/$\mu$g of protein) for “no inhibitor.” The amounts of UDP-4K6DG were calculated from peak areas and expressed as the amounts of UDP-glucose equivalent, obtained by the triplicate experiments.

| Compound                          | Relative activity (%) |
|-----------------------------------|-----------------------|
| No inhibitor                      | $100 \pm 1.8$         |
| $1 \mathrm{mM} \mathrm{UDP}-\mathrm{L}$-rhamnose | $18.1 \pm 0.6$        |
| $0.1 \mathrm{mM} \mathrm{UDP}-\mathrm{L}$-rhamnose | $80.4 \pm 2.8$        |
| $1 \mathrm{mM} \mathrm{UDP}-\mathrm{D}$-xylose    | $20.9 \pm 0.9$        |
| $0.1 \mathrm{mM} \mathrm{UDP}-\mathrm{D}$-xylose   | $92.3 \pm 2.3$        |
| $1 \mathrm{mM} \mathrm{UDP}$                  | $59.0 \pm 0.8$        |
| $0.1 \mathrm{mM} \mathrm{UDP}$                 | $94.9 \pm 1.9$        |
| $1 \mathrm{mM} \mathrm{UTP}$                   | $95.9 \pm 1.6$        |
| $1 \mathrm{mM} \mathrm{UMP}$                   | $76.6 \pm 0.8$        |
| $1 \mathrm{mM} \mathrm{D}$-glucose             | $98.2 \pm 0.7$        |
| $1 \mathrm{mM} \mathrm{UDP}-\mathrm{L}$-arabinose | $92.0 \pm 6.19$       |
| $1 \mathrm{mM} \mathrm{UDP}-\mathrm{D}$-galactose | $104.7 \pm 2.4$       |
| $1 \mathrm{mM} \mathrm{UDP}-N$-acetyl-$\mathrm{D}$-galactosamine | $102.4 \pm 2.5$       |
| $1 \mathrm{mM} \mathrm{UDP}-N$-acetyl-$\mathrm{D}$-glucosamine | $102.7 \pm 4.7$       |
| $1 \mathrm{mM} \mathrm{UDP}-\mathrm{D}$-glucuronic acid | $102.3 \pm 0.3$       |
| $1 \mathrm{mM} \mathrm{GDP}-\mathrm{D}$-mannose | $107.9 \pm 4.7$       |
| $1 \mathrm{mM} \mathrm{GDP}-\mathrm{L}$-fucose | $104.9 \pm 2.1$       |
| $1 \mathrm{mM} \mathrm{CMP}-\mathrm{D}$-sialic acid | $104.3 \pm 0.9$       |

FIGURE 7. Regulation of UDP-sugars by feedback inhibition in the cytoplasm of plant cells. RHM2(N), the N-terminal domain of UDP-Rha synthase. UDP-Rha(C), C-terminal domain of UDP-Rha synthase. UER1, UDP-4K6DG 3,5-epimerase and UDP-4KR 4-keto-reductase. UGE, UDP-Glc 4-epimerase (8). UGD, UDP-Glc dehydrogenase (33). sUXS, soluble type UDP-GlcA decarboxylase (34). mUXS, membrane type UDP-GlcA decarboxylase (34, 35). UXE, UDP-Xyl 4-epimerase (37). AXS, UDP-Api and UDP-Xyl synthase (38). UGE, UDP-Glc 4-epimerase (40). GAE, UDP-GlcA 4-epimerase (41, 42).

FEBRUARY 23, 2007 • VOLUME 282 • NUMBER 8                                                                 JOURNAL OF BIOLOGICAL CHEMISTRY 5401
tein must be close to accelerate a reaction. Indeed, slow growth and cell aggregation phenotypes were observed in RHM2-N overexpressing yeast cells. This is probably due to a dominant negative effect caused by production of excess UDP-4K6DG, as co-expression of either RHM2-C or UER1 with RHM2-N diminished the defects (data not shown). It seems plausible that there was selective pressure to fuse the domains, as this would stimulate the reactions and prevent accumulation of toxic 4-keto intermediates.

Our functional domain analysis of RHM2/MUM4 and chemical structure analysis of the reaction products revealed that RHM2-C is responsible for 3,5-epimeration and, like UER1, 4-keto reduction of UDP-4K6DG and UDP-4KR (8). The large number of matching sequences in an expressed sequence tag database (7, 9) and the massively parallel signature sequencing data of *A. thaliana* (30) suggest that UER1 is abundantly expressed and thus, may compensate for lagging function of the C-terminal domains of RHM proteins such that over-accumulation of 4-keto intermediates is avoided. Indeed, a remnant of the reaction product (UDP-4K6DG) was observed in an *in vitro* assay of UDP-Rha synthesis. Thus it is also likely that UER1 supports the function of the RHM2-C domain by mediating accumulation of 4-keto intermediates.

We purified His~6~-tagged RHM2-N protein and analyzed the properties and kinetics of this enzyme. Fig. 7 shows an abbreviated model of regulation of UDP-sugars in the cytoplasm of plant cells. An inhibition analysis revealed that UDP-Rha inhibits conversion of 1 mM UDP-Glc by RHM2-N even at a low concentration (0.1 mM). We therefore propose the presence of a feedback mechanism that maintains the UDP-sugar pool *in vivo* by inhibiting UDP-Rha synthesis. It is known that GDP-Man 4,6-dehydratase is strongly inhibited by the final product GDP-Fuc, indicating that GDP-Man 4,6-dehydratase may help maintain intracellular levels of GDP-Man (31). The final product UDP-Rha may similarly inhibit UDP-Glc 4,6-dehydratase activity of RHM2/MUM4.

Interestingly, the UDP-Glc 4,6-dehydratase activity of RHM2/MUM4 was inhibited by UDP-Xyl to a degree comparable with inhibition by UDP-Rha. This suggests that UDP-Xyl also regulates RHM2/MUM4 activity *in vivo*. UDP-Xyl is a final product of conversion of UDP-Glc to UDP-Xyl via UDP-GlcA in the plant cytoplasm (22, 32–36). UDP-Xyl also inhibits its related enzyme activities (UDP-Glc dehydrogenase, UDP-Glc pyrophosphorylase, and UDP-GlcA decarboxylase) (34). Such a feedback inhibition loop could regulate conversion of UDP-Glc to UDP-GlcA, UDP-Xyl, UDP-Ara, UDP-GalA, and UDP-D-apiose (UDP-Api) in plants, suggesting that UDP-Xyl plays a key role to regulate the overall UDP-sugar pool. As it is important for plant cells to maintain a ready supply of UDP-Glc, UDP-Xyl may inhibit not only its own synthetic pathway but also the production of UDP-Rha from UDP-Glc. However, further investigation will be required to better understand the significance of inhibition and feedback.

Unlike UDP-Rha and UDP-Xyl, UDP-Ara did not inhibit RHM2-N activity, despite the fact that UDP-Ara is synthesized by 4-epimeration of UDP-Xyl (UDP-Xyl 4-epimerase) (37). The enzymes (UDP-Xyl 4-epimerase) that convert UDP-Xyl to UDP-Ara are type II membrane proteins, suggesting these reactions occur on the lumenal side of the Golgi apparatus (37). Because UDP-Ara is mainly localized in the Golgi lumen and not in the cytoplasm, there may be no reason for inhibitory regulation by UDP-Ara.

It will be interesting to address whether RHM2-N is inhibited by another final product, UDP-Api, but the fact that UDP-Api is not commercially available at present makes it difficult to test. However, there is less D-apiose in the cell wall than there is L-rhamnose or D-xylose (38), suggesting that less UDP-Api is produced and needed in the cytoplasm. Therefore, the effect of UDP-Api on RHM2-N activity may not be a strong one. Additionally, we cannot exclude the possibility that UDP-Rha inhibits another *de novo* pathway, such as via inhibition of UGD. It has been reported that a UDP-Glc pyrophosphorylase catalyzes formation of various UDP-sugars from monosaccharide 1-phosphates at the end of the salvage pathways in higher plants (39); however, a salvage pathway for UDP-Rha has not yet been identified. Further studies are required before we will fully understand the mechanisms that control regulation of the UDP-sugar pool in plant cells.

Acknowledgments—We thank Dr. Ko Hayama for ESI-MS analysis. We thank Drs. Yasunori Chiba and Morihisa Fujita, and Hiroto Hirayama and Toshihiko Kitajima for many helpful discussions. We are indebted to Drs. Takehiko Yoko-o, Yoh-ichi Shimma, Xiao-Dong Gao, and Michiyo Okamoto for stimulating discussions.

---

**REFERENCES**

1. Ridley, B. L., O’Neill, M. A., and Mohnen, D. (2001) *Phytochemistry* **57**, 929–967
2. Ikan, R. (1999) *Naturally Occurring Glycosides*, Wiley, Chichester, UK
3. O’Neill, M. A., Warrenfeltz, D., Kates, K., Pellerin, P., Doco, T., Darvill, A. G., and Albersheim, P. (1996) *J. Biol. Chem.* **271**, 22923–22930
4. Dong, C., Beis, K., Giraud, M. F., Blankenfeldt, W., Allard, S., Major, L. L., Kerr, I. D., Whitfield, C., and Naismith, J. H. (2003) *Biochem. Soc. Trans.* **31**, 532–536
5. Jiang, X. M., Neal, B., Santiago, F., Lee, S. J., Romana, L. K., and Reeves, P. R. (1991) *Mol. Microbiol.* **5**, 695–713
6. Kamsteeg, J., Van Brederode, J., and Van Nigtevecht, G. (1978) *FEBS Lett.* **91**, 281–284
7. Seifert, G. J. (2004) *Curr. Opin. Plant Biol.* **7**, 277–284
8. Watt, G., Leoff, C., Harper, A. D., and Bar-Peled, M. (2004) *Plant Physiol.* **134**, 1337–1346
9. Reiter, W. D., and Vanzin, G. F. (2001) *Plant Mol. Biol.* **47**, 95–113
10. Usadel, B., Kuschinsky, A. M., Rosso, M. G., Eckermann, N., and Pauly, M. (2004) *Plant Physiol.* **134**, 286–295
11. Western, T. L., Young, D. S., Dean, G. H., Tan, W. L., Samuels, A. L., and Haughn, G. W. (2004) *Plant Physiol.* **134**, 296–306
12. Diet, A., Link, B., Seifert, G. J., Schellenberg, B., Wagner, U., Pauly, M., Reiter, W. D., and Ringli, C. (2006) *Plant Cell* **18**, 1630–1641
13. Brachmann, C. B., Davies, A., Cost, G. J., Caputo, E., Li, J., Hieter, P., and Boeke, J. D. (1998) *Yeast* **14**, 115–132
14. Sherman, F. (1991) *Methods Enzymol.* **194**, 3–21
15. Gietz, D., St. Jean, A., Woods, R. A., and Schiestl, R. H. (1992) *Nucleic Acids Res.* **20**, 1425
16. Seki, M., Carninci, P., Nishiyama, Y., Hayashizaki, Y., and Shinozaki, K. (1998) *Plant J.* **15**, 707–720
17. Seki, M., Narusaka, M., Kamiya, A., Ishida, J., Satou, M., Sakurai, T., Nakajima, M., Enju, A., Akiyama, K., Oono, Y., Muramatsu, M., Hayashizaki, Y., Kawai, J., Carninci, P., Itoh, M., Ishii, Y., Arakawa, T., Shibata, K., Shinagawa, A., and Shinozaki, K. (2002) *Science* **296**, 141–145
18. Nakayama, K., Maeda, Y., and Jigami, Y. (2003) *Glycobiology* **13**, 673–680

5402 JOURNAL OF BIOLOGICAL CHEMISTRY
VOLUME 282 • NUMBER 8 • FEBRUARY 23, 2007
Characterization of Plant UDP- _L_ -rhamnose Synthase in A. thaliana

19. Hill, J. E., Myers, A. M., Koerner, T. J., and Tzagoloff, A. (1986) *Yeast* 2, 163–167
20. Rossmann, M. G., and Argos, P. (1978) *Mol. Cell. Biochem.* 21, 161–182
21. Duax, W. L., Ghosh, D., and Pletnev, V. (2000) *Vitam. Horm.* 58, 121–148
22. Oka, T., and Jigami, Y. (2006) *FEBS J.* 273, 2645–2657
23. Kang, Y. B., Yang, Y. H., Lee, K. W., Lee, S. G., Sohng, J. K., Lee, H. C., Liou, K., and Kim, B. G. (2006) *Biotechnol. Bioeng.* 93, 21–27
24. Nakamura, Y., Gojobori, T., and Ikemura, T. (2000) *Nucleic Acids Res.* 28, 292
25. Graninger, M., Nidetzky, B., Heinrichs, D. E., Whitfield, C., and Messner, P. (1999) *J. Biol. Chem.* 274, 25069–25077
26. Jones, E. W. (1991) *Methods Enzymol.* 194, 429–453
27. Egelund, J., Skjot, M., Geshi, N., Ulvskov, P., and Petersen, B. L. (2004) *Plant Physiol.* 136, 2609–2620
28. Bonin, C. P., Potter, I., Vanzin, G. F., and Reiter, W. D. (1997) *Proc. Natl. Acad. Sci. U. S. A.* 94, 2085–2090
29. Bonin, C. P., and Reiter, W. D. (2000) *Plant J.* 21, 445–454
30. Meyers, B. C., Lee, D. K., Vu, T. H., Tej, S. S., Edberg, S. B., Matvienko, M., and Tindell, L. D. (2004) *Plant Physiol.* 135, 801–813
31. Somoza, J. R., Menon, S., Schmidt, H., Joseph-McCarthy, D., Dessen, A.,
32. Stahl, M. L., Somers, W. S., and Sullivan, F. X. (2000) *Structure* 8, 123–135
33. Turner, W., and Botha, F. C. (2002) *Arch. Biochem. Biophys.* 407, 209–216
34. Tenhaken, R., and Thulke, O. (1996) *Plant Physiol.* 112, 1127–1134
35. Harper, A. D., and Bar-Peled, M. (2002) *Plant Physiol.* 130, 2188–2198
36. Pattathil, S., Harper, A. D., and Bar-Peled, M. (2005) *Planta* 221, 538–548
37. Bindschedler, L. V., Wheatley, E., Gay, E., Cole, J., Cottage, A., and Bolwell, G. P. (2005) *Plant Mol. Biol.* 57, 285–301
38. Burget, E. G., Verma, R., Molhoj, M., and Reiter, W. D. (2003) *Plant Cell* 15, 523–531
39. Ahn, J. W., Verma, R., Kim, M., Lee, J. Y., Kim, Y. K., Bang, J. W., Reiter, W. D., and Pai, H. S. (2006) *J. Biol. Chem.* 281, 13708–13716
40. Kotake, T., Yamaguchi, D., Ohzono, H., Hojo, S., Kaneko, S., Ishida, H. K., and Tsumuraya, Y. (2004) *J. Biol. Chem.* 279, 45728–45736
41. Barber, C., Rosti, J., Rawat, A., Findlay, K., Roberts, K., and Seifert, G. J. (2006) *J. Biol. Chem.* 281, 17276–17285
42. Molhoj, M., Verma, R., and Reiter, W. D. (2004) *Plant Physiol.* 135, 1221–1230
43. Usadel, B., Schluter, U., Molhoj, M., Gipmans, M., Verma, R., Kossmann, J., Reiter, W. D., and Pauly, M. (2004) *FEBS Lett.* 569, 327–331

Glycobiology and Extracellular Matrices: Functional Analysis of Arabidopsis thaliana RHM2/MUM4, a Multidomain Protein Involved in UDP-D-glucose to UDP-L-rhamnose Conversion

Takuji Oka, Tadashi Nemoto and Yoshifumi Jigami

J. Biol. Chem. 2007, 282:5389-5403.  
doi: 10.1074/jbc.M610196200 originally published online December 26, 2006

---

Access the most updated version of this article at doi: [10.1074/jbc.M610196200](http://dx.doi.org/10.1074/jbc.M610196200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 41 references, 20 of which can be accessed free at  
[http://www.jbc.org/content/282/8/5389.full.html#ref-list-1](http://www.jbc.org/content/282/8/5389.full.html#ref-list-1)
