
THE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 282, NO. 11, pp. 7790–7798, March 16, 2007  
© 2007 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in the U.S.A.

# Modification of Lipopolysaccharide with Colanic Acid (M-antigen) Repeats in *Escherichia coli*

Received for publication, November 30, 2006, and in revised form, January 4, 2007 Published, JBC Papers in Press, January 16, 2007, DOI 10.1074/jbc.M611034200

**Timothy C. Meredith**, **Uwe Mamat**, **Zbigniew Kaczynski**, **Buko Lindner**, **Otto Holst**, and **Ronald W. Woodard**

From the †Department of Medicinal Chemistry, University of Michigan, Ann Arbor, Michigan 48109, the §Division of Structural Biochemistry and ||Division of Immun chemistry, Research Center Borstel, Leibniz Center for Medicine and Biosciences, D-23845 Borstel, Germany, and the ¶Faculty of Chemistry, University of Gdansk, PL-80952 Gdansk, Poland

Colanic acid (CA) or M-antigen is an exopolysaccharide produced by many enterobacteria, including the majority of *Escherichia coli* strains. Unlike other capsular polysaccharides, which have a close association with the bacterial surface, CA forms a loosely associated saccharide mesh that coats the bacteria, often within biofilms. Herein we show that a highly mucoid strain of *E. coli* K-12 ligates CA repeats to a significant proportion of lipopolysaccharide (LPS) core acceptor molecules, forming the novel LPS glycoform we call $M_{\mathrm{LPS}}$. $M_{\mathrm{LPS}}$ biosynthesis is dependent upon (i) CA induction, (ii) LPS core biosynthesis, and (iii) the O-antigen ligase WaaL. Compositional analysis, mass spectrometry, and nuclear magnetic resonance spectroscopy of a purified $M_{\mathrm{LPS}}$ sample confirmed the presence of a CA repeat unit identical in carbohydrate sequence, but differing at multiple positions in anomeric configuration and linkage, from published structures of extracellular CA. The attachment point was identified as O-7 of the L-glycero-D-manno-heptose of the outer LPS core, the same position used for O-antigen ligation. When O-antigen biosynthesis was restored in the K-12 background and grown under conditions meeting the above specifications, only $M_{\mathrm{LPS}}$ was observed, suggesting *E. coli* can reversibly change its proximal covalently linked cell surface polysaccharide coat from O-antigen to CA in response to certain environmental stimuli. The identification of $M_{\mathrm{LPS}}$ has implications for potential underlying mechanisms coordinating the synthesis of various surface polysaccharides.

Enteric bacteria synthesize and display a complex array of various cell surface polysaccharides. There can be at least six distinct saccharide polymers simultaneously present within the glycocalyx of a typical strain of *Escherichia coli*. At present, the known components of the saccharide matrix include lipopolysaccharide (LPS)² O-antigens (1), enterobacterial common antigen (ECA) (2), capsular polysaccharides (K-antigen) (3), poly β-1,6-N-acetyl-D-glucosamine (PNAG) (4), the β-1,4-glucan polymer bacterial cellulose (5), and colanic acid (CA or M-antigen) (6). The tremendous diversity within the serotype specific repeat units [~170 O-antigens, ~80 K-antigens (7)], coupled with regulation of expression levels between polysaccharide classes, facilitates the expression of a multitude of glycocalyx compositions. For each strain and/or given growth environment, a number of cell coat polysaccharide states can be sampled to attain a balance that is suitable for a particular niche.

The network of *E. coli* surface polysaccharides can be further subdivided into those that are tightly associated or covalently linked to the outer membrane (OM) (O/K-antigens, ECA) and those that are loosely associated, called exo- or slime polysaccharides. The exopolysaccharides (in particular CA and PNAG (8)) are integral components of biofilms, acting as the “cement,” which holds together the various protein, lipid, and polysaccharide components (9). The vast majority of CA is secreted into this extracellular milieu, with no evidence existing of a lipid anchor tethering CA chains to the OM. Disruption of the putative acetyl transferase gene *wcaF* from the CA gene cluster in *E. coli* K-12 severely curtailed the maturation and development of the complex three-dimensional architecture typically associated with robust biofilms (10).

CA is a polyanionic heteropolysaccharide containing a repeat unit with D-glucose, L-fucose, D-galactose, and D-glucuronic acid sugars that are nonstoichiometrically decorated with O-acetyl and pyruvate side chains (6, 11). The CA polysaccharide repeat is assembled on the membrane lipid undecaprenol pyrophosphate (Und-PP) by a series of glycosyl transferases on the cytoplasmic face of the inner membrane, after which the single repeat is flipped to the periplasmic side and polymerized by the Wzy-dependent pathway (reviewed in Refs. 1 and 3). Subsequently, the polymer is presumably cleaved from the Und-PP anchor, transported across the periplasm, and excreted into the extracellular space in a poorly understood process. The genetic determinant for CA biosynthesis resides on the 19 gene *wca* (*cps*) cluster (12, 13) and is tightly regulated by a complex signal transduction cascade governed by the *rcs* (regulator of capsule synthesis) phosphorelay system (14, 15). While nor-

* This work was supported by National Institutes of Health Grant R01 AI61531 (to R. W. W.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked "advertisement" in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The on-line version of this article (available at http://www.jbc.org) contains supplemental Tables S1–S4.
* To whom correspondence should be addressed: College of Pharmacy, 428 Church St., Ann Arbor, MI 48109-1065. Tel.: 734-764-7366; Fax: 734-763-2022; E-mail: rww@umich.edu.
* The abbreviations used are: LPS, lipopolysaccharide; Kdo, 3-deoxy-D-manno-oct-2-ulosonic acid; A5P, D-arabinose 5-phosphate; OM, outer membrane; ECA, enterobacterial common antigen; CA, colanic acid; OS, oligosaccharide; Und-PP, undecaprenol pyrophosphate; FPA, p-fluorophenylalanine; HMBC, heteronuclear multiple bond coherence; HMQC, heteronuclear single quantum coherence-distortionless enhancement by polarization transfer; Pyr, pyruvate; ROESY, rotating frame Overhauser effect spectroscopy; MOPS, 4-morpholinepropanesulfonic acid; Tricine, N-[2-hydroxy-1,1-bis(hydroxymethyl)ethyl]glycine.

7790 JOURNAL OF BIOLOGICAL CHEMISTRY VOLUME 282 • NUMBER 11 • MARCH 16, 2007

Colanic Acid-modified LPS

TABLE 1  
Bacterial strains and plasmids  

| Strains/plasmid | Description | Source or Ref. |
|-----------------|-------------|----------------|
| BW30270         | $E. coli$ K-12 MG1655; $rph^{+} fnr^{+} wbbL$ | $E. coli$ Genetic Stock Center (CGSC 7925) |
| F11119-41       | Wild-type $E. coli$ isolate with O:16 antigen; K1:H$^{-}$ | H. Steinrück |
| TCM31           | BW30270 $\Delta waaL$ | This study |
| TCM33           | TCM31(pT7waaL) | This study |
| TCM40           | BW30270(pT7wbbL); O:16$^{+}$ | This study |
| KPM22           | TCM15 derivative; LPS$^{-}$ | Ref. 29 |
| KPM25           | KPM22 with pT7kdsD | Ref. 29 |
| KPM72           | KPM22 $\Delta waaL$ | This study |
| KPM73           | KPM72(pT7waaL) | This study |
| KPM77           | KPM22(pT7wbbL); O:16$^{+}$ | This study |
| pT7waaL         | pT7-7 with $E. coli$ K-12 $waaL$; Amp$^{R}$ | This study |
| pT7wbbL         | pT7-7 with $E. coli$ F11119-41 $wbbL$; Amp$^{R}$ | This study |

mally not expressed in planktonic cultures, a number of sub-  
optimal culture conditions modestly induce the production  
of CA, including low growth temperatures on solid surfaces,  
osmotic shock (16), desiccation (17), and high concentra-  
tions of zinc (18), detergents (19), or $\beta$-lactams (20). Low  
level and transient induction has complicated studies on  
envelope changes, which accompany CA biosynthesis. In  
this report, a mutant strain of $E. coli$ K-12 that produces  
copious quantities of CA when cultured in hypotonic growth  
medium has been used to isolate a novel LPS glycoform con-  
taining CA repeats that we call $M_{LPS}$ (for M-antigen). Fur-  
ther, it is shown that $M_{LPS}$ has a structure differing from the  
reported structures of extracellular CA and that $M_{LPS}$, not  
LPS O-antigen, is the dominant smooth LPS glycoform syn-  
thesized when CA is highly induced.

**EXPERIMENTAL PROCEDURES**

**Bacterial Strains and Growth Conditions—**All strains used  
were derivatives of $E. coli$ K-12 strain MG1655 and are listed in  
Table 1. Bacteria were grown in either Luria-Bertani (LB) (10  
g/liter Bacto tryptone; 5 g/liter yeast extract) medium contain-  
ing the indicated amount of NaCl (5 g/liter or 10 g/liter for CA  
inducing and non-inducing conditions, respectively) or in 0.6%  
D-glucose MOPS minimal media supplemented with thiamine  
(1 $\mu$g/ml) (21). Cultures were grown aerobically at 37°C with  
shaking (250 rpm). To partially restore LPS biosynthesis in the  
KPM series, 30 $\mu$M D-arabinose 5-phosphate (A5P) was added  
along with 10 $\mu$M D-glucose 6-phosphate (G6P) to induce the  
hexose sugar phosphate transporter (22). The amino acid ana-  
log p-fluorophenylalanine (FPA) was used at 80 $\mu$M to induce  
CA biosynthesis in wild-type strains (23).

Gene deletions were constructed using the phage $\lambda$ Red  
recombinase system according to the procedure of Datsenko  
and Wanner (24), except chloramphenicol was used at 10  
$\mu$g/ml for selection, and temperature-sensitive plasmids were  
cured at 37°C in $\Delta$API strains (KPM designation). Primers  
H1P1WaaL-H2P2WaaL were used to construct the insert cas-  
sette targeting $waaL$. All strains were colony-purified, tested for  
loss of all antibiotic resistances, and the relevant locus  
sequenced to confirm the site of deletion. Both the $waaL$  
(WaaLF-WaaLR) and $wbbL$ (WbbLF-WbbLR) genes were  
amplified and cloned into the expression vector pT7-7 using  
standard molecular biology techniques and maintained with  
100 $\mu$g/ml ampicillin. Primer sequences are listed in Table S1 of  
Supplemental Information.

Quantitation of CA—CA was estimated according to a pub-  
lished procedure (23). Liquid cultures were immersed in a boil-  
ing water bath for 15 min to release extracellular polysacchar-  
ides, and then clarified by centrifugation (10 min, 8000 × g).  
The supernatant was extensively dialyzed against distilled  
water before being assayed for $\omega$-deoxyhexose (L-fucose), a  
constituent of CA, by a colorimetric reaction using authentic  
L-fucose as standard (25). CA levels are reported as g of nondia-  
lyzable $\omega$-deoxyhexose/unit $A_{600\text{ nm}}$/ml of culture.

LPS SDS-PAGE and O:16/ECA Immunoblots—Culture sam-  
ples for LPS analysis were washed twice with Dulbecco's phos-  
phate-buffered saline, and the cell pellets resuspended in lysis  
buffer (200 mM Tris, pH 6.8, 2% SDS, 4% 2-mercaptoethanol,  
10% glycerol). LPS samples were separated by Tricine-SDS-  
PAGE (26) and visualized by silver staining (27). Immunoblots  
were developed using either Ab K9-7/75 for the O:16 antigen or  
mAb 898 for ECA (28) as previously described (29).

Large Scale $M_{LPS}$ Purification—For isolation and purification  
of $M_{LPS}$, KPM22 was grown in 10 liters of LB medium (5 g/liter  
NaCl) supplemented with G6P (10 $\mu$M) and A5P (10 $\mu$M) at  
30°C with vigorous shaking (250 rpm). After growth of the  
culture to a density of 1.0 at 600 nm, additional G6P/A5P (10/15  
$\mu$M) was added and grown until the cell density ceased increas-  
ing. At this point, cells were harvested and total LPS extracted  
as described (29).

Purification of Oligosaccharides—Approximately 60 mg of  
total LPS was hydrolyzed in acetic acid buffer (1.5 M  
CH$_3$COONa, pH 4.4, 100°C, 2 h). Lipid A was removed by  
centrifugation (12,000 × g, 1 h). The carbohydrate portion  
(supernatant) was lyophilized, reconstituted, and then fraction-  
ated by gel-permeation chromatography (GPC) on a TSK-40  
HW column (2 cm × 120 cm) using 50 mM pyridine acetate  
buffer (pH 4.4) as eluent. Fractions were monitored using a  
differential refractometer. Aliquots of purified oligosaccharides  
were de-O-acetylated using absolute hydrazine (37°C, 30 min).

Compositional Analysis—The sugar components of the oli-  
gosaccharides were identified as methyl glycoside acetate deriv-  
atives (2 M HCl in methanol, 85°C, 16 h, then acetylation).  
Methylation analysis, the determination of the absolute config-  
urations of the sugar components, and GLC/GLC-MS were  
performed as previously described (30–33).

Mass Spectrometry and NMR Spectroscopy—Mass spectra  
were recorded in the negative ion mode using a 7 Tesla ESI  
FT-MS instrument (Bruker ApexII) (29), and NMR spectra

Colanic Acid-modified LPS

**FIGURE 1.** A, hypotonic sensitivity of KPM22. Strains were grown in LB medium containing various concentrations of sodium chloride. The culture turbidity was measured after 16 h. Wild-type (diamonds), KPM22 (circles), KPM25 [(KPM22(pT7kdsD)] (squares). B, quantitation of CA in the exopolysaccharide fraction as determined by the amount of nondialyzable ω-deoxyhexose per unit $A_{600\ \mathrm{nm}}$ per ml of culture. C, SDS-PAGE LPS profile of KPM22 grown in normal and hypotonic LB medium supplemented with A5P.

were recorded on a Bruker DRX Avance 600 MHz spectrometer at 27°C (34).

**RESULTS**

**Hypotonic Media Cause Growth Inhibition, CA Induction, and an Altered LPS Profile in KPM22—E. coli K-12 has two copies of the A5P isomerase (API) gene (22, 35), whose product (A5P) is a precursor of the inner LPS core moiety 3-deoxy-D-manno-oct-2-ulosonic acid (Kdo). Strain KPM22 was constructed by deleting both API genes, resulting in an LPS layer containing predominantly the nonglycosylated LPS intermediate lipid IV$_A$ (29). Despite having a markedly compromised OM, KPM22 is normally nonmucoid and has only a 50% longer doubling time (29). Lowering the NaCl concentration, however, revealed a salt-dependent mucoid phenotype on agar and a decrease in growth that could be suppressed by complementation with the plasmid encoded API gene kdsD (KPM25) (Fig. 1A). At an NaCl concentration of 5 g/liter (~85 mM), the mucoid phenotype was readily apparent while the growth density was not appreciably diminished, and was thus used as the CA-inducing conditions in subsequent studies.

The mucoid phenotype is a characteristic marker of CA induction (6). To confirm CA is induced by hypotonic media in KPM22, extracellular polysaccharides were isolated and assayed for nondialyzable ω-deoxyhexose content, which is representative of the amount of the CA carbohydrate L-fucose (23). At 10 g/liter NaCl (~170 mM), KPM22 produced low amounts of CA comparable to the wild-type strain BW30270

**FIGURE 2.** A, M$_{\text{LPS}}$ synthesis in wild-type E. coli K-12 (lanes 1 and 2) and KPM22 (lanes 3–7). LPS synthesis was restored for KPM22 where indicated (+) by the inclusion of A5P in the medium while CA synthesis was induced by the addition of FPA (+). Block arrow denotes the LPS core position, while the shaded arrow that of an M-LPS band containing a single CA repeat unit. B, immunoblot detecting the presence of ECA. The bracket denotes the position of M$_{\text{LPS}}$ in the corresponding silver-stained gel. Lane 1, wild-type BW30270; lane 2, BW30270 + FPA; lane 3, KPM22 + A5P in inducing medium (85 mM NaCl).

(Fig. 1B). However, growth in 5 g/liter NaCl significantly increased the amount of nondialyzable ω-deoxyhexose, indicating lower ionic strength medium induces CA in KPM22. A similar, though more modest, increase in CA production was observed in the wild-type genetic background by adding the CA-inducing amino acid analog FPA to the growth medium. FPA is believed to induce CA production by inactivating a regulatory protein after it is incorporated into the polypeptide chain (23).

The SDS-PAGE LPS profile was examined to determine whether CA induction is accompanied by changes to the LPS core of KPM22. When preparations were isolated from hypotonic media, a distinct ladder-like banding of high molecular mass LPS species of unknown composition was readily observed (Fig. 1C). As many as 8–10 additional bands with a staining intensity inversely proportional to size were observed, presumably reflecting the presence of a novel LPS glycoconjugate differing in the number of polysaccharide repeat units.

M$_{\text{LPS}}$ Is Dependent on CA Induction and a Complete K-12 LPS Core—By adding A5P and lowering the concentration of NaCl in the growth medium, both the LPS core and CA synthesis, respectively, in KPM22 can be systematically varied and their involvement in the production of the new banding pattern evaluated. High molecular mass bands were observed only when A5P was added to hypotonic medium (Fig. 2A, lane 4). FPA could be used in place of hypotonic medium to induce CA in KPM22 cultures, resulting in the appearance of high molecular mass bands (Fig. 2A, lane 7). Addition of FPA to wild type resulted in similar LPS banding patterns (lane 2), suggesting the extra bands are not particular to the KPM22 genetic background. As a fraction of ECA is known to be ligated to the LPS core to form ECA$_{\text{LPS}}$ (2), ECA immunoblots were performed to exclude the possibility of ECA$_{\text{LPS}}$ being the unknown LPS glycoconjugate. Whereas the ECA Ab did detect bands, they did not correspond to the CA-dependent silver stainable bands (Fig. 2B). Because the unknown glycoconjugate required both a complete K-12 LPS core and CA induction, the bands were tentatively identified as M$_{\text{LPS}}$.

Compositional Analysis of M$_{\text{LPS}}$ Oligosaccharide (OS) Fractions—Four OS fractions (OS-I to -IV) were isolated after hydrolysis and separation by gel permeation chromatography

Colanic Acid-modified LPS

Carbohydrate analysis of OS-II identified Glc, Gal, Hep, and Fuc as the main constituents and smaller amounts of GlcA and Pyr-Gal. The absolute configurations were assigned as L for Fuc residues and D for the other sugars. Methylation analysis of OS-II revealed the presence of 4,6-disubstituted Gal, 4-substituted GlcA, 3-substituted Gal, 3-substituted Fuc, 4-substituted Fuc, and 3-substituted Glc, as well as all of the constituents of the standard E. coli K-12 LPS structure, glycoform I (36). OS-III also contained all the constituents of glycoform I, along with a minor amount of 7-substituted Hep, as described earlier as the linkage point of a terminal β-GlcpNAc residue in LPS glycoform II (36). The increased ratio of 7-substituted Hep to terminal Hep in OS-II in comparison to OS-III implicated O-7 Hep as the candidate CA-LPS core linkage position.

NMR Spectroscopy of OS-II—A comprehensive NMR investigation of OS-II was undertaken to confirm the structure and determine the CA linkage position to the LPS core. To facilitate peak assignments, a portion of OS-II was de-O-acetylated and analyzed in tandem with the parent oligosaccharide. The ¹H NMR spectrum of OS-II contained approximately a dozen overlapping signals in the anomic region between δ 5.454 and δ 4.440, three signals of O-acetyl methyl groups, of a pyruvate methyl group, and several overlapping signals characteristic of 6-deoxy-sugar methyl groups (Fig. 4, A and B). HMQC analysis helped to deconvolute the overlapping signals in the anomic region (data not shown). In total, seventeen anomic proton signals were resolved along with three proton signals shifted downfield because of geminal O-acetylation (δ 5.227, δ 5.083, δ 4.910). Three of the anomic proton signals (labeled D, E, and F in OS-II structure (Fig. 7)) gave rise to two signals attributed to non-stoichiometric O-2/O-3 acetylation of residue E. Anomeric configurations and monosaccharide conformations (pyranose for all residues) were assigned on the basis of ¹H and ¹³C NMR chemical shifts and ³J<sub>H-1,H-2</sub> and ¹J<sub>C-1,H-1</sub> coupling constants (tabulated lists of NMR assignments can be found under Supplemental Information, Tables S3 and S4).

FIGURE 3. ESI FT-ICR MS spectra of OS-fractions I–III. Mass assignments and the corresponding structures are listed in Table S2 of Supplemental Information.

and subsequently analyzed by high-resolution ESI FT-ICR MS (Fig. 3. Detailed mass assignments can be found under Supplemental Information, Table S2). Fractions OS-III (4.0 mg) and OS-IV (8.7 mg) mainly contained the unligated OS of LPS glycoform I with non-stoichiometric P and P-EtN substitutions (36, 37). Molecular ion peaks were observed for fractions OS-II (5.5 mg) and OS-I (2.2 mg) that are in excellent agreement with the calculated masses of OS-III/IV ligated to either one or two CA repeat units, respectively. Fraction OS-I also contained molecular species lacking one pyruvate unit. Approximately ~30% (mol percent) of the total recovered LPS-derived oligosaccharides were modified with CA repeats.

Colanic Acid-modified LPS

A.
![Image Description](image1)

B.
![Image Description](image2)

FIGURE 4. ${}^{1}$H NMR spectra of OS-II (A) and de-O-acetylated OS-II (B) of $M_{LPS}$ from *E. coli* KPM22. The spectra were recorded at 600 MHz and 27 °C. The letters refer to the carbohydrate residues as shown in Fig. 7. The arabic numerals refer to the protons in the respective residues. Chemical shift values and assignments can be found in Tables S3 and S4 of Supplemental Information.

A.
![Image Description](image3)

B.
![Image Description](image4)

FIGURE 5. Sections of the HMBC spectra of OS-II (A) and de-O-acetylated OS-II (B). The spectra were recorded at 600 MHz and 27 °C. The letters refer to the carbohydrate residues as shown in Fig. 7 and Supplemental Information Tables S3 and S4. The arabic numerals refer to the protons in the respective residues.

The ${}^{31}$P NMR spectrum had four phosphate signals, with the most prominent resonance being assigned to a monophosphate residue attached to O-4 of the 3,7-disubstituted $\alpha$-Hepp (M ($\delta$ 0.70)). A second residue was linked to O-4 of 3-substituted $\alpha$-Hepp (O ($\delta$ 1.09)) and was non-stoichiometrically substituted by a 2-aminoethanol phosphate appendage ($\delta -10.89$ and $\delta -11.60$).

The protons of three O-acetyl methyl groups were observed in the ${}^{1}$H NMR spectrum in a ratio of $\sim$1:0.5:0.5. Residue C (3-substituted $\alpha$-Galp) was stoichiometrically O-acetylated at position O-2 whereas residue E (4-substituted $\alpha$-Fucp) was either substituted by O-acetyl groups at positions O-2 or O-3. The HMBC spectrum showed H-6 protons of two different Fuc residues (substituted and unsubstituted at position O-4), one of which had multiple signals because of non-stoichiometric O-acetylation (Fig. 5).

On the basis of methylation analysis and NMR data (38), the R-configured pyruvate residues were located to O-4 and O-6 of 4,6-disubstituted $\alpha$-Galp (A). Additionally, a carbon resonance at $\delta$ 63.68 was observed, which is typical for C-5 of an $\alpha$-Galp residue bearing a pyruvic ketal substituent at O-4 and O-6 (39).

An HSQC-DEPT experiment was performed to distinguish between secondary and primary carbon atoms. Two low-field shielded cross-peaks of primary C-7 of 7-substituted Hepp (G) at $\delta_{C,H}$ 3.856/73.07 and at $\delta_{C,H}$ 4.057/73.07 were observed. The presence/absence of these signals has been previously described for different glycoforms of the core region depending on a substitution at O-7 (36). The positions of glycosylation assigned by methylation analysis were confirmed by the low field shifted signals of carbon atoms (underlined values in Tables S3 and S4 of Supplemental Information).

ROESY experiments revealed the sequence of the sugar residues in the oligosaccharide (Fig. 6). Strong inter-residual NOE contacts were observed between protons A1/B4, B1/C3, B1/C4, C1/D3, D1/E4, E1/F3, F1/G7, G1/H6, H1/I2, I1/K3, and K1/M3 unequivocally confirming the proposed structure

WaaL Is Necessary for CA Ligation to the LPS Core—Because the LPS core substitution positions of both CA and the O:16 antigen are identical (40) and may thus share the same LPS core acceptor, the role of the O:16 antigen ligase WaaL in $M_{LPS}$ biosynthesis was investigated. Isogenic constructs with the $waaL$ gene deleted were made in both wild type and KPM22 backgrounds. Deletion of $waaL$ had no appreciable affect on the LPS profiles in a wild type (TCM31) or KPM22 genetic background (KPM72) under conditions that did not induce CA synthesis in comparison to their respective isogenic $waaL^+$ parent strains (Fig. 8A). When CA was induced, the K-12 LPS core remained the only observable band (lanes 2 and 5). Complementation with plasmid-encoded $waaL$ restored $M_{LPS}$ biosynthesis to wild-type levels (TCM33 and KPM73), indicating CA is dependent upon the WaaL O-antigen ligase for linkage to the LPS core.

$M_{LPS}$ Is the Dominant Conjugated LPS Glycoform during CA Induction—*E. coli* K-12 has an IS5 insertion within $wbbL$, a L-rhamnosyl transferase from the O-antigen gene cluster (41) and is unable to synthesize its O:16 repeat despite having the other necessary genetic determinants. As a result, the LPS core is unligated or “rough” unless a functional copy of $wbbL$ is provided. Complementation of *E. coli* K-12 with $wbbL$ in wild type (TCM40) or in KPM22 (KPM77) resulted in the characteristic modal distribution of O-antigen repeat lengths as visualized by immunoblotting with anti-O:16 Ab when CA was not induced (Fig. 8B, lanes 2 and 7). $M_{LPS}$ bands did not cross-react with the O:16 antibody (lanes 3 and 5), confirming the high molecular mass LPS bands were not caused by O:16 reversion. Moreover, restoration of O:16 was not observed in KPM77 when grown

Colanic Acid-modified LPS

is triggered by a variety of environmental stresses (16–20). Whether certain stress stimuli induce CA by merely mimicking conditions experienced by cells during a particular biofilm development phase/environment, or CA production alone is a coping mechanism outside a biofilm setting is unclear. A unifying theme among CA-inducing stresses is that they all disturb the integrity of the cell envelope, leading to the suggestion that CA synthesis is a response to sensed OM instability originating from a damaged cell envelope (16). Indeed, genetic lesions which truncate the inner LPS core are intimately associated with an *rcsC*-dependent mucoid phenotype (42). Accumulating LPS precursors and sugar nucleotides from O-antigen biosynthetic pathways have been proposed as direct CA-inducing signals (42, 43).

We previously reported the construction of a strain of *E. coli* K-12 (KPM22) that, in contrast to other inner core LPS mutants (42), is nonmucoid despite lacking the entire LPS core structure (29). The ΔAPI genotype (Δ*kdsD*Δ*gutQ*) of KPM22 blocks biosynthesis of Kdo, resulting in the deepest rough LPS mutant possible and an OM of correspondingly compromised integrity (see Ref. 29 for a discussion). While KPM22 is normally nonmucoid, there is a salt-dependent growth defect in hypotonic media that is immediately preceded by robust CA production (Fig. 1A). A5P and the API protein(s) themselves are clearly not necessary for CA synthesis, nor is an OM LPS layer needed for the activity of the OM embedded CA assembly protein complex. This may not be the case for *Yersinia pestis*, where a pair of API protein domains have been implicated in expression of a biofilm exopolysaccharide (44), and in the capsular polysaccharides of *Neisseria meningitidis* (45) and *E. coli* Group 2 capsules (46, 47). Osmoprotectants such as sucrose are unable to substitute for NaCl in suppressing the KPM22 growth defect (data not shown), indicating uncharged solutes cannot substitute for electrolytes. OM instability may only be manifested and sensed at low ionic strength because of less screening of charge repulsion between phospholipids/lipid IV<sub>A</sub> headgroups, coupled with the higher intrinsic turgor pressure imparted by a hypotonic environment. LPS truncation would therefore appear to be an indirect signal for CA induction, with the direct signal being the resultant OM distension, which in turn is sensed by an OM-associated protein or complex. The OM lipoprotein RcsF may be such a candidate sensor capable of transmitting OM perturbation input to the *rcs* phosphorelay (48).

CA has traditionally been considered as exclusively an exopolysaccharide that is only loosely associated with the surface. Apparently, CA can under certain conditions be efficiently attached to the surface through a covalent linkage to LPS to form M<sub>LPS</sub>. Approximately ~30% of the fractionated oligosaccharide molecules were ligated, with up to 10 repeats visible in a nonmodal distribution. Compositional analysis and NMR spectroscopy were used to make signal assignments and ultimately to elucidate the structure of OS-II (Fig. 7). CA repeats are ligated to O-7 of the outer core Hepp (G) of LPS glycoform I. The LPS-ligated CA repeat structure is similar to earlier reports of CA exopolysaccharides produced by a number of other enterobacteria (11, 49, 50). Whereas the sugar composition and the carbohydrate sequence are the same, both Galp residues (A and C) and of Fucp (D) are α-anomers instead of the

FIGURE 6. Sections of the ROESY spectra of OS-II (A) and de-O-acetylated OS-II (B). The spectra were recorded at 600 MHz and 27°C. The letters refer to the carbohydrate residues as shown in Fig. 7, and the arabic numerals refer to the protons in the respective residues. The inter-residual NOE contacts are underlined.

under CA-inducing conditions (*lane* 6), as determined by the lack of any high molecular mass silver staining material and anti-O:16 immunoblotting. The LPS profile revealed only M<sub>LPS</sub> and unligated K-12 core, identical to the O:16 negative sample (*lane* 5). The shift to M<sub>LPS</sub> as the predominant smooth LPS glycoform was not observed in the complemented wild-type sample with CA induced by FPA (*lane* 4). This suggests that there may be differences in the molecular signaling mechanisms at work in CA induction by FPA versus hypotonic growth of KPM22.

DISCUSSION

The essential role of CA in *E. coli* biofilm maturation is well established (10). In addition to biofilm formation, CA induction

Colanic Acid-modified LPS

Py→4,6)-α-Galp-(1→4)-β-GlcpA-(1→3)-α-Galp-(1→3)-α-Fucp-(1→4)-α-Fucp-(1→3)-β-Glcp F  
O-Ac  
↓  
2  
O-Ac  
↓  
2  
α-Hepp  
G  
1  
↓  
7  
α-Galp  
L  
↓  
6  
α-Glcp-(1→2)-α-Glcp-(1→3)-α-Glcp-(1→3)-α-Hepp-(1→3)-α-Hepp-(1→5)-α-Kdop-  
H I K M P  
P=P-EtN  

FIGURE 7. Chemical structure of OS-II isolated from the total LPS fraction of E. coli KPM22 grown in hypotonic medium after hydrolysis and removal of lipid A. Non-stoichiometric substitutions are underlined and italicized. Letter assignments correspond to peak assignments in the text and in Fig. 4. The large block arrow denotes ligation position of the initiating D-Glc residue from the CA repeat to LPS core.

A.  
Lane 1 2 3 4 5 6  

B.  
Lane 1 2 3 4 5 6 7  

FIGURE 8. LPS whole cell lysate profiles. A, silver-stained LPS gel examining the role of *waaL* in the biosynthesis of M<sub>LPS</sub>. Lane 1, TCM31; lane 2, TCM31 + FPA; lane 3, TCM33 + FPA; lane 4, KPM72; lane 5, KPM72 + A5P; lane 6, KPM73 + A5P. Block arrow denotes the position of the LPS core while the shaded arrow that of an M<sub>LPS</sub> band composed of one or two CA repeat units. B, silver-stained LPS gel (top panel) and immunoblot (bottom panel) using O:16 antibody. Lane 1, E. coli F11119–41(O:16); lane 2, TCM40; lane 3, wild-type BW30270 + FPA; lane 4, TCM40 + FPA; lane 5, KPM22 + A5P; lane 6, KPM77 + A5P (85 mM NaCl); lane 7, KPM77 + A5P (170 mM NaCl). Bracket defines the high modal O-antigen repeat bands while the block arrow denotes the unligated LPS core.

necessary step for the synthesis or transport of extracellular CA (Fig. 1B).

The O-antigen ligase WaaL is an essential component in the ligation of a number of Und-PP linked polysaccharides to the LPS core, which is a versatile glycosylation acceptor in E. coli (52, 53). Capped LPS glycoforms not only encompass O-antigen, but also enterobacterial common antigen [ECA<sub>LPS</sub>] (2), a subset of Group 1 and 4 capsules [K<sub>LPS</sub>] (54), and now CA [M<sub>LPS</sub>]. The flexibility of the WaaL-dependent ligation machinery in E. coli K-12 to accommodate diverse Und-PP donors is impressive, particularly considering CA (Glcp, residue (F) in Fig. 7) does not share the same initiating glycosyl residue as O:16 antigen and ECA (GlcpNAc). The substrate promiscuity of the ligation machinery may be by design in order to efficiently attach both the O:16 repeat and the CA repeat to the LPS core depending on the needs of the cell and not simply a consequence of recognizing the Und-PP lipid carrier. Studies are beginning to reveal a level of complexity in the recognition of both the donor (40, 55) and the acceptor (56) that invoke a multiprotein assembly apparatus, suggesting the participation of both polysaccharide repeat specific and generic (*i.e.* WaaL) components. It will be interesting to examine the distribution of M<sub>LPS</sub> biosynthesis among CA-expressing E. coli strains that have any of the other LPS core types (57).

There are clear parallels in the assembly of K<sub>LPS</sub>, ECA<sub>LPS</sub>, and certain O-antigens with M<sub>LPS</sub>. All the heteropolysaccharide repeat units are synthesized in a more or less identical manner by the Wzy-dependent pathway (1, 3). The respective pathways diverge at postassembly of the repeat in that the polysaccharide can either be ligated to a lipid anchor (*i.e.* LPS and/or glycerophospholipids), secreted as an exopolysaccharide, or both, as is the case for CA and Group 1 K-antigens. The similarity between CA and Group 1 K-antigens extends to the protein level, as certain membrane translocation components can be interchanged between the two systems, including Wza, Wzb, and Wzc CA proteins with their K30-antigen counterparts (58). However, there are major differences in the regulatory networks controlling their expression and likely in their respective cellular functions (reviewed in Refs. 3 and 59). Groups 1 K-antigens are more closely associated with virulence and are expressed at physiological temperatures, whereas CA is normally not expressed and is therefore generally thought to have a function outside of a host-pathogen relationship.

Another difference becomes apparent when the *wbbL* lesion from the O:16 antigen biosynthetic pathway of *E. coli* K-12 is repaired. Whereas $K_{\mathrm{LPS}}$ is co-expressed with their O-antigens (60), no O:16 antigen was visualized in KPM22 when grown under CA-inducing conditions with A5P by immunoblotting (Fig. 8B). The loss of O:16 could be attributed to a number of factors, the most apparent ones being: (i) an assembly defect in the KPM22 membrane when cultured in hypotonic medium, (ii) titration of Und-PP lipid carrier, LPS core acceptor, and/or the ligation machinery by the CA pathway, and (iii) regulated gene silencing of the O:16 antigen gene cluster. We currently favor the latter (iii), as the KPM22 envelope, albeit compromised, is still capable of assembling the other Wzy-dependent polymer ECA (Fig. 2B). Second, Und-PP-linked O-antigen precursors are normally detectable by immunoblotting (61). Limiting amounts of LPS core acceptor would seem an unlikely answer, as both SDS-PAGE analysis and LPS fractionation detected large amounts of unligated LPS glycoform I. Most compelling, the *wbbL* plasmid is difficult to maintain in the KPM22 genetic background when either LPS synthesis is not enabled by withholding A5P from the medium or when CA is not induced. Unligatable Und-PP polysaccharide precursors are known to be toxic, presumably by draining the cellular Und-PP pools into dead end products, leading to membrane perturbation and cessation of essential Und-PP-dependent pathways such as peptidoglycan synthesis (62). Without A5P addition to KPM77 [KPM22(pT7*wbbL*)] cultures, Und-PP O:16 cannot be ligated as the attachment site is distal to Kdo (Fig. 7). Plasmid pT7*wbbL* transformed at a high efficiency and was stable without A5P in the KPM22 genetic background only when CA was induced. CA induction in the wild-type strain by the aromatic amino acid analog FPA did not repress O:16 antigen biosynthesis. FPA is thought to induce CA by incorporation into a CA regulatory protein, whose activity in turn is altered (23). CA synthesis is more aptly described as derepression as opposed to regulated induction to an environmental signal, and cannot be equated with hypotonic CA induction in KPM22. Collectively, the data suggest the O-antigen gene cluster may be co-regulated and repressed when CA is induced.

The possibility that *E. coli* can in effect change its polysaccharide coat from an O-antigen LPS surface to $M_{\mathrm{LPS}}$ in response to OM perturbation raises the question of what advantage does $M_{\mathrm{LPS}}$ impart to a bacterium with a destabilized OM. Unlike serotype-specific capsular and O-antigens, the genetic determinants for CA synthesis are widely distributed among different strains of *E. coli* that express a multitude of O-antigens of variable composition, charge, length, and hydrophobicity (3). Remodeling to $M_{\mathrm{LPS}}$ would result in a surface with increased net negative charge. Perhaps $M_{\mathrm{LPS}}$ facilitates homotropic interactions with the extracellular CA fraction through salt bridges, as has been proposed for some authentic capsular antigens (63). A more intimate CA surface association could conceivably stabilize a traumatized OM. While generally not considered a virulence factor, CA does enhance survival of enterohemorrhagic *E. coli* in simulated gastrointestinal fluids (64). The negatively charged $M_{\mathrm{LPS}}$ layer may provide additional extracellular buffering capacity to an acid-stressed OM in comparison to other lipid A-linked polysaccharides. Alternatively,

the advantage may be conferred by the very removal of the O-antigen itself. A number of bacteriophages use O-antigens as receptors (65). Provided the CA induction kinetics are sufficiently robust, phage infection sensed through OM destabilization may be countered by replacement of the receptor and/or CA-masking of underlying receptors, implicating CA as a potential defense mechanism. If present, $M_{\mathrm{LPS}}$ may also be more advantageous than O-antigen in the unique environment of biofilms. Smooth O-antigens normally project away from the surface (~30–100 repeat units), shielding the surface from contact with macromolecules and/or other cells. In contrast, $M_{\mathrm{LPS}}$ expression in *E. coli* K-12 is non-modal, with a decidedly lower average number of repeat units. The greater surface accessibility could foster cell-to-cell contacts by exposing surface proteins and other adhesion factors that figure prominently in biofilm communities. Definitive answers await a better understanding of both the environmental signals and the regulatory systems that control $M_{\mathrm{LPS}}$ biosynthesis in *E. coli*.

Acknowledgments—We thank other members of the Woodard laboratory for helpful discussions, D. Bitter-Suermann (Medical University Hannover, Germany) for kindly providing the anti-ECA mAb, H. Steinrück (Robert Koch Institute, Berlin, Germany) for anti-O:16 Ab and *E. coli* strain F11119-41, H. Käβner (RCB) for NMR measurements, H. Moll (RCB) for help with GC-MS experiments, and Brigitte Kunz (RCB) for MS measurements.

---

**REFERENCES**

1. Raetz, C. R., and Whitfield, C. (2002) *Annu. Rev. Biochem.* **71**, 635–700
2. Kuhn, H. M., Meier-Dieter, U., and Mayer, H. (1988) *FEMS Microbiol. Rev.* **4**, 195–222
3. Whitfield, C. (2006) *Annu. Rev. Biochem.* **75**, 39–68
4. Wang, X., Preston, J. F., 3rd, and Romeo, T. (2004) *J. Bacteriol.* **186**, 2724–2734
5. Zogaj, X., Nimtz, M., Rohde, M., Bokranz, W., and Romling, U. (2001) *Mol. Microbiol.* **39**, 1452–1463
6. Grant, W. D., Sutherland, I. W., and Wilkinson, J. F. (1969) *J. Bacteriol.* **100**, 1187–1193
7. Ørskov, I., Ørskov, F., Jann, B., and Jann, K. (1977) *Bacteriol. Rev.* **41**, 667–710
8. Agladze, K., Wang, X., and Romeo, T. (2005) *J. Bacteriol.* **187**, 8237–8246
9. Sutherland, I. (2001) *Microbiology* **147**, 3–9
10. Danese, P. N., Pratt, L. A., and Kolter, R. (2000) *J. Bacteriol.* **182**, 3593–3596
11. Garegg, P. J., Lindberg, B., Onn, T., and Sutherland, I. W. (1971) *Acta Chem. Scand.* **25**, 2103–2108
12. Stevenson, G., Andrianopoulos, K., Hobbs, M., and Reeves, P. R. (1996) *J. Bacteriol.* **178**, 4885–4893
13. Stout, V. (1996) *J. Bacteriol.* **178**, 4273–4280
14. Gottesman, S., Trisler, P., and Torres-Cabassa, A. (1985) *J. Bacteriol.* **162**, 1111–1119
15. Majdalani, N., and Gottesman, S. (2005) *Annu. Rev. Microbiol.* **59**, 379–405
16. Sledjeski, D. D., and Gottesman, S. (1996) *J. Bacteriol.* **178**, 1204–1206
17. Ophir, T., and Gutnick, D. L. (1994) *Appl. Environ. Microbiol.* **60**, 740–745
18. Hagiwara, D., Sugiura, M., Oshima, T., Mori, H., Aiba, H., Yamashino, T., and Mizuno, T. (2003) *J. Bacteriol.* **185**, 5735–5746
19. Rajagopal, S., Sudarsan, N., and Nickerson, K. W. (2002) *Appl. Environ. Microbiol.* **68**, 4117–4121
20. Sailer, F. C., Meberg, B. M., and Young, K. D. (2003) *FEMS Microbiol. Lett.* **226**, 245–249
21. Neidhardt, F. C., Bloch, P. L., and Smith, D. F. (1974) *J. Bacteriol.* **119**,

Colanic Acid-modified LPS

736–747

22. Meredith, T. C., and Woodard, R. W. (2005) *J. Bacteriol.* 187, 6936–6942
23. Kang, S., and Markovitz, A. (1967) *J. Bacteriol.* 93, 584–591
24. Datsenko, K. A., and Wanner, B. L. (2000) *Proc. Natl. Acad. Sci. U. S. A.* 97, 6640–6645
25. Dische, Z., and Shettles, L. B. (1948) *J. Biol. Chem.* 175, 595–603
26. Lesse, A. J., Campagnari, A. A., Bittner, W. E., and Apicella, M. A. (1990) *J. Immunol. Methods* 126, 109–117
27. Hitchcock, P. J., and Brown, T. M. (1983) *J. Bacteriol.* 154, 269–277
28. Peters, H., Jurs, M., Jann, B., Jann, K., Timmis, K. N., and Bitter-Suermann, D. (1985) *Infect. Immun.* 50, 459–466
29. Meredith, T. C., Aggarwal, P., Mamat, U., Lindner, B., and Woodard, R. W. (2006) *ACS Chem. Biol.* 1, 33–42
30. Haseley, S. R., Holst, O., and Brade, H. (1997) *Eur. J. Biochem.* 247, 815–819
31. Ciucanu, I., and Kerek, F. (1984) *Carbohydr. Res.* 131, 209–217
32. Kaczynski, Z., Karapetyan, G., Evidente, A., Iacobellis, N. S., and Holst, O. (2006) *Carbohydr. Res.* 341, 285–288
33. Leontein, K., Lindberg, B., and Lönngren, J. (1978) *Carbohydr. Res.* 62, 359–362
34. Theilacker, C., Kaczynski, Z., Kropec, A., Fabretti, F., Sange, T., Holst, O., and Huebner, J. (2006) *Infect. Immun.* 74, 5703–5712
35. Meredith, T. C., and Woodard, R. W. (2003) *J. Biol. Chem.* 278, 32771–32777
36. Müller-Loennies, S., Lindner, B., and Brade, H. (2003) *J. Biol. Chem.* 278, 34090–34101
37. Holst, O., Zähringer, U., Brade, H., and Zamojski, A. (1991) *Carbohydr. Res.* 215, 323–335
38. Garegg, P. J., Jansson, P.-E., Lindberg, B., Lindh, F., Lönngren, J., Kvarnström, I., and Nimmich, W. (1980) *Carbohydr. Res.* 78, 127–132
39. Jansson, P.-E., Lindberg, J., and Widmalm, G. (1993) *Acta Chem. Scand.* 47, 711–715
40. Feldman, M. F., Marolda, C. L., Monteiro, M. A., Perry, M. B., Parodi, A. J., and Valvano, M. A. (1999) *J. Biol. Chem.* 274, 35129–35138
41. Liu, D., and Reeves, P. R. (1994) *Microbiology* 140, 49–57
42. Parker, C. T., Kloser, A. W., Schnaitman, C. A., Stein, M. A., Gottesman, S., and Gibson, B. W. (1992) *J. Bacteriol.* 174, 2525–2538
43. El-Kazzaz, W., Morita, T., Tagami, H., Inada, T., and Aiba, H. (2004) *Mol.*

Microbiol. 51, 1117–1128

44. Tan, L., and Darby, C. (2006) *Mol. Microbiol.* 61, 861–870
45. Tzeng, Y. L., Datta, A., Strole, C., Kolli, V. S., Birck, M. R., Taylor, W. P., Carlson, R. W., Woodard, R. W., and Stephens, D. S. (2002) *J. Biol. Chem.* 277, 24103–24113
46. Meredith, T. C., and Woodard, R. W. (2006) *Biochem. J.* 395, 427–432
47. Cieslewicz, M., and Vimr, E. (1997) *Mol. Microbiol.* 26, 237–249
48. Majdalani, N., Heck, M., Stout, V., and Gottesman, S. (2005) *J. Bacteriol.* 187, 6770–6778
49. Lawson, C. J., McCleary, C. W., Nakada, H. I., Rees, D. A., Sutherland, I. W., and Wilkinson, J. F. (1969) *Biochem. J.* 115, 947–958
50. Sutherland, I. W. (1969) *Biochem. J.* 115, 935–945
51. Cescutti, P., Toffanin, R., Pollesello, P., and Sutherland, I. W. (1999) *Carbohydr. Res.* 315, 159–168
52. Heinrichs, D. E., Yethon, J. A., and Whitfield, C. (1998) *Mol. Microbiol.* 30, 221–232
53. Whitfield, C., Amor, P. A., and Koplin, R. (1997) *Mol. Microbiol.* 23, 629–638
54. Whitfield, C., and Roberts, I. S. (1999) *Mol. Microbiol.* 31, 1307–1319
55. Marolda, C. L., Vicarioli, J., and Valvano, M. A. (2004) *Microbiology* 150, 4095–4105
56. Kaniuk, N. A., Vinogradov, E., and Whitfield, C. (2004) *J. Biol. Chem.* 279, 36470–36480
57. Whitfield, C., Kaniuk, N., and Frirdich, E. (2003) *J. Endotoxin Res.* 9, 244–249
58. Reid, A. N., and Whitfield, C. (2005) *J. Bacteriol.* 187, 5470–5481
59. Whitfield, C., and Paiment, A. (2003) *Carbohydr. Res.* 338, 2491–2502
60. MacLachlan, P. R., Keenleyside, W. J., Dodgson, C., and Whitfield, C. (1993) *J. Bacteriol.* 175, 7515–7522
61. Abeyrathne, P. D., Daniels, C., Poon, K. K., Matewish, M. J., and Lam, J. S. (2005) *J. Bacteriol.* 187, 3002–3012
62. Stevenson, G., Neal, B., Liu, D., Hobbs, M., Packer, N. H., Batley, M., Redmond, J. W., Lindquist, L., and Reeves, P. (1994) *J. Bacteriol.* 176, 4144–4156
63. Jann, B., and Jann, K. (1990) *Curr. Top. Microbiol. Immunol.* 150, 19–42
64. Mao, Y., Doyle, M. P., and Chen, J. (2006) *Lett. Appl. Microbiol.* 42, 642–647
65. Lindberg, A. A. (1973) *Annu. Rev. Microbiol.* 27, 205–241

Glycobiology and Extracellular Matrices:  
Modification of Lipopolysaccharide with  
Colanic Acid (M-antigen) Repeats in  
Escherichia coli  

Timothy C. Meredith, Uwe Mamat, Zbigniew  
Kaczynski, Buko Lindner, Otto Holst and  
Ronald W. Woodard  
J. Biol. Chem. 2007, 282:7790-7798.  
doi: 10.1074/jbc.M611034200 originally published online January 16, 2007  

Access the most updated version of this article at doi: [10.1074/jbc.M611034200](http://dx.doi.org/10.1074/jbc.M611034200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](http://www.jbc.org/site/affinitysites/index.xhtml).

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

Supplemental material:  
[http://www.jbc.org/content/suppl/2007/01/17/M611034200.DC1.html](http://www.jbc.org/content/suppl/2007/01/17/M611034200.DC1.html)

This article cites 60 references, 35 of which can be accessed free at  
[http://www.jbc.org/content/282/11/7790.full.html#ref-list-1](http://www.jbc.org/content/282/11/7790.full.html#ref-list-1)
