
Communications

**Biosynthetic Enzymes**

DOI: 10.1002/anie.200604194

# Characterization of the Enzyme BtrD from *Bacillus circulans* and Revision of Its Functional Assignment in the Biosynthesis of Butirosin**

Andrew W. Truman, Fanglu Huang, Nicholas M. Llewellyn, and Jonathan B. Spencer*

The 2-deoxystreptamine(2-DOS)-containing aminoglycosides are a clinically important class of antibiotics, whose members include neomycin, gentamycin, and the semisynthetic amikacin. A detailed understanding of the biosynthetic enzymes that construct these natural products is required for the rational engineering of biosynthetic pathways to produce novel aminoglycosides. Butirosin (5) is a 2-DOS aminoglycoside that contains a novel (S)-4-amino-2-hydroxybutyrate side chain. In 2005, Kudo et al.[1] proposed that BtrD, a protein encoded by the butirosin gene cluster,[2] functions as a nucleotidyltransferase that specifically synthesizes thymidine 5′-diphosphoglucosamine (TDP-GlcN) from thymidine 5′-triphosphate (TTP) and glucosamine-1-phosphate, thus generating the sugar donor for a putative glycosyltransferase, BtrM (Scheme 1a). It was also proposed that homologous enzymes in other gene clusters also function as hexose-1-phosphate nucleotidyltransferases.[1]

Here, we report that this characterization is incorrect and that BtrD actually functions as a deacetylase (Scheme 1b), which is consistent with its phylogenetic similarity to characterized deacetylases PIG-L (glycosylphosphatidylinositol biosynthesis[3]), MshB (mycothiol biosynthesis[4]) and Orf2* (teicoplanin biosynthesis[5]). These all have a conserved domain called COG2120 and function in an analogous fashion to incorporate glucosamine into natural products. In each case, glucosamine originates from uridine diphospho-N-acetyl-D-glucosamine (UDP-GlcNAc), an abundant primary metabolite essential for cell wall biosynthesis. A UDP-N-acetyl-glucosaminyl transferase glycosylates an aglycone, then the aminosugar is subsequently deacetylated. A similar pathway also exists in the biosynthesis of lipid A using the phylogenetically unrelated deacetylase LpxC.[6] This biosynthetic strategy ensures that UDP-GlcNAc is not unnecessarily hydrolyzed (a transformation that could be detrimental to cell wall biosynthesis), and the rate of UDP-GlcNAc sequestration is controlled by glycosyltransferase activity and the rate of aglycone production.

The recent discovery[5] that Orf2*, a protein encoded in the teicoplanin gene cluster, functions as a deacetylase prompted a reinvestigation into the function of BtrD. The gene *btrD* was cloned from cosmid DNA and BtrD was expressed in *E. coli* BL21(DE3) as an N-terminally His₆-tagged protein. Unexpectedly, incubation of the enzyme with TTP or uridine triphosphate with glucosamine-1-phosphate or glucose-1-phosphate gave no reaction.[7] The utility of BtrD as a deacetylase was therefore explored.

As has been previously observed,[1] BtrD does not deacetylate UDP-GlcNAc, so a pathway of glucosamine

[*] A. W. Truman, Dr. F. Huang, N. M. Llewellyn, Dr. J. B. Spencer  
University of Cambridge  
University Chemical Laboratory  
Lensfield Road, Cambridge CB21EW (UK)  
Fax: (+44) 122-333-6362  
E-mail: jbs20@cam.ac.uk  

[**] This research was supported financially by the BBSRC (support for J.B.S.), the BBSRC and St. John’s College, Cambridge (studentship for A.W.T.), and the Marshall Scholarships and the National Science Foundation (support for N.M.L.). Many thanks go to Professor Anthony Clarke for the donation of AAC(2')-Ia.

Supporting information for this article is available on the WWW under http://www.angewandte.org or from the author.

incorporation analogous to the biosyntheses of mycothiol, glycosylphosphatidylinositol, and teicoplanin was investigated. To accomplish this, the synthesis of acetylated biosynthetic intermediates was required. N-acetylation is a common resistance mechanism towards aminoglycosides,[8] and a number of coenzyme A-dependent aminoglycoside 2′-N-acetyltransferases (AAC(2′)) have been reported.[9] The well-characterized[10] AAC(2′)-Ia from *Providencia stuartii* was expressed as an N-terminally His<sub>6</sub>-tagged protein in *E. coli* and its activity exploited to selectively 2′-N-acetylate paromamine (2), neamine (3), ribostamycin (4), and butirosin (5).

These compounds represent all major intermediates beyond 2-DOS in butirosin biosynthesis (Scheme 1), so activity with at least one of these substrates would be expected if BtrD functions as a deacetylase. Incubation of BtrD with each of these substrates and analysis by LC/ESI-MS demonstrated that BtrD is active only with 2′-N-acetylparomamine (1); this was evident from the disappearance of the signal at *m/z* 366.1 at *t* = 6.0 min and the appearance of the signal at *m/z* 324.2 at *t* = 10.0 min attributed to paromamine (2; Figure 1 a). The transformation was also observed in situ by <sup>1</sup>H NMR spectroscopy (Figure 1 b). This clearly shows the movement of the singlet for the 2′-N-acetyl methyl group

from δ = 1.92 ppm for **1** to δ = 1.79 ppm for free acetate. Since BtrD showed no activity with the other acetylated compounds, it strongly suggests that the timing of deacetylation in butirosin biosynthesis is directly after the formation of 2′-N-acetylparomamine (**1**; Scheme 1b). This is consistent with the observation that paromamine restored butirosin production in a *btrD* gene disrupted mutant of *Bacillus circulans.*[1]

The reassignment of BtrD as a deacetylase raises the possibility that all proteins containing the COG2120 domain are deacetylases. This domain has a conserved sequence motif near the N terminus ((A/P)HXDD) and another towards the middle of the protein (HXDH). Crystallographic studies[4] imply that these conserved regions participate in metal binding, substrate binding, and base catalysis for MshB. Figure 2 clearly demonstrates the presence of these conserved

![Figure 2](#fig2)

Figure 2. Partial amino acid sequence alignment of BtrD and aminoglycoside gene cluster homologues with known deacetylases. Residues proposed (from MshB crystallization studies) to participate in metal binding (▲), substrate binding (●) and base catalysis (★) are indicated. Fully conserved residues have white text on a red background and similar (at least 70%) amino acids are framed in blue (the similar residues are colored red).

regions in a wide array of homologues encoded in aminoglycoside gene clusters. In addition to these highly conserved regions, other more isolated residues proposed to bind to the glucosaminyl substrate (Arg68 and Asp95 in MshB[4]) are also well conserved throughout these proteins (see the Supporting Information).

The homologue in the apramycin gene cluster, AprN, suggests a common biosynthetic pathway to paromamine prior to formation of apramycin’s novel fused ring system (Scheme 2). It is worth noting that BtrD exhibits relatively poor overall identity to the aminoglycoside gene cluster homologues, probably because butirosin’s producer (*Bacillus circulans*) is the only non-actinomycete aminoglycoside producer.

A glucosaminyl group (sometimes heavily derivatized) is present in almost every natural product whose gene cluster encodes a protein with the COG2120 domain, thus suggesting a common function. Notable exceptions are chloroeremycin[11] and balhimycin,[12] glycopeptide antibiotics whose gene clusters contain putative proteins with high identity to Orf2* and lincomycin.[13] Although all three of these molecules contain aminosugars, it is not clear whether they are derived from glucosamine. Further experimental work will be needed

Angew. Chem. Int. Ed. 2007, 46, 1462–1464 © 2007 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim www.angewandte.org 1463

Communications

Keywords: aminoglycosides · biosynthesis · butirosin · deacetylation

[1] F. Kudo, K. Kawabe, H. Kuriki, T. Eguchi, K. Kakinuma, *J. Am. Chem. Soc.* 2005, 127, 1711–1718.

[2] Y. Ota, H. Tamegai, F. Kudo, H. Kuriki, A. Koike-Takeshita, T. Eguchi, K. Kakinuma, *J. Antibiot.* 2000, 53, 1158–1167.

[3] R. Watanabe, K. Ohishi, Y. Maeda, N. Nakamura, T. Kinoshita, *Biochem. J.* 1999, 339, 185–192.

[4] a) G. L. Newton, Y. Av-Gay, R. C. Fahey, *J. Bacteriol.* 2000, 182, 6958–6963; b) J. T. Maynes, C. Garen, M. M. Cherney, G. Newton, D. Arad, Y. Av-Gay, R. C. Fahey, M. N. James, *J. Biol. Chem.* 2003, 278, 47166–47170; c) A. A. McCarthy, N. A. Peterson, R. Knijff, E. N. Baker, *J. Mol. Biol.* 2004, 335, 1131–1141.

[5] a) A. W. Truman, L. Robinson, J. B. Spencer, *ChemBioChem* 2006, 7, 1670–1675; b) T.-L. Li, F. Huang, S. F. Haydock, T. Mironenko, P. F. Leadlay, J. B. Spencer, *Chem. Biol.* 2004, 11, 107–119.

[6] J. E. Jackman, C. R. H. Raetz, C. A. Fierke, *Biochemistry* 1999, 38, 1902–1911.

[7] A mixture of nucleotidyltransferase activity similar to that reported by Kudo et al. and nucleotide 5′-triphosphate phosphatase activity was observed in the cell-free supernatant (possibly a glucose-1-phosphate thymidytransferase) of both BL21 containing the insert and those that did not. This activity was lost when BtrD was purified by nickel-affinity chromatography.

[8] K. J. Shaw, P. N. Rather, R. S. Hare, G. Miller, *Microbiol. Rev.* 1993, 57, 138–163.

[9] D. R. Macinga, P. N. Rather, *Front. Biosci.* 1999, 4, 132–140.

[10] a) M. Chevereau, P. J. L. Daniels, J. Davies, F. LeGoffic, *Biochemistry* 1974, 13, 598–603; b) K. Franklin, A. J. Clarke, *Antimicrob. Agents Chemother.* 2001, 45, 2238–2244.

[11] A. M. A. van Wageningen, P. N. Kirkpatrick, D. H. Williams, B. R. Harris, J. K. Kershaw, N. J. Lennard, M. Jones, S. J. M. Jones, P. J. Solenberg, *Chem. Biol.* 1998, 5, 155–162.

[12] S. Pelzer, R. Süßmuth, D. Heckmann, J. Recktenwald, P. Huber, G. Jung W. Wohlleben, *Antimicrob. Agents Chemother.* 1999, 43, 1565–1573.

[13] a) U. Peschke, H. Schmidt, H.-Z. Zhang, W. Piepersberg, *Mol. Microbiol.* 1995, 16, 1137–1156; b) N. M. Brahme, J. E. Gonzalez, S. Mitzak, J. R. Rolls, E. J. Hessler, L. H. Hurley, *J. Am. Chem. Soc.* 1984, 106, 7878–7883.

[14] F. Huang, S. F. Haydock, T. Mironenko, D. Spiteller, Y. Li, J. B. Spencer, *Org. Biomol. Chem.* 2005, 3, 1410–1418.

Scheme 2. A proposed general biosynthetic pathway for glucosamine incorporation into aminoglycosides. Blue rings indicate a second sugar that is likely to be incorporated in an analogous fashion. The BtrD homologue in each aminoglycoside cluster is listed in parentheses.

Another detail to be confirmed is the biosynthetic pathway of aminoglycosides that incorporate more than one glucosamine group, such as neomycin. The neomycin gene cluster [14] contains only one BtrD homologue (Neo16), suggesting it may act twice in the formation of neomycin (Scheme 2).

In conclusion we have shown that BtrD functions as a deacetylase in the formation of butirosin, therefore strongly suggesting that UDP-GlcNAc is the true substrate for the glycosyltransferase BtrM. Since there are many homologues of BtrD encoded in the gene clusters of other natural products, it implies a common strategy for incorporating a glucosamine moiety into secondary metabolites.

Received: October 13, 2006  
Published online: January 17, 2007

www.angewandte.org

© 2007 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim

Angew. Chem. Int. Ed. 2007, 46, 1462–1464
