
REVIEW

Type II polyketide synthases: gaining a deeper insight into enzymatic teamwork

Christian Hertweck,$^{a}$ Andriy Luzhetskyy,$^{b}$ Yuri Rebets$^{b}$ and Andreas Bechthold*$^{b}$

Received (in Cambridge, UK) 15th August 2006  
First published as an Advance Article on the web 22nd November 2006  
DOI: 10.1039/b507395m  

Covering: 2002–2005  

This review covers advances in understanding of the biosynthesis of polyketides produced by type II PKS systems at the genetic, biochemical and structural levels.

---

### Contents
1. Introduction
2. Biological activities of type II PKS-derived natural products
3. Enzymes involved in the formation of aromatic polyketides
   - 3.1 Production of aromatic polyketides by teamwork between enzymes
   - 3.2 Polyketide chain assembly by the minimal PKS
   - 3.3 Chain length
   - 3.4 PKS priming
   - 3.5 Ketoreductases
   - 3.6 Cyclases
   - 3.7 Aromatases
4. Polyketide-tailoring reactions
   - 4.1 Methyltransferases
   - 4.2 Oxygenases
   - 4.3 Glycosyltransferases
5. Regulation of aromatic polyketide biosynthesis
   - 5.1 Genes that affect secondary metabolism and morphological differentiation
   - 5.2 Genes that affect secondary metabolism and growth rate
   - 5.3 General signals affecting secondary metabolism
   - 5.4 SARP, pathway-specific regulators affecting secondary metabolism
   - 5.5 Negative regulators of secondary metabolism biosynthesis
   - 5.6 Post-translational modifications of biosynthetic enzymes
6. Acknowledgements
7. References

---

### 1 Introduction

Natural products formed by type II polyketide synthases (PKSs) comprise an important and structurally diverse group of bacterial secondary metabolites.$^{1}$ Many of these compounds or their semisynthetic derivatives have emerged as clinically useful drugs or are promising drug candidates. The soil-borne and marine Gram-positive actinomycetes are a particularly rich source of such bioactive polyphenols, and represent the only known group of organisms that employ type II PKS systems for polyketide biosynthesis. With the advent of molecular tools and recombinant methods applicable to actinomycetes,$^{2,3}$ it has become feasible to investigate bacterial aromatic polyketide biosynthesis at the genetic and biochemical levels,$^{4}$ which has finally provided a basis for engineering novel natural product derivatives.$^{5-8}$ The enzymology of type II PKS systems has been the subject of excellent general reviews by Rawlings,$^{9}$ Shen,$^{10}$ and Reeves,$^{11}$ as well as specialized reviews on PKS starter units by Moore and Hertweck,$^{12}$ polyketide tailoring reactions by Rohr and coworkers,$^{13}$ and pathway engineering approaches by Mendez and Salas.$^{5}$ Nonetheless, the rapid developments in both cloning and analyzing type II PKS gene clusters, and biochemical as well as structure-biological investigations,$^{14}$ call for an update on the recent developments in type II PKS research. After a short introduction into the mode of action of the most valuable aromatic polyketides, this review summarizes the recent literature (2002–2005) pertaining to the type II PKS genes and enzymes that catalyze and regulate the biosynthesis of aromatic polyketides.

---

### 2 Biological activities of type II PKS-derived natural products

On the basis of the polyphenolic ring system and their biosynthetic pathways, metabolites produced by type II PKSs are classified as anthracyclines, angucyclines, aureolic acids, tetracyclines, tetracenomycins, pradimicin-type polyphenols, and benzoisochromanquinones. This brief overview introduces bioactive aromatic polyketides that have been the subject of recent biosynthetic studies at the molecular level.

Anthracyclines rank among the most effective anticancer drugs ever developed.$^{15,16}$ The first anthracyclines were isolated in the 1950s and were named rhodomycin and cinerubcin.$^{17,18}$ A few years later daunorubicin and doxorubicin were discovered.$^{19,20}$ The aglycone of anthracyclines consists of a tetracyclic ring system with quinone-hydroquinone groups in rings C and B, a methoxy substituent at C-4 in ring D and a short side chain at C-9, with a carbonyl at C-13. A sugar, daunosamine, is attached at C-7 of ring A (Fig. 1). The difference between doxorubicin and daunorubicin is that the side chain of doxorubicin terminates with a primary alcohol and daunorubicin terminates with a methyl. This minor structural difference, however, has an important impact on the

---

$a$ Leibniz Institute for Natural Product Research and Infection Biology, HKI, Beutenbergstr. 11a, 07745, Jena, Germany  
$b$ Universität Freiburg, Institut für Pharmazeutische Wissenschaften, Pharmazeutische Biologie und Biotechnologie, Stefan-Meier-Straße 19, 79104, Freiburg, Germany. E-mail: andreas.bechthold@pharmazie.uni-freiburg.de
Christian Hertweck (born 1969) studied chemistry, and obtained his Ph.D. on the subject of natural product synthesis under the supervision of Professor Boland at the Kekulé-Institute for Organic Chemistry and Biochemistry, Bonn, and at the Max-Planck-Institute for Chemical Ecology, Jena. In 1999, he became a Humboldt (Feodor Lynen) postdoctoral fellow of Professors Floss and Moore at the University of Washington, Seattle. From 2000 to 2005 he was head of a Junior Research Group at the Hans-Knöll-Institute, Jena. In 2006 he was appointed as Full Professor for Natural Product Chemistry at the Friedrich-Schiller-University and Head of the Department of Biomolecular Chemistry at the Leibniz-Institute for Natural Product Research and Infection Biology, Jena. His current research interests involve the elucidation and manipulation of microbial biosynthetic pathways.

Andriy Luzhetskyyy was born in Busovisko, Ukraine, in 1977. He studied genetics and biotechnology at Ivan Franko National University, Lviv, and in 2003 obtained his Ph.D. on the subject of the genetics of streptomycetes under the supervision of Professor Fedorenko, Lviv. In 2001 and 2002 he was an INTAS fellow of Professor Bechthold at the Albert-Ludwigs Universität, Freiburg. In 2004 he started his work as a postdoctoral researcher in the laboratory of Professor Bechthold. His major interests are the elucidation of biosynthetic pathways of antibiotics, the generation of new hybrid natural products through combinatorial biosynthesis, and the development of new genetic tools for the manipulation of actinomycetes.

Yuriy Rebets was born in Zhovkva, Ukraine, in 1978. He studied biology at Ivan Franko National University, Lviv. In 2001 he joined a graduate programme under the supervision of Dr Fedorenko, and in 2006 he obtained his Ph.D. on the subject of molecular genetics. In 2003 and 2004 he was a research fellow of Professor Nakamura at Niigata University of Pharmacy and Applied Life Sciences, Niigata, Japan. From 2002–2005 he was a DAAD fellow of Professor Bechthold at the Albert-Ludwigs Universität, Freiburg. In 2006 he started his work as a postdoctoral researcher in the laboratory of Professor Nakamura at the Niigata University of Pharmacy and Applied Life Sciences, Niigata, Japan. His current research is devoted to control of antibiotic production at the level of pathway-specific regulatory genes, and investigation of ion transporters in bacteria.

Andreas Bechthold (born 1962) studied pharmacy, and obtained his Ph.D. on the subject of natural product biosynthesis under the supervision of Professors Heide and Leistner at Rheinische Friedrich-Wilhelms-Universität, Bonn. In 1992, he became a DFG (Deutsche Forschungsgemeinschaft) postdoctoral fellow of Professor Floss at the University of Washington, Seattle. In 1993, he became a JSPS (Japanese Society for Pharmaceutical Sciences) postdoctoral fellow of Professor Tabata at Kyoto University. From 1994 to 1999 he was head of a Junior Research Group at Eberhard-Karls-Universität, Tübingen. In 2000 he was appointed as Associate Professor for Pharmaceutical Biology at the Christian-Albrechts-Universität, Kiel. In 2001 he was appointed as Full Professor for Pharmaceutical Biology and Biotechnology at the Albert-Ludwigs-Universität, Freiburg. His current research interests involve the elucidation and manipulation of microbial biosynthetic pathways.

---

spectrum of activity. Doxorubicin is an essential component in treatments for breast cancer, childhood solid tumours, soft tissue sarcomas, and aggressive lymphomas, while daunorubicin shows activity in acute lymphoblastic or myeloblastic leukaemias. Unfortunately, the use of these drugs may cause serious problems, such as cardiomyopathy and congestive heart failure. In addition, drug-resistant tumour cells survive treatment with both anticancer drugs.²¹ Attempts to generate or find novel anthracyclines with improved activity or reduced toxicity has resulted only in a few valuable compounds. Among these, epirubicin and idarubicin (Fig. 1) are now used in cancer therapy.²¹ Epirubicin differs from doxorubicin in the stereochemistry at position C-4 of the sugar. It shows different pharmacokinetic parameters from doxorubicin,

resulting in an enhanced total body clearance. Idarubicin does not contain the 4-methoxy group that is present in ring D of daunorubicin, and shows a broader spectrum of activity. Pirarubicin, aclacinomycin A (aclarubicin) and nogalamycin (Fig. 1) are further anthracyclines that have received clinical approval.²¹

It was indicated in early reports that pirarubicin is less cardiotoxic than doxorubicin, but disadvantages concerning the activity and toxicity of this drug were described later.²² The activity of anthracyclines seems to be based on different mechanisms of action. Intercalation into DNA, generation of free radicals, alkylation, DNA cross-linking, interference with helicase activity, inhibition of topoisomerase II and induction of apoptosis have been considered as being the most important mechanisms.²¹

This journal is © The Royal Society of Chemistry 2007  
Nat. Prod. Rep., 2007, 24, 162–190 | 163
at positions C-2 and C-6. Both compounds are DNA-dependent RNA polymerase inhibitors forming symmetrical dimers with DNA; the sugars of the trisaccharide side chains are essential for stabilizing the dimers. Interestingly, the acetoxy groups in two of the sugars of chromomycin contribute distinctively to DNA complex formation by providing an additional H-bond with the 2-amino groups of G bases and thus adding more specificity to the DNA binding.²⁹⁻³¹

One of the most important classes of antibiotics formed by type II PKSs are the tetracyclines. The first generation of tetracyclines, introduced in the 1950–1960s, includes the broad-spectrum antibiotics tetracycline, oxytetracycline, chlorotetracycline and demeclocycline (Fig. 3). The second generation of tetracyclines includes doxycycline and minocycline (Fig. 3).³² Tetracyclines are amphoteric due to a dimethylamino group and an acidic phenolic residue. Tetracyclines inhibit protein synthesis by interfering with the binding of aminoacylated tRNA to the A-site of the 30S subunit. In 2001 six different tetracycline-binding sites on the 30S subunit were identified.³³ Binding site I is located near to the A-site, the docking place of the aminoacylated tRNA. The five other binding sites are found at various locations in the 30S subunit, which explains the sometimes contradictory biochemical and functional data for tetracycline binding to the 30S subunits. The intensive use of tetracyclines has caused bacterial resistance against tetracyclines, resulting in severe drawbacks in their use. Attempts to develop tetracycline analogues that overcome these resistance mechanisms led to the discovery of the glycylcyclines. The most potent glycylcycline is the 9-tert-butylglycylamido derivative of minocycline, also known as tigecycline (Fig. 3). The glycylcyclines exhibit antibacterial activities comparable to the first-generation tetracyclines, but have a higher potency against tetracycline-resistant pathogens with efflux and ribosomal protection mechanisms of resistance.³⁴ Despite the structural similarities between the linear tetracyclic ring systems of the tetracyclines, anthracyclines, and aureolic acid precursors, they differ in their cyclization patterns and thus belong to separate classes of aromatic polyketides. The tetracenomycins (tetracenomycin and elloramycin) (Fig. 3), which have been isolated from *Streptomyces glaucescens* and *Streptomyces olivaceus*, also form a separate group of tetracyclic antibiotics, which exhibit moderate antibacterial activities.³⁵

Landomycin A and urdamycin A belong to the angucycline type of antibiotics (Fig. 4). The biological activities of the angucycline group metabolites are not restricted to any particular type of action. To date, five main areas of biological activities have been described: cytostatic activities, enzyme inhibition activities, inhibition of platelet aggregation, antibacterial and antiviral activities.³⁶ One of the most important compounds in the angucycline group of antibiotics is landomycin A. It possesses strong antitumour activities, in particular against prostate cancer cell lines. It inhibits thymidine uptake in murine smooth muscle cells and cell cycle progression.³⁷ Its unusual activities depend on the extended oligosaccharide chain, since landomycins with shorter sugar chains, such as landomycin E, containing three sugars, have much weaker antitumour activity.³⁸

Benzo[a]naphthacenes, such as the pradimicins and pluramycins (Fig. 4), also feature an angular polyphenolic ring system, and may be regarded as ‘extended angucyclines’. The pradimicins exhibit broad-spectrum antifungal and antiviral activity.³⁹ It has been

The formation of an anthracycline-DNA complex relies on defined structural determinants. The planar ring system is important for intercalation into DNA. Crystallographic studies of a complex of daunomycin and duplex DNA also showed the amino sugar daunosamine to be located in the minor groove of the DNA duplex, with the protonated daunosamine potentially contributing to the non-covalent binding of daunomycin. The amino and hydroxyl groups of the sugar face out of the minor groove, possibly contributing to the topoisomerase-inhibiting activity of the drug.²³⁻²⁶ Interestingly, DNA binding has also been shown for the sugar components of aclacinomycin A and nogalamycin.²⁷

The aureolic acids represent another group of potent anticancer drugs. Members of this family are chromomycin and mithramycin (Fig. 2), olivomycin, chromocyclomycin, UCH 9 and durhamycin A.²⁸ Mithramycin (which has been used clinically in several cancer therapies and Paget’s disease), as well as the chromomycins, contain a tricyclic chromophore with two aliphatic side chains at C-3 and C-7. Two sugar side chains are attached to each aglycone

Fig. 2 Structures of aureolic acid antibiotics.

This finding is of particular interest because some isoenzymes of GST play a crucial role in the detoxification of tumour cell lines that have acquired antitumour drug resistance.

The pradimicin-related pluramycins altromycin B and hedamycin (Fig. 4) are strong alkylating agents. They contain a 4H-anthra[1,2-b]pyrene structural motif. Both compounds exhibit antitumour and antimicrobial activities. The mechanism of action is by direct interaction with DNA through intercalation between the bases of the nucleotide, and alkylation of a guanine residue at 5′-PyG-3′ sites. NMR studies revealed that the dimethylated amino-substituted sugar vancosamine, a component of both compounds, interacts directly with pyrimidine units of the DNA.⁴⁴

Gilvocarcins belong to the benzo[d]naphtho[1,2-b]pyran-6-one C-glycoside antibiotics.⁴⁵ The most widely used compound is gilvocarcin V⁴⁶ (Fig. 4), which shows excellent antitumour activity and remarkably low toxicity.⁴⁷ The jadomycins are a unique family of angucycline-derived antibiotics because of their pentacyclic 8H-benz[b]oxazolo[3,2-f]phenanthridine backbone. Jadomycin B (Fig. 4) is the principal product of *Streptomyces venezuelae* ISP5230 when this strain is fermented under stress, such as heat shock, ethanol treatment, or phage infection.⁴⁸ The jadomycins show biological activity against Gram-positive and Gram-negative bacteria, and against yeast.⁴⁹ Interestingly, both gilvocarcin V and jadomycin B are angucycline derivatives (see below).

The benzoisochromanquinone class of antibiotics⁵⁰ comprises the antibiotics actinorhodin from *Streptomyces coelicolor*, medermycin from *Streptomyces* sp. AM7161, frenolicin from *Streptomyces roseofulvus*, griseusin A from *Streptomyces griseus*, and the antineoplastic agent granaticin from *Streptomyces olivaceus* (Fig. 5). The blue pigment actinorhodin is the best-studied type II PKS metabolite to date.⁴⁹ It should be highlighted once again that the pioneering experiments by Hopwood and co-workers on combining actinorhodin and medermycin biosynthesis genes led to the production of the first type II PKS hybrid metabolite, mederrhodin.⁵¹

Various polycyclic polyketide metabolites do not fit into the above-mentioned general classes. For example, the R1128 complex

Fig. 4 Structures of landomycin A, urdamycin A, altromycin, hedamycin, pradimicin, pluramycin, benastatin, gilvocarcin V and jadomycin B.

comprises several anthraquinones, which have been identified as novel non-steroidal estrogen-receptor antagonists.⁵²

The potent antitumour polyketide glycosides chartreusin (Fig. 6) and elsamicin feature a rare pentacyclic bislactone aglycone, named chartarin, which results from an unusual rearrangement sequence.⁵³⁻⁵⁶ Pharmacological studies revealed that chartreusin and its derivatives exert their antitumour activities through binding to DNA,⁵⁷ radical-mediated single strand breaks, and inhibition of topoisomerase II.⁵⁸ Resistomycin (Fig. 6) is an unusual naphthanthrene derivative isolated from *Streptomyces resistomycificus*⁵⁹ that exhibits a variety of pharmacologically relevant properties, e.g. inhibition of HIV-1 protease,⁶⁰ as well as RNA and DNA polymerase, and activity against Gram-positive bacteria and mycobacteria.⁶¹ More recently, it has also been

Fig. 5 Structures of selected benzoisochromanequinones.

implicated as a modulator of apoptosis, and may thus serve as  
a valuable tool in cell biology. ${}^{62}$  

The structurally fascinating $\beta$-rubromycin ${}^{63}$ and griseorhodin  
A, ${}^{64}$ as well as fredericamycin ${}^{65}$ (Fig. 6) are potent antitumoural  
agents. According to their biosynthetic models, these unusual  
polyphenols share early biosynthetic steps with pradimicin-type  
metabolites and then undergo intriguing oxidative rearrangement  
cascades. ${}^{64,66,67}$  

The most unusual metabolite formed by type II PKS is the  
bacteriostatic agent enterocin ${}^{68,69}$ (vulgamycin) ${}^{70}$ (Fig. 6), which  
has been isolated from terrestrial and marine *Streptomyces* species.  
Interestingly, the only aromatic part of the heavily rearranged  
molecule is the benzoyl-derived starter unit. ${}^{70-72}$

## 3 Enzymes involved in the formation of aromatic  
polyketides  

### 3.1 Production of aromatic polyketides by teamwork between  
enzymes  

In analogy to type II bacterial and plant fatty acid syntheses  
(FASs), type II PKSs are comprised of several individual  
enzymes. ${}^{4,73-75}$ Sequence information from numerous type II PKS  
gene clusters revealed a hallmark for the biosynthesis of aromatic  
polyketides: in all cases, a minimal set of iteratively used enzymes,  
each expressed from a distinct gene, is involved. In general, this so-  
called 'minimal PKS' consists of two ketosynthase units ($\mathrm{KS}_{\alpha}$ and  
$\mathrm{KS}_{\beta}$) and an acyl carrier protein (ACP), which serves as an anchor  
for the growing polyketide chain. ${}^{10}$ With a few exceptions, genes  
encoding these three proteins are grouped together, and show a  
typical $\mathrm{KS}_{\alpha} / \mathrm{KS}_{\beta} / \mathrm{ACP}$ architecture. Additional PKS subunits,  
including ketoreductases, cyclases and aromatases define the  
folding pattern of the nascent poly-$\beta$-keto intermediate. Finally,

Fig. 6 Structures of chartreusin, resistomycin, $\beta$-rubromycin, grise-  
orhodin A, fredericamycin A and enterocin.  

the polyphenols are tailored by oxygenases, glycosyl and methyl  
transferases. Typically, genes encoding these enzymes are clustered  
together. Since 2000, over 14 publications on the identification,  
cloning and sequencing of type II PKS gene clusters have appeared  
in the literature, highlighting the great interest in these systems  
(Table 1). In addition, with the advent of partial and whole-genome  
sequencing programs, as-yet uncharacterized and/or silent type II  
PKS gene clusters have been detected in actinomycetes. ${}^{76-79}$  

### 3.2 Polyketide chain assembly by the minimal PKS  

The minimal PKS catalyses the iterative decarboxylative condensation of malonyl-CoA extender units with an acyl starter unit (Fig. 7). Both subunits, $\mathrm{KS}_{\alpha}$ and $\mathrm{KS}_{\beta}$, show high sequence similarities. However, in contrast to $\mathrm{KS}_{\alpha}$, the $\mathrm{KS}_{\beta}$ component is lacking an active site cysteine, which is crucial for polyketide

KS
priming

$\begin{array}{c}
\text { KS }_{\alpha} \\
\text { SH }
\end{array}$

$\begin{array}{c}
\text { KS }_{\beta} \\
\text { SH }
\end{array}$

ACP

$\begin{array}{c}
\text { KS }_{\alpha} \\
\text { S }
\end{array}$

$\begin{array}{c}
\text { KS }_{\beta} \\
\text { S }
\end{array}$

ACP

$\begin{array}{c}
\text { KS }_{\alpha} \\
\text { HS }
\end{array}$

$\begin{array}{c}
\text { KS }_{\beta} \\
\text { S }
\end{array}$

ACP

$-\mathrm{CO}_{2}$

Claisen
condensation

$\begin{array}{c}
\text { R } \\
\text { O }=\stackrel{\mathrm{S}}{\longrightarrow} \mathrm{O} \\
\mathrm{R}
\end{array}$

MCAT or
self-
malonylation

$\begin{array}{c}
\text { O } \\
\mathrm{R}
\end{array}$

$\begin{array}{c}
\text { O } \\
\mathrm{R}
\end{array}$

iteration

Fig. 7 Basic mechanisms of aromatic polyketide biosynthesis.

Table 1 Type II PKS gene clusters analyzed since 2000.

| Metabolite       | Abbrev. | Strain/source                     | Year | Ref. |
|------------------|---------|------------------------------------|------|------|
| Enterocin        | Enc     | *Streptomyces maritimus*          | 2000 | 80   |
| R1128            | Zhu     | *Streptomyces sp.* R1128         | 2000 | 81   |
| Nogalamycin      | Sno     | *Streptomyces nogalater*          | 2001 | 82   |
| Rubromycin       | Rub     | *Streptomyces collinus*           | 2001 | 67$^{a}$ |
| Griseorhodin     | Grh     | *Streptomyces sp.* JP95           | 2002 | 64   |
| Medermycin       | Med     | *Streptomyces sp.* AM-7161        | 2003 | 83   |
| Gilvocarcin      | Gil     | *Streptomyces griseoflavus*      | 2003 | 84   |
| Hedamycin        | Hed     | *Streptomyces griseoruber*       | 2004 | 85   |
| Chromomycin      | Cmm     | *Streptomyces griseus*            | 2004 | 30   |
| Resistomycin     | Rem     | *Streptomyces resistomycificus*   | 2004 | 86   |
| Cervimycin       | Cer     | *Streptomyces tendae*             | 2004 | 87$^{b}$ |
| Chartreusin      | Cha     | *Streptomyces chartreusis*       | 2005 | 56   |
| Oviedomycin      | Ovi     | *Streptomyces antibioticus*       | 2005 | 88   |
| Fredericamycin   | Fdm     | *Streptomyces griseus*            | 2005 | 66   |

$^{a}$ Cluster sequence only published in GenBank. $^{b}$ Only partial sequence published.

Fig. 8 Model for a minimal PKS complex.

component of a minimal PKS, a malonyl-CoA:ACP transferase (MCAT). Since genes encoding MCATs are absent in most type II PKS gene clusters, it was proposed that an endogenous MCAT is recruited from fatty acid biosynthesis.$^{98,99}$ An alternative model was proposed by Simpson and co-workers, who demonstrated by *in vitro* studies that the ACP is capable of self-malonylation, albeit only at elevated concentrations of malonyl-CoA. Recently, *in vitro* self-malonylation of a chemically synthesized ACP was observed, clearly ruling out any contamination of purified ACP with traces of MCAT.$^{100}$ *In vivo*, however, at physiologically relevant concentrations, MCAT-dependent malonyl-ACP formation is still a likely alternative.$^{90,101-103}$ In this context the catalysis, specificity, and ACP docking site of *Streptomyces coelicolor* malonyl-CoA:ACP transacylase has been investigated.$^{104}$

Polyketide assembly by type II PKSs is notoriously difficult to study, since it is carried out by a multienzyme complex and not by a single enzyme. Furthermore, the proposed poly-$\beta$-keto intermediates are highly unstable and prone to spontaneous cyclization. Consequently, the biosynthetic machinery has long been regarded as a 'black box'. The recent structural study by Stroud, Khosla and co-workers, in which the authors were able to 'catch a type II PKS in action', was a breakthrough.$^{105}$ They provided direct proof for a KS heterodimer in showing that KS and CLF (or KS$_{\beta}$) have evolved highly complementary contacts. In addition, they deduced from the structure that a protein cleft keeps the nascent polyketide chain extended. In this fashion the reactive ketide groups are separated and probably contact the KS-CLF mainly as enols.$^{105}$ This study complements well the previously available structures of type II PKS components; primarily an X-ray structure of a priming KS (KAS III homologue) from the R1128 pathway,$^{106}$ and NMR solution structures of ACPs from the actinorhodin,$^{107}$ oxytetracycline$^{108}$ and frenolicin$^{109}$ PKSs.

3.3 Chain length

The chain length of polyketides synthesized by type II PKS is usually either 16 (octaketides, such as actinorhodin), 20 (decaketides, such as tetracenomycin), or 24 (dodecaketides, such as pradimicin). In a few cases, for example in the biosynthesis of griseorhodin (a tridecaketide), benastatin (a tetradecaketide) and fredericamycin (a pentadecaketide), even longer chains are formed (Fig. 9).

*In vivo* experiments with various PKS hybrids suggested that the KSβ controls the number of Claisen condensations during polyketide assembly, and was thus termed chain length factor (CLF). Furthermore, hybrid KS and CLF genes designed from sequences of the *act* (octaketide) and *tcm* (decaketide) genes reinforced the importance of CLF in defining chain length.

How is chain length determined? In principle, the size of the polyketide could be controlled by counting the number of condensation reactions, as has been suggested for 6-methylsalicylic acid (6-MSAS) or by measuring the length of the chain, as in fatty acid or plant polyketide biosynthesis. Cox and co-workers have addressed this issue by *in vitro* directed polyketide biosynthesis using a set of acylated C17S actinorhodin ACP surrogates and a mutated KSα/KSβ system, which is not capable of synthesizing the endogenous acetyl ACP starter. It was found that the number of elongations is dependent of the length of the starter unit, and consequently, the *act min* PKS governs chain length by measurement. Analogous observations with the cervimycin (*cer*) PKS strongly support the model of a protein cavity, which controls the size of the growing chain (C. Hertweck, unpublished observations).

Results from homology modelling based on crystal structures of the KS homodimer from *E. coli* FAS suggested that regions defining chain length are located at the interface of the dimer. Khosla and co-workers found that residues 109, 112 and 116 in the *act* CLF serve as gatekeepers in the polyketide tunnel, and that reducing the size of these residues lengthens the channel,

allowing two more elongation cycles. Stroud, Khosla and co-workers extended their model of this ‘gating mechanism’ in the context of KS/CLF structural studies, including Trp194, Leu143 and Phe140. They proposed that novel chain lengths could be engineered by opening or closing the gates.

Although the KSβ (CLF) represents the primary determinant of polyketide chain length, it may not be the only crucial regulatory element. Other type II PKS components, in particular cyclases, can also influence the number of chain elongations, as demonstrated by Shen *et al.* in a remarkable study on *whiE* spore pigment biosynthesis (see below). Almost simultaneously, Hunter and co-workers reported that disruption of an aromatase/cyclase from the oxytetracycline (*otc*) gene cluster of *Streptomyces rimosus* results in the production of novel polyketides with shorter chain lengths.

Consequently, it seems that the chain length is—at least to a limited degree—determined not only by the CLF, but also by the entire PKS complex. Interestingly, in the context of engineering hybrid systems, a fundamental incompatibility between antibiotic and spore pigment biosynthesis has been observed. Also it should be noted that to date no factor mediating the release of the fully elongated polyketide chain has been identified.

3.4 PKS priming

In most cases aromatic polyketide synthases are formally primed with acetate. At least in actinorhodin and tetracenomycin biosynthesis, it has been implicated that the PKS is primed by decarboxylation of a malonyl unit to yield an acetyl-S-KS intermediate (Fig. 10). Leadlay and co-workers demonstrated that the KSβ (CLF) has decarboxylase activity towards malonyl-CoA, in close analogy to the KSα of modular PKS (see above). Stroud *et al.* argued against this model on the basis of the solved *act* KS/CLF protein structure and suggested that CLF is a function of the KSα domain, and thus this issue is still a matter of debate. It is also relevant to point out that in the case of decarboxylation there is

![Figure 9](#fig9)

Fig. 9 Examples of varying polyketide chain lengths.

Fig. 10 Mechanism of type II PKS priming. a) Priming involving decarboxylation. b) Priming involving a KSIII/ACP module. c) Priming involving the direct loading of an acyl-CoA starter.

the requirement for transfer of the resulting acetate unit to the active site of the KS domain.

Various PKS employ alternative primers, such as propionate, (iso)butyrate, malonamate and benzoate. The biosynthesis and attachment of such alternative type II PKS primers has been extensively reviewed by Moore and Hertweck.¹² In short, in addition to decarboxylation of malonates, two alternative pathways have been proposed for non-acetate PKS primers.

One alternative involves an additional KSIII (and ACP) for the biosynthesis and attachment of short-chain fatty acids (Fig. 10). The latter scenario was first described in the context of anthracycline biosynthesis. It should be noted that in these cases, decarboxylative chain initiation by malonyl-ACP should even be suppressed in favour of unusual starter units.

Sequence analyses of the daunorubicin (*dnr*) and doxorubicin (*dox*) gene clusters by the Hutchinson¹²⁰ and Strohl¹²¹ groups revealed the presence of two additional PKS components, which were implicated in starter unit selection and chain initiation. These additional activities are represented by a FabH-like KS¹²² and an AT, encoded by *dpsC* and *dpsD*, respectively. As in bacterial fatty acid biosynthesis,¹²³ it has been suggested that DpsC functions as a KSIII, which catalyzes the first condensation of malonyl-CoA with propionyl-CoA to form a β-ketoester. Subsequently, the diketide would be loaded onto the min PKS by means of the AT DpsD.¹²¹,¹²² Independent *in vivo* studies by the Hutchinson and Strohl groups on the function of DpsC and DpsD showed that in the absence of the priming KSIII component DpsC, the *dps* PKS exhibits a relaxed starter unit specificity for acetyl-CoA instead of propionyl-CoA. However, propionate-derived polyketides may also be formed by the *dps* PKS in the absence of heterologously produced DpsC and DpsD.¹²⁴,¹²⁵ More recent *in vitro* studies

by Hutchinson and co-workers unequivocally demonstrated that DpsC is indeed responsible for the choice of the correct starter unit for daunorubicin biosynthesis.¹²⁶,¹²⁷

Other examples involving related PKS priming mechanisms include the biosynthesis of frenolicin, R1128, and hedamycin. In all cases, genes encoding KSIII components were identified in the respective gene clusters. Analysis of the frenolicin (*frn*) biosynthetic genes by the Hopwood group not only revealed an ORF encoding a FabH orthologue (FrnI), but also an additional ACP (FrnJ), which may serve as an intermediate anchor for the diketide.¹²⁸ Most likely the ACP-bound β-keto ester is processed to the fully saturated butyryl moiety before priming the PKS. Since no β-keto-processing enzymes were found in the sequenced *frn* gene locus, it was proposed that these activities were recruited from fatty acid metabolism (Fig. 11).

A similar scenario was found in the pathway of the R1128 complex, which incorporates butyryl, valeryl or 4-methylvaleryl starters. In addition to the KSIII gene (*zhuH*), Khosla and coworkers identified further malonyl AT (*ZhuC*) and priming ACP (*ZhuG*) components encoded by the R1128 (*zhu*) biosynthetic gene cluster. In analogy to anthracycline biosynthesis, it is assumed that ZhuH functions as a priming KS that is solely dedicated to the first malonyl-CoA condensation cycle.⁸¹ *In vitro* studies on the substrate specificity of ZhuH by Meadows and Khosla strongly supported this model.¹²⁹ In addition, it was found that the additional ACP (*ZhuG*) is indispensable for the incorporation of non-acetate starter units.¹³⁰ It should be highlighted that the *zhu* ketosynthases in the initiation (ZhuH) and elongation modules (ZhuA) of aromatic polyketide synthases have orthogonal acyl carrier protein specificity.¹³⁰ The crystal structure of the priming KSIII (ZhuH) has been disclosed by the Stroud and Khosla

DpsC  
Frnl  
ZhuH  

KSIII  

DpsD  
ZhuC?  

AT  

ACP  

SH  

O  

R  

HO  

O  

SCoA  

DpsG?  
FrnJ  
ZhuG  

DpsC  
Frnl  
ZhuH  

DpsG?  
ZhuG  

KSIII  

ACP  

S  

S  

O  

R  

O  

R  

O  

OH  

1° metabolism?  

FrnJ  
ZhuG  

ZhuG  

ACP  

ZhuC?  

AT  

S  

O  

R  

O  

S  

DpsA DpsB  

DpsG  

KSα KSβ  

ACP  

S  

O  

R  

O  

OH  

aklavinone  

FrnA FrnB  

FrnN  

ZhuA ZhuB  

ZhuN  

KSα KSβ  

ACP  

S  

O  

R  

O  

OH  

frenolicin  
R1128  
complex  

OH  

O  

COOH  

frenolicin B  

OH  

O  

OH  

O  

OH  

R = CH₃: R1128A  
R = CH₂CH₃: R1128B  
R = CH(CH₃)₂: R1128C  
R = CH₂CH₂CH₃: R1128  

OH  

O  

COOCH₃  

OH  

OH  

aklavinone  

Fig. 11 PKS priming involving KSIII components.

groups.¹⁰⁶ Furthermore, Khosla *et al.* succeeded in engineering modified aromatic polyketides by recombination of the R1128 'loading module' consisting of ZhuC, ZhuH, and ZhuG, with other min PKS components.¹¹⁹,¹³¹,¹³²

In the biosynthesis of hedamycin, which belongs to the plu- ramycin group of antibiotics, an iterative type I PKS is utilized for the formation of a short-chain starter unit (Fig. 12). Thorson and co-workers reported that an additional ketosynthase (KSIII), encoded by *hedS*, presumably plays a role in starter unit specificity in utilizing the unusual hexenoate starter unit in favour of the regular malonyl decarboxylation mechanism.⁸⁵ Here, the KSIII component represents a unique link between a type I (HedT)

HedT  

KS₀  

AT  

ACP  

SH  

O  

O  

OH  

HedU  

KS(I)  

AT  

DH  

KR  

ACP  

KS(II)  

iteration  

O  

S  

O  

HedS  

KSIII  

F  

S  

O  

O  

HedCDEF  

O  

O  

O  

O  

O  

S-ACP  

O  

O  

O  

O  

O  

Me₂N  

HO  

O  

OH  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

O  

Oand a type II PKS (HedCDE). In addition, a gene encoding a putative acyl transferase (HedF) has been located in the *hed* gene cluster, which might be involved in malonyl-CoA:ACP transfer or in loading the hexadienoate primer onto the PKS.

In conclusion, there is increasing evidence that KSIII components serve as gatekeepers for the min PKS. The presence of a KSIII and an additional ACP is always a good indication for non-acetate priming.¹³³ While—at least empirically—the choice of starters can be rationalized by the presence of these components, the exact PKS priming mechanism remains obscure.

The second alternative priming pathway is the direct loading by an acyl-CoA ligase and an AT, which are responsible for activation and transfer of the starters onto the PKS, respectively (Fig. 10). This mechanism has been proposed for malonamate and benzoate starters in the tetracycline and enterocin pathways, respectively¹² (Fig. 13). It would be rational to assume that these carboxylic acids, which clearly cannot be loaded by any of the other two priming modes, are activated as CoA thioesters by an acyl-CoA ligase (LIG) and then transferred onto the KS by an acyl transferase. Analysis of the enterocin biosynthesis gene cluster⁸⁰ by Moore and co-workers immediately lent support to this model, since gene candidates for AT (*encL*) and LIG (*encN*) were identified. The involvement of the ligase EncL was demonstrated by *in vivo* and *in vitro* studies.¹³⁴,¹³⁵ Substrate flexibility of both EncL and the *enc* PKS allowed for the activation and attachment of benzoyl-CoA surrogates. In a study on precursor-directed biosynthesis of enterocin and wailupemycin derivatives, it was shown that a number of non-natural aromatic and alicyclic primers were accepted by the *enc* PKS and fully processed.¹³⁴ Surprisingly, however, EncL does not seem to be required for chain initiation,¹³⁶ and also in this case the exact mechanism of KS priming remains to be established.

To date, the biosynthesis of the putative malonamyl-CoA starter in tetracycline and cervimycin biosynthesis has not been fully elucidated. However, it is likely that it involves an asparagine synthase-like amidase.⁸⁷

### 3.5 Ketoreductases

Ketoreductases (KRs) catalyze the stereospecific hydrogen transfer from NAD(P)H onto a keto group, resulting in the formation of a secondary alcohol. In aromatic polyketide biosynthesis, this reaction can either influence the orientation of the poly-β-keto chain for a favoured aldol condensation, which delineates the first ring closure, or define the configuration of a persisting carbinoi moiety. In most cases, KRs are an integral part of bacterial aromatic polyketide pathways. In such systems, KRs can be the first enzymes to affect the nascent polyketide chain prior to cyclization. Since the ketoreduction results in a transition from sp² to sp³ hybridization, inducing a defined bend of the poly-β-keto intermediate, the carbon chain may take up a favoured orientation for an aldol reaction between the carbons involved in the first ring formation. Consequently, ketoreduction has a pronounced role in dictating the structure of the final product, and thus at least indirectly contributes to the polyketide cyclization process⁹ (Fig. 14). On the other hand, incomplete KR-deficient sets of type II PKS enzymes may result in incorrectly folded polyketides. When the act KR was omitted from its endogenous type II PKS, the correctly folded act octaketide SEK4, as well as the incorrectly folded shunt product SEK4b, was formed.⁹⁴

Another very important observation of such mixed first ring cyclization events was reported by Kunnari *et al.* with randomly generated *S. peucetius* and *S. galilaeus* KR mutants.¹³⁷ Inactivation and complementation experiments using mithramycin, nogalamycin and aclacinomycin biosynthesis genes further support the involvement of ketoreductase in regiospecific cyclization processes¹³⁷,¹³⁸ (Fig. 15). It is remarkable that all type II PKS-derived polyketides that undergo ketoreduction, including benzisochromanones, tetracyclines, and anthracyclines, are similarly reduced at the ninth carbon from the carboxy terminus of the assembled polyketide. This strict regiochemistry is independent from the polyketide chain length (octaketide to decaketide) and the nature of the starter unit.⁹

Similarly, it is assumed that a KR may also be involved in preforming the ‘bent’ part of the polycyclic ring system of pradimicin-type polyketides (Fig. 16). In this regard it should be noted that KR may occur in cyclase-reductase didomain enzymes, as in griseorhodin biosynthesis.⁶⁴

Structural investigations of KR proteins could shed light on the biochemical basis of this important step in polyketide biosynthesis. In two independent studies the crystal structure of the actinorhodin polyketide reductase (ActIII) has been investigated. Hadfield *et al.* proposed a mechanism for ACP and polyketide binding,¹³⁹ whereas Korman *et al.* focused on cofactor binding

![Fig. 13](PKS priming with benzoate and malonamate.)

Fig. 13 PKS priming with benzoate and malonamate.
Fig. 14 Impact of the KR on cyclization pattern.

Fig. 15 Regioselective ketoreduction.

Fig. 16 Ketoreduction in pradimicin-type polyketide biosynthesis.

and substrate specificity.¹⁴⁰ Apart from the *act* KR, which was the first cloned KR controlling C-9 ketoreduction, genes encoding closely related enzymes were found in various other gene clusters involved in benzoisochromanonequinone biosynthesis, e.g. medermycin,⁸³ granaticin,¹⁴¹ and enterocin.⁸⁰ In many cases it has been possible to interchange ketoreductases from these different systems, and KRs have been shown to function with hybrid PKSs.⁸⁰,⁹³ Iterative type II polyketide synthases, cyclases and ketoreductases exhibit context-dependent behaviour in the biosynthesis of linear and angular decapolyketides.¹⁴² From

various mix-and-match experiments, it was hypothesized that the polyketide chain is completely assembled before reduction of the keto group takes place. However, as a more recent study revealed, the model for aromatic polyketide biosynthesis is likely to be oversimplified. While most minimal PKSs are able to operate in the absence of a KR and assemble polyketide chains, the enterocin PKS is surprisingly unable to produce polyketides without its endogenous KR. This suggests that if chain growth terminates at this stage, truncated pentaketides are either not released from the PKS complex, or are simply catabolized. Point mutations in the active site (S to A, and Y to F) of EncD implied that the KR is required to serve as a catalytic and not just a structural role. Consequently, ketoreduction in enterocin biosynthesis occurs during the chain elongation and not after, as proposed for most other type II PKS systems. Furthermore, it is remarkable that although co-expression of the act minimal PKS genes with encD results in the formation of the same shunt products, DMAP and aloesaponarin, absolutely no polyketide production was observed even when the encD null mutant was complemented with the functionally equivalent act KR. The observed differences between the enc and act KRs suggest that there may be two pathways for ketoreduction in type II PKS systems, either during polyketide assembly (EncD) or post-polyketide assembly (ActIII). The likely mode of catalysis by EncD involves its association with the KS and reduction of the KS-bound pentaketide intermediate. Two routes involving either cyclized (Staunton model) or open-chain intermediates are conceivable (Fig. 17).

In a few aromatic polyketide biosynthetic pathways the action of an additional KR can result in a chiral carbon centre, which does not undergo elimination and is still present in the final product. A prominent example is the antitumoural polyphenol doxorubicin from S. peucetius, which has chiral secondary hydroxy groups at C-17 and on the side chain. The C-17 keto group is stereospecifically reduced by aklaviketone reductase (DnrH), a result which has been experimentally proven by expression and feeding experiments (Fig. 18). Genes encoding related enzymes were found in the aclacinomycin/aklavinone (aknU), nogalamycin (snoaF) and chartreusin (chaL) biosynthesis gene clusters. Inactivation and complementation experiments showed that DnrU is required for reduction of the side chain keto group of anthracyclic polyketides.

![Figure 18](https://i.imgur.com/1234567.png)

Fig. 18 C-17 ketoreduction in anthracycline biosynthesis.

The benzoisochromanequinone frameworks of actinorhodin (or medermycin) and granaticin feature chiral carbon centres with opposite configuration. The key to the S- or R-configuration of the pyran rings are C-3 KRs, which catalyze hydride transfer from the pro-S or pro-R site, respectively. Ichi nose et al. demonstrated that ActVI-ORFI converts a bicyclic intermediate to a chiral alcohol, which is subsequently converted to a key intermediate in BIQ biosynthesis, (S)-DNPA. An orthologue of ActVI-ORFI is encoded in the medermycin (med) biosynthesis gene cluster (med-12) (Fig. 19). A corresponding KS from the granaticin pathway exhibits the opposite stereospecificity, yielding the enantiomeric alcohol. The C-3 KRs are capable of converting even synthetic surrogates to their corresponding secondary alcohols with very high enantioselectivity. It has been shown previously that inactivation of act-ORFl results in the formation of DMAC and aloesaponarin. Surprisingly, co-expression of the act min PKS, CYC and aromatase genes with gra KR, as well as complementation of the act KR with the graC-3 KR (Gra-ORF6) revealed that the KRs act context-dependently.

### 3.6 Cyclases

In the current model for aromatic polyketide biosynthesis, highly reactive poly-β-keto intermediates are formed, which can undergo spontaneous aldol reactions without enzyme stabilization or directed cyclizations (Fig. 20). The occurrence of spontaneous cyclizations resulting from incomplete sets of type II PKS systems was most impressively demonstrated by Shen et al. in a study on spore pigment biosynthesis. The metabolic profile of a

Fig. 17 Model of ketoreduction in the enterocin pathway.

So-called cyclases (CYCs), which function in a ‘chaperone-like’ manner, help in directing nascent polyketide intermediates into particular reaction channels. In the presence of these enzymes, spontaneous aldol chemistry is efficiently suppressed.

Since biochemical investigations of cyclases and their unstable substrates are particularly difficult, to date very little is known about the exact mode of action of these intriguing cofactor-free enzymes. With a few exceptions, their function could only be deduced by inactivation and recombination experiments, and elucidation of the (shunt) metabolites produced. Also, it could be demonstrated that (at least in e.g. anthracycline biosynthesis) cyclase-catalyzed C–C bond formations take place prior to dehydration mediated by aromatases (ARO, see below). Analyses of deduced gene products from various type II PKS gene clusters revealed that cyclases may occur as ‘didomain’ enzymes with internally duplicated motifs, such as ActVII in actinorhodin biosynthesis, 154,155 and that expression of the N- or C-termini alone resulted in loss of function. Alternatively, CYCs may also exist as ‘monodomain’ enzymes, e.g. TcmN.156 In various cases, these CYC domains are fused with other functionalities, for example as CYC/MT, or CYC/KR. Despite the large number of cloned CYC genes, the only biochemically characterized cyclases are the TcmF2 cyclase157 and TcmN158 from the tetracenomycin pathway, and the ester cyclases SnoaL and AknH involved in nogalamycin and aclacinomycin biosynthesis, respectively.159 In solution, these cyclases exist as a homotrimer (TcmF2 CYC), a homodimer (TcmN), and a monomer (SnoaL). Interestingly, the different groups of cyclases do not share unique common motifs (except for HxGTHxDxPxH, which is likely to form part of the active site of cyclases), and exhibit no phylogenetic relationship.

There is an ongoing discussion on whether or not KSα directs first ring cyclization. It has been reported that the first ring cyclization largely depends on the type of KS and, more importantly, on regioselective ketoreduction. However, it was implicated that folding of the polyketide chain is not dictated by the minimal polyketide synthase in the biosynthesis of mithramycin and anthracyclines, but rather by ketoreduction and cyclases.138 Also, in tetracenomycin biosynthesis, the cyclase TcmN seems to play a crucial role in first ring formation, directing the initial cyclization between C-9 and C-14. Meurer et al. concluded that KS, CYC and KR components exhibit context-dependent behaviour in the biosynthesis of linear and angular decapolyketides.142

Early inactivation and recombination experiments, which have been thoroughly reviewed by Rawlings9 and Shen,10 gave the first clue on the functions of downstream cyclases in tetracenomycin biosynthesis, such as TcmN, TcmI, and TcmJ, which were identified as second, third and fourth ring cyclases, respectively10,157,158 (Fig. 21).

The cyclization events in angucycline biosynthesis have also been the subject of many studies. Recently, functional analyses of jadI revealed the role of the deduced gene product in angucyclinone cyclization.160 Hautala et al. studied second and third ring cyclization in anthracycline biosynthesis, and identified dpsY and snoaM as cyclase genes from S. peucetius and S. nogalater, respectively. From analysis of the metabolites produced by mutants they concluded that the third ring closure is most likely mediated by the same enzyme that catalyzes the second ring closure.161 It is remarkable that in many cases cyclase genes are not interchangeable. While the incompatibility of enzymes

This journal is © The Royal Society of Chemistry 2007

Fig. 21 Cyclization events in tetracenomycin biosynthesis.

involved in the biosynthesis of non-reduced (tcm) and partially reduced (jad, dps) polyketides can be rationalized from the deviating folding of the polyketide chain, in other cases the failure of hybrid systems lacks an explanation. A recent example was reported by Wohlert *et al.*, who found that heterologously produced cyclases DpsY and JadI are incompatible in the presence of both the tetracenomycin (tcm) and daunorubicin (dps) PKSs. Conversely, Metsä-Ketelä *et al.* succeeded in engineering anthracycline biosynthesis (nogalamycin, aclacinomycin) toward angucyclines by co-expressing the angucycline-specific cyclase gene *pgaF* from a silent gene cluster. In this study, PgaF, which is similar to angucycline cyclases LanF, UrdF and JadI from the landomycin, urdamycin and jadomycin biosynthetic pathways, respectively, was identified as the third and fourth ring cyclase in angular systems (Fig. 22).

In anthracycline biosynthesis, cyclases such as CmmY (chromomycin), SnoaM (nogalamycin), and AknW (aclarubicin) are required for the complete cyclization of the second (and most probably the third) ring of linear polyketides. The fourth ring formation is catalyzed by a group of ester cyclases (DnrD, DauD, RdmA, AcmA, SnoaL). The enzyme-catalyzed aldol reaction usually results in the formation of a secondary alcohol with 9R configuration, except for nogalamycin and steffimycin, which are both 9S-configured. The cyclase SnoaL, which dictates the 9S configuration of anthracyclines from *S. nogalater*, has been characterized by Torkkell *et al.* (Fig. 23). In addition, Schneider, Mäntsälä and co-workers were able to solve the protein crystal structure of SnoaL, providing the first insight into a novel mechanism for enzymatic aldol condensation. Individual amino acids were spotted that differ between the

Fig. 22 Directing anthracycline biosynthesis towards angucyclines.

Fig. 23 Stereochemistry of the fourth ring ester cyclases.

9R and 9S specific enzymes, and the recent structural analysis of AknH helped to shed more light on the molecular basis of enantiospecificity of the ester cyclases.

It is remarkable that only very few basic cyclization patterns are realized in aromatic polyketide biosynthesis. Virtually all aromatic polyketides from bacteria are either linear (e.g. biq, anthracycline, aureolic acids, tetracenomycins and tetracyclines) or angular (e.g. angucyclines and pradimicin-type polyketides). A common structural feature is that in these systems rings are annellated only on one or two sides. A clear exception is the antibacterial polyketide resistomycin from *Streptomyces resistomycificus*. The naphthanthrene framework of this rare metabolite contains multiply annellated rings, and represents the only ‘discoid’ polyketide ring system found so far.

10x malonyl-CoA

EnzS
O
O
O
O
OH
OH
OH
OH

RemABCI
TcmKLMN

EnzS
O
O
O
O
OH
OH
OH
OH

TcmIJ

dehydrorabelomycin
angular

tetracenomycin F1
linear

resistomycin
discoid

Fig. 24 U-shape vs. S-shape folding in angular, linear and discoid polyketide pathways.

and Hertweck cloned and sequenced the gene locus encoding resistomycin (rem) biosynthesis, and identified several cyclase gene candidates, remI, remF, and remL.86 Mutational analyses and recombination experiments, together with elucidation of the shunt metabolites, produced gave an unexpected result. The resistomycin pathway shares early biosynthetic steps with tetracenomycin, but diverges from all other paths by folding the polyketide chain into an S-shape—as opposed to the regular U-shape (Fig. 24).

3.7 Aromatases

Apart from cyclases, the cyclization process is often supported by so-called aromatases. These enzymes are known to dehydrate cyclic alcohols to yield aromatic ring systems. In anthracycline biosynthesis, aromatases AknE1, RdmK, and SnoaE172,173 are responsible for the aromatization of the A-rings. Recently Yang, Rohr and co-workers identified JadH as an aromatase involved in the biosynthesis of jadomacin A and B. When jadH was deleted in the genome of Streptomyces venezuelae ISP 5230, a novel angucycline metabolite, 2,3-dehydro-UWM6, was formed. 2,3-Dehydro-UWM6 and UWM6, obtained by expression of the jadomycin PKS gene cluster in S. lividans, could be transformed into jadomycin B by a ketosynthase (jadA) mutant of S. venezuelae. These angucycline intermediates were also converted to the non-glycosylated jadomycin B derivative jadomycin A by transformants of the heterologous host S. lividans expressing the jadFGH oxygenases in vivo. After overexpression JadH was isolated for enzyme assays. UW6 was converted to rabelomycin and 2,3-dehydro-UW6 to dehydrorabelomycin, indicating that JadH possesses dehydrase and oxygenase activity174 (Fig. 25).

A further example of an aromatase is LanV, which is involved in landomycin biosynthesis. Expression of lanV in the rabelomycin-producing urdM mutant of Streptomyces fradiae resulted in the formation of 9-C-D-olivosyltetrangulol, urdamycin B and urdamycinone B. This result indicated that LanV is a bifunctional enzyme: it catalyzes the ketoreduction step at position C-6 of the polyketide core followed by a dehydration reaction, and thus aromatization of ring A of the angucycline (Fig. 26).175

Fig. 25 Ring aromatization in jadomycin biosynthesis.

in rhodomycin biosynthesis in *Streptomyces purpurascens*, has also been elucidated. RdmB strongly resembles DnrK (52% identical amino acids) and shows the typical methyltransferase fold and a SAM binding site. However, most surprisingly RdmB does not act as a methyltransferase but as a regiospecific hydroxylase. ${}^{178}$

### 4.2 Oxygenases

Oxidative tailoring reactions, which are catalyzed by oxygenases, are the key to the vast structural diversity of polyketides and to their biological activity. ${}^{13,179,180}$ In general, these enzymes promote the incorporation of oxygen into a substrate. Oxygenases are divided into monooxygenases and dioxygenases, depending on the number of oxygen atoms that are inserted into the substrates. Since monooxygenases reduce the second oxygen atom of dioxygen to water, they have also been termed ‘mixed function oxygenases’. Several types of oxygenases are known that play a role in post-PKS II modification: cytochrome P-450 monooxygenases (CYP 450), flavin-dependent mono- and dioxygenases, and anthrone oxygenases. These enzymes catalyze different reactions, e.g. hydroxylation, epoxidation, quinone formation, and oxidative rearrangement of the Baeyer–Villiger or the Favorskii type. Oxygenases differ significantly in their mode of action, requirement of cofactors and degree of flexibility toward the substrate.

#### Anthrone-type oxygenases. The oxidation of naphthacenone-and anthrone-type precursors of aromatic polyketides to the corresponding quinone derivatives is a common and important tailoring step in the biosynthesis of aromatic polyketides. Anthrone oxygenases, classified as internal monooxygenases, are responsible for these reactions. Interestingly, these enzymes lack the requirement for any cofactor. ${}^{181}$ Anthrone oxygenases use their substrates as a reducing equivalent for the reduction of one oxygen atom (from dioxygen) to water. ${}^{181}$ Quinone-forming monooxygenases involved in aromatic polyketide biosynthesis were first studied by Shen and Hutchinson in 1993 in the context of tetracenomycin biosynthesis in *S. glaucescens*. The monooxygenase TcmH was found to catalyze the conversion of the naphthacenone tetracenomycin F1 at position C-5 into the 5,12-naphthacenequinone tetracenomycin D3 (Fig. 27). ${}^{157}$ TcmH has a molecular weight of 12.5 kDa and forms a homotrimer in solution, as revealed by gel filtration. Inhibitor studies suggested that sulfhydryl groups and histidine residues are essential for catalysis. The second quinone-forming monooxygenase studied was ActVA-orf6 from *S. coelicolor* A3(2), which shows 39% identity to TcmH. ${}^{182}$ It was proposed that ActVA-orf6 catalyzes the formation of dihydrokalafungin (Fig. 27), an intermediate in actinorhodin biosynthesis. Since the natural substrate of ActVA-orf6 was not available, its catalytic activity was studied using tetracenomycin F1, the substrate of TcmH. Besides this tetracyclic compound, the protein also oxidizes tricyclic anthrones, suggesting a relaxed substrate specificity. ${}^{183}$ The ActVA-orf6 protein also does not require exogenous cofactors or metal ions, and there is no indication of any bound cofactors. In contrast to TcmH, which presumably possesses catalytically relevant sulfhydryl groups, ${}^{157}$ ActVA-orf6 does not contain any cysteine residues. Site-directed mutagenesis experiments implicate that residue His-52 is important for the catalytic activity of ActVA-orf6, as well as of TcmH. ${}^{181}$ Another internal monooxygenase is AknX, which has been purified and

---

**Fig. 26** Function of LanV as ketoreductase and aromatase involved in landomycin biosynthesis.

## 4 Polyketide-tailoring reactions

### 4.1 Methyltransferases

Most methyltransferases use the ubiquitous cofactor S-adenosyl-L-methionine (SAM) and transfer its activated methyl group to nitrogen, carbon or oxygen in a variety of target molecules. To date over twenty deduced amino acid sequences have been reported that might function as methyltransferases in *Streptomyces*. Among these are DnrK from *S. peuceticus*, DauK from *S.* sp. C5, CmmM1 from *S. griseus*, OxyF from *S. rimosus*, GilMT from *S. griseoflavus*, AlpH from *S. ambofaciens* and TcmN and TcmO from *S. glaucescens*, all involved in the biosynthesis of aromatic polyketides. ${}^{176}$ The degree of amino acid conservation among these enzymes is low, although they often share a chain-fold consisting of a central β-sheet with surrounding α-helices. In addition, a SAM binding site can be found in all of these enzymes. The function of some of these genes has been identified by gene disruption experiments and structure elucidation of the metabolites produced by the mutants. Recently the crystal structure of a ternary complex of DnrK with bound SAM and 4-methoxy-ε-rhodomycin T has been solved. ${}^{177}$ DnrK, a carminomycin 4-O-methyltransferase, methylates the 4-hydroxyl group of daunorubicin at a very late step in the biosynthesis. DnrK is a homodimer, and the subunit displays the typical fold of small molecule O-methyltransferases. The structure provides insights into the recognition of the anthracycline substrate and also suggests conformational changes as part of the catalytic cycle of the enzyme. The position and orientation of the bound ligands are consistent with an $S_N$2 mechanism of methyl transfer. Mutagenesis experiments on a putative catalytic base confirm that DnrK most likely acts as an entropic enzyme, in that rate enhancement is mainly due to orientational and proximity effects. DnrK exhibits a rather relaxed substrate specificity, as it accepts carminomycin, 13-deoxycarminomycin and ε-rhodomycin T as substrates. ${}^{177}$ The crystal structure of RdmB, an enzyme involved

that only four amino acid residues, Pro-44, Gly-45, Ala-65 and Trp-67, are fully conserved in these proteins.

Flavin-dependent oxygenases. Flavoenzymes are very versatile catalysts and are involved not only in the activation of oxygen for oxidation and hydroxylation reactions, but also in dehydrogenation reactions, in light emission, and in one- and two-electron transfer reactions. Flavin-dependent oxygenases often show high homologies within two regions of their amino acid sequences. The first region, which is usually located close to the N-terminus of the protein, has been implicated in the binding of the adenosine moiety of the FAD cofactor, whereas the second motif is associated with the binding of the flavin portion of FAD. Flavoprotein-catalyzed oxidative reactions associated with post-PKS modifications include hydroxylations, epoxidations, Baeyer–Villiger and Favorskii rearrangements. Baeyer–Villiger rearrangements are one of the most widespread FAD-dependent oxygenase reactions in the biosynthesis of type II PKS-derived metabolites. In general, flavin-containing Baeyer–Villiger monooxygenases (BVMO) employ NADPH and molecular oxygen to catalyze the insertion of an oxygen atom into a carbon–carbon bond of a carbonyl substrate. The catalytic mechanism of BVMO, which has been reviewed in detail, involves the formation of two crucial intermediates: flavin peroxide generated by the reaction of the reduced flavin with molecular oxygen, and a ‘Criegee’ intermediate resulting from the attack of the flavin peroxide onto the substrate. One of the best examples indicating the importance of BVMO as post-PKS II enzymes has been outlined in the biosynthetic pathway of the clinically used antitumour antibiotic mithramycin (Fig. 28) produced by S. argillaceus. The process that is responsible for conferring biological activity to the molecule has been identified by Prado *et al.* as the Baeyer–Villiger oxygenation of the tetracyclic intermediate premithramycin B, which is followed by hydrolytic ring-opening

Fig. 27 Reactions catalyzed by the anthrone-type oxygenases TcmH, ActVA-orf6, and AknX.

characterized by Ebizuka and co-workers more recently. This enzyme is involved in the oxidation of aklanonic acid anthrone to aklanonic acid (Fig. 27). In accordance with the findings regarding ActVA-orf6, Cys residues of AknX seem not to be important for catalytic activity of the enzyme, as a C52S mutant of AknX retained its oxygenase activity. Shen and Hutchinson proposed that the reaction mechanism of TcmH follows a radical process, which includes the generation of a superoxide anion radical. However, the anthrone oxygenase activity of AknX was not affected by superoxide dismutase. Moreover, mutants of AknX His-49, which corresponds to His-52 of ActVA6 and TcmH, retained 40–45% enzyme activity. This implies that His-49 –at least in the AknX enzyme reaction–is not an essential residue. Kinetic analyses of AknX mutants suggested that the Trp-67 residue plays a key role in the AknX oxygenation reaction. Due to the unique features of quinone-forming monooxygenases, it was assumed that they might share a novel protein fold. In 2003, X-ray analyses of ActVA-orf6 crystals revealed a ferredoxin-like fold with a novel dimeric assembly, but also indicating that the widely represented ferredoxin fold may sustain another functionality. The structure of ActVA-Orf6 suggests that four residues may be important for binding and/or catalysis: Tyr51, Asn62 and Trp66, which belong to β-sheets, and Tyr72 as part of an α-helix.

In addition to *aknX*, *tcmH*, and *actVA6*, several genes, such as *elmH* of *Streptomyces olivaceus* Tu2353, *dnrG* of *S. peucetius*, *dauG* of *Streptomyces* sp. strain C5, *snoaB* of *S. nogalater*, *mtmOIII* of *S. argillaceus*, and *frnU* of *S. roseofulvus*, have been reported to be possibly involved in similar oxygenation reactions. The putative aklanonic acid anthrone oxygenases AknX, DnrG, DauG, and SnoaB are closely related to each other and have a distal relationship with the other oxygenases like TcmH, ActVA6, MtmOIII and FrnU. Their amino acid sequence alignment shows

Fig. 28 Baeyer–Villiger oxygenation of premithramycin B leads to the active form of mithramycin.

of the lactone intermediate and the formation of a tricyclic mithramycin precursor.¹⁹³ Recently, Rohr and co-workers have purified the protein MtmOIV, which is responsible for the oxidative C–C bond cleavage of premithramycin B that leads to the active molecule mithramycin after decarboxylation. The authors isolated the lactone intermediate, premithramycin B-lactone, proving the Baeyer–Villiger mechanism in the formation of MtmOIV.¹⁹⁴

The involvement of the three another oxygenases (mtmOI, mtmOII, mtmOIII) in mithramycin biosynthesis was extensively reviewed by Rohr and co-workers in 2002.¹³ BVMOs have been investigated, in particular in the biosynthesis of angucyclines, such as urdamycin, jadomycin, and gilvocarcin. A unique hydroxylation mechanism was described for the biosynthesis of urdamycin A. The bifunctional oxygenase/reductase UrdM introduces the angular 12b-hydroxy group into urdamycin molecules by a Baeyer–Villiger mechanism.⁸⁴,¹⁹⁵ The accumulation of rabelomycin and urdamycin L in *S.* fradiae ΔurdM can be explained by the reaction scheme shown in Fig. 29.

![Fig. 29](https://i.imgur.com/1234567.png)

**Fig. 29** 12b-Hydroxylation of urdamycin A catalyzed by UrdM involves a Baeyer–Villiger monooxygenation reaction. Enz = enzyme.

Recently, Rohr and co-workers have demonstrated that BVMOs are involved in ring-opening reactions during the biosynthesis of jadomycin and gilvocarcin, something which is essential for the biological activity of these polyketides.⁴⁹,⁸⁴,¹⁹⁶

One of the key steps in gilvocarcin and jadomycin biosynthesis is the oxidative rearrangement sequence by which their unique chromophores are generated. GilOI and GilOIV, both responsible for the oxidative C–C bond cleavage, are the key enzymes in gilvocarcin biosynthesis. Gene inactivation of *gilOI* and *gilOIV* and structural elucidation of the metabolites produced by the mutants helped deduction of the ring-opening mechanisms (Fig. 30). It was shown that GilOI and GilOIV cooperate in a sequential action to promote the C–C bond cleavage, with GilOIV acting prior to GilOI.¹⁹⁶

It has been shown that JadF, JadH, and JadG are involved in extremely complex oxygenation/dehydration processes during jadomycin A biosynthesis. The inactivation and heterologous expression experiments show that JadF, JadH and JadG are arranged in an oligomeric protein complex.¹⁹⁷ Strikingly, non-functionality of one of these enzymes causes the loss of the activity of the other two enzymes. In addition to oxygenase activity, JadF and JadH also function as 2,3- and 4a,12b-dehydratases, respectively, and are responsible for the aromatization of ring A.¹⁷⁴ Apparently, these additional enzyme functions are not dependent on correct protein

![Fig. 30](https://i.imgur.com/8901234.png)

**Fig. 30** Model for the stepwise oxidative C–C bond cleavage in the biosyntheses of the gilvocarcins and the jadomycins.

complex formation. A general scheme of oxidative rearrangements during the biosyntheses of the jadomycins and the gilvocarcins is presented in Fig. 30.

The FAD-dependent monooxygenase ChaZ, which seems to work as a Baeyer–Villigerase, plays a key role in the biosynthesis of the potent antitumour agent and topoisomerase inhibitor chartreusin. The unusual structure of the bislactone aglycone and puzzling results of ${}^{13}$C labelling experiments hinted towards an intriguing biosynthetic scheme. Hertweck and co-workers demonstrated that inactivation of *chaZ* results in the accumulation of the anthracyclic intermediate or shunt product resomycin C,${}^{198}$ which revealed that chartreusin biosynthesis involves the unprecedented rearrangement of an anthracyclic ring system. According to the ${}^{13}$C-acetate labelling pattern, a quinone carbon bond is cleaved and a new C–C bond is formed between the carbonyl and the unsubstituted carbon of the C ring. A hydroxyl group could then be introduced either during the ring cleavage or by hydroxylation of the angucyclic intermediate by the putative hydroxylase ChaX. After formation of the polycyclic framework, the quinone oxygen, probably in the enol form, could attack the methyl ester carbonyl with substitution of methanol, forming the first lactone ring. Finally, the putative dioxygenase ChaP, which belongs to the vicinal oxygen chelate (VOC) superfamily of enzymes, could disrupt the dione moiety by attack of a dioxygen species and loss of carbon by rearrangement of pradimicin-type polyketides, yielding the spiro metabolite griseorhodin oxide. Subsequent lactone formation would ultimately furnish the chartarin aglycone (Fig. 31).${}^{56}$

Baeyer–Villigerases have also been implicated in the biosynthesis of the rare spiroketal metabolites rubromycin and griseorhodin. Cloning and analysis of the griseorhodin biosynthesis gene cluster from a tunicate-derived *Streptomyces* strain revealed the presence of a plethora of genes encoding putative oxidoreductases.${}^{64}$ On the basis of deduced gene functions, Li and Piel established a fascinating albeit hypothetical biosynthetic scheme. In this scheme, the formation of the spiroketal pharmacophore would require the cleavage of three C–C bonds to extrude a $C_2$ unit, followed by the rearrangement of a proposed pradimicin-type intermediate. Li and Piel pointed out the striking similarity between GrhO6 and MtmOIV, which suggests that this enzyme acts as a Baeyer–Villigerase. The model involving a tridecaketide intermediate is strongly supported by independent studies of Minas, Bailey and co-workers, who found that heterologous expression of a fragment of the rubromycin biosynthetic gene cluster from *S.* *collinus* in *S.* *coelicolor* resulted in the formation of the hexacyclic shunt product collinone.${}^{67}$ Interestingly, fredericamycin, which features a unique spiro carbon centre, seems to share early biosynthetic steps with griseorhodin and rubromycin. Most recently, Hutchinson, Shen and co-workers unveiled the molecular basis for fredericamycin biosynthesis by sequence analysis of the fredericamycin biosynthesis gene cluster from *S.* *griseus*.${}^{66}$

![Figure 31](attachment:Fig31.png)

Fig. 31 Model of chartreusin biosynthesis involving the oxidative rearrangement of an anthracyclic precursor.

Favorskiiase. In contrast to Baeyer–Villiger rearrangements, the occurrence of Favorskii-type oxidative rearrangements of α-ketones to the corresponding acids or esters is scarce. To date only one ‘favorskiiase’, EncM, which is involved in enterocin biosynthesis, has been characterized. The presumed substrate of EncM is a linear C-9-reduced octaketide that is oxidized at C-12 to form the 11,12,13-trione intermediate (Fig. 32). Moore and co-workers demonstrated that EncM facilitates two aldol condensations (between C-6 and C-11, and C-7 and C-14), generating chiral centres at each position. Usually, cyclases catalyze such aldol condensation reactions during aromatic polyketide assembly. However, the enterocin gene cluster is an exception among all other known type II PKS gene sets, as it lacks typical cyclase- and aromatase-encoding genes. EncM also participates in two heterocycle-forming reactions during the formation of desmethyl-5-deoxyenterocin. It would be tempting to employ this unique enzyme in combinatorial biosynthesis approaches to produce novel derailed polyketide metabolites. Unfortunately, the co-expression of *encM* with actinorhodin PKS genes did not lead to the production of any new compounds, suggesting that EncM may be specific for its endogenous type

II PKS or for benzoyl-primed polyketide precursors. The final hydroxylation step from 5-deoxyenterocin to enterocin is catalyzed by EncR, a cytochrome P-450 monooxygenase (see below) assisted by the translationally coupled ferredoxin-like *encQ* gene product.

Cytochrome-dependent P450 monooxygenases. Surprisingly, to date few CYP450 monooxygenases have been described in the context of bacterial aromatic polyketide biosynthesis. In addition to EncR, the monooxygenase responsible for the transformation of deoxyenterocin into enterocin (see above), DoxA from *S. peucetius* (and *S.* sp. strain C5) has been studied. The cytochrome P450 oxygenase participates in the biosynthesis of the clinically important antitumour antibiotics daunorubicin (DNR) and doxorubicin (DXR). DoxA exhibits a broad substrate specificity, catalyzing the anthracycline hydroxylation on both C-13 and C-14. Priestley and co-workers demonstrated that DoxA catalyzes the conversion of the anthracycline analogue desacetyladiamycin into the novel anthracycline metabolite 10-hydroxydesacetyladiamycin (Fig. 33). In these experiments DoxA showed an absolute requirement for molecular oxygen and NADPH.

![Figure 32](https://i.imgur.com/1234567.png)

Fig. 32 Proposed biosynthetic pathway of enterocin, indicating the role of EncM in post-PKS modifications.

![Figure 33](https://i.imgur.com/7654321.png)

Fig. 33 Enzymatic reactions catalyzed by DoxA.

4.3 Glycosyltransferases

Many bacterial aromatic polyketides contain deoxysugars as pivotal structural elements. The formation and attachment of these sugars is catalyzed by several enzymes encoded by genes that are also located in the biosynthetic gene clusters. The formation of nucleotide-activated sugars has been reviewed several times²⁰³,²⁰⁴ and is therefore not part of this review. This chapter focuses on the most important glycosyltransferases (GTs) involved in the glycosylation of aromatic polyketides, which are also of great interest for the engineering of novel bioactive metabolites.

The majority of GTs discovered in actinomycetes that are associated with secondary metabolism belong to the GT-1 family of inverting enzymes, using the CAZy classification system. This system is based on substrate specificity, type of catalyzed reaction and amino acid sequence similarity.²⁰⁵ So far, no crystal structure of a glycosyltransferase involved in the biosynthesis of type II PKS-derived compounds has been solved. However, GTs that decorate aromatic compounds are very similar to other microbial glycosyltransferases, for which three-dimensional structures have been disclosed.²⁰⁶–²⁰⁸ It is most likely that these enzymes show the so called GT-B fold, originally found in a phage T4 DNA-glucosyltransferase.²⁰⁹ This GT-B fold is characterized by two similar Rossmann-like folds, with the N-terminal domain providing the acceptor-binding site and the C-terminal domain providing the donor-binding site. Both domains are connected by a linker region, and the active site is located between these two domains. It has been proposed that the donor sugar binds to the C-terminal domain and, in addition, interacts with amino acids of the N-terminal domain.²⁰⁶,²⁰⁷,²¹⁰ Experimental evidence for the involvement of the N-terminal part came from a gene shuffling experiment using urdGT1b and urdGT1c. Genetic engineering of the enzyme regions responsible for selecting sugar and acceptor substrates in urdamycin GTs UrdGT1b and UrdGT1c has provided GTs with combined and/or novel specificity, leading to a new urdamycin derivative with an unusual branched saccharide side chain. Critical amino acids responsible for selective sugar binding as well as aglycone recognition were located in the N-terminal part.²¹¹,²¹²

Difficulties in obtaining highly purified active GTs and NDP-deoxysugars have hampered biochemical studies on the natural product GTs, and thus only a limited number of glycosyl-transferases have received detailed biochemical study. Examples are AknK and AknS, both involved in the biosynthesis of aclacinomycin.¹⁸²,²¹³,²¹⁴ AknS is a priming GT catalyzing the attachment of L-rhodosamin to the C-7 hydroxyl of the tetracyclic scaffold of aclacinomycin. The protein was overexpressed in *E. coli* in soluble form and purified as a His6-tagged fusion protein. Since dTDP-rhodosamine was not available to study the activity of AknS, dTDP-2-deoxyfucose was used as substrate. Enzyme activity was detectable, as AknS converted around 10% of 100 μM aklavinone to 2-deoxyfucosyl-aklavinone in 120 min. Surprisingly, 60% aklavinone was converted to 2-deoxyfucosyl-aklavinone when AknT was included in the reaction mixture, indicating that AknT is a potential activator or chaperone for AknS. Walsh and co-workers also reported the overexpression and purification of AknK. In *in vitro* experiments, AknK transferred 2-deoxy-L-fucose to the rhodosaminyl-aklavinone as well as 2-deoxy-L-fucose to 2-deoxyfucosyl-rhodosaminyl-aklavinone (the second transfer with lower efficiency). Both enzymes, AknS and AknK, have been studied on their ability to accept different sugar donors and aglycones. Beside dTDP-2-deoxy-L-fucose, the AknS/AknT complex also accepted dTDP-4-amino-2-deoxy-L-rhamnose as a substrate. The catalytic efficiencies were comparable to those seen with dTDP-2-deoxy-L-fucose. AknS/AknT also accepted rhodomycinone as substrate, which contains an additional hydroxyl group at position C-10. AknK also showed some substrate flexibility. Different deoxysugars (dTDP-2-deoxy-L-fucose and dTDP-L-daunosamine) and different aglycones (such as rhodosaminyl-aklavinone, daunomycin, adriamycin, and idarubicin) were accepted as substrates. As a result, a number of new potentially active anthracycline glycosides were generated.

The functions of many other GTs have been elucidated by gene deletion experiments and by combinatorial biosynthesis. In particular, GTs involved in the biosynthesis of the mithramycin (for details see ref. 13), urdamycin and landomycin sugar side chains have been investigated in detail. The urdamycin biosynthetic cluster revealed four GT-encoding genes, *urdGT2*, *urdGT1a*, *urdGT1b* and *urdGT1c*.²¹⁵–²¹⁸ The trisaccharide of urdamycin A is connected to the aglycone by an unusual C-glycosidic bond, which is of particular interest from the pharmaceutical viewpoint since they are stable towards glycosidase degradation. Inactivation of *urdGT2* gene resulted in production of three new compounds: urdamycins I, J and K. All lack the C-glycosidically attached D-olivose, showing UrdGT2 to be a C-olivosyltransferase.²¹⁸ The *S. fradiae* Tü2717ΔurdGT1a strain accumulates 12b-derhodinosyl-urdamycins, proving that UrdGT1a attaches L-rhodinose to the 12b-OH position of aquayamycin. *S. fradiae* Ax, a mutant lacking *urdGT1a*, *urdGT1b* and *urdGT1c*, produces aquayamycin together with urdamycinone B. Expression of *urdGT1a*, *urdGT1b* and *urdGT1c* individually and in various combinations in this mutant showed that UrdGT1c catalyzes the transfer of L-rhodinose to the 3'-OH group of the first attached D-olivose, and that UrdGT1b is responsible for attaching the terminal D-olivose, finishing the biosynthesis of urdamycin A.²¹⁷,²¹⁸ (Fig. 34). Landomycin A is a hexasaccharide glycoside produced by *Streptomyces cyanogenus*.²¹⁹ The hexasaccharide chain consists of six deoxygenated sugars—four D-olivose and two L-rhodinose moieties. Surprisingly, only four GT genes (*lanGT1*, *lanGT2*, *lanGT3* and *lanGT4*) were identified within the landomycin biosynthetic cluster.²²⁰ Moreover, heterologous expression of the landomycin cluster in a *S. fradiae* mutant led to the production of landomycin A, proving that all four *lanGT*s are responsible for the formation and attachment of the hexasaccharide chain.²²¹ Inactivation of *lanGT2* led to the production of tetrangulol, which is the progenitor of the landomycin aglycone.²²² The lack of the sugar side chain indicates that LanGT2 catalyzes the priming glycosyltransfer during the hexasaccharide biosynthesis, which is the attachment of a D-olivose to O-8 of the polyketide backbone.

In contrast to landomycinone, tetrangulol lacks two hydroxy groups at C-6 and C-11. It has been shown by feeding experiments that the O-atoms at positions 6 and 11 derive from molecular oxygen. Recently, both oxygenases catalyzing these reactions, LanM2 and LanZ4/Z5, were identified.³⁸,¹⁷⁹ As both hydroxyls are missing in tetrangulol, the oxygenation steps apparently occur after the attachment of the first D-olivose moiety by LanGT2. Remarkably, the oxygen at C-6 is first introduced by an acetate building block during the polyketide assembly, subsequently removed by

Fig. 34 Biosynthesis of the sugar side chains of landomycin A and urdamycin A.

deoxygenation and reintroduced by an oxygenase. This unusual, energy-consuming pathway can be explained by the narrow substrate specificity of LanGT2, which accepts only tetrangulol as acceptor substrate and not landomycinone. The S. cyanogenus mutant containing a deleted *lanGT1* gene produced two new compounds, landomycin H and 8-olivosyl-landomycinone. Both lack the second D-olivose, indicating that LanGT1 is the second D-olivosyltransferase of the landomycin pathway.²² The D-olivosyltransferase function of LanGT1 has also been proved by heterologous expression.²³ A LanGT4 mutant accumulated landomycin D with two sugars, indicating that LanGT4 attaches L-rhodinose to the 3-OH group of the second D-olivose moiety.¹⁸⁰ A mutant lacking *lanGT3* produced landomycin E (consisting of three sugars attached to the aglycone), indicating that LanGT3 is an olivosyltransferase responsible for the transfer of the fourth sugar.²⁴ Finally, attachment of the last two deoxysugars (D-olivose and L-rhodinose) is catalyzed by iteratively acting LanGT1 and LanGT4, as shown by bioconversion experiments²²⁵ (Fig. 34).

More than 40 compounds have been generated by combinatorial biosynthesis using only GT genes from the landomycin and urdamycin clusters, indicating the potency of genetic methods for the generation of novel compounds.³⁸,¹⁸⁰,²¹⁷,²¹⁸,²²⁶–²²⁹

5 Regulation of aromatic polyketide biosynthesis

Streptomycetes are saprophytic bacteria typically found in terrestrial and marine ecosystems. During their life cycle they are exposed to a huge diversity of environmental signals, such as nutritional starvation, osmotic pressure and pH shift. Secondary metabolism causes additional commitments on the cells in the use of energy and materials required for production of antibiotics. Due to these reasons, biosynthetic pathways leading to the formation of secondary metabolites are under strict regulation by internal and external signals also affecting the morphological differentiation of colonies. In all cases studied, the onset of antibiotic production occurs in the late exponential or early stationary phase of growth in liquid culture, indicating temporal control of antibiotic production.²³⁰,²³¹ This control of antibiotic production is performed by various mechanisms and regulatory elements. In several studies on antibiotic biosynthesis genes, the accumulation of mRNA in the course of growth indicated that the control of their expression is carried out at the level of transcription by means of dedicated transcriptional factors.²³²,²³³ The activity of these regulators is controlled by other regulatory mechanisms of higher rank. All of the regulatory elements actively interact with each other to ensure flexible and sensitive control over gene expression. We will focus in this chapter on the regulation of type II PKS biosynthetic pathways including more general and highly pathway-specific gene functions.

5.1 Genes that affect secondary metabolism and morphological differentiation

Mutations affecting both secondary metabolite formation and morphological differentiation have been described for *S. coelicolor*. These genes were named *bld* ('bald' – due to the lack of an aerial mycelium and spore formation).²³⁴ Almost all *bld* mutants are conditional and can restore the wild-type phenotype upon changes in cultivation conditions. The best studied among them is the *bldA* locus, which codes for a minor tRNA translating the leucine UUA codon.²³⁵ This codon is extremely rare in GC-rich genomes of actinomycetes, and is implicated in the regulation of gene expression by the translational selection of mRNA.²³⁶ The presence of rare leucine TTA codons within the coding region of many SARP genes (see below) is well documented.²³⁶,²³⁷ It has been shown that the *bldA* primary transcript has to undergo post-translational maturation, and the active tRNA is only formed at the beginning of the stationary phase that coincides with the onset of antibiotic production.²³⁸ In the context of actinorhodin biosynthesis, it has been demonstrated that the
expression of an *actII-ORF4* carrying a mutated TTA codon led to *bldA*-independent antibiotic biosynthesis.²³⁷,²³⁹ However, the exact function of the *bldA* product (tRNA<sup>UA</sup>) is still not fully understood. Recent studies on the regulatory gene *ccaR* and some antibiotics resistance genes revealed that they are mistranslated in *bldA* mutants of *S. clavuligerus* and *S. coelicolor*.²⁴⁰,²⁴¹

TTA codons were found within the *lndI* and *lanI* genes that control the production of landomycins E and A in *S. globisporus* and *S. cyanogenus*, respectively. The availability of leucyl tRNA<sup>UA</sup> in *S. globisporus* cells is a limiting factor for *lndI* gene expression. The presence of the rare TTA codon within *lndI* caused a delay between gene transcription and translation of its mRNA and replacement of TTA with CTC resulted in the loss of this delay.²⁴²

Other *bld* genes encode a sigma factor (*BldN*),²⁴³ an anti-sigma factor (*BldG*),²⁴⁴ an oligopeptide importer (*BldK*),²⁴⁵ a response regulator required for morphogenesis (*BldM*),²⁴⁶ a transcriptional regulator (*BldD*),²⁴³ a putative transcriptional factor of the MerR family (*BldC*),²⁴⁷ and a DNA binding protein (*BldB*).²⁴⁸ However, not all *bld* genes are implicated in regulation of production of actinorhodin and other antibiotics by *S. coelicolor*. Some of them influence only the morphological differentiation.

### 5.2 Genes that affect secondary metabolism and growth rate

A regulatory function has also been described for ppGpp (guanosine tetraphosphate) in initiating actinorhodin biosynthesis under nitrogen limitation. The exact mechanism is not yet understood, but when a modified *relA* gene, encoding a ppGpp synthetase, was highly expressed in *S. coelicolor*, transcription of *actII-orf4*, the pathway-specific regulator of actinorhodin biosynthesis (see below), occurred.²⁴⁹ A ppGpp-independent signalling pathway (as yet unknown) must exist under phosphate limitation. Mutations of the two-component system *PhoR-PhoP* in *S. lividans* resulted in reduced levels of alkaline phosphatase activity, reduced phosphate transport at low phosphate concentrations, and increased production of actinorhodin. This might indicate that a phosphorylated *PhoP* is repressing transcription of actinorhodin biosynthetic genes. In addition, inactivation of the polyphosphate kinase PPK involved in the regulation of intracellular phosphate concentration also resulted in an increase in actinorhodin production.²⁵⁰

### 5.3 General signals affecting secondary metabolism

γ-Butyrolactones are produced by many streptomycetes. A-factor, which has been studied in detail in *S. griseus*, is the best characterized γ-butyrolactone. It is involved in binding to the cytoplasmic protein ArpA (A-factor receptor protein) which, after binding, is released from the *adpA* promoter, resulting in AdpA formation. AdpA is required for StrR formation, which regulates the onset of streptomycin biosynthesis.²⁵¹

The involvement of γ-butyrolactones in type II PKS-derived natural product formation has been studied for actinorhodin biosynthesis in *S. coelicolor*. SCB1, one of four γ-butyrolactones found in the strain,²⁵²,²⁵³ evokes the production of actinorhodin. The SCB1 binding factors have been named ScbA and ScbR. Interestingly, mutations in *scbA* and *scbR* led to an increase and to a reduction in actinorhodin biosynthesis respectively.²³¹,²⁵³ The exact function of these proteins in regulating actinorhodin biosynthesis

is not fully understood. ScbR is not interacting with the promoter region of *actII-ORF4*, but is binding to the promoter of a pathway-specific regulatory gene (*kasO*) that seems to control the biosynthesis of a putative type I PKS-derived polyketide. This might be explained by a regulatory cross-talk between both metabolic pathways.²³¹

Recently a gene, *jadW1*, coding for a γ-butyrolactone-associated protein (similar to *afsA* and *scbA* of *S. griseus* and *S. coelicolor*) has also been found in *S. venezuelae*. Disruption of *jadW1* resulted in the shut-down of chloramphenicol and jadomycin B production, indicating the involvement of this protein in the regulation of these processes in *S. venezuelae*. Furthermore, the precise analysis of the *jadR2* gene product showed that it can act as a γ-butyrolactone-binding protein, indicating the involvement of these compounds in the jadomycin biosynthesis regulation.²⁵⁴

#### 5.4 SARP, pathway-specific regulators affecting secondary metabolism

Many genes coding for a pathway-specific transcriptional regulator belong to the SARP (*Streptomyces* Antibiotics Regulatory Protein) family. Some of these proteins seem to recognize heptameric repeats within the promoter region of genes that they regulate. The DNA-binding domain of proteins of the SARP family resembles the DNA binding domain of the osmoregulator OmpR, which has been the subject of several studies.²⁵⁵–²⁵⁷ SARPs have been found in many type II PKS gene clusters.

ActII-ORF4 was shown to interact with promoters of the ‘early’ *act* genes and with one promoter that directs the expression of *act* genes encoding enzymes involved in tailoring reactions.²⁵⁸ High copy number plasmids carrying these promoter regions reduced *actI-ORF1* and *actVI-ORFA* transcription in the wild type as judged by an overall decrease in actinorhodin production. The phenotype was restored by the introduction of additional copies of *actII-ORF4*. ActII-ORF4 has a higher affinity to the *actVI-ORF1-ORFA* intergenic region than to promoters of early biosynthetic genes. This seems to prevent the accumulation of intermediates of the biosynthesis, which might elicit oxidative stress.

The recent *Streptomyces* genome sequencing projects set the stage for studying the global gene expression in *S. coelicolor* using microarray technology.²⁷,²⁹ It was found that six genes named *eca* (expression coordinated with *act*) distant from the *act* locus have the temporal pattern of transcription that is highly coordinated with the expression of *actII-ORF4*.²³³ The 5′-regions of some of *eca* genes contain imperfect repeats, which are known to be ActII-ORF4 recognition and binding sites.²³² In the absence of *actII-ORF4*, genes governing biosynthesis of undecylprodigiosin (*red*) are characterized by an elevated level of transcription. On the other hand, constitutive expression of *actII-ORF4* causes a decreased transcription rate of *red* and enhancement of *cda* (governing biosynthesis of calcium-dependent antibiotics) gene activity.²³³ *actII-ORF4* also influences the expression of the *whiE* locus and the non-ribosomal peptide coelichelin biosynthesis gene cluster, as well as catalase and superoxide dismutase genes. In this way it also increases the antioxidative defence before the onset of actinorhodin production. Furthermore, it was shown that ActII-ORF4 activates the expression of silent type II PKS genes from *S. antibioticus* after introduction into *S. coelicolor*.²⁵⁹ This data indicates that SARPs can act not only within the respectivebiosynthesis gene clusters, but also pleiotropically influence the expression of other genes at the level of mRNA. Moreover, these experiments demonstrated that the regulatory gene *redZ*, which was believed to be pathway-specific, controls the expression of all type II PKS-derived secondary metabolites known to be produced by *S. coelicolor*.

Another higher-rank actinorhodin-associated transcriptional regulator is AtrA, which has been identified in the crude protein extract of *S. coelicolor* A3(2). It binds to the *actII-ORF4* promoter region.²⁶⁰ *AtrA*, which has been localized outside the actinorhodin biosynthetic gene cluster, encodes a protein belonging to the TetR family of transcriptional factors. Interestingly, an *atrA* mutant produces actinorhodin several hours later than the WT-strain. This can be rationalized by considering that the synthesis of ActII-ORF4 is not perfectly blocked in the *atrA* mutant, and after accumulation of a sufficient amount of ActII-ORF4, actinorhodin is produced.²⁶⁰

Several regulatory genes have been identified in the daunorubicin biosynthetic gene cluster. DnrI, which has high similarity to ActII-ORF4, controls the transcription of all structural genes and *drrA* and *drrB*, which encode ATP-dependent transporter systems.²⁶¹ Introduction of *dnrI* into the *S. coelicolor actII-ORF4* mutant restored actinorhodin biosynthesis, showing the interchangeability of these two regulators.²⁶² Biochemical analyses revealed that DnrI exists as a homodimer that recognizes specific sequences within the respective promoters.²⁶³ The C-terminal region of the protein plays an important role in the formation of protein–protein contacts. Truncation of only 10 amino acid residues from this part of DnrI led to complete loss of the ability of the protein to activate daunorubicin production, while its DNA-binding activity remained intact.²⁶⁴ In contrast, changes within the N-terminal region of DnrI reduced its DNA-binding affinity.

Two divergently transcribed genes, *dnrN* and *dnrO*, have been found in the daunorubicin gene cluster.²⁶¹ ORF *dnrN* encodes a transcriptional regulator controlling *dnrI* transcription.²⁶⁵ Several regions within the *dnrN* promoter show similarity to daunorubicin intercalation sites. Daunorubicin might bind to the promoter of *dnrN*, leading to a decrease in *dnrN* transcription and antibiotic production.²⁶¹

DnrO is a transcriptional factor with a typical helix-turn-helix (HTH) DNA-binding structure in its N-terminus, characteristic of proteins belonging to the DeoR family.²⁶⁶ It was shown that DnrO binds to the intergenic region between *dnrN* and *dnrO* controlling the expression of both proteins.²⁶¹,²⁶⁴,²⁶⁶

In analogy to ActII-ORF4 and DnrI, JadR1 functions as a SARP in jadomycin biosynthesis.²⁶⁷ Disruption of *jadR1* led to the complete cessation of jadomycin B production, while the introduction of additional *jadR1* copies into *S. venezuelae* resulted in an increased titre of jadomycin B.

The regulatory proteins involved in landomycin A and landomycin E biosynthesis are LanI and LndI.²⁶⁸ Both regulators are very similar to JadR1. Interestingly, *lanI* and *lndI* are located on opposite sites of the gene clusters. Disruption of *lndI* caused complete loss of landomycin E production, while its overexpression led to an increase in antibiotic biosynthesis.²⁶⁸,²⁶⁹ LndI binds to promoter regions of the structural landomycin E genes and to its own promoter, indicating an autoregulation process.²⁴² The same effect on landomycin A production was observed when LanI was investigated (Y. Rebets, unpublished work).

Five putative regulatory genes were found within the angucycline-like antibiotic alpomycin and orange pigment biosynthesis gene cluster of *S. ambofaciens*, located within terminal inverted repeats of the chromosome of the strain.²⁶ One of them, *alpV*, codes for a transcriptional activator of the SARP family. Disruption of this gene caused complete cessation of both alpomycin and orange pigment production, as a consequence of failure in the activation of structural gene expression.²⁷⁰

### 5.5 Negative regulators of secondary metabolism biosynthesis

Only a few examples of negatively acting regulators are known to participate in the control of biosynthetic pathways involving type II PKS. The best-studied cases are the regulation of the non-polyketide antibiotic methylenomycin biosynthesis and transcriptional control of resistance genes.²⁷¹–²⁷³ However, several pathways, in which negative regulators directly affect the production of secondary metabolites have been reported.

The *absA1A2* locus of *S. coelicolor* encodes a two-component histidine kinase-response regulator pair. Recently it was shown that the phosphorylated form of AbsA2 acts a negative regulator. Mutations in the *absA1A2* locus that enhanced antibiotic biosynthesis impair AbsA2 kinase activity, while mutations that repress antibiotic synthesis impair phosphatase activity.²⁷⁴

One gene of the jadomycin biosynthetic gene cluster, *jadR2*, encodes a protein with significant similarity to the group of TetR transcriptional factors with the typical HTH DNA-binding domain,⁴⁸ and to the A-factor receptor proteins from various streptomycetes.²⁵⁴ Disruption of *jadR2* led to increased jadomycin B production (even in the absence of stress factors), confirming that JadR2 acts as a repressor of jadomycin B biosynthesis.

### 5.6 Post-translational modifications of biosynthetic enzymes

*S. coelicolor* proteome investigations revealed numerous post-translational protein modifications. It was shown that 6 out of the 14 identified actinorhodin biosynthesis enzymes undergo modification by covalent attachment of small molecules or partial proteolytic digestion with the removal of leader peptides.²⁷⁵,²⁷⁶ In the case of the dehydratase ActIV-ORF3 the proteolytic cut-out of the first 31 or 34 amino acid residues assists the translocation of the protein across the membrane. Apparently this protein is secreted by (and non-covalently associated with) the cell wall.

The putative proteinase gene named *prxl* was identified within the *lnd* gene cluster.²⁷⁷ Disruption of this gene caused a significant decrease in antibiotic biosynthesis. As the wild phenotype could not be restored by overexpression of the positive regulator *lndI*, a direct influence of Prx1 activity on structural genes or their products has to be assumed.

## 6 Acknowledgements

The authors’ original research on type II PKS has been generously funded by the Deutsche Forschungsgemeinschaft, DFG, (HE 3469/1 to C.H.; BE 1389/6-1 to A.B.) and by a BMBF research grant as part of the GenoMik competence network ‘Secondary Metabolite Biosynthesis Genes from Streptomycetes’ (to C.H. and A.B.). We thank Dr Mikko Metsä-Ketelä, University of Turku, Finland, for critically reading the manuscript and for helpful comments.
7 References

1. D. O'Hagan, *The Polyketide Metabolites*, Ellis Horwood, Chichester, 1991.
2. T. Kieser, M. J. Bibb, M. J. Buttner, K. F. Chater and D. A. Hopwood, *Practical Streptomyces Genetics*, The John Innes Foundation, Chichester, 2000.
3. B. A. Pfeifer and C. Khosla, *Microbiol. Mol. Biol. Rev.*, 2001, **65**, 106.
4. D. A. Hopwood, *Chem. Rev.*, 1997, **97**, 2465.
5. C. Mendez and J. A. Salas, *Comb. Chem. High Throughput Screening*, 2003, **6**, 513.
6. A. Luzhetskyy and A. Bechthold, *Mol. Microbiol.*, 2005, **58**, 3.
7. B. S. Moore and J. Piel, *Antonie van Leeuwenhoek*, 2000, **78**, 391.
8. C. Khosla, R. S. Gokhale, J. R. Jacobsen and D. E. Cane, *Annu. Rev. Biochem.*, 1999, **68**, 219.
9. B. J. Rawlings, *Nat. Prod. Rep.*, 1999, **16**, 425.
10. B. Shen, *Top. Curr. Chem.*, 2000, **209**, 1.
11. C. D. Reeves, *Crit. Rev. Biotechnol.*, 2003, **23**, 95.
12. B. S. Moore and C. Hertweck, *Nat. Prod. Rep.*, 2002, **19**, 70.
13. U. Rix, C. Fischer, L. L. Remsing and J. Rohr, *Nat. Prod. Rep.*, 2002, **19**, 542.
14. G. Schneider, *Curr. Opin. Struct. Biol.*, 2005, **15**, 629.
15. R. B. Weiss, *Semin. Oncol.*, 1992, **19**, 670.
16. G. Batist, *Cancer Chemother. Biol. Response Modif.*, 2001, **19**, 47.
17. H. Brockmann and K. Bauer, *Naturwissenschaften*, 1950, **37**, 492.
18. L. Ettlinger, E. Gäumann, R. Hutter, W. Keller-Schierlein, F. Kradolfer, L. Neipp, V. Prelog, P. Reusser and H. Zähner, *Chem. Ber.*, 1959, **92**, 1867.
19. H. Di Marco, R. Silvestrini, M. Gaetani, M. Soldati, P. Orezzi, T. Dasdia, B. M. Scarpino and L. Valentini, *Nature*, 1964, **201**, 706.
20. H. Di Marco, R. Si, M. Gaetani and B. M. Scarpino, *Cancer Chemother.*, 1969, **53**, 33.
21. G. Minotti, P. Menna, E. Salvatorelli, G. Cairo and L. Gianni, *Pharmacol. Rev.*, 2004, **56**, 185.
22. E. Koh, Y. Ueda, T. Nakamura, A. Kobayashi, S. Katsuta and H. Takahashi, *Pediatr. Res.*, 2002, **51**, 256.
23. E. Ragg, R. Mondelli, C. Battistini, A. Garbesi and F. P. Colonna, *FEBS Lett.*, 1988, **236**, 231.
24. F. Zunino, G. Pratesi and P. Perego, *Biochem. Pharmacol.*, 2001, **61**, 933.
25. M. Trieb, C. Rauch, B. Wellenzohn, F. Wibowo, T. Loerting, E. Mayer and K. R. Liedl, *J. Biomol. Struct. Dyn.*, 2004, **21**, 713.
26. J. S. Thorson and T. Vogt, *Glycosylated natural products*, ed. C.-H. Wong, Wiley-VCH, Weinheim, 2003.
27. S. P. Sim, B. Gatto, C. Yu, A. A. Liu, T. K. Li, D. S. Pilch, E. J. LaVoie and L. F. Liu, *Biochemistry*, 1997, **36**, 13285.
28. J. Rohr, C. Méndez and J. A. Salas, *Bioorg. Chem.*, 1999, **27**, 41.
29. M. H. Hou and A. H. Wang, *Nucleic Acids Res.*, 2005, **33**, 1352.
30. N. Menendez, M. Nur-e-Alam, A. F. Brana, J. Rohr, J. A. Salas and C. Mendez, *Chem. Biol.*, 2004, **11**, 21.
31. M. A. Keniry, E. A. Owen and R. H. Shafer, *Biopolymers*, 2000, **54**, 104.
32. I. Chopra and M. Roberts, *Microbiol. Mol. Biol. Rev.*, 2000, **65**, 232.
33. M. Pioletti, F. Schlunzen, J. Harms, R. Zarivach, M. Gluhmann, H. Avila, A. Bashan, H. Bartels, T. Auerbach, C. Jacobi, T. Hartsch, A. Yonath and F. Franceschi, *EMBO J.*, 2001, **20**, 1829.
34. L. R. Peterson, *Diagn. Microbiol. Infect. Dis.*, 2005, **52**, 163.
35. E. Egert, M. Noltemeyer, J. Siebers, J. Rohr and A. Zeeck, *J. Antibiot.*, 1992, **45**, 1190.
36. J. Rohr and R. Thiericke, *Nat. Prod. Rep.*, 1992, **9**, 103.
37. B. R. T. Crow, R. Smith, Y. Guo, K. Ramos and G. Sulikowski, *Bioorg. Med. Chem. Lett.*, 1999, **9**, 1663.
38. B. Ostash, U. Rix, L. L. Rix, T. Liu, F. Lombó, A. Luzhetskyy, O. Gromyko, C. Wang, A. F. Brana, C. Mendez, J. A. Salas, V. Fedorenko and J. Rohr, *Chem. Biol.*, 2004, **11**, 547.
39. M. Tsunakawa, M. Nishio, H. Ohkuma, T. Tsuno, M. Konishi, T. Naito, T. Oki and H. Kawaguchi, *J. Org. Chem.*, 1989, **54**, 2532.
40. S. Okuyama, M. Kakushima, H. Kamachi, M. Konishi and T. Oki, *J. Antibiot.*, 1993, **46**, 500.
41. T. Aoyagi, T. Aoyama, F. Kojima, N. Matsuda, M. Maruyama, M. Hamada and T. Takeuchi, *J. Antibiot.*, 1992, **45**, 1385.
42. T. Aoyama, F. Kojima, F. Abe, Y. Muraoka, H. Naganawa, T. Takeuchi and T. Aoyagi, *J. Antibiot.*, 1993, **46**, 914.
43. T. Aoyama, F. Kojima, T. Yamazaki, T. Tatee, F. Abe, Y. Muraoka, H. Naganawa, T. Aoyagi and T. Takeuchi, *J. Antibiot.*, 1993, **46**, 712.
44. E. A. Owen, G. A. Burley, J. A. Carver, G. Wickham and M. A. Keniry, *Biochem. Biophys. Res. Commun.*, 2002, **290**, 1602.
45. K. Takahashi, M. Yoshida, F. Tomita and K. Shirahata, *J. Antibiot.*, 1981, **34**, 271.
46. D. M. Balitz, F. A. O’Herron, J. Bush, D. M. Vyas, D. E. Nettleton, R. E. Grulich, W. T. Bradner, T. W. Doyle, E. Arnold and J. Clardy, *J. Antibiot.*, 1981, **34**, 1544.
47. M. Morimoto, S. Okubo, F. Tomita and H. Marumo, *J. Antibiot.*, 1981, **34**, 701.
48. K. Yang, L. Han and L. C. Vining, *J. Bacteriol.*, 1995, **177**, 6111.
49. S. W. Ayer, A. G. McInnes, P. Thibault, J. A. Walter, J. L. Doull and T. Parnell, *Tetrahedron Lett.*, 1991, **32**, 6301.
50. K. Ichinose, T. Taguchi, Y. Ebizuka and D. A. Hopwood, *Actinomycetologica*, 1988, **12**, 99.
51. S. Omura, H. Ikeda, F. Malpartida, H. Kieser and D. A. Hopwood, *Antimicrob. Agents Chemother.*, 1986, **29**, 13.
52. Y. Hori, S. Takase, N. Shigematsu, T. Goto, M. Okuhara and M. Kohsaka, *J. Antibiot.*, 1993, **46**, 1063.
53. B. E. Leach, K. M. Calhoun, L. E. Johnson, C. M. Teeters and W. G. Jackson, *J. Am. Chem. Soc.*, 1953, **75**, 4011.
54. E. Simonitsch, W. Eisenhuth, O. A. Stamm and H. Schmid, *Helv. Chim. Acta*, 1964, **47**, 1459.
55. M. Konishi, K. Sugawara, F. Kofu, Y. Nishiyama, K. Tomita, T. Miyaki and H. Kawaguchi, *J. Antibiot.*, 1986, **39**, 784.
56. Z. Xu, K. Jakobi, K. Welzel and C. Hertweck, *Chem. Biol.*, 2005, **12**, 579.
57. X. Salas and J. Portugal, *FEBS Lett.*, 1991, **292**, 223.
58. A. Lorico and B. H. Long, *Eur. J. Cancer*, 1993, **29A**, 1985.
59. H. Brockmann and T. Reschke, *Tetrahedron Lett.*, 1968, **27**, 3167.
60. B. E. Roggo, F. Petersen, R. Delmendo, H. B. Jenny, H. H. Peter and J. Roesel, *J. Antibiot.*, 1994, **47**, 136.
61. I. Haupt, U. Waehnert, C. Pitra, G. Loeber, G. Luck and K. Eckardt, *Z. Allg. Mikrobiol.*, 1975, **15**, 411.
62. Y. Shiono, N. Shiono, S. Seo, S. Oka and Y. Yamazaki, *Z. Naturforsch.*, 2002, **57c**, 923.
63. C. Puder, S. Loya, A. Hizi and A. Zeeck, *Eur. J. Org. Chem.*, 2000, 729.
64. A. Li and J. Piel, *Chem. Biol.*, 2002, **9**, 1017.
65. R. C. Pandey, M. W. Toussaint, R. M. Stroshane, C. C. Kalita, A. A. Aszalos, A. L. Garretson, T. T. Wei, K. M. Byrne, R. F. J. Geoghegan and R. J. White, *J. Antibiot.*, 1981, **34**, 1389.
66. E. Wendt-Pienkowski, Y. Huang, J. Zhang, B. Li, H. Jiang, H. Kwon, C. R. Hutchinson and B. Shen, *J. Am. Chem. Soc.*, 2005, **127**, 16442.
67. R. Martin, O. Sterner, M. A. Alvarez, E. De Clercq, J. E. Bailey and W. Minas, *J. Antibiot.*, 2001, **54**, 239.
68. N. Miyairi, H.-I. Sakai, T. Konomi and H. Imanaka, *J. Antibiot.*, 1976, **29**, 227.
69. Y. Tokuma, N. Miyairi and Y. Morimoto, *J. Antibiot.*, 1976, **29**, 1114.
70. H. Seto, T. Sato, S. Urano, J. Uzawa and H. Yonehara, *Tetrahedron Lett.*, 1976, 4367.
71. C. Hertweck and B. S. Moore, *Tetrahedron*, 2000, **56**, 9115.
72. C. Hertweck, A. P. Jarvis, L. Xiang, B. S. Moore and N. J. Oldham, *ChemBioChem*, 2001, **2**, 784.
73. J. W. Campbell and J. E. J. Cronan, *Annu. Rev. Microbiol.*, 2001, **55**, 305.
74. B. J. Rawlings, *Nat. Prod. Rep.*, 1998, **15**, 275.
75. D. J. Payne, P. V. Warren, D. J. Holmes, Y. Ji and J. T. Lonsdale, *Drug Discovery Today*, 2001, **6**, 537.
76. X. Pang, B. Aigle, J.-M. Girarget, S. Mangenot, J.-L. Pernodet, B. Decaris and P. Leblond, *Antimicrob. Agents Chemother.*, 2004, **48**, 575.
77. H. Ikeda, J. Ishikawa, A. Hanamoto, M. Shinose, H. Kikuchi, T. Shiba, Y. Sakaki, M. Hattori and S. Omura, *Nat. Biotechnol.*, 2003, **21**, 526.
78. S. Omura, H. Ikeda, J. Ishikawa, A. Hanamoto, C. Takahashi, M. Shinose, Y. Takahashi, H. Horikawa, H. Nakazawa, T. Osonoe, H. Kikuchi, T. Shiba, Y. Sakaki and M. Hattori, *Proc. Natl. Acad. Sci. U. S. A.*, 2001, **98**, 12215.
79. S. D. Bentley, K. F. Chater, A. M. Cerdeno-Tarraga, G. L. Challis, N. R. Thomson, K. D. James, D. E. Harris, M. A. Quail, H. Kieser, D. Harper, A. Bateman, S. Brown, G. Chandra, C. W. Chen, M. Collins, A. Cronin, A. Fraser, A. Goble, J. Hidalgo, T. Hornsby, S. Howarth, C. H. Huang, T. Kieser, L. Larke, L. Murphy, K. Oliver, S. O'Neil, E. Rabbinowitsch, M. A. Rajandream, K. Rutherford, S. Rutter, K. Seeger, D. Saunders, S. Sharp, R. Squares, S. Squares, K. Taylor,

T. Warren, A. Wietzorrek, J. Woodward, B. G. Barrell, J. Parkhill and D. A. Hopwood, *Nature*, 2002, **417**, 141.

80 J. Piel, C. Hertweck, P. Shipley, D. S. Hunt, M. S. Newman and B. S. Moore, *Chem. Biol.*, 2000, **7**, 943.

81 T. Marti, Z. Hu, N. L. Pohl, A. N. Shah and C. Khosla, *J. Biol. Chem.*, 2000, **275**, 33443.

82 S. Torkkeli, T. Kunnari, K. Palmu, P. Mäntsälä, J. Hakala and K. Ylihonko, *Mol. Genet. Genomics*, 2001, **266**, 276.

83 K. Ichinose, M. Ozawa, K. Itou, K. Kunieda and Y. Ebizuka, *Microbiology*, 2003, **149**, 1633.

84 C. Fischer, F. Lipata and J. Rohr, *J. Am. Chem. Soc.*, 2003, **125**, 7818.

85 T. Bililign, C.-G. Hyun, J. S. Williams, A. M. Czisny and J. S. Thorson, *Chem. Biol.*, 2004, **11**, 959.

86 K. Jakobi and C. Hertweck, *J. Am. Chem. Soc.*, 2004, **126**, 2298.

87 K. Herold, Z. Xu, F. A. Gollmick, U. Gräfe and C. Hertweck, *Org. Biomol. Chem.*, 2004, **2**, 2411.

88 F. Lombó, A. F. Brana, J. A. Salas and C. Mendez, *ChemBioChem*, 2004, **5**, 1181.

89 R. McDaniel, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *J. Am. Chem. Soc.*, 1993, **115**, 11671.

90 W. Bao, E. Wendt-Pienkowski and C. R. Hutchinson, *Biochemistry*, 1998, **37**, 8132.

91 C. Bisang, P. F. Long, J. Cortes, J. Westcott, J. Crosby, A. L. Matharu, R. Cox, T. J. Simpson, J. Staunton and P. F. Leadlay, *Nature*, 1999, **401**, 502.

92 R. McDaniel, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *Science*, 2005, **375**, 549.

93 H. Fu, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *J. Am. Chem. Soc.*, 1994, **116**, 4166.

94 H. Fu, D. A. Hopwood and C. Khosla, *Chem. Biol.*, 1994, **1**, 205.

95 B. J. Rawlings, *Nat. Prod. Rep.*, 2001, **18**, 231.

96 B. J. Rawlings, *Nat. Prod. Rep.*, 2001, **18**, 190.

97 J. Staunton and K. J. Weissman, *Nat. Prod. Rep.*, 2001, **18**, 380.

98 T. S. Hitchman, J. Crosby, K. J. Byrom, R. J. Cox and T. J. Simpson, *Chem. Biol.*, 1998, **5**, 35.

99 A. L. Matharu, R. J. Cox, J. Crosby, K. J. Byrom and T. J. Simpson, *Chem. Biol.*, 1998, **5**, 699.

100 C. J. Arthur, A. Szafranska, S. E. Evans, S. C. Findlow, S. G. Burston, P. Owen, I. Clark-Lewis, T. J. Simpson, J. Crosby and M. P. Crump, *Biochemistry*, 2005, **44**, 1541.

101 C. W. Carreras and C. Khosla, *Biochemistry*, 1998, **37**, 2084.

102 W. P. Revill, M. J. Bibb and D. A. Hopwood, *J. Bacteriol.*, 1995, **177**, 3946.

103 J. Dreier, A. N. Shah and C. Khosla, *J. Biol. Chem.*, 1999, **274**, 25108.

104 A. T. Keatinge-Clay, A. A. Shelat, D. F. Savage, S. C. Tsai, L. J. Miercke, J. D. O’Connell, C. Khosla and R. M. Stroud, *Structure*, 2003, **11**, 147.

105 A. T. Keatinge-Clay, D. A. Maltby, K. F. Medzihradszky, C. Khosla and R. M. Stroud, *Nat. Struct. Mol. Biol.*, 2004, **11**, 888.

106 H. Pan, S.-C. Tsai, E. S. Meadows, L. J. W. Miercke, A. T. Keatings-Clay, J. O’Connell, C. Khosla and R. M. Stroud, *Structure*, 2002, **10**, 1559.

107 M. P. Crump, J. Crosby, C. E. Dempsey, J. A. Parkinson, M. M. and T. J. Simpson, *Biochemistry*, 1997, **36**, 6000.

108 S. C. Findlow, C. Winsor, T. J. Simpson, J. Crosby and M. P. Crump, *Biochemistry*, 2003, **42**, 8423.

109 Q. Li, C. Khosla, J. D. Puglisi and X.-C. Liu, *Biochemistry*, 2003, **42**, 4648.

110 K. K. Burson and C. Khosla, *Tetrahedron*, 2000, **56**, 9401.

111 I. D. G. Campuzano and P. M. Shoolingin-Jordan, *Biochem. Soc. Trans.*, 1998, **26**, S284.

112 M. Moche, K. Dehesh, P. Edwards and Y. Lindqvist, *J. Mol. Biol.*, 2001, **305**, 491.

113 J. M. Jez, M. B. Austin, J. L. Ferrer, M. E. Bowman, J. Schroder and J. P. Noel, *Chem. Biol.*, 2000, **7**, 919.

114 K. Herold, F. A. Gollmick, I. Groth, M. Roth, K.-D. Menzel, U. Möllmann, U. Gräfe and C. Hertweck, *Chem.–Eur. J.*, 2005, **11**, 5523.

115 W. Huang, J. Jia, P. Edwards, K. Dehesh, G. Schneider and Y. Lindqvist, *EMBO J.*, 1998, **17**, 1183.

116 Y. Tang, S.-C. Tsai and C. Khosla, *J. Am. Chem. Soc.*, 2003, **125**, 12708.

117 Y. M. Shen, P. Yoon, T.-W. Yu, H. G. Floss and B. S. Moore, *Proc. Natl. Acad. Sci. U. S. A.*, 1999, **96**, 3622.

118 H. Petkovic, A. Thamchaipenet, L.-H. Zhou, D. Hranueli, P. Raspor, P. G. Waterman and I. S. Hunter, *J. Biol. Chem.*, 1999, **274**, 32829.

119 T. S. Lee, C. Khosla and Y. Tang, *J. Am. Chem. Soc.*, 2005, **127**, 12254.

120 C. R. Hutchinson, *Chem. Rev.*, 1997, **97**, 2525.

121 J. Ye, M. L. Dickens, R. Plater, Y. Li, J. Lawrence and W. R. Strohl, *J. Bacteriol.*, 1994, **176**, 6270.

122 A. Grimm, K. Madduri, A. Ali and C. R. Hutchinson, *Gene*, 1994, **151**, 1.

123 L. Han, S. Lobo and K. Reynolds, *J. Bacteriol.*, 1998, **180**, 4481.

124 G. Meurer and C. R. Hutchinson, *J. Am. Chem. Soc.*, 1995, **117**, 5899.

125 V. B. Rajgarhia and W. R. Strohl, *J. Bacteriol.*, 1997, **179**, 2690.

126 W. Bao, P. J. Sheldon, E. Wendt-Pienkowski and C. R. Hutchinson, *J. Bacteriol.*, 1999, **181**, 4690.

127 W. Bao, P. J. Sheldon and C. R. Hutchinson, *Biochemistry*, 1999, **38**, 9752.

128 M. J. Bibb, D. H. Sherman, S. Omura and D. A. Hopwood, *Gene*, 1994, **142**, 31.

129 E. S. Meadows and C. Khosla, *Biochemistry*, 2001, **40**, 14855.

130 Y. Tang, T. S. Lee, S. Kobayashi and C. Khosla, *Biochemistry*, 2003, **42**, 6588.

131 Y. Tang, N. Y. Lee, H. Y. Lee and C. Khosla, *Tetrahedron*, 2004, **60**, 7659.

132 Y. Tang, T. S. Lee and C. Khosla, *PLoS Biol.*, 2004, **2**, 0227.

133 J. A. Salas, *Chem. Biol.*, 2004, **11**, 892.

134 J. A. Kalaitzis, M. Izumikawa, L. Xiang, C. Hertweck and B. S. Moore, *J. Am. Chem. Soc.*, 2003, **125**, 9290.

135 L. Xiang and B. S. Moore, *J. Bacteriol.*, 2003, **185**, 399.

136 C. Hertweck, L. Xiang, J. A. Kalaitzis, Q. Cheng, M. Palzer and B. S. Moore, *Chem. Biol.*, 2004, **11**, 461.

137 T. Kunnari, K. Ylihonko, A. Hautala, K. D. Klika, P. Mäntsälä and J. Hakala, *Bioorg. Med. Chem. Lett.*, 1999, **9**, 2639.

138 J. Kantola, G. Blanco, A. Hautala, T. Kunnari, J. Hakala, C. Mendez, K. Ylihonko, P. Mäntsälä and J. Salas, *Chem. Biol.*, 1997, **4**, 751.

139 A. T. Hadfield, C. Limpkin, W. Teartasin, T. J. Simpson, J. Crosby and M. P. Crump, *Structure*, 2004, **12**, 1865.

140 T. P. Korman, J. A. Hill, T. N. Y. Vu and S. C. Tsai, *Biochemistry*, 2004, **43**, 14529.

141 D. Tornus and H. G. Floss, *J. Antibiot.*, 2001, **54**, 91.

142 G. Meurer, M. Gerlitz, E. Wendt-Pienkowski, L. C. Vining, J. Rohr and C. R. Hutchinson, *Chem. Biol.*, 1997, **4**, 433.

143 P. L. Bartel, C.-B. Zhu, J. S. Lampel, D. C. Dosch, N. C. Connors, W. R. Strohl, J. M. Beale and H. G. Floss, *J. Bacteriol.*, 1990, **172**, 4816.

144 M. L. Dickens, J. Ye and W. R. Strohl, *J. Bacteriol.*, 1996, **178**, 3384.

145 K. Räty, J. Kantola, A. Hautala, J. Hakala, K. Ylihonko and P. Mäntsälä, *Gene*, 2002, **293**, 115.

146 N. Tsukamoto, I. Fujii, Y. Ebizuka and U. Sankawa, *J. Bacteriol.*, 1994, **176**, 2473.

147 N. Lomovskaya, S. L. Otten, Y. Doi-Katayama, L. Fonstein, X.-C. Liu, T. Takatsu, A. Inventi-Solari, S. Filippini, F. Torti, A. L. Colombo and C. R. Hutchinson, *J. Bacteriol.*, 1999, **181**, 305.

148 K. Ichinose, C. Surti, T. Taguchi, F. Malpartida, K. I. Booker-Milburn, G. R. Stephenson, Y. Ebizuka and D. A. Hopwood, *Bioorg. Med. Chem. Lett.*, 1999, **9**, 395.

149 K. Ichinose, T. Taguchi, D. Bedford, Y. Ebizuka and D. A. Hopwood, *J. Bacteriol.*, 2001, **183**, 3247.

150 T. Taguchi, Y. Ebizuka, D. A. Hopwood and K. Ichinose, *J. Am. Chem. Soc.*, 2001, **123**, 11376.

151 C. E. Anson, M. J. Bibb, K. I. Booker-Milburn, C. Clissold, P. J. Haley, D. A. Hopwood, K. Ichinose, W. P. Revill, G. R. Stephenson and C. M. Surti, *Angew. Chem., Int. Ed.*, 2000, **39**, 224.

152 M. A. Fernandez-Moreno, E. Martinez, J. L. Caballero, K. Ichinose, D. A. Hopwood and F. Malpartida, *J. Biol. Chem.*, 1994, **269**, 24854.

153 T. Taguchi, Y. Ebizuka, D. A. Hopwood and K. Ichinose, *Tetrahedron Lett.*, 2000, **41**, 5253.

154 M. J. Bibb, D. H. Sherman, S. Omura and D. A. Hopwood, *Gene*, 1994, **142**, 31.

155 R. McDaniel, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *J. Am. Chem. Soc.*, 1994, **116**, 10855.

156 R. J. X. Zawada and C. Khosla, *J. Biol. Chem.*, 1997, **272**, 16184.

157 B. Shen and C. R. Hutchinson, *Biochemistry*, 1993, **32**, 149.

158 B. Shen and C. R. Hutchinson, *Proc. Natl. Acad. Sci. U. S. A.*, 1996, **93**, 6600.

159 A. Sultana, P. Kallio, A. Jansson, J. S. Wang, J. Niemi, P. Mantsala and G. Schneider, *EMBO J.*, 2004, **23**, 1911.

160 K. Kulowski, E. Wendt-Pienkowski, L. Han, K. Yang, L. C. Vining and C. R. Hutchinson, *J. Am. Chem. Soc.*, 1999, **121**, 1786.

161 A. Hautala, S. Torkkell, K. Räty, T. Kunnari, J. Kantola, P. Mäntsälä, J. Hakala and K. Ylihonko, *J. Antibiot.*, 2003, **56**, 143.

162 S.-E. Wohlert, E. Wendt-Pienkowski, W. Bao and C. R. Hutchinson, *J. Nat. Prod.*, 2001, **64**, 1077.

163 M. Metsä-Ketelä, K. Palmu, T. Kunnari, K. Ylihonko and P. Mäntsälä, *Antimicrob. Agents Chemother.*, 2003, **47**, 1291.

164 L. Westrich, S. Domann, B. Faust, D. Bedford, D. A. Hopwood and A. Bechthold, *FEMS Microbiol. Lett.*, 1999, **170**, 381.

165 H. Decker and S. Haag, *J. Bacteriol.*, 1995, **177**, 6126.

166 L. Han, Y. K., E. Ramalingam, R. H. Mosher and L. C. Vining, *Microbiology*, 1994, **140**, 3379.

167 S. Torkkell, K. Ylihonko, J. Hakala, M. Skurnik and P. Mantsala, *Mol. Gen. Genet.*, 1997, **256**, 203.

168 K. Raty, T. Kunnari, J. Hakala, P. Mantsala and K. Ylihonko, *Mol. Gen. Genet.*, 2000, **264**, 164.

169 S. Torkkell, T. Kunnari, K. Palmu, J. Hakala, P. Mäntsälä and K. Ylihonko, *Antimicrob. Agents Chemother.*, 2000, **44**, 396.

170 P. Kallio, A. Sultana, J. Niemi, P. Mantsala and G. Schneider, *J. Mol. Biol.*, 2006, **17**, 210.

171 H. Brockmann, E. Meyer, K. Schremp, F. Reiners and T. Reschke, *Chem. Ber.*, 1969, **102**, 1224.

172 J. Y. Chung, I. Fujii, S. Harada, U. Sankawa and Y. Ebizuka, *J. Bacteriol.*, 2002, **184**, 6115.

173 K. Ylihonko, J. Tuukkanen, S. Jussila, L. Cong and P. Mantsala, *Mol. Gen. Genet.*, 1996, **251**, 113.

174 Y. H. Chen, C. C. Wang, L. Greenwell, U. Rix, D. Hoffmeister, L. C. Vining, J. Rohr and K. Q. Yang, *J. Biol. Chem.*, 2005, **280**, 22508.

175 A. Mayer, T. Taguchi, A. Linnenbrink, C. Hofmann, A. Luzhetskyy and A. Bechthold, *ChemBioChem*, 2005, **6**, 2312.

176 A. Jansson, J. Niemi, Y. Lindqvist, P. Mantsala and G. Schneider, *J. Mol. Biol.*, 2003, **334**, 269.

177 A. Jansson, H. Koskineni, P. Mantsala, J. Niemi and G. Schneider, *J. Biol. Chem.*, 2004, **279**, 41149.

178 A. Jansson, J. Niemi, P. Mäntsälä and G. Schneider, *Acta Crystallogr.*, Sect. D: *Biol. Crystallogr.*, 2003, **59**, 1637.

179 L. Zhu, B. Ostash, U. Rix, E. A. M. Nur, A. Mayers, A. Luzhetskyy, C. Mendez, J. A. Salas, A. Bechthold, V. Fedorenko and J. Rohr, *J. Org. Chem.*, 2005, **70**, 631.

180 A. Luzhetskyy, L. Zhu, M. Gibson, M. Fedoryshyn, C. Durr, C. Hofmann, D. Hoffmeister, B. Ostash, C. Mattingly, V. Adams, V. Fedorenko, J. Rohr and A. Bechthold, *ChemBioChem*, 2005, **6**, 675.

181 S. Fetzner, *Appl. Microbiol. Biotechnol.*, 2002, **60**, 243.

182 J. L. Caballero, E. Martinez, F. Malpartida and D. A. Hopwood, *Mol. Gen. Genet.*, 1991, **230**, 401.

183 S. G. Kendrew, D. A. Hopwood and E. N. Marsh, *J. Bacteriol.*, 1997, **179**, 4305.

184 G. Sciara, S. G. Kendrew, A. E. Miele, N. G. Marsh, L. Federici, F. Malatesta, G. Schimperna, C. Savino and B. Vallone, *EMBO J.*, 2003, **22**, 205.

185 E. R. Rafanan, Jr., L. Le, L. Zhao, H. Decker and B. Shen, *J. Nat. Prod.*, 2001, **64**, 444.

186 J. Ye, M. L. Dickens, R. Plater, Y. Li, J. Lawrence and W. R. Strohl, *J. Bacteriol.*, 1994, **176**, 6270.

187 F. Lombo, G. Blanco, E. Fernandez, C. Mendez and J. A. Salas, *Gene*, 1996, **172**, 87.

188 A. W. Munro, P. Taylor and M. D. Walkinshaw, *Curr. Opin. Biotechnol.*, 2000, **11**, 369.

189 M. W. Fraaije and A. Mattevi, *Trends Biochem. Sci.*, 2000, **25**, 126.

190 L. Xiang, J. A. Kalaitzis and B. S. Moore, *Proc. Natl. Acad. Sci. U. S. A.*, 2004, **101**, 15609.

191 N. M. Kamerbeek, D. B. Janssen, W. J. H. van Berkel and M. W. Fraaije, *Adv. Synth. Catal.*, 2003, **345**, 667.

192 A. Baeyer and V. Villiger, *Ber. Dtsch. Chem. Ges.*, 1899, **32**, 3625.

193 L. Prado, E. Fernández, U. Weißbach, G. Blanco, L. M. Quirós, A. F. Brana, C. Méndez, J. Rohr and J. A. Salas, *Chem. Biol.*, 1999, **6**, 19.

194 M. Gibson, M. Nur-e-alam, F. Lipata, M. A. Oliveira and J. Rohr, *J. Am. Chem. Soc.*, 2005, **127**, 17594.

195 U. Rix, L. L. Remsing, D. Hoffmeister, A. Bechthold and J. Rohr, *ChemBioChem*, 2003, **4**, 109.

196 T. Liu, C. Fischer, C. Beninga and J. Rohr, *J. Am. Chem. Soc.*, 2004, **126**, 12262.

197 U. Rix, C. Wang, Y. Chen, F. M. Lipata, L. L. Remsing Rix, L. M. Greenwell, L. C. Vining, K. Yang and J. Rohr, *ChemBioChem*, 2005, **6**, 838.

198 R. P. Maskey, I. Grün-Wollny and H. Laatsch, *J. Antibiot.*, 2003, **56**, 795.

199 A. Faworsky, *J. Prakt. Chem.*, 1895, **51**, 533.

200 J. Piel, K. Hoang and B. S. Moore, *J. Am. Chem. Soc.*, 2000, **122**, 5415.

201 R. J. Walczak, J. V. Hines, W. R. Strohl and N. D. Priestley, *Org. Lett.*, 2001, **3**, 2277.

202 R. J. Walczak, M. L. Dickens, N. D. Priestley and W. R. Strohl, *J. Bacteriol.*, 1999, **181**, 298.

203 L. Rodriguez, I. Aguirrezabalaga, N. Allende, A. F. Brana, C. Mendez and J. A. Salas, *Chem. Biol.*, 2002, **9**, 721.

204 A. Trefzer, J. A. Salas and A. Bechthold, *Nat. Prod. Rep.*, 1999, **16**, 283.

205 Y. Hu and S. Walker, *Chem. Biol.*, 2002, **9**, 1287.

206 A. M. Mulichak, W. Lu, H. C. Losey, C. T. Walsh and R. M. Garavito, *Biochemistry*, 2004, **43**, 5170.

207 A. M. Mulichak, H. C. Losey, W. Lu, Z. Wawrzak, C. T. Walsh and R. M. Garavito, *Proc. Natl. Acad. Sci. U. S. A.*, 2003, **100**, 9238.

208 A. M. Mulichak, H. C. Losey, C. T. Walsh and R. M. Garavito, *Structure*, 2001, **9**, 547.

209 S. Morera, A. Imberty, U. Aschke-Sonnenborn, W. Ruger and P. S. Freemont, *J. Mol. Biol.*, 1999, **292**, 717.

210 A. Vrielink, W. Ruger, H. P. Driessen and P. S. Freemont, *EMBO J.*, 1994, **13**, 3413.

211 D. Hoffmeister, K. Ichinose and A. Bechthold, *Chem. Biol.*, 2001, **8**, 557.

212 D. Hoffmeister, B. Wilkinson, G. Foster, P. J. Sidebottom, K. Ichinose and A. Bechthold, *Chem. Biol.*, 2002, **9**, 287.

213 W. Lu, C. Leimkuhler, G. J. Gatto, Jr., R. G. Kruger, M. Oberthur, D. Kahne and C. T. Walsh, *Chem. Biol.*, 2005, **12**, 527.

214 W. Lu, C. Leimkuhler, M. Oberthur, D. Kahne and C. T. Walsh, *Biochemistry*, 2004, **43**, 4548.

215 J. Rohr and A. Zeeck, *J. Antibiot.*, 1987, **40**, 459.

216 B. Faust, D. Hoffmeister, G. Weitnauer, L. Westrich, S. Haag, P. Schneider, H. Decker, E. Kunzel, J. Rohr and A. Bechthold, *Microbiology*, 2000, **146**, 147.

217 D. Hoffmeister, K. Ichinose, S. Domann, B. Faust, A. Trefzer, G. Drager, A. Kirschning, C. Fischer, E. Kunzel, D. Bearden, J. Rohr and A. Bechthold, *Chem. Biol.*, 2000, **7**, 821.

218 A. Trefzer, D. Hoffmeister, E. Kunzel, S. Stockert, G. Weitnauer, L. Westrich, U. Rix, J. Fuchsner, K. U. Bindseil, J. Rohr and A. Bechthold, *Chem. Biol.*, 2000, **7**, 133.

219 T. Henkel and J. Rohr, *J. Antibiot.*, 1990, **43**, 492.

220 L. Westrich, S. Domann, B. Faust, D. Bedford, D. A. Hopwood and A. Bechthold, *FEMS Microbiol. Lett.*, 1999, **170**, 381.

221 U. von Mulert, A. Luzhetskyy, C. Hofmann, A. Mayer and A. Bechthold, *FEMS Microbiol. Lett.*, 2004, **230**, 91.

222 A. Luzhetskyy, T. Taguchi, M. Fedoryshyn, C. Durr, S. E. Wohlrert, V. Novikov and A. Bechthold, *ChemBioChem*, 2005, **6**, 1406.

223 A. Trefzer, C. Fischer, S. Stockert, L. Westrich, E. Kunzel, U. Girreser, J. Rohr and A. Bechthold, *Chem. Biol.*, 2001, **8**, 1239.

224 A. Luzhetskyy, T. Liu, M. Fedoryshyn, B. Ostash, V. Fedorenko, J. Rohr and A. Bechthold, *ChemBioChem*, 2004, **5**, 1567.

225 A. Luzhetskyy, M. Fedoryshyn, C. Durr, T. Taguchi, V. Novikov and A. Bechthold, *Chem. Biol.*, 2005, **12**, 725.

226 E. Künzel, B. Faust, C. Oelkers, U. Weissbach, D. W. Bearden, G. Weitnauer, L. Westrich, A. Bechthold and J. Rohr, *J. Am. Chem. Soc.*, 1999, **121**, 11058.

227 A. Trefzer, G. Blanco, L. Remsing, E. Kunzel, U. Rix, F. Lipata, A. F. Brana, C. Mendez, J. Rohr, A. Bechthold and J. A. Salas, *J. Am. Chem. Soc.*, 2002, **124**, 6056.

228 D. Hoffmeister, M. Weber, G. Dräger, K. Ichinose, C. Dürr and A. Bechthold, *ChemBioChem*, 2004, **5**, 369.

229 D. Hoffmeister, G. Drager, K. Ichinose, J. Rohr and A. Bechthold, *J. Am. Chem. Soc.*, 2003, **125**, 4678.

230 M. Bibb, *Microbiology*, 1996, **142**, 1335.

231 M. J. Bibb, *Curr. Opin. Microbiol.*, 2005, **8**, 208.

232 J. Huang, C. J. Lih, K. H. Pan and S. N. Cohen, *Genes Dev.*, 2001, **15**, 3183.

233 J. Huang, J. Shi, V. Molle, B. Sohlberg, D. Weaver, M. J. Bibb, N. Karoonuthaisiri, C. J. Lih, C. M. Kao, M. J. Buttner and S. N. Cohen, *Mol. Microbiol.*, 2005, **58**, 1276.

234 K. F. Chater, *Annu. Rev. Microbiol.*, 1993, **47**, 685.

235 E. J. Lawlor, H. A. Baylis and K. F. Chater, *Genes Dev.*, 1987, **1**, 1305.

236 F. Wright and M. J. Bibb, *Gene*, 1992, **113**, 55.

237 M. A. Fernandez-Moreno, J. A. Caballero, D. A. Hopwood and F. Malpartida, *Cell*, 1991, **66**, 769.

238 B. K. Leskiw, R. Mah, E. J. Lawlor and K. F. Chater, *J. Bacteriol.*, 1993, **175**, 1995.

239 R. Passantino, A.-M. Puglia and K. F. Chater, *J. Mol. Microbiol.*, 1991, **2059**.

240 B. K. Leskiw, E. J. Lawlor, J. M. Fernandez-Abalos and K. F. Chater, *Proc. Natl. Acad. Sci. U. S. A.*, 1991, **88**, 2461.

241 N. K. Trepanier, S. E. Jensen, D. C. Alexander and B. K. Leskiw, *Microbiology*, 2002, **148**, 643.

242 Y. V. Rebets, B. O. Ostash, M. Fukuhara, T. Nakamura and V. O. Fedorenko, *FEMS Microbiol. Lett.*, 2006, **256**, 30.

243 M. J. Bibb, V. Molle and M. J. Buttner, *J. Bacteriol.*, 2000, **182**, 4606.

244 D. R. Bignell, J. L. Warawa, J. L. Strap, K. F. Chater and B. K. Leskiw, *Microbiology*, 2000, **146**, 2161.

245 J. R. Nodwell, K. McGovern and R. Losick, *Mol. Microbiol.*, 1996, **22**, 881.

246 V. Molle and M. J. Buttner, *Mol. Microbiol.*, 2000, **36**, 1265.

247 A. C. Hunt, L. Servin-Gonzalez, G. H. Kelemen and M. J. Buttner, *J. Bacteriol.*, 2005, **187**, 716.

248 M. K. Pope, B. Green and J. Westpheling, *J. Bacteriol.*, 1998, **180**, 1556.

249 R. Chakraburty and M. Bibb, *J. Bacteriol.*, 1997, **179**, 5854.

250 A. Hesketh, J. Sun and M. Bibb, *Mol. Microbiol.*, 2001, **39**, 136.

251 S. Horinouchi, Y. Ohnishi and D.-K. Kang, *J. Ind. Microbiol. Biotechnol.*, 2001, **27**, 177.

252 H. Onaka, T. Nakagawa and S. Horinouchi, *Mol. Microbiol.*, 1998, **28**, 743.

253 E. Takano, T. Nihira, Y. Hara, J. J. Jones, C. J. Gershater, Y. Yamada and M. Bibb, *J. Biol. Chem.*, 2000, **275**, 11010.

254 L. Wang and L. C. Vining, *Microbiology*, 2003, **149**, 1991.

255 A. Wietzorrek and M. Bibb, *Mol. Microbiol.*, 1997, **25**, 1181.

256 E. Martinez-Hackert and A. M. Stock, *Structure*, 1997, **5**, 109.

257 E. Martinez-Hackert and A. M. Stock, *J. Mol. Biol.*, 1997, **269**, 301.

258 P. Arias, M. A. Fernandez-Moreno and F. Malpartida, *J. Bacteriol.*, 1999, **181**, 6958.

259 V. Colombo, M. Fernandez-de-Heredia and F. Malpartida, *Microbiology*, 2001, **147**, 3083.

260 G. C. Uguru, K. E. Stephens, J. A. Stead, J. E. Towle, S. Baumberg and K. J. McDowall, *Mol. Microbiol.*, 2005, **58**, 131.

261 S. L. Otten, J. Ferguson and C. R. Hutchinson, *J. Bacteriol.*, 1995, **177**, 1216.

262 K. J. Stutzman-Engwall, S. L. Otten and C. R. Hutchinson, *J. Bacteriol.*, 1992, **174**, 144.

263 L. Tang, A. Grimm, Y.-X. Zhang and C. R. Hutchinson, *Mol. Microbiol.*, 1995, **22**, 801.

264 P. J. Sheldon, S. B. Busarow and C. R. Hutchinson, *Mol. Microbiol.*, 2002, **44**, 449.

265 K. Furuya and C. R. Hutchinson, *J. Bacteriol.*, 1996, **178**, 6310.

266 S. L. Otten, C. Olano and C. R. Hutchinson, *Microbiology*, 2000, **146**, 1457.

267 K. Yang, L. Han, J. He, L. Wang and L. C. Vining, *Gene*, 2001, **279**, 165.

268 Y. Rebets, B. Ostash, A. Luzhetskyy, D. Hoffmeister, A. Brana, C. Mendez, J. A. Salas, A. Bechthold and V. Fedorenko, *FEMS Microbiol. Lett.*, 2003, **222**, 149.

269 O. Gromyko, Y. Rebets, B. Ostash, A. Luzhetskyy, M. Fukuhara, A. Bechthold, T. Nakamura and V. Fedorenko, *J. Antibiot.*, 2004, **57**, 383.

270 B. Aigle, X. Pang, B. Decaris and P. Leblond, *J. Bacteriol.*, 2005, **187**, 2491.

271 K. F. Chater and C. J. Bruton, *EMBO J.*, 1985, **4**, 1893.

272 K. F. Chater and M. Bibb, *Biotechnology. Vol. 7: Products of Secondary Metabolism*, VCH, Weinheim, 1997.

273 P. Guilfoile and C. R. Hutchinson, *J. Bacteriol.*, 1992, **174**, 3659.

274 N. L. Sheeler, S. V. MacMillan and J. R. Nodwell, *J. Bacteriol.*, 2005, **187**, 687.

275 A. R. Hesketh, G. Chandra, A. D. Shaw, J. J. Rowland, D. B. Kell, M. J. Bibb and K. F. Chater, *Mol. Microbiol.*, 2002, **46**, 917.

276 A. R. Hesketh and K. F. Chater, *J. Ind. Microbiol. Biotechnol.*, 2003, **30**, 523.

277 L. Dutko, Y. Rebets, B. Ostash, A. Luzhetskyy, A. Bechthold, T. Nakamura and V. Fedorenko, *FEMS Microbiol. Lett.*, 2004, **255**, 280.
