
# Light-regulated transcriptional networks in higher plants

**Yuling Jiao, On Sun Lau and Xing Wang Deng**

Abstract | Plants have evolved complex and sophisticated transcriptional networks that mediate developmental changes in response to light. These light-regulated processes include seedling photomorphogenesis, seed germination and the shade-avoidance and photoperiod responses. Understanding the components and hierarchical structure of the transcriptional networks that are activated during these processes has long been of great interest to plant scientists. Traditional genetic and molecular approaches have proved powerful in identifying key regulatory factors and their positions within these networks. Recent genomic studies have further revealed that light induces massive reprogramming of the plant transcriptome, and that the early light-responsive genes are enriched in transcription factors. These combined approaches provide new insights into light-regulated transcriptional networks.

---

### Phototropism
Directional plant growth that is determined by the direction of the light source.

### Gravitropism
A growth movement in response to gravity.

Light is one of the most important environmental factors for plants, as it provides the source of energy for plant life. It is therefore not surprising that plants have adopted the ability to sense multiple parameters of ambient light signals, including light quantity (fluence), quality (wavelength), direction and duration. Light signals are perceived through at least four distinct families of photoreceptors, which include phytochromes, cryptochromes, phototropins and unidentified ultraviolet B (UVB) photoreceptor(s) (BOX 1). Plant responses to light occur in the context of multiple developmental processes, including seed germination, seedling photomorphogenesis, phototropism, gravitropism, chloroplast movement, shade avoidance, circadian rhythms and flower induction (BOX 2).

Transcriptional regulatory networks have a key role in mediating light signalling through the coordinated activation and repression of specific downstream genes. Therefore, there is considerable interest in elucidating the hierarchy of networks that are formed by transcription factors, and in identifying the key regulatory elements in different light-responsive developmental processes. For each developmental response, more than one photoreceptor can contribute to the perception of light signals, indicating that signal integration points for different light signals must exist in transcriptional hierarchies. Added to this complexity are organ-specific and developmental stage-specific responses to light, which represent a multitude of variations among light-responsive transcriptional networks. Furthermore, light and other environmental stimuli often work together to trigger specific developmental responses, indicating the existence of integration points between these different signalling networks.

Extensive progress has been made in recent years towards characterizing the organization of light-regulated transcriptional networks in the model plant *Arabidopsis thaliana*; in particular, in characterizing the networks that regulate seedling photomorphogenesis. Classical genetic and molecular approaches have identified various regulators downstream of photoreceptors¹. Many of these encode transcription factors, as well as kinases, phosphatases and degradation-pathway proteins. Although some of these regulators are specific for light quality, others regulate signal transduction networks in response to various light signals, representing potential signal integration points. Furthermore, recent genomic analysis has started to address how light influences transcription at the genomic scale. Light induces massive reprogramming of the plant transcriptome. For example, we now know that a significant portion of the genome, at least 20% in both *A. thaliana* and rice, shows differential expression between seedlings that are under photomorphogenesis and those that are under skotomorphogenesis²⁻⁴. Light effects are so profound that most of the major biochemical pathways that are located within the main subcellular organelles are coordinately regulated by light³,⁴.

Here we review the current understanding of light-regulated transcriptional networks, derived mainly from research in *A. thaliana*. We first provide a brief overview

Department of Molecular, Cellular and Developmental Biology, 165 Prospect Street, Yale University, New Haven, Connecticut 06520-8104, USA.
Correspondence to X.W.D.
e-mail: xingwang.deng@yale.edu
doi:10.1038/nrg2049

REVIEWS

Box 1 | Light signals and photoreceptors

| Cryptochrome |
| --- |
| Pterin | FAD |
| CNT/PHR | CCT/DAS |

| Phytochrome |
| --- |
| PΦB |
| PHY | PAS | PAS | HKRD |

UVA Blue Red Far red  
300 400 500 600 700 800  

FMN FMN  
LOV LOV Serine/threonine kinase  

Phototropin  

To monitor the light environment, plants have evolved a series of photoreceptors. Cryptochromes and phototropins perceive blue and ultraviolet A (UVA) wavelengths. Phytochromes predominantly absorb the far-red and red wavelengths, and an unidentified photoreceptor, or photoreceptors, absorbs UVB.

In higher plants, phytochromes form a small family, which further evolved independently in dicots<sup>138</sup>. There are five phytochromes (PHYA to PHYE) in *Arabidopsis thaliana*. PHYA is a type I phytochrome, which is most abundant in the dark and degrades rapidly after light exposure. All other phytochromes are relatively stable in the light and are classified as type II (REF. 139). The phytochromes are dimeric chromoproteins. Each polypeptide consists of an N-terminal photosensory domain that covalently binds a single bilin chromophore (PΦB), followed by a C-terminal domain that contains several motifs and functions in dimerization, light-dependent nuclear localization and, possibly, regulation of signalling<sup>140</sup>.

There are two well-characterized cryptochromes in *A. thaliana*<sup>5</sup>, CRY1 and CRY2, and a more divergent CRY3 (REFS 7, 141). CRY1 and CRY2 have an N-terminal photolyase-related (PHR) domain (CNT) and a less-conserved, intrinsically unstructured C-terminal DAS domain (CCT), which is not present in CRY3 (REFS 141, 142). The PHR domain non-covalently binds to two chromophores, a flavin adenine dinucleotide (FAD), and a pterin. CCT mediates a constitutive light response through direct interaction with CONSTITUTIVE PHOTOMORPHOGENIC 1 (COP1) (REFS 143–145).

Phototropins are plant-specific blue light receptors, which have a photosensory N-terminal half and a C-terminal half with serine/threonine kinase function<sup>146</sup>. The N terminus contains two flavin mononucleotide (FMN) chromophore-binding LOV domains (LOV1 and LOV2).

of the basic principles of light-responsive signal transduction. We then focus our discussion on four important light-regulated developmental processes — seedling photomorphogenesis, seed germination, shade avoidance and the photoperiod response — and highlight emerging insights regarding the integration of information that is provided by different light qualities, the generation of organ-specific responses and the interaction of light-responsive transcriptional networks with inputs from other environmental stimuli.

### Photoreceptor regulation and activity

As photosensory switches, photoreceptors are tightly controlled by light in multiple ways. Photoreceptor genes are largely ubiquitously expressed, and the regulation of their functions is mainly at the post-translational level.

Light can modulate photoreceptor activity by inducing changes that alter their cellular localization. Phytochromes are synthesized in the inactive Pr form, and are activated on light absorption by conversion to the biologically active Pfr form<sup>5</sup>. The photoconversion of phytochromes results in their translocation from the cytoplasm into the nucleus, which is crucial for allowing them to interact with transducers in initiating downstream transcriptional cascades<sup>5</sup>. In terms of the cryptochromes, *A. thaliana* **CRY2** is constitutively nuclear-localized whereas **CRY1** is nuclear in the dark but largely cytoplasmic under light<sup>6</sup>. Nuclear-localized cryptochromes closely interact with the chromatin<sup>6</sup>. On the other hand, CRY3 has a dual targeting signal that mediates its transport to chloroplasts and mitochondria, suggesting a potential role in regulating transcription in organelles<sup>7</sup>. Both of the *A. thaliana* phototropins, **PHOT1** and **PHOT2**, are largely associated with the plasma membrane, although following activation by light, a fraction of PHOT1 is released to the cytoplasm<sup>1</sup>. The contribution of phototropins on transcriptional regulation, however, is relatively small, and only a limited number of genes are under their control.

Photoreceptors are also subject to phosphorylation control. For example, the phosphorylation of **PHYA** modulates photoresponses in several ways: through controlling the subcellular localization of PHYA, its stability and its affinity towards downstream signal transducers<sup>8,9</sup>. In several cases, signal transduction in response to light is thought to involve kinase activity of the photoreceptors themselves — for example, in the case of the phytochromes<sup>10</sup>. It has been proposed that phytochromes have intrinsic kinase activity, with the Pfr form being more active<sup>11</sup>. Controversially, however, *in vivo* studies have indicated that the proposed C-terminal

---

Chromophore  
The part of a molecule that absorbs specific wavelengths of light and is responsible for its colour.

Box 2 | Light-regulated plant development

Germination  
Photomorphogenesis  
Plant growth and response to environment  
Shade avoidance  

- Photoperiod and flowering  
- Phototropism  
- Chloroplast movement  
- Stomatal opening  

Phytochromes  
Cryptochromes  
Phototropins  

Light controls growth and development throughout the plant life cycle. In unfavourable environmental conditions, an intact and healthy seed remains dormant in a dry state<sup>147</sup>. In *Arabidopsis thaliana*, seed dormancy is terminated by environmental signals such as light, temperature, nutrient availability and duration of storage in the dried state<sup>147</sup>. Low-fluence red light induces germination, which can be inhibited by subsequent far-red light treatment<sup>5</sup>. The phytochrome PHYB is the photoreceptor largely responsible for red:far-red reversible control of seed germination with the help of the phytochromes PHYA and PHYE.

After germination, seedlings follow one of two developmental patterns. Skotomorphogenesis (or etiolation) in the dark is characterized by long hypocotyls, closed cotyledons protected by apical hooks in *A. thaliana*, and the development of proplastids into etioplasts. By contrast, growth in the light results in photomorphogenesis (or de-etiolation) characterized by short hypocotyls, expanded open cotyledons and the development of mature green chloroplasts that can photosynthesize. A wide spectrum of light, in particular far-red, red, blue and ultraviolet (UV) light conditions, induces photomorphogenesis. PHYA is the primary photoreceptor under far-red light in *A. thaliana*, whereas PHYB has a major role under white or red light with the aid of PHYA, PHYC and PHYD. Rice PHYA and PHYB equally contribute to seedling photomorphogenesis under red light and both rice PHYA and PHYC are involved in far-red light responses<sup>148</sup>. Both CRY1 and CRY2 cryptochromes are responsible for photomorphogenesis under blue and UVA light.

When plants grow in close proximity there is competition for light. Higher plants have evolved an impressive capacity to avoid shade. A plant canopy is associated with a reduction in the ratio of red:far-red light. Changes in the red:far-red ratio are detected as a change in the relative proportions of Pr and Pfr forms of phytochromes and PHYB has the most significant role<sup>5</sup>.

The perception of photoperiod (or day length) is crucial for plants to adjust their development to fit into annual seasonal changes. The interaction of light signals with intrinsic circadian rhythms measures changes in day length. In *A. thaliana*, both phytochromes and cryptochromes contribute to synchronizing the circadian clock. The perception of day length is an important signal in the control of flowering.

Several other transient developmental processes, including phototropism, chloroplast movement and stomatal opening, are under light control mainly through phototropins<sup>146</sup>. These rapid light-responsive processes are not under extensive transcriptional regulation, and are therefore beyond the scope of this Review.

---

Proplastid  
Precursors of plastids, which are plant organelles that include chloroplasts.

Etioplast  
An immature chloroplast that has not been exposed to light.

kinase-related domain of **PHYB** is dispensable for signalling<sup>12</sup>. In *A. thaliana*, the autophosphorylation of CRY1 and CRY2 is also important for their functions<sup>6</sup>. It has been suggested that light activation of the N terminus of CRY1 (CNT) induces a conformational change in its C terminus (CCT) (BOX 1), allowing its autophosphorylation and dimerization, and possible interactions with downstream partner proteins<sup>13</sup>. Phototropins also have well-established kinase activity. The blue-light-triggered autophosphorylation of these receptors initiates the transduction of the light signal<sup>14,15</sup>, involving several downstream signalling pathways<sup>1</sup>.

REVIEW S

The activation of photoreceptors, mainly phytochromes and cryptochromes, can significantly affect transcription through signal transduction pathways and, in a few cases, by direct effects on transcription factors.

### Regulation of transcription factors by light

Light-responsive transcription factors have been identified through screens for light-responsive *cis*-element (LRE)-binding proteins and through genetic analyses of mutants that are deficient in their response to specific types of light. Some of these transcription factors are regulated by just one type of light, whereas many more respond to a wide spectrum of light. Transcriptional regulation, post-translational modification and degradation of these transcription factors are all important in the light-regulated control of development.

Both positive and negative transcriptional regulation of transcription factors by light has been documented. For example, the transcription of COMMON PLANT REGULATORY FACTORS 1 (CPRF1) from parsley is rapidly induced by light¹⁶ (FIG. 1a). CPRF1 has the ability to bind to G-box, a well-defined LRE¹⁶. However, CPRF1 levels increase only transiently after light treatment, and transcription might be blocked by the binding of CPRF1 to its own promoter¹⁷. Such a ‘gas-and-brake’ mechanism is widely seen in light-regulated networks, as discussed later. It is worth noting that downstream targets of CPRF1 and other CPRFs are still unknown owing to the arduous nature of carrying out genetic studies in parsley.

Several basic post-translational mechanisms are involved in regulating transcription factor activities in response to light. The phosphorylation of transcription factors is a common modification that can influence their ability to bind to promoters (FIG. 1b). For example, the level of G-BOX BINDING FACTOR 1 (GBF1) is constant but its affinity for the G-box is modulated by its phosphorylation status: its phosphorylation by nuclear CASEIN KINASE II (CKII) enables G-box binding¹⁸,¹⁹. Light might also regulate the subcellular localization of transcription factors through phosphorylation²⁰ (FIG. 1c). For example, CPRF2 from parsley is localized in the cytosol in the dark and treatment with light causes an import in the nucleus²¹; light-dependent *in vivo* phosphorylation of CPRF2 is probably the key event that triggers its nuclear import²².

Finally, recent advances have demonstrated the importance of ubiquitin-mediated proteolysis in light signalling²³. Suppression of photomorphogenesis in dark-grown seedlings requires the repressor CONSTITUTIVE PHOTOMORPHOGENIC 1 (COP1), a RING-finger type ubiquitin E3 ligase²⁴. In the dark, some transcription factors that positively regulate gene expression in response to light, such as LONG AFTER FAR-RED LIGHT 1 (LAF1), are ubiquitylated by COP1 under far-red light for subsequent degradation by the 26S proteasome, with the help of the PHYTOCHROME A SUPPRESSOR 1 (SPA1) protein²⁵ (FIG. 1d). Light inhibits its E3 ligase activity in part by excluding COP1 from the nucleus²⁶. In many cases, light affects multiple steps in the regulation of a transcription factor, thereby achieving extensive flexibility and precision. For example, the multi-facet regulation of two transcription factors, ELONGATED HYPOCOTYL 5 (HY5), a positive regulator, and PHYTOCHROME INTERACTING FACTOR 3 (PIF3), a negative regulator for seedling photomorphogenesis, has been well defined and is illustrated in FIG. 1e,f.

Adding a further level of complexity to the regulation of transcription in response to light, certain light-regulated transcription factor families tend to form dimers, such as bHLH (basic helix–loop–helix) and bZIP (basic leucine zipper) families²⁷⁻²⁹. Various degrees of homodimerization and heterodimerization suggest that these transcription factor families have the potential to participate in multiple sets of combinatorial interactions, endowing them with the capacity to function in the regulation of several transcriptional programmes³⁰,³¹. It is reasonable to suspect that such interactions are involved in integrating signals from different light signalling branches and from light and other factors, such as temperature. In addition, interactions between different families further extend the complexity of regulation³²,³³.

#### LREs in transcriptional regulation

LREs, which commonly occur in light-regulated promoters, are essential for light-controlled transcriptional activity³⁴,³⁵. A combination of various methods has been used to identify these LREs. Traditional deletion and mutagenesis analysis of promoters of known light-responsive genes has been used to pinpoint LREs, and footprinting and gel-retardation assays have been used to screen for binding motifs of known light-responsive transcription factors³⁴. Recently developed computational approaches use genome-scale expression data from microarray studies to look at enriched sequence elements among promoters of co-expressed or differentially expressed genes³⁶.

A range of LREs have been documented in different promoters, many of which positively or negatively mediate gene expression in response to light. Although many LREs and their binding proteins have been identified, no single element is found in all light-regulated promoters, suggesting a complex light-regulation network and a lack of a universal switch. It has been suggested that combinations of LREs, rather than individual elements, could confer proper light-responsiveness to a light-insensitive basal promoter. Most information relating to LREs has been derived from studies on photomorphogenesis, which are discussed in detail in the next section.

#### Seedling photomorphogenesis networks

The process of seedling photomorphogenesis, during which plants undergo profound developmental changes, is one of the most extensively studied light-regulated responses (BOX 2). Extensive forward genetic studies have identified many key factors in the signalling network. In addition, large-scale gene-expression analyses have expanded our understanding of photomorphogenesis to the genomic scale. A picture is emerging in which signals from different photoreceptors are highly integrated, but at the same time light-regulated networks are able

---

**Ubiquitin E3 ligase**

Enzymes that covalently attach ubiquitin to a lysine residue on a target protein.

**26S proteasome**

A large, ATP-dependent, multicatalytic protease, which degrades ubiquitylated proteins to short peptides.

Figure 1 | General mechanisms of transcription factor regulation by light.

a | Transcription of the G-box-binding transcription factor COMMON PLANT REGULATORY FACTORS 1 (CPRF1) is induced by light. CPRF1 represses its own transcription, resulting in the tight control of its expression.

b | Unphosphorylated G-BOX BINDING FACTOR 1 (GBF1) lacks affinity for its target genes. Phosphorylation by CASEIN KINASE II (CKII) allows GBF1 to bind to promoters that contain G-boxes.

c | Red light induces phosphorylation and nuclear translocation of the G-box-binding transcription factor CPRF2.

d | LONG AFTER FAR-RED LIGHT 1 (LAF1) is a positive regulator of gene expression downstream of far-red light, and is ubiquitylated by COP1–SPA1 (CONSTITUTIVE PHOTOMORPHOGENIC 1–PHYTOCHROME A SUPPRESSOR 1) in the dark, which leads to its degradation²⁵. Light negatively regulates the nuclear level of COP1 and thereby allows the accumulation of LAF1 (REF. 26).

e | Some transcription factors are regulated by light in many ways. The level of ELONGATED HYPOCOTYL 5 (HY5) is regulated at the transcriptional level by light⁷⁰. In addition, the nuclear level of COP1 regulates HY5 by targeted ubiquitin-mediated proteolysis²³,¹⁴⁹. HY5 is also preferentially phosphorylated within its COP1 binding domain in the dark⁷². Unphosphorylated HY5 interacts more strongly with COP1, is the preferred substrate for degradation and has higher binding affinity for target promoters⁷².

f | PHYTOCHROME INTERACTING FACTOR 3 (PIF3) induces the expression of skotomorphogenesis genes, thereby acting as a negative regulator of photomorphogenesis. Red light induces the nuclear import of the Pfr form of phytochromes, allowing them to interact with PIF3. Subsequent rapid phosphorylation of PIF3 renders it unstable under light¹⁰, as phosphorylated PIF3 is ubiquitylated and degraded⁶⁴.

to direct organ-specific responses. Transcription factors seem to be key players in these networks and the molecular mechanisms of their light-mediated control are starting to be revealed.

**Light-responsive transcription factors.** An important finding from microarray analysis is the enrichment of transcription factors in light-responsive genes during photomorphogenesis, especially shortly after light exposure when light-stimulated photomorphogenesis is barely observable²³⁷,³⁸. For example, among functionally classifiable early light-responsive genes within 1 hour of far-red or red light exposure, 44% (for far-red light) and 25% (for red light) encode transcription factors. Given that *A. thaliana* uses 5–6% of its genome to encode transcription factors³⁹, this enrichment is significant. A similar time-course study identified 64 early responsive transcription factors that respond to blue light³⁷. The rapid responsiveness of these transcription factors indicates that they might be integral components of a primary light-regulated transcriptional network. Besides gene activation, microarray data suggest the existence of at least one other pathway, which is initiated by the early repression of transcriptional cascades, as revealed by the fact that the expression of a large number of transcription factors is rapidly repressed by light²³⁷,³⁸.

Interestingly, PHYA dominates in regulating the transcription of early responding genes under both far-red light and red light²,⁴⁰, although PHYB instead of PHYA is described as the main red light receptor in the literature¹. One explanation to this is that, although PHYB dominates the long-term red light suppression of hypocotyl cell elongation, PHYA and other phytochromes have a significant role in the apical-zone

responses of hook opening, cotyledon expansion and chloroplast biogenesis³⁸. Considering that the less vacuolated apical hook and cotyledon cells are probably the predominant mRNA source rather than the hypocotyl cells, which are large in size but vacuolated, the observed expression patterns probably reflect the light-induced responses of the apical hook and cotyledons, rather than the hypocotyls⁴⁰.

**Light-quality-specific signalling.** Several transcription factors, which include both positive and negative regulators, have been genetically identified as acting downstream of specific photoreceptors or sets of photoreceptors in photomorphogenesis. Although some transcription factors predominantly respond to one type of light, others respond to two or more.

Genetic and genomic analyses suggest the existence of several signalling pathways downstream of PHYA in photomorphogenesis⁴¹,⁴² (FIG. 2). FAR-RED IMPAIRED RESPONSE 1 (*FAR1*) and FAR-RED ELONGATED HYPOCOTYL 3 (*FHY3*) are both novel transposon-derived putative transcription factors, which interact with each other and are specific to far-red light⁴³⁻⁴⁵, whereas LAF1 is a transcription factor that is homologous to the R2R3-MYB family of DNA-binding proteins⁴⁶. Loss-of-function mutants of *far1*, *fhy3* or *laf1* show developmental defects in PHYA-mediated seedling photomorphogenesis in response to far-red light, whereas they show no obvious phenotype under other light qualities⁴³⁻⁴⁶. A similar light-hyposensitive phenotype was found, under both far-red and blue light conditions⁴⁷,⁴⁸, in mutants of the LONG HYPOCOTYL IN FAR-RED (*HFR1*) gene (which encodes a putative bHLH transcription factor⁴⁷,⁴⁹,⁵⁰), implying its role in both PHYA and cryptochrome signalling. FHY3 and FAR1

![Diagram](attachment:diagram.png)

Figure 2 | **Transcriptional networks for seedling photomorphogenesis.** A simplified overview of the network involved in this process is shown. Key regulators of this light-regulated transcriptional network have been identified in *Arabidopsis thaliana*, and suggest the existence of separate intermediate networks that are dedicated to each photoreceptor group. A group of PIF transcription factors interact directly with phytochromes and function mainly as repressors of photomorphogenesis. Key transcription factors, such as HY5, serve as signal integration points of major branches downstream of all photoreceptors. The COP/DET/FUS class of factors act as light-inactivatable repressors of photomorphogenesis. Bold lines indicate the convergence pathway.
most likely act more upstream in the network, close to PHYA⁴¹, whereas HFR1 probably acts further downstream as microarray studies found that HFR1 controls the expression of a smaller subset of genes⁵¹,⁵². HY5, a bZIP transcription factor, might represent another branch under FHY3/FAR1, although its function is not limited to far-red light⁴² (FIG. 2).

Two Dof family transcription factors, COGWHEEL 1 (COG1; also known as RPP1) and OBF4 BINDING PROTEIN 3 (OBP3), are involved in red light signalling. COG1 is a negative regulator under both red and far-red light⁵³, whereas OBP3 acts as a positive regulator for the inhibition of hypocotyl elongation and a negative regulator for cotyledon expansion in both PHYB and CRY1 signalling pathways⁵⁴. Another two G-box and Z-box-binding transcription factors are also involved in light signalling. In *A. thaliana* MYC2, a bHLH transcription factor, functions as a repressor of blue and far-red light-mediated seedling de-etiolation⁵⁵, whereas GBF1, a bZIP transcription factor, functions as a repressor for blue-light-mediated inhibition of hypocotyl elongation but a positive regulator for cotyledon expansion⁵⁶.

A breakthrough concept of phytochrome regulation of gene expression has come from the identification of a subfamily of phytochrome-interacting bHLH transcription factors, which have been designated PIFs (phytochrome interacting factors). These PIFs for the first time define direct links between photoreceptors and transcriptional regulation, and selectively interact with the Pfr form of phytochromes⁵⁷. PIF3, PIF4, PIF5 (also known as PIL6) and PIF6 (also known as PIL2) interact mainly with PHYB, whereas PIF1 (also known as PIL5) can bind to both PHYA and PHYB⁵⁷–⁶². Loss-of-function mutants of *pif3*, *pif4* and *pif5* display shorter hypocotyls under red light⁶⁰,⁶³–⁶⁶, similar to the *pif1* mutant under far-red light⁶¹,⁶², implying that PIFs are mainly negative regulators of phytochrome signalling. However, although highly similar in sequence, the roles of these PIFs are not completely overlapping. For example, PIF3 differentially affects distinct branches of PHYB signalling as a positive regulator of early chloroplast development and of the expression of nuclear-encoded photosynthetic genes⁶⁶, whereas PIF1 inhibits the accumulation of protochlorophyllide, the immediate precursor of chlorophyll. The excess of such free protochlorophyllide accumulation can lead to lethal seedling bleaching on exposure to light⁶¹. Therefore the stability of PIF proteins is kept under tight light control. In fact, one of the main functions of phytochromes is to downregulate PIF proteins, at least initially, through light-induced degradation¹⁰,⁶⁷ (FIG. 1f). Interestingly, PIF proteins are not completely removed by light-induced degradation. PIF1 and PIF3 proteins decline rapidly to a basal steady-state level after initial light exposure, but re-accumulate in the subsequent dark period, signifying that they could also function during diurnal cycles⁶⁶,⁶⁷.

(UVR8) is a UVB-specific signalling component that orchestrates the expression of a range of genes with vital UVB-protective functions. UVR8 regulates the expression of the transcription factor HY5 when the plant is exposed to UVB⁶⁸. This protein is located principally in the nucleus and associates with chromatin through histones in the HY5 promoter region, providing a mechanistic basis for its involvement in regulating transcription⁶⁸. In addition, COP1, a negative regulator of the visible light response, is a crucial positive regulator of responses to low levels of UVB, coordinating both HY5-dependent and HY5-independent pathways and eventually resulting in UVB tolerance⁶⁹.

**Integration of light-quality-specific signals.** Distinct light qualities, which are mediated through different photoreceptors (BOX 1), have similar effects on the transcriptomes during *A. thaliana* and rice seedling photomorphogenesis, consistent with phenotypic observations³,⁴. It is therefore reasonable to suggest that one or more integration points exist for light signals. Accordingly, a few light-induced transcription factors are identified as key regulators during seedling photomorphogenesis. The *hy5* mutant, which is deficient for a bZIP transcription factor, shows a partially etiolated phenotype in a wide spectrum of light⁷⁰,⁷¹, suggesting that HY5 acts downstream of all photoreceptors. HY5 levels are rapidly regulated both transcriptionally and post-translationally²³,⁷⁰,⁷² (FIG. 1e). *In vitro* analysis showed that HY5 directly binds to the promoters of several light-inducible genes⁷³,⁷⁴, and a recent chromatin-immunoprecipitation analysis in combination with a whole-genome tiling microarray revealed that HY5 binds directly to a large number of genomic sites, mainly at the promoter regions of annotated genes⁷⁵. It seems that HY5 directly mediates both upregulation and downregulation of gene expression by light. The transcriptional regulation attributable to HY5 is included largely within genes that are regulated by light and comprises ~20% of all light-regulated genes⁷⁶. Therefore, HY5 is likely to be a regulator that is positioned high up in the hierarchy of transcription factors at a branch of the transcriptional cascade that is involved in seedling photomorphogenesis (FIG. 2).

Similarly, CIRCADIAN CLOCK ASSOCIATED 1 (CCA1) and LATE ELONGATED HYPOCOTYL (LHY), both encoding partially redundant MYB transcription factors⁷⁷,⁷⁸, are among the early induced genes that are under phytochrome and cryptochrome control during seedling photomorphogenesis²,³⁷,³⁸. CCA1 directly interacts with the promoters of target genes to transduce light signals⁷⁷. Importantly, both CCA1 and LHY are components of the plant circadian clock⁷⁸,⁷⁹, and therefore seem to provide a link between light signals and this endogenous mechanism in the control of gene expression.

**Organ-specific responses.** Each organ type exhibits distinct developmental responses to light (BOX 2), although they seem to share common photoreceptors¹. Not surprisingly, light-regulated transcriptomes in different

Chromatin immunoprecipitation  
A technique that is used to determine whether a particular protein, for example, a transcription factor, can bind to a specific region on a chromatin *in vivo*.  

UVB light (with a wavelength of 280–320 nm) is another component of sunlight, and is sensed by plants as both an informational signal and an environmental stress factor. *Arabidopsis thaliana* UVB RESISTANCE 8  
REVIEW S

a
Far-red
light

b
Light

c
Light

Shoot-specific
response

Far-red
light

GATA
Basal

Induction-specific
flanking sequence

Shoot-specific
flanking sequence

Far-red
light

G
GATA
Basal

Repression-specific
flanking sequence

Root-specific
flanking sequence

Figure 3 | Light-responsive promoters. a | Promoters that contain a basal element and either a G-box or a GATA-box only cannot effectively respond to far-red light. Promoters with paired light-responsive cis-elements (LREs; those that contain both a G-box and a GATA-box) are able to respond to a wide spectrum of light signals, including far-red light. b | Induction or repression of PHYA (phytochrome A)-responsive genes is specified according to the conserved flanking sequences around the G-box core. The combination of the G-box core, the flanking sequence and another LRE defines transcriptional activities. c | Organ-specific light responses are defined by conserved flanking sequences around the G-box core.

organs have limited overlaps^{4,80}. The transcriptomes of A. thaliana cotyledons, hypocotyls and roots are all regulated by light to a similar degree. However the overlaps among light-regulated genes are small and only less than 1% of all light-regulated genes (~12) were found to be induced or repressed by light in all three tissues^{4,80}. Such organ-specific transcription profiles suggest a spatial involvement of different signalling cascades in different organs and cell types^{80}, with distinctions at several levels. For instance, although early red-light-responsive genes are mainly under PHYA control^{40}, suppression of hypocotyl cell elongation under red light is largely mediated by PHYB, whereas red-light-imposed cotyledon cell expansion and hook unbinding are mediated by other phytochromes^{38}. Furthermore, transcription factors in several branches at different positions in the hierarchy, such as PIF3, OBP3 and GBF1, show distinct organ-specific effects during photomorphogenesis. It is likely that distinct organs recruit different transcriptional networks, which share some transcription factors in common.

Targeted mutagenesis studies of 32 representative phytochrome-induced genes during seedling photomorphogenesis suggest that 63% (20) of them have light-dependent morphological phenotypes, specifically in inhibition of hypocotyls or stimulation of cotyledon growth^{81}. A small proportion, 22% (7), have both diagnostic phenotypes. The divergent functions of these immediate downstream genes could imply the immediate divergence of phytochrome signalling. For the other 12 genes with mutant lines that still lack identifiable photomorphogenic phenotypes, functional redundancy, functional specificity (such as wavelength and developmental specificity) and a possible transient role during de-etiolation are possible explanations^{81}.

Combinatorial LRE functions. Distinct combinations of transcription factors that bind at LREs provide one key regulation point in transcriptional cascades that are induced in response to different light signals or within different organs during photomorphogenesis (FIG. 3). Whereas individual LRE-containing promoters primarily respond to a specific wavelength of light, no reported LRE has activity in an organ-specific manner, suggesting that combinatorial functions of distinct elements are important for light-regulated promoter activities^{32,82–84}. With synthetic promoter-reporter constructs containing single or paired LREs, it has been demonstrated that promoters containing paired elements can respond to a wide range of strengths in light signal and a broad spectrum of light^{83,84}.

Whereas the well-defined, widely conserved core LREs are essential for light responses, the conserved flanking consensus sequences can specify the mode of regulation (repression versus activation) or organ specificity. From an analysis of PHYA-regulated genes in A. thaliana, two distinct sequences flanking a G-box core were identified, one predominating in PHYA-inducible promoters and the other in PHYA-repressed promoters^{36}. Similarly, different flanking sequences of the G-box core and other LREs were recruited to convey organ specificity in light regulation^{4}. These specificity-defining flanking sequences around core LREs might indicate that different members of the same family of DNA-binding proteins can mediate induction and repression with distinct spatial patterns^{36}. Further mutagenesis studies will be needed to confirm this suggestion.

Chromatin remodelling. Chromatin modifications are important for light-regulated gene expression. So far, none of the identified light-regulated chromatin

REVIEW

as loss-of-function mutants for $DET1$ develop in darkness in the same way as light-grown seedlings⁹⁹. $DET1$ encodes a nuclear-localized protein that does not have any detectable DNA-binding activity⁹⁰. Instead, $DET1$ is part of a protein complex (CDD) with DAMAGED DNA-BINDING PROTEIN 1 (DDB1), a protein implicated in chromatin modification, possibly through the recruitment of histone acetyltransferases⁹¹, and COP10, a ubiquitin-conjugating enzyme variant⁹². The CDD complex might bind to histones directly in modifying chromatin structure, as tomato $DET1$ interacts with the non-acetylated tail of histone H2B (REF. 93). In addition, the *Drosophila melanogaster* $DET1$ homologue associates with chromatin⁹⁴. The binding between $DET1$ and chromatin can be specific, occurring through promoter elements and proteins such as the transcription factors CCA1 and HY5 to induce or repress gene transcription⁹⁵. $DET1$ might in part regulate chromatin conformation, thereby affecting the expression of many genes involved in photomorphogenesis.

### Transcriptional networks during seed germination

Following the identification of two bHLH transcription factors that repress germination and maintain dormancy, the genetic hierarchies that integrate external light and temperature signals with endogenous signals that break seed dormancy have started to be explained (BOX 2). Seeds produced by a loss-of-function mutant of $PIF1$ display reduced dormancy and light signals are no longer needed for germination as they are in wild-type seeds⁶²,⁹⁶. Conversely, $PIF1$-overexpressing lines require increased exposure to red light for germination, further confirming that $PIF1$ is a repressor of germination⁶². As a putative transcription factor, $PIF1$ negatively regulates endogenous levels of active gibberellic acid (GA) to maintain dormancy by transcriptionally repressing GA biosynthesis genes and activating GA degradation genes directly or indirectly⁹⁶,⁹⁷. The protein level of $PIF1$ is tightly controlled by light through its interactions with PHYB and PHYA, preferentially the Pfr forms⁶¹,⁶². PHYA and PHYB significantly decrease the stability of $PIF1$ under light as part of the process of breaking seed dormancy⁶⁷,⁹⁷ (FIG. 4).

The bHLH transcription factor SPATULA (SPT) has been recently identified as another repressor of germination⁹⁶. Loss-of-function $spt$ mutant seeds show insensitivity to cold stratification, so that their germination does not need stratification even when freshly harvested. On the other hand, $spt$ mutants have only a slight increase in germination in darkness⁹⁶, and SPT does not interact with phytochromes⁵⁷. So, SPT contributes predominantly to temperature-controlled seed dormancy. Similar to $PIF1$, SPT negatively regulates endogenous levels of GA by transcriptionally repressing the GA biosynthesis gene, and possibly by other mechanisms. A model therefore emerges in which there is a division of labour between $PIF1$ and SPT with respect to their abilities to sense different dormancy-breaking signals, that is, light and temperature (FIG. 4). It would be interesting to further study how $PIF1$ and SPT act together to integrate light and temperature signals.

---

**Figure 4 | A model for the light and temperature control of seed germination.**

Both PHYTOCHROME INTERACTING FACTOR 1 ($PIF1$) and SPATULA ($SPT$) inhibit germination by regulating the level of active gibberellic acid (GA). They act as transcriptional repressors of GA biosynthetic genes ($GA3ox1$ and $GA3ox2$), which are required to convert the GA precursor to its active form. In addition, $PIF1$ activates the expression of a GA catabolic gene ($GA2ox2$). Under inductive light conditions (in which a relatively high red:far-red ratio converts sufficient phytochrome B to the active, nuclear-localized Pfr form), phytochrome binding triggers the degradation of $PIF1$ through ubiquitin-mediated proteolysis, thereby blocking the repression of GA levels. Similarly, cold treatment represses $SPT$ activity to promote germination.

---

**Nuclear matrix**  
The network of fibres found throughout the inside of a cell nucleus.

**Histone acetyltransferase**  
A type of enzyme that acetylates conserved lysine amino acids on histones by transferring an acetyl group from acetyl CoA to lysine to form $\varepsilon$-$N$-acetyl lysine.

**Histone deacetylase**  
A type of enzyme that removes an acetyl group from histones, which usually allows histones to bind DNA and inhibit gene transcription.

**Gibberellic acid**  
A phytohormone involved in promoting stem elongation, seed germination, mobilization of food reserves in seeds and other processes.

**Cold stratification**  
A dormancy-breaking process of treating seeds with a period of moist cold, which a seed must endure before germination.

REVIEW S

Transcriptional networks during shade avoidance

To date, five genes have been identified as primary targets of phytochromes during shade-avoidance responses in *A. thaliana*, *ATHB2* (also known as *HAT4*), *ATHB4*, *PIF3-LIKE 1* (*PIL1*), *PHY RAPIDLY REGULATED 1* (*PAR1*) and *HFR1* (REFS 98,99). The transcript levels of these genes increase within a few minutes of low red:far-red exposure (the situation under a dense canopy) (BOX 2). Except for *PAR1*, the function of which is largely unexplored, all the other genes encode transcription factors. Both *ATHB2* and *ATHB4* encode plant-specific homeodomain-leucine zipper transcription factors<sup>100</sup>. The function of *ATHB2* has been well established: this protein promotes the shade-avoidance response by acting as a repressor of gene expression<sup>101</sup>. The expression of *ATHB2* is itself under tight light-regulated transcriptional control (FIG. 5). *ATHB2* binds to its own promoter and represses transcription, forming a negative autoregulatory loop to efficiently maintain the optimal *ATHB2* level<sup>102</sup>. Finally, the bHLH protein *PIL1* is required for both the normal shade-avoidance elongation response and in the plant circadian system<sup>103–105</sup>.

A dynamic balance of positive and negative transcriptional regulation that functions as a gas-and-brake mechanism during the shade-avoidance response was recently uncovered. Most genes for which expression is rapidly induced under low red:far-red conditions are downregulated after prolonged exposure to this type of light source with a few exceptions, including *HFR1* (REF 99). Furthermore, the decrease in the expression of many shade-avoidance-related genes that is provoked by prolonged low red:far-red exposure is essentially abolished in *hfr1* mutants. Therefore, *HFR1* negatively regulates other targets to attenuate the shade-avoidance response<sup>99</sup> (FIG. 5). This negative regulatory mechanism is central to the attenuation of virtually all plant responses to canopy shade<sup>99</sup>. Such a negative controller of the shade-avoidance response ensures that an exaggerated reaction does not occur when the plant is unsuccessful in escaping canopy shade.

Networks involved in the photoperiod response

Circadian rhythms are driven by endogenous biological clocks that confer a 24-hour rhythm on many biochemical and physiological processes and seasonal rhythms, such as flowering (BOX 2). In higher plants, as in animals and fungi, endogenous biological clocks are based on simple autoregulatory negative-feedback loops, which are synchronized by entraining stimuli, in particular light and temperature (FIG. 6). A few recent studies have significantly extended earlier models by integrating all known clock components into a functional molecular circuit<sup>106–108</sup>. The *A. thaliana* central loop comprises two morning-expressing MYB transcription factors *CCA1* and *LHY*<sup>78,79</sup>, and the evening-expressing *TIMING OF CAB EXPRESSION 1* (*TOC1*; also known as *PRR1*)<sup>109</sup>. A model has been proposed in which *CCA1* and *LHY* work redundantly to repress *TOC1* expression by directly binding to the evening element of its promoter, a motif that is over-represented in the promoters of evening-phased genes<sup>110,111</sup>. *TOC1* closes the loop as an indirect positive regulator of *CCA1* and *LHY* expression<sup>111,112</sup>. Similarly, *EARLY FLOWERING 4* (*ELF4*), *GIGANTEA* (*GI*) and *LUX ARRHYTHMO* (*LUX*; also known as *PCL1*) are thought to form feedback loops with *CCA1/LHY*<sup>78,111,113–118</sup>. An additional negative-feedback loop involves *CCA1/LHY* and three *TOC1* paralogues: *PRR5*, *PRR7* and *PRR9*. Under steady-state entrained conditions, these three factors repress the expression of *LHY* and *CCA1* and their own expression is positively regulated by *LHY* and *CCA1* (REFS 119–121).

Multiple light and temperature input points precisely modulate the circadian clock by acting in environmental entrainment pathways. Light synchronizes the endogenous clock at several points (FIG. 6). Light activates the transcription of the *LHY* and *CCA1* genes<sup>111</sup>, and transcripts of *TOC1* and *EFL4*, the other arm of the feedback loops, are also positively regulated by light<sup>115</sup>. *ELF3*, a *PHYB*-interacting protein<sup>122</sup>, negatively affects light input on multiple genes in the clock<sup>115,122,123</sup>, and seems to gate light signals that originate from both red and blue photoreceptors<sup>123</sup>. In addition, *FYH3* specifically gates phytochrome signalling to the circadian clock<sup>124</sup>, and *TOC1* interacts with *PIF* proteins and *PIL1*, as shown in yeast two-hybrid assays<sup>105</sup>, although a recent report showed that *PIF3* has a minor role, if any, in mediating

Figure 5 | Transcriptional networks involved in the shade-avoidance response. A low red:far-red light ratio in a canopy is sensed through the phytochromes *PHYB*, *PHYD* and *PHYE*. The perception of a varying red:far-red ratio is achieved through a change in the equilibrium of the Pr and Pfr forms of phytochromes, as the absorption spectra of the two forms partially overlap and neither of the photoconversions can be driven to completion. A dynamic balance between transcriptional activation and repression downstream of phytochromes regulates the shade-avoidance response. Phytochrome-mediated signals rapidly induce the transcription of *LONG HYPOCOTYL IN FAR-RED* (*HFR1*), *ATHB2* and other early response transcription factors. *HFR1* negatively regulates the transcription of other transcription factors to ensure that an exaggerated response does not occur. The shade-avoidance response is further attenuated by the self-repression feedback of *ATHB2*. *PAR1*, *PHY RAPIDLY REGULATED 1*; *PIL1*, *PIF3-LIKE 1*.

Figure 6 | The photoperiod response in *Arabidopsis thaliana*. The three transcription feedback loops of the circadian oscillator, which all consist of CIRCADIAN CLOCK ASSOCIATED 1 (CCA1) and LATE ELONGATED HYPOCOTYL (LHY), are shown in the shaded areas. In the first loop, CCA1 and LHY repress TIMING OF CAB EXPRESSION 1 (TOC1) expression by binding to the evening element (EE) of its promoter, while TOC1 acts as a positive regulator for CCA1 and LHY expression. Similarly, EARLY FLOWERING 4 (ELF4), GIGANTEA (GI) and LUX ARRHYTHMO (LUX) form another feedback loop with CCA1/LHY. By contrast, in the third feedback loop, CCA1 and LHY positively regulate the expression of PRR5, PRR7 and PRR9, and these three proteins repress the expression of CCA1 and LHY. The grey shaded area in the box indicates activities that peak in the subjective night, whereas the blue area indicates activities that peak during the subjective day. Light signals are perceived by phytochromes (PHYs) and cryptochromes (CRYs), and possibly by ZEITLUPE (ZTL) and LOV KELCH PROTEIN 2 (LKP2). Dashed lines indicate the extrinsic inputs that regulate gene expression.

light input to the clock<sup>125</sup>. Finally, two novel proteins ZEITLUPE (ZTL; also known as ADO1) and LOV KELCH PROTEIN 2 (LKP2) precisely regulate TOC1 levels through degradation<sup>126–128</sup>, which is reminiscent of the roles of COP1 during seedling photomorphogenesis. ZTL is a key component of an SCF complex that specifically targets TOC1 for degradation<sup>126,129,130</sup>. Given that the activity of ZTL shows fluence rate dependency and the N-terminal LOV domain (BOX 2) of ZTL and LKP2 possesses light-sensing properties, a role for ZTL and LKP2 in light input has been suggested<sup>126,131</sup>.

The circadian clock gates light responses. For example, transcription of the CHLOROPHYLL A/B BINDING PROTEIN (CAB) gene, which encodes a key photosynthetic enzyme, is induced in response to short light pulses applied during the subjective day. By contrast, only minimal transcriptional responses are observed when identical light signals are applied during the subjective night<sup>132</sup>. In fact, many components of the circadian clock are also involved in other light-regulated processes. Through these connecting points, the circadian clock mediates the photoperiod response by integrating with developmental pathways. For instance, mutants of *elf3*, *gi* or *prr5prr7prr9* show phenotypes that indicate

Fluence rate  
The light irradiance that is incident from all angles onto a small region of space.

Subjective day  
An artificially defined daytime in circadian experiments.

Thermocycles  
Alternating warm and cool conditions.

temperature sensitivity of light signalling pathways indicates that light and temperature pathways might be highly integrated<sup>137</sup>.

**Conclusions and perspectives**

It has now been well established that light regulates complex remodelling of the transcriptome during many developmental processes in plants. Genetic and genomic studies have identified a growing list of light-controlled genes, and we now have a general understanding of the basic framework of light-regulated signalling networks, in particular photomorphogenesis. However, there are many gaps in our understanding of the structure and mechanisms of these transcriptional networks, each of which control a different developmental response.

Although many transcription factors have been defined as key factors in the light-regulated transcriptional network, their downstream targets are mostly unclear. The ChIP-on-chip approach has recently been used to systematically identify direct targets of HY5, a key transcription factor in seedling photomorphogenesis<sup>75</sup>. The integration of genome-wide expression profiles with binding data from ChIP-on-chip assays will help in filling the major gaps in our understanding of the architecture of these transcriptional regulatory networks.

The level of organ-specific light responses on gene expression has recently been reported in *A. thaliana* and rice<sup>4,80</sup>. However, the molecular basis for organ specificity has barely been explored. A reverse genetic approach applied to genes that are involved in the early response to PHYA has shown that many of these genes have organ-specific effects, which might not be large enough to be detected in a forward genetic screen<sup>81</sup>. A greater understanding of the components involved in organ specificity will therefore require more detailed genomic and genetic studies, such as identifying photo-receptor-regulated early response genes in specific organs by expression profiling and functional analysis using reverse genetic approaches.

Special attention has been paid in recent years to the question of how light responses are integrated with intrinsic signals (such as those from the circadian clock) and other environmental signals (such as temperature). In addition, crosstalk has been shown to exist among different physiological responses to light, as various key regulators in the light signalling pathways have several seemingly discrete physiological roles. The molecular mechanisms for many of these connections remain to be understood. It is likely that some proteins, especially transcription factors, have several roles during different light-regulated physiological processes. Further genetic studies of other functions of previously identified factors should shed light on this issue.

Finally, although we have a good understanding of how plants distinguish differences in light quality, it is still unclear how plants distinguish differences in light quantity, duration and direction. Such regulation could be at both the photoreceptor level and at the level of downstream signalling networks. Resolving these questions requires the identification of key steps in signalling under specific conditions, through precisely designed genetic and genome-wide transcriptomic studies.


1. Chen, M., Chory, J. & Fankhauser, C. Light signal transduction in higher plants. *Annu. Rev. Genet.* **38**, 87–117 (2004).
   - This review provides a comprehensive, up-to-date overview of plant photoreceptors.
2. Tepperman, J. M., Zhu, T., Chang, H.-S., Wang, X. & Quail, P. H. Multiple transcription-factor genes are early targets of phytochrome A signaling. *Proc. Natl Acad. Sci. USA* **98**, 9437–9442 (2001).
   - This work identifies the transcriptional cascades that lie downstream of PHYA, which involve multiple transcription factors as primary targets.
3. Ma, L. *et al.* Light control of *Arabidopsis* development entails coordinated regulation of genome expression and cellular pathways. *Plant Cell* **13**, 2589–2607 (2001).
   - A landmark paper that demonstrates the massive effects of light on genome-wide transcription.
4. Jiao, Y., Ma, L., Strickland, E. & Deng, X. W. Conservation and divergence of light-regulated genome expression patterns during seedling development in rice and *Arabidopsis*. *Plant Cell* **17**, 3239–3256 (2005).
   - This paper compares light-regulated transcription profiles in *A. thaliana* and rice. Organ specificity of the transcriptome and the evolution of the light-regulated transcriptome are also discussed.
5. Quail, P. H. Phytochrome photosensory signalling networks. *Nature Rev. Mol. Cell Biol.* **3**, 85–93 (2002).
6. Lin, C. & Shalitin, D. Cryptochrome structure and signal transduction. *Annu. Rev. Plant Biol.* **54**, 469–496 (2003).
7. Kleine, T., Lockhart, P. & Batschauer, A. An *Arabidopsis* protein closely related to Synechocystis cryptochrome is targeted to organelles. *Plant J.* **35**, 93–103 (2003).
8. Kim, J.-I. *et al.* Phytochrome phosphorylation modulates light signaling by influencing the protein–protein interaction. *Plant Cell* **16**, 2629–2640 (2004).
9. Ryu, J. S. *et al.* Phytochrome-specific type 5 phosphatase controls light signal flux by enhancing phytochrome stability and affinity for a signal transducer. *Cell* **120**, 395–406 (2005).
10. Al-Sady, B., Ni, W., Kircher, S., Schäfer, E. & Quail, P. H. Photoactivated phytochrome induces rapid PIF3 phosphorylation prior to proteasome-mediated degradation. *Mol. Cell* **23**, 439–446 (2006).
11. Yeh, K.-C. & Lagarias, J. C. Eukaryotic phytochromes: light-regulated serine/threonine protein kinases with histidine kinase ancestry. *Proc. Natl Acad. Sci. USA* **95**, 13976–13981 (1998).
12. Matsushita, T., Mochizuki, N. & Nagatani, A. Dimers of the N-terminal domain of phytochrome B are functional in the nucleus. *Nature* **424**, 571–574 (2003).
13. Sang, Y. *et al.* N-terminal domain-mediated homodimerization is required for photoreceptor activity of *Arabidopsis* CRYPTOCHROME 1. *Plant Cell* **17**, 1569–1584 (2005).
14. Christie, J. M. *et al.* *Arabidopsis* NPH1: a flavoprotein with the properties of a photoreceptor for phototropism. *Science* **282**, 1698–1701 (1998).
15. Solomon, M., Christie, J. M., Knieb, E., Lempert, U. & Briggs, W. R. Photochemical and mutational analysis of the FMN-binding domains of the plant blue light receptor, phototropin. *Biochemistry* **39**, 9401–9410 (2000).
16. Weisshaar, B., Armstrong, G. A., Block, A., da Costa e Silva, O. & Hahlbrock, K. Light-inducible and constitutively expressed DNA-binding proteins recognizing a plant promoter element with functional relevance in light responsiveness. *EMBO J.* **10**, 1777–1786 (1991).
17. Feldbrugge, M. *et al.* Functional analysis of a light-responsive plant bZIP transcriptional regulator. *Plant Cell* **6**, 1607–1621 (1994).
18. Klimczak, L. J., Schindler, U. & Cashmore, A. R. DNA binding activity of the *Arabidopsis* G-box binding factor GBF1 is stimulated by phosphorylation by casein kinase II from broccoli. *Plant Cell* **4**, 87–98 (1992).
19. Klimczak, L. J. *et al.* Reconstitution of *Arabidopsis* casein kinase II from recombinant subunits and phosphorylation of transcription factor GBF1. *Plant Cell* **7**, 105–115 (1995).
20. Harter, K. *et al.* Light-regulated modification and nuclear translocation of cytosolic G-box binding factors in parsley. *Plant Cell* **6**, 545–559 (1994).
21. Kircher, S. *et al.* Nuclear import of the parsley bZIP transcription factor CPRF2 is regulated by phytochrome photoreceptors. *J. Cell Biol.* **144**, 201–211 (1999).
22. Wellmer, F. *et al.* Phosphorylation of the parsley bZIP transcription factor CPRF2 is regulated by light. *J. Biol. Chem.* **274**, 29476–29482 (1999).
23. Osterlund, M. T., Hardtke, C. S., Wei, N. & Deng, X. W. Targeted destabilization of HY5 during light-regulated development of *Arabidopsis*. *Nature* **405**, 462–466 (2000).
   - A landmark paper that shows that transcription factors, which are downstream of light-signalling pathways, are regulated at the level of protein degradation by COP1 and other proteins. This work demonstrates for the first time that photomorphogenesis is governed by proteolysis.
24. Yi, C. & Deng, X. W. COP1 — from plant photomorphogenesis to mammalian tumorigenesis. *Trends Cell Biol.* **15**, 618–625 (2005).
25. Seo, H.-S. *et al.* LAF1 ubiquitination by COP1 controls photomorphogenesis and is stimulated by SPA1. *Nature* **423**, 995–999 (2003).
26. von Arnim, A. G. & Deng, X. W. Light inactivation of *Arabidopsis* photomorphogenic repressor COP1 involves a cell-specific regulation of its nucleocytoplasmic partitioning. *Cell* **79**, 1035–1045 (1994).
27. Jakoby, M. *et al.* bZIP transcription factors in *Arabidopsis*. *Trends Plant Sci.* **7**, 106–111 (2002).
28. Toledo-Ortiz, G., Huq, E. & Quail, P. H. The *Arabidopsis* basic/helix-loop-helix transcription factor family. *Plant Cell* **15**, 1749–1770 (2003).

© 2007 Nature Publishing Group

29. Holm, M., Ma, L.-G., Qu, L.-J. & Deng, X.-W. Two interacting bZIP proteins are direct targets of COP1-mediated control of light-dependent gene expression in *Arabidopsis*. *Genes Dev.* 16, 1247–1259 (2002).

30. Schindler, U., Menkens, A. E., Beckmann, H., Ecker, J. R. & Cashmore, A. R. Heterodimerization between light-regulated and ubiquitously expressed *Arabidopsis* GBF bZIP proteins. *EMBO J.* 11, 1261–1273 (1992).

31. Sprenger-Haussels, M. & Weisshaar B. Transactivation properties of parsley proline-rich bZIP transcription factors. *Plant J.* 22, 1–8 (2000).

32. Feldbrugge, M., Sprenger, M., Hahlbrock, K. & Weisshaar, B. PcMYB1, a novel plant protein containing a DNA-binding domain with one MYB repeat, interacts *in vivo* with a light-regulatory promoter unit. *Plant J.* 11, 1079–1093 (1997).

33. Zimmermann, I. M., Heim, M. A., Weisshaar, B. & Uhrig, J. F. Comprehensive identification of *Arabidopsis thaliana* MYB transcription factors interacting with R/B-like BHLH proteins. *Plant J.* 40, 22–34 (2004).

34. Terzaghi, W. B. & Cashmore, A. R. Light-regulated transcription. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* 46, 445–474 (1995).

35. Arguello-Astorga, G. & Herrera-Estrella, L. Evolution of light-regulated plant promoters. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* 49, 525–555(1998).

36. Hudson, M. E. & Quail, P. H. Identification of promoter motifs involved in the network of phytochrome A-regulated gene expression by combined analysis of genomic sequence and microarray data. *Plant Physiol.* 133, 1605–1616 (2003). This paper describes a genome-scale approach to identifying promoter *cis*-elements in PHYA-responding genes.

37. Jiao, Y. *et al.* A genome-wide analysis of blue-light regulation of *Arabidopsis* transcription factor gene expression during seedling development. *Plant Physiol.* 133, 1480–1493 (2003).

38. Tepperman, J. M. *et al.* Expression profiling of PHYB mutant demonstrates substantial contribution of other phytochromes to red-light-regulated gene expression during seedling de-etiolation. *Plant J.* 38, 725–739 (2004).

39. Riechmann, J. L. *et al.* *Arabidopsis* transcription factors: genome-wide comparative analysis among eukaryotes. *Science* 290, 2105–2110 (2000).

40. Tepperman, J. M., Hwang, Y.-S. & Quail, P. H. PHYA dominates in transduction of red-light signals to rapid-responding genes at the initiation of *Arabidopsis* seedling deetiolation. *Plant J.* 48, 728–742 (2006).

41. Wang, H. *et al.* Analysis of far-red light-regulated genome expression profiles of phytochrome A pathway mutants in *Arabidopsis*. *Plant J.* 32, 723–733 (2002). This work exploits global expression analysis to understand the effects of mutations in key regulators in the PHYA-signalling pathway.

42. McCormac, A. C. & Terry, M. J. Light-signalling pathways leading to the co-ordinated expression of HEMA1 and *Lhcb* during chloroplast development in *Arabidopsis thaliana*. *Plant J.* 32, 549–559 (2002).

43. Hudson, M., Ringli, C., Boylan, M. T. & Quail, P. H. The FAR1 locus encodes a novel nuclear protein specific to phytochrome A signaling. *Genes Dev.* 13, 2017–2027 (1999).

44. Wang, H & Deng, X. W. *Arabidopsis* PHY3 defines a key phytochrome A signaling component directly interacting with its homologous partner FAR1. *EMBO J.* 21, 1339–1349 (2002).

45. Hudson, M. E., Lisch, D. R. & Quail, P. H. The *PHY3* and *FAR1* genes encode transposase-related proteins involved in regulation of gene expression by the phytochrome A-signaling pathway. *Plant J.* 34, 453–471 (2003).

46. Ballesteros, M. L. *et al.* LAF1, a MYB transcription activator for phytochrome A signaling. *Genes Dev.* 15, 2613–2625 (2001).

47. Fairchild, C. D., Schumaker, M. A. & Quail, P. H. HFR1 encodes an atypical bHLH protein that acts in phytochrome A signal transduction. *Genes Dev.* 14, 2377–2391 (2000).

48. Duek, P. D. & Fankhauser, C. HFR1, a putative bHLH transcription factor, mediates both phytochrome A and cryptochrome signalling. *Plant J.* 34, 827–836 (2003).

49. Soh, M.-S., Kim, Y.-M., Han, S.-J. & Song, P.-S. REP1, a basic helix–loop–helix protein, is required for a branch pathway of phytochrome A signaling in *Arabidopsis*. *Plant Cell* 12, 2061–2074 (2000).

50. Spiegelman, J. I. *et al.* Cloning of the *Arabidopsis RSF1* gene by using a mapping strategy based on high-density DNA arrays and denaturing high-performance liquid chromatography. *Plant Cell* 12, 2485–2498 (2000).

51. Kim, Y.-M., Woo, J.-C., Song, P.-S. & Soh, M.-S. HFR1, a phytochrome A-signalling component, acts in a separate pathway from HY5, downstream of COP1 in *Arabidopsis thaliana*. *Plant J.* 30, 711–719 (2002).

52. Yang, K.-Y., Kim, Y.-M., Lee, S., Song, P.-S. & Soh, M.-S. Overexpression of a mutant basic helix–loop–helix protein HFR1, HFR1-δN105, activates a branch pathway of light signaling in *Arabidopsis*. *Plant Physiol.* 133, 1630–1642 (2003).

53. Park, D.-H. *et al.* The *Arabidopsis COG1* gene encodes a Dof domain transcription factor and negatively regulates phytochrome signaling. *Plant J.* 34, 161–171 (2003).

54. Ward, J. M., Cufr, C. A., Denzel, M. A. & Neff, M. M. The Dof transcription factor OBP3 modulates phytochrome and cryptochrome signaling in *Arabidopsis*. *Plant Cell* 17, 475–485 (2005).

55. Yadav, V., Mallappa, C., Gangappa, S. N., Bhatia, S. & Chattopadhyay, S. A basic helix-loop-helix transcription factor in *Arabidopsis*, MYC2, acts as a repressor of blue light-mediated photomorphogenic growth. *Plant Cell* 17, 1953–1966 (2005).

56. Mallappa, C., Yadav, V., Negi, P. & Chattopadhyay, S. A bzip transcription factor, G-box binding factor 1, regulates blue light mediated photomorphogenic growth in *Arabidopsis*. *J. Biol. Chem.* 31, 22190–22199 (2006).

57. Khanna, R. *et al.* A novel molecular recognition motif necessary for targeting photoactivated phytochrome signaling to specific basic helix–loop–helix transcription factors. *Plant Cell* 16, 3033–3044 (2004).

58. Ni, M., Tepperman, J. M. & Quail, P. H. PIF3, a phytochrome-interacting factor necessary for normal photoinduced signal transduction, is a novel basic helix–loop–helix protein. *Cell* 95, 657–667 (1998).

59. Zhu, Y., Tepperman, J. M., Fairchild, C. D. & Quail, P. H. Phytochrome B binds with greater apparent affinity than phytochrome A to the basic helix–loop–helix factor PIF3 in a reaction requiring the PAS domain of PIF3. *Proc. Natl Acad. Sci. USA* 97, 13419–13424 (2000).

60. Huq, E. & Quail, P. H. PIF4, a phytochrome-interacting bHLH factor, functions as a negative regulator of phytochrome B signaling in *Arabidopsis*. *EMBO J.* 21, 2441–2450 (2002).

61. Huq, E. *et al.* Phytochrome-interacting factor 1 is a critical bHLH regulator of chlorophyll biosynthesis. *Science* 305, 1937–1941 (2004).

62. Oh, E. *et al.* PIL5, a phytochrome-interacting basic helix–loop–helix protein, is a key negative regulator of seed germination in *Arabidopsis thaliana*. *Plant Cell* 16, 3045–3058 (2004). References 61 and 62 describe the identification of an additional branch of the PHYA-signalling network, which negatively regulates chlorophyll biosynthesis and represses seed germination.

63. Kim, J. *et al.* Functional characterization of phytochrome interacting factor 3 in phytochrome-mediated light signal transduction. *Plant Cell* 15, 2399–2407 (2003).

64. Bauer, D. *et al.* Constitutive photomorphogenesis 1 and multiple photoreceptors control degradation of phytochrome interacting factor 3, a transcription factor required for light signaling in *Arabidopsis*. *Plant Cell* 16, 1433–1445 (2004).

65. Fujimori, T., Yamashino, T., Kato, T. & Mizuno, T. Circadian-controlled basic/helix–loop–helix factor, PIL6, implicated in light-signal transduction in *Arabidopsis thaliana*. *Plant Cell Physiol.* 45, 1078–1086 (2004).

66. Monte, E. *et al.* The phytochrome-interacting transcription factor, PIF3, acts early, selectively, and positively in light-induced chloroplast development. *Proc. Natl Acad. Sci. USA* 101, 16091–16098 (2004). A comprehensive study of the multiple facets of PIF3, a PHYA-interacting transcription factor that regulates a branch of the PHYA-signalling network.

67. Shen, H., Moon, J. & Huq, E. PIF1 is regulated by light-mediated degradation through the ubiquitin-26S proteasome pathway to optimize photomorphogenesis of seedlings in *Arabidopsis*. *Plant J.* 44, 1023–1035 (2005).

68. Brown, B. A. *et al.* A UV-B-specific signaling component orchestrates plant UV protection. *Proc. Natl Acad. Sci. USA* 102, 18225–18230 (2005).

69. Oravecz, A., *et al.* CONSTITUTIVELY PHOTOMORPHOGENIC1 is required for the UV-B response in *Arabidopsis*. *Plant Cell* 18, 1975–1990 (2006).

70. Oyama, T., Shimura, Y. & Okada, K. The *Arabidopsis HY5* gene encodes a bZIP protein that regulates stimulus-induced development of root and hypocotyl. *Genes Dev.* 11, 2983–2995 (1997).

71. Ulm, R., *et al.* Genome-wide analysis of gene expression reveals function of the bZIP transcription factor HY5 in the UV-B response of *Arabidopsis*. *Proc. Natl Acad. Sci. USA* 101, 1397–1402 (2004).

72. Hardtke, C. S. *et al.* HY5 stability and activity in *Arabidopsis* is regulated by phosphorylation in its COP1 binding domain. *EMBO J.* 19, 4997–5006 (2000).

73. Ang, L.-H. *et al.* Molecular interaction between COP1 and HY5 defines a regulatory switch for light control of *Arabidopsis* development. *Mol. Cell* 1, 213–222 (1998).

74. Chattopadhyay, S., Ang, L.-H., Puente, P., Deng, X.-W. & Wei, N. *Arabidopsis* bZIP protein HY5 directly interacts with light-responsive promoters in mediating light control of gene expression. *Plant Cell* 10, 673–683 (1998).

75. Lee, J. *et al.* Analysis of transcription factor HY5 genomic binding sites revealed its hierarchical role in light regulation of development. *Plant Cell* (in the press).

76. Ma, L. *et al.* Genomic evidence for COP1 as a repressor of light-regulated gene expression and development in *Arabidopsis*. *Plant Cell* 14, 2383–2398 (2002).

77. Wang, Z.-Y. *et al.* A Myb-related transcription factor is involved in the phytochrome regulation of an *Arabidopsis Lhcb* gene. *Plant Cell* 9, 491–507 (1997).

78. Schaffer, R. *et al.* The late elongated hypocotyl mutation of *Arabidopsis* disrupts circadian rhythms and the photoperiodic control of flowering. *Cell* 93, 1219–1229 (1998).

79. Wang, Z.-Y. & Tobin, E. M. Constitutive expression of the CIRCADIAN CLOCK ASSOCIATED 1 (*CCA1*) gene disrupts circadian rhythms and suppresses its own expression. *Cell* 93, 1207–1217 (1998).

80. Ma, L. *et al.* Organ-specific expression of *Arabidopsis* genome during development. *Plant Physiol.* 138, 80–91 (2005).

81. Khanna, R. *et al.* Functional profiling reveals that only a small number of phytochrome-regulated early-response genes in *Arabidopsis* are necessary for optimal deetiolation. *Plant Cell* 18, 2157–2171 (2006).

82. Degenhardt, J. & Tobin, E. M. A DNA binding activity for one of two closely defined phytochrome regulatory elements in an *Lhcb* promoter is more abundant in etiolated than in green plants. *Plant Cell* 8, 31–41 (1996).

83. Puente, P., Wei, N. & Deng, X. W. Combinatorial interplay of promoter elements constitutes the minimal determinants for light and developmental control of gene expression in *Arabidopsis*. *EMBO J.* 15, 3732–3743 (1996).

84. Chattopadhyay, S., Puente, P., Deng, X. W. & Wei, N. Combinatorial interaction of light-responsive elements plays a critical role in determining the response characteristics of light-regulated promoters in *Arabidopsis*. *Plant J.* 15, 69–77 (1998).

85. Chua, Y.-L., Brown, A. P. & Gray, J. C. Targeted histone acetylation and altered nuclease accessibility over short regions of the pea plastocyanin gene. *Plant Cell* 13, 599–612 (2001).

86. Chua, Y.-L., Watson, L. A. & Gray, J. C. The transcriptional enhancer of the pea plastocyanin gene associates with the nuclear matrix and regulates gene expression through histone acetylation. *Plant Cell* 15, 1468–1479 (2003).

87. Bertrand, C. *et al.* *Arabidopsis* HAF2 gene encoding TATA-binding protein (TBP)-associated factor TAF1, is required to integrate light signals to regulate gene expression and growth. *J. Biol. Chem.* 280, 1465–1473 (2005).

88. Benhamed, M., Bertrand, C., Servet, C. & Zhou, D.-X. *Arabidopsis GCN5, HD1,* and *TAF1/HAF2* interact to regulate histone acetylation required for light-responsive gene expression. *Plant Cell* 18, 2893–2903 (2006). This paper, with reference 87, demonstrates that an HAT and an HDAC are involved in transcriptional regulation downstream of multiple photoreceptors, indicating that chromatin remodelling is a convergence point in light-regulated transcription.

REVIEWS

89. Chory, J., Peto, C., Feinbaum, R., Pratt, L. & Ausubel, F. *Arabidopsis thaliana* mutant that develops as a light-grown plant in the absence of light. *Cell* **58**, 991–999 (1989).

90. Pepper, A., Delaney, T., Washburn, T., Poole, D. & Chory, J. *DET1*, a negative regulator of light-mediated development and gene expression in *Arabidopsis*, encodes a novel nuclear-localized protein. *Cell* **78**, 109–116 (1994).

91. Schroeder, D. F. *et al.* De-etiolated 1 and damaged DNA binding protein 1 interact to regulate *Arabidopsis* photomorphogenesis. *Curr. Biol.* **12**, 1462–1472 (2002).

92. Yanagawa, Y. *et al.* *Arabidopsis COP10* forms a complex with DDB1 and *DET1 in vivo* and enhances the activity of ubiquitin conjugating enzymes. *Genes Dev.* **18**, 2172–2181 (2004).

93. Benvenuto, G., Formiggini, F., Laflamme, P., Malakhov, M. & Bowler, C. The photomorphogenesis regulator *DET1* binds the amino-terminal tail of histone H2B in a nucleosome context. *Curr Biol.* **12**, 1529–1534 (2002).

References 91–93 describe the interaction of *DET1*, *COP10* and *DDB1* as a protein complex and suggest that chromatin remodelling might be involved in photomorphogenesis.

94. Berloco, M., Fanti, L., Breling, A., Orlando, V. & Pimpinelli, S. The maternal effect gene, *abnormal oocyte (abo)*, of *Drosophila melanogaster* encodes a specific negative regulator of histones. *Proc. Natl Acad. Sci. USA* **98**, 12126–12131 (2001).

95. Maxwell, B. B., Andersson, C. R., Poole, D. S., Kay, S. A. & Chory, J. HY5, Circadian Clock-Associated 1, and a *cis*-element, *DET1* dark response element, mediate *DET1* regulation of chlorophyll a/b-binding protein 2 expression. *Plant Physiol.* **133**, 1565–1577 (2003).

96. Penfield, S. *et al.* Cold and light control seed germination through the bHLH transcription factor SPATULA. *Curr. Biol.* **5**, 378–381 (2005).

97. Oh, E. *et al.* Light activates the degradation of PIL5 protein to promote seed germination through gibberellin in *Arabidopsis*. *Plant J.* **47**, 124–139 (2006).

References 96 and 97 provide evidence that light signal is transduced through a phytochrome-interacting bHLH transcription factor to control seed germination by regulating endogenous GA level.

98. Roig-Villanova, I., Bou, J., Sorin, C., Devlin, P. F. & Martinez-Garcia, J. F. Identification of primary target genes of phytochrome signaling. Early transcriptional control during shade avoidance responses in *Arabidopsis*. *Plant Physiol.* **141**, 85–96 (2006).

99. Sessa, G. *et al.* A dynamic balance between gene activation and repression regulates the shade avoidance response in *Arabidopsis*. *Genes Dev.* **19**, 2811–2815 (2005).

This paper describes a negative regulatory mechanism in shade-avoidance response.

100. Carabelli, M., Sessa, G., Baima, S., Morelli, G. & Ruberti, I. The *Arabidopsis Athb-2* and -4 genes are strongly induced by far-red-rich light. *Plant J.* **4**, 469–479 (1993).

101. Steindler, C. *et al.* Shade avoidance responses are mediated by the ATHB-2 HD-zip protein, a negative regulator of gene expression. *Development* **126**, 4235–4245 (1999).

102. Oghishi, M., Oka, A., Morelli, G., Ruberti, I. & Aoyama, T. Negative autoregulation of the *Arabidopsis* homeobox gene *ATHB-2*. *Plant J.* **25**, 389–398 (2001).

103. Salter, M. G., Franklin, K. A. & Whitelam, G. C. Gating of the rapid shade-avoidance response by the circadian clock in plants. *Nature* **426**, 680–683 (2003).

104. Devlin, P. F., Yanovsky, M. J. & Kay, S. A. A genomic analysis of the shade avoidance response in *Arabidopsis*. *Plant Physiol.* **133**, 1617–1629 (2003).

105. Yamashino, T. *et al.* A Link between circadian-controlled bHLH factors and the APRR1/TOC1 quintet in *Arabidopsis thaliana*. *Plant Cell Physiol.* **44**, 619–629 (2003).

106. Locke, J. C. *et al.* Extension of a genetic network model by iterative experimentation and mathematical analysis. *Mol. Syst. Biol.* **1**, 2005.0013 (2005).

107. Zeilinger, M. N., Farré, E. M., Taylor, S. R., Kay, S. A. & Doyle, F. J. III. A novel computational model of the circadian clock in *Arabidopsis* that incorporates PRR7 and PRR9. *Mol. Syst. Biol.* **2**, 58 (2006).

108. Locke, J. C. *et al.* Experimental validation of a predicted feedback loop in the multi-oscillator clock of *Arabidopsis thaliana*. *Mol. Syst. Biol.* **2**, 59 (2006).

109. Strayer, C. *et al.* Cloning of the *Arabidopsis* clock gene *TOC1*, an autoregulatory response regulator homolog. *Science* **289**, 768–771 (2000).

110. Harmer, S. L. *et al.* Orchestrated transcription of key pathways in *Arabidopsis* by the circadian clock. *Science* **290**, 2110–2113 (2000).

111. Alabadí, D. *et al.* Reciprocal regulation between *TOC1* and *LHY/CCA1* within the *Arabidopsis* circadian clock. *Science* **293**, 880–883 (2001). A landmark paper that reports the first model for the plant circadian clock, following the observation of a reciprocal regulation between *CCA1/LHY* and *TOC1*.

112. Alabadí, D., Yanovsky, M. J., Más, P., Harmer, S. L. & Kay, S. A. Critical role for *CCA1* and *LHY* in maintaining circadian rhythmicity in *Arabidopsis*. *Curr. Biol.* **12**, 757–761 (2002).

113. Doyle, M. R. *et al.* The *ELF4* gene controls circadian rhythms and flowering time in *Arabidopsis thaliana*. *Nature* **419**, 74–77 (2002).

114. Mizoguchi, T. *et al.* *LHY* and *CCA1* are partially redundant genes required to maintain circadian rhythms in *Arabidopsis*. *Dev. Cell* **2**, 629–641 (2002).

115. Kikis, E. A., Khanna, R. & Quail, P. H. *ELF4* is a phytochrome-regulated component of a negative-feedback loop involving the central oscillator components *CCA1* and *LHY*. *Plant J.* **44**, 300–313 (2005).

116. Mizoguchi *et al.* Distinct roles of *GIGANTEA* in promoting flowering and regulating circadian rhythms in *Arabidopsis*. *Plant Cell* **17**, 2255–2270 (2005).

117. Hazen, S. P. *et al.* *LUX ARRHYTHMO* encodes a Myb domain protein essential for circadian rhythms. *Proc. Natl Acad. Sci. USA* **102**, 10387–10392 (2005).

118. Onai, K. & Ishiura, M. *PHYTOCLOCK 1* encoding a novel GARP protein essential for the *Arabidopsis* circadian clock. *Genes Cells* **10**, 963–972 (2005).

119. Farre, E. M., Harmer, S. L., Harmon, F. G., Yanovsky, M. J. & Kay, S. A. Overlapping and distinct roles of *PRR7* and *PRR9* in the *Arabidopsis* circadian clock. *Curr. Biol.* **15**, 47–54 (2005).

120. Nakamichi, N. *et al.* The *Arabidopsis* pseudo-response regulators, *PRR5* and *PRR7*, coordinately play essential roles for circadian clock function. *Plant Physiol.* **46**, 609–619 (2005).

121. Nakamichi, N., Kita, M., Ito, S., Yamashino, T. & Mizuno, T. PSEUDO-RESPONSE REGULATORS, *PRR9*, *PRR7* and *PRR5*, together play essential roles close to the circadian clock of *Arabidopsis thaliana*. *Plant Cell Physiol.* **46**, 686–698 (2005).

122. Liu, X.-L., Covington, M. F., Fankhauser, C., Chory, J. & Wagner, D. R. *ELF3* encodes a circadian clock-regulated nuclear protein that functions in an *Arabidopsis PHYB* signal transduction pathway. *Plant Cell* **13**, 1293–1304 (2001).

123. Covington, M. F. *et al.* *ELF3* modulates resetting of the circadian clock in *Arabidopsis*. *Plant Cell* **13**, 1305–1315 (2001).

124. Allen, T. *et al.* *Arabidopsis* *FHY3* specifically gates phytochrome signaling to the circadian clock. *Plant Cell* **18**, 2506–2516 (2006).

125. Viczián, A. *et al.* Functional characterization of phytochrome interacting factor 3 for the *Arabidopsis thaliana* circadian clockwork. *Plant Cell Physiol.* **46**, 1591–1602 (2005).

126. Somers, D. E., Schultz, T. F., Milnamow, M. & Kay, S. A. *ZEITLUPE* encodes a novel clock-associated PAS protein from *Arabidopsis*. *Cell* **101**, 319–329 (2000).

127. Jarillo, J. A. *et al.* An *Arabidopsis* circadian clock component interacts with both *CRY1* and *PHYB*. *Nature* **410**, 487–490 (2001).

128. Schultz, T. F., Kiyosue, T., Yanovsky, M., Wada, M. & Kay, S. A. A role for LKP2 in the circadian clock of *Arabidopsis*. *Plant Cell* **13**, 2659–2670 (2001).

129. Más, P., Kim, W.-Y., Somers, D. E. & Kay, S. A. Targeted degradation of *TOC1* by ZTL modulates circadian function in *Arabidopsis thaliana*. *Nature* **426**, 567–570 (2003).

130. Han, L., Mason, M., Risseeuw, E. P., Crosby, W. L. & Somers, D. E. Formation of an SCF<sup>ZTL</sup> complex is required for proper regulation of circadian timing. *Plant J.* **40**, 291–301 (2004).

131. Imaizumi, T., Tran, H. G., Swartz, T. E., Briggs, W. R. & Kay, S. A. *FKF1* is essential for photoperiodic-specific light signalling in *Arabidopsis*. *Nature* **426**, 302–306 (2003).

132. Millar, A. J. & Kay, S. A. Integration of circadian and phototransduction pathways in the network controlling *CAB* gene transcription in *Arabidopsis*. *Proc. Natl Acad. Sci. USA* **93**, 15491–15496 (1996).

133. Huq, E., Tepperman, J. M. & Quail, P. H. *GIGANTEA* is a nuclear protein involved in phytochrome signaling in *Arabidopsis*. *Proc. Natl Acad. Sci. USA* **97**, 9789–9794 (2000).

134. Michael, T. P., Salome, P. A. & McClung, C. R. Two *Arabidopsis* circadian oscillators can be distinguished by differential temperature sensitivity. *Proc. Natl Acad. Sci. USA* **100**, 6878–6883 (2003).

135. Salome, P. A. & McClung, C. R. *PSEUDO-RESPONSE REGULATOR 7* and *9* are partially redundant genes essential for the temperature responsiveness of the *Arabidopsis* circadian clock. *Plant Cell* **17**, 791–803 (2005).

136. Gould, P. D. *et al.* The molecular basis of temperature compensation in the *Arabidopsis* circadian clock. *Plant Cell* **18**, 1177–1187 (2006).

137. Halliday, K. J. & Whitelam, G. C. Changes in photoperiod or temperature alter the functional relationships between phytochromes and reveal roles for PHYD and PHYE. *Plant Physiol.* **131**, 1913–1920 (2003).

138. Mathews, S. & Sharrock, R. A. The phytochrome gene family in grasses (Poaceae): a phylogeny and evidence that grasses have a subset of the loci found in dicot angiosperms. *Mol. Biol. Evol.* **13**, 1141–1150 (1996).

139. Sharrock, R. A. & Clack, T. Patterns of expression and normalized levels of the five *Arabidopsis* phytochromes. *Plant Physiol.* **130**, 442–456 (2002).

140. Krall, L. & Reed, J. W. The histidine kinase-related domain participates in phytochrome B function but is dispensable. *Proc. Natl Acad. Sci. USA* **97**, 8169–8174 (2000).

141. Brudler, R. *et al.* Identification of a new cryptochrome class. Structure, function, and evolution. *Mol. Cell* **11**, 59–67 (2003).

142. Partch, C. L., Clarkson, M. W., Ozgur, S., Lee, A.-L. & Sancar, A. Role of structural plasticity in signal transduction by the cryptochrome blue-light photoreceptor. *Biochemistry* **44**, 3795–3805 (2005).

143. Yang, H.-Q. *et al.* The C termini of *Arabidopsis* cryptochromes mediate a constitutive light response. *Cell* **103**, 815–827 (2000).

144. Wang, H., Ma, L.-G., Li, J.-M., Zhao, H.-Y. & Deng, X.-W. Direct interaction of *Arabidopsis* cryptochromes with COP1 in light control development. *Science* **294**, 154–158 (2001).

145. Yang, H.-Q., Tang, R.-H. & Cashmore, A. R. The signaling mechanism of *Arabidopsis CRY1* involves direct interaction with COP1. *Plant Cell* **13**, 2573–2587 (2001).

146. Briggs, W. R. & Christie, J. M. Phototropins 1 and 2: versatile plant blue-light receptors. *Trends Plant Sci.* **7**, 204–210 (2002).

147. Koornneef, M., Bentsink, L. & Hilhorst, H. Seed dormancy and germination. *Curr. Opin. Plant Biol.* **5**, 33–36 (2002).

148. Takano, M. *et al.* Distinct and cooperative functions of phytochromes A, B, and C in the control of deetiolation and flowering in rice. *Plant Cell* **17**, 3311–3125 (2005).

149. Saijo, Y. *et al.* The COP1–SPA1 interaction defines a critical step in phytochrome A-mediated regulation of *HY5* activity. *Genes Dev.* **17**, 2642–2647 (2003).

Acknowledgements  
This area of research in the authors’ laboratory has been supported by a US National Institutes of Health grant to X.W.D.

Competing interests statement  
The authors declare no competing financial interests.

DATABASES  
The following terms in this article are linked online to:  
TAIR: http://www.arabidopsis.org  
CCA1 | COP1 | CRY1 | CRY2 | DET1 | FAR1 | FHY3 | GCN5 | HAF2 | HD1 | HFR1 | HY5 | LAF1 | LHY | SPA1 | PIF3 | PHOT1 | PHOT2 | PHYA | PHYB  

FURTHER INFORMATION  
Deng laboratory homepage: http://www.yale.edu/denglab  
Access to this links box is available online.
