
Communications

**Biosynthesis Mechanisms**

**A Type II Polyketide Synthase from the Gram-Negative Bacterium Stigmatella aurantiaca Is Involved in Aurachin Alkaloid Biosynthesis**

Axel Sandmann, Jeroen Dickschat, Holger Jenke-Kodama, Brigitte Kunze, Elke Dittmann, and Rolf Müller*

**Dedicated to Professor Thomas Hartmann on the occasion of his 70th birthday**

Over the last two decades, myxobacteria have emerged as a rich source of secondary metabolites (SMs) with potent biological activities.[1] These compounds often contain structural elements rarely found in SMs from other sources.[2] Interestingly, several SM-multiproducers are known among the myxobacteria, including *Stigmatella aurantiaca* Sga15, from which the aurachins,[3] myxochelins,[4] myxalamids, and stigmatellins[5,6] have been isolated. Whereas the molecular details of myxochelin, myxalamid, and stigmatellin biosyntheses have been described in detail,[7] no such information is available about the pathway to the aurachins.

The aurachins are rare quinoline alkaloids that also incorporate an isoprenoid side chain. Owing to their structural similarity to vitamin K, they act as potent inhibitors of electron transport in the respiratory chain.[2] The structurally related isoprenylated quinoline antibiotics are potent growth inhibitors of *Helicobacter pylori*,[8] while two members of the 4-hydroxy-2-alkylquinoline family (HAQs), 3,4-dihydroxy-2-heptylquinoline (PQS)[9] and 4-hydroxy-2-heptylquinoline, function as signaling molecules in cell–cell communication in *Pseudomonas aeruginosa*.[10]

HAQ biosynthesis has been shown to result from the “head-to-head” condensation of anthranilic acid (1) and β-keto fatty acids.[11] Feeding experiments with *S. aurantiaca* revealed that anthranilate—which is curiously formed by a plant-type 3-deoxy-D-arabinooheptulosonate-7-phosphate (DAHP) synthase instead of by the standard bacterial enzyme[12]—is also the starting unit of aurachin biosynthesis. Based on these experiments, 1 was speculated to be extended by acetoacetate, leading to the hypothetical early-pathway intermediate 4-hydroxy-2-methylquinoline (3; Scheme 1a).[13] An attractive alternative mechanism, however, is that intermediate 3 arises from the elongation of activated anthranilate by two malonyl-CoA extender units, catalyzed by a type III polyketide synthase (PKS). In support of this hypothesis, a plant typeIII PKS (acridone synthase) is responsible for formation of acridone (9), in which N-methylated anthranilate 8 is condensed with three units of malonyl-CoA (Scheme 1b).[14] TypeIII PKSs, also known as chalcone synthases (CHSs), were believed to be present only in plants but were recently discovered in a number of bacteria.[15,16]

As initial experiments based on degenerate PCR did not yield type III PKS products from genomic DNA of *S. aurantiaca*, we instead employed a mariner-based transposon mutagenesis strategy to identify the genes for aurachin biosynthesis. After screening by HPLC-MS of extracts from a 4096-member library of *S. aurantiaca* Sga15 mutants grown in microtiter plates (see the Supporting Information),[17–19] we identified one clone (4.59) that produced significantly less aurachin (Figure 1b) than the wild-type cells (Figure 1a). This reduced titer was confirmed by comparisons of extracts from 50-mL mutant cultures and from wild-type *S. aurantiaca* Sga15. The chromosomal recovery of the transposon from mutant 4.59 together with DNA sequences flanking the transposable element in the genome[17] led to the identification of the gene cluster for aurachin biosynthesis (see Table 1). Sequencing of the recovery plasmid (p4.59/NotI) revealed eleven open reading frames (ORFs) distributed across 11191 bp of the *S. aurantiaca* Sga15 chromosome, with an average guanine + cytosine content of 66.66%, which can be regarded as typical for this bacterial species (unpublished data; the gene cluster has been deposited in the EMBL database under accession number AM404078).

Sequence analysis identified the transposition site within *orf1*. However, the corresponding gene product shows similarity to hydrolases and therefore is not thought to be involved in the aurachin biosynthesis. Further sequence analysis revealed additional *orfs* downstream of *orf1*. These genes encode proteins with significant similarity to a ben-

[*] Dr. A. Sandmann, Dr. J. Dickschat,[+] Prof. Dr. R. Müller Pharmaceutical Biotechnology, Saarland University P.O. Box 151150, 66041 Saarbrücken (Germany) Fax: (+49) 681-302-5473 E-mail: rom@mx.uni-saarland.de Homepage: http://www.myxo.uni-saarland.de Dipl.-Biol. H. Jenke-Kodama, Prof. Dr. E. Dittmann Institute of Biology, Humboldt University Invalidenstrasse 43, 10115 Berlin (Germany) Dr. B. Kunze Helmholtz-Zentrum für Infektionsforschung GmbH Inhoffenstrasse 7, 38124 Braunschweig (Germany) [+] Current address: Department of Biochemistry, University of Cambridge 80 Tennis Court Road, Cambridge CB21GA (UK) [**] The authors thank Dr. K. Weissman and Dr. H. B. Bode for critical review of this manuscript. This work was supported by DFG Priority Programme 1152 “Evolution of Metabolic Diversity”. Supporting information for this article is available on the WWW under http://www.angewandte.org or from the author.

a)

+ coenzyme A

AuaE

+ 2 malonyl-CoA
- H₂O, - 3 CO₂
- 3 CoASH

AuaB, C, and D

+ farnesyl-PP

AuaA

b)

8

COSCoA

NH + 3 malonyl-CoA
- 2H₂O, - 3 CO₂,
- 4 CoASH

9

---

**Scheme 1.** a) Proposed biosynthetic pathway for the formation of aurachins D–A (4: aurachin D; 5: aurachin C; 6: aurachin B; 7: aurachin A). The biosynthetic steps that are catalyzed by the gene products (AuaA–AuaE) from the aurachin core gene cluster are shown. The intramolecular 1,2-migration of the farnesyl residue during the transformation of 5 to 6 is accompanied by the loss of the labeled carbonyl oxygen; this is compatible with a Wagner–Meerwein-type rearrangement (G. Höfle, personal communication and Ref.[13]). b) Biosynthesis of the plant metabolite acridone for comparison.

zoate-CoA ligases (AuaE), a set of proteins typical for type II PKS systems (AuaDCB), and a prenyltransferase (AuaA; see Table 1). Type II PKSs are multienzyme complexes consisting of a single set of proteins that are used iteratively. To date, they have been described exclusively from actinomycetes, where they are involved in the biosynthesis of polycyclic aromatic compounds such as actinorhodin.[20,21] The “minimal” polyketide synthase is common to all type II PKSs, and comprises an acyl carrier protein (ACP; AuaB), and a heterodimer consisting of a ketosynthase (KSα; AuaC) and a “chain-length factor” (CLF or KSβ; AuaD). The KSα subunit is responsible for the iterative condensation of malonyl-CoA building blocks, whereas KSβ is involved in decarboxylation of the starter unit and determination of chain length. AuaB and AuaC show only weak overall homology (see Table 1) to their respective enzyme families, but the conserved active site residues are present. In AuaD, which is homologous to known CLFs, a threonine residue replaces the active-site glutamine or glutamate (see Figure 3).[15,21]

In addition, a prenyltransferase (AuaA) is part of the gene cluster (Figure 2). AuaA shows similarities to the 4-hydroxybenzoate:polyprenyldiphosphate 3-polyprenyltransferase UbiA from *E. coli*,[22] which prenylates 4-hydroxybenzoate with farnesyl diphosphate. The enzyme AuaA is predicted by the program SOSUI to contain ten putative transmembrane helices (http://bp.nuap.nagoya-u.ac.jp/sosui). The benzoate-CoA ligase like protein AuaE shares strong similarities with EncN, which is part of the type II PKS involved in enterocin biosynthesis.[23,24] To prove the involvement of the *aua* gene cluster in aurachin biosynthesis, gene-disruption experiments were performed (see the Supporting Information).[25] As shown by HPLC-MS analysis, disruption of *auaA*, *auaD*, or *auaE* leads to abolition of aurachin production (Figure 1c), demonstrating the essential role of the *aua* operon. However, polar effects on downstream genes caused by the

---

**Figure 1.** HPLC-MS analysis of extracts of the wild-type *S. aurantiaca* Sga15 and *aua* knockout mutants; a) wild type; b) mutant 4.59; c) Sga-D_AS (the extracted-ion chromatograms of Sga-A_AS and Sga-E_AS are the same); d) Sga-D_AS after complementation with 3. The extracted-ion chromatograms of the aurachin derivatives from different culture extracts are shown. 4: [M+H]⁺ 364; 5: [M+H]⁺ 380; 6: [M+H]⁺ 380; 7: [M+H]⁺ 396.

Communications

Table 1: Proteins encoded in the sequenced region including the core gene cluster responsible for aurachin biosynthesis and their putative functions.

| Protein (gene) | Size (in Da, bp) | Proposed homologue, origin (identity [%], similarity [%]; Access no.) |
|----------------|------------------|---------------------------------------------------------------|
| AuaA (auaA)    | 36605, 981       | UbiA prenyltransferase family, Flavobacteria bacterium BBFL7 (28, 44; EAS 21036) |
| AuaB (auaB)    | 9572, 255        | acyl carrier protein, Enterococcus faecalis (34, 60; AAO 82791) |
| AuaC (auaC)    | 42842, 1221      | β-ketoacyl synthase, Desulfitobacterium hafniense (40, 59; YP_518890) |
| AuaD (auaD)    | 41891, 1242      | 3-oxoacyl-ACP synthase II, Streptomyces avermitilis (31, 45; NP_824120) |
| AuaE (auaE)    | 53329, 1494      | benzoate-CoA ligase family, Rhodopseudomonas palustris (37, 52; YP_568017) |
| ORF1 (orf1)    | 24354, 675       | carboxymethylenebutenolidase, Rhodopseudomonas palustris (45, 56; NP_946545) |
| ORF2 (orf2)    | 24958, 663       | hypothetical protein, Flavobacterium johnsoniae (41, 59; ZP_01247630) |
| ORF3 (orf3)    | 28683, 813       | putative secreted hydrolase, Streptomyces coelicolor (56, 70; CAC16457) |
| ORF4 (orf4)    | 20379, 543       | transcriptional regulatory protein, Bradyrhizobium japonicum (40, 62; BAC47519) |
| ORF5 (orf5)    | 45410, 1286      | protein of unknown function, delta proteobacterium MLMS-1 (28, 49; EAT 04615) |
| ORF6 (orf6)    | 47699, 1335      | protein of unknown function, delta proteobacterium MLMS-1 (28, 49; EAT 04615) |

![Diagram](attachment:diagram.png)

Figure 2. Top: Genetic organization of the gene cluster for aurachin biosynthesis. The location and transcriptional orientation of the genes are indicated by arrows. The flags show the sequence regions used for plasmid integration in the different knockout mutants generated in this work (see the Supporting Information), and the transposition site of magellan4 in mutant 4.59 is marked with a triangle. Bottom: The transcriptional overlap of *auaE* and *orf1*. The arrows indicate the direction of transcription of the aurachin operon and of the additional *orfs* located up- and downstream of the aurachin core cluster. Black arrows indicate the minimal PKS, gray arrows mark the remaining genes of the core cluster, and white arrows label the genes that encode proteins with an unknown function. The position of the Tn5-promoter in mutant 4.59 presumably activating transcription of the 3′ fragment of *orf1* is indicated.

D to aurachins C (5), B (6), and A (7) could not be identified, and they must therefore be located at alternative sites in the genome of *S. aurantiaca* Sga15. Although biosynthetic gene clusters with split organization are known,[17,26] this type of arrangement is still regarded as exceptional.[27]

In common with the aurachin type II PKS, several other bacterial aromatic PKSs use starter units other than acetyl-CoA and malonyl-CoA.[28] For example, the doxorubicin and daunorubicin PKSs are primed by propionyl-CoA,[29] whereas frenolicin biosynthesis starts with butyryl-CoA.[30] The enterocin type II PKS from *S. maritimus* uses as a starting moiety benzoyl-CoA, which is derived from benzoate in an EncN-and ATP-dependent reaction.[31] Prior to this work, the enterocin synthase was the only type II PKS known to employ an aromatic starter unit. In order to verify the suggested biosynthetic pathway to the aurachins, the hypothetical intermediate 3 was synthesized (see the Supporting Information) and administered to the cultures of the mutant strains Sga-D_AS (*auaD*−) and Sga-E_AS (*auaE*; see the Supporting Information). Production of metabolites 4–7 by the mutants was restored, as determined by HPLC-MS analysis (Figure 1d). In addition, GC-MS analysis of wild-type extracts derivatized with *N*-methyl-*N*-trimethylsilyltrifluoroacetamide (MSTFA) and comparison to the synthesized intermediate as a standard revealed small amounts of intermediate 3. Intermediate 3 was not identified in extracts of mutants Sga-D_AS and Sga-E_AS, while feeding of intermediate 3 failed to restore aurachin production in the mutant Sga-A_AS (*auaA*−, data not shown). Furthermore, 3 accumulated in the mutant Sga-A_AS, presumably because it could not be processed further to the aurachins (see the Supporting Information). Taken together, the results of these experiments are entirely consistent with the proposed order of transformations in aurachin biosynthesis.

To our knowledge, this is the first time a type II PKS has been identified from a Gram-negative bacterium. Based on the assignment of the gene cluster, a biosynthetic scheme for aurachins can be proposed (Scheme 1). Activation of 1 to anthraniloyl-CoA (2) by AuaE and transfer to the acyl carrier protein AuaB results in anthraniloyl-S-AuaE. The minimal PKS (consisting of primed AuaB plus AuaC and D) forms 3 by extension of 2 with two malonyl-CoA units, followed by release of the carboxylic acid along with decarboxylation of the second extender unit. Prenylation with farnesyl diphosphate catalyzed by AuaA would then yield aurachin D (4). Genes presumably involved in the transformation of aurachin

www.angewandte.org © 2007 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim Angew. Chem. Int. Ed. 2007, 46, 2712–2716

Unusually, the 3′-ends of *orf1* and *auaE* transcriptionally overlap (see Figure 2), which may explain the finding that mutant 4.59 generated very low amounts of 4, 5, and 6, while production of 7 was at essentially the level of the wild-type cells (nevertheless only representing a minor fraction of the total aurachins). In this mutant, transposition occurred in *orf1*, and presumably resulted in its induction (Figure 2). Owing to the overlap with *auaE*, this induction may have decreased the expression of the *aua* operon. As a result, formation of 4 likely decreased, but the expression of the additional enzymes proposed to catalyze the transformation from 4 to 7 would have been unaffected, leading to essentially complete turnover of the intermediates to 7.

To discern the evolutionary relationship between the aurachin PKS and several type II and type III PKSs (including acridone synthase), we analyzed a set of sequences from bacteria and plants representing all major type II and type III PKS systems identified to date (Figure 3).[16] We used archaeal FabH homologues as the outgroup and observed two major clades. The first contains bacterial FabH sequences and all chalcone synthase-like sequences from bacteria and plants, while the second comprises KSα and KSβ proteins of bacterial type II PKS. Addition of further bacterial type II PKS did not change the tree topology at all (data not shown). AuaC clearly belongs to the subclade that contains well-characterized KSα sequences from actinobacteria, although it is only distantly related and therefore forms a separate branch.

AuaD occupies an independent branch within the type II clade of the phylogenetic tree and therefore presumably represents a new sequence type of type II PKS CLF. Typically the CLF active site contains a highly conserved glutamine (Figure 3), which renders it an effective decarboxylase for chain initiation.[32,33] WhiE from *Streptomyces coelicolor* is the only known example in which the active-site glutamine is replaced by a glutamate residue. In AuaD, the active-site glutamine is substituted by a threonine, which may indicate that AuaD is inactive and therefore not required for aurachin biosynthesis. Disruption of AuaD may therefore only abolish aurachin biosynthesis as a result of polar effects (see above). On the other hand, the active-site glutamine may not be required for chain-length control and only serve to mediate protein/protein interactions between the two subunits.[34] If AuaD is really required, it could still serve as chain-length factor as previously discussed on the basis of the KSα/KSβ crystal structure.[34] Taken together, these observations suggest that the aurachin PKS belongs to a new group of PKSs that are undergoing an evolutionary transition from type II PKS into type III PKS and/or FABs or vice versa. Experiments are currently underway in our laboratory to evaluate this hypothesis and to verify the postulated biosynthetic mechanism.

---

Received: August 28, 2006  
Revised: November 13, 2006  
Published online: March 5, 2007  

Keywords: alkaloids · biosynthesis · myxobacteria · natural products · polyketides

---

**Figure 3.** a) Phylogenetic relationship of AuaC and AuaD with selected type II and type III PKSs. For all major nodes, the Bayesian support values are given above the branches. Branch lengths indicate the number of inferred amino acid changes per position. b) Active-site sequences of typical KSα and CLF proteins of type II PKSs relative to those of AuaC and AuaD. These alignments were generated using the sequences of frenolicin (Fren), enterocin (Enc), oxytetracycline (Otc), tetracenomycin (Tcm), *S. coelicolor* spore pigment (WhiE), and actinorhodin (Act) subunits.

| KSα subunits | KSβ subunits (CLFs) |
|--------------|---------------------|
| Fren_KS      | AVVSTGCTSGIDS       | Fren_CLF     | AVVVADQAGGLDA |
| Enc          | SVISAGCTSGIDA       | Enc          | GVVAEQAGGLDA |
| Oct          | GVVSAAGCTSGIDV      | Oct          | GALVAEQAGGLDA |
| Tcm          | TVVSTGCTSGLDA       | Tcm          | SVFVTEQAGGLDA |
| WhiE         | QTVSTGCTSGLDA       | whiE         | GVVAADEAGGLDA |
| ActI         | TMVSTGCTSGLDS       | Act          | SALVAEQAGGLDA |
| Aua          | LVIPTACAAGNSA       | Aua          | MTLSSRTAASLDA |

---

[1] K. Gerth, S. Pradella, O. Perlova, S. Beyer, R. Müller, *J. Biotechnol.* **2003**, *106*, 233.  
[2] H. Reichenbach, G. Höfle in *Drug Discovery from Nature* (Eds.: S. Grabley, R. Thiericke), Springer, Berlin, **1999**, p. 149.  
[3] B. Kunze, G. Höfle, H. Reichenbach, *J. Antibiot.* **1987**, *40*, 258.  
[4] B. Kunze, N. Bedorf, W. Kohl, G. Höfle, H. Reichenbach, *J. Antibiot.* **1989**, *42*, 14.

[5] K. Gerth, R. Jansen, G. Reifenstahl, G. Höfle, H. Irschik, B. Kunze, H. Reichenbach, G. Thierbach, *J. Antibiot.* **1983**, 36, 1150.

[6] B. Kunze, T. Kemmer, G. Höfle, H. Reichenbach, *J. Antibiot.* **1984**, 37, 454.

[7] H. B. Bode, R. Müller, *J. Ind. Microbiol. Biotechnol.* **2006**, 33, 577.

[8] K. A. Dekker, T. Inagaki, T. D. Gootz, L. H. Huang, Y. Kojima, W. E. Kohlbrenner, Y. Matsunaga, P. R. McGuirk, E. Nomura, T. Sakakibara, S. Sakemi, Y. Suzuki, Y. Yamauchi, N. Kojima, *J. Antibiot.* **1998**, 51, 145.

[9] E. C. Pesci, J. B. Milbank, J. P. Pearson, S. McKnight, A. S. Kende, E. P. Greenberg, B. H. Iglewski, *Proc. Natl. Acad. Sci. USA* **1999**, 96, 11229.

[10] E. Deziel, F. Lepine, S. Milot, J. He, M. N. Mindrinos, R. G. Tompkins, L. G. Rahme, *Proc. Natl. Acad. Sci. USA* **2004**, 101, 1339.

[11] F. Bredenbruch, M. Nimtz, V. Wray, M. Morr, R. Müller, S. Häussler, *J. Bacteriol.* **2005**, 187, 3630.

[12] B. Silakowski, B. Kunze, R. Müller, *Arch. Microbiol.* **2000**, 173, 403.

[13] G. Höfle, H. Reichenbach in *Sekundärmetabolismus bei Mikroorganismen* (Eds.: W. Kuhn, H. P. Fiedler), Attempo, Tübingen, **1995**, p. 61.

[14] K. T. Junghanns, R. E. Kneusel, A. Baumert, W. Maier, D. Gröger, U. Matern, *Plant Mol. Biol.* **1995**, 27, 681.

[15] M. B. Austin, J. P. Noel, *Nat. Prod. Rep.* **2003**, 20, 79.

[16] F. Gross, N. Luniak, O. Perlova, N. Gaitatzis, H. Jenke-Kodama, K. Gerth, D. Gottschalk, E. Dittmann, R. Müller, *Arch. Microbiol.* **2006**, 185, 28.

[17] A. Sandmann, F. Sasse, R. Müller, *Chem. Biol.* **2004**, 11, 1071.

[18] M. Kopp, H. Irschik, F. Gross, O. Perlova, A. Sandmann, K. Gerth, R. Müller, *J. Biotechnol.* **2004**, 107, 29.

[19] M. Kopp, H. Irschik, S. Pradella, R. Müller, *ChemBioChem* **2005**, 6, 1277.

[20] B. J. Rawlings, *Nat. Prod. Rep.* **1999**, 16, 425.

[21] J. Staunton, K. J. Weissman, *Nat. Prod. Rep.* **2001**, 18, 380.

[22] M. Melzer, L. Heide, *Biochim. Biophys. Acta* **1994**, 1212, 93.

[23] C. Hertweck, B. S. Moore, *Tetrahedron* **2000**, 56, 9115.

[24] J. Piel, C. Hertweck, P. R. Shipley, D. M. Hunt, M. S. Newman, B. S. Moore, *Chem. Biol.* **2000**, 7, 943.

[25] S. Beyer, B. Kunze, B. Silakowski, R. Müller, *Biochim. Biophys. Acta* **1999**, 1445, 185.

[26] T. W. Yu, L. Bai, D. Clade, D. Hoffmann, S. Toelzer, K. Q. Trinh, J. Xu, S. J. Moss, E. Leistner, H. G. Floss, *Proc. Natl. Acad. Sci. USA* **2002**, 99, 7968.

[27] S. C. Wenzel, R. Müller, *Curr. Opin. Chem. Biol.* **2005**, 9, 447.

[28] B. S. Moore, C. Hertweck, *Nat. Prod. Rep.* **2002**, 19, 70.

[29] C. R. Hutchinson, *Chem. Rev.* **1997**, 97, 2525.

[30] M. J. Bibb, D. H. Sherman, S. Omura, D. A. Hopwood, *Gene* **1994**, 142, 31.

[31] M. Izumikawa, Q. Cheng, B. S. Moore, *J. Am. Chem. Soc.* **2006**, 128, 1428.

[32] J. Dreier, C. Khosla, *Biochemistry* **2000**, 39, 2088.

[33] C. Bisang, P. F. Long, J. Cortés, J. Westcott, J. Crosby, A.-L. Matharu, R. J. Cox, T. J. Simpson, J. Staunton, P. F. Leadlay, *Nature* **1999**, 401, 502.

[34] A. T. Keatinge-Clay, D. A. Maltby, K. F. Medzihradszky, C. Khosla, R. M. Stroud, *Nat. Struct. Mol. Biol.* **2004**, 11, 888.
