
Current Molecular Medicine 2007, 7, 199-218

# Enzymes in the Conversion of Cholesterol into Bile Acids

Maria Norlin and Kjell Wikvall*

Division of Biochemistry, Department of Pharmaceutical Biosciences, University of Uppsala, Box 578, SE-751 23 Uppsala, Sweden

**Abstract:** This article aims to give an overview on the characterization, properties and regulation of enzymes, particularly the cytochrome (CYP) P450 enzymes, in the formation of bile acids from cholesterol. Bile acids are biologically active molecules that promote absorption of dietary lipids in the intestine and stimulate biliary excretion of cholesterol. Bile acids and oxysterols, formed from cholesterol, act as ligands to nuclear receptors regulating the expression of important genes in cholesterol homeostasis. Thus, the bioactivation of cholesterol into bile acids is crucial for regulation of cholesterol homeostasis. The primary human bile acids, cholic acid and chenodeoxycholic acid, are formed from cholesterol via several pathways involving many different enzymes. Many of these enzymes are cytochrome P450 (CYP) enzymes, introducing a hydroxyl group in the molecule. The “classic” pathway of bile acid formation starts with a 7α-hydroxylation of cholesterol by CYP7A1 in the liver. The “acidic” pathway starts with a hepatic or extrahepatic 27-hydroxylation by CYP27A1. There also exist some quantitatively minor pathways which may be of importance under certain conditions. Formation of cholic acid requires insertion of a 12α-hydroxyl group performed by CYP8B1. Oxysterols are precursors to bile acids, participate in cholesterol transport and are known to affect the expression of several genes in cholesterol homeostasis. Enzymes with capacity to form and metabolize oxysterols are present in liver and extrahepatic tissues. The enzymes, nuclear receptors and transcription factors involved in bile acid biosynthesis are potential pharmaceutical targets for the development of new drugs to control hypercholesterolemia and to prevent atherosclerosis and other diseases related to disturbed cholesterol homeostasis. The review will also discuss some inborn errors of bile acid biosynthesis and the recently acquired knowledge on the genetic defects underlying these diseases.

**Keywords:** Cholesterol, bile acids, oxysterols, CYP450 enzymes, pathways, tissue distribution, gene regulation, genetic disease.

## INTRODUCTION

Björkhem and Eggertsen [4], Russell [5], and Chiang [6].

Bile acids are formed from cholesterol in the liver and secreted via bile into the intestine where they promote absorption of dietary lipids, including fat-soluble vitamins. Bile acids also promote biliary excretion of unmetabolized cholesterol into bile. The enzymatic conversion of cholesterol into bile acids and the biliary excretion of cholesterol are the most important mechanisms for the ultimate removal of cholesterol from the body. Recent studies have revealed that bile acids are ligands to nuclear receptors regulating the expression of important genes in cholesterol homeostasis. Thus, the formation of bile acids is in several ways crucial for regulation of the cholesterol homeostasis. Considering that bile acids are involved in hepatic, biliary, and intestinal disease and also in diseases related to disturbances in cholesterol homeostasis, it is not surprising that bile acid research has been subject to increasing interest during recent years. During the last ten years reviews covering various aspects on bile acid research have been written by Princen [1], Hofmann [2], Vlahcevic et al. [3],

Because of the importance of cholesterol metabolism in the etiology of atherosclerosis, systems regulating cholesterol metabolism offer a number of potential pharmaceutical targets for the development of new drugs to control hypercholesterolemia and favorably impact the process of atherosclerosis. An increased knowledge of the many enzymes in bile acid biosynthesis should contribute to new possibilities for treatment of diseases related to cholesterol and bile acid metabolism.

This article aims to give an overview on the characterization, properties and regulation of enzymes, particularly the cytochrome (CYP) P450 enzymes, in the formation of bile acids from cholesterol. Enzymatic formation and metabolism of oxysterols (hydroxylated derivatives of cholesterol) are also included. In addition to being intermediates in bile acid biosynthesis, oxysterols are important regulators in lipid homeostasis and are involved in cholesterol transport. The review also describes some inborn errors of bile acid biosynthesis involving mutations of genes encoding bile acid biosynthetic enzymes.

Before discussing the enzymes in the conversion of cholesterol into bile acids, a general background on cholesterol homeostasis will be given.

**GENERAL BACKGROUND ON CHOLESTEROL HOMEOSTASIS**

Cholesterol is an essential structural component of all cellular and intracellular membranes and serves as a precursor for biosynthesis of a variety of products with specific biological activities such as bile acids and steroid hormones. Notwithstanding the essential role of cholesterol in multiple physiological processes, accumulation of this sterol is a risk factor for the development of disease. Excess cholesterol in the blood circulation may accumulate in macrophages and vascular cells and lead to atherosclerosis [7]. Increased amount of cholesterol in the bile may result in gallstones [8]. Disturbance of cholesterol homeostasis in the brain is reported to be associated with neurodegeneration [9,10]. The maintenance of adequate cholesterol levels in various tissues and cells requires the complex interaction of a number of physiological factors. The mechanisms for regulation of cholesterol homeostasis include effects on biosynthesis, uptake, esterification, and metabolism.

Cholesterol may be obtained in the diet or synthesized in the body from acetate [11]. Biosynthesis of cholesterol which involves approximately 30 enzymatic steps can take place in most cell types. Absorption of cholesterol from the intestine requires solubilization in micelles containing bile acids, phospholipids, fatty acids, and glycerides. The absorbed cholesterol is incorporated in chylomicrons, large lipoprotein particles which are secreted in the lymph vessels and enter the bloodstream via the thoracic duct. Increase of dietary cholesterol increases plasma cholesterol levels in most humans.

About 50-75% of the total cholesterol is synthesized endogenously [11]. A large part of the endogenously formed cholesterol is synthesized in the liver. The rate-limiting step is the conversion of 3-hydroxy-3-methylglutaryl coenzyme A (HMG-CoA) to mevalonic acid catalyzed by HMG-CoA reductase [1,11]. The activity of this enzyme is regulated by negative feed-back mechanisms. Compounds reported to down-regulate HMG-CoA reductase include mevalonic acid and oxysterols such as 25-hydroxycholesterol and 27-hydroxycholesterol [11,12]. Transcriptional down-regulation of HMG-CoA reductase by oxysterols occurs via sterol regulatory element binding proteins (SREBPs). Disturbed cholesterol biosynthesis during embryogenesis may lead to organ defects or in severe cases to intrauterine or neonatal death [13]. Cholesterol can be stored in the cells as cholesteryl esters after esterification by acyl coenzyme A:cholesterol acyltransferase (ACAT) [1,11].

About 7% of the body cholesterol circulates in plasma incorporated in different types of lipoproteins

[1,11]. The lipoproteins consist of a hydrophobic core of triglycerides and cholesteryl esters and a hydrophilic surface of phospholipids, unesterified cholesterol and apolipoproteins [14]. The dominating cholesterol carrier in fasting plasma is low-density lipoprotein (LDL).

Binding of LDL to specific receptors on the cell surface leads to cellular uptake of LDL cholesterol. The number of LDL receptors adjusts to cellular cholesterol demand [11]. The uptake of LDL cholesterol by the cell induces a negative feed-back signal which leads to down-regulation of the synthesis of LDL receptors and suppression of HMG-CoA reductase. Consequently the LDL receptors play an important role in cholesterol homeostasis. Excess cholesterol can be removed from extrahepatic cells by interaction with high-density lipoprotein (HDL) particles and transported as HDL to the liver where it is taken up and metabolized (reverse cholesterol transport) [15]. An additional mechanism for removal of cholesterol from extrahepatic tissues, involving conversion of cholesterol into 27-hydroxycholesterol, has been described [16]. 27-Hydroxycholesterol and other 27-oxygenated steroids are more polar than cholesterol and can be transported from the cells more efficiently than cholesterol. 27-Oxygenated steroids are taken up and metabolized by the liver.

The most cholesterol-rich organ in the body is the brain where cholesterol plays an important role as a component in myelin [9,10]. Myelin is essential for the propagation of nerve impulses in myelinated nerve fibers. Cholesterol homeostasis in the brain involves enzymatic conversion of cholesterol to 24-hydroxycholesterol which is more easily transported through the brain-blood barrier than cholesterol [10]. In this way, there is a flux of 24-hydroxycholesterol from the brain to the circulation and the oxysterol can ultimately be taken up and metabolized by the liver.

The quantitatively most important metabolic pathway in elimination of cholesterol from the body is formation of bile acids in the liver [1-6,11]. The liver also secretes some unmetabolized cholesterol directly into the bile. Reported mechanisms for regulation of bile acid biosynthesis include enzyme induction by oxysterols, feed-back inhibition by bile acids and effects of various hormones.

**PATHWAYS AND ENZYMES OF BILE ACID BIOSYNTHESIS**

The primary bile acids in most mammalian species are cholic acid and chenodeoxycholic acid [3] (Fig. (1)). Some species also produce other major bile acids, such as hyocholic acid in the pig and the muricholic acids in rats and mice. In hyocholic acid, the 12α-hydroxyl group present in cholic acid is replaced by a 6α-hydroxyl group. In α- and β-muricholic acids, the 12α-hydroxyl group is replaced by a 6β-hydroxyl group [1,17,18]. All primary bile acids contain a 7α-hydroxyl group.

Enzymes in the Conversion of Cholesterol

Current Molecular Medicine, 2007, Vol. 7, No. 2 201

![Diagram](attachment:enzymes_in_conversion_of_cholesterol.png)

Fig. (1). Major pathways in bile acid formation: the neutral (classic) and acidic (alternative) pathways.
3β-HSD, 3β-hydroxy-Δ⁵-C₂₇-steroid dehydrogenase; 3α-HSD, 3α-hydroxysteroid dehydrogenase; 5β-reductase, 3-oxo-Δ⁴-steroid 5β-reductase.

202 Current Molecular Medicine, 2007, Vol. 7, No. 2

The primary bile acids formed in the liver are conjugated with glycine and taurine before secretion into the bile which empties into the upper portion of the small intestine. In the intestine, primary bile acids are converted to secondary bile acids, e.g. deoxycholic acid and lithocholic acid, by microbial enzymes. Most of the bile acid pool is reabsorbed and returned to the liver via the portal vein to be used over again [1,11]. Biosynthesis of bile acids from cholesterol in the liver involves a variety of different enzymes in endoplasmic reticulum, mitochondria, cytosol and peroxisomes [1,3,5].

According to current concepts there are two main pathways, the "classic or neutral" and the "alternative or acidic" pathway [19] as shown in Fig. (1). The classic pathway has come to be known as the neutral pathway of bile acid biosynthesis because bile acid precursors proceed through most of the pathway as neutral steroids. The alternative pathway is called the acidic pathway because bile acid precursors proceed through the cascade of intermediates as carboxylic acids. Quantitatively minor pathways involving 24-hydroxylation of cholesterol ("24-hydroxylase pathway") or 25-hydroxylation of bile acid intermediates ("25-hydroxylase pathway") will be briefly discussed.

Neutral or Classic Pathway

The neutral pathway starts with 7α-hydroxylation of cholesterol by cholesterol 7α-hydroxylase (CYP7A1), a microsomal cytochrome P450 enzyme (Fig. (1)). The 7α-hydroxylation of cholesterol is rate-limiting in the neutral pathway. CYP7A1 will be discussed in detail later in this review.

The formed 7α-hydroxycholesterol is then converted to 7α-hydroxy-4-cholesten-3-one by an NAD-dependent microsomal 3β-hydroxy-Δ⁵-C₂₇ steroid dehydrogenase/isomerase [20-23] (Fig. (1)). The purification of 3β-hydroxy-Δ⁵-C₂₇-steroid dehydrogenase from rabbit and pig liver revealed an absolute preference for C₂₇-sterol substrates [20,21], implying that this enzyme is different from 3β-hydroxy-Δ⁵-steroid dehydrogenases acting on C₁₉- and C₂₁-steroids in steroid hormone biosynthesis. cDNA encoding the human 3β-hydroxy-Δ⁵-C₂₇-steroid dehydrogenase was isolated by Schwarz et al. [22]. When transfected into cultured cells, the 3β-hydroxy-Δ⁵-C₂₇-steroid dehydrogenase cDNA encoded an enzyme with essentially the same substrate specificity as the purified porcine enzyme [21]. The predicted enzyme product shares 34% identity with the 3β-hydroxy-Δ⁵-steroid dehydrogenases active on C₁₉ and C₂₁ steroids in steroid hormone biosynthesis [22]. The human 3β-hydroxy-Δ⁵-C₂₇-steroid dehydrogenase shares approximately 85% identity with the mouse [23] and porcine microsomal 3β-hydroxy-Δ⁵-C₂₇-steroid dehydrogenase sequences [24]. The molecular genetics behind the rare inborn error 3β-hydroxy-Δ⁵-C₂₇-steroid dehydrogenase deficiency will be described later in this review.

For the formation of cholic acid, a 12α-hydroxyl group must be introduced in the steroid nucleus producing 7α,12α-dihydroxy-4-cholesten-3-one (Fig. (1)). 12α-Hydroxylation is catalyzed by the microsomal sterol 12α-hydroxylase CYP8B1. CYP8B1 will be discussed in detail later in this review.

The further conversion of the 7α-hydroxy-4-cholesten-3-one and 7α,12α-dihydroxy-4-cholesten-3-one intermediates into chenodeoxycholic and cholic acids requires two reduction steps in the liver cytosol catalyzed by 3-oxo-Δ⁴-steroid 5β-reductase [25] and 3α-hydroxysteroid dehydrogenase [26] (Fig. (1)). These enzymes are members of a large family of enzymes, the aldo-keto reductases (reviews, see Russell [5] and Penning [27]).

Formation of bile acids requires side chain cleavage which is initiated by oxidation of the side chain. An essential enzyme required for initiation of side chain oxidation is sterol 27-hydroxylase (CYP27A1), a mitochondrial cytochrome P450 which introduces a hydroxyl group in the 27-position. Substrates for sterol 27-hydroxylase are for instance the intermediates 5β-cholestane-3α,7α-diol and 5β-cholestane-3α,7α,12α-triol in the neutral pathway and cholesterol in the acidic pathway (Fig. (1)). CYP27A1 will be discussed in detail later in this review.

The 27-hydroxylated sterols are subsequently oxidized into the corresponding C₂₇ acids. Studies with purified CYP27A1 [28-30] and with cultured cells transfected with CYP27A1 cDNA [31] are consistent with one enzyme, i.e. CYP27A1, performing the complete oxidation of C₂₇-alcohol into C₂₇ acid. Also, there are other enzyme systems able to catalyze the oxidation to C₂₇ acids, including an NAD-dependent dehydrogenase system in mitochondria [32] and alcohol- and aldehyde dehydrogenases in cytosol [33].

For the final steps of side-chain cleavage, the C₂₇ acid precursors of cholic acid and chenodeoxycholic acid, 3α,7α,12α-trihydroxy-5β-cholestanoic acid and 3α,7α-dihydroxy-5β-cholestanoic acid, respectively, are converted into coenzyme A-thioesters [5,34]. The coenzyme A-activation can be catalyzed by either the peroxisomal and microsomal very long chain acyl-CoA synthetase or the microsomal bile acid-CoA synthetase in humans. The cholestanyl-CoA thioesters are subsequently chain-shortened by one cycle of β-oxidation in the peroxisomes, after which the two coenzyme A-thioesters formed are conjugated with taurine or glycine, catalyzed by bile acid:CoA-amino acid N-acyltransferase [5,34].

Acidic or Alternative Pathway

A pathway for bile acid biosynthesis in rats not involving 7α-hydroxylation of cholesterol was suggested as early as 1967 by Mitropoulos and Myant [35], who obtained a small conversion of labeled cholesterol into 3β-hydroxy-5-cholenoic acid,

lithocholic acid and chenodeoxycholic acid in rat liver
mitochondria. Administration of 27-hydroxycho-
lesterol to rats, hamsters, rabbits and humans results
in the formation of 7α-hydroxylated products [36-38].
Sterol 27-hydroxylase catalyzes the 27-hydroxylation
of cholesterol, the first step in the acidic pathway
[39] (Fig. (1)). The 27-hydroxylation has been
suggested to be rate-limiting for this pathway, at
least in the rat [3]. The formed 27-hydroxycholesterol
may undergo oxidation to 3β-hydroxy-5-cholestenoic
acid [30,41]. In the acidic pathway

hydroxycholesterol and 3β-hydroxy-5-cholestenoic
acid are further 7α-hydroxylated by a microsomal
enzyme referred to as oxysterol 7α-hydroxylase or
CYP7B1 (Fig. (1)). This enzyme does, however, not
7α-hydroxylate cholesterol [40]. In addition, CYP7A1
is able to catalyze 27-hydroxycholesterol 7α-
hydroxylation. CYP7A1 and CYP7B1 will be
discussed in detail later in this review. Mitochondrial
7α-hydroxylase activity also has been reported. The
dehydrogenase/isomerase active towards 7α-
hydroxycholesterol in the neutral pathway oxidizes


A
CHOLESTEROL $\stackrel{\text { CYP46A1 }}{\longrightarrow}$ 24-hydroxycholesterol
$\downarrow$
$\begin{array}{c}
\text { CYP7A1 } \\
\text { CYP39A1 }
\end{array}$
7α,24-dihydroxycholesterol
$\downarrow$
$3 \beta-\mathrm{HSD}$
7α,24-dihydroxy-4-cholesten-3-one
$\downarrow$
$\downarrow$
Enzymes?
$\downarrow$
CHENODEOXYCHOLIC ACID


B
CHOLESTEROL
$\downarrow$
$\downarrow$
neutral
pathway
$\begin{array}{ll}
\text { 5β-cholestane-3α,7α,12α-triol } & \text { 25-hydroxylase } \\
\text { CYP27A1 } & \text { pathway }
\end{array}$
$\begin{array}{ll}
\text { CYP3A4 } & \text { (CYP3A5) }
\end{array}$
5β-cholestane-
3α,7α,12α,27-tetrol
$\begin{array}{ll}
\text { 5β-cholestane- } & \text { Enzyme? }
\end{array}$
3α,7α,12α,25-tetrol
$\begin{array}{ll}
\text { 5β-cholestane- } & 
\end{array}$
3α,7α,12α,24,25-pentol
$\downarrow$
CHOLIC ACID


Fig. (2). Minor pathways in bile acid formation.
A, 24-Hydroxylase pathway, starting with a 24S-hydroxylation of cholesterol in the brain.
B, In the 25-hydroxylase pathway, side-chain oxidation in cholic acid biosynthesis starts with a CYP3A4-mediated
microsomal 25-hydroxylation. For comparison, the CYP27A1-mediated 27-hydroxylation in the neutral pathway is also
shown.

204 Current Molecular Medicine, 2007, Vol. 7, No. 2

also 7α,27-dihydroxycholesterol into 7α,27- dihydroxy-4-cholestene-3-one [21,22] (Fig. (1)).

Studies on the substrate specificity of purified pig liver microsomal 3β-hydroxy-Δ5-C27-steroid dehydrogenase showed that one single enzyme can perform the oxidation/isomerization reaction on all potential intermediates in major and alternative pathways of bile acid biosynthesis [21]. The fact that only bile acids with 3β-hydroxy-Δ5 configuration are found in children with familial intrahepatic cholestasis due to 3β-hydroxy-Δ5-C27-steroid dehydrogenase deficiency, supports the contention that only one enzyme is involved in the oxidation of the 3β-hydroxy group [23]. The major product of the acidic pathway is considered to be chenodeoxycholic acid [3,19].

CYP3A11 carries out the 25-hydroxylation in cholic acid biosynthesis and is responsible for the formation of various other side-chain hydroxylated metabolites in the mouse. The other enzymes in the 25-hydroxylation pathway in different species have not to date been characterized.

### Other Pathways

Recently, Norlin et al. [48] showed that cholestaneone, a cholestanol precursor, is a substrate for CYP27A1. The results suggest the possible existence of an additional minor pathway for bile acid biosynthesis involving 27-hydroxylation of 4-cholestene-3-one (cholestaneone) by CYP27A1. The 27-hydroxycholestaneone formed may be 7α-hydroxylated by CYP7A1, and the resulting 7α,27-dihydroxy-4-cholestene-3-one may be further converted into bile acids [48].

Cholestanol is known to be degraded to 5α-bile acids, allocholic and allochenodeoxycholic acid, by liver enzymes similar to those involved in biosynthesis of 5β-bile acids. Allo-bile acids are minor constituents in mammalian bile, ranging from 1% or less in human to more than 5% in rabbits [49].

### Contributions of the Different Pathways for Bile Acid Formation

The relative contribution of various pathways for bile acid formation is not fully assessed. This may vary between different species, with age as well as with various physiological and pathological conditions. The neutral pathway is considered the quantitatively most important pathway for formation of primary bile acids in healthy humans. In rats, the contribution of the acidic pathway seems to be as much as 50% of total bile acid biosynthesis [3]. During normal conditions, the acidic pathway of bile acid synthesis is considered quantitatively less important than the neutral pathway in humans [4]. From results of a kinetic study with human subjects, it was calculated that the endogenous production of 27-hydroxycholesterol amounted to approximately 9% of the total formation of bile acids [50]. However, the acidic pathway may be critical during certain conditions. It appears that the acidic pathway can compensate and maintain bile acid formation under conditions where the neutral pathway is repressed [19,51]. A major function of the acidic pathway, starting with an extrahepatic 27-hydroxylation, could be to provide an important means for removal of cholesterol from extrahepatic tissues e.g. vascular endothelium and macrophages. Also, some data indicate that the acidic pathway may play an important role for human infants [52] (see below).

It has been estimated that about 5% of total bile acids are formed by the 25-hydroxylase pathway in healthy humans [53]. However, when the 27-hydroxylase pathway for side chain oxidation is blocked, e.g. in patients with the inherited lipid storage disease cerebrotendinous xanthomatosis

### 24-Hydroxylase Pathway

Cholesterol is 24S-hydroxylated in the brain by the cholesterol 24-hydroxylase, CYP46A1. The formation of 24S-hydroxycholesterol represents an important mechanism by which cholesterol is secreted from the brain [4,10]. 24-Hydroxycholesterol reaching the liver by the circulation may then be converted into bile acids (Fig. (2A)). Some of the enzymatic steps have been characterized in studies with purified and/or recombinant enzymes. 24-Hydroxycholesterol can be converted to 7α,24-dihydroxycholesterol by CYP7A1 in liver [42] or by another hepatic 7α-hydroxylase, CYP39A1 [43]. The relative importance of CYP7A1 and CYP39A1 in the metabolism of 24-hydroxycholesterol is not known. 7α,24-Dihydroxycholesterol is a substrate for the 3β-hydroxy-Δ5-C27-steroid dehydrogenase [22] resulting in the formation of 7α,24-dihydroxy-4-cholestene-3-one. This intermediate may then be converted into bile acids, at least chenodeoxycholic acid, by enzymes participating in the major pathways [44]. However, the enzymes involved in the final steps of the 24-hydroxylase pathway have not been identified.

### 25-Hydroxylase Pathway

During certain conditions side-chain degradation in cholic acid biosynthesis may start with a microsomal 25-hydroxylation of 5β-cholestane-3α,7α,12α-triol (25-hydroxylase pathway) (Fig. (2B)). In this pathway, the 25-hydroxylation is followed by microsomal 24-hydroxylation to yield 5β-cholestane-3α,7α,12α,24,25-pentol which is converted into cholic acid by soluble enzymes [45]. The 25-hydroxylation of 5β-cholestane-3α,7α,12α-triol, the intermediate in cholic acid formation, was shown to be catalyzed by CYP3A4 in human liver [46]. CYP3A4 also catalyzes the formation of small amounts of 23- and 24-hydroxylated products. Recombinant CYP3A4 does not 25-hydroxylate 5β-cholestane-3α,7α-diol, the corresponding intermediate in chenodeoxycholic acid formation. This finding is in accordance with the concept that the 25-hydroxylase pathway is particularly important for cholic acid biosynthesis [45,46]. Honda et al. [47] reported that murine

(CTX), the 25-hydroxylase pathway appears to be
the only possibility to form bile acids. Patients with
CTX also excrete gram amounts of 5β-cholestane-
3α, 7α, 12α, 25-tetrol [54].

The recently described 24-hydroxylase pathway
and the pathway involving 27-hydroxylation
of cholestenone are probably responsible for less than
1% of total formation of bile acids [4].

CYTOCHROME (CYP) P450 ENZYMES IN
BILE ACID BIOSYNTHESIS

CYP7A1 (Cholesterol 7α-Hydroxylase)

The first and rate-limiting step in the neutral or
classic pathway is 7α-hydroxylation of cholesterol,
catalyzed by the microsomal cholesterol 7α-
hydroxylase, CYP7A1 (Fig. (1)). Early studies
indicated that the hepatic cholesterol 7α-
hydroxylating enzyme is a specific microsomal
cytochrome P450 that is induced by biliary drainage
or cholestyramine and inhibited by bile acids [55].
Cholesterol 7α-hydroxylase has been purified from
rat liver [56, 59] and pig liver [59]. cDNA encoding for
CYP7A1 was first isolated from rat in 1989-1990 [60-
62] and subsequently from human [63-65], hamster
[66] and mouse [67]. Studies on mice with a
disruption in the Cyp7a1 gene [68, 69] demonstrated
a crucial role for this enzyme in bile acid
biosynthesis. Most Cyp7a1-deficient animals died
within the first 18 days of life, if not treated with
vitamins and cholic acid, due to liver failure and
deficiencies of fat-soluble vitamins. It is notable that
mice and humans with cholesterol 7α-hydroxylase
deficiency exhibit different phenotypes [70].
Cholesterol 7α-hydroxylase deficiency in humans
with mutations in the CYP7A1 gene will be described
later in this review.

Although CYP7A1 is mainly known for its
important role in cholesterol 7α-hydroxylation, this
enzyme is able to catalyze 7α-hydroxylation also of
27-hydroxycholesterol and other oxysterols, as
shown by Norlin et al. [42, 71]. Results supporting a
role for CYP7A1 as an oxysterol 7α-hydroxylase also
are reported by Dueland et al. [74] and Pandak et
al. [75]. In addition to data on recombinant
expression of human and rat CYP7A1 cDNA, Norlin
et al. found that oxysterol 7α-hydroxylase activity in
rat liver was significantly increased by treatment with
cholestyramine, an inducer of CYP7A1, supporting a
physiological role for CYP7A1 in oxysterol 7α-
hydroxylation [71]. The relative importance of
CYP7A1 and CYP7B1 in hepatic 27-
hydroxycholesterol 7α-hydroxylation is not known.
The Vmax/Km values for 7α-hydroxylation of
cholesterol and 27-hydroxycholesterol by
recombinant CYP7A1 are of a similar order of
magnitude [71]. However, the very high amount of
cholesterol in liver cells may compete with 27-
hydroxycholesterol for the active site of CYP7A1 and
therefore substantially reduce CYP7A1-mediated 27-
hydroxycholesterol 7α-hydroxylation in vivo. On the

other hand a similar situation may occur for CYP7B1-
mediated 27-hydroxycholesterol 7α-hydroxylation
since there are several CYP7B1-substrates present
in liver, for instance dehydroepiandrosterone
(DHEA), that may compete with 27-hydroxycho-
lesterol for the active site of CYP7B1 [40, 72, 73]. For
extrahepatic 27-hydroxycholesterol 7α-hydroxylation,
however, CYP7B1 should be the most important
enzyme, since it is expressed in most tissues of the
body [6].

Regulation

The main target for regulation of bile acid
biosynthesis is CYP7A1. Several mechanisms for
regulation of CYP7A1 have been described. The
regulation of CYP7A1 will be summarized in this
review. The reader is referred to excellent reviews by
Chiang for detailed discussion on regulatory
mechanisms [6, 76-78].

An overview of reported regulators of CYP7A1 is
given in Fig. (3). Much experimental evidence
suggests that bile acids inhibit bile acid biosynthesis
by inhibiting CYP7A1 gene transcription via negative
feedback control. Bile acids bind as ligands to the
nuclear receptor FXR (farnesyl X receptor) which
forms a heterodimer with the retinoid X receptor
(RXR). Activated FXR does not bind to the bile acid
responsive elements (BARE) in the CYP7A1 gene.
The suppression of the CYP7A1 promoter activity
through the activated FXR-RXR complex is
apparently mediated by an indirect mechanism
involving interaction with other transcription proteins,
including SHP-1 (small heterodimer partner) and FTF
(human α-fetoprotein transcription factor) also called
LRH-1. FTF binds to BARE-II in the CYP7A1
promoter and stimulate the hepatic expression of
CYP7A1. SHP-1 represses several target genes
such as that encoding FTF. It appears that bile acid-
activated FXR induces SHP, which subsequently
interacts with FTF and inhibits its trans-activation of
the CYP7A1 gene [6, 76-78].

Experiments with SHP null mice have indicated
the existence of SHP-independent mechanisms for
suppression of the CYP7A1 gene by bile acids
[6, 79]. It has been suggested that bile acids induce
inflammatory cytokines (tumor necrosis factor α and
IL-1β) and stimulate mitogen-activated protein kinase
(MAPK) signaling pathways leading to the activation
of cJun N-terminus kinase (JNK). JNK may
phosphorylate and inactivate transcription factors
such as HNF-4α, which is a crucial factor stimulating
the hepatic expression of CYP7A1 [6, 80, 81]. In
addition, the pregnane X receptor (PXR) and the
vitamin D receptor (VDR) have been identified as bile
acid-activated receptors [82-84]. PXR and VDR may
suppress CYP7A1 by SHP-independent mechanisms
[76, 78, 85].

Cholestyramine, a drug used in the treatment of
hyperlipoproteinaemia, increases the levels of
cholesterol 7α-hydroxylase by binding to bile acids in
the intestine and preventing their reabsorption to the
liver [86].

206 Current Molecular Medicine, 2007, Vol. 7, No. 2

Norlin and Wikvall

![Diagram](attachment:diagram.png)

Fig. (3). An overview of reported regulators of CYP7A1.

+, upregulation; -, downregulation. See text for details, abbreviations and reference. FXR, PXR, LXRα, VDR and PPARα heterodimerize with RXR.

Another mechanism proposed for regulation of cholesterol 7α-hydroxylase involves the nuclear receptor LXRα (liver X receptor alpha) [87]. This receptor has been shown to mediate induction of CYP7A1 gene expression by oxysterols in rodents. A critical role for LXRα in induction of cholesterol 7α-hydroxylase in the mouse is supported by results obtained with mice with a disruption in the LXRα gene [88]. LXRα-deficient mice were defective in cholesterol 7α-hydroxylase induction mechanisms and accumulated very large amounts of cholesterol in the liver when fed a cholesterol-rich diet. LXRα is activated by oxysterols, metabolites of cholesterol, and enhances clearance of cholesterol by increasing bile acid synthesis and excretion. Ligand-bound LXRα binds and induces transcription of the CYP7A1 gene in a heterodimer complex with RXR. Compared with rats and mice, LXRα has much less effect on the hamster and human CYP7A1 genes, which lack the LXRα-binding site found in the mouse and rat genes [89, 90]. This explains why an increase of dietary cholesterol is reported to increase transcription of cholesterol 7α-hydroxylase in some species but decrease the transcription in others [3].

In one study, peroxisome proliferator-activated receptor α (PPARα) was shown to inhibit CYP7A1 gene transcription [91]. In another study, however, it was suggested that PPARα may not be a major influence on CYP7A1 activity under normal physiological conditions [92].

Cholesterol 7α-hydroxylase levels are also influenced by hormones [3, 93, 94]. Thyroid hormone and glucocorticoids are reported to stimulate cholesterol 7α-hydroxylase expression. Glucocorticoids stimulate the promoter of the rat CYP7A1 gene whereas insulin inhibits promoter activity. Glucagon was recently shown to inhibit human CYP7A1 in human hepatocytes [94].

Evidence for a posttranscriptional regulation of cholesterol 7α-hydroxylase has been reported, but most of the data available suggest that the regulation is predominantly on a transcriptional level [3, 6, 95-97].

**Tissue Distribution and Expression During Development**

CYP7A1 is considered a liver-specific enzyme. However, one study reports expression of CYP7A1 in human prostate [98]. The expression of hepatic CYP7A1 in several species has been reported to be low in early life and to increase severalfold with age [24, 99].

**CYP7B1 (Oxysterol 7α-Hydroxylase)**

In the search for a 7α-hydroxylating enzyme participating in the acidic pathway (Fig. (1)), Toll et al. [100] described in 1992 the substrate specificity of a pig liver microsomal oxysterol 7α-hydroxylase that is active on 27-oxygenated sterols. An enzyme

Enzymes in the Conversion of Cholesterol

Current Molecular Medicine, 2007, Vol. 7, No. 2 207

preparation catalyzing 7α-hydroxylation of 27-
oxygenated sterols could also be isolated free from
cholesterol 7α-hydroxylase activity and it was
concluded that this oxysterol 7α-hydroxylase was
different from CYP7A1 [100]. Subsequently, several
groups reported the 7α-hydroxylation of 27-
hydroxycholesterol by liver microsomes from humans
[101,102] and hamster [103]. The oxysterol 7α-
hydroxylase enzyme attracted even more interest
when experiments with Cyp7α-deficient mice showed
that 90% of the mice with a disruption in the Cyp7a
gene died within the first three weeks of life, but in
those that survived the pathological
symptoms
resolved, coinciding with an upregulation of oxysterol
7α-hydroxylase activity [68,69].

cDNA encoding CYP7B1 was first isolated from
rat hippocampus in 1995 [104] and subsequently
from mouse [72] and human liver [52]. The native
oxysterol 7α-hydroxylase enzyme has been purified
from pig liver and characterized [40].

CYP7B1 is structurally similar to CYP7A1 but has
a different and broader substrate specificity. The
purified hepatic enzyme and the heterologously
expressed enzyme [72,73] catalyze 7α-hydroxylation
of the oxysterols 25-hydroxycholesterol and 27-
hydroxycholesterol but not of cholesterol and show
very low activity towards 24-hydroxycholesterol
[40,72]. In addition to 25- and 27-hydroxycho-
lesterol,
dehydroepiandrosterone (DHEA),
pregnenolone and 5α-androstane-3β,17β-diol are
7α-hydroxylated by CYP7B1 [40,73,105,106].

The metabolic activity of this enzyme may be
important for controlling the levels of oxysterols,
which are potent regulators of lipid metabolism [107].

Oxysterols have many cellular effects including
down-regulating the synthesis of cholesterol
and
fatty acids, increasing bile acid synthesis
by
induction of cholesterol 7α-hydroxylase and affecting
cholesterol transport and elimination [11,12,77,87].
High amounts of cellular oxysterol may result in
cytotoxicity. Li-Hawkins et al. [106] constructed mice
without CYP7B1 and reported that plasma and
tissue levels of 25- and 27-hydroxycholesterol were
markedly elevated in Cyp7b1(-/-) knockout animals.
They concluded that a major physiological role of the
CYP7B1 oxysterol 7α-hydroxylase, at least in the
mouse, is to metabolize 25- and 27-hydroxycho-
lesterol and that loss of this enzyme in the liver is
compensated for by increases in the synthesis of bile
acids by other pathways. CYP7B1 has attracted
considerable interest in recent years since it also
metabolizes certain neurosteroids and is reported to
play a role in endocrine function [73,108].

In humans, the absence of CYP7B1 appears to
have serious consequences. Setchell et al. [52]
described a newborn child with severe neonatal
cholestasis with a mutation in the CYP7B1 gene.
The finding that lack of CYP7B1 proved to be lethal,
indicates that this enzyme may be important in early
human life. CYP7B1 deficiency caused by mutation
of the CYP7B1 gene will be discussed later in this
review.

Regulation

An overview of reported regulators of CYP7B1 is
given in Fig. (4). The 5′ flanking region of the human
CYP7B1 gene was isolated in 1999 by Wu et al.
[109]. A role for the general transcription activator
Sp1 in the basal transcription of human CYP7B1 has


![Diagram](attachment:diagram.png)

Fig. (4). An overview of reported regulators of CYP7B1.
+, upregulation; -, downregulation; See text for details, abbreviations and reference. Squalenstatin, a HMG-CoA reductase
inhibitor.

208 Current Molecular Medicine, 2007, Vol. 7, No. 2

been suggested [110]. Norlin and Chiang [111] reported suppression of human CYP7B1 transcription by sterol response element binding protein (SREBP), revealing a link between oxysterol-sensitive regulators and oxysterol metabolism. Recently, it was reported that estrogens and androgens can regulate human CYP7B1 levels [112, 113]. This regulatory mechanism may be related to the endocrine functions of CYP7B1. Also, estradiol has been reported to affect certain aspects of cholesterol homeostasis, including bile acid biosynthetic pathways and bile acid transport [114, 115].

Although studies in rats and mice have indicated some effect by bile acids on rodent CYP7B1 expression [75, 116], regulation of the human gene by bile acids appears modest, with suppression only by chenodeoxycholic acid (M. Norlin, E. Owsley, J. Y. L. Chiang, unpublished observations). The 5′ flanking region of the rat CYP7B1 gene has not been investigated and the molecular basis for the inhibitory effect of bile acids has not been characterized. Pandak et al. [116] reported that CYP7B1 levels in rat hepatocytes are affected by cholesterol and squalenol, a cholesterol synthesis inhibitor. Both CYP7B1 and CYP7A1 in the rat were found to undergo diurnal variation, with peak and trough values for CYP7B1 lagging approximately 6 hours behind CYP7A1 [116]. Thyroid hormones, glucocorticoids and cytokines are also reported to affect CYP7B1 levels [116-118].

Tissue Distribution and Expression During Development

CYP7B1 is widely expressed in many human tissues. In addition to liver, CYP7B1 is expressed for instance in brain, kidney, lung, heart, pancreas and prostate [72, 73, 109, 112]. It has been suggested that 7α-hydroxylation of oxysterols could play a role in modulating their effects on cholesterol homeostasis [119, 120]. The role of CYP7B1 in extrahepatic tissues remains unclear. The substrate specificity and tissue distribution pattern indicate that CYP7B1 may have multiple functions in control of steroid levels [73, 106, 108].

The available data on developmental expression of CYP7B1 appear contradictory in some ways. The reasons for the observed discrepancies, however, may be differences between species and/or tissue-specific differences in development. Studies on CYP7B1 expression in development have been performed with humans, mice and pigs. Setchell et al. [52] described a newborn human child with a mutation in the CYP7B1 gene who suffered from severe neonatal cholestasis and liver failure. This finding indicates that CYP7B1 is expressed in human liver at birth and that it may be very important for bile acid synthesis in early life. Furthermore, recent studies by Tang et al. [112] report CYP7B1 expression in several human fetal tissues, including liver. In contrast to the situation in humans, CYP7B1

is not present in livers of newborn mice but is upregulated later in life as shown by Schwarz et al. [69]. In extrahepatic tissues, however, murine CYP7B1 appears to be expressed both before and at birth. Bean et al. [121] studied CYP7B1 expression in mice fetuses and observed widespread expression in fetal extrahepatic tissues which became restricted to fewer organs towards birth. Consistently with the findings by Schwarz et al. [69], Bean et al. [121] reported that hepatic CYP7B1 expression in mice was undetectable at birth. Studies on developmental CYP7B1 expression in pigs indicate an expression pattern similar to humans with hepatic CYP7B1 expression present in newborns. Studies by Norlin and collaborators [24, 112] indicate that both porcine and human CYP7B1 levels increase with age in the liver but decrease in the kidney. In general, extrahepatic CYP7B1 expression appears higher in fetal than in adult tissues. The findings on tissue-specific and age-related variations in CYP7B1 expression [24, 112, 121] suggest that there may be a tissue-specific developmental regulation of this enzyme. It is possible that this might be connected to different functions for CYP7B1 in different tissues.

Mitochondrial Oxysterol 7α-Hydroxylase

Axelson et al. [122] reported that pig liver contains a mitochondrial sterol 7α-hydroxylase active on 27-oxygenated sterols. Shoda et al. [102] demonstrated the formation of 7α-hydroxylated bile acid precursors from 27-oxygenated sterols in human liver mitochondria. The results suggest the presence in human liver mitochondria of a 7α-hydroxylase active toward 3β-hydroxy-5-cholestenoic acid but essentially inactive toward 27-hydroxycholesterol. The oxysterol 7α-hydroxylase, CYP7B1, in human liver microsomes appears to have a different and broader substrate specificity. The presence of an oxysterol 7α-hydroxylase in mitochondria is also reported by Ren et al. [116] in studies with rat liver.

CYP8B1 (Sterol 12α-Hydroxylase)

Sterol 12α-hydroxylation is a reaction unique for cholic acid biosynthesis (Fig. (1)). Early studies indicated that the hepatic 12α-hydroxylating enzyme is a specific microsomal cytochrome P450 induced by starvation and diabetes [55, 123].

A sterol 12α-hydroxylase was purified to apparent homogeneity from rabbit liver microsomes in 1992 by Ishida et al. [124]. The molecular cloning of the rabbit enzyme was reported in 1996 by Eggertsen et al. [125]. The deduced primary structure showed the highest overall sequence identity with prostacyclin synthase, CYP8A1. Consequently, sterol 12α-hydroxylase was designated CYP8B1. CYP8B1 mRNA was found to be exclusively expressed in the liver [125]. Characterization of the CYP8B1 gene structure in human and mouse revealed that the gene is devoid of introns [126], a feature that is unique among mammalian cytochrome P450 genes.

Enzymes in the Conversion of Cholesterol

Regulation

An overview of reported regulators of CYP8B1 is given in Fig. (5). Microsomal sterol 12α-hydroxylase activity in rats is known to be affected by fasting/starvation, streptozotocin-induced diabetes, insulin, thyroid hormones and bile acids [55,123]. The nuclear receptor PPARα might mediate the fasting-induced stimulation of CYP8B1. Hunt et al. [127] reported that treatment of mice with a peroxisome proliferator or fasting for 24 h both induced the sterol 12α-hydroxylase mRNA in liver. Using the PPAR knockout mouse model, it was found that the induction by both treatments was dependent on PPARα.

The elevated hepatic CYP8B1 level in streptozotocin-induced diabetic rats was suppressed by insulin administration and the CYP8B1 mRNA level in cultivated rat hepatoma cells was strongly suppressed by insulin [128,129]. These observations indicate that CYP8B1 expression is dominantly regulated by the direct action of insulin on hepatocytes. The rat liver sterol 12α-hydroxylase activity is stimulated by thyroidectomy and inhibited by thyroid hormone. Andersson et al. [130] found that thyroid hormone decreased CYP8B1 activity and mRNA in vivo. However, no hitherto described thyroid hormone response elements could be identified in the rat CYP8B1 promoter.

CYP8B1 has been reported to be regulated by members of the SREBP family. The rat CYP8B1 promoter contains several sterol response elements (SRE) and is activated by the SREBP-1 isoform whereas a second isoform, SREBP-2, has been reported to suppress the promoter activity in an indirect manner [131,132].

Early studies suggested that sterol 12α-hydroxylase activity in rat liver microsomes increases with cholestyramine feeding and decreases with bile acid feeding. Bile acids inhibit CYP8B1 mRNA expression in rats [133]. It is believed that downregulation by bile acids is mediated, in a similar way as the regulation of CYP7A1, by the FXR receptor via interaction of SHP with HNF4α or FTF (LRH-1). BARE identified in rat and human CYP8B1 promoters have overlapping FTF and HNF4α sites [6,133,134]. In addition, it has been suggested that a SHP-independent mechanism may play an important role in mediating bile acid feedback inhibition of the CYP8B1 gene transcription [6]. This mechanism involves bile acid-induced inflammatory cytokines and MAPK signaling pathways leading to inhibition of the trans-activation by HNF-4α on the CYP8B1 gene [135]. Suppression of the sterol 12α-hydroxylase activity has been suggested as a possible therapeutic strategy for dissolution of cholesterol gallstones [126].

![Diagram](attachment:diagram.png)

Fig. (5). An overview of reported regulators of CYP8B1.
+, upregulation; -, downregulation. See text for details, abbreviations and reference. FXR heterodimerizes with RXR.

210 Current Molecular Medicine, 2007, Vol. 7, No. 2

Tissue Distribution and Expression During Development

CYP8B1 is considered a liver-specific enzyme [125, 128]. CYP8B1 has been found to be expressed in both fetal and adult human liver samples, although to a much less extent in the fetal liver [136].

In the pig, cholic acid is not detectable in bile of adult animals whereas hyocholic acid is found in amounts equal to cholic acid in humans. The pathway leading to formation of hyocholic acid was believed to be species-specific and to have evolved in the pig to compensate for a non-existent or deficient cholic acid biosynthesis. However, recent data show a high level of cholic acid in bile of fetal pig [137]. A gene encoding the key enzyme in synthesis of cholic acid, the sterol 12α-hydroxylase (CYP8B1) is in fact present in the pig genome [136]. In contrast to the situation in human, the expression of CYP8B1 and formation of cholic acid appears to be restricted to the early fetal age in pig. Also in this species the gene lacks introns [136].

CYP27A1 (Sterol 27-Hydroxylase)

A 27-hydroxylation is required for degradation of the C27-sterol side chain in connection with bile acid biosynthesis (Fig. (1)). The 27-hydroxylation and the subsequent oxidation of the 27-hydroxysteroid into the corresponding 27-carboxy steroid (C27 acid) are catalyzed by mitochondrial sterol 27-hydroxylase (CYP27A1). For detailed review of studies on this enzyme up to about 1991 the reader is referred to a previous review article [138]. CYP27A1 seems, however, to have roles beyond bile acid formation, as illustrated by patients with a deficient sterol 27-hydroxylase. This disease will be discussed later in this review.

Sterol 27-hydroxylase was first purified from mitochondria of rabbit liver in 1984 [39, 139, 140] and subsequently from rat liver [141] and pig kidney [142]. cDNA encoding for sterol 27-hydroxylase was first isolated from rabbit in 1989 [143], and subsequently from rat [144], human [31] and pig [145]. Sterol 27-hydroxylase participates in several different metabolic reactions in cholesterol homeostasis and vitamin D3 metabolism. The catalytical properties and substrate specificity have been studied with purified native liver enzyme and with recombinant human CYP27A1 [29, 30, 39, 41, 48, 139, 146-150]. This mitochondrial cytochrome P450 enzyme catalyzes 27-hydroxylation of intermediates in the neutral pathway of bile acid biosynthesis such as 5β-cholestane-3α, 7α, 12α-triol and 5β-cholestane-3α, 7α-diol, as well as 27-hydroxylation of cholesterol in the acidic pathway (Fig. (1)). Small amounts of 24- and 25-hydroxycholesterol also may be formed. The enzyme is also known to produce 27-hydroxycholesterol in extrahepatic cells [16, 142]. In general, the 27-hydroxylation is more efficient with polar sterols and

sterols with 3-oxo-Δ4 structure, for instance 4-cholesten-3-one (cholestenone), a precursor of cholestanol [48]. Furthermore, CYP27A1 performs 25-hydroxylation of vitamin D3, the first step of vitamin D3 bioactivation, as well as 25-hydroxylation of 1α-hydroxyvitamin D3, a non-physiological compound which is used as a drug in treatment of diseases caused by disturbances in calcium metabolism [139]. Although it is considered to be more important for the 25-hydroxylation step, CYP27A1 is also able to catalyze both 1α- and 27-hydroxylations of 25-hydroxyvitamin D3 [146, 147, 149, 150].

CYP27A1 mRNA and protein have been found in numerous tissues [143]. A major biologic role of CYP27A1 in extrahepatic tissues is believed to be the generation of ligands such as 27-hydroxycholesterol and 3β-hydroxy-5-cholestenoic acid, which regulate the expression of nuclear receptors that govern many aspects of cholesterol homeostasis in various cells. 27-Hydroxycholesterol formed by CYP27A1 suppresses, via SREBP, the rate-limiting enzyme in cholesterol biosynthesis, HMG-CoA reductase [77, 107, 151]. There are several possible mechanisms in which CYP27A1 could play important roles for elimination of cholesterol from extrahepatic cells. 27-Hydroxycholesterol upregulates the genes for ABC-transport proteins in e.g. macrophages and thus stimulates the transport of cholesterol from the cell via reverse (HDL) cholesterol transport [77, 152]. In addition, 27-hydroxycholesterol and other 27-oxygenated products formed by CYP27A1 are excreted from the cell more efficiently than cholesterol. 27-Oxygenated products formed in extrahepatic tissues can be transported to the liver and eliminated through conversion to bile acids [16, 153]. The function of CYP27A1 in extrahepatic tissues may also be related to the role of LDL. It is reported that the suppressive effect of LDL on HMG-CoA reductase is decreased when 27-hydroxylation of LDL cholesterol in fibroblasts is inhibited [154].

The molecular genetics behind the sterol 27-hydroxylase deficiency resulting in the disease cerebrotendinous xanthomatosis (CTX) in humans will be discussed later in this review. Mice with a disrupted cyp27a1 gene (cyp27(-/-)) are reported to have normal plasma levels of cholesterol, retinol, tocopherol, and 1,25-dihydroxyvitamin D. Excretion of fecal bile acids is markedly decreased, and formation of bile acids from tritium-labeled 7α-hydroxycholesterol is much less than normal. Somewhat unexpectedly, no CTX-related pathological abnormalities have been observed in mice [155].

Regulation

An overview of reported regulators of CYP27A1 is given in Fig. (6). Many studies on regulation of sterol 27-hydroxylase have been performed with the rat as a model. Bile acids and cholesterol appear, however, to affect this enzyme differently in different species. Sterol 27-hydroxylase activity and mRNA levels are

Enzymes in the Conversion of Cholesterol

decreased by bile acids in the rat [156] but unaffected in the rabbit [157, 158] and mouse [159]. Cholesterol is reported to increase sterol 27- hydroxylase levels in the rabbit [157] but not in the rat [156] or mouse [159]. It has been reported that there is a coordinate regulation of sterol 27- hydroxylase (CYP7A1) and cholesterol 7α- hydroxylase (CYP27A1) by bile acids in rats [160- 162]. In humans, however, there appears to be little or no coordinate regulation of CYP7A1 and CYP27A1 at the transcriptional level, and human CYP27A1 is not subject to a negative feedback control by bile acids [163]. The results underline that marked species differences may exist in mechanisms for control of synthesis of bile acids and cholesterol homeostasis.

Regulation of CYP27A1 is of interest for several reasons. For instance, this enzyme has been suggested to be rate-limiting in the acidic pathway. In addition, the 27-hydroxycholesterol product formed by this enzyme influences several aspects of cholesterol homeostasis. An inhibitory effect of insulin on CYP27A1 transcription has been reported in studies with rat hepatocytes [164]. In a study on the hormonal regulation of the human CYP27A1 gene, it was found that growth hormone, insulin growth factor-1 (IGF-1) and dexamethasone increased both the transcriptional activity and the endogenous CYP27A1 enzyme activity in HepG2 cells, whereas thyroxine and phorbol-12-myristate- 13-acetate (PMA) repressed the activities markedly [165]. Mechanisms for up-regulation of the human CYP27A1 gene, such as by hormones, are of interest in therapy since anti-atherogenic effects have been ascribed to the CYP27A1 enzyme. This

enzyme and its gene may be potential targets for drug development. Recently, the up-regulation of CYP27A1 in cultured macrophages by TGF-β1 was reported [166].

### Tissue Distribution and Expression During Development

CYP27A1 is expressed in liver and many extrahepatic tissues, including kidney, lung, macrophages, brain, vascular endothelium, skin and prostate [16, 143, 167, 168].

In a study on the expression of CYP27A1 in pig liver during development, it was reported that the levels of CYP27A1 mRNA and enzymatic activities are about the same in newborns and six months old pigs [24]. The presence of similar levels of CYP27A1 in the newborn and in the mature state is in agreement with multiple important roles for this enzyme [16, 143, 167, 168].

#### CYP3A Subfamily

In 1999, CYP3A4 was shown to catalyze the 6α- hydroxylation of the hepatotoxic lithocholic acid and other bile acids [169], and the 25-hydroxylation of the bile acid precursor 5β-cholestane-3α,7α,12α-triol [46] (Fig. (2B)). Since then, several groups have reported that lithocholic acid and 5β-cholestane- 3α,7α,12α-triol are ligands for the pregnane X receptor (PXR) which is a regulator of the CYP3A4 gene further indicating a link between CYP3A4 and bile acid levels [170-173]. In addition, the involvement of members of the CYP3A subfamily in hydroxylations of bile acids in several different positions has recently been reported [174].

![Diagram](attachment:diagram.png)

Fig. (6). An overview of reported regulators of CYP27A1.  
+, upregulation; -, downregulation. See text for details, abbreviations and reference.

The human CYP3A subfamily includes CYP3A4, CYP3A5, and CYP3A7. The CYP3A enzymes have been estimated to be associated with over 40% of drug metabolism. CYP3A4 is the most abundantly expressed form of P450 in human liver. CYP3A5 appears to be present in only 10-30% of human subjects. In addition to CYP3A4, CYP3A5 is able to form the 25-hydroxylated product from 5β-cholestane-3α,7α,12α-triol [46]. CYP3A7 is expressed during human fetal life.

CYP4A21 (Porcine Taurochenodeoxycholic Acid 6α-Hydroxylase)

Hyocholic acid is a major primary bile acid in the pig. The key enzyme for hyocholic acid biosynthesis, performing 6α-hydroxylation of chenodeoxycholic acid (or its conjugate), has been characterized and designated CYP4A21 [17, 21]. The 6α-hydroxylase is a member of the CYP4A subfamily, enzymes known to catalyze hydroxylation of fatty acids and prostaglandins. CYP4A21 is unique among CYP4A

A

![27-hydroxycholesterol](https://i.imgur.com/1.png)
27-hydroxycholesterol

![25-hydroxycholesterol](https://i.imgur.com/2.png)
25-hydroxycholesterol

![24-hydroxycholesterol](https://i.imgur.com/3.png)
24-hydroxycholesterol

B

| Reported roles of oxysterols in bile acid biosynthesis/cholesterol metabolism |
|---------------------------------------------------------------------------|
| **27-hydroxy-cholesterol**                                                |
| - quantitatively important as bile acid precursor                          |
| - ligand of LXR                                                          |
| - suppresses SREBP                                                       |
| - upregulates ABC transport proteins                                     |
| - role in cholesterol elimination from extrahepatic cells, e.g. macrophages |
| **25-hydroxy-cholesterol**                                                |
| - ligand of LXR                                                          |
| - suppresses SREBP                                                       |
| **24-hydroxy-cholesterol**                                                |
| - role in brain cholesterol homeostasis/cholesterol transport from brain   |
| - minor contribution as bile acid precursor                               |
| - ligand of LXR                                                          |

Fig. (7). Oxysterols of interest in cholesterol homeostasis.
A, Structures of 24S, 25- and 27-hydroxycholesterol.
B, Overview of reported roles of oxysterols in bile acid biosynthesis/cholesterol metabolism.

Enzymes in the Conversion of Cholesterol

Current Molecular Medicine, 2007, Vol. 7, No. 2 213

enzymes in that it participates in bile acid
biosynthesis and do not possess fatty acid
hydroxylase activity [17, 18]. Sequence comparison
between the CYP4A21 gene and other porcine
CYP4A genes indicates that the unique catalytic
properties of CYP4A21 could originate from gene-
conversion events during evolution [18].

FORMATION AND METABOLISM OF
OXYSTEROLS

The oxysterols 24S-hydroxycholesterol, 25-
hydroxycholesterol, and 27-hydroxycholesterol are
intermediates in bile acid biosynthesis, participate in
transport of cholesterol from the periphery to the
liver, and affect transcriptional proteins regulating the
expression of genes in cholesterol homeostasis (Fig.
(7)). The role of oxysterols in cholesterol homeostasis
has been reviewed by Björkhem [175]. The enzymes
involved in the formation of the oxysterols have been
reviewed by Russell [176].

CYP27A1, expressed in most tissues, converts
cholesterol into 27-hydroxycholesterol and to a
lesser extent 24- and 25-hydroxycholesterol [177].

Cholesterol is converted to 24S-hydroxycho-
lesterol by CYP46A1 (cholesterol 24-hydroxylase)
which is primarily expressed in the brain. Lund et al.
[178] isolated cDNAs that encode murine and
human cholesterol 24-hydroxylases. It was recently
reported that the substrate specificities of CYP46A1
are not limited to cholesterol but include a number of
structurally diverse compounds such as C21-steroids
and some xenobiotics [179]. Thus, CYP46A1 may
play several roles in the brain.

cDNAs encoding cholesterol 25-hydroxylases
were isolated from the mouse and human [180]. The
cholesterol 25-hydroxylase enzymes, expressed in
most murine and human tissues, are small,
hydrophobic proteins that are not cytochrome P-450
enzymes but rather belongs to a smaller family of
non-heme iron containing proteins.

Metabolism of the oxysterols in the liver and other
tissues involves 7α-hydroxylation and 27-
oxygenation. The metabolism is believed to produce
metabolites that are less active as ligands for nuclear
receptors and transcription factors [181]. 27-
Hydroxycholesterol is converted to bile acids via the
acidic pathway. 24-Hydroxycholesterol and 25-
hydroxycholesterol may also ultimately be converted
to bile acids [5, 48].

The 7α-hydroxylation of oxysterols can be
performed by at least three different 7α-
hydroxylases, CYP7A1, CYP7B1, and CYP39A1
[42, 43, 71, 72, 106]. Interestingly, the three oxysterol
7α-hydroxylases exhibit different substrate specificity
for various oxysterols. Hepatic CYP7A1 is able to
catalyze the 7α-hydroxylation of 20(S)-, 24-, 25- and
27-hydroxycholesterol [42, 71]. CYP7B1, expressed
in most tissues, is active towards 25- and 27-
hydroxycholesterol, but shows very low activity


towards 24-hydroxycholesterol [42, 72, 106].
CYP39A1 is a microsomal cytochrome P450 enzyme
that has preference for 7α-hydroxylation of 24-
hydroxycholesterol and is expressed in the liver
[43, 105].

The oxysterols can also be 27-oxygenated by
CYP27A1, expressed in several tissues. CYP27A1 is
able to introduce a 27-hydroxyl group in 24-
hydroxycholesterol and 25-hydroxycholesterol [48]
and to oxidize the 27-hydroxysterol into the
corresponding C27 acid [28-30, 41].

INBORN ERRORS OF HUMAN BILE ACID
BIOSYNTHESIS

Excellent reviews covering inborn errors of bile
acid biosynthesis have been written by Björkhem
and Boberg [182], Bove et al. [183-185], and
Russell [5]. The reader is referred to these articles for
a more detailed description of the diseases. This
review will summarize diseases involving mutations in
genes encoding enzymes in bile acid formation.

Cholesterol 7α-Hydroxylase Deficiency

A metabolic disorder presenting with elevated
plasma cholesterol levels caused by a homozygous
deletion mutation in the CYP7A1 gene in a family of
English and Celtic origin was recently described
[70, 186]. The mutation leads to a frameshift resulting
in the synthesis of a truncated protein with no
enzymatic activity. The high levels of LDL cholesterol
in the CYP7A1-deficient adult subjects were found to
be resistant to treatment with hydroxymethylglutaryl
coenzyme A (HMG-CoA) inhibitors. Individuals that
are heterozygous for the mutation were also found
to be hyperlipidemic, indicating that the disorder is
inherited in a codominant fashion [186].

Oxysterol 7α-Hydroxylase (CYP7B1) Deficiency

Setchell et al. [52] described a newborn child with
severe neonatal cholestasis with a mutation in the
CYP7B1 gene. The condition was lethal. The patient
had markedly elevated levels of 24-, 25-, and 27-
hydroxycholesterol and excreted unsaturated
monohydroxy bile acids. The liver failure is consistent
with the known hepatotoxic properties of oxysterols
and unsaturated monohydroxy bile acids. This
patient had a normal CYP7A1 gene. The finding that
the CYP7B1 gene defect was lethal indicates that
CYP7B1 in humans may be important in infancy.

3β-Hydroxy-Δ5-C27-Steroid Dehydrogenase
Deficiency

This disease was first described by Clayton et al.
[187]. The clinical manifestations may begin at any
age during childhood and include jaundice, fat
malabsorption with signs of fat-soluble vitamin
deficiency, and progressive cholestasis. Virtually no
chenodeoxycholic acid or cholic acid is formed (Fig.

214 Current Molecular Medicine, 2007, Vol. 7, No. 2

(1)). Instead, accumulation in blood and urine of hepatotoxic bile acids with 3β-hydroxy-Δ⁵ configuration are found [182]. The human gene encoding the enzyme was isolated and a 2-bp deletion mutation, eliminating the enzymatic activity, was identified in a patient with 3β-hydroxy-Δ⁵-C₂₇ steroid dehydrogenase deficiency [22]. Oral bile acid substitution therapy is curative and it has been suggested that, with timely intervention, this liver disease may be prevented [184].

[52,106]. Finally, CYP27A1-deficiency in humans causes CTX with cholesterol and cholestanol accumulation throughout the body, whereas mice with the same defect have a bile acid defect but do not develop signs of CTX such as generalized accumulation of cholesterol and cholestanol in their tissues [47,155,191]. It has been suggested that the species differences between humans and mice may reflect differences in how bile acid synthesis is regulated in the two species for example via nuclear receptors [192].

### Sterol 27-Hydroxylase Deficiency

Patients with the inherited lipid storage disease cerebrotendinous xanthomatosis (CTX), a rare autosomal gene defect, have a deficient sterol 27-hydroxylase due to point mutations of the CYP27A1 gene [188]. A large number of mutations leading to CTX have been described in this gene [189]. Biochemical characteristics of CTX are markedly reduced production of chenodeoxycholic acid and increased formation of various 25-hydroxylated C₂₇ bile alcohols [182]. Manifestations of this genetically determined CYP27A1 deficiency range from accumulation of bile alcohols and cholestanol to accelerated atherosclerosis, osteoporosis and progressive neurologic impairment. The major symptoms in CTX are considered to be due to the generalized accumulation of cholesterol and cholestanol in almost every tissue, including the nervous system [182,190]. There are several hypotheses for the mechanisms behind the dramatically increased synthesis of cholestanol in patients with CTX [182]. This may be due to increased cholesterol biosynthesis or increased utilization of bile acid intermediates as cholestanol precursors. Other possibilities are increased activity of 3β-hydroxy-Δ⁵-C₂₇-steroid dehydrogenase or the abolished activity of CYP27A1, which efficiently metabolizes 4-cholesten-3-one (cholesteneone), an important cholestanol precursor [48]. During normal conditions, excess 4-cholesten-3-one may be continuously drained from the cholestanol-forming pathways by 27-hydroxylation resulting in bile acid formation [48]. CTX can be successfully treated by oral bile acid therapy [182].

### FUTURE PERSPECTIVES

Further study of bile acid biosynthetic enzymes and their genes is of great interest because of the potential benefits for the development of new drugs to control hypercholesterolemia and to prevent atherosclerosis and other diseases related to disturbed cholesterol homeostasis. However, information on the properties and regulation of these enzymes is important not only in therapy but also in understanding development of diseases related to disturbances in cholesterol homeostasis. Most studies on enzymes in cholesterol metabolism have been performed with mice and rats. It is evident that there are important differences in e.g. regulation of the enzymes between animals and humans. It would be of interest to clarify the mechanisms that are involved behind such species differences. Although CYP7A1 has been subject to intense study, much less is known about the regulation of the human CYP7B1 and CYP27A1 genes. It should be important to understand more about how the human CYP7B1 and CYP27A1 genes are regulated in hepatic and extrahepatic tissues. The importance of metabolic conversions and mechanisms for transport of cholesterol between tissues, such as formation of 27-hydroxycholesterol for transport from lung and macrophages and 24-hydroxycholesterol from brain, also are subjects of increasing interest. Furthermore, we expect more information to become available on the roles of nuclear receptors and their various ligands for regulation of bile acid formation, as well as the identities of transcription factors and intracellular signalling pathways.

### Differences Between Data on Mice and Humans with Deficiencies in Bile Acid Biosynthetic Enzymes

From studies using different species it seems that humans and mice exhibit different phenotypes for deficiencies in bile acid biosynthetic enzymes. Some important examples will be given. The CYP7A1-deficient humans were hyperlipidemic whereas the Cyp7α1-deficient mice were normolipidemic [70,159,186]. In humans, this deficiency appears not to be lethal, whereas Cyp7α1-deficient mice were severely impacted [68]. A human infant with CYP7B1 deficiency died with severe biliary cholestasis. In contrast, Cyp7b1-deficient mice appear relatively healthy and have no disturbed bile acid production

### ACKNOWLEDGEMENTS

Research described here from the authors' laboratory is supported by grants from the Swedish Research Council – Medicine (projects 03X-218, 03P-14802 and 03X-14769).

### ABBREVIATIONS

ABCA1 = ATP binding cassette protein A1  
BA = bile acid  
BARE = bile acid response element  
CTX = cerebrotendinous xanthomatosis  
Norlin and Wikvall

Enzymes in the Conversion of Cholesterol

CYP = cytochrome P450  
FTF = α-fetoprotein transcription factor  
FXR = farnesoid X receptor  
GC/MS = gas chromatography-mass spectrometry  
HDL = high density lipoprotein  
HMG-CoA = 3-hydroxy-3-methylglutaryl coenzyme A  
HNF4 = hepatocyte nuclear factor 4  
IGF-1 = insulin-like growth factor-1  
IL-1 = interleukin 1  
JNK = Jun N-terminus kinase  
LDL = low density lipoprotein  
LRH-1 = liver receptor homolog 1  
LXR = liver X receptor  
MAPK = mitogen-activated protein kinase  
PMA = Phorbol-12-myristate-13-acetate  
POEL = polyoxyethylene 10 lauryl ether  
PPAR = peroxisome proliferator activated receptor  
PXR = pregnane X receptor  
RXR = retinoid X receptor  
SDS-PAGE = sodium dodecyl sulfate-polyacrylamide gel electrophoresis  
SHP = small heterodimer partner  
SREBP = sterol response element binding protein  
TGF-β1 = transforming growth factor β1  
VDR = vitamin D receptor  

THE FOLLOWING TRIVIAL NAMES ARE USED  
Chenodeoxycholic acid, 3α,7α-dihydroxy-5β-cholanoic acid  
Cholic acid, 3α,7α,12α-trihydroxy-5β-cholanoic acid  
Deoxycholic acid, 3α,12α-dihydroxy-5β-cholanoic acid  
Hyocholic acid, 3α,6α,7α-trihydroxy-5β-cholanoic acid  
Lithocholic acid, 3α-hydroxy-5β-cholanoic acid  
7α-hydroxycholesterol, 5-cholestene-3β,7α-diol  
27-hydroxycholesterol, 5-cholestene-3β,27-diol  
7α,27-dihydroxycholesterol, 5-cholestene-3β,7α,27-triol  
25-hydroxycholesterol, 5-cholestene-3β,25-diol  
24-hydroxycholesterol, 5-cholestene-3β,24-diol  
22-hydroxycholesterol, 5-cholestene-3β,22-diol  

20-hydroxycholesterol, 5-cholestene-3β,20-diol  
dehydroepiandrosterone, 5-androsten-3β-ol-17-one  
pregnenolone, 5-pregnen-3β-ol-20-one  
7-dehydrocholesterol, 5,7-cholestadien-3β-ol  
vitamin D₃, cholecalciferol  
7-oxocholesterol, 5-cholesten-3β-ol-7-one  

REFERENCES  
[1] Princen, H.M.G., Post, S.M. and Twisk, J. (1997) *Curr. Pharm. Des.*, **3**, 59-84.  
[2] Hofmann, A.F. (1999) *Arch. Intern. Med.*, **59**, 2647-2658.  
[3] Vlahcevic, Z.R., Pandak, M.P. and Stravitz, R.T. (1999) *Gastroenterol. Clin. North Am.*, **28**, 1-25.  
[4] Björkhem, I. and Eggertsen, G. (2001) *Curr. Opin. Lipidol.*, **12**, 97-103.  
[5] Russell, D.W. (2003) *Annu. Rev. Biochem.*, **72**, 137-174.  
[6] Chiang, J.Y. (2004) *J. Hepatol.*, **40**, 539-551.  
[7] Martin, M.J., Hulley, S.B., Browner, W.S., Kuller, L.H. and Wentworth, D. (1986) *Lancet*, **25**, 933-936.  
[8] Bennion, L.J. and Grundy, S.M. (1978) *N. Engl. J. Med.*, **299**, 1161-1167.  
[9] Reiss, A.B., Siller, K.A., Rahman, M.M., Chan, E.S., Ghiso, J., de Leon, M.J. (2004) *Neurobiol. Aging*, **25**, 977-989.  
[10] Björkhem, I. and Meaney, S. (2004) *Arterioscler. Thromb. Vasc. Biol.*, **24**, 806-815.  
[11] Vlahcevic, Z.R., Hylemon, P.B. and Chiang, J.Y. (1994) in *The Liver: Biology and Pathobiology, 3rd ed.* (Arias, I.M., Ed.), pp. 379-389, Raven Press Ltd. New York.  
[12] Parish, E.J., Parish, S.C. and Li, S. (1999) *Crit. Rev. Biochem. Mol. Biol.*, **34**, 265-272.  
[13] Tint, G.S., Irons, M., Elias, E.R., Batta, A.K., Frieden, R., Chen, T.S. and Salen, G. (1994) *N. Engl. J. Med.*, **330**, 107-113.  
[14] Mahley, R.W., Innerarity, T.L., Rall, S.C. and Weisgraber, K.H. (1984) *J. Lipid Res.*, **25**, 1277-1294.  
[15] Oram, J.F. and Yokoyama, S. (1996) *J. Lipid Res.*, **37**, 2473-2491.  
[16] Babiker, A., Andersson, O., Lund, E., Xiu, R.-J., Deeb, S., Reshef, A., Leitersdorf, E., Diczfalussy, U. and Björkhem, I. (1997) *J. Biol. Chem.*, **272**, 26253-26261.  
[17] Lundell, K., Hansson, R. and Wikvall, K. (2001) *J. Biol. Chem.*, **276**, 9606-9612.  
[18] Lundell, K. (2004) *Biochem. J.*, **378**, 1053-1058.  
[19] Axelsson, M. and Sjövall, J. (1990) *J. Steroid Biochem.*, **36**, 631-640.  
[20] Wikvall, K. (1981) *J. Biol. Chem.*, **256**, 3376-3380.  
[21] Furster, C., Zhang, J. and Toll, A. (1996) *J. Biol. Chem.*, **271**, 20903-20907.  
[22] Schwarz, M., Wright, A.C., Davis, D.L., Nazer, H., Björkhem, I. and Russell, D.W. (2000) *J. Clin. Invest.*, **106**, 1175-1184.  
[23] Ichimiya, H., Egestad, B., Nazer, H., Baginski, E.S., Clayton, P.T. and Sjövall, J. (1991) *J. Lipid Res.*, **32**, 829-841.  
[24] Norlin, M. (2002) *J. Lipid Res.*, **43**, 721-731.  
[25] Kondo, K.H., Kai, M.H., Setoguchi, Y., Eggertsen, G., Sjöblom, P., Setoguchi, T., Okuda, K.I. and Björkhem, I. (1994) *Eur. J. Biochem.*, **219**, 357-363.  
[26] Usui, E., Okuda, K., Kato, Y. and Noshiro, M. (1994) *J. Biochem. (Tokyo)*, **115**, 230-237.  
[27] Penning, T.M., Jin, Y., Steckelbroeck, S., Lanisnik Rizner, T. and Lewis, M. (2004) *Mol. Cell. Endocrinol.*, **215**, 63-72.  
[28] Dahlbäck, H. and Holmberg, I. (1990) *Biochem. Biophys. Res. Commun.*, **167**, 391-395.  
[29] Holmberg-Bethsholtz, I., Lund, E., Björkhem, I. and Wikvall, K. (1993) *J. Biol. Chem.*, **268**, 11079-11085.  
[30] Holmberg-Bethsholtz, I. and Wikvall, K. (1995) *Steroid Biochem. Mol. Biol.*, **55**, 115-119.  
[31] Cali, J.J. and Russell, D.W. (1991) *J. Biol. Chem.*, **266**, 7774-7778.  
[32] Dahlbäck, H., Danielsson, H., Gustafsson, M., Sjövall, J. and Wikvall, K. (1988) *Biochem. Biophys. Res. Commun.*, **153**, 267-274.

[33] Okuda, A. and Okuda, K. (1983) *J. Biol. Chem.*, 258, 2899-2905.

[34] Solaas, K., Ulvestad, A., Soreide, O. and Kase, B.F. (2000) *J. Lipid Res.*, 41, 1154-1162.

[35] Mitropoulos, K.A. and Myant, N.B. (1967) *Biochem. J.*, 103, 472-479.

[36] Javitt, N.B., Kok, E., Carubbi, F., Bizzard, T., Gut, M. and Byon, C.Y. (1986) *J. Biol. Chem.*, 261, 12486–12489.

[37] Ayaki, Y., Kok, E. and Javitt, N.B. (1989) *J. Biol. Chem.*, 264, 3818–3821.

[38] Anderson, K.E., Kok, E. and Javitt, N.B. (1972) *J. Clin. Invest.*, 51, 112–117.

[39] Wikvall, K. (1984) *J. Biol. Chem.*, 259, 3800-3804.

[40] Norlin, M. and Wikvall, K. (1998) *Biochim. Biophys. Acta*, 1390, 269-281.

[41] Pikuleva, I.A., Babiker, A., Waterman, M.R. and Björkhem, I. (1998) *J. Biol. Chem.*, 273, 18153-18160.

[42] Norlin, M., Toll, A., Björkhem, I. and Wikvall, K. (2000) *J. Lipid Res.*, 41, 1629-1639.

[43] Li-Hawkins, J., Lund, E.G., Bronson, A.D. and Russell, D.W. (2000) *J. Biol. Chem.*, 275, 16543-16549.

[44] Björkhem, I., Andersson, U., Ellis, E., Alvelius, G., Ellegard, L., Diczfalussy, U., Sjövall, J. and Einarsson, C. (2001) *J. Biol. Chem.*, 276, 37004-37010.

[45] Shefer, S., Cheng, F.W., Dayal, B., Hauser, S., Tint, G.S., Salen, Gb and Mosbach, E.H. (1976) *J. Clin. Invest.*, 57, 897-903.

[46] Furster, C. and Wikvall, K. (1999) *Biochim. Biophys. Acta*, 1437, 46-52.

[47] Honda, A., Salen, G., Matsuzaki, Y., Batta, A.K., Xu, G., Leitersdorf, E., Tint, G.S., Erickson, S.K., Tanaka, N. and Shefer, S. (2001) *J. Biol. Chem.*, 276, 34579-34585.

[48] Norlin, M., von Bahr, S., Björkhem, I. and Wikvall, K. (2003) *J. Lipid Res.*, 44, 1515-1522.

[49] Björkhem, I. (1985) In *New Comprehensive Biochemistry*, Vol. 12: Sterols and Bile Acids (Danielsson, H. and Sjövall, J., Eds.), pp. 231-278, Elsevier, Amsterdam.

[50] Duane, W.C. and Javitt, N.B. (1999) *J. Lipid Res.*, 40, 1194-1199.

[51] Schwarz, M., Lund, E.G., Setchell, K.D., Kayden, H.J., Zerwekh, J.E., Björkhem, I., Herz, J. and Russell, D.W. (1996) *J. Biol. Chem.*, 271, 18024-18031.

[52] Setchell, K.D., Schwarz, M., O'Connell, N.C., Lund, E.G., Davis, D.L., Lathe, R., Thompson, H.R., Wesley Tyson, R., Sokol, R.J. and Russell, D.W. (1998) *J. Clin. Invest.*, 102, 1690-1703.

[53] Duane, W.C., Pooler, P.A. and Hamilton, J.N. (1988) *J. Clin. Invest.*, 82, 82-85.

[54] Björkhem, I. and Muri Moberg, K. (1995) In *The metabolic and molecular bases of inherited disease*, $7^{th}$ Ed, Vol 2, (Scriver, C.R., Beaudet, A.L., Sly, W.S. and Valle, D., Eds.), pp. 2073-2099, McGraw-Hill, Inc., New York.

[55] Wikvall, K. (1991) In *Frontiers in Biotransformation*, Vol. 6, (Ruckpaul, K. and Rein, H., eds.), pp. 147-182, Akademie-Verlag, Berlin and Taylor and Francis, London.

[56] Andersson, S., Boström, H., Danielsson, H. och Wikvall, K. (1985) *Methods Enzymol.*, 111 B, 364-377.

[57] Ogishima, T., Deguchi, S. and Okuda, K. (1987) *J. Biol. Chem.*, 262, 7646-7650.

[58] Chiang, J.Y., Miller, W.F. and Lin, G.M. (1990) *J. Biol. Chem.*, 265, 3889-3897.

[59] Wikvall, K. (1988) *Biochem. (Life Sci. Adv.)*, 7, 77-82.

[60] Noshiro, M., Nishimoto, M., Morohashi, K. and Okuda, K. (1989) *FEBS Lett.*, 257, 97-100.

[61] Jelinek, D.F., Andersson, S. Slaughter, C.A. and Russell, D.W. (1990) *J. Biol. Chem.*, 265, 8190-8197.

[62] Li, Y.C., Wang, D.P. and Chiang, J.Y. (1990) *J. Biol. Chem.*, 265, 12012-12019.

[63] Noshiro, M. and Okuda, K. (1990) *FEBS Lett.*, 268, 137-140.

[64] Cohen, J.C., Cali, J.J., Jelinek, D.F., Mehrabian, M., Sparkes, R.S., Lusis, A.J., Russell, D.W. and Hobbs, H.H. (1992) *Genomics*, 14, 153-161.

[65] Nishimoto, M., Noshiro, M. and Okuda, K. (1993) *Biochim. Biophys. Acta*, 1172, 147-150.

[66] Crestani, M., Galli, G. and Chiang, J.Y. (1993) *Arch. Biochem. Biophys.*, 306, 451-460.

[67] Tzung, K.W., Ishimura-Oka, K., Kihara, S., Oka, K. and Chan, L. (1994) *Genomics*, 21, 244-247.

[68] Ishibashi, S., Schwarz, M., Frykman, P.K., Herz, J. and Russell, D.W. (1996) *J. Biol. Chem.*, 271, 18017-18023.

[69] Schwarz, M., Lund, E.G., Setchell, K.D., Kayden, H.J., Zerwekh, J.E., Björkhem, I., Herz, J., and Russell, D.W. (1996) *J. Biol. Chem.*, 271, 18024-18031.

[70] Beigneux, A., Hofmann, A.F. and Young, S.G. (2002) *J. Clin. Invest.*, 110, 29-31.

[71] Norlin, M., Andersson, U., Björkhem, I. and Wikvall, K. (2000) *J. Biol. Chem.*, 275, 34046-34053.

[72] Schwarz, M., Lund, E.G., Lathe, R., Björkhem, I. and Russell, D.W. (1997) *J. Biol. Chem.*, 272, 23995-24001.

[73] Rose, K.A., Stapleton, G., Dott, K., Kieny, M.P., Best, R., Schwarz, M., Russell, D.W., Björkhem, I., Seckl, J. and Lathe, R. (1997) *Proc. Natl. Acad. Sci. USA*, 94, 4925-4930.

[74] Dueland, S., Trawick, J.D., Nenseter, M.S., MacPhee, A.A. and Davis, R.A. (1992) *J. Biol. Chem.*, 267, 22695-22698.

[75] Pandak, W.M, Hylemon, P.B., Ren, S., Marques, D., Gil, G., Redford, K., Mallonee, D. and Vlahcevic, Z.R. (2002) *Hepatology*, 35, 1400-1408.

[76] Chiang, J.Y. (2003) *Am. J. Physiol. Gastrointest. Liver Physiol.*, 284, G349-356.

[77] Chiang, J.Y. (2002) *Endocr. Rev.*, 23, 443-463.

[78] Riddick, D.S., Lee, C., Bhathena, A., Timsit, Y.E., Cheng, P.Y., Morgan, E.T., Prough, R.A., Ripp, S.L., Miller, K.K., Jahan, A. and Chiang, J.Y. (2004) *Drug Metab. Dispos.*, 32, 367-375.

[79] Kerr, T.A., Saeki, S., Schneider, M., Schaefer, K., Berdy, S., Redder, T., Shan, B., Russell, D.W. and Schwarz, M. (2002) *Dev. Cell*, 2, 713-720.

[80] Gupta, S., Stravitz, R.T., Dent, P. and Hylemon P.B. (2001) *J. Biol. Chem.*, 276, 15816-15822.

[81] Gupta, S., Natarajan, R., Payne, S.G., Studer, E.J., Spiegel, S., Dent, P. and Hylemon, P.B. (2004) *J. Biol. Chem.*, 279, 5821-5828.

[82] Staudinger, J.L., Goodwin, B., Jones, S.A., Hawkins-Brown, D., MacKenzie, K.I., LaTour, A., Liu, Y., Klaassen, C.D., Brown, K.K., Reinhard, J., Willson, T.M., Koller, B.H. and Kliewer, S.A. (2001) *Proc. Natl. Acad. Sci. USA*, 98, 3369-3374.

[83] Wang, L., Lee, Y.K., Bundman, D., Han, Y., Thevananther, S., Kim, C.S., Chua, S.S., Wei, P., Heyman, R.A., Karin, M. and Moore, D.D. (2002) *Dev. Cell.*, 2, 721-731.

[84] Makishima, M., Lu, T.T., Xie, W., Kerr Whitfield, G., Domoto, H., Evans, R.M., Haussler, M.R. and Mangelsdorf, D.J. (2002) *Science*, 296, 1313-1316.

[85] Li, T. and Chiang, J.Y. (2005) *Am. J. Physiol. Gastrointest. Liver Physiol.*, 288, G74-84.

[86] Brown, M.J.G. and Boyd, G.S. (1974) *Eur. J. Biochem.*, 44, 37-47.

[87] Lehmann, J.M., Kliewer, S.A., Moore, L.B., Smith-Oliver, T.A., Oliver, B.B., Su, J.L., Sundseth, S.S., Winegar, D.A., Blanchard, D.E., Spencer, T.A. and Willson, T.M. (1997) *J. Biol. Chem.*, 272, 3137-3140.

[88] Peet, D.J., Turley, S.D., Ma, W., Janowski, B.A., Lobaccaro, J.M., Hammer, R.E. and Mangelsdorf, D.J. (1998) *Cell*, 93, 693-704.

[89] Menke, J.G., Macnaul, K.L., Hayes, N.S., Baffic, J., Chao, Y.S., Elbrecht, A., Kelly, L.J., Lam, M.H., Schmidt, A., Sahoo, S., Wang, J., Wright, S.D., Xin, P., Zhou, G., Moller, D.E. and Sparrow, C.P. (2002) *Endocrinology*, 143, 2548-2558.

[90] Chiang, J.Y., Kimmel, R. and Stroup, D. (2001) *Gene*, 262, 257-265.

[91] Marrapodi, M. and Chiang, J.Y. (2000) *J. Lipid Res.*, 41, 514-520.

[92] Patel, D.D., Knight, B.L., Soutar, A.K., Gibbons, G.F. and Wade, D.P. (2000) *Biochem. J.*, 351, 747-753.

[93] Crestani, M., Stroup, D. and Chiang, J.Y. (1995) *J. Lipid Res.*, 36, 2419-2432.

[94] Song, K-H. and Chiang, J.Y. (2006) *Hepatology*, 43, 117-125.

[95] Agellon, L.B. and Cheema, S.K. (1997) *Biochem. J.*, 328, 393-399.

[96] Baker, D.M., Wang, S.L., Bell, D.J., Drevon, C.A. and Davis, R.A. (2000) *J. Biol. Chem.*, 275, 19985-19991.

[97] Stroup, D. and Ramsaran, J.R. (2005) *Biochem. Biophys. Res. Commun.*, 329, 957-965.

[98] Steckelbroeck, S., Watzka, M., Lütjohann, D., Makiola, P., Nassen, A., Hans, W.H.J., Clusmann, H., Reissinger, A., Ludwig, M., Siekmann, L. and Klingmüller, D. (2002) *J. Neurochem.*, 83, 713-726.

Enzymes in the Conversion of Cholesterol

[99] Massimi, M., Lear, S.R., Huling, S.L., Jones, A.L. and Erickson, S.K. (1998) *Hepatology*, **28**, 1064-1072.

[100] Toll, A., Shoda, J., Axelson, M., Sjövall, J. and Wikvall, K. (1992) *FEBS Lett.*, **296**, 73-76.

[101] Björkhem, I., Nyberg, B. and Einarsson, K. (1992) *Biochim. Biophys. Acta*, **1128**, 73-76.

[102] Shoda, J., Toll, A., Axelson, M., Pieper, F., Wikvall, K. and Sjövall, J. (1993) *Hepatology*, **17**, 395-403.

[103] Martin, K.O., Budai, K. and Javitt, N.B. (1993) *J. Lipid Res.*, **34**, 581-588.

[104] Stapleton, G., Steel, M., Richardson, M., Mason, J.O., Rose, K.A., Morris, R.G. and Lathe, R. (1995) *J. Biol. Chem.*, **270**, 29739-29745.

[105] Li-Hawkins, J., Lund, E.G., Bronson, A.D. and Russell, D.W. (2000) *J. Biol. Chem.*, **275**, 16543-16549.

[106] Li-Hawkins, J., Lund, E.G., Turley, S.D. and Russell, D.W. (2000) *J. Biol. Chem.*, **275**, 16536-16542.

[107] Schroepfer, G.J., Jr. (2000) *Physiol. Rev.*, **80**, 361-554.

[108] Weihua, Z., Lathe, R., Warner, M. and Gustafsson, J.Å. (2002) *Proc. Natl. Acad. Sci. USA*, **99**, 13589-13594.

[109] Wu, Z., Martin, K.O., Javitt, N.B., and Chiang, J.Y. (1999) *J. Lipid Res.*, **40**, 2195-2203.

[110] Wu, Z. and Chiang, J.Y. (2001) *Gene*, **272**, 191-197.

[111] Norlin, M. and Chiang, J.Y.L. (2004) *Biochem. Biophys. Res. Commun.*, **316**, 158-164.

[112] Tang, W., Eggertsen, G., Chiang, J.Y. and Norlin, M. (2006) *J. Steroid Biochem. Mol. Biol.*, **100**, 42-51.

[113] Tang, W. and Norlin, M. (2006) *Biochem. Biophys. Res. Commun.*, **344**, 540-546.

[114] Koopen, N.R., Post, S.M., Wolters, H., Havinga, R., Stellaard, F., Boverhof, R., Kuipers, F. and Princen, H.M. (1999) *J. Lipid Res.*, **40**, 100-108.

[115] Bossard, R., Stieger, B., O'Neill, B., Fricker, G. and Meier, P.J. (1993) *J. Clin. Invest.*, **91**, 2714-2720.

[116] Ren, S., Marques, D., Redford, K., Hylemon, P.B., Gil, G., Vlahcevic, Z.R. and Pandak, W.M. (2003) *Metabolism*, **52**, 636-642.

[117] Dulos, J., van der Vleuten, M.A., Kavelaars, A., Heijnen, C.J. and Boots, A.M. (2005) *Arthritis Rheum.*, **52**, 770-778.

[118] Dulos, J., Kaptein, A., Kavelaars, A., Heijnen, C. and Boots, A. (2005) *Arthritis Res. Ther.*, **7**, R1271-1280.

[119] Axelson, M. and Larsson, O. (1996) *J. Biol. Chem.*, **271**, 12724-12736.

[120] Zhang, J., Dricu, A. and Sjövall, J. (1997) *Biochim. Biophys. Acta*, **1344**, 241-249.

[121] Bean, R., Seckl, J.R., Lathe, R. and Martin, C. (2001) *Mol. Cell. Endocrinol.*, **174**, 137-144.

[122] Axelson, M., Shoda, J., Sjövall, J., Toll, A. and Wikvall, K. (1992) *J. Biol. Chem.*, **267**, 1701-1704.

[123] Wikvall, K. (1990) *Curr. Opin. Lipidol.*, **1**, 248-254.

[124] Ishida, H., Noshiro, M., Okuda, K. and Coon, M.J. (1992) *J. Biol. Chem.*, **267**, 21319-21323.

[125] Eggertsen, G., Olin, M., Andersson, U., Ishida, H., Kubota, S., Hellman, U., Okuda, K.I. and Björkhem, I. (1996) *J. Biol. Chem.*, **271**, 32269-32275.

[126] Gåfvels, M., Olin, M., Chowdhary, B.P., Raudsepp, T., Andersson, U., Persson, B., Jansson, M., Björkhem, I. and Eggertsen, G. (1999) *Genomics*, **56**, 184-196.

[127] Hunt, M.C., Yang, Y.Z., Eggertsen, G., Carneheim, C.M., Gåfvels, M., Einarsson, C. and Alexson, S.E. (2000) *J. Biol. Chem.*, **275**, 28947-28953.

[128] Ishida, H., Kuruta, Y., Gotoh, O., Yamashita, C., Yoshida, Y. and Noshiro, M. (1999) *J. Biochem. (Tokyo)*, **126**, 19-25.

[129] Ishida, H., Yamashita, C., Kuruta, Y., Yoshida, Y. and Noshiro, M. (2000) *J. Biochem. (Tokyo)*, **12**, 57-64.

[130] Andersson, U., Yang, Y.Z., Björkhem, I., Einarsson, C., Eggertsen, G. and Gåfvels, M. (1999) *Biochim. Biophys. Acta*, **1438**, 167-174.

[131] Yang, Y., Eggertsen, G., Gåfvels, M., Andersson, U., Einarsson, C., Björkhem, I. and Chiang, J.Y. (2004) *Biochem. Biophys. Res. Commun.*, **320**, 1204-1210.

[132] del Castillo-Olivares, A. and Gil, G. (2002) *J. Biol. Chem.*, **277**, 6750-6757.

[133] Vlahcevic, Z.R., Eggertsen, G., Björkhem, I., Hylemon, P.B., Redford, K. and Pandak, W.M. (2000) *Gastroenterology*, **118**, 599-607.

[134] Zhang, M. and Chiang, J.Y. (2001) *J. Biol. Chem.*, **276**, 41690-41699.

[135] Jahan, A. and Chiang, J.Y. (2005) *Am. J. Physiol. Gastrointest. Liver Physiol.*, **288**, G685-695.

[136] Lundell, K. and Wikvall, K. (2003) *Biochim. Biophys. Acta*, **1634**, 86-96.

[137] Kuramoto, T., Miyamoto, J., Konishi, M., Hoshita, T., Masui, T. and Une, M. (2000) *Biol. Pharm. Bull.*, **23**, 1143–1146.

[138] Wikvall, K. (1993) In *Handbook of Experimental Pharmacology*, Vol. 105 Cytochrome P450 (Schenkman J.B. and Greim, H., eds.) pp. 705-718, Springer-Verlag, Berlin.

[139] Dahlbäck, H. and Wikvall, K. (1988) *Biochem. J.*, **252**, 207-213.

[140] Dahlbäck, H. (1988) *Biochem. Biophys. Res. Commun.*, **157**, 30-36. Erratum In: *Biochem. Biophys. Res. Commun.* (1989) **59**, 370.

[141] Okuda, K., Masumoto, O. and Ohyama, Y. (1988) *J. Biol. Chem.*, **263**, 18138-18142.

[142] Bergman, T. and Postlind, H. (1991) *Biochem. J.*, **276**, 427-432.

[143] Andersson, S., Davis, D.L., Dahlbäck, H., Jörnvall, H. and Russell, D.W. (1989) *J. Biol. Chem.*, **264**, 8222-8229.

[144] Usui, E., Nishiro, M., Ohyama, Y. and Okuda, K. (1990) *FEBS Lett.*, **274**, 175-177.

[145] Postlind, H., Hosseinpour, F., Norlin, M. and Wikvall, K. (2000) *Biochem. J.*, **347**, 349-356.

[146] Axén, E., Postlind, H., Sjöberg, H. and Wikvall, K. (1994) *Proc. Natl. Acad. Sci. USA*, **19**, 10014-10018.

[147] Pikuleva, I.A., Björkhem, I. and Waterman, M.R. (1997) *Arch. Biochem. Biophys.*, **343**, 123-130.

[148] Pikuleva, I. and Javitt, N.B. (2003) *Arch. Biochem. Biophys.*, **420**, 35-39.

[149] Sawada, N., Sakaki, T., Ohta, M. and Inouye, K. (2000) *Biochem. Biophys. Res. Commun.*, **273**, 977-984.

[150] Araya, Z., Hosseinpour, F., Bodin, K. and Wikvall, K. (2003) *Biochim. Biophys. Acta*, **1632**, 40-47.

[151] Parish, E.J., Parish, S.C. and Li, S. (1995) *Lipids*, **30**, 247-251.

[152] Fu, X., Menke, J.G., Chen, Y., Zhou, G., MacNaul, K.L., Wright, S.D., Sparrow, C.P. and Lund, E.G. (2001) *J. Biol. Chem.*, **276**, 38378-38387.

[153] Lund, E., Andersson, O., Zhang, J., Babiker, A., Ahlborg, G., Diczfalusy, U., Einarsson, K., Sjövall, J. and Björkhem, I. (1996) *Arterioscler. Thromb. Vasc. Biol.*, **16**, 208-212.

[154] Axelson, M. and Larsson, O. (1995) *J. Biol. Chem.*, **270**, 15102-15110.

[155] Rosen, H., Reshef, A., Maeda, N., Lippoldt, A., Shpizen, S., Triger, L., Eggertsen, G., Björkhem, I. and Leitersdorf, E. (1998) *J. Biol. Chem.*, **273**, 14805-14812.

[156] Vlahcevic, Z.R., Jairath, S.K., Heuman, D.M., Stravitz, R.T., Hylemon, P.B., Avadhani, N.G. and Pandak, W.M. (1996) *Am. J. Physiol.*, **270**, G646-652.

[157] Xu, G., Salen, G., Shefer, S., Tint, G.S., Nguyen, L.B., Parker, T.T., Chen, T.S., Roberts, J., Kong, X. and Greenblatt, D. (1998) *J. Lipid Res.*, **39**, 1608-1615.

[158] Araya, Z., Sjöberg, H. and Wikvall, K. (1995) *Biochem. Biophys. Res. Commun.*, **216**, 868-873.

[159] Schwarz, M., Russell, D.W., Dietschy, J.M., and Turley, S.D. (2001) *J. Lipid Res.*, **42**, 1594-1603.

[160] Stravitz, R.T., Vlahcevic, Z.R., Russell, T.L., Heizer, M.L., Avadhani, N.G. and Hylemon, P.B. (1996) *J. Steroid Biochem. Mol. Biol.*, **57**, 337-347.

[161] Twisk, J., de Wit, E.C.M. and Princen, H.M.G. (1995) *Biochem. J.*, **305**, 505-511.

[162] Rao, Y.P., Vlahcevic, Z.R., Stravitz, R.T., Mallonee, D.H., Mullick, J., Avadhani, N.G. and Hylemon, P.B. (1999) *J. Steroid Biochem. Mol. Biol.*, **70**, 1-14.

[163] Björkhem, I., Araya, Z., Rudling, M., Angelin, B., Einarsson, C. and Wikvall, K. (2002) *J. Biol. Chem.*, **277**, 26804-26807.

[164] Twisk, J., Hoekman, M.F., Lehmann, E.M., Meijer, P., Mager, W.H. and Princen, H.M. (1995) *Hepatology*, **21**, 501-510.

[165] Araya, Z., Tang, W. and Wikvall, K. (2003) *Biochem. J.*, **372**, 529-534.

[166] Hanson, M., Wikvall, K. and Babiker, A. (2005) *Biochim. Biophys. Acta*, **1687**, 44-51.

[167] Shanahan, C.M., Carpenter, K.L. and Cary, N.R. (2001) *Atherosclerosis*, **154**, 269-276.

[168] Prosser, D.E., Guo, Y., Jia, Z. and Jones, G. (2006) *Biophys. J.*, **90**, 3389-3409.

[169] Araya, Z. and Wikvall, K. (1999) *Biochim. Biophys. Acta*, 1438, 47-54.

[170] Staudinger, J.L., Goodwin, B., Jones, S.A., Hawkins-Brown, D., MacKenzie, K.I., LaTour, A., Liu, Y., Klaassen, C.D., Brown, K.K., Reinhard, J., Willson, T.M., Koller, B.H. and Kliewer, S.A. (2001) *Proc. Natl. Acad. Sci. USA*, 98, 3369-3374.

[171] Xie, W., Radominska-Pandya, A., Shi, Y., Simon, C.M., Nelson, M.C., Ong, E.S., Waxman, D.J. and Evans, R.M. (2001) *Proc. Natl. Acad. Sci. USA*, 98, 3375-3380.

[172] Dussault, I., Yoo, H.D., Lin, M., Wang, E., Fan, M., Batta, A.K., Salen, G., Erickson, S.K. and Forman, B.M. (2003) *Proc. Natl. Acad. Sci. USA*, 100, 833-838.

[173] Goodwin, B., Gauthier, K.C., Umetani, M., Watson, M.A., Lochansky, M.I., Collins, J.L., Leitersdorf, E., Mangelsdorf, D.J., Kliewer, S.A. and Repa, J.J. (2003) *Proc. Natl. Acad. Sci. USA*, 100, 223-228.

[174] Bodin, K., Lindbom, U. and Diczfalussy, U. (2005) *Biochim. Biophys. Acta*, 1687, 84-93.

[175] Björkhem, I. (2002) *J. Clin. Invest.*, 110, 725-730.

[176] Russell, D.W. (2000) *Biochim. Biophys. Acta*, 1529, 126-135.

[177] Lund, E., Björkhem, I., Furster, C. and Wikvall, K. (1993) *Biochim. Biophys. Acta*, 1166, 177-182.

[178] Lund, E.G., Guileyardo, J.M. and Russell, D.W. (1999) *Proc. Natl. Acad. Sci. USA*, 96, 7238-7243.

[179] Mast, N., Norcross, R., Andersson, U., Shou, M., Nakayama, K., Björkhem, I. and Pikuleva, I.A. (2003) *Biochemistry*, 42, 14284-13292.

[180] Lund, E.G., Kerr, T.A., Sakai, J., Li, W.P. and Russell, D.W. (1998) *J. Biol. Chem.*, 273, 34316-34327.

[181] Martin, K.O., Reiss, A.B., Lathe, R. and Javitt, N.B. (1997) *J. Lipid Res.*, 38, 1053-1058.

[182] Björkhem, I. and Boberg, K.M. (2001) In *The Metabolic Basis of Inherited Disease*. (Scriver, C.R., Beaudet, A.L., Sly, W.S. and Valle, D. Eds.) pp. 2961-2988, McGraw-Hill Inc., New York.

[183] Bove, K.E. (2000) *Clin. Liver Dis.*, 4, 831-848.

[184] Bove, K.E., Daugherty, C.C., Tyson, W., Mierau, G., Heubi, J.E., Balistreri, W.F. and Setchell, K.D. (2000) *Pediatr. Dev. Pathol.*, 3, 1-16.

[185] Bove, K.E., Heubi, J.E., Balistreri, W.F. and Setchell, K.D. (2004) *Pediatr. Dev. Pathol.*, 7, 315-334.

[186] Pullinger, C.R., Eng, C., Salen, G., Shefer, S., Batta, A.K., Erickson, S.K., Verhagen, A., Rivera, C.R., Mulvihill, S.J., Malloy, M.J. and Kane, J.P. (2002) *J. Clin. Invest.*, 110, 109-117.

[187] Clayton, P.T., Leonard, J.V., Lawson, A.M., Setchell, K.D.R., Andersson, S., Egestad, B. and Sjövall, J. (1987) *J. Clin. Invest.*, 79, 1031-1038.

[188] Cali, J.J., Hsieh, C.L., Francke, U. and Russell, D.W. (1991) *J. Biol. Chem.*, 266, 7779-7783.

[189] Rystedt, E., Olin, M., Seyama, Y., Buchmann, M., Berstad, A., Eggertsen, G. and Björkhem, I. (2002) *J. Intern. Med.*, 252, 259-264.

[190] Moghadasian, M.H., Salen, G., Frohlich, J.J. and Scudamore, C.H. (2002) *Arch. Neurol.*, 59, 527-529.

[191] Honda, A., Salen, G., Matsuzaki, Y., Batta, A.K., Xu, G., Leitersdorf, E., Tint, G.S., Erickson, S.K., Tanaka, N. and Shefer, S. (2001) *J. Lipid Res.*, 42, 291-300.

[192] Honda, A., Salen, G., Matsuzaki, Y., Batta, A.K., Xu, G., Hirayama, T., Tint, G.S., Doy, M. and Shefer, S. (2005) *J. Lipid Res.*, 46, 287-296.

Copyright of Current Molecular Medicine is the property of Bentham Science Publishers Ltd. and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.
