
# T-lymphocyte navigation and migration: beyond the PI3K paradigm

**L. Smith, A. Webb and S.G. Ward¹**

Department of Pharmacy and Pharmacology, University of Bath, Claverton Down, Bath BA2 7AY, U.K.

---

## Abstract

The co-ordinated and directional trafficking of T-lymphocytes in lymphoid and peripheral tissues is an important process in lymphoid development, immunosurveillance and immune responses. Members of the chemokine superfamily play a key role in providing navigational cues for T-cells and chemokine receptors couple with a wide range of biochemical signals including phosphoinositide lipid metabolism, elevation of intracellular calcium levels, activation of a wide array of protein kinases as well as small GTPases. One of the most robust biochemical signals elicited by chemokines in T-lymphocytes is the activation of several members of the PI3K (phosphoinositide 3-kinase) family. In many cell systems, PI3Ks are known to contribute to several aspects of the migratory machinery, although their role in T-cell migration has been unclear and will be considered in the present paper.

---

## Introduction

Trafficking of T-lymphocytes in lymphoid and peripheral tissues during immunosurveillance and immune responses involves a multi-step adhesion cascade of selectin- and integrin- (and their corresponding vascular ligands) supported rolling, chemokine-mediated integrin activation and firm integrin-mediated adhesion to the microvasculature endothelium followed by transmigration through the vessel wall and further migration in extravascular tissue [1,2]. The process of migration requires the cell to be polarized with the molecular processes at the front (leading edge) of a moving cell being distinct from those at the back (uropod). Establishing and maintaining cell polarity in response to extracellular stimuli appears to be mediated by a set of interlinked positive-feedback loops involving PI3Ks (phosphoinositide 3-kinases), Rho family GTPases, integrins, microtubules and vesicular transport. The relative contributions of these various signals depend on the cell type and the specific stimulus [3]. These intracellular signals result in reorganization of the cytoskeleton and cell adhesion causing the cells to send out pseudopodia and crawl up the chemoattractant gradient.

the migratory machinery including gradient sensing, actin reorganization and general cell motility [3–5]. The concept of PI3K being a key player in gradient sensing and/or cellular polarization was originally cemented by reports that PI(3,4,5)P₃ (phosphatidylinositol 3,4,5-trisphosphate) accumulates at the leading edge after exposure of *Dictyostelium* to chemoattractant stimuli [6,7]. These seminal observations led to a model for cell guidance mechanisms, whereby gradient sensing depends on the balance between excitation (via PI3K) and inhibition [via either substrate depletion and/or the lipid phosphatase PTEN (phosphatase and tensin homologue deleted on chromosome 10)] processes. Hence, binding/activation sites for PI3Ks are generated by chemoattractant stimulation of *Dictyostelium*, which ultimately confines PI(3,4,5)P₃ to the leading edge of chemotaxing cells [8]. Moreover, PI(3,4,5)P₃ at the leading edge stimulates its own accumulation via activation of one or more Rho GTPases [Rac, Cdc42 (cell division cycle 42) and/or Rho] that in turn activate PI3K and increase PI(3,4,5)P₃ formation [9]. PI(3,4,5)P₃ is further restricted to the leading edge by exclusion of its metabolizing enzyme PTEN from this region. Although this model has gained some credence in the context of neutrophil migration, extrapolation of the *Dictyostelium* model looks increasingly tenuous when applied to mammalian cells [10].

---

## PI3K and cell migration: the popular paradigm

In several cell systems, PI3K-dependent signalling events have been demonstrated to contribute to several aspects of

---

## Evidence of a role for class I PI3Ks in T-lymphocyte migration

PI3K activation is a signalling event shared by most chemokine receptors expressed on T-cells (Table 1). Signalling via CXCR4 (CXC chemokine receptor 4) has been studied in most detail and SDF-1 (stromal-derived factor-1)/CXCL12 (CXC chemokine ligand 12) triggers a transient pertussis toxin-sensitive accumulation of PI(3,4,5)P₃ in the Jurkat leukaemic cell line [11]. Similar lipid accumulation in murine

---

**Key words:** cell migration, chemokine, GTPase, phosphoinositide 3-kinase (PI3K), protein kinase C (PKC), T-lymphocyte.

**Abbreviations used:** CCL, CC chemokine ligand; CCR4, CC chemokine receptor 4; CXCL12, CXC chemokine ligand 12; CXCR, CXC chemokine receptor; DAG, diacylglycerol; DOCK, downstream of Crk-180 homologue; GPCR, G-protein-coupled receptor; MTOC, microtubule organizing centre; PBL, peripheral blood-derived human T-lymphocyte; PDK-1, 3′-phosphoinositide-dependent kinase-1; PI3K, phosphoinositide 3-kinase; PI(3,4,5)P₃, phosphatidylinositol 3,4,5-trisphosphate; PKC, protein kinase C; PLC, phospholipase C; PTEN, phosphatase and tensin homologue deleted on chromosome 10; ROCK, Rho-associated kinase; TNFα, tumour necrosis factor α.

¹To whom correspondence should be addressed (email S.G.Ward@bath.ac.uk).

Table 1 | Activation of PI3K by lymphoid chemokine receptors and sensitivity of migratory responses to pharmacological and genetic manipulation

Activation of PI3K by chemokines that show promiscuity towards receptors is attributed to signalling via all the receptors bound by these ligands, even though such details have not necessarily been investigated. BLC, B-lymphocyte chemoattractant; IP-10: interferon-$\gamma$-inducible protein 10; I-TAC, interferon-$\gamma$-inducible T-cell $\alpha$-chemoattractant; KI, kinase-inactive; LARC, liver and activation-regulated chemokine; MCP-1, monocyte chemotactic protein-1; MDC, macrophage-derived chemokine; Mig, monokine induced by interferon-$\gamma$; MIP-1, macrophage inflammatory protein 1; n.d., not determined; TARC: thymus-and activation-related chemokine; TECK: thymus-expressed chemokine.

| Receptor | Expression | Ligand | PI3K activation | Effect of inhibitors on T-cell migration | Effect of gene targeting on T-cell migration | Reference(s) |
|----------|------------|--------|------------------|-------------------------------------------|---------------------------------------------|---------------|
| CCR1     | Effector T-cells | RANTES/CCL5 | Yes | Inhibition | PI3K$\gamma$$^{-/-}$ | [11] |
|          |            | MIP-1$\alpha$/CCL3 |  |  | p110$\delta$KI |  |
|          |            | MCP-2/CCL8 |  |  |  |  |
|          |            | MCP-3/CCL7 |  |  |  |  |
| CCR2     | Effector T-cells | MCP-1/CCL2 | Yes | n.d. |  |  |
|          |            | MCP-2/CCL8 |  |  |  |  |
|          |            | MCP-3/CCL7 |  |  |  |  |
|          |            | MCP-4/CCL13 |  |  |  |  |
| CCR3     | Effector T-cells (Th2) | Eotaxin-1/CCL11 | n.d. | n.d. |  | [47] |
|          |            | Eotaxin-2/CCL24 |  |  |  |  |
|          |            | Eotaxin-3/CCL26 |  |  |  |  |
|          |            | RANTES/CCL5 |  |  |  |  |
|          |            | MCP-2/CCL8 |  |  |  |  |
|          |            | MCP-3/CCL7 |  |  |  |  |
|          |            | MCP-4/CCL13 |  |  |  |  |
| CCR4     | Effector T-cells (Th2) | MDC/CCL22 | Yes | No effect |  | [16] |
|          |            | TARC/CCL17 | Yes | No effect |  |  |
| CCR5     | Effector T-cells (Th1) | RANTES/CCL5 | Yes | Inhibition |  | [11] |
|          |            | MIP-1$\alpha$/CCL3 |  |  |  |  |
|          |            | MIP-1$\beta$/CCL4 | Yes | n.d. |  |  |
|          |            | MCP-2/CCL8 |  |  |  |  |
|          |            | MCP-3/CCL7 |  |  |  |  |
| CCR6     | Effector T-cells, B-cells, memory T-cells | LARC/CCL20 | No | n.d. |  | [47] |
| CCR7     | Naive central memory B-cells, mature medullary thymocytes | SLC/CCL21 | Yes | No effect | Partial inhibition | [12,14] |
|          |            | ELC/CCL19 | Yes | No effect | No effect |  |
| CCR8     | Effector T-cells (Th2) | I-309/CCL1 | Yes | Partial inhibition |  | * |
| CCR9     | Memory T-cells, B-cells, immature thymocytes | TECK/CCL25 | Yes | n.d. |  | * |
| CXCR3    | Effector T-cells | I-TAC/CXCL11 | Yes | No effect |  | [17] |
|          |            | Mig/CXCL9 | Yes |  |  |  |
|          |            | IP-10/CXCL10 | Yes |  |  |  |
| CXCR4    | Naive memory T-cells, B-cells, thymocytes | SDF-1/CXCL12 | Yes | Inhibition of primary cells; no effect on activated cells | Partial inhibition | [22] |
|          |            |  |  |  | No effect |  |
| CXCR5    | Follicular B helper T-cells, B-cells | BLC/CXCL13 | Yes | Inhibition of B-cell migration |  | [22] |
| CXCR6    | Effector T-cells | CXCL16 | Yes | n.d. |  | * |

* L. Smith, E. Hickman, J. Westwick and S.G. Ward, unpublished work.
and human T-lymphocytes stimulated with ELC (Epstein–Barr virus-induced receptor ligand chemokine)/CCL19 (CC chemokine ligand 19) and SLC (secondary lymphoid tissue chemokine)/CCL21 has also been reported [12]. The use of PI3K inhibitors has also revealed that *in vitro* chemotaxis across synthetic membranes of PBLs (peripheral blood-derived human T-lymphocytes) and NK (natural killer) cells in response to CXCL12 [as well as other chemokines such as RANTES (regulated upon activation, normal T-cell expressed and secreted)/CCL5] can be severely attenuated by PI3K inhibitors [11]. During migration and chemotaxis, there are at least two chemokine-mediated processes operating, namely chemokine-mediated integrin activation to mediate cell adhesion and chemokine-induced chemotaxis (that is likely to be integrin-independent). There is some evidence that enhanced integrin mobility in chemokine-stimulated lymphocytes is dependent on PI3K activation [12]. However, rapid integrin activation on rolling lymphocytes does not generally depend on PI3K activation processes [13].

### PI3K isoforms provide dispensable signals for T-lymphocyte migration

In experiments that more closely mimic the physiological conditions encountered by lymphocytes undergoing transendothelial migration, PI3K activation appears to be a dispensable signal for chemokine-stimulated T-cell migration [12,14]. Furthermore, CCR4 (CC chemokine receptor 4)-ligand-stimulated migration of human Th2 differentiated cells to either CCR4 ligands [15,16] and CXCR3 ligand-stimulated migration of PBLs[17] across synthetic membranes in *in vitro* assays are also insensitive to PI3K inhibitors. Curiously, migration of freshly isolated CD4<sup>+</sup> or CD8<sup>+</sup> T-cells in response to CXCL12 in these *in vitro* assays is sensitive to inhibition of PI3K, suggesting that the utilization of PI3K in migration responses is dependent on cell activation and differentiation status (L. Smith, E. Hickman, J. Westwick and S.G. Ward, unpublished work).

### Insights from gene targeting: getting the context right

Chemokine interaction with GPCRs (G-protein-coupled receptors) on lymphocytes has been shown to depend predominantly on Gαi proteins [18]. This has led to the assumption that these receptors are coupled with the βγ-dependent p110γ isoform. Indeed, this does appear to be the case, although several chemokine receptors can additionally activate other PI3K isoforms [10,11]. Significant progress has been made in resolving the confusion concerning the specific role of PI3K in lymphocyte migration. This has been made possible by availability of mice deficient in the 110γ catalytic isoform as well as analysis of mice expressing a mutant, catalytically inactive class 1A p110δ isoform. The *in vitro* migration of p110γ-deficient CD4<sup>+</sup> and CD8<sup>+</sup> T-cells to CCL19 and CXCL12 and CCL21 is significantly (but incompletely) decreased compared with cells from wild-type mice. In contrast, T-cell responses were largely unaffected by p110δ deficiency [19]. Hence, in settings where T-cell migration required PI3K activation, the p110γ isoform appears to be the dominant isoform utilized. This correlates with observations that p110γ-selective inhibitors reduce numbers of CD4<sup>+</sup> memory T-cells in models of systemic lupus [20]. Interestingly, B-cell migration to chemokines was not significantly affected by p110γ deficiency, thus implicating involvement of other PI3K isoforms (most notably p110δ), or signalling pathways in B-cell migration [19,21]. The ability of p110δ to function downstream of chemokine receptors in lymphocyte chemotactic responses is consistent with the finding that a broad spectrum loss-of-function mutant that disrupts all class 1A catalytic isoforms reduced chemotactic responses of leukaemic T-cells to CXCL12 [22]. Together, these results indicate that individual lymphoid chemokine receptors will probably have a differing dependence on PI3K-dependent signals for achieving ordered migration.

The Rac GEF (guanine nucleotide-exchange factor) protein DOCK (downstream of Crk-180 homologue)-2 has been proposed to mediate a PI3K-independent pathway sufficient to elicit T-lymphocyte migration [23]. However, further analysis has revealed that optimal T-cell migration *in vitro* and *in vivo* in response to CCL21, CCL19 or CXCL12 is actually dependent on expression of both DOCK-2 and p110γ [21]. So, DOCK-2-deficient T-lymphocytes can mount a 110γ-dependent residual migratory response. There is also indirect evidence for cross-talk between PI3K-dependent signals and DOCK-2. Hence, in mammalian cells, the DOCK-2-related protein DOCK-180 forms a complex with the adaptors Crk and Elmo. This complex activates several Rac-dependent pathways including actin remodelling and cell migration and both Elmo and DOCK-180 contain structural domains that probably interact with PI(3,4,5)P<sub>3</sub> [24–27].

### Do class II PI3Ks play a role in cell migration?

Mammalian class II PI3Ks predominantly include the ubiquitously expressed PI3KC2α and PI3KC2β [28]. Several chemokine receptors have been reported to stimulate class II PI3K activity [11,28]. Pharmacological tools such as wortmannin and LY294002 have limited actions on the class II PI3Ks [29,30]. The wide use of these tools, coupled with their relative weakness against class II PI3Ks compared with class I PI3Ks, means that the biological role of these class II PI3Ks may have been seriously underestimated, particularly in the context of PI3K inhibitor-resistant T-cell migration described earlier [17]. Interestingly, class II PI3KC2β and its primary product PI3P (phosphatidylinositol 3-phosphate) have been implicated in the regulation of cell adhesion, actin reorganization and migration in response to lysophosphatidic acid-dependent cell migration and wound healing in non-immune cell systems [31,32]. The role of class II PI3Ks in T-lymphocyte migration responses to chemokines remains relatively unexplored and it is at present unclear how GPCRs couple with class II PI3Ks.
Figure 1 | Model of signalling events contributing to chemokine-mediated lymphocyte migration

The mechanism by which GPCRs couple with DOCK-2 is unclear but Rho GTPases are implicated in other systems. The γ isoform of p110 is likely to be the major PI3K isoform activated, but other class 1A p110 isoforms can contribute to PI(3,4,5)P₃ accumulation. Relative contribution of each pathway will vary according to cell activation state and subtype. See the text for further details.

![Diagram](attachment:diagram.png)

PKC (protein kinase C) isoforms contribute to T-lymphocyte migration

Activation of PLC (phospholipase C), calcium mobilization and activation of DAG (diacylglycerol)-dependent PKCs by chemokines have been proposed as regulators of cell adhesion and migration [33,34]. However, studies with mice deficient in PLCβ1 and PLCβ3 indicated that the PLC pathway is not required for chemotaxis in neutrophils, although its role in T-lymphocytes was not investigated [35]. Certainly, during LFA-1 (lymphocyte function-associated antigen 1)-mediated locomotion of activated T-cells, PKCβ1 and PKCδ associate with the microtubules in the uropod, the trailing extension of the migrating T-cells. Curiously, while PKCδ is associated with the MTOC (microtubule organizing centre) but not the microtubules, PKCβ1 is located to the MTOC and along the microtubules in the trailing cell extensions [36]. This may be indicative of discrete functions for individual PKC isoforms in T-cell migration.

PKCδ has been implicated in cell motility and migratory responses in many non-immune cells [37]. In T-cell models where migration to CCR4 stimulation can occur independently of PI3K [15], use of pharmacological tools has indicated that PLC-dependent, calcium-independent mechanisms are required for directional migration of T-lymphocytes in response to the CCR4 ligands CCL17 and CCL22. This points towards the involvement of receptor-regulated DAG effectors in migratory responses to CCR4 stimulation. Further pharmacological intervention implicated a role for DAG-dependent PKC isoforms in the CCR4 migratory response, in particular PKCδ [16]. Activation of both PKCβ and PKCδ is known to involve the so-called ‘master kinase’ PDK-1 (3′-phosphoinositide-dependent kinase-1) which phosphorylates several PKC isoforms in their activation loop, which is necessary for their catalytic activation [38]. It is worth noting that PKCδ is often a functional enzyme in the absence of phosphorylation in the activation loop and so can be activated independently of PDK-1 and PI3K [38]. In this regard, PKCδ contains multiple sites for tyrosine phosphorylation, some of which have been demonstrated to influence its activation [39]. It is not well understood how PKC isoforms regulate cell

motility/migration, although it is likely that PKC isoforms exert effects on actin reorganization/polymerization [40,41] as well as cause changes in integrin affinity and lateral mobility [42]. There is strong pharmacological and genetic evidence that Rho and its effector ROCK (Rho-associated kinase) signalling is a key component of T-cell migration and adhesion in response to several chemokines in mature T-cells and thymocytes [15,43,44]. Activation of Rho/ROCK has been implicated in the formation of the uropod and myosin assembly in other systems and the effects on integrin adhesion observed in lymphocytes appear to occur at least in part via activation of the atypical ζ isoform of PKC, another known substrate for PDK-1.

# Concluding remarks

There are several limitations in the strategies that have been employed to date, to understand the relative contribution of PI3K catalytic isoforms to T-lymphocyte migrational responses to chemokines (summarized in Figure 1). These include reliance on the use of transformed cell lines, use of inhibitors that do not adequately discriminate between catalytic isoforms as well as the failure to adequately assess the role of individual class I and class II PI3K isoforms in primary human T-cell migration. While PI3K activation seems to be a conserved biochemical response common to most chemokines, it is now clear that this can be a dispensable signal for directional migration of T-cells [10,15,17,45]. The relative involvement of individual pathways in chemokine-stimulated migration of T-cells is likely to be context-dependent, being determined by individual receptor ligands, cell type (or subtype, e.g. Th1 compared with Th2) and cell status (e.g. naïve or activated). In TNFα (tumour necrosis factor α)-primed human neutrophils for example, fMet-Leu-Phe (N-formylmethionyl-leucylphenylalanine; ‘chemotactic peptide’) induces a biphasic increase in PI(3,4,5)P₃ accumulation, in which the first phase is dependent on the p110γ isoform [46]. The second phase of PI(3,4,5)P₃ accumulation is dependent on prior exposure to TNFα and is driven predominantly by p110δ [46]. The balance of signalling via individual PI3K isoforms compared with naïve cells may therefore be altered in activated T-cells that have been exposed to cytokines.

The differing dependence of individual chemokine receptors on PI3K isoforms in different cell types and at different stages of activation, makes it difficult to design a ‘one fits all’ drug to inhibit inflammatory recruitment of cells. Given the current interest in targeting PI3K isoforms to block migration of leucocytes to the site of inflammatory lesion, it is essential that we have complete understanding of the mechanisms guiding T-lymphocyte migration in the future. This may open new avenues for potential therapeutic interventions in inflammatory and autoimmune disease settings.

# References

1. von Andrian, U.H. and Mempel, T.R. (2003) Nat. Rev. Immunol. **3**, 867–878

2. Miyasaka, M. and Tanaka, T. (2004) Nat. Rev. Immunol. **4**, 360–370
3. Ridley, A.J., Schwartz, M.A., Burridge, K., Firtel, R.A., Ginsberg, M.H., Borisy, G., Parsons, J.T. and Horwitz, A.R. (2003) Science **302**, 1704–1709
4. Merlot, S. and Firtel, R.A. (2003) J. Cell Sci. **116**, 3471–3478
5. Hogg, N., Laschinger, M., Giles, K. and McDowall, A. (2003) J. Cell Sci. **116**, 4695–4705
6. Iijima, M. and Devreotes, P. (2002) Cell **109**, 599–610
7. Funamoto, S., Meili, R., Lee, S., Parry, L. and Firtel, R.A. (2002) Cell **109**, 611–623
8. Huang, Y.E., Iijima, M., Parent, C.A., Funamoto, S., Firtel, R.A. and Devreotes, P. (2003) Mol. Biol. Cell **14**, 1913–1922
9. Weiner, O.D., Neilsen, P.O., Prestwich, G.D., Kirschner, M.W., Cantley, L.C. and Bourne, H.R. (2002) Nat. Cell Biol. **4**, 509–513
10. Ward, S.G. (2004) Trends Immunol. **25**, 67–74
11. Sotsios, Y. and Ward, S.G. (2000) Immunol. Rev. **177**, 217–235
12. Constantin, G., Majeed, M., Giagulli, C., Picio, L., Kim, J.Y., Butcher, E. and Laudanna, C. (2000) Immunity **13**, 759–769
13. Grabovsky, V., Feigelson, S., Chen, C., Bleijs, D.A., Peled, A., Cinnamon, G., Baleux, F., Arenzana-Seisdedos, F., Lapidot, T., van Kooyk, Y. et al. (2000) J. Exp. Med. **192**, 495–506
14. Cinnamon, G., Shinder, V. and Alon, R. (2001) Nat. Immunol. **2**, 515–521
15. Cronshaw, D.G., Owen, C., Brown, Z. and Ward, S.G. (2004) J. Immunol. **172**, 7761–7770
16. Cronshaw, D.G., Kouroumalis, A., Parry, R., Webb, A., Brown, Z. and Ward, S.G. (2006) J. Leukocyte Biol. **79**, 1369–1380
17. Smit, M.J., Verdijk, P., van der Raaij-Helmer, E.M., Navis, M., Hensbergen, P.J., Leurs, R. and Tensen, C.P. (2003) Blood **102**, 1959–1965
18. Han, S.B., Moratz, C., Huang, N.N., Kelsall, B., Cho, H., Shi, C.S., Schwartz, O. and Kehrl, J.H. (2005) Immunity **22**, 343–354
19. Reif, K., Okkenhaug, K., Sasaki, T., Penninger, J.M., Vanhaesebroeck, B. and Cyster, J.G. (2004) J. Immunol. **173**, 2236–2240
20. Barber, D.F., Bartolome, A., Hernandez, C., Flores, J.M., Redondo, C., Fernandez-Arias, C., Camps, M., Ruckle, T., Schwarz, M.K., Rodriguez, S. et al. (2005) Nat. Med. **11**, 933–935
21. Nombela-Arrieta, C., Lacalle, R.A., Montoya, M.C., Kunisaki, Y., Megias, D., Marques, M., Carrera, A.C., Manes, S., Fukui, Y., Martinez, A.C. and Stein, J.V. (2004) Immunity **21**, 429–441
22. Curnock, A.P., Sotsios, Y., Wright, K.L. and Ward, S.G. (2003) J. Immunol. **170**, 4021–4030
23. Fukui, Y., Hashimoto, O., Sanui, T., Oono, T., Koga, H., Abe, M., Inatoshi, A., Noda, M., Oike, M., Shirai, T. and Sasazuki, T. (2001) Nature **12**, 826–830
24. Reif, K. and Cyster, J.G. (2002) Trends Cell Biol. **12**, 368–373
25. Sanui, T., Inayoshi, A., Noda, M., Iwata, E., Stein, J.V., Sasazuki, T. and Fukui, Y. (2003) Blood **102**, 2948–2950
26. Lemmon, M.A. and Ferguson, K.M. (2000) Biochem. J. **350**, 1–18
27. Cote, J.F., Motoyama, A.B., Bush, J.A. and Vuori, K. (2005) Nat. Cell Biol. **7**, 797–807
28. Foster, F.M., Traer, C.J., Abraham, S.M. and Fry, M.J. (2003) J. Cell Sci. **116**, 3037–3040
29. Brown, R.A., Domin, J., Arcaro, A., Waterfield, M.D. and Shepherd, P.R. (1999) J. Biol. Chem. **274**, 14529–14532
30. Domin, J., Pages, F., Volinia, S., Rittenhouse, S.E., Zvelebil, M.J., Stein, R.C. and Waterfield, M.D. (1997) Biochem. J. **326**, 139–147
31. Maffucci, T., Cooke, F.T., Foster, F.M., Traer, C.J., Fry, M.J. and Falasca, M. (2005) J. Cell Biol. **169**, 789–799
32. Domin, J., Harper, L., Aubyn, D., Wheeler, M., Florey, O., Haskard, D., Yuan, M. and Zicha, D. (2005) J. Cell. Physiol. **205**, 452–462
33. Tan, S.L. and Parker, P.J. (2003) Biochem. J. **376**, 545–552
34. Ward, S.G., Bacon, K. and Westwick, J. (1998) Immunity **9**, 1–11
35. Li, Z., Jiang, H., Xie, W., Zhang, Z., Smrcka, A.V. and Wu, D. (2000) Science **287**, 1046–1049
36. Volkov, Y., Long, A., McGrath, S., Ni Eidhin, D. and Kelleher, D. (2001) Nat. Immunol. **2**, 508–514
37. Iwabu, A., Smith, K., Allen, F.D., Lauffenburger, D.A. and Wells, A. (2004) J. Biol. Chem. **279**, 14551–14560
38. Newton, A.C. (2003) Biochem. J. **370**, 361–371
39. Steinberg, S.F. (2004) Biochem. J. **384**, 449–459
40. Hartwig, J.H., Thelen, M., Rosen, A., Janmey, P.A., Nairn, A.C. and Aderem, A. (1992) Nature **356**, 618–622
41. Sasahara, Y., Rachid, R., Byrne, M.J., de la Fuente, M.A., Abraham, R.T., Ramesh, N. and Geha, R.S. (2002) Mol. Cell **10**, 1269–1281
42. Giagulli, C., Scarpini, E., Ottoboni, L., Narumiya, S., Butcher, E.C., Constantin, G. and Laudanna, C. (2004) Immunity **20**, 25–35

198 Biochemical Society Transactions (2007) Volume 35, part 2

43 Li, Z., Dong, X., Wang, Z., Liu, W., Deng, N., Ding, Y., Tang, L., Hla, T., Zeng, R., Li, L. and Wu, D. (2005) Nat. Cell Biol. **7**, 399–404  
44 Vielkind, S., Gallagher-Gambarelli, M., Gomez, M., Hinton, H.J. and Cantrell, D.A. (2005) J. Immunol. **175**, 350–357  
45 Lacalle, R.A., Gomez-Mouton, C., Barber, D.F., Jimenez-Baranda, S., Mira, E., Martinez, A.C., Carrera, A.C. and Manes, S. (2004) J. Cell Sci. **117**, 6207–6215  

46 Condliffe, A.M., Davidson, K., Anderson, K.E., Ellson, C.D., Crabbe, T., Okkenhaug, K., Vanhaesebroeck, B., Turner, M., Webb, L., Wymann, M.P. et al. (2005) Blood **106**, 1432–1440  
47 Tilton, B., Ho, L., Oberlin, E., Loetscher, P., Baleux, F., Clark-Lewis, I. and Thelen, M. (2000) J. Exp. Med. **192**, 313–324  

Received 20 November 2006

©2007 Biochemical Society
