
THE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 282, NO. 21, pp. 15812–15822, May 25, 2007  
© 2007 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in the U.S.A.

# Structural and Mutational Studies of Anthocyanin Malonyltransferases Establish the Features of BAHD Enzyme Catalysis*

Received for publication, January 23, 2007, and in revised form, March 1, 2007 Published, JBC Papers in Press, March 23, 2007, DOI 10.1074/jbc.M700638200

Hideaki Unno ${}^{1}$, Fumiko Ichimaida ${}^{1}$, Hirokazu Suzuki ${}^{1}$, Seiji Takahashi ${}^{1}$, Yoshikazu Tanaka ${}^{1}$, Atsushi Saito ${}^{2}$, Tokuzo Nishino ${}^{4}$, Masami Kusunoki ${}^{2}$, and Toru Nakayama ${}^{3}$

From the ${}^{\dagger}$Institute for Protein Research, Osaka University, 3-2 Yamada-oka, Suita, Osaka 565-0871, Japan, ${}^{S}$Department of Biomolecular Engineering, Graduate School of Engineering, Tohoku University, 6-6-11 Aobayama, Sendai, Miyagi 980-8579, Japan, and ${}^{\ddagger}$Suntory Research Center, Shimamoto-cho, Mishima-gun, Osaka 618-8503, Japan

---

The BAHD family is a class of acyl-CoA-dependent acyltransferases that are involved in plant secondary metabolism and show a diverse range of specificities for acyl acceptors. Anthocyanin acyltransferases make up an important class of the BAHD family and catalyze the acylation of anthocyanins that are responsible for most of the red-to-blue colors of flowers. Here, we describe crystallographic and mutational studies of three similar anthocyanin malonyltransferases from red chrysanthemum petals: anthocyanidin 3-O-glucoside-6″-O-malonyltransferase (Dm3MaT1), anthocyanidin 3-O-glucoside-3″, 6″-O-dimalonyltransferase (Dm3MaT2), and a homolog (Dm3MaT3). Mutational analyses revealed that seven amino acid residues in the N- and C-terminal regions are important for the differential acyl-acceptor specificity between Dm3MaT1 and Dm3MaT2. Crystallographic studies of Dm3MaT3 provided the first structure of a BAHD member, complexed with acyl-CoA, showing the detailed interactions between the enzyme and acyl-CoA molecules. The structure, combined with the results of mutational analyses, allowed us to identify the acyl-acceptor binding site of anthocyanin malonyltransferases, which is structurally different from the corresponding portion of vinorine synthase, another BAHD member, thus permitting the diversity of the acyl-acceptor specificity of BAHD family to be understood.

transferases plays versatile roles in the biosyntheses of these metabolites, making a very important contribution to the establishment of the structural and functional diversities of the metabolites (1). Although these acyltransferases show only low sequence similarities (15–30% identity) with each other, they share two highly conserved sequences, suggesting that they evolved from a common ancestor. Thus, a single protein family has been proposed for these diversified acyltransferases of biological and industrial significance and is referred to as the BAHD${}^{4}$ family (1–4).

Anthocyanin acyltransferases (AATs) form an important class of this family, catalyzing the transfer of an acyl group from acyl-CoA to a sugar moiety of an anthocyanin (2), a class of flavonoids that is the origin of most of the orange to blue colors of flowers and that plays important roles in plant reproduction and survival (5). The AAT-catalyzed acylation of anthocyanins is important for the stabilization, accumulation in vacuoles, and modulating the coloration of the pigments (2). It has been proposed that AAT catalysis proceeds through the formation of a ternary complex consisting of acyl-CoA, an anthocyanin, and the enzyme, where a general-base amino acid residue deprotonates a hydroxy group of the anthocyanin substrate, thereby promoting its nucleophilic attack on the carbonyl of the thioester of acyl-CoA (6).

The red color of the chrysanthemum (*Dendranthema × morifolium*) mainly arises from cyanidin 3-O-3″,6″-O-dimalonylglucoside (Fig. 1, compound 3), which contains two malonyl groups (7, 8). During the biosynthesis of the pigment the first malonylation takes place at the 6″-position of the cyanidin 3-O-glucoside (1) (termed 3MaT1 activity) and is catalyzed by either

---

Plants produce more than $2 \times 10^{5}$ types of secondary metabolites, many of which are of biomedical, pharmaceutical, and agricultural importance. A class of acyl-CoA-dependent acyl-

* This work was supported by the National Project on Protein Structure and Functional Analysis promoted by the Ministry of Education, Culture, Sports, Science and Technology of Japan. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
${}^{S}$ The on-line version of this article (available at http://www.jbc.org) contains supplemental Figs. 1S–3S.
The nucleotide sequence(s) reported in this paper has been submitted to the DDBJ/GenBank${}^{\mathrm{TM}}$/EBI Data Bank with accession number(s) AB290338.
The atomic coordinates and structure factors (code 2E1U, 2E1T, and 2E1V) have been deposited in the Protein Data Bank, Research Collaboratory for Structural Bioinformatics, Rutgers University, New Brunswick, NJ (http://www.rcsb.org/).
${}^{1}$ These authors are equal contributors.
${}^{2}$ To whom correspondence may be addressed. Tel. and Fax: 81-6-6879-8636; E-mail: kusunoki@protein.osaka-u.ac.jp.
${}^{3}$ To whom correspondence may be addressed. Tel. and Fax: 81-22-795-7270; E-mail: nakayama@seika.che.tohoku.ac.jp.

---

$^{4}$ The abbreviations used are: BAHD, benzyl alcohol acetyl-, anthocyanin-O-hydroxycinnamoyl-, anthranilate-N-hydroxycinnamoyl/benzoyl-, and deacetylvindoline acetyltransferase; AAT, anthocyanin acyltransferase; Dm3MaT1, malonyl-CoA:anthocyanidin 3-O-glucoside-6″-O-malonyltransferase of *D. x morifolium*; MaT, malonyltransferase; Dm3MaT2, malonyl-CoA:anthocyanidin 3-O-glucoside-3″,6″-O-dimalonyltransferase of *D. x morifolium*; Dm3MaT3, an AAT homolog of *D. x morifolium*; Gt5AT, hydroxycinnamoyl-CoA:anthocyanin 3,5-O-diglucoside 6″-O-hydroxycinamoyltransferase of *G. triflora*; Pf3AT, hydroxycinnamoyl-CoA:anthocyanin 3-O-glucoside 6″-O-hydroxycinamoyltransferase of *P. frutescens*; Ss5MaT1, malonyl-CoA:anthocyanin 5-O-glucoside-6″-O-malonyltransferase of *S. splendens*; Ss5MaT2, malonyl-CoA:anthocyanin 5-O-glucose-4″-O-malonyltransferase of *S. splendens*; Sc3MaT, malonyl-CoA:anthocyanidin 3-O-glucoside-6″-O-malonyltransferase of *S. cruentus*; kb, kilobase(s).

15812 JOURNAL OF BIOLOGICAL CHEMISTRY VOLUME 282 • NUMBER 21 • MAY 25, 2007

Structures of a BAHD Family Enzyme

EXPERIMENTAL PROCEDURES

**Materials**

Anthocyanins were generous gifts from Prof. Masa-atsu Yamaguchi, Minami-Kyushu University. All other chemicals used were of analytical grade or sequencing grade, as appropriate.

**Enzyme Assay**

The standard reaction mixture (50 μl) consisted of 20 mM potassium phosphate, pH 7.0, 60 μM malonyl-CoA, 120 μM cyanidin 3-O-glucoside, and enzyme. The mixture without enzyme was incubated at 30 °C for 10 min. The reaction was started by the addition of the enzyme. After incubation at 30 °C for 20 min, the reaction was stopped by adding 50 μl of ice-cold 0.5% (v/v) trifluoroacetic acid. Anthocyanins in the resulting mixture were analyzed by reversed-phase high performance liquid chromatography using a Shimadzu Prominence LC solution system (Shimadzu, Kyoto, Japan) with an Asahipak ODP-50 4E column (4.6 × 250 mm; Shoko, Tokyo, Japan). Chromatographic conditions were as follows: solvent A, 0.1% (v/v) trifluoroacetic acid; solvent B, 0.1% (v/v) trifluoroacetic acid and 90% (v/v) acetonitrile; column temperature, ambient; detection at 520 nm; flow rate, 0.7 ml/min. The column was equilibrated with 20% B before use. After injection, the column was initially developed with 20% B for 3 min followed by a linear gradient from 20 to 31% B in 15 min followed by that from 31 to 56% B in 1 min. The column was then washed with 56% B for 4 min followed by a linear gradient from 56 to 20% B in 1 min.

**Enzyme Kinetics**

Assays for the initial velocity for 3MaT1 and 3MaT2 activities were carried out under steady-state conditions using a standard assay system (see above) with varying concentrations of substrates. Typically, the anthocyanin and malonyl-CoA concentrations were varied in the range of 5–54 and 5–30 μM, respectively, in the assays for Dm3MaT1, Dm3MaT2, and their mutants. Double reciprocal plots of the activities of these AATs generally yielded a family of apparently parallel lines in the range of substrate concentrations employed, as has been observed for other AATs (6). The apparent kinetic parameters and their standard deviations were determined by fitting the initial velocity data by a nonlinear least squares method using a computer program (two-dimensional scientific graph-plotting tool program) to Equation 1, which best describes the observed parallel patterns of double reciprocal plots,

\[
\frac{1}{v} = \frac{K_{\text{mA}}}{(k_{\text{cat}}[E][A])} + \frac{(1 + K_{\text{mB}}/[B])}{k_{\text{cat}}[E]}
\]

where [A] and [B] denote the substrate concentrations, [E] denotes enzyme concentration, \( k_{\text{cat}} \) denotes the catalytic rate constant, and \( K_{\text{mA}} \) and \( K_{\text{mB}} \), respectively, denote [A] and [B].

Structures of a BAHD Family Enzyme

| Dm3MaT1 | MASNSIVTILEQSRISSPPGTIGERSLPLTFFDIGWVPFPPVHHVFFYRFPHSKSHFLET | 60 |
| Dm3MaT2 | MASNPIVTILEHSRISSPPGTIGERSLPLTFFDITWLAFPPVHHVFFYAFQHSHKSHFLET | 60 |
| Dm3MaT3 | MASLPILTVLEQSQVSPPPDTLGDKSLQLTFFDFFWLRSPPINNLFFYELPITRSQFTET | 60 |

| Dm3MaT1 | VVPNLKHSLSLALQHFFPFASNLYVSPNADDFGVIRKPEIRHVEGDYVALTFAECS-LDF | 119 |
| Dm3MaT2 | VVPNLKHSLSLTQHFFPFAGNLFVFPNADDYGVIRKPEIRHVEGDYVALTFAECPTLDF | 120 |
| Dm3MaT3 | VVPNIKHSLSTITLKHFPFVGKLVVYP------APTKKPEICYVEGDSVAVTFAECN- LDL | 114 |

| Dm3MaT1 | NDLTGNHPRKCENFYPLVPPLGNVVKMADCVTIPPLFSVQVTYFRDSGISIGMTNHHSLGD | 179 |
| Dm3MaT2 | NDLTGNHPRECENFHPLVPPLGDLVKMYDYVTIPLLSVQVTYFKDSGISIGMTNHTVAD | 180 |
| Dm3MaT3 | NELTGNHPRNCDKFYDLVIPILGESTRLSDCIKIPFLFSVQVTLFPNQGIAIGITNHCLGD | 174 |

| Dm3MaT1 | ASTRLGFLKVWTSIAKSGG-DQSLLMNGSLPVLDRLIDVPKLDEYRLRHTSLETF---YQ | 235 |
| Dm3MaT2 | ASTRLGFLKAWTSIAKSGR-DQSFVMNGSPPVLDRLIDPKLDEFRLRHTSLETL---YQ | 236 |
| Dm3MaT3 | ASTRFCFLKAWTSIARGSNNDESFLANGTRPLYDRIIKYPMLDEAYLKRAKVESFNEDYV | 234 |

| Dm3MaT1 | PPSLVGPTKKVRATFILSRTNINQLKKRVLTQIPTLEYISSFTVTCGYIWSCIAKSLVKM | 295 |
| Dm3MaT2 | PRSLVGPTKKVRATFILSRTNINQLKKRVLTQIPTLEYISSFTVTCGYIWSCIAKSLVKM | 296 |
| Dm3MaT3 | TQSLAGPSDKLRATFILTRAVINQLKDRVLAQLPTLEYVSSFTVACAYIWSCIAK------ | 289 |

| Dm3MaT1 | GEKKGEDELEQFICTADCRSRMDPIPI PSTYFGNCGAPCVTTIKNVVLSSENGVFAAKLI | 355 |
| Dm3MaT2 | GEKKGEDELEQFIISVDCRSRMNPIPI PSTYLGNCGAPCITTIKNVVLSSENGFVFTAKLI | 356 |
| Dm3MaT3 | ---SRNDKLQLFGFPIDRRARMKPIPI PTAYFGNCJGGCAAIAKTNLLIGKEGFITAAKLI | 346 |

| Dm3MaT1 | GEAINKMVKN-KEGILKDAER---WHDAFKIPARKIGVAGTPKLNFYIDD FGWKPKQNE | 411 |
| Dm3MaT2 | SEAINKMVKN-KEGILKDAER---WHEGFKIPARKIGVAGTPKLNFYIDD FGWKPKQYE | 413 |
| Dm3MaT3 | GENLHKTILTDYKDGVLKDDMESFNDLVSEGMPTMTWVSGTPKLRFYDMDFGWGKPKKLE | 406 |

| Dm3MaT1 | TISIDYNGSVAINASKESTQDFEIGLCFSNMQMEAFADIFNHGLESEI | 459 |
| Dm3MaT2 | AISIDYKGSVAINASKESTQDFEIGLCFPNMQMKAFADIFNHGFESEV | 460 |
| Dm3MaT3 | TVSIDHNGAISINSCKESNEDLEIGVCISATQMEDFVHIFDDGLKAYL | 454 |

FIGURE 2. Amino acid sequence comparison of Dm3MaT1, Dm3MaT2, and Dm3MaT3. Arrows above the Dm3MaT1 sequence show regions A, B, and C that were defined in region swapping studies. The box labeled with motif 2 shows the motif that is specifically conserved among AATs, and those labeled with motif 1 and motif 3 show motifs that are highly conserved among BAHD family of enzymes, including AATs. Filled circles below the Dm3MaT3 sequence indicate the location of amino acid residues corresponding to those important for 3MaT2 activity of Dm3MaT2 as revealed by mutational studies (see also Fig. 4). Amino acid residues that are different from those of Dm3MaT1 are shown in red.

which give a $k_{cat}[E]/2$ value in the presence of a saturating concentration of a counter substrate. Note that our previous product inhibition studies showed that the AAT-catalyzed malonyl transfer proceeds via a ternary complex mechanism (6), consistent with the results of the present crystallographic studies (see below). Under some circumstances, double reciprocal plots of ternary complex mechanisms are known to yield parallel patterns (10).

### Region Swapping and Site-specific Mutagenesis

The expression plasmids pQE-Dm3MaT1 and pQE-Dm3MaT2 (8) share unique restriction enzyme sites (SphI, PmlI, SpeI, and SalI in the order of 5′→3′ orientation) in each AAT sequence. Digestion of each of these plasmids with SphI-PmlI produces two DNA fragments that are 0.5 and 3.3 kb in size. The 0.5-kb fragment derived from pQE-Dm3MaT2 were ligated with the 3.3-kb fragment from pQE-Dm3MaT1 to produce the plasmid pQE-Dm3MaT(211), which expresses a Dm3MaT1 mutant in which region A is replaced with that of Dm3MaT2 (see Fig. 2 and supplemental Fig. 1S). Likewise, the 0.5-kb fragment obtained from pQE-Dm3MaT1 was ligated with the 3.3-kb fragment from pQE-Dm3MaT2 to produce a

plasmid expressing Dm3MaT(122). Plasmids expressing Dm3MaT(121) and Dm3MaT(212) were prepared by exchanging the PmlI/SpeI-digested DNA fragments of pQE-Dm3MaT1 and pQE-Dm3MaT2, and those expressing Dm3MaT(112) and Dm3MaT(221) were prepared by exchanging the SpeI/SalI-digested DNA fragments of pQE-Dm3MaT1 and pQE-Dm3MaT2. Escherichia coli JM109 cells were transformed with the resulting plasmids to overexpress each region-swapped mutant. The plasmids for the expression of site-specific mutants of Dm3MaT(212) was prepared by PCR using pQE-Dm3MaT(212) as a template and mutagenesis primers (not shown).

#### cDNA Cloning, Protein Expression, and Purification of Dm3MaT3

cDNA Cloning and Expression of Dm3MaT3—The Dm3MaT3 cDNA was isolated from red buds of *D. morifolium* essentially as described previously (8). The Dm3MaT3 cDNA was cloned into pBluescript SK−. From the resulting plasmid, the full-length Dm3MaT3 cDNA was amplified by PCR using the primers 5′-TGCGAGCATATGGCCTCTCTTC-3′ and 5′-CCCGGGGATCCTTAAGG-3′ to introduce NdeI and BamHI restriction sites (respective underlined sites) containing in-frame start and stop codons into the 5′- and 3′-ends of the cDNA, respectively. The amplified fragment was cloned into a pCR4Blunt-TOPO vector (Invitrogen) and sequenced to confirm the absence of PCR errors. The fragment was then digested with NdeI and BamHI and ligated with an expression vector, pET-15b (Novagen, Madison, WI), which had previously been digested with NdeI and BamHI. The resulting plasmid, pET-Dm3MaT3, which encodes an N-terminal in-frame fusion of Dm3MaT3 with a His₆ tag, was used to transform *E. coli* BL21(DE3) cells. After preculturing the transformant cells at 37 °C overnight in Luria-Bertani broth containing 50 μg/ml ampicillin, 10 ml of the culture was inoculated into 2000 ml of the same medium. After incubation at 30 °C until the $A_{600}$ reached 0.5, isopropyl 1-β-D-thiogalactoside was added to the broth to a final concentration of 0.4 mM followed by cultivation at 30 °C for 14 h.

Purification of Recombinant Dm3MaT3—All subsequent operations were conducted at 0–4 °C. The cells were harvested by centrifugation (5000 × g, 15 min) and resuspended in buffer H (20 mM potassium phosphate, pH 7.4, containing 15 mM 2-mercaptoethanol, 10 mM imidazole, and 0.5 M NaCl). The cells were disrupted at 4 °C by 10 cycles of ultrasonication

(Where one cycle corresponds to 10 kHz for 10 s followed by an interval of 50 s). Cell debris was removed by centrifugation (5000 × g, 15 min). Polyethyleneimine was slowly added to the supernatant solution to a final concentration of 0.3% (v/v). The mixture was allowed to stand at 4 °C for 30 min followed by centrifugation (8000 × g, 15 min) and filtration through a 0.22-μm filter. The supernatant was applied to a HisTrap HP column (5 ml; GE Healthcare) equilibrated with buffer H. The column was extensively washed with buffer H, and the enzyme was eluted with buffer H containing 200 mM imidazole. The active fractions were collected, concentrated, and equilibrated with buffer T (20 mM Tris-HCl, pH 7.4, containing 15 mM 2-mercaptoethanol) by ultrafiltration with an Amicon Ultra-15 centrifugal filter device (30,000 molecular weight cut-off; Millipore, Bedford, MA). The enzyme was then digested with thrombin (Novagen; 1 unit/mg Dm3MaT3 protein) at 4 °C for 16 h in 20 mM Tris-HCl, pH 8.4, containing 0.15 M NaCl and 2.5 mM CaCl₂ to remove the His₆ tag from the recombinant enzyme. The resulting mixture was applied to a HisTrap HP column (5 ml) equilibrated with buffer H. The column was extensively washed with buffer H, and the flow-through fractions, which contained Dm3MaT3, were collected. The enzyme solution was concentrated and equilibrated with buffer Q (10 mM potassium phosphate, pH 7.4, containing 15 mM 2-mercaptoethanol) by ultrafiltration. The resulting enzyme solution was applied to a HiTrap Q HP column (5 ml; GE Healthcare) equilibrated with buffer Q at a flow rate of 1 ml/min using ÄKTA purifier (GE Healthcare). The column was washed with the same buffer. The enzyme was eluted with a linear gradient of 0–600 mM NaCl in buffer Q. The enzyme solution was concentrated and equilibrated with buffer A (0.01 M Hepes-NaOH, pH 7.0, containing 15 mM 2-mercaptoethanol) as described above. Proteins were quantified by the method of Bradford using a kit (Bio-Rad) with bovine serum albumin as the standard.

For the production of selenomethionyl Dm3MaT3, *E. coli* BL21-CodonPlus(DE3)-RIPL cells (Stratagene, La Jolla, CA) harboring pET-Dm3MaT3 were grown in 10 liters of a minimal medium (Se-Met core medium; Wako, Tokyo, Japan) supplemented with L-selenomethionine (2.5 mg/ml, final concentration) at 35 °C. The selenomethionyl Dm3MaT3 was purified from crude extracts of the recombinant cells essentially as described above, except that an extra HiTrap Q chromatography was carried out before HisTrap column chromatography.

### Crystallization

Dm3MaT3 crystals were grown at 20 °C over a period of 1–3 weeks by the sitting-drop vapor-diffusion method. Dm3MaT3 crystals complexed with malonyl-CoA were obtained by mixing 2 μl of protein solution including 2 mM malonyl-CoA with 2 μl of reservoir solution (30% (w/v) polyethylene glycol 8000 (PEG8000, Hampton Research), 100 mM sodium cacodylate, pH 6.5, 120 mM ammonium sulfate, and 10% glycerol). Crystals of substrate-free native and selenomethionyl Dm3MaT3 were obtained by mixing 2 μl of the protein solution with 2 μl of reservoir solution (25% (w/v) polyethylene glycol 8000 (PEG8000, Hampton Research), 100 mM sodium cacodylate (pH 6.5), 100 mM ammonium sulfate, 10% glycerol, and 10% KCl). Crystals were frozen at 90 K.

---

### Structures of a BAHD Family Enzyme

#### Data Collection, Structure Solution, and Refinement

All data sets were collected on beamline BL-5A at the photon factory (KEK, Tsukuba, Japan). Measurements were performed at 90 K with λ = 0.97909 Å (Se peak), λ = 0.97928 Å (Se edge), λ = 0.95000 Å high-remote), and λ = 1.000 Å (native-free and native-complex). Complete data sets were collected through contiguous rotation ranges at a given wavelength before proceeding to the next wavelength. The data sets were processed and scaled with HKL2000 (11). All data sets belonged to space group P2₁ with two molecules per asymmetric unit. The Dm3MaT3 structure was solved by the multiwavelength anomalous diffraction method, with selenomethionyl Dm3MaT3 crystal.

Initial phases were determined by the SOLVE program (12). Phase improvement by density modification including non-crystallography symmetry averaging was performed by the RESOLVE program (13). A model was then obtained by automatic tracing with ARP/wARP program (14). The structure was then rebuilt by O (15) and refined using Refmac (16), with 5% of the data set aside as a free set. In subsequent refinement, the selenomethionyl Dm3MaT3 data set was replaced with native-free and native-complex data sets. Malonyl-CoA models were fitted into the substrate binding sites based on non-crystallography symmetry averaged difference electron density maps (Fig. 3C). Of 454 residues, 14 residues (N-terminal 5 residues, C-terminal 4 residues, and residues 365–369) in chains A and B are not visible in the electron density and are probably disordered. All figures were produced using POVSCRIPT+ (17), PyMOL (18), and RASTER3D (19). The Ca atoms of native-complex of Dm3MaT3 and vinorine synthase structures were superimposed with the program LSQKAB (20).

---

### RESULTS AND DISCUSSION

#### Region Swapping and Site-specific Mutagenesis of Dm3MaT1 and Dm3MaT2—To explore the amino acid residues responsible for the difference in the acyl-acceptor specificity of malonyl transfer between Dm3MaT1 and Dm3MaT2, we first carried out region swapping between these enzymes. Namely, regions A, B, and C of these two enzymes (see Fig. 2) were shuffled to obtain six swapped mutants: Dm3MaT(112), Dm3MaT(121), Dm3MaT(211), Dm3MaT(122), Dm3MaT(212), and Dm3MaT(221) (see also supplemental Fig. 1S). In the Dm3MaT(112) mutant, for example, regions A and B come from Dm3MaT1, and region C comes from Dm3MaT2. These mutants were examined for their 3MaT1 and 3MaT2 activities (Table 1; see also Fig. 1). The results showed that Dm3MaT(122), a Dm3MaT2 mutant containing region A of Dm3MaT1, as well as Dm3MaT(121) and Dm3MaT(112) displayed no appreciable 3MaT2 activity, although they showed 3MaT1 activities that were comparable with that of Dm3MaT1. These results suggest that region A of Dm3MaT2 contains an amino acid residue(s) that is important for 3MaT2 activity. Dm3MaT(211) also did not show 3MaT2 activity, suggesting that region B and/or C of Dm3MaT2 may also contain an amino acid residue(s) that is important for 3MaT2 activity. A comparison of the 3MaT2 activities of Dm3MaT(212) and Dm3MaT(221) showed that the apparent $k_{\text{cat}}/K_m$ value for 3MaT2 activity of the former mutant was significant.

Structures of a BAHD Family Enzyme

(A)  
(C)

(B)

amino acid residues unique to Dm3MaT2 in domain C except for Tyr-411, Ala-413, and Lys-419 were replaced by those that were unique to Dm3MaT1, displayed a significant level of 3MaT2 activity (Table 1). These results indicate the importance of Tyr-411, Ala-413, and Lys-419 for the 3MaT2 activity of Dm3MaT2. Considering the above findings, we concluded that Thr-35, Leu-37, Ala-38, Gln-51, Tyr-411, Ala-413, and Lys-419 are important for 3MaT2 activity of Dm3MaT2. This conclusion was further supported by the fact that mutant M7, 7-fold mutant of Dm3MaT1 (G35T/V37L/P38A/ P51Q/N410Y/T412A/N418K), displayed a significant level of 3MaT2 activity (Table 1).

Phylogenetics, Specificity, and Biochemical Properties of Dm3MaT3—During the course of our attempts to find the structural basis of the above conclusion, we identified a novel AAT homolog, Dm3MaT3, from red chrysanthemum petals, which unlike Dm3MaT1 and Dm3MaT2 yields protein crystals suitable for x-ray crystallographic studies (for details of the isolation and expression of Dm3MaT3 cDNA, see “Experimental Procedures”). The Dm3MaT3 cDNA coded for a protein of 454 amino acids with a calculated molecular weight of 50,805. The deduced amino acid sequence of Dm3MaT3 (DDBJ accession number, AB290338; Fig. 2) showed the highest sequence similarity to those of Dm3MaT1 (55% identity) and Dm3MaT2 (53%) as well as the malonyl-CoA:anthocyanidin 3-O-glucoside-6"-O-malonyltransferases of other Compositae plants, such as those of Senecio cruentus (Sc3MaT) (21) (54%) and Dahlia variabilis (Dv3MaT) (22) (52%). It also shows a 40% sequence identity to malonyl-CoA:anthocyanin 5-O-glucoside-6"-O-malonyltransferase of Salvia splendens petals (Ss5MaT1) (23). The primary structure of Dm3MaT1 contains sequences that are absolutely conserved among all members of the BAHD family: His-Xaa₃-Asp (motif 1) and Asp-Phe-Gly-Trp-Gly (motif 3) (1, 2) (Fig. 2). It also contains the sequence that is uniquely conserved among the AATs: Tyr-Phe-Gly-Asn-Cys (motif 2) (2). As expected, a molecular phylogenetic analysis clearly indicated that Dm3MaT3 belongs to the AAT-related subfamily and is most closely related to Compositae anthocyanin malonyltransferases (see supplemental Fig. 2S).

The Dm3MaT3 protein was expressed under the control of a T5 promoter in E. coli BL21(DE3) cells, and the AAT activities of the expressed product were examined. The recombinant

FIGURE 3. Structures of Dm3MaT3 complexed with malonyl-CoA. A, ribbon diagram showing the tracing of the main chain of Dm3MaT3 in complex with malonyl-CoA. Helices and strands are shown as ribbons in green and yellow, respectively, and are numbered from the N terminus, whereas loops are shown as thin lines. Bound malonyl-CoA molecule is shown as sticks. B, a stereo view of Dm3MaT3 molecule complexed with malonyl-CoA. The locations of motifs 1, 2, and 3, each of which forms a part of a loop, is marked in red. C, model and non-crystallography symmetry averaged $2F_{o} - F_{c}$ electron density map of malonyl-CoA. The electron map (green) is contoured at $1\sigma$. Carbon, oxygen, nitrogen, phosphorus, and sulfur atoms are colored white, red, blue, orange, and yellow, respectively.

icantly higher than that of the latter, leading us to propose that region C, but not region B, contains an amino acid residue(s) that is important for 3MaT2 activity.

To identify the amino acid residues that are important for 3MaT2 activity in regions A and C, 9 site-specific mutants of Dm3MaT(212) were created in which each or a set of amino acid residues unique to Dm3MaT2 in regions A and C of Dm3MaT(212) were replaced by those unique to Dm3MaT1 (see Table 1), and their 3MaT and 3MaT2 activities were examined. The results showed that the following single amino acid substitutions in region A resulted in a significant loss of 3MaT2 activity of Dm3MaT(212) (relative $k_{cat}/K_{m}$ for acyl acceptor, <20%): T35L, L37V, A38P, and Q51P. However, M8 mutant, which is an 8-fold Dm3MaT(212) mutant containing P5S, H11Q, A49R, T72A, G81S, F84Y, F86S, and Y92F in region A, displayed a considerable level of 3MaT2 activity. These results indicate that Thr-35, Leu-37, Ala-38, and Gln-51 of Dm3MaT2 are important for 3MaT2 activity. As for region C, each of the Y411N, A413T, and K419N substitutions in Dm3MaT(212) resulted in a significant loss of 3MaT2 activity (relative $k_{cat}/K_{m}$ for acyl acceptor, <20%), whereas the 14-fold mutant of Dm3MaT(212), mutant CM14, where all of the

15816 JOURNAL OF BIOLOGICAL CHEMISTRY
VOLUME 282 • NUMBER 21 • MAY 25, 2007

Structures of a BAHD Family Enzyme

TABLE 1  
3MaT1 and 3MaT2 activities of mutants  

The apparent kinetic parameters and their standard deviations were determined as described under Experimental Procedures. Values in parentheses indicate the relative percentage of the $k_{cat}/K_m$ value for 3MaT2 activity, with that of Dm3MaT(212) taken to be 100%.

| Enzyme                     | $k_{cat}$ ($s^{-1}$) | $k_{cat}/K_m$ for acyl donor ($s^{-1} \mu M^{-1}$) | $k_{cat}/K_m$ for acyl acceptor ($s^{-1} \mu M^{-1}$) | $k_{cat}$ ($s^{-1}$) | $k_{cat}/K_m$ for acyl donor ($s^{-1} mM^{-1}$) | $k_{cat}/K_m$ for acyl acceptor ($s^{-1} mM^{-1}$) |
|----------------------------|-----------------------|----------------------------------------------------|-----------------------------------------------------|-----------------------|-------------------------------------------------|-----------------------------------------------------|
| Dm3MaT2                    | $4.1 \pm 0.2$         | $2.0 \pm 0.4$                                      | $0.63 \pm 0.15$                                     | $0.27 \pm 0.03$       | $43 \pm 11$                                       | $57 \pm 29 (150)^a$                                |
| Dm3MaT(221)                | $4.4 \pm 0.1$         | $1.1 \pm 0.1$                                      | $0.43 \pm 0.05$                                     | $0.012 \pm 0.002$     | $5.1 \pm 0.4$                                     | $9.0 \pm 0.7 (24)$                                 |
| Dm3MaT(212)                | $4.3 \pm 0.1$         | $6.0 \pm 0.8$                                      | $1.6 \pm 0.1$                                       | $0.093 \pm 0.010$     | $23 \pm 4$                                        | $38 \pm 5 (100)$                                   |
| Dm3MaT(122)                | $14 \times 10^{-3} \pm 2 \times 10^{-3}$ | $5.2 \times 10^{-3} \pm 0.7 \times 10^{-3}$ | $2.2 \times 10^{-3} \pm 0.4 \times 10^{-3}$        | $<0.001$              |                                                     |                                                   |
| Dm3MaT(112)                | $3.8 \pm 0.1$         | $1.8 \pm 0.2$                                      | $1.2 \pm 0.2$                                       | $<0.001$              |                                                     |                                                   |
| Dm3MaT(121)                | $3.1 \pm 0.1$         | $0.11 \pm 0.04$                                    | $1.8 \pm 0.6$                                       | $<0.001$              |                                                     |                                                   |
| Dm3MaT(211)                | $2.7 \pm 0.1$         | $0.83 \pm 0.08$                                    | $0.40 \pm 0.05$                                     | $<0.001$              |                                                     |                                                   |
| Dm3MaT1                    | $11 \pm 0.2$          | $4.0 \pm 0.1$                                      | $2.7 \pm 0.1$                                       | $<0.001$              |                                                     |                                                   |

**Site-specific mutants of Dm3MaT1(212)**  
| Mutant | $k_{cat}$ ($s^{-1}$) | $k_{cat}/K_m$ for acyl donor ($s^{-1} \mu M^{-1}$) | $k_{cat}/K_m$ for acyl acceptor ($s^{-1} \mu M^{-1}$) | $k_{cat}$ ($s^{-1}$) | $k_{cat}/K_m$ for acyl donor ($s^{-1} mM^{-1}$) | $k_{cat}/K_m$ for acyl acceptor ($s^{-1} mM^{-1}$) |
|--------|-----------------------|----------------------------------------------------|-----------------------------------------------------|-----------------------|-------------------------------------------------|-----------------------------------------------------|
| T35G   | $1.5 \pm 0.1$         | $0.42 \pm 0.05$                                    | $0.31 \pm 0.03$                                     | $0.010 \pm 0.001$     | $0.70 \pm 0.15$                                   | $2.8 \pm 0.5 (7)$                                  |
| L37V   | $3.9 \pm 0.1$         | $1.1 \pm 0.3$                                      | $1.9 \pm 0.1$                                       | $0.027 \pm 0.003$     | $5.2 \pm 1.3$                                     | $4.7 \pm 0.9 (12)$                                |
| A38P   | $3.4 \pm 0.1$         | $3.2 \pm 0.9$                                      | $2.4 \pm 0.9$                                       | $0.015 \pm 0.001$     | $1.5 \pm 0.3$                                     | $4.2 \pm 0.8 (11)$                                |
| Q51P   | $4.2 \pm 0.1$         | $13 \pm 4$                                         | $2.5 \pm 0.3$                                       | $0.013 \pm 0.001$     | $3.2 \pm 0.5$                                     | $4.8 \pm 0.7 (12)$                                |
| M8$^a$ | $3.2 \pm 0.1$         | $3.6 \pm 0.4$                                      | $1.5 \pm 0.4$                                       | $0.041 \pm 0.001$     | $7.1 \pm 0.6$                                     | $15 \pm 1 (40)$                                   |
| Y411N  | $1.2 \pm 0.1$         | $0.70 \pm 0.09$                                    | $0.67 \pm 0.16$                                     | $<0.001$              |                                                     |                                                   |
| A413T  | $2.3 \pm 0.1$         | $3.1 \pm 0.8$                                      | $2.9 \pm 1.1$                                       | $0.015 \pm 0.002$     | $22 \pm 6$                                        | $2.6 \pm 0.4 (7)$                                 |
| K419N  | $6.4 \pm 0.1$         | $6.8 \pm 0.8$                                      | $2.8 \pm 0.4$                                       | $0.020 \pm 0.003$     | $30 \pm 9$                                        | $6.5 \pm 1.0 (17)$                                |
| CM14$^b$ | $3.2 \pm 0.1$        | $6.2 \pm 1.2$                                      | $2.1 \pm 0.4$                                       | $0.015 \pm 0.001$     | $20 \pm 3$                                        | $16 \pm 7 (41)$                                   |

**Seven-fold mutant of Dm3MaT1**  
| Mutant | $k_{cat}$ ($s^{-1}$) | $k_{cat}/K_m$ for acyl donor ($s^{-1} \mu M^{-1}$) | $k_{cat}/K_m$ for acyl acceptor ($s^{-1} \mu M^{-1}$) | $k_{cat}$ ($s^{-1}$) | $k_{cat}/K_m$ for acyl donor ($s^{-1} mM^{-1}$) | $k_{cat}/K_m$ for acyl acceptor ($s^{-1} mM^{-1}$) |
|--------|-----------------------|----------------------------------------------------|-----------------------------------------------------|-----------------------|-------------------------------------------------|-----------------------------------------------------|
| M7$^c$ | $4.2 \pm 0.1$         | $1.9 \pm 0.1$                                      | $3.4 \pm 0.6$                                       | $0.02 \pm 0.002$      | $3.8 \pm 0.4$                                     | $9.1 \pm 1.6 (24)$                                |

$a$ Eight-fold mutant in region A; i.e. P5S/H11Q/A49R/T72A/G81S/F84Y/F86S/Y92F mutant of Dm3MaT1(212).  
$b$ Fourteen-fold mutant in region B; i.e. I308C/S309T/V310A/N317D/L325F/I333V/T351A/S356G/E378D/G379A/P440S/K445E/F455L/V459I mutant of Dm3MaT(212).  
$c$ G35T/V37L/P38A/P51Q/N410Y/T412A/N418K mutant of Dm3MaT1.

Dm3MaT3 showed 3MaT1 activity but no 3MaT2 activity (Fig. 1); it catalyzed the transfer of a malonyl group from malonyl-CoA ($K_m$ 35 ± 11 $\mu$M) to cyanidin 3-O-glucoside ($K_m$ 470 ± 100 $\mu$M) to produce cyanidin 3-O-6"-O-malonylglucoside ($k_{cat}$ 0.061 ± 0.013 $s^{-1}$). Dm3MaT3 could also act on pelargonidin 3-O-glucoside ($k_{cat}$ 0.022 ± 0.002 $s^{-1}$; $K_m$ for malonyl-CoA, 35 ± 11 $\mu$M; $K_m$ for pelargonidin 3-O-glucoside, 120 ± 12 $\mu$M) and delphinidin 3-O-glucoside ($k_{cat}$, 0.021 ± 0.001 $s^{-1}$; $K_m$ for malonyl-CoA, 24 ± 2 $\mu$M; $K_m$ for delphinidin 3-O-glucoside, 170 ± 12 $\mu$M). However, flavonol 3-O-glucoside (quercetin 3-O-glucoside), cyanidin 3-O-6"-O-malonylglucoside, pelargonidin 3,5-O-diglucoside, pelargonidin 3-O-6"-O-malonyl-glucoside 5-O-glucoside did not serve as acyl acceptors. For acyl donors, Dm3MaT3 was highly specific for malonyl-CoA. Succinyl-CoA served as a very weak substrate (relative activity, 5.1% of activity for malonyl-Co-A), whereas methylmalonyl-CoA, acetyl-CoA, p-coumaroyl-CoA, and caffeoyl-CoA were inert as acyl donors. Thus, the observed acyl-donor specificity unambiguously showed that Dm3MaT3 is a malonyltransferase. It should be noted in this respect that the AAT-related subfamily of the BAHD family (see supplemental Fig. 2S) can be specific for phenolics other than anthocyanins, as exemplified by a malonyltransferase of *Nicotiana tabacum* (24), and physiological malonyl acceptors of Dm3MaT3 in the chrysanthemum plant might not be anthocyanins, although it displayed AAT activities. The molecular weight of the native Dm3MaT3 was estimated to be 60,000 by gel permeation high performance liquid chromatography on Superdex 200 (GE Healthcare), indicating that Dm3MaT3 is monomeric. The recombinant enzyme showed a maximum activity at pH 7.5 and was stable over the pH range 4.5–8.0 (at 20 °C for 20 h) and at temperatures up to 40 °C (at pH 7.0 for 20 min).

Overall Structure of Dm3MaT3—We obtained three types of Dm3MaT3 crystals (native Dm3MaT3 (termed “native-free”), native Dm3MaT3 complexed with malonyl-CoA (termed “native-complex”), and selenomethionyl Dm3MaT3). The structure of Dm3MaT3 was solved by the multiwavelength anomalous diffraction method using selenomethionyl Dm3MaT3 and was refined to a resolution of 2.2 Å (native-free Dm3MaT3), 2.1 Å (native-complex Dm3MaT3), and 1.8 Å (selenomethionyl Dm3MaT3). The models were refined to final crystallographic $R/R_{free}$ values of 19.4/24.8% (native-free Dm3MaT3), 19.6/23.4% (native-complex Dm3MaT3), and 19.6/23.4% (selenomethionyl Dm3MaT3). The refinement statistics and model quality parameters are listed in Table 2. The asymmetric unit contains two molecules in all types of the crystals. Although several hydrogen-bonding and hydrophobic interactions between these two molecules in the asymmetric unit are observed, Dm3MaT3 is most likely to be monomeric in solution, judging from the gel filtration results (see above), which are consistent with other BAHD members that have been characterized (1).

The crystal structures of Dm3MaT3 contain 17 $\beta$-strands and 17 $\alpha$-helices, each numbered from the N terminus as shown in Fig. 3A (see also supplemental Fig. 3S). The $\beta$-strands form three groups of $\beta$-sheets; i.e. group A ($\beta1$, $\beta6$, $\beta9$, $\beta10$, $\beta3$, and $\beta15$), group B ($\beta13$, $\beta12$, $\beta14$, $\beta16$, $\beta17$, $\beta11$, and $\beta7$), and group C ($\beta2$, $\beta5$, $\beta4$, end of $\beta9$, and $\beta8$). In the native-complex structure of Dm3MaT3, the electron densities of malonyl-CoA were identified on one of two faces of the Dm3MaT3 molecule (Fig. 3, A and C), and this face is referred to as the “front face.” Another face of the enzyme molecule, which is opposite to the malonyl-CoA-bound face, is referred to as the “back face” (Fig. 4). Most of the $\beta$-sheets mentioned above are sandwiched

Structures of a BAHD Family Enzyme

TABLE 2  
Data collection and refinement statistics  
Numbers in parentheses are for the highest shell.

| Crystal type | Native-free | Native complex | SeMet-Dm3MaT3 |
|--------------|-------------|----------------|---------------|
|              |             | Peak           | Edge          | Remote       |
| Data collection and processing statistics | | | | |
| Space group | P2₁ | P2₁ | P2₁ | |
| Unit cell dimension (Å) | | | | |
| $a$ (Å) | 52.653 | 52.646 | 52.777 | |
| $b$ (Å) | 122.980 | 122.922 | 122.908 | |
| $c$ (Å) | 70.268 | 69.848 | 70.332 | |
| $\beta$ (°) | 94.212 | 94.158 | 94.381 | |
| Wavelength (Å) | 1.0000 | 1.0000 | 0.97909 | 0.97928 | 0.95000 |
| Resolution (Å) | 50.00-2.20 (2.28-2.20) | 35.49-2.10 (2.18-2.10) | 70.18-1.80 (1.86-1.80) | 70.18-1.90 (1.97-1.90) | 70.18-1.90 (1.97-1.90) |
| Measured | 158,660 | 202,428 | 640,126 | 292,976 | 293,469 |
| Mean $I/\sigma I$ | 18.5 (3.6) | 19.7 (3.9) | 31.0 (5.4) | 26.8 (5.8) | 27.4 (4.9) |
| Redundancy | 3.9 (3.0) | 4.1 (3.0) | 7.9 (6.8) | 4.2 (4.1) | 4.2 (4.0) |
| Completeness (%) | 90.2 (59) | 95.8 (74.1) | 98.6 (93.0) | 99.0 (97.3) | 99.0 (96.6) |
| $R_{\text{merge}}$ (%) | 7.3 (22) | 7.0 (21) | 6.4 (27.6) | 5.0 (20.3) | 4.9 (22.5) |
| Refinement statistics | | | | |
| Resolution | 33.88-2.20 | 50.00-2.10 | 30.46-1.80 | | |
| Protein atoms | 6932 | 6932 | 6932 | | |
| Ligand atoms | 108 | | | | |
| Water molecule | 697 | 463 | 946 | | |
| $R_{\text{work}}/R_{\text{free}}$ (%) | 19.4/24.8 | 19.6/23.4 | 19.6/23.4 | | |
| Root mean square deviations | | | | |
| Bond lengths (Å) | 0.012 | 0.011 | 0.009 | | |
| Bond angles (°) | 1.348 | 1.401 | 1.104 | | |

${}^a R_{\text{merge}} = 100 \Sigma |I - \langle I\rangle| / \Sigma I$, where $I$ is the observed intensity, and $\langle I\rangle$ is the average intensity of multiple observations of symmetry-related reflections.

(A)  
(B)

mean square deviation value of 3.6 Å for 413 Ca atoms with a 20% sequence identity (9). Structural differences between Dm3MaT3 and vinorine synthase are described below in detail. The other structurally related proteins retrieved by means of the Dali server include polyketide synthetase-associated protein 5 from *Mycobacterium tuberculosis* with a root mean square deviation value of 4.1 Å for 388 Ca atoms with a 14% sequence identity (25).

Identification of Acyl-CoA Binding Site and Active-site Channel—In the native-complex structure of Dm3MaT3, well defined electron densities of malonyl-CoA were identified in a pocket surrounded by α10, α11, α15, β12, and β14 on the front face of the Dm3MaT3 molecule (Figs. 3, B and C, and 4A), which forms the malonyl-CoA binding site. This acyl-CoA binding site is connected to a channel that lies between β-sheet groups A and B and penetrates through the Dm3MaT3 molecule. His-170 of Dm3MaT3, which corresponds to an invariant residue in motif 1 of the BAHD sequences, was identified in the middle of the channel in close proximity to the thioester carbonyl carbon of the bound malonyl-CoA (Fig. 5).

Our previous alanine-scanning mutagenesis studies of Ss5MaT1 showed that the replacement of each of His and Asp

(C)

FIGURE 4. Surface diagram showing the front face (A) and the back face (B) of Dm3MaT3. The coordinates shown as sticks are for bound malonyl-CoA. This figure was created with PyMOL program (18). The location in Dm3MaT3 of amino acid residues that correspond to those important for the 3MaT2 activity of Dm3MaT2 revealed by mutational studies are shown in green. C, a close-up view of the acyl-acceptor binding pocket.

between the faces. The front and back faces of the Dm3MaT3 molecule are mainly composed of α-helices (α1, α2, α5, α6, α9, α10, α11, α12, α13, α15, α16, and α17 on the front face; α3, α4, α7, α8, and α14 on the back face) and several loop structures. The overall structure of Dm3MaT3 is most similar to that of vinorine synthase, a BAHD family enzyme, with a root

Structures of a BAHD Family Enzyme

ilar to each other (see below). Therefore, the ability of BAHD enzymes to bind an acyl-CoA moiety might have evolved with the specific spatial arrangement of polar groups being conserved, as the result of the specific folding of the polypeptide chain (see below) rather than from the conservation of specific amino acid residues.

Concerning interactions with the malonyl portion of the bound malonyl-CoA, the thioester carbonyl oxygen of malonyl-CoA is hydrogen-bonded to His-170 of motif 1, which in turn is hydrogen-bonded with the side chain of Asn-319 of motif 2 (not shown). The side chain of Arg-178 and the main-chain N atom of Gly-386 are involved in hydrogen-bonding interactions with the carboxyl oxygen atom of a malonyl group (Fig. 5), thereby fixing the terminal carboxyl group of malonyl-CoA in the complex. Arg-178 and Gly-386 are also conserved in Dm3MaT1, Dm3MaT2, and other anthocyanin malonyltransferases that are specific for anthocyanidin 3-O-glucosides. However, Arg-178 is not conserved in anthocyanin malonyltransferases that are specific for anthocyanin 5-O-glucosides such as those from *S. splendens* (Ss5MaT1 (23) and Ss5MaT2 (26)) (see supplemental Fig. 3S). It is noteworthy here that the molecular phylogenetic tree of AATs (supplemental Fig. 2S) implies that the acyl-donor specificities of AATs are diversified after the divergence of plant species. Dm3MaT1, Dm3MaT2, and Dm3MaT3 are all enzymes of Compositae plants, whereas Ss5MaT1 and Ss5MaT2 are produced by labiate plants. Thus, the conservation of Arg-178 among the three Dm3MaT enzymes may be explained in phylogenetic terms rather than functional terms. Thus, the specificity of BAHD enzymes for the acyl portion of acyl-CoA may also be due to the specific spatial arrangement of functional groups rather than the conservation of specific amino acid residues, similar to the case for the specificity for the CoA portion of acyl-CoA (see above).

Identification of Acyl-acceptor Binding Site—The malonyl-CoA binding site connects via the active-site channel to another pocket (Figs. 3B and 4B), which is located on the back face of the Dm3MaT3 molecule. This pocket is composed of Phe-35, Trp-36, Leu-37, Pro-40, Ile-42, Asn-44, Arg-178, Pro-301, Trp-383, Ser-385, Thr-407, Ser-409, His-412, and Asn-413 and appears to be sufficiently wide to accommodate an acyl-acceptor (anthocyanin) molecule. A sequence comparison (Fig. 2) shows that the amino acid residues (Thr-35, Leu-37, Ala-38, Gln-51, Tyr-411, Ala-413, and Lys-419 of Dm3MaT2) that are important for the ability of Dm3MaT2 to act on cyanidin 3-O-6″-O-malonylglucoside correspond to Phe-35, Leu-37, Arg-38, Pro-

FIGURE 5. Close-up view of the malonyl-CoA binding pocket in the native-complex structure of Dm3MaT3. Bound malonyl-CoA is shown as thick sticks, whereas the amino acid residues of Dm3MaT3 that interact with bound malonyl-CoA are shown as thin sticks. Carbon, oxygen, nitrogen, phosphorus, and sulfur atoms are colored gray, red, blue, salmon, and yellow, respectively. Dotted lines indicate hydrogen bonds. Amino acid residues without dotted lines form hydrophobic interactions with bound malonyl-CoA. Amino acid residues whose main chain atoms form hydrogen bonds with the malonyl-CoA molecule are underlined.

residues of motifs 1 and 3, respectively, causes the nearly complete loss of catalytic activity, suggesting that one of these residues acts as a catalytic base in the general acid/base mechanism of AAT catalysis (6). The crystal structures of Dm3MaT3 show that the Asp residue in motif 3 (Asp-396) is too far from the bound substrate to interact with it (Fig. 3B). Thus, the His residue of motif 1 most likely fulfills the role of the general base catalyst, and the channel forms the active site of the enzyme.

Interaction of Dm3MaT3 with Bound Malonyl-CoA—The native-complex structure of Dm3MaT3 provides the first crystal structure of a BAHD member complexed with acyl-CoA, permitting detailed information relative to interactions between the enzyme and acyl-donor substrate to be elucidated. In the native-complex structure, the malonyl-CoA molecule interacts with His-170, Ala-175, Arg-178, Lys-260, Glu-271, Tyr-272, Val-273, Ser-274, Ser-275, Phe-276, Pro-301, Ile-302, Asp-303, Arg-307, Gly-386, Thr-387, and Lys-389 of Dm3MaT3 (Fig. 5), where the underlined residues are those that interact with the malonyl group, with the others interacting with the CoA moiety of the acyl-donor molecule.

Fourteen hydrogen-bonding interactions between the Dm3MaT3 protein and the CoA portion of malonyl-CoA were identified, five of which are mediated by main-chain atoms of Dm3MaT3 (Fig. 5). A sequence comparison (see supplemental Fig. 3S) revealed that the amino acid residues involved in CoA binding are not strictly conserved among members of the BAHD family, even among AATs, although the folds of the acyl-CoA binding site of the BAHD enzymes appear to be sim-

Structures of a BAHD Family Enzyme

(A)                                                                 (B)

FIGURE 6. Superposition of Dm3MaT3 and vinorine synthase (ribbon diagram). A, front face of enzyme molecules showing the acyl-CoA binding site. B, back face of enzyme molecules showing the acyl-acceptor binding site. The two structures are superimposed as described under “Experimental Procedures.” The main chain tracing of Dm3MaT3 and vinorine synthase is shown in green and magenta, respectively. Malonyl-CoA molecule that is bound to Dm3MaT3 is shown as sticks. Dm3MaT3 structures shown in A and B of this figure correspond to those shown in A and B, respectively, of Fig. 4.

51, Leu-405, Thr-407, and Asn-413 of Dm3MaT3, all of which, except for Pro-51, are among the amino acid residues that form the pocket at the back face of the Dm3MaT3 molecule (Fig. 4). The inner part of the pocket is accessible to the malonyl moiety of the bound malonyl-CoA through the active-site channel (Fig. 4, B and C). All of these observations led to the conclusion that the pocket on the back face of the Dm3MaT3 molecule serves as the acyl-acceptor binding site of Dm3MaT3, unambiguously establishing the spatial arrangement of acyl-CoA and acyl-acceptor binding sites in the enzyme molecule, both of which are connected by an active-site channel. Such a spatial arrangement of substrate binding sites and active site was previously proposed for the vinorine synthase structure (9) and is similar to those of several other acyl-CoA-dependent acyltransferases, such as Azotobacter vinelandii dihydrolipoyl transacetylase (Ref. 27; Protein Data Bank (PDB) codes 1EAD and 1EAB) and mouse carnitine acetyltransferase (Ref. 28; PDB codes 1NDB and 1NDI), which show essentially no sequence similarity to Dm3MaT3.

Comparison with Vinorine Synthase Structure—The β-sheet groups C and B of Dm3MaT3 consist of five and seven β-strands, respectively, whereas those of vinorine synthase contains only two and six short β-strands, respectively. The crystal structures of Dm3MaT3 (even the 1.8-Å resolution structure of selenomethionyl Dm3MaT3) contain three disordered regions (N-terminal residues Met-1–Pro-5, residues Asp-365–Phe-369, and C-terminal residues Lys-451–Leu-454), whereas that of vinorine synthase contains two disordered regions (N-terminal residues Met-1–Pro-3 and residues Ser-235–Glu-239).

The superposition of crystal structures of Dm3MaT3 and vinorine synthase reveals that, whereas the structures of the front faces of these enzymes are similar to each other, the structures of the back faces differ significantly (Fig. 6). Namely, the arrangements of secondary structures on the front face of both enzymes can be well superposed, showing similar structures around the acyl-CoA binding pockets in both enzymes (Fig. 6A). By contrast, the spatial arrangement of secondary structures (α-helices and loops) on the back face of Dm3MaT3 cannot be superimposed over that of vinorine synthase; hence, the structures around the acyl-acceptor binding sites of both enzymes are different. These observations permitted the unique structural characteristics of the BAHD family concerning the specificities for the acyl-donor and acceptor substrates to be rationalized. The common ability of BAHD members to utilize acyl-CoAs appears to be maintained through the structural conservation of the front face of the enzyme molecules. However, the fact that the BAHD family displays versatility with respect to acyl-acceptor substrates should arise from the differential architectures of the acyl-acceptor binding sites, which are constructed with differential spatial arrangements of secondary structures on the back face of the enzyme molecules. BAHD family enzymes with different biochemical functions show a only low sequence similarity to each other (~20% identity), ensuring such differential architectures for the acyl-acceptor binding site.

Induced Fit of Dm3MaT3 upon Malonyl-CoA Binding—A comparison of native-free and native-complex structures reveals that the overall structures of malonyl-CoA-bound and -unbound forms of Dm3MaT3 are essentially identical with each other, showing a root mean square deviation value of 0.327 Å for 440 Ca atoms. However, these structures show some movements of the backbone (Ca atoms of Pro-301–Ile-302 and Gly-386–Lys-389) and side-chain positions (Arg-307 and Lys-389) around the malonyl-CoA binding site upon malonyl-CoA binding. The backbone of Pro-301–Ile-302 and Gly-386–Lys-389 is shifted outward by more than 0.5 Å, with a maximum shift of the Ca atom of Thr-387 by 1.18 Å, thus making it possible to accommodate a malonyl-CoA molecule in the pocket. Moreover, upon malonyl-CoA binding, the side chains of Arg-307 and Lys-389 rotate and form hydrogen bonds with the CoA portion of malonyl-CoA. For comparison, the two types of ligand-free crystal structures (i.e. structures of native-free and selenomethionyl Dm3MaT3s) could be regarded as essentially identical with each other (root mean square deviation of 0.178 Å for 440 Ca atoms), where the backbone shifts around residues Pro-301–Ile-302 and Gly-386–Lys-389 are only less than 0.3 Å. The observed deviations around malonyl-CoA binding site upon binding of malonyl-CoA can be consistently explained in terms of an induced fit of the Dm3MaT3 molecule.

Previous kinetic studies of malonyl transfer catalyzed by Ss5MaT1 suggested that the kinetic mechanism could be most consistently described in terms of a ternary-complex mechanism, where malonyl-CoA was the first substrate to bind to the enzyme, and CoA-SH was the last product to dissociate from the enzyme-product complex (6). This predicted order of substrate binding and product release prompted us to examine whether the binding of malonyl-CoA entails any conformational changes in the acyl-acceptor binding site in the

15820 JOURNAL OF BIOLOGICAL CHEMISTRY
VOLUME 282 • NUMBER 21 • MAY 25, 2007

(A)                                                                 (B)

![Image Description](#)

FIGURE 7. Schematic representation of the substrate binding sites and active-site channel in ternary complexes of Dm3MaT1 (A) and Dm3MaT2 (B). Enzyme molecules are shown in gray. The structure of the bound anthocyanin is shown as a quinonoidal base structure.

Dm3MaT3 molecule to allow for the subsequent binding of an acyl acceptor (anthocyanin). However, the main-chain and side-chain conformations of the acyl-acceptor binding sites of the native-complex structure were essentially identical to those of the native-free structure (shift of all atoms of amino acid residues forming acyl-acceptor binding sites, less than 0.3 Å). Thus, sequential binding for the malonyl-CoA and acyl acceptor appears to be less likely in the case of Dm3MaT3 catalysis.

Specificity and Mechanism in AAT Catalysis—The results obtained in the present studies provide a structural basis for general insights into the specificity and mechanism of BAHD catalysis. In the discussion below the amino acid residues of BAHD enzymes are numbered so as to correspond to the numbering of Dm3MaT3 based on the alignment shown in Fig. 2 (see also supplemental Fig. 3S), and the positional numbering, according to this notation, is indicated by an asterisk.

Thus far AATs have been strictly classified into two distinct categories on the basis of their acyl-donor specificity, *i.e.*, aliphatic AATs (*e.g.*, Dm3MaT1, Dm3MaT2, Dm3MaT3, and Ss5MaT1) and aromatic AATs (*e.g.* hydroxycinnamoyl-CoA: anthocyanin 3,5-O-diglucoside 6″-O-hydroxycinnamoyltransferase, Gt5AT, which is involved in the biosynthesis of gentiodelphin in the blue flowers of gentian (*Gentiana triflora*) (29), and hydroxycinnamoyl-CoA:anthocyanin 3-O-glucoside 6″-O-hydroxycinnamyltransferase, Pf3AT, which is involved in the biosynthesis of shisonin in the red forms of *Perilla frutescens* (30)). Aliphatic acyltransferases specifically utilize aliphatic acyl-CoAs (such as malonyl-CoA) but do not act on aromatic acyl-CoAs (such as *p*-coumaroyl-CoA and caffeoyl-CoA); likewise, aromatic AATs utilize aromatic acyl-CoAs exclusively but do not act on aliphatic acyl-CoAs. The native-complex structure of Dm3MaT3 revealed that Arg-178 and Gly-386 are involved in binding the malonyl group of malonyl-

CoA. A sequence comparison (see supplemental Fig. 3S) revealed that Arg-178 of Dm3MaT3 is specifically replaced by Phe in both Gt5AT and Pf3AT sequences. Phe-178* (*i.e.* Phe-182 of Gt5AT and Phe-170 of Pf3AT) of these aromatic AATs would be expected to interact with the aromatic ring moiety of a bound hydroxycinnamoyl-CoA, and no other amino acid residues near acyl-CoA binding site would be predicted to be capable of entering into aromatic interactions with a bound acyl-CoA. Thus, Phe178* is likely to be a key residue that governs the preference of Gt5AT and Pf3AT for aromatic acyl-CoA.

In Dm3MaT1 and Dm3MaT2 catalysis, the acyl-acceptor binding site, containing a set of Dm3MaT1-type amino acid residues (Gly-35*, Val-37*, Pro-38*, Pro-51*, Asn-405*, Thr-407*, and Asn-413*), would only allow the binding of cyanidin 3-O-glucoside (Fig. 7), whereas that containing a set of Dm3MaT2-type residues (Thr-35*, Leu-37*, Ala-38*, Tyr-405*, Ala-407*, and Lys-413*) would allow the productive binding of the mono-malonylated product (cyanidin 3-O-6″-O-malonylglucoside) in addition to that of the non-malonylated substrate (cyanidin 3-O-glucoside) (Fig. 7). Although details of the nature of anthocyanin binding in the pocket remains to be clarified, Thr-407* of Dm3MaT1 should be located near the dead end of the anthocyanin binding pocket (Fig. 4C), and the replacement of Thr-407* by alanine likely provides a wider space in the pocket, making some contribution to the productive binding of the mono-malonylated product in Dm3MaT2 (Fig. 7). Gln-51* of Dm3MaT2 is remote from the pocket (Fig. 4A), so that it is unlikely that this residue is directly involved in interactions with a bound anthocyanin. Rather, this residue may play a structural role in maintaining the conformation of the enzyme in such a way that affects the shape of the acyl-acceptor binding site. After the formation of a ternary complex consisting of malonyl-CoA, anthocyanin, and enzyme, the hydroxy group of anthocyanin to be activated would be placed in the active-site channel near the thioester carbonyl carbon of the bound malonyl-CoA and His-170*, which then abstracts a proton from the hydroxy group, triggering an acyl transfer reaction (6). It should be noted that Trp-36 of Dm3MaT3 is located at the anthocyanin binding pocket and is conserved in all other AATs (see supplemental Fig. 3S), implying that this aromatic residue may be crucial for the acyl acceptor specificity of the anthocyanins. This residue would be a good target in a mutagenesis study to alter the acyl acceptor specificity of a BAHD family enzyme, and such work is currently under way.

In addition to His-170*, several amino acid residues have been shown to be functionally important for AAT catalysis as the result of alanine-scanning mutagenesis studies of Ss5MaT1

Structures of a BAHD Family Enzyme

(6), *i.e.* Asp-396*, Tyr-48*, Asp-174*, Arg-305*, and Asn-319*. Asp-396* (in motif 3) is too far from the bound substrate to interact with it (Fig. 3), so that the Asp residue in motif 3 could play a structural role in enzyme catalysis. Asn-319* (in motif 2) is hydrogen-bonded with His-170* and is thereby likely to affect the role of the catalytic His-170*. Tyr-48* is distant from the active-site channel and substrate binding sites and probably plays an important role in maintaining the catalytically active structure of the enzyme. Asp-174* (in motif 1) forms a hydrogen bond with Arg-305* that occurs near the malonyl-CoA binding pocket (not shown), so that this interaction may be important for the efficient binding of malonyl-CoA.

**Conclusion—** The crystal structures of Dm3MaT3 along with the results of mutagenesis studies, intended to explore key residues responsible for differential acyl-acceptor specificity between Dm3MaT1 and Dm3MaT2, allowed us to identify the acyl-CoA and acyl-acceptor binding sites in AATs. This finding provides concrete evidence for the spatial arrangement of acyl-donor and acyl-acceptor binding sites connected by an active-site channel in BAHD enzymes. The front faces of BAHD enzyme molecules, which contain the acyl-CoA binding site, are structurally similar to each other, rationalizing the common ability of BAHD enzymes to utilize acyl-CoA. By contrast, the structures around the acyl-acceptor binding site on the back face of BAHD enzymes can be different from each other, thus explaining the diversity of acyl-acceptor specificities of the BAHD family. These findings establish the structural basis for general insights into the specificity and mechanism of BAHD catalysis.

---

**Acknowledgments—** We thank Drs. N. Igarashi and N. Matsugaki and the technical staff at synchrotron beamline BL-5A of the Photon Factory, High Energy Accelerator Research Organization, Japan, for assistance with the data collection.

---

**REFERENCES**

1. D'Auria, J. C. (2006) *Curr. Opin. Plant Biol.* **9**, 331–340
2. Nakayama, T., Suzuki, H., and Nishino, T. (2003) *J. Mol. Catal. B Enzym.* **23**, 117–132
3. St-Pierre, B., and De Luca, V. (2000) in *Evolution of Metabolic Pathways, Recent Advances in Phytochemistry,* (Romeo, J. T., Ibrahim, R., Varin, L., and De Luca, V., eds) Vol. 34. pp. 285–315, Elsevier Science Ltd., Oxford
4. St-Pierre, B., Laflamme, P., Alarco, A.-M., and De Luca, V. (1998) *Plant J.* **14**, 703–713
5. Strack, D., and Wray, V. (1994) in *The Flavonoids* (Harbone, J. B., ed) pp. 1–22, Chapman & Hall, Washington, D. C.
6. Suzuki, H., Nakayama, T., and Nishino, T. (2003) *Biochemistry* **42**, 1764–1771
7. Ino, I., Nishiyama, H., and Yamaguchi, M.-A. (1993) *Phytochemistry* **32**, 1425–1426
8. Suzuki, H., Nakayama, T., Yamaguchi, M., and Nishino, T. (2004) *Plant Sci.* **166**, 89–96
9. Ma, X., Koepke, J., Panjikar, S., Fritzsch, G., and Stöckigt, J. (2005) *J. Biol. Chem.* **280**, 13576–13583
10. Segel, I. H. (1975) *Enzyme Kinetics: Behavior, and Analysis of Rapid Equilibrium and Steady-state Enzyme Systems,* pp. 623–625, John Wiley & Sons, Inc., New York
11. Otwinowski, Z., and Minor, W. (1997) *Methods Enzymol.* **276**, 307–326
12. Terwilliger, T. C., and Berendzen, J. (1999) *Acta Crystallogr. Sect. D.* **55**, 849–861
13. Terwilliger, T. C. (2003) *Acta Crystallogr. Sect. D.* **59**, 38–44
14. Perrakis, A., Morris, R., and Lamzin, V. S. (1999) *Nat. Struct. Biol.* **6**, 458–463
15. Jones, T. A., Zou, J. Y., Cowan, S. W., and Kjeldgaard, M. (1991) *Acta Crystallogr. Sect. A.* **47**, 110–119
16. Murshudov, G. N., Vagin, A. A., and Dodson, E. J. (1997) *Acta Crystallogr. Sect. D.* **53**, 240–255
17. Fenn, T. D., Ringe, D., and Petsko, G. A. (2003) *J. Appl. Crystallogr.* **36**, 944–947
18. DeLano, W. L. (2002) *The PyMOL Molecular Graphics Systems,* DeLano Scientific, San Carlos, CA
19. Merritt, E. A., and Murphy, M. E. (1994) *Acta Crystallogr. Sect. D Biol. Crystallogr.* **50**, 869–873
20. Kabsch, W. (1976) *Acta Crystallogr. Sect. A.* **32**, 922–923
21. Suzuki, H., Sawada, S., Yonekura-Sakakibara, K., Nakayama, T., Yamaguchi, M.-A., and Nishino, T. (2003) *Plant Biotech.* **20**, 229–234
22. Suzuki, H., Nakayama, T., Yonekura-Sakakibara, K., Fukui, Y., Nakamura, N., Yamaguchi, M.-A., Tanaka, Y., Kusumi, T., and Nishino, T. (2002) *Plant Physiol.* **130**, 2142–2151
23. Suzuki, H., Nakayama, T., Yonekura-Sakakibara, K., Fukui, Y., Nakamura, N., Nakao, M., Tanaka, Y., Yamaguchi, M.-A., Kusumi, T., and Nishino, T. (2001) *J. Biol. Chem.* **276**, 49013–49019
24. Taguchi, G., Shitchi, Y., Shirasawa, S., Yamamoto, H., and Hayashida, N. (2005) *Plant J.* **42**, 481–491
25. Buglino, J., Onwueme, K. C., Ferreras, J. A., Quadri, L. E., and Lima, C. D. (2004) *J. Biol. Chem.* **279**, 30634–30642
26. Suzuki, H., Sawada, S., Watanabe, K., Nagae, S., Yamaguchi, M. A., Nakayama, T., and Nishino, T. (2004) *Plant J.* **38**, 994–1003
27. Mattevi, A., Obmolova, G., Schulze, E., Kalk, K. H., Westphal, A. H., de Kok, A., and Hol, W. G. (1992) *Science* **255**, 1544–1550
28. Jogl, G., and Tong, L. (2003) *Cell* **112**, 113–122
29. Fujiwara, H., Tanaka, Y., Yonekura-Sakakibara, K., Fukuchi-Mizutani, M., Nakao, M., Fukui, Y., Yamaguchi, M., Ashikari, T., and Kusumi, T. (1998) *Plant J.* **16**, 421–431
30. Yonekura-Sakakibara, K., Tanaka, Y., Fukuchi-Mizutani, M., Fujiwara, H., Fukui, Y., Ashikari, T., Murakami, Y., Yamaguchi, M.-A., and Kusumi, T. (2000) *Plant Cell Physiol.* **41**, 495–502

Enzyme Catalysis and Regulation:  
Structural and Mutational Studies of  
Anthocyanin Malonyltransferases Establish  
the Features of BAHD Enzyme Catalysis  

Hideaki Unno, Fumiko Ichimaida, Hirokazu  
Suzuki, Seiji Takahashi, Yoshikazu Tanaka,  
Atsushi Saito, Tokuzo Nishino, Masami  
Kusunoki and Toru Nakayama  
J. Biol. Chem. 2007, 282:15812-15822.  
doi: 10.1074/jbc.M700638200 originally published online March 23, 2007  

Access the most updated version of this article at doi: [10.1074/jbc.M700638200](http://dx.doi.org/10.1074/jbc.M700638200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinitysites).  

Alerts:  
- [When this article is cited](#)  
- [When a correction for this article is posted](#)  

[Click here](#) to choose from all of JBC's e-mail alerts  

Supplemental material:  
[http://www.jbc.org/content/suppl/2007/03/27/M700638200.DC1.html](http://www.jbc.org/content/suppl/2007/03/27/M700638200.DC1.html)  

This article cites 27 references, 6 of which can be accessed free at  
[http://www.jbc.org/content/282/21/15812.full.html#ref-list-1](http://www.jbc.org/content/282/21/15812.full.html#ref-list-1)
