
REVIEW

**Biosynthesis of aminocyclitol-aminoglycoside antibiotics and related compounds**

Patricia M. Flatt and Taifo Mahmud*

Received (in Cambridge, UK) 30th August 2006  
First published as an Advance Article on the web 21st December 2006  
DOI: 10.1039/b603816f  

Covering: 1978 to June 2006  

This review covers the biosynthesis of aminocyclitol-aminoglycoside antibiotics and related compounds, particularly from the molecular genetic perspectives. 195 references are cited.

---

1. Introduction  
2. Biosynthesis of myo-inositol-derived aminoglycoside antibiotics  
   2.1 The origin of myo-inositol 1-phosphate  
   2.2 Biosynthesis of streptomycin  
   2.3 Biosynthesis of bluensomycin  
   2.4 Biosynthesis of spectinomycin  
   2.5 Biosynthesis of fortimicin (astromicin) and related compounds  
   2.6 Biosynthesis of kasugamycin  
   2.7 Biosynthesis of hygromycin A  
3. Biosynthesis of 2-deoxy-scyllo-inosose-derived aminoglycoside antibiotics  
   3.1 The 2-deoxystreptamine core unit  
   3.2 Discovery of 2-deoxy-scyllo-inosose synthases  
   3.3 Biosynthesis of butirosin  
   3.4 Biosynthesis of related 4, 5-disubstituted aminoglycosides  
4. Biosynthesis of 2-epi-5-epi-valiolone-derived aminocyclitols  
   4.1 Biosynthetic origin of the core C<sub>7</sub>-cyclitols  
   4.2 Biosynthesis of acarbose  
   4.3 Biosynthesis of validamycin  
5. Biosynthesis of cyclopentitol-derived aminocyclitols  
   5.1 Biosynthetic origin of pactamycin  
   5.2 Biosynthetic origin of allosamidin  
   5.3 Biosynthetic origin of trehazolin  
   5.4 Biosynthetic origin of aristeromycin and neplanocin A  
   5.5 Biosynthetic origin of the cyclopentitol moiety of queuosine and epoxyqueuosine  
   5.6 Biosynthetic origin of the cyclopentitol moiety linked to bacteriohopanetetrol  
6. Summary and perspectives  
7. Acknowledgements  
8. References  

Department of Pharmaceutical Sciences, Oregon State University, Corvallis, OR, 97331-3507, USA  

Dr Patricia Flatt received a PhD degree in biochemistry from Vanderbilt University under the directions of Professor Jennifer Pietenpol in the Center in Molecular Toxicology in 2000. Subsequently, she received a post-doctoral fellowship from the NIH to work under the guidance of Dr William Gerwick in the study of bioorganic and marine natural products biosynthesis at Oregon State University from 2000–2003. Upon the completion of her post-doctoral training, she was promoted to the rank of Assistant Professor of Sr. Research in the College of Pharmacy at Oregon State University. Her broad research interests include the biosynthesis of secondary metabolites from marine cyanobacteria and soil-derived actinomycete species, with a special emphasis on the biosynthesis of C<sub>7</sub>N aminocyclitols and their incorporation into unique aglycone structures.

Dr Taifo Mahmud received a PhD degree in natural products chemistry from Osaka University under the directions of Professors Isao Kitagawa and Motomasa Kobayashi in 1997. Subsequently, he received a three-year postdoctoral training from Professor Heinz G. Floss at the University of Washington in Seattle, where he was then promoted to the rank of Research Assistant Professor in Chemistry. After six enjoyable years living in Seattle, he moved to Oregon State University College of Pharmacy as an Assistant Professor of Medicinal Chemistry. Dr Mahmud’s research interests are broadly in bioorganic and natural products chemistry, biosynthesis of microbial secondary metabolites, and the interface of molecular genetics, enzymology, and chemistry to create and develop novel pharmaceutically active leads. One of his current projects focuses on the biosynthetic studies and developments of aminocyclitol-containing bioactive natural products.

# 1 Introduction

Aminoglycoside antibiotics have long been known for their important biological activities. Many of them, e.g., amikacin (1, semi-synthetic), gentamicin (2), kanamycin (3), neomycin (4), netilmicin (5, semi-synthetic), streptomycin (6), and tobramycin (7), have been used clinically for decades as potent antimicrobial agents (Fig. 1). Other analogs, e.g., hygromycin A (8) and spectinomycin (9), are used frequently as animal medicines in veterinary and agricultural applications, and as selection markers in molecular genetic studies. Although in general these antibiotics are known as the family of aminocyclitol or aminoglycoside antibiotics, they can be divided into four groups based on the biosynthesis of their core aminocyclitol moieties. Two groups of biosynthetic pathways have been characterized in depth and include aminoglycosides originating from i) the *myo*-inositol pathway, e.g., streptomycin (6) and hygromycin A (8), and ii) the 2-deoxy-scyllo-inosose pathway, e.g., gentamicin (2), kanamycin (3), and butirosin (10). A third group of aminocyclitol/aminoglycoside natural products represents compounds that contain a C<sub>7</sub>N aminocyclitol moiety, exemplified by the potent α-glucosidase inhibitor acarbose (11) and the antifungal-antibiotic validamycin A (12). Acarbose is one of a new generation of anti-type II insulin-independent antidiabetic drugs available on the market, whereas validamycin has been widely used in Asia as a crop protectant. Finally, a fourth group represents natural products that contain an aminocyclitol moiety, such as the cytotoxic antibiotic pactamycin (13) and the trehalase inhibitor trehazolin (14).

A number of very good reviews have previously been published on the biosynthesis of some aminoglycoside antibiotics and C<sub>7</sub>N aminocyclitol-derived natural products. However, since that time, there have been numerous reports elucidating the biosynthetic gene clusters required for the biosynthesis of this diverse set of compounds. The purpose of this review is to compile and compare the genetic architecture of these biosynthetic gene clusters. Elucidation of genetic similarities within these pathways offers a unique opportunity to explore novel approaches to the engineering of antibiotic biosynthesis as well as providing genetic tools to screen and elucidate secondary metabolite pathways from previously uncharacterized organisms.

This review will be divided into sections based on the biosynthetic origin of the incorporated aminocyclitol and will include:

![Figure 1](https://i.imgur.com/yourimageurl.png)

*Fig. 1*

i) myo-inositol-derived aminocyclitols, ii) 2-deoxy-scylllo-inosose-derived aminocyclitols, iii) 2-epi-5-epi-valiolone-derived aminocyclitols and iv) unique cyclopentitol-derived aminocyclitols. The review will end with a global comparison of the similarities within these pathways and their potential use in drug discovery.

## 2 Biosynthesis of myo-inositol-derived aminoglycoside antibiotics

### 2.1 The origin of myo-inositol 1-phosphate

Inositol-containing molecules are critical players of many cellular processes in addition to aminoglycoside antibiotic production and are involved in signal transduction pathways, stress response, and cell wall biogenesis. ${}^{10-12}$ myo-Inositol 1-phosphate (MIP) synthase performs the first step in the synthesis of all inositol-derived compounds, the conversion of glucose 6-phosphate (15) to MIP (18), and has been reported from both prokaryotic and eukaryotic sources. ${}^{13}$ Primary sequence analysis reveals that MIP synthases of prokaryotic origin comprise a very diverse set of enzymes that are distinct from their eukaryotic counterparts. ${}^{14}$ In spite of this striking diversity, it appears that the core three-dimensional structure and overall reaction mechanism of MIP synthases are well conserved. ${}^{15-18}$ Furthermore, the reaction represents the first committed step in inositol biosynthesis and can act as a rate-limiting step in the synthesis of aminoglycoside antibiotics. Thus, the biochemical activity and reaction mechanism of MIP synthase is of considerable interest.

The reaction mechanism of MIP synthase is proposed to involve three distinct steps, including an oxidation, an intramolecular aldol cyclization and a reduction (Scheme 1). ${}^{13,15}$ Crystallographic data indicate that MIP synthase utilizes two cofactors, NAD${}^{+}$ and the monovalent cation, NH${}_{4}^{+}$. ${}^{16,18}$ Other classes of enzymes containing NAD${}^{+}$ and a cation cofactor include dehydroquinate (DHQ) synthases and alcohol dehydrogenases. ${}^{19,20}$ In fact, even though MIP synthases and DHQ synthases share little structural similarity when overlaid, the proximity of the NH${}_{4}^{+}$ and NAD${}^{+}$ cofactors within MIP synthase are within 1.5 Å of the Zn${}^{2+}$ and NAD${}^{+}$ cofactors of DHQ synthase. ${}^{16}$ With such close proximity, it has been proposed that the NH${}_{4}^{+}$ cofactor acts as a Lewis acid to stabilize the negative charge on an oxygen atom during enzyme catalysis. Inhibitor binding studies with MIP synthases suggest that this class of enzyme binds exclusively to the open form of glucose 6-phosphate (15) rather than the cyclized substrate. ${}^{16,21}$ Once bound to the enzyme, 15 is oxidized in an NAD${}^{+}$-dependent reaction to the C-5 ketone (16). Following oxidation, the substrate is proposed to undergo phosphate-mediated enolization. During this process the ammonium cation is proposed to stabilize the negative charge on the C-5 ketone. The nucleophilic attack of C-6 on C-1 is promoted by charge stabilization of the C-1 oxygen by a conserved lysine residue. Subsequent protonation of the C-1 oxygen and reduction at the C-5 position by NADH completes MIP biosynthesis.

In aminoglycoside biosynthesis MIP is converted to myo-inositol (19), which then undergoes a number of pathway specific modifications prior to its incorporation into the final antibiotic structure. 19 has been shown to be the direct precursor for numerous antibiotics including streptomycin (6), spectinomycin (9), the fortimicins (e.g., 20), kasugamycin (21), and hygromycin A (8). The focus of this section of the review will be to elaborate on the biochemical mechanisms and processes used by microorganisms to structurally modify the myo-inositol core unit during the biosynthesis of these interesting and highly functional aminoglycoside antibiotics.

### 2.2 Biosynthesis of streptomycin

Pioneering work by Piepersberg and coworkers on the biosynthesis of streptomycin and related compounds, such as 5'-hydroxystreptomycin (22) and bluensomycin (23) (Fig. 2), has provided an excellent foundation for the exploration of the genes and biochemical mechanisms involved in the biosynthesis of this class of aminoglycoside-antibiotic. ${}^{7,22}$ Streptomycin-related aminoglycosides are primarily 3-ring structures derived from a complex, branched biosynthetic pathway originating from glucose 6-phosphate (15) (Scheme 2). The biosynthesis proceeds through the formation and ligation of three major activated intermediates including a scylllo-inosose-derived aminocyclitol (i.e., streptidine 6-phosphate (28) or bluensidine 6-phosphate (45)), an activated 6-deoxyhexose component (i.e., dTDP-L-dihydrostreptose (34)), and an aminoheose derivative (i.e., NDP-N-methyl-L-glucosamine (40)). ${}^{7}$

![Figure 2](attachment://image1.png)

**Fig. 2**

**Scheme 1 Proposed mechanism of myo-inositol 1-phosphate synthase.**

18 myo-inositol 1-phosphate
HO
HO
PO OH

15 glucose 6-phosphate
HO
HO
OH OH

29 glucose 1-phosphate
HO
HO
OH OP

36 dCDP-D-Glucose
HO
HO
OH OCDP

19 myo-inositol
HO
HO
HO OH

24 scyllo-inosose
HO
HO
OH O

26 scyllo-inosamine 4-phosphate
PO
HO
NH₂
HO

25 scyllo-inosamine
HO
HO
NH₂
HO

27 amidino-scyllo-inosamine
4-phosphate
PO
HO
NH
NH
HO

28 streptidine 6-phosphate
HN
NH₂
HO
HO
HN
NH₂

30 dTDP-D-Glucose
HO
HO
OdTDP

31 4-keto-6-deoxy-
dTDP-D-glucose
O
HO
HO
OdTDP

33 dTDP-L-rhamnose
HO
HO
OdTDP

34 dTDP-L-dihydrostreptose
H₃C
HO
HO
OH

35 O-1,4-α-L-dihydrostreptosyl-streptidine 6-phosphate
H₂N
NH
HO
HO
OP
H₂N
NH
HO
HO
H₃C
O
HO
O
H₃CHN
O
HO
OH

37
HO
HO
OCDP

38
HO
HO
H₂N
OCDP

39
HO
HO
H₃CHN
OCDP

40 CDP-N-methyl-L-glucosamine
OH
NHCH₃
HO
OH ONDP

6 streptomycin
H₂N
NH
HO
HO
O
H₃C
O
H₃CHN
O
HO
OH

42 streptomycin 6-phosphate
H₂N
NH
HO
HO
OP
H₃C
O
H₃CHN
O
HO
OH

41 dihydrostreptomycin 6-phosphate
H₂N
NH
HO
HO
OP
H₃C
O
H₃CHN
O
HO
OH

Scheme 2 Proposed biosynthetic mechanism of streptomycin biosynthesis. The proposed enzymes involved in the pathway from the S. griseus and S. glaucescens biosynthetic gene clusters are listed. Enzymes denoted with an * have been biochemically characterized.

The molecular mechanisms for the biosynthesis of streptomycin have been studied in depth over the past two decades in a number of soil-derived Actinomycetes, including Streptomyces griseus N2-3-11, Streptomyces glaucescens 22794, and Streptomyces bikiniensis ATCC 11062.⁷ Experiments on streptomycin biosynthesis in S. griseus have revealed over 29 genes involved in the pathway.²³⁻²⁵ Many homologs and a few additional orfs have been discovered in S. glaucescens.²⁶,²⁷ The production of streptomycin (6) is controlled through an A-factor regulatory cascade that leads to the activation and expression of a regulatory transcription factor in the streptomycin biosynthetic pathway called StrR (Fig. 3B).²⁸,²⁹ The A-factor, 2-isocapryloyl-3R-hydroxymethyl-γ-butyrolactone (43), is synthesized during late bacterial growth phase through an enzymatic condensation reaction involving the AfsA protein (Fig 3A).³⁰,³¹ When critical levels of the A-factor are reached, it acts as a microbial hormone, binding with the transcriptional inhibitor ArpA.³² Following A-factor binding, the ArpA homodimer dissociates from the DNA allowing the transcriptional activation of the adpA gene locus. AdpA acts as a transcriptional activator and binds to a promoter region

RNA Polymerase
ArpA dimer

adpA

A factor

mRNA

AdpA

S. griseus

sts
str Z Y U V W
B2 M L E D R A BI F G H I K S T O N
StrR
resistance

S. glaucescens (streptomycin producer)
str O N B2 M L E D R T S K A B1 F G H P Q U X V W

S. hygroscopicus form glebosus (bluensomycin producer)
blm E D T S B F G H U X A Y C V W

S. bluensis (bluensomycin producer)
blu N M L R E D T S (Q?) B F G H

Transcription Factor
Phosphotransferase
Involved in 2-deoxystreptamine biosynthesis
Involved in dTDP-L-dihydro-streptose biosynthesis
Promoter elements
Involved in NDP-N-methyl-L-glycosamine biosynthesis
Involved in streptomycin biosynthesis
ABC transporter/efflux pump
O-carbamoyltransferase
Unknown function

Fig. 3 Regulation, expression, and comparison of the biosynthetic gene clusters for streptomycin and bluensomycin biosynthesis. (A) Synthesis and function of the A-factor, 2-isocapryloyl-3R-hydroxymethyl-γ-butyrolactone in the regulation of streptomycin biosynthesis. Binding of the A-factor to the ArpA homodimer, activates the transcription of the *adpA* gene locus. AdpA binds to and activates the promoter upstream of the gene encoding the streptomycin regulatory protein, StrR. StrR then mediates the transcriptional upregulation of the remaining streptomycin biosynthetic genes. (B–E) Graphical representation of the biosynthetic gene clusters for streptomycin biosynthesis from (B) *S. griseus*, (C) *S. glaucescens*, and bluensomycin biosynthesis from (D) *S. hygroscopicus*, and (E) *S. bluensis*.

upstream of *strR* and *strA* in the streptomycin biosynthetic gene cluster. Strains of *S. griseus* containing a knockout mutation in the *adpA* locus were unable to activate *strR* and *strA* transcription or produce streptomycin.²⁸ Extrachromosomal expression of AdpA restored antibiotic production. Interestingly, expression of *adpA* in the wild type strain from a high copy expression plasmid increased streptomycin production 10-fold.³² Other studies on the regulation of streptomycin biosynthesis have also demonstrated that exogenous levels of S-adenosylmethionine (SAM) acts as a potent activator of *adpA* transcription and subsequently streptomycin production.³³,³⁴ Interestingly, the addition of SAM acts independently of the A Factor and does not involve binding of the transcriptional repressor ArpA.³³ Overall, understanding the fundamental regulatory cascades involved in antibiotic production can provide useful tools for upregulating and optimizing antibiotic production.
The $strR$ gene locus encodes a transcriptional activator and appears to be the sole regulatory element in the streptomycin biosynthetic pathway. DNA binding domains for StrR have been found and characterized upstream of $strB1$, $strR$, $stsC$, $stsU$, and $stsV$, and although no StrR binding domains have been found upstream of $strD$, $strN$, and $strO$, transcription of these orfs was found to be dependent on StrR (Fig. 3B). ${ }^{28}$ Interestingly, synteny homology analysis has revealed that the 32.6 Kb streptomycin biosynthetic gene cluster was likely inserted into the chromosome of $S$. griseus by horizontal gene transfer, and that the only regulatory element encoded within this region is StrR. ${ }^{28}$ In addition to $strR$, the A-factor cascade induces the expression of StrA, a streptomycin 6-phosphotransferase enzyme and provides the primary mechanism of self-resistance to streptomycin in $S$. griseus. ${ }^{28}$ Enzymatic characterization of StrA has shown that it phosphorylates intracellular streptomycin (6) to yield streptomycin 6-phosphate (42), an inactive compound. ${ }^{35}$ Final activation of 6 only occurs through an extracellular dephosphorylation event mediated by the phosphatase StrK (Scheme 2). ${ }^{36}$

The $strR$ and $strA$ genes are located in the center of the streptomycin cluster with the remaining orfs extending on either side of the initiation operon (Fig 3B). The biochemical mechanisms of several key enzymes involved in the formation of the monomeric units have been described, and candidate orfs for many of the remaining reactions have been deduced from BLAST homology searches. In addition, the order and progression of the individual biosynthetic steps in streptomycin biosynthesis has been studied in depth using a mutagenesis approach. Mutant strains of producing organisms were created using chemical or UV-exposure, growth at increased temperatures and via spontaneous mechanisms. Resulting mutants were blocked at various stages of streptomycin biosynthesis. For a complete list of streptomycin mutant strains, please refer to an in depth review by Piepersberg. ${ }^{7}$ Combined with the molecular and genetic information available on the biosynthetic genes involved in this pathway, these data provide experimental evidence to support a putative enzymatic pathway for streptomycin biosynthesis (Scheme 2).

Production of the streptidine 6-phosphate intermediate (28) begins with the conversion of D-glucose 6-phosphate (15) to MIP (18). The MIP synthase responsible for this activity is not clustered with the streptomycin biosynthetic genes, suggesting that this reaction step is mediated by a MIP synthase that is also involved in primary metabolism. Dephosphorylation of 18 by StrO is then predicted to yield myo-inositol (19). Oxidation of 19 followed by transamination by StsC are predicted to yield scyllo-inosamine (25). The recombinant expression and in vitro activity of the StsC protein as a scyllo-inosose aminotransferase has been confirmed. ${ }^{25}$ This would be followed by phosphorylation at C-4 by StsE and amidino transfer by StrB1 to give amidino-scyllo-inosamine 4-phosphate (27). Conversion to 28 is predicted to occur through a second round of oxidation, transamination, and amidination via StsB, StsA, and either StrB1 or StrB2 (Scheme 2). The biochemical activity of StrB1 has been characterized in vitro and shown to mediate the transfer of the amidino functionality to both positions in 28. ${ }^{37,38}$ However, StrB2 shows high homology with StrB1 and may also function in vivo in the amidination process during streptidine 6-phosphate biosynthesis.

All of the enzymatic functions required for the formation of the second (dTDP-L-dihydrostreptose (34)) and third (CDP-N-

methyl-L-glucosamine (40)) monomeric functional units in 6 are easily predicted from the orfs within the cluster, except for the conversion of dTDP-L-rhamnose (33) to 34 (Scheme 2). However, there are a number of putative orfs within the cluster that do not have homology with other characterized enzymes. Thus, there are several candidates for this conversion. The StrH protein is predicted to mediate the ligation of 28 with 34. ${ }^{36}$ The intermediate 35 is then condensed with 40 to form dihydrostreptomycin 6-phosphate (41). The putative orf that mediates this transition and the following dehydrogenation step are currently unknown. Further conversion of 42 to the active compound only occurs when 42 is released from the cell. The extracellularly-localized streptomycin phosphatase, StrK, completes streptomycin biosynthesis and activates the compound through the dephosphorylation of the C-6 position. ${ }^{36}$

### 2.3 Biosynthesis of bluensomycin

Bluensomycin (23) biosynthesis resembles streptomycin (6) formation except that the streptidine 6-phosphate (28) is replaced with a bluensidine subunit (44) (Fig. 4). ${ }^{22}$ 44 differs from streptidine only in that the guanidino functionality at position 1 is replaced with an O-carbamoyl moiety (Fig 4). Due to these structural similarities, it has been postulated that the genes encoding bluensomycin biosynthesis either share an ancestral evolutionary relationship with the streptomycin biosynthetic gene cluster, or possibly involve a degenerative evolution of the streptomycin biosynthetic gene cluster to yield bluensomycin production. ${ }^{39-41}$ To examine this relationship, a number of genetic and biochemical studies have been done in two different bacterial isolates that produce bluensomycin ($Streptomyces hygroscopicus$ var. glebosus ATCC 14607 and $Streptomyces$ bluensis DSM 40564). ${ }^{25,40,42}$ Preliminary experiments showed that when protein extracts from $S$. hygroscopicus were incubated with L-[guanidino-${ }^{14} \mathrm{C}]$ arginine and carbamoyl phosphate, there was production of both [${ }^{14} \mathrm{C}]$-amidino-scyllo-inosamine 4-phosphate ([${ }^{14} \mathrm{C}]$-27) and [guanidino-${ }^{14} \mathrm{C}]$ bluensidine 6-phosphate (45) (Fig. 4), suggesting that functional enzymes required for the production of the streptidine monomer are retained in the bluensomycin producer. ${ }^{41}$ However, under normal in vivo conditions, the production of streptidine 6-phosphate (28) or the end product streptomycin remains undetected. One possibility to explain this dichotomy is that enzymes within the bluensomycin pathway that function at intermediate stages during bluensidine 6-phosphate (45) production, may require the presence of the O-carbamoyl functionality in order for substrate to be recognized. Thus in vivo, only the production of bluensomycin would be favored.

![Figure 4](#fig4)

It has also been postulated that the gene cassette responsible for the production and incorporation of the first guanidino group into streptidine 6-phosphate (28) has been duplicated in the

This journal is © The Royal Society of Chemistry 2007
Nat. Prod. Rep., 2007, 24, 358–392 | 363
15D-glucose 6-phosphate

18 myo-inositol 1-phosphate
SpcA
'SpeA' *
↓
19 myo-inositol
SpcB
or H
'SpeB' *
↓
24 scyllo-inosose
SpcC
or E
'SpcS2' *
↓
25 scyllo-inosamine
SpcB
or H
'SpeH'
↓
HO
HO
NH₂
46
O OH
SpcC
or E
'SpcS1'
↓
HO
HO
NH₂
47 H₂N OH
SpcG
↓
'SpcM'
↓
HO
HO
NHCH₃
NH OH
H₃C
48 actinamine

29 glucose 1-phosphate
SpcK
'SpcD' *
↓
30 dTDP-D-glucose
SpcJ
'SpcE'
↓
31 4-keto-6-deoxy-
dTDP-D-glucose
Spci
'Spel'
↓
HO
O
HO
OdTDP
49
↓
O
HO
OdTDP
50 dTDP-actinospectose
↓
HO
NHCH₃
OH
O
OH
H
O
CH₃
9 spectinomycin

Scheme 3 Proposed biosynthetic mechanism of spectinomycin biosynthesis. The proposed enzymes involved in the pathway from *S. flavopersicus* (right side) and *S. spectabilis* (left side) are indicated. Enzymes denoted with an * have been biochemically characterized.

streptomycin producer. Whereas in the bluensomycin producer, there has been no gene duplication. In support of this hypothesis, initial analysis of the partial bluensomycin biosynthetic gene cluster (*blm* in *S. hygroscopicus* and *blu* in *S. bluensis*) has revealed the presence of only one aminotransferase (*blmS*), one amidinotransferase (*blmB*), and a novel *orf* encoding an O-carbamoyltransferase (*blmC*) (Fig. 3D).<sup>40,42</sup> Furthermore, Southern analysis using genomic DNA from *S. hygroscopicus* has revealed homologs for two of the three aminotransferases in the streptomycin cluster (*stsC* and *stsS*), whereas a homolog for *stsA* gene could not be found.<sup>25</sup> StsA is believed to function in the second transamination step in streptidine 6-phosphate biosynthesis. Continued sequencing and analysis of the entire *blm/blu* pathway should help to resolve some of the interesting questions addressing the evolutionary relationships and mechanisms of horizontal gene transfer of antibiotic production pathways.

### 2.4 Biosynthesis of spectinomycin

Structurally, spectinomycin (9) is a disaccharide composed of an aminocyclitol monomer (actinamine (48)) and a glucose-6-phosphate-derived moiety (actinospectose) (Scheme 3). Biochemically, the early stages of actinamine and actinospectose biosynthesis are similar to streptomycin and bluensomycin biosynthesis. Furthermore, two partial biosynthetic gene clusters for the production of 9 have been isolated from two different soil bacteria, *Streptomyces flavopersicus* NRLL B 2820 and *Streptomyces spectabilis* ATCC 27741, revealing several homologs with the streptomycin (*str/sts*) and bluensomycin (*blm/blu*) pathways (Fig. 5).<sup>43–45</sup> Unfortunately, each laboratory group involved in the isolation of the spectinomycin cluster used the same abbreviation (*spc*) to annotate the genes within the spectinomycin cluster, and thus, some of the *orfs* from the *S. flavopersicus* cluster share the same name as non-homologous counterparts from the *S. spectabilis* cluster. For the sake of discussion, the genes from *S. flavopersicus* will be used as the primary nomenclature, followed

**S. flavopersicus** (spectinomycin producer)
A
SfORF1 insA spcA spcC spcF spcH spcJ
insA1 strK strA strB spcR spcN spcT spcD spcX spcE spY spcG spcl spcK

**S. spectabilis** (spectinomycin producer)
B
speV 'spcD' 'spcE' spel speH 'spcM' 'spcG' speY 'spcS2' speX speD 'spcS1' speB speA speT speN

Transcription Factor
Phosphotransferase
Involved in actinamine biosynthesis
Involved in actinospectose biosynthesis
Remnant streptomycin biosynthetic genes
Not involved in biosynthesis
Glycosyltransferase
ABC transporter/efflux pump
Transposable elements
Unknown function

Fig. 5 Comparison of biosynthetic gene clusters for spectinomycin production. Graphical representation of the biosynthetic gene clusters for spectinomycin biosynthesis from (A) *S. flavopersicus* and (B) *S. spectabilis*. Homologous *orfs* for the biosynthesis of the actinospectose and actinamine subunits are present in each cluster, however, the arrangement of the genes within the cluster are different.
by the *S. spectabilis* gene of interest in parentheses and in single quotation marks.

Formation of the aminocyclitol portion of the molecule begins with the conversion of D-glucose to D-glucose 6-phosphate (15) and on to myo-inositol 1-phosphate (18). As in streptomycin and bluensomycin biosynthesis, the genes required for these chemical modifications are not clustered with the other genes for spectinomycin biosynthesis (Fig. 5). The formation of 48 proceeds through the dephosphorylation of 18 by the SpcA ('SpeA') myo-inositol monophosphatase enzyme. Characterization of the 'SpeA' protein from *S. spectabilis* has confirmed this activity.⁴³ This reaction is followed by two rounds of oxidation and transamination by the putative dehydrogenases, SpcB ('SpeB') and/or SpcH ('SpeH') and the putative aminotransferases, SpcC ('SpcS2') and/or SpcE ('SpcS1') (Scheme 3). Cell-free activity assays using recombinantly expressed 'SpeB' and 'SpcS2' from *S. spectabilis* demonstrated that these enzymes are involved in the conversion of myo-inositol (19) to scyllo-inosose (24) and of 24 to scyllo-inosamine (25), respectively.⁴³ The final processing of the diamino intermediate to 48 involves the methylation of the amine functionalities at positions 1 and 3 by SpcG ('SpcM'). Since SpcG ('SpcM') is the only methyltransferase isolated within the gene cluster, it is proposed to mediate both methylation events.

The biosynthesis of the actinospectose unit is less clear. Currently, the only characterized enzymatic reaction in this branch is the thymidylation of 29 by the dTDP-glucose synthase SpcK ('SpcD').⁴⁶ Further dehydration via the enzymatic activities of SpcJ ('SpcE') is predicted to yield 4-keto-6-deoxy-dTDP-D-glucose (31) that could then be directly coupled with actinamine (48) and further processed to yield 9 (Scheme 3). Alternatively, 31 may undergo isomerization and dehydration prior to the incorporation with 48. However, the later proposal is less likely due to the instability of the proposed intermediates 49 and 50.

Resistance to 9 is mediated by the phosphotransferase protein SpcN ('SpeN'). Characterization of the SpcN protein from *S. flavopersicus* has shown that SpcN phosphorylates 9 and dihydrospectinomycin in an ATP-dependent manner.⁴⁴ In addition, SpcN displayed low tolerance for alternate substrates, including related aminoglycosides such as 6.⁴⁴ Furthermore, regulation of antibiotic production is likely controlled by the SpcR transcriptional activator. SpcR retains moderate homology with the streptomycin StrR regulator (29% identity/43% similarity).⁴⁴

Interestingly, the flanking regions of the spectinomycin biosynthetic gene cluster in *S. flavopersicus* have revealed the presence of several transposase sequences intermingled with two complete orfs from the streptomycin biosynthetic gene cluster, encoding the streptomycin 6-phosphate phosphatase, StrK and the streptomycin resistance protein, StrA.⁴⁴ Since an alternate phosphatase was not found within the spc cluster, the StrK protein serves as a primary candidate for the extracellular dephosphorylation and activation of spectinomycin. Interestingly, a partial sequence was recovered encoding StrB1. This orf was found to be disrupted and replaced with an incomplete transposase gene sequence. Gene sequence identities for the StrK and StrA proteins are close to 70%, whereas sequence similarity for spc biosynthetic genes with the str/sts pathway typically range from 29 to 43% identity.⁴⁴ Thus, the presence of remnant streptomycin biosynthetic elements adjacent to the spc gene cluster and neighboring transposase sequences suggests that this area of the chromosome has a high level of

---

**Figure 6**

instability and that horizontal gene transfer has likely given rise to the spectinomycin gene cluster in *S. flavopersicus*.

### 2.5 Biosynthesis of fortimicin (astromicin) and related compounds

The fortimicins (*e.g.*, 20) and related compounds, including the istamycins (*e.g.*, 51), dactimicin (52), and sannamycin A (53) constitute a group of pseudodisaccharides that share a number of unique structural features, including an *N*-glycyl functionality (Fig. 6). The biosynthetic origin of fortimicin has been extensively studied by Nakayama and Hasegawa's groups from the Tokyo Research Laboratories of Kyowa Hakko Kogyo Co., LTD. in Japan. In these investigations, the use of blocked mutants from the fortimicin A producer, *M. olivasterospora* KY 11518 that are only capable of producing intermediates in the fortimicin pathway, has been instrumental in understanding the biosynthetic route to fortimicin production, as well as providing a means to uncover the fortimicin biosynthetic gene cluster (*fms*).⁴⁷⁻⁵⁰ Based on the data from blocked mutants, the proposed pathway for fortimicin biosynthesis is initiated by the conversion of myo-inositol (19) to scyllo-inosose (24) with further transamination to form scyllo-inosamine (25) (Scheme 4). dTDP-glucosamine (56) then forms a 1,6 linkage with 25 to form the core structure of the fortimicin class of aminoglycoside antibiotics.

A number of tailoring reactions follow, including the transamination of the C-4 position of 25, instead of the C-3 position found in streptidine-derived aminoglycosides. The C-4 amine functionality is subsequently methylated. Transamination followed by *C*-methylation is then predicted to occur at the C-6′ position of the glucosamine moiety. The resulting intermediate (63) undergoes a didehydroxylation reaction at the C-3′ and C-4′ positions of the glucosamine functionality, followed by the epimerization of the functional groups at the C-3 and C-4 positions of the scyllo-inosamine moiety to yield fortimicin B (67). A glycytransferase is then predicted to mediate the conversion of 67 to 20. 20, as well as other aminoglycosides of this class, can be further modified with a unique *N*-formimidoyl functionality to yield 68. In Scheme 4, the steps in the pathway are indicated where mutants were blocked during fortimicin biosynthesis and the accumulation of intermediate products occurred.

Complementation studies of the blocked mutants of the fortimicin producer were conducted using a genomic DNA library prepared from the wild type *M. olivasterospora*.⁴⁹,⁵⁰ Results from these studies revealed a number of putative genes involved in

This journal is © The Royal Society of Chemistry 2007   Nat. Prod. Rep., 2007, 24, 358–392 | 365
![](https://i.imgur.com/yourimage.png)

Scheme 4 Proposed biosynthetic mechanism of fortimicin biosynthesis. The proposed mechanism for fortimicin biosynthesis was generated through the study of mutant strains blocked at different positions within the pathway.

fortimicin biosynthesis. A complementation map representing the putative gene cluster is shown in Fig. 7. All of the genes represented on this map were isolated by complementation except for *fms13* and *fms14*. The mutant blocked prior to the enzyme reaction mediated by the *fms13* gene product was highly resistant to transformation. Fortunately, work with the sannamycin producer, *Streptomyces sannanensis* IFO 14239, revealed a sannamycin β-glycyl transferase that mediates the conversion of sannamycin B (54) to sannamycin A (53), a homologous enzymatic step to that predicted in the fortimicin pathway. This enzyme was utilized as a heterologous probe to isolate a homologous gene, *fms13* from the fortimicin producer that was localized upstream of *fms3*, -4, and -5 (Fig. 7). The gene *fms14*, on the other hand, was deduced using a reverse genetics approach. N-Formimidoyl transferase activity was detected in extract of wild type *M. olivasterospora* and led to the isolation and purification of the active protein. N-Terminal sequencing and reverse translation was used to predict the gene sequence within this region. PCR was subsequently utilized to clone the *fms14* gene.

Gene sequences for *fms8*, *fms11*, and *fms14* have been obtained, as well as a resistance gene, *fmrO* located downstream of *fms11* (Fig. 7). Complementation analysis with the *fms8* gene sequence led to the rescue of a mutant blocked prior to the didehydroxylation step. Interestingly, the sequence of *fms8* revealed that it shares high similarity with *nmrA*, a neomycin resistance gene known to encode an enzyme that mediates the phosphorylation and inactivation of neomycin (4). In fact, expression of *nmrA* in the blocked mutant of *M. olivasterospora* could also rescue the mutant phenotype, whereas heterologous expression of *fms8* was capable of mediating resistance to both 4 and 20. Thus, the *fms8* gene product is predicted to mediate the phosphorylation of the intermediate and that this phosphorylation event is critical for the further processing of the fortimicin product.

Recently submitted sequence data from two additional bacterial isolates are currently available in the GenBank Database [gene clusters for fortimicin (Accession # AJ628421) and istamycin (Accession # AJ845083)] (Table 1). Interestingly, the putative fortimicin cluster contains *orfs* encoding a MIP phosphatase

Micromonospora olivasterospora KY11518 (fortimicin producer)
![](https://i.imgur.com/yourimage2.png)

Fig. 7 Analysis of the putative fortimicin biosynthetic gene cluster. Complementation analysis revealed the presence of several putative *orfs* involved in fortimicin biosynthesis. Although many of these constructs have not been sequenced, their ability to complement specific mutants within the fortimicin pathway have been established and are indicated in Scheme 4. *orfs* that have been sequenced are indicated with blue arrows.

Table 1 A list of the aminoglycoside/aminocyclitol antibiotic biosynthetic gene clusters and producing organisms that are currently available from GenBank

| ID | Compound       | Producing organism                              | Gene cluster | GenBank#   |
|----|----------------|-------------------------------------------------|--------------|------------|
| 1  | Streptomycin   | Streptomyces griseus DSM 40236                  | str/sts      | AJ862840    |
| 2  | Streptomycin   | Streptomyces griseus                           | str/sts      | YO8763      |
|    |                |                                                 |              | X62567      |
| 3  | Streptomycin   | Streptomyces glaucescens ATCC 13032             | str          | AJ006985    |
| 4  | Spectinomycin  | Streptomyces spectabilis ATCC 27741             | spc          | AF244574    |
| 5  | Spectinomycin  | Streptomyces flavopersicus NRRL 2820            | spc          | U70376      |
| 6  | Fortimicin     | Micromonospora olivasterospora DSM 43868        | for          | AJ628421    |
| 7  | Istamycin      | Streptomyces tenjimariensis ATCC 31603          | ist          | AJ845083    |
| 8  | Kasugamycin    | Streptomyces kasugaensis M 338-M1               | kas          | AB005901    |
|    |                |                                                 |              | AB120043    |
| 9  | Hygromycin A   | Streptomyces hygroscopicus NRRL 2388           | hyg          | DQ314862    |
| 10 | Butirosin      | Bacillus circulans SANK 72073                  | btr          | AB097196    |
| 11 | Butirosin      | Bacillus circulans SANK 72073                  | btr          | AJ494863    |
| 12 | Butirosin      | Bacillus circulans ATCC 21557                  | btr          | AJ781030    |
| 13 | Ribostamycin   | Streptomyces ribosidificus ATCC 21294          | rac/rbm      | AJ748131    |
| 14 | Ribostamycin   | Streptomyces ribosidificus NRRL B-11466         | rib          | AJ744850    |
| 15 | Neomycin       | Streptomyces fradiae MCIMB 8233                 | neo          | AJ843080    |
| 16 | Neomycin       | Streptomyces fradiae NBRC 12773                 | neo          | AB211959    |
| 17 | Neomycin       | Streptomyces fradiae ATCC 10745                 | nac/nem      | AJ786317    |
| 18 | Neomycin       | Streptomyces fradiae DSM 40063                 | neo          | AJ629247    |
| 19 | Gentamicin     | Micromonospora echinospora ATCC 15835          | gnt          | AY524043    |
| 20 | Gentamicin     | Micromonospora echinospora ATCC 15835          | gac/gtm      | MPU575934   |
| 21 | Gentamicin     | Micromonospora echinospora DSM 43036           | gen          | AJ628149    |
| 22 | Kanamycin      | Streptomyces kanamyceticus 21-18               | orf          | AB164642    |
|    |                |                                                 |              | AB254080    |
| 23 | Kanamycin      | Streptomyces kanamyceticus ATCC 12853          | kac/kan      | AJ582817    |
| 24 | Kanamycin      | Streptomyces kanamyceticus DSM 40500           | kan          | AJ628422    |
| 25 | Tobramycin     | Streptomyces tenebrarius ATCC 17920            | tac/tbm      | AJ579650    |
| 26 | Tobramycin     | Streptomyces sp. DSM 40477                      | tob          | AJ810851    |
| 27 | Lividomycin    | Streptomyces lividus CBS 844.73                | liv          | AJ748832    |
| 28 | Paromomycin    | Streptomyces rimosus NRRL 2455                 | par          | AJ628955    |
| 29 | Apramycin      | Streptomyces sp. DSM 40477                     | apr          | AJ629123    |
| 30 | Apramycin      | Streptoalloteichus hindustanus DSM44523T       | apr          | AJ875019    |
| 31 | Hygromycin B   | Streptomyces hygroscopicus DSM 40578           | hyg          | AJ628642    |
| 32 | Acarbose       | Actinoplanes sp. SE 50/110                     | acb          | Y18523      |
| 33 | Validamycin    | Streptomyces hygroscopicus subsp. jinggangensis 5008 | val          | AQ753181    |
|    |                |                                                 |              | DQ164098    |
| 34 | Validamycin    | Streptomyces hygroscopicus subsp. limoneus     | vld          | DQ223652    |

(forA) and a myo-inositol dehydrogenase (forG), supporting a biosynthetic origin that includes a myo-inositol intermediate (Scheme 4). However, the putative istamycin gene cluster lacks any homologous genes for myo-inositol biosynthesis. Alternatively, the istamycin cluster contains an orf that encodes a 2-deoxy-scyllo-inosose synthase, suggesting that the origin of the istamycins is more closely related to 2-deoxystreptamine-derived aminoglycosides than to myo-inositol-derived compounds (Table 2).

2.6 Biosynthesis of kasugamycin

Kasugamycin (21) is a biologically active aminoglycoside produced by the soil bacterium, Streptomyces kasugaensis M 338-M1 and is widely used in Japan as an agricultural supplement to prevent rice blast disease. An in depth series of studies on the biosynthetic origins of kasugamycin have been done using a combination of radioactive and stable isotope-labeled feeding experiments (Fig. 8). Results from these experiments, clearly establish that (1) the D-inositol functionality originates from myo-inositol, (2) the kasugamine moiety is derived from glucose via an N-glucosamine intermediate, and (3) the two carbon side chain and imino functionality originate from glycine. These experiments combined with recent discoveries about the genes required for the production of 21 have led to a proposed pathway for kasugamycin biosynthesis as depicted in Scheme 5.

The identification of a resistance gene, kac²⁷³, from a different strain of S. kasugaensis MB 273-C4 has been instrumental in the cloning and characterization of the putative kasugamycin biosynthetic gene cluster (kas) (Fig. 9). The kac²⁷³ gene sequence was used as a heterologous probe to screen and clone several small DNA fragments from S. kasugaensis M 338-MI. Results included the cloning of a homologous resistance gene designated kac³³⁸. Expression of kac³³⁸ in E. coli conferred resistance to

Table 2 Sequence homology of the enzymes involved in the production of 4,5-disubstituted deoxystreptamine-derived aminoglycosides*

| Predicted function | ID |
|--------------------|----|
|                    | 13 | 14 | 15 | 16 | 17 | 18 | 10 | 11 | 12 | 27 | 28 |
| DOI-synthase       | RbmA | RibC | Neo7 | NeoC | NemA | NeoC | BtrC | BtrC | LivC | ParC |
| DOI-aminotransferase | RbmB | RibS | Neo6 | NeoB | NemB | NeoS | BtrS | BtrR | LivS | ParS |
| Dehydrogenase      | RbmC | RibE | Neo5 | NeoA | NemC | NeoE |      |      | LivE | ParE |
| Glycosyltransferase | RbmD | RibM | Neo8 | NeoD | NemD | NeoM | BtrM | BtrM | LivM | ParM |
| ABC transporter     | RbmE | RibT | Neo9 | NeoE | NemE | NeoT | BtrW |      | LivT | ParT |
| ABC transporter     | RbmF | RibU | Neo10 | NeoF | NemF | NeoU | BtrX |      | LivU | ParU |
| Dehydrogenase      | RbmG | RibQ | Neo11 | NeoG | NemG | NeoQ | BtrQ | BtrQ | LivQ | ParQ |
| Aminotransferase   | RbmH | RibB | Neo18 | NeoN |      | NeoB | BtrB | BtrB | LivB | ParB |
| Acetyltransferase  | RbmI | AacC8 | Neo20 |      |      | AacC8 |      |      |      | aacC8 |
| Putative regulator | RacA | RibI | Neo4 |      | Nac1 | NeoI |      |      | LivI | ParI |
| Putative regulator | RacB | RibH | Neo3 |      | Nac3 | NeoH |      |      | LivH | ParH |
| Putative regulator | RacC | RibG | Neo2 |      | Nac4 | NeoG |      |      | LivG | ParG |
| Phosphotransferase | Rph  | Rph  | Neo1 |      | NeoU | Aph  | AphA |      |      | ParR |
| Putative phosphatase | RacH | RibP | Neo13 |      | NeoI | Nac23 | NeoP | BtrP | BtrP | LivP | ParP |
| Glycosyltransferase | RacI | RibF | Neo15 |      | NeoK |      | NeoF |      | LivF | ParF |
| Deacetylase        | RacJ | RibD | Neo16 |      | NeoL | Nac25 | NeoD |      | LivD | ParD |
| Unknown            | RacK | RibL | Neo17 |      | NeoM | Nac26 | NeoL | BtrL | BtrL | LivL | ParL |
| Unknown            | RacL/RacM | RibA | Neo19 |      | NeoO |      | NeoA | BtrA | BtrA | LivA | ParA |
| Unknown            | RbmX | RibX | Neo14 |      |      | NeoJ | NeoX |      | LivX | ParX |
| Oxidoreductase/epimerase | RbmN | RibN | Neo12 |      | Nac22 | NeoH | NeoN |      | LivN | ParN |
| LuxR response regulator |      |      | Neo21 |      |      | NeoQ | NeoR |      |      |      |
| Putative regulator |      |      | Orf6 |      | NemH | OrfSfrA 10.3 |      |      |      |      |
| UDP-transferase    |      |      |      |      |      |      | BtrD |      |      | BtrD |
| Oxidoreductase     |      |      |      |      |      |      | BtrE |      |      | BtrE |
| Dehydrogenase      |      |      |      |      |      |      | BtrF |      |      | BtrF |
| Unknown            |      |      |      |      |      |      | BtrG |      |      | BtrG |
| Unknown            |      |      |      |      |      |      | BtrH |      |      | BtrH |
| PCP-domain         |      |      |      |      |      |      | BtrI |      |      | BtrI |
| Carbamoyl phosphate |      |      |      |      |      |      | BtrJ |      |      | BtrJ |
| Synthase           |      |      |      |      |      |      |      |      |      |      |
| Decarboxylase      |      |      |      |      |      |      | BtrK |      |      | BtrK |
| Monooxygenase      |      |      |      |      |      |      | BtrO |      |      | BtrO |
| Putative regulator |      |      |      |      |      |      | BtrR1 | BtrW |      |      |
| Oxidoreductase     |      |      |      |      |      |      | BtrU |      | BtrV |      |
| Putative regulator |      |      |      |      |      |      | BtrR2 | BtrU |      |      |

*Sequence comparisons were made using the BLAST database and ClustalW sequence alignment tool. The indicated ID numbers correspond to the ID number listed in Table 1.

kasugamycin, suggesting that $kac^{338}$ mediates self-resistance in the producing strain. ${ }^{60}$ Since analysis of many biosynthetic gene clusters has shown that resistance genes are a common feature in such biosynthetic gene clusters of secondary metabolites, the regions of DNA adjacent to $kac^{338}$ were sequenced to reveal several genes putatively involved in kasugamycin biosynthesis. ${ }^{60-63}$ More recently, an additional gene cluster for kasugamycin biosynthesis was isolated from the strain variant, S. kasugaensis KACC 20262. ${ }^{64}$ The $ksR$ gene locus from this cluster was also shown to mediate kasugamycin resistance.

BLAST analysis of putative kasugamycin biosynthetic genes has allowed the formulation of a proposed biosynthetic pathway for the synthesis of kasugamycin, as presented in Scheme 5. The biosynthesis proceeds through the conjugation of glucose 1-phosphate (29) with UDP followed by transamination at the C-2 position. The enzymes that catalyze these steps have not been found in the sequenced portion of the kas cluster; however, additional biosynthetic genes may still be localized downstream of $kasI$. The $kasF$ gene encodes a protein that shares homology with $N$-acetyltransferases, suggesting that this enzyme functions

S. kasugaensis (kasugamycin producer)

![Diagram](https://i.imgur.com/yourimageurl.png)

Transcription Factor  
Promoter elements  
Involved in kasugamycin biosynthesis  
Resistance gene  
Unknown function  

Fig. 9 Kasugamycin biosynthetic gene cluster from Streptomyces kasugaensis.

producer has two mechanisms of self-resistance; one involving the modification of the compound by acetylation and the second through the transport of the compound out of the cell *via* the KasK, KasL, and KasM transport complex. Overall regulation of pathway expression appears to be controlled by the putative transcription factor KasT. KasT shares high homology with the StrR (50% identity) and SpcR (44% identity) transcriptional regulators from the streptomycin and spectinomycin gene clusters, respectively. Furthermore, initial experiments using gel shift assays have revealed that KasT can physically interact with several putative promoter sites within the *kas* cluster, including the intergenic region between *kasU* and *kasJ*, *kasM* and *kasN*, as well as *kasQ* and *kasR* as depicted in Fig. 9.

### 2.7 Biosynthesis of hygromycin A

Hygromycin A (8) and B (109) were originally isolated from *S.* hygroscopicus* strain NRRL 2388. However, even though they share the same name, 8 has a unique and different structure when compared to the 2-deoxy-scyllo-inosose-derived aminoglycoside 109. Thus, the biosynthesis of each of these compounds will be addressed separately. Hygromycin A (8) has been shown to have broad spectrum antibiotic activity mediated through the inhibition of ribosomal peptidyl transferase activity. Renewed interest in the structure and activity of 8 has been sparked due to its effective use in the treatment of swine dysentery caused by the anaerobic spirochete *Seripulina (Treponema) hyodysenteriae*. The development of detailed structure-activity relationship data has been aided by the synthesis of over 100 analogs of the hygromycin A structure. Results indicate that the aminocyclitol functionality is essential for biological activity, whereas the 6-deoxy-5-keto-D-arabino-hexofuranose moiety can be readily modified without loss of activity. Current work on the biosynthetic origin of 8 is being utilized to isolate and characterize the biosynthetic gene cluster responsible for hygromycin A biosynthesis.

The proposed biosynthetic route to 8 has been deduced primarily from data from isotope-labeled feeding experiments. Both the 6-deoxy-5-keto-D-arabino-hexofuranose and the aminocyclitol moieties of 8 appear to be derived from glucose through a branched biosynthetic pathway (Fig. 10). Biosynthetic feeding experiments have shown that both D-[1,2-<sup>13</sup>C]-glucose and [1-<sup>13</sup>C]-mannose are incorporated into the 6-deoxy-5-keto-D-arabino-hexofuranose

**Scheme 5 Proposed biosynthetic mechanism of kasugamycin biosynthesis**

in the acetylation of the C-2 amine. The N-acetylglucosamine derivative (72) would then undergo a series of dehydration steps by KasD, and the pyridoxamine 5-phosphate (PMP) dependent enzymes, KasR and KasP. This is predicted to be followed by the transamination of the C-4 position by KasC and by the addition of a glycyl functionality to the C-4 amine by KasN. KasN shares homology with glycine oxidase (32% identity) from *Bacillus halodurans* and Fms14 (31% identity), the unique N-formimidoyl fortimicin A synthase. The glycyl intermediate (78) would be further processed by epimerization of the C-2 N-acetyl functionality by KasQ followed by loss of the acetyl group. The KasA protein shares homology with N-acetyl glucosaminyl transferases and is predicted to catalyze the 1,4 linkage of the aminosugar with myo-inositol (19). KasJ is predicted to then mediate the epimerization of the myo-inositol functionality at the C-1 position to yield 21. Further biochemical characterization of pathway intermediates and enzymatic activities is required to confirm the predicted biosynthetic steps.

In addition to the genes predicted to be involved in the actual biosynthesis of kasugamycin, several *orfs* were found to be involved in the regulation or transport of the aminoglycoside (Fig. 9). For example, a cassette of *orfs* including *kasK*, *kasL*, and *kasM* show high homology with ABC and related transporters. Overexpression of these three genes in the *E. coli* strain JM109 conferred resistance to kasugamycin, suggesting that the kasugamycin

**Fig. 10**

The formation of the 3,4-dihydroxy-α-methylcinnamic acid functionality likely requires the activity of a polyketide synthase to elongate a 4-hydroxybenzoyl-CoA starter unit with a methylmalonyl-CoA extender unit as both [carboxy-¹³C]-4-hydroxybenzoic acid and [2,3-¹³C₂]propionate were incorporated into the hygromycin A backbone (Fig. 10).⁶⁸ The origin of the 4-hydroxybenzoyl unit is predicted to be formed via the shikimate pathway as a direct product of chorismate metabolism (Scheme 8). The alternative hypothesis that the 4-hydroxybenzoyl unit originates from the degradation of phenylalanine or tyrosine was ruled out as neither of these labeled compounds was incorporated into the final structure.⁶⁸ Furthermore, growth of the 8 producer in the presence of a shikimate pathway inhibitor, N-(phosphonomethyl)-glycine led to decreased biosynthesis of 8 (20–40% inhibition). Addition of 4-hydroxybenzoic acid (83) to the growth medium restored wild type levels of 8 production.⁶⁸ Reynolds and coworkers recently submitted the DNA sequence of a putative hygromycin A biosynthetic gene cluster to GenBank (DQ314862) (Table 1). Further analysis of this cluster will undoubtedly shed light on the enzymatic details of 8 biosynthesis.

**Scheme 6** Proposed biosynthetic route to the 6-deoxy-5-keto-D-arabino-hexofuranose moiety of hygromycin A.

ring structure, suggesting that the biosynthetic route is initiated by the conversion of glucose 6-phosphate (15) to mannose 1-phosphate (84) via D-fructose 6-phosphate (Scheme 6).⁶⁸ The proposed subsequent steps include the conversion of 84 to an NDP-activated L-fucose (87) (Scheme 6). To generate 87 from NDP-mannose (85) would require the elimination of water to produce an NDP-4-keto-6-deoxy-D-mannose (86), followed by the epimerization of the C-3 and C-5 positions of the hexose ring structure, and a reduction at the C-4 position. Interestingly, a homologous GDP-mannose 4,6-dehydratase required for the conversion of 85 to 86 was recently found in the biosynthetic gene cluster for the structurally related compound antibiotic A201A.⁶⁹ The NDP-L-fucose (87) would then have to undergo ring contraction to the furanose and oxidation at C-5 to yield the 6-deoxy-5-keto-D-arabino-hexofuranose (88) required for attachment to the C-4 hydroxyl of the 3,4-dihydroxy-α-methylcinnamic acid moiety of 8.

The aminocyclitol core structure is also derived from glucose, however, this branch of the pathway is speculated to involve a myo-inositol intermediate (89) to yield the unique neo-inosamine-2 (90) found in the final hygromycin A structure (Scheme 7). Biosynthetic feeding experiments with D-[1,2-¹³C]-glucose showed intact labeling at the C-1 and C-2 positions, consistent with the proposal that 90 is produced via the oxidation and subsequent transamination of the C-5 hydroxyl of myo-inositol (19).⁶⁸ Additional biosynthetic feeding experiments using L-[methyl-¹³C]-methionine demonstrated that S-adenosylmethionine is the source of the methylene bridge between the C-4 and C-5 hydroxyl groups in 8 (Fig. 10).⁶⁸

**Scheme 7** Proposed biosynthetic route to the aminocyclitol moiety of hygromycin A.

**Scheme 8** Proposed biosynthetic route to the 3,4-dihydroxy-α-methylcinnamic acid-derived moiety of hygromycin A.

## 3 Biosynthesis of 2-deoxy-scyllo-inosose-derived aminoglycoside antibiotics

### 3.1 The 2-deoxystreptamine core unit

The core structural motif found in the majority of antibiotics in this class is a 2-deoxystreptamine (95) unit (Fig. 11).⁷⁰ This moiety plays a critical role in the biological function of this class of aminoglycoside antibiotics and has been shown to interact directly with the 16S ribosomal RNA subunit.⁷¹⁻⁷³ The positively charged nature of this group of compounds combined with a high degree of flexibility facilitates their interaction with the negatively charged RNA backbone. The result is that a majority of these antibiotics mediate their action by disrupting mRNA processing and causing the misreading of the genetic code. Thus, a bacteriocidal effect is observed for the bulk of these compounds, rather than a bacteriostatic effect associated with other classes of antibiotics. For a complete review on this process, please refer to Schroeder *et al.*⁷⁴

As described below, 2-deoxy-scyllo-inosose (DOI) synthase is a key metabolic enzyme in the formation of 95, catalyzing the first committed step in the pathway (the conversion of glucose 6-phosphate (15) to DOI (96)) (Scheme 9). Candidate enzymes

for the second enzymatic step, the transamination of the C-3 position, have also been discovered in a number of aminoglycoside producers. Additional oxidation and transamination of the C-1 position are predicted to complete the production of 95 (Scheme 10). Incorporation of 95 into a wide variety of pseudotrisaccharide structural motifs increases the diversity of this class of antibiotics. The resulting compounds are often divided into two major structural groups based on the position of the glycosidic linkages with the 95 moiety. Typical aminoglycosides will show either a 4,5- or 4,6-disaccharide linkage pattern with 95.

For a number of the 4,5- and 4,6-linkage classes of DOI-derived aminoglycosides, further processing of the 2-deoxystreptamine unit occurs to form a common pseudodisaccharide intermediate called paromamine (97). Representatives of the aminoglycosides containing 97 include 4,5-linked antibiotics such as the butirosins (e.g., 10 and 98), the neomycins (e.g., 4 and 99), the lividomycins (e.g., 100), the paromomycins (e.g., 101 and 102), and ribostamycin (103) (Fig. 11). Similarly, 4,6-linked aminoglycosides originating from 97 include the kanamycins (e.g., 3, 104, 105), tobramycins (e.g., 7), gentamicins (e.g., 2, 106, 107) and other related compounds (Fig. 11). DOI-derived aminoglycosides that do not contain a paromamine moiety include the apramycin-related compounds (e.g., 108), and the hygromycin B/destomycin group (e.g., 109) (Fig. 11). Interestingly, in the hygromycin B/destomycin group, some related compounds, such as SS-56-C (111) appear to be derived from a hybrid pathway involving myo-inositol (19) rather than a DOI (96) intermediate.

D-glucose

$\downarrow$

$\begin{aligned}
& \text { HO } \\
& \text { HO } \\
& \text { OPO }_{3}^{2-} \\
& \text { OH }
\end{aligned}$

$\begin{aligned}
& \text { *BtrC } \\
& \text { RbmA/Neo7* } \\
& \text { ParC/LivC }
\end{aligned}$

$\begin{aligned}
& \text { HO } \\
& \text { HO } \\
& \text { O } \\
& \text { OH }
\end{aligned}$

15

$\begin{aligned}
& \text { *BtrS } \\
& \text { RbmB/Neo6* } \\
& \text { ParS/LivS }
\end{aligned}$

$\begin{aligned}
& \text { HO } \\
& \text { HO } \\
& \text { O } \\
& \text { OH }
\end{aligned}$

96 2-deoxy-scyllo-inosose

$\begin{aligned}
& \text { *BtrE } \\
& \text { RbmC/Neo5 } \\
& \text { ParE/LivE }
\end{aligned}$

$\begin{aligned}
& \text { H }_{2} \text { N } \\
& \text { HO } \\
& \text { OH }
\end{aligned}$

116 2-deoxy-scyllo-inosamine

$\begin{aligned}
& \text { H }_{2} \text { N } \\
& \text { HO } \\
& \text { OH }
\end{aligned}$

117 2-deoxy-3-keto-scyllo-inosamine

$\downarrow$

$\begin{aligned}
& \text { *BtrS } \\
& \text { RbmB/Neo6* } \\
& \text { ParS/LivS }
\end{aligned}$

$\begin{aligned}
& \text { BtrM } \\
& \text { RbmD } \\
& \text { Neo8 } \\
& \text { LivM } \\
& \text { ParM }
\end{aligned}$

$\begin{aligned}
& \text { H }_{2} \text { N } \\
& \text { HO } \\
& \text { OH }
\end{aligned}$

95 2-deoxystreptamine

$\begin{aligned}
& \text { NH }_{2} \\
& \text { HO } \\
& \text { O } \\
& \text { H }_{2} \text { N }
\end{aligned}$

$\begin{aligned}
& \text { HO } \\
& \text { O } \\
& \text { H }_{2} \text { N } \\
& \text { OH }
\end{aligned}$

103 ribostamycin

$\begin{aligned}
& \text { R1 } \\
& \text { H }
\end{aligned}$

$\begin{aligned}
& \text { R2 } \\
& \text { OH }
\end{aligned}$

$\begin{aligned}
& \text { HO } \\
& \text { O } \\
& \text { H }_{2} \text { N } \\
& \text { OH }
\end{aligned}$

119 xylostasin

$\begin{aligned}
& \text { H } \\
& \text { OH }
\end{aligned}$

$\begin{aligned}
& \text { OH } \\
& \text { O } \\
& \text { H }_{2} \text { N } \\
& \text { OH }
\end{aligned}$

4 neomycin B

$\begin{aligned}
& \text { HO } \\
& \text { O } \\
& \text { H }_{2} \text { N } \\
& \text { OH }
\end{aligned}$

118 neamine (neomycin A)

$\begin{aligned}
& \text { HO } \\
& \text { O } \\
& \text { H }_{2} \text { N } \\
& \text { OH }
\end{aligned}$

97 paromamine

$\begin{aligned}
& \text { HO } \\
& \text { O } \\
& \text { H }_{2} \text { N } \\
& \text { OH }
\end{aligned}$

101 paromomycin I

$\begin{aligned}
& \text { HO } \\
& \text { O } \\
& \text { H }_{2} \text { N } \\
& \text { OH }
\end{aligned}$

100 lividomycin A

$\begin{aligned}
& \text { HO } \\
& \text { O } \\
& \text { H }_{2} \text { N } \\
& \text { OH }
\end{aligned}$

99 butirosin A

$\begin{aligned}
& \text { R1 } \\
& \text { OH }
\end{aligned}$

$\begin{aligned}
& \text { R2 } \\
& \text { H }
\end{aligned}$

10 butirosin B

$\begin{aligned}
& \text { H } \\
& \text { OH }
\end{aligned}$

$\begin{aligned}
& \text { * denotes enzymes that have been characterized }
\end{aligned}$

---

**Scheme 10** Proposed biosynthetic pathway for the production of 4,5-disubstituted aminoglycosides. Enzymes denoted with an * have been biochemically characterized.

Findings such as these suggest that genetic recombination between antibiotic biosynthetic pathways occurs in nature and can be utilized to create novel secondary metabolites with unique biogenic origins.

### 3.2 Discovery of 2-deoxy-scyllo-inosose synthases

Kakinuma's research group at the Tokyo Institute of Technology has led the research in the understanding of the biosynthetic pathways for 2-deoxystreptamine (95) derived aminoglycosides. This work began with the isolation and purification of the class of enzymes required for the first committed step in the biosynthesis of 95, involving the conversion of glucose 6-phosphate (15) to 2-deoxy-scyllo-inosose (96). Characterization of this enzymatic process began with the study of crude protein extracts from the neomycin (4) and butirosin (10) producers, *Streptomyces fradiae* IFO13147 and *Bacillus circulans* SANK 72073, respectively.

These studies led to the isolation and purification of a single enzyme, 2-deoxy-scyllo-inosose (DOI) synthase that shares high homology with dehydroquinate (DHQ) synthase from the shikimate pathway.

Analysis of the DOI synthase from the butirosin producer (BtrC), revealed that this enzyme mediates the conversion of 15 to 96 through a multi-step process that involves the oxidation of the C-4 position of 15, followed by the elimination of the phosphate group to form an enolate intermediate (113) (Scheme 9). The subsequent reduction of the C-4 position, ring-opening, and an aldol-type condensation between C-1 and C-6 results in the production of 96. Analysis of the primary sequence and biochemical reaction mechanism of BtrC demonstrates that this enzyme class is mechanistically similar to DHQ synthase (34.1% identity with DHQ synthase from *Bacillus subtilis*). BtrC also shares homology with the amino-DHQ synthase, RifG (28.8% identity) from the rifamycin biosynthetic gene cluster.
to DHQ synthase, BtrC has a pH optimum between 7.5–8.5 and requires NAD⁺ as a cofactor.⁸⁵ In addition, both enzymes require a divalent metal cofactor as well. However, BtrC works optimally using Co²⁺ and is completely inhibited by Cu²⁺ or Zn²⁺, whereas DHQ synthase shows high activity with either Zn²⁺ or Co²⁺. Recent mechanism-based inhibitor analysis of the BtrC enzyme has revealed that the conserved residue K141 (K152 in DHQ synthase), is important for enzymatic activity.⁹¹

In addition to BtrC, a number of DOI synthases have recently been isolated from several aminoglycoside biosynthetic gene clusters, including the ribostamycin, neomycin, kanamycin, tobramycin, and gentamicin gene clusters.⁷⁶,⁷⁸,⁷⁹,⁸²,⁹² Comparison of the active site residues of the crystallized DHQ synthase from *Aspergillus nidulans* with the new class of DOI synthases shows several key differences within the active site, including the conversion of R264 and N268 to a glycine residue and an acidic aspartate, respectively. These alterations are highly conserved within all of the available sequences for DOI synthases creating a novel sequence signature for this subfamily. Furthermore, site-directed mutagenesis of DHQ synthase converting R264 to an alanine residue shows dramatically diminished levels of activity, suggesting that the active site and catalytic properties of DHQ synthases and DOI synthases are significantly different.⁹³ Further analysis of the recently reported crystal of BtrC should lend valuable insight into the resulting structural differences in the active site pocket of DOI synthases.⁹⁴

Interestingly, the BtrC protein was originally isolated as a heterodimer containing a 20 kD binding partner.⁸⁹,⁹⁵ Heterologous expression of BtrC in the absence of the binding partner gave full enzymatic activity.⁹⁵ However, it was noted that heterologously expressed BtrC displayed increased protein instability in the absence of the binding partner. Molecular cloning of the gene encoding the BtrC binding protein, *btrC2* showed that *btrC2* is not clustered with *btrC* or the other butirosin biosynthetic genes.⁹⁵ However, the importance of *btrC2* in butirosin biosynthesis is exemplified by the fact that a *btrC2* mutant strain of *B. circulans* grew more slowly and was incapable of antibiotic production. Coexpression and purification of BtrC and BtrC2 in *E. coli* demonstrated that the proteins co-migrated into the same fractions during the purification process and migrated as a single band using native SDS-PAGE analysis. It was also noted that in the coexpressed fractions, BtrC was stable during ammonium sulfate precipitation and dialysis, whereas BtrC expressed alone would aggregate under these conditions. Thus, a major role for BtrC2 in butirosin biosynthesis appears to be the ability to regulate the stability of BtrC. DHQ synthases, on the other hand, do not appear to require such a binding partner.

### 3.3 Biosynthesis of butirosin

A mechanism for the biosynthesis of the butirosins was first proposed based on a combination of biosynthetic feeding experiments and analysis of idiophoretic mutants that were blocked in different phases of butirosin biosynthesis.⁹⁶⁻¹⁰⁰ The proposed pathway includes the formation of 2-deoxystreptamine (95) followed by the joining of this module at the C-4 position with the C-1 position of glucosamine to form the paromamine (97) intermediate (Scheme 10). Oxidation and transamination of 97 are predicted to generate a neamine pseudodisaccharide (118) that is converted to

ribostamycin (103) by the 1–5 linkage of a ribose moiety with 95. Conversion of 103 to butirosin B (10) would then be mediated via the addition of a 4-amino-2-hydroxybutyrate functionality to the C-1 amine of 103 (Scheme 10).

Characterization of the genes for butirosin biosynthesis was first attempted by examining the genetic locus of the self-resistance gene, *butA* from the butirosin producer, *B. circulans*.¹⁰¹ Analysis of the neighboring *orfs*, revealed a putative transport protein, *butB*; however, no additional butirosin biosynthetic genes were located within the region, suggesting that the *butA* resistance gene is not clustered with butirosin biosynthetic genes. Thus, an alternate strategy for isolating the biosynthetic gene cluster was pursued via the purification of enzymatic activities from *B. circulans*, most notably, the isolation and purification of the DOI synthase, BtrC.⁷⁷,¹⁰² A gene walking approach was then used to isolate a biosynthetic gene cluster for butirosin formation (*btr*) that contains 25 *orfs* (Fig. 12A).

The disruption of numerous genes within the cluster, including *btrR1, btrM, btrB, btrC,* and *btrD* demonstrate their involvement in butirosin biosynthesis.⁷⁷,¹⁰² Furthermore the *in vitro* characterization of several recombinant proteins from the cluster, combined with BLAST homology analysis has allowed the proposal of a putative biosynthetic route for the production of butirosin (Scheme 10). As previously mentioned, characterization of BtrC clearly demonstrates its role in the production of DOI (96) from glucose 6-phosphate (15) as the first step in the formation of 95.⁸⁵ The second step involved in the biosynthesis requires the conversion of DOI to 2-deoxy-scyllo-inosamine (116). The enzyme required for this transformation was subsequently identified and characterized at about the same time by two different research groups. Kakinuma’s group demonstrated the activity of BtrS as a PLP-dependent aminotransferase required for the conversion of DOI to 2-deoxy-scyllo-inosamine, through the isolation, cloning and recombinant expression of the *btrS* gene.⁷⁵ Simultaneously, Spencer’s group cloned and expressed the identical gene locus obtaining similar results as those found by Kakinuma.¹⁰³ Unfortunately, due to the timing and overlap of the experiments, two different names were given for the same gene locus (*btrS* by Kakinuma’s group and *btrR* by Spencer’s group). The conversion of 116 to 95 requires the activity of the dehydrogenase (BtrE) and a second transamination that is also mediated by BtrS (Scheme 10).¹⁰⁴

In a branched pathway, similar to streptomycin biosynthesis, the second structural moiety, UDP-glucosamine is proposed to originate from D-glucose.¹⁰² Conversion of D-glucose 1-phosphate (29) to UDP-glucose (69) is proposed to be mediated by BtrD. Inactivation studies disrupting the *btrD* gene locus have demonstrated that the mutant strain is blocked prior to paromamine (97) formation, as antibiotic production could be restored by the inclusion of 97 in the growth medium, but not by the addition of 96 or 95.¹⁰² Further analysis of the *in vitro* activity of BtrD revealed that BtrD is capable of converting D-glucose-1 phosphate (29) to UDP-glucose (69) and to a lesser extent dTDP could be used as a nucleotide donor as well.¹⁰⁵ Interestingly, glucosamine 1-phosphate could also be utilized as a substrate, albeit at reduced rates. Thus, it is proposed that the nucleotidylation conjugation of 29 precedes the transamination step; however, experimental evidence will be required to confirm this process.
B. circulans SANK 72073 (butirosin producer)

btr R1 U R2 T S N M L A B C D E F G H I J K O P V Q W X

S. ribosidificus ATCC 21294 (ribostamycin producer)

racG  
F E D  
neo12 and neo14  
homologs  

rac  
rph  
rbm  
C B A D C B A D E F G N X H K L M N I  

S. fradiae (neomycin producer)  

neo  
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21  

S. tenebrarius (tobramycin producer)  

tac  
tbm  
E D C E B A B C D X  

S. kanamyceticus (kanamycin producer)  

orf  
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21  

M. echinospora (gentamicin producer)  

gac  
gtm  
N M L K J L H G F B A C E I D J A B C D E F G H I K  

Aminoglycoside 3-N-acetyltransferase  
Phosphotransferase  
Ribosomal methyltransferase  
Phosphatase  
N-acetylhexosamine deacetylase  
Regulatory Protein  
Efflux Pump/Transporter  
DOI Synthase  
DOI Aminotransferase  
Dehydrogenase  
Aminotransferase  
Methyltransferase  
Glycosyltransferase  
Btr cassette for side chain assembly  
Conserved hypothetical protein  
Unknown  

Fig. 12 Comparison of the characterized biosynthetic gene clusters for the 2-deoxy-scyllo-inosose-derived aminoglycosides. Homologous genes are compared from the biosynthetic gene clusters for (A) butirosin, (B) ribostamycin, (C) neomycin, (D) tobramycin, (E) kanamycin, (F) gentamicin production.

The putative glycosyltransferase, BtrM, is proposed to mediate the transfer of the glucosamine moiety to 95 yielding 97. A third transamination step is required for the production of the neamine intermediate (118). This could be mediated by one of the putative dehydrogenases in the pathway followed by transamination by BtrS, BtrB, or an as of yet, unidentified aminotransferase. BtrM is then proposed to mediate a second round of glycosylation with NDP-ribose or -xylose as putative substrates. Glycosylation with ribose will yield ribostamycin (103) as an intermediate in butirosin production, whereas glycosylation with xylose will yield xylostasin (119).

Conversion to butirosin A (98) (xylose-containing) or B (10) (ribose-containing) requires the formation and addition of a 4-amino-2-hydroxybutyrate (120) functionality. Incorporation of 120 is of special interest as it has been shown to retain the biological activity of butirosin in many strains of bacteria that show resistance to other aminoglycoside antibiotics. In fact, due to the position of 120 at the C-1 amine of the 2-deoxystreptomine core, several aminoglycoside modifying enzymes, including N-aminoglycoside acetyltransferases, are unable to modify and inactivate butirosin. The usefulness of the 4-amino-2-hydroxybutyrate side chain is further highlighted by the synthetic analogs of kanamycin (3) and dibekacin (126) (amikacin (1) and arbekacin (127), respectively) that are also capable of evading inactivation in drug-resistant bacterial strains (Fig. 13).

Biosynthetic feeding experiments have determined that L-glutamate and γ-aminobutyric acid (GABA) can serve as precursors for the butirosin side chain, whereas ornithine and free 120 were not incorporated. Further analysis of several enzymes within the btr cluster suggests that the formation of the 4-amino-2-hydroxybutyrate side chain requires the loading and covalent thiol linkage of L-glutamate by BtrJ to the ACP-like BtrI protein (Scheme 11). The L-glutamyl-BtrI intermediate (121) is then proposed to undergo decarboxylation by BtrK and α-hydroxylation by BtrO. Interestingly, the amine functionality of the nascent 121-BtrI intermediate appears to undergo

3.4 Biosynthesis of related 4,5-disubstituted aminoglycosides

As indicated above, ribostamycin (103) has long been proposed to be an intermediate in the production of 4,5-disubstituted aminoglycosides, including butirosin (10) and neomycin (4). Support for this hypothesis comes from mutant strains of butirosin- and neomycin-producing bacteria that alternatively produce 103. To further establish this pathway, putative biosynthetic gene clusters for 103 and 4 have been isolated from *Streptomyces ribosidificus* and *S. fradiae*, respectively (Table 1). BLAST analysis of the orfs within these pathways reveals a core set of genes that are shared with the butirosin pathway (Table 2). These include homologs of the DOI synthase (BtrC), the DOI aminotransferase (BtrS), the putative paromamine dehydrogenase (BtrQ), an additional aminotransferase (BtrB), the glycosyltransferase (BtrM), and two proteins with unknown function (BtrA and BtrL). Additional functional homologs include a pair of putative oxidoreductase/dehydrogenase proteins, represented by BtrE and BtrF from the butirosin pathway and Neo5 and Neo12 in the neomycin pathway. Corresponding homologs of Neo5 and Neo12 have also been found in the ribostamycin producer, suggesting that this core set of proteins functions in the biosynthesis of ribostamycin as delineated in Scheme 10. Functional characterization of the DOI synthase, RbmA from the ribostamycin cluster, and the DOI aminotransferase Neo6 from the neomycin cluster have confirmed their biological activity. Most notably, Neo6 was shown to be capable of mediating both transamination reactions required for 2-deoxystreptamine (95) biosynthesis, demonstrating that, as in the case of butirosin biosynthesis, a single aminotransferase is sufficient for the production of 95. Concurrent isolation and expression of the neomycin biosynthetic genes required for 2-deoxystreptamine biosynthesis from a strain variant of *S. fradiae* by Kakinuma and coworkers has confirmed these results (Table 3). In addition, the dehydrogenase, NeoA (homolog of Neo5 and BtrE), was shown to catalyze the oxidation of 116 to 117 prior to conversion to 95 by NeoB (Neo6).

During the direct comparison of the neomycin and ribostamycin gene clusters, an error was discovered in the published sequence for the ribostamycin cluster. In the publication by Subba et al., Fig. 1 shows a comparison of the ribostamycin and butirosin biosynthetic pathways. In this Figure, the order of the ribostamycin genes is correct; however, the orientation of the arrows representing the direction of transcription has inadvertently been reversed. Therefore, the genomic DNA sequence submitted to GenBank was reanalyzed using the NCBI OrfFinder bioinformatics tool, and the corrected version of the ribostamycin gene cluster is presented in Fig. 12. Reanalysis of the genetic sequence also revealed two additional orfs (*rbmN* and *rbmX*) corresponding to the published sequences of *neo12* and *neo14* from the neomycin producer (Fig. 12). Overall, the ribostamycin and neomycin producers share at least 20 homologous biosynthetic proteins with greater than 90% overall identity. These include the core set of orfs predicted for the biosynthesis of ribostamycin (103). Additional homologs are proposed to be involved in the regulation of pathway expression and transport/resistance mechanisms involved in the cellular processing and release of antibiotic compounds. In addition to the high level of homology shared within these pathways, all of the identified homologs are transcribed in the same order, suggesting that these organisms

This journal is © The Royal Society of Chemistry 2007

| ID | Predicted function | 15 | 22 | 23 | 24 | 25 | 26 | 20 | 19 | 21 | 6 | 7 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| DOI-synthase | Neo7 | Orf11 | KanA | TbmA | TobC | GtmA | GntB | GenC | IstC |  |  |  |
| DOI-aminotransferase | Neo6 | Orf10 | KanB | TbmS1 | TobS1 | GtmB | GntF | GenS1 | IstS |  |  |  |
| Dehydrogenase | Neo5 | Orf16 | KanD | KanS2 | GtmD | Gntp | GenE | ForS |  |  |  |
|  | Neo5 | Orf6 | KanK | KanE | TacD | TobE | Gach | ForE |  |  |  |
| Glycosyltransferase | Neo8 | Orf14 | KanE | KanM1 | TbmD | TobM1 | Gtml | GenM1 | ForM | IstM |  |  |
| Dehydrogenase | Neo9 | Orf20 | KanF | KanM2 | TobM2 | GtmG | GntZ | GenM2 |  |  |  |  |
| Aminotransferase | Neo11 | Orf9 | KanL | KanD1 | TobQ | GacJ | GenQ | ForQ | IstQ |  |  |  |
|  | Neo18 | Orf8 | KacL | KanB | TobB | GacE | GenB1 | ForB | IstB |  |  |  |
| Putative regulator | Neo4 | KacM | KanI | GntJ | GenB2 |  |  |  |  |  |  |  |
| Putative regulator | Neo3 | Orf3 | KanH | GacC | GntL | GenB3 |  |  |  |  |  |  |
| Putative regulator | Neo2 | Orf2 | KacN | KanG | GacK | GntW | GenB4 |  |  |  |  |  |
| Phosphotransferase | Neo1 | KacA | KanN | TobN | GtmJ | Gntl | GenP | ForP | IstP |  |  |  |
| Deacetylase | Neo16 | Orf15 | KanC | KanD2 | TobD2 | GtmC | GntC | GenD2 | ForD | IstD |  |  |
| Dehydrogenase | Neo20 | Kac(Orf1) | KanM | KanD3 | TobT |  |  |  |  |  |  |  |
| Acetyltransferase | Neo20 | Kac(Orf1) | KanM | KanA | TobZ |  |  |  |  |  |  |  |
| Carbamoyltransferase |  |  |  | TACA |  |  |  |  |  |  |  |  |
| Putative regulator | Orf5 | KanL | KanR |  |  |  |  |  |  |  |  |  |
| Putative transport protein | Orf7 | KanJ | KanT | TbmE |  |  |  |  |  |  |  |  |
| Dioxygenase | Orf17 | KacB | KanJ |  |  |  |  |  |  |  |  |  |
| Oxidoreductase | Orf18 | KacC | KanK |  |  |  |  |  |  |  |  |  |
| Dehydrogenase |  |  |  |  |  |  |  |  |  |  |  |  |
| Ribosomal methyltransferase | Kmr(Orf19) | Kmr** | FmrO** | GtmH | GntY | GenD3 |  |  |  |  |  |  |
| tRNA ribosyltransferase |  |  |  | GtmF | GrmA | GrmB | FmrB |  |  |  |  |  |
| Methyltransferase |  |  |  | Gacl | GntV | GenO |  |  |  |  |  |  |
| Methyltransferase |  |  |  | Gtml | GntE | GenD1 |  |  |  |  |  |  |
| Unknown |  |  |  | Gnts | GenF | FosF |  |  |  |  |  |  |
| GTP cyclohydrolase |  |  |  | GacD | GntK | GenK | ForK |  |  |  |  |  |
| Efflux pump |  |  |  | GacF | GntM | GenX | FosC |  |  |  |  |  |
| Membrane antipporter |  |  |  | GacA | GntG | GenW | FosC |  |  |  |  |  |
| Putative regulator |  |  |  | GtmK | GntO | GenV | ForY |  |  |  |  |  |
| Regulatory element |  |  |  | Gacl | GntQ | GenY | FosE |  |  |  |  |  |
| 6-Pyruvoyl tetrahydropterin Synthase |  |  |  | GntR | GntN | GenU | FosG |  |  |  |  |  |
| Exporter protein |  |  |  | GacG | GntJ | GenJ | ForJ |  |  |  |  |  |

either shared a recent common ancestor, or that the antibiotic gene cassette encoding aminoglycoside production underwent recent horizontal transfer between these species. The results from these initial publications have since been strengthened by additional data from the cloning and sequencing of putative ribostamycin and neomycin gene clusters from strain variants of *S. ribosidificus* and *S. fradiae*, and with the isolation of putative gene clusters for paromomycin and lividomycin biosynthesis (Table 1).

The paromomycins (*e.g.*, 101 and 102) and lividomycins (*e.g.*, 100) represent 4,5-disubstituted aminoglycosides that arise through the ligation of ribose with paromamine rather than the neamine moiety found in ribostamycin (103), neomycin (4), and butirosin (10) (Scheme 10). Similar to neomycin production, the incorporation of an additional aminosugar unit results in the production of paromomycin I (101). Production of lividomycin A (100) from 101 is then easily envisioned *via* the incorporation of an additional α-D-glucose (Scheme 10). Recently, the putative biosynthetic gene clusters for both 101 and 100 have been submitted to GenBank (Table 1). Comparison of the *orfs* within each of these clusters also reveals striking similarities with the ribostamycin and neomycin gene clusters, suggesting a common evolutionary origin for all of these pathways (Table 2). Within these highly related pathways, the formation of a pseudotrisaccharide core structure incorporating either paromamine (97) or neamine (118) with ribose is highly conserved genetically. Structural variation largely arises in the secondary modification of this core structure with additional sugar or aminosugar residues. Thus, from a genetic perspective it is extremely surprising that only a single putative glycosyltransferase is encoded in each of the pathways. Further biochemical characterization of the activity of glycosyltransferase homologs from the different producers combined with the analysis of mutant strains blocked at different intermediate stages in production may lend insight into the process of antibiotic diversification observed in these related *Streptomyces* species.

### 3.5 Biosynthesis of tobramycin and kanamycin

The kanamycins and tobramycin represent the class of 4,6-disubstituted aminoglycosides that includes additional family members, such as the closely related nebramycins (*e.g.*, 125) (Fig. 13).<sup>8</sup> In this class of compounds the 2-deoxystreptamine (95) is substituted at the C-4 position by an aminoglucose functionality (*i.e.*, 6-amino-6-deoxy-D-glucose in kanamycin A, neosamine in kanamycin B, or D-glucosamine in kanamycin C) and at the C-6 position by a kanosamine moiety (3-amino-3-deoxy-D-glucose) (Fig. 11). Variation in the pattern of amination gives rise to tobramycin (7) and the nebramycins (125) (Scheme 11). Although the kanamycins show broad antibacterial activity, resistance to this antibiotic is widespread and thus, its clinical utility is limited.<sup>112,113</sup> However, synthetic modification of the core kanamycin structure has yielded clinically relevant derivatives, such as amikacin (1), dibekacin (126), and arbekacin (127), that have more resilient activity against antibiotic-resistant strains of bacteria.<sup>114–116</sup> These results have stimulated the interest in the isolation and characterization of the biosynthetic gene clusters from the kanamycin and tobramycin producers for possible use in combinatorial biosynthesis.

Initial detection of genes required for kanamycin and tobramycin biosynthesis was made possible by the isolation and characterization of a dTDP-glucose-4,6-dehydratase gene from *Streptomyces tenebrarius* H6 that shares homology with the StrE dehydratase from the streptomycin gene cluster.<sup>117</sup> *S.* *tenebrarius* H6 produces a number of antibiotics, including kanamycin B (104), tobramycin (7), and apramycin (108).<sup>118–120</sup> Thus, it was unclear to which antibiotic pathway the putative dehydratase belonged. Disruption of this gene locus resulted in mutant strains incapable of producing 104 and 7, but that retained 108 production.<sup>117</sup> Comparative studies in a strain variant of *S.* *tenebrarius* ATCC 17920 led to the isolation of a putative biosynthetic gene cluster for tobramycin biosynthesis.<sup>79</sup> Recombinant expression of the TmbA protein from this cluster confirmed its activity as a DOI synthase related to BtrC.<sup>81</sup> Further analysis of this gene cluster revealed several *orfs* with homology to the core structural genes required for 2-deoxystreptamine biosynthesis, including a DOI aminotransferase (TmbB) and a DOIA dehydrogenase (TacD) (Table 3). Additional biosynthetic components include a glycosyltransferase (TmbD) related to BtrM and an additional dehydrogenase (TacB) and aminotransferase (TacC) related to BtrQ and BtrB, respectively, suggesting that the formation of a paromamine (97) intermediate occurs in a fashion similar to that of the 4,5-disubstituted aminoglycosides.

The subsequent isolation and analysis of the kanamycin biosynthetic gene cluster from different strain variants of *Streptomyces kanamyceticus* provides additional support for the proposed biosynthetic pathway of this group of compounds (Table 3).<sup>80,83</sup> The genetic organization and high homology shared between the tobramycin and kanamycin gene clusters attests to their close evolutionary history (Fig. 12). The core set of genes required for paromamine biosynthesis are highly conserved within these biosynthetic gene clusters (Table 3). Additional analysis reveals a novel cassette of *orfs* that are unique to this class of compounds. These encode a unique set of putative dehydrogenase, oxidoreductase, and dioxygenase proteins as well as an additional aminotransferase that shares homology with BtrS (Table 3). These *orfs* are predicted to be involved in the biosynthesis of the kanosamine moiety. A key difference from the 4,5-disubstituted aminoglycoside pathways, is that the kanamycin and tobramycin gene clusters contain two glycosyltransferase *orfs*, which may explain how the incorporation of the kanosamine moiety occurs at the C-6 position of 2-deoxystreptamine (95).

Additional *orfs* found within the kanamycin gene cluster correspond to regulatory and resistance genes (Table 3). It is worth mentioning that antibiotic-producing organisms typically employ three main mechanisms of resistance to avoid toxicity to their own antibiotics: (1) intracellular modification of the antibiotic, (2) modification or replacement of the target with an antibiotic resistant version, and (3) efficient removal of the antibiotic from the cells using transmembrane efflux pumps.<sup>121</sup> Interestingly, the kanamycin gene cluster appears to include all three resistance mechanisms.<sup>83</sup> Effective modification of kanamycin occurs by acetylation of the 6′ amino functionality by the Kac resistance protein encoded by *orf1*. The Kmr protein encoded by *orf19* is a 16S methyltransferase that effectively modifies the ribosomal target of kanamycin and *orf7* encodes a putative kanamycin efflux protein. Even in the presence of numerous resistance mechanisms, it has been noted that antibiotic-producing strains still display
toxicity to their own antibiotics at high concentrations. This can be problematic in the optimization of strains for the elevated production of antibiotics, and it has been noted that strains that produce increased levels of antibiotics also show increased antibiotic resistance. ${ }^{12}$

Interestingly, in the attempt to optimize the production of kanamycin, Murakami and coworkers noted that the resulting high-producing strains appeared to increase the copy number of the biosynthetic gene cluster. ${ }^{83}$ In a recently published follow-up report, Yanai *et al.* have nicely demonstrated that the high-producing *S. kanamyceticus* strain 12-6 contains a 145 kb amplifiable unit of DNA (AUD) housing among other genes, the kanamycin biosynthetic gene cluster. ${ }^{123}$ Researchers discovered that if the bacteria are grown under the selective pressure of kanamycin that multiple copies of the AUD are effectively amplified within the genome of this organism generating a mixed population of bacteria that contain up to an amazing 36 copies of the kanamycin biosynthetic gene cluster! Removal of the kanamycin selection pressure from the growth medium results in the destabilization of the AUD region and reduction in the high copy number of kanamycin biosynthetic genes. The proposed mechanism for AUD amplification involves a rolling circle model, enabling the effective recombination and amplification of the AUD locus during DNA replication. ${ }^{123}$ Rolling circle amplification is proposed to be mediated by a series of CRISPER motifs (clustered regularly interspaced short palindromic repeats) and provides a highly interesting mechanism for the rearrangement and amplification of antibiotic biosynthetic genes. It will be interesting to see if similar mechanisms are also involved in the horizontal transfer of biosynthetic genes between bacterial species.

### 3.6 Biosynthesis of gentamicin

The gentamicins represent a complex group of 2-deoxystreptamine-derived aminoglycosides produced by a number of actinomycetes from the *Micromonospora* genus. ${ }^{7}$ Other related compounds include sisomicin (128) and verdamicin (129) which are characterized by the incorporation of an unsaturated aminosugar (Fig. 14). ${ }^{8}$ The primary structural difference in the gentamicin class of antibiotics compared with other 4,6-disubstituted aminoglycosides, is the prevalence of C-methyl and N-methyl substituents (Fig. 11). Isolation of a biosynthetic gene cluster for gentamicin production from *M. echinospora* ATCC 15835 was accomplished by two independent research groups, yielding an identical set of orfs (Table 3). ${ }^{82,84}$ Comparison of the gac/gtm cluster with those of other 2-deoxystreptamine-derived aminoglycosides reveals conservation of the same set of core biosynthetic genes required for the biosynthesis of the paromamine (97) intermediate. Similar to

other 4,6-disubstituted aminoglycosides, the gentamicin cluster contains two glycosyltransferases (GtmE and GtmG), suggesting that the ligation of an aminosugar to the C-6 position of 2-deoxystreptamine (95) requires a unique glycosyltransferase. The gentamicin cluster also contains a number of unique features, including a total of 6 aminotransferases, 4 methyltransferases, a putative GTP cyclohydrolase, and several orfs involved in export, antibiotic resistance, and transcriptional regulation (Table 3). A proposed pathway for the biosynthesis of a number of gentamicin analogs from 97 is presented in Scheme 12.

Interestingly, the recent direct submission of putative sequence data for the fortimicin biosynthetic gene cluster has revealed that the fortimicin pathway shares a high degree of homology with the gentamicin gene cluster (Table 3). The fortimicins and gentamicins are both produced by microorganisms within the same genus, *Micromonospora*, suggesting that the clusters may have evolved from a common ancestor. A total of 17 orfs from the putative fortimicin cluster share significant homology with the gentamicin cluster, including a pair of aminotransferases, a pair of dehydrogenases, a glycosyltransferase, and two methyltransferases. The fortimicin pseudodissacharide structure shares similarity with the paromamine-derived portion of the gentamicins, namely the incorporation and modification of glucosamine to yield a purpurosamine analog that has undergone two rounds of transamination, has been didehydroxylated at the C-3′ and C-4′ positions and methylated at the C-6′ position. However, the aminocyclitol moieties of fortimicin and gentamicin are formed through different intermediates. The fortimicins originate from a myo-inositol intermediate, whereas, the gentamicins are produced via a DOI intermediate. Further analysis of the biosynthetic genes supports this proposal, as the fortimicin gene cluster lacks a homolog for DOI synthase (Table 3). Instead, the putative fortimicin proteins ForA and ForG share homology with myo-inositol biosynthetic enzymes and lack corresponding homologs in the gentamicin cluster. Overall, the DOI-derived aminoglycosides represent a robust class of compounds that share many biosynthetic features with the myo-inositol-derived compounds.

---

### 4 Biosynthesis of 2-epi-5-epi-valiolone-derived aminocyclitols

The third prominent class of aminocyclitol natural products contains a C7N aminocyclitol unit in their core structure. Members of this family of natural products include the α-glucosidase inhibitors acarbose (11), the oligostatins (*e.g.*, 137), and the trestatins (*e.g.*, 138), the trehalase inhibitor validamycin A (12), the antibiotic pyralomicin 1a (139), and the antitumor agent cetoniacytone (140) (Fig. 15). Their chemical structures most commonly contain an unsaturated aminocyclitol unit, valienamine (141), although in some compounds modified forms of valienamine are present instead. The antidiabetic agent acarbose (11) and the antifungal antibiotic validamycin A (12) have become prominent members of this family of natural products due to their significant biomedical and agricultural uses. 11 is one of the components in a complex mixture of pseudo-oligosaccharides produced by *Actinoplanes* strain SE 50/110 with potent inhibitory activity against various sugar hydrolytic enzymes. ${ }^{124}$ Currently, it is clinically used as an anti type-II insulin independent antidiabetic agent. ${ }^{125}$ 12,

![Chemical Structures](attachment:chemical_structures.png)

Fig. 14

378 | Nat. Prod. Rep., 2007, 24, 358–392 This journal is © The Royal Society of Chemistry 2007
4.1 Biosynthetic origin of the core C<sub>7</sub>-cyclitols

Through the seminal work of the groups of Pape, Floss, and Rinehart,<sup>128,129</sup> the C<sub>7</sub>N moiety present in validamycin and acarbose has been established to be formed from a C<sub>2</sub>+C<sub>2</sub>+C<sub>3</sub>-derived 7-carbon sugar phosphate related to the pentose phosphate pathway. Later, Floss and co-workers synthesized a series of radioactive and stable-isotope labeled possible precursors and carried out feeding experiments with the acarbose and validamycin producing strains, *Actinoplanes* sp. and *Streptomyces hygroscopicus* var. *limoneus*, respectively. The results indicated that 2-epi-5-epi-valiolone (142) is the common precursor for the biosynthesis of the core C<sub>7</sub>-cyclitol moieties in acarbose (11) and validamycin A (12).<sup>130–132</sup> In the case of 12, besides 142, 5-epi-valiolone (143), valienone (144) and validone (145) were also incorporated (Fig. 16). Similar results have also been shown in the biosynthetic studies of pyralomicin 1a (139), in which 142 and 143 were incorporated into 139.<sup>133</sup>

The involvement of 2-epi-5-epi-valiolone (142) in acarbose and validamycin biosynthesis has also been confirmed by genetic and biochemical approaches.<sup>134–136</sup> Piepersberg and co-workers cloned the biosynthetic gene cluster of acarbose in *Actinoplanes* sp. SE50/110 and found a gene (*acbC*) in the cluster that encodes a protein homologous to DHQ synthases, amino-DHQ synthases, and DOI synthases. The *acbC* gene was heterologously expressed in *Streptomyces lividans* and the product was found to catalyze the conversion of sedoheptulose 7-phosphate (146) to 2-epi-5-epi-valiolone (142).<sup>134</sup> More recently, the biosynthetic gene cluster of validamycin has also become available.<sup>135,136</sup> Using the *acbC* gene as heterologous probe, Deng and co-workers identified the validamycin biosynthetic gene cluster in *Streptomyces hygroscopicus* var. *jinggangensis* 5008, from which *valA*, an *acbC* homolog, was cloned.<sup>135,136</sup> The *valA* gene was heterologously overexpressed in *E. coli* and the product was characterized by TLC and GC-MS, revealing the function of ValA as the 2-epi-5-epi-valiolone synthase. Very recently, an almost identical gene, *vldA*, was identified in another validamycin producing strain *S. hygroscopicus* var. *limoneus*.<sup>137</sup> The recombinant VldA protein, obtained from *E. coli*, efficiently consumed 146, which was generated *in situ* using hydroxypyruvate, ribose 5-phosphate, and transketolase, to give a new product that has an *R<sub>f</sub>* value on TLC similar with that reported for 142.<sup>137</sup> Thus, the biosynthesis of the core aminocyclitol, 2-epi-5-epi-valiolone (142), is mediated by a new family of sugar phosphate cyclases that are mechanistically and structurally similar to DHQ and DOI synthases, but retain strong substrate specificity for 146.

4.2 Biosynthesis of acarbose

The biosynthesis of acarbose has been studied by a combination of conventional feeding experiments using isotopically labeled compounds and by genetic and biochemical approaches.<sup>128,130,134,138–142</sup> From feeding experiments, a number of important facts about acarbose biosynthesis have been learned: (a) the C<sub>7</sub>N aminocyclitol moiety of acarbose is not derived from the shikimate pathway or related pathways, but rather from a C7 sugar phosphate derived from the pentose phosphate pathway;<sup>128</sup> (b) the α-nitrogen of glutamate is the most efficient source of the acarbose nitrogen;<sup>141</sup> (c) the maltose unit of acarbose arises from maltotriose present

Scheme 12 Proposed biosynthetic route to gentamicin and related compounds.

which was isolated from the fermentation broths of *Streptomyces hygroscopicus* var. *limoneus* and *S. hygroscopicus* var. *jinggangensis*, is a trehalase inhibitor widely used in Asia for the treatment of sheath blight disease of rice plants and damping-off of vegetables caused by the fungus *Rhizoctonia solani*.<sup>126,127</sup>

137 oligostatin C m = 0, n = 2

138 Trestatin A (n = 2); R = H

140 cetoniacytone A

141 valienamine

Fig. 15

142 2-epi-5-epi-valiolone
143 5-epi-valiolone
144 valienone
145 validone

Fig. 16

AcbR, is predicted to yield NDP-1-epi-valienol 7-phosphate (152). The AcbA, AcbB, and AcbV proteins are predicted to catalyze the conversion of glucose 6-phosphate (15) to 4-amino-4,6-dideoxy-dTDP-glucose (153), which is then coupled with 152 by either AcbS or AcbI to give dTDP-acarviose 7-phosphate (154). Further extension with two units of glucose yields 11.

### 4.3 Biosynthesis of validamycin

Although the valienamine moiety in both acarbose (11) and validamycin A (12) is derived from the same precursors, 2-epi-5-epi-valiolone (142), further downstream their biosynthetic pathways appear to be different. Feeding experiments with isotopically labeled precursors to the validamycin producing *S. hygroscopicus* var. *limoneus* revealed the incorporation of 142, 143, 144, and 145 into 12 (Fig. 16). Based on the results of these feeding experiments, it was proposed that in validamycin A biosynthesis 142 is epimerized at the C-2 stereocenter followed by dehydration between C-5 and C-6 to give 144. The latter compound was found to be a common precursor for both the unsaturated and saturated cyclitol moieties of 12.

Identification and functional analysis of the biosynthetic gene cluster of validamycin in *S. hygroscopicus* var. *jinggangensis* 5008 has been reported recently by Deng and co-workers, revealing 16 structural genes, 2 regulatory genes, 5 genes related to transport, transposition/integration, tellurium resistance, and another 4 genes with no obvious identity. The involvement of the gene cluster in validamycin biosynthesis was confirmed by deletion of a 30 kb DNA fragment from this cluster in the chromosome.

protein heterologously expressed in *E. coli* revealed the function of this protein as a 2-epi-5-epi-valiolone synthase. Among 16 structural genes, only eight were found to be essential for the synthesis of validamycin A in a heterologous host, *S. lividans* 1326. Those are *valA* (encoding a 2-epi-5-epi-valiolone synthase), *valB* (encoding a nucleotidyltransferase), *valC* (encoding a cyclitol kinase), *valG* (encoding a glycosyltransferase), *valK* (encoding an epimerase/dehydratase), *valL* (encoding a validoxylamine A 7'-phosphate synthase), *valM* (encoding a aminotransferase), and *valN* (encoding a cyclitol reductase) (Fig. 17 and Table 4).<sup>135</sup> In vivo inactivation of the putative glycosyltransferase gene (*valG*) abolished the final attachment of glucose for validamycin (12) production, and resulted in the accumulation of the precursor, validoxylamine A (162) (Scheme 14).<sup>135</sup> Complementation with *valG* restored the normal production of validamycin A, and in vitro enzymatic assays using the recombinant ValG protein demonstrated the glycosylation of validoxylamine A to validamycin A. Based on these genetic and biochemical data as well as the previously reported feeding experiments Deng and co-workers then proposed the biosynthetic pathway to validamycin A as shown in Scheme 14.<sup>135</sup>

More recently, two other validamycin biosynthetic gene clusters from related strains of bacteria have been reported.<sup>137,144</sup> Deng and co-workers identified the biosynthetic gene cluster of jinggangmycin in *S. hygroscopicus* 10–22.<sup>144</sup> The chemical structure of jinggangmycin was recently proposed to be identical to validamycin A, based on comparisons of its HPLC retention time, MS/MS fragmentation pattern, and bioactivity, as well as the sequence similarity of their biosynthetic gene clusters. While only part of the cluster (ca. 6 kb) has been sequenced, three open reading frames identified in the clusters, *orf1-orf3*, showed 99% identity at the nucleotide level and 99% similarity at the amino acid level to *valA*, *valB*, and *valC*, respectively. In addition, Suh and co-workers independently sequenced a 37 kb DNA fragment harboring a complete set of validamycin biosynthetic genes from *S. hygroscopicus* var. *limoneus* KCCM 11405 (IFO 12704).<sup>137</sup> Direct comparison of the validamycin cluster from

---

**Scheme 13** Proposed biosynthetic pathway to acarbose.

that resulted in the loss of validamycin production. In addition, *valA*, a homolog of the *acbC* gene of acarbose biosynthesis, was disrupted in the validamycin producer and found to be essential for the production of 12. Biochemical studies using the ValA

---

**S. hygroscopicus var. jinggangensis 5008**

![Diagram](https://example.com/diagram1.png)

**S. hygroscopicus var. limoneus KCCM 11405**

![Diagram](https://example.com/diagram2.png)

**Validamycin biosynthetic genes**
**Transporter/efflux pump**
**Possible tailoring genes**
**Unknown/unrelated open reading frames**
**Transposase gene**

---

**Fig. 17** Genetic organization of the validamycin biosynthetic gene clusters from *S. hygroscopicus* var. *jinggangensis* 5008 and *S. hygroscopicus* var. *limoneus* KCCM 11405. Blue arrows are genes proposed to be involved in the biosynthesis of validamycin A. Dashed lines show the inverted locations of three almost identical genes in strains 5008 and KCCM 11405.

Table 4 Comparison of related genes present in the val, vld, and acb clusters and their putative function

| Val protein | Vld protein (% sim.) $^{a}$ | Acb protein (% sim.) $^{a}$ | Putative function |
|-------------|-------------------------------|-------------------------------|---------------------|
| ValA        | VldA (100)                   | AcbC (72)                    | 2-epi-5-epi-Valiolone synthase |
| ValB        | VldB (99)                    | AcbR (72)                    | Nucleotidylyltransferase |
| ValC        | VldC (99)                    | AcbM (41)                    | Cyclitol kinase |
| ValD        | VldX (93)                    | —                            | Glyoxalase |
| ValE        | —                            | —                            | Oxidoreductase |
| ValF        | —                            | —                            | Oxidoreductase |
| ValG        | VldK (98)                    | —                            | Glycosyltransferase |
| ValH        | VldJ (99)                    | —                            | Transport protein |
| ValI        | VldI (99)                    | —                            | Glycosylhydrolase |
| ValJ        | VldW (97)                    | —                            | Oxidoreductase |
| ValK        | VldD (100)                   | —                            | Epimerase/dehydratase |
| ValL        | VldE (98)                    | —                            | Validoxylamine A 7'-phosphate synthase |
| ValM        | VldF (99)                    | —                            | Cyclitol aminotransferase |
| ValN        | VldG (99)                    | —                            | Cyclitol reductase |
| ValO        | VldH (96)                    | —                            | Phosphatase/phosphohexomutase |
| ValP        | —                            | —                            | Regulatory protein |
| ValQ        | —                            | —                            | Regulatory protein |
| Orf4        | VldV (100)                   | —                            | Unknown |

$^{a}$ % similarity to the corresponding Val protein at the amino acid level.

---

Scheme 14 Biosynthetic pathway to validamycin proposed by Deng and co-workers.
(ValC) might have broad substrate specificity, activating all these cyclitols and making it possible for them to be incorporated into the biosynthetic pathway. Alternatively, ValC may only utilize valienone (144) and/or validone (145) as substrate. Suh and co-workers<sup>137</sup> proposed a rather intriguing hypothesis regarding the function of *vldC* (Scheme 15). They speculated that VldC is a 5-epi-valiolone 7-kinase that phosphorylates 5-epi-valiolone (143) to its 5-epi-valiolone 7-phosphate (163). The latter compound could be converted to 5-epi-valiolone 5-phosphate (164) by the action of a phosphomutase (VldH) setting the stage for a dehydration reaction leading to 144. However, inactivation of the *vldH* homolog, *valO*, in *S*. hygroscopicus 5008 did not abolish the production of 12,<sup>135</sup> which raises questions whether VldH is really involved in the biosynthesis of 12. Alternatively, VldC was also proposed to function as a glucose kinase, the product of which is converted to NDP-glucose by the action of VldB. Our recent investigation of the function of ValC has shown that the kinase phosphorylates valienone (144) and validone (145) to their corresponding 7-phosphorylated derivatives, but not 2-epi-5-epi-valiolone (142), 5-epi-valiolone (143), or glucose.<sup>†</sup>

(Scheme 14) and/or other reductases/dehydrogenases, e.g., ValE, ValF or ValJ. However, the heterologous expression experiments in *S*. lividans suggest that ValE, ValF and ValJ are not necessary for the biosynthesis of validamycin A.<sup>135</sup> Similarly, inactivation of *vldW*, which corresponds to *valJ*, did not abolish the production of validamycin A in *S*. hygroscopicus var. limoneus,<sup>137</sup> which leaves this part of the pathway unclear. It was proposed that these oxidoreductases might be involved in the biosynthesis of the hydroxylated analogs of 12, validamycins B (165) and/or G (166) (Fig. 18). Transamination of the keto group of 159 catalyzed by the aminotransferase ValM (or VldF) would give validamine 7-phosphate (160).

![Chemical Structure](https://i.imgur.com/1234567.png)

Fig. 18

In the other branch of the pathway, 156 is either phosphorylated at C-1 by an unidentified kinase to generate the 1,7-diphosphate derivative as proposed in acarbose biosynthesis or transformed into valienol 1-phosphate (157) through catalysis by a phosphoglucomutase. The product is subsequently converted to an NDP-valienol derivative (158) by the nucleotidyl transferase ValB (or VldB). Condensation of 158 with 160 is predicted to give phosphorylated validoxylamine A (161). However, an enzyme required for the dephosphorylation of 161 to validoxylamine A (162) has not been found within the cluster.<sup>135</sup> A non-specific sugar phosphatase may be able to carry out this hydrolytic reaction. Finally, 162 is converted to validamycin A (12) through catalysis by the glycosyltransferase ValG using UDP-glucose as the sugar donor. Interestingly, the location of the glycosyltransferase genes is different in the two clusters (Fig. 17). In the 5008 strain, *valG* is located upstream of the main cluster, and in the KCCM 11405 strain the homologous gene (*vldK*) is located downstream of the main cluster.<sup>137</sup> Although Suh and co-workers excluded *vldK* from the proposed structural genes for validamycin biosynthesis, the high similarity (98%) between VldK and ValG suggest that *vldK* is also involved in the biosynthesis of 12.

### 5 Biosynthesis of cyclopentitol-derived aminocyclitols

Cyclopentitol-derived aminocyclitols are relatively rare in nature. However, their biological activities and their unique structures have been attractive objects of investigation. One of the first compounds to be isolated and the hallmark of this class of natural products is pactamycin (13) (Fig. 1), which is an antibiotic antitumor agent obtained from *Streptomyces pactum.*<sup>145</sup> Another interesting cyclopentitol-derived natural product is allosamidin (167), a chitinase inhibitor isolated from *Streptomyces* sp. no. 1713 (Fig. 19).<sup>146</sup> This compound has a unique trisaccharide structure consisting of a cyclopentitol moiety called allosamizoline (168) and two units of *N*-acetyl-D-allosamine. Interestingly, the

† Article submitted to *ChemBioChem*.

This journal is © The Royal Society of Chemistry 2007  
Nat. Prod. Rep., 2007, 24, 358–392 | 383  
regularis. \({ }^{153}\) Adecypenol (172), the adenosine deaminase inhibitor, was first isolated from the culture broth of an actinomycete strain OM-3223 by Omura et al. \({ }^{151}\) 172 is a unique carbocyclic nucleoside consisting of a ring-expanded (“fat”) purine-based nucleic acid structure and a cyclopentene ring. While no study on its biosynthesis has been reported, the formation of the cyclopentene ring in adecypenol can be proposed to resemble that of neplanocin A (170).

Queuosine (173) and epoxyqueuosine (174) are modified nucleosides identified in certain tRNAs in both bacteria and eukarya. However, only bacteria are capable of synthesizing these compounds *de novo*, while eukaryotes obtain queuosine from their diet or intestinal flora. \({ }^{154}\)

Cyclopentitol moieties are also found attached to bacterial-derived hopanoids. Bacteriohopanetetrol derivatives 175 and 176 (Fig. 19) linked either to glucosamine through a glycosidic linkage or to a cyclopentane ring through an ether bond were found in *Zymomonas mobilis*, *Methylobacterium organophilum*, and other bacteria. \({ }^{155,156}\)

Pioneered by the work of Rinehart and co-workers on the biosynthesis of the anti-tumor antibiotic pactamycin (13), \({ }^{157}\) a number of studies on the biosynthesis of cyclopentitol-derived aminocyclitols have been reported in the literature, mainly involving feeding experiments with isotopically labeled precursors. Thus far, except for queuosine (173), \({ }^{158}\) no complete biosynthetic gene cluster for any compound of this class has been published. Therefore, compared to the other classes of aminocyclitol natural products discussed above, relatively little is known about the biosynthesis of cyclopentitol-containing compounds.

Results of feeding experiments with isotopically labeled precursors revealed that most of the cyclopentitol moieties are derived from glucose or glucosamine (177). Labeling and coupling patterns indicated that there are at least two different modes of cyclization involved in the process: (1) cyclization between C-1 and C-5 of glucose (e.g., for pactamycin (13), trehazolin (14), allosamidin (167), bacteriohopanetetrol ether (176)), or (2) between C-2 and C-6 (e.g., for neplanocin A (170) and aristeromycin (171)). These cyclization reactions have been proposed to occur via either a DHQ synthase-like pathway or an *myo*-inositol 1-phosphate synthase-like mechanism. \({ }^{159}\) However, in queuosine biosynthesis, the cyclopentitol unit is derived from the cofactor S-adenosylmethionine (SAM) by an unprecedented mechanism. \({ }^{154,160-163}\)

### 5.1 Biosynthetic origin of pactamycin

Studies on the biosynthetic origin of pactamycin (13) have previously been reviewed by Rinehart et al. \({ }^{157}\) This structurally intriguing antibiotic consists of three cyclic units, in which each is derived from a different biosynthetic pathway (Fig. 20). The cyclopentitol portion of the molecule, which is considered the core moiety, is derived from glucose or its activated analog, whereas the formation of the 6-methylsalicylic acid and the *m*-aminoacetophenone moieties involves a polyketide synthase (PKS) and a pathway similar to the shikimate pathway, respectively. \({ }^{157}\) While the formation of 6-methylsalicylic acid by PKS has been studied to some extent, \({ }^{164-166}\) particularly in fungal systems, the mode of formation of *m*-aminoacetophenone remains elusive. Feeding experiments carried out by Rinehart and co-workers using \([6-{ }^{13} \mathrm{C}]\) glucose and \([2-\) \({ }^{13} \mathrm{C}]\) acetate showed that three carbons (C-2", C-6", and C-8") of

cyclopentanoid structure of allosamizoline is coupled with a dimethylaminooxazine ring (Fig. 19). A similar structure is also found in the potent trehalase inhibitor, trehazolin (14), which is produced by *Micromonospora coriacea* (SANK 62192). \({ }^{147}\) Trehazolin (14) is a pseudodisaccharide with an *N*-glycosidic linkage between the glucose residue and the aminocyclitol derivative, trehalamine (169) (Fig. 19). \({ }^{148}\) Similar to allosamizoline (168), the latter compound consists of a cyclopentitol structure fused with an aminooxazine ring. However, there are some significant differences between the two compounds, which will be discussed in more detail in Section 5.3.

Cyclopentitol moieties are also found in carbocyclic nucleosides and related compounds, e.g., neplanocin A (170), \({ }^{149}\) aristeromycin (171), \({ }^{150}\) adecypenol (172), \({ }^{151}\) queuosine (173), \({ }^{152}\) and its analog epoxyqueuosine (174) (Fig. 19). Neplanocin A (170) and aristeromycin (171) were isolated from *Streptomyces citricolor*, although the former compound is also produced by *Ampullariella*

Fig. 20

$m$-aminoacetophenone were labeled by $[6-{ }^{13} \mathrm{C}]$glucose and only C-8" was labeled by $[2-{ }^{13} \mathrm{C}]$acetate (Fig. 20). ${ }^{167}$ While the labeling of C-8" by $[6-{ }^{13} \mathrm{C}]$glucose presumably occurs via acetate, labeling of C-2" and C-6" is exactly the result that one would expect if a pathway related to shikimate biosynthesis is involved. Based on careful studies of the labeling and coupling pattern observed in the product after feeding experiments with $[\mathrm{U}-{ }^{13} \mathrm{C}_{6}]$glucose, Rinehart concluded that $m$-aminobenzoic acid, the putative precursor of $m$-aminoacetophenone, is derived from 3-deoxy-D-arabino-heptulosonic acid (DAHP) and that the amino group is introduced into the compound at the stage of dehydroquinate (DHQ) or dehydrosikimate, ${ }^{167}$ not at a stage earlier than DHQ as in the biosynthesis of 3-amino-5-hydroxybenzoic acid (AHBA), the common precursor of rifamycin, ansamitocin, geldanamycin, and mitomycin. ${ }^{168-171}$

The cyclopentitol backbone of pactamycin (13) is derived from glucose by carbon-carbon bond formation between C-1 and C-5. ${ }^{172}$ Two plausible routes were proposed: (1) glucose may be converted directly to the cyclopentane unit by cyclization between C-1 and C-5 in a reaction analogous to an aldol cyclization triggered by a carbonyl group either at C-4 or at C-6; or (2) cyclization could proceed through an inosose, which is then rearranged to a cyclopentane. One of the intriguing features in pactamycin biosynthesis is that the compound is highly aminated and methylated. Results of feeding experiments revealed that the two $N$-methyl carbons and three carbons of the substituents on the cyclopentane ring are derived from methionine. ${ }^{172}$

### 5.2 Biosynthetic origin of allosamidin

The biosynthetic origin of allosamidin (167) has been studied using a number of isotopically labeled precursors fed to the cultures of *Streptomyces* sp. AJ 9463. ${ }^{173}$ Feeding experiments with $[1-{ }^{13} \mathrm{C}-]$ and D-$[6-{ }^{13} \mathrm{C}]$glucose as well as doubly labeled D-$[1-{ }^{13} \mathrm{C}, 2-{ }^{15} \mathrm{~N}]$-glucosamine showed that the $N$-acetyl-D-allosamine and allosamizoline (168) moieties of 167 are derived from D-glucosamine (177), whereas the dimethylaminoxazoline moiety was found to be derived from L-arginine (178) and methionine (179) (Fig. 21). ${ }^{173}$ Feeding experiments with $[7-{ }^{14} \mathrm{C}]$-dimethylaminoxazoline (180) showed no incorporation of this compound into allosamidin, suggesting that 180 is not one of the intermediates in the

allosamidin pathway. As $N$-acetyl-D-allosamine is a C-3 epimer of $N$-acetyl-glucosamine, the conversion of $N$-acetyl-glucosamine to $N$-acetyl-D-allosamine has been speculated to be due to the action of an epimerase. ${ }^{173}$

To study the mode of cyclopentane ring formation of 167, Sakuda et al. carried out feeding experiments with a series of deuterium-labeled glucosamine and glucose analogs. ${ }^{174}$ The results showed that deuterium from [3-${ }^{2} \mathrm{H}]$- and [4-${ }^{2} \mathrm{H}]$-D-glucosamine (177), and [1-${ }^{2} \mathrm{H}]$-D-glucose was incorporated at C-3, C-4, and C-1 of allosamizoline (168), respectively, whereas deuterium on C-5 and one of the two deuterium atoms on C-6 of glucosamine were lost during the formation of the cyclopentane ring. Further experiments using stereospecifically labeled $(6 R)-$ and $(6 S)-[6-{ }^{2} \mathrm{H}]$-D-glucose revealed that the $(6 R)$-deuterium, not the $(6 S)$-deuterium, of glucose was incorporated into the proS position at C-6 of 168. Based on these results, Sakuda et al. proposed that the cyclization process involves an intermediate with a 6-aldehyde group and the inversion of stereochemistry of the C-6 hydrogen is due to stereospecific oxidation and reduction at C-6 during the formation of 168. ${ }^{174}$

Besides allosamidin (167), *Streptomyces* sp. AJ 9463 also produced demethylallosamidin (183) and didemethylallosamidin (184) suggesting that the later compounds may be the precursor of 167 (Scheme 16). ${ }^{175}$ However, feeding experiments with ${ }^{14} \mathrm{C}$-labeled 183 and 184 showed that only 183 was incorporated into 167. It was proposed that the first $N$-methyl group is introduced before the cyclization of the aminoxazole ring during 167 biosynthesis. However, feeding experiments with the acyclic derivative compound 181 showed no incorporation of the compound into 167, despite the fact that the compound was efficiently taken up by the cells. ${ }^{175}$ While no experimental data is currently available, the involvement of methylaminoxazole (186) in allosamidin biosynthesis should also be considered. Attachment of 186 to $N$-acetyl-D-allosamide (185) or its dimeric analog would give intermediates, which can be incorporated into allosamidin without the intermediacy of the acyclic intermediate.

### 5.3 Biosynthetic origin of trehazolin

Although the structure of trehalamine (169), the cyclopentane core unit of trehazolin (14), resembles that of allosamizoline (168), there are a number of significant differences between the two moieties. While 168 has a nitrogen atom at the C-2 position, which is derived from glucosamine (177), 169 contains a nitrogen atom

$\begin{aligned}
& \text{(181)} \\
& \text{(182)} \\
& \text{(184 didemethylallosamidin)} \\
& \text{(183 demethylallosamidin)} \\
& \text{(167 allosamidin)} \\
& \text{(185)} \\
& \text{(186)}
\end{aligned}$

Scheme 16 Four different plausible cyclization mechanisms of the cyclopentane ring of trehazolin.

at the C-1 position. Consequently, the oxazine moiety in the two compounds is fused in an opposite orientation. Furthermore, 169 has a hydroxyl group at C-5, while there is no hydroxyl group at C-5 in 168.

Preliminary work by Sakuda and co-workers revealed that the cyclopentane unit of 14 is derived from glucose through a carbon-carbon bond formation between C-1 and C-5 (Fig. 22). As has been shown in the biosynthetic studies of allosamidin (167), the aminooxazine moiety of 14 is also derived from the amidino group of arginine (178). However, in the case of trehalamine (169), the two nitrogen atoms and the quaternary carbon atom all originated from L-arginine (178), while only one of the nitrogen atoms of arginine was incorporated into 168. The second nitrogen atom comes from glucosamine (177), a direct precursor of the cyclopentitol moiety of 168.

$\begin{aligned}
& \text{D-glucose} \\
& \text{14} \\
& \text{178 L-arginine} \\
& \text{D-glucose}
\end{aligned}$

Fig. 22

Based on analogy to the biosynthetic mechanisms of natural products with cyclohexane rings, four different plausible cyclization mechanisms of the cyclopentane ring of trehazolin were proposed from glucose as the tentative substrate (Scheme 17). Mechanisms A–C are analogous to the myo-inositol 1-phosphate synthase mechanism, whereas mechanism D is analogous to the DHQ synthase mechanism. To investigate which one of them would be the most likely mechanism involved in the cyclopentane ring formation in trehazolin (14) biosynthesis, Sakuda and co-

$\begin{aligned}
& \text{Scheme 17 Four different plausible cyclization mechanisms of the} \\
& \text{cyclopentane ring of trehazolin.}
\end{aligned}$

workers carried out feeding experiments with [1-²H]-, [2-²H]-, [4-²H]-, [6,6-²H₂]-, and [U-²H₇]-D-glucose, as well as with [1-¹³C]-D-fructose. The results showed that deuterium on C-1, C-3, C-4, and C-6 of glucose was retained in the cyclopentitol moiety of 14. While the involvement of mechanism D may be ruled out because deuterium on C-4 was retained in the product, it turned out that it is difficult to draw conclusions based upon feeding experiment results alone. For example, mechanisms B and C require hydrogen abstraction from C-6 and C-4 of glucose, respectively. However, the retention of deuterium at C-4 and C-6 of the cyclopentitol moiety still cannot rule out the involvement of mechanisms B or C, as deuterium extracted from C-4 or C-6 may be reincorporated during subsequent reduction at those positions if the cyclization reaction involves an enzyme-bound NAD⁺ or

NADP⁺ as seen in the reaction mechanism of DHQ synthase and myo-inositol 1-phosphate synthase reactions. On the other hand, deuterium loss at C-2 can be interpreted as an indication of the involvement of mechanism A. However, it may also be due to rapid interconversion of glucose and fructose, especially since [1-¹³C]-D-fructose was also efficiently incorporated into the cyclopentitol moiety. Moreover, the deuterium loss was also observed in the glucose part of 14, leaving the crucial question of whether mechanism A is involved in the biosynthesis unanswered. Similar observations have also been reported for the biosynthesis of calditol in *Sulfolobus solfataricus*.^{177}

### 5.4 Biosynthetic origin of aristeromycin and neplanocin A

The biosynthesis of the nucleoside antibiotic aristeromycin (171) was first investigated by Parry and co-workers using radioactive and ¹³C-labeled precursors fed to cultures of the producing strain *Streptomyces citricolor*.^{178} The results of these studies revealed that the cyclopentane ring of 171 is generated by C–C bond formation between C-2 and C-6 of D-glucose and that adenine ring of the antibiotic is synthesized via common pathways present in bacteria. Later, Parry and Johnson reported the synthesis of tetrol 188 and aminotriol 189 and, using isotope dilution studies, concluded that 188 and 189 were intermediates in the biosynthetic pathway to 171 (Scheme 18).^{179,180} However, using a series of cosynthesis experiments with a number of mutants of *S. citricolor* that were blocked in their ability to synthesize neplanocin A (170) and aristeromycin (171), Turner and co-workers successfully identified a compound that they believe to be involved in the biosynthesis of 170 and 171.^{159,181} The compound was identified as the unsaturated tetrol 190, which restores the production of 170 and 171 in mutant CC940 (Scheme 19). The results led them to propose a revised pathway for the biosynthesis of 170 and 171. Compound 190 was then prepared with ¹³C label at C-6 and fed to cultures of *S. citricolor*, giving rise to [6′-¹³C]aristeromycin (35% yield) and [6′-¹³C]neplanocin A (8% yield). Turner and co-workers also prepared the enone 187, enamine 191, and 1-epi-tetrol 192, and fed them to mutant CC940 (Scheme 19). The results showed that only the enone 187 was incorporated into 170 and 171, whereas the enamine 191 and the 1-epi-tetrol 192 were not. Interestingly, when the tetrol 188 and aminotriol 189 were prepared and fed to mutant CC940, no production of 171 was observed.^{181,182} Later, Parry and co-workers reported that partially purified cell-free extract of the aristeromycin producer *S. citricolor* catalyzed the NADPH-dependent reduction of 170 to 171 that involves transfer of the 4 pro-R hydrogen atom of NADPH to the 6′β position of 171.^{183}

---

### Scheme 18 Parry–Johnson’s pathway to neplanocin A and aristeromycin.

---

### Scheme 19 Turner’s pathway to neplanocin A and aristeromycin.

---

### 5.5 Biosynthetic origin of the cyclopentitol moiety of queuosine and epoxyqueuosine

The hypermodified nucleoside queuosine (173), which contains a cyclopentenediol group attached to a (7-aminomethyl)-7-deazaguanine core structure, and its analog epoxyqueuosine (174) are widely found in bacteria and eukarya as products of the post-translational processing of tRNA. These compounds occur exclusively at position 34 in the anticodons of tRNAs coding for the amino acids asparagine, aspartic acid, histidine, and tyrosine.^{154} Seminal work by Kersten and Iwata-Reuyl and co-workers has shown that the cyclopentitol moiety of 173 and 174 is derived from S-adenosylmethionine (SAM) (193) (Scheme 20).^{154,160–163} The enzyme S-adenosylmethionine:tRNA ribosyltransferase-isomerase (QueA) catalyzes the unprecedented ribosyl transfer from the cofactor SAM (193) to a modified tRNA precursor to generate epoxyqueuosine (oQ)-tRNA (196).^{160} The latter is then deoxygenated enzymatically through a cobamide-dependent reduction to generate queuosine (Q)-tRNA (197).^{184} A comprehensive review on the biosynthesis of queuosine and its derivatives has been published rather recently by Iwata-Reuyl.^{185}

---

### 5.6 Biosynthetic origin of the cyclopentitol moiety linked to bacteriohopanetetrol

Bacteriohopanetetrol (198) and its derivatives are triterpenes of bacterial origin that are essential for the survival of the bacteria due to their ability to modulate the properties of the cellular membranes. The bacteriohopanetetrol glycoside (175) and bacteriohopanetetrol ether (176), which contain a glycosidically linked glucosamine (177) and an ether linked cyclopentitol ring, respectively, are among the major constituents of the bacterial membrane-associated hopanoids.^{186} Through the excellent work of Rohmer and others, it is known that the isoprenoid building blocks of the triterpene core structure are derived from the methylerythritol phosphate pathway. Earlier studies feeding ¹³C-labeled glucose to *Zymomonas mobilis* showed that the polyhydroxylated C₅ side chain of 198 is derived from D-ribose and the cyclopentitol moiety attached to it is derived from a hexose through C–C bond formation between C-1 and C-5.^{187} More recent studies using deuterated N-acetyl-D-glucosamine fed to cultures of *Z. mobilis* showed that N-acetyl-D-glucosamine is a common precursor of the glucosamine (177) and the cyclopentitol moieties in 175 and

193 S-adenosylmethionine

$\ce{->[-adenine][+H^+] 194 preQ_1-tRNA}$

$\ce{->[-methionine] 195}$

197 Q-tRNA

$\ce{<=>[B_{12}] 196 oQ-tRNA}$

**Scheme 20** Formation of the cyclopentitol moiety of the hypermodified nucleoside queuosine and epoxyqueuosine.

176, respectively. It was also reported that the incorporation of deuterium into the final products was much more efficient when the producer was fed with $N$-acetyl-$D$-[6,6-${}^{2}$H${}_{2}$]glucosamine (around 20% incorporation for both compounds 175 and 176) than with $D$-[6,6-${}^{2}$H${}_{2}$]glucosamine (around 8% incorporation for 175 and less than 5% for 176). The lower incorporation of $D$-[6,6-${}^{2}$H${}_{2}$]glucosamine was rationalized to be due to isotope dilution, which occurs *via* de novo biosynthesis of glucosamine from the non-labeled glucose present in the cells, as well as catabolism of the deuterated glucosamine the resulting metabolites of which were incorporated into anabolic pathways leading to the isoprene units and the pentose derivatives. The latter notion was supported by the fact that deuterium incorporation was also observed for the fragments corresponding to 198. Interestingly, this phenomenon was not observed in feeding experiments with $N$-acetyl-$D$-[6,6-${}^{2}$H${}_{2}$]glucosamine. Nevertheless, it should be noted that the lower incorporation of $D$-[6,6-${}^{2}$H${}_{2}$]glucosamine may also be a result of a less efficient uptake through the bacterial cell wall due to the free amino group on its structure.

Based on the results of these feeding experiments, it was proposed that UDP-$N$-acetyl-$D$-glucosamine could be a common precursor of both glycoside 175 and ether 176. A glycosylation of bacteriohopanetetrol (198) with UDP-$N$-acetyl-$D$-glucosamine as sugar donor would lead to 175, which could be further converted into 176 as shown in Scheme 21. While the proposed biosynthetic pathway to 176 is attractive, there is no direct evidence to show that

198 bacteriohopanetetrol

$\ce{->[1) glycosylation][UDP-GlcNAc]}$

$\ce{->[2) deacetylation][UDP] 175}$

199 $\ce{<=>[R-O^+] 200}$

$\ce{->[R-O^+] 201 ->[R-O^+] 202}$

176

**Scheme 21** Proposed mechanism for the formation of the cyclopentitol moiety linked to bacteriohopanetetrol.
176 is derived from 175. It is also possible that the cyclopentitol moiety is formed independently, activated (e.g., with phosphate or UDP), and attached to 198. However, such a mechanism would require a unique enzyme for the ether bond formation. Clearly, further experiments are needed to shed more light on the biosynthesis of this interesting natural product.

## Summary and perspectives

Overall, the technological advances in molecular genetics, DNA sequencing, and bioinformatics throughout the last decade, have truly revolutionized our ability to decode and understand how genes control cellular functions. A primary impact of these technologies in the field of microbial secondary metabolism has been the discovery, sequencing, and comparison of the biosynthetic gene clusters required for the production of many biologically active metabolites. These data have allowed the unraveling and utilization of complex metabolic pathways for a multitude of scientific endeavors, including those having medicinal, technological, and other scholarly merit.

At the forefront of this field have been investigations into the biosynthesis of polyketide and nonribosomal peptide-derived compounds. Results from this robust compilation of studies have enabled the expansion of our understanding of secondary metabolite biosynthesis to include additional molecular scaffolds, such as terpene, and aminoglycoside biosynthesis. Although the work on the genetics of aminoglycoside biosynthesis began in the mid 1980's, the growth of this field accelerated only recently when sequence data for many of the gene clusters have become available. This information has been used to target and study discreet functions within pathways through a combination of technologies including targeted gene disruption, recombinant protein expression, and semisynthetic combinatorial experiments. Together with results from conventional feeding experiments, it is now known that the aminocyclitol/aminoglycoside natural products are derived from a number of distinct pathways, which can be differentiated by the mode of formation of their core structures. Synthesis of these core elements involves discrete enzymes that catalyze the key cyclization process of a sugar phosphate intermediate specific to each pathway. For example, the first committed step in the biosynthesis of myo-inositol-derived aminoglycosides and 2-deoxy-scyllo-inosose-derived aminoglycosides is catalyzed by two distinct families of sugar phosphate cyclases, the myo-inositol 1-phosphate synthases and the 2-deoxy-scyllo-inosose (DOI) synthases, respectively. In the case of the C<sub>7</sub>N aminocyclitol compounds, the cyclization is catalyzed by 2-epi-5-epi-valiolone synthases that utilize sedoheptulose 7-phosphate as substrate. While some studies on the biosynthetic origins of pactamycin, trehazolin, and allosamidin have been carried out, little is know about the mode of formation of the unique aminocyclopentitol moieties in these compounds.

The availability of genetic information for aminocyclitol/aminoglycoside biosynthesis has opened new opportunities to study broader questions of pathway expression such as the transcriptional regulation of pathway elements or the complete or partial expression of a pathway in a heterologous host. These data can then be used in the generation of new or derivative aminoglycoside compounds with the end goal of enhancing the biological activity of medicinally relevant compounds. For example,

the StrM (epimerase) and StrL (reductase) from the streptomycin biosynthetic gene cluster have been used in a combinatorial strategy to produce novel derivatives in the methymycin/pikromycin producer, *Streptomyces venezuelae*. Similarly, enzymes involved in the formation of unusual sugars have also been utilized from a number of other biosynthetic pathways as well, including the tylosin, erythromycin, and desosamine pathways. Other challenging end goals include the ability to improve strain expression of biosynthetic pathways to optimize compound production for commercial development.

In addition to the direct utilization of pathway elements for compound production and development, sequence data can be utilized as a bioinformatics tool to isolate predicted biosynthetic gene clusters from known producing organisms and to discover and characterize novel pathways in previously uncharacterized organisms. This technology has already been successfully used in a vast number of experiments (several cited within this review) to isolate and characterize related biosynthetic gene clusters. For example, the *btrC* and *btrS* genes from the butirosin pathway have been used either directly as heterologous probes or in sequence alignments to design degenerate primers for DOI synthases and DOI aminotransferases that have been successfully used to identify the ribostamycin, neomycin, tobramycin, kanamycin and gentamicin gene clusters. Similarly, the validamycin biosynthetic gene cluster was successfully isolated using the 2-epi-5-epi-valiolone synthase gene (*acbC*) from the acarbose pathway as both a heterologous probe and a tool to develop degenerate PCR primers. Even within the scope of these exciting results, the full potential for the use of homologous sequences is only beginning to be realized. For example, reverse genetic approaches using heterologous screening technologies, degenerate primer development, and PCR amplification have enabled the high throughput screening and isolation of homologous enzymes from previously uncharacterized organisms. These organisms can then be selected for further in depth study by more traditional chemistry approaches. Furthermore, the usefulness of these targeted sequences as *in silico* tools to analyze the mother lode of data continually being generated by genome sequencing projects offers tremendous potential to enrich our knowledge of how microorganisms produce secondary metabolites, as well as to understand the evolutionary mechanisms leading to the incredible richness and diversity of microbial compounds produced in nature.

## Acknowledgements

The authors are deeply indebted to Professors H. G. Floss and F. Kudo for their invaluable comments and suggestions for the improvement of this review article.

## References

1. E. J. Begg and M. L. Barclay, *Br. J. Clin. Pharmacol.*, 1995, **39**, 597.
2. L. G. Biehl, *Vet. Clin. North. Am. Food Anim. Pract.*, 1986, **2**, 481.
3. A. Nakagawa, T. Fujimoto, S. Omura, J. C. Walsh, R. L. Stotish and B. George, *J. Antibiot.*, 1987, **40**, 1627.
4. S. Schwarz, C. Kehrenberg, S. A. Salmon and J. L. Watts, *J. Antimicrob. Chemother.*, 2004, **53**, 379.
5. K. L. Rinehart and R. M. Stroshane, *J. Antibiot.*, 1976, **29**, 319.
6. H. G. Floss, P. J. Keller and J. M. Beale, *J. Nat. Prod.*, 1986, **49**, 957.
7. W. Piepersberg, *Biotechnology*, 1995, **28**, 531.
8 W. Piepersberg, P. M. Diaz-Guardamino Uribe, A. Stratmann, H. Thomas, U. F. Wehmeier and C. Zhang, in *Microbial Secondary Metabolites: Biosynthesis, Genetics and Regulation*, ed. F. Fierro and J. F. Martin, Research Signpost, Kerala, India, 2002.

9 T. Mahmud, *Nat. Prod. Rep.*, 2003, **20**, 137.

10 K. J. Catt, L. Hunyady and T. Balla, *J. Bioenerg. Biomembr.*, 1991, **23**, 7.

11 J. H. Exton, *FASEB J.*, 1988, **2**, 2670.

12 B. B. Biswas, B. Ghosh and A. L. Majumder, *Subcell. Biochem.*, 1984, **10**, 237.

13 A. L. Majumder, M. D. Johnson and S. A. Henry, *Biochim. Biophys. Acta*, 1997, **1348**, 245.

14 A. L. Majumder, A. Chatterjee, D. K. Ghosh and M. Majee, *FEBS Lett.*, 2003, **553**, 3.

15 M. W. Loewus, F. A. Loewus, G. U. Brillinger, H. Otsuka and H. G. Floss, *J. Biol. Chem.*, 1980, **255**, 11710.

16 A. J. Stein and J. H. Geiger, *J. Biol. Chem.*, 2002, **277**, 9484.

17 R. Kniewel, J. A. Buglino, V. Shen, T. Chadha, A. Beckwith and C. D. Lima, *J. Struct. Funct. Genomics*, 2002, **2**, 129.

18 R. A. Norman, M. S. McAlister, J. Murray-Rust, F. Movahedzadeh, N. G. Stoker and N. Q. McDonald, *Structure (Cambridge, MA, U.S.)*, 2002, **10**, 393.

19 E. P. Carpenter, A. R. Hawkins, J. W. Frost and K. A. Brown, *Nature*, 1998, **394**, 299.

20 K. L. Kavanagh, M. Klimacek, B. Nidetzky and D. K. Wilson, *J. Biol. Chem.*, 2002, **277**, 43433.

21 F. Tian, J. L. Montchamp and J. W. Frost, *J. Org. Chem.*, 1996, **61**, 7373.

22 W. Piepersberg, in *Streptomycin and Related Antibiotics*, ed. L. C. Vin- ing and C. Stuttard, Butterworth Heinemann, Newton, MA, 1995.

23 J. Distler, A. Ebert, K. Mansouri, K. Pissowatzki, M. Stockmann and W. Piepersberg, *Nucleic Acids Res.*, 1987, **15**, 8041.

24 J. Distler, C. Braun, A. Ebert and W. Piepersberg, *Mol. Gen. Genet.*, 1987, **208**, 204.

25 J. Ahlert, J. Distler, K. Mansouri and W. Piepersberg, *Arch. Microbiol.*, 1997, **168**, 102.

26 S. Beyer, J. Distler and W. Piepersberg, *Mol. Gen. Genet.*, 1996, **250**, 775.

27 S. Beyer, G. Mayer and W. Piepersberg, *Eur. J. Biochem.*, 1998, **258**, 1059.

28 A. Tomono, Y. Tsai, H. Yamazaki, Y. Ohnishi and S. Horinouchi, *J. Bacteriol.*, 2005, **187**, 5595.

29 S. Horinouchi, *Front. Biosci.*, 2002, **7**, d2045.

30 N. Ando, N. Matsumori, S. Sakuda, T. Beppu and S. Horinouchi, *J. Antibiot.*, 1997, **50**, 847.

31 T. Umeyama, P. C. Lee, K. Ueda and S. Horinouchi, *Microbiology*, 1999, **145(9)**, 2281.

32 Y. Ohnishi, S. Kameyama, H. Onaka and S. Horinouchi, *Mol. Microbiol.*, 1999, **34**, 102.

33 S. K. Shin, D. Xu, H. J. Kwon and J. W. Suh, *FEMS Microbiol. Lett.*, 2006, **259**, 53.

34 N. Saito, K. Kurosawa, J. Xu, S. Okamoto and K. Ochi, *Actinomyce- tologica*, 2003, **17**, 47.

35 C. K. Lim, M. C. Smith, J. Petty, S. Baumberg and J. C. Wootton, *J. Gen. Microbiol.*, 1989, **135**, 3289.

36 K. Mansouri and W. Piepersberg, *Mol. Gen. Genet.*, 1991, **228**, 459.

37 E. Fritsche, A. Bergner, A. Humm, W. Piepersberg and R. Huber, *Biochemistry*, 1998, **37**, 17664.

38 K. Pissowatzki, K. Mansouri and W. Piepersberg, *Mol. Gen. Genet.*, 1991, **231**, 113.

39 J. B. Walker, *J. Biol. Chem.*, 1974, **249**, 2397.

40 J. B. Walker, *J. Bacteriol.*, 1990, **172**, 5844.

41 J. B. Walker, *Methods Enzymol.*, 1975, **43**, 429.

42 Y. G. Jung, S. H. Kang, C. G. Hyun, Y. Y. Yang, C. M. Kang and J. W. Suh, *FEMS Microbiol. Lett.*, 2003, **219**, 285.

43 J. You-Young, S. Kim, Y. Yang, C. Kang, J. Sohng and J. Suh, *J. Microbiol. Biotechnol.*, 2003, **13**, 906.

44 T. V. M. Stumpp, *Biosynthese und Resistenz in den Produzen- tenstNmmen der Gattung Streptomyces*, Thesis, University of Stuttgart, 2003.

45 J. K. Sohng, T. J. Oh, J. H. Cha, J. J. Hahn, J. W. Kim, J. W. Suh and H. C. Lee, *J. Biochem. Mol. Biol. Biophys.*, 2001, **5**, 209.

46 C. Hyun, S. S. Kim, J. K. Sohng, J. Hahn, J. Kim and J. Suh, *FEMS Microbiol. Lett.*, 2000, **183**, 183.

47 Y. Odakura, H. Kase, S. Itoh, S. Satoh, S. Takasawa, K. Takahashi, K. Shirahata and K. Nakayama, *J. Antibiot.*, 1984, **37**, 1670.

48 M. Hasegawa, *Gene*, 1992, **115**, 85.

49 T. Dairi, T. Ohta, E. Hashimoto and M. Hasegawa, *Mol. Gen. Genet.*, 1992, **232**, 262.

50 T. Dairi, T. Ohta, E. Hashimoto and M. Hasegawa, *Mol. Gen. Genet.*, 1992, **236**, 39.

51 T. Ohta, E. Hashimoto and M. Hasegawa, *J. Antibiot.*, 1992, **45**, 1167.

52 T. Dairi, K. Yamaguchi and M. Hasegawa, *Mol. Gen. Genet.*, 1992, **236**, 49.

53 H. Umezawa, M. Hamada, Y. Suhara, T. Hashimoto and T. Ikekawa, *Antimicrob. Agents Chemother.*, 1965, **5**, 753.

54 T. Ishiyama, I. Hara, M. Matsuoka, K. Sato, S. Shimada, R. Izawa, T. Hashimoto, M. Hamada, Y. Okami, T. Takeuchi and H. Umezawa, *J. Antibiot.*, 1965, **18**, 115.

55 Y. Fukagawa, T. Sawa, T. Takeuchi and H. Umezawa, *J. Antibiot.*, 1968, **21**, 50.

56 Y. Fukagawa, T. Sawa, T. Takeuchi and H. Umezawa, *J. Antibiot.*, 1968, **21**, 182.

57 Y. Fukagawa, T. Sawa, T. Takeuchi and H. Umezawa, *J. Antibiot.*, 1968, **21**, 185.

58 Y. Fukagawa, T. Sawa, I. Homma, T. Takeuchi and H. Umezawa, *J. Antibiot.*, 1968, **21**, 358.

59 Y. Fukagawa, T. Sawa, I. Homma, T. Takeuchi and H. Umezawa, *J. Antibiot.*, 1968, **21**, 410.

60 S. Ikeno, T. Tsuji, K. Higashide, N. Kinoshita, M. Hamada and M. Hori, *J. Antibiot.*, 1998, **51**, 341.

61 S. Ikeno, D. Aoki, M. Hamada, M. Hori and K. S. Tsuchiya, *J. Antibiot.*, 2006, **59**, 18.

62 S. Ikeno, D. Aoki, K. Sato, M. Hamada, M. Hori and K. S. Tsuchiya, *J. Antibiot.*, 2002, **55**, 1053.

63 S. Ikeno, Y. Yamane, Y. Ohishi, N. Kinoshita, M. Hamada, K. S. Tsuchiya and M. Hori, *J. Antibiot.*, 2000, **53**, 373.

64 Y. Y. Jo, J. Liu, Y. Y. Jin, Y. Y. Yang and J. W. Suh, *J. Microbiol. Biotechnol.*, 2005, **15**, 491.

65 R. L. Mann, R. M. Gale and R. F. Van Abeele, *Antibiot. Chemother.*, 1953, **3**, 1279.

66 R. L. Mann and W. W. Bromer, *J. Am. Chem. Soc.*, 1958, **80**, 2714.

67 S. F. Hayashi, L. J. Norcia, S. B. Seibel and A. M. Silvia, *J. Antibiot.*, 1997, **50**, 514.

68 E.-S. E. Habib, J. N. Scarsdale and K. A. Reynolds, *Antimicrob. Agents Chemother.*, 2003, **47**, 2065.

69 I. Saugar, E. Sanz, M. A. Rubio, J. C. Espinosa and A. Jimenez, *Eur. J. Biochem.*, 2002, **269**, 5527.

70 G. F. Busscher, F. P. Rutjes and F. L. van Delft, *Chem. Rev.*, 2005, **105**, 775.

71 D. Fourmy, M. I. Recht, S. C. Blanchard and J. D. Puglisi, *Science*, 1996, **274**, 1367.

72 D. Fourmy, M. I. Recht and J. D. Puglisi, *J. Mol. Biol.*, 1998, **277**, 347.

73 D. Moazed and H. F. Noller, *Nature*, 1987, **327**, 389.

74 R. Schroeder, C. Waldsich and H. Wank, *EMBO J.*, 2000, **19**, 1.

75 H. Tamegai, E. Nango, M. Kuwahara, H. Yamamoto, Y. Ota, H. Kuriki, T. Eguchi and K. Kakinuma, *J. Antibiot.*, 2002, **55**, 707.

76 F. Huang, S. F. Haydock, T. Mironenko, D. Spiteller, Y. Li and J. B. Spencer, *Org. Biomol. Chem.*, 2005, **3**, 1410.

77 F. Kudo, M. Numakura, H. Tamegai, H. Yamamoto, T. Eguchi and K. Kakinuma, *J. Antibiot.*, 2005, **58**, 373.

78 B. Subba, M. K. Kharel, H. C. Lee, K. Liou, B. G. Kim and J. K. Sohng, *Mol. Cells*, 2005, **20**, 90.

79 M. K. Kharel, B. Subba, H. C. Lee, K. Liou, J. S. Woo and J. K. Sohng, *Biotechnol. Lett.*, 2003, **25**, 2041.

80 M. K. Kharel, B. Subba, D. B. Basnet, J. S. Woo, H. C. Lee, K. Liou and J. K. Sohng, *Arch. Biochem. Biophys.*, 2004, **429**, 204.

81 M. K. Kharel, D. B. Basnet, H. C. Lee, K. Liou, J. S. Woo, B. G. Kim and J. K. Sohng, *FEMS Microbiol. Lett.*, 2004, **230**, 185.

82 M. K. Kharel, D. B. Basnet, H. C. Lee, K. Liou, Y. H. Moon, J. J. Kim, J. S. Woo and J. K. Sohng, *Mol. Cells*, 2004, **18**, 71.

83 K. Yanai and T. Murakami, *J. Antibiot.*, 2004, **57**, 351.

84 J. Unwin, S. Standage, D. Alexander, T. Hosted, Jr., A. C. Horan and E. M. Wellington, *J. Antibiot.*, 2004, **57**, 436.

85 F. Kudo, H. Tamegai, T. Fujiwara, U. Tagami, K. Hirayama and K. Kakinuma, *J. Antibiot.*, 1999, **52**, 559.

86 N. Yamauchi and K. Kakinuma, *J. Org. Chem.*, 1995, **60**, 5614.

87 F. Kudo, N. Yamauchi, R. Suzuki and K. Kakinuma, *J. Antibiot.*, 1997, **50**, 424.

88 Z. Huang, K. Kakinuma and T. Eguchi, *Bioorg. Chem.*, 2005, **33**, 82.

89 F. Kudo, Y. Hosomi, H. Tamegai and K. Kakinuma, *J. Antibiot.*, 1999, **52**, 81.

90 P. R. August, L. Tang, Y. J. Yoon, S. Ning, R. Muller, T. W. Yu, M. Taylor, D. Hoffmann, C. G. Kim, X. Zhang, C. R. Hutchinson and H. G. Floss, *Chem. Biol.*, 1998, **5**, 69.

91 E. Nango, T. Eguchi and K. Kakinuma, *J. Org. Chem.*, 2004, **69**, 593.

92 M. L. Thuy, M. K. Kharel, R. Lamichhane, H. C. Lee, J. W. Suh, K. Liou and J. K. Sohng, *Biotechnol. Lett.*, 2005, **27**, 465.

93 A. Park, H. K. Lamb, C. Nichols, J. D. Moore, K. A. Brown, A. Cooper, I. G. Charles, D. K. Stammers and A. R. Hawkins, *Protein Sci.*, 2004, **13**, 2108.

94 E. Nango, T. Kumasaka, T. Sato, N. Tanaka, K. Kakinuma and T. Eguchi, *Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.*, 2005, **61**, 709.

95 H. Tamegai, E. Nango, A. Koike-Takeshita, F. Kudo and K. Kakinuma, *Biosci., Biotechnol. Biochem.*, 2002, **66**, 1538.

96 T. Furumai, K. Takeda, A. Kinumaki, Y. Ito and T. Okuda, *J. Antibiot.*, 1979, **32**, 891.

97 K. Takeda, A. Kinumaki, S. Okuno, T. Matsushita and Y. Ito, *J. Antibiot.*, 1978, **31**, 1039.

98 K. Takeda, A. Kinumaki, H. Hayasaka, T. Yamaguchi and Y. Ito, *J. Antibiot.*, 1978, **31**, 1031.

99 K. Takeda, K. Aihara, T. Furumai and Y. Ito, *J. Antibiot.*, 1979, **32**, 18.

100 D. Takeda, S. Okuno, Y. Ohashi and T. Furumai, *J. Antibiot.*, 1978, **31**, 1023.

101 E. ubert-Pivert and J. Davies, *Gene*, 1994, **147**, 1.

102 Y. Ota, H. Tamegai, F. Kudo, H. Kuriki, A. Koike-Takeshita, T. Eguchi and K. Kakinuma, *J. Antibiot.*, 2000, **53**, 1158.

103 F. Huang, Y. Li, J. Yu and J. B. Spencer, *Chem. Commun.*, 2002, 2860.

104 K. Yokoyama, F. Kudo, M. Kuwahara, K. Inomata, H. Tamegai, T. Eguchi and K. Kakinuma, *J. Am. Chem. Soc.*, 2005, **127**, 5869.

105 F. Kudo, K. Kawabe, H. Kuriki, T. Eguchi and K. Kakinuma, *J. Am. Chem. Soc.*, 2005, **127**, 1711.

106 T. L. Nagabhushan, G. H. Miller and M. J. Weinstein, in *Structure-activity relationships in aminoglycoside-aminocyclitol antibiotics*, ed. A. Whelton and H. C. Neu, Dekker, New York, 1982.

107 T. Yukita, H. Nishida, T. Eguchi and K. Kakinuma, *J. Antibiot.*, 2003, **56**, 497.

108 Y. Li, N. M. Llewellyn, R. Giri, F. Huang and J. B. Spencer, *Chem. Biol.*, 2005, **12**, 665.

109 T. Fujiwara, T. Tanimoto, K. Matsumoto and E. Kondo, *J. Antibiot.*, 1978, **31**, 966.

110 H. Baud, A. Betencourt, M. Peyre and L. Penasse, *J. Antibiot.*, 1977, **30**, 720.

111 F. Kudo, Y. Yamamoto, K. Yokoyama, T. Eguchi and K. Kakinuma, *J. Antibiot.*, 2005, **58**, 766.

112 K. Smalla, L. S. Van Overbeek, R. Pukall and J. D. Van Elsas, *FEMS Microbiol. Ecol.*, 1993, **13**, 47.

113 L. G. Leff, J. R. Dana, J. V. McArthur and L. J. Shimkets, *Appl. Environ. Microbiol.*, 1993, **59**, 417.

114 H. Kawaguchi, T. Naito, S. Nakagawa and K. I. Fujisawa, *J. Antibiot.*, 1972, **25**, 695.

115 S. Kondo, K. Inuma, H. Yamamoto, K. Maeda and H. Umezawa, *J. Antibiot.*, 1973, **26**, 412.

116 H. Umezawa, S. Umezawa, T. Tsuchiya and Y. Okazaki, *J. Antibiot.*, 1971, **24**, 485.

117 Y. Du, T. Li, Y. G. Wang and H. Xia, *Curr. Microbiol.*, 2004, **49**, 99.

118 Y. Han, L. Mu, J. Li and X. Chen, *Kangshengsu*, 1986, **11**, 183.

119 R. Q. Thompson and E. A. Presti, *Antimicrob. Agents Chemother.*, 1967, **7**, 332.

120 H. D. Zhang, X. Wu and S. Liu, *Weishengwu. Xuebao*, 1987, **27**, 357.

121 H. Umezawa, *Jpn. J. Antibiot.*, 1979, **32 Suppl**, S1.

122 R. Crameri and J. E. Davies, *J. Antibiot.*, 1986, **39**, 128.

123 K. Yanai, T. Murakami and M. Bibb, *Proc. Natl. Acad. Sci. U. S. A.*, 2006, **103**, 9661.

124 D. D. Schmidt, W. Frommer, B. Junge, L. Muller, W. Wingender, E. Truscheit and D. Schafer, *Naturwissenschaften*, 1977, **64**, 535.

125 L. K. Campbell, J. R. White and R. K. Campbell, *Ann. Pharmacother.*, 1996, **30**, 1255.

126 M. Shibata, K. Mori and M. Hamashima, *J. Antibiot.*, 1982, **35**, 1422.

127 N. Asano, T. Yamaguchi, Y. Kameda and K. Matsui, *J. Antibiot.*, 1987, **40**, 526.

128 U. Degwert, R. van Hulst, H. Pape, R. E. Herrold, J. M. Beale, P. J. Keller, J. P. Lee and H. G. Floss, *J. Antibiot.*, 1987, **40**, 855.

129 T. Toyokuni, W. Z. Jin and K. L. Rinehart, Jr., *J. Am. Chem. Soc.*, 1987, **109**, 3481.

130 T. Mahmud, I. Tornus, E. Egelkrout, E. Wolf, C. Uy, H. G. Floss and S. Lee, *J. Am. Chem. Soc.*, 1999, **121**, 6973.

131 H. Dong, T. Mahmud, I. Tornus, S. Lee and H. G. Floss, *J. Am. Chem. Soc.*, 2001, **123**, 2733.

132 T. Mahmud, S. Lee and H. G. Floss, *Chem. Rec.*, 2001, **1**, 300.

133 H. Naganawa, H. Hashizume, Y. Kubota, R. Sawa, Y. Takahashi, K. Arakawa, S. G. Bowers and T. Mahmud, *J. Antibiot.*, 2002, **55**, 578.

134 A. Stratmann, T. Mahmud, S. Lee, J. Distler, H. G. Floss and W. Piepersberg, *J. Biol. Chem.*, 1999, **274**, 10889.

135 L. Bai, L. Li, H. Xu, K. Minagawa, Y. Yu, Y. Zhang, X. Zhou, H. G. Floss, T. Mahmud and Z. Deng, *Chem. Biol.*, 2006, **13**, 387.

136 Y. Yu, L. Bai, K. Minagawa, X. Jian, L. Li, J. Li, S. Chen, E. Cao, T. Mahmud, H. G. Floss, X. Zhou and Z. Deng, *Appl. Environ. Microbiol.*, 2005, **71**, 5066.

137 D. Singh, M. J. Seo, H. J. Kwon, A. Rajkarnikar, K. R. Kim, S. O. Kim and J. W. Suh, *Gene*, 2006, **376**, 13.

138 C. S. Zhang, A. Stratmann, O. Block, R. Bruckner, M. Podeschwa, H. J. Altenbach, U. F. Wehmeier and W. Piepersberg, *J. Biol. Chem.*, 2002, **277**, 22853.

139 C. S. Zhang, M. Podeschwa, H. J. Altenbach, W. Piepersberg and U. F. Wehmeier, *FEBS Lett.*, 2003, **540**, 47.

140 K. Arakawa, S. G. Bowers, B. Michels, V. Trin and T. Mahmud, *Carbohydr. Res.*, 2003, **338**, 2075.

141 S. Lee and E. Egelkrout, *J. Antibiot.*, 1998, **51**, 225.

142 S. Lee, B. Sauerbrei, J. Niggemann and E. Egelkrout, *J. Antibiot.*, 1997, **50**, 954.

143 U. F. Wehmeier and W. Piepersberg, *Appl. Microbiol. Biotechnol.*, 2004, **63**, 613.

144 X. Jian, X. Pang, Y. Yu, X. Zhou and Z. Deng, *Antonie Van Leeuwenhoek*, 2006, **90**, 29.

145 B. K. Bhuyan, *Appl. Microbiol.*, 1962, **10**, 302.

146 S. Sakuda, A. Isogai, S. Matsumoto and A. Suzuki, *J. Antibiot.*, 1987, **40**, 296.

147 O. Ando, H. Satake, K. Itoi, A. Sato, M. Nakajima, S. Takahashi, H. Haruyama, Y. Ohkuma, T. Kinoshita and R. Enokita, *J. Antibiot.*, 1991, **44**, 1165.

148 O. Ando, M. Nakajima, K. Hamano, K. Itoi, S. Takahashi, Y. Takamatsu, A. Sato, R. Enokita, T. Okazaki, H. Haruyama and T. Kinoshita, *J. Antibiot.*, 1993, **46**, 1116.

149 M. Hayashi, S. Yaginuma, H. Yoshioka and K. Nakatsu, *J. Antibiot.*, 1981, **34**, 675.

150 T. Kusaka, H. Yamamoto, M. Shibata, M. Muroi and T. Kishi, *J. Antibiot.*, 1968, **21**, 255.

151 S. Omura, H. Tanaka, H. Kuga and D. Imamura, *J. Antibiot.*, 1986, **39**, 309.

152 S. Nishimura, *Prog. Nucleic Acid Res. Mol. Biol.*, 1983, **28**, 49.

153 S. Yaginuma, N. Muto, M. Tsujino, Y. Sudate, M. Hayashi and M. Otani, *J. Antibiot.*, 1981, **34**, 359.

154 R. K. Slany, M. Bosl, P. F. Crain and H. Kersten, *Biochemistry*, 1993, **32**, 7811.

155 G. Flesch and M. Rohmer, *Biochem. J.*, 1989, **262**, 673.

156 G. Flesch and M. Rohmer, *Eur. J. Biochem.*, 1988, **175**, 405.

157 K. L. Rinehart, Jr., D. D. Weller and C. J. Pearce, *J. Nat. Prod.*, 1979, **43**, 1.

158 J. S. Reader, D. Metzgar, P. Schimmel and V. de Crecy-Lagard, *J. Biol. Chem.*, 2004, **279**, 6280.

159 G. N. Jenkins and N. J. Turner, *Chem. Soc. Rev.*, 1995, 169.

160 R. K. Slany, M. Bosl and H. Kersten, *Biochimie*, 1994, **76**, 389.

161 S. G. Van Lanen and D. Iwata-Reuyl, *Biochemistry*, 2003, **42**, 5312.

162 S. D. Kinzie, B. Thern and D. Iwata-Reuyl, *Org. Lett.*, 2000, **2**, 1307.

163 S. G. Van Lanen, S. D. Kinzie, S. Matthieu, T. Link, J. Culp and D. Iwata-Reuyl, *J. Biol. Chem.*, 2003, **278**, 10491.

164 J. T. Kealey, L. Liu, D. V. Santi, M. C. Betlach and P. J. Barr, *Proc. Natl. Acad. Sci. U. S. A.*, 1998, **95**, 505.

165 P. Lu, A. Zhang, L. M. Dennis, A. M. Dahl-Roshak, Y. Q. Xia, B. Arison, Z. An and J. S. Tkacz, *Mol. Genet. Genomics*, 2005, **273**, 207.

166 N. Yalpani, D. J. Altier, E. Barbour, A. L. Cigan and C. J. Scelonge, *Plant Cell*, 2001, **13**, 1401.

167 K. L. Rinehart, Jr., M. Potgieter and D. L. Delaware, *J. Am. Chem. Soc.*, 1981, **103**, 2099.

168 H. G. Floss, *Nat. Prod. Rep.*, 1997, **14**, 433.

169 K. Arakawa, R. Muller, T. Mahmud, T. W. Yu and H. G. Floss, *J. Am. Chem. Soc.*, 2002, **124**, 10644.

170 H. G. Floss, *J. Nat. Prod.*, 2006, **69**, 158.

171 T. W. Yu, R. Muller, M. Muller, X. Zhang, G. Draeger, C. G. Kim, E. Leistner and H. G. Floss, *J. Biol. Chem.*, 2001, **276**, 12546.

172 D. D. Weller and K. L. Rinehart, Jr., *J. Am. Chem. Soc.*, 1978, **100**, 6757.

173 Z. Y. Zhou, S. Sakuda and Y. Yamada, *J. Chem. Soc., Perkin Trans. 1*, 1992, 1649.

174 S. Sakuda, Y. Sugiyama, Z. Y. Zhou, H. Takao, H. Ikeda, K. Kakinuma, Y. Yamada and H. Nagasawa, *J. Org. Chem.*, 2001, **66**, 3356.

175 Z. Y. Zhou, S. Sakuda, M. Kinoshita and Y. Yamada, *J. Antibiot.*, 1993, **46**, 1582.

176 Y. Sugiyama, H. Nagasawa, A. Suzuki and S. Sakuda, *J. Antibiot.*, 2002, **55**, 263.

177 A. Gambacorta, G. Caracciolo, D. Trabasso, I. Izzo, A. Spinella and G. Sodano, *Tetrahedron Lett.*, 2002, **43**, 451.

178 R. J. Parry, V. Bornemann and R. Subramanian, *J. Am. Chem. Soc.*, 1989, **111**, 5819.

179 R. J. Parry, K. Haridas, R. De Jong and C. R. Johnson, *Tetrahedron Lett.*, 1990, **31**, 7549.

180 R. J. Parry, K. Haridas, R. De Jong and C. R. Johnson, *J. Chem. Soc., Chem. Commun.*, 1991, 740.

181 J. M. Hill, G. N. Jenkins, C. P. Rush, N. J. Turner, A. J. Willets, A. D. Buss, M. J. Dawson and B. A. M. Rudd, *J. Am. Chem. Soc.*, 1995, **117**, 5391.

182 J. Hanrahan, H. Navratilova, K. A. Shoberu, D. H. G. Crout, D. W. Hutchinson, S. Roberts and E. M. H. Wellington, *J. Chem. Soc., Perkin Trans. 1*, 1994, 3533.

183 R. J. Parry and Y. Jiang, *Tetrahedron Lett.*, 1994, **35**, 9665.

184 B. Frey, J. McCloskey, W. Kersten and H. Kersten, *J. Bacteriol.*, 1988, **170**, 2078.

185 D. Iwata-Reuyl, *Bioorg. Chem.*, 2003, **31**, 24.

186 J. M. Renoux and M. Rohmer, *Eur. J. Biochem.*, 1985, **151**, 405.

187 M. Rohmer, B. Sutter and H. Sahm, *J. Chem. Soc., Chem. Commun.*, 1989, 1471.

188 S. P. Vincent, P. Sinay and M. Rohmer, *Chem. Commun.*, 2003, 782.

189 A. M. Hill, *Nat. Prod. Rep.*, 2006, **23**, 256.

190 K. J. Weissman and P. F. Leadlay, *Nat. Rev. Microbiol.*, 2005, **3**, 925.

191 J. Grunewald and M. A. Marahiel, *Microbiol. Mol. Biol. Rev.*, 2006, **70**, 121.

192 H. Yamase, L. Zhao and H. W. Liu, *J. Am. Chem. Soc.*, 2000, **122**, 12397.

193 X. M. He and H. W. Liu, *Annu. Rev. Biochem.*, 2002, **71**, 701.

194 C. G. Hyun, S. Y. Kim, J. H. Hur, M. J. Seo, J. W. Suh and S. O. Kim, *J. Microbiol.*, 2005, **43**, 313.

195 A. Nagaya, S. Takeyama and H. Tamegai, *Biosci., Biotechnol., Biochem.*, 2005, **69**, 1389.
