
THE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 282, NO. 23, pp. 16829–16837, June 8, 2007  
© 2007 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in the U.S.A.

# Defining Paxilline Biosynthesis in *Penicillium paxilli*

## FUNCTIONAL CHARACTERIZATION OF TWO CYTOCHROME P450 MONOOXYGENASES*§

Received for publication, February 23, 2007, and in revised form, April 3, 2007 Published, JBC Papers in Press, April 11, 2007, DOI 10.1074/jbc.M701626200

Sanjay Saikia †, Emily J. Parker §1, Albert Koulman ‡, and Barry Scott ‡2

From the †Institute of Molecular Biosciences and the §Institute of Fundamental Sciences, Massey University, Private Bag 11 222, Palmerston North, New Zealand and the ‡AgResearch Grasslands Research Centre, Tennent Drive, Private Bag 11 008, Palmerston North, New Zealand

---

Indole diterpenes are a large, structurally and functionally diverse group of secondary metabolites produced by filamentous fungi. Biosynthetic schemes have been proposed for these metabolites but until recently none of the proposed steps had been validated by biochemical or genetic studies. Using *Penicillium paxilli* as a model experimental system to study indole diterpene biosynthesis we previously showed by deletion analysis that a cluster of seven genes is required for paxilline biosynthesis. Two of these *pax* genes, *paxP* and *paxQ* (encoding cytochrome P450 monooxygenases), are required in the later steps in this pathway. Here, we describe the function of *paxP* and *paxQ* gene products by feeding proposed paxilline intermediates to strains lacking the *pax* cluster but containing ectopically integrated copies of *paxP* or *paxQ*. Transformants containing *paxP* converted paspaline into 13-desoxypaxilline as the major product and β-PC-M6 as the minor product. β-PC-M6, but not α-PC-M6, was also a substrate for PaxP and was converted to 13-desoxypaxilline. *paxQ*-containing transformants converted 13-desoxypaxilline into paxilline. These results confirm that paspaline, β-PC-M6, and 13-desoxypaxilline are paxilline intermediates and that paspaline and β-PC-M6 are substrates for PaxP, and 13-desoxypaxilline is a substrate for PaxQ. PaxP and PaxQ also utilized β-paxitriol and α-PC-M6 as substrates converting them to paxilline and α-paxitriol, respectively. These findings have allowed us to delineate clearly the biosynthetic pathway for paxilline for the first time.

mevalonate, respectively (5–8), very little is known about their biosynthetic pathways. Biosynthetic schemes have been proposed for these metabolites based on chemical identification of likely intermediates and precursor-feeding studies from the organism of interest and related filamentous fungi (9–11). However, until recently none of the proposed steps had been validated by biochemical or genetic studies.

The cloning and characterization of a cluster of genes from *Penicillium paxilli* necessary for biosynthesis of the indole diterpene, paxilline, has helped to understand the genetics and biochemistry of this important class of secondary metabolites (12). Analysis of the *pax* gene cluster has also facilitated the isolation of *pax* gene orthologues from *Aspergillus flavus* and *Neotyphodium lolii*, fungi that produce the related, but more complex indole diterpenes aflatrem and lolitrem, respectively (13, 14). Based on gene disruption and chemical complementation studies, a cluster of seven genes including *paxG* (encoding a geranylgeranyl diphosphate synthase), *paxM* (encoding a FAD-dependent monooxygenase), *paxC* (encoding a prenyltransferase), *paxP* and *paxQ* (encoding two cytochrome P450 monooxygenases), and *paxA* and *paxB* (encoding two putative membrane proteins)³, has been shown to be essential for paxilline biosynthesis (12, 15). Recently, we have shown that four of these genes, *paxG*, *paxM*, *paxB*, and *paxC*, are required for the biosynthesis of the proposed first stable indole diterpene intermediate, paspaline (16). Later steps in the paxilline biosynthesis pathway involve two cytochrome P450 monooxygenases, PaxP and PaxQ, which are proposed to utilize paspaline and 13-desoxypaxilline as their respective substrates (15). These compounds together with paspaline B, PC-M6, and β-paxitriol have been proposed to form a metabolic grid for paxilline biosynthesis (11). The formation of paxilline from PC-M6 was proposed to occur either via 13-desoxypaxilline or via β-paxitriol suggesting a bifurcation at the penultimate step to paxilline biosynthesis (11).

The objectives of this study were to identify the substrates and products for PaxP and PaxQ and to assess the involvement of proposed paxilline intermediates. Given the versatility of cytochrome P450 enzymes to catalyze multiple steps (17, 18), we sought to define the roles that PaxP and PaxQ play in the conversion of paspaline to paxilline.

---

Indole diterpenes are a large, structurally and functionally diverse group of secondary metabolites produced mainly by the *Penicillium*, *Aspergillus*, *Claviceps*, and *Neotyphodium* species of filamentous fungi (1, 2). Most of these metabolites are potent tremorgenic mammalian toxins (3) and some have anti-insect properties (4). Although much work has been done to demonstrate the origins of indole and diterpene components of these metabolites from tryptophan or a tryptophan precursor and

---

* This work was supported by a grant from the Royal Society of New Zealand (Marsden Grant MAU010). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
§ The on-line version of this article (available at http://www.jbc.org) contains supplemental Figs. 1 and 2.
¹ Present address: Dept. of Chemistry, Canterbury University, Private Bag 4800, Christchurch, New Zealand.
² To whom correspondence should be addressed: Tel.: +64 6 3505168; Fax: +64 6 3505688; E-mail: d.b.scott@massey.ac.nz.
³ B. Monahan, S. Saikia, and B. Scott, unpublished results.

---

## EXPERIMENTAL PROCEDURES

### Bacterial Strains and Plasmids—The bacterial strains and plasmids used are listed in Table 1. *Escherichia coli* strain XL

P. paxilli Cytochrome P450 Monooxygenases

TABLE 1  
Strains and plasmids  

| Strains and plasmids | Plasmid/Identifier | Relevant characteristics$^a$ | Source/Ref. |
|----------------------|--------------------|-------------------------------|-------------|
| Plasmids             |                    |                              |             |
| pGEM®-T Easy         |                    | Amp<sup>R</sup>               | Promega     |
|                      | pII99              | Amp<sup>R</sup>/Gen<sup>R</sup> (PtrpC-nptII-TtrpC) | 20        |
|                      | pSS1               | pII99 containing *paxP* on a 2.68-kb EcoRI fragment; Amp<sup>R</sup>/Gen<sup>R</sup> | This study |
|                      | pSS2               | pII99 containing *paxQ* on a 3.02-kb HindIII fragment; Amp<sup>R</sup>/Gen<sup>R</sup> | This study |
| Fungal strains (*P. paxilli*) |            |                              |             |
| PN2013               |                    | Wild-type; paxilline-positive | 21          |
| PN2253               | LM662              | PN2013/Δ*paxT-paxD*; Hyg<sup>R</sup>; paxilline-negative | 12        |
| PN2250               | CY2                | PN2013/Δ(*pax* cluster and flanking sequences); Hyg<sup>R</sup>; paxilline-negative | 22        |
| PN2258               | LMP1               | PN2013/Δ*paxP*; Hyg<sup>R</sup>; paxilline-negative | 15        |
| PN2523               | LMP1/pSS1-T4       | PN2258/pSS1; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-positive | This study |
| PN2259               | LMQ226             | PN2013/Δ*paxQ*; Hyg<sup>R</sup>; paxilline-negative | 15        |
| PN2527               | LMQ226/pSS2-T4     | PN2259/pSS2; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-positive | This study |
| PN2536               | LM662/pII99-T2     | PN2253/pII99; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2537               | LM662/pSS1-T2      | PN2253/pSS1; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2538               | LM662/pSS1-T10     | PN2253/pSS1; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2539               | CY2/pII99-T5       | PN2250/pII99; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2540               | CY2/pSS1-T9        | PN2250/pSS1; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2541               | CY2/pSS1-T10       | PN2250/pSS1; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2542               | LM662/pII99-T3     | PN2253/pII99; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2543               | LM662/pSS2-T9      | PN2253/pSS2; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2544               | LM662/pSS2-T14     | PN2253/pSS2; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2545               | CY2/pII99-T6       | PN2250/pII99; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2546               | CY2/pSS2-T7        | PN2250/pSS2; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| PN2547               | CY2/pSS2-T10       | PN2250/pSS2; Hyg<sup>R</sup> Gen<sup>R</sup>; paxilline-negative | This study |
| Bacterial strains (*E. coli*) |           |                              |             |
| XL1-Blue             |                    | supE44 hsdR17 recA1 endA1 gyrA46 thi relA1 lac<sup>-</sup> | 19        |
|                      |                    | F' [proAB<sup>+</sup> lacI<sup>q</sup> lacZ ΔM15 Tn10 (Tet<sup>R</sup>)] |             |
| PN1994               |                    | XL1-Blue/pSS1                  | This study  |
| PN1995               |                    | XL1-Blue/pSS2                  | This study  |

$^a$ Amp<sup>R</sup>, ampicillin-resistant; Gen<sup>R</sup>, Geneticin-resistant; Hyg<sup>R</sup>, hygromycin-resistant.

1-Blue (19) served as the host for routine cloning. The transformants of this host were grown on LB agar plates supplemented with ampicillin (100 μg/ml) for selection. pGEM®-T Easy (Promega) and pII99 (20) plasmids were used for cloning.

Fungal Strains and Growth Conditions—The fungal strains used are listed in Table 1. Cultures of wild-type *P. paxilli* Bainier (PN2013 = ATCC 26601) (21), *paxP* and *paxQ* deletion mutant strains LMP1 and LMQ226, respectively (15), *pax* cluster deletion mutant strains CY2 (22) and LM662 (12), and their derivatives were routinely grown in *Aspergillus* complete medium at 22°C for 4–6 days as described previously (16). Fungal cultures used for the isolation of genomic DNA, preparation of protoplasts, and indole diterpene analysis were also grown as described previously (16). The growth conditions for fungal cultures used for the isolation of RNA were similar to those for indole diterpene analysis.

Molecular Biology—Plasmid DNA was isolated and purified by alkaline lysis using a Bio-Rad Quantum Prep® Plasmid Miniprep Kit (Bio-Rad). Genomic DNA was isolated using a modification of the method of Yoder (23) as described previously (22). Total RNA was isolated from frozen mycelium using TRIzol® reagent (Invitrogen). Before RT-PCR, isolated RNA was treated with DNase in 50 μl reaction volume containing 5 μg of RNA, 1× DNase I reaction buffer (Invitrogen) and 5 units of DNase I, amplification grade (Invitrogen), following the manufacturer's instructions. DNA fragments and PCR products were purified using QIAquick gel extraction and a PCR purification kit (Qiagen). DNA fragments were sequenced by the dideoxynucleotide chain-termination method (24) using Big-Dye (Version 3) chemistry (PerkinElmer Life Sciences) with oligonucleotide primers (Sigma Genosys). Products were separated on an ABI Prism 377 sequencer (Perkin-Elmer Life Sciences).

PCR and RT-PCR Conditions—PCR of *paxP* and *paxQ* for cloning was carried out in 50-μl reaction volume containing 1× reaction buffer (Roche Applied Science), a 50 μM concentration of each dNTP, a 300 nM concentration of each primer, 0.75 unit of Expand High Fidelity Enzyme Mix (Roche Applied Science), and 15 ng of genomic DNA. PCR for screening for *paxP* and *paxQ* was carried out in 50-μl reaction volume containing 1× reaction buffer (Roche Applied Science), a 100 μM concentration of each dNTP, a 200 nM concentration of each primer, 2 units of TaqDNA polymerase (Roche Applied Science), and 10 ng of genomic DNA. RT-PCR for *paxP* and *paxQ* expression was performed in 25 μl reaction volume containing 1× reaction mix (a 200 μM concentration of each dNTP) (Invitrogen), a 200 nM concentration of each primer, 0.5 μl of RT/Platinum® Taq Mix (Invitrogen), and 100 ng of DNase-treated RNA.

The thermocycle conditions routinely used with Expand High Fidelity Enzyme Mix and TaqDNA polymerase were one cycle at 94°C for 2 min, 30 cycles at 94°C for 30 s, 55°C for 30 s, and 72°C for 1 min (per kb) and one cycle at 72°C for 5 min, and with RT/Platinum® Taq Mix one cycle at 50°C for 30 min and 94°C for 2 min, 35 cycles at 94°C for 30 s, 55°C for 30 s, and 72°C for 1 min (per kb) and one cycle at 72°C for 10 min. Reactions were carried out in a PC-960, PC-960G (Corbett Research, Mortlake, Australia), or Mastercycler® gradient (Eppendorf, Hamburg, Germany) thermocycler.

$^4$ The abbreviations used are: RT, reverse transcription; HPLC, high performance liquid chromatography; MOPS, 3-(N-morpholino)propanesulfonic acid.

The primers used for amplifying *paxP* and *paxQ* for cloning were as follows: *paxP* (paxPEcoRIF2, 5′-TTGAATTCATGCA-TGGTAAGCAGCCG-3′, and paxPEcoRIR, 5′-GGGAATTC-CGGCCATTCAATCTCAAG-3′), and *paxQ* (paxQHindIIIF2, 5′-GAAAAGCTTTACTCTGACACACTCCGC-3′ and pax-QHindIIIR, 5′-TTGAAGCTTCCCTATTGTGGCGCAGTC-3′). The primers contained mismatches (shown in bold) relative to the genomic sequences to introduce either EcoRI or HindIII sites, as appropriate, at both ends. These primers were also used for screening the transformants for integration of *paxP* and *paxQ*. The primers used for RT-PCR analysis of *paxP* and *paxQ* were as follows: *paxP* (SS1, 5′-TTCACGGACCACGTTGG-TGG-3′, and SS2, 5′-TGCTCATGGCACTGGTGTGC-3′) and *paxQ* (SS3, 5′-GGAAGTCTGCATCTGCAACG-3′, and SS4, 5′-AGAACCTGGGTCGTGGTATG-3′). The primers spanned one of the introns so that targets only from RNA or cDNA would be primed.

**Construction of Plasmids Encoding PaxP and PaxQ Proteins—** To construct an expression plasmid encoding PaxP, wild-type genomic DNA was used as a template in a PCR using the forward primer paxPEcoRIF2 and the reverse primer pax-PEcoRIR. The 2.68-kb amplified fragment was digested with EcoRI and ligated into EcoRI-digested pII99 vector yielding the plasmid pSS1. To obtain an expression plasmid encoding PaxQ, wild-type genomic DNA was used as a template in a PCR using the primers paxQHindIIIF2 and paxQHindIIIR. The 3.02-kb amplified fragment was digested with HindIII and subcloned into HindIII-digested pII99 to generate the plasmid pSS2.

**P. paxilli Transformation—** Protoplasts of *P. paxilli paxP* and *paxQ* deletion mutant strains LMP1 and LMQ226, respectively, and *pax* cluster deletion mutant strains CY2 and LM662 were prepared and transformed with 5 μg of circular plasmids pII99, pSS1, or pSS2 as described previously (16). Transformants were selected on regeneration medium supplemented with either hygromycin (100 μg/ml) or Geneticin (150 μg/ml) (Roche Applied Science).

**Synthesis of PC-M6 and Paxitriol—** Indole diterpenes PC-M6 and paxitriol were synthesized by sodium borohydride reduction of 13-desoxypaxilline and paxilline, respectively, using the method described by Miles *et al.* (25). Approximately 9 mg (0.021 mmol) of each of authentic 13-desoxypaxilline and paxilline (AgResearch, Ruakura, New Zealand) was reduced separately for 10 min with 12 (0.317 mmol) and 9 mg (0.238 mmol) of sodium borohydride in 9 ml of methanol, respectively. The products of the reduction reaction were checked by normal-phase TLC (chloroform:acetone 9.5:0.5) to ensure no starting material remain. Ten ml of de-ionized water was then added to the reaction mixture followed by the addition of 2 ml of 10% (v/v) hydrochloric acid. The products were extracted four times with 10 ml of dichloromethane and the pooled extract was dried by the addition of anhydrous magnesium sulfate and then filtered. The filtrate was dried *in vacuo* and the residue dissolved in methanol. A sample of this was run on reverse-phase HPLC, as described previously (16), which detected two peaks with a 1:2 ratio in both the reduction samples. The peaks in the paxilline reduction sample had retention times of 3.55 and 4.30 min and those in the 13-desoxypaxilline reduction sample had

retention times of 5.85 and 7.75 min. The fractions corresponding to the two peaks were separated, pooled, and purified by reverse-phase HPLC. The purified products of paxilline reduction were identified as α-paxitriol and β-paxitriol based on comparison to authentic standards (AgResearch, Ruakura, New Zealand). The purified products of 13-desoxypaxilline reduction were identified as α-PC-M6 and β-PC-M6 based on the NMR analysis of β-PC-M6 (Varian INOVA 500 equipped with an inverse-detection capillary probe) and the conversion of α-PC-M6 to α-paxitriol. PC-M6s were analyzed by LC-MSMS (see Supplemental Data) as described previously (16).

**Feeding of Precursor Metabolites to Fungal Cultures—** Mycelium from liquid cultures of *P. paxilli* was grown for 4 days as described previously (16). On day 4, mycelium was washed three times with 10 ml of 1 mM MOPS buffer (pH 6.5) (Sigma-Aldrich) and 0.5 g of washed mycelium transferred to two flasks each containing 12.5 ml of 20 mM MOPS buffer (pH 7.0) containing 4% (v/v) glycerol. To one flask, precursor metabolite was added in two equal doses of 100 μg (in 100 μl of acetone) at 24-h intervals during incubations at 28 °C with shaking at 200 rpm. To the other flask, 100 μl of acetone was added as an external control. On day 6, mycelium was harvested for indole diterpene analysis as described previously (16).

**Indole Diterpene Analysis—** Fungal mycelia was analyzed for indole diterpenes by normal-phase TLC and reverse-phase HPLC as described previously (16). In reverse-phase HPLC analysis, the characteristic feature of an indole moiety showing an absorption maximum at 230 nm and an absorption minimum at 280 nm was employed to confirm the presence of an indole diterpene in a sample. However, all reverse-phase HPLC traces reported here are shown for 230 nm wavelength.

**RESULTS**

**Complementation of P. paxilli paxP and paxQ Deletion Mutants with Respective Wild-type Genes—** Previously, it was shown that the deletion mutants of *paxP* (LMP1) and *paxQ* (LMQ226) were blocked for paxilline biosynthesis and accumulated paspaline and 13-desoxypaxilline, respectively (15). To confirm that disruption of *paxP* and *paxQ* genes was responsible for the block in paxilline biosynthesis complementation tests were carried out with LMP1 and LMQ226. The complementation constructs, pSS1 (*paxP*) and pSS2 (*paxQ*), were prepared by cloning PCR-amplified copies of *paxP* and *paxQ* genes into the geneticin-resistant vector pII99, respectively (Fig. 1). Protoplasts of deletion mutants LMP1 and LMQ226 were transformed with pSS1 and pSS2, respectively. Ten arbitrarily selected Geneticin-resistant transformants, from each transformation, were analyzed by normal-phase TLC (data not shown) and reverse-phase HPLC (Fig. 2) for their ability to synthesize paxilline. These analyses showed that the introduction of pSS1 and pSS2 into LMP1 and LMQ226, respectively, was able to restore paxilline biosynthesis in the recipient strains (Fig. 2). However, the deletion mutants containing the base vector, pII99, were not complemented for paxilline biosynthesis (data not shown). These results confirmed that the replacement of

P. paxilli Cytochrome P450 Monooxygenases

A

B

C

FIGURE 1. *paxP* and *paxQ* genes of *P. paxilli*. A, physical map of *paxP* and *paxQ* genes in the paxilline biosynthesis gene cluster showing the primers used for constructing the complementation vectors, screening gene integrations, and expression analysis. The restriction sites marked by an asterisk are introduced by PCR for cloning. B, complementation vectors pSS1 and pSS2 constructed by cloning PCR-amplified copies of *paxP* and *paxQ* genes into the geneticin-resistant base vector plI99, respectively. C, deletion mutants showing the regions deleted with respect to the *pax* cluster. The double-headed arrow indicates that the deletion in CY2 includes the entire *pax* cluster as well as the flanking sequences extending beyond the region shown in A.

Transformants for Functional Analysis of *paxP* and *paxQ*—To confirm the function of *paxP* and *paxQ*, complementation constructs pSS1 (*paxP*) and pSS2 (*paxQ*) were transformed into the protoplasts of the deletion mutant LM662 (12) that lacks the entire *pax* gene cluster and also into CY2 (22) that also lacks the entire *pax* gene cluster plus considerable flanking sequences (Fig. 1C). An arbitrary selection of Geneticin-resistant transformants was screened by PCR (data not shown) for ectopic integration of *paxP* or *paxQ*. Transformants containing a PCR product of the correct size were further assessed for *paxP* or *paxQ* expression by RT-PCR analysis using primers that spanned one of the introns so that targets only from RNA or cDNA would be primed. A transcript for *paxP* or *paxQ* was detected in all the PCR-positive transformants analyzed (shown in Fig. 3 for the transformants that were further used for functional analysis). As expected, no transcript for *paxP* or *paxQ* was detected in LM662, CY2, LM662/plI99, and CY2/plI99 strains. This analysis confirmed that both *paxP* and *paxQ* were

FIGURE 2. Genetic complementation of paxilline biosynthesis in *paxP* (LMP1) and *paxQ* (LMQ226) deletion mutants. Mycelial extracts of *P. paxilli* *paxP* and *paxQ* deletion mutants and their derivatives containing ectopically integrated copies of *paxP* and *paxQ* genes, respectively, were analyzed by reverse-phase HPLC for indole diterpenes. A, *paxP* deletion mutant LMP1, deletion mutant LMP1 containing *paxP* (LMP1/pSS1-T4), paxilline and paspaline standards. B, *paxQ* deletion mutant LMQ226, deletion mutant LMQ226 containing *paxQ* (LMQ226/pSS2-T4), paxilline and 13-desoxypaxilline standards.

*paxP* and *paxQ* genes caused the block for paxilline biosynthesis in LMP1 and LMQ226 mutants, respectively, and that the *paxP* and *paxQ* complementation constructs are functional. These complementation constructs were further used for functional analysis of *paxP* and *paxQ*.
expressed but at higher levels in the LM662 background than in  
the CY2 background. Transformants that showed high levels of  
$paxP$ or $paxQ$ expression were selected for further $paxP$ and  
$paxQ$ functional analysis experiments.  

**Functional Analysis of $paxP$—** To test that paspaline is a sub-  
strate for PaxP, LM662 and CY2 strains containing $paxP$ were  

A  
![Image Description](#)  

B  
![Image Description](#)  

FIGURE 3. Expression analysis of $paxP$ and $paxQ$ genes. RT-PCR analysis of  
total RNA isolated from 6 day old mycelia of LM662 and CY2 deletion mutants  
containing either $paxP$ (LM662/pSS1-T2, LM662/pSS1-T10, CY2/pSS1-T9,  
CY2/pSS1-T10) (A) or $paxQ$ (LM662/pSS2-T9, LM662/pSS2-T14, CY2/pSS2-T7,  
CY2/pSS2-T10) (B). These transformants were confirmed by PCR to contain  
the respective genes. Expression was also examined in the wild-type and in  
LM662 and CY2 deletion mutants and their derivatives containing the base  
vector pII99 (LM662/pII99-T2, CY2/pII99-T5 in A and LM662/pII99-T3 CY2/  
pII99-T6 in B). Numbers on the right correspond to the fragment sizes indi-  
cated in base pairs.  

incubated with paspaline for 2 days and chloroform:methanol  
(2:1) extracts of these transformants analyzed for indole diter-  
penes by normal-phase TLC (data not shown) and reverse-  
phase HPLC (Fig. 4). Extracts of LM662/pSS1 and CY2/pSS1  
transformants had two new indole diterpenes, 13-desoxypaxil-  
line and $\beta$-PC-M6, as major and minor products, respectively,  
that were absent in LM662 and CY2 (Fig. 4) and transformants  
containing the base vector pII99 (data not shown). Paspaline  
was almost completely converted to 13-desoxypaxilline by the  
LM662/pSS1 transformants, whereas the CY2/pSS1 transfor-  
mants showed only partial conversion of the added paspaline to  
indole diterpene products. Analysis of additional $paxP$-con-  
taining transformants showed that the degree of paspaline con-  
version correlated with the level of $paxP$ expression. These  
results confirmed that paspaline is a substrate for PaxP and is  
converted to 13-desoxypaxilline as the major product and  
$\beta$-PC-M6 as the minor product and that PaxP is capable of  
catalyzing more steps than originally proposed (15).  

**Functional Analysis of $paxQ$—** To test whether 13-desoxy-  
paxilline is a substrate for PaxQ, LM662, and CY2 strains that  
contained $paxQ$ were incubated with 13-desoxypaxilline,  
together with appropriate controls. Normal-phase TLC (data  
not shown) and reverse-phase HPLC analysis of extracts of  
LM662 and CY2 strains containing $paxQ$ identified paxilline as  
the indole diterpene product (Fig. 5). As expected, no indole  
diterpenes other than the added 13-desoxypaxilline, were  
detected in extracts of LM662 and CY2 deletion mutants (Fig.  
5) and LM662 and CY2 strains containing the base vector pII99  
(data not shown). Similar to the paspaline feeding experiments  
(see above), the LM662/pSS2 transformants were more effi-  
cient at converting the added 13-desoxypaxilline to paxilline  
than the CY2/pSS2 transformants, with the degree of conver-  
sion correlating with the level of $paxQ$ expression. These anal-  
yses confirmed that 13-desoxypaxilline is a substrate for PaxQ.  

PC-M6 Feeding Studies—Although PC-M6 has been pro-  
posed as a paxilline intermediate (11), there is no biochemical  
evidence to support this hypothesis. To test whether PC-M6 is  
an intermediate for paxilline biosynthesis and a substrate for  
PaxP, the $\alpha$- and $\beta$-diastereomers of PC-M6 were chemically  
synthesized, purified, and incubated (see “Experimental Procedures”) with  
the same transformants described  
above. Extracts of $\beta$-PC-M6-fed  
strains containing a functional PaxP,  
LM662/pSS1-T10 and LMQ226,  
were able to convert $\beta$-PC-M6 to  
13-desoxypaxilline (Fig. 6A), whereas  
the strains containing PaxQ or the  
controls were unable to metabolize  
the added substrate (Fig. 6A). The  
$\beta$-PC-M6-fed LMQ226 mutant  
showed an increased accumulation of  
13-desoxypaxilline which is most  
likely the result of catalysis by PaxP  
(Fig. 6A). Although the $paxP$ deletion  
mutant LMP1 was unable to utilize  
$\beta$-PC-M6 as a substrate, it showed an  
increased accumulation of paspaline  

![Image Description](#)  

FIGURE 4. Reverse-phase HPLC analysis of fungal cultures incubated with paspaline. Paspaline was incu-  
bated with liquid cultures of deletion mutants LM662 and CY2 and their derivatives containing $paxP$ (LM662/  
pSS1-T10, CY2/pSS1-T9). Extracts were analyzed for indole diterpenes by reverse-phase HPLC. The peaks indi-  
cated by arrows corresponded to $\beta$-PC-M6, as confirmed later (shown in Fig. 6A). The topmost trace corresponds  
to the standards. The standard fed is indicated in gray.  

JUNE 8, 2007 • VOLUME 282 • NUMBER 23  
JOURNAL OF BIOLOGICAL CHEMISTRY 16833  
P. paxilli Cytochrome P450 Monooxygenases

Similar analysis of extracts of α-PC-M6-fed strains containing a functional PaxQ, LM662/pSS2-T9 and LMP1, showed accumulation of a new indole diterpene, α-paxitriol (Fig. 6B). However, the strains containing PaxP, LMQ226, and LM662/pSS1-T10, and the control LM662 were unable to utilize the added α-PC-M6 as a substrate (Fig. 6B). While a peak of similar retention time to α-paxitriol is visible in the reverse phase-HPLC traces of these strains (Fig. 6B), this peak does not have the characteristic absorption feature of an indole moiety at 230 and 280 nm (see “Experimental Procedures”). Extracts of α-PC-M6-fed *paxP* deletion mutant LMP1 showed an increased accumulation of paspaline (Fig. 6B) similar to that observed in extracts of β-PC-M6-fed LMP1 mutant (see above). Although these results confirmed that α-PC-M6 is a substrate for PaxQ, it is unlikely to be an intermediate for paxilline biosynthesis since paxilline precursors have a C-10 β-stereochemistry.

**Paxitriol Feeding Studies—To test whether α- and β-paxitriol are intermediates in the paxilline biosynthesis pathway and substrates for either PaxP or PaxQ, the two diastereomers were chemically synthesized and fed to the strains described above. Unlike the controls, extracts of β-paxitriol-fed strains containing a functional PaxP, LM662/pSS1-T10 and LMQ226, showed a new indole diterpene, paxilline (Fig. 7A). These results confirmed that β-paxitriol is a substrate for PaxP. The conversion of β-paxitriol to paxilline by the *paxQ* deletion mutant LMQ226 suggests that the metabolism of β-paxitriol occurs after the steps catalyzed by PaxQ. Taken together, these results suggest that β-paxitriol is not a true intermediate in paxilline biosynthesis. These results further confirm that the oxidation at C-10 is a catalytic function of PaxP, which catalyzes the conversion of β-PC-M6 to 13-desoxypaxilline and β-paxitriol to paxilline, both requiring C-10 oxidation. Similar incubation studies with α-paxitriol showed no conversion of**

in response to β-PC-M6 feeding. Taken together, these results confirmed that β-PC-M6 is a substrate for PaxP and an intermediate for paxilline biosynthesis.

FIGURE 5. Reverse-phase HPLC analysis of fungal cultures incubated with 13-desoxypaxilline. 13-Desoxypaxilline was incubated with liquid cultures of deletion mutants LM662 and CY2 and their derivatives containing *paxQ* (LM662/pSS2-T9, CY2/pSS2-T7). Extracts were analyzed for indole diterpenes by reverse-phase HPLC. The topmost trace corresponds to the standards. The standard fed is indicated in gray.

FIGURE 6. Reverse-phase HPLC analysis of fungal cultures incubated with PC-M6. Liquid cultures of deletion mutants LMP1, LMQ226 and LM662, and LM662 transformants containing *paxP* (LM662/pSS1-T10) and *paxQ* (LM662/pSS2-T9) were incubated with either β-PC-M6 (A) or α-PC-M6 (B). Extracts were analyzed for indole diterpenes by reverse-phase HPLC. The topmost trace in each panel corresponds to the standards. The standards fed are indicated in gray.

P. paxilli Cytochrome P450 Monooxygenases

deletion is not yet defined (22), the presence of a gene encoding a regulatory factor within the deleted region cannot be ruled out. Unlike other filamentous fungi, no regulatory genes have been found to be associated with paxilline biosynthesis (28–32). Although it was shown earlier that two putative regulatory genes, *paxR* and *paxS*, that encode putative transcription factors are associated with the *pax* cluster (12), mutants deleted for these genes are still able to synthesize paxilline suggesting that these two genes do not have a regulatory role in paxilline biosynthesis.⁵

In precursor-feeding experiments, although paxilline was the only conversion product of 13-desoxypaxilline by PaxQ, PaxP converted paspaline to both 13-desoxypaxilline and β-PC-M6. These results demonstrate that PaxP is able to catalyze multiple oxidation steps both at a single carbon atom and at different carbon atoms involving removal of the C-30 methyl group followed by C-10 oxidation, via β-PC-M6. Similar sequential loss of a C-14 methyl group by a P450 enzyme has been reported in sterol biosynthesis (33, 34). In *Fusarium fujikuroi*, the multifunctional cytochrome P450 monooxygenase, GA₁₄-oxidase, was shown to catalyze four sequential oxidations at four different carbon atoms in a series of oxidations from ent-kaurenoic acid to GA₁₄ (35). Oxidation at multiple sites by a single P450 enzyme has also been shown for the fungus *Curvularia lunata* in which the cytochrome P450 enzyme, P450lun, catalyzes bifunctionally 11β- and 14α-hydroxylations of 11-deoxycortisol to form cortisol (36). In *F. fujikuroi*, another cytochrome P450 enzyme, ent-kaurene oxidase, catalyzes three sequential oxidations at a single carbon atom in the conversion of ent-kaurene to ent-kaurenoic acid (35, 37). Similar multiple catalytic activities have been suggested for the cytochrome P450 enzymes encoded by genes within the gibberellin cluster identified in *Phaeosphaeria* sp. (38). Another indole diterpene paspaline B has previously been isolated from *P. paxilli* and identified as the first oxidized analogue of paspaline (11). However, in our precursor-feeding experiments, no indole diterpenes other than β-PC-M6 and 13-desoxypaxilline were detected as the conversion products of paspaline by PaxP. In addition, incubation of β-PC-M6 with a *pax* cluster negative strain contained

FIGURE 7. Reverse-phase HPLC analysis of fungal cultures incubated with paxitriol. Liquid cultures of deletion mutants LMQ226 and LM662 and LM662 transformant containing *paxP* (LM662/pSS1-T10) were incubated with either β-paxitriol (*A*) or α-paxitriol (*B*). Extracts were analyzed for indole diterpenes by reverse-phase HPLC. The topmost trace in each panel corresponds to the standards. The standards fed are indicated in gray.

the added substrate by the tested strains (Fig. 7*B*) confirming that it was not an intermediate for paxilline biosynthesis.

DISCUSSION

In this study we have biochemically characterized the two cytochrome P450 monooxygenases, PaxP and PaxQ, and demonstrated that these enzymes alone are responsible for the conversion of paspaline to paxilline. Incubation of paspaline and β-PC-M6, and 13-desoxypaxilline with two *pax* cluster deletion mutants, CY2 and LM662, containing either PaxP or PaxQ resulted in the conversion of the fed compounds into indole diterpene products confirming that these compounds are substrates for PaxP and PaxQ, respectively, and intermediates for paxilline biosynthesis.

Both P450 enzymes were more efficient in their ability to convert their respective substrates in the LM662 background than in the CY2 background. This could be attributed to a comparatively higher level of expression of the corresponding genes in the LM662 background than in the CY2 background which in turn might be attributable to the site of integration (26, 27). Given that CY2 is the result of a single crossover associated with a subsequent large deletion (>100 kb) and that one end of this

⁵ L. McMillan, S. Munday-Finch, and B. Scott, unpublished results.

P. paxilli Cytochrome P450 Monooxygenases

PaxG  
↓  
GGPP  

PaxM  
↓  

PaxC  
↓  

$\begin{aligned} & \text { paspaline } \\ & \beta-\mathrm{PC}-\mathrm{M} 6 \end{aligned}$  

PaxP  
→  

$\begin{aligned} & \text { 13-desoxypaxilline } \\ & \text { paxilline } \end{aligned}$  

PaxQ  
→  

$\begin{aligned} & \alpha-\mathrm{PC}-\mathrm{M} 6 \\ & \alpha-\text { paxitriol } \end{aligned}$  

PaxQ  
→  

terpendoles  
lolitrem  

janthitrem  
penitrem  

FIGURE 8. Proposed pathway for paxilline biosynthesis showing the various steps catalyzed by the two cytochrome P450 monooxygenases, PaxP and PaxQ. The reactions in the major pathway leading to the biosynthesis of paxilline are denoted by thick black arrows, and the reactions unrelated to the major paxilline biosynthesis pathway are shown by black arrows. Gray arrows indicate proposed pathways for the formation of more complex indole diterpenes. Positions of carbon atoms acted upon by PaxP or PaxQ are numbered.

ing only PaxP resulted in its conversion to 13-desoxypaxilline confirming that β-PC-M6 is also a substrate for PaxP. Although PaxP was shown to catalyze the C-10 oxidation of β-paxitriol to produce paxilline, β-paxitriol was not identified as a pathway intermediate in the feeding experiments. These results clearly indicate that the oxidation of the C-10 hydroxyl group does not require a distinct dehydrogenase enzyme, as proposed previously (15), but is achieved by the ketone functionality of PaxP alone.

In a previous study on the biosynthesis of paxilline from paspaline it was proposed that a bifurcation existed at PC-M6 and there were two alternative pathways from PC-M6 to paxilline requiring different enzymatic transformations. In this scheme it was proposed that paxilline could be synthesized either from β-paxitriol or from 13-desoxypaxilline (11). This suggests C-13 hydroxylation of PC-M6 as the source of β-paxitriol. However, our precursor-feeding experiments have shown that this reaction, although similar to the PaxQ conversion of 13-desoxypaxilline to paxilline, is not catalyzed by PaxQ. Furthermore, incubation of β-paxitriol with a paxQ deletion mutant complemented paxilline biosynthesis. Taken together, these precursor-feeding studies strongly suggest that β-paxitriol is not a part of the paxilline biosynthesis pathway but could be involved in steps post-paxilline (Fig. 8). It is possible that an enzyme outside the pax cluster is capable of converting paxilline to β-paxitriol.

Incubation of α-paxitriol with strains containing pax cluster negative backgrounds and also with a strain containing only PaxP confirmed that α-paxitriol was not a substrate for any Pax enzymes. However, α-PC-M6 was shown to be a substrate for PaxQ being converted to α-paxitriol. Given that the paxilline intermediates have a β-configuration at C-10, acceptance of a substrate that has a α-configuration by a Pax enzyme is unusual. α-Paxitriol is more likely a precursor for lolitrem and terpendoles that have the α-configuration at C-10 (Fig. 8) (9, 25, 39). The formation of α-paxitriol by PaxQ could be a nonspecific intrinsic reaction or a promiscuous catalytic activity of PaxQ. The biosynthetic significance, if any, of this reaction is not known. In Penicillium janczewskii and Penicillium janthinellum, incorporation of labeled β-paxitriol and not α-paxitriol into penitrem A and E and janthitrem B and C, respectively, suggests that β-paxitriol is an immediate precursor for the complex indole diterpenes with β-stereochemistry (Fig. 8) (39). These observations highlight the substrate stereospecificity of enzymes involved in the biosynthesis of indole diterpenes. Moreover, indole diterpenes that have been reported from fungi belonging to the taxonomic order Eurotiales (Aspergillus and Penicillium) have β-stereochemistry and those belonging to the order Clavicipitales (Epichloë and Claviceps) have α-stereochemistry (1). Recent identification of PaxP and PaxQ homologues in the lolitrem cluster (40) could help in understanding the substrate stereospecificities of these P450 enzymes in indole diterpene biosynthesis.

Our study illustrates for the first time how paspaline is elaborated to paxilline and defines the pathway and the gene products responsible for paxilline biosynthesis. It also proposes a role for PaxP (and by analogy LtmP) in laying down the C-10 stereochemistry of the indole diterpene intermediates. It is

now possible to test this hypothesis by introducing either or both LtmP and LtmQ in *P. paxilli* and check if this introduction enables *P. paxilli* to form indole diterpenes with α-stereochemistry.

Acknowledgments—We thank S. Munday-Finch and C. Miles for providing authentic samples of paxilline, paspaline, α-paxitriol, and β-paxitriol.

---

**REFERENCES**

1. Parker, E. J., and Scott, B. (2004) in *Handbook of Industrial Mycology*, (An, Z., ed) pp. 405–426, Marcel Dekker, New York
2. Scott, B., Jameson, G. B., and Parker, E. J. (2004) in *Advances in Fungal Biotechnology for Industry, Agriculture, and Medicine* (Tkacz, J. S., and Lange, L., eds) pp. 163–198, Kluwer Academic/Plenum Publishers, New York
3. Steyn, P. S., and Vleggaar, R. (1985) *Fortschr. Chem. Org. Naturst.* **48**, 1–80
4. Gloer, J. B. (1995) *Acc. Chem. Res.* **28**, 343–350
5. Acklin, W., Weibel, F., and Arigoni, D. (1977) *Chimia* **31**, 63
6. Byrne, K. M., Smith, S. K., and Ondeyka, J. G. (2002) *J. Am. Chem. Soc.* **124**, 7055–7060
7. de Jesus, A. E., Gorst-Allman, C. P., Steyn, P. S., van Heerden, F. R., Vleggaar, R., Wessels, P. L., and Hull, W. E. (1983) *J. Chem. Soc. Perkin Trans.* **1**, 1863–1868
8. Laws, I., and Mantle, P. G. (1989) *J. Gen. Microbiol.* **135**, 2679–2692
9. Gatenby, W. A., Munday-Finch, S. C., Wilkins, A. L., and Miles, C. O. (1999) *J. Agric. Food Chem.* **47**, 1092–1097
10. Mantle, P. G., and Weedon, C. M. (1994) *Phytochemistry* **36**, 1209–1217
11. Munday-Finch, S. C., Wilkins, A. L., and Miles, C. O. (1996) *Phytochemistry* **41**, 327–332
12. Young, C., McMillan, L., Telfer, E., and Scott, B. (2001) *Mol. Microbiol.* **39**, 754–764
13. Young, C. A., Bryant, M. K., Christensen, M. J., Tapper, B. A., Bryan, G. T., and Scott, B. (2005) *Mol. Genet. Genomics* **274**, 13–29
14. Zhang, S., Monahan, B. J., Tkacz, J. S., and Scott, B. (2004) *Appl. Environ. Microbiol.* **70**, 6875–6883
15. McMillan, L. K., Carr, R. L., Young, C. A., Astin, J. W., Lowe, R. G., Parker, E. J., Jameson, G. B., Finch, S. C., Miles, C. O., McManus, O. B., Schmalhofer, W. A., Garcia, M. L., Kaczorowski, G. J., Goetz, M., Tkacz, J. S., and Scott, B. (2003) *Mol. Genet. Genomics* **270**, 9–23
16. Saikia, S., Parker, E. J., Koulman, A., and Scott, B. (2006) *FEBS Lett.* **580**, 1625–1630
17. Ro, D. K., Arimura, G., Lau, S. Y., Piers, E., and Bohlmann, J. (2005) *Proc. Natl. Acad. Sci. U. S. A.* **102**, 8060–8065
18. Tudzynski, B. (2005) *Appl. Microbiol. Biotechnol.* **66**, 597–611
19. Bullock, W. O., Fernandez, J. M., and Short, J. M. (1987) *BioTechniques* **5**, 376–379
20. Namiki, F., Matsunaga, M., Okuda, M., Inoue, I., Nishi, K., Fujita, Y., and Tsuge, T. (2001) *Mol. Plant Microbe Interact.* **14**, 580–584
21. Itoh, Y., Johnson, R., and Scott, B. (1994) *Curr. Genet.* **25**, 508–513
22. Young, C., Itoh, Y., Johnson, R., Garthwaite, I., Miles, C. O., Munday-Finch, S. C., and Scott, B. (1998) *Curr. Genet.* **33**, 368–377
23. Yoder, O. C. (1988) *Adv. Plant Pathol.* **6**, 93–112
24. Sanger, F., Nicklen, S., and Coulson, A. R. (1977) *Proc. Natl. Acad. Sci. U. S. A.* **74**, 5463–5467
25. Miles, C. O., Wilkins, A. L., Gallagher, R. T., Hawkes, A. D., Munday, S. C., and Towers, N. R. (1992) *J. Agric. Food Chem.* **40**, 234–238
26. Malonek, S., Rojas, M. C., Hedden, P., Gaskin, P., Hopkins, P., and Tudzynski, B. (2005) *Appl. Environ. Microbiol.* **71**, 1462–1472
27. Chiou, C. H., Miller, M., Wilson, D. L., Trail, F., and Linz, J. E. (2002) *Appl. Environ. Microbiol.* **68**, 306–315
28. Mihlan, M., Homann, V., Liu, T. W., and Tudzynski, B. (2003) *Mol. Microbiol.* **47**, 975–991
29. Tudzynski, B., Homann, V., Feng, B., and Marzluf, G. A. (1999) *Mol. Gen. Genet.* **261**, 106–114
30. Bok, J. W., and Keller, N. P. (2004) *Eukaryot. Cell* **3**, 527–535
31. Bok, J. W., Balajee, S. A., Marr, K. A., Andes, D., Nielsen, K. F., Frisvad, J. C., and Keller, N. P. (2005) *Eukaryot. Cell* **4**, 1574–1582
32. Bok, J. W., Hoffmeister, D., Maggio-Hall, L. A., Murillo, R., Glasner, J. D., and Keller, N. P. (2006) *Chem. Biol.* **13**, 31–37
33. Fischer, R. T., Stam, S. H., Johnson, P. R., Ko, S. S., Magolda, R. L., Gaylor, J. L., and Trzaskos, J. M. (1989) *J. Lipid Res.* **30**, 1621–1632
34. Lamb, D. C., Kelly, D. E., Manning, N. J., Kaderbhai, M. A., and Kelly, S. L. (1999) *FEBS Lett.* **462**, 283–288
35. Rojas, M. C., Hedden, P., Gaskin, P., and Tudzynski, B. (2001) *Proc. Natl. Acad. Sci. U. S. A.* **98**, 5838–5843
36. Suzuki, K., Sanga, K., Chikaoka, Y., and Itagaki, E. (1993) *Biochim. Biophys. Acta* **1203**, 215–223
37. Tudzynski, B., Hedden, P., Carrera, E., and Gaskin, P. (2001) *Appl. Environ. Microbiol.* **67**, 3514–3522
38. Kawaide, H. (2006) *Biosci. Biotechnol. Biochem.* **70**, 583–590
39. Penn, J., and Mantle, P. G. (1994) *Phytochemistry* **35**, 921–926
40. Young, C. A., Felitti, S., Shields, K., Spangenberg, G., Johnson, R. D., Bryan, G. T., Saikia, S., and Scott, B. (2006) *Fungal Genet. Biol.* **43**, 679–693

Enzyme Catalysis and Regulation:
Defining Paxilline Biosynthesis in
Penicillium paxilli: FUNCTIONAL
CHARACTERIZATION OF TWO
CYTOCHROME P450
MONOOXYGENASES

Sanjay Saikia, Emily J. Parker, Albert
Koulman and Barry Scott
J. Biol. Chem. 2007, 282:16829-16837.
doi: 10.1074/jbc.M701626200 originally published online April 11, 2007

Access the most updated version of this article at doi: [10.1074/jbc.M701626200](http://dx.doi.org/10.1074/jbc.M701626200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](http://www.jbc.org/site/misc/affinity-sites.xhtml).

Alerts:
- [When this article is cited](http://www.jbc.org/cgi/alerts/doi?doi=10.1074%2Fjbc.M701626200)
- [When a correction for this article is posted](http://www.jbc.org/cgi/alerts/correction?doi=10.1074%2Fjbc.M701626200)

[Click here](http://www.jbc.org/cgi/alerts/register) to choose from all of JBC's e-mail alerts

Supplemental material:
[http://www.jbc.org/content/suppl/2007/04/13/M701626200.DC1.html](http://www.jbc.org/content/suppl/2007/04/13/M701626200.DC1.html)

This article cites 38 references, 11 of which can be accessed free at
[http://www.jbc.org/content/282/23/16829.full.html#ref-list-1](http://www.jbc.org/content/282/23/16829.full.html#ref-list-1)
