
Available online at www.sciencedirect.com

ScienceDirect

Biochimica et Biophysica Acta 1773 (2007) 1177–1195

Review

Ras oncogenes and their downstream targets

Krishnaraj Rajalingam, Ralf Schreck, Ulf R. Rapp, Štefan Albert*

University of Würzburg, Institut für Medizinische Strahlenkunde und Zellforschung, Versbacherstr. 5, D-97078 Würzburg, Germany

Received 8 August 2006; received in revised form 17 January 2007; accepted 19 January 2007

Available online 28 January 2007

---

### Abstract

RAS proteins are small GTPases, which serve as master regulators of a myriad of signaling cascades involved in highly diverse cellular processes. RAS oncogenes have been originally discovered as retroviral oncogenes, and ever since constitutively activating RAS mutations have been identified in human tumors, they are in the focus of intense research. In this review, we summarize the biochemical properties of RAS proteins, trace down the evolution of RAS signaling and present an overview of the spatio-temporal activation of major RAS isoforms. We further discuss RAS effector pathways, their role in normal and transformed cell physiology and summarize ongoing attempts to interfere with aberrant RAS signaling. Finally, we comment on the role of micro RNAs in modulating RAS expression, contribution of RAS to stem cell function and on high-throughput analyses of RAS signaling networks.

© 2007 Elsevier B.V. All rights reserved.

Keywords: RAS; Small GTPases; RAS effector pathways; Mitogenic cascade; Oncogenic signaling; Mutational activation

---

### 1. Introduction

RAS proteins control cellular signaling pathways responsible for growth, migration, adhesion, cytoskeletal integrity, survival and differentiation. RAS proteins belong to the large family of small GTPases, which are activated in response to various extracellular stimuli. For instance, RAS proteins are activated in response to growth factor stimulation and subsequently bind to numerous effector proteins leading to the activation of several signaling cascades within the cell. Since defects in RAS signaling may result in malignant transformation, the activation of RAS proteins is tightly controlled in normal cells. In fact, RAS genes were originally identified as retroviral oncogenes in the 1960/70s from the genome of Harvey and Kirsten rat sarcoma viruses [1]. RAS research has gained momentum in 1980s, when constitutively activating RAS mutations were identified in human tumors [1]. It is estimated now that approximately 20% of all human tumors have activating mutations in one of the RAS genes. Therefore, multiple approaches are undertaken to develop tumor therapies efficiently targeting RAS and RAS effector pathways. In the past decades, research on RAS signaling has shed a lot of light on the regulation of RAS effectors, signaling cascades and their cross talk in modulating various cellular processes. In this review, we attempt to present an overview of the recent advances made in the field of RAS biology.

---

### 2. Structure and biochemistry of small GTPases

Due to their preeminent role in tumorigenesis, RAS proteins are the founding members of the superfamily of small GTP binding proteins, also called RAS-like GTPases or RAS superfamily of GTP-binding proteins. A common feature of RAS proteins is that they function in signal transduction across membranes, in particular in signal transfer induced by growth factors. Their primary role is to assemble transient signaling complexes at the membrane that activate signal transduction pathways coordinating transcription, cell shape and migration, endocytosis, cell survival and cell cycle progression, differentiation, senescence and more.

The question arises how many RAS family members are there to fulfill all of these different functions? The main problem is to define appropriate criteria for the entry of RAS and RAS-like proteins into this superfamily as well as for their classification into subfamilies. In general, small G-proteins are in the range of 20 to 29 kDa, share sequence homologies and

* Corresponding author. Tel.: +49 931 201 45141; fax: +49 931 201 45835.
E-mail address: stefan.albert@mail.uni-wuerzburg.de (Š. Albert).

0167-4889/$ - see front matter © 2007 Elsevier B.V. All rights reserved.
doi:10.1016/j.bbamcr.2007.01.012

common motifs and are nearly identical at their tertiary structure, which is composed of six β-sheets surrounded by α-helices. More than 150 RAS-like genes have been identified in mammalian genomes [2,3]. The whole RAS superfamily has been divided into five subfamilies: RAS, RHO, RAB, ARF and Ga subunits of heterotrimeric G-proteins [2]. In contrast, Channing Der and colleagues did not include Ga subunits under this superfamily and assigned RAN to a distinct RAS subfamily [3]. Both groups have included RAP, RAL, RHEB and other GTPases into the subfamily of RAS proteins, which trigger effector pathways not primarily used by RAS [1,3]. Other classifications rely on the simplistic view that there are only three true RAS proteins (H-, N-, and KRAS), and subsume other family members as RAS-like proteins. However, MRAS, RRAS and ERAS definitely belong to the RAS subfamily, as they signal through at least one of the RAS effector pathways, sometimes in a cell type- and/or adaptor-dependent manner [4–6]. We realize that the classification of RAS proteins based on homology and function bears many caveats that have to be re-addressed in future.

Molecular processes underlying the function of RAS as a binary switch are a paradigm for the whole superfamily of small G-proteins. They alternate between GTP- and GDP-bound conformations, where the GTP-bound conformation represents the “On” and GDP-bound the “Off” state. Upon binding, two regions of RAS undergo dramatic structural changes depending on the type of bound nucleotide. These are called Switch region I (aa 30–38) and Switch region II (aa 59–67) and form an interaction surface for effector molecules in a GTP-dependent manner [7], see Fig. 1. The effector loop is a fingerprint of small GTPases determining the specificity of effector binding to a given GTPase.

In the presence of magnesium ions, the association of guanine nucleotides with small G-proteins is generally very tight. The dissociation constants of GDP and GTP are in the picomolar range (Kd approximately 10–100 pM) [8], and therefore G-proteins are usually present in a complex with either GTP or GDP. Although RAS proteins have a measurable GTPase activity on their own (GTP hydrolysis rate constant: 0.028 min−1) [9], it is too low to be relevant at the physiological level. This fact implies that transitions between the “On” and “Off” states of RAS require additional help from specialized proteins accelerating GTP hydrolysis. These GTPase accelerators are termed GTPase activating proteins or GAPs and increase the GTP hydrolysis of RAS by approximately 100 000 fold [10]. In order to recharge the GTPase, the binding of GDP must be weakened by the action of guanine nucleotide exchange factors or GEFs, which catalyze its replacement with GTP.

As will be discussed later, the oncogenic mutants of RAS have amino acid exchanges predominantly at three residues: G12, G13 and Q61 [11]. These substitutions prevent the intrinsic and GAP catalyzed hydrolysis of GTP, thereby generating permanently active RAS molecules with severe consequences for the cell. The solution of the crystal structure of a RAS·GDP·p120GAP complex mimicking the transition state of GTP-hydrolysis shed light on the molecular basis of the oncogenicity of activated RAS mutants [12]. Wittinghofer and coworkers described that the presence of any side chain at position G12 and partially also at position G13 is preventing the proper insertion of the catalytic “arginine finger” provided in trans by RasGAP into a position near the β- and γ-phosphates of GTP. In complex with wild-type RAS this positive arginine charge is neutralizing the negative charge, which is required for the cleavage of the bond between the β- and γ-phosphates. RAS Q61 is located in the switch II region and is mostly conserved among small G-proteins and is thought to activate the water molecule for an attack of the γ-phosphate of GTP. The relocation and fixation of Q61 in a position suitable for catalysis is responsible for the so-called allosteric part of GAP activity, which is detected as residual activity in GAP mutants with a substituted arginine finger [13].

It is worth mentioning that some GTPases of the RAS superfamily are less addicted to glutamine at positions homologous to Q61. For example, RAP GTPases with threonine at the Q61 position use the asparagine “thumb” of Rap1GAP, which takes the role of Q61 [14]. As a second example, GTP hydrolysis by RAB GTPases with Q61 substituted by leucine is efficiently accelerated by their cognate GAPs to a degree similar to wild type proteins. This acceleration is fully dependent on the presence of the arginine finger [15]. It can be concluded, that despite similarities in the overall structure and buildup of the catalytic center, there is no common reaction mechanism for GTP hydrolysis by small G-proteins [16].

Due to problems with X-ray diffraction of crystallized full-length proteins, structural studies of RAS proteins have been done so far only with C-terminally truncated RAS proteins containing the complete GTPase domain but lacking up to 23 C-terminal residues. In comparison to their full-length counterparts.

parts, truncated RAS proteins were shown to have identical biochemical parameters, such as GTP/GDP binding and hydrolysis [9]. However, the hypervariable C-terminal regions of RAS proteins are the sites of posttranslational modifications determining subcellular localization as well as some biological properties of RAS isoforms (discussed in detail later).

### 3. On the evolution of small G-proteins

#### 3.1. The early evolution of eukaryotes and the functional diversification of small G-proteins

RAS signaling seems to be relatively recent in terms of evolution even if the whole RAS superfamily is considered. The division into membrane-enclosed sub-compartments dramatically increased the complexity of eukaryotic cells and brought about a demand for communication and precisely regulated trafficking between individual compartments. As a result of multiplication and functional diversification, small G-proteins became central regulatory units of nearly all aspects of intra- and intercellular communication [17]. The following processes are regulated by small G-proteins: import of proteins from the cytoplasm into the endoplasmic reticulum (ER) by a signal recognition particle (SRP), whose β-subunit SRPβ is a G-protein. Vesicle budding as well as coat assembly on the surface of transport vesicles by Sar and Arf GTPases. In addition, vesicle docking and targeting are regulated by RAB GTPases, cytoskeleton and cell shape are under the control of RHO-GTPases and exchange of proteins and nucleic acids between nucleus and cytoplasm is governed by RAN GTPase. However, it should be emphasized that this subdivision based on functional criteria is only arbitrary and cross talk between different subfamilies of the RAS superfamily has been described [18].

#### 3.2. Co-evolution of RAS signaling and intracellular processing of extracellular signals

One of the most surprising findings in the RAS field is that none of the so far sequenced plant genomes encoded either RAS or RasGAP proteins [19], although they are particularly rich in RAB and RHO-like GTPases [20]. This absence has been linked to the lack of receptor tyrosine kinases in plants, which are utilizing a different set of receptor serine/threonine- and histidine kinases instead [21]. Therefore, it is conceivable that animals and plants diverged prior to the occurrence of the RAS signaling module. However, the overall scenario seems to be a bit more complicated. For example, while baker’s yeast *Saccharomyces cerevisiae* possesses two redundant RAS genes, which are involved in cAMP signaling [22], fission yeast *Schizosaccharomyces pombe* contains a single RAS gene, which was shown recently to signal through two different cascades depending on its intracellular localization [23]. Amoeba *Dictyostelium discoideum* contains several RAS proteins regulating actin assembly and mobility, endocytosis and developmental patterning [24], but no RAF gene [25]. The genomes of *Giardia* and *Trypanosoma*, primitive unicellular eukaryotes, contain no

RAS gene [26], which might be either due to genome reduction or constitute a further proof of the later origin of RAS.

In metazoans, the major effectors of RAS are RAF kinases regulating the mitogenic cascade, one of the MAP kinase cascades. Since multicellular plants also contain MAP kinase cascades, it may be expected that they also contain RAF as a MAP kinase regulator. In this scenario RAF would presumably not be hooked up with a RAS binding domain (RBD) but instead be outfitted with other regulatory domains. In fact, as many as 52 RAF-related kinases have been identified in the *Arabidopsis* genome and have been shown to be significantly divergent at non-catalytic sequences in comparison to metazoans [27].

Plant genomes, as already mentioned, contain no RAS genes at all. As expected, we have not been able to detect kinases bearing regions with a homology to the RBD of RAF by database analysis of the *Arabidopsis* genome. This observation gains further support by genetic analyses of ethylene signaling in plants, where one of the presumable RAF orthologs, CTR1, is under the control of the ethylene receptor histidine kinase ETR1 [28]. On the other hand, the large number of proteins containing a RAF-like kinase domain in early diverged plants supports the view that the tyrosine kinase-like (TKL) branch of kinases, including RAF kinases, is of ancient origin.

Taken together, we propose that three major events have been involved in the evolution of RAS signaling (see Fig. 2). (I) Functional diversification of G-proteins accompanying sub-cellular compartmentalization in early eukaryotes. (II) Invention of RAS at the level of development of opisthokonts/amoebozoa lineage. In its archaic form, RAS was not feeding through RAF into the MAP kinase cascade. (III) Gain of a RBD by (one of) the preexisting RAF-like kinase(s) by domain shuffling and the appearance of receptor tyrosine kinase signaling in conjunction with multicellularity in metazoans. This enabled RAF to become the primary messenger of RAS-mediated signals from receptor tyrosine kinases to the MEK-ERK pathway in this lineage. In contrast, although the multicellularity and the

![Figure 2](#fig2)

Fig. 2. Major steps in the evolution of RAS and RAF signaling. The eukaryotic branch of the tree of life was adopted from <http://tolweb.org> and [225]. (I) The compartmentalization of eukaryotic cells into membrane-enclosed organelles was accompanied by gene duplication and functional diversification of small G-proteins. (II) Genomic evidence puts the appearance of RAS to a common ancestor of opisthokonts and slime molds. (III) RAF kinases linking RAS and MAPK signaling accompanied the development of multicellularity in animals. For more details, see text.

development of the tyrosine kinase-like branch of the plant kinome evolved in parallel to the multicellularity in the animal kingdom, the final two steps were never performed and therefore the tyrosine kinase branch of the kinome and RAS are missing in plants. Instead, other signaling pathways and signaling molecules have been generated, which led to the differences in intra- and intercellular signaling observed today.

4. Functional diversity of RAS isoforms by differential expression, localization and spatio-temporal signaling

4.1. Expression analyses and knock-out studies of RAS isoforms

The expression analysis of RAS isoforms revealed significant variations between tissues as well as during development. For instance in mice, HRAS transcripts are highly expressed in brain, muscle and skin and lowest in liver, while KRAS transcripts are readily detected in gut, lung and thymus. NRAS transcripts are primarily expressed in the testis and thymus [29]. Differential expression is also observed during mouse development where NRAS expression is high at day 10 of gestation and the expression of KRAS is reduced during the end of gestation [30]. These descriptive analyses clearly suggested that the three isoforms of RAS perform distinct cell type and tissue specific functions. Mouse knock-out studies revealed an obligatory role for K- but not H- or NRAS during mouse development [31–34]. While mice deficient for HRAS or NRAS lack any obvious phenotype, KRAS mutated mice developed embryos, which died between 12 and 14 days of gestation, with fetal liver defects and evidence of anemia. The phenotype observed in KRAS mutant mice may be attributed to the predominant expression of this isoform in a critical cell type (e.g. within the fetal liver) [32].

4.2. Differential activation and localization of RAS isoforms

Another line of evidence often considered for individual roles of RAS isoforms pertains to the analysis of constitutively activating RAS mutations in human cancer. Though the mutated forms of the three RAS genes produce the same phenotype in in vitro transformation assays [35], the oncogenic forms of K-, H-, and NRAS are preferentially detected in specific tumor types. For example, KRAS mutations occur primarily in colon and pancreatic cancers, NRAS mutations are often detected in myeloid leukemia and HRAS mutations are high in bladder carcinomas [35]. The isoforms display very high degree of sequence homology and the major differences are confined to the hypervariable region (HVR), at the C-terminus. The HVR encompass approximately 25 aa, which are required for the posttranslational modifications that direct plasma membrane anchoring as well as trafficking of newly synthesized RAS proteins to the inner surface of the plasma membrane from the cytosolic surface of the ER [36]. The first modification described for RAS was the attachment of a 15-carbon isoprenyl group to C186 by farnesyl transferase, which is irreversible and accompanied by cleavage of the three C-terminal amino acids and subsequent carboxymethylation of the C-terminal cysteine. Farnesylation is found in all RAS proteins and is essential for their biological function [37]. However, it has been shown that the C-terminal farnesylation does not suffice for proper targeting of RAS to membranes. Each of the three prototype RAS proteins (H-, N-, and KRAS) undergoes additional modifications. Whereas HRAS is palmitoylated on C181 and C184, NRAS is palmitoylated on C181 and KRAS4B does not have any palmitoylation at all. Instead, KRAS4B possesses the so-called polybasic region, a stretch of lysines, which are believed to interact with negatively charged head groups of plasma membrane lipids. While palmitoylated H- and NRAS enter the exocytic pathway through Golgi to reach the plasma membrane, KRAS4B bypasses the Golgi. The trafficking of alternatively spliced KRAS4A is believed to follow H- and NRAS as this splice variant lacks the polybasic domain and as it is palmitoylated at C180. The key enzyme responsible for palmitoylation of H- and NRAS is RAS palmitoyl transferase (RPT), which was cloned from S. cerevisiae [38]. The various stages of RAS modification and intracellular trafficking are summarized in Fig. 3.

Recent studies with live cell imaging, fluorescent resonance energy transfer (FRET), fluorescent recovery after photobleaching (FRAP), electron microscopy (EM) and single particle tracking analysis (SPT) have shed more light into the dynamics of RAS proteins and their interactions with membranes [36]. These studies revealed that RAS generates signal outputs when associated with non-plasma membranes as well. Interestingly, differences in the HVR directed RAS proteins to different microdomains of the plasma membrane. The plasma membrane, which was long considered to be a uniform lipid bilayer, has turned out to constitute a complex mosaic of microdomains with various compositions of proteins and lipids [39]. Palmitoylation of RAS isoforms may increase their membrane affinity so that they can be captured by the exocytotic pathway to the trans Golgi and subsequently be sorted to the various plasma membrane microdomains. Association of non-palmitoylated RAS is detected in the ER by live cell imaging, although the association is weak and can be dislodged by hypotonic lysis of cells [40,41]. In contrast to H- and NRAS, KRAS4B cannot be detected at the Golgi and has been shown to interact with microtubules via the polybasic domain. In addition, prenylation and methylation of KRAS4B is also required for this interaction [42,43]. In addition, the two palmitoylated RAS proteins were shown to undergo ubiquitination and are subsequently targeted to endosomes, a cellular localization that prevents them from activation of the MAP cascade [44].

4.3. Spatio-temporal activation of RAS

The spatio-temporal activation of RAS isoforms and their downstream signaling cascades is highly influenced by their distinct microdomain localization within the plasma membrane. A heterogeneous mixture of lipids and proteins leads to the organization of various microdomains of which lipid rafts and caveolae are well characterized [39]. These microdomains serve

K. Rajalingam et al. / Biochimica et Biophysica Acta 1773 (2007) 1177–1195

![Diagram](attachment:diagram.png)

Fig. 3. Modifications and localization of RAS proteins. RAS proteins are synthesized in the cytosol and farnesylated by proteinfarnesyl transferase (PFT) at the cysteine residue of the CAAX motif (A is an aliphatic amino acid and X is any amino acid). The cleavage of AAX tripeptide and methylation occurs at the cytosolic surface of the ER and are mediated by RAS converting enzyme 1 (Rce 1) and Isoprenylcysteine carboxylmethyl transferase (Icmt), respectively. H- and NRAS further undergo palmitoylation in the hypervariable region (HVR) on the Golgi to reach the plasma membrane via vesicular transport, while KRAS reaches the plasma membrane by a yet uncharacterized pathway. Depalmitoylation of H- and NRAS will lead to the localization of these two isoforms in the Golgi.

as signaling platforms, as the increased association or exclusion of specific signaling proteins to these structures confers specificity and intensity of signaling processes. Quantitative EM analysis of intact plasma membrane sheets revealed that neither HRAS nor KRAS are particularly localized in the caveolae, although some of the downstream RAS effectors have been detected there [45,46]. In fact, most of the RAS proteins were found in non-raft part of the plasma membrane. For instance, KRAS is predominantly not associated with lipid rafts irrespective of its activation status. GTP-HRAS is also primarily localized outside lipid rafts, whilst 50% of GDP-HRAS is detected in the caveolae [46]. Based on these results, it has been proposed that HRAS is in a dynamic equilibrium between the lipid raft and non-raft microdomains of the plasma membrane, a process which is regulated by GTP loading.

The localization of H- and KRAS to plasma membrane microdomains is not disrupted by cholesterol depletion suggesting that both isoforms function in non-raft microdomains, also called nanoclusters in the plasma membrane. A recent study by Hancock and colleagues revealed that the two palmitates of HRAS have distinct biological roles. While monopalmitoylation of C181 can efficiently target HRAS to plasma membrane, it fails to segregate HRAS from cholesterol rich to cholesterol independent microdomains. On the other hand, monopalmitoylation of C184 does not permit efficient trafficking beyond the Golgi apparatus, although this mutant once in the plasma membrane can segregate HRAS from rafts to non-raft microdomains upon GTP loading [47]. NRAS also seems to exhibit GTP dependent changes in its localization like HRAS as GTP loaded NRAS is predominantly localized to lipid rafts, while the GDP loaded form is primarily confined to the non-raft microdomains [47]. In addition to the modifications in the anchor region and the activation state, the linker domain of the HVR also contributes to the HRAS microlocalization as deletion of this domain confines HRAS primarily to lipid rafts [48].

Apart from these modifications, additional proteins interacting with RAS are also modulating the microlocalization of RAS isoforms. For instance, the lectin galectin-1 interacts strongly with activated HRAS and to a lesser extent with KRAS but not with NRAS or inactive K- or HRAS. Downregulation of galectin-1 inhibits cellular transformation with G12V HRAS, while overexpression of galectin increases the size of G12V HRAS clusters leading to increased activation of the MAPK pathway [49,50]. Galectin-3, which is expressed frequently in human tumors, specifically binds to activated KRAS4B and stabilizes GTP-KRAS by reducing the efficiency of p120<sup>GAP</sup>-mediated GTP hydrolysis [51]. Another protein that interacts with the C-terminus of HRAS is Phosphodiesterase δ (PDEδ). It binds more strongly to GDP- than GTP-HRAS and when ectopically expressed results in the redistribution of HRAS from the plasma membrane to the cytosol [52]. Moreover, scaffolding proteins also influence coupling of active RAS with its effectors. For instance, leucine rich repeat protein Sur-8 interacts with active RAS and RAF and induces MAPK activation [53].

1182 K. Rajalingam et al. / Biochimica et Biophysica Acta 1773 (2007) 1177–1195

### 4.4. RAS signaling from endomembranes

The first indication that RAS/MAPK signaling occurs in membranes apart from the plasma membrane was the detection of Shc, Grb2–SOS and phosphorylated RAF in the endosomal fraction isolated from cells after EGF treatment [54]. These observations were subsequently supported by expression of the dominant-negative K44A mutant of dynamin, which blocked the activation of ERK induced by G-protein coupled receptors [55] as well as neurite outgrowth in response to NGF in PC12 cells [56]. As endosomes are derived from the plasma membrane, the observation of transmission of signaling complexes from the plasma membrane to the endosomes was not surprising. The presence of a significant pool of H- and NRAS in the Golgi raised the possibility that RAS/MAPK signaling may indeed take place in this organelle. Using a fluorescently labeled RAF-RBD Chiu et al. demonstrated the activation of RAS at Golgi and plasma membrane [57]. While the activation of RAS at the plasma membrane was transient, activation of RAS at the Golgi was delayed and sustained [57]. In addition, RAS-GRP, a RAS GEF (discussed below), localized to Golgi and seems to be responsible for the activation of RAS in this compartment. The acylation/deacylation cycle of RAS also contributes to the activation of RAS at this organelle [58]. For example, expression of palmitoylation-deficient mutants of H- and NRAS provoke their accumulation in the ER and cytosol [41]. Components of RAS/MAPK have also been reported on mitochondria. The localization of Ras and its effectors in the mitochondria are discussed in detail in an accompanying review.

Besides subcellular localization, the intensity and duration of RAS signaling profoundly influence downstream signaling networks and the final signal output [59,60]. One of the finest examples we would like to highlight here are the studies performed with the rat PC12 pheochromocytoma cells [60]. When treated with NGF, PC12 cells express neuronal markers and show neurite outgrowth. However, when treated with insulin or EGF, they show only a weak proliferative response. Though both receptors activate RAS, the timing of RAS/MAPK activation with both the receptors was different. While treatment with EGF leads to a transient activation of ERK, treatment with NGF caused a sustained RAS/MAPK activation leading to differentiation of these cells. These observations have also been attributed to the sustained activation of Rap1b with NGF but not with EGF [61,62].

### 4.5. RAS GEFs

As mentioned above, the activity of RAS proteins is tightly controlled by switching between the GDP- and GTP-bound states. Although RAS proteins possess intrinsic GTP-ase activity, regulatory proteins like guanine nucleotide exchange factors (GEFs) and GTPase activating proteins (GAPs) profoundly influence the activation of RAS. As the cellular amount of GTP is tenfold higher than GDP, GEFs promote the formation of active GTP–RAS, while GAPs stimulate the intrinsic rate of GTP hydrolysis and thereby the formation of GDP-RAS. Oncogenic RAS mutations at residues G12, G13 and Q61 render them constitutively active (GTP–RAS) as they are impaired in their intrinsic and GAP-mediated GTP hydrolysis. Three main classes of RAS GEFs are currently known with a common CDC25 homology catalytic domain and an N-terminal RAS exchange motif: SOS, RAS-GRF and RAS-GRP. SOS and RAS-GRF also serve as GEFs for Rac GTPases.

The role of SOS in receptor tyrosine kinase-mediated activation of RAS is very well established. SOS, in addition to the SOS homology domain encompasses a Dbl homology (DH) domain and pleckstrin homology (PH) domain. The DH homology domain mediates the GDP/GTP exchange on Rho GTPases, while the PH domain is responsible for membrane localization. In resting cells, SOS is primarily found to be associated with the Src homology 3 (SH3) domains of the Grb2 adaptor protein and upon receptor stimulation the Grb2–SOS complex is recruited to the plasma membrane. This recruitment is primarily mediated by the interaction of the SH2 domain of Grb2 with the tyrosine phosphorylated residues in the cytoplasmic domain of the activated receptor or to the tyrosine phosphorylated Shc, another adaptor protein implied in this process [63]. This association with the activated receptor brings SOS to the vicinity of RAS leading to the formation of GTP–RAS. Grb2–SOS complex is also utilized by G protein coupled receptors to activate RAS through Ca²⁺ dependent activation of PYK2 [64]. Recent structural studies suggested RAS–GTP as an allosteric activator of SOS. Binding of RAS–GDP to the distal allosteric site in SOS reduced the activity of SOS, while binding of RAS–GTP to this site enhances the activity of SOS [65,66].

RAS–Guanine Nucleotide Releasing Factors or RAS–GRFs are also members of the GEF family [67]. RAS–GRFs share 80% homology and have a common modular structure: C-terminal guanine nucleotide exchange domain with the DH and PH domains in the center of the protein. In addition, an IQ domain (short calmodulin-binding motif containing I and Q residues) is present at the N-terminus and required for activation. RAS–GRFs act on H-, N- and KRAS as well as on RRAS, and the RAS–GEF activity of these proteins is triggered by the association of Ca²⁺-Calmodulin with the IQ domain. Studies with *grf-1* or *grf-2* deficient mice revealed that both RAS–GRF1 and RAS–GRF2 function in coupling of N-methyl-D-aspartate glutamate ligand-gated ion channel receptors to the activation of MAPK signaling in the neurons of adult mice [68,69].

RAS guanine nucleotide releasing proteins or RAS–GRPs are additional members of the GEF family and were initially cloned from brain and lymphoid tissues [70,71]. RAS–GRPs, also called CalDAG-GEFs, constitute a group of GEFs possessing calcium and diacylglycerol binding domains and can be triggered by diacylglycerol and phorbol ester. RAS–GRPs also associate with endomembranes and activate RAS. For instance, GRP1 and GRP3 localize to Golgi thereby activating Golgi-associated RAS [72], while RAS–GRP2 activates K- and NRAS [73]. DAG binding recruits RAS–GRPs to plasma membrane. RAS–GRP1-deficient animals showed impaired T-cell development and impaired T-cell

stimulated activation of RAS [74], whereas RasGRP-2 deficient mice were severely compromised in integrin signaling and displayed increased bleeding [75].

### 4.6. RAS GAPs

p120<sup>GAP</sup> was the first identified member of the GAP family. In addition to the catalytic domain, this protein also harbors SH2, SH3 and PH domains and phospholipid binding motifs. RAS-GAP was the first protein found to interact with the so-called effector domain of RAS [76]. Consequently, RAS-GAP was originally perceived as a major RAS signal terminator. Indeed, initial evidence suggested that RAS-GAP is in a complex with receptor and non-receptor tyrosine kinases [77,78]. However, in contrast to GEFs, there are only a few details known with respect to pathways modulating RAS-GAP activity. Yang et al. demonstrated that the partial cleavage of RAS-GAP by caspase-3 is required to activate AKT leading to cell survival under mild stress conditions [79]. Neurofibromin, another RAS-GAP, acts as a tumor suppressor gene and it is lost in the autosomal dominantly inherited disorder neurofibromatosis type 1 (NF1). Intracellular levels of neurofibromin have been shown to be dynamically regulated by the ubiquitin-proteasome pathway [80]. Degradation is rapidly triggered in response to growth factors and requires sequences adjacent to the catalytic GAP-related domain of neurofibromin. However, whereas degradation is rapid, neurofibromin levels are re-elevated shortly after growth factor treatment. Accordingly, NF1-deficient mouse embryonic fibroblasts exhibit an enhanced activation of RAS, prolonged RAS and ERK activities, and proliferate in response to sub-threshold levels of growth factors. Recent evidence suggests a crucial role for NF1 in the activation of mTOR, an evolutionarily conserved serine/threonine protein kinase that regulates cell growth and proliferation in yeast, flies, and mammals. mTOR is constitutively activated in cells derived from NF1-deficient mice and this aberrant activation depends on the phosphorylation of tuberin by activated AKT [81]. Additional RAS-GAPs have been identified and their mode of activation has been characterized. The Ca<sup>2+</sup>-promoted RAS inactivator CAPRI was identified as a RAS-GAP stimulated by elevated intracellular Ca<sup>2+</sup> levels leading to attenuation of RAS activation and MAPK activity [82]. Another member of the GAP1 family responding to Ca<sup>2+</sup> spikes is RAS GTPase-activating-like RASAL, which is highly expressed in follicular cells of the thyroid and the adrenal medulla [83]. RASAL oscillates between the cytosol and the plasma membrane in response to Ca<sup>2+</sup> spikes thereby decoding the complex Ca<sup>2+</sup> oscillations into a dynamic regulation in the activation of RAS [84].

Finally, GAP1 IP4BP and GAP1<sup>m</sup> are other members of this GAP1 family with a PH domain. GAPIP4BP is constitutively associated with the plasma membrane mediated by PIP2 binding, the RASGAP activity is regulated by Inositol 1,3,4,5-tetrakisphosphate [85]. GAP1<sup>m</sup> is localized to the perinuclear region of the cytoplasm and upon activation of PI3K gets recruited to the plasma membrane by PIP3 binding

[82,86]. SynGAP, is another member of this family, which is selectively expressed in the brain and is highly enriched at excitatory synapses [87]. SynGAP regulates synaptic plasticity and MAPK signaling in neurons [88]. In the following we will discuss individual RAS effector pathways after a brief comment on the historical events leading to the discovery of RAF as the first RAS effector.

## 5. RAS and the mitogenic cascade, brief history

In the mid 1980s, several labs [89–92] explored the connection between growth factor/cytokine signaling and intracellular oncoproteins by growth factor abrogation experiments with cells that specifically depend on a particular factor for growth and survival. Gene transfer experiments with retroviruses and different classes of oncogenes alone or in combination were subsequently used to establish the broad scheme of signal transduction cascades connecting cell surface receptors with cell cycle progression, differentiation and cell survival processes [92,93]. A hierarchical sequence was drafted that had protein tyrosine kinases on top followed by RAS GTPases, RAF serine/threonine kinase and oncogene class transcription factors, such as Myc, AP1 and Ets. Later, we found that MAP kinase, another serine/threonine kinase, was also constitutively active in v-Raf transformed NIH 3T3 cells leading to the identification of the RAF-MEK-ERK segment of the mitogenic cascade [94] thereby placing Myc, AP1 and Ets downstream of MAP kinase. The discovery of RAF as an effector and the observations leading to the discovery are discussed below.

## 6. RAS effectors

RAS effectors are defined as proteins with a strong affinity to GTP-RAS, whose binding is impaired by mutations within the core effector domain. The binding of RAS effector proteins to GTP-RAS triggers distinct signaling cascades. However, the notion that GDP-RAS does not have any functional role has been challenged by the recent observation that GDP-RAS indeed interacts with several effector proteins and modulates downstream signaling events. For instance, GDP-RAS binds to the transcription factor Aiolos, thereby modulating the nuclear translocation of Aiolos and the expression of anti-apoptotic protein BCL-2 [95,96]. In 1993, RAF kinase was first discovered as a RAS effector followed by Ral guanine nucleotide dissociation stimulator (RalGDS) and phosphatidylinositol 3-kinase (PI3K). Apart from RAF, PI3K, RalGDS and p120<sup>GAP</sup> the growing family of RAS effector proteins includes Rin1, Tiam, Af6, Nore1, PLCε and PKCζ. Currently there are more than 10 different RAS effectors (see Fig. 4), and several of them contain functionally related isoforms [97]. RAS effector proteins are characterized by the presence of a putative RAS binding domain or RBD. At least three distinct RBDs are recognized: (1) the RBD of RAF and TIAM 1, (2) the RBD from PI3K, and (3) the RAS association (RA) domains of RalGDS and AF6. The structures of the four RBDs solved so far displayed the same topology, the ubiquitin fold (ββαββαβ),

that was first presented in public by Ulf R. Rapp at a NATO conference in Spetsai in August 1992 (mentioned in [104]).

(1) Antisense and dominant-negative RAF blocked transformation by RAS oncogenes in culture [105]. Dominant-negative RAF mutants included a kinase dead version of full-length RAF as well as a more potent fragment of 257 aa N-terminal fragment called RAF-C4, which encompasses the cysteine rich domain (CRD). Point mutation within the CRD abolished the dominant-negative effect of RAF-C4, and the synergistic interaction between RAS and full-length RAF as examined in transformation or receptor gene assays [106].

(2) RAS was shown to activate RAF following growth factor induction of NIH 3T3 cells and NGF induction of PC12 cells and this activation was blocked with a dominant-negative RAS (M17 RAS [107]).

(3) Protein kinase Cα (PKCα), which is being activated at the plasma membrane was observed to phosphorylate RAF [108].

(4) As activation of PKC and of RAS occurs at the plasma membrane, whereas RAF shows cytosolic staining, and because RAS was thought to be a timer of direct protein–protein interactions (Dieter Gallwitz, personal communication), RAS was proposed to bring (translocate) RAF to the plasma membrane into the neighborhood of active PKC. Based on these findings Rapp et al. proposed in 1992 that RAF and a lipid-derived cofactor directly interacted with RAS at or near the CRD [104]. In contrast, the prevailing view at the time was that RAS–GAP had hallmarks of a RAS effector [109,110], and no other candidates were seriously considered. To evaluate the question as to the nature of a link between RAS and RAF we settled on a collaboration with Joe Avruch’s laboratory leading to a report in the spring of 1993 on “p21-Rafbind to the amino terminal regulatory region of c-raf1” [111]. The question about lipid cofactors was advanced by the finding that phosphatidyl serine binds to the CRD and recently we deciphered that the farnesyl moiety of RAS also has to interact with this domain of C-RAF for efficient RAS/C-RAF coupling to take place at least in vitro and probably also at the membranes. Farnesyl interaction with the CRD is thought to be required for transmission of a conformational activation signal to the RAF kinase domain ([112] and Fischer A, Hekman M and Rapp UR, unpublished observations).

In 1993, four groups reported that RAF specifically interacts with activated GTP–RAS proteins [111,113–115]. The paper by Moodie and coworkers was published a month before the others [113]. This paper had been inspired by the membrane translocation hypotheses proposed by Ulf R. Rapp, which he had brought up in personal discussions with Weber during a seminar at the NCI in Frederick in fall 1992, an incident that Weber acknowledged at a AACR symposium on “Oncogenes and Antioncogenes in Differentiation, Development and Human Cancer” in Big Sky Resort, Montana, that they both

Fig. 4. RAS and RAS effector pathways. GTP–RAS binds to numerous effectors to trigger various signaling cascades, which in turn modulate different cellular processes ranging from cell growth, survival, cell migration, differentiation and death. For more details, see text.

suggesting a similar mode of interaction of RAS with its effectors [98].

### 6.1. RAF and the MEK/ERK cascade

More than 24 years ago, Ulf R. Rapp and colleagues cloned an oncogene acquired by the murine retrovirus 3611-MSV that was generated by targeted retroviral transduction [99]. Since 3611-MSV induced rapidly growing fibrosarcoma in mice, the transduced oncogene was named *v-raf* and the cellular homologue was called *c-raf-1* (-2 and higher numbers designate pseudogenes) [99]. Later, Klaus Bister and coworkers demonstrated that the naturally occurring avian retrovirus Mill-Hill No. 2 (MH2) carried a second cell-derived sequence besides *myc*, which was named *v-mil* after the virus [100]. By direct sequencing of both viral genomes, it was later proved that 3611-MSV and MH2 have integrated orthologues of the same gene into their genomes [101]. Homologues of *C-RAF* were also identified in *Drosophila melanogaster* (*D-RAF*) and *Caenorhabditis elegans* (*lin-45*). Two related RAF genes *A-RAF* and *B-RAF* were subsequently cloned. *B-RAF* appears to be the founder of the family and the orthologue of *D-RAF*. All three RAF isoforms share two conserved regions, CR1 and CR2, at the N-terminus, while a third conserved region CR3, encodes the kinase domain at the C-terminus. In regard to RAS/RAF function, the role of RAS as a signal transducer downstream of growth factor receptors was suggested by the observation that epidermal growth factor increases the amount of GTP–RAS and that microinjection of a monoclonal antibody to RAS blocked serum induced DNA synthesis [102]. The RAS block was relieved by the cytoplasmic oncoprotein MOS and the candidate downstream serine/threonine kinase effector RAF [103]. These findings left open whether RAF kinases function downstream or parallel to RAS and whether, in case of a downstream connection, the interaction was direct or indirect. The following lines of evidence pointed to a direct effector function of RAF

attended in February 1–6, 1993 and where Weber presented the data that subsequently appeared in the *Science* paper. Weber, in his seminar at NCI had favoured the idea of a “chaotic signaling network” that did not work with vectorial signal transfer modules.

Although it is still too early to come up with a final assessment of receptor/RAS-mediated assembly of signaling complexes at the plasma membrane, the current view of the role of the mitogenic cascade clearly involves the use of building blocks in specific combination and sequence. Much of the regulation lies in fact on the assembly process (RAS-RAF-MEK-ERK) at the site of receptor activation in the membranes, where the activation of signaling through the cascade may be triggered by the successful assembly of parts docking on each other with the assistance of scaffolding proteins [116]. Localization of MAP kinase signaling to these membrane sites is probably not only required for the integration of transcriptional activity in the nucleus by the mobile effector kinase ERK, but also for dynamic membrane processes and cytoskeletal rearrangements that are involved in endocytosis and cell migration (discussed in other chapters).

Later on, studies from several groups revealed that the RAS isoforms activate RAF with various intensities. For instance, Voice et al. demonstrated that KRAS is the most potent activator of C-RAF followed by N- and HRAS [117]. The RAF-RBD has been extensively studied by various groups to understand the spatio-temporal activation of RAS, as already discussed above [58,118]. In unstimulated cells, C-RAF is inactive and to a large extent localized in the cytosol. Upon RAS activation, RAF undergoes conformational changes, phosphorylation and hetero-oligomerisation leading to its full activation at the plasma membrane. The ability of RAS to activate C-RAF is regulated by binding of other proteins like 14-3-3 and prohibitin to C-RAF[119–121]. The authors would like to refer to accompanying reviews as well as recent ones on RAF kinases for detailed information [122,123].

### 6.2. PI3K

In addition to RAF, the interaction of PI3K with RAS and its role in various cellular processes have been thoroughly characterized. In its active form, PI3K is composed of a regulatory p85 subunit and a catalytic p110 subunit. In 1991, Lapetina and coworkers reported the association of PI3K activity with RAS, while the direct interaction of the catalytic subunit of PI3K with active RAS was demonstrated by Julian Downward and colleagues [124,125]. PI3K encompasses an RBD domain for interaction with active RAS, and the structure of the PI3K-RBD was solved either alone or in combination with RAS [98]. Among the RAS isoforms, HRAS was identified to be a more potent activator of the PI3K in comparison to KRAS [126]. Upon activation of PI3K the second messenger lipid phosphatidylinositol (3,4,5) triphosphate or PIP₃ is generated. PIP₃ recruits phosphatidylinositol-dependent kinase 1 (PDK1) and PKB/AKT to the plasma membrane, where PDK1 phosphorylates and activates AKT [127–129]. Three major isoforms of AKT are recognized: AKT1, AKT2, and AKT3. While AKT1 promotes cell proliferation and survival, AKT2 is associated with insulin-mediated metabolic processes. AKT3 deficient mice have reduced cell size and cell number suggesting a crucial role of AKT isoforms in modulating cell death and proliferation [129]. Some of the initial evidence for the contribution of PI3K to malignant transformation was the discovery of amplification of genes encoding the p110 subunit of PI3K and AKT2 in ovarian, breast and pancreatic cancer [130]. The high activity of PI3K in tumor cells is attributed to the loss of the phosphatidylinositol 3 kinase phosphatase with tensin homology or PTEN lipid phosphatase. PTEN catalyzes the removal of the D3 phosphate from PIP₃ to terminate downstream signaling, and is now thought to be the most commonly mutated tumor suppressor in humans, after p53 [127].

#### 6.3. Ral GEFs

RAL (RAS-Like) family GTPases were identified on the basis of their sequence homology to RAS. Two RAL genes, RALA and RALB, are ubiquitously expressed in humans and share 80% homology. RAL-GEFs serve as a link between the RAS and the Ral family of GTPases. Four distinct RAL-GEFs have been identified as effectors of RAS with RA domains: RalGDS, RGL, RGl2/Rlf and RGl3 [97,131]. Initial studies with NIH 3T3 fibroblasts suggested a minor role for RAL-GEFs during transformation. However, recent studies revealed a crucial role for RAL-GEFs in the transformation of human cells [132]. While activation of RAF alone was sufficient for RAS-mediated transformation of NIH 3T3 fibroblasts, activation of this pathway alone did not transform human BJ fibroblasts, MCF-10A human breast epithelial cells or RIE rat intestinal epithelial cells [132]. These data suggested that species and/or cell type specific differences for RAS induced transformation exist. These observations led Weinberg and colleagues to decipher the requirements for transformation of various human cell types. They demonstrated that immortalized HEK cells require the specific activation of RAL-GEFs and PI3K but not RAF for transformation in vitro and anchorage-independent growth [132]. These observations demonstrated the crucial role of RalGEFs in human cell transformation. Furthermore, White and colleagues provided evidence for the role of RAL-GEFs in tumor cell growth and cell survival. By applying an RNA interference-mediated loss of function approach, they demonstrated that RALA is required for the anchorage-independent proliferation of transformed cells, while RALB is required for the survival of the transformed but not normal cells [133]. RAL signaling has also been implicated in the regulation of endocytosis or exocytosis, actin organization, cell migration and gene expression [131,134–136].

#### 6.4. Tiam

T lymphoma invasion and metastasis protein 1 (Tiam 1) was identified in a retroviral insertional mutagenesis screen for genes conferring invasiveness to otherwise non-invasive murine T-lymphoma cells [137]. Furthermore, Tiam1 was characterized
to be a specific GEF for RAC with a RAS binding domain thereby linking the activation of RAS to RAC [138]. The role of Tiam in the RAS–RAC cross talk is further confirmed by the studies in *Tiam1*-deficient mice. *Tiam1*<sup>−/−</sup> mice were resistant to the development of RAS-induced skin tumors initiated with 7,12-dimethylbenzanthracene and the fibroblasts derived from them were resistant to foci formation by oncogenic RAS [139]. Tiam1 was also implied recently in neurotrophin induced Schwann cell migration [140,141]. In addition to RAS, Tiam1 is also involved in Rap-1 mediated cell spreading [142]. Membrane localization of Tiam1 is crucial for its ability to induce Rac-mediated membrane ruffles and activation of JNK. The intracellular localization and activation of Tiam1 is modulated by N-terminal myristoylation, phosphorylation and phosphoinositol binding (for review [143]).

### 6.5. RASSF

RASSF (RAS association domain family) proteins are a family of RAS effectors with a RA domain without catalytic function. RASSF family members have been shown to function as tumor suppressors as supported by the loss of expression of RASSF family members in a variety of human tumors [144,145]. RASSF1 was first identified as a candidate tumor suppressor gene situated on chromosome 3p21.3 and is frequently inactivated by DNA methylation [145]. Additionally, overexpression of RASSF proteins provoked anti-proliferative and pro-apoptotic responses. RASSF2 and RASSF4 specifically interact with KRAS to induce cell cycle arrest and induce apoptosis [146,147]. In addition, RASSF2 was identified as a novel tumor suppressor in colorectal tumors and its expression was silenced by DNA methylation [148]. The idea that epigenetic inactivation of RASSF2 plays a key role in KRAS mediated transformation is supported by the findings that KRAS/BRAF mutations were identified more frequently in colorectal tumors with methylation of RASSF2. In addition, methylation of RASSF2 was also detected in colorectal tumors without RAS/BRAF mutations suggesting silencing of RASSF as a crucial event of colorectal tumorigenesis [148]. A recent study from Neel and colleagues suggested a role of RASSF1A for death receptor induced BAX conformational changes leading to permeabilization of mitochondrial outer membranes and apoptosis [149]. This pro-apoptotic effect of RASSF1A is accomplished by a complex formation with a BH-3 like protein MAP-1. Interaction with RASSF1A promotes binding of MAP-1 with BAX leading the activation of BAX [149]. Ectopic expression of RASSF5 (also called NORE1) induced apoptosis in a RAS dependent manner and impaired the ability to grow in soft agar [146]. Khokhlatchev and colleagues demonstrated the involvement of RAS–Nore–Mst1 (mammalian ste20-like kinase) complexes in apoptotic processes [150].

### 6.6. RIN and PLC

RIN1 is a RAS effector with RA domain and it functions as a GEF for Rab5/Vps21 like proteins thereby facilitating RAS regulated endocytosis [151]. Three family members are known: RIN1, RIN2, and RIN3. Whereas RIN1 is primarily cytosolic, RIN2 and RIN3 are localized to endocytic vesicles.

PLC was originally identified as a RAS binding protein in a yeast two hybrid screen performed with *Let-60* (RAS encoding gene in *C. elegans*) as a bait [152]. Cloning of human and rat homologues of PLC210, designated PLCε, revealed its domain structure with a RAS GTPase binding domain. PLCε has been shown to be activated by growth factors like PDGF and EGF in a RAS and RAP1 dependent manner [153,154]. For instance, rapid activation of PLCε by PDGF is mediated by RAS, while sustained activation of PLCε is mediated by Rap1 [154]. PLCε<sup>−/−</sup> mice showed a delayed onset and markedly reduced incidence of carcinogen-induced skin squamous tumors. These data confirmed the crucial role of PLCε in the progression of carcinomas [155].

### 6.7. AF6 and PKCζ

The growing family of RAS effectors also include AF6 and PKCζ which are implied in various cellular processes such as cell adhesion and transcription, respectively [156–159]. AF6 (also called Afadin) was cloned as an interacting partner of *ALL-1* gene from leukemia’s [160], an actin binding protein with two RA domains in its N-terminus and a PDZ domain, which assists in interacting with tight junction protein ZO-1 [161]. Consequently, AF6 was found to colocalize with ZO-1 at the tight junctions and gene inactivation studies in mouse revealed a crucial role of AF6 in maintaining epithelial cell–cell junctions and cell polarity [156]. AF6 has also been shown to negatively regulate Rap1 mediated cell adhesion [157]. PKCζ is similar to RAF in structure and it was shown to be required during oocyte maturation in *Xenopus* [162]. Moscat and colleagues demonstrated direct interactions between RAS and PKCζ in vitro and in vivo [162]. The various roles of PKCζ have been comprehensively discussed in a recent review [163].

Putting together, there are a large number of RAS effectors that perform versatile roles in response to various stimuli. As mentioned before, activation of various receptors such as GPCRs, RTKs and integrins by different ligands leads to the activation of RAS, which can then transmit highly diverse signals by binding to several effectors. Many of the divergent signaling pathways initiated by RAS exhibit significant cross talk, which may finally converge at some level to confer a specific phenotype. For instance, the activation of RALGDS can lead to the activation of RAF under some conditions via PLD [164]. In case of growth factors, activation of RAS alone could suffice to bring about most if not all of the phenotypes in the recipient cells. For instance, RAS activation can lead to the activation of RAB5 via RIN1 leading to endocytosis, which is definitely required for EGFR signaling to ensue. Along the same line, by triggering the activation of RAS, EGF can also accomplish cytoskeletal changes and cell migration via RAC and AF6. Thus RAS serve as a crucial signaling hub for the convergence and divergence of signals by switching between the GDP and GTP bound forms.
K. Rajalingam et al. / Biochimica et Biophysica Acta 1773 (2007) 1177–1195

7. Targeting RAS pathways in human cancer

Due to its central role in intracellular signal transduction and malignant transformation, a plethora of drugs targeting RAS proteins or RAS effector pathways have been developed with the aim to either correct or eliminate aberrant RAS signaling. Early observations documenting that posttranslational modifications such as farnesylation, C-terminal peptide cleavage, and carboxymethylation are essential for oncogenic RAS mutants directed the initial development of farnesylation inhibitors targeting RAS. The first farnesyl transferase inhibitors (FTIs) mimicked the structure of the CAAX peptidic motif of RAS and/or the structure of the farnesyl substrate. Some FTIs initially showed promising pharmacological properties such as low ID50, pronounced tumor regression in animal models, high membrane permeability and low toxicity. For example, the FTI tipifarnib received FDA approval for the treatment of older patients with myelodysplastic syndrome, and Phase III trials suggest that tipifarnib might also be a drug of choice for treatment of older patients with acute myeloid leukemia [165]. Surprisingly, oncogenic RAS was not inhibited by FTIs and was geranylgeranylated by the related geranylgeranyl transferase I upon FTI exposure [166]. Thus although initially developed as RAS inhibitors with promising results in animal tumor models, FTIs seem to act via off-target inhibition of hitherto unidentified substrate(s).

The repair of defective GTPase activity of mutant RAS by GTP derivatives bearing residues required for GTP hydrolysis has been reported [167,168]. However, this avenue of RAS inhibition still awaits additional investigation to solve its major problems, such as specificity or transport of compounds modified by triphosphates through biological membranes.

The development of drugs inhibiting the interaction of RAS with its effectors [169], immunological approaches and mutant-specific siRNAs are also still in their infancies [165]. In contrast, a couple of novel approaches are trying to target aberrant RAS signaling by the inhibition of its downstream effector pathways. For example, inhibition of MEK by CI-1040 stopped the growth of some cell lines expressing Q61R NRAS [170]. Also the inhibition of MEK was surprisingly efficient for cell lines harboring the V600E B-RAF mutation [170]. However, another study demonstrated that certain tumor cell lines may develop resistance to CI-1040 by increasing endogenous KRAS expression [171]. It is expected that despite the inherent problems with RAS as a target, the ongoing development of RAS pathway inhibitors will lead to additional therapies in future, which in combination with a precise molecular classification of tumors and additional classical treatment modalities will add in to the ultimate goal of all tumor therapies, which is complete and irreversible remission.

8. Novel avenues of RAS research

8.1. Regulating RAS by microRNAs

[172]. MicroRNAs are small, non-coding RNAs modulating the expression of target mRNAs at a post-transcriptional level [173]. They are involved in the regulation of a wide variety of cellular processes. It was estimated that the number of human miRNAs is as high as 1000 [174] and that miRNAs may regulate approximately 30% of all genes [175]. One of the earliest reports linking miRNAs to cancer described that two miRNAs, miR-15 and miR-16, are frequently deleted in chronic lymphocytic lymphoma patients [176]. In the meantime, more publications have appeared and proven that either the deregulated expression or mutations of miRNAs are linked to cancer and that miRNAs may function either as tumor suppressors or oncogenes [177–179].

In 2005, it was discovered by in silico analysis that let-60, the *C. elegans* ortholog of the *RAS* oncogene, harbors let-7 target sites in its 3′-untranslated region (UTR) [172]. let-7 is a heterochronic switch gene, whose loss causes reiterations of larval cell fates in the adult, whereas overexpression of let-7 evokes premature expression of adult fates during larval stages [180]. It encodes a 22 nucleotide RNA negatively regulating the expression of protein-coding genes that contain regions of complementarity in their 3′-UTRs by target mRNA degradation [181]. Using various experimental strategies Slack went on and proved that the amount of let-60/RAS is regulated by let-7 family members in the context of vulva development and in human cell lines [172]. Since various let-7 family members have been mapped to chromosomal regions frequently deleted in lung tumors [182], and let-7 expression was reported being reduced in lung tumors in association with shortened post-operative survival [183], microarray analysis was performed on tumor and matched adjacent non-tumor tissue from cancer patients. Relative let-7 expression was reduced in 12 of 12 lung cancers, 4 of 6 colon cancers and 2 of 3 breast cancers and increased RAS protein levels were correlated with reduced let-7 expression [172]. let-7 is also part of a unique miRNA expression profile that may be applicable as diagnostic and prognostic marker for human lung cancer [184].

In summary, all data implicate that let-7 is a bona fide tumor suppressor – at least in lung tumors – and that its effect is mediated at least partially at the post-transcriptional level via down-regulation of RAS expression. It remains to be addressed (1) whether there are additional let-7 targets in lung tumorigenesis, (2) at which phase of tumor development and by which mechanism let7 is downregulated, and (3) what is the overall contribution of let7 to lung tumorigenesis?

8.2. Somatic and germline mutations in Human RAS genes

As mentioned above, RAS proteins possess multiple functions, which differ depending on factors such as species, genetic background, tissue or cell type and microenvironment. As diverse as the regular functions of RAS proteins are as diverse are their contributions to malignant phenotypic conversion. RAS family members are involved in the execution of almost any step in models of multi-step tumorigenesis [185] or, in other words, contribute to each hallmark of cancer [186]. It has been calculated that RAS is oncogenically activated in more

than 15% of human tumors and mutations have been found at residues 12, 13, 59 and 61, with positions 12 and 61 being the most common [35]. By August 2006 the frequency of somatic mutations in the main three RAS genes in the Catalogue of Somatic Mutations in Cancer [187] was: (1) HRAS: mutated in approximately 4% of samples analyzed with the highest frequency in salivary gland (20%), urinary tract (12%) and cervix (9%). (2) KRAS: mutated in approximately 21.7% of samples analyzed with the highest frequency in pancreas (59%), biliary tract (32%), large intestine (30%), small intestine (27%) and lung (19%). (3) NRAS: mutated in approximately 8.2% of samples analyzed with the highest frequency in skin (17%), nervous system (16%) and hematopoietic and lymphoid tissues (11%). Interestingly, a comparison of 12 activating amino acid changes in KRAS at codons 12 and 13 revealed that the site and the type of codon change vary significantly from tumor type to tumor type [188]. For example, G13D KRAS is found almost exclusively in soft tissue, stomach and thyroid tumors, whereas G12D and G12V KRAS are more ubiquitously detected [188]. Unraveling the molecular mechanisms underlying the genetic specificity and lineage dependency of the individual RAS mutations is still an ongoing challenge in the field.

Although somatic point mutations in human RAS genes are already known for many years, germline mutations in RAS as well as in other components of RAS signaling pathways have been detected only recently. These mutations have now been proposed to be a unifying theme for a set of rare, phenotypically overlapping developmental syndromes [189], which have been summarized under the term “Neuro-Cardio-Facial-Cutaneous” (NCDF) syndromes. These include Neurofibromatosis Type I (NF1), Noonan syndrome (NS), LEOPARD syndrome, Cardiofacio-cutaneous syndrome (CFC) and Costello syndrome (CS), for recent reviews [190,191]. In general, patients with these diseases are characterized by facial dysmorphisms, heart defects and short stature. In addition, skin and genital malformations, mental retardation and predisposition to certain malignancies have been described [189]. De novo germline mutations have been reported for example for KRAS in Noonan syndrome [192] and in CFC [193], and for HRAS in Costello Syndrome [194–197]. In addition, mutations of RAS effectors such as BRAF or MEK1/2 have also been detected in CFC [198].

The consequences of RAS germline mutations found in NCDF syndromes are less well characterized. For example, the KRAS germline mutations V14I, T58I, and D153V in NS and P34R, which are present in CFC patients, are rare in COSMIC. Recombinant V41I and T58I proteins displayed defective intrinsic GTP hydrolysis and impaired responsiveness to GTPase activating proteins and hematopoietic progenitors transfected with viruses encoding these mutants were hypersensitive to growth factors [192]. In contrast, the heterozygous germline HRAS missense mutations described in approximately 85% of CS patients are frequently found also in somatic tumors. Whereas G12S HRAS is with 75% the most frequent HRAS germline mutation, other mutations are represented at a lower frequency [194–197]. Interestingly, more than 50% of patients with the less frequent G12A mutation develop tumors including rhabdomyosarcoma, ganglioneuroblastoma and bladder carci-

noma, whereas the overall malignancy frequency in all CS patients with HRAS mutations is between 11 and 17% [197]. This indicates that the wild-type copy of HRAS, which is still present in most of the CS patients, might antagonize the function of mutated HRAS, which was already proposed in a different context [96], and that the wild-type copy might cope differently with germline mutations depending on the specific mutation and the cell type involved. As described for some germline KRAS mutations, HRAS mutations are hypersensitive to growth factors, as shown by cell proliferation assays with fibroblasts of selected CS patients [194].

In summary, the discovery of germline mutations in RAS and other genes involved in RAS signaling was a major breakthrough, but lots of open questions remain. (1) The number of analyzed NCDF patients and the number of identified individual germline mutations is still too low to draw far-reaching conclusions. For example, most of the studies described data from less than 100 patients. The same is true for the predicted predisposition for malignancies in certain NCDF syndromes. (2) The functional consequences of individual germline mutations have to be addressed. Do these mutants engage other signaling pathways than the respective wild-type proteins? Are their ways to establish mouse models reflecting the human phenotype? Why are these germline mutations primarily dominant and the phenotypic effects with respect to increased proliferation relatively mild? How do mutated RAS genes evoke the complex phenotype pattern in NCDF patients and what is the cause of the high variability in tissue-specific penetrance? (3) Since there are still NCDF patients with no germline mutation in the above mentioned genes, the mutational status of other players involved in RAS signaling including scaffold proteins needs to be analyzed in future.

### 8.3. Untangling the complexity of RAS function by large scale proteomic and genomic approaches

Very early on in the 40 years of research on RAS it became apparent that large-scale approaches are necessary to decipher the full complexity of RAS function. So far only a few proteomic approaches on RAS have been published. Cheng and colleagues investigated mechanisms of RAS-mediated transformation by proteomic profiling of G12V HRAS immortalized human ovarian epithelial cell lines [199]. Using peptide mass fingerprinting 16 out of 32 proteins found to be associated with RAS-mediated transformation had not been linked previously to RAS or RAS mediated transformation [199]. The protein targets were classified according to function to processes such as metabolism, redox balance, calcium signaling, apoptosis and protein methylation. One key finding of this study was that Caspase 4 maturation was blocked by G12V HRAS at the post-translational level, which might partially explain, why HRAS transformed cells evade apoptosis [199]. Earlier in 2006 the group of Bakh dissected the targets of mutated HRAS proteins in NIH 3T3 cells [200,201]. Either stable or tetracycline-inducible cell lines expressing G12V or G12R HRAS were generated, and their proteomes characterized by 2-DE, quantitative imaging and MALDI-TOF MS. Overall, 213

protein spots were changed in HRAS transformed cell lines, and upon further characterization finally 64 polypeptides were identified to be consistently deregulated by mutated HRAS (36 up-/28 down-regulated). Many HRAS targets identified were associated with the cytoskeleton or molecular chaperones [200]. Unfortunately, the results provided were presented as common changes in HRAS mutants and eminent questions, e.g. what are the main differences between different HRAS mutants, were not disclosed. Finally, the groups of Casal and Nebrada analyzed the effects of G12V RAS on proteins differentially expressed at cell membranes in either wild-type or p38α MAPK deficient mouse embryo fibroblasts by 2D-DIGE coupled to MALDI-TOF MS [202]. 42 differentially expressed proteins were identified and seven proteins previously linked to transformation were verified by Western blot analysis. Whereas e.g. prohibitin was down-regulated, the cytoplasmic tyrosine kinase FAK2 and specific proteins involved in glucose metabolism or mitochondrial respiration were up-regulated [202]. This publication demonstrated the feasibility of RAS proteome studies at the sub-cellular level. Overall, proteomic approaches seem to provide promising starting points with respect to RAS target discovery and subsequent functional investigations.

Large-scale genomic expression profiling approaches for the analysis of RAS function have been performed more frequently. Initially, subtractive suppression hybridization was used for genome-wide surveys of RAS transformation targets in rat fibroblasts [203], thyroid cells [204] or ovarian epithelial cells [205]. Common and distinct targets in cells transformed by G12D NRAS, G12V KRAS or G12V HRAS were identified and 61 putative RAS targets sensitive to MEK inhibition described [203]. Di Lauro and coworkers concentrated on the identification of 57 immediate early genes induced by activated HRAS in differentiated, epithelial thyroid cells [204]. In G12V KRAS transformed ovarian epithelial cells more than 200 genes with altered expression were identified, 79 targets being sensitive to inhibition of MEK/ERK or PI3K [205]. A whole range of studies has been using high-density microarrays of tumor cell lines or tumor xenograft tissues to detect oncogenic RAS-dependent transcriptional changes. For example, 584 upregulated genes have been identified in an in vitro model of human pancreatic duct epithelial carcinogenesis, which is dependent on G12V KRAS [206]. The transcriptional effects of G13D KRAS in combination with increased hepatocyte growth factor signaling were evaluated in a model of human colorectal cancer [207]. 139 differentially expressed genes were identified in human melanoma cell lines carrying the Q61R NRAS mutation [208]. Although these publications provide hundreds of targets for specific RAS mutations in various in vitro and in vivo tumor scenarios, further in-depth functional analysis and independent confirmation by standardized array technologies and post-array computational analysis is often missing.

Another approach using microarray methods is to detect expression signatures specific for certain tumors and/or RAS mutants. This might be informative in the future for the evaluation of clinical cancer outcome or patient management. For example, Tyler Jacks and colleagues identified a G12D

KRAS expression signature by cross-species gene expression analysis of mouse and human lung tumors [209]. This signature encompasses a set of 89 genes, which are indicative of KRAS mutations and provide a source of novel potential effectors of oncogenic KRAS activity in human cancer [209]. Also, oligonucleotide array analysis of 20 different samples isolated from human colorectal cancer patients with either BRAF or KRAS mutation revealed an expression pattern of 98 genes that is able to distinguish between BRAF and KRAS mutant groups [210]. Out of these signatures for various tumor types it will be important to identify those targets that are relevant for oncogene addiction, a phenomenon describing the acquired dependence of tumor cells on an activated oncogene for their survival and/or proliferation [116,211,212]. Analyses of this type may provide molecular targets for future therapeutic approaches by targeting mutation-specific events in tumorigenesis.

Eric Holland and colleagues also performed microarray-based assays to analyze how oncogenic RAS and AKT signaling contribute to glioblastoma [213]. It was demonstrated that the main effect of KRAS/AKT signaling is not at the level of transcription, but the differential recruitment of already transcribed specific mRNAs to polysomes. 426 individual mRNAs were detected to be regulated by RAS/AKT at the level of polysome loading within 2 h after blocking RAS/AKT signals. The contribution of translational control to transformation by activated KRAS was recently independently confirmed [214]. It needs now to be determined whether this translational control is necessary or sufficient to induce or maintain tumor formation [215]. In order to get a complete picture of RAS function it may be necessary to look in detail not only at the level of transcription and translation but also on additional aspects of RAS regulation at a global scale. For example, a genome-wide study on RAS-dependent miRNA expression profiles in various human tumors is so far missing.

### 8.4. RAS and tumor stem cells

Sophisticated mouse tumor models have been generated and have contributed significant insights into the isoform-specific actions of mutated RAS genes, for comprehensive reviews [1,216,217]. Due to species-specific differences and lack of full correspondence to the human phenotype these models have to be treated with caution. However, one major outcome of these studies was that the given cellular context is of special importance for tumor development and maintenance [218], and the hypothesis was raised that certain RAS-induced tumors developed from stem cells or stem cell-like progenitor cells [216]. For example, Tyler Jacks and colleagues isolated bronchioalveolar stem cells (BASCs) exhibiting self-renewal and multipotency in clonal assays [219]. Conditional expression of G12D KRAS in these BASCs in vitro or in vivo resulted in an expansion of BASC number and subsequent differentiation along the alveolar lineage. In naphthalene-treated mice expressing activated KRAS, an increase of size and number of lung tumors were noted. The authors suggested that the microenvironment conditions synergize with activated KRAS and claimed that they have found the target cell population, from which

human lung adenocarcinoma may arise. Also the targeting of activated RAS to different stem and progenitor cells of the skin resulting in different types of benign and malignant tumors suggested that the nature of the cell in which tumor initiation occurs and its microenvironment are main determinants of the malignant potential (discussed in [220]). In this context it is also noteworthy that mouse embryonic stem cells express a RAS-like gene called ERAS, which is important for the tumor-like properties of these cells [6] and that expression of activated MRAS in hematopoietic stem cells initiated leukemogenic transformation, immortalization and preferential generation of mast cells [221]. Future studies have to address whether other stem cell populations are also prone to RAS transformation and by which mechanisms RAS-dependent tumorigenesis is initiated. It is also conceivable that therapeutic approaches will be developed that aim to suppress RAS-dependent tumor formation by influencing the respective microenvironment.

## 9. Concluding remarks

The first cellular RAS oncogene was cloned in 1982 and recently the Beatson Institute for Cancer Research in Glasgow, UK has successfully organized a wonderful meeting on RAS signaling and cancer to commemorate 24 years of intense RAS research. There is an old saying that “familiarity breeds contempt”, which is however not true in the context of RAS research. The field has grown tremendously in the past years with more than 150 family members and as many as 10 effector pathways have been identified so far for RAS alone. The functional interplay of many of these effector pathways needs to be worked out in future in more detail. Apart from the effector cascades identified, the RAF/MEK/ERK pathway as well as the PI3K pathway still happened to be the most valuable targets for cancer therapy. The recent identification of RalGEFs as crucial regulators of human cell transformation has opened up a new window in the search for new targets for therapeutic intervention. The use of large-scale RNA interference libraries shows considerable promise in quickly uncovering the crucial components of RAS induced signaling cascades. The recent discovery that RAS oncogene expression is modulated by microRNAs has added a new facet to RAS research. In the days to come, microRNAs regulating other downstream targets of the RAS signaling cascades will be deciphered as well. There is no doubt that these studies will keep up the momentum and excitement of this ever growing field of RAS biology, thereby retaining RAS at the heart of cancer research for the next decades. Finally it is befitting to quote the concluding lines from Robert Frost’s famous poem:

> The woods are lovely, dark, and deep,  
> But I have promises to keep,  
> And miles to go before I sleep,  
> And miles to go before I sleep.

## Acknowledgements

Authors are supported by SFBs 487 and 581, GRK 1142 and Transregio 17.

---

### References

[1] M. Malumbres, M. Barbacid, RAS oncogenes: the first 30 years, Nat. Rev., Cancer 3 (2003) 459–465.

[2] J. Colicelli, Human RAS superfamily proteins and related GTPases, Sci. STKE 2004 (2004) RE13.

[3] K. Wennerberg, K.L. Rossman, C.J. Der, The Ras superfamily at a glance, J. Cell. Sci. 118 (2005) 843–846.

[4] B.M. Marte, P. Rodriguez-Viciana, S. Wennstrom, P.H. Warne, J. Downward, R-Ras can activate the phosphoinositide 3-kinase but not the MAP kinase arm of the Ras effector pathways, Curr. Biol. 7 (1997) 63–70.

[5] P. Rodriguez-Viciana, J. Oses-Prieto, A. Burlingame, M. Fried, F. McCormick, A phosphatase holoenzyme comprised of Shoc2/Sur8 and the catalytic subunit of PP1 functions as an M-Ras effector to modulate Raf activity, Mol. Cell 22 (2006) 217–230.

[6] K. Takahashi, K. Mitsui, S. Yamanaka, Role of ERas in promoting tumour-like properties in mouse embryonic stem cells, Nature 423 (2003) 541–545.

[7] A. Wittinghofer, N. Nassar, How Ras-related proteins talk to their effectors, Trends Biochem. Sci. 21 (1996) 488–491.

[8] R.S. Goody, M. Frech, A. Wittinghofer, Affinity of guanine nucleotide binding proteins for their ligands: facts and artefacts, Trends Biochem. Sci. 16 (1991) 327–328.

[9] J. John, I. Schlichting, E. Schiltz, P. Rosch, A. Wittinghofer, C-terminal truncation of p21H preserves crucial kinetic and structural properties, J. Biol. Chem. 264 (1989) 13086–13092.

[10] P. Gideon, J. John, M. Frech, A. Lautwein, R. Clark, J.E. Scheffler, A. Wittinghofer, Mutational and kinetic analyses of the GTPase-activating protein (GAP)-p21 interaction: the C-terminal domain of GAP is not sufficient for full activity, Mol. Cell. Biol. 12 (1992) 2050–2056.

[11] M. Barbacid, Ras genes, Annu. Rev. Biochem. 56 (1987) 779–827.

[12] K. Scheffzek, M.R. Ahmadian, W. Kabsch, L. Wiesmuller, A. Lautwein, F. Schmitz, A. Wittinghofer, The Ras–RasGAP complex: structural basis for GTPase activation and its loss in oncogenic Ras mutants, Science 277 (1997) 333–338.

[13] D.L. Graham, J.F. Eccleston, P.N. Lowe, The conserved arginine in rho-GTPase-activating protein is essential for efficient catalysis but not for complex formation with Rho.GDP and aluminum fluoride, Biochemistry 38 (1999) 985–991.

[14] O. Daumke, M. Weyand, P.P. Chakrabarti, I.R. Vetter, A. Wittinghofer, The GTPase-activating protein Rap1GAP uses a catalytic asparagine, Nature 429 (2004) 197–201.

[15] A. De Antoni, J. Schmitzova, H.H. Trepte, D. Gallwitz, S. Albert, Significance of GTP hydrolysis in Ypt1p-regulated endoplasmic reticulum to Golgi transport revealed by the analysis of two novel Ypt1-GAPs, J. Biol. Chem. 277 (2002) 41023–41031.

[16] A. Wittinghofer, Phosphoryl transfer in Ras proteins, conclusive or elusive? Trends Biochem. Sci. 31 (2006) 20–23.

[17] G. Jekely, Small GTPases and the evolution of the eukaryotic cell, BioEssays 25 (2003) 1129–1138.

[18] D. Bar-Sagi, A. Hall, Ras and Rho GTPases: a family reunion, Cell 103 (2000) 227–238.

[19] S.Y. Jiang, S. Ramachandran, Comparative and evolutionary analysis of genes encoding small GTPases and their activating proteins in eukaryotic genomes, Physiol. Genomics 24 (2006) 235–251.

[20] A.H. Valster, P.K. Hepler, J. Chernoff, Plant GTPases: the Rhos in bloom, Trends Cell Biol. 10 (2000) 141–146.

[21] V. Vernoud, A.C. Horton, Z. Yang, E. Nielsen, Analysis of the small GTPase gene superfamily of Arabidopsis, Plant Physiol. 131 (2003) 1191–1208.

[22] D. Broek, N. Samiy, O. Fasano, A. Fujiyama, F. Tamanoi, J. Northup, M. Wigler, Differential activation of yeast adenylate cyclase by wild-type and mutant RAS proteins, Cell 41 (1985) 763–769.

[23] B. Onken, H. Wiener, M.R. Philips, E.C. Chang, Compartmentalized signaling of Ras in fission yeast, Proc. Natl. Acad. Sci. U. S. A. 103 (2006) 9045–9050.

[24] J.R. Chubb, R.H. Insall, Dictyostelium: an ideal organism for genetic dissection of Ras signalling networks, Biochim. Biophys. Acta 1525 (2001) 262–271.

[25] J.M. Goldberg, G. Manning, A. Liu, P. Fey, K.E. Pilcher, Y. Xu, J.L. Smith, The dictyostelium kinome—Analysis of the protein kinases from a simple model organism, PLoS Genet. 2 (2006) e38.

[26] M.C. Field, Signalling the genome: the Ras-like small GTPase family of trypanosomatids, Trends Parasitol. 21 (2005) 447–450.

[27] A. Champion, M. Kreis, K. Mockaitis, A. Picaud, Y. Henry, Arabidopsis kinome: after the casting, Funct. Integr. Genomics 4 (2004) 163–187.

[28] C. Chang, The ethylene signal transduction pathway in Arabidopsis: an emerging paradigm? Trends Biochem. Sci. 21 (1996) 129–133.

[29] J. Leon, I. Guerrero, A. Pellicer, Differential expression of the ras gene family in mice, Mol. Cell. Biol. 7 (1987) 1535–1540.

[30] R. Muller, D.J. Slamon, E.D. Adamson, J.M. Tremblay, D. Muller, M.J. Cline, I.M. Verma, Transcription of c-onc genes c-rasKi and c-fms during mouse development, Mol. Cell. Biol. 3 (1983) 1062–1069.

[31] L.M. Esteban, C. Vicario-Abejon, P. Fernandez-Salguero, A. Fernandez-Medarde, N. Swaminathan, K. Yienger, E. Lopez, M. Malumbres, R. McKay, J.M. Ward, A. Pellicer, E. Santos, Targeted genomic disruption of H-ras and N-ras, individually or in combination, reveals the dispensability of both loci for mouse growth and development, Mol. Cell. Biol. 21 (2001) 1444–1452.

[32] L. Johnson, D. Greenbaum, K. Cichowski, K. Mercer, E. Murphy, E. Schmitt, R.T. Bronson, H. Umanoff, W. Edelmann, R. Kucherlapati, T. Jacks, K-ras is an essential gene in the mouse with partial functional overlap with N-ras, Genes Dev. 11 (1997) 2468–2481.

[33] K. Koera, K. Nakamura, K. Nakao, J. Miyoshi, K. Toyoshima, T. Hatta, H. Otani, A. Aiba, M. Katsuki, K-ras is essential for the development of the mouse embryo, Oncogene 15 (1997) 1151–1159.

[34] H. Umanoff, W. Edelmann, A. Pellicer, R. Kucherlapati, The murine N-ras gene is not essential for growth and development, Proc. Natl. Acad. Sci. U. S. A. 92 (1995) 1709–1713.

[35] J.L. Bos, ras oncogenes in human cancer: a review, Cancer Res. 49 (1989) 4682–4689.

[36] J.F. Hancock, R.G. Parton, Ras plasma membrane signalling platforms, Biochem. J. 389 (2005) 1–11.

[37] J.F. Hancock, H. Paterson, C.J. Marshall, A polybasic domain or palmitoylation is required in addition to the CAAX motif to localize p21ras to the plasma membrane, Cell 63 (1990) 133–139.

[38] S. Lobo, W.K. Greentree, M.E. Linder, R.J. Deschenes, Identification of a Ras palmitoyltransferase in Saccharomyces cerevisiae, J. Biol. Chem. 277 (2002) 41268–41273.

[39] S. Mukherjee, F.R. Maxfield, Membrane domains, Annu. Rev. Cell Dev. Biol. 20 (2004) 839–866.

[40] J.F. Hancock, A.I. Magee, J.E. Childs, C.J. Marshall, All ras proteins are polyisoprenylated but only some are palmitoylated, Cell 57 (1989) 1167–1177.

[41] E. Choy, V.K. Chiu, J. Silletti, M. Feoktistov, T. Morimoto, D. Michaelson, I.E. Ivanov, M.R. Philips, Endomembrane trafficking of ras: the CAAX motif targets proteins to the ER and Golgi, Cell 98 (1999) 69–80.

[42] Z. Chen, J.C. Otto, M.O. Bergo, S.G. Young, P.J. Casey, The C-terminal polylysine region and methylation of K-Ras are critical for the interaction between K-Ras and microtubules, J. Biol. Chem. 275 (2000) 41251–41257.

[43] J.A. Thissen, J.M. Gross, K. Subramanian, T. Meyer, P.J. Casey, Prenylation-dependent association of Ki-Ras with microtubules. Evidence for a role in subcellular trafficking, J. Biol. Chem. 272 (1997) 30362–30370.

[44] N. Jura, E. Scotto-Lavino, A. Sobczyk, D. Bar-Sagi, Differential modification of Ras proteins by ubiquitination, Mol. Cell 21 (2006) 679–687.

[45] P. Liu, Y. Ying, R.G.W. Anderson, Platelet-derived growth factor activates mitogen-activated protein kinase in isolated caveolae, Proc. Natl. Acad. Sci. U. S. A. 94 (1997) 13666–13670.

[46] I.A. Prior, A. Harding, J. Yan, J. Sluimer, R.G. Parton, J.F. Hancock, GTP-dependent segregation of H-ras from lipid rafts is required for biological activity, Nat. Cell Biol. 3 (2001) 368–375.

[47] S. Roy, S. Plowman, B. Rotblat, I.A. Prior, C. Muncke, S. Grainger, R.G. Parton, Y.I. Henis, Y. Kloog, J.F. Hancock, Individual palmitoyl residues serve distinct roles in H-ras trafficking, microlocalization, and signaling, Mol. Cell. Biol. 25 (2005) 6722–6733.

[48] M. Jaumot, J. Yan, J. Clyde-Smith, J. Sluimer, J.F. Hancock, The linker domain of the Ha-Ras hypervariable region regulates interactions with exchange factors, Raf-1 and phosphoinositide 3-kinase, J. Biol. Chem. 277 (2002) 272–278.

[49] A. Paz, R. Haklai, G. Elad-Sfadia, E. Ballan, Y. Kloog, Galectin-1 binds oncogenic H-Ras to mediate Ras membrane anchorage and cell transformation, Oncogene 20 (2001) 7486–7493.

[50] G. Elad-Sfadia, R. Haklai, E. Ballan, H.J. Gabius, Y. Kloog, Galectin-1 augments Ras activation and diverts Ras signals to Raf-1 at the expense of phosphoinositide 3-kinase, J. Biol. Chem. 277 (2002) 37169–37175.

[51] G. Elad-Sfadia, R. Haklai, E. Balan, Y. Kloog, Galectin-3 augments K-Ras activation and triggers a Ras signal that attenuates ERK but not phosphoinositide 3-kinase activity, J. Biol. Chem. 279 (2004) 34922–34930.

[52] V. Nancy, I. Callebaut, A. El Marjou, J. de Gunzburg, The delta subunit of retinal rod cGMP phosphodiesterase regulates the membrane association of Ras and Rap GTPases, J. Biol. Chem. 277 (2002) 15076–15084.

[53] Y. Wu, M. Han, Suppression of activated Let-60 ras protein defines a role of Caenorhabditis elegans Sur-1 MAP kinase in vulval differentiation, Genes Dev. 8 (1994) 147–159.

[54] G.M. Di Guglielmo, P.C. Baass, W.J. Ou, B.I. Posner, J.J. Bergeron, Compartmentalization of SHC, GRB2 and mSOS, and hyperphosphorylation of Raf-1 by EGF but not insulin in liver parenchyma, EMBO J. 13 (1994) 4269–4277.

[55] Y. Daaka, L.M. Luttrell, S. Ahn, G.J. Della Rocca, S.S. Ferguson, M.G. Caron, R.J. Lefkowitz, Essential role for G protein-coupled receptor endocytosis in the activation of mitogen-activated protein kinase, J. Biol. Chem. 273 (1998) 685–688.

[56] Y. Zhang, D.B. Moheban, B.R. Conway, A. Bhattacharyya, R.A. Segal, Cell surface Trk receptors mediate NGF-induced survival while internalized receptors regulate NGF-induced differentiation, J. Neurosci. 20 (2000) 5671–5678.

[57] V.K. Chiu, T. Bivona, A. Hach, J.B. Sajous, J. Silletti, H. Wiener, R.L. Johnson, A.D. Cox, M.R. Philips, Ras signalling on the endoplasmic reticulum and the Golgi, Nat. Cell Biol. 4 (2002) 343–350.

[58] A. Mor, M.R. Philips, Compartmentalized Ras/MAPK signaling, Annu. Rev. Immunol. 24 (2006) 771–800.

[59] A. Robubi, T. Mueller, J. Fueller, M. Hekman, U.R. Rapp, T. Dandekar, B-Raf and C-Raf signaling investigated in a simplified model of the mitogenic kinase cascade, Biol. Chem. 386 (2005) 1165–1171.

[60] V. Wixler, U. Smola, M. Schuler, U. Rapp, Differential regulation of Raf isoforms by growth versus differentiation inducing factors in PC12 pheochromocytoma cells, FEBS Lett. 385 (1996) 131–137.

[61] R.D. York, H. Yao, T. Dillon, C.L. Ellig, S.P. Eckert, E.W. McCleskey, P.J.S. Stork, Rap1 mediates sustained MAP kinase activation induced by nerve growth factor, Nature 392 (1998) 622–626.

[62] S. Sasagawa, Y.i. Ozaki, K. Fujita, S. Kuroda, Prediction and validation of the distinct dynamics of transient and sustained ERK activation, Nat. Cell Biol. 7 (2005) 365–373.

[63] J. Downward, The GRB2/Sem-5 adaptor protein, FEBS Lett. 338 (1994) 113–117.

[64] H. Avraham, S.Y. Park, K. Schinkmann, S. Avraham, RAFTK/Pyk2-mediated cellular signalling, Cell. Signal. 12 (2000) 123–133.

[65] H. Sondermann, S.M. Soisson, S. Boykevisch, S.S. Yang, D. Bar-Sagi, J. Kuriyan, Structural analysis of autoinhibition in the Ras activator Son of sevenless, Cell 119 (2004) 393–405.

[66] S.M. Margarit, H. Sondermann, B.E. Hall, B. Nagar, A. Hoelz, M. Pirruccello, D. Bar-Sagi, J. Kuriyan, Structural evidence for feedback activation by Ras.GTP of the Ras-specific nucleotide exchange factor SOS, Cell 112 (2003) 685–695.

[67] N. Mitin, K.L. Rossman, C.J. Der, Signaling interplay in Ras superfamily function, Curr. Biol. 15 (2005) R563–R574.

[68] G. Krapivinsky, L. Krapivinsky, Y. Manasian, A. Ivanov, R. Tyzio, C. Pellegrino, Y. Ben-Ari, D.E. Clapham, I. Medina, The NMDA receptor is

coupled to the ERK pathway by a direct interaction between NR2B and RasGRF1, Neuron 40 (2003) 775–784.

[69] X. Tian, T. Gotoh, K. Tsuji, E.H. Lo, S. Huang, L.A. Feig, Developmentally regulated role for Ras-GRFs in coupling NMDA glutamate receptors to Ras, Erk and CREB, EMBO J. 23 (2004) 1567–1575.

[70] C.E. Tognon, H.E. Kirk, L.A. Passmore, I.P. Whitehead, C.J. Der, R.J. Kay, Regulation of RasGRP via a phorbol ester-responsive C1 domain, Mol. Cell. Biol. 18 (1998) 6995–7008.

[71] J.O. Ebinu, D.A. Bottorff, E.Y. Chan, S.L. Stang, R.J. Dunn, J.C. Stone, RasGRP, a Ras guanyl nucleotide-releasing protein with calcium- and diacylglycerol-binding motifs, Science 280 (1998) 1082–1086.

[72] M.J. Caloca, J.L. Zugaza, X.R. Bustelo, Exchange factors of the RasGRP family mediate Ras activation in the Golgi, J. Biol. Chem. 278 (2003) 33465–33473.

[73] J. Clyde-Smith, G. Silins, M. Gartside, S. Grimmond, M. Etheridge, A. Apolloni, N. Hayward, J.F. Hancock, Characterization of RasGRP2, a plasma membrane-targeted, dual specificity Ras/Rap exchange factor, J. Biol. Chem. 275 (2000) 32260–32267.

[74] N.A. Dower, S.L. Stang, D.A. Bottorff, J.O. Ebinu, P. Dickie, H.L. Ostergaard, J.C. Stone, RasGRP is essential for mouse thymocyte differentiation and TCR signaling, Nat. Immunol. 1 (2000) 317–321.

[75] J.R. Crittenden, W. Bergmeier, Y. Zhang, C.L. Piffath, Y. Liang, D.D. Wagner, D.E. Housman, A.M. Graybiel, CalDAG–GEFI integrates signaling for platelet aggregation and thrombus formation, Nat. Med. 10 (2004) 982–986.

[76] H. Adari, D.R. Lowy, B.M. Willumsen, C.J. Der, F. McCormick, Guanosine triphosphatase activating protein(GAP) interacts with the p 21 ras effector binding domain, Science 240 (1988) 518–521.

[77] D.R. Kaplan, D.K. Morrison, G. Wong, F. McCormick, L.T. Williams, PDGF beta-receptor stimulates tyrosine phosphorylation of GAP and association of GAP with a signaling complex, Cell 61 (1990) 125–133.

[78] A. Kazlauskas, C. Ellis, T. Pawson, J.A. Cooper, Binding of GAP to activated PDGF receptors, Science 247 (1990) 1578–1581.

[79] J.Y. Yang, D. Michod, J. Walicki, B.M. Murphy, S. Kasibhatla, S.J. Martin, C. Widmann, Partial cleavage of RasGAP by caspases is required for cell survival in mild stress conditions, Mol. Cell. Biol. 24 (2004) 10425–10436.

[80] K. Cichowski, S. Santiago, M. Jardim, B.W. Johnson, T. Jacks, Dynamic regulation of the Ras pathway via proteolysis of the NF1 tumor suppressor, Genes Dev. 17 (2003) 449–454.

[81] C.M. Johannessen, E.E. Reczek, M.F. James, H. Brems, E. Legius, K. Cichowski, The NF1 tumor suppressor critically regulates TSC2 and mTOR, Proc. Natl. Acad. Sci. U. S. A. 102 (2005) 8573–8578.

[82] P.J. Lockyer, S. Kupzig, P.J. Cullen, CAPRI regulates Ca 2+-dependent inactivation of the Ras-MAPK pathway, Curr. Biol. 11 (2001) 981–986.

[83] M. Allen, S. Chu, S. Brill, C. Stotler, A. Buckler, Restricted tissue expression pattern of a novel human rasGAP-related gene and its murine ortholog, Gene 218 (1998) 17–25.

[84] S.A. Walker, S. Kupzig, D. Bouyoucef, L.C. Davies, T. Tsuboi, T.G. Bivona, G.E. Cozier, P.J. Lockyer, A. Buckler, G.A. Rutter, Identification of a Ras GTPase-activating protein regulated by receptor-mediated Ca 2+ oscillations, EMBO J. 23 (2004) 1749–1760.

[85] P.J. Cullen, J.J. Hsuan, O. Truong, A.J. Letcher, T.R. Jackson, A.P. Dawson, R.F. Irvine, Identification of a specific Ins(1,3,4,5)P4-binding protein as a member of the GAP1 family, Nature 376 (1995) 527–530.

[86] P.J. Lockyer, S. Wennstrom, S. Kupzig, K. Venkateswarlu, J. Downward, P.J. Cullen, Identification of the ras GTPase-activating protein GAP1(m) as a phosphatidylinositol-3,4,5-trisphosphate-binding protein in vivo, Curr. Biol. 9 (1999) 265–268.

[87] J.H. Kim, D. Liao, L.F. Lau, R.L. Huganir, SynGAP: a synaptic RasGAP that associates with the PSD-95/SAP90 protein family, Neuron 20 (1998) 683–691.

[88] G. Rumbaugh, J.P. Adams, J.H. Kim, R.L. Huganir, SynGAP regulates synaptic strength and mitogen-activated protein kinases in cultured neurons, Proc. Natl. Acad. Sci. U. S. A. 103 (2006) 4344–4351.

[89] H.A. Armelin, M.C. Armelin, K. Kelly, T. Stewart, P. Leder, B.H. Cochran, C.D. Stiles, Functional role for c-myc in mitogenic response to platelet-derived growth factor, Nature 310 (1984) 655–660.

[90] E. Blasi, B.J. Mathieson, L. Varesio, J.L. Cleveland, P.A. Borchert, U.R. Rapp, Selective immortalization of murine macrophages from fresh bone marrow by a raf/myc recombinant murine retrovirus, Nature 318 (1985) 667–670.

[91] U.R. Rapp, J.L. Cleveland, K. Brightman, A. Scott, J.N. Ihle, Abrogation of IL-3 and IL-2 dependence by recombinant murine retroviruses expressing v-myc oncogenes, Nature 317 (1985) 434–438.

[92] U.R. Rapp, J.L. Cleveland, S.M. Storm, T.W. Beck, M. Huleihel, Transformation by raf and myc oncogenes, Princess Takamatsu Symp. 17 (1986) 55–74.

[93] U.R. Rapp, J.L. Cleveland, T.I. Bonner, S.M. Storm, The Raf oncogene family, in: T. Curran, E.P. Reddy, A. Skalka (Eds.), The Oncogene Handbook, The Elsevier Science Publishers, The Netherlands, 1988, pp. 213–253.

[94] J.M. Kyriakis, H. App, X.F. Zhang, P. Banerjee, D.L. Brautigan, U.R. Rapp, J. Avruch, Raf-1 activates MAP kinase–kinase, Nature 358 (1992) 417–421.

[95] F. Romero, Aiolos transcription factor controls cell death in Tcells by regulating Bcl-2 expression and its cellular localization, EMBO J. 18 (1999) 3419–3430.

[96] A. Singh, A.P. Sowjanya, G. Ramakrishna, The wild-type Ras: road ahead, FASEB J. 19 (2005) 161–169.

[97] G.A. Repasky, E.J. Chenette, C.J. Der, Renewing the conspiracy theory debate: does Raf function alone to mediate Ras oncogenesis? Trends Cell Biol. 14 (2004) 639–647.

[98] C. Herrmann, Ras–effector interactions: after one decade, Curr. Opin. Struct. Biol. 13 (2003) 122–129.

[99] U.R. Rapp, M.D. Goldsborough, G.E. Mark, T.I. Bonner, J. Groffen, F.H. Reynolds Jr., J.R. Stephenson, Structure and biological activity of v-raf, a unique oncogene transduced by a retrovirus, Proc. Natl. Acad. Sci. U. S. A. 80 (1983) 4218–4222.

[100] H.W. Jansen, T. Patschinsky, K. Bister, Avian oncovirus MH2: molecular cloning of proviral DNA and structural analysis of viral RNA and protein, J. Virol. 48 (1983) 61–73.

[101] H.W. Jansen, R. Lurz, K. Bister, T.I. Bonner, G.E. Mark, U.R. Rapp, Homologous cell-derived oncogenes in avian carcinoma virus MH2 and murine sarcoma virus 3611, Nature 307 (1984) 281–284.

[102] T. Kamata, J.R. Feramisco, Epidermal growth factor stimulates guanine nucleotide binding activity and phosphorylation of ras oncogene proteins, Nature 310 (1984) 147–150.

[103] M.R. Smith, S.J. DeGudicibus, D.W. Stacey, Requirement for c-ras proteins during viral oncogene transformation, Nature 320 (1986) 540–543.

[104] U.R. Rapp, J.T. Bruder, J. Troppmair, Role of the Raf signal transduction pathway in Fos/Jun regulation and determination of cell fates, in: P.E. Angel, P.A. Herrlich (Eds.), The FOS and JUN Families of Transcription Factors, CRC Press, Boca Raton, 1994, pp. 221–247.

[105] W. Kolch, G. Heidecker, P. Lloyd, U.R. Rapp, Raf-1 protein kinase is required for growth of induced NIH/3T3 cells, Nature 349 (1991) 426–428.

[106] J.T. Bruder, G. Heidecker, U.R. Rapp, Serum-, TPA-, and Ras-induced expression from Ap-1/Ets-driven promoters requires Raf-1 kinase, Genes Dev. 6 (1992) 545–556.

[107] J. Troppmair, J.T. Bruder, H. App, H. Cai, L. Liptak, J. Szeberenyi, G.M. Cooper, U.R. Rapp, Ras controls coupling of growth factor receptors and protein kinase C in the membrane to Raf-1 and B-Raf protein serine kinases in the cytosol, Oncogene 7 (1992) 1867–1873.

[108] W. Kolch, G. Heidecker, G. Kochs, R. Hummel, H. Vahidi, H. Mischak, G. Finkenzeller, D. Marme, U.R. Rapp, Protein kinase C alpha activates RAF-1 by direct phosphorylation, Nature 364 (1993) 249–252.

[109] H.R. Bourne, D.A. Sanders, F. McCormick, The GTPase superfamily: a conserved switch for diverse cell functions, Nature 348 (1990) 125–132.

[110] H.R. Bourne, D.A. Sanders, F. McCormick, The GTPase superfamily: conserved structure and molecular mechanism, Nature 349 (1991) 117–127.

[111] X.F. Zhang, J. Settleman, J.M. Kyriakis, E. Takeuchi-Suzuki, S.J. Elledge, M.S. Marshall, J.T. Bruder, U.R. Rapp, J. Avruch, Normal and

oncogenic p21ras proteins bind to the amino-terminal regulatory domain of c-Raf-1, Nature 364 (1993) 308–313.

[112] M. Hekman, H. Hamm, A.V. Villar, B. Bader, J. Kuhlmann, J. Nickel, U.R. Rapp, Associations of B- and C-Raf with cholesterol, phosphatidylserine, and lipid second messengers: preferential binding of Raf to artificial lipid rafts, J. Biol. Chem. 277 (2002) 24090–24102.

[113] S.A. Moodie, B.M. Willumsen, M.J. Weber, A. Wolfman, Complexes of Ras.GTP with Raf-1 and mitogen-activated protein kinase kinase, Science 260 (1993) 1658–1661.

[114] A.B. Vojtek, S.M. Hollenberg, J.A. Cooper, Mammalian Ras interacts directly with the serine/threonine kinase Raf, Cell 74 (1993) 205–214.

[115] P.H. Warne, P.R. Viciana, J. Downward, Direct interaction of Ras and the amino-terminal region of Raf-1 in vitro, Nature 364 (1993) 352–355.

[116] U.R. Rapp, R. Gotz, S. Albert, BuCy RAFs drive cells into MEK addiction, Cancer Cell 9 (2006) 9–12.

[117] J.K. Voice, R.L. Klemke, A. Le, J.H. Jackson, Four human Ras homologs differ in their abilities to activate Raf-1, induce transformation, and stimulate cell motility, J. Biol. Chem. 274 (1999) 17164–17170.

[118] J.F. Hancock, Ras proteins: different signals from different locations, Nat. Rev., Mol. Cell Biol. 4 (2003) 373–384.

[119] M. Hekman, S. Wiese, R. Metz, S. Albert, J. Troppmair, J. Nickel, M. Sendtner, U.R. Rapp, Dynamic changes in C-Raf phosphorylation and 14-3-3 protein binding in response to growth factor stimulation: differential roles of 14-3-3 protein binding sites, J. Biol. Chem. 279 (2004) 14074–14086.

[120] K. Rajalingam, C. Wunder, V. Brinkmann, Y. Churin, M. Hekman, C. Sievers, U.R. Rapp, T. Rudel, Prohibitin is required for Ras-induced Raf-MEK-ERK activation and epithelial cell migration, Nat. Cell Biol. 7 (2005) 837–843.

[121] K. Rajalingam, T. Rudel, Ras-Raf signaling needs prohibitin, Cell Cycle 4 (2005) 1503–1505.

[122] R. Schreck, U.R. Rapp, Raf kinases: oncogenesis and drug discovery, Int. J. Cancer 119 (2006) 2261–2271.

[123] J.M. Kyriakis, The integration of signaling by multiprotein complexes containing Raf kinases, Biochim. Biophys. Acta 1773 (2007) 1238–1247, doi:10.1016/j.bbamcr.2006.11.003.

[124] P. Rodriguez-Viciana, P.H. Warne, R. Dhand, B. Vanhaesebroeck, I. Gout, M.J. Fry, M.D. Waterfield, J. Downward, Phosphatidylinositol-3-OH kinase as a direct target of Ras, Nature 370 (1994) 527–532.

[125] A. Sjolander, K. Yamamoto, B.E. Huber, E.G. Lapetina, Association of p21ras with phosphatidylinositol 3-kinase, Proc. Natl. Acad. Sci. U. S. A. 88 (1991) 7908–7912.

[126] J. Yan, S. Roy, A. Apolloni, A. Lane, J.F. Hancock, Ras isoforms vary in their ability to activate Raf-1 and phosphoinositide 3-kinase, J. Biol. Chem. 273 (1998) 24052–24056.

[127] M. Cully, H. You, A.J. Levine, T.W. Mak, Beyond PTEN mutations: the PI3K pathway as an integrator of multiple inputs during tumorigenesis, Nat. Rev., Cancer 6 (2006) 184–192.

[128] M.P. Scheid, J.R. Woodgett, Phosphatidylinositol 3′ kinase signaling in mammary tumorigenesis, J. Mammary Gland Biol. Neoplasia 6 (2001) 83–99.

[129] M.P. Scheid, J.R. Woodgett, PKB/AKT: functional insights from genetic models, Nat. Rev., Mol. Cell Biol. 2 (2001) 760–768.

[130] J. Luo, B.D. Manning, L.C. Cantley, Targeting the PI3K-Akt pathway in human cancer: rationale and promise, Cancer Cell 4 (2003) 257–262.

[131] E. Feinstein, Ral-GTPases: good chances for a long-lasting fame, Oncogene 24 (2005) 326–328.

[132] A. Rangarajan, S.J. Hong, A. Gifford, R.A. Weinberg, Species- and cell type-specific requirements for cellular transformation, Cancer Cell 6 (2004) 171–183.

[133] Y. Chien, M.A. White, RAL GTPases are linchpin modulators of human tumour-cell proliferation and survival, EMBO Rep. 4 (2003) 800–806.

[134] C. Rosse, A. Hatzoglou, M.C. Parrini, M.A. White, P. Chavrier, J. Camonis, RalB mobilizes the exocyst to drive cell migration, Mol. Cell. Biol. 26 (2006) 727.

[135] J.H. Camonis, M.A. White, Ral GTPases: corrupting the exocyst in cancer cells, Trends Cell Biol. 15 (2005) 327–332.

[136] L.A. Feig, Ral-GTPases: approaching their 15 minutes of fame, Trends Cell Biol. 13 (2003) 419–425.

[137] G.G. Habets, E.H. Scholtes, D. Zuydgeest, R.A. van der Kammen, J.C. Stam, A. Berns, J.G. Collard, Identification of an invasion-inducing gene, Tiam-1, that encodes a protein with homology to GDP-GTP exchangers for Rho-like proteins, Cell 77 (1994) 537–549.

[138] J.M. Lambert, Q.T. Lambert, G.W. Reuther, A. Malliri, D.P. Siderovski, J. Sondek, J.G. Collard, C.J. Der, Tiam1 mediates Ras activation of Rac by a PI(3)K-independent mechanism, Nat. Cell Biol. 4 (2002) 621–625.

[139] A. Malliri, R.A. van der Kammen, K. Clark, M. van der Valk, F. Michiels, J.G. Collard, Mice deficient in the Rac activator Tiam 1 are resistant to Ras-induced skin tumours, Nature 417 (2002) 867–871.

[140] J. Yamauchi, Y. Miyamoto, A. Tanoue, E.M. Shooter, J.R. Chan, Ras activation of a Rac1 exchange factor, Tiam1, mediates neurotrophin-3-induced Schwann cell migration, Proc. Natl. Acad. Sci. U. S. A. 102 (2005) 14889–14894.

[141] J. Yamauchi, J.R. Chan, Y. Miyamoto, G. Tsujimoto, E.M. Shooter, The neurotrophin-3 receptor TrkC directly phosphorylates and activates the nucleotide exchange factor Dbs to enhance Schwann cell migration, Proc. Natl. Acad. Sci. U. S. A. 102 (2005) 5198–5203.

[142] W.T. Arthur, L.A. Quilliam, J.A. Cooper, Rap1 promotes cell spreading by localizing Rac guanine nucleotide exchange factors, J. Cell Biol. 167 (2004) 111–122.

[143] A.E. Mertens, R.C. Roovers, J.G. Collard, Regulation of Tiam1-Rac signalling, FEBS Lett. 546 (2003) 11–16.

[144] S. Tommasi, R. Dammann, S.G. Jin, X. Zhang, X.J. Avruch, G.P. Pfeifer, RASSF3 and NORE1: identification and cloning of two human homologues of the putative tumor suppressor gene RASSF1, Oncogene 21 (2002) 2713–2720.

[145] R. Dammann, C. Li, J.H. Yoon, P.L. Chin, S. Bates, G.P. Pfeifer, Epigenetic inactivation of a RAS association domain family protein from the lung tumour suppressor locus 3p21.3, Nat. Genet. 25 (2000) 315–319.

[146] M.D. Vos, A. Martinez, C.A. Ellis, T. Vallecorsa, G.J. Clark, The pro-apoptotic Ras effector NoREL may serve as a Ras-regulated tumor suppressor in the lung, J. Biol. Chem. 278 (2003) 21938–21943.

[147] M.D. Vos, C.A. Ellis, C. Elam, A.S. Ulku, B.J. Taylor, G.J. Clark, RASSF2 is a novel K-Ras-specific effector and potential tumor suppressor, J. Biol. Chem. 278 (2003) 28045–28051.

[148] K. Akino, M. Toyota, H. Suzuki, H. Mita, Y. Sasaki, M. Ohe-Toyota, J.P. Issa, Y. Hinoda, K. Imai, T. Tokino, The Ras effector RASSF2 is a novel tumor-suppressor gene in human colorectal cancer, Gastroenterology 129 (2005) 156–169.

[149] S. Baksh, S. Tommasi, S. Fenton, V.C. Yu, L.M. Martins, G.P. Pfeifer, F. Latif, J. Downward, B.G. Neel, The tumor suppressor RASSF1A and MAP-1 link death receptor signaling to Bax conformational change and cell death, Mol. Cell 18 (2005) 637–650.

[150] A. Khokhlatchev, S. Rabizadeh, R. Xavier, M. Nedwidek, T. Chen, X.F. Zhang, B. Seed, J. Avruch, Identification of a novel Ras-regulated proapoptotic pathway, Curr. Biol. 12 (2002) 253–265.

[151] G.G. Tall, M.A. Barbieri, P.D. Stahl, B.F. Horazdovsky, Ras-activated endocytosis is mediated by the Rab5 guanine nucleotide exchange activity of RIN1, Dev. Cell 1 (2001) 73–82.

[152] M. Shibatohge, K.i. Kariya, Y. Liao, C.D. Hu, Y. Watari, M. Goshima, F. Shima, T. Kataoka, Identification of PLC210, a Caenorhabditis elegans phospholipase C, as a putative effector of Ras, J. Biol. Chem. 273 (1998) 6218–6222.

[153] T.G. Jin, T. Satoh, Y. Liao, C. Song, X. Gao, K.i. Kariya, C.D. Hu, T. Kataoka, Role of the CDC25 homology domain of phospholipase epsilon in amplification of Rap1-dependent signaling, J. Biol. Chem. 276 (2001) 30301–30307.

[154] C. Song, T. Satoh, H. Edamatsu, D. Wu, M. Tadano, X. Gao, T. Kataoka, Differential roles of Ras and Rap1 in growth factor-dependent activation of phospholipase C epsilon, Oncogene 21 (2002) 8105–8113.

[155] Y. Bai, H. Edamatsu, S. Maeda, H. Saito, N. Suzuki, T. Satoh, T. Kataoka, Crucial role of phospholipase Cepsilon in chemical carcinogen-induced skin tumor development, Cancer Res. 64 (2004) 8808–8810.

[156] A.B. Zhadanov, D.W. Provance Jr., C.A. Speer, J.D. Coffin, D. Goss, J.A. Blixt, C.M. Reichert, J.A. Mercer, Absence of the tight junctional protein AF-6 disrupts epithelial cell–cell junctions and cell polarity during mouse development, Curr. Biol. 9 (1999) 880–888.

[157] Z. Zhang, H. Rehmann, L.S. Price, J. Riedl, J.L. Bos, AF6 negatively regulates Rap1-induced cell adhesion, J. Biol. Chem. 280 (2005) 33200–33205.

[158] A. Toker, Signaling through protein kinase C, Front. Biosci. 3 (1998) D1134–D1147.

[159] M.S. Marshall, Ras target proteins in eukaryotic cells, FASEB J. 9 (1995) 1311–1318.

[160] R. Prasad, Y. Gu, H. Alder, T. Nakamura, O. Canaani, H. Saito, K. Huebner, R.P. Gale, P.C. Nowell, K. Kuriyama, Cloning of the ALL-1 fusion partner, the AF-6 gene, involved in acute myeloid leukemias with the t(6;11) chromosome translocation, Cancer Res. 53 (1993) 5624–5628.

[161] M. Kuriyama, N. Harada, S. Kuroda, T. Yamamoto, M. Nakafuku, A. Iwamatsu, D. Yamamoto, R. Prasad, C. Croce, E. Canaani, K. Kaibuchi, Identification of AF-6 and canoe as putative targets for Ras, J. Biol. Chem. 271 (1996) 607–610.

[162] M.T. Diaz-Meco, J. Lozano, M.M. Municio, E. Berra, S. Frutos, L. Sanz, J. Moscat, Evidence for the in vitro and in vivo interaction of Ras with protein kinase C zeta, J. Biol. Chem. 269 (1994) 31706–31710.

[163] J. Moscat, P. Rennert, M.T. az-Meco, PKC[zeta] at the crossroad of NF-[kappa]B and Jak1//Stat6 signaling pathways, Cell Death Differ. 13 (2005) 702–711.

[164] T. Joseph, A. Bryant, P. Frankel, R. Wooden, E. Kerkhoff, U.R. Rapp, D.A. Foster, Phospholipase D overcomes cell cycle arrest induced by high-intensity Raf signaling, Oncogene 21 (2002) 3651–3658.

[165] B.B. Friday, A.A. Adjei, K-ras as a target for cancer therapy, Biochim. Biophys. Acta 1756 (2005) 127–144.

[166] D.B. Whyte, P. Kirschmeier, T.N. Hockenberry, I. Nunez-Oliva, L. James, J.J. Catino, W.R. Bishop, J.K. Pai, K- and N-Ras are geranylgeranylated in cells treated with farnesyl protein transferase inhibitors, J. Biol. Chem. 272 (1997) 14459–14464.

[167] M.R. Ahmadian, T. Zor, D. Vogt, W. Kabsch, Z. Selinger, A. Wittinghofer, K. Scheffzek, Guanosine triphosphatase stimulation of oncogenic Ras mutants, Proc. Natl. Acad. Sci. U. S. A. 96 (1999) 7065–7070.

[168] M. Kosloff, Z. Selinger, Substrate assisted catalysis—Application to G proteins, Trends Biochem. Sci. 26 (2001) 161–166.

[169] M. Spoerner, T. Graf, B. Konig, H.R. Kalbitzer, A novel mechanism for the modulation of the Ras-effector interaction by small molecules, Biochem. Biophys. Res. Commun. 334 (2005) 709–713.

[170] D.B. Solit, L.A. Garraway, C.A. Pratilas, A. Sawai, G. Getz, A. Basso, Q. Ye, J.M. Lobo, Y. She, I. Osman, T.R. Golub, J. Sebolt-Leopold, W.R. Sellers, N. Rosen, BRAF mutation predicts sensitivity to MEK inhibition, Nature 439 (2006) 358–362.

[171] Y. Wang, B.K. Van, P. Jiang, S. Przybranowski, C. Omer, J. Sebolt-Leopold, A role for K-ras in conferring resistance to the MEK inhibitor, CI-1040, Neoplasia 7 (2005) 336–347.

[172] S.M. Johnson, H. Grosshans, J. Shingara, M. Byrom, R. Jarvis, A. Cheng, E. Labourier, K.L. Reinert, D. Brown, F.J. Slack, RAS is regulated by the let-7 microRNA family, Cell 120 (2005) 635–647.

[173] D.P. Bartel, MicroRNAs: genomics, biogenesis, mechanism, and function, Cell 116 (2004) 281–297.

[174] I. Bentwich, A. Avniel, Y. Karov, R. Aharonov, S. Gilad, O. Barad, A. Barzilai, P. Einat, U. Einav, E. Meiri, E. Sharon, Y. Spector, Z. Bentwich, Identification of hundreds of conserved and nonconserved human microRNAs, Nat. Genet. 37 (2005) 766–770.

[175] B.P. Lewis, C.B. Burge, D.P. Bartel, Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets, Cell 120 (2005) 15–20.

[176] G.A. Calin, C.D. Dumitru, M. Shimizu, R. Bichi, S. Zupo, E. Noch, H. Aldler, S. Rattan, M. Keating, K. Rai, L. Rassenti, T. Kipps, M. Negrini, F. Bullrich, C.M. Croce, Frequent deletions and down-regulation of micro-RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia, Proc. Natl. Acad. Sci. U. S. A. 99 (2002) 15524–15529.

[177] A. Esquela-Kerscher, F.J. Slack, Oncomirs—MicroRNAs with a role in cancer, Nat. Rev., Cancer 6 (2006) 259–269.

[178] S.M. Hammond, MicroRNAs as oncogenes, Curr. Opin. Genet. Dev. 16 (2006) 4–9.

[179] Y.S. Lee, A. Dutta, MicroRNAs: small but potent oncogenes or tumor suppressors, Curr. Opin. Investig. Drugs 7 (2006) 560–564.

[180] B.J. Reinhart, F.J. Slack, M. Basson, A.E. Pasquinelli, J.C. Bettinger, A.E. Rougvie, H.R. Horvitz, G. Ruvkun, The 21-nucleotide let-7 RNA regulates developmental timing in Caenorhabditis elegans, Nature 403 (2000) 901–906.

[181] S. Bagga, J. Bracht, S. Hunter, K. Massirer, J. Holtz, R. Eachus, A.E. Pasquinelli, Regulation by let-7 and lin-4 miRNAs results in target mRNA degradation, Cell 122 (2005) 553–563.

[182] G.A. Calin, C. Sevignani, C.D. Dumitru, T. Hyslop, E. Noch, S. Yendamuri, M. Shimizu, S. Rattan, F. Bullrich, M. Negrini, C.M. Croce, Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers, Proc. Natl. Acad. Sci. U. S. A. 101 (2004) 2999–3004.

[183] J. Takamizawa, H. Konishi, K. Yanagisawa, S. Tomida, H. Osada, H. Endoh, T. Harano, Y. Yatabe, M. Nagino, Y. Nimura, T. Mitsudomi, T. Takahashi, Reduced expression of the let-7 microRNAs in human lung cancers in association with shortened postoperative survival, Cancer Res. 64 (2004) 3753–3756.

[184] N. Yanaihara, N. Caplen, E. Bowman, M. Seike, K. Kumamoto, M. Yi, R.M. Stephens, A. Okamoto, J. Yokota, T. Tanaka, G.A. Calin, C.G. Liu, C.M. Croce, C.C. Harris, Unique microRNA molecular profiles in lung cancer diagnosis and prognosis, Cancer Cell 9 (2006) 189–198.

[185] B. Vogelstein, K.W. Kinzler, The multistep nature of cancer, Trends Genet. 9 (1993) 138–141.

[186] D. Hanahan, R.A. Weinberg, The hallmarks of cancer, Cell 100 (2000) 57–70.

[187] COSMIC, <http://www.Sanger.Ac.Uk/Genetics/CGP/Cosmic/>.

[188] O.M. Sieber, S.R. Tomlinson, I.P. Tomlinson, Tissue, cell and stage specificity of (epi)mutations in cancers, Nat. Rev., Cancer 5 (2005) 649–655.

[189] M. Bentires-Alj, M.I. Kontaridis, B.G. Neel, Stops along the RAS pathway in human genetic disease, Nat. Med. 12 (2006) 283–285.

[190] A. Roberts, J. Allanson, S.K. Jadico, M.I. Kavamura, J. Noonan, J.M. Opitz, T. Young, G. Neri, The cardiofaciocutaneous syndrome, J. Med. Genet. 43 (2006) 833–842.

[191] M. Tartaglia, B.D. Gelb, Noonan syndrome and related disorders: genetics and pathogenesis, Annu. Rev. Genomics Hum. Genet. 6 (2005) 45–68.

[192] S. Schubbert, M. Zenker, S.L. Rowe, S. Boll, C. Klein, G. Bollag, I. van der Burgt, L. Musante, V. Kalscheuer, L.E. Wehner, H. Nguyen, B. West, K.Y. Zhang, E. Sisternans, A. Rauch, C.M. Niemeyer, K. Shannon, C.P. Kratz, Germline KRAS mutations cause Noonan syndrome, Nat. Genet. 38 (2006) 331–336.

[193] T. Niihori, Y. Aoki, Y. Narumi, G. Neri, H. Cave, A. Verloes, N. Okamoto, R.C. Hennekam, G. Gillessen-Kaesbach, D. Wieczorek, M.I. Kavamura, K. Kurosawa, H. Ohashi, L. Wilson, D. Heron, D. Bonneau, G. Corona, T. Kaname, K. Naritomi, C. Baumann, Germline KRAS and BRAF mutations in cardio-facio-cutaneous syndrome, Nat. Genet. 38 (2006) 294–296.

[194] Y. Aoki, T. Niihori, H. Kawame, K. Kurosawa, H. Ohashi, Y. Tanaka, M. Filocamo, K. Kato, Y. Suzuki, S. Kure, Y. Matsumara, Germline mutations in HRAS proto-oncogene cause Costello syndrome, Nat. Genet. 37 (2005) 1038–1040.

[195] A.L. Estep, W.E. Tidyman, M.A. Teitell, P.D. Cotter, K.A. Rauen, HRAS mutations in Costello syndrome: detection of constitutional activating mutations in codon 12 and 13 and loss of wild-type allele in malignancy, Am. J. Med. Genet., Part A 140 (2006) 8–16.

[196] K.W. Gripp, A.E. Lin, D.L. Stabley, L. Nicholson, C.I. Scott Jr., D. Doyle, Y. Aoki, Y. Matsumara, E.H. Zackai, P. Lapunzina, A. Gonzalez-Meneses, J. Holbrook, C.A. Agresta, I.L. Gonzalez, K. Sol-Church, HRAS mutation analysis in Costello syndrome: genotype and phenotype correlation, Am. J. Med. Genet., Part A 140 (2006) 1–7.

[197] B. Kerr, M.A. Delrue, S. Sigaudy, R. Perveen, M. Marche, I. Burgelin, M. Stef, B. Tang, O.B. Eden, J. O’Sullivan, A. De Sandre-Giovannoli, W. Reardon, C. Brewer, C. Bennett, O. Quarell, E. M’Cann, D. Donnai, F. Stewart, R. Hennekam, H. Cave, Genotype–phenotype correlation in Costello syndrome: HRAS mutation analysis in 43 cases, J. Med. Genet. 43 (2006) 401–405.

[198] P. Rodriguez-Viciana, O. Tetsu, W.E. Tidyman, A.L. Estep, B.A. Conger, M.S. Cruz, F. McCormick, K.A. Rauen, Germline mutations in genes within the MAPK pathway cause cardio-facio-cutaneous syndrome, Science 311 (2006) 1287–1290.

[199] T. Young, F. Mei, J. Liu, R.C. Bast Jr., A. Kurosky, X. Cheng, Proteomics analysis of H-RAS-mediated oncogenic transformation in a genetically defined human ovarian cancer model, Oncogene 24 (2005) 6174–6184.

[200] J.W. Park, S. Kim, Y.Y. Bahk, A proteomic approach for dissecting H-Ras signaling networks in NIH/3T3 mouse embryonic fibroblast cells, Proteomics 6 (2006) 2433–2443.

[201] J.W. Park, S. Kim, K.J. Lim, R.J. Simpson, Y.S. Kim, Y.Y. Bahk, A proteomic approach for unraveling the oncogenic H-Ras protein networks in NIH/3T3 mouse embryonic fibroblast cells, Proteomics 6 (2006) 1175–1186.

[202] P. Alfonso, I. Dolado, A. Swat, A. Nunez, A. Cuadrado, A.R. Nebreda, J.I. Casal, Proteomic analysis of p38alpha mitogen-activated protein kinase-regulated changes in membrane fractions of RAS-transformed fibroblasts, Proteomics 6 (Suppl 1) (2006) S262–S271.

[203] J. Zuber, O.I. Tchernitsa, B. Hinzmann, A.C. Schmitz, M. Grips, M. Hellriegel, C. Sers, A. Rosenthal, R. Schafer, A genome-wide survey of RAS transformation targets, Nat. Genet. 24 (2000) 144–152.

[204] G. Cobellis, C. Missero, B. Simionati, G. Valle, R. Di Lauro, Immediate early genes induced by H-Ras in thyroid cells, Oncogene 20 (2001) 2281–2290.

[205] O.I. Tchernitsa, C. Sers, J. Zuber, B. Hinzmann, M. Grips, A. Schramme, P. Lund, A. Schwendel, A. Rosenthal, R. Schafer, Transcriptional basis of KRAS oncogene-mediated cellular transformation in ovarian epithelial cells, Oncogene 23 (2004) 4536–4555.

[206] J. Qian, J. Niu, M. Li, P.J. Chiao, M.S. Tsao, In vitro modeling of human pancreatic duct epithelial cell transformation defines gene expression changes induced by K-ras oncogenic activation in pancreatic carcinogenesis, Cancer Res. 65 (2005) 5045–5053.

[207] I.M. Seiden-Long, K.R. Brown, W. Shih, D.A. Wigle, N. Radulovich, I. Jurisica, M.S. Tsao, Transcriptional targets of hepatocyte growth factor signaling and Ki-ras oncogene activation in colorectal cancer, Oncogene 25 (2006) 91–102.

[208] S. Bloethner, B. Chen, K. Hemminki, J. Muller-Berghaus, S. Ugurel, D. Schadendorf, R. Kumar, Effect of common B-RAF and N-RAS mutations on global gene expression in melanoma cell lines, Carcinogenesis 26 (2005) 1224–1232.

[209] A. Sweet-Cordero, S. Mukherjee, A. Subramanian, H. You, J.J. Roix, C. Ladd-Acosta, J. Mesirov, T.R. Golub, T. Jacks, An oncogenic KRAS2

expression signature identified by cross-species gene-expression analysis, Nat. Genet. 37 (2005) 48–55.

[210] I.J. Kim, H.C. Kang, S.G. Jang, K. Kim, S.A. Ahn, H.J. Yoon, S.N. Yoon, J.G. Park, Oligonucleotide microarray analysis of distinct gene expression patterns in colorectal cancer tissues harboring BRAF and K-ras mutations, Carcinogenesis 27 (2006) 392–404.

[211] J. Jonkers, A. Berns, Oncogene addiction: sometimes a temporary slavery, Cancer Cell 6 (2004) 535–538.

[212] I.B. Weinstein, Cancer. Addiction to oncogenes—The Achilles heel of cancer, Science 297 (2002) 63–64.

[213] V.K. Rajasekhar, A. Viale, N.D. Socci, M. Wiedmann, X. Hu, E.C. Holland, Oncogenic Ras and Akt signaling contribute to glioblastoma formation by differential recruitment of existing mRNAs to polysomes, Mol. Cell 12 (2003) 889–901.

[214] J. Spence, B.M. Duggan, C. Eckhardt, M. McClelland, D. Mercola, Messenger RNAs under differential translational control in Ki-ras-transformed cells, Mol. Cancer Res. 4 (2006) 47–60.

[215] V.K. Rajasekhar, E.C. Holland, Postgenomic global analysis of translational control induced by oncogenic signaling, Oncogene 23 (2004) 3248–3264.

[216] K.P. Janssen, M. Abal, F. El Marjou, D. Louvard, S. Robine, Mouse models of K-ras-initiated carcinogenesis, Biochim. Biophys. Acta 1756 (2005) 145–154.

[217] C.J. Kemp, Multistep skin cancer in mice as a model to study the evolution of cancer cells, Semin. Cancer Biol. 15 (2005) 460–473.

[218] C. Guerra, N. Mijimolle, A. Dhawahir, P. Dubus, M. Barradas, M. Serrano, V. Campuzano, M. Barbacid, Tumor induction by an endogenous K-ras oncogene is highly dependent on cellular context, Cancer Cell 4 (2003) 111–120.

[219] C.F. Kim, E.L. Jackson, A.E. Woolfenden, S. Lawrence, I. Babar, S. Vogel, D. Crowley, R.T. Bronson, T. Jacks, Identification of bronchioalveolar stem cells in normal lung and lung cancer, Cell 121 (2005) 823–835.

[220] J. Perez-Losada, A. Balmain, Stem-cell hierarchy in skin cancer, Nat. Rev., Cancer 3 (2003) 434–443.

[221] X. Guo, L. Stratton, J.W. Schrader, Expression of activated M-Ras in hemopoietic stem cells initiates leukemogenic transformation, immortalization and preferential generation of mast cells, Oncogene 25 (2006) 4241–4244.

[222] H.J. Bernstein, Recent changes to RasMol, recombining the variants, Trends Biochem. Sci. 25 (2000) 453–455.

[223] L. Tong, M.V. Milburn, A.M. de Vos, S.H. Kim, Structure of ras proteins, Science 245 (1989) 244.

[224] A.J. Scheidig, C. Burmester, R.S. Goody, The pre-hydrolysis state of p21 (ras) in complex with GTP: new insights into the role of water molecules in the GTP hydrolysis reaction of ras-like proteins, Structure 7 (1999) 1311–1324.

[225] S.L. Baldauf, The deep roots of eukaryotes, Science 300 (2003) 1703–1706.
