
The RNA Degradosome of Escherichia coli: An mRNA-Degrading Machine Assembled on RNase E

Agamemnon J. Carpousis

Laboratoire de Microbiologie et Génétique Moléculaires, Unité Mixte de Recherche 5100, Centre National de la Recherche Scientifique et Université Paul Sabatier, 31062 Toulouse, France; email: carpousi@ibcg.biotoul.fr

Annu. Rev. Microbiol. 2007. 61:71–87

First published online as a Review in Advance on April 20, 2007

The Annual Review of Microbiology is online at micro.annualreviews.org

This article's doi: 10.1146/annurev.micro.61.080706.093440

Copyright © 2007 by Annual Reviews. All rights reserved

0066-4227/07/1013-0071$20.00

**Key Words**

DEAD-box RNA helicase, natively unstructured protein, mRNA degradation, PNPase

**Abstract**

The RNA degradosome of *Escherichia coli* is a multiprotein complex involved in the degradation of mRNA. The principal components are RNase E, PNPase, RhlB, and enolase. RNase E is a large multidomain protein with an N-terminal catalytic region and a C-terminal noncatalytic region that is mostly natively unstructured protein. The noncatalytic region contains sites for binding RNA and for protein-protein interactions with other components of the RNA degradosome. Several recent studies suggest that there are alternative forms of the RNA degradosome depending on growth conditions or other factors. These alternative forms appear to modulate RNase E activity in the degradation of mRNA. RNA degradosome-like complexes appear to be conserved throughout the Proteobacteria, but there is a surprising variability in composition that might contribute to the adaptation of these bacteria to the enormously wide variety of niches in which they live.

E and PNPase are important enzymes for the maturation of stable RNA and the degradation of mRNA (10, 16, 21, 26). RNase E cleavage is generally believed to be the rate-limiting step in mRNA degradation (40). Enolase is a glycolytic enzyme whose role in mRNA degradation is still a mystery. RhlB is a DEAD-box RNA helicase that facilitates the degradation of mRNA by RNase E and PNPase.

## Contents

- INTRODUCTION ..... 72
- RNase E–MEDIATED mRNA DEGRADATION IN ESCHERICHIA COLI ..... 72
- THE STRUCTURE AND FUNCTION OF RNase E ..... 73
- THE STRUCTURE AND FUNCTION OF PNPase ..... 75
- THE STRUCTURE AND FUNCTION OF RhlB ..... 75
- THE STRUCTURE AND FUNCTION OF ENOLASE ..... 77
- PUTTING RNase E, PNPase, RhlB, AND ENOLASE TOGETHER: ASSEMBLING THE RNA DEGRADOSOME ..... 77
- ALTERNATIVE FORMS OF THE RNA DEGRADOSOME IN ESCHERICHIA COLI ..... 78
- PHYSIOLOGICAL STUDIES OF THE FUNCTION OF THE RNA DEGRADOSOME IN ESCHERICHIA COLI ..... 80
- AN RhlB-PNPase COMPLEX IN THE ABSENCE OF THE PROTEIN SCAFFOLD OF RNase E ..... 81
- THE PHYLOGENETIC DISTRIBUTION OF RNase E AND THE EVOLUTION OF THE RNA DEGRADOSOME ..... 82

## RNase E–MEDIATED mRNA DEGRADATION IN ESCHERICHIA COLI

In this review I focus on mRNA degradation because RNase E, PNPase, and RhlB each have an important role in this process. Under normal laboratory growth conditions, the principal pathway for mRNA degradation in *E. coli* is mediated by RNase E (Figure 1). RNase E inactivates polyribosomal mRNA by endoribonucleolytic cleavage to produce mRNA fragments, which are digested to nucleotides by exoribonucleases with help from RhlB and poly(A) polymerase (PAP). In wild-type cells, it is difficult to detect the mRNA fragments depicted in Figure 1 because they are rapidly digested by the exoribonucleases. That is, the initial cleavage by RNase E is the slow step that corresponds to the functional degradation of mRNA, whereas the overall process corresponds to the chemical degradation of mRNA. The elucidation of this pathway is based on the biochemical characterization of the enzymes in Figure 1 and on genetic studies employing *E. coli* strains with mutations in the genes encoding these enzymes.

RNase E is the only essential enzyme in this pathway, although it is not clear if the loss of viability is due to defective mRNA degradation or to another function such as the maturation of ribosomal or transfer RNA (20, 63, 76). Strains in which RNase II and PNPase, or RNase R and PNPase, have been inactivated are also not viable (21). Strains in which RNase II and RNase R have been inactivated are viable, suggesting that PNPase

## INTRODUCTION

The RNA degradosome was discovered in two different laboratories during work on the purification and characterization of *Escherichia coli* ribonuclease E (RNase E) and the identification of factors affecting the activity of the RNA-degrading enzyme, polynucleotide phosphorylase (PNPase) (9, 11, 23, 70). The major components of this multiprotein complex include RNase E, PNPase, RNase helicase B (RhlB), and enolase (54, 71). RNase

### RNA degradosome:
bacterial multiprotein complex involved in RNA processing and degradation

### PNPase:
polynucleotide phosphorylase

alone can perform all the essential functions of  
the exoribonucleases. The 3′ polyadenylation  
of mRNA fragments by PAP activates their  
degradation by the exoribonucleases (10, 18,  
22). The role of the DEAD-box RNA heli-  
case, RhlB, is discussed below. Three of the  
enzymes in this pathway, RNase E, PNPase,  
and RhlB, are major components of the RNA  
degradosome.

Although the pathway depicted in  
Figure 1 is useful for thinking about mRNA  
decay in *E. coli*, the possibility that alternate  
pathways (co)exist needs to be considered.  
The inactivation of RNase E slows mRNA  
degradation but does not completely block  
it. Other endoribonucleases such as RNase  
III, RNase P, RNase G, and RNase Z have  
been implicated to a limited extent in mRNA  
decay (16, 26, 41, 65, 66). However, strains  
in which RNase E and each of the other  
endoribonucleases have been inactivated ex-  
hibit mRNA degradation rates that are only  
slightly slower than when RNase E alone has  
been inactivated. In a strain in which RNase  
E, the exoribonucleases, and PAP have been  
inactivated, mRNA degradation is completely  
blocked (28). Taken together, these results  
suggest that, in the absence of RNase E activ-  
ity, *E. coli* has a secondary mRNA degradation  
pathway that appears to be mediated by PAP  
and the exoribonucleases. Recent microarray  
studies suggest that mRNA polyadenylation  
in *E. coli* is more widespread than previously  
believed and that the steady-state level of  
a significant cross section of transcripts  
increases when the gene encoding PNPase  
is disrupted (56, 57). Furthermore, there  
appears to be a subtle but intriguing interplay  
in the *rpsO* mRNA between degradation  
initiated by RNase E cleavage and degra-  
dation initiated by PAP (51). Important  
questions include what proportion of mRNA  
degradation is initiated by 3′ polyadenyla-  
tion and whether this proportion changes  
in response to growth conditions. Does  
control of this pathway involve the RNA  
degradosome?

---

**a Primary endoribonuclease cleavage**

![Diagram showing primary endoribonuclease cleavage](https://i.imgur.com/yourimageurl.png)

**b Secondary exoribonuclease digestion**

![Diagram showing secondary exoribonuclease digestion](https://i.imgur.com/yourimageurl.png)

**Figure 1**  
The principal pathway for mRNA degradation in *E. coli*. RNase E, PNPase,  
and RhlB are components of the RNA degradosome. (*a*) RNase E initially  
inactivates polyribosomal mRNA by cleaving in the translation initiation  
region (*bold arrow*) or in the intercistronic regions of polycistronic messages.  
As the elongating ribosomes finish translating the functionally inactivated  
mRNA, RNase E further cleaves the ribosome-free mRNA (*dashed arrows*).  
(*b*) Fragments of mRNA produced by RNase E are digested to nucleotides  
by the exonucleases. RNA helicase B (RhlB) and poly(A) polymerase (PAP)  
are accessory factors that facilitate exoribonucleolytic activity.

---

**THE STRUCTURE AND FUNCTION OF RNase E**

RNase E is a large, 1061-amino-acid multidomain protein that can be roughly divided into  
two halves. The N-terminal half of RNase E contains the catalytic domain, whereas the C-  
terminal half is an exceptionally long stretch of natively unstructured protein that has no  
known enzymatic activity. The structure of the catalytic domain of RNase E, complexed with RNA, has recently been solved by X-ray  
crystallography (6). Briefly, the enzyme is a homotetramer that can be viewed as a dimer  
of dimers. The catalytic site is structurally related to DNase I. The overall structure of  
the tetramer is important for formation of

www.annualreviews.org • The Bacterial RNA Degradosome

the catalytic site, which is consistent with previous biochemical work suggesting that protomers by themselves have little or no activity. Other important features include an S1 RNA binding domain, a site forming a pocket that specifically interacts with the terminus of 5′-monophosphorylated RNA, and a channel that permits access to the catalytic site of single-stranded RNA, but not double-stranded RNA. The structure helps researchers to better understand many characteristics of the enzyme elucidated in previous biochemical and genetic studies, including the specificity for single-stranded RNA and the activation of catalytic activity by 5′-monophosphorylated RNA.

The noncatalytic region of RNase E contains two RNA binding sites and three protein binding sites involved in the assembly of the RNA degradosome. The region containing the protein binding sites has also been called the scaffold because it is the framework upon which the other components of the RNA degradosome are assembled (17, 33, 79). Most of the noncatalytic region of RNase E is predicted to be natively unstructured by sequence analysis. Evidence supporting this prediction, including studies by circular dichroism and X-ray scattering, has recently been obtained (5). Natively unstructured protein (also known as intrinsically unstructured protein) is protein with little or no propensity to fold under native conditions (78). Although unfolded protein is often found as relatively short stretches in proteins that are normally folded, some proteins are completely unfolded under native conditions. These proteins have mechanical properties that can be used as molecular springs, rulers, and spacers. The underlying physical force is due to the significant entropic penalty involved in compressing or stretching unfolded protein. Given the large size of the natively unstructured region in RNase E (more than 500 residues), the mechanical properties of this region could be important for RNA degradosome function.

The recently solved crystal structure of enolase bound to its cognate site in RNase

E shows that the interaction involves little if any folding of RNase E (12). A polypeptide from the region of RNase E that binds enolase fits snugly into a cleft at the interface of the protomers that comprise the enolase dimer. An interaction network of mostly hydrogen bonds involves 13 amino acids from RNase E. The overall structure of enolase is unperturbed by the interaction, consistent with previous work showing that enolase in the RNA degradosome has full activity (71). The RNase E polypeptide forms a single turn of the α-helix. It is not known if this α-helix preexisted in solution before complex formation, but the region that binds enolase is one of several small islands in the noncatalytic region of RNase E predicted to have the capacity to fold (5). Sequence comparisons of RNase E homologs from other γ-Proteobacteria revealed a short conserved motif of about 25 amino acids that corresponds to the enolase binding site. Because the motif is larger than the interaction detected in the crystal structure, it has been suggested that there could be a second interaction involving an additional 5 to 10 residues. Taken together, these results show that enolase interacts with RNase E by using a relatively small segment (15 to 25 residues) in the noncatalytic region. Although structures of RhlB and PNPase complexed to segments of RNase E have not yet been solved, it seems likely that these interactions are similar to the interaction elucidated for enolase (5, 12, 50).

Obtaining molecular models for the catalytic domain of RNase E and other components of the RNA degradosome has been tremendously stimulating. The experimental confirmation that much of the noncatalytic region of RNase E is intrinsically unstructured helps researchers to better understand certain work performed over the past decade in which an extensive deletion analysis of the noncatalytic region of RNase E was performed (17, 33, 44, 64, 79). With the possible exception of one of the RNA binding domains, deletion of part or the entire noncatalytic region of RNase E has at most a small effect on the

Carpousis

catalytic activity of RNase E. The deletion  
of the catalytic domain or of a specific RNA  
or protein binding site does not affect binding  
at other sites that have not been deleted.  
Relatively small polypeptides corresponding  
to binding sites in the noncatalytic region of  
RNase E have full binding activity. All these  
results point to a structure in which the catalytic  
domain of RNase E and the individual  
binding sites act independently of each other:  
a string of pearls model, in which each pearl  
is structurally autonomous.

**THE STRUCTURE AND**  
**FUNCTION OF PNPAse**

PNPase acts as a 3′ exoribonuclease in *E. coli*.  
It degrades RNA from the 3′ end, releasing  
nucleotides in a processive 3′ to 5′ degradation  
reaction. However, PNPase is a phosphorylase: It uses phosphate to cleave phosphodiester bonds. There is a significant difference between hydrolysis of a polynucleotide and phosphorolysis of a polynucleotide: Hydrolysis is a favorable reaction, whereas the phosphorolysis is a reaction close to equilibrium. Indeed, PNPase was discovered as the first RNA-synthesizing enzyme (27). It is now known that PNPase is primarily an RNA-degrading enzyme and that the concentration of free phosphate in the cell favors phosphorolysis. Nevertheless, there have been reports that PNPase can occasionally add 3′-polynucleotide extensions to mRNA or mRNA decay fragments (55, 57). Although intriguing, the physiological relevance of the synthetic reaction remains to be elucidated.

The structure of the PNPase from *Streptomyces antibioticus* has been solved by X-ray crystallography (50, 74, 75). Briefly, PNPase is a homotrimer. Each protomer is composed of four domains: two PH domains, which are structural elements that form the catalytic site, and two RNA binding domains. The PH domains are related by sequence and structure and are believed to have arisen by an ancient gene duplication-fusion event. The PH domains form a hexameric ring, with the catalytic site in the central channel formed by the ring. The RNA binding domains are on one face of the ring, forming a cap that could control access of the RNA substrate to the catalytic core and facilitate processivity. In a recent study in which the RNA binding domains of *E. coli* PNPase were deleted, the interaction between PNPase and RNase E was unperturbed (73). Thus, RNase E interacts with the catalytic core of PNPase. The threefold symmetry of PNPase suggests that it has three independent sites for the interaction with RNase E.

It is not yet known if the interaction between RNase E and PNPase alters the structure of PNPase. As described in the next section, the interaction between RNase E and RhlB significantly stimulates RhlB activity. Thus, not all protein-protein interactions in the RNA degradosome are neutral with respect to enzymatic activity.

**THE STRUCTURE AND**  
**FUNCTION OF RhlB**

The DEAD-box RNA helicases are a ubiquitous family of enzymes (46, 77). Although called DEAD-box proteins, D-E-A-D is one of nine motifs within a structurally conserved catalytic core composed of about 400 amino acids. In vitro, these proteins often have RNA-dependent ATPase activity and ATPase-dependent RNA unwinding activity. However, these helicases generally unwind only a few base pairs of duplex RNA. The DEAD-box proteins often act locally at specific sites to remodel RNA-RNA or RNA-protein interactions. They have specific roles in one or a few steps in processes such as translation initiation, mRNA splicing, and ribosome assembly. The conserved catalytic core, which is the motor of these proteins, is directed to specific targets by accessory domains that are present either in the same protein or within a multiprotein complex.

*E. coli* encodes five DEAD-box proteins: CsdA (also known as DeaD), DbpA, RhlB, RhlE, and SrmB (31). CsdA, DbpA, and SrmB

www.annualreviews.org • The Bacterial RNA Degradosome

have roles in ribosome assembly (13, 14, 34). The function of RhlE is not known. Strains in which the genes encoding each of these proteins have been disrupted are viable. This is in contrast to yeast, which has more than 20 DEAD-box proteins. Most of them are essential for viability (46, 77). The smaller number of DEAD-box proteins in *E. coli* could be due to its translation initiation mechanism, which does not require a DEAD-box RNA helicase, and the lack of a spliceosome-based RNA splicing system. One possible explanation for the nonessential nature of the *E. coli* DEAD-box proteins is that they could have overlapping functions. However, this appears unlikely because *E. coli* strains with multiple mutations inactivating more than one DEAD-box protein are also viable (31; V. Khemici & A.J. Carpousis, unpublished results). The dispensability of the *E. coli* DEAD-box proteins suggests that, like many other pathways in *E. coli*, ribosome assembly and mRNA degradation are highly streamlined, requiring few accessory factors. As discussed below, strains in which RhlB has been inactivated have clear-cut defects in mRNA degradation, some of which have been analyzed in detail.

A functional interaction between RhlB and PNPase has been elucidated in biochemical experiments. PNPase specifically acts on single-stranded RNA substrates, although partially structured RNA substrates can be degraded in vitro. In kinetic experiments, pausing of the PNPase at RNA stem-loops or more complicated structures can be detected as intermediates in degradation. These results are usually interpreted as follows. PNPase pauses at structured RNA because only single-stranded RNA can thread into the catalytic core. However, terminal base pairs in duplex RNA frequently open spontaneously (breathe), and PNPase can nibble duplex RNA as it breathes. Repeated extragenic palindromic (REP) sequences present in the noncoding regions of many polycistronic mRNAs are a particularly effective barrier against digestion by PNPase and other exoribonucleases (53, 62). The RNA degradosome was shown to degrade mRNA substrates containing REP elements in a reaction that required ATP hydrolysis (71). In subsequent reconstitution experiments, a functional interaction between PNPase and RhlB was shown to depend on the binding of these enzymes to the protein scaffold of RNase E (17). The reaction catalyzed by RhlB is unlikely to involve the unwinding of long stretches of duplex RNA. All that is required is a little push from ATP hydrolysis by RhlB to melt a few terminal base pairs and thus facilitate threading into the catalytic site.

Highly purified RhlB has no detectable RNA-dependent ATPase activity, even though this activity was detected in the RNA degradosome. A polypeptide derived from the noncatalytic region of RNase E, which included the RhlB binding site, strongly stimulated the RNA-dependent ATPase activity of RhlB (79). RhlB and the RNase E polypeptide formed a 1:1 complex, and the RNA-dependent ATPase activity at saturating concentrations of the RNase E polypeptide was comparable to the activity measured in the RNA degradosome. In other work, an amino acid substitution in the noncatalytic region of RNase E that disrupted the interaction with RhlB also interfered with the stimulation of RNA-dependent ATPase activity (37). A complex between RhlB and the RNase E polypeptide also has ATPase-dependent RNA unwinding activity (12a). In RNA unwinding experiments performed with a smaller RNase E polypeptide in which RNA binding sites flanking the RhlB binding site were deleted, RNA unwinding activity diminished significantly. The importance of these regions for RNA binding was confirmed in gel retardation and quantitative filter binding assays. Taken together, these results show that the binding of RhlB to RNase E has two major effects: (a) The protein-protein interaction activates the ATPase activity of RhlB, and (b) the RNA binding sites provided by RNase E participate in the RNA unwinding reaction.

The structure of RhlB has been modeled on the basis of other DEAD-box proteins

whose structures have been solved by X-ray crystallography (12a). RhlB is essentially composed of the conserved DEAD-box catalytic core and a short basic C-terminal extension that appears to be involved in nonspecific RNA binding. The work described in this section strongly suggests that the binding of RhlB to RNase E and thus its association with the RNA degradosome determine its specificity in mRNA degradation.

**THE STRUCTURE AND FUNCTION OF ENOLASE**

Enolase is a glycolytic enzyme. Sequence comparisons and structures solved by X-ray crystallography show that the enzymes from *E. coli* and yeast are highly conserved (39). The enzyme catalyzes the reversible dehydration of 2-phosphoglycerate to phosphoenolpyruvate. Enolase is an abundant protein in *E. coli* and only a small proportion is bound to RNase E as a component of the RNA degradosome (47, 71). A number of functions for enolase in the RNA degradosome have been suggested, including the possibility that it serves as a sensor linking the energetic state of the cell to mRNA degradation. Microarray analysis suggests that enolase affects the degradation of certain messages that encode enzymes involved in energy-generating pathways (1, 2), and enolase and RNase E are involved in the response to phosphosugar stress (59, 60). Nevertheless, the physiological importance of linking the energetic state of the cell to mRNA degradation and the molecular basis for such regulation are not clear.

**PUTTING RNase E, PNPase, RhlB, AND ENOLASE TOGETHER: ASSEMBLING THE RNA DEGRADOSOME**

In work in which RNase E was purified by classical biochemical techniques, PNPase, RhlB, and enolase copurified in stoichiometric amounts (3, 9, 23, 71). Attempts to determine the molecular weight of the RNA degradosome using these preparations were ambiguous. In velocity sedimentation experiments, the RNA degradosome exhibited considerable size heterogeneity, with sedimentation constants ranging from 8S to 16S on glycerol gradients (9). RNase E, PNPase, RhlB, and enolase cosedimented in approximately equal proportions in the RNA degradosome fractions. Using native molecular mass markers, we estimated the RNA degradosome to have a molecular mass ranging from 160 to 460 kDa. However, by gel permeation, the RNA degradosome was estimated to have a molecular mass of at least 1500 kDa (A.J. Carpousis, unpublished results). The size heterogeneity is not likely due to RNA binding because the RNA degradosome preparations were free of detectable nucleic acid. For these reasons, we suggested that the RNA degradosome might be a dynamic complex (9). That is, there might be an equilibrium between different size complexes. The discrepancy between molecular mass estimated by sedimentation and by gel permeation could be due to the natively unfolded nature of the noncatalytic region of RNase E. The molecular mass estimated by velocity sedimentation is likely a significant underestimation owing to the open conformation of a complex containing extended regions of natively unstructured protein.

Figure 2 depicts the basic building block of the RNA degradosome on the basis of a recently proposed model (50). Four noncatalytic regions extend from the tetrameric catalytic core of RNase E. Each noncatalytic region associates with a monomer of RhlB, a dimer of enolase, and a trimer of PNPase. This complex has a molecular mass of about 2000 kDa. Note that each PNPase depicted in Figure 2 interacts with only one noncatalytic region. If each PNPase can interact with three RNase E noncatalytic regions, it is possible to build a larger model with a molecular mass of about 4000 kDa (50). The 4000-kDa complex is a closed structure in which all possible protein-protein interactions are filled. However, the structure shown in Figure 2

RNase E as the central component. The important concept is that the RNA degradosome could be a dynamic structure in which each component interacts freely with its nearest neighbors. The structure could be a molecular domain within which it is possible to channel an RNA substrate. That is, once an RNA substrate interacts with one of the components in the RNA degradosome, the probability of escaping from the complex would be lower than the probability of interacting with other components. These ideas are highly speculative, but it is evident that the influence of the natively unstructured protein on the structure and function of the RNA degradosome raises a number of interesting and important questions for future research.

The models for the RNA degradosome discussed in this section beg the question of whether such structures actually exist in *E. coli*. The cell biology of the RNA degradosome is a largely uncharted area of research that is ripe for exploration. A recent article, based on in vivo fluorescence tagging of proteins, suggests that the major components of the RNA degradosome colocalize in a filamentous structure (75a). The validity of this observation and the underlying molecular basis for forming a filamentous structure remains to be established.

**ALTERNATIVE FORMS OF THE RNA DEGRADOSOME IN ESCHERICHIA COLI**

RNase E copurifies and coprecipitates with other proteins that have been detected in substoichiometric amounts relative to the major components. These proteins include polyphosphate kinase (PPK), DnaK, and GroEL (3, 54, 72a). Furthermore, in a recent systematic analysis of protein-protein interactions in *E. coli*, RNase E, RhlB, enolase, and PNPase formed an interaction network as expected, but interactions with other proteins were detected, including the RNA binding protein Hfq (4). PPK is an enzyme that catalyzes the reversible polymerization of

Figure 2  
A model of the structure of the RNA degradosome. The components of the RNA degradosome are color coded. The noncatalytic region extending from the catalytic core of RNase E has been drawn symmetrically. However, the natively unstructured protein in each noncatalytic extension is likely to form a dynamic, random coil-like structure, and each extension is expected to act independently of the other noncatalytic extensions.

can be extended to form open structures that are heterogeneous in size. The formation of the 4000-kDa closed complex may depend on the mechanical properties of the natively unstructured protein. If the entropic penalty for closed structure formation is too high, then a dynamic series of open structures could predominate.

The structural model shown in Figure 2 has a number of interesting functional consequences. First, the catalytic site of RNase E appears to be at the center of the RNA degradosome. Second, if the natively unstructured protein is not constrained, RhlB, enolase, and PNPase are part of a protein cloud that surrounds the catalytic core of RNase E. This view is diametrically opposed to models in which each component lies at fixed distances in a rigid structure. Figure 2 can be redrawn to put PNPase at the center by creating interactions between one trimer of PNPase and three noncatalytic regions of RNase E. Thus, it is probably a mistake to think of

polyphosphate from ATP. In vitro, PPK binds RNA, and polyphosphate inhibits RNA degradosome activity (3). However, the physiological significance of these results is not known. The association of DnaK and GroEL could simply be a reflection of their chaperone activity. Hfq is discussed below.

In addition to proteins present in stoichiometric amounts, proteins loosely associated with the RNA degradosome could also have important roles. During its purification, the RNA degradosome copurified with a PAP activity that was removed in the last step of the purification. Furthermore, in vitro results have demonstrated a protein-protein interaction between PAP and RNase E (72). Unfortunately, the physiological significance of this interaction is unclear, since to date the only specific in vivo pathways that have been elucidated for PAP involve RNase R (15, 35) and the RNA binding protein Hfq (29, 58).

There is evidence that the RNA degradosome is modified during cold shock (68). The DEAD-box RNA helicase, CsdA, which is a cold shock protein, associates with the RNA degradosome at low temperature. CsdA reconstituted into the RNA degradosome can replace the function of RhlB in vitro. However, analysis of the site where CsdA binds to RNase E revealed that it is distinct from the RhlB binding site (37). This secondary site binds CsdA, RhlE, and SrmB indiscriminately in vitro and, like CsdA, RhlE can replace RhlB activity in vitro. This work has raised more questions than it has answered. First, CsdA in the cold shock degradosome does not appear to replace RhlB, rather it is added to the complex. Nevertheless, even if structured mRNA is more difficult to degrade in the cold, the advantage of having two RNA helicases in the complex is not clear. Much more work is required to understand how the RNA degradosome is adapted to the cold. Second, although CsdA is overexpressed at low temperature, it is also present under normal growth conditions. Thus, how the RNA degradosome is remodeled in response to cold shock remains to be clarified.

Hfq is an abundant *E. coli* RNA binding protein involved in a variety of processes including the posttranscriptional control of gene expression by small noncoding regulatory RNA (sRNA) (8, 25, 80). Two well-characterized sRNAs, RyhB and SgrS, as well as many other sRNAs, require Hfq for function. Hfq bound to the RyhB and SgrS protects them from degradation by RNase E and mediates their interaction with target mRNA. RyhB and SgrS block translation by binding to the translation initiation region in the target mRNA. Upon translation repression, both the sRNA and the target mRNA are degraded in a coordinated reaction that is mediated by RNase E (52, 60). The interaction of the sRNA with the mRNA is sufficient to inhibit translation. The degradation of the mRNA appears to be a secondary event that makes inhibition irreversible (61). A surprising finding was that in RNase E pull-down experiments a complex between RNase E, Hfq, and the SgrS was identified (60). Other results showed that the protein scaffold region of RNase E was necessary for the interaction with Hfq and that the RNase E–Hfq interaction was necessary for the degradation of the target mRNA. Furthermore, the RNase E–Hfq complex is distinct from the RNA degradosome in that it did not contain RhlB, enolase, or PNPase. Taken together, these results provide clear-cut evidence for an RNase E-based complex containing Hfq that is distinct from the RNA degradosome.

Recently, two protein inhibitors of RNase E named RraA and RraB have been identified in genetic screens (24, 43). These proteins inhibit RNase E activity in vivo and in vitro and appear to work by interacting with the noncatalytic region of RNase E. RraA binding cannot be localized to a single specific site in the noncatalytic region of RNase E, whereas RraB interacts in the same region where RhlB binds. The failure to localize RraA binding to a single region suggests that it recognizes multiple sites in the noncatalytic region. In vivo expression of RraA or RraB had differential effects on the level of certain specific transcripts

www.annualreviews.org • The Bacterial RNA Degradosome
and on the composition of the RNA degradosome in an RNase E pull-down experiment (24). Both inhibitors significantly reduced the amount of PN Pase in the RNA degradosome. RraA also significantly reduced the amount of RhlB and enolase. In these experiments, CsdA bound to RNase E was detected in the cells where RraB was expressed. This is the first work to suggest that *E. coli* has protein factors that can modulate the composition of the RNA degradosome.

*E. coli* appears to have several different RNase E–based multiprotein complexes and the composition of the complexes can be modulated by factors such as cold shock, or RraA and RraB. The principal questions raised by these observations are how do changes in the composition of the RNA degradosome affect RNase E activity, how do factors such as RraA and RraB remodel the RNA degradosome, and what are the signals that promote remodeling?

### PHYSIOLOGICAL STUDIES OF THE FUNCTION OF THE RNA DEGRADOSOME IN *ESCHERICHIA COLI*

Strains of *E. coli* expressing truncated forms of RNase E in which part or all of the noncatalytic region have been deleted are viable (38, 79). Strains in which the genes encoding RhlB, enolase, and PN Pase have been disrupted are also viable, although growth of a strain lacking enolase requires special feeding conditions. These results demonstrate that the assembly of the RNA degradosome is not essential in *E. coli*. Nevertheless, recent work has revealed a variety of effects on mRNA degradation in these mutant strains. One of the original alleles producing RNase E lacking the noncatalytic region, *rne131*, was isolated in a screen for suppressors of a temperature-sensitive allele of *mukB* (38). In an *rne131* strain, mRNA degradation is slower (48). In fact, the effect of *rne131* is comparable to the effect of a temperature-sensitive *rne* allele at the nonpermissive temperature. The *rne131* strain grows normally. Functions such as the processing of ribosomal and transfer RNA are not affected. Taken together, these results suggest that the noncatalytic region of RNase E serves as an adapter that helps target RNase E to mRNA.

Subsequent work suggested that the major element involved in mRNA degradation was an RNA binding domain in the noncatalytic region of RNase E, not the protein scaffold to which RhlB, enolase, and PN Pase bind (64). Nevertheless, recent genome-wide surveys of mRNA steady-state levels and decay rates suggest that the assembly of the RNA degradosome is necessary for normal mRNA decay (1, 2). That is, in a strain with the *rne131* allele as well as strains in which the genes encoding RhlB, enolase, or PN Pase have been inactivated, the levels of many mRNAs are significantly perturbed. Imbalances in gene expression could be one explanation why strains with mutations disrupting the noncatalytic region of RNase E are easily outcompeted when grown in mixed cultures with their wild-type parent (44).

mRNA synthesized by the bacteriophage T7 RNA polymerase are exceptionally sensitive to degradation by RNase E (30). This instability is due to the rate of RNA elongation of the T7 RNA polymerase, which is eightfold faster than the host enzyme (49). The T7 RNA polymerase rapidly outpaces translating ribosomes, producing long stretches of ribosome-free mRNA. A functional *lacZ* message transcribed by the T7 RNA polymerase in a strain with the *rne131* mutation produces 10- to 20-fold more β-galactosidase than in the wild-type control (44, 48). The production of β-galactosidase was used to analyze a series of deletions disrupting elements in the noncatalytic region of RNase E. This study showed that the disruption of the RhlB binding site together with the flanking RNA binding domains had an effect on β-galactosidase production that was equal to the disruption of the entire noncatalytic region (44). It was shown subsequently that disrupting the gene encoding RhlB also
significantly increased β-galactosidase production (36). Taken together, these results suggest RhlB in the RNA degradosome acts in concert with the flanking RNA binding sites to facilitate the degradation of the T7-lacZ mRNA. RhlB appears to be acting in a pathway involving the endoribonucleolytic activity of RNase E. This in vivo work is the first time that RhlB has been implicated in a functional interaction with the catalytic activity of RNase E.

In a double-mutant strain in which the genes encoding both RhlB and PAP have been disrupted, the degradation of mRNA fragments containing REP elements was severely impaired (35). These mRNA fragments appeared to be completely stable. In single-mutant strains, defects in REP mRNA degradation were detectable but less severe than in the double-mutant strain. This result suggests that RhlB and PAP are acting in separate pathways because both enzymes need to be inactivated to completely block REP mRNA degradation. Further work showed that RhlB and PNase are in the same pathway and that both enzymes need to be bound to RNase E as part of the RNA degradosome for this pathway to work in vivo. This is the first work to demonstrate a requirement for RNA degradosome assembly for the activity of an mRNA-degrading pathway. This work concurs with previous in vitro results showing that RhlB and PNase in the absence of the protein scaffold of RNase E cannot degrade mRNA fragments containing REP elements (17). PAP and the exoribonuclease RNase R form the second pathway involved in the degradation of mRNA containing REP elements (15, 35). Taken together, this work shows that RhlB and PAP act in separate, parallel pathways. The PAP–RNase R pathway works independently of the RNA degradosome.

The work presented in this section shows that RhlB as a component of the RNA degradosome acts in vivo in different pathways, one involving RNase E and the other involving PNase. The function of both pathways

requires the assembly of the RNA degradosome. The dynamic nature of the natively unstructured protein in the noncatalytic region of RNase E could be important for the capacity of RhlB to function in both pathways. It should be interesting to test this idea in future work on the RNA degradosome.

**AN RhlB-PNase COMPLEX IN THE ABSENCE OF THE PROTEIN SCAFFOLD OF RNase E**

Lin-Chao and colleagues (45, 47) have recently published biochemical, two-hybrid, and immunoprecipitation experiments showing that RhlB can interact directly with PNase. An antibody against RhlB can coimmunoprecipitate PNase in a strain with RNase E lacking the protein scaffold. Lin-Chao and colleagues have inferred from these results that RhlB is the β-subunit of PNase, a 48-kDa polypeptide that copurified with PNase in certain purification procedures. However, the interaction in vitro of RhlB with PNase is weaker than the interaction of RhlB with RNase E (47), and the coimmunoprecipitation of PNase and RhlB is much more efficient in a strain containing intact RNase E (45, 79).

The claim that RhlB is the β-subunit of PNase is in direct conflict with previously published work. During the purification and characterization of the RNA degradosome, I obtained a sample of PNase containing the β-subunit from Claude Portier (67). This preparation was compared side-by-side with the RNA degradosome preparation. In gel electrophoresis, the 48-kDa protein in the PNase preparation comigrated with the 48-kDa protein in the RNA degradosome preparation, which is now known to be enolase (9, 71). Furthermore, the 48-kDa protein from each preparation was analyzed by protein fingerprinting using partial proteolysis with several different proteases. The β-subunit and enolase were identical (9). However, no direct physical

interaction between enolase and PNPase was detected in subsequent work and a possible explanation of this apparent contradiction was suggested previously (79). In preparations of PNPase containing the β-subunit, the noncatalytic region of RNase E was likely degraded to proteolytic fragments. Enolase is the closest protein to PNPase on the scaffold and the two proteins could have been tethered by proteolytic fragments. The identity of the β-subunit of PNPase is mainly of historical interest. The question of whether there is an RhlB-PNPase pathway in vivo that functions independently of the RNase E protein scaffold is, however, an issue that needs to be addressed.

**THE PHYLOGENETIC DISTRIBUTION OF RNase E AND THE EVOLUTION OF THE RNA DEGRADOSOME**

In *E. coli*, there is significant sequence conservation between RNase G and the catalytic core of RNase E. The difference between the two enzymes is that RNase G lacks the extended noncatalytic region found in RNase E. These enzymes are the founders of the RNase E/G family of ribonucleases distributed throughout the Bacteria (7, 19, 42). In the Proteobacteria, most of the bacteria whose genomes have been completely sequenced contain identifiable orthologs of RNase E and RNase G. Outside of the Proteobacteria, many, but not all, bacteria have ribonucleases that are members of the RNase E/G family. However, these organisms often encode a single gene and the longer RNase E-like form is not always present. For example, *Synechocystis*, a cyanobacteria, encodes a short form of the enzyme, whereas *Streptomyces coelicolor*, an actinobacteria, encodes an unusual form in which the catalytic domain is flanked by long N- and C-terminal extensions (33, 42).

A striking feature of RNase E within the Proteobacteria is that, although the catalytic domain of the enzyme is well conserved, the noncatalytic region is highly plastic in both primary sequence and length (50). Furthermore, to the limited extent that the RNA degradosome has been characterized in other Proteobacteria, its composition is variable. That is, *Pseudomonas syringae* has a multiprotein complex containing RNase E, RNase R, and a DEAD-box protein; *Rhodobacter capsulatus* has a multiprotein complex containing RNase E, two DEAD-box proteins, and the Rho transcription termination factor (32, 69). Thus, the phylogenetic plasticity of the noncatalytic region of RNase E appears to be associated with a functional plasticity in the composition of the RNA degradosome. Within the γ-Proteobacteria, the noncatalytic region of the RNase E of *E. coli* and related enterobacteria contains a set of conserved, short-sequence motifs that correspond to the sites for binding RhlB, enolase, and PNPase (12, 50). Thus, an *E. coli*-like RNA degradosome appears to be conserved in the enterobacteria.

Taken together, these observations suggest that most Proteobacteria contain an RNase E-based multiprotein complex. Nonetheless, there is only a limited sequence conservation of the noncatalytic region among the enterobacteria. In addition to the DEAD-box proteins, RNase R and the Rho transcription termination factor have been identified as components of the RNA degradosome in Proteobacteria distantly related to *E. coli*. Given that only two bacteria other than *E. coli* have been analyzed so far, this list could be much longer. The plasticity of the noncatalytic region is likely related to its natively unfolded structure (50). The emerging story suggests a remarkable conservation of RNA degradosome-like complexes but a surprising variability in composition. It is likely that the central role of the RNA degradosome in the posttranscriptional regulation of gene expression and the remodeling of this complex during evolution contributes to the adaptation of bacteria to the enormously wide variety of niches in which they live.

SUMMARY POINTS

1. The noncatalytic region of *E. coli* RNase E is a large 500-amino-acid stretch of natively unstructured protein with interspersed regions involved in RNA binding and protein-protein interactions with the other components of the RNA degradosome.
2. The principal form of the RNA degradosome in *E. coli* contains PNPAse, RhlB, and enolase, although recent work from several different studies suggests that alternative RNase E–based complexes exist under certain growth conditions.
3. RNase E–based multiprotein complexes are likely to be found throughout the Proteobacteria, although the composition of the associated proteins appears to be variable.

DISCLOSURE STATEMENT

The author is not aware of any biases that might be perceived as affecting the objectivity of this review.

ACKNOWLEDGMENTS

I would like to acknowledge all my collaborators who have participated in the identification and characterization of the RNA degradosome and in particular the PhD students who have worked in my group over the past decade. I thank I. Iost (Paris) and M. Dreyfus (Paris) for critical reading of this manuscript. My group is supported by the Centre National de la Recherche Scientifique with additional funding from the Agence Nationale de la Recherche (NT05_1-44659).

LITERATURE CITED

1. Bernstein JA, Khodursky AB, Lin PH, Lin-Chao S, Cohen SN. 2002. Global analysis of mRNA decay and abundance in *Escherichia coli* at single-gene resolution using two-color fluorescent DNA microarrays. *Proc. Natl. Acad. Sci. USA* 99:9697–702
2. Bernstein JA, Lin PH, Cohen SN, Lin-Chao S. 2004. Global analysis of *Escherichia coli* RNA degradosome function using DNA microarrays. *Proc. Natl. Acad. Sci. USA* 101:2758–63
3. Blum E, Py B, Carpousis AJ, Higgins CF. 1997. Polyphosphate kinase is a component of the *Escherichia coli* RNA degradosome. *Mol. Microbiol.* 26:387–98
4. Butland G, Peregrin-Alvarez JM, Li J, Yang W, Yang X, et al. 2005. Interaction network containing conserved and essential protein complexes in *Escherichia coli*. *Nature* 433:531–37
5. Callaghan AJ, Aurikko JP, Ilag LL, Gunter Grossmann J, Chandran V, et al. 2004. Studies of the RNA degradosome-organizing domain of the *Escherichia coli* ribonuclease RNase E. *J. Mol. Biol.* 340:965–79
6. Callaghan AJ, Marcaida MJ, Stead JA, McDowall KJ, Scott WG, Luisi BF. 2005. Structure of *Escherichia coli* RNase E catalytic domain and implications for RNA turnover. *Nature* 437:1187–91
7. Carpousis AJ. 2002. The *Escherichia coli* RNA degradosome: structure, function and relationship to other ribonucleolytic multienzyme complexes. *Biochem. Soc. Trans.* 30:150–55
8. Carpousis AJ. 2003. Degradation of targeted mRNAs in *Escherichia coli*: regulation by a small antisense RNA. *Genes Dev.* 17:2351–55

9. Carpousis AJ, Van Houwe G, Ehretsmann C, Krisch HM. 1994. Copurification of *E. coli* RNase E and PNase: evidence for a specific association between two enzymes important in RNA processing and degradation. *Cell* 76:889–900

10. Carpousis AJ, Vanzo NF, Raynal LC. 1999. mRNA degradation: a tale of poly(A) and multiprotein machines. *Trends Genet.* 15:24–28

11. Causton H, Py B, McLaren RS, Higgins CF. 1994. mRNA degradation in *Escherichia coli*: a novel factor which impedes the exoribonucleolytic activity of PNase at stem-loop structures. *Mol. Microbiol.* 14:731–41

12. Chandran V, Luisi BF. 2006. Recognition of enolase in the *Escherichia coli* degradosome. *J. Mol. Biol.* 358:8–15

12a. Chandran V, Poljak L, Vanzo NF, Leroy A, Miguel RN, et al. 2007. Recognition and cooperation between the ATP-dependent RNA helicase RhlB and ribonuclease RNase E. *J. Mol. Biol.* 367:113–32

13. Charollais J, Dreyfus M, Iost I. 2004. CsdA, a cold-shock RNA helicase from *Escherichia coli*, is involved in the biogenesis of 50S ribosomal subunit. *Nucleic Acids Res.* 32:2751–59

14. Charollais J, Pflieger D, Vinh J, Dreyfus M, Iost I. 2003. The DEAD-box RNA helicase SrmB is involved in the assembly of 50S ribosomal subunits in *Escherichia coli*. *Mol. Microbiol.* 48:1253–65

15. Cheng ZF, Deutscher MP. 2005. An important role for RNase R in mRNA decay. *Mol. Cell* 17:313–18

16. Coburn GA, Mackie GA. 1999. Degradation of mRNA in *Escherichia coli*: an old problem with some new twists. *Prog. Nucleic Acid Res. Mol. Biol.* 62:55–108

17. Coburn GA, Miao X, Briant DJ, Mackie GA. 1999. Reconstitution of a minimal RNA degradosome demonstrates functional coordination between a 3′ exonuclease and a DEAD-box RNA helicase. *Genes Dev.* 13:2594–603

18. Cohen SN. 1995. Surprises at the 3′ end of prokaryotic RNA. *Cell* 80:829–32

19. Condon C, Putzer H. 2002. The phylogenetic distribution of bacterial ribonucleases. *Nucleic Acids Res.* 30:5339–46

20. Deana A, Belasco JG. 2004. The function of RNase G in *Escherichia coli* is constrained by its amino and carboxyl termini. *Mol. Microbiol.* 51:1205–17

21. Deutscher MP. 2006. Degradation of RNA in bacteria: comparison of mRNA and stable RNA. *Nucleic Acids Res.* 34:659–66

22. Dreyfus M, Regnier P. 2002. The poly(A) tail of mRNAs: bodyguard in eukaryotes, scavenger in bacteria. *Cell* 111:611–13

23. Ehretsmann CP, Carpousis AJ, Krisch HM. 1992. Specificity of *Escherichia coli* endoribonuclease RNase E: in vivo and in vitro analysis of mutants in a bacteriophage T4 mRNA processing site. *Genes Dev.* 6:149–59

24. Gao J, Lee K, Zhao M, Qiu J, Zhan X, et al. 2006. Differential modulation of *E. coli* mRNA abundance by inhibitory proteins that alter the composition of the degradosome. *Mol. Microbiol.* 61:394–406

25. Gottesman S. 2004. The small RNA regulators of *Escherichia coli*: roles and mechanisms. *Annu. Rev. Microbiol.* 58:303–28

26. Grunberg-Manago M. 1999. Messenger RNA stability and its role in control of gene expression in bacteria and phages. *Annu. Rev. Genet.* 33:193–227

27. Grunberg-Manago M, Ortiz PJ, Ochoa S. 1955. Enzymatic synthesis of nucleic acidlike polynucleotides. *Science* 122:907–10

28. Hajnsdorf E, Braun F, Haugel-Nielsen J, Regnier P. 1995. Polyadenylation destabilizes the rpsO mRNA of *Escherichia coli*. *Proc. Natl. Acad. Sci. USA* 92:3973–77

29. Hajnsdorf E, Regnier P. 2000. Host factor Hfq of *Escherichia coli* stimulates elongation of poly(A) tails by poly(A) polymerase I. *Proc. Natl. Acad. Sci. USA* 97:1501–5

30. Iost I, Dreyfus M. 1995. The stability of *Escherichia coli lacZ* mRNA depends upon the simultaneity of its synthesis and translation. *EMBO J.* 14:3252–61

31. Iost I, Dreyfus M. 2006. DEAD-box RNA helicases in *Escherichia coli*. *Nucleic Acids Res.* 34:4189–97

32. Jäger S, Fuhrmann O, Heck C, Hebermehl M, Schiltz E, et al. 2001. An mRNA degrading complex in *Rhodobacter capsulatus*. *Nucleic Acids Res.* 29:4581–88

33. Kaberdin VR, Miczak A, Jakobsen JS, Lin-Chao S, McDowall KJ, von Gabain A. 1998. The endoribonucleolytic N-terminal half of *Escherichia coli* RNase E is evolutionarily conserved in *Synechocystis* sp. and other bacteria but not the C-terminal half, which is sufficient for degradosome assembly. *Proc. Natl. Acad. Sci. USA* 95:11637–42

34. Karginov FV, Uhlenbeck OC. 2004. Interaction of *Escherichia coli* DbpA with 23S rRNA in different functional states of the enzyme. *Nucleic Acids Res.* 32:3028–32

35. Khemici V, Carpousis AJ. 2004. The RNA degradosome and poly(A) polymerase of *Escherichia coli* are required in vivo for the degradation of small mRNA decay intermediates containing REP-stabilizers. *Mol. Microbiol.* 51:777–90

36. Khemici V, Poljak L, Toesca I, Carpousis AJ. 2005. Evidence in vivo that the DEAD-box RNA helicase RhlB facilitates the degradation of ribosome-free mRNA by RNase E. *Proc. Natl. Acad. Sci. USA* 102:6913–18

37. Khemici V, Toesca I, Poljak L, Vanzo NF, Carpousis AJ. 2004. The RNase E of *Escherichia coli* has at least two binding sites for DEAD-box RNA helicases: functional replacement of RhlB by RhlE. *Mol. Microbiol.* 54:1422–30

38. Kido M, Yamanaka K, Mitani T, Niki H, Ogura T, Hiraga S. 1996. RNase E polypeptides lacking a carboxyl-terminal half suppress a *mukB* mutation in *Escherichia coli*. *J. Bacteriol.* 178:3917–25

39. Kuhnle K, Luisi BF. 2001. Crystal structure of the *Escherichia coli* RNA degradosome component enolase. *J. Mol. Biol.* 313:583–92

40. Kushner SR. 2002. mRNA decay in *Escherichia coli* comes of age. *J. Bacteriol.* 184:4658–65

41. Lee K, Bernstein JA, Cohen SN. 2002. RNase G complementation of *rne* null mutation identifies functional interrelationships with RNase E in *Escherichia coli*. *Mol. Microbiol.* 43:1445–56

42. Lee K, Cohen SN. 2003. A *Streptomyces coelicolor* functional orthologue of *Escherichia coli* RNase E shows shuffling of catalytic and PNPase-binding domains. *Mol. Microbiol.* 48:349–60

43. Lee K, Zhan X, Gao J, Qiu J, Feng Y, et al. 2003. RraA: a protein inhibitor of RNase E activity that globally modulates RNA abundance in *E. coli*. *Cell* 114:623–34

44. Leroy A, Vanzo NF, Sousa S, Dreyfus M, Carpousis AJ. 2002. Function in *Escherichia coli* of the noncatalytic part of RNase E: role in the degradation of ribosome-free mRNA. *Mol. Microbiol.* 45:1231–43

45. Lin PH, Lin-Chao S. 2005. RhlB helicase rather than enolase is the beta-subunit of the *Escherichia coli* polynucleotide phosphorylase (PNPase)-exoribonucleolytic complex. *Proc. Natl. Acad. Sci. USA* 102:16590–95

46. Linder P. 2006. Dead-box proteins: a family affair—active and passive players in RNP-remodeling. *Nucleic Acids Res.* 34:4168–80

47. Liou GG, Chang HY, Lin CS, Lin-Chao S. 2002. DEAD box RhlB RNA helicase physically associates with exoribonuclease PNPase to degrade double-stranded

RNA independent of the degradosome-assembling region of RNase E. *J. Biol. Chem.* 277:41157–62

48. Lopez PJ, Marchand I, Joyce SA, Dreyfus M. 1999. The C-terminal half of RNase E, which organizes the *Escherichia coli* degradosome, participates in mRNA degradation but not rRNA processing in vivo. *Mol. Microbiol.* 33:188–99

49. Makarova OV, Makarov EM, Sousa R, Dreyfus M. 1995. Transcribing of *Escherichia coli* genes with mutant T7 RNA polymerases: stability of lacZ mRNA inversely correlates with polymerase speed. *Proc. Natl. Acad. Sci. USA* 92:12250–54

50. Marcaida MJ, DePristo MA, Chandran V, Carpousis AJ, Luisi BF. 2006. The RNA degradosome: life in the fast lane of adaptive molecular evolution. *Trends Biochem. Sci.* 31:359–65

51. Marujo PE, Braun F, Haugel-Nielsen J, Le Derout J, Arraiano CM, Regnier P. 2003. Inactivation of the decay pathway initiated at an internal site by RNase E promotes poly(A)-dependent degradation of the rpsO mRNA in *Escherichia coli*. *Mol. Microbiol.* 50:1283–94

52. Masse E, Escorcia FE, Gottesman S. 2003. Coupled degradation of a small regulatory RNA and its mRNA targets in *Escherichia coli*. *Genes Dev.* 17:2374–83

53. McLaren RS, Newbury SF, Dance GS, Causton HC, Higgins CF. 1991. mRNA degradation by processive 3′-5′ exoribonucleases in vitro and the implications for prokaryotic mRNA decay in vivo. *J. Mol. Biol.* 221:81–95

54. Miczak A, Kaberdin VR, Wei CL, Lin-Chao S. 1996. Proteins associated with RNase E in a multicomponent ribonucleolytic complex. *Proc. Natl. Acad. Sci. USA* 93:3865–69

55. Mohanty BK, Kushner SR. 2000. Polynucleotide phosphorylase functions both as a 3′→5′ exonuclease and a poly(A) polymerase in *Escherichia coli*. *Proc. Natl. Acad. Sci. USA* 97:11966–71

56. Mohanty BK, Kushner SR. 2003. Genomic analysis in *Escherichia coli* demonstrates differential roles for polynucleotide phosphorylase and RNase II in mRNA abundance and decay. *Mol. Microbiol.* 50:645–58

57. Mohanty BK, Kushner SR. 2006. The majority of *Escherichia coli* mRNAs undergo post-transcriptional modification in exponentially growing cells. *Nucleic Acids Res.* 34:5695–704

58. Mohanty BK, Maples VF, Kushner SR. 2004. The Sm-like protein Hfq regulates polyadenylation dependent mRNA decay in *Escherichia coli*. *Mol. Microbiol.* 54:905–20

59. Morita T, Kawamoto H, Mizota T, Inada T, Aiba H. 2004. Enolase in the RNA degradosome plays a crucial role in the rapid decay of glucose transporter mRNA in the response to phosphosugar stress in *Escherichia coli*. *Mol. Microbiol.* 54:1063–75

60. Morita T, Maki K, Aiba H. 2005. RNase E-based ribonucleoprotein complexes: mechanical basis of mRNA destabilization mediated by bacterial noncoding RNAs. *Genes Dev.* 19:2176–86

61. Morita T, Mochizuki Y, Aiba H. 2006. Translational repression is sufficient for gene silencing by bacterial small noncoding RNAs in the absence of mRNA destruction. *Proc. Natl. Acad. Sci. USA* 103:4858–63

62. Newbury SF, Smith NH, Robinson EC, Hiles ID, Higgins CF. 1987. Stabilization of translationally active mRNA by prokaryotic REP sequences. *Cell* 48:297–310

63. Ow MC, Kushner SR. 2002. Initiation of tRNA maturation by RNase E is essential for cell viability in *E. coli*. *Genes Dev.* 16:1102–15

64. Ow MC, Liu Q, Kushner SR. 2000. Analysis of mRNA decay and rRNA processing in *Escherichia coli* in the absence of RNase E-based degradosome assembly. *Mol. Microbiol.* 38:854–66

65. Ow MC, Perwez T, Kushner SR. 2003. RNase G of *Escherichia coli* exhibits only limited functional overlap with its essential homologue, RNase E. *Mol. Microbiol.* 49:607–22

66. Perwez T, Kushner SR. 2006. RNase Z in *Escherichia coli* plays a significant role in mRNA decay. *Mol. Microbiol.* 60:723–37

67. Portier C. 1975. Quaternary structure of polynucleotide phosphorylase from *Escherichia coli*: evidence of a complex between two types of polypeptide chains. *Eur. J. Biochem.* 55:573–82

68. Prud'homme-Genereux A, Beran RK, Iost I, Ramey CS, Mackie GA, Simons RW. 2004. Physical and functional interactions among RNase E, polynucleotide phosphorylase and the cold-shock protein, CsdA: evidence for a ‘cold shock degradosome’. *Mol. Microbiol.* 54:1409–21

69. Purusharth RI, Klein F, Sulthana S, Jager S, Jagannadham MV, et al. 2005. Exoribonuclease R interacts with endoribonuclease E and an RNA helicase in the psychrotrophic bacterium *Pseudomonas syringae* Lz4W. *J. Biol. Chem.* 280:14572–78

70. Py B, Causton H, Mudd EA, Higgins CF. 1994. A protein complex mediating mRNA degradation in *Escherichia coli*. *Mol. Microbiol.* 14:717–29

71. Py B, Higgins CF, Krisch HM, Carpousis AJ. 1996. A DEAD-box RNA helicase in the *Escherichia coli* RNA degradosome. *Nature* 381:169–72

72. Raynal LC, Carpousis AJ. 1999. Poly(A) polymerase I of *Escherichia coli*: characterization of the catalytic domain, an RNA binding site and regions for the interaction with proteins involved in mRNA degradation. *Mol. Microbiol.* 32:765–75

72a. Regonesi ME, Del Favero M, Basilico F, Briani F, Benazzi L, et al. 2006. Analysis of the *Escherichia coli* RNA degradosome composition by a proteomic approach. *Biochimie* 88:151–61

73. Stickney LM, Hankins JS, Miao X, Mackie GA. 2005. Function of the conserved S1 and KH domains in polynucleotide phosphorylase. *J. Bacteriol.* 187:7214–21

74. Symmons MF, Jones GH, Luisi BF. 2000. A duplicated fold is the structural basis for polynucleotide phosphorylase catalytic activity, processivity, and regulation. *Struct. Fold. Des.* 8:1215–26

75. Symmons MF, Williams MG, Luisi BF, Jones GH, Carpousis AJ. 2002. Running rings around RNA: a superfamily of phosphate-dependent RNases. *Trends Biochem. Sci.* 27:11–18

75a. Taghbalout A, Rothfield L. 2007. RNaseE and the other constituents of the RNA degradosome are components of the bacterial cytoskeleton. *Proc. Natl. Acad. Sci. USA* 104:1667–72

76. Tamura M, Lee K, Miller CA, Moore CJ, Shirako Y, et al. 2006. RNase E maintenance of proper FtsZ/FtsA ratio required for nonfilamentous growth of *Escherichia coli* cells but not for colony-forming ability. *J. Bacteriol.* 188:5145–52

77. Tanner NK, Linder P. 2001. DExD/H box RNA helicases: from generic motors to specific dissociation functions. *Mol. Cell* 8:251–62

78. Tompa P. 2002. Intrinsically unstructured proteins. *Trends Biochem. Sci.* 27:527–33

79. Vanzo NF, Li YS, Py B, Blum E, Higgins CF, et al. 1998. Ribonuclease E organizes the protein interactions in the *Escherichia coli* RNA degradosome. *Genes Dev.* 12:2770–81

80. Wagner EG, Altuvia S, Romby P. 2002. Antisense RNAs in bacteria and their genetic elements. *Adv. Genet.* 46:361–98

Annual Review of  
Microbiology  

Volume 61, 2007  

Contents  

Frontispiece  
Margarita Salas …………xiv  

40 Years with Bacteriophage φ29  
Margarita Salas …………1  

The Last Word: Books as a Statistical Metaphor for Microbial Communities  
Patrick D. Schloss and Jo Handelsman …………23  

The Mechanism of Isoniazid Killing: Clarity Through the Scope  
of Genetics  
Catherine Vilchèze and William R. Jacobs, Jr. …………35  

Development of a Combined Biological and Chemical Process for  
Production of Industrial Aromatics from Renewable Resources  
F. Sima Sariaslani …………51  

The RNA Degradosome of Escherichia coli: An mRNA-Degrading  
Machine Assembled on RNase E  
Agamemnon J. Carpousis …………71  

Protein Secretion in Gram-Negative Bacteria via the Autotransporter  
Pathway  
Nathalie Dautin and Harris D. Bernstein …………89  

Chlorophyll Biosynthesis in Bacteria: The Origins of Structural and  
Functional Diversity  
Aline Gomez Maqueo Chew and Donald A. Bryant …………113  

Roles of Cyclic Diguanylate in the Regulation of Bacterial Pathogenesis  
Rita Tamayo, Jason T. Pratt, and Andrew Camilli …………131  

Aggresomes and Pericentriolar Sites of Virus Assembly:  
Cellular Defense or Viral Design?  
Thomas Wileman …………149  

As the Worm Turns: The Earthworm Gut as a Transient Habitat for  
Soil Microbial Biomes  
Harold L. Drake and Marcus A. Horn …………169  

Biogenesis of the Gram-Negative Bacterial Outer Membrane  
Martine P. Bos, Viviane Robert, and Jan Tommassen ........................... 191  

SigB-Dependent General Stress Response in Bacillus subtilis and  
Related Gram-Positive Bacteria  
Michael Hecker, Jan Pané-Farré, and Uwe Völker .......................... 215  

Ecology and Biotechnology of the Genus Shewanella  
Heidi H. Hau and Jeffrey A. Gralnick ................................. 237  

Nonhomologous End-Joining in Bacteria: A Microbial Perspective  
Robert S. Pitcher, Nigel C. Brissett, and Aidan J. Doherty ............... 259  

Postgenomic Adventures with Rhodobacter sphaeroides  
Chris Mackenzie, Jesus M. Eraso, Madhusudan Choudhary, Jung Hyeob Roh,  
Xiaohua Zeng, Patrice Bruscella, Ágnes Puskás, and Samuel Kaplan ......... 283  

Toward a Hyperstructure Taxonomy  
Vic Norris, Tanneke den Blaauwen, Roy H. Doi, Rasika M. Harshey,  
Laurent Janniere, Alfonso Jiménez-Sánchez, Ding Jun Jin,  
Petra Anne Levin, Eugenia Mileykovskaya, Abraham Minsky,  
Gradimir Misovic, Camille Ripoll, Milton Saier, Jr., Kirsten Skarstad,  
and Michel Thellier .................................................... 309  

Endolithic Microbial Ecosystems  
Jeffrey J. Walker and Norman R. Pace ................................. 331  

Nitrogen Regulation in Bacteria and Archaea  
John A. Leigh and Jeremy A. Dodsworth .............................. 349  

Microbial Metabolism of Reduced Phosphorus Compounds  
Andrea K. White and William W. Metcalf ............................. 379  

Biofilm Formation by Plant-Associated Bacteria  
Thomas Danhorn and Clay Fuqua ...................................... 401  

Heterotrimeric G Protein Signaling in Filamentous Fungi  
Liande Li, Sara J. Wright, Svetlana Krystofova, Gyungsoon Park,  
and Katherine A. Borkovich ........................................... 423  

Comparative Genomics of Protists: New Insights into the Evolution  
of Eukaryotic Signal Transduction and Gene Regulation  
Vivek Anantharaman, Lakshminarayan M. Iyer, and L. Aravind ............. 453  

Lantibiotics: Peptides of Diverse Structure and Function  
Joanne M. Willey and Wilfred A. van der Donk ........................ 477  

The Impact of Genome Analyses on Our Understanding of  
Ammonia-Oxidizing Bacteria  
Daniel J. Arp, Patrick S.G. Chain, and Martin G. Klotz ................ 503  

Morphogenesis in Candida albicans  
Malcolm Whiteway and Catherine Bachewich 529  

Structure, Assembly, and Function of the Spore Surface Layers  
Adriano O. Henriques and Charles P. Moran, Jr. 555  

Cytoskeletal Elements in Bacteria  
Peter L. Graumann 589  

Indexes  

Cumulative Index of Contributing Authors, Volumes 57–61 619  

Cumulative Index of Chapter Titles, Volumes 57–61 622  

Errata  

An online log of corrections to Annual Review of Microbiology articles may be found  
at http://micro.annualreviews.org/
