
# Unusual sugar biosynthesis and natural product glycodiversification

Christopher J. Thibodeaux${}^{1}$, Charles E. Melançon${}^{2}$ & Hung-wen Liu${}^{1,2,3}$

The enzymes involved in the biosynthesis of carbohydrates and the attachment of sugar units to biological acceptor molecules catalyse an array of chemical transformations and coupling reactions. In prokaryotes, both common sugar precursors and their enzymatically modified derivatives often become constituents of biologically active natural products through the action of glycosyltransferases. Recently, researchers have begun to harness the power of these biological catalysts to alter the sugar structures and glycosylation patterns of natural products both *in vivo* and *in vitro*. Biochemical and structural studies of sugar biosynthetic enzymes and glycosyltransferases, coupled with advances in bioengineering methodology, have ushered in a new era of drug development.

Given the numerous physiological roles of carbohydrates, which encompass many aspects of normal cellular function and survival, it is not surprising that both monosaccharide and polysaccharide structures vary markedly throughout nature, providing the chemical diversity required to fulfil a vast range of biological tasks. Two general strategies are used by nature to expand the diversity of carbohydrate structure and function, a process referred to as glycodiversification. In one strategy, an unusual sugar structure can be generated through enzymatic modification of the functional groups of a common sugar precursor. This mode of diversification exists in all living organisms but is most elaborate in prokaryotes, which manufacture highly modified sugars as building blocks for cell wall and secondary metabolite biosynthesis. The second strategy is to assemble different glycoforms by altering the attachment of activated sugar substrates to ‘acceptor’ molecules such as natural product aglycones, proteins or other sugars — a feat performed by glycosyltransferases. In this case, the identity of the sugar, the number of sugar moieties, or the regiochemistry or stereochemistry of sugar attachment to the acceptor molecule can all be varied to generate structural diversity. Thus, the abundance of natural sugar structures is ultimately derived from the activities of the enzymes responsible for sugar biosynthesis and attachment.

In this review we summarize some of the strategies used by sugar biosynthetic enzymes to generate these myriad carbohydrate structures. Special emphasis is given to the enzymes of bacterial deoxysugar biosynthesis, both because these sugars exhibit abundant structural modifications not seen in eukaryotic sugars and because many of these sugars become appendages on biologically active natural products${}^{1,2}$. We also discuss the structure and mechanisms of glycosyltransferases, which ultimately determine the feasibility of natural product enzymatic glycodiversification. Finally, we highlight recent progress in the fields of combinatorial biosynthesis and chemoenzymatic/enzymatic synthesis, and demonstrate how these bioengineering methods have been successfully used to alter the sugar components of natural products.

## Monosaccharide activation

In order to be used by the biosynthetic enzymes and glycosyltransferases within a cell, monosaccharides must first be activated as either nucleoside monophosphate (NMP) or nucleoside diphosphate (NDP) derivatives. Most NDP-sugars (such as NDP-α-D-glucose, 3, Fig. 1) originate from primary metabolic intermediates such as fructose-6-phosphate and glucose-6-phosphate (1). Activation typically involves the coupling of a sugar-1-phosphate (2) to an NMP moiety from the corresponding nucleotide triphosphate (NTP) by a nucleotidylyltransferase${}^{3}$. The sugar-1-phosphate is either the product of an anomer kinase reaction or is generated from a sugar-6-phosphate by an appropriate mutase. Interestingly, only ten NDP-sugars are used as the basic construction units for most biosynthetic glycosylation reactions in eukaryotic cells${}^{4}$.

In contrast to eukaryotes, the structures of prokaryotic NDP-sugars are much more diverse (Fig. 1). Because most unusual sugars are 6-deoxyhexoses${}^{5}$, the immediate biosynthetic precursor of most deoxysugars is an NDP-4-keto-6-deoxy-D-hexose (usually NDP-4-keto-6-deoxy-α-D-glucose, 4). From this common intermediate, different combinations of oxidation, reduction, deoxygenation, epimerization, isomerization, group-transfer and rearrangement reactions yield the vast array of diverse NDP-sugar structures observed across the prokaryotic kingdom. In most cases, the enzymes involved in these transformations have only recently been identified and mechanistically characterized${}^{5-7}$. Importantly, there is often a high degree of amino-acid sequence similarity among enzymes catalysing similar reactions in separate deoxysugar biosynthetic pathways. This has facilitated the identification and characterization of deoxysugar biosynthetic loci, and has also enabled the pathway engineering studies discussed below.

### Diverse mechanisms for unusual sugar biosynthesis

Although many sugar biosynthetic enzymes have been studied, those involved in deoxygenation are among the most mechanistically diverse and intriguing known${}^{6-9}$, and thus deserve special attention. The specific C-O bond cleavage strategy used by each type of deoxygenating enzyme in the biosynthesis of deoxyhexoses is determined by the location of the scissile C–O bond (either α or β) relative to an essential 4-keto group on the sugar ring${}^{9}$. As depicted in Fig. 2a, C6 deoxygenation is catalysed by an NDP-hexose-4,6-dehydratase (E${}_{od}$), which belongs to

---

${}^{1}$ Institute for Cellular and Molecular Biology, 1 University Station A4810, University of Texas at Austin, Austin, Texas 78712, USA. ${}^{2}$ Department of Chemistry and Biochemistry, 1 University Station A5300, University of Texas at Austin, Austin, Texas 78712, USA. ${}^{3}$ Division of Medicinal Chemistry, College of Pharmacy, 1 University Station A1935, University of Texas at Austin, Austin, Texas 78712, USA.

the short-chain dehydrogenase/reductase (SDR) family and relies on a tightly bound NAD⁺ cofactor to oxidize the hydroxyl group at C4 of NDP-hexose (such as NDP-α-D-glucose, 3) to give a 4-ketosugar intermediate (5). The C4 oxidation activates the C5 proton for abstraction, allowing elimination of a water molecule across the C5–C6 bond to form 6. Hydride transfer from the transient NADH to C6 of 6 leads to the NDP-4-keto-6-deoxy-α-D-glucose product (4). The installation of a keto group at C4 provides the necessary ring activation for subsequent deoxygenation at C2, C3 and C4, as well as many other transformations in sugar biosynthesis.

A similar reaction sequence (namely, α-H⁺ abstraction followed by β-C–O bond cleavage) occurs in the C2 deoxygenation catalysed by TylX3 in the TDP-L-mycarose (7, Fig. 1) biosynthetic pathway of Streptomyces fradiae (Fig. 2b). Here, the 4-keto group of 8 helps lower the pKa of the C3 proton, facilitating deprotonation at C3 and the elimination of the C2 hydroxyl group to afford 10. Zn²⁺, which is required for catalysis by TylX3, may have a role in C3 deprotonation by activating an active-site-bound water molecule as a base. Alternatively, it may stabilize the enediolate intermediate (9) or act as a general acid in the elimination of the C2 hydroxyl group (9 → 10). Tautomerization of 10 yields the unstable 3,4-diketone 11, which is reduced by the next enzyme in the pathway, the NADPH-dependent 3-ketoreductase, TylC1, to give the 2,6-dideoxyhexose product 12.

The mechanism of C3 deoxygenation in the biosynthesis of 3,6-dideoxysugars (such as 13, Fig. 1) is more complex than C2 and C6 deoxygenation because the scissile C–O bond is adjacent to the 4-keto group. Thus, C–O bond cleavage cannot be triggered by an initial α-H⁺ abstraction event. This reaction is catalysed by the combined action of two enzymes, a 3-dehydrase (E₁) and a reductase (E₃), originally isolated from the CDP-L-ascarylose (13) biosynthetic pathway of Yersinia pseudotuberculosis (Fig. 2c). Catalysis is initiated by the formation of a Schiff base between the 4-keto group of 14 and the pyridoxamine 5′-phosphate (PMP) cofactor of E₁, followed by deprotonation at C4′ and 1,4-dehydration to give 15 as the intermediate. Subsequent reduction catalysed by E₃ through two one-electron transfers from NADH to generate a radical intermediate 16 is mediated by a relay system consisting of the FAD and [2Fe–2S] centre in E₃ and a [2Fe–2S] centre in E₁. Hydrolysis of the reduced species 17 gives the 3,6-dideoxyhexose product 18 and regenerates PMP. The elaborate and complex reaction performed by E₁/E₃ and its homologues in other unusual sugar pathways is unprecedented in enzyme catalysis.

Recently, the mechanism of C4 deoxygenation catalysed by the enzymes DesI and DesII during the biosynthesis of TDP-D-desosamine (19, Fig. 1) from Streptomyces venezuelae has been explored. The aminotransferase DesI catalyses the conversion of 8 to TDP-4-amino-4,6-α-D-dideoxyglucose, whereas DesII, a member of the radical S-adenosyl-L-methionine (SAM) superfamily, is responsible for the subsequent oxidative deamination reaction. Although the DesII reaction is probably initiated by C3 hydrogen abstraction, its mechanistic details remain obscure¹⁰. Cumulatively, these examples demonstrate the elegance and versatility of nature’s sugar biosynthetic machinery. The fact that several mechanisms have evolved for C–O bond cleavage, using enzymes that accept similar substrates but require distinct sets of cofactors, underscores the importance of these modifications in the natural amplification of carbohydrate structural diversity.

### Evolution in unusual sugar biosynthesis

We have noticed that many of the enzymes involved in unusual sugar biosynthesis (including Eod, Fig. 2a) are members of the short-chain dehydrogenase/reductase (SDR) family¹¹. Enzymes in this family use a conserved protein fold to bind a nicotinamide adenine dinucleotide cofactor (either NAD⁺ or NADP⁺, denoted NAD(P)⁺) that has an essential role in catalysis. Interestingly, the nicotinamide cofactor is most often used as a true prosthetic group by the sugar biosynthetic enzymes of the SDR family, rather than as a co-substrate as is typically seen in most NAD(P)-dependent enzymes. Catalysis by these enzymes is always initiated by the oxidation of one of the sugar hydroxyl groups by the NAD(P)⁺ prosthetic group. After the requisite reaction chemistry, the oxidized nicotinamide coenzyme is regenerated at the end of each catalytic cycle (Fig. 3).

UDP-D-galactose-4-epimerase from Escherichia coli¹²,¹³, CDP-D-tyvelose-2-epimerase from Y. pseudotuberculosis¹⁴ and ADP-β-L-glycero-D-manno-heptose-6-epimerase from E. coli¹⁵ each catalyse epimerization of a hydroxyl group at an unactivated position of a hexose, and all three are members of the SDR superfamily (Fig. 3a). After oxidation of the target hydroxyl

Figure 1 | Structural diversity of prokaryotic NDP-sugars derived from NDP-4-keto-6-deoxy-α-D-glucose. NDP-α-D-glucose (3) is generated from common metabolites through the action of hexose kinases, phosphohexose mutases and nucleotidylyltransferases. Most prokaryotic deoxysugars are 6-deoxyhexoses and are synthesized from 3 via an NDP-4-keto-6-deoxy-α-D-glucose intermediate (4), which is the product of an NDP-glucose-4,6-dehydratase reaction — the committed step in 6-deoxysugar biosynthesis. PPi, pyrophosphate.

group by NAD(P)⁺, the ketosugar intermediate of each respective reaction (20, 21 or 22) rotates in the enzyme’s active site such that the opposite face of the sugar keto group becomes exposed to the transient NAD(P)H. Internal hydride return from NAD(P)H reduces the keto group and completes the isomerization. In the reaction catalysed by GDP-D-mannose-3,5-epimerase/reductase from *Arabidopsis thaliana*, oxidation of the C4 hydroxyl facilitates C5 epimerization through active site acid/base chemistry¹⁶ (23 → 24 → 25, Fig. 3b). The epimerized sugar can then be reduced at C4 to give GDP-β-L-gulose (26), or it can be further epimerized at C3 (25 → 27 → 28 → 29, Fig. 3b) before C4 reduction to give GDP-β-L-galactose (30).

The reaction catalysed by another SDR enzyme, UDP-α-D-sulphoquinovose synthase (SQD1) from *A. thaliana*¹⁷, involves the formation of an α,β-unsaturated intermediate similar to 6 in the reaction catalysed by Eₐ (Fig. 2a). However, instead of a nucleophilic hydride attack on C6 of 6, *in vitro* studies have shown that SQD1 uses sulphite as the nucleophile¹⁸. After sulphite addition, NADH reduction of the 4-keto group completes the catalytic cycle to give 31 (Fig. 1). As a final demonstration of the versatility of this class of sugar biosynthetic enzyme, UDP-glucuronic acid decarboxylase has been shown to catalyse the decarboxylation of UDP-glucuronic acid (32) to give UDP-xylose (33, Fig. 3c). Once again, oxidation of the C4 hydroxyl group precedes decarboxylation and facilitates the formation of a transient enolate (34). Stereospecific protonation at C5 (34 → 35) followed by internal hydride return at C4 gives the final product (33)¹⁹.

Cumulatively, the SDR family enzymes illustrate a powerful evolutionary strategy conceived by nature for the synthesis of diverse carbohydrate structures. These enzymes use a common protein fold and have similar cofactor requirements to perform various chemical transformations. Because of the central role of SDR enzymes in carbohydrate biosynthesis, and the availability of X-ray crystal structures of several of these enzymes¹¹,¹³,¹⁶,¹⁷, this family of catalysts is an attractive target for structure-based protein-engineering efforts to construct sugar-modifying enzymes with customized activities.


### Structures and mechanisms of glycosyltransferases

Despite the impressive variety of carbohydrate structures found in nature, most of the glycosyltransferases responsible for the attachment of NDP-sugars to their acceptor molecules fall into one of two structurally distinct families²⁰⁻²². In the glycosyltransferase-A (GT-A) family, the overall enzyme topology consists of separate, but closely associated, NDP-sugar-binding and acceptor-binding domains²³⁻²⁵. The NDP-sugar-binding domain generally contains a conserved Asp-X-Asp amino-acid motif that binds a divalent metal ion (either Mg²⁺ or Mn²⁺); however, some GT-A enzymes do not require this metal ion for activity²⁶,²⁷. When present, the metal ion coordinates the diphosphate moiety of the NDP-sugar and electrostatically stabilizes the developing negative charge on the NDP leaving group during catalysis²⁸. The GT-A family includes most of the eukaryotic glycosyltransferases responsible for glycan biosynthesis in the Golgi and endoplasmic reticulum, as well as many prokaryotic glycosyltransferases²⁹. Members of the GT-B family contain two Rossmann-type domains separated by a deep cleft²⁵,²⁹ and, unlike their counterparts in the GT-A family, the GT-B enzymes characterized so far do not require a metal ion for NDP-sugar binding and catalysis²⁹. Members of the GT-B family are also dispersed throughout nature, but include most prokaryotic glycosyltransferases that glycosylate secondary metabolites and the O-linked β-N-acetylglucosamine (O-GlcNAc) transferase (OGT), which is involved in the glycosylation of cytosolic and nuclear proteins in eukaryotes²⁹ (see page 1017).

Mechanistically, glycosyltransferases can be classified as either inverting or retaining, depending on the overall stereochemical course of the reaction they catalyse³⁰ (Fig. 4). The stereochemistry of the transfer reaction does not correlate directly with a specific glycosyltransferase structural family, because both the GT-A and GT-B families include inverting and retaining glycosyltransferases²⁵,³¹. Structural³²⁻³⁴ and mechanistic²⁸,³⁵ data, combined with *ab initio* molecular orbital calculations³⁶, support an in-line Sₙ₂ attack mechanism for inverting glycosyltransferases (Fig. 4a).

The mechanism of retaining glycosyltransferases is still ambiguous. It was originally thought to involve a double displacement reaction analogous to that of retaining glycosidases³⁷,³⁸ (Fig. 4b). However, structural³⁹⁻⁴² and biochemical⁴³,⁴⁴ studies of several retaining glycosyltransferases in the presence of substrate analogues failed to identify an active-site nucleophile that could be involved in forming the enzyme–sugar adduct. This led to the proposal of a front-side Sₙ2 (or Sₙi) attack mechanism involving a late oxocarbenium-like transition state³⁹, in which the new glycosidic linkage between the acceptor and donor is formed on the same side of the sugar ring as the scissile glycosidic linkage to the NDP leaving group. Recently, however, Lairson *et al.* identified a covalent attachment between sugar and enzyme in a mutant α1,4-galactosyltransferase C (LgtC) from *Neisseria meningitidis*³⁷, rekindling the mechanistic debate about this important class of enzyme.

In addition to the canonical glycosyltransferase-catalysed reaction involving sugar transfer from an NDP-sugar to the hydroxyl group of an aglycone acceptor, several unusual modes of glycosylation exist in nature. It has recently been demonstrated that the glycosyltransferases of certain macrolide and anthracycline antibiotics require an auxiliary or accessory protein for efficient glycosyl transfer⁴⁵⁻⁴⁸. Although the exact biochemical function of this protein is not yet clear, it seems to be involved in activating its partner glycosyltransferase for catalysis, perhaps either by helping the glycosyltransferase fold into an active conformation or by inducing a conformational change that results in activation. In addition to the ubiquitous O-glycosides, N-, C- and S-glycosides have also been found in nature. Perhaps the best-known example of N-glycosylation is the reaction catalysed by the heteromeric, multisubunit oligosaccharyltransferase (OST) complex that initiates N-glycan biosynthesis in eukaryotes. In this case, an oligosaccharide core is transferred from a dolichol-pyrophosphate donor to the asparagine residue of an Asn-X-Ser/Thr-X motif (where X is not Pro) of the target protein substrate in the endoplasmic reticulum⁴⁹. Despite its ubiquity in eukaryotic cells, the catalytic mechanism and the contributions of the individual OST subunits to substrate recognition and catalysis remain largely unexplored⁴⁹.

Although N-glycosides are much less common in prokaryotes, several N-glycosyltransferases involved in the biosynthesis of indolocarbozoles⁵⁰⁻⁵² and the single-subunit OST from *Campylobacter jejuni*⁵³ have recently been identified and characterized. C-glycosides have been found in several natural aromatic products from prokaryotes and plants⁵⁴, and the mechanism of C-glycosylation has been proposed to involve nucleophilic attack by an activated aromatic ring on the anomeric carbon of the NDP-sugar⁵⁵ (Fig. 4c). Although several prokaryotic natural products contain thiosugars (such as 36, Fig. 1), authentic

Figure 3 | Examples of sugar biosynthetic enzymes from the short-chain dehydrogenase/reductase enzyme family. **a**, Epimerization at unactivated carbon centres catalysed by UDP-α-D-galactose-4-epimerase, CDP-α-D-tyvelose-2-epimerase and ADP-β-L-glycerol-D-manno-heptose-6-epimerase. Each involves oxidation of the hydroxyl group at the chiral centre to be epimerized, followed by rotation of the keto intermediate in the active site and reduction from the opposite face to complete the epimerization. **b**, In the mechanism of GDP-α-D-mannose-3,5-epimerase/reductase, oxidation to the keto intermediate followed by C5 epimerization and C4 reduction yields GDP-β-L-gulose (23 → 24 → 25 → 26). Intermediate 25 can also be epimerized at C3 and reduced at C4 to yield GDP-β-L-galactose (25 → 27 → 28 → 29 → 30). **c**, Reaction catalysed by UDP-glucuronic acid decarboxylase.

Figure 4 | Mechanism of glycosyltransferases.  
a, For inverting glycosyltransferases, general base (B)-catalysed in-line S<sub>N</sub>2 attack at C1 of the NDP-sugar donor by the acceptor leads to inversion of stereochemistry at the anomic carbon.  
b, The double displacement mechanism for retaining glycosyltransferases involves the S<sub>N</sub>2 attack of an active site nucleophile (Nu) at C1 of the NDP-sugar to form a covalent sugar-enzyme adduct, followed by an S<sub>N</sub>2 attack of the acceptor at C1 to form the glycosidic linkage with retention of anomic configuration.  
c, The putative mechanism for C-glycosylation in aryl-C-glycosides involves aromatic substitution facilitated by an ortho/para-directing hydroxyl group.

S-glycosyltransferases have not yet been identified in these organisms. To the best of our knowledge, UGT74B1, which is involved in glucosinate biosynthesis in *A. thaliana*<sup>56</sup>, is the only S-glycosyltransferase to have been characterized *in vitro*<sup>57</sup>.

### Bioengineering glycosylated natural products

Not all biologically active natural products are suitable for clinical use, owing to problems with solubility, bioavailability, toxicity or ineffectiveness against resistant organisms, making preparation of their analogues important in drug-discovery efforts. As a result of their structural complexity, many natural products are challenging to prepare or modify through purely synthetic methods. Because of the difficulties often encountered in total synthesis, re-engineering of the existing biosynthetic machinery offers an alternative approach for the creation of natural product analogues with several distinct advantages: access to chemical derivative structures not easily obtained by synthetic means, ease of compound production, reduced production of toxic by-products, scalability of the process, and the potential for performing combinatorial biosynthesis.

The sequences of more than 100 gene clusters involved in the production of glycosylated natural products are known, and the activities of many sugar biosynthetic enzymes and glycosyltransferases have been characterized. This means that these gene/enzyme components can potentially be used in their natural cellular contexts or in artificially assembled *in vivo* or *in vitro* systems for the synthesis of novel glycosylated products. All successful bioengineering applications ultimately depend on the tolerance of these biosynthetic enzymes for alternative substrates. Particularly important is the availability of 'substrate-flexible' glycosyltransferases, which, as highlighted below, have recently been used for the glycoengineering of secondary metabolites.

### Glycodiversification of natural products *in vivo*

An overview of the common strategies for biological glycodiversification of natural products is outlined in Fig. 5. Precursor-directed biosynthesis involves feeding analogues of a natural product precursor to cell cultures of the producing organism to effect structural changes in a natural product<sup>58</sup>. Mutational biosynthesis (or mutasynthesis) is a variation on precursor-directed biosynthesis that involves blocking the formation of an intermediate that is being replaced by the exogenously fed analogue through gene disruption<sup>59</sup>. Metabolic pathway engineering, or combinatorial biosynthesis, is another emerging technique used to alter the structure of a given natural product *in vivo*, and has recently been used in the glycodiversification of several classes of natural product aglycone. In one early glycodiversification study, expression of a glycosyltransferase from the vancomycin biosynthetic pathway of *Amycolatopsis orientalis* in *Streptomyces toyocaensis*, the producer of the structurally similar antibiotic A47934, resulted in the formation of a glucosylated A47934 derivative<sup>60</sup>.

In a separate study, the daunorubicin biosynthetic pathway of *Streptomyces peucetius* was altered to produce epirubicin, a semi-synthetic antitumour drug with improved therapeutic properties. Structurally, daunorubicin and epirubicin differ only in the configuration of the C4 hydroxyl group of the attached sugar, which is L-daunosamine (37, Fig. 1) in daunorubicin. In this experiment, the gene encoding the proposed sugar 4-ketoreductase of the daunorubicin pathway, *dnrV*, was disrupted and replaced with either *avrB* or *eryBIV*, two *dnrV* homologues from the avermectin and erythromycin biosynthetic pathways, respectively. These two genes were predicted to encode enzymes that could catalyse the reduction of 4-ketosugars with the opposite stereochemistry to DnrV. Indeed, both replacement mutants made significant quantities of epirubicin<sup>61</sup>. Interestingly, the epirubicin sugar generated by these studies, 4-epi-L-daunosamine, does not occur naturally, illustrating the potential of bioengineering methods to construct novel sugar structures using combinations of natural biosynthetic enzymes.

Although the relaxed substrate tolerance of natural product glycosyltransferases was implicated by these early bioengineering results, a more detailed investigation of the substrate specificity of a natural product glycosyltransferase (DesVII, a desosaminyltransferase) was only conducted recently in studies of TDP-D-desosamine (19, Fig. 1) biosynthesis in the methymycin/pikromycin pathway of *S. venezuelae*. As shown in Fig. 6a, individual disruption of four separate D-desosamine biosynthetic genes, *desI*, *desII*, *desV* and *desVI*, led to the formation of new macrolide analogues (41–44) carrying sugars corresponding to the accumulated D-desosamine pathway intermediates<sup>62–65</sup> (8 and 38–40). Expression of gene cassettes encoding TDP-D-dihydrostreptose (45, Fig. 1) biosynthetic genes (*strM* and *strL*) from *Streptomyces griseus*<sup>66</sup>, or TDP-D-mycaminose (46, Fig. 1) biosynthetic genes (*tyl1a* and *tylB*) from *S. fradiae*<sup>67</sup> in an *S. venezuelae* mutant lacking a functional *desI* gene (*KdesI*) led to novel methymycin/pikromycin derivatives bearing L-rhamnose (47) or D-mycaminose (48), respectively. In a separate

study, a library of 14-membered ring macrolactones was also shown to be glycosylated by DesVII (ref. 68). Together, these results clearly demonstrated that DesVII is promiscuous towards both its sugar and aglycone substrates.

A second example of a substrate-promiscuous glycosyltransferase that has been used to create an array of glycosylated natural product analogues *in vivo* is ElmGT from the elloramycin pathway of *Streptomyces olivaceus*. Using a combination of gene-knockout, heterologous-expression and feeding experiments in several producing and non-producing *Streptomyces* strains, ElmGT was used to generate a total of 12 new elloramycin derivatives, each bearing a unique sugar structure^{69–74}. Cumulatively, these and other^{51,75–80} examples firmly established the feasibility of using pathway engineering to produce novel natural product glycoforms through the assembly of custom-designed sugar biosynthetic pathways and substrate-promiscuous glycosyltransferases.

Although many natural product glycosyltransferases exhibit inherent substrate flexibility, others may be amenable to protein engineering to further alter or expand their donor and/or acceptor substrate specificities. In a remarkable set of experiments, Hoffmeister *et al.* constructed several chimaeric variants of two urdamycin A glycosyltransferases, UrdGT1b and UrdGT1c, which have 91% sequence identity yet show different specificities for both sugar and acceptor substrates. *In vivo* analysis of the activities of a series of UrdGT1b/UrdGT1c chimaeras allowed the identification of a 30-amino-acid region near the amino terminus of both proteins that is responsible for both donor and acceptor substrate recognition. Through mutagenesis, 10 of the 18 non-identical amino acids in this region were shown to be important for specificity. This allowed the construction of a library of 2,048 variants harbouring different combinations of parental amino-acid sequences, and these variants were subsequently screened for new activities *in vivo*^{81,82}. Several of these chimaeric enzymes exclusively catalysed the production of a novel derivative of urdamycin that contained a branched sugar appendage. Although this feat of engineering has not yet been duplicated in other systems, recent progress in the development of high-throughput screening methods for glycosyltransferase activity^{83,84} will facilitate future efforts to construct glycosyltransferases with unnatural activities and substrate specificities. Another significant limitation to successful glycosyltransferase engineering is the dearth of structural data on these enzymes. So far, the crystal structures of only a handful of natural product glycosyltransferases have been reported^{85–88}.

![Diagram](attachment:diagram.png)

Figure 5 | Overview of bioengineering strategies. A producing strain can be manipulated to generate new natural products through a combination of feeding experiments (precursor-directed biosynthesis (**a**) or mutasynthesis (**b**)) and/or genetic manipulation (gene disruption or gene disruption/heterologous expression). In metabolic pathway engineering (**c**), a sugar biosynthetic intermediate is rerouted to generate a new product through gene disruption and heterologous gene expression.

In combinatorial biosynthesis (**d**), sugar biosynthetic pathways from separate organisms can be re-engineered and ‘combined’ in a single organism to give hybrid or novel natural products. In enzymatic/chemoenzymatic synthesis (**e**), sugar biosynthetic enzymes with natural or engineered substrate flexibility are isolated and used for the construction of natural product libraries *in vitro*, using purified aglycone and NDP-sugar substrates.

As the three-dimensional structures of more glycosyltransferases become available, it may become possible to identify key amino acids that could be mutated to alter the substrate specificity or catalytic activity of these glycosyltransferases.

**Glycodiversification of natural products in vitro**

The *in vitro* demonstration of natural product glycosyltransferase activity has facilitated both enzymatic and chemoenzymatic synthesis of diverse natural product glycoforms. The availability of various NDP-sugar and aglycone substrates is crucial to the success of this approach. Recently, improvements in chemical methods for NDP-sugar synthesis and the development of *in vitro* glycorandomization (IVG, also referred to as glycodiversification or glyco-optimization) have made small libraries of NDP-sugars accessible for *in vitro* glycosylation assays. IVG is a chemoenzymatic method that uses engineered substrate-flexible anomeric sugar kinases and nucleotidylyltransferases for the conversion of diverse sugars to their NDP-activated forms. Application of IVG to the GtfE-catalysed glycosylation of the vancomycin aglycone yielded more than 30 vancomycin analogues with altered sugar structures, including three with sugars containing an azide functional group that permitted further combinatorial functionalization using the Huisgen 1,3-dipolar cycloaddition ('click' chemistry). IVG has also been used to assess glycosyltransferase substrate flexibility and to generate glycosylated natural product analogues of methymycin and novobiocin.

Recently, the Eguchi and Thorson research groups demonstrated that several natural product glycosyltransferase-catalysed reactions are reversible *in vitro*. In the presence of a glycosylated natural product and NDP, these glycosyltransferases can synthesize the corresponding NDP-sugar and aglycone. Thus, the reversibility of the glycosyltransferase-catalysed reaction could provide easy access to various NDP-sugars for *in vitro* studies. Next, they exploited this reversibility to perform one-pot sugar and aglycone exchange reactions. This methodology, combined with IVG, led to the synthesis of more than 70 derivatives of the antitumour compound calicheamicin (Fig. 6b). These demonstrations of glycosyltransferase utility are likely to affect many areas of glycobiology and glycochemistry research.

**Conclusions and future directions**

The enzymes involved in the biosynthesis of the unusual sugars in bacteria are excellent targets for drug development, because most of these sugars are absent in humans. Many of them are important constituents of the bacterial cell wall, and their biosyntheses involve

---

**Figure 6 | Glycodiversification of natural products.**  
*a*, *In vivo* glycodiversification in *S. venezuelae*. The D-desosamine biosynthetic pathway (8 → 38 → 39 → 40 → 19) was altered either through gene disruption to give 41–44 or through a combination of gene disruption and heterologous gene expression to give 47 and 48. In all cases, the endogenous glycosyltransferase-auxiliary-protein pair DesVII and DesVIII coupled the 'unnatural' sugars to the native *S. venezuelae* aglycones (in which R represents methynolide or pikrolide). The particular *S. venezuelae* mutant that was used in each study is shown in red (for example, in KdesI, the *desI* gene is disrupted). The names of the enzymes involved in the chemical transformations are shown in blue. Other enzymatic modifications that are required to generate the observed products are shown in purple. The enzymes responsible for the N-acetylation in 42 and 44 and the keto-group reduction in 41 and 43 are unknown.  
*b*, Applications of *in vitro* glycorandomization. After removal of the 3-O-methyl-rhamnose moiety of eight different calicheamicin analogues by means of a reverse-glycosyltransferase reaction, the calicheamicin glycosyltransferase, CalG1, was found to couple 10 different TDP-sugars to 8 calicheamicin aglycones, yielding a 72-member glycorandomized library. (Eight of the possible 80 combinations were not obtained.)
mechanistically novel enzymatic steps that have been actively targeted for inhibitor development<sup>94–97</sup>. Thus, future mechanistic and structural studies of bacterial carbohydrate biosynthetic enzymes will undoubtedly continue to present researchers with alternative drug targets and inspiration for the development of new antibiotics. As an alternative approach to drug design, engineering natural product sugar structures through the manipulation of sugar biosynthetic machinery is an effective means by which to generate glycoconjugates with new or enhanced biological activities. The viability of *in vivo* bioengineering strategies, such as combinatorial biosynthesis, and *in vitro* chemoenzymatic synthesis methods have infused new excitement into glycodiversification efforts<sup>93</sup>. Rapid advances in the identification and sequencing of natural product biosynthetic gene clusters, coupled with the isolation and the structural and functional characterization of the encoded enzymes, will undoubtedly augment our ability to customize the glycosylation patterns of natural products through these bioengineering approaches. The power, elegance and utility of nature’s biosynthetic machinery has clearly ushered in a new era of carbohydrate research, and has ensured a promising future for natural product glycodiversification in the search for new and improved drugs.

**Note added in proof:** Lovering *et al.* recently demonstrated that the glycosyltransferase domain of a bifunctional glycosyltransferase/transpeptidase enzyme involved in peptidoglycan biosynthesis in *Staphylococcus aureus* adopts a structural fold that is distinct from that of the GT-A and GT-B glycosyltransferase families, making it the first member of a new family of glycosyltransferases to be structurally characterized<sup>98</sup>.

1. Weymouth-Wilson, A. C. The role of carbohydrates in biologically active natural products. *Nat. Prod. Rep.* **14**, 99–110 (1997).
2. Thorson, J. S., Hosted, T. J., Jiang, J., Biggins, J. B. & Ahlert, J. Nature’s carbohydrate chemists: the enzymatic glycosylation of bioactive bacterial metabolites. *Curr. Org. Chem.* **5**, 139–167 (2001).
3. Barton, W. A. *et al.* Structure, mechanism and engineering of a nucleotidylyltransferase as a first step toward glycorandomization. *Nature Struct. Biol.* **8**, 545–551 (2001).
4. Varki, A. *et al.* Essentials of Glycobiology (Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York, 1999).
5. Johnson, D. A. & Liu, H.-w. in Comprehensive Natural Product Chemistry (eds Barton, D. H. R., Meth-Cohn, O. & Nakanishi, K.) 311–365 (Elsevier, Amsterdam, 1999).
6. He, X., Agnihotri, G. & Liu, H.-w. Novel enzymatic mechanisms in carbohydrate metabolism. *Chem. Rev.* **100**, 4615–4661 (2000).
7. He, X. M. & Liu, H.-w. Formation of unusual sugars: mechanistic studies and biosynthetic applications. *Annu. Rev. Biochem.* **71**, 701–754 (2002).
8. Hallis, T. M. & Liu, H.-w. Learning Nature’s strategies for making deoxy sugars: pathways, mechanisms, and combinatorial applications. *Acc. Chem. Res.* **32**, 579–588 (1999).
9. He, X. & Liu, H.-w. Mechanisms of enzymatic C–O bond cleavages in deoxyhexose biosynthesis. *Curr. Opin. Chem. Biol.* **6**, 590–597 (2002).
10. Szu, P.-h., He, X., Zhao, L. & Liu, H.-w. Biosynthesis of TDP-β-desosamine: identification of a strategy for C4 deoxygenation. *Angew. Chem. Int. Ed.* **44**, 6742–6746 (2005).
11. Oppermann, U. *et al.* Short-chain dehydrogenases/reductases (SDR): the 2002 update. *Chem. Biol. Interact.* **143–144**, 247–253 (2003).
12. Frey, P. A. The Leloir pathway: a mechanistic imperative for three enzymes to change the stereochemical configuration of a single carbon in galactose. *FASEB J.* **10**, 461–470 (1996).
13. Thoden, J. B., Frey, P. A. & Holden, H. M. Molecular structure of the NADH/UDP-glucose abortive complex of UDP-galactose-4-epimerase from *Escherichia coli*: implications for the catalytic mechanism. *Biochemistry* **35**, 5137–5144 (1996).
14. Hallis, T. M., Zhao, Z. & Liu, H.-w. New insights into the mechanism of CDP-β-tyvelose 2-epimerase: an enzyme catalyzing epimerization at an unactivated stereocenter. *J. Am. Chem. Soc.* **122**, 10493–10503 (2000).
15. Morrison, J. P., Read, J. A., Coleman, W. G. & Tanner, M. E. Dismutase activity of ADP-L-glycerol-D-manno-heptose 6-epimerase: evidence for a direct oxidation/reduction mechanism. *Biochemistry* **44**, 5907–5915 (2005).
16. Major, L. L., Wolucka, B. A. & Naismith, J. H. Structure and function of GDP-mannose-3′,5′-epimerase: an enzyme which performs three chemical reactions at the same active site. *J. Am. Chem. Soc.* **127**, 18309–18320 (2005).
17. Mulichak, A. M., Theisen, M. J., Essigmann, B., Benning, C. & Garavito, R. M. Crystal structure of SQD1, an enzyme involved in the biosynthesis of the plant sulfolipid headgroup donor UDP-sulfoquinovose. *Proc. Natl Acad. Sci. USA* **96**, 13097–13102 (1999).
18. Sanda, S., Leustek, T., Theisen, M. J., Garavito, R. M. & Benning, C. Recombinant Arabidopsis SQD1 converts UDP-glucose and sulfite to the sulfolipid head group precursor UDP-sulfoquinovose *in vitro*. *J. Biol. Chem.* **276**, 3941–3946 (2001).
19. Schutzbach, J. S. & Feingold, D. S. Biosynthesis of uridine diphosphate β-xylose IV. Mechanism of action of uridine diphosphoglucuronate carboxy-lyase. *J. Biol. Chem.* **245**, 2476–2482 (1970).
20. Davies, G. J., Gloster, T. M., & Henrissat, B. Recent structural insights into the expanding world of carbohydrate-active enzymes. *Curr. Opin. Struct. Biol.* **15**, 637–645 (2005).
21. Campbell, J. A., Davies, G. J., Bulone, V. & Henrissat, B. A classification of nucleotide-diphospho-sugar glycosyltransferases based on amino acid sequence similarities. *J. Biochem.* **326**, 929–942 (1997).

22. Coutinho, P. M., Deleury, E., Davies, G. J. & Henrissat, B. An evolving hierarchical family classification for glycosyltransferases. *J. Mol. Biol.* **328**, 307–317 (2003).
23. Unligil, U. M. & Rini, J. M. Glycosyltransferase structure and mechanism. *Curr. Opin. Struct. Biol.* **10**, 510–517 (2000).
24. Bourne, Y. & Henrissat, B. Glycoside hydrolases and glycosyltransferases: families and functional modules. *Curr. Opin. Struct. Biol.* **11**, 593–600 (2001).
25. Breton, C., Mucha, J. & Jeanneau, C. Structural and functional features of glycosyltransferases. *Biochimie* **83**, 713–718 (2001).
26. Pak, J. E. *et al.* X-ray crystal structure of leukocyte type core 2 β1,6-N-acetylglucosaminyl-transferase. Evidence for a convergence of metal ion-independent glycosyltransferase mechanism. *J. Biol. Chem.* **281**, 26693–26701 (2006).
27. Chiu, C. P. *et al.* Structural analysis of the sialyltransferase CstII from *Campylobacter jejuni* in complex with a substrate analogue. *Nature Struct. Mol. Biol.* **11**, 163–170 (2004).
28. Murray, B. W., Takayama, S., Schultz, J. & Wong, C.-H. Mechanism and specificity of human α-1,3-fucosyltransferase V. *Biochemistry* **35**, 11183–11195 (1996).
29. Hu, Y. & Walker, S. Remarkable structural similarities between diverse glycosyltransferases. *Chem. Biol.* **9**, 1287–1296 (2002).
30. Sinnott, M. L. Catalytic mechanisms of enzymic glycosyl transfer. *Chem. Rev.* **90**, 1171–1202 (1990).
31. Gibson, R. P., Turkenburg, J. P., Charnock, S. J., Lloyd, R. & Davies, G. J. Insights into trehalose synthesis provided by the structure of the retaining glycosyltransferase OtsA. *Chem. Biol.* **9**, 1337–1346 (2002).
32. Pedersen, L. C., Darden, T. A. & Negishi, M. Crystal structure of β1,3-glucuronyltransferase I in complex with active donor substrate UDP-GlcUA. *J. Biol. Chem.* **277**, 21869–21873 (2002).
33. Charnock, S. J. & Davies, G. J. Structure of the nucleotide-diphospho-sugar transferase, SpsA from *Bacillus subtilis*, in native and nucleotide-complexed forms. *Biochemistry* **38**, 6380–6385 (1999).
34. Tarbouriech, N., Charnock, S. J. & Davies, G. J. Three-dimensional structures of the Mn and Mg dTDP complexes of the family GT-2 glycosyltransferase SpsA: a comparison with related NDP-sugar glycosyltransferases. *J. Mol. Biol.* **314**, 655–661 (2001).
35. Qiao, L., Murray, B. W., Shimazaki, M., Schultz, J. & Wong, C.-H. Synergistic inhibition of human α-1,3-fucosyltransferase V. *J. Am. Chem. Soc.* **118**, 7653–7662 (1996).
36. Tvaroska, I., Andre, I. & Carver, J. P. *Ab initio* molecular orbital study of the catalytic mechanism of glycosyltransferases: description of reaction pathways and determination of transition-state structures for inverting *N*-acetylglucosaminyltransferases. *J. Am. Chem. Soc.* **122**, 8762–8776 (2000).
37. Lairson, L. L. *et al.* Intermediate trapping on a mutant retaining α-galactosyltransferase identifies an unexpected aspartate residue. *J. Biol. Chem.* **279**, 28339–28344 (2004).
38. Zechel, D. L. & Withers, S. G. Glycosidase mechanisms: anatomy of a finely tuned catalyst. *Acc. Chem. Res.* **33**, 11–18 (2000).
39. Persson, K. *et al.* Crystal structure of the retaining galactosyltransferase LgtC from *Neisseria meningitidis* in complex with donor and acceptor sugar analogs. *Nature Struct. Biol.* **8**, 166–175 (2001).
40. Pedersen, L. C. *et al.* Crystal structure of an α-1,4-*N*-acetylhexosaminyltransferase (EXTL2), a member of the exostosin gene family involved in heparan sulfate biosynthesis. *J. Biol. Chem.* **278**, 14420–14428 (2003).
41. Boix, E. *et al.* Structure of UDP complex of UDP-galactose:β-galactoside-α-1,3-galactosyltransferase at 1.53-Å resolution reveals a conformational change in the catalytically important C terminus. *J. Biol. Chem.* **276**, 48608–48614 (2001).
42. Boix, E., Zhang, Y., Swaminathan, G. J., Brew, K. & Acharya, K. R. Structural basis of ordered binding of donor and acceptor substrates to the retaining glycosyltransferase, α-1,3-galactosyltransferase. *J. Biol. Chem.* **277**, 28310–28318 (2002).
43. Ly, H. D., Lougheed, B., Wakarchuk, W. W. & Withers, S. G. Mechanistic studies of a retaining α-galactosyltransferase from *Neisseria meningitidis*. *Biochemistry* **41**, 5075–5085 (2002).
44. Zhang, Y. *et al.* Roles of individual enzyme-substrate interactions by α-1,3-galactosyltransferase in catalysis and specificity. *Biochemistry* **42**, 13512–13521 (2003).
45. Borisova, S. A. *et al.* Substrate specificity of the macrolide-glycosylating enzyme pair DesVII/DesVIII: opportunities, limitations, and mechanistic hypotheses. *Angew. Chem. Int. Ed. Engl.* **45**, 2748–2753 (2006).
46. Kao, C.-L., Borisova, S. A., Kim, H. J., & Liu, H.-w. Linear aglycones are the substrates for glycosyltransferase DesVII in methymycin biosynthesis: analysis and implications. *J. Am. Chem. Soc.* **128**, 5606–5607 (2006).
47. Yuan, Y. *et al.* In vitro reconstitution of EryCIII activity for the preparation of unnatural macrolides. *J. Am. Chem. Soc.* **127**, 14128–14129 (2005).
48. Lu, W. *et al.* AknT is an activating protein for the glycosyltransferase AknS in L-amino-deoxysugar transfer to the aglycone of aclacinomycin A. *Chem. Biol.* **12**, 527–534 (2005).
49. Imperiali, B. & Tai, V. W. -F. in Carbohydrate-based Drug Discovery Vol. 1 (ed. Wong, C. -H.) 281–303 (Wiley-VCH, Weinheim, 2003).
50. Onaka, H., Taniguchi, S.-i., Igarashi, Y. & Furumai, T. Characterization of the biosynthetic gene cluster of rebeccamycin from *Lechevalieria aerocolonigenes* ATCC 39243. *Biotechnol. Biochem.* **67**, 127–138 (2003).
51. Salas, A. P. *et al.* Deciphering the late steps in the biosynthesis of the anti-tumour indolocarbazole staurosporine: sugar donor substrate flexibility of the StaG glycosyltransferase. *Mol. Microbiol.* **58**, 17–27 (2005).
52. Gao, Q., Zhang, C., Blanchard, S. & Thorson, J. S. Deciphering indolocarbazole and enediyne aminodeoxyxypentose biosynthesis through comparative genomics: insights from the AT2433 biosynthetic locus. *Chem. Biol.* **13**, 733–743 (2006).
53. Wacker, M. *et al.* N-linked glycosylation in *Campylobacter jejuni* and its functional transfer into *E. coli*. *Science* **298**, 1790–1793 (2002).
54. Hultin, P. G. Bioactive C-glycosides from bacterial secondary metabolism. *Curr. Topics Med. Chem.* **5**, 1299–1331 (2005).
55. Bililign, T., Hyun, C.-G., Williams, J. S., Czisny, A. M. & Thorson, J. S. The hedamycin locus implicates a novel aromatic PKS priming mechanism. *Chem. Biol.* **11**, 959–969 (2004).
56. Halkier, B. A. & Gershenzon, J. Biology and biochemistry of glucosinolates. *Annu. Rev. Plant Biol.* **57**, 303–333 (2006).
57. Grubb, C. D. *et al.* Arabidopsis glucosyltransferase UGT74B1 functions in glucosinolate biosynthesis and auxin homeostasis. *Plant J.* **40**, 893–908 (2004).

58. Thiericke, R. & Rohr, J. Biological variation of microbial metabolites by precursor-directed biosynthesis. *Nat. Prod. Rep.* **10**, 265–289 (1993).

59. Weist, S. & Sussmuth, R. D. Mutational biosynthesis — a tool for the generation of structural diversity in the biosynthesis of antibiotics. *Appl. Microbiol. Biotech.* **68**, 141–150 (2005).

60. Solenberg, P. J. *et al.* Production of hybrid glycopeptide antibiotics *in vitro* and in *Streptomyces toyocaensis*. *Chem. Biol.* **4**, 195–202 (1997).

61. Madduri, K. *et al.* Production of the antitumor drug epirubicin (4′-epidoxorubicin) and its precursor by a genetically engineered strain of *Streptomyces peucetius*. *Nature Biotechnol.* **16**, 69–74 (1998).

62. Zhao, L., Sherman, D. H. & Liu, H.-w. Biosynthesis of desosamine: construction of a new methymycin/neomethymycin analogue by deletion of a desoamine biosynthetic gene. *J. Am. Chem. Soc.* **120**, 10256–10257 (1998).

63. Zhao, L., Que, N. L. S., Xue, Y., Sherman, D. H. & Liu, H.-w. Mechanistic studies of desosamine biosynthesis: C-4 deoxygenation precedes C-3 transamination. *J. Am. Chem. Soc.* **120**, 12159–12160 (1998).

64. Zhao, L., Borisova, S., Yeung, S. M. & Liu, H. Study of C-4 deoxygenation in the biosynthesis of desosamine: evidence implicating a novel mechanism. *J. Am. Chem. Soc.* **123**, 7909–7910 (2001).

65. Borisova, S. A., Zhao, L., Sherman, D. H. & Liu, H. W. Biosynthesis of desosamine: construction of a new macrolide carrying a genetically designed sugar moiety. *Org. Lett.* **1**, 133–136 (1999).

66. Yamase, H., Zhao, L. & Liu, H.-w. Engineering a hybrid sugar biosynthetic pathway: production of l-rhamnose and its implication on dihydrostreptose biosynthesis. *J. Am. Chem. Soc.* **122**, 12397–12398 (2000).

67. Melancon, C. E., Yu, W. L. & Liu, H. W. TDP-mycaminose biosynthetic pathway revised and conversion of desosamine pathway to mycaminose pathway with one gene. *J. Am. Chem. Soc.* **127**, 12240–12241 (2005).

68. Tang, L. & McDaniel, R. Construction of desosamine containing polyketide libraries using a glycosyltransferase with broad substrate specificity. *Chem. Biol.* **8**, 547–555 (2001).

69. Decker, H., Haag, S., Udvarnoki, G. & Rohr, J. Novel genetically engineered tetracenomycins. *Angew. Chem. Int. Ed. Engl.* **34**, 1107–1110 (1995).

70. Wohlert, S.-E. *et al.* Novel hybrid tetracenomycins through combinatorial biosynthesis using a glycosyltransferase encoded by the *elm* genes in cosmid 16F4 and which shows broad sugar substrate specificity. *J. Am. Chem. Soc.* **120**, 10596–10601 (1998).

71. Rodriguez, L. *et al.* Engineering deoxysugar biosynthetic pathways from antibiotic-producing microorganisms: a tool to produce novel glycosylated bioactive compounds. *Chem. Biol.* **9**, 721–729 (2002).

72. Fischer, C. *et al.* Digitoxosyltetracenomycin C and glucosyltetracenomycin C, two novel elloramycin analogues obtained by exploring the sugar donor substrate flexibility of glycosyltransferase ElmGT. *J. Nat. Prod.* **65**, 1685–1689 (2002).

73. Lombó, F. *et al.* Engineering biosynthetic pathways for deoxysugars: branched-chain sugar pathways and derivatives from the antitumor tetracenomycin. *Chem. Biol.* **11**, 1709–1718 (2004).

74. Pérez, M. *et al.* Combining sugar biosynthesis genes for the generation of l- and d-amicetose and formation of two novel antitumor tetracenomycins. *Chem. Commun. (Camb.)* **12**, 1604–1606 (2005).

75. Zhang, C. *et al.* RebG- and RebM-catalyzed indolocarbazole diversification. *ChemBioChem* **7**, 795–804 (2006).

76. Sanchez, C. *et al.* Combinatorial biosynthesis of antitumor indolocarbazole compounds. *Proc. Natl Acad. Sci. USA* **102**, 461–466 (2005).

77. Trefzer, A. *et al.* Elucidation of the function of two glycosyltransferase genes (*lanGT1* and *lanGT4*) involved in landomycin biosynthesis and generation of new oligosaccharide antibiotics. *Chem. Biol.* **8**, 1239–1252 (2001).

78. Torkkell, S. *et al.* The entire nogalamycin biosynthetic gene cluster of *Streptomyces nogalater*: characterization of a 20-kb DNA region and generation of hybrid structures. *Mol. Gen. Genet.* **266**, 276–288 (2001).

79. Wohlert, S.-E. *et al.* Insights about the biosynthesis of the avermectin deoxysugar I-oleandrose through heterologous expression of *Streptomyces avermitilis* deoxysugar genes in *Streptomyces lividans*. *Chem. Biol.* **8**, 681–700 (2001).

80. Gaisser, S. *et al.* A defined system for hybrid macrolide biosynthesis in *Saccharopolyspora erythraea*. *Mol. Microbiol.* **36**, 391–401 (2000).

81. Hoffmeister, D. *et al.* Engineered urdamycin glycosyltransferases are broadened and altered in substrate specificity. *Chem. Biol.* **9**, 287–295 (2002).

82. Hoffmeister, D., Ichinohe, K. & Bechthold, A. Two sequence elements of glycosyltransferases involved in urdamycin biosynthesis are responsible for substrate specificity and enzymatic activity. *Chem. Biol.* **8**, 557–567 (2001).

83. Love, K. R., Swoboda, J. G., Noren, C. J. & Walker, S. Enabling glycosyltransferase evolution: a facile substrate-attachment strategy for phage-display enzyme evolution. *ChemBioChem* **7**, 753–756 (2006).

84. Aharoni, A. *et al.* High-throughput screening methodology for the directed evolution of glycosyltransferases. *Nature Methods* **3**, 609–614 (2006).

85. Mulichak, A. M., Losey, H. C., Walsh, C. T. & Garavito, R. M. Structure of the UDP-glucosyltransferase GtfB that modifies the heptapeptide aglycone in the biosynthesis of vancomycin group antibiotics. *Structure* **9**, 547–557 (2001).

86. Mulichak, A. M. *et al.* Structure of the TDP-epi-vancosaminyltransferase GtfA from the chloroeremomycin biosynthetic pathway. *Proc. Natl Acad. Sci. USA* **100**, 9238–9243 (2003).

87. Mulichak, A. M., Lu, W., Losey, H. C., Walsh, C. T. & Garavito, R. M. Crystal structure of vancosaminyltransferase GtfD from the vancomycin biosynthetic pathway: interactions with acceptor and nucleotide ligands. *Biochemistry* **43**, 5170–5180 (2004).

88. Offen, W. *et al.* Structure of a flavonoid glucosyltransferase reveals the basis for plant natural product modification. *EMBO J.* **25**, 1396–1405 (2006).

89. Fu, X. *et al.* Antibiotic optimization via *in vitro* glycorandomization. *Nature Biotechnol.* **21**, 1467–1469 (2003).

90. Albermann, C. *et al.* Substrate specificity of NovM: implications for novobiocin biosynthesis and glycorandomization. *Org. Lett.* **5**, 933–936 (2003).

91. Zhang, C. *et al.* Exploiting the reversibility of natural product glycosyltransferase-catalyzed reactions. *Science* **313**, 1291–1294 (2006).

92. Minami, A., Kakinuma, K. & Eguchi, T. Aglycon switch approach toward unnatural glycosides from natural glycoside with glycosyltransferase VinC. *Tetrahedr. Lett.* **46**, 6187–6190 (2005).

93. Melancon, C. E., Thibodeaux, C. J. & Liu, H.-w. Glyco-stripping and glyco-swapping. *ACS Chem. Biol.* **1**, 499–504 (2006).

94. Grison, C., Petek, S., Finance, C. & Coutrot, P. Synthesis and antibacterial activity of mechanism-based inhibitors of KDO8P synthase and DAH7P synthase. *Carbohydrate Res.* **340**, 529–537 (2005).

95. Carlson, E. E., May, J. F. & Kiessling, L. L. Chemical probes of UDP-galactopyranose mutase. *Chem. Biol.* **13**, 825–837 (2006).

96. van Heijenoort, J. Recent advances in the formation of the bacterial peptidoglycan monomer unit. *Nat. Prod. Rep.* **18**, 503–519 (2001).

97. Zöeby, A. E., Sanschagrin, F. & Levesque, R. C. Structure and function of the Mur enzymes development of novel inhibitors. *Mol. Microbiol.* **47**, 1–12 (2003).

98. Lovering, A. L., de Castro, L. H., Lim, D. & Strynadka, N. C. J. Structural insight into the transglycosylation step of bacterial cell-wall biosynthesis. *Science* **315**, 1402–1405 (2007).


**Acknowledgements** We thank the National Institutes of Health for their generous support of this work.


**Author Information** Reprints and permissions information is available at npg.nature.com/reprintsandpermissions. The authors declare no competing financial interests. Correspondence should be addressed to H.-w.L. (h.w.liu@mail.utexas.edu).
