
Available online at www.sciencedirect.com

ScienceDirect

Bioorganic & Medicinal Chemistry 15 (2007) 4360–4368

---

### Unique O-ribosylation in the biosynthesis of butirosin

Fumitaka Kudo,$^{a}$ Takuya Fujii,$^{a}$ Shunsuke Kinoshita$^{a}$ and Tadashi Eguchi$^{b,*}$

**$^{a}$Department of Chemistry, Tokyo Institute of Technology, 2-12-1 O-okayama, Meguro-ku, Tokyo 152-8551, Japan**

**$^{b}$Department of Chemistry & Materials Science, Tokyo Institute of Technology, 2-12-1 O-okayama, Meguro-ku, Tokyo 152-8551, Japan**

Received 1 March 2007; revised 19 April 2007; accepted 20 April 2007  
Available online 25 April 2007

---

#### Abstract—Using a comparative genetics approach, one or more of the BtrA, BtrL, BtrP, and BtrV proteins encoded in the butirosin biosynthetic gene cluster (*btr*) from *Bacillus circulans* SANK72073 were identified as being responsible for an O-ribosylation process leading to the formation of ribostamycin, a key intermediate in this, and related antibiotic biosynthetic pathways. Functional analysis of the recombinantly expressed proteins revealed that both BtrL and BtrP were responsible for the ribosylation of neamine, using 5-phosphoribosyl-1-diphosphate (PRPP) as the ribosyl donor. Further detailed analysis indicated that this process occurs via two discrete steps: with BtrL first catalyzing the phosphoribosylaion of neamine to form 5″-phosphoribostamycin, followed by a BtrP-catalyzed dephosphorylation to generate ribostamycin. To the best of our knowledge, this is the first time that the functional characterization of a glycosyltransferase from an aminoglycoside biosynthetic gene cluster has been reported.

© 2007 Elsevier Ltd. All rights reserved.

---

### 1. Introduction

Aminocyclitol-containing compounds such as kanamycin, neomycin, butirosin, and streptomycin are a clinically important class of antibiotic agents. Members of this antibiotic family contain one of two common aminocyclitol units: streptamine or 2-deoxystreptamine (DOS), which are decorated with diverse structural amino sugars, deoxysugars, and further functionalized sugars. Glycosyl transfer reactions play a critical role in generating structural diversity within these aminoglycosides by catalyzing the formation of various linkages between the aminocyclitol and sugar units. It is envisaged that by using a ‘biosynthetic engineering’ approach, the rational re-assortment or rearrangement of these glycosyltransferases within biosynthetic gene clusters may lead to the formation of various new aminoglycoside antibiotics. It may also be possible to create new glycosides (or glycoside intermediates) by using combinations of these enzymes in vitro. However, to the best of our knowledge, there are no published reports describing the functional characterization of any glycosyltransferases derived from aminoglycoside biosynthetic pathways.

Keywords: 5-Phosphoribosyl-1-diphosphate:neamine phosphoribosyltransferase; 5″-Phosphoribostamycin phosphatase; Ribosylation; Butirosin; Ribostamycin.

* Corresponding author. E-mail: eguchi@cms.titech.ac.jp

The biochemical analysis of this class of enzymes is therefore of critical importance.

The majority of aminoglycoside antibiotics contain DOS, which is an aminocyclitol found only in this class of antibiotics. Two enzymes from the butirosin biosynthetic pathway in *Bacillus circulans* ${}^{1-3}$ were previously identified as being responsible for the formation of DOS: the 2-deoxy-scyllo-inosose (DOI) synthase BtrC and the Gln:DOI aminotransferase BtrS (also referred to as BtrR). This finding leads to the subsequent identification of many other DOS-containing antibiotic biosynthetic gene clusters. ${}^{4}$ Recent comparative genetic analyses of gene clusters responsible for the biosynthesis of this group of aminoglycosides have revealed that homologous genes appear to be involved in the construction of common intermediates, such as DOS, paromamine, neamine, and ribostamycin. ${}^{4,5}$ Using this approach, we recently established that the last remaining enzyme of unknown function from the DOS biosynthetic pathway was a 2-deoxy-scyllo-inosamine dehydrogenase. ${}^{6}$ Furthermore, several genes from the butirosin biosynthetic gene cluster were also found to be involved in the specific biosynthesis of 4-amino-2-hydroxybutyric acid. ${}^{7}$ However, a number of proteins encoded in the butirosin biosynthetic gene cluster remain to be characterized. The functions of the *btrA*, *L*, *P*, and *V* encoded proteins have previously been inferred by comparison with the DOS-containing biosynthetic gene clusters

© 2007 Elsevier Ltd. All rights reserved.

doi:10.1016/j.bmc.2007.04.040

responsible for the biosynthesis of neomycin and ribostamycin. 4 These proteins were predicted to be responsible for the biosynthesis of the common ribostamycin structure, which is putatively formed via an O-ribosylation of neamine (Scheme 1). In this paper, we characterize, after their heterologous expression in *Escherichia coli*, that BtrL is a 5-phosphoribosyl-1-diphosphate:neamine phosphoribosyltransferase and BtrP is a phosphoribostamycin phosphatase, responsible for the formation of the key ribostamycin intermediate in the biosynthesis of butirosin. To the best of our knowledge, this is the first report describing the functional characterization of a glycosyltransferase from an aminoglycoside biosynthetic gene cluster.

To investigate the hypothesized ribosylation of neamine, we first determined whether 5-phosphoribosyl-1-diphosphate (PRPP) was a potential ribosyl donor. PRPP is known to be a common ribosyl donor for the reactions of many phosphoribosyltransferases from nucleotide, tryptophan, histidine, and nicotinamide biosynthetic pathways,9 and is also a substrate for the recently characterized decaprenyl-phosphate 5-phosphoribosyltransferase.10 Cell-free extracts containing the recombinant BtrA, L, P, and V proteins were incubated with neamine and PRPP. The enzymatic reaction mixture was treated with 2,4-dinitrofluorobenzene, and then analyzed by HPLC and LC-ESI-MS. As shown in Figure 2, tetraakis-(2,4-dinitrophenyl)-ribostamycin was detected, indicating that ribostamycin was formed as a reaction product. This derivative was not formed in a control reaction without PRPP. These results clearly demonstrated that one of the BtrA, L, P, or V proteins, or a certain combination of them, was responsible for the ribosylation of neamine, using PRPP as a ribosyl donor.

### 2. Results

#### 2.1. Identification of the enzymes responsible for the ribosylation of neamine

The (plasmid-encoded) BtrA, P, and V proteins were expressed in *E. coli* in a soluble form, by incubating the cells at a relatively low temperature (15–18 °C) after induction with isopropyl β-D-thiogalactoside (IPTG). However, soluble BtrL was only obtained by co-expression with the GroESL chaperones8 at low temperature (Fig. 1).

![Scheme 1](https://i.imgur.com/1234567.png)

**Scheme 1.** Proposed formation of the ribostamycin intermediate in the butirosin biosynthetic pathway. Comparative genetic analysis suggested that one or more of the BtrA, L, P, and/or V proteins encoded in the butirosin biosynthetic gene cluster were responsible for the ribosylation of neamine to form ribostamycin. The structure of the biosynthetically related aminoglycoside antibiotic neomycin B is also shown.

![Figure 1](https://i.imgur.com/ABCDEFG.png)

**Figure 1.** Expression of recombinant BtrA, L, P, and V proteins in *E. coli*. Images of SDS–polyacrylamide gels, with lanes that contain aliquots of (supernatant) cell free extracts (CFEs) prepared from induced cultures of *E. coli* harboring the relevant expression plasmid, or empty expression plasmid. Lane 1: CFE from *E. coli*/pET30b; lane 2: CFE from *E. coli*/pET-btrA; lanes 3, 6, 8: CFE from *E. coli*/pET21a; lane 4: CFE from *E. coli*/pET-btrL; lane 5: CFE from *E. coli*/pET-btrL + pREP4-groESL; lane 7: CFE from *E. coli*/pET-btrP; lane 9: CFE from *E. coli*/pET-btrV; M, protein marker. The solid arrows indicate the positions of the over-expressed BtrA, L, P, and V proteins. The broken arrow (lane 5) indicates the co-expressed GroEL.

F. Kudo et al. / Bioorg. Med. Chem. 15 (2007) 4360–4368

![Diagram](https://i.imgur.com/yourimage.png)

Figure 2. Analysis of products formed after incubation of neamine, 5-phosphoribosyl-1-diphosphate (PRPP), and a mixture of the BtrA, L, P, and V proteins. The enzymatic product mixtures were reacted with 2,4-dinitrofluorobenzene prior to analysis by HPLC and LC-electrospray ionization mass spectrometry (LC-ESI-MS). (a) HPLC chromatogram of products formed by a mixture of BtrA, L, P, and V CFEs (50 μL of each, prepared in Hepes buffer, 50 mM, pH 8.0), neamine (8.9 mM), PRPP (8.9 mM), and MgCl₂ (1.1 mM), after incubation at 37°C for 12 h. (b) HPLC chromatogram of products formed by an identical enzymatic incubation, co-injected with an authentic sample of ribostamycin (which had been similarly reacted with 2,4-dinitrofluorobenzene). (c) HPLC chromatogram of products formed by an identical enzymatic incubation without PRPP. (d) ESI-mass spectrum of the 2,4-dinitrofluorobenzene-derivatized enzymatic reaction product. The arrows indicate the position of the tetrakis-(2,4-dinitrophenyl)-ribostamycin derivative.

processes, their order of addition was investigated. After neamine and PRPP had been incubated with either BtrL or BtrP, the enzymes were inactivated by heat treatment, then the corresponding other protein was added to the solution to catalyze the second step. This revealed that the BtrL reaction product was converted to ribostamycin by BtrP (Fig. 3). The BtrL reaction product was then analyzed by LC-ESI-MS, and a peak with a mass that was identical to that of phosphorylated ribostamycin was observed (Fig. 3c). These results clearly demonstrated that BtrL catalyzes the phosphoribosylation of neamine using PRPP as a phosphoribosyl donor to generate 5′-phosphoribostamycin. BtrP then catalyzes a dephosphorylation of 5′-phosphoribostamycin to form ribostamycin (Fig. 3).

### 2.2. Biochemical properties of BtrP

To further investigate the biochemical properties of these unique ribosylation enzymes, BtrP was purified by ion exchange chromatography on DEAE-Sephadex, and then by gel-filtration to near homogeneity (Fig. 4a). p-Nitrophenyl phosphate was used then as a substrate to examine its phosphatase activity. As we expected, BtrP was able to hydrolyze p-nitrophenyl phosphate to nitrophenol and inorganic orthophosphate (Fig. 4b). It was also found that the presence of EDTA did not affect the activity of BtrP (Fig. 4c). This suggested that BtrP did not need a (divalent) metal ion for activity, or if one was required, then it was bound extremely tightly in the active site. Other substrates with structures similar to ribose 5-phosphate were next examined to investigate the substrate specificity of BtrP. Of the phosphate-containing compounds tested, only fructose-1,6-bisphosphate was rapidly hydrolyzed by BtrP, while PRPP, fructose-6-phosphate, ribose-5-phosphate, and nucleotides (AMP, CMP, GMP, UMP, and dTMP) were not hydrolyzed (Fig. 4d).

### 2.3. Biochemical properties of BtrL

A coupled assay including both BtrL and BtrP was used to investigate the phosphoribosylation activities of BtrL, as BtrP had no detectable PRPP hydrolysis activity. BtrL activity was determined by quantifying the amount of ribostamycin formed after incubation with neamine and PRPP. The biochemical properties of BtrL were determined without purification (i.e., using freshly prepared cell-free extracts of induced cells), because its activity decreased rapidly (within a few hours) after disruption of the harvested *E. coli* cells. Attempts to stabilize the BtrL protein proved largely unsuccessful. Consequently, BtrL was found to have optimum catalytic activity at pH 7.5, at a temperature between 27°C and 32°C, and essentially required a divalent metal ion: optimally Mg²⁺, Ni²⁺, or Co²⁺ (Fig. 5).

We next investigated the ability of BtrL to utilize alternative phosphoribosyl acceptors, using a BtrP-coupled assay under optimal reaction conditions (pH 7.5, 32°C, 1 mM MgCl₂). As shown in Figure 6, BtrL also catalyzed the ribosylation of paromamine using PRPP,

F. Kudo et al. / Bioorg. Med. Chem. 15 (2007) 4360–4368

Figure 3. Two-step ribosylation of neamine catalyzed by the BtrL and BtrP proteins. An incubation of BtrL with neamine and PRPP was quenched by heat treatment, then incubated with BtrP. After derivatization with 2,4-dinitrofluorobenzene, enzymatic product mixtures were analyzed by HPLC and LC-ESI-MS. (a) HPLC chromatogram of derivatized ribostamycin formed after stepwise reaction with BtrL/BtrP proteins. (b) HPLC chromatogram of derivatized enzymatic product co-injected with authentic ribostamycin derivative. (c) ESI-mass spectrum of the dinitrophenyl derivative of 5′-phosphoribostamycin intermediate formed after incubation of neamine, PRPP, and BtrL. The arrows indicate the position of the tetrakis-(2,4-dinitrophenyl)-ribostamycin derivative.

Figure 4. Phosphatase activity of BtrP. (a) Purification of BtrP protein as analyzed by SDS–PAGE. Lane 1: pellet (insoluble protein) obtained after lysis of induced *E. coli*/pET-btrP cells; lane 2: CFE (supernatant) from induced *E. coli*/pET-btrP; lane 3: BtrP after purification by DEAE chromatography; lane 4: BtrP after purification by gel filtration chromatography; M, protein marker. (b) Analysis of *p*-nitrophenolate produced after incubation of BtrP (20 μM) with *p*-nitrophenyl phosphate (100 μM). (c) Divalent metal requirement for BtrP-mediated hydrolysis of *p*-nitrophenyl phosphate (metal/EDTA added to 1 mM). (d) Substrate specificity of BtrP-mediated reaction. Fructose-1,6-bisphosphate (F-1,6-P) and *p*-nitrophenyl phosphate (*p*NPP) were hydrolyzed by BtrP, but PRPP and the other tested phosphates were not.

to generate the corresponding product, but did not accept DOS or kanamycin A as substrates. This suggests that BtrL prefers the pseudo-disaccharide structure of neamine/paromamine.

### 3. Discussion

In this paper, we describe the functional characterization of the BtrL and BtrP proteins from the butirosin

F. Kudo et al. / Bioorg. Med. Chem. 15 (2007) 4360–4368

BtrP shares homology with fructose 2,6-bisphosphatases, phosphoglycerate mutases, and phosphohistidine phosphatases. Therefore, it was initially predicted to have some kind of phosphatase activity involved in the conversion of neamine to ribostamycin. Here, we clearly demonstrate that BtrP functions as a 5′-phosphoribostamycin phosphatase in the two-step ribosylation process. Catalytically important residues present in related phosphatases are highly conserved in BtrP. The BtrP-mediated phosphohydrolase process is thus predicted to occur via a very similar catalytic mechanism to those of related enzymes. However, structural homology studies do not suggest a clear mode for substrate recognition: as 5′-phosphoribostamycin is a very different molecule from fructose 1,6-bisphosphate (and other biosynthetic genes.

We show that the recombinant BtrL protein is a PRPP:neamine phosphoribosyltransferase, which produces 5′-phosphoribostamycin. This intermediate is then hydrolyzed by the 5′-phosphoribostamycin phosphatase, BtrP, to form ribostamycin. BtrL and its homologs from other ribostamycin-containing amino-glycoside biosynthetic gene clusters have a distinctive PRPP binding motif, which is also found in other phosphoribosyltransferases (Fig. 7). A comparative structural analysis of several phosphoribosyltransferases has shown that the adjacent Asp-Asp, or Glu-Asp, residues within this motif play key catalytic roles, and are responsible for divalent metal binding. In the BtrL protein, this corresponds to residues E217 and D218. However, apart from this PRPP binding motif, the rest of the BtrL sequence shares little homology with any of the other phosphoribosyltransferases. Determining the structure of the BtrL protein would therefore be invaluable in helping to elucidate the residues responsible for PRPP and neamine/paromamine substrate recognition and binding.

| BtrL | IVILEDDQPHGGT |
| --- | --- |
| NeoM | LVLVDDQPDTGAT |
| Upp | VLLVDPMLATGGS |
| Apt | VVIVDDILLATGGT |
| Hprt | VLIVEDIIDTGKT |
| Purl | VVLVDDSIVRGTT |
| Prs | AILVDDMIDTGGT |

Figure 7. The predicted PRPP binding motif in BtrL aligned with those from other phosphoribosyltransferases. BtrL, neamine phosphoribosyltransferase from *Bacillus circulans* (residues 213–225); NeoM, BtrL homolog in the neomycin producer *Streptomyces fradiae* (residues 241–254); Upp, uracil phosphoribosyltransferases; Apt, adenine phosphoribosyltransferases; Hprt, hypoxanthine-guanine phosphoribosyltransferases; Purl, amido phosphoribosyltransferases; Prs, PRPP synthases. The block-shaded letters indicate the two characteristic acidic amino acid residues.

Figure 6. The BtrL and BtrP catalyzed stepwise ribosylation of paromamine. (a) HPLC chromatogram of (DNP-derivatized) ribosylated paromamine formed after incubation of paromamine, PRPP, and the BtrL/BtrP proteins. (b) Identical incubation without PRPP. (c) Identical incubation without paromamine. (d) ESI-mass spectrum of (DNP-derivatized) ribosylated paromamine formed by incubating BtrL, BtrP, paromamine, and PRPP. The solid arrows indicate the position of the ribosylated paromamine derivative, and the broken arrow indicates the paromamine derivative.

related substrates). Consequently, solving the BtrP structure would also be required to elucidate the substrate recognition mechanism at the molecular level.

In our previous report, we demonstrated that the disruption of the $b t r P / V$ genes resulted in the abolition of antibiotic production. ${ }^{13}$ The results presented here suggest that this occurs via the accumulation of the 5"-phosphoribostamycin intermediate, which has no antibiotic activity. It was previously postulated that 5"-phosphoribostamycin was an inactive form of ribostamycin and was formed by a putative phosphotransferase enzyme. ${ }^{14}$ Because no obvious resistance gene exists in the btr gene cluster, BtrP appears to be related to self-resistance in $B$. circulans. Consequently, antibiotic production may be controlled by BtrP-mediated dephosphorylation combined with phosphorylation by an aminoglycoside phosphotransferase (APH) which has been previously identified in $B$. circulans. ${ }^{15}$

The functions of BtrA and V have yet to be determined. ORFs homologous to BtrA and V have been identified in the toxin complex genes of *Photorhabdus luminescens*, where they appear to be co-transcribed. The functions of BtrA and V may therefore be related to each other, somehow involved in ribostamycin production in the relevant antibiotic-producing organisms. Ribostamycin itself is toxic to the producer, and thus it is possible that BtrA and V may control the amount of ribostamycin biosynthesized. However, further functional analysis of these proteins is required to unequivocally establish their biosynthetic roles.

In summary, we have used a combination of comparative genetics and functional analyses to demonstrate that the BtrL and BtrP proteins from the butirosin biosynthetic pathway catalyze a unique two-step ribosylation of neamine, to form ribostamycin. Our results show that this two-pronged approach is a very effective method for establishing the activities of genes of unknown or poorly defined function within biosynthetic gene clusters.

### 4. Experimental

#### 4.1. General procedures

General DNA manipulations in *E. coli* were performed according to standard protocols. ${ }^{16}$ *Escherichia coli* DH5$\alpha$ was used as a host strain for the routine subcloning of DNA fragments. *Escherichia coli* BL21(DE3) was used for expression of the $b t r A$, $L$, $P$, and $V$ genes. The pT7Blue (Novagen) and pLITMUS28 (New England Biolabs) plasmids were routinely used as plasmid vectors for subcloning and sequencing. The pET21a(+) or pET30b(+) (Novagen) plasmids were used as vectors for the protein expression. The plasmid pREP4-groESL ${ }^{8}$ was used to co-express the GroES and GroEL proteins in addition to the expression vector for the $b t r L$ gene. *Escherichia coli* harboring plasmids were cultured in Luria-Bertani (LB) medium with appropriate antibiotics (100 $\mu \mathrm{g} / \mathrm{mL}$ of ampicillin or $30 \mu \mathrm{g} / \mathrm{mL}$ of kanamycin) at $37^{\circ} \mathrm{C}$ overnight before extracting plasmids. Purification of plasmids was performed using the GFX micro Plasmid Prep Kit (Amersham Bioscience).

Oligonucleotides were purchased from Sigma Genosys. Restriction enzymes and modification enzymes were purchased from TaKaRa. PCR was performed using the GeneAmp PCR System 9700 (Perkin-Elmer Applied Biosystems) with KOD-plus-DNA polymerase (TOYOBO). DNA sequence analysis was performed using a LONG READER 4200 (Li-Cor) and a Thermo Sequenase Cycle Sequencing Kit according to the manufacturer's protocol. HPLC analysis of enzymatic products was performed on a Hitachi L-7100 Pump equipped with an L-7405 UV Detector, an L-7300 Column Oven, and a D-2500 Chromato-Integrator. A LCQ mass spectrometer (Finnigan) fitted with a NANOSPACE HPLC and SE-1 UV detector (Shiseido, Japan) was used for the LC-ESI-MS analysis. FPLC (Pharmacia Biotech.) was used for protein purification. Other chemicals were of highest grade commercially available.

#### 4.2. Expression of BtrA, L, P, and V

Chromosomal DNA extracted from $B$. circulans SANK72073 was used as a template for PCR amplification of the $b t r A$, $L$, $P$, and $V$ genes. The $b t r L$ gene was amplified by PCR with the primers pETbtrL-f: $5^{\prime}-\mathrm{AGA}$ GGCTGCCGCATATGAAG-3' and pETbtrL-r: $5^{\prime}-$ CCGAAGGAAATTCATACAAAG-3'. PCR conditions were: $94^{\circ} \mathrm{C}, 2 \mathrm{~min}$ denature; 30 cycles of $94^{\circ} \mathrm{C}$ for $15 \mathrm{~s}, 55^{\circ} \mathrm{C}$ for $30 \mathrm{~s}$, and $68^{\circ} \mathrm{C}$ for $2 \mathrm{~min}$ for DNA extension; in $1 \mu \mathrm{L}$ of $10 \times$ KOD buffer, $0.8 \mu \mathrm{L}$ of dNTP (2.5 mM each), $0.4 \mu \mathrm{L}$ of $\mathrm{MgSO}_{4}(25 \mathrm{mM}), 0.1 \mu \mathrm{L}$ of pETbtrL-f $(50 \mu \mathrm{M}), 0.1 \mu \mathrm{L}$ of pETbtrL-r $(50 \mu \mathrm{M}), 0.2 \mu \mathrm{L}$ of KOD-plus polymerase, $0.5 \mu \mathrm{L}$ of template ($16 \mathrm{ng}$ genomic DNA/$\mu \mathrm{L}$ ), and $6.9 \mu \mathrm{L}$ of water (total $10 \mu \mathrm{L}$ ). The amplified PCR product was extracted from an agarose gel using Easytrap Ver.2 (TaKaRa). The obtained DNA was further treated with Ex Taq polymerase (TaKaRa) to attach an A residue at the end of the PCR product (standard concentrations, $72^{\circ} \mathrm{C}, 1 \mathrm{~min}$ ). The resulting DNA was ligated with the T-vector of pT7Blue, and the DNA sequence of the obtained clone was confirmed. The plasmid was digested with NdeI and EcoRI, and the resulting DNA fragment was inserted into the corresponding site of pET21a (Novagen). After confirmation of the sequence, the obtained expression plasmids for BtrL (pET-btrL) and pREP4-groESL were co-introduced into *E. coli* BL21(DE3) by a standard chemical transformation. *Escherichia coli* cells harboring pET-btrL/pREP4-groESL were grown in LB medium supplemented with kanamycin and ampicillin at $37^{\circ} \mathrm{C}$ to an OD $_{600}$ of 0.6–1.0. At this point, the fermentation temperature was lowered to $15^{\circ} \mathrm{C}$, isopropyl $\beta-\mathrm{D}$ -thiogalactoside (IPTG) was added to a final concentration of $0.1 \mathrm{mM}$ to induce expression. The culture was incubated at $15^{\circ} \mathrm{C}$ for $24 \mathrm{~h}$ and the cells were then harvested by centrifugation ($6000 \mathrm{rpm} \times 30 \mathrm{~min}$ ).

The $b t r P$ gene was similarly amplified using the primers pETbtrP-f: $5^{\prime}-\mathrm{GGAGAGATCATATGCGATTG}-3^{\prime}$ and pETbtrP-r: $5^{\prime}-\mathrm{CACTTGATACAGCGAATTCC}-3^{\prime}$ except using an extension time of $1 \mathrm{~min}$. The amplified
DNA was ligated with the T-vector of pT7Blue and then cloned into pET21a. The obtained expression plasmid for BtrP (pET-btrP) was introduced into *E. coli* BL21(DE3) and BtrP was similarly expressed at 15°C for 24h after adding IPTG to 1mM.

The *btrV* gene was amplified with the primers pETbtrV-f: 5′-GATGAAATCATATGGATAAC-3′ and pET-btrV-r: 5′-CTTGTTGTAGTAGGGGTCCA-3′ using an annealing temperature of 48 °C. The PCR extension time was 1 min. The amplified DNA was cloned into the T-vector of pT7Blue and then subcloned into pET21a (NdeI-EcoRI). The resultant BtrV expression plasmid (pET-btrV) was introduced into *E. coli* BL21(DE3) and BtrV was expressed at 18°C for 20h after adding IPTG to 0.3mM.

A plasmid containing the entire *btrA* gene (3675bp) was constructed from pDS1, 5, and 12, which were previously cloned from the chromosomal DNA derived from *B. circulans* SANK 72073. First, pDS12 was digested with NcoI/EcoRI and the fragment containing the pUC19 backbone was purified. pDS1 was digested with NcoI/EcoRI and the fragment containing a section of the *btrA* gene was recovered. The two fragments were then ligated to obtain a plasmid containing the *btrA* gene lacking an NcoI fragment (ca. 1kb). This NcoI fragment was removed from pDS5 and cloned into the NcoI site of the plasmid, after treatment with alkaline phosphatase (BAP) to obtain a plasmid containing the entire *btrA* gene (pUC19-btrA). The orientation of the NcoI fragment in the pUC19-btrA plasmid was confirmed by digestion with EcoRV and PstI. The N-terminal part of the *btrA* gene was separately amplified using the primers pETbtrA-f: 5′-CTTCATATGAATAGCTTCGG-3′ and pETbtrA-r: 5′-AGCGGGCGATATCCTCCTTA-3′ using the same PCR conditions as above, except for the extension time (1 min) and was then cloned into pT7Blue. After confirmation of the sequence, the NdeI/AccI fragment from the amplified DNA was ligated with pET30 that had been digested with NdeI/BamHI and the AccI/BamHI fragment (ca. 3.3kb) from pUC19-btrA, to generate the expression plasmid pET-btrA. After confirmation of the sequence, the pET-btrA plasmid was introduced into *E. coli* BL21(DE3) and BtrA was expressed at 18°C for 20h after addition of 0.3 mM IPTG.

The induced cell pellets (typically 3g from a 600 mL culture) were suspended in 7 volumes of 50 mM Hepes buffer (pH 8.0) and disrupted by sonic oscillation for 5 × 2 min (Sonifier Type-250, Branson). After centrifugation (10,000 rpm × 10 min), the supernatants were used as cell free extracts (CFEs) containing the expressed proteins.

### 4.3. Analysis of the ribosylation reaction

Reaction products formed after incubation of the CFEs with neamine and PRPP were converted to the corresponding dinitrophenyl derivatives (tetrakis-(2,4-dinitrophenyl)-ribostamycin) using 2,4-dinitrofluorobenzene (DNFB), to facilitate analysis. Ribostamycin (authentic sample) in 50 μL of 50 mM Hepes buffer (pH 8.0) was treated with 20 μL of 10% solution of DNFB in methanol, 10 μL of DMSO, and 3 μL of 2M NaOH at 98°C for 1h, and then ca. 8 μL of 2M NaOH was added to adjust the pH of the solution to pH 8–9. After an additional incubation for 1h at 98°C, the dinitrophenyl (DNP) derivatives were extracted with ethyl acetate (150 μL × 3) and the combined organic extracts were concentrated using a centrifugal evaporator. The residue was dissolved in 50 μL of CH₃CN and an aliquot of the solution (2 μL) was analyzed by HPLC using a PEGASIL ODS column (4.6 × 250 mm, Senshu Scientific, Japan). Samples were eluted with 50% (or 53%) aqueous CH₃CN at a flow rate of 1 mL/min, monitoring the elution at 350 nm. The DNP derivative of the authentic ribostamycin sample eluted at 25.6 min. A standard curve for HPLC analysis was prepared by reacting 10, 50, 100, and 1000 μM of ribostamycin in 50 μL of 50 mM Hepes buffer (pH 8.0) with DNFB using identical conditions. DNP-ribostamycin was also analyzed by LC-ESI mass spectrometry using a Mightysil RP-18 GP column (Kanto Chemical, Japan). An aliquot of the solution (5 μL) was injected and eluted with 10% aqueous CH₃CN for 10 min, and 80% aqueous CH₃CN for 40 min at a flow rate 50 μL/min. The elution was monitored at 350 nm and analyzed in the negative mode. LC-ESI-MS (negative): 25.2 min, *m/z* 1117.2 (M–H)⁻; calc'd for C₄₁H₄₂N₁₂O₂₆: 1118.2.

All combinations of the separate cell-free extracts (CFEs) containing BtrA, L, P, and/or V were examined to monitor ribostamycin formation after incubation with neamine and 5-phosphoribosyl-1-diphosphate (PRPP). The enzyme reaction mixtures contained 50 μL of each CFE, 10 μL of 200 mM PRPP (final concentration 8.9 mM), 10 μL of 200 mM neamine (final concentration 8.9 mM), and 5 μL of 50 mM MgCl₂ (final concentration 1.1 mM), plus 50 mM Hepes buffer (pH 8.0) to a final volume of 225 μL and were incubated at 37°C for 12 h. After heat treatment (98°C for 2 min), the precipitated protein was removed by centrifugation (12,000 rpm × 30 min) and the resulting supernatant was passed through an ultra-filtration membrane (Microcon YM-10, Millipore). The solution obtained was concentrated on a centrifugal evaporator and the residue was dissolved in 20 μL of 50 mM Hepes buffer (pH 8.0). Derivatization with DNFB was then performed as described above. Samples for HPLC and LC-ESI-MS analysis were similarly prepared.

### 4.4. Confirmation of the stepwise reaction

Using the conditions described above, an incubation of BtrL CFE (50 μL), neamine, and PRPP (37°C for 12 h) was heat quenched (98°C for 2 min), then BtrP CFE (50 μL) was added, and the mixture was incubated for a further 12 h at 37°C. The reverse order of CFE addition (i.e., BtrP then BtrL) was used in a separate reaction, performed under analogous conditions. The enzyme reaction products were then derivatized and analyzed by HPLC as described above.The derivatization and analysis of the product formed after incubation of BtrL CFE, neamine, and PRPP was performed as above except extracting after acidification with 2M HCl to pH 4–5. An aliquot of the solution was analyzed by LC-ESI-MS (as described above, except eluting with 10% aqueous CH₃CN for 10 min, then 50% aqueous CH₃CN for 40 min, at a flow rate 50 μL/min). The elution was monitored at 350 nm and analyzed in the negative mode. LC-ESI-MS (negative): 13.5 min, *m/z* 1197.2 (*M*−H)⁻; calc'd for C₄₁H₄₁N₁₂O₂₉P 1198.2.

### 4.5. Purification of BtrP

The cell free extract containing the BtrP protein (prepared from 3 g wet cell pellet using the method described above) was loaded onto a column (2.5 × 6 cm) of DEAE-Sephrose Fast Flow that had been equilibrated with 50 mM Tris buffer (pH 8.0). The adsorbed proteins were eluted using the same buffer with linear gradient of NaCl (0 to 400 mM, total 1 L). The fractions containing BtrP were concentrated to ca. 2 mL by ultrafiltration (Vivaspin 20, 10,000 MW, Vivascience). The concentrate was then loaded onto a Hi Load 26/60 Superdex 200 pg (FPLC) that had been equilibrated with 50 mM Tris buffer (pH 8.0). After isocratic elution, the fraction containing BtrP was concentrated (to ca. 300 μM) and the solution was stored at −78 °C. The molecular weight of native BtrP was estimated to be 50 kDa by comparison with the retention time of protein standards, which indicates that BtrP exists as a homodimer.

### 4.6. Properties of BtrP

BtrP phosphatase activities were assayed using 100 μM of *p*-nitrophenyl phosphate as substrate, by monitoring the characteristic absorbance of *p*-nitrophenolate at 410 nm (*ε*~max~ 1.75 × 10⁴ M⁻¹cm⁻¹). To establish the metal ion requirement, MgCl₂, MnCl₂, and EDTA were used at a final concentration of 1 mM. The substrate specificity of BtrP was investigated by separate incubations with 100 μM of PRPP, fructose 1,6-bisphosphate, fructose 6-phosphate, ribose 5-phosphate, AMP, CMP, GMP, UMP, and dTMP in 50 mM Tris buffer (pH 8.0), for 1 h at room temperature. The reaction was initiated by the addition of BtrP (final concentration 20 μM), and an aliquot of the product solution (100 μL) was reacted with 1 mL of BIOMOL GREEN (Funakoshi) at room temperature for 1 h. Released free inorganic monophosphate was quantified using the absorbance at 620 nm.

### 4.7. Properties of BtrL

The biochemical properties of BtrL (as a CFE) were characterized using a coupled assay that included the BtrP CFE, by quantifying the amount of ribostamycin formed. The enzyme reaction mixture contained 75 μL of BtrL cell free extract, 5 μL of 200 mM neamine (final concentration 10 mM), 5 μL of 200 mM PRPP (final concentration 10 mM), 5 μL of 20 mM MgCl₂ (final concentration 1 mM), and 10 μL of 200 μM BtrP (final concentration 20 μM) in 50 mM Hepes buffer containing 10% glycerol. After 3-h incubation at 37 °C, the reaction was quenched by addition of 100 μL of acetone. The reaction mixture was concentrated using a centrifugal evaporator and the residue was dissolved in 50 μL of water. The product mixture was derivatized with DNFB as above. The optimum pH was determined by performing identical enzymatic incubations (at 37 °C) containing 50 mM Hepes buffer with pH values of 6.5–9.0. To determine the metal ion requirement, MgCl₂, MnCl₂, CoCl₂, and NiCl₂ were used at a final concentration of 1 mM. ‘Metal-free’ conditions were achieved by adding EDTA to a final concentration of 1 mM. For the determination of optimum reaction temperature, the enzyme reaction was performed at a range of temperatures between 27 and 42 °C. Under optimum conditions (50 mM HEPES buffer (pH 7.5) containing 10% glycerol with 1 mM of MgCl₂ and 20 μM of BtrP at 32 °C), the substrate specificity of BtrL/BtrP was investigated using 10 mM of PRPP and a variety of ribosyl acceptors, including: paromamine, DOS, and kanamycin A (at a final concentration of 10 mM). The reaction products were similarly converted to the dinitrophenyl derivatives, which were analyzed by HPLC and LC-ESI-MS with slightly modified separation conditions. Paromamine; HPLC, 57% aqueous CH₃CN; LC-ESI-MS, 10% aqueous methanol for 10 min, and 70% aqueous methanol for 40 min. DOS; LC-ESI-MS, 10% aqueous methanol for 10 min, and 70% aqueous methanol for 40 min. Kanamycin A; LC-ESI-MS, 10% aqueous CH₃CN for 10 min, and 80% aqueous CH₃CN.

---

### Acknowledgments

We thank Prof. Craig A. Townsend at The Johns Hopkins University for generously supplying us with the pREP4-groESL plasmid, and also Dr. Rory M. Watt at The University of Hong Kong, who helped us with the preparation of this manuscript. This work was supported in part by a Grant-in-Aid for Scientific Research from the Ministry of Education, Culture, Science, Sports, and Technology, Japan, and the Uehara Memorial Foundation.

---

### References and notes

1. Kudo, F.; Tamegai, H.; Fujiwara, T.; Tagami, U.; Hirayama, K.; Kakinuma, K. *J. Antibiot.* **1999**, *52*, 559.
2. Tamegai, H.; Nango, E.; Kuwahara, M.; Yamamoto, H.; Ota, Y.; Kuriki, H.; Eguchi, T.; Kakinuma, K. *J. Antibiot.* **2002**, *55*, 707.
3. Huang, F.; Li, Y.; Yu, J.; Spencer, J. B. *Chem. Commun.* **2002**, 2860.
4. Llewellyn, N. M.; Spencer, J. B. *Nat. Prod. Rep.* **2006**, *23*, 864.
5. Huang, F.; Haydock, S. F.; Mironenko, T.; Spiteller, D.; Li, Y.; Spencer, J. B. *Org. Biomol. Chem.* **2005**, *3*, 1410.
6. Kudo, F.; Yamamoto, Y.; Yokoyama, K.; Eguchi, T.; Kakinuma, K. *J. Antibiot.* **2005**, *58*, 766.
7. Li, Y.; Llewellyn, N. M.; Giri, R.; Huang, F.; Spencer, J. B. *Chem. Biol.* **2005**, *12*, 665.
8. Cole, P. A. *Structure* **1996**, *4*, 239.
9. Sinha, S. C.; Smith, J. L. *Curr. Opin. Struct. Biol.* **2001**, *11*, 733.
10. Huang, H.; Scherman, M. S.; D'Haeze, W.; Vereecke, D.; Holsters, M.; Crick, D. C.; McNeil, M. R. *J. Biol. Chem.* 2005, 280, 24539.
11. Lundegaard, C.; Jensen, K. F. *Biochemistry* 1999, 38, 3327.
12. Rigden, D. J.; Lamani, E.; Mello, L. V.; Littlejohn, J. E.; Jedrzejas, M. *J. Mol. Biol.* 2003, 328, 909.
13. Kudo, F.; Numakura, M.; Tamegai, H.; Yamamoto, H.; Eguchi, T.; Kakinuma, K. *J. Antibiot.* 2005, 58, 373.
14. Kida, M.; Asako, T.; Yoneda, M.; Mitsuhashi, S. *Microb. Drug Resist.* 1975, 441.
15. Aubert-Pivert, E.; Davies, J. *Gene* 1994, 147, 1.
16. Sambrook, J.; Fritsch, E. F.; Maniatis, T. *Molecular Cloning: A Laboratory Manual*, 2nd ed.; Cold Spring Harbor Laboratory: Cold Spring Harbor, NY, 1989.
17. Ota, Y.; Tamegai, H.; Kudo, F.; Kuriki, H.; Koike-Takeshita, A.; Eguchi, T.; Kakinuma, K. *J. Antibiot.* 2000, 53, 1158.
