
Environmental Microbiology (2007) 9(6), 1535–1540

doi:10.1111/j.1462-2920.2007.01272.x

# Escherichia coli has multiple enzymes that attack TNT and release nitrogen for growth

M. Mar González-Pérez, Pieter van Dillewijn, Rolf-M. Wittich and Juan L. Ramos*

Estación Experimental del Zaidín, Consejo Superior de Investigaciones Científicas, Apdo. Correos 419, E-18008 Granada, Spain.

## Summary

There has been a growing interest in the degradation of 2,4,6-trinitrotoluene (TNT) over the last decade, ever since its removal from polluted sites was declared an international environmental priority. Certain aerobic and anaerobic microorganisms are capable of using TNT as an N source, although very few studies have proven the mineralization of this compound. An unexpected observation in our laboratory led us to discover that certain *Escherichia coli* bench laboratory strains have multiple enzymes that attack TNT. One of the NemA products is responsible for the release of nitrite from the nitroaromatic ring: among the metabolites observed *in vitro* include Meisenheimer dihydride complexes of TNT from which 2-hydroxylamino-6-nitrotoluene is slowly formed during their rearomatization under concomitant release of nitrite. Furthermore, NemA, together with NfsA and NfsB reduce the nitro groups on the aromatic ring to the corresponding hydroxylamine derivatives, which probably results in the release of ammonium ions which can, in turn be used as a nitrogen source by *E. coli* for growth.

## Introduction

TNT (2,4,6-trinitrotoluene) degradation has attracted the attention of many scientists in the last decade, ever since a number of environmental protection agencies declared the removal of TNT a priority (Esteve-Núñez *et al.*, 2001; Ramos *et al.*, 2004). TNT is a major contaminant in many military sites and explosive manufacturing factories, where production and decommissioning activities generate large quantities of this explosive as a waste product (Esteve-Núñez *et al.*, 2001). Much of this waste is deposited in the soil and in unlined lagoons from which it can reach groundwaters through leaching (Selim *et al.*, 1995).

*Pseudomonas* sp. strain JLR11, isolated from a wastewater treatment plant, is able to use nitrate, nitrite and TNT as the N source under oxic- and anoxic conditions (Esteve-Núñez and Ramos, 1998; Esteve-Núñez *et al.*, 2000). The utilization of TNT as an N-source involves the removal of nitrogen from the nitroarene ring. This is achieved either through the direct removal of nitrite, which is in turn reduced to ammonium ions (Esteve-Núñez and Ramos, 1998; Kitts *et al.*, 2000; Riefler and Smets, 2002; Kwak *et al.*, 2003), or through the removal of nitrogen as ammonium after the reduction of a nitro group on the ring to its hydroxylamine derivative, which upon a Bamberger-like reaction could form intermediates, which could promote the release of ammonium (Caballero and Ramos, 2006). The reduction of a nitro group to its corresponding hydroxylamine moiety is catalysed by nitroreductases (Hughes *et al.*, 1998a; Caballero *et al.*, 2005).

In the course of our attempts to clone and identify other genes of *Pseudomonas* JLR11 encoding proteins involved in TNT metabolism, we designed a cloning strategy using *Escherichia coli* as a host strain. The series of controls indicated that *E. coli* was able to use TNT as the sole nitrogen source, thereby confirming recent findings (Stenuit *et al.*, 2006). In fact, it has long been known that *E. coli* cells growing aerobically or anaerobically are able to reduce up to two of the nitro groups of TNT to yield 2,4-diamino-6-nitrotoluene (McCormick *et al.*, 1976; Ederer *et al.*, 1997; Fuller and Manning, 1997). Recently, Yin and colleagues (2004) have identified hydroxylaminodinitrotoluene isomers as potential pathway intermediates in the reduction pathway. This finding is in agreement with the description of two nitroreductases (NfsA and NfsB) in *E. coli*, and the identification of a xenobiotic reductase [N-ethylmaleimide (NEM) reductase] that reduces TNT *in vitro* (Kwak *et al.*, 2003; Williams *et al.*, 2004). This prompted us to analyse whether the *nfsA*, *nfsB* and *nemA* gene products were involved in TNT metabolism in *E. coli*.

## Results and discussion

*Escherichia coli* AB1157 is a wild-type strain that efficiently uses TNT as an N source (our own observations) and for which the *nfsA* and the double *nfsA/nfsB* mutants

Received 20 November, 2006; accepted 4 January, 2007. *For correspondence. E-mail jlramos@eez.csic.es; Tel. (+34) 958 181608; Fax (+34) 958 135740.*

© 2007 The Authors  
Journal compilation © 2007 Society for Applied Microbiology and Blackwell Publishing Ltd

1536 M. M. González-Pérez, P. van Dillewijn, R.-M. Wittich and J. L. Ramos

Table 1. *Escherichia coli* strains used in this study and their ability to grow at the expense of TNT as the sole N source.

| Strain       | Genotype      | cfu ml⁻¹     | %TNT remaining |
|--------------|---------------|--------------|----------------|
| AB1157       | wt            | 1.2 × 10⁸    | 0              |
| NFR402       | nfsA          | 1 × 10⁸      | 5              |
| ABnfsB       | nfsB          | 1 × 10⁸      | 5              |
| ABnemA       | nemA          | 1.1 × 10⁸    | 5              |
| NFR502       | nfsA/nfsB     | 1 × 10⁷      | 70             |
| NFR402nemA   | nfsA/nemA     | 0.9 × 10⁷    | 20             |
| ABnfsBnemA   | nfsB/nemA     | 9 × 10⁷      | 20             |
| 502nemA      | nfsA/nfsB/nemA| 3 × 10⁵      | 100            |

The strains were grown under aerobic conditions in minimal medium with glucose as a C source and TNT as an N source. The initial number of cells determined as cfu ml⁻¹ was in the range of 3–6 × 10⁵. The initial TNT concentration was 300 μM (100%). Values in the Table correspond to measurements done after 48 h incubation at 30°C with shaking. Results are the mean of three replicates. Standard errors were below 5% of the given values.

were not produced. These results are consistent with those by Kwak and colleagues (2003) who analysed the nitroreductase activity of *E. coli* NfsA and NfsB with TNT as substrate *in vitro*. However, our study describes the first direct implication of these two nitroreductases in *in vivo* TNT degradation.

The limited consumption of TNT when NemA was combined with either NfsA or NfsB indicates that NEM reductase could also contribute to TNT metabolism in *E. coli*. To further confirm this hypothesis, we decided to construct the triple mutant strain 502nemA in which the three genes had been inactivated (Table 1). In this case the mutant strain was not able to metabolize TNT and did not grow with this nitroarene as an N source (Table 1). This result confirms that NfsA, NfsB and NEM reductase all play a role in TNT metabolism in *E. coli*.

N-ethylmaleimide reductase (Miura et al., 1997) belongs to a set of proteins of the OYE protein family that have two potential enzymatic activities on TNT: they reduce the aromatic ring to produce a set of Meisenheimer hydride complexes, ‘orange products’, and release nitrite from the biotransformed TNT molecule (Williams et al., 2004). On the other hand, they are also able to reduce nitro groups on the TNT aromatic ring (French et al., 1995). To clarify the role of NemA in TNT metabolism we decided to analyse the *in vivo* and *in vitro* activity of the protein with TNT. To this end the *nemA* gene was cloned into pET28b to yield pET28bNEM. Plasmid pET28bNEM has an IPTG-inducible promoter that allows the synthesis of large amounts of protein (almost 1% of total protein, not shown). In a series of resting cell assays we observed that *E. coli* (pET28b-NEM) consumed TNT three times faster than cells that did not bear this plasmid. Up to 300 μM of nitrite was released from TNT supplied in excess in the culture medium (Fig. 1).

To unequivocally confirm the involvement *in vitro* of denitrating activity mediated by NEM reductase, we purified the enzyme to homogeneity, as described in Experimental procedures, and carried out *in vitro* assays with TNT as the substrate and NADPH as the electron donor. These assays showed transient production of the Meisenheimer monohydride complex followed by Meisenheimer dihydride complex formation and concomitant orange colouring of the reaction mixture and nitrite release. The activity of transformation was 5 mU × mg⁻¹ protein for nitrite release. On the other hand, both 2-hydroxylamino-4,6-dinitrotoluene (2HADNT) and 4-hydroxylamino-2,6-dinitrotoluene (4HADNT) were produced rapidly *in vitro* and then consumed thereafter. The specific activity for total HADNT production was 50 mU HADNT × mg⁻¹ protein within the first 20 min of the reaction. Therefore, the NEM reductase protein may exhibit dual enzymatic activities, as it was reported for the xenobiotic reductase XenB of *Pseudomonas fluorescens* (Bleher et al., 1999),

© 2007 The Authors
Journal compilation © 2007 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, **9**, 1535–1540

E. coli has multiple enzymes that attack TNT 1537

of the Meisenheimer dihydride complex (Fig. 2). The signals derived from NMR were as follows (proton chemical shifts converted to the TMS (tetramethylsilane) scale, in CDCl₃): the singlet at δ = 2.42 p.p.m. represents the methyl group while the doublet at δ = 7.44 (J = 8.09 Hz) followed by a triplet at δ = 7.19 p.p.m. (J_AB = 8.14, J_BC = 8.12 Hz) and another doublet at δ = 7.01 p.p.m. (J = 7.99 Hz) represents a classical ABC spectrum of three adjacent, non-equivalent protons, and the broad hydroxylamine signal found at 5.21 p.p.m.

The rearomatization of Meisenheimer hydride complexes with nitrite release has been shown before for trinitrophenol (picric acid) degradation in Rhodococcus (Hoffmann et al., 2004). Furthermore, Vorbeck and colleagues (1998) found that the Meisenheimer monohydride complex of TNT will rearomatize to TNT, suggesting that rearomatization of the Meisenheimer dihydride complex of TNT to a product such as 2-hydroxylamino-6-nitrotoluene is not unfeasible. Nevertheless, the process of rearomatization is a slow abiotic process, which does not occur at rates high enough to explain the large amounts of nitrite released in the in vitro reaction with NEM reductase. Recently, Stenuit and colleagues (2006) suggested that nitrite release results from the dimerization of HADNT with Meisenheimer dihydride complexes as previously had been suggested by Pak and colleagues (2000), to form isomeric derivatives of substituted biphenyls.

The utilization of N derived from TNT via the denitration process requires the reduction of nitrite to ammonium via assimilatory nitrite reductase activity. Escherichia coli does not possess an assimilatory nitrite reductase and therefore this type of reaction cannot be the source of N for growth (Neidhardt, 1996). Therefore, we hypothesize that E. coli assimilates N from TNT via the release of ammonium from TNT similar to Pseudomonas putida JLR11 mutants disabled in nitrite reductase (Caballero and Ramos, 2006). In this strain, which can use TNT as an N-source, the release of ammonium from TNT has been observed (Caballero and Ramos, 2006). In the case of E. coli, reactions catalyzed by NfsA, NfsB and NEM reductase reduce a nitro group on the ring to its hydroxylamine derivative. The removal of nitrogen as ammonium then takes place after the hydroxylamine derivative undergoes a Bamberger-like rearrangement to intermediates propitious for ammonium release by as yet unidentified enzymes (Fig. 2, upper right) (Hughes et al., 1998b; Nadeau et al., 2003; Williams et al., 2004; Stenuit et al., 2006).

The ability to degrade aromatic compounds is frequently associated to soil bacteria such as Pseudomonas strains (Díaz et al., 2001; Esteve-Núñez et al., 2001). Here we confirm that easy to manipulate E. coli strains are able to use TNT as a nitrogen source which opens a

© 2007 The Authors  
Journal compilation © 2007 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, **9**, 1535–1540

1538 M. M. González-Pérez, P. van Dillewijn, R.-M. Wittich and J. L. Ramos

![Diagram](attachment:chemical-reaction-diagram.png)

Fig. 2. Proposed bacterial pathways for removal of nitrogen from TNT through reduction of the aromatic ring and subsequent rearomatization, and by reduction of nitro groups, which could be hydrolytically released as ammonium upon a previous Bamberger rearrangement.

wide series of opportunities for biotechnological applications for the removal of TNT from polluted sites in ex situ applications, in particular because the strains used in this study tolerate subproducts. Structural evidence for a denitrated compound obtained from biologically produced Meisenheimer dihydride complexes has been provided here for the first time. Furthermore, the triple mutant that is unable to attack TNT is now an excellent host to search for enzymatic activities present in other microorganisms and will favour genomic and metagenomic approaches.

### Experimental procedures

#### Chemicals
TNT was obtained from Union Española de Explosivos (Madrid, Spain) and was more than 99% pure. 4-hydroxylamino-2,6-dinitrotoluene was obtained from AccuStandard.

#### Organism, culture medium and growth conditions
The E. coli strains used in this study are listed and described in Table 1. The strains were grown on M9 minimal medium with glucose (20 mM) as a carbon source at 30°C. When TNT was used as the sole N source, it was supplied at 100 mg l⁻¹ or as crystal in excess of its solubility in water; and ammonium was omitted from M9 medium. Growth was measured by counting cfu ml⁻¹ after spreading serial dilutions on Luria–Bertani (LB) medium plates (Carpenter et al., 1978).

#### Analytical methods
Nitrite concentration was determined by the method of Snell (Snell and Snell, 1949). TNT concentration was analysed by high-performance liquid chromatography on a Hewlett-Packard model 1050 chromatograph equipped with a diode array detector and a 5-µm C18RP column (UltraCarb C30 Phenomenex, 15 cm by 4.6 mm). The column was first washed with a mixture of acetonitrile and a solution of 1%

© 2007 The Authors  
Journal compilation © 2007 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, **9**, 1535–1540
(v/v) acetic acid in water (2:8 [v/v]) for 2 min. A linear gradient was then applied to reach 100% (v/v) acetonitrile over 18 min. The flow was kept constant at 1 ml min⁻¹, and the detector was set at 230 and 254 nm to detect aromatic compounds. To detect hydroxylaminodinitrotoluenes, the same column was used and the samples run for 20 min in a 30% methanol : water solution at 0.9 ml min⁻¹ flow rate with the detector set at 230 nm.

### DNA techniques

Preparation of chromosomal DNA, digestion with restriction enzymes and electrophoresis were carried out using standard methods (Sambrook et al., 1989).

### Escherichia coli mutant strains construction

For generation of single, double and triple mutant strains we used the pKO3 vector system (Link et al., 1997). 2.5–3 kb DNA fragments containing the target genes (*nfsB* and/or *nemA*) and flanking regions were amplified by polymerase chain reaction (PCR) and cloned in pGemT-easy vector cloning kit (Promega, Madison, WI). Km or Gm cassettes from pUC18K (Menard et al., 1993) and pCR2.1-Gm, respectively, were then introduced into a unique site within the coding region of the target genes. The nature of the knock out was confirmed by PCR amplification using primers based on *nfsB* and *nemA* gene sequences, and primers based on the sequence of the antibiotic resistance marker were used to inactivate the target gene. The *nfsB*-Gm and *nemA*-Km alleles were released after BamHI digestion, gel purified and cloned into the single BamHI site in pKO3 vector. This construction was transformed into *E. coli*, and Kmᴿ or Gmᴿ transformants were selected and used for gene replacement exactly as described by Link and colleagues (1997).

### Overexpression of *nemA* and purification of NemA

The *nemA* gene (1100 bp) was PCR-amplified from *E. coli* chromosomal DNA, by using primers with BamHI and HindIII (5′-CGGGATCCTGCAGGCAAAAT-3′ and 5′-CCCAAGCTT CAACGTCCGGGTAATC-3′ respectively) sites. After digestion with these restriction enzymes, the PCR products were ligated into pET28b(+) vector (Novagen) previously digested with BamHI and HindIII. The resulting plasmid, pET29b-NemA, contained the *nemA* coding sequence in frame with a DNA sequence encoding a His₆-tag at its 3′ end. For NemA-His₆ purification, pET28b-NemA was transformed into *E. coli* BL21 (pLys). The cells were grown in several 1-l batches at 30°C in 2 × YT culture medium (Sambrook et al., 1989) with 50 μg ml⁻¹ kanamycin to an A₆₆₀ between 0.5 and 0.7 and then induced with 1 mM IPTG. Cells were harvested after overnight induction at 20°C, resuspended in 25 mM sodium phosphate (pH 7.0), 0.5 M NaCl, 5% (v/v) glycerol and protease inhibitor cocktail (Complete™, Roche) and broken by treatment with 20 μg ml⁻¹ of lysozyme and French press. Following centrifugation at 20 000 g for 30 min, the protein was found predominantly (more than 80%) in the soluble fraction. NemA-His₆ was purified by nickel affinity chromatography and eluted with a continuous imidazole gradient. Peak fractions were pooled and dialysed against the gel-filtration buffer (500 mM NaCl, 10% glycerol, 25 mM sodium phosphate buffer pH 7.5, 0.1 mM EDTA and 1 μM FMN). The dialysed sample was then run over a gel-filtration column (Pharmacia) in the same buffer and the peak fractions pooled. Protein concentrations were determined using the Bio-Rad Protein Assay kit.

### Enzyme assays

The aromatic ring reductase activity of NemA was determined measuring the concentration of TNT reduced in the presence of NADPH at 30°C. Assay conditions were: 50 mM phosphate buffer (pH 7.0), 0.1 mM of NADPH, 0.1 mM of TNT and a suitable amount of enzyme. One unit of activity was defined as the amount of enzyme required to release 1 μmol of nitrite per minute from TNT under the standard assay conditions. The nitroreductase activity of NemA was determined at 25°C in 25 mM phosphate buffer (pH 7.5) with 0.2 mM TNT, 0.7 mM NADPH. One unit of activity was defined as the amount of enzyme required to release 1 μmol of hydroxylaminodinitrotoluene per minute from TNT under standard assay conditions.

### Acknowledgements

This study was supported by a grant from the European Commission (MADOX QLRT-2001-00345) and a grant from Ministerio de Educación y Ciencia (BIO 2003-0505). We thank Alí Haïdour from the University of Granada for running the NMR spectrum and Fernando Lafont from the University of Córdoba for mass spectrometry analysis. We also thank M. Mar Fandila and Carmen Lorente for secretarial assistance.

### References

Blehert, D.S., Fox, B.G., and Chambliss, G.H. (1999) Cloning and sequence analysis of two *Pseudomonas* flavoprotein xenobiotic reductases. *J Bacteriol* **181**: 6254–6263.

Caballero, A., and Ramos, J.L. (2006) A double mutant of *Pseudomonas putida* JLR11 deficient in the synthesis of the nitroreductase PnrA and assimilatory nitrite reductase NasB is impaired for growth on 2,4,6-trinitrotoluene (TNT). *Environ Microbiol* **8**: 1306–1310.

Caballero, A., Lázaro, J.J., Ramos, J.L., and Esteve-Núñez, A. (2005) PnrA, a new nitroreductase-family enzyme in the TNT-degrading strain *Pseudomonas putida* JLR11. *Environ Microbiol* **7**: 1211–1219.

Carpenter, D.F., McCormick, N.G., Cornell, J.H., and Kaplan, A. (1978) Microbial transformation of ¹⁴C-labeled 2,4,6-trinitrotoluene in an activated-sludge system. *Appl Environ Microbiol* **35**: 949–954.

Díaz, E., Ferrández, A., Prieto, M.A., and García, J.L. (2001) Biodegradation of aromatic compounds by *Escherichia coli*. *Microbiol Mol Biol Rev* **65**: 523–569.

Ederer, M.M., Lewis, T.A., and Crawford, R.L. (1997) 2,4,6-Trinitrotoluene transformation by Clostridia isolated from a munition-fed bioreactor: comparison with non-adapted bacteria. *J Indust Microbiol Biotech* **18**: 82–88.

© 2007 The Authors  
Journal compilation © 2007 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, **9**, 1535–1540
1540 M. M. González-Pérez, P. van Dillewijn, R.-M. Wittich and J. L. Ramos

Esteve-Núñez, A., and Ramos, J.L. (1998) Metabolism of 2,4,6-trinitrotoluene by *Pseudomonas* sp. JLR11. *Environ Sci Tech* **32**: 3802–3808.

Esteve-Núñez, A., Luchessi, G., Philipps, B., Schink, B., and Ramos, J.L. (2000) Respiration of 2,4,6-trinitrotoluene by *Pseudomonas* sp. strain JLR11. *J Bacteriol* **182**: 1352–1355.

Esteve-Núñez, A., Caballero, A., and Ramos, J.L. (2001) Biological degradation of 2,4,6-trinitrotoluene. *Microbiol Mol Biol Rev* **65**: 335–352.

French, C.E., Nicklin, S., and Bruce, N.C. (1995) Sequence and properties of pentaerythritol tetranitrate reductase from *Enterobacter cloacae* PB2. *J Bacteriol* **178**: 6623–6627.

French, C.E., Nicklin, S., and Bruce, N.C. (1998) Aerobic degradation of 2,4,6-trinitrotoluene by *Enterobacter cloacae* PB2 and by pentaerythritol tetranitrate reductase. *Appl Environ Microbiol* **64**: 2864–2868.

Fuller, M.E., and Manning, J.F., Jr (1997) Aerobic Gram-positive and Gram-negative bacteria exhibit differential sensitivity to and transformation of 2,4,6-trinitrotoluene (TNT). *Curr Microbiol* **35**: 77–83.

Hoffmann, K.W., Knackmuss, H.J., and Heiss, G. (2004) Nitrite elimination and hydrolytic ring cleavage in 2,4,6-trinitrophenol (picric acid) degradation. *Appl Environ Microbiol* **70**: 2854–2860.

Hughes, J.B., Wang, C., Yesland, K., Bhadra, R., Richardson, A., Bennet, G., and Rudolph, F. (1998a) Reduction of 2,4,6-trinitrotoluene by *Clostridium acetobutylicum* through hydroxylamino intermediates. *Environ Toxicol Chem* **17**: 343–348.

Hughes, J.B., Wang, C., Yesland, K., Richardson, A., Bhadra, R., Bennett, G., and Rudolph, F. (1998b) Bamberger rearrangement during TNT metabolism by *Clostridium acetobutylicum*. *Environ Sci Technol* **32**: 494–500.

Kitts, C.L., Green, C.E., Otley, R.A., Alvarez, M.A., and Unkefer, P.J. (2000) Type I nitroreductases in soil enterobacteria reduce TNT (2,4,6-trinitrotoluene) and RDX (hexahydro-1,3,5-trinitro-1,3,5-triazine). *Can J Microbiol* **46**: 278–282.

Kwak, Y.H., Lee, D.S., and Kim, H.B. (2003) *Vibrio harveyi* nitroreductase is also a chromate reductase. *Appl Environ Microbiol* **69**: 4390–4395.

Link, A.J., Philips, D., and Church, G.M. (1997) Methods for generating precise deletions and insertions in the genome of wild-type *Escherichia coli*: applications to open reading frame characterization. *J Bacteriol* **179**: 6228–6237.

McCalla, D.R., Christel, K., and Green, M.H. (1978) Genetics of nitrofurazone resistance in *Escherichia coli*. *J Bacteriol* **133**: 10–16.

McCormick, N.G., Feeherry, F.E., and Levinson, H.S. (1976) Microbial transformation of 2,4,6-trinitrotoluene and other nitroaromatic compounds. *Appl Environ Microbiol* **31**: 949–958.

Menard, R., Sansonetti, P.J., and Parsot, C. (1993) Non polar mutagenesis of the *ipa* genes defines IpaB, IpaC and IpaD as effectors of *Shigella flexneri* entry into epithelial cells. *J Bacteriol* **175**: 5899–5906.

Miura, K., Tomioka, Y., Suzuki, H., Yonezawa, M., Hishinuma, T., and Mizuyaki, M. (1997) Molecular cloning of the *nemA*

gene encoding N-ethylmaleimide reductase from *Escherichia coli*. *Biol Pharm Bull* **20**: 110–112.

Nadeau, L.J., He, Z., and Spain, J.C. (2003) Bacterial conversion of hydroxylamino aromatic compounds by both lyase and mutase enzymes involves intramolecular transfer of hydroxyl groups. *Appl Environ Microbiol* **69**: 2786–2793.

Neidhardt, F.C. (1996) *Escherichia coli and Salmonella Cellular and Molecular Biology*. Washington, DC, USA: American Society for Microbiology Press.

Pak, J.W., Knoke, K.L., Noguera, D.R., Fox, B.G., and Chambliss, G.H. (2000) Transformation of 2,4,6-trinitrotoluene by purified xenobiotic reductase B from *Pseudomonas fluorescens* I-C. *Appl Environ Microbiol* **66**: 4742–4750.

Ramos, J.L., Caballero, A., Duque, E., van Dillewijn, P., González-Pérez, M.M., and Esteve-Núñez, A. (2004) Physiological evidence for respiration of TNT by *Pseudomonas* sp. JLR11. In *Pseudomonas*, Vol. 3. Ramos, J.L. (ed.). London, UK: Kluwer Academic/Plenum Publisher, pp. 229–240.

Riefler, P.-G., and Smets, B.F. (2002) NAD(P)H: Flavin mononucleotide oxidoreductase inactivation during 2,4,6-trinitrotoluene reduction. *Appl Environ Microbiol* **68**: 1690–1696.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) *Molecular Cloning: A Laboratory Manual*, 2nd edn. Cold Spring Harbor, NY, USA: Cold Spring Harbor Laboratory Press.

Selim, H.M., Xue, S.K., and Iskandar, I.K. (1995) Transport of 2,4,6-trinitrotoluene and hexahydro-1,3,5-trinitro-1,3,5-triazine in soils. *Soil Sci* **160**: 328–339.

Snell, F.D., and Snell, C.T. (1949) *Colorimetric Methods of Analysis*, Vol. 3. New York, USA: Van Nostrand Co., pp. 804–805.

Stenuit, B., Eyers, L., Rozenberg, R., Habib-Jiwan, J.L., and Agathos, S.N. (2006) Aerobic growth on 2,4,6-trinitrotoluene (TNT) as sole nitrogen source by *Escherichia coli* and evidence of TNT denitrification with whole cells and cell-free extracts. *Appl Environ Microbiol* **72**: 7945–7948.

Vorbeck, C., Lenke, H., Fischer, P., Spain, J.C., and Knackmuss, H.-J. (1998) Initial reductive reactions in aerobic microbial metabolism of 2,4,6-trinitrotoluene. *Appl Environ Microbiol* **64**: 246–252.

Whiteway, J., Koziarz, P., Veall, J., Sandhu, N., Kumar, P., Hoecher, B., and Lambert, I.B. (1998) Oxygen-insensitive nitroreductases: analysis of the roles of * nfsA* and * nfsB* in development of resistance to 5-nitrofuran derivatives in *Escherichia coli*. *J Bacteriol* **180**: 5529–5539.

Williams, R.E., Rathbone, D.A., Scrutton, N.S., and Bruce, N.C. (2004) Biotransformation of explosives by the old yellow enzyme family of flavoproteins. *Appl Environ Microbiol* **70**: 3566–3574.

Yin, H., Wood, T.K., and Smets, B.F. (2004) Reductive transformation of TNT by *Escherichia coli*: pathway description. *Appl Microbiol Biotechnol* **67**: 397–404.

Zенко, S., Hoike, H., Kumar, A.N., Jayaraman, R., Tanokura, M., and Saigo, K. (1996) Biochemical characterization of NfsA. The *Escherichia coli* major nitroreductase exhibiting a high amino acid sequence homology to Frp, a *Vibrio harveyi* flavin oxidoreductase. *J Bacteriol* **178**: 4508–4514.
