
Chlorophyll Biosynthesis in Bacteria: The Origins of Structural and Functional Diversity

Aline Gomez Maqueo Chew  
and Donald A. Bryant  

Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802; email: aug111@psu.edu, dab14@psu.edu

Annu. Rev. Microbiol. 2007. 61:113–129

First published online as a Review in Advance on June 5, 2007

The Annual Review of Microbiology is online at micro.annualreviews.org

This article’s doi: 10.1146/annurev.micro.61.080706.093242

Copyright © 2007 by Annual Reviews. All rights reserved

0066-4227/07/1013-0113$20.00

**Key Words**

bacteriochlorophyll, photosynthesis, phototrophy, tetrapyrrole

**Abstract**

The use of photochemical reaction centers to convert light energy into chemical energy, chlorophototrophy, occurs in organisms belonging to only five eubacterial phyla: Cyanobacteria, Proteobacteria, Chlorobi, Chloroflexi, and Firmicutes. All chlorophototrophs synthesize two types of pigments: (a) chlorophylls and bacteriochlorophylls, which function in both light harvesting and uniquely in photochemistry; and (b) carotenoids, which function primarily as photoprotective pigments but can also participate in light harvesting. Although hundreds of carotenoids have been identified, only 12 types of chlorophylls (Chl $a$, $b$, $d$; divinyl-Chl $a$ and $b$; and $8^{1}$-hydroxy-Chl $a$) and bacteriochlorophylls (BChl $a$, $b$, $c$, $d$, $e$, and $g$) are currently known to occur in bacteria. This review summarizes recent progress in the identification of genes and enzymes in the biosynthetic pathways leading to Chls and BChls, the essential tetrapyrrole cofactors of photosynthesis, and addresses the mechanisms for generating functional diversity for solar energy capture and conversion in chlorophototrophs.

Contents
INTRODUCTION 114
CHLOROPHYLL AND BACTERIOCHLOROPHYLL BIOSYNTHESIS IN BACTERIA 114
The Core Pathway:
Protoporphyrin-IX to Chlorophyllide $a$ 115
Further Modifications to the Chlorin Ring: BChl $a$ 117
Further Modifications to the Chlorin Ring: Chlorobium chlorophylls: BChl $c, d$, and $e$ 120
Further Modifications to the Bacteriochlorin Ring: BChl $b$ and BChl $g$ 121
Aerobic versus Anaerobic Synthesis: AcsF, BchE, and Formation of the C-3 and C7 Formyl Groups 121
CLOSING REMARKS 122

(green sulfur bacteria, GSB) and Cyanobacteria are photolithoautotrophs, but only some members of the Chloroflexi (filamentous anoxygenic phototrophs, FAPs), Proteobacteria (purple bacteria, PB), and Firmicutes (heliobacteria) are phototrophic.

All chlorophototrophs synthesize two types of pigments: $(a)$ bacteriochlorophylls (BChls) and/or chlorophylls (Chls), which function in both light harvesting and photochemistry, and $(b)$ carotenoids, which primarily act as photoprotective pigments but can also function in light harvesting (19). Twelve types of Chls (Chl $a, b, d$; divinyl-Chl $a$ and $b$; $8^{1}$-hydroxy-Chl $a$) and BChls (BChl $a, b$, $c, d, e$, and $g$) have been identified; the diversity in their structures results almost exclusively from enzymes that act late in the Chl/BChl biosynthetic pathway to modify the basic chlorin structure. In striking contrast, hundreds of carotenoids have been identified (19). Their structural diversity results both from late-acting enzymes, which add diverse functional groups, and from core-pathway enzymes, which establish asymmetry in the isoprenoid backbone. This review describes the genes and the corresponding enzymes that synthesize the hub metabolite, chlorophyllide $a$ (Chlide $a$), as well as those enzymes that modify this intermediate to produce the structurally and functionally diverse Chls and BChls used for solar energy capture and conversion by chlorophototrophs.

INTRODUCTION

Two mechanisms exist for the collection and conversion of solar energy into chemical energy (14). The first uses proteorhodopsins or bacteriorhodopsins, which are retinal-binding proteins that act as light-driven proton or chloride pumps. Halobacteria from the domain Archaea were the first identified retinalphototrophs, but more recently it has been discovered that many Proteobacteria and possibly other members of the domain Bacteria employ retinal-binding proteins for light-energy conversion and light-dependent signal transduction (14, 70, 73). The second mechanism for light capture and conversion is dependent upon photochemical reaction centers (RCs), which contain chlorophylls, Mg-containing tetrapyrrole prosthetic groups, as essential cofactors. Chlorophototrophs occur in only five eubacterial phyla: Cyanobacteria, Proteobacteria, Chlorobi, Chloroflexi, and Firmicutes (14). All described Chlorobi

Retinalphototroph: a phototroph that uses either bacteriorhodopsin or proteorhodopsin for the light-driven production of proton-motive force

Chlorophototroph: an organism that has photochemical reaction centers and performs chlorophyll-based phototrophy

GSB: green sulfur bacteria

FAP: filamentous anoxygenic phototroph

PB: purple bacteria

Gomez Maqueo Chew • Bryant

ANRV322-MI61-07 ARI 6 August 2007 16:26

![Figure 1](#fig1)

Pathway for the biosynthesis of Chlide *a* from Proto-IX. See text for additional details.

confer specific advantages for the particular ecological niches of different chlorophototrophic organisms. The spectral properties of the antenna structures are determined by the arrangements and environments of their pigments, whether they are self-assembled or organized within a protein scaffold, as well as by the chemical and spectroscopic properties of these pigments (34). All chlorophototrophs share a set of specific enzymes for the initial steps in the biosynthesis of their BChls and Chls (10, 83); however, a diverse set of biosynthetic enzymes also exists that collectively creates chemical and functional diversity through the production of highly specialized (B)Chls.

### The Core Pathway: Protoporphyrin-IX to Chlorophyllide *a*

The metabolic precursor of all (B)Chls is protoporphyrin IX (Proto-IX) (Figure 1a). Proto-IX, which is also the precursor of heme and heme derivatives such as the bilin chromophores of the phycobiliproteins (87), is synthesized from eight molecules of 5-aminolevulinic acid (ALA). ALA can be synthesized through two pathways: the C₅ pathway from 2-oxoglutarate/glutamate and the C₄ + C₁ pathway from succinate and glycine (7). Six enzymes (ALA dehydratase, porphobilinogen deaminase, uroporphyrinogen III synthase, uroporphyrinogen III decarboxylase, coproporphyrinogen III oxidase, and protoporphyrinogen IX oxidase) synthesize Proto-IX from ALA. Beale (6, 7) recently reviewed the synthesis of ALA and Proto-IX.

Mg-chelatase, a multisubunit enzyme encoded by three genes, catalyzes the insertion of Mg²⁺ into Proto-IX, which is the first committed step in (B)Chl biosynthesis (Figure 1a) (10, 11, 27, 82–84). The genes *chlI/bchI* and *chlD/bchD* encode two small subunits and *chlH/bchH* or *bchH*-like genes (*bchS* and *bchT*) encode the large, Proto-IX-binding subunit (42, 83, 85). The development of an overproduction system for the Mg-chelatase has allowed the function of the chelatase to be studied in vitro (26, 27, 84). The reaction takes place in two ATP-dependent steps: activation and chelation (82). Subunits I and D are members of the AAA+ superfamily, and both have several Mg-ATPase-binding motifs (41). In the presence of Mg²⁺ and ATP, subunits I and D aggregate into heptamers and hexamers, respectively, which form a

www.annualreviews.org • Chlorophyll Biosynthesis in Bacteria 115

**Figure 1**

Pathway for the biosynthesis of Chlide *a* from Proto-IX. See text for additional details.

stable complex that can interact with the H subunit (26, 68). Recent kinetic studies have shown that the synthesis of one Mg-Proto-IX requires the hydrolysis of ~15 ATP molecules by Mg-chelatase (67). The H subunit binds the Proto-IX and Mg²⁺ substrates and likely contains the active site for chelation (38, 66).

PChlide:  
protochlorophyllide  

POR:  
protochlorophyllide oxidoreductase  

Cyanobacteria and PB have single copies of each of the three genes that encode the Mg-chelatase (85). However, GSB and FAPs usually have single copies of the *bchI* and *bchD* genes but two or three homologs of *bchH*, denoted *bchH*, *bchS*, and *bchT* (18, 21). Analyses of mutants of each of these genes in the GSB *Chlorobium tepidum* have shown that, although each can function as the only Mg-chelatase large subunit in the organism, (B)Chl biosynthesis is reduced to different extents in the three mutants. A mutant lacking a functional copy of *bchS* produces only 10% of wild-type levels of BChl *c* and excretes large amounts of Proto-IX into the growth medium (22). These additional copies of the Mg-chelatase large subunit in GSB and FAPs probably function in substrate channeling or end-product regulation of the pathway. In *Rhodobacter capsulatus* and *Synechocystis sp.* PCC 6803, BchM, the enzyme responsible for the subsequent methylation step in the (B)Chl biosynthetic pathway, requires the substrate, Mg-Proto IX (Figure 1b), to be bound to BchH/ChlH for activity (38, 71). After the methylation of Mg-Proto IX to form Mg-Proto-IX-monomethylester (Figure 1c), the C-13 propionic acid chain is oxidized and cyclized to form the isocyclic, fifth ring of the (B)Chl molecule (5, 83). This reaction is discussed in more detail below.

The product of the cyclization reaction is 3,8-divinyl-protochlorophyllide (DV-PChlide) (Figure 1d) (76). Marine *Prochlorococcus* spp. are unique because they lack the reductase for the C-8 vinyl group (54) and thus produce divinyl-Chl *a* and divinyl-Chl *b* (17, 61). The 8-vinyl reductase step is the earliest step in Chl biosynthesis that leads to chemical diversity in this class of molecules. *Prochlorococcus* spp. have somewhat enhanced

blue-light absorption because of the presence of these two vinyl groups on their Chls. However, in other bacteria the next step in the (B)Chl biosynthesis pathway is the reduction of the C-8 vinyl group of DV-PChlide (11, 54). Until recently, it was thought that the enzyme responsible for this activity in GSB and PB was BchJ (75, 87). However, recent evidence has shown that another enzyme, BciA, is responsible for this function in *C. tepidum* and probably in other organisms as well (30). BciA can reduce the 8-vinyl group of DV-PChlide in an NADPH-dependent reaction, and a *bciA* mutant of *C. tepidum* only synthesizes C-8 vinyl (B)Chls. These results indicate that a C-8 ethyl substituent is not a requirement for activity of the light-independent (dark-operative) PChlide-oxidoreductase (DPOR) or for the activities of the other downstream enzymes of (B)Chl biosynthesis in GSB (30). This finding is also true for the light-dependent protochlorophyllide oxidoreductase (POR) of cyanobacteria. POR from *Thermosynechococcus elongatus* BP-1 has equal activities in vitro with monovinyl and divinyl substrates (37). The *bciA* gene is not universally distributed among chlorophototrophs, and the genes/enzymes responsible for this reaction in cyanobacteria, most PB, FAPs, and even some GSB have not yet been identified.

After the reduction of the C-8 vinyl group, POR catalyzes the reduction of the C-17/C-18 double bond of the D-ring of PChlide *a* (Figure 1e) to form Chlide *a* (Figure 1f) (76, 83). As noted above, two types of structurally unrelated PORs, DPOR and POR, are found in chlorophototrophs (74). DPOR is found in all anoxygenic photosynthetic bacteria, and cyanobacteria have both POR and DPOR (86). POR is a member of the RED (reductases, epimerases, dehydrogenases) protein superfamily, which includes enzymes found in bacteria, plants, and humans (36). The catalytic site residues are highly conserved, and mutagenic studies with higher-plant homologs have found that they are essential for catalysis (69). POR

is unique because it is one of the only two known enzymes other than photochemical reaction centers with light-driven catalysis (36, 46). Detailed kinetic studies have shown that NADPH is required for POR activity and that POR has equal affinity for monovinyl and divinyl substrates (37).

DPOR is a nitrogenase-like enzyme with three subunits encoded by *chlN/bchN*, *chlB/bchB*, and *chlL/bchL* (4, 74). ChlL/BchL is most similar to its nitrogenase homolog NifH (33% identity and 50% similarity). The cysteine residues that ligate the intrasubunit [4Fe-4S] cluster of the nitrogenase NifH dimer are conserved in ChlL/BchL. Recent EPR analyses have shown that a BchL dimer from *R. capsulatus* carries an oxygen-sensitive [4Fe-4S] cluster similar to that found in NifH (55). ChlB/BchB and ChlN/BchN are only 30% similar to their nitrogenase homologs NifK and NifD, respectively (57); however, like their nitrogenase counterparts, the N and B subunits form a heterotetramer (24). Sequence analyses of the N and B proteins have shown that these proteins do not have the requisite number of conserved cysteines to assemble a P cluster or the FeMo cofactor of the nitrogenase NifK and NifD heterotetramer. Instead, the conserved cysteine arrangement in ChlN/BchN and ChlB/BchB seems to be more consistent with a cluster similar to the one found in the nitrogenase accessory proteins NifE and NifN. The activity of DPOR is dependent upon both ATP hydrolysis and a reductant, which is most likely ferredoxin in vivo (57).

Depending on the organism, the resulting Chlide *a* (Figure 1f and Figure 2f) can be esterified by ChlG, the Chl *a*-specific synthase, to form Chl *a* (Figure 2g) (6, 76), or it can be further modified to produce several additional products. These modifications include oxygenation either at C-3 to produce Chlide *d* (Figure 2k) (52) or at C-7 to produce Chlide *b* (Figure 2b) (79), further reduction of the chlorin macrocycle to produce bacteriochlorophyllide *a* (BChlide *a*) (Figure 2l) (16), and removal of the C-13² methylcar-

boxyl moiety to produce 3-vinyl-BChlide *d* (Figure 2t) (22). Chl *a* has absorption maxima in methanol at 432 and 665 nm and is a minor pigment in all GSB; all cyanobacteria produce at least some Chl *a* or divinyl-Chl *a* (17, 54).

### Further Modifications to the Chlorin Ring: BChl *a*

Because of various modifications to Chlide *a*, the range of light that can be absorbed by (bacterio)chlorins for photosynthesis can be broadened to span from ~350 to ~1050 nm in the UV and near infrared. BChl *a*, the pigment found in PB and the second most abundant Chl in Chlorobi and green-colored FAPs, has absorbance maxima of 367 nm and 770 nm in methanol (43), and BChl *b* has maxima at 373 and 795 nm (15). These spectral characteristics allow PB to utilize light at wavelengths other than those absorbed by the Chl *a*- and Chl *b*-containing cyanobacteria and algae, which are generally found higher in the water column (61). In the first step of synthesis of BChl *a* from Chlide *a*, the double bond between the C-7 and C-8 carbons of Chlide *a* is reduced by Chlide-oxidoreductase (COR) (11, 16). COR, like DPOR, has three different subunits, BchX, BchY, and BchZ, that are homologous to nitrogenase subunits NifH, NifD, and NifK, respectively (56). The subunits of COR apparently arose by gene duplication and divergence from those of BchN, BchB, and BchlL (42, 47).

Following the reduction of the B-ring of the tetrapyrrole, BchF hydrates the C-3 vinyl group of BChlide *a* to introduce a C-3¹-hydroxyl group (64) (Figure 2l, m). This hydration reaction is followed by dehydrogenation of the C-3 side chain by BchC to form the C-3¹-keto group (11, 76) (Figure 2m, n). In PB, the *bchC* gene is found in an operon with the three BChl *a*-specific genes that encode COR (8). The transcription factor CrtJ represses this operon in *R. capsulatus* under aerobic conditions (63). The *bchC* gene is also found immediately downstream of *bchF* and upstream of *bchX* in Chlorobi. The final

**Photosynthesis:** the use of light to provide energy and reductants for carbon dioxide reduction

Figure 2

Known and proposed pathways for the biosynthesis of Chl $a$, $b$, $d$, and $8^{1}$-hydroxy-Chl $a$ and of Bchl $a$, $b$, $c$, $d$, $e$, and $g$ from a proposed Chlide $a$ intermediate. Steps that remain to be elucidated are indicated with question marks. The (B)Chl synthase (ChlG, BchG) and geranylgeranyl reductase reactions (BchP, ChlP) have been combined into one reaction step for convenience. See text for additional details.

![f](image1.png)

NADPH +  
Geranylgeranyl-PPi  

ChlG  

ChlP  

PPi  

HOOC  
H₃COOC  

Chlorophyllide a  

O₂ + NADPH  

CAO  

Fdx_red  

BchX  
BchY  
BchZ  

Fdx_ox  

Spontaneous  
dehydration  

HOOC  
H₃COOC  

3-vinyl-bacteriochlorophyllide a  

HOOC  
H₃COOC  

Hydroxyethyl-bacteriochlorophyllide a  

BchF  

?  

HOOC  
H₃COOC  

Bacteriochlorophyllide a  

BchC  

?  

HOOC  
H₃COOC  

Bacteriochlorophyllide b  

NADPH +  
Geranylgeranyl-PPi  

ChlG  

ChlP  

PPi  

HOOC  
H₃COOC  

Chlorophyll ap  

HOOC  
H₃COOC  

Chlorophyll b  

NADPH +  
Geranylgeranyl-PPi  

ChlG  

ChlP  

PPi  

HOOC  
H₃COOC  

Chlorophyll bₚ  

Farnesyl-PPi  

BchG  

PPi  

HOOC  
H₃COOC  

Bacteriochlorophyll g  

HOOC  
H₃COOC  

Bacteriochlorophyll gₚ  

H₂O  

light  

HOOC  
H₃COOC  

8¹-hydroxy-chlorophyll aₚ  

NADPH +  
Geranylgeranyl-PPi  

BchG  

BchP  

PPi  

HOOC  
H₃COOC  

Bacteriochlorophyll aₚ  

NADPH +  
Geranylgeranyl-PPi  

BchG  

BchP  

PPi  

HOOC  
H₃COOC  

Bacteriochlorophyll bₚ  

Figure 2  
(Continued)

step in BChl $a$ biosynthesis is the addition of  
a phytyl tail by the BChl $a$-specific synthase  
BchG (1, 25, 48) (Figure 2$n$, $o$). It has been  
proposed that both ChlG and BchG attach  
a geranylgeranyl-pyrophosphate tail, which  
is later sequentially saturated three times by  
ChlP/BchP to form the phytyl tail (2, 3), to  
Chlide and BChlide, respectively. However,  
some evidence suggests that ChlP/BchP  
can also saturate the geranylgeranyl-  
pyrophosphate prior to the addition of the  
isoprenoid tail to the (B)Chlide (80).

**Further Modifications to the Chlorin Ring: Chlorobium chlorophylls: BChl $c$, $d$, and $e$**

Chlorobi synthesize Chl $a$ and BChl $a$ and one  
of three so-called *Chlorobium* chlorophylls,  
which are denoted BChl $c$, $d$, and $e$. All are  
actually chlorins rather than bacteriochlorins  
(see Figure 2$v,x,z$), and they usually have  
unsaturated farnesyl rather than phytyl tails  
(9, 12). The esterifying alcohols for BChl $c$  
in Chloroflexi strains are usually a mixture  
of straight-chain alcohols (44). Chlorobi and  
some Chloroflexi methylate these antenna  
BChls at the C-20, C-$8^{2}$, and C-$12^{1}$ positions  
(28). All these antenna pigments have a C-$3^{1}$-  
hydroxyl group that allows them to chelate the  
Mg atom of a neighboring BChl molecule to  
form the large, self-assembling aggregates in  
chlorosomes (20). Finally, all these BChls lack  
C-$13^{2}$-methylcarboxyl side chains, which are  
found in all other (B)Chls but which would  
interfere with their ability to form the large  
pigment aggregates found in chlorosomes  
(20, 21).

Until very recently, the branching point  
for the biosynthetic pathway leading to BChl  
$c$ from that for Chl $a$ and BChl $a$ in *C.*  
*tepidum* was unknown. Because of the ex-  
istence of multiple large subunits of Mg-  
chelatase, the branch point leading to BChl  
$c$ had initially been suggested to occur at the  
level of Proto-IX (13, 21). An alternative pos-  
sibility, now excluded by mutational analyses,  
is that a BChl $c$-specific oxidative cyclase was  

responsible for the formation of the isocyclic  
E-ring and the oxidative decarboxylation at  
C-13 (22). However, the accumulation of very  
large amounts of DV-PChlide in a $bchJ$ mu-  
tant of *C. tepidum*, which produces only 10%  
of the wild-type level of BChl $c$ (30), suggests  
that the pathway leading to BChl $c$, $d$, and $e$  
diverges from that for Chl $a$ and BChl $a$ only  
after Chlide $a$ is formed (see Figure 2$f-t$). If  
this deduction is correct, then the only pro-  
cess in the synthesis of BChl $c$, $d$, and $e$ that is  
not yet genetically and/or biochemically de-  
fined in GSB and FAPs is the removal of the  
C-$13^{2}$-methylcarboxyl moiety. This reaction  
might be catalyzed by a pheophorbide sim-  
ilar to those involved in the degradation of  
Chls in higher plants (77, 78).

Two radical-SAM (S-adenosyl-L-  
methionine) oxidative methyltransferases,  
BchQ and BchR, add the methyl groups to  
the C-$8^{2}$ ethyl and C-$12^{1}$ methyl carbons,  
respectively (21, 22; A. Gomez Maqueo  
Chew, N.-U. Frigaard & D.A. Bryant,  
manuscript submitted). The addition of up  
to three methyl groups at C-$8^{2}$ and a single  
methyl group at C-$12^{1}$ does not change the  
absorbance properties of monomeric BChl $c$,  
but these modifications can further tune the  
absorbance maximum of the aggregated BChl  
$c$, $d$ or $e$ in chlorosomes by up to 15 nm and  
increase the half-bandwidth of the absorption  
spectrum (9, 12). Furthermore, mutants of  
*C. tepidum*, which are unable to perform the  
methylation reactions at the C-$8^{2}$, C-$12^{1}$,  
and C-20 positions, are unable to modulate  
correctly their cellular content of BChl $c$ in  
a light-intensity-dependent manner (22, 31).  
The difference between BChl $d$ and BChl $c$  
or BChl $e$ is the addition of the C-20 methyl  
group by BchU (Figure 2$w,x-z$) (50). Wada  
et al. (81) have determined the structure of  
the BchU methyltransferase and have further  
shown that BchU probably acts after the  
hydration of the C-3 vinyl group (J. Harada,  
personal communication). The C-20 methyl  
group causes a red-shift of ~12 nm in the Q$_{y}$  
absorbance of BChl. This absorption shift  
confers a significant growth rate advantage
to cells at low intensities of far-red light and additionally shifts the absorption away from that of Chl $a$ (50).

The C-3${}^{1}$ carbons of BChl $c$, $d$, and $e$ are chiral centers (see Figure 2$u-z$). The stereochemistry at C-3${}^{1}$ is known to be affected by the methylation status at C-8${}^{2}$ and C-12${}^{1}$ in such a way that the most highly methylated BChl $c$, $d$, and $e$ homologs have predominantly $S$ stereochemistry, and the least methylated BChl $c$, $d$, and $e$ homologs have predominantly $R$ stereochemistry (40, 72). Database searches show that all GSB genomes encode at least two homologs of $bchF$. One of these two genes, denoted $bchV$, is paralogous to $bchF$ and is not required for viability in *C. tepidum*, but the apparent $bchF$ ortholog is required for viability, presumably because this enzyme is also required for the synthesis of BChl $a$ and Chl $a$ (22) (see Figure 2$l,m$). The methylated homologs of 3-vinyl-BChl $c$ that accumulate in the $bchV$ mutant are hypermethylated, whereas those BChl $c$ molecules that are hydrated at C-3${}^{1}$ are hypomethylated (29). These results suggest that BchF hydrates the C-3 vinyl group of 3-vinyl-BChlide $d$ to produce predominantly $R$ stereochemistry at C-3${}^{1}$ and BchV produces predominantly $S$ stereochemistry (A. Gomez Maqueo Chew & D.A. Bryant, unpublished data). This would account for the distribution of BChl $c$ C-3${}^{1}$ $R$ and $S$ epimers found in the *C. tepidum* chlorosomes (40). This observation also implies that hydration of the C-3 vinyl group probably occurs after methylation at C-8${}^{2}$ and C-12${}^{1}$ or at least is strongly influenced by these methylation events. As occurs in the synthesis of both Chl $a$ and BChl $a$, the last step in the biosynthesis of BChl $c$ is the addition of the farnesyl tail to BChlide $c$, $d$, or $e$ (Figure 2$u,w,y$) by a BChl $c$-, $d$-, or $e$-specific synthase BchK (23).

### Further Modifications to the Bacteriochlorin Ring: BChl $b$ and BChl $g$

Both BChl $b$ (Figure 2$q$), found in some PB, and BChl $g$ (Figure 2$s$), the only BChl found

in heliobacteria, have ethylidine groups at their C-8 positions. These molecules are true bacteriochlorins with reduced B- and D-rings, and consistent with this assertion, the *Heliobacterium modesticaldum* genome appears to contain homologs of the $bchX$, $bchY$, and $bchZ$ genes that encode Chlide $a$ reductase ([http://genomes.tgen.org/index.html](http://genomes.tgen.org/index.html); R.E. Blankenship, personal communication). BChl $b$ has a C-3 acetyl side chain similar to that of BChl $a$; however, the C-8 ethyldiene side chain causes a red-shift of the $Q_{y}$ absorbance to 795 nm in methanol, compared with the 770-nm $Q_{y}$ absorption maximum of BChl $a$ (39). This property causes an in vivo $Q_{y}$ absorption maximum of >1014 nm (39) and confers a selective advantage to BChl $b$-producing organisms in specific niches in which there is little absorbance by water. BChl $g$ has its $Q_{y}$ absorption maximum at 760 nm in methanol (51). It is not known whether the C-8 ethyldiene side chains of BChls $b$ and $g$ are produced enzymatically or merely by a chemical reaction. Heliobacteria also produce another unique chlorophyll: C-8${}^{1}$-hydroxy-Chl $a$ (Figure 2$aa$). BChl $g$ is readily photoisomerized into C-8${}^{1}$-hydroxy-Chl $a$ in the presence of oxygen, although this Chl also occurs naturally in heliobacterial reaction centers (51, 53). This suggests that the addition of the C-8${}^{1}$ hydroxyl can also occur under anoxic conditions, under which the hydroxyl group must originate from water.

### Aerobic versus Anaerobic Synthesis: AcsF, BchE, and Formation of the C-3 and C7 Formyl Groups

The oxidation and cyclization of the C-13 propionate methyl ester of Mg-protoporphyrin-monomethylester (Figure 1$c$) to form the E-ring of PChlide is a key step in the synthesis of all (B)Chls (5, 83). This multistep reaction is catalyzed in vivo by one of two structurally unrelated enzymes (31, 60). The first enzyme, AcsF, is an oxygen-dependent, di-iron oxygenase found in cyanobacteria and other organisms, including PB, that
synthesize Chls in oxic environments (62). The second enzyme, BchE, is an oxygen-sensitive, radical-SAM enzyme found in GSB and other organisms that live in anoxic environments. Some PB, which produce BChl *a* under anoxic and microaerophilic conditions, have both AcsF and BchE and regulate the activity of the corresponding proteins in an oxygen-dependent manner (58). A homolog of BchE encoded in the *Synechocystis* sp. PCC 6803 genome was recently shown to complement a BchE-deficient mutant of *R. capsulatus* (10). However, inactivation of the cyanobacterial gene produced no effect on Chl biosynthesis in *Synechocystis* sp. PCC 6803, which also encodes two enzymes similar to AcsF. The direct substitution of an oxygen-dependent (oxygenase) enzyme in aerobes for a radical-SAM enzyme used in anaerobes is not unusual (45). These two types of Mg-protoporphyrin-monomethylester oxidative cyclase illustrate how enzyme diversification has occurred to produce (B)Chl enzymes better adapted to particular environments (e.g., oxic versus anoxic).

Another similar case is the introduction of a formyl group at the C-7 position in Chl *b* (Figure 2*i*) and BChl *e* (Figure 2*y,z*). Chl *b* is structurally similar to Chl *a*, and prochlorophytes (*Prochlorothrix, Prochloron,* and *Prochlorococcus* spp.) are the only prokaryotes that can produce Chl *b* (or divinyl-Chl *b*) as do higher plants (17, 61). BChl *e* and BChl *c* only differ in that the former has a formyl group at the C-7 position (Figure 2*z*) (13). The addition of this formyl group causes a red-shift of 33 nm in the Soret region and a blue-shift of 15 nm of the *Q*<sub>y</sub> peaks of Chl *b* with respect to the maxima of Chl *a*. Similarly, the addition of a C-7 formyl group to BChl *c* causes a 35-nm red-shift of the Soret region and a 12-nm blue-shift in the *Q*<sub>y</sub> of its absorption spectrum. These changes greatly improve the light-harvesting capabilities of Chl *b* and BChl *e* in the blue region of the visible spectrum.

The enzyme responsible for this reaction in prochlorophytes, Chl *a* oxygenase, was initially identified in green algae and is highly similar in cyanobacteria (79). Chl *a* oxygenase is a Rieske-containing, nonheme-iron monooxygenase that uses molecular O<sub>2</sub> and NADPH to perform two successive hydroxylations at the C-7<sup>1</sup> position of Chlide *a*. The resulting aldehyde hydrate undergoes spontaneous dehydration to form Chlide *b* (59). Brown-colored Chlorobi are strict anaerobes, and thus the synthesis of BChl *e* must occur by a completely different mechanism. A putative C-7<sup>1</sup> hydratase has been identified by comparative genomics, and this gene occurs in a gene cluster that includes the carotenoid cyclase, CruB, which is also uniquely found in brown-colored Chlorobi (49). This putative C-7<sup>1</sup> hydratase appears to be yet another member of the radical-SAM superfamily, and its gene lies next to a gene encoding a third hydratase of the BchF/BchV family (49).

The cyanobacterium *Acaryochloris marina* synthesizes Chl *d*, which has a formyl group at the C-3 position that causes a red-shift of the *Q*<sub>y</sub> absorption (Figure 2*k*) (52). Nothing is currently known about the enzyme(s) involved in the likely conversion of Chlide *a* to Chlide *d*. It is easy to imagine that *A. marina* might perform multiple hydration reactions at the C-3<sup>1</sup> position of Chlide *a*, or alternatively, the formyl group could be formed through an oxidative mechanism related to that for the formation of the formyl group of Chl *b*. Consistent with this latter possibility, the unfinished genome of *A. marina* (http://genomes.tgen.org/index.html) (R.E. Blankenship, personal communication) contains sequences with significant similarity to the Chlide *a* oxygenase, a member of the Rieske-nonheme iron oxygenase family of proteins.

**CLOSING REMARKS**

Although Chls and BChls found in nature are diverse in their structure and spectral properties, the enzymes responsible for most of the core biosynthetic pathway are highly conserved among all photosynthetic organisms.

When a powerful selective pressure, oxygen, apparently inactivated oxygen-independent enzymes in (B)Chl biosynthesis, unrelated proteins with the same catalytic function but with completely different structures and mechanisms, sometimes even using oxygen as a substrate, then evolved. Because the biosynthesis of Chls $b$ (and probably Chl $d$) requires oxygen as a cofactor, these Chls likely arose only after the invention of Photosystem II and oxygen-evolving photosynthesis by ancestral cyanobacteria.

Although the order of the reactions leading to the synthesis of some (B)Chls is not yet completely resolved, it is nevertheless now possible to postulate that all Chls and BChls are synthesized from Chlide $a$. (The exceptions are divinyl-Chl $a$ and divinyl-Chl $b$, which must be synthesized from 3,8-divinyl-Chlide $a$.) If validated, this finding has implications for the evolution of Chl-based photosynthesis and supports the Granick hypothesis, which states that biosynthetic pathways evolve as organisms evolve (32, 33, 65). As specifically applied to photosynthesis, this hypothesis states that the evolution of Chl and BChl biosynthesis should mirror the evolution of photosynthesis. From the hub intermediate Chlide $a$, modifications of the chlorin or bacteriochlorin ring lead to a family of molecules, each of which is tuned to allow specific light-wavelength absorption by the organisms that possess these diverse molecules (Figure 2).

The GSB are particularly important and provide support for the Granick hypothesis. All GSB synthesize Chl $a$ esterified with $\Delta$-2,6-phytadienol, BChl $a$ esterified with phytol, and either BChl $c$, $d$, or $e$ esterified with farnesol (22). All of these molecules appear to be synthesized from the central intermediate, Chlide $a$. Because antenna functions are expected to have evolved after reaction centers, it is not surprising to find that BChl $c$ is most likely synthesized from an intermediate of the core biosynthetic pathway such as Chlide $a$. The Type I reaction centers of *C. tepidum* require both Chl $a$ and BChl $a$ as cofactors but do not contain BChl $c$. It is not too difficult to imagine that, as oxygenic phototrophs arose and flourished, the reaction centers of the ancestors of GSB exchanged Chl $a$ molecules for BChl $a$ molecules in order to compete with ancestral cyanobacteria for light. This would be similar to the subsequent invention of Chl $d$ by the ancestors of *A. marina* and related organisms. In an analogous manner, BChl $a$, BChl $b$, and BChl $g$ might have arisen by extensions of the biosynthetic pathway from Chlide $a$ and BChlide $a$. As more and more genomes of diverse chlorophototrophs are being sequenced (14), phylogenetic analyses may provide further support for this view of the evolution of Chl biosynthesis.

---

**SUMMARY POINTS**

1. A core set of biosynthetic enzymes is responsible for the synthesis of the Chlide precursor of all BChls and Chls.
2. Organisms have adapted to growth under specific light conditions by evolving enzymes that modify the chlorin and bacteriochlorin rings. These modifications change the absorption properties of the (B)Chls and thereby change the properties of the reaction center and antennae complexes that contain these molecules.
3. Oxygen has conferred a strong selective pressure on Chl biosynthesis. Aerobes have evolved some oxygen-stable and oxygen-dependent isoenzymes that can perform the same functions as the structurally distinct, oxygen-independent enzymes found in anaerobes.
4. All major types of Chls and BChls can be biosynthesized from the hub intermediate, Chlide $a$.

FUTURE ISSUES

1. Although all the genes involved in the synthesis of Chl $a$ in plants have been identified, one enzyme and its gene(s) remain to be identified in the synthesis of Chl $a$ in most cyanobacteria and many other organisms: the C-8 vinyl reductase. The genomes of most cyanobacteria, PB, and FAPs do not encode a homolog of $bciA$, which encodes the C-8 vinyl reductase in 9 of 11 sequenced GSB strains.

2. One key step in the synthesis of BChl $c$, $d$ and $e$, the removal of the C-13² carboxymethyl group of the E-ring, remains to be elucidated. The mechanism by which the C-7 formyl group of BChl $e$ is introduced is not yet established.

3. The mechanism of formation of the ethylidine groups of BChl $b$ and BChl $g$, whether chemically or enzymatically mediated, is still unknown.

4. It remains to be determined whether the formation of the C-3 formyl moiety of Chl $d$ is catalyzed by a Rieske-containing monooxygenase, a hydratase, or some other type of enzyme.

5. The specific role of BchJ in regulating BChl biosynthesis at the level of PChlide in GSB and PB is unknown.

6. Regulation of Chl and BChl biosynthesis by metabolites has not yet been clearly resolved, and the roles of substrate channeling and substrate binding proteins are not yet well understood.


DISCLOSURE STATEMENT

The authors are not aware of any biases that might be perceived as affecting the objectivity of this review.


ACKNOWLEDGMENTS

Work from the author’s laboratory was supported by grants MCB-0523100 and MCB-0519743 from the National Science Foundation and DE-FG02-94ER20137 from the Department of Energy.


LITERATURE CITED

1. Addlesee HA, Fiedor L, Hunter CN. 2000. Physical mapping of $bchG$, $orf427$, and $orf177$ in the photosynthesis gene cluster of *Rhodobacter sphaeroides*: functional assignment of the bacteriochlorophyll synthetase gene. *J. Bacteriol.* 182:3175–82

2. Addlesee HA, Gibson LCD, Jensen PE, Hunter CN. 1996. Cloning, sequencing and functional assignment of the chlorophyll biosynthesis gene, $chlP$, of *Synechocystis* sp. PCC 6803. *FEBS Lett.* 389:126–30

3. Addlesee HA, Hunter CN. 1999. Physical mapping and functional assignment of the geranylgeranyl-bacteriochlorophyll reductase gene, $bchP$, of *Rhodobacter sphaeroides*. *J. Bacteriol.* 181:7248–55

4. Armstrong GA. 1998. Greening in the dark: light-independent chlorophyll biosynthesis from anoxygenic photosynthetic bacteria to gymnosperms. *J. Photochem. Photobiol. B Biol.* 43:87–100

5. Beale SI. 1999. Enzymes of chlorophyll biosynthesis. *Photosynth. Res.* 60:43–73  
6. Beale SI. 2005. Green genes gleaned. *Trends Plant Sci.* 10:309–12  
7. Beale SI. 2006. Biosynthesis of 5-aminolevulinic acid. See Ref. 35, pp. 147–58  
8. Biel AJ, Marrs BL. 1983. Transcriptional regulation of several genes for bacteriochlorophyll biosynthesis in *Rhodopseudomonas capsulata* in response to oxygen. *J. Bacteriol.* 156:686–94  
9. Bobe FW, Pfennig N, Swanson KL, Smith KM. 1990. Red shift of absorption maxima in *Chlorobiineae* through enzymic methylation of their antenna bacteriochlorophylls. *Biochemistry* 29:4340–48  
10. Bolivar DW. 2006. Recent advances in chlorophyll biosynthesis. *Photosynth. Res.* 89:1–22  
11. Bolivar DW, Suzuki JY, Beatty JT, Dobrowolski JM, Bauer CE. 1994. Directed mutational analysis of bacteriochlorophyll *a* biosynthesis in *Rhodobacter capsulatus*. *J. Mol. Biol.* 237:622–40  
12. Borrego CM, Gerola PD, Miller M, Cox RP. 1999. Light intensity effects on pigment composition and organisation in the green sulfur bacterium *Chlorobium tepidum*. *Photosynth. Res.* 59:159–66  
13. Brockmann H. 1976. Bacteriochlorophyll *e*: structure and stereochemistry of a new type of chlorophyll from *Chlorobiaceae*. *Philos. Trans. R. Soc. London Ser. B* 273:277–85  
14. Bryant DA, Frigaard NU. 2006. Prokaryotic photosynthesis and phototrophy illuminated. *Trends Microbiol.* 14:488–96  
15. Bryantseva IA, Gorlenko VM, Komantseva EI, Imhoff JF. 2000. *Thioalkalicoccus limnaeus* gen. nov., sp. nov., a new alkaliphilic purple sulfur bacterium with bacteriochlorophyll *b*. *Int. J. Syst. Evol. Microbiol.* 50:2157–63  
16. Burke DH, Alberti M, Hearst JE. 1993. The *Rhodobacter capsulatus* chlorin reductase-encoding locus, *bchA*, consists of three genes, *bchX*, *bchY*, and *bchZ*. *J. Bacteriol.* 175:2407–13  
17. Chisholm SW, Frankel SL, Goericke R, Olson RJ, Palenik B, et al. 1992. *Prochlorococcus marinus* nov. gen., nov. sp.: an oxyphototrophic marine prokaryote containing divinyl chlorophyll *a* and chlorophyll *b*. *Arch. Microbiol.* 157:297–300  
18. Eisen JA, Nelson KE, Paulsen IT, Heidelberg JF, Wu M, et al. 2002. The complete genome sequence of *Chlorobium tepidum* TLS, a photosynthetic, anaerobic, green-sulfur bacterium. *Proc. Natl. Acad. Sci. USA* 99:9509–14  
19. Frank H, Young A, Britton G, Cogdell RJ, eds. 1999. *Advances in Photosynthesis and Respiration*. Vol. 8: *The Photochemistry of Carotenoids*. Dordrecht, The Neth.: Kluwer Acad. 399 pp.  
20. Frigaard NU, Bryant DA. 2006. Chlorosomes: antenna organelles in photosynthetic green bacteria. In *Complex Intracellular Structures in Prokaryotes*, ed. JM Shively, pp. 80–114. Berlin/Heidelberg: Springer-Verlag  
21. Frigaard NU, Gomez Maqueo Chew A, Li H, Maresca JA, Bryant DA. 2003. *Chlorobium tepidum*: insights into the structure, physiology, and metabolism of a green sulfur bacterium derived from the complete genome sequence. *Photosynth. Res.* 78:93–117  
22. Frigaard NU, Gomez Maqueo Chew A, Maresca JA, Bryant DA. 2006. Bacteriochlorophyll biosynthesis in green bacteria. See Ref. 35, pp. 201–21  
23. Frigaard NU, Voigt GD, Bryant DA. 2002. *Chlorobium tepidum* mutant lacking bacteriochlorophyll *c* made by inactivation of the *bchK* gene, encoding bacteriochlorophyll *c* synthase. *J. Bacteriol.* 184:3368–76  
24. Fujita Y, Bauer CE. 2000. Reconstitution of light-independent protochlorophyllide reductase from purified BchL and BchN-BchB subunits. In vitro confirmation of nitrogenase-like features of a bacteriochlorophyll biosynthesis enzyme. *J. Biol. Chem.* 275:23583–88  

25. Garcia-Gil LJ, Gich FB, Fuentes-Garcia X. 2003. A comparative study of *bchG* from green photosynthetic bacteria. *Arch. Microbiol.* 179:108–15

26. Gibson LCD, Jensen PE, Hunter CN. 1999. Magnesium chelatase from *Rhodobacter sphaeroides*: initial characterization of the enzyme using purified subunits and evidence for a BchI-BchD complex. *Biochem. J.* 337:243–51

27. Gibson LCD, Willows RD, Kannangara CG, Vonwettstein D, Hunter CN. 1995. Magnesium-protoporphyrin chelatase of *Rhodobacter sphaeroides*: reconstitution of activity by combining the products of the *bchH*, *bchI*, and *bchD* genes expressed in *Escherichia coli*. *Proc. Natl. Acad. Sci. USA* 92:1941–44

28. Gich F, Airs RL, Danielsen M, Keely BJ, Abella CA, et al. 2003. Characterization of the chlorosome antenna of the filamentous anoxygenic phototrophic bacterium *Chloronema* sp. strain UdG9001. *Arch. Microbiol.* 180:417–26

29. Gomez Maqueo Chew A, Bryant DA. 2004. Identification of BchV, a C-3(1) hydratase specific for hypermethylated bacteriochlorophyll *c* in *Chlorobium tepidum*. In *Photosynthesis: Fundamental Aspects to Global Perspectives, Proc. Int. Congr. Photosynth.*, 12th, Montreal, ed. A van der Est, D Bruce, pp. 875–77. Lawrence, KS: Allen

30. Gomez Maqueo Chew A, Bryant DA. 2006. Characterization of a plant-like protochlorophyllide *a* divinyl reductase in green sulfur bacteria. *J. Biol. Chem.* 282:2967–75

31. Gough SP, Petersen BO, Duus JO. 2000. Anaerobic chlorophyll isocyclic ring formation in *Rhodobacter capsulatus* requires a cobalamin cofactor. *Proc. Natl. Acad. Sci. USA* 97:6908–13

32. Granick S. 1957. Speculations on the origins and evolution of photosynthesis. *Ann. NY Acad. Sci.* 69:292–308

33. Granick S. 1965. Evolution of heme and chlorophyll. In *Evolving Genes and Proteins*, ed. V Bryson, HG Vogel, pp. 67–68. New York: Academic

34. Green BR, Parson WW, eds. 2003. *Advances in Photosynthesis and Respiration*. Vol. 13: *Light-Harvesting Antennas in Photosynthesis*. Dordrecht, The Neth.: Kluwer Acad. 513 pp.

35. Grimm B, Porra RJ, Rüdiger W, Scheer H, eds. 2006. *Advances in Photosynthesis and Respiration*. Vol. 25: *Chlorophylls and Bacteriochlorophylls: Biochemistry, Biophysics, Functions and Applications*. Dordrecht, The Neth.: Springer. 603 pp.

36. Heyes DJ, Heathcote P, Rigby SE, Palacios MA, van Grondelle R, et al. 2006. The first catalytic step of the light-driven enzyme protochlorophyllide oxidoreductase proceeds via a charge transfer complex. *J. Biol. Chem.* 281:26847–53

37. Heyes DJ, Kruk J, Hunter CN. 2006. Spectroscopic and kinetic characterization of the light-dependent enzyme protochlorophyllide oxidoreductase (POR) using monovinyl and divinyl substrates. *Biochem. J.* 394:243–48

38. Hinchigeri SB, Hundle B, Richards WR. 1997. Demonstration that the BchH protein of *Rhodobacter capsulatus* activates S-adenosyl-L-methionine:magnesium protoporphyrin IX methyltransferase. *FEBS Lett.* 407:337–42

39. Hoogewerf GJ, Jung DO, Madigan MT. 2003. Evidence for limited species diversity of bacteriochlorophyll *b*-containing purple nonsulfur anoxygenic phototrophs in freshwater habitats. *FEMS Microbiol. Lett.* 218:359–64

40. Huster MS, Smith KM. 1990. Biosynthetic studies of substituent homologation in bacteriochlorophylls *c* and *d*. *Biochemistry* 29:4348–55

41. Jensen PE, Gibson LC, Hunter CN. 1999. ATPase activity associated with the magnesium-protoporphyrin IX chelatase enzyme of *Synechocystis* PCC 6803: evidence for ATP hydrolysis during Mg²⁺ insertion, and the MgATP-dependent interaction of the ChlI and ChlD subunits. *Biochem. J.* 339:127–34

42. Jensen PE, Reid JD, Hunter CN. 2000. Modification of cysteine residues in the ChlI and ChlH subunits of magnesium chelatase results in enzyme inactivation. *Biochem. J.* 352:435–41

43. Kolber ZF, Plumley FG, Lang AS, Beatty JT, Blankenship RE, et al. 2001. Contribution of aerobic photoheterotrophic bacteria to the carbon cycle in the ocean. *Science* 292:2492–95

44. Larsen KL, Cox RP, Miller M. 1994. Effects of illumination intensity on bacteriochlorophyll *c* homolog distribution in *Chloroflexus aurantiacus* grown under controlled conditions. *Photosynth. Res.* 41:151–56

45. Layer G, Heinz DW, Jahn D, Schubert WD. 2004. Structure and function of radical SAM enzymes. *Curr. Opin. Chem. Biol.* 8:468–76

46. Lebedev N, Tinko MP. 1998. Protochlorophyllide photoreduction. *Photosynth. Res.* 58:2–23

47. Lockhart PJ, Larkum AWD, Steel MA, Waddell PJ, Penny D. 1996. Evolution of chlorophyll and bacteriochlorophyll: the problem of invariant sites in sequence analysis. *Proc. Natl. Acad. Sci. USA* 93:1930–34

48. Lopez JC, Ryan S, Blankenship RE. 1996. Sequence of the *bchG* gene from *Chloroflexus aurantiacus*: relationship between chlorophyll synthase and other polyphenyltransferases. *J. Bacteriol.* 178:3369–73

49. Maresca JA. 2007. *The genetic basis for pigment variation among green sulfur bacteria*. PhD thesis. Penn. State Univ. 165 pp.

50. Maresca JA, Gomez Maqueo Chew A, Ponsati MR, Frigaard NU, Ormerod JG, et al. 2004. The *bchU* gene of *Chlorobium tepidum* encodes the C-20 methyltransferase in bacteriochlorophyll *c* biosynthesis. *J. Bacteriol.* 186:2558–66

51. Michalski TJ, Hunt JE, Bowman MK, Smith U, Bardeen K, et al. 1987. Bacteriopheophytin *g*: properties and some speculations on a possible primary role for bacteriochlorophylls *b* and *g* in the biosynthesis of chlorophylls. *Proc. Natl. Acad. Sci. USA* 84:2570–74

52. Miyashita H, Ikemoto H, Kurano N, Adachi K, Chihara M, et al. 1996. Chlorophyll *d* as a major pigment. *Nature* 383:402

53. Mizoguchi T, Oh-oka H, Tamiaki H. 2005. Determination of stereochemistry of bacteriochlorophyll *gF* and 81-hydroxy-chlorophyll *aF* from *Heliobacterium modesticaldum*. *Photochem. Photobiol.* 81:666–73

54. Nagata N, Tanaka R, Satoh S, Tanaka A. 2005. Identification of a vinyl reductase gene for chlorophyll synthesis in *Arabidopsis thaliana* and implications for the evolution of *Prochlorococcus* species. *Plant Cell* 17:233–40

55. Nomata J, Kitashima M, Inoue K, Fujita Y. 2006. Nitrogenase Fe protein-like Fe-S cluster is conserved in L-protein (BchL) of dark-operative protochlorophyllide reductase from *Rhodobacter capsulatus*. *FEBS Lett.* 580:6151–54

56. Nomata J, Mizoguchi T, Tamiaki H, Fujita Y. 2006. A second nitrogenase-like enzyme for bacteriochlorophyll biosynthesis: reconstitution of chlorophyllide *a* reductase with purified X-protein (BchX) and YZ-protein (BchY-BchZ) from *Rhodobacter capsulatus*. *J. Biol. Chem.* 281:15021–28

57. Nomata J, Swem LR, Bauer CE, Fujita Y. 2005. Overexpression and characterization of dark-operative protochlorophyllide reductase from *Rhodobacter capsulatus*. *Biochim. Biophys. Acta* 1708:229–37

58. Oh JI, Eraso JM, Kaplan S. 2000. Interacting regulatory circuits involved in orderly control of photosynthesis gene expression in *Rhodobacter sphaeroides* 2.4.1. *J. Bacteriol.* 182:3081–87

59. Oster U, Tanaka R, Tanaka A, Rudiger W. 2000. Cloning and functional expression of the gene encoding the key enzyme for chlorophyll *b* biosynthesis (CAO) from *Arabidopsis thaliana*. *Plant J.* 21:305–10

60. Ouchane S, Steunou AS, Picaud M, Astier C. 2004. Aerobic and anaerobic Mg-protoporphyrin monomethyl ester cyclases in purple bacteria: a strategy adopted to bypass the repressive oxygen control system. *J. Biol. Chem.* 279:6385–94

61. Partensky F, Hess WR, Vaulot D. 1999. *Prochlorococcus*, a marine photosynthetic prokaryote of global significance. *Microbiol. Mol. Biol. Rev.* 63:106–27

62. Pinta V, Picaud M, Reiss-Husson F, Astier C. 2002. *Rubrivivax gelatinosus* *acsF* (previously *orf358*) codes for a conserved, putative binuclear-iron-cluster-containing protein involved in aerobic oxidative cyclization of Mg-protoporphyrin IX monomethylester. *J. Bacteriol.* 184:746–53

63. Ponnampalam SN, Elsen S, Bauer CE. 1998. Aerobic repression of the *Rhodobacter capsulatus bchC* promoter involves cooperative interactions between CrtJ bound to neighboring palindromes. *J. Biol. Chem.* 273:30757–61

64. Porra RJ, Schafer W, Gadon N, Katheder I, Drews G, et al. 1996. Origin of the two carbonyl oxygens of bacteriochlorophyll *a*: demonstration of two different pathways for the formation of ring E in *Rhodobacter sphaeroides* and *Roseobacter denitrificans*, and a common hydratase mechanism for 3-acetyl group formation. *Eur. J. Biochem.* 239:85–92

65. Raymond J, Zhaxybayeva O, Gogarten JP, Gerdes SY, Blankenship RE. 2002. Whole genome analysis of photosynthetic prokaryotes. *Science* 298:1616–20

66. Reid JD, Hunter CN. 2002. Current understanding of the function of magnesium chelatase. *Biochem. Soc. Trans.* 30:643–45

67. Reid JD, Hunter CN. 2004. Magnesium-dependent ATPase activity and cooperativity of magnesium chelatase from *Synechocystis* sp. PCC6803. *J. Biol. Chem.* 279:26893–99

68. Reid JD, Siebert CA, Bullough PA, Hunter CN. 2003. The ATPase activity of the ChlI subunit of magnesium chelatase and formation of a heptameric AAA(+) ring. *Biochemistry* 42:6912–20

69. Reinbothe C, Buhr F, Bartsch S, Desvignes C, Quigley F, et al. 2006. In vitro-mutagenesis of NADPH:protochlorophyllide oxidoreductase B: Two distinctive protochlorophyllide binding sites participate in enzyme catalysis and assembly. *Mol. Genet. Genomics* 275:540–52

70. Sharma AK, Spudich JL, Doolittle WF. 2006. Microbial rhodopsins: functional versatility and genetic mobility. *Trends Microbiol.* 14:463–69

71. Shepherd M, McLean S, Hunter CN. 2005. Kinetic basis for linking the first two enzymes of chlorophyll biosynthesis. *FEBS J.* 272:4532–39

72. Smith KM, Simpson DJ. 1986. Stereochemistry of the bacteriochlorophyll-*e* homologs. *J. Chem. Soc. Chem. Commun.* 22:1682–84

73. Spudich JL. 2006. The multitalented microbial sensory rhodopsins. *Trends Microbiol.* 14:480–87

74. Suzuki JY, Bauer CE. 1995. A prokaryotic origin for light-dependent chlorophyll biosynthesis of plants. *Proc. Natl. Acad. Sci. USA* 92:3749–53

75. Suzuki JY, Bauer CE. 1995. Altered monovinyl and divinyl protochlorophyllide pools in *bchJ* mutants of *Rhodobacter capsulatus*. Possible monovinyl substrate discrimination of light-independent protochlorophyllide reductase. *J. Biol. Chem.* 270:3732–40

76. Suzuki JY, Bolivar DW, Bauer CE. 1997. Genetic analysis of chlorophyll biosynthesis. *Annu. Rev. Genet.* 31:61–89

77. Suzuki Y, Amano T, Shioi Y. 2006. Characterization and cloning of the chlorophyll-degrading enzyme pheophorbidase from cotyledons of radish. *Plant Physiol.* 140:716–25

78. Suzuki Y, Doi M, Shioi Y. 2002. Degradation of chlorophylls: two reaction pathways in the formation of pyropheophorbide *a*. *Photosynth. Res.* 74:225–33

79. Tanaka A, Ito H, Tanaka R, Tanaka NK, Yoshida K, et al. 1998. Chlorophyll *a* oxygenase (CAO) is involved in chlorophyll *b* formation from chlorophyll *a*. *Proc. Natl. Acad. Sci. USA* 95:12719–23

80. Tanaka R, Oster U, Kruse E, Rudiger W, Grimm B. 1999. Reduced activity of geranylgeranyl reductase leads to loss of chlorophyll and tocopherol and to partially geranylgeranylated chlorophyll in transgenic tobacco plants expressing antisense RNA for geranylgeranyl reductase. *Plant Physiol.* 120:695–704

81. Wada K, Yamaguchi H, Harada J, Niimi K, Osumi S, et al. 2006. Crystal structures of BchU, a methyltransferase involved in bacteriochlorophyll *c* biosynthesis, and its complex with S-adenosylhomocysteine: implications for reaction mechanism. *J. Mol. Biol.* 360:839–49

82. Walker CJ, Willows RD. 1997. Mechanism and regulation of Mg-chelatase. *Biochem. J.* 327:321–33

83. Willows RD. 2003. Biosynthesis of chlorophylls from protoporphyrin IX. *Nat. Prod. Rep.* 20:327–41

84. Willows RD, Beale SI. 1998. Heterologous expression of the *Rhodobacter capsulatus bchl*, -D, and -H genes that encode magnesium chelatase subunits and characterization of the reconstituted enzyme. *J. Biol. Chem.* 273:34206–13

85. Willows RD, Gibson LCD, Kanangara CG, Hunter CF, von Wettstein D. 1996. Three separate proteins constitute the magnesium chelatase of *Rhodobacter sphaeroides*. *Eur. J. Biochem.* 235:438–43

86. Yamazaki S, Nomata J, Fujita Y. 2006. Differential operation of dual protochlorophyllide reductases for chlorophyll biosynthesis in response to environmental oxygen levels in the cyanobacterium *Leptolyngbya boryana*. *Plant Physiol.* 142:911–22

87. Yaronskaya E, Grimm B. 2006. The pathway from 5-aminolevulinic acid to protochlorophyllide and protoheme. See Ref. 35, pp. 173–88

---

**RELATED RESOURCES**

Olson JM, Blankenship RE. 2004. Thinking about the evolution of photosynthesis. *Photosynth. Res.* 80:373–86

Porra RJ. 1997. Recent progress in porphyrin and chlorophyll biosynthesis. *Photochem. Photobiol.* 65:492–516

Tanaka R, Tanaka A. 2007. Tetrapyrrole biosynthesis in higher plants. *Annu. Rev. Plant Biol.* 58:321–46

Xiong J, Bauer CE. 2002. Complex evolution of photosynthesis. *Annu. Rev. Plant Biol.* 53:503–21

Annual Review of  
Microbiology  

Volume 61, 2007  

Contents  

Frontispiece  
Margarita Salas …………xiv  

40 Years with Bacteriophage φ29  
Margarita Salas …………1  

The Last Word: Books as a Statistical Metaphor for Microbial Communities  
Patrick D. Schloss and Jo Handelsman …………23  

The Mechanism of Isoniazid Killing: Clarity Through the Scope  
of Genetics  
Catherine Vilchèze and William R. Jacobs, Jr. …………35  

Development of a Combined Biological and Chemical Process for  
Production of Industrial Aromatics from Renewable Resources  
F. Sima Sariaslani …………51  

The RNA Degradosome of Escherichia coli: An mRNA-Degrading  
Machine Assembled on RNase E  
Agamemnon J. Carpousis …………71  

Protein Secretion in Gram-Negative Bacteria via the Autotransporter  
Pathway  
Nathalie Dautin and Harris D. Bernstein …………89  

Chlorophyll Biosynthesis in Bacteria: The Origins of Structural and  
Functional Diversity  
Aline Gomez Maqueo Chew and Donald A. Bryant …………113  

Roles of Cyclic Diguanylate in the Regulation of Bacterial Pathogenesis  
Rita Tamayo, Jason T. Pratt, and Andrew Camilli …………131  

Aggresomes and Pericentriolar Sites of Virus Assembly:  
Cellular Defense or Viral Design?  
Thomas Wileman …………149  

As the Worm Turns: The Earthworm Gut as a Transient Habitat for  
Soil Microbial Biomes  
Harold L. Drake and Marcus A. Horn …………169  

Biogenesis of the Gram-Negative Bacterial Outer Membrane  
Martine P. Bos, Viviane Robert, and Jan Tommassen ………………………………………… 191  

SigB-Dependent General Stress Response in Bacillus subtilis and  
Related Gram-Positive Bacteria  
Michael Hecker, Jan Pané-Farré, and Uwe Völker ………………………………………… 215  

Ecology and Biotechnology of the Genus Shewanella  
Heidi H. Hau and Jeffrey A. Gralnick ……………………………………………………… 237  

Nonhomologous End-Joining in Bacteria: A Microbial Perspective  
Robert S. Pitcher, Nigel C. Brissett, and Aidan J. Doherty ……………………………… 259  

Postgenomic Adventures with Rhodobacter sphaeroides  
Chris Mackenzie, Jesus M. Eraso, Madhusudan Choudhary, Jung Hyeob Roh,  
Xiaohua Zeng, Patrice Bruscella, Ágnes Puskás, and Samuel Kaplan …………………… 283  

Toward a Hyperstructure Taxonomy  
Vic Norris, Tanneke den Blaauwen, Roy H. Doi, Rasika M. Harshey,  
Laurent Janniere, Alfonso Jiménez-Sánchez, Ding Jun Jin,  
Petra Anne Levin, Eugenia Mileykovskaya, Abraham Minsky,  
Gradimir Miscevic, Camille Ripoll, Milton Saier, Jr., Kirsten Skarstad,  
and Michel Thellier …………………………………………………………………………… 309  

Endolithic Microbial Ecosystems  
Jeffrey J. Walker and Norman R. Pace ……………………………………………………… 331  

Nitrogen Regulation in Bacteria and Archaea  
John A. Leigh and Jeremy A. Dodsworth …………………………………………………… 349  

Microbial Metabolism of Reduced Phosphorus Compounds  
Andrea K. White and William W. Metcalf ………………………………………………… 379  

Biofilm Formation by Plant-Associated Bacteria  
Thomas Danhorn and Clay Fuqua …………………………………………………………… 401  

Heterotrimeric G Protein Signaling in Filamentous Fungi  
Liande Li, Sara J. Wright, Svetlana Krystofova, Gyungsoon Park,  
and Katherine A. Borkovich ………………………………………………………………… 423  

Comparative Genomics of Protists: New Insights into the Evolution  
of Eukaryotic Signal Transduction and Gene Regulation  
Vivek Anantharaman, Lakshminarayan M. Iyer, and L. Aravind ………………………… 453  

Lantibiotics: Peptides of Diverse Structure and Function  
Joanne M. Willey and Wilfred A. van der Donk ………………………………………… 477  

The Impact of Genome Analyses on Our Understanding of  
Ammonia-Oxidizing Bacteria  
Daniel J. Arp, Patrick S.G. Chain, and Martin G. Klotz ………………………………… 503  

Morphogenesis in Candida albicans  
Malcolm Whiteway and Catherine Bachewich 529  

Structure, Assembly, and Function of the Spore Surface Layers  
Adriano O. Henriques and Charles P. Moran, Jr. 555  

Cytoskeletal Elements in Bacteria  
Peter L. Graumann 589  

Indexes  

Cumulative Index of Contributing Authors, Volumes 57–61 619  

Cumulative Index of Chapter Titles, Volumes 57–61 622  

Errata  

An online log of corrections to Annual Review of Microbiology articles may be found  
at http://micro.annualreviews.org/
