
# Gram-positive three-component antimicrobial peptide-sensing system

Min Li*, Yuping Lai*†, Amer E. Villaruz*, David J. Cha*, Daniel E. Sturdevant§, and Michael Otto*‡

*Laboratory of Human Bacterial Pathogenesis and §Research and Technology Branch, Research Technologies Section, Genomics Unit, Rocky Mountain Laboratories, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Hamilton, MT 59840; and †School of Life Science, East China Normal University, Shanghai 200062, China

Edited by Emil C. Gotschlich, The Rockefeller University, New York, NY, and approved April 19, 2007 (received for review March 8, 2007)

To survive during colonization or infection of the human body, microorganisms must circumvent mechanisms of innate host defense. Antimicrobial peptides represent a key component of innate host defense, especially in phagocytes and on epithelial surfaces. However, it is not known how the clinically important group of Gram-positive bacteria sense antimicrobial peptides to coordinate a directed defensive response. By determining the genome-wide gene regulatory response to human β-defensin 3 in the nosocomial pathogen *Staphylococcus epidermidis*, we discovered an antimicrobial peptide sensor system that controls major specific resistance mechanisms of Gram-positive bacteria and is unrelated to the Gram-negative PhoP/PhoQ system. It contains a classical two-component signal transducer and an unusual third protein, all of which are indispensable for signal transduction and antimicrobial peptide resistance. Furthermore, our data indicate that a very short, extracellular loop with a high density of negative charges in the sensor protein is responsible for antimicrobial peptide binding and the observed specificity for cationic antimicrobial peptides. Our study shows that Gram-positive bacteria have developed an efficient and unique way of controlling resistance mechanisms to antimicrobial peptides, which may provide a promising target for antimicrobial drug development.

innate host defense | *Staphylococcus epidermidis*

Mechanisms of innate host defense play a crucial role in preventing bacterial infection and colonization. The production of antimicrobial peptides (AMPs) is an evolutionarily conserved mechanism of innate host defense and found in virtually all groups of organisms, including amphibians, insects, and other invertebrates. Notably, recent research has demonstrated that AMPs also have a key function in human immune defenses by contributing to the microbiocidal activity of neutrophils, platelets, and epithelial cells. Therefore, bacteria need specific mechanisms of resistance to AMPs to colonize or invade the human body (1). Bacterial resistance mechanisms to AMPs differ with regard to efficiency, specificity, and distribution among species (2). AMPs may be actively extruded from the cytoplasmic membrane, their main target, or inactivated by secreted bacterial proteases. Further, many bacteria make use of the fact that most AMPs are cationic and sequester or repel cationic AMPs by electrostatic interaction on the cell surface. As a sign of the long interplay with bacteria during evolution, the host has invented tricks to circumvent bacterial AMP resistance mechanisms (3). This may occur, for example, by stabilizing AMPs against proteolytic inactivation, such as by the introduction of disulfide bridges or other posttranslational modifications. In addition, many human tissues produce anionic AMPs, such as the dermcidins (4), that are not subject to the bacterial resistance mechanisms based on the repulsion of positively charged molecules. However, anionic AMPs are rare and not as microbiocidal as their cationic counterparts, which suggests that evasion of bacterial resistance mechanisms needs to be carefully weighed against efficiency during evolution.

specific for a bacterial strain or group. For example, because of the different molecular composition of their cell surface, the alteration of surface charge as a resistance mechanism is accomplished by largely unrelated molecular procedures among Gram-positive and Gram-negative bacteria. A prominent mechanism of resistance in Gram-negative bacteria is the incorporation of positively charged aminoarabinose in lipid A, which reduces the anionic character of the cell surface and thus the attraction of cationic AMPs (5). In contrast, Gram-positive bacteria, which do not have lipid A, achieve the same goal by modifying teichoic acids with D-alanyl groups or by including positively charged phospholipids in the cellular membrane (6–10).

Probably because they represent a significant energetic burden to the bacteria, many mechanisms of resistance are subject to gene regulation, ascertaining that they are only active when needed. To this end, bacteria must have sensors for AMPs and corresponding gene regulatory mechanisms. However, we know only one such example, the regulation of lipid A modification by the *Salmonella typhimurium* PhoP/PhoQ two-component regulator (11, 12), homologues of which are widespread among Gram-negative bacteria. The PhoQ membrane histidine kinase part is activated when cationic AMPs displace divalent cations from an extracellular negatively charged loop of the protein. After phosphorylation by activated PhoQ, the PhoP response regulator protein regulates target gene expression. In contrast, although Gram-positive bacteria comprise a series of the most significant pathogens, it is not known whether there are sensors for AMPs in Gram-positive bacteria that trigger a gene regulatory response aimed to combat the activity of AMPs.

## Results

The *Staphylococcus epidermidis* Gene Regulatory Response to Human β-Defensin 3 (hBD3) Is Defined by Up-Regulation of Specific Resistance Mechanisms to Cationic Antimicrobial Peptides. To investigate the gene regulatory response of Gram-positive pathogens to human AMPs, we developed a model system based on the interaction of *S.* *epidermidis* and the cationic human AMP hBD3. *S.* *epidermidis* is the most prominent commensal organism on the human skin and the most frequent cause of medical device-associated infections (13). It is therefore vital to this organism to have countermeasures

Contributions: M.L. and Y.L. contributed equally to this work; M.L., Y.L., and M.O. designed research; M.L., Y.L., A.E.V., D.J.C., D.E.S., and M.O. performed research; M.L., Y.L., D.E.S., and M.O. analyzed data; and M.O. wrote the paper.

The authors declare no conflict of interest.

This article is a PNAS Direct Submission.

Abbreviations: AMP, antimicrobial peptide; hBD3, human β-defensin 3.

Data deposition: The data reported in this paper have been deposited in the Gene Expression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE7163).

‡Present address: Division of Dermatology, University of California at San Diego, 3350 La Jolla Village Drive, La Jolla, CA 92161.

†To whom correspondence should be addressed. E-mail: motto@niaid.nih.gov.

This article contains supporting information online at [www.pnas.org/cgi/content/full/0702159104/DC1](http://www.pnas.org/cgi/content/full/0702159104/DC1).

© 2007 by The National Academy of Sciences of the USA

Table 1. Genome-wide regulatory response of S. epidermidis WT strain 1457 to hBD3

| Gene number | Gene product name | Factor of regulation |
|-------------|-------------------|----------------------|
| SERP0518    | D-alanine-activating enzyme (DltA, EC 6.3.2.-) | 2.09* |
| SERP0519    | Protein DltB | 3.33* |
| SERP0520    | D-alanyl carrier protein DltC | 3.22* |
| SERP0521    | Protein DltD precursor | 4.26* |
| SERP0930    | Lysyltransferase (MprF, EC 2.3.2.3) | 2.81* |
|             | Putative AMP resistance mechanism |  |
| SERP0314    | ABC transporter ATP-binding protein | 3.23* |
| SERP0315    | ABC transporter permease protein | 3.93* |
|             | Further up-regulated genes |  |
| SE0106      | Arginine deiminase (EC 3.5.3.6) | 2* |
| SERP0020    | ABC transporter permease protein | 2.29* |
| SERP0336    | EMG2 protein | 2.02* |
| SERP0387    | Histidinol-phosphate aminotransferase (EC 2.6.1.9) | 2.05* |
| SERP0573    | Oligopeptide transport ATP-binding protein OppF | 2.05* |
| SERP0712    | Hypothetical cytosolic protein | 2.07* |
| SERP1470    | Hypothetical protein | 2.05* |
| SERP1772    | Hypothetical cytosolic protein | 2.15* |
| SERP2170    | Pyruvate, phosphate dikinase (EC 2.7.9.1) | 2.07* |
| SERP2442    | Hypothetical protein | 2.3* |
|             | Down-regulated genes |  |
| SERP0971    | Hypothetical protein | 0.49† |
| SERP1394    | Amino acid transport ATP-binding protein | 0.42† |
| SERP1395    | Arginine transport system permease protein ArtQ | 0.45† |
| SERP1951    | Hypothetical protein | 0.33† |
| SERP1952    | ABC transporter permease protein | 0.41† |

Displays is a complete list of statistically significant changes of gene expression (factor of >2) in an experiment with whole-genome S. epidermidis microarrays. SERP, strain RP62A; SE, strain ATCC12228. *, up-regulation; †, down-regulation.

against AMPs both in its natural habitat, the human skin, and during infection of the human body. hBD3 is a main component of innate immune defense especially on epithelia and represents the only beta defensin with considerable activity against microorganisms at physiological salt concentrations (14, 15).

We used whole-genome microarrays of S. epidermidis to detect genes specifically activated (or repressed) by hBD3. We established (16) an appropriate subinhibitory concentration of hBD3 (10 μg/ml) to be used in gene expression experiments, which was also used in the present study. First, there was a significant increase in the expression of three genetic loci: (i) the *dlt* operon, which is responsible for the D-alanylation of teichoic acids (genes SERP0518–SERP0521) (17); (ii) the *mprF* gene, which is responsible for the biosynthesis of lysyl-phosphatidylglycerol (gene SERP0930) (10); and (iii) a putative transporter system of unknown function (genes SERP0314 and SERP0315) with homology to the Staphylococcus aureus *vraF* and *vraG* genes (Table 1). The function of the former two loci is to increase the concentration of positive charges on the bacterial surface, thereby causing AMP resistance by repulsion of cationic AMPs (9, 10). Increased transcription of all 3 systems with hBD3 was confirmed by quantitative RT-PCR (Fig. 1A). A small number of other genes showed differential expression at low levels. However, this was usually not accompanied by differential expression in genes belonging to the same locus, except for two ABC transporters, which were slightly down-regulated (genes SERP1394, SERP1395, SERP1951, SERP1952). Thus, the

Fig. 1. Gene regulatory responses of S. epidermidis to the antimicrobial peptide hBD3. (A) Quantitative RT-PCR of selected genes from Table 1. (B) Arrangement of *aps* and *vraFG*-homologous genes in the S. epidermidis genome. (C) Quantitative RT-PCR of the two hBD3-regulated target genes *dltB* and *mprF* in WT, *apsS* deletion, genetically complemented *apsS* deletion, and respective control strains. (D) Quantitative RT-PCR of the two hBD3-regulated target genes *dltB* and *mprF* in WT, *apsS*, *apsR*, and *apsX* deletion strains. hBD3 was used at the subinhibitory concentration of 10 μg/ml in all experiments. *, P < 0.05; **, P < 0.01; ***, P < 0.001, vs. corresponding data without hBD3.

gene regulatory response to hBD3 is highly specific and appropriate for the defense against AMPs, inasmuch as it almost exclusively comprises increased expression of loci described to confer specific resistance to cationic AMPs in Staphylococcus (9, 10). In addition, the regulated transporter system may be involved in AMP resistance. This hypothesis, which remains to be tested, is supported by the specificity of the regulatory response, the reported involvement of the S. aureus homologue in resistance to the cationic antibiotic vancomycin (18), and the fact that transporters represent common resistance factors to AMPs (2).

A Three-Component Sensor/Regulator System Transfers the AMP Signal. We discovered that a regulatory system with a yet-unknown function is encoded in the genome of S. epidermidis next to the hBD3-regulated *vraFG*-homologous transporter genes (Fig. 1B). Because regulatory systems in bacteria are often encoded next to the genes they regulate, we hypothesized that this regulatory system is involved in sensing cationic peptides and controlling target gene expression. To investigate this hypothesis, we constructed gene deletion mutants in all of the three putative components of the system, which based on gene similarity comprises the typical components of a bacterial two-component regulatory system (19), a histidine kinase and a response regulator, and as an unusual feature, a third component of unknown function. Gene deletion mutants were constructed by using the method of Bae et al. (20) without insertion of an antibiotic resistance cassette. Importantly, we ascertained by quantitative RT-PCR measurements that expression levels of the genes downstream of the respective deletion site within the putative operon were not altered compared with the WT strain. Furthermore, these experiments demonstrated that the system is not autoregulatory.

To determine whether this system is involved in the regulation of the identified target genes under the influence of cationic AMPs, we analyzed the genome-wide gene regulatory response in the S. epidermidis WT strain in comparison with the three deletion strains with and without addition of hBD3

Table 2. Genome-wide regulatory responses in aps gene deletion strains with and without addition of hBD3, selected genes from experiments with whole-genome S. epidermidis microarrays.

| Gene product               | $apsX^{+}/WT^{-}$ | $apsR^{+}/apsX^{-}$ | $apsS^{+}/apsR^{-}$ | $apsX^{-}/WT^{-}$ | $apsR^{-}/WT^{+}$ | $apsS^{-}/WT^{+}$ | $apsS^{+}/WT^{+}$ |
|----------------------------|-------------------|---------------------|---------------------|--------------------|--------------------|--------------------|--------------------|
| DltA                       | 2.09*             | 1.00†              | 1.24†              | 1.05†             | 0.38‡              | 0.19‡              | 0.22‡              |
| DltB                       | 3.33*             | 1.01†              | 1.16†              | 0.98†             | 0.40‡              | 0.11‡              | 0.18‡              |
| DltC                       | 3.22*             | 1.00†              | 1.54†              | 1.26†             | 0.31‡              | 0.10‡              | 0.13‡              |
| DltD                       | 4.26*             | 0.88†              | 1.29†              | 1.09†             | 0.38‡              | 0.11‡              | 0.16‡              |
| MprF                       | 2.81*             | 1.10†              | 0.72†              | 0.81†             | 0.70†              | 0.28‡              | 1.12†              |
| ABC transporter            | 3.23*             | 1.17†              | 1.46†              | 1.03†             | 0.91†              | 0.23‡              | 0.56‡              |
| ATP-binding protein        |                   |                     |                     |                    |                    |                    |                    |
| ABC transporter permease protein | 3.93*             | 1.06†              | 1.24†              | 1.09†             | 0.73†              | 0.19‡              | 0.54‡              |

The entire list is shown in SI Table 4. *, up-regulation; †, no significant change; ‡, down-regulation; +, with hBD3; −, without hBD3.

A

ApsS
extracellular loop
transmembrane segment 1 (inside to outside) transmembrane segment 2 (outside to inside)
MNNFRWFWFIKSRIINWILWILFLNIILLGVAYIDYEISVESVFYIVILNVGLSILFLLTFVKEVRLSK
HFYEDKEIEEIKHKDLAETPFQQQVIDLYRHIAAQKEKVVEQQLQIKNHEQTITEFVHDIKTPVTAMKL
LIDQENDDQRKRALLFEWSRINEMLDKQLYLTRLETHHRDMYFDYISLKRMVIDEIQVTRHISQAKGIGF
ELDFKDEQKVYTDVKWCRMMIRQVLNSNLKYSDNSTINLSGYNIEGHVVLKIKDYGRGISKRDLPRIFDR
GFTSTTDDRNDTASSGMGLYLVQSVEKOLGIEVKVDSIVGKGTTTFYIFPQONEIIERMSKVTRLSF

B
kDa
wt ΔapsS

D
relative quantity dltB
no antiserum 1:10000 1:5000 1:2000 1:1000

C
insoluble
cytosolic
membrane

Fig. 3. Role of the ApsS extracellular loop in antimicrobial peptide binding. (A) Amino acid sequence of the ApsS sensor showing the locations of transmembrane and extracellular loop segments. Negatively charged amino acids in the extracellular loop are marked. (B) Reaction of antiserum with total cell extracts of WT and apsS deletion strains. (C) Subcellular location of ApsS. The membrane fraction was prepared by ultracentrifugation for 90 min at 105,000 × g and resuspending the pellet in buffer containing 1% Triton X-100. The supernatant yielded the cytoplasmic fraction, whereas still insoluble material was again centrifuged and directly resuspended in SDS/PAGE loading buffer. (D) Quantitative RT-PCR of dltB gene expression in S. epidermidis WT strain. Samples were incubated with different antiserum concentrations in 10 mM PBS/100 mM NaCl for 1 h before hBD3 at 10 μg/ml was added, and samples were incubated for 3 h more before RNA isolation. In all experiments, cells were harvested at OD600 nm = 3 after inoculation from a preculture grown overnight. ***, P < 0.001 vs. sample without antiserum.

with an anionic AMP for their capacities to influence target gene expression and confirmed that the effect is mediated via aps by using the apsS deletion mutant as control (Fig. 4). In contrast to the anionic AMP dermcidin, all tested cationic AMPs were active in

relative quantity (dltB)

without peptide dermcidin hBD3 tachyplesin nisin LL-37 magainin histatin brevinin without peptide dermcidin hBD3 tachyplesin nisin LL-37 magainin histatin brevinin
wt apsS

Fig. 4. Specificity of the aps response for cationic AMPs. Quantitative RT-PCR measurements of the dltB target gene under the influence of subinhibitory concentrations of various AMPs in the WT and apsS deletion strains are shown. ***, P < 0.001; **, P < 0.01 vs. corresponding sample without peptide. For most peptides, concentrations were the same as for hBD3 (2 μM, which is ≈10 μg/ml for hBD3). Because of significant killing activity at this concentration, lower concentrations were used for brevinin (1 μM) and nisin (0.1 μM).

Fig. 5. Antimicrobial resistance conferred by the aps system. Killing assays of WT, aps deletion, genetically complemented, and respective control strains with the cloning vector are shown.

this test, confirming the notion that binding occurs to the extracellular anionic loop and the aps system reacts specifically to the presence of cationic peptides. Moreover, these results show that the system can be activated by a wide variety of cationic peptides irrespective of their structure, including peptides with alpha-helical structure (magainin, LL-37), bridged structure (hBD3, tachyplesin, brevinin), the histidine-rich histatin, or the rigid lantibiotic nisin.

Binding of cationic AMPs to the Gram-negative AMP sensor protein PhoQ can be competed by divalent cations, such as Mg2+ (12). To investigate whether the ApsS protein functions in a similar fashion, we determined the transcription of the aps target genes dltB and the AraC-type regulator gene at different concentrations of Mg2+. Whereas addition of Mg2+ ions influenced the absolute expression levels of both genes, a comparable relative effect was observed in the apsS deletion and WT strains, indicating that the influence of Mg2+ on aps target genes is not primarily mediated via aps. Thus, it appears that activation of the aps system is not mediated by competitive binding of cationic AMPs and Mg2+.

The aps System Is Crucial for Antimicrobial Peptide Resistance. To determine whether the regulatory function of aps is crucial for the establishment of antimicrobial peptide resistance, we performed killing assays of WT and apsS, apsR, and apsX deletion strains in comparison, using hBD3 as standard AMP (Fig. 5). Resistance to hBD3 was significantly decreased in all three deletion mutants compared with the WT strain and restored in genetically complemented mutant strains. This was confirmed by determination of minimal inhibitory concentration values (Table 3). These findings demonstrate that functionality of the aps system plays a key role in antimicrobial peptide resistance in S. epidermidis and confirm that all three components of the system are indispensable for its function.

Discussion

The importance of AMP sensing mechanisms for the survival of bacterial pathogens has been frequently emphasized, and the

Table 3. Minimal inhibitory concentration values against hBD3 (µg/ml)

| MIC | WT | apsX | apsX(pT181) | apsX(pTapsX) | apsR | apsR(pT181) | apsR(pTapsR) | apsS | apsS(pT181) | apsS(pTapsS) |
|-----|----|------|-------------|--------------|------|-------------|-------------|------|-------------|---------------|
| 90% inhibition | 264 | 72 | 80 | 180 | 54 | 64 | 224 | 128 | 136 | 224 |
| 50% inhibition | 192 | 48 | 96 | 24 | 32 | 128 | 64 | 160 |  |

MIC, minimal inhibitory concentration.

PhoP/PhoQ AMP sensory system of Gram-negative bacteria has been suggested as a premier target for antimicrobial drug discovery (12, 22). However, many significant human pathogens are Gram-positive and do not have homologues of PhoP/PhoQ. We hypothesized that an AMP-sensing mechanism exists in Gram-positive bacteria and investigated this hypothesis in the Gram-positive pathogen *S. epidermidis*. *S. epidermidis* is the most frequent colonizer of the human skin and may cause serious infections especially on indwelling medical devices (13).

Here, we demonstrate that *S. epidermidis* senses antimicrobial peptides by a three-component sensor/regulator system, which we termed *aps* for antimicrobial peptide sensor. Cationic, but not anionic, antimicrobial peptides induce a gene regulatory response via *aps* that comprises the up-regulation of the two described main resistance mechanisms to cationic antimicrobial peptides in Gram-positive bacteria, i.e., *dlt*-operon mediated D-alanylation of teichoic acids and MprF-mediated incorporation of lysyl-phosphatidylglycerol in the cytoplasmic membrane (2). A third *aps*-regulated gene locus with similarity to ABC transporter genes may also be involved in AMP resistance (Fig. 6). In general, transporter systems that confer resistance to AMPs may function by exporting the AMP from the target cytoplasmic membrane or, alternatively, by import of the AMP into the cell to forward it to proteolytic inactivation (23, 24). Notably, deletion of any component of the regulatory system led to a significant decrease in antimicrobial peptide resistance. These findings emphasize the importance of controlling AMP resistance mechanisms. On the other hand, considering the very specific gene regulatory response, they underline the key role of the regulated target mechanisms for AMP resistance.

The *aps* regulatory system is very well conserved among staphylococci. The ApsS and ApsX proteins show identity values of 40–45% among staphylococci, whereas the response regulator protein ApsR is even more highly conserved. In addition, there are ApsR- and ApsS-homologous proteins in other Gram-positive bacteria, including the VirRS regulatory system of *Listeria monocytogenes* (25), suggesting that these bacteria might be able to respond to cationic AMPs in the same way as *S. epidermidis*. Similar to *S. epidermidis*, the VirRS regulator of *L. monocytogenes* controls expression of the *dlt* and *mprF* loci (6, 25). However, the *L. monocytogenes vir* system differs from the *aps* locus of *S. epidermidis*, inasmuch as the *virS* and *virR* genes are not adjacent and there is no *apsX* homologue, which appears to be exclusive for staphylococci. Of note, the Aps proteins do not share pronounced sequence similarity with the corresponding proteins of the PhoP (31% identity)/PhoQ (10% identity) system. In addition, our data indicate that cationic AMPs activate the *aps* system by direct binding to an extracellular loop (Fig. 6). Although this is reminiscent of PhoQ activation, the extremely short length of the ApsS extracellular loop represents a dramatic difference. Moreover, our data do not support that ApsS responds to binding of divalent cations, which is a key feature of the PhoQ protein (12). This shows that staphylococci, and likely other Gram-positive bacteria, have found a different way to sense cationic AMPs and control the respective defensive mechanisms. In addition, our data indicate that the

![Figure 6](attachment:figure_6.png)

Fig. 6. The *aps* system. Model of *aps* function and regulated target mechanisms. Cationic AMPs that usually function by pore formation in the cytoplasmic membrane (shown on the right), bind to the anionic loop of ApsS (left) and trigger activation of the *aps* system (gray), leading to activation of *aps*-regulated target mechanisms: (*i*) D-alanylation of teichoic acids by the *dlt* system (purple), (*ii*) lysinylation of phospholipids by MprF (yellow), and (*iii*) putative transport of AMPs through an ABC transporter (blue). The transporter may work either by expelling the pore-forming peptides from the membrane or by importing them into the cell for proteolytic inactivation. ApsX does not contain a putative signal peptide sequence (Signal P software, Version 3.0; Technical University of Denmark, www.cbs.dtu.dk). It contains a possible transmembrane sequence at the very C terminus with most of the protein predicted to be cytoplasmic. It was therefore drawn as cytoplasmic, but membrane-attached.
reported influence on cations on *dlt* gene expression (26) is not mediated by the *aps* system.

The *aps* system exclusively responds to cationic AMPs and controls mechanisms of resistance that are specific for cationic AMPs and Gram-positive bacteria. This accounts at least for the characterized *dlt* and *mprF* systems, whereas the role and specificity of the *aps*-regulated putative AMP transporter remain to be elucidated. These findings underpin the importance of cationic AMPs in innate host defense and the notion that noncationic AMPs, such as the human dermcidin, represent an evolutionary response of the host aimed to circumvent bacterial defense mechanisms. We have recently found that *S*. *epidermidis* can respond to anionic and cationic AMPs by an up-regulation of proteolytic defense mechanisms that proved especially effective for the defense against the anionic dermcidin (16). However, in contrast to *aps* control, that response is far less specific and probably represents a phenomenon more similar to the general stress response.

Data obtained with the *aps* deletion mutants suggest that the regulatory task of the *aps* system exceeds the response to cationic AMPs and includes regulatory networking and the control of several vital functions such as autolysis. However, deletion of the regulator is not likely to be found in nature, and the dramatic consequences might thus not represent a common physiologic situation. Interestingly, most genes and loci that were affected under these conditions were up-regulated in the deletion strains, suggesting that the changes might be due to derepression and a possibly different aspect of regulation by *aps*. The molecular details of the *aps* regulatory mechanism, especially the unusual ApsX protein, certainly warrant further investigation.

We identified a Gram-positive antimicrobial peptide-sensor and regulator with the same task, but significant difference in its composition and target resistance mechanisms to the PhoP/PhoQ system of Gram-negative bacteria, indicating that Gram-positive bacteria have their own, unique way of controlled defense to cationic antimicrobial peptides. This mechanism is likely of great importance for the efficient colonization and infectivity of opportunistic human pathogens, such as, especially, the staphylococci.

---

### Materials and Methods

**Bacterial Strains and Growth Conditions, Construction of Deletion Mutants, Complementation Plasmids, Killing and Minimal Inhibitory Concentration Assays, Statistical and Sequence Analyses, and Microarray Experiments and Analyses.** Detailed protocols are described in **SI Table 5** and **SI Text**.

#### Quantitative RT-PCR
Oligonucleotide primers and probes (**SI Table 6**) were designed with Primer Express software, Version 2.0 (Applied Biosystems, Foster City, CA) and synthesized by Applied Biosystems. The experiments were performed in triplicate as described in ref. 27, with 16S rRNA as a control.

#### Zymographic Analysis
Bacteria were grown for 12 h in TSB, cells were harvested and resuspended in 1% SDS, and autolysins were isolated by boiling the cells with 1% SDS for 10 min. Bacteriolytic enzyme profiles were analyzed as described in ref. 28.

#### Immunological Procedures
Rabbit antiserum to the extracellular loop of ApsS was developed by Sigma–Genosys (The Woodlands, TX) against the conjugated peptide YIDYEISVESVF. This antiseraum was blocked with cell extract from the *apsS* deletion mutant strain obtained by vortexing cell suspension with glass beads from 500 ml of culture grown to OD~600~ = 3.0. Blocked antiserum stock was obtained after adding the cell extract to 100 ml of 1:100 diluted antiserum, incubating for 16 h at 4°C, and final centrifugation for 30 min at 28,000 × g at 4°C.

---

We thank Taeok Bae for plasmid pKOR1 and Frank DeLeo for critically reading the manuscript. This work was supported by the Intramural Research Program of the National Institute of Allergy and Infectious Diseases.

---

### References

1. Hancock RE, Diamond G (2000) *Trends Microbiol* 8:402–410.
2. Peschel A (2002) *Trends Microbiol* 10:179–186.
3. Peschel A, Sahl HG (2006) *Nat Rev Microbiol* 4:529–536.
4. Schittek B, Hipfel R, Sauer B, Bauer J, Kalbacher H, Stevanovic S, Schirle M, Schroeder K, Blin N, Meier F, Rassner G, Garbe C (2001) *Nat Immunol* 2:1133–1137.
5. Shafer WM, Casey SG, Spitznagel JK (1984) *Infect Immun* 43:834–838.
6. Thedieck K, Hain T, Mohamed W, Tindall BJ, Nimtz M, Chakraborty T, Wehland J, Jansch L (2006) *Mol Microbiol* 62:1325–1339.
7. Abachin E, Poyart C, Pellegrini E, Milohanic E, Fiedler F, Berche P, Trieu-Cuot P (2002) *Mol Microbiol* 43:1–14.
8. Kovacs M, Halfmann A, Fedtke I, Heintz M, Peschel A, Vollmer W, Hakenbeck R, Bruckner R (2006) *J Bacteriol* 188:5797–5805.
9. Peschel A, Otto M, Jack RW, Kalbacher H, Jung G, Gotz F (1999) *J Biol Chem* 274:8405–8410.
10. Peschel A, Jack RW, Otto M, Collins LV, Staubitz P, Nicholson G, Kalbacher H, Nieuwenhuizen WF, Jung G, Tarkowski A, *et al.* (2001) *J Exp Med* 193:1067–1076.
11. Trent MS, Pabich W, Raetz CR, Miller SI (2001) *J Biol Chem* 276:9083–9092.
12. Bader MW, Sanowar S, Daley ME, Schneider AR, Cho U, Xu W, Klevit RE, Le Moual H, Miller SI (2005) *Cell* 122:461–472.
13. Vuong C, Otto M (2002) *Microbes Infect* 4:481–489.
14. Harder J, Bartels J, Christophers E, Schroder JM (2001) *J Biol Chem* 276:5707–5713.
15. Harder J, Schroder JM (2005) *Chem Immunol Allergy* 86:22–41.
16. Lai Y, Villaruz AE, Li M, Cha DJ, Sturdevant DE, Otto M (2007) *Mol Microbiol* 63:497–506.
17. Perego M, Glaser P, Minutello A, Strauch MA, Leopold K, Fischer W (1995) *J Biol Chem* 270:15598–15606.
18. Kuroda M, Kuwahara-Arai K, Hiramatsu K (2000) *Biochem Biophys Res Commun* 269:485–490.
19. Igo MM, Slauach JM, Silhavy TJ (1990) *New Biol* 2:5–9.
20. Bae T, Schneewind O (2006) *Plasmid* 55:58–63.
21. Cheung AL, Zhang G (2002) *Front Biosci* 7:d1825–d1842.
22. Brodsky IE, Gunn JS (2005) *Mol Interv* 5:335–337.
23. Parra-Lopez C, Baer MT, Groisman EA (1993) *EMBO J* 12:4053–4062.
24. Otto M, Gotz F (2001) *Res Microbiol* 152:351–356.
25. Mandin P, Fsihi H, Dussurget O, Vergassola M, Milohanic E, Toledo-Arana A, Lasa I, Johansson J, Cossart P (2005) *Mol Microbiol* 57:1367–1380.
26. Koprivnjak T, Mlakar V, Swanson L, Fournier B, Peschel A, Weiss JP (2006) *J Bacteriol* 188:3622–3630.
27. Yao Y, Vuong C, Kocianova S, Villaruz AE, Lai Y, Sturdevant DE, Otto M (2006) *J Infect Dis* 193:841–848.
28. Vuong C, Gotz F, Otto M (2000) *Infect Immun* 68:1048–1053.