
Cannabidiolic-acid synthase, the chemotype-determining enzyme in the fiber-type *Cannabis sativa*

Futoshi Taura*, Supaart Sirikantaramas¹, Yoshinari Shoyama, Kazuyoshi Yoshikai, Yukihiro Shoyama, Satoshi Morimoto

*Graduate School of Pharmaceutical Sciences, Kyushu University, Fukuoka 812-8582, Japan*

Received 14 February 2007; accepted 15 May 2007

Available online 25 May 2007

Edited by Mark Stitt

---

**Abstract** Cannabidiolic-acid (CBDA) synthase is the enzyme that catalyzes oxidative cyclization of cannabigerolic-acid into CBDA, the dominant cannabinoid constituent of the fiber-type *Cannabis sativa*. We cloned a novel cDNA encoding CBDA synthase by reverse transcription and polymerase chain reactions with degenerate and gene-specific primers. Biochemical characterization of the recombinant enzyme demonstrated that CBDA synthase is a covalently flavinylated oxidase. The structural and functional properties of CBDA synthase are quite similar to those of tetrahydrocannabinolic-acid (THCA) synthase, which is responsible for the biosynthesis of THCA, the major cannabinoid in drug-type *Cannabis* plants.

© 2007 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.

Keywords: Cannabidiolic-acid synthase; Cannabinoid; Biosynthesis; Molecular cloning; *Cannabis sativa*

---

and chronic neurodegeneration [5,6]. Thus, cannabinoids such as THC and CBD are regarded as promising medicinal resources for treating various diseases [7].

Cannabinoids are classified into two types, neutral cannabinoids and cannabinoid acids, based on whether they contain a carboxyl group or not. It is known that, in fresh plants, the concentrations of neutral cannabinoids are much lower than those of cannabinoid acids. Thus, THC and CBD are derived artificially from their acidic precursors tetrahydrocannabinolic-acid (THCA) and cannabidiolic acid (CBDA) by nonenzymatic decarboxylation [8,9] (Fig. 1). With respect to the biosynthesis of cannabinoids, we have previously reported the identification and purification of novel enzymes, THCA synthase and CBDA synthase, which are specifically expressed in respective drug-type (THCA-rich) and fiber-type (CBDA-rich) chemical phenotypes of *C. sativa* [10,11]. These enzymes are the first cannabinoid synthases to be studied, and potentially attractive targets for various biotechnological applications as they produce the direct precursors of pharmacologically active cannabinoids. In addition, these enzymes catalyze an unique biosynthetic reaction, the stereospecific oxidative cyclization of the geranyl group of cannabigerolic-acid (CBGA) (Fig. 1), for which no equivalent reaction has been reported to date. Furthermore, the structural, functional, and genetic relationship between THCA synthase and CBDA synthase is also of great interest.

The interesting properties of cannabinoid synthases led us to attempt the molecular cloning of these enzymes, and recently, we have successfully obtained a cDNA encoding THCA synthase from a drug-type *Cannabis* plant (Mexican strain) [12]. The biochemical characterization of the recombinant enzyme demonstrated that THCA synthase is a covalently flavinylated oxidase [12]. In addition, we have developed THCA-producing tobacco hairy roots by using recombinant techniques, opening the way to the biotechnological production of THCA [12]. In contrast to THCA synthase, CBDA synthase has remained to be further studied at the molecular level. In the present study, we cloned and characterized a novel cDNA (*CBDAS*) encoding CBDA synthase from a fiber-type *C. sativa* (CBDA strain).

---

### 1. Introduction

Cannabinoids, which are found only in *Cannabis sativa*, are secondary metabolites possessing alkylresorcinol and monoterpenoid moieties in their molecules. More than 60 cannabinoids have been isolated from marijuana or fresh *Cannabis* leaves [1]. Among them, tetrahydrocannabinol (THC) is the well-known psychoactive component of marijuana [2]. Recent studies have demonstrated that this cannabinoid exerts a variety of therapeutic activities through the cannabinoid receptors CB1 and CB2, which exist in the mammalian brain and immune cells, respectively [3,4]. Besides THC, the pharmacological properties of cannabidiol (CBD), an isomer of THC, have also attracted a great deal of attention although this cannabinoid does not activate cannabinoid receptors. For example, it has been reported that CBD is a potent antioxidative and anti-inflammatory agent providing neuroprotection against acute

---

*Corresponding author. Fax: +81 92 642 6582.  
E-mail address: taura@phar.kyushu-u.ac.jp (F. Taura).*

¹ Present address: Graduate School of Pharmaceutical Sciences, Chiba University, Chiba 263-8522, Japan.

**Abbreviations:** CBD, cannabidiol; CBDA, cannabidiolic-acid; CBDAS, cannabidiolic-acid synthase cDNA; CBGA, cannabigerolic-acid; THC, tetrahydrocannabinol; THCA, tetrahydrocannabinolic-acid

---

### 2. Materials and methods

#### 2.1. Plant materials and reagents

*C. sativa* plants (CBDA strain) [13] were cultivated in the herbal garden of the Graduate School of Pharmaceutical Sciences, Kyushu University. CBDA was purified from dried leaves of *C. sativa* [14], and CBGA was chemically synthesized as described previously [11].

(5′-RACE) [16]. The first round of PCR was performed with gene-specific primer e and adapter primer i in the presence of poly(dA)-tailed cDNA. The cDNA fragment was obtained by nested PCR with gene-specific primer f and adapter primer j. All PCR products were cloned into the vector pUC119 and sequenced on an Applied Biosystems 310 genetic analyzer.

### 2.5. Expression of CBDA synthase in insect cell culture

Full-length CBDAS was amplified using gene-specific primers g and h with a proofreading polymerase (KOD DNA polymerase, Toyobo). The amplified cDNA was ligated into the vector pFastBacI (Invitrogen), and a recombinant baculovirus harboring CBDAS was prepared and amplified according to the manufacturer’s instructions (Invitrogen).

For production and purification of the recombinant CBDA synthase, a 1000-ml suspension culture of *Spodoptera frugiperda* cells (~2.0 × 10⁶ cells/ml) was infected with virus at a multiplicity of infection of 1.0. The culture was harvested 72 h after infection, and centrifuged. The supernatant was directly applied to a hydroxylapatite column (1.0 × 10.0 cm) equilibrated with 10 mM phosphate buffer (pH 7.0) containing 3 mM mercaptoethanol (buffer A). The bound proteins were eluted with a 600-ml linear gradient of buffer A to 0.5 M potassium phosphate buffer (pH 7.0) containing 3 mM mercaptoethanol. Fractions containing CBDA synthase activity were applied to another hydroxylapatite column (1.0 × 10.0 cm). The recombinant CBDA synthase was eluted with a 600-ml gradient of NaCl (0–2 M) in buffer A. The most active fractions were used for the biochemical characterization.

### 2.6. Spectroscopic characterization of CBDA synthase and its peptide fragment

The absorption spectrum of the recombinant CBDA synthase was measured on a Hitachi Model U-2001 spectrophotometer. For fluorescence analysis, the recombinant enzyme dissolved in 100 mM sodium citrate buffer (pH 4.0) was irradiated at 450 nm, and the fluorescence emission spectrum was measured on a Hitachi Model F-2000 fluorometer. Flavin peptide was prepared and characterized as follows. 100 μg of CBDA synthase was digested with trypsin and chymotrypsin, and the resulting flavin peptide solution was treated with phosphodiesterase. Fluorescence emission spectra of peptide samples before and after phosphodiesterase treatment were analyzed as described above. The release of AMP from flavin peptide was detected enzymatically by using adenylate kinase [17].

### 2.7. Construction of H114A mutant

The mutant cDNA was prepared by a PCR-based mutagenesis [18]. The oligonucleotide primer used was 5′-AAGCGGTGGGCGCTGGCTGAG-3′ in the sense direction (New Ala codon in boldface). The prepared mutant cDNA was introduced into pFastbacI, and the H114A enzyme was expressed by the baculovirus expression system.

### 2.8. CBDA synthase Assay

The standard reaction mixture consisted of 200 μM CBGA, 0.1% (w/v) Triton X-100, and 100 mM sodium citrate buffer (pH 5.0) in a total volume of 500 μl. The reaction was started by adding 100 μl of enzyme solution, and the mixture was incubated at 30 °C for 24 h. After termination of the reaction with 600 μl of methanol, a 100-μl aliquot was subjected to analytical HPLC [11].

To examine the molecular oxygen requirement of CBDA synthase, the substrate and enzyme solutions were pre-incubated with glucose oxidase and catalase in the presence of glucose [19]. The hydrogen peroxide generated after the CBDA synthase reaction was quantified using a horseradish peroxidase with 4-aminoantipyrene as substrate [20].

## 3. Results and discussion

### 3.1. cDNA cloning and structural characteristics of CBDA synthase

The cloning of CBDAS was carried out by reverse transcription and polymerase chain reactions. To design degenerate

F. Taura et al. / FEBS Letters 581 (2007) 2929–2934

Table 1  
N-terminal and internal amino acid sequences of CBDA synthase  

| Protein/peptide | Sequence |
|------------------|----------|
| N-terminal       | NPRENFLKXFSQYIPNNATNLKLVY |
| Glu-C-1          | XAIPFPHRAGILYEL |
| Glu-C-2          | XVFVQILEKLYEEDI |

The letter X indicates that the identity of the amino acid is ambiguous.

PCR primers, we analyzed partial amino acid sequences of CBDA synthase purified from *C. sativa*. First, we determined the N-terminal sequence of the enzyme containing 25 amino acid residues (Table 1). The purified enzyme was then treated with endoproteinase Glu-C, and the resulting fragments (Glu-C-1 and Glu-C-2) were N-terminally sequenced (Table 1). When PCR was carried out using the degenerate primers designed from the N-terminal and Glu-C-1 sequences, a ~1200-bp PCR product was obtained. The amplification of cDNA fragments containing 3′- and 5′-end regions was achieved by 3′- and 5′-RACE, respectively. Then, PCR with gene-specific primers successfully amplified the full-length cDNA named CBDAS (GenBank™ Accession No. AB292682). CBDAS is the second gene involved in cannabinoid biosynthesis to be cloned.

This gene consisted of a 1632-nucleotide open reading frame encoding a 544-amino acid polypeptide (Fig. 2). The deduced


ATGAAGTGCTCAACATTCTCCTTTTGGTTTGTTCGCAAGATAATATTTTCTTTTTCTCATTC AATATCCAAACTTCCATTGCTAATCCT 90
MetLysCysSerThrPheSerPheTrpPheValCysLysIleIlePhePhePheSerPheAsnIleGlnThrSerIleAlaAsnPro 30
CGAGAAAACTTCCTTAAATGCTTCTCGCAAATATATTTCCAATAATGCAACAAAATCTAAAAC TCGTATACACTCAAAACAACCATTGTAT 180
ArgGluAsnPheLeuLysCysPheSerGlnTyrIleProAsnAsnAlaThrAsnLeuLysLeuValTyrThrGlnAsnAsnProLeuTyr 60
ATGTCTGTCCTAAATTCGACAATACACAATCTTAGATTCACCTCTGACACAACCCCAAAACC ACTTGTTATCGTCACTCCTTCACATGTC 270
MetSerValLeuAsnSerThrIleHisAsnLeuArgPheThrSerAspThrThrProLysProLeuValIleValThrProSerHisVal 90
TCTCATATCCAAGGCATATTCTATGCTCCAAGAAAGTTGGCTTGCAGATTCGAACTCGAAGT GGGTGCATGATTCTGAGGGCATGTC 360
SerHisIleGlnGlyThrIleLeuCysSerLysValGlyLeuGlnIleArgThr ArgSerGlyGlyHis AspSerGluGlyMetSer 120
TACATATCTCAAGTCCCATTGTATAGACTTGGAGAAACATGCGTTCAATCAAAATAGATGTT CATAGCCAAACTGCATGGGTTGAA 450
TyrIleSerGlnValProPheValIleValAspLeuArgAsnMetArgSerIleLysIleAspValHisSerGlnThrAlaTrpValGlu 150
GCCGGAGCTACCCCTTGGAGAAGTTTATTATTTGGGTTAATGAGAAAAATGGAATCTTAGTT TGGGCTGGGTATTGCCCTACTGTTGC 540
AlaGlyAlaThrLeuGlyValTyrTyrTrpValAsnGluLysAsnGluSerLeuAlaAlaGlyTyrCysProThrValCys 180
GCAGGTGGACATTTTGGTGGAGGAGGCTATGGACCATTGATGAGAAAACTATGGGCCTCGCG GCTGATAATATCATTGATGCACACTTAGTC 630
AlaGlyGlyHisPheGlyGlyGlyTyrGlyProLeuMetArgAsnTyrGlyLeuAlaAlaAspAsnIleIleAspAlaHisLeuVal 210
AACGTTCATGGAAAAGTGCTAGATCGAAAATCTATGGGGGAAGATCTTTTGGGCTTTACGTG GTGTTGGAGCAGAAAGCTTCGGAATC 720
AsnValHisGlyLysValLeuAspArgLysSerMetGlyGluAspLeuPheTrpAlaLeuArgGlyGlyAlaGluSerPheGlyIle 240
ATTGTAGCATGGAAAATTAGACTGGTTGCTGCCAAAGTCTACTATGTTTAGTGTAAAAAGAT CATGGGAGATACATGAGCTTGTCAAG 810
IleValAlaTrpLysIleArgLeuValAlaValProLysSerThrMetPheSerValLysIleIleMetGluIleHisGluLeuValLys 270
TTAGTTAACAAATGGCAAAATATTGCTTACAAGTATGCAAAAGATTTATTACTCATGACTC ATTCAAACTAGGAACATTACAGATAAT 900
LeuValAsnLysTrpGlnAsnIleAlaTyrLysTyrAspLysAspLeuLeuLeuMetThrHisPheIleThrArgAsnIleThrAspAsn 300
CAAGGGAAGAATAAGACAGCAATACACACTTACTTCTCTTCAGTTTTCCCTTGGGAGTGGAT AGTCTAGTCGACTTGATGAACAAGAGT 990
GlnGlyLysAsnLysThrAlaIleHisThrTyrPheSerSerValPheLeuGlyValAspSerLeuValAspLeuMetAsnLysSer 330
TTT CCTGAGTTGGGTATTTAAAAAAAAACGGATTGCAGACAATTGGAGCTGGATTGATCTATCATCTTCTATAGTGGTGTGTAAATTACGAC 1080
PheProGluLeuGlyIleLysThrAspArgGlnLeuSerTrpIleAspThrIleIlePheTyrSerGlyValValAsnTyrAsp 360
ACTGATAATTTTAACAAGGAAATTTTGCTTAGATAGATCCGGCTGGGCAAAAGGTGCTTTCA AGATTAAGTTAGACTACGTTAAGAAACCA 1170
ThrAspAsnPheAsnLysGluIleLeuLeuAspArgSerAlaGlyGlnAsnGlyAlaPheLysIleLysLeuAspTyrValLysLysPro 390
ATTCCAGAATCTGTATTTTGTCCAAAATTTTGGAAAAATATATGAAGAAGATATAGGAGCTGGGATGTGCGTTGTACCCCTTACGGTGGT 1260
IleProGluSerValPheValGlnIleLeuGluLysLeuTyrGluGluAspIleGlyAlaGlyMetTyrAlaLeuTyrProTyrGlyGly 420
ATAATGGATGAGATTTTCAGAATCAGCAATTCCTCATCGAGCTGGAATCTTGTATGGAGTTATGGTACATATGTAGTTGGGAGAAG 1350
IleMetAspGluIleSerGluSerAlaIleProPheProHisArgAlaIleLeuTyrGluLeuTrpTyrIleCysSerTrpGluLys 450
CAAGAAGATAACGAAAAGCATCTAAACTGGATTAGAAATATTTTATAACTTTCATGACTCCT ATGTGTC CAAAATCCAAGAATTGGCATAT 1440
GlnGluAspAsnGluLysHisLeuAsnTrpIleArgAsnIleTyrAsnPheMetThrProTyrValSerLysAsnProArgLeuAlaTyr 480
CTCAATTATAGAGACCTTGATATAGGAATAAATGATCCCAAAGAATCCAAATAATTACACACA AGCAGTATTTGGGGTGAGAAGTATTTT 1530
LeuAsnTyrArgAspLeuAspIleGlyIleAsnAspProLysAsnProAsnTyrThrGlnAlaArgIleTrpGlyGluLysTyrPhe 510
GGTAAAAATTTTGACAGGCTAGTAAAAAGTGGAAACCCCTGGTTGATCCCAAATAACTTTTT TAGAAACGAACAAAGCATCCCACCTCTTCCA 1620
GlyLysAsnPheAspArgLeuValLysThrLeuValAspProAsnAsnPhePheArgAsnGluGlnSerIleProProLeuPro 540
CGGCATCGTCATTTA
ArgHisArgHisEND


Fig. 2. Nucleotide and deduced amino acid sequences of CBDA synthase. The amino acid sequences determined from the purified native enzyme are underlined. The putative signal peptide cleavage site is indicated by an arrow. The asterisks indicate potential Asn-glycosylation sites. The putative flavin-binding site consensus sequence is boxed.

primary structure contained all the partial amino acid sequences. PSORT analysis (http://psort.nibb.ac.jp/) of the sequence indicated that the first 28 amino acid residues constitute a cleavable signal peptide (Fig. 2), being consistent with the N-terminal start position of CBDA synthase. Hence, we concluded that mature CBDA synthase consists of 517 amino acid residues. However, the theoretical molecular weight (58,863 Da) calculated from the amino acid composition of the mature protein is apparently lower than that (~74 kDa) of the enzyme purified from *C. sativa* [11]. The difference may be due to post-translational modifications such as glycosylation, since 7 possible Asn-glycosylation sites were confirmed to be present in mature CBDA synthase (Fig. 2).

It is of great interest to reveal the structural homology between CBDA synthase and THCA synthase as these enzymes catalyze similar oxidocyclization reactions with the common substrate CBGA. Hence, we aligned the amino acid sequences of these cannabinoid synthases, and found that the primary structure of CBDA synthase has a surprising level of homology to that of THCA synthase (83.9% identity in a 544-amino acid overlap). The high sequence identity clearly indicated the evolutionary relationship between these cannabinoid synthases, and suggested that CBDA synthase and THCA synthase developed from a common ancestor via changes in the amino acid residues governing the structure of their products.

CBDA synthase also showed considerable homology (40–50% identity) to several oxidoreductases such as berberine bridge enzyme in *Eschscholzia californica*, which is involved in alkaloid biosynthesis, and nectarin V in *Nicotiana langsdorffii × N. sanderae*, which acts as a glucose oxidase [21,22]. These plant enzymes as well as THCA synthase are flavoenzymes having a covalently attached FAD molecule. Furthermore, motif analysis (http://myhits.isb-sib.ch/cgi-bin/motif_scan/) suggested that CBDA synthase is also a flavinylated enzyme; the sequence (Arg-Ser-Gly-Gly-His), which is characteristic of FAD-binding sites in the above flavoenzymes [12,21,22], was confirmed in CBDA synthase (Fig. 2). It is interesting to note that flavoenzymes having apparently different catalytic functions show high sequence homology to each other.

### 3.2. Heterologous expression of CBDA synthase in insect cell cultures

We attempted the heterologous expression of CBDA synthase using a baculovirus-insect cell expression system. The enzyme expression was induced by infecting the insect cells with a baculovirus harboring *CBDAS*. After 3 days of incubation, the CBDA synthase activity in the cell extract and in the culture medium was measured. As a result, activity was observed only in the culture medium, indicating that most of the enzyme was secreted from insect cells. The recombinant CBDA synthase in the insect culture medium was readily purified to a homogeneous protein with a molecular mass of ~62 kDa by hydroxylapatite column chromatography (Fig. 3A). The N-terminal amino acid sequence of the recombinant enzyme was identical to that of the native enzyme, suggesting that the signal peptide was correctly cleaved in insect cells to sort the mature polypeptide into the secretory pathway. These results implied that CBDA synthase is a secreted biosynthetic enzyme as in the case of THCA synthase that operates in the apoplastic space of the glandular trichome of *C. sativa* [23].

Because the recombinant enzyme was catalytically active, it is evident that *CBDAS* encodes an active CBDA synthase. However, it was quite difficult to perform the kinetic evaluation of the reaction because the recombinant enzyme showed much less activity than the native CBDA synthase, and prolonged incubation was required to detect the synthesized CBDA in a HPLC analysis. The weak catalytic activity of the recombinant enzyme raised the possibility that another gene encoding more active enzyme has not been found. Thus, we made further efforts to clone another candidate gene by using various PCR conditions. However, we could not amplify gene fragments other than *CBDAS* from the cDNA solution. On the other hand, two gene fragments, both of which contain a 1635-bp intronless open reading frame, were obtained when PCRs were performed with genomic DNA as the template. The genes, named *CBDAS2* and *CBDAS3* (GenBank™ Accession Nos. AB292683 and AB292684), encoded 545-amino acid proteins that share 84.6% and 84.0% identity with the primary structure deduced from *CBDAS*, respectively. In spite of the

![Figure 3](#fig-3)

**Fig. 3.** Analyses of flavin attachment to the recombinant CBDA synthase. (A) SDS-PAGE analysis of the purified recombinant CBDA synthase. Lane 1, molecular mass standards; lanes 2 and 3, 5 μg of the enzyme. Protein bands were visualized by Coomassie blue staining (lanes 1 and 2) or transillumination at 366 nm (lane 3). (B) The absorption spectrum of the enzyme (4.2 mg/ml) dissolved in 10 mM sodium phosphate buffer (pH 7.0) containing 3 mM mercaptoethanol. (C) The fluorescence emission spectrum of the enzyme (1 mg/ml) dissolved in 10 mM sodium citrate buffer (pH 4.0). The sample was irradiated at 450 nm.

structural similarity, the partial amino acid sequences of native CBDA synthase were not completely conserved in CBDAS2 and CBDAS3, and further, the recombinant enzymes prepared from these clones did not exhibit CBDA synthase activity (Data not shown). From these results, we concluded that CBDAS is the dominantly expressed cDNA for CBDA synthase, and the weak activity of the recombinant enzyme might be due to incorrect folding resulting from the differences between the insect and plant protein biosynthetic machinery. Further studies are now in progress to prepare a recombinant CBDA synthase with catalytic activity comparable to that of the native enzyme. The effective expression of the enzyme will be helpful in developing a biotechnological production system for CBDA, as we previously reported on artificial THCA production in transgenic tobacco expressing THCA synthase [12].

### 3.3. Coenzyme binding and reaction mechanism of CBDA synthase

The reaction mechanism of CBDA synthase has been unclear because this enzyme requires neither coenzymes nor cofactors for the oxidation of CBGA [11]. However, in the present study, the sequence analysis indicated the possibility that CBDA synthase possesses a flavin that acts as a coenzyme. We thus next attempted to identify the coenzyme that binds to this enzyme by conducting various spectroscopic analyses of the recombinant enzyme.

The concentrated solution of the purified recombinant CBDA synthase gave a yellow coloration and transillumination of the enzyme at 366 nm showed autofluorescence on SDS-PAGE (Fig. 3A). In addition, the enzyme solution showed absorbance maxima at 365 nm and 450 nm (Fig. 3B), and a fluorescence emission maximum at 515 nm (Fig. 3C). Further, trichloroacetic-acid treatment did not release the compound exhibiting autofluorescence from the enzyme. These properties were consistent with those of flavoproteins [24,25], indicating that CBDA synthase has a covalently bound flavin.

The majority of covalently flavinylated enzymes contain FAD, but several enzymes possessing FMN have been identified [26]. We next attempted to identify the flavin that attaches to CBDA synthase as follows. A flavin-bound peptide was prepared by partial hydrolysis of the recombinant enzyme with trypsin and chymotrypsin. Phosphodiesterase treatment of this peptide resulted in a ~1.5-fold increase of fluorescence emission at 515 nm, along with the release of AMP, suggesting that FMN was formed by cleavage of the phosphodiester bond in FAD [25]. These results showed that FAD binds to the enzyme. Based on a comparison of the molar extinction coefficient of the enzyme at 450 nm with that of authentic FAD, the molar ratio of FAD to CBDA synthase was concluded to be 1:1. We also confirmed that, as reported for THCA synthase, the FAD-binding site of CBDA synthase is His-114 in the flavinylation consensus sequence (Arg-Ser-Gly-Gly-His) because a mutant with a site-directed mutation at this position (H114A) did not show spectroscopic characteristics. In addition, the H114A mutant enzyme exhibited no CBDA synthase activity, indicating that the CBDA synthase reaction is dependent on FAD.

We previously reported that THCA synthase requires molecular oxygen to oxidize CBGA and produces hydrogen peroxide as well as THCA demonstrating that THCA synthase is an oxidase like structurally related enzymes such as berberine bridge enzyme [12]. The structural similarity between CBDA synthase and THCA synthase suggests that the CBDA synthase reaction proceeds through a mechanism similar to that of the THCA synthase reaction. Therefore, we evaluated the effect of molecular oxygen on the CBDA synthase reaction using a glucose-glucose oxidase-catalase system, which completely consumes the molecular oxygen in assay solutions [19]. Consequently, the CBDA synthase reaction was completely inhibited by this treatment, confirming that the reaction absolutely depends on molecular oxygen. In addition, the enzyme reaction produced hydrogen peroxide proportional to CBDA.

Based on these biochemical properties of the enzyme, we propose here the reaction mechanism of CBDA synthase (Fig. 4), which is similar to the previously reported mechanism for

![image](https://i.imgur.com/your_image_url.png)

Fig. 4. The reaction mechanisms of CBDA synthase and THCA synthase. Two electrons from the substrate are accepted by enzyme-bound FAD, and then transferred to molecular oxygen to re-oxidize FAD. CBDA and THCA are synthesized from the ionic intermediates via stereoselective cyclization by the enzymes. R, the rest of the FAD molecule. B, the proposed basic residue of the enzyme.

THCA synthase [12] (Fig. 4). The reactions of both enzymes are initiated by the transfer of a hydride ion from the diallylic position of CBGA to the reactive N-5 position of the isoalloxazine ring of FAD [27]. The important difference between the two reactions is seen in the proton transfer step; i.e. CBDA synthase abstracts a proton from the terminal methyl group of CBGA, whereas a proton is removed from a hydroxyl group of the substrate in the THCA synthase reaction. After the protons elimination, the final step of the reaction, stereoselective ring closure to form CBDA and THCA, takes place in the active sites of each cannabinoid synthase. On the other hand, the hydride ion is transferred from the reduced flavin to molecular oxygen, resulting in the formation of hydrogen peroxide and re-activation of the flavin for the next cycle. Because of these functional similarities, only a small number of amino acid residues would determine the product specificity of the two cannabinoid synthases. Recently, we started the X-ray crystallographic analysis of cannabinoid synthases to unequivocally determine the reaction mechanisms, including the amino acid residues regulating the product specificity, at the atomic level. We have already obtained crystals of the recombinant THCA synthase by a hanging drop vapor diffusion method, and reported their preliminary characterization [28]. Subsequent studies should demonstrate the structure-function relationship of the enzyme active sites, and may provide a rational strategy for controlling the oxidocyclization reaction to effectively produce natural and unnatural cannabinoids.

In conclusion, we have characterized a novel cDNA encoding CBDA synthase, and provided evidence for a close relationship between CBDA synthase and THCA synthase, suggesting that CBDA and THCA, the chemotype-determining cannabinoids, are biosynthesized with similar reaction mechanisms in fiber-type and drug-type *Cannabis* plants, respectively.

Acknowledgments: We thank Dr. Yuji Ito for determining amino acid sequences. This research was partially supported by Grants-in-Aid (Nos. 14771317 and 18780089 for F.T. and No. 10672108 for S.M.) from the Ministry of Education, Science, Sports, and Culture of Japan.

### References

[1] Turner, C.E., Elsohly, M.A. and Boeren, E.G. (1980) Constituents of *Cannabis sativa* L. XVII. A review of the natural constituents. J. Nat. Prod. 43, 169–234.

[2] Gaoni, R. and Mechoulam, R. (1964) Isolation, structure, and partial synthesis of an active constituent of hashish. J. Am. Chem. Soc. 86, 1946–1947.

[3] Matsuda, L.A., Lolait, S.J., Brownstein, M.J., Young, A.C. and Bonner, T.I. (1990) Structure of a cannabinoid receptor and functional expression of the cloned cDNA. Nature 346, 561–564.

[4] Munro, S., Thomas, K.L. and Abu-Shaar, M. (1993) Molecular characterization of a peripheral receptor for cannabinoids. Nature 365, 61–65.

[5] Hampson, A.J., Grimaldi, M., Axelrod, J. and Wink, D. (1998) Cannabidiol and Δ⁹-tetrahydrocannabinol are neuroprotective antioxidants. Proc. Natl. Acad. Sci. USA 95, 8268–8273.

[6] Lastres-Becker, I., Molina-Holgado, F., Ramos, J.A., Mechoulam, R. and Fernandez-Ruiz, J. (2005) Cannabinoids provide neuroprotection against 6-hydroxydopamine toxicity in vivo and in vitro: relevance to Parkinson’s disease. Neurobiol. Dis. 19, 96–107.

[7] Pertwee, R.G. (2006) Cannabinoid pharmacology: the first 66 years. Br. J. Pharmacol. 147, S163–S171.

[8] Yamauchi, T., Shoyama, Y., Aramaki, H., Azuma, T. and Nishioka, I. (1967) Tetrahydrocannabinolic acid, a genuine substance of tetrahydrocannabinol. Chem. Pharm. Bull. 15, 1075–1076.

[9] Kimura, M. and Okamoto, K. (1970) Distribution of tetrahydrocannabinolic acid in fresh wild *Cannabis*. Experientia 26, 819–820.

[10] Taura, F., Morimoto, S., Shoyama, Y. and Mechoulam, R. (1995) First direct evidence for the mechanism of Δ¹-tetrahydrocannabinolic acid biosynthesis. J. Am. Chem. Soc. 117, 9766–9767.

[11] Taura, F., Morimoto, S. and Shoyama, Y. (1996) Purification and characterization of cannabidiolic-acid synthase from *Cannabis sativa* L. Biochemical analysis of a novel enzyme that catalyzes the oxidocyclization of cannabigerolic acid to cannabidiolic acid. J. Biol. Chem. 271, 17411–17416.

[12] Sirikantaramas, S., Morimoto, S., Shoyama, Y., Ishikawa, Y., Wada, Y., Shoyama, Y. and Taura, F. (2004) The gene controlling marijuana psychoactivity. Molecular cloning and heterologous expression of Δ¹-tetrahydrocannabinolic acid synthase from *Cannabis sativa* L. J. Biol. Chem. 279, 39767–39774.

[13] Shoyama, Y., Yagi, M., Nishioka, I. and Yamauchi, T. (1975) Biosynthesis of cannabinoid acids. Phytochemistry 14, 2189–2192.

[14] Taura, F., Morimoto, S. and Shoyama, Y. (1995) Cannabinerolic acid, a cannabinoid from *Cannabis sativa*. Phytochemistry 39, 457–458.

[15] Chomczynski, P. and Sacchi, N. (1987) Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction. Anal. Biochem. 162, 156–159.

[16] Frohman, M.A., Dush, M.K. and Martin, G.R. (1988) Rapid production of full-length cDNAs from rare transcripts: amplification using a single gene-specific oligonucleotide primer. Proc. Natl. Acad. Sci. USA 85, 8998–9002.

[17] Barzu, O. and Michelson, S. (1983) Simple and fast purification of *Escherichia coli* adenylate kinase. FEBS Lett. 153, 280–284.

[18] Higuchi, R., Krummel, B. and Saiki, R.K. (1988) A general method of in vitro preparation and specific mutagenesis of DNA fragments: study of protein and DNA interactions. Nucleic Acids Res. 16, 7351–7367.

[19] Fabian, J. (1965) Simple method of anaerobic cultivation, with removal of oxygen by a buffered glucose oxidase-catalase system. J. Bacteriol. 89, 921.

[20] Green, M.J. and Hill, H.A. (1984) Chemistry of dioxygen. Methods Enzymol. 105, 3–22.

[21] Kutchan, T.M. and Dittrich, H. (1995) Characterization and mechanism of the berberine bridge enzyme, a covalently flavinylated oxidase of benzophenanthridine alkaloid biosynthesis in plants. J. Biol. Chem. 270, 24475–24481.

[22] Carter, C.J. and Thornburg, R.W. (2004) Tobacco nectarin V is a flavin-containing berberine bridge enzyme-like protein with glucose oxidase activity. Plant Physiol. 134, 460–469.

[23] Sirikantaramas, S., Taura, F., Tanaka, Y., Ishikawa, Y., Morimoto, S. and Shoyama, Y. (2005) Tetrahydrocannabinolic acid synthase, the enzyme controlling marijuana psychoactivity, is secreted into the storage cavity of the glandular trichomes. Plant Cell Physiol. 46, 1578–1582.

[24] Singer, T.P. and McIntire, W. (1984) Covalent attachment of flavin to flavoproteins: occurrence, assay, and synthesis. Methods Enzymol. 106, 369–378.

[25] Koziol, J. (1971) Fluorometric analyses of riboflavin and its coenzymes. Methods Enzymol. 18, 253–285.

[26] Decker, K.F. (1993) Biosynthesis and function of enzymes with covalently bound flavin. Annu. Rev. Nutr. 13, 17–41.

[27] Singer, T.P. and Edmondson, D.E. (1978) Flavoproteins (overview). Methods Enzymol. 53, 397–418.

[28] Shoyama, Y., Takeuchi, A., Taura, F., Tamada, T., Adachi, M., Kuroki, R., Shoyama, Y. and Morimoto, S. (2005) Crystallization of Δ¹-tetrahydrocannabinolic acid (THCA) synthase from *Cannabis sativa*. Acta Cryst. F 61, 799–801.
