
HER3 and mutant EGFR meet MET

Carlos L Arteaga

Non-small-cell lung cancers with activating mutations in the epidermal growth factor receptor develop resistance to tyrosine kinase inhibitors by recruiting the MET receptor kinase to activate HER3 and the PI3K-Akt cell survival pathway.

Tyrosine kinase inhibitors (TKIs) are a major advance in cancer treatment. ATP-competitive small molecule inhibitors block the catalytic activity of protein tyrosine kinases in cancer cells and thereby interfere with the phosphorylation of cellular substrates. The therapeutic inactivation of essential kinases, however, creates selective pressures for the development of mechanisms of resistance. There are several mechanisms of escape from TKIs, such as amplification of an oncogenic kinase gene, production of a drug-resistant variant of the targeted kinase, substitution of the signaling function of the targeted kinase(s) by upregulation of compensatory pathways, and overexpression of transporters involved in drug efflux, among others. In a recent study, Engelman *et al.* report that non-small-cell lung cancers (NSCLC) overcome inhibition of the epidermal growth factor receptor (EGFR) tyrosine kinase by amplifying the *MET* oncogene to maintain Akt signaling.

Acquired resistance to TKIs has been observed in several types of cancer. For example, chronic myelogenous leukemia (CML) develops resistance to the Abl kinase inhibitor imatinib by acquiring point mutations in the Abl tyrosine kinase domain. One of these mutations in Abl impairs the ability of the kinase to adopt a conformation to which imatinib binds. Similar mutations in the kinase domains of the receptor kinases Kit and platelet-derived growth factor receptor-α have been shown to weaken the interaction of ATP-mimetics like imatinib, also resulting in drug resistance.

Activating somatic mutations in the EGFR gene have been identified in a cohort of NSCLC. These 'gain-of-function' mutations are either substitutions or short, in-frame deletions or insertions clustered around the region encoding the ATP-binding pocket of the receptor's tyrosine kinase domain. Mutant EGFR phosphorylates the kinase-impaired receptor HER3 (also known as Erb3), which then potently activates the phosphatidylinositol-3 kinase (PI3K) and Akt cell survival pathways (Fig. 1). The majority of individuals with such tumors initially show a good clinical response to EGFR TKIs such as gefitinib, but they almost invariably develop drug resistance after an average of 12 months. A secondary mutation in exon 20 (T790M) is found in about half of patients with a primary mutation that have grown resistant after an initial response to EGFR TKIs. More recently, a new secondary mutation (D761Y) was found in a brain metastasis of a lung cancer initially responsive to gefitinib. So far, T790M and D761Y are the only acquired alterations identified in the EGFR associated with escape from TKI therapy.

Engelman *et al.* isolated gefitinib-resistant clones from HCC827 lung cancer cells which harbor a drug-sensitive activating mutation in the EGFR. The selected resistant cells maintained activation of HER3 and of PI3K and Akt in the presence of gefitinib (Fig. 1). In search of the cause, the authors found that the resistant cells had amplified the *MET* oncogene, which encodes a transmembrane tyrosine kinase receptor that is activated by hepatocyte growth factor (also called scatter factor). *MET* gene amplification or activating mutations are found across a spectrum of primary human cancers. Treatment of the resistant cells with a TKI specific for MET or gefitinib alone did not inhibit cell viability or affect HER3 and Akt phosphorylation. Strikingly, the combination of both drugs stalled resistant cell growth and inhibited HER3 and Akt phosphorylation. Expression of HER3-specific short hairpin RNAs knocked down HER3 protein levels, decreasing active Akt and cell viability. This result supports a central role for HER3 in the survival of drug-resistant cells. What's more, these data imply that the resistant cells had adapted to drug-mediated EGFR inhibition by recruiting compensatory inputs to activate HER3 and PI3K-Akt.

To evaluate the clinical relevance of this cell-culture finding, Engelman *et al.* examined *MET* copy number in EGFR mutant lung cancers that had initially responded to an TKI therapy but had then grown resistant. *MET* gene amplification was detected in 4 of 18 (22%) resistant lung cancers. Two tumors showed *MET* amplification in the resistant but not in the pretherapy specimens, indicating that therapeutic pressure may have 'induced' this alteration or selected a minority of tumor cells harboring it. Taken together, these data represent not only the first report of a kinase other than a receptor of the EGFR family that activates HER3, but also the first genetic alteration not involving the EGFR that is potentially associated with acquired resistance to EGFR TKIs in humans.

As the authors propose, the relevance of *MET* as a mechanism of escape of NSCLC from anti-EGFR therapy needs to be validated in prospective trials of *MET* inhibitors in combination with EGFR TKIs. Nevertheless, the acquired amplification of *MET* in resistant NSCLC and its clear ability to activate the HER3-PI3K-Akt axis in gefitinib-resistant cells lead one to anticipate that this inconvenient encounter of mutant EGFR, HER3 and *MET* will be clinically important.

This study underscores the role of the PI3K-Akt survival pathway as a preferential mechanism of drug resistance to EGFR inhibitors. It also confirms the importance of HER3 as a therapeutic target in NSCLC. Indeed, downregulation of HER3 signaling has been shown to be required for the antitumor effect of EGFR TKIs. Conversely, persistence of HER3 phosphorylation and its association with PI3K in cells treated with an EGFR TKI are both markers of drug resistance. For example, the gefitinib-resistant EGFR T790M mutant continues to activate HER3 and PI3K-Akt in the presence of the TKI. It is possible that inhibiting PI3K or HER3 alone would be sufficient to nullify *MET*-induced resistance to gefitinib in

Carlos L. Arteaga is in the Departments of Medicine and Cancer Biology and the Breast Cancer Research Program, Vanderbilt-Ingram Comprehensive Cancer Center; Vanderbilt University School of Medicine, Nashville, Tennessee 37232, USA.
e-mail: carlos.arteaga@vanderbilt.edu

NEWS AND VIEWS

NSCLC. This question will require further investigation.

In another recent study, Engelman *et al.* selected for gefitinib resistance in a different lung cancer cell line (H3255 cells). These cells, which have a drug-sensitive EGFR allele, acquired a secondary T790M mutation in a small fraction of amplified receptor alleles¹². In a similar study with a cell line that had been derived from an individual with CML, Ricci *et al.* selected for resistance against the Abl kinase inhibitor imatinib¹³. In a fraction of the amplified Abl alleles, the resistant cells had the ‘gatekeeper’ T315I mutation¹³, also found in individuals with imatinib-resistant CML², that prevents inhibitor binding. The discovery of T315I, T790M and MET amplification in cancer cells in culture indicates that a broader use of tumor cell lines that depend highly on activated oncogenes may be profitable. Targeting those pathogenic oncogenes *ex vivo* should allow further discovery of clinically-relevant mechanisms of resistance to therapy.

The success of this approach with CML and NSCLC cells in culture also indicates that, for oncogene-reliant cells, the tumor microenvironment may not influence the development of drug-resistance mechanisms. On the other hand, the independent presence of two resistance mechanisms, EGFR-T790M and MET amplification, in tumors at different metastatic sites in a single subject in this study¹ and the apparent selection against EGFR-T790M in TKI-resistant brain metastases⁹ would imply organ-specific, tumor-microenvironmental, or pharmacological influences on mechanisms of drug resistance in some cases.

One final insight from the study by Engelman *et al.*¹ is the reminder that new and targetable genetic alterations can be detected in tumors that are rebiopsied at the time of drug resistance. Of note, in only eight subjects were paired tumor specimens—that is, from before treatment and after the development of resistance—available. This number is surprisingly low and indicates considerable room for improvement. For example, it is known that approximately 10% of NSCLC harbor EGFR mutations and that close to 70% of those cancers will respond clinically to EGFR TKIs, though invariably these cancers will develop resistance to these drugs⁵. Resistance-associated mutations have been detected in specimens from drug-resistant NSCLC, CML and gastrointestinal stromal tumors, thus confirming the obvious: that under therapeutic pressure, tumors modify those same drug targets or signaling programs essential for their initial pathogenesis.


![Figure 1](#fig1)

**Figure 1** Combined inhibition of mutant EGFR and MET reverses resistance to the EGFR TKI gefitinib. (a) Activated EGFR associates with and phosphorylates the kinase-impaired HER3 receptor. Phosphorylated HER3 binds the regulatory subunit of PI3K (p85) leading to activation of PI3K (p110α) and its downstream kinase Akt, essential to cell growth and survival. The EGFR kinase inhibitor gefitinib prevents phosphorylation of both EGFR and HER3 and, as a result, the PI3K-Akt pathway is uncoupled from HER3 and inactivated. (b) In gefitinib-resistant cells, Engelman *et al.*¹ found that the oncogenic receptor tyrosine kinase MET could also phosphorylate HER3, leading to activation of the PI3K-Akt pathway. Pharmacological inhibition or either EGFR or MET alone was insufficient to inactivate the HER3-PI3K-Akt axis or stall resistant cell growth. (c) The combination of inhibitors can prevent the phosphorylation of HER3 and the activation of Akt in resistant cells, reducing cell viability.

Finally, for many of those cancers showing a resistance-associated alteration, investigational drugs are becoming available.

These observations beg the following question: is it possible that this low number of paired specimens in the study discussed here reflects wide unavailability of drug-resistant cancer specimens that can and should be used now to discover other genetic alterations that cause drug resistance? I suspect the sad answer to this question is yes. If so, what is preventing us from having a wider resource of such tumor tissues that should allow the open-ended discovery of targetable mechanisms of resistance

to treatment? The study by Engelman *et al.*<sup>1</sup> reminds us one more time that we should, when medically possible, rebiopsy every single tumor that becomes resistant to a molecularly targeted drug in order to find compensatory mechanisms of escape that can be eliminated therapeutically.

COMPETING INTERESTS STATEMENT

The author declares no competing financial interests.

1.  Engelman, J.A. *et al.* Science; published online 26 April 2007 (doi:10.1126/science.1141478).
2.  Shah, N.P. *et al.* Cancer Cell **2**, 117–125 (2002).
3.  Daub, H., Specht, K. & Ullrich, A. Nat. Rev. Drug

4.  Sharma, S.V., Bell, D.W., Settleman, J. & Haber, D.A. Nat. Rev. Cancer **7**, 169–181 (2007).
5.  Riely, G.J. *et al.* Clin. Cancer Res. **12**, 839–844 (2006).
6.  Pao, W. *et al.* PLoS Med. **2**, e73 (2005).
7.  Kobayashi, S. *et al.* N. Engl. J. Med. **352**, 786–792 (2005).
8.  Kosaka, T. *et al.* Clin. Cancer Res. **12**, 5764–5769 (2006).
9.  Balak, M.N. *et al.* Clin. Cancer Res. **12**, 6494–6501 (2006).
10. Boccaccio, C. & Comoglio, P.M. Nat. Rev. Cancer **6**, 637–645 (2006).
11. Engelman, J.A. *et al.* Proc. Natl. Acad. Sci. USA **102**, 3788–3793 (2005).
12. Engelman, J.A. *et al.* J. Clin. Invest. **116**, 2695–2706 (2006).
13. Ricci, C. *et al.* Cancer Res. **62**, 5995–5998 (2002).

NEWS AND VIEWS

NATURE MEDICINE | VOLUME 13 | NUMBER 6 | JUNE 2007 | 677
