
Full text provided by www.sciencedirect.com

ELSEVIER

# Maturation and degradation of RNA in bacteria

Ciarán Condon

RNA decay plays an important role, not only in recycling nucleotides but also in determining the rapidity with which cells can react to changing growth conditions. The degradation process can be regulated, thus providing an often-underestimated means of controlling gene expression. Recent developments in the field of RNA maturation and decay in two key model organisms, *Escherichia coli* and *Bacillus subtilis*, include the resolution of the structures of many of the participants in these processes in *E. coli* and the identification of an enzyme in *B. subtilis* that appears to fit the bill as a major player in RNA decay in this organism.

**Addresses**

CNRS UPR 9073 (affiliated with Université de Paris 7 — Denis Diderot), Institut de Biologie Physico-Chimique, 13 rue Pierre et Marie Curie, 75005 Paris, France

Corresponding author: Condon, Ciarán ([condon@ibpc.fr](mailto:condon@ibpc.fr))

---

**Current Opinion in Microbiology 2007, 10:271–278**

This review comes from a themed issue on RNA techniques  
Edited by Renée Schroeder and Emmanuelle Charpentier  

Available online 7th June 2007  
1369-5274/$ – see front matter  
© 2007 Elsevier Ltd. All rights reserved.  
DOI [10.1016/j.mib.2007.05.008](https://doi.org/10.1016/j.mib.2007.05.008)

## Introduction

The degradation of RNA was long considered a constitutive process that only came into play at the end of the usefulness of an mRNA for translation. That perception has changed over time, however, and it is now well established that degradation of RNA can play an important role in the control of gene expression. Bacterial RNA degradation has been best studied in *Escherichia coli* and significant headway has been made recently in *Bacillus subtilis*, where the enzymes and pathways appear to be radically different. In this review, the techniques routinely employed in studying RNA maturation and degradation in bacteria are briefly discussed. The principles of RNA maturation and decay in *E. coli* and *B. subtilis* are developed in more detail, placing particular emphasis on discoveries made in the past two years and, where interesting, highlighting the methods used.

## Techniques used in the study of RNA maturation and decay

### Detection and quantification of RNA

The techniques used to study the phenomena of RNA maturation and degradation have not changed much in 30 years. Northern blot analysis still remains the method of choice to detect RNAs *in vivo*. Northern blots have the advantage of providing a visual confirmation of RNA quality and size. Many researchers have switched to real-time RT-PCR analysis to study the fate of a single RNA under many different experimental conditions. Once the effort has been made to optimize the method for a favorite RNA, many samples can be processed rapidly and variations in low-abundance RNAs can be detected more readily than by Northern blot. In bacteria, RNA half-lives are usually measured by quantifying the amount of RNA remaining at different times after addition of rifampicin to cultures to inhibit new initiation events by RNA polymerase. Although this method does have secondary effects on cellular metabolism and may not always give a precise absolute measurement of half-life, it is still a valid method to compare half-lives in two experimental conditions, wild type versus mutant, for example. Global effects of RNase mutations on mRNA half-lives have been successfully analyzed using gene arrays performed on RNAs isolated at different times after rifampicin addition (e.g. [1]).

### Mapping of 5′ and 3′ extremities of RNA

Whereas mapping the 5′ extremity of primary transcripts or processing/degradation intermediates is a straightforward exercise using a primer-extension assay, mapping the 3′ end provides a little more of a challenge. RNase protection assays have given way to both 5′ and 3′ RACE (rapid amplification of cDNA ends) experiments, in which an oligonucleotide is ligated to either the 5′ or 3′ end, and the ligated species is amplified by RT-PCR. The PCR fragment is then cloned and sequenced to determine the 5′ or 3′ extremities. A variation of this technique (5′/3′ RACE), in which RNAs are circularized by ligation, reverse transcribed across the ligated junction and amplified by PCR, permits simultaneous mapping of both 5′ and 3′ ends (e.g. [2]).

### Assay of ribonuclease activity *in vitro*

To be able to associate an activity with a particular enzyme, particularly when identifying new RNases, requires the setting up of processing assays *in vitro*. With the development of protein-tagging techniques, purification of enzymes has become much simpler. In general, assays of purified proteins are performed with radiolabelled substrate RNAs, synthesized by T7 RNA polymerase *in vitro*. The products are then analyzed on polyacrylamide sequencing gels, which give a better overview than, say, filter-binding assays. Fluorescence-based assays have recently become popular for labor-intensive kinetic studies. Some groups have used such an assay for *E. coli* RNase

www.sciencedirect.com                                                                 Current Opinion in Microbiology 2007, 10:271–278

E endonucleolytic activity, where the substrate has a fluorescence tag and quencher on either side of the cleavage site. Cleavage of this substrate by RNase E causes an easily detectable increase in fluorescence that can be measured in real time (e.g. [3]).

# Recent developments in the field of RNA maturation and decay

## Messenger RNA decay under 'normal' growth conditions in *E. coli*

*E. coli* mRNAs are generally degraded by endonucleolytic cleavage, followed by 3′-to-5′ exonucleolytic RNA decay (Figure 1). The principal instigator of this process under equilibrium growth conditions is the essential ribonuclease RNase E, with minor roles attributed to other more specialized endonucleases, such as its paralog RNase G, RNase III, RNase P and, more recently, RNase Z [4]. RNase E does not generally act alone; it has a long unstructured C-terminal tail that serves as a scaffold for the 3′-to-5′ exoribonuclease polynucleotide phosphorylase (PNPase), an ATP-dependent helicase (RhlB) and enolase. This complex, which can also have other proteins [such as poly(A) polymerase] attached in sub-stoichiometric amounts, is called the RNA degradosome and has been found under different guises in many bacteria (below). Although it cleaves mRNAs internally, RNase E is a 5′-end-dependent nuclease. It requires at least four nucleotides of single-stranded RNA at the 5′ end to be able to bind efficiently and it has a significant preference for 5′-monophosphorylated extremities over the 5′-triphosphate ends of primary transcripts. Using an interesting approach, based on the ability to ligate monophosphorylated but not triphosphorylated ends of primary transcripts, Belasco and colleagues (H Celesnik, A Deana, JG Belasco, personal communication) have proposed that a pyrophosphatase acts on mRNAs to enable RNase E to launch its initial attack, perhaps analogous to the decapping step of eukaryotic mRNAs.

The 3′ end of bacterial mRNAs is protected from attack by 3′-to-5′ exonucleases by secondary structure, in most cases a Rho-independent transcription terminator. Endonucleolytic cleavages generate new unprotected 3′ ends that are rapidly degraded by exonucleases. The main exoribonucleases acting on *E. coli* mRNAs are RNase II, PNPase and RNase R. RNase II can only degrade single-stranded RNA and is completely inhibited by secondary structure. Both PNPase and RNase R can get through secondary structure, however, provided a 3′ toehold is available. This toehold often comes in the form of a poly(A) tail, synthesized by poly(A) polymerase (PAP1) with the help of the small RNA-binding protein Hfq. Although a toehold of 7 nt is sufficient for RNase R [5] because of its intrinsic ability to advance processively through secondary structure, PNPase might also require assistance from a helicase for particularly stable structures. Degradation of highly structured repeated extragenic palindromic (REP) elements, for example, can be accomplished by a combination either of RNase R and PAP1 [6], or of degradosome-bound PNPase and RhlB [7]. Multiple rounds of polyadenylation and exonucleolytic degradation might be required before secondary structures are completely degraded.

Several lines of evidence suggest that endonucleolytic cleavage by RNase E provides the signal for polyadenylation of fragments. This could be due to the ability of PAP1 to recognize 5′ monophosphorylated fragments, but could also be due to direct targeting of PAP1 to these fragments through its interaction with the degradosome. This would explain how 3′ fragments containing terminator hairpins or REP sequences protect primary transcripts, but are degraded once these mRNAs are cleaved by RNase E. A gene-array-based approach, combined with 3′ RACE (primed with oligo dT), was recently used to support the idea that Rho-dependent terminator hairpins are preferential sites of polyadenylation [8].

Although it was long assumed that the degradosome-associated helicase was dedicated to assisting PNPase degrade secondary structures, RhlB has also recently been shown to promote RNase E cleavage of transcripts synthesized by T7 RNA polymerase *in vivo* [9]. Translation is uncoupled from transcription of these RNAs because of the rapid elongation rate of T7 RNA polymerase, and RhlB functions perhaps by remodeling the RNA to reveal cleavage sites previously embedded in secondary structures. It will be interesting to see whether this observation also applies to native cellular mRNAs, under conditions of translation repression, for example, where transcription and translation are also uncoupled.

*E. coli* exoribonucleases RNase II and RNase R accumulate fragments in the range 2–5 nt as end products. Another essential enzyme, called oligoribonuclease (Orn), is required to convert these short products to mononucleotides. It has recently been shown that Orn activity is inhibited by the high intracellular levels of 3′-phosphoadenosine-5′-phosphate (pAp) that accumulate in response to lithium stress [10]. Although a similar effect of pAp on the activity of the *Saccharomyces cerevisiae* 5′-to-3′ exoribonuclease Xrn1 has also been observed, the potential regulation of ribonuclease activity by products of intermediate metabolism has remained relatively unexplored.

## Variations on the degradosome theme

Interesting variants of the degradosome have been found in other bacteria, usually by co-immunoprecipitation techniques. In *Streptomyces*, the catalytic domain of RNase E occupies a central position in the protein, with a scaffold domain on either side. Both scaffold domains

Figure 1

(a) E. coli paradigm:
Pyrophosphatase?
↓
ppp
Ribosome
Ribosome
RNaseE
OligoRNase
RNase R
PNPase
RNase II
RNase R
PNPase

(b) B.subtilis models:
(i)
Ribosome
Ribosome
RNase J1/J2
5′-to-3′ exo
(ii)
Ribosome
RNase J1/J2
endo?
PNPase
RNase R
RNase J1/J2
5′-to-3′ exo

Principal mRNA decay pathways in *E. coli* and *B. subtilis*. (a) The *E. coli* paradigm. For a general description, see text. Endonucleolytic cleavage of mRNAs by 5′-monophosphate-dependent RNase E is indicated by scissors symbols. 3′-to-5′ exonucleolytic degradation of the cleaved intermediates by the major exoribonucleases RNase R, PNPase and RNase II is indicated by the ‘Pacman’ symbols. Fragments containing RNA structure, such as terminator hairpins, require polyadenylation before being degraded. Small fragments in the 2–5 nt range are degraded by oligoribonuclease.

(b) *B. subtilis* models. (i) RNAs are degraded/matured from the 5′ end by the 5′-to-3′ exoribonuclease activity of RNase J1/J2. This enzyme is thought to have both exonucleolytic and endonucleolytic activities, indicated by the combined Pacman/scissors symbols. (ii) RNAs are first cleaved by an endoribonuclease, possibly also RNase J1/J2. The newly generated unprotected 3′ ends are degraded by 3′-to-5′ exoribonucleases, primarily PNPase and RNase R, while the newly generated monophosphorylated 5′ ends might be substrates of the 5′-to-3′ exoribonuclease activity of RNase J1/J2.

overexpression of two recently identified inhibitors of RNase E, called RraA and RraB [14].

**Structures of the major players**

Major advances have been made in the past few years in terms of the understanding of the functioning of ribonucleases at the molecular level. The crystal structures of PNPAse, RNase III and oligoribonuclease were solved a number of years ago. The structures of RNase II [15**, 16] and of the N-terminal catalytic domain of RNase E were solved more recently [17**]. The structures of most of the major ribonucleases involved in *E. coli* mRNA decay are therefore now known. The structure of RNase II bound to single-stranded RNA explains the processivity of the enzyme to the 10 nt range, after which it becomes distributive [15**]. An anchor domain is separated by about 10 nt from the catalytic site; once the RNA is too short to be bound by both domains, it is released at each catalytic cycle.

The structure of the catalytic domain of RNase E shows it forms tetramers held together by a pair of Zn atoms, the so-called 'Zn-link' [17**, 18]. The Zn-link is not required for core catalytic activity, however [19]. The structure also provides a molecular explanation for the enzyme's preference for RNA fragments bearing 5'-monophosphate residues over 5'-triphosphorylated or 5'-hydroxylated termini [17**]. A number of specific interactions were observed between the terminal phosphate and residues of a 5'-sensor pocket. A model has been proposed whereby binding of the 5'-monophosphate by the 5'-sensor domain causes the neighboring S1 domain to clamp down on the RNA downstream [17**], increasing the catalytic power of the enzyme [20**]. The crystal structure of an enolase dimer bound to its binding site in the RNase E scaffold region has also been solved at high resolution [21]. The quaternary structures of RNase E and the other components of the core degradosome have inspired the proposal of a theoretical supramolecular degradation complex of >4 MDa (larger than the ribosome) that satisfies all of the available binding sites of the different partners [22**].

**Targeting of mRNA degradation by non-coding RNAs**

The translational regulation of gene expression by noncoding RNAs (ncRNAs) has, in many cases, been shown to lead to degradation of both the target and regulatory RNA. Although the primary effect of ncRNAs is thought to be inhibition of translation initiation [23], subsequent degradation of the target can play an important part in determining the sensitivity of control. Degradation can be initiated by RNase III (through cleavage of the duplex formed between the ncRNA and its target [24]) or by RNase E (in cases in which RNase E cleavage sites become exposed as a result of altered folding of the target or the lack of ribosomes). Hfq, which plays an important role as an RNA chaperone in bringing imperfectly paired target and ncRNAs together, has been proposed to help recruit RNase E to the targets through a direct protein-protein interaction [25**]. A parallel has thus been drawn between this system and the eukaryotic RISC complex involved in RNA interference. Although a classical RISC-mediated silencing phenomenon has not yet been demonstrated in bacteria, it is interesting to note that *Aquifex aeolicus* does have a homolog of argonaute, one of the key players in this phenomenon [26].

**Messenger RNA degradation in *B. subtilis***

Although there were some early indications that mRNA degradation in *B. subtilis* differed somewhat from that in *E. coli*, this point was really driven home upon completion of the sequences of the two genomes. *B. subtilis* lacks homologues of some of the key enzymes of RNA degradation in *E. coli*, namely RNase II, oligoribonuclease and RNase E (and therefore cannot form *E. coli*-type degradosomes). In contrast to *E. coli*, where 3'-to-5' exonucleolytic degradation is thought to be primarily hydrolytic, this is principally a phosphorolytic process in *B. subtilis*. PNPAse has been established one of the primary 3'-to-5' exonucleases in this organism, with RNase R also playing a significant back-up role in degrading decay intermediates ending in secondary structure [27**, 28]. Polyadenylation has also been observed in *B. subtilis* [29], despite the lack of a PAP1 homolog; however, its role in mRNA stability has not yet been addressed.

A characterizing feature of *B. subtilis* mRNA degradation has been the often dramatic effect of 5'-stabilizers in this organism [30**]. Roadblocks such as bound or stalled ribosomes, bound proteins or strong secondary structures near the 5' end of mRNAs have significant positive effects on the stability of the downstream portion (up to several kilobases) of the mRNA, whereas the upstream portion is usually undetectable (Figure 1). RNAs with similar features are unstable in *E. coli*. Whereas these results were best explained by a 5'-to-3' exonucleolytic activity, up to very recently such an activity was not thought to exist in bacteria. It is now known, however, that the essential enzyme RNase J1 (and its paralog J2) of *B. subtilis* has both endonucleolytic and 5'-to-3' exonucleolytic activity [31, 32**]. Thus, exonucleolytic degradation in this organism can occur from either the 5' or 3' extremity, similar to what is observed in eukaryotes. 5'-to-3' exonucleolytic degradation of the *cryIIIA* mRNA leader by RNase J1 as far as a bound 30S ribosomal subunit results in stabilization of the downstream portion of this mRNA. The 5'-to-3' exonucleolytic activity of RNase J1 prefers single-stranded monophosphorylated 5'-extremities over double-stranded or triphosphorylated 5' ends, and therefore could, in theory at least, degrade the products of its own endonucleolytic activity.

Although there are 11 known endoribonucleases in *B. subtilis* (Table 1), it is not yet clear whether mRNA cleavage plays a major role in mRNA degradation in this

Table 1

Comparison of ribonucleases present in *E. coli* and *B. subtilis.*

| Enzyme       | Exoribonucleases | Endoribonucleases |
|--------------|------------------|-------------------|
|              | *E. coli* gene   | *B. subtilis* gene | Homology<sup>a</sup> | Enzyme | *E. coli* gene | *B. subtilis* gene | Homology<sup>a</sup> |
| PNPase       | *pnp*           | *pnpA*            | 52/69%              | RNase III | *rnc*         | *rncS*<sup>b</sup> | 36/60%              |
| RNase R      | *rnr*           | *mr*              | 39/59%              | RNase P  | *mpAB*<sup>b</sup> | *rnpAB*<sup>b</sup> | 36/63%              |
| RNase PH     | *rph*           | *rph*             | 56/73%              | RNase Z  | *rbn/elaC*    | *rnz*<sup>b</sup>  | 47/64%              |
| RNase II     | *rnb*           | –                 |                     | MazF/EndoA | *mazF*        | *ndoA*            | 32/48%              |
| RNase D      | *rnd*           | –                 |                     | ChpBK     | *chpBK*       | –                  |                     |
| RNase T      | *rnt*           | –                 |                     | YoeB      | *yoeB*        | –                  |                     |
| Oligo-RNase  | *orn*<sup>b</sup>| –                 |                     | RNase E   | *rne*<sup>b</sup> | –                  |                     |
| YhaM         | –               | *yhaM*            |                     | RNase G   | *rng*         | –                  |                     |
| RNase J1     | –               | *mjA*<sup>b</sup> |                     | RNase I   | *rna*         | –                  |                     |
| RNase J2     | –               | *mjB*             |                     | RNase LS  | *rnlA*        | –                  |                     |
|              |                  |                   |                     | RNase HI  | *rnhA*        | –                  |                     |
|              |                  |                   |                     | RNase HII | *rnhB*        | *rnhB*             | 48/65%              |
|              |                  |                   |                     | RNase HIII| –             | *rnhC*             |                     |
|              |                  |                   |                     | RNase M5  | –             | *rnmV*             |                     |
|              |                  |                   |                     | RNase Bsn | –             | *yurl*             |                     |
|              |                  |                   |                     | YhcR      | –             | *yhcR*             |                     |
|              |                  |                   |                     | RNase J1  | –             | *rnjA*<sup>c</sup> |                     |
|              |                  |                   |                     | RNase J2  | –             | *rnjB*             |                     |

RNases in common between *E. coli* and *B. subtilis* are in bold font.

<sup>a</sup> Homology is given as % identity and % similarity.

<sup>b</sup> Essential genes.

<sup>c</sup> Although a protein with 46% similarity to *E. coli* RNase HI, YpdQ, exists in *B. subtilis*, it has been shown not to possess RNase H activity.

Stable RNA maturation and decay

Ribosomal RNAs (rRNAs) and transfer RNAs (tRNAs) are generally synthesized as larger precursors that require processing of both their 5′ and 3′ ends to become functional molecules, and there is significant overlap between the enzymes involved in mRNA degradation and stable RNA maturation [41]. As most of the recent progress has been made on tRNA maturation, I focus on this area for the purposes of this review.

The pathways for tRNA maturation are fairly well understood in both *E. coli* and *B. subtilis* (Figure 2). Processing at the 5′ side is catalyzed by the quasi-universal ribozyme RNase P (this enzyme has been found in every organism studied thus far but *A. aeolicus*). Processing at the 3′-extremity is essentially an exonucleolytic process in *E. coli*, catalyzed by any one of a number of redundant 3′-to-5′ exonucleases, including RNase PH, RNase T, RNase D and RNase II, with the first two playing key roles. Processing of the 3′ end of tRNAs in *B. subtilis* can be either endonucleolytic or exonucleolytic, depending on whether the tRNA gene directly encodes the CCA motif present at the 3′-extremity of all mature tRNAs [42]. In contrast to *E. coli*, where all tRNAs have an encoded CCA motif, this sequence is lacking from about one third of *B. subtilis* tRNA genes and must be added post-transcriptionally by tRNA nucleotidyl transferase. Transfer RNAs with an encoded CCA are matured by an exonucleolytic pathway involving redundant enzymes, much like in *E. coli*. One of the main enzymes involved is RNase PH, with minor roles associated with RNase R, PNPase and YhaM. Another key component of this pathway, thought to play a role analogous to RNase T in removing the nucleotide 3′ to the CCA motif, remains to be identified.

tRNAs lacking an encoded CCA motif are primarily matured by a widely conserved and essential endonuclease in *B. subtilis* called RNase Z. RNase Z cleaves these tRNAs just downstream of the discriminator base in most cases. It is inhibited by the presence of a CCA motif, thus preventing futile cycles of CCA removal and addition in the cell. A particular form of RNase Z has been identified in *Thermotoga maritima*, however, that can also cleave tRNAs downstream of the CCA [43]. RNase Z is also present in *E. coli* (also known as RNase BN [44]), where it can provide sufficient tRNA maturation activity for cell survival in the absence of all of the components of the exonucleolytic pathway, despite inhibition of enzyme activity by the CCA motif.

The structures of RNase Z from *E. coli*, *T. maritima* and *B. subtilis* (both free and tRNA-bound forms) have been solved in the past two years [45–47,48]. The enzyme consists of a core dimer of β-lactamase domains, with each monomer having a long flexible arm that extends from the protein to clamp the tRNA into position. The properties

---

**Figure 2**

(a) Exonucleolytic pathway  
(b) Endonucleolytic pathway  

![Diagram](attachment:diagram.png)

Exonucleolytic and endonucleolytic tRNA maturation pathways in *E. coli* and *B. subtilis*. (a) Maturation of the 3′ end of tRNAs by exoribonucleases (Pacman symbols) in *E. coli* and *B. subtilis*. The principle enzymes identified thus far are in bold type. Maturation of the 5′ end by RNase P is indicated by the scissors symbol. (b) Maturation of the 3′ end of tRNAs by the endonucleolytic pathway. Endonucleolytic cleavages by RNase P and RNase Z are indicated by scissors symbols. RNase Z cleaves downstream of the discriminator nucleotide (N) of the precursor tRNA.
of RNase Z have been reviewed in some detail recently [49,50].

Stable RNAs are generally only turned over under stress conditions or when they are incorrectly folded. This latter process has been termed quality control and the principal pathway in *E. coli* is very similar to that observed for mRNAs, that is, polyadenylation of the fragments and degradation by 3′-to-5′ exoribonucleases, principally RNase R and PNPase [41].

### Conclusions and perspectives

Although we have come a long way towards understanding RNA maturation and decay in *E. coli*, this organism clearly has not yet revealed all its secrets; the recent discovery of toxin–antitoxin systems with RNase activity, just to cite one example, came as a surprise to many. It is also evident from the discovery of 5′-to-3′ exoribonuclease activity in *B. subtilis* that we should consider other model organisms before we can claim even a remote understanding of these processes in the bacterial kingdom as a whole. One cannot help feeling that only the tip of the proverbial iceberg has yet been revealed!

### Acknowledgements

I apologize to those whose papers were not cited due to constraints on space and on the number of citations; I arbitrarily chose a cut-off date of 2003/2004. I thank members of my lab for their contributions to our work in this domain. This work was supported by funds from the Centre National de Recherche Scientifique (CNRS), Université Paris VII-Denis Diderot, ACI Jeunes Chercheurs from the Ministère de l’Education Nationale, and the Agence Nationale de la Recherche (ANR).

### References and recommended reading

Papers of particular interest, published within the period of review, have been highlighted as:

- of special interest
- of outstanding interest

1. Hambraeus G, von Wachenfeldt C, Hederstedt L: Genome-wide survey of mRNA half-lives in *Bacillus subtilis* identifies extremely stable mRNAs. *Mol Genet Genomics* 2003, **269**:706-714.
2. Britton RA, Wen T, Schaefer L, Pellegrini O, Uicker WC, Mathy N, Tobin C, Daou R, Szyk J, Condon C: Maturation of the 5′ end of *Bacillus subtilis* 16S rRNA by the essential ribonuclease YkqC/RNase J1. *Mol Microbiol* 2007, **63**:127-138.
3. Redko Y, Tock MR, Adams CJ, Kaberdin VR, Grasby JA, McDowall KJ: Determination of the catalytic parameters of the N-terminal half of *Escherichia coli* ribonuclease E and the identification of critical functional groups in RNA substrates. *J Biol Chem* 2003, **278**:44001-44008.
4. Perwez T, Kushner SR: RNase Z in *Escherichia coli* plays a significant role in mRNA decay. *Mol Microbiol* 2006, **60**:723-737.
5. Vincent HA, Deutscher MP: Substrate recognition and catalysis by the exoribonuclease RNase R. *J Biol Chem* 2006, **281**:29769-29775.
6. Cheng ZF, Deutscher MP: An important role for RNase R in mRNA decay. *Mol Cell* 2005, **17**:313-318. Evidence implicating RNase R in mRNA decay.
7. Khemici V, Carpousis AJ: The RNA degradosome and poly(A) polymerase of *Escherichia coli* are required in vivo for the degradation of small mRNA decay intermediates containing REP-stabilizers. *Mol Microbiol* 2004, **51**:777-790.

First indication that the role of RhlB helicase in the degradosome is not restricted to facilitating PNPase activity.

8. Mohanty BK, Kushner SR: The majority of *Escherichia coli* mRNAs undergo post-transcriptional modification in exponentially growing cells. *Nucleic Acids Res* 2006, **34**:5695-5704.

9. Khemici V, Poljak L, Toesca I, Carpousis AJ: Evidence in vivo that the DEAD-box RNA helicase RhlB facilitates the degradation of ribosome-free mRNA by RNase E. *Proc Natl Acad Sci USA* 2005, **102**:6913-6918.

10. Mechold U, Ogryzko V, Ngo S, Danchin A: Oligoribonuclease is a common downstream target of lithium-induced pAp accumulation in *Escherichia coli* and human cells. *Nucleic Acids Res* 2006, **34**:2364-2373.

11. Purusharth RI, Klein F, Sulthana S, Jager S, Jagannadham MV, Evguenieva-Hackenberg E, Ray MK, Klug G: Exoribonuclease R interacts with endoribonuclease E and an RNA helicase in the psychrotrophic bacterium *Pseudomonas syringae* Lz4W. *J Biol Chem* 2005, **280**:14572-14578. First indication that an RNase other than PNPase can interact with RNase E.

12. Chen C, Deutscher MP: Elevation of RNase R in response to multiple stress conditions. *J Biol Chem* 2005, **280**:34393-34396.

13. Prud'homme-Genereux A, Beran RK, lost I, Ramey CS, Mackie GA, Simons RW: Physical and functional interactions among RNase E, polynucleotide phosphorylase and the cold-shock protein, CsdA: evidence for a ‘cold shock degradasome’. *Mol Microbiol* 2004, **54**:1409-1421.

14. Gao J, Lee K, Zhao M, Qiu J, Zhan X, Saxena A, Moore CJ, Cohen SN, Georgiou G: Differential modulation of *E. coli* mRNA abundance by inhibitory proteins that alter the composition of the degradosome. *Mol Microbiol* 2006, **61**:394-406.

15. Frazao C, McVey CE, Amblar M, Barbas A, Vonrhein C, Arraiano CM, Carrondo MA: Unravelling the dynamics of RNA degradation by ribonuclease II and its RNA-bound complex. *Nature* 2006, **443**:110-114. Resolution of the structure of RNase II bound to RNA.

16. Zuo Y, Vincent HA, Zhang J, Wang Y, Deutscher MP, Malhotra A: Structural basis for processivity and single-strand specificity of RNase II. *Mol Cell* 2006, **24**:149-156.

17. Callaghan AJ, Marcaida MJ, Stead JA, McDowall KJ, Scott WG, Luisi BF: Structure of *Escherichia coli* RNase E catalytic domain and implications for RNA turnover. *Nature* 2005, **437**:1187-1191. Resolution of the structure of the catalytic domain of RNase E bound to RNA.

18. Callaghan AJ, Redko Y, Murphy LM, Grossmann JG, Yates D, Garman E, Ilag LL, Robinson CV, Symmons MF, McDowall KJ *et al.*: Zn-link: a metal-sharing interface that organizes the quaternary structure and catalytic site of the endoribonuclease, RNase E. *Biochemistry* 2005, **44**:4667-4675.

19. Caruthers JM, Feng Y, McKay DB, Cohen SN: Retention of core catalytic functions by a conserved minimal ribonuclease E peptide that lacks the domain required for tetramer formation. *J Biol Chem* 2006, **281**:27046-27051.

20. Jiang X, Belasco JG: Catalytic activation of multimeric RNase E and RNase G by 5′-monophosphorylated RNA. *Proc Natl Acad Sci USA* 2004, **101**:9211-9216. Indication that a monophosphate group at the 5′ extremity of an RNA increases the catalytic power rather than the affinity of RNase E.

21. Chandran V, Luisi BF: Recognition of enolase in the *Escherichia coli* RNA degradosome. *J Mol Biol* 2006, **358**:8-15.

22. Marcaida MJ, DePristo MA, Chandran V, Carpousis AJ, Luisi BF: The RNA degradosome: life in the fast lane of adaptive molecular evolution. *Trends Biochem Sci* 2006, **31**:359-365. Recent review on the RNA degradosome and its variants.

23. Morita T, Mochizuki Y, Aiba H: Translational repression is sufficient for gene silencing by bacterial small noncoding RNAs in the absence of mRNA destruction. *Proc Natl Acad Sci USA* 2006, **103**:4858-4863.
24. Huntzinger E, Boisset S, Saveanu C, Benito Y, Geissmann T, Namane A, Lina G, Etienne J, Ehresmann B, Ehresmann C *et al.*: Staphylococcus aureus RNAIII and the endoribonuclease III coordinately regulate *spa* gene expression. *EMBO J* 2005, 24:824-835.

25. Morita T, Maki K, Aiba H: RNase E-based ribonucleoprotein complexes: mechanical basis of mRNA destabilization mediated by bacterial noncoding RNAs. *Genes Dev* 2005, 19:2176-2186. Evidence that Hfq may recruit RNase E to sites of control by ncRNAs.

26. Yuan YR, Pei Y, Ma JB, Kuryavyi V, Zhadina M, Meister G, Chen HY, Dauter Z, Tuschl T, Patel DJ: Crystal structure of *A. aeolicus* argonaute, a site-specific DNA-guided endoribonuclease, provides insights into RISC-mediated mRNA cleavage. *Mol Cell* 2005, 19:405-419.

27. Deikus G, Babitzke P, Bechhofer DH: Recycling of a regulatory protein by degradation of the RNA to which it binds. *Proc Natl Acad Sci USA* 2004, 101:2747-2751. Shows the key role of PNPase in RNA degradation in *B. subtilis*.

28. Oussenko IA, Abe T, Ujiie H, Muto A, Bechhofer DH: Participation of 3′-to-5′ exoribonucleases in the turnover of *Bacillus subtilis* mRNA. *J Bacteriol* 2005, 187:2758-2767.

29. Campos-Guillen J, Bralley P, Jones GH, Bechhofer DH, Olmedo-Alvarez G: Addition of poly(A) and heteropolymeric 3′ ends in *Bacillus subtilis* wild-type and polynucleotide phosphorylase-deficient strains. *J Bacteriol* 2005, 187:4698-4706.

30. Sharp JS, Bechhofer DH: Effect of 5′-proximal elements on decay of a model mRNA in *Bacillus subtilis*. *Mol Microbiol* 2005, 57:484-495. Comprehensive characterization of the effects of a 5′-stabiliser in *B. subtilis*.

31. Even S, Pellegrini O, Zig L, Labas V, Vinh J, Brechemmier-Baey D, Putzer H: Ribonucleases J1 and J2: two novel endoribonucleases in *B. subtilis* with functional homology to *E. coli* RNase E. *Nucleic Acids Res* 2005, 33:2141-2152.

32. Mathy N, Bénard L, Pellegrini O, Daou R, Wen T, Condon C: 5′-to-3′ exoribonuclease activity in bacteria: role of RNase J1 in rRNA maturation and 5′ stability of mRNA. *Cell* 2007, 129:681-692. First demonstration of 5′-to-3′ exoribonuclease activity in bacteria.

33. Gerdes K, Christensen SK, Lobner-Olesen A: Prokaryotic toxin–antitoxin stress response loci. *Nat Rev Microbiol* 2005, 3:371-382. Comprehensive review on toxin–antitoxin systems in bacteria.

34. Buts L, Lah J, Dao-Thi MH, Wyns L, Loris R: Toxin–antitoxin modules as bacterial metabolic stress managers. *Trends Biochem Sci* 2005, 30:672-679.

35. Condon C: Shutdown decay of mRNA. *Mol Microbiol* 2006, 61:573-583.

36. Pellegrini O, Mathy N, Gogos A, Shapiro L, Condon C: The *Bacillus subtilis* ydcDE operon encodes an endoribonuclease of the MazF/PemK family and its inhibitor. *Mol Microbiol* 2005, 56:1139-1148.

37. Kamada K, Hanaoka F: Conformational change in the catalytic site of the ribonuclease YoeB toxin by YefM antitoxin. *Mol Cell* 2005, 19:497-509.

38. Zhang Y, Zhu L, Zhang J, Inouye M: Characterization of ChpBK, an mRNA interferase from *Escherichia coli*. *J Biol Chem* 2005, 280:26080-26088.

39. Odaert B, Saida F, Aliprandi P, Durand S, Crechet JB, Guerois R, Laalami S, Uzan M, Bontems F: Structural and functional studies of RegB, a new member of a family of sequence-specific ribonucleases involved in mRNA inactivation on the ribosome. *J Biol Chem* 2006, 282:2019-2028.

40. Zhang Y, Zhang J, Hara H, Kato I, Inouye M: Insights into the mRNA cleavage mechanism by MazF, an mRNA interferase. *J Biol Chem* 2005, 280:3143-3150.

41. Deutscher MP: Degradation of RNA in bacteria: comparison of mRNA and stable RNA. *Nucleic Acids Res* 2006, 34:659-666. Review highlighting the extensive overlap between stable RNA and mRNA maturation and degradation.

42. Wen T, Oussenko IA, Pellegrini O, Bechhofer DH, Condon C: Ribonuclease PH plays a major role in the exonucleolytic maturation of CCA-containing tRNA precursors in *Bacillus subtilis*. *Nucleic Acids Res* 2005, 33:3636-3643.

43. Minagawa A, Takaku H, Takagi M, Nashimoto M: A novel endonucleolytic mechanism to generate the CCA 3′ termini of tRNA molecules in *Thermotoga maritima*. *J Biol Chem* 2004, 279:15688-15697.

44. Ezraty B, Dahlgren B, Deutscher MP: The RNase Z homologue encoded by *Escherichia coli* elaC gene is RNase BN. *J Biol Chem* 2005, 280:16542-16545.

45. Li de la Sierra-Gallay I, Pellegrini O, Condon C: Structural basis for substrate binding, cleavage and allostery in the tRNA maturase RNase Z. *Nature* 2005, 433:657-661.

46. Ishii R, Minagawa A, Takaku H, Takagi M, Nashimoto M, Yokoyama S: Crystal structure of the tRNA 3′ processing endoribonuclease tRNase Z from *Thermotoga maritima*. *J Biol Chem* 2005, 280:14138-14144.

47. Kostelecky B, Pohl E, Vogel A, Schilling O, Meyer-Klaucke W: The crystal structure of the zinc phosphodiesterase from *Escherichia coli* provides insight into function and cooperativity of tRNase Z-family proteins. *J Bacteriol* 2006, 188:1607-1614.

48. Li de la Sierra-Gallay I, Mathy N, Pellegrini O, Condon C: Structure of the ubiquitous 3′ processing enzyme RNase Z bound to transfer RNA. *Nat Struct Mol Biol* 2006, 13:376-377. Structure of the tRNA 3′ maturase RNase Z bound to tRNA.

49. Vogel A, Schilling O, Spath B, Marchfelder A: The tRNase Z family of proteins: physiological functions, substrate specificity and structural properties. *Biol Chem* 2005, 386:1253-1264.

50. Redko Y, Li de la Sierra-Gallay I, Condon C: When all’s zed and done: the structure and function of RNase Z in prokaryotes. *Nat Rev Microbiol* 2007, 5:278-286. Recent comprehensive review on RNase Z in prokaryotes.
