
Rv1106c from *Mycobacterium tuberculosis* Is a 3β-Hydroxysteroid Dehydrogenase†

Xinxin Yang,‡ Eugenie Dubnau,$^{§}$ Issar Smith,$^{§}$ and Nicole S. Sampson*,‡

Department of Chemistry, Stony Brook University, Stony Brook, New York 11794-3400, and Public Health Research Institute Center, New Jersey Medical School, University of Medicine and Dentistry of New Jersey, 225 Warren Street, Newark, New Jersey 07103

Received April 12, 2007; Revised Manuscript Received May 16, 2007

ABSTRACT: New approaches are required to combat *Mycobacterium tuberculosis* (*Mtb*), especially the multi-drug resistant and extremely drug resistant organisms (MDR-TB and XDR-TB). There are many reports that mycobacteria oxidize 3β-hydroxysterols to 3-ketosteroids, but the enzymes responsible for this activity have not been identified in mycobacterial species. In this work, the *Rv1106c* gene that is annotated as a 3β-hydroxysteroid dehydrogenase in *Mtb* has been cloned and heterologously expressed. The purified enzyme was kinetically characterized and found to have a pH optimum between 8.5 and 9.5. The enzyme, which is a member of the short chain dehydrogenase superfamily, uses NAD⁺ as a cofactor and oxidizes cholesterol, pregnenolone, and dehydroepiandrosterone to their respective 3-keto-4-ene products. The enzyme forms a ternary complex with NAD⁺ binding before the sterol. The enzyme shows no substrate preference for dehydroepiandrosterone versus pregnenolone with second-order rate constants (*k*~cat~/K~m~) of 3.2 ± 0.4 and 3.9 ± 0.9 μM⁻¹ min⁻¹, respectively, at pH 8.5, 150 mM NaCl, 30 mM MgCl₂, and saturating NAD⁺. Trilostane is a competitive inhibitor of dehydroepiandrosterone with a *K*~i~ of 197 ± 8 μM. The expression of the 3β-hydroxysteroid dehydrogenase in *Mtb* is intracellular. Disruption of the 3β-hydroxysteroid dehydrogenase gene in *Mtb* abrogates mycobacterial cholesterol oxidation activity. These data are consistent with the *Rv1106c* gene being the one responsible for 3β-hydroxysterol oxidation in *Mtb*.

Tuberculosis is an opportunistic infection caused by *Mycobacterium tuberculosis* (*Mtb*)¹ in individuals with HIV-AIDS that is estimated to infect 30% of the world’s population (1, 2). The World Health Organization estimates that 2 million people die every year from tuberculosis. Drug resistance to front-line *Mtb* drugs rifampicin and isoniazid has emerged (3, 4), and additional resistance to second-line drugs is emerging (5, 6). It is clear that new approaches are required to combat these multi-drug resistant and extremely (or extensively) drug resistant organisms (7–9).

The complete genome sequences of microorganisms are a rich source for mining new drug targets. However, oftentimes, biochemical functions have been assigned to genes purely on the basis of the homology of their sequence to the sequences of gene products which are themselves poorly characterized. One such gene is *Mtb* gene *Rv1106c*. This gene encodes a putative cholesterol (3β-hydroxysterol) dehydrogenase (10) that is also found in the genomes of related mycobacteria (11, 12). Many actinomycetes and pseudomonads can utilize sterols as a sole carbon source, and 3β-hydroxysteroid oxidation is the first step in this catabolic or degradative pathway (13–15). However, unlike other actinomycetes, *Mtb* cannot survive on sterols alone (16); it will grow if supplemented with asparagine, citrate, and Tyloxapol (15). Moreover, reports that mycobacteria synthesize cholesterol (17) have not been substantiated (18). Thus, it is intriguing to consider other possible roles for this enzymatic function in the infectious life cycle of mycobacteria.

*Rv1106c* is annotated as a cholesterol dehydrogenase because it is 74% identical with the *Nocardia* sp. cholesterol dehydrogenase. Cholesterol dehydrogenase is a NAD⁺- or NADP⁺-dependent enzyme and catalyzes two sequential reactions: oxidation of cholesterol (3β-hydroxysterol) to cholest-5-en-3-one and isomerization of cholest-5-en-3-one to cholest-4-en-3-one. The expression of *Nocardia* cholesterol dehydrogenase is reported to be transcriptionally activated by the addition of cholesterol to culture medium (19, 20). The *Mycobacterium marinum* (pathogen that causes fish and amphibian tuberculosis) homologue is preferentially expressed in the frog granuloma (21). Thus, it appears that transcription and expression may be cholesterol-dependent and may require the host to provide the sterol activator.

Gene database analysis has revealed a superfamily of proteins that in addition to the bacterial cholesterol dehy-

---

† Financial support from the National Institutes of Health [Grants AI065251 (N.S.S.), AI065987 (I.S.), and RR021008 (N.S.S.) and NIAID Contract HHSN266200400091C (Colorado State University)] and the National Science Foundation (Grant CHE0131146 for NMR spectrometers) is gratefully acknowledged.

* To whom correspondence should be addressed: Stony Brook University, Stony Brook, NY 11794-3400. Phone: (631) 632-7952. Fax: (631) 632-5731. E-mail: nicole.sampson@stonybrook.edu.

‡ Stony Brook University.

$^{§}$ University of Medicine and Dentistry of New Jersey.

¹ Abbreviations: *Mtb*, *Mycobacterium tuberculosis*; MDR-TB, multi-drug resistant *Mtb*; XDR-TB, extremely drug resistant *Mtb*; DHEA, dehydroepiandrosterone; rH₆3BHSD, recombinant 3β-hydroxysteroid dehydrogenase with an N-terminal six-histidine tag; r3BHSD, recombinant 3β-hydroxysteroid dehydrogenase without an N-terminal six-histidine tag; IMAC, immobilized metal ion affinity chromatography; TAPS, *N*-tris(hydroxymethyl)methyl-3-aminopropanesulfonic acid.

Mtb Sterol Dehydrogenase

Biochemistry, Vol. 46, No. 31, 2007 9059

FIGURE 1: Unrooted phylogenetic tree for 3β-hydroxysteroid dehydrogenase encoded by *Rv1106c* and related proteins. The values are relative evolutionary distance. The tree was generated using a ClustalW 1.82 alignment and CLC Free Workbench 3. Proteins are identified by species, enzyme, and Uniprot entry.

drogenase includes mammalian 3β-hydroxysteroid dehydrogenases, plant dihydroflavonol reductases, bacterial UDP-glucose-4-epimerases, and viral 3β-hydroxysteroid dehydrogenases (22, 23) (Figure 1). There appears to be an ancient evolutionary relationship between plant flavonol synthesis and mammalian steroid hormone synthesis, and in fact, many flavonols are agonists or antagonists of mammalian steroid hormone receptors (24). The viral 3β-hydroxysteroid dehydrogenases are genetically closest to the mammalian 3β-hydroxysteroid dehydrogenases and the vaccinia virus dehydrogenase functions in viral steroid hormone synthesis. A mutant strain of vaccinia virus carrying a knockout deletion of the vaccinia gene encoding 3β-hydroxysteroid dehydrogenase is attenuated in mice (25). Unlike the wild-type virus, the mutant does not induce the formation of corticosterone, a suppressor of the host inflammatory response to infection. The bacterial 3β-hydroxysteroid dehydrogenases are more distantly related to the mammalian dehydrogenases than their viral counterparts (Figure 1); however, they may serve a function similar to that of the viral enzymes, that is, steroid hormone synthesis, or they may be involved in bacterial flavonol synthesis. Both activities could lead to immune suppression. The ability of actinomycetes such as *Mtb* or *Nocardia* to persist in the intracellular milieu requires mechanisms for abrogating the normal host immune response, and targeting the glucocorticoid receptors may be one such mechanism.

The similarities among the enzymes of this superfamily hint at the potential catalytic properties of *Rv1106c* from *Mtb*. However, the physicochemical characteristics of this putative 3β-hydroxysteroid dehydrogenase from *Mtb* have not been defined, and its real enzymatic function has not yet been confirmed. Before assigning physiological function, we undertook a fundamental characterization of the chemistry catalyzed by this putative 3β-hydroxysteroid dehydrogenase encoded by *Rv1106c*, and the results of those studies are presented here.

### EXPERIMENTAL PROCEDURES

#### Materials and General Methods. Cholesterol, cholest-4-en-3-one, cholest-5-en-3-one, NAD⁺, NADH, 5-pregnen-3β-ol-20-one, and trilostane were purchased from Sigma-Aldrich (St. Louis, MO). Triton X-100 and progesterone were from Aldrich Fine Chemical Co. (Milwaukee, WI). NADP⁺ was from Research Products International Co. (Mount Prospect, IL). Restriction endonucleases and T4 DNA ligase were from New England Biolabs (Beverly, MA). Oligonucleotides were

Expression and Purification of the Rv1106c Gene Product in Escherichia coli. BL21(DE3)pLysS transformed with pET28b-1106c or pET29a-1106c was cultured overnight in LB/kanamycin medium. Starter cultures were used to inoculate 2×YT medium (30 μg/mL kanamycin) at a ratio of 1:100 for ~3 h until the OD<sub>600</sub> reached 0.6–0.8. Isopropyl β-D-thiogalactoside (IPTG) was added at a final concentration of 0.4 mM to induce protein expression. Cultures were grown at 25 °C for an additional 8 h or at 18 °C for an additional 20 h. The cells were collected by centrifugation at 5000g for 30 min. The cell pellet (5 g) was suspended in 30 mL of buffer A and lysed with a French press at 10 000 psi. All subsequent steps were conducted at 4 °C. Cell debris was removed by centrifugation at 135000g for 3 h. The supernatant was precipitated with ammonium sulfate, and the 5–10% (w/v) fraction was collected. This pellet was suspended in buffer A (5 mL) and dialyzed (NMWCO 6000–8000) against buffer A. The dialysate was loaded onto a column of DEAE-cellulose (30 mm × 25 cm, DE-52) equilibrated with buffer A. Fractions were collected by gradient elution (from 0.1 to 0.4 M NaCl), and 10 mL fractions were collected. Fractions with activity were combined, concentrated, and desalted by ultrafiltration (NMWCO 10 000) to a final volume of 5 mL. This active fraction was then applied to Q-Sepharose (10 mL bed volume) equilibrated in buffer B, and the flow-through was collected and concentrated to a final volume of 2 mL. The concentrate was loaded on a Sephacryl S-200 high-resolution column (20 mm × 120 cm) and eluted with buffer B. Pure fractions containing dehydrogenase activity were collected and concentrated.

The identity of the purified protein was confirmed by tryptic digestion and MALDI mass fingerprinting. Dehydrogenase (~5 μg in gel) was digested with trypsin (400 ng) in 20 μL of 50 mM NH<sub>4</sub>HCO<sub>3</sub> for 17 h. The tryptic fragments were extracted with a 60% CH<sub>3</sub>CN/H<sub>2</sub>O mixture and 0.1% TFA (50 μL) three times. The combined extracts were dried and redissolved in 5 μL of a 0.1% TFA/H<sub>2</sub>O mixture. MALDI-TOF mass spectra were acquired in positive ion mode using a saturated solution of α-cyano-4-hydroxycinnamic acid as the matrix.

Activity Assays. Initially, the 3β-hydroxysteroid dehydrogenase was assayed at 30 °C for less than 10% of the reaction with monitoring of NADH absorbance at 340 nm in buffer C (pregnenolone, DHEA, quercetin, or dihydroquercetin as the substrate) or buffer D (cholesterol as the substrate). Dehydroepiandrosterone, pregnenolone, quercetin, or dihydroquercetin stock solutions were prepared in ethanol (3 mM), and cholesterol stocks were prepared in propan-2-ol. The final volume of alcohol was kept constant in all assays and was 5%. Reactions were initiated by the addition of dehydrogenase. Later activity measurements were taken as described below for enzyme kinetics.

Product Analysis by TLC and HPLC Spectroscopy. Enzyme assay solutions (1 mL) at various extents of reaction were extracted with an equal volume of ethyl acetate five times. The ethyl acetate extracts were combined and dried under a stream of N<sub>2</sub>. Then the dried residue was dissolved in 50 μL of ethyl acetate, applied to silica gel thin layer chromatography plates, and separated using a 4:1 hexane/ethyl acetate mixture or a 5:1 dichloromethane/ethyl acetate mixture. To improve resolution, the plate was developed three times, drying the plate between each run. Steroid products were visualized by UV and staining with phosphomolybdic acid in an ethanolic solution (10%, w/v). Authentic standards were run on the same plate and cospotted with the reaction mixtures.

Samples for HPLC analyses (27) were analyzed on a Shimadzu Corp. (Columbia, MD) system comprised of an SCL-10A VP system controller, two LC-10AD solvent pumps, and a model SPA-20A Prominence UV-vis spectrometer. The following conditions were used: stationary phase, Microsorb-MV C-18 column (Rainin Instrument Corp., Woburn, MA; 5 μm, 100 Å, 4.6 mm × 250 mm); mobile phase, solvent A, CH<sub>3</sub>CN/H<sub>2</sub>O (90:10, v/v), and solvent B, CH<sub>3</sub>CN/propan-2-ol (85:15, v/v). Samples were eluted for 25 min isocratically with solvent A, followed by a 10 min linear gradient to 100% solvent B, followed by a 45 min isocratic elution with solvent B; the flow rate was 1.25 mL/min, and peaks were detected at 212 and 240 nm. Sample aliquots (200 μL) were injected directly from enzyme assay solutions incubated at 30 °C that contained 120 μM cholesterol, 5% 2-propanol, 1.4 mM NAD, and 0.41 pM dehydrogenase in buffer D. Calibration curves for cholesterol and cholest-5-en-3-one were prepared by the integration of peak area detected at 212 nm for standard solutions ranging in concentration from 0 to 1500 μM. Calibration curves for cholest-4-en-3-one were prepared analogously by the integration of peak area detected at 212 and 240 nm.

Enzyme Kinetics. The 3β-hydroxysteroid dehydrogenase was assayed at 30 °C for less than 10% of the reaction with monitoring of NADH absorbance at 340 nm in buffer E (pregnenolone or DHEA as the substrate) or buffer F (cholesterol as the substrate). DHEA or pregnenolone stock solutions were prepared in ethanol (3 mM), and cholesterol stocks were prepared in propan-2-ol. No oxidation of ethanol or propanol was detected in control assays. The enzymatic activity was insensitive to alcohol concentrations between 2.5% (v/v) and 5% (v/v), and the final volume of alcohol was kept constant in all assays at 5% (v/v). Reactions were initiated by the addition of dehydrogenase. Initial velocities were measured for varying NAD<sup>+</sup> concentrations (350–22400 μM) at different fixed concentrations of DHEA or pregnenolone (30–300 μM). Initial velocities were measured for varying trilostane concentrations (0–1000 μM) at different fixed concentrations of NAD<sup>+</sup> (88–5600 μM) with DHEA at a concentration of 120 μM or at different fixed concentrations of DHEA (11–240 μM) with NAD<sup>+</sup> at a concentration of 350 μM. All points are the average of triplicate data points.

Initial velocities were globally fit to the following equations using Grafit (version 4.0.10).

Ternary complex Michaelis–Menten equation

\[
\nu = (V_{\mathrm{m}}[A][B])/(K_{\mathrm{ia}}K_{\mathrm{mb}} + K_{\mathrm{mb}}[A] + K_{\mathrm{ma}}[B] + [A][B])
\]

Competitive inhibition equation

\[
\nu = (V_{\mathrm{m}}[S])/[K_{\mathrm{m}}(1 + [I]/K_{\mathrm{ic}}) + [S]]
\]

Uncompetitive inhibition equation

\[
\nu = (V_{\mathrm{m}}[S])/[K_{\mathrm{m}} + [S](1 + [I]/K_{\mathrm{iu}})]
\]

Mtb Sterol Dehydrogenase

Mixed inhibition equation

\[
\nu = (V_{\mathrm{m}}[S]) / [K_{\mathrm{m}}(1 + [I] / K_{\mathrm{ic}}) + [S](1 + [I] / K_{\mathrm{iu}})]
\]

Substrate NAD⁺ inhibition equation

\[
\nu = (V_{\mathrm{m}}[\mathrm{NAD}^{+}]) / (K_{\mathrm{m}} + [\mathrm{NAD}^{+}] + [\mathrm{NAD}^{+}]^{2} / K_{\mathrm{iuNAD}^{+}})
\]

Fixed second-substrate Michaelis–Menten equation

\[
\nu = (V_{\mathrm{m}}[S]) / (K_{\mathrm{m}} + [S])
\]

Rate pH dependence equation

\[
\text{rate} = (\text{rate}_{\mathrm{m}} \times 10^{\mathrm{pH} - \mathrm{p}K_{\mathrm{a}}}) / (10^{\mathrm{pH} - \mathrm{p}K_{\mathrm{a}}} + 1)
\]

where \( K_{\mathrm{ma}} \) and \( K_{\mathrm{mb}} \) are the Michaelis constants for A and B at saturating concentrations of B and A, respectively, \( K_{\mathrm{m}} \) is the apparent Michaelis constant for a varied substrate at a fixed concentration of the second substrate, \( K_{\mathrm{ia}} \) is the dissociation constant for the enzyme and A, \( V_{\mathrm{m}} \) is the maximum velocity, S is the varied substrate, \( K_{\mathrm{ic}} \) and \( K_{\mathrm{iu}} \) are the competitive and uncompetitive inhibition constants, respectively, rate is either \( k_{\mathrm{cat}} \) or \( k_{\mathrm{cat}} / K_{\mathrm{m}} \), and rate\(_{\mathrm{m}}\) is the maximal value of either \( k_{\mathrm{cat}} \) or \( k_{\mathrm{cat}} / K_{\mathrm{m}} \).

The pH dependence of the 3β-hydroxysteroid dehydrogenase-catalyzed reaction was determined in 100 mM TAPS hydrochloride buffer, 150 mM NaCl, 30 mM MgCl₂, and 2.8 mM NAD⁺ at different fixed concentrations of DHEA (11–240 μM) between pH 6.5 and 9.0. Initial velocity data were fit to eq 6 to determine the apparent \( k_{\mathrm{cat}} \) and \( k_{\mathrm{cat}} / K_{\mathrm{m}} \) values at each pH. The apparent p\( K_{\mathrm{a}} \) values were then determined from these rate constants and eq 7.

**Culture of CDC1551 and Rv1106c Expression Analysis.** The Mtb strains used in this study were derivatives of Mtb CDC1551. The Mt1137 ORF corresponds to the Rv1106c ORF in Mtb H37Rv. The Mt1137 transposon mutant (28) was obtained through TARGET, part of NIAID contract HHSN266200400091C for Tuberculosis Vaccine Testing and Research Materials awarded to Colorado State University. The Mtb cells were cultured at 37 °C in Middlebrook 7H9 (liquid) or 7H10 (solid) (Difco) medium that was supplemented with 10% albumin–dextrose–sodium chloride complex (ADN) (29), 0.2% glycerol, and 0.05% Tween 80. The kanamycin concentration used for the transposon mutant in Mtb was 20 μg/mL. Induction with cholesterol was achieved by addition of cholesterol (final concentration of 1 mg/mL) solubilized in Tween 80 [final concentration of 5% (w/v)] to cultures at the mid-logarithmic phase. Cultures were incubated for 24 h at 37 °C and harvested by centrifugation. The culture filtrate was collected and sterile filtered for HPLC assay and enzymatic analysis. The cell pellet was washed by resuspension in 1 mL of 50 mM sodium phosphate buffer (pH 7) containing 5% (v/v) propan-2-ol; the cells were pelleted again, and the process was repeated two more times to remove cholesterol from the medium. The pellet was resuspended in the same buffer, and the cells were lysed by bead beating twice for 1 min, with incubation on ice for 2 min between each bead beating. The cell lysate was centrifuged to remove the cell debris, and the supernatant was sterilized by filtration and analyzed for enzymatic activity via a HPLC assay. Lysates were incubated with 150

Biochemistry, Vol. 46, No. 31, 2007 9061

μM cholesterol and 3.5 mM NAD⁺ in buffer D at 30 °C for 72 h and then directly injected onto a C18 reversed-phase HPLC column for analysis as described above. A unit of cholesterol oxidation activity was defined as 1 μmol of cholest-4-en-3-one per minute per milligram of protein under these assay conditions.

Other Analyses. The primary sequences of proteins related to Rv1106c were obtained from a BLASTP search initiated on the NIH server (http://www.ncbi.nlm.nih.gov/BLAST/). Multiple-sequence alignments and the phylogenetic tree were produced by the Protein Information Resource (30) using CLUSTALW 1.82 (31) and CLC free workbench 3.2 with default parameters. Signal peptide analysis was performed on the Center for Biological Sequence Analysis server (http:// www.cbs.dtu.dk/services/) using SignalP 3.0 (32) and Secretome 2.0 (33). Operon prediction was analyzed on The Institute for Genomic Research website (http://www.tigr.org/ tigr-scripts/operons/operons.cgi) (34).

**RESULTS AND DISCUSSION**

**Bioinformatics Analysis of Rv1106c.** Protein family analysis indicated that Rv1106c is most closely related to the Nocardia species cholesterol dehydrogenase (75% identical, UniProtKB entry Q03704) and other mycobacterial homologues (Figure 1). It shares 24% amino acid identity with the vaccinia virus 3β-hydroxysteroid dehydrogenase (UniProtKB entry O57245) and 29% identity with the human homologue (UniProtKB entry P14060).

In the Mtb H37Rv genome, the Rv1106c gene is located within 10 bases of the xseB (Rv1107c) and xseA (Rv1108c) genes, probable exonuclease VII small and large subunits, and 18 bases of a possible nitrobenzyl esterase (Rv1105c). Similar gene pairs are found only in mycobacterial genomes, e.g., Mycobacterium bovis, Mycobacterium avium paratuberculosis, and Mycobacterium leprae. The Nocardia farcinica 3β-hydroxysteroid dehydrogenase homologue nfa34580 is not paired with the exonuclease VII subunits. Signal peptide analysis indicated that the probability that the Rv1106c gene product is secreted classically with a signal peptide or nonclassically is less than 0.5 or 5%, respectively. The Rv1106c gene product appears to be a member of the short chain alcohol dehydrogenase superfamily (35). The glycine-rich sequence Gly-X-X-Gly-X-X-Gly, synonymous with the Rossman fold sequence Gly-X-Gly-X-X-Gly for AMP binding, is present as Gly²¹-Gly-Gly-Ala-Gly-Phe-Val-Gly²⁷. The same Gly-X-X-Gly-X-X-Gly variation of the motif is present in mammalian 3β-hydroxysteroid dehydrogenases (36). Moreover, the short chain alcohol dehydrogenase active site motif Tyr-X-X-X-Lys appears as Tyr¹⁵⁸-Thr-Glu-Thr-Lys¹⁶² in the translated Rv1106 gene. Last, a single aspartate (Asp³⁶ in the human type 1 3β-hydroxysteroid dehydrogenase) thought to form hydrogen bonds with the 2′,3′-hydroxyls of the cofactor ribose dictates dehydrogenase specificity for NAD⁺ rather than NADP⁺ (37). This aspartate is conserved in the Rv1106c sequence (Asp⁴⁵), suggesting that the Rv1106c gene product will preferentially use NAD⁺ as a cofactor.

Recombinant Expression Studies. Attempts to express the Rv1106c gene in M. smegmatis behind the acetamidase promoter were unsuccessful. Rv1106c was expressed in E. coli BL21(DE3)pLysS with an N-terminal His₆ tag.

9062 Biochemistry, Vol. 46, No. 31, 2007

Table 1: Tryptic Peptides from rH₆3BHSD Identified by MALDI-TOF Mass Spectrometry

| Residues | Sequences |
|----------|-----------|
| 1-50     | MGSSHHHHHHSSGLVPRGSHMLRRMGDASLTTELGRVLVTGGAGFVGANL |
| 51-100   | VTLLDRGHWVRSFDRAPSLLPAHPQLEVLQGDITDADVCAAAVDGIDTI |
| 101-150  | FHTAAIIELMGGASVTDEYRQSFAYNVGGTENLLHAGQRAGVQRFVYTS |
| 151-200  | SNSVVMGGQNIAGGDETLPYTDRFNDLYTETKVVAERFVLAQNGVDGMLT |
| 201-250  | CAIRPSGIWNGDQTMFRKLFESVLKGHVKVLVGRKSARLDNSYVHNLIH |
| 251-300  | GFILAAAHLVPDGTTAPGQAYFINDAEPINMFEFARPVLEACGQRWPKMRI |
| 301-350  | SGPAVRWMTGWQLHFRFGFPAPPLLEPLAVERLYLDNYFSIAKARRDLG |
| 351-392  | YEPLFTTQQALTECLPYYVSLFEQMKNEARAEKTAATVKPKL |

Scheme 1: Reaction Catalyzed by Mtb 3β-Hydroxysteroid Dehydrogenase (Rv1106c)

![Chemical Structure](chemical_structure_image.png)

cholesterol, cholesteneone: X = CH₃CH(CH₂)₃CH(CH₃)₂, Y = H  
pregnenolone, progesterone: X = COCH₃, Y = H  
dehydroepiandrosterone, androsterone: X, Y = O  

pregnenolone, and dehydroepiandrosterone), two flavonols (quercetin and dihydroquercetin), and nicotinamide cofactors. The cofactor or the sterols were omitted from the assay mixtures as negative controls. Ketone product was formed only in reaction mixtures containing NAD⁺ and 3β-hydroxysterol. NADP⁺ did not function as a cofactor with any of the three sterols at concentrations up to 10 mM. The cofactor preference is analogous to that reported for the *Nocardia* cholesterol dehydrogenase (20) and consistent with the conservation of Asp⁴⁵ in the short chain alcohol dehydrogenase family. rH₆3BHSD did not catalyze oxidation of quercetin with NAD⁺ or reduction of dihydroquercetin with NADH. The sterol reaction mixtures were analyzed by TLC and products compared to authentic standards. The enzyme catalyzed the oxidation and isomerization of cholesterol, pregnenolone, and dehydroepiandrosterone into cholest-4-en-3-one, progesterone, and androsterone, respectively (Scheme 1). In addition, the cholesterol reaction course was followed by reversed-phase C-18 chromatography, and the identities of the intermediate and product, cholest-5-en-3-one and cholest-4-en-3-one, respectively, were confirmed by co-injection with standards. The Rv1106c open reading frame clearly encodes a 3β-hydroxysterol dehydrogenase, not a dihydroflavonol reductase.

The rH₆3BHSD enzyme activity with cholesterol was measured as a function of pH, and the rate plateaued at pH 8.5. Next the enzyme activity was measured under four different detergent conditions: 0.05% Triton X-100, 2% Triton X-100, 1% nonyl glucoside, and 1% sodium cholate. There was no activity in nonyl glucoside and very little activity in sodium cholate. The highest activity was obtained with 2% Triton X-100. However, the enzyme activity was not saturated with cholesterol in any of the detergent systems. The highest cholesterol concentration that could reliably be attained was 1 mM cholesterol in 2% Triton X-100. At this cholesterol concentration, the specific activity of rH₆3BHSD was 2.90 μmol min⁻¹ mg⁻¹ in 100 mM triethanolamine hydrochloride (pH 8.5).

Dehydroepiandrosterone and pregnenolone have sufficient aqueous solubility that the use of detergent micelles is not required. Both substrates yielded normal saturation kinetics with the *Mtb* dehydrogenase. The lack of saturation with cholesterol may be due to cholesterol being a poor substrate.

expressed protein was present in both the soluble and membrane fractions. The soluble expressed protein did not bind to Ni²⁺ or Co²⁺ IMAC columns in its native form, suggesting that the N-terminus is not exposed in the folded protein. However, it could be purified by IMAC under denaturing conditions (6 M guanidinium hydrochloride), thus confirming that the expressed protein contained an N-terminal His tag. The protein identity was confirmed by MALDI-TOF mass analysis of a tryptic peptide mixture generated by in-gel digestion. Eighteen percent of the sequence was covered (Table 1). Expression levels in *E. coli* BL21(DE3)pLysS were 10-fold lower without the N-terminal fusion. The expressed proteins with and without the N-terminal His₆ tag were purified by conventional methods to yield rH₆3BHSD and r3BHSD, respectively. The apparent molecular mass was determined by SDS-PAGE to be 43 kDa, as expected for a protein that is not post-translationally processed. On a native gel, the apparent molecular mass of the recombinant protein was that of a monomer.

Reactions Catalyzed by the Recombinant Enzyme. rH₆3BHSD was assayed with three sterols (cholesterol,

for the enzyme. Alternatively, it may be due to a low affinity of the enzyme for the detergent micelle surface. After optimization of pH and cation conditions (vide infra), rH₆-3BHSD had the highest specific activity with dehydroepiandrosterone as the substrate; the activity was 2.5–3-fold higher with dehydroepiandrosterone than with cholesterol in 0.05% Triton X-100 or pregnenolone. The same relative activities were obtained when each of the three sterols was assayed in the presence of 0.05% Triton X-100 in the optimized buffer conditions of 100 mM TAPS hydrochloride, 150 mM NaCl, 30 mM MgCl₂ (pH 8.5), and 2.8 mM NAD⁺ (vide infra). The specific activities measured were 0.15, 0.36, and 0.28 μM min⁻¹ mg⁻¹ for cholesterol, dehydroepiandrosterone, and pregnenolone, respectively. Dehydroepiandrosterone was used as the substrate in subsequent studies because its solubility and kinetic behavior made it the most amenable to the detailed analysis.

Both the recombinant Rv1106c with (rH₆3BHSD) and without (r3BHSD) an N-terminal histidine tag were assayed with a fixed concentration of dehydroepiandrosterone (300 μM), and the NAD⁺ concentration was varied from 1 to 28 mM in 100 mM triethanolamine hydrochloride at pH 8.5 and 30 °C. The apparent \( K_m \) values for NAD⁺ were 2.4 ± 0.7 and 2.2 ± 0.3 mM, respectively, demonstrating that the N-terminal histidine tag did not affect substrate binding. Because the r3BHSD protein was expressed at low levels and could not be purified to homogeneity, its \( k_{cat} \) could not be accurately determined. All subsequent experiments were performed with N-terminally His-tagged enzyme rH₆3BHSD because it was expressed at higher levels and was easier to purify.

The substrate specificities reported for the *Nocardia* cholesterol dehydrogenase were similar but not identical to those observed for rH₆3BHSD (20). The *Nocardia* enzyme utilizes both cholesterol and pregnenolone, whereas no activity with dehydroepiandrosterone was observed. However, the assay conditions used for the latter sterols were not reported. Therefore, it is unclear if the difference in specificity is due to buffer conditions or true differences between the enzymes. Kishi et al. (20) reported a \( K_m \) of 2.5 mM for cholesterol in 1.87% Triton X-100 for the *Nocardia* cholesterol dehydrogenase. However, the highest cholesterol concentration that was used was 2.9 mM, suggesting that as with the *Mtb* enzyme studied in this work, saturation of the *Nocardia* enzyme with cholesterol was not possible.

**pH Dependence.** Initial velocity kinetic patterns and apparent kinetic constants \( K_m \) and \( k_{cat} \) with dehydroepiandrosterone as the substrate were measured at a fixed concentration of NAD⁺ (2.8 mM) in 100 mM TAPS hydrochloride with 150 mM NaCl and 30 mM MgCl₂ at pH 9.5, 9.0, 8.5, 8.0, 7.5, 7.0, and 6.5. No catalytic activity was detected below pH 7.5. The apparent pKₐ for \( k_{cat} \) is 7.9 ± 0.1, and the apparent pKₐ for \( k_{cat}/K_m \) is 8.1 ± 0.1. The pH optimum of the enzyme is 8.5–9.5 (Figure 2). This pH optimum range is comparable to that reported for the *Nocardia* cholesterol dehydrogenase (20). Further experiments were conducted at pH 8.5 to optimize activity while minimizing decomposition of the NAD⁺ cofactor during the assay.

**Cation and Ionic Strength Dependence.** rH₆3BHSD was assayed with 10 and 1 mM CaCl₂, MgCl₂, 1 mM FeCl₂, CuCl₂, ZnCl₂, or AgNO₃, CuCl₂, or 50 mM EDTA in the

FIGURE 2: pH dependence of the reaction catalyzed by rH₆3BHSD with DHEA as the varied substrate. Conditions: 2.8 mM NAD⁺, 100 mM TAPS hydrochloride buffer, 150 mM NaCl, 30 mM MgCl₂, pH 8.5, and 30 °C. (A) pH dependence of \( k_{cat} \). (B) pH dependence of \( k_{cat}/K_m \). The curves are fits to eq 7. The data shown are the average of three independent experiments, and the errors are the standard deviation of the measurements.

presence of 150 μM dehydroepiandrosterone and 3.5 mM NAD⁺ in 100 mM triethanolamine hydrochloride (pH 8.5) (Figure 3A). EDTA, Fe²⁺, Zn²⁺, Ag⁺, and Cu²⁺ inhibited enzymatic activity. Mg²⁺ and Ca²⁺ were the best activators of enzymatic activity. Then, rH₆3BHSD was assayed as a function of NaCl concentration in the presence of 150 μM dehydroepiandrosterone and 2.8 mM NAD⁺ in 100 mM TAPS hydrochloride (pH 8.5) (Figure 3B). The activity increased with an increase in NaCl concentration and plateaued at ~300 mM. A NaCl concentration of 150 mM was used in subsequent assays as this represented a physiological ionic strength. Next, rH₆3BHSD was assayed with 10 and 1 mM CaCl₂, MgCl₂, 1 mM FeCl₂, CuCl₂, ZnCl₂, or AgNO₃, CuCl₂, or 50 mM EDTA in the presence of 150 μM dehydroepiandrosterone, 150 mM NaCl, and 2.8 mM NAD⁺ in 100 mM TAPS hydrochloride (pH 8.5) (Figure 3C). The highest activities were observed with Ca²⁺, Mg²⁺, and K⁺. Different concentrations of Mg²⁺ were assayed in the presence of 150 mM NaCl, and the activity plateaued between 20 and 40 mM (Figure 3C). Subsequent assays were performed with 150 mM NaCl and 30 mM MgCl₂.

Steady-State Kinetic Assays. Initial velocity data with rH₆-3BHSD and varied substrate concentrations produced an intersecting line pattern (Figure S1 of the Supporting Information) consistent with a sequential binding mechanism which is expected for an alcohol dehydrogenase reaction. Concentrations of NAD⁺ higher than 5.6 mM inhibit the dehydrogenase reaction (Figure 4). The substrate inhibition by NAD⁺ could be overcome by increasing the concentration of dehydroepiandrosterone. This competitive inhibition suggests either that NAD⁺ binds to the E–NAD⁺ complex to

9064 Biochemistry, Vol. 46, No. 31, 2007

**FIGURE 3: Cation dependence of rH₆3BHSD activity.** (A) Specific activity in the presence of assorted cations or EDTA. Conditions: 150 μM DHEA, 3.5 mM NAD⁺, 100 mM triethanolamine hydrochloride buffer, pH 8.5, and 30 °C. (B) Specific activity as a function of NaCl concentration. Conditions: 150 μM DHEA, 2.8 mM NAD⁺, 100 mM TAPS hydrochloride buffer, pH 8.5, 150 mM NaCl, and 30 °C. (C) Specific activity in the presence of 150 mM NaCl and assorted cations or EDTA. Conditions: 150 μM DHEA, 2.8 mM NAD⁺, 100 mM TAPS hydrochloride buffer, pH 8.5, 150 mM NaCl, and 30 °C. The data shown are the average of three independent experiments, and the errors are the standard deviation of the measurements.

form a dead-end E–(NAD⁺)₂ complex or that the E–NAD⁺ complex is itself a dead-end complex. Which species is formed depends on the order of substrate binding in the enzyme-catalyzed reaction. Further analysis with additional inhibitors is required to determine the reaction order (vide infra).

Because of substrate inhibition, the highest concentration of NAD⁺ used for fits of initial velocity data to eq 1 was 4.2 mM. The ternary complex steady-state kinetic parameters were derived by globally fitting the initial velocity data to eq 1 for an ordered reaction (Table 2). Inhibitors of the reaction catalyzed by rH₆3BHSD were required for the determination of the order of substrate binding. A steroid-

**FIGURE 4: Steady-state kinetics of rH₆3BHSD.** A plot of initial velocity vs NAD⁺ concentration at a fixed DHEA concentration was fitted to eq 5. Conditions: 120 μM DHEA, 100 mM TAPS hydrochloride buffer, 150 mM NaCl, 30 mM MgCl₂, pH 8.5, and 30 °C. The data shown are the average of three independent experiments, and the errors are the standard deviation of the measurements.

derived inhibitor and a cofactor-derived inhibitor were sought.

Trilostane [(2α,4α,5α,17β)-4,5-epoxy-17-hydroxy-3-oxoandrostan-2-carbonitrile] is a known competitive inhibitor (vs pregnenolone) of mammalian 3β-hydroxysteroid dehydrogenases (38). It was expected that trilostane would be an inhibitor of the Mtb 3β-hydroxysteroid dehydrogenase as well because the reactions catalyzed by both enzymes, and thus, the substrates, intermediates, and products, are the same. Indeed, trilostane is a micromolar inhibitor of rH₆3BHSD (Figure S2 of the Supporting Information). Moreover, the cofactor product of oxidation, NADH, was also found to be an inhibitor of the rH₆3BHSD-catalyzed reaction. The order of the reaction catalyzed by rH₆3BHSD was determined by analyzing patterns of inhibition by trilostane and NADH (Table 3 and Figures S2 and S3). Competitive inhibition by trilostane with respect to dehydroepiandrosterone and competitive inhibition by NADH with respect to NAD⁺ suggested that the reaction has a compulsory order. If the order of binding were random, mixed inhibition would have been observed. Uncompetitive inhibition by trilostane with respect to NAD⁺ and mixed inhibition by NADH with respect to DHEA indicate that the NAD⁺ binds to the enzyme first followed by DHEA in the formation of a ternary complex. This order of reaction is observed for a large number of NAD(P)-dependent dehydrogenases. This information was used to fit an ordered ternary mechanism to the kinetic data acquired for both dehydroepiandrosterone and pregnenolone as substrates. Although *k*~cat~ is larger for dehydroepiandrosterone, the enzyme has a larger *K*~m~ for this substrate compared to pregnenolone. Thus, the second-order or specificity rate constants (*k*~cat~/*K*~m~) are approximately the same for both steroids. A similar lack of specificity has been observed for the human 3β-hydroxysteroid dehydrogenases (39). The human enzymes catalyze the dehydrogenase reaction at second-order rates similar to those catalyzed by rH₆3BHSD.

Furthermore, the competitive substrate inhibition by NAD⁺ that is observed must be due to the formation of an E–(NAD⁺)₂ dead-end or nonproductive complex. The structural explanation for how this complex may be formed awaits three-dimensional structural data that are unavailable at present. The initial velocity data versus NAD⁺ concentra-

Mtb Sterol Dehydrogenase

Table 2: Michaelis-Menten Constants for rH₆3BHSD

| substrate A | substrate B       | $K_{ia} (\mu M)^a$ | $K_{ma} (\mu M)^a$ | $K_{mb} (\mu M)^a$ | $k_{cat} (s^{-1})^a$ | $k_{cat}/K_{mb} (\mu M^{-1} \text{min}^{-1})$ |
|-------------|-------------------|--------------------|--------------------|--------------------|----------------------|--------------------------------------------|
| NAD⁺        | DHEA              | 347 ± 28           | 270 ± 37           | 221 ± 22           | 11.6 ± 0.73          | 3.2 ± 0.4                                 |
| NAD⁺        | pregnenolone      | 213 ± 162          | 172 ± 52           | 23 ± 5             | 1.48 ± 0.09          | 3.9 ± 0.9                                 |

$a$ A compulsory order ternary complex mechanism as described in eq 1 was fit to the initial velocity data to yield $K_{ia}$, $K_{mB}$, and $K_{mA}$. Conditions: 100 mM TAPS, 150 mM NaCl, 30 mM MgCl₂ buffer, pH 8.5, and 30°C.

---

Table 3: Inhibition of rH₆3BHSD

| inhibitor   | pattern type$^b$ | $K_{ic} (\mu M)^a$ | $K_{iu} (\mu M)^a$ | pattern type$^b$ | $K_{ic} (\mu M)^a$ | $K_{iu} (\mu M)^a$ |
|-------------|------------------|--------------------|--------------------|------------------|--------------------|--------------------|
| trilostane  | C                | 197 ± 8$^c$        | na$^d$             | UC               | na$^d$             | 180 ± 11$^e$       |
| NADH        | mixed            | 245 ± 68           | 116 ± 22           | C                | 72 ± 3             | na$^{df}$         |

$a$ A compulsory order ternary complex mechanism was fit to eqs 2–4 with the initial velocity data, and the best fit was used to yield $K_{ic}$ or $K_{iu}$. Conditions: 100 mM TAPS hydrochloride buffer, pH 8.5, 150 mM NaCl, 30 mM MgCl₂, and 30°C. $^b$ C, competitive inhibition; UC, uncompetitive inhibition; mixed, mixed inhibition. $^c$ The NAD⁺ concentration was fixed at 350 μM. $^d$ Not applicable. $^e$ The DHEA concentration was fixed at 120 μM. $^f$ The DHEA concentration was fixed at 90 μM.

of the MT1137 gene reduces the cholesterol oxidation activity of Mtb at least 90-fold. In addition, no cholesterol oxidation activity was observed in either the wild-type or mutant culture filtrates.

The implications of this experiment are several-fold. First, upon disruption of a single gene, all detectable cholesterol oxidation activity is abrogated. This result implies that there is a single sterol-oxidizing enzyme and that this enzyme is the Rv1106c-encoded 3β-hydroxysteroid dehydrogenase. Second, cholesterol oxidation activity was observed only in cell lysates; no activity was observed in culture filtrates. This result suggests that Mtb does not secrete the 3β-hydroxysteroid dehydrogenase encoded by Rv1106c. This observation is consistent with a proteomic analysis that identified the Mtb Rv1106c gene product in the cytosol (40, 41). Third, disruption of the enzymatic activity with a single knockout bodes well for the discovery of small molecule inhibitors that can completely inhibit Mtb cholesterol oxidation in vivo. However, the importance of the latter type of discovery efforts will need to be validated by host-infection virulence studies, which are underway.

Summary. Mycobacteria and related actinomycetes have long been reported to oxidize environmental cholesterol (13, 14). However, the actual identity and genotype of the strains isolated and studied have not always been clear (42). With the recent complete sequencing of several bacterial genomes, including that of Mtb (10, 43), it is now possible to connect bacterial phenotypes to specific genes. In this work, genome mining was used to identify a potential 3β-hydroxysteroid oxidizing enzyme from Mtb, the product of the Rv1106c gene.

Elucidation of the kinetic profile for the heterologously expressed Mtb Rv1106c gene product clearly establishes that the enzyme is a 3β-hydroxysteroid dehydrogenase that oxidizes the 3-hydroxyl and isomerizes the α,γ-unsaturated ketone into the conjugated α,β-unsaturated ketone of at least three sterol substrates, cholesterol, dehydroepiandrosterone, and pregnenolone. Under the conditions optimized in this work, the enzyme is equally specific for dehydroepiandrosterone and pregnenolone and 3-fold less specific for cholesterol. Although the maximal catalytic activity is 10-fold lower with pregnenolone as a substrate than with dehydroepiandrosterone, the apparent binding constant ($K_m$) for pregnenolone is 10-fold better (Table 2). Thus, the in vivo substrate concentrations, in the case of cholesterol membrane composition, will dictate the substrate selection. At present, these concentrations and membrane compositions are unknown.

The intracellular expression of the Mtb Rv1106c gene product is more consistent with the use of 3β-hydroxy sterols for steroid hormone biosynthesis, for example, by mycobacterial conversion of cholesterol into glucocorticoids as seen in the case of vaccinia virus (25) or androgens, rather than a role in energy metabolism. Mtb does not grow on

cholesterol as a sole carbon source. Moreover, all characterized examples of actinomycetes that catabolize 3β-hydroxy-sterols for energy use, e.g., *Rhodococcus* (44, 45) and *Streptomyces* (46), secrete the sterol oxidizing enzyme. Last, in contrast to the *Mtb Rv1106c* gene product, 3-hydroxy steroid oxidases known to be involved in primary metabolism are more specific for cholesterol than dehydroepiandrosterone or pregnenolone (47–49).

The intracellular expression of the enzyme suggests that 3β-hydroxysterols must be taken into the mycobacterium from its environment, e.g., the host cell macrophage. An alternative possibility is that the enzymatic substrate is synthesized by the mycobacterium. However, no evidence for the mycobacterial biosynthesis of cholesterol has yet been obtained (17, 18). Future experiments will explore the possible physiological and pathological roles of this enzyme in steroid biosynthesis and degradation.

### ACKNOWLEDGMENT

We thank Yelena Altshuller of the Molecular Cloning Facility at Stony Brook University for isolating the original *Rv1106c* clone by PCR from genomic DNA.

### SUPPORTING INFORMATION AVAILABLE

Double-reciprocal and secondary plots for two-substrate steady-state kinetics and inhibition studies. This material is available free of charge via the Internet at http://pubs.acs.org.

### REFERENCES

1. Kochi, A. (1991) The global tuberculosis situation and the new control strategy of the World Health Organization, *Tubercle* 72, 1–6.
2. Raviglione, M. C., Snider, D. E., Jr., and Kochi, A. (1995) Global epidemiology of tuberculosis. Morbidity and mortality of a worldwide epidemic, *J. Am. Med. Assoc.* 273, 220–226.
3. Byarugaba, D. K. (2004) Antimicrobial resistance in developing countries and responsible risk factors, *Int. J. Antimicrob. Agents* 24, 105–110.
4. Mwinga, A., and Bernard Fourie, P. (2004) Prospects for new tuberculosis treatment in Africa, *Trop. Med. Int. Health* 9, 827–832.
5. Raviglione, M. C., and Smith, I. M. (2007) XDR tuberculosis: Implications for global public health, *N. Engl. J. Med.* 356, 656–659.
6. (2006) XDR-TB: A global threat, *Lancet* 368, 964.
7. Rattan, A., Kalia, A., and Ahmad, N. (1998) Multidrug-resistant *Mycobacterium tuberculosis*: Molecular perspectives, *Emerging Infect. Dis.* 4, 195–209.
8. Marris, E. (2006) Extreme TB strain threatens HIV victims worldwide, *Nature* 443, 131.
9. Cohen, J. (2006) Infectious disease. Extensively drug-resistant TB gets foothold in South Africa, *Science* 313, 1554.
10. Cole, S. T., Brosch, R., Parkhill, J., Garnier, T., Churcher, C., Harris, D., Gordon, S. V., Eiglmeier, K., Gas, S., Barry III, C. E., Tekaia, F., Badcock, K., Basham, D., Brown, D., Chillingworth, T., Connor, R., Davies, R., Devlin, K., Feltwell, T., Gentles, S., Hamlin, N., Holroyd, S., Hornsby, T., Jagels, K., Krogh, A., McLean, J., Moule, S., Murphy, L., Oliver, K., Osborne, J., Quail, M. A., Rajandream, M. A., Rogers, J., Rutter, S., Seeger, K., Skelton, J., Squares, R., Squares, S., Sulston, J. E., Taylor, K., Whitehead, S., and Barrell, B. G. (1998) Deciphering the biology of *Mycobacterium tuberculosis* from the complete genome sequence, *Nature* 393, 537–544.
11. Eiglmeier, K., Simon, S., Garnier, T., and Cole, S. T. (2001) The integrated genome map of *Mycobacterium leprae*, *Lepr. Rev.* 72, 462–469.
12. Garnier, T., Eiglmeier, K., Camus, J. C., Medina, N., Mansoor, H., Pryor, M., Duthoy, S., Grondin, S., Lacroix, C., Monsempe, C., Simon, S., Harris, B., Atkin, R., Doggett, J., Mayes, R., Keating, L., Wheeler, P. R., Parkhill, J., Barrell, B. G., Cole, S. T., Gordon, S. V., and Hewinson, R. G. (2003) The complete genome sequence of *Mycobacterium bovis*, *Proc. Natl. Acad. Sci. U.S.A.* 100, 7877–7882.
13. Tak, J. (1942) On bacteria decomposing cholesterol, *Antonie van Leeuwenhoek* 8, 32–40.
14. Turfitt, G. E. (1944) The microbiological degradation of steroids. 2. Oxidation of cholesterol by *Proactinomyces* spp., *Biochem. J.* 38, 492–496.
15. Van der Geize, R., Yam, K., Heuser, T., Wilbrink, M. H., Hara, H., Anderton, M. C., Sim, E., Dijkhuizen, L., Davies, J. E., Mohn, W. W., and Eltis, L. D. (2007) A gene cluster encoding cholesterol catabolism in a soil actinomycete provides insight into *Mycobacterium tuberculosis* survival in macrophages, *Proc. Natl. Acad. Sci. U.S.A.* 104, 1947–1952.
16. Av-Gay, Y., and Sobouti, R. (2000) Cholesterol is accumulated by mycobacteria but its degradation is limited to non-pathogenic fast-growing mycobacteria, *Can. J. Microbiol.* 46, 826–831.
17. Lamb, D. C., Kelly, D. E., Manning, N. J., and Kelly, S. L. (1998) A sterol biosynthetic pathway in *Mycobacterium*, *FEBS Lett.* 437, 142–144.
18. Jackson, C. J., Lamb, D. C., Marczylo, T. H., Parker, J. E., Manning, N. L., Kelly, D. E., and Kelly, S. L. (2003) Conservation and cloning of CYP51: A sterol 14α-demethylase from *Mycobacterium smegmatis*, *Biochem. Biophys. Res. Commun.* 301, 558–563.
19. Horinouchi, S., Ishizuka, H., and Beppu, T. (1991) Cloning, nucleotide sequence, and transcriptional analysis of the NAD(P)-dependent cholesterol dehydrogenase gene from a *Nocardia* sp. and its hyperexpression in *Streptomyces* spp., *Appl. Environ. Microbiol.* 57, 1386–1393.
20. Kishi, K., Watazu, Y., Katayama, Y., and Okabe, H. (2000) The characteristics and applications of recombinant cholesterol dehydrogenase, *Biosci., Biotechnol., Biochem.* 64, 1352–1358.
21. Ramakrishnan, L., Federspiel, N. A., and Falkow, S. (2000) Granuloma-specific expression of *Mycobacterium* virulence proteins from the glycine-rich PE-PGRS family, *Science* 288, 1436–1439.
22. Baker, M. E., and Blasco, R. (1992) Expansion of the mammalian 3β-hydroxysteroid dehydrogenase/plant dihydroflavonol reductase superfamily to include a bacterial cholesterol dehydrogenase, a bacterial UDP-galactose-4-epimerase, and open reading frames in vaccinia virus and fish lymphocystis disease virus, *FEBS Lett.* 301, 89–93.
23. Baker, M. E., Luu-The, Y., Simard, J., and Labrie, F. (1990) A common ancestor for mammalian 3β-hydroxysteroid dehydrogenase and plant dihydroflavonol reductase, *Biochem. J.* 269, 558–559.
24. Baker, M. E. (1992) Evolution of regulation of steroid-mediated intercellular communication in vertebrates: Insights from flavonoids, signals that mediate plant-rhizobia symbiosis, *J. Steroid Biochem. Mol. Biol.* 41, 301–308.
25. Reading, P. C., Moore, J. B., and Smith, G. L. (2003) Steroid hormone synthesis by vaccinia virus suppresses the inflammatory response to infection, *J. Exp. Med.* 197, 1269–1278.
26. Changsen, C., Franzblau, S. G., and Palittapongarnpim, P. (2003) Improved green fluorescent protein reporter gene-based microplate screening for antituberculosis compounds by utilizing an acetamidase promoter, *Antimicrob. Agents Chemother.* 47, 3682–3687.
27. Sampson, N. S., and Kass, I. J. (1997) Isomerization, but not oxidation, is suppressed by a single point mutation, E361Q, in the reaction catalyzed by cholesterol oxidase, *J. Am. Chem. Soc.* 119, 855–862.
28. Lamichhane, G., Zignol, M., Blades, N. J., Geiman, D. E., Dougherty, A., Grosset, J., Broman, K. W., and Bishai, W. R. (2003) A postgenomic method for predicting essential genes at subsaturation levels of mutagenesis: Application to *Mycobacterium tuberculosis*, *Proc. Natl. Acad. Sci. U.S.A.* 100, 7213–7218.
29. Jacobs, W. R., Jr., Kalpana, G. V., Cirillo, J. D., Pascopella, L., Snapper, S. B., Udani, R. A., Jones, W., Barletta, R. G., and Bloom, B. R. (1991) Genetic systems for mycobacteria, *Methods Enzymol.* 204, 537–555.
30. Wu, C. H., Yeh, L. S., Huang, H., Arminski, L., Castro-Alvear, J., Chen, Y., Hu, Z., Kourtesis, P., Ledley, R. S., Suzek, B. E., Vinayaka, C. R., Zhang, J., and Barker, W. C. (2003) The Protein Information Resource, *Nucleic Acids Res.* 31, 345–347.
31. Chenna, R., Sugawara, H., Koike, T., Lopez, R., Gibson, T. J., Higgins, D. G., and Thompson, J. D. (2003) Multiple sequence

alignment with the Clustal series of programs, *Nucleic Acids Res.* 31, 3497–3500.

32. Bendtsen, J. D., Nielsen, H., von Heijne, G., and Brunak, S. (2004) Improved prediction of signal peptides: SignalP 3.0, *J. Mol. Biol.* 340, 783–795.

33. Bendtsen, J. D., Kiemer, L., Fausboll, A., and Brunak, S. (2005) Non-classical protein secretion in bacteria, *BMC Microbiol.* 5, 58.

34. Ermolaeva, M. D., White, O., and Salzberg, S. L. (2001) Prediction of operons in microbial genomes, *Nucleic Acids Res.* 29, 1216–1221.

35. Jornvall, H., Persson, B., Krook, M., Atrian, S., Gonzalez-Duarte, R., Jeffery, J., and Ghosh, D. (1995) Short-chain dehydrogenases/reductases (SDR), *Biochemistry* 34, 6003–6013.

36. Simard, J., Ricketts, M. L., Gingras, S., Soucy, P., Feltus, F. A., and Melner, M. H. (2005) Molecular biology of the 3β-hydroxy-steroid dehydrogenase/Δ5-Δ4 isomerase gene family, *Endocr. Rev.* 26, 525–582.

37. Thomas, J. L., Duax, W. L., Addlagatta, A., Brandt, S., Fuller, R. R., and Norris, W. (2003) Structure/function relationships responsible for coenzyme specificity and the isomerase activity of human type 1 3β-hydroxysteroid dehydrogenase/isomerase, *J. Biol. Chem.* 278, 35483–35490.

38. Potts, G. O., Creange, J. E., Hardomg, H. R., and Schane, H. P. (1978) Trilostane, an orally active inhibitor of steroid biosynthesis, *Steroids* 32, 257–267.

39. Thomas, J. L., Mason, J. I., Brandt, S., Spencer, B. R., Jr., and Norris, W. (2002) Structure/function relationships responsible for the kinetic differences between human type 1 and type 2 3β-hydroxysteroid dehydrogenase and for the catalysis of the type 1 activity, *J. Biol. Chem.* 277, 42795–42801.

40. Rosenkrands, I., King, A., Weldingh, K., Moniatte, M., Moertz, E., and Andersen, P. (2000) Towards the proteome of *Mycobacterium tuberculosis*, *Electrophoresis* 21, 3740–3756.

41. Rosenkrands, I., Weldingh, K., Jacobsen, S., Hansen, C. V., Florio, W., Gianetri, I., and Andersen, P. (2000) Mapping and identification of *Mycobacterium tuberculosis* proteins by two-dimensional gel electrophoresis, microsequencing and immunodetection, *Electrophoresis* 21, 935–948.

42. Stadtman, T. C. (1955) Cholesterol dehydrogenase from a *Mycobacterium*, *Methods Enzymol.* 1, 678–681.

43. Camus, J. C., Pryor, M. J., Medigue, C., and Cole, S. T. (2002) Re-annotation of the genome sequence of *Mycobacterium tuberculosis* H37Rv, *Microbiology* 148, 2967–2973.

44. Sojo, M., Bru, R., LopezMolina, D., GarciaCarmona, F., and Arguelles, J. C. (1997) Cell-linked and extracellular cholesterol oxidase activities from *Rhodococcus erythropolis*. Isolation and physiological characterization, *Appl. Microbiol. Biotechnol.* 47, 583–589.

45. Elalami, A., Kreit, J., Filali-Maltouf, A., Boudrant, J., and Germain, P. (1999) Characterization of a secreted cholesterol oxidase from *Rhodococcus* sp. GKI (CIP 105335), *World J. Microbiol. Biotechnol.* 15, 579–585.

46. Murooka, Y., Ishizaki, T., Nimi, O., and Maekawa, N. (1986) Cloning and expression of a *Streptomyces* cholesterol oxidase gene in *Streptomyces lividans* with plasmid pIJ702, *Appl. Environ. Microbiol.* 52, 1382–1385.

47. MacLachlan, J., Wotherspoon, A. T., Ansell, R. O., and Brooks, C. J. (2000) Cholesterol oxidase: Sources, physical properties and analytical applications, *J. Steroid Biochem. Mol. Biol.*, 72, 169–195.

48. Sampson, N. S., Kass, I. J., and Ghoshroy, K. B. (1998) Assessment of the role of an Ω loop of cholesterol oxidase: A truncated loop mutant has altered substrate specificity, *Biochemistry* 37, 5770–5778.

49. Smith, A. G., and Brooks, C. J. W. (1977) The substrate specificity and stereochemistry, reversibility and inhibition of the 3-oxo steroid Δ⁴-Δ⁵ isomerase component of cholesterol oxidase, *Biochem. J.* 167, 121–129.
