
Enzyme Catalysis and Regulation:  
Investigation of Early Tailoring Reactions  
in the Oxytetracycline Biosynthetic  
Pathway  

Wenjun Zhang, Kenji Watanabe, Clay C. C.  
Wang and Yi Tang  
J. Biol. Chem. 2007, 282:25717-25725.  
doi: 10.1074/jbc.M703437200 originally published online July 12, 2007  

Access the most updated version of this article at doi: [10.1074/jbc.M703437200](http://dx.doi.org/10.1074/jbc.M703437200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).  

Alerts:  
- [When this article is cited](#)  
- [When a correction for this article is posted](#)  

[Click here](#) to choose from all of JBC's e-mail alerts  

Supplemental material:  
[http://www.jbc.org/content/suppl/2007/07/13/M703437200.DC1.html](http://www.jbc.org/content/suppl/2007/07/13/M703437200.DC1.html)  

This article cites 45 references, 10 of which can be accessed free at  
[http://www.jbc.org/content/282/35/25717.full.html#ref-list-1](http://www.jbc.org/content/282/35/25717.full.html#ref-list-1)

Investigation of Early Tailoring Reactions in the Oxytetracycline Biosynthetic Pathway*

Received for publication, April 25, 2007, and in revised form, June 26, 2007 Published, JBC Papers in Press, July 12, 2007, DOI 10.1074/jbc.M703437200

Wenjun Zhang†, Kenji Watanabe§, Clay C. C. Wang§, and Yi Tang‡

From the †Department of Chemical and Biomolecular Engineering University of California, Los Angeles, California 90095 and §Department of Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles California 90089

Tetracyclines are aromatic polyketides biosynthesized by bacterial type II polyketide synthases. The amidated tetracycline backbone is biosynthesized by the minimal polyketide synthases and an amidotransferase homologue OxyD. Biosynthesis of the key intermediate 6-methylpretetramid requires two early tailoring steps, which are cyclization of the linearly fused tetracyclic scaffold and regioselective C-methylation of the aglycon. Using a heterologous host (CH999)/vector pair, we identified the minimum set of enzymes from the oxytetracycline biosynthetic pathway that is required to afford 6-methylpretetramid *in vivo*. Only two cyclases (OxyK and OxyN) are necessary to completely cyclize and aromatize the amidated tetracyclic aglycon. Formation of the last ring via C-1/C-18 aldol condensation does not require a dedicated fourth-ring cyclase, in contrast to the biosynthetic mechanism of other tetracyclic aromatic polyketides, such as daunorubicin and tetracenomycin. Acetyl-derived polyketides do not undergo spontaneous fourth-ring cyclization and form only anthracene carboxylic acids as demonstrated both *in vivo* and *in vitro*. OxyF was identified to be the C-6 C-methyltransferase that regioselectively methylates pretetramid to yield 6-methylpretetramid. Reconstitution of 6-methylpretetramid in a heterologous host sets the stage for a more systematic investigation of additional tetracycline downstream tailoring enzymes and is a key step toward the engineered biosynthesis of tetracycline analogs.

Tetracyclines are aromatic polyketides biosynthesized by soil-borne *Streptomyces* bacteria (1, 2). It is well known that the carbon skeleton of an aromatic polyketide is assembled through stepwise decarboxylative condensation of malonate equivalents catalyzed by the minimal PKS,² which consists of a ketosynthase/chain length factor heterodimer (KS-CLF or KSα-KSβ), an acyl-carrier protein (ACP), and a malonyl-CoA:ACP acyl-transferase (3). Dedicated tailoring enzymes transform the highly reactive poly-β-ketone backbone into fused, richly substituted compounds (4). The biosynthesis of tetracyclines has been studied using blocked mutants of the chlorotetracycline producer *Streptomyces aureofaciens* (5–11). However, the underlying enzymology of several key tailoring steps that give rise to its structural features remain unresolved, including cyclization of the tetracyclic scaffold and C-methylation of C-6 to afford the key intermediate 6-methylpretetramid (Fig. 1). The oxytetracycline (*oxy*) biosynthetic gene cluster from *Streptomyces rimosus* has been completely sequenced, hence allowing a thorough investigation of the biochemical basis of these features (12–16).

During tetracycline biosynthesis, the C-9 reduced decaketide backbone first undergoes C-7/C-12 intramolecular aldol condensation to fix the regioselectivity of the D ring followed by three sequential cyclization reactions to form a fully aromatized, tetracyclic intermediate pretetramid (Fig. 1). Biosynthesis of aureolic acids such as mithramycin presumably follows identical cyclization patterns, as inferred from the structure of premithramycinone, an early intermediate during mithramycin biosynthesis (17). The biochemical basis of the C-1/C-18 cyclization reaction, which affords the last rings in both tetracycline and mithramycin, is not well understood. Combinatorial biosynthesis using mithramycin and tetracenomycin biosynthetic genes indicated that MtmX may function as a fourth ring cyclase (18), although no direct biochemical evidence is available. The *oxy* gene cluster encodes an MtmX homologue, OxyI, which may be involved in A ring cyclization. Compounds in the anthracycline family, such as daunorubicin, undergo identical cyclization reactions to fix the first three rings (Fig. 1) (19). Formation of the fourth, non-aromatic ring to yield alkaviketone during daunorubicin biosynthesis is catalyzed by the aklanonic acid methyl ester cyclase DnrD via distinct C-2/C-19 connectivity (20, 21). The tetracenomycin family of compounds is constructed from fully aromatized intermediate tetracenomycin F1, but the regioselectivities are significantly different, starting with the first cyclization, which occurs between C-9 and C-14. The fourth ring cyclization takes place between C-2/C-19 and is catalyzed by a dedicated cyclase TcmI (22, 23).

Another important structural feature of tetracyclines is a methyl group installed on C-6 of second ring. The methylation modification occurs relatively early during tetracycline biosynthesis as evidenced by isolation of the intermediate 6-methylpretetramid (Fig. 1) (8). The C-methylation reaction is likely

* This work was supported by a UCLA startup grant, University of California Cancer Research Coordinating Committee funds, National Science Foundation Chemical, Bioengineering, Environmental, and Transport Systems Grant 0545860, NIGMS, National Institutes of Health Grant RO1-GM75857 (to C. C. C. W.), and American Cancer Society Grant RSG-06-010-01-CDD (to C. C. C. W.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
§ The on-line version of this article (available at http://www.jbc.org) contains supplemental Fig. S1 and Tables S1–S4.
¹ To whom correspondence should be addressed: Dept. of Chemical and Biomolecular Engineering, University of California Los Angeles, Los Angeles, CA 90095. Tel.: 810-825-0375; E-mail: yitang@ucla.edu.
² The abbreviations used are: PKS, polyketide synthase; LC, liquid chromatography; MS, mass spectroscopy; HPLC, high performance liquid chromatography; EA, ethyl acetate; DMAC, 3,8-dihydroxy-methylanthraquinone carboxylic acid; HR-ESIMS, high resolution electrospray ionization-MS.

Tetracycline Biosynthesis

![Diagram](https://i.imgur.com/1234567.png)

FIGURE 1. The different cyclization patterns for various four-ring aromatic polyketides. TcmI is required for the C-2/C-19 cyclization in TcmF2 to yield TcmF1. DnrD is required for the C-2/C-19 cyclization in alkanonic acid methyl ester to yield alkaviketone. During tetracycline biosynthesis, cyclization between C-1/C-18 completes formation of the tetracyclic scaffold, whereas C-6 methylation yields the key intermediate 6-methylpretetramid. The enzymes indicated with a box are cyclases.

catalyzed by one of two methyltransferases in the oxy gene cluster (OxyF and OxyT), both containing putative S-adenosylmethionine binding domains. C-Methyltransferases are found infrequently among bacterial type II PKSs, including MtmMII in the mithramycin gene cluster (24), RemG and RemH from the resistomycin gene cluster (25), and the recently identified BenF and NapB2 from the benastatin (26) and napyradiomycin (27) gene clusters, respectively. All these C-methyltransferases modify the respective aromatic aglycons with unique regiospecificity. Therefore, identification and reconstitution of the dedicated 6-methyltransferase in the oxytetracycline biosynthetic pathway will add to the molecular toolbox of tailoring enzymes.

Gene disruption through homologous recombination followed by structure elucidation of the metabolites produced by the mutants has been widely used to access enzyme functions in polyketide biosynthetic pathways. However, gene inactivation has been demonstrated to be of limited use in studying oxytetracycline biosynthesis in *S. rimosus*. Genomic deletions of *oxyK* (28) and *oxyS* (29) in the oxy gene cluster led to the recovery of truncated polyketides only (30), and the resulting phenotypes cannot be directly associated with the functions of the deleted genes. The exact causes for biosynthesis of these truncated compounds in *S. rimosus* mutants are not understood. Purification of target enzymes followed by *in vitro* biochemical characterizations are complicated by the inaccessibility of the substrates for the cyclization and methylation reactions of interest here. As a result of these technical difficulties, investigation of these early tailoring reactions can be performed through the systematic and incremental reconstitution of the pathway in a heterologous host. Using a heterologous *Streptomyces* host and shuttle vectors containing different combinations of *oxy* genes, we recently reported the biosynthesis of novel amidated polyketides by coexpressing the *oxy* minimal PKS and OxyD (16, 31). These studies verified that the amidotransferase homologue OxyD is a key enzyme in the biosynthesis of the amidated tetracycline backbone.

The main goal of this work is to elucidate the enzymes involved in formation of the tetracyclic scaffold and C-methylation of the aromatic aglycon. We compiled the minimal set of enzymes required to reconstitute 6-methylpretetramid biosynthesis in the heterologous host. These results will enable additional investigation of oxytetracycline downstream tailoring steps and lead to the rational and combinatorial biosynthesis of tetracycline analogs.

## EXPERIMENTAL PROCEDURES

### Bacterial Strains and General Techniques for DNA Manipulation—*Streptomyces coelicolor* strain CH999 (32) was used as a host for transformation of shuttle vectors. Protoplast preparation and polyethylene glycol-assisted transformation were performed as described by Hopwood et al. (33). *Escherichia coli* XL-1 Blue (Stratagene) was used for the manipulation of plasmid DNA. PCR was performed using Platinum Pfx DNA polymerase (Invitrogen). PCR products were first cloned into a pCR-Blunt vector (Invitrogen) followed by DNA sequencing. Unmethylated DNA was obtained using the methyrase-deficient strain GM2163 (New England Biolabs). T4 DNA ligase (Invitrogen) was used for ligation of restriction fragments. The cosmid clone pYT264, which harbors the entire *oxy* gene cluster (16), was used as the template for the amplification of individual *oxy* genes.

### Construction of Plasmids—Primers used to amplify the individual genes are listed in supplemental Table S4. Multicistronic cassettes were constructed using the compatible XbaI/SpeI/NheI cohesive ends for most of the genes. Different combinations of genes were introduced into pWJ35 (16) or pRM5 (32) to yield the constructs shown in Table 1.

Culture Conditions, Extractions, and Small-scale Analysis—Strains were grown on solid R5 plates with 50 mg/liter thio-strepton at 30 °C for 7–10 days. For LC/MS and analytical HPLC analysis, a well pigmented plate was chopped into fine pieces and extracted with 50 ml of ethyl acetate (EA)/methanol (MeOH)/acetic acid (89%/10%/1%). Extracts were dried over anhydrous Na₂SO₄. Solvent was removed *in vacuo*, and the residue was dissolved in 0.5 ml of dimethyl sulfoxide (Me₂SO). The polyketide products were separated by reverse-phase HPLC and detected at 280 and 410 nm using an analytical C₁₈ column (Varian Pursuit 5 μm, 250 mm × 4.6 mm) with a linear gradient of 5% acetonitrile (CH₃CN) in water (0.1% trifluoroacetic acid) to 95% CH₃CN in water (0.1% trifluoroacetic acid) over 30 min with a flow rate of 1 ml/min. HPLC retention times (*t*~R~) were as follows: WJ83T1 (1), 26.4 min; WJ83Q2 (2a/b), 17.8 min; WJ83Q3 (3), 29.0 min; WJ83Q1 (4), 16.2 min; desmethylaklalonic acid (5), 22.8 min; WJ119 (6), 22.5 min; DMAC, 20.5 min.

Spectroscopic Analysis—High resolution mass spectrometry was performed at the Vincent Coates Foundation Mass Spectrometry Laboratory at Stanford University using a Micromass Q-Tof hybrid quadrupole-time of flight LC-MS. NMR spectra were obtained on Bruker DRX-500 and DRX-600 spectrometers at the NMR facility in the Department of Chemistry and Biochemistry at UCLA. ¹H and ¹³C chemical shifts were referenced to the solvent peak (Me₂SO-*d*₆) δ 2.49 and 39.5 ppm, respectively. Standard parameters were used for one- and two-dimensional NMR experiments, which included ¹H, ¹³C, nuclear Overhauser effect, heteronuclear multiple quantum correlation (¹H,¹³C), and heteronuclear multiple-bond correlation (¹H,¹³C).

Isolation of WJ83T1 (1) from S. coelicolor CH999/pWJ83—One hundred R5 plates (3 liter) streaked with CH999/pWJ83 were incubated at 30 °C for 10 days. The plates were chopped into fine pieces and extracted with 3 liter of EA/MeOH/acetic acid (94%/5%/1%). The solvent was removed *in vacuo*, and the residue was partially dissolved in 100 ml of MeOH. The methanol-insoluble fraction collected by filtration was dissolved in 5 ml Me₂SO, filtered, and purified with a preparative reverse-phase HPLC column (SunFire Prep C18 OBD column, 5 μm, 19 × 50 mm). A 20–40% CH₃CN in water (10 mM triethylamine) gradient was used over 30 min with a flow rate of 3 ml/min to give purple fractions containing 1 with a retention time of 13.8 min. The eluent was extracted with ethyl acetate and dried in *vacuo* to give pure solid 1 (approximate yield of 55 mg/liter). WJ83T1 (1): HR-ESIMS *m/z* = 364.0450 (C₁₉H₁₀NO₇ [M-H]⁻, 364.0457 calculated); UV (CH₃CN/H₂O/trifluoroacetic acid) λ~max~, 278 and 500 nm; see supplemental Table S1 for NMR spectral data.

Isolation of WJ83Q1–3 from S. coelicolor CH999/pWJ83—From the same extract as above, the methanol soluble fraction was dried *in vacuo* and redissolved in 5 ml of EA. The residue was first chromatographed on a normal-phase silica gel (150 g) column and eluted with ethyl acetate in 1% acetic acid to give two impure fractions containing 3 and 4 (*R*~f~ values if 0.72 and 0.3, respectively). Further purification of 3 and 4 were achieved on a preparative reverse-phase HPLC column (XTerra Prep MS C18 OBD column, 5 μm, 19 × 50 mm), and a 30–80% CH₃CN in water (0.1% trifluoroacetic acid) gradient was used over 45 min with a flow rate of 3 ml/min. 4 and 3 crystallized as pure yellow/orange needles after elution from the column, with retention times of 8.6 and 17.8 min and approximate yields of 5 and 2 mg/liter, respectively. The MeOH-soluble fraction of the crude extract was also directly applied to the preparative reverse-phase HPLC column to give pure 2 (approximate yield of 8 mg/liter with retention time of 11.2 min at the same gradient as above). WJ83Q2 (2): HR-ESIMS *m/z* = 338.0662 (C₁₈H₁₂NO₆ [M-CO₂-H]⁻, 338.0665 calculated); UV (CH₃CN/H₂O/trifluoroacetic acid) λ~max~, 228, 240, 257, 289, and 430 nm. WJ83Q3 (3): HR-ESIMS *m/z* = 319.0610 (C₁₉H₁₁O₅ [M-H]⁻, 319.0607 calculated); UV (CH₃CN/H₂O/trifluoroacetic acid) λ~max~, 240, 267, 284, and 412 nm. WJ83Q1 (4): HR-ESIMS *m/z* = 296.0565 (C₁₆H₁₀NO₅ [M-H]⁻, 296.0559 calculated); UV (CH₃CN/H₂O/trifluoroacetic acid) λ~max~: 228, 258, 290, and 430 nm; see supplemental Tables S2 and S3 for NMR spectral data.

Isolation of WJ119 (6) from S. coelicolor CH999/pWJ119—Seventy R5 plates (2 liter) streaked with the CH999/pWJ119 were incubated at 30 °C for 8 days. The plates were extracted with 2 liter of EA/MeOH/acetic acid (94%/5%/1%). The solvent was removed *in vacuo*, and the residue was first chromatographed on a normal-phase silica gel (100 g) column and eluted with EA in 1% acetic acid to give impure yellow fractions containing 6 (*R*~f~ value 0.38). Further purification of 6 was achieved on a preparative reverse-phase HPLC column (SunFire Prep C18 OBD column, 5 μm, 19 × 50 mm). A 5–95% CH₃CN in water (10 mM triethylamine) gradient was used over 30 min with a flow rate of 3 ml/min to give yellow fractions containing 6 with a retention time of 12.4 min. The pH of the eluent was adjusted to ~2 with 6 N HCl, extracted with EA, and dried in *vacuo* to give pure solid 6 (approximate yield of 58 mg/liter). WJ119 (6): HR-ESIMS *m/z* = 380.0780 (C₂₀H₁₄NO₇ [M-H]⁻, 380.0770 calculated); UV (CH₃CN/H₂O/trifluoroacetic acid) λ~max~, 220, 268, 300, and 435 nm; see supplemental Table S1 for NMR spectral data.

Enzyme Purification and in Vitro Assays—Gris ARO/CYC, OxyN, OxyI, and holo-OxyC were expressed and purified from *E. coli* strains BL21(DE3)/pWJ184, BL21(DE3)/pWJ181, BL21(DE3)/pWJ189, and BAP1/pWJ66, respectively (see supplemental data for details). ActIII ketoreductase and malonyl-CoA:ACP acyltransferase were expressed in *E. coli* and purified as described previously (34, 35). OxyAB was purified from CH999/pWJ131. The *in vitro* assays converting malonyl-CoA into aromatic polyketides were performed in reaction buffer (100 mM NaH₂PO₄ (pH 7.4), 2 mM MgCl₂, and 10% glycerol). Each reaction mixture contained 2 mM malonyl-CoA, 300 nM malonyl-CoA:ACP acyltransferase, 20 μM holo-OxyC, and 10 μM OxyAB in a final volume of 100 μl. All other enzymes were added to final concentrations between 20 and 100 μM. NADPH (2 mM final concentration) was added as needed. Reactions were initiated by adding the substrate malonyl-CoA and incubated at 30 °C for 2 h. The reaction mixture was extracted with EA/acetic acid (99%/1%), and the organic phase was dried in a SpeedVac and redissolved in 20 μl of Me₂SO and analyzed with HPLC.

Tetracycline Biosynthesis

TABLE 1  
Plasmid constructions and resulting polyketide products  
*S. coelicolor* strain CH999 was used as the host for polyketide biosynthesis. Each plasmid is derived from pRM5.

| Plasmid   | Genes                          | Major product       | Minor products            | Reference |
|-----------|--------------------------------|---------------------|---------------------------|-----------|
| pYT319    | oxyABC                         | SEK15               | SEK15b                   | (24)      |
| pWJ85     | oxyABCD                        | SEK15, WJ85         | WJ85b                    | (43)      |
| pWJ35     | oxyABCDJ                       | WJ35, RM20b         | RM20, RM20c              | (24)      |
| pWJ83     | oxyABCDJKN                     | 1                   | 2-5, DMAC                | This work |
| pWJ196    | oxyABCDJK/dpsY                 | 1                   | 2-5, DMAC                | This work |
| pWJ83c    | oxyABCJKN                      | 5                   | DMAC                     | This work |
| pWJ90     | oxyABCDJNI                     | 1                   | 2-5, DMAC                | This work |
| pWJ120    | oxyABCDJKNT                    | 1                   | 2-5, DMAC                | This work |
| pWJ119    | oxyABCDJKNF                    | 6                   | 2-5                      | This work |
| pWJ123    | actI-III/actVII/actIV/oxyF      | DMAC                | Aloesaponarin II          | This work |
| pWJ180    | pdmABCD/oxyF                   | TW95a,b             |                           | This work |
| pWJ190    | tcmKLMN/oxyF                   | TcmF2               | RM80                     | This work |

RESULTS  

We constructed a series of plasmids to identify the minimal set of enzymes required to afford the key intermediate 6-methylpretetramid. The genes of interest were systematically introduced into the *E. coli/S. coelicolor* shuttle vector as shown in Table 1. The plasmids were transformed into the engineered *S. coelicolor* host CH999 (32), and the production of polyketides by the host/vector pairs were assayed by LC/MS.

Functional Study of OxyN—Our previous study had identified that OxyABCDJ were essential and sufficient to produce the amidated decaketide tetracycline backbone with good yield in CH999 (16). Furthermore, OxyK was identified by Petkovic *et al.* (28) and confirmed in our laboratory (36) to be the bifunctional cyclase/dehydratase that is responsible for formation of the D ring through C-7/C-12 cyclization (Fig. 2). OxyN showed strong sequence similarity to DpsY (37) and MtmY (38), which catalyze cyclization of the second ring during daunorubicin and mithramycin biosynthesis, respectively. We tested the function of OxyN through coexpression with the oxy minimal PKS, OxyJ, OxyK, and OxyD in plasmid pWJ83.

CH999/pWJ83 grown on R5 agar developed rust-colored, non-diffusible pigmentation in the mycelia. Several new biosynthetic products were detected from the CH999/pWJ83 extract and were partitioned with methanol and purified separately. The methanol-insoluble fractions contained the most abundant compound produced by this strain. WJ83T1 (1) was isolated at ~55 mg/liter after reversed-phase HPLC purification. The UV spectrum of the compound in dilute acid showed absorption maxima at 278 nm and a broad peak at 500 nm that is indicative of an extended chromophore. The compound exhibited a bathochromatic shift and developed a deep purple color with the addition of NaOH (*λ*~max~ = 566 nm). HR-ESIMS indicated the molecular formula of C~19~H~11~NO~7~ for 1, consistent with the molecular composition of an amidated decaketide (Fig. 2). The ^13^C NMR spectrum (supplemental Table S1) showed 19 signals composed of 5 aromatic C-H residues and 14 quaternary carbons, suggesting a completely aromatized scaffold. Two downfield signals (*δ*~C-6~ = 183.1 and *δ*~C-13~ = 184.5 ppm) were indicative of conjugated carbonyls. The ^1^H NMR spectrum contained two broad signals corresponding to the free amide protons (*δ* = 7.87 and 9.67 ppm). Strong ^1^H,^13^C heteronuclear multiple-bond correlations of H-8/C-12, H-4/C-14, H-4/C-16, H-2/C-16, and H-2/C-18 together with the nuclear Overhauser effect coupling of H-2/H-4 established the polyhydroxyl naphthacenedione structure of 1. Accumulation of the linear, tetracyclic 1 as a major product in CH999/pWJ83 demonstrates that only two cyclases (OxyK and OxyN) were necessary to cyclize all four rings in tetracycline. Oxidation of ring C to a quinone was likely a spontaneous, nonenzymatic modification that is widely observed for aromatic polyketides (32, 39–41). LC-MS analysis indicated the likely presence of trace amount of the acetyl-primed version of 1 (*m/z* 364.1) in the extract of CH999/pWJ83.

Numerous anthraquinone compounds were recovered from the methanol-soluble fraction of the CH999/pWJ83 extract. Reverse-phase HPLC purification yielded an orange-pigmented metabolite WJ83Q2 (2a) at ~8 mg/liter (Fig. 2). WJ83Q2 has strong UV absorptions at 228, 257, and 430 nm and has a molecular formula of C~19~H~14~NO~8~. The ^1^H spectrum showed four aromatic proton signals, two methylene singlets and two broad ^1^H signals at *δ*~H~ = 7.12 and 7.52 ppm identified as amide protons. 2a is the amidated version of the well-known metabolite aklanonic acid, which is an intermediate in the anthracycline biosynthetic pathways (Fig. 1) (42). NMR analysis in Me~2~SO-*d*~6~ also showed both the C-17/C-18 keto (2a) and enol (2b) isoforms were present with an equilibrium ratio of 2 to 3. Furthermore, after 2 days at room temperature in Me~2~SO-*d*~6~, 2a and 2b were completely decarboxylated to 2c and 2d as indicated by NMR and LC-MS analysis (supplemental Table S2). The acetyl-primed version of 2, desmethylaklanoic acid (5), was also purified from the methanol-soluble fraction (~4.5 mg/liter).

Two minor orange-pigmented metabolites, WJ83Q3 (3) and WJ83Q1 (4), were isolated from CH999/pWJ83 (Fig. 3). Compound 3 (2 mg/liter) had a molecular composition of C~19~H~12~O~5~, which indicated that it may be an acetyl-primed decaketide that has undergone loss of one CO~2~. NMR spectra (supplemental Table S3) confirmed regiospecific cyclization of D, C, and B rings and a unique O-15/C-19 cyclization to yield a fused γ-pyrone. The 4H-anthra[1,2-b]-pyran-4,7,12-trione aglycon is present in the well-known pluramycin family of polyketides (43, 44), but identification of 3 in the extract of CH999/pWJ83 at an appreciable level was unexpected. A similar compound, pre-mithramycinone H was previously recovered by Rohr and coworkers (45) from an engineered strain of *Streptomyces argillaceus* in which a foreign oxygenase TcmH was coexpressed with the mithramycin biosynthetic enzymes.

Tetracycline Biosynthesis

![Chemical Structures and Pathways]

FIGURE 2. The biosynthetic pathway for 1, 2, and 6. Additional metabolite (5) isolated from the strains discussed in this study is boxed in dash lines. Compounds 3 and 4 are shown in Fig. 3. The block arrows indicate the oxy biosynthetic pathway. The thin arrows indicate shunt pathways or oxidation reactions observed in this study. The filled block arrow indicates a collection of downstream tailoring reactions. Pathway A and pathway B emerging from anthracene carbonyl-S-OxyC lead to formation of naphthacene and anthracene carboxylic acids, respectively. During tetracycline biosynthesis, the spontaneous pathway A dominates. OxyC is the acyl-carrier protein of the oxy pathway.

Compound 4 (~5 mg/liter) has the molecular formula C₁₆H₁₁NO₅ and the structure of the novel amidated anthraquinone derived from an amidated nonaketide backbone (9 C═O) was determined as shown in Fig. 3 (see supplemental Table S3 for NMR data). The presence of an amidated, truncated shunt product is not surprising and further confirmed that OxyAB may not be able to exert complete chain length control. Hunter and co-workers (28, 29) have reported the isolation of truncated polyketides from *S. rimosus* mutant strains containing genomic deletions in *oxyK* and *oxyS*. However, in contrast to those studies, the full-length products 1–3 were the dominant metabolites produced by CH999/pWJ83. Other truncated polyketides produced by this strain include the acetyl-primed octaketide 3,8-dihydroxy-methylanthraquinone carboxylic acid (DMAC) (32), which was recovered at yield <0.5 mg/liter.

We then constructed pWJ196 in which *oxyN* was replaced with *dpsY* from the daunorubicin biosynthetic pathway. CH999/pWJ196 produced the identical distribution of polyketides with 1 being the dominant product. This is unexpected since DpsY was previously demonstrated to only catalyze cyclization of the second and possibly the third rings during daunorubicin synthesis (46).

To compare the cyclization events between acetyl-derived and malonamoyl-derived polyketides, we constructed plasmid pWJ83c in which *oxyD* was removed from pWJ83. CH999/pWJ83c produced 5 as the major product, whereas no trace of tetracyclic compounds can be detected by LC-MS analysis. This provided direct evidence that the terminal amide unit, which is installed by OxyD, is important for the final cyclization step leading to the formation of 1. To further demonstrate that acetyl-derived decaketides cannot form tetracyclic aglycons with only two cyclases, we performed in vitro assays with purified *oxy* PKS components. OxyAB was purified from CH999/pWJ131 with a yield of ~2 mg/liter (supplemental data). Holo-OxyC and OxyN were expressed and purified from *E. coli* expression strains. OxyJ and OxyK formed inclusion bodies when expressed in *E. coli*. As a substitute, the functionally equivalent ketoreductase ActIII (16, 34) and cyclase Gris ARO

Tetracycline Biosynthesis

![Chemical Structure 1](image1.png)

---

![Chemical Structure 2](image2.png)

FIGURE 3. Biosynthetic pathways for 3 and 4. Both compounds were isolated from CH999/pWJ83 as minor metabolites. 3 is an acetyl-primed decaketide, and 4 is an amidated nonaketide.

were expressed, purified, and assayed (47). Malonyl-CoA and malonyl-CoA:ACP acyltransferase were added to different combinations of PKS enzymes, and the polyketide products were extracted and analyzed by HPLC (supplemental Fig. S1). The minimal PKS (OxyABC) produced SEK15, whereas the sequential addition of ActIII and Gris ARO led to the synthesis of RM20b and SEK43 as major products, respectively. Addition of OxyN led to formation of 5, whereas no trace of tetracyclic compounds can be detected by LC-MS analysis, consistent with in vivo results.

**Functional Study of OxyI—**OxyI was originally assigned (16) as the fourth ring cyclase during oxytetracycline biosynthesis based on its high sequence homology to MtmX in the mithramycin biosynthetic pathway. To examine if coexpression of OxyI can lead to complete cyclization of the decaketides characterized in CH999/pWJ83, we constructed pWJ90, in which *oxyI* was inserted immediately downstream of *oxyN*. The LC-MS profile of the extract of CH999/pWJ90 was identical to that of CH999/pWJ83 with all the aforementioned products present at comparable yields, and the ratio of 1 to 2 remained at ~7:1. These results indicate that OxyI has no apparent cyclase functions in the oxy biosynthetic pathway, and cyclization of the fourth ring in tetracycline can take place in the presence of only two cyclases.

**Functional Study of OxyF—**OxyF and OxyT are the two putative S-adenosylmethionine-dependent methyltransferases present in the *oxy* gene cluster (16). To identify the enzyme responsible for C-6 methylation that affords 6-methylpretetramid, we constructed shuttle vectors in which the gene encoding OxyT or OxyF was added to pWJ83 to yield pWJ120 and pWJ119, respectively.

Extraction of CH999/pWJ120 yielded the same metabolite profile as that of CH999/pWJ83, indicating OxyT has no apparent function in this strain. In contrast, the extract of CH999/pWJ119 contained <1% of 1 observed in CH999/pWJ83 as a result of OxyF coexpression, with the accompanying appearance of a new major metabolite WJ119 (6), which had a distinct UV spectrum with λ<sub>max</sub> at 435 nm. The new compound was highly soluble in methanol and was isolated (58 mg/liter) after normal-phase silica gel chromatography and reversed-phase HPLC purification. HR-ESIMS indicated the molecular formula of C<sub>20</sub>H<sub>15</sub>NO<sub>7</sub> for 6, which is consistent with the introduction of an extra methyl group to 1. Heteronuclear single quantum correlation experiments (supplemental Table S1) confirmed the presence of a new methyl group (δ<sub>C-20</sub> = 39.3, δ<sub>H-20</sub> = 1.48 ppm), whereas the heteronuclear multiple-bond correlation of H-20 to C-5, C-6, and C-7 established the carbon-carbon connectivity between the methyl group (C-20) and C-6 in ring C. The chemical shift of C-6 (δ<sub>C-6</sub> = 69.8 ppm) indicates it is a quaternary sp<sup>3</sup> carbon atom connected to an OH group and is evidence for a non-aromatic ring C. We concluded that 6 is the oxidized form of 6-methylpretetramid with a hydroxyl group introduced at C-6 (Fig. 2), likely as a result of air oxidation during purification. Biosynthesis of 6 by CH999/pWJ119 proved the involvement of OxyF in the C-6 methylation reaction. The anthraquinones 2–5 were also present in the extract of CH999/pWJ119 at the same levels, whereas LC-MS indicated the possible presence of the acetyl-primed version of WJ119 in very trace amounts.

To probe whether OxyF can be combinatorially combined with other aromatic PKSs toward the synthesis of novel, methylated polyketides, we coexpressed OxyF in CH999 strains producing DMAC (*act* PKS, CH999/pWJ123) (32), dodecaketides (*pdm* PKS, CH999/pWJ180) (48), or tetracenomycin F2 (*tcm* PKS, pWJ190) (49). No new metabolites were detected in those strains, indicating OxyF has high specificity toward its natural substrates in the *oxy* biosynthetic pathway.

## DISCUSSION

In this work we successfully constructed heterologous strains that synthesized key intermediates of the oxytetracycline biosynthetic pathway. This work is an important step in understanding and engineering tetracycline biosynthesis because 1) the presence of the carbamoyl moiety in 1 and 6 further validates the essential role of OxyD for tetracycline biosynthesis, 2) complete cyclization of the amidated decaketide backbone confirms the role of the cyclases in the *oxy* cluster, 3) regioselective C-6 methylation specifies the function of the C-methyltransferase, OxyF, and 4) the strain CH999/pWJ119 is a suitable platform for systematic and combinatorial analysis of additional *oxy* downstream tailoring enzymes.

The various metabolites recovered from CH999/pWJ83 and CH999/pWJ119 provide insights into the possible mechanisms of these early tailoring steps. Formation of ring C clearly proved that OxyN catalyzes the C-5/C-14 aldol reaction. No additional enzyme was needed to completely cyclize ring B, consistent with the biosynthetic mechanism of most aromatic polyketides, such as alkanonic acid and DMAC (37, 46). Surprisingly, cyclization of the ring A of amidated polyketides to yield pretetramid occurred without coexpression of additional cyclases. This is in contrast to the assembly of alkylacyl-primed, tetracyclic polyketides (Fig. 1), in which cyclization of the fourth ring is catalyzed by dedicated enzymes (20, 22). This was in fact observed in the extracts of CH999/pWJ83 and CH999/pWJ83c, where acetyl-primed deca- ketides did not undergo fourth ring cyclization (see 3 and 5).

Amide-dependent formation of ring A is unlikely to be catalyzed by OxyN, since the replacement of OxyN with DpsY led to the exact same product distribution. It is difficult to rationalize

that DpsY from a propionyl-primed pathway has evolved the required substrate specificity toward malonamyl-primed polyketides. We, therefore, propose cyclization of ring A occurs spontaneously only in the presence of the terminal amide. As shown in Fig. 2, intramolecular C-1/C-18 aldol condensation of the anthracene carbonyl-S-OxyC followed by concomitant release of the OxyC-SH affords the naphthacene compound pretetramid (pathway A), whereas spontaneous hydrolysis of the tricyclic intermediate yields the anthracene carboxylic acid (pathway B). The ratio of tetracyclic to tricyclic compounds is, thus, dictated by the relative rates of these two pathways *in vivo*. We hypothesize that installment of the amide group at C-19 may stabilize the anthracene carbonyl-S-OxyC complex against spontaneous hydrolysis, possibly by formation of hydrogen bonds with amino acid residues on helix 2 of OxyC. The polyketide-OxyC interaction increases the relative rate of pathway A and leads to the synthesis of the key intermediate pretetramid. As a result, a high ratio of 1 to 2 (~7:1) was observed in the extract of CH999/pWJ83. In the absence of the stabilizing polyketide-OxyC interactions, as in the alkylacyl-primed polyketides, hydrolysis of the thioester bond occurs readily, and only trace amounts of the tetracyclic compounds can be synthesized. The same polyketide-OxyC interaction facilitated by the amide start unit may also account for the unusual cyclization patterns observed for the polyketides WJ35 (16) and WJ85 (31).

Our work also unequivocally proved that OxyF is the C-6 methyltransferase in the oxy biosynthetic pathway. Near complete methylation of pretetramid to 6-methylpretetramid was observed upon coexpression of OxyF, whereas the yields of tricyclic compounds 2–5 remain unchanged. We did not detect any derivatives of 2 and 5 that are methylated at the C-6 positions. From these *in vivo* results we reason the OxyF-catalyzed methylation reaction occurs on the tetracyclic scaffold, and anthracene compounds are not substrates of OxyF (Fig. 2). The proposed mechanism for OxyF is shown in Fig. 4. Base-catalyzed deprotonation of free C-13 phenolic OH leads to the generation of a C-6 carbanion. The nucleophilic C-6 then attacks the $\mathrm{CH}_{3}^{+}$ equivalent residing on S-adenosylmethionine to form the observed carbon-carbon bond. The methylation reaction must

FIGURE 4. Proposed reaction mechanism of the C-6 methyltransferase OxyF.

![Figure 4](https://i.imgur.com/1234567.png)

FIGURE 5. The observed ratios for amidated polyketides to nonamidated polyketide produced by the various heterologous host/vector pairs. As the number of tailoring enzymes is increased, the amount of amidated polyketides increases correspondingly. S. coelicolor CH999 is used as host in these studies.

![Figure 5](https://i.imgur.com/7654321.png)

Tetracycline Biosynthesis

proceed before the spontaneous oxidation of the C ring, as installment of a quinone at C-6 deactivates formation of the nucleophilic C-6.

Another interesting finding is that there is significant increase in the relative yields of malonamyl-derived to acetyl-derived metabolites when more tailoring enzymes are coexpressed with the oxy minimal PKS (Fig. 5). With only the extended oxy minimal PKS (OxyABCD, in CH999/pWJ85), equal amounts of amidated (WJ85) to non-amidated (SEK15) polyketides were isolated (31). Insertion of the C-9 reductase OxyJ increases the ratio to ~4 with WJ35 becoming the major compound over RM20/b/c (16). The addition of the first ring cyclase OxyK further increases the amount of amidated polyketides (36). In this work, when OxyN and OxyF were coexpressed, amidated polyketides were produced ~10 times the level of non-amidated polyketides. These systematic increases in the amount of malonamyl-derived compounds strongly suggest the downstream enzymes enforce significant influence on the incorporation of the amide functionality during polyketide synthesis. Although we confirmed the essential role of OxyD in introducing the amide group during tetracycline synthesis, the exact timing and substrates of the reaction have not been determined.

The striking observation in Fig. 5 can be explained by several possible models; 1) increased activity of OxyD in the presence of tailoring enzymes, which results in elevated levels of the malonamyl primer unit; 2) increased selectivity of the minimal PKS toward the malonamyl starter unit through unknown protein-protein interactions with the tailoring enzymes (this was previously observed for the enterocin minimal PKS where coexpression of the endogenous ketoreductase is essential for the synthesis of benzoyl-primed polyketides (36)); 3) the incorporation of the malonamyl primer unit favors turnover of downstream tailoring steps. Our investigation of cyclization and methylation reactions in this work showed that the presence of amide unit is necessary for the complete cyclization of the tetracycline aglycon, which then serves as substrates for next tailoring enzyme OxyF. The final mechanism that can satisfactorily explain the trend in Fig. 5 may be a combination of the models above, and we are currently investigating this intriguing phenomenon.

---

Acknowledgments—The NMR instrumentation at UCLA is supported in part by National Science Foundation Grants CHE9974928 and CHE0116853. We thank Prof. Mike Jung for helpful discussions.

---

REFERENCES

1. Kim, E. S., Bibb, M. J., Butler, M. J., Hopwood, D. A., and Sherman, D. H. (1994) *Gene* (Amst.) **141**, 141–142
2. Rawlings, B. J. (1999) *Nat. Prod. Rep.* **16**, 425–484
3. Hertweck, C., Luzhetskyy, A., Rebets, Y., and Bechthold, A. (2007) *Nat. Prod. Rep.* **24**, 162–190
4. Rix, U., Fischer, C., Remsing, L. L., and Rohr, J. (2002) *Nat. Prod. Rep.* **19**, 542–580
5. McCormick, J. R. D., and Jensen, E. R. (1965) *J. Am. Chem. Soc.* **87**, 1794–1795
6. McCormick, J. R. D., and Jensen, E. R. (1968) *J. Am. Chem. Soc.* **90**, 7126–7127
7. McCormick, J. R. D., and Jensen, E. R. (1969) *J. Am. Chem. Soc.* **91**, 206–208
8. McCormick, J. R. D., Jensen, E. R., Arnold, N. H., Corey, H. S., Joachim, U. H., Johnson, S., Miller, P. A., and Sjolande, N. O. (1968) *J. Am. Chem. Soc.* **90**, 7127–7128
9. McCormick, J. R. D., Jensen, E. R., Johnson, S., and Sjolande, N. O. (1968) *J. Am. Chem. Soc.* **90**, 2201–2202
10. McCormick, J. R. D., Joachim, U. H., Jensen, E. R., Johnson, S., and Sjolande, N. O. (1965) *J. Am. Chem. Soc.* **87**, 1793–1794
11. McCormick, J. R. D., Sjolander, N. O., and Johnson, S. (1963) *J. Am. Chem. Soc.* **85**, 1692–1694
12. Butler, M. J., Binnie, C., Hunter, I. S., Sugden, D. A., and Warren, M. (1990) *Dev. Ind. Microbiol.* **31**, 41–50
13. Hunter, I. S. (2002) in *Microbial Secondary Metabolites: Biosynthesis Genetics and Regulation* (Fierro, F., and Martin, J. F., eds) pp. 141–166 Research Signpost, Lucknow, India
14. McDowall, K. J., Doyle, D., Butler, M. J., Binnie, C., Warren, M., and Hunter, I. S. (1990) in *Genetics and Product Formation in Streptomyces* (Noack, D., Krugel, H., and Baumberg, S., eds), pp. 105–116 Plenum Press, New York
15. Petkovic, H., Cullum, J., Hraneli, D., Hunter, I. S., Peric-Concha, N., Pigac, J., Thamchaipenet, A., Vujaklija, D., and Long, P. F. (2006) *Microbiol. Mol. Biol. Rev.* **70**, 704–728
16. Zhang, W. J., Ames, B. D., Tsai, S. C., and Tang, Y. (2006) *Appl. Environ. Microbiol.* **72**, 2573–2580
17. Rohr, J., Weissbach, U., Beninga, C., Kunzel, E., Siems, K., Bindseil, K. U., Lombó, F., Prado, L., Brana, A. F., Mendez, C., and Salas, J. A. (1998) *Chem. Commun.* **437–438**
18. Kunzel, E., Wohlert, S. E., Beninga, C., Haag, S., Decker, H., Hutchinson, C. R., Blanco, G., Mendez, C., Salas, J. A., and Rohr, J. (1997) *Chem.-Eur. J.* **3**, 1675–1678
19. Hautala, A., Torkkell, S., Raty, K., Kunnari, T., Kantola, J., Mantsala, P., Hakala, J., and Ylihonko, K. (2003) *J. Antibiot.* (Tokyo) **56**, 143–153
20. Kendrew, S. G., Katayama, K., Deutsch, E., Madduri, K., and Hutchinson, C. R. (1999) *Biochemistry* **38**, 4794–4799
21. Kantola, J., Kunnari, T., Hautala, A., Hakala, J., Ylihonko, K., and Mantsala, P. (2000) *Microbiology* **146**, 155–163
22. Thompson, T. B., Katayama, K., Watanabe, K., Hutchinson, C. R., and Rayment, I. (2004) *J. Biol. Chem.* **279**, 37956–37963
23. Hutchinson, C. R. (1997) *Chem. Rev.* **97**, 2525–2536
24. Lozano, M. J., Remsing, L. L., Quiros, L. M., Brana, A. F., Fernandez, E., Sanchez, C., Mendez, C., Rohr, J., and Salas, J. A. (2000) *J. Biol. Chem.* **275**, 3065–3074
25. Jakobi, K., and Hertweck, C. (2004) *J. Am. Chem. Soc.* **126**, 2298–2299
26. Xu, Z., Schenk, A., and Hertweck, C. (2007) *J. Am. Chem. Soc.* **129**, 6022–6030
27. Winter, J. M., Moffitt, M. C., Zazopoulos, E., McAlpine, J. B., Dorrestein, P. C., and Moore, B. S. (2007) *J. Biol. Chem.* **282**, 16362–16368
28. Petkovic, H., Thamchaipenet, A., Zhou, L. H., Hraneli, D., Raspor, P., Waterman, P. G., and Hunter, I. S. (1999) *J. Biol. Chem.* **274**, 32829–32834
29. Peric-Concha, N. A., Borovicka, B., Long, P. F., Hrnueli, D., Waterman, P. G., and Hunter, I. S. (2005) *J. Biol. Chem.* **280**, 37455–37460
30. Deseo, M. A., Hunter, S., and Waterman, P. G. (2005) *J. Antibiot.* (Tokyo) **58**, 822–827
31. Zhang, W., Watanabe, K., Wang, C. C., and Tang, Y. (2006) *J. Nat. Prod.* **69**, 1633–1636
32. McDaniel, R., Ebert-Khosla, S., Hopwood, D. A., and Khosla, C. (1993) *Science* **262**, 1546–1550
33. Hopwood, D., Bibb, M. J., Chater, K. F., Kieser, T., Bruton, C. J., Kieser, H. M., Lydiate, D. J., Smith, C. P., Ward, J. M., and Schrempf, H. (1985) *Genetic Manipulation of Streptomyces: A Laboratory Manual*, The John Innes Foundation, Norwich, UK
34. Korman, T. P., Hill, J. A., Vu, T. N., and Tsai, S. C. (2004) *Biochemistry* **43**, 14529–14538
35. Summers, R. G., Ali, A., Shen, B., Wessel, W. A., and Hutchinson, C. R. (1995) *Biochemistry* **34**, 9389–9402
36. Hertweck, C., Xiang, L., Kalaitzis, J. A., Cheng, Q., Palzer, M., and Moore, B. S. (2004) *Chem. Biol.* **11**, 461–468
37. Lomovskaya, N., Doi-Katayama, Y., Filippini, S., Nastro, C., Fonstein, L., Gallo, M., Colombo, A. L., and Hutchinson, C. R. (1998) *J. Bacteriol.* **180**,

2379–2386

38. Prado, L., Lombo, F., Brana, A. F., Mendez, C., Rohr, J., and Salas, J. A. (1999) *Mol. Genet.* 261, 216–225

39. Tang, Y., Lee, T. S., and Khosla, C. (2004) *PLoS Biol.* 2, 227–238

40. McDaniel, R., Ebert-Khosla, S., Hopwood, D. A., and Khosla, C. (1995) *Nature* 375, 549–554

41. Yu, T. W., Shen, Y., McDaniel, R., Floss, H. G., Khosla, C., Hopwood, D., and Moore, B. S. (1998) *J. Am. Chem. Soc.* 120, 7749–7759

42. Eckardt, K., Tresselt, D., Schumann, G., Ihn, W., and Wagner, C. (1985) *Antibiot. (Tokyo)* 38, 1034–1039

43. Bililign, T., Hyun, C. G., Williams, J. S., Czisny, A. M., and Thorson, J. S. (2004) *Chem. Biol.* 11, 959–969

44. Schumacher, R. W., Davidson, B. S., Montenegro, D. A., and Bernan, V. S. (1995) *J. Nat. Prod.* 58, 613–617

45. Lombo, F., Kunzel, E., Prado, L., Brana, A. F., Bindseil, K. U., Frevert, J., Bearden, D., Mendez, C., Salas, J. A., and Rohr, J. (2000) *Angew. Chem. Int. Ed. Engl.* 39, 796–799

46. Wohlert, S. E., Wendt-Pienkowski, E., Bao, W., and Hutchinson, C. R. (2001) *J. Nat. Prod.* 64, 1077–1080

47. Zawada, R. J., and Khosla, C. (1999) *Chem. Biol.* 6, 607–615

48. Lee, T. S., Khosla, C., and Tang, Y. (2005) *J. Am. Chem. Soc.* 127, 12254–12262

49. Bao, W., Wendt-Pienkowski, E., and Hutchinson, C. R. (1998) *Biochemistry* 37, 8132–8138
