
Molecular Microbiology (2007) 65(5), 1321–1333

doi:10.1111/j.1365-2958.2007.05869.x  
First published online 31 July 2007

# Identification of subtilisin, Epr and Vpr as enzymes that produce CSF, an extracellular signalling peptide of *Bacillus subtilis*

Sara Lanigan-Gerdes,$^{1}$ Alek N. Dooley,$^{2,3}$ Kym F. Faull$^{2,3,4}$ and Beth A. Lazazzera$^{1,4*}$

$^{1}$ Department of Microbiology, Immunology and Molecular Genetics, $^{2}$The Pasarow Mass Spectrometry Laboratory, The NPI-Semel Institute for Neuroscience and Human Behavior, $^{3}$Department of Psychiatry and Biobehavioral Sciences, and $^{4}$Molecular Biology Institute, University of California Los Angeles, 405 Hilgard Avenue, Los Angeles, CA 90095, USA.

## Summary

Cell–cell communication regulates many important processes in bacteria. Gram-positive bacteria use peptide signals for communication, such as the Phr pentapeptides of *Bacillus subtilis*. The Phr pentapeptides are secreted with a pro domain that is cleaved to produce an active signalling peptide. To identify the protease(s) involved in production of the mature Phr signalling peptides, we developed assays for detecting cleavage of one of the *B. subtilis* Phr pentapeptides, CSF, from the proCSF precursor. Using both a cellular and a mass spectrometric approach, we determined that a sigma-H-regulated, secreted, serine protease(s) cleaved proCSF to CSF. Mutants lacking the three proteases that fit these criteria, subtilisin, Epr and Vpr, had a defect in CSF production. Purified subtilisin and Vpr were shown to be capable of processing proCSF as well as at least one other Phr peptide produced by *B. subtilis*, PhrA, but they were not able to process the PhrE signalling peptide of *B. subtilis*, indicating that there are probably other unidentified proteases involved in Phr peptide production. Subtilisin, Epr and Vpr are members of the subtilisin family of proteases that are widespread in bacteria, suggesting that many bacterial species may be capable of producing Phr signalling peptides.

## Introduction

Quorum sensing is a widespread form of bacterial communication in which signalling molecules are secreted

and elicit a cellular response when the signalling molecule reaches a critical environmental concentration (Waters and Bassler, 2005). A wide variety of bacterial species use quorum sensing to regulate many important processes including extracellular enzyme production, antibiotic production, virulence, biofilm formation and genetic exchange (Winzer and Williams, 2001; Kleerebezem, 2004; Parsek and Greenberg, 2005; Waters and Bassler, 2005). Thus, bacteria can conserve expression of the genes necessary for these processes until there is a sufficient density of bacteria present to efficiently carry out that process or until the diffusion rate in the environment is sufficiently low to prevent loss of secreted products (Redfield, 2002; Hense *et al.*, 2007).

Gram-positive bacteria use peptides for quorum sensing (Lyon and Novick, 2004). One signalling peptide that contributes to quorum sensing in the Gram-positive bacterium *Bacillus subtilis* is the Competence and Sporulation Factor (CSF) (Pottathil and Lazazzera, 2003). CSF is a secreted ERGMT pentapeptide that is transported into the cell by an oligopeptide permease when the peptide reaches a critical concentration in the environment (Lazazzera *et al.*, 1997). Once internalized, CSF interacts with intracellular receptors, two of which, RapC and RapB, belong to the tetratricopeptide repeat (TPR) family of proteins (Perego and Brannigan, 2001). CSF is able to stimulate the quorum response and sporulation by antagonizing RapC and RapB activity respectively (Perego, 1997; Core and Perego, 2003). RapC inhibits the activity of the transcription factor ComA that activates the quorum response, one part of which is genetic competence development (Solomon *et al.*, 1996; Core and Perego, 2003; Comella and Grossman, 2005). RapB dephosphorylates Spo0F, a component of the phosphorelay that activates sporulation (Tzeng *et al.*, 1998). At high concentrations CSF also inhibits ComA-controlled genes by an unknown mechanism (Lazazzera *et al.*, 1997). The gene for RapC is co-transcribed with *phrC*, which encodes a 40-amino-acid precursor protein for CSF (Lazazzera *et al.*, 1999). PhrC has a predicted signal sequence for export by the Sec machinery (Stephenson *et al.*, 2003). How secreted PhrC is processed to release the C-terminal CSF pentapeptide is unknown.

CSF is a member of a family of peptides produced by *B. subtilis* and other Gram-positive, spore-forming

Accepted 5 July, 2007. *For correspondence.* E-mail bethl@microbio.ucla.edu; Tel. (+1) 310 794 4804; Fax (+1) 310 206 5231.

© 2007 The Authors  
Journal compilation © 2007 Blackwell Publishing Ltd

1322 S. Lanigan-Gerdes, A. N. Dooley, K. F. Faull and B. A. Lazazzera

bacteria. There are eight *phr* genes encoded on the *B. subtilis* chromosome and at least four *phr* genes encoded on *B. subtilis* plasmids that are similar to *phrC* in that they encode small proteins, approximately 40 amino acids in length, with a signal sequence for export (Fig. 1) (Pottathil and Lazazzera, 2003; Hayashi *et al.*, 2006). These *phr* genes are adjacent to the genes encoding their cognate Rap. Many of these *rap/phr* pairs regulate a process induced during the transition to stationary-phase growth. For example, *rapA/phrA* and *rapE/phrE* regulate sporulation, *rapG/phrG* regulates extracellular protease production and *rapI/phrI* regulates transfer of a conjugative DNA element (Perego and Hoch, 1996; Jiang *et al.*, 2000; Ogura *et al.*, 2003; Auchtung *et al.*, 2005). Whether these *rap/phr* genes mediate quorum-sensing control or an autocrine-timing mechanism is a matter of debate (Perego and Brannigan, 2001; Pottathil and Lazazzera, 2003). The active signalling peptide produced by these *phr* genes appears to be a five-amino-acid peptide derived from proteolytic processing of the Phr proteins (Perego and Brannigan, 2001). The identity of the protease(s) that carries out this processing step is unknown.

Several *rap/phr*-like pairs have been identified on each genome of such Gram-positive endospore formers as *Bacillus cereus*, *B. thuringiensis*, *B. stearothermophilus*, *B. anthracis*, *B. halodurans*, *B. pumilus* and *Clostridium acetobutylicum* (Perego and Brannigan, 2001; Slamti and Lereclus, 2002; Pottathil and Lazazzera, 2003; Bongiorni *et al.*, 2006). However, functionally similar *rap/phr* cassettes may be more widespread than a search for genes with similarity at the primary sequence level suggests. These identified *rap/phr*-like pairs encode a protein with a TPR domain, as found in the Rap proteins, and a small protein with a signal sequence for export. The member of this family, outside *B. subtilis*, with the most well-defined function is the *plcR/papR* pair of *B. cereus* and

*B. thuringiensis* that regulates virulence (Slamti and Lereclus, 2005). Again, the active signalling peptide produced by *papR* is a pentapeptide, but the mechanism of release of the pentapeptide from PapR is unknown.

Identifying the protease that processes Phr proteins to their mature peptide form will reveal which bacteria are capable of producing Phr-type signalling peptides. In the long term, development of inhibitors that specifically target the Phr-processing protease would allow questions to be asked such as what is the effect of simultaneously shutting down all Phr peptide production and could pave the way for the development of new antimicrobials against some pathogens. To address how Phr pentapeptides are released from their precursor proteins, we developed two different but complementary *in vitro* assays for measuring cleavage of proCSF to CSF. We found that proCSF processing to CSF is carried out by secreted serine proteases that are partially dependent on an alternative sigma factor, sigma-H, for expression. We show that a mutant lacking three secreted serine proteases regulated by sigma-H, subtilisin, Epr and Vpr, produced less CSF and had less proCSF-processing activity. The role of these proteases in processing was direct, as purified subtilisin and Vpr cleaved proCSF to release CSF. A role for subtilisin and Vpr in processing other Phr peptides of *B. subtilis* is also presented. This work has important implications for how similar Phr-type signalling peptides are produced by other bacteria.

## Results

### Development of a biological assay for measuring cleavage of proCSF to CSF

To identify the protease(s) that is responsible for cleaving the Phr proteins of *B. subtilis* to their mature pentapeptide

| *PhrA* | MKSKWMSGLLLVAVGFSFTQVMVHAGETANTEGKTFHIA | ARNOT |
| --- | --- | --- |
| *PhrC* | MKLKSKLFLVICLAAAAIFTAAGVSANAEALDFHVT | ERGMT |
| *PhrE* | MKSKLFISLSAVLIGLAFFGSMYNGEMKEA | SRNVT LAPTHEFLV |
| *PhrF* | MKLKSLLLLSCLALSTVFVATTIANAPTHQIEVA | ORGMI |
| *PhrG* | MKRFLIGAVAAVILSGWFIADHQTH SQEMKVA | EKMIG |
| PhrH | MPIKKKVMMCLAVTLVFGSMSFPTLTNSGGFKEST | DRNTT YIDHSPYKLSDQKKALS |
| *PhrI* | MKISRILLAAVILSSVFSITYLQSDHNETIKVAA | DRVGA |
| PhrK | MKKLVLCVSILAVILSGVALTQLSTDPSNIQVA | ERPVG G |

<points x1="1587" y1="2510" alt="cleavage sites">cleavage sites</points>

**Fig. 1.** Alignment of the *B. subtilis* Phr proteins. The amino acid sequence of the eight Phr proteins are aligned based on the sequence (underlined) of their mature or predicted mature pentapeptide sequence. The arrows point to location of cleavage sites to release the mature peptides. The asterisks preceding the name of the Phr protein indicate Phr proteins for which there are experimental data supporting the identity of the mature peptide: PhrA (Perego, 1997), PhrC (Solomon *et al.*, 1996), PhrE (Jiang *et al.*, 2000), PhrF (Bongiorni *et al.*, 2005), PhrG (Ogura *et al.*, 2003) and PhrI (Auchtung *et al.*, 2005).

© 2007 The Authors

Journal compilation © 2007 Blackwell Publishing Ltd, *Molecular Microbiology*, **65**, 1321–1333

Identification of pro-CSF processing proteases 1323

Fig. 2. Development of an assay to detect cleavage of proCSF to CSF.

A. The sequence of synthetic proCSF is shown along with the site of cleavage that results in release of CSF (mature peptide).

B and C. proCSF was incubated with BAL941 (∆phrC), and the pmoles of CSF produced was measured using the biological assay. Data shown are representative of at least three independent experiments. Error bars represent standard error of the mean of the replicate points from one experiment. (B) proCSF was incubated with cells, which had been grown to an OD₆₀₀ of 1.0, for the amount of time indicated on the axis. (C) proCSF was incubated with serial, twofold dilutions of BAL941 cells for 40 min.

form, we developed a biological assay to monitor cleavage of a synthetic proCSF to CSF by a sensitive and simple procedure (Lazazzera et al., 1999). The synthetic proCSF corresponded to the C-terminal 15 amino acids of PhrC, the portion of PhrC that follows the signal sequence and predicted signal peptidase cleavage site (Fig. 2A). Despite the predicted signal peptidase cleavage site, Stephenson et al. (2003) could not find evidence that this site was cleaved by signal peptidases. Thus, this synthetic proCSF could correspond to the portion of PhrC that is released by a signal peptidase or is presented at the exterior surface of the membrane.

The source of the Phr-processing protease(s) was intact *B. subtilis* cells. Intact cells allowed us to specifically assay proteases that are extracytoplasmic, the predicted location of Phr processing. To obtain cells with likely high levels of the processing protease(s), the cells were grown to late exponential phase in a minimal medium, conditions under which cells have been shown to produce high levels of CSF (Lazazzera et al., 1999). The cells were of strain BAL941, which has a deletion of *phrC* so that any CSF detected in the assay has come from the exogenously added, synthetic proCSF.

Synthetic proCSF was incubated with the BAL941 cells at 37°C. Cells were then removed, and the medium was fractionated on a reverse-phase C18 column to separate proCSF from CSF. The amount of CSF that eluted from the column was then quantified by determining the level of β-galactosidase-specific activity after treatment of cells containing the ComA-controlled reporter fusion, *srfA-lacZ*, with dilutions of the eluates. The level of β-galactosidase-specific activity obtained for the column elutes was compared with that obtained with known concentrations of synthetic CSF in the linear portion of the dose–response curves to determine the amount of CSF that was produced from proCSF.

Using this cleavage assay, we observed production of CSF from proCSF. The amount of CSF increased linearly for the first 80 min of incubation, after which processing of proCSF to CSF was complete (Fig. 2B). In contrast, incubation of proCSF alone or cells alone did not result in the production of CSF (data not shown), indicating that proCSF cleavage requires a cellular factor. This assay was sensitive to the amount of processing activity added to the assay. We incubated twofold, serial dilutions of the cells with proCSF for 40 min and then assayed for the amount of CSF produced. With decreasing amount of cells incubated with proCSF there was a corresponding decrease in the level of CSF produced over a 10-fold range of cell concentrations (Fig. 2C).

### Inhibition of proCSF cleavage with a serine protease inhibitor

To further the goal of identifying the Phr-processing protease(s), we used inhibition assays to determine the type of protease that was responsible for proCSF cleavage. We treated cells with various protease inhibitors, prior to the addition of proCSF, and then assayed for CSF production (Table 1). CSF production appeared to increase slightly in the presence of pepstatin, EDTA, E64 or dichloroisocoumarin, an aspartic, metallo, cysteine and serine protease inhibitor respectively. This may be due to inhibition of proteases that are non-specifically degrading CSF or the proCSF precursor. In contrast, addition of phenylmethylsulphonyl fluoride (PMSF), a serine protease inhibitor, significantly reduced the amount of CSF that was produced. The reduction in proCSF cleavage was dose dependent, and no mature CSF could be detected when the PMSF concentration was at or above 0.1 mM (data not shown). PMSF is an irreversible serine protease inhibitor that causes sulphonylation of active site serine residues rendering them inactive. The ability of PMSF to inactivate the proCSF cleavage activity indicates that a serine protease(s) is responsible for proCSF cleavage.

© 2007 The Authors  
Journal compilation © 2007 Blackwell Publishing Ltd, *Molecular Microbiology*, **65**, 1321–1333

1324 S. Lanigan-Gerdes, A. N. Dooley, K. F. Faull and B. A. Lazazzera

Table 1. Inhibition of ProCSF cleavage by PMSF.

| Inhibitor | Type of protease inhibited | Concentration used (mM) | Relative amount of CSF produced$^a$ |
|-----------|-----------------------------|--------------------------|--------------------------------------|
| EDTA      | Metalloprotease             | 100                      | 1.69 ± 0.42                          |
| Pepstatin | Aspartic protease           | 10                       | 1.5 ± 0.39                           |
| E-64      | Cysteine protease           | 100                      | 1.36 ± 0.31                          |
| Dichloroisocoumarin | Serine protease          | 100                      | 1.77 ± 0.51                          |
| Phenylmethylsulphonyl fluoride (PMSF)   | Serine protease          | 0.1                                  | 0.04 ± 0.04                          |

a. proCSF was incubated with BAL950 ($\Delta phrC$, $\Delta opp$, $\Delta comQ$) cells, which had been grown to an OD$_{600}$ of 1.0, and the amount of CSF produced was quantified using a biological assay. The amount of CSF produced with inhibitor added was divided by the amount of CSF produced in an identical reaction without inhibitor added. The average value obtained from at least three independent experiments is shown ± the standard error of the mean.

### Determination of the subcellular location of the proCSF-processing protease(s)

Having identified the proCSF-processing protease(s) as a serine protease, we next sought to determine the cellular location of the enzyme. *B. subtilis* cultures were first separated into cells and the culture supernatant fraction that should contain secreted proteases. These culture fractions were incubated with proCSF and assayed for the amount of CSF produced using the biological assay described above (i.e., *srfA-lacZ* induction). The cells were observed to produce CSF, but the culture supernatant fraction had processing activity below our limits of detection (Fig. 3A).

As the majority of the processing activity appeared to be cell-associated, the cells were fractionated into a cytoplasm/membrane fraction and a cell-wall fraction (see Experimental procedures). Analysis of these subcellular fractions by SDS-PAGE revealed distinct protein staining patterns, indicating fractionation had been achieved (data not shown). However, Western blot analysis using antibodies directed against a membrane protein (F-type ATP synthase) and a cytoplasmic protein (ectopically expressed green fluorescent protein) indicated that the cell-wall fraction was contaminated with cytoplasm and membrane material (data not shown). These two subcellular fractions were incubated with proCSF and the level of CSF was determined using the biological assay. Only the cell-wall fraction produced CSF (Fig. 3B). Cleavage of proCSF by the cell-wall fraction was abolished by the addition of PMSF (data not shown), suggesting that the protease(s) in the cell-wall fraction cleaving proCSF is the same protease(s) on the intact cells that cleaved proCSF. Although the cell-wall fraction is contaminated with cytoplasm and membrane material, that the cell-wall fraction had activity while the cytoplasm/membrane fraction did not indicates that the protease(s) responsible for proCSF cleavage is cell wall-associated.

### Development of a mass spectrometric assay for measuring cleavage of proCSF to CSF

To confirm the results from the biological assay, we developed a liquid chromatography-tandem mass spectrometry

(LC-MS/MS) assay for CSF by recording the intensity of the transition of the doubly charged parent ion (m/z 297.2) to the singly charged fragment ion (m/z 386.5) under collisionally activated dissociation (CAD) multiple reaction monitoring (MRM) conditions. Under the prescribed chromatography conditions, synthetic CSF had a retention time of 6 min. Neither the proCSF peptide nor the cell-wall fraction of *B. subtilis* separately yielded a significant LC-MS/MS-MRM response for CSF (Fig. 3C and data not shown). In contrast, when the cell-wall fraction was incubated with proCSF, a significant MRM response was recorded at the appropriate retention time (Fig. 3C), indicating that proCSF was cleaved into the CSF pentapeptide. The CSF MRM response was abolished when cell-wall fraction was incubated with proCSF in the presence of PMSF (data not shown), indicating that a serine protease(s) enriched in the cell-wall fraction of *B. subtilis* cleaves proCSF to CSF.

### The proCSF-processing protease(s) is regulated by sigma-H

To gain greater insight into the potential identity of the Phr-processing protease(s), we sought to identify what factors regulate its production. Lazazzera *et al.* (1999) previously suggested that the alternative sigma factor, sigma-H, stimulated CSF production post-transcriptionally. One of the two known promoters for *phrC* (the gene for CSF) is recognized by sigma-H (Carter *et al.*, 1990). Thus, to confirm the post-transcriptional role of sigma-H in CSF production, we removed *phrC* from the endogenous locus and re-introduced *phrC* on the chromosome at the *thrC* locus under the control of the IPTG-inducible, $P_{spacy}$ promoter. The levels of CSF produced by this strain were significantly higher than in an isogenic strain lacking sigma-H (Fig. 4A), confirming that sigma-H has a post-transcriptional role in regulating CSF production.

We then asked whether the post-transcriptional role of sigma-H in CSF production was to stimulate expression of the proCSF-processing protease(s). Therefore, we incubated cells lacking sigma-H, BAL1190 ($\Delta sigH$, $\Delta phrC$), and cells containing sigma-H, BAL941 ($\Delta phrC$), with

© 2007 The Authors
Journal compilation © 2007 Blackwell Publishing Ltd, *Molecular Microbiology*, **65**, 1321–1333

Role of sigma-H-regulated, secreted, serine proteases in proCSF cleavage

Our experiments have indicated that there is at least one sigma-H-regulated, serine protease associated with the cell wall that cleaves proCSF to CSF. Among the genes identified as regulated, either directly or indirectly, by sigma-H (Britton et al., 2002), we could identify three genes, *aprE* (encoding subtilisin), *epr* and *vpr*, that encode serine proteases, as assessed by the Pfam database (Bateman et al., 2004), and signal sequences for export, as determined by PSORT (http://psort.hgc.jp) (Nakai and Kanehisa, 1991). To determine if one or more of these serine proteases is involved in CSF production, we made strains that lacked one or a combination of *aprE*, *epr* and *vpr* and assayed these strains for the level of endogenous CSF production, by the amount of CSF that accumulated in the culture supernatants (Fig. 5A). A strain lacking either subtilisin, Epr or Vpr showed no statistically significant decrease in CSF production as compared with the wild-type strain. Mutants lacking any two of the proteases also did not show a reduction in CSF production. We do note that there was a small increase in the level of CSF production for mutants lacking subtilisin alone or one other proteases. The reason for this is unknown, but could be due to increased levels of other proteases in the absence of subtilisin. Only the mutant lacking all three proteases showed a statistically significant defect in CSF production (*P* < 1 × 10⁻⁴). These data indicate that all three proteases are involved in CSF production and that they are redundant with each other. That the mutant lacking all three proteases still produced some CSF suggests that at least one other protease is involved in CSF production.

To further confirm that subtilisin, Epr and Vpr are involved in CSF production, we determined whether the mutant lacking all three proteases had decreased expression of a gene regulated by CSF. We measured the level of *srfA–lacZ* expression in strains containing or lacking *aprE*, *epr* and *vpr*. The mutant strain lacking all three proteases had *srfA–lacZ* expression levels fourfold less than those of the wild-type strain (Fig. 5B).

A simple model for the role of subtilisin, Epr and Vpr in CSF production is that they cleave PhrC to CSF. We tested this model by determining whether cells of the *aprE*, *epr* and *vpr* mutant strain had a defect in processing synthetic proCSF to CSF. Cells of strains BAL2838 (*ΔphrC*, *aprE*, *epr*, *vpr*) and BAL941 (*ΔphrC*) were incubated with proCSF, and CSF production was assayed using the biological assay. In comparison with the wild-type strain, the strain lacking subtilisin, Epr and Vpr produced 10-fold less CSF (Fig. 5C), indicating that the mutant lacking *aprE*, *epr* and *vpr* is defective in the cleavage event to release mature CSF.

1326 S. Lanigan-Gerdes, A. N. Dooley, K. F. Faull and B. A. Lazazzera

![Figure](https://i.imgur.com/yourimageurl.png)

Fig. 4. Sigma-H regulates expression of the proCSF-processing protease(s). Data shown are from a representative experiment of at least three independent experiments. Error bars represent standard error of the mean of the replicate points from one experiment.

A. The relative amount of endogenous CSF present in culture medium during exponential growth was measured. Strains used are as follows: WT (BAL948 – thrC::P<sub>spachy</sub>-phrC, ΔphrC) (circles), sigH (BAL949 – thrC::P<sub>spachy</sub>-phrC, ΔsigH, ΔphrC) (diamonds) and phrC (BAL941 – ΔphrC) (squares). The relative amount of CSF produced was determined by dividing the level of CSF produced for each sample by the level of CSF produced by samples of BAL948 at an OD<sub>600</sub> of 1.0.

B. proCSF was incubated with washed cells of the following strains: sigH (BAL1190 – ΔsigH, ΔphrC) (circles) and WT (BAL941 – ΔphrC) (diamonds). The amount of CSF produced was determined using the biological assay.

To determine whether subtilisin, Epr and Vpr are directly involved in cleaving proCSF to CSF, we analysed the ability of purified subtilisin and Vpr to cleave proCSF. Epr was not tested as we were unsuccessful in overexpressing and purifying this protein from Escherichia coli. The subtilisin that was used in these assays was a commercially available subtilisin that had been purified from Bacillus licheniformis (Sigma). Subtilisin from B. licheniformis and B. subtilis have 79% amino acid similarity and have been shown to have similar substrate preferences (Corvey et al., 2003; Gosalia et al., 2005). The Vpr used in these assays was a Vpr–His<sub>6</sub> fusion protein overexpressed in and purified from E. coli. As a control to demonstrate that any proCSF cleavage activity observed with Vpr–His<sub>6</sub> was due to Vpr and not a contaminating protease, we also analysed the cleavage activity of Vpr that has the catalytic site serine, S534, changed to an alanine, VprS534A. The wild-type and mutant Vpr preparations exhibit identical protein profiles as determined by SDS-PAGE and Coomassie stain (data not shown).

Subtilisin and the purified preparations of Vpr and VprS534A were incubated with synthetic proCSF and the amount of CSF produced was measured using the biological assay. Subtilisin (Fig. 5D) and wild-type Vpr (Fig. 5E) cleaved proCSF to CSF, but not VprS534A (Fig. 5E). Subtilisin and Vpr incubated alone did not result in CSF activity in the assay (data not shown). These data indicate that subtilisin and Vpr are directly involved in processing proCSF to CSF.

The ability of subtilisin and Vpr to cleave proCSF to the expected pentapeptide was determined using the LC-MS/MS-MRM procedure described above. Incubation of either protease alone did not result in the production of a significant CSF MRM response (Fig. 5F). However, when either enzyme was incubated with proCSF, a significant response was seen (Fig. 5F). Such a response was not observed with the catalytically inactive VprS534A (Fig. 5F). These data indicate that subtilisin and Vpr cleave proCSF to release the mature CSF pentapeptide.

### Subtilisin and Vpr can cleave other Phr peptides

All Phr proteins of B. subtilis are predicted to be cleaved to release a pentapeptide that is the mature signalling peptide. For at least five of the Phr proteins, the mature signalling peptide appears to be derived from the C-terminal five amino acids, as CSF is from its precursor protein (Fig. 1). Thus, we predicted that the proteases involved in releasing mature CSF would also play a role in releasing the signalling peptides that appear to be derived from the C-terminus of their precursor protein. We used an LC-MS/MS-MRM procedure to test the ability of subtilisin and Vpr to release the PhrA<sub>pep</sub> signalling peptide, ARNQT, from the C-terminus of synthetic proPhrA (corresponds to the portion of PhrA that is C-terminal to the signal sequence) (Fig. 6A). The intensity of the MRM response for the transition of the doubly charged parent (m/z 295.3) to the singly charged fragment (m/z 342.4) was recorded (Fig. 6B). A significant response was not detected when proPhrA, subtilisin or Vpr was incubated separately (Fig. 6B). In contrast, a significant response was detected when proPhrA was incubated with either subtilisin or Vpr (Fig. 6B), demonstrating that both of these proteases can cleave proPhrA to PhrA<sub>pep</sub>. As expected, VprS534A incubated with proPhrA did not result in a detectable response (Fig. 6B). These results suggest that subtilisin and Vpr play

© 2007 The Authors  
Journal compilation © 2007 Blackwell Publishing Ltd, *Molecular Microbiology*, **65**, 1321–1333

Fig. 5. Subtilisin, Epr and Vpr are, in part, responsible for proCSF cleavage to CSF.

A. A strain lacking *aprE*, *epr* and *vpr* accumulates less CSF in the culture medium than a wild-type strain. The following strains were grown in minimal media to OD<sub>600</sub> of 2 and the amount of endogenous CSF production was measured: WT (JH642 – wild type), *aprE* (BAL1108), *vpr* (BAL793), *epr* (BAL2243), *vpr, aprE* (BAL1885), *epr, aprE* (BAL2264), *epr, vpr* (BAL2262) and *vpr, aprE, epr* (BAL2246). The amount of CSF produced for each strain was divided by the amount of CSF produced by the WT strain. Shown is the average value from at least three independent experiments. Error bars represent standard error of the mean.

B. A strain lacking *aprE, epr* and *vpr* expresses a CSF-controlled reporter fusion, *srfA–lacZ*, at lower levels than a wild-type stain. Strains were grown in minimal media and, at the indicated OD<sub>600</sub> values, were assayed for β-galactosidase-specific activity. Data shown are representative of two independent experiments. Strains used are WT (BAL199; circles) and *aprE, epr, vpr* (BAL2849; squares).

C–E. The amount of CSF produced was measured using a biological assay. Data shown are representative of three independent experiments, and the error bars represent standard error of the mean of the replicate points from one experiment. (C) Strains containing a Δ*phrC* mutation and that either lacked *aprE, epr* and *vpr* (BAL2838; circles) or were otherwise wild type (BAL941-WT; diamonds) were grown in minimal media to OD<sub>600</sub> = 2. Washed cells were serially diluted twofold, incubated with proCSF. (D) Plotted is the concentration of subtilisin that was incubated with proCSF. (E) Plotted is the concentration of Vpr (circles) or VprS534A (squares) that was incubated with proCSF.

F. Purified subtilisin (S), Vpr (V) and VprS534A (V*) were incubated with proCSF for 40 min and the presence of ERGMT was determined by LC-MS/MS-MRM. Plotted is the intensity of the signal for parent to fragment ion transition versus whether proCSF was present and which protease was added to the reaction.


a role in releasing Phr signalling peptides derived from the C-terminus of their precursor proteins.

For at least one, and possibly three, of the *B. subtilis* Phr proteins, the mature signalling peptide is released from an internal portion of the precursor protein (Fig. 1)

(Jiang et al., 2000; Pottathil and Lazazzera, 2003; Hayashi et al., 2006). To test whether subtilisin and Vpr participate in the production of these signalling peptides, we assayed the ability of these proteases to release the PhrE<sub>pep</sub> signalling peptide, SRNVVT, from synthetic

© 2007 The Authors  
Journal compilation © 2007 Blackwell Publishing Ltd, Molecular Microbiology, 65, 1321–1333

1328 S. Lanigan-Gerdes, A. N. Dooley, K. F. Faull and B. A. Lazazzera

sensing form has remained elusive. Here, we studied production of the CSF signalling peptide from its precursor protein and identified three redundant proteases, subtilisin, Epr and Vpr, that process at least a subset of the Phr precursor proteins to their mature pentapeptide form.

Subtilisin, Epr and Vpr all play a role in proCSF processing to CSF as evidenced by a mutant lacking these proteases having a defect in CSF production, and purified subtilisin and Vpr cleaving synthetic proCSF to CSF. Purified subtilisin and Vpr were also able to cleave synthetic proPhrA to PhrA<sub>pep</sub>, suggesting that, as for CSF, subtilisin, Epr and Vpr all play a role in PhrA<sub>pep</sub> production by cells. Both PhrA<sub>pep</sub> and CSF are derived from the C-terminal five amino acids of their Phr precursor.

Purified subtilisin and Vpr were unable to cleave synthetic proPhrE to PhrE<sub>pep</sub>. PhrE<sub>pep</sub>, unlike CSF and PhrA<sub>pep</sub>, is derived from an internal portion of its precursor protein, and thus, two cleavage reactions are necessary for release of the signalling peptide. Although subtilisin or Vpr alone was insufficient to produce PhrE<sub>pep</sub>, they may still be necessary. Subtilisin and/or Vpr may be responsible for cleaving either N- or C-terminally to mature PhrE<sub>pep</sub>, and another protease may be required for the remaining cleavage event. Refinement of our mass spectrometry assay is underway to test this hypothesis. Identification of the protease(s) that processes PhrE<sub>pep</sub> has implications for our understanding of the range of bacteria that could produce Phr signalling peptides.

Only a mutant lacking subtilisin, Epr and Vpr showed a defect in CSF production, and this mutant still produced small, but detectable, levels of CSF. Thus, there appears to be significant redundancy with at least four proteases involved in Phr processing. The unidentified protease(s) appears to be a serine protease, as addition of PMSF completely abolished proCSF-processing activity associated with cells, and future experiments are aimed at identifying this entity. Possible candidates include Bpr and WprA, secreted serine proteases of *B. subtilis* that are inhibited by PMSF (Sloma et al., 1990; Babe and Schmidt, 1998). The depth of redundancy for processing Phr proteins may increase the robustness of this step against mutations and other stresses.

Testing subcellular fractions for proCSF-processing activity indicated that this activity is cell wall-associated. Such a location would concentrate this processing activity near the site of Phr protein secretion, allowing for efficient cleavage to Phr<sub>pep</sub>. Processing of Phr signalling peptides by cell-wall proteases differs from the processing mechanisms described for other signalling peptides produced by Gram-positive bacteria. Other signalling peptides, such as the Agr peptide of *Staphylococcus aureus*, are processed by integral membrane proteins (Lyon and Novick, 2004).

A cell-wall location for subtilisin, Epr and Vpr is surprising as these proteases have previously been shown to be

Discussion

The Phr signalling peptides of *B. subtilis* were first identified in 1996 (Perego and Hoch, 1996; Solomon et al., 1996). Despite this, the mechanism by which their precursor proteins are processed to their mature quorum-

secreted into the culture medium and not associated with
the cell wall (Antelmann *et al.*, 2002; Corvey *et al.*, 2003;
Kho *et al.*, 2005). That a mutant lacking subtilisin, Epr and
Vpr had less cell-associated proCSF-processing protease
activity is consistent with these proteases being cell wall-
associated under the conditions used in this study. At this
time, we do not know the mechanism by which these
proteases associate with the cell wall. The differing results
for the location of these proteases could be due to the
growth conditions used in these studies; minimal medium
and exponential phase were used in this study and spora-
tion medium and early stationary phase in the previous
studies. It is possible that the cellular location of these
proteases may be regulated by the growth conditions.
Indeed, we have observed that the proCSF cleavage
activity was no longer cell-associated but was associated
with the culture supernatants when the cells were grown
in sporulation medium to early stationary phase (S.
Lanigan-Gerdes and B.A. Lazazzera, unpubl. data). This
would suggest that, under conditions where the proPhr-
processing proteases are extracellular, Phr peptide pro-
duction would be strongly influenced by the diffusivity of
the extracellular medium. Understanding regulation of
protease localization will enhance our understanding of
the regulation of Phr signalling peptide production and
quorum sensing in *B. subtilis*.

Subtilisin, Epr and Vpr are secreted members of the
subtilisin family of proteases that is conserved among
many bacterial species (Rawlings *et al.*, 2004). This sug-
gests that other bacteria growing in the vicinity of *B.* *subtilis*
could participate in Phr<sub>pep</sub> production. Furthermore, other
components of the signalling pathway, the Sec secretary
system and the oligopeptide permease, are similarly con-
served, suggesting that many bacterial species are
capable of producing and responding to their own Phr
signalling peptides, provided they have the appropriate
cytoplasmic receptor proteins. Several *rap/phr* signalling
cassettes are encoded on potentially mobile genetic ele-
ments and one has been shown to be encoded on and
regulate a conjugative transposon that could be trans-
ferred from *B. subtilis* into *B. anthracis*, *B. licheniformis*
and *Listeria monocytogenes* (Auchtung *et al.*, 2005).
These bacteria, in addition to the secretary system, have
subtilisin-like proteases and an oligopeptide permease
indicating that they should be capable of producing and
sensing the Phr signalling peptide that is encoded on the
transposon.

Identification of subtilisin, Epr and Vpr as having a role
in the production of at least some of the Phr peptides of
*B. subtilis* fills a major gap in our knowledge of how these
signalling peptides are produced. These findings shed
light on how the genes for this peptide-signalling system
could be transferred and function in other bacterial
species and the range of bacterial species capable of

producing these signalling peptides. For those genomes
in which genes for Phr peptides have been identified but
for which it has not been possible to identify the mature
signalling peptide into which they are processed, analysis
of the products of incubation of the subtilisin-like protease
from these bacteria with the Phr precursor protein may
identify the mature signalling peptide. Understanding Phr
peptide signalling in other bacteria will elucidate the range
of processes regulated by this form of communication.

### Experimental procedures

#### Reagents

Peptides were synthesized by Bio-Synthesis Incorporated
(Lewisville, TX). Subtilisin was purchased from Sigma-Aldrich
Company.

#### Growth conditions

*Bacillus subtilis* cells were grown with shaking at 37°C in S7
minimal medium (Vasantha and Freese, 1980), except that
MOPS (morpholinepropanesulphonic acid) was used at 50
rather than 100 mM. This medium contained 1% glucose,
0.1% glutamate, and required amino acids (tryptophan, phe-
nylalanine and, when necessary, threonine) at 50 μg ml<sup>-1</sup>.

#### Strain construction

*Bacillus subtilis* strains used in this study are listed in
Table 2. These were constructed by transformation with
chromosomal DNA or plasmids using standard protocols
(Harwood and Cutting, 1990). The following alleles utilized in
this study have been previously described: Δ*comQ::spc*
(Magnuson *et al.*, 1994), Δ*opp::erm* (formerly Δ*spoOK::erm*;
LeDeaux and Grossman, 1995), Δ*sigH::cat* (formerly
Δ*spoOH::cat*, Jaacks *et al.*, 1989), Δ*sigH::cat::spc* (Auchtung
*et al.*, 2006), *aprE::pBL188(cat)* (Hamon *et al.*, 2004) and
*amyE::[srfA-lacZΩ374(neo)]* (Solomon *et al.*, 1995). The
*aprE::pBL188(cat::erm)* allele was constructed by transform-
ing the appropriate strains with the plasmid pCm::Er
(Steinmetz and Richter, 1994). Erythromycin-resistant trans-
formants were selected and then screened for chloram-
phenicol sensitivity to confirm insertion of *erm* into the *cat*
gene.

The Δ*phrC* mutant alleles were constructed as follows. The
Δ*phrC::tet* allele was constructed by cloning upstream and
downstream regions of *phrC* on either side of *tet*. A 643 bp
upstream fragment was PCR amplified with primers BL153
(5′-GGAATTCTACGTGGAGCAGGAAAC-3′) and BL154
(5′-AAGGATCCTGCGGCCAAACAAAATAAC-3′) that encode
EcoRI and BamHI sites (underlined sequence) respectively.
This PCR fragment was cloned into the EcoRI and BamHI
sites of pGK68. pGK68 (a gift of A.D. Grossman) is pGEM-
3Zf(+)-*cat*-1 (Youngman *et al.*, 1989) with *tet* inserted at the
Xbal site. A 439 bp downstream fragment was PCR amplified
using primers PhrC1 (5′-GCGAGAAGGGGTTTTTC-3′) and
BL157 (5′-CTATTATCACCTGTTCTC-3′). This PCR fragment
was blunt-cloned into a PstI-digested and end-filled pGK68

© 2007 The Authors
Journal compilation © 2007 Blackwell Publishing Ltd, *Molecular Microbiology*, **65**, 1321–1333

1330 S. Lanigan-Gerdes, A. N. Dooley, K. F. Faull and B. A. Lazazzera

Table 2. Strains used in this study.

| Strain name | Relevant genotypes$^a$ |
|-------------|-------------------------|
| JH642       | trpC2, pheA1            |
| BAL793      | vpr::pBL36(cat)         |
| BAL941      | $\Delta phrC::tet$      |
| BAL950      | $\Delta phrC::tet$, $\Delta comQ::spc$, $\Delta opp::erm$ |
| BAL1108     | $aprE::pBL188(cat)$     |
| BAL1190     | $\Delta sigH::cat$, $\Delta phrC::tet$ |
| BAL1885     | $vpr::pBL36(cat)$, $aprE::pBL188(cat::erm)$ |
| BAL2243     | $epr::pBL503(tet)$      |
| BAL2246     | $vpr::pBL36(cat)$, $aprE::pBL188(cat::erm)$, $epr::pBL503(tet)$ |
| BAL2262     | $vpr::pBL36(cat)$, $epr::pBL503(tet)$ |
| BAL2264     | $aprE::pBL188(cat)$, $epr::pBL503(tet)$ |
| BAL2838     | $vpr::pBL36(cat)$, $aprE::pBL188(cat::erm)$, $epr::pBL503(tet)$, $\Delta phrC::spc$ |
| BAL2849     | $amyE::[srfA-lacZ\Omega374(neo)]$, $vpr::pBL36(cat)$, $aprE::pBL188(cat::erm)$, $epr::pBL503(tet)$ |
| BAL199      | $amyE::srfA-lacZ\Omega374(neo)$ |
| BAL946      | $amyE::[srfA-lacZ\Omega374(neo)]$, $thrC::[P_{spachy}(erm)]$, $\Delta phrC::tet$ |
| BAL948      | $thrC::[P_{spachy}-phrC(erm)]$, $\Delta phrC::tet$ |
| BAL949      | $thrC::[P_{spachy}-phrC(erm)]$, $\Delta sigH::cat::spc$, $\Delta phrC::tet$ |

a. All strains are derivatives of JH642 and contain $trpC2$, $pheA1$ mutations.

plasmid containing the $phrC$-upstream region, to yield plasmid, pBL124. The $\Delta phrC::spc$ allele was constructed by swapping $spc$ for $tet$ on pBL124. The $spc$ gene was obtained by BamHI and SalI digestion of pJL73 (LeDeaux and Grossman, 1995) and then cloned into the corresponding sites of pBL124, to yield pBL123. pBL124 and pBL123 were transformed into $B$. *subtilis*, and tetracycline- or spectinomycin-resistant transformants were selected as appropriate. These transformants were then screened for chloramphenicol sensitivity to test that replacement of $phrC$ had occurred rather than integration of the plasmid, which carries $cat$.

The $thrC::P_{spachy}$-$phrC(erm)$ allele was constructed to assess the effect of mutations on the post-transcription expression of $phrC$. As a first step, pBL112 was constructed as a vector for replacement of the $thrC$ locus with a gene under the control of $P_{spachy}$, an IPTG-inducible promoter (Quisel et al., 2001). A 1.6 kb DNA fragment was released from pPL82 (Quisel et al., 2001) with BamHI and EcoRI and was cloned into the corresponding sites of pRDC19 (Arigoni et al., 1998). A 188 bp fragment containing $phrC$ was then PCR amplified using primers PhrC7 ($5'$-ATAAGCTTACAA AGGAGTGAAAGGTTTGT-3') and PhrC8 ($5'$-ATGCATGCGAA AAACCCCTTCTCGCTAA-3') that encode HinDIII and SphI sites (underlined sequence) respectively. This PCR product was cloned into the HinDIII and SphI sites of pBL112, to yield plasmid pBL24. $B$. *subtilis* cells were transformed with pBL24 and pBL112, and erythromycin-resistant transformants were selected. Transformants were checked for replacement of the $thrC$ locus, with either the $P_{spachy}$-$phrC(erm)$, in the case of pBL24, or $P_{spachy}(erm)$, in the case of pBL112, by the lack of spectinomycin resistance associated with the plasmids and auxotrophy for threonine.

The $vpr::pBL36(cat)$ and $epr::pBL503(tet)$ alleles were constructed as follows. Internal fragments of $vpr$ and $epr$ were PCR amplified from the $B$. *subtilis* chromosome using the following pairs of primers: Vpr1 ($5'$-GCTGCCGCTCAATG AATACGCCGTCACTTT-3') and Vpr2 ($5'$-GCTTAATCGTT GGGACAGACATGCCTGGCA-3') for $vpr$, and BL783 ($5'$- GCAAAACAGCAGCGAGAAAGAGG-3') and BL784 ($5'$-TAG CCGTTATGCTTGGCTCCGAT-3') for $epr$. The PCR product

for $vpr$ was blunt-cloned into the Smal site of pGEM-3Zf(+)- cat-1 to yield pBL36. The PCR product for $epr$ was blunt- cloned into the Smal site of pGK68, and then an EcoRI to HinDIII fragment containing $epr$ and $tet$ was subcloned into the corresponding sites in pBluescript II KS(+) (Stratagene) to yield pBL503. pBL36 ($vpr$) and pBL503 ($epr$) were integrated into the chromosome of $B$. *subtilis* at their corresponding gene by transforming $B$. *subtilis* cells and selecting either chloramphenicol- or tetracycline-resistant transformants.

### Protein methods

Protein concentrations were determined by Bradford assay (Bollag et al., 1996) using the Coomassie Plus Assay Reagent (Pierce, Rockford, IL). Samples were analysed for protein content, when appropriate, by SDS-PAGE (Bollag et al., 1996).

### Preparation of cellular fractions

To prepare cells or cell-free culture supernatants for proCSF-processing protease assays, $B$. *subtilis* cultures were grown in minimal medium to an OD$_{600}$ of between 1 and 2.5. The cells were harvested by centrifugation (8000 g, 20 min) and washed with fresh minimal medium containing 5 μg ml$^{-1}$ chloramphenicol. The cells were then re-suspended in fresh minimal medium containing 5 μg ml$^{-1}$ chloramphenicol to the same volume as the volume of the culture supernatant from the first centrifugation step. This ensured normalization of the cells and supernatant fractions. The culture supernatant passed through a 0.2 μM pore size filter and was retained as the cell-free culture supernatant.

To prepare subcellular fractions, a 250 ml culture of $B$. *subtilis* was grown to an OD$_{600}$ of 2.5. The cells were harvested by centrifugation (8000 g, 20 min) and washed with Fractionation buffer (50 mM Tris-pH 7.5, 10 mM EDTA). All subsequent steps were carried out at 4°C. Harvested cells were re-suspended in 20 ml of Fractionation buffer and lysed by passage through a French press. The lysate was centrifuged
(11 000 g, 30 min) and the resulting supernatant was separated into a cell wall-enriched fraction and a cytoplasm and membrane fraction based on the protocol of (Soldo *et al.*, 1999). Briefly, the cell-wall material was pelleted by centrifugation twice at 27 000 g for 15 min. The supernatant was collected as the cytoplasm and membrane fraction. The cell-wall pellet was washed and re-suspended in 1 ml of Fractionation buffer. Western blots were performed as described in Bollag *et al.* (1996) to assess the purity of the cell fractions. Anti-F-type ATP synthase antisera (Agisera, Sweden) and anti-GFP antisera (Molecular Probes) were used to detect a membrane and cytoplasm protein marker respectively.

### Overexpression of Vpr

The pBL631 plasmid that overexpresses the Vpr–His₆ fusion protein in *E. coli* was constructed as follows. The *vpr* gene was PCR amplified from the *B. subtilis* genome using primers BL793 (5′-CGCGGATCCTTGAAAAAGGGGATCATTCGCTTTC-3′) and BL794 (5′-ACGCGTCGACGTCTTCAACAGTGAAAGGTTCTTCGGGAC-3′) that encode BamHI and Sall sites (underlined sequence) respectively. The resulting PCR product was cloned into the BamHI and Sall sites of pET21 (Novagen), placing *vpr* under the control of a T7 promoter and fusing it to a C-terminal His₆ tag. The pBL635 plasmid, which overexpresses a mutant form of Vpr–His₆, VprS534A, in *E. coli* cells, was created by using the QuikChange Site-Directed Mutagenesis Kit (Stratagene). pBL631 was used as a template and primers BL798 (5′-GGATCAAAAACAA GGAACAGCAATGGGCATCGGCCTCATTTGCCGGAGCG-3′) and BL799 (5′-CGCTCCGGCAATATGAGGCGATGCCATT GCTGTTCCCTTGTTTTGATCC-3′) (altered nucleotides are underlined) were used to change the catalytic site serine S534 to an alanine.

The pBL631 and pBL635 plasmids were introduced into *E. coli* BL21 (λDE3, F′4Z::Tn5 pro+) that encodes T7 RNA polymerase under the control of an IPTG-inducible promoter. The resulting *E. coli* strains were grown in LB at 37°C with shaking to an OD~600~ of 0.5. IPTG was then added to 1 mM to induce expression of Vpr and VprS534A. Incubation of the cells continued until the culture reached an OD~600~ of 2.

### Purification of Vpr

Cells overexpressing Vpr or VprS534A were harvested by centrifugation. The cell pellet was re-suspended in 25 ml of a buffer containing 50 mM NaH₂PO₄ and 300 mM NaCl₂ (pH 8) and then lysed by passage through a French press twice. The lysate was centrifuged (11 000 g, 4°C, 10 min), imidazole was added to the supernatant (100 mM final concentration) and the supernatant was passed over a 5 ml HiTrap Chelating HP column (Amersham) charged with Ni²⁺. The column was eluted with a 30 ml linear gradient of imidazole from 10 mM to 1 M. Vpr eluted around 65 mM imidazole and was visualized on a Coomassie-stained SDS-PAGE gel as a band of approximately 68 kDa.

### In vitro proPhr cleavage assay

Unless otherwise indicated, 100 pmol of synthetic proCSF (NAEALDFHVTERGMT), proPhrE (NGEMKEASRNVTLAPTHEFLV) or proPhrA (GETANTEGKTFHIAARNQT) was added to a 1 ml volume and incubated at 37°C for 40 min. Cells and culture supernatants were assayed in a defined minimal growth medium containing 5 µg ml⁻¹ chloramphenicol. Cellular fractions and purified enzymes were assayed in Fractionation buffer. The reactions were centrifuged (16 000 g, 1 min), and the supernatants were filtered through a 0.2 μM pore size filter. CSF production was then monitored using either a biological or a mass spectrometric assay, as described below. PhrA~pep~ and PhrE~pep~ production was monitored using a mass spectrometric assay.

#### Quantification of CSF using a biological assay

Sufficient trifluoroacetic acid (TFA) was added to samples for quantification to achieve a pH of 2. Each sample was passed through 1-ml C18 reverse-phase column (Sep-Pak Plus, Waters), which had been previously activated with acetonitrile and equilibrated with 0.1% aqueous TFA. After loading the sample, the column was washed with 3 ml of 0.1% TFA, and then eluted with 3 ml of 11% acetonitrile, 0.1% TFA solution. The eluates were dried in a vacuum centrifuge and re-suspended in a minimal growth media containing 50 µg ml⁻¹ bovine serum albumin. Ten, twofold serial dilutions were made of each sample and these were incubated with strain BAL125 (∆phrC, *srfA-lacZ*) for 70 min at 37°C as described previously (Lazazzera *et al.*, 1999). The level of β-galactosidase-specific activity for the cells treated with column eluates was compared with the level of activity for the cells treated with known concentrations of synthetic CSF to determine the concentration of CSF in each eluate.

#### Mass spectrometric assay for Phr peptides

Samples were injected (typically 50 μl) onto a reverse-phase HPLC column (Thermo Aquasil C18, 5 mm particle size, 300 Å pore diameter, 100 × 2.1 mm) equilibrated with aqueous 0.1% formic acid and eluted (40 μl min⁻¹) with an increasing concentration of acetonitrile (time/% acetonitrile; 0/0, 20/0, 50/95, 55/95). The effluent from the column was directed at an Ionspray™ ion source connected to a tuned and calibrated triple quadrupole mass spectrometer (API III⁺, Perkin-Elmer Sciex™, Thornhill, Canada). Mass spectra were obtained at instrument conditions sufficient to resolve the isotopes of the PPG/NH₄⁺ singly charged calibrant ion at m/z 906 with 40% valley using hydrocarbon- and particulate-depleted air (so-called zero air) for spray nebulization, and the vapours of liquid nitrogen for the curtain gas, both at a flow rate of 0.6 l min⁻¹, with a 0.3 Da step size during data acquisition, and an orifice voltage of 80.

Analysis of the three peptides revealed the following ions: CSF showed an intense ion at m/z 593.8 (100%) and a less intense signal at m/z 297.2 (30%), corresponding to the singly and doubly protonated molecule respectively [calculated for (MH)⁺ and {(M+2H)/2}⁺ as 593.27 and 297.13 Da respectively]; PhrA~pep~ showed an intense ion at m/z 295.3, corresponding to a doubly protonated parent [calculated for {(M+2H)/2}⁺ at 295.15 Da]; and PhrE~pep~ showed an intense ion at m/z 576.1, corresponding to a singly protonated parent [calculated for (MH)⁺ at 576.3 Da].

© 2007 The Authors  
Journal compilation © 2007 Blackwell Publishing Ltd, *Molecular Microbiology*, **65**, 1321–1333
1332 S. Lanigan-Gerdes, A. N. Dooley, K. F. Faull and B. A. Lazazzera

During MS/MS using argon CAD gas: doubly protonated CSF [instrumental setting for CAD gas thickness (CGT) of 80] fragmented to a singly charged ion at m/z 386.5, tentatively identified as the \(a_4\) – methionine side-chain ion (minimal fragmentation was obtained under these conditions from the singly charged parent); doubly protonated PhrA\(_{\text{pep}}\) fragmented (CGT 130) to a singly charged ion at m/z 342.4, tentatively identified as the \(b_3\) ion; and singly protonated PhrE\(_{\text{pep}}\) fragmented (CGT 170) to a singly charged ion at m/z 227.2, tentatively identified as the \(b_2\)-NH\(_3\) ion (Roepstorff and Fohlman, 1984; Biemann, 1988).

Resolution of the mass spectrometer (both Q1 and Q3) was deliberately degraded to increase sensitivity for the MRM experiments. In this case the isotopes of the PPG/NH\(_4^+\) singly charged calibrant ion at m/z 906 were not resolved from one another. MRM data using the above-described transitions were collected [Tune\(^{\text{TM}}\) version 2.5 (FPU)] and displayed (MacSpec\({}^{\text{TM}}\), version 3.3) using instrument manufacturer-supplied software.

### Assay for endogenous CSF production by cells

To test for levels of endogenous CSF production, cells were grown in minimal media at 37°C to an OD\(_{600}\) of 0.2, at which time 1 mM IPTG was added, when necessary, to induce *phrC* expression. At the optical densities indicated, a 7 ml sample of the culture was harvested. The cells were removed by centrifugation, the supernatant was filtered through a 0.2 μM pore-size filter and the filtered supernatant was stored at –20°C. CSF in the supernatant was quantified using the biological assay described above.

### Acknowledgements

We thank Cathleen Collins for assistance with experiments to determine the role of sigma-H in CSF production. We thank members of the Lazazzera laboratory for helpful discussions regarding this work. We would also like to thank Dr Alan Grossman of MIT for providing bacterial strains. This work was supported in part by the NIH Public Health service Grant AI48616 to B.A.L. S.L.-G. was supported in part by the USPHS National Research Service Award GM07104. The mass spectrometer used in this work was purchased in part with funds from the W.M. Keck Foundation.

### References

Antelmann, H., Yamamoto, H., Sekiguchi, J., and Hecker, M. (2002) Stabilization of cell wall proteins in *Bacillus subtilis*: a proteomic approach. *Proteomics* **2**: 591–602.

Arigoni, F., Talabot, F., Peitsch, M., Edgerton, M.D., Meldrum, E., Allet, E., *et al.* (1998) A genome-based approach for the identification of essential bacterial genes. *Nat Biotechnol* **16**: 851–856.

Auchtung, J.M., Lee, C.A., Monson, R.E., Lehman, A.P., and Grossman, A.D. (2005) Regulation of a *Bacillus subtilis* mobile genetic element by intercellular signaling and the global DNA damage response. *Proc Natl Acad Sci USA* **102**: 12554–12559.

Auchtung, J.M., Lee, C.A., and Grossman, A.D. (2006) Modulation of the ComA-dependent quorum response in *Bacillus subtilis* by multiple Rap proteins and Phr peptides. *J Bacteriol* **188**: 5273–5285.

Babe, L.M., and Schmidt, B. (1998) Purification and biochemical analysis of WprA, a 52-kDa serine protease secreted by *B. subtilis* as an active complex with its 23-kDa propeptide. *Biochim Biophys Acta* **1386**: 211–219.

Bateman, A., Coin, L., Durbin, R., Finn, R.D., Hollich, V., Griffiths-Jones, S., *et al.* (2004) The Pfam protein families database. *Nucleic Acids Res* **32**: D138–D141.

Biemann, K. (1988) Contributions of mass spectrometry to peptide and protein structure. *Biomed Environ Mass Spectrom* **16**: 99–111.

Bollag, D.M., Rozycki, M.D., and Edelstein, S.J. (1996) *Protein Methods*. New York: Wiley-Liss.

Bongiorni, C., Ishikawa, S., Stephenson, S., Ogasawara, N., and Perego, M. (2005) Synergistic regulation of competence development in *Bacillus subtilis* by two Rap-Phr systems. *J Bacteriol* **187**: 4353–4361.

Bongiorni, C., Stoessel, R., Shoemaker, D., and Perego, M. (2006) Rap phosphatase of virulence plasmid pXO1 inhibits *Bacillus anthracis* sporulation. *J Bacteriol* **188**: 487–498.

Britton, R.A., Eichenberger, P., Gonzalez-Pastor, J.E., Fawcett, P., Monson, R., Losick, R., and Grossman, A.D. (2002) Genome-wide analysis of the stationary-phase sigma factor (sigma-H) regulon of *Bacillus subtilis*. *J Bacteriol* **184**: 4881–4890.

Carter, H.L., 3rd, Tatti, K.M., and Moran, C.P., Jr (1990) Cloning of a promoter used by sigma H RNA polymerase in *Bacillus subtilis*. *Gene* **96**: 101–105.

Comella, N., and Grossman, A.D. (2005) Conservation of genes and processes controlled by the quorum response in bacteria: characterization of genes controlled by the quorum-sensing transcription factor ComA in *Bacillus subtilis*. *Mol Microbiol* **57**: 1159–1174.

Core, L., and Perego, M. (2003) TPR-mediated interaction of RapC with ComA inhibits response regulator-DNA binding for competence development in *Bacillus subtilis*. *Mol Microbiol* **49**: 1509–1522.

Corvey, C., Stein, T., Dusterhus, S., Karas, M., and Entian, K.D. (2003) Activation of subtilin precursors by *Bacillus subtilis* extracellular serine proteases subtilisin (AprE), WprA, and Vpr. *Biochem Biophys Res Commun* **304**: 48–54.

Gosalia, D.N., Salisbury, C.M., Ellman, J.A., and Diamond, S.L. (2005) High throughput substrate specificity profiling of serine and cysteine proteases using solution-phase fluorogenic peptide microarrays. *Mol Cell Proteomics* **4**: 626–636.

Hamon, M.A., Stanley, N.R., Britton, R.A., Grossman, A.D., and Lazazzera, B.A. (2004) Identification of AbrB-regulated genes involved in biofilm formation by *Bacillus subtilis*. *Mol Microbiol* **52**: 847–860.

Harwood, C.R., and Cutting, S.M. (1990) *Molecular Biological Methods for Bacillus*. Chichester: John Wiley & Sons.

Hayashi, K., Kensuke, T., Kobayashi, K., Ogasawara, N., and Ogura, M. (2006) *Bacillus subtilis* RghR (YvaN) represses *rapG* and *rapH*, which encode inhibitors of expression of the *srfA* operon. *Mol Microbiol* **59**: 1714–1729.

Hense, B.A., Kuttler, C., Muller, J., Rothballer, M., Hartmann, A., and Kreft, J.U. (2007) Does efficiency sensing unify diffusion and quorum sensing? Nat Rev Microbiol 5: 230–239.

Jaacks, K.J., Healy, J., Losick, R., and Grossman, A.D. (1989) Identification and characterization of genes controlled by the sporulation-regulatory gene spo0H in Bacillus subtilis. J Bacteriol 171: 4121–4129.

Jiang, M., Grau, R., and Perego, M. (2000) Differential processing of propeptide inhibitors of Rap phosphatases in Bacillus subtilis. J Bacteriol 182: 303–310.

Kho, C.W., Park, S.G., Cho, S., Lee do, H., Myung, P.K., and Park, B.C. (2005) Confirmation of Vpr as a fibrinolytic enzyme present in extracellular proteins of Bacillus subtilis. Protein Expr Purif 39: 1–7.

Kleerebezem, M. (2004) Quorum sensing control of lantibiotic production; nisin and subtilin autoregulate their own biosynthesis. Peptides 25: 1405–1414.

Lazazzera, B.A., Solomon, J.M., and Grossman, A.D. (1997) An exported peptide functions intracellularly to contribute to cell density signaling in B. subtilis. Cell 89: 917–925.

Lazazzera, B.A., Kurtser, I.G., McQuade, R.S., and Grossman, A.D. (1999) An autoregulatory circuit affecting peptide signaling in Bacillus subtilis. J Bacteriol 181: 5193–5200.

LeDeaux, J.R., and Grossman, A.D. (1995) Isolation and characterization of kinC, a gene that encodes a sensor kinase homologous to the sporulation sensor kinases KinA and KinB in Bacillus subtilis. J Bacteriol 177: 166–175.

Lyon, G.J., and Novick, R.P. (2004) Peptide signaling in Staphylococcus aureus and other Gram-positive bacteria. Peptides 25: 1389–1403.

Magnuson, R., Solomon, J., and Grossman, A.D. (1994) Biochemical and genetic characterization of a competence pheromone from B. subtilis. Cell 77: 207–216.

Nakai, K., and Kanehisa, M. (1991) Expert system for predicting protein localization sites in Gram-negative bacteria. Proteins 11: 95–110.

Ogura, M., Shimane, K., Asai, K., Ogasawara, N., and Tanaka, T. (2003) Binding of response regulator DegU to the aprE promoter is inhibited by RapG, which is counteracted by extracellular PhrG in Bacillus subtilis. Mol Microbiol 49: 1685–1697.

Parsek, M.R., and Greenberg, E.P. (2005) Sociomicrobiology: the connections between quorum sensing and biofilms. Trends Microbiol 13: 27–33.

Perego, M. (1997) A peptide export-import control circuit modulating bacterial development regulates protein phosphatases of the phosphorelay. Proc Natl Acad Sci USA 94: 8612–8617.

Perego, M., and Brannigan, J.A. (2001) Pentapeptide regulation of aspartyl-phosphate phosphatases. Peptides 22: 1541–1547.

Perego, M., and Hoch, J.A. (1996) Cell–cell communication regulates the effects of protein aspartate phosphatases on the phosphorelay controlling development in Bacillus subtilis. Proc Natl Acad Sci USA 93: 1549–1553.

Pottathil, M., and Lazazzera, B.A. (2003) The extracellular

Phr peptide-Rap phosphatase signaling circuit of Bacillus subtilis. Front Biosci 8: d32–45.

Quisel, J.D., Burkholder, W.F., and Grossman, A.D. (2001) In vivo effects of sporulation kinases on mutant Spo0A proteins in Bacillus subtilis. J Bacteriol 183: 6573–6578.

Rawlings, N.D., Tolle, D.P., and Barrett, A.J. (2004) MEROPS: the peptidase database. Nucleic Acids Res 32: D160–D164.

Redfield, R.J. (2002) Is quorum sensing a side effect of diffusion sensing? Trends Microbiol 10: 365–370.

Roepstorff, P., and Fohlman, J. (1984) Proposal for a common nomenclature for sequence ions in mass spectra of peptides. Biomed Mass Spectrom 11: 601.

Slamti, L., and Lereclus, D. (2002) A cell–cell signaling peptide activates the PlcR virulence regulon in bacteria of the Bacillus cereus group. EMBO J 21: 4550–4559.

Slamti, L., and Lereclus, D. (2005) Specificity and polymorphism of the PlcR-PapR quorum-sensing system in the Bacillus cereus group. J Bacteriol 187: 1182–1187.

Sloma, A., Rufo, G.A., Jr, Rudolph, C.F., Sullivan, B.J., Theriault, K.A., and Pero, J. (1990) Bacillopeptidase F of Bacillus subtilis: purification of the protein and cloning of the gene. J Bacteriol 172: 1470–1477.

Soldo, B., Lazarevic, V., Pagni, M., and Karamata, D. (1999) Teichuronic acid operon of Bacillus subtilis 168. Mol Microbiol 31: 795–805.

Solomon, J.M., Magnuson, R., Srivastava, A., and Grossman, A.D. (1995) Convergent sensing pathways mediate response to two extracellular competence factors in Bacillus subtilis. Genes Dev 9: 547–558.

Solomon, J.M., Lazazzera, B.A., and Grossman, A.D. (1996) Purification and characterization of an extracellular peptide factor that affects two different developmental pathways in Bacillus subtilis. Genes Dev 10: 2014–2024.

Steinmetz, M., and Richter, R. (1994) Easy cloning of mini-Tn10 insertions from the Bacillus subtilis chromosome. J Bacteriol 176: 1761–1763.

Stephenson, S., Mueller, C., Jiang, M., and Perego, M. (2003) Molecular analysis of Phr peptide processing in Bacillus subtilis. J Bacteriol 185: 4861–4871.

Tzeng, Y.L., Feher, V.A., Cavanagh, J., Perego, M., and Hoch, J.A. (1998) Characterization of interactions between a two-component response regulator, Spo0F, and its phosphatase, RapB. Biochemistry 37: 16538–16545.

Vasantha, N., and Freese, E. (1980) Enzymes changes during Bacillus subtilis sporulation caused by deprivation of guanine nucleotides. J Bacteriol 144: 1119–1125.

Waters, C.M., and Bassler, B.L. (2005) Quorum sensing: cell-to-cell communication in bacteria. Annu Rev Cell Dev Biol 21: 319–346.

Winzer, K., and Williams, P. (2001) Quorum sensing and the regulation of virulence gene expression in pathogenic bacteria. Int J Med Microbiol 291: 131–143.

Youngman, P., Poth, H., Green, B., York, K., Olmedo, G., and Smith, K. (1989) Methods for Genetic Manipulation, Cloning, and Functional Analysis of Sporulation Genes in Bacillus subtilis. Washington, DC: American Society for Microbiology.
