
Menaquinone Biosynthesis in *Escherichia coli*: Identification of 2-Succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylate as a Novel Intermediate and Re-Evaluation of MenD Activity†

Ming Jiang, Yang Cao, Zu-Feng Guo, Minjiao Chen, Xiaolei Chen, and Zhihong Guo*

Department of Chemistry, Center for Cancer Research, The Hong Kong University of Science and Technology, Clear Water Bay, Kowloon, Hong Kong SAR, China

Received April 29, 2007; Revised Manuscript Received July 27, 2007

ABSTRACT: Menaquinone is an electron carrier in the respiratory chain of *Escherichia coli* during anaerobic growth. Its biosynthesis involves (1R,6R)-2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylic acid (SHCHC) as an intermediate, which is believed to be derived from isochorismate and 2-ketoglutarate by one of the biosynthetic enzymes—MenD. However, we found that the genuine MenD product is 2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic acid (SEPHCHC), rather than SHCHC. This is supported by the following findings: (i) isochorismate consumption and SHCHC formation are not synchronized in the enzymic reaction, (ii) the rate of SHCHC formation is independent of the enzyme concentration, (iii) SHCHC is not formed in weakly acidic or neutral solutions in which the isochorismate substrate is readily consumed by MenD, and (iv) the MenD turnover product, formed under conditions disabling SHCHC formation, possesses spectroscopic characteristics consistent with the structure of SEPHCHC and spontaneously undergoes 2,5-elimination to form SHCHC and pyruvate in weakly basic solutions. Two properties of the intermediate, ultraviolet transparency and chemical instability, provide a rationale for the fact that SHCHC has been consistently mistaken as the MenD product. In accordance with these findings, MenD was rediscovered to be a highly efficient enzyme with a high second-order rate constant and should be renamed SEPHCHC synthase. Intriguingly, the enzymatic activity responsible for conversion of SEPHCHC into SHCHC appears not to associate with any of the known enzymes in menaquinone biosynthesis but is present in the crude extract of *E. coli* K12, suggesting that a genuine SHCHC synthase remains to be identified to fully elucidate the ubiquitous biosynthetic pathway.

Menaquinone is derived from the common branch-point intermediate, chorismate, of the shikimate pathway in bacteria (1, 2). It is a lipid-soluble molecule that serves important biological functions. In most anaerobes and all Gram-positive aerobes, menaquinone is the sole electron transporter in the respiratory chain and is essential for their survival (3, 4). It also serves as an electron carrier in facultative Gram-negative aerobes, such as *Escherichia coli*, but only during their anaerobic growth. In contrast, menaquinone is a vitamin (K₂) in humans and animals; it plays a totally different role, serving as an enzyme cofactor involved in glutamate γ-carboxylation that is essential in blood clotting (5, 6). Due to its absence in humans and animals, menaquinone biosynthesis is an appealing target for development of antibiotics for pathogenic microbes that depend on the naphthoquinone for survival, such as *Mycobacterium tuberculosis*—an obligate aerobic Gram-positive pathogen that is a worldwide public health threat (7). In recent years, the prospect of developing antitubercular drugs from inhibitors of menaquinone biosynthesis has stimulated new interest in this old pathway (8–10).

Biosynthesis of menaquinone is best known in *E. coli*. The proposed biosynthetic pathway is illustrated in Scheme 1, involving products of eight genes (*menA*, *ubiE*, and *menFDHBCE*). The branch-point intermediate from the shikimate pathway, chorismate, is first converted to isochorismate by an isochorismate synthase, MenF (11, 12), which is isofunctional to EntC in enterobactin biosynthesis (13) but specifically dedicated to menaquinone biosynthesis. Isochorismate is subsequently transformed to the first aromatic intermediate, *o*-succinylbenzoate (OSB),¹ by two gene products—MenD and MenC (14, 15). Next, an ATP-dependent acyl-CoA ligase, MenE (16), activates the carboxyl group in the succinyl side chain for cyclization to form the naphthaloid skeleton, which is catalyzed by MenB (17). The latter enzyme was previously proposed to also hydrolyze 1,4-dihydroxynaphthoic acid CoA ester (DHNA-CoA). More recently, an open reading frame, *yfbB*, adjacent to the *menD* gene was suggested to be responsible for the DHNA-CoA hydrolysis and designated MenH (4). The proposed role of

---

¹ Abbreviations: OSB, *o*-succinylbenzoate; SHCHC, (1R,6R)-2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylic acid; SEPHCHC, 2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic acid; DHB, 3,5-dihydroxybenzoic acid; DHNA-CoA, 1,4-dihydroxynaphthoic acid coenzyme A thioester; ThDP, thiamine diphosphate; α-KG, 2-ketoglutarate; IPTG, isopropyl β-D-thiogalactopyranoside; HPLC, high-pressure liquid chromatography.

† This work was financially supported by HKUST and RGC of the Hong Kong SAR government.

* To whom correspondence should be addressed. Phone: 852-2358-7352. Fax: 852-2358-1594. E-mail: chguo@ust.hk.

10980 Biochemistry, Vol. 46, No. 38, 2007 Jiang et al.

Scheme 1: Proposed Biosynthesis of Menaquinone (Vitamin K₂) in *E. coli*${}^{a}$

![Chemical Structures and Reactions]

$a$ $\alpha$-KG = $\alpha$-ketoglutarate, SHCHC = (1R,6R)-2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate, OSB = $o$-succinyl-1-benzoate, DHNA = 1,4-dihydroxy-2-naphthanoate, CoA = coenzyme A, DMKH2 = demethylmenaquinol, MKH2 = menaquinol, AdoMet = S-adenosylmethionine, and AdoHcy = S-adenosylhomocysteine.

MenH was supported by confirmation of DHNA-CoA as the MenB product (8) and experimental demonstration of thioesterase activity of the protein (18). In the final steps of the biosynthesis, MenA carries out decarboxylative prenylation on the DHNA intermediate (19) and an $S$-adenosylmethionine (SAM)-dependent methyltransferase involved in the biosynthesis of ubiquinone, UbiE, adds a methyl group to the naphthalenoid ring to afford the reduced form (DMKH₂) of vitamin K₂ (20). The originally misannotated $menG$ gene product turned out to be a protein inhibitor (RraA) of RNase E involved in regulation of RNA abundance, instead of a SAM-dependent methyltransferase responsible for the methylation of the naphthalenoid ring (21, 22). Among the biosynthetic enzymes, the OSB synthase (MenC) is most thoroughly studied as a member of the enolase superfamily (23–28), with its X-ray structure solved in the presence of substrate (26). The crystallographic structure of a MenB analogue from *M. tuberculosis* also has been determined, gleaning important insights into the reaction mechanism in the ring formation of the naphthalenoid skeleton (8, 9).

The first committed step in menaquinone biosynthesis involves decarboxylation of 2-ketoglutarate to generate a succinic semialdehyde–ThDP anion for addition to C-2 of isochorismate. The required 2-ketoglutarate decarboxylase activity was originally thought to come from the first protein of the 2-ketoglutarate dehydrogenase complex (29, 30), but was later found to be from a distinct enzyme coded by the $menD$ gene (31–34). 2-Succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylic acid (SHCHC) was considered to be the MenD product on the basis that it was formed from isochorismate and 2-ketoglutarate by a crude extract of a $menC^{-}menD^{+}$ mutant (35). MenD was proposed to be a unique member of the ThDP-dependent decarboxylase superfamily catalyzing a Michael-type addition to isochorismate (36, 37), with a catalytic mechanism as shown in Scheme 2. Here, we present evidence to show that the turnover product of MenD is 2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic acid (8, SEPHCHC) instead of SHCHC.

## MATERIALS AND METHODS

### General Information

Chorismic acid, 2-ketoglutarate, thiamine diphosphate (ThDP), reduced $\beta$-nicotinamide adenine dinucleotide (NADH), L-lactate dehydrogenase, isopropyl $\beta$-thiogalactopyranoside (IPTG), buffers, and salts were purchased from Sigma. DNA-manipulating biochemicals, including restriction enzymes, T4 DNA ligase, and associated reagents, were obtained from New England Biolabs. Other chemicals and biochemicals in reagent kits were purchased from vendors as specified below. UV–vis absorbance was measured using a Perkin-Elmer Lambda 900 UV/vis/NIR spectrometer with temperature control by a water circulator connected to a water-thermostatable eight-cell holder. PCR amplifications were performed using a PTC-200 Peltier thermal cycler from MJ Research. Protein chromatography was performed using a Bio-Rad BioLogic

A Novel Intermediate in Menaquinone Biosynthesis

**Scheme 2: Proposed Mechanism of the MenD-Catalyzed Reaction**

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

Thiamine diphosphate (ThDP)

HR workstation in a refrigerated chamber. HPLC analysis and purification were carried out using a Waters 600E system with model 2487 dual λ absorbance detector. Mass spectroscopy was carried out using a QSTAR XL mass spectrometer manufactured by Applied Biosystems Co. 1H NMR analysis was performed with a JEOL EX 400 NMR spectrometer.

**Expression and Purification of Enzymes.** Overproduction and purification of EntC was performed as described previously (13). The genes menC and menD were amplified from the genomic DNA of E. coli and subcloned into a modified version of pET32a (a DNA fragment coding a start codon, a hexahistidine tag, and a thrombin site replaces the sequence from the start codon of the thioredoxin gene to the last nucleotide of the EcoRV restriction site of the vector) between BamHI and EcoRI (or XhoI) for expression as a protein with an extra sequence of MHHHHHHSSGLVPRGS at the N-terminus. The primers in the gene amplification were G CGC GGA TCC ATG CGT AGC GCG CAG GTA TAC CGC TGG (menC forward), C CGG AAT TCA TAA CAA CCG CTC CAG TGC ATC AAC (menC reverse), GC GGA TCC ATG TCA GTA AGC GCA TTT AAC CGA CGC TGG GCG (menD forward), and GA CTC GAG TCA TAA ATG GCT TAC CTG CGC CAG AAG TTG (menD reverse). MenC and MenD were also expressed as an untagged protein in the same vector by using GG CAT ATG CGT AGC GCG CAG and CG CAT ATG TCA GTA AGC GC as forward primers for amplification of their respective genes and using the 5′-end NdeI restriction site for subcloning. The recombinant plasmids were introduced into the host cell BL21 (DE3), and the gene products were expressed at 18 °C for 16 h in Luria broth containing 0.1 mM IPTG. The tagged proteins were purified from the crude extracts prepared from the harvested cells first by metal chelating chromatography using a 5 mL HiTrap Chelating HP column (Amersham Biosciences) and then by gel filtration using Sephacryl S-200 beads (Amersham Biosciences). The untagged proteins were partially purified by fractionation with ammonium precipitation and by anion exchange chromatography using a 1 mL HiTrap Q FF column (Amersham Biosciences) and a linear gradient of 0–500 mM NaCl. Active fractions were identified by assay for either MenC or MenD activity. SDS-PAGE found the obtained tagged proteins were purified to homogeneity with purity greater than 95%, while the untagged proteins were greater than 70% in purity. These proteins were quantified with a Coomassie Blue protein assay kit (Pierce) and stored in 50 mM Tris-HCl buffer (pH 7.8) containing 10% glycerol and 50 mM NaCl at −20 °C until use. Unless stated otherwise, the enzymes MenC, MenD, and EntC refer to the recombinant proteins with a hexahistidine tag at the N-terminus.

**Chemoenzymatic Preparation of Isochorismate, SEPHCHC, and SHCHC.** For preparation of isochorismate, chorismic acid (4 mg) was incubated with 5 μM EntC in 400 μL of 50 mM Tris-HCl buffer (pH 7.8) containing 50 mM NaCl at 37 °C for 2 h. To prepare SHCHC, 600 μL of a solution containing 25 μM MenD, 50 mM 2-ketoglutarate, 75 μM ThDP, and 7.5 mM MgSO₄ was added to the reaction mixture used for preparation of isochorismate. After incubation at 25 °C for 5 h, the reaction mixture was adjusted to pH 12 by adding 1.0 N NaOH and incubated at room temperature for another 1 h. The preparation of SEPHCHC was carried out in 25 mM sodium phosphate buffer (pH 6.5), and other reaction components were identical to those in SHCHC preparation. SEPHCHC solution was used directly after removal of the enzymes by ultrafiltration with an Amicon YM-10 (Millipore, molecular weight cutoff 10 000). Isochorismate and SHCHC were further purified separately using an Xterra preparative C18 reversed-phase column (10 μm particle size, 19 × 150 mm) on a Waters 600 HPLC system, using water containing 1% formic acid for elution at 10 mL/min. Under these HPLC conditions, chorismic acid, isochorismic acid, and SHCHC were eluted at 8.42, 13.5, and 5.35 min, respectively. Fractions containing these products were collected separately and lyophilized. Isochorismic acid and SHCHC were redissolved in pure water and quantified by absorbance at 278 nm (ε = 8300 M⁻¹·cm⁻¹) (38) and 290 nm (use the extinction coefficient ε = 4000 M⁻¹·cm⁻¹ at 293 nm) (24), respectively. SEPHCHC used in the 1H NMR analysis was from a chemoenzymatic

preparation using equimolar chorismate (4 mg in 0.6 mL) and 2-ketoglutarate (29.5 mM) as substrates. At completion of the reaction, the mixture was added to 30 μL of 6 N HCl and extracted with ethyl acetate (4 × 1.0 mL). The organic solvent was evaporated to obtain 4.2 mg of SEPHCHC (72%), as determined by converting a fraction of it to SHCHC in 0.01 N NaOH and measuring the absorbance at 290 nm.

**Enzyme Activity Assay.** The activity of MenD was assayed in buffers with varied pH values. Typically, the reaction mixture in an appropriate buffer contained 2.5 μM isochorismate, 130 nM MenD, 660 μM 2-ketoglutarate, 50 μM ThDP, and 5 mM MgSO₄. The reaction was initiated by adding the enzyme and monitored in real time with the absorbance change at 278 or 290 nm. To analyze product formation, the reaction was stopped either at an appropriate time or after 4 h of incubation by acidification with trifluoroacetic acid. To determine the decomposition products of the true turnover product of the MenD-catalyzed reaction, the product mixture obtained from a reaction carried out in 50 mM sodium phosphate buffer (pH 7.0) was adjusted to pH 12–13 with NaOH and incubated at room temperature for 1 h for analysis of SHCHC formation by HPLC. In the meantime, an aliquot of the basified solution was readjusted to pH 7.0 with 6 N HCl to detect formation of pyruvate by L-lactate dehydrogenase and NADH to monitor NADH oxidation at 340 nm.

Using SHCHC as the substrate, MenC activity was assayed in a mixture containing 50 mM Tris—HCl (pH 7.8), 50 mM NaCl, and 5 mM MgSO₄ by monitoring the decrease in absorbance at 290 nm. The activity of MenC (20 μg/mL) and the cell-free extract (600 μg/mL) of *E. coli* toward SEPHCHC was determined by HPLC in 50 mM sodium phosphate buffer at pH 7.0, using SHCHC as a positive control. In the assay of SHCHC synthase activity of the bacterial extract, MenC was also added to convert the SHCHC product to OSB for easy detection in HPLC because SHCHC coeluted with a large amount of chromogenic metabolites from the crude extract. The crude extract was prepared from a 1 L culture of *E. coli* K12 inoculated by 10 mL of overnight culture and grown at 37 °C for 30 h in LB medium that was sealed with 2 cm of mineral oil and contained 40 mM fumarate and 40 mM glycerol. Before the activity assay, the crude extract was extensively dialyzed with 50 mM sodium phosphate buffer (pH 7.0) and the total protein concentration was determined with the Coomassie Blue protein assay kit (Pierce).

HPLC analysis was performed using a Nova-Pak C18 analytical column (4 μm particle size, 3.9 × 150 mm), using water containing 1% formic acid for elution at 1 mL/min. Eluted products were detected with UV absorption at 238 and 278 nm. Under these conditions, isochorismic acid, SHCHC, and OSB were eluted at 5.5, 3.2, and 11.4 min, respectively. Retention times for ThDP and 2-ketoglutarate were 3.3 and 1.5 min, respectively. The macromolecular species in the reaction mixtures were removed by ultrafiltration using Amicon YM-10 (Millipore) before HPLC analysis.

**Spontaneous Conversion of SEPHCHC to SHCHC.** A concentrated SEPHCHC solution was prepared from chorismate using the conditions described above. The accurate concentration of SEPHCHC was determined by the absorbance at 290 nm after it was completely converted to SHCHC by diluting 100-fold into 0.01 N NaOH and incubation at room temperature for 20 min. To determine the kinetics of the spontaneous conversion, SEPHCHC was diluted 50-fold into 200 mM sodium phosphate buffer at varied pH and monitored in real time for an absorbance increase at 290 nm at 37 °C. The first-order rate constant was determined by curve-fitting the obtained experimental data. All experiments were performed in triplicate.

---

**RESULTS**

**Asynchronous Substrate Consumption and SHCHC Formation in MenD Catalysis.** According to the biosynthetic pathway for menaquinone as shown in Scheme 1, the ultraviolet absorption spectra of isochorismate and SHCHC can be exploited to develop a spectroscopic assay for determination of MenD activity. Previous studies have demonstrated that SHCHC absorbs strongly at 290 nm (24), whereas isochorismate absorption is maximum at 278 nm (38), but lower at 290 nm (ε = 2900 M⁻¹·cm⁻¹). Thus, the initial reaction rate for the MenD-catalyzed reaction can be determined by an expected linear increase of the absorbance at 290 nm (*A*₂₉₀). However, an *A*₂₉₀ increase was not observed in such an experiment carried out at pH 8.0 in Tris—HCl buffer. Instead, a linear decrease of *A*₂₉₀ was recorded in the early stage of the reaction; a very small *A*₂₉₀ increase was detected only after a long period of time (~7 min, Figure 1A). The observed *A*₂₉₀ decrease coincided with a linear decrease in absorbance (*A*₂₇₈) at 278 nm, the absorption maximum of isochorismate, indicating that the absorbance decrease most likely resulted from the consumption of the isochorismate substrate and that the SHCHC formation was not significant in the early stage of the reaction.

To corroborate the spectroscopic observations, the enzymic reaction was quenched with trifluoroacetic acid at varied time points and the reaction product was analyzed by HPLC after removal of the enzyme. Under the given conditions, consumption of isochorismate was readily detected in the reaction quenched during the linear decrease of the absorbance at either 278 or 290 nm (*t* = 4 min, Figure 1B). At the end of the observed *A*₂₉₀ decrease (*t* = 7 min), the isochorismate consumption was complete. However, SHCHC was not detectable during this substrate consumption period (*t* = 0–7 min) despite the fact that a small amount of this product was possible due to its overlap with ThDP at 3.3 min in the HPLC trace. The product was not detected until 45 min after the complete consumption of isochorismate. Afterward, the amount of SHCHC in the reaction was found to gradually increase with time (data not shown), coinciding with the small linear *A*₂₉₀ increase in the spectroscopic monitoring. At *t* = 24 h, an appreciable amount of SHCHC was formed (Figure 1B), but the slow SHCHC formation extended beyond 36 h. Consistent with the spectroscopic observations, these HPLC analyses show that the early absorbance decrease in the enzymatic reaction is indeed due to the consumption of the isochorismate substrate and that the later linear *A*₂₉₀ increase is due to the slow formation of SHCHC.

Results from both the spectroscopic monitoring and HPLC analysis indicate that the MenD-catalyzed reaction is biphasic, consisting of a fast phase of isochorismate consumption

A Novel Intermediate in Menaquinone Biosynthesis

Biochemistry, Vol. 46, No. 38, 2007 10983

(A)

| Absorbance |
| --- |
| 0.085 |
| 0.08 |
| 0.075 |
| 0.07 |
| 0.065 |
| 0.06 |
| 0.055 |
| 0.05 |
| 0.045 |
| 0.04 |

| Time (s) |
| --- |
| 0 | 200 | 400 | 600 | 800 | 1000 | 1200 |

(B)

| SHCHC |
| --- |
| SHCHC standard |
| DHB |
| A₂₉₀ |
| 24 h |
| 7 min |
| 4 min |
| solvent |
| front |
| ThDP |
| IS |
| 0 min |

| Time (min) |
| --- |
| 0 | 2 | 4 | 6 | 8 | 10 | 12 | 14 | 16 | 18 |

FIGURE 1: Time course of the MenD-catalyzed reaction at 290 or 278 nm (A) and HPLC analysis of the reaction mixture at various time points (B). DHB = 3,5-dihydroxybenzoic acid, an internal standard, IS = isochorismate, and ThDP = thiamine diphosphate. Reaction conditions: 66 nM MenD, 660 μM 2-ketoglurate, 2.5 μM isochorismate, 2.0 μM ThDP in 50 mM Tris-HCl buffer (pH 8.0) containing 5.0 mM MgSO₄. The reaction was carried out at room temperature (22.5 °C).

without detectable formation of SHCHC and a very slow phase of SHCHC formation. The lack of synchronization of the substrate consumption with the SHCHC formation strongly suggests that a yet unidentified intermediate is formed from isochorismate in the fast phase and subsequently converted very slowly to SHCHC in the second phase.

Spontaneous SHCHC Formation in the MenD-Catalyzed Reaction. The exceedingly slow formation of SHCHC in the second phase of the MenD-catalyzed reaction raises the possibility that it is nonenzymatic. To test this, the amount of MenD was varied and the reaction was again monitored at 290 nm at room temperature. As expected, the initial rate

Table 1: Rate Dependence of the Two-Phase MenD-Catalyzed Reaction on the Concentration of the Enzyme

| [MenD] | relative rate | relative rate |
| --- | --- | --- |
| (nM) | first phase$^a$ | second phase$^b$ | [MenD] | first phase$^a$ | second phase$^b$ |
|  |  |  | (nM) |  |  |
| 67.0 | 1.0 ± 0.1 | 1.0 ± 0.2 | 267 | 6.0 ± 0.1 | 0.86 ± 0.13 |
| 134 | 3.1 ± 0.1 | 1.1 ± 0.1 | 400 | 11.0 ± 0.1 | 1.4 ± 0.1 |
| 200 | 3.9 ± 0.1 | 0.94 ± 0.02 | 534 | 14.0 ± 0.1 | 0.79 ± 0.06 |

$^a$ The rate of the first phase equals the slope of the $A_{290}$ decrease in the period from 0 to 180 s. $^b$ The second-phase rate is the slope of the $A_{290}$ increase over a period of 10 min after the turning point from a linear $A_{290}$ decrease to a linear increase (e.g., as in Figure 1A). Reaction conditions: MenD at indicated concentrations, 660 μM 2-ketoglutarate, 2.5 μM isochorismate, 2.0 μM ThDP in 50 mM Tris-HCl buffer (pH 8.0) containing 5.0 mM MgSO₄. The reactions were carried out at room temperature (22.5 °C).

of isochorismate disappearance in the first phase increased with increasing MenD concentration (Table 1). However, the slope of the second-phase absorbance increase at 290 nm, which indicates the rate of SHCHC formation, is independent of the concentration of MenD in the reaction mixture. These results indeed strongly support that the SHCHC formation is not catalyzed by MenD but is most likely spontaneous under the reaction conditions.

To examine the dependence of SHCHC formation on other reaction conditions, the MenD-catalyzed reaction was carried out in buffers with varied acidity (pH value). Interestingly, the second-phase SHCHC formation was essentially undetectable by spectroscopic monitoring at low pH (pH 6.0–8.0) in phosphate buffer, as shown in Figure 2A. It became detectable at pH 8.5, and its rate was increased as the pH value of the reaction buffer was increased. Under the given conditions, the isochorismate substrate was readily consumed at a varied rate, indicating that the enzyme was fully functional. In Tris-HCl buffer, the second-phase SHCHC formation was more pronounced and became optically observable at pH 7.5–8.0 (Figure 1), and its rate was increased much more rapidly with increasing pH (data not shown), suggesting that it is catalyzed by base. To elaborate this, the enzymatic reaction was carried out in phosphate buffer (pH 7.0) to ensure that isochorismate was completely consumed. After ultrafiltration, the enzyme-free reaction mixture was basified to pH 12–13 and a large amount of SHCHC was formed as the only product within 10 min as detected by both spectroscopic monitoring (data not shown) and HPLC (Figure 2B), indicating that the SHCHC formation was indeed a base-susceptible spontaneous reaction from an unknown intermediate. This SHCHC precursor, therefore, is the true turnover product of MenD.

In all the preceding experiments, the recombinant MenD in the enzymatic reaction carries an N-terminal hexahistidine tag. To be sure that the lack of SHCHC formation activity of MenD was not due to the appendage, *menD* was resubcloned to express the wild-type protein. Due to the high level of overexpression, the untagged protein was purified to >80% by ammonium sulfate precipitation combined with anion exchange column chromatography. The activity of the untagged protein was found to be identical to that of the tagged protein in that they both demonstrated high activity in consuming the isochorismate substrate but no SHCHC formation activity at pH 6–8 in phosphate buffer (data not

shown). This further supports that the true turnover product of MenD is a precursor of SHCHC.

**SEPHCHC as the True MenD Product and Its Base Liability.** In an attempt to identify other products in the spontaneous transformation of the genuine MenD product into SHCHC under basic conditions (Figure 2B), a part of the basified reaction mixture was readjusted to a neutral pH (7.0) and added to L-lactate dehydrogenase and NADH. Real-time monitoring of the absorbance at 340 nm clearly indicated that pyruvate was present in the basified reaction mixture but was absent in the enzyme-free reaction mixture before basification (Figure 2C). On the basis of its decomposition into pyruvate and SHCHC, the true MenD product was inferred to be 2-succinyl-4-enolpyruvyl or ((1-carboxyethenyl)oxy)-6-hydroxy-3-cyclohexene-1-carboxylic acid (SEPHCHC 8), the immediate SHCHC precursor in the previously proposed reaction mechanism of the MenD-catalyzed reaction (Scheme 2).

Analysis of the MenD product formed at pH 7.0 by electron spread ionization (ESI) mass spectrometry found only a major ion ([M + Na]+) at *m/z* = 351.0573 (Figure 3A), consistent with the calculated molecular weight of SEPHCHC (328.0790). Fragmentation of this molecular ion into SHCHC and pyruvic acid by MS/MS analysis (Figure 3B) also supported the proposed structure of the MenD product. In addition, a ¹H NMR spectrum (400 MHz, CD₃OD) was obtained for the turnover product (Figure 4). Its resonance signals (*δ*, ppm), compared to those in the NMR spectra of SHCHC (26, 35), chorismic acid (39), and isochorismic acid (40), were easily assignable to the structure of SEPHCHC: four succinyl protons at 2.95 (1H, m), 2.78 (1H, m), and 2.56 (2H, m), six cyclohexene ring protons at 3.05 (1H, m), 3.76 (1H, m), 4.04 (1H, m), 4.50 (1H, m), 5.94 (olefinic, 1H, m), and 5.80 (olefinic, 1H, m), and one doublet at 5.42 (1H, d, *J* = 3.0 Hz) belonging to the 5-enolpyruvyl group. The signal for the other 5-enolpyruvyl proton should be submerged in the suppressed water signal at 4.85 ppm (*δ*) where the signal for the corresponding proton in either chorismic acid (*δ* 4.93; 39) or isochorismic acid (*δ* 4.96; 40) was found nearby. These results confirmed that the MenD product was SEPHCHC.

To quantitatively determine the base liability of SEPHCHC, its concentrated solution in phosphate buffer (pH 7.0) was diluted 50-fold in buffers with varied pH values. Its conversion to SHCHC at 37 °C was monitored by absorbance at 290 nm and found to follow first-order kinetics. The rate constants and half-times are given in Table 2. From this table, it is easy to understand that SHCHC is formed at a varied rate in the MenD-catalyzed reaction in neutral and basic solutions, but it is not detectable by HPLC (Figure 2B) in low-pH solutions.

**Re-Evaluation of the Kinetic Parameters of the MenD-Catalyzed Reaction.** A continuous assay, which was based on isochorismate consumption as indicated by the absorbance decrease at 278 nm (*ε* = 8300 M⁻¹·cm⁻¹) (38), was used to reassess the MenD activity. The kinetic constants of the MenD-catalyzed reaction were determined (Figure 5) as shown in Table 3. Noticeably, the enzyme has a nanomolar range Michaelis constant, which is only found for low-concentration metabolites such as insulin (*K*~M~ = 22.2 nM, insulinase) (41) and DNA (*K*~M~ = 1.5 nM, DNA ligase) (42). This low *K*~M~ should make the isochorismate synthase (MenF)-catalyzed reaction virtually irreversible and help to

A Novel Intermediate in Menaquinone Biosynthesis

Biochemistry, Vol. 46, No. 38, 2007 10985

(A)

| 392 |
| --- |
| 380 |
| 360 |
| 340 |
| 320 |
| 300 |
| 280 |
| 260 |
| 240 |
| 220 |
| 200 |
| 180 |
| 160 |
| 140 |
| 120 |
| 100 |
| 80 |
| 60 |
| 40 |
| 20 |
| 0 |

m/z, amu

(B)

| 9.0 |
| --- |
| 8.5 |
| 8.0 |
| 7.5 |
| 7.0 |
| 6.5 |
| 6.0 |
| 5.5 |
| 5.0 |
| 4.5 |
| 4.0 |
| 3.5 |
| 3.0 |
| 2.5 |
| 2.0 |
| 1.5 |
| 1.0 |
| 0.5 |
| 0.0 |

m/z, amu

FIGURE 3: Mass spectrum of SEPHCHC by electron spread ionization mass spectrometry (A) and MS/MS analysis of the molecular ion [M + Na]⁺ at m/z = 351.0573 (B).

10986 Biochemistry, Vol. 46, No. 38, 2007

SEPHCHC

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

FIGURE 4: \({}^{1}\)H NMR spectrum of SEPHCHC in \(d_{4}\)-methanol. Signals labeled with an asterisk are from impurities because their intensity is much less than that for a proton. SEPHCHC was obtained by ethyl acetate extraction of the acidified reaction mixture from a scaled-up chemoenzymatic preparation using equimolar chorismate and 2-ketoglutarate as substrates and evaporation of the organic solvent.

Table 2: First-Order Rate Constant and Half-Life of the Spontaneous SEPHCHC Decomposition to SHCHC at Various pH Values in 200 mM Phosphate Buffer (37 °C)

| pH | \(k\) (\(min^{-1}\)) | \(t_{1/2}\) |
|----|---------------------|-------------|
| 6.0 | \((4.80 \pm 0.34) \times 10^{-3}\) | \(24.2 \pm 1.7 \, h\) |
| 6.5 | \((5.17 \pm 0.34) \times 10^{-3}\) | \(22.4 \pm 1.5 \, h\) |
| 7.0 | \((1.23 \pm 0.07) \times 10^{-3}\) | \(9.40 \pm 0.58 \, h\) |
| 7.5 | \((1.32 \pm 0.05) \times 10^{-3}\) | \(8.77 \pm 0.30 \, h\) |
| 8.0 | \((2.90 \pm 0.04) \times 10^{-3}\) | \(3.99 \pm 0.05 \, h\) |
| 9.0 | \((1.51 \pm 0.05) \times 10^{-2}\) | \(45.8 \pm 1.7 \, min\) |
| 10.0 | \((8.04 \pm 0.16) \times 10^{-2}\) | \(8.64 \pm 0.17 \, min\) |
| 11.0 | \((1.47 \pm 0.05) \times 10^{-1}\) | \(4.71 \pm 0.15 \, min\) |
| 12.0 | \((2.33 \pm 0.01) \times 10^{-1}\) | \(2.97 \pm 0.01 \, min\) |

![Graph](https://i.imgur.com/graph_image.png)

divert chorismate into the menaquinone pathway in competition with many other chorismate-utilizing biosynthetic branches (I). It also provides a rationale for the linear \(A_{278}\) or \(A_{290}\) decrease in the entire reaction course of the MenD-catalyzed reaction (Figures 1A and 2A). In addition, MenD has a much higher catalytic efficiency (~100 fold higher) than the previous value that was obtained on the basis of the formation of SHCHC (36). Moreover, the catalysis of MenD also demonstrates a strong dependence on magnesium ion. Titration of the enzyme activity with the cofactors found that the dissociation constants of MenD—Mg\(^{2+}\) for ThDP and MenD—ThDP for Mg\(^{2+}\) are 2.4 and 80 μM, respectively. From Figure 2A, it is obvious that MenD achieves optimal activity in phosphate buffer at pH 7.0–8.0.

**SHCHC Synthase Activity in E. coli Extract.** To test whether MenC can convert SEPHCHC to SHCHC, a necessary step in the biosynthesis of menaquinone, SEPHCHC was incubated with recombinant MenC in a buffer (pH 7.0) in which spontaneous SHCHC formation was very slow.

FIGURE 5: Michaelis-Menten kinetics of the MenD-catalyzed reaction. The reaction rate was determined in the presence of saturating concentrations of 2-ketoglutarate, ThDP, and MgSO₄ at pH 7.8 and room temperature. Inset: Lineweaver-Burk plot from which \(K_{M}\) and \(k_{cat}\) were determined.

Only a very small amount of OSB, which likely resulted from the slow spontaneous transformation of SEPHCHC into SHCHC, was formed in the reaction, whereas it was readily formed from SHCHC under identical conditions (Figure 6). Again, to make sure the inability of MenC to use SEPHCHC as a substrate is not due to the N-terminus hexahistidine tag in the used recombinant MenC, the wild-type protein without any additional sequence was expressed and purified, only to be found to behave exactly the same as the hexahistidine-tagged MenC. These results clearly indicate that another enzyme is needed to synthesize SHCHC in menaquinone biosynthesis.

A Novel Intermediate in Menaquinone Biosynthesis

Biochemistry, Vol. 46, No. 38, 2007 10987

subsequently converted to OSB by MenC. OSB was not further transformed because ATP was needed for the subsequent metabolism of OSB by MenE, but was not present in the reaction mixture. These experiments strongly support the existence of a novel SHCHC synthase, which is yet to be identified.

## DISCUSSION

For a long time, SHCHC has been generally accepted to be the only biosynthetic intermediate between isochorismate and OSB in menaquinone biosynthesis, which is thought to be synthesized by MenD for subsequent conversion into OSB by MenC. Supporting evidence for this view includes the finding of only two groups of mutants (menC and menD) (14, 15), which are blocked in OSB synthesis, and the consistent observation of SHCHC formation in a number of cell-free assays of MenD activity (30–36). However, our investigation of the MenD-catalyzed reaction unambiguously demonstrates that the current biosynthetic pathway of menaquinone as shown in Figure 1 is not accurate and needs a revision to include a new intermediate.

Through real-time monitoring of the MenD-catalyzed reaction, the turnover product was found likely to be SEPHCHC instead of SHCHC. The finding of asynchronous consumption of the isochorismate substrate and SHCHC formation in the enzymic reaction provided the first clue that SHCHC is not the genuine turnover product of MenD, because it was not possible to detect the lag time of the product formation in the enzymatic reaction with the employed UV-monitoring method. This suspicion was supported by the subsequent finding that the rate of SHCHC formation was unaffected by the concentration of the enzyme. Moreover, the absence of SHCHC formation in the enzymatic reaction in low-pH phosphate buffer (pH 6.0–8.0), in which isochorismate is readily consumed, leaves no doubt that SHCHC is not the real product of the MenD-catalyzed reaction. On the basis of the formation of SHCHC and pyruvate in the basification of the MenD turnover product mixture, SEPHCHC was deduced to be the genuine product of MenD, which was supported by spectroscopic analyses. In accordance with these findings, we propose that MenD be designated SEPHCHC synthase.

Experimental observations found that the newly identified MenD product is UV-inactive (260–400 nm) and base-labile. These properties are due to the absence of a π-conjugated electron system and the presence of an acidic α-proton (position 2) of the succinyl carbonyl group in SEPHCHC; the observed specific decomposition of SEPHCHC is due to the thermodynamically favorable formation of an extensive π-conjugation system in SHCHC, which is initiated by dissociation of the acidic proton (Scheme 3). This is the most likely underlying reason for the fact that SHCHC has been consistently mistaken as the MenD product. In previous investigations (30–36, 43), the cell-free assay of MenD activity was exclusively conducted at high pH (pH 8.0 or 8.5) in Tris buffer. Some of these studies also relied on conversion of the presumed SHCHC product to OSB under strong basic conditions (33, 34, 43). These assay conditions inevitably allowed spontaneous decomposition of SEPHCHC, the true MenD product, to form a substantial amount of the detectable SHCHC according to its base lability, whereas

FIGURE 6: Detection of SHCHC synthase activity in the crude extract of E. coli K12. The crude extract from the bacteria grown under anaerobic conditions was extensively dialyzed in 50 mM sodium phosphate buffer (pH 7.0). The reactions contained 200 μM SEPHCHC, 6 mg/mL crude extract, and 100 μg/mL MenC where appropriate in 50 mM sodium phosphate buffer (pH 7.0) and were carried out at room temperature for 1.5 h. Macromolecular components were removed by ultrafiltration before analysis. The early part of the elution profile (2–5 min) indicates the presence of a large amount of chromogenic materials from the crude extract and is omitted for simplicity.

Table 3: Kinetic and Cofactor Binding Constants of MenD$^a$

| substrate or cofactor | $K_M$ or $K_D (\mu M)$ | $k_{cat} (\text{min}^{-1})$ | $k_{cat}/K_M (M^{-1}\cdot\text{min}^{-1})$ |
|------------------------|-------------------------|-------------------------------|-------------------------------------------|
| isochorismate         | 0.053 ± 0.003           | 3.0 ± 0.3                     | (5.6 ± 0.8) × 10⁷                         |
| 2-ketoglutarate        | 1.5 ± 0.3               | 4.5 ± 0.4                     | (3.2 ± 0.7) × 10⁶                         |
| ThDP                   | 2.4 ± 0.3               |                               |                                           |
| Mg²⁺                  | 80 ± 8                  |                               |                                           |

$a$ The reactions were carried out in 50 Tris-HCl buffer (pH 7.8) at room temperature (22.5 ± 0.5 °C). In determining $K_M$ for isochorismate, 2-ketoglutarate was at a saturating concentration of 660 μM, while isochorismate was saturating at 4.0 μM in the determination of $K_M$ for 2-ketoglutarate. ThDP and Mg²⁺ in the kinetic experiments were at a saturating concentration of 50 μM and 5 mM, respectively. In titration of the enzyme activity with a cofactor, both substrates and the other cofactor were at a saturating concentration.

A dialyzed crude extract prepared from E. coli K12 grown under anaerobic conditions was subsequently tested for SHCHC synthase activity. Exogenous MenC was added to the reaction to convert the generated SHCHC to OSB for the purpose of easy detection. In phosphate buffer at pH 7.0, very little OSB was formed in incubating SEPHCHC with MenC, as a result of the slow decomposition of the substrate into SHCHC. In contrast, OSB was readily formed from SHCHC in the presence of MenC or SEPHCHC in the presence of the crude extract and MenC (Figure 6). This indicated that there was indeed SHCHC synthase activity in the extract to convert SEPHCHC to SHCHC, which was

10988 Biochemistry, Vol. 46, No. 38, 2007

**Scheme 3: Base-Catalyzed 2,5-Elimination Reaction of SEPHCHC (8)**

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

**8. SEPHCHC**

the existence of the true turnover product was masked because of its lack of UV absorption. Consequently, the degradation product SHCHC was consistently mistaken as the only product of the enzymatic reaction, while the true MenD product eluded identification. In this way, the spectroscopic and chemical properties of SEPHCHC created the illusion that SHCHC is the only intermediate between isochorismate and OSB in menaquinone biosynthesis.

More support for the incorrect view that only SHCHC is involved in the transformation from isochorismate to OSB is the finding of only two groups of mutants (*menC* and *menD*) that were blocked in the synthesis of OSB (14, 15). The failure of these studies to identify the genuine SHCHC synthase, which is needed to convert SEPHCHC to SHCHC in menaquinone biosynthesis, is unlikely due to a mistake in the original genetic screening but is likely inevitable because of the revealed base lability of its substrate. Conceivably, SHCHC formed from the spontaneous decomposition of the accumulated SEPHCHC in a knockout mutant of the SHCHC synthase would have partially complemented the loss of the enzymic activity. Although the spontaneous transformation is slow at physiological pH (~7.5), a substantial amount of SHCHC should have been formed for subsequent conversion to OSB by MenC because of the prolonged anaerobic growth necessary for the mutant selection (14, 15). As a result, the loss of the OSB synthesis phenotype would not have been clearly discernible for knockout mutants of the SHCHC synthase. Thus, the chemical instability of SEPHCHC most likely obscured the presence of the SHCHC synthase in the genetic studies, in addition to creating the illusion that SHCHC was the turnover product of MenD in cell-free assays. Coincidentally, the two consequences of the unique properties of SEPHCHC are completely consistent with each other and self-sufficient to create the erroneous biosynthetic picture that OSB synthesis from isochorismate involves SHCHC as the only intermediate between two consecutive enzymes, MenD and MenC. In turn, this inaccurate view has helped the true MenD product evade identification until now.

The detection of SHCHC synthase activity in the crude extract of *E. coli* K12 suggests that the bacterial genome codes an enzyme to convert SEPHCHC into SHCHC. This enzyme activity is unlikely to be associated with any of the menaquinone biosynthetic enzymes so far identified because their functions are already assigned. In addition, the gene of this SHCHC synthase is also unlikely to locate in the proximity of the *menA*, *ubiE*, and *menFDHBCE* gene loci (Figure 1), because all the neighboring genes have been unambiguously assigned functions that are unrelated to menaquinone biosynthesis. Despite the lack of clues for its convenient identification, the SHCHC synthase could be isolated from the crude extract of *E. coli* according to its well-defined activity and identified through protein sequencing or peptide fingerprinting using proteomic techniques.

However, this SHCHC synthase will not likely be amenable to identification through the traditional screening of knockout mutants since such mutants are unable to generate a clearly discernible phenotype, as pointed out above.

In summary, we have demonstrated that SEPHCHC is a new intermediate in the biosynthesis of menaquinone, serving as the true product in the first committed step catalyzed by MenD. In its new capacity, MenD is a highly efficient enzyme and is thus renamed SEPHCHC synthase. In addition, the activity of a new enzyme (SHCHC synthase) capable of converting the newly identified intermediate to SHCHC was detected in the crude extract of *E. coli*. These findings suggest that the biosynthetic pathway of menaquinone in *E. coli* as well as all other menaquinone-utilizing microorganisms should be modified, because the biosynthetic enzymes so far identified are conserved across species (44). With revelation of the new intermediate, the stage is set for the identification of its processing enzyme to fully elucidate the ubiquitous biosynthetic pathway.

---

**REFERENCES**

1. Bentley, R. (1990) The shikimate pathway-A metabolic tree with many branches, *Crit. Rev. Biochem. Mol. Biol.* 25, 307–384.
2. Dosselaere, F., and Vanderleyden, J. (2001) A metabolic node in action: chorismate-utilizing enzymes in microorganisms, *Crit. Rev. Microbiol.* 27, 75–131.
3. Meganathan, R. (1996) Biosynthesis of the isoprenoid quinones menaquinone (vitamin K2) and ubiquinone (coenzyme Q), in *Escherichia coli and Salmonella: cellular and molecular biology* (Neidhardt, F. C., Curtiss, R., III, Ingraham, J. L., Lin, E. C. C., Low, K. B., Magasanik, B., Reznikoff, W. S., Riley, M., Schaechter, M., and Umbarger, H. E., Eds.) 2nd ed., Vol. 1, pp 642–656, American Society for Microbiology, Washington, DC.
4. Meganathan, R. (2001) Biosynthesis of menaquinone (Vitamin K2) and ubiquinone (Coenzyme Q): A perspective on enzymatic mechanisms, *Vitam. Horm.* 61, 173–218.
5. Suttie, J. W. (1995) The importance of menaquinones in human nutrition, *Annu. Rev. Nutr.* 15, 399–417.
6. Dowd, P., Ham, S. W., Naganathan, S., and Hershline, R. (1995) The mechanism of action of vitamin K, *Annu. Rev. Nutr.* 15, 419–440.
7. World Health Organization. (2002) *Stop TB annual report*, WHO Report WHO/CDS/STB/2002.17, World Health Organization, Geneva, Switzerland.
8. Truglio, J. J., Theis, K., Feng, Y., Gajda, R., Machutta, C., Tonge, P. J., and Kisker, C. (2003) Crystal structure of *Mycobacterium tuberculosis* MenB, a key enzyme in vitamin K₂ biosynthesis, *J. Biol. Chem.* 278, 42352–42360.
9. Johnston, J. M., Arcus, V. L., and Baker, E. N. (2005) Structure of naphthoate synthase (MenB) from *Mycobacterium tuberculosis* in both native and product-bound forms, *Acta Crystallogr. D61*, 1199–1206.
10. Johnston, J. M., Arcus, V. L., Morton, C. J., Parker, M. W., and Baker, E. N. (2003) Crystal structure of a putative methyltransferase from *Mycobacterium tuberculosis*: Misannotation of a genome clarified by protein structural analysis, *J. Bacteriol.* 185, 4057–4065.
11. Daruwala, R., Kwon, O., Meganathan, R., and Hudspeth, M. E. S. (1996) A new isochorismate synthase specifically involved in menaquinone (vitamin K2) biosynthesis encoded by the *menF* gene, *FEMS Microbiol. Lett.* 140, 159–163.

A Novel Intermediate in Menaquinone Biosynthesis

12. Daruwala, R., Bhattacharyya, D. K., Kwon, O., and Meganathan, R. (1997) Menaquinone (Vitamin K2) biosynthesis: Overexpression, purification, and characterization of a new isochorismate synthase from *Escherichia coli*, *J. Bacteriol.* 179, 3133–3138.

13. Liu, J., Quinn, N., Berchtold, G. A., and Walsh, C. T. (1990) Overexpression, purification, and characterization of isochorismate synthase (EntC), the first enzyme involved in the biosynthesis of enterobactin from chorismate, *Biochemistry* 29, 1417–1425.

14. Guest, J. R. (1979) Anaerobic growth of *Escherichia coli* with fumarate as terminal electron acceptor: Genetic studies with menaquinone and fluoroacetate-resistant mutants, *J. Gen. Microbiol.* 115, 259–271.

15. Meganathan, R., and Bentley, R. (1983) Thiamine pyrophosphate requirement for *o*-succinylbenzoic acid synthesis in *Escherichia coli* and evidence for an intermediate, *J. Bacteriol.* 153, 739–746.

16. Kwon, O., Bhattacharyya, D. K., and Meganathan, R. (1996) Menaquinone (Vitamin K2) biosynthesis: Overexpression, purification, and properties of *o*-succinylbenzoyl-coenzyme A synthetase from *Escherichia coli*, *J. Bacteriol.* 178, 6778–6781.

17. Sharma, V., Suvarna, K., Meganathan, R., and Hudspeth, M. E. S. (1992) Menaquinone (Vitamin K2) biosynthesis: Nucleotide sequence and expression of the *menB* gene from *Escherichia coli*, *J. Bacteriol.* 174, 5057–5062.

18. Kuznetsova, E., Proudfoot, M., Sanders, S. A., Reinking, J., Savchenko, A., Arrowsmith, C. H., Edwards, A. M., and Yakunin, A. F. (2005) Enzyme genomics: Application of general enzymatic screens to discover new enzymes, *FEMS Microbiol. Rev.* 29, 263–279.

19. Suvarna, K., Stevenson, D., Meganathan, R., and Hudspeth, M. E. S. (1998) Menaquinone (Vitamin K2) biosynthesis: Localization and characterization of the *menA* gene from *Escherichia coli*, *J. Bacteriol.* 180, 2782–2787.

20. Lee, P. T., Hsu, A. Y., Ha, H. T., and Clarke, C. F. (1997) A C-methyltransferase involved in both ubiquinone and menaquinone biosynthesis: isolation and identification of the *Escherichia coli ubiE* gene, *J. Bacteriol.* 179, 1748–1754.

21. Lee, K., Zhan, X., Gao, J., Qiu, J., Feng, Y., Meganathan, R., Cohen, S. N., and Georgiou, G. (2003) RraA: a protein inhibitor of RNase E activity that globally modulates RNA abundance in *E. coli*, *Cell* 114, 623–634.

22. Monzingo, A. F., Gao, J., Qiu, J., Georgiou, G., and Robertus, J. D. (2003) The X-ray structure of *Escherichia coli* RraA (MenG), a protein inhibitor of RNA processing, *J. Mol. Biol.* 332, 1015–1024.

23. Taylor, E. A., Palmer, D. R. J., and Gerlt, J. A. (2001) The lesser “burden borne” by *o*-succinylbenzoate synthase: an “easy” reaction involving a carboxylate carbon acid, *J. Am. Chem. Soc.* 123, 5824–5825.

24. Palmer, D. R. J., Garrett, J. B., Sharma, V., Meganathan, R., Babbitt, P. C., and Gerlt, J. A. (1999) Unexpected divergence of enzyme function and sequence: “N-acylamino acid racemase” is *o*-succinylbenzoate synthase, *Biochemistry* 38, 4252–4258.

25. Thompson, T. B., Garrett, J. B., Taylor, E. A., Meganathan, R., Gerlt, J. A., and Rayment, I. (2000) Evolution of enzymatic activity in the enolase superfamily: Structure of *o*-succinylbenzoate synthase from *Escherichia coli* in complex with Mg²⁺ and *o*-succinylbenzoate, *Biochemistry* 39, 10662–10676.

26. Klenchin, V. A., Ringia, E. A. T., Gerlt, J. A., and Rayment, I. (2003) Evolution of enzymatic activity in the enolase superfamily: Structural and mutagenic studies of the mechanism of the reaction catalyzed by *o*-succinylbenzoate synthase from *Escherichia coli*, *Biochemistry* 42, 14427–14433.

27. Ringia, E. A. T., Garrett, J. B., Thoden, J. B., Holden, H. M., Rayment, I. and Gerlt, J. A. (2004) Evolution of enzymatic activity in the enolase superfamily: Functional studies of the promiscuous *o*-succinylbenzoate synthase from *Amycolatopsis*, *Biochemistry* 43, 224–229.

28. Thoden, J. B., Ringia, E. A. T., Garrett, J. B., Gerlt, J. A., Holden, H. M., and Rayment, I. (2004) Evolution of enzymatic activity in

the enolase superfamily: Structural studies of the promiscuous *o*-Succinylbenzoate Synthase from *Amycolatopsis*, *Biochemistry* 43, 5716–5727.

29. Campbell, I. M. (1969) The roles of alanine, aspartate and glutamate in lawsone biosynthesis in impatiens balsamina, *Tetrahedron Lett.* 10, 4777–4780.

30. Meganathan, R. (1981) Enzymes from *Escherichia coli* synthesize *o*-succinylbenzoic acid, an intermediate in menaquinone (Vitamin K2) biosynthesis, *J. Biol. Chem.* 256, 936–9368.

31. Meganathan, R., and Bentley, R. (1983) Thiamine pyrophosphate requirement for *o*-succinylbenzoic acid synthesis in *Escherichia coli* and evidence for an intermediate, *J. Bacteriol.* 153, 739–746.

32. Marley, M. G., Meganathan, R., Bentley, R. (1986) Menaquinone (Vitamin K₂) biosynthesis in *Escherichia coli*: synthesis of *o*-succinylbenzoate does not require the decarboxylase activity of the ketoglutarate dehydrogenase complex, *Biochemistry* 25, 1304–1307.

33. Popp, J. L. (1989) Sequence and overexpression of the *menD* Gene from *Escherichia coli*, *J. Bacteriol.* 171, 4349–4354.

34. Palaniappan, C., Sharma, V., Hudspeth, M. E. S., and Meganathan, R. (1992) Menaquinone (Vitamin K2) biosynthesis: Evidence that the *Escherichia coli menD* gene encodes both 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylic acid synthase and α-ketoglutarate decarboxylase activities, *J. Bacteriol.* 174, 8111–8118.

35. Emmons, G. T., Campbell, I. M., and Bentley, R. (1985) Vitamin K (menaquinone) biosynthesis in bacteria: purification and probable structure of an intermediate prior to *o*-succinylbenzoate, *Biochem. Biophys. Res. Commun.* 131, 956–960.

36. Bhasin, M., Billinsky, J. L., and Palmer, D. R. J. (2003) Steady-state kinetics and molecular evolution of *Escherichia coli* MenD [(1R,6R)-2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate synthase], an anomalous thiamin diphosphate-dependent decarboxylase-carboligase, *Biochemistry* 42, 13496–13504.

37. Sieminska, E. A. L., Macova, A., Palmer, D. R. J., and Sanders, D. A. R. (2005) Crystallization and preliminary X-ray analysis of (1R,6R)-2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate (SHCHC) synthase (MenD) from *Escherichia coli*, *Acta Crystallogr. F61*, 489–492.

38. Rusnak, F., Liu, J., Quinn, N., Berchtold, G. A., and Walsh, C. T. (1990) Subcloning of the enterobactin biosynthetic gene *entB*: expression, purification, characterization, and substrate specificity of isochorismatase synthase, *Biochemistry* 29, 1425–1435.

39. McGowan, D. A., Berchtold, G. A. (1982) Total synthesis of racemic chorismic acid, *J. Am. Chem. Soc.* 104, 1153–1154.

40. Busch, F. R., Berchtold, G. A. (1983) Total synthesis of racemic isochorismic acid, *J. Am. Chem. Soc.* 105, 3346–3347.

41. Burghen, G. A., Kitabchi, A. E., Brush, J. S. (1972) Characterization of a rat liver protease with specificity for insulin, *Endocrinology* 91, 633–642.

42. Weiss, B., Jacquemin-Sablon, A., Live, T. R., Fareed, G. C., Richardson, C. C. (1968) Enzymatic breakage and joining of deoxyribonucleic acid. VI. Further purification and properties of polynucleotide ligase from *Escherichia coli* infected with bacteriophage T4, *J. Biol. Chem.* 243, 4543–4555.

43. Popp, J. L., Berliner, C., and Bentley, R. (1989) Vitamin K (menaquinone) biosynthesis in bacteria: high-performance liquid chromatographic assay of the overall synthesis of *o*-succinylbenzoic acid and of 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylic acid synthase, *Anal. Biochem.* 178, 306–310.

44. Glasner, M. E., Fayazmanesh, N., Chiang, R. A., Sakai, A., Jacobson, M. P., Gerlt, J. A., and Babbitt, P. C. (2006) Evolution of structure and function in the *o*-succinylbenzoate synthase/*N*-acylamino acid racemase family of the enolase superfamily, *J. Mol. Biol.* 306, 228–250.
