
Available online at www.sciencedirect.com

ScienceDirect

Biochimica et Biophysica Acta 1781 (2008) 16–25

**Clostridium scindens baiCD and baiH genes encode stereo-specific 7α/7β-hydroxy-3-oxo-Δ⁴-cholenoic acid oxidoreductases**

Dae-Joong Kang${}^{a,1}$, Jason M. Ridlon${}^{a,1}$, Doyle Ray Moore II${}^{b}$, Stephen Barnes${}^{b}$, Phillip B. Hylemon${}^{a,*}$

${}^{a}$ Department of Microbiology and Immunology, Medical College of Virginia Campus, Virginia Commonwealth University, Richmond, VA 23298, USA  
${}^{b}$ Department of Pharmacology and Toxicology, University of Alabama at Birmingham, Birmingham, AL 35294, USA  

Received 17 April 2007; received in revised form 16 October 2007; accepted 22 October 2007  
Available online 7 November 2007

---

### Abstract

Secondary bile acids, formed by intestinal bacteria, are suggested to play a significant role in cancers of the gastrointestinal tract in humans. Bile acid 7α/β-dehydroxylation is carried out by a few species of intestinal clostridia which harbor a multi-gene bile acid inducible (*bai*) operon. Several genes encoding enzymes in this pathway have been cloned and characterized. However, no gene product(s) has yet been assigned to the production of 3-oxo-Δ⁴-cholenoic acid intermediates of cholic acid (CA), chenodeoxycholic acid (CDCA) or ursodeoxycholic acid (UDCA). We previously reported that the *baiH* gene encodes an NADH:flavin oxidoreductase (NADH:FOR); however, the role of this protein in bile acid 7-dehydroxylation is unclear. Homology searches and secondary structural alignments suggest this protein to be similar to flavoproteins which reduce α/β-unsaturated carbonyl compounds. The *baiH* gene product was expressed in *Escherichia coli*, purified and discovered to be a stereo-specific NAD(H)-dependent 7β-hydroxy-3-oxo-Δ⁴-cholenoic acid oxidoreductase. Additionally, high sequence similarity between the *baiH* and *baiCD* gene products suggests the *baiCD* gene may encode a 3-oxo-Δ⁴-cholenoic acid oxidoreductase specific for CDCA and CA. We tested this hypothesis using cell extracts prepared from *E. coli* overexpressing the *baiCD* gene and discovered that it encodes a stereo-specific NAD(H)-dependent 7α-hydroxy-3-oxo-Δ⁴-cholenoic acid oxidoreductase.

© 2007 Elsevier B.V. All rights reserved.

Keywords: Bile acid 7-dehydroxylation; Flavoprotein; Old yellow enzyme; Enoate reductase; 2,4-dienoyl-CoA reductase; Ursodeoxycholic acid; Clostridium; baiCD; baiH

---

### 1. Introduction

Bile acids are synthesized in the liver from cholesterol. Prior to secretion, bile acids are conjugated to either glycine or taurine and actively transported from the liver via canalicular transporters [1]. Conjugated bile acids – also termed bile salts – are stored in the gallbladder along with unesterified cholesterol, phospholipids and conjugated bilirubin. Upon meal induced hormonal stimulation, the gallbladder contracts, emptying its contents into the small intestine. Bile salts promote the solubilization and absorption of fats, cholesterol, and fat-soluble vitamins in the small intestine. The majority of bile salts are recovered in the ileum by active transport and returned to the liver in the portal circulation [1]. This process of enterohepatic circulation of bile salts occurs several times each day. During this cycling, several hundred milligrams of bile salts are lost into the human colon.

Colonic bacteria are responsible for the biotransformation of conjugates of cholic acid (CA) (3α, 7α, 12α-trihydroxy-5β-cholan-24-oic acid) and chenodeoxycholic acid (CDCA) (3α,7α-dihydroxy-5β-cholan-24-oic acid) into the unconjugated secondary bile acids deoxycholic acid (DCA) (3α,12α-

---

Data presented at Digestive Disease Week 2006, Los Angeles CA, USA (Ref. [57]).

* Corresponding author. Department of Microbiology and Immunology, Medical College of Virginia Campus, Virginia Commonwealth University, P.O. Box 980678, Richmond, VA 23298-0678, USA. Tel.: +1 804 828 2331; fax: +1 804 828 0676.

E-mail address: hylemon@vcu.edu (P.B. Hylemon).

¹ Authors contributed equally to this manuscript.

1388-1981/$ - see front matter © 2007 Elsevier B.V. All rights reserved.  
doi:10.1016/j.bbalip.2007.10.008
dihydroxy-5β-cholan-24-oic acid) and lithocholic acid (LCA) (3α-hydroxy-5β-cholan-24-oic acid), respectively [2]. Unlike rodents, the human liver lacks the ability to 7α-hydroxylate secondary bile acids. Lithocholic acid, due to its intrinsic cytotoxicity is detoxified by species-specific hydroxylation or sulfation and generally rapidly eliminated from the circulating bile acids. DCA is retained to a larger extent due to its higher solubility and can accumulate to high levels in the bile of some individuals and some species, notably the rabbit [2]. Exposure of the colonic epithelium to secondary bile acids is strongly suggested by epidemiologic [3–5], animal models [6–8], and cell culture and signaling studies [9–13] to promote the colon carcinogenesis process.

Conjugated bile salts undergo deconjugation mediated by bacterial hydrolases in the distal small intestine and large intestine. Unconjugated bile acids can then be reversibly epimerized at the 3, 7 and 12 hydroxy groups (reviewed by Refs. [2,14]). Ursodeoxycholic acid (UDCA) (3α, 7β-dihydroxy-5β-cholan-24-oic acid) makes up a small percentage of the biliary pool in healthy individuals (<5%). UDCA is an important therapeutic agent for the treatment of cholestatic liver disease [15–18] and its administration has been suggested to prevent colon cancer [19–21]. UDCA is capable of being directly 7β-dehydroxylated to LCA [22] as well as epimerized to CDCA and then 7α-dehydroxylated to LCA [23–25]. However, genes involved in the 7β-dehydroxylation of UDCA have not yet been identified.

Several genes encoding enzymes in the bile acid 7α-dehydroxylating pathway from the human fecal isolate Clostridium scindens VPI 12708 have been cloned and characterized [2]. The current model suggests that 7α-hydroxy (CA, CDCA) and 7β-hydroxy (UDCA) bile acids are actively transported across the bacterial membrane by an H⁺ dependent bile salt transporter (baiG) [26], conjugated to coenzyme A (baiB) [27], and oxidized at the 3-hydroxy group by NAD(P)(H)-dependent 3α-hydroxysteroid dehydrogenase (baiA) [28]. Oxidation of the 3α-hydroxy group is followed by the formation of 3-oxo-Δ⁴-cholenoic acid-CoA intermediates by unidentified NAD(P)(H)-dependent 3-oxo-Δ⁴-cholenoic acid oxidoreductase(s). At some point in the pathway, the CoA-ester is hydrolyzed or more likely the CoA moiety is transferred to incoming primary bile acids [2,29–31]. The baiF gene product has been shown to have bile acid-CoA hydrolase activity [32]; however, recent evidence indicates that the baiF belongs to a family of enzymes that have CoA transferase activity [2,30]. The gene encoding the bile acid 7α-dehydratase has been shown to be the baiE gene product; however, the 7β-dehydratase has not been identified, although is predicted to be encoded by the baiI, a homologue of the baiE located on the same bai operon [33]. 7α/β-dehydration yields a 3-oxo-4,6-choladienoic acid intermediate which is then reduced in three sequential steps to DCA or LCA, or to the 5α epimers (A/B ring juncture in the cis configuration) allodeoxycholic acid or alloolithocholic acid, respectively. Gene products catalyzing reactions in the reductive arms of the pathway have yet to be elucidated.

We have previously reported that the baiH gene product has NADH:flavin oxidoreductase activity (NADH:FOR) [34,35]. In addition, the baiCD gene product is found to share significant primary and secondary structural identities with the baiH gene product and related proteins whose structures have been elucidated. However, the role of the baiH and baiCD gene products in bile acid 7-dehydroxylation is unclear. In the current article, we provide evidence that the baiCD and baiH gene products from C. scindens VPI 12708 encode stereo-specific NAD(H)-dependent 3-oxo-Δ⁴-cholenoic acid oxidoreductases involved in bile acid 7α-dehydroxylation and bile acid 7β-dehydroxylation, respectively.

## 2. Materials and methods

### 2.1. Bacterial strains, culture media, and conditions

C. scindens VPI 12708 (formerly Eubacterium sp. strain VPI 12708) was cultivated under anaerobic conditions in Brain Heart Infusion (BHI) broth containing sodium cholate (100 μM) and maintained as previously described [36]. Clostridium absonum (ATCC 27555) was purchased from ATCC. C. absonum was initially grown in cooked meat broth for 24 h at 37 °C and subsequently stored at 4 °C and used for no more than 3 weeks as starter culture source. The cells were cultured in BHI supplemented with 0.2% fructose (w/v). Escherichia coli BL21-CodonPlus (DE3)-RIL was purchased from Stratagene (La Jolla, CA) and used for expression of recombinant proteins. Recombinant E. coli cells were grown with shaking in modified TYGPN (tryptone 20 g, yeast extract 10 g, 0.8% glycerol, 5 g Na₃HPO₄, 10 g KNO₃, and 3 μM hematin/l; pH 7.0) containing 100 μg/ml ampicillin.

### 2.2. Polyacrylamide gel electrophoresis and immunoblot analysis

Protein samples were separated and analyzed on a 10% SDS-polyacrylamide gel electrophoresis (PAGE) [37]. Protein samples were prepared for SDS-PAGE by heating for 5 min at 100 °C in an equal volume of sample buffer (0.1 M Tris-HCl, 5% SDS, 0.9% 2-mercaptoethanol (2ME), 20% glycerol, pH 6.8). Gels were stained with 0.2% (w/v) Coomassie brilliant blue R-250 (Sigma, St. Louis, MO) in ethanol:acetic acid:water (30:10:60, v/v/v). A broad-range protein weight marker (Bio-Rad, Hercules, CA) was used for protein size determinations. For western blot analysis, proteins were similarly separated by SDS-PAGE, and electrophoretically transferred to a nitrocellulose membrane by electroblotting using a Bio-Rad Immuno-Blot assay kit. Antiserum against purified baiH gene product was raised in rabbits as described previously [35]. Detection of the baiH gene product on nitrocellulose membranes was performed using a goat anti-rabbit alkaline phosphatase conjugate.

### 2.3. PCR and recombinant DNA methods

Total genomic DNA from C. scindens VPI 12708 was purified as described previously [38] and used as a template for PCR amplification of the baiH gene using primers baiHF–PstI (5′-TAGCACTGGCAGGCCTATGT-3′) containing PstI restriction site and baiHR–SacI (5′-ATCAGAGGCTCATTCGCCTCAC-3′) containing SacI restriction site. The PCR reaction mixture contained 100 ng of C. scindens VPI 12708 genomic DNA, 100 pmol each primer, 0.3 mM dNTPs, Pfu polymerase buffer, and 2.5 U Pfu DNA polymerase in a total volume of 50 μl. Cycling parameters included an initial 3 min denaturation at 94 °C followed by 30 cycles of amplification 94 °C for 35 s, 52 °C for 30 s and 68 °C for 2 min followed by one additional 7 min elongation at 68 °C. PCR was performed on a DNA Thermal cycler (MJ research, Watertown, MA). The PCR product and pSport1 (Invitrogen, Carlsbad, CA) vector were double-digested with PstI and SacI, gel purified (GENECLEAN spin kit) and ligated with T4 DNA ligase (NEB). The plasmid was transformed into E. coli DH5α (Bio-line, Randolph, MA) by electroporation using an Eppendorf 2510 electroporator at 2500 V. The recombinant plasmid (pSport1-baiH) was purified from E. coli DH5α and transformed into E. coli BL21-CodonPlus (DE3)-RIL for over-expression of baiH gene product. The sequence was verified at the Virginia Commonwealth University Nucleic Acid Core Facility using standard primers.
metabolites were extracted with ethyl acetate, dried under a stream of nitrogen gas, dissolved in 100 μl MeOH, spotted, separated by thin layer chromatography on Baker flex silica gel IB-2 plates (J.T. Baker Chemical Co, Philipsburg, NJ) using a solvent system of iso-octane:ethyl acetate:acetic acid (15:10:2 v/v/v) and radiolabeled bile acid metabolites were visualized by autoradiography (Kodak MS film, Rochester, N.Y). Sections of TLC silica gel containing bile acid reaction products were scraped and radioactivity quantified by liquid scintillation spectrometry (Packard, Meriden, CT). Extracts prepared from *E. coli* containing pSport plasmids without *baiH* or *baiCD* gene inserts served as negative controls.

### 2.5. Expression and purification of recombinant *baiH* gene product

*E. coli* BL21-CodonPlus (DE3)-RIL harboring pSport1-*baiH* were grown in 5 ml of TYGPN medium (tryptone 20 g; yeast extract 10 g; glycerol 87 mM; sodium phosphate 35 mM, pH 7.0; KNO₃ 100 mM; hematin 3 μM) with 100 μg of ampicillin/ml at 37 °C for 8 h. The culture was stored at 4 °C overnight. The next day, cells were collected by centrifugation (3000 ×g, 10 min) and suspended in 5 ml of TYGPN medium containing 100 μg/ml ampicillin. This cell suspension was used to inoculate 2 l flasks containing 600 ml TYGPN broth medium containing 100 μg/ml ampicillin. The cells were cultivated at 37 °C with shaking at 250 rpm for about 3 h to reach an A₆₀₀ of about 0.4 at which point the cultures were induced by addition of isopropyl-β-D-thiogalactopyranoside (IPTG) to 1 mM final concentration. Cultures were grown at 25 °C for an additional 4 h at which time the cells were harvested by centrifugation at 5500 ×g, 15 min at 4 °C. The cells were suspended in 400 ml of buffer A (50 mM sodium phosphate buffer; pH 6.8, 2% (w/v) glycerol, 15 mM of 2-mercaptoethanol (2ME)) cells were lysed by sonication twice for 10 min intervals and cell debris removed by centrifugation at 21,000 ×g for 30 min. The supernatant fluid containing soluble recombinant BaiH was removed for further purification.

All purification methods were followed essentially as described previously with some modifications [34]. Frozen cells were suspended in buffer A, disrupted by sonication on ice, and centrifuged at 21,000 ×g for 30 min. Protein was concentrated and protein under 50 kDa removed using Centriprep YM50 (50 kDa cut-off) concentrators (Millipore, Billerica, MA). The protein solution was then applied to a 2 × 10 cm Protein-Pak DEAE 40HR column (Waters, Milford, MA) equilibrated with buffer A. After the column was washed with 60 ml of buffer A, NADH:FOR was eluted with a 500 ml linear gradient of NaCl (0 to 1.0 M) in buffer A at a flow rate of 2 ml min⁻¹. Fractions with a yellow color and NADH:FOR activity were pooled and dialyzed overnight against 2 l of the same buffer. The dialyzed solution was loaded onto a 1.0 × 5.5 cm Cibacron blue 3GA agarose column (Sigma, St. Louis, MO) equilibrated with buffer A. The column was then washed with 100 ml of buffer A, and protein partially eluted with 20 ml of buffer A containing 1 mM NADH, 1 mM FAD, and 20 mM dithiothreitol (DTT). The remaining protein was eluted with buffer A containing 1 M KCl. Active protein fractions eluted with NAD and FAD were pooled and loaded onto a 1.0 × 6.4 cm Bio-Scale CHT5-1 hydroxyapatite column (Bio-Rad, Hercules, CA) equilibrated with buffer C (5 mM potassium phosphate buffer pH 7.0, 2% glycerol, 15 mM 2ME). After the column was washed with buffer C at

### 2.4. Enzyme assays

NADH:FOR activity was determined at 25 °C by monitoring the oxidation of NADH at 340 nm (ε₃₄₀ = 6.22 mM⁻¹ cm⁻¹) using a Beckman DU-640 spectrophotometer (Fullerton, CA). The standard assay mixture (1 ml) contained 100 mM sodium phosphate buffer (pH 6.8), 150 μM NADH, 150 μM FAD and an appropriate amount of enzyme added to initiate the reaction. Assays were performed under aerobic conditions in a 1 ml quartz cuvette with a 1 cm pathlength. One unit of NADH:FOR activity is defined as the amount of enzyme catalyzing the oxidation of 1 μmol of NADH min⁻¹ mg⁻¹ of protein. Protein concentrations were determined according to Bradford [39] using the Bio-Rad protein assay kit, with BSA as standard for routine assays.

3-oxo-Δ⁴-cholenoic acid oxidoreductase activity assays were performed in reaction mixtures containing the following: 50 mM sodium phosphate buffer (pH 6.8), 1.5 mM NAD⁺, 225 μM [24-¹⁴C] 3-dehydro-UDCA (3-oxo-7β-hydroxy) or [24-¹⁴C] 3-dehydro-CDCA (3-oxo-7α-hydroxy) with a specific activity of 0.01 μCi μmol⁻¹ substrate) and purified recombinant BaiH protein (100 μg). The reaction mixture (1 ml) was incubated at 37 °C for 4 h. Reactions were stopped by lowering the pH to ~3.0 with 1 N HCl. Radiolabeled bile acid

Table 1  
Purification scheme of *baiH* gene protein in *E. coli* BL21 CodonPlus (DE3)-RIL (pSport1-*baiH*)

| Step                  | Total volume (ml) | Total protein (mg/ml) | Total activity (U) | Specific activity (fold) | Recovery (%) |
|-----------------------|-------------------|------------------------|--------------------|--------------------------|--------------|
| Cell-free extract     | 35                | 410                    | 93.3               | 0.7                      | 100          |
| Centriprep treatment  | 20                | 175                    | 80                 | 1.2                      | 86           |
| DEAE 40HR cellulose   | 11                | 92                     | 66                 | 3                        | 71           |
| Cibacron blue 3GA agarose | 5.2              | 63                     | 55.5               | 8                        | 60           |
| Hydroxyapatite CHT    | 2.3               | 28                     | 35.1               | 16.4                     | 38           |

D. Kang et al. / Biochimica et Biophysica Acta 1781 (2008) 16–25

![image](https://i.imgur.com/your_image_url.png)

Fig. 2. TLC autoradiograph of BaiCD and BaiH catalyzed NAD(H)-dependent 3-oxo-Δ⁴-cholenoic acid oxidoreductase activity assays. The reactions were carried out in a 500 μl volume consisting of 50 mM sodium phosphate buffer (pH 6.8) containing 225 μM of each respective substrate with a specific activity of 0.01 μCi μmol⁻¹. A: [24-¹⁴C] 3-dehydro-CDCA standard, B: [24-¹⁴C] 3-dehydro-UDCA, C: [24-¹⁴C] 3-dehydro-CDCA + 100 μg cell-free extract of *E. coli* (plasmid, no insert) D: [24-¹⁴C] 3-dehydro-CDCA + 1.6 U purified BaiH, E: [24-¹⁴C] 3-dehydro-CDCA + 100 μg *E. coli* BL21 (DE3) expressing *baiCD*. F: [24-¹⁴C] 3-dehydro-UDCA + 100 μg *E. coli* BL21 (DE3) (plasmid, no insert), G: [24-¹⁴C] 3-dehydro-UDCA + 100 μg cell-free extract of *E. coli* BL21 (DE3) expressing *baiCD*, H: [24-¹⁴C] 3-dehydro-UDCA + 1.6 U purified BaiH. All reactions were performed with 1.5 mM NAD⁺ as co-substrate.

1 ml min⁻¹, active fractions were eluted with a linear gradient of 0 to 0.5 M potassium phosphate (pH 7.0). The purified protein from this step was dialyzed with buffer A, concentrated by a Centriprep YM50 concentrator and stored in buffer A containing 50% glycerol (w/v) at −20 °C. Protein remained active for 4 weeks under these conditions.

### 2.6. Preparation of bile acid substrates

[24-¹⁴C]-CDCA was purchased from American Radiolabeled Chemicals, Inc. (St. Louis, Mo), and determined to be 99.9% pure by TLC. *C. absonum* expresses both 7α-HSDH and 7β-HSDH and can be used to efficiently epimerize CDCA to UDCA [40,41]. [24-¹⁴C]-UDCA was biologically synthesized from [24-¹⁴C]-CDCA using *C. absonum* ATCC 27555 grown in BHI. *C. absonum* was induced at O.D₆₀₀ nm 0.4 by addition of 200 μM sodium chenodeoxycholate (CDCA) plus 2 μCi of [24-¹⁴C]-CDCA for production of [24-¹⁴C]-UDCA (0.01 μCi μmol⁻¹). After 24 h growth following induction, the pH of the culture medium was lowered to ~pH 3 and bile acid metabolites were extracted twice with ethyl acetate, dried under a stream of nitrogen gas and dissolved in 100 μl of absolute methanol (MeOH). [24-¹⁴C]-CDCA, [24-¹⁴C] 7-oxo-LCA and [24-¹⁴C]-UDCA were separated by TLC as described above. Non-radiolabeled CDCA and UDCA standards (Sigma) were also chromatographed along with the radiolabeled reaction products and this portion of the plate stained with phosphomolybdic acid for detection. Radiolabeled bands co-migrating with UDCA standard (Sigma) were scraped and extracted with ethyl acetate, dried, and suspended in MeOH.

[24-¹⁴C]-UDCA was converted to [24-¹⁴C] 3-dehydro-UDCA (3-oxo-7β-hydroxy) enzymatically using commercially available 3α-hydroxysteroid dehydrogenase (Sigma, St. Louis, Mo) in the following reaction mixture: 0.5 ml total volume 50 mM sodium phosphate buffer (pH 6.8), 1 U 3α-hydroxysteroid dehydrogenase, 1.5 mM NAD⁺, 100 μM unlabeled UDCA and 1 μCi [24-¹⁴C]-UDCA. The reaction mixture was allowed to incubate for 4 h at 37 °C. Bile acids were separated by TLC as described above and radiolabeled bile acid reaction products were detected by autoradiography.

### 2.7. Purification and mass spectrometry of bile acid products

The analysis and identification of BaiH and BaiCD catalyzed bile acid reaction product was carried out on a Shimadzu HPLC (Kyoto, Japan) with a SIL-HTC Autosampler by reverse-phase HPLC using a Brownlee RP-300 (Perkin Elmer, Wellesley, MA) C8 7 m 2.1 i.d. × 100 mm column. The mobile phase consisted of 10 mM ammonium acetate (solvent A) and 10 mM ammonium acetate in acetonitrile (solvent B). The column was equilibrated in solvent A. After injection of a 5 ml sample of the bile acid, solvent B was increased from 0–100% at 10% min⁻¹. The column was regenerated by switching back to solvent A and re-equilibrating for 5 min. The flow rate was 0.2 ml min⁻¹ and the eluted material was passed into the electrospray ionization interface of a MDS Sciex/Applied Biosystems API 4000 Q Trap (Foster city, CA) operating in the negative ion mode. Mass spectra were recorded over a 350–700 *m/z* range.

### 2.8. Protein sequence analysis

Homology searches with the *baiH* amino acid sequence from *C. scindens* VPI 12708 (AAC45417) were made against the UniProt database. Multiple sequence alignments were constructed with the T-COFFEE program available on the World Wide Web ([http://www.ch.embnet.org/software/TCoffee.html](http://www.ch.embnet.org/software/TCoffee.html)) [42]. *C. scindens* VPI 12708 *baiH* (AAC45417) and *Clostridium hiranonis baiCD* (AAF22846) genes were sequenced previously [43]. Secondary structural predictions were carried out with the PSIPRED program ([http://bioinf.cs.ucl.ac.uk/psipred/](http://bioinf.cs.ucl.ac.uk/psipred/)) [44,45]. Secondary structural predictions were compared with the known secondary structure from the crystal structure of 2,4-dienoyl-CoA reductase (*dcr*) from *E. coli* [46].

### 2.9. Nucleotide sequence accession number

The *baiC* and *baiD* genes from *C. scindens* were originally reported as separate genes, however, after discovery that the *baiCD* is a single gene product in *C. hiranonis*, the *baiC* and *baiD* gene region was resequenced from *C. scindens* and found to encode a single gene (D.H. Mallonee and P.B. Hylemon, unpublished data). Also, a *baiH* gene has been located downstream of the *bai* operon in *C. hiranonis* and has not been reported previously (J.E. Wells and P.B. Hylemon, unpublished data). GenBank accession numbers for the *baiCD* gene from *C. scindens* VPI 12708 is EF539210 and the *baiH* gene from *C. hiranonis* TO931 is EF539211.

![image](https://i.imgur.com/your_image_url.png)

Fig. 3. SDS-PAGE of cell extract of recombinant *E. coli* BL21 (DE3) overexpressing *baiCD* gene. (M) Marker (UI) uninduced, (I) induced 1 mM IPTG. 30 μg protein loaded and gel stained with Coomassie brilliant blue R-250. A large band at ~70 kDa was detected in IPTG induced cell extracts indicating overexpression of *baiCD* gene.

3. Results

3.1. Purification of recombinant BaiH

Initially, the baiH gene was cloned into a pET24a(+) vector with C-terminal hexa-histidine tag to make a BaiH-His recombinant. The BaiH-His showed NADH:FOR activity in crude extract prior to nickel affinity chromatography. However, following nickel affinity chromatography, the purified enzyme lost detectable NADH:FOR activity. Problems with His-tag use has also been reported for the YqjM protein from *Bacillus subtilis*; a baiH homologue [47]. The quaternary structure of recombinant His-tagged YqjM was reported to be altered by the His-tag resulting in expression of an inactive enzyme. This is not the case with the baiH gene product as the enzyme showed NADH:FOR activity in crude extract similar to previously reported in recombinant BaiH lacking affinity tag [34] before His-binding chromatography (data not shown). The BaiH is known to contain 1 mol copper per subunit [34]. While the mechanism for this loss of activity is not known, we speculate that the nitriloacetic acid, which chelates Ni²⁺, Cu²⁺, and Co²⁺, and in turn binds the hexa-histidine tail, may have chelated copper from the enzyme causing inactivation [48,49]. Addition of various concentrations of Cu²⁺ to the pure enzyme did not restore activity. Therefore, we opted to clone the gene into the pSport1 vector without an affinity tag.

SDS-PAGE analysis of the active fractions after CHT5-1 hydroxyapatite chromatography revealed a highly enriched protein with a subunit molecular mass of 72 kDa that was reactive against anti-BaiH antiserum (Fig. 1). The specific activity and purification fold were dramatically increased by Cibacron blue 3GA and hydroxyapatite chromatography (Table 1). The baiH gene product was purified 23-fold over the crude extract. The final enzyme preparation had a NADH:

FOR specific activity of 16.4 U mg⁻¹. NADH:FOR enzyme activity existed in bright yellow fractions as observed previously [34].

3.2. Stereospecific NAD(H)-dependent 3-oxo-Δ⁴-cholenoic acid oxidoreductase activity

We tested the hypothesis that the baiH gene product encodes an NAD(H)-dependent 3-oxo-Δ⁴-cholenoic acid oxidoreductase by addition of substrates [24-¹⁴C] 3-dehydro-UDCA or [24-¹⁴C] 3-dehydro-CDCA with NAD⁺ and purified BaiH. A single reaction product of more hydrophilic character was detected by autoradiography following TLC when [24-¹⁴C] 3-dehydro-UDCA and NAD⁺ were substrates (Fig. 2). Previous studies have shown the specificity of this enzyme for NADH [34] and the BaiH did not show 3-oxo-Δ⁴-cholenoic acid oxidoreductase activity with NADPH (data not shown). The bile acid metabolite generated following BaiH catalyzed reaction with NAD⁺ was scraped from TLC plates, pooled, further purified by ion exchange chromatography and identified by electrospray mass spectrometry (data not shown). The deprotonated molecular ion given by the product was 2 Da lighter than that given by the 3-dehydro-CDCA substrate, consistent with a loss of 2 hydrogens from the substrate and potentially indicating a 3-dehydro-Δ⁴-CDCA structure. Previous observations showed loss of the 5β-hydrogen during incubation of [5β-³H] [24-¹⁴C]-CA to CA induced whole cells and cell extracts of *C. scindens* VPI 12708 [50]. Taken together, these observations strongly suggest the formation of a 3-dehydro-Δ⁴-UDCA structure.

In addition, we overexpressed the baiCD gene in *E. coli* BL21 (DE3) and tested for 3-oxo-Δ⁴-cholenoic acid oxidoreductase activity. The baiCD gene product was detected by SDS-PAGE at the expected molecular mass of 70 kDa (Fig. 3).

---

Table 2  
Characteristics of selected BaiH homologues  

| Enzyme               | Organism                | Subunit $M_r$ (kDa) | Quaternary Structure | Cofactor          | Electron Donor       | Electron Acceptor    | Percent identity/similarity to BaiH | Accession Number | Reference |
|----------------------|-------------------------|---------------------|-----------------------|--------------------|-----------------------|----------------------|--------------------------------------|-------------------|-----------|
| BaiH                 | *Clostridium scindens* | 72                  | Homotrimer            | 1FAD, Fe-S         | 3-dehydro-UDCA        | NAD⁺                 | 100/100                              | AAC 45417         | This      |
|                      |                         |                     |                       |                    |                       |                      |                                      |                   | [2,15]    |
| BaiCD                | *C. scindens*           | 70                  | ND                    | ND                 | 3-dehydro-CDCA        | NAD⁺                 | 32/46                               | EF539210          | This      |
|                      |                         |                     |                       |                    |                       |                      |                                      |                   | study     |
| Enoate reductase     | *Clostridium tyrobutyricum* | 73                  | Homododecamer        | 1FAD, 1FMN         | NADH                  | α, β-unsaturated carbonyls | 29/48                               | CAA 71086         | [51]      |
| 2,4-dienoyl-CoA reductase | *Escherichia coli* | 73                  | Monomer               | 1FAD, 1FMN         | NADPH                 | Unsaturated fatty acyl-CoA | 29/49                               | 1PS9A             | [21]      |
| Old yellow enzyme (OYE 2) | *Saccharomyces cerevisiae* | 49.9               | Homodimer            | 1FMN               | NADPH                 | α, β-unsaturated carbonyls | 23/42                                | QO3558            | [57,61]   |
| YqjM (OYE)           | *Bacillus subtilis*     | 37.4                | Homotetramer         | 1FMN               | NADPH                 | α, β-unsaturated carbonyls and nitroaromatics | 29/49            | BAA 12619 | [14,26]   |
| NoxB-1               | *Archaeoglobus fulgidus* | 69                  | Monomer               | FAD                | NADH                  | Unknown              | 35/55                               | NP069291          | [25]      |

a Data inconclusive (2).  
b Alignment between full length OYE 2 and amino acids 1–457 of BaiH.  
c Alignment between full length YqjM and amino acids 1–351 of BaiH.

12708baiH
12708H MD MKHSR_LFSPL_QIGSL_TLSNR_VGMAPM_SMD-YEAA_DGTVPKRLADV_FVRR_AEGGT_GYVM_IDAVTIDSK---_YPYMGNNTTA_LDRDELVPQFKEFADRVKEAGSTLVPQIIH----P_GPE 111
TO931H MD MKNSK_LFSPL_TIGSL_TLNRR_VGMAPM_SMD-YEAA_DGTVPKRLADI_FVRR_AEGGT_GYVTI_DAVTIDSK---_YKYMGNTTA_LDSDDLVTQFKEFATR_VREAGSTLIPQVIH----P_GPE 111
12708CD -- MSYEA_LFSP_FKVRGL_ELKNR_IVL_PGMNTK-MAKNKHDIGEDMIA_YHVARAKAGCALNIFECVALCPAP- HAYMYMG----LYTDHHVEQLKKLLTDAVHEAG_GKMGIQWLW----G_GFS 108
TO931CD -- MSYDAL_LFSP_FKIRGL_ELKNR_IVL_PGMNTK-MAKNKHDLSDDMIAYHVARAKAGCALNIFECVALCPAP- HAYMYMG----LYNDNHVAQLKKLLTDAVHEVG_GKMAVQLWH----G_GFS 108
enr    -- MKNKS_LFEPIKIG_KVEKN_KISMAPM_GAFG_LVDNEG_CYNQ_RAVD_YYVERAKGGT_GLITSITK_VE_NEID_KVVPG_VIPIIS_INPGRF_IMTSS_EMTERVHAYGSKIFLQLTMGFGRS_GAP 118
dcr    -- MSYPS_LFAPLD_LGFTT_LKNR_VLMGSMHTG--LEEYP_DGAERLAAFYAERARHGVALIVSGGIAPDLTG----VGMEGGAM_LNDASQ_IPHRTITEAVHQEGGKIALQTLH------- T 105

dcr

12708baiH
12708H SVCG YRH_IAPLGPSA_NTNA_NCHVS_RSISID_EIHDIIKQFGQAARRA_EEAGCGAISLHCAHAYMLPGS_FLSP_LRNRKRMEYGG_SLDNRARFVIEMI_EARRNVSPDFPIFLR_IS-------- 224
TO931H SICG YRH_IAPLGPSV_NTNA_NCHVS_RAISVDEIHEI_IKQFGQAARRVE_EAGCGGIGLHCAHAYMLPGS_FLSP_LRNRKRMEYGG_CLDNRARFVIEMI_EBVRNVSPDFPIMLRIS-------- 224
12708CD PQ   MFFDETN---------TLET_PDTLTVERIHEI_VEEFGRGAR_MAVQAGFD_AVEFHAAH_SYLP-H_EFLSP_GMNKRTDEYGG_SFENRCRFCYEVVQAIRSNIPDDMPFFMRADC-------- 211
TO931CD PQ   MFFDKTN---------TLET_PDTITVERIHEI_VKEFGEARRA_VEAGFD_AVEFHAAH_SYLP-H_EFLSP_GMNKRTDEYGG_NFENRCRFCFVVEAIR_ANIPEDMPFFMRVDC-------- 211
enr    GT   LLTSQPVSASSVP_NYWDPTVTC_RELTTSBEVWIVAKFIQGAAIAQKAGFD_GVEIHAVHEGY_LLDOFTLSIFNRRTDKYGG_DLRRGLQLP_IEIVQGIKAQVGSDFPVGLRYSVKSCIKD 238
dcr    GRY  YQPHLVAPSAL_QAPI_NRFV_PHELSHEEI_LQLIDNFARCAQLAREAGYDGVEVMGS-EGY_LINEFLTTLRTNQRSDQWGGDYRMRFAVEVVARVERVGNDFTIIYRLS---------- 217

dcr

12708baiH
12708H -------G DER_MIGGNSLEDMLYLAPKFVE_AGVNMFEV_SGG-TQYEGLEHI_IPCQNKS_RGVNVYBAS_EIKKVV_GIPVYAVGKIND_IRYAAEIVERGLVDGVAMGRPLLADPD_LCKKAVEG 336
TO931H -------G DERMVGGNSLEDMLYLAPKEAAG_SMLEV_SGG-TQYEGLEHI_IPSQNKSIGVNVHEAS_EIKKVV_DVPVYAVGKIND_IRYAAEIVERGLVDGVSIGRPLLADPD_LCNKAKKEN 336
12708CD ------- IDELME_QTMTEEIVTFINKCAEL_GVDVADLSRGNNATSFA_TVYEVPPFNLAHGFNIENIYN_IKKQINIPVMGVGRINTGEMANKVIEEGKF_DLVGIGRAQLADPNWTTKVREG 324
TO931CD ------- IDELM_DEVMTEEIVEFINRCADL_GVDVADLSRGNAQSFA_TVYEVPPFNLOHGFNIENIYN_IKKQIKIPVMGVGRINTGEMANQVIAADGKF_DLVGIGRAQLADQDWVAKVREG 324
enr    WRQQGLPD EDYVEKGRDLEE_GLESPQIL_EAAG_YDELNTDVG--TYDAWYWSHPPPLYQKD_GLYLPYTQ_ELEKVVKIPVIVAGKLGVPQE_AEKALDEGADMIGLARPLLSDAYWPK_KVLSG 356
dcr    ------- MLDLVEDGGTFAETVELAQAI_EAAG_ATIINTGI_G---WHEARIPTIATP_VPRGAFSWVTRK_LKGHSVSLPLVTTN_RINDPQVADDILSRGDA DMVSMARPFLADAELLSKAQSG 327

dcr

12708baiH
12708H QF DEITPCASC_GGS_CISRSEA_APEC_H--C_HINPRLG_REYEFPDVPAEKS_KKVL_VIGAGPGGM_MAAVTAAERGH_DVTWEAADKI_GGQLNLA_VVAPGKO_EMTQWMVHLNYRA_KKAGVKFEF 454
TO931H LF DEITPCASC_GGS_CISRTPDRPQCR--CHINPRVGFEBYDYPEVPAEKS_KKVL_VVAGAGPGGM_MAAVTAAERGH_DVTLWEADTQIGGQINLAVVAPGKO_EMTKWLSHLNYRA_KKAGVKMVL 454
12708CD KEDLIRHCIGC_DQGCYDAVINPKMKHITCTHNPGCLCLEY-QGMPKTDAPKKVMIVGGGMAGMAIAAEVLKT_RGHNPFEA_SDKLAGQFRLAGVAPMKQDWADVAB_WEAEVERLGEIVRL 445
TO931CD KEDLIRHCIGC_DQGCYDAVINPQMTHITCTRNPHLCLEY-KGMPKTDEPKKVMIIGGGMAGILAAEVLKK_RGHEPFEA_SDHLAGQFVLAGKAPMKEDWAAAAK_WEAEVARLGEIVRY 443
enr    HPERRPCIGC_HVACLGRGFEKPLS--CAVNPAAGRERYEI_RPAI_DKKVL_IAGGGVAGM_EAARMAVL_RGHKVTL_YESTDQLGGEIVPGSVPDFKIDDRL_LDWYRNEMKELKINVIF 474
dcr    RADEINTCIGCNQA CLDQIFVGKVTs--CLVNPRACHETKM_PILPAVQKKNLAVVGAGPAG_LAFAINAAARGHQVTLFDAHSE_IGGQFN_IAKQIPGKEEF_YETLRYYRRMEVTGVTLLK 445

dcr

12708baiH
12708H NKEATAEDVKALAPBAVIVATGA_KPLV_PPI_KGTQDYP_VLTA_HDFLRGKFVIPKG_RVCVLGGGAVACETAE_TALEN_ARPN-----------------SYTRGYD----ASIGDIDVTLVEMLPQ 556
TO931H GK_BATVENIKEFAP_BAVIVATGA_RPLV_PPI_KGTQDYP_VLTA_HDFLRGKFVIPKG_KVCVLGGGAVACETAE_TVLEN_ARPN-----------------AFTR_GFD----ASIGDVDVTLVEMLPQ 556
12708CD NTEVTAEE_TIKEF_NPDNVITAUGSTYALPEIPGID_SPSVYSQYQVLKG-EVNPTGRVAV_ICGGLVGTEVAELLASRGAQV-----------------IAIERKG----V_GTGLSLMRRMFMNPE 544
TO931CD NTKVTPELIEEFAPDHVVIAIGSDY_VAPAIPGID_SDKVYTQYQVLKG-EVEPKGHAVAV_VGCGLVGTEVAQYLAARGAQV-----------------TAIERKG----V_GTGLSLMRRMFMNPE 544
enr    NTEVT_DKLVGKEQPDVVIVATGA_NDVKIKLPGMEKD KVSTA DILNG_TKKSGKN_VLIVGGGLVGCE TA_LYLA_KAGKKV-----------------AI VEAKDKILD_AGKPIPHMNKIMLEDL 579
dcr    NHTVTA_DQLQAF--DETILASGIVPRT_PPIDGIDHPKVL_SYLDVLRD-KAPVGN_KVAIIGCGGIGFD TAMYL_SQPGESTSQNIAGFCNEWGID_SSLQQAGGLSPQ_GMQIPRSP_RQIVMLQ 564

dcr

12708baiH
12708H LLTGV----C APNREP_LIRKLK-S_KGVHINVNTKIMEVTD_HEVKVQRQDGQTQEW_LEGF DYVL_FGLGS RNY_DPSETLK E FVP_EVHVIC--DAVRARQASYAMWEGFEKAYS L 669
TO931H LLTGV----C APNRTPL_LIRKLK-N_KGVHINVNTKILEVTD_HDVKVQRADGAEEW_LKGF DYIL_FGLGS RNY_DPSEQIKEFVP_EVHV VG--DAKRARQASFAMWEAFEAAYS L 661
12708CD F KYYK---- IAKMSGTNVTALEQGKVHY_IMTDKKTKEVT_QGVLECDAT-----------V ICTGITARPSDGLKARCEELGIP_EVIG--DAA GARDCTIATREGYDAGMAI 639
TO931CD F KYYK---- INKMSGTNIVGIEPGKLHY_IMTNKKTQEVT_EGVLECDA A-----------V ICTGITARPSEDLQEKCKELGVPFN_VIG--DAA GARDARIA TQEGYEVGMSI 639
enr    IKKYNN--- IKVITGNSILLEVTDAGAVLIDSFKQQEV_SADTVVISIG-------------F KSNRKLYNKLHGKVT_DLYLIG--DAYQAANIMDAIWSGNEIGLNC 667
dcr    R KASKPGQQ_LGKT TGWIHR TTLSS_RGVKMPIPGVSY_QKIDDDGGLHVING--ETQ_VLAVDNVVICA_GQEPNRA LAQPLIDSGKT_VHLIGGC DVAMELDARRAI AQGTRLALETI 672

dcr

Fig. 4. Amino acid sequence alignment of the BaiH (AAC45417) from C. scindens VPI 12708 with protein homologues: 2,4-dienoyl-CoA reductase (dcr) (AAC76116) from *E. coli*, 2-enoate reductase (CAA71086) from *Clostridium tyrobutyricum*, the baiCD gene product from *Clostridium scindens* (EF539210) and *Clostridium hiranonas* (AAF22846) and the baiH gene product from *C. hiranonis* (EF539211). Secondary structural elements based on the crystal structure of DCR [21] are shown (below alignment). Predicted secondary structural elements of the BaiH protein are also shown (above alignment). Arrows correspond to beta-sheets and loops to alpha helices. Residues corresponding to FMN binding (■), FAD binding (*), and cysteine residues involved in a 4Fe–4S center (♦) from the crystal structure of DCR are also shown. The alignment was generated with the T-COFFEE program [42] and sequence similarities/identities highlighted with Boxshade 3.1 (http://www.ch.embnet.org/software/BOX_form.html).

Addition of *baiCD* overexpressed *E. coli* cell extract to reaction mixture containing 1.5 mM NAD⁺ resulted in a product of greater hydrophilicity than the starting substrate [24-¹⁴C] 3-dehydro-CDCA (Fig. 2). This product was not seen using cell extract of *E. coli* BL21 (DE3) containing pSport1 vector without *baiCD* gene insert. BaiCD did not recognize the substrate [24-¹⁴C] 3-dehydro-UDCA. The product derived from reaction with [24-¹⁴C] 3-dehydro-CDCA was scraped from TLC plates, pooled and subjected to mass spectrometry (data not shown). The major mass ion was 2 amu less than the 3-dehydro-CDCA substrate and thus consistent with the loss of 2 hydrogens from the substrate bile acid and potentially a 3-dehydro-Δ⁴-CDCA structure.

### 3.3. Amino acid sequence alignment and secondary structural prediction of the BaiH

Protein–protein BLAST searches of the *baiH* gene product suggest homology with a family of pyridine-nucleotide dependent flavoproteins (Table 2) including NADH oxidase from *Thermoanaerobacter brockii* (CAA47660) [51], 2-enoyl reductase from *Clostridium tyrobutyricum* (CAA71086) [52] and *Moorella thermoacetica* (CAA76082) [52], 2,4-dienoyl-CoA reductase from several bacterial species, and Old yellow enzyme (OYE) from *Thermoanaerobacter tengcongensis* (NP_623692) [53]. *E. coli* 2,4-dienoyl-CoA reductase (*dcr*) (AAC76116) is a 73 kDa protein which uses NADPH as an electron donor and contains a 4Fe–4S cluster, 1 mol FMN, and 1 mol FAD per mole subunit [46]. The *dcr* gene product from *E. coli* shares 32% identity and 49% similarity at the amino acid sequence level with the *baiH* gene product from *C. scindens* VPI 12708. A 2.4 Å structure has been solved for the *E. coli dcr* gene product allowing a basis of comparison for secondary structure prediction of the *baiH* gene product (Fig. 4). The predicted secondary structure of the BaiH is very similar to that of the *dcr* gene product suggesting a similar TIM-barrel fold. It is interesting to note the conserved cysteine consensus pattern of C-(2X)-C-(2-3X)-C-(11-12X)-C between the BaiH, *E. coli* Dcr and other homologues suggesting that the BaiH and BaiCD contain 4Fe–4S centers.

## 4. Discussion

In this present study, we present evidence that the *baiCD* and *baiH* gene products encode NAD⁺-dependent 3-oxo-Δ⁴-cholenoic acid oxidoreductases recognizing 7α-hydroxy and 7β-hydroxy bile acids, respectively (Fig. 5). Previously, we demonstrated loss of the 5β-hydrogen during 7α-dehydroxylation of [5β-³H]+[24-¹⁴C]-CA both *in vivo* and from cell extracts of CA induced *C. scindens* whole cells suggesting an introduction of a C₄–C₅ double bond [50]. Addition of [24-¹⁴C] 3-oxo-Δ⁴-cholenoic acid to CA induced whole cells and cell extracts of *C. scindens* VPI 12708 resulted in conversion to [24-¹⁴C] DCA [54]. Subsequent studies elucidated the first two steps of the bile acid 7α/β-dehydroxylation pathway; CoA ligation [27] and oxidation of the 3α-hydroxy group producing a 3-dehydro-CA-CoA intermediate [28]. Collectively, these data suggested the presence of an enzyme catalyzing the region-selective oxidation at C₄–C₅ yielding a 3-dehydro-Δ⁴-CA-CoA intermediate. Others have speculated that the *baiH* and *baiCD* genes were involved in C–C double bond oxidation/reduction based on homology to enzymes with similar function [55,56]. The fact that there are two homologous flavin oxidoreductases of similar *M*ᵣ (70–72 kDa) with 32% identity and 46% similarity that are co-expressed from a *bai* operon suggests a possible similar function.

We tested this hypothesis by incubation of a crude cell extract of *E. coli* overexpressing the *baiCD* gene with [24-¹⁴C] 3-dehydro-UDCA or [24-¹⁴C] 3-dehydro-CDCA, and indeed the reaction yielded a 3-dehydro-Δ⁴-CDCA metabolite as suggested by mass spectrometry [57]. 3-dehydro-UDCA did not appear to be a substrate of the *baiCD* gene product. Bile acid 7α-hydroxy groups are axial while 7β-hydroxy groups are equatorial. These data suggest that the key to substrate recognition between the *baiCD* and *baiH* is the stereochemistry of the 7-hydroxy group, whose orientation apparently requires a different architecture in the substrate binding pocket. Taken together these results suggest that these homologous enzymes act at an analogous step in the 7-dehydroxylation pathway, and therefore, two distinct branches of the oxidative arm of the pathway exist in *C. scindens* and *C. hiranonis*.

The BaiH is a homotrimer composed of 72 kDa subunits flavoprotein containing 1 mol of FMN and 1 mol of FAD per subunit. BaiH is homologous to a class of flavoenzymes involved in reduction of unsaturated fatty acids and aldehydes (Table 2) [34]. Enoate reductase (*enr*) from *C. tyrobutyricum* catalyzes the reduction of C–C double bonds of several α/β-unsaturated aldehydes as well as cyclic ketones and methylketones [52]. Enr is a dodecamer with subunit *M*ᵣ of 73 kDa each containing a 4Fe–4S cluster and 1 mol of FMN and 1 mol of FAD [58]. 2,4-dienoyl-CoA reductase from *E. coli* is a 73 kDa

![Proposed reactions catalyzed by the *baiCD* and *baiH* gene products. R group may be the carboxylic acid or the coenzyme A conjugate.](image.png)

Fig. 5. Proposed reactions catalyzed by the *baiCD* and *baiH* gene products. R group may be the carboxylic acid or the coenzyme A conjugate.

monomeric enzyme that reduces C–C double bonds with NADPH and contains FMN, FAD as well as a 4Fe–4S cluster [46,59]. The noxB gene from *Archaeoglobus fulgidus* encodes an NADH oxidase of unknown physiological function [55]. The noxB gene product is a 69 kDa monomer containing 1 mol FAD and shares the same conserved ferrodoxin domain (C-(2X)-C-(2-3X)-C-(11-12X)-C) with each of these proteins including the baiH and baiCD gene products [55]. The noxB-2 gene shares 98.9% identity with the noxB-1 gene from *A. fulgidus* and is found upstream of medium-chain acyl-CoA ligase in this bacterium suggesting these genes are also involved in fatty acid metabolism [55]. The more distantly related Old yellow enzyme (OYE) family homologues are found both in eukaryotes such as *Saccharomyces cerevisiae* (OYE1, OYE2) [56] and *S. carlsbergensis* (OYE1) [60], as well as the prokaryotes including *B. subtilis* (YqjM) [61], and *Shewanella oneidensis* (SYE1–SYE4) [62]. OYE genes have been extensively characterized both biochemically and structurally [58,61,63]. Numerous compounds act as electron acceptors including various quinones, as well as α, β-unsaturated aldehydes and ketones which are reduced at the olefinic bond [63]; however, only recently have physiologically relevant electron acceptors been identified. The oye2 and oye3 genes from *S. cerevisiae*, syel from *S. oneidensis*, and the yqjM gene from *B. subtilis* were shown to be inducible by small α/β-unsaturated aldehydes during oxidative stress responses to lipid peroxidation [47,62,64] and actin cytoskeletal oxidative damage in yeast [65]. Detoxification does not appear to be the function of the bile acid 7α/β-dehydroxylating pathways, but rather the substrates appear to serve as electron sinks to enhance fermentation [2,14].

Electron flow in the *E. coli* DCR as determined by redox titrations [66], and crystallography [46] show that two reducing equivalents are supplied to FAD by direct hydride transfer and follow an electron flow from FAD to a 4Fe–4S cluster to FMN and finally to the oxidized substrate. OYE's reduce C–C double bonds through a ping-pong bi-bi mechanism in which FMN is reduced by NADPH, followed by substrate binding and hydride transfer from FMNH₂ to the Cβ carbon of the substrate [67]. Additional studies will be required to determine the mechanism of the reaction catalyzed by the baiCD and baiH gene products.

We observed low NAD(H)-dependent 3-oxo-Δ⁴-cholenoic acid oxidoreductase activity based on the amount of products formed, as observed by autoradiography of TLC (Fig. 2). However, we would predict that the physiological substrates for these enzymes are the CoA conjugates of 3-oxo-cholanoic acids. This is based on the observations that the baiA gene product encodes a 3α-hydroxysteroid dehydrogenase that has high specificity for coenzyme A conjugated bile acids [28]. However, large quantities of the CoA derivatives of 3-dehydro-CDCA and 3-dehydro-UDCA will have to be chemically synthesized to test this hypothesis. In this regard, we have detected NAD(H)-dependent 3-oxo-Δ⁴-cholenoic acid oxidoreductase activity using small quantities of enzymatically generated radiolabeled CoA conjugates of 3-dehydro-UDCA and 3-dehydro-CDCA (D. Kang, J.M. Ridlon, and P.B. Hylemon, unpublished data). Moreover, other members of this gene family recognize substrates linked to CoA (Table 2). Future mechanistic and

structural studies will require chemical synthesis of CoA conjugated bile acid intermediates.

This is the first report of a gene product specifically involved in 7β-dehydroxylation of UDCA. We predict that the bail gene encodes a bile acid 7β-dehydratase, based on its amino acid sequence identity (20%) and similarity (40%) with the baiE gene product as well as co-expression of this gene product during CA induction in *C. scindens* VPI. We have previously demonstrated that the baiE gene encodes a bile acid 7α-dehydratase [33]. Determining the function of the bail as well as identifying genes involved in reducing the 3-oxo-4,6-choladienoic structure to LCA will be the focus of future research.

In summary, we have provided data that indicate the baiCD and baiH genes encode NAD(H)-dependent 3-oxo-Δ⁴-cholenoic acid oxidoreductases. These results can account for the ability of this organism to use UDCA as a substrate for 7β-dehydroxylation, and provide additional support for formation of a 3-oxo-Δ⁴-cholenoic acid intermediate during bile acid 7-dehydroxylation.

## Acknowledgements

This work was supported by an NIH grant PO1-DK38030, R01-DK46390, Shared Instrumentation Grant from NCRR (S10 RR-19231), and Mass Spectrometry Shared Facility grant (P30 CA-13148). We wish to thank Dr. Alan F. Hofmann for his advice during the preparation of this manuscript.

## References

[1] Z.R. Vlahcevic, D.M. Heuman, P.B. Hylemon, Physiology and pathophysiology of enterohepatic circulation of bile acids, in: D. Zakim, T. Boyer (Eds.), 3rd ed., Hepatology: a Textbook of Liver Diseases, vol. 1, W. B. Saunders, Philadelphia, PA, 1996, pp. 376–417.

[2] J.M. Ridlon, D. Kang, P.B. Hylemon, Bile salt biotransformations by human intestinal bacteria, J. Lipid Res 47 (2006) 241–259.

[3] E. Bayerdörffer, G.A. Mannes, W.O. Richter, T. Ochsenkühn, B. Wiebecke, W. Kopcke, G. Paumgartner, Increased serum deoxycholic acid levels in men with colorectal adenomas, Gastroenterology 104 (1) (1993) 145–151.

[4] B.S. Reddy, E.L. Wynder, Metabolic epidemiology of colon cancer. Fecal bile acids and neutral sterols in colon cancer patients and patients with adenomatous polyps, Cancer 39 (1977) 2533–2539.

[5] M.J. Hill, Bile flow and colon cancer, Mutat. Res 238 (1990) 13–32.

[6] T. Narisawa, N.E. Magadia, J.H. Weisburger, E.L. Wynder, Promoting effect of bile acids on colon carcinogenesis after intrarectal instillation of N-methyl-N′-nitro-N-nitrosoguanidine in rats, J. Natl. Cancer Inst 53 (4) (1974) 1093–1097.

[7] B.S. Reddy, T. Narasawa, J.H. Weisburger, E.L. Wynder, Promoting effect of sodium deoxycholate on colon adenocarcinomas in germfree rats, J. Natl. Cancer Inst 56 (2) (1976) 441–442.

[8] I. Zusman, M. Chevion, N. Kitrosski, Effects of N′-methyl-N′-nitro-N-nitrosoguanidine and deoxycholic acid on the content of free radicals in rat serum, Exp. Toxicol. Pathol 44 (4) (1992) 187–189.

[9] K. Cheng, J. Raufman, Bile acid-induced proliferation of a human colon cancer cell line is mediated by transactivation of epidermal growth factor receptors, Bioch. Pharm 70 (2005) 1035–1047.

[10] P.A. Craven, J. Pfanstiel, F.R. DeRubertis, Role of activation of protein kinase C in the stimulation of colonic epithelial proliferation and reactive oxygen formation by bile acids, J. Clin. Invest 79 (1987) 532–541.

[11] T. Ochsenkuhn, E. Bayerdörffer, A. Meining, M. Schinkel, C. Thiede, V. Nussler, M. Sackmann, R. Hatz, A. Neubauer, G. Paumgartner, Colonic

mucosal proliferation is related to serum deoxycholic acid levels, Cancer 85 (8) (1999) 1664–1669.

[12] R. Pai, A.S. Tarnawski, T. Tran, Deoxycholic acid activates beta-catenin signaling pathway and increases colon cell cancer growth and invasiveness, Mol. Biol. Cell 15 (5) (2004) 2156–2163.

[13] P. Pérez-Ramos, N. Olmo, J. Turnay, E. Lecona, G. González de Buitrago, M.T. Portolés, M.A. Lizarbe, Effect of bile acids on butyrate-sensitive and -resistant human colon adenocarcinoma cells, Nutr. Cancer 53 (2) (2005) 208–219.

[14] M. Begley, C.G. Gahan, C. Hill, The interaction between bacteria and bile, FEMS Microbiol. Rev 29 (4) (2005) 625–665.

[15] P.N. Maton, G.M. Murphy, R.H. Dowling, Ursodeoxycholic acid treatment of gallstones. Dose–response study and possible mechanism of action, Lancet 2 (1977) 1297–1301.

[16] G. Salen, A. Colalillo, D. Verga, E. Bagan, G.S. Tint, S. Shefer, Effect of high and low doses of ursodeoxycholic acid on gallstone dissolution in humans, Gastroenterology 78 (1980) 1412–1418.

[17] G.S. Tint, G. Salen, A. Colalillo, D. Graber, D. Verga, J. Speck, S. Shefer, Ursodeoxycholic acid: a safe and effective agent for dissolving cholesterol gallstones, Ann. Intern. Med. 97 (1982) 351–356.

[18] A. Pares, L. Caballeria, J. Rodes, Excellent long-term survival in patients with primary biliary cirrhosis and biochemical response to ursodeoxycholic acid, Gastroenterology 130 (3) (2006) 715–720.

[19] D.L. Earnest, H. Holubec, R.K. Wali, C.S. Jolley, M. Bissonette, A.K. Bhattacharyya, H. Roy, S. Khare, T.A. Brasitus, Chemoprevention of azoxymethane-induced colonic carcinogenesis by supplemental dietary ursodeoxycholic acid, Cancer Res 54 (19) (1994) 5071–5074.

[20] E. Im, J.D. Martinez, Ursodeoxycholic acid (UDCA) can inhibit deoxycholic acid (DCA)-induced apoptosis via modulation of EGFR/Raf-1/ERK signaling in human colon cancer cells, J. Nutr 134 (2004) 483–486.

[21] S. Khare, S. Cerda, R.K. Wali, F.C. von Lintig, M. Tretiakova, L. Joseph, D. Stoiber, G. Cohen, K. Nimmagadda, J. Hart, M.D. Sitrin, G.R. Boss, M. Bissonnette, Ursodeoxycholic acid inhibits Ras mutations, wild-type Ras activation, and cyclooxygenase-2 expression in colon cancer, Cancer Res 63 (13) (2003) 3517–3523.

[22] B.A. White, R.J. Fricke, P.B. Hylemon, 7β-Dehydroxylation of ursodeoxycholic acid by whole cells and cell extracts of the intestinal anaerobic bacterium *Eubacterium* species VPI 12708, J. Lipid Res 23 (1982) 145–153.

[23] S. Hirano, N. Masuda, Epimerization of the 7-hydroxyl group of bile acids by the combination of two kinds of microorganisms with 7α- and 7β-hydroxysteroid dehydrogenase activity, respectively, J. Lipid Res 22 (1981) 1060–1068.

[24] P. Lepercq, P. Gerard, F. Beguet, P. Raibaud, J.P. Grill, P. Relano, C. Cayuela, C. Juste, Epimerization of chenodeoxycholic acid to ursodeoxycholic acid by *Clostridium baratii* isolated from human feces, FEMS Microbiol. Lett 235 (1) (2004) 65–72.

[25] J.D. Sutherland, I.A. Macdonald, The metabolism of primary, 7-oxo- and 7β-hydroxy bile acids by *Clostridium absonum*, J. Lipid Res 23 (1982) 726–732.

[26] D.H. Mallonee, P.B. Hylemon, Sequencing and expression of a gene encoding a bile acid transporter from *Eubacterium* sp. strain VPI 12708, J. Bacteriol 178 (24) (1996) 7053–7058.

[27] D.H. Mallonee, J.L. Adams, P.B. Hylemon, The bile acid-inducible *baiB* gene from *Eubacterium* sp. strain VPI 12708 encodes a bile acid enzyme A ligase, J. Bacteriol 174 (1992) 2065–2071.

[28] D.H. Mallonee, W.B. White, P.B. Hylemon, Expression in *E. coli* and characterization of a bile acid-inducible 3α-hydroxysteroid dehydrogenase from *Eubacterium* sp. strain VPI 12708, Curr. Microbiol 30 (1995) 259–263.

[29] C. Engemann, T. Elssner, S. Pfeifer, C. Krumbholz, T. Maier, H.-P. Kleber, Identification and functional characterization of genes and corresponding enzymes involved in carnitine metabolism of *Proteus* sp, Arch. Microbiol 183 (2005) 176–189.

[30] J. Heider, A new family of CoA-transferases, FEBS Letters 509 (2001) 345–349.

[31] P. Stenmark, D. Gurmu, P. Nordlund, Crystal structure of CiaB, Type-III CoA transferase in carnitine metabolism, Biochemistry 43 (2004) 13996–14003.

[32] H.Q. Ye, D.H. Mallonee, J.E. Wells, I. Björkhem, P.B. Hylemon, The bile acid-inducible *baiF* gene from *Eubacterium* sp. strain VPI 12708 encodes a bile acid-coenzyme A hydrolase, J. Lipid Res 40 (1999) 17–23.

[33] J.A. Dawson, D.H. Mallonee, I. Björkhem, P.B. Hylemon, Expression and characterization of a *C₂₄* bile acid 7α-dehydratase from *Eubacterium* sp. strain VPI 12708, J. Lipid Res 37 (1996) 1258–1267.

[34] S.F. Baron, P.B. Hylemon, . Expression of the bile acid-inducible NADH: flavin oxidoreductase gene of *Eubacterium* sp. VPI 12708 in *Escherichia coli*, Biochim. Biophys. Acta 1249 (2) (1995) 145–154.

[35] C.V. Franklund, S.F. Baron, P.B. Hylemon, Characterization of *baiH* gene encoding a bile acid-inducible NADH:flavin oxidoreductase from *Eubacterium* sp. strain VPI 12708, J. Bacteriol 175 (1993) 3002–3012.

[36] B.A. White, R.H. Lipsky, R.J. Fricke, P.B. Hylemon, Bile acid induction specificity of 7α-dehydroxylase activity in an intestinal *Eubacterium* species, Steroids 35 (1980) 103–109.

[37] U.K. Laemmli, Cleavage of structural proteins during the assembly of the head of bacteriophage T4, Nature 227 (1970) 680–685.

[38] J. Marmur, A procedure for the isolation of deoxyribonucleic acid from microorganisms, J. Mol. Biol. 3 (1961) 208–218.

[39] M.M. Bradford, A rapid and sensitive method for the quantification of microgram quantities of protein utilizing the principle of protein-dye binding, Anal. Biochem. 72 (1976) 248–254.

[40] I.A. Macdonald, D.M. Hutchison, T.P. Forrest, Formation of urso- and ursodeoxy-cholic acids from primary bile acids by *Clostridium absonum*, J. Lipid Res 22 (1981) 458–466.

[41] I.A. Macdonald, P.D. Roach, Bile salt induction of 7α- and 7β-hydroxysteroid dehydrogenases in *Clostridium absonum*, Biochim. Biophys. Act 665 (1981) 262–269.

[42] C. Notredame, D. Higgins, J. Heringa, T-Coffee: a novel method for multiple sequence alignments, J. Mol. Biol 302 (2000) 205–217.

[43] J.E. Wells, P.B. Hylemon, Identification and characterization of a bile acid 7α-dehydroxylating operon in *Clostridium sp.* strain TO-931, a highly active 7α-dehydroxylating strain isolated from human feces, Appl. Environ. Microbiol 66 (2000) 1107–1113.

[44] D.T. Jones, Protein secondary structure prediction based on position-specific scoring matrices, J. Mol. Biol. 292 (1999) 195–202.

[45] L.J. McGuffin, K. Bryson, D.T. Jones, The PSIPRED protein structure prediction server, Bioinformatics 16 (2000) 404–405.

[46] P.A. Hubbard, X. Liang, H. Schulz, J.P. Kim, The crystal structure and reaction mechanism of *Escherichia coli* 2,4-dienoyl-CoA reductase, J. Biol. Chem 278 (39) (2003) 37553–37560.

[47] T.B. Fitzpatrick, S. Auweter, K. Kitzing, T. Clausen, N. Amrhein, Macheroux, Structural and functional impairment of an Old yellow enzyme homologue upon affinity tag incorporation, Protein Expr. Purif 36 (2004) 280–291.

[48] Y. Li, J. Chen, J. Liu, X. Yang, K. Wang, Binding of Cu²⁺ to S-adenosyl-L-homocysteine hydrolase, J. Inorg. Biochem 98 (2004) 977–983.

[49] E. Sulkowski, Purification of proteins by IMAC, Trends Biotechnol 3 (1985) 1–7.

[50] P.B. Hylemon, P.D. Melone, C.V. Franklund, E. Lund, I. Björkhem, Mechanism of intestinal 7α-dehydroxylation of cholic acid: evidence that allo-deoxycholic acid is an inducible side-product, J. Lipid Res 32 (1991) 89–95.

[51] X.L. Liu, R.K. Scopes, Cloning, sequencing and expression of the gene encoding NADH oxidase from the extreme anaerobic thermophile *Thermoanaerobium brockii*, Biochim. Biophys. Acta 1174 (2) (1993) 187–190.

[52] F. Rohdich, A. Wiese, R. Feicht, H. Simon, A. Bacher, Enoate reductases of *Clostridia*. Cloning, sequencing, and expression, J. Biol. Chem 276 (8) (2001) 5779–5787.

[53] Q. Bao, Y. Tian, W. Li, Z. Xu, Z. Xuan, S. Hu, W. Dong, J. Yang, Y. Chen, Y. Xue, Y. Xu, X. Lai, L. Huang, X. Dong, Y. Ma, L. Ling, H. Tan, R. Chen, J. Wang, J. Xu, H. Yang, A complete sequence of the *T. tengcongensis* genome, Genome Res. 12 (5) (2002) 689–700.

[54] I. Björkhem, K. Einarsson, P.D. Melone, P.B. Hylemon, Mechanism of intestinal formation of deoxycholic acid from cholic acid in humans: evidence for a 3-oxo-Δ⁴-steroid intermediate, J. Lipid Res 30 (1989) 1033–1039.

[55] S.W. Kengen, J. van der Oost, W.M. de Vos, Molecular characterization of H₂O₂-forming NADH oxidases from *Archaeoglobus fulgidus*, Eur. J. Biochem. 270 (2003) 2885–2894.

[56] K. Stott, K. Saito, D.J. Thiele, V. Massey, Old yellow enzyme: the discovery of multiple isozymes and a family of related proteins, J. Biol. Chem 268 (9) (1993) 6097–6106.

[57] J.M. Ridlon, P.B. Hylemon, D. Kang, D. Mallonee, J.E. Wells, D.R. Moore, S. Barnes, The *baiCD* and *baiH* genes from *Clostridium scindens* encode 3-oxo-Δ⁴-steroid oxidoreductases specific for 7α and 7β-hydroxy bile acids, respectively, Gastroenterology 130(4) (Suppl. 2) (2006) A–829.

[58] S. Kuno, A. Bacher, H. Simon, Structure of enoate reductase from a *Clostridium tyrobutyricum* (C. spec. La1), Biol. Chem. Hoppe-Seyler 366 (1985) 463–472.

[59] V. Dommes V, W.H. Kunau, 2,4-Dienoyl coenzyme A reductases from bovine liver and *Escherichia coli*. Comparison of properties, J. Biol. Chem 259 (1984) 1781–1788.

[60] K. Saito, D.J. Thiele, M. Davio, O. Lockridge, V. Massey, The cloning and expression of a gene encoding Old yellow enzyme from *Saccharomyces carlsbergensis*, J. Biol. Chem 266 (31) (1991) 20720–20724.

[61] K. Kitzing, T.B. Fitzpatrick, C. Wilken, J. Sawa, G.P. Bounekov, P. Macheroux, T. Clausen, The 1.3 Å crystal structure of the flavoprotein

[62] A. Brigé, D. van den Hemel, W. Carpentier, L. De Smet, J.J. Van Beeumen, Comparative characterization and expression analysis of the four Old yellow enzyme homologues from *Shewanella oneidensis* indicate differences in physiological function, Biochem. J 394 (2006) 335–344.

[63] A. Vaz, D.N.S. Chakraborty, V. Massey, Old yellow enzyme: aromatization of cyclic enones and the mechanism of a novel dismutation reaction, Biochemistry 34 (1995) 4246–4256.

[64] E.W. Trotter, E.J. Collinson, I.W. Dawes, C.M. Grant, Old yellow enzymes protect against acrolein toxicity in the yeast *Saccharomyces cerevisiae*, Appl. Environ. Microbiol 72 (7) (2006) 4885–4892.

[65] B.K. Haarer, D.C. Amberg, Old yellow enzyme protects the actin cytoskeleton from oxidative stress, Mol. Biol. Cell 15 (2004) 4522–4531.

[66] X. Liang, C. Thorpe, H. Schulz, 2,4-Dienoyl-CoA reductase from *Escherichia coli* is a novel iron–ulfur flavoprotein that functions in fatty acid beta-oxidation, Arch. Biochem. Biophys 380 (2) (2000) 373–379.

[67] B.J. Brown, Z. Deng, P.A. Karplus, V. Massey, On the active site of Old yellow enzyme. Role of histidine 191 and asparagine 194, J. Biol. Chem 273 (49) (1998) 32753–32762.
