
Vol 450|29 November 2007|doi:10.1038/nature06384

REVIEWs

**Protein translocation across the eukaryotic endoplasmic reticulum and bacterial plasma membranes**

Tom A. Rapoport¹

A decisive step in the biosynthesis of many proteins is their partial or complete translocation across the eukaryotic endoplasmic reticulum membrane or the prokaryotic plasma membrane. Most of these proteins are translocated through a protein-conducting channel that is formed by a conserved, heterotrimeric membrane-protein complex, the Sec61 or SecY complex. Depending on channel binding partners, polypeptides are moved by different mechanisms: the polypeptide chain is transferred directly into the channel by the translating ribosome, a ratcheting mechanism is used by the endoplasmic reticulum chaperone BiP, and a pushing mechanism is used by the bacterial ATPase SecA. Structural, genetic and biochemical data show how the channel opens across the membrane, releases hydrophobic segments of membrane proteins laterally into lipid, and maintains the membrane barrier for small molecules.

For almost 40 years, researchers have been fascinated by the question of how proteins are transported across or are integrated into membranes. Pioneering work by G. Palade¹ demonstrated that in eukaryotic cells secretory proteins cross the endoplasmic reticulum membrane before being transported in vesicles to the plasma membrane. The laboratories of G. Blobel and C. Milstein then discovered that these proteins are directed to the endoplasmic reticulum membrane by signal sequences²³. A little later, signal sequences were also found to direct the translocation of proteins across the bacterial plasma membrane⁴⁵. Genetic experiments identified components required for translocation, initially in bacteria and later in yeast⁶⁻⁸, and the establishment of an *in vitro* system initiated biochemical studies⁹. All of these achievements set the stage for investigations into the molecular mechanism of translocation, which will be the focus of this review.

Proteins transported across the eukaryotic endoplasmic reticulum membrane or the prokaryotic plasma membrane include soluble proteins, such as those ultimately secreted from the cell or localized to the endoplasmic reticulum lumen, and membrane proteins, such as those in the plasma membrane or in other organelles of the secretory pathway. Soluble proteins cross the membrane completely and usually have amino-terminal, cleavable signal sequences, the major feature of which is a segment of 7–12 hydrophobic amino acids. Membrane proteins have different topologies in the lipid bilayer, with one or more transmembrane segments composed of about 20 hydrophobic amino acids; the hydrophilic regions of these proteins either cross the membrane or remain in the cytosol. Both types of proteins are handled by the same machinery within the membrane: a protein-conducting channel. The channel allows soluble polypeptides to cross the membrane and hydrophobic transmembrane segments of membrane proteins to exit laterally into the lipid phase.

**Structure of the translocation channel**

The translocation channel is formed from a conserved heterotrimeric membrane protein complex, called the Sec61 complex in eukaryotes and the SecY complex in bacteria and archaea (for more details, see refs 10 and 11). The α- and γ-subunits show significant sequence conservation, and both subunits are essential for the function of the channel and for cell viability. The β-subunits are not essential; they are similar in eukaryotes and archaea, but show no obvious homology to the corresponding subunit in bacteria.

The α-subunit forms the pore of the channel, as initially shown by experiments in which photoreactive probes were systematically placed at different positions of a stalled translocating polypeptide¹²; all positions predicted to be within the membrane cross-linked only to the α-subunit of the Sec61 complex, indicating that this subunit surrounds the polypeptide chain during its passage across the membrane. In addition, experiments in which the purified Sec61/SecY complex was reconstituted into proteoliposomes showed that it is the essential membrane component for protein translocation¹³⁻¹⁵. The channel has an aqueous interior, as demonstrated by electrophysiology experiments¹⁶ and by measurements of the fluorescence lifetime of probes incorporated into a translocating polypeptide chain¹⁷,¹⁸.

The crystal structure of an archael SecY complex provided important insight into how the α-subunit forms the channel¹⁰. The structure is probably representative of complexes from all species, as indicated by sequence conservation and by the similarity to a lower-resolution structure of the *Escherichia coli* SecY complex, determined by electron microscopy from two-dimensional crystals¹⁹,²⁰. Viewed from the cytosol, the channel has a square shape (Fig. 1a). The α-subunit is divided into two halves, transmembrane segments 1–5 and 6–10. The loop between transmembrane segments 5 and 6 at the back of the α-subunit serves as a hinge, allowing the α-subunit to open at the front—the ‘lateral gate’. The γ-subunit links the two halves of the α-subunit at the back by extending one transmembrane segment diagonally across their interface. The β-subunit makes contact only with the periphery of the α-subunit, probably explaining why it is dispensable for the function of the complex.

The ten helices of the α-subunit form an hourglass-shaped pore that consists of cytoplasmic and external funnels, the tips of which meet about half way across the membrane (Fig. 1b). Whereas the

---

¹Howard Hughes Medical Institute and Department of Cell Biology, Harvard Medical School, 240 Longwood Avenue, Boston, Massachusetts 02115, USA.

cytoplasmic funnel is empty, the external funnel is plugged by a short helix. The crystal structure therefore represents a closed channel but, as will be discussed later, biochemical data indicate how it can open and translocate proteins. The constriction of the hourglass-shaped channel is formed by a ring of six hydrophobic residues that project their side chains radially inward. The residues forming this ‘pore ring’ are amino acids with bulky, hydrophobic side chains.

**Different modes of translocation**

The channel alone is a passive pore; it must associate with partners that provide a driving force for translocation. Depending on the partner, there are three known ways in which the channel can function.

In co-translational translocation, the main partner is the ribosome. This mode of translocation is found in all cells and is used for the translocation of secretory proteins as well as for the integration of most membrane proteins. Co-translational translocation begins with a targeting phase. The signal or transmembrane sequence of a growing polypeptide chain is recognized by the signal-recognition particle (SRP); after this, the ribosome–nascent-chain–SRP complex binds to the membrane, first by an interaction between SRP and its membrane receptor, and then by an interaction between the ribosome and

![Figure 1 | The translocation channel.](image1.png)

*Figure 1 | The translocation channel.* **a,** View from the cytosol of the crystal structure of the SecY complex from *Methanococcus jannaschii*. The α-subunit consists of two halves, transmembrane segments 1–5 and 6–10 (in blue and red, respectively), which can open the lateral gate at the front (purple double-headed arrow). The β- and γ-subunits are shown in grey. In the closed channel, the plug (in yellow) is in the centre of the α-subunit. Plug movement towards the back (black double-headed arrow) opens the channel across the membrane. The pore-ring residues are indicated in green. **b,** Cross-sectional view of the channel from the side.

![Figure 2 | Model of co-translational translocation.](image2.png)

*Figure 2 | Model of co-translational translocation.* The scheme is mostly based on experiments with the eukaryotic system, but is probably similar for all organisms.

the translocation channel (Fig. 2; for review of the targeting phase, see refs 21 and 22). The elongating polypeptide chain subsequently moves directly from the tunnel inside the ribosome into the associated membrane channel. GTP hydrolysis is required for chain elongation by the ribosome, but polypeptide movement through the channel is independent of nucleotide hydrolysis²³. In the case of membrane proteins, certain polypeptide segments do not enter the channel, but instead emerge from the ribosome–channel junction into the cytosol²⁴, generating a cytosolic domain.

In most if not all cells, some proteins are transported after completion of their synthesis, that is, post-translationally. This pathway seems to be used by a larger fraction of proteins in simpler organisms, such as bacteria and yeast, perhaps because in these fast-growing cells translocation does not always keep pace with translation. This pathway is used mostly by soluble proteins, such as secretory proteins, which possess only moderately hydrophobic signal sequences that cause them to escape recognition by the SRP during their synthesis²⁵,²⁶. These proteins need to remain unfolded or loosely folded after their release from the ribosome²⁷. Post-translational translocation occurs by different mechanisms in eukaryotes and bacteria.

During post-translational translocation in yeast, and probably in all eukaryotes, the channel partners with another membrane-protein complex, the tetrameric Sec62/Sec63 complex, and with the luminal chaperone BiP, a member of the Hsp70 family of ATPases²⁸,²⁹. In *Saccharomyces cerevisiae*, the Sec62/Sec63 complex consists of the essential Sec62 and Sec63 proteins as well as the dispensable Sec71 (also known as Sec66) and Sec72 proteins. Mammalian cells only have Sec62 and Sec63 (refs 30 and 31). Translocation begins with the binding of a translocation substrate to the channel (Fig. 3). During this step, all cytosolic chaperones are released from the substrate³². Once the polypeptide is inserted into the channel, its translocation occurs by a ratcheting mechanism³³. The polypeptide chain in the channel can slide in either direction by brownian motion, but

![Figure 3 | Model of post-translational translocation in eukaryotes.](image3.png)

*Figure 3 | Model of post-translational translocation in eukaryotes.* It is possible that oligomers of the Sec61 complex mediate translocation, similar to the situation with the other modes of translocation (Figs 2 and 4).

NATURE|Vol 450|29 November 2007

and SecA interact with cytosolic loops in the carboxy-terminal half of Sec61/SecY⁵⁵, and these might transmit conformational changes to other parts of the molecule, resulting in transient displacement of the plug and continuous opening and closing of the lateral gate. This is supported by the observation of increased ion conductance when non-translating ribosomes are bound to the channel¹⁶.

The second step is the intercalation of the hydrophobic segment of a signal sequence into the lateral gate. Photocrosslinking experiments with a yeast *in vitro* system show that the hydrophobic region of a bound signal sequence forms a helix of about two turns, which is intercalated between transmembrane segments 2b and 7 (ref. 56). The signal sequence can also be crosslinked to phospholipid molecules, indicating that it sits at the interface between channel and lipid. The binding of the signal sequence would separate transmembrane segments 2b and 7 and further destabilize plug interactions, causing the plug to move from the centre of Sec61/SecY into a cavity at the back of the molecule. Disulphide-bridge crosslinking shows that the plug indeed comes close to the transmembrane segment of the γ-subunit during translocation⁵⁷,⁵⁸. This model is also consistent with the observation that many mutations that allow the translocation of proteins with defective signal sequences (signal-suppressor mutations) would be expected to destabilize the closed channel¹⁰,⁵⁹.

Finally, the open state of the channel would be fixed by the insertion of the polypeptide segment distal to the signal sequence into the pore proper. During subsequent translocation, the signal sequence stays put, whereas the rest of the polypeptide moves through the pore. The plug could only return to the centre of Sec61/SecY when the polypeptide chain has left the pore. At some point, the signal sequence is cleaved by signal peptidase and is then further degraded by the signal peptide peptidase, a presenilin-like enzyme that cleaves the hydrophobic segment within the membrane⁶⁰.

## The pore

The crystal structure indicates that a single copy of the Sec61/SecY complex forms the pore; according to this model, a polypeptide

Figure 4 | Model of post-translational translocation in bacteria.

its binding to BiP inside the lumen of the endoplasmic reticulum prevents movement back into the cytosol, resulting in net forward translocation. ATP-bound BiP with an open peptide-binding pocket interacts with the J-domain of Sec63, which causes rapid ATP hydrolysis and closure of the peptide-binding pocket around the translocation substrate. J-domain-activated BiP has a low binding specificity³⁴, allowing it to interact with essentially any polypeptide segment that emerges from the channel into the lumen of the endoplasmic reticulum. When the polypeptide has moved sufficiently in the forward direction, the next BiP molecule can bind. This process is repeated until the polypeptide chain has completely traversed the channel. Finally, exchange of ADP for ATP opens the peptide-binding pocket and releases BiP.

In bacterial post-translational translocation, the channel partners with the cytosolic ATPase SecA. SecA has several domains, including two nucleotide-binding folds (NBF1 and NBF2) that bind the nucleotide between them and move relative to one another during the ATP hydrolysis cycle. The other domains also move, perhaps allowing SecA to alternate between the closed and open conformations that are observed in crystal structures³⁵,³⁶. A large groove in the open state might close around the polypeptide chain, because it is similar in dimensions to those seen in other proteins that interact with a wide range of substrates. Several experiments indicate that SecA functions as a monomer during translocation³⁷⁻⁴⁰, but the issue is still controversial⁴¹⁻⁴³. The translocation of many substrates begins with their binding to SecB, a cytosolic chaperone⁴⁴ (Fig. 4). Next, SecA interacts with SecB and accepts the polypeptide, probably binding both the signal sequence and the segment following it⁴⁵⁻⁴⁷. The subsequent transfer of the polypeptide into the channel requires a full cycle of ATP hydrolysis by SecA⁴⁸. Once inserted into the channel, the substrate is translocated by a ‘pushing’ mechanism⁴⁹. Although the details are not yet clear, a plausible mechanism assumes that the polypeptide-binding groove of SecA closes around the polypeptide chain and moves towards the channel, pushing the polypeptide into it (Fig. 4). The size of SecA means that it is unlikely that it inserts deeply into the SecY channel, as proposed earlier⁴⁹,⁵⁰. On nucleotide hydrolysis, the groove opens, releases the peptide, and moves away to ‘grab’ the next segment of the substrate. This cycle continues until the entire polypeptide is translocated. An electrochemical gradient across the membrane stimulates translocation *in vitro* and is required *in vivo*⁵¹, but it is unclear how the gradient is used.

Archaea probably have both co- and post-translational translocation⁵²,⁵³, but it is unknown how post-translational translocation occurs because these organisms lack SecA, the Sec62/Sec63 complex and BiP.

## Opening the channel across the membrane

In all modes, the translocation of a secretory protein begins with its insertion into the channel. The polypeptide inserts as a loop (Fig. 5a), with the signal sequence intercalated into the walls of the channel and the segment distal to it located in the pore proper⁵⁴. Opening of the channel for loop insertion probably occurs in two steps. The first is the binding of a channel partner—the ribosome, the Sec62/Sec63 complex or SecA. This event probably destabilizes interactions that keep the plug in the centre of the Sec61/SecY molecule. The ribosome

Figure 5 | Different stages of translocation. **a**, Translocation of a secretory protein. The red line indicates the hydrophobic region of a signal sequence. Depicted is the co-translational mode of translocation, but similar schemes can be envisioned for the other modes. For simplicity, only the translocating Sec61/SecY copy is shown. **b**, Translocation of membrane proteins. When a hydrophobic transmembrane sequence (in red) has emerged from the ribosome, it can bind reversibly in several conformations. If the hydrophobic sequence is long and the N terminus is not retained in the cytosol, it can flip across the membrane (upper panel). If the N terminus is retained in the cytosol and the polypeptide chain is further elongated by the translating ribosome (indicated by the loop between the ribosome and channel), the C terminus can translocate across the membrane (lower panel).

would move from the cytoplasmic funnel, through the pore ring, into the external funnel. Previously, it was thought that several copies of the Sec61/SecY complex would assemble to form a hydrophilic pore in the membrane, but the crystal structure shows that, similar to all other membrane proteins, a single SecY complex has an entirely hydrophobic belt around its exterior surface¹⁰. Systematic disulphide cross-linking experiments, in which one cysteine is placed in a translocation substrate and another at various positions of SecY, show that the polypeptide chain indeed passes through the centre of the SecY complex and makes contact with it only at the waist of the hourglass-shaped channel⁶¹. The aqueous interior of the channel, its shape and the lack of interactions of the hydrophobic pore residues with the hydrophilic polypeptide backbone all help to minimize the energy required to move a translocation substrate through the membrane.

The diameter of the pore ring, as observed in the crystal structure, is too small to allow the passage of most polypeptide chains. Because even peptide loops of 13 residues and bulky side chains can move through the channel⁶²,⁶³, pore widening has to be postulated, which could be mediated by movements of the helices to which the pore-ring residues are attached. The flexibility of the pore ring is supported by molecular dynamics simulations as well as by electrophysiology experiments⁶⁴–⁶⁷. The intercalation of a signal sequence at the front of Sec61/SecY may cause additional widening of the pore, as is required for loop insertion of a polypeptide chain. The estimated maximum dimensions of the pore on the basis of the crystal structure are 15 × 20 ångström (Å) (ref. 10). A polypeptide in the pore could therefore form an α-helix, but no tertiary structure, in agreement with experimental data⁶⁸.

Results from fluorescence-quenching experiments have indicated that the pore is much larger (40–60 Å) than conceivable with a single Sec61/SecY molecule⁶⁹. The fluorescent probes were incorporated into translocating nascent chains that are associated with stalled, membrane-bound ribosomes; quenching agents as large as NAD⁺ were able to move through the channel to collide with the probes. These data could be reconciled with the crystal structure if two or more Sec61/SecY complexes associated at their front surfaces and opened their lateral gates to fuse their pores into a larger channel. Although a different arrangement—a back-to-back orientation—is suggested by the two-dimensional structure of the *E. coli* SecY complex and by cross-linking data¹⁹,⁷⁰, the model would be consistent with an electron microscopy structure in which a translating *E. coli* ribosome was proposed to be associated with two nearly front-to-front oriented SecY molecules⁷¹. However, this structure is based on a low-resolution electron-density map (~15 Å), and the docking of the SecY crystal structure required its drastic modification. The position and orientation of both SecY molecules are different from that of the single SecY molecule that is seen in more recent electron microscopy structures of non-translating ribosome-SecY complexes⁷². Disulphide-bridge cross-linking experiments argue against fusion of different pores because they show that, during SecA-mediated translocation, both the signal sequence and the mature region of a polypeptide chain are located in the same SecY molecule⁴⁸. A detergent-solubilized translocation intermediate also contains just one copy of SecY associated with one SecA and one translocation substrate molecule³⁸.

If, then, the channel is formed from only one Sec61/SecY molecule, how can one rationalize the fluorescence-quenching results? Although there is currently no good explanation, it should be noted that the fluorescent probes were located deep inside the ribosome, and therefore the same large diameter (40–60 Å) must be assumed for the ribosome tunnel, a size that does not agree with that seen in ribosome structures (<20 Å) determined by crystallography or cryo-electron microscopy⁷³–⁷⁵. One might argue that the tunnel could widen under certain conditions, but this could not be caused simply by the arrival of a nascent polypeptide, because a chain is present in some of the structures. It is also difficult to see how the rigid ribosomal RNA that lines most of the tunnel could undergo such a

dramatic change. It has been proposed that the structural methods give inaccurate answers because they are obtained in detergent⁷⁶, but the disulphide cross-linking experiments were performed with functional translocation intermediates in intact membranes⁴⁸.

### Oligomeric translocation channels

Although the pore is formed by only one Sec61/SecY molecule, translocation of a polypeptide chain seems to be mediated by oligomers. This is best supported by the observation that a SecY molecule defective in SecA-mediated translocation can be rescued by linking it covalently with a wild-type SecY copy⁴⁸. Disulphide-bridge cross-linking showed that SecA binds through its NBF1 domain to a non-translocating SecY copy, and moves the polypeptide chain through a neighbouring SecY copy⁴⁸ (Fig. 4). In this model, the static interaction with the non-translocating SecY molecule would prevent complete detachment of SecA when its peptide-binding domain moves away from the translocating SecY molecule to ‘grab’ the next polypeptide segment.

The Sec61/SecY complex probably forms oligomers during co-translational translocation as well. When a ribosome-nascent-chain-SRP complex binds to the SRP receptor, a domain of SRP undergoes a conformational change that exposes a site on the ribosome to which a single Sec61/SecY molecule could bind⁷⁷—probably the one seen in recent electron microscopy structures of non-translating ribosome-SecY complexes⁷². The bound SecY molecule is close to the point where a polypeptide exits the ribosome and could thus become the translocating copy. At a later stage of translocation, SRP completely detaches from the ribosome, and one or more additional copies of the Sec61/SecY complex may associate, as suggested by cross-linking and freeze-fracture electron microscopy experiments⁷⁸,⁷⁹. These copies could stabilize the ribosome-channel junction and possibly recruit other components, such as signal peptidase and oligosaccharyl transferase, which are needed for polypeptide modification, or the translocon-associated protein complex (TRAP), the function of which is still unclear. On termination of translocation, dissociation of the Sec61/SecY oligomers could facilitate the release of the ribosome from the membrane⁷⁸. Dissociable oligomers may also allow the Sec61/SecY complex to change channel partners and modes of translocation.

Electron microscopy structures of detergent-solubilized ribosome-channel complexes suggested the presence of three or four Sec61 molecules⁸⁰,⁸¹. However, the low resolution of these structures makes it difficult to distinguish between protein and additional density contributed by lipid and detergent. It is therefore possible that only one Sec61 molecule is present, whereas the other Sec61 copies were lost during solubilization, similar to the dissociation seen with SecA-interacting SecY oligomers³⁸.

The emerging concept of homo-oligomeric channels, in which only one copy is active at any given time, may be a common theme in protein translocation. Such a situation may apply for PapC, which is involved in the secretion of pilus subunits across the outer membrane of *E. coli*⁸², and for Tom40 and Tim22, which are involved in protein transport across the outer and inner mitochondrial membrane⁸³,⁸⁴, respectively.

### Membrane-protein integration

During the synthesis of a membrane protein, all transmembrane segments move from the aqueous interior of the channel, through the lateral gate, into the lipid phase. The gate is formed where short segments of four transmembrane segments at the front of Sec61/SecY link the two halves of the molecule¹⁰. The resulting seam in the wall of the channel is probably weak once these transmembrane segments are no longer contacted and stabilized by the plug. The lateral gate may therefore continuously open and close, exposing polypeptide segments located in the aqueous channel to the surrounding hydrophobic lipid phase. Segments that are sufficiently long and hydrophobic to span the entire membrane would exit the channel through
the lateral gate, simply by partitioning between aqueous and hydrophobic environments. This model is supported by photo-cross-linking experiments that examined the lateral exit of a transmembrane segment in different translocation intermediates^{85}, as well as by the agreement between a hydrophobicity scale derived from peptide partitioning into an organic solvent and the tendency of a peptide to span the membrane^{86}. The model is also consistent with molecular dynamics simulations that indicate that lipid molecules do not pass the lateral gate rapidly^{87}. The size of the channel indicates that transmembrane segments exit laterally one by one or in pairs. Hydrophilic segments between the transmembrane segments would move alternately from the ribosome, through the aqueous channel, to the external side of the membrane, or emerge between the ribosome and channel into the cytosol. Movement into the cytosol would utilize a gap between the ribosome and channel, which can be visualized in electron microscopy structures^{81}.

In contrast to a signal sequence, which always has its N terminus in the cytosol, the first transmembrane segment of a membrane protein can have its N terminus on either side of the membrane, depending on the amino-acid sequence of the protein. In a multi-spanning membrane protein, the first transmembrane segment often determines the orientation of the subsequent ones^{88}, although there are exceptions in which internal transmembrane segments have a preferred orientation regardless of the behaviour of preceding transmembrane segments (for a review, see ref. 89).

A model for how the orientation of the first transmembrane segment is determined is shown in Fig. 5. When a hydrophobic segment emerges from the ribosome, it can intercalate reversibly in two different orientations into the lateral gate (Fig. 5b). If the hydrophobic sequence is long and the N terminus is not retained in the cytosol by positive charges or by the folding of the preceding polypeptide segment (for a review, see ref. 89), it can flip across the channel and subsequently exit it laterally into the lipid phase. If the N terminus is retained in the cytosol and the polypeptide chain is further elongated, the C terminus can translocate across the channel, inserting the polypeptide as a loop, as in the case of a secretory protein.

### Maintaining the permeability barrier

The channel must prevent the free movement of small molecules, such as ions or metabolites, both in its resting state and when translocating a polypeptide. Maintaining the membrane barrier is particularly important for prokaryotes, because the proton gradient across the membrane is the main source of their energy. The endoplasmic reticulum membrane may be somewhat leaky to small molecules^{90}, but it must also prevent the free flow of ions.

Results from fluorescence-quenching experiments with endoplasmic reticulum membranes suggest a complex molecular mechanism to maintain the membrane permeability barrier. In this model, the resting channel has a pore size of 9–15 Å, which is closed at the luminal end by BiP, either directly or indirectly^{91}. During the translocation of a secretory protein, the channel widens to 40–60 Å, and the luminal seal is lost, replaced by a cytoplasmic seal from the translating ribosome^{69,91}. When a multi-spanning membrane protein is synthesized, the seals provided by the ribosome and BiP alternate, depending on whether the nascent chain is directed to the endoplasmic reticulum lumen or the cytosol^{92}. The pore is closed by BiP before the transmembrane segment in a nascent chain reaches the channel, implying that the ribosome recognizes the nascent chain as a membrane protein. Fluorescence resonance energy transfer (FRET) and chemical modification experiments support the idea that a transmembrane segment can form an α helix inside the ribosome^{93,94}, but it is difficult to see how the ribosome tunnel, with its mostly hydrophilic surface, could recognize a long hydrophobic sequence. In addition, consecutive transmembrane segments move in the same direction inside the ribosome, but would have to transmit opposing signals to the ribosome-associated channel. A tight seal between the ribosome and channel is also at odds with electron microscopy

structures that reveal a gap of 12–15 Å between them^{80,95}. Finally, this model does not explain how the membrane barrier is maintained in the absence of a ribosome (in post-translational translocation) or in the absence of BiP (in prokaryotes).

The crystal structure indicates a simpler model, in which the membrane barrier is formed by the channel itself, with both the plug and pore ring contributing to the seal^{10}. Electrophysiology experiments show that the resting SecY channel, reconstituted in the absence of other components into a planar membrane, is indeed impermeable to ions and water, and opens on plug displacement^{67}. In the active channel, the pore ring would fit like a gasket around the translocating polypeptide chain, thereby restricting the passage of small molecules during protein translocation. The seal would not be expected to be perfect; in fact, a partial loss of the electrochemical gradient is observed on accumulation of an arrested translocation intermediate in *E. coli* membranes^{96}. Leakage is probably compensated for by powerful ion pumps. During the synthesis of a multi-spanning membrane protein, the seal would be provided in an alternating manner either by the nascent chain in the pore or—once the chain has left the pore—by the plug returning to the centre of Sec61/SecY. This model needs further experimental verification, but it would explain how the membrane barrier can be maintained in both co- and post-translational translocation, and why a gap between the ribosome and channel may not compromise the barrier.

Surprisingly, plug-deletion mutants are viable in *S. cerevisiae* and *E. coli*, and have only moderate translocation defects^{97–99}. However, the crystal structure of these mutants shows that new plugs are formed from neighbouring polypeptide segments^{99}. The new plugs still seal the closed channel, but they have lost many interactions that normally keep the plug in the centre of SecY. This results in continuous channel opening and closing, as observed in electrophysiology experiments^{67}. In addition, facilitated channel opening in the plug-deletion mutants permits polypeptides with defective or even missing signal sequences to be translocated^{98,99}. The plug sequences are only poorly conserved among Sec61/SecY channels, supporting the idea that promiscuous segments can seal the channel and lock it in its closed state.

### Perspective

Progress during the past several years has led to a detailed understanding of protein translocation, in particular of the function of the Sec61/SecY channel. Nevertheless, major questions in the field are still controversial and unresolved. Further progress will require the combination of different approaches. To address how the channel maintains the permeability barrier, electrophysiology experiments are needed to complement the fluorescence-quenching method, particularly because results from the latter are difficult to reconcile with structural data. Important questions in co-translational translocation include how the SRP receptor and channel collaborate, how many Sec61/SecY complexes participate in translocation, and how the ribosome ultimately dissociates from the channel. The precise role of the Sec62/Sec63 components in post-translational translocation, and the mechanism by which SecA moves polypeptides, are also important issues for the future. Membrane-protein integration is still particularly poorly understood, and new methods are required to follow the integration of individual transmembrane segments during the synthesis of a multi-spanning membrane protein. Another unresolved issue concerns other translocation components, such as the translocon-associated membrane protein (TRAM) protein and the translocon-associated protein (TRAP) complex in mammalian cells, or the YidC, SecD and SecF proteins in prokaryotes. These components may be required as chaperones for the folding of transmembrane segments, or to increase the efficiency of translocation of some substrates, but their precise functions remain to be clarified. Much of the progress in the field will depend on the generation of high-resolution structures, with the ‘holy grail’ being a picture of the ‘translocon in action’, in which a channel associated with both a
partner and a translocating polypeptide chain is visualized at atomic detail. However, even lower resolution electron microscopy structures of active translocation complexes are eagerly awaited. The study of the Sec translocation system remains an exciting area of research and is likely to serve as a paradigm for other protein translocation systems, such as those in mitochondria, chloroplasts and peroxisomes.

1. Palade, G. Intracellular aspects of the process of protein synthesis. *Science* **189**, 347–358 (1975).
2. Blobel, G. & Sabatini, D. D. Ribosome-membrane interaction in eukaryotic cells. *Biomembranes* **2**, 193–195 (1971).
3. Milstein, C., Brownlee, G. G., Harrison, T. M. & Mathews, M. B. A possible precursor of immunoglobulin light chains. *Nature New Biol.* **239**, 117–120 (1972).
4. Inouye, H. & Beckwith, J. Synthesis and processing of an Escherichia coli alkaline phosphatase precursor in vitro. *Proc. Natl Acad. Sci. USA* **74**, 1440–1444 (1977).
5. Halegoua, S., Sekizawa, J. & Inouye, M. A new form of structural lipoprotein of outer membrane of Escherichia coli. *J. Biol. Chem.* **252**, 2324–2330 (1977).
6. Emr, S. D., Hanley-Way, S. & Silhavy, T. Suppressor mutations that restore export of a protein with a defective signal sequence. *Cell* **23**, 79–88 (1981).
7. Oliver, D. B. & Beckwith, J. E. coli mutant pleiotropically defective in the export of secreted proteins. *Cell* **25**, 765–772 (1981).
8. Deshaies, R. J. & Schekman, R. A yeast mutant defective at an early stage in import of secretory protein precursors into the endoplasmic reticulum. *J. Cell Biol.* **105**, 633–645 (1987).
9. Blobel, G. & Dobberstein, B. Transfer of proteins across membranes. II. Reconstitution of functional rough microsomes from heterologous components. *J. Cell Biol.* **67**, 852–862 (1975).
10. Van den Berg, B. *et al.* X-ray structure of a protein-conducting channel. *Nature* **427**, 36–44 (2004).
11. Osborne, A. R., Rapoport, T. A. & van den Berg, B. Protein translocation by the Sec61/SecY channel. *Annu. Rev. Cell Dev. Biol.* **21**, 529–550 (2005).
12. Mothes, W., Prehn, S. & Rapoport, T. A. Systematic probing of the environment of a translocating secretory protein during translocation through the ER membrane. *EMBO J.* **13**, 3937–3982 (1994).
13. Akimaru, J., Matsuyama, S., Tokuda, H. & Mizushima, S. Reconstitution of a protein translocation system containing purified SecY, SecE, and SecA from Escherichia coli. *Proc. Natl Acad. Sci. USA* **88**, 6545–6549 (1991).
14. Brundage, L., Hendrick, J. P., Schiebel, E., Driessen, A. J. & Wickner, W. The purified E. coli integral membrane protein SecY/E is sufficient for reconstitution of SecA-dependent precursor protein translocation. *Cell* **62**, 649–657 (1990).
15. Gorlich, D. & Rapoport, T. A. Protein translocation into proteoliposomes reconstituted from purified components of the endoplasmic reticulum membrane. *Cell* **75**, 615–630 (1993).
16. Simon, S. M. & Blobel, G. A protein-conducting channel in the endoplasmic reticulum. *Cell* **65**, 371–380 (1991).
17. Crowley, K. S., Reinhart, G. D. & Johnson, A. E. The signal sequence moves through a ribosomal tunnel into a noncytoplasmic aqueous environment at the ER membrane early in translocation. *Cell* **73**, 1101–1115 (1993).
18. Crowley, K. S., Liao, S. R., Worrell, V. E., Reinhart, G. D. & Johnson, A. E. Secretory proteins move through the endoplasmic reticulum membrane via an aqueous, gated pore. *Cell* **78**, 461–471 (1994).
19. Breyton, C., Haase, W., Rapoport, T. A., Kuhlbrandt, W. & Collinson, I. Three-dimensional structure of the bacterial protein-translocation complex SecYEG. *Nature* **418**, 662–665 (2002).
20. Bostina, M., Mohsin, B., Kuhlbrandt, W. & Collinson, I. Atomic model of the E. coli membrane-bound protein translocation complex SecYEG. *J. Mol. Biol.* **352**, 1035–1043 (2005).
21. Luirink, J. & Sinning, I. SRP-mediated protein targeting: structure and function revisited. *Biochim. Biophys. Acta* **1694**, 17–35 (2004).
22. Halic, M. & Beckmann, R. The signal recognition particle and its interactions during protein targeting. *Curr. Opin. Struct. Biol.* **15**, 116–125 (2005).
23. Connolly, T. & Gilmore, R. Formation of a functional ribosome–membrane junction during translocation requires the participation of a GTP-binding protein. *J. Cell Biol.* **103**, 2253–2261 (1986).
24. Mothes, W. *et al.* Molecular mechanism of membrane protein integration into the endoplasmic reticulum. *Cell* **89**, 523–533 (1997).
25. Ng, D. T., Brown, J. D. & Walter, P. Signal sequences specify the targeting route to the endoplasmic reticulum membrane. *J. Cell Biol.* **134**, 269–278 (1996).
26. Huber, D. *et al.* Use of thioredoxin as a reporter to identify a subset of Escherichia coli signal sequences that promote signal recognition particle-dependent translocation. *J. Bacteriol.* **187**, 2983–2991 (2005).
27. Huber, D. *et al.* A selection for mutants that interfere with folding of Escherichia coli thioredoxin-1 in vivo. *Proc. Natl Acad. Sci. USA* **102**, 18872–18877 (2005).
28. Deshaies, R. J., Sanders, S. L., Feldheim, D. A. & Schekman, R. Assembly of yeast Sec proteins involved in translocation into the endoplasmic reticulum into a membrane-bound multisubunit complex. *Nature* **349**, 806–808 (1991).
29. Panzner, S., Dreier, L., Hartmann, E., Kostka, S. & Rapoport, T. A. Posttranslational protein transport in yeast reconstituted with a purified complex of Sec proteins and Kar2p. *Cell* **81**, 561–570 (1995).

30. Meyer, H. A. *et al.* Mammalian Sec61 is associated with Sec62 and Sec63. *J. Biol. Chem.* **275**, 14550–14557 (2000).
31. Tyedmers, J. *et al.* Homologs of the yeast Sec complex subunits Sec62p and Sec63p are abundant proteins in dog pancreas microsomes. *Proc. Natl Acad. Sci. USA* **97**, 7214–7219 (2000).
32. Plath, K. & Rapoport, T. A. Spontaneous release of cytosolic proteins from posttranslational substrates before their transport into the endoplasmic reticulum. *J. Cell Biol.* **151**, 167–178 (2000).
33. Matlack, K. E., Misselwitz, B., Plath, K. & Rapoport, T. A. BiP acts as a molecular ratchet during posttranslational transport of prepro-α factor across the ER membrane. *Cell* **97**, 553–564 (1999).
34. Misselwitz, B., Staeck, O. & Rapoport, T. A. J proteins catalytically activate Hsp70 molecules to trap a wide range of peptide sequences. *Mol. Cell* **2**, 593–603 (1998).
35. Hunt, J. F. *et al.* Nucleotide control of interdomain interactions in the conformational reaction cycle of SecA. *Science* **297**, 2018–2026 (2002).
36. Osborne, A. R., Clemons, W. M. Jr & Rapoport, T. A. A large conformational change of the translocation ATPase SecA. *Proc. Natl Acad. Sci. USA* **101**, 10937–10942 (2004).
37. Or, E., Navon, A. & Rapoport, T. Dissociation of the dimeric SecA ATPase during protein translocation across the bacterial membrane. *EMBO J.* **21**, 4470–4479 (2002).
38. Duong, F. Binding, activation and dissociation of the dimeric SecA ATPase at the dimeric SecYEG translocase. *EMBO J.* **22**, 4375–4384 (2003).
39. Or, E., Boyd, D., Gon, S., Beckwith, J. & Rapoport, T. The bacterial ATPase SecA functions as a monomer in protein translocation. *J. Biol. Chem.* **280**, 9097–9105 (2004).
40. Alami, M., Dalal, K., Lelj-Garolla, B., Sligar, S. G. & Duong, F. Nanodiscs unravel the interaction between the SecYEG channel and its cytosolic partner SecA. *EMBO J.* **26**, 1995–2004 (2007).
41. Jilaveanu, L. B., Zito, C. R. & Oliver, D. Dimeric SecA is essential for protein translocation. *Proc. Natl Acad. Sci. USA* **102**, 7511–7516 (2005).
42. de Keyzer, J. *et al.* Covalently dimerized SecA is functional in protein translocation. *J. Biol. Chem.* **280**, 35255–35260 (2005).
43. Mitra, K., Frank, J. & Driessen, A. Co- and post-translational translocation through the protein-conducting channel: analogous mechanisms at work? *Nature Struct. Mol. Biol.* **13**, 957–964 (2006).
44. Randall, L. L. *et al.* Binding of SecB to ribosome-bound polypeptides has the same characteristics as binding to full-length, denatured proteins. *Proc. Natl Acad. Sci. USA* **94**, 802–807 (1997).
45. Lill, R., Dowhan, W. & Wickner, W. The ATPase activity of SecA is regulated by acidic phospholipids, SecY, and the leader and mature domains of precursor proteins. *Cell* **60**, 271–280 (1990).
46. Randall, L. L. & Hardy, S. J. SecB, one small chaperone in the complex milieu of the cell. *Cell. Mol. Life Sci.* **59**, 1617–1623 (2002).
47. Musial-Siwek, M., Rusch, S. L. & Kendall, D. A. Selective photoaffinity labeling identifies the signal peptide binding domain on SecA. *J. Mol. Biol.* **365**, 637–648 (2006).
48. Osborne, A. R. & Rapoport, T. A. Protein translocation is mediated by oligomers of the SecY complex with one SecY copy forming the channel. *Cell* **129**, 97–110 (2007).
49. Economou, A. & Wickner, W. SecA promotes preprotein translocation by undergoing ATP-driven cycles of membrane insertion and deinsertion. *Cell* **78**, 835–843 (1994).
50. Kim, Y. J., Rajapandi, T. & Oliver, D. SecA protein is exposed to the periplasmic surface of the E. coli inner membrane in its active state. *Cell* **78**, 845–853 (1994).
51. Schiebel, E., Driessen, A. J., Hartl, F. U. & Wickner, W. ΔμH+ and ATP function at different steps of the catalytic cycle of preprotein translocase. *Cell* **64**, 927–939 (1991).
52. Irhimovitch, V. & Eichler, J. Post-translational secretion of fusion proteins in the halophilic archaea Haloferax volcanii. *J. Biol. Chem.* **278**, 12881–12887 (2003).
53. Ortenberg, R. & Mevarech, M. Evidence for post-translational membrane insertion of the integral membrane protein bacterioopsin expressed in the heterologous halophilic archaeon Haloferax volcanii. *J. Biol. Chem.* **275**, 22839–22846 (2000).
54. Shaw, A. S., Rottier, P. J. & Rose, J. K. Evidence for the loop model of signal-sequence insertion into the endoplasmic reticulum. *Proc. Natl Acad. Sci. USA* **85**, 7592–7596 (1988).
55. Raden, D., Song, W. & Gilmore, R. Role of the cytoplasmic segments of Sec61α in the ribosome-binding and translocation-promoting activities of the Sec61 complex. *J. Cell Biol.* **150**, 53–64 (2000).
56. Plath, K., Mothes, W., Wilkinson, B. M., Stirling, C. J. & Rapoport, T. A. Signal sequence recognition in posttranslational protein transport across the yeast ER membrane. *Cell* **94**, 795–807 (1998).
57. Harris, C. R. & Silhavy, T. J. Mapping an interface of SecY (PrIA) and SecE (PrIG) by using synthetic phenotypes and in vivo cross-linking. *J. Bacteriol.* **181**, 3438–3444 (1999).
58. Tam, P. C., Maillard, A. P., Chan, K. K. & Duong, F. Investigating the SecY plug movement at the SecYEG translocation channel. *EMBO J.* **24**, 3380–3388 (2005).
59. Smith, M. A., Clemons, W. M. Jr, DeMars, C. J. & Flower, A. M. Modeling the effects of prl mutations on the Escherichia coli SecY complex. *J. Bacteriol.* **187**, 6454–6465 (2005).

60. Weihofen, A., Binns, K., Lemberg, M. K., Ashman, K. & Martoglio, B. Identification of signal peptide peptidase, a presenilin-type aspartic protease. *Science* 296, 2215–2218 (2002).

61. Cannon, K. S., Or, E., Clemons, W. M. Jr, Shibata, Y. & Rapoport, T. A. Disulfide bridge formation between SecY and a translocating polypeptide localizes the translocation pore to the center of SecY. *J. Cell Biol.* 169, 219–225 (2005).

62. Tani, K., Tokuda, H. & Mizushima, S. Translocation of proOmpA possessing an intramolecular disulfide bridge into membrane vesicles of *Escherichia coli*. Effect of membrane energization. *J. Biol. Chem.* 265, 17341–17347 (1990).

63. Kurzchalia, T. V. *et al.* tRNA-mediated labelling of proteins with biotin. A nonradioactive method for the detection of cell-free translation products. *Eur. J. Biochem.* 172, 663–668 (1988).

64. Gumbart, J. & Schulten, K. Molecular dynamics studies of the archaeal translocon. *Biophys. J.* 90, 2356–2367 (2006).

65. Tian, P. & Andricioaei, I. Size, motion, and function of the SecY translocon revealed by molecular dynamics simulations with virtual probes. *Biophys. J.* 90, 2718–2730 (2006).

66. Haider, S., Hall, B. A. & Sansom, M. S. Simulations of a protein translocation pore: SecY. *Biochemistry* 45, 13018–13024 (2006).

67. Saparov, S. M. *et al.* Determining the conductance of the SecY protein translocation channel for small molecules. *Mol. Cell* 26, 501–509 (2007).

68. Kowarik, M., Kung, S., Martoglio, B. & Helenius, A. Protein folding during cotranslational translocation in the endoplasmic reticulum. *Mol. Cell* 10, 769–778 (2002).

69. Hamman, B. D., Chen, J. C., Johnson, E. E. & Johnson, A. E. The aqueous pore through the translocon has a diameter of 40–60 Å during cotranslational protein translocation at the ER membrane. *Cell* 89, 535–544 (1997).

70. Kaufmann, A., Manting, E. H., Veenendaal, A. K., Driessen, A. J. & van der Does, C. Cysteine-directed cross-linking demonstrates that helix 3 of SecE is close to helix 2 of SecY and helix 3 of a neighboring SecE. *Biochemistry* 38, 9115–9125 (1999).

71. Mitra, K. *et al.* Structure of the *E. coli* protein-conducting channel bound to a translating ribosome. *Nature* 438, 318–324 (2005).

72. Ménetret, J. F. *et al.* Ribosome binding of a single copy of the SecY complex: implications for protein translocation. *Mol. Cell* (in the press).

73. Voss, N. R., Gerstein, M., Steitz, T. A. & Moore, P. B. The geometry of the ribosomal polypeptide exit tunnel. *J. Mol. Biol.* 360, 893–906 (2006).

74. Halic, M. *et al.* Following the signal sequence from ribosomal tunnel exit to signal recognition particle. *Nature* 444, 507–511 (2006).

75. Mitra, K. *et al.* Elongation arrest by SecM via a cascade of ribosomal RNA rearrangements. *Mol. Cell* 22, 533–543 (2006).

76. Johnson, A. E. Maintaining the permeability barrier during protein trafficking at the endoplasmic reticulum membrane. *Biochem. Soc. Trans.* 31, 1227–1231 (2003).

77. Halic, M. *et al.* Signal recognition particle receptor exposes the ribosomal translocon binding site. *Science* 312, 745–747 (2006).

78. Schaletzky, J. & Rapoport, T. A. Ribosome binding to and dissociation from translocation sites of the endoplasmic reticulum membrane. *Mol. Biol. Cell* 17, 3860–3869 (2006).

79. Hanein, D. *et al.* Oligomeric rings of the Sec61p complex induced by ligands required for protein translocation. *Cell* 87, 721–732 (1996).

80. Beckmann, R. *et al.* Architecture of the protein-conducting channel associated with the translating 80S ribosome. *Cell* 107, 361–372 (2001).

81. Ménetret, J. F. *et al.* Architecture of the ribosome-channel complex derived from native membranes. *J. Mol. Biol.* 348, 445–457 (2005).

82. Thanassi, D. G., Stathopoulos, C., Karkal, A. & Li, H. Protein secretion in the absence of ATP: the autotransporter, two-partner secretion and chaperone/ usher pathways of Gram-negative bacteria. *Mol. Membr. Biol.* 22, 63–72 (2005).

83. Ahting, U. *et al.* The TOM core complex: the general protein import pore of the outer membrane of mitochondria. *J. Cell Biol.* 147, 959–968 (1999).

84. Rehling, P. *et al.* Protein insertion into the mitochondrial inner membrane by a twin-pore translocase. *Science* 299, 1747–1751 (2003).

85. Heinrich, S. U., Mothes, W., Brunner, J. & Rapoport, T. A. The Sec61p complex mediates the integration of a membrane protein by allowing lipid partitioning of the transmembrane domain. *Cell* 102, 233–244 (2000).

86. Hessa, T. *et al.* Recognition of transmembrane helices by the endoplasmic reticulum translocon. *Nature* 433, 377–381 (2005).

87. Gumbart, J. & Schulten, K. Structural determinants of lateral gate opening in the protein translocon. *Biochemistry* 46, 11147–11157 (2007).

88. Wessels, H. P. & Spiess, M. Insertion of a multispanning membrane protein occurs sequentially and requires only one signal sequence. *Cell* 55, 61–70 (1988).

89. Rapoport, T. A., Goder, V., Heinrich, S. U. & Matlack, K. E. Membrane-protein integration and the role of the translocation channel. *Trends Cell Biol.* 14, 568–575 (2004).

90. Le Gall, S., Neuhof, A. & Rapoport, T. The endoplasmic reticulum membrane is permeable to small molecules. *Mol. Biol. Cell* 15, 447–455 (2004).

91. Hamman, B. D., Hendershot, L. M. & Johnson, A. E. BiP maintains the permeability barrier of the ER membrane by sealing the lumenal end of the translocon pore before and early in translocation. *Cell* 92, 747–758 (1998).

92. Liao, S., Lin, J., Do, H. & Johnson, A. E. Both lumenal and cytosolic gating of the aqueous ER translocon pore are regulated from inside the ribosome during membrane protein integration. *Cell* 90, 31–41 (1997).

93. Woolhead, C. A., McCormick, P. J. & Johnson, A. E. Nascent membrane and secretory proteins differ in FRET-detected folding far inside the ribosome and in their exposure to ribosomal proteins. *Cell* 116, 725–736 (2004).

94. Lu, J. & Deutsch, C. Secondary structure formation of a transmembrane segment in Kv channels. *Biochemistry* 44, 8230–8243 (2005).

95. Ménetret, J. F. *et al.* The structure of ribosome-channel complexes engaged in protein translocation. *Mol. Cell* 6, 1219–1232 (2000).

96. Schiebel, E. & Wickner, W. Preprotein translocation creates a halide anion permeability in the *Escherichia coli* plasma membrane. *J. Biol. Chem.* 267, 7505–7510 (1992).

97. Junne, T., Schwede, T., Goder, V. & Spiess, M. The plug domain of yeast Sec61p is important for efficient protein translocation, but is not essential for cell viability. *Mol. Biol. Cell* 17, 4063–4068 (2006).

98. Maillard, A. P., Lalani, S., Silva, F., Belin, D. & Duong, F. Deregulation of the SecYEG translocation channel upon removal of the plug domain. *J. Biol. Chem.* 282, 1281–1287 (2007).

99. Li, W. *et al.* The plug domain of the SecY protein stabilizes the closed state of the translocation channel and maintains a membrane seal. *Mol. Cell* 26, 511–521 (2007).

Acknowledgements | thank A. Osborne, Y. Shibata, B. van den Berg and K. Matlack for critical reading of the manuscript, and W. Li and V. Goder for help with the figures.

Author Information | Reprints and permissions information is available at www.nature.com/reprints. Correspondence and requests for materials should be addressed to T.A.R. (tom_rapoport@hms.harvard.edu).
