
Genome-wide gene expression profiling reveals renal genes regulated during metabolic acidosis

Marta Nowik, M. Rita Lecca, Ana Velic, Hubert Rehrauer, André W. Brändli and Carsten A. Wagner

*Physiol. Genomics* 32:322-334, 2008. First published 4 December 2007; doi:10.1152/physiolgenomics.00160.2007

---

**You might find this additional info useful...**

**Supplemental material** for this article can be found at:
/content/suppl/2007/12/06/00160.2007.DC1.html

**This article cites** 49 articles, 33 of which can be accessed free at:
/content/32/3/322.full.html#ref-list-1

**This article has been cited by** 21 other HighWire hosted articles, the first 5 are:

- Impaired expression of key molecules of ammoniagenesis underlies renal acidosis in a rat model of chronic kidney disease  
  Remy Bürki, Nilufar Mohebbi, Carla Bettoni, Xueqi Wang, Andreas L. Serra and Carsten A. Wagner  
  *Nephrol. Dial. Transplant.*, December 18, 2014; .  
  [Abstract] [Full Text] [PDF]

- Angiotensin II type 2 receptor regulates ROMK-like K⁺ channel activity in the renal cortical collecting duct during high dietary K⁺ adaptation  
  Yuan Wei, Yi Liao, Beth Zavilowitz, Jin Ren, Wen Liu, Pokman Chan, Rajeev Rohatgi, Genevieve Estilo, Edwin K. Jackson, Wen-Hui Wang and Lisa M. Satlin  
  *Am J Physiol Renal Physiol*, October 1, 2014; 307 (7): F833-F843.  
  [Abstract] [Full Text] [PDF]

- Autophagic Clearance of Mitochondria in the Kidney Copes with Metabolic Acidosis  
  Tomoko Namba, Yoshitsugu Takabatake, Tomonori Kimura, Atsushi Takahashi, Takeshi Yamamoto, Jun Matsuda, Harumi Kitamura, Fumio Niimura, Taiji Matsusaka, Hirotsugu Iwatani, Isao Matsui, Junya Kaimori, Hidetaka Kioka, Yoshitaka Isaka and Hiromi Rakugi  
  *JASN*, October , 2014; 25 (10): 2254-2266.  
  [Abstract] [Full Text] [PDF]

- Reduction of Proteinuria through Podocyte Alkalinization  
  Mehmet M. Altintas, Kumiko Moriwaki, Changli Wei, Clemens C. Möller, Jan Flesche, Jing Li, Suma Yaddanapudi, Mohd Hafeez Faridi, Markus Gödel, Tobias B. Huber, Richard A. Preston, Jean X. Jiang, Dontscho Kerjaschki, Sanja Sever and Jochen Reiser  
  *J. Biol. Chem.*, June 20, 2014; 289 (25): 17454-17467.  
  [Abstract] [Full Text] [PDF]

- Autophagic Clearance of Mitochondria in the Kidney Copes with Metabolic Acidosis  
  Tomoko Namba, Yoshitsugu Takabatake, Tomonori Kimura, Atsushi Takahashi, Takeshi Yamamoto, Jun Matsuda, Harumi Kitamura, Fumio Niimura, Taiji Matsusaka, Hirotsugu Iwatani, Isao Matsui, Junya Kaimori, Hidetaka Kioka, Yoshitaka Isaka and Hiromi Rakugi  
  *JASN*, April 3, 2014; .  
  [Abstract] [Full Text] [PDF]

**Updated information and services** including high resolution figures, can be found at:
/content/32/3/322.full.html

**Additional material and information** about *Physiological Genomics* can be found at:
http://www.the-aps.org/publications/pg

---

This information is current as of February 18, 2015.

---

*Physiological Genomics* publishes results of a wide variety of studies from human and from informative model systems with techniques linking genes and pathways to physiology, from prokaryotes to eukaryotes. It is published quarterly in January, April, July, and October by the American Physiological Society, 9650 Rockville Pike, Bethesda MD 20814-3991. Copyright © 2008 by the American Physiological Society. ISSN: 1094-8341, ESSN: 1531-2267. Visit our website at http://www.the-aps.org/.

Physiol Genomics 32: 322–334, 2008.
First published December 4, 2007; doi:10.1152/physiolgenomics.00160.2007.

# Genome-wide gene expression profiling reveals renal genes regulated during metabolic acidosis

Marta Nowik,$^{1}$ M. Rita Lecca,$^{2}$ Ana Velic,$^{1}$ Hubert Rehrauer,$^{3}$ André W. Brändli,$^{2}$ and Carsten A. Wagner$^{1}$

$^{1}$Institute of Physiology and Zurich Center for Human Integrative Physiology (ZIHP), University of Zurich; $^{2}$Institute of Pharmaceutical Sciences, Department of Chemistry and Applied Biosciences, ETH Zurich; and $^{3}$Functional Genomics Center Zurich, Zurich, Switzerland

Submitted 18 July 2007; accepted in final form 21 November 2007

Nowik M, Lecca MR, Velic A, Rehrauer H, Brändli AW, Wagner CA. Genome-wide gene expression profiling reveals renal genes regulated during metabolic acidosis. *Physiol Genomics* 32: 322–334, 2008. First published December 4, 2007; doi:10.1152/physiolgenomics.00160.2007.—Production and excretion of acids are balanced to maintain systemic acid-base homeostasis. During metabolic acidosis (MA) excess acid accumulates and is removed from the body, a process achieved, at least in part, by increasing renal acid excretion. This acid-secretory process requires the concerted regulation of metabolic and transport pathways, which are only partially understood. Chronic MA causes also morphological remodeling of the kidney. Therefore, we characterized transcriptional changes in mammalian kidney during MA to gain insights into adaptive pathways. Total kidney RNA from control and 2- and 7-days NH₄Cl treated mice was subjected to microarray gene profiling. We identified 4,075 transcripts significantly (*P* < 0.05) regulated after 2 and/or 7 days of treatment. Microarray results were confirmed by qRT-PCR. Analysis of candidate genes revealed that a large group of regulated transcripts was represented by different solute carrier transporters, genes involved in cell growth, proliferation, apoptosis, water homeostasis, and ammoniagenesis. Pathway analysis revealed that oxidative phosphorylation was the most affected pathway. Interestingly, the majority of acutely regulated genes after 2 days, returned to normal values after 7 days suggesting that adaptation had occurred. Besides these temporal changes, we detected also differential regulation of selected genes (SNAT3, PEPCK, PDG) between early and late proximal tubule. In conclusion, the mammalian kidney responds to MA by temporally and spatially altering the expression of a large number of genes. Our analysis suggests that many of these genes may participate in various processes leading to adaptation and restoration of normal systemic acid-base and electrolyte homeostasis.

kidney; microarray; acid-base; ammoniagenesis; remodeling

SYSTEMIC pH in extracellular fluids is maintained within the physiological range through the cooperation of various body buffering systems including regulated exhalation of CO₂ by the lungs, release of buffers from bone, and acid or base excretion by the kidney (16), whereby the kidney plays a central role in maintaining systemic acid-base homeostasis. Chemical buffering and ventilation contribute to the immediate and chronic response to changes in systemic acid-base status, whereas the kidney is rather responsible for both intermediate and chronic adaptation. The kidney participates in maintaining pH homeostasis by excreting the net excess of acid in urine (23). This is achieved by three main processes: 1) the reabsorption of filtered bicarbonate, 2) the excretion of acids/protons, and 3) the synthesis of ammonia and use of ammonia, phosphate, and citrate as titratable acids increasing the kidneys' ability to excrete protons. These processes take place in different segments of the nephron and are precisely regulated and coordinated (23).

Disturbances in acid-base balance lead to and require an adaptive increase in renal acid excretion that involves activation and/or regulation of various pathways (16). These pathways include increased ammoniagenesis, changes in electrolyte and water handling, excretion of titratable acids, increased synthesis of bicarbonate, stimulation of proton secretion, and also extensive remodeling of the nephron with cellular hypertrophy of various nephron segments (16).

However, these adaptive changes are only partially understood. Two recent publications employed different experimental approaches to identify changes in renal gene or protein expression in chronically acidic animals. Cheval and colleagues (12) searched for altered transcripts in the mouse outer medullary collecting duct using serial analysis of gene expression (SAGE), whereas Curthoys et al. (15) applied two-dimensional gel electrophoresis and subsequent mass spectroscopy to detect altered protein expression in rat proximal tubules after acidosis. Here, we used microarray-based gene expression profiling as an alternative approach to perform a genome-wide analysis of altered transcripts in response to acute (2 days) and chronic (7 days) metabolic acidosis in mouse kidneys. Our data demonstrate a massive regulation of various transcripts after 2 days of NH₄Cl-loading with most genes returning to normal levels after 7 days. Several pathways including ammoniagenesis, oxidative phosphorylation, and general transport processes were highly overrepresented. Furthermore, gene regulation was spatially and temporally regulated in dissected early and late proximal tubules. Thus, our data suggest a concerted, spatially and temporally regulated, response of the kidney to systemic metabolic acidosis that may contribute to the kidneys' ability to restore systemic acid-base balance.

## METHODS

Animals. All experiments were performed on 12-wk-old C57BL/6J male mice and according to Swiss Animal Welfare laws, with approval of the local veterinary authority (Kantonales Veterinäramt Zürich).

Metabolic cages. To induce metabolic acidosis, C57BL/6J mice (10 animals per group) were given 0.28 M NH₄Cl/2% sucrose in drinking water for 2 or 7 days, whereas the control group received

only 2% sucrose in drinking water. All animals were adapted to metabolic cages for 3 days before data collection. Forty-eight hours before death, mice were housed in metabolic cages and had free access to standard mouse chow and drinking water. Daily chow, water intake, and body weights were measured, and urine was collected under mineral oil. At the end of the experiment, mice were anesthetized, and heparinized venous blood was collected and analyzed immediately for pH, blood gases, and electrolytes on a Radiometer ABL 505 (Radiometer, Copenhagen, Denmark) blood gas analyzer. Serum was collected and frozen until further analysis. Both kidneys were harvested, immediately frozen in liquid nitrogen, and stored at $-80^{\circ} \mathrm{C}$ until mRNA extraction.

Urinary pH was measured using a pH microelectrode (691 pH-meter, Metroholm). Urinary creatinine was measured by the Jaffe method (42). Ammonium in urine was measured by the method of Berthelot (5). Urinary and serum phosphate was measured using a commercial kit (Sigma Diagnostics, Munich, Germany). Urinary electrolytes ($\mathrm{Na}^{+}, \mathrm{K}^{+}, \mathrm{Ca}^{2+}, \mathrm{Mg}^{2+}, \mathrm{Cl}^{-}, \mathrm{SO}_{4}^{2-}$) were measured by ion chromatography (Metrohm ion chromatograph, Herisau, Switzerland).

**Preparation of isolated proximal tubule segments.** Control animals received 2% sucrose in their drinking water for 2 days, whereas the acidotic group received $0.28 \mathrm{M} \mathrm{NH}_{4} \mathrm{Cl}$ plus 2% sucrose in their drinking water for 2 days. Animals were otherwise housed in standard cages and had free access to standard rodent chow. Mice were anesthetized and perfused through the left heart ventricle with $20 \mathrm{ml}$ of a solution containing $140 \mathrm{mM}$ sucrose, $28 \mathrm{mM} \mathrm{Na}_{2} \mathrm{HPO}_{4}$, and $112 \mathrm{mM} \mathrm{NaH}_{2} \mathrm{PO}_{4}$. Kidneys were immediately removed, the capsule removed, and thin coronal slices containing both cortex and medulla were prepared. The inner medulla was removed. The tissue was dissected under a stereo microscope at $4^{\circ} \mathrm{C}$ using two fine forceps (Dumont no. 5). The total number of 50 S1 and S2 segments, followed by 50 S3 segments, was collected and placed in $300 \mu \mathrm{l}$ RLT-Buffer (Qiagen, Basel, Switzerland) containing $3 \mu \mathrm{l}$ 2-mercaptoethanol. S1/S2 and S3 segments of proximal tubules were dissected separately from six controls and six acidotic animals. Collected tissue was immediately frozen at $-80^{\circ} \mathrm{C}$ until mRNA extraction.

**RNA extraction from total kidney.** Snap-frozen kidneys (10 kidneys for each condition) were homogenized in RLT-Buffer (Qiagen) supplemented with 2-mercaptoethanol to a final concentration of 1%. Total RNA was extracted from $200 \mu \mathrm{l}$ aliquots of each homogenized sample using the RNeasy Mini Kit (Qiagen) according to the manufacturer's instructions. Quality and concentration of the isolated RNA preparations were analyzed by electrophoresis and spectroscopically using the 2100 Bioanalyzer (Agilent Technologies) and the ND-1000 spectrophotometer (NanoDrop Technologies), respectively. Total RNA samples were stored at $-80^{\circ} \mathrm{C}$.

**RNA extraction from microdissected nephron segments.** After thawing on ice, each sample was vortexed and total RNA was extracted using the RNeasy Micro Kit (Qiagen) according to the manufacturer's instructions. Quality and concentration of the isolated RNA preparations were analyzed using the ND-1000 spectrophotometer (NanoDrop Technologies). Total RNA samples were stored at $-80^{\circ} \mathrm{C}$.

**Quantitative real-time PCR.** Each RNA sample was diluted to $100 \mathrm{ng} / \mu \mathrm{l}$, and $3 \mu \mathrm{l}$ was used as a template for reverse transcription using the TaqMan Reverse Transcription Kit (Applied Biosystems, Forster City, CA). Quantitative real-time (qRT-PCR) was performed on the ABI PRISM 7700 Sequence Detection System (Applied Biosystems). Primers for all genes of interest were designed using Primer Express from Applied Biosystems (Supplementary Table S1). Probes were labeled with the reporter dye FAM at the 5′ end and the quencher dye TAMRA at the 3′ end (Microsynth, Balgach, Switzerland). The specificity of all primers was first tested in a standard PCR and always resulted in a single product of the expected size on $1 \%$ agarose gels (data not shown). Real-time PCR reactions were performed using the TaqMan Universal PCR Master Mix (Applied Biosystems). Briefly, $3.5 \mu \mathrm{l}$ cDNA, $1 \mu \mathrm{l}$ of each primer (25 $\mu \mathrm{M}$ ), $0.5 \mu \mathrm{l}$ labeled probe (5 $\mu \mathrm{M}$ ), $6.5 \mu \mathrm{l}$ RNase-free water, $12.5 \mu \mathrm{l}$ TaqMan Universal PCR Master Mix reached $25 \mu \mathrm{l}$ of final reaction volume. Reaction conditions were: denaturation at $95^{\circ} \mathrm{C}$ for 10 min followed by 40 cycles of denaturation at $95^{\circ} \mathrm{C}$ for 15 s and annealing/elongation at $60^{\circ} \mathrm{C}$ for 60 s with autoramp time. All reactions were run in duplicate. To analyze the data, we set the threshold to 0.06 as this value had been determined to be in the linear range of the amplification curves for all mRNAs in all experimental runs. The expression of gene of interest was calculated in relation to hypoxanthine guanine phosphoribosyl transferase (HPRT) or β-actin. Relative expression ratios were calculated as $R=2^{\left[\mathrm{Ct}(\mathrm{HPRT} / \beta-\text { actin })-\mathrm{Ct}(\text { test gene })\right]}$, where Ct represents the cycle number at the threshold 0.06.

**Labeling of cRNA, microarray hybridization, and image processing.** Digoxigenin (DIG)-UTP-labeled cRNA was generated and linearly amplified from total RNA using the Nano Amp RT-IVT Labeling Kit (Applied Biosystems) and the manufacturer’s protocol. Briefly, $1 \mu \mathrm{g}$ of total RNA was used to perform the 1st- and 2nd-strand synthesis of cDNA. The purified cDNA was transcribed in vitro in presence of $200 \mathrm{nM}$ DIG-11 UTP to produce DIG-labeled cRNA, followed by purification. The quality and amount of labeled cRNA were assessed with the Agilent 2100 Bioanalyzer and the ND-1000 spectrophotometer, respectively. Labeled cRNA preparations were hybridized to mouse microarrays (Mouse Genome Survey Microarray, 33,148 features; Applied Biosystems). Microarray hybridization, chemiluminescence detection, image acquisition, and analysis were performed using the Applied Biosystems Chemiluminescence Detection Kit and Applied Biosystems 1700 Chemiluminescent Microarray Analyzer following the manufacturer’s protocols. Each microarray was first prehybridized at $55^{\circ} \mathrm{C}$ for 1 h in hybridization buffer with blocking agent. The DIG-labeled cRNA targets ($20 \mu \mathrm{g}$) were first fragmented to 100–400 bases by incubation with fragmentation buffer at $60^{\circ} \mathrm{C}$ for 30 min, mixed with internal control target (ICT, 24-mer oligo labeled with LIZ fluorescent dye), and hybridized to each prehybridized microarray in $1.5-\mathrm{ml}$ volume at $55^{\circ} \mathrm{C}$ for 16 h using an Minitron Incubator Shaker (Infors HT). After hybridization, the microarrays were washed with hybridization wash buffer and chemiluminescence rinse buffer. Enhanced chemiluminescent signals were generated by first incubating the microarrays with antidigoxigenin Fab fragment antibodies coupled to alkaline phosphatase (150 U, Roche Applied Science) for 20 min, then washing with Chemiluminescence Enhancing Solution, and finally adding Chemiluminescence Substrate. Images were collected for each microarray using Applied Biosystems 1700 Chemiluminescent Microarray Analyzer. Images were autogrided, and the chemiluminescent signals were quantified, corrected for background, and spatially normalized using the Expression Array System Analyzer Software Version 1.1.1 (Applied Biosystems).

**Membrane preparation and western blot analysis.** Mice were anesthetized with ketamine-xylazine intraperitoneally, and kidneys were removed and rapidly frozen in liquid nitrogen. For total membranes isolation kidneys were homogenized in an ice-cold K-HEPES buffer ($200 \mathrm{mM}$ mannitol, $80 \mathrm{mM} \mathrm{K}-\mathrm{HEPES}, 41 \mathrm{mM} \mathrm{KOH}, \mathrm{pH} 7.5$) with pepstatin, leupeptin, K-EDTA, and phenylmethylsulfonyl fluoride as protease inhibitors. Samples were centrifuged at $1,000 \mathrm{~g}$ for 10 min at $4^{\circ} \mathrm{C}$, and the supernatant was saved. Subsequently, the supernatant was centrifuged at $41^{\prime} 000 \mathrm{~g}$ for 1 h at $4^{\circ} \mathrm{C}$, and the pellet was resuspended in K-HEPES buffer containing protease inhibitors. Brush border membranes (BBM) were prepared as described previously using the $\mathrm{Mg}^{2+}$-precipitation technique (6). After measurement of the total protein concentration (Bio-Rad, Hercules, CA), $50 \mu \mathrm{g}$ of crude membrane proteins or $10 \mu \mathrm{g}$ of BBM proteins were solubilized in Laemmli sample buffer, and SDS-PAGE was performed on 10%

polyacrylamide gels. For immunoblotting, the proteins were transferred electrophoretically to polyvinylidene fluoride membranes (Immobilon-P; Millipore, Bedford, MA). After blocking with 5% milk powder in Tris-buffered saline/0.1% Tween-20 for 60 min, the blots were incubated with the primary antibodies and mouse monoclonal anti-β-actin antibody (42 kDa; Sigma, St. Louis, MO) 1:5,000 either for 2 h at room temperature or overnight at 4°C. The primary antibodies were: rabbit anti-mouse NHE3 (1:5,000) (generated against a KLH-linked peptide, Pineda Antibody Services, Berlin, Germany), rabbit anti-mouse SNAT3 previously described (32), rabbit anti-aquaporin 2 (1:5,000) (kindly provided by J. Loffing, University of Fribourg, Switzerland), rabbit anti-pendrin (1:5,000). The membranes were then washed three times, blocked for 1 h, and again incubated for 1 h at room temperature with the secondary goat anti-rabbit or donkey anti-goat antibodies (1:5,000) linked to alkaline phosphatase (Promega). The protein signal was detected with the CDP Star chemiluminescence system (Roche Diagnostics, Basel, Switzerland) using the DIANA III-chemiluminescence detection system (Raytest, Straubenhardt, Germany). All images were analyzed using appropriate software (Advanced Image Data Analyser AIDA, Raytest) to calculate the protein of interest/actin ratio.

Microarray data analysis. RNA samples (four biological replicates per condition) were transcribed into DIG-labeled cRNAs and hybridized to Mouse Genome Survey Arrays (32,996 probes representing 32,389 curated genes) from Applied Biosystems. The arrays were scanned and quantified using Expression Array System Analyzer Software Version 1.1.1 (Applied Biosystems). Intensity readouts were normalized across arrays with Bioconductor's vsn package (24). Genes were defined as present and used in the subsequent analysis if they satisfied the intensity threshold of 2,000 in all four replicates of at least one condition. Among the present genes, the differentially expressed ones were identified by applying Student's t-test to the log-intensity data. For the significant genes fold-changes between the treated and untreated conditions were also computed on log-intensity data.

Hierarchical clustering. Clustering was performed using GeneSpring GX Version 7.3 software (Agilent Technologies). Microarray data were normalized by dividing the expression values of each gene by the median of its expression values across all samples. To remove genes that do not show significant expression changes, we first applied a one-way ANOVA analysis to select among the present genes only those that showed a significant variation across the time points (*P* value threshold: 0.05). The selected genes were then clustered using Spearman correlation as distance measure and the average linkage algorithm for merging clusters.

Identification of genes and biological processes involved in kidney adaptation to metabolic acidosis. PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System (1) (http://www.pantherdb.org) was used to classify significantly regulated transcripts. Pathway analysis of significantly regulated genes was performed using MetaCore (GeneGo, St. Joseph, MI) and WebGestalt toolkit (http://bioinfo.vanderbilt.edu/webgestalt). MetaCore is a web-based software suite for functional analysis of experimental data based on a manually curated database of human protein-protein, protein-DNA and protein compound interactions, and metabolic and signaling pathways. Gene Ontology (GO) annotations were used as indicators of biological function. Associations with GO biological process, molecular function, and cellular component groups were obtained through MetaCore. WebGestalt includes information from Kyoto Encyclopedia of Genes and Genomes (KEGG) database and organizes genes based on the KEGG biochemical pathway.

Statistical analysis. Results are expressed as means ± SE. All data were tested for significance using ANOVA and unpaired Student’s test where appropriate. Only values with *P* < 0.05 were considered as significant.


RESULTS

Acid-base status of animals. Animals were given 0.28 M NH₄Cl with 2% sucrose in the drinking water for the 48 h or 7 days to induce metabolic acidosis. The control group received only 2% sucrose in the drinking water for 2 or 7 days. This treatment has been previously shown to induce metabolic acidosis (32, 45). As we did not detect any difference in the 2- or 7-day control groups, all data were subsequently pooled. Blood and urine samples were analyzed to monitor systemic acid-base status and renal acid excretion (Table 1).

Addition of 0.28 M NH₄Cl to the drinking water resulted in metabolic acidosis in both treated groups as was evident from the reduction of blood pH and bicarbonate levels and increase in chloride and sodium concentrations (Table 1). Potassium levels remained unchanged. As expected, urine pH values of both treated groups were more acidic compared with the control group, and urinary ammonium excretion was increased accordingly. Urine output was significantly reduced in both treated groups. Thus, the treatment with NH₄Cl resulted in the expected hyperchloremic metabolic acidosis and its consequences.

Genome-wide comparison of renal gene expression profiles of control and acidotic mice reveals large number of differentially regulated genes. Transcriptional changes in the total kidney during metabolic acidosis were investigated by genome-wide gene expression profiling using Applied Biosystems Mouse Genome Survey Arrays (32,996 probes representing 32,389 curated genes that target 44,498 transcripts). In a first step, changes in the expression of SNAT3 (Slc38a3) and

Table 1. Summary of blood and urine analysis

| Control | 2 Days NH₄Cl | 7 Days NH₄Cl |
| --- | --- | --- |
| **Blood** |  |  |
| pH | 7.24 ± 0.03 | 7.10 ± 0.02 | 7.19 ± 0.03<sup>ns</sup> |
| pCO₂, mmHg | 47.1 ± 2.2 | 40.9 ± 2.7<sup>ns</sup> | 46.1 ± 3.7<sup>ns</sup> |
| [HCO₃⁻], mM | 17.0 ± 0.9 | 12.1 ± 0.9<sup>‡</sup> | 16.5 ± 0.8<sup>*</sup> |
| [Na⁺], mM | 144.2 ± 0.6 | 148.7 ± 0.7<sup>†</sup> | 145.5 ± 1.2<sup>ns</sup> |
| [K⁺], mM | 5.2 ± 0.5 | 4.9 ± 0.7<sup>ns</sup> | 5.16 ± 0.3<sup>ns</sup> |
| [Cl⁻], mM | 113.4 ± 0.5 | 128.5 ± 1.3<sup>‡</sup> | 121.8 ± 1.5<sup>‡</sup> |
| Osmolality, mOsm/kg H₂O | 274 ± 5 | 279 ± 6<sup>ns</sup> | 285 ± 5<sup>ns</sup> |
| **Urine** |  |  |
| pH | 6.50 ± 0.15 | 5.40 ± 0.05<sup>‡</sup> | 5.43 ± 0.04<sup>‡</sup> |
| Creatinine, mg/dl | 51.3 ± 5.1 | 73.0 ± 2.8<sup>‡</sup> | 60.8 ± 3.9<sup>ns</sup> |
| NH₃/NH₄⁺/(mM)/crea, mg/dl | 0.9 ± 0.5 | 6.5 ± 0.5<sup>‡</sup> | 7.1 ± 0.6<sup>‡</sup> |
| Na⁺(mM)/crea, mg/dl | 2.4 ± 0.5 | 2.3 ± 0.2<sup>ns</sup> | 2.6 ± 0.1<sup>ns</sup> |
| K⁺(mM)/crea, mg/dl | 5.4 ± 0.8 | 5.2 ± 0.4<sup>ns</sup> | 6.1 ± 0.3<sup>ns</sup> |
| Ca²⁺(mM)/crea, mg/dl | 0.05 ± 0.02 | 0.09 ± 0.01<sup>ns</sup> | 0.08 ± 0.01<sup>ns</sup> |
| Mg²⁺(mM)/crea, mg/dl | 0.44 ± 0.10 | 0.53 ± 0.03<sup>ns</sup> | 0.63 ± 0.02<sup>ns</sup> |
| Cl⁻(mM)/crea, mg/dl | 4.1 ± 0.7 | 8.8 ± 0.4<sup>‡</sup> | 11.0 ± 0.7<sup>‡</sup> |
| Pi(mM)/crea, mg/dl | 2.4 ± 0.4 | 2.5 ± 0.4<sup>ns</sup> | 2.3 ± 0.3<sup>ns</sup> |
| SO₄²⁻(mM)/crea, mg/dl | 0.7 ± 0.1 | 0.7 ± 0.1<sup>ns</sup> | 0.7 ± 0.1<sup>ns</sup> |
| Osmolality, mOsm/kg H₂O | 1577 ± 142 | 3770 ± 285<sup>‡</sup> | 3332 ± 232<sup>‡</sup> |
| Urine volume, ml | 2.4 ± 0.4 | 0.8 ± 0.1<sup>‡</sup> | 1.2 ± 0.2<sup>†</sup> |
| Urine volume/BW | 0.09 ± 0.02 | 0.03 ± 0.01<sup>†</sup> | 0.05 ± 0.01<sup>ns</sup> |
| Weight change, % | +1.9 ± 3.0 | −5.3 ± 0.04<sup>*</sup> | −2.5 ± 0.6<sup>ns</sup> |
| Water intake, ml/g BW | 0.23 ± 0.01 | 0.15 ± 0.01<sup>‡</sup> | 0.18 ± 0.01<sup>†</sup> |
| Food intake, mg/g BW | 0.13 ± 0.01 | 0.12 ± 0.00<sup>ns</sup> | 0.15 ± 0.01<sup>ns</sup> |

Summary of blood and urine data from mice receiving 0.28 M NH₄Cl plus 2% sucrose in drinking water for 2 or 7 days or only 2% sucrose (control group); *n* = 10 per group. Urinary excretion was normalized against creatinine (crea). BW, body weight; ns, not significant. *P* < 0.05, †*P* < 0.01, ‡*P* < 0.001.

PEPCK, two genes known to be upregulated in mouse kidney during metabolic acidosis (32), were tested using quantitative real-time PCR. As expected, SNAT 3 and PEPCK mRNA levels were elevated three- to fourfold after 2 and 7 days (data not shown). Subsequently, four animals from each group with the highest degree of similarity in terms of SNAT 3 and PEPCK expression were selected for microarray analysis. In the 12 samples hybridized on the microarrays (four biological replicates per condition) we detected a total of 12,984 genes (40% of the total number present on the array) expressed in kidneys of all groups of mice. We found that 4,075 transcripts were significantly (*P* < 0.05) regulated in response to the acid load after 2 and/or 7 days (Supplementary Tables S2 and S3). The majority of transcripts (3,070) responded only to 2 days NH₄Cl treatment, and only 322 responded exclusively to the chronic acid load (7 days). Out of the 3,070 genes acutely regulated, 1,475 were upregulated (69 with fold change ≥2) and 1,595 were downregulated (108 with fold change ≤2). Out of the 322 genes regulated only after 7 days treatment, 142 were upregulated (7 with fold change ≥2) and 180 were downregulated (3 with fold change ≤2). We found that 675 transcripts were similarly regulated in response to acute and chronic acid load. Moreover, 333 genes were consistently upregulated and 342 were downregulated. Seven transcripts showed decreased expression after 2 days acidosis and increased expression after 7 days. Only one gene coding for fucosyltransferase 11 (Fut11) was upregulated after 2 days and downregulated after 7 days. Selected genes affected by 2 and/or 7 days of NH₄Cl-loading are listed in Tables 2–5.

Pathway analysis. Pathway analysis using two different web-based softwares (MetaCore and WebGestalt) revealed that the most affected pathway in acidotic mice was oxidative phosphorylation (41 upregulated genes assigned to KEGG “mmu00190” pathway; 35 upregulated genes assigned to MetaCore's metabolic map “oxidative phosphorylation pathway”). As expected, in acid-loaded animals we found upregulation of a number of genes coding for enzymes and transporters important for ammonia synthesis (SNAT 3, PDG, PEPCK, Glud1). Interestingly, increased ammoniagenesis was concomitant with activation of the TCA (tricarboxylic acid) cycle pathway (10 upregulated and 2 downregulated genes assigned to KEGG “mmu00020” pathway) and glycolysis/gluconeogenesis pathway (8 upregulated and 5 downregulated genes assigned to KEGG “mmu00010” pathway).

The largest group of transcripts affected by metabolic acidosis encoded solute carrier (Slc) transporters (62 genes downregulated, 35 genes upregulated). Monocarboxylic acid transporters (Slc16 family) and mitochondrial carriers (Slc25 family) were the most represented with 13 and 16 genes significantly regulated after 2 and/or 7 days of acid-loading, respectively.

Co-regulated gene clusters. Hierarchical clustering (17) was performed on 4,389 acidosis-responsive genes obtained after one-way ANOVA test. This allowed the identification of at least six distinct clusters of genes sharing similar expression profiles across the three experimental conditions (Fig. 1). Most genes were either found in cluster 1 or 3. Genes of cluster 1 were upregulated more than twofold after 2 days and less than twofold after 7 days of metabolic acidosis (Fig. 1). This cluster included 1,278 genes, which encode gene products mainly involved in glutamine and glutamate metabolism: glutamate dehydrogenase 1 (Glud1), glutamine transporter SNAT 3 (Slc38a3), asparagine synthetase (Asns), glutamate-cysteine ligase (Gclm), guanine nucleotide binding protein, alpha 11 (Gna11), citrate synthase (Cs), glycolytic pathway, and ATP synthesis (hexokinase, phosphor-fructo-kinase, ATP synthase, NADH dehydrogenase, cytochrome c) (Supplementary Table S5). These findings are in good accordance with the observed increased utilization of glutamine, lactate, and citrate to generate ammonia and ATP during kidney metabolic acidosis. The increased production of ATP

Table 2. Genes upregulated after 2 days acid-loading

| Gene Symbol | Gene Name | Fold Change | GenBank Acc. No. |
|-------------|-----------|-------------|------------------|
| Ugt8a       | UDP galactosyltransferase 8A | 11.38 | NM_011674 |
| 1700047E16Rik | RIKEN cDNA 1700047E16 gene | 10.84 | NM_173431 |
| Fgfrl1      | fibroblast growth factor receptor-like 1 | 7.65 | XM_983846; NM_054071 |
| Grem2       | gremlin 2 homolog, cysteine knot superfamily (Xenopus laevis) | 5.26 | NM_011825 |
| Slc25a25    | solute carrier family 25 (mitochondrial carrier, phosphate carrier), member 25 | 4.32 | NM_146118 |
| Slc38a3     | solute carrier family 38, member 3; SNAT 3 | 3.75 | NM_023805 |
| Slc4a7      | solute carrier family 4, sodium bicarbonate cotransporter, member 7 | 3.33 | NM_001033270 |
| Slc13a2     | solute carrier family 13 (sodium-dependent dicarboxylate transporter), member 2 | 3.29 | NM_022411 |
| Pck2        | phosphoenolpyruvate carboxykinase 2 (mitochondrial) | 3.28 | NM_028994 |
| AQP2        | aquaporin 2 | 3.20 | NM_009699 |
| Slc14a2     | solute carrier family 14 (urea transporter), member 2 | 2.94 | NM_030683; NM_207651 |
| Slc16a6     | solute carrier family 16 (monocarboxylic acid transporters), member 6 | 2.62 | NM_001029842; NM_134038 |
| Mdh2        | malate dehydrogenase 2, NAD (mitochondrial) | 2.38 | NM_008617 |
| AQP4        | aquaporin 4 | 2.12 | NM_009700 |
| Car2        | carbonic anhydrase 2 | 1.69 | NM_009801 |
| PDG         | glutaminase | 1.63 | NM_001081081 |
| Slc16a7     | solute carrier family 16 (monocarboxylic acid transporters), member 7 | 1.56 | NM_011391 |
| Avpr2       | arginine vasopressin receptor 2 | 1.50 | NM_019404 |
| Slc13a3     | solute carrier family 13 (sodium-dependent dicarboxylate transporter), member 3 | 1.49 | NM_054055 |
| AQP3        | aquaporin 3 | 1.41 | NM_016689 |
| Cs          | citrate synthase | 1.36 | NM_026444 |
| Aco2        | aconitase 2 | 1.30 | NM_080633 |
| Mdh1        | malate dehydrogenase 1 (soluble) | 1.19 | NM_008618 |

List of selected genes upregulated after 2 days of NH₄Cl-loading. For each gene, the symbol, name, fold change, and GenBank Reference Sequence are given.

326 GENOME-WIDE DETECTION OF RENAL GENES REGULATED DURING ACIDOSIS

Table 3. Genes upregulated after 7 days acid-loading

| Gene Symbol | Gene Name | Fold Change | GenBank Acc. No. |
|-------------|-----------|-------------|------------------|
| 2200001I15Rik | RIKEN cDNA 2200001I15 gene | 3.89 | NM_183278 |
| Slc38a3 | solute carrier family 38, member 3; SNAT 3 | 3.86 | NM_023805 |
| Cul7 | cullin 7 | 3.78 | NM_025611 |
| Slc25a25 | solute carrier family 25 (mitochondrial carrier, phosphate carrier), member 25 | 3.44 | NM_146118 |
| Pck2 | phosphoenolpyruvate carboxykinase 2 (mitochondrial) | 3.04 | NM_028994 |
| Pck1 | phosphoenolpyruvate carboxykinase 1, cytosolic | 2.71 | NM_011044 |
| Slc13a2 | solute carrier family 13 (sodium-dependent dicarboxylate transporter), member 2 | 2.41 | NM_022411 |
| Slc16a6 | solute carrier family 16 (monocarboxylic acid transporters), member 6 | 2.11 | NM_001029842; NM_134038 |
| AQP2 | aquaporin 2 | 1.72 | NM_009699 |
| Car2 | carbonic anhydrase 2 | 1.53 | NM_009801 |
| Slc13a3 | solute carrier family 13 (sodium-dependent dicarboxylate transporter), member 3 | 1.52 | NM_054055 |
| PDG | glutaminase | 1.48 | NM_001081081 |
| Aco2 | aconitase 2 | 1.38 | NM_080633 |
| Slc25a10 | solute carrier family 25 (mitochondrial carrier, dicarboxylate transporter), member 10 | 1.37 | NM_013770 |
| Cs | citrate synthase | 1.16 | NM_026444 |
| Mdh1 | malate dehydrogenase 1 (soluble) | 1.20 | NM_008618 |

List of selected genes upregulated after 7 days of NH4Cl-loading. For each gene, the symbol, name, fold change, and GenBank Reference Sequence are given.

may be used to reabsorb sodium in the proximal convoluted tubule either during acute or chronic acidosis (22). Cluster 3 contains 2,044 genes, whose expression levels were induced (>2-fold) after 2 days but returned to control levels during chronic acid load (Fig. 1). These genes, were mainly represented by genes encoding cytoskeletal proteins (α-tubulin, destrin), solute carrier transporters (mainly sodium and glutamate transporters), transcription factors (Foxp4, Foxp1, Wt1, Krüppel-like factors 5 and 7), chloride transporting proteins: chloride channel 3 (Clcn3), potassium/chloride transporter KCC3 (Slc12a4), and TGF-β and Wnt signaling molecules: MAD homolog 7 (Smad7), frizzled homolog 2 (Fzd2) (Supplementary Table S7).

Interestingly, only a small group of transcripts (214 genes) showed a late response with increased gene expression levels after 7 days of NH4Cl loading (cluster 2, Fig. 1). Cluster 2 includes, along with signaling molecules of the small GTPase family and G protein signaling pathway, genes encoding cytoskeleton and Wnt signaling modulators such as the mammalian counterpart of *Drosophila* pygopus 1 (Supplementary Table S6). Of note, the recently identified (Na⁺,K⁺)/H⁺ exchanger NHE7 (SLC9A7) was highly upregulated during chronic acidosis; the latter localizes predominantly to the trans-Golgi network (34), and it has been suggested to play a pivotal role in the control of organellar ion homeostasis, of which the molecular mechanisms are still largely unknown (29).

Moreover, we found three small clusters of genes (clusters 4–6) with the following characteristics: cluster 4 contains 226 genes, which include the aspartate/glutamate transporter Slc1a6, the sodium/glucose cotransporter (Slc5a11), the sodium bicarbonate cotransporter Slc4a9, the gulonolactone oxidase (Gulo), and a member of the organic anion transporters (Slc22a19) (Supplementary Table S8). These genes showed a gradual decrease in expression levels during metabolic acidosis. Cluster 5 consists of 442 genes that are transiently repressed after 2 days of acidosis but returned to control levels

Table 4. Genes downregulated after 2 days acid-loading

| Gene Symbol | Gene Name | Fold Change | GenBank Acc. No. |
|-------------|-----------|-------------|------------------|
| Dst | dystonin | 0.11 | NM_133833; NM_134448 |
| Angptl4 | angiopoietin-like 4 | 0.23 | NM_020581 |
| Slc7a7 | solute carrier family 7 (cationic amino acid transporter, y+ system), member 7 | 0.29 | NM_011405 |
| Slc34a3 | solute carrier family 34 (sodium phosphate), member 3; NaPiIIc | 0.32 | NM_080854 |
| GPI | glucose phosphate isomerase | 0.38 | NM_008155 |
| Dpp7 | dipeptidylpeptidase 7 | 0.39 | NM_031843 |
| Slc5a12 | solute carrier family 5 (sodium/glucose cotransporter), member 12 | 0.39 | NM_001003915 |
| ASS1 | argininosuccinate synthetase 1 | 0.42 | NM_007494 |
| Slc2a2 | solute carrier family 2 (facilitated glucose transporter), member 2 | 0.43 | NM_031197 |
| Fbp2 | fructose bisphosphatase 2 | 0.46 | NM_007994 |
| Gatm | L-arginine:glycine amidinotransferase | 0.46 | NM_025961 |
| Slc5a11 | solute carrier family 5 (sodium/glucose cotransporter), member 11 | 0.48 | NM_146198 |
| G6pc | glucose-6-phosphatase, catalytic | 0.54 | NM_008061 |
| Asl | argininosuccinate lyase | 0.54 | NM_133768 |
| Slc5a2 | solute carrier family 5 (sodium/glucose cotransporter), member 2 | 0.56 | NM_133254 |
| Slc25a15 | ORNT1, solute carrier family 25 (mitochondrial carrier ornithine transporter), member 15 | 0.59 | NM_011017, NM_181325 |
| Dao1 | D-amino acid oxidase 1 | 0.61 | NM_010018 |
| Slc16a10 | solute carrier family 16 (monocarboxylic acid transporters), member 10, TAT1 | 0.61 | NM_028243 |
| Slc16a2 | solute carrier family 16 (monocarboxylic acid transporters), member 2 | 0.68 | NM_009197 |
| Slc26a4 | solute carrier family 26, member 4; pendrin | 0.70 | NM_011867 |
| Agmat | agmatine ureohydrolase (agmatinase) | 0.77 | XM_915401, XM_980293, XM_983114 |

List of selected genes downregulated after 2 days of NH4Cl-loading. For each gene, the symbol, name, fold change, and GenBank Reference Sequence are given.

GENOME-WIDE DETECTION OF RENAL GENES REGULATED DURING ACIDOSIS 327

Table 5. Gene downregulated after 7 days acid-loading

| Gene Symbol | Gene Names                                                                 | Fold Change | GenBank Acc. No. |
|-------------|---------------------------------------------------------------------------|-------------|------------------|
| Ces1        | carboxylesterase 1                                                        | 0.07        | NM_021456        |
| Slc7a12     | solute carrier family 7 (cationic amino acid transporter, y+ system), member 12 | 0.25        | NM_080852        |
| Angptl4     | angiopoietin-like 4                                                       | 0.40        | NM_020581        |
| Fbp2        | fructose bisphosphatase 2                                                | 0.43        | NM_007994        |
| Slc34a3     | solute carrier family 34 (sodium phosphate), member 3; NaPiIIc              | 0.44        | NM_080854        |
| Slc7a7      | solute carrier family 7 (cationic amino acid transporter, y+ system), member 7 | 0.45        | NM_011405        |
| Dpp7        | dipeptidylpeptidase 7                                                     | 0.47        | NM_031843        |
| Slc5a11     | solute carrier family 5 (sodium/glucose cotransporter), member 11           | 0.56        | NM_146198        |
| Slc26a4     | solute carrier family 26, member 4; pendrin                               | 0.56        | NM_011867        |
| Slc2a2      | solute carrier family 2 (facilitated glucose transporter), member 2         | 0.65        | NM_031197        |
| ASS1        | argininosuccinate synthetase 1                                            | 0.66        | NM_007494        |
| Gatm        | L-arginine:glycine amidinotransferase                                     | 0.71        | NM_025961        |

List of selected genes downregulated after 7 days of NH₄Cl-loading. For each gene, the symbol, name, fold change, and GenBank Reference Sequence are given.

after 7 days (Supplementary Table S9). This cluster was characterized by genes encoding carbonic anhydrase 4 (Car4) as well as proteins involved in regulation of cell growth such as Fas apoptotic molecule 3 (Faim3), growth arrest specific 2 (Gas2), growth differentiation factor 3 (Gdf3), cytoskeleton-associated protein 2 (Ckap2), nibrin (Nbn), insulin-like growth factor binding protein 4 (Igfbp4), and angiopoietin-like gene (Angptl4). Many members of the SLC family were also represented in this cluster: sodium/glucose cotransporter Slc5a12, the facilitated glucose transporter Slc2a2, the cation transporters (Slc7a2, Slc7a9, Slc22a5), monocarboxylic transporters (Slc16a10, Slc16a2), the acetyl-CoA transporter Slc33a1, and many others (Slc1a6, Slc5a1, Slc2a2, Slc34a3, Slc7a7). Finally, cluster 6 included 185 genes with increased expression levels in 7 days-treated mice but decreased expression in 2 days-treated mice (Supplementary Table S10). These gene transcripts include the amiloride-sensitive cation channel Accn1, the potassium channel Kcnk4, the glucose 6-phosphatase (G6pc), and an uncharacterized carbohydrate kinase. In addition, many genes encoding gene products of unknown function are also present in this cluster. Further characterization of these orphan genes may provide novel insights into the mechanisms of adaptation to metabolic acidosis.

Taken together, our findings highlight and confirm the pivotal role of ammoniagenesis, ATP synthesis, and sodium reabsorption as major players during acute and chronic metabolic acidosis. Furthermore, they address the potential adaptive mechanisms during chronic metabolic acidosis, which may be represented by complex signaling networks involving the fine-tuning of cytoskeleton organization, cell proliferation, cell differentiation, and apoptotic responses. Further investigations are needed to characterize not only the metabolic pathways directing these cellular modifications during chronic acidosis, but also the large number of differentially regulated genes, of which the structure and the function are still unknown.

Validation of candidate genes by quantitative real-time PCR. Candidate genes emerging from the microarray analysis were validated by qRT-PCR in a separate group of mice treated identically as the mice used above. Fifteen differently regulated genes were selected for further validation, including SNAT3 (Slc38a3/SN1), phosphoenolpyruvate carboxykinase (PEPCK), aquaporin-2 (AQP2), aquaporin-3 (AQP3), vasopressin 2 receptor (Avrp2), carbonic anhydrase 2 (Car2), phosphate-dependent glutaminase (PDG), the sodium-dependent phosphate cotransporter NaPiIIa (Slc34a1), the sodium-dependent phosphate cotransporter NaPiIIc (Slc34a3), the monocarboxylate transporters MCT8 (Slc16a2) and MCT2 (Slc16a7), the sodium-dependent bicarbonate transporter NBCn1 (Slc4a7), the Cl⁻/HCO₃⁻ exchanger pendrin (Slc26a4), malate dehydrogenase 2 (Mdh2), and fructose bisphosphatase 2 (Fbp2). These genes represented transcripts, which showed positive or negative regulation over a wide range. All genes, except Mdh2, showed the predicted changes in expression level (Fig. 2), similar in fold change to microarray data (slope of line 0.89, R² = 0.73 for 2 days acidosis vs. control, slope of 0.97, R² = 0.94 for 7 days acidosis vs. control, respectively). Thus, the regulation of all but one selected genes could be confirmed by qRT-PCR using a separate group of animals. This demonstrates a remarkably high degree of reproducibility and sensitivity of the gene expression profiles obtained by microarray analysis.

Validation of candidate genes by Western blotting. It has been previously demonstrated that changes in mRNA expression levels do not necessarily translate into changes in the corresponding protein levels (21). Therefore, we investigated whether the mRNA levels as determined by microarray analysis and validated by qRT-PCR correlate with changes in protein levels assessed by Western blotting.

Crude membrane fractions prepared from total kidney of control and acidic animals were used to test the expression levels of the glycosylated and nonglycosylated forms of the aquaporin 2 water channel, the Cl⁻/HCO₃⁻ exchanger pendrin, and the glutamine transporter SNAT3 (Fig. 3). The protein levels of SNAT3 (Slc38a3) were significantly increased after 2 and 7 days of NH₄Cl treatment (268.7 ± 17.9%, 257.6 ± 32.9%, respectively), that was in good agreement with the strong increase found at the mRNA level (3.8, 3.9-fold by microarray; 5.4, 3.9-fold by qRT-PCR). Similarly, the expression levels of the glycosylated (224.1 ± 20.7%, 323.2 ± 34.7%) and nonglycosylated (218.4 ± 15%, 275.9 ± 29.6%) AQP2 were significantly increased in acidic animals (3.2, 1.7-fold by microarray; 2.2, 1.8-fold by qRT-PCR). Downregulation of mRNA levels of Cl⁻/HCO₃⁻ exchanger pendrin observed in microarray experiments in the kidneys of 2 and 7 days treated animals (0.73- and 0.58-fold, respectively) and further confirmed by qRT-PCR (0.51- and 0.42-fold, respectively) was in agreement with decreased abundance of the protein (84.0 ± 6.0% and 67.8 ± 5.8%, respectively). Thus,

GENOME-WIDE DETECTION OF RENAL GENES REGULATED DURING ACIDOSIS

analyzed dissected early (S1/S2 convoluted) and late (S3 straight) proximal tubule segments from control mice and animals acid-loaded for 2 days ($n = 3-6$ animals for each group). Total RNA was extracted and used to assess abundance of several transcripts by qRT-PCR. Analysis was performed for PEPCK, SNAT3 (Slc38a3), PDG, Mdh2, and Fbp2, genes involved in ammoniagenesis, the Krebs cycle, and gluconeogenesis, and for ORNT1 (Slc25a15), $y^{+}$-LAT-1, and Gatm, which are genes important in arginine metabolism (Figs. 6 and 7). Under control conditions, we observed similar expression of PEPCK in early and late parts of the proximal tubule that was highly elevated in acidic mice (10-fold in early, 3.6-fold in late proximal tubule). The glutamine transporter SNAT3 (Slc38a3) was in very low abundance in the early proximal tubule compared with the late segment in control mice; however, in acidic animals mRNA expression was increased to a similar level in both proximal tubule segments. Under control condi-

Fig. 1. Hierarchical clustering reveals 6 distinct clusters of coexpressed genes. The cluster image shows the different classes of gene expression profiles identified. We selected 4,389 genes, whose expression levels changed in response to 2 and 7 days of $\mathrm{NH}_{4} \mathrm{Cl}$ administration. The expression pattern of each gene is displayed here as a horizontal stripe. For each gene, the ratio of the transcript expression level to the median of its expression level across the three time points is represented by a color, according to the color scale on the right. Experiments relative to each time point are shown in columns. Numbers indicate the identified clusters.

regulation of transcripts by acidosis could be confirmed for at least some selected genes suggesting that the transcriptional changes observed may translate into altered protein expression and may eventually alter functional properties of cells and nephron segments.

Spatial regulation of genes: qRT-PCR of dissected nephron segments. Microarray analysis and consecutive real-time PCR had been performed on the total kidney, which may mask even stronger changes in transcript levels, if occurring only in a subpopulation of cells along the nephron. Pathway analysis of regulated transcripts suggested a strong representation of genes involved in ammoniagenesis, the Krebs cycle, and gluconeogenesis, processes known to be highly active in the proximal tubule (Fig. 4). Additionally we found a large number of genes involved in arginine metabolism that were downregulated in acidic mice (Fig. 5). Renal arginine synthesis and metabolism take place in the proximal tubule (7). Therefore, we

Physiol Genomics • VOL 32 • www.physiolgenomics.org

GENOME-WIDE DETECTION OF RENAL GENES REGULATED DURING ACIDOSIS

**Fig. 3. Metabolic acidosis affects protein expression levels. Expression of selected proteins in total kidney membrane fractions after 2 and 7 days acid loading. A:** abundance of glycosylated and nonglycosylated variants of aquaporin 2 water channel (AQP2), the chloride/anion exchanger pendrin (SLC26A4), and the glutamine transporter SNAT3 (SLC38A3). The immunoblots were stripped and reprobed for all proteins and β-actin to control for equal loading. **B:** bar graphs summarizing data from immunoblotting. All data were normalized against actin. *P < 0.05, **P < 0.01, ***P < 0.001.*

tions expression of PDG was similar in both, early and late proximal tubules, and was further increased 3.1-fold in the S1/S2 segment and 1.9-fold in the S3 segment from acid-loaded animals. In contrast, in control animals Fbp2 was 3.5-fold higher expressed in the early proximal tubule; however, in acidotic animals its expression was reduced to a similar extent. We did not observe any changes in Mdh2 expression, which was in agreement with qRT-PCR data from the total kidney but in contrast to the data from microarray analysis.

We also tested genes involved in proximal tubule arginine metabolism and transport and found that under control conditions expression of all analyzed transcripts (ORNT1, y+

Blood

Lumen

Fig. 4. The kidney ammoniagenesis pathway. Glutamine (Gln) is transported across the basolateral plasma membrane via the Na+/glutamine transporter SNAT3 (Slc38a3) and then taken up into mitochondria via a yet unknown carrier. In mitochondria, Gln is converted to glutamate (Glu) by the phosphate-dependent glutaminase (PDG) and then to α-ketoglutarate (α-KG) by action of the glutamate dehydrogenase (Glud1) to fuel into the TCA (Krebs) cycle. These 2 reactions result in production of 2 NH₄⁺ molecules that are secreted into the luminal fluid through a mechanism involving transport via the apical Na⁺/H⁺ exchanger NHE3. Eventually, glucose is produced by the cytosolic phosphoenolpyruvate carboxykinase (PEPCK). Green rectangles denote transporters and enzymes upregulated, red downregulated, yellow not regulated on microarrays. The white rectangle denotes a putative glutamine transporter not identified yet on a molecular basis. Other abbreviations used: ACO2, acetyl-CoA hydrolase; Cit, citrate; CS, citrate synthase; Fbp2, fructose bisphosphatase 2; Fum, fumarate; Glul, glutamine synthase; Glut2 (Slc2a2), glucose transporter; GPI, glucose phosphate isomerase; G6pc, glucose-6-phosphatase; Isocit, isocitrate; Mal, malate; MDH1/2, malate dehydrogenase 1/2; NaDC1 (Slc13a2), Na⁺/dicarboxylate cotransporter, NaDC3 (Slc13a3), Na⁺/dicarboxylate cotransporter; OAA, oxaloacetate; PEP, phosphoenolpyruvate; Succ, succinate.

330 GENOME-WIDE DETECTION OF RENAL GENES REGULATED DURING ACIDOSIS

Blood                                                                 Lumen

Fig. 5. The arginine metabolism pathway in proximal tubules. L-arginine is synthesized from L-citrulline via action of argininosuccinate synthetase (ASS1) and argininosuccinate lyase (ASL). Arginine may be also converted by the enzyme glycine amidinotransferase (Gatm) to guanidinoacetate or by argininodecarboxylase (ADC) to agmatine. Agmatine is then converted to putrescine by an agmatinase (Agmat). Arginine may be also metabolized to 2-oxo-5-guani dinopentanoate by D-amino acid oxidase 1 (Dao1). In addition, arginine is also a precursor of nitric oxide (NO) while converted back to citrulline via action of nitric oxide synthase (eNOS). The bulk of reabsorbed and newly synthesized arginine is, however, transported out of the proximal tubule cell via the basolateral system y⁺L amino acid transporter y⁺-LAT1 (Slc7a7) and is metabolized in other organs. Other abbreviations used: B⁰ Na⁺-dependent neutral amino acid transporter; ORNT1 (Slc25a15), mitochondrial ornithine (citrulline transporter); OTC, ornithine carbamoyltransferase; TAT1 (Slc16a10), monocarboxylic acid transporter. Green rectangles denote transporters and enzymes upregulated, red downregulated, yellow not regulated on microarrays. White rectangle denotes not identified transporters.

![Diagram](#)

LAT1, and Gatm) was higher in the proximal convoluted tubule than proximal straight tubule. Furthermore, acid load decreased expression of these genes only in the early segment. Thus, analysis of two consecutive segments of the proximal tubule from control and acidotic animals demonstrated a spatial arrangement of transcript expression levels and that acidosis differently affected transcript levels of the same genes in adjacent segments. In the case of SNAT3, similar results had been obtained recently by immunohistochemistry in kidneys from control and acidotic animals (32).

DISCUSSION

The renal adaptation to metabolic acidosis involves the regulation of various enzymes and transport proteins. Despite the identification of proteins and mechanisms regulated during metabolic acidosis, the processes involved in the adaptation of the kidney remain only partially understood. Here we used a microarray-based genome-wide gene expression profiling approach to characterize transcriptional changes in total kidney during metabolic acidosis.

![Bar Graphs](#)

Fig. 6. Spatial expression and regulation of genes in the proximal tubule. Expression of mRNA for PEPCK, SNAT3, phosphate-dependent glutaminase (PDG), malate dehydrogenase 2 (Mdh2), and fructose bisphosphatase 2 (Fbp2) in microdissected early (S1/S2) or late (S3) segments of the proximal tubules from control and 2 days acidotic mice. HPRT, hypoxanthine guanine phosphoribosyl transferase. Data are given as means ± SE; n = 3–6. *P < 0.05, **P < 0.01.

Physiol Genomics • VOL 32 • www.physiolgenomics.org

GENOME-WIDE DETECTION OF RENAL GENES REGULATED DURING ACIDOSIS 331

Fig. 7. Spatial expression and regulation of genes involved in arginine synthesis and metabolism in the proximal tubule. Expression of mRNA coding for glycine amidinotransferase (Gatm), the arginine efflux system y⁺L amino acid transporter y⁺-LAT1 (Slc7a7), and the ornithine transporter (Slc25a15) in microdissected early (S1/S2) or late (S3) segments of the proximal tubules from control and 2 days acidotic mice. Data are given as means ± SE; n = 3–6 animals. *P < 0.05, **P < 0.01.

Analysis of microarrays revealed that nearly one-third of all genes present on the chip could be detected in kidney tissue at significant levels. This number is considerably higher than previous reports on total kidney using microarrays that covered the mouse genome only partly (40, 47) or using microdissected nephron segments analyzed by SAGE (11, 50). However, these reports did not include all kidney structures into their analyses, which may explain this discrepancy (44).

The relative expression of >4,000 genes was affected by acid loading for 2 or 7 days, representing almost one third of all expressed genes. Interestingly, cluster analysis detected a clear time course of changes with the majority of transcripts regulated only after 2 days treatment and only a smaller number responded to 7 days acid loading. This temporal change in gene expression was paralleled by the return of systemic acid-base parameters toward control values. It remains to be examined whether this apparent normalization of acid-base status and transcript levels is mediated by posttranscriptional changes, i.e., protein expression levels or posttranslational protein modifications. Quantitative real-time RT-PCR of 15 differentially regulated candidate transcripts validated the microarray findings. Correlation between microarray and qRT-PCR data was remarkably high, confirming the accuracy and reliability of the microarray technology used here. Moreover, protein abundance of AQP2, pendrin, and SNAT3 was in agreement with microarray and qRT-PCR findings. Many of the genes known to be regulated in response to acid loading (e.g., Slc38a3, PEPCK, Slc13a2) could be detected in our lists of regulated genes providing an additional and independent confirmation (Table 6). We further compared our data with a list of proteins detected by proteome analysis of rat proximal tubule during metabolic acidosis (15). Curthoys et al. (15) identified 21 proteins to be upregulated during acidosis of which 10 were also significantly upregulated on mRNA level in our set of data [i.e., transketolase, dimethyl glycine dehydrogenase, phosphoenolpyruvate carboxykinase, glutamate dehydrogenase, and glutathione S-transferases (Pi-class and Mu-1, selenium binding protein, glutaminase)]. Nine mRNAs were not regulated, and two RNAs (phenylalanine-4-hydroxylase, acetyl-CoA thiolase) were downregulated as determined by microarray analysis. Similarly, out of the 16 proteins decreased in acidotic rat proximal tubule, we confirmed three candidates on mRNA level (argininosuccinate synthetase, transglutaminase, Arg-Gly amidinotransferase), one was found to be upregulated (enolase), the other mRNAs were not identified or found to be not regulated. This high degree of similarity between proteome and microarray data may encourage further efforts to decipher the renal response to acidosis on mRNA and protein levels.

Table 6. Previously reported acidosis-regulated genes

| Gene Symbol | Gene Name | Fold Change | Reference No. | GenBank Acc. No. |
|-------------|-----------|-------------|----------------|------------------|
| Slc38a3     | SNAT3     | 3.75        | (27, 32)       | NM_023805         |
| Slc4a7      | NBCn1     | 3.33        | (28)           | NM_001033270      |
| AQP2        | aquaporin 2 | 3.20        | (3)            | NM_009699         |
| Slc13a2     | NaCl, NaDC-1, SDCT1 | 3.29        | (4)            | NM_022411         |
| Glud1       | glutamate dehydrogenase | 2.04        | (39, 52)         | NM_008133         |
| Car2        | carbonic anhydrase 2 | 1.69        | (12, 41, 48)    | NM_009801         |
| PDG         | phosphate-dependent glutaminase | 1.63        | (14, 25, 26)    | NM_001081081      |
| Fxyd4       | CHIF (channel inducing factor) | 1.49        | (9, 10)          | NM_033648         |
| Slc7a7      | y(+) -LAT1 | 0.279       | (32)           | NM_011405         |
| Slc26a4     | pendrin   | 0.70        | (19, 36)       | NM_011867         |
| Slc7a9      | b⁰⁺AT     | 0.47        | (32)           | NM_021291         |
| Car4        | carbonic anhydrase 4 | 0.466       | (51)           | NM_007607         |
| Glul        | glutamine synthase | 0.79        | (13)           | NM_008131         |
| Slc13a1     | NaSi-1    | 0.58        | (37)           | NM_019481         |
| Pck1        | phosphoenolpyruvate carboxykinase 1, cytosolic | nd | (25, 32)       | NM_011044         |

List of genes reported previously by others to be regulated by metabolic acidosis and also detected here by microarray analysis. For each gene, the symbol, name, fold change after 2 and/or 7 days acidosis as detected by microarray analysis, references, and GenBank Reference Sequence are given.

Pathway analysis revealed that regulated genes represent a large variety of GO categories and KEGG pathways, suggesting that the kidney is strongly affected as a whole and responds at many different levels. Analysis of statistically significant transcripts revealed that expression of 97 genes coding different solute carrier transporters was affected by acid loading. Mitochondrial carriers (Slc25) and monocarboxylic acid transporters (Slc16) were the most represented. Membrane transporters are important in maintaining cellular homeostasis and mediate the reabsorptive and excretory processes along the nephron. Thus, regulation of such a large number of solute carriers is not unexpected. However, the most represented pathway in both analyses was oxidative phosphorylation. The apparent overrepresentation of oxidative phosphorylation together with regulation of mitochondrial transporters may indicate the requirement of kidney cells for energy substrates such as ATP to adapt to acid-loading. Transport pathways involved in acid excretion or bicarbonate reclamation such as H⁺-ATPases, Na⁺/HCO₃⁻ cotransporter or Na⁺/H⁺-exchangers, and Na⁺/K⁺-ATPase are driven directly by ATP hydrolysis or depend on the sodium gradient and membrane potential generated by ATP-dependent processes. Along the same line, we detected 13 genes of the mitochondrial transporter family (Slc25) to be regulated by metabolic acidosis. Interestingly, the mitochondrial phosphate carrier (Slc25a25) was one of the genes with the highest elevation of expression during metabolic acidosis. These transporters participate in the import of substrates and release of metabolites from mitochondria (35). Also, expression of genes that belong to metabolism of amino acids, lipids, and carbohydrates was affected by metabolic acidosis.

Increased ammoniagenesis is a major process contributing to the renal response to metabolic acidosis (14). Ammonia is synthesized mainly in the proximal tubule from glutamine taken up from blood possibly by the Na⁺-dependent amino acid transporter SNAT3 (Slc38a3). During metabolic acidosis SNAT3 is upregulated on mRNA and protein levels, and its expression spreads from the late proximal tubule to the early proximal tubule (27, 32, 43). In our experiments, the expression of SNAT3 was strongly elevated after 2 and 7 days of acid loading, which was confirmed by qRT-PCR and Western blotting. We also observed upregulation of many additional genes involved in renal ammoniagenesis namely the phosphate-dependent glutaminase (PDG), glutamate dehydrogenase (Glud1), mitochondrial malate transporter (Slc25a10), malate dehydrogenase 1 (Mdh1), and phospho-enolpyruvate carboxykinase (PEPCK), combined with downregulation of glutamine synthetase (Glul) (Supplementary Tables S2 and S3, Fig. 5). Effects of metabolic acidosis on expression of PDG, Glud1, PEPCK, and Glul were previously reported in other studies (Table 6); however, our data expand the list of genes considerably and suggest a concerted regulation of the whole pathway. Activation of ammoniagenesis was concomitant with increased expression of TCA cycle enzymes: α-ketoglutarate dehydrogenase (αKGD), malate dehydrogenase 2 (MDH2), citrate synthase (CS), and aconitase 2 (Aco2). The enhanced efflux of malate from mitochondria during acidosis has been described previously (33). However, in contrast to our data it has been reported that flux through citrate synthase is reduced. The apparent activation of the TCA observed here is in agreement with upregulation of the sodium-dependent citrate transporters NaDC1 (Slc13a2) (4) and NaDC3 (Slc13a3). NaDC1 and NaDC3 transport succinate, citrate and α-ketoglutarate, TCA cycle intermediates, and it has been suggested that NaDC1 regulation may provide more substrate for bicarbonate production.

Previous studies linked metabolic acidosis to increased renal gluconeogenesis (18, 39). Others have found no evidence for increased glucose release from kidney during acidosis (20). Our data indicate that gluconeogenesis may be reduced given that the key enzymes of glucose synthesis, glucose 6-phosphatase and fructose bisphosphatase, were severely downregulated (Fig. 5, Supplementary Table S2). It remains to be clarified if these apparent discrepancies are due to the different animal species used. Moreover, our data reflect only changes on the transcriptional level but not on the level of enzyme activity.

Both ammoniagenesis and gluconeogenesis take place in proximal tubule cells (14). Since the proximal tubule can be subdivided into at least two distinct subsegments, the early or convoluted tubule (S1/S2 segment) and the late or straight proximal tubule (S3 segment), we asked whether there is a differential transcriptional response to acidosis. Interestingly, qRT-PCR of S1/S2 and S3 segments demonstrated both spatial and temporal patterns of gene expression and regulation. Under control conditions, components of the gluconeogenic pathway were mostly found in the early proximal tubule, where downregulation during metabolic acidosis was more pronounced. In contrast, key transporters and enzymes of ammoniagenesis were mostly localized to the late proximal tubule and were strongly upregulated both in the early and late proximal tubule during metabolic acidosis. The very low mRNA expression of SNAT3 in the early proximal tubule is in agreement with the low abundance of the protein in this part of the nephron as detected by immunohistochemistry (32). Adaptation to metabolic acidosis involves also nephron remodeling increasing the number of acid-secretory type A intercalated cells and leading to hypertrophy of various cell types along the nephron (2, 30, 31, 49). In the present study we observed a large number of transcripts (both up- and downregulated) that are involved in cell death/apoptosis, growth, and differentiation. Therefore, it may be possible that the increased number of type A intercalated cells during chronic acidosis may involve proliferation combined with apoptotic removal of type B intercalated cells. In fact, we have observed during preliminary experiments that type A intercalated cells stain positive for various markers of proliferation, whereas type B intercalated were devoid of such markers (D. Bacic, M. Nowik, M. LeHir, B. Kaissling, and C. A. Wagner, unpublished results). Similarly, Cheval et al. (12) observed by SAGE analysis transcripts of proliferative pathways in isolated mouse outer medullary collecting duct. Moreover, proximal tubular cells also show prominent staining of proliferative markers during chronic acidosis (D. Bacic, M. Nowik, M. LeHir, B. Kaissling, and C. A. Wagner, unpublished observations).

We also observed upregulation of several genes involved in renal water handling, among them the water channels AQP1, AQP2, AQP3, AQP4, the urea transporter UT-A1, and the vasopressin receptor V2R. Regulation of AQP2 during NH₄Cl-loading has been previously reported (3). Acidotic mice, however, consumed ~20% less water than control animals (data not shown). Thus, we cannot exclude that the addition of NH₄Cl to drinking water induced also a mild

dehydration that may contribute to the regulation observed. Water homeostasis in metabolic acidosis deserves further investigation as deranged water homeostasis has also been observed in inborn or acquired syndromes of renal tubular acidosis (8, 38, 46).

In summary, in our study >4,000 transcripts were significantly (*P* < 0.05) regulated during metabolic acidosis, which was ~40% of all genes expressed in the kidney. Major pathways regulated affect energy homeostasis, acid excretion, water and electrolyte balance, as well as pathways involved in apoptosis and cell proliferation. The latter may contribute to the remodeling of the kidney. Our data provide a genome-wide view on the complex regulatory and adaptive processes during metabolic acidosis and will allow elucidating the exact role of regulated pathways.

### ACKNOWLEDGMENTS

The use of the ZIHP Core Facility for Rodent Physiology is gratefully acknowledged.

### GRANTS

This study was supported by the 6th Framework EU project EuReGene (005085) to A. W. Brändli and C. A. Wagner. M. Nowik is supported by a PhD student fellowship from the Zurich Center for Integrative Physiology (ZIHP).

### REFERENCES

1. SRI International. Panther classification system, www.pantherdb.org.
2. Al-Awqati Q. Plasticity in epithelial polarity of renal intercalated cells: targeting of the H<sup>+</sup>-ATPase and band 3. *Am J Physiol Cell Physiol* 270: C1571–C1580, 1996.
3. Amlal H, Sheriff S, Soleimani M. Upregulation of collecting duct aquaporin-2 by metabolic acidosis: role of vasopressin. *Am J Physiol Cell Physiol* 286: C1019–C1030, 2004.
4. Aruga S, Wehrli S, Kaisling B, Moe OW, Preisig PA, Pajor AM, Alpern RJ. Chronic metabolic acidosis increases NaDC-1 mRNA and protein abundance in rat kidney. *Kidney Int* 58: 206–215, 2000.
5. Berthelot M. Violet d'aniline. *Rep Chim App* 1: 284, 1859.
6. Biber J, Stieger B, Haase W, Murer H. A high yield preparation for rat kidney brush border membranes. Different behaviour of lysosomal markers. *Biochim Biophys Acta* 647: 169–176, 1981.
7. Brosnan ME, Brosnan JT. Renal arginine metabolism. *J Nutr* 134: 2791S–2795S, 2004.
8. Bruce LJ, Cope DL, Jones GK, Schofield AE, Burley M, Povey S, Unwin RJ, Wrong O, Tanner MJ. Familial distal renal tubular acidosis is associated with mutations in the red cell anion exchanger (Band 3, AE1) gene. *J Clin Invest* 100: 1693–1707, 1997.
9. Capurro C, Coutry N, Bonvalet JP, Escoubet B, Garty H, Farman N. Cellular localization and regulation of CHIF in kidney and colon. *Am J Physiol Cell Physiol* 271: C753–C762, 1996.
10. Capurro C, Coutry N, Bonvalet JP, Escoubet B, Garty H, Farman N. Specific expression and regulation of CHIF in kidney and colon. *Ann NY Acad Sci* 834: 562–564, 1997.
11. Chabardes-Garonne D, Mejean A, Aude JC, Cheval L, Di Stefano A, Gaillard MC, Imbert-Teboul M, Wittner M, Balian C, Anthouard V, Robert C, Segurens B, Wincker P, Weissenbach J, Doucet A, Elalouf JM. A panoramic view of gene expression in the human kidney. *Proc Natl Acad Sci USA* 100: 13710–13715, 2003.
12. Cheval L, Morla L, Elalouf JM, Doucet A. The kidney collecting duct acid-base “regulon”. *Physiol Genomics* 27: 271–281, 2006.
13. Conjard A, Komaty O, Delage H, Boghossian M, Martin M, Ferrier B, Bavere! G. Inhibition of glutamine synthetase in the mouse kidney: a novel mechanism of adaptation to metabolic acidosis. *J Biol Chem* 278: 38159–38166, 2003.
14. Curthoys NP, Gstraunthaler G. Mechanism of increased renal gene expression during metabolic acidosis. *Am J Physiol Renal Physiol* 281: F381–F390, 2001.
15. Curthoys NP, Taylor L, Hoffert JD, Knepper MA. Proteomic analysis of the adaptive response of rat renal proximal tubules to metabolic acidosis. *Am J Physiol Renal Physiol* 292: F140–F147, 2007.
16. DuBose T Jr, Alpern RJ. Renal tubular acidosis. In: *The Metabolic and Molecular Bases of Inherited Disease*, edited by Scriver CR, Beaudet AL, Sly WS, Valle D. New York: McGraw-Hill, 2001, p. 4983–5021.
17. Eisen MB, Spellman PT, Brown PO, Botstein D. Cluster analysis and display of genome-wide expression patterns. *Proc Natl Acad Sci USA* 95: 14863–14868, 1998.
18. Ekberg K, Landau BR, Wajngot A, Chandramouli V, Efendic S, Brunengraber H, Wahren J. Contributions by kidney and liver to glucose production in the postabsorptive state and after 60 h of fasting. *Diabetes* 48: 292–298, 1999.
19. Frische S, Kwon TH, Frokiaer J, Madsen KM, Nielsen S. Regulated expression of pendrin in rat kidney in response to chronic NH4Cl or NaHCO3 loading. *Am J Physiol Renal Physiol* 284: F584–F593, 2003.
20. Goldstein L. Renal substrate utilization in normal and acidotic rats. *Am J Physiol Renal Fluid Electrolyte Physiol* 253: F351–F357, 1987.
21. Greenbaum D, Colangelo C, Williams K, Gerstein M. Comparing protein abundance and mRNA expression levels on a genomic scale. *Genome Biol* 4: 117, 2003.
22. Halperin ML, Vinay P, Gougoux A, Pichette C, Jungas RL. Regulation of the maximum rate of renal ammoniagenesis in the acidic dog. *Am J Physiol Renal Fluid Electrolyte Physiol* 248: F607–F615, 1985.
23. Hamm LL, Alpern RJ. Cellular mechanisms of renal tubular acidification. In: *The Kidney (3rd ed.)*, edited by Seldin DW, Giebisch G. Philadelphia, PA: Lippincott, 2001, p. 1995–2013.
24. Huber W, von Heydebreck A, Sültmann H, Poustka A, Vingron M. Variance stabilization applied to microarray data calibration and to the quantification of differential expression. *Bioinformatics (Oxford)* 18, Suppl 1: S96–S104, 2002.
25. Hwang JJ, Curthoys NP. Effect of acute alterations in acid-base balance on rat renal glutaminase and phosphoenolpyruvate carboxykinase gene expression. *J Biol Chem* 266: 9392–9396, 1991.
26. Hwang JJ, Perera S, Shapiro RA, Curthoys NP. Mechanism of altered renal glutaminase gene expression in response to chronic acidosis. *Biochemistry* 30: 7522–7526, 1991.
27. Karinch AM, Lin CM, Wolfgang CL, Pan M, Souba WW. Regulation of expression of the SN1 transporter during renal adaptation to chronic metabolic acidosis in rats. *Am J Physiol Renal Physiol* 283: F1011–F1019, 2002.
28. Kwon TH, Fulton C, Wang W, Kurtz I, Frokiaer J, Aalkjaer C, Nielsen S. Chronic metabolic acidosis upregulates rat kidney Na-HCO cotransporters NBCn1 and NBC3 but not NBC1. *Am J Physiol Renal Physiol* 282: F341–F351, 2002.
29. Lin PJ, Williams WP, Luu Y, Molday RS, Orlowski J, Numata M. Secretory carrier membrane proteins interact and regulate trafficking of the organellar (Na<sup>+</sup>,K<sup>+</sup>)/H<sup>+</sup> exchanger NHE7. *J Cell Sci* 118: 1885–1897, 2005.
30. Madsen KM, Tisher CC. Response of intercalated cells of rat outer medullary collecting duct to chronic metabolic acidosis. *Lab Invest* 51: 268–276, 1984.
31. Madsen KM, Verlander JW, Kim J, Tisher CC. Morphological adaptation of the collecting duct to acid-base disturbances. *Kidney Int Suppl* 33: S57–S63, 1991.
32. Moret C, Dave MH, Schulz N, Jiang JX, Verrey F, Wagner CA. Regulation of renal amino acid transporters during metabolic acidosis. *Am J Physiol Renal Physiol* 292: F555–F566, 2007.
33. Nissim I, Nissim I, Yudkoff M. Adaptation of renal tricarboxylic acid cycle metabolism to various acid-base states: study with [3-13C,5-15N]glutamine. *Miner Electrolyte Metab* 17: 21–31, 1991.
34. Numata M, Orlowski J. Molecular cloning and characterization of a novel (Na<sup>+</sup>,K<sup>+</sup>)/H<sup>+</sup> exchanger localized to the trans-Golgi network. *J Biol Chem* 276: 17387–17394, 2001.
35. Palmieri L, Lasorsa FM, Vozza A, Agrimi G, Fiermonte G, Runswick MJ, Walker JE, Palmieri F. Identification and functions of new transporters in yeast mitochondria. *Biochim Biophys Acta* 1459: 363–369, 2000.
36. Petrovic S, Wang Z, Ma L, Soleimani M. Regulation of the apical Cl<sup>-</sup>/HCO-3 exchanger pendrin in rat cortical collecting duct in metabolic acidosis. *Am J Physiol Renal Physiol* 284: F103–F112, 2003.
37. Puttaparthi K, Markovich D, Halaihel N, Wilson P, Zajicek HK, Wang H, Biber J, Murer H, Rogers T, Levi M. Metabolic acidosis regulates rat renal Na-Si cotransport activity. *Am J Physiol Cell Physiol* 276: C1398–C1404, 1999.

38. Rodriguez-Soriano J, Vallo A, Castillo G, Oliveros R. Natural history of primary distal renal tubular acidosis treated since infancy. *J Pediatr* 101:669–676, 1982.

39. Schoolwerth AC, deBoer PA, Moorman AF, Lamers WH. Changes in mRNAs for enzymes of glutamine metabolism in kidney and liver during ammonium chloride acidosis. *Am J Physiol Renal Fluid Electrolyte Physiol* 267: F400–F406, 1994.

40. Schwab K, Patterson LT, Aronow BJ, Luckas R, Liang HC, Potter SS. A catalogue of gene expression in the developing kidney. *Kidney Int* 64: 1588–1604, 2003.

41. Schwartz GJ, Winkler CA, Zavilowitz BJ, Bargiello T. Carbonic anhydrase II mRNA is induced in rabbit kidney cortex during chronic metabolic acidosis. *Am J Physiol Renal Fluid Electrolyte Physiol* 265: F764–F772, 1993.

42. Seaton B, Ali A. Simplified manual high performance clinical chemistry methods for developing countries. *Med Lab Sci* 41: 327–336, 1984.

43. Solbu TT, Boulland JL, Zahid W, Lyamouri Bredahl MK, Amiry-Moghaddam M, Storm-Mathisen J, Roberg BA, Chaudhry FA. Induction and targeting of the glutamine transporter SN1 to the basolateral membranes of cortical kidney tubule cells during chronic metabolic acidosis suggest a role in pH regulation. *J Am Soc Nephrol* 16: 869–877, 2005.

44. Soutourina O, Cheval L, Doucet A. Global analysis of gene expression in mammalian kidney. *Pflügers Arch* 450: 13–25, 2005.

45. Stehberger PA, Schulz N, Finberg KE, Karet FE, Giebisch G, Lifton RP, Geibel JP, Wagner CA. Localization and regulation of the

ATP6V0A4 (a4) vacuolar H+-ATPase subunit defective in an inherited form of distal renal tubular acidosis. *J Am Soc Nephrol* 14: 3027–3038, 2003.

46. Stehberger PA, Shmukler BE, Stuart-Tilley AK, Peters LL, Alper SL, Wagner CA. Distal renal tubular acidosis in mice lacking the AE1 (band3) Cl⁻/HCO₃⁻ exchanger (slc4a1). *J Am Soc Nephrol* 18: 1408–1418, 2007.

47. Stuart RO, Bush KT, Nigam SK. Changes in global gene expression patterns during development and maturation of the rat kidney. *Proc Natl Acad Sci USA* 98: 5649–5654, 2001.

48. Tsuruoka S, Kittelberger AM, Schwartz GJ. Carbonic anhydrase II and IV mRNA in rabbit nephron segments: stimulation during metabolic acidosis. *Am J Physiol Renal Physiol* 274: F259–F267, 1998.

49. Verlander JW, Madsen KM, Cannon JK, Tisher CC. Activation of acid-secreting intercalated cells in rabbit collecting duct with ammonium chloride loading. *Am J Physiol Renal Fluid Electrolyte Physiol* 266: F633–F645, 1994.

50. Virlon B, Cheval L, Buhler JM, Billon E, Doucet A, Elalouf JM. Serial microanalysis of renal transcriptomes. *Proc Natl Acad Sci USA* 96: 15286–15291, 1999.

51. Winkler CA, Kittelberger AM, Schwartz GJ. Expression of carbonic anhydrase IV mRNA in rabbit kidney: stimulation by metabolic acidosis. *Am J Physiol Renal Physiol* 272: F551–F560, 1997.

52. Wright PA, Packer RK, Garcia-Perez A, Knepper MA. Time course of renal glutamate dehydrogenase induction during NH4Cl loading in rats. *Am J Physiol Renal Fluid Electrolyte Physiol* 262: F999–F1006, 1992.
