
Characterization of tRNA-dependent Peptide Bond Formation by MurM in the Synthesis of Streptococcus pneumoniae Peptidoglycan*${}^{S}$

Received for publication, September 28, 2007, and in revised form, December 7, 2007. Published, JBC Papers in Press, December 12, 2007, DOI 10.1074/jbc.M708105200

Adrian J. Lloyd${}^{\dagger 1}$, Andrea M. Gilbey${}^{\ddagger}$, Anne M. Blewett${}^{\ddagger}$, Gianfranco De Pascale${}^{S}$, Ahmed El Zoeiby${}^{\P}$, Roger C. Levesque${}^{\P}$, Anita C. Catherwood${}^{\ddagger}$, Alexander Tomasz${}^{\|}$, Timothy D. H. Bugg${}^{S}$, David I. Roper${}^{\ddagger}$, and Christopher G. Dowson${}^{\ddagger}$

From the Departments of ${}^{\ddagger}$Biological Sciences and ${}^{S}$Chemistry, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom, the ${}^{\P}$Centre de Recherche sur la Fonction, Structure et Ingénierie des Protéines, Faculté de Médecine, Pavillon Charles-Eugène Marchand, Université Laval, Ste-Foy, Quebec G1K 7P4, Canada, and the ${}^{\|}$Laboratory of Microbiology, The Rockefeller University, New York, New York 10021

---

MurM is an aminoacyl ligase that adds $L$-serine or $L$-alanine as the first amino acid of a dipeptide branch to the stem peptide lysine of the pneumococcal peptidoglycan. MurM activity is essential for clinical pneumococcal penicillin resistance. Analysis of peptidoglycan from the highly penicillin-resistant *Streptococcus pneumoniae* strain 159 revealed that *in vivo* and *in vitro*, in the presence of the appropriate acyl-tRNA, MurM${}_{159}$ alanylated the peptidoglycan $\epsilon$-amino group of the stem peptide lysine in preference to its seryl ation. However, in contrast, identical analyses of the penicillin-susceptible strain Pn16 revealed that MurM${}_{\text{Pn16}}$ activity supported seryl ation more than alanylation both *in vivo* and *in vitro*. Interestingly, both MurM${}_{\text{Pn16}}$ acylation activities were far lower than the alanylation activity of MurM${}_{159}$. The resulting differing stem peptide structures of 159 and Pn16 were caused by the profoundly greater catalytic efficiency of MurM${}_{159}$ compared with MurM${}_{\text{Pn16}}$ brought about by sequence variation between these enzymes and, to a lesser extent, differences in the *in vivo* tRNA${}^{\text{Ala}}$:tRNA${}^{\text{Ser}}$ ratio in 159 and Pn16. Kinetic analysis revealed that MurM${}_{159}$ acted during the lipid-linked stages of peptidoglycan synthesis, that the $D$-alanyl-$D$-alanine of the stem peptide and the lipid II $N$-acetylglucosaminyl group were not essential for substrate recognition, that $\epsilon$-carboxylation of the lysine of the stem peptide was not tolerated, and that lipid II-alanine was a substrate, suggesting an evolutionary link to staphylococcal homologues of MurM such as FemA. Kinetic analysis also revealed that MurM recognized the acceptor stem and/or the T$\Psi$C loop stem of the tRNA${}^{\text{Ala}}$. It is anticipated that definition of the minimal structural features of MurM substrates will allow development of novel resistance inhibitors that will restore the efficacy of $\beta$-lactams for treatment of pneumococcal infection.

The peptidoglycan in *Streptococcus pneumoniae* and other Gram-positive pathogens is composed of a carbohydrate polymer consisting of alternating residues of $N$-acetylglucosamine and $N$-acetylmuramic acid. Appended to the $N$-acetylmuramic acid residue is the “stem peptide” composed of up to five amino acids, $L$-alanyl-$\gamma$-$D$-glutamyl-$L$-lysyl-$D$-alanyl-$D$-alanine. The stem peptides are themselves cross-linked between the $\epsilon$-amino group of the lysine of a pentapeptide and the carbonyl group of the fourth position $D$-alanine of an adjacent stem (1). The pneumococcal stem peptide is further modified in *S. pneumoniae* where the lysyl residue $\epsilon$-amino group is substituted by a dipeptide branch consisting of $L$-alanine or $L$-serine followed invariably by $L$-alanine (2–4).

The stem peptide is constructed in the cytoplasm appended to a UDP nucleotide (Fig. 1) in a series of reactions catalyzed by MurA to F, where MurC, -D, -E, and -F are responsible for the ATP-dependent ligation of $L$-alanine, $D$-glutamate, $L$-lysine, and $D$-alanyl-$D$-alanine, respectively, to form UDP-$N$-acetylmuramyl-$L$-alanyl-$\gamma$-$D$-glutamyl-$L$-lysyl-$D$-alanyl-$D$-alanine (UDP-MurNAcAEKAA) (1). The phospho-$N$-acetylmuramyl pentapeptide is transferred from this species by MraY to a membrane-bound undecaprenyl-phosphate carrier to form lipid I (undecaprenyl-pyrophosphoryl-$N$-acetylmuramyl-$L$-alanyl-$\gamma$-$D$-glutamyl-$L$-lysyl-$D$-alanyl-$D$-alanine), which is then glycosylated with UDP-$N$-acetylglucosamine by MurG to form lipid II (undecaprenyl-pyrophosphoryl-$N$-acetylmuramyl ($N$-acetylglucosaminyl)-$L$-alanyl-$\gamma$-$D$-glutamyl-$L$-lysyl-$D$-alanyl-$D$-alanine) (1) (Fig. 1). The dipeptide branch is added to the stem peptide lysine at some point after the stem peptide is constructed (5–9) (e.g. lipid II in Fig. 1). After transport to the outer face of the cytoplasmic membrane, lipid II is polymerized by transglycosylation. This nascent peptidoglycan is given structural rigidity by transpeptidation between the position 3 lysine (with or without a dipeptide branch) and the fourth position $D$-alanine of adjacent stem peptides (Fig. 1) (1).

The pneumococcal genes encoding the enzymes that construct the dipeptide branch, MurM and MurN, add the first and second amino acids to the stem peptide lysine, respectively (10, 11). *S. pneumoniae* has acquired related MurM sequences within its genome by homologous recombination

* This work was supported by Wellcome Trust Grant 066443. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
${}^{S}$ The on-line version of this article (available at http://www.jbc.org) contains supplemental Materials and Methods, Results, additional references, Figs. S1–S5, and Table S1.
${}^{1}$ To whom correspondence should be addressed. Tel.: 44-2476-522568; Fax: 44-2476-523701; E-mail: adrian.lloyd@warwick.ac.uk.

Enzymology of the Aminoacyl Ligase MurM

UDP-MurNAc-Ala-Glu-Lys
MurE
ADP + P<sub>i</sub>
L-Lys

UDP-MurNAc-Ala-Glu
MurD
ADP + P<sub>i</sub>
ATP
D-Glu

UDP-MurNAc-Ala
MurC
ADP + P<sub>i</sub>
ATP
L-Ala

UDP-MurNAc
MurB
ATP
NADP⁺
NADPH

UDP-enol-pyruvyl-GlcNAc
MurA
P<sub>i</sub>
Phospho-enol pyruvate

ATP
MurF
D-Ala
ADP + P<sub>i</sub>

L-I
MurG
D-Ala
UDP-GlcNAc
Ser-tRNA<sup>Ser</sup>
or
Ala-tRNA<sup>Ala</sup>
D-Ala
D-Ala
L-Lys
D-Glu
L-Ala
D-Ala
L-Lys-Ser
D-Glu
L-Ala
D-Ala
L-Lys-Ser-Ala
D-Glu
L-Ala
D-Ala
MurNAc-GlcNAc
P

L-II
MurM
D-Ala
MurNAc-GlcNAc
P
D-Ala
L-Ala
D-Glu
L-Lys-Ser-Ala
D-Ala
D-Ala
L-Ala
D-Glu
L-Lys-Ser-Ala
D-Ala
D-Ala
MurNAc-GlcNAc
P

Cytoplasm

undecaprenyl phosphate

Cell Membrane

Penicillin binding protein transpeptidase

Nascent Peptidoglycan

-MurNAc-GlcNAc-MurNAc-GlcNAc-
L-Ala
D-Glu
L-Lys-Ser-Ala
D-Ala
L-Ala
D-Glu
L-Lys-Ser-Ala
D-Ala

Extracellular Space

FIGURE 1. The peptidoglycan synthetic pathway. Intracellular cytoplasmic and cell membrane-bound steps are shown with the polymerization reactions that generate the nascent peptidoglycan on the extracellular face of the cell membrane. L-I and L-II denote lipid I and lipid II, respectively. The C-terminal D-alanine that is lost in transpeptidation is shown in yellow on a green background, and the remaining stem peptide amino acids are in black. The GlcNAc residues of the carbohydrate backbone of the peptidoglycan are violet; the serine or alanine added by MurM is in light green, and the alanine added by MurN is brown. The undecaprenyl-phosphate carrier that is cycled between the extracellular and intracellular faces of the cytoplasmic membrane is in red. For purposes of simplification, the figure is drafted assuming that MurM and MurN act exclusively after the formation of lipid II, where formally, they could utilize any intermediate after the MurE step.

to create a family of mosaics of related murM genes (12, 13). This has endowed the resulting family of MurM variants with vastly differing levels of activity in vivo and differing amino acid specificities for incorporation of alanine and serine (2, 12, 14).

Although not absolutely essential for high level penicillin resistance in laboratory strains of S. pneumoniae, clinical strains of this organism depend upon the activity of MurM for high level penicillin resistance (14–16). Despite the medical importance of this protein, knowledge of the enzyme biochemistry of MurM is sketchy and inferred. Unavailability of the peptidoglycan precursor substrates of MurM has previously restricted analysis of this protein to what can be deduced from molecular genetics, bioinformatics, and analysis of peptidoglycan precursor pools within S. pneumoniae. Recent successes in the in vitro synthesis of these precursors in our laboratory and elsewhere have, however, made the analysis of MurM enzymology a realistic proposition (17–19).

We have therefore carried out a characterization of the enzymatic properties of MurM from two clinical isolates of S. pneumoniae, one highly penicillin-resistant (159) and the other penicillin-sensitive (Pn16), that has allowed us in this paper to 1) confirm the type of enzymatic reaction carried out by MurM; 2) correlate the enzyme biochemistry of MurM with the final composition of the peptidoglycan of these two strains; 3) deduce the specificity of MurM for its peptidoglycan precursor substrates, allowing delineation of what is required for substrate binding by this enzyme; and 4) define those regions of the tRNA<sup>Ala</sup> substrate of MurM that are required for binding and catalysis.

MATERIALS AND METHODS

Chemicals

RNA species corresponding to the sequence of the full-length pneumococcal tRNA<sup>Ala</sup><sub>UGC</sub> isoacceptor GGG GCC UUA GCU CAG CUG GGA GAG CGC CUG CUU UGC ACG CAG GAG GUC AGC GGU UCG AUC CCG CUA GGC UCC ACC A and corresponding to the 3′-aminoacylation site, acceptor stem, and TΨC loop and stem of the full-length pneumococcal tRNA<sup>Ala</sup><sub>UGC</sub>, GGG GCC UAG CGG UUC GAU CCC GCU AGG CUC CAC CA (RNA minihelix), were synthesized, purified, and supplied with a 5′-phosphorylation by Dharmacon Inc. S. pneumoniae Pn16 MurE (MurE<sub>Pn16</sub>) and Pseudomonas

MARCH 7, 2008 • VOLUME 283 • NUMBER 10 JOURNAL OF BIOLOGICAL CHEMISTRY 6403

# Enzymology of the Aminoacyl Ligase MurM

aeruginosa MurA, MurB, MurC, MurD, MurE, and MurF were overexpressed and purified (18, 20). Bacterial purine nucleoside phosphorylase (Sigma) was repurified (21). Pig heart isocitrate dehydrogenase (NADP⁺; Sigma) was re-purified by elution from Sepharose 4B-Procion Blue MX2G with 1 mM NADPH. Undecaprenyl-MurNAc(GlcNAc)-L-alanyl-γ-D-glutamyl-L-lysyl(εN-L-alanine)-D-alanyl-D-alanine (lipid II-Ala) was a generous gift from Dr. G. dePascale (Warwick University). Other chemicals are recorded in the supplemental Materials and Methods and were sourced as in Ref. 22 or were from Sigma or Melford Laboratories Ltd.

## Escherichia coli Strains and Plasmids

Details of *E. coli* strains and plasmids used in this study are indicated in supplemental Materials and Methods.

## S. pneumoniae Strains and Isolation of Pneumococcal DNA

Pn16 (110K/70) serotype 42, was isolated in Papua, New Guinea and was penicillin-sensitive (minimum inhibitory concentration < 0.016 μg·ml⁻¹) (23). 159 serotype 19A was isolated in Hungary (15) and was penicillin-resistant (minimum inhibitory concentration > 16 μg·ml⁻¹). Strains were propagated on brain heart infusion agar containing 5% (v/v) sheep blood at 37°C in 5% (v/v) CO₂ or in liquid medium in brain heart infusion broth at 37°C in 5% (v/v) CO₂. DNA was extracted from lawns of pneumococci on brain heart infusion blood agar as described (23).

## Micrococcus flavus Membranes

Details of preparation of *M. flavus* membranes are recorded in supplemental Materials and Methods.

## Peptidoglycan Analysis

Peptidoglycan was extracted from late exponential phase *S. pneumoniae*, purified, and digested with muramidase, and the resulting stem peptides were extracted and fractionated by reverse phase HPLC² on a Vydac 218TP54 column (4, 24). Peptidoglycan fragment structural assignments were made according to Refs. 2–4, 11.

## Protein Analytical Methods

SDS-PAGE, N-terminal protein sequencing, protein assays, and Western blotting for histidine tags were performed according to Ref. 22 and references therein.

## Identification and Sequencing of the murM Alleles from Pn16 and 159

To sequence the Pn16 and 159 *murM* genes, primers were designed using the *S. pneumoniae* R6 genome sequence (25) at the J. Craig Ventner Institute (formally The Institute for Genome Research) web site to amplify the region between 232 nucleotides 5′ to the initiator ATG (primer 1, supplemental Table 1) of the *murM* gene to 162 nucleotides 3′ of the *murM*

---

TAA stop codon (primer 2, supplemental Table 1) by PCR. DNA sequence between these primers was amplified by Pwo DNA polymerase according to the manufacturer's instructions. and a product of the correct size (1.6 kb) was purified using a Qiagen spin column and sequenced in both directions.

### Cloning, Overexpression, and Purification of MurM from Pn16 and 159

To construct an expression vector carrying a *murM* allele with a 3′ sequence encoding a hexahistidine (His₆) peptide, 1.3-kb fragments containing the *murM* allele from Pn16 and 159 were amplified by PCR from the appropriate pneumococcal DNA. Because of sequence divergence at the 5′ end of the open reading frame between the *murM* alleles, a 5′ primer for each gene was designed incorporating an NdeI restriction site for amplification of *murM*~*Pn16*~ and *murM*~*159*~, respectively (primers 3 and 4, supplemental Table 1). A single 3′ primer for the amplification of both alleles was designed to incorporate a 3′ XhoI site and eliminate the 3′ stop codon (primer 5, supplemental Table 1). On PCR with PWO polymerase, products of the correct mass for *murM*~*Pn16*~ and *murM*~*159*~ were obtained, purified, restricted with NdeI and XhoI, and ligated into similarly restricted pET21b as described in Ref. 22. Clones carrying the recombinant *murM*~*Pn16*~ and *murM*~*159*~ genes were verified by sequencing, and one correct clone was retained for expression of each protein (pET21b::*murM*~*Pn16*~ and pET21b::*murM*~*159*~).

To overexpress the MurM proteins, 650-ml cultures of *E. coli* C41 (DE3)/pRIL, harboring either pET21b::*murM*~*Pn16*~ or pET21b::*murM*~*159*~ in Luria Broth (LB) + 50 μg/ml carbenicillin + 30 μg/ml chloramphenicol, were grown at 37°C to an *A*~600~ of 0.6–1.0, when MurM expression was induced by 0.5 mM isopropyl β-D-thiogalactopyranoside, concurrent with the growth temperature being reduced to 25°C. *E. coli* cells were harvested after 4 h and washed at 4°C in 50 mM HEPES, 1 mM MgCl₂, pH 7.5, and 2 mM β-mercaptoethanol.

Analysis of whole cells and subcellular fractions thereof by SDS-PAGE and Western blotting suggested that MurM~*Pn16*~ and MurM~*159*~ were mostly insoluble but could be solubilized by 1 M NaCl.³ To purify MurM~*Pn16*~ or MurM~*159*~, all steps were performed at < 4°C. Cell pellets suspended/g in 3 ml of 50 mM HEPES, 1 mM MgCl₂, 0.5 mM EGTA, 2 mM β-mercaptoethanol, 1 μM leupeptin, 1 μM pepstatin, and 0.2 mM phenylmethanesulfonyl fluoride, pH 7.0, for 30 min and then centrifuged for 30 min at 100,000 × g. The supernatant was retained, and the 100,000 × g pellet was re-extracted as above, and the supernatants were combined. The supernatant was sequentially fractionated between 25 and 50% saturation ammonium sulfate and by gel exclusion chromatography on a 500-ml Sephacryl S-200 column in 50 mM NaH₂PO₄, 0.5 M NaCl, 0.2 mM phenylmethanesulfonyl fluoride, 1 μM leupeptin, 1 μM pepstatin, pH 7.0 (phosphate buffer). Fractions containing

---

² The abbreviations used are: HPLC, high pressure liquid chromatography; CHAPS, 3-[(3-cholamidopropyl)dimethylammonio]1-propanesulfonic acid; MOPS, 4-morpholinepropanesulfonic acid; DTT, dithiothreitol; DAP, meso-diaminopimelic acid; AlaRS, alanyl-tRNA<sup>Ala</sup> synthetase; SerRS, seryl-tRNA<sup>Ser</sup> synthetase; ES-MS, electrospray-mass spectrometry.

³ A.J. Lloyd, unpublished data.

MurM by Western blot were further purified by immobilized metal affinity chromatography on a 25-ml column of cobalt-talon resin (Clontech) in phosphate buffer. Once unbound proteins were eluted, MurM was eluted by a 0–0.2 M imidazole gradient. The purity and identity of the final products of these purifications were assessed by SDS-PAGE, Western blotting, and N-terminal sequencing.

**Cloning, Overexpression, and Purification of Alanyl-tRNA<sup>Ala</sup> Synthetase (AlaRS) and Seryl-tRNA<sup>Ser</sup> Synthetase (SerRS) from Pn16 and MurF from 159**

Using the pneumococcal sequences in Ref. 26, the above genes were cloned and overexpressed, and their products were purified by immobilized metal affinity chromatography and anion exchange chromatography. Details of these procedures are given in supplemental Materials and Methods.

**Synthesis of UDP-linked Peptidoglycan Precursors**

**UDP-N-acetylmuramyl-L-alanyl-γ-D-glutamyl-L-lysine (UDP-MurNAcAEK)**

Syntheses were conducted at 37°C overnight in 2-ml volumes in air-tight tubes with no head space, in 50 mM HEPES, 1 mM dithiothreitol (DTT), 50 mM KCl, 10 mM MgCl₂ adjusted to pH 7.5, 3.65 μM MurA, 7.24 μM MurB, 3.50 μM MurC, 9.91 μM MurD, 9.09 μM MurE<sub>Pn16</sub>, 0.4 μmol·min⁻¹·ml⁻¹ NADP⁺-linked isocitrate dehydrogenase, 6.7 μmol·min⁻¹·ml⁻¹ rabbit muscle pyruvate kinase, 26.7 mM DL-isocitrate, 79.8 mM phosphoenolpyruvate, 13.3 mM UDP-GlcNAc, 0.1 mM NADPH, 5 mM ATP, 20 mM L-alanine, 22.9 mM D-glutamate and 15 mM L-lysine.

**UDP-MurNAcAEKAA**

Syntheses were conducted as for UDP-MurNAcAEK, except that the phosphoenolpyruvate concentration was 99.8 mM and the incubations also contained 15 mM D-alanyl-D-alanine and 9.09 μM MurF<sub>159</sub>.

**UDP-N-acetylmuramyl-L-alanyl-γ-D-glutamyl-meso-diaminopimelyl-D-alanyl-D-alanine (UDP-MurNAcAE(DAP)AA)**

Syntheses were conducted as described for UDP-MurNAc-AEKAA, except that lysine was replaced by 30 mM meso-diaminopimelic acid (DAP), and both MurE<sub>Pn16</sub> and MurF<sub>159</sub> were replaced by *P. aeruginosa* MurE and MurF.

In all cases, the UDP-MurNAc peptide product was freed from protein by centrifugation through a Mr 10,000 cutoff membrane, and the filtrate was fractionated on a 50-ml column of Source 30Q anion exchange resin from which it was eluted using a 0–1 M ammonium acetate gradient at pH 7.5. Fractions containing the UDP-MurNAcAEK product were identified enzymatically utilizing MurF<sub>159</sub> and by negative ion electrospray-mass spectrometry (ES-MS) for this and all other UDP-MurNAc peptides. All products were lyophilized three times versus water and stored in solution at –20°C.

**Synthesis of Lipid-linked Peptidoglycan Precursors**

All syntheses were conducted using *Micrococcus flavus* membranes essentially as described in Ref. 19. The lipid I or lipid II products were purified as described previously (19).

**Analysis of Lipid-linked Peptidoglycan Precursors**

To assay lipid-linked precursors, 50 μl of lipid I or II species suspended in 50 mM HEPES, 10 mM MgCl₂, 30 mM KCl, and 1.5% (w/v) CHAPS, pH 7.6, were added to 50 μl of 1 M HCl. Samples were boiled for 30 min and neutralized with 2 M NaOH. The phosphate released was assayed according to Refs. 21, 27. Synthesis of lipid precursors was confirmed by TLC on silica and by negative ion ES-MS as in Ref. 19.

**Sequencing of tRNA<sup>Ala</sup> and tRNA<sup>Ser</sup> Genes from Pn16 and 159**

Four tRNA<sup>Ser</sup> genes, tRNA<sup>Ser(1)</sup>, tRNA<sup>Ser(2)</sup>, tRNA<sup>Ser(3)</sup>, and tRNA<sup>Ser(4)</sup> corresponding to (anticodon/locus tag) GCU/SP2253, UGA/SP2258, UGA/SP2291, and GGA/SP2247, respectively, and four UGC anticodon tRNA<sup>Ala</sup> genes (1–4 with locus tags SP2270, SP2282, SP2295, and SP2243, respectively) were identified in the *S. pneumoniae* TIGR4 genome (26). Of these, tRNA<sup>Ala(2)</sup><sub>UGC</sub> and tRNA<sup>Ala(3)</sup><sub>UGC</sub> were located within blocks of sequence that were identical for 3.294 kb 5′ and 1.995 kb 3′ to the gene of interest, and were not amenable to PCR amplification. For the remaining genes, primers 12–23 were designed starting at 250 bp upstream and 250 bp downstream of the mature tRNA sequence (supplemental Table 1), and genes encoding Pn16 tRNA<sup>Ala(1)</sup>, Pn16 tRNA<sup>Ala(4)</sup>, Pn16 tRNA<sup>Ser(1–4)</sup>, and 159 tRNA<sup>Ala(1)</sup> were amplified with TaqDNA polymerase; 159 tRNA<sup>Ala(4)</sup> and 159 tRNA<sup>Ser(2)</sup> were amplified by platinum Pfx DNA polymerase, and 159 tRNA<sup>Ser(4)</sup> was amplified with PWO DNA polymerase. All products were of the expected size (0.6 kb) and were sequenced.

No conditions could be found for the amplification of 159 tRNA<sup>Ser(3)</sup>. 159 tRNA<sup>Ser(1)</sup> was amplified with platinum Pfx DNA polymerase; however, most unexpectedly, a clean 2-kb product was obtained, the 3′ termini of which were tagged with ATP and TA cloned into a linearized pCR®2.1 vector, according to the manufacturer’s instructions (Invitrogen). The 2-kb insert was then sequenced using the vector-specific m13 primer sequences (supplemental Table 1) either side of the insert.

**Total tRNA Preparation, Aminoacyl-tRNA Preparation, and tRNA<sup>Ala</sup> or tRNA<sup>Ser</sup> Determination**

Techniques employed to isolate, preparatively acylate, and assay tRNA<sup>Ala</sup> and tRNA<sup>Ser</sup> are described in supplemental Materials and Methods.

**MurM Enzyme Assays**

**Spectrophotometric**

This assay followed the cycling of tRNA<sup>Ser</sup> between MurM and SerRS. To a 0.2-ml assay was added 50 mM HEPES, 30 mM KCl, 10 mM MgCl₂, pH 7.6, 1 mM DTT, 1.5% (w/v) CHAPS, 0.25 mM NADH, 2 mM phosphoenolpyruvate, 0.2 mM ATP, 6.2 mg·ml⁻¹ total tRNA<sub>159</sub> (1.5 μM in terms of tRNA<sup>Ser</sup><sub>159</sub>), 10 mM L-serine, 4.89 μM SerRS, 52.5 μmol·min⁻¹·ml⁻¹ myokinase, 6.60 μmol·min⁻¹·ml⁻¹ pyruvate kinase, 10.50 μmol·min⁻¹·ml⁻¹ lactate dehydrogenase, and 0.14 μM MurM<sub>159</sub>. The ΔA<sub>340</sub> of

# Enzymology of the Aminoacyl Ligase MurM

NADH ($\epsilon_{340 \, \text{nm}} = 6220 \, \text{M}^{-1} \cdot \text{cm}^{-1}$) was followed at $37^\circ \text{C}$, and MurM$_{159}$ activity was then initiated with $25 \, \mu \text{M}$ lipid II.

## Radiochemical

These assays were designed to follow the transfer of label from $[^3\text{H}]$alanyl-tRNA$^{\text{Ala}}$ and $[^3\text{H}]$seryl-tRNA$^{\text{Ser}}$ to the peptidoglycan precursor. Initial experiments examining the stability of the aminoacyl linkage to the tRNA suggested that at $37^\circ \text{C}$ at the pH employed in the spectrophotometric method (7.6) the half-life of *M. flavus* $[^3\text{H}]$alanyl-tRNA$^{\text{Ala}}$ was 9.8 min. However, this could be extended to 46 min by dropping the pH of the assay to 6.8.³ Therefore, to avoid interference by depletion of acyl-tRNA substrate in MurM assays, initial rate data were usually obtained at pH 6.8 within the first 10 min of reaction, where loss of alanyl-tRNA$^{\text{Ala}}$ through chemical deacylation was $<5\%$.

### Transfer of $^3\text{H}$-Amino-acid between $[^3\text{H}]$Acyl-tRNA and Lipid-linked Peptidoglycan Precursors

To follow generation of $^3\text{H}$-alanylated lipid precursors, an assay mix typically in a final volume of $35 \, \mu \text{l}$ routinely contained $50 \, \text{mM}$ MOPS, $30 \, \text{mM}$ KCl, $10 \, \text{mM}$ MgCl₂, pH 6.8, $1.5\%$ (w/v) CHAPS, $1 \, \text{mM}$ DTT, $1 \, \text{mM}$ L-alanine, $10 \, \mu \text{M}$ lipid substrate, and MurM (as indicated). Reactions were initiated by (unless otherwise indicated) the addition of $0.45 \, \mu \text{M}$ $[^3\text{H}]$alanyl-tRNA$^{\text{Ala}}$ ($800 - 1000 \, \text{cpm} \cdot \text{pmol}^{-1}$ unless stated otherwise) and were incubated at $37^\circ \text{C}$ for times specified in the text (although initial rate data were usually taken from the first 2 min of reaction). Reactions were terminated at the appropriate time by the addition of $35 \, \mu \text{l}$ of ice-cold $6 \, \text{M}$ pyridinium acetate, pH 4.5, and $70 \, \mu \text{l}$ of ice-cold butan-1-ol. The incubations were rapidly mixed and centrifuged for 5 min at $1^\circ \text{C}$ at $13,000 \times g$, after which the butan-1-ol phase was washed with $70 \, \mu \text{l}$ of water and then assayed for $^3\text{H}$ by liquid scintillation counting. To follow generation of $^3\text{H}$-serylated lipid precursors, exactly the same procedure was followed, except that the $1 \, \text{mM}$ L-alanine and $0.45 \, \mu \text{M}$ $[^3\text{H}]$alanyl-tRNA$^{\text{Ala}}$ were replaced with $1 \, \text{mM}$ L-serine and $0.45 \, \mu \text{M}$ $[^3\text{H}]$seryl-tRNA$^{\text{Ser}}$.

### Transfer of $^3\text{H}$-Amino-acid between $[^3\text{H}]$Acyl-tRNA and UDP-linked Peptidoglycan Precursors

To follow generation of $^3\text{H}$-alanylated UDP-MurNAc peptide precursors, an assay mix to follow the MurM-catalyzed UDP-MurNAc-peptide-dependent loss of label from *M. flavus* $[^3\text{H}]$alanyl-tRNA$^{\text{Ala}}$ was devised. In a final volume of $80 \, \mu \text{l}$, assays contained $50 \, \text{mM}$ MOPS, $30 \, \text{mM}$ KCl, $10 \, \text{mM}$ MgCl₂, pH 6.8, $1.5\%$ (w/v) CHAPS, $1 \, \text{mM}$ DTT, $1 \, \text{mM}$ L-alanine, UDP-MurNAc peptide substrate (as indicated in the text), and $0.58 \, \mu \text{M}$ MurM$_{159}$. Reactions were initiated by $0.45 \, \mu \text{M}$ $[^3\text{H}]$alanyl-tRNA$^{\text{Ala}}$ and were incubated at $37^\circ \text{C}$ where $10-\mu \text{l}$ samples were taken up to 3 min. Remaining $[^3\text{H}]$alanyl-tRNA$^{\text{Ala}}$ was quantitated by trichloroacetic acid precipitation as described in supplemental Materials and Methods relative to control incubations carried out without MurM or UDP-MurNAc peptide.

## Kinetic Data Analysis

Nonlinear regression analyses of dependences of MurM initial velocity on substrate concentration were performed using GraphPad Prism 4 software.

---

**FIGURE 2.** Fingerprints of stem peptides fractionated by reverse phase HPLC from muramidase digests of peptidoglycans from Pn16 and 159. Stem peptides were isolated from pneumococcal peptidoglycans as under “Materials and Methods” and fractionated by reverse phase HPLC with a C-18 column eluted by an 80-min gradient between 0 and 15% (v/v) acetonitrile in 0.1% (v/v) trifluoroacetic acid (4). Peptide elution was monitored at 215 nm, and peptides were identified by comparison with elution of known peptidoglycan fragments (2, 4, 11, 12, 14). Stem peptide sequences assigned to each species are in black type; amino acids added by MurM are in red type; amino acids added by MurN are in purple type. Labels 1–9 on the Pn16 HPLC trace and 1 and 3 and I–IX on the 159 HPLC trace correspond to stem peptides categorized in Refs. 2, 4, 11, 12, 14.

---

## RESULTS

### Peptidoglycan Analysis Reveals the Relative Activity and Amino Acid Specificity of MurM in Vivo

To gain insight into the activity of MurM in vivo, we examined the peptide structure of muramidase digests of Pn16 and 159 peptidoglycan by reverse-phase HPLC (Fig. 2). This revealed that the major cross-linked species in Pn16 peptidoglycan is dimeric, composed of two adjacent peptides, without any dipeptide branch between them (peak 4). Unbranched single peptide species (monomers, peaks 1 and 2) also predominated in the first 15 min of the Pn16 chromatogram. Beyond 25 min, multimers of cross-linked peptidoglycan precursors (peaks 5–9) eluted (Fig. 2). These formed the minority of the Pn16 material fractionated and are composed of cross-linked peptide dimers (peaks 5–7) and trimers (peaks 8 and 9), which

are linked by dipeptide branches which mostly had serine attached to the lysine of the peptidoglycan peptide (Fig. 2).

This suggested that MurM activity in Pn16 is predominantly serine-specific. In contrast, after 20 min the majority of material eluted on chromatography of muramidase digests of 159 peptidoglycan was composed of monomers (peaks I–III), dimers (peaks IV–VI), and trimers (peaks VII–IX), all of which were substituted by dipeptide branches (Fig. 2). It was clear that *in vivo* MurM activity was considerably greater in 159 than in Pn16. Furthermore, *in vivo* MurM<sub>159</sub> specificity was heavily biased in favor of addition of alanine onto the stem peptide in preference to serine, where the converse was true for MurM<sub>Pn16</sub> (Fig. 2).

### murM<sub>159</sub> and murM<sub>Pn16</sub> Genes Display Marked Sequence Divergence That Underpins the Variation in Peptidoglycan Structure between 159 and Pn16

murM genes display mosaic sequences resulting from homologous recombination among natural populations of streptococci (12, 13). To determine whether this might underlie the marked differences between the stem peptide branching in Pn16 and 159, the murM<sub>Pn16</sub> and murM<sub>159</sub> alleles were sequenced. Both genes encoded proteins of 406 amino acids. Comparison of the inferred amino acid sequences of MurM<sub>Pn16</sub> and MurM<sub>159</sub> in supplemental Fig. 1a revealed considerable amino acid sequence divergence (18%) between the two enzymes overall, with the N-terminal 61 amino acids, residues 115–149 and residues 229–298, differing in sequence by as much as 31, 42, and 44%, respectively.

Clearly, the sequence divergence and differing genetic lineages of murM<sub>Pn16</sub> and murM<sub>159</sub> could contribute to the very differing activities of MurM<sub>Pn16</sub> and MurM<sub>159</sub> *in vivo*. MurM species with a threonine or lysine in position 260 insert predominantly serine or alanine (14). Consistent with this observation and the *in vivo* preferences of MurM<sub>Pn16</sub> and MurM<sub>159</sub>, these proteins possessed a threonine and lysine, respectively, at position 260 (supplemental Fig. S1a).

### Overexpression and Purification of MurM<sub>159</sub> and MurM<sub>Pn16</sub>

To characterize the enzymology of MurM<sub>159</sub> and MurM<sub>Pn16</sub> and relate their *in vitro* behavior to their activity *in vivo*, we cloned, overexpressed, and purified MurM<sub>159</sub> and MurM<sub>Pn16</sub> as their C-terminal (C-(His<sub>6</sub>)) fusions. Pilot experiments in *E. coli* C41(DE3)/pRIL demonstrated that MurM<sub>Pn16</sub> overexpression was in large excess over that of MurM<sub>159</sub> as evidenced by SDS-PAGE and Western blot analysis of whole cells. Subcellular fractionation suggested that both MurM species were almost completely expressed as inclusion bodies. Attempts to obtain soluble MurM, including varying growth temperature and co-expression with chaperones, failed.<sup>4</sup> However, both MurM species could be solubilized by 1 M NaCl. Therefore, we developed a high ionic strength-tolerant purification protocol as follows: high salt solubilization, followed by ammonium sulfate precipitation, followed by gel filtration, followed by immobilized cobalt affinity chromatography to purify MurM.

---

<sup>4</sup>A.M. Gilbey and A.J. Lloyd, unpublished data.

The protocol yielded, per liter of culture, ~3 and 0.5 mg of MurM<sub>Pn16</sub> and MurM<sub>159</sub>, respectively, at a purity in excess of 95% (supplemental Fig. S2a). Confirmation of the identity of the purified products involved Western blot identification of the C-(His)<sub>6</sub> tag on the purified proteins (supplemental Fig. S2b), whereas N-terminal sequencing revealed N-terminal sequences of MurM<sub>Pn16</sub> and MurM<sub>159</sub> were MYRYQIGIPT and MYRYQLG, respectively, where the 6th residue (isoleucine) of MurM<sub>Pn16</sub> is substituted by a leucine in MurM<sub>159</sub> (supplemental Fig. S2c). These results exactly matched N-terminal sequences inferred from sequencing the *murM*<sub>159</sub> and *murM*<sub>Pn16</sub> genes.

### MurM Substrate Synthesis

To provide MurM substrates for enzymological studies, we synthesized UDP-MurNAcAEK, UDP-MurNAcAEKAA, and UDP-MurNAcAE(DAP)AA from UDP-GlcNAc, the required amino acids, and the appropriate combinations of MurA→F from *P. aeruginosa* and *S. pneumoniae*. The purity of the peptides after purification was ≥98% by analytical anion exchange on MonoQ™ FPLC. Syntheses were confirmed by negative ion ES-MS (observed m<sup>-2/2</sup>/expected m<sup>-2/2</sup>) as follows: UDP-MurNAcAEK (502.6326/502.6308); UDP-MurNAcAEKAA (573.6773/573.6679); and UDP-MurNAcAE(DAP)AA (595.6585/595.6629).

The UDP-MurNAc peptides were converted to their lipid I or lipid II derivatives using the MraY and MurG activity associated with *M. flavus* membranes. Both UDP-MurNAcAEKAA and UDP-MurNAcAE(DAP)AA were completely converted to their lipid II derivatives in the presence of UDP-GlcNAc as judged by TLC. Likewise, UDP-MurNAcAEKAA was completely converted to its lipid I derivative in the absence of UDP-GlcNAc. However, even under regimes employing extended incubation times and doubling the quantity of *M. flavus* membranes and UDP-GlcNAc, it proved impossible to completely convert UDP-MurNAcAEK to its corresponding lipid II derivative as determined by TLC. Therefore, UDP-MurNAcAEK was only converted to its lipid I derivative.

After purification of the lipid I and II species as in Ref. 19, they were judged to be at least 95% pure by TLC of the final purified products. Again, syntheses were confirmed by negative ion ES-MS (observed m<sup>-2/2</sup>/expected m<sup>-2/2</sup>) as follows: lipid II (936.4341/936.5222); lipid I (834.8976/834.9828); lipid II(DAP) (958.5496/958.5174); and lipid I-AEK (763.9486/763.9457).

### MurM Depends upon tRNA for the Transfer of Alanine or Serine to Lipid II to Generate Lipid II-Ala or Lipid II-Ser in Vitro

To determine whether MurM<sub>159</sub> depended on tRNA for the acylation of lipid II, we carried out a series of incubations designed to generate alanyl-tRNA<sup>Ala</sup> or seryl-tRNA<sup>Ser</sup> *in situ* to determine whether MurM<sub>159</sub> could then transfer the aminoacyl group to lipid II.

Transfer of Alanine—On TLC analysis of the butan-1-ol-soluble components of incubations involving alanyl-tRNA<sup>Ala</sup>, a product was observed (*R*<sub>f</sub> = 0.27) that was absent if ATP, tRNA<sub>159</sub>, AlaRS, lipid II, MurM, or alanine was omitted (Fig. 3a). This product was purified by anion exchange chromatog-

Enzymology of the Aminoacyl Ligase MurM

FIGURE 3. Transfer of alanine and serine to lipid II as mediated by MurM<sub>159</sub> and tRNA<sub>159</sub>. 1.14 μM MurM<sub>159</sub> was incubated in 50 mM HEPES, 10 mM MgCl<sub>2</sub>, 30 mM KCl, pH 7.6, and 1.5% (v/v) CHAPS, with 1 mM DTT, 5.9 μM AlaRS (or 19.5 μM SerRS), 10 mM L-alanine (or 10 mM L-serine), 0.43 mM lipid II, 5 mM ATP, and 1.72 mg·ml<sup>-1</sup> total tRNA<sub>159</sub> (2.34 μM tRNA<sup>Ala</sup> or 0.944 μM tRNA<sup>Ser</sup> total isoacceptors) at 37 °C for 60 min. Incubations omitting MurM<sub>159</sub>, lipid II, ATP, tRNA<sub>159</sub>, aminoacyl-tRNA synthetase, or L-amino acid were also carried out. Reactions were terminated and extracted with 1 volume of ice-cold 6 M pyridinium acetate and 2 volumes of ice-cold butan-1-ol. The butan-1-ol phases were dried. Samples were resuspended in 10 μl of 2:3:1 chloroform:methanol:water and fractionated by TLC on silica according to Ref. 19. The TLC plate was stained by iodine vapor. Lane 1 corresponds to the complete incubation. Lanes 2–7 are identical to lane 1 except that incubations omitted MurM, lipid II, ATP, tRNA<sub>159</sub>, AlaRS, and alanine, respectively. The product unique to the complete incubation is highlighted (→). b, purification of lipid II-Ala. Incubations were resuspended in 10 ml of 2:3:1 chloroform:methanol:water, loaded onto a 3-ml column of DEAE-Sephacel (acetate counter ion), and isocratically eluted with increasing [NH<sub>4</sub>HCO<sub>3</sub>]. 0.5 ml of each wash was dried and analyzed by TLC as for a. Samples loaded in lanes 1–5 are from eluates from washes containing 2 parts chloroform to 3 parts methanol to 1 part 0.15, 0.2, 0.25, 0.3, and 1.0 M NH<sub>4</sub>HCO<sub>3</sub>, respectively. Species labeled (i and ii) correlate in abundance with lipid II and lipid II-Ala as determined by ES-MS analysis (Not shown except for final wash, see c). c, ES-MS analysis of lipid II-Ala. 0.5 ml of DEAE-Sephacel washes were dried down, washed with water, dried down, and resuspended in 10 mM ammonium acetate:methanol (3:7) and analyzed by ES-MS in negative ion mode. Results from the 2:3:1 chloroform:methanol:NH<sub>4</sub>HCO<sub>3</sub> (1 M) wash are shown. m<sup>-1</sup>/1, m<sup>-2</sup>/2, and m<sup>-3</sup>/3 signify m/z values for the singly, doubly, and triply charged anions of lipid II-Ala (expected values: 1945.0899, 972.0410, and 647.6914, respectively). d, incubations to demonstrate seryl transfer to lipid II. Legend as for a where sample loaded in lane 1 corresponds to the complete incubation. Samples loaded in lanes 2–7 are identical to that in lane 1 except that incubations omitted MurM, lipid II, ATP, tRNA<sub>159</sub>, SerRS, and L-serine, respectively. e, purification of lipid II-Ser. Legend as for b except that sample loaded in lane 1 is the flow-through of the applied sample; lanes 2–9 are from eluates from washes containing 2 parts chloroform to 3 parts methanol to 1 part 0.00, 0.05, 0.10, 0.15, 0.2, 0.25, 0.3, and 1.0 M NH<sub>4</sub>HCO<sub>3</sub>, respectively. Lane 10 was loaded with 17 nmol of lipid II. f, ES-MS analysis of lipid II-Ser. Legend as for c. Results from the 2:3:1 chloroform:methanol:NH<sub>4</sub>HCO<sub>3</sub> (1 M) wash are shown. m<sup>-1</sup>/1, m<sup>-2</sup>/2, and m<sup>-3</sup>/3 signify m/z values for the singly, doubly, and triply charged anions of lipid II-Ser (expected values: 1961.0848, 980.0385, and 653.0231, respectively).

uct was lipid II-Ala. This was further confirmed by ES-MS fragmentation analysis of this species by collision-induced dissociation (supplemental Fig. S3).

Transfer of Serine—On TLC analysis of the butan-1-ol-soluble components from the incubations involving serine, tRNA<sup>Ser</sup>, SerRS, ATP, MurM<sub>159</sub>, and lipid II, no unique product was observed (Fig. 3d). Purification of the components of this incubation (19) (Fig. 3e) yielded a product that on negative ion ES-MS (Fig. 3f) was characterized by a major doubly charged species with an m/z of 980.5023, associated with triply and singly charged species with m/z values of 653.3488 and 1961.1393, which correspond precisely with the expected m/z values 980.0385, 653.0231, and 1961.0848 for the doubly, triply, and singly charged lipid II-Ser. No ions with m/z values suggestive of lipid II were detected in this sample. The identification of the MurM<sub>159</sub> product as lipid II-Ser was further confirmed by ES-MS fragmentation analysis of this species by collision-induced dissociation (supplemental Fig. S3).

MurM Regenerates Unacylated tRNA on Acylation of Lipid II

To further confirm the coupling of SerRS activity to that of MurM<sub>159</sub> implied above, and to demonstrate that unacylated tRNA was the other product of MurM<sub>159</sub>, a continuous spectrophotometric method for monitoring this process was devised. Where SerRS was not limiting for MurM<sub>159</sub> activity, with catalytic concentrations of Ser-tRNA<sup>Ser</sup>, synthesis of lipid II-Ser from lipid II by MurM<sub>159</sub> generated equivalent amounts of tRNA<sup>Ser</sup>. This could be reacylated by SerRS, generating an equivalent amount of adenosine 5′-monophosphate, which could be coupled to yield a continuous spectrophotometric signal (“Materials and Methods,” see “Spectrophotometric”; supplemental Fig. S4a). This assay system generated a SerRS-, tRNA<sup>Ser</sup>-, ATP-, serine-, lipid II-, and MurM<sub>159</sub>-dependent decrease in NADH absorbance (supplemental Fig. S4b). This confirmed that where MurM<sub>159</sub> accumulated lipid II-Ser (above), tRNA<sup>Ser</sup> was generated by this

raphy (19) (Fig. 3b). Negative ion ES-MS analysis (Fig. 3c) showed the mass spectrum of the product was dominated by a doubly charged species with an (observed/expected for lipid II-Ala) m/z of 971.9925/972.0410, associated with triply and singly charged species with m/z values of 647.9958/647.6914 and 1945.6179/1945.0899, respectively, suggesting the prod-

# Enzymology of the Aminoacyl Ligase MurM

## TABLE 1
### Demonstration of acyl-tRNA dependence of MurM from Pn16 and 159

Assays were for 60 min in 50 μl of 10 μM lipid II and 0.29 μM *M. flavus* \[^3H\]alanyl-tRNA^Ala^ (6669 cpm/assay; 457 cpm·pmol^−1^) or 0.26 μM *M. flavus* \[^3H\]seryl-tRNA^Ser^ (3971 cpm/assay; 303 cpm·pmol^−1^). MurM~Pn16~ and MurM~159~ were 5.81 and 2.91 μM, respectively. Butanol-1-extracted counts/min are averages of duplicate incubations that varied by ≤10%. Background was 8.5 cpm.

| \[^3H\]Acyl-tRNA | Incubation         | MurM origin | Butan-1-ol-soluble \[^3H\) minus background | Label in butanol phase |
|------------------|--------------------|-------------|--------------------------------------------|-------------------------|
|                  |                    |             | \(cpm\)                                    | \%                      |
| *M. flavus*      | Complete           | 159         | 5217                                       | 78                      |
| \[^3H\]alanyl-tRNA^Ala^ | Pn16            |             | 462                                        | 6.9                     |
|                  | No MurM            | None        | 221                                        | 3.3                     |
|                  | No lipid II        | 159         | 203                                        | 3.0                     |
|                  |                    | Pn16        | 170                                        | 2.5                     |
|                  | Complete + 0.1 mg·ml^−1^ RNase A | 159         | 162                                        | 2.4                     |
|                  |                    | Pn16        | 208                                        | 3.1                     |
| *M. flavus*      | Complete           | 159         | 492                                        | 12.3                    |
| \[^3H\]seryl-tRNA^Ser^ | Pn16            |             | 130                                        | 3.2                     |
|                  | No MurM            | None        | 209                                        | 5.2                     |
|                  | No Lipid II        | 159         | 145                                        | 3.7                     |
|                  |                    | Pn16        | 156                                        | 3.9                     |
|                  | Complete + 0.1 mg·ml^−1^ RNase A | 159         | 335                                        | 8.4                     |
|                  |                    | Pn16        | 310                                        | 7.8                     |

enzyme and was recycled with SerRS. The NADH oxidation kinetics suggested consumption of 83% of the lipid II over the duration of the assay. This and the kinetics of the process in supplemental Fig. S4b leads us to tentatively suggest that there is molar equivalence between lipid II-Ser and tRNA^Ser^ generation.

## The True Amino Acid Substrates of MurM Are Aminoacyl-tRNAs

To establish that the actual MurM tRNA substrate was an acyl-tRNA, using *M. flavus* total tRNA, we precharged \[^3H\]alanyl-tRNA^Ala^ and tested the ability of MurM to transfer \[^3H\]alanine from the \[^3H\]alanyl-tRNA to lipid II, using the vastly enhanced solubility of lipid II in butan-1-ol relative to \[^3H\]alanyl-tRNA^Ala^ to separate the \[^3H\]acyl-tRNA substrate from the \[^3H\]acylated lipid II product (“Materials and Methods,” see under “Transfer of \[^3H\]-Amino-acid between \[^3H\]AcyI-tRNA and Lipid-linked Peptidoglycan Precursors”).

MurM~159~ catalyzed the transfer of \[^3H\]alanyl groups from \[^3H\]alanyl-tRNA^Ala^ to lipid II as evidenced by the incorporation of 78% of the \[^3H\] added to the assay into butan-1-ol-soluble material in the complete incubation (Table 1). Controls minus MurM~159~ or lipid II accumulated only 3.3 or 3.0% of the \[^3H\] added to the incubation into butan-1-ol-extractable products, demonstrating the essential requirement of these components for MurM~159~ activity (Table 1). To demonstrate the requirement of MurM~159~ for the tRNA portion of the \[^3H\]alanyl-tRNA^Ala^ substrate, complete reactions were treated with 0.1 mg·ml^−1^ RNase A, which reduced incorporation of \[^3H\] into lipid products to 2.4%, comparable with control values obtained without lipid II or MurM~159~ (Table 1). Similar results were obtained with MurM~Pn16~; however, the incorporation of radioactivity in the presence of all components was 8.8% that of the MurM~159~ incubations (Table 1).

These assays were conducted for 60 min. However, under initial rate conditions, accumulation of radiolabeled lipid-II-Ala product was linear for about 3 min, and within this time frame the relationship between the measured rate and protein concentration was linear.

## TABLE 2
### Activity of MurM~Pn16~ and MurM~159~ with \[^3H\]alanyl and \[^3H\]seryl-tRNA species from 159 and Pn16

Assays were performed at 10 μM lipid II and 0.45 μM pneumococcal \[^3H\]acyl-tRNA from Pn16 or 159 as indicated in the text. [MurM~Pn16~] and [MurM~159~] were 4.2 and 0.03 μM, respectively. Specific activities were calculated from initial rates of time courses, typically from the first 3 min of data.

| \[^3H\]Acyl-tRNA | MurM species | tRNA source | MurM specific activity ± S.E. |
|------------------|--------------|-------------|-------------------------------|
| \[^3H\]Alanyl-tRNA^Ala^ | Pn16       | Pn16        | 0.172 ± 0.025 nmol·min^−1^·mg(MurM)^−1^ |
|                      | 159         | Pn16        | 0.175 ± 0.022                |
|                      | 159         | Pn16        | 2.095 ± 0.285                |
|                      | 159         | Pn16        | 1.924 ± 0.466                |
| \[^3H\]Seryl-tRNA^Ser^ | Pn16       | Pn16        | 0.370 ± 0.038                |
|                      | 159         | Pn16        | 0.247 ± 0.025                |
|                      | 159         | Pn16        | 0.312 ± 0.055                |
|                      | 159         | Pn16        | 0.281 ± 0.096                |

### MurM~159~ and MurM~Pn16~ Activity with Crude Acyl-tRNAs Reflects the Branch Composition of the Stem Peptides of 159 and Pn16 Peptidoglycans

*M. flavus* \[^3H\]seryl-tRNA^Ser^ only barely supported the activity of MurM~159~ (Table 1). Therefore, to assay the relative abilities of MurM~Pn16~ and MurM~159~ to serylate or alanylate lipid II in vitro, crude tRNA~Pn16~ and tRNA~159~ were charged with \[^3H\]serine or \[^3H\]alanine, and the Pn16 and 159 \[^3H\]acyl-tRNAs were tested as substrates for acylation of lipid II by MurM~Pn16~ and MurM~159~, respectively.

Initial velocity measurements (*n* = 5) demonstrated that when MurM~159~ was challenged with lipid II and either \[^3H\]alanyl-tRNA^Ala^ or \[^3H\]seryl-tRNA^Ser^ from 159, the enzyme was 6.9-fold more active with \[^3H\]alanyl-tRNA^Ala^ than with \[^3H\]seryl-tRNA^Ser^ (Table 2; *p* < 0.0005 by Student’s *t* test). This result was entirely consistent with the observed prevalence of insertion of alanine in preference to serine into peptidoglycan branches by MurM~159~ in vivo. When these experiments were repeated with MurM~Pn16~ and either \[^3H\]alanyl-tRNA^Ala^ or \[^3H\]seryl-tRNA^Ser^ from Pn16, it was apparent that there had been a switch in specificity in that, unlike MurM~159~, MurM~Pn16~ was 2.2-fold more active with \[^3H\]seryl-tRNA^Ser^ than it was with \[^3H\]alanyl-tRNA^Ala^ (Table 2; *p* < 0.0005), which is entirely consistent with the preference of serine over alanine displayed by the MurM~Pn16~ in vivo. The intrinsic alanylation activity of MurM~159~ was also 11.3- and 5.2-fold greater than the lipid II-alanylation (*p* < 0.0005) and serylolation (*p* < 0.0005) activities of MurM~Pn16~ (Table 2). These data are consistent with the far higher levels of peptidoglycan branching in the 159 peptidoglycan compared with that of Pn16.

### The Competence of tRNA^Ala^ and tRNA^Ser^ from 159 and Pn16 as MurM Substrates Does Not Influence Branch Composition of the 159 and Pn16 Peptidoglycan

MurM~159~ supported lipid II alanylation to the greatest extent; however, it was not clear whether the tRNA pool in 159 was better suited to supporting MurM activity than the tRNA pool in Pn16. To address this issue, the sequencing of tRNA genes from 159 and Pn16 was undertaken to determine whether there were sequence differences in the tRNA genes that could explain the enhanced levels of MurM activity in 159.

It was only possible to sequence tRNA^Ala(1)^ and tRNA^Ala(4)^ from Pn16 and 159 (see “Materials and Methods”) which showed that

# Enzymology of the Aminoacyl Ligase MurM

tRNA<sup>Ala(1)</sup> and tRNA<sup>Ala(4)</sup> were identical in sequence and identical from either organism. All four tRNA<sup>Ser</sup> genes from Pn16 were sequenced and were found to be identical in sequence and arrangement to the tRNA<sup>Ser</sup> genes found in penicillin-sensitive pneumococci such as TIGR4 and R6 (25, 26). In the case of 159, tRNA<sup>Ser(4)</sup> and tRNA<sup>Ser(2)</sup> genes were identical to those in Pn16. The gene encoding 159 tRNA<sup>Ser(1)</sup>, although identical in sequence to that of Pn16, had, as indicated by BLAST analysis (28), been subject to an IS1167 transposon insertion immediately 5′ to its start increasing the size of the insert from 573 to 2043 bp. The remaining tRNA<sup>Ser(3)</sup> gene could not be isolated by PCR suggesting sequence rearrangement local to this gene.

It was thus clear that no complete conclusion regarding tRNA sequence identity and MurM activity could be drawn. Thus, to determine whether the *in vitro* activity of MurM<sub>Pn16</sub> and MurM<sub>159</sub> was either a function of their amino acid sequence divergence or of the relative ability of the tRNAs of 159 and Pn16 to serve as MurM<sub>159</sub> and MurM<sub>Pn16</sub> substrates, MurM<sub>Pn16</sub> was assayed with 159 <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> and 159 <sup>3</sup>H]seryl-tRNA<sup>Ser</sup>, whereas MurM<sub>159</sub> was assayed with Pn16 <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> and Pn16 <sup>3</sup>H]seryl-tRNA<sup>Ser</sup> (*n* = 5 for all measurements).

MurM<sub>159</sub> was equally active with Pn16 <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> and 159 <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> (Table 2; *p* < 0.3). Similarly, the activity of MurM<sub>Pn16</sub> with 159 <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> was equal to that with Pn16 <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> (Table 2; *p* < 0.45). Evidently, both 159 and Pn16 <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> could support enhanced MurM<sub>159</sub> activity, but 159 <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> could not elevate the specific activity of MurM<sub>Pn16</sub> to that of the 159 enzyme. This strongly suggested that *in vivo*, in 159 the enhanced insertion of alanine into peptidoglycan branches was a function of MurM<sub>159</sub> but not of the presence of an intrinsically efficient alanyl-tRNA<sup>Ala</sup> substrate of MurM in 159.

MurM<sub>159</sub> was equally active with Pn16 <sup>3</sup>H]seryl-tRNA<sup>Ser</sup> and 159 <sup>3</sup>H]seryl-tRNA<sup>Ser</sup> (Table 2; *p* < 0.25), and the activity of MurM<sub>Pn16</sub> with Pn16 <sup>3</sup>H]seryl-tRNA<sup>Ser</sup> was only 1.5 times that with 159 <sup>3</sup>H]seryl-tRNA<sup>Ser</sup> (Table 2; *p* < 0.0005). This suggested that enhanced insertion of serine over alanine in Pn16 peptidoglycan branches was a function of the acyl-tRNA specificity of MurM<sub>Pn16</sub> but not caused by a particularly competent seryl-tRNA<sup>Ser</sup> substrate of this MurM in Pn16.

the tRNA<sup>Ser</sup> pool was the same size in both 159 and Pn16 (0.549 ± 0.096 pmol of tRNA<sup>Ser</sup>/μg of total tRNA (*n* = 5 cultures) and 0.519 ± 0.099 pmol of tRNA<sup>Ser</sup>/μg of total tRNA (*n* = 3 cultures) respectively; *p* < 0.3). The tRNA<sup>Ala</sup>:tRNA<sup>Ser</sup> ratio in 159 (2.3) relative to Pn16 (1.2) was raised 2-fold, which could contribute to the enhanced levels of stem peptide branch alanylation in 159 relative to that in Pn16.

## Selection of <sup>3</sup>H]Acyl-tRNA Substrates for Kinetic Analysis of the Peptidoglycan Precursor Substrate Specificity of MurM

To establish the position of MurM within peptidoglycan synthesis and to determine what portions of the peptidoglycan precursor substrate were crucial for catalysis required a considerable quantity of acyl-tRNA. Therefore, *M. flavus* <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> was chosen for these experiments. However, it was necessary to demonstrate that this choice was appropriate, especially as *M. flavus* <sup>3</sup>H]seryl-tRNA<sup>Ser</sup> had proven to be essentially catalytically inert (Table 1). Therefore, the initial velocity of MurM<sub>159</sub> was determined over a variety of lipid II concentrations between 0 and 50 μM in the presence of 0.45 μM *M. flavus* <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup>. The experiment was then repeated with 0.45 μM <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub>, obtained by alanylation of a synthetic RNA sequence corresponding to the 76 nucleotides of *S. pneumoniae* tRNA<sup>Ala</sup> (see chemicals, Fig. 4, *a*, *i*, and *b*, *i*).

The velocities obtained at increasing lipid II concentrations in the presence of *M. flavus* <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> were related to those obtained in the presence of <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> by a straight line represented by a linear regression equation (*r*² = 0.9358) of *V*<sub>o</sub> (alanyl-tRNA<sup>Ala</sup><sub>UGC</sub>) = 0.9234 ± 0.0344 (*V*<sub>o</sub> [*M. flavus* Alanyl-tRNA<sup>Ala</sup>]) (Fig. 4*a*, *i*, inset), which demonstrated that MurM activities supported by *M. flavus* <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup> accurately reflected those supported by pure pneumococcal <sup>3</sup>H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub>. Indeed, both data sets delineated hyperbolic dependences of *V*<sub>o</sub> on lipid II concentration which, when fitted by nonlinear regression to the Michaelis-Menten equation ([S] = [lipid II]; Equation 1),

\[
V_{o} = \frac{V_{\text{max}}[S]}{K_{m}^{\text{app}} + [S]}
\]

yielded identical values for the *k*<sub>cat</sub><sup>app</sup> and *K*<sub>m</sub><sup>app</sup> of MurM<sub>159</sub> for lipid II (Table 3).

## Changes in the Ratio of tRNA<sup>Ala</sup> to tRNA<sup>Ser</sup> in 159 and Pn16 Could Influence the Stem Peptide Branch Composition of 159 and Pn16 Peptidoglycans

Although the competence of alanyl-tRNA<sup>Ala</sup> and seryl-tRNA<sup>Ser</sup> from 159 and Pn16 to perform as MurM<sub>159</sub> and MurM<sub>Pn16</sub> substrates was equivalent *in vitro*, *in vivo*, perturbations in the tRNA<sup>Ala</sup>:tRNA<sup>Ser</sup> ratio possibly influenced by the transposon insertion 5′ to 159 tRNA<sup>Ser</sup><sub>GCU</sub> could affect the substrate availability for MurM and so could impact on the peptidoglycan composition of 159 and Pn16. Therefore, pneumococcal AlaRS and SerRS were used to assay the relative concentration of tRNA<sup>Ala</sup> and tRNA<sup>Ser</sup> in Pn16 and 159 total tRNA. This revealed that the 159 tRNA<sup>Ala</sup> pool (1.363 ± 0.102 pmol of tRNA<sup>Ala</sup>/μg of total tRNA; *n* = 4 cultures) was 2.2-fold larger than the Pn16 tRNA<sup>Ala</sup> pool (0.5973 ± 0.107 pmol of tRNA<sup>Ala</sup>/μg of total tRNA; *n* = 3 cultures; *p* < 0.001), whereas

## Kinetic Analysis of MurM<sub>159</sub> Suggests That Stem Peptide Branches Are Added Exclusively during the Lipid-linked Stages of Cell Wall Synthesis

To determine whether there was any preference displayed by MurM<sub>159</sub> toward lipid I or lipid II, both peptidoglycan precursors were tested as MurM<sub>159</sub> substrates in the butan-1-ol extraction assay. MurM<sub>159</sub> displayed a hyperbolic dependence on lipid I (Fig. 4*a*, *ii*). The similarity in *K*<sup>app</sup><sub>m</sub> and the modest diminution in the lipid I *k*<sup>app</sup><sub>cat</sub> when compared with lipid II (Table 3) indicated that lipid I would be as significant as lipid II as a MurM<sub>159</sub> substrate.

To determine whether MurM<sub>159</sub> substrate specificity extended to the cytoplasmic peptidoglycan precursors, the MurM<sub>159</sub> and UDP-MurNAc peptide-dependent consumption

# Enzymology of the Aminoacyl Ligase MurM

(a): $V_o$ vs. [Lipid Substrate] | (b): Conversion
--- | ---
(i) | (i)
![Graph](https://via.placeholder.com/300x200?text=(i)) | ![Diagram](https://via.placeholder.com/300x200?text=(i))
(ii) | (ii)
![Graph](https://via.placeholder.com/300x200?text=(ii)) | ![Diagram](https://via.placeholder.com/300x200?text=(ii))
(iii) | (iii)
![Graph](https://via.placeholder.com/300x200?text=(iii)) | ![Diagram](https://via.placeholder.com/300x200?text=(iii))
(iv) | (iv)
![Graph](https://via.placeholder.com/300x200?text=(iv)) | ![Diagram](https://via.placeholder.com/300x200?text=(iv))

of $M$. *flavus* ${}^{[3]}$H]alanyl-tRNA${}^{\text{Ala}}$, MurM${}_{159}$ activity was clearly detectable (supplemental Fig. S5). Comparison of $k_{\text{cat}}^{\text{app}}:K_m^{\text{app}}$ ratios (Table 3) suggested that UDP-MurNAcAEKAA was used 3300 times less efficiently than lipid II. This and the inability of UDP-MurNAcAEK to function as a MurM${}_{159}$ substrate (Table 3) suggests that stem peptide branches are added exclusively to the lipid-linked peptidoglycan precursors *in vivo*.

## Characterization of the Response of MurM${}_{159}$ to Peptidoglycan Precursors and Precursor Analogues

A Complete Five-amino Acid Stem Peptide Is Not Required for MurM${}_{159}$ Activity—To determine the importance of the stem peptide D-alanyl-D-alanine to the binding of peptidoglycan precursors to MurM${}_{159}$, we tested lipid I-AEK as a MurM${}_{159}$ substrate. MurM${}_{159}$ was able to alanylate lipid I-AEK in a MurM${}_{159}$-dependent manner. The relationship between MurM${}_{159}$ velocity and lipid I AEK was characterized by substrate inhibition at >40 $\mu$M lipid I-AEK (Fig. 4a, iii). Analysis of the data suggested that it could be fitted by nonlinear regression to Equation 2 ([S] = [lipid I-AEK] (29)),

$$
V_o = \frac{V_{\text{max}}^{\text{app}}[S]}{K_m^{\text{app}} + [S]\left(1 + [S]/K_s\right)}
$$

where $K_s$ is the binding constant for the lipid I-AEK in a catalytically unproductive mode. The similarity of the $k_{\text{cat}}^{\text{app}}:K_m^{\text{app}}$ ratio for both lipid I-AEK and lipid I, 0.46 and 0.21 s${}^{-1}\cdot$mm${}^{-1}$, respectively (Table 3), suggests that the D-alanyl-D-alanine terminus contributes little to productive binding of the stem peptide by MurM${}_{159}$. The lipid I-AEK substrate inhibition suggested that loss of the D-alanyl-D-alanine relaxed the constraints placed upon stem peptide binding permitting the formation of an inactive MurM${}_{159}$-stem peptide substrate complex.

## MurM${}_{159}$ Cannot Tolerate a Negatively Charged Carboxyl Proximal to Its Site of Aminoacylation of Lipid II—To determine what effect the introduction of a negatively charged car-

FIGURE 4. Kinetics of dependence of MurM${}_{159}$ activity on lipid substrates. Initial velocity ($V_o$) is in units of nmol lipid-${}^{[3]}$H]Ala$\cdot$min${}^{-1}\cdot$mg (MurM${}_{159}$)${}^{-1}$. Unless otherwise stated, all data were obtained with 0.45 $\mu$M $M$. *flavus* ${}^{[3]}$H]alanyl-tRNA${}^{\text{Ala}}$. $a$, $i$, lipid II dependence in the presence of ${}^{[3]}$H]alanyl-tRNA${}^{\text{Ala}}$ from $M$. *flavus* or synthetic ${}^{[3]}$H]alanyl-tRNA${}^{\text{Ala}}_{\text{UGC}}$. $V_o$ is plotted versus [lipid II]. $M$. *flavus* ${}^{[3]}$H]alanyl-tRNA${}^{\text{Ala}}$ data are represented by filled triangles. Data obtained with 0.45 $\mu$M synthetic ${}^{[3]}$H]alanyl-tRNA${}^{\text{Ala}}_{\text{UGC}}$ is shown by filled squares. [MurM${}_{159}$] was 21 nM. Data were fitted by nonlinear regression to Equation 1 in the text. Inset, $V_o$ at a given [lipid II] with ${}^{[3]}$H]alanyl-tRNA${}^{\text{Ala}}_{\text{UGC}}$ (■) are plotted against $V_o$ at the same [lipid II] with $M$. *flavus* ${}^{[3]}$H]alanyl-tRNA${}^{\text{Ala}}$ (▲). The data were fitted by linear regression yielding the equation, $V_o$ (Alanyl-tRNA${}^{\text{Ala}}_{\text{UGC}}$) = 0.9234 ± 0.0344($V_o$ [$M$. *flavus* alanyl-tRNA${}^{\text{Ala}}$]). $a$, $ii$, lipid I dependence, $V_o$ is plotted versus [lipid I]. [MurM${}_{159}$] was 21 nM. Data were fitted to Equation 1 in the text. $a$, $iii$, lipid I-AEK dependence, $V_o$ is plotted versus [lipid I-AEK]. [MurM${}_{159}$] was 33 nM. Data were fitted to Equation 2 in the text. $a$, $iv$, lipid II-Ala dependence, $V_o$ is plotted versus [lipid II-Ala]. [MurM${}_{159}$] was 100 nM. Data were fitted to Equation 1 in the text. $b$, $i$–$iv$, lipid substrates conversions. Panels show the lipid substrates and products that participate in and result from alanylation of lipid II ($b$, $i$), lipid I ($b$, $ii$), lipid I-AEK ($b$, $iii$), and lipid II-Ala ($b$, $iv$).

of ${}^{[3]}$H]alanyl-tRNA${}^{\text{Ala}}$ was measured as a function of time-dependent loss of trichloroacetic acid-precipitable radioactivity. Despite the slight loss of label due to chemical deacylation

# Enzymology of the Aminoacyl Ligase MurM

## TABLE 3
Kinetics of utilization of peptidoglycan precursor and peptidoglycan precursor derivatives by MurM₁₅₉

All assays were performed for 1 min at 0.45 μM [³H]alanyl-tRNA<sup>Ala</sup>. All assays with lipid derivatives followed the butan-1-ol extraction method (see under “Materials and Methods,” “Transfer of <sup>3</sup>H-Amino-acid between [³H]Acyl-tRNA and Lipid-linked Peptidoglycan Precursors”), all assays with UDP-MurNAc substrates used the trichloroacetic acid precipitation procedure (see under “Materials and Methods,” “Transfer of <sup>3</sup>H-Amino-acid between [³H]Acyl-tRNA and UDP-linked Peptidoglycan Precursors”). Lipid I-AEK kinetic constants were determined according to Equation 2; all other kinetic constants were determined according to Equation 1 (see text). Fitting data to both equations was performed by nonlinear regression. NA means not applicable.

| Peptidoglycan precursor substrate | [Substrate] range | [³H]Alanyl-tRNA<sup>Ala</sup> source | [MurM₁₅₉]<sub>assay</sub> | $k_{\text{cat}}^{\text{app}}$ | $K_{m}^{\text{app}}$ | $K_{s}$ | $k_{\text{cat}}^{\text{app}} \cdot K_{m}^{\text{app}}$ |
|-----------------------------------|-------------------|-------------------------------------|--------------------------|----------------------------|------------------------|----------------|---------------------------------------|
| Lipid II                          | 0 to 50 μM       | M. flavus                         | 0.021                    | $0.033 \pm 0.0039$         | $34.78 \pm 7.29$      | NA             | 0.948                               |
| Lipid II                          | 0 to 50 μM       | Synthetic                         | 0.021                    | $0.032 \pm 0.0042$         | $37.45 \pm 8.59$      | NA             | 0.854                               |
| Lipid II-Ala                      | 0 to 200 μM      | M. flavus                         | 0.100                    | $0.0031 \pm 0.002$         | $236.8 \pm 153.3$     | NA             | 0.0131                              |
| Lipid II(DAP)                     | 10 μM, 100 μM    | M. flavus                         | 0.028                    | No activity                | NA                     | NA             | N/A                                 |
| Lipid I                           | 0 to 50 μM       | M. flavus                         | 0.021                    | $0.017 \pm 0.002$         | $37.37 \pm 6.99$      | NA             | 0.205                               |
| Lipid I-AEK                       | 0 to 80 μM       | M. flavus                         | 0.033                    | $0.0157 \pm 0.0037$        | $31.03 \pm 6.33$      | $51.98 \pm 5.57$ | 0.460                               |
| UDP-MurNAcAEKAA                   | 0 to 4.54 mM     | M. flavus                         | 1.130                    | $7.02 \cdot 10^{-4} \pm 2.1 \cdot 10^{-5}$ | $2445 \pm 1499$ | NA             | $2.87 \cdot 10^{-4}$               |
| UDP-MurNAcAEK                     | 100 μM           | M. flavus                         | 0.576                    | No activity                | NA                     | NA             | NA                                  |

boxyl would have on the activity of MurM₁₅₉, we synthesized lipid II(DAP), where DAP is a derivative of lysine that has been carboxylated on the ε-carbon atom to which the side chain ε-amino group is attached. Attempts to use lipid II(DAP) as a substrate for MurM₁₅₉ failed completely (Table 3), suggesting that the active site of MurM₁₅₉ could not tolerate a negative charge proximal to the lysine ε-amino group that undergoes alanylation by MurM₁₅₉.

**MurM₁₅₉ Can Alanylate Its Product, Lipid II-Ala—Lipid II-Ala is the product of MurM₁₅₉. We therefore initially tested lipid II-Ala for MurM₁₅₉ product inhibition. However, to our surprise, MurM₁₅₉ transferred [³H]alanine from *M. flavus* [³H]alanyl-tRNA<sup>Ala</sup> to lipid II-Ala. The relationship between $V_{o}$ and lipid II-Ala was hyperbolic (Fig. 4a, iv) and was fitted by nonlinear regression to Equation 1. Comparing $k_{\text{cat}}^{\text{app}}/K_{m}^{\text{app}}$ for lipid II-Ala and lipid II (Table 3) suggested that the extra alanylation of the lipid II substrate reduced the catalytic efficiency of MurM₁₅₉ 72-fold relative to lipid II.**

### Characterization of the Dependence of MurM₁₅₉ on [³H]Alanyl-tRNA<sup>Ala</sup>

**tRNA Secondary Modifications Are Not Required for Recognition by MurM—To study the interactions of the alanyl-tRNA<sup>Ala</sup> with MurM₁₅₉, we used pure pneumococcal tRNA<sup>Ala</sup> (UGC isoacceptor; tRNA<sup>Ala</sup><sub>UGC</sub>), synthesized with a 5′-phosphate (Fig. 5a, ii) but otherwise devoid of any secondary modifications to its bases. Analysis of the data in Tables 2 and 4 and preliminary time course experiments of lipid II-alanylation by MurM₁₅₉, with [³H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> or crude 159 [³H]alanyl-tRNA<sup>Ala</sup>, revealed that the pure tRNA substrate supported catalysis 8-fold more rapidly than the crude 159 tRNA substrate. These results indicated that secondary modifications were not important for the recognition by MurM₁₅₉ of alanyl-tRNA<sup>Ala</sup>.**

**The Anticodon and Variable and D-loops and Stems of the [³H]Alanyl-tRNA<sup>Ala</sup> Are Not Required for Substrate Recognition by MurM₁₅₉—To determine the regions of the tRNA important for MurM₁₅₉ recognition, a 5′-phosphorylated 35-nucleotide minihelix of the tRNA<sup>Ala</sup><sub>UGC</sub> isoacceptor comprising the (nucleotides in full-length tRNA<sup>Ala</sup><sub>UGC</sub>) 3′-CCA aminoacylation site (74–76), acceptor stem (73–66 and 1–7), TΨC loop (54–60), and stem (61–65 and 53–49) was made (Fig. 5b, ii) and was chargeable with [³H]alanine by AlaRS.**

Pilot experiments showed the [³H]alanyl-RNA minihelix supported MurM₁₅₉ activity. To further characterize these interactions, we compared the kinetics of dependence of MurM₁₅₉ activity on [³H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> with those on the [³H]alanyl-RNA minihelix (Table 4). Both relationships were hyperbolic (Fig. 5, $a$, $i$, and $b$, $i$). However, MurM₁₅₉ concentrations in these assays were significant compared with those of the alanyl-RNAs (Table 4).

To obtain the kinetic constants that characterized the dependence of MurM₁₅₉ on its alanylated RNA substrates, the data were fitted by nonlinear regression to Equation 3 below (29, 30), which accounts for [MurM₁₅₉] ([E]).

$$
V_{o} = \frac{V_{\text{max}}^{\text{app}}}{2 \cdot [E]} \cdot \left\{[E] + [S] + K_{m}^{\text{app}} - \sqrt{([E] + [S] + K_{m}^{\text{app}})^{2} - (4 \cdot [E] \cdot [S])}\right\}
$$

(Eq. 3)

The MurM₁₅₉ $k_{\text{cat}}^{\text{app}}:K_{m}^{\text{app}}$ ratios for [³H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> and [³H]alanyl-RNA minihelix (Table 4) were identical ($0.101$ and $0.099$ s⁻¹·μM⁻¹, respectively), indicating that both RNA substrates were catalytically equivalent. Thus, MurM₁₅₉:tRNA substrate recognition was likely to be limited to the alanyl-acceptor stem, the TΨC loop, and the TΨC stem and did not involve the anticodon, D, or variable loops and stems.

### Comparison of the Dependences of MurM₁₅₉ and MurM<sub>Pn16</sub> on [³H]Alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> and Lipid II

MurM<sub>Pn16</sub> was considerably less active than MurM₁₅₉ in vivo and in vitro. To characterize this discrepancy kinetically, the dependence of MurM<sub>Pn16</sub> on lipid II and [³H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> was analyzed (Table 4). The catalytic efficiency ($k_{\text{cat}}^{\text{app}}:K_{m}^{\text{app}}$) of MurM<sub>Pn16</sub> for lipid II was 72.3-fold lower than that for the 159 enzyme (Table 4), whereas the $k_{\text{cat}}^{\text{app}}:K_{m}^{\text{app}}$ of MurM<sub>Pn16</sub> for [³H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> was 46-fold lower than the corresponding value for the 159 enzyme (Table 4). These data are consistent with the performance of MurM<sub>Pn16</sub> and MurM₁₅₉ in vivo in Pn16 and 159.

## DISCUSSION

Peptide bond formation is crucial to the synthesis of the peptidoglycan, which involves three of the four commonly used

Enzymology of the Aminoacyl Ligase MurM

(ai)
![Graph](graph1.png)

(aii)
![Structure Diagram](structure_diagram1.png)

(aiii)
![RNA Structure](rna_structure1.png)

(bi)
![Graph](graph2.png)

(bii)
![Structure Diagram](structure_diagram2.png)

(biii)
![RNA Structure](rna_structure2.png)

FIGURE 5. Kinetics of dependence of $MurM_{159}$ activity on RNA substrates. $V_o$ is in nmol lipid II-$[^3H]$Ala·min${}^{-1}$·mg $(MurM_{159})^{-1}$. $a$, $i–iii$, alanyl-tRNA${}^{Ala}}_{UGC}$: $a$, $i$, dependence of $MurM_{159}$ on $[^3H]$alanyl-tRNA${}^{Ala}}_{UGC}$. $V_o$ is plotted versus $[^3H]$alanyl-tRNA${}^{Ala}}_{UGC}$. The [lipid II] was 10 $\mu$M, and $[MurM_{159}]$ was 67.2 nM. Data were fitted to Equation 3 in the text. $a$, $ii$, secondary structure of the pneumococcal tRNA${}^{Ala}$ UGC isoacceptor. Sequence was determined from the gene sequence determined under “Results” and was synthesized as an oligonucleotide for use as a MurM substrate. The anticodon and all stems and loops are labeled, and the G3:U70 base pair in the acceptor stem that is crucial for recognition by AlaRS is highlighted in white against gray (*). $a$, $iii$, tertiary structure of the pneumococcal tRNA${}^{Ala}$ UGC isoacceptor. A model of tRNA${}^{Ala}$ based on that of tRNA${}^{Phe}$ is shown, with the major structural elements labeled. $b$, $i–iii$, alanyl-minihelix. $b$, $i$, dependence of $MurM_{159}$ on $[^3H]$alanyl-minihelix]. $V_o$ is plotted versus $[^3H]$alanyl-minihelix]. The [lipid II] was 10 $\mu$M, and $[MurM_{159}]$ was 16.6 nM. Data were fitted to Equation 3 in the text. $b$, $ii$, secondary structure of the minihelix derivative of the pneumococcal tRNA${}^{Ala}$ UGC isoacceptor. Sequence of the acceptor stem and T$\psi$C loop included the contiguous sequence of nucleotides 1–7 and 49–76 of the full-length tRNA${}^{Ala}$ UGC. The G3:U70 base pair in the acceptor stem crucial for recognition by AlaRS is highlighted in white against gray (*). $b$, $iii$, tertiary structure of the minihelix derivative of the pneumococcal tRNA${}^{Ala}$ UGC isoacceptor. A model of the tertiary structure of the portion of tRNA${}^{Ala}$ recapitulated by the minihelix based on the structure of tRNA${}^{Phe}$ is shown, with the major structural elements labeled accordingly.

TABLE 4  
Comparison of the substrate dependencies of $MurM_{Pn16}$ and $MurM_{159}$  

All assays were performed for 1 min. All assays followed the butan-1-ol extraction method. Lipid II kinetic constants were determined according to Equation 1, and all other kinetic constants were determined according to Equation 3 to account for the relatively high $[MurM]:[^3H]$alanyl-(t)RNA substrate] ratio (see text). Fitting data to both equations was performed by nonlinear regression.

| MurM | Varied          | $[MurM]$ | [Substrate]     | Co-substrate       | $K_m^{\text{app}}$ | $k_{\text{app}}$ | $k_{\text{cat}}^{\text{app}}:K_m^{\text{app}}$ |
|------|-----------------|----------|------------------|--------------------|---------------------|-------------------|-----------------------------------------------|
|      | substrate       |          | range            | concentration      | $\mu M$             | $s^{-1}$          | $s^{-1}\cdot\mu M^{-1}$                       |
| Pn16 | Lipid II        | 450      | $nM$             | $0.45 \mu M$ $M.$flavus $[^3H]$alanyl-tRNA${}^{Ala}$ | $236.8 \pm 153.3$ | $0.0031 \pm 0.0017$ | $0.0131 \times 10^{-3}$                      |
|      | $[^3H]$Alanyl-tRNA${}^{Ala}}_{UGC}$ | 743      | $\mu M$            | $10 \mu M$ lipid II | $0.589 \pm 0.038$ | $0.0013 \pm 0.0003$ | $0.0022$                                      |
| 159  | Lipid II        | 21       | $0–50$           | $0.45 \mu M$ $M.$flavus $[^3H]$alanyl-tRNA${}^{Ala}$ | $34.78 \pm 7.29$  | $0.033 \pm 0.0039$   | $0.948 \times 10^{-3}$                      |
|      | $[^3H]$alanyl-tRNA${}^{Ala}}_{UGC}$ | 67.2     | $0–0.5$           | $10 \mu M$ lipid II | $0.209 \pm 0.007$ | $0.021 \pm 0.002$    | $0.101$                                       |
|      | $[^3H]$Ala-RNA minihelix | 16.6     | $0–0.3$           | $10 \mu M$ lipid II | $0.756 \pm 0.048$ | $0.075 \pm 0.023$    | $0.099$                                       |

biochemical mechanisms of amino acid carboxyl activation. The stem peptide is constructed from amino acids that are activated by esterification to phosphate (31); the stem peptides are cross-linked after activation by esterification to a highly reactive serine within a penicillin-binding protein transpeptidase (32), or inter-stem peptide branches are built up by condensation of L-amino acids activated by esterification to their cognate tRNA (5–9, 33–36). It is with this latter class of reactions that this paper is concerned.

S. pneumoniae, probably because of the plasticity of its genome is unique because of the strain-dependent and highly non-uniform chemical nature and abundance of dipeptide

# Enzymology of the Aminoacyl Ligase MurM

branches in its peptidoglycan (12, 14), as exemplified here with strains Pn16 and 159.

Sequence similarity to *S. aureus fmhb* (*femX*) led Filipe and Tomasz (10) to insertionally disrupt the pneumococcal *murM* sequence. The resulting peptidoglycan phenotype identified the *murM* gene as being responsible for the addition of the first amino acid of the pneumococcal stem peptide branch (10). We therefore overexpressed and purified MurM<sub>Pn16</sub> and MurM<sub>159</sub> to correlate their enzymological properties with their contribution to the structure of pneumococcal peptidoglycan for the first time. The purification of both enzymes was complicated because their solubility depended upon maintenance of a high ionic strength. This feature is also shared with the MurM homologue *Enterococcus faecalis* BppA1 (33).

A further hurdle in the study of MurM was the generation of peptidoglycan precursors. Although the synthesis of these molecules in vitro had been described previously, NADPH oxidase activity of MurB, coupled with its sensitivity to substrate inhibition by NADPH, required use of very high concentrations of NADPH and MurB for synthesis of UDP-MurNAc peptides by these methods (17, 18, 37–40). Here, however, by recycling the NADP<sup>+</sup> co-factor with isocitrate dehydrogenase, we maintained sub-inhibitory NADPH concentrations that were ~1% of the final yield of UDP-MurNAc peptide. Similarly, by recycling ADP produced by MurC to -F (Fig. 1), with pyruvate kinase to reform ATP, UDP-MurNAc peptide purification was simplified because, although ADP co-purified with many UDP-MurNAc peptides, ATP eluted after all of them.<sup>3</sup>

Our results conclusively showed for the first time that MurM<sub>159</sub> and, by inference, MurM<sub>Pn16</sub> supported dipeptide branch synthesis by transferring either alanyl or seryl residues from alanyl-tRNA<sup>Ala</sup> or seryl-tRNA<sup>Ser</sup> to the ε-amino group of the lipid II stem peptide lysine (Figs. 3, S3, and S4). This is consistent with inter-stem branch synthesis in other Gram-positive organisms where L-amino acids or glycine are activated by esterification to tRNA prior to insertion into the stem peptide (5–9, 33–36). The work here is an advance on previous analyses of the MurM homologues *E. faecalis* BppA1 and *S. aureus* FemX, -A, or -B. In these cases, no demonstration of the transfer of aminoacyl groups from a pre-acylated tRNA to a peptidoglycan precursor was achieved, and so no kinetic analyses of acyl-tRNA usage could be performed (8, 33), as was done here. Furthermore, the BppA1 results were based upon utilization of UDP-MurNAc pentapeptide as the stem peptide substrate, where, as the authors of this work concede, it is more likely that the true BppA1 substrate was lipid I or lipid II (33). Unlike the studies of the *E. faecalis* and *Staphylococcus aureus* enzymes, our reconstruction of peptidoglycan intermediate synthesis in vitro allowed us to probe in detail the interaction of MurM with its lipid substrates (Fig. 4; Table 3) (8, 33). Finally, the ES-MS fragmentation results presented here are the first direct confirmation that the site of aminoacylation of a lipid peptidoglycan precursor by this class of enzymes is the ε-amino group of the lipid II lysine (Fig. S3). This contrasts with previous studies of the *S. aureus* FemXAB (8), where the site of glycylation of lipid II was not identified.

The markedly different branching phenotype displayed by Pn16 and 159, corresponding to the in vivo activity of MurM

(Fig. 2), was reflected in vitro by MurM<sub>159</sub> activity with pneumococcal alanyl-tRNA<sup>Ala</sup>, which was considerably greater than with seryl-tRNA<sup>Ser</sup> and greater than any acyl-tRNA-supported MurM<sub>Pn16</sub> activity (Table 2). This was consistent with the almost entire alanylation of stem peptides in 159 (Fig. 2). For MurM<sub>Pn16</sub>, seryl-tRNA<sup>Ser</sup> was a better substrate than alanyl-tRNA<sup>Ala</sup>, consistent with the residual amounts of serylation and alanylation (where the serylation was greater than alanylation) of stem peptides in Pn16 (Fig. 2; Table 2). The markedly differing properties of MurM<sub>Pn16</sub> and MurM<sub>159</sub> were also reflected in their kinetics of substrate dependence, where consistent with their in vivo activity, both lipid II and alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> were used considerably less efficiently by the Pn16 enzyme compared with its 159 counterpart (Table 4).

Involvement of both MurM and pneumococcal tRNA in the synthesis of dipeptide branches suggested that the tRNA or the MurM protein had been modified to reflect the differing nature of the peptidoglycan in Pn16 and 159. Analysis of pneumococcal genomes coupled with our own sequencing of tRNA genes of 159 and Pn16 failed to reveal any sequence differences that could account for the elevated levels of peptidoglycan branching in 159. Furthermore, equal amounts of [³H]seryl-tRNA<sup>Ser</sup> and [³H]alanyl-tRNA<sup>Ala</sup> from either 159 or Pn16 were equally competent at supporting MurM<sub>159</sub> and MurM<sub>Pn16</sub> activity, suggesting that tRNA<sup>Ala</sup> or tRNA<sup>Ser</sup> from 159 was as capable of supporting enhanced peptidoglycan branching as tRNA<sup>Ala</sup> or tRNA<sup>Ser</sup> from Pn16.

Nevertheless, one of the 159 tRNA<sup>Ser</sup><sub>GCU</sub> genes was directly adjacent to the 3′ end of an IS1167 transposon insertion that was absent in Pn16. tRNA gene rearrangement by transposon insertion is well known, and it is conceivable that this interfered with transcription of this 159 tRNA<sup>Ser</sup><sub>GCU</sub> gene, elevating the tRNA<sup>Ala</sup>:tRNA<sup>Ser</sup> ratio in 159 relative to Pn16 (41, 42). This suggests that alterations in tRNA expression as opposed to the tRNA itself may contribute to the high levels of alanylation of peptidoglycan of 159 relative to those in the Pn16 peptidoglycan.

Homologous recombination of streptococcal genes generates a myriad of gene products of variable sequence that has allowed the pneumococcus and other streptococci to develop resistance to major classes of antibiotic as evidenced by sequence differences that have generated β-lactam-resistant penicillin-binding proteins and sulfonamide-resistant dihydropteroate synthase variants (43, 44). This type of genetic rearrangement has led to a series of families of *murM* alleles, of which *murM*<sub>Pn16</sub> is 99% identical to the *murMA* class of *murM* alleles typified by *S. pneumoniae* R6 *murM* and of which *murM*<sub>159</sub> is 100% identical to the *murMB1* allele typified by Hungarian isolate Hun663 (supplemental Fig. S1b) (12, 25). Such strain-dependent sequence variation of MurM is believed to underpin variations in MurM activity and peptidoglycan composition (12, 14) similar to those observed here.

What does our data reveal about tRNA recognition by MurM? MurM<sub>Pn16</sub> and MurM<sub>159</sub> can utilize alanyl-tRNA<sup>Ala</sup> and seryl-tRNA<sup>Ser</sup> (Table 2). Therefore, anticodon recognition is not important, which is also so for AlaRS and SerRS (45, 46). This correlates with the [³H]alanyl-minihelix data, which suggested that no more than the acceptor stem and the TΨC loop

of tRNA<sup>Ala</sup> were required for interaction with MurM<sub>159</sub> (Table 4; Fig. 5b, *i–iii*). Similar conclusions have been drawn regarding the *Lactobacillus* FemX (47).

AlaRS recognition of tRNA<sup>Ala</sup> depends on the highly conserved mispaired G:U at position 3:70 of the acceptor stem of the tRNA (Fig. 5a, *ii*) (45). Although it is possible that this recognition element is targeted by pneumococcal MurM, very recent work by Villet *et al.* (48) showed that *Lactobacillus* FemX does not depend upon the G3:U70 base pair for alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> substrate recognition, but instead it depends principally upon the first 2 bp in the acceptor stem of tRNA<sup>Ala</sup><sub>UGC</sub> (G2:C71 and G1:C72; Fig. 5a, *ii*). All of the *S.* pneumoniae tRNA<sup>Ser</sup> and tRNA<sup>Ala</sup><sub>UGC</sub> species have acceptor stems whose first 2 bp are G2:C71 and G1:C72 (25, 26 and data reported herein). Additionally, this acceptor stem motif is shared by pneumococcal tRNA isoacceptors for (amino acid/anticodon) Y/GUA, R/CCG, I/GAU, F/GAA, M/CAU, D/GUC, and V/UAC (25, 26, 49). Thus, if MurM shares the *Lactobacillus* FemX mode of tRNA recognition (48), then discrimination between seryl and alanyl addition and limitation of MurM activity to just alanylation and serylation of the stem peptide largely results from recognition of the aminoacyl moiety of the acyl-tRNA substrate by this enzyme.

Comparison of the rates of MurM<sub>159</sub> supported by crude 159 [³H]alanyl-tRNA<sup>Ala</sup> and synthetic [³H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> suggested that the latter was an 8-fold better substrate. This result highlighted the redundancy of tRNA secondary modifications in MurM catalysis. Additionally, these data suggested that the crude pneumococcal tRNA pool contains components that inhibit utilization of alanyl-tRNA<sup>Ala</sup> by MurM<sub>159</sub>. These components could well be unacylated tRNA species with the G2:C71 and G1:C72 motif (above), including the tRNA<sup>Ser</sup> isoacceptors which implies a subtlety in the interaction of MurM with the pneumococcal tRNA pool, which might impact upon the final composition of the cell wall.

What do the MurM alanyl-tRNA<sub>UGC</sub> kinetics reveal about the relationship between peptidoglycan and protein synthesis? *K*<sub>*m*</sub><sup>app</sup> values of MurM<sub>159</sub> and MurM<sub>Pn16</sub> for [³H]alanyl-tRNA<sup>Ala</sup><sub>UGC</sub> were considerably higher (Table 4) than the 6.2 nM *K*<sub>D</sub> of EF-TU for alanyl-tRNA<sup>Ala</sup> (50). Thus there is potential for considerable competition between ribosomal protein and cell wall synthesis for alanyl-tRNA<sup>Ala</sup> particularly where amino acids and therefore aminoacyl-tRNA species are in short supply and where, necessarily, alanyl-tRNA<sup>Ala</sup> will be drawn into protein synthesis. In the staphylococci, this issue is resolved by mutations in the TΨC and D-loops of two specialized UCC isoacceptor tRNA<sup>Gly</sup> species that bar involvement of these tRNA<sup>Gly</sup><sub>UCC</sub> species in protein synthesis, thus reserving a pool of glycyl-tRNA<sup>Gly</sup> for peptidoglycan synthesis (51–54). In the case of the lactobacilli, maintenance of cell wall branching despite the demands of protein synthesis for alanyl-tRNA<sup>Ala</sup> probably depends upon the high *k*<sub>cat</sub> of FemX (47) relative to that of MurM.

*S.* pneumoniae is distinguishable from *Lactobacillus viridescens* and *S.* aureus because it is not essential for pneumococcal peptidoglycan to be entirely composed of branched stem peptides, nor even to contain any at all (2, 4, 10, 55). It is therefore likely that the pneumococcus is more tolerant to loss of acyl-tRNA from peptidoglycan to protein synthesis than *L.* viridescens and *S.* aureus. Nevertheless, it should be noted that such diminution of pneumococcal stem peptide branching may well adversely impact upon the attachment of important cell surface protein adhesins and virulence determinants ligated to branched stem peptides via sortases prior to peptidoglycan cross-linking (56).

What can be concluded from the *in vitro* peptidoglycan intermediate substrate specificity of MurM? MurM probably utilizes lipid I and/or II *in vivo* as suggested by Filipe *et al.* (14). This is inconsistent with exclusive specificity of *S.* aureus FemX for lipid II (8). Usage of UDP-MurNAcAEKAA by MurM<sub>159</sub> was insignificant (supplemental Fig. S5; Table 3) in comparison with that of *L.* viridescens FemX, which utilizes only this substrate (5).

The presentation of the preferred peptidoglycan precursor substrates to MurM and *L.* viridescens FemX is likely to be radically different. In the case of MurM, its lipid II substrate is tethered to a phospholipid bilayer by an undecaprenyl tail, which is therefore unlikely to interact with the enzyme, whereas the rest of the molecule, including the stem peptide (pyrophosphoryl-(GlcNAc)-MurNAcAEKAA), extends from the membrane surface (57) into the active site of MurM. In contrast, the stem peptide of the soluble peptidoglycan substrate of the *L.* viridescens FemX is tethered to UDP, which is available for and is indeed essential for interaction with this enzyme (58, 59). The vastly differing properties and availabilities of the groups appended to the MurNAc-stem peptide of UDP-MurNAcAEKAA and lipid I or II probably have precluded the evolution of a FemX or MurM active site that can bind both substrates and might dictate whether peptidoglycan stem peptide branching is a cytoplasmic or a membrane-bound process.

Usage by MurM of either lipid I or lipid II *in vivo* suggests the N-acetylglucosaminyl group of lipid II was not essential for recognition of the lipid precursor by MurM (Fig. 4; Table 3). It also indicated that *in vivo* MurG probably tolerates lipid I species with an acylation of the stem peptide lysine ε-NH<sub>2</sub> group to avoid wasteful accumulation of lipid I precursors. This is likely because *M.* flavus MurG can even process lipid I modified by the addition of bulky pyrene-based fluorophores to the stem peptide lysine (19).

To extend our substrate specificity studies, we examined the consequences of carboxylation of the ε-carbon atom of the stem peptide lysine on MurM<sub>159</sub> catalysis. MurM<sub>159</sub> was entirely intolerant of this modification as judged by the observation that lipid II(DAP) was not a substrate. This has also been observed for *S.* aureus FemA and *L.* viridescens FemX (8, 9). The lysine-containing stem peptide-specific members of the FemXAB family contain a catalytic aspartate (47), which is similarly conserved in MurM (Asp-107; Fig. S1a). It is possible that a substrate (DAP) carboxyl adjacent to the ε-NH<sub>2</sub> undergoing acylation, would perturb the environment of the catalytic carboxyl of Asp-107 of MurM sufficiently to prevent catalysis. In this context, it is interesting to note that *Streptomyces coelicolor* FemX and VanK, which append glycine onto an LL-DAP-containing stem peptide, do not possess conserved active site aspartates analogous to *L.* viridescens FemX Asp-109 or *S.* pneumoniae MurM Asp-107 (60, 61).

Our results indicated that the stem peptide C-terminal D-alanyl-D-alanine was not essential for MurM<sub>159</sub> catalysis (Fig. 4a,

# Enzymology of the Aminoacyl Ligase MurM

These results contrasted starkly with those of Maillard *et al.* (58) who found a stem peptide D-alanyl-D-alanine C terminus was required in *L. viridescens* FemX substrates. In this context, the lipid I-AEK substrate inhibition of MurM<sub>159</sub> might originate in a set of interactions such as those evident in the *L. viridescens* FemX:UDP-MurNAcAEKAA crystal structure (58), which is not a bound conformation of the substrate that can be catalytically active.

Surprisingly, we found that MurM<sub>159</sub> could append an alanyl group from \[^3H\]alanyl-tRNA<sup>Ala</sup> onto lipid II-Ala (Fig. 4a, iv). Although this was not evident from the ES-MS analysis of the products of MurM (Figs. 3 and supplemental Fig. S3), the kinetics of alanylation of lipid II-Ala were so poor in the absence of lipid II (Fig. 4a, iv; Table 3) than in its presence, it is unlikely that lipid II-Ala-Ala formation would have been detected by the experiments represented by supplemental Fig. S3 and Fig. 3.

The alanylation of lipid II-Ala by MurM is the activity of MurN *in vivo* (11). This showed that MurM is an enzymatic homologue of *S. aureus* FemA (8). A similar flexibility in the active site of *S. aureus* FemX also allows it under forcing conditions to add two glycine residues instead of one (8). Although clearly of mechanistic and evolutionary relevance, *in vivo*, mutational inactivation of MurN caused the branches of the stem peptides to be substituted by a single alanine or serine in *S. pneumoniae*, and consequently, the alanylation by MurM<sub>159</sub> of lipid II-Ala cannot be physiologically significant (11).

What relevance have our results for pneumococcal penicillin resistance? Clinically, MurM is essential for high level penicillin resistance (14–16). Our data demonstrate the central role played by tRNA in streptococcal cell wall synthesis, and link tRNA metabolism with β-lactam resistance. Interestingly, competition of peptidoglycan branching and ribosomal protein synthesis for acyl tRNA might influence the response of pneumococcal infection to β-lactam therapy, which in turn might be modified by ribosomally directed antibiotics such as erythromycin and chloramphenicol. Our results have revealed many of the important features of the substrates of MurM involved in binding and catalysis. Pursuit of the structural basis of these interactions and methods for their disruption to furnish novel antibiotic therapies is currently underway in our laboratory.

---

Acknowledgments—We thank Dr. E. Breukink (University of Utrecht, The Netherlands) for assistance with lipid I and II synthesis, Dr. M. Mystry (MRC Unit, University of Leicester, UK) for N-terminal sequencing, and S. Slade and H. Bird (Dept. of Biological Sciences, University of Warwick, UK) for mass spectrometric and DNA sequence analyses.

---

## REFERENCES

1. Bugg, T. D. H. (1999) in Comprehensive Natural Products Chemistry (Pinto, M., ed) Vol. 3, pp. 241–294, Elsevier, Oxford, UK
2. Garcia-Bustos, J., and Tomasz, A. (1990) Proc. Natl. Acad. Sci. U. S. A. 87, 5415–5419
3. Garcia-Bustos, J. F., Chait, B. T., and Tomasz, A. (1988) J. Bacteriol. 170, 2143–2147
4. Severin, A., and Tomasz, A. (1996) J. Bacteriol. 178, 168–174
5. Plapp, R., and Strominger, J. L. (1970) J. Biol. Chem. 245, 3667–3674
6. Matsuhashi, M., Dietrich, C. P., and Strominger, J. L. (1967) J. Biol. Chem. 242, 3191–3206
7. Petit, J.-F., Strominger, J. L., and Söll, D. (1968) J. Biol. Chem. 243, 757–767
8. Schneider, T., Senn, M. M., Berger-Bachii, B., Tossi, A., Sahl, H.-G., and Wiedmann, I. (2004) Mol. Microbiol. 53, 675–685
9. Hegde, S., and Schraeder, T. E. (2001) J. Biol. Chem. 276, 6998–7003
10. Filipe, S. R., and Tomasz, A. (2000) Proc. Natl. Acad. Sci. U. S. A. 97, 4891–4896
11. Filipe, S. R., Pinho, M. G., and Tomasz, A. (2000) J. Biol. Chem. 275, 27768–27774
12. Filipe, S. R., Severina, E., and Tomasz, A. (2000) J. Bacteriol. 182, 6798–6805
13. Chesnel, L., Carapito, R., Croizé, J., Dideberg, O., Vernet, T., and Zapun, A. (2005) Antimicrob. Agents Chemother. 49, 2895–2902
14. Filipe, S. R., Severina, E., and Tomasz, A. (2001) J. Biol. Chem. 276, 39618–39628
15. Barcus, V. A., Ghanekar, K., Yeo, M., Coffey, T. J., and Dowson, C. G. (1995) FEMS Microbiol. Lett. 126, 299–303
16. Smith, A. M., and Klugman, K. P. (2001) Antimicrob. Agents Chemother. 45, 2393–2396
17. Reddy, S. G., Waddell, S. T., Kuo, D. W., Wong, K. K., and Pompliano, D. L. (1999) J. Am. Chem. Soc. 121, 1175–1178
18. El Zoeiby, A., Sanschagrin, F., Havugimana, P. C., Garnier, A., and Levesque, R. C. (2001) FEMS Microbiol. Lett. 201, 229–235
19. Breukink, E., van Heusden, H. E., Vollmerhaus, P. J., Swiezewska, E., Brunner, L., Walker, S., Heck, A. J. R., and de Kruijff, B. (2003) J. Biol. Chem. 278, 19898–19903
20. Blewett, A. M., Lloyd, A. J., Echalier, A., Fulop, V., Dowson, C. G., Bugg, T. D. H., and Roper, D. I. (2004) Acta. Crystallogr. Sect. D Biol. Crystallogr. 60, 359–361
21. Webb, M. R. (1992) Proc. Natl. Acad. Sci. U. S. A. 89, 4884–4887
22. Lloyd, A. J., Brandish, P. E., Gilbey, A. M., and Bugg, T. D. H. (2003) J. Bacteriol. 186, 1747–1757
23. Whatmore, A. M., Barcus, V. A., and Dowson, C. G. (1999) J. Bacteriol. 181, 3144–3154
24. Garcia-Bustos, J. F., and Tomasz, A. (1987) J. Bacteriol. 169, 447–453
25. Hoskins, J., Alborn, W. E., Jr., Arnold, J., Blaszczak, L. C., Burgett, S., DeHoff, B. S., Estrem, S. T., Fritz, L., Fu, D. J., Fuller, W., Geringer, C., Gilmour, R., Glass, J. S., Khoja, H., Kraft, A. R., Lagace, R. E., LeBlanc, D. J., Lee, L. N., Lefkowitz, E. J., Lu, J., Matsushima, P., McAhren, S. M., McHenney, M., McLeaster, K., Mundy, C. W., Nicas, T. I., Norris, F. H., O’Gara, M., Peery, R. B., Robertson, G. T., Rockey, P., Sun, P. M., Winkler, M. E., Yang, Y., Young-Bellido, M., Zhao, G., Zook, C. A., Baltz, R. H., Jaskunas, S. R., Rosteck, P. R., Jr., Skatrud, P. L., and Glass, J. I. (2001) J. Bacteriol. 183, 5709–5717
26. Tettelin, H., Nelson, K. E., Paulsen, I. T., Eisen, J. A., Read, T. D., Peterson, S., Heidelberg, J., DeBoy, R. T., Haft, D. H., Dodson, R. J., Durkin, S., Gwinn, M., Kolonay, J. F., Nelson, W. C., Peterson, J. D., Umayam, L. A., White, O., Salzberg, S. L., Lewis, M. R., Radune, D., Holtzapple, E., Khouri, H., Wolf, A. M., Utterback, T. R., Hansen, C. L., McDonald, L. A., Feldblyum, T. V., Angiuoli, S., Dickinson, T., Hickey, E. K., Holt, I. E., Loftus, B. J., Yang, F., Smith, H. O., Venter, J. C., Dougherty, B. A., Morrison, D. A., Hollingshead, S. K., and Fraser, C. M. (2001) Science 293, 498–506
27. Lloyd, A. J., Thomann, H.-U., Ibba, M., and Söll, D. (1995) Nucleic Acids Res. 23, 2886–2892
28. Altschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990) J. Mol. Biol. 215, 403–410
29. Segel, I. H. (1975) Enzyme Kinetics: Behaviour and Analysis of Rapid Equilibrium and Steady-state Enzyme Systems, pp. 18–99, Wiley-Interscience, New York
30. Henderson, P. J. F. (1973) Biochem. J. 135, 101–107
31. El Zoeiby, A., Sanschagrin, F., and Levesque, R. C. (2003) Mol. Microbiol. 47, 1–12
32. Goffin, C., and Ghysen, J.-M. (2002) Microbiol. Mol. Biol. Rev. 66, 702–738
33. Bouhss, A., Josseaume, N., Allanic, D., Crouvoisier, M., Gutmann, L., Mainard, J.-L., Mengin-Lecreulx, D., van Heijenoort, J., and Arthur, M. (2001) J. Bacteriol. 183, 5122–5127
34. Linnet, P. E., Roberts, R., and Strominger, J. L. (1974) J. Biol. Chem. 249, 2497–2506

**6416 JOURNAL OF BIOLOGICAL CHEMISTRY**
**VOLUME 283 • NUMBER 10 • MARCH 7, 2008**

35. Roberts, W. S. L., Strominger, J. L., and Söll, D. (1968) *J. Biol. Chem.* 243, 749–756
36. Roberts, W. S. L., Petit, J.-F., and Strominger, J. L. (1968) *J. Biol. Chem.* 243, 768–772
37. Dhalla, A. M., Yanchunas, J., Jr., Ho, H.-T., Falk, P. J., Villafranca, J. J., and Robertson, J. G. (1995) *Biochemistry* 34, 5390–5402
38. Sylvester, D. R., Alvarez, E., Patel, A., Ratnam, K., Kallender, H., and Wallis, N. G. (2001) *Biochem. J.* 355, 431–435
39. Benson, T. E., Walsh, C. T., and Massey, V. (1997) *Biochemistry* 36, 796–805
40. Nishida, S., Kurokawa, K., Matsuo, M., Sakamoto, S., Ueno, K., Kita, K., and Sekimizu, K. (2006) *J. Biol. Chem.* 281, 1714–1724
41. Williams, K. P. (2002) *Nucleic Acids Res.* 30, 866–875
42. Zhou, L., Hui, F. M., and Morrison, D. A. (1995) *Plasmid* 33, 127–138
43. Dowson, C. G., Coffey, T. J., and Spratt, B. G. (1994) *Trends Microbiol.* 2, 361–366
44. Jönsson, M., Ström, K., and Swedberg, G. (2003) *Microb. Drug Resist.* 9, 147–153
45. Ribas de Pouplana, L., Musier-Forsyth, K., and Schimmel, P. (2005) in *The Aminoacyl-tRNA Synthetases* (Ibba, M., Francklyn, C., and Cusack, S., eds) pp. 241–249, R. G. Landes Co., Austin, TX
46. Weygand-Durasevic, I., and Cusack, S. (2005) in *The Aminoacyl-tRNA Synthetases* (Ibba, M., Francklyn, C., and Cusack, S., eds) pp. 177–192, R. G. Landes Co., Austin, TX
47. Hegde, S. S., and Blanchard, J. S. (2003) *J. Biol. Chem.* 278, 22861–22867
48. Villet, R., Fonvielle, M., Busca, P., Chemama, M., Maillard, A. P., Hugonet, J.-E., Dubost, L., Marie, A., Josseaume, N., Mesnage, S., Mayer, C., Valéry, J.-M., Ethève-Quelquejeu, M., and Arthur, M. (2007) *Nucleic Acids Res.* 35, 6870–6883
49. Sprinzl, M., and Vassilenko, K. S. (2005) *Nucleic Acids Res.* 33, D139–D140
50. LaRiviere, F. J., Wolfson, A. D., and Uhlenbeck, O. C. (2001) *Science* 294, 165–168
51. Bumstead, R. M., Dahl, J. L., Söll, D., and Strominger, J. L. (1968) *J. Biol. Chem.* 243, 779–782
52. Stewart, T. S., Roberts, R. J., and Strominger, J. L. (1971) *Nature* 230, 36–38
53. Roberts, R. J. (1974) *J. Biol. Chem.* 249, 4787–4796
54. Roberts, R. J., Lovinger, G. G., Tamura, T., and Strominger, J. L. (1974) *J. Biol. Chem.* 249, 4781–4786
55. Schleifer, K. H., and Kandler, O. (1972) *Bacteriol. Rev.* 36, 407–477
56. Marraffini, L. A., DeDent, A. C., and Schneewind, O. (2006) *Microbiol. Mol. Rev.* 70, 192–221
57. Ganchev, D. N., Hasper, H. E., Breukink, E., and de Kruijff, B. (2006) *Biochemistry* 45, 6195–6202
58. Maillard, A. P., Biarrotte-Sorin, S., Villet, R., Mesnage, S., Bouhss, A., Sougakoff, W., Mayer, C., and Arthur, M. (2005) *J. Bacteriol.* 187, 3833–3838
59. Plapp, R., and Strominger, J. L. (1970) *J. Biol. Chem.* 245, 3675–3682
60. Hong, H.-J., Hutchings, M. I., Hill, L. M., and Buttner, M. J. (2006) *J. Biol. Chem.* 280, 13055–13061
61. Bentley, S. D., Chater, K. F., Cerdeno-Tarraga, A. M., Challis, G. L., Thompson, N. R., James, K. D., Harris, D. E., Quail, M. A., Kieser, H., Harper, D., Bateman, A., Brown, S., Chandra, G., Chen, C. W., Collins, M., Cronin, A., Fraser, A., Goble, A., Hidalgo, J., Hornsby, T., Howarth, S., Huang, C. H., Kieser, T., Larke, L., Murphy, L., Oliver, K., O’Neil, S., Rabbinowitsch, E., Rajandream, M. A., Rutherford, K., Rutter, S., Seeger, K., Saunders, D., Sharp, S., Squares, R., Squares, S., Taylor, K., Warren, T., Wietzorrek, A., Woodward, J., Barrell, B. G., Parkhill, J., and Hopwood, D. A. (2002) *Nature* 417, 141–147

Enzyme Catalysis and Regulation: Characterization of tRNA-dependent Peptide Bond Formation by MurM in the Synthesis of *Streptococcus pneumoniae* Peptidoglycan

Adrian J. Lloyd, Andrea M. Gilbey, Anne M. Blewett, Gianfranco De Pascale, Ahmed El Zoeiby, Roger C. Levesque, Anita C. Catherwood, Alexander Tomasz, Timothy D. H. Bugg, David I. Roper and Christopher G. Dowson

J. Biol. Chem. 2008, 283:6402-6417.  
doi: 10.1074/jbc.M708105200 originally published online December 12, 2007

---

Access the most updated version of this article at doi: [10.1074/jbc.M708105200](http://dx.doi.org/10.1074/jbc.M708105200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- [When this article is cited](#)
- [When a correction for this article is posted](#)

[Click here](#) to choose from all of JBC's e-mail alerts

Supplemental material:  
[http://www.jbc.org/content/suppl/2007/12/13/M708105200.DC1.html](http://www.jbc.org/content/suppl/2007/12/13/M708105200.DC1.html)

This article cites 58 references, 40 of which can be accessed free at  
[http://www.jbc.org/content/283/10/6402.full.html#ref-list-1](http://www.jbc.org/content/283/10/6402.full.html#ref-list-1)
