
# Protein Translocation Across the Bacterial Cytoplasmic Membrane

**Arnold J.M. Driessen and Nico Nouwen**

Department of Molecular Microbiology, Groningen Biomolecular Sciences and Biotechnology Institute and the Zernike Institute for Advanced Materials, University of Groningen, 9751 NN, Haren, The Netherlands; email: a.j.m.driessen@rug.nl, nico.nouwen@mpl.ird.fr

---

### Key Words

chaperone, membrane protein, proton motive force, SecA, SecY, translocase

---

### Abstract

About 25% to 30% of the bacterial proteins function in the cell envelope or outside of the cell. These proteins are synthesized in the cytosol, and the vast majority is recognized as a ribosome-bound nascent chain by the signal recognition particle (SRP) or by the secretion-dedicated chaperone SecB. Subsequently, they are targeted to the Sec translocase in the cytoplasmic membrane, a multimeric membrane protein complex composed of a highly conserved protein-conducting channel, SecYEG, and a peripherally bound ribosome or ATP-dependent motor protein SecA. The Sec translocase mediates the translocation of proteins across the membrane and the insertion of membrane proteins into the cytoplasmic membrane. Translocation requires the energy sources of ATP and the proton motive force (PMF) while the membrane protein insertion is coupled to polypeptide chain elongation at the ribosome. This review summarizes the present knowledge of the mechanism and structure of the Sec translocase, with a special emphasis on unresolved questions and topics of current research.

Contents
INTRODUCTION 644
TARGETING AND RECOGNITION 645
Signal Sequences 645
Targeting Routes 646
SecB 646
Signal Recognition Particle and Its Receptor 648
SecA 649
TRANSLOCATION 651
SecYEG 651
SecDF(yajC) 655
Mechanisms and Energetics 656
MEMBRANE PROTEIN INSERTION 659
Mechanisms and Lateral Pore Opening 659
YidC 660
SUBCELLULAR LOCALIZATION OF THE SEC SYSTEM 661

INTRODUCTION

Bacteria are generally simple structures that lack a membrane-bound nucleus. Because of this, they are described as prokaryotes, which also include the Archaea that represent a separate kingdom of life. Bacteria may be conveniently divided into two further groups, depending upon their ability to retain a crystal violet-iodine dye complex, termed the Gram's stain. Gram-negative and gram-positive bacteria have fundamentally different structures, related to the composition of the cell envelope. Gram-positive bacteria have many layers of peptidoglycan that surround the cytoplasmic membrane, whereas the cell envelope of gram-negative bacteria is more complex. Above the cytoplasmic membrane is a periplasm with some layers of peptidoglycan, and beyond this layer lies an outer membrane, which contains phospholipids and lipopolysaccharides. Some bacteria have an atypical cell envelope, such as the mycobacteria that contain a thick hydrophobic my-

Translocase: a membrane-embedded protein complex that mediates the translocation of polypeptides from one side of the membrane to the other side

colic acid layer surrounding the cytoplasmic membrane. Each of these envelope structures represents a subcompartment in the cell with its own set of resident proteins. Proteins are synthesized at ribosomes localized in the cytosol, and all proteins that function outside of the cytosol either need to insert or need to pass the cytoplasmic membrane to reach their final destination. The major route for protein transport across and into the cytoplasmic membrane is the Sec translocase (Figure 1). In its minimal form, the Sec translocase consists of a protein-conducting channel, the SecYEG complex, and a peripherally associated component that delivers energy for the transport process. This component is either a translating ribosome or SecA, an ATP-driven motor protein. Other Sec proteins may associate with the Sec translocase to provide it with additional functionalities. The protein-conducting channel is a highly conserved complex with homologs present beyond the prokaryotic kingdom, such as in the chloroplast thylakoid membrane and the endoplasmic reticulum of eukaryotes.

An excellent historical overview of the first discovery of the Sec translocase in bacteria until the establishment of a reconstituted in vitro translocation system, based on purified components, was written by Wickner and coworkers (1) more than 15 years ago for the *Annual Review of Biochemistry*. Extensive genetic analysis, using *Escherichia coli* as a model organism, has resulted in the identification of all major genes (and proteins) (for a review, see Reference 2). Another major milestone has been the complete functional in vitro reconstitution of the protein translocation reaction using purified components (3). This opened a new era of research allowing an understanding of the protein translocation process at the molecular level. Since the early 1990s, major progress has been made, and recent high-resolution structures of subunits of the Sec translocase represent new milestones (4, 5). Supported by extensive biochemical analysis, the most complete view of the molecular basis of a protein translocation and membrane protein

Figure 1

Scheme of protein targeting to the Sec translocase. The bacterial Sec translocase is a protein complex in the cytoplasmic membrane (CM), which comprises a peripheral motor domain SecA (green), the protein-conducting channel, SecYEG (orange), and the accessory proteins SecDF(yajC) (pink) and YidC (red). Signal peptidase (SPase) is a membrane-bound peptidase that cleaves the signal sequence from preproteins at the periplasmic face of the membrane. (a) Secretory proteins (yellow) are posttranslationally targeted to the Sec translocase by virtue of their signal sequence, which is recognized directly by SecA, the motor domain of the Sec translocase, or by the aid of the molecular chaperone SecB (blue). (b) Membrane proteins and some preproteins are cotranslationally targeted to the Sec translocase as ribosome-bound nascent chains by the SRP and the SRP-receptor FtsY (purple). (c) Some membrane proteins insert into the cytoplasmic membrane via YidC. Abbreviation: PMF, proton motive force.

insertion pathway is now possible. This review summarizes the recent progress on the Sec translocase, with an emphasis on the structural and mechanistic aspects, and does not discuss other translocases of the cytoplasmic membrane involved in the translocation of folded proteins or subsets of envelope proteins.

### TARGETING AND RECOGNITION

Secretory proteins (preproteins) and membrane proteins are synthesized at ribosomes in the cytosol. To reach their destination outside of the cytosol or in the cytoplasmic

### Signal Sequences

Secretory proteins (preproteins) are synthesized with an N-terminal extension, the signal sequence, which is removed after translocation (for a review, see Reference 6). Signal sequences have a tripartite structure with an

Preprotein: the precursor form of a secretory protein, which is usually an unfolded state of the protein with an N-terminal signal sequence

Signal sequence: an N-terminal extension of secretory proteins that functions as a sorting and recognition signal for the translocase

N-terminal region encompassing one to three positively charged amino acid residues (the N domain), a hydrophobic core region consisting of 10–15 residues (the H domain), and a more polar C terminus, which constitutes the signal peptidase cleavage site (C domain). These domains show little sequence conservation, but the presence of an N-terminal signal sequence can conveniently be predicted with computer algorithms such as SignalP (7). Signal peptidase is a membrane-bound enzyme that utilizes a Lys-Ser catalytic dyad for signal peptide cleavage (8). As shown by structural studies, the signal peptide cleavage site at the –3 and –1 positions contains amino acid residues with small neutral side chains that allow the signal peptide to dock into the catalytic site of signal peptidase. An α-helix-destabilizing amino acid residue is often found at position –6 and/or in the middle of the H domain (6). Nuclear magnetic resonance studies on the conformation of signal peptides in membrane mimetic environments suggest the presence of two structural domains: a stable α-helix N-terminal domain and a more flexible C domain (9, 10). Although mutagenesis studies have clearly established the importance of various domains of the signal sequence for translocation, deficiencies in one region can often be compensated by improving the quality of another region (11). This suggests a multifaceted mechanism of signal-sequence recognition by the Sec translocase. Integral membrane proteins generally do not contain a signal sequence, but instead, their hydrophobic transmembrane segments (TMSs) function as an internal signal for targeting and insertion.

**Transmembrane segment (TMS):** stretch of hydrophobic amino acid residues that anchors membrane proteins in the phospholipid bilayer

**Ribosome-nascent chain (RNC):** a complex of a ribosome, an mRNA, and a partially synthesized polypeptide chain

**Signal recognition particle (SRP):** complex of a 4.5S RNA molecule and the GTPase P48, which recognizes hydrophobic transmembrane segments in ribosome-nascent membrane proteins

article (SRP) (Figure 1, step b). These two pathways diverge at an early stage when the nascent chain emerges from the ribosome (12) because of a competition between the SRP and the peptidyl-prolyl *cis-trans* isomerase trigger factor for binding to the nascent chain (13). The strength of interaction of the SRP with signal sequences increases with the hydrophobicity of the H region (14). Owing to this high selectivity, the SRP binds directly to hydrophobic TMSs of nascent cytoplasmic membrane proteins, directing them into the cotranslational targeting pathway. Trigger factor shunts nascent preproteins into the SecB pathway by blocking the interaction between the SRP and the signal peptide (13). While the nascent chain grows, SecB or other chaperones may associate with the nascent preproteins, which are then targeted posttranslationally to the Sec translocase.

**SecB**

It has been known for a long time that synthesis and translocation of preproteins in *E. coli* are uncoupled events (15). Because proteins need to be translocated in an unfolded conformation and because unfolded proteins are unstable in the cytosol, the unfolded state of preproteins is stabilized by molecular chaperones prior to translocation. SecB is a secretion-dedicated chaperone (for a review, see Reference 16) found mostly in the α-, β-, and γ-proteobacteria (17). SecB is a homotetrameric protein that is organized as a dimer of dimers (18) (Figure 2). The structure of the *Haemophilus influenzae* SecB shows the presence of ~70-Å-long channels located on either side of the SecB tetramer that may be involved in the binding of unfolded polypeptide segments. Consistent with biochemical data, each binding groove seems to contain two subsites that may recognize distinct features of preprotein substrates. One subsite is a deep cleft, lined with mostly conserved aromatic residues and suited for binding of hydrophobic and aromatic regions of polypeptides, and the other subsite forms a shallow open groove

**Targeting Routes**

*E. coli* contains two major targeting routes that direct proteins to the Sec translocase (Figure 1). Most preproteins are targeted via the molecular chaperone SecB (Figure 1, step a) whereas cytoplasmic membrane proteins are targeted as ribosome-bound nascent chains (RNCs) by the signal recognition par-

Figure 2

Structure of SecB, a secretion-dedicated molecular chaperone. A ribbon drawing of the structure of the *H. influenza* SecB tetramer is shown in complex with peptides that correspond to the C-terminal SecB-binding domain of SecA (Protein Data Bank number 1OZB) (29) in two orthogonal views. (a) Front view showing the four-stranded β-sheet of each monomer and the packing of the dimer. (b) Side view showing the dimer-dimer interface formed by α-helices, and as indicated by the shaded ellipse, the long putative polypeptide-binding grooves on the sides of the SecB tetramer. The two SecA C-terminal peptides are shown as ribbon drawings in gray with the central zinc ion indicated as a sphere. Each subunit in the SecB tetramer is shown in a different color.

with a hydrophobic surface that might be involved in the binding of β-pleated sheets (18). The position of the binding sites suggests that polypeptides are wrapped around the tetrameric SecB protein and supported by electron paramagnetic resonance experiments (19). The mechanism by which SecB differentiates between secretory and nonsecretory proteins remains poorly understood. Various methods have been used to identify SecB-binding sites in preproteins (for an overview, see Reference 16). SecB only interacts with unfolded polypeptides and appears to associate with mature regions that are normally buried in the folded structure (20). On the basis of peptide-binding studies, a general SecB-binding motif has been defined that consists of approximately nine amino acid residues enriched in aromatic and basic residues, whereas acidic residues are strongly disfavored (21). Such peptide sequences are found with similar frequency in both cytoplasmic and secretory proteins, and this situation does not explain the high selectivity of SecB in vivo where it is associated with only a subset of long nascent secretory proteins (22). Selectivity in binding has been attributed to a kinetic partitioning between polypeptide folding and association with SecB (23). Herein, the signal sequence could indirectly affect the selectivity of SecB by retarding the folding of the preprotein. Because the rate of SecB association with polypeptide substrates is limited only by the rate of collision, which is much faster than folding (24), the specific and selective interaction with preproteins is likely a

complex and delicate mechanism with molecular features that remain unknown.

Possibly more general chaperones substitute for the SecB function in bacteria that lack SecB (25). A general chaperone activity has also been associated with SecB in chaperone-compromised cells (26), but a key feature that distinguishes SecB from other chaperones is its ability to interact with high affinity [K<sub>d</sub> ~10–30 nanomolars (nM)] with the SecYEG channel-bound SecA protein (27). The highly conserved 22 C-terminal amino acid residues of SecA constitute a genuine SecB-binding domain (28). Tetrameric SecB binds the dimeric form of SecA (28), consistent with the ability of SecB to bind two C-terminal SecA peptides (29) (Figure 2). The SecA C terminus is a highly flexible, positively charged region with three cysteines and a histidine residue that together coordinate a zinc atom that is needed for SecB binding (30). The latter stabilizes the fold of the C terminus of SecA (29). SecA-SecB binding is mediated through an electrostatic interaction of the positively charged SecA C termini with the negatively charged patch present on both sides of the SecB tetramer (29, 31) (Figure 2). This SecA-SecB complex can be envisaged as a symmetric structure wherein the C termini of the two SecA protomers embrace the tetrameric SecB with the postulated peptide-binding grooves on SecB aligned with the protein-binding sites on SecA (29).

SecA and SecB interact in solution with low affinity (K<sub>d</sub> ~1.6 μM) (16). This interaction involves the C terminus of SecB (32), which may insert at the dimer interface of SecA and bind the N terminus of SecA (29, 33). SecB does not interact with ATP. The transfer of the secretory protein from SecB to the SecYEG-bound form of SecA therefore occurs independently of energy. However, for membrane release, SecB depends on the binding of ATP to SecA, and in this respect, it seems to act as a cochaperone of SecA (28). Also, high-affinity SecA-SecB binding involves the dimeric SecA (31), whereas low-affinity binding is observed only

with the monomeric form of SecA (34). This has led to the suggestion that a SecA dimer-to-monomer transition underlies the mechanism of SecB release from the membrane (34).

**Signal Recognition Particle and Its Receptor**

Approximately 20% the proteome of *E. coli* concerns cytoplasmic membrane proteins (35). Most of these proteins enter the membrane via the Sec translocase. Unlike preproteins, membrane proteins are targeted to the membrane in a cotranslational manner. This involves a bacterial homolog of the eukaryotic SRP (for a review, see Reference 36). The bacterial SRP is of a lesser complexity compared to its eukaryotic counterparts. The *E. coli* SRP is composed of a complex of a 4.5S RNA and a 48-kDa GTPase P48 or Ffh (for fifty-four homolog) (37) that interacts specifically with the signal sequence or hydrophobic TMSs of nascent proteins (14, 38). In eukaryotes, this interaction results in a translational arrest, but this phenomenon has not been observed in bacteria. RNC-bound SRP interacts with FtsY (39), a prokaryotic homolog of the α-subunit of the SRP receptor. Bacteria lack a homolog of the β-subunit of the SRP receptor, and it is generally believed that FtsY fulfills the function of both the SRα- and SRβ-subunits of the SRP receptor (40). FtsY binds to membranes via the anionic phospholipids (41) but also interacts directly with the SecYEG channel (42). The interaction between FtsY and Ffh changes the nucleotide-binding affinity of both proteins and allows them to bind GTP (43, 44). Upon GTP hydrolysis by both the SRP and FtsY, the RNC-SRP-FtsY complex dissociates, and the released RNC complex is transferred to SecYEG (45), which binds the ribosome directly (46).

Ffh is composed of two domains, the NG domain (N, amino terminal; G, GTPase), which contains a GTP-binding site, and a methionine-rich C-terminal M domain, which is involved in the binding of the signal sequence and SRP RNA (47). The central

G domain is related to the p21Ras GTPase family. FtsY contains a similar NG domain and a very acidic N-terminal domain that is involved in membrane targeting (48). In both FtsY and Ffh, the N domain is closely associated with the G domain, and the connecting region between the two domains is highly conserved (49, 50). The interaction between both proteins appears to occur mainly via their structurally related NG domains (49, 51). The 4.5S RNA and the M domain of Ffh are not required for the stimulation of GTPase activity in vitro (52, 53). However, the 4.5S RNA coordinates the interaction of the SRP and FtsY (52) with recruitment of the ribosome and the subsequent transfer of the RNC to the SecYEG complex (54).

The signal-sequence-binding site of Ffh localizes to the M domain. The structure of the M domain of *Thermus aquaticus* Ffh showed that it contains a deep groove that is lined almost exclusively with the side chains of conserved hydrophobic residues and that includes all conserved methionines (55). The dimensions and hydrophobic character of the groove suggest that it forms the signal-sequence-binding pocket of the SRP (56). Close to this groove, part of the conserved RNA IV domain binds to the M domain. This suggests that the functional signal-sequence-binding site consists of both protein and RNA. Recent cryo-electron microscopy structures of RNC complexes with the SRP of the *E. coli* (57) and the mammalian system (58) indicate that the signal sequence is presented at the ribosomal tunnel exit in a manner that allows it to slide directly into the proposed signal-sequence-binding site on the SRP. Further analysis awaits high-resolution structures of the SRP with a bound signal sequence.

## SecA

SecA is a central component of the Sec translocase, which functions as an ATP-dependent motor protein (Figure 1). It interacts with nearly all other components involved in protein translocation, and its ATPase activity is allosterically regulated by unfolded preproteins, the SecYEG complex, acidic phospholipids, and by SecB (59, 60). In recent years, high-resolution structures of SecA proteins of various species have been solved (4, 61–64), providing detailed insight into their overall domain organization. In most of these crystal forms, SecA is dimeric with its protomers arranged in an antiparallel fashion, except for the *Thermus thermophilus* SecA, which has been crystallized as a parallel dimer (64). The SecA protomer (Figure 3) can be subdivided into several structural subdomains [nucleotide-binding folds 1 and 2 (NBF1 and NBF2); the preprotein cross-linking domain (PPXD); the α-helical scaffold domain (HSD); the α-helical wing domain (HWD); and the C-terminal linker (CTL)]. The actual motor function of SecA, i.e., the conversion of chemical energy into movement is performed by the “DEAD motor” core that is also found in DNA/RNA helicases (65). The DEAD motor consists of two RecA-like NBFs, NBF1 and NBF2. At the interface of these two domains, ATP can be bound and hydrolyzed to induce the conformational changes necessary for preprotein translocation. SecA interacts with preproteins via its PPXD (66, 67), a region that is inserted in NBF1 but that forms a separate domain in SecA (4) (Figure 3). Interestingly, an extended α-helix of the HSD contacts all other domains of SecA and therefore likely plays an important role in the catalytic cycle of SecA.

SecA is a soluble protein that localizes both to the cytosol and the cytoplasmic membrane (68). Its association with the membrane occurs via low- and high-affinity interactions with anionic phospholipids (59) and the SecYEG complex (27), respectively. The CTL domain of SecA has a dual function, including its involvement in lipid binding (69) and in SecB binding, as described in the previous section. SecA exists in a dynamic equilibrium between a monomeric and dimeric form (*K*~d~ ~ 0.1 μM) (70). In the cytosol and when purified, SecA is mainly dimeric (71). Although the exact features of the dimer

ANRV 345-BI77-26 ARI 15 May 2008 0:31

![Diagram](attachment:diagram.png)

Figure 3

Structure of SecA, the motor domain of the Sec translocase. The structure of the *Mycobacterium tuberculosis* SecA (Protein Data Bank number 1NKT) (61) shows the different subdomains (NBF1 and NBF2, nucleotide-binding folds 1 and 2; PPXD, preprotein cross-linking domain; HSD, α-helical scaffold domain; HWD, α-helical wing domain) in color and the corresponding linear display of the domains. The C-terminal linker (CTL) domain was not resolved in the structure. The second protomer of the dimeric SecA is represented as a (gray) ribbon. The intramolecular region of ATP hydrolysis 1 (IRA1), which controls hydrolysis of ATP at NBF1, localizes to the HSD (as indicated) and to a central opening at the SecA dimer interface (*shaded region*).

650 Driessen • Nouwen

interface differ between the various crystal structures of SecA, the C-terminal part of the extended HSD participates in dimer formation (4, 61, 62).

ATP binding and hydrolysis involves both the NBF1 and NBF2 of SecA. Both NBFs contain the highly conserved Walker A and the less-conserved Walker B motifs organized in a RecA structural fold (72). Both NBFs are essential for the ATPase activity of SecA and translocation (73). NBF1 is the catalytically important nucleotide-binding site (Kd, ADP ~ 0.13 μM) (74) that is regulated by NBF2 (73, 75). Both NBFs operate in concert to bind a single nucleotide comparable to the DEAD-box helicases (4, 62).

Several functionally important regions in SecA have been defined. The intramolecular regulator of ATP hydrolysis 1 (IRA1) region is a global regulator of the ATP turnover of SecA. This region localizes in the HSD. It forms a helix-loop-helix structure that contacts on one side NBF2 and on the other side the PPXD (Figure 3). For soluble SecA, deletion of IRA1, or disturbance of the IRA1-NBF2 or IRA1-PPXD interaction, results in an elevated preprotein-uncoupled ATPase activity (76). Therefore, the IRA1 domain likely prevents uncontrolled ATP hydrolysis of SecA in the cytosol. The central feature of the mechanism used by preproteins to control the ATPase cycle of SecA is, however, a highly conserved salt bridge, termed Gate 1, that controls the opening/closure of the nucleotide-binding cleft (77) in conjunction with the binding signal generated at the PPXD. This relay mechanism is only active after binding of SecA to SecYEG and allows the functional coupling between the preprotein binding and release mechanism and the ATPase cycle. Structural and functional data indicate an allosteric communication between the DEAD motor domain and the PPXD of individual monomers within the SecA dimer, wherein conserved regions lining the nucleotide cleft undergo cycles of disorder-order transitions (78). In a crystal form of a high [Mg²⁺]-enforced monomeric

and nucleotide-free *Bacillus subtilis* SecA (79), the PPXD, HSD, and HWD have undergone a drastic conformational change as compared to the protomer in the crystallized dimer, whereas the NBFs have remained at the same position. This form has been proposed to represent the “open” conformation where the HSD/HWD and the PPXD constitute a large groove that in its dimensions is similar to peptide-binding grooves observed in chaperones and peptide-binding proteins. SecA contains separate binding sites for the signal sequence and for the mature domain of preproteins (59), and an additional potential peptide-binding site locates to the interface of the NBF1, the HSD, and the PPXD (4).

Although it is generally accepted that SecA is dimeric in the cytosol, its translocation-active quaternary structure is currently under debate. In vitro, dimeric SecA dissociates into monomers upon an interaction with anionic phospholipids (80, 81), whereas signal peptides have been reported to either dissociate the dimer (81, 82) or promote its oligomerization (80). Also, it has been argued that SecA functions as a monomer because of the similarity of the DEAD motor domain with that of helicases (79). However, various lines of evidence indicate that in in vitro translocation assays SecA functions and remains dimeric (71, 83, 84). In addition, attempts to create a stable monomeric SecA form by site-directed mutagenesis and truncations result in a severe loss of activity (34, 81, 85).

## TRANSLOCATION

After targeting to the cytoplasmic membrane, preproteins are translocated across this membrane via a complex that is composed of several integral membrane proteins. Below, we describe the structure of this integral membrane protein complex and how it may facilitate protein translocation.

### SecYEG

The *E. coli* protein-conducting channel consists of three integral membrane proteins,

SecA/ribosome-  
associating groups  

a  
Cytosol  

Periplasm  

Plug  

b  

SecE  

SecY  
(TMSs 6–10)  

Lateral  
gate  

SecY  
(TMSs 1–5)  

SecG  
(Secβ)  

Hinge  

Pore ring  

Plug  

c  

Front to front  

SecE  

SecY  

Lateral  
gate  

Central  
pore  

Back to back  

**Figure 4**  
Structure of SecYEG, the protein-conducting channel of the Sec translocase (5). $(a)$ Membrane cross section and $(b)$ a cytosolic view of the structure of the $M$. jannaschii SecYE$\beta$ (Protein Data Bank number 1RHZ) (5) and $(c)$ a schematic presentation of possible SecYEG dimer configurations. The protein-conducting channel consists of three subunits: the SecY (Sec61$\alpha$) that is embraced by the SecE (Sec61$\gamma$) subunit and the peripheral bound SecG (Sec61$\beta$) protein. The channel forms an hourglass-like structure with a pore ring of hydrophobic amino acid residues at its constriction. The pore is closed at the periplasmic side by a plug formed by a short $\alpha$-helix of a periplasmic loop that folds back into the funnel. The two halves of the clamshell-like structure of SecY are indicated as TMS1–5 and TMS6–10 and are connected by a hinge region in the back. The clamshell opening in the front may form a lateral gate to the lipid bilayer. Signal-sequence insertion into lateral gate is thought to widen the central pore opening and to destabilize the plug, resulting in the opening of a vectorial water-filled channel (5).

termed SecY, SecE, and SecG, that together form a stable complex (3). The heterotrimeric organization of this complex, which corresponds to Sec61$\alpha\gamma\beta$ in eukaryotes, is conserved in all three kingdoms of life (86). The X-ray crystallography structure of an idle SecYE$\beta$ heterotrimer of the archaeon Methanococcus jannaschii (5) (Figure 4) and the cryo-EM structure of a active $E$. coli SecYEG complex, bound cotranslationally to an RNC (87), have revealed detailed aspects of the structure of the heterotrimeric complex and the highly dynamic nature of the translocation pore. In agreement with its universal conservation, the overall structure of the $M$. jannaschii SecYE$\beta$ superimposes with the low-resolution two-dimensional cryo-EM electron density map of the $E$. coli SecYEG complex (88). The two complexes differ only slightly in conformation; however, compared

to that of $M$. jannaschii, the $E$. coli complex contains three additional TMSs: two from SecE and one from SecG. The structure confirms the presence of 10 $\alpha$-helical segments in SecY (Sec61$\alpha$) with the N- and C-terminal ends at the cytosolic side of the membrane, and the single TMSs of SecE (Sec61$\gamma$) and of Sec61$\beta$ (SecG) expose only their N terminus to the cytosol. In $E$. coli, SecE consists of three TMSs, but a truncate, which resembles Sec61$\gamma$ with only the C-terminal TMS, suffices for functionality (89). SecY is organized into N- and C-terminal domains, compromising TMS1–5 and TMS6–10, respectively (Figure 4). Both domains are connected by a periplasmic loop between TMS5 and TMS6, yielding an overall structure that resembles a clamshell with a central funnel-like pore. Both domains of SecY are held together by SecE, which acts as a molecular

clamp. One part of the clamp is formed by the conserved amphipathic cytoplasmic loop that connects TMS2 with TMS3 of SecE, and the other part is formed by TMS3 of SecE at the “back” of SecY (5, 88). SecG makes only limited contact with SecY and localizes at the periphery of the complex (88). The *M. jannaschii* SecYEβ structure likely represents the “closed” state of the translocation channel (5), but its overall organization suggests a possible mechanism of channel opening. At the cytosolic face of the membrane, the proposed channel is shaped like an inverted funnel with a diameter of ~20–25 Å at its widest point and ~4 Å at its narrowest point, which is formed by a ring of isoleucine residues that constrict the channel (pore ring). At the periplasmic face of the membrane, the structure shows a cavity, which is closed by the first periplasmic loop of SecY that folds back as a distorted α-helix (TMS2a) into the funnel-like cavity. This structure is termed the plug and is believed to be displaced from the periplasmic cavity upon the initiation of protein translocation (90), which would result in the opening of the pore to the periplasmic side of the membrane thus forming a continuous vectorial aqueous path across the membrane. Pore opening is probably accompanied by an overall expansion of the SecY clam-like structure, allowing insertion of the signal sequence plus the early mature region of a preprotein as a hairpin-like structure. Opening of the channel might be achieved by a hinge movement of the loop region that connects TMS5 and TMS6 of SecY (5). A small conformational change in this region may move the two domains of SecY away from each other, thereby creating an opening between TMS2 and TMS7 toward the lipid bilayer, which is at the “front” of the SecYEG complex. Intercalation of the signal sequence between TMS2 and TMS7 at the lipid bilayer interface (91) and displacement of the plug region would result in a channel that is sufficiently large to permit the translocation of unfolded polypeptide segments through its center. This open state of the channel has an hourglass shape with hydrophilic funnels on both sides of the central constriction. The hydrophobic pore ring may fit like a gasket around the translocating polypeptide chain, thereby providing a seal that restricts the passage of ions and other small molecules during protein translocation. Mutations in the pore ring indeed result in transient openings of the SecYEG channel (92). However, the diameter of the pore ring is likely too small to allow passage of an extended polypeptide chain, and therefore, additional widening of the pore is necessary. The latter may occur by the movement of the α-helices to which the pore region residues are attached. The high plasticity of the SecY channel also follows from observations that, in the absence of SecE, SecY is intrinsically unstable and readily degraded by the membrane-bound protease FtsH (93).

Cysteine cross-linking studies with the *E. coli* SecY, based on the *M. jannaschii* SecYEβ structure, and fluorescence studies confirm that a translocating nascent polypeptide chain passes through the membrane by the postulated water-filled channel and suggest that the pore ring residues form a tight hydrophobic ring that surrounds the translocating polypeptide chain (94, 94a). Double cysteine SecY mutants showed that the plug domain moves away from the channel opening upon initiation of translocation (95) and that, in the displaced state, there is a constant channel opening (92). Moreover, deletion studies of the plug domain suggest that it mainly serves to stabilize the closed state of the SecYEG channel (96). Based on the X-ray structure, the maximum dimensions of the pore are ~15–20 Å (5), which accommodates polypeptide segments with an α-helical secondary structure. However, the Sec translocase can also translocate larger disulfide bond-stabilized tertiary loops in secretory proteins (97) as well as proteins, conjugated with bulky fluorescent dyes (98), and organic molecules (99). A multitude of biochemical (100–102) and structural (87, 103–105) data demonstrate that the SecYEG (and Sec61αβγ complex) complex assembles into highly ordered oligomers, with tetramers and dimers as the most dominant
forms. SecA and the ribosome recruit SecYEG monomers to form such oligomers (105), but SecYEG also has a strong tendency to dimerize in the absence of these ligands. A larger pore might be generated when two (or more) SecYEG complexes associate into a “front-to-front” organization in which the lateral openings fuse to form a consolidated pore (106). In the two-dimensional crystals of the *E. coli* SecYEG, the molecules are arranged as a “back-to-back” dimer (88) (Figure 4c), consistent with cross-linking studies on the idle state of the complex (107). However, a cryo-EM study of the *E. coli* SecYEG complex bound cotranslationally to a ribosome-nascent polypeptide complex suggests a dimeric arrangement of SecYEG channels in a front-to-front arrangement in its active state (87) (Figure 4c). But only one channel is used to translocate the polypeptide chain, which is consistent with cross-linking data (108). In addition to dimeric SecYEG complexes, higher ordered, possibly tetrameric complexes have been observed (100, 102, 104). In eukaryotes, such complexes have been observed when the Sec61 complex is in association with ribosomes, but their functional role is not known. Obviously, in order to settle controversies on subunit stoichiometry, the X-ray structure of an active SecYEG channel is the ultimate goal for future research.

Because SecYEG does not utilize nucleotides to generate energy, it must associate with cellular components that provide the driving force necessary for polypeptide translocation or insertion. For cotranslational membrane protein insertion, SecYEG associates with the ribosome (46), whereas in the posttranslational mode, it interacts with SecA (27). The ribosome and SecA interact with similar regions of SecYEG, which may suggest a similar type of channel-opening mechanism. Structural and biochemical data indicate that ribosomes associate with the SecYEG complex at three distinct sites, two of which are formed by the pairs of long cytoplasmic loops of SecY (C4 and C5, respectively) (87, 109, 110) (Figure 4). The third connection is mediated by the cytoplasmic loop of SecG and the two N-terminal TMSs of SecE. SecA binds the same C4 and C5 loop regions of SecY (111, 112) with SecG (113) and with a region at the interface between TMS4 and C3 of SecY that is in direct contact with SecG (111). Importantly, interacting regions are located in both halves of the SecY clamshell (111), which provides a means of communication between SecA and SecY to sense or induce the transition from a closed to an open state of the pore. In a dimeric arrangement of SecYEG, separation of the two SecY domains takes place at the dimer interface, and thus the opening of one subunit is directly transmitted to the neighboring subunit. It should be noted that the features that mediate the third ribosome-SecYEG connection (SecG and the SecE extension) are not essential for cell viability or protein translocation (114, 115). Ribosome-induced opening of the SecYEG channel may be mediated primarily by the C4/C5 connections, and the third connection plays an auxiliary role. The stimulatory role of SecG in SecA-dependent translocation may be based on similar interactions.

The classical view is that channel opening is mediated by the signal sequence and that the ribosome and SecA merely function to release this domain to the SecYEG channel. In another view, the ribosome and SecA have been proposed to be actively engaged in the channel-opening mechanism (106) (see also the section Mechanisms and Energetics). The SecA-dependent opening mechanism might, however, differ in several aspects from that of the ribosome as SecA has been proposed to penetrate deeply into the SecYEG channel (116, 117). Moreover, SecY mutants of the C4 loop are severely defective in SecA-mediated posttranslational protein translocation (118) but normally allow the SecA-dependent translocation of periplasmic domains of membrane proteins that are targeted to the Sec translocase via the SRP pathway (119). The latter involves the ribosome instead of SecA to initiate translocation,suggesting different requirements for SecA and the ribosome in channel opening. SecG is in close proximity to C3 and TMS4 of SecY, which is the region where SecA is postulated to (partially) insert or interact with the SecYEG complex (120, 121). This may explain why SecG promotes the SecA conformational cycle. SecG has been proposed to invert the membrane topology of its two TMSs during a catalytic cycle of the Sec translocase (122). Structural studies, however, provide no clues on how this process may occur (5). Moreover, a topologically fixed SecG variant has been shown to be fully functional (123). Therefore, the presumed topology inversion most likely represents a conformational change within a highly dynamic region of the SecYEG complex.

Genetic studies have identified a series of mutations in sec genes that can suppress the translocation defect of preproteins with a defective signal sequence (124). Interestingly, many of these mutations are in secY (prlA) and localize to the pore-facing side of TMS7 and the plug domain (TMS2a) (for a review, see Reference 2). In these mutants, the SecY-SecE interaction is destabilized (125). Prl mutations do not merely compensate for the signal-sequence defect by restoring the recognition, but also are thought to facilitate the opening of the channel and either stabilize the open or destabilize the closed state of the channel. With the native complex, the open state is supposed to be stabilized by the signal sequence, SecA, and/or the ribosome (5). Indeed, several of the prlA mutations have been shown to stabilize the interaction between SecY and SecA, and in vitro these mutants are hyperactive (126) and less dependent on the proton motive force (PMF) (127) for translocation. It thus appears that prlA mutations alter the channel conformation, and as a result of this conformational change, SecA is bound tightly to SecY, which permits a more efficient initiation of translocation. This results in a more efficient translocation of wild-type preproteins and of the translocation of signal-sequence defective preproteins above background levels. This concept has been termed proofreading (124) and might be directly related to the mechanism by which SecY activates the ATPase catalytic cycle of SecA. SecA binds with high affinity (Kd ~10 nM) to the SecYEG complex. This binding reaction converts SecA to an activated state (128) that is primed to interact with preproteins and SecB (27, 28). Priming is manifested by an acceleration of the rate of nucleotide exchange on SecA (128), which removes the rate-limiting step in ATP hydrolysis, namely ADP release (129). The tight link with SecYEG binding prevents the uncontrolled hydrolysis of ATP by the cytosolic SecA. Possibly, prlA mutants resemble a constitutively activated state of the SecYEG-bound SecA.

### SecDF(yajC)

The SecYEG channel can associate with another heterotrimeric membrane complex consisting of the SecD, SecF, and YajC proteins (130). In E. coli, SecD and SecF are polytopic membrane proteins with a large periplasmic domain (131). This structural organization is reminiscent of the Resistance-Nodulation-Cell Division family of membrane proteins that includes multidrug-resistance pumps (132). YajC is a membrane protein with a single TMS and a large cytosolic domain. Although YajC associates with SecDF, it is not needed for functionality, but cells lacking SecD and SecF are cold sensitive, which prevents growth, and are severely defective in protein translocation (133). Membranes depleted of SecDF(yajC) (134) or containing SecDF mutants are severely defective in vitro protein translocation (135), but SecDF(yajC) is not needed for translocation per se, as shown by reconstitution studies (3). Possibly SecDF(yajC) functions in a step downstream of the primary translocation reaction, after the release of proteins from the SecYEG channel exit site (136), or in the clearance of the pore for processed signal sequences or phospholipids. SecDF(yajC) have been proposed to regulate the catalytic cycle

**Polytopic membrane protein:** a membrane protein that contains more than one transmembrane segment

www.annualreviews.org • Bacterial Protein Translocation 655of SecA, thereby controlling the movement of the polypeptide chain through the SecYEG channel (137). However, this effect is presumably indirect as Archaea lack a SecA homolog but contain the SecD and SecF proteins that are needed for protein translocation (86, 138). Another possibility is that SecDF(yajC) is catalytically involved in the formation or regulation of an oligomeric SecYEG pore complex, which would be consistent with their lower abundance as compared to the number of SecYEG complexes in the cytoplasmic membrane.

## Mechanisms and Energetics

The driving force for preprotein translocation is provided by ATP hydrolysis at SecA and by the PMF (139). These energy sources are employed at different stages of the process. ATP is essential for the initiation of the translocation and is used throughout the process. Several of the intermediate steps during ATP-dependent translocation have been resolved (140, 141). Protein translocation starts with the binding of ATP to SecA, which allows the insertion of a hairpin-like loop structure of the signal sequence with the early mature protein region into the translocation pore. This step can be stimulated by the PMF, which likely affects the orientation of the signal sequence in the translocation pore (142, 143). ATP hydrolysis results in a release of the bound preprotein from SecA (140), after which SecA can either dissociate from SecYEG or rebind to the partially translocated preprotein trapped in the SecYEG pore. This rebinding results in an ATP-independent translocation of a polypeptide segment of about 2–2.5 kDa, and subsequent ATP binding causes the translocation of another 2–2.5 kDa (140, 141). Multiple rounds of nucleotide binding and hydrolysis by SecA drives the stepwise translocation of the preprotein, whereby each turnover of SecA results in the translocation of ~5 kDa in two consecutive steps (141). The exact step size, however, has not yet been precisely defined, but the time frame for translocation increases linearly with the length of the preprotein as predicted for a stepping mechanism (144). Physiochemical properties of the translocating polypeptide segment as, for instance, the presence of relatively hydrophobic segments likely influence also step size and translocation kinetics (145). Once ATP hydrolysis has dissociated the preprotein from SecA, the PMF can drive translocation further and, in late stages, even complete translocation in the absence of ATP (97, 140, 141, 146). Although PMF-driven translocation occurs independently of SecA and ATP, these two modes of translocation are tightly interrelated. For instance, the PMF has been shown to stimulate the release of ADP from SecA (129), and it promotes a translocation-related conformational change of SecA (147). The PMF also seems to modulate the opening or formation of the translocation channel (97, 127). The intermediate stages of translocation are reversible, and temperature-dependent retrograde movements of the polypeptide chain can occur in the absence of SecA, ATP, and the PMF (140, 146).

Importantly, the Sec translocase functions in the translocation of unfolded polypeptides, whereas stably folded structures generally cause a translocation arrest (148). However, the Sec system is capable of unfolding and translocating the tightly folded human cardiac Ig-like domain I27 when fused to the C terminus of a preprotein (149). Because of the requirement for unfolding, this process is accompanied by an increased utilization of ATP, suggesting the participation of SecA in active unfolding, which is similar to that shown for some cytosolic chaperones. Indeed, a chaperone activity has been suggested for SecA; its proposed function is in a quality control mechanism that assists the folding of signal-sequence-less proteins, thereby excluding them from the secretion process (150).

The exact molecular mechanism by which SecA promotes translocation is largely unresolved. On the basis of protease resistance
and labeling with chemical reagents added from outside of the cell, it has been proposed that SecA inserts, with almost its entire mass, deeply into the SecYEG channel (116, 151). Considering the molecular dimensions of SecA, it seems unlikely that a SecYEG pore can accommodate such a large structure. Probably, SecA adopts a protease-resistant conformation upon SecYEG, preprotein, and nucleotide binding (152) and is accessible from the periplasmic face of the membrane to small labeling reagents that permeate the SecYEG pore (153). Indeed, the chemical modification sites are spread out over the entire SecA molecule (4, 154).

The structural information on SecA and the SecYEG complex, either alone or in association with the ribosome, now provides some clues concerning the possible mode of action of SecA. Basically there are two major mechanistic models that describe the role of SecA as a motor protein, i.e., the power-stroke model and the Brownian ratchet (155). In the power-stroke mechanism, ATP binding and hydrolysis induce conformational changes in SecA that are translated into a mechanical force, which is imposed on the bound protein substrate. This pushing force then drives the movement of the preprotein through the SecYEG channel. With the Brownian ratchet, SecA biases the random Brownian motion of a translocating unfolded polypeptide chain (156). In this model, spontaneous reversible movement (hysteresis) of the polypeptide chain in the SecYEG channel is coupled to energy-dependent trapping by SecA. Trapping prevents retrograde movement, thereby providing directionality to the translocation process. The observed step-wise translocation mechanism linked to nucleotide-dependent conformational changes in SecA suggest a power-stroke mechanism (140, 141, 144). Because of the homology of the NBFs of SecA to the corresponding RecA domains of the DEAD helicase family, it has been postulated that the translocation-active SecA is monomeric (79) and functions according to the inchworm mechanism proposed for the

helicas PcrA (72, 157). However, DEAD helicases contain two substrate-binding sites with different affinities, and so far for the monomeric SecA, only one peptide-binding site has been detected. For this reason, the SecYEG channel has been implicated in the formation of the second peptide-binding site (81, 108).

Other models implicate the dimer as the translocation-active state of SecA (71). The structures of the antiparallel SecA dimer show a central opening (4, 61). Another view of a power-stroke mechanism is the piston model, which proposes that the central SecA opening aligns with the SecYEG pore and that it traps the preprotein in the initial SecA-bound state. By means of a nucleotide-dependent power stroke, the SecA-bound preprotein is pushed through the pore (61). This model was further refined in the molecular peristalsis model (106) (Figure 5). According to this model, the dimeric SecA docks onto a (front-to-front) SecYEG dimer creating a large vestibule between the two protein complexes (Figure 5a). An unfolded preprotein gains access to the PPXD regions of the SecA dimer that are exposed to the inner surface of the vestibule via the central opening. Subsequently, binding of ATP induces a conformational change in SecA, resulting in a different dimer interface (Figure 5b). This closes the central opening in the SecA dimer, thereby trapping the bound preprotein, with a concomitant reduction in the volume of the vestibule and an opening of the SecYEG channel. The latter would result from a reduction of the distance between the interaction sites of the individual SecA monomers and the cytosolic loops of two SecY proteins. The strict coupling between a reduction in cavity volume and opening of the SecYEG channel directs the polypeptide segment present in the central cavity into the translocation pore, and backsliding is prevented by the closure of the central opening in the SecA dimer. Next, ATP is hydrolyzed (Figure 5c), which results in a reversal of the SecA dimer conformational change, and coupled with the opening and closing of the

C
SecB
Cytoplasm

N
SecA
dimer
ADP
ADP
ADP
ADP
ADP
ADP
a
b
c
d
SecYEG₁
SecYEG₂
Periplasm

Figure 5

Model for Sec-mediated protein translocation. The peristalsis model, proposed in Reference 106, is based on SecA dimer structures from *M. tuberculosis* (open pore) (61) and *B. subtilis* (closed pore) (4) and proposes that the nucleotide-dependent changes in the SecA dimer conformation coordinate the SecYEG channel opening and closing. (a) The SecA dimer in an open-pore conformation is bound to the dimer of SecYEG, creating a large central cavity in between SecYEG and SecA. In this state, it accepts the preprotein from Sec B. Because of Brownian motion, the polypeptide passes through the central opening in the SecA dimer into the cavity where the signal sequence of the protein binds to the PPXD of one of the SecA protomers while the cavity fills up with protein. The PPXD of the second SecA protomer may also participate in the binding of the protein in the central cavity. (b) Conformational changes (arrow) owing to ATP binding (ADP/ATP exchange) to SecA result in the closure of the SecA central opening concomitantly with an opening of the SecYEG channel. The conformational change of the SecA dimer results in a reduction of the cavity volume and the release of secretory proteins from the PPXD(s) of SecA. The signal sequence and early mature region of the secretory protein are forced to move into the open SecYEG pore as a looped structure. (c) ATP hydrolysis reverses the SecA conformational change, which results in the reopening of the central SecA channel and closure of the SecYEG pore, allowing a new stretch of secretory protein to enter the cavity. (d) This cycle of nucleotide-dependent alternating opening and closing of the central opening in the SecA dimer and the pore in SecYEG is repeated until translocation of the polypeptide is completed. The proton motive force (PMF) may drive the translocation of larger polypeptide segments when SecA has released the secretory protein after ATP hydrolysis. How this translocation energy is coupled is unclear, but this activity may be effected by a PMF-dependent opening or widening of the SecYEG pore. The figure was adapted with permission from Reference 106.

channels in SecA and SecYEG, respectively. Further diffusion of nontranslocated polypeptide substrate into the cavity and subsequent binding of ATP (Figure 5d) result in the translocation of another segment. Alternatively, ATP hydrolysis releases SecA from the polypeptide chain and the SecYEG channel whereafter SecA is replaced by another SecA protein or possibly, in the case of a nascent membrane protein, the (re-)association of the ribosome with SecYEG. Notably, in the peristalsis model, the actual movement of the polypeptide is driven by Brownian motion, whereas a power stroke is employed to decrease the cavity size and to open the SecYEG channel. In the inchworm and piston models, the step size of translocation depends on the size of the lever arm. To account for a step size of ~25 amino acids (140, 141), a very large conformational change is needed

that reaches a distance of ~85 Å (one amino acid is 3.3 Å). Interestingly, the large step size might be explained by the peristalsis mechanism, as it can be determined only by the volume of the vestibule formed between SecA and SecYEG. Because these models are built on static structures of SecA and SecYEG, experiments should be designed to provide direct experimental evidence for the postulated mechanistic models.

### MEMBRANE PROTEIN INSERTION

A remarkable feature of the SecYEG complex is that it is capable of two seemingly opposite functions, i.e., the translocation of polar polypeptide segments across the cytoplasmic membrane and the insertion of hydrophobic TMSs of membrane proteins into the cytoplasmic membrane. Membrane proteins exist in many different topologies with one or multiple TMSs. In contrast to the N-terminus of a signal sequence that always remains in the cytosol, the first TMS of a nascent membrane protein will either face the cytosol or will be translocated to the periplasm. Although there are some exceptions, the first TMS of a polytopic membrane protein often determines the orientation of subsequent TMSs, which need to alternate in orientation. The topology of membrane proteins mostly follows the positive-inside rule (158). Statistical analysis of bacterial membrane proteins showed a very strong distribution of positively charged residues in cytoplasmic loops rather than in periplasmic loops (158). This has been attributed to the orientation of the transmembrane electrical potential (negative in the cytoplasm) (159) and to the presence of negatively charged phospholipids in the cytoplasmic membrane, which may interact with the positively charged amino acid residues in the cytosolic loops (160). The alteration of the charge distribution in the regions flanking a TMS often results in a reversal of the topology. However, other features of a TMS such

as its length and mean hydrophobicity are also important for membrane insertion (161).

#### Mechanisms and Lateral Pore Opening

The requirement for specific components of the Sec translocase for membrane integration varies for different classes of membrane proteins. Many membrane proteins integrate into the membrane independently of SecA (162, 163), but SecA is required for the translocation of large periplasmic loops (164). The mechanism by which hydrophobic TMSs partition from an aqueous interior of the translocation channel into the hydrophobic interior of the phospholipid membrane is largely unresolved. Hydrophobic regions in preproteins can serve as stop-transfer sequences, causing a translocation arrest and a lateral release from the translocation channel (161, 165, 166). The stop-transfer function of these sequences correlates to their mean hydrophobicity and is reinforced by the presence of positive charges in the flanking region of the hydrophobic domain (167). In *E. coli*, the presence of a synthetic stop-transfer region in a preprotein induced the release of SecA from SecYEG, whereupon the hydrophobic domain partitioned into the membrane (165, 166). Membrane integration is likely a kinetically controlled phenomenon in which slow translocation of hydrophobic sequences is critical for stable integration into the lipid bilayer. Indeed, moderate hydrophobic regions can escape membrane insertion as a result of rapid translocation (165).

The structure of the SecYEG channel suggests that the channel has a lateral gate through which TMSs can partition into the lipid bilayer (5). This lateral gate is formed by the relative short helices of TMS2b, -3, -7, and -8, which localize to both SecY halves (Figure 4). The gate may undergo opening and closing during the translocation of a polypeptide chain (92, 96), and possibly even fluctuates thereby temporarily exposing translocating polypeptide segments to the
hydrophobic interior of the lipid bilayer. This may enable the equilibration between the two phases. Alternatively, TMSs may already be sensed at an early stage by the ribosome (or SecA), which may alter or open the lateral gate to the lipid bilayer in a coordinated manner. TMSs likely integrate into the membrane by a simple partitioning between the SecYEG pore and the lipid bilayer. This passive-partitioning model is supported by the observation that a hydrophobicity scale, derived from peptide interactions with an organic solvent, can be used to predict the tendency for a TMS to integrate into the membrane (168).

The insertion of membrane proteins with multiple TMSs is probably much more complicated because a translocation pore consisting of a single SecYEG channel is too small to store several TMSs. Cross-linking studies suggest that the TMSs of multispansing membrane proteins leave the channel one at a time or perhaps in pairs (169). After moving through the lateral gate, some TMSs are directly exposed to lipids, and others remain associated with SecYEG (or bind to the YidC protein—see the YidC section, below.) until termination of translation (169). Another possibility is that the TMSs of multispansing membrane proteins are assembled by an oligomeric complex of SecYEG and are released into the lipid bilayer after dissociation of the ribosome and the SecYEG oligomer. While the newly synthesized membrane protein inserts into the membrane, it folds into its final conformation. This not only involves extensive helix-helix packing but also specific interactions with phospholipids that are critical for the proper topological folding of membrane proteins (169a).

### YidC

Although most membrane proteins insert into the membrane via the Sec system, in recent years, YidC has been identified as a novel and essential membrane protein that facilitates the insertion of a subset of membrane proteins on its own and in cooperation with the Sec system (Figure 1, step c) (for reviews see References 170 and 171). YidC is functionally and structurally homologous to Oxa1 in mitochondria and Alb3 in chloroplasts. The latter two proteins function as membrane protein insertases in these organelles and play an important role in the membrane assembly of energy-transducing complexes. YidC can also associate with SecYEG (172) and with SecDF(yajC) (173), where it contacts the TMSs of newly synthesized membrane proteins and, in some cases, facilitates insertion. In *E. coli*, YidC is involved in the functional assembly of the F₁F₀ ATPase and cytochrome *bo₃* quinol oxidase (170). Remarkably, YidC catalyzes the membrane insertion of small phage coat proteins, such as M13 procoat and Pf3, that previously were thought to insert spontaneously into the membrane (174, 175). The membrane-insertase function of YidC was demonstrated by reconstitution studies in which purified YidC was shown to catalyze the membrane insertion of the F₀ subunit c of the F₁F₀ ATPase (176) and the Pf3 coat (177). Finally, YidC has also been implicated in the proper folding of LacY, a lactose/H⁺ symporter (178). In its Sec-independent activity, YidC seems to function primarily in the insertion of small (mono- and bitopic) membrane proteins that need to assemble into larger membrane protein complexes. Possibly, it also actively participates in the assembly of membrane protein complexes as a chaperone. Within the context of SecYEG, YidC may stabilize moderately hydrophobic TMSs after they are inserted into the membrane. This could facilitate their association with subsequently inserting TMSs, whereafter they can be released as a properly folded membrane protein into the lipid bilayer. The mitochondrial Oxa1 was shown to bind the ribosome via its C-terminal tail (179), a region lacking in bacterial YidC proteins. Some substrates of the YidC only pathway require the SRP for membrane targeting, and one of the remaining questions is how these proteins are targeted correctly to YidC insteadof to SecYEG, which by default accepts RNC complexes. Other YidC substrates do not depend on the SRP for targeting (176, 177) and must be recognized directly by YidC.

**SUBCELLULAR LOCALIZATION OF THE SEC SYSTEM**

Asymmetric localization of proteins is essential for many basic biological processes in bacteria, such as motility and chemotaxis. Also, some bacterial infection-related proteins seem to locate specifically to one of the cellular poles. This led to the suggestion that components of the Sec system may be spatially organized in the cytoplasmic membrane. By means of immunofluorescent microscopy and protein fusions with green fluorescent protein, several of the Sec components were localized into helical arrays in the rod-shaped bacteria *E. coli* (180) and *B. subtilis* (181). This “Sec coil” appears distinctly different from the MreB coil, an actin-like cytoskeletal protein. However, other studies suggest that the Sec system is evenly distributed within the cytoplasmic membrane of *E. coli* (182). In the coccoid *Streptococcus pyogenes*, the Sec system localized to a unique single microdomain termed the ExPortal site close to the division septum (183). Possible explanations for this phenomenon include Sec system interactions with murein, with actin cytoskeletons, or with specific lipid domains. Clustering of the Sec system in the cytosolic membrane with specific associated factors might allow for the selection of particular secretory proteins that can be targeted to unique subcellular sites, such as the pole(s) or the mid cell. However, the mechanisms responsible for this spatial localization remain mysterious.

---

### SUMMARY POINTS

1. Bacterial secretory proteins are synthesized in the cytosol with an N-terminal signal sequence and are targeted to the Sec translocase by the molecular chaperone SecB or as RNCs by an SRP.
2. The Sec translocase is embedded in the cytoplasmic membrane and consists of a protein-conducting channel (SecYEG) and an ATP-dependent motor domain, which also functions as a preprotein receptor (SecA).
3. SecB transfers the preprotein to the SecYEG-bound form of SecA and dissociates from the membrane upon the binding of ATP to SecA.
4. Protein translocation is a step-wise process whereby cycles of ATP binding and hydrolysis by SecA permit the translocation of polypeptide segments with a discrete length.
5. Proteins translocate in an unfolded conformation and pass through the membrane via an aqueous pore formed by the SecYEG complex.
6. Bacterial cytoplasmic membrane proteins are targeted to the Sec translocase as RNCs by an SRP and cotranslationally integrate into the cytoplasmic membrane via the SecYEG protein-conducting channel.
7. Some cytoplasmic membrane proteins, most notably subunits of the main energy-transducing complexes, integrate into the membrane with the aid of the YidC protein.
FUTURE ISSUES

1. How does SecA drive translocation of polypeptide chains through the SecYEG channel? How are the energy sources, ATP and the PMF, employed in this process?
2. How are TMSs of membrane proteins inserted into the cytoplasmic membrane? How is the lateral pore in SecYEG gated? How are the actions of SecA and the ribosome coordinated during membrane protein insertion and translocation of extracellular loops?
3. How do SecYEG complexes assemble into functional units? What is the stoichiometry of the active SecA-SecYEG complex? How do the interacting partners SecDF(yajC) and YidC cooperate with the SecYEG pore?
4. How does YidC recognize its substrates, and how does it catalyze membrane protein insertion?
5. How are certain secretory and membrane proteins targeted to specific subcellular sites?

DISCLOSURE STATEMENT

The authors are not aware of any biases that might be perceived as affecting the objectivity of this review.

ACKNOWLEDGMENTS

We apologize to authors whose work we were unable to cite because of space limitations.

LITERATURE CITED

1. Wickner W, Driessen AJM, Hartl FU. 1991. *Annu. Rev. Biochem.* 60:101–24
2. Danese PN, Silhavy TJ. 1998. *Annu. Rev. Genet.* 32:59–94
3. Brundage L, Hendrick JP, Schiebel E, Driessen AJM, Wickner W. 1990. *Cell* 62:649–57
4. Hunt JF, Weinkauf S, Henry L, Fak JJ, McNicholas P, et al. 2002. *Science* 297:2018–26
5. van den Berg B, Clemons WM Jr, Collinson I, Modis Y, Hartmann E, et al. 2004. *Nature* 427:36–44
6. von Heijne G. 1990. *J. Membr. Biol.* 115:195–201
7. Nielsen H, Engelbrecht J, Brunak S, von Heijne G. 1997. *Protein Eng.* 10:1–6
8. Paetzell M, Dalbey RE, Strynadka NC. 1998. *Nature* 396:186–90
9. Rizo J, Blanco FJ, Kobe B, Bruch MD, Gierasch LM. 1993. *Biochemistry* 32:4881–94
10. Chupin V, Killian JA, Breg J, de Jongh HH, Boelens R, et al. 1995. *Biochemistry* 34:11617–24
11. Hikita C, Mizushima S. 1992. *J. Biol. Chem.* 267:12375–79
12. Koch HG, Hengelage T, Neumann-Haefelin C, MacFarlane J, Hoffschulte HK, et al. 1999. *Mol. Biol. Cell* 10:2163–73
13. Beck K, Wu LF, Brunner J, Muller M. 2000. *EMBO J.* 19:134–43
14. Valent QA, Kendall DA, High S, Kusters R, Oudega B, Luirink J. 1995. *EMBO J.* 14:5494–505
15. Randall LL. 1983. *Cell* 33:231–40

16. Fekkes P, Driessen AJM. 1999. *Microbiol. Mol. Biol. Rev.* 63:161–73
17. van der Sluis EO, Driessen AJM. 2006. *Trends Microbiol.* 14:105–8
18. Xu Z, Knafels JD, Yoshino K. 2000. *Nat. Struct. Biol.* 7:1172–77
19. Crane JM, Suo Y, Lilly AA, Mao C, Hubbell WL, Randall LL. 2006. *J. Mol. Biol.* 363:63–74
20. Randall LL, Topping TB, Suciu D, Hardy SJ. 1998. *Protein Sci.* 7:1195–200
21. Knoblauch NT, Rudiger S, Schonfeld HJ, Driessen AJM, Schneider-Mergener J, Bukau B. 1999. *J. Biol. Chem.* 274:34219–25
22. Randall LL, Topping TB, Hardy SJ, Pavlov MY, Freistroffer DV, Ehrenberg M. 1997. *Proc. Natl. Acad. Sci. USA* 94:802–7
23. Hardy SJ, Randall LL. 1991. *Science* 251:439–43
24. Fekkes P, den Blaauwen T, Driessen AJM. 1995. *Biochemistry* 34:10078–85
25. Wild J, Altman E, Yura T, Gross CA. 1992. *Genes Dev.* 6:1165–72
26. Ullers RS, Luirink J, Harms N, Schwager F, Georgopoulos C, Genevaux P. 2004. *Proc. Natl. Acad. Sci. USA* 101:7583–88
27. Hartl FU, Lecker S, Schiebel E, Hendrick JP, Wickner W. 1990. *Cell* 63:269–79
28. Fekkes P, van der Does C, Driessen AJM. 1997. *EMBO J.* 16:6105–13
29. Zhou J, Xu Z. 2003. *Nat. Struct. Biol.* 10:942–47
30. Fekkes P, de Wit JG, Boorsma A, Friesen RH, Driessen AJM. 1999. *Biochemistry* 38:5111–16
31. Fekkes P, de Wit JG, van der Wolk JPW, Kimsey HH, Kumamoto CA, Driessen AJM. 1998. *Mol. Microbiol.* 29:1179–90
32. Randall LL, Crane JM, Liu G, Hardy SJ. 2004. *Protein Sci.* 13:1124–33
33. Crane JM, Mao C, Lilly AA, Smith VF, Suo Y, et al. 2005. *J. Mol. Biol.* 353:295–307
34. Randall LL, Crane JM, Lilly AA, Liu G, Mao C, et al. 2005. *J. Mol. Biol.* 348:479–89
35. Krogh A, Larsson B, von Heijne G, Sonnhammer EL. 2001. *J. Mol. Biol.* 305:567–80
36. Egea PF, Stroud RM, Walter P. 2005. *Curr. Opin. Struct. Biol.* 15:213–20
37. Poritz MA, Bernstein HD, Strub K, Zopf D, Wilhelm H, Walter P. 1990. *Science* 250:1111–17
38. Luirink J, High S, Wood H, Giner A, Tollervey D, Dobberstein B. 1992. *Nature* 359:741–43
39. Bernstein HD, Poritz MA, Strub K, Hoben PJ, Brenner S, Walter P. 1989. *Nature* 340:482–86
40. Zelazny A, Seluanov A, Cooper A, Bibi E. 1997. *Proc. Natl. Acad. Sci. USA* 94:6025–29
41. de Leeuw E, te Kaat K, Moser C, Menestrina G, Demel R, et al. 2000. *EMBO J.* 19:531–41
42. Angelini S, Deitermann S, Koch HG. 2005. *EMBO Rep.* 6:476–81
43. Shan SO, Walter P. 2005. *Biochemistry* 44:6214–22
44. Reyes CL, Rutenber E, Walter P, Stroud RM. 2007. *PLoS ONE* 2:e607
45. Valent QA, Scotti PA, High S, de Gier JW, von Heijne G, et al. 1998. *EMBO J.* 17:2504–12
46. Prinz A, Behrens C, Rapoport TA, Hartmann E, Kalies KU. 2000. *EMBO J.* 19:1900–6
47. Walter P, Johnson AE. 1994. *Annu. Rev. Cell Biol.* 10:87–119
48. Powers T, Walter P. 1997. *EMBO J.* 16:4880–86
49. Freymann DM, Keenan RJ, Stroud RM, Walter P. 1997. *Nature* 385:361–64
50. Newitt JA, Bernstein HD. 1997. *Eur. J. Biochem.* 245:720–29
51. Montoya G, Svensson C, Luirink J, Sinning I. 1997. *Nature* 385:365–68
52. Peluso P, Herschlag D, Nock S, Freymann DM, Johnson AE, Walter P. 2000. *Science* 288:1640–43

53. Macao B, Luirink J, Samuelsson T. 1997. Mol. Microbiol. 24:523–34
54. Bradshaw N, Walter P. 2007. Mol. Biol. Cell 18:2728–34
55. Keenan RJ, Freymann DM, Walter P, Stroud RM. 1998. Cell 94:181–91
56. Batey RT, Rambo RP, Lucast L, Rha B, Doudna JA. 2000. Science 287:1232–39
57. Schaffitzel C, Oswald M, Berger I, Ishikawa T, Abrahams JP, et al. 2006. Nature 444:503–6
58. Halic M, Gartmann M, Schlenker O, Mielke T, Pool MR, et al. 2006. Science 312:745–47
59. Lill R, Dowhan W, Wickner W. 1990. Cell 60:271–80
60. Miller A, Wang L, Kendall DA. 2002. Biochemistry 41:5325–32
61. Sharma V, Arockiasamy A, Ronning DR, Savva CG, Holzenburg A, et al. 2003. Proc. Natl. Acad. Sci. USA 100:2243–48
62. Papanikolau Y, Papadovasilaki M, Ravelli RB, McCarthy AA, Cusack S, et al. 2007. J. Mol. Biol. 366:1545–57
63. Zimmer J, Li W, Rapoport TA. 2006. J. Mol. Biol. 364:259–65
64. Vassylyev DG, Mori H, Vassylyeva MN, Tsukazaki T, Kimura Y, et al. 2006. J. Mol. Biol. 364:248–58
65. Koonin EV, Gorbalenya AE. 1992. FEBS Lett. 298:6–8
66. Kimura E, Akita M, Matsuyama S, Mizushima S. 1991. J. Biol. Chem. 266:6600–6
67. Papanikou E, Karamanou S, Baud C, Frank M, Sianidis G, et al. 2005. J. Biol. Chem. 280:43209–17
68. Cabelli RJ, Dolan KM, Qian LP, Oliver DB. 1991. J. Biol. Chem. 266:24420–27
69. Breukink E, Nouwen N, van Raalte A, Mizushima S, Tommassen J, de Kruijff B. 1995. J. Biol. Chem. 270:7902–7
70. Woodbury RL, Hardy SJ, Randall LL. 2002. Protein Sci. 11:875–82
71. Driessen AJM. 1993. Biochemistry 32:13190–97
72. Ye J, Osborne AR, Groll M, Rapoport TA. 2004. Biochim. Biophys. Acta 1659:1–18
73. Mitchell C, Oliver D. 1993. Mol. Microbiol. 10:483–97
74. Sato K, Mori H, Yoshida M, Mizushima S. 1996. J. Biol. Chem. 271:17439–44
75. Sianidis G, Karamanou S, Vrontou E, Boulias K, Repanas K, et al. 2001. EMBO J. 20:961–70
76. Karamanou S, Vrontou E, Sianidis G, Baud C, Roos T, et al. 1999. Mol. Microbiol. 34:1133–45
77. Karamanou S, Gouridis G, Papanikou E, Sianidis G, Gelis I, et al. 2007. EMBO J. 26:2904–14
78. Keramisanou D, Biris N, Gelis I, Sianidis G, Karamanou S, et al. 2006. Nat. Struct. Mol. Biol. 13:594–602
79. Osborne AR, Clemons WM Jr, Rapoport TA. 2004. Proc. Natl. Acad. Sci. USA 101:10937–42
80. Benach J, Chou YT, Fak JJ, Itkin A, Nicolae DD, et al. 2003. J. Biol. Chem. 278:3628–38
81. Or E, Navon A, Rapoport T. 2002. EMBO J. 21:4470–79
82. Musial-Siwek M, Rusch SL, Kendall DA. 2005. Biochemistry 44:13987–96
83. Jilaveanu LB, Zito CR, Oliver D. 2005. Proc. Natl. Acad. Sci. USA 102:7511–16
84. de Keyzer J, van der Sluis EO, Spelbrink RE, Nijstad N, de Kruijff B, et al. 2005. J. Biol. Chem. 280:35255–60
85. Karamanou S, Sianidis G, Gouridis G, Pozidis C, Papanikolau Y, et al. 2005. FEBS Lett. 579:1267–71
86. Pohlschroder M, Prinz WA, Hartmann E, Beckwith J. 1997. Cell 91:563–66
87. Mitra K, Schaffitzel C, Shaikh T, Tama F, Jenni S, et al. 2005. Nature 438:318–24

88. Breyton C, Haase W, Rapoport TA, Kuhlbrandt W, Collinson I. 2002. *Nature* 418:662–65  
99. Murphy CK, Beckwith J. 1994. *Proc. Natl. Acad. Sci. USA* 91:2557–61  
90. Harris CR, Silhavy TJ. 1999. *J. Bacteriol.* 181:3438–44  
91. Plath K, Mothes W, Wilkinson BM, Stirling CJ, Rapoport TA. 1998. *Cell* 94:795–807  
92. Saparov SM, Erlandson K, Cannon K, Schaetzky J, Schulman S, et al. 2007. *Mol. Cell* 26:501–9  
93. Kihara A, Akiyama Y, Ito K. 1995. *Proc. Natl. Acad. Sci. USA* 92:4532–36  
94. Cannon KS, Or E, Clemons WM Jr, Shibata Y, Rapoport TA. 2005. *J. Cell Biol.* 169:219–25  
94a. Bol R, de Wit JG, Driessen AJM. 2007. *J. Biol. Chem.* In press. doi/10.1074/jbc.M702140200  
95. Tam PC, Maillard AP, Chan KK, Duong F. 2005. *EMBO J.* 24:3380–88  
96. Li W, Schulman S, Boyd D, Erlandson K, Beckwith J, Rapoport TA. 2007. *Mol. Cell* 26:511–21  
97. Tani K, Tokuda H, Mizushima S. 1990. *J. Biol. Chem.* 265:17341–47  
98. de Keyzer J, van der Does C, Driessen AJM. 2002. *J. Biol. Chem.* 277:46059–65  
99. Tani K, Mizushima S. 1991. *FEBS Lett.* 285:127–31  
100. Bessonneau P, Besson V, Collinson I, Duong F. 2002. *EMBO J.* 21:995–1003  
101. Duong F. 2003. *EMBO J.* 22:4375–84  
102. Tziatzios C, Schubert D, Lotz M, Gundogan D, Betz H, et al. 2004. *J. Mol. Biol.* 340:513–24  
103. Meyer TH, Ménetret JF, Breitling R, Miller KR, Akey CW, Rapoport TA. 1999. *J. Mol. Biol.* 285:1789–800  
104. Manting EH, van der Does C, Remigy H, Engel A, Driessen AJM. 2000. *EMBO J.* 19:852–61  
105. Scheuring J, Braun N, Nothdurft L, Stumpf M, Veenendaal AK, et al. 2005. *J. Mol. Biol.* 354:258–71  
106. Mitra K, Frank J, Driessen AJM. 2006. *Nat. Struct. Mol. Biol.* 13:957–64  
107. Kaufmann A, Manting EH, Veenendaal AKJ, Driessen AJM, van der Does C. 1999. *Biochemistry* 38:9115–25  
108. Osborne AR, Rapoport TA. 2007. *Cell* 129:97–110  
109. Cheng Z, Jiang Y, Mandon EC, Gilmore R. 2005. *J. Cell Biol.* 168:67–77  
110. Raden D, Song W, Gilmore R. 2000. *J. Cell Biol.* 150:53–64  
111. van der Sluis EO, Nouwen N, Koch J, de Keyzer J, van der Does C, et al. 2006. *J. Mol. Biol.* 361:839–49  
112. Mori H, Ito K. 2006. *Proc. Natl. Acad. Sci. USA* 103:16159–64  
113. Nagamori S, Nishiyama K, Tokuda H. 2002. *J. Biochem.* 132:629–34  
114. Nishiyama K, Hanada M, Tokuda H. 1994. *EMBO J.* 13:3272–77  
115. Schatz PJ, Bieker KL, Ottemann KM, Silhavy TJ, Beckwith J. 1991. *EMBO J.* 10:1749–57  
116. Economou A, Wickner W. 1994. *Cell* 78:835–43  
117. Economou A, Pogliano JA, Beckwith J, Oliver DB, Wickner W. 1995. *Cell* 83:1171–81  
118. Mori H, Ito K. 2001. *Proc. Natl. Acad. Sci. USA* 98:5128–33  
119. de Keyzer J, Regeling A, Driessen AJM. 2007. *FEBS Lett.* 581:1859–64  
120. Matsumoto G, Mori H, Ito K. 1998. *Proc. Natl. Acad. Sci. USA* 95:13567–72  
121. Suzuki H, Nishiyama K, Tokuda H. 1998. *Mol. Microbiol.* 29:331–41  
122. Nishiyama K, Suzuki T, Tokuda H. 1996. *Cell* 85:71–81

123. van der Sluis EO, van der Vries E, Berrelkamp G, Nouwen N, Driessen AJM. 2006. *J. Bacteriol.* 188:1188–90

124. Bieker-Brady K, Silhavy TJ. 1992. *EMBO J.* 11:3165–74

125. Duong F, Wickner W. 1999. *EMBO J.* 18:3263–70

126. van der Wolk JPW, Fekkes P, Boorsma A, Huie JL, Silhavy TJ, Driessen AJM. 1998. *EMBO J.* 17:3631–39

127. Nouwen N, de Kruijff B, Tommassen J. 1996. *Proc. Natl. Acad. Sci. USA* 93:5953–57

128. Natale P, Swaving J, van der Does C, de Keyzer J, Driessen AJM. 2004. *J. Biol. Chem.* 279:13769–77

129. Shiozuka K, Tani K, Mizushima S, Tokuda H. 1990. *J. Biol. Chem.* 265:18843–47

130. Duong F, Wickner W. 1997. *EMBO J.* 16:2756–68

131. Gardel C, Johnson K, Jacq A, Beckwith J. 1990. *EMBO J.* 9:3209–16

132. Saier MHJ, Paulsen IT, Sliwinski MK, Pao SS, Skurray RA, Nikaido H. 1998. *FASEB J.* 12:265–74

133. Pogliano JA, Beckwith J. 1994. *EMBO J.* 13:554–61

134. Arkowitz RA, Wickner W. 1994. *EMBO J.* 13:954–63

135. Nouwen N, Piwowarek M, Berrelkamp G, Driessen AJM. 2005. *J. Bacteriol.* 187:5857–60

136. Matsuyama S, Fujita Y, Mizushima S. 1993. *EMBO J.* 12:265–70

137. Duong F, Wickner W. 1997. *EMBO J.* 16:4871–79

138. Hand NJ, Klein R, Laskewitz A, Pohlschroder M. 2006. *J. Bacteriol.* 188:1251–59

139. Driessen AJM. 1992. *EMBO J.* 11:847–53

140. Schiebel E, Driessen AJM, Hartl FU, Wickner W. 1991. *Cell* 64:927–39

141. van der Wolk JPW, de Wit JG, Driessen AJM. 1997. *EMBO J.* 16:7297–304

142. van Dalen A, Killian A, de Kruijff B. 1999. *J. Biol. Chem.* 274:19913–18

143. Nouwen N, Tommassen J, de Kruijff B. 1994. *J. Biol. Chem.* 269:16029–33

144. Tomkiewicz D, Nouwen N, van Leeuwen R, Tans S, Driessen AJM. 2006. *J. Biol. Chem.* 281:15709–13

145. Sato K, Mori H, Yoshida M, Tagaya M, Mizushima S. 1997. *J. Biol. Chem.* 272:5880–86

146. Driessen AJM, Wickner W. 1991. *Proc. Natl. Acad. Sci. USA* 88:2471–75

147. Nishiyama K, Fukuda A, Morita K, Tokuda H. 1999. *EMBO J.* 18:1049–58

148. Arkowitz RA, Joly JC, Wickner W. 1993. *EMBO J.* 12:243–53

149. Nouwen N, Berrelkamp G, Driessen AJM. 2007. *J. Mol. Biol.* 372:422–33

150. Eser M, Ehrmann M. 2003. *Proc. Natl. Acad. Sci. USA* 100:13231–34

151. Eichler J, Wickner W. 1997. *Proc. Natl. Acad. Sci. USA* 94:5574–81

152. van der Does C, Manting EH, Kaufmann A, Lutz M, Driessen AJM. 1998. *Biochemistry* 37:201–10

153. van der Does C, den Blaauwen T, de Wit JG, Manting EH, Groot NA, et al. 1996. *Mol. Microbiol.* 22:619–29

154. Jilaveanu LB, Oliver DB. 2007. *J. Biol. Chem.* 282:4661–68

155. Tomkiewicz D, Nouwen N, Driessen AJM. 2007. *FEBS Lett.* 581:2820–28

156. Simon SM, Peskin CS, Oster GF. 1992. *Proc. Natl. Acad. Sci. USA* 89:3770–74

157. Velankar SS, Soultanas P, Dillingham MS, Subramanya HS, Wigley DB. 1999. *Cell* 97:75–84

158. von Heijne GV. 1986. *EMBO J.* 5:3021–27

159. Andersson H, von Heijne G. 1994. *EMBO J.* 13:2267–72

160. van Klompenburg W, Nilsson I, von Heijne G, de Kruijff B. 1997. *EMBO J.* 16:4261–66

161. Saaf A, Wallin E, von Heijne G. 1998. *Eur. J. Biochem.* 251:821–29

162. Werner PK, Saier MH Jr, Muller M. 1992. *J. Biol. Chem.* 267:24523–32  
163. MacFarlane J, Muller M. 1995. *Biochem. Soc. Trans.* 23:560S  
164. Saaf A, Andersson H, Gafvelin G, von Heijne G. 1995. *Mol. Membr. Biol.* 12:209–15  
165. Duong F, Wickner W. 1998. *EMBO J.* 17:696–705  
166. Sato K, Mori H, Yoshida M, Tagaya M, Mizushima S. 1997. *J. Biol. Chem.* 272:20082–87  
167. Yamane K, Akiyama Y, Ito K, Mizushima S. 1990. *J. Biol. Chem.* 265:21166–71  
168. Hessa T, Kim H, Bihlmaier K, Lundin C, Boekel J, et al. 2005. *Nature* 433:377–81  
169. Beck K, Eisner G, Trescher D, Dalbey RE, Brunner J, Muller M. 2001. *EMBO Rep.* 2:709–14  
169a. Wang X, Bogdanov M, Dowhan W. 2002. *EMBO J.* 21:5673–81  
170. van der Laan M, Nouwen NP, Driessen AJM. 2005. *Curr. Opin. Microbiol.* 8:182–87  
171. Dalbey RE, Kuhn A. 2004. *J. Cell Biol.* 166:769–74  
172. Scotti PA, Urbanus ML, Brunner J, de Gier JW, von Heijne G, et al. 2000. *EMBO J.* 19:542–49  
173. Nouwen N, Driessen AJM. 2002. *Mol. Microbiol.* 44:1397–405  
174. Samuelson JC, Jiang F, Yi L, Chen M, de Gier JW, et al. 2001. *J. Biol. Chem.* 276:34847–52  
175. Chen M, Xie K, Jiang F, Yi L, Dalbey RE. 2002. *Biol. Chem.* 383:1565–72  
176. van der Laan M, Bechtluft P, Kol S, Nouwen N, Driessen AJM. 2004. *J. Cell Biol.* 165:213–22  
177. Serek J, Bauer-Manz G, Struhalla G, Van den Berg L, Kiefer D, et al. 2004. *EMBO J.* 23:294–301  
178. Nagamori S, Smirnova IN, Kaback HR. 2004. *J. Cell Biol.* 165:53–62  
179. Szyrach G, Ott M, Bonnefoy N, Neupert W, Herrmann JM. 2003. *EMBO J.* 22:6448–57  
180. Shiomi D, Yoshimoto M, Homma M, Kawagishi I. 2006. *Mol. Microbiol.* 60:894–906  
181. Campo N, Tjalsma H, Buist G, Stepniak D, Meijer M, et al. 2004. *Mol. Microbiol.* 53:1583–99  
182. Brandon LD, Goehring N, Janakiraman A, Yan AW, Wu T, et al. 2003. *Mol. Microbiol.* 50:45–60  
183. Rosch J, Caparon M. 2004. *Science* 304:1513–15

AR345-FM    ARI    7 May 2008    14:43

$\mathbf{A R}$

**Annual Review of Biochemistry**

Volume 77, 2008

---

## Contents

### Prefatory Chapters

- **Discovery of G Protein Signaling**  
  *Zvi Selinger* ………… 1

- **Moments of Discovery**  
  *Paul Berg* ………… 14

### Single-Molecule Theme

- **In singulo Biochemistry: When Less Is More**  
  *Carlos Bustamante* ………… 45

- **Advances in Single-Molecule Fluorescence Methods for Molecular Biology**  
  *Chirlmin Joo, Hamza Balci, Yuji Ishitsuka, Chittanon Buranachai, and Taekjip Ha* ………… 51

- **How RNA Unfolds and Refolds**  
  *Pan T.X. Li, Jeffrey Vieregg, and Ignacio Tinoco, Jr.* ………… 77

- **Single-Molecule Studies of Protein Folding**  
  *Alessandro Borgia, Philip M. Williams, and Jane Clarke* ………… 101

- **Structure and Mechanics of Membrane Proteins**  
  *Andreas Engel and Hermann E. Gaub* ………… 127

- **Single-Molecule Studies of RNA Polymerase: Motoring Along**  
  *Kristina M. Herbert, William J. Greenleaf, and Steven M. Block* ………… 149

- **Translation at the Single-Molecule Level**  
  *R. Andrew Marshall, Colin Echeverría Aitken, Magdalena Dorywalska, and Joseph D. Puglisi* ………… 177

- **Recent Advances in Optical Tweezers**  
  *Jeffrey R. Moffitt, Yann R. Chemla, Steven B. Smith, and Carlos Bustamante* ………… 205

### Recent Advances in Biochemistry

- **Mechanism of Eukaryotic Homologous Recombination**  
  *Joseph San Filippo, Patrick Sung, and Hannah Klein* ………… 229

Structural and Functional Relationships of the XPF/MUS81 Family of Proteins  
Alberto Ciccia, Neil McDonald, and Stephen C. West  . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Maturation of Iron-Sulfur Proteins in Eukaryotes: Mechanisms,  
Connected Processes, and Diseases  
Roland Lill and Ulrich Mühlenhoff ………………………………………… 669  

CFTR Function and Prospects for Therapy  
John R. Riordan ……………………………………………………………… 701  

Aging and Survival: The Genetics of Life Span Extension  
by Dietary Restriction  
William Mair and Andrew Dillin ………………………………………… 727  

Cellular Defenses against Superoxide and Hydrogen Peroxide  
James A. Imlay ……………………………………………………………… 755  

Toward a Control Theory Analysis of Aging  
Michael P. Murphy and Linda Partridge ………………………………… 777  

Indexes  

Cumulative Index of Contributing Authors, Volumes 73–77 …………… 799  
Cumulative Index of Chapter Titles, Volumes 73–77 …………………… 803  

Errata  

An online log of corrections to *Annual Review of Biochemistry* articles may be found  
at http://biochem.annualreviews.org/errata.shtml
