
APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Feb. 2008, p. 1294–1298
0099-2240/08/$08.00+0 doi:10.1128/AEM.02660-07
Copyright © 2008, American Society for Microbiology. All Rights Reserved.

**Determination of Glycosyltransferase Specificities for the *Escherichia coli* O111 O Antigen by a Generic Approach**

Gordon Stevenson, Manuela Dieckelmann, and Peter R. Reeves*

School of Molecular and Microbial Biosciences, University of Sydney, Sydney, New South Wales 2006, Australia

Received 23 November 2007/Accepted 27 November 2007

We describe a bacterial strain developed to facilitate the determination of glycosyltransferase (GT) specificities for O antigens of known structure and gene cluster sequence. For proof of principle for the approach, the strain was used to determine the specificity of the *Escherichia coli* O111 O-antigen GT genes.

The O antigen is a repeat unit polysaccharide that is part of the lipopolysaccharide (LPS). It is very diverse, with about 190 forms known for *Escherichia coli* (including *Shigella* strains) (5) and 46 for *Salmonella enterica* (19). The genes responsible for each O-antigen form are generally in a gene cluster that maps to a specific locus, which is between *galF* and *gnd* for *E. coli* (22). The structure and O-antigen gene cluster of *E. coli* O111 are shown in Fig. 1.

In *E. coli*, all but a few of the O-antigen repeat units (O units) are synthesized by the Wzy-dependent mechanism on a carrier lipid (22, 26). The first sugar is added as a sugar phosphate to undecaprenyl phosphate (UndP) to give a pyrophosphate linkage. The other sugars are added sequentially by glycosyltransferases (GTs) from nucleotide diphospho (NDP) sugars (22). The completed O unit is then translocated to the outer face of the membrane and polymerized on undecaprenol pyrophosphate (UndPP) before being added to lipid A/core, which has been synthesized and translocated separately (9, 26) (Fig. 2). The genes generally fall into three classes, those for the synthesis of precursors of O-antigen components, those for the assembly of the O unit (mostly GTs), and those for processing the O unit (22). The first class comprises genes for the synthesis of precursors for sugars and other components not otherwise available in cells of that species. Many of these genes have been characterized and are usually recognizable in the gene cluster. The third class comprises genes for processing the O unit and for Wzy-dependent synthesis; they comprise *wzx* for translocation, *wzy* for polymerization, and *wzz* for the determination of O-antigen chain length. *Wzz* is generally conserved in sequence and easily recognized, while *Wzx* and *Wzy*, although very variable in sequence, both have a recognizable conserved secondary structure (22). Thus, genes of the first and third classes are generally identifiable from sequence alone, whereas genes for the assembly of the repeat unit are often identified in only the generic sense as GT genes, but not in terms of the linkage involved.

It is now common to assign function to all but the GT genes in the sequence of a repeat unit gene cluster. The problem is the enormous diversity of glycosidic linkages, with about 20 known sugars in *E. coli* O antigens and many different combinations of donor sugar, acceptor sugar, and anomerism (24). The number is enormously greater if we consider all repeat-unit polysaccharides. GTs have been grouped in various ways, of which the Pfam (12) and Cazy (4) groupings are the most useful for identifying GT genes in a generic sense. However, they provide limited guidance for allocating a GT to a specific linkage, and only rarely is it possible to do the assignment with confidence from sequence comparisons alone. This inability to assign GT function is the major limitation in our interpretation of gene function for bacterial polysaccharides.

The few GT genes that have been allocated to a specific linkage were mostly characterized biochemically by demonstrating a specific GT activity (e.g., see reference 17), which is difficult as the substrates are activated precursors (usually an NDP sugar) of which many are not generally available and the acceptor is usually an incomplete O unit, also not readily available. Even fewer have been characterized by individually mutating the relevant genes and characterizing the intermediate that accumulates (e.g., see reference 21), which has its own problems as discussed below.

In this paper, we describe a protocol to determine GT specificity that can be applied generally in *E. coli*. The protocol uses an *E. coli* K-12 tester strain (P5814) into which O-antigen gene clusters can be cloned for determination of their GT specificities. The principle is to determine the mass of the intermediates that accumulate after mutating each of the putative GT genes and in this way determine the order in which they act. This, in conjunction with the structure, can serve to allocate a function to each GT with reasonable confidence. Strain P5814 was created from *E. coli* K-12 strain JM109 (lab name P4971), *endA1 recA1 gyrA96 thi hsdR17 relA1 supE44 Δ(lac-proAB)* (*F' traD36 proAB laqI<sup>q</sup>ZΔM15*) (31) by making a series of deletions or substitutions to resolve two problems that had to be addressed.

The first problem is that GTs can carry out reactions that are additional to their usual reaction if alternate substrates are available. *E. coli* K-12 has three major polysaccharides, and it is quite possible that an expected intermediate will not accumulate but will instead be used as a substrate by another GT in the absence of competition from the original GT. An example is the formation in a clone of *E. coli* O4 DNA in *E. coli* K-12

* Corresponding author. Mailing address: School of Molecular and Microbial Biosciences, University of Sydney, Sydney, NSW 2006, Australia. Phone: 612-93512536. Fax: 612-93514571. E-mail: reeves@angis.usyd.edu.au.
† Supplemental material for this article may be found at http://aem.asm.org/.
∇ Published ahead of print on 21 December 2007.

Vol. 74, 2008

GLYCOSYL TRANSFERASE SPECIFICITY 1295

$\begin{aligned} & \text { Col } \\ & \alpha \mid_{1,3} \\ & \downarrow \\ & \left[\mathrm{Glc}^{\alpha}_{1,4} \mathrm{Gal}^{\alpha}_{1,3} \mathrm{GlcNac}\right] \beta \rightarrow \\ & \alpha \mid_{1,6} \\ & \text { Col } \end{aligned}$

| wbdH | gmd | manC | manB | colB | colA | wzx | wzy | wbdL | wbdM |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13Kb |

FIG. 1. The E. coli O111 O antigen and gene cluster. At the top is the structure (based on data from reference 24). At the bottom is a depiction of the gene cluster showing GDP-colitose pathway genes manB, manC, colA, and colB; processing genes wzx and wzy; and GT genes wbdH, wbdL, and wbdM (adapted from reference 27). Molecular sizes are indicated below.

of a novel repeat unit involving GTs from both donor and host (16, 25). This is a risk whenever an incomplete gene cluster is generated.

We therefore deleted the gene clusters for enterobacterial common antigen (ECA), O-antigen, and colanic acid (CA) synthesis by using the methods of Datsenko and Wanner (6) and Kang et al. (15), using the lambda Red recombinase. The kanamycin resistance cassette of pKD4 (6) was amplified using oligonucleotides that include segments homologous to DNA flanking the region to be replaced, together with an I-SceI site in one primer and FRT sites (FLP recognition sites) in both (Table 1; see the supplemental material for all primer details). The PCR products of primer pairs 5464 and 5465, 5460 and 5461, and 5564 and 5565 were used to replace the ECA, O-antigen, and CA gene clusters, respectively, with a kanamycin cassette by homologous recombination. Each operation required the addition of pKD46 to provide the lambda recombinase and its induction by arabinose prior to electroporation of the appropriate PCR product and selection of the deletion strain with kanamycin.

In each case, the kan cassette was then removed by introducing a bridging sequence (a short double-stranded DNA) comprising sequences that now flank the inserted kanamycin cassette, such that homologous recombination will remove it. The bridging sequences were made using oligonucleotide pairs 5512 and 5513, 5555 and 5556, or 5566 and 5567, respectively, for ECA, O antigen, or CA. Each pair hybridized via complementary sequences at their ends, and extension with Klenow fragment gave double-stranded DNA with each end suitable for recombination into one of the flanking regions. Plasmid pBC-I-SceI, which contains the cat gene for chloramphenicol resistance and constitutively expresses the I-SceI nuclease (15), was introduced with the bridging sequence to provide selection against the presence of the kan cassette by cutting the associated I-SceI sites. pBC-I-SceI was then removed by screening colonies for a chloramphenicol-sensitive variant.

The second problem in mutating GT genes is that the intermediate that accumulates may be damaging to the cell. This has been studied in S. enterica LT2 for both O antigen (18, 32) and ECA synthesis (20). The work cited was done in S. enterica but applies to three intermediates and is probably a widespread phenomenon. To avoid this problem, we put the initiation of O-unit synthesis under tight control so that it would not start until induced. Most E. coli strains have GlcNAc or

Periplasm

O unit

$\begin{array}{lllllll} 1 & 2 & 3 & 4 & 5 & \text { Lipid A } \\ \hline \text { P } & \text { P } & \text { P } & \text { P } & \text { core } \\ \hline \end{array}$

$\begin{array}{llllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll
TABLE 1. Oligonucleotides used to make DNA segments for chromosomal insertions and replacements$^a$

| Oligonucleotide no. | Primer direction | Length (bases) | Function | Gene or cluster affected | Comments |
|---------------------|------------------|----------------|----------|-------------------------|----------|
| 5504                | F                | 62             | kan cassette insertion     | rhaBAD   | 42bK-12(4095513–4095472) + 20bpKD4 (31–50) |
| 5505                | R                | 79             | kan cassette insertion     | rhaBAD   | 42bK-12(4091429–4091470) + I-SceI+19bpKD4 (1525–1507) |
| 5506                | F                | 62             | wecA replaces rhaBAD       | rhaBAD   | 45bK-12(4095513–4095469) + 17bK-12(3965942–3965958) |
| 5507                | R                | 62             | wecA replaces rhaBAD       | rhaBAD   | 45bK-12(4091429–4091473) + 17bK-12(3967039–3967023) |
| 5564                | F                | 62             | kan cassette insertion     | CA       | 42bK-12(2135542–2135501) + 20bpKD4 (31–50) |
| 5565                | R                | 79             | kan cassette insertion     | CA       | 42bK-12(3978839–2118182) + I-SceI+19bpKD4 (1525–1507) |
| 5566                | F                | 52             | kan cassette removal       | CA       | 42bK-12(2135542–2135501) + 10bK-12(2113963–2113972) |
| 5567                | R                | 52             | kan cassette removal       | CA       | 42bK-12(2113931–2113972) + 10bK-12(2135501–2135492) |
| 5464                | F                | 80             | kan cassette insertion     | ECA      | 42bK-12(3965820–3965861) + I-SceI+20bpKD4 (31–50) |
| 5465                | R                | 62             | kan cassette insertion     | ECA      | 42bK-12(3978839–3978798) + 20kbpKD4 (1526–1507) |
| 5512                | F                | 50             | kan cassette removal       | ECA      | 40bK-12(3965660–3965699) + 10bK-12(397801–397810) |
| 5513                | R                | 50             | kan cassette removal       | ECA      | 40bK-12(3978839–3978800) + 10bK-12(3965699–3965690) |
| 5460                | F                | 80             | kan cassette insertion     | OAg      | 42bK-12(2110912–2110871) + I-SceI+20bpKD4 (31–50) |
| 5461                | R                | 62             | kan cassette insertion     | OAg      | 42bK-12(2099651–2099692) + 29kbpKD4 (1526–1507) |
| 5555                | F                | 63             | kan cassette removal       | OAg      | 43bK-12(2111499–2111458) + 20bK-12(2099292–2099273) |
| 5556                | R                | 62             | kan cassette removal       | OAg      | 42bK-12(2099251–2099292) + 20bK-12(2111458–2111477) |
| 5603                | F                | 61             | cat cassette insertion     | wbdH     | 42bO111(697–738) + 19bpKD3P2 (1525–1507) |
| 5604                | R                | 62             | cat cassette insertion     | wbdH     | 45bO111(1974–1930) + 17bpKD3P1 (137–153) |
| 3621                | F                | 55             | cat cassette insertion     | wbdL     | 34bO111(10918–10951) + 21bpKD3 (773–753) |
| 3622                | R                | 55             | cat cassette insertion     | wbdL     | 34bO111(11855–11822) + 21bpKD3 (115–134) |
| 5759                | F                | 99             | cat cassette insertion     | wbdM     | 80bO111(11746–11825) + 19bpKD3P2 (1525–1507) |
| 5760                | R                | 100            | cat cassette insertion     | wbdM     | 80bO111(13025–12946) + 20bpKD3P1 (31–50) |
| 5598                | F                | 79             | cat cassette insertion     | wcaJ     | 42bK-12(2119620–2119579) + I-SceI+19bpKD3P2 (1525–1507) |
| 5597                | R                | 80             | cat cassette insertion     | wcaJ     | 42bK-12(2118141–2118182) + I-SceI+20bpKD3P1 (31–50) |

$^a$ F, forward; R, reverse; b, base; OAg, O antigen. GenBank accession numbers: *E. coli* K-12, U00096.2; *E. coli* O111, AF078736; pKD3, AY048742; pKD4, AY048743.P1 and P2 sequences are primers used by Datsenko and Wanner (6). I-SceI is the 18-bp sequence TAGGGATAACAGGGTAA. The oligonucleotide DNA sequences are in Table S2 in the supplemental material.

wecA gene flanked by sequences based on the segments upstream of the rhaB gene and downstream of rhaD (generated using primers 5506 and 5507) was incorporated using the Red system, such that the start codon of wecA was in the same position as that of rhaB previously, to give a cell in which the expression of wecA would fall under the rhaBAD promoter, which is tightly controlled by RhaS and inducible by rhamnose (30).

Strain P5814 is used by introducing into it a series of plasmids that carry a cloned O-antigen gene cluster and then generating nonpolar mutations separately in each putative GT gene. In the absence of the complete O antigen, the intermediate should be added to lipid A/core as shown by Feldman et al. (10), and the relative size of the LPS and, hence, of the O-unit intermediate can be determined by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) after inducing O-unit synthesis in each of the knockout strains.

We used a derivative of plasmid pPR1231 (2) that carries the *E. coli* O111 gene cluster to establish the utility of the tester strain. pPR1231 also carries part of the CA gene cluster, including wcaJ, the CA IT gene. We found that wcaJ interfered with our assays, and so replaced it with a cat cassette already flanked by FRT sites using primers 5597 and 5598 and the Red system as described above, followed by removal of the cassette by transformation in plasmid pCP-20, which carries the FLP recombinase to excise DNA between FRT sites (6). Ampicillin-resistant colonies were screened to find a chloramphenicol-sensitive colony, which was named pPR2105 and electroporated into P5814.

After rhamnose induction, strain P5814(pPR2105) gave a typical O-antigen ladder (Fig. 3A, lane 2), but in the absence of

FIG. 3. (A) Rhamnose induction of wecA and effect on O-antigen synthesis. Results of SDS-PAGE of LPS from strain P5814 carrying the O111 gene cluster (pPR2105). Lane 1, not induced by rhamnose; lane 2, rhamnose induced. (B) Results of SDS-PAGE of LPS from P5814 carrying pPR2102 and derivatives. Lanes 1 to 4, all with rhamnose induction, contain strain P5814 carrying, in lane 1, no plasmid; lane 2, pPR2102; lane 3, pPR2104; lane 4, pPR2103. Lane 5, strain P5814 carrying pPR2105 with no rhamnose induction. Arrows indicate direction of electrophoresis. Arrowheads indicate bands comprising LPS with a single complete or incomplete O unit or O-antigen polymer that are referred to in the text. Note that lane 5 is the same as lane 1 in Fig. 3A.

GT knockout strains, there should be no significant accumulation of intermediate.

The O111 O unit (Fig. 1) has five sugar residues, comprising GlcNAc, galactose, glucose, and two colitose residues. The *wzx* and *wzy* genes and genes for the synthesis of GDP-colitose are identified (Fig. 2), but genes for UDP-Glc, UDP-GlcNAc, and UDP-Gal are housekeeping genes and map elsewhere (27). The GlcNAc residue is added by WecA as GlcNAc-P, and four other GT steps are required. However, the O111 gene cluster has only three putative GT genes, and it is presumed that both colitose residues are added by one GT (27).

We used the Red system to replace, separately, *wbdH*, *wbdL*, and *wbdM*, the three putative GT genes in pPR2105, with a chloramphenicol cassette, using primer pairs 5603 and 5604, 3621 and 3622, and 5759 and 5760 to generate plasmids pPR2102, pPR2103, and pPR2104, respectively. We looked at the intermediates that accumulated after the addition of rhamnose by using SDS-PAGE (17% acrylamide) and phenol-water extracts of LPS (14) with silver staining (Fig. 3B). In the absence of the O111 clone, a band of lipidA/core-GlcNAc could be seen (lane 1). The deletion of *wbdH* (lane 2) gives the same pattern as seen before the O111 clone is added, indicating that *wbdH* is responsible for the addition of the O111 second sugar (in this case, galactose), as only lipidA/core and lipidA/core-GlcNAc are made. This is as expected due to the good homology with *wbbP*, an *S.* dysenteriae gene that carries out the same function as WbdH, adding galactose as the second sugar to UndPP-GlcNAc (13). The deletion of *wbdM* or *wbdL* gives larger fragments (Fig. 3B, lanes 3 and 4, respectively). The inferred order of function is WbdH, WbdM, and WbdL, as this is the order of increasing size of the intermediates made by the mutants (Fig. 3B). Figure 3, lane 5, shows the full O unit.

We conclude that WecA, WbdH, WbdM, and WbdL add, respectively, GlcNAc, Gal, Glc, and colitose. From the spacing of the bands, it appears that in the absence of *wbdM*, no colitose is added and that WbdM is responsible for the addition of both colitose moieties. There is the possibility of the mutations being polar and preventing the expression of genes downstream, although this is not expected from the nature of the deletions. However, for *wbdH*, we confirmed that this was not the case by transfer of the plasmid pWL1059, which carries a cloned *wbbP* gene (11), and showed that it complemented the *wbdH* mutation, allowing the synthesis of full LPS (data not shown). This was important as the phenotype of the *wbdH* mutant is the same as that of P5814 without the pWL1059 plasmid, and the complementation shows that, while *wbdH* is inactivated in pPR2102, *wbdM* and *wbdL* are still functional. In the case of *wbdM* and *wbdL*, we conclude that the mutation in *wbdL* is not polar on *wbdM*, as the *wbdL* mutant produces a larger intermediate than the *wbdM* mutant and therefore cannot have fewer functions than the *wbdM* mutant. For *wbdM*, the matter does not arise as it is the last gene in the gene cluster. It is interesting that, after rhamnose induction, strains carrying mutations in *wbdM* or *wbdL* grew poorly, with indications of lysis, suggesting that the accumulation of intermediates was indeed damaging.

The tester strain P5814 should be applicable to all *E.* coli strains (including Shigella) that have GlcNAc or GalNAc as the first sugar. Some *E.* coli strains that include GalNAc have a *gne* gene in the gene cluster, but it can also be outside, as in *E.* coli O157 (28), and would then require the addition of a *gne* gene. P5814 should also be useful as a tester strain for *S.* enterica and Yersinia spp., as most of these use GlcNAc or GalNAc as the initial sugar with WecA as the IT and are expected to express the O-antigen gene cluster when cloned into *E.* coli K-12, as shown experimentally for the more divergent Yersinia (3). Extension to species with different initial sugars would require modification of the tester strain to change the IT gene, but in principle, the strategy could be adapted to a range of species and to capsules as well as O antigens.

We thank L. Feng and L. Wang for providing plasmid pWL1059. We thank the Australian Research Council for grant support.

### REFERENCES

1. Alexander, D. C., and M. A. Valvano. 1994. Role of the *rfe* gene in the biosynthesis of the *Escherichia coli* O7-specific lipopolysaccharide and other O-specific polysaccharides containing N-acetylglucosamine. J. Bacteriol. 176: 7079–7084.
2. Bastin, D. A., L. K. Romana, and P. R. Reeves. 1991. Molecular cloning and expression in *Escherichia coli* K-12 of the *rfb* gene cluster determining the O antigen of an *E.* coli O111 strain. Mol. Microbiol. 5:2223–2231.
3. Bengoechea, J. A., E. Pinta, T. Salminen, C. Oertelt, O. Holst, J. Radziejewska-Lebrecht, Z. Piotrowska-Seget, R. Venho, and M. Skurnik. 2002. Functional characterization of Gne (UDP-N-acetylglucosamine-4-epimerase), Wzz (chain length determinant), and Wzy (O-antigen polymerase) of *Yersinia enterocolitica* serotype O:8. J. Bacteriol. 184:4277–4287.
4. Coutinho, P. M., E. Deleury, G. J. Davies, and B. Henrissat. 2003. An evolving hierarchical family classification for glycosyltransferases. J. Mol. Biol. 328:307–317.
5. Cunneen, M., and P. R. Reeves. 2007. The *Yersinia kristensenii* O11 O antigen gene cluster was acquired by lateral gene transfer from an *Escherichia coli*-like ancestor and incorporated at a novel chromosomal locus. Mol. Biol. Evol. 24:1355–1365.
6. Datsenko, K. A., and B. L. Wanner. 2000. One-step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. Proc. Natl. Acad. Sci. USA 97:6640–6645.
7. Dean, C. R., C. V. Franklund, J. D. Retief, M. J. Coyne, Jr., K. Hatano, D. J. Evans, G. B. Pier, and J. B. Goldberg. 1999. Characterization of the serotype O11 O-antigen locus of *Pseudomonas aeruginosa* PA103. J. Bacteriol. 181: 4275–4284.
8. DebRoy, C., P. M. Fratamico, E. Roberts, M. A. Davis, and Y. Liu. 2005. Development of PCR assays targeting genes in O-antigen gene clusters for detection and identification of *Escherichia coli* O45 and O55 serogroups. Appl. Environ. Microbiol. 71:4919–4924.
9. Doerrler, W. T., and C. R. H. Raetz. 2002. ATPase activity of the MsbA lipid flippase of *Escherichia coli*. J. Biol. Chem. 277:36697–36705.
10. Feldman, M. F., C. L. Marolda, M. A. Monteiro, M. B. Perry, A. J. Parodi, and M. A. Valvano. 1999. The activity of a putative polyisoprenol-linked sugar translocase (*Wzx*) involved in *Escherichia coli* O antigen assembly is independent of the chemical structure of the O repeat. J. Biol. Chem. 274:35129–35138.
11. Feng, L., A. V. Perepelov, G. Zhao, S. D. Shevelev, Q. Wang, S. N. Senchenkova, A. S. Shashkov, Y. Geng, P. R. Reeves, Y. A. Knirel, and L. Wang. 2007. Structural and genetic evidence that the *Escherichia coli* O148 O antigen is the precursor of the *Shigella dysenteriae* type 1 O antigen and identification of a glycosyltransferase gene. Microbiology 153:139–147.
12. Finn, R. D., J. Mistry, B. Schuster-Bockler, S. Griffiths-Jones, V. Hollich, T. Lassmann, S. Moxon, M. Marshall, A. Khanna, R. Durbin, S. R. Eddy, E. L. Sonnhammer, and A. Bateman. 2006. Pfam: clans, web tools and services. Nucleic Acids Res. 34:D247–D251.
13. Gohmann, S., P. A. Manning, C. A. Alpert, M. J. Walker, and K. N. Timmis. 1994. Lipopolysaccharide O-antigen biosynthesis in *Shigella dysenteriae* serotype1: analysis of the plasmid-carried *rfp* determinant. Microb. Pathog. 16:53–64.
14. Guard-Petter, J., B. Lakshmi, R. Carlson, and K. Ingram. 1995. Characterization of lipopolysaccharide heterogeneity in *Salmonella enteritidis* by an improved gel electrophoresis method. Appl. Environ. Microbiol. 61:2845–2851.
15. Kang, Y., T. Durfee, J. D. Glasner, Y. Qiu, D. Frisch, K. M. Winterberg, and F. R. Blattner. 2004. Systematic mutagenesis of the *Escherichia coli* genome. J. Bacteriol. 186:4921–4930.
16. Kogan, G., G. Haraguchi, S. I. Hull, R. A. Hull, A. S. Shashkov, B. Jann, and K. Jann. 1993. Structural analysis of O4-reactive polysaccharides from recombinant *Escherichia coli*: changes in the O-specific polysaccharide induced by cloning of the *rfb* genes. Eur. J. Biochem. 214:259–265.
17. Liu, D., A. M. Haase, L. Lindqvist, A. A. Lindberg, and P. R. Reeves. 1993. Glycosyl transferases of O-antigen biosynthesis in *Salmonella enterica*: iden-

18. Mäkelä, P. H., and B. A. D. Stocker. 1984. Genetics of lipopolysaccharide, p. 59–137. *In* E. T. Rietschel (ed.), Handbook of endotoxins, vol. I. Elsevier Science Publishers, Amsterdam, The Netherlands.

19. Popoff, M. Y., and L. Le Minor. 2001. Antigenic formulas of the *Salmonella* serovars, 8th revision. WHO Collaborating Centre for Reference and Research on *Salmonella*. Institut Pasteur, Paris, France.

20. Rick, P. D., S. Wolski, K. Barr, S. Ward, and L. Ramsay-Sharer. 1988. Accumulation of a lipid-linked intermediate involved in enterobacterial common antigen synthesis in *Salmonella typhimurium* mutants lacking dTDP-glucose pyrophosphorylase. J. Bacteriol. **170**:4008–4014.

21. Riley, J. G., M. Menggad, P. J. Montoya-Peleaz, W. A. Szarek, C. L. Marolda, M. A. Valvano, J. S. Schutzbach, and I. Brockhausen. 2005. The *wbbD* gene of *E. coli* strain VW187 (O7:K1) encodes a UDP-Gal:GlcNAcα-pyrophosphate-R β1,3-galactosyltransferase involved in the biosynthesis of O7-specific lipopolysaccharide. Glycobiology **15**:605–613.

22. Samuel, G., and P. R. Reeves. 2003. Biosynthesis of O-antigens: genes and pathways involved in nucleotide sugar precursor synthesis and O-antigen assembly. Carbohydr. Res. **338**:2503–2519.

23. Shepherd, J. G., L. Wang, and P. R. Reeves. 2000. Comparison of O-antigen gene clusters of *Escherichia coli* (*Shigella*) Sonnei and *Plesiomonas shigelloides* O17: Sonnei gained its current plasmid-borne O-antigen genes from *P. shigelloides* in a recent event. Infect. Immun. **68**:6056–6061.

24. Stenutz, R., A. Weintraub, and G. Widmalm. 2006. The structures of *Escherichia coli* O-polysaccharide antigens. FEMS Microbiol. Rev. **30**:382–403.

25. Stevenson, G., B. Neal, D. Liu, M. Hobbs, N. H. Packer, M. Batley, J. W. Redmond, L. Lindquist, and P. R. Reeves. 1994. Structure of the O-antigen of *Escherichia coli* K-12 and the sequence of its *rfb* gene cluster. J. Bacteriol. **176**:4144–4156.

26. Valvano, M. A. 2003. Export of O-specific lipopolysaccharide. Front. Biosci. **8**:S452–S471.

27. Wang, L., H. Curd, W. Qu, and P. R. Reeves. 1998. Sequencing of *Escherichia coli* O111 O-antigen gene cluster and identification of O111-specific genes. J. Clin. Microbiol. **36**:3182–3187.

28. Wang, L., S. Huskic, A. Cisterne, D. Rothemund, and P. R. Reeves. 2002. The O-antigen gene cluster of *Escherichia coli* O55:H7 and identification of a new UDP-GlcNAc C4 epimerase gene. J. Bacteriol. **184**:2620–2625.

29. Wang, W., A. V. Perepelov, L. Feng, S. D. Shevelev, Q. Wang, S. N. Senchenkova, W. Han, Y. Li, A. S. Shashkov, Y. A. Knirel, P. R. Reeves, and L. Wang. 2007. A group of *Escherichia coli* and *Salmonella enterica* O antigen lipopolysaccharides sharing a common backbone structure. Microbiology **153**:2159–2167.

30. Wickstrum, J. R., T. J. Santangelo, and S. M. Egan. 2005. Cyclic AMP receptor protein and RhaR synergistically activate transcription from the L-rhamnose-responsive *rhaSR* promoter in *Escherichia coli*. J. Bacteriol. **187**:6708–6718.

31. Yanisch-Perron, C., J. Vieira, and J. Messing. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene **33**:103–119.

32. Yuasa, R., M. Levinthal, and H. Nikaido. 1969. Biosynthesis of cell wall lipopolysaccharide in mutants of *Salmonella*. J. Bacteriol. **100**:433–444.
