
Inhibition Kinetics and Emodin Cocrystal Structure of a Type II Polyketide Ketoreductase${}^{\dagger,\ddagger}$

Tyler Paz Korman,${}^{\ddagger,\S}$ Yu-Hong Tan,${}^{\ddagger}$ Justin Wong,${}^{\ddagger,\S}$ Ray Luo,${}^{\ddagger}$ and Shiou-Chuan Tsai*,$^{\ddagger,\S}$

Department of Molecular Biology and Biochemistry and Department of Chemistry, University of California, Irvine, California 92697

Received August 14, 2007; Revised Manuscript Received December 6, 2007

ABSTRACT: Type II polyketides are a class of natural products that include pharmaceutically important aromatic compounds such as the antibiotic tetracycline and antitumor compound doxorubicin. The type II polyketide synthase (PKS) is a complex consisting of 5–10 standalone domains homologous to fatty acid synthase (FAS). Polyketide ketoreductase (KR) provides regio- and stereochemical diversity during the reduction. How the type II polyketide KR specifically reduces only the C9 carbonyl group is not well understood. The cocrystal structures of actinorhodin polyketide ketoreductase (actKR) bound with NADPH or NADP${}^{+}$ and the inhibitor emodin were solved with the wild type and P94L mutant of actKR, revealing the first observation of a bent $p$-quinone in an enzyme active site. Molecular dynamics simulation help explain the origin of the bent geometry. Extensive screening for *in vitro* substrates shows that unlike FAS KR, the actKR prefers bicyclic substrates. Inhibition kinetics indicate that actKR follows an ordered Bi Bi mechanism. Together with docking simulations that identified a potential phosphopantetheine binding groove, the structural and functional studies reveal that the C9 specificity is a result of active site geometry and substrate ring constraints. The results lay the foundation for the design of novel aromatic polyketide natural products with different reduction patterns.

The pharmaceutical potential of bacterial or fungal natural products is illustrated by the large number of compounds that are clinically applied as therapeutics. Many pharmaceutically relevant natural products are derived from polyketides and are used as antibiotics (tetracyclines, actinorhodin), anticancer (doxorubicin), antiviral (rebeccamycin derivatives), and cholesterol-lowering (statins) compounds (1). The antibiotics such as tetracycline and actinorhodin are biosynthesized from acyl-CoA thiosters by type II polyketide synthases (PKSs${}^{1}$), which are structurally and functionally related to the type II fatty acid synthase (FAS) (2). Compared to the type I FAS and PKS, which have enzyme domains covalently linked together, the type II FAS and PKS consist of 5–10 standalone enzymes that catalyze the condensation of malonyl extender units iteratively, followed by chain modifications, to produce the aromatic polyketides (3, 4).

A major difference between the type II PKS and FAS is the degree of reduction of the polyketide product. Whereas FAS typically have ketoreductase (KR), enoyl reductase (ER), and dehydratase (DH) domains that catalyze iterative reductions to produce a fully reduced, long-chain aliphatic fatty acid, the type II PKS either lacks any reduction domains or has a single KR domain that specifically reduces one carbonyl (normally the C9-carbonyl) group of the polyketide chain. As a result, the unreduced or singly reduced polyketide chain can form cyclized products that vary in their chain length, reduction levels, and presence of one or more rings and chiral centers.

The focus of this study is the type II KR, a key modifying enzyme in the biosynthesis of polycyclic, aromatic polyketides. The polyketide chain is first assembled by the minimal PKS (acyl-carrier protein, ketosynthase, and chain length factor), followed by KR reduction at a specific position and cyclization/aromatization of the polyketide chain (Figure 1A). Previous work suggests that the regiospecificities of ketoreduction, cyclization, and aromatization are closely related to one another (5, 6). Further, experiments from over 50 cloned type II PKSs have found that, except in rare cases, the type II KR specifically reduces the C9 carbonyl group, as demonstrated by the product outcome during the biosyntheses of actinorhodin (7), doxorubicin (8), R1128 (9), and enterocin (10, 11). Similar to actinorhodin, all of these polyketides are cyclized at the C7–C12 position (Figure 1A), although in special cases, a C5–C10 cyclized product also affords a C7-reduced product by KR (12). Despite extensive genetic analysis of type II PKS, the structure-function relationship that leads to the C9-specificity of KR is not well understood (13).

Earlier, we solved the cocrystal structures of actinorhodin KR (actKR) bound with either the cofactor NADP${}^{+}$ or

---

${}^{\dagger}$ This work is supported by the Pew Foundation and National Institute of General Medical Sciences (NIGMS R01GM076330).

${}^{\ddagger}$ The atomic coordinates have been deposited in the Protein Data Bank (accession code 2RH4, 2RHC, and 2RHR).

* Author to whom correspondence should be addressed. Phone 949-824-4486, e-mail sctsai@uci.edu, fax 949-824-8552.

${}^{\ddagger}$ Department of Molecular Biology and Biochemistry.

${}^{\S}$ Department of Chemistry.

${}^{1}$ Abbreviations: KR, ketoreductase; FabG, $\beta$-ketoacyl [acyl carrier protein] reductase; Act, actinorhodin; PKS, polyketide synthase; NADP, nicotinamide adenine dinucleotide phosphate; NADPH, reduced nicotinamide adenine dinucleotide phosphate; SDR, short-chain dehydrogenase/reductase; T3HNR, 1,3,8-trihydroxynaphthalene reductase; T4HNR, 1,3,6,8-tetrahydroxynaphthalene; ACP, acyl carrier protein.

1838 Biochemistry, Vol. 47, No. 7, 2008

domain of 6-deoxyerythronolide synthase (DEBS KR1) (21) have given insight into the molecular events and substrate specificity of the KRs. However, to date there is no *in vitro* kinetic information for any type II polyketide modifying enzymes. Here, we describe an *in vitro* assay for actKR activity with the substrate analogues trans-1-decalone, 2-decalone, and α-tetralone (Figure 1B, 7–9). Additionally, we report inhibition kinetics for actKR using the plant polyketide emodin. The assay results elucidate the catalytic mechanism of actKR with respect to substrate binding and product release. Herein, we also report the crystal structure of the inhibitor emodin bound in the KR active site. Previously, no polyketide KR structure has been reported with substrate or inhibitor bound. Surprisingly, we found that the p-quinone emodin is bent in the actKR active site. In combination with the kinetic data, the KR−emodin cocrystal structures allow the identification of residues important for enzyme catalysis and substrate binding, as well as molecular features important for control of the reduction stereo- and regiospecificity.

**MATERIALS AND METHODS**

*Chemicals, Strains, and DNA Manipulation.* NADPH, trans-1-decalone, 2-decalone, and α-tetralone were purchased from Sigma and were the highest grade available. DMSO, and all other reagents were ACS grade purchased from Fluka. *Escherichia coli* strain DH5α was used to prepare mutant and WT plasmid DNA. The S144A, Y157A, and P94L mutations were introduced using the Stratagene Quick-Change Kit. Synthetic oligonucleotides were from Operon. Transformants were selected on media supplemented with 50 μg mL⁻¹ kanamycin as the selectivity marker. The point mutations were confirmed by sequence analysis. *E. coli* strain BL21 λ(DE3) was used for recombinant protein expression.

*Expression and Purification of Recombinant Proteins.* The actIII gene was cloned into the pET28b vector to produce plasmid pYT238 as described previously (15). Following transformation of plasmid pYT238 into *E. coli* strain BL21-(DE3) (Novagen), 1 L of LB media containing 100 μg/mL kanamycin was inoculated with the transformed BL21 cells at 37 °C until the OD₆₀₀ ~ 0.6, and protein expression was induced by 1 mM IPTG overnight at 18 °C. The cells were harvested by centrifugation (6000g × 30 min) and resuspended in lysis buffer (50 mM Tris-Cl, pH 7.4, 0.3 M NaCl, 5 mM imidazole, 5% glycerol). The cells were lysed on ice by sonication (5 × 30 s) and the debris removed by centrifugation (14000 rpm × 45 min). The recombinant His-tagged protein was purified by Ni-NTA affinity chromatography (2 mL, Qiagen) and eluted at 20, 40, 60, 100, and 150 mM imidazole. ActKR was eluted as >95% pure protein at 60 mM imidazole and was dialyzed overnight against 4 L of 50 mM Tris-Cl, pH 7.5, 0.3 M NaCl, 10% glycerol. The protein was concentrated to 10 mg/mL with Vivaspin 30,-000 MWCO concentrators.

*In Vitro Kinetic Assays for actKR.* Kinetic parameters were determined spectrophotometrically on a Cary 3E UV−vis spectrophotometer (Varian). Steady-state kinetic parameters were determined by monitoring the change in absorbance at 340 nm from the conversion of NADPH (ε₃₄₀ = 6220 M⁻¹ cm⁻¹) to NADP⁺ over 5 min. The use of trans-1-decalone, 2-decalone, and α-tetralone as substrates for reductase activity has been reported for the FAS and the Type I PKS

Emodin Cocrystal Structures

**Table 1: Crystallization, Data Collection and Refinement Statistics**

|  | KR-NADPH-emodin | KR-NADP⁺-emodin | P⁹⁴L-NADPH-emodin |
| --- | --- | --- | --- |
| **A. Crystallization** |  |  |  |
| 4 M sodium formate | 4 M sodium formate | 4 M sodium formate |
| 50 mM Tris-Cl, 0.3 M NaCl, | 50 mM Tris-Cl, 0.3 M NaCl, | 50 mM Tris-Cl, 0.3 M NaCl, |
| 10% glycerol (pH 7.5) | 10% glycerol (pH 7.5) | 10% glycerol (pH 7.5) |
| **B. Crystallographic Data** |  |  |  |
| space group | P3₂21 | P3₂21 | P3₂21 |
| cell dimension (Å) | 104.0 104.0 123.4 | 103.9, 103.9, 123.2 | 104.7, 104.7, 123.6 |
| α = β = 90, γ = 120 | α = β = 90, γ = 120 | α = β = 90, γ = 120 |
| resolution (Å) | 2.30 | 2.10 | 2.50 |
| mosaicity (deg) | 0.2 | 0.2 | 0.2 |
| no. of observations | 844700 | 1570585 | 1573966 |
| no. of unique reflections | 34835 | 45482 | 27840 |
| completeness % (last shell) | 100.0 (100.0) | 99.9 (100.0) | 99.3 (100.0) |
| I/σ(I) (last shell) | 33.0 (5.9) | 30.9 (5.3) | 21.3 (5.7) |
| Rmerge % (last shell) | 7.7 (49.9) | 7.7 (49.1) | 9.3 (49.9) |
| **C. Refinement** |  |  |  |
| resolution (Å) | 2.30 | 2.10 | 2.50 |
| no. of reflections | 33667 | 43397 | 26627 |
| no. of protein atoms | 3851 | 3804 | 3804 |
| no. of cofactor atoms | 94 | 94 | 94 |
| no. of ligand atoms | 20 | 20 | 40 |
| average B-factor protein | 39.9 | 35.1 | 52.6 |
| average B-factor solvent | 46.6 | 49.5 | 59.7 |
| average B-factor emodin | 73.7 | 79.7 | 65.9 |
| average B-factor cofactor | 36.4 | 29.8 | 54.4 |
| no. of waters | 267 | 326 | 173 |
| Rfree % | 20.4 | 21.3 | 21.6 |
| Rcrys % | 17.9 | 18.5 | 18.9 |
| **D. Geometry** |  |  |  |
| RMS bonds (Å) | 0.006 | 0.005 | 0.008 |
| RMS angles (deg) | 1.29 | 1.26 | 1.70 |
| RMS B main chain | 1.27 | 1.28 | 1.39 |
| RMS B side chain | 2.2 | 2.14 | 2.18 |
| Ramachandran plot (%) |  |  |  |
| most favored | 89.7 | 90.1 | 88.2 |
| favored | 10.3 | 9.4 | 11.8 |
| generously allowed | 0.0 | 0.5 | 0.0 |

KR domains (20, 21). For actKR, all assays were performed in 400 mM KPi buffer, pH 7.4, and were initiated with the addition of the enzyme. The enzyme concentration varied between 100 nM and 5 μM. Because of the low solubility of α-tetralone in water, the temperature was kept constant at 30 °C in assay buffer containing 2% DMSO. The Michaelis-Menten constants Km and kcat for each ketone substrate were obtained by varying the substrate concentration in the presence of 50 μM NADPH. The Michaelis-Menten constants for NADPH were obtained by varying the NADPH concentration in the presence of 2 mM trans-1-decalone. A reaction with NADPH in the buffer containing 2% DMSO was used as control and did not show any effect on the change in absorbance. Data were fitted directly to the Michaelis-Menten equation, using the program Kaleida-graph (Synergy).

**Crystallization of actKR-Cofactor-Emodin (Ternary) Complexes.** Growth conditions for the trigonal crystals (space group P3₂21) containing actKR in complex with either NADPH or NADP⁺ were previously reported simultaneously by our group and Hadfield et al. (14, 15). Crystals of actKR wild-type or mutant complexes with cofactor (NADP⁺ or NADPH) and emodin grew within 3 days at room temperature by sitting-drop vapor diffusion in 3.8–4.8 M sodium formate (14, 15). Emodin was added to 10 mg/mL actKR containing 5 mM NADP(H) to a final concentration of 250 μM, where the final concentration of DMSO was 1% (v/v).

The drop was produced by mixing 2 μL of the purified protein solution with 2 μL of the well buffer over 500 μL of the well solution. The crystals of the ternary complexes yielded the same space group and similar cell dimensions as the actKR-NADP(H) binary complex (Table 1).

**Data Collection.** X-ray diffraction data for the ternary complexes of actKR were collected at the Stanford Synchrotron Radiation Laboratory (SSRL) to 2.1 Å. Crystals were flash frozen in the well solution plus 30% v/v glycerol. The diffraction intensities were integrated, reduced, and scaled using the program HKL2000 (22). The crystal space groups for all ternary complexes are P3₂21, and cell dimensions varied by 1–2 Å. A summary of the crystallographic data is shown in Table 1.

**Molecular Replacement and Refinement.** The structures of the actKR ternary complexes were solved by molecular replacement with CNS (23), using the coordinates for the actKR-NADPH structure as the search model (19). The actKR dimer was used for cross-rotation and translation search with the data from 15 to 4 Å. Once a suitable solution was found, a rigid body refinement was performed, treating the noncrystallographically related monomers as rigid bodies. Because of the flexibility of the loop region between residues 200–214, the starting model deleted this loop region in both monomers. A preliminary round of refinement using torsion angle simulated annealing, followed by energy minimization, positional, and individual B-factor refinement reduced Rcrys.

to 24–28%. The molecular models were gradually improved by sequential rounds of manual rebuilding using the program QUANTA (Accelrys), followed by refinement utilizing the maximum likelihood based approach (CNS), using all data to the highest resolution. Electron density maps at this stage showed clear density for the bound cofactor, inhibitor emodin, as well as the excluded 200–214 loop region (23). The emodin model was generated using PRODRG (24) and fitted to the difference maps using SWISS-PDB Viewer (25), and loop residues 200–214 were added in QUANTA. The topology and parameter files for emodin were generated using XPLO2D (26). Following positional refinement of the inhibitor, waters were added for final refinement of the models. The presence of emodin was confirmed by generating a simulated annealing omit map in the region of the bound inhibitor. Table 1 lists the statistics for refinement and components of the final models.

**Model Docking.** Docking between act KR-NADPH and trans-1-decalone, 2-decalone, and various putative conformations of the natural phosphopantetheinylated substrate were performed using ICM-Pro (27). The A chain from the KR-NADPH structure was defined as static. The binding pocket of actKR was defined by the 10 conserved residues, P94, G95, G96, T145, Q149, V151, F189, V198, R220, and L258, along with the catalytic tetrad N114, S144, Y157, and K161. Different binding conformations were searched using a default thoroughness of 2. Each compound was docked 10 times to ensure consistent docking simulation.

**Molecular Dynamics Simulation of Inhibitor Binding.** To study the molecular energies of emodin in bent or flat geometries (EMB and EML, respectively), initial pdb structures for both conformations were optimized with Gaussian 03B3LYP (28) using the 6-311+G(d,p) basis set for the ab initio calculation. To study the influence of protein environment to the geometry preferences of EMB and EML, Langevin dynamics (LD) simulations for both geometries in both free (solution) and enzyme-bound states were performed in implicit solvent (29) with default parameters in the AMBER 9 simulation package (30). The cavity radii are taken from a previous study (31). SHAKE (32) was turned on for bonds containing hydrogen atoms, so that a time step of 2 fs could be used in the leapfrog (33) numerical integrator for LD simulations. Each LD simulation was started after a brief steepest descent minimization of 500 steps to relax any possible clashes. After heating for 20 ps from 0 to 298 K, a production run was performed for 280 ps at 298K.

**RESULTS AND DISCUSSION**

**In Vitro Assay of actKR: A Significant Preference for Bicyclic Substrates.** Previous biosynthetic experiments using a Streptomyces host have implicated actKR in the first ring cyclization of the polyketide substrate (Figure 1A, from 0 to 1) (6, 34, 35). This raises the question whether the substrate of actKR is the linear polyketide 0 or the cyclized polyketides (such as 1 or 5, Figure 1A) and requires an in-depth analysis of actKR. However, the natural substrates of type II polyketide KRs are inherently unstable due to the presence of multiple ketone groups (36) (Figure 1A, intermediates 0–6). This difficulty raises the issue of finding a suitable in vitro substrate for the type II polyketide KRs.

---

Table 2: Kinetic Parameters for the Reduction of trans-1-Decalone, 2-Decalone, and α-Tetralone by actKR

| Compound          | $k_{\text{cat}}$ (s⁻¹)   | $K_{\text{m}}$ (mM)    | $k_{\text{cat}}/K_{\text{m}}$ (s⁻¹ mM⁻¹) |
|-------------------|--------------------------|------------------------|-----------------------------------------|
| NADPH$^a$         | 1.75 ± 0.09              | 0.005 ± 0.001          | 350 ± 70                                |
| trans-1-decalone$^b$ | 2.6 ± 0.1               | 0.79 ± 0.09            | 3.2 ± 0.3                               |
| 2-decalone$^b$     | 0.16 ± 0.02              | 3.9 ± 1.0              | 0.04 ± 0.01                             |
| α-tetralone$^c$   | 0.073 ± 0.003            | 5.7 ± 0.5              | 0.013 ± 0.001                           |

$a$ Determined in the presence of 2 mM trans-1-decalone.  
$b$ Determined in the presence of 0.2 mM NADPH.  
$c$ Determined in the presence of 0.2 mM NADPH at 30 °C and 2% Me₂SO.  
$d$ The values of $k_{\text{cat}}$ and $K_{\text{m}}$ for trans-1-decalone and 2-decalone are apparent values due to the presence of different enantiomeric forms of the substrate.

Previously, the assay for actKR activity in vitro involved a cell-free assay, in which each component of the minimal PKS (KS, CLF, and ACP, Figure 1A) must be purified separately and incubated with KR, followed by monitoring the formation of radiolabeled mutactin product by TLC (6). Such an assay is highly dependent on the activity of components other than KR itself, such as KS, CLF, and ACP, and does not distinguish between possible intermediates (Figure 1A, intermediates 0, 1, or 5). In order to isolate the single ketoreduction event and clarify mechanistic issues concerning the KR stereo- and regiospecificity, there is a need to identify suitable in vitro substrates for the type II polyketide KR.

We screened a wide range of potential substrate candidates (Figure 1B), such as the bicyclic, trans-1- or 2-decalones (7 and 8) and tetralone (9), acyl-CoAs (10 and 11), and the monocyclic 1,3-diketocyclohexanones (12–16). Previous studies with FAS (20, 37) and type I polyketide KRs (21) have shown that monocyclic ketones (such as 12–16) of various length and substitution patterns can be used as in vitro substrates for these KRs. However, in the case of actKR, we could not detect enzyme activity for any linear or mono-cyclic ketones, as well as acetoacetyl-CoA or acetoacetyl-ACP. On the other hand, we can detect enzyme activity for bicyclic ketone substrates such as trans-1-decalone (7), 2-decalone (8), and α-tetralone (9) (Table 2). Therefore, actKR shows a clear preference for bicyclic substrates.

The dependence on a sterically constrained substrate is not without precedent. Two of the best studied fungal reductases, 1,3,8 reductase (T3HNR) and 1,3,6,8-tetrahydroxynaphthalene (T4HNR), share 30% and 25% sequence identity with actKR, respectively (38–40). The products of T3HNR and T4HNR, scytalone and vermelone, are structurally similar to the first ring C9 reduced product in actKR biosynthesis (Figure 1A, intermediate 1) (38). The sequence homology with T3HNR and T4HNR, in combination with the strong preference for bicyclic substrates, points to the possibility that in the absence of downstream ARO and CYC domains, actKR may reduce an intermediate with both the first and second ring cyclized (intermediate 5 that can also lead to mutactin, Figure 1A), and the actual substrate for actKR may be a tautomerized form of the bicyclic intermediate 5 (Figure 1C, 5, as well as its aromatized form and the aromatic tautomers).

**The Importance of Substrate Flexibility: Probing the Substrate Specificity for 1-Decalone, 2-Decalone, and Tetralone.** Among the bicyclic substrates, actKR shows a distinct preference for trans-1-decalone (Table 2). The $K_{\text{m}}$

Emodin Cocrystal Structures

values of 0.79 mM for *trans*-1-decalone and 0.0049 mM for NADPH agree well with published data for DEBS KR1 (21) (1.2 and 0.011 mM for *trans*-1-decalone and NADPH, respectively), although the $k_{cat}/K_m$ is an order of magnitude higher for actKR (3.23 s⁻¹ mM⁻¹ for actKR versus 0.450 s⁻¹ mM⁻¹ for DEBS KR1). Therefore, despite the sequence homology shared between actKR and DEBS KR1 (17% identical, 35% similar), the catalytic efficiency and substrate specificity for the *in vitro* substrates are different between type I and type II polyketide KRs.

In comparison to 1- and 2-decalone, the aromatic α-tetralone is a much poorer substrate, with an 8-fold higher $K_m$ (5.6 ± 0.36 mM) and a 200-fold lower $k_{cat}/K_m$ than that of *trans*-1-decalone. The apparent differences in binding and efficiency between *trans*-1-decalone and α-tetralone could be a result of decreased second-ring flexibility in the aromatic α-tetralone substrate.

Interestingly, 2-decalone (Figure 1B, compound 8) is a poorer KR substrate than *trans*-1-decalone, with an 80-fold lower $k_{cat}/K_m$. In the natural substrate 1 or 5, the C7–C12 cyclization restricts the reduction to the C9 position of the polyketide chain (Figure 1A) (3). 2-Decalone mimics the first two rings in intermediates 1 and 5, with its carbonyl group corresponding to the natural C9 ketone of intermediate 1 (Figure 1A,B). If it is assumed that the first ring (C7–C12) cyclization occurs before reduction of the C9 carbonyl of the tautomers (35, 36), the 2-decalone ketone group should be more readily reduced than the ketone of *trans*-1-decalone. So why do we observe the opposite trend that $k_{cat}/K_m$ of 2-decalone is smaller than *trans*-1-decalone? The first possible explanation is due to the presence of isomers. In the commercially available 2-decalone, the cis isomer and both enantiomers of the trans substrate are present. The potential nonreactivity of *cis*-2-decalone has been reported previously in screens for stereoselective reductions by alcohol dehydrogenase in *D. grovesii* (41). Since the cis and trans isomers are 1:1 in ratio, the presence of the cis isomer will decrease the activity by half. However, even if only one of the eight possible 2-decalone isomers are reactive, the activity will only decrease to 1/8, and this still does not account for the 80 fold $k_{cat}/K_m$ difference between 1- and 2-decalone. A second possible explanation is that 1- and 2-decalone have different docking modes in the actKR substrate pocket, which is important for orienting the ketone group for ketoreduction. Indeed, docking simulation suggests that *trans*-1-decalone and *trans*-2-decalone have different binding modes. Docking for both (9R)-*trans*-1-decalone and (9S)-*trans*-1-decalone consistently predicts the same conformation for the ketone in an appropriate orientation for hydride transfer (Figure 3A) and an average calculated binding energy ($\Delta G$) of −30.2 kcal/mol. In contrast, when either (9R)-*trans*-2-decalone, (9S)-*trans*-2-decalone, or *cis*-2-decalone was used as the substrate, the docking position and orientation varied over every docking run, and with a much smaller binding energy ($\Delta G = -22.69$, $-22.02$, and $-25.49$ kcal/mol for 9(R)-*trans*-9(S)-*trans*- and *cis*-2-decalones, respectively) (Figure 3B). Specifically, about 40% of docking runs orient the ketone of 2-decalone within hydrogen-bonding distance of the Thr145 side chain, thus misorienting the ketone out of the range of the oxyanion hole (side chains of Tyr157 and Ser144) and away from the catalytic tetrad. Therefore, the docking simulation indicates that the observed higher $k_{cat}$

1842 Biochemistry, Vol. 47, No. 7, 2008

(A)  
![Image](image1.png)

(B)  
![Image](image2.png)

FIGURE 3: Docking simulation between actKR and *trans*-1- or 2-decalone. (A) *trans*-1-Decalone binds tightly to actKR with a single predicted conformation. (B) 2-decalone binds less tightly to actKR with multiple possible binding modes.

(A)  
![Image](image3.png)

(B)  
![Image](image4.png)

(C)  
![Image](image5.png)

FIGURE 4: Enzyme kinetics of actKR. (A) The double reciprocal plot (1/*v* versus 1/[trans-1-decalone]) altering the concentration of trans-1-decalone and NADPH eliminates a ping-pong mechanism (which will produce parallel lines) as a possible mechanism. (B) The double reciprocal plot that changes the concentration of the inhibitor AMP, NADPH, and trans-1-decalone indicates that AMP is a competitive inhibitor (right) of NADPH but a noncompetitive inhibitor for trans-1-decalone (left). (C) The double reciprocal plot that changes the concentration of the inhibitor emodin, NADPH, and trans-1-decalone indicates that emodin is a competitive inhibitor of trans-1-decalone (left) but a uncompetitive inhibitor for NADPH (right). The results from A, B, and C indicate that actKR proceeds via an ordered Bi Bi mechanism.

result is consistent with an ordered Bi Bi mechanism, where binding of NADPH is followed by substrate binding, ketone reduction, and product release.

The **actKR-NADP(H)-Emodin Crystal Structure Shows a Bent p-Quinone**. The ternary structure of actKR bound with the cofactor NADP⁺ or NADPH (denoted as NADP(H)) and the inhibitor emodin was crystallized in the same crystallization solution, with the same hexagonal space group (P3₂-21) as the binary KR-cofactor complex (15). Each crystallographic asymmetric unit contains two monomers (A and B), while the 2-fold crystallographic axis generates the biological tetramer (Figure 5A). The A chain of KR-NADPH-emodin structure shows emodin electron density in the \(3F_{o} - 2F_{c}\) map (contoured at 1 sigma), and it has an overall rmsd of 0.20 and 0.34 Å with the KR-NADP⁺ and KR-NADPH structures, respectively, although in both structures the emodin does have an elevated *B*-factor relative to the rest of the protein (Figure 6A, 6C, Table 1). The hydrogen-bonding network, observed in the binary complex structure between the cofactor, N114, K161, S144, Y157, and the four waters are conserved in the emodin-bound ternary structure (Figures 2 and 6). All amino acids for both monomers could be built into the electron density, including the loop region between α6 and α7 in both monomers as well as the six residues (cloning artifact) preceding the N-terminal methionine in monomer B. The overall rmsd between monomers A and B is 0.48 Å, although there is a significant movement of ~7.9 Å in the flexible loop region between α6 and α7 (Figure 5B, right panel). A close inspection of the electron density near the active site of monomer A shows some density contribution from a neighboring monomer that contacts the conserved NNAG motif in the long loop region between β4 and α5. An inspection of symmetrically related molecules in the unit cell shows that the density corresponds to residues −6 to 0 from monomer B′ of the \(Y-X, -X, Z+1/3\) symmetry mate. Although the first six residues do not interact directly with the active site, Val −5 comes within 6 Å of the bound emodin and P94 stacks with H0 of monomer B. The crystal structure also shows that the stacking of proline and histidine residues (P94-H0′-P⁻4′-H201) locks the N-terminus of monomer B in place. Inspection of the previously reported binary structures shows that these crystal contacts are conserved between the KR-NADPH, KR-NADP⁺, and KR-NADP(H)-emodin structures.

In both actKR-NADP⁺-emodin and actKR-NADPH-emodin structures, the inhibitor emodin binds in the substrate binding cleft of monomer A (see electron density maps in Figure 6A,C). To our surprise, in both ternary structures, the bound emodin was not planar in the substrate binding pocket but was found to be bent ~63° from planarity with

Emodin Cocrystal Structures

(A)

(B)
Monomer B Close form
Monomer A Open form

FIGURE 5: (A) The tetramer (in different colors) cocrystal structure of wild type actKR-NADPH-emodin, in which the cofactor NADPH (in sticks) is bound to all four monomers, whereas well-defined electron density of emodin (in spheres) can be found in monomers A and C. (B) The asymmetric unit contains monomers A (middle panel) and B (left panel) in open and close conformations, respectively. Surface potentials were colored from negative (red) to positive (blue). When the open and closed conformations are overlapped (right panel, open in yellow, close in purple), the major conformational change is in the flexible α6-α7 region, with a 7.9 Å difference between the two monomers.

the C10 in the appropriate orientation instead of the C6 hydroxyl (Figure 6A-D). In addition, an unbiased \(2F_{o} - F_{c}\) simulated annealing omit map (not shown) in this region shows partial density for the core of the bent emodin, confirming the bent p-quinone geometry. In order for the anthraquinone to adopt this bent conformation, the quinone moiety of emodin would either need to be reduced, harbor a radical or a semi-quinone, or be bent due to steric constraints in the active site of actKR. The first two possibilities were eliminated by a lack of any detectable signal in single turnover and EPR experiments (data not shown). Furthermore, when we solved the crystal structure of mutant P94L (active with trans-1-decalone with a \(k_{\text{cat}}/K_{\text{m}}\) of 1.02 ± 0.59 s⁻¹ mM⁻¹) bound with NADPH and emodin, we found that emodin binds tightly and adopts a flat conformation in the active site as evident in its slightly lower B factors and the presence of well-defined electron density for two molecules of emodin modeled at full occupancy in the unbiased \(2F_{o} - F_{c}\) simulated-annealing omit map (Figure 6E,F, contoured at 1σ). This indicates that emodin is uniquely bent in the wild type actKR active site, and that, although emodin has elevated B factors in the wild type structure, the bent electron density is not a crystallization artifact. In the wild type enzyme, the hydrogen bond between emodin

(Biochemistry, Vol. 47, No. 7, 2008 1843)

(A)

P94
Y157
S144
NADPH
Wild Type actKR-NADPH-emodin:
Bent Emodin

(C)

P94
Y157
S144
NADP⁺
Wild Type actKR-NADP⁺-emodin:
Bent Emodin

(E)

P94L
Y157
S144
NADPH
P94L: actKR-NADPH-emodin: Flat Emodin

(F)

NADPH
1st emodin
2nd emodin

FIGURE 6: (A and B) The wild type actKR-NADPH-emodin cocrystal structure shows that the quinone carbonyl group of emodin is in the three-point docking position. Surprisingly, the p-quinone is bent, as seen in the \(3F_{o} - 2F_{c}\) density contoured at 1σ. In addition, an unbiased \(2F_{o} - F_{c}\) simulated annealing omit map (not shown) in this region contoured at 1σ also shows partial density for the core of the bent emodin, confirming the bent geometry. (B) With the view rotated 90°. (C and D) Similarly, the wild type actKR-NADP⁺-emodin cocrystal structure shows that the quinone carbonyl group of emodin is in the three-point docking position (\(2F_{o} - F_{c}\) density map is shown, contoured at 0.6s), and the p-quinone is also bent, as shown in D with the view rotated 90°. (E and F) In comparison, the P94L actKR-NADPH-emodin cocrystal structure shows a flat p-quinone in the \(2F_{o} - F_{c}\) simulated annealing omit map contoured at 1s. Also, unexpectedly, there are two molecules of emodin bound in the P94L mutant active site with the first molecule (purple) at full occupancy while there is only partial density in the sa-omit map for the second molecule when also modeled at full occupancy (density map not shown). This result indicates that a change in the actKR active site geometry affects substrate/inhibitor binding, and also that the observed bent emodin in the wild type is not a crystallization defect. The electron density surrounding emodin is shown in blue mesh.

C10 and active site residues may help stabilize the bent conformation, so that an abortive complex (KR-NADP(H)-emodin) is stable enough to be crystallized (Figure 6A-D). In addition, the presence of two molecules of emodin bound in the P94L mutant active site (Figure 6E,F) demonstrates that a change in the actKR active site geometry will significantly affect the substrate/inhibitor specificity. Although this is the first report of a bent p-quinone bound in an enzyme binding pocket, recent small molecule crystal structures of chemically similar hydroxyanthraquinone compounds also show that it is possible for the p-quinone to be bent in solution without extensive energy penalties due to the presence of different tautomers (Figure 1D, emodin, and its tautomers) (48).

To look into the bent emodin energetics, we performed molecular dynamics simulation to analyze the energy penalty

for the bent versus flat forms of emodin in the actKR active site using the linear (EML) and bent (EMB) conformations of emodin built using PRODRG (24). Two angles, one for atoms C4, C2, and C15 and the other for atoms C8, C11, and C19, respectively, were used to monitor the geometry preferences between EML and EMB. We found that EMB remains bent, and EML is planar after energy minimization. The molecular dynamic simulation shows that the energy difference between a bent and planar emodin is similar in solution and in the actKR binding pocket. Therefore, if a *p*-quinone can be bent in the solution, as reported before by small molecular crystallography, our calculation shows that actKR will not prevent the bending from happening.

On the basis of the above results, it is possible that a tautomeric form of emodin is bound to the active site (Figure 1D), which holds the *p*-quinone in the bent geometry and positions the C10 quinone carbonyl near the oxyanion hole in a conformation that is not favorable for reduction. It is apparent that the bent emodin maintains extensive hydrogen-bonding and hydrophobic interactions with actKR that are necessary for its inhibitory characteristics (*K*<sub>i</sub> of ~15 μM). This extensive hydrogen-bonding network includes the C6 hydroxyl that hydrogen-bonds with Gln149 and Thr145, as well as the C3 methyl that interacts with Pro94 and Ala154. Val151 comes within 3.3 Å of the reduced emodin. Also, the aromatic residue Phe189 comes within 3.6 Å of aromatic ring C, possibly to help orient the bound inhibitor. These extra interactions may stabilize the bent emodin in the active site, facilitating crystallization of the actKR–NADP(H)–emodin ternary complex.

**The Open Form versus the Closed Form.** The greatest difference between the Type II polyketide KRs and other SDRs (such as type I polyketide KRs, FabG (49), and tropinone reductase (50)) is a 10-residue insertion (residues 199–209) between helices 6 and 7. Although the length is widely conserved in type II KRs, the amino acid composition of the loop varies except for Y202 and W206. The length of this region in modular (type I) polyketide KRs is not as uniformly conserved as in type II polyketide KRs, making this 10-residue insertion a unique feature of type II polyketide KR. Because the type II polyketide KRs have a higher sequence identity with the fungal PKS or FAS KRs, it is noteworthy that Y202 (a conserved residue in this 10-residue region) is also conserved and stacks directly with bound inhibitors in the T3HN reductase structures, similar to the actKR–emodin structure (39). Furthermore, when the monomers A and B of the emodin-bound structure are superimposed, there is a large shift in this loop region (residues 200–214), especially surrounding the Cα of Glu207 (7.9 Å between monomers A and B, Figure 5B, right panel). The importance of this flexible loop region has been described for the homologous T3HN reductase from *M. grisea* (40) and the 7α-hydroxysteroid dehydrogenase from *E. coli* (51). This loop region forms half of the substrate binding pocket and is the least conserved region among SDRs (52), accounting for the different SDR substrate specificities. The α6-α7 region also has the highest *B*-factor in the actKR crystal structure. A comparison of monomers A and B in the published binary actKR–NADPH structure (14, 15) or the actKR–NADP(H)–emodin ternary structures (this work) show that there is a significant difference in the loop regions between monomers A and B. In the ternary actKR–NADP

(H)–emodin complex, this difference is highlighted by the fact that clear electron density for the bent emodin is observed in monomer A but not in monomer B.

The observed conformational flexibility in the 10-residue insertion loop may have a profound influence on the binding of the natural polyketide substrate. When actKR adopts a closed conformation (Figure 5B, left panel) with NADPH bound as in monomer B, we could not observe electron density corresponding to emodin. However, in monomer A, where the emodin density is well defined, actKR adopts an open conformation, presumably in an orientation that mimics substrate binding or product release (Figure 5B, middle panel). Therefore, the opening and closing of the actKR pocket may be related with substrate and product binding.

**Substrate Specificity and Protein Flexibility.** The importance of protein flexibility on ligand docking has been recently reviewed (53). In light of the flexible 10-residue insert discussed above, and in combination with kinetic data and docking simulations, we have further investigated the correlation between substrate specificity and protein flexibility as follows: docking simulation shows that 10-carbon, bicyclic substrates such as *trans*-1- and 2-decalone can fit in the active site, but do not possess the necessary hydrophilic substituents as in the natural substrate, to reinforce the C9 regiospecificity. To determine the importance of hydrophilic substituents in the polyketide chain for substrate binding, we docked actKR with C7–C12 cyclized intermediates (1 and 5, Figure 1A) containing the phosphopantetheine (PPT) group. The docked substrates mimic the natural polyketide intermediates that are tethered to acyl carrier protein via the PPT group. We found that the use of different monomers (A or B) result in very different docking results. When the closed form of actKR (monomer B) is used, the cyclized ring cannot enter the closed-off active site (Figure 5B, left panel). On the other hand, when the open form of actKR (monomer A) is used (Figure 5B, middle panel), multiple docking runs consistently dock the C9 position of mono- and bicyclic intermediates 1 and 5 (Figure 1A) in the correct orientation in the vicinity of the oxyanion hole (Figure 7A). Therefore, the docking simulation indicates that the closed form blocks the binding of an incoming polyketide substrate, while the open form is presumably the conformation adopted by actKR prior to substrate binding and/or product release.

Significantly, multiple runs dock the PPT group to a unique groove that is only present in the open form (Figure 7B,C). This groove contains a pocket of three arginines, R38, R65, and R93, D109, and T113. All except R65 are highly conserved in type II polyketide KRs. These residues form a pocket that is predicted to interact strongly with the phosphate in the PPT group to help anchor the polyketide substrate. Interestingly, this same region was recently identified as the probable location for ACP and phosphopantetheine docking in SCO1815, the KR involved in biosynthesis of R1128 in *S. coelicolor* (54). Furthermore, the docking results suggest that the positioning of P94 can influence the bending of the PPT arm, further guiding the orientation of the substrate. The conclusion for the above simulation is that both protein flexibility and substrate chemical properties (such as the ring constraints) are important for actKR to properly orient its substrate for regiospecific ketoreduction.

**Biological Significance.** Polyketides have been recognized as one of the most important classes of natural products for

Emodin Cocrystal Structures

(A)

(B)

Docked  
substrates  

FIGURE 7: Multiple docking rounds of possible actKR intermediates (1 and 5 in Figure 1A) show high consistency in docking results. (A) In the hydride transfer site, the C9 carbonyl groups of the monocyclic 1 (magenta) and the bicyclic 5 (yellow) are consistently docked to the optimal position for ketoreduction. Further, at least the constraints from the first ring are necessary to properly orient the substrate for C9 reduction. (B) The phosphopantetheine (PPT) group, which is tethered between acyl carrier protein and the polyketide substrate, is consistently docked into a groove on the actKR surface, where the positively charged groove interacts with the PPT phosphate group.

medical applications. The PKS is a multidomain enzyme complex that produces a huge variety of polyketides via a controlled variation of building blocks and modification reactions such as chain reduction and cyclization. However, it is unclear whether polyketide cyclizations (that lead to the formation of 1 and 5, Figure 1A) occur before or after ketoreduction. Our kinetic analyses show that similar to other SDR proteins, the order of substrate and cofactor binding in actKR follows an ordered Bi Bi mechanism, where the cofactor NADPH binds before the ketone substrate. However, in vitro, the actKR (and likely all type II polyketide KRs) has a unique preference for bicyclic substrates, indicating that the C7–C12 cyclized intermediates 1 or 5 are the most likely substrate of actKR (Figure 1A). Therefore, the C9 regiospecificity results from the dual constraints of the three-point docking in the active site and the C7–C12 ring geometry of the substrate. The importance of cyclization and substitution pattern can be seen in the actKR–NADP(H)–emodin ternary structure, which also reveals a bent p-quinone in an enzyme active site for the first time. The emodin cocrystal structure, in combination with docking studies, suggest conserved residues in the binding pocket of Type II KRs, namely G95, G96, T145, Q149, V151, M194, V198, Y202, and the lesser conserved P94 help guide substrate binding with a marked preference for cyclic, geometrically constrained substrates. Docking simulations further support the importance of the open conformation for substrate binding and identified a highly conserved groove for PPT binding. Therefore, the actKR substrate specificity is defined by a combination of enzyme conformation, specific molecular interactions between the substrate and active site residues, and substrate and protein flexibility. Because of the dynamic nature of the binding cleft, it should be possible for KR to be altered in a way to accept substrates with variable chain lengths or cyclization patterns. In conclusion, we have conducted detailed kinetic and structural analysis of a polyketide KR domain and, for the first time, reported an inhibitor-bound polyketide KR structure that enables us to elucidate the molecular basis of KR specificity, which in turn will facilitate the development of “unnatural natural products” via protein engineering of polyketide synthase.

ACKNOWLEDGMENT

We thank Prof. Yi Tang for providing the pYT238 plasmid, Prof. Markus Ribbe for the EPR analyses, and Prof. Chaitan Khosla for providing the pRZ153 plasmid. Thanks to Mr. Peter Smith for reading this manuscript and for his insightful suggestions. Portions of this research were carried out at the Stanford Synchrotron Radiation Laboratory, a national user facility operated by Stanford University on behalf of the U.S. Department of Energy, Office of Basic Energy Sciences. The SSRL Structural Molecular Biology Program is supported by the Department of Energy, Office of Biological and Environmental Research, and by the National Institutes of Health, National Center for Research Resources, Biomedical Technology Program, and the National Institute of General Medical Sciences.

REFERENCES

1. Cane, D. E., Walsh, C. T., and Khosla, C. (1998) Harnessing the biosynthetic code: combinations, permutations, and mutations, Science 282, 63–68.
2. Hopwood, D. A. (1997) Genetic Contributions to Understanding Polyketide Synthases, Chem. Rev. 97, 2465–2498.
3. McDaniel, R., Ebert-Khosla, S., Hopwood, D. A., and Khosla, C. (1995) Rational design of aromatic polyketide natural products by recombinant assembly of enzymatic subunits, Nature 375, 549–554.
4. Bibb, M. J., Biro, S., Motamedi, H., Collins, J. F., and Hutchinson, C. R. (1989) Analysis of the nucleotide sequence of the Streptomyces glaucescens tcmI genes provides key information about the enzymology of polyketide antibiotic biosynthesis, EMBO J. 8, 2727–2736.
5. Zawada, R. J., and Khosla, C. (1997) Domain analysis of the molecular recognition features of aromatic polyketide synthase subunits, J. Biol. Chem. 272, 16184–16188.
6. Zawada, R. J., and Khosla, C. (1999) Heterologous expression, purification, reconstitution and kinetic analysis of an extended type II polyketide synthase, Chem. Biol. 6, 607–615.
7. Malpartida, F., and Hopwood, D. A. (1984) Molecular cloning of the whole biosynthetic pathway of a Streptomyces antibiotic and its expression in a heterologous host, Nature 309, 462–464.
8. Meurer, G., Gerlitz, M., Wendt-Pienkowski, E., Vining, L. C., Rohr, J., and Hutchinson, C. R. (1997) Iterative type II polyketide synthases, cyclases and ketoreductases exhibit context-dependent behavior in the biosynthesis of linear and angular decapolyketides, Chem. Biol. 4, 433–443.
9. Marti, T., Hu, Z., Pohl, N. L., Shah, A. N., and Khosla, C. (2000) Cloning, nucleotide sequence, and heterologous expression of the biosynthetic gene cluster for R1128, a non-steroidal estrogen receptor antagonist. Insights into an unusual priming mechanism, J. Biol. Chem. 275, 33443–33448.
10. Kalaitzis, J. A., and Moore, B. S. (2004) Heterologous biosynthesis of truncated hexaketides derived from the actinorhodin polyketide synthase, J. Nat. Prod. 67, 1419–1422.
11. Hertweck, C., Xiang, L., Kalaitzis, J. A., Cheng, Q., Palzer, M., and Moore, B. S. (2004) Context-dependent behavior of the enterocin iterative polyketide synthase; a new model for ketoreduction, Chem. Biol. 11, 461–468.

12. McDaniel, R., Ebert-Khosla, S., Fu, H., Hopwood, D. A., and Khosla, C. (1994) Engineered biosynthesis of novel polyketides: influence of a downstream enzyme on the catalytic specificity of a minimal aromatic polyketide synthase, *Proc. Natl. Acad. Sci. U.S.A.* 91, 11542–11546.

13. Rix, U., Fischer, C., Remsing, L. L., and Rohr, J. (2002) Modification of post-PKS tailoring steps through combinatorial biosynthesis, *Nat. Prod. Rep.* 19, 542–580.

14. Hadfield, A. T., Limpkin, C., Teartasin, W., Simpson, T. J., Crosby, J., and Crump, M. P. (2004) The crystal structure of the actIII actinorhodin polyketide reductase: proposed mechanism for ACP and polyketide binding, *Structure (Camb.)* 12, 1865–1875.

15. Korman, T. P., Hill, J. A., Vu, T. N., and Tsai, S. C. (2004) Structural analysis of actinorhodin polyketide ketoreductase: cofactor binding and substrate specificity, *Biochemistry* 43, 14529–14538.

16. Rossmann, M. G., and Argos, P. (1981) Protein folding, *Annu. Rev. Biochem.* 50, 497–532.

17. Holzbaur, I. E., Ranganathan, A., Thomas, I. P., Kearney, D. J., Reather, J. A., Rudd, B. A., Staunton, J., and Leadlay, P. F. (2001) Molecular basis of Celmer’s rules: role of the ketosynthase domain in epimerisation and demonstration that ketoreductase domains can have altered product specificity with unnatural substrates, *Chem. Biol.* 8, 329–340.

18. Nakajima, K., Kato, H., Oda, J., Yamada, Y., and Hashimoto, T. (1999) Site-directed mutagenesis of putative substrate-binding residues reveals a mechanism controlling the different stereospecificities of two tropinone reductases, *J. Biol. Chem.* 274, 16563–16568.

19. Zhang, Y. M., Wu, B., Zheng, J., and Rock, C. O. (2003) Key residues responsible for acyl carrier protein and beta-ketoacyl-acyl carrier protein reductase (FabG) interaction, *J. Biol. Chem.* 278, 52935–52943.

20. Dutler, H., Kull, A., and Mislin, R. (1971) Fatty acid synthetase from pig liver. 2. Characterization of the enzyme complex with oxidoreductase activity for alicyclic ketones as a fatty acid synthetase, *Eur. J. Biochem.* 22, 213–217.

21. Ostergaard, L. H., Kellenberger, L., Cortes, J., Roddis, M. P., Deacon, M., Staunton, J., and Leadlay, P. F. (2002) Stereochemistry of catalysis by the ketoreductase activity in the first extension module of the erythromycin polyketide synthase, *Biochemistry.* 41, 2719–2726.

22. Otwinowski, Z., and Minor, W. (1997) Processing of X-ray diffraction data collected in oscillation mode, *Macromol. Crystallogr. Part A,* 1997 276, 307–326.

23. Bruner, A. T., Adams, P. D., Clore, G. M., DeLano, W. L., Gros, P., Grosse-Kunstleve, R. W., Jiang, J. S., Kuszewski, J., Nilges, M., Pannu, N. S., Read, R. J., Rice, L. M., Simonson, T., and Warren, G. L. (1998) Crystallography & NMR system: A new software suite for macromolecular structure determination, *Acta Crystallogr., Sect. D: Biol. Crystallogr.* 54 (Pt 5), 905–921.

24. Schuttelkopf, A. W., and van Aalten, D. M. (2004) PRODRG: a tool for high-throughput crystallography of protein-ligand complexes, *Acta Crystallogr., Sect. D: Biol. Crystallogr.* 60, 1355–1363.

25. Guex, N., and Peitsch, M. C. (1997) SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling, *Electrophoresis* 18, 2714–2723.

26. Kleywegt, G. J., and Jones, T. A. (1998) Databases in protein crystallography, *Acta Crystallogr., Sect. D: Biol. Crystallogr.* 54, 1119–1131.

27. Abagyan, R., Totrov, M., and Kuznetsov, D. (1994) Icm - a New Method for Protein Modeling and Design – Applications to Docking and Structure Prediction from the Distorted Native Conformation, *J. Comput. Chem.* 15, 488–506.

28. Foresman, J. B., and Frisch, A. (1996) in *Exploring Chemistry With Electronic Structure Methods: A Guide to Using Gaussian*, 2nd ed., Gaussian, Inc., Wallingford.

29. Lu, Q., and Luo, R. (2003) A Poisson-Boltzmann dynamics method with nonperiodic boundary condition, *J. Chem. Phys.* 119, 11035–11047.

30. Case, D. A., Darden, T. E. C., III; Simmerling, C.; Wang, J.; Duke, R. E.; Luo, R. Merz, K. M.; Pearlman, D. A.; Crowley, M.; Walker, R.; Zhang, W.; Wang, B.; Hayik, S.; Roitberg, A.; Seabra, G.; Wong, K.; Paesani, F.; Wu, X.; Brozell, S.; Tsui, V.; Gohlke, H.; Yang, L.; Tan, C.; Mongan, J.; Hornak, V.; Cui, G.; Beroza, P.; Mathews, D. H.; Schafmeister, C.; Ross, W. S., and Kollman, P. A. (2006) AMBER 9, San Francisco.

31. Lwin, T. Z. Zhou, R. H., and Luo, R. (2006) Is Poisson-Boltzmann theory insufficient for protein folding simulations? *J. Chem. Phys.* 124, 034902.

32. Ryckaert, J. P., Ciccotti, G., and Berendsen, H. J. C. (1977) Numerical-Integration of Cartesian Equations of Motion of a System with Constraints - Molecular-Dynamics of N-Alkanes, *J. Comput. Phys.* 23, 327–341.

33. Hockney, R. W., and Eastwood, J. W. (1981) *Computer Simulations Using Particles*, McGraw-Hill, New York.

34. Kantola, J., Blanco, G., Hautala, A., Kunnari, T., Hakala, J., Mendez, C., Ylihonko, K., Mantsala, P., and Salas, J. (1997) Folding of the polyketide chain is not dictated by minimal polyketide synthase in the biosynthesis of mithramycin and anthracycline, *Chem. Biol.* 4, 751–755.

35. Fu, H., Ebert-Khosla, S., Hopwood, D. A., and Khosla, C. (1994) Engineered Biosynthesis of Novel Polyketides - Dissection of the Catalytic Specificity of the Act Ketoreductase, *J. Am. Chem. Soc.* 116, 4166–4170.

36. Fu, H., Hopwood, D. A., and Khosla, C. (1994) Engineered biosynthesis of novel polyketides: evidence for temporal, but not regiospecific, control of cyclization of an aromatic polyketide precursor, *Chem. Biol.* 1, 205–210.

37. Joshi, A. K., and Smith, S. (1993) Construction, expression, and characterization of a mutated animal fatty acid synthase deficient in the dehydrase function, *J. Biol. Chem.* 268, 22508–22513.

38. Simpson, T. J., and Weerasooriya, M. K. B. (2000) NMR studies of tautomerism in the fungal melanin biosynthesis intermediate 1,3,8-trihydroxynaphthalene, *J. Chem. Soc., Perkin Trans. 1* 2771–2775.

39. Andersson, A., Jordan, D., Schneider, G., and Lindqvist, Y. (1996) Crystal structure of the ternary complex of 1,3,8-trihydroxynaphthalene reductase from Magnaporthe grisea with NADPH and an active-site inhibitor, *Structure* 4, 1161–1170.

40. Andersson, A., Jordan, D., Schneider, G., and Lindqvist, Y. (1997) A flexible lid controls access to the active site in 1,3,8-trihydroxynaphthalene reductase, *FEBS Lett.* 400, 173–176.

41. Carballeira, J. D., Alvarez, E., Campillo, M., Pardo, L., and Sinisterra, J. V. (2004) Diplogelasinospora grovesii IMI 171018, a new whole cell biocatalyst for the stereoselective reduction of ketones, *Tetrahedron Asymmetry* 15, 951–962.

42. Zhang, C., Teng, L., Shi, Y., Jin, J., Xue, Y., Shang, K., and Gu, J. (2002) Effect of emodin on proliferation and differentiation of 3T3-L1 preadipocyte and FAS activity, *Chin. Med. J. (Beijing, China, Engl. Ed.)* 115, 1035–1038.

43. Anderson, J. A., Lin, B. K., Williams, H. J., and Scott, A. I. (1988) Deoxygenation of Phenolic Natural-Products - Enzymatic Conversion of Emodin to Chrysophanol, *J. Am. Chem. Soc.* 110, 1623–1624.

44. Winberg, J. O., and McKinley-McKee, J. S. (1994) Drosophila melanogaster alcohol dehydrogenase: product-inhibition studies, *Biochem. J.* 301 (Pt 3), 901–909.

45. Lee, J. K., Koo, B. S., Kim, S. Y., and Hyun, H. H. (2003) Purification and characterization of a novel mannitol dehydrogenase from a newly isolated strain of Candida magnoliae, *Appl. Environ. Microbiol.* 69, 4438–4447.

46. Poddar, S., and Henkin, J. (1984) Isotope, Pulse Chase, Stopped-Flow, and Rapid Quench Studies on the Kinetic Mechanism of Bovine Dihydropteridine Reductase, *Biochemistry* 23, 3143–3148.

47. Fromm, H. J. (1979) Use of competitive inhibitors to study substrate binding order, *Methods Enzymol.* 63, 467–486.

48. Norvez, S., Tournilhac, F. G., Bassoul, P., and Herson, P. (2001) Mesomorphism and polar distortion in 1,4,5,8-tetrasubstituted anthraquinones and anthracenes, *Chem. Mater.* 13, 2552–2561.

49. Price, A. C., Zhang, Y. M., Rock, C. O., and White, S. W. (2004) Cofactor-induced conformational rearrangements establish a catalytically competent active site and a proton relay conduit in FabG, *Structure (Camb.)* 12, 417–428.

50. Nakajima, K., Yamashita, A., Akama, H., Nakatsu, T., Kato, H., Hashimoto, T., Oda, J., and Yamada, Y. (1998) Crystal structures of two tropinone reductases: different reaction stereospecificities in the same protein fold, *Proc. Natl. Acad. Sci. U.S.A.* 95, 4876–4881.

51. Tanaka, N., Nonaka, T., Tanabe, T., Yoshimoto, T., Tsuru, D., and Mitsui, Y. (1996) Crystal structures of the binary and ternary complexes of 7 α-hydroxysteroid dehydrogenase from Escherichia coli, *Biochemistry* 35, 7715–7730.

52. Oppermann, U., Filling, C., Hult, M., Shafqat, N., Wu, X., Lindh, M., Shafqat, J., Nordling, E., Kallberg, Y., Persson, B., and

Emodin Cocrystal Structures

Jornvall, H. (2003) Short-chain dehydrogenases/reductases (SDR): the 2002 update, *Chem. Biol. Interact.* 143–144, 247–253.

53. May, A., and Zacharias, M. (2005) Accounting for global protein deformability during protein-protein and protein-ligand docking, *Biochim. Biophys. Acta* 1754 (1–2), 225–231.

54. Tang, Y., Lee, H. Y., Tang, Y., Kim, C. Y., Mathews, I., and Khosla, C. (2006) Structural and functional studies on SCO1815: a $\beta$-ketoacyl-acyl carrier protein reductase from *Streptomyces coelicolor* A3(2), *Biochemistry* 45, 14085–14093.
BI7016427
