
# A Revised Pathway Proposed for Staphylococcus aureus Wall Teichoic Acid Biosynthesis Based on In Vitro Reconstitution of the Intracellular Steps

**Stephanie Brown,**¹ **Yu-Hui Zhang,**¹ and **Suzanne Walker**¹,*

¹Department of Microbiology and Molecular Genetics, Harvard Medical School, 200 Longwood Avenue, Boston, MA 02115, USA  
*Correspondence: suzanne_walker@hms.harvard.edu  
DOI 10.1016/j.chembiol.2007.11.011

## SUMMARY

Resistance to every family of clinically used antibiotics has emerged, and there is a pressing need to explore unique antibacterial targets. Wall teichoic acids (WTAs) are anionic polymers that coat the cell walls of many Gram-positive bacteria. Because WTAs play an essential role in *Staphylococcus aureus* colonization and infection, the enzymes involved in WTA biosynthesis are proposed to be targets for antibiotic development. To facilitate the discovery of WTA inhibitors, we have reconstituted the intracellular steps of *S. aureus* WTA biosynthesis. We show that two intracellular steps in the biosynthetic pathway are different from what was proposed. The work reported here lays the foundation for the discovery and characterization of inhibitors of WTA biosynthetic enzymes to assess their potential for treating bacterial infections.

## INTRODUCTION

Wall teichoic acids (WTAs) are anionic polymers of Gram-positive bacteria that are covalently attached to the peptidoglycan chains (Figure 1) (Neuhaus and Baddiley, 2003). In *Staphylococcus aureus*, WTAs have been shown to play essential roles in host-tissue colonization and the spread of infection (Weidenmaier et al., 2004, 2005). They contribute to bacterial resistance to human lysozyme, which is a component of the innate immune response that destroys bacteria by hydrolyzing the peptidoglycan layers (Bera et al., 2007). WTAs have also been shown to play a role in biofilm formation (Gross et al., 2001; Vinogradov et al., 2006; Walter et al., 2007). Therefore, inhibitors of the WTA biosynthetic pathway may be useful for treating infections caused by *S. aureus* and other Gram-positive pathogens (Weidenmaier et al., 2003). In order to identify and characterize such inhibitors, it is important to understand the steps involved in WTA biosynthesis. Although *S. aureus* strains are known to make polyribitol phosphate (polyRboP) WTAs (Fiedler and Glaser, 1974; Harrington and Baddiley, 1985; Heckels et al., 1975; Yokoyama et al., 1986), the biosynthetic pathway is not well characterized, and the proposed pathway (Figure 2A) is largely based on studies done with *B. subtilis* (Lazarevic et al., 2002; Qian et al., 2006; Ward, 1981).

WTAs consist of a disaccharide-based linkage unit and a repeating polyol-phosphate polymer such as polyglycerol phosphate (polyGroP) or polyRboP (Neuhaus and Baddiley, 2003). WTAs are synthesized on a diphospholipid carrier anchored in the cytoplasmic membrane and are then translocated through a two-component transporter to the outside of the cell, where they are attached by a phosphodiester linkage to the C6 hydroxyl of the N-acetyl muramic acid sugars of peptidoglycan (Figure 1) (Neuhaus and Baddiley, 2003). The most common linkage unit, found in all *B. subtilis* and *S. aureus* strains, is ManNAc-β-(1,4)-GlcNAc-(GroP)n, where n is proposed to be 2 in *B. subtilis* W23 and 3 in *S. aureus* (Harrington and Baddiley, 1985; Yokoyama et al., 1986). Common repeat units include polyGroP, found in *B. subtilis* 168, and polyRboP, found in *B. subtilis* W23 as well as *S. aureus* (Neuhaus and Baddiley, 2003; Yokoyama et al., 1986). Functions have been established for most of the *tag* (for teichoic acid glycerol) genes in *B. subtilis* 168 (Bhavsar et al., 2005; Ginsberg et al., 2006; Qian et al., 2006; Schertzer et al., 2005; Schertzer and Brown, 2003), and putative functions were assigned to the *tar* (for teichoic acid ribitol) genes involved in polyRboP-WTA biosynthesis based on sequence homology to the *tag* genes (Lazarevic et al., 2002; Qian et al., 2006) (Figure 2A). Thus, it was proposed that polyRboP-WTA biosynthesis starts with the TarO-mediated transfer of GlcNAc to a membrane-anchored undecaprenyl carrier lipid. Consistent with this, an *S. aureus* SA113 strain in which *tarO* is disrupted does not express WTAs (Weidenmaier et al., 2004). WTA biosynthesis was proposed to continue with the TarA-mediated transfer of ManNAc to the C4 hydroxyl of GlcNAc to form ManNAc-β-(1,4)-GlcNAc-pp-undecaprenyl, whereupon TarB adds glycerol-3-phosphate to the C4 hydroxyl of the ManNAc moiety (D’Elia et al., 2006; Lazarevic et al., 2002; Qian et al., 2006). The proposed functions of TarA and TarB are thus identical to the established functions of TagA and TagB (Ginsberg et al., 2006; Qian et al., 2006). After the TarB step, the pathways for polyGroP- and polyRboP-WTA biosynthesis diverge. Whereas TagF is a polymerase that adds dozens of GroP units to the disaccharide-based linkage unit in *B. subtilis* 168 (Schertzer et al., 2005; Schertzer and Brown, 2003), TarF is a putative primase. *B. subtilis* TarF was proposed to add one GroP to the disaccharide linkage unit, whereas *S. aureus* TarF was proposed to add two GroPs (D’Elia et al., 2006; Qian et al., 2006). (The *tarO*, *tarA*, *tarB*, and *tarF* genes found in polyRboP-WTA-forming bacteria are sometimes annotated as *tagO*, *tagA*, *tagB*, and *tagF* because of their similarity to the genes involved polyGroP-WTA biosynthesis.) Two additional gene products, a putative ribitol-5-phosphate primase, TarK, and a putative RboP polymerase, TarL, are proposed to build the poly-ribitol-phosphate polymer (D’Elia et al., 2006; Lazarevic

Chemistry & Biology

A Revised Pathway for S. aureus WTA Biosynthesis

mic DNA. The PCR product was cloned into a pET24b(+) vector for expression as a C-terminal hexa-His-tagged protein. The protein was overexpressed in *E. coli* strain Rosetta2(DE3)pLysS and was purified by Ni²⁺-affinity chromatography. SA640 is proposed to transfer ManNAc from UDP-ManNAc to a GlcNAc-pp-undecaprenyl carrier lipid. We have previously shown that *B. subtilis* TagA accepts GlcNAc-pp-lipid substrates containing farnesyl chains and is functional in the absence of membranes (Zhang et al., 2006). Therefore, we incubated SA640 with UDP-ManNAc and the synthetic GlcNAc-pp-farnesyl acceptor substrate under the same conditions used to characterize *B. subtilis* TagA (Zhang et al., 2006). We have previously described the synthesis of these substrates (Ginsberg et al., 2006). HPLC analysis (Figures 3A and 3B) of the reaction showed the disappearance of UDP-ManNAc and the concomitant appearance of a UDP peak. The reaction product was found to have a retention time of 11.7 min, identical to an authentic standard generated in the TagA reaction and previously characterized by MS and ¹H-NMR (Figures 3C and 3D) (Ginsberg et al., 2006; Zhang et al., 2006). The reaction was quenched, and the lipid-linked product was separated from other reaction components over a Phenomenex Strata C18-E column and was analyzed by MS. The product was found to have a mass of 787.3, consistent with compound **2b** (calc’d m/z: 787.2819). These experiments verify that SA640 encodes the UDP-ManNAc transferase TarA, which makes compound **2** (Figure 2B).

et al., 2002; Qian et al., 2006). For *S. aureus* NCTC8325, the source of the enzymes described below, the genes suggested to be responsible for each step in polyRboP-WTA biosynthesis are indicated in Figure 2A.

The pathway proposed in Figure 2A was recently questioned by Qian et al. (2006), who carried out a genomic analysis of several fully sequenced *S. aureus* strains. Their analysis shows that the candidates for TarK and TarL, the two RboP transferases, are very similar in all *S. aureus* strains. Because of the high sequence homology between TarK and TarL, Qian et al. (2006) suggested that both gene products may function as polymerases, and that there may not be a distinct RboP primase and a distinct RboP polymerase (Qian et al., 2006). Here, we report the in vitro reconstitution of the intracellular steps downstream of TarO in the WTA biosynthetic pathway in *S. aureus*. In addition to TarD, TarI, and TarJ, which supply the CDP-glycerol and CDP-ribitol substrates (Badurina et al., 2003; Pereira and Brown, 2004), we show that TarA, TarB, TarF, and TarL are required to convert the GlcNAc-pp-lipid product of TarO to the full-length polyribitol polymer. Based on our studies, we propose a revised WTA biosynthetic pathway for *S. aureus* in which the number of GroP units added by TarF is altered and the proposed polyRboP primase (TarK) is omitted (compare Figures 2A and 2B). This work clarifies the biosynthetic pathway in *S. aureus*, and it may also enable efforts to discover and characterize WTA inhibitors that can be used to assess whether the WTA pathway is a target for therapeutic intervention.

## RESULTS

### In Vitro Reconstitution of S. aureus TarA

The SAOUHSC_00640 gene, encoding SA640, a putative N-acetylmannosaminyl transferase with 55% similarity to *B. subtilis* 168 TagA, was PCR amplified from *S. aureus* NCTC8325 genomic DNA. The PCR product was cloned into a pET24b(+) vector for expression as a C-terminal hexa-His-tagged protein. The protein was overexpressed in *E. coli* strain Rosetta2(DE3)pLysS and was purified by Ni²⁺-affinity chromatography. SA640 is proposed to transfer ManNAc from UDP-ManNAc to a GlcNAc-pp-undecaprenyl carrier lipid. We have previously shown that *B. subtilis* TagA accepts GlcNAc-pp-lipid substrates containing farnesyl chains and is functional in the absence of membranes (Zhang et al., 2006). Therefore, we incubated SA640 with UDP-ManNAc and the synthetic GlcNAc-pp-farnesyl acceptor substrate under the same conditions used to characterize *B. subtilis* TagA (Zhang et al., 2006). We have previously described the synthesis of these substrates (Ginsberg et al., 2006). HPLC analysis (Figures 3A and 3B) of the reaction showed the disappearance of UDP-ManNAc and the concomitant appearance of a UDP peak. The reaction product was found to have a retention time of 11.7 min, identical to an authentic standard generated in the TagA reaction and previously characterized by MS and ¹H-NMR (Figures 3C and 3D) (Ginsberg et al., 2006; Zhang et al., 2006). The reaction was quenched, and the lipid-linked product was separated from other reaction components over a Phenomenex Strata C18-E column and was analyzed by MS. The product was found to have a mass of 787.3, consistent with compound **2b** (calc’d m/z: 787.2819). These experiments verify that SA640 encodes the UDP-ManNAc transferase TarA, which makes compound **2** (Figure 2B).

### In Vitro Reconstitution of S. aureus TarB

The SAOUHSC_00643 gene, encoding SA643, a putative glycerol phosphotransferase with 53% homology to TagB from *B. subtilis* 168, was PCR amplified from *S. aureus* NCTC8325 genomic DNA. The gene was cloned into a pET24b(+) vector, overexpressed in *E. coli* strain Rosetta2(DE3)pLysS as a C-terminal hexa-His-tagged protein, and purified via Ni²⁺-affinity chromatography. SA643 is proposed to transfer glycerol-3-phosphate from CDP-glycerol to the disaccharide-lipid product of TarA. Therefore, we examined its ability to transfer GroP to compound **2b**, which we have previously shown to be a substrate for *B. subtilis* 168 TagB (Ginsberg et al., 2006). Reactions containing ManNAc-β-(1,4)-GlcNAc-pp-lipid (the TarA product) and CDP-glycerol were monitored by HPLC. The CDP-glycerol peak (retention time of 7.56 min) decreased in intensity, and a new peak, corresponding to CMP (retention time of 3.85 min), appeared. Under the same conditions, SA643 was unable to utilize CDP-ribitol as a substrate, as judged by the stability of the CDP-ribitol peak (retention time of 7.54 min) in the HPLC trace (Figures 4A–4C). Using radiolabeled [¹⁴C]-CDP-glycerol, we could also monitor the SA643 reaction by polyacrylamide gel electrophoresis followed by phosphorimaging analysis. The gel (Figure 4D) shows a new spot in the enzyme reaction that is not present in the heat-treated control. This new spot has the same mobility as the authentic product generated by reacting [¹⁴C]-CDP-glycerol and compound **2b** with *B. subtilis* 168 TagB. The lipid-linked product from a TarB reaction utilizing nonradioactive substrates was purified over a C18 column and subjected to MS analysis. The purified product has a mass of 941.3, consistent with structure **3b** (calc’d m/z: 941.2851). These results verify that SA643 encodes the glycerol phosphate transferase TarB, which makes compound **3** (Figure 2B).

Chemistry & Biology

A Revised Pathway for S. aureus WTA Biosynthesis

**Figure 2. Proposed and Reconstituted WTA Biosynthetic Pathway in S. aureus**

(A) The proposed polyribitol-phosphate WTA biosynthetic pathway. The S. aureus NCTC8325 gene encoding each putative protein is shown by its locus tag number in the block arrow next to the enzyme name.

(B) The revised polyribitol-phosphate WTA pathway in S. aureus based on in vitro reconstitution. R = undecaprenyl (compounds 1a–5a) for the natural lipid carrier or farnesyl (compounds 1b–5b) for the alternative lipid used in the in vitro experiments. Differences between the proposed and reconstituted pathways are highlighted by larger bold text.

---

**In Vitro Reconstitution of S. aureus TarF**

SAOUHSC_00223, encoding SA223, which is 67% similar to B. subtilis W23 TarF, was cloned from S. aureus NCTC8325 genomic DNA, overexpressed in E. coli, and purified over a Ni²⁺-affinity column. S. aureus TarF is proposed to transfer two units of glycerol-3-phosphate from CDP-glycerol to compound 3. To assess the function of SA223, we incubated the purified enzyme with purified [¹⁴C]-glycerol-3-phosphate-ManNAc-β-(1,4)-GlcNAc-pp-lipid (3b) and either [¹⁴C]-CDP-glycerol or CDP-ribitol. Reactions were monitored by polyacrylamide gel electrophoresis (Figure 5A). A lower spot appeared in the [¹⁴C]-CDP-glycerol reaction that was not present in the heat-treated controls or in the CDP-ribitol reaction. The new spot migrates faster than the starting material, but it was found to depend on the presence of both [¹⁴C]-CDP-glycerol and the TarB product. Therefore, we carried out an enzymatic reaction with nonradiolabeled 3b and CDP-glycerol, purified the lipid-linked material over a C18 column, and subjected the material to MS analysis. The product was found to have an m/z of 1095.3 (calc’d m/z: 1095.2882), which corresponds to the addition of one glycerol-3-phosphate unit to compound 3b to form compound 4b. No products containing additional GroP units were observed under any conditions. We have concluded that SA223 is TarF, and that TarF in S. aureus NCTC8325 functions to add one unit of glycerol-3-phosphate to compound 3 to produce (glycerol-3-phosphate)₂-ManNAc-β-(1,4)-GlcNAc-pp-lipid (compound 4)

14 Chemistry & Biology 15, 12–21, January 2008 ©2008 Elsevier Ltd All rights reserved

Chemistry & Biology

A Revised Pathway for S. aureus WTA Biosynthesis

![](image.png)

Figure 3. HPLC and LC-MS Chromatograms of the TarA Enzymatic Reaction

(A and B) HPLC chromatograms after incubation of UDP-ManNAc with compound **1b** and (A) heat-treated SA640 or (B) active enzyme.

(C and D) Retention times and intensities of the extracted product ions (m/z = 787.3) for (C) the authentic product, compound **2b**, and (D) the product generated by incubating UDP-ManNAc and compound **1b** with SA640.

(E) The in vitro reaction catalyzed by SA640 (TarA); the experimental m/z value is shown (calculated m/z: 787.2819).

(Figure 2B). Unless the functions of TarF homologs in other *S. aureus* strains are different, we propose that *S. aureus* TarF, like *B. subtilis* W23 TarF, transfers a single GroP unit.

### In Vitro Reconstitution of the Ribitolphosphotransferase Polymerase, TarL

All sequenced *S. aureus* strains contain two genes that share high homology to one another and to the *B. subtilis* W23 *tarL* and *tarK* genes (Qian et al., 2006). In *B. subtilis* W23, TarK is proposed to be a primase that adds one or a few units of ribitol-5-phosphate, whereas TarL is proposed to be a ribitol-5-phosphate polymerase that adds many RboP units (D’Elia et al., 2006; Lazarevic et al., 2002; Qian et al., 2006). Qian et al. (2006) have suggested that the two candidate genes for TarK and TarL in *S. aureus* may actually be duplicates of one another and may have similar functions. To assess the functions of the two putative RboP transferases in *S. aureus* NCTC8325 (SA222 and SA227), we cloned the genes for these proteins from genomic DNA into pET24b(+), expressed them in *E. coli*, and purified the expressed proteins by Ni²⁺-affinity column chromatography. SA222 was incubated with \[^{14}C\]-**4b** and CDP-ribitol, and the reaction mixture was analyzed by polyacrylamide gel electrophoresis; however, no change in the mobility of the radiolabeled starting material was observed. Similarly, no change was observed when CDP-glycerol was included in the reaction. Concerned that a product involving the addition of a single RboP unit might not separate from the starting material under the gel-electrophoresis conditions used, we analyzed a set of nonradioactive reactions by HPLC. SA222 was separately incubated with compound **3b** and compound **4b** and either CDP-ribitol or CDP-glycerol, but we did not observe under any conditions a decrease in peaks corresponding to starting material or the appearance of a peak for CMP (data not shown).

Unable to reconstitute any activity for SA222, we subjected SA227 to a similar series of experiments. SA227 was incubated with \[^{14}C\]-**4b** and either CDP-ribitol or CDP-glycerol, and the reactions were analyzed by polyacrylamide gel electrophoresis

(Figure 6A). In this case, \[^{14}C\]-**4b** reacted to produce a higher band in the presence of CDP-ribitol, but not in the presence of CDP-glycerol. No slower-migrating band was visible in the heat-treated control or in reactions containing \[^{14}C\]-**3b** rather than \[^{14}C\]-**4b**. We also used \[2-³H\]-CDP-ribitol as a substrate along with \[^{14}C\]-**4b** in the SA227 reactions. The reaction mixtures were spotted onto Whatman 3MM paper strips, and the strips were developed by paper chromatography. Nonpolymeric starting materials migrated up the strips, whereas polymeric product remained at the origin. Scintillation counting showed that the material at the origin was labeled with both \[^{14}C\) and \[^{3}H\) in a ratio of >1:17 under the reaction conditions used, which confirms the attachment of multiple \[2-³H\]-ribitol-5-phosphate units to the \[^{14}C\]-TarF product. Taken together, these results show that SA227 transfers multiple RboP units to the disaccharide **4b**, which contains two GroP units, but not to the disaccharide **3b**, which contains one GroP unit.

We incubated the product of the SA227 reaction with SA222 and CDP-ribitol, but we did not detect any change in the length of the polyRboP-WTA polymer, suggesting that SA222 does not further extend this product.

The results reported above establish that SA227 is a ribitol-5-phosphate polymerase that transfers ribitol-5-phosphate from CDP-ribitol to compound **4** to form the (ribitol-5-phosphate)_n-(glycerol-3-phosphate)_2-ManNAc-β-(1,4)-GlcNAc-pp-undecaprenyl product **5** (Figure 2B). We have hence identified SA227 as the polymerase TarL. We have also concluded that a ribitol-5-phosphate primase is not required for the synthesis of poly-RboP-WTA polymers in *S. aureus*, as it apparently is in *B. subtilis* W23 (Lazarevic et al., 2002). The function of SA222, suggested first to be a primase and later to be another polymerase, remains to be established.

### In Vitro Reconstitution of WTA Synthesis

We carried out a tandem reaction of TarA, TarB, TarF, and TarL to produce WTA polymers in vitro from the GlcNAc-pp-lipid substrate **1b**. The synthetic WTA polymers formed in vitro were

Chemistry & Biology 15, 12–21, January 2008 ©2008 Elsevier Ltd All rights reserved 15

Chemistry & Biology

A Revised Pathway for S. aureus WTA Biosynthesis

![](image.png)

Figure 4. HPLC Chromatograms and Radioactive Gel Analysis of TarB Enzymatic Reactions

(A–C) HPLC chromatograms after incubation of CDP-glycerol or CDP-ribitol after incubation with compound **2b** and either heat-treated or untreated SA643. (A) CDP-glycerol reaction with heat-treated enzyme. (B) CDP-glycerol reaction with active enzyme. (C) CDP-ribitol reaction with active enzyme.

(D) Autoradiogram of a polyacrylamide gel for reactions of compound **2b** and SA643 in the presence of \[^{14}C\]-CDP-glycerol. (+) and (−) symbols designate the presence or absence, respectively, of the components indicated on the left; (Δ) indicates heat treatment. Lane 1, heat-treated enzyme reaction; lane 2, active enzyme reaction; lane 3, authentic standard of \[^{14}C\]-3b generated by using previously characterized *B. subtilis* TagB.

(E) The in vitro reaction catalyzed by TarB (SA643); the experimental m/z value is shown (calculated m/z: 941.2851).

analyzed on a polyacrylamide gel that also contained WTAs extracted from S. aureus NCTC8325 cells (Figure 6B). The synthetic polymers could be stained with Alcian blue, an established method for staining WTAs. The most abundant polymers synthesized in vitro are ~10 units smaller than the most abundant extracted WTAs, which have been estimated to contain ~40 RboP units (Ward, 1981). We were unable to significantly increase the lengths of the WTA polymers produced in vitro by manipulating substrate ratios or enzyme concentrations. The molecular basis for the control of polymer length is not known, but it is evident from the distribution of polymeric products that TarL is a processive enzyme. Additional factors may control polymer lengths and product distributions in bacterial cells, explaining the longer lengths of the natural WTA polymers. Whether WTA polymer length is of any biological significance is not known.

DISCUSSION

Bacterial resistance to all main classes of antibiotics has developed. Most current antibiotics target a few essential bacterial pathways, including DNA replication, cell wall biosynthesis, bacterial protein biosynthesis, and folate biosynthesis (Walsh and Wright, 2005). It remains to be seen how many new inhibitors can be developed to these well-studied pathways. There exists a need to explore new targets. Virulence factors, which are involved in the progression of disease in the host, but are not critical for the survival of the bacterium in vitro, are intriguing targets. Virulence factors include toxins as well as factors important for colonization and infection of host tissue (Marra, 2006). Recent studies support the promise of small-molecule inhibitors of virulence factors for treating bacterial infection (Hung et al., 2005).

S. aureus, one of the major nosocomial pathogens, is covered in WTAs, anionic polymers that are attached to peptidoglycan and that play a variety of important but poorly understood roles in the biology of these and many other Gram-positive organisms (Gotz, 2004). Recent experiments showed that deleting the first gene in the WTA biosynthetic pathway, *tarO*, produces mutants that are viable in vitro but are unable to colonize epithelial or endothelial tissue (Weidenmaier et al., 2004, 2005). These studies highlighted the essential role that WTAs play in S. aureus infection and have prompted interest in understanding the WTA biosynthetic pathway in more detail since it may be a target for antibacterial agents. Brown and coworkers recently reported that several genes downstream of *tarO* in the proposed S. aureus biosynthetic pathway cannot be deleted (D’Elia et al., 2006), and it was suggested that deletions of these genes may be lethal because toxic intermediates accumulate in the cell and/or because a blockade in the WTA biosynthetic pathway diverts building blocks, such as undecaprenyl phosphate, that would otherwise be recycled and used in essential metabolic pathways such as

16 Chemistry & Biology 15, 12–21, January 2008 © 2008 Elsevier Ltd All rights reserved

Chemistry & Biology

A Revised Pathway for S. aureus WTA Biosynthesis

**Figure 5. Radioactive Gel Analysis of TarF Enzymatic Reactions**

(A) Autoradiogram of a polyacrylamide gel for TarF-catalyzed reactions of \[^{14}C\]-3b and either \[^{14}C\]-CDP-glycerol or CDP-ribitol. (+) and (−) symbols designate the presence or absence, respectively, of the components indicated on the left; (Δ) indicates heat treatment. Lane 1, heat-treated enzyme reaction containing \[^{14}C\]-CDP-glycerol; lane 2, active enzyme reaction containing \[^{14}C\]-CDP-glycerol; lane 3, heat-treated enzyme reaction containing CDP-ribitol; lane 4, active enzyme reaction containing CDP-ribitol.

(B) The in vitro reaction catalyzed by TarF (SA223); the experimental m/z value is shown (calculated m/z: 1095.2882).

peptidoglycan biosynthesis (D’Elia et al., 2006). The fact that disruption of *tar* genes leads to a lethal phenotype increases interest in the WTA biosynthetic pathway as a target for intervention, but it also makes it challenging to characterize the pathway in detail by using a genetic approach. Therefore, we decided to establish the functions of several of the genes proposed to be involved in WTA biosynthesis via in vitro reconstitution of key intracellular steps.

We cloned, overexpressed, and purified eight *S.* aureus NCTC8325 enzymes, tentatively identified as TarA, TarB, TarD, TarF, TarI, TarJ, TarK, and TarL. Five of these enzymes were proposed to be involved in the conversion of compound 1 to compound 5 via the linear sequence of steps shown in Figure 2A. The other three, TarD, TarI, and TarJ (Badurina et al., 2003; Pereira and Brown, 2004), were cloned to enable the synthesis of CDP-glycerol and CDP-ribitol in both radiolabeled and nonradio-labeled forms. In addition, we synthesized via chemical methods both UDP-ManNAc and compound 1b, which are the donor and acceptor substrates, respectively, for TarA. The acceptor substrate we prepared, compound 1b, differs from the natural substrate in that it contains a farnesyl chain rather than the undecaprenyl carrier lipid, but we have previously established that other WTA biosynthetic enzymes are not very sensitive to the structure of the attached lipid chain (Ginsberg et al., 2006; Zhang et al., 2006). Therefore, there was precedent for the use of synthetic alternative substrates in reconstituting the WTA biosynthetic pathway of *S.* aureus.

We were able to reconstitute the activities of four of the five Tar pathway enzymes proposed to be involved in the conversion of compound 1 to compound 5. Although the four enzymes, TarA, TarB, TarF, and TarL, operate on a membrane-anchored, undecaprenyl-containing substrate in cells, none of them are predicted to have membrane-spanning domains, and none require the undecaprenyl substrate or a membrane interface for activity. Kinetic studies have not been done, but the enzymes have reasonable activity with the alternative substrate since all of the reactions largely went to completion in 2 hr. We were not able to reconstitute the activity of the fifth putative Tar pathway enzyme, the TarK/TarL homolog SA222; however, the work reported here clearly shows that it is not required to form polyRboP-WTA polymers.

Based on the results of the reconstitution experiments, we have revised the *S.* aureus *tar* pathway in two ways. First, we have altered the proposed function of TarF. This enzyme had been thought to add two GroP units (D’Elia et al., 2006; Harrington and Baddiley, 1985; Yokoyama et al., 1986), but we have shown here that it adds only one. There may be several explanations for the discrepancy between earlier results and the results reported here. For example, the *S.* aureus strain that was originally analyzed to determine the structure of the linkage unit may have been unusual in having a TarF that adds two GroP units. Alternatively, the *S.* aureus strain we used as a source of genomic DNA may be unusual in having a TarF that adds only one unit. There may be an as yet undiscovered enzyme in this or other *S.* aureus strains that adds a third GroP unit; if so, however, the third GroP unit is not required for extension of the polyRboP chain. Finally, it is possible that the original structural analyses were incorrect. Analysis of polymers fractionated from complex biological mixtures is more complicated than analysis of the products of an enzymatic reaction in vitro. At this point, we cannot distinguish among the possibilities. Therefore, we have revised the function of *S.* aureus TarF in Figure 2B with the caveat that generality has not yet been established for all *S.* aureus TarF homologs.

The other revision is more substantive. We have shown that TarL, the poly(RboP) polymerase, can act directly on the TarF product (compound 4), indicating that there is no need in *S.* aureus for an RboP primase. The putative primase was thus removed from the proposed pathway (compare Figures 2A and 2B). The function of SA222, a candidate for the primase encoding a gene product that shows 88% end-to-end homology to SA227, which we have identified as TarL, could not be established from these in vitro experiments. However, all sequenced *S.* aureus strains, which now number 12, contain a pair of genes

Chemistry & Biology 15, 12–21, January 2008 ©2008 Elsevier Ltd All rights reserved 17

Chemistry & Biology

A Revised Pathway for S. aureus WTA Biosynthesis

**Figure 6. Analysis of TarL, SA227, Enzymatic Reactions**

(A) Autoradiogram of a polyacrylamide gel for TarL-catalyzed reactions of \[^{14}C\]-4b or \[^{14}C\]-3b and either CDP-glycerol or CDP-ribitol. (+) and (−) symbols designate the presence or absence, respectively, of the components indicated on the left; (Δ) indicates heat treatment. Only lane 3, which contains \[^{14}C\]-4b and CDP-ribitol in addition to active enzyme, shows the disappearance of radiolabeled starting material and the formation of a radiolabeled higher product.

(B) Polyacrylamide gel of WTAs stained with silver and Alcian blue according to an established method for detecting WTA polymers. Lane 1, WTAs synthesized in vitro from UDP-ManNAc, compound 1b, CDP-glycerol, and CDP-ribitol by the tandem action of TarA, TarB, TarF, and TarL. Lane 2, WTAs extracted from S. aureus NCTC8325 cells.

(C) The in vitro reaction catalyzed by TarL (SA227).

steps of the WTA biosynthetic pathway in a clinically relevant pathogen paves the way for the discovery of inhibitors.

**EXPERIMENTAL PROCEDURES**

**Reagents**

Vectors, expression hosts, and His-Bind resin were obtained from Novagen. Acrylamide solutions were purchased from National Diagnostics, TEMED was purchased from American Bioanalytical, and TBE was purchased from Bio-Rad. All other enzymes, reagents, and buffers were obtained from Sigma-Aldrich.

**Cloning, Expression, and Purification of Putative Enzymes**

The SAOUHSC_00640, SAOUHSC_00643, SAOUHSC_00223, SAOUHSC_00222, SAOUHSC_00227, and tarD, tarl, and tarJ genes were PCR amplified from Staphylococcus aureus NCTC8325 genomic DNA. The primer pairs used for amplification are included in Table S1 (see the Supplemental Data available with this article online). The PCR products were subcloned into pET24b(+) (Novagen) at the Xhol and BamHI restriction sites for expression in E. coli Rosetta2(DE3)pLysS (Novagen) as C-terminal hexa-His-tagged proteins. The proteins were expressed in mid-log phase cultures after induction with 0.5 mM or 1 mM IPTG for 4 hr at 37°C, except for SA222 and SA227, which were induced at 16°C for overnight growth. Cells were lysed by freeze-thaw, sonication, and French pressure at 16,000 lb/in² in buffer supplemented with rLysozyme and Benzonase (Novagen) and protease inhibitor cocktail (CalBioChem). SA222 and SA227 lysis buffer was 50 mM HEPES (pH 7.0), 200 mM NaCl, 0.5% CHAPS, 5% glycerol buffer. The remaining proteins were lysed in 100 mM Tris-HCl (pH 7.5), 500 mM NaCl, 0.6% CHAPS, 0.5% Triton X-100. Clarified lysate was purified by Ni²⁺-affinity chromatography (Novagen His-Bind resin), yielding 5 mg/l TarA, 5 mg/l TarD, 1 mg/l TarB, 8 mg/l TarF, 7 mg/l Tarl, 3 mg/l TarJ, 0.5 mg/l TarK as determined by Lowry Assay. The proteins were stored as 20% and 50% glycerol stocks at both −80°C and −20°C.

**Preparation of UDP-ManNAc, \[^{14}C\]-CDP-Glycerol, Ribitol-5-Phosphate, and Compounds 1b, 2b, and 3b**

UDP-ManNAc, compound 1b, and compound 2b were prepared as we previously described (Ginsberg et al., 2006). \[^{14}C\]-CDP-glycerol was prepared as

**SIGNIFICANCE**

Wall teichoic acids (WTAs) are virulence factors in Gram-positive bacteria (Weidenmaier et al., 2004). Their involvement in bacterial colonization and infection suggest that WTA biosynthesis may be a target for therapeutic intervention. Molecules that inhibit WTA biosynthesis could be used to determine whether targeting virulence factors would have efficacy as antibiotics. Using in vitro reconstitution of enzymatic activity, we have delineated several intracellular steps in the WTA biosynthetic pathway in S. aureus. To our knowledge, this is the first biochemical study assigning the function of gene products involved in the synthesis of WTAs from a polyribitol-forming strain, and the work has led to two corrections in the proposed biosynthetic pathway in S. aureus. The reconstitution of the intracellular

18 Chemistry & Biology 15, 12–21, January 2008 ©2008 Elsevier Ltd All rights reserved

Chemistry & Biology

A Revised Pathway for S. aureus WTA Biosynthesis

previously described by using purified TarD (Badurina et al., 2003). Ribitol-5-phosphate was prepared as described previously (Bigham et al., 1984; Zolli et al., 2001). Briefly, 0.1 g ribulose-5-phosphate was dissolved in water. After the solution pH was confirmed to be above 7, sodium borohydride was added to give a 1:1 molar ratio. The mixture was incubated at room temperature overnight and was quenched with acetic acid (to pH 5), and NaOH was added to adjust the pH to 8. The addition and removal by rotary evaporator of MeOH was used to rid the resulting borate salts. The reaction products were analyzed by Benedict’s Reagent to confirm the disappearance of ribulose-5-phosphate. The reaction was purified by using Bio-Rad P2 gel. \[^{14}C\]-radiolabeled compound **3b** was prepared by incubating TarB (500 nM) overnight with 10 μM compound **2** and 30 μM \[^{14}C\]-CDP-glycerol at room temperature in 20 mM Tris-HCl (pH 7.5), 100 mM NaCl, and 10 mM MgCl₂ buffer. The product, \[^{14}C\]-labeled **3b**, was purified with a C18 column (Accubond SPE ODS-C18) by washing with 100% water and eluting product with 100% ethanol.

### Preparation of CDP-Ribitol and \[^{2-3}H\]-CDP-Ribitol

Nonradiolabeled CDP-ribitol was prepared by incubating TarI overnight with 1 mM ribitol-5-phosphate and 2 mM CTP in 50 mM HEPES (pH 8), 10 mM MgCl₂, 1 mM DTT, and 1 U inorganic pyrophosphatase (Pereira and Brown, 2004). The reaction was monitored by HPLC by using an anion-exchange column, Phenosphere 5μ SAX 250 × 4.6 mm, 5 μm (Phenomenex), to record the disappearance of the CTP peak and production of a new peak (Buffer A: 5 mM NH₄H₂PO₄ [pH 2.8]; Buffer B: 750 mM NH₄H₂PO₄ [pH 3.7]; linear gradient of 0%–20% solution B over 60 min, UV monitored at 271 nm). The identity of CDP-ribitol was confirmed by mass spectroscopy. CDP-ribitol was used unpurified in subsequent enzymatic reactions after spinning the reaction in a 3000 MWCO filter to remove the enzyme. \[^{2-3}H\]-radiolabeled CDP-ribitol was prepared by extension of a previously outlined protocol to make \[^{2-3}H\]-ribitol-5-phosphate and CDP-ribitol (Landau et al., 1996; Pereira and Brown, 2004). A duplicate nonradiolabeled reaction was used to monitor the reaction progress by analytical anion-exchange HPLC (Phenosphere 5μ SAX 250 × 4.6 mm, 5 μm [Phenomenex]). A 1:200 mixture of \[^{2-3}H\]-D-glucose:D-glucose (2 mM total), 3 mM ATP, 4 mM NADP, 1.5 U glucose-6-phosphate dehydrogenase, 1 U hexokinase, 1 U 6-phosphogluconate dehydrogenase was incubated at 30°C in 100 mM triethanolamine buffer (pH 7.6), 10 mM MgCl₂. After 3 hr, 1 μM ZnCl₂, 1 mM DTT, and TarJ were added. After an additional 4 hr, the reaction was spun in a 3000 MWCO filter to remove the enzymes. The resulting ribitol-5-phosphate product was purified by using a previously established method (Bartlett, 1959). Briefly, the reaction was applied to a 1 ml Dowex 1 × 8–400 anion-exchange resin (Cl⁻ form). The product was purified by using a stepwise gradient of HCl and was found to elute with 0.02 N HCl. The pH of the collected product was adjusted to 7 by using NaOH, and the solvent was evaporated to concentrate the product. 1 U inorganic pyrophosphatase, 2 mM CTP, and TarI were added to the purified ribitol-5-phosphate, and the reaction was allowed to proceed for 1 hr at 30°C in 100 mM triethanolamine buffer (pH 7.6), 10 mM MgCl₂. The resulting \[^{2-3}H\]-CDP-ribitol was used in subsequent enzymatic reactions.

of 500 nM TarF was incubated with 50 μM compound **2b**, 500 nM TarB, and 350 μM CDP-glycerol in buffer (20 mM Tris-HCl [pH 7.5], 100 mM NaCl, and 10 mM MgCl₂). The products were purified by using a Phenomenex Strata C18-E column by washing with 100% water and eluting product with 50:50 water:methanol. MS-TOF experiments were performed on a Waters mass spectrometer. The purified compounds (TarA, TarB, and TarF products) were run in negative ion mode by using water with 0.1% ammonium formate as solvent. For ESI-MS analysis, mass spectra were acquired by using an Agilent 1100 series LC/MSD mass spectrometer. After the mass of the purified compound was obtained, unpurified TarA product and purified TagA product (obtained as described previously [Ginsberg et al., 2006]) were subjected to LC-MS analysis (extracted ion of 787.3 from the LC trace) by using a Zorbax 300SB-C18, 5 μm, 4.6 × 250 mm (Agilent) column. The compounds were eluted at a flow rate of 0.5 ml/min by using a step gradient: 0%–60% solution B for 8 min, 60%–100% solution B for 30 s, and 100% solution B for 7.5 min (solution A, water, solution B, methanol, both solutions were supplemented with 0.1% ammonium hydroxide as a solvent modifier).

### TarB, TarF, TarK, and TarL Enzymatic Reactions for Polyacrylamide Gel Electrophoresis Analysis

Reactions containing 1 μM purified compound **2b** and 2 μM \[^{14}C\]-CDP-glycerol were incubated with TarB (500 nM). TarF reactions were performed by using 500 nM enzyme, 2 μM purified \[^{14}C\]-labeled compound **3b**, and 4 μM either CDP-ribitol or \[^{14}C\]-CDP-glycerol. TarK and TarL reactions were performed by reacting 500 nM enzyme with 1 μM purified \[^{14}C\]-labeled compound **4b** or 1 μM purified \[^{14}C\]-labeled compound **3b** and 200 μM CDP-ribitol or CDP-glycerol. TarB, TarF, TarK, and TarL enzymatic reactions were performed in buffer containing 20 mM Tris-HCl (pH 7.5), 100 mM NaCl, and 10 mM MgCl₂. All enzymatic reactions were quenched with an equal volume of DMF after 2.5 hr. \[^{14}C\]-TagB product was obtained by using \[^{14}C\]-CDP-glycerol and by following our previously established method (Ginsberg et al., 2006).

### Polyacrylamide Gel Electrophoresis Assay for TarB, TarF, TarK, and TarL

The acrylamide gels used are similar to those described previously (Pollack and Neuhaus, 1994). Bio-Rad minigels (7.0 cm × 8.3 cm [H × W]; 1.0 mm thickness) were constructed of 20% acrylamide/0.25 M TBE by mixing 4 ml of a prepared acrylamide stock solution (Protogel, 30% [w/v] acrylamide: 0.8% [w/v] bisacrylamide), 0.25 ml water, 1.69 ml 10× TBE solution, 5 μl TEMED, and 20 μl 10% ammonium persulfate. A total of 2 μl quenched enzymatic reactions were added to 2 μl 2× loading buffer consisting of 50% glycerol and 0.1% bromophenol blue. The gels were electrophoresed in 0.25 M TBE buffer running at a constant 100V for 107 min by using the Bio-Rad Mini-PROTEAN system. The gels were dried between two sheets of cellophane, and the dried gels were exposed to a tritium storage phosphor screen (GE Healthcare) for ~48 hr. The screen was imaged by using a Typhoon 9400 imager and analyzed by using the ImageQuant TL computer software.

### Paper Chromatography Assay to Determine the Incorporation of RboP Repeats

Reactions (6 μl) containing 0.5 μM purified compound **4b**, 100 μM \[^{2-3}H\]-CDP-ribitol, and 500 nM TarL enzyme were incubated at room temperature for 2.5 hr in 20 mM Tris base (pH 7.5), 10 mM MgCl₂, and 100 mM NaCl. The reaction was performed in triplicate. The reactions were quenched with 6 μl DMF and 3 μl spotted onto 3 MM Whatman paper in triplicate. The reactions were spotted 3 cm from the bottom of the paper and were developed by using 5:3 isobutyric acid:1 M NH₄OH. The solvent was allowed to run to the top of the 20 cm paper. The paper was dried, and the origin was cut out and placed in eppendorf tubes. Water (750 μl) was added, and the tubes were placed at 100°C for 4 hr. Heating samples allowed for a consistent amount of radioactive quenching (10% for 14C and 20% for 3H). Ultima Gold scintillation cocktail (5 ml) was added, and the samples were analyzed by using a scintillation counter.

### WTA Polyacrylamide Gel Electrophoresis for TarL and Natural WTAs

A 150 μl reaction containing 1 μM purified compound **1b** and 4 μM UDP-ManNAc was incubated at room temperature for 30 min with 500 nM TarA in 20 mM Tris-HCl (pH 7.9) and 500 mM NaCl. 5 μM CDP-glycerol, 400 μM CDP-ribitol, and 10 mM MgCl₂ were added. Water (600 ml) was then added to dilute the

### HPLC Assay for TarA and TarB

TarA (500 nM) was incubated at room temperature for 2.5 hr with 50 μM UDP-ManNAc and 100 μM compound **1b** in buffer (20 mM Tris-HCl [pH 7.9], 500 mM NaCl). TarB (500 nM) was incubated at room temperature for 2.5 hr with 50 μM CDP-glycerol or 50 μM CDP-ribitol and 100 μM compound **2b** in buffer (20 mM Tris-HCl [pH 7.5], 100 mM NaCl, 10 mM MgCl₂). Reactions were quenched with an equal volume of DMF, applied to an analytical anion-exchange HPLC column (Phenosphere 5μ SAX 250 × 4.6 mm, 5 μm [Phenomenex]). TarA reactions were eluted by using a linear gradient of 0%–5% solution B over 30 min (buffer A: 5 mM NH₄H₂PO₄ [pH 4.5]; buffer B: 750 mM NH₄H₂PO₄ [pH 3.7]). TarB reactions were eluted by using a linear gradient from 0% solution B to 100% solution B over 25 min (buffer A: 5 mM NH₄H₂PO₄ [pH 2.8]; buffer B: 750 mM NH₄H₂PO₄ [pH 3.7]). UDP-ManNAc and UDP were monitored at 260 nM. CDP-glycerol and CMP were monitored at 271 nM.

### HPLC-MS and MS Assays

TarA and TarB reaction conditions were the same as described above, except TarA reactions utilized 50 μM compound **2b** and 150 μM compound **1b** and TarB reactions utilized 50 μM compound **3b** and 200 μM CDP-glycerol. A total

Chemistry & Biology 15, 12–21, January 2008 ©2008 Elsevier Ltd All rights reserved 19

higher concentration of salt that was found to be slightly inhibitory to later-stage enzymes, 500 nM each of TarB, TarF, and TarL were added, and the reaction was allowed to proceed for an additional 2 hr at room temperature. The reactions were quenched with 600 μl MeOH and evaporated by using a speed vacuum. The reaction was resuspended in 15 μl 50:50 water:DMF and 120 mM NaOH (so as to mimic the final step in WTA isolation from cells in which the linkage between ManNAc and GroP is cleaved). The reaction mixture was shaken at room temperature for 15 min, and the reaction was loaded onto a 16 cm (1.0 mm thickness) polyacrylamide gel. For the separating gel, a 30% total acrylamide and 6% crosslinking solution was made by adding 340 mg Bisacrylamide to 30 ml Protogel solution. A total of 19.9 ml of this acrylamide solution was added to 10 ml 3M Tris(HCl) (pH 8.5), 300 μl 10% ammonium persulfate, and 30 μl TEMED. The stacking gel was prepared by mixing 1 ml Protogel solution, 3 ml 3 M Tris(HCl) (pH 8.5), 6 ml water, 100 μl 10% ammonium persulfate, and 10 μl TEMED. The running buffer consisted of 0.1 M Tris base and 0.1 M Tricine (pH 8.1). A lane containing bromophenol blue was used as a marker of gel progress. The gel was electrophoresed until the bromophenol blue was 1 inch from the bottom. The gel was stained with Alcian blue and silver staining as described previously (Wolters et al., 1990). Wild-type WTA was a gift from Timothy Meredith.

**Supplemental Data**

Supplemental Data include a table listing the primer pairs used for PCR amplification and are available at [http://www.chembiol.com/cgi/content/full/15/1/12/DC1/](http://www.chembiol.com/cgi/content/full/15/1/12/DC1/).

**ACKNOWLEDGMENTS**

This work was supported by a National Institutes of Health grant. S.B. is funded by a National Science Foundation Fellowship. We thank Timothy Meredith (Harvard Medical School, Boston, MA) for providing extracted WTAs from S. aureus NCTC8325. We thank Xiao Fang for assistance in cloning TarI and TarJ.

Received: September 26, 2007  
Revised: November 9, 2007  
Accepted: November 14, 2007  
Published: January 25, 2008

**REFERENCES**

Badurina, D.S., Zolli-Juran, M., and Brown, E.D. (2003). CTP:glycerol 3-phosphate cytidylyltransferase (TarD) from Staphylococcus aureus catalyzes the cytidylyl transfer via an ordered Bi-Bi reaction mechanism with micromolar K(m) values. Biochim. Biophys. Acta 1646, 196–206.

Bae, T., Banger, A.K., Wallace, A., Glass, E.M., Aslund, F., Schneewind, O., and Missiakas, D.M. (2004). Staphylococcus aureus virulence genes identified by bursa aurealis mutagenesis and nematode killing. Proc. Natl. Acad. Sci. USA 101, 12312–12317.

Bartlett, G.R. (1959). Methods for the isolation of glycolytic intermediated by column chromatography with ion exchange resins. J. Biol. Chem. 234, 459–465.

Bera, A., Biswas, R., Herbert, S., Kulauzovic, E., Weidenmaier, C., Peschel, A., and Gotz, F. (2007). Influence of wall teichoic acid on lysozyme resistance in Staphylococcus aureus. J. Bacteriol. 189, 280–283.

Bhavsar, A.P., Truant, R., and Brown, E.D. (2005). The TagB protein in Bacillus subtilis 168 is an intracellular peripheral membrane protein that can incorporate glycerol phosphate onto a membrane-bound acceptor in vitro. J. Biol. Chem. 280, 36691–36700.

Bigham, E.C., Gragg, C.E., Hall, W.R., Kelsey, J.E., Mallory, W.R., Richardson, D.C., Benedict, C., and Ray, P.H. (1984). Inhibition of arabinose 5-phosphate isomerase. An approach to the inhibition of bacterial lipopolysaccharide biosynthesis. J. Med. Chem. 27, 717–726.

D’Elia, M.A., Pereira, M.P., Chung, Y.S., Zhao, W., Chau, A., Kenney, T.J., Sulavik, M.C., Black, T.A., and Brown, E.D. (2006). Lesions in teichoic acid biosynthesis in Staphylococcus aureus lead to a lethal gain of function in the otherwise dispensable pathway. J. Bacteriol. 188, 4183–4189.

Fiedler, F., and Glaser, L. (1974). The synthesis of polyribitol phosphate. I. Purification of polyribitol phosphate polymerase and lipoteichoic acid carrier. J. Biol. Chem. 249, 2684–2689.

Ginsberg, C., Zhang, Y.H., Yuan, Y., and Walker, S. (2006). In vitro reconstitution of two essential steps in wall teichoic acid biosynthesis. ACS Chem. Biol. 1, 25–28.

Gotz, F. (2004). Staphylococci in colonization and disease: prospective targets for drugs and vaccines. Curr. Opin. Microbiol. 7, 477–487.

Gross, M., Cramton, S.E., Gotz, F., and Peschel, A. (2001). Key role of teichoic acid net charge in Staphylococcus aureus colonization of artificial surfaces. Infect. Immun. 69, 3423–3426.

Harrington, C.R., and Baddiley, J. (1985). Biosynthesis of wall teichoic acids in Staphylococcus aureus H, Micrococcus varians and Bacillus subtilis W23. Involvement of lipid intermediates containing the disaccharide N-acetylmannosaminyl N-acetylglucosamine. Eur. J. Biochem. 153, 639–645.

Heckels, J.E., Archibald, A.R., and Baddiley, J. (1975). Studies on the linkage between teichoic acid and peptidoglycan in a bacteriophage-resistant mutant of Staphylococcus aureus H. Biochem. J. 149, 637–647.

Hung, D.T., Shakhnovich, E.A., Pierson, E., and Mekalanos, J.J. (2005). Small-molecule inhibitor of Vibrio cholerae virulence and intestinal colonization. Science 310, 670–674.

Landau, B.R., Wahren, J., Chandramouli, V., Schumann, W.C., Ekberg, K., and Kalhan, S.C. (1996). Contributions of gluconeogenesis to glucose production in the fasted state. J. Clin. Invest. 98, 378–385.

Lazarevic, V., Abellan, F.X., Moller, S.B., Karamata, D., and Mael, C. (2002). Comparison of ribitol and glycerol teichoic acid genes in Bacillus subtilis W23 and 168: identical function, similar divergent organization, but different regulation. Microbiology 148, 815–824.

Marra, A. (2006). Targeting virulence for antibacterial chemotherapy: identifying and characterising virulence factors for lead discovery. Drugs R D. 7, 1–16.

Neuhaus, F.C., and Baddiley, J. (2003). A continuum of anionic charge: structures and functions of D-alanyl-teichoic acids in gram-positive bacteria. Microbiol. Mol. Biol. Rev. 67, 686–723.

Pereira, M.P., and Brown, E.D. (2004). Bifunctional catalysis by CDP-ribitol synthase: convergent recruitment of reductase and cytidylyltransferase activities in Haemophilus influenzae and Staphylococcus aureus. Biochemistry 43, 11802–11812.

Pollack, J.H., and Neuhaus, F.C. (1994). Changes in wall teichoic acid during the rod-sphere transition of Bacillus subtilis 168. J. Bacteriol. 176, 7252–7259.

Qian, Z., Yin, Y., Zhang, Y., Lu, L., Li, Y., and Jiang, Y. (2006). Genomic characterization of ribitol teichoic acid synthesis in Staphylococcus aureus: genes, genomic organization and gene duplication. BMC Genomics 7, 74.

Schertzer, J.W., and Brown, E.D. (2003). Purified, recombinant TagF protein from Bacillus subtilis 168 catalyzes the polymerization of glycerol phosphate onto a membrane acceptor in vitro. J. Biol. Chem. 278, 18002–18007.

Schertzer, J.W., Bhavsar, A.P., and Brown, E.D. (2005). Two conserved histidine residues are critical to the function of the TagF-like family of enzymes. J. Biol. Chem. 280, 36683–36690.

Vinogradov, E., Sadovskaya, I., Li, J., and Jabbouri, S. (2006). Structural elucidation of the extracellular and cell-wall teichoic acids of Staphylococcus aureus MN8m, a biofilm forming strain. Carbohydr. Res. 341, 738–743.

Walsh, C., and Wright, G. (2005). Introduction: antibiotic resistance. Chem. Rev. 105, 391–394.

Walter, J., Loach, D.M., Alqumber, M., Rockel, C., Hermann, C., Pfitzenmaier, M., and Tannock, G.W. (2007). D-alanyl ester depletion of teichoic acids in Lactobacillus reuteri 100-23 results in impaired colonization of the mouse gastrointestinal tract. Environ. Microbiol. 9, 1750–1760.

Ward, J.B. (1981). Teichoic and teichuronic acids: biosynthesis, assembly, and location. Microbiol. Rev. 45, 211–243.

Weidenmaier, C., Kristian, S.A., and Peschel, A. (2003). Bacterial resistance to antimicrobial host defenses—an emerging target for novel antiinfective strategies? Curr. Drug Targets 4, 643–649.

Chemistry & Biology

A Revised Pathway for S. aureus WTA Biosynthesis

Weidenmaier, C., Kokai-Kun, J.F., Kristian, S.A., Chanturiya, T., Kalbacher, H., Gross, M., Nicholson, G., Neumeister, B., Mond, J.J., and Peschel, A. (2004). Role of teichoic acids in Staphylococcus aureus nasal colonization, a major risk factor in nosocomial infections. Nat. Med. 10, 243–245.

Weidenmaier, C., Peschel, A., Xiong, Y.Q., Kristian, S.A., Dietz, K., Yeaman, M.R., and Bayer, A.S. (2005). Lack of wall teichoic acids in Staphylococcus aureus leads to reduced interactions with endothelial cells and to attenuated virulence in a rabbit model of endocarditis. J. Infect. Dis. 191, 1771–1777.

Wolters, P.J., Hildebrandt, K.M., Dickie, J.P., and Anderson, J.S. (1990). Polymer length of teichuronic acid released from cell walls of Micrococcus luteus. J. Bacteriol. 172, 5154–5159.

Yokoyama, K., Miyashita, T., Araki, Y., and Ito, E. (1986). Structure and functions of linkage unit intermediates in the biosynthesis of ribitol teichoic acids in Staphylococcus aureus H and Bacillus subtilis W23. Eur. J. Biochem. 161, 479–489.

Zhang, Y.H., Ginsberg, C., Yuan, Y., and Walker, S. (2006). Acceptor substrate selectivity and kinetic mechanism of Bacillus subtilis TagA. Biochemistry 45, 10895–10904.

Zolli, M., Kobric, D.J., and Brown, E.D. (2001). Reduction precedes cytidylyl transfer without substrate channeling in distinct active sites of the bifunctional CDP-ribitol synthase from Haemophilus influenzae. Biochemistry 40, 5041–5048.
