
REVIEW ARTICLE

Penicillin-binding proteins and β-lactam resistance

André Zapun¹, Carlos Contreras-Martel² & Thierry Vernet¹

¹Laboratoire d’Ingénierie des Macromolécules, Institut de Biologie Structurale Jean-Pierre Ebel, UMR 5075-CNRS, CEA, Université Joseph Fourier Grenoble, France; and ²Laboratoire des Protéines Membranaires, Institut de Biologie Structurale Jean-Pierre Ebel, UMR 5075-CNRS, CEA, Université Joseph Fourier, Grenoble, France

Correspondence: Thierry Vernet, Laboratoire d’Ingénierie des Macromolécules, Institut de Biologie Structurale Jean-Pierre Ebel, UMR 5075-CNRS, CEA, Université Joseph Fourier, 41, rue Jules Horowitz, 38027 Grenoble, France. Tel.: +33 4 38 78 96 81; fax: +33 4 38 78 54 94; e-mail: thierry.vernet@ibs.fr

Received 6 June 2007; revised 13 September 2007; accepted 23 October 2007. First published online February 2008.

DOI:10.1111/j.1574-6976.2007.00095.x

Editor: Jacques Coyette

Keywords
Staphylococcus aureus; Streptococcus pneumoniae; Neisseria; Enterococcus

Abstract
A number of ways and means have evolved to provide resistance to eubacteria challenged by β-lactams. This review is focused on pathogens that resist by expressing low-affinity targets for these antibiotics, the penicillin-binding proteins (PBPs). Even within this narrow focus, a great variety of strategies have been uncovered such as the acquisition of an additional low-affinity PBP, the over-expression of an endogenous low-affinity PBP, the alteration of endogenous PBPs by point mutations or homologous recombination or a combination of the above.

What are PBPs?
Since their discovery as targets of the β-lactams, the penicillin-binding proteins (PBPs) have been the subject of intense research, particularly regarding their role in the resistance to β-lactams of some important pathogens such as Staphylococcus aureus, Enterococci and Streptococcus pneumoniae.
The penicillin-binding domains of PBPs are transpeptidases or carboxypeptidases involved in peptidoglycan metabolism (Goffin & Ghuyse, 2002; Sauvage *et al.*, 2008). These domains harbor three specific motifs: SXXK, (S/Y)XN and (K/H)(S/T)G, which define the active-site serine penicillin-recognizing enzymes family (for ASPRE) that also includes the class A and C β-lactamases. The serine of the SXXK motif is central to the catalytic mechanism. The serine attacks the carbonyl of the penultimate D-Ala amino acid of the stem peptide, which releases the last D-Ala amino acid from the ‘donor’ peptide and forms a covalent acyl–enzyme complex. In transpeptidases, the carbonyl of the D-Ala amino acid, now forming an ester linkage with the active site serine, then undergoes an attack from a primary amine linked in various ways to the third residue of a second ‘acceptor’ stem peptide. A peptide bridge is then created between two stem peptides, forming a link between glycan strands (Fig. 1a). In DD-carboxypeptidases, the acyl-enzyme intermediate is hydrolyzed; this process eliminates ‘donor’ stem peptides from the peptidoglycan.
β-Lactams mimic the D-Ala-D-Ala dipeptide in an elongated conformation, particularly regarding the distribution of three electrostatic-negative wells (Fig. 1b–d), and act as suicide inhibitors. The active site serine attacks the carbonyl of the β-lactam ring, resulting in the opening of the ring and formation of a covalent acyl-enzyme complex. This complex is hydrolyzed very slowly, thus effectively preventing further reactions.
The following kinetic model describes the reaction of PBPs with β-lactams. A noncovalent complex EI is formed between the enzyme E and the inhibitor I, with the dissociation constant K_d, from which acylation proceeds to form the covalent complex EI* with the rate k_2. EI* is hydrolyzed at the rate k_3 to regenerate E and an inactivated product P (Fig. 2). The rate described by k_3 is negligible on the time scale of a bacterial generation. The following nomenclature

FEMS Microbiol Rev 32 (2008) 361–385
© 2008 Federation of European Microbiological Societies Published by Blackwell Publishing Ltd. All rights reserved

Fig. 1. (a) Catalysis of transpeptidation. Fragments of glycan strands are represented by chains of hexagones standing for the hexoses N-acetyl glucosamine (G) and N-acetyl muramic acid (M). The 'donor' pentapeptide is depicted on the upper glycan strand, whereas the 'acceptor' is attached here on the lower strand. The peptides shown are those from Streptococcus pneumoniae. The second and third amino acids may differ in various species. Note that in many instances, including in Streptococcus pneumoniae, various intervening amino acids are attached to the third residue of the acceptor peptide, and provide the free amine that attacks the acyl-enzyme intermediate. Such stem peptides are called 'branched'. Structural similarity between β-lactams and the natural substrate of the PBPs. (b) N-Acyl-d-Alanyl-d-Alanine peptide. (c) Penicillin backbone. (d) Cephalosporin backbone. The regions of negative electrostatic potential are indicated by arcs.

E + I $\xrightleftharpoons[k_{-1}]{k_1}$ EI $\xrightarrow{k_2}$ EI* $\xrightarrow{k_3}$ E + P

$K_d = k_{-1}/k_1$

Fig. 2. Kinetic scheme of the reaction between a PBP (E) and a β-lactam (I). EI represents a preacylation noncovalent complex. EI* represents the covalent acyl-enzyme complex. P is the open inactivated product.

will be used throughout this review. The constants $k_2$ and $k_3$ describe the acylation and deacylation rates, respectively. The second-order rate constant $k_2/K_d$ is the efficiency of acylation, which allows calculation of the overall acylation rate at a given antibiotic concentration. The inhibitory potency of a β-lactam for a PBP is given by the $c_{50}$, which is the antibiotic concentration resulting in the inhibition of half the PBP molecules at a steady state (when the acylation and deacylation reactions proceed at the same rate). The value of $c_{50}$ is equal to the ratio $k_3/(k_2/K_d)$. Here, as in the literature in general, PBPs are said to be or to have high or low affinity for β-lactams. This affinity implicitly refers to the $c_{50}$, and should not be confused with the strength of a noncovalent interaction.

Despite detailed kinetic studies and several crystal structures of PBPs, the reaction mechanism is still a matter of debate, and likely differs between various ASPRE enzymes, and even for a single protein and different β-lactams (Oliva et al., 2003).

### PBP-based β-lactam resistance

Inhibition of PBPs produces an imbalance in cell wall metabolism, resulting in growth inhibition or lysis. The link between PBP inhibition and the biological outcome remains poorly understood [e.g. Escherichia coli (de Pedro et al., 2002), Staphylococcus aureus (Giesbrecht et al., 1998), Enterococcus hirae (Pucci et al., 1986), Streptococcus pneumoniae (Mascher et al., 2006)]. Despite the ignorance of the detailed consequences of β-lactam treatment, various means of resistance have been uncovered and investigated, such as a

Low-affinity PBPs

decreased permeability of the outer membrane, export of the antibiotics (these mechanisms are restricted to gram-negative bacteria), degradation of the antibiotic by β-lactamases or utilization of PBPs with low affinity for the β-lactams. The following sections will be concerned with PBPs of organisms that exploit this latter strategy. Note that the emphasis of this review is mostly biochemical, and that the term resistance is often used to qualify sometimes modest decreases of susceptibility; the definition of resistance in this review is thus rather biological, meaning a decreased susceptibility resulting from selection, rather than medical according to susceptibility thresholds set by various national agencies.

**Staphylococcus aureus**

After the spread of *Staphylococcus aureus* strains resistant to penicillin through the acquisition of a β-lactamase, methicillin was introduced, which was not degraded by β-lactamases. A methicillin-resistant strain was isolated soon afterwards (Jevons et al., 1963). The so-called methicillin-resistant *Staphylococcus aureus* (MRSA) strains are particularly dangerous as they exhibit blanket resistance to virtually all β-lactams, often associated with resistance to other classes of antibiotics.

The β-lactam resistance of MRSA strains results from the expression, in addition to the four native PBPs, of a fifth low-affinity PBP termed PBP2a or PBP2′ (Hartman & Tomasz, 1984; Utsui & Yokota, 1985). Note that the expression of PBP2a (about 800 molecules cell⁻¹) does not alter the amount of the other PBPs (comprised individually between 150 and 450 molecules cell⁻¹) (Pucci & Dougherty, 2002). PBP2a is the product of the *mecA* gene whose transcription is controlled by the *mecI* and *mecR1* regulatory elements. MecI represses *mecA* transcription (Garcia-Castellanos et al., 2004). By analogy with the homologous BlaI and BlaR1 system that controls the expression of the β-lactamase BlaZ, the Mec system is thought to function as follows (Zhang et al., 2001): MecR1 is a signal-transduction protein with an extracellular penicillin-binding domain that senses the presence of β-lactams and activates its cytoplasmic domain. The intracellular domain of MecR1 is a protease that undergoes activation, which results directly or indirectly in the cleavage of the MecI repressor. The *mecA* gene and its regulatory system are found on a mobile genetic element called the staphylococcal cassette chromosome *mec* that integrates in the chromosome (Ito et al., 2001). A detailed presentation of these genetic elements can be found elsewhere (Hiramatsu et al., 2001; Ender et al., 2004).

Interestingly, the intact *mec* system does not confer resistance, as the expression of PBP2a is normally well repressed. Only a few β-lactams can alleviate this repression. Mutations, for example in *mecI* or in the operator region,

lead to de-repression of *mecA*. Even so, strains with unrestricted expression of PBP2a exhibit resistance only in a small subpopulation, when maintained without β-lactam selection. A homogenous resistant population is selected only upon exposure to β-lactams. When the antibiotic pressure is removed, heterogeneity is restored, indicating that the functioning of PBP2a in cell wall synthesis bears a cost that is best avoided in the absence of β-lactams. The genetic determinants of homogenous resistance in wild strains remain mysterious.

The class B PBP2a lacks the glycosyltransferase activity that is also required for peptidoglycan synthesis. Although PBP2a supports all the transpeptidase activity when this activity is inhibited by β-lactams in the four native PBPs, the presence of the glycosyltransferase domain of the class A PBP2 is nevertheless required (Pinho et al., 2001a,b). Most surprisingly, the presence of the native class B PBP1 is also required, even though the transpeptidation can be carried out by PBP2a (Pereira et al., 2007).

Other genes are necessary for the expression of the resistance conferred by PBP2a. Over 30 of these auxiliary genes, *aux* or *fem* (for factor essential for methicillin resistance) have been identified (De Lencastre et al., 1999). Several *fem* genes are involved in cell wall metabolism; other genes participate into putative regulatory or sensory functions. The *femAB* operon products, for example, add the second to fifth glycine residues to the peptidoglycan precursor to form the pentaglycine branch that serves as the cross-bridge of staphylococcal peptidoglycan (Henze et al., 1993; Stranden et al., 1997). The trivial expectation that PBP2a requires ‘acceptor’ peptides with a pentaglycine branch turned out to be naïve, for PBP2a can confer resistance to Enterococci, which lack *femAB* and have alternative peptidoglycan cross-bridges (Arbeloa et al., 2004a).

PBP2a belongs to a subgroup of class B PBPs characterized by an insertion of about 100 residues following the transmembrane anchor. This group includes chromosomally encoded PBP5 from Enterococci, which are low-affinity PBPs involved in some degree of β-lactam resistance. Other members of this subgroup are present in *Bacillus subtilis* and related species, in *Listeria monocytogenes* and *innocua* and in *Clostridium acetobutylicum*, although these do not appear to confer reduced susceptibility to β-lactams. The origin of PBP2a remains mysterious. Close homologues have been found in susceptible and resistant *Staphylococcus sciuri* strains, and in resistant *Staphylococcus vitulinus, capitis* and *kloosii* (Wu et al., 1998; Couto et al., 2003; Schnellmann et al., 2006) (Fig. 3). The *mec* system may thus have spread from a hitherto unidentified species, not only to *Staphylococcus aureus* but also to at least 14 other pathogenic or commensal staphylococcal species (Pierre et al., 1990b; Yasuda et al., 2000; Busscher et al., 2006; Schnellmann et al., 2006).

364

|  | 330 | 340 | 350 | 360 | 370 | 380 | 390 | 400 | 410 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Q4GXX7 | N |  |  |  | N | D.A.K |  |  |  |
| Q4GXY0 | N |  |  |  | N | D.A.K |  |  |  |
| Q53707 |  |  |  |  |  |  |  |  |  |
| P07944 |  |  |  |  |  |  |  |  |  |
| Q3T5M1 |  |  |  |  |  |  |  |  |  |
| Q4GXY3 |  |  |  |  |  |  |  |  |  |
| Q5HJW3 | TIDAKVQKSIYNNMKNDYGSGTAIHPQTGELLALVSTPSYDVYPFMYGMSNEEYNKLTEDKKEPLLLNKFQITTS PGSTQKILTAMIGLNN |  |  |  |  |  |  |  |  |
| Q5HK31 |  |  |  |  |  |  |  |  |  |
| O54283 |  |  |  |  |  |  |  |  |  |
| O54272 | R | D | K |  | N | ND.D.K | D |  |  |
| P96018 | R | D |  |  | N | D.D.K | D |  |  |
| O54277 | R | D |  |  | N | D.D.K | D | I |  |
| Q2VTQ2 | SR | D |  |  | N | D.D.K | D |  |  |

|  | 420 | 430 | 440 | 450 | 460 | 470 | 480 | 490 | 500 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Q4GXX7 | Q.N | N |  | D |  | Q |  |  |  |
| Q4GXY0 | Q.N | N |  | D |  | Q |  |  |  |
| Q53707 |  |  |  |  |  |  |  |  |  |
| P07944 |  |  |  |  |  |  |  |  |  |
| Q3T5M1 |  |  |  |  |  |  |  |  |  |
| Q4GXY3 |  |  |  |  |  | I |  |  |  |
| Q5HJW3 | KTLDDKTSYKIDGKGWQDKSWGGYNVTRYEVVNGNIDLKQAIESSDNIFFARVALELGSKKFEKGMKKLGVGEDIPSDYPFYNAQISNK |  |  |  |  |  |  |  |  |  |
| Q5HK31 |  |  |  |  |  |  |  |  |  |
| O54283 |  |  |  |  |  |  |  |  |  |
| O54272 | G | N | D | D |  | E | R |  |  |
| P96018 | G | N | D | AD |  | E | R |  |  |
| O54277 | N | D | D |  |  | E | R | Y |  |
| Q2VTQ2 | N | D | D |  |  | E | R |  |  |

|  | 510 | 520 | 530 | 540 | 550 | 560 | 570 | 580 | 590 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Q4GXX7 | D |  | S | V | E | R | P | V |  |
| Q4GXY0 | D |  | S | V | F | E | R | P | V |
| Q53707 |  |  |  |  |  |  | N |  |  |
| P07944 |  |  |  |  |  |  | N |  |  |
| Q3T5M1 |  |  |  |  |  |  |  |  |  |
| Q4GXY3 |  |  |  |  |  |  |  |  |  |
| Q5HJW3 | NLDNEILLADSGYGQGEILINPVQILSIYSALENNGNINAPHLLKDTKNVWKKNIIISKENINLLTDGMQQVVNKTHKEDIYRSYANLIG |  |  |  |  |  |  |  |  |  |
| Q5HK31 |  |  |  |  |  |  |  |  |  |
| O54283 |  |  |  |  |  |  |  |  |  |
| O54272 | K | V | V |  | Q | K | R | V |  |
| P96018 | K | V | V |  | Q | K | R | V |  |
| O54277 | K | V | V |  | Q | K | R | V |  |
| Q2VTQ2 | K | V | V |  | Q | K | R | V |  |

|  | 600 | 610 | 620 | 630 | 640 | 650 | 660 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Q4GXX7 |  |  |  | D |  |  |  |
| Q4GXY0 |  |  |  | D |  |  |  |
| Q53707 |  | S |  |  |  |  |  |
| P07944 |  |  |  |  |  |  |  |
| Q3T5M1 |  |  |  |  |  |  |  |
| Q4GXY3 |  |  |  |  |  |  |  |
| Q5HJW3 | KSGTAELKMKQGETGRQIGWFISYDKDNPNMMMMAINVKDVQDKGMASYNAKISGKVYDELYENGNKKYDIDE |  |  |  |  |  |  |  |
| Q5HK31 |  |  |  |  |  |  |  |
| O54283 |  | P | HL | P |  |  |  |
| O54272 | Q | N |  | D | D | K.T.R | K |  |
| P96018 | Q | I |  | D | D | K.T.R | K |  |
| O54277 | Q |  |  | E | D | D | K.T.R | K |
| Q2VTQ2 | Q | N |  | D | D | K.T.N | K |  |

Fig. 3. Sequence alignment of staphylococcal PBP2a transpeptidase domains 327–668 (designated by their Uniprot accession numbers). Three types of sequences have been found. Sequences on a white background are from MRSA. The most commonly found sequence (Q5HJW3) in MRSA has also been isolated from *Staphylococcus kloosii*, *sciuri*, *vitulinus*, haemolyticus, *epidermidis* and *capitis*. The catalytic motifs are in black boxes. Sequences in dark gray were isolated from *Staphylococcus vitulinus*, *capitis* and *kloosii*. Sequences in light gray are from *Staphylococcus sciuri* strains.

Low-affinity PBPs

PBP2a reacts with β-lactams extremely slowly. The acylation efficiency of PBP2a by penicillin G, characterized by the second-order rate constant $k_2/K_\mathrm{d}=15\,\mathrm{M}^{-1}\,\mathrm{s}^{-1}$, is one to three orders of magnitude slower than that of the four native PBPs from *Staphylococcus aureus* (Chambers et al., 1994; Graves-Woodward & Pratt, 1998; Lu et al., 1999). The acylation rate of PBP2a at therapeutic concentrations of β-lactams is thus negligible compared with the bacterial generation time ($t_{1/2}$ for acylation > 1 h with 10 μM of penicillin).

The low efficiency of acylation appears to result both from a poor ‘true’ affinity of PBP2a for the β-lactams, with dissociation constants ($K_\mathrm{d}$) of the preacylation complex in the millimolar range, and extremely slow acylation rates ($k_2$) ranging from 0.2 to 0.001 s⁻¹ (Lu et al., 1999; Fuda et al., 2004). The acylation rate $k_2$ of PBP2a by penicillin G, for example, is three orders of magnitude slower than that of the susceptible PBP2x from *Streptococcus pneumoniae* (Lu et al., 1999, 2001).

The structure of a soluble form of PBP2a has been solved to a resolution of 1.8 Å (Lim & Strynadka, 2002). The active site of PBP2a appears to be somewhat closed without a bound antibiotic, with the active site S403 poorly positioned for a nucleophilic attack and a twisting of strand β3. The penicillin-binding domain of ASPRE proteins can be described as consisting of two subdomains, α-helical and α/β, surrounding the active site. The structures of PBP2a with bound antibiotics revealed a tilt of the whole helical subdomain with respect to the α/β-subdomain (O. Dideberg, pers. commun.). This rotation opens and substantially rearranges the active site. The Oγ of S403 is displaced by 1.8 Å (with nitrocefin), whereas strand β3 is straightened (Fig. 4). It has been argued that this rearrangement is costly and impedes acylation. Although possible, this and other explanations rely on the assumption that the conformation of the acyl-enzyme complexes are relevant to the transition state of the acylation reaction. In this regard, it must be remembered that there is a complete absence of correlation between the efficiency of acylation ($k_2/K_\mathrm{d}$) and the strength of the noncovalent interaction between the covalently bound antibiotic and the PBP, as demonstrated with *Escherichia coli* PBP5 (Beadle et al., 2001). Therefore, analysis of the complementarity of bound open antibiotics may bear little relevance to the understanding of the acylation process.

However, support for a substantial rearrangement of PBP2a upon acylation is provided by circular dichroism measurements that showed a diminution of the helix content with kinetics consistent with the acylation rates by several β-lactams (Fuda et al., 2004, 2006). Binding of synthetic peptidoglycan fragments also caused rearrangements detected by circular dichroism (Fuda et al., 2005). As binding of the peptidoglycan surrogate molecules led to an increase of the acylation rate by β-lactams, it has been

![Figure 4](https://i.imgur.com/yourimageurl.png)

Fig. 4. Superposition of the active site of *Staphylococcus aureus* PBP2a without (magenta) and with (green) bound methicillin (shown green sticks). The Oγ of the catalytic S403 is shown as a red ball. Note the displacement of the active site serine and the conformational change of strand β3, which are further apart following acylation by methicillin. The figure was prepared using the PDB entries #1 MWU and #1 MVR (Lim & Strynadka, 2002).

proposed that the catalytic activity of PBP2a, and possibly of PBPs in general, might be stimulated allosterically by the peptidoglycan (Fuda et al., 2004, 2006).

In the past decade, several novel carbapenems and cephalosporins have been described that are good inhibitors of PBP2a, including some in clinical trials (e.g. Hebeisen et al., 2001 and references therein; Fuda et al., 2006; Page, 2006). Detailed kinetic studies have shown that the improved efficacy of three novel cephalosporins resulted from better preacylation binding, with $K_\mathrm{d}$ in the range of tens of micromolar, whereas the acylation rate $k_2$ was unchanged compared with other antibiotics (Fuda et al., 2006).

Mildly resistant strains of *Staphylococcus aureus* have also been isolated that lack both *mecA* and β-lactamases. In one clinical strain, resistance to a small set of structurally related cephalosporin was correlated with poor binding to PBP2, while PBP3 was absent or did not react with penicillin (Georgopapadakou et al., 1982). Alterations of penicillin binding by PBP1, PBP2 and an elevated amount of PBP4 were observed in some strains (Tomasz et al., 1989; Henze & Berger-Bachi, 1996). The acylation rates of PBP1 and PBP2 were decreased, and the deacylation rates increased (Chambers et al., 1994). The S569A and A576S substitutions were shown to cause a 10-fold decrease of the acylation efficiency of PBP2 (Hackbarth et al., 1995). Another variant with the A450D and A462V substitutions surrounding the SXN motif and the Q629D mutation has a $k_2/K_\mathrm{d}$ reduced 20-fold (Hackbarth et al., 1995). A laboratory mutant selected with ceftizoxime has the single substitution P458L close to the SXN catalytic motif (Leski & Tomasz, 2005).

Thus, *Staphylococcus aureus* has been found to resist β-lactams in three ways: using β-lactamases to degrade

FEMS Microbiol Rev **32** (2008) 361–385

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved
the antibiotic, by lowering the affinity of its endogenous PBPs for β-lactams and most dangerously through the recruitment of an additional PBP that is unaffected by β-lactams.

### Enterococci

Enterococci have an intrinsic low susceptibility or resistance to β-lactams. *Enterococcus faecalis* typically has minimum inhibitory concentrations (MICs) for penicillin of 2–8 mg L⁻¹ (e.g. Perez et al., 1987), and *Enterococcus faecium* of 16–32 mg L⁻¹ (e.g. Moellering et al., 1979). These important human pathogens have been the subject of intense molecular studies, together with *Enterococcus hirae*, which is more of a veterinary concern.

The intrinsic moderate resistance to β-lactams results from the presence of a particular high-molecular-weight (MW) class B PBP, which takes over the transpeptidase function of the other PBPs when these are inhibited by antibiotics (Fontana et al., 1983, 1985). Three lines of evidence in early studies of an *Enterococcus hirae* strain and several derivatives (initially identified as *Streptococcus faecium* ATCC 9790) led to this conclusion. Firstly, one of the high-MW PBPs (PBP5) has a much lower affinity for penicillin, and spontaneous mutants with greater resistance have elevated amounts of this PBP (Fontana et al., 1983). Secondly, a mutant hypersensitive to penicillin lacks PBP5 expression (Fontana et al., 1985). Finally, saturation of PBP5 with β-lactams leads to bacterial death (Lleo et al., 1987). Note that a peculiar strain of *Enterococcus hirae* were found to express a second low-affinity PBP of the same family, termed PBP3r, in addition to PBP5. In contrast to PBP5, PBP3r is plasmid-encoded (Raze et al. 1998).

Subsequent studies uncovered the same mechanism causing intrinsic β-lactam resistance in *Enterococcus faecium* (Williamson et al., 1985; Sifaoui et al., 2001) and *Enterococcus faecalis* (Duez et al., 2001). Note that, confusingly, the low-affinity PBP of *Enterococcus faecalis* is sometimes termed PBP4. The elevated levels of resistance exhibited by clinical isolates of *Enterococcus faecium* was found to arise from increased expression of PBP5 and mutations of PBP5 that further decrease its affinity for β-lactams (Klare et al., 1992; Fontana et al., 1994). Strains with low susceptibility (MIC for ampicillin of 8 mg L⁻¹) appear to rely mainly on the overexpression of PBP5, the reasons for which are still unclear. Extremely resistant strains (MIC for ampicillin of up to 512 mg L⁻¹) appear to combine both overexpression and reduced affinity (Klare et al., 1992; Fontana et al., 1994) or use only the latter mechanism (Zorzi et al., 1996; Rybkine et al., 1998). Note that the exclusive use of the PBP5 transpeptidase, when the others are inhibited by β-lactams, does not modify the composition of the peptidoglycan cross-bridges (Sifaoui et al., 2001).

Several point mutations in PBP5 were found to be correlated with a low affinity for β-lactams and high resistance of *Enterococcus faecium* (Fig. 5a, Table 1) (Ligozzi et al., 1996; Zorzi et al., 1996; Rybkine et al., 1998). However, as isolates are not isogenic, assessment of the effect of various PBP5 sequences awaited their introduction into a single strain. When three PBP5 sequences from strains with MICs for ampicillin of 2, 24 and 512 mg L⁻¹ were introduced in a strain with no PBP5 expression (MIC ampicillin of 0.03 mg L⁻¹), the resulting strains had MICs of 6, 12 and 20 mg L⁻¹, respectively (Sifaoui et al., 2001). These results show that variants of PBP5 confer different MICs, but that this effect is strongly modulated by unknown factors (Sifaoui et al., 2001; Jureen et al., 2004). The particular mutation M485A was hypothesized to have a very important effect as it was found in two highly resistant strains and is located close to the second catalytic motif SXN482 (Zorzi et al., 1996). When introduced individually, this mutation caused only a small increase of resistance, correlated with a modest decrease of the affinity for penicillin (Zorzi et al., 1996; Sifaoui et al., 2001; Rice et al., 2004). Smaller effects resulted from three other mutations investigated individually: I499T, E629V and the introduction of an additional Ser466’s. However, when combined with the other mutations, in particular M485A, the additional Ser466’ caused a nearly threefold increase of the MIC for ampicillin (Rice et al., 2004). This study also revealed that the various individual mutations tested had different effects on the MICs of different β-lactams.

Enterococcal PBP5 belongs to the same subgroup of class B PBPs as staphylococcal PBP2a (el Kharroubi et al., 1991). The crystal structure of *Enterococcus faecium* PBP5 bound to penicillin was solved to a resolution of 2.4 Å (Sauvage et al., 2002). The resistance of the originating strain (D63r, MIC for penicillin of 70 mg L⁻¹) appears to result from overproduction of the same PBP5 found in the parental strain (D63) (Zorzi et al., 1996). The structure is therefore that of a ‘wild-type’ PBP5, without substitutions that further decrease the affinity for β-lactams. The efficiency of acylation by penicillin of D63r *Enterococcus faecium* PBP5 defined by the second-order rate constant $k_2/K_\mathrm{d} = 20 \, \mathrm{M}^{-1} \, \mathrm{s}^{-1}$ is similar to that of *Staphylococcus aureus* PBP2a, that is two to three orders of magnitude slower than that of a ‘regular’ high-affinity PBP (Zorzi et al., 1996). Investigation of the apparent affinity and reaction rate of a variety of β-lactams with a soluble form of PBP5 has shown that the acylation is sensitive to the various substituents grafted onto the antibiotic, thus offering some scope for improving drugs against enterococci (Hujer et al., 2005).

As no structure was obtained in the absence of antibiotic, no comment could be made regarding a possible rearrangement upon acylation, although some loop residues, which are conserved in this subgroup of PBPs (residues 461–465),
Low-affinity PBPs

(a)
350 360 370 380 390 400 410 420 430 440 450
Q47801 P
Q47783 S
Q9S6C2 D
Q47763 D
Q93NP3
Q3XZN6
Q59438
Q93T65
Q47759 TIDAKAQKTAFDLSGGKAGSTVATTPKTGDLLALASSPSYDPNKMTNGISQEDYKAYEENPEQPFISR FATGYAPGSTFKMITAAIGLDNGTIDPNEVLTINGLKWO
Q47751 S
460 470 480 490 500 510 520 530 540 550 560
Q K T D A
Q T K I D
S Q A KL T D
S Q A K T D
Q T K T D
Q K T D
Q
KDSSWGSYQVTRVS-DVSHVDLKTALIYSDNIYMAQETLKMGEKNFRAGLDKFIFGEDLDLPISMNPAQSNEESFNSDILLADTGYGQGELLINPIQQAMYSVFANNGITLVY
Q K I D
570 580 590 600 610 620 630 640 650 660 670
L V S
L V S
V S
V S
Y E V A S
D S
PKLIADKETDKKNVIGETAVQTIVPDLREVVQDVNGTAHSL SALGIPLAAKTGTAEIKEKQDEKGKENSFLFAFNPDNQGYMMVSMLENKEDDSATKRAPELLQYLNQNYO
R

(b)
350 360 370 380 390 400 410 420 430 440 450
Q9K3C9
Q6PY39 A
Q9AM37
Q47800 TIDSGVQQQAFADFKRPGSAVITDPQKGDLLATVSSPSYDPNKMANGISQKEYDAYNNNKDLPTARFATGYAPGSTFKIITGAIGLDAGTLKPDEELEINGLKWO
Q6PY40 T
Q5EFS1 T
Q47759 AKA.KT.DSLGGKA.T.A.T.KT.LA.T.ED.K.EE.PEQ.IS.M.A.N.ID.N.V.T.
460 470 480 490 500 510 520 530 540 550 560
QDKSWGGYFATRVKEASPVNLR TALVNSDNIIYFAQOTLRMGEDKFRAGLNKFIFGEELDLPIAMTPAQISNEDKFNSEILLADTGYGQQLLISPIQQATMYSVFQNNGTLVY
S S
S.S.QV.SDV.H.D.K.IY M.E.K.KN.D.S.N.ES.D.E.N.A.A
570 580 590 600 610 620 630 640 650 660 330
PKLVLDKETKKKDNVISANAANTIATDLLGSVEDPSGVYNMYNPNFSLAAKTGTAEIKDKQDTDGKENSFLLTLD RSNNKFLTIMVENSGENGSA TDISKPLIDYLEATIK
IA.D.K.GET.VQ.VP.REV.Q.VN.TAHSL SALGIP E.EK.FAFNPD.QGYMMVS.L.KEDDD.KRAPE.LQ.NQNYO

Fig. 5. Sequence variation of *Enterococcus faecium* PBP5 (a) and *Enterococcus faecalis* PBP4 (b) transpeptidase domain. An *Enterococcus faecium* PBP5 sequence (Uniprot number #47759) is also shown in (b) to illustrate the degree of divergence with *Enterococcus faecalis* PBP4. Catalytic motifs are blackened. The PBP5M485A substitution that was shown to increase resistance is highlighted in gray. The PBP4 mutations in positions 520 and 605 associated with greater resistance are in gray.

FEMS Microbiol Rev 32 (2008) 361–385
© 2008 Federation of European Microbiological Societies
Published by Blackwell Publishing Ltd. All rights reserved

Table 1. Characteristics of Enterococcus faecium strains and their PBP 5, for which sequences are publicly available

| Strain   | MIC (mg L⁻¹)         | Uniprot # | Expression level       | $k_{2}/K_{d}$ (M⁻¹s⁻¹)* | MIC (mg L⁻¹) conferred to a recipient strain* |
|----------|----------------------|-----------|------------------------|-------------------------|-----------------------------------------------|
| BM4107   | 2(Amp) *             | Q93T65    | Unchanged *            | 6                       |                                               |
| D366     | 16(Pen) *            | Q47751    | Unchanged *            | 17                      |                                               |
| D63      | 5(Pen) *             | Q47759    | Unchanged *            | 24                      |                                               |
| D63r     | 70(Pen) *            |           | Highly increased *     | 20                      |                                               |
| D344     | 24(Amp) *            | Q93NP3    | Highly increased *     | 17                      | 12                                            |
|          | 64(Pen) *            |           |                        |                         |                                               |
| EFM-1    | 90(Pen) *            | Q47783    | Unchanged *            | 1.5                     |                                               |
| 9439     | 128(Amp) *           | Q47801    | Increased §            |                         |                                               |
| C68      | 256(Amp) ‖           | Q9S6C2    |                        |                         |                                               |
| H80721   | 512(Amp) *           | Q47763    | Unchanged *            | <1.3                    | 20                                            |
|          | 512(Pen) *           |           |                        |                         |                                               |

*Zorzi et al. (1996).  
†Sifaoui et al. (2001).  
‡Ligozzi et al. (1996).  
§Fontana et al. (1994).  
‖Rice et al. (2001).

may have been pushed aside (Sauvage et al., 2002). Based on this structure, explanations have been advanced for the various mutations that increase the resistance (Sauvage et al., 2002).

PBP 5, as a class B PBP, does not support the necessary glycosyltransferase activity for peptidoglycan synthesis. Deletion studies in *Enterococcus faecalis* have demonstrated that the glycosyltransferase activity must be provided by at least one of the two class A PBPs encoded by *ponA* or *pbpF* (Arbeloa et al., 2004b).

Note also that four isolates of *Enterococcus faecalis* were found to exhibit high resistance to ampicillin and imipenem without overexpression of PBP 4. Instead, the resistance is due to two substitutions, P520S and Y605H, in PBP 4 (Ono et al., 2005). β-Lactamase-producing strains of *Enterococcus faecalis* have also been isolated in nosocomial infections (Murray, 1992), demonstrating again that several mechanisms can be exploited by the same pathogen.

### Streptococcus pneumoniae

The expression of β-lactamases or additional low-affinity PBPs has never been reported in pneumococcus. Instead, β-lactam-resistant strains of *Streptococcus pneumoniae* always harbor modified versions of their own class A and class B PBPs that are poorly acylated by β-lactams (Hakenbeck et al., 1980; Zighelboim & Tomasz, 1980).

Of the six PBPs from *Streptococcus pneumoniae*, PBP1a, PBP2b, PBP2x and sometimes PBP2a were found to be altered in resistant clinical isolates. These modified PBPs bound less radio-labeled antibiotic, whereas the affinity of PBP1b and PBP 3 was unchanged (Laible et al., 1991).
Low-affinity PBPs

emergence of pneumococcal resistance. Commensal streptococci, such as *Streptococcus oralis* and *Streptococcus mitis*, have acquired resistance through point mutations selected by exposure to β-lactam treatment for various diseases. Fragments of genes encoding low-affinity PBPs were subsequently exchanged between related streptococcal species, including *Streptococcus pneumoniae*, and selected by antibiotherapy (Dowson *et al.*, 1990). The recognition of these gene transfers in streptococci has led to the concept of a global gene pool of altered *pbp* sequences for β-lactam resistance (Reichmann *et al.*, 1997).

Comparison of the PBP2x sequence of a drug-susceptible strain of *Streptococcus mitis*, and sequence blocks, derived from this ancestral sequence, but found in resistant pneumococcal strains identified a few substitutions that are likely important for the resistance (Chi *et al.*, 2007). The role of some of these substitutions in the resistance has been verified experimentally [e.g. in positions 338, 339, 364, 369, 371, 605, (Smith & Klugman, 2005; Carapito *et al.*, 2006)], thus supporting the proposed scenario for the emergence of resistance.

Because *Streptococcus pneumoniae* can easily exchange genetic material, closely related strains can differ in capsular biosynthetic genes (hence serotype) and *pbp* genes. Conversely, identical *pbp* alleles or capsular biosynthetic genes can be found in unrelated strains (Coffey *et al.*, 1991; Zhou *et al.*, 2000). Nevertheless, despite the complications that horizontal gene transfers bring to the definition of pneumococcal lineage, it appears that the worldwide spread of pneumococcal β-lactam resistance results from the dispersion of a limited number of successful clones (McGee *et al.*, 2001).

Besides recombination, point mutations that occurred in pneumococcus have certainly contributed to the resistance phenomenon. A case in point is the T550A substitution in PBP2x that confers resistance to cephalosporins but susceptibility to penicillin. This substitution was found in the laboratory (Grebe & Hakenbeck, 1996; Krauss *et al.*, 1996; Sifaoui *et al.*, 1996), as well as in PBP2x from clinical isolates where it was caused by a mutation either within a mosaic (Coffey *et al.*, 1995) or a ‘virgin’ *pbp2x* gene (Asahi *et al.*, 1999).

Selection in the laboratory demonstrated that PBP2x and PBP2b are the primary resistance determinants for cefotaxime and piperacillin, respectively (Laible & Hakenbeck, 1987; Grebe & Hakenbeck, 1996), suggesting that PBP2x and PBP2b are the essential PBPs most reactive towards cefotaxime and piperacillin, respectively. Indeed, cefotaxime does not react with PBP2b (Hakenbeck *et al.*, 1987). Surprisingly, the point mutations selected in the laboratory do not match the substitutions of clinical isolates, with the exception of the aforementioned T550A in PBP2x (Grebe & Hakenbeck, 1996; Krauss *et al.*, 1996; Sifaoui *et al.*, 1996)

and T446A in PBP2b (Grebe & Hakenbeck, 1996). This discrepancy could reflect limited sampling, or the most useful substitutions may be different in the native PBPs from *Streptococcus pneumoniae*, as selected in the laboratory, or in the PBPs from the commensal species where they were originally selected in their host.

Transfer to a susceptible strain of *pbp2x* genes from clinically resistant isolates decreases the susceptibility to cephalosporins and most penicillins (Dowson *et al.*, 1989b; Laible *et al.*, 1991; Reichmann *et al.*, 1996; Smith & Klugman, 1998; Asahi *et al.*, 1999; du Plessis *et al.*, 2002; Chesnel *et al.*, 2003; Pernot *et al.*, 2004). Introduction of mosaic *pbp2b* genes causes a modest reduction of the susceptibility to piperacillin (Pagliero *et al.*, 2004). Lower susceptibility is achieved on transfer of both mosaic *pbp2x* and *pbp2b* genes (Dowson *et al.*, 1989b; Reichmann *et al.*, 1996; Smith & Klugman, 1998; du Plessis *et al.*, 2002). Still higher levels of resistance result from the additional introduction of a mosaic *pbp1a* gene (Barcus *et al.*, 1995; Reichmann *et al.*, 1996; Smith & Klugman, 1998; du Plessis *et al.*, 2002; Chesnel *et al.*, 2003). A high level of resistance restricted to the cephalosporins is obtained following transformation of a susceptible strain with mosaic *pbp2x* and *pbp1a* (Munoz *et al.*, 1992; Coffey *et al.*, 1995; Reichmann *et al.*, 1996; Smith & Klugman, 1998; Chesnel *et al.*, 2003). These observations can be rationalized by a threshold effect (Fig. 6).

These laboratory findings are mirrored in clinical strains (Sanbongi *et al.*, 2004; Ubukata *et al.*, 2004). Most resistant clinical isolates harbor three mosaic *pbp* genes encoding PBP1a, PBP2b and PBP2x (e.g. Ferroni & Berche, 2001; Overweg *et al.*, 2001; du Plessis *et al.*, 2002; Nagai *et al.*, 2002; Nichol *et al.*, 2002). However, some weakly resistant strains have mosaic alleles of *pbp2x* and *pbp2b* only (e.g. Ferroni & Berche, 2001; Nichol *et al.*, 2002), and at least one strain has only *pbp2x* modified (Nichol *et al.*, 2002). Some isolates with cephalosporin resistance, yet susceptible to penicillin, were found to have mosaic *pbp2x* and *pbp1a*

![Affinity MIC Affinity MIC Affinity MIC](image.png)

Fig. 6. Schematic representation of the threshold effect that may account for the relationship between the identity of the altered PBPs and the level of pneumococcal resistance. The PBP with the highest affinity for the β-lactam considered sets the susceptibility threshold of the recipient strain (horizontal arrow). The sequence of introduction of altered PBPs, which produces an incremental increase of resistance, depends on the relative affinities of the PBPs of the susceptible strain for a particular β-lactam. The depicted example could apply to PBP2x (gray) and PBP1a (white) with cefotaxime.

FEMS Microbiol Rev **32** (2008) 361–385

© 2008 Federation of European Microbiological Societies Published by Blackwell Publishing Ltd. All rights reservedwhile retaining a ‘virgin’ *pbp2b* (McDougal *et al.*, 1995; Smith & Klugman, 2001).

By measuring the decrease in intrinsic fluorescence of a recombinant soluble form of PBP2x upon antibiotic binding, the overall acylation efficiency \( k_2 / K_{\mathrm{d}} \) was found to be between 60 000 and 110 000 M⁻¹ s⁻¹ for penicillin (Jamin *et al.*, 1993; Mouz *et al.*, 1998; Di Guilmi *et al.*, 2000; Chesnel *et al.*, 2002). The deacylation rate \( k_3 \) measured in different ways is between 0.8 and 5 s⁻¹ (Jamin *et al.*, 1993; Di Guilmi *et al.*, 2000; Lu *et al.*, 2001; Chesnel *et al.*, 2002). The very fast acylation and slow deacylation reactions result in a \( c_{50} \) that lies in the micromolar range, which is consistent with MIC of susceptible strains (Lu *et al.*, 2001; Chesnel *et al.*, 2003). Attempts have been made to delineate the dissociation constant \( K_{\mathrm{d}} \) and the rate of acylation \( k_2 \) with penicillin. One study found a \( K_{\mathrm{d}} \) of 0.9 mM and a \( k_2 \) of 180 s⁻¹ (Lu *et al.*, 2001), whereas a second study reported a \( K_{\mathrm{d}} \) of 20 mM and a \( k_2 \) of 1600 s⁻¹ (Thomas *et al.*, 2001). Penicillin thus has a very poor ‘true’ affinity for PBP2x, a finding that presumably applies to β-lactams and PBPs in general. The efficacy of β-lactams does not result from a particularly good fit of the antibiotic to its target (\( K_{\mathrm{d}} \)), but rather from the extremely high rate of acylation (\( k_2 \)).

The crystal structure of PBP2x from the susceptible strain R6 was solved to a resolution of 2.4 Å (Pares *et al.*, 1996; Gordon *et al.*, 2000). The extracellular part of PBP2x consists of a transpeptidase domain, flanked by an elongated N-terminal domain and a small globular C-terminal domain. Regarding the precise mechanism of acylation by antibiotics, the crystal structure and theoretical studies have left the question open (e.g. Oliva *et al.*, 2003).

Identifying amino acid substitutions that are relevant to the reduction of affinity of a particular PBP is a difficult task. Owing to the process of recombination, neutral substitutions have likely been imported together with the ones that provide antibiotic resistance (‘hitchhiking’). Indeed, even genes neighboring *pbp2b* or *pbp1a* have been incidentally modified through recombination of large DNA fragments (Coffey *et al.*, 1999; Enright & Spratt, 1999). Nevertheless, a number of likely important substitutions were proposed based on their absence in susceptible strains, presence in many resistant strains and proximity to the catalytic motifs: T338A, T338G, T338P and M339F found within the SXXK motif (Coffey *et al.*, 1995; Reichmann *et al.*, 1996; Mouz *et al.*, 1998; Asahi *et al.*, 1999), H394Y and M400T that surround the SXN motif (Laible & Hakenbeck, 1987; Coffey *et al.*, 1995; Nagai *et al.*, 2002; Nichol *et al.*, 2002) and L546V, T550A and Q552E, which are close to the KTG motif (Coffey *et al.*, 1995; Asahi *et al.*, 1999; Mouz *et al.*, 1999; Ferroni & Berche, 2001). These substitutions do not appear randomly, and some sequence families can be identified.

Examination of about one hundred public sequences of the transpeptidase domain of PBP2x reveals three broad families. An alignment is provided in Fig. 7 to convey graphically a sense of the diversity of sequences, and of the difficulty of classifying those sequences resulting from the mosaicity. One family contains nonmosaic sequences. The mosaicity complicates the two other families and the grouping would differ for various sequence blocks. Nevertheless, the pattern emerging from a minimal overlap of substitutions in the transpeptidase domain suggests that two mechanisms have been selected to reduce the affinity of PBP2x for β-lactams (Pernot *et al.*, 2004).

One family of sequences is characterized by the T338A substitution. A subset of these sequences also have the adjacent M339F substitution. These latter sequences are from strains with particularly high levels of resistance (Coffey *et al.*, 1995; Asahi *et al.*, 1999; Ferroni & Berche, 2001; Nagai *et al.*, 2002; Chesnel *et al.*, 2003; Sanbongi *et al.*, 2004). PBP2x with the T338A/M339F substitutions have an efficiency of acylation by penicillin reduced more than 1000-fold (Lu *et al.*, 2001; Chesnel *et al.*, 2003), mostly due to slower acylation (\( k_2 \) decreased 300-fold), although weaker preacylation binding (\( K_{\mathrm{d}} \) fourfold higher) also contributes to the extremely poor affinity (Lu *et al.*, 2001; Chesnel *et al.*, 2003). These PBP2x variants also have significantly faster deacylation kinetics (\( k_3 \) increased 40–70-fold), an effect mostly due to the M339F substitution (Di Guilmi *et al.*, 2000; Chesnel *et al.*, 2003). The slow acylation and fast deacylation combine to elevate the \( c_{50} \) (concentration of antibiotic resulting in the steady-state acylation of half the enzyme) by four to five orders of magnitude (Lu *et al.*, 2001; Chesnel *et al.*, 2003).

The individual reversion of the 41 substitutions of a transpeptidase domain of PBP2x from a particularly resistant strain uncovered the role of four mutations, I371T, R384G, M400T and N605T, in addition to those in positions 338 and 339 (Carapito *et al.*, 2006) (Fig. 8). Reversion of the six substitutions nearly restored a normal and rapid acylation rate. Introduction of five of the substitutions into PBP2x from a susceptible strain diminished the reactivity with β-lactams, almost to the level of the original PBP2x from the resistant strain. These effects measured *in vitro* were mirrored by the expected phenotypic consequences *in vivo* (Carapito *et al.*, 2006). With a different PBP2x, a similar experiment where the effect of individual reversions was directly monitored *in vivo* also identified positions 371 and 384 as important (Smith & Klugman, 2005). A surprising discrepancy is the identification of Y595F as important, but not of N605T, into the *in vivo* study (Smith & Klugman, 2005), whereas the opposite was found in the *in vitro* screening (Carapito *et al.*, 2006).

The side chain of T338 is pointing away from the active site cavity and is hydrogen bonded to a buried water molecule. It has been proposed that suppression of the hydrogen bonding by replacement of T338 can lead to
Low-affinity PBPs

Q54533 L...QV.L.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q8GPK0 L...QV.L.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q83XA8 L...QV.L.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q9RES8 L...QV.L.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y78 .....QV.L.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y54 .....QV.L.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q9RES6 .....QV.L.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y57 .....QV.L.....L.....N.....A.TSS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN.....T.N...........T.E.SPNDK.TTE.SAI....I.AETVT....LVI.K..F.D...E.V.T.
Q75Y71 .....QV.L.....L.....N.....A.TSS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN.....T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y58 .....QV.L.....L.....N.....A.TSS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN.....T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
O34006 L...QV.L.....L.....N.....A.TSS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN.....T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y59 .....QV.L.....L.....N.....AF.SS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........S..T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y52 .....QV.L.....L.....N.....AF.SS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........ITSV...S..E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y46 .....QV.L.....L.....N.....AF.SS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN.....T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q9RES7 .....QV.L.....LQ..N.....AFTSS..SY.F..T.ATG.....TS.K....S......L....TNSG......E.TNHI.L.Y.II.V...SN..V.T.N...F........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y56 .....QV.L.....L.....N.....AFTSS..SY.F..T.ATG.....TS.K....S......L....TNSV......KE.....I.L.Y.II.V...SN..V.T.N...F........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q9S415 .....QV.L.....LQ..N.....AF.SS..SY.F..T.ATG.....TS.K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q83XA7 .....QV.L.....L.....N.....**AF.SS..SY.F..T.**ATG**L....**AS**K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...**A**.**A**E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q54793 L...QV.L.....L.....N.....AF.SS..SY.F..T.TG......TS..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q54792 L...QV.L.....L.....N.....AF.SS..SY.F..T.TG......TS..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVISK..F.D...E.V.T.
Q75Y69 .....QV.L.....L.....N.....AF.SS..SY.F..T.TG......TS..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y45 L...QV.L.....LQ..N.....AF.SS..SY.F..T.TG..L....TS..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y48 .....QV..T.....L.....N.....A.TSS..SY.F..T.DTG..L....S..S....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q9RCR8 .....QV..T.......N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q9RCR1 .....QV..T.......N.K.A.T....SY.F..T.ATG..L....S..K....S......LL....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y62 .....QV..T.......L.....N.....A.TSS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y60 .....QV..T.......L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y49 .....TRQVNL.S....L.......A.TSS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y44 .....QV.L..S....L.......A.TSS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q75Y67 .....QV........LK.A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN.....T.N...........T.E.SPNDK.TTE.SAI....AI..ETVT....LVI.K..F.D...E.V.T.
Q8GPJ4 .....QV........N.......LK.A.TSS..SY.F..T.GTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........V..S.TD..TN..AAI..AI..E.VS..L....AAFS..VE....
Q75Y66 .....QV........L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........V..S.TD..TN..AAI..AI..E.VS..L....AAFS..IVE....
Q75Y74 .....QV........A.TSS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN.....T.N...........T.E.SPNDK.TTE.SAI.E.AI..E.VT..L....AAFS..IVE....
Q75Y47 L...QV.L.....L.....N.....A.TSS..SY.F..T.TG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN.....T.N...........VT.E.SPNDK.TTE.SAI....IIA.AI..LVI..TAFS..E.....
Q75Y64 .....QV..T..N.......LK.T.S..Y..V..DT..L.L.S.T........L....TNSV......KE.....I.L.Q.II.V...SN..V.T.N...AA...S.........................A.F.....E.....
Q54701 .....QV........LQ..N.....AITS.SY.F.VT.DTG..L....S..S......LL....TNSV......KE.....I.I.Y.II.V...SN..V.T.N...........T.E.......................K...............
Q9RES9 .....QV..T.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K...S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T........................V...............
Q8G918 .....QV..T.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T........................V...............
Q54937 .....QV..T.....L.....N.....A.TSS..SY.F..T.DTG..L....S..K....S......L....TNSV......E.TNHI.L.Y.II.V...SN..V.T.N...........T........................G...............
Q75Y68 .....QV........LK.TSS.SY..V..DT..L.L.S.T........L....TNSV......KE.....I.I.Y.II.V...SN..V.T.N.......E.SPND.TTE.SAI.........................V...............
Q75Y77 .....QV........LK.TSS.SY..V..DT..L.L.S.T........L....TNSV......KE.....I.I.Y.II.V...SN..V.T.N.......E.SPND.TTE.SAI.AI..ETV....LVI.K..F.D...E.V.T.
P72530 L...QV.L.....L.....ND.KGA.SS..Y.S.VT..G......IET....S....TL..V......Q.Q.......I....PN..V.A.N........E.SPND.TTE.SAI.E.TI..ETV....LVI.K..F.D...E.I.T.
P72528 L...QV.L.V....L.....ND.KGA.TSS..Y.S.V....G......TIET....S....TL..V......Q.Q.......I....PN..V.A.N...F........E.SPND.TTE.SAI.E.TI..ETV....LVI.K..F.D...E.I.T.
P72529 L...QV.L.....L.....ND.KGA.TSS..Y.S.VT..G......IET....S....TL..V......Q.Q.......I....SN..LLTIN........T.EYSPNDK.TTE.SAI....I.E.VR.S........F.......
P72531 L...QV.L.....LQ..N.....GA.TSS..Y.S.VT..G......IET....S....L....TNSV......KE.....I.L.Q.II.V...SN..V.T.N...F........E.SPND..S.TSAIN.I..E.VR.S.......F.......
Q9RCR4 .....QV.L.....L.....ND.K.A.T.S....Y.......DTG......S..T....S..MA.L....TNSV......KE.....I.L.Y.II....B.SN.PV.T.N........E.SPND.TTE.SAI....AI..E.VT..ILVI.K..F.......
Q75Y73 A...HV.L.....ND.K.A.TS....Y.......DTG......S....T..MA.L..L...SV......A.SEH.M....N.N.....TEN..V.....V........E.SPNDK.TTE.SAI.E.AI..E.VT....LVI.K..F.D...E.V.T.
Q9RCR3 .....QV........R........A........V.N........S.......TL.N.VR......GQ.H.R...A.N...E.SNN.V.A.N........E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q8GPJ9 .....QV........A........V........V.N........S.......TL.N.VR......GQ.H.R...A.N...E.SNN.V.A.N........E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q9RCR5 .....QV.L.....LQ..N.....A.T.....Y..V..DT..L.L.S.P........TL..V........I........E.SNN.V.A.N........E.SPND.TTE.SAI.E.TI..ETV....LVI.K..F.D...E.I.T.
Q9RCR6 .....QV..T........LK..TSS.SSY.......DTG..L.Y.S....S..MA.L..L...V.......Q.H..R...A..N...E.SN..V.A.N........E.SPND.TTE.SAI.E.AI..ETV....LVI.K.AF.D...E.I.T.
Q8GPK1 .....QV..T........LK..TSS.SY.......DTG..L.Y.S....S..MA.L..L...V.......Q.H..R...A..N...E.SN..V.A.N........E.SPND.TTE.SAI.E.AI..ETV....LVI.K.AF.D...E.I.T.
Q8GPJ5 .....QV........N.......LK..T........V..DT..L.L.S........L........V........I........N........N........T.E.SPNDK.TTE.SAI.E.AI..ETVT....LVI.K..F.D...E.V.T.
Q9RCR7 .....QV..T..N.......K..TSS.SY.F..V..GT..L.L.S........L.TL..VR........H..R.G.II.V.E.AN..V...N..I..VVH.......E.SPN...TAE.SAI..AII.E.V..V.LVI.K.AF.D...E.IQ.T
Q54781 .....QV..T..N.......LK..TSSSE.SY.F..V..GT..L.L.S........L.TL..VR........H..R.G.II.V.E.AN..V...V..YG.I..VVH.......E.SPN...TAE.SAI..AII.E.V..V.LVI.K.AF.D...E.IQ.T
Q75Y55 .....QV..T..N.......K..T..T........L..Q.II.V...SN..V.T.N...AV...S..SPND..TTE.SAI....AI..ETV....LVI..AAFS..VE.....
P59676 QVSQRMDIPLTDQEVTDEIESTMMAADNGVNLIAIDEGRMFSAHMTQANVPANAQSIVFSIPDTAKQINKDSLTNVLVYSPTVTLTQVLDYSAVSMSPNVYSGIQLGANSDNTTAEQVSQQSPVDDELVTNSAKNALIAVGTETLANELEQSTKAA
P14677 ..........................................................................................................................................................................
Q75Y82 ..........................................................................................................................................................................
Q54748 ..........................................................................................................................................................................
Q75Y80 ..........................................................................................................................................................................
Q75Y83 ..........................................................................................................................................................................
Q9RET2 ..........................................................................................................................................................................
Q93IQ1 ..........................................................................................................................................................................
Q75Y63 .....QV........LK..T.....D..V.....L..L............L..L..SV............
Q75Y72 .....QV........P...........V........................Q.H...H........ISNH.....N
Q8GPJ6 .....QV........K.A.T.....VVT..G........................V...........H...K........N...........N...........V...........
Q93IQ0 .....QV........LK.P.T.....VVT..G............K........G...........V...........H...K........TN........N........F........V...........
Q8G8S1 I..QV........N.K.A.T.S...Y.....V.....G........................S...........A.F.....ET....
Q9RETO .....QV........SY.......DTG..L....S........MA.L........I........E.SNN.V.A.N........A.F.....E.....
Q54750 .....QV........SY.......DTG.LL..S........MA.L........I........E.SNN.V.A.N........K........A.F.....E.....
Q70B25 .....QV........MA.L..L..SV.....E.VDH.MT..A..N.ETSNN.V....H........F.....E.....
Q9RCR2 .....QV........MA.L..L..SV.....E.VDH.MT..A..N.ETSNN.V....H........F.....E.....
Q9R315 .....QV........MA.L..L..SV.....E.VDH.MT..A..N.ETSNN.V....H........F.....E.....
Q8GPJ3 .....QV........K........G............V...........I........E.SNN.V.A.N.D........E.SPNDK.TTE.SAI.........................E.....
Q54780 .....QV........P...........A........................Q.H........E.TN........N........P..T..E.SPND.TTE.STI....I..E.VTS..L....AAFS..E.....
Q75Y65 ..F.QV..T..N.......LK..T.....D..V.....VL...........L..SV.....E.VDY.M........N........N...........V..S.D.I.S.TSAI....I..ETVT....LVI.K..F.D...E.V.T.
Q8GPJ7 .....QV..T..N.....A..LK.P.T.....VV.....G............L..V.....Q.H...H........T........N...........V..S.TD..TN..AAI..AI..E.VT..ILVI.K.AF.D...E.V.T.
Q9RET1 .....QV..T..N.......LK..T........V.....G............I........TL.NSV......Q.H..R...A.N...E.SNH.V.A.N........V..S.TD..TN..AAI..AI..E.VS..L....AAFS..VE....
Q8GPJ8 .....QV..T..N.......LK..T........V.....G............I........TL.NSV......Q.H..R...A.N...E.SNN.V.A.N........V..S.TD..TN..AAI..AI..E.VS..L....AAFS..VE....
Q75Y75 .....QV..T..N.......LK..A.TSS.Y..VT..VG.....S........TL..V........Q.H..I.Q.II.V...SN..V.T.N........V..S.TD..TN..AAI..AI..E.VT..L....AAFS..IVE....
Q8RT80 .....QV..T..N.......LK..T........V........L........L....TNSV......KE.....I.L.Q.II.V...SN..V.T.N........V..S.TD..TN..AAI..AI..E.VT..L....AAFS..VE....
Q75Y81 .....QV..T..N.......LK..T........V........L........L....DTG......S....T..MA.L....TNSV.K.KE.....I.L.H.II.V...SN..V.T.N........V..S.TD..TN..AAI..AI..E.VA.T.....A.F..D..E.....


Fig. 7. Alignment of PBP2x publicly available sequences (aligned and clustered with CLUSTALW). Only positions where at least one sequence differs from the R6 reference (Uniprot accession number #P59676) are shown. The tranpeptidase domain is in light gray. Substitutions at positions 338, 339 and 552 are highlighted in dark grays. Although the mosaicity confounds efforts to classify these sequences unambiguously, this representation allows to visualize that sequences characterized by a mutation in position 338 (denoted by the thick black line on the right of the alignment) differ substantially from sequences with the Q552E substitution (denoted by the thin black on the right of the alignment), although a few sequences harbor both mutations. The absence of a line on the right of the alignment denotes nonmosaic sequences or sequences with few substitutions. The crystal structures of the high-affinity PBP2x from strains R6 (# P59676), as well as the two low-affinity proteins from strains Sp328 (# O34006) and 5259 (# Q70B25) have been solved, revealing two modes of reducing the affinity for β-lactams. The corresponding sequences are in bold characters. Also in bold is PBP2x from strain 5204 (# Q83KA7) with substitutions experimentally identified as contributing to the resistance highlighted in black.

destabilization of the active site due to the loss of the water molecule (Mouz *et al.*, 1998). Resolution of the structure of PBP2x from the resistant strain Sp328, which belongs to the family defined by the T338A substitution, has confirmed the absence of the buried water molecule (Dessen *et al.*, 2001).

The M339F mutation alone, introduced in the reference R6 PBP2x, reduces the efficiency of acylation by penicillin by sixfold and reduces the susceptibility (Chesnel *et al.*, 2003). A combination of the M339F and T338A mutations produces a greater effect. The structure of the latter double

Fig. 8. Overall structure of the transpeptidase domain of PBP2x from strain R6, showing the distribution of the 41 positions substituted in the β-lactam-resistant strain 5204. The position of the active site S337 is in red. Positions found to affect the efficiency of acylation experimentally are in magenta.

mutant has been solved to a resolution of 2.4 Å. The salient feature of the mutated active site is the reorientation of the hydroxyl of the catalytic S337 that is now pointing away from the active site center and is hydrogen-bonded to the main chain nitrogen of T550 instead of to K340 (Chesnel et al., 2003) (Fig. 9).

Mutations I371T and R384G impart a great flexibility to the loop spanning residues 365–394, as noted from the structure of Sp328. This instability extends in part to the SXN motif in positions 395–397, with S395 being somewhat displaced. The flexibility of this region generates a more accessible ‘open’ active site that may better accommodate alternative physiological substrates with branched stem peptides (Dessen et al., 2001). Mutations M400T and N605T certainly have indirect effects as they are quite far from the active site.

A second family of PBP2x molecules from resistant strains can be defined by the presence of the Q552E substitution. Introduction of this single substitution into PBP2x reduces about fourfold the efficiency of acylation and modestly reduces the susceptibility of the recipient R6 strain

(Mouz et al., 1999; Pernot et al., 2004). The structure of a PBP2x from a clinical strain that possesses the Q552E substitution has been solved to a resolution of 3 Å. This PBP2x has an efficiency of acylation reduced more than 15-fold (Pernot et al., 2004). The only significant difference found in comparison with the structure of R6 PBP2x is the displacement of strand β3, which carries the KTG motif (Pernot et al., 2004). This displacement of 0.5 Å narrows the active site, and is reminiscent of the closed conformation of PBP2a from *Staphylococcus aureus* (Lim & Strynadka, 2002). In addition to this conformational effect, the introduction of a negative charge into position 552 greatly affects the entry of the active site and does not favor binding of β-lactams, which are negatively charged (Pernot et al., 2004). Consequently, it appears that two distinct mechanisms have been selected that reduce the reactivity of PBP2x towards β-lactams. One mechanism primarily affects the chemistry of the active site S337, whereas the second mechanism hinders acylation by requiring an opening of the active site. These two mechanisms may be a reflection of two major sources of exogenous genetic material that have been incorporated into strains of *Streptococcus pneumoniae*. Note that a few sequences of PBP2x have both T338A and Q552E substitutions and may thus combine the effect of both mechanisms.

Another significant substitution is T550A, which confers resistance to cephalosporins only, both in laboratory and clinical strains (Coffey et al., 1995; Grebe & Hakenbeck,

Fig. 9. Superposition of the active site of wild-type R6 PBP2x (green) and of the double mutant T338A/M339F (cyan). Residues from the catalytic motifs are shown along the mutated amino acids. Note that the hydroxyl of the catalytic S337 is pointing towards strand β3 in the mutant, whereas it is pointing to the active site cavity in the wild-type enzyme.

Low-affinity PBPs

1996; Sifaoui *et al.*, 1996; Asahi *et al.*, 1999). When the T550A point mutation occurs within a mosaic PBP2x, which contains the T338A/M339F double mutation, it further increases the resistance to cephalosporins, while it almost abolishes resistance to penicillin (Coffey *et al.*, 1995). This effect is mirrored in the acylation efficiency of a T550A point mutant of R6 PBP2x, which is decreased 20-fold towards cefotaxime and unaffected towards penicillin (Mouz *et al.*, 1999). This effect has been rationalized by the abolition of the hydrogen-bond between T550 and the carboxylate that is attached to the six-member ring of second- and third-generation cephalosporins (Gordon *et al.*, 2000).

Note that ceftobiprole, a novel cephalosporine active against MRSA, shows great promise for the treatment of infections caused by *Streptococcus pneumoniae* with resistance to most β-lactams. Ceftobiprole MICs are not > 1 mg L⁻¹, even with strains that have altered PBP2x, PBP2b and PBP1a, including PBP2x with the major substitutions T338A, M339F, I371T, R384G and M400T (Davies *et al.*, 2006). Whether ceftobiprole reacts with PBP2x molecules that have the cephalosporine-specific mutation T550A remains to be established.

PBP2b, the other class B PBP from *Streptococcus pneumoniae*, has not been subjected to such thorough investigations, presumably because of the absence of a high-resolution structure. Over 90 sequences are available and two substitutions, T446A or T446S and E476G, are always found in PBP2b from clinically resistant strains. The probable importance of these two substitutions was pointed out in numerous studies (Dowson *et al.*, 1989b; Smith & Klugman, 1995; Ferroni & Berche, 2001; Sanbongi *et al.*, 2004). The T446A mutation, which is immediately adjacent to the SXN motif, is also selected by piperacillin in the laboratory (Grebe & Hakenbeck, 1996). T446A is the only substitution that has been characterized biochemically (Pagliero *et al.*, 2004) and it reduces the affinity for penicillin by 60%. The affinity of various PBP2b molecules from clinical isolates with six to 43 mutations in addition to T446A is reduced by 90–99%.

In addition to the mutations in positions 446 and 476, some PBP2b sequences are distinguished by other salient features such as the substitution of six to seven adjacent residues at position 426/427–432 (Dowson *et al.*, 1989a; Smith & Klugman, 1995). Three related PBP2b sequences from Japanese isolates are noteworthy by the insertion of three residues (SWY) after position 422 (Yamane *et al.*, 1996). This is one of two occurrences of a change in the number of residues in a mosaic PBP. The other case was found in PBP1a. In all other cases, the total length of the proteins and the position of the catalytic motifs are fully conserved, despite extensive sequence remodeling. Seven related sequences from Korean clinical strains show a sub-

stitution within the third catalytic motif KTG, which is changed to KSG (Song *et al.*, 2000). In contrast to PBP2x and PBP1a, where mutations within the first catalytic motif are commonplace, a single case was reported of a V388A substitution within the SVVK motif (Kell *et al.*, 1993). The recent emergence of strains that show a particularly high resistance to amoxicillin, relative to other β-lactams, appears to result from a set of 10 substitutions in the region 591–640 surrounding the third catalytic motif KTG (du Plessis *et al.*, 2002; Kosowska *et al.*, 2004). The relative importance of these and other mutations for the resistance awaits investigation, and mechanistic insight will require the resolution of the crystal structure of PBP2b from susceptible and resistant strains.

PBP1a may be considered as the most important and troublesome PBP clinically. Indeed, the resistance potentially provided by mosaic PBP2x and PBP2b is capped by the presence of a ‘virgin’ PBP1a, which still warrants some efficacy to β-lactam therapy. A high level of resistance depends on a modified PBP1a. About 50 PBP1a sequences are publicly available. The T371A substitution within the first catalytic motif, analogous to the T338A mutation in PBP2x, is commonly found in PBP1a sequences from resistant strains (Asahi & Ubukata, 1998; Smith & Klugman, 1998; Ferroni & Berche, 2001; Nagai *et al.*, 2002; Nichol *et al.*, 2002) (Fig. 10). Reversion of this substitution reduced

![Figure 10](https://example.com/image.png)

Fig. 10. Overall structure of the transpeptidase domain of PBP1a. The position of the active site S370 is shown with a red sphere. Yellow spheres mark the positions of substitutions that are important for resistance, T371A and TSQF(574-577)NTGY.

but did not abrogate the resistance that PBP1a confers in addition to PBP2x and PBP2b (Smith & Klugman, 1998). Some mosaic sequences lack the T371A mutation, including PBP1a from a highly resistant Hungarian isolate (MIC for penicillin of $16 \mathrm{mg} \mathrm{L}^{-1}$) (Smith & Klugman, 2003). Another remarkable feature is the mutation of a stretch of four residues (TSQF to NTGY) at positions 574–577, which is observed in all the mosaic sequences (Fig. 10). Reversion of this set of substitutions decreased the additional resistance conferred by PBP1a (Smith & Klugman, 2003). A similar effect of the reversion was found for the L539W substitution, although the sequence in which the experiment was performed is the only one that presents this particular mutation (Smith & Klugman, 2003). Much remains to be learned about the detailed mechanism by which the reactivity of PBP1a is reduced.

Although PBP2x, PBP2b and PBP1a are the major PBPs responsible for the resistance of *Streptococcus pneumoniae*, a number of studies have hinted at the possible involvement of various other PBPs. Transfer of a high level of resistance from a strain of *Streptococcus mitis* to a laboratory strain of *Streptococcus pneumoniae* was shown to require transfer of the genes encoding the five high-MW PBPs (Hakenbeck et al., 1998). A point mutation in the low-MW PBP3 was found to contribute to the resistance of a strain selected on cefotaxime in the laboratory (Krauss et al., 1996). In contrast to these laboratory experiments, examination of the PBPs from clinical isolates failed to reveal significant modification of PBP1b or PBP3 (Du Plessis et al., 2000; Sanbongi et al., 2004). Early studies, which examined various strains through the labeling of PBPs with radioactive penicillin, found several instances where binding to PBP2a was diminished in resistant strains (Laible et al., 1991; Reichmann et al., 1997). Also, transfer in the laboratory of resistance from a *Streptococcus mitis* strain to *Streptococcus pneumoniae* involved modification of PBP2x, PBP2b, PBP1a and PBP2a, but not of PBP1b and PBP3 (Reichmann et al., 1996). Various combinations of point mutations, including silent ones, were observed in some PBP2a sequences, suggesting events of intraspecies recombination (Chesnel et al., 2005). The role of PBP2a in β-lactam resistance is now firmly established in at least one instance (Smith et al., 2005). A strain isolated from an AIDS patient was found to harbor a mosaic PBP2a in addition to mosaic PBP2x, PBP2b and PBP1a. Transformation experiments demonstrated that this PBP2a variant is indeed responsible for an elevated resistance to various β-lactams. The sequence shows 25 substitutions including 12 within the transpeptidase domain. The absence of a crystal structure precludes a detailed analysis, but it is noteworthy that the threonine following the catalytic serine is replaced by an alanine, like in numerous variants of PBP2x and PBP1a.


Both class B PBPs, PBP2x and PBP2b, are essential in *Streptococcus pneumoniae*, which is consistent with the selection of variants of these proteins by β-lactams (Kell et al., 1993). PBP1b and PBP3 are not essential (Schuster et al., 1990; Hoskins et al., 1999), which is again consistent with the fact that these proteins are not involved in the resistance process. PBP1a and PBP2a are not essential individually, but one of them must be present and functional (Hoskins et al., 1999; Paik et al., 1999). The fact that PBP1a, rather than PBP2a, is the main target of antibiotic selective pressure may be due to PBP2a having a low intrinsic affinity for β-lactams (Zhao et al., 2000).

A puzzling discovery was made, which is directly related to PBP-based β-lactam resistance. Clinically resistant isolates have an abnormal peptidoglycan structure with an elevated proportion of cross-bridges that involve branched stem-peptides (Garcia-Bustos & Tomasz, 1990). Instead of having the L-Lys of the ‘acceptor’ peptide cross-linked directly to the D-Ala of the ‘donor’ peptide, there are intervening L-Ala-L-Ala or L-Ala-L-Ser dipeptides. The genetic determinants of this cell wall abnormality could nevertheless be separated from the resistance determinants (the mosaic *pbp* genes) (Severin et al., 1996). The genes responsible for the synthesis of branched precursors were found to constitute the *murMN* operon (Filipe & Tomasz, 2000), also known as the *fibAB* operon (Weber et al., 2000). Mosaic *murM* genes often increase the resistance level conferred by a set of mosaic *pbp* genes (Filipe & Tomasz, 2000; Smith & Klugman, 2001). A naïve explanation is that mosaic PBPs prefer branched substrates. However, deletion of *murM* abolishes the resistance but does not impact the growth rate in the absence of antibiotic challenge (Filipe & Tomasz, 2000), demonstrating that mosaic PBPs can efficiently use linear precursors. The situation is reminiscent of the role of the *femAB* operon in *Staphylococcus aureus*, which is required for expression of *mecA*-based resistance, while the *mecA*-encoded PBP2a can nevertheless function with alternative substrates produced in the absence of *femAB* (Stranden et al., 1997; Arbeloa et al., 2004a). It has been proposed that branched stem-peptides may be superior competitors against β-lactams for the active site of some PBPs of resistant strains, or that they may be involved in some signaling function of cell wall metabolism, or that they play a particular role in the integrity of the peptidoglycan, a role that becomes critical when some PBPs are inhibited by antibiotics (Filipe & Tomasz, 2000).

Besides MurM, other unknown factors modulate β-lactam resistance. Indeed, five clinical isolates with significantly different levels of resistance were found to have the same MurM allele and strictly identical sequence of their penicillin-binding domains, for the six PBPs (Chesnel et al., 2005). For example, a functional two-component signal-transducing system CiaRH is necessary to tolerate

Low-affinity PBPs

low-affinity PBP2x variants, even in the absence of antibiotic challenge (Mascher *et al.*, 2006).

Although much is known about the biochemistry of the PBPs, the MurM and CiaRH complications highlight the limited understanding of the physiological function of the PBPs in cell wall metabolism, both in the absence and in the presence of antibiotics.

### Neisseria

*Neisseria meningitidis* and *Neisseria gonorrhoeae* have acquired reduced susceptibility to penicillin via two routes: β-lactamase production (Bergström *et al.*, 1978) and modified PBPs. The modification of at least one chromosomally encoded PBP will be discussed below.

*Neisseria* contain only three PBPs called PBP1, PBP2 and PBP3, respectively of class A, of class B and a low-MW carboxypeptidase. Gonococcal strains with reduced susceptibility to β-lactams without β-lactamase were found to exhibit reduced labeling of PBP2 and PBP1 by radiolabeled penicillin (Dougherty *et al.*, 1980). The full level of resistance of a donating clinical strain could be conferred to a susceptible recipient strain upon transformation, resulting in diminished binding of radiolabeled penicillin to PBP1 and PBP2 (Dougherty, 1986). Reduced labeling of PBP2 was observed in meningococci (Mendelman *et al.*, 1988). PBP2 is encoded by the *penA* gene, which is mosaic in resistant strains of *N. gonorrhoeae* (Spratt, 1988) and *N. meningitidis* (Spratt *et al.*, 1989). The mechanism of the acquisition of nonplasmidic resistance in *Neisseria* is therefore similar to that of *Streptococcus pneumoniae*.

The origin of the sequence fragments that are found in mosaic *penA* genes of gonococci and meningococci has been investigated in some depth. Several commensal species, such as *Neisseria flavescens*, *Neisseria cinerea* or *Neisseria perflava*, appear to have each contributed sequence blocks (Spratt *et al.*, 1989, 1992; Bowler *et al.*, 1994; Ameyama *et al.*, 2002). *Neisseria flavescens* isolates from the preantibiotic era are relatively resistant to penicillin and have a PBP2 with an intrinsic low affinity for penicillin (Bowler *et al.*, 1994). Transfer in the laboratory of the *penA* gene from such *N. flavescens* isolates could indeed reduce the susceptibility of *N. meningitidis* (Bowler *et al.*, 1994). In contrast, *N. cinerea* is not naturally resistant, and accordingly, no resistance was conferred upon transfer of the *penA* gene from this species (Bowler *et al.*, 1994). Instead, PBP2 sequences of *N. cinerea* found in resistant meningococci have an additional aspartic acid following D345, which is not present in susceptible *N. cinerea* strains (Bowler *et al.*, 1994). This insertion was also found in mosaic PBP2 sequences from most resistant gonococcal strains. It is sufficient to decrease the reactivity of PBP2 for β-lactams and to confer some resistance to *N. gonorrhoeae* (Brannigan

et al., 1990). A clinically resistant strain was later discovered that only has this additional aspartic acid (Ito *et al.*, 2005). The sequence identity between *Neisseria* PBP2 and PBPs of known structures is too low to obtain a reliable alignment. Nevertheless, by simply aligning the SXXK and SXN motifs, it appears that the insertion following position 345 is in a region close in space from the SXN motif. This loop was found to be destabilized in a low-affinity PBP2x of *Streptococcus pneumoniae* (Dessen *et al.*, 2001).

Thus, *penA* alleles conferring penicillin resistance have arisen both from the recruitment of sequence blocks from naturally resistant species, such as *N. flavescens*, and new mutations such as a codon insertion. As commensal *Neisseria* species readily exchange genetic material, the *penA* alleles conferring resistance may be considered as forming a common gene pool shared by several species (Saez-Nieto *et al.*, 1990; Lujan *et al.*, 1991).

The peptidoglycan composition is considerably altered in a gonococcal strain with altered PBP1 and PBP2, even in the absence of antibiotics (Dougherty, 1985). The cell wall of a meningococcal strain with altered *penA* alleles has a greater amount of unprocessed pentapeptides, suggesting that the transpeptidase and/or carboxypeptidase activity of low-affinity PBP2 is modified (Garcia-Bustos & Dougherty, 1987; Antignac *et al.*, 2003).

Early studies showed that PBP1, the class A PBP, also had decreased reactivity for penicillin in gonococci (Dougherty *et al.*, 1980), but subsequent studies failed to uncover mosaicity in the *ponA* gene encoding PBP1. More recently, an allele of *ponA* encoding PBP1 with the single substitution L421P was found to contribute to the high resistance of some *N. gonorrhoeae* strains (Ropp *et al.*, 2002). The L421P substitution was shown *in vitro* to diminish about fourfold the acylation efficiency of PBP1 by various β-lactams (Ropp *et al.*, 2002).

Note that three non-*pbp* loci have been found to contribute to β-lactam resistance in *Neisseria* species. The *mtr* locus encodes an efflux pump (Hagman *et al.*, 1995), while *penB* codes for a porin (Gill *et al.*, 1998). The nature of the third locus *penC*, which is required to allow phenotypic expression of the *ponA* mutation, remains undetermined (Ropp *et al.*, 2002).

### Other pathogens

Modified PBPs to resist β-lactams have been found in a few other pathogens, including species where the most frequent mode of resistance is the production of a β-lactamase.

Resistant clinical isolates of *Haemophilus influenzae* usually evade the action of β-lactams by producing one of two β-lactamases (Tristram *et al.*, 2007). However, the number of β-lactamase-negative ampicillin-resistant (BLNAR) strains is increasing, particularly in Japan (Hasegawa *et al.*,
2004). First documented in 1980 (Markowitz, 1980), BLNAR strains express low-affinity PBP3, the division-specific class B PBP (Mendelman *et al.*, 1984; Parr & Bryan, 1984; Malouin *et al.*, 1987; Clairoux *et al.*, 1992).

Sequencing revealed in excess of 20 mutation patterns for the transpeptidase domain of PBP3, with a number of mutations ranging from one to nine, affecting 21 different positions (Ubukata *et al.*, 2001; Dabernat *et al.*, 2002; Kaczmarek *et al.*, 2004). These sequences are not mosaic but show an accumulation of point mutations. Some sequences are characterized by the presence of an R517H substitution, while others have the N526K mutation. Both substitutions are relatively close to the third KTG514 catalytic motif. Position 517 with respect to the KTG motif is analogous to the position 552 in PBP2x from *Streptococcus pneumoniae*, which is also mutated in some strains (Pernot *et al.*, 2004). Sequences that contain the N526K substitution may also possess the three additional mutations M377I, S385T and L389F surrounding the second SSN381 catalytic motif. These mutations are associated with a high resistance to cefotaxime in Japan (Dabernat *et al.*, 2002). S385T and L389F increase the resistance conferred by N526K (Osaki *et al.*, 2005). Modeling of the structure of *H. influenzae* PBP3 on that of *Streptococcus pneumoniae* PBP2x showed that residues 517, 526, 377, 385 and 389 are probably lining the active site cavity (Ubukata *et al.*, 2001).

The affinity for penicillin of a few *H. influenzae* PBP3 variants has been measured *in vitro* (Kaczmarek *et al.*, 2004). Variants with the N526K mutation have a lower affinity than without, in agreement with the resistance level of the originating strains. Surprisingly, a PBP3 with only the R517H substitution had the same high affinity as a wild-type PBP3. Therefore, this substitution cannot confer resistance in isolation.

BLNAR strains with a high level of resistance combine mechanisms that involve alteration of PBP3 and an efflux pump (Kaczmarek *et al.*, 2004). A low affinity PBP3 can also be found in strains expressing a β-lactamase, and both mechanisms can cooperate to increase the resistance or to resist to combinations of β-lactams and β-lactamase inhibitors such as the widely used amoxicillin/clavulanate formulations (Dabernat *et al.*, 2002; Matic *et al.*, 2003).

The genome of *Helicobacter pylori* encodes three PBPs. These are the homologues of the class B PBP2 and PBP3, and of the class A PBP1a from *Escherichia coli*. A few stable amoxicillin-resistant strains have been reported (van Zwet *et al.*, 1998; Han *et al.*, 1999; Gerrits *et al.*, 2002; Okamoto *et al.*, 2002). In one case, the resistance was shown to result entirely from the single point mutation S414R in PBP1a (Gerrits *et al.*, 2002). Two other resistant strains were found to have the three substitutions T556S, N562Y and T593A as well as the insertion of a Glu after residue 464 (Okamoto *et al.*, 2002). One strain had ten substitutions, all of them in

the second half of the transpeptidase domain, including the T556S and N562Y mutations (Kwon *et al.*, 2003). T556S is within the third catalytic motif KTG. *In vitro* selection on amoxicillin also yielded strains with modified PBP1a (DeLoney & Schiller, 2000). The PBP1a of one such strain had four substitutions, including the S414R mutation (Paul *et al.*, 2001).

To the authors' knowledge, no clinical isolates of *Escherichia coli* are known to resist through the expression of modified PBPs. However, as a laboratory workhorse, *Escherichia coli* was used to demonstrate that β-lactam pressure can select for altered PBPs (Spratt, 1978). Several point mutations in PBP3, the class B PBP dedicated to division, were found to reduce susceptibility. Interestingly, the substitution T308A, next to the active site S307, is analogous to the PBP2x T338A and PBP1a T371A that contribute to resistance to *Streptococcus pneumoniae* (Hedge & Spratt, 1985a,b). Another mutation was found in the second catalytic motif, changing SSN361 into SSS361 (Hedge & Spratt, 1985a).

A few reports must be added to complete this overview of pathogens with modified PBPs. PBP alteration has also been found in imipenem-resistant clinical isolates of *Proteus mirabilis* (Neuwirth *et al.*, 1995) and *Pseudomonas aeruginosa* (Bellido *et al.*, 1990). A cefsulodin-resistant clinical isolate of *P. aeruginosa* also had one PBP with reduced affinity, although not the same as the imipenem-resistant isolate (Gotoh *et al.*, 1990). Overexpression of PBP3, in addition to decreased outer-membrane permeability, was found in a highly resistant strain of *Salmonella muenchen* (Bellido *et al.*, 1989). The various levels of resistance of several strains of *Acinetobacter calcoaceticus* could be correlated with the production of PBPs with altered expression or affinity for β-lactams (Obara & Nakae, 1991). Also, a nosocomial imipenem-resistant strain of *Acinetobacter baumannii* was found to have PBPs with low-affinity for β-lactamase inhibitors, in particular clavulanate, which are normally used against this bacteria (Urban *et al.*, 1995).

In the laboratory, imipenem could select a resistant clone of *Acinetobacter baumannii* with an altered PBP (Gehrlein *et al.*, 1991). Alterations of PBP3 or PBP2 were selected in laboratory mutants of *Listeria monocytogenes* (Gutkind *et al.*, 1990; Pierre *et al.*, 1990a). Altered PBPs were also found in laboratory-resistant mutants of the *Bacteroides fragilis* group (Wexler & Hableian, 1990) and of *Rhodococcus equi* (Nordmann *et al.*, 1993).

Pathogens have been submitted to severe antibiotic pressure over the past five decades, leading to the emergence of resistant strains. In a natural setting as well, β-lactam-producing bacteria need to be protected against the drugs of their own making. Two examples have been documented, which involve low-affinity PBPs. The expression of a particular PBP is responsible in part for the resistance of

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved   FEMS Microbiol Rev **32** (2008) 361–385
Low-affinity PBPs

Table 2. High molecular weight PBPs of organisms that resist β-lactams by expressing low affinity PBPs

| Class A (bifunctional) | Class B (monofunctional) |
|------------------------|--------------------------|
| Staphylococcus aureus  | PBP 2                   | PBP 1                  | PBP 3                  | PBP2a mecA |
| Enterococci            | PBP1a                   | PBP1b                  | PBP2a                  | PBPC(B)    | PBP2b                  | pbpB       | pbpA       |
|                        | ponA                    | pbpZ                   | pbpF                   |            |                        |            |            |
| Streptococcus pneumoniae | PBP1a                   | PBP1b                  | PBP2a                  | PBP2x      | PBP2b                  |            |            |
| Neisseria              | PBP1                    |                        | PBP2                   | penA       |                        |            |            |
|                        | ponA                    |                        |                        |            |                        |            |            |
| Haemophilus influenzae | PBP1a                   | PBP1b                  | PBP3                   | ftsI       | PBP2                   |            |            |
| Helicobacter pylori    | PBP1a                   |                        | PBP3                   |            | PBP2                   |            |            |

Low-affinity PBPs are underlined. Discontinuous underline indicate intrinsic low affinity. Italic characters indicate mosaicity. Nonitalic characters indicate point mutations. An arrow indicate overexpression. Bold characters indicate acquisition of exogenous origin. Alternative gene names are given below their respective product.

β-lactam-producing Streptomyces clavuligerus (Paradkar et al., 1996). None of the eight PBPs of cephamycin C-producing Nocardia lactamdurans bind the β-lactam secreted by this bacteria, although it also expresses a β-lactamase (Coque et al., 1993).

### Concluding remarks: are the PBPs sustainable targets?

β-Lactam treatments for staphylococcal, enterococcal and pneumococcal infections are increasingly ineffective. Table 2 summarizes the various PBP-based mechanisms of resistance that have been uncovered. The isolation of strains with modified PBPs from species producing β-lactamases is worrisome. The efficacy of β-lactams may thus be compromised even with the advent of efficient β-lactamase inhibitors. One should therefore ask whether PBPs are still valid targets for future antimicrobial therapies.

Five decades of β-lactam therapy have largely validated the targeting of PBPs. In which direction should the research effort be headed? Detailed kinetic studies of the reaction between PBPs and β-lactams have shown that these antibiotics are a poor fit to the enzyme active site, which guarantees the broad specificity of these drugs. However, clinically useful improvements of the affinity of the preacylation complex can be obtained, as demonstrated with novel cephalosporines active against MRSA and a host of other resistant pathogens (Fuda et al., 2006; Page, 2006).

Restoring a rapid acylation rate in low-affinity PBPs might be more difficult. With this aim, instead of studying the reaction between PBPs and β-lactams, research should be directed towards the physiological reaction of transpeptidation. Indeed, the following paradox needs to be resolved. β-Lactams are mimics of the natural substrate of PBPs, so that explanations of the low reactivity of altered PBPs for the drugs should apply equally well to the reaction with the natural substrates. However, this is not the case, as resistant pneumococci are fit, indicating that the physiological reactions catalyzed by altered low-affinity PBPs are not severely affected. Understanding how the natural PBP substrates, or partner proteins, maintain the reactivity of the catalytic serine even in PBPs from resistant bacteria should aid in the design of novel compounds. Such new drugs could react with all PBPs, regardless of their reactivity with β-lactams (Zervosen et al., 2004). Alternatively, new molecules might serve as adjuvants to restore or maintain the reactivity of all PBPs towards traditional β-lactams (Fuda et al., 2005).

### Acknowledgements

This work was supported by the European Commission Sixth Framework Program grants LSMH-CT-COBRA 2003-503335 and LSMH-CT-EUR-INTAFAR 2004-512138.

### References

Ameyama S, Onodera S, Takahata M, Minami S, Maki N, Endo K, Goto H, Suzuki H & Oishi Y (2002) Mosaic-like structure of penicillin-binding protein 2 Gene (penA) in clinical isolates of Neisseria gonorrhoeae with reduced susceptibility to cefixime. Antimicrob Agents Chemother 46: 3744–3749.

Antignac A, Boneca IG, Rousselle JC, Namane A, Carlier JP, Vazquez JA, Fox A, Alonso JM & Taha MK (2003) Correlation between alterations of the penicillin-binding protein 2 and modifications of the peptidoglycan structure in Neisseria meningitidis with reduced susceptibility to penicillin G. J Biol Chem 278: 31529–31535.

Arbeloa A, Hugonnet JE, Sentilhes AC, Josseaume N, Dubost L, Monsempes C, Blanot D, Brouard JP & Arthur M (2004a) Synthesis of mosaic peptidoglycan cross-bridges by hybrid peptidoglycan assembly pathways in gram-positive bacteria. J Biol Chem 279: 41546–41556.

Arbeloa A, Segal H, Hugonnet JE, Josseaume N, Dubost L, Brouard JP, Gutmann L, Mengin-Lecreulx D & Arthur M (2004b) Role of class a penicillin-binding proteins in PBP5-

mediated beta-lactam resistance in *Enterococcus faecalis*.  
*J Bacteriol* **186**: 1221–1228.

Asahi Y & Ubukata K (1998) Association of a thr-371 substitution  
in a conserved amino acid motif of penicillin-binding protein  
1A with penicillin resistance of *Streptococcus pneumoniae*.  
*Antimicrob Agents Chemother* **42**: 2267–2273.

Asahi Y, Takeuchi Y & Ubukata K (1999) Diversity of  
substitutions within or adjacent to conserved amino acid  
motifs of penicillin-binding protein 2X in cephalosporin-  
resistant *Streptococcus pneumoniae* isolates. *Antimicrob Agents  
Chemother* **43**: 1252–1255.

Barcus VA, Ghanekar K, Yeo M, Coffey TJ & Dowson CG (1995)  
Genetics of high level penicillin resistance in clinical isolates of  
*Streptococcus pneumoniae*. *FEMS Microbiol Lett* **126**: 299–303.

Beadle BM, Nicholas RA & Shoichet BK (2001) Interaction  
energies between beta-lactam antibiotics and *E. coli* penicillin-  
binding protein 5 by reversible thermal denaturation. *Protein  
Sci* **10**: 1254–1259.

Bellido F, Vladoianu IR, Auckenthaler R, Suter S, Wacker P, Then  
RL & Pechere JC (1989) Permeability and penicillin-binding  
protein alterations in *Salmonella muenchen*: stepwise  
resistance acquired during beta-lactam therapy. *Antimicrob  
Agents Chemother* **33**: 1113–1115.

Bellido F, Veuthey C, Blaser J, Bauernfeind A & Pechere JC (1990)  
Novel resistance to imipenem associated with an altered PBP-4  
in a *Pseudomonas aeruginosa* clinical isolate. *J Antimicrob  
Chemother* **25**: 57–68.

Bergström S, Norlander L, Norqvist A & Normark S (1978)  
Contribution of a TEM-1-like beta-lactamase to penicillin  
resistance in *Neisseria gonorrhoeae*. *Antimicrob Agents  
Chemother* **13**: 618–623.

Bowler LD, Zhang QY, Riou JY & Spratt BG (1994) Interspecies  
recombination between the penA genes of *Neisseria  
meningitidis* and commensal *Neisseria* species during the  
emergence of penicillin resistance in *N. meningitidis*: natural  
events and laboratory simulation. *J Bacteriol* **176**: 333–337.

Brannigan JA, Tirodimos IA, Zhang QY, Dowson CG & Spratt BG  
(1990) Insertion of an extra amino acid is the main cause of  
the low affinity of penicillin-binding protein 2 in penicillin-  
resistant strains of *Neisseria gonorrhoeae*. *Mol Microbiol* **4**:  
913–919.

Busscher JF, van Duijkeren E & Sloet van Oldruitenborgh-  
Oosterbaan MM (2006) The prevalence of methicillin-  
resistant staphylococci in healthy horses in the Netherlands.  
*Vet Microbiol* **113**: 131–136.

Carapito R, Chesnel L, Vernet T & Zapun A (2006) Pneumococcal  
beta-lactam resistance due to a conformational change in  
penicillin-binding protein 2x. *J Biol Chem* **281**: 1771–1777.

Chambers HF, Sachdeva MJ & Hackbarth CJ (1994) Kinetics of  
penicillin binding to penicillin-binding proteins of  
*Staphylococcus aureus*. *Biochem J* **301**(Part 1): 139–144.

Chesnel L, Zapun A, Mouz N, Dideberg O & Vernet T (2002)  
Increase of the deacylation rate of PBP2x from *Streptococcus  
pneumoniae* by single point mutations mimicking the class a  
beta-lactamases. *Eur J Biochem* **269**: 1678–1683.


Chesnel L, Pernot L, Lemaire D, Champelovier D, Croize J,  
Dideberg O, Vernet T & Zapun A (2003) The structural  
modifications induced by the M339F substitution in PBP2x  
from *Streptococcus pneumoniae* further decreases the  
susceptibility to beta-lactams of resistant strains. *J Biol Chem*  
**278**: 44448–44456.

Chesnel L, Carapito R, Croize J, Dideberg O, Vernet T & Zapun A  
(2005) Identical penicillin-binding domains in penicillin-  
binding proteins of *Streptococcus pneumoniae* clinical isolates  
with different levels of beta-lactam resistance. *Antimicrob  
Agents Chemother* **49**: 2895–2902.

Chi F, Nolte O, Bergmann C, Ip M & Hakenbeck R (2007)  
Crossing the barrier: evolution and spread of a major class of  
mosaic pbp2x in *Streptococcus pneumoniae*, *S. mitis* and  
*S. oralis*. *Int J Med Microbiol* **297**: 503–512.

Clairoux N, Picard M, Brochu A, Rousseau N, Gourde P,  
Beauchamp D, Parr TR Jr, Bergeron MG & Malouin F (1992)  
Molecular basis of the non-beta-lactamase-mediated  
resistance to beta-lactam antibiotics in strains of *Haemophilus  
influenzae* isolated in Canada. *Antimicrob Agents Chemother*  
**36**: 1504–1513.

Coffey TJ, Dowson CG, Daniels M, Zhou J, Martin C, Spratt BG  
& Musser JM (1991) Horizontal transfer of multiple penicillin-  
binding protein genes, and capsular biosynthetic genes, in  
natural populations of *Streptococcus pneumoniae*. *Mol  
Microbiol* **5**: 2255–2260.

Coffey TJ, Daniels M, McDougal LK, Dowson CG, Tenover FC &  
Spratt BG (1995) Genetic analysis of clinical isolates of  
*Streptococcus pneumoniae* with high-level resistance to  
expanded-spectrum cephalosporins. *Antimicrob Agents  
Chemother* **39**: 1306–1313.

Coffey TJ, Daniels M, Enright MC & Spratt BG (1999) Serotype  
14 variants of the Spanish penicillin-resistant serotype 9V  
clone of *Streptococcus pneumoniae* arose by large  
recombinational replacements of the cpsA-pbp1a region.  
*Microbiology* **145**(Part 8): 2023–2031.

Coque JJ, Liras P & Martin JF (1993) Genes for a beta-lactamase,  
a penicillin-binding protein and a transmembrane protein are  
clustered with the cephamycin biosynthetic genes in *Nocardia  
lactamdurans*. *EMBO J* **12**: 631–639.

Couto I, Wu SW, Tomas A & de Lencastre H (2003)  
Development of methicillin resistance in clinical isolates of  
*Staphylococcus sciuri* by transcriptional activation of the mecA  
homologue native to s. *J Bacteriol* **185**: 645–653.

Dabernat H, Delmas C, Seguy M, Pelissier R, Faucon G,  
Bennamei S & Pasquier C (2002) Diversity of beta-lactam  
resistance-conferring amino acid substitutions in penicillin-  
binding protein 3 of *Haemophilus influenzae*. *Antimicrob  
Agents Chemother* **46**: 2208–2218.

Davies TA, Shang W & Bush K (2006) Activities of ceftobiprole  
and other β-lactams against *Streptococcus pneumoniae* clinical  
isolates from the United States with defined substitutions in  
penicillin-binding proteins PBP1a, PBP2b and PBP2x.  
*Antimicrob Agents Chemother* **50**: 2530–2532.

Low-affinity PBPs

De Lencastre H, Wu SW, Pinho MG, Ludovice AM, Filipe S, Gardete S, Sobral R, Gill S, Chung M & Tomasz A (1999) Antibiotic resistance as a stress response: complete sequencing of a large number of chromosomal loci in *Staphylococcus aureus* strain COL that impact on the expression of resistance to methicillin. *Microb Drug Resist* 5: 163–175.

DeLoney CR & Schiller NL (2000) Characterization of an *in vitro*-selected amoxicillin-resistant strain of *Helicobacter pylori*. *Antimicrob Agents Chemother* 44: 3368–3373.

de Pedro MA, Holtje JV & Schwarz H (2002) Fast lysis of *Escherichia coli* filament cells requires differentiation of potential division sites. *Microbiology* 148: 79–86.

Dessen A, Mouz N, Gordon E, Hopkins J & Dideberg O (2001) Crystal structure of PBP2x from a highly penicillin-resistant *Streptococcus pneumoniae* clinical isolate: a mosaic framework containing 83 mutations. *J Biol Chem* 276: 45106–45112.

Di Guilmi AM, Mouz N, Petillot Y, Forest E, Dideberg O & Vernet T (2000) Deacylation kinetics analysis of *Streptococcus pneumoniae* penicillin-binding protein 2x mutants resistant to beta-lactam antibiotics using electrospray ionization-mass spectrometry. *Anal Biochem* 284: 240–246.

Dougherty TJ (1985) Analysis of *Neisseria gonorrhoeae* peptidoglycan by reverse-phase, high-pressure liquid chromatography. *J Bacteriol* 163: 69–74.

Dougherty TJ (1986) Genetic analysis and penicillin-binding protein alterations in *Neisseria gonorrhoeae* with chromosomally mediated resistance. *Antimicrob Agents Chemother* 30: 649–652.

Dougherty TJ, Koller AE & Tomasz A (1980) Penicillin-binding proteins of penicillin-susceptible and intrinsically resistant *Neisseria gonorrhoeae*. *Antimicrob Agents Chemother* 18: 730–737.

Dowson CG, Hutchison A, Brannigan JA, George RC, Hansman D, Linares J, Tomasz A, Smith JM & Spratt BG (1989a) Horizontal transfer of penicillin-binding protein genes in penicillin-resistant clinical isolates of *Streptococcus pneumoniae*. *Proc Natl Acad Sci USA* 86: 8842–8846.

Dowson CG, Hutchison A & Spratt BG (1989b) Extensive re-modelling of the transpeptidase domain of penicillin-binding protein 2B of a penicillin-resistant South African isolate of *Streptococcus pneumoniae*. *Mol Microbiol* 3: 95–102.

Dowson CG, Hutchison A, Woodford N, Johnson AP, George RC & Spratt BG (1990) Penicillin-resistant viridans streptococci have obtained altered penicillin-binding protein genes from penicillin-resistant strains of *Streptococcus pneumoniae*. *Proc Natl Acad Sci USA* 87: 5858–5862.

Duez C, Zorzi W, Sapunaric F, Amoroso A, Thamm I & Coyette J (2001) The penicillin resistance of *Enterococcus faecalis* JH2-2r results from an overproduction of the low-affinity penicillin-binding protein PBP4 and does not involve a psr-like gene. *Microbiology* 147: 2561–2569.

Du Plessis M, Smith AM & Klugman KP (2000) Analysis of penicillin-binding protein lb and 2a genes from *Streptococcus pneumoniae*. *Microb Drug Resist* 6: 127–131.


du Plessis M, Bingen E & Klugman KP (2002) Analysis of penicillin-binding protein genes of clinical isolates of *Streptococcus pneumoniae* with reduced susceptibility to amoxicillin. *Antimicrob Agents Chemother* 46: 2349–2357.

el Kharroubi A, Jacques P, Piras G, Van Beeumen J, Coyette J & Ghuy森 JM (1991) The *Enterococcus hirae* R40 penicillin-binding protein 5 and the methicillin-resistant *Staphylococcus aureus* penicillin-binding protein 2 are similar. *Biochem J* 280: 463–469.

Ender M, McCallum N, Adhikari R & Berger-Bachi B (2004) Fitness cost of SCCmec and methicillin resistance levels in *Staphylococcus aureus*. *Antimicrob Agents Chemother* 48: 2295–2297.

Enright MC & Spratt BG (1999) Extensive variation in the ddl gene of penicillin-resistant *Streptococcus pneumoniae* results from a hitchhiking effect driven by the penicillin-binding protein 2b gene. *Mol Biol Evol* 16: 1687–1695.

Ferroni A & Berche P (2001) Alterations to penicillin-binding proteins 1A, 2B and 2X amongst penicillin-resistant clinical isolates of *Streptococcus pneumoniae* serotype 23F from the nasopharyngeal flora of children. *J Med Microbiol* 50: 828–832.

Filipe SR & Tomasz A (2000) Inhibition of the expression of penicillin resistance in *Streptococcus pneumoniae* by inactivation of cell wall muropeptide branching genes. *Proc Natl Acad Sci USA* 97: 4891–4896.

Fontana R, Cerini R, Longoni P, Grossato A & Canepari P (1983) Identification of a streptococcal penicillin-binding protein that reacts very slowly with penicillin. *J Bacteriol* 155: 1343–1350.

Fontana R, Grossato A, Rossi L, Cheng YR & Satta G (1985) Transition from resistance to hypersusceptibility to beta-lactam antibiotics associated with loss of a low-affinity penicillin-binding protein in a *Streptococcus faecium* mutant highly resistant to penicillin. *Antimicrob Agents Chemother* 28: 678–683.

Fontana R, Aldegheri M, Ligozzi M, Lopez H, Sucari A & Satta G (1994) Overproduction of a low-affinity penicillin-binding protein and high-level ampicillin resistance in *Enterococcus faecium*. *Antimicrob Agents Chemother* 38: 1980–1983.

Fuda C, Suvorov M, Vakulenko SB & Mobashery S (2004) The basis for resistance to beta-lactam antibiotics by penicillin-binding protein 2a of methicillin-resistant *Staphylococcus aureus*. *J Biol Chem* 279: 40802–40806.

Fuda C, Heseк D, Lee M, Morio K, Nowak T & Mobashery S (2005) Activation for catalysis of penicillin-binding protein 2a from Methicillin-resistant *Staphylococcus aureus* by bacterial cell wall. *J Am Chem Soc* 127: 2056–2057.

Fuda C, Heseк D, Lee M, Heilmayer W, Novak R, Vakulenko SB & Mobashery S (2006) Mechanistic basis for the action of new cephalosporin antibiotics effective against methicillin- and vancomycin-resistant *Staphylococcus aureus*. *J Biol Chem* 281: 10035–10041.

Garcia-Bustos J & Dougherty TJ (1987) Alterations in peptidoglycan of *Neisseria gonorrhoeae* induced by sub-MICs

of beta-lactam antibiotics. Antimicrob Agents Chemother 31: 178–182.

Garcia-Bustos J & Tomasz A (1990) A biological price of antibiotic resistance: major changes in the peptidoglycan structure of penicillin-resistant pneumococci. Proc Natl Acad Sci USA 87: 5415–5419.

Garcia-Castellanos R, Mallorqui-Fernandez G, Marrero A, Potempa J, Coll M & Gomis-Ruth FX (2004) On the transcriptional regulation of methicillin resistance: MecI repressor in complex with its operator. J Biol Chem 279: 17888–17896.

Gehrlein M, Leying H, Cullmann W, Wendt S & Opferkuch W (1991) Imipenem resistance in Acinetobacter baumanii is due to altered penicillin-binding proteins. Chemotherapy 37: 405–412.

Georgopapadakou NH, Smith SA & Bonner DP (1982) Penicillin-binding proteins in a Staphylococcus aureus strain resistant to specific beta-lactam antibiotics. Antimicrob Agents Chemother 22: 172–175.

Gerrits MM, Schuijffel D, van Zwet AA, Kuipers EJ, Vandenbroucke-Grauls CM & Kusters JG (2002) Alterations in penicillin-binding protein 1A confer resistance to beta-lactam antibiotics in Helicobacter pylori. Antimicrob Agents Chemother 46: 2229–2233.

Giesbrecht P, Kersten T, Maidhof H & Wecke J (1998) Staphylococcal cell wall: morphogenesis and fatal variations in the presence of penicillin. Microbiol Mol Biol Rev 62: 1371–1414.

Gill MJ, Simjee S, Al-Hattawi K, Robertson BD, Easmon CS & Ison CA (1998) Gonococcal resistance to beta-lactams and tetracycline involves mutation in loop 3 of the porin encoded at the penB locus. Antimicrob Agents Chemother 42: 2799–2803.

Goffin C & Ghuysen JM (2002) Multimodular penicillin-binding proteins: an enigmatic family of orthologs and paralogs. Microbiol Mol Biol Rev 62: 1079–1093.

Gordon E, Mouz N, Duee E & Dideberg O (2000) The crystal structure of the penicillin-binding protein 2x from Streptococcus pneumoniae and its acyl-enzyme form: implication in drug resistance. J Mol Biol 299: 477–485.

Gotoh N, Nunomura K & Nishino T (1990) Resistance of Pseudomonas aeruginosa to cefsulodin: modification of penicillin-binding protein 3 and mapping of its chromosomal gene. J Antimicrob Chemother 25: 513–523.

Graves-Woodward K & Pratt RF (1998) Reaction of soluble penicillin-binding protein 2a of methicillin-resistant Staphylococcus aureus with beta-lactams and acyclic substrates: kinetics in homogeneous solution. Biochem J 332(Part 3): 755–761.

Grebe T & Hakenbeck R (1996) Penicillin-binding proteins 2b and 2x of Streptococcus pneumoniae are primary resistance determinants for different classes of beta-lactam antibiotics. Antimicrob Agents Chemother 40: 829–834.

Gutkind GO, Ogueta SB, de Uriaga AC, Mollerach ME & de Torres RA (1990) Participation of PBP 3 in the acquisition of

dicloxacillin resistance in Listeria monocytogenes. J Antimicrob Chemother 25: 751–758.

Hackbarth CJ, Kocagoz T, Kocagoz S & Chambers HF (1995) Point mutations in Staphylococcus aureus PBP 2 gene affect penicillin-binding kinetics and are associated with resistance. Antimicrob Agents Chemother 39: 103–106.

Hagman KE, Pan W, Spratt BG, Balthazar JT, Judd RC & Shafer WM (1995) Resistance of Neisseria gonorrhoeae to antimicrobial hydrophobic agents is modulated by the mtrRCDE efflux system. Microbiology 141(Part 3): 611–622.

Hakenbeck R, Tarpay M & Tomasz A (1980) Multiple changes of penicillin-binding proteins in penicillin-resistant clinical isolates of Streptococcus pneumoniae. Antimicrob Agents Chemother 17: 364–371.

Hakenbeck R, Tornette S & Adkinson NF (1987) Interaction of non-lytic beta-lactams with penicillin-binding proteins in Streptococcus pneumoniae. J Gen Microbiol 133(Part 3): 755–760.

Hakenbeck R, König A, Kern I, van der Linden M, Keck W, Billot-Klein D, Legrand R, Schoot B & Gutmann L (1998) Acquisition of five high-Mr penicillin-binding protein variants during transfer of high-level beta-lactam resistance from Streptococcus mitis to Streptococcus pneumoniae. J Bacteriol 180: 1831–1840.

Han SR, Bhakdi S, Maurer MJ, Schneider T & Gehring S (1999) Stable and unstable amoxicillin resistance in Helicobacter pylori: should antibiotic resistance testing be performed prior to eradication therapy? J Clin Microbiol 37: 2740–2741.

Hartman BJ & Tomasz A (1984) Low-affinity penicillin-binding protein associated with beta-lactam resistance in Staphylococcus aureus. J Bacteriol 158: 513–516.

Hasegawa K, Chiba N, Kobayashi R, Murayama SY, Iwata S, Sunakawa K & Ubukata K (2004) Rapidly increasing prevalence of beta-lactamase-nonproducing, ampicillin-resistant Haemophilus influenzae type b in patients with meningitis. Antimicrob Agents Chemother 48: 1509–1514.

Hebeisen P, Heinze-Krauss I, Angehrn P, Hohl P, Page MGP & Then RL (2001) In vitro and in vivo properties of Ro 63–9141, a novel broad-spectrum cephalosporin with activity against methicillin-resistant Staphylococci. Antimicrob Agents Chemother 45: 825–836.

Hedge PJ & Spratt BG (1985a) Amino acid substitutions that reduce the affinity of penicillin-binding protein 3 of Escherichia coli for cephalixin. Eur J Biochem 151: 111–121.

Hedge PJ & Spratt BG (1985b) Resistance to beta-lactam antibiotics by re-modelling the active site of an E. coli penicillin-binding protein. Nature 318: 478–480.

Henze U, Sidow T, Wecke J, Labischinski H & Berger-Bachi B (1993) Influence of femB on methicillin resistance and peptidoglycan metabolism in Staphylococcus aureus. J Bacteriol 175: 1612–1620.

Henze UU & Berger-Bachi B (1996) Penicillin-binding protein 4 overproduction increases beta-lactam resistance in Staphylococcus aureus. Antimicrob Agents Chemother 40: 2121–2125.

Hiramatsu K, Cui L, Kuroda M & Ito T (2001) The emergence and evolution of methicillin-resistant *Staphylococcus aureus*. *Trends Microbiol* **9**: 486–493.

Hoskins J, Matsushima P, Mullen DL, Tang J, Zhao G, Meier TI, Nicas TI & Jaskunas SR (1999) Gene disruption studies of penicillin-binding proteins 1a, 1b, and 2a in *Streptococcus pneumoniae*. *J Bacteriol* **181**: 6552–6555.

Hujer AM, Kania M, Gerken T, Anderson VE, Buynak JD, Ge X, Caspers P, Page MGP, Rice LB & Bonomo RA (2005) Structure-activity relationships of different β-lactam antibiotics against a soluble form of *Enterococcus faecium* PBP5, a type II bacterial transpeptidase. *Antimicrob Agents Chemother* **49**: 612–618.

Ito M, Deguchi T, Mizutani KS, Yasuda M, Yokoi S, Ito S, Takahashi Y, Ishihara S, Kawamura Y & Ezaki T (2005) Emergence and spread of *Neisseria gonorrhoeae* clinical isolates harboring mosaic-like structure of penicillin-binding protein 2 in central Japan. *Antimicrob Agents Chemother* **49**: 137–143.

Ito T, Katayama Y, Asada K, Mori N, Tsutsumimoto K, Tiensasitorn C & Hiramatsu K (2001) Structural comparison of three types of staphylococcal cassette chromosome mec integrated in the chromosome in methicillin-resistant *Staphylococcus aureus*. *Antimicrob Agents Chemother* **45**: 1323–1336.

Jamin M, Damblon C, Millier S, Hakenbeck R & Frere JM (1993) Penicillin-binding protein 2x of *Streptococcus pneumoniae*: enzymic activities and interactions with beta-lactams. *Biochem J* **292**(Part 3): 735–741.

Jevons MP, Coe AW & Parker MT (1963) Methicillin resistance in staphylococci. *Lancet* **1**: 904–907.

Jureen R, Mohn SC, Harthug S, Haarr L & Langeland N (2004) Role of penicillin-binding protein 5 C-terminal amino acid substitutions in conferring ampicillin resistance in Norwegian clinical strains of *Enterococcus faecium*. *Apmis* **112**: 291–298.

Kaczmarek FS, Gootz TD, Dib-Hajj F, Shang W, Hallowell S & Cronan M (2004) Genetic and molecular characterization of beta-lactamase-negative ampicillin-resistant *Haemophilus influenzae* with unusually high resistance to ampicillin. *Antimicrob Agents Chemother* **48**: 1630–1639.

Kell CM, Jordens JZ, Daniels M, Coffey TJ, Bates J, Paul J, Gilks C & Spratt BG (1993) Molecular epidemiology of penicillin-resistant pneumococci isolated in Nairobi, Kenya. *Infect Immun* **61**: 4382–4391.

Klare I, Rodloff AC, Wagner J, Witte W & Hakenbeck R (1992) Overproduction of a penicillin-binding protein is not the only mechanism of penicillin resistance in *Enterococcus faecium*. *Antimicrob Agents Chemother* **36**: 783–787.

Kosowska K, Jacobs MR, Bajaksouzian S, Koeth L & Appelbaum PC (2004) Alterations of penicillin-binding proteins 1A, 2X, and 2B in *Streptococcus pneumoniae* isolates for which amoxicillin MICs are higher than penicillin MICs. *Antimicrob Agents Chemother* **48**: 4020–4022.

Krauss J, van der Linden M, Grebe T & Hakenbeck R (1996) Penicillin-binding proteins 2x and 2b as primary PBP targets in *Streptococcus pneumoniae*. *Microb Drug Resist* **2**: 183–186.


Kwon DH, Dore MP, Kim JJ, Kato M, Lee M, Wu JY & Graham DY (2003) High-level beta-lactam resistance associated with acquired multidrug resistance in *Helicobacter pylori*. *Antimicrob Agents Chemother* **47**: 2169–2178.

Laible G & Hakenbeck R (1987) Penicillin-binding proteins in beta-lactam-resistant laboratory mutants of *Streptococcus pneumoniae*. *Mol Microbiol* **1**: 355–363.

Laible G, Spratt BG & Hakenbeck R (1991) Interspecies recombinational events during the evolution of altered PBP 2x genes in penicillin-resistant clinical isolates of *Streptococcus pneumoniae*. *Mol Microbiol* **5**: 1993–2002.

Leski TA & Tomasz A (2005) Role of penicillin-binding protein 2 (PBP2) in the antibiotic susceptibility and cell wall cross-linking of *Staphylococcus aureus*: evidence for the cooperative functioning of PBP2, PBP4, and PBP2A. *J Bacteriol* **187**: 1815–1824.

Ligozzi M, Pittaluga F & Fontana R (1996) Modification of penicillin-binding protein 5 associated with high-level ampicillin resistance in *Enterococcus faecium*. *Antimicrob Agents Chemother* **40**: 354–357.

Lim D & Strynadka NC (2002) Structural basis for the beta-lactam resistance of PBP2a from methicillin-resistant *Staphylococcus aureus*. *Nat Struct Biol* **9**: 870–876.

Lleo MM, Canepari P, Cornaglia G, Fontana R & Satta G (1987) Bacteriostatic and bactericidal activities of beta-lactams against *Streptococcus (Enterococcus) faecium* are associated with saturation of different penicillin-binding proteins. *Antimicrob Agents Chemother* **31**: 1618–1626.

Lu WP, Sun Y, Bauer MD, Paule S, Koenigs PM & Kraft WG (1999) Penicillin-binding protein 2a from methicillin-resistant *Staphylococcus aureus*: kinetic characterization of its interactions with beta-lactams using electrospray mass spectrometry. *Biochemistry* **38**: 6537–6546.

Lu WP, Kincaid E, Sun Y & Bauer MD (2001) Kinetics of beta-lactam interactions with penicillin-susceptible and -resistant penicillin-binding protein 2x proteins from *Streptococcus pneumoniae*. Involvement of acylation and deacylation in beta-lactam resistance. *J Biol Chem* **276**: 31494–31501.

Lujan R, Zhang QY, Saez Nieto JA, Jones DM & Spratt BG (1991) Penicillin-resistant isolates of *Neisseria lactamica* produce altered forms of penicillin-binding protein 2 that arose by interspecies horizontal gene transfer. *Antimicrob Agents Chemother* **35**: 300–304.

Malouin F, Schryvers AB & Bryan LE (1987) Cloning and expression of genes responsible for altered penicillin-binding proteins 3a and 3b in *Haemophilus influenzae*. *Antimicrob Agents Chemother* **31**: 286–291.

Markowitz SM (1980) Isolation of an ampicillin-resistant, non-beta-lactamase-producing strain of *Haemophilus influenzae*. *Antimicrob Agents Chemother* **17**: 80–83.

Martin C, Sibold C & Hakenbeck R (1992) Relatedness of penicillin-binding protein 1a genes from different clones of penicillin-resistant *Streptococcus pneumoniae* isolated in South Africa and Spain. *Embo J* **11**: 3831–3836.

Mascher T, Heintz M, Zahner D, Merai M & Hakenbeck R (2006) The CiaRH system of *Streptococcus pneumoniae* prevents lysis during stress induced by treatment with cell wall inhibitors and by mutations in pbp2x involved in beta-lactam resistance. *J Bacteriol* 188: 1959–1968.

Matic V, Bozdogan B, Jacobs MR, Ubukata K & Appelbaum PC (2003) Contribution of beta-lactamase and PBP amino acid substitutions to amoxicillin/clavulanate resistance in beta-lactamase-positive, amoxicillin/clavulanate-resistant *Haemophilus influenzae*. *J Antimicrob Chemother* 52: 1018–1021.

McDougal LK, Rasheed JK, Biddle JW & Tenover FC (1995) Identification of multiple clones of extended-spectrum cephalosporin-resistant *Streptococcus pneumoniae* isolates in the United States. *Antimicrob Agents Chemother* 39: 2282–2288.

McGee L, McDougal L, Zhou J *et al.* (2001) Nomenclature of major antimicrobial-resistant clones of *Streptococcus pneumoniae* defined by the pneumococcal molecular epidemiology network. *J Clin Microbiol* 39: 2565–2571.

Mendelman PM, Chaffin DO, Stull TL, Rubens CE, Mack KD & Smith AL (1984) Characterization of non-beta-lactamase-mediated ampicillin resistance in *Haemophilus influenzae*. *Antimicrob Agents Chemother* 26: 235–244.

Mendelman PM, Campos J, Chaffin DO, Serfass DA, Smith AL & Saez-Nieto JA (1988) Relative penicillin G resistance in *Neisseria meningitidis* and reduced affinity of penicillin-binding protein 3. *Antimicrob Agents Chemother* 32: 706–709.

Moellering RC Jr, Korzeniowski OM, Sande MA & Wennersten CB (1979) Species-specific resistance to antimicrobial synergism in *Streptococcus faecium* and *Streptococcus faecalis*. *J Infect Dis* 140: 203–208.

Munoz R, Dowson CG, Daniels M, Coffey TJ, Martin C, Hakenbeck R & Spratt BG (1992) Genetics of resistance to third-generation cephalosporins in clinical isolates of *Streptococcus pneumoniae*. *Mol Microbiol* 6: 2461–2465.

Mouz N, Gordon E, Di Guilmi AM, Petit I, Petillot Y, Dupont Y, Hakenbeck R, Vernet T & Dideberg O (1998) Identification of a structural determinant for resistance to beta-lactam antibiotics in gram-positive bacteria. *Proc Natl Acad Sci USA* 95: 13403–13406.

Mouz N, Di Guilmi AM, Gordon E, Hakenbeck R, Dideberg O & Vernet T (1999) Mutations in the active site of penicillin-binding protein PBP2x from *Streptococcus pneumoniae*. Role in the specificity for beta-lactam antibiotics. *J Biol Chem* 274: 19175–19180.

Murray BE (1992) Beta-lactamase-producing enterococci. *Antimicrob Agents Chemother* 36: 2355–2359.

Nagai K, Davies TA, Jacobs MR & Appelbaum PC (2002) Effects of amino acid alterations in penicillin-binding proteins (PBPs) 1a, 2b, and 2x on PBP affinities of penicillin, ampicillin, amoxicillin, cefditoren, cefuroxime, cefprozil, and cefaclor in 18 clinical isolates of penicillin-susceptible, -intermediate, and -resistant pneumococci. *Antimicrob Agents Chemother* 46: 1273–1280.


Neuwirth C, Siebor E, Duez JM, Pechinot A & Kazmierczak A (1995) Imipenem resistance in clinical isolates of *Proteus mirabilis* associated with alterations in penicillin-binding proteins. *J Antimicrob Chemother* 36: 335–342.

Nichol KA, Zhanel GG & Hoban DJ (2002) Penicillin-binding protein 1A, 2B, and 2X alterations in Canadian isolates of penicillin-resistant *Streptococcus pneumoniae*. *Antimicrob Agents Chemother* 46: 3261–3264.

Nordmann P, Nicolas MH & Gutmann L (1993) Penicillin-binding proteins of *Rhodococcus equi*: potential role in resistance to imipenem. *Antimicrob Agents Chemother* 37: 1406–1409.

Obara M & Nakae T (1991) Mechanisms of resistance to beta-lactam antibiotics in *Acinetobacter calcoaceticus*. *J Antimicrob Chemother* 28: 791–800.

Okamoto T, Yoshiyama H, Nakazawa T, Park ID, Chang MW, Yanai H, Okita K & Shirai M (2002) A change in PBPl is involved in amoxicillin resistance of clinical isolates of *Helicobacter pylori*. *J Antimicrob Chemother* 50: 849–856.

Oliva M, Dideberg O & Field MJ (2003) Understanding the acylation mechanisms of active-site serine penicillin-recognizing proteins: a molecular dynamics simulation study. *Proteins* 53: 88–100.

Ono S, Muratani T & Matsumoto T (2005) Mechanisms of resistance to imipenem and ampicillin in *Enterococcus faecalis*. *Antimicrob Agents Chemother* 49: 2954–2958.

Osaki Y, Sanbongi Y, Ishikawa M, Kataoka H, Suzuki T, Maeda K & Ida T (2005) Genetic approach to study the relationship between penicillin-binding protein 3 mutations and *Haemophilus influenzae* beta-lactam resistance by using site-directed mutagenesis and gene recombinants. *Antimicrob Agents Chemother* 49: 2834–2839.

Overweg K, Bogaert D, Sluijter M, de Groot R & Hermans PW (2001) Molecular characteristics of penicillin-binding protein genes of penicillin-nonsusceptible *Streptococcus pneumoniae* isolated in the Netherlands. *Microb Drug Resist* 7: 323–334.

Page MGP (2006) Anti-MRSA beta-lactams in development. *Curr Opin Pharmacol* 6: 480–485.

Paglieri E, Chesnel L, Hopkins J, Croize J, Dideberg O, Vernet T & Di Guilmi AM (2004) Biochemical characterization of *Streptococcus pneumoniae* penicillin-binding protein 2b and its implication in beta-lactam resistance. *Antimicrob Agents Chemother* 48: 1848–1855.

Paik J, Kern I, Lurz R & Hakenbeck R (1999) Mutational analysis of the *Streptococcus pneumoniae* bimodular class A penicillin-binding proteins. *J Bacteriol* 181: 3852–3856.

Paradkar AS, Aidoo KA, Wong A & Jensen SE (1996) Molecular analysis of a beta-lactam resistance gene encoded within the cephamycin gene cluster of *Streptomyces clavuligerus*. *J Bacteriol* 178: 6266–6274.

Pares S, Mouz N, Petillot Y, Hakenbeck R & Dideberg O (1996) X-ray structure of *Streptococcus pneumoniae* PBP2x, a primary penicillin target enzyme. *Nat Struct Biol* 3: 284–289.

Parr TR Jr & Bryan LE (1984) Mechanism of resistance of an ampicillin-resistant, beta-lactamase-negative clinical isolate of

© 2008 Federation of European Microbiological Societies
Published by Blackwell Publishing Ltd. All rights reserved

Low-affinity PBPs

Haemophilus influenzae type b to beta-lactam antibiotics.
Antimicrob Agents Chemother 25: 747–753.

Paul R, Postius S, Melchers K & Schafer KP (2001) Mutations of the Helicobacter pylori genes rdxA and pbp1 cause resistance against metronidazole and amoxicillin. Antimicrob Agents Chemother 45: 962–965.

Pereira SF, Henriques AO, Pinho MG, de Lencastre H & Tomasz A (2007) Role of PBP1 in cell division of Staphylococcus aureus. J Bacteriol 189: 3525–3531.

Perez JL, Riera L, Valls F, Berrocal CI & Berrocal L (1987) A comparison of the in-vitro activity of seventeen antibiotics against Streptococcus faecalis. J Antimicrob Chemother 20: 357–362.

Pernot L, Chesnel L, Le Gouellec A, Croize J, Vernet T, Dideberg O & Dessen A (2004) A PBP2x from a clinical isolate of Streptococcus pneumoniae exhibits an alternative mechanism for reduction of susceptibility to beta-lactam antibiotics. J Biol Chem 279: 16463–16470.

Pierre J, Boisivon A & Gutmann L (1990a) Alteration of PBP 3 entails resistance to imipenem in Listeria monocytogenes. Antimicrob Agents Chemother 34: 1695–1698.

Pierre J, Williamson R, Bornet M & Gutmann L (1990b) Presence of an additional penicillin-binding protein in methicillin-resistant Staphylococcus epidermidis, Staphylococcus haemolyticus, Staphylococcus hominis, and Staphylococcus simulans with a low affinity for methicillin, cephalothin, and cefamandole. Antimicrob Agents Chemother 34: 1691–1694.

Pinho MG, de Lencastre H & Tomasz A (2001a) An acquired and a native penicillin-binding protein cooperate in building the cell wall of drug-resistant staphylococci. Proc Natl Acad Sci USA 98: 10886–10891.

Pinho MG, Filipe SR, de Lencastre H & Tomasz A (2001b) Complementation of the essential peptidoglycan transpeptidase function of penicillin-binding protein 2 (PBP2) by the drug resistance protein PBP2A in Staphylococcus aureus. J Bacteriol 183: 6525–6531.

Pucci MJ & Dougherty TJ (2002) Direct quantitation of the numbers of individual penicillin-binding proteins per cell in Staphylococcus aureus. J Bacteriol 184: 588–591.

Pucci MJ, Hinks ET, Dicker DT, Higgins ML & Daneo-Moore L (1986) Inhibition of beta-lactam antibiotics at two different times in the cell cycle of Streptococcus faecium ATCC 9790. J Bacteriol 165: 682–688.

Raze D, Dardenne O, Hallut S, Martinez-Bueno M, Coyette J & Ghysen JM (1998) The gene encoding the low-affinity penicillin-binding protein 3r in Enterococcus hirae S185R is borne on a plasmid carrying other antibiotic resistance determinants. Antimicrob Agents Chemother 42: 534–539.

Reichmann P, König A, Marton A & Hakenbeck R (1996) Penicillin-binding proteins as resistance determinants in clinical isolates of Streptococcus pneumoniae. Microb Drug Resist 2: 177–181.

Reichmann P, König A, Linares J, Alcaide F, Tenover FC, McDougal L, Swidsinski S & Hakenbeck R (1997) A global gene pool for high-level cephalosporin resistance in

commensal Streptococcus species and Streptococcus pneumoniae. J Infect Dis 176: 1001–1012.

Rice LB, Carias LL, Hutton-Thomas R, Sifaoui F, Gutmann L & Rudin SD (2001) Penicillin-binding protein 5 and expression of ampicillin resistance in Enterococcus faecium. Antimicrob Agents Chemother 45: 1480–1486.

Rice LB, Bellais S, Carias LL, Hutton-Thomas R, Bonomo RA, Caspers P, Page MGP & Gutmann L (2004) Impact of specific pbp5 mutations on expression of β-lactam resistance in Enterococcus faecium. Antimicrob Agents Chemother 48: 3028–3032.

Ropp PA, Hu M, Olesky M & Nicholas RA (2002) Mutations in ponA, the gene encoding penicillin-binding protein 1, and a novel locus, penC, are required for high-level chromosomally mediated penicillin resistance in Neisseria gonorrhoeae. Antimicrob Agents Chemother 46: 769–777.

Rybkin T, Mainardi JL, Sougakoff W, Collatz E & Gutmann L (1998) Penicillin-binding protein 5 sequence alterations in clinical isolates of Enterococcus faecium with different levels of beta-lactam resistance. J Infect Dis 178: 159–163.

Saez-Nieto JA, Lujan R, Martinez-Suarez JV, Berron S, Vazquez JA, Vinas M & Campos J (1990) Neisseria lactamica and Neisseria polysaccharea as possible sources of meningococcal beta-lactam resistance by genetic transformation. Antimicrob Agents Chemother 34: 2269–2272.

Sanbongi Y, Ida T, Ishikawa M, Osaki Y, Kataoka H, Suzuki T, Kondo K, Ohsawa F & Yonezawa M (2004) Complete sequences of six penicillin-binding protein genes from 40 Streptococcus pneumoniae clinical isolates collected in Japan. Antimicrob Agents Chemother 48: 2244–2250.

Sauvage E, Kerff F, Terrak M, Ayala J & Charlier P (2008) The penicillin-binding proteins: structure and role in peptidoglycan biosynthesis. FEMS Microbiol Rev 32: doi: 10.1111/j.1574-6976.2008.00105.x

Sauvage E, Kerff F, Fonze E et al. (2002) The 2.4-A crystal structure of the penicillin-resistant penicillin-binding protein PBP5fm from Enterococcus faecium in complex with benzylpenicillin. Cell Mol Life Sci 59: 1223–1232.

Schnellmann C, Gerber V, Rossano A, Jaquier V, Panchaud Y, Doherr MG, Thomann A, Straub R & Perreten V (2006) Presence of new mecA and mph(C) variants conferring antibiotic resistance in Staphylococcus spp. isolated from the skin of horses before and after clinic admission. J Clin Microbiol 44: 4444–4454.

Schuster C, Dobrinski B & Hakenbeck R (1990) Unusual septum formation in Streptococcus pneumoniae mutants with an alteration in the D,D-carboxypeptidase penicillin-binding protein 3. J Bacteriol 172: 6499–6505.

Severin A, Figueiredo AM & Tomasz A (1996) Separation of abnormal cell wall composition from penicillin resistance through genetic transformation of Streptococcus pneumoniae. J Bacteriol 178: 1788–1792.

Sibold C, Henriksen J, König A, Martin C, Chalkley L & Hakenbeck R (1994) Mosaic pbpX genes of major clones of penicillin-resistant Streptococcus pneumoniae have evolved

FEMS Microbiol Rev 32 (2008) 361–385

© 2008 Federation of European Microbiological Societies Published by Blackwell Publishing Ltd. All rights reserved

from pbpX genes of a penicillin-sensitive Streptococcus oralis. Mol Microbiol 12: 1013–1023.

Sifaoui F, Kitzis MD & Gutmann L (1996) In vitro selection of one-step mutants of Streptococcus pneumoniae resistant to different oral beta-lactam antibiotics is associated with alterations of PBP2x. Antimicrob Agents Chemother 40: 152–156.

Sifaoui F, Arthur M, Rice L & Gutmann L (2001) Role of penicillin-binding protein 5 in expression of ampicillin resistance and peptidoglycan structure in Enterococcus faecium. Antimicrob Agents Chemother 45: 2594–2597.

Smith AM & Klugman KP (1995) Alterations in penicillin- binding protein 2B from penicillin-resistant wild-type strains of Streptococcus pneumoniae. Antimicrob Agents Chemother 39: 859–867.

Smith AM & Klugman KP (1998) Alterations in PBP 1A essential- for high-level penicillin resistance in Streptococcus pneumoniae. Antimicrob Agents Chemother 42: 1329–1333.

Smith AM & Klugman KP (2001) Alterations in MurM, a cell wall muropeptide branching enzyme, increase high-level penicillin and cephalosporin resistance in Streptococcus pneumoniae. Antimicrob Agents Chemother 45: 2393–2396.

Smith AM & Klugman KP (2003) Site-specific mutagenesis analysis of PBP 1A from a penicillin-cephalosporin-resistant pneumococcal isolate. Antimicrob Agents Chemother 47: 387–389.

Smith AM & Klugman KP (2005) Amino acid mutations essential to production of an altered PBP 2X conferring high-level beta- lactam resistance in a clinical isolate of Streptococcus pneumoniae. Antimicrob Agents Chemother 49: 4622–4627.

Smith AM, Feldman C, Massidda O, McCarthy K, Ndiweni D & Klugman KP (2005) Altered PBP 2A and its role in the development of penicillin, cefotaxime, and ceftriaxone resistance in a clinical isolate of Streptococcus pneumoniae. Antimicrob Agents Chemother 49: 2002–2007.

Song JH, Yang JW, Jin JH *et al.* (2000) Molecular characterization of multidrug-resistant Streptococcus pneumoniae isolates in Korea. The Asian network for Surveillance of resistant pathogens (ANSORP) study group. J Clin Microbiol 38: 1641–1644.

Spratt BG (1978) Escherichia coli resistance to beta-lactam antibiotics through a decrease in the affinity of a target for lethality. Nature 274: 713–715.

Spratt BG (1988) Hybrid penicillin-binding proteins in penicillin-resistant strains of Neisseria gonorrhoeae. Nature 332: 173–176.

Spratt BG, Zhang QY, Jones DM, Hutchison A, Brannigan JA & Dowson CG (1989) Recruitment of a penicillin-binding protein gene from Neisseria flavescens during the emergence of penicillin resistance in Neisseria meningitidis. Proc Natl Acad Sci USA 86: 8988–8992.

Spratt BG, Bowler LD, Zhang QY, Zhou J & Smith JM (1992) Role of interspecies transfer of chromosomal genes in the evolution of penicillin resistance in pathogenic and commensal Neisseria species. J Mol Evol 34: 115–125.


Stranden AM, Ehlert K, Labischinski H & Berger-Bachi B (1997) Cell wall monoglycine cross-bridges and methicillin hypersusceptibility in a femAB null mutant of methicillin- resistant Staphylococcus aureus. J Bacteriol 179: 9–16.

Thomas B, Wang Y & Stein RL (2001) Kinetic and mechanistic studies of penicillin-binding protein 2x from Streptococcus pneumoniae. Biochemistry 40: 15811–15823.

Tomasz A, Drugeon HB, de Lencastre HM, Jabes D, McDougall L & Bille J (1989) New mechanism for methicillin resistance in Staphylococcus aureus: clinical isolates that lack the PBP 2a gene and contain normal penicillin-binding proteins with modified penicillin-binding capacity. Antimicrob Agents Chemother 33: 1869–1874.

Tristram S, Jacobs MR & Appelbaum PC (2007) Antimicrobial resistance in Haemophilus influenzae. Clin Microbiol Rev 20: 368–389.

Ubukata K, Shibasaki Y, Yamamoto K, Chiba N, Hasegawa K, Takeuchi Y, Sunakawa K, Inoue M & Konno M (2001) Association of amino acid substitutions in penicillin-binding protein 3 with beta-lactam resistance in beta-lactamase- negative ampicillin-resistant Haemophilus influenzae. Antimicrob Agents Chemother 45: 1693–1699.

Ubukata K, Chiba N, Hasegawa K, Kobayashi R, Iwata S & Sunakawa K (2004) Antibiotic susceptibility in relation to penicillin-binding protein genes and serotype distribution of Streptococcus pneumoniae strains responsible for meningitis in Japan, 1999 to 2002. Antimicrob Agents Chemother 48: 1488–1494.

Urban C, Go E, Mariano N & Raha JJ (1995) Interaction of sulbactam, clavulanic acid and tazobactam with penicillin- binding proteins of imipenem-resistant and -susceptible Acinetobacter baumannii. FEMS Microbiol Lett 125: 193–197.

Utsui Y & Yokota T (1985) Role of an altered penicillin-binding protein in methicillin- and cephem-resistant Staphylococcus aureus. Antimicrob Agents Chemother 28: 397–403.

van Zwet AA, Vandenbroucke-Grauls CM, Thijs JC, van der Wouden EJ, Gerrits MM, Kusters JG & Vandenbrouke-Grauls CM (1998) Stable amoxicillin resistance in Helicobacter pylori. Lancet 352: 1595.

Weber B, Ehlert K, Diehl A, Reichmann P, Labischinski H & Hakenbeck R (2000) The fib locus in Streptococcus pneumoniae is required for peptidoglycan crosslinking and PBP-mediated beta-lactam resistance. FEMS Microbiol Lett 188: 81–85.

Wexler HM & Halebian S (1990) Alterations to the penicillin- binding proteins in the Bacteroides fragilis group: a mechanism for non-beta-lactam mediated cefoxitin resistance. J Antimicrob Chemother 26: 7–20.

Williamson R, le Bouguenec C, Gutmann L & Horaud T (1985) One or two low affinity penicillin-binding proteins may be responsible for the range of susceptibility of Enterococcus faecium to benzylpenicillin. J Gen Microbiol 131(Part 8): 1933–1940.

Wu S, de Lencastre H & Tomasz A (1998) Genetic organization of the mecA region in methicillin-susceptible and methicillin-

Low-affinity PBPs

resistant strains of *Staphylococcus sciuri*. *J Bacteriol* 180: 236–242.

Yamane A, Nakano H, Asahi Y, Ubukata K & Konno M (1996) Directly repeated insertion of 9-nucleotide sequence detected in penicillin-binding protein 2B gene of penicillin-resistant *Streptococcus pneumoniae*. *Antimicrob Agents Chemother* 40: 1257–1259.

Yasuda R, Kawano J, Onda H, Takagi M, Shimizu A & Anzai T (2000) Methicillin-resistant coagulase-negative staphylococci isolated from healthy horses in Japan. *Am J Vet Res* 61: 1451–1455.

Zervosen A, Lu WP, Chen Z, White RE, Demuth TP Jr & Frere JM (2004) Interactions between penicillin-binding proteins (PBPs) and two novel classes of PBP inhibitors, arylalkylidene rhodanines and arylalkylidene iminothiazolidin-4-ones. *Antimicrob Agents Chemother* 48: 961–969.

Zhang HZ, Hackbarth CJ, Chansky KM & Chambers HF (2001) A proteolytic transmembrane signaling pathway and resistance to beta-lactams in staphylococci. *Science* 291: 1962–1965.

Zhao G, Meier TI, Hoskins J & McAllister KA (2000) Identification and characterization of the penicillin-binding protein 2a of *Streptococcus pneumoniae* and its possible role in resistance to beta-lactam antibiotics. *Antimicrob Agents Chemother* 44: 1745–1748.

Zhou J, Enright MC & Spratt BG (2000) Identification of the major Spanish clones of penicillin-resistant pneumococci via the Internet using multilocus sequence typing. *J Clin Microbiol* 38: 977–986.

Zighelboim S & Tomasz A (1980) Penicillin-binding proteins of multiply antibiotic-resistant South African strains of *Streptococcus pneumoniae*. *Antimicrob Agents Chemother* 17: 434–442.

Zorzi W, Zhou XY, Dardenne O, Lamotte J, Raze D, Pierre J, Gutmann L & Coyette J (1996) Structure of the low-affinity penicillin-binding protein 5 PBP5fm in wild-type and highly penicillin-resistant strains of *Enterococcus faecium*. *J Bacteriol* 178: 4948–4957.
