
Characterization of the Histidine-containing Phosphotransfer Protein B-mediated Multistep Phosphorelay System in Pseudomonas aeruginosa PAO1*${}^{S}$

Received for publication, October 26, 2007, and in revised form, February 4, 2008 Published, JBC Papers in Press, February 5, 2008, DOI 10.1074/jbc.M708836200

Jye-Lin Hsu ${}^{\dagger}$, Hsuan-Cheng Chen ${}^{\ddagger}$, Hwei-Ling Peng ${}^{S}$, and Hwan-You Chang ${}^{1}$

From the ${}^{\dagger}$Institute of Molecular Medicine, National Tsing Hua University, 101 Guang Fu Rd. 2nd Sec., Hsin Chu 300 and the ${}^{S}$Department of Biological Science and Technology, National Chiao Tung University, 75 Po Ai Street, Hsin Chu, Taiwan, Republic of China

Certain bacterial two-component sensor kinases possess a histidine-containing phosphotransfer (Hpt) domain to carry out a multistep phosphotransferring reaction to a cognate response regulator. *Pseudomonas aeruginosa* PAO1 contains three genes that encode proteins with an Hpt domain but lack a kinase domain. To identify the sensor kinase coupled to these Hpt proteins, a phosphorelay profiling assay was performed. Among the 12 recombinant orphan sensor kinases tested, 4 of these sensors (PA1611, PA1976, PA2824, and RetS) transferred the phosphoryl group to HptB (PA3345). The *in vivo* interaction between HptB and each of the sensors was also confirmed using the bacterial two-hybrid assay. Interestingly, the phosphoryl groups from these sensors all appeared to be transferred via HptB to PA3346, a novel phosphatase consisting of an N-terminal receiver domain and a eukaryotic type Ser/Thr phosphatase domain, and resulted in a significant increase of its phosphatase activity. The subsequent reverse transcription-PCR analysis revealed an operon structure of *hptB-PA3346-PA3347*, suggesting a coordinate expression of the three genes to carry out a signal transduction. The possibility was supported by the analysis showing PA3347 is able to be phosphorylated on Ser-56, and this phosphoryl group could be removed by PA3346 protein. Finally, analysis of PA3346 and PA3347 gene knock-out mutants revealed that these genes are associated with bacterial swarming activity and biofilm formation. Together, these results disclose a novel multistep phosphorelay system that is essential for *P. aeruginosa* to respond to a wide spectrum of environmental signals.

response to environmental challenges (1). Typical 2CSs are composed of a sensor and a response regulator. The sensor is normally a transmembrane histidine kinase that detects a specific environmental stimulus and auto-phosphorylates a conserved histidine residue within its transmitter domain. The phosphoryl group is subsequently transferred from the histidine residue to an aspartic acid on the cognate regulator, which then activates the expression of genes required for countering the environmental stress. In addition to this basic "His → Asp" type of phosphotransfer mechanism, more complex multistep phosphorelay 2CSs also exist, in which the sensor harbors two extra domains as follows: a receiver domain containing a phosphor-accepting Asp and a Hpt domain. The most well characterized examples of the complex type 2CS are the anaerobic regulator ArcAB of *Escherichia coli* (2) and virulence-associated regulator BvgAS of *Bordetella* spp. (3). Both are capable of performing a multistep His → Asp → His → Asp phosphorelay.

An intermediate group of sensors are known as the hybrid sensors. The hybrid-type sensors, which contain a kinase and a receiver domain but lack an Hpt domain, are believed to require another protein to provide the Hpt domain for their signal transduction (Fig. 1) (4, 5). An example of such a system has been demonstrated in *E. coli*, in which the hybrid sensor RcsC is dependent on YojN, an Hpt domain-containing protein, to signal and activate the response regulator RcsB (5, 6). Another example was found recently in yeast, in which YPD1, also an Hpt module protein, could transfer phosphoryl groups from sensor kinase SLN1 to two downstream response regulatory proteins, SSK1 and SKN7 (7–9). These findings suggest that Hpt module-containing proteins function as intermediate transducers in multistep phosphorelay reactions (5, 10, 11).

*Pseudomonas aeruginosa* is a Gram-negative pathogen causing many acute and chronic infections, particularly in hospitalized individuals. The bacterium is responsible for the majority of morbidity and mortality in patients afflicted with cystic fibrosis (11, 12). The bacterium is also ubiquitous in the environment and is well known for its multidrug resistance. The strong capability of the bacterium to adapt to different environments might be partly explained by the presence of more than 60 sets of 2CSs (13, 14). A total of 12 hybrid-type kinases and 3 putative Hpt-module proteins have been annotated among the large

* This work was supported by the National Research Program for Genomic Medicine and National Science Council of Taiwan, Republic of China. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
${}^{S}$ The on-line version of this article (available at http://www.jbc.org) contains supplemental Table S1 and Figs. S1–S3.
${}^{1}$ To whom correspondence should be addressed: 101 Guang Fu Rd. 2nd Sec., Institute of Molecular Medicine, National Tsing Hua University, Hsin Chu, 300, Taiwan, Republic of China. Tel.: 886-3-5742910; Fax: 886-3-5742910; E-mail: hychang@life.nthu.edu.tw.
${}^{2}$ The abbreviations used are: 2CSs, two-component regulatory systems; α-RNAP, α-subunit of RNA polymerase; DIFMUP, 6,8-difluoro-4-methylumbelliferyl phosphate; MALDI-TOF, matrix-assisted laser desorption ioniza-

tion time-of-flight; PP2C, protein phosphatase 2C; RT, reverse transcription; X-gal, 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside.

HptB-mediated Phosphorelay in P. aeruginosa

TABLE 1  
Bacterial strains and plasmids used in this study

| Strains or plasmids | Descriptions                                                                                   | Refs. or source |
|---------------------|-----------------------------------------------------------------------------------------------|-----------------|
| Strains             |                                                                                               |                 |
| E. coli             |                                                                                               |                 |
| NovaBlue (DE3)      | endA1 hsdR17 supE44 thi-1 recA1 gyrA96 relA1 lac [F' proAB lacIq ZΔM15 Tn10]                      | Novagen         |
| XL-1 Blue           | MRF' K, Δ(mcrA) 183Δ[(mcrCB-hsdSMR-mrr) 173endA1supE44thi-1recA1gyrA96relA1 lac (F' proAB lacIq ZΔM15Tn5(Km<sup>r</sup>))] | Novagen         |
| P. aeruginosa       |                                                                                               |                 |
| PAO1                | Nonmucoid wild type strain                                                                   | Laboratory stock |
| MPA45               | PAO1ΔhptB                                                                                    | 16              |
| MJL46               | PAO1ΔPA3346                                                                                  | This study      |
| MJL47               | PAO1ΔPA3347                                                                                  | This study      |
| Plasmids            |                                                                                               |                 |
| pET30a-b            | His tag protein expression vector, Km<sup>r</sup>                                             | Novagen         |
| pET100              | His tag protein expression vector, Ap<sup>r</sup>                                              | Invitrogen      |
| pMMB66              | Ap<sup>r</sup>; broad-host-range expression vector                                             | 42              |
| pEX18Tc             | Tc<sup>r</sup>; oriT<sup>+</sup>sacB<sup>+</sup>, gene replacement vector                        | 43              |
| pGEX-5X-1           | GST tag protein expression vector, Ap<sup>r</sup>                                              | Amersham Biosciences |
| pBT                 | Cm<sup>r</sup>, p15A origin of replication, lac-UV5, λcl open reading frame                       | Stratagene      |
| pTRG                | Tc<sup>r</sup>, ColE1 origin of replication, lac-UV5 promoter, RNAPα open reading frame          | Stratagene      |
| pEHptA              | Km<sup>r</sup>; a hptA containing fragment cloned into pET30a                                   | 16              |
| pEHptB              | Km<sup>r</sup>; a hptB containing fragment cloned into pET30b                                   | 16              |
| pEHptC              | Km<sup>r</sup>; a hptC containing fragment cloned into pET30b                                   | 16              |
| pE16HD              | Km<sup>r</sup>; a PA1611HD containing fragment cloned into pET30a                               | 16              |
| pE1243              | Ap<sup>r</sup>, a fragment containing residues 456–859 of PA1243 coding region cloned into pET100 | This study      |
| pE1396              | Ap<sup>r</sup>, a fragment containing residues 172–541 of PA1396 coding region cloned into pET100 | This study      |
| pE1976              | Ap<sup>r</sup>, a fragment containing residues 460–882 of PA1976 coding region cloned into pET100 | This study      |
| pE1992              | Ap<sup>r</sup>, a fragment containing residues 160–565 of PA1992 coding region cloned into pET100 | This study      |
| pE2177              | Ap<sup>r</sup>, a fragment containing residues 310–700 of PA2177 coding region cloned into pET100 | This study      |
| pE2583              | Ap<sup>r</sup>, a fragment containing residues 589–993 of PA2583 coding region cloned into pET100 | This study      |
| pE2824              | Ap<sup>r</sup>, a fragment containing residues 248–787 of PA2824 coding region cloned into pET100 | This study      |
| pE3271              | Ap<sup>r</sup>, a fragment containing residues 749–1160 of PA3271 coding region cloned into pET100 | This study      |
| pE3462              | Ap<sup>r</sup>, a fragment containing residues 383–920 of PA3462 coding region cloned into pET100 | This study      |
| pE3974              | Ap<sup>r</sup>, a fragment containing residues 386–796 of PA3974 coding region cloned into pET100 | This study      |
| pE4856              | Ap<sup>r</sup>, a fragment containing residues 390–943 of RetS coding region cloned into pET100   | This study      |
| pE46R               | Km<sup>r</sup>; a fragment containing the receiver domain of regulator PA3346 from residue 1 to 161 cloned into pET30a | 16              |
| pE34                | Km<sup>r</sup>; the fragment containing the receiver domain of regulator PA0034 from residues 1 to 115 cloned into pET30a | This study      |
| pE1397              | Ap<sup>r</sup>, a fragment containing entire PA1397 coding region cloned into pET100             | This study      |
| pE2798              | Ap<sup>r</sup>, a fragment containing entire PA2798 coding region cloned into pET100             | This study      |
| pE3604              | Ap<sup>r</sup>, a fragment containing entire PA3604 coding region cloned into pET100             | This study      |
| pE3714              | Ap<sup>r</sup>, a fragment containing entire PA3714 coding region cloned into pET100             | This study      |
| pE4843              | Ap<sup>r</sup>, a fragment containing entire PA4843 coding region cloned into pET100             | This study      |
| pE5364              | Ap<sup>r</sup>, a fragment containing entire PA5364 coding region cloned into pET100             | This study      |
| pTRG-1611           | Ap<sup>r</sup>, a fragment containing residues 202–652 of PA1611 coding region cloned into pTRG   | This study      |
| pTRG-1976           | Ap<sup>r</sup>, a fragment containing residues 468–882 of PA1976 coding region cloned into pTRG   | This study      |
| pTRG-2824           | Ap<sup>r</sup>, a fragment containing residues 248–787 of PA2824 coding region cloned into pTRG   | This study      |
| pTRG-RetS           | Ap<sup>r</sup>, a fragment containing residues 392–859 of RetS coding region cloned into pTRG    | This study      |
| pBT-HptB            | Ap<sup>r</sup>, a fragment containing entire HptB coding region cloned into pBT                  | This study      |
| pMMB46              | Ap<sup>r</sup>, a fragment containing entire PA3346 coding region fused with a N-terminal His tag cloned into pMMB66 | This study      |
| pMMB47              | Ap<sup>r</sup>, a fragment containing entire PA3347 coding region fused with a N-terminal His tag cloned into pMMB66 | This study      |
| pBM2                | Ap<sup>r</sup>, a fragment containing entire hptB coding region cloned into pMMB66               | 16              |
| pGEX47              | Ap<sup>r</sup>, a fragment containing entire PA3347 coding region fused with a N-terminal GST tag cloned into pGEX-5X-1 | This study      |
| pE3346              | Ap<sup>r</sup>, a fragment containing entire PA3346 coding region cloned into pET100             | This study      |

number of 2CS proteins (Fig. 1). It is not clear, however, whether these hybrid sensors can indeed transmit the phosphoryl signal to the Hpt proteins. Furthermore, unlike the operon organization of most of the 2CS genes, the majority of the hybrid sensor genes does not link to a response regulator-encoding gene. Additionally, eight “orphan” response regulator genes, which do not link to any sensor gene, were also found in the *P. aeruginosa* genome (Fig. 1) (13–15). These orphan response regulators are therefore good candidates for receiving signals from the Hpt proteins.

We have demonstrated in a previous report that 1 of the 12 hybrid sensors, encoded by PA1611, is capable of transferring a phosphoryl group to 1 of the 3 Hpt proteins (16). We here extend the study to identify the multistep phosphorelay pathways among these sensor kinases, Hpt proteins, and orphan response regulators by performing *in vitro* phototransfer reactions systematically. In addition, we further evaluated the functional roles of the signaling pathway mediated by HptB. Our results reveal how these 2CS members are organized into complex signaling regulatory pathways, allowing *P. aeruginosa* to respond to a variety of different environmental signals flexibly and efficiently.

EXPERIMENTAL PROCEDURES

Bacterial Strains, Plasmids, and Growth Conditions—The bacterial strains and plasmids used in this study are listed in Table 1. All *P. aeruginosa* mutants used in this study were derived from wild type strain PAO1. Both *E. coli* and *P. aeruginosa* strains were propagated in Luria-Bertani (LB) broth or on LB agar at 37°C. The concentrations of antibiotics used for

selection of recombinant *E. coli* were 100 μg/ml ampicillin, 25 μg/ml kanamycin, and/or 5 μg/ml tetracycline, whereas 400 μg/ml carbenicillin and/or 50 μg/ml tetracycline were used for the selection of *P. aeruginosa* derivatives.

RT-PCR—Total RNA was extracted from mid-log phase *P. aeruginosa* PAO1 using the TRI reagent (Molecular Research Center, Inc., Cincinnati, OH). Contaminating DNA was eliminated from the RNA samples with RNase-free DNase (Promega). cDNA was reverse-transcribed from RNA using the SuperScript first-strand synthesis system (Invitrogen) following the manufacturer's recommendations. PCR was carried out at 95 °C for 1 min, 64 °C for 1 min, and 72 °C for 1 min per cycle for 30 cycles, with a final extension at 72 °C for 5 min, and the reaction products were analyzed on a 2% agarose gel. The primers used in this study are listed in the supplemental Table S1.

Cloning, Expression, and Purification of His-tagged Proteins—The recombinant DNA manipulations were carried out essentially as described (17). All restriction endonucleases and DNA-modifying enzymes were obtained from New England Biolabs and were used at the conditions suggested by the supplier. To generate recombinant proteins containing an N-terminal His<sub>6</sub> tag, the coding region for each of the desired proteins was first amplified by PCR, cloned into an expression vector (pET30a or pET100), and transformed into *E. coli* Novablue (DE3). The transformants were induced with 0.1 mM isopropyl β-D-thiogalactopyranoside, collected by centrifugation, resuspended in lysis buffer (20 mM Tris-HCl, pH 8.0, 1 mM phenylmethylsulfonyl fluoride, and 500 mM NaCl), and disrupted on ice by ultrasonication. After centrifugation at 24,000 × g for 20 min at 4 °C to remove debris, the clarified cell lysate was loaded onto a nickel-charged column (Amersham Biosciences) and washed, and the His<sub>6</sub>-tagged proteins were then eluted and analyzed by SDS-PAGE. The compositions of wash and elution buffers, as well as the chromatography procedure, were essentially as described by the manufacturer (Amersham Biosciences). The purified proteins were dialyzed against storage buffer (50 mM Tris-HCl, pH 8.0, 200 mM KCl, 10 mM MgCl<sub>2</sub>, 0.1 mM EDTA, 1 mM dithiothreitol, 50% glycerol), and their concentrations were determined by the Bradford method using a Bio-Rad kit.

In Vitro Phosphorelay Assays—The signal transduction between the sensors and the Hpt proteins and between the Hpt proteins and the orphan regulators was tested by in vitro phosphorelay assays. The composition of the phosphorelay reaction was as described previously (16) and contained 100 mM Tris-HCl, pH 8.0, 50 mM KCl, 5 mM MgCl<sub>2</sub>, 1 mM dithiothreitol, and 1 μCi of [γ-<sup>32</sup>P]ATP at a final concentration of 2.5 μM. For autophosphorylation of the sensor, 1 μM of recombinant sensor protein was used, and the reaction was performed at 25 °C for 30 min. Phosphorelay from sensors to different Hpt proteins was performed by adding 1 μM of purified HptA, HptB, or HptC protein to the autokinase reaction mixture. The phosphorylation of response regulators was performed by incubating the purified regulator protein of interest (1 μM) in the reaction buffer with sensor kinase PA2824 and HptB at 25 °C for 30 min to allow for autophosphorylation of the sensor protein, and later at 37 °C for 30 min to allow for phosphorelay to HptB, and then finally to the regulator. In the reverse phosphorelay assay, <sup>32</sup>P-phosphorylated HptB was first prepared by using PA1611

HptB-mediated Phosphorelay in *P. aeruginosa*

sensor kinase as stated above. The sensor protein with no detectable autokinase activity was then added individually to the mixtures with excess ATP (0.5 mM) and incubated at 25 °C for 15 min. All the phosphorelay reactions were terminated by adding an equal volume of SDS-PAGE loading buffer (50 mM Tris-Cl, pH 6.8, 100 mM dithiothreitol, 2% SDS, 0.1% bromphenol blue, 10% glycerol) and resolved by 15% SDS-PAGE, and the phosphorylation pattern was visualized by autoradiography. All phosphorelay assays were performed independently at least three times.

Bacterial Two-hybrid Assay—DNA fragments encoding C-terminal cytoplasmic regions of hybrid sensors (PA1611, PA1976, PA2824, and RetS) and full-length HptB were cloned, respectively, at the 3′ end of genes encoding λ-CI repressor protein domain carried on pTRG vector and α-subunit RNA polymerase (α-RNAP) domain on pBT vector as described by the manufacturer (Stratagene). The resulting gene fusion constructs, pTRG-1611, pTRG-1976, pTRG-2824, pTRG-RetS, and pBT-HptB, were confirmed by DNA sequencing. The positive controls used were pTRG-GAL11P and pBT-LGF2 (Stratagene). Derivatives of pTRG and pBT were co-transformed into *E. coli* XL1-Blue MRF′ Kan cells and selected on LB plates supplemented with 250 μg/ml carbenicillin, 25 μg/ml chloramphenicol, and 50 μg/ml kanamycin. Then single colonies were patched on X-gal indicator plates (LB-agar plates supplemented with 350 μg/ml carbenicillin, 25 μg/ml chloramphenicol, 15 μg/ml tetracycline, 50 μg/ml kanamycin, 50 μg/ml X-gal, 0.2 mM phenylethyl β-D-thiogalactoside, and 20 μM isopropyl 1-thio-β-D-galactopyranoside) for 17–24 h at 37 °C.

Phosphorylation and Dephosphorylation of PA3347 Protein in Vitro—The open reading frame of PA3347 was cloned into the expression vector pGEX-5X-1 and overexpressed as an N-terminal glutathione S-transferase (GST) fusion protein. The recombinant PA3347 protein was purified through a Glutathione-Superflow column under the conditions recommended by the manufacturer (Amersham Biosciences). The purified fusion protein was analyzed by SDS-PAGE and subsequently subjected to the phosphorylation assay. The *P. aeruginosa* cell lysate used for phosphorylation of the GST-PA3347 protein was prepared from exponentially grown bacteria by ultrasonication on ice followed by centrifugation at 24,000 × g for 20 min at 4 °C. Approximately 2.4 μM purified GST-PA3347 fusion protein was incubated with 10 mg of prepared lysate and 0.15 μCi of [γ-<sup>32</sup>P]ATP at 25 °C for 1 h, precipitated with Glutathione-Superflow beads, and subsequently analyzed by SDS-PAGE. Protein phosphatase activity of PA3346 was determined by incubation of the purified PA3346 protein with phosphorylated PA3347 at 37 °C for 30 min. When indicated, 10 units of calf intestinal alkaline phosphatase (New England Biolabs) or 50 mM EDTA was included in the reaction as a control. All reactions were terminated by adding an equal volume of SDS-PAGE loading buffer and analyzed by SDS-PAGE followed by autoradiography.

Construction of GST-PA3347 Site-specific Mutations—Site-directed mutagenesis was performed using the Stratagene QuikChange site-directed mutagenesis kit. Oligonucleotide primer sequences are provided in the supplemental material.

HptB-mediated Phosphorelay in P. aeruginosa

Hybrid Sensor Kinase                      Hpt Protein                     Response Regulator
PA1243                                    PA0034
PA1396                                    PA1397
PA1611                                    PA2798
PA1976                                    PA3346
PA1992                                    PA3604
PA2177                                    PA3714
PA2583                                    PA4781
PA2824                                    PA4843
PA3271                                    PA5364
PA3462
LadS (PA3974)
RetS (PA4856)

FIGURE 1. Schematic diagram showing the phosphorelay in 2CSs with a hybrid type sensor. The *P*. aeruginosa gene index number of the orphan sensors, Hpt proteins, and response regulators investigated in this study are listed below the diagram. The underlines indicate the only previously known coupled sensor kinase and response regulator genes.

In Vitro Phosphatase Activity Assays of PA3346—To detect phosphatase activity of phosphorylated PA3346, 5 μg of PA3346 purified from *E. coli* was preincubated with 1 μg of sensor (PA2824) and 1 μg of HptB in 50 μl of buffer containing 200 mM Tris-HCl, pH 8.0, 100 mM KCl, 10 mM MgCl₂, 1 mM dithiothreitol, and 2.5 μM ATP for 10 min at 25°C and then 5 min at 37°C. The fluorogenic compound 6,8-difluoro-4-methylumbelliferyl phosphate (DIFMUP) and its dephosphorylated standard 6,8-difluoro-4-methylumbelliferyl were purchased from Molecular Probes. For kinetic analysis, the DIFMUP stock solution was prediluted in reaction buffer and added to the reaction mixture to initiate the reaction at 25°C. Fluorescent excitation of hydrolyzed DIFMUP and the standard DIFMU were measured at 355 nm, and emission was detected at 460 nm in a fluorescence plate reader (VITOR³, PerkinElmer Life Sciences).

Construction of Isogenic Mutants—The allelic exchange strategy was used to generate gene-specific mutants in *P*. aeruginosa. DNA fragments ~1 kb in size flanking both sides of PA3346 and PA3347 were PCR-amplified by specific primer pairs (supplemental Table S1) and cloned into the suicide vec-

A
+ + + +
PA1611
+ + - -
HptA
- - + -
HptB
- - - +
HptC

B
+ + + +
PA1976
+ + - -
HptA
- + + -
HptB
- - + -
HptC

C
+ + + +
PA2824
+ + - -
HptA
- - + -
HptB
- - - +
HptC

D
+ + + +
PA2177
+ + - -
HptA
- + + -
HptB
- - + -
HptC

E
+ + + +
PA2583
+ + - -
HptA
- - + -
HptB
- - - +
HptC

FIGURE 2. Transduction of phosphoryl signals from orphan sensors to Hpt proteins. Sensor proteins were individually incubated with [γ-³²P]ATP either in the absence (-) or presence (+) of an Hpt protein, resolved by SDS-PAGE, and Coomassie Blue staining (right figure). The phosphorylation profiles of sensors and target Hpt protein were detected by autoradiography (right figure). The sensor kinases used were PA1611 (A), PA1976 (B), PA2824 (C), PA2177 (D), and PA2583 (E). The protein bands of the expected size for the sensor kinase tested and HptB protein are indicated by arrowheads.

9936 JOURNAL OF BIOLOGICAL CHEMISTRY
VOLUME 283 • NUMBER 15 • APRIL 11, 2008

HptB-mediated Phosphorelay in P. aeruginosa

A PA1611 + HptB
B PA1611 + HptB

FIGURE 3. Reverse phosphorylation of sensor proteins with phosphorylated HptB. The sensor kinase PA1611 was incubated with HptB and with a sensor protein unable to perform autophosphorylation. The proteins were resolved by SDS-PAGE and subjected to autoradiography (A) and Coomassie Blue staining (B). The phosphorylated sensors and HptB are marked by filled arrowheads, whereas the position of phosphorylated RetS is indicated by empty arrowheads.

FIGURE 4. Bacterial two-hybrid analyses of interactions between hybrid sensors and HptB. E. coli XL1-Blue cells carrying various combinations of pTRG and pBT-derived plasmids of the two-hybrid systems were propagated on indicator plates. The plasmids harbored by the testing bacterial strains are indicated below the plates. \( T^+ \), pTRG-GAL11p and pBT-LGF2 (positive control); \( T^- \), pTRG and pBT vectors (negative control).

tor pEX18Tc. The generated plasmids were mobilized from E. coli SM10 into P. aeruginosa PAO1 through conjugation. The tetracycline-resistant transconjugants were selected positively with 5% sucrose for loss of the vector. PA3346 and PA3347 deletion mutants were verified by PCR and Southern blotting analysis (supplemental Fig. S1). Complementation of the gene defects was performed by cloning DNA fragments comprising either the PA3346 or the PA3347 open reading frame with an N-terminal His tag sequence into the shuttle vector pMMB66 to result in pMMB46 and pMMB47, which were then mobilized into the mutant bacteria by conjugation.

Bacteria Motility Assays—The plates for the swarming assay contained M8 medium base supplemented with 0.5% Bacto-Agar, 0.02% glucose, and 0.5% casamino acids (18). The bacterial strains to be tested were transferred from an overnight culture plate using a sterile toothpick and incubation was done at \( 37^\circ \mathrm{C} \) for the swarming assay. The assays were performed independently at least three times.

Biofilm Formation Assay—Biofilm formation was assayed in 96-well polystyrene microtiter plates. Bacterial strains to be tested were grown overnight at \( 37^\circ \mathrm{C} \) in LB broth and diluted 1:50 in fresh media into the 96-well plates the next morning. The plates were incubated at \( 37^\circ \mathrm{C} \) for different times without shaking. The extent of biofilm formation was determined by staining with 1% crystal violet as described previously (16) and expressed as the absorbance at 595 nm.

RESULTS

Cloning, Expression, and Purification of P. aeruginosa Hybrid Sensors—As the first step toward elucidating the signaling pathways exerted by the 12 hybrid-type sensors, we cloned all the genes encoding these sensors (listed in Fig. 1), overexpressed them with a terminal His₆ fusion in E. coli, and purified the gene products. To eliminate potential insolubility problems, the N-terminal transmembrane domains of the sensors were removed. The remaining C-terminal cytoplasmic regions containing the His-containing transmitter and the Asp-containing receiver domain are predicted to possess auto-phosphorylation activity, as well as the ability to transfer the phosphoryl group to the corresponding response regulators (19). All the sensor proteins, with the exception of PA1992, which did not yield sufficient quantities for subsequent study, could be purified to apparent homogeneity in soluble form by a single step affinity chromatography (data not shown).

Sensor Proteins PA1611, PA1976, and PA2824 Phosphorylate HptB—To identify the relationships between the three Hpt proteins and the hybrid-type sensor histidine kinases, the phosphotransfer profiling assay (20) was employed. In the assay, the purified sensor proteins were incubated with [γ-³²P]ATP to assess their autophosphorylation activity, and five of the tested sensors (PA1611, PA1976, PA2177, PA2583, and PA2824) showed varying degrees of autophosphorylation (Fig. 2, A–E, 1st lane). The phosphorylated sensor proteins were incubated with an Hpt protein, and the transfer of the phosphoryl group was examined by autoradiography (Fig. 2, A–E). The results showed that sensors PA1611, PA1976, and PA2824 could transfer a phosphoryl group specifically to HptB but not to either HptA or HptC (Fig. 2, A–C). Both sensors PA2177 and PA2583 were unable to phosphorylate any of the Hpt proteins in vitro despite their relatively high auto-kinase activity (Fig. 2, D and E). Sensors were purified and tested in three independent assays.

RetS (PA4856) Also Interacts with HptB—To investigate whether any of the sensor proteins incapable of autophosphorylation could interact with HptB, a reverse phosphorylation

HptB-mediated Phosphorelay in P. aeruginosa

A  
+ - + + PA2824  
+ - - + HptB  
- + + + PA3346  

B  

FIGURE 5. Identification of orphan response regulators targeted by HptB. A, reconstitution of PA2824-HptB-PA3346 phosphorelay *in vitro*. The reactions were analyzed by SDS-PAGE and subjected to autoradiography. The *plus* and *minus* signs indicate the presence or absence of PA2824, HptB, and PA3346 in the *in vitro* phosphorelay assay. B, capability of orphan response regulators for receiving phosphoryl signals from HptB. Phosphorylated HptB was used as the phosphoryl group donor in the *in vitro* phosphorelay assay, and the proteins were resolved by SDS-PAGE and subjected to autoradiography. The *P. aeruginosa* gene name or index numbers are indicated for each orphan response regulator. The sensor protein used to initiate phosphorelay was PA2824.

A  
PA3348 → PA3347 → PA3346 → PA3345  
H1     H2     H3  
100 bp  

B  
M 1 2 3 4 5 6 7 8 9  

FIGURE 6. PA3347, PA3346, and *hptB* are organized as an operon. A, gene map of the *hptB* gene cluster. The thick arrows denote open reading frames. Lines between closed circles below the arrows indicate the intergenic regions amplified by PCR with the indicated primer pairs. B, RT-PCR analysis of the intergenic regions in the *hptB* gene cluster. PCR products were resolved on a 2% agarose gel. The amplified products from primer sets H1, H2, and H3 are shown in lanes 1–3 (695 bp), lanes 4–6 (146 bp), and lanes 7–9 (344 bp), respectively. Lanes 1, 4, and 7 represent the PCR products from genomic DNA template as positive controls. Lanes 2, 5, and 8 represent the PCR products using cellular RNA without reverse transcription as negative controls. Lanes 3, 6, and 9 show the PCR products amplified from randomly primed cDNA of *P. aeruginosa*. The sizes of the DNA markers are indicated at left. The arrowheads indicate the PCR products of the predicted size.

experiment was performed as described (21, 22). Phosphorylated HptB, prepared with [γ-³²P]ATP and sensor PA1611, was used in this study as the phosphoryl group donor. In five independent assays, the appearance of a phosphorylated RetS band accompanied with a significant reduction of isotope signal of HptB (Fig. 3) could be detected indicating that, besides PA1611, PA1976, and PA2824, RetS is another sensor that participates in HptB-mediated phosphorelay.

PA1611, PA1976, PA2824, and RetS Interact with HptB in Vivo—To further verify whether the interaction between these sensors, PA1611, PA1976, PA2824, RetS, and HptB indeed occurs in vivo, a bacterial two-hybrid assay was performed. The cytoplasmic domain of the sensors was constructed onto the target vector pTRG to produce a recombinant protein fused to the λ-CI domain. The full-length HptB was cloned at the 3′ end of the gene encoding the α-RNAP domain on the bait vector pBT to produce an α-RNAP-HptB fusion protein. Interaction between the sensor and HptB fusion proteins would allow the λ-CI to bind to the operator region and recruit α-RNAP to initiate transcription of the *ampR* and *lacZ* reporter genes. Fig. 4 shows the result of such an analysis, where the strains carrying both plasmids were grown on X-gal indicator plates supplemented with carbenicillin. All tested hybrid sensors (PA1611, PA1976, PA2824, and RetS) were able to interact with HptB as reflected by the significant growth of the bacterial strains. No interaction was detected either between the sensor fusion proteins and λ-CI without fused HptB or between HptB fusion and α-RNAP. Together, these results strongly suggest that sensors PA1611, PA1976, PA2824, and RetS interact with HptB via their cytoplasmic domains and are able to transfer the phosphoryl group to HptB.

PA3346 Is the Only Orphan Response Regulator Phosphorylated by HptB—We have previously shown that HptB (PA3345) could relay the phosphoryl signal to a response regulator (PA3346) (16). In addition to PA3346, there are seven response regulator-encoding genes in the *P. aeruginosa* PAO1 genome that do not physically link to a sensor gene and are considered “orphans” (Fig. 1). Although the response regulator PA1397 has not been classified as an orphan, it was of particular interest to this study because it resides next to the gene encoding hybrid sensor PA1396 and may be a target of HptB. To explore whether any of these response regulators could serve as a downstream target of HptB, these response regulators were first synthesized in *E. coli* using standard recombinant DNA techniques and used in the phosphorelay assay. With the only exception of PA4781, the other seven response regulators (PA0034, PA2798, PA3604, PA3714, PA4843, PA5364, and PA1397) could be successfully overexpressed and purified to homogeneity in soluble form using nickel chelate affinity chromatography (supplemental Fig. S2). The phosphorylation pattern of the seven response regulators was then determined in three independent assays by co-incubation with PA2824 sensor kinase and HptB. As shown in Fig. 5A, PA3346 could efficiently receive the phosphoryl signal relayed from PA2824 to HptB. All other tested response regulators were unable to receive the phosphoryl group from HptB (Fig. 5B).

PA3347, PA3346, and *hptB* Are Organized as an Operon—Bioinformatic analysis of the *P. aeruginosa* PAO1 genome has revealed that *hptB* (PA3345), PA3346, and PA3347 are closely clustered together with an intergenic distance of 66 and 1 bp, respectively. In addition, all three genes are transcribed in the same direction starting at PA3347 and proceeding through *hptB*, suggesting that they are organized as an operon and

HptB-mediated Phosphorelay in P. aeruginosa

PA3346, and *hptB*, but not PA3348, are indeed organized as an operon.

**PA3347 Is Phosphorylated Both in Vivo and in Vitro—The response regulator PA3346 encodes a protein of 571 amino acids with two notable features: an N-terminal phosphoryl signal receiver domain and a protein phosphatase 2C (PP2C)-like domain. The PP2C domain shares 21% sequence similarity with the phosphatase domain of RsbU in *Bacillus subtilis* (Fig. 7A) (23). PA3346 has two conserved amino acid clusters, which are thought to be required for the function of both prokaryotic and eukaryotic PP2C members, such as RsbU, SpoIIE, and RsbX in *B. subtilis* and TPD1 and PTC1 in yeast (24) (Fig. 7A). RsbU is an important stress regulator and is known to exert its function through modulation of the phosphorylation of RsbV, a σ factor antagonist (23, 25). Interestingly, PA3347 also contains a σ factor antagonist domain (Fig. 7B). The alignment of the amino acid sequences of PA3347 and other σ factor antagonists, RsbV, RsbS, and SpoIIAA, has revealed an overall 16% sequence identity among these proteins. A serine residue known to be essential for the regulation of RsbV and SpoIIA activity (26, 27) was also found to be conserved in PA3347 at amino acid position 56 (Fig. 7B). In analogy to the relationship between SpoIIAA and SpoIIAB and between RsbU and RsbV, we predict that PA3346 may regulate PA3347 through dephosphorylation.**

To investigate whether PA3347 is indeed phosphorylated in vivo, recombinant His<sub>6</sub>-PA3347 was synthesized in *P. aeruginosa* PAO1 and purified by using a nickel-charged column. The eluted sample was further resolved by SDS-PAGE, and the protein band was excised and subjected to proteolytic digestion with trypsin, followed by matrix-assisted laser desorption ionization time-of-flight (MALDI-TOF) mass spectrometry analysis. Analysis of the mass spectrometry spectrum revealed two major peaks of 1850.994 and 1930.963 Da in mass, which are consistent with those predicted for the unphosphorylated and phosphorylated Ser-56-containing peptides (NATYLDSSALGMLLLR), 1849.997 and 1929.963 Da, respectively (Fig. 8A). The phospho-

FIGURE 7. Multiple sequence alignment of *P. aeruginosa* PA3346 and PA3347. A, sequence of the PA3346 phosphatase domain compared with other PP2C proteins in *B. subtilis* using RPSBLAST. PP2C domains are denoted by black boxes. Two conserved phosphatase motifs are denoted by Box 1 and Box 2. B, sequence alignment of *P. aeruginosa* PA3347 with various anti-σ antagonists of *B. subtilis*. Conserved residues are shaded in black and identical residues are shaded in gray. The conserved Ser-56 residue subject to phosphorylation regulation is marked by an asterisk. The GenBank™ accession numbers for each of the protein sequences are as follows: SpoIIIE, U26835; RsbU, L35574; RsbX, M34995; SpoIIAA, AAA22789.1; RsbV, L35574; RsbS, L35574.

responsible for the same functional task. To determine whether these genes are indeed transcribed in the same unit, an RT-PCR experiment was performed. PCR products of 146 and 344 bp in length, comprising the intergenic region of *hptB*-PA3346 and of PA3346–PA3347, respectively, can be clearly observed in Fig. 6. The same primer sets were unable to generate any PCR product with the same RNA template without reverse transcription. Similarly, no RT-PCR product was found when the primer pair designed to amplify the intergenic region between PA3347 and PA3348 was used. Our results thus confirmed that PA3347,

HptB-mediated Phosphorelay in P. aeruginosa

A

B
1 2 3 4
WT WT S56A S57A

C
PA3346 (μg)
0 5 10 20 CIP EDTA

D

FIGURE 8. PA3347 is subject to phosphorylation and can be dephosphorylated by PA3346. A, MALDI-TOF mass spectrum of tryptic fragments of PA3347 purified from *P. aeruginosa*. SP denotes the Ser-56-containing peptide, NATYLDSSALGMLLLR. Modifications of the peptide are shown as M, oxidation; Y, phosphorylation. The observed *m/z* values and the peak assignments are also indicated. Calculated monoisotopic masses of peptide with or without modifications were as follows: SP, 1849.9971; SP+M, 1865.9920; SP+Y, 1929.9634; SP+M+Y, 1945.9584. B, in vitro phosphorylation of purified PA3347. Purified wild type (WT), S56A, or S57A GST-PA3347 protein was incubated for 1 h with [γ-³²P]ATP in the absence (lane 1) or presence (lanes 2–4) of total *P. aeruginosa* cell lysate. GST-PA3347 was pulled down with Glutathione-Superflow beads and analyzed by SDS-PAGE (right), followed by autoradiography (left). C, dephosphorylation of GST-PA3347 by purified PA3346. GST-PA3347 was phosphorylated with [γ-³²P]ATP by *P. aeruginosa* cell extract, precipitated with Glutathione-Superflow beads, and incubated with 0, 5, 10, or 20 μg of purified PA3346, 50 mM EDTA, or calf intestinal alkaline phosphatase (CIP), respectively. Top, autoradiogram; bottom, Coomassie Blue-stained SDS-polyacrylamide gel. D, extent of dephosphorylation relative to the untreated sample was measured using the image analysis software TotalLab version 1.00 and plotted.

rylated Ser-56-containing peptide peak, however, was not observed in PA3347 synthesized in *E. coli* (supplemental Fig. S3).

More direct evidence indicating that PA3347 is phosphorylated in *P. aeruginosa* came from three independent in vitro phosphorylation assays. As the identity of the kinase responsible for PA3347 phosphorylation remains elusive, whole *P. aeruginosa* cell extract, which should provide the required kinase activity for PA3347 phosphorylation, was used in this assay with [γ-³²P]ATP and GST-PA3347 purified from *E. coli*. Phosphorylation of GST-PA3347 could be clearly observed after incubation with the cell lysate (Fig. 8B, lane 2). No auto-phosphorylation activity was observed for PA3347 (Fig. 8B, lane 1).

Ser-56 Is the Only Phosphorylation Residue of PA3347—Because mass spectrometry analysis indicated that Ser-56-containing peptides were phosphorylated, the two serine residues in the peptide, Ser-56 and Ser-57, were individually substituted with alanine in PA3347. In the phosphorylation experiment, no radioactive signal was detected on PA3347-S56A, whereas phosphorylation of PA3347-S57A could be clearly observed (Fig. 8B) that was even stronger than the wild type protein. These results suggest that the conserved Ser-56 residue is subject to phosphorylation regulation.

PA3346 Encodes a Novel Ser/Thr Phosphatase for PA3347 Protein—The next question was whether PA3346 could serve as a functional phosphatase for PA3347. The recombinant GST-PA3347 was first phosphorylated using [γ-³²P]ATP, and whole *P. aeruginosa* cell lysate, purified on glutathione-Sepharose, was then used as the substrate for testing phosphatase activity of recombinant full-length PA3346. As shown in Fig. 8, C and D, the degree of phosphorylation of PA3347 showed a considerable inverse correlation with the input amounts of PA3346, indicating that PA3347 is indeed a target of PA3346 phosphatase activity. Because PA3347 is phosphorylated only on

Ser-56, the result is consistent with the bioinformatics finding that PA3346 is a Ser/Thr phosphatase. The phosphatase activity of PA3346 was partially inhibited by the presence of EDTA (Fig. 8, C and D), a known inhibitor of protein phosphatases 2B and 2C (28, 29). The results suggest that a divalent cation is required in the reaction catalyzed by PA3346. Calf intestine alkaline phosphatase was unable to dephosphorylate PA3347 (Fig. 8, C and D).

**Phosphorylation of PA3346 Enhances Its Phosphatase Activity—** Because PA3346 is a response regulator with a functional phosphatase domain, we test whether phosphorylation of PA3346 in its receiver domain would affect its phosphatase activity. The phosphatase activity of PA3346 was determined

![Figure 9](#fig9)

**FIGURE 9.** Activation of PA3346 phosphatase activity by phosphorylation. Phosphatase activity was monitored by measuring fluorescence at indicated times in a fluorescence microplate reader. ▲, PA3346 directly purified from *E. coli*; ●, PA3346 previously phosphorylated by 1 μg of PA2824 and 1 μg of HptB; ○, 1 μg of PA2824; □, 1 μg of HptB; △, blank.

using the synthetic substrate DiFMUP, which has been used for the measurement of the activity of several Ser/Thr phosphatases (30, 31). The \(K_m\) and \(V_{\text{max}}\) values of PA3346 in utilizing DiFMUP as the substrate were found to be 176.33 ± 14.61 μM and 3.18 ± 0.31 μM min⁻¹ mg⁻¹, respectively. PA3346 purified from *E. coli* exhibited a weak phosphatase activity, but the activity increased by 3-fold when the protein was subject to phosphorylation by incubating with ATP, sensor PA2824 and HptB (Fig. 9). The result therefore indicates that phosphorylated PA3346 is the active form of the enzyme.

**HptB-mediated Signaling Pathway Controls Swarming Activity in P. aeruginosa PAO1—** We have previously shown that HptB is involved in several biological properties of *P. aeruginosa*, including swarming activity and biofilm formation (16). To further elucidate the functional roles of PA3346 and PA3347, deletion mutants at these two loci were generated, and their behaviors were evaluated. Analysis of swarming ability revealed that the PA3346 and PA3347 mutant, designated MJL46 and MJL47, respectively, significantly exceeded the wild type strain, whereas swarming in MPA45 was severely defective (Fig. 10A). The swarming phenotypes of MJL46, MJL47, and MPA45 could be restored by complementation with a plasmid expressing a functional PA3346, PA3347, and *hptB* gene, respectively (Fig. 10A). In the complementation with a plasmid-borne PA3347 gene, the swarming ability of MJL47 was even decreased.

**PA3346 Affects Biofilm Formation in P. aeruginosa—** In addition to motility, the kinetics of biofilm formation in these mutants was also examined (Fig. 10B). The biofilm-forming activity of MJL47 was indistinguishable from that of PAO1 with the amount of total biofilm formed reaching its highest point at ~8 h. On the other hand, it took 10 h for MJL46 to accumulate biofilm to the highest level. Unlike MJL46, the *hptB* mutant MPA45 synthesized and disintegrated biofilm at a faster rate than the wild type strain (Fig. 10B). The biofilm formation in MJL46 [pMMB46] was the same as PAO1. However, the complementary strains MPA45 [pBM2] and MJL47 [pMMB47] exhibited a lower biofilm-forming activity than that of the wild type strain (Fig. 10B). The growth rates of these mutant strains in LB broth were essentially the same (data not shown) and thus apparently did not contribute to the difference in the biofilm forming activity.

![Figure 10](#fig10)

**FIGURE 10.** Phenotypic characterization of mutants defective in *hptB*, PA3346, or PA3347. A, motility of *P. aeruginosa* PAO1, MPA45, MJL46, MJL47, and the complemented strains on a swarming plate. B, biofilm formation was evaluated by crystal violet staining and expressed as OD₅₉₅. The average zone expansion from three independent experiments is displayed. *, statistically significant difference from the wild type (*p* < 0.005).

**DISCUSSION**

Based on the analysis of phosphorelay among 12 hybrid sensors, 3 Hpt proteins and 9 orphan response regulators, we have established a novel signaling network in *P. aeruginosa* PAO1. Unlike conventional 2CS in which signal trans-

HptB-mediated Phosphorelay in P. aeruginosa

Although all recombinant sensors in this study contain a seemingly similar domain organization, it is not clear why only five of the sensor proteins exhibited autokinase activity. One possible explanation is that the receiver domain in some hybrid sensors is auto-inhibitory for the kinase activity, as demonstrated previously in *Agrobacterium tumefaciens* VirA (32). Nevertheless, by taking advantage of the reversibility of phosphotransfer between HptB and sensor kinases, an additional sensor protein that interacts with HptB was identified. This finding also implies that Hpt proteins may relay signals between sensor kinases to initiate new signaling processes. The extent of the reverse phosphorylation *in vivo* is not clear, but it is likely to be low because other factors, such as spatial and temporal patterns of protein synthesis, may restrict the chance of interaction.

The importance of the eukaryotic type Ser/Thr kinases and phosphatases in prokaryotic signal transduction has become an interesting issue since their first discovery in *Myxococcus xanthus* (33). Bacterial homologues of protein phosphatases have been described to be necessary for cellular functions such as growth, differentiation, and virulence (28, 29), although information concerning their endogenous substrates and activating signals has been limited, particularly in Gram-negative bacteria. Proteins with a PP2C domain are known to play an important role in response to environmental stresses or energy starvation in Gram-positive bacteria (23, 26). According to the current model of σ<sup>B</sup> regulation in *B. subtilis*, a PP2C phosphatase triggers the dephosphorylation of an anti-σ antagonist, which in turn binds to an anti-σ factor to result in the release of σ<sup>B</sup>, and eventually leads to transcription initiation of the target regulon (25, 34). We hypothesize that a similar model could be applied to the HptB-PA3346-PA3347 signaling system, in which PA3347 phosphorylation, controlled by PA3346 phosphatase activity and a yet to be identified protein kinase, affects the anti-σ binding activity of PA3346 and consequently modulates the expression of downstream target genes. The protein phosphatase identified in *P. aeruginosa* so far are Stp1 and PppA (35, 36), which share weak homology with PA3346. In addition, no investigation of anti-σ antagonists in Gram-negative bacteria has been reported (34, 37). Therefore, this work creates a new direction for the study of this interesting type of regulatory system in Gram-negative bacteria.

Functional studies of the isogenic mutants relating to the HptB-mediated signal transduction indicate that this pathway is involved in regulation of swarming activity and biofilm formation. The mutants investigated in this study displayed quite distinct swarming phenotypes. The swarming

FIGURE 11. A hypothetical scheme for the HptB-mediated phosphorelay system. Upon activation by environmental stress, sensor kinases PA1611, PA1976, PA2824, and RetS autophosphorylate and transfer a phosphoryl group specifically to HptB, which in turn relays the signal to response regulator PA3346. The phosphorylation regulates the Ser/Thr phosphatase activity of PA3346, resulting in increasing its phosphatase activity and dephosphorylation of PA3347. PA3347 is previously phosphorylated at Ser-56 by an unknown kinase, and the phosphorylation may modulate the binding activity of PA3347 by which it releases an anti-σ factor (an unknown factor X) and leads to the expression of genes associated with swarming activity and biofilm formation.

duction from a sensor to a paired response regulator is a common feature, the signaling network elucidated in this study consists of multiple hybrid sensors (PA1611, PA1976, PA2824, and RetS), a single Hpt protein (HptB), and a single response regulator (PA3346) (Fig. 11). Such a signal convergent network reflects how *P. aeruginosa* could counter a wide range of environmental stresses by evolving a large number of sensor proteins while maintaining a small set of mediator proteins (HptB) to perform the essential responses, such as increasing swarming to move away from an unfavorable environment or providing sufficient phenotypic plasticity to cause infections.

Trans-phosphorylation between noncognate 2CS pairs has been reported for organisms other than *P. aeruginosa*. For example, in *Caulobacter crescentus*, an Hpt module-containing protein was identified as a functional phosphotransferase and is known to be activated by three different hybrid sensors. This pathway finally activates the response regulator, TacA, which plays a key role in bacterial morphogenesis (21). Similar to *C. crescentus*, *P. aeruginosa* also utilizes an Hpt protein as the mediator to integrate signals transmitted from multiple sensors prior to relaying to a single response regulator, PA3346, whereby this pathway appears to have substantial influence over reactions to different environmental stimuli. Both of the systems support the idea that cross-talk in bacterial signal transduction takes place. It is of interest to note that, in the absence of HptB, a slight phosphorylation could be detected in PA3346 co-incubated with PA2824 (Fig. 5A) suggesting certain some hybrid kinase may also interact with regulators independent of HptB. Nevertheless, the impact of such cross-talk is likely to be small, because of the relatively weak reaction compared with that mediated by HptB.

motility is totally abolished in the *hptB* mutant, although the activity is enhanced in the PA3346 and PA3347 mutants. The different phenotypes exhibited by the mutants strongly imply that additional regulatory components also participate in this pathway. In our hypothesis, unphosphorylated PA3347 could bind and regulate the function of an unknown anti-σ factor that regulates the swarming phenotype. Because in the gene knock-out mutant MJL47 the anti-σ factor is able to activate gene expression without inhibitory control, the mutant performs the same phenotype as the phosphatase mutant, MJL46, increasing the swarming activity. However, in our model, we cannot explain the decrease of swarming activity in the *hptB* mutant. HptB may be capable of regulating a response regulator, which is not included in this study and remains to be discovered.

Until now, the functions of only a few hybrid sensors in *P. aeruginosa* have been identified. One example is PA4856, which encodes RetS (RtsM) (38, 39). RetS has been shown to be a crucial sensor required for controlling pleiotropic phenotypes such as the expression of the type III secretion system (40) and the inhibition of biofilm development and exopolysaccharide production (38, 39). Similarly, sensor PA2824 was also found to be a negative regulator of biofilm formation (38). Although previous studies have shown RetS signals through the GacS/GacA/RsmZ global regulatory pathway, the missing link between sensor RetS and its direct regulatory factor has not been found. Our data indicate that PA2824 and RetS are capable of phosphorylating HptB, suggesting that these two sensors exert their function through the HptB-PA3346-PA3347 signaling pathway. On the other hand, the hybrid sensor LadS (PA3974) is known to play a counter-regulatory role against RetS (38, 41). LadS was unable to phosphorylate HptB, suggesting that it carries out the regulation through a pathway distinct from HptB-mediated signal transduction.

This study has established a significantly expanded picture of the HptB-mediated signaling pathway, which controls swarming activity and biofilm development in *P. aeruginosa* (Fig. 11). Nevertheless, our understanding of the phosphorelay system is far from complete. First, several key elements in the pathways remain to be identified and characterized, including the environmental signals detected by the HptB-regulating sensors, the kinase responsible for phosphorylation of PA3347, the anti-σ factor that interacts with PA3347, and the downstream target genes regulated by the pathway. Second, the mechanism by which HptB integrates multiple signals and relays an adequate signal to downstream effectors will also need to be determined. Finally, a detailed molecular mechanism on how phosphorylation would affect the protein activity requires further investigation.

---

**REFERENCES**

1. Bijlsma, J. J., and Groisman, E. A. (2003) *Trends Microbiol.* **11**, 359–366
2. Kato, M., Mizuno, T., Shimizu, T., and Hakoshima, T. (1997) *Cell* **88**, 717–723
3. Akerley, B. J., Cotter, P. A., and Miller, J. F. (1995) *Cell* **80**, 611–620
4. Rogov, V. V., Bernhard, F., Lohr, F., and Dotsch, V. (2004) *J. Mol. Biol.* **343**, 1035–1048
5. Takeda, S., Fujisawa, Y., Matsubara, M., Aiba, H., and Mizuno, T. (2001)
6. Lai, Y. C., Peng, H. L., and Chang, H. Y. (2003) *J. Bacteriol.* **185**, 788–800
7. Janiak-Spens, F., Cook, P. F., and West, A. H. (2005) *Biochemistry* **44**, 377–386
8. Xu, Q., Porter, S. W., and West, A. H. (2003) *Structure (Camb.)* **11**, 1569–1581
9. Posas, F., Wurgler-Murphy, S. M., Maeda, T., Witten, E. A., Thai, T. C., and Saito, H. (1996) *Cell* **86**, 865–875
10. Matsubara, M., and Mizuno, T. (2000) *FEBS Lett.* **470**, 118–124
11. Winstanley, C., Kaye, S. B., Neal, T. J., Chilton, H. J., Miksch, S., and Hart, C. A. (2005) *J. Med. Microbiol.* **54**, 519–526
12. Jesaitis, A. J., Franklin, M. J., Berglund, D., Sasaki, M., Lord, C. I., Bleazard, J. B., Duffy, J. E., Beyenal, H., and Lewandowski, Z. (2003) *J. Immunol.* **171**, 4329–4339
13. Rodrigue, A., Quentin, Y., Lazdunski, A., Mejean, V., and Foglino, M. (2000) *Trends Microbiol.* **8**, 498–504
14. Stover, C. K., Pham, X. Q., Erwin, A. L., Mizoguchi, S. D., Warrener, P., Hickey, M. J., Brinkman, F. S., Hufnagle, W. O., Kowalik, D. J., Lagrou, M., Garber, R. L., Goltry, L., Tolentino, E., Westbrock-Wadman, S., Yuan, Y., Brody, L. L., Coulter, S. N., Folger, K. R., Kas, A., Larbig, K., Lim, R., Smith, K., Spencer, D., Wong, G. K., Wu, Z., Paulsen, I. T., Reizer, J., Saier, M. H., Hancock, R. E., Lory, S., and Olson, M. V. (2000) *Nature* **406**, 959–964
15. Chen, Y. T., Chang, H. Y., Lu, C. L., and Peng, H. L. (2004) *J. Mol. Evol.* **59**, 725–737
16. Lin, C. T., Huang, Y. J., Chu, P. H., Hsu, J. L., Huang, C. H., and Peng, H. L. (2006) *Res. Microbiol.* **157**, 169–175
17. Sambrook, J., and Russell, D. W. (2001) *Molecular Cloning: A Laboratory Manual*, 3rd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY
18. Kohler, T., Curty, L. K., Barja, F., van Delden, C., and Pechere, J. C. (2000) *J. Bacteriol.* **182**, 5990–5996
19. Yamamoto, K., Hirao, K., Oshima, T., Aiba, H., Utsumi, R., and Ishihama, A. (2005) *J. Biol. Chem.* **280**, 1448–1456
20. Skerker, J. M., Prasol, M. S., Perchuk, B. S., Biondi, E. G., and Laub, M. T. (2005) *Plos Biol.* **3**, e334
21. Biondi, E. G., Skerker, J. M., Arif, M., Prasol, M. S., Perchuk, B. S., and Laub, M. T. (2006) *Mol. Microbiol.* **59**, 386–401
22. West, A. H., and Stock, A. M. (2001) *Trends Biochem. Sci.* **26**, 369–376
23. Delumeau, O., Dutta, S., Brigulla, M., Kuhnke, G., Hardwick, S. W., Volker, U., Yudkin, M. D., and Lewis, R. J. (2004) *J. Biol. Chem.* **279**, 40927–40937
24. Adler, E., Donella-Deana, A., Arigoni, F., Pinna, L. A., and Stragler, P. (1997) *Mol. Microbiol.* **23**, 57–62
25. Vijay, K., Brody, M. S., Fredlund, E., and Price, C. W. (2000) *Mol. Microbiol.* **35**, 180–188
26. Arigoni, F., Duncan, L., Alper, S., Losick, R., and Stragier, P. (1996) *Proc. Natl. Acad. Sci. U. S. A.* **93**, 3238–3242
27. Lucet, I., Borriss, R., and Yudkin, M. D. (1999) *J. Bacteriol.* **181**, 3242–3245
28. Jin, H., and Pancholi, V. (2006) *J. Mol. Biol.* **357**, 1351–1372
29. Rajagopal, L., Clancy, A., and Rubens, C. E. (2003) *J. Biol. Chem.* **278**, 14429–14441
30. Gee, K. R., Sun, W. C., Bhalgat, M. K., Upson, R. H., Klaubert, D. H., Latham, K. A., and Haugland, R. P. (1999) *Anal. Biochem.* **273**, 41–48
31. Pastula, C., Johnson, I., Beechem, J. M., and Patton, W. F. (2003) *Comb. Chem. High Throughput Screen* **6**, 341–346
32. Chang, C. H., Zhu, J., and Winans, S. C. (1996) *J. Bacteriol.* **178**, 4710–4716
33. Munoz-Dorado, J., Inouye, S., and Inouye, M. (1991) *Cell* **67**, 995–1006
34. Bork, P., Brown, N. P., Hegyi, H., and Schultz, J. (1996) *Protein Sci.* **5**, 1421–1425
35. Mukhopadhyay, S., Kapral, V., Xu, W., and Chakrabarty, A. M. (1999) *J. Bacteriol.* **181**, 6615–6622
36. Mougous, J. D., Gifford, C. A., Ramsdell, T. L., and Mekalanos, J. J. (2007) *Nat. Cell Biol.* **9**, 797–803
37. Hughes, K. T., and Mathee, K. (1998) *Annu. Rev. Microbiol.* **52**, 231–286
38. Goodman, A. L., Kulasekara, B., Rietsch, A., Boyd, D., Smith, R. S., and Lory, S. (2004) *Dev. Cell* **7**, 745–754

APRIL 11, 2008 • VOLUME 283 • NUMBER 15

HptB-mediated Phosphorelay in P. aeruginosa

39. Laskowski, M. A., Osborn, E., and Kazmierczak, B. I. (2004) *Mol. Microbiol.* **54**, 1090–1103
40. Mougous, J. D., Cuff, M. E., Raunser, S., Shen, A., Zhou, M., Gifford, C. A., Goodman, A. L., Joachimiak, G., Ordonez, C. L., Lory, S., Walz, T., Joachimiak, A., and Mekalanos, J. J. (2006) *Science* **312**, 1526–1530
41. Ventre, I., Goodman, A. L., Vallet-Gely, I., Vasseur, P., Soscia, C., Molin, S.,
42. Bleves, S., Lazdunski, A., Lory, S., and Filloux, A. (2006) *Proc. Natl. Acad. Sci. U. S. A.* **103**, 171–176
43. Furste, J. P., Pansegrau, W., Frank, R., Blocker, H., Scholz, P., Bagdasarian, M., and Lanka, E. (1986) *Gene (Amst.)* **48**, 119–131
44. Hoang, T. T., Karkhoff-Schweizer, R. R., Kutchma, A. J., and Schweizer, H. P. (1998) *Gene (Amst.)* **212**, 77–86

Mechanisms of Signal Transduction:  
Characterization of the  
Histidine-containing Phosphotransfer  
Protein B-mediated Multistep  
Phosphorelay System in *Pseudomonas*  
*aeruginosa* PAO1  

Jye-Lin Hsu, Hsuan-Cheng Chen, Hwei-Ling  
Peng and Hwan-You Chang  
*J. Biol. Chem.* 2008, 283:9933-9944.  
doi: 10.1074/jbc.M708836200 originally published online February 5, 2008  

Access the most updated version of this article at doi: [10.1074/jbc.M708836200](http://dx.doi.org/10.1074/jbc.M708836200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](http://www.jbc.org/site/affinitysites/index.xhtml).  

Alerts:  
- [When this article is cited](http://www.jbc.org/cgi/alerts/doi?doi=10.1074%2Fjbc.M708836200&alerttype=citation)  
- [When a correction for this article is posted](http://www.jbc.org/cgi/alerts/doi?doi=10.1074%2Fjbc.M708836200&alerttype=correction)  

[Click here](http://www.jbc.org/cgi/alerts/register) to choose from all of JBC's e-mail alerts  

Supplemental material:  
[http://www.jbc.org/content/suppl/2008/02/07/M708836200.DC1.html](http://www.jbc.org/content/suppl/2008/02/07/M708836200.DC1.html)  

This article cites 42 references, 13 of which can be accessed free at  
[http://www.jbc.org/content/283/15/9933.full.html#ref-list-1](http://www.jbc.org/content/283/15/9933.full.html#ref-list-1)
