
myo-Inositol Catabolism in *Bacillus subtilis*

Received for publication, September 26, 2007, and in revised form, February 19, 2008

Ken-ichi Yoshida$^{\dagger 1}$, Masanori Yamaguchi$^{S}$, Tetsuro Morinaga$^{\ddagger}$, Masaki Kinehara$^{\ddagger}$, Maya Ikeuchi$^{\ddagger}$, Hitoshi Ashida$^{\ddagger}$, and Yasutaro Fujita$^{\P}$

From the $^{\dagger}$Department of Agrobioscience, Graduate School of Agricultural Science, Kobe University Kobe 657-8501, $^{S}$Central Research Laboratories, Hokko Chemical Industry Co., Ltd, Atsugi 243-0023 and $^{\P}$Department of Biotechnology, Faculty of Life Science and Biotechnology, Fukuyama University, Fukuyama 729-0292, Japan

The *iolABCDEF*GHIJ operon of *Bacillus subtilis* is responsible for myo-inositol catabolism involving multiple and stepwise reactions. Previous studies demonstrated that IolG and IolE are the enzymes for the first and second reactions, namely dehydrogenation of myo-inositol to give 2-keto-myo-inositol and the subsequent dehydration to 3D-(3,5/4)-trihydroxycyclohexane-1,2-dione. In the present studies the third reaction was shown to be the hydrolysis of 3D-(3,5/4)-trihydroxycyclohexane-1,2-dione catalyzed by IolD to yield 5-deoxy-D-glucuronic acid. The fourth reaction was the isomerization of 5-deoxy-D-glucuronic acid by IolB to produce 2-deoxy-5-keto-D-gluconic acid. Next, in the fifth reaction 2-deoxy-5-keto-D-gluconic acid was phosphorylated by IolC kinase to yield 2-deoxy-5-keto-D-gluconic acid 6-phosphate. IolR is known as the repressor controlling transcription of the *iol* operon. In this reaction 2-deoxy-5-keto-D-gluconic acid 6-phosphate appeared to be the intermediate acting as inducer by antagonizing DNA binding of IolR. Finally, IolJ turned out to be the specific aldolase for the sixth reaction, the cleavage of 2-deoxy-5-keto-D-gluconic acid 6-phosphate into dihydroxyacetone phosphate and malonic semialdehyde. The former is a known glycolytic intermediate, and the latter was previously shown to be converted to acetyl-CoA and CO₂ by a reaction catalyzed by IolA. The net result of the inositol catabolic pathway in *B. subtilis* is, thus, the conversion of myo-inositol to an equimolar mixture of dihydroxyacetone phosphate, acetyl-CoA, and CO₂.

genes/Klebsiella mobilis) (3), Rhizobium leguminosarum bv. viciae (4), Sinorhizobium meliloti (5), Sinorhizobium fredii (6), Corynebacterium glutamicum (7), and Lactobacillus casei (8) can grow on MI as the sole carbon source. MI catabolism in A. aerogenes was studied biochemically, and a pathway of the catabolism finally yielding acetyl-CoA and dihydroxyacetone phosphate (DHAP) was proposed (9). However, our knowledge of the molecular genetics of bacterial MI catabolism has been restricted to *B. subtilis* (1, 10–12). In *B. subtilis*, the *iol* divergon, comprising the operons *iolABCDEF*GHIJ and *iolRS* (1), and the *iolT* gene (12) were shown to be required for inositol catabolism (Fig. 1). Nowadays, a large number of bacteria have genes annotated *iol* in their genome sequence, but the annotation is only based on sequence similarity to *B. subtilis iol* genes, as relatively few studies have been done to demonstrate the participation of the deduced *iol* genes in MI catabolism.

In *B. subtilis*, a repressor encoded by *iolR* is responsible for the regulation of all the *iol* genes (11, 12). In the absence of MI in the growth medium, the IolR repressor binds to the operator site within the promoter regions to repress the transcription. In its presence, however, MI is taken up by the cell and converted to a catabolic intermediate that acts as an inducer by antagonizing IolR, thereby inducing the *iol* divergon and *iolT* (11, 12). Consequently, inactivation of *iolR* makes the transcription of the *iol* divergon and the *iolT* gene constitutive (1).

Some enzymes involved in MI catabolism have been characterized. Inositol dehydrogenase encoded by *iolG* is responsible for the first step of the degradation cascade by converting MI to 2-keto-MI (2KMI), conversion of compound [1] to [3] see (Fig. 1) (10). Recently IolG was also shown to be able to act on D-chiro-inositol (DCI, compound [2]) to yield 1-keto-DCI (compound [4]), and IolI interconverts 1-keto-DCI and 2KMI, indicating that not only MI but also DCI is metabolized through the MI catabolic pathway (Fig. 1) (14). In addition, pinitol (3-O-methyl-DCI) contained in soybean appeared to be an alternative substrate of IolG, and this substrate was degraded depending on the presence of functional *iol* genes (15). 2KMI dehydratase, encoded by *iolE*, is responsible for the second step to produce 3D-(3,5/4)-trihydroxycyclohexane-1,2-dione (THcHDO, compound [5], formerly called D-2,3-diketo-4-dioxy-epi-inositol) (13). The *iolT* and *iolF* genes encode the primary and secondary inositol transporters, respectively (Fig. 1) (12). The *iolA* gene was shown to encode malonic semialdehyde (MSA) dehydrogenase (16), supposed to be involved in MI catabolism based on the assumption that the catabolic pathway

* This work was supported in part by a grant-in-aid for the Encouragement of Young Scientists from the Ministry of Education, Culture, Sports, Science, and Technology of Japan (to K. Y.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
$^{1}$ To whom correspondence should be addressed: Dept. of Agrobioscience, Graduate School of Agricultural Science, Kobe University, 1-1 Rokkodai, Nada, Kobe 657-8501, Japan. Tel.: 81-78-803-5862; Fax: 81-78-803-5815; E-mail: kenyoshi@kobe-u.ac.jp.
$^{2}$ The abbreviations used are: MI, myo-inositol; DCI, d-chiro-inositol; 5DG, 5-deoxy-glucuronic acid; DHAP, dihydroxyacetone phosphate; DKG, 2-deoxy-5-keto-D-gluconic acid; DKGP, DKG 6-phosphate; ESI-TOF, electrospray ionization-time-of-flight; 2KMI, 2-keto-myo-inositol; LB, Luria-Bertani broth; MSA, malonic semialdehyde; THcHDO, 3D-(3,5/4)-trihydroxycyclohexane-1,2-dione; IPTG, isopropyl 1-thio-β-D-galactopyranoside; MS, mass spectroscopy.

B. subtilis Inositol Catabolism

TABLE 1  
Bacterial strains and plasmids used in this study  

| Strains | Genotype | Reference or source |
|---------|----------|---------------------|
| B. subtilis | trpC2metC7 | Laboratory collection |
| 60015 | trpC2metC7iolR::cat | 1 |
| YF244 | trpC2metC7iolR::catiolB52 | This work |
| YF258 | trpC2metC7iolR::catiolB58 | This work |
| YF259 | trpC2metC7iolR::catiolC62 | This work |
| YF260 | recA1endA1gyrA96thi-1hsdR17(rK–mK+)e14–(mcrA–)supE44relA1 | 20 |
| E. coli | Δ(lac-proAB)/F[traD36proAB+ lacIq lacZΔM15] |  |
| JM109 | F–ompT hsdS(rK–mK–) dcm gal (DE3) tonA | Novagen |
| BL21(DE3) |  |  |

| Plasmids | Genotype | Reference or source |
|----------|----------|---------------------|
| pDGHisC | ampPT7'His6-tag | K. Asai |
| pET30a | kan | Novagen |
| pPIOLf | ampPiol | 11 |
| pIOLB | ampPlac-iolB | 13 |
| pIOLBC | ampPlac-iolBC | 13 |
| pIOLB-His | ampPT7-iolB-his | This work |
| pIOLC | ampPlac-iolC | 13 |
| pIOLD | ampPlac-iolD | 13 |
| pIOLDE | ampPlac-iolDE | 13 |
| pIOLJ-His | ampPT7-iolJ-his | This work |
| pIOLR3 | ampPlac-iolR | 11 |
| pG-KJE8 | catdnaKdnaJgrpEgroESgroEL | 24 |
| pUC18 | amplacZM15 | 20 |
| pUC19 | amplacZM15 | 20 |

is similar to that proposed by Anderson and Magasanik for A. aerogenes (9). But the reaction step producing MSA has not been identified. Homology searches indicated possible functions for the products of the iolC and iolJ genes (13), but none has been identified experimentally. In the present studies we tried to identify the functions of iolD, iolB, iolC, and iolJ, aiming to define the whole pathway of MI catabolism and to understand how it is linked to the transcriptional regulation of the iol genes.

EXPERIMENTAL PROCEDURES

**Bacterial Strains, Plasmids, and Growth Conditions—** Bacterial strains and plasmids used in this study are listed in Table 1. B. subtilis 60015 (trpC2 metC7) is our standard laboratory strain. B. subtilis strains YF244, YF258, YF259, and YF260 are 60015 derivatives. YF244 possesses the iolR::cat mutation inactivating the IolR repressor to allow constitutive expression of the iol divergon (1). In a previous study, some iol mutants defective in MI utilization were obtained by ethyl methanesulfonate mutagenesis, and subsequent detection of colonies were unable to ferment inositol on tetrazolium-containing plates (17). The complementation test through transformation using various PCR fragments (18) covering each of the iol genes demonstrated that some appeared to possess mutations localized within iolB and iolC. Subsequent nucleotide sequencing revealed three mutations of iolB52 (Gln-137 (CAA) to stop (TAA)), iolB58 (Glu-162 (GAA) to Lys (AAA)), and iolC62 (Ala-269 (GCC) to Thr (ACC)) as described previously (13). Strains YF258, YF259, and YF260 have iolB52, iolB58, and iolC62, respectively, besides iolR::cat, which allows the constitutive iol expression (1). B. subtilis cells were grown on tryptose blood agar base (Difco) supplemented with 0.18% glucose at 30°C and incubated in S6 medium (19) containing 0.5% casamino acids (Difco), 50 μg/ml each of Trp and Met, and supplemented with or without 10 mM MI at 37°C with shaking.

Escherichia coli strains JM109 (20) and BL21(DE3) (Novagen) were used as the hosts for plasmid constructions and expression of C-terminal His6-tag fusion proteins, respectively. E. coli cells were grown in Luria-Bertani broth (LB) (21) and TGA (22) media at 37°C. Plasmids pUC18, pUC19 (20), pET30a (Novagen), and pDGHisC (a gift from Kei Asai, Saitama University) were used as cloning vectors. pDGHisC is a pDG148 (23) derivative designed to express a desired gene as a C-terminal His6 tag fusion under the control of T7 promoter in E. coli. pG-KJE8 (24) was used for supplying chaperons to stabilize overproduced fusion proteins. When required, media were supplemented with ampicillin (50 μg/ml), arabinose (1 mg/ml), chloramphenicol (15 and 35 μg/ml for B. subtilis and E. coli, respectively), IPTG (0.5 mM), kanamycin (50 μg/ml), and tetracycline (5 ng/ml).

**Cloning and Expression of the iol Genes in E. coli and Purification of the Gene Product—** The native iolB, iolC, iolD, and iolE genes were cloned into the pUC18 vector and expressed in E. coli cells grown in TGA medium as described previously (13). To express iolB and iolJ and produce the corresponding proteins as C-terminal His6-tag fusions, pIOLB-His and pIOLJ-His, respectively, were constructed as follows. DNA fragments covering the iolB or iolJ coding regions, associated with their Shine-Dalgarno sequence and a short linker enabling the C-terminal His6 tag fusion at the head and tail, respectively, were amplified from DNA of strain 60015 by PCR using the specific primers. The primers for the iolB PCR fragment were iolBE (CCCGAATTCTAGAAAGAGGGAGGGCTTCTC, an EcoRI site is indicated in bold) and iolBHIsC (CCATCGATGCGTTCTAAAAATCCATTCATG, a ClaI site is in bold), and those for the iolJ fragment were iolJE (CCGAATTCTTCAGCATGTAATGGATAAAGG, an EcoRI site is in bold) and iolJHisE (CCGGAATTCCGCCCACCTGCTGCTTAGCTG, in an EcoRI site is in bold). The iolB fragment was trimmed with EcoRI and

Clal, whereas the *iolJ* fragment was cut with EcoRI alone, and each of these fragments was ligated to the arm of pDGHisC that had been cleaved with the respective enzyme(s). The ligated DNA mixtures were used for the transformation of JM109 to ampicillin resistance to result in plasmids pIOLB-His and pIOJE-His, which carried the respective *iol* genes as C-terminal His₆ tag fusion placed under the control of the pDGHisC-borne T7 promoter. To express *iolA* as a C-terminal His₆ tag fusion, pIOLA-His was constructed similarly as above but using pET30a (Novagen) as the cloning vector. DNA fragments covering the *iolA* coding region and a short linker enabling the C-terminal His₆ tag fusion at the tail was amplified from DNA of strain 60015 by PCR using the specific primers, *iolA*-NdeI (GGA-ATTCCATATGATGGCAGAAAATCAGAAAAATTA, an NdeI site is indicated in bold) and *iolA*-NotI (ATAAGAATG-CGGCCGCGTTGAAATCAGGTGCCGGAT, an NotI site in bold). The PCR fragment was trimmed with NdeI and NotI and ligated to the arm of pET30a that had been cleaved with the same enzymes. The ligated DNA mixtures were used for the transformation of JM109 to kanamycin resistance to result in the plasmid pIOLA-His, which carried *iolA* as C-terminal His₆ tag fusion placed under the control of the pDGHisC-borne T7 promoter. Correct construction of the plasmids was confirmed by nucleotide sequencing. To express the His₆ tag fusion of *iolB*, *iolJ*, and *iolA* genes, pIOLB-His, pIOLJ-His, and pIOLA-His, respectively, were introduced into BL21(DE3) cells together with pG-KJE8. BL21(DE3) carrying both plasmids was inoculated into TGA medium containing ampicillin, arabinose, chloramphenicol, and tetracycline and grown at 37°C with shaking. At an optical density of the cultures of 0.35 at 600 nm, production of the C-terminal His₆ tag fusion proteins was induced for 2 h by the addition of IPTG. The cells were harvested, treated with lysozyme, and disrupted by brief sonication, and the soluble fraction was obtained after centrifugation. The His₆ tag fusion proteins were purified from the soluble fraction using the His-Bond system (Novagen), employing the standard procedure as recommended by the supplier. The purified proteins were subjected to Sephadex G-25 (Amersham Biosciences) gel filtration to remove imidazole and salts. The purity of the His₆ tag fusion proteins was verified by SDS-PAGE and subsequent immunoblot analyses using anti-His tag antibody (Nacalai Tesque).

**Enzyme Assays—THcHDO hydrolase activity** was measured as follows. The reaction substrate, THcHDO, was prepared by the enzymatic conversion of 2KMI and purified as described (13). JM109 cells carrying pUC18, pIOLB, pIOLC, or pIOLD were grown with IPTG, and the protein extracts were prepared as described (13). The hydrolase assay was performed by the procedures modified from those previously reported (3). 0.98 ml of assay mixture was prepared to contain 50 mM Tris-Cl (pH 8.0), 0.1 mM glutathione, 0.05 mM CoCl₂, and sufficient THcHDO (giving an absorbance of 0.6 at 260 nm; ~0.1 mM). After adding 20 μl of the protein extract containing ~60 μg of protein to the mixture, the rate of decrease in the absorbance at 260 nm was measured, and consumption of the substrate THcHDO was calculated from its estimated molar absorption coefficient of 6000 (3).

---

**B. subtilis Inositol Catabolism**

Phosphorylation triggered by the addition of 5-deoxy-glucuronic acid (5DG) was monitored as follows. The reaction substrate, 5DG, was prepared by the enzymatic conversion of 2KMI and purified as described below. JM109 cells carrying pUC18, pIOLB, pIOLC, or pIOLBC were grown with IPTG, and the protein extracts were prepared as described (13). *B. subtilis* cells were grown with and without inositol, and the extracts were prepared in the same way as described for the inositol dehydrogenase assay (1). Phosphorylation was monitored by the procedures modified from those reported previously (9). 0.9 ml of assay mixture was prepared to contain 50 mM Tris-Cl (pH 8.0), 2.5 mM MgCl₂, 1.25 mM phosphoenolpyruvate, 0.5 mM ATP, 3.0 mM KCl, 10 mM 2-mercaptoethanol, 10 μg/ml of pyruvate kinase, 5 μg/ml of lactate dehydrogenase, 0.25 mM NADH, and 100 μl of protein extract (~200–300 μg of protein). 0.1 ml of 20 mM 5DG or H₂O was added to the mixture, and the rate of decrease in the absorbance of NADH at 340 nm was measured.

IolJ aldolase activity was measured as follows. The substrate was the reaction product of the IolBC reaction prepared as described below. IolJ-His was purified as indicated above and used for the enzyme assay. The IolJ reaction was supposed to produce DHAP, and the production of DHAP was monitored by the procedures modified from those reported previously (9). The purified IolJ-His protein was serially diluted in test tubes, each containing the reaction mixture (100 mM Tris-Cl (pH 7.6), 3.3 mM DKG, 1.43 mM NADH, and glycerol-1-phosphate dehydrogenase (0.1 unit/μl)), and incubated at 24°C for 30 min. Each of the reaction mixtures was diluted 25 times immediately after the reaction to measure the absorbance of NADH at 340 nm.

The IolJ reaction was supposed to produce MSA besides DHAP, and MSA would be the substrate of IolA MSA dehydrogenase, which was shown to convert MSA to acetyl-CoA and CO₂ with the specific reduction of NAD⁺ to NADH (16). Thus, MSA was detected as follows by the procedures modified from those reported previously (9). The purified IolJ-His protein was serially diluted in test tubes, each containing the reaction mixture (100 mM Tris-Cl (pH 8.0), 3.3 mM DKG, IolA-His (5.7 μg), 3.0 mM CoA, and 3.0 mM NAD⁺), and incubated at 24°C for 60 min. Each of the reaction mixtures was diluted 25 times immediately after the reaction to measure the absorbance of NADH at 340 nm.

**Purification and Structural Analysis of the Reaction Products—** The product of the IolD reaction was synthesized and characterized as follows. JM109 cells carrying pIOLDE (13) were grown in 100 ml of LB containing ampicillin and IPTG for 16 h, harvested, suspended in 2.2 ml of water, treated with lysozyme, disrupted by sonication, and centrifuged to obtain the supernatant solution containing both the IolD and IolE enzymes. The enzyme solution (~3 ml) was mixed with 250 mg of 2KMI in 9 ml of 33 mM phosphate buffer (pH 8.0) and incubated at 25°C, allowing the reaction to proceed. During the reaction the pH values were maintained at 8.0 by adding NaOH when required. After 20 h appropriate amounts of 6 N H₂SO₄ were added to drop the pH down to 3.0, and the reaction was terminated. The mixture was heated at 85°C for 10 min to denature the proteins

and then centrifuged for 10 min at 5000 rpm. The supernatant solution was saved, concentrated, and dried. The dried solute was dissolved in 20 ml of methanol/ethyl acetate (4:1, v/v), and the solution was passed through a column filled with 12 ml silica gel (Merck) to remove the remaining substrate and intermediates. The eluent was concentrated once again and dried to obtain 119 mg of pale yellow compound. The ${}^{1}$H NMR spectrum of this compound was obtained in $D_{2}O$.

The IolBC reaction product was identified as follows. JM109 cells carrying pIOLBC (13) were grown in 100 ml of LB containing ampicillin and IPTG for 16 h, harvested, suspended in 2.2 ml of water, treated with lysozyme, disrupted by sonication, and centrifuged to obtain the supernatant solution containing both the IolB and IolC enzymes. The enzyme solution ($\sim 3 \mathrm{ml}$) and 222 mg of 5DG were mixed to make 13 ml of reaction mixture (pH 8.0, adjusted with NaOH) containing $90 \mathrm{mM} \mathrm{ATP}, 3 \mathrm{mM} \mathrm{KCl}$, and $3 \mathrm{mM} \mathrm{MgCl}_{2}$. The mixture was incubated at $26^{\circ} \mathrm{C}$, allowing the reaction to proceed. 24 h later the reaction was terminated by the addition of HCl to lower the pH to 2–3. After being concentrated to 10 ml and mixed with 40 ml of methanol, the mixture was centrifuged to recover the supernatant solution. The solution was concentrated to 10 ml, neutralized with NaOH to adjust the pH to 7.0, and combined and mixed well with $2.5 \mathrm{ml}$ of $\mathrm{BaCl}_{2}$. The mixture was centrifuged to obtain the white sediment, which was suspended in $10 \mathrm{ml}$ of water, mixed with $10 \mathrm{ml}$ of Duolite C20 resin ($\mathrm{H}^{+}$ type; Sumitomo Chemical), and stirred for $2 \mathrm{~h}$ at room temperature. The resin was removed by filtration and washed with $20 \mathrm{ml}$ of water. All of the filtrate ($\sim 30 \mathrm{ml}$) was centrifuged to obtain the clear supernatant. After adjusting its pH to 3.0 with HCl, the supernatant was concentrated and dried to obtain the solute. The dried solute was dissolved in $4 \mathrm{ml}$ of methanol/water (3:1, v/v), and subjected to column chromatography through $25 \mathrm{ml}$ of silica gel with a mobile eluent of methanol/water (9:1, v/v) to collect the eluent in fractions. The desired fractions were combined, concentrated, and then dried to obtain $60 \mathrm{mg}$ of a pale brown compound. The ${}^{1} \mathrm{H}$ NMR spectrum of this compound was obtained in $D_{2} O$.

The IolB-His reaction product was characterized as follows. The purified IolB-His protein ($\sim 150 \mu \mathrm{g}$) and $10 \mathrm{mg}$ of 5DG were combined in $0.14 \mathrm{ml}$ of reaction mixture (pH 8.0, adjusted with NaOH). The reaction mixture was incubated at $26^{\circ} \mathrm{C}$ for 16 h. The reaction was terminated by the addition of $\mathrm{H}_{2} \mathrm{SO}_{4}$ to lower the pH to 2–3 and centrifuged to save the clear supernatant solution, which was concentrated and dried to obtain the solute. The dried solute was suspended in $5 \mathrm{ml}$ of chloroform/methanol (1:1, v/v) and centrifuged to save the supernatant solution. The solution was passed through a column filled with $2 \mathrm{ml}$ of silica gel, and the eluent was concentrated once again and dried to obtain $9 \mathrm{mg}$ of pale yellow compound. The ${}^{1} \mathrm{H}$ NMR spectrum of this compound was obtained in $D_{2} O$.

Mass spectra of the IolD and IolBC reaction products were obtained by ESI-TOF/MS (Mariner Biospectrometry Workstation; Applied Biosystems). The products were dissolved in 50% acetonitrile containing 1% triethylamine for anion analysis.

DNase I Footprinting Experiment—DNase I footprinting experiments were performed as described (11). The IolR protein was produced in JM109 carrying pIOLR3, and the crude extract was used for the experiments as described (11). The probe DNA for the gel retardation and DNase I footprinting experiments was prepared by PCR amplification of a DNA stretch containing the iol promoter region using plasmid DNA of pPIOLF(11) as a template and a pair of specific primers of M4 and RV (Takara). The 5'-terminus of the RV primer had been labeled with a Megalabel kit and [$\gamma^{-32} \mathrm{P}] \mathrm{ATP}$ before the PCR amplification. DNase I footprinting reactions were done in the presence and absence of intermediates of the MI catabolic pathway.

## RESULTS

The Third Step Reaction of the MI Catabolic Pathway Is Catalyzed by IolD Hydrolyzing THcHDO to 5DG—Previous studies revealed that IolB, IolC, IolD, IolE, and IolG were all required for the initial reaction steps of MI catabolism to produce an intermediate acting as the inducer in the cell to antagonize DNA binding of IolR to induce transcription of the iol operon (13). Because IolG and IolE catalyze the first and second reactions, respectively (10, 13), either IolB, IolC, IolD, or combinations of these must be required for the third reaction.

The substrate of the third reaction is THcHDO, which is the product of the IolE reaction. In A. aerogenes, the third reaction was demonstrated to be ring scission (hydrolysis) of the substrate to yield 2-deoxy-5-keto-D-gluconic acid (DKG, compound [7]) in Fig. 1) as the product (25). Assuming that the third reaction in B. subtilis could be similar to that in A. aerogenes, THcHDO hydrolyzing activity in the protein extracts of the E. coli cells expressing iolB, iolC, or iolD was measured (Table 2). The results clearly indicated that only the extract prepared from cells expressing iolD contained a large amount of the THcHDO hydrolyzing activity. From these results, we concluded that iolD encoded the B. subtilis THcHDO hydrolase. The ESI-TOF mass spectrum of the isolated product gave a major peak of an anion at $m / z 177.0484$ (data not shown). It was likely that this anion could correspond to the molecular ion of $\left[\mathrm{C}_{6} \mathrm{H}_{10} \mathrm{O}_{6}-\mathrm{H}\right]^{-}$, suggesting that it might be DKG. However, its ${}^{1} \mathrm{H}$ NMR spectra revealed that the product of the B. subtilis enzyme reaction was not DKG but 5DG (compound [6]) in Fig. 1). Actually, the product was an $\sim 1: 1$ mixture of compounds [12] and [13], $\alpha$- and $\beta$-anomers of the five-membered ring compounds (Fig. 2), indicating that these were produced after spontaneous formation of a hemiacetal linkage between the Cl aldehyde and the C4 hydroxyl groups of 5DG (compound [6]). Therefore, the IolD THcHDO hydrolase of B. subtilis could cleave the C2–C3 bond of THcHDO to yield 5DG (Fig. 1).

Kinase Activity Triggered by 5DG Addition Requires Both IolB and IolC—In A. aerogenes, the next step is phosphorylation of the product of the third reaction, DKG, and the reaction is mediated by a kinase requiring ATP (9). In B. subtilis, the product of the third step appeared to be 5DG, and we were not sure if such phosphorylation would occur on 5DG itself. Nevertheless, we tried to detect the possible kinase activity, which might be triggered by 5DG addition, in the protein extracts of the E. coli cells expressing iolB and/or iolC. In this detection system, the activity was judged spectrophotometrically as described (9) by the difference between the rates of decrease in

B. subtilis Inositol Catabolism

iolT and iolF [3]

[1]


![Structure 1](structure_1.png)

Cell membrane

[2]


![Structure 2](structure_2.png)

?

[iolG]


![Structure 3](structure_3.png)

NAD⁺
↓
NADH+H⁺

[iolG]


![Structure 4](structure_4.png)

[ioll]
↑

[iolE]


![Structure 5](structure_5.png)

[4]

H₂O

ADP
↑
ATP

[iolC]


![Structure 6](structure_6.png)

[8]

[iolB]


![Structure 7](structure_7.png)

[7]

[iolD]


![Structure 8](structure_8.png)

[6]

H₂O

[5]


![Structure 9](structure_9.png)

[iolJ]


![Structure 10](structure_10.png)

[9]

CoA

[iolA]


![Structure 11](structure_11.png)

[10]

NADH+H⁺
↓
NAD⁺

CH₃—C—CoA
[11]

CO₂

FIGURE 1. MI catabolic pathway and functional activities of the B. subtilis iol genes. B. subtilis iol genes proven to encode the enzymes involved in the various reaction steps of the MI catabolic pathway are shown. Compounds: [1], MI; [2], DCI; [3], 2KMI; [4], 1-keto-DCI; [5], THchDO; [6], 5DG; [7], DKG; [8], DKGP; [9], DHAP; [10], MSA; [11], acetyl-CoA. Carbon numbering is defined for MI, 2KMI, THchDO, 5DG, DKG, and DKGP; for the former three the definition from previous studies is applied (3, 25).

APRIL 18, 2008 • VOLUME 283 • NUMBER 16 JOURNAL OF BIOLOGICAL CHEMISTRY 10419

# B. subtilis Inositol Catabolism

## TABLE 2
THcHDO-hydrolyzing activity in the protein extracts of the E. coli cells

| Plasmid in JM109 | THcHDO hydrolyzing activity$^a$ |
|------------------|---------------------------------|
| pUC18            | $14.3 \pm 14.2$                |
| pIOLB            | $20.2 \pm 12.1$                |
| pIOLC            | $26.6 \pm 11.4$                |
| pIOLD            | $253.0 \pm 17.7$               |

**nmol/min/mg of protein**

$^a$ THcHDO hydrolyzing activity in E. coli JM109 carrying the plasmids grown in TGA medium containing ampicillin; IPTG was determined. Mean values ± S.D. of three independent measurements are shown.

---

### Structure Determination of the Product of the IolD Reaction

#### Compound [12]
![Structure of Compound [12]](structure_12.png)

| 5DG ($\alpha$-anomer) | Summary of ${}^{1}$H-NMR (300MHz, D$_2$O) |
|------------------------|------------------------------------------|
| 85.05 (1H, d, H-1)     |                                          |
| 83.96 (1H, t, H-2)     |                                          |
| 84.04 (1H, dd, H-3)    |                                          |
| 84.43 (1H, dt, H-4)    |                                          |
| 82.41 (2H, dd, H-5)    |                                          |

#### Compound [13]
![Structure of Compound [13]](structure_13.png)

| 5DG ($\beta$-anomer) | Summary of ${}^{1}$H-NMR (300MHz, D$_2$O) |
|-----------------------|------------------------------------------|
| 85.28 (1H, d, H-1)    |                                          |
| 83.98 (1H, t, H-2)    |                                          |
| 84.13 (1H, dd, H-3)   |                                          |
| 84.47 (1H, dt, H-4)   |                                          |
| 82.54 (2H, dd, H-5)   |                                          |

---

## TABLE 3
Kinase activity triggered by 5DG addition in the protein extracts of E. coli and B. subtilis cells

| $\Delta A_{340}$/min/mg of protein |
|------------------------------------|
| -5DG                              |
| +5DG                              |

| Plasmid in JM109$^b$ | -5DG | +5DG |
|----------------------|------|------|
| pUC18                | 1.40 | 0.99 |
| pIOLB                | 0.71 | 0.58 |
| pIOLC                | 0.91 | 1.41 |
| pIOLBC               | 0.79 | 2.10 |

| Strain (relevant genotype$^c$) | -5DG | +5DG |
|-------------------------------|------|------|
| 60015 (wild type)              | 0.12 | 0.11 |
| YF244 ($iolR::cat$)            |      |      |
| $-$MI                         | 0.23 | 0.88 |
| $+$MI                         | 0.27 | 0.82 |
| YF258 ($iolR::cat iolB 52$), $-$MI | 0.15 | 0.31 |
| YF259 ($iolR::cat iolB 58$), $-$MI | 0.12 | 0.31 |
| YF260 ($iolR::cat iolC 62$), $-$MI | 0.14 | 0.13 |

$^a$ Kinase activity triggered by 5DG addition was investigated in the protein extracts of the cells. Values shown are the rates of decrease in absorbance of NADH at 340 nm ($\Delta A_{340}$/min)/mg of protein in the presence of 5DG (+5DG) and in its absence ($-$5DG). A higher kinase activity triggered by 5DG addition resulted in a more pronounced decrease in absorbance at 340 nm. Only a set of representative data is shown, but at least three independent measurements were repeated with similar results.

$^b$ E. coli JM109 cells carrying the indicated plasmid were grown in TGA medium containing ampicillin and 1 mM IPTG.

$^c$ B. subtilis cells were grown in S6 medium with (+) or without ($-$) 10 mM MI as indicated.

---

### DKGP
Summary of ${}^{1}$H-NMR (300MHz, D$_2$O)

|                       |
|-----------------------|
| 82.42 (2H, d, H-2)    |
| 84.35 (1H, dt, H-3)    |
| 84.33 (1H, d, H-4)     |
| 84.65 (1H, dd, H-6a)   |
| 84.75 (1H, dd, H-6b)   |

---

### FIGURE 3. Structure determination of the product of the IolBC reaction.
The IolBC reaction product was identified as compound [8], DKGP (with carbon and proton numbering). A summary table of its ${}^{1}$H NMR spectrum analysis (top) and defined structural formula (bottom) are given.

![Structure of Compound [8]](structure_8.png)

---

The phosphorylated product of the kinase reaction involving both IolB and IolC was extracted as a barium salt form and found to give a very complicated ESI-TOF mass spectrum, probably due to its lower purity as well as differential ionization (data not shown). However, its ${}^{1}$H NMR spectra clearly revealed that the product was DKGP (compound [8]) in Figs. 1 and 3; ${}^{1}$H

DKG

Summary of ${}^{1}$H-NMR (300MHz, D$_{2}$O)

$\delta$2.44 (2H, d, H-2)  
$\delta$4.27 (1H, dt, H-3)  
$\delta$4.23 (1H, d, H-4)  
$\delta$4.41 (1H, d, H-6a)  
$\delta$4.53 (1H, d, H-6b)  

![Structure](https://i.imgur.com/structure.png)

FIGURE 4. Structure determination of the product of the IolB reaction. The IolB reaction product was identified as compound [7], DKG (with carbon and proton numbering). A summary table of its ${}^{1}$H NMR spectrum analysis (top) and defined structural formula (bottom) are given.

NMR summary shown in Fig. 3), which turned out to be the same compound as in *A. aerogenes* (9).

**IolB Isomerizes 5DG to DKG, and IolC Phosphorylates DKG to DKGP—**The kinase reaction involving both IolB and IolC produced DKGP from 5DG. On the other hand, a homology search revealed that IolB exhibited similarities to a number of 4-deoxy-L-threo-5-hexulose uronate isomerases, and its amino acid sequence contains the motif of 5-keto-4-deoxyuronate isomerase as suggested in the BSORF data base. These facts prompted us to test whether IolB might be an isomerase converting 5DG to DKG. IolB was produced in *E. coli* as a C-terminal His$_{6}$ tag fusion protein and purified as IolB-His. The purified IolB-His was incubated with 5DG, and the reaction product was obtained. Although its ESI-TOF mass analysis was not performed, its ${}^{1}$H NMR spectrum was clear enough to indicate that it contained an equimolar mixture of 5DG and DKG (compound [7]) in Figs. 1 and 4; ${}^{1}$H NMR summary shown in Fig. 4). These results allowed us to conclude that IolB encodes the isomerase responsible for the fourth-step reaction, thereby converting 5DG to DKG reversibly. The amino acid sequence of IolC exhibits significant similarity to fructo-kinases and in its amino acid sequence contains a carbohydrate kinase motif of the pfkB family as listed in the BSORF data base, suggesting that IolC could be a kinase phosphorylating the C6 hydroxyl group of DKG (Fig. 4). Thus, the conversion of 5DG to DKGP appeared to involve two successive reactions, namely the fourth reaction of the pathway catalyzed by IolB isomerizing 5DG to DKG and then the fifth reaction by IolC, thereby phosphorylating DKG to DKGP.

---

![DNA Footprinting](https://i.imgur.com/dna_footprinting.png)

FIGURE 5. DKGP antagonizes the DNA binding function of IolR. DNase I footprinting analysis of the interaction of IolR with the *iol* promoter region was performed as described in the text. DNase I footprints of the 5'-end-labeled coding strand of the probe DNA were prepared as described in the text. Lanes 1–10 contained 0.04 pmol of the ${}^{32}$P-labeled probe DNA in the reaction mixture (50 $\mu$l). Lanes 1–9 contained 4.0 $\mu$g of the protein extract of JM109 cells bearing pIOLR3, whereas lane 10 contained that of the cells bearing pUC19. Lanes 1, 5, 9, and 10 contained no additional chemicals, whereas lanes 2, 3, 4, 6, 7, and 8 did contain MI, 2KMI, THcHDO, 5DG (~1:1 mixture of $\alpha$- and $\beta$-anomers), DKG/5DG (~1:1 mixture), and DKGP, respectively (each approximately at 10 mM). Lanes G, A, T, and C contained the products of the respective sequencing reactions performed with the same primers as used for the probe preparation. The enhanced cleavage sites and protected regions are indicated by arrowheads and vertical lines on the right side, and nucleotide numbers are shown on the left (+1 is the transcription start nucleotide of the *iol* operon).

The Product of the IolC Reaction, DKGP, Antagonizes IolR Binding to DNA—IolG, IolE, IolD, IolB, and IolC appeared to catalyze the initial five reaction steps of the MI catabolic pathway in this order (Fig. 1), and previously all of these proteins were also shown to be required for the production of an intermediate acting as the inducer in cell (13). These findings suggested that the product of the IolC reaction might be the inducer that antagonizes DNA binding of IolR. It is known that the IolR repressor binds to DNA containing the *iol* promoter region, thereby altering the DNase I digestion pattern (11). Therefore, each of the intermediates appearing in the catabolic pathway was tested for its ability to interfere with DNA binding of IolR (Fig. 5). None of the compounds MI, 2KMI, THcHDO, the mixture of $\alpha$-and $\beta$-anomers of 5DG, or DKG altered the

B. subtilis Inositol Catabolism

DNase I digestion pattern upon IolR binding (11), whereas only DKGP rendered the digestion pattern almost the same as that without IolR binding. The results clearly indicated that the reaction product of the IolC reaction, DKGP, antagonized IolR for its DNA binding, revealing DKGP as the intermediate that functions as the inducer in the cell.

**IolJ Is an Aldolase Acting on DKGP to Produce DHAP and MSA—In A. aerogenes it was shown that an aldolase converted DKGP into two C3 compounds, DHAP (compound [9]) in Fig. 1) and MSA (compound [10]) in Fig. 1). The amino acid sequence of IolJ shares high similarities with those of many aldolases (1). IolJ was produced in E. coli as a C-terminal His₆ tag fusion protein and purified as IolJ-His. IolJ-His was incubated with DKGP to test its aldolase activity, employing the assay system previously described (9). The aldolase reaction would produce DHAP, which is known to serve as the specific substrate for glycerol-1-phosphate dehydrogenase involving the oxidation of NADH. Under the conditions we used, the more IolJ-His was contained in the reaction mixture, the more DHAP was produced (Fig. 6A), clearly indicating that IolJ-His had the aldolase activity producing DHAP from DKGP. When fructose bisphosphate was used as another substrate in the same system as above, no production of DHAP was observed (data not shown), suggesting that IolJ might not function as fructose-bisphosphate aldolase as annotated in databases.

The *iolA* gene product was previously characterized as MSA dehydrogenase (16), which acts on MSA to convert it into acetyl-CoA and CO₂ with reduction of NAD⁺ to NADH. Because MSA is very unstable and its authentic standard for NMR analysis is not available, production of MSA was also shown in the enzymatic way employing IolA. IolA was produced in E. coli and purified as a C-terminal His₆ tag fusion protein, IolA-His. When the IolJ aldolase reaction was performed in the presence of IolA-His together with CoA and NAD⁺, the specific reduction of NAD⁺ to NADH was observed, suggesting that MSA produced by the aldolase reaction could serve as the substrate for the successive IolA reaction. The more IolJ-His was contained in the reaction mixture, the more MSA was produced (Fig. 6B), although the production was as low as 10% that of DHAP. Probably under the conditions we used, most of MSA was lost due to its instability and possible side reactions. Nevertheless the results clearly indicated that IolJ-His functioned to produce MSA from DKGP. We, thus, concluded that IolJ was the aldolase acting on DKGP, thereby converting it into DHAP and MSA. DHAP is a known glycolytic intermediate, and MSA is converted into acetyl-CoA and CO₂ with reduction of NAD⁺ by the MSA dehydrogenase of the *iolA* gene product.

---

### DISCUSSION

The work described here together with previously published works revealed that MI catabolism of B. subtilis comprises seven stepwise reactions (Fig. 1). The first step is the dehydrogenation of MI to 2KMI, which is catalyzed by IolG, and the second step is the successive dehydration by IolE to give THcHDO. The third step is ring scission of THcHDO, catalyzed by IolD, to yield 5DG, and the fourth step is the isomer-

---

FIGURE 6. Aldolase activity of IolJ-His producing DHAP (A) and MSA (B) from DKGP. A, DHAP production. The experiment was performed as described under “Experimental Procedures.” Amounts of DHAP produced in the reaction tube within 30 min (nmol) were calculated from the decrease in the absorbance of NADH at 340 nm and plotted against the amounts of IolJ-His (µg). B, MSA production. The experiment was performed as described under “Experimental Procedures.” Amounts of MSA produced in the reaction tube within 60 min (nmol) were calculated from the increase in the absorbance of NADH at 340 nm and plotted against the amounts of IolJ-His (µg). Only the sets of representative data are shown, but at least three independent measurements were repeated with similar results.

ization of 5DG by IolB to produce DKG. The fifth step is the phosphorylation of DKG by IolC, yielding DKGP, and the sixth step is the aldolase reaction by IolJ, cleaving DKGP into DHAP and MSA. The last, seventh, step is catalyzed by IolA, thereby converting MSA to acetyl-CoA and CO₂. The net result of the inositol catabolic pathway in B. subtilis is the conversion of one MI molecule to yield one DHAP, one acetyl-CoA, and one CO₂ molecule, with acquisition of two NADH molecules and the consumption of an ATP molecule.

In A. aerogenes, the ring scission of THcHDO, was initially reported to give 4-deoxy-5-ketohexonic acid as the product (25). This was reinvestigated 5 years later, however, with the result that the product should be DKG (26). This suggests
that the C2–C3 bond of THcHDO (compound [5]) in Fig. 1)  
was hydrolyzed upon the ring scission to yield DKG in a  
single step, taking into account the possible keto-enol tau-  
tomerization of THcHDO (26). In *B. subtilis*, our results  
indicated that the product of the ring scission catalyzed by  
IolD was not DKG but 5DG (compound [6]) in Fig. 1). After  
its isolation, 5DG was found in an equimolar mixture of  
compounds [12] and [13] (Fig. 2), which were α- and β-ano-  
mers produced after the spontaneous formation of a hemi-  
acetal linkage between its C1 aldehyde and the C4 hydroxyl  
groups, suggesting that both in *A. aerogenes* and *B. subtilis*  
the C2–C3 bond of THcHDO was cut but the product was  
different. However, we showed that in *B. subtilis* 5DG was  
successively isomerized by IolB enzyme to yield DKG, which  
was the same product identified in *A. aerogenes* after the ring  
scission. Therefore, the conversion of THcHDO to DKG in  
*B. subtilis* requires the successive hydrolysis and isomeriza-  
tion reactions corresponding to the third and fourth steps of  
the pathway catalyzed by IolD and IolB, respectively (Fig. 1).  
The previous studies on MI catabolism in *A. aerogenes* were  
performed in a biochemical way mostly using crude extracts  
or partially purified enzymes prepared from mutant strains  
(3, 9, 25, 26). It is possible that both the active counterparts  
for IolD hydrolase and IolB isomerase were contained  
together in the crude extract and enzyme preparations,  
resulting in DKG as the major product so that the interme-  
diates, α- and β-anomers of 5DG, might be barely detectable.  
Using genetic tests, all functional *iolB*, *iolC*, *iolD*, *iolE*, and  
*iolG* genes were previously shown to be required for the pro-  
duction of an intermediate acting as the inducer in the cell (13).  
Consistently, the fifth step of the pathway was the phosphoryl-  
ation of DKG by IolC to produce DKGP (Fig. 1). This antago-  
nizes IolR for its DNA binding activity and reveals that DKGP is  
the inducer in the cell (Fig. 5). IolR belongs to the DeoR family  
of bacterial transcription repressors. Because members of this  
family are known to interact with an inducer of phosphorylated  
sugar (27), our results are well in agreement with this feature of  
the DeoR family members. Probably because DKGP is the spe-  
cific and first-appearing phosphorylated compound in the MI  
catabolic pathway (Fig. 1), it may be more distinguishable than  
the other intermediates. IolR may, thus, have evolved to recog-  
nize the level of DKGP as the induction signal to sense the  
availability of MI as carbon source. In these studies we showed  
that the next successive step was the aldolase reaction catalyzed  
by IolJ, thereby converting DKGP into DHAP and MSA. The  
IolJ aldolase reaction might be a rate-limiting step in the path-  
way, and when enough MI was supplied, its substrate DKGP  
could be quickly accumulated to make IolR recognize the  
induction signal. DHAP is a known glycolytic intermediate and  
is, thus, readily metabolized through the glycolysis pathway,  
whereas MSA must be processed further in a specific way by  
IolA MSA dehydrogenase (16). Upon the addition of MI into  
the growth medium, we witnessed that a *B. subtilis* mutant  
defective in *iolA* started to die (data not shown),³ suggesting  
that some toxic effects might be provoked by MSA accumu-  
lated in the cell.

³ K. Yoshida, unpublished observation.

---

Two *iol* genes, *iolH* and *iolS*, have remained uncharacterized.  
IolH is paralogous to both IolI and IolE. The *iolI* gene product  
was shown to be 2KMI/1-keto-DCI isomerase involved in DCI  
metabolism (14), and IolE is known as 2KMI dehydratase indis-  
pensable for MI catabolism (13). This suggests that IolH may  
also act on a certain ketose as its isomerase or dehydratase. It is  
unlikely, however, that IolH serves as another 2KMI dehy-  
dratase because the *iolE*-defective mutant did not possess this  
enzyme activity at all and, thus, could not grow on inositol  
(13). Finally, IolS was reported to be homologous to pyri-  
doxal reductase of *Schizosaccharomyces pombe* (28). The  
corresponding gene was cloned and expressed in *E. coli*, and  
this actually showed some pyridoxal reductase activity. This  
fact may imply a possible relationship between the MI and  
vitamin B6 metabolism, which would require further studies  
to be elucidated.

Acknowledgments—We thank Ken-ichi Tsurusaki for expert assist-  
ance with ESI-TOF mass analysis, Kei Asai for providing pDGHisC,  
Hideki Ikeda, Hideko Ikeda, Takahiro Ogawa, Kaoru Omae, and  
Akiko Tsukuda for technical assistance, Sierd Bron for a critical  
review of the manuscript and valuable advice, and Gerrit Poelarends  
for the generous gift of materials and inspiring discussion.

---

**REFERENCES**

1. Yoshida, K., Aoyama, D., Ishio, I., Shibayama, T., and Fujita, Y. (1997) *J. Bacteriol.* **179**, 4591–4598  
2. Vidal-Leiria, M., and van Uden, N. (1973) *Biochim. Biophys. Acta* **293**, 295–303  
3. Berman, T., and Magasanik, B. (1966) *J. Biol. Chem.* **241**, 800–806  
4. Poole, P. S., Blyth, A., Reid, C. J., and Walters, K. (1994) *Microbiology* **140**, 2787–2795  
5. Galbraith, M. P., Feng, S. F., Borneman, J., Triplett, E. W., de Bruijn, F. J., and Rossbach, S. (1998) *Microbiology* **144**, 2915–2924  
6. Jiang, G., Krishnan, A. H., Kim, Y.-M., Wacek, T. J., and Krishnan, H. B. (2001) *J. Bacteriol.* **183**, 2595–2604  
7. Krings, E., Krumbach, K., Bathe, B., Kelle, R., Wendisch, V. F., Sahm, H., and Eggeling, L. (2006) *J. Bacteriol.* **188**, 8054–8061  
8. Yebra, M. J., Zuniga, M., Beaufils, S., Perez-Martinez, G., Deutscher, J., and Monedero, V. (2007) *Appl. Environ. Microbiol.* **73**, 3850–3858  
9. Anderson, W. A., and Magasanik, B. (1971) *J. Biol. Chem.* **246**, 5662–5675  
10. Fujita, Y., Shindo, K., Miwa, Y., and Yoshida, K. (1991) *Gene (Amst.)* **108**, 121–125  
11. Yoshida, K., Shibayama, T., Aoyama, D., and Fujita, Y. (1999) *J. Mol. Biol.* **285**, 917–929  
12. Yoshida, K., Yamamoto, Y., Omae, K., Yamamoto, M., and Fujita, Y. (2002) *J. Bacteriol.* **184**, 983–991  
13. Yoshida, K., Yamaguchi, M., Ikeda, H., Omae, K., Tsurusaki, K., and Fujita, Y. (2004) *Microbiology* **150**, 571–580  
14. Yoshida, K., Yamaguchi, M., Morinaga, T., Ikeuchi, M., Kinehara, M., and Ashida, H. (2006) *Appl. Environ. Microbiol.* **72**, 1310–1315  
15. Morinaga, T., Yamaguchi, M., Makino, Y., Nanamiya, H., Takahashi, K., Yoshikawa, H., Kawamura, F., Ashida, H., and Yoshida, K. (2006) *Biosci. Biotechnol. Biochem.* **70**, 1913–1920  
16. Stines-Chaumeil, C., Talfournier, F., and Branlant, G. (2006) *Biochem. J.* **395**, 107–115  
17. Fujita, Y., and Fujita, T. (1983) *J. Bacteriol.* **154**, 864–869  
18. Fujita, Y., Yoshida, K., Miwa, Y., Yanai, N., Nagakawa, E., and Kasahara, Y. (1998) *J. Bacteriol.* **180**, 4309–4313  
19. Fujita, Y., and Freese, E. (1981) *J. Bacteriol.* **145**, 760–767  
20. Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) *Gene (Amst.)* **33**, 103–119  
B. subtilis Inositol Catabolism

21. Sambrook, J., and Russell, D. W. (2001) *Molecular Cloning: A Laboratory Manual*, 3rd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
22. Kaempfer, R. O., and Magasanik, B. (1967) *J. Mol. Biol.* 27, 453–468
23. Stragier, P., Bonamy, C., and Karmazyn-Campelli, C. (1988) *Cell* 52, 697–704
24. Nishihara, K., Kanemori, M., Kitagawa, M., Yanagi, H., and Yura, T. (1998) *Appl. Environ. Microbiol.* 64, 1694–1699
25. Berman, T., and Magasanik, B. (1966) *J. Biol. Chem.* 241, 807–813
26. Anderson, W. A., and Magasanik, B. (1971) *J. Biol. Chem.* 246, 5653–5661
27. van Rooijen, R. J., and de Vos, W. M. (1990) *J. Biol. Chem.* 265, 18499–18503
28. Nakano, M., Morita, T., Yamamoto, T., Sano, H., Ashiuchi, M., Masui, R., Kuramitsu, S., and Yagi, T. (1999) *J. Biol. Chem.* 274, 23185–23190

Metabolism and Bioenergetics:  
**myo-Inositol Catabolism in Bacillus subtilis**

Ken-ichi Yoshida, Masanori Yamaguchi,  
Tetsuro Morinaga, Masaki Kinehara, Maya  
Ikeuchi, Hitoshi Ashida and Yasutaro Fujita  
*J. Biol. Chem.* 2008, 283:10415-10424.  
doi: 10.1074/jbc.M708043200 originally published online February 28, 2008

---

Access the most updated version of this article at doi: [10.1074/jbc.M708043200](http://dx.doi.org/10.1074/jbc.M708043200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- **When this article is cited**
- **When a correction for this article is posted**

[Click here](https://www.jbc.org/site/alerts) to choose from all of JBC's e-mail alerts

This article cites 27 references, 20 of which can be accessed free at  
[http://www.jbc.org/content/283/16/10415.full.html#ref-list-1](http://www.jbc.org/content/283/16/10415.full.html#ref-list-1)
