
Environmental Microbiology (2008) 10(6), 1547–1556

doi:10.1111/j.1462-2920.2008.01570.x

# 6-Oxocyclohex-1-ene-1-carbonyl-coenzyme A hydrolases from obligately anaerobic bacteria: characterization and identification of its gene as a functional marker for aromatic compounds degrading anaerobes

Kevin Kuntze,$^{1}$ Yoshifumi Shinoda,$^{2}$ Housna Moutakki,$^{3}$ Michael J. McInerney,$^{3}$ Carsten Vogt,$^{4}$ Hans-Hermann Richnow$^{4}$ and Matthias Boll$^{1*}$

$^{1}$ Institute of Biochemistry, University of Leipzig, Leipzig, Germany.

$^{2}$ Department of Environmental Microbiology, Gakuen University, Kyoto, Japan.

$^{3}$ Department of Botany and Microbiology, University of Oklahoma, Norman, OK, USA.

$^{4}$ Department of Isotope Biogeochemistry, Helmholtz Centre for Environmental Research, Leipzig, Germany.

## Summary

In anaerobic bacteria, most aromatic growth substrates are channelled into the benzoyl-coenzyme A (CoA) degradation pathway where the aromatic ring is dearomatized and cleaved into an aliphatic thiol ester. The initial step of this pathway is catalysed by dearomatizing benzoyl-CoA reductases yielding the two electron-reduction product, cyclohexa-1,5-diene-1-carbonyl-CoA, to which water is subsequently added by a hydratase. The next two steps have so far only been studied in facultative anaerobes and comprise the oxidation of the 6-hydroxyl-group to 6-oxocyclohex-1-ene-1-carbonyl-CoA (6-OCH-CoA), the addition of water and hydrolytic ring cleavage yielding 3-hydroxypimelyl-CoA. In this work, two benzoate-induced genes from the obligately anaerobic bacteria, *Geobacter metallireducens* (*bamA*$_{Geo}$) and *Syntrophus aciditrophicus* (*bamA*$_{Syn}$), were heterologously expressed in *Escherichia coli*, purified and characterized as 6-OCH-CoA hydrolases. Both enzymes consisted of a single 43 kDa subunit. Some properties of the enzymes are presented and compared with homologues from facultative anaerobes.

An alignment of the nucleotide sequences of *bamA*$_{Geo}$ and *bamA*$_{Syn}$ with the corresponding genes from facultative anaerobes identified highly conserved DNA regions, which enabled the discrimination of genes coding for 6-OCH-CoA hydrolases from those coding for related enzymes. A degenerate oligonucleotide primer pair was deduced from conserved regions and applied in polymerase chain reaction reactions. Using these primers, the expected DNA fragment of the 6-OCH-CoA hydrolase genes was specifically amplified from the DNA of nearly all known facultative and obligate anaerobes that use aromatic growth substrates. The only exception was the aromatic compound-degrading *Rhodopseudomonas palustris*, which uniquely uses a modified benzoyl-CoA degradation pathway. Using the oligonucleotide primers, the expected DNA fragment was also amplified in a toluene-degrading and a *m*-xylene-degrading enrichment culture demonstrating its potential use in less defined bacterial communities. The gene probe established in this work provides for the first time a general tool for the detection of a central functionality in aromatic compound-degrading anaerobes.

## Introduction

In the last decade, the number of isolated and characterized anaerobic bacteria with the capacity to use aromatic growth substrates has continuously increased. Meanwhile, the important role that these bacteria play in the overall carbon cycle and in the bioremediation of aromatic xenobiotics is now clearly recognized. In anaerobic bacteria, the oxygen-dependent processes involved in the aerobic aromatic metabolism are completely replaced by a set of unprecedented enzymatic reactions, many of which follow unique biochemical principles. In the last several years, many degradation pathways for anaerobic aromatic metabolism have been elucidated and some of the key enzymes have been characterized in detail (Schink *et al.*, 2000; Boll *et al.*, 2002; Gibson and Harwood, 2002; Diaz, 2004; Boll and Fuchs, 2005).

Received 9 September, 2007; accepted 21 December, 2007. *For correspondence. E-mail boll@uni-leipzig.de; Tel. (+49) 341 9736996; Fax (+49) 341 9736910.*

© 2008 The Authors  
Journal compilation © 2008 Society for Applied Microbiology and Blackwell Publishing Ltd

1548 K. Kuntze et al.

In all anaerobic bacteria, most aromatic growth substrates (e.g. toluene, phenol, cresols, xylenes, ethylbenzene, benzoate analogues, etc.) are channelled to the central intermediate benzoyl-coenzyme A (CoA) prior to dearomatization and ring cleavage; the only exceptions are aromatic compounds with at least two meta-positioned hydroxyl groups (Schink *et al.*, 2000; Boll, 2005). In facultative anaerobes such as the denitrifying model organism *Thauera aromatica*, benzoyl-CoA is further metabolized by the so-called benzoyl-CoA degradation pathway resulting in dearomatization and cleavage of the ring. This pathway comprises the following four enzymatic steps (Fig. 1): (i) the ATP-dependent, reductive dearomatization of benzoyl-CoA to cyclohexa-1,5-diene-1-carbonyl-CoA (dienoyl-CoA, compound 2 in Fig. 1) catalysed by benzoyl-CoA reductase (Boll and Fuchs, 1995; Boll *et al.*, 2000), (ii) the addition of water by a dienoyl-CoA hydratase forming 6-hydroxycyclohex-1-ene-1-carbonyl-CoA (6-OH-cyclohexenoyl-CoA, compound 3) (Laempe *et al.*, 1998), (iii) the oxidation of the hydroxyl group to 6-oxocyclohex-1-ene-1-carbonyl-CoA (6-OCH-CoA, compound 4) by an alcohol dehydrogenase, and finally (iv) hydrolytic ring cleavage and the addition of a water molecule to the double bond yielding 6-hydroxypimelyl-CoA (Laempe *et al.*, 1999). The latter two partial reactions are probably both catalysed by the 6-OCH-CoA hydrolase, also referred to as the ring-opening hydrolase.

Amino acid sequence comparisons of the corresponding gene products suggested that enzymes highly similar to those of the benzoyl-CoA pathway in *T. aromatica* are present in most other facultative anaerobes that use aromatic growth substrates such as species in the genera *Thauera*, *Azoarcus* and *Magnetospirillum* (Gibson and Harwood, 2002; Lopez Barragan *et al.*, 2004a). The only known exception is *Rhodopseudomonas palustris*, where benzoyl-CoA is reduced twice by two electrons to a cyclic monoenoyl-CoA compound (Gibson and Gibson, 1992; Gibson and Harwood, 2002). As a consequence, the enzymes involved in the subsequent reactions involved in benzoyl-CoA degradation in *R. palustris* are less similar to those from other facultative anaerobes and cannot be uniquely identified by their amino acid sequence (Egland *et al.*, 1997). This expectation will not always be mentioned in the following.

Recently, insights into the metabolism of aromatic compounds in obligate anaerobes were obtained. Studies with the Fe(III)-respiring *Geobacter metallireducens* (Wischgoll *et al.*, 2005) and the sulfate-reducing *Desulfovibrio multivorans* (Peters *et al.*, 2004) indicated that the benzoyl-CoA is reduced by a completely different type of dearomatizing enzyme that contains Mo- and Se-protein components. However, in spite of the different types of benzoyl-CoA reductases in obligate anaerobes, deduced amino acid sequences of further putative enzymes of the benzoyl-CoA degradation pathway suggested that the product of the dearomatization process, dienoyl-CoA, was the same in both facultative and obligate anaerobes. This suggestion was confirmed by the characterization of highly specific dienoyl-CoA hydratases (catalysing step II in Fig. 1) in *G. metallireducens* and in the fermenting, *Syntrophus aciditrophicus* (Peters *et al.*, 2007).

The presence of highly similar dienoyl-CoA hydratases in both facultative and obligate anaerobes suggests that all enzymatic reactions involved in the conversion from dienoyl-CoA to the first aliphatic product 3-hydroxypimelyl-CoA (Fig. 1, steps II–IV) are identical in facultative and obligate anaerobes that utilize aromatic growth substrates. To test this hypothesis, we heterologously expressed two benzoate-induced genes putatively coding for 6-OCH-CoA hydrolases from the obligately anaerobic Deltaproteobacterium, *G. metallireducens* (referred to as *bamA*<sub>Geo</sub>) and *S. aciditrophicus* (referred to as *bamA*<sub>Syn</sub>) in *Escherichia coli* and identified the products as 6-OCH-CoA hydrolases. The genes coding for 6-OCH-CoA hydrolases were found to be highly conserved in all anaerobic bacteria that use aromatic growth substrates. A set of degenerate oligonucleotide gene primers were developed that specifically amplified the gene of 6-OCH-CoA hydrolases in polymerase chain reactions (PCR) in

![Figure 1](#fig1)

Fig. 1. Reactions of the benzoyl-CoA degradation pathway. The reactions shown are catalysed by (I) benzoyl-CoA reductase, (II) cyclohexa-1,5-diene-1-carbonyl-CoA hydratase, (III) 6-hydroxycyclohex-1-ene-1-carbonyl-CoA dehydrogenase and (IV) 6-oxocyclohex-1-ene-1-carbonyl-CoA hydrolase (6-OCH-CoA hydrolase).

© 2008 The Authors  
Journal compilation © 2008 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, **10**, 1547–1556

nearly all known anaerobic bacteria with the capacity to degrade aromatic growth substrates and in a toluene- and *m*-xylene-degrading, sulfate-reducing enrichment culture.

**Results**

**Heterologous expression of *bamA*<sub>Geo</sub> and *bamA*<sub>Syn</sub>**

The benzoate-induced *bamA*<sub>Geo</sub> gene (gi 78223295) of *G. metallireducens*, annotated as an enoyl-CoA hydratase/isomerase, was suggested to code for a putative 6-OCH-CoA hydrolase (Wischgoll et al., 2005). The deduced amino acid sequence of an open reading frame of the fermenting, benzoate-degrading *S. aciditrophicus* strain (gi 85860871) is annotated as an enoyl-CoA hydratase/isomerase family protein (McInerney et al., 2007). The corresponding deduced amino acid sequence of BamA<sub>Geo</sub> shows 49% amino acid identity to the BamA<sub>Syn</sub> gene product, 75% amino acid identity to 6-OCH-CoA hydrolase in *T. aromatica* encoded by *oah*, and 50–52% amino acid identity to corresponding gene product in *Azoarcus* species. The genes *bamA*<sub>Geo</sub> and *bamA*<sub>Syn</sub> were heterologously expressed in *E. coli* with an N-terminal 6× His-Tag; after completion of expression a 6-OCH-hydrolase activity was found in the soluble protein fraction. The soluble gene products obtained from these expressions were used for further studies. On SDS-polyacrylamide gels, the overexpressed proteins had masses of 43 kDa corresponding to the predicted masses derived from the gene sequences (41.9 kDa for *bamA*<sub>Geo</sub> and 43.1 kDa for *bamA*<sub>Syn</sub>; Fig. 2).

| kDa | M | 1 | 2 | 3 | 4 |
|-----|---|---|---|---|---|
| 116 |   |   |   |   |   |
| 66.2|   |   |   |   |   |
| 45  |   |   |   |   |   |
| 35  |   |   |   |   |   |
| 25  |   |   |   |   |   |

Fig. 2. Purification of His-tagged BamA from *Geobacter metallireducens* and *Syntrophus aciditrophicus* after heterologous expression in *Escherichia coli*. Cell extracts from *E. coli* containing overexpressed BamA<sub>Geo</sub> (lane 1) or BamA<sub>Syn</sub> (lane 3). Enzymes purified after Ni-chelating chromatography: BamA<sub>Geo</sub> (lane 2) and BamA<sub>Syn</sub> (lane 4).

© 2008 The Authors  
Journal compilation © 2008 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, **10**, 1547–1556

**Gene probe for ring-opening hydrolases 1549**

**Purification and characterization of *bamA*<sub>Geo</sub> and *bamA*<sub>Syn</sub> gene products**

BamA<sub>Geo</sub> and BamA<sub>Syn</sub> were purified by Ni-chelating affinity chromatography. Both proteins eluted at an imidazole concentration between 50 and 200 mM with high purity (> 98%) (Fig. 2). The native molecular masses determined by gel filtration were 240 ± 15 kDa for BamA<sub>Geo</sub> and 265 ± 15 kDa for BamA<sub>Syn</sub>. These data suggest an α<sub>6</sub> subunit composition of both BamA<sub>Geo</sub> and BamA<sub>Syn</sub>.

The activities of recombinant BamA<sub>Geo</sub> and BamA<sub>Syn</sub> were determined by a discontinuous assay. For this purpose 6-OCH-CoA was synthesized from benzoyl-CoA by the combined action of purified benzoyl-CoA reductase, dienoyl-CoA hydratase and 6-hydroxycyclohexenoyl-CoA dehydrogenase (catalysing reactions I–III in Fig. 1). The latter two enzymes were heterologously produced in *E. coli* by expressing the *bamR* (hydratase) and *bamQ* (dehydrogenase) genes from *G. metallireducens* (Wischgoll et al., 2005; Peters et al., 2007). Both BamA<sub>Geo</sub> and BamA<sub>Syn</sub> converted 6-OCH-CoA to 3-hydroxypimelyl-CoA without accumulation of a reaction intermediate (reaction IV in Fig. 1), with specific activities of 2.0 ± 0.3 μmol min<sup>−1</sup> mg<sup>−1</sup> (BamA<sub>Geo</sub>) and 0.5 ± 0.1 μmol min<sup>−1</sup> mg<sup>−1</sup> (BamA<sub>Syn</sub>) respectively (Fig. 3). Due to the difficult enzymatic synthesis of the substrate and the use of a discontinuous assay, the *K*<sub>m</sub> values for the enzymes could only be roughly estimated by fitting the kinetic data to the Michaelis–Menten equation. The apparent *K*<sub>m</sub> values for 6-OCH-CoA determined were 70 ± 25 μM for BamA<sub>Geo</sub> and 115 ± 30 μM for BamA<sub>Syn</sub>.

*Syntrophus aciditrophicus* is able to grow on crotonate either in the presence or in the absence of a H<sub>2</sub>-consuming organism. In the latter case, a fraction of crotonate is oxidized to acetate, thereby forming ATP, whereas the remaining portion is used to reoxidize reduced cofactors by a reversal of β-oxidation reactions including the carboxylation of crotonyl-CoA and acetyl-CoA condensation steps. In the course of such reactions, cyclohexane carboxylate and even some benzoate are formed from crotonate, which requires cyclohexane ring formation and aromatization reactions (Mouttaki et al., 2007). The possibility that BamA<sub>Syn</sub> catalyses cyclohexane ring formation was tested by using pimelyl-CoA or 3-hydroxypimelyl-CoA as substrates (0.2 mM each). No consumption of either substrate was detected by high-performance liquid chromatography (HPLC) analysis.

**Phylogenetic analysis of BamA proteins**

A BLAST analysis of BamA<sub>Geo</sub> revealed highly similar open reading frames in all anaerobic bacteria that use aromatic growth substrates (49–77% amino acid sequence identity).

1550 K. Kuntze et al.

A

|  |  |
| --- | --- |
| 0,08 |  |
| 0,06 |  |
| 0,04 |  |
| 0,02 |  |
| 0 |  |
|  | 0 5 10 15 |
|  | Time (min) |

B

|  |  |
| --- | --- |
| 0,08 |  |
| 0,06 |  |
| 0,04 |  |
| 0,02 |  |
| 0 |  |
|  | 0 5 10 15 |
|  | Time (min) |

C

|  |  |
| --- | --- |
| 0,02 |  |
| 0,015 |  |
| 0,01 |  |
| 0,005 |  |
| 0 |  |
|  | 210 260 310 360 |
|  | nm |

D

|  |  |
| --- | --- |
| 0,06 |  |
| 0,05 |  |
| 0,04 |  |
| 0,03 |  |
| 0,02 |  |
| 0,01 |  |
| 0 |  |
|  | 210 260 310 360 |
|  | nm |

Fig. 3. High-performance liquid chromatography (HPLC) analysis of 6-OCH-CoA hydrolase activity. The 6-OCH-CoA was enzymatically synthesized from benzoyl-CoA by purified enzymes of the benzoyl-CoA degradation pathway (reaction I–III, Fig. 1). HPLC chromatograms showing substrate consumption and product formation after 0.1 min (A) and 10 min (B) of incubation. (C) and (D) show typical UV-visible spectra of the components indicated in (A) and (B). The UV spectra are identical to those for 6-OCH-CoA (A and C) and 3-hydroxypimelyl-CoA (B and D) as previously reported (Laempe *et al.*, 1999). The large peak eluting at approximately 7 min represents NAD⁺, which was used in the assay to form 6-OCH-CoA from 2-OH-cyclohexenoyl-CoA.

ity) with the exception of *R. palustris*. All deduced BamA gene products were part of a gene cluster that contained the genes of the upper benzoyl-CoA degradation pathway (reactions I–IV in Fig. 1) and therefore are very likely 6-OCH-CoA hydrolases. In contrast, related enzymes annotated as enoyl-CoA hydratases/isomerases from bacteria that cannot degrade aromatic compounds under anaerobic conditions were less similar (< 35% sequence identity). Dihydroxynaphthoate synthases play a role in menaquinone synthesis by forming the second ring with o-succinylbenzoate as the substrate (Meganathan, 2001). Phylogenetic analysis (Fig. 4) showed that BamA-like proteins group into two subclusters of the Geobacter–Thauera type and the Azoarcus–Syntrophus type.

### Construction and application of a gene probe for 6-OCH-CoA hydrolases

A more careful comparison of the amino acid sequences of 6-OCH-CoA hydrolases and related enoyl-CoA hydratases and dihydroxynaphthoate synthases revealed two regions that were 100% conserved in all 6-OCH-CoA hydrolases but not in other related enzymes (Fig. 5A). A high degree of similarity was even observed at the nucleotide level for these regions (Fig. 5B). This finding allowed the construction of degenerate oligonucleotide primers that matched the highly conserved nucleotide regions in the *bamA* genes. Using these primers an approximately 300 bp nucleotide sequence within the *bamA* gene was amplified in PCR reactions. Chromosomal DNA was isolated from diverse anaerobic bacteria that can use aromatic growth substrates including *T. aromatica*, *Azoarcus evansii*, *Azoarcus EbN-1*, *G. metallireducens*, *R. palustris*, *S. aciditrophicus*, *Magnetospirillum* species TS-6 and CC-26, and *D. multivorans*. The negative controls were DNA from *Geobacter sulfurreducens* and *E. coli*, neither of which can grow with aromatic compounds under anaerobic conditions.

Among several oligonucleotide primer pairs tested, *BamA*-SP9/*BamA*-ASP1 (Fig. 5C) proved to be the most

---

Fig. 4. Tree of 6-OCH hydrolases (right, those with amino acid sequence identities to *BamA*ₐₑₒ > 50%) and deduced dihydroxynaphthoate synthases/enoyl-CoA hydratases from aerobic organisms (sequence identities to *BamA*ₐₑₒ < 40%). A., *Azoarcus*; M., *Magnetospirillum*; G., *Geobacter*.

|  |  |
| --- | --- |
| Rhodococcus sp. RHA1 |  |
| Xanthobacter autotrophicus Py2 |  |
| Ralstonia eutropha H16 |  |
| Rhodopseudomonas palustris BisB18 |  |
| Nocardioides sp. JS614 |  |
| Corynebacterium efficiens YS-314 |  |
|  | BamAₛᵥₙ |
|  | A. sp. EbN1 |
|  | A. sp. CIB |
|  | A. evansii |
|  | M. magneticum AMB-1 |
|  | M. magnetotacticum MS-1 |
|  | T. aromatica |
|  | BamAₐₑₒ |
|  | G. sp. FRC-32 |
|  | G. bemidjiensis Bem |

© 2008 The Authors  
Journal compilation © 2008 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, **10**, 1547–1556

Gene probe for ring-opening hydrolases 1551

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
| A |  |  |  |  |
| G. met | EGLYN..AWIIFNNPRQNSYTTEMVKATILAFRRASVDR | NGMRIGGGQEIGMACDFSIAQDLAKFGQAG |  |  |
| G. bem | QGLYN..AWIIFNNPKQFNSYTTDMMVKATILAFRRASVDR | NGMRIGGGQEIGMACDFSVAHDLANFGQAG |  |  |
| G. FRC | EGIYN..AWIIFNNPKQNSYTTEMVKATILAFRRASADR | NGMRIGGGQEIGMACDFSIAQDLANFGQAG |  |  |
| T. aro | PGLYN..VWISLDNPKQNSYTTDMMVKGLILAFRAASCAR | NGMRIGGGQEIGMAADFTVAQDLANFGQAG |  |  |
| M. AMB | EGLYN..AWITLDNQKQNSYTTDMMVKGVIMAFRDASNAR | NGMRIGGGQEIGMAADFSVAQDLAKFGQAG |  |  |
| M. MS1 | EGLYN..AWITLDNQKQNSYTTDMMVKGVIMAFRDASNAR | NGMRIGGGQEIGMAADFSVAQDLAKFGQAG |  |  |
| A. CIB | PGLYA..AWIIILNNPTQNSYTTEMVKAVIAGFQRASSDR | NGMRVGGGQEIGMATDLTVTSDMAVFGQAG |  |  |
| A. Ebn | PGLYA..AWIILNNPKQNSYTTEMVKAIIAGFQRASSDR | NGMRVGGGQEIGMATDLTITSDMAIFGQAG |  |  |
| A. eva | PGLYS..AWITLNNPTQNSYTTEMVKAVIAGFQRASSDR | NGMRVGGGQEIGMATDLTITSMAVFGQAG |  |  |
| S. aci | PGLYS..AWIRLNNPGQNSYTTEMVKGVIAGFENSSTDTR | NGMRVAGGQEIGTATDITVSSDLAIFGQAG |  |  |
| R. RHA | YSAQNGVARITINRPEKYNAFREEETLDDLIAAFSEAEADT | KGYAVGGGNEMQMLCDLTLASDDSIFGQSG |  |  |
| X. aut | YEVRGGAAWITINRPDKYNAFRGHTVDELIHAFQTAGWDR | NGFAIGGGNVLATICDLTIASDKAQFGQVG |  |  |
| R. eut | YEVRNGAAWILINRPEKMNAFRGRTCDELIHAINRAGYDR | QGYAVGGGNVLCTICDFTIASDKAVFGQVG |  |  |
| R. pal | YDIRNGVAWIIIINRPDKMNAFRGTTCDELIKALYKAGYDK | QGYAIGGGNVLATICDLTICSEKAIFGQVG |  |  |
| N. JS | YVVEDSAAIITISRPERYNAFRAKTVDDELIKAFRLAWADR | NGIAVGGGHVLHVLCDVSIASENARFGQAG |  |  |
| C. eff | YERRNSAAVITINRPQRYNAFRAQTVEELIKAFKMAWADN | NGIAIGGGHV LHVLCDLTIAASTARFGQSG |  |  |

|  | Q | Y/F | N | S | Y | T | T | E/D |  | G | Q | E | I | G | M/T | A |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| B |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| G. met | CAG | TAC | AAC | TCC | TAC | ACC | ACT | GAC | ... | GGC | CAG | GAA | ATC | GGC | ATG | GCC |
| G. bem | CAG | TTC | AAC | TCC | TAT | ACC | ACC | GAC | ... | GGC | CAG | GAG | ATC | GGC | ATG | GCC |
| G. FRC | CAA | TAC | AAC | TCC | TAC | ACG | ACG | GAA | ... | GGC | CAG | GAA | ATC | GGC | ATG | GCC |
| T. aro | CAG | TAC | AAC | TCC | TAC | ACC | ACC | GAT | ... | GGT | CAG | GAG | ATC | GGT | ATG | GCG |
| M. AMB | CAG | TAC | AAT | TCC | TAC | ACC | ACC | GAC | ... | GGC | CAG | GAA | ATC | GGC | ATG | GCC |
| M. MS1 | CAG | TAC | AAT | TCC | TAC | ACC | ACC | GAC | ... | GGC | CAG | GAA | ATC | GGC | ATG | GCC |
| A. CIB | CAG | TAC | AAC | TCC | TAC | ACC | ACC | GAG | ... | GGT | CAG | GAA | ATC | GGC | ATG | GCG |
| A. Ebn | CAG | TAC | AAC | TCC | TAC | ACC | ACC | GAA | ... | GGC | CAG | GAA | ATC | GGC | ATG | GCG |
| A. eva | CAG | TAC | AAC | TCC | TAC | ACC | ACC | GAG | ... | GGC | CAG | GAA | ATC | GGC | ATG | GCG |
| S. aci | CAG | TAC | AAT | TCT | TAT | ACA | ACC | GAG | ... | GGT | CAG | GAA | ATC | GGC | ACT | GCC |

|  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
1552 K. Kuntze et al.

a study with the enzyme from *T. aromatica*, which was suggested to introduce two water molecules into its substrate without the release of an intermediate (Laempe *et al.*, 1999). One water molecule is added to the double bond of the 6-hydroxycyclohexane ring (enoyl-CoA hydratase-like activity), and another water molecule is involved in hydrolytic ring cleavage (hydrolase activity). Though unlikely, Laempe and colleagues (1999) did not completely rule out that the 6-OCH-CoA hydrolase preparations contained an enoyl-CoA hydratase impurity catalysing the addition of water to the double bond. With the heterologous expression of active *bamA* genes from two obligate anaerobes and the identification of the reaction product of the purified 6-OCH-CoA hydrolases reported, there remains now little doubt that both of the water addition reactions are catalysed by a single enzyme. The order of the two partial reactions is not known due to the unavailability of the potential reaction intermediate, 2-oxo-6-hydroxycyclohexane-1-carbonyl-CoA.

All 6-OCH-CoA hydrolases are distinguished from their closest related homologues, dihydroxynaphthoate synthases, by a large C-terminal extension of about 160 amino acids, which results in a much higher molecular mass of the subunit (approximately 43 kDa) in comparison with that of other enoyl-CoA hydratases or naphthoate synthases (25–30 kDa). The function of this extension is not known. There are no amino acid sequences similarities in this extension to any other known enzyme type. As no reaction intermediates during the two partial reactions are released, the presence of a second catalytic centre in this region is rather unlikely.

The catalytic properties (*v*<sub>max</sub> and *K*<sub>m</sub>) of the enzymes from *T. aromatica*, *G. metallireducens* and *S. aciditrophicus* appear to be similar, with the latter exhibiting a slightly decreased catalytic efficiency. The native masses of the BamA<sub>Geo</sub> and BamA<sub>Syn</sub> are clearly larger than that of the enzyme from *T. aromatica*, suggesting an *a*<sub>6</sub> rather than an *a*<sub>4</sub> composition of the *T. aromatica* enzyme (Laempe *et al.*, 1999).

Fig. 6. Polymerase chain reaction amplification of *bamA* genes from different bacteria with a degenerate *bamA*-oligonucleotide gene probe. Notably, in *Rhodopseudomonas palustris* benzoyl-CoA is reduced twice to cyclohexenoyl-CoA. As a consequence, the subsequent steps differ from the benzoyl-CoA pathway.

contaminated aquifer underlying a former coal hydrogenation and benzene production plant near Zeitz, Germany (Schirmer *et al.*, 2006). Chromosomal DNA isolated from the three cultures gave in all cases an amplicon of the expected size. With *D. multivorans* and the *m*-xylene-degrading enrichment culture, a single DNA sequence was obtained and deposited at the GenBank (Accession No. EU268441 and EU268442). In case of the toluene-degrading enrichment culture a mixture of DNA sequences was obtained suggesting that the culture consisted of several different, *bamA* gene-containing species. A phylogenetic analysis of the DNA sequences obtained indicated that *bamA* genes from sulfate-reducing bacteria group into the *Azoarcus/Syntrophus* cluster (Fig. 7).

Discussion

Properties of ring-opening 6-OCH hydrolases

In this work, 6-OCH-CoA hydrolases have been characterized for the first time from obligate anaerobes. Due to the lack of a commercial source for the substrate and the difficulties of its enzymatic synthesis, the initial characterization was limited to only a few properties. So far, all information about 6-OCH-CoA hydrolases is derived from

Fig. 7. Phylogenetic tree of DNA fragments obtained (300 bp) using the *bamA* gene probe as oligonucleotide primer in PCR reactions. *D.*, *Desulfovibrio*.

Role of BamA<sub>syn</sub>

The intriguing observation that S. aciditrophicus produces cyclohexane carboxylate and benzoate when grown in pure culture on crotonate requires a reversal of the β-oxidation reactions and a ring-forming enzyme (Moutaki *et al.*, 2007). The homologue dihydroxynaphthoate synthase catalyses such a ring formation reaction albeit at the expense of CoA thiol ester hydrolysis. The possibility that BamA<sub>syn</sub> catalyses a homologous reaction can now be ruled out and suggests that BamA<sub>syn</sub> is only involved in benzoyl-CoA degradation. There are other highly similar copies of *bamA*-like genes in the genome of S. aciditrophicus; their products might catalyse ring formation from 3-hydroxypimelyl-CoA or pimelyl-CoA (McInerney *et al.*, 2007). Other scenarios would involve ring formation from 3-oxopimelyl-CoA or from a pimelyl-di-CoA thiol ester.

Molecular detection of anaerobic bacteria that degrade aromatic compounds

The presence of 6-OCH-CoA hydrolases in nearly all tested anaerobes confirms that the benzoyl-CoA degradation pathway (Fig. 1) is present in all anaerobes that use aromatic growth substrates. In a recent study, we suggested that the gene coding for dienoyl-CoA hydratase (*bamR*) of the benzoyl-CoA pathway might serve as a potential target for gene probes (Peters *et al.*, 2007). However, the similarities at the nucleotide level were not sufficient to use *bamR* as a target for PCR amplification (not shown). In contrast, *bamA* genes [referred to as *oah* in *T. aromatica* or *bzdY* in *Azoarcus* strain CIB (Lopez Barragan *et al.*, 2004b)] have regions with remarkable nucleotide sequence conservation, which can be used for gene probes or primer binding.

There are few instances where molecular approaches have been used to detect characteristic genes coding for enzymes involved in anaerobic aromatic metabolism. These studies have been limited to the detection of genes from bacteria that grow on toluene (Beller *et al.*, 2002; Winderl *et al.*, 2007) or to facultative anaerobes containing ATP-dependent benzoyl-CoA reductases (Hosoda *et al.*, 2005; Song and Ward, 2005). The *bssA* gene coding for the large subunit of benzylsuccinate synthase, a key enzyme in anaerobic toluene metabolism (Leuthner *et al.*, 1998), is highly conserved and gene probes have successfully been applied for enrichment cultures and environmental samples. In contrast, studies with gene probes for the four genes of ATP-dependent benzoyl-CoA reductases were complicated by several difficulties. First, there are two types of benzoyl-CoA reductases, the Azoarcus or *bzd* type and the Thauera or *bcr* type (Boll, 2005; Song and Ward, 2005). Amino acid sequence identities between the two types are rather low, which made it impossible to establish a single degenerate oligonucleotide probe to detect both types of BCR enzymes. Second, in general, obligate anaerobes such as iron-reducing, sulfate-reducing or fermenting bacteria do not contain the ATP-dependent benzoyl-CoA reductases, but rather a totally different Mo- and Se-containing benzoyl-CoA-dearomatizing enzyme complex (Peters *et al.*, 2004; Wischgoll *et al.*, 2005). Thus, gene probes for ATP-dependent benzoyl-CoA reductases can at best only detect the corresponding genes from facultative anaerobes. The third difficulty is the similarity in the amino acid sequence between ATP-dependent benzoyl-CoA reductases and ATP-dependent 2-hydroxylacyl-CoA dehydratases. The latter are involved in amino acid fermentation in *Clostridia* but not in anaerobic aromatic metabolism (Kim *et al.*, 2004). The ATP-binding α- and δ-subunits of Azoarcus-type BCRs share higher amino acid sequence similarities with the ATP-binding subunits of 2-hydroxylacyl-CoA dehydratases than with the corresponding BCR subunits from *Thauera* or *Magnetospirillum* species (Boll, 2005). As a result, false-positive results have been obtained, e.g. the identification of a putative ATP-dependent benzoyl-CoA reductase genes in *G. metallireducens*, although the genome of the latter does not contain genes for this enzyme (Hosoda *et al.*, 2005).

The single degenerate PCR gene probe established in this work provides a new and widely applicable tool for the detection of all types of anaerobic bacteria capable of degrading aromatic compounds. Future applications for the tool comprise (i) the assessment of the anaerobic aromatic degradation potential during bioremediation processes, (ii) the detection of novel bacteria with the capacity to degrade aromatic compounds, and (iii) the exploration of the biodiversity of microbial communities which are responsible for degradation processes of aromatic compounds. The *bamA* gene probe may be used in combination with probes for the degradation of individual compounds such as the *bssA* gene probe. Combinations of general probes for all anaerobic aromatic degraders with probes for specific processes (e.g. toluene degradation) can be applied in real-time PCR or microarray assays and will allow investigators to obtain a more thorough picture of the composition of microbial communities and the metabolic pathways involved in the bioremediation of aromatic compounds.

Experimental procedures

**Growth of bacterial cells and preparation of cell extracts**

The following strains were obtained from Deutsche Sammlung von Mikroorganismen und Zellkulturen (DSMZ) or from the culture collections of the Professor G. Fuchs (Institute for

1554 K. Kuntze et al.

Biology II, University of Freiburg, Germany) or M. McInerney; if not available from DSMZ, the original descriptions are referenced: G. metallireducens GS-15 (DSMZ-No. 7210), T. aromatica K172 (DSMZ-No. 6984), Azoarcus EbN1 (Rabus and Widdel, 1995), A. evansii KB740 (DSMZ-No. 6898), D. multivorans (DSMZ-No. 7210), S. aciditrophicus SB (Jackson et al., 1999), Magnetospirillum TS-6 (Shinoda et al., 2005), Magnetospirillum CC-26 (Shinoda et al., 2000), R. palustris (DSMZ-No. 123), G. sulfurreducens (DSMZ-No. 12127). Two sediment-free sulfate-reducing mixed cultures degrading toluene or *m*-xylene were enriched from different anoxic groundwater samples of a BTEX contaminated aquifer (Knöller et al., 2006). Both cultures have been cultivated for 3–4 years with their respective substrates and were regularly diluted within this time by transfer of 10–20% of the culture volume into fresh medium (see below).

For DNA extraction, all strains were cultured in 10 ml Hungate tubes under anaerobic conditions as described by DSMZ (http://www.dsmz.de/) or by the references given above. The mixed cultures were cultured in a freshwater mineral salt medium described elsewhere (Knöller et al., 2006) spiked with either toluene (2% dissolved in 2,2,4,4,6,8,8 heptamethylnonane: (Knöller et al., 2006) or *m*-xylene (by addition of pure *m*-xylene to final concentrations of 100–300 µM) as sole source of carbon and energy. For biochemical studies, G. metallireducens and S. aciditrophicus were cultured anaerobically in a mineral salts medium using 10 or 200 l fermentors as described (Lovley and Lonergan, 1990; Jackson et al., 1999). The cells were harvested by centrifugation (10 000 g) during the exponential growth phase and were stored by freezing in liquid nitrogen. For the preparation of crude extracts, frozen cells were suspended in 20 mM triethanolamine hydrochloride/ KOH buffer pH 7.3 (1 g of cells in 1.5 ml of buffer) with 10 mM MgCl₂, 10% glycerol and 0.1 mg of DNase I. Cell lysates were obtained by passage through a French pressure cell at 137 MPa. After centrifugation at 100 g (1 h, 4°C), the supernatant was used for further studies.

### Synthesis of CoA esters

6-OH-cyclohexenoyl-CoA was enzymatically synthesized from benzoate with an enriched benzoate-CoA ligase, benzoyl-CoA reductase and dienoyl-CoA hydratase from T. aromatica as described (Laempe et al., 1998). 3-Hydroxypimelyl-CoA and pimelyl-CoA were synthesized as described (Laempe et al., 1999). Isolation and tests for purity of the CoA esters were performed by preparative HPLC (Laempe et al., 1999).

### Cloning and expression of genes

The *bamA*<sub>Geo</sub> gene (gi 78223357) was amplified from G. metallireducens DNA by using the primer pair 5′-ATGGCGAGAACAGACGAG-3′ (forward primer)/5′-CTTCTGCGCCGTCCGGC-3′ (reverse primer); for *bamA*<sub>Syn</sub> gene (gi 85860871), the primer pair 5′-ATGAGTTTAGTTGGATGCCAAG-3′ (forward primer)/5′-TTTTGCCAGCGGTTTTCGAG-3′ (reverse primer) was used. Primers were designed in a way that the native stop codon was removed. Cloning positioned the gene of interest in frame with DNA encoding a C-terminal peptide containing six histidines. Primers were synthesized by biomers.net GmbH (Ulm, Germany). The following PCR programmes using a Taq- and *pfu*-Polymerase (20:1) were applied for amplification: 30 cycles each of 94°C for 1 min, 63°C for 1 min and 72°C for 70 s (*bamA*<sub>Geo</sub>); and 30 cycles 94°C for 1 min, 65°C for 1 min and 72°C for 69 s (*bamA*<sub>Syn</sub>). The programmes started with 5 min at 94°C for activation and ended with 5 min at 72°C. The amplified genes were transferred into the pCR®T7/CT-TOPO® expression vector according to the manufacturer’s instruction (Invitrogen™ Life Technologies, Karlsruhe, Germany). The constructs were transformed into One Shot® TOP10F’ and One Shot® BL21(DE3)pLysS cells for propagation, analysis and maintenance, and expression of the fusion protein, respectively, according to the manufacturer’s instructions. Expression of the genes was carried out by incubating the cells at 25°C overnight and then adding 0.8 mM isopropyl-D-thiogalactopyranoside when the cells reached an OD<sub>578</sub> = 0.5. Cells were harvested at an OD<sub>578</sub> = 2.9 (*bamA*<sub>Geo</sub>) or OD<sub>578</sub> = 3.4 (*bamA*<sub>Syn</sub>) by centrifugation at 13 700 g for 15 min at 4°C and then stored at –80°C until used.

### Purification of His-tagged BamA proteins

The overexpressed His-tagged BamA<sub>Geo</sub> and BamA<sub>Syn</sub> were purified by Ni-Chelating Sepharose™ Fast Flow chromatography (GE Healthcare, Munich) at 6°C. Briefly, 4 g of E. coli One shot®BL21(DE3)pLzsS cells (wet mass) which contained the recombinant fusion protein was re-suspended in 8 ml of 20 mM Tris/HCl buffer, pH 7.9 (buffer A). After equilibration of the Ni-Chelating Sepharose column (His-Trap HP, GE healthcare, 1 ml volume) with buffer A containing 250 mM KCl and 10% glycerol (equilibration buffer), 5 ml of the cell extract was applied to the column at a flow rate of 0.1 ml min⁻¹. The column was washed with 6 column volumes of buffer A at a flow rate of 0.1 ml min⁻¹. Then, a linear gradient of 0% to 100% of buffer B [20 mM Tris/HC (pH 7.9), 250 mM KCl, 10% Glycerin and 500 mM imidazole] at a flow rate of 0.5 ml min⁻¹ was applied. The fusion proteins eluted between 50 and 200 mM imidazole. The purity of both BamA proteins was determined by SDS-page electrophoresis.

### Enzyme assay

The 6-OCH-CoA hydrolase activity from both organisms was determined by a coupled, discontinuous assay by monitoring substrate consumption and product formation at 30°C as described (Laempe et al., 1999). Briefly, The assay solution contained 50 mM potassium phosphate buffer (pH 7.0), 0.3 mM NAD⁺, 0.3 mM pyruvate, 40 units of lactate dehydrogenase and 0.15 mg of purified NAD⁺-dependent 6-OH-cyclohexenoyl-CoA dehydrogenase from G. metallireducens, which had been heterologously expressed in E. coli (not shown, specific activity was 35 µmol min⁻¹ mg⁻¹). The reactions were routinely started by adding 0.2 mM 6-OH-cyclohexenoyl-CoA.

The *K*<sub>m</sub> values for 6-OCH-CoA by BamA were estimated by varying the concentration of 6-OH-cyclohexenoyl-CoA from

0.02 mM to 0.4 mM. Kinetic data were analysed by fitting them to the Michaelis–Menten equation by using the Prism software package (GraphPad, San Diego, CA).

### Determination of the molecular mass of the proteins

The native molecular mass of the overexpressed BamA proteins was determined by using a FPLC Superdex 200 HR 16/60 gel filtration column (GE Healthcare, diameter 16 mm, bed volume 120 ml). After equilibration of the individual columns with 20 mM triethanolamine (pH 7.8), 150 mM KCl, and 10% (v/v) glycerol (BamA<sub>Geo</sub>) or 10 mM Tris-HCl buffer (pH 7.5) containing 100 mM KCl (BamA<sub>Syn</sub>), 100 μl of the protein solution (approximately 1.5 mg) was applied at a flow rate of 0.2 ml min<sup>-1</sup>.

### PCR studies with the bamA gene probe

Database searches for related *bamA* gene sequences were conducted using BLAST (http://www.ncbi.nlm.nih.gov/blast/) and the GenBank database. Multiple alignment of the sequences was carried out with a MultAlin interface (http://bioinfo.genopole-toulouse.prd.fr/multalin/). A degenerate primer pair was deduced from regions that were conserved in *bamA* genes but not in related enoyl-CoA hydratases from organisms that cannot grow anaerobically with aromatic compounds.

The genomic DNA was extracted from different bacteria as described previously (Cheng and Jiang, 2006). The yield of extracted DNA was checked by UV spectroscopy (ND-1000 Spectrophotometer, NanoDrop®, Implen GmbH, Germany, Muenchen). The integrity of the DNA was checked by amplification of the 16S rRNA gene by using the primer set UniBac27f/Univ1492r (Weisburg et al., 1991).

*bamA* fragments were amplified by PCR by using the deduced gene probe PCR primer (see Results) and a mastercycler (Eppendorf). Optimal PCR conditions were determined empirically. The PCR conditions used for the *bamA* gene probe were as follows: 10 min at 94°C (initial denaturation, activation polymerase), 35 cycles of amplification (30 s at 94°C, 45 s at 59°C, 60 s at 72°C), and 10 min at 72°C for terminal extension. One 25 μl PCR reaction contained nuclease-free H<sub>2</sub>O, 2.5 μl of 10× PCR buffer, 2 mM dNTPS, 1.2 U Taq polymerase (BioTherm™), 30 pmol of each primer (biomers.net GmbH, Ulm) and 200–400 ng of template DNA. The amplifications were checked by standard gel electrophoresis and ethidium bromide staining. The appropriately sized amplicons (~300 bp) from DNA of pure cultures were obtained by excision from a agarose gel and purified (QIAquick Gel Extraction Kit, Quiagen, Hilden, Germany) prior to sequencing. Sequencing was performed on a ABI 3700 DNA Analyser (Amersham Pharmacia Biotech, Germany, Freiburg) by using BigDye terminator chemistry according to the manufacturer’s introductions.

### Further determinations

SDS-PAGE was performed as described previously (Laemmli, 1970). Proteins were visualized with Coomassie blue staining (Zehr et al., 1989). Protein concentration was

Acknowledgements

This work was funded by the German Research Council (Deutsche Forschungsgemeinschaft) (to M.B.) and by the US Department of Energy Grant DE-FG03-96-ER-20212 (to M.J.M.).

References

Beller, H.R., Kane, S.R., Legler, T.C., and Alvarez, P.J. (2002) A real-time polymerase chain reaction method for monitoring anaerobic, hydrocarbon-degrading bacteria based on a catabolic gene. *Environ Sci Technol* **36**: 3977–3984.

Boll, M. (2005) Dearomatizing benzene ring reductases. *J Mol Microbiol Biotechnol* **10**: 132–142.

Boll, M., and Fuchs, G. (1995) Benzoyl-coenzyme A reductase (dearomatizing), a key enzyme of anaerobic aromatic metabolism. ATP dependence of the reaction, purification and some properties of the enzyme from *Thauera aromatica* strain K172. *Eur J Biochem* **234**: 921–933.

Boll, M., and Fuchs, G. (2005) Unusual reactions involved in anaerobic metabolism of phenolic compounds. *Biol Chem* **386**: 989–997.

Boll, M., Laempe, D., Eisenreich, W., Bacher, A., Mittelberger, T., Heinze, J., and Fuchs, G. (2000) Nonaromatic products from anoxic conversion of benzoyl-CoA with benzoyl-CoA reductase and cyclohexa-1,5-diene-1-carbonyl-CoA hydratase. *J Biol Chem* **275**: 21889–21895.

Boll, M., Fuchs, G., and Heider, J. (2002) Anaerobic oxidation of aromatic compounds and hydrocarbons. *Curr Opin Chem Biol* **6**: 604–611.

Bradford, M.M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal Biochem* **72**: 248–254.

Chakraborty, R., and Coates, J.D. (2004) Anaerobic degradation of monoaromatic hydrocarbons. *Appl Microbiol Biotechnol* **64**: 437–446.

Cheng, H.R., and Jiang, N. (2006) Extremely rapid extraction of DNA from bacteria and yeasts. *Biotechnol Lett* **28**: 55–59.

Diaz, E. (2004) Bacterial degradation of aromatic pollutants: a paradigm of metabolic versatility. *Microbiology* **7**: 173–180.

Egland, P.G., Pelletier, D.A., Dispensa, M., Gibson, J., and Harwood, C.S. (1997) A cluster of bacterial genes for anaerobic benzene ring biodegradation. *Proc Natl Acad Sci USA* **94**: 6484–6489.

Gibson, K.J., and Gibson, J. (1992) Potential early intermediates in anaerobic benzoate degradation by *Rhodopseudomonas palustris*. *Appl Environ Microbiol* **58**: 696–698.

Gibson, J., and Harwood, C.S. (2002) Metabolic diversity in aromatic compound utilization by anaerobic microbes. *Annu Rev Microbiol* **56**: 345–369.

Heider, J. (2007) Adding handles to unhandy substrates:

anaerobic hydrocarbon activation mechanisms. *Curr Opin Chem Biol* 11: 188–194.

Hosoda, A., Kasai, Y., Hamamura, N., Takahata, Y., and Watanabe, K. (2005) Development of a PCR method for the detection and quantification of benzoyl-CoA reductase genes and its application to monitored natural attenuation. *Biodegradation* 16: 591–601.

Jackson, B.E., Bhupathiraju, V.K., Tanner, R.S., Woese, C.R., and McInerney, M.J. (1999) *Syntrophus aciditrophicus* sp. nov., a new anaerobic bacterium that degrades fatty acids and benzoate in syntrophic association with hydrogen-using microorganisms. *Arch Microbiol* 171: 107–114.

Kim, J., Hetzel, M., Boiangiu, C.D., and Buckel, W. (2004) Dehydration of (R)-2-hydroxyacyl-CoA to enoyl-CoA in the fermentation of alpha-amino acids by anaerobic bacteria. *FEMS Microbiol Rev* 28: 455–468.

Knöller, K., Vogt, C., Richnow, H.H., and Weise, S.M. (2006) Sulfur and oxygen isotope fractionation during benzene, toluene, ethyl benzene, and xylene degradation by sulfate-reducing bacteria. *Environ Sci Technol* 40: 3879–3885.

Laemmli, U.K. (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. *Nature* 227: 680–685.

Laempe, D., Eisenreich, W., Bacher, A., and Fuchs, G. (1998) Cyclohexa-1,5-diene-1-carbonyl-CoA hydratase [corrected], an enzyme involved in anaerobic metabolism of benzoyl-CoA in the denitrifying bacterium *Thauera aromatica*. *Eur J Biochem* 255: 618–627.

Laempe, D., Jahn, M., and Fuchs, G. (1999) 6-Hydroxycyclohex-1-ene-1-carbonyl-CoA dehydrogenase and 6-oxocyclohex-1-ene-1-carbonyl-CoA hydrolase, enzymes of the benzoyl-CoA pathway of anaerobic aromatic metabolism in the denitrifying bacterium *Thauera aromatica*. *Eur J Biochem* 263: 420–429.

Leuthner, B., Leutwein, C., Schulz, H., Horth, P., Haehnel, W., Schiltz, E., *et al.* (1998) Biochemical and genetic characterization of benzylsuccinate synthase: a new glycyl radical enzyme catalysing the first step in anaerobic toluene metabolism. *Mol Microbiol* 28: 615–628.

Lopez Barragan, M.J., Diaz, E., Garcia, J.L., and Carmona, M. (2004a) Genetic clues on the evolution of anaerobic catabolism of aromatic compounds. *Microbiology* 150: 2018–2021.

Lopez Barragan, M.J., Carmona, M., Zamarro, M.T., Thiele, B., Boll, M., Fuchs, G., *et al.* (2004b) The *bzd* gene cluster, coding for anaerobic benzoate catabolism, in *Azoarcus* sp. strain CIB. *J Bacteriol* 186: 5762–5774.

Lovley, D.R., and Lonergan, D.J. (1990) Anaerobic oxidation of toluene, phenol, and *p*-Cresol by the dissimilatory iron-reducing organism, GS-15. *Appl Environ Microbiol* 56: 1858–1864.

McInerney, M.J., Rohlin, L., Mouttaki, H., Kim, U., Krupp, R.S., Rios-Hernandez, L., *et al.* (2007) The genome of *Syntrophus aciditrophicus*: life at the thermodynamic limit

of microbial growth. *Proc Natl Acad Sci USA* 104: 7600–7605.

Meganathan, R. (2001) Biosynthesis of menaquinone (vitamin K2) and ubiquinone (coenzyme Q): a perspective on enzymatic mechanisms. *Vitam Horm* 61: 173–218.

Mouttaki, H., Nanny, M.A., and McInerney, M.J. (2007) Cyclohexane carboxylate and benzoate formation from crotonate in *Syntrophus aciditrophicus*. *Appl Environ Microbiol* 73: 930–938.

Peters, F., Rother, M., and Boll, M. (2004) Selenocysteine-containing proteins in anaerobic benzoate metabolism of *Desulfovibrio multivorans*. *J Bacteriol* 186: 2156–2163.

Peters, F., Shinoda, Y., McInerney, M.J., and Boll, M. (2007) Cyclohexa-1,5-diene-1-carbonyl-coenzyme A (CoA) hydratases of *Geobacter metallireducens* and *Syntrophus aciditrophicus*: evidence for a common benzoyl-CoA degradation pathway in facultative and strict anaerobes. *J Bacteriol* 189: 1055–1060.

Rabus, R., and Widdel, F. (1995) Anaerobic degradation of ethylbenzene and other aromatic hydrocarbons by new denitrifying bacteria. *Arch Microbiol* 163: 96–103.

Schink, B., Philipp, B., and Müller, J. (2000) Anaerobic degradation of phenolic compounds. *Naturwissenschaften* 87: 12–23.

Schirmer, M., Dahmke, A., Dietrich, P., Dietze, M., Gödeke, S., Richnow, H.H., *et al.* (2006) Natural attenuation research at the contaminated megasite Zeitz. *J Hydrol* 328: 393–407.

Shinoda, Y., Sakai, Y., Ue, M., Hiraishi, A., and Kato, N. (2000) Isolation and characterization of a new denitrifying spirillum capable of anaerobic degradation of phenol. *Appl Environ Microbiol* 66: 1286–1291.

Shinoda, Y., Akagi, J., Uchihashi, Y., Hiraishi, A., Yukawa, H., Yurimoto, H., *et al.* (2005) Anaerobic degradation of aromatic compounds by *Magnetospirillum* strains: isolation and degradation genes. *Biosci Biotechnol Biochem* 69: 1483–1491.

Song, B., and Ward, B.B. (2005) Genetic diversity of benzoyl coenzyme A reductase genes detected in denitrifying isolates and estuarine sediment communities. *Appl Environ Microbiol* 71: 2036–2045.

Weisburg, W.G., Barns, S.M., Pelletier, D.A., and Lane, D.J. (1991) 16S ribosomal DNA amplification for phylogenetic study. *J Bacteriol* 173: 697–703.

Winderl, C., Schaefer, S., and Lueders, T. (2007) Detection of anaerobic toluene and hydrocarbon degraders in contaminated aquifers using benzylsuccinate synthase (*bssA*) genes as a functional marker. *Environ Microbiol* 9: 1035–1046.

Wischgoll, S., Heintz, D., Peters, F., Erxleben, A., Sarnighausen, E., Reski, R., *et al.* (2005) Gene clusters involved in anaerobic benzoate degradation of *Geobacter metallireducens*. *Mol Microbiol* 58: 1238–1252.

Zehr, B.D., Savin, T.J., and Hall, R.E. (1989) A one-step, low background coomassie staining procedure for polyacrylamide gels. *Anal Biochem* 182: 157–159.
