
Review

Sumoylating and desumoylating enzymes at nuclear pores: underpinning their unexpected duties?

Benoît Palancade and Valérie Doye

Institut Curie, Centre de Recherche & UMR144 CNRS, 26 rue d'Ulm, F-75248 Paris, France

Modulation of protein activities by SUMO-dependent modification has emerged as a key feature of cellular regulation. Evidence of the localization of different enzymes of the sumoylation-desumoylation cycle at nuclear pore complexes (NPCs), and of its biological relevance, has steadily accumulated over the past ten years. Recent findings indicate that, beyond nucleocytoplasmic transport, sumoylation processes underpin newly emerging, and initially unexpected, roles for NPCs in a broad array of biological functions. These include cell division, DNA repair, DNA replication and mRNA quality control. Most of these functions were initially discovered through genetic studies in budding yeast, but the localization of SUMO-proteases at NPCs in higher eukaryotes suggests that at least some of these mechanisms might have been conserved during evolution.

Introduction

A fundamental characteristic of living cells is the controlled distribution of macromolecules within different cellular compartments in which they fulfill their respective functions. In eukaryotes, the paradigm for such a phenomenon is represented by the nucleus, which is separated from the rest of the cell by the nuclear envelope, a double lipid membrane that is perforated by nuclear pore complexes (NPCs). NPCs are conserved macromolecular assemblies formed by proteins called nucleoporins (reviewed in [1]), and their canonical function is the selective and directional transport of proteins and ribonucleoparticles between the nucleus and the cytoplasm (Figure 1 and references therein). As detailed in this review, however, recent findings suggest that the function of NPCs is not solely restricted to nucleocytoplasmic exchanges.

Over the past 10 years, the covalent attachment of small modifier polypeptides has emerged as an essential determinant of proteins fate. Modification by members of the SUMO (small ubiquitin-like modifier) family has been shown to modulate multiple cellular processes (reviewed in [2,3]). SUMOs are small (10–11 kDa) polypeptides, which are conserved among eukaryotes. Some organisms, such as the budding yeast *Saccharomyces cerevisiae*, the fission yeast *Schizosaccharomyces pombe*, the nematode *Caenorhabditis elegans* and the fruit fly *Drosophila melanogaster*, have a single gene encoding a SUMO polypeptide (e.g., SMT3 in budding yeast), whereas the genomes of vertebrates and plants encode several SUMO proteins (also called Sentrins in mammals; reviewed in [3]). Like ubiquitin, SUMO polypeptides are synthesized as precursors that need to be processed before being conjugated to their targets (Figure 2, step 1). Sumoylation (Figure 2, steps 1–5) then relies on the formation of an isopeptide bond between the processed C-terminus of SUMO and the ε-amino group of an acceptor lysine within the target protein (which can be another SUMO in the case of SUMO chain synthesis [Figure 2, step 5]). The sumoylation process requires the action of a cascade of enzymes: an E1-activating enzyme (the Uba2–Aos1 heterodimer), an E2-conjugating enzyme (Ubc9) and, in most cases, one of the SUMO E3 ligases that enhance E2-mediated sumoylation. Desumoylation (e.g., deconjugation [Figure 2, step 6] and chain-editing [Figure 2, step 7]) involves SUMO-proteases (also termed SUMO-isopeptidases), a family of conserved proteins some of which, such as budding yeast Ulp1 and vertebrate SENP1 and SENP2, can also process SUMO precursors (Figure 2, step 1; for review, see [4,5]). Sumoylation generally modifies the properties of its target proteins by affecting their abilities to interact with partners and/or by preventing or promoting other posttranslational modifications. These changes can, in turn, affect the subcellular localization of the targeted proteins, their biological activity and, in some cases, their stability (reviewed in [2,3]).

Because one of the first described sumoylated proteins was a component of nuclear pores [6,7], sumoylation has been linked with NPCs and nucleocytoplasmic transport since its discovery [8,9]. While the impact of sumoylation on various cellular processes has been extensively reviewed (see [3]), we focus here on recent studies that shed light on the multiple links between sumoylation or desumoylation and nuclear transport machineries. First, we review the molecular mechanisms underlying the targeting of sumoylation and desumoylation enzymes to NPCs. We then summarize the impact of NPC-localized sumoylation or desumoylation enzymes on cellular sumoylation patterns. Finally, we discuss the role of these enzymes in newly emerging non-canonical NPC functions such as cell division, DNA repair, DNA replication and mRNA quality control.

Sumoylation enzymes are associated with NPCs in vertebrates

The first hints of a connection between nuclear pores and sumoylation processes came from the observation that

Corresponding authors: Palancade, B. (palancad@curie.fr); Doye, V. (vdoye@curie.fr).

Review                                                                                     Trends in Cell Biology Vol.18 No.4

(a)                                                                                       (b)

![Diagram](attachment:diagram.png)

Figure 1. Basic mechanisms of nucleocytoplasmic transport. Schematic shows only factors that are referred to in the text and clusters them according to their respective functions (for extensive descriptions of these processes, see [67,68]). (a) Protein transport. Proteins (in pink) containing a targeting signal such as a NLS (nuclear localization signal) or NES (nuclear export signal) are recognized by receptors of the karyopherin superfamily, respectively, importins or exportins (in yellow). A typical karyopherin is represented by importin-β (Kap95 in budding yeast), which mediates import of classical NLS-containing cargos through the intermediary adaptor importin-α (Kap60 or Srp1 in budding yeast). Translocation through the nuclear pore relies on the dynamic interaction between a karyopherin and a specific subset of nucleoporins containing phenylalanine-glycine (FG) repeats. The directionality and irreversibility of transport are provided by the nucleocytoplasmic gradient of the small GTPase Ran, which in its nuclear-enriched GTP-bound form is required to dissociate import complexes and to assemble export complexes. The Ran-GTP gradient is maintained by Ran cofactors, namely Ran GDP-GTP exchange factors (RanGEFs), which are located in the nucleus, and RanGTPase-activating proteins (RanGAPs), located in the cytoplasm (or, in the case of mammalian RanGAP1, at the cytoplasmic face of NPCs). (b) mRNA export. Upon synthesis, mRNAs are processed and assembled with export adaptors such as Yra1 (ALY in mammals, in pink), which can in turn recruit the export receptor, namely the conserved dimer Mex67-Mtr2 (TAP-p15 in mammals, in yellow). Unlike karyopherins, this export receptor interacts with its cargo in a Ran-GTP independent manner, although it also directly binds to FG-containing nucleoporins.

vertebrate RanGAP1, the GTPase-activating protein for Ran (a small GTPase involved in nucleocytoplasmic transport), can be conjugated with SUMO-1 [6,7]. This SUMO modification promotes the interaction of RanGAP1 with RanBP2-Nup358, a giant nucleoporin localized on the cytosolic filaments of NPCs [6,7,10] [Figure 3a and 3b, panels (i) and (i)']. At around the same time, other investigators showed that RanBP2 associated with a modified form of RanGAP1 and with Ubc9 [11], an E2-like enzyme for SUMO conjugation [12,13]. Although vertebrate Ubc9 is predominantly nuclear, a fraction of it consistently localizes to NPCs [13–15] (Figure 3b, panels (ii) and (ii)'). Structural studies, as well as in vitro approaches, have further revealed the formation of a ternary complex containing RanBP2-Nup358, SUMO-1-modified RanGAP1 and Ubc9 [16,17]. Within this complex, SUMO-1-modified RanGAP1 is resistant to SENP2 protease activity [14], a property that might explain the remarkable abundance of SUMO-conjugated RanGAP1 in comparison with that of most other SUMO-conjugates [6,7]. The link between this NPC-localized complex and sumoylation was strengthened by the finding that RanBP2 exhibits SUMO E3 ligase activity and enhances the sumoylation of Ubc9 substrates [18]. However, whether RanGAP1 sumoylation takes place at NPCs, and to what extent RanBP2 E3 activity contributes to this process in vivo, remain unclear (discussed in [16]).

The in vivo requirement for both the E2 Ubc9 and the E3 RanBP2 for proper targeting of SUMO-modified RanGAP1 to the cytoplasmic side of NPCs is now well established ([19] and references therein), but we must emphasize that RanPB2 is a vertebrate-specific E3 ligase. Moreover, the other SUMO E3 ligases from vertebrates or other eukaryotes described to date do not localize at the nuclear envelope. Furthermore, Ubc9 is only found at NPCs in vertebrates (reviewed in [3]). In plants, the anchoring of RanGAP1 at the nuclear envelope relies on a totally different mechanism, namely the use of a specific N-terminal tryptophan- or proline-rich (WPP) domain [20]; in budding yeast, RanGAP1 is largely cytoplasmic and cannot be detected at the nuclear envelope at all.

Although the overexpression of a fragment of RanBP2 that sequesters most Ubc9 within the cytoplasm affects the localization of SUMO and/or sumoylated proteins, this does not significantly impair the NPC localization of Ubc9 or SUMO-1 modified RanGAP1 [15]. Moreover, none

Review                                                                                     Trends in Cell Biology Vol. 18 No. 4

| Reaction                      | Enzymes          |
|-------------------------------|------------------|
| #                             | Name             | Budding yeast       | Mammals            |
| ①                            | SUMO processing  | Ulp1 (Ulp2?)        | SENP1, SENP2,      |
|                               |                   |                     | SENP3?, SENP5?,    |
|                               |                   |                     | SENP7?             |
| ②                            | SUMO activation (E1) | Uba2/Aos1         |                    |
| ③                            | SUMO conjugation (E2) | Ubc9              | Ubc9               |
| ④, ⑤                        | SUMO ligation (E3) | Siz1, Siz2         | RanBP2, PC2,       |
|                               |                   | Mms21, Zip3        | PIAS1, PIASxα,     |
|                               |                   |                     | PIASxβ, PIAS3,     |
|                               |                   |                     | PIASy, MMS21       |
| ⑥                            | SUMO deconjugation | Ulp1, Ulp2         | SENP1, SENP2,      |
|                               |                   |                     | SENP3, SENP5,      |
|                               |                   |                     | SENP7?             |
| ⑦                            | SUMO chain editing | Ulp2 (Ulp1?)       | SENP6, SENP7?      |

Figure 2. The sumoylation and desumoylation pathway and its enzymes. The sequential reactions involved in sumoylation (steps 1–5, in green) and desumoylation (steps 6 and 7, in orange) are represented and referred to by numbers in the accompanying table, in which the budding yeast and mammalian enzymes are indicated (reviewed in [3, 5]). The enzymes that have a reported localization at the NPCs are highlighted in red (see also Figure 3). Note that SUMO-processing (step 1), SUMO deconjugation (step 6) and SUMO chain-editing (step 7) activities are carried out by a family of enzymes termed SUMO-proteases (or SUMO isopeptidases). Not all the SUMO-proteases equally support each of these activities. For instance, in budding and fission yeasts, Ulp1 is the prominent SUMO-processing enzyme in vivo, whereas SUMO chain-editing mainly relies on Ulp2 in vivo [21, 69, 70]. Note also that, in mammals, SUMO-proteases exhibit differential specificities towards the different available SUMO paralogs (reviewed in [5]).

of the identified in vitro targets of RanBP2 has yet been  
Multiple determinants are required for proper NPC  
reported to rely solely on this E3 ligase in vivo (reviewed in  
localization of budding yeast Ulp1  
[3]). Future studies aimed at characterizing in vivo targets  
In budding yeast, the N-terminal domain of Ulp1 contains  
will thus be required to determine the potential function(s),  
binding sites for two distinct nuclear transport receptors  
beyond the localization of RanGAP1, of this well-characterized NPC-localized sumoylation complex.  
(karyopherins), Kap121 (amino acid residues 1–150) and  
Kap60–Kap95 (residues 150–340), both of which are  
required for efficient targeting of Ulp1 to the NPC ([28];  
see Figure 4a). Unlike classical karyopherin-cargo interactions (see Figure 1), these interactions occurred in a Ran-  
GTP-insensitive manner, suggesting the possibility of a non-conventional tethering of Ulp1 to the NPC through permanent interactions with karyopherins [28]. While this first study was performed using a mildly overexpressed green fluorescent fusion protein GFP-Ulp1, it was subsequently shown that Ulp1-GFP expressed at endogenous levels was also partially mislocalized in kap121 mutant cells at the restrictive temperature (37°C). Although the localization of Ulp1-GFP at the nuclear envelope was not significantly impaired in a kap95 mutant, the Ulp1Δ150–340 deletion, which lacks the Kap60-Kap95 binding domain, was entirely mislocalized in kap121 mutant cells at 37°C, suggesting that a complex interplay between these two karyopherin pathways is required for proper targeting of Ulp1 to the nuclear envelope (discussed in [32]).  

The molecular mechanisms underlying the localization of Ulp1 at NPCs turned out to have further levels of complexity. Another study revealed the involvement of the nuclear-basket proteins Nup60 and Mlp1-Mlp2 in this process ([33]; Figure 3a). In agreement with a potential role for Mlp1-Mlp2 in tethering Ulp1 to the NPC, Ulp1 is excluded from NPCs in the region of the nuclear envelope

Multiple pathways contribute to proper NPC  
localization of Ulp1/SENP2 SUMO-proteases  
In contrast to the SUMO conjugation and ligation machinery, the localization of SUMO-proteases at NPCs has been conserved during evolution (Figure 3b). In budding yeast, the SUMO-protease Ulp1 is localized at nuclear pores [Figure 3b, panel (v)], whereas its unique paralog, Ulp2, is only present within the nucleoplasm [21]. Localization of Ulp1 at the nuclear periphery is also a feature of the unique Schizosaccharomyces ortholog [22] and of Ulp1's characterized orthologs in Drosophila [23] and Arabidopsis thaliana (ESD4, Figure 3b, panel (iv); [24]). Among the characterized Ulp1-related SUMO-proteases in mammals (termed SENPx, for Sentrin protease x), SENP1 and SENP2, which exhibit the highest similarity with yeast Ulp1, are also associated with the nuclear envelope [Figure 3b, panels (i), (ii) and (iii)], [14, 25, 26], although there is some variability in localization of SENP1 depending on the species and cell type [Figure 3b, panels (iii)] and (iii)" and references therein]. Deletion analyses of Ulp1 or SENP2, as well as analyses of alternatively spliced variants of rat or mouse SENP2, have revealed the involvement of their non-catalytic N-terminal domains in targeting to the NPC [14, 25, 27–31].

Review                                                                                     Trends in Cell Biology Vol.18 No.4

(a) Mammals                                                                                       Yeast
Cytoplasm

Ubc9  
Nup358/RanBP2  
SUMO-RanGAP1  

Nup214  
Nup98  
Nup88  
Rae1  

Nup107 Nup133 Nup43  
Nup96 Sec13 Nup37  
Nup85 Seh1  
Nup160  

gp210  
Nup53 Nup205  
Pom121 Nup188  
Ndc1 Nup93 Nup155  

SENPl?  
Nup160  
Nup85 Seh1  
Nup96 Sec13 Nup37  
Nup107 Nup133 Nup43  

Ubc9?  
Nup50  
Nup98  
Rae1  

SENPl2  
Nup153  
Tpr  

Glel  
Nup159  
Nup82  

Nup42  
Nup116  
Rae1  
Nup100  

Nup84 Nup133  
Nup145-C Sec13  
Nup85 Seh1  
Nup120  

Nup192  
Nsp1  
Nup188  
Nic96  

Nup57  
Nup53 Nup170  
Pom34  
Nup59 Nup157  
Pom152  
Ndc1  

Nic96  
Nup120  
Nup188  
Nup85 Seh1  
Nup192  
Nup145-C Sec13  
Nup84 Nup133  

Nup116  
Rae1  
Nup1  

Nup2  
Mlp1 Mlp2  
Ulp1  

Nucleus

(b) RanGAP1 SENP2 Ubc9 ESD4

(i)  
(ii)  
(iii)  
(iv)  
(v)

SENPl  

Ulp1  

TRENDS in Cell Biology

Figure 3. NPC architecture and localization of sumoylation and desumoylation enzymes. (a) Simplified model of the organization of nucleoporins within the NPC framework in mammals (left) and in budding yeast (right). Nucleoporins are colored to reflect the different subcomplexes that compose the whole NPC (reviewed in [1], see also [71] for the yeast NPC). Two of the yeast subcomplexes are more specially referred to in the text: the Nup84 complex (Nup107–160 complex in mammals, boxed in purple) is a symmetrically localized essential building block of NPCs, which has been conserved throughout evolution; the yeast Nup60–Mlp1–Mlp2 complex (boxed in blue) contains three proteins that are localized at the nucleoplasmic side of NPCs. Although biochemical evidence for this complex is lacking, Mlp1 and Mlp2 anchoring to NPCs has been shown to depend on Nup60, and the nup60Δ and mlp1Δ mlp2Δ mutants have strongly overlapping phenotypes (see text for details). Note that Tpr, the human ortholog of Mlp1–Mlp2, is bound to NPCs through interaction with Nup153, which is thought to be a functional homolog of Nup60 (discussed in [72]). Enzymes of the SUMO cycle are boxed in red and placed in the scheme according to their putative localization within the NPC. Lines connect them to their potential anchoring determinants (see text for details). (b) Localization of some enzymes of the sumoylation cycle at the nuclear periphery. (i), RanGAP1 (green) and SENP2 (red) localization at nuclear pores in HeLa cells. (i)', four-fold magnification of the inset in panel (i) (reproduced with permission from [14]). Note that the SENP2 signal is present in a ring internal to the RanGAP1 signal, consistent with the localization of these two proteins on the nuclear and cytoplasmic sides of the NPCs respectively. (ii) and (ii)', localization of Ubc9 in mammalian COS-7 cells, (ii) without or (ii)' with digitonin permeabilisation before fixation. Ubc9 is mainly nuclear, but its enrichment at the nuclear periphery becomes clearly detectable following digitonin treatment (reproduced with permission from [15]). (iii), (iii)', (iii)", SENP1 localization can differ depending on the species and the cell type. SENP1 can be observed in the nucleus and at the nuclear periphery in human HEp2 cells (iii, reproduced with permission from [26]). By contrast, *Drosophila* SENP1 is localized predominantly in the nucleus and at the plasma membrane in salivary gland cells, but within the cytoplasm in foregut cells (iii)' and (iii)", respectively, reproduced with permission from [73]). (iv), Localization of the SUMO protease ESD4 in roots cells of *Arabidopsis thaliana* seedlings (reproduced with permission from [38]). (v), Ulp1 localization at the nuclear periphery in budding yeast. Note the asymmetric localization of Ulp1 to a restricted area of the nuclear envelope (see text for details).

that is juxtaposed to the unique nucleolus (see Figure 3b, panel (v) [33]), a localization shared by Mlp1, Mlp2 and their binding partner Pml39 [34,35]. Consistent with this, it was recently reported that in the absence of Escl, a non- NPC nuclear envelope protein that affects nuclear basket assembly, Ulp1 relocates along with Nup60, Mlp1 and Mlp2 to a few perinuclear aggregates [36]. In addition, mutations affecting the Nup84 complex (see Figure 3a) also

Review

Trends in Cell Biology Vol.18 No.4

(a)
![Diagram](https://i.imgur.com/placeholder.png)

(b)
![Diagram](https://i.imgur.com/placeholder.png)

Figure 4. Yeast Ulp1: domain organization and involvement in non-canonical NPC functions. (a) Functional domains of the SUMO-protease Ulp1. The domains delineated by amino acids 1–150 and 151–340 bind to Kap121 and Kap60-Kap95, respectively. The coiled-coil domain (residues 340–403) functions as a NES, possibly preventing the nuclear accumulation of Ulp1. (Domain organization is adapted from [28]). (b) A summary of the different non-canonical NPC functions involving Ulp1-dependent (de)sumoylation. The different proteins of the nuclear envelope, NPCs and karyopherins required for targeting, tethering and/or stabilizing Ulp1 at the NPCs are represented (see text for details). The various processes that are affected in cells harboring mutated nucleoporins, karyopherins and/or Ulp1 are boxed. For each, potential sumoylated targets are indicated; note that, impaired sumoylation in nucleoporins, karyopherins or Ulp1 mutant strains has only been definitively shown for Yku70 and Cdc3 [32,37,43].

strongly impair the localization of Ulp1 at the nuclear envelope [37]. Since these mutations do not affect the targeting of Nup60, Mlp1 or Mlp2 at NPCs [37], the mis-localization of Ulp1 is not the result of either Nup60 or Mlp1–Mlp2 mis-targeting, but of some other defect, possibly the loss of a direct interaction between Ulp1 and the Nup84 complex. Finally, it is worth noting that the overexpression of Ulp1 or of its Kap121-binding site led to their uniform distribution within the inner nuclear envelope, even in NPC-free areas [32,37]. This suggests the existence of additional, NPC-independent, binding sites at the nuclear envelope.

What could determine the respective contributions of karyopherins, Nup60–Mlp1–Mlp2, the Nup84 complex and potentially other nuclear envelope tethers in the targeting of Ulp1 to the NPC and, in some instances, the inner nuclear membrane? Considering the cytoplasmic mislocalization of Ulp1 seen in kap121 mutants or after treatment with metabolic poisons [32], it is tempting to speculate that the energy-requiring karyopherin-dependent nuclear import of Ulp1 might be a preliminary step, which is required for the subsequent binding of Ulp1 to the NPC or to nuclear envelope anchors. As discussed previously [37], the Nup84 and Nup60–Mlp1–Mlp2 NPC subcomplexes could subsequently provide either independent binding sites or two successive anchors, with the Nup60–Mlp1–Mlp2 site probably providing the final binding site for Ulp1 at NPCs. Loss of NPC-associated Ulp1 in nucleoporin mutants, but not in kap121 mutants, is associated with proteasomal-dependent degradation of Ulp1 [37]. According to the above scenario, this might reflect the decreased stability of Ulp1 that is mislocalized within the nucleus. Alternatively, this degradation could hint at a more indirect involvement of the Nup84 and/or Nup60–Mlp1–Mlp2 complexes in the regulation of Ulp1 expression, post-translational modification or stability [33,37]. Extensive investigation of the direct binding partners of Ulp1 at NPCs are required to clarify how karyopherins and nucleoporins cooperate to maintain proper levels of Ulp1 at nuclear pores.

**Are similar determinants involved in NPC localization of SUMO proteases in metazoans?**

Direct interactions between Ulp1 and nucleoporins have not yet been reported in budding yeast, but it is worth noting that the *Arabidopsis thaliana* Ulp1 ortholog ESD4 interacts with NUA (Nuclear Pore Anchor, an Mlp1–Mlp2 ortholog) in a two-hybrid assay [38]. Although *nua* mutants phenocopy *esd4* mutants and exhibit a similar rise in cellular sumoylation levels, the localization of GFP–ESD4 at the nuclear periphery is not affected in these mutants, suggesting that the interaction between ESD4 and NUA might be indirect and that other proteins might tether ESD4 to the nuclear envelope [38]. In vertebrates, the N-terminal NPC-targeting domain of SENP2 interacts with the C-terminal karyopherin-binding domain of Nup153, a putative Nup60 ortholog [14,25]. On the basis of these assays, however, the existence of an indirect karyopherin-mediated interaction cannot be excluded. In the future, additional studies aimed at determining the contribution of karyopherins and nucleoporins (including

Nup153, Mlp orthologs and the Nup107–160 complex) in vertebrates will be required to determine whether common targeting and stabilization mechanisms act upon the evolutionarily conserved SUMO proteases at the nuclear envelope.

Reciprocal interplay between sumoylation processes and nucleocytoplasmic transport

Several studies have revealed that sumoylation can contribute to the regulation of the nucleocytoplasmic transport machinery. For instance, the anchoring of sumoylated RanGAP1 at NPCs in vertebrates might contribute to the efficiency of karyopherin-dependent import or export by maintaining a RanGTP gradient. In addition, yeast *uba2* and *ulp1* mutants, which are defective in SUMO activation and protease activities respectively, are specifically impaired in the import of proteins that contain a classical nuclear localization sequence (cNLS). These mutants also accumulate the cNLS adaptor Kap60 in the nucleus [39], raising the possibility that sumoylation or desumoylation regulates the recycling of Kap60. An attractive, yet still hypothetical, target for sumoylation might be the nucleoporin Nup2, which is required for Kap60 recycling and was found to be part of the SUMO-modified proteome [40]. Finally, export of yeast pre-ribosomes was recently shown to rely on a functional sumoylation machinery ([41] and references therein). Because of the association of SUMO-modifying enzymes with NPCs, it has also been proposed that the sumoylation and/or desumoylation of specific targets could be coupled to their translocation through NPCs, although there is no direct evidence for such a mechanism ([9] and references therein).

The nucleocytoplasmic transport machinery can also regulate the sumoylation of non-nuclear targets. In budding yeast, components of the septin ring, which assemble at the bud neck during mitosis, are sumoylated by the Siz1 E3 ligase before anaphase. This sumoylation process relies on the active nuclear export of Siz1 by Kap142–Msn5 that potentially depends on Siz1 phosphorylation in mitosis [32,42]. At cytokinesis, Ulp1-mediated desumoylation of septins may rely on the karyopherin-dependent targeting of Ulp1 to the bud neck [32,43]. This last process could be triggered by previously described molecular rearrangements within the NPCs that affect the interaction between Kap121 and specific nucleoporins in mitosis ([44]; discussed in [32]).

Functional links between sumoylation enzymes, transport factors and nucleoporins might also exist outside the NPC in species that, unlike yeasts, are characterized by an open mitosis (i.e. mitosis in which the nuclear envelope breaks down and then re-forms around the two sets of separated chromosomes). In mammals, such a link is indicated by the shared kinetochore localization of sumoylated RanGAP1, the E3 RanBP2, the exportin CRM1 and the Nup107–160 complex ([45,46] and references therein). These proteins are known to regulate the attachment of kinetochores to microtubules during mitosis, but the extent to which sumoylation contributes to these processes and the nature of potential SUMO targets remain to be determined.

Review

NPC association of SUMO-proteases controls cellular sumoylation patterns

As discussed above, the timely regulated control of Ulp1 localization can affect the sumoylation of specific substrates during mitosis in budding yeast. However, mutations that impair the NPC-targeting of SUMO-proteases were also shown to cause more complex changes in global cellular sumoylation patterns. In yeast ulp1Δ cells, the expression of mutant forms of Ulp1 that lacked the N-terminal NPC targeting domain (Figure 4a) led to the accumulation of SUMO conjugates, suggesting that this domain may confer specificity for Ulp1 substrates [27]. In addition, the increase in sumoylated species observed in cells that lacked the other yeast SUMO-protease, Ulp2, can be reverted when N-terminal deletion mutants of Ulp1 are expressed. This indicates that, once mislocalized, Ulp1 might target genuine Ulp2 substrates [27]. Similar results were obtained in mammalian cells that overexpressed a truncated GFP-SENP2 fusion lacking its N-terminal NPC targeting domain. This fusion accumulated in the nucleus and led to an overall decrease in global sumoylation [25]. It is also interesting that the overexpression of N-terminal-truncated forms of Ulp1 that are mislocalized within the nucleus had a dominant-lethal effect on yeast viability, unless fused to a cytoplasmic anchor or to a NES sequence [28,47]. Moreover, such Ulp1 fusions could rescue the lethal phenotype of a ulp1Δ strain, indicating that exclusion from the nucleoplasm, rather than NPC localization, might be the predominant function of Ulp1's NPC-targeting domain [28].

Complex alterations of sumoylation profiles, including both increased and decreased levels of sumoylation of specific targets, were also observed upon mutating Nup60–Mlp1–Mlp2, the Nup84 complex or Esc1, which affect the localization of Ulp1 at NPCs [33,36,37,48]. As discussed previously [33,37], both the loss of Ulp1's ability to target natural substrates and an increased access to nuclear Ulp2 targets might cause these complex modifications of sumoylation patterns. Indeed, a moderate targeting of Ulp1 towards Ulp2 substrates could underlie the suppression of ulp2Δ mutant growth defects observed in esc1Δ or nup60Δ strains [36]. However, because the overall level of Ulp1 is decreased in the nucleoporin mutants (a property not shared by esc1Δ cells), some of these sumoylation defects could also be caused by impairment of SUMO processing (Figure 2, step 1), a process that mainly relies on Ulp1 in yeast [21]. In these mutants, a reduction in the availability of conjugatable SUMO might impinge on the sumoylation of a broad range of targets (discussed in [48]).

Ulp1-dependent sumoylation or desumoylation processes underpin newly emerging non-canonical NPC functions

Recent studies have uncovered unexpected functional connections between NPCs and different aspects of nuclear metabolism, namely DNA repair, DNA replication, mRNA quality control and nuclear organization, that are largely independent of the canonical function of NPC in nucleocytoplasmic transport [33,34,37,49–51]. Connections between these functions and sumoylation or desumoylation processes are now emerging (Figure 4).

Clonal lethality and regulation of 2μ circle levels

The first demonstration of such a link was the observation in budding yeast that mlp1Δmlp2Δ strains and nup60Δ strains exhibit irregularly shaped colonies (referred to as 'nibbled') [33]. This phenotype reflects clonal lethality that was caused by increased copy number of the 2μ circle, a selfish DNA element that is naturally present in yeast strains ([33] and references therein). Mutants that have impairments in the enzymes of the SUMO pathways, including ulp1 and the E3 ligase mutant siz1Δsiz2Δ, also form nibbled colonies [52,53], and so it was hypothesized and subsequently demonstrated that clonal lethality in the nucleoporin mutants was due to the altered localization or stability of Ulp1. Accordingly, these phenotypes were suppressed in the nucleoporin mutants by restoring Ulp1 levels [33].

Possible mechanisms to explain the effect of SUMO on the accumulation of 2μ circles are now emerging. Indeed, among the four proteins encoded by the 2μ circle and necessary for its replication, two, including the Flp1 recombinase, are modified by SUMO. A direct role for Flp1 sumoylation in this pathway is suggested by previous studies which showed that increased expression of Flp1 increases 2μ levels and that sumoylation negatively regulates Flp1 levels ([53] and references therein). As previously discussed [52,53], however, accumulation of 2μ circles might also result from defects in plasmid partitioning. Accordingly, other sumoylated host proteins that have a possible involvement in this process, including topoisomerase II (Top2), the chromatin remodeling complex subunit Rsc2, and the cohesin subunits Smc1 and Scc1, might also contribute to this phenotype ([52,53] and references therein). Future analysis of the sumoylation status of these proteins in ulp1 and nucleoporin mutants should provide further insight into the molecular mechanisms that connect nucleoporins to clonal lethality.

Sumoylation processes: the missing link between nucleoporins and DNA repair

Independently, a synthetic lethal screen revealed a strong and specific genetic relationship between mutations that impair the Nup84 or Nup60–Mlp1–Mlp2 complexes and mutations that affect either the Rad52 pathway of homologous-recombination-mediated-DNA repair or Rad27, an enzyme of the DNA replication machinery. These genetic interactions correlated with an exacerbated sensitivity to DNA-damage-inducing treatments and with the accumulation of Rad52-DNA-repair foci in the nucleoporin mutants ([37,50] and references therein). The fact that similar DNA repair and replication defects had previously been reported for a ulp1 mutant [54] subsequently led to the demonstration that the accumulation of DNA damage in these nucleoporin mutants was mediated by loss of ULP1 function [37]. Impaired processing of SUMO precursors (Figure 2, step 1) by Ulp1 might underlie these phenotypes as similar defects were also reported in mutants of the E2 ligase Ubc9 and of the E3s Siz1–Siz2 or Mms21 [48,55,56].

Further studies in budding yeast revealed that decreased sumoylation of Yku70, a protein involved in the repair of DNA breaks through non-homologous end-joining (NHEJ,

an alternative to the homologous recombination pathway), might contribute to the DNA-repair phenotypes shared by nucleoporin and ulp1 mutants [37]. As this observation was the result of a ‘candidate approach’, other SUMO targets that are involved in DNA recombination, repair or replication, including for example the recombination protein Rad52 [57] and the DNA-replication factor PCNA [58], might also be affected in these mutants.

Interestingly, in mammalian cells, DNA-damage-induced assembly of the DNA-repair protein Rad51 into DNA-repair foci is impaired upon overexpression of a RanBP2 domain that sequesters the E2 Ubc9 in the cytoplasm [15]. This indicates that DNA-repair processes are also dependent on regulation by SUMO in vertebrates, although the underlying mechanism appears to be distinct from that in yeast.

### NPC basket assembly, SUMO protease regulation, and mRNA surveillance

In budding yeast, the basket-associated nucleoporins Mlp1, Mlp2 and Nup60 are involved in a quality-control pathway that prevents the export of improperly processed mRNAs [34, 59, 60]. The observation that Esc1, which controls nuclear basket assembly, is involved in both Ulp1 localization at the nuclear envelope and the retention of unspliced pre-mRNA in the nucleus, suggested a potential function for Ulp1 in this mRNA surveillance process. Consistently, Ulp1 was shown to prevent leakage of pre-mRNAs by a mechanism that is genetically linked to ESC1 and MLP1 [36]. Interestingly, a mutant allele of the mRNA export factor Yra1 was suppressed by deletion of MLP2 or PML39 (which encodes an NPC-associated effector of the Mlp proteins that is involved in retention of pre-mRNAs) [35, 60], and ULP1 was found as a high-copy-number suppressor of another yra1 mutant [61]. These data suggest the existence of a complex pathway that prevents improper mRNA export and involves both nuclear basket proteins and the SUMO-protease Ulp1. Whether Ulp1 directly or indirectly regulates components of the Mlp1–Mlp2 pathway, or affects other factors involved in mRNA metabolism, some of which belong to the SUMO-modified yeast proteome [40], remains unknown.

### Other shared functions of nucleoporins and SUMO proteases?

In yeast, Nup60–Mlp1–Mlp2 [49, 62, 63], the Nup84 complex [51] and Esc1 [64, 65] also contribute to nuclear organization by controlling the length and nuclear envelope anchoring of telomeres, and subtelomeric transcriptional repression. Because several telomere-associated proteins, including Yku70, are sumoylated [40, 66], efforts should be made to discover whether these functions also require Ulp1-mediated control of sumoylation patterns.

Finally, although the above examples are taken from budding yeast, the regulation of complex cellular processes by both nucleoporins and SUMO proteases extends to multicellular organisms. Indeed, Arabidopsis plants bearing mutations in the genes encoding NUA (Nuclear pore anchor, the Mlp1–Mlp2 ortholog) and ESD4 (the Ulp1/SENP ortholog) exhibit similar phenotypes, including

increased abundance of SUMO conjugates, deregulation of the timing of flowering and developmental defects ([24, 38] and references therein). In multicellular organisms, NPC-dependent regulation of SUMO protease function might thus impinge on complex developmental and physiological processes.

### Concluding remarks

Over the past decade, the localization of enzymes of the SUMO cycle at NPCs has been well documented. While the localization of the SUMO-conjugating machinery at the cytosolic side of the NPC is restricted to vertebrates, the anchoring of SUMO proteases at the inner side of the nuclear envelope appears to be a conserved eukaryotic feature. However, the molecular mechanisms underlying this localization and the extent to which they have been conserved between species remain to be fully understood. In addition, the use of genetically amenable organisms, mainly budding yeast, has revealed that several non-canonical NPC functions involve sumoylation or desumoylation processes. It will now be essential to determine to what extent the localization of SUMO enzymatic machineries at NPCs is crucial for the control of these biological processes. In particular, analyses in yeast of the respective contributions of the SUMO-processing and the deconjugation functions of Ulp1 might help to determine whether its altered localization and/or levels ultimately lead to the nucleoporin mutant phenotypes. Moreover, the inventory of SUMO targets that specifically regulate each of these biological functions also remains to be assessed in most cases, a task that will likely necessitate, in addition to candidate approaches, comparative analyses of SUMO-modified proteomes in dedicated mutants of nucleoporins and of sumoylation or desumoylation enzymes.

### Acknowledgements

We thank the reviewers for fruitful comments that led to improvement of the manuscript, and colleagues for permission to use previously published images. We apologize to colleagues whose work was not directly cited because of space limitation. We thank Alain Nicolas, Andrés Aguilera and Xiaolan Zhao for fruitful collaboration on these topics during the past years. Research in our laboratory was funded by a collaborative program between the Institut Curie and the Commissariat à l’Energie Atomique (PIC Paramètres Epigénétiques), the Ligue Nationale contre le Cancer (Equipe Labellisée 2006), the Association pour la Recherche contre le Cancer (ARC), and the Ministère de l’Education Nationale, de la Recherche et de l’Enseignement Supérieur (A.C.I. ‘Jeunes chercheurs’).

### References

1. Tran, E.J. and Wente, S.R. (2006) Dynamic nuclear pore complexes: life on the edge. *Cell* 125, 1041–1053
2. Zhao, J. (2007) Sumoylation regulates diverse biological processes. *Cell. Mol. Life Sci.* 64, 3017–3033
3. Geiss-Friedlander, R. and Melchior, F. (2007) Concepts in sumoylation: a decade on. *Nat. Rev. Mol. Cell Biol.* 8, 947–956
4. Dohmen, R.J. (2004) SUMO protein modification. *Biochim. Biophys. Acta* 1695, 113–131
5. Mukhopadhyay, D. and Dasso, M. (2007) Modification in reverse: the SUMO proteases. *Trends Biochem. Sci.* 32, 286–295
6. Matunis, M.J. *et al.* (1996) A novel ubiquitin-like modification modulates the partitioning of the Ran-GTPase-activating protein RanGAP1 between the cytosol and the nuclear pore complex. *J. Cell Biol.* 135, 1457–1470
7. Mahajan, R. *et al.* (1997) A small ubiquitin-related polypeptide involved in targeting RanGAP1 to nuclear pore complex protein RanBP2. *Cell* 88, 97–107

Review

8 Pichler, A. and Melchior, F. (2002) Ubiquitin-related modifier SUMO1 and nucleocytoplasmic transport. *Traffic* 3, 381–387

9 Melchior, F. *et al.* (2003) SUMO: ligases, isopeptidases and nuclear pores. *Trends Biochem. Sci.* 28, 612–618

10 Matunis, M.J. *et al.* (1998) SUMO-1 modification and its role in targeting the Ran GTPase-activating protein, RanGAP1, to the nuclear pore complex. *J. Cell Biol.* 140, 499–509

11 Saitoh, H. *et al.* (1997) RanBP2 associates with Ubc9p and a modified form of RanGAP1. *Proc. Natl. Acad. Sci. U. S. A.* 94, 3736–3741

12 Saitoh, H. *et al.* (1998) Ubc9p and the conjugation of SUMO-1 to RanGAP1 and RanBP2. *Curr. Biol.* 8, 121–124

13 Lee, G.W. *et al.* (1998) Modification of Ran GTPase-activating protein by the small ubiquitin-related modifier SUMO-1 requires Ubc9, an E2-type ubiquitin-conjugating enzyme homologue. *J. Biol. Chem.* 273, 6503–6507

14 Zhang, H. *et al.* (2002) Enzymes of the SUMO modification pathway localize to filaments of the nuclear pore complex. *Mol. Cell. Biol.* 22, 6498–6508

15 Saitoh, H. *et al.* (2002) Perturbation of SUMOlation enzyme Ubc9 by distinct domain within nucleoporin RanBP2/Nup358. *J. Biol. Chem.* 277, 4755–4763

16 Bernier-Villamor, V. *et al.* (2002) Structural basis for E2-mediated SUMO conjugation revealed by a complex between ubiquitin-conjugating enzyme Ubc9 and RanGAP1. *Cell* 108, 345–356

17 Reverter, D. and Lima, C.D. (2005) Insights into E3 ligase activity revealed by a SUMO-RanGAP1-Ubc9-Nup358 complex. *Nature* 435, 687–692

18 Pichler, A. *et al.* (2002) The nucleoporin RanBP2 has SUMO1 E3 ligase activity. *Cell* 108, 109–120

19 Zhu, S. *et al.* (2006) SUMO modification through rapamycin-mediated heterodimerization reveals a dual role for Ubc9 in targeting RanGAP1 to nuclear pore complexes. *Exp. Cell Res.* 312, 1042–1049

20 Rose, A. and Meier, I. (2001) A domain unique to plant RanGAP is responsible for its targeting to the plant nuclear rim. *Proc. Natl. Acad. Sci. U. S. A.* 98, 15377–15382

21 Li, S.J. and Hochstrasser, M. (2000) The yeast ULP2 (SMT4) gene encodes a novel protease specific for the ubiquitin-like Smt3 protein. *Mol. Cell. Biol.* 20, 2367–2377

22 Taylor, D.L. *et al.* (2002) Cell-cycle-dependent localisation of Ulp1, a Schizosaccharomyces pombe Pmt3 (SUMO)-specific protease. *J. Cell Sci.* 115, 1113–1122

23 Smith, M. *et al.* (2004) *Drosophila* Ulp1, a nuclear pore-associated SUMO protease, prevents accumulation of cytoplasmic SUMO conjugates. *J. Biol. Chem.* 279, 43805–43814

24 Murtas, G. *et al.* (2003) A nuclear protease required for flowering-time regulation in Arabidopsis reduces the abundance of SMALL UBIQUITIN-RELATED MODIFIER conjugates. *Plant Cell* 15, 2308–2319

25 Hang, J. and Dasso, M. (2002) Association of the human SUMO-1 protease SENP2 with the nuclear pore. *J. Biol. Chem.* 277, 19961–19966

26 Bailey, D. and O'Hare, P. (2004) Characterization of the localization and proteolytic activity of the SUMO-specific protease, SENP1. *J. Biol. Chem.* 279, 692–703

27 Li, S.J. and Hochstrasser, M. (2003) The Ulp1 SUMO isopeptidase: distinct domains required for viability, nuclear envelope localization, and substrate specificity. *J. Cell Biol.* 160, 1069–1081

28 Panse, V.G. *et al.* (2003) Unconventional tethering of Ulp1 to the transport channel of the nuclear pore complex by karyopherins. *Nat. Cell Biol.* 5, 21–27

29 Kadoya, T. *et al.* (2002) Desumoylation activity of Axam, a novel Axin-binding protein, is involved in downregulation of beta-catenin. *Mol. Cell. Biol.* 22, 3803–3819

30 Nishida, T. *et al.* (2001) Characterization of a novel mammalian SUMO-1/Smt3-specific isopeptidase, a homologue of rat axam, which is an axin-binding protein promoting beta-catenin degradation. *J. Biol. Chem.* 276, 39060–39066

31 Best, J.L. *et al.* (2002) SUMO-1 protease-1 regulates gene transcription through PML. *Mol. Cell* 10, 843–855

32 Makhnevych, T. *et al.* (2007) The role of karyopherins in the regulated sumoylation of septins. *J. Cell Biol.* 177, 39–49

33 Zhao, X. *et al.* (2004) Mlp-dependent anchorage and stabilization of a desumoylating enzyme is required to prevent clonal lethality. *J. Cell Biol.* 167, 605–611

34 Galy, V. *et al.* (2004) Nuclear retention of unspliced mRNAs in yeast is mediated by perinuclear Mlp1. *Cell* 116, 63–73

35 Palancade, B. *et al.* (2005) Pml39, a novel protein of the nuclear periphery required for nuclear retention of improper messenger ribonucleoparticles. *Mol. Biol. Cell* 16, 5258–5268

36 Lewis, A. *et al.* (2007) A nuclear envelope protein linking nuclear pore basket assembly, SUMO protease regulation, and mRNA surveillance. *J. Cell Biol.* 178, 813–827

37 Palancade, B. *et al.* (2007) Nucleoporins prevent accumulation of DNA damages by modulating Ulp1-dependent sumoylation processes. *Mol. Biol. Cell* 18, 2912–2923

38 Xu, X.M. *et al.* (2007) NUCLEAR PORE ANCHOR, the Arabidopsis homolog of Tpr/Mlp1/Mlp2/megator, is involved in mRNA export and SUMO homeostasis and affects diverse aspects of plant development. *Plant Cell* 19, 1537–1548

39 Stade, K. *et al.* (2002) A lack of SUMO conjugation affects cNLS-dependent nuclear protein import in yeast. *J. Biol. Chem.* 277, 49554–49561

40 Hannich, J.T. *et al.* (2005) Defining the SUMO-modified proteome by multiple approaches in Saccharomyces cerevisiae. *J. Biol. Chem.* 280, 4102–4110

41 Panse, V.G. *et al.* (2006) Formation and nuclear export of preribosomes are functionally linked to the small-ubiquitin-related modifier pathway. *Traffic* 7, 1311–1321

42 Johnson, E.S. and Gupta, A.A. (2001) An E3-like factor that promotes SUMO conjugation to the yeast septins. *Cell* 106, 735–744

43 Takahashi, Y. *et al.* (2000) Yeast Ulp1, an Smt3-specific protease, associates with nucleoporins. *J. Biochem. (Tokyo)* 128, 723–725

44 Makhnevych, T. *et al.* (2003) Cell cycle regulated transport controlled by alterations in the nuclear pore complex. *Cell* 115, 813–823

45 Arnaoutov, A. and Dasso, M. (2005) Ran-GTP regulates kinetochore attachment in somatic cells. *Cell Cycle* 4, 1161–1165

46 Zuccolo, M. *et al.* (2007) The human Nup107-160 nuclear pore subcomplex contributes to proper kinetochore functions. *EMBO J.* 26, 1853–1864

47 Mossessova, E. and Lima, C.D. (2000) Ulp1-SUMO crystal structure and genetic analysis reveal conserved interactions and a regulatory element essential for cell growth in yeast. *Mol. Cell* 5, 865–876

48 Chen, X.L. *et al.* (2007) Topoisomerase I-dependent viability loss in Saccharomyces cerevisiae mutants defective in both SUMO conjugation and DNA repair. *Genetics* 177, 17–30

49 Galy, V. *et al.* (2000) Nuclear pore complexes in the organization of silent telomeric chromatin. *Nature* 403, 108–112

50 Loeillet, S. *et al.* (2005) Genetic network interactions among replication, repair and nuclear pore deficiencies in yeast. *DNA Repair (Amst.)* 4, 459–468

51 Therizols, P. *et al.* (2006) Telomere tethering at the nuclear periphery is essential for efficient DNA double strand break repair in subtelomeric region. *J. Cell Biol.* 172, 189–199

52 Dobson, M.J. *et al.* (2005) The 2 μm plasmid causes cell death in Saccharomyces cerevisiae with a mutation in Ulp1 protease. *Mol. Cell. Biol.* 25, 4299–4310

53 Chen, X.L. *et al.* (2005) Misregulation of 2 μm circle copy number in a SUMO pathway mutant. *Mol. Cell. Biol.* 25, 4311–4320

54 Soustelle, C. *et al.* (2004) A new Saccharomyces cerevisiae strain with a mutant Smt3-deconjugating Ulp1 protein is affected in DNA replication and requires Srs2 and homologous recombination for its viability. *Mol. Cell. Biol.* 24, 5130–5143

55 Zhao, X. and Blobel, G. (2005) A SUMO ligase is part of a nuclear multiprotein complex that affects DNA repair and chromosomal organization. *Proc. Natl. Acad. Sci. U. S. A.* 102, 4777–4782

56 Jacquiau, H.R. *et al.* (2005) Defects in SUMO (small ubiquitin-related modifier) conjugation and deconjugation alter cell sensitivity to DNA topoisomerase I-induced DNA damage. *J. Biol. Chem.* 280, 23566–23575

57 Sacher, M. *et al.* (2006) Control of Rad52 recombination activity by double-strand break-induced SUMO modification. *Nat. Cell Biol.* 8, 1284–1290

58 Pfander, B. *et al.* (2005) SUMO-modified PCNA recruits Srs2 to prevent recombination during S phase. *Nature* 436, 428–433

59 Green, D.M. *et al.* (2003) The C-terminal domain of myosin-like protein 1 (Mlp1p) is a docking site for heterogeneous nuclear

Review

ribonucleoproteins that are required for mRNA export. *Proc. Natl. Acad. Sci. U.S.A.* 100, 1010–1015

60 Vinciguerra, P. *et al.* (2005) Perinuclear Mlp proteins downregulate gene expression in response to a defect in mRNA export. *EMBO J.* 24, 813–823

61 Kashyap, A.K. *et al.* (2005) Biochemical and genetic characterization of Yra1p in budding yeast. *Yeast* 22, 43–56

62 Feuerbach, F. *et al.* (2002) Nuclear architecture and spatial positioning help establish transcriptional states of telomeres in yeast. *Nat. Cell Biol.* 4, 214–221

63 Hediger, F. *et al.* (2002) Myosin-like proteins 1 and 2 are not required for silencing or telomere anchoring, but act in the Tel1 pathway of telomere length control. *J. Struct. Biol.* 140, 79–91

64 Andrulis, E.D. *et al.* (2002) Esc1, a nuclear periphery protein required for Sir4-based plasmid anchoring and partitioning. *Mol. Cell. Biol.* 22, 8292–8301

65 Taddei, A. *et al.* (2004) Separation of silencing from perinuclear anchoring functions in yeast Ku80, Sir4 and Esc1 proteins. *EMBO J.* 23, 1301–1312

66 Wohlschlegel, J.A. *et al.* (2004) Global analysis of protein sumoylation in *Saccharomyces cerevisiae*. *J. Biol. Chem.* 279, 45662–45668

67 Pemberton, L.F. and Paschal, B.M. (2005) Mechanisms of receptor-mediated nuclear import and nuclear export. *Traffic* 6, 187–198

68 Kohler, A. and Hurt, E. (2007) Exporting RNA from the nucleus to the cytoplasm. *Nat. Rev. Mol. Cell Biol.* 8, 761–773

69 Zhou, L. and Watts, F.Z. (2005) Nep1, a *Schizosaccharomyces pombe* deneddylating enzyme. *Biochem. J.* 389, 307–314

70 Bylebyl, G.R. *et al.* (2003) The SUMO isopeptidase Ulp2 prevents accumulation of SUMO chains in yeast. *J. Biol. Chem.* 278, 44113–44120

71 Alber, F. *et al.* (2007) The molecular architecture of the nuclear pore complex. *Nature* 450, 695–701

72 Hase, M.E. and Cordes, V.C. (2003) Direct interaction with nup153 mediates binding of Tpr to the periphery of the nuclear pore complex. *Mol. Biol. Cell* 14, 1923–1940

73 Kim, Y.H. *et al.* (2005) Desumoylation of homeodomain-interacting protein kinase 2 (HIPK2) through the cytoplasmic-nuclear shuttling of the SUMO-specific protease SENP1. *FEBS Lett.* 579, 6272–6278

---

**Endeavour**

The quarterly magazine for the history and philosophy of science.

You can access *Endeavour* online on ScienceDirect, where you'll find book reviews, editorial comment and a collection of beautifully illustrated articles on the history of science.

Featuring:

- Information revolution: William Chambers, the publishing pioneer by A. Fyfe
- Does history count? by K. Anderson
- Waking up to shell shock: psychiatry in the US military during World War II by H. Pols
- Deserts on the sea floor: Edward Forbes and his azoic hypothesis for a lifeless deep ocean by T.R. Anderson and T. Rice
- 'Higher, always higher': technology, the military and aviation medicine during the age of the two world wars by C. Kehrt
- Bully for *Apatosaurus* by P. Brinkman

Coming soon:

- Environmentalism out of the Industrial Revolution by C. Macleod
- Pandemic in print: the spread of influenza in the Fin de Siècle by J. Mussell
- Earthquake theories in the early modern period by F. Willmoth
- Science in fiction - attempts to make a science out of literary criticism by J. Adams
- The birth of botanical *Drosophila* by S. Leonelli

And much, much more...

*Endeavour* is available on ScienceDirect, www.sciencedirect.com
