
JOURNAL OF BACTERIOLOGY, July 2008, p. 4460–4469
0021-9193/08/$08.00+0 doi:10.1128/JB.00270-08
Copyright © 2008, American Society for Microbiology. All Rights Reserved.

Functional Analysis of the Protein Machinery Required for Transport  
of Lipopolysaccharide to the Outer Membrane of *Escherichia coli*

Paola Sperandeo,$^{1}$ Fion K. Lau,$^{2}$ Andrea Carpentieri,$^{3}$ Cristina De Castro,$^{3}$ Antonio Molinaro,$^{3}$  
Gianni Dehò,$^{4}$ Thomas J. Silhavy,$^{2}$ and Alessandra Polissi$^{1*}$

Dipartimento di Biotecnologie e Bioscienze, Università di Milano-Bicocca, Milan, Italy$^{1}$; Department of Molecular Biology,  
Princeton University, Princeton, New Jersey 08544$^{2}$; Dipartimento di Chimica Organica e Biochimica, Università di  
Napoli Federico II, Naples, Italy$^{3}$; and Dipartimento di Scienze biomolecolari e Biotecnologie,  
Università degli Studi di Milano, Milan, Italy$^{4}$

Received 22 February 2008/Accepted 8 April 2008

Lipopolysaccharide (LPS) is an essential component of the outer membrane (OM) in most gram-negative bacteria, and its structure and biosynthetic pathway are well known. Nevertheless, the mechanisms of transport and assembly of this molecule at the cell surface are poorly understood. The inner membrane (IM) transport protein MsbA is responsible for flipping LPS across the IM. Additional components of the LPS transport machinery downstream of MsbA have been identified, including the OM protein complex LptD/LptE (formerly Imp/RlpB), the periplasmic LptA protein, the IM-associated cytoplasmic ATP binding cassette protein LptB, and LptC (formerly YrbK), an essential IM component of the LPS transport machinery characterized in this work. Here we show that depletion of any of the proteins mentioned above leads to common phenotypes, including (i) the presence of abnormal membrane structures in the periplasm, (ii) accumulation of de novo-synthesized LPS in two membrane fractions with lower density than the OM, and (iii) accumulation of a modified LPS, which is ligated to repeating units of colanic acid in the outer leaflet of the IM. Our results suggest that LptA, LptB, LptC, LptD, and LptE operate in the LPS assembly pathway and, together with other as-yet-unidentified components, could be part of a complex devoted to the transport of LPS from the periplasmic surface of the IM to the OM. Moreover, the location of at least one of these five proteins in every cellular compartment suggests a model for how the LPS assembly pathway is organized and ordered in space.

The cell envelope of gram-negative bacteria consists of an inner (IM) and an outer membrane (OM) separated by an aqueous compartment, the periplasm, which contains the peptidoglycan layer. The OM is an asymmetric bilayer, with phospholipids in the inner leaflet and lipopolysaccharides (LPS) facing outward (29, 32). The OM is an effective permeability barrier that protects the cells from toxic compounds, such as antibiotics and detergents, thus allowing bacteria to inhabit several different and often hostile environments. LPS is responsible of most of the permeability properties of the OM and consists of the lipid A moiety (a glucosamine-based phospholipid) linked to the short core oligosaccharide and the distal O-antigen polysaccharide chain. The core oligosaccharide can be further divided into an inner core, composed of 3-deoxy-D-manno-octulosanate (KDO) and heptose, and an outer core, which has a somewhat variable structure. LPS is essential in most gram-negative bacteria, with the notable exception of *Neisseria meningitidis* (39).

The biogenesis of the OM implies that the individual components are transported from the site of synthesis to their final destination outside the IM by crossing both hydrophilic and hydrophobic compartments. The machinery and the energy source that drive this process are not yet understood. The lipid A-core moiety and the O-antigen repeat units are synthesized at the cytoplasmic face of the IM and are separately exported via two independent transport systems, namely, the O-antigen transporter Wzx (13, 17) and the ATP binding cassette (ABC) transporter MsbA that flips the lipid A-core moiety from the inner leaflet to the outer leaflet of the IM (12, 28, 45). O-antigen repeat units are then polymerized in the periplasm by the Wzy polymerase and ligated to the lipid A-core moiety by the WaaL ligase (reference 29 and references therein). *Escherichia coli* K-12 LPS is missing the O antigen, as an IS5 insertion disrupts its synthesis (18). Very recently, a modified LPS in which repeating units of colanic acid, a cell surface polysaccharide synthesized by enteric bacteria in the presence of envelope-damaging stresses (42), are ligated to the core oligosaccharide in a WaaL-dependent manner has been described (21).

How LPS reaches the OM is less well understood. A protein complex in the OM of *E. coli* composed of LptD (formerly Imp), an essential β-barrel OM protein (6), and LptE (formerly RlpB), an essential OM lipoprotein, has recently been implicated in LPS assembly (43). Depletion of either protein results in similar OM biogenesis defects, including increased LPS levels, abnormal membrane structures, and activation of the OM enzyme PagP (43). These findings indicate that the LptD/LptE complex is responsible for LPS assembly at the outer surface of the OM (43). LptD has also been shown to be

* Corresponding author. Mailing address: Dipartimento di Biotecnologie e Bioscienze, Università di Milano-Bicocca, Piazza della Scienza 2, 20126 Milan, Italy. Phone: 39-02-64483431. Fax: 39-02-6448-3450. E-mail: alessandra.polissi@unimib.it.
† Supplemental material for this article may be found at http://jb.asm.org/.
Published ahead of print on 18 April 2008.

TABLE 1. Bacterial strains and plasmids

| Strain or plasmid | Relevant characteristics | Reference |
|-------------------|-------------------------|-----------|
| Strains           |                         |           |
| AM604             | MC4100 ara⁺             | 43        |
| AM661             | AM604 ΔlptD::kan/araBp-lptD | 20        |
| AM679             | AM604 ΔlptE::kan/araBp-lptE | This study |
| AM689             | AM604 ΔlptE::kan Δ(latt-lom) ::bla araBp-lptE | 43        |
| BB-3              | BW25113 Ω(kan araC araBp-lptC) 1 | 36        |
| BB-4              | BW25113 Ω(kan araC araBp-lptA) 1 | 36        |
| BW25113           | Δ(araD-araB) 567 ΔlacZ 4787 (::rrnB-4) lacIq λ⁻ rpoS 396 (Am) Δ(rhaD-rhaB) rrnB-4 hsdR 514 | 11        |
| DH5α              | Δ(argF-lac169) φ80dlacZ 58 (M15) glnV 44 (AS) λ⁻ rfbD1 gyrA96 recA1 endA1 spoT1 thi-1 hsdR17 | 15        |
| DY378             | W3110 λcI857 Δ(cro-bioA) | 44        |
| FL750             | AM661 ΔwaaL::cam         | This study |
| FL752             | AM679 ΔwaaL::cam         | This study |
| FL905             | AM604 Ω(kan araC araBp-lptC) 1 | This study |
| FL907             | AM604 Ω(kan araC araBp-lptA) 1 | This study |
| FL909             | MC4100-BB3 ΔwaaL::cam    | This study |
| FL911             | MC4100-BB4 ΔwaaL::cam    | This study |
| MC4100            | F⁻ araD139 Δ(argF-lac) U169 rpsL150 relA1 flb5301 deoC1 ptsF25 thi | 7         |
| MG1655            | LAM rph-1               | 1         |
| Plasmids          |                         |           |
| pBAD18            | Arabinoe-inducible expression vector, Ampʳ | 14        |
| pBADlptE          | pBAD18 araBp-lptE       | This study |
| pGS100            | pGZ119EH derivative, contains TIR sequence downstream of ptac | 36        |
| pGS108            | pGS100 ptac-lptC-His₆   | This study |
| pKD3              | oriRγ; Ampʳ Camʳ; source of cat cassette | 11        |
| pKD4              | oriRγ; Ampʳ Kanʳ; source of kan cassette | 11        |

required for proper transport of LPS to the cell surface of *N. meningitidis* (5). This conclusion was based on loss of surface accessibility of LPS to neuraminidase and loss of lipid A modification by the OM deacylase PagL (5).

More recently, two additional essential *E. coli* proteins, LptA and LptB (formerly YhbN and YhbG, respectively), have been implicated in LPS transport to the OM (35). LptA is a periplasmic protein (38), whereas LptB, a cytoplasmic protein possessing the ABC signature, has been found to be associated with the IM (40). Mutants depleted of LptA and/or LptB have abnormal membrane structures in the periplasm, produce an anomalous LPS form characterized by ladderlike banding of higher-molecular-weight species, and, more importantly, do not transport de novo-synthesized LPS to the OM (35). Based on these findings, it has been suggested that LptA and LptB, together with as-yet-unidentified transmembrane partners, may form a membrane-associated complex required for LPS transport across the periplasm (35).

In this paper we provide evidence that LptA, LptB, LptD, LptE, and YrbK (renamed LptC), an essential IM protein characterized here, are all members of the same LPS transport machinery. In fact, depletion of any of these proteins blocks the LPS assembly pathway in nearly the same fashion, which results in very similar phenotypes. Moreover, the location of at least one of these five proteins in every cellular compartment suggests a model for how the LPS assembly pathway is organized and ordered in space.

MATERIALS AND METHODS

Bacterial strains, plasmids, and media. The bacterial strains and plasmids used in this work are listed in Table 1. Plasmid pBADlptE was constructed as follows. Primers ACM 146 (5′-AAAGAATTCGCGCGGGAGGAAGC-3′) and ACM 147 (5′-TTATCTAGACGCGGAGTTGTCC-3′) were used to amplify the *lptE* gene from MC4100. The PCR product was digested with EcoRI and XbaI and ligated to pBAD18 cut with the same enzymes. The construct was transformed into AM604. Primers ACM 143 (5′-GTAAAGTGATTTACGTAC CAGGTA AACTCCTCAATCTGGTCGTGGCTAAGTG TAGGGTGGAGC TGCTTC-3′) and ACM 145 (5′-CTAATCGGGTAGATATCACGGCCGGGG ATCAAACACGGTCGCATTAACCGTCATATGAATATCCCTTAC-3′) were used to amplify the kanamycin resistance cassette from pKD4 (11) by PCR. In the product of this reaction the *kan* cassette is flanked by 50-bp regions with homology to the *lptE* locus. These oligonucleotides were used in a recombineering reaction to replace the chromosomal *lptE* gene by using the protocol of Copeland et al. (9). Strain DY378 carrying *araBp-lptE* was used as a recipient strain. Strain AM679 was obtained by introduction of the *lptE::kan* allele by P1 transduction into AM604 carrying pBADlptE.

Depletion mutants FL905 and FL907 were created by P1 transduction transferring the *kan-araBp-lptC* and *kan-araBp-lptA* cassettes from mutants BB-3 and BB-4, respectively (36), in AM604 (MC4100 *ara*⁺) (43).

*waaL* was inactivated by using a λ-Red recombineering technique adapted from previously described methods (10), as described above. A chloramphenicol cassette was engineered into the *waaL* gene by PCR amplifying the pKD3 plasmid (11) with primer FL9 (5′-ATGCTAACATCCTTTAAACTTCATTGAAAACCTTACA CTCTGAAAATCGTG TAGGCTGGAGCTGCTTC-3′) and primer FL10 (5′-TTA ATTAATTGTATTGTTACGATTATTAATGACGAGTAAGAGGACTATAGC ATATGAATATCCTCCTTAG-3′). The PCR product was introduced into DY378 by using a standard electroporation technique. The *waaL::cam* locus was transferred into FL905, FL907, AM661, and AM679 by P1 transduction (34). Bacteria were grown in LD broth (33). When required, 0.2% l-arabinose (as an inducer of the *araBp* promoter), 0.2% glucose, 0.5 mM isopropyl-β-d-thiogalactopyranoside (IPTG) for overexpression of LptC with a C-terminal His₆ tag (LptC-H) or 1 mM IPTG, 100 μg/ml ampicillin, 50 μg/ml kanamycin, and 25 μg/ml chloramphenicol were added. Solid media were the same as the media described above except that they contained 1% agar.

Plasmid pGS108 expresses LptC-H under control of a *ptac* (IPTG-inducible) promoter. It was constructed by cloning the *lptC* open reading frame obtained by PCR amplification of MG1655 DNA with primers AP54 (5′-CGAGAGGAAT TCACCATGAGTAAAGCCAGACGTTGGG-3′) and AP63 (5′-GTGATCAC ATCTAGATCAGTGGTGGTGGTGGTGGGCTGAGTTTGT TTGT-3′). The PCR product was digested with EcoRI and XbaI and ligated to pGS100 cut with the same enzymes. The EcoRI-XbaI insert in pGS108 was verified by DNA sequencing.

LPS extraction and analysis by immunoblotting. Bacterial cultures grown at 30°C in LD medium or LD medium with arabinose to an optical density at 600
nm (OD<sub>600</sub>) of 0.2 were harvested by centrifugation, washed in LD medium, and diluted 250- or 20-fold in LD medium with or without arabinose. Samples at an OD<sub>600</sub> of 2 were taken at different time points, and LPS was extracted from the cell pellets by a mini phenol-water extraction procedure as described previously (30). Briefly, the cells were resuspended in water and pelleted by centrifugation (5 min, 10,000 × g) to remove the exopolysaccharides. The pellet was resuspended in 0.5 ml phosphate buffer (pH 7) and thoroughly vortexed; 0.5 ml phenol equilibrated with 0.1 M Tris-HCl at pH 5.5 was added, and the suspension was vortexed. The tubes were placed in a 65°C heating block for 15 min, thoroughly vortexed every 5 min, and then cooled in ice. After centrifugation (5 min, 10,000 × g), the water phase was removed, dialyzed (2,000-molecular-weight cutoff) against phosphate buffer (pH 7), and dried under a vacuum. Air-dried material was then dissolved in 30 μl sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) sample buffer. LPS was separated by Tricine-SDS-PAGE as described previously (37). LPS were transferred onto a nitrocellulose membrane (Whatman, Inc.) at 100 V for 1 h using an electroblotting apparatus (Bio-Rad) and were immunodetected using a 1:3,000 dilution of the anti-LPS WN1 222-5 monoclonal antibody (HyCult Biotechnology b.v.).

Analysis of LPS composition and analytical methods. The LPS fraction was separated using gel permeation chromatography on a Sephadex G-100 column (100 by 3 cm) eluted with a buffer consisting of 0.2 M NaCl, 0.25% deoxycholate, 1 mM EDTA, 0.02% NaN<sub>3</sub>, and 10 mM Tris in water (pH 8.0) at 20°C. The results were monitored with a differential refractometer (Knauer), and 3-ml fractions were collected and were successively dialyzed against 0.2 M NaCl, 1 mM EDTA, 10 mM Tris-HCl, 0.02% (wt/vol) NaN<sub>3</sub> buffer for 3 days, then against 0.2 M NaCl buffer for 3 days, and finally against water for 3 days. Each fraction was analyzed by an SDS-PAGE procedure as described previously (37). Determination of sugar residues and of their absolute configurations, gas-liquid chromatography (GLC), and GLC-mass spectrometry (MS) were all carried out as described previously (22, 23). Monosaccharides were identified as acetylated O-methyl glycoside derivatives. After methanolysis (2 M HCl–methanol, 85°C, 24 h) and acetylation with acetic anhydride in pyridine (85°C, 30 min), the sample was analyzed by GLC-MS. A linkage analysis was carried out by methylation of the complete LPS fraction and also of the dephosphorylated LPS as described previously (8), and identical results were obtained. The sample was hydrolyzed with 4 M trifluoroacetic acid (100°C, 4 h), carbonyl reduced with NaBD<sub>4</sub>, carboxyl methylated, carboxyl reduced, acetylated, and analyzed by GLC-MS. Oligosaccharide OS1 was obtained by hydrolysis of LPS with acetate buffer at pH 4.5 at 100°C for 2 h (22).

NMR spectroscopy. For structural assignment, one-dimensional and two-dimensional <sup>1</sup>H nuclear magnetic resonance (NMR) spectra were recorded using a solution consisting of 0.3 mg in 0.5 ml of D<sub>2</sub>O at 300 K and pD 7 (uncorrected value) with a Bruker 600 DRX spectrometer equipped with a cryoprobe. Spectra were calibrated with internal acetone (δ<sub>H</sub> 2.225, δ<sub>C</sub> 31.45). <sup>31</sup>P NMR experiments were carried out using a Bruker DRX-400 spectrometer, and aqueous 85% phosphoric acid was used as the external reference (0.00 ppm). A rotating frame Overhauser enhancement spectroscopy experiment was performed using data sets of 4,096 × 1,024 points, and 64 scans were acquired with a mixing time of 200 ms; a double quantum-filtered phase-sensitive correlation spectroscopy experiment was performed with an acquisition time of 0.258 s using data sets of 4,096 × 1,024 points, and 128 scans were acquired; and a total correlation spectroscopy experiment was performed with a spinlock time of 100 ms using data sets of 4,096 × 1,024 points, and 64 scans were acquired. In all homonuclear experiments the data matrix was zero filled in the F1 dimension to obtain a matrix of 4,096 × 2,048 points and was resolution enhanced in both dimensions by a shifted sine-bell function before Fourier transformation. Coupling constants were determined on a first-order basis using two-dimensional phase-sensitive double quantum-filtered correlation spectroscopy (27). The heteronuclear single quantum coherence and heteronuclear multiple bond correlation experiment spectra were measured in the <sup>1</sup>H-detected mode with proton decoupling in the <sup>13</sup>C (or <sup>31</sup>P) domain, using data sets of 2,048 × 512 points, and 64 scans were acquired for each t1 value. The experiments were carried out in the phase-sensitive mode (38) <sup>1</sup>H, <sup>13</sup>C heteronuclear multiple bond correlation was optimized for a 6-Hz coupling constant, and <sup>1</sup>H, <sup>31</sup>P heteronuclear single quantum coherence was optimized for an 8-Hz coupling constant. In all the heteronuclear experiments the data matrix was extended to 2,048 × 1,024 points using forward linear prediction extrapolation.

MS. Negative Reflectron matrix-assisted laser desorption ionization (MALDI) spectra were recorded using a Voyager DE STR instrument (Applied Biosystems, Framingham, MA). The MALDI matrices were prepared by dissolving 20 mg of 2,3-dihydroxybenzoic acid in 1 ml of acetonitrile-water (90:10, vol/vol). Typically, 1 μl of matrix was applied to the metallic sample plate, and 1 μl of analyte was then added. The acceleration and reflector voltages were set up as follows: a target voltage of 20 kV and the first grid at 95% of the target voltage, with delayed extraction at 600 ns to obtain the best signal-to-noise ratios and the best possible isotopic resolution. Each spectrum represents the sum of 1,500 laser pulses from randomly chosen spots per sample position. All analyses were conducted in triplicate.

Raw data were analyzed using the computer software provided by the manufacturers and are expressed as monoisotopic masses.

[<sup>3</sup>H]GlnNAc pulse-labeling and cell fractionation. Mutants were grown in LD medium with arabinose to an OD<sub>600</sub> of 0.2 at 30°C in the presence of 0.2% N-acetylglucosamine (GlnNAc) to induce GlnNAc uptake. Cells were then harvested, washed in LD medium, diluted 200- and 50-fold (FL905 and FL907) and 20-fold (AM689 and AM661) in 50 and 100 ml fresh medium with and without arabinose, respectively, and incubated with aeration at 30°C. N-Acetyl[<sup>3</sup>H]-glucosamine ([<sup>3</sup>H]GlnNAc) pulse-labeling (see below) of mutants grown with arabinose was performed when the cultures reached an OD<sub>600</sub> of 0.3 to 0.4, whereas depleted cultures grown without arabinose were labeled 1 h after the cultures reached the maximal OD<sub>600</sub> (OD<sub>600</sub> between 0.2 and 0.4). [<sup>3</sup>H]-GlnNAc pulse-labeling was performed by adding [<sup>3</sup>H]GlnNAc (1.5 μCi ml<sup>−1</sup>) to a culture, followed by a chase after 2 min with 0.4% nonradioactive GlnNAc. After a 5-min chase, cells were chilled in ice and harvested by centrifugation. IM and OM were separated by discontinuous sucrose density gradient centrifugation of a total membrane fraction obtained by spheroplast lysis as described previously (26). Step gradients were prepared by layering 2 ml each of 50, 45, 40, 35, and 30% (wt/vol) sucrose solutions over a 55% sucrose cushion (0.5 ml).

Fractions (300 μl) were collected from the top of the gradient, and 16 μl of each fraction was transferred onto a nitrocellulose membrane at 100 V for 1 h or at 12 V overnight in an electroblotting apparatus (PBI) for protein profile analysis. The IM profile of the fractions was determined by immunoblotting using a 1:10,000 dilution of anti-LptD polyclonal antibody which cross-reacts with an 55-kDa IM protein (43). The distribution of the OM on the nitrocellulose membrane was immunodetected using a 1:5,000 dilution of anti-LamB polyclonal antibody.

To estimate the LPS contents in the gradient fractions, 16-μl portions of the fractions were separated by Tricine-SDS-PAGE and immunoblotting performed as described above.

To estimate <sup>3</sup>H incorporation, 50 μl of each fraction dissolved in 4 ml scintillation liquid (Ready safe; Beckman Coulter) was counted with a liquid scintillation counter (LS6500; Beckman).

Cellular localization of LptC. Cultures of BW25113 containing plasmid pGS108 were grown in LD medium to an OD<sub>600</sub> of 0.7, and LptC-H was induced for 2 h with 0.5 mM IPTG. Periplasmic, cytoplasmic, IM, and OM fractions were prepared as described previously (25). Equal volumes of the fractions were fractionated by 10% SDS-PAGE. The tagged protein was detected by Western blotting using anti-His<sub>6</sub> monoclonal antibody (Roche). An antibody against the IM protein YidC (19) was used as a control for good fractionation.

Electron microscopy. Samples used for electron microscopy were prepared using the method of Ogura et al. (24). Thin (70-nm) sections were obtained by using a diamond knife in a Leica UC6 ultramicrotome and were observed at 80 kV with a Zeiss 912AB transmission electron microscope equipped with an Omega energy filter. Micrographs were captured by using a digital camera obtained from Advanced Microscopy Techniques and were saved as TIFF files onto a Dell personal computer.

---

**RESULTS**

LptC is an essential IM protein. In previous work it was demonstrated that *lptC*, an open reading frame upstream of *lptA* and *lptB*, is essential, as the *lptC* conditional expression mutant BB-3 (Table 1), which harbors this gene under control of the arabinose-inducible *araBp* promoter, was unable to grow in the absence of the inducer (36).

The *lptC* gene encodes a 191-amino-acid protein (predicted molecular mass, 21.7 kDa). Primary sequence analysis revealed the presence of one transmembrane domain. To determine the subcellular localization of LptC, we constructed a tagged version of the protein carrying a C-terminal His<sub>6</sub> tag (LptC-H) under control of the IPTG-inducible *ptac* promoter (plasmid pGS108). This construct produced a functional LptC-H pro-
in LPS assembly, discrepancies were observed when results obtained under different experimental conditions and with different genetic backgrounds were compared. Similarities or differences in the terminal phenotypes of depletion strains could provide important insights into the mechanism. Thus, to address these important issues, we constructed a set of isogenic depletion strains (MC4100 derivatives) for each of the five essential proteins and determined the terminal phenotypes using a standardized set of biochemical and microscopic techniques as described below.

Structural abnormalities in cells depleted of LptA-LptB LptC, LptD, and LptE. Mutant cells depleted of LptA-LptB, LptD, and LptE exhibit striking alterations in envelope structure, such as abnormal membrane structures and accumulation of “extra” membrane material in the periplasm (35, 43). The isogenic depletion strains were grown to exponential phase and then shifted to media lacking arabinose. Samples were taken 300 min after the shift to depletion conditions and then processed for electron microscopy as described in Material and Methods. As shown in Fig. 2B, depleted cells of all strains contained strikingly similar multilayer membranous bodies that protruded into the periplasmic space that were absent in the nondepleted control. These data strongly implicate LptC in LPS biogenesis, and they suggest that there are similar defects in LPS transport in all of the depletion strains.

Depletion of LptA-LptB, LptC, LptD, and LptE affects LPS transport to the OM. We labeled the isogenic strains depleted of LptA-LptB (FL907), LptC (FL905), LptD (AM661), and LptE (AM689) and nondepleted controls with \[^{3}H\]GlnNAc and fractionated the membranes using sucrose density gradient centrifugation. Fractions collected from the gradients were assayed both for incorporated radioactivity and by immunoblotting with antibodies against LPS, the OM proteins LamB and OmpA, and a 55-kDa IM protein (43). The results are shown in Fig. 3. In nondepleted FL907 cells most of the radioactivity was found in fractions 19 to 23 corresponding to the OM, where most of the total LPS and OmpA and LamB proteins equilibrate (density, 1.20 g ml\(^{-1}\)). Upon LptA-LptB depletion the radioactivity appeared in two lighter peaks equilibrating around fractions 5 to 7 and 13 (1.15 and 1.17 g ml\(^{-1}\),

FIG. 2. Cell morphology upon depletion of LptA-LptB, LptE, LptD, and LptC. Cells grown in the presence (+) or in the absence (-) of arabinose were prepared for electron microscopy as described in Materials and Methods. Scale bars, 0.5 μm.

A

| % of total radioactivity |
| --- |
| 40 |
| 35 |
| 30 |
| 25 |
| 20 |
| 15 |
| 10 |
| 5 |
| 0 |
| 0 | 5 | 10 | 15 | 20 | 25 | 30 | 35 |
| fraction |

+ ara
- ara
+ ara
- ara
+ ara
- ara

LPS
LamB
OmpA
55 kDa

B

| % of total radioactivity |
| --- |
| 40 |
| 35 |
| 30 |
| 25 |
| 20 |
| 15 |
| 10 |
| 5 |
| 0 |
| 0 | 5 | 10 | 15 | 20 | 25 | 30 | 35 |
| fraction |

+ ara
- ara
+ ara
- ara
+ ara
- ara

LPS
LamB
OmpA
55 kDa

C

| % of total radioactivity |
| --- |
| 40 |
| 35 |
| 30 |
| 25 |
| 20 |
| 15 |
| 10 |
| 5 |
| 0 |
| 0 | 5 | 10 | 15 | 20 | 25 | 30 | 35 |
| fraction |

+ ara
- ara
+ ara
- ara
+ ara
- ara

LPS
LamB
OmpA
55 kDa

D

| % of total radioactivity |
| --- |
| 40 |
| 35 |
| 30 |
| 25 |
| 20 |
| 15 |
| 10 |
| 5 |
| 0 |
| 0 | 5 | 10 | 15 | 20 | 25 | 30 | 35 |
| fraction |

+ ara
- ara
+ ara
- ara
+ ara
- ara

LPS
LamB
OmpA
55 kDa

4464

respectively), with the former peak corresponding to the IM, as judged by the 55-kDa IM protein profile. Moreover, a substantial amount of the total LPS equilibrated in fractions lighter than the OM (fractions 1 to 15) (Fig. 3A). This is consistent with previous data obtained in a different strain background (BB-4) harboring the same conditional mutation and with strains depleted of each of the two proteins individually (35). Moreover, as previously observed (35), in the lighter fractions a modified form of LPS, characterized by ladderlike banding of higher-molecular-weight species, was visible. However, the accumulation of total LPS in lighter fractions upon depletion did not cause a pronounced change in the OM density, as judged by the LamB and OmpA profile (Fig. 3A, right).

A fractionation profile very similar to that obtained upon LptA-LptB depletion was exhibited by cells depleted of LptD, LptE, and LptC. In nondepleted strains AM661 and AM689 and FL905 (Fig. 3B, C, and D), most of total and newly synthesized LPS was found in the fractions in which the OM markers LamB and OmpA sedimented (fractions 19 to 23) (compare Fig. 3A with Fig. 3B, C, and D). In depleted cells, similar to what was observed upon LptAB depletion, radioactivity appeared in two lighter peaks equilibrating at fractions 7 and 13. Moreover, in the three depleted strains a substantial amount of total LPS was found to be associated with fractions 1 to 17 with density lower than that of the OM. The ladderlike banding of high-molecular-weight LPS species observed upon LptA-LptB depletion was also apparent in the other three depleted strains. Contrary to what was previously reported (43), an increase in OM density was not observed upon LptD and LptE depletion, as judged by the OmpA and LamB profiles. In AM661 depleted cells the OM density remained the same (Fig. 3B, right), whereas in AM689 depleted cells it was shifted slightly toward lower density as the OM equilibrated around fraction 19 instead of fraction 21 (Fig. 3C, right). The discrepancies with previously published data are thus likely due to differences in the methods used for cell breakage and membrane fractionation. In any event, the data presented here were obtained with isogenic strains and standardized protocols, and thus the results are directly comparable.

In summary, the membrane fractionation patterns of all of the depletion strains grown under permissive conditions were quite similar. The OM equilibrated around fractions 19 to 21, where most of the radioactivity and total LPS were found (Fig. 3). In contrast, in all depleted strains the radioactivity appeared in two lighter peaks equilibrating around fractions 7 and 13, and a substantial amount of LPS was shifted toward lighter fractions (fractions 1 to 17), showing the ladderlike banding of high-molecular-weight species also observed previously (35). These data indicate that under depletion conditions de novo-synthesized LPS does not reach the OM and accumulates in the lighter part of the gradient in a modified form.

Altogether, these results suggest (i) that LptA, LptB, LptC, LptD, and LptE operate in the same pathway (namely, the transport of de novo-synthesized LPS to the OM) and (ii) that when this pathway is impaired by the loss of any one of these proteins, LPS-incorporated radioactivity and an anomalous form of LPS, visible as a ladder of bands migrating more slowly than the native LPS, accumulate in light membrane fractions.

Colanic acid is ligated to LPS in cells depleted of LptA-LptB. In order to elucidate the chemical structure of the high-molecular-mass, ladderlike bands observed when LPS transport is impaired, we isolated the LPS fraction produced by cells depleted of LptA-LptB and carried out a structural analysis. As the high-molecular-mass species represent a very small fraction of the total LPS, they were purified by gel permeation chromatography with sodium deoxycholate. Using gel filtration with a denaturing agent, we were able to separate the various LPS molecular species into eight different fractions (see Fig. S1 in the supplemental material). The fraction with the greatest relative abundance of high-molecular-weight LPS species was subjected to mild hydrolysis with acetate buffer to split the lipid A from the core oligosaccharide fraction (fraction OS1).

The OS1 fraction was subjected to full structural elucidation by chemical analysis, MS, and NMR spectroscopy, and all data converged to indicate the presence of a colanic acid repeating unit attached to the outer core region of the LPS at O-7 of the heptose residue, as described previously for a mucoid mutant of *E. coli* K-12 defective in KDO biosynthesis (21). Monosaccharide analyses showed the expected residues of the core region of LPS and colanic acid residues. In order to confirm the structural hypothesis described above, the oligosaccharide portion was analyzed by MALDI-time of flight MS and NMR spectroscopy. The MALDI-MS spectrum (Fig. 4A) showed a main molecular ion at 1796.7 that could be assigned to the complete core region of wild-type *E. coli* K-12 LPS, as previously observed (21). In agreement, peaks related to the same glycoform either lacking a phosphate group or possessing an additional 2-amino-ethyl phosphate residue were also present. At higher molecular weights of the same spectrum an ion peak at *m/z* 2834.8 was present, which was in agreement with the presence of the same core glycoform that also possessed a bis-acetylated colanic acid repeating unit (∆*m/z* 1038) lacking a pyruvate group. Other minor peaks were also present that could be assigned to the same glycoform that in addition possessed a 2-amino-ethyl phosphate residue (*m/z* 2957.6) and/or lacked a phosphate (*m/z* 2754.8) or possessed a further pyru-

FIG. 3. Membrane fractionation of cells depleted of LptA-LptB, LptD, LptE, and LptC. FL907, AM661, AM689, and FL905 cultures were grown with arabinose to an OD<sub>600</sub> of 0.2, harvested, and resuspended in an arabinose-supplemented or arabinose-free medium. About 1 h after the cultures had reached the maximal OD<sub>600</sub> (OD<sub>600</sub> between 0.2 and 0.6), cells were pulse-labeled for 2 min with [³H]GlnNAc and chased for 5 min with 0.4% nonradioactive GlnNAc; the nondepleted cultures were pulse-labeled when the same OD<sub>600</sub> was reached, as described in Materials and Methods. Total membranes prepared from cells were fractionated by sucrose density gradient. Fractions were collected from the top of the gradient and immunoblotted using antibodies recognizing LPS, LamB, and a 55-kDa IM protein as indicated. Fractions were also analyzed for total incorporated radioactivity. The panels on the left show the percentages of the total incorporated radioactivity for nondepleted (■) and depleted (○) mutant cells. The panels on the right show the LamB and OmpA profiles of nondepleted (+ ara) and depleted (− ara) mutant cells. The OmpA protein cross-reacts with the LamB antibody. (A) FL907 cells depleted and not depleted of LptA-LptB. (B) AM661 cells depleted and not depleted of LptD. (C) AM689 cells depleted and not depleted of LptE. (D) FL905 cells depleted and not depleted of LptC.

4466 SPERANDEO ET AL.

![Image Description](#)

FIG. 4. (a) MALDI-MS spectrum of the product OS1. The main molecular ion at *m/z* 1796.7 consists either of hexa-acylated lipid A or the complete core region of wild-type *E. coli* K-12 LPS, whereas the higher-molecular-mass ion peaks can be assigned to the same oligosaccharide that in addition bears one or more *bis*-acetylated colanic acid repeating units (*Δm/z* 1038) lacking a pyruvate group. (b) <sup>1</sup>H NMR spectrum of the O-deacetylated OS1 product, in which the great heterogeneity of the sample due to nonstoichiometric substitutions and reducing KDO arrangements is evident. (Inset) Anomeric assignments of the colanic acid single repeating unit as shown in Table S1 in the supplemental material. (c) Repeating unit of colanic acid. Residues are α-configured unless stated otherwise.

Pyr-Gal-(1→4)-β-GlcA-(1→3)-GalAc-(1→3)-FucAc(1→3)-β-Glc-(1→7)-Hep-core-lipid A

In conclusion, chemical analysis, MS, and NMR data completely support the hypothesis that colanic acid repeating units are attached at position O-7 of the outer core heptose residue (Fig. 4C). The same structural analysis protocol applied to wild-type *E. coli* LPS and LPS purified from BB-4 grown with arabinose revealed the expected canonical structure (data not shown).

O-antigen ligase modifies LPS in cells depleted of LptA-LptB LptC, LptD, and LptE. As both colanic acid and the O-antigen are ligated to the *L*-glycero-*D*-manno-heptose residue of the LPS outer core (13), we tested the possibility that the O-antigen ligase WaaL was responsible for this LPS modification. Disruptions of the *waaL* gene were generated in FL905, FL907, AM661, and AM679 mutant backgrounds, and the LPS profiles were analyzed for all strains grown with and without arabinose. As shown in Fig. 5, cells depleted of each protein displayed the same ladderlike banding of high-molecular-mass LPS species in total LPS preparations, confirming that this anomalous LPS form is produced when transport of LPS to the OM is impaired. Disruptions of *waaL* in each strain completely abolished the formation of the higher-molecular-weight LPS bands in cells depleted of LptE, LptD, LptA-LptB, and LptC (Fig. 5A, B, C, and D, respectively), indicating that colanic acid is dependent upon the WaaL O-antigen ligase for linkage to the LPS core.

Vol. 190, 2008

LptC AND LPS TRANSPORT MACHINERY 4467

A  
B  
C  
D  

ara + - + -  
waaL + + - -  

FIG. 5. WaaL dependence of anomalous LPS production. The strains carrying a functional O-antigen ligase (+ waaL) or in which the O-antigen ligase is disrupted (- waaL) were grown with arabinose (+ ara) or without arabinose (- ara) as described in Material and Methods. The LPS profile was determined by Western blotting using anti-LPS WN1 222-5 antibodies. (A) LptE depletion, 300 min. (B) LptD depletion, 360 min. (C) LptAB depletion, 240 min. (D) LptC depletion, 360 min.

DISCUSSION

Previous studies identified four essential proteins that are required for LPS transport from the IM to the OM (35, 43). Here we identified a fifth essential protein, LptC, that is also required. Similarities or differences in the terminal phenotypes of strains depleted of each component could provide important insights into the mechanism. Although different phenotypes were reported, it was not possible to make meaningful comparisons because of different experimental conditions and genetic backgrounds. This work provides compelling evidence, based on analysis of isogenic strains, that five proteins (LptA, LptB, LptC, LptD, and LptE) cooperate in the LPS assembly pathway to the OM. Cells depleted of any of these proteins have nearly identical phenotypes: (i) they have abnormal membrane structures in the periplasm; (ii) they do not transport de novo-synthesized LPS to the OM and instead accumulate the newly synthesized molecules in two membrane fractions with lower densities than the OM; and (iii) they accumulate a modified LPS ligated to repeating units of colanic acid.

LptC, the newly identified member of the LPS transport machinery, is an IM protein that could be part of a complex together with LptB, the cytoplasmic ABC component of a transporter, and the periplasmic protein LptA (35). LptB, which has been shown to be in a 140-kDa IM complex (40), could provide the energy from ATP hydrolysis to extract LPS from the periplasmic surface of the IM and deliver it to the LptD/LptE complex in the OM. As secondary structure prediction of LptC showed that there is only one putative transmembrane helix, additional integral membrane components may still be missing in this system. Very recently, two essential IM proteins, LptG and LptF (formerly YjgP and YjgQ, respectively), were identified and proposed to be the missing transmembrane components of the ABC transporter that together with LptB function to extract LPS from the IM en route to the

OM (31). LptD and LptE could then receive the LPS at the inner leaflet of the OM, flipping the LPS across the OM or in both of these processes (43). The five proteins described in this study operate in the LPS assembly pathway downstream of MsbA and are located in each of the cellular compartments of a gram-negative bacterium, thus suggesting how the components are organized and ordered in space, as well as the likely order of the flow of LPS molecules from the IM to the cell surface (Fig. 6). We show here that if this assembly pathway is broken at any place, LPS accumulates at the outer surface of the IM. The buildup of LPS molecules in this location allows modification with colanic acid in a reaction catalyzed by the WaaL ligase, an enzyme located in the IM facing the periplasm. This LPS modification has recently been reported by Meredith and coworkers for an E. coli mutant (KPM22) that is deficient in KDO synthesis and overproduces the YhjD flippase (21). Our data show that the higher-molecular-weight species of LPS that accumulate in the depletion strains are located largely in the IM. We believe that the extra LPS molecules form protrusions that extend into the periplasm, accounting for the extra membranes observed by electron microscopy in cells depleted of LptA-LptB, LptC, LptE, and LptD in this and previous studies (35, 43). In support of this hypothesis, it can be noted that for mutants defective in the MsbA flippase electron micrographs show membrane invaginations that extend into the cytoplasm, probably because of the tension created by the buildup of LPS on the inner surface of the IM (12). In any case, the data described above suggest that the appearance of LPS modified with colanic acid is diagnostic for defects in LPS transport occurring downstream of the MsbA-mediated flipping of the lipid A-core moiety to the periplasmic face of the IM.

The molecular mechanism of LPS transport across the periplasm is still not understood. The periplasmic protein LptA, possibly together with additional unknown partners, may work as a periplasmic LPS shuttle. However, Tefsen and
coworkers (41) have shown that LPS is still transported to the OM in spheroplasts, which should be effectively drained of periplasmic contents. Alternatively, LPS transport could proceed via multiprotein complexes working at sites of membrane adhesion known as Bayer bridges (2, 3), which might remain intact in spheroplasts. When LPS transport is blocked, newly synthesized LPS consistently accumulates in two novel membrane fractions; the lighter fraction has the density (1.15 g ml⁻¹) typical of the IM (26), and the heavier fraction has a density intermediate between that of the IM and that of the OM (35). We do not know the nature of the novel membrane fraction with intermediate density. It could correspond to the abnormal membrane fraction that is formed when the LPS transport machinery is blocked (Fig. 2). Alternatively, we speculate that this novel fraction could correspond to the OM_L fraction described by Ishidate and coworkers and thought to represent a native structure arising from zones of association of the IM and OM where newly synthesized LPS transiently accumulates on its way to the OM (16). If the hypothesis that LPS travels across the periplasm through points of contact between the IM and OM is correct, then it could be that LptA, LptB, LptC, LptD, LptE, and possibly additional proteins function at these zones of adhesion and could contribute to their formation (4). Characterization of the membrane structures formed when any of the five proteins are depleted may help to clarify this aspect of the LPS transport process. Finally, two challenges for future studies are to determine whether there are other protein components of the LPS transport machinery and to define the molecular role of each protein and the interactions of the proteins with each other.

### ACKNOWLEDGMENTS

We thank Pierre Genevoux for the gift of YidC antibodies. We also thank Roberto Assandri for technical assistance. T.J.S. and F.K.L. were supported by NIGMS award GM34821. P.S. was supported by an Ingenio-Finlombarda fellowship. This work was partially supported by Fondazione Cariplo grant 2005.1076/10.4878.

### REFERENCES

1. Bachmann, B. J. 1987. Derivatives and genotypes of some mutant derivatives of *Escherichia coli* K-12, p. 1191–1219. In J. L. Ingraham, K. B. Low, B. Magasanik, M. Schaechter, and H. E. Umbarger (ed.), *Escherichia coli* and *Salmonella typhimurium*: cellular and molecular biology. ASM Press, Washington, DC.
2. Bayer, M. E. 1968. Areas of adhesion between wall and membrane of *Escherichia coli*. J. Gen. Microbiol. **53**:395–404.
3. Bayer, M. E. 1991. Zones of membrane adhesion in the cryofixed envelope of *Escherichia coli*. J. Struct. Biol. **107**:268–280.
4. Bos, M. P., V. Robert, and J. Tommassen. 2007. Biogenesis of the gram-negative bacterial outer membrane. Annu. Rev. Microbiol. **61**:191–214.
5. Bos, M. P., B. Tefsen, J. Geurtsen, and J. Tommassen. 2004. Identification of an outer membrane protein required for the transport of lipopolysaccharide to the bacterial cell surface. Proc. Natl. Acad. Sci. USA **101**:9417–9422.
6. Braun, M., and T. J. Silhavy. 2002. Imp/OstA is required for cell envelope biogenesis in *Escherichia coli*. Mol. Microbiol. **45**:1289–1302.
7. Casadaban, M. J. 1976. Transposition and fusion of the lac genes to selected promoters in *Escherichia coli* using bacteriophage lambda and Mu. J. Mol. Biol. **104**:541–555.
8. Ciucanu, I., and F. Kerek. 1984. A simple method for the permethylation of carbohydrates. Carbohydr. Res. **131**:209–217.
9. Copeland, N. G., N. A. Jenkins, and D. L. Court. 2001. Recombineering: a powerful new tool for mouse functional genomics. Nat. Rev. Genet. **2**:769–779.
10. Costantino, N., and D. Court. 2003. Enhanced levels of lambda Red-mediated recombinants in mismatch repair mutants. Proc. Natl. Acad. Sci. USA **100**:15748–15753.
11. Datsenko, K. A., and B. L. Wanner. 2000. One-step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. Proc. Natl. Acad. Sci. USA **97**:6640–6645.
12. Doerrler, W. T., M. C. Reedy, and C. R. Raetz. 2001. An *Escherichia coli* mutant defective in lipid export. J. Biol. Chem. **276**:11461–11464.
13. Feldman, M. F., C. L. Marolda, M. A. Monteiro, M. B. Perry, A. J. Parodi, and M. A. Valvano. 1999. The activity of a putative polyisoprenol-linked sugar translocase (Wzx) involved in *Escherichia coli* O antigen assembly is independent of the chemical structure of the O repeat. J. Biol. Chem. **274**:35129–35138.
14. Guzman, L. M., D. Belin, M. J. Carson, and J. Beckwith. 1995. Tight regulation, modulation, and high-level expression by vectors containing the arabinose PBAD promoter. J. Bacteriol. **177**:4121–4130.
15. Hanahan, D. 1983. Studies on transformation of *Escherichia coli* with plasmids. J. Mol. Biol. **166**:557–580.
16. Ishidate, K., E. S. Creeger, J. Zrike, S. Deb, B. Glauner, T. J. MacAlister, and L. I. Rothfield. 1986. Isolation of differentiated membrane domains from *Escherichia coli* and *Salmonella typhimurium*, including a fraction containing attachment sites between the inner and outer membranes and the murein skeleton of the cell envelope. J. Biol. Chem. **261**:428–443.
17. Liu, D., R. A. Cole, and P. R. Reeves. 1996. An O-antigen processing function for Wzx (RfbX): a promising candidate for O-unit flippase. J. Bacteriol. **178**:2102–2107.
18. Liu, D., and P. R. Reeves. 1994. *Escherichia coli* K12 regains its O antigen. Microbiology **140**:49–57.
19. Luirink, J., G. von Heijne, E. Houben, and J. W. de Gier. 2005. Biogenesis of inner membrane proteins in *Escherichia coli*. Annu. Rev. Microbiol. **59**:329–355.
20. McCandlish, A. C. 2006. Ph.D. thesis. Princeton University, Princeton, NJ.
21. Meredith, T. C., U. Mamat, Z. Kaczynski, B. Lindner, O. Holst, and R. W. Woodard. 2007. Modification of lipopolysaccharide with colanic acid (M-antigen) repeats in *Escherichia coli*. J. Biol. Chem. **282**:7790–7798.
22. Molinaro, A., B. Lindner, C. De Castro, B. Nolting, A. Silipo, R. Lanzetta, M. Parrilli, and O. Holst. 2003. The structure of lipid A of the lipopoly-saccharide from *Burkholderia caryophylli* with a 4-amino-4-deoxy-L-arabino-pyranose 1-phosphate residue exclusively in glycosidic linkage. Chemistry **9**:1542–1548.
23. Muller-Loennies, S., B. Lindner, and H. Brade. 2003. Structural analysis of oligosaccharides from lipopolysaccharide (LPS) of *Escherichia coli* K12 strain W3100 reveals a link between inner and outer core LPS biosynthesis. J. Biol. Chem. **278**:34090–34101.
24. Ogura, T., K. Inoue, T. Tatsuta, T. Suzuki, K. Karata, K. Young, L. H. Su, C. A. Fierke, J. E. Jackman, C. R. Raetz, J. Coleman, T. Tomoyasu, and H. Matsuzawa. 1999. Balanced biosynthesis of major membrane components through regulated degradation of the committed enzyme of lipid A biosynthesis by the AAA protease FtsH (HflB) in *Escherichia coli*. Mol. Microbiol. **31**:833–844.
25. Oliver, D. B., and J. Beckwith. 1982. Regulation of a membrane component required for protein secretion in *Escherichia coli*. Cell **30**:311–319.
26. Osborn, M. J., J. E. Gander, E. Parisi, and J. Carson. 1972. Mechanism of assembly of the outer membrane of *Salmonella typhimurium*. Isolation and characterization of cytoplasmic and outer membrane. J. Biol. Chem. **247**:3962–3972.
27. Piantini, U., O. W. Sorensen, and R. R. Ernst. 1982. Multiple quantum filters for elucidating NMR coupling networks. J. Am. Chem. Soc. **104**:6800–6801.
28. Polissi, A., and C. Georgopoulos. 1996. Mutational analysis and properties of the *msbA* gene of *Escherichia coli*, coding for an essential ABC family transporter. Mol. Microbiol. **20**:1221–1233.
29. Raetz, C. R., and C. Whitfield. 2002. Lipopolysaccharide endotoxins. Annu. Rev. Biochem. **71**:635–700.
30. Reuhs, B. L., R. W. Carlson, and J. S. Kim. 1993. *Rhizobium fredii* and *Rhizobium meliloti* produce 3-deoxy-d-manno-2-octulosonic acid-containing polysaccharides that are structurally analogous to group II K antigens (capsular polysaccharides) found in *Escherichia coli*. J. Bacteriol. **175**:3570–3580.
31. Ruiz, N., L. S. Gronenberg, D. Kahne, and T. J. Silhavy. 2008. Identification of two inner-membrane proteins required for the transport of lipopolysaccharide to the outer membrane of *Escherichia coli*. Proc. Natl. Acad. Sci. USA **105**:5537–5542.
32. Ruiz, N., D. Kahne, and T. J. Silhavy. 2006. Advances in understanding bacterial outer-membrane biogenesis. Nat. Rev. Microbiol. **4**:57–66.
33. Sabbattini, P., F. Forti, D. Ghisotti, and G. Dehò. 1995. Control of transcription termination by an RNA factor in bacteriophage P4 immunity: identification of the target sites. J. Bacteriol. **177**:1425–1434.
34. Silhavy, T. J., M. L. Barman, and L. W. Enquist. 1984. Experiments with gene fusion. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY.
35. Sperandeo, P., R. Cescutti, R. Villa, C. Di Benedetto, D. Candia, G. Dehò, and A. Polissi. 2007. Characterization of *lptA* and *lptB*, two essential genes implicated in lipopolysaccharide transport to the outer membrane of *Escherichia coli*. J. Bacteriol. **189**:244–253.
36. Sperandeo, P., C. Pozzi, G. Dehò, and A. Polissi. 2006. Non-essential KDO biosynthesis and new essential cell envelope biogenesis genes in the *Escherichia coli yrbG-yhbG* locus. Res. Microbiol. **157**:547–558.
37. Sprott, G. D., S. F. Kova, and C. A. Schnaitman. 1994. Cell fractionation, p. 72–103. In P. Gerhardt, R. G. E. Murray, W. A. Wood, and N. R. Krieg.
(ed.), Methods for general and molecular bacteriology. ASM Press, Washington, DC.

38. States, D. J., R. A. Haberkorn, and D. J. Ruben. 1982. A two-dimensional nuclear Overhauser experiment with pure absorption phase in four quadrants. J. Magn. Reson. 48:286–292.

39. Steeghs, L., R. den Hartog, A. den Boer, B. Zomer, P. Roholl, and P. van der Ley. 1998. Meningitis bacterium is viable without endotoxin. Nature 392:449–450.

40. Stenberg, F., P. Chovanec, S. L. Maslen, C. V. Robinson, L. L. Ilag, G. von Heijne, and D. O. Daley. 2005. Protein complexes of the Escherichia coli cell envelope. J. Biol. Chem. 280:34409–34419.

41. Tefsen, B., J. Geurtsen, F. Beckers, J. Tommassen, and H. de Cock. 2005. Lipopolysaccharide transport to the bacterial outer membrane in spheroplasts. J. Biol. Chem. 280:4504–4509.

42. Whitfield, C., and I. S. Roberts. 1999. Structure, assembly and regulation of expression of capsules in *Escherichia coli*. Mol. Microbiol. 31:1307–1319.

43. Wu, T., A. C. McCandlish, L. S. Groenberger, S. S. Chng, T. J. Silhavy, and D. Kahne. 2006. Identification of a protein complex that assembles lipopolysaccharide in the outer membrane of *Escherichia coli*. Proc. Natl. Acad. Sci. USA 103:11754–11759.

44. Yu, D., H. M. Ellis, E. C. Lee, N. A. Jenkins, N. G. Copeland, and D. L. Court. 2000. An efficient recombination system for chromosome engineering in *Escherichia coli*. Proc. Natl. Acad. Sci. USA 97:5978–5983.

45. Zhou, Z., K. A. White, A. Polissi, C. Georgopoulos, and C. R. Raetz. 1998. Function of *Escherichia coli* MsbA, an essential ABC family transporter, in lipid A and phospholipid biosynthesis. J. Biol. Chem. 273:12466–12475.
