
A 52-kDa Leucyl Aminopeptidase from *Treponema denticola* Is a Cysteinyglycinase That Mediates the Second Step of Glutathione Metabolism*

Received for publication, February 7, 2008, and in revised form, April 28, 2008. Published, JBC Papers in Press, May 15, 2008, DOI 10.1074/jbc.M801034200

Lianrui Chu$^{1}$, Yanlai Lai$^{\dagger}$, Xiaoping Xu$^{\S}$, Scott Eddy$^{\dagger}$, Shuang Yang$^{\dagger}$, Li Song$^{\dagger}$, and David Kolodrubetz$^{\ddagger}$

From the Departments of $^{\dagger}$Orthodontics, $^{\S}$Periodontics, and $^{\ddagger}$Microbiology and Immunology, University of Texas Health Science Center at San Antonio, San Antonio, Texas 78229

---

The metabolism of glutathione by the periodontal pathogen *Treponema denticola* produces hydrogen sulfide, which may play a role in the host tissue destruction seen in periodontitis. H₂S production in this organism has been proposed to occur via a three enzyme pathway, γ-glutamyltransferase, cysteinyglycinase (CGase), and cystalyisin. In this study, we describe the purification and characterization of *T*. *denticola* CGase. Standard approaches were used to purify a 52-kDa CGase activity from *T*. *denticola*, and high pressure liquid chromatography electrospray ionization tandem mass spectrometry analysis of this molecule showed that it matches the amino acid sequence of a predicted 52-kDa protein in the *T*. *denticola* genome data base. A recombinant version of this protein was overexpressed in and purified from *Escherichia coli* and shown to catalyze the hydrolysis of cysteinyglycine (Cys-Gly) with the same kinetics as the native protein. Surprisingly, because sequence homology indicates that this protein is a member of a family of metalloproteases called M17 leucine aminopeptidases, the preferred substrate for the *T*. *denticola* protein is Cys-Gly (*k*~cat~/K~m~ of 8.2 μM⁻¹ min⁻¹) not L-Leu-p-NA (*k*~cat~/K~m~ of 1.1 μM⁻¹ min⁻¹). The activity of CGase for Cys-Gly is optimum at pH 7.3 and is enhanced by Mn²⁺, Co²⁺, or Mg²⁺ but not by Zn²⁺ or Ca²⁺. Importantly, in combination with the two other previously purified *T*. *denticola* enzymes, γ-glutamyltransferase and cystalyisin, CGase mediates the in vitro degradation of glutathione into the expected end products, including H₂S. These results prove that *T*. *denticola* contains the entire three-step pathway to produce H₂S from glutathione, which may be important for pathogenesis.

Glutathione (L-γ-glutamyl-L-cysteinyglycine) is believed to be the major source for H₂S production in the oral cavity; human cells, especially polymorphonuclear leucocytes, have high concentrations (up to 4 mM) of glutathione that can be released when host cells are damaged in the periodontal pocket. Although a number of oral bacteria have been tested, only a few of them are able to catabolize glutathione into H₂S (3, 11, 12). *Treponema denticola*, which appears to play a significant role in the development of acute and chronic periodontal diseases in humans (13–17), is the only oral pathogen in which the proteins involved in this catabolic pathway have begun to be identified and characterized (18–22). Glutathione catabolism to H₂S (and glutamate, glycine, ammonia, and pyruvate) has been proposed to occur via a three-step enzyme pathway in this spirochete (12). In the first step, glutathione is split into glutamate and Cys-Gly. This dipeptide is then hydrolyzed into glycine and L-cysteine followed by the breakdown of L-cysteine into pyruvate, ammonia, and H₂S. The enzymes involved in the first and third steps have been purified from *T*. *denticola* and characterized (19–30). γ-Glutamyltransferase (GGT)² is a 27-kDa protein that catalyzes the cleavage of glutathione into glutamate and Cys-Gly. Cystalyisin (L-cysteine desulfhydrase) is a 46-kDa protein that converts L-cysteine into H₂S, ammonia, and pyruvate. However, neither of these enzymes can utilize Cys-Gly as a substrate (18, 19, 25), indicating that *T*. *denticola* has an additional enzyme, presumably a cysteinyglycinase (CGase), to cleave the peptide bond of Cys-Gly.

In the present study, the purification and characterization of a CGase from *T*. *denticola* is described. The *T*. *denticola* CGase is a 52-kDa protein that should be a leucyl aminopeptidase based upon its sequence homology to members of that protein family (31). Characterization of a recombinant version of the *T*. *denticola* 52-kDa protein indicates that its preferred substrate is Cys-Gly, not leucine aminopeptides. The *T*. *denticola* CGase can, in combination with GGT and cystalyisin from *T*. *denticola*, catalyze the production of H₂S from glutathione in vitro, thus proving that the proposed three-step pathway for glutathione catabolism to H₂S in *T*. *denticola* is correct.

---

The volatile sulfur compound H₂S can be produced by the metabolic activity of numerous oral bacteria, including several periodontal pathogens (1–3). This gas, which is malodorous and highly toxic (4–6), is found in high concentrations in periodontal pockets (7–9) and may play a role in some of the tissue destruction seen in periodontal diseases (7, 8, 10). H₂S can be produced from the metabolism of several molecules, but glutathione (L-γ-glutamyl-L-cysteinyglycine) is believed to be the major source for H₂S production in the oral cavity; human cells, especially polymorphonuclear leucocytes, have high concentrations (up to 4 mM) of glutathione that can be released when host cells are damaged in the periodontal pocket. Although a number of oral bacteria have been tested, only a few of them are able to catabolize glutathione into H₂S (3, 11, 12). *Treponema denticola*, which appears to play a significant role in the development of acute and chronic periodontal diseases in humans (13–17), is the only oral pathogen in which the proteins involved in this catabolic pathway have begun to be identified and characterized (18–22). Glutathione catabolism to H₂S (and glutamate, glycine, ammonia, and pyruvate) has been proposed to occur via a three-step enzyme pathway in this spirochete (12). In the first step, glutathione is split into glutamate and Cys-Gly. This dipeptide is then hydrolyzed into glycine and L-cysteine followed by the breakdown of L-cysteine into pyruvate, ammonia, and H₂S. The enzymes involved in the first and third steps have been purified from *T*. *denticola* and characterized (19–30). γ-Glutamyltransferase (GGT)² is a 27-kDa protein that catalyzes the cleavage of glutathione into glutamate and Cys-Gly. Cystalyisin (L-cysteine desulfhydrase) is a 46-kDa protein that converts L-cysteine into H₂S, ammonia, and pyruvate. However, neither of these enzymes can utilize Cys-Gly as a substrate (18, 19, 25), indicating that *T*. *denticola* has an additional enzyme, presumably a cysteinyglycinase (CGase), to cleave the peptide bond of Cys-Gly.

---

* This work was supported, in whole or in part, by National Institutes of Health Grant RO1 DE-13819. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
¹ To whom correspondence should be addressed: University of Texas Health Science Center at San Antonio, San Antonio, TX 78229. Tel.: 210-567-6269; Fax: 210-567-6858; E-mail: chul@uthscsa.edu.

² The abbreviations used are: GGT, γ-glutamyltransferase; HPLC, high pressure liquid chromatography; ESI-MS/MS, electrospray ionization tandem mass spectrometry; CGase, cysteinyglycinase; L-Leu-p-NA, L-leucine-p-nitroanilide; PBS, phosphate-buffered saline; LAP, leucine aminopeptidase.

# Characterization of Cysteinylglycinase from T. denticola

## EXPERIMENTAL PROCEDURES

### Materials and Bacterial Strains

Unless otherwise indicated, all of the chemicals and reagents were obtained from Sigma. *T. denticola* 35404 and other treponema ATCC strains were from the American Type Culture Collection (Manassas, VA), and the clinic isolates were from Dr. Stanley Holt (32). All of the bacteria used in this study were cultured anaerobically in a Coy anaerobic chamber (5% CO₂, 10% H₂, and 85% N₂) at 37°C in GM-1 broth (33) supplemented with 3.4% rabbit serum.

### Enzyme Assays

The standard reaction buffer for CGase assays was 50 mM Tris-HCl (pH 7.3) with 0.2 mM MnCl₂. Protein samples were incubated in the reaction buffer with 2 mM Cys-Gly, unless another concentration is indicated, for 20 min at 37°C, and the reactions were stopped by the addition of 5% trichloroacetic acid. The amount of the reaction product, L-cysteine, produced was measured as described by Gaitonde (34). L-Cysteine concentrations, as the optical density at 560 nm, were calculated from a standard curve with known amounts of L-cysteine, after subtracting a blank.

Leucine aminopeptidase activity was measured using L-Leu-p-NA as substrate (35). The reaction was done in 50 mM Tris-HCl (pH 8.0) with 0.2 mM MnCl₂ and 2 mM L-Leu-p-NA, unless noted otherwise. Enzyme was added, and the reaction was incubated at 37°C for 20 min before being stopped by cooling on ice. The concentration of the reaction product was measured by the absorbance at 405 nm. All of the enzyme assays were carried out in triplicate unless otherwise indicated.

### Fractionation and Purification of Native CGase from T. denticola

The purification of the CGase activity from *T. denticola* 35404 was carried out in three steps.

#### Step 1: Initial Enrichment of CGase

Briefly, whole cell pellets were diluted with PBS buffer containing 2 mM phenylmethylsulfonyl fluoride to a protein concentration of 5 mg/ml. The cells were sonicated (Branson Cell Disrupter 200; Sonic Power Co.) for 3 min at maximum output at 4°C. The soluble material was separated from cell and membrane debris by two consecutive centrifugations at 20,000 × g, 45 min. Stepwise ammonium sulfate precipitation (2.0, 2.8, and 3.6 M ammonium sulfate) of the soluble cell fraction was then used to further enrich the CGase activity. The precipitate formed at 3.6 M ammonium sulfate was collected by centrifugation, dissolved in 15 ml of 10 mM phosphate buffer (pH 7.5), and dialyzed at 4°C against the same buffer with 2 mM β-mercaptoethanol and 0.02 mM Mn²⁺ (final concentrations) for 48 h with three changes of the same buffer.

#### Step 2: Ultrafiltration by Microcon

Approximately 15 ml of the 3.6 M ammonium sulfate fraction (3.5 mg protein/ml) was applied to a Microcon 100 filter device and centrifuged at 8,000 × g at 4°C for 10 min. The flow-through (<100-kDa proteins) was then put onto a Microcon 50 filter device and centrifuged to remove proteins smaller than 50 kDa and to concentrate the CGase activity. This material was suspended in 50 mM Tris-HCl and dialyzed overnight at 4°C against a large volume of the same buffer containing 2 mM β-mercaptoethanol and 0.2 mM Mn²⁺.

#### Step 3: Purification by HiTrap Sepharose Q FF Ion Exchange Columns

After centrifugation, 14,000 rpm in a microcentrifuge for 15 min, of the dialysate from the previous step to remove precipitated protein, about 5 ml of sample was loaded into a 1-ml HiTrap Sepharose Q FF cation exchange column (GE Healthcare). Chromatography was carried out on a Beckman Coulter System Gold HPLC system at a flow rate of 1 ml/min (Beckman Coulter, Inc., Fullerton, CA). The column was washed with 50 mM Tris-HCl (pH 8.0) for 5 min to remove unbound protein, and then the bound protein, including CGase, was eluted by a 0–0.5 M gradient of NaCl in 50 mM Tris-HCl, pH 8.0. The eluted proteins were monitored by a UV detector at 280 nm. The CGase activity eluted at about 0.18 M NaCl and was usually distributed across one to three fractions.

### Identification of the Purified T. denticola Protein with CGase Activity

The three Coomassie-stained bands seen after the final purification step were excised from an SDS gel and identified, after digestion with trypsin, by capillary HPLC electrospray ionization tandem mass spectrometry (36). The resulting CID spectra were searched against the NCBI nr data base by means of Mascot (Matrix Science). Cross-correlation with X! Tandem and determination of probabilities of correct protein identification were done with Scaffold (Proteome Software).

### Cloning and Sequencing of the *cga* Gene and Purification of Recombinant CGase Protein from Escherichia coli

#### Cloning and Sequencing of the *cga* Gene in E. coli

Based upon the genomic DNA sequence of *T. denticola* strain 35405 (31), two primers, (forward, 5′-CCGCTCGAGATGAAATTATAATTGCAAAAAAAG-3′; reverse: 5′-GGGGTACCATTTTGCTTCCCTGCGGGC-3′) were designed to amplify the entire TDE0300 open reading frame, which we will refer to as *cga*, and to put an Xhol site (forward) and KpnI site (reverse) at the ends of the PCR product. With *T. denticola* 35404 genomic DNA as template, a 1.6-kb fragment was amplified by PCR, using standard protocols, and ligated into the Xhol/KpnI sites of the expression vector pRsetA (Invitrogen). The insert from a plasmid containing the *cga* gene of strain 35404 was sequenced, independently from both strands, in the Center for Advanced DNA Technology at the University of Texas Health Science Center at San Antonio.

#### Expression and Purification of Recombinant CGase

To express recombinant CGase, isopropyl β-D-thiogalactopyranoside (final concentration, 0.5 mM) was added to a culture of *E. coli* cells containing the expression plasmid. After 4 h, the bacterial culture was centrifuged at 6,000 × g for 5 min, and the cells were washed once using 20 mM PBS (pH 7.4). The cells were resuspended in 60 ml of PBS and sonicated (Branson Sonifier 450, VWR Scientific) for 5 min on ice. The soluble fraction of the cell sonicate was recovered after centrifugation at 16,500 × g for 30 min. Because the recombinant CGase has a His₆ tag, it was purified on a 2.5-ml nickel-nitrilotriacetic acid gel column (Qiagen). After loading 15 ml of the soluble cell

material onto the column, it was washed with 20× volume of 50 mM Tris-HCl (pH 8.0), 300 mM NaCl, and 20 mM imidazole. A step gradient elution with imidazole (40–250 mM in the wash buffer) was used to elute the CGase. The identity of the purified recombinant CGase was confirmed by HPLC ESI-MS/MS as described above for the native protein.

### Enzymatic Properties of the T. denticola CGase Protein

#### Determination of Optimal pH—The effect of pH on CGase activity was determined in the standard reaction buffer, except the pH was varied from pH 5.0 to 9.5 in ~0.3 to 0.5 pH units. CGase (final concentration, 5 μg/ml) and either 2 mM Cys-Gly or L-Leu-p-NA were added, and the reaction was done at 37°C for 10 min.

#### Substrate Specificity—To assess the substrate specificity profile of the T. denticola 52-kDa CGase, its activity with a number of different substrates was measured in the standard reaction buffer at 37°C for 15 min. At the end of the reaction, the concentrations of the appropriate products, L-cysteine, glycine, H₂S, pyruvate, or ammonia, were determined (12, 37–39). Because the pH optima could vary for different substrates, each substrate was tested at two different pH values. Initially, each substrate was used at a concentration of 2 mM; if there was enzyme activity at that concentration, then enzyme activity was measured over a range of substrate concentrations. The \( K_m \), \( V_{\text{max}} \), and \( k_{\text{cat}} \) values were calculated as previously described (25).

#### Effect of Cations on CGase Activity—For this set of reactions, the control reaction contained 5 μg/ml (final concentration) of purified recombinant CGase with 2 mM Cys-Gly in 50 mM Tris-HCl (pH 7.3), 0.2 mM MnCl₂ and was incubated at 37°C for 20 min. To see whether metal ions were required for enzyme activity, 0.2 mM EDTA was added to one set of tubes. To determine which cations were best at enhancing activity, 2 mM of different cations, MgCl₂, MnCl₂, CaCl₂, CoCl₂, or ZnCl₂, were preincubated separately with the enzyme in 50 mM Tris-HCl (pH 7.3) for 20 min at 37°C, and then 2 mM Cys-Gly was added, and the reaction was allowed to proceed for an additional 20 min.

### Antibody Preparation and Immunodepletion of CGase from T. denticola Lysate

Antibody against the purified recombinant CGase was produced in New Zealand White rabbits (100 μg of purified protein mixed with Freund’s incomplete adjuvant/each injection) as described previously (18). Rabbit anti-CGase IgG (20 mg) was purified, coupled to activated Sepharose 4B (40), and resuspended in 6 ml of distilled H₂O. For immunodepletion, 1 × 10¹⁰ T. denticola cells were lysed by 1% Triton X-100 in PBS containing 2 mM β-ME and 0.2 mM MnCl₂. The lysate was clarified by centrifugation at 12,000 × g for 5 min. One ml of the soluble material was combined with 0.3 ml of anti-CGase-IgG-Sepharose 4B and mixed gently by shaking at 25°C for 30 min. The supernatant was collected for analysis of CGase activity. Bacterial lysate that was not put over the Sepharose 4B column and cell supernatant that was depleted on a Sepharose 4B column that had been coupled to preimmune serum IgG were used as controls.

---

### TABLE 1  
**Purification of CGase from T. denticola 35404**  
Cys-Gly was the substrate used to assay enzyme activity.

| Fractionations                | Total protein (mg) | Enzymatic activity (mmol/mg/min) | Yield of protein (%) | Retained activity (%) |
|------------------------------|--------------------|----------------------------------|----------------------|-----------------------|
| Whole cell sonicated          | 425.5              | 1.3                              | 100.0                | 100                   |
| Soluble cell fraction         | 250.2              | 1.9                              | 58.8                 | 85                    |
| 2.8–3.6 M ASP precipitation    | 81.4               | 4.5                              | 19.1                 | 66                    |
| Microcon cut-off (50–100 kDa)  | 50.5               | 38.25                            | 11.9                 | 49                    |
| Second HiTrap Q FF column     | 0.25               | 165.4                            | 0.05                 | 7.5                   |

---

### Analysis of CGase Enzyme Activity and Gene Distribution in Other T. denticola Strains

After 2 days of growth in an anaerobic chamber, cells of various *Treponema* strains were harvested by centrifugation, washed once with PBS, and then assayed, at 1 mg protein/ml, for CGase activity. To see whether the *cga* gene was present in other strains of *T. denticola* or in other bacteria, genomic DNA was isolated from isolated from several *T. denticola*, *Treponema vincentii*, and *E. coli* strains. The DNAs were used as templates for PCR. Two primers, the forward primer (5′-ATGAAATTAAATTTGCAAAAAAAAG-3′), which starts exactly at the first codon of the *cga* open reading frame, and the reverse primer (5′-ATATTTGCTTCCCTGCGGGC-3′), which is 18 bases after the end of the *cga* open reading frame, were used for DNA amplification using standard reaction conditions.

---

### Reconstituting Glutathione Catabolism in vitro

Recombinant *T. denticola* GGT and cystealysin were purified from *E. coli* cells with the appropriate expression clones, as previously described (18, 19). Purified CGase, GGT, and cystealysin, either alone or in various combinations, were added to the standard reaction buffer so that the final concentration of each enzyme was 2 μg/ml. Glutathione was added (final concentration, 2 mM), and the reaction was allowed to proceed for 30 min at 37°C. The reaction was quenched on ice, and then the production of H₂S, ammonia, pyruvate, glutamic acid/glutamine and glycine was determined (12, 37, 38, 39). As a control, *T. denticola* 35404 cells, at a final concentration of 0.5 mg protein/ml, were also incubated with 2 mM glutathione under the same conditions as the purified proteins.

---

### RESULTS

#### Purification of Cysteinylglycinase from T. denticola—We have proposed that glutathione is catabolized by a three-step pathway in *T. denticola* and have previously identified and characterized the enzymes involved in steps 1 and 3. However, the *T. denticola* enzyme (CGase) involved in the proposed second step, cleavage of Cys-Gly into glycine and L-cysteine, has not been identified. To rectify this, we first showed that sonicated extracts of *T. denticola* strain ATCC 35404 had CGase activity (Table 1). The protein responsible for this activity was then purified more than 120-fold with a 7.5% yield of the initial total enzyme activity. Almost all of the enzyme activity was found in one fraction from the final HPLC ion exchange purification (Fig. 1a). This fraction contained one major protein band, with an approximate molecular mass of 52 kDa, on an

# Characterization of Cysteinylglycinase from T. denticola

SDS-PAGE gel (Fig. 1b) and two minor protein bands with molecular masses of 35 and 19 kDa.

To determine which of the three proteins was most likely to be the CGase, the three protein species were digested *in situ* with trypsin, and the resulting peptides were subjected to HPLC ESI-MS/MS analysis. The tandem mass spectra obtained from the 52-kDa major protein band had a significant match to the amino acid sequence of a predicted 52-kDa protein (Gene ID TDE0300) in the genomic database of *T. denticola* strain 35405 (The Institute for Genomic Research, Rockville, MD) (31). The two minor protein bands (19 and 35 kDa) also matched up with a part of the amino acid sequence of the predicted TDE0300 protein, suggesting that they are proteolytic fragments of the full-length protein. We concluded that the 52-kDa protein encoded by TDE0300 has CGase activity.

**Cloning, Sequencing, and Analysis of the 52-kDa CGase Gene of *T. denticola***—To prove that the correct protein had been identified as a cysteinylglycinase, it is easiest to clone its gene, which we will refer to as *cga*, in *E. coli* so that the recombinant gene product can be overexpressed and shown to have CGase activity. To accomplish this, the gene encoding the 52-kDa protein first had to be cloned and sequenced, because we identified the protein in a strain (strain 35404) different from the one whose genome has been sequenced (strain 35405). Thus, PCR with primers designed from the known sequence of strain 35405 (31) was used to amplify a 1.6-kb DNA fragment from *T. denticola* strain 35404. This PCR fragment, which contains the entire *cga* gene, was cloned and sequenced. The open reading frame of the *T. denticola* 35404 *cga* gene consists of 1428 bp encoding 476 amino acids (predicted molecular mass of 52.2 kDa). The *cga* nucleotide sequence from strain 35404 differs from the published 35405 sequence at 12 positions; nine of the differences result in amino acid changes in the predicted protein sequence (Fig. 2). Somewhat surprisingly, only five of the nine changes are conservative amino acid substitutions.

By sequence homology, the CGase protein from *T. denticola* is a member of the M17 family of metal-dependent leucyl aminopeptidases. This family is defined by a cluster of seven amino acids, five of which are involved in metal binding and two of which have a direct role in catalysis (41, 42). The *T. denticola* protein has all seven of these amino acids (Fig. 2).

**Overexpression, Purification, and Initial Characterization of Recombinant CGase of *T. denticola***—To investigate further the properties of CGase from *T. denticola* and to prove that the correct protein had been identified as a cysteinylglycinase, the cloned recombinant CGase was overexpressed in and purified from *E. coli*. The purified recombinant CGase migrated as a single band of the expected molecular mass on an SDS gel (Fig. 3a, inset). The purified recombinant CGase was subjected to HPLC ESI-MS/MS analysis, which confirmed that the correct protein had been purified because all of the peptides found matched to the *T. denticola* *cga* sequence. In addition, protein extract from *E. coli* cells with empty vector was subjected to the same purification scheme, and there was no CGase activity retained on the nickel-nitrilotriacetic acid affinity column. Overall, this simple purification scheme resulted in an ~170-fold purification of the protein with a yield of 75% of the total starting enzyme activity (data not shown).

The ability of the purified recombinant protein to hydrolyze Cys-Gly was compared with the activity of the protein purified directly from *T. denticola*. The native CGase had a $K_m$ of 0.48 mM and a $k_{cat}$ of 2,800 min⁻¹, and the recombinant CGase had a $K_m$ of 0.44 mM and a $k_{cat}$ of 3,600 min⁻¹. Because the kinetic

**FIGURE 1.** Fractionation of 52-kDa CGase from *T. denticola* ATCC 35404 on a HiTrap Q FF column. *a*, fractions from the separation of a partially purified protein extract on a HiTrap Sepharose Q FF ion exchange column were assayed for CGase activity (squares) and total protein (diamonds, as OD₂₈₀). *b*, SDS-PAGE analysis (18) of the proteins present in the sample that was loaded onto the ion exchange column (lane 1) and the proteins present in fraction 15 that had the peak CGase activity (lane 2). Proteins of known molecular weight were run in lane MW, and the gel was stained with 0.025% Coomassie Brilliant Blue R-250.

| T.d-35404 | MKFNIAKKGG VVAQLVFEEK IEGGYLNHLK EKELFSGKAE EVYYTLSNL KAQLFIGLGK | 60 |
| --- | --- | --- |
| T.d-35405 | **\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\
properties of the recombinant enzyme were essentially the same as those of the native enzyme and because it was easier to get purified recombinant CGase protein from *E. coli* rather than directly from *T. denticola*, the recombinant CGase was used for further characterization of this protein.

To corroborate that the 52-kDa protein represented the major, or only, CGase activity of *T. denticola*, a polyclonal rabbit antisera was prepared against the recombinant CGase protein. The monospecific antibodies were coupled to Sepharose 4B and then used to immunodeplete the soluble fraction of sonicated *T. denticola* cells. 96% of the CGase activity was depleted by immunodepletion, indicating that the 52-kDa CGase protein was the major, and probably only, CGase in this strain when grown under standard conditions (data not shown).

**Enzymatic Properties of the *T. denticola* 52-kDa Enzyme—** Because the 52-kDa protein was purified on the basis of its CGase activity, but the deduced amino acid sequence indicated that it was member of the M17 family of leucyl aminopeptidases, we first examined the substrate specificity of the purified recombinant protein (at pH 7.3). Experiments with Cys-Gly as substrate gave a $K_m$ of 0.44 mM (Fig. 3a) and a $k_{cat}/K_m$ ratio of 8.2 $\mu M^{-1}$ min$^{-1}$ (Table 2). When L-Leu-p-NA was used as the substrate, the $K_m$ was 1.72 mM (Fig. 3b), and the $k_{cat}/K_m$ ratio was 1.1 $\mu M^{-1}$ min$^{-1}$ (Table 2). No activity was detected when Glu-Cys-Gly, Asp-Gly, or L-Cys were used as substrates (Table 2). Thus, we conclude that Cys-Gly is the preferred substrate for this enzyme and thus consider it to be a CGase, even though it also has leucyl aminopeptidase activity. As a control, the substrate specificity of *T. denticola* CGase was compared with that of leucine aminopeptidase (LAP) from porcine kidney (Sigma). Although both of the substrates were hydrolyzed by both

![Figure 3](#fig3)

FIGURE 3. Analysis of the Kinetics of the *T. denticola* 52-kDa protein with Cys-Gly and L-Leu-p-NA as substrate. $a$, samples containing 5 $\mu g/ml$ of the purified recombinant *T. denticola* TDE0300 protein (shown in the SDS gel in the inset) were assayed for Cys-Gly hydrolysis at different concentrations of substrate. $b$, samples containing 5 $\mu g/ml$ of the purified recombinant *T. denticola* TDE0300 protein were assayed for leucine aminopeptidase activity at different concentrations of the substrate, L-Leu-p-NA.

enzymes, the results clearly showed that Cys-Gly was the better substrate for the *T. denticola* CGase and that Leu-p-NA was the better substrate for porcine LAP (Fig. 4). This confirms the conclusion that, unlike most other members of the M17 family of leucyl aminopeptidases, the *T. denticola* enzyme is preferentially a CGase.

To see whether altering the reaction conditions might preferentially alter the activity of the enzyme toward one substrate versus the other, the reaction temperature, ionic strength, and pH were varied. Altering the ionic strength (0.01–0.3 mM Tris-HCl) of the reaction buffer did not significantly change the activity of the enzyme, and the optimal reaction temperature was 37°C for both substrates (data not shown). However, pH did have a differential effect on CGase activity. The amidolytic activity of the enzyme toward Cys-Gly was maximal at pH 7.3/7.4 and declined fairly rapidly at higher or lower pH conditions (Fig. 5). On the other hand, the activity of the *T. denticola* enzyme with L-Leu-p-NA as substrate was highest at a pH of 8.0, and it was not as sensitive to increases or decreases in pH (Fig. 5). However, even at pH 8.0, the preferred substrate for the *T. denticola* enzyme is still Cys-Gly (Table 2).

The M17 family members are metalloproteases, so the *T. denticola* CGase is likely to have a metal requirement. To prove this, the activity of CGase was tested in the presence of EDTA; as expected there was almost no activity in the presence of this chelator (Table 3). Various metals were then added back in excess to determine which, if any, altered enzyme activity. Mn$^{2+}$, Co$^{2+}$, and Mg$^{2+}$ enhanced enzyme activity, whereas Zn$^{2+}$ and Ca$^{2+}$ supported very little CGase activity (Table 3). This result, which suggests that Mn$^{2+}$, Co$^{2+}$, or Mg$^{2+}$ is the preferred metal for activation of CGase, is similar to the result seen with bovine lens LAP, which catalyzes Cys-Gly hydrolysis in the presence of Mn$^{2+}$ but not in the presence of Zn$^{2+}$ (35).

**CGase Activity and the cga Gene Are Present in Multiple *T. denticola* Strains—** To demonstrate that the CGase activity from *T. denticola* was not strain-specific, enzyme assays were done on extracts from five other *T. denticola* strains as well as from *T. vincentii*. ATCC strains 35405 and 33520 and clinical isolates GM-1, MS-25, and SW-1 all hydrolyzed Cys-Gly with approximately the same activity as strain 35404; *T. vincentii* did not have CGase activity (data not shown). PCR was used to show that the activity in the different strains of *T. denticola* is most likely caused by the *cga* gene. Primers, designed from the known sequences of the *cga* genes from strains 35404 and 35405, amplified identically sized fragments ($\approx$1.6 kb) from all six *T. denticola* strains (data not shown). No PCR products were found when the same primers were used with DNA

TABLE 2  
Kinetic parameters of recombinant *T. denticola* CGase with different substrates  

Each value represents the mean for three experiments. The errors for each value were ≤10% of the mean. ND, no activity detected.

| Substrate | $K_m$ | $k_{cat}$ | $k_{cat}/K_m$ | $K_m$ | $k_{cat}$ | $k_{cat}/K_m$ |
|-----------|-------|------------|---------------|-------|------------|---------------|
|           | $mM$  | $min^{-1}$ | $\mu M^{-1} min^{-1}$ | $mM$  | $min^{-1}$ | $\mu M^{-1} min^{-1}$ |
| Cys-Gly   | 0.44  | $3.6 \times 10^3$ | 8.2         | 0.39  | $2.6 \times 10^3$ | 6.6         |
| Leu-p-NA  | 1.72  | $1.8 \times 10^3$ | 1.1         | 1.79  | $2.5 \times 10^3$ | 1.4         |
| Asp-Gly   | ND    | ND          | ND            | ND    | ND          | ND            |
| Glu-Cys-Gly | ND    | ND          | ND            | ND    | ND          | ND            |
| L-Cys     | ND    | ND          | ND            | ND    | ND          | ND            |

JULY 11, 2008 • VOLUME 283 • NUMBER 28                                                                                     JOURNAL OF BIOLOGICAL CHEMISTRY 19355

Characterization of Cysteinylglycinase from T. denticola

**FIGURE 4.** Comparison of the enzyme activity of the *T. denticola* 52-kDa protein and the porcine kidney leucine aminopeptidase. The enzyme activities of the *T. denticola* 52-kDa protein (CGase) and the porcine kidney LAP were compared using Cys-Gly and L-Leu-p-NA as indicated. The same amount of protein (5 μg/ml) was used in all of the experiments, and each experiment was done three times. The error bars indicate the standard errors.

**TABLE 3**

*T. denticola* CGase activity is enhanced by metals

The control reaction measured Cys-Gly hydrolysis in 50 mM Tris/HCl (pH 7.3) and 0.2 mM MnCl₂ for 20 min at 37°C.

| Reagent    | Concentration ($mM$) | Relative activity (%) |
|------------|----------------------|-----------------------|
| Control    |                      | 100                   |
| EDTA       | 0.2                  | 2 ± 0                 |
| Co²⁺       | 2.0                  | 580 ± 18              |
| Mn²⁺       | 2.0                  | 510 ± 18              |
| Mg²⁺       | 2.0                  | 258 ± 18              |
| Ca²⁺       | 2.0                  | 22 ± 1                |
| Zn²⁺       | 2.0                  | 23 ± 0                |

**FIGURE 6.** Three purified *T. denticola* enzymes can, only in combination, produce H₂S from glutathione. H₂S production from GSH was determined for different combinations, as shown, of three purified *T. denticola* proteins. The production of H₂S from glutathione was also measured with *T. denticola* (Td) cells as a positive control. CGA, purified recombinant *T. denticola* 52-kDa protein; GGT, purified *T. denticola* γ-glutamyltransferase (19); Cysta, *T. denticola* cystalyisin (L-cysteine desulfhydrase) (18).

expected, only the combination of all three enzymes led to the production of H₂S from glutathione (Fig. 6 and data not shown). Importantly, the other four products, ammonia, pyruvate, glutamic acid, and glycine, of the proposed enzyme pathway were also produced in equimolar amounts in the reaction with all three enzymes (data not shown). These results show that the proposed pathway for *T. denticola* catabolism of glutathione into H₂S is plausible and that the enzymes involved have been identified. However, genetic experiments in which the genes encoding these enzymes are mutated will be needed to prove that this pathway is used by *T. denticola* in vivo.

**DISCUSSION**

The CGase from *T. denticola* is a 52-kDa protein that is a member of the M17 family of leucyl aminopeptidases. Kinetic analysis of the purified protein showed that its preferred substrate is Cys-Gly, although it can also hydrolyze a leucine dipeptide. This result is not unprecedented; *E. coli* aminopeptidases

**Glutathione Catabolic Pathway Can Be Recreated in Vitro—**

We have proposed that *T. denticola* uses a three enzyme pathway for the catalysis of glutathione into H₂S (12). To prove that GGT, cystalyisin, and CGase can function in a catabolic pathway as proposed, the three purified enzymes (Refs. 18 and 19 and this report) were used to reproduce the pathway in vitro. As

from *T. vincentii* or from *E. coli* strains TB-1 and BL21 (DE3). These results suggest that all strains of *T. denticola* are likely to contain a *cga* gene closely related to the one in strain 35404 and that these genes express a protein with CGase activity.

A and B, which are M17 leucine aminopeptidases, have cysteinyglycinase activity (43). Purified aminopeptidase B was shown to prefer Cys-Gly over Leu-Gly as a substrate (44) as we have found for the *T. denticola* CGase. Jösch et al. (45) identified a cytosolic leucine aminopeptidase as the major CGase in rat liver. However, their study did not directly compare the kinetics of their enzyme with Cys-Gly versus leucine dipeptide, so it is not clear which is the preferred substrate. On the other hand, Cappiello et al. (46, 47) identified the Cys-Gly hydrolyzing activity in bovine lens and showed that the protein previously characterized as an M17 leucine aminopeptidase was the enzyme involved. They compared the kinetics of their protein with Cys-Gly versus Leu-Gly and found that Cys-Gly was the preferred substrate, at least with Mn²⁺ as the cation. This is the same result we found for the *T. denticola* CGase. All four of these enzymes, from *T. denticola*, *E. coli*, rat liver, and bovine eye, have identical amino acids at the seven residues involved in metal binding and enzyme catalysis in the M17 family of aminopeptidases (41, 42). This suggests that the hundreds of other members of this family of proteins are likely to have CGase activity if they are assayed under the right conditions.

Two other periodontal pathogens, *Fusobacterium nucleatum* and *Porphyromonas gingivalis*, have been shown to be able to catabolize Cys-Gly into H₂S (48, 49), implying that they have a CGase enzyme. However, the proteins responsible for this activity in these two bacteria have not been identified. Based upon a homology search of the Oral Pathogens Data Base, *F. nucleatum* has an M17 family member, but *P. gingivalis* does not. Thus, we hypothesize that the leucine aminopeptidase is the likely CGase from *F. nucleatum* but that *P. gingivalis* will use a different peptidase to cleave Cys-Gly.

When the *T. denticola* CGase protein sequence was used in a blastp search for homologues in the total protein data base at NCBI, *Leptospira interrogans* was the only spirochete that had a homologue. This result is not particularly surprising because the analysis of the genome sequence of *T. denticola* (31) indicates that 46% of the *T. denticola* genes have no homologue in any of the three other spirochetes whose genomes have been sequenced (*Treponema pallidum*, *Borrelia burgdorferi*, and *L. interrogans*), and 32% of the *T. denticola* genes have a homologue in only one or two of the other spirochetes. CGase is one of 288 (10% of total) *T. denticola* genes that only have *Leptospira* homologues (31). Interestingly, there were hundreds of other bacteria with proteins more similar to the *T. denticola* CGase than was the *L. interrogans* leucine aminopeptidase. Atop the list were several species of *Clostridium* whose proteins shared 48% identity with the *T. denticola* CGase as opposed to the 31% amino acid sequence identity between the spirochetal leucine aminopeptidases. Once again, this was not particularly surprising if the results from whole genome sequence analysis are considered; a fair number of other *T. denticola* proteins are more closely related to proteins from *Clostridium* spp. than they are to proteins from other spirochetes (31). This suggests that the acquisition of the glutathione catabolic pathway by *T. denticola* was a more recent evolutionary event than was the divergence of the spirochetes into different species.

By immunodepletion, *T. denticola* would appear to have only one protein with CGase activity. Although this is similar to the results found in rat liver, where the leucine aminopeptidase appears to be the major, if not only, cysteinyglycinase (45), the results are quite different from those found in *E. coli*, the only other bacterium where CGase activity has been well characterized. In *E. coli*, there are four aminopeptidases that have CGase activity (43). Two of them, aminopeptidases A and B, are M17 leucine aminopeptidases, but the other two *E. coli* proteins are from different, unrelated metalloprotease families. Given the *E. coli* result, it is somewhat surprising that *T. denticola* appears to have only one CGase because the *T. denticola* genome encodes at least 55 peptidases (Los Alamos Oral Pathogens Data Base). Perhaps there is another aminopeptidase with Cys-Gly activity whose synthesis is induced only under certain growth conditions. This possibility is being explored.

The optimal pH for several other M17 family members has been determined to be somewhat basic (pH 7.9–8.5) with Leu-X as substrate (35, 46, 50). However, in the one case where it was examined, bovine lens aminopeptidase (46), the optimal pH for Cys-Gly hydrolysis with the same enzyme was more acidic (pH 6.8). This is very similar to what was found with the *T. denticola* CGase, which had an optimal pH of 8.0 with Leu-Xaa as substrate but a pH optimum of 7.3 when cleaving Cys-Gly. Interestingly, the metal dependence of the *T. denticola* and bovine proteins were also basically the same. The CGase activity from bovine lens required Mn²⁺ as a cation; Zn²⁺ did not support CGase activity. Similarly, the activity of the *T. denticola* CGase is strongly activated by Mn²⁺ (and Co²⁺ and Mg²⁺) but barely activated at all by Zn²⁺ (and Ca²⁺). As has been suggested and discussed by Cappiello et al. (47), these pH profile and metal dependence results indicate that although the catalytic mechanism and metal-binding residues are the same for the two substrates, the substrate recognition “pockets” for the two substrates differ. As more M17 leucine aminopeptidases are tested for their ability to hydrolyze Cys-Gly, it will be interesting to see whether there are conserved amino acid residues that define the ability, or lack thereof, to hydrolyze Cys-Gly.

The identification and characterization of CGase from *T. denticola*, and the use of the previously purified GGT and cystatysin enzymes, has confirmed that this organism contains the proposed three-step pathway that can produce H₂S from glutathione (12). However, in addition to being able to be metabolized into H₂S, glutathione and its catabolic intermediate Cys-Gly have been shown to have important functions in other bacteria (51). Both glutathione and Cys-Gly can play critical roles in maintaining or altering the redox status of a cell and can help protect components of the cell from oxidative damage. These two thiol-containing molecules can also modify some proteins at cysteine and influence their functions. Because GGT can obviously alter the levels of glutathione in the cell and CGase can do the same for Cys-Gly, these two enzymes are very likely to have critical functions in *T. denticola*, besides being part of the pathway to make H₂S. Thus, genetic experiments are in progress to assess the *in vivo* roles of these enzymes to see whether they have additional roles in *T. denticola* physiology and pathogenesis.

Characterization of Cysteinylglycinase from T. denticola

Acknowledgments—We are grateful to Dr. Sue Weintraub and Kevin Hakala of the University of Texas Health Science Center at San Antonio Institutional Mass Spectrometry Laboratory for scientific input in the protein identification by HPLC ESI-MS/MS analysis. We also thank Dr. Bjorn Steffensen for kindly providing HPLC supplies and Dr. An-Ping Lin for discussions of the analysis of the enzyme kinetics.

---

**REFERENCES**

1. Persson, S., Claesson, R., and Carlsson, J. (1989) Oral Microbiol. Immunol. 4, 169–172
2. Claesson, R., Edlund, M. B., Persson, S., and Carlsson, J. (1990) Oral Microbiol. Immunol. 5, 137–142
3. Persson, S., Edlund, M. B., and Claesson, R. (1990) Oral Microbiol. Immunol. 5, 195–201
4. United States National Research Council (1979) Hydrogen Sulfide in Medical and Biologic Effects of Environmental Pollutants, pp. 1–50, University Park Press, Baltimore, MD
5. Beauchamp, R. O., Bus, J. S., Jr., Popp, J. A., Boreiko, C. J., and Andjelkovich, D. A. (1984) Crit. Rev. Toxicol. 13, 25–97
6. Reiffenstein, R. J., Hulbert, W. C., and Roth, S. H. (1992) Annu. Rev. Pharmacol. Toxicol. 32, 109–134
7. Morhart, R. E., Mata, L. J., Sinsky, A. J., and Harris, R. S. (1970) J. Periodontol. 41, 644–649
8. Horowitz, A., and Folke, L. E. (1973) J. Periodontal. 44, 390–395
9. Persson, S. (1992) Oral Microbiol. Immunol. 7, 378–379
10. Rizzo, A. A. (1967) Periodontics 5, 233–236
11. Carlsson, J., Larsen J. T., and Edlund, M. B. (1993) Oral Microbiol. Immunol. 8, 42–45
12. Chu, L., Dong, Z., Xu, X., Cappelli, D., and Ebersole, J. (2002) Infect. Immun. 70, 1113–11207
13. Simonson, L. G., Goodman, C. H., Bial, J. J., and Morton, H. E. (1988) Infect. Immun. 56, 726–728
14. Moore, W. E. C. (1987) J. Periodont. Res. 22, 335–341
15. Aimetti, M., Romano, F., and Nessi, F. (2007) J. Periodontol. 78, 1718–1723
16. Sela, M. N. (2001) Crit. Rev. Oral Biol. Med. 12, 399–413
17. Holt, S. C., and Ebersole, J. L. (2005) Periodontol. 38, 72–122
18. Chu, L., Ebersole, J. L., Kurzben, G. P., and Holt, S. C. (1997) Infect. Immun. 65, 3231–3238
19. Chu, L., Xu, X., Dong, Z., Cappelli, D., and Ebersole, J. D. (2003) Infect. Immun. 71, 335–342
20. Chu, L., and Holt, S. C. (1994) Microb. Pathog. 16, 197–212
21. Chu, L., Burgum, A., Kolodrubetz, D., and Holt, S. C. (1995) Infect. Immun. 63, 4448–4455
22. Makinen, P. L., and Makinen, K. K. (1997) Infect. Immun. 65, 685–691
23. Chu, L., Ebersole, J. L., Kurzben, G. P., and Holt, S. C. (1999) Clin. Infect. Dis. 28, 442–450
24. Chu, L., Ebersole, J. L., and Holt, S. C. (1999) Oral Microbiol. Immunol. 14, 293–303
25. Kurzben, G. P., Chu, L., Ebersole, J. L., and Holt, S. C. (1999) Oral Microbiol. Immunol. 14, 153–164
26. Krupka, H. I., Huber, R., Holt, S. C., and Clausen, T. (2000) EMBO J. 19, 3168–3178
27. Bertoldi, M., Cellini, B., Clausen, T., and Voltattorni, C. B. (2002) Biochemistry 41, 9153–9164
28. Cellini, B., Bertoldi, M., Paiardini, A., D'Aguanno, S., and Voltattorni, C. B. (2004) J. Biol. Chem. 279, 36898–36905
29. Cellini, B., Bertoldi, M., Montioli, R., Laurents, D. V., Paiardini, A., and Voltattorni, C. B. (2006) Biochemistry 45, 14140–14154
30. Cellini, B., Montioli, R., Bossi, A., Bertoldi, M., Laurents, D. V., and Voltattorni, C. B. (2006) Arch. Biochem. Biophys. 455, 31–39
31. Seshadri, R., Myers, G. S., Tettelin, H., Eisen, J. A., Heidelberg, J. F., Dodson, R. J., Davidsen, T. M., DeBoy, R. T., Fouts, D. E., Haft, D. H., Selengut, J., Ren, Q., Brinkac, L. M., Madupu, R., Kolonay, J., Durkin, S. A., Daugherty, S. C., Shetty, J., Shvartsbeyn, A., Gebregeorgis, E., Geer, K., Tsegaye, G., Malek, J., Ayodeji, B., Shatsman, S., McLeod, M. P., Smajs, D., Howell, J. K., Pal, S., Amin, A., Vashisth, P., McNeill, T. Z., Xiang, Q., Sodergren, E., Baca, E., Weinstock, G. M., Norris, S. J., Fraser, C. M., and Paulsen, I. T. (2004) Proc. Natl. Acad. Sci. U. S. A. 101, 5646–5651
32. Weinberg, A., and Holt, S. C. (1990) Infect. Immun. 58, 1720–1729
33. Blakemore, R. P., and Canale-Parola, E. (1976) J. Bacteriol. 128, 616–622
34. Gaitonde, M. K. (1967) Biochem. J. 104, 627–633
35. Morty, R. E., and Morehead, J. (2002) J. Biol. Chem. 277, 26057–26065
36. Thomas, J. A., Hardies, S. C., Rolando, M., Hayes, S., Lieman, K., Carroll, C. A., Weintraub, S. T., and Serwer, P. (2007) Virology 368, 405–421
37. Siegel, L. M. (1965) Anal. Biochem. 11, 126–132
38. Zheng, L., White, R. H., Cash, V. L., Jack, R. F., and Dean, D. R. (1993) Proc. Natl. Acad. Sci. U. S. A. 90, 2754–2758
39. Bauer, J. D., Ackermann, P. G., and Toro, P. (1974) Clinical Laboratory Methods, pp. 399–401, The C. V. Mosby Company, St. Louis, MO
40. Chu, L., Kong, X., Shao, J., Lee, M., and Xie, J. (1986) Shanghai J. Immunol. 6, 243–248
41. Colloms, S. D. (2004) in Handbook of Proteolytic Enzymes (Barrett, A. J., Rawlings, N. D., and Woesner, J. F., eds) 2nd Ed., pp. 905–908, Elsevier/Academic Press, San Diego, CA
42. Strater, N., and Lipscomb, W. N. (2004) in Handbook of Proteolytic Enzymes, (Barrett, A. J., Rawlings, N. D., and Woesner, J. F., eds) 2nd Ed., pp. 896–899, Elsevier/Academic Press, San Diego, CA
43. Suzuki, H., Kamatani, S., Kim, E. S., and Kumagai, H. (2001) J. Bacteriol. 183, 1489–1490
44. Suzuki, H., Kamatani, S., and Kumagai, H. (2001) Biosci. Biotechnol. Biochem. 65, 1549–1558
45. Jösch, C., Klotz, L. O., and Sies, H. (2003) Biol. Chem. 384, 213–218
46. Cappiello, M., Lazzarotti, A., Buono, F., Scaloni, A., D'Ambrosio, C., Amodeo, P., Méndez, B. L., Pelosi, P., Del Corso, A., and Mura, U. (2004) Biochem. J. 378, 35–44
47. Cappiello, M., Cappiello, M., Alterio, V., Amodeo, P., Del Corso, A., Scali, A., Pedone, C., Moschini, R., De Donatis, G. M., De Simone, G., and Mura, U. (2006) Biochemistry 45, 3226–3234
48. Carlsson, J., Larsen, J. T., and Edlund, M. B. (1994) Oral Microbiol. Immunol. 9, 297–300
49. Tang-Larsen, J., Claesson, R., Edlund, M. B., and Carlsson, J. (1995) J. Periodont. Res. 30, 390–395
50. Dong, L., Cheng, N., Wang, M. W., Zhang, J., Shu, C., and Zhu, D. X. (2005) Oral Microbiology 151, 2017–2023
51. Smirnova, G. V., and Oktyabrsky, O. N. (2005) Biochemistry (Moscow) 70, 1459–1473

Enzyme Catalysis and Regulation:
A 52-kDa Leucyl Aminopeptidase from
Treponema denticola Is a Cysteinylglycinase
That Mediates the Second Step of
Glutathione Metabolism

Lianrui Chu, Yanlai Lai, Xiaoping Xu, Scott
Eddy, Shuang Yang, Li Song and David
Kolodrubetz
J. Biol. Chem. 2008, 283:19351-19358.
doi: 10.1074/jbc.M801034200 originally published online May 15, 2008

Access the most updated version of this article at doi: [10.1074/jbc.M801034200](http://dx.doi.org/10.1074/jbc.M801034200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- [When this article is cited](#)
- [When a correction for this article is posted](#)

[Click here](#) to choose from all of JBC's e-mail alerts

This article cites 47 references, 16 of which can be accessed free at
[http://www.jbc.org/content/283/28/19351.full.html#ref-list-1](http://www.jbc.org/content/283/28/19351.full.html#ref-list-1)
