
Microbiology (2008), 154, 1570–1583

DOI 10.1099/mic.0.2008/016840-0

**Review**

The bacterial type VI secretion machine: yet another player for protein transport across membranes

Alain Filloux,$^{1,2}$ Abderrahman Hachani$^{1,2}$ and Sophie Bleves$^{2}$

$^{1}$Imperial College London, Division of Cell and Molecular Biology, Centre for Molecular Microbiology and Infection, South Kensington Campus, Flowers Building, London SW7 2AZ, UK

$^{2}$Laboratoire d'Ingénierie des Systèmes Macromoléculaires, UPR9027, CNRS-IBSM, 31 Chemin Joseph Aiguier, 13402 Marseille cedex 20, France

Several secretion systems have evolved that are widespread among Gram-negative bacteria. Recently, a new secretion system was recognized, which is named the type VI secretion system (T6SS). The T6SS components are encoded within clusters of genes initially identified as IAHP for IcmF-associated homologous proteins, since they were all found to contain a gene encoding an IcmF-like component. IcmF was previously reported as a component of the type IV secretion system (T4SS). However, with the exception of DotU, other T4SS components are not encoded within T6SS loci. Thus, the T6SS is probably a novel kind of complex multi-component secretion machine, which is often involved in interaction with eukaryotic hosts, be it a pathogenic or a symbiotic relationship. The expression of T6SS genes has been reported to be mostly induced *in vivo*. Interestingly, expression and assembly of T6SSs are tightly controlled at both the transcriptional and the post-translational level. This may allow a timely control of T6SS assembly and function. Two types of proteins, generically named Hcp and VgrG, are secreted via these systems, but it is not entirely clear whether they are truly secreted effector proteins or are actually components of the T6SS. The precise role and mode of action of the T6SS is still unknown. This review describes current knowledge about the T6SS and summarizes its hallmarks and its differences from other secretion systems.

---

**Introduction**

Interaction between bacteria and hosts ranges from a commensal collaboration to a competition that may result in host death (Merrell & Falkow, 2004). Such interaction is guided by a communication/signalling game between the host and the pathogen, which, as in a game of chess, aims to influence the way you would like your opponent to play. Among the tools used by bacteria to influence the host response, secretion machines that deliver proteins and toxins into the environment and within a eukaryotic target cell are crucial for virulence and survival within hosts (Caron *et al.*, 2006; Cossart & Sansonetti, 2004; Mota *et al.*, 2005). These proteins are transported across the membranes of the bacteria, and eventually of the host, by means of specific devices called secretion systems (SSs). During the last 20 years, it has been found that Gram-negative bacteria have evolved several SSs, which have been identified by types (Fig. 1) (Filloux, 2004; Saier, 2006). Five types have been defined, i.e. type I to type V (T1SS to T5SS). The SSs vary in complexity but all use a single polypeptide or a supra-macromolecular complex to build a path through the bacterial cell envelope. SSs can be recognized by a set of core components used to build up the secretion device. Recent studies have led to the identification of a new type of SS, named T6SS. The T6SS components are encoded within gene clusters that vary in organization. These clusters were initially named IAHP, for IcmF-associated homologous proteins, because they contain a gene encoding an IcmF-like component (Das & Chaudhuri, 2003). It thus all started with the finding that a known T4SS component, IcmF, was encoded within a conserved gene cluster among Gram-negative bacteria but whose other genes had no homology with T4SS components. These novel genes were likely to encode the components of a novel secretion machine or T6SS.

**The type IVB secretion system**

*Legionella pneumophila* is a facultative intracellular pathogen, and when it grows inside human cells or amoebae it is able to inhibit phagosome–lysosome fusion. *L. pneumophila* pathogenesis requires 26 dot (*defect in organelle trafficking*) and *icm* (*intracellular multiplication*) genes (Segal *et al.*, 2005). These genes are essential for altering the endocytic pathway and for replication of *L. pneumophila* inside the host cells (Sexton & Vogel, 2002). The *L. pneumophila* dot/icm genes encode components of the type IVB SS, and share extensive similarities with the *trb/tra* genes found on the IncI plasmids and involved in bacterial

The bacterial type VI secretion machine

Fig. 1. Type I–V secretion systems in Gram-negative bacteria. Type I, type III and type IV SSs (left) are believed to transport proteins in one step from the bacterial cytosol to the bacterial cell surface and external medium. In the case of type III and type IV SSs, the proteins are transported from the bacterial cytoplasm to the target cell cytosol. One exception for type IV is the pertussis toxin, which is secreted in two steps and released into the extracellular medium. This exception is represented by the dotted arrow, which connects Sec and the type IV SS. Type II and type V SSs transport proteins in two steps. In that case, proteins are first transported to the periplasm via the Sec or Tat system before reaching the cell surface. Type Va is a putative autotransporter, indicating that the C-terminus of the protein forms the outer-membrane channel (cylinder) whereas the N-terminus (pink line) is exposed to the surface or released by proteolytic cleavage (scissors). C, bacterial cytoplasm; IM, bacterial inner membrane; P, bacterial periplasm; OM, bacterial outer membrane; ECM, extracellular milieu; PM (brown zone), host cell plasma membrane. When appropriate, coupling of ATP hydrolysis to transport is highlighted. Arrows indicate the route followed by transported proteins.

conjugation. The type IVA archetypes are the T-DNA transferring system called VirB in *Agrobacterium tumefaciens* and the Tra/Trb conjugative system from IncP plasmids (Christie et al., 2005). The relationship between the B and the A subgroups of the T4SS is limited to a few components.

Whereas the Tra/Trb systems of the IncI plasmids are conjugation machines that deliver nucleoprotein complexes (Wilkins & Thomas, 2000), the *L. pneumophila* type IVB SS (Dot/Icm) is known to deliver proteins into target cells. In *L. pneumophila* Dot/Icm-dependent effectors have been characterized. Among them, RalF was shown to be required for the localization of ARF (ADP-ribosylation factor) on phagosomes containing *L. pneumophila* (Nagai et al., 2002).

### The DotU/IcmF paradigm

In contrast to most *dot/icm* genes, *icmF* is partially required for *L. pneumophila* replication in macrophages (Purcell & Shuman, 1998). The *icmF* gene is located at one end of the *dot/icm* cluster, downstream of a gene designated *dotU* or *icmH*. DotU and IcmF localize to the *L. pneumophila* inner membrane (Sexton et al., 2004). IcmF contains several transmembrane domains and a putative Walker A nucleotide-binding motif whereas DotU contains a transmembrane segment in its C-terminal region. *dotU* and *icmF* mutants have similar intracellular growth phenotypes, i.e. partially defective in replication within macrophages (Sexton et al., 2004; Zusman et al., 2004), which suggested that the DotU and IcmF proteins worked together. Furthermore, the lack of IcmF resulted in a reduced level

http://mic.sgmjournals.org
of DotU protein, suggesting that the two proteins interact. It was also shown that the lack of DotU and/or IcmF affected the stability of three other Dot proteins. This suggests that DotU/IcmF assist in assembly and stability of a functional Dot–Icm complex and that in their absence the complex is not maintained in a fully active form (Sexton *et al.*, 2004; VanRheenen *et al.*, 2004).

IcmF and DotU orthologues have been found in a wide range of Gram-negative bacterial species. In many cases, these genes are linked, but no other T4SS genes have been found in their vicinity. Instead, another set of conserved genes are systematically found, which were originally known as IAHP (Das & Chaudhuri, 2003). Each gene cluster encodes at least a dozen proteins with various degrees of conservation.

### The Rhizobium leguminosarum Imp system

The first report describing genes belonging to IAHP clusters was published in 1997 (Roest *et al.*, 1997). *Rhizobium leguminosarum* is a plant symbiont, which induces formation of nitrogen-fixing nodules. The host specificity is determined by the Sym plasmid (Van Brussel *et al.*, 1986). The authors described a *R.* *leguminosarum* strain capable of nodulation on *Vicia sativa* but not on *Pisum sativum* and *Vicia hirsuta*. In these last species small nodules were formed on the plant roots, but they were not able to fix nitrogen. This strain was designated RBL5523. Tn5 mutagenesis on RBL5523 allowed the identification of a mutant, RBL5787, restored in its ability to nodulate *P.* *sativum* and *V.* *hirsuta*. Because the locus is important for nodulation or subsequent stages of symbiosis, it was named *imp* (*impaired in nodulation*). In 2003, another study reported the sequence of a 33 kb region around the Tn5 inserted in RBL5787 (Bladergroen *et al.*, 2003), revealing a putative operon of 14 genes (Fig. 2) named *impA–impN*. These genes mostly encode proteins with unknown function, but ImpK and ImpL have similarities with DotU and IcmF, respectively. ImpK is similar to DotU but has a C-terminal extension with similarity to OmpA, an *Escherichia coli* outer-membrane protein, and to the flagellar torque-generating protein MotB (De Mot & Vanderleyden, 1994). This conserved C-terminal domain is described as a peptidoglycan-anchoring domain. DotU and IcmF are known T4SS-like components, which encouraged the authors to test whether *imp* mutants were defective in protein secretion. They showed that four proteins were lacking in the culture supernatant of the *imp* mutant (RLB5787), as compared to RLB5523, including a homologue of the signal-peptide-containing protein RbsB (ribose-binding protein). Furthermore, they showed that when spent growth medium of RBL5523 was used to inoculate the *imp* mutant RBL5787, a reduction in nodules and nitrogen fixation could be observed. In conclusion, they suggested that the Imp system encoded components of a secretion machine and that Imp-dependently secreted proteins could block the colonization/infection process in

pea plants. Obviously, a new SS was born, even though not yet named T6SS.

### The T6SS in Vibrio cholerae

The work defining the *imp* locus in *R.* *leguminosarum* has been concomitant with work on *Vibrio cholerae*, which started with the characterization of an IcmF homologue. The *V.* *cholerae icmF* gene was identified as being induced *in vivo* in a rabbit model of infection (Das *et al.*, 2000, 2002). However, the most significant work that revealed the role of the *V.* *cholerae* IAHP cluster, and which identified it as the new T6SS, was published in 2006 (Pukatzki *et al.*, 2006). The authors used the O37 serogroup *V.* *cholerae* strain V52, which, in contrast to the O1 strain, is capable of evading amoeboid killing when plated with *Dictyostelium discoideum*. Transposon mutagenesis on strain V52 identified a series of mutants that were attenuated for their virulence on *Dictyostelium*. Transposon mapping located most of the insertions in a cluster of genes called *vas* for ‘virulence associated secretion’. Interestingly, an insertion was found in *vasK* (VAC0120), which corresponds to the previously identified *icmF*-like gene (Das *et al.*, 2002). The other *vas* genes, namely VCA0107 to VCA0120, had homologies with most of the *imp* genes previously described (Fig. 2) (Bladergroen *et al.*, 2003). This indicated that the *imp* genes from *R.* *leguminosarum* and the *vas* genes from *V.* *cholerae* encoded a related system, whose role is important for pea infection and *Dictyostelium* killing, respectively. Like the *imp* gene cluster, the *vas* cluster encodes IcmF and DotU homologues. It should be noted that the DotU homologue of *V.* *cholerae* (VCA0115/VasF) does not contain the OmpA/MotB C-terminal extension found in ImpK, which indicated that this is not a hallmark for DotU orthologues encoded within IAHP/T6SS gene clusters. All the hallmark features found within characterized T6SSs are described in Table 1. Pukatzki *et al* (2006) investigated whether the Vas system may be involved in protein secretion. Analysis of *vas* mutants’ culture supernatant, and comparison with the parental strain, revealed the absence of a 28 kDa protein, which appeared to be Hcp, a ‘Haemolysin A co-regulated protein’ (Williams *et al.*, 1996), and of proteins called VgrG. Analysis of an O1 serogroup strain sensitive to *Dictyostelium*, N16961, showed that it failed to secrete Hcp and VgrG proteins, similarly to the V52 *vas* mutants. *Dictyostelium* resistance thus appeared to be dependent on the secretion of Hcp and VgrG proteins. It should be noted that in the study by Pukatzki *et al.* (2006), secretion of VgrG1 and VgrG2 was shown to be Hcp-dependent, even though it was unclear how Hcp could serve this role. The VgrG1, VgrG2, VgrG3 and Hcp proteins do not contain an N-terminal signal peptide. Interestingly, secreted *V.* *cholerae* proteins such as chitinase, neuraminidase, PrtV and HlyA contain a signal peptide and are still secreted in the *vas* mutants. These observations suggested that the Vas/IAHP/T6SS is responsible for the secretion of proteins
The bacterial type VI secretion machine

---

**R. leguminosarum**

![Diagram of R. leguminosarum T6SS gene cluster](https://i.imgur.com/placeholder.png)

**V. cholerae**

![Diagram of V. cholerae T6SS gene clusters](https://i.imgur.com/placeholder.png)

**P. aeruginosa**

HSI-I

![Diagram of P. aeruginosa HSI-I T6SS gene cluster](https://i.imgur.com/placeholder.png)

HSI-II

![Diagram of P. aeruginosa HSI-II T6SS gene cluster](https://i.imgur.com/placeholder.png)

HSI-III

![Diagram of P. aeruginosa HSI-III T6SS gene cluster](https://i.imgur.com/placeholder.png)

---

Fig. 2. T6SS gene clusters. *R. leguminosarum* T6SS genes are labelled *impA* to *impN*. *V. cholerae* genes are indicated by the number of the annotated gene (e.g. VC0107) and when applicable with the given gene name, i.e. *vas*, *hcp* or *vgrG*. The three *P. aeruginosa* T6SS clusters are presented (HSI-I, HSI-II and HSI-III). The genes are indicated by the number of their annotation (e.g. PA0074) and when applicable with the given gene name. In addition to the gene nomenclature presented by Mougous et al. (2006), genes with unknown function or homologues have been named *hsi*. The gene letter given corresponds to the *R. leguminosarum* homologue. Thus *hsiA* is an *impA* homologue. We have indicated the gene encoding a putative lipoprotein as *lip*, the gene encoding the sigma factor activator as *sfa* and the gene encoding the putative Ser/Thr phosphatase from HSI-I as *pppB*. In all cases homologous genes are represented with the same colour or motif. In the case of *P. aeruginosa* HSI-II and *V. cholerae* distal *hcp* and *vgrG* genes have also been indicated. The *P. aeruginosa* HSI-II *orfX* gene is bordered with dashed lines indicating that it might be a misannotated gene. The HSI-I cluster is represented from PA0074 to PA0091. It should be noted that PA0071–PA0073 are likely to be part of HSI-I since they are upregulated in a *P. aeruginosa retS* mutant (Mougous et al., 2006). They have not been indicated in this figure simply because no homologues have yet been reported in other T6SS clusters.

---

lacking signal peptides, like the T4SS and T3SS but in contrast to the T2SS (Filloux, 2004).

## The T6SS in Salmonella enterica

Early in the discovery of the T6SS, a complete gene cluster, named *sci* (*Salmonella enterica* centisome 7 genomic island), was described in *S. enterica* (Folkesson et al., 2002). The island is 47 kb long and harbours 37 genes. From *sciA* to *sciY*, many genes could be identified as core genes for the T6SS, including 9 of the 15 *vas* genes found in *V. cholerae*. The Sci island includes genes encoding an Hcp-like protein (*sciK*) and a VgrG-like protein (*vgrS*) (Table 1). In addition, genes involved in fimbrial assembly (*saf*) or invasin production (*pagN*) were found. A complete deletion of the Sci genomic island resulted in decreased ability of *S. enterica* to enter eukaryotic cells. This result is different from data obtained with a *sciS* (*icmF*-like) transposon mutant (Parsons & Heffron, 2005). In that case, it was observed that SciS limits intracellular growth in macrophages at late stages of infection and attenuates the lethality of *S. enterica* in a murine host. *sciS* was maximally expressed at a late stage of infection, and was shown to be negatively regulated by SsrB, part of the SsrA/SsrB two-

Table 1. Some features of T6SSs in various bacteria

| Bacterium                          | T6SS genes                  | Secreted proteins                     | Putative secreted proteins           | Regulatory aspects*                                                                 | Notable features†                                                                 |
|-------------------------------------|----------------------------|----------------------------------------|---------------------------------------|------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|
| Burkholderia mallei                 | tss                         | Hcp1, VgrG1                           | TssB, TssM                          | VirAG two-component system BMAA 1517 (AraC-type regulator)                            | Attenuated virulence in hamsters                                                   |
| Edwardsiella ictaluri               | eip                         | EvpC (Hcp)                            |                                     | EsrB (response regulator, SsrB-like)                                                 | Anti-Eip produced during catfish infection                                         |
| Edwardsiella tarda                  | evp                         | EvpP (Hcp-like)                       |                                     | Temperature-dependent                                                                  | Attenuated virulence in blue gourami fish                                          |
| Escherichia coli (EAEC)             | sci-I                       | Hcp-like                              |                                     |                                                                                        |                                                                                  |
|                                    | sci-II (aai)                | AaiC                                  | AaiG (VgrG-like)                    | AggR (AraC-type regulator)                                                             |                                                                                  |
| Francisella tularensis              | igl, pdp, pig               |                                      | PigG (VgrG-like)                    | MglA master regulator Induced in vivo (macrophages)                                   | Required for intramacrophage growth; no ClpV-homologue                             |
| Pseudomonas aeruginosa              | hsi-I                       | Hcp1                                  | VgrG1                               | Two-component system sensor RetS                                                      | Attenuated virulence in rat lung infection; anti-Hcp1 produced in CF patients       |
|                                    |                             |                                       |                                     | Two-component system sensor LadS                                                     |                                                                                  |
|                                    |                             |                                       |                                     | PpkA/PppAS/T kinase-phosphatase                                                       |                                                                                  |
|                                    | hsi-II                      | VgrG2, Hcp2                           |                                     | Putative σ⁵⁴ activator (PA1663)                                                        |                                                                                  |
|                                    |                             |                                       |                                     | Stk1/Stp1S/T kinase-phosphatase                                                       |                                                                                  |
|                                    | hsi-III                     | Hcp3                                  |                                     | Putative σ⁵⁴ activator (PA2359)                                                        | No S/T kinase-phosphatase or Fha                                                  |
| Pectobacterium atrosepticum         |                             | Hcp1, -2, -3, -4, VgrG                |                                     | Induced by host-plant extract                                                          |                                                                                  |
| Rhizobium leguminosarum             | imp                         | RbsB homologue                        |                                     | Temperature-dependent                                                                  | No ClpV homologue                                                                |
|                                    |                             |                                       |                                     | ImpM/ImpNS/T kinase-phosphatase                                                       | No lipoprotein-encoding gene; signal peptide in RbsB; C-terminal extension in DotU |
| Salmonella enterica subspecies I    | sci                         | SciK (Hcp), VgrS (VgrG)               |                                     | Induced in vivo (macrophage and rabbit ileal loop) Response regulator SsrB            | Increased bacterial number in macrophages Hypervirulent in mice                     |
| Vibrio cholerae                     | vas                         | Hcp1, -2, VgrG1, -2, -3               |                                     | Induced in vivo (rabbit model for cholera) RpoN (σ⁵⁴) Putative σ⁵⁴ activator (VasH)   | Attenuated virulence on Dictyostelium and macrophages VgrG1 covalently cross-links actin |

* Regulatory components that control T6SS expression and function have been indicated as well as conditions in which T6SS genes are specifically controlled. The role of the putative σ⁵⁴ activator in expression of T6SS genes has not been demonstrated. However, in V. cholerae, a putative σ⁵⁴ activator is required for the T6SS-dependent phenotype as well as RpoN (σ⁵⁴).

† The attenuation or hypervirulence is indicated when it has been observed for at least some of the described T6SS mutants.
component system. However, the two studies are not really comparable in terms of models (J774 macrophage vs Hep2 cells) and in the study by Folkesson *et al.* (2002) a strain with a complete deletion of the Sci genomic island was used. The Sci island also contains *saf* fimbrial genes which may be required for invasion.

# The T6SS in *Pseudomonas aeruginosa*

Three IAHP/T6SS clusters are found in *Pseudomonas aeruginosa* (Fig. 2, Table 1) (Das & Chaudhuri, 2003). It has been demonstrated that one of these clusters, named HSI-I (*Hcp1* secretion island I) (Mougous *et al.*, 2006) and spanning genes identified as PA0071 to PA0091, contributed to *P. aeruginosa* pathogenesis. Previous studies also found that *P. aeruginosa* PAO1 mutants affected in PA0073, PA0077, PA0082, PA088 and PA0090 showed an attenuated virulence phenotype in a model of chronic infection in the rat lung (Potvin *et al.*, 2003). This suggested that the HSI-I genes are induced *in vivo*, as is the case with the homologous *V. cholerae vas* genes. In laboratory growth conditions, the HSI-I genes could be induced by deleting the *retS* gene, encoding a two-component system sensor protein, which is part of a signalling pathway important for establishing chronic infections (Goodman *et al.*, 2004). In a *retS* mutant, an Hcp-like protein could be detected in the culture supernatant. The gene encoding the Hcp protein is PA0085 and is part of the HSI-I gene cluster (Table 1). Mougous and collaborators further assessed whether mutation in HSI-I genes could interfere with Hcp secretion. The deletion of either PA0090 or PA0077 (*icmF*-like gene) genes resulted in a dramatic decrease of Hcp secretion. In *V. cholerae*, in addition to Hcp protein, VgrG proteins are secreted in a T6SS-dependent manner (Pukatzki *et al.*, 2006). In the study by Mougous and collaborators, even though a VgrG protein is encoded within the HSI-I gene cluster (PA0091), no indication was obtained about its secretion (Table 1).

## Some T6SS components

The T6SS machine, like other SSs, is defined by a set of original components. The T6SS gene clusters usually encode between 12 and 25 proteins, among which only DotU and IcmF-like proteins are found in another SS (T4SS). The cluster does not necessarily contain the genes encoding the Hcp and VgrG proteins, which may be distant from the T6SS locus. Most of the genes encode proteins with unknown function but some have known characteristics.

ATPases are frequently used by SSs to energize the transport process. In the case of the T6SS, a gene encoding a ClpB homologue is most frequently found. The Clp/Hsp100 family belongs to the ring-forming AAA⁺ superfamily of ATPases (Neuwald *et al.*, 1999). The ClpB homologues found in T6SS clusters could be distinguished as a subfamily, which was named ClpV. In prokaryotes,

ClpB members are key players in protein quality control. They form hexameric rings and use ATP hydrolysis to insert substrates to be unfolded within their central channel, that is the proteolytic chamber, for further degradation (Weibezahn *et al.*, 2004). Unlike ClpB, it was shown that ClpV from enterohaemorrhagic *E. coli* (EHEC) and *Salmonella typhimurium* failed to solubilize aggregated proteins (Schlieker *et al.*, 2005). Moreover, substrates stimulating the activity of ClpB ATPases failed to do so with ClpV. Therefore, the function of ClpV might be different from that of other ClpB members, such as providing energy-dependent coupled transport of poly-peptides (Yeo & Waksman, 2004). Nevertheless, due to the similarity with the Clp/Hsp100 family, another possibility is that the ClpV component is required to unfold/fold components of the T6SS machine to be assembled or effectors to be secreted. A similar role was proposed for ATPases involved in the T3SS (Akeda & Galan, 2005). The energy specifically required for the movement of proteins through the T6SS could then be provided by other components and might alternatively involve the proton-motive force.

Apart from *clpV*, another gene is frequently found in T6SS clusters, which encodes a putative lipoprotein. This is for example the case for PA0080 (HSI-I) in *P. aeruginosa* or VCA0113 (*VasD*) in *V. cholerae* (Fig. 2). In most cases, the amino acid in position +2 after the putative cleavage site is a serine, indicating that it is a putative outer-membrane lipoprotein (Yamaguchi *et al.*, 1988). Whether this protein contributes to pore formation in the membrane is unknown and proof for such a hypothesis may await the demonstration that it could be inserted into the outer membrane independently of its lipid anchor.

## What is secreted, what is injected and what are the effectors needed for?

Most recent studies have suggested that proteins most likely to be secreted by the T6SS belong to the Hcp or VgrG families.

### The Hcp family

The secretion of Hcp proteins in a T6SS-dependent manner was most significantly demonstrated in *V. cholerae* (Pukatzki *et al.*, 2006). In this bacterium there are two *hcp* genes (*hcpA*, -B or *hcp1*, -2), encoding identical proteins (Table 1). The *hcp* gene products have a predicted size of about 28 kDa, but they show an aberrant mobility on gels since their apparent molecular mass is 18–19 kDa. The *hcp2* gene (VC0017) is in the vicinity of the *V. cholerae* T6SS cluster (*vas*; VC0107–VC0123) whereas *hcp1* is more distantly located (VC1415). The Hcp proteins do not contain a canonical signal peptide, indicating that they are not secreted in a Sec- or Tat-dependent manner, and probably cross the bacterial cell envelope in a single step (Pallen *et al.*, 2003). Interestingly, an *hcp1/hcp2* mutant is

http://mic.sgmjournals.org   1575
A. Filloux, A. Hachani and S. Bleves

avirulent, whereas individual *hcp1* or *hcp2* single mutants retain virulence. This indicated that T6SS-dependent secretion of at least one Hcp protein is required and sufficient for virulence.

The secretion of Hcp protein was confirmed in *P. aeruginosa* (Mougous *et al.*, 2006). Each of the *P. aeruginosa* T6SS clusters contains or is associated with *hcp* genes, *hcp1* (PA0085/HSI-I), *hcp2* (PA1512/HSI-II) or *hcp3* (PA2367/HSI-III) (Fig. 2). The function of the HSI-I/T6SS system was studied in a *retS* background, which allowed overproduction of HSI-I genes and abundant secretion of Hcp1 (Mougous *et al.*, 2006). The secretion of Hcp1 was abolished after introducing a mutation in the *icmF1* or *clpV1* genes, indicating that, as with *V. cholerae*, Hcp secretion in *P. aeruginosa* is T6SS-dependent. In conditions where Hcp secretion is defective, the protein was found accumulated in the periplasm (Mougous *et al.*, 2007). This is a puzzling observation, which needs to be confirmed, since Hcp1 does not contain a signal peptide to allow inner membrane translocation. Interestingly, Hcp1 was found in the sputum of cystic fibrosis (CF) patients colonized with *P. aeruginosa*, and CF sera contained Hcp1 antibodies, which suggested that Hcp1 secretion is relevant not only *in vitro* but also *in vivo*.

The X-ray crystal structure of *P. aeruginosa* Hcp1 was obtained at a resolution of 1.95 Å (0.195 nm), and showed that the protein could assemble into hexameric rings with an inner diameter of 40 Å (4 nm) (Fig. 3) (Mougous *et al.*, 2006). The Hcp function is still unknown, and despite the fact that Hcp is considered as a secreted protein, one may consider that it may also be used to assemble a conduit at the bacterial cell surface through which other effector molecules might be transported to the host cell.

(vas), whereas *vgrG3* is located within the *vas* gene cluster (Pukatzki *et al.*, 2006) (Fig. 2). Mutations in *vgrG1* and *vgrG2* resulted in lack of Hcp secretion (Pukatzki *et al.*, 2007), suggesting that these proteins may also serve a T6SS function despite the fact that they appeared to be secreted substrates. Several *vgrG* genes located outside T6SS clusters are found in association with *hcp* genes. The Vgr proteins were initially considered as accessory components of the Rhs (recombination hot spot) family (Wang *et al.*, 1998b). Rhs components are complex genetic composites, and in *E. coli*, eight components, RhsA–H, are known. Rhs components include a core ORF (G/C rich) and an adjacent core extension (A/T-rich) with linked downstream ORFs (A/T-rich). Two ORFs immediately upstream of the *rhsG* core were identified and named *vgrG* and *hcp* (Wang *et al.*, 1998b). Although they are not homologous, *vgrG* and the *RhsG* core ORFs are similar in a number of respects. They are both large, their predicted products are hydrophilic, and both are characterized by a regularly repeated peptide motif. These motifs are YDxxGRL(I/T) for the RhsG core protein and a Val-Gly dipeptide motif (Valine glycine repeats) in the case of VgrG. In the case of the RhsE element, a *vgr* gene, *vgrE*, was also found upstream of the *rhsE* core ORF (Wang *et al.*, 1998b). Interestingly, within the *S. enterica* T6SS gene cluster (*sci*), in addition to the *vgrS* gene, two genes were annotated *rhs1* and *rhs2*, *rhs2* encoding a protein with similarity to the *E. coli* RhsE protein (Folkesson *et al.*, 2002).

Characterization of VgrG proteins revealed further remarkable features. In *V. cholerae*, the VgrG1 C terminus shares a domain (ACD) with the RtxA toxin, which mediates actin cross-linking (Sheahan *et al.*, 2004). The functionality of the VgrG1 ACD domain was tested in COS-7 cells. As with RtxA ACD, ectopic expression of VgrG1 ACD produces cell rounding and actin cross-linking within the transfected cells. The activity of the VgrG1 ACD domain was confirmed *in vitro* using a purified VgrG1 protein, monomeric G-actin and cytoplasmic extracts of amoebae or macrophages (Pukatzki *et al.*, 2007). The VgrG proteins

The VgrG family

The T6SS-dependent secretion of VgrG proteins was demonstrated in *V. cholerae*. VgrG2 is encoded by a gene linked to *hcp2*, and in the vicinity of the T6SS gene clusters

---

Fig. 3. *P. aeruginosa* Hcp1 structure, reproduced from Mougous *et al.* (2006) (*Science* **312**, 1526–1530) with permission from AAAS. Hcp1 forms rings as seen from electron microscopy and single-particle analysis of purified material (inset scale bar, 10 nm). The rings are hexameric as shown by the top view of a ribbon representation. The individual subunits are coloured differently to highlight their organization.

containing a C-terminal extension were named ‘evolved VgrGs’. In *V. cholerae*, VgrG3 has a peptidoglycan-binding domain, whereas no extension was found in VgrG2.

Independently of C-terminal extensions, VgrGs share a conserved region, which contains two domains that showed similarities with the gp5 and gp27 proteins that constitute the bacteriophage T4 tail spike. Dimer of trimers of gp5 and gp27 constitutes the tail spike, which is used for puncturing the bacterial envelope and allows DNA injection into the bacterial cytoplasm (Kanamaru *et al.*, 2002; Rossmann *et al.*, 2004). Pull-down experiments using *V. cholerae* VgrG-directed antibodies revealed that these proteins form homotrimeric or heterotrimeric complexes (Pukatzki *et al.*, 2007). The assembly of the VgrG proteins into a putative tail-spike-like structure suggests that the resulting structure may be used as a puncturing device, to allow perforation of the bacterial cell envelope or of the host cell membrane, or both. In any case, VgrGs appear to be not simply secreted proteins but structural components of the T6SS machine. This is reminiscent of the other family of secreted proteins, Hcp, which form hexameric rings with a central channel of 40 Å (4 nm) (Mougous *et al.*, 2006). The tube formed by the bacteriophage T4 gp5/gp27 complex is about 30 Å (3 nm) in diameter (Kanamaru *et al.*, 2002). Pukatzki and colleagues speculated that the VgrG complex may adopt a similar structure and could be surrounded by the Hcp rings, both forming the tube for T6SS substrates. If that were the case, Hcps and VgrGs may be appended at the bacterial surface and possibly released into the culture supernatant by shearing.

The presence of catalytic domains in some VgrG proteins does not support the idea of VgrGs being solely T6SS components. Considering the example of the VgrG1 ACD domain, one has to think that it should be translocated into the host cytosol to be able to promote actin cross-linking. This transport is unlikely to be performed after release of VgrG1 into the extracellular medium, since *V. cholerae* culture supernatants containing VgrGs were not able to provoke macrophage rounding. This observation anyhow provides evidence that the T6SS is intimately linked with the process of bacteria–host interaction, since remodelling of the cytoskeleton suggests injection of effector proteins/domains into host cells.

Finally, other proteins have been proposed to be secreted in a T6SS-dependent manner, but whether they are injected into the host cell, attached to the bacterial cell surface or simply released into the milieu needs further investigation. One striking example is the *R. leguminosarum* RbsB-like protein (ribose-binding protein) reported as a T6SS substrate (Bladergroen *et al.*, 2003). In Gram-negative bacteria, RbsB homologues contain a signal peptide and are localized to the periplasm, where they are involved in binding substrates, such as ribose or the AI-2 signalling molecule (Shao *et al.*, 2007). Bladergroen *et al.* (2003) confirmed that the *R. leguminosarum* RbsB does contain a signal peptide, which is not in agreement with T6SS

The bacterial type VI secretion machine

transporting uniquely substrates lacking signal peptides. This might still be possible since both signal-peptide-containing (pertussis toxin) and signal-peptide-lacking substrates (*Helicobacter pylori* CagA) are transported in a T4SS-dependent manner (Backert & Meyer, 2006). Whether RbsB secretion in *R. leguminosarum* is significant remains unclear, but strikingly, a gene encoding an RbsB homologue is found downstream of the *vgrG3* gene in the *V. cholerae* T6SS cluster (Bladergroen *et al.*, 2003).

## The key role of threonine phosphorylation in type VI secretion

Another remarkable feature of T6SS gene clusters is that they generally encode Ser/Thr kinases and phosphatases (Fig. 2, Table 1). Mukhopadhyay *et al.* (1999) reported Hank’s-type Ser/Thr kinase and Ser/Thr phosphatase in *P. aeruginosa*. The corresponding genes, *stk1* and *stpl*, are located downstream of a gene encoding an IcmF-like protein, which is now recognized as part of the HSI-II/T6SS system of *P. aeruginosa*. In *Yersinia*, the Ser/Thr kinase YpkA is involved in T3SS virulence (Cornelis, 1998), and it is considered that some bacterial Ser/Thr kinases have host targets due to their similarity with mammalian proteins. Mukhopadhyay *et al.* (1999) suggested that Stk1 and Stp1 could be translocated in host cells by a mechanism involving IcmF and thus the T6SS. It is now clear that it is not the case.

In *P. aeruginosa*, in addition to *stk1* and *stpl*, genes encoding Ser/Thr kinases and phosphatases, *ppkA* and *pppA*, respectively, are found in the T6SS/HSI-I cluster. The *ppkA* gene was previously found specifically induced within the host and required for virulence in neutropenic mice (Wang *et al.*, 1998a; Motley & Lory, 1999). Since the T6SS/HSI-I gene cluster is upregulated in the *retS* background, Mougous and collaborators could monitor T6SS function when introducing mutations in either *ppkA* or *pppA* (Mougous *et al.*, 2007). They showed that PpkA and PppA have antagonistic activities since a *ppkA* mutation resulted in defective Hcp1 secretion, whereas mutation in *pppA* resulted in increased Hcp1 secretion. The antagonistic effect of PpkA and PppA on T6SS function was confirmed by investigating T6SS localization. Using a ClpV1-GFP chimera, it was shown that the *P. aeruginosa* ClpV1 protein is localized in foci in the bacterial cell (Mougous *et al.*, 2006). However, ClpV1 was found evenly distributed throughout the cell in *hcp1* or *icmFl* mutants, indicating that the T6SS machine assembles as a macromolecular complex. Interestingly, the localization of ClpV1 into foci was observed in a *pppA* mutant, but not in a *ppkA* mutant. This suggested that both T6SS assembly and Hcp1 secretion require phosphorylation by PpkA and are prevented by dephosphorylation through PppA. Mougous and colleagues recognized a gene from the T6SS/HSI-I cluster encoding a protein with an FHA (forkhead-associated) domain. Proteins with FHA domains have affinity for phosphothreonine and have been

A. Filloux, A. Hachani and S. Bleves

implicated in phosphorylation-dependent signalling (Pallen *et al.*, 2002). The gene was named *fha1* and a mutation in this gene resulted in defective Hcp1 secretion. The phosphorylation status of Fha1 was shown both *in vitro* and *in vivo* to be PpkA/PppA-dependent and occurred on Thr-362. Overall, Mougous and collaborators suggest that, in standard conditions, the level of Fha1 phosphorylation is kept low by PppA. However, upon sensing of unknown environmental cues by PpkA, the kinase may overrule PppA activity and Fha phosphorylation initiates a signal transduction cascade that results in T6SS assembly and function. In conclusion, PpkA/PppA/Fha play a crucial role in controlling activity of the *P. aeruginosa* T6SS/HSI-I at the post-translational level. Whether this is applicable to all known T6SSs is questionable. Some clusters do not contain Ser/Thr kinase-phosphatase and Fha homologues (Fig. 2, Table 1). That is true for the *P. aeruginosa* T6SS/HSI-III but one may suggest that it uses heterologous components from the HSI-I or HSI-II systems. In the case of *V. cholerae*, the T6SS gene cluster encodes an Fha homologue (VCA0112) but no Ser/Thr kinase-phosphatase (Table 1). This could suggest that Fha could be phosphorylated by a kinase encoded by a gene outside the T6SS gene cluster.

### Controlling the expression of T6SS genes

Independently of post-translational control of T6SS function, expression of T6SS genes is highly regulated. In *R. leguminosarum*, T6SS secretion is temperature-dependent, since putative T6SS effectors were optimally secreted at 24 °C and only weakly at 20 °C (Bladergroen *et al.*, 2003). Other observations have revealed that the T6SS could be induced under *in vivo* conditions. That was the case for *V. cholerae* (Das *et al.*, 2000), for which induction of the *icmF* gene (*vasK*) was shown by comparing the global transcription response of *V. cholerae* cells grown overnight in rabbit ileal loops (*in vivo*) and in rich media (*in vitro*).

In *S. enterica*, induction of T6SS genes, and more precisely *sciS*, encoding the IcmF-like protein, was shown to occur inside macrophages (Parsons & Heffron, 2005). Expression of a *sciS-lacZ* transcriptional fusion was not observed when *Salmonella* was grown *in vitro*. *Salmonella* invasion and multiplication in macrophages relies on two T3SSs. T3SS-1 is involved in earlier killing, whereas T3SS-2 mediates host cell death at later stage (18–24 h post-infection). However, bacteria can persist in macrophages beyond 24 h, and this seems to be dependent on silencing the T6SS. Indeed, a *sciS* mutant is able to survive and multiply in macrophages at a late stage post-infection without lysis of the host cell. The response regulator SsrB, part of the two-component system SsrA/SsrB, negatively controls expression of *sciS*. In contrast, it was shown that SsrA/SsrB positively controls genes encoded on *Salmonella* pathogenicity island 2 (SPI-2), including the T3SS-2 genes (Garmendia *et al.*, 2003). Finally, in contrast to T3SS, the *Salmonella* T6SS favours bacterial persistence, and limits intracellular replication and bacterial load in host tissues, which is consistent with the hypervirulent phenotype of a *sciC* mutant in mice (Parsons & Heffron, 2005).

In *P. aeruginosa*, a balance between expression of genes involved in bacterial persistence and chronic infection versus genes involved in cytotoxicity and acute infection is nicely documented. As previously mentioned, expression of the T6SS/HSI-I gene cluster from *P. aeruginosa* is induced *in vitro* in a *retS* mutant. RetS is a hybrid sensor, which was identified as positively controlling expression of genes required for cytotoxicity (T3SS and exoenzymes) and negatively controlling genes required for biofilm formation (exopolysaccharide biogenesis gene clusters *pel* and *psl*) (Goodman *et al.*, 2004). The negative control of the *P. aeruginosa* T6SS by RetS is reminiscent of the negative control of the *Salmonella* T6SS by SsrB. It is also important to recall that antibodies against T6SS components were found in sera of CF patients chronically infected with *P. aeruginosa* (Mougous *et al.*, 2006), which supports the idea that T6SS favours persistence. Interestingly, in *P. aeruginosa* another hybrid sensor, LadS, is antagonist to RetS and acts positively on T6SS gene expression (Ventre *et al.*, 2006). Since T6SS gene expression could not be seen *in vitro* except in a *retS* mutant, one may speculate that the T6SS genes may be controlled *in vivo* upon recognition of specific host signals by the LadS/RetS pathway.

In *P. aeruginosa*, LadS and RetS do not influence expression of T6SS genes from HSI-II and HSI-III. Within each cluster, one gene encodes a σ⁵⁴ activator: PA1663 for HSI-II and PA2359 for HSI-III (Fig. 2, Table 1). Activators of bacterial σ⁵⁴-RNA polymerase holoenzyme use ATP hydrolysis to activate transcription by promoting the transition from a closed RNA polymerase (RNAP) promoter complex to a transcriptionally competent open complex (Rappas *et al.*, 2005). In the *V. cholerae* T6SS gene cluster, a σ⁵⁴ activator is encoded by *vasH*. A mutant in *vasH* was attenuated in *Dictyostelium*, as was a mutant affected in *rpoN* (Pukatzki *et al.*, 2006), the gene encoding the σ⁵⁴ alternative subunit of RNAP, which suggests that VasH plays a role in the likely σ⁵⁴-dependent expression of *V. cholerae* T6SS genes.

Overall, regulatory processes appear to tightly control T6SS gene expression during the host-pathogen interaction and prevent premature or inappropriate expression of these genes, which may negatively affect *in vivo* survival.

### The T6SSs in other Gram-negative bacteria: hallmarks and differences?

Besides the early discovered, or archetypal, T6SSs of *R. leguminosarum*, *V. cholerae*, *S. enterica* and *P. aeruginosa*, our knowledge about T6SSs in other bacteria is fragmentary. The data collected do not really contribute to further understanding of the T6SSs but point to a certain degree of variability.

Burkholderia mallei, Burkholderia pseudomallei and Burkholderia thailandensis possess multiple T6SS clusters, i.e. four, six and five, respectively. In B. mallei, one T6SS gene cluster (tssA–tssN) is upregulated upon overproduction of the VirAG two-component system (Schell et al., 2007), which resulted in an increased T6SS-dependent secretion of the Hcp1 protein. A B. mallei hcp mutant is attenuated in the hamster infection model (Schell et al., 2007). As with P. aeruginosa Hcp1, antibodies against B. mallei Hcp1 were found in sera of infected animals. Although VgrG proteins are encoded within Burkholderia T6SS clusters, their secretion has not been demonstrated. Interestingly, a non-Hcp and non-VgrG putative T6SS substrate was proposed to be the B. mallei TssB protein. The tssB gene is part of the T6SS cluster, and whereas a tssB mutant was not impaired in Hcp1 secretion it was attenuated for virulence in hamsters, similarly to a hcp mutant (Table 1). This observation suggested that if TssB is not part of the T6SS machine, it is involved in a T6SS-dependent phenotype and might be a T6SS effector (Schell et al., 2007). Finally, tssM encodes a protein containing an ubiquitin-specific protease domain (Schell et al., 2007). Since ubiquitination of proteins does not occur in bacteria, one can speculate that TssM is injected into the host cell.

In the case of Edwardsiella tarda, the T6SS cluster was named evp and contains 16 genes (Zheng et al., 2005; Zheng & Leung, 2007). Mutations in evp genes resulted in attenuated virulence in blue gourami fish (Table 1). A previous report on Edwardsiella ictaluri (Moore et al., 2002) identified eip genes similar to the imp genes from Rhizobium, and eip gene products were recognized by the catfish immune system during infection, indicating that they are produced in vivo. In Ed. tarda, it was shown that 13 evp genes were required for secretion of EvpC, an Hcp homologue (Rao et al., 2004), and EvpP, a secreted protein with no homology with either VgrG or Hcp proteins (Zheng & Leung, 2007) (Table 1). Sequence variation between Hcps is high, and EvpP shares several features with this family. It has no canonical signal peptide and runs aberrantly on polyacrylamide gels. Such aberrant migration was observed with V. cholerae Hcps. The formation of a T6SS complex involving the IcmF-like component EvpO was tested using a two-hybrid approach. Relevant interactions with EvpA, EvpL (lipoprotein) and EvpN (DotU) were found. No interaction between EvpH (ClpV) and any other Evp components was found, contradictory to results in P. aeruginosa, which suggests an interaction between ClpV1, Hcp1 and IcmF1 (Mougous et al., 2006).

In the case of enteroaggregative E. coli (EAEC), expression of chromosomal genes that are part of a T6SS cluster (aaiA–aaiY; AggR-activated island) is under the control of the virulence-plasmid-encoded AggR (aggregative adherence) transcriptional regulator (Dudley et al., 2006), a member of the AraC family (Sheikh et al., 2002). This plasmid encodes additional virulence factors such as the Pet enterotoxins, the aggregative adherence fimbriae (aafDA) (Natario et al., 1994) and the dispersin (aap) (Sheikh et al., 2002). The aai cluster is located in an EAEC pathogenicity island (117 kb) and is not found on the E. coli K-12 genome. It was shown that the aai cluster is required for secretion of AaiC. AaiC has no similarity with Hcp proteins, but like EvpP from Ed. tarda, has a size of about 18–19 kDa and does not contain a signal peptide. The aai cluster contains a gene encoding a VgrG-like protein but its secretion was not observed (Table 1). A second AggR-independent T6SS was found on the EAEC genome. The cluster consists of 21 genes and was named EAECSci-I, whereas the aai cluster was named EAECSci-II. In the EAECSci-I/T6SS cluster an hcp-like gene encodes a product that is secreted in a T6SS-dependent manner (Table 1).

In Pectobacterium atrosepticum, which causes diseases in potatoes and other plants, the production of four Hcps was observed (Mattinen et al., 2007). Mutation in one hcp gene did not impair virulence but overproduction of Hcp increased virulence. In vivo induction of T6SS gene expression is likely to occur in P. atrosepticum, since secretion of VgrG and Hcp proteins was induced by addition of potato tuber and stem extracts to the bacterial cultures (Mattinen et al., 2007).

In Francisella tularensis, a gene cluster encompassing 16–19 genes and located on a pathogenicity island (FPI) is needed for intramacrophage growth and virulence in chicken embryos (de Bruin et al., 2007; Nano et al., 2004; Nano & Schmerk, 2007). The organization of these genes is conserved in Francisella species, but is distinct from the well-characterized T6SS organization in other species. Two genes, iglA and iglB, have similarities to impB and impC, respectively. Expression of the iglAB genes is induced during growth in macrophages and is under the positive control of MglA (de Bruin et al., 2007) (Table 1), previously described as a master regulator for Francisella virulence (Lauriano et al., 2004). MglA is similar to the E. coli stringent starvation protein A (SspA), an RNAP-associated protein (Hansen et al., 2005). The mglA gene is linked with mglB, which is also required for intracellular growth of Francisella, and which encodes a protein similar to SspB from E. coli (Baron & Nano, 1998). The pdpB (pathogenicity determinant protein) gene encodes an IcmF-like protein, whereas the pigF (pathogenicity island gene) gene has similarities to dotU. The pigB gene encodes a protein with similarity to the VgrG-family of proteins. The iglD gene encodes a protein with some similarities to ImpJ, particularly in the N-terminal domain. The iglD gene is essential for intracellular replication in primary human monocyte-derived macrophages (Santic et al., 2007). Other genes, iglC, pigC and pigG, seem to be unique to Francisella. Finally, no gene encoding a homologue to ClpV protein is found, which overall makes this system at the limits of variation to classify it as a T6SS.

### Evolutionary aspects of T6SSs

The screening of databases shows that there are almost 100 different bacterial species with a T6SS but it has been

A. Filloux, A. Hachani and S. Bleves

experimentally studied in only a few. Whereas it is mainly found in pathogenic or symbiotic bacteria of the phylum Proteobacteria, the T6SS is also found in non-pathogenic soil bacteria such as *Pseudomonas putida* and *Myxococcus xanthus* (Bingle et al., 2008). Recently, a phylogenetic analysis distinguished four branches (A–D) within the T6SS tree (Bingle et al., 2008). Interestingly, multiple T6SS clusters from a single bacterium are found in different groups. For example, *P*. aeruginosa HSI-I is in group A (together with *B*. pseudomallei and *S*. typhimurium), HSI-II is in group B (together with *R*. leguminosarum) and HSI-III is in group D (together with *V*. cholerae). The three T6SS clusters are found in all *P*. aeruginosa genome sequences that are available, i.e. PAO1, PA14 and PA7 (http://www.pseudomonas.com). *Yersinia pestis* has five T6SS loci, which are spread between the A, C and D groups. This observation indicates that multiple T6SS loci may not have arisen through duplication but rather been acquired by horizontal gene transfer. The T6SS from *F*. tularensis is phylogenetically distant from the other T6SSs, and constitutes an outward fifth branch. This confirms the weak sequence similarities with other T6SS components that we have described above.

### Conclusion

The hallmark of the T6SS is a set of approximately 15 genes found within clusters, which may vary in terms of organization. It is thought that T6SS genes encode components of a secretion machine and specific effectors. In several species, multiple T6SS loci can be found, which in some cases are incomplete. It is not known whether those loci that are most incomplete cooperate with other T6SSs. Consequently, it is difficult to unambiguously define the core components for a functional system. T6SS components that have been discussed are presented in a speculative model in Fig. 4.

Most T6SS clusters contain a gene encoding an AAA⁺ ATPase, ClpV, which may be the T6SS motor. ClpV should be considered as a core component; however, in *R*. leguminosarum, which is a T6SS archetype, no *clpV*-like gene is found in the *imp* cluster. One possibility is that some T6SSs use a protein encoded by a *clpV* gene located elsewhere on the chromosome. In order to cross the bacterial cell envelope, a membrane channel is necessary. Apart from DotU and IcmF none of the T6SS components possess transmembrane domains. There is a putative outer-membrane lipoprotein but this is unlikely to form a pore in the membrane if it is only anchored via the lipid modification.

The VgrG proteins from *V*. cholerae may form a puncturing device similar to the bacteriophage tail spike. This device may be used to perforate the bacterial envelope, and further the host cell membrane, to transport effectors all the way. Another class of proteins, the Hcps, have been shown to form hexameric rings, which may form an extracellular conduit that extends or surrounds the tube

Fig. 4. Schematic representation of a T6SS. Some T6SS components discussed in the text are displayed in the bacterial cell envelope. The ClpV ATPase (orange) may help to transport Hcp (yellow) and VgrG (dark green) across the cell envelope. Although Hcps could be secreted independently, they are represented as a channel through and outside the cell envelope as discussed in the text. The ‘VgrG’ puncturing device (dark green arrows) could be involved either in injecting the C-terminus of evolved VgrG (dark green circle) into the eukaryotic cell or in releasing VgrG into the milieu or at the bacterial cell surface. Lip (pink) is a putative outer-membrane lipoprotein. IcmF (blue) and DotU (red) are inner-membrane proteins. In some cases ClpV has been shown to interact with IcmF. The level of phosphorylation of the Fha protein (brown) regulates T6SS activity. STK (dotted light green) is the Ser/Thr kinase, whereas STP (light green) is the Ser/Thr phosphatase. The colour coding corresponds to that used for the corresponding genes in Fig. 2. C, bacterial cytoplasm; IM, bacterial inner membrane; P, bacterial periplasm; OM, bacterial outer membrane; ECM, extracellular milieu. PM (brown zone), host cell plasma membrane.

formed by the VgrG proteins. These observations are difficult to reconcile with the idea that Hcps and VgrGs are the T6SS substrates. One may compare this with the T3SS, which transports the translocator proteins to be inserted into the host cell membrane and subsequently the effectors to be translocated into the host cell cytosol (Edqvist et al., 2007). T6SSs could contribute to assembling VgrGs and Hcps outside the bacterial cell, into a putative conduit that
may be further used by as yet unknown T6SS effectors to be injected into host cells.

Some VgrGs may have a dual role, somewhat similar to what has been described for autotransporters, in which both the transporter and the catalytic domain are carried on a single polypeptide (Henderson *et al.*, 2004). Indeed, some VgrGs possess a C-terminal extension, such as an actin cross-linking domain in the *V*. *cholerae* VgrG1. This domain could be transferred across the bacterial and host membranes through the VgrG puncturing device, and eventually released by proteolytic cleavage into the host cell (Fig. 4). The presence of a tropomyosin-like domain, which putatively manipulates actin filaments, at the C-terminus of one of the *Yersinia* VgrGs also supports the idea that the T6SS is an injection machine (Pukatzki *et al.*, 2007). The presence of domains at the C-terminus of VgrG proteins, which have homology with bacterial adhesins, such as YadA, pertactin, mannose-binding domain or fibronectin-like domain, suggests that other transported domains may not be injected but remain attached at the bacterial cell surface (Pukatzki *et al.*, 2007). The identification of Hcps and VgrGs in the supernatant may be an artefact due to mechanical release of a surface-exposed structure into the medium, as with bacterial flagellin (Fernandez & Berenguer, 2000).

Whether substrates other than Hcps or VgrGs could travel through T6SSs is under debate and few candidates have been proposed. Whether all T6SS substrates are translocated into host cells or whether some are released into the extracellular medium is under debate too. One such controversy is illustrated by the finding that, in *R*. *leguminosarum*, extracellular addition of T6SS-dependent secreted protein prevents the formation of nodules on pea plants (Bladergroen *et al.*, 2003). In other words T6SS substrates may act even though added exogenously. In the case of *V*. *cholerae* this was shown not to be the case and instead, as with the T3SS or T4SS, bacterial–host contact is needed to transport T6SS effectors into the cell. Too little is known at the moment to overspeculate and it will be wise to await the elucidation of the biochemical function and target of the putative T6SS secreted effectors before we reconsider what could be their likely final localization.

T6SS genes are mostly not induced in laboratory growth conditions but are induced *in vivo* during infection. Several two-component systems have been identified that may play a crucial role in inducing/repressing expression of the T6SS genes while the bacteria are colonizing the host and encountering different environmental conditions. The T6SS seems to be involved in the intracellular behaviour of bacteria and allows chronic and persistent infection (Yahr, 2006). Furthermore, in some cases, the activity of the T6SS has been shown to be dependent on the phosphorylation status of Fha, which is subject to the antagonistic activity of Ser/Thr kinase-phosphatase. The Ser/Thr kinase-phosphatase/Fha system is lacking in the T6SS of some bacterial species or is incomplete

(The bacterial type VI secretion machine)

(Table 1). The presence of such a phosphorylation system that times T6SS assembly and function (Kulasekara & Miller, 2007) may not be mandatory, but may provide a useful switch to control delivery of effectors only upon detection of specific environmental signals. In bacterial species lacking such post-translational control, the T6SS may function constitutively once assembled.

The function and characteristics of the T6SS are far from being understood but the system appears as a novel key player in bacterial pathogenesis and bacteria–host interaction. It may become in the future another target in the hunt for novel antimicrobials, which will help fight human morbidity and mortality due to persistent bacterial infections.

### Acknowledgements

We thank Éric Durand for designing Fig. 1 on the secretion systems. We thank Neil Fairweather for helpful discussion. Alain Filloux is supported by the Royal Society.

### References

Akeda, Y. & Galan, J. E. (2005). Chaperone release and unfolding of substrates in type III secretion. *Nature* **437**, 911–915.

Backert, S. & Meyer, T. F. (2006). Type IV secretion systems and their effectors in bacterial pathogenesis. *Curr Opin Microbiol* **9**, 207–217.

Baron, G. S. & Nano, F. E. (1998). MglA and MglB are required for the intramacrophage growth of *Francisella novicida*. *Mol Microbiol* **29**, 247–259.

Bingle, L. E. H., Bailey, C. E. & Pallen, M. J. (2008). Type VI secretion: a beginner’s guide. *Curr Opin Microbiol* **11**, 3–8.

Bladergroen, M. R., Badelt, K. & Spaink, H. P. (2003). Infection-blocking genes of a symbiotic *Rhizobium leguminosarum* strain that are involved in temperature-dependent protein secretion. *Mol Plant Microbe Interact* **16**, 53–64.

Caron, E., Crepin, V. F., Simpson, N., Knutton, S., Garmedia, J. & Frankel, G. (2006). Subversion of actin dynamics by EPEC and EHEC. *Curr Opin Microbiol* **9**, 40–45.

Christie, P. J., Atmakuri, K., Krishnamoorthy, V., Jakubowski, S. & Cascales, E. (2005). Biogenesis, architecture, and function of bacterial type IV secretion systems. *Annu Rev Microbiol* **59**, 451–485.

Cornelis, G. R. (1998). The *Yersinia* Yop virulon, a bacterial system to subvert cells of the primary host defense. *Folia Microbiol (Praha)* **43**, 253–261.

Cossart, P. & Sansonetti, P. J. (2004). Bacterial invasion: the paradigms of enteroinvasive pathogens. *Science* **304**, 242–248.

Das, S. & Chaudhuri, K. (2003). Identification of a unique IAHP (IcmF associated homologous proteins) cluster in *Vibrio cholerae* and other proteobacteria through in silico analysis. *In Silico Biol* **3**, 287–300.

Das, S., Chakrabortty, A., Banerjee, R., Roychoudhury, S. & Chaudhuri, K. (2000). Comparison of global transcription responses allows identification of *Vibrio cholerae* genes differentially expressed following infection. *FEMS Microbiol Lett* **190**, 87–91.

Das, S., Chakrabortty, A., Banerjee, R. & Chaudhuri, K. (2002). Involvement of in vivo induced *icmF* gene of *Vibrio cholerae* in

http://mic.sgmjournals.org   1581
motility, adherence to epithelial cells, and conjugation frequency. Biochem Biophys Res Commun 295, 922–928.

de Bruin, O. M., Ludu, J. S. & Nano, F. E. (2007). The Francisella pathogenicity island protein IglA localizes to the bacterial cytoplasm and is needed for intracellular growth. BMC Microbiol 7, 1–10.

De Mot, R. & Vanderleyden, J. (1994). The C-terminal sequence conservation between OmpA-related outer membrane proteins and MotB suggests a common function in both gram-positive and gram-negative bacteria, possibly in the interaction of these domains with peptidoglycan. Mol Microbiol 12, 333–334.

Dudley, E. G., Thomson, N. R., Parkhill, J., Morin, N. P. & Nataro, J. P. (2006). Proteomic and microarray characterization of the AggR regulon identifies a pheU pathogenicity island in enteroaggregative Escherichia coli. Mol Microbiol 61, 1267–1282.

Edqvist, P. J., Aili, M., Liu, J. & Francis, M. S. (2007). Minimal YopB and YopD translocator secretion by Yersinia is sufficient for Yop-effector delivery into target cells. Microbes Infect 9, 224–233.

Fernandez, L. A. & Berenguer, J. (2000). Secretion and assembly of regular surface structures in Gram-negative bacteria. FEMS Microbiol Rev 24, 21–44.

Filloux, A. (2004). The underlying mechanisms of type II protein secretion. Biochim Biophys Acta 1694, 163–179.

Folkesson, A., Lofdahl, S. & Normark, S. (2002). The Salmonella enterica subspecies I specific centisome 7 genomic island encodes novel protein families present in bacteria living in close contact with eukaryotic cells. Res Microbiol 153, 537–545.

Garmendia, J., Beuzon, C. R., Ruiz-Albert, J. & Holden, D. W. (2003). The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Salmonella typhimurium SPI-2 type III secretion system. Microbiology 149, 2385–2396.

Goodman, A. L., Kulasekara, B., Rietsch, A., Boyd, D., Smith, R. S. & Lory, S. (2004). A signaling network reciprocally regulates genes associated with acute infection and chronic persistence in Pseudomonas aeruginosa. Dev Cell 7, 745–754.

Hansen, A. M., Qiu, Y., Yeh, N., Blattner, F. R., Durfee, T. & Jin, D. J. (2005). SspA is required for acid resistance in stationary phase by downregulation of H-NS in Escherichia coli. Mol Microbiol 56, 719–734.

Henderson, I. R., Navarro-Garcia, F., Desvaux, M., Fernandez, R. C. & Ala’Aldeen, D. (2004). Type V protein secretion pathway: the autotransporter story. Microbiol Mol Biol Rev 68, 692–744.

Kanamaru, S., Leiman, P. G., Kostyuchenko, V. A., Chipman, P. R., Mesyanzhinov, V. V., Arisaka, F. & Rossmann, M. G. (2002). Structure of the cell-puncturing device of bacteriophage T4. Nature 415, 553–557.

Kulasekara, H. D. & Miller, S. I. (2007). Threonine phosphorylation times bacterial secretion. Nat Cell Biol 9, 734–736.

Lauriano, C. M., Barker, J. R., Yoon, S. S., Nano, F. E., Arulanandam, B. P., Hassett, D. J. & Klose, K. E. (2004). MglA regulates transcription of virulence factors necessary for Francisella tularensis intraamoebae and intramacrophage survival. Proc Natl Acad Sci U S A 101, 4246–4249.

Mattinen, L., Nissinen, R., Riipi, T., Kalkkinen, N. & Pirhonen, M. (2007). Host-extract induced changes in the secretome of the plant pathogenic bacterium Pectobacterium atrosepticum. Proteomics 7, 3527–3537.

Merrell, D. S. & Falkow, S. (2004). Frontal and stealth attack strategies in microbial pathogenesis. Nature 430, 250–256.

Moore, M. M., Fernandez, D. L. & Thune, R. L. (2002). Cloning and characterization of Edwardsiella ictaluri proteins expressed and

recognized by the channel catfish Ictalurus punctatus immune response during infection. Dis Aquat Organ 52, 93–107.

Mota, L. J., Sorg, I. & Cornelis, G. R. (2005). Type III secretion: the bacteria-eukaryotic cell express. FEMS Microbiol Lett 252, 1–10.

Motley, S. T. & Lory, S. (1999). Functional characterization of a serine/threonine protein kinase of Pseudomonas aeruginosa. Infect Immun 67, 5386–5394.

Mougous, J. D., Cuff, M. E., Raunser, S., Shen, A., Zhou, M., Gifford, C. A., Goodman, A. L., Joachimiak, G., Ordoñez, C. L. & other authors (2006). A virulence locus of Pseudomonas aeruginosa encodes a protein secretion apparatus. Science 312, 1526–1530.

Mougous, J. D., Gifford, C. A., Ramsdell, T. L. & Mekalanos, J. J. (2007). Threonine phosphorylation post-translationally regulates protein secretion in Pseudomonas aeruginosa. Nat Cell Biol 9, 797–803.

Mukhopadhyay, S., Kapral, V., Xu, W. & Chakrabarty, A. M. (1999). Characterization of a Hank’s type serine/threonine kinase and serine/threonine phosphoprotein phosphatase in Pseudomonas aeruginosa. J Bacteriol 181, 6615–6622.

Nagai, H., Kagan, J. C., Zhu, X., Kahn, R. A. & Roy, C. R. (2002). A bacterial guanine nucleotide exchange factor activates ARF on Legionella phagosomes. Science 295, 679–682.

Nano, F. E. & Schmerk, C. (2007). The Francisella pathogenicity island. Ann N Y Acad Sci 1105, 122–137.

Nano, F. E., Zhang, N., Cowley, S. C., Klose, K. E., Cheung, K. K., Roberts, M. J., Ludu, J. S., Letendre, G. W., Meierovices, A. I. & other authors (2004). A Francisella tularensis pathogenicity island required for intramacrophage growth. J Bacteriol 186, 6430–6436.

Nataro, J. P., Yikang, D., Yingkang, D. & Walker, K. (1994). AggR, a transcriptional activator of aggregative adherence fimbria I expression in enteroaggregative Escherichia coli. J Bacteriol 176, 4691–4699.

Neuwald, A. F., Aravind, L., Spouge, J. L. & Koonin, E. V. (1999). AAA+: a class of chaperone-like ATPases associated with the assembly, operation, and disassembly of protein complexes. Genome Res 9, 27–43.

Pallen, M., Chaudhuri, R. & Khan, A. (2002). Bacterial FHA domains: neglected players in the phospho-threonine signalling game? Trends Microbiol 10, 556–563.

Pallen, M. J., Chaudhuri, R. R. & Henderson, I. R. (2003). Genomic analysis of secretion systems. Curr Opin Microbiol 6, 519–527.

Parsons, D. A. & Heffron, F. (2005). sciS, an icmF homolog in Salmonella enterica serovar Typhimurium, limits intracellular replication and decreases virulence. Infect Immun 73, 4338–4345.

Potvin, E., Lehoux, D. E., Kukavica-Ibrulj, I., Richard, K. L., Sanschagrin, F., Lau, G. W. & Levesque, R. C. (2003). In vivo functional genomics of Pseudomonas aeruginosa for high-throughput screening of new virulence factors and antibacterial targets. Environ Microbiol 5, 1294–1308.

Pukatzki, S., Ma, A. T., Sturtevant, D., Krastins, B., Sarracino, D., Nelson, W. C., Heidelberg, J. F. & Mekalanos, J. J. (2006). Identification of a conserved bacterial protein secretion system in Vibrio cholerae using the Dictyostelium host model system. Proc Natl Acad Sci U S A 103, 1528–1533.

Pukatzki, S., Ma, A. T., Revel, A. T., Sturtevant, D. & Mekalanos, J. J. (2007). Type VI secretion system translocates a phage tail spike-like protein into target cells where it cross-links actin. Proc Natl Acad Sci U S A 104, 15508–15513.

Purcell, M. & Shuman, H. A. (1998). The Legionella pneumophila icmGCDJBF genes are required for killing of human macrophages. Infect Immun 66, 2245–2255.

Rao, P. S., Yamada, Y., Tan, Y. P. & Leung, K. Y. (2004). Use of proteomics to identify novel virulence determinants that are required for *Edwardsiella tarda* pathogenesis. *Mol Microbiol* 53, 573–586.

Rappas, M., Schumacher, J., Beuron, F., Niwa, H., Bordes, P., Wigneshweraraj, S., Keetch, C. A., Robinson, C. V., Buck, M. & Zhang, X. (2005). Structural insights into the activity of enhancer-binding proteins. *Science* 307, 1972–1975.

Roest, H. P., Mulders, I. H., Spaink, H. P., Wijffelman, C. A. & Lugtenberg, B. J. (1997). A *Rhizobium leguminosarum* biovar *trifolii* locus not localized on the *sym* plasmid hinders effective nodulation on plants of the pea cross-inoculation group. *Mol Plant Microbe Interact* 10, 938–941.

Rossmann, M. G., Mesyanzhinov, V. V., Arisaka, F. & Leiman, P. G. (2004). The bacteriophage T4 DNA injection machine. *Curr Opin Struct Biol* 14, 171–180.

Saier, M. H., Jr (2006). Protein secretion and membrane insertion systems in gram-negative bacteria. *J Membr Biol* 214, 75–90.

Santic, M., Molmeret, M., Barker, J. R., Klose, K. E., Dekanic, A., Doric, M. & Abu Kwaik, Y. (2007). A *Francisella tularensis* pathogenicity island protein essential for bacterial proliferation within the host cell cytosol. *Cell Microbiol* 9, 2391–2403.

Schell, M. A., Ulrich, R. L., Ribot, W. J., Brueggemann, E. E., Hines, H. B., Chen, D., Lipscomb, L., Kim, H. S., Mrázek, J. & other authors (2007). Type VI secretion is a major virulence determinant in *Burkholderia mallei*. *Mol Microbiol* 64, 1466–1485.

Schlieker, C., Zentgraf, H., Dersch, P. & Mogk, A. (2005). ClpV, a unique Hsp100/Clp member of pathogenic proteobacteria. *Biol Chem* 386, 1115–1127.

Segal, G., Feldman, M. & Zusman, T. (2005). The Icm/Dot type-IV secretion systems of *Legionella pneumophila* and *Coxiella burnetii*. *FEMS Microbiol Rev* 29, 65–81.

Sexton, J. A. & Vogel, J. P. (2002). Type IVB secretion by intracellular pathogens. *Traffic* 3, 178–185.

Sexton, J. A., Miller, J. L., Yoneda, A., Kehl-Fie, T. E. & Vogel, J. P. (2004). *Legionella pneumophila* DotU and IcmF are required for stability of the Dot/Icm complex. *Infect Immun* 72, 5983–5992.

Shao, H., James, D., Lamont, R. J. & Demuth, D. R. (2007). Differential interaction of *Aggregatibacter (Actinobacillus) actinomy cetemcomitans* LsrB and RbsB proteins with autoinducer 2. *J Bacteriol* 189, 5559–5565.

Sheahan, K. L., Cordero, C. L. & Satchell, K. J. (2004). Identification of a domain within the multifunctional *Vibrio cholerae* RTX toxin that covalently cross-links actin. *Proc Natl Acad Sci U S A* 101, 9798–9803.

Sheikh, J., Czeczulin, J. R., Harrington, S., Hicks, S., Henderson, I. R., Le Bouguenec, C., Gounon, P., Phillips, A. & Nataro, J. P. (2002). A novel dispersin protein in enteroinvasive *Escherichia coli*. *J Clin Invest* 110, 1329–1337.

Van Brussel, A. A., Zaat, S. A., Cremers, H. C., Wijffelman, C. A., Pees, E., Tak, T. & Lugtenberg, B. J. (1986). Role of plant root exudate and Sym plasmid-localized nodulation genes in the synthesis by *Rhizobium leguminosarum* of Tsr factor, which causes thick and short roots on common vetch. *J Bacteriol* 165, 517–522.

VanRheenen, S. M., Dumenil, G. & Isberg, R. R. (2004). IcmF and DotU are required for optimal effector translocation and trafficking of the *Legionella pneumophila* vacuole. *Infect Immun* 72, 5972–5982.

Ventre, I., Goodman, A. L., Vallet-Gely, I., Vasseur, P., Soscia, C., Molin, S., Bleves, S., Lazdunski, A., Lory, S. & Filloux, A. (2006). Multiple sensors control reciprocal expression of *Pseudomonas aeruginosa* regulatory RNA and virulence genes. *Proc Natl Acad Sci U S A* 103, 171–176.

Wang, J., Li, C., Yang, H., Mushegian, A. & Jin, S. (1998a). A novel serine/threonine protein kinase homologue of *Pseudomonas aeruginosa* is specifically inducible within the host infection site and is required for full virulence in neutropenic mice. *J Bacteriol* 180, 6764–6768.

Wang, Y. D., Zhao, S. & Hill, C. W. (1998b). *Rhs* elements comprise three subfamilies which diverged prior to acquisition by *Escherichia coli*. *J Bacteriol* 180, 4102–4110.

Weibezahn, J., Tessarz, P., Schlieker, C., Zahn, R., Maglica, Z., Lee, S., Zentgraf, H., Weber-Ban, E. U., Dougan, D. A. & other authors (2004). Thermotolerance requires refolding of aggregated proteins by substrate translocation through the central pore of ClpB. *Cell* 119, 653–665.

Wilkins, B. M. & Thomas, A. T. (2000). DNA-independent transport of plasmid primase protein between bacteria by the I1 conjugation system. *Mol Microbiol* 38, 650–657.

Williams, S. G., Varcoe, L. T., Attridge, S. R. & Manning, P. A. (1996). *Vibrio cholerae* Hcp, a secreted protein coregulated with HlyA. *Infect Immun* 64, 283–289.

Yahr, T. L. (2006). A critical new pathway for toxin secretion? *N Engl J Med* 355, 1171–1172.

Yamaguchi, K., Yu, F. & Inouye, M. (1988). A single amino acid determinant of the membrane localization of lipoproteins in *Escherichia coli*. *Cell* 53, 423–432.

Yeo, H. J. & Waksman, G. (2004). Unveiling molecular scaffolds of the type IV secretion system. *J Bacteriol* 186, 1919–1926.

Zheng, J. & Leung, K. Y. (2007). Dissection of a type VI secretion system in *Edwardsiella tarda*. *Mol Microbiol* 66, 1192–1206.

Zheng, J., Tung, S. L. & Leung, K. Y. (2005). Regulation of a type III and a putative secretion system in *Edwardsiella tarda* by EsrC is under the control of a two-component system, EsrA-EsrB. *Infect Immun* 73, 4127–4137.

Zusman, T., Feldman, M., Halperin, E. & Segal, G. (2004). Characterization of the *icmH* and *icmF* genes required for *Legionella pneumophila* intracellular growth, genes that are present in many bacteria associated with eukaryotic cells. *Infect Immun* 72, 3398–3409.
