
# REVIEWS

## Methanogenic archaea: ecologically relevant differences in energy conservation

**Rudolf K. Thauer**, **Anne-Kristin Kaster**, **Henning Seedorf**, **Wolfgang Buckel** and **Reiner Hedderich**

### Abstract | Most methanogenic archaea can reduce CO₂ with H₂ to methane, and it is generally assumed that the reactions and mechanisms of energy conservation that are involved are largely the same in all methanogens. However, this does not take into account the fact that methanogens with cytochromes have considerably higher growth yields and threshold concentrations for H₂ than methanogens without cytochromes. These and other differences can be explained by the proposal outlined in this Review that in methanogens with cytochromes, the first and last steps in methanogenesis from CO₂ are coupled chemiosmotically, whereas in methanogens without cytochromes, these steps are energetically coupled by a cytoplasmic enzyme complex that mediates flavin-based electron bifurcation.

---

**Syntrophic**  
A nutritional situation in which two or more organisms combine their metabolic capabilities to catabolize a substance that cannot be catabolized by either one of them alone.

Methanogenic archaea are a phylogenetically diverse group of strictly anaerobic Euryarchaeota with an energy metabolism that is restricted to the formation of methane from CO₂ and H₂, formate, methanol, methylamines and/or acetate¹⁻⁵. Despite this restriction, these organisms have a quantitatively important role in the global carbon cycle (FIG. 1). It is estimated that approximately 1 billion tons (1 giga ton) of methane per year is formed globally by methanogenic archaea in anoxic environments, such as freshwater sediments, swamps, paddy fields, landfills and the intestinal tracts of ruminants and termites⁴. This means that approximately 2% of the net CO₂ that is fixed annually into biomass by photosynthesis (70 giga tonnes of carbon per year) ends up primarily as methane.

Methanogenesis from the biomass in the anoxic ecosystems that are mentioned above is catalysed by a syntrophic association between anaerobic bacteria, protozoa and/or anaerobic fungi, and syntrophic bacteria, acetogenic bacteria and methanogenic archaea (FIG. 1). The anaerobic bacteria, protozoa and fungi (mainly present in the rumen of ruminants) hydrolyse biopolymers to monomers and lipids to glycerol and long-chain fatty acids, and ferment these, together with syntrophic bacteria, to acetic acid, CO₂ and H₂ (REFS 6,7). These are the sole fermentation products only when polymer hydrolysis is the rate-limiting step

in methanogenesis from biomass and when the H₂ concentration is kept below 10 Pa by methanogenic archaea (reaction 1, see BOX 1)⁸⁻¹⁰ and/or acetogenic bacteria (2 CO₂ + 4 H₂ → CH₃COO⁻ + H⁺ + 2 H₂O; ΔG°' equals –95 kJ per mole)¹¹ (see BOX 1 for details of the calculation of ΔG°'). Because the free energy change (ΔG) of acetogenesis from H₂ and CO₂ increases with increasing H₂ concentration, increasing pH and decreasing temperature, acetogenic bacteria preferentially channel H₂ and CO₂ into acetic acid synthesis at high H₂ concentrations, a pH of more than 7 and low temperatures. By contrast, at low H₂ concentrations, a pH of less than 7 and high temperatures, acetogenic bacteria channel acetic acid into H₂ and CO₂ formation¹²,¹³. The methanogens then finally convert acetate, H₂ and CO₂ to methane¹³. In ruminants and termites, the acetic acid that is formed by fermentation is resorbed from the intestinal tract and used by the host as an energy source. In the intestinal tract, methanogens that can grow on H₂ and CO₂ or formate are mainly found¹⁴,¹⁵.

Methane is a major end product of anaerobic biomass degradation only in anoxic environments where the concentrations of sulphate, nitrate, Mn(IV) or Fe(III) are low. In the presence of these electron acceptors, methanogenesis is out-competed by anaerobic respiration, mainly for thermodynamic reasons. Thus, in the upper layer of marine sediments, where the

*Max Planck Institute for Terrestrial Microbiology,*  
*Karl-von-Frisch-Strasse,*  
*D-35043 Marburg, Germany.*

*Laboratory for Microbiology,*  
*Faculty of Biology, Philipps University Marburg,*  
*D-35032 Marburg, Germany.*

Correspondence to R.K.T.  
e-mail: thauer@mpi-marburg.mpg.de  

doi:10.1038/nrmicro1931  
Published online 30 June 2008

---

NATURE REVIEWS | MICROBIOLOGY  
© 2008 Macmillan Publishers Limited. All rights reserved.

Troposphere

$\mathrm{CO}_{2}(380 \mathrm{ppm})$

Photo-oxidation

$0.6 \mathrm{Gt}$ per year

Methane from other sources (for example, gas-pipe leakages and biomass fires)

Methane released from anoxic environments (for example, paddy fields and wetlands) directly into the troposphere

Oxic environments

$\mathrm{CO}_{2}$

Oxygenic photosynthesis

Biomass

~70 Gt C per year (100%)

Aerobic oxidation

Aerobic methanotrophic bacteria

$\mathrm{CO}_{2}$

Aerobic oxidation

Acetate$^{-}$

$0.6 \mathrm{Gt}$ methane per year

$0.6 \mathrm{Gt}$ per year

Anoxic environments

Biomass

Anaerobic oxidation with sulphate, nitrate and Fe(III)

Biomass (~2%)

Bacteria, protozoa and fungi

Acetate$^{-}+\mathrm{H}^{+}$

Acetogenic bacteria

$\mathrm{CO}_{2}+\mathrm{H}_{2}$

Methanogens with cytochromes

$1 \mathrm{Gt}$ per year

$\mathrm{CH}_{4}$

$1 \mathrm{Gt}$ per year

$\mathrm{CH}_{4}$

$1 \mathrm{Gt}$ per year

Recalcitrant organic material

Microbial and thermogenic

Methane hydrates (>10,000 Gt methane)

Geochemical

Figure 1 | Methane as an intermediate in the global carbon cycle. Continuous arrows indicate a reaction and dashed arrows indicate diffusion and/or convection. In anoxic environments (for example, freshwater sediments, swamps, paddy fields, landfills and the intestinal tracts of ruminants and termites), approximately 1 giga ton (Gt) of methane ($10^{15} \mathrm{~g}$) is formed per year from acetate, $\mathrm{CO}_{2}$ and $\mathrm{H}_{2}$ through the metabolic activity of methanogenic archaea. Almost the same amount of methane is released into the environment from melting methane hydrates. From the $2 \mathrm{Gt}$ of methane that is produced per year, ~0.6 Gt is oxidized to $\mathrm{CO}_{2}$ by aerobic bacteria, ~1 Gt is oxidized by anaerobic archaea and ~0.4 Gt escapes into the atmosphere. Another 0.2 Gt per year is released into the atmosphere from other sources, such as gas-pipe leakages and the burning of biomass. In the atmosphere, most of the methane is photo-oxidized to $\mathrm{CO}_{2}$. Only 0.03 Gt per year is removed from the atmosphere by aerobic bacteria that live in soils and water. The concentration of methane in the troposphere has increased over the past 100 years from 0.9 to 1.8 parts per million (ppm), which is of concern as methane is a potent greenhouse gas. For literature, see REFS 123–125.

sulphate concentration is usually high (near to $30 \mathrm{mM}$), methanogenesis is restricted to substrates such as methylamines (glycine-betain fermentation products) that generally cannot be metabolized by sulphate-reducing bacteria.

FIGURE 1 shows that methane can be oxidized to $\mathrm{CO}_{2}$ by microorganisms not only aerobically but also anaerobically; the most dominant anaerobic process is the anaerobic oxidation of methane (AOM) with sulphate$^{16}$. AOM usually occurs when methane is the only available electron donor and the concentration of suitable electron acceptors is high, which is the case in marine sediments that are located directly above methane hydrates (high sulphate concentrations) or organic-rich freshwater sediments that contain high concentrations of nitrate$^{17}$. Interestingly, AOM with sulphate involves archaea that are closely related to methanogens containing cytochromes, and even more interestingly, AOM with sulphate seems to involve the nickel-containing enzyme methyl-coenzyme M reductase$^{18,19}$, which catalyses the methane-forming step in all methanogenic archaea$^{20-24}$.

This Review focuses on the energy metabolism of methanogenic archaea that grow on $\mathrm{H}_{2}$ and $\mathrm{CO}_{2}$, with an emphasis on the differences between methanogens with and without cytochromes. We first describe the differences in the electron-carrier apparatus that are reflected in differences in growth yields ($Y_{\mathrm{CH}_{4}}$; the amount of dried cells in grams (g) per mole of methane), ATP gains (moles of ATP per mole of methane) and $\mathrm{H}_{2}$ threshold concentrations. We then outline how methanogens with and without cytochromes conserve energy during growth on $\mathrm{CO}_{2}$ and $\mathrm{H}_{2}$, and highlight how in methanogens with cytochromes, the first and last steps in methanogenesis from $\mathrm{CO}_{2}$ are coupled chemiosmotically, whereas the available evidence indicates that in methanogens without cytochromes, these steps are energetically coupled by a cytoplasmic enzyme

Box 1 | The reactions involved in CO₂ or methanol reduction with H₂ to methane

In the table, reactions 1–6, 8, 10, 15 and 16 are catalysed by cytoplasmic proteins^{4,93} and reactions 7, 9 and 11–14 are catalysed by membrane-associated enzyme complexes. In methanogens with cytochromes, CoM-S-S-CoB reduction with H₂ generally proceeds through reactions 11 and 12, whereas in methanogens without cytochromes, CoM-S-S-CoB reduction with H₂ involves reaction 15. The standard free energy change (ΔG°') was calculated from equilibrium constants or from the standard free energies of formation at 25°C with H₂, CO₂ and CH₄ in the gaseous state at 10⁵ Pa, H₂O in the liquid state, pH at 7.0 and all other compounds at 1 molar activity^{4,8}.

| Reaction number | Equation                                                                                   | ΔG°' (kJ per mole)                                                                 |
|------------------|-------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|
| 1                | 4 H₂ + CO₂ → CH₄ + 2 H₂O                                                                  | –131                                                                                |
| 2                | CO₂ + MFR + Fd²⁻_{red} + 2 H⁺ ⇌ CHO-MFR + Fd⁺_{ox} + H₂O                                  | 0                                                                                  |
| 3                | CHO-MFR + H₄MPT ⇌ CHO-H₄MPT + MFR                                                        | –5                                                                                 |
| 4                | CHO-H₄MPT + H⁺ ⇌ CH≡H₄MPT⁺ + H₂O                                                          | –5                                                                                 |
| 5                | CH≡H₄MPT⁺ + F₄₂₀H₂ ⇌ CH₂=H₄MPT + F₄₂₀⁺ + H⁺                                               | +6                                                                                 |
| 6                | CH₂=H₄MPT + F₄₂₀H₂ ⇌ CH₃-H₄MPT + F₄₂₀                                                    | –6                                                                                 |
| 7                | CH₃-H₄MPT + HS-CoM ⇌ CH₃-S-CoM + H₄MPT                                                   | –30 (coupled with 2 Na⁺ translocations)                                            |
| 8                | CH₃-S-CoM + HS-CoB ⇌ CH₄ + CoM-S-S-CoB                                                   | –30                                                                                |
| 9                | H₂ + Fd⁺_{ox} ⇌ Fd²⁻_{red} + 2 H⁺                                                       | +16 (coupled to 2 H⁺, or possibly 2 Na⁺, translocations)                         |
| 10               | H₂ + F₄₂₀ ⇌ F₄₂₀H₂ (x 2)                                                                 | –11                                                                                |
| 11               | H₂ + MP ⇌ MPH₂                                                                            | –50 (coupled with 2 H⁺ translocations)                                             |
| 12               | MPH₂ + CoM-S-S-CoB ⇌ MP + HS-CoM + HS-CoB                                                | –5 (coupled with 2 H⁺ translocations)                                              |
| 13               | ADP + Pᵢ ⇌ ATP + H₂O                                                                     | –32 (coupled to 4 H⁺, or possibly 4 Na⁺, translocations)                          |
| 14               | 2 H⁺ (outside) + 1 Na⁺ (inside) ⇌ 2 H⁺ (inside) + 1 Na⁺ (outside)                        | 0                                                                                  |
| 15               | 2 H₂ + CoM-S-S-CoB + Fd⁺_{ox} ⇌ HS-CoM + HS-CoB + Fd²⁻_{red} + 2 H⁺                      | –39                                                                                |
| 16               | CH₃OH + HS-CoM ⇌ CH₃-S-CoM + H₂O                                                         | –17.5                                                                              |

F₄₂₀, coenzyme F₄₂₀; Fd, ferredoxin; H₄MPT, tetrahydrosarcinapterin; HS-CoB, coenzyme B; HS-CoM, coenzyme M; MFR, methanofuran; MP, methanophenazine.

complex that mediates flavin-based electron bifurcation (a coupling mechanism that was recently discovered in clostridia)^{25-27}. Finally, we point out how by involving flavoprotein-linked electron bifurcation one can also explain how *Methanosphaera stadtmanae* can grow on methanol and H₂, for which there has previously been no convincing explanation.

---

### Electron bifurcation
Separation of the two electrons from ubiquinol at the quinol oxidation site of the bc₁ complex (complex III) of the respiratory chain, which leads to a bifurcation of the two electrons to a high and a low potential pathway.

### Methanogens with and without cytochromes
Five orders of methanogenic archaea have been identified (BOX 2): Methanopyrales, Methanococcales, Methanobacteriales, Methanomicrobiales and Methanosarcinales. The order with the deepest root among the Euryarchaeota is the Methanopyrales and the one that branches off last is the Methanosarcinales¹. All members of the Methanosarcinales contain cytochromes^{28-31} and methanophenazine (a functional menaquinone analogue)^{32-34} and have a broad substrate spectrum. For example, *Methanosarcina barkeri* can use all the methanogenic substrates discussed above, except for formate³⁵. The members of the other four orders lack cytochromes and methanophenazine and reduce CO₂ with H₂ to methane, although some members can also use formate as an electron donor³⁶. Only one methanogen without cytochromes, namely *M. stadtmanae*, cannot reduce CO₂ to methane. This human intestinal archaeon is instead dependent on methanol and H₂ as energy sources³⁷. The differences between methanogenic archaea with and without cytochromes that can grow on H₂ and CO₂ are summarized in BOX 2. In the following sections, the differences in growth yields, ATP gains, H₂ thresholds and upper temperature growth optima are discussed.

#### Growth yields and ATP gains.
Methanogens with cytochromes that can grow on H₂ and CO₂, such as *M. barkeri*, have a much higher growth yield on H₂ and CO₂ than methanogens without cytochromes (BOX 2): the reported growth yields are 6.4 g per mole for *M. barkeri* at 37°C³⁸; 1.4 g per mole for *Methanobrevibacter arboriphilus*³⁹; 1.3 g per mole for *Methanobacterium bryantii*⁴⁰; 1.9 g per mole for *Methanothermobacter thermoautotrophicus* (formerly known as *Methanobacterium thermoautotrophicum* strain delta H)⁴¹,⁴²; and up to 3 g per mole for *Methanothermobacter marburgensis* (formerly known as *Methanobacterium thermoautotrophicum* strain Marburg)⁴³. In a recent comparative study in which the same minimal growth medium was used, *M. barkeri* (which has cytochromes) was shown to grow at 37°C

Box 2 | Selected differences between methanogenic archaea

The complete genome sequences of 6 methanogens with cytochromes and 15 methanogens without cytochromes (including 4 *Methanococcus maripaludis* strains) and the incomplete genome sequences of another 34 methanogens are available in the Genomes OnLine Database (see Further information). Methanogens with cytochromes all belong to the order of Methanosarcinales (which includes the Methanosarcina, Methanoaeta and Methanobolus genera). Methanogens without cytochromes include the Methanobacteriales, Methanococcales, Methanomicrobiales and Methanopyrales orders.

**Methanogens with cytochromes**

- Contain methanophenazine (a functional menaquinone analogue).
- Growth on H₂ and CO₂ is restricted to some Methanosarcina species; most can grow on acetate, methanol and methylamines and cannot grow on formate³⁵.
- Threshold H₂ partial pressure is generally >10 Pa.
- Growth yields on H₂ and CO₂ of up to 7 g per mole of methane.
- Doubling times are generally >10 hours.
- No hyperthermophilic species.

**Methanogens without cytochromes**

- Do not contain methanophenazine.
- Can grow on H₂ and CO₂, except for Methanospaera stadtmanae; cannot grow on acetate or methylamines and many can grow on formate³⁶.
- Threshold H₂ partial pressure is generally <10 Pa.
- Growth yields on H₂ and CO₂ of up to 3 g per mole of methane.
- Doubling times can be as low as 1 hour.
- Many hyperthermophilic species.

high ATP gain of >1 for M. barkeri, which was calculated from the growth yield of 7.2 g per mole, indicates that this organism should not be able to grow at a pH₂ that is below 10³ Pa, which is what is actually observed.

**H₂ thresholds.** In the reduction of CO₂ with H₂ to methane (reaction 1, see BOX 1), the thermodynamic equilibrium (∆G′ equals 0 kJ per mole) is theoretically reached at 25°C and a pH₂ of approximately 0.1 Pa (assuming that the partial pressure of CO₂ is equal to the partial pressure of methane; ∆G′ = ∆G°′ + 2.3 RT log [CH₄]/[H₂]⁴ × [CO₂], in which R is the gas constant and T is the temperature in degrees Kelvin). However, when this is coupled with the phosphorylation of ADP (4 H₂ + CO₂ + n ADP + n Pi → CH₄ + n ATP + 3 H₂O), the theoretical H₂ threshold concentration is higher. Therefore, if n equals 0.5 and there is a phosphorylation potential of –50 kJ per mole, the threshold pH₂ is ~2 Pa, whereas if n equals 1, the threshold is ~30 Pa. This explains why the final concentration of H₂ (the threshold concentration) in the presence of methanogens is generally much higher than 0.1 Pa (∆G of <0 kJ per mole)¹⁰,¹¹,⁴⁸⁻⁵³ (for different results, see REFS 54,55). The experimentally determined threshold is always lower than the calculated threshold, mainly owing to the partial uncoupling of methanogenesis from ATP synthesis (which decreases n, the ATP gain) and to the decrease in the phosphorylation potential at low rates of methanogenesis that is caused by low H₂ concentrations. Functionally distinct genes have been shown to be regulated by hydrogen limitation and growth rate in methanogenic archaea⁵⁶. Nonetheless, organisms with a higher ATP gain have a higher H₂ threshold concentration. Indeed, methanogens without cytochromes cease the oxidation of H₂ at pH₂ 1–10 Pa, whereas methanogens with cytochromes cease oxidation at a concentration that is at least tenfold higher¹,².

The inability of methanogens with cytochromes to compete with methanogens without cytochromes for H₂ under the conditions that prevail in their natural environments probably explains why many methanogens with cytochromes, including *Methanosarcina acetivorans*, *Methanobolus tindarius* and *Methanotherix soehngenii*, have lost the ability to grow on H₂ and CO₂. Most of the genes that are required for growth on H₂ and CO₂ are present in the genome of *M. acetivorans*, but do not seem to be transcribed⁵⁷⁻⁵⁹.

The different thresholds for H₂ could also explain why growth on formate seems to be restricted to methanogens without cytochromes. In methanogens, formate is converted through the coenzyme F₄₂₀-dependent formate dehydrogenase (HCOO⁻ + H⁺ + F₄₂₀ ⇌ CO₂ + F₄₂₀H₂; ∆G°′ equals –14 kJ per mole) and the F₄₂₀-reducing hydrogenase FrhABC (F₄₂₀H₂ ⇌ F₄₂₀ + H₂; ∆G°′ equals +11 kJ per mole) to CO₂ and H₂, from which methane is subsequently formed⁶⁰. The production of H₂ as an intermediate in methanogenesis from formate is a problem for organisms with a high H₂ threshold if they are in competition with other organisms with lower H₂ thresholds in their natural environments⁶¹.

M. barkeri  
(with cytochromes)  
4 H₂ + CO₂ → CH₄ + 2 H₂O  

EchA–F  
VhoACG–  
HdrDE  

methyltetrahydromethanopterin: coenzyme M methyltransferase, which is involved in CO₂ reduction to methane and acetate disproportionation into methane and CO₂, is a primary sodium ion pump that requires high sodium ion concentrations for activity (~1 mM Na⁺ to obtain 0.5 V_max⁶⁵). In line with this interpretation is the fact that methane formation from methanol and H₂ in cell suspensions of M. barkeri⁶²,⁶⁶ and M. stadtmanae⁶⁷, which does not involve the sodium ion-translocating methyltransferase, is sodium ion independent down to concentrations of 0.3 mM.

Energy conservation involving cytochromes  
The reduction of CO₂ with H₂ to methane and energy conservation in methanogens with cytochromes is considered to involve 13 reactions (reactions 2–14, see BOX 1) in which methanofuran (MFR; a 2-aminomethylfuran derivative), tetrahydrosarcinapterin (H₄MPT; a tetrahydrofolate analogue) and coenzyme M (HS-CoM; also known as 2-thioethanesulphonate) are C₁-unit carriers³⁴ and ferredoxin (Fd), coenzyme F₄₂₀ (a 5′-deazaflavin derivative; E⁰′ equals –360 mV), coenzyme B (HS-CoB; also known as 7-thioheptanoyl-o-phospho-L-threonine; E⁰′ equals –140 mV)³⁴ and methanophenazine (MP; E⁰′ equals –165 mV)³⁴ are electron carriers³⁻⁵ (FIG. 2). Reactions 2–6, 8 and 10 (BOX 1) are catalysed by cytoplasmic enzymes and reactions 7, 9 and 11–14 (BOX 1) are catalysed by membrane-protein complexes. This information is integrated into FIG. 2, in which the reactions that are involved in the reduction of CO₂ with H₂ to methane, their topology and their coupling with the translocation of protons or sodium ions across the cytoplasmic membrane are shown.

Methanogens with cytochromes contain numerous ferredoxins that harbour at least two [4Fe–4S] clusters, which indicates that they can accept more than one electron. For calculation purposes, the standard redox potential (E⁰′) of the ferredoxin that is involved in reactions 2 and 9 (BOX 1) was set at –500 mV, which is the E⁰′ of the CO₂/CHO-MFR couple⁴. Thus, the ΔG′ of reaction 2 (BOX 1), which has been shown to operate near equilibrium in vivo, becomes 0 kJ per mole⁴¹. The ferredoxin that is involved in reactions 2 and 9 (BOX 1) and its redox potential are not yet known.

The membrane proteins involved. The energy conservation that is associated with the reduction of CO₂ with H₂ to methane in methanogens with cytochromes involves six membrane-associated protein complexes: methyl-H₄MPT–coenzyme M methyltransferase (MtrA–H; reaction 7, BOX 1), energy-converting [NiFe] hydrogenase (EchA–F; reaction 9, BOX 1), methanophenazine-reducing [NiFe] hydrogenase (VhoACG; reaction 11, BOX 1), methanophenazine-dependent heterodisulphide reductase (HdrDE; reaction 12, BOX 1), an A₁A₀-ATP synthase (AhaA–K; reaction 13, BOX 1) and an Na⁺/H⁺ antiporter (reaction 14, BOX 1).

MtrA–H contains a corrinoid that is bound to the MtrA subunit, and has been shown to couple reaction 7 (BOX 1) with the translocation of two sodium ions⁶⁵,⁶⁸. EchA–F⁶⁹,⁷⁰ is a nickel–iron–sulphur protein in which

Figure 2 | The coupling sites that are proposed to be involved in energy conservation in Methanosarcina barkeri growing on CO₂ and H₂. The numbers in bold correspond to the reaction numbers in BOX 1. The first and last steps are chemiosmotically coupled. The ATP gain (moles of ATP per mole of methane) is assumed to be 1.5. The redox potentials are standard potentials at pH 7.0 (E⁰′). The E⁰′ of ferredoxin was set at –500 mV, which is the E⁰′ of the CO₂/CHO-MFR couple (discussed in the main text). C₁ units and the cytochrome b subunits VhoC and HdrE are highlighted in red. Fd, ferredoxin; H₄MPT, tetrahydrosarcinapterin; HS-CoB, coenzyme B; HS-CoM, coenzyme M; MFR, methanofuran.

---

The upper temperature limit for growth. Another interesting difference between methanogens with and without cytochromes is that at low temperatures (4°C), methanogens with cytochromes predominate, whereas at higher temperatures (>60°C), it is mainly methanogens without cytochromes that are abundant¹¹,⁴⁹; methanogens with cytochromes have not yet been found to grow at temperatures above 60°C. Hyperthermophily seems to be restricted to methanogens without cytochromes, such as *Methanopyrus kandleri* (*T*ₘₐₓ of 110°C), *Methanocaldococcus jannaschii* (*T*ₘₐₓ of 88°C) and *Methanothermus fervidus* (*T*ₘₐₓ of 97°C). Notably, the ΔG′ that is associated with methane formation from H₂ and CO₂ decreases with increasing temperature from –131 kJ per mole at 25°C to –100 kJ per mole at 100°C¹². At high temperatures and a low *pH₂* (<10 Pa), CO₂ reduction with H₂ is not exergonic enough to allow the growth of methanogens with an ATP gain of 1 or more.

Common dependence on sodium ions. Both methanogens with and without cytochromes require high concentrations of sodium ions for growth and methane formation (~1 mM Na⁺ to obtain 0.5 V_max⁶²⁻⁶⁴). This sodium ion dependence can be explained by the finding that

Hyperthermophily  
A growth temperature optimum of 80°C or higher.

Corrinoid  
A cobalt-containing tetrapyrrole, such as vitamin B₁₂ or coenzyme B₁₂.
the EchE subunit harbours the active-site \[NiFe\] centre${}^{71,72}$, and is assumed to be proton-translocating, as the conversion of CO to $\mathrm{CO}_{2}$ and $\mathrm{H}_{2}$ in cell suspensions of $M$. barkeri$, which involves the EchA–F complex, does not seem to be sodium ion dependent${}^{69,70,73-75}$. However, there are also indications that the first step in $\mathrm{CO}_{2}$ reduction with $\mathrm{H}_{2}$ in $M$. barkeri$ — the reduction of $\mathrm{CO}_{2}$ with $\mathrm{H}_{2}$ to formyl-MFR (reactions 2 plus 9; see BOX 1; $\Delta G^{\prime \prime}$ equals +16 kJ per mole) — could be directly driven by the electrochemical sodium ion potential. For example, methanogenesis from $\mathrm{CO}_{2}$ and $\mathrm{H}_{2}$ is not affected by protonophores when the $\mathrm{Na}^{+} / \mathrm{H}^{+}$ antiporter is inhibited${}^{76,77}$, which is difficult to explain if we assume that reaction 9 (BOX 1) is coupled to proton translocation. VhoACG${}^{78-80}$ is a nickel–iron–sulphur haemoprotein in which VhoA harbours the active-site \[NiFe\] centre and VhoC (a $b$-type cytochrome) harbours the haem group. In the cytoplasmic membrane, the VhoACG complex is orientated with its active site facing the periplasm${}^{81}$. HdrDE is an iron–sulphur haemoprotein in which HdrD harbours an unusual active-site FeS cluster and HdrE (a $b$-type cytochrome) harbours the haem group${}^{82-85}$. The exergonic reduction of the heterodisulphide with $\mathrm{H}_{2}$ ($\Delta G^{\prime \prime}$ equals –55 kJ per mole; reactions 11 and 12, see BOX 1), which is catalysed by the VhoACG and HdrDE complexes, is coupled with the build-up of an electrochemical proton potential${}^{86,86}$ and is likely to have a proton to electron ratio of 2 (REFS 78,79). AhaA–K${}^{80}$ is a proton-translocating ATP synthase that is likely to have a proton to ATP stoichiometry of 4 (REFS 87–89). The $\mathrm{Na}^{+} / \mathrm{H}^{+}$ antiporter Nha from methanogens is related to the $\mathrm{Na}^{+} / \mathrm{H}^{+}$ antiporter in *Escherichia coli*${}^{90}$, for which the proton to sodium ion stoichiometry has been determined to be 2 (REFS 91,92). However, it should be noted that in methanogens with cytochromes (FIG. 2), Nha is assumed to have an opposite function to that of the antiporter in *E*. coli: in *E*. coli (which lacks a primary sodium ion pump), Nha is assumed to be involved in the build-up of the sodium motive force, whereas in $M$. barkeri$, Nha is assumed to be involved in the build-up of the proton motive force.

In the metabolic scheme shown in FIG. 2, the ions involved and the stoichiometries of chemiosmotic coupling have been chosen to best fit all of the experimental data, including the ATP gain of >1. It should be noted, however, that there is considerable uncertainty surrounding the prediction of ATP gains from growth yields and the determination of the number and type of cations that are translocated by the energy-converting membrane complexes. Because six cation-translocating complexes are involved, it is almost impossible to measure the activity of one complex *in vivo* without interference from one of the other complexes.

The cytoplasmic enzymes involved. Reactions 2–6, 8 and 10 (BOX 1) are catalysed by cytoplasmic enzymes. The enzymes that catalyse reactions 3–6 are composed of only one type of subunit and do not possess a prosthetic group. Reaction 2 is catalysed by a molybdenum or tungsten iron–sulphur protein (Fmd or Fwd; formylmethanofuran dehydrogenase) that contains five different subunits, reaction 8 is catalysed by the nickel porphinoid ${\mathrm{F}}_{430}$-harbouring methyl-coenzyme M reductase (Mcr or Mrt), which contains three different types of subunit, and reaction 10 is catalysed by a nickel–iron–sulphur flavoprotein (Frh; an ${\mathrm{F}}_{420}$-reducing hydrogenase) that is composed of three different types of subunit${}^{4,93}$.

An alternative pathway. In some *Methanosarcina* species, the reduction of methanophenazine with $\mathrm{H}_{2}$ (reaction 11, see BOX 1) can also be catalysed by the cytoplasmic ${\mathrm{F}}_{420}$-reducing hydrogenase FrhABC (reaction 10, BOX 1) in combination with a membrane-associated, energy-conserving ${\mathrm{F}}_{420}{\mathrm{H}}_{2}$-dehydrogenase complex (FpoABCDHIJKLMNO) that is related to NADH dehydrogenase, which catalyses the reduction of methanophenazine with ${\mathrm{F}}_{420}{\mathrm{H}}_{2}$ (${\mathrm{F}}_{420}{\mathrm{H}}_{2}+\mathrm{MP} \rightarrow {\mathrm{F}}_{420}+\mathrm{MPH}_{2}$; $\Delta G^{\prime \prime}$ equals –38 kJ per mole). This reaction is coupled to the translocation of two protons across the cytoplasmic membrane${}^{94}$.

Energy conservation not involving cytochromes

It is generally thought that $\mathrm{CO}_{2}$ reduction with $\mathrm{H}_{2}$ to methane and energy conservation in methanogens without cytochromes proceeds in principle as it does in methanogens with cytochromes. Only the reduction of the heterodisulphide CoM-S-S-CoB with $\mathrm{H}_{2}$ (reactions 11 and 12, BOX 1), which does not involve cytochromes and methanophenazine, was known to be different. However, the enzyme complex that catalyses this reaction was thought to also be membrane associated and proton translocating. In the following sections, we present an alternative picture.

The similarities. Most of the enzymes and coenzymes that are involved in the reduction of $\mathrm{CO}_{2}$ with $\mathrm{H}_{2}$ to methane in methanogens with cytochromes are also found in methanogens without cytochromes. The enzymes that catalyse reactions 2–10, 13 and 14 (BOX 1) are phylogenetically related and have similar structures and identical cellular locations. Only the energy-converting hydrogenase (reaction 9, BOX 1) has a substantially different number of subunits: in $M$. barkeri$, the Ech complex contains only 6 subunits, whereas the Eha and Ehb complexes contain at least 16 subunits${}^{95}$. Interestingly, the hydrogenases Ech, Eha and Ehb are phylogenetically related to complex I (the NADH–ubiquinol oxidoreductase complex) of the respiratory chain. *E*. coli complex I contains 14 subunits and the complex I of mitochondria contains more than 40 subunits${}^{96}$. There are some differences in the structure of the coenzymes MFR, ${\mathrm{H}}_{4}\mathrm{MPT}$ and ${\mathrm{F}}_{420}$, but these are not functionally important. *In vitro*, the enzymes from methanogens without cytochromes can use the coenzymes from methanogens with cytochromes and *vice versa*, although generally with a lower catalytic efficiency${}^{4}$.

The apparent lack of a coupling site. Methanogens with and without cytochromes differ in how they reduce CoM-S-S-CoB with $\mathrm{H}_{2}$. Instead of the membrane complexes VhoACG (reaction 11, BOX 1) and HdrDE
A way out of the dilemma? How methanogens without cytochromes conserve energy during methanogenesis from CO₂ even though the components that are involved in heterodisulphide reduction with H₂ are thought to be cytoplasmic and have no association with the cytoplasmic membrane clearly remains an important unsolved question.

A similar question regarding the energy metabolism of *Clostridium kluyveri*, which does not contain cytochromes, menaquinone or ubiquinone²⁵⁻²⁷, was recently solved. This Gram-positive anaerobic bacterium grows on ethanol and acetate, with the formation of butyrate (and caproate) and H₂ as fermentation products; H₂ bubbles out of the culture. All of the enzymes that are involved in this fermentation have been shown to be cytoplasmic and to use either NAD as an electron acceptor or NADH as an electron donor. Only the [FeFe]–hydrogenase that is involved in H₂ formation is specific for ferredoxin (Fd²⁺ + 2 H⁺ ⇌ Fd²⁻ + H₂; ΔG°' equals 0 kJ per mole). In clostridia, ferredoxin harbours 2 [4Fe–4S] clusters, each of which can be reduced by 1 electron at a redox potential of approximately –420 mV. As all of the enzymes that are involved in ethanol oxidation are NAD specific, in *C. kluyveri*, ferredoxin must be reduced by NADH (*E*°' equals –320 mV), which is an endergonic reaction that requires the input of energy to proceed (NADH + Fd²⁺ ⇌ NAD⁺ + Fd²⁻ + H⁺; ΔG°' equals +20 kJ per mole).

The reaction that is most likely to provide the energy required for ferredoxin reduction is the strongly exergonic reduction of crotonyl-CoA to butyryl-CoA (*E*°' equals –10 mV) with NADH (*E*°' equals –320 mV), which is catalysed by a complex of butyryl-CoA dehydrogenase (Bcd) and two electron-transfer flavoproteins (EtfAB) (NADH + crotonyl-CoA + H⁺ → NAD⁺ + butyryl-CoA; ΔG°' equals –60 kJ per mole). How the endergonic reduction of ferredoxin with NADH is coupled to the exergonic reduction of crotonyl-CoA with NADH has remained a mystery, however, for more than 30 years. Chemiosmotic coupling was excluded by showing that the enzymes which catalyse both reactions are not associated with the cytoplasmic membrane²⁶,²⁷. Herrmann *et al.*²⁵ proposed that the Bcd–EtfAB complex could catalyse the reaction 2 NADH + crotonyl-CoA + Fd²⁺ → 2 NAD⁺ + butyryl-CoA + Fd²⁻ (ΔG°' equals –40 kJ per mole), a hypothesis that was subsequently verified by Li and colleagues²⁶.

FAD is probably involved in the coupling of ferredoxin reduction with NADH to the reduction of crotonyl-CoA with NADH, as each of the three different subunits of the Bcd–EtfAB complex contains an FAD molecule and there are no other prosthetic groups²⁵,²⁶. In some flavoproteins, the flavin nucleotide can be reduced by one electron to a stable semiquinone flavin radical (FADH or FMNH), which can then be reduced by a second electron to the fully reduced flavin nucleotide (FADH₂ or FMNH₂). The first one-electron reduction generally has a higher positive redox potential than the second reduction⁹⁹. For example, in flavodoxin from *Acidaminococcus fermentans*, the first FMN reduction step has a redox potential of approximately –60 mV and the second reduction step has

Figure 3 | The reaction catalysed by the butyryl-CoA dehydrogenase (Bcd)–electron transfer flavoprotein (EtfAB) complex from *Clostridium kluyveri*. The enzyme complex couples the endergonic reduction of ferredoxin (Fd) with NADH to the exergonic reduction of crotonyl-CoA with NADH by flavin-based electron bifurcation (right). The redox potentials are standard potentials at pH 7.0 (*E*°'). FADH, FAD reduced by one electron; FADH₂, FAD reduced by two electrons (according to Li and colleagues²⁶).

REVIEW S

![Diagram](#)

Figure 4 | Proposed scheme for the reduction of CoM-S-S-CoB with H₂ that is catalysed by the hydrogenase (MvhADG)–heterodisulphide reductase (HdrABC) complex in methanogens without cytochromes. The enzyme complex is proposed to couple the endergonic reduction of ferredoxin with H₂ to the exergonic reduction of CoM-S-S-CoB with H₂ by flavin-based electron dismutation that involves the FAD in HdrA. The redox potentials are standard potentials at pH 7.0 (E′). The E′ of ferredoxin was set at –500 mV, which is the E′ of the CO₂/CHO-MFR couple (discussed in the main text). The sequence of HdrB contains ten conserved cysteines that are organized into two so-called CCG domains (CX₃₁₋₃₉CCX₃₅₋₃₆CXXC). The carboxy-terminal CCG domain is involved in an unusual [4Fe–4S] cluster formation and the amino-terminal domain is involved in zinc binding. The zinc in HdrB is ligated by three sulphurs and one histidine nitrogen, as revealed by Zn-K-edge X-ray absorption spectroscopy⁸². The ‘4C’ in HdrA represents a conserved sequence motif that contains four cysteines, and in HdrA from Methanococcus spp., one of the four cysteines is a selenocysteine. Fd, ferredoxin; HS-CoB, coenzyme B; HS-CoM, coenzyme M.

reduction of ferredoxin (–500 mV) (reaction 15, see BOX 1) by flavin-based electron bifurcation (FIG. 4).

This proposal, which for the first time provides a function for the FAD-harbouring subunit HdrA (FIG. 4), is substantiated by the following experimental results. First, cell extracts of *M. thermoautotrophicus* have been shown to slowly catalyse the reduction of CO₂ to methane only upon spiking with methyl-coenzyme M or CoM-S-S-CoB¹⁰¹,¹⁰². This phenomenon is referred to in the literature as the RPG effect. The RPG effect indicates that the first step in methanogenesis from CO₂ and H₂ is somehow coupled to the reduction of CoM-S-S-CoB. Larger particles were removed from the cell extracts by centrifugation at 25,000 g, making the presence of vesicles — and thus chemiosmotic coupling — unlikely. Cell extracts of the cytochrome-containing *M. barkeri* do not show the RPG effect. Second, cell extracts of *M. thermoautotrophicus* have been shown to catalyse the CoM-S-S-CoB-dependent reduction of CO₂ with H₂ to formyl-MFR¹⁰³. As CO₂ reduction to formyl-MFR is ferredoxin dependent, this finding indicates that the reduction of ferredoxin with H₂ is CoM-S-S-CoB dependent. Based on this finding, Rouvière and Wolfe¹⁰² proposed in 1988 that CO₂ reduction to formyl-MFR and CoM-S-S-CoB reduction with H₂ have a cytoplasmic component, probably a ferredoxin, in common. Finally, cell extracts of *M. thermoautotrophicus* have been shown to catalyse the CoM-S-S-CoB-dependent reduction of metronidazole with H₂ (REF. 104). CoM-S-S-CoB could not be substituted by any other disulphide in promoting metronidazole reduction, and metronidazole is known to be spontaneously reduced by ferredoxin. These findings again indicate that the reduction of ferredoxin with H₂ is CoM-S-S-CoB dependent and correspond with the observation that the MvhADG–HdrABC complex catalyses the reduction of CoM-S-S-CoB rather than the reduction of other disulphides⁸³.

Most of these results have been repeated with cell extracts of *M. marburgensis*, but an unambiguous demonstration that the purified MvhADG–HdrABC complex catalyses reaction 15 (BOX 1) has not yet been achieved. The complex does seem to be more labile and more susceptible to uncoupling than the Bcd–EtfAB complex from *C. kluyveri*.

**A proposed metabolic scheme.** If the idea that in methanogens without cytochromes the reduction of heterodisulphide with H₂ is coupled to the reduction of ferredoxin with H₂ (reaction 15, BOX 1; FIG. 4) by flavin-based electron bifurcation is correct, then the metabolic scheme shown in FIG. 2 changes to that shown in FIG. 5.

In FIG. 5, it is assumed that the A₁A₀-ATP synthase is sodium ion translocating rather than proton translocating. This assumption is based on three findings. First, ATP synthesis in *M. marburgensis*, which is driven by valinomycin-mediated potassium ion efflux, is significantly enhanced in the presence of sodium ions¹⁰⁵,¹⁰⁶. Second, at Vmax concentrations of sodium ions, methanogenesis from CO₂ and H₂ and ATP synthesis in cell suspensions of *M. marburgensis* are almost completely insensitive to protonophores¹⁰⁷⁻¹⁰⁹. And third, dicyclohexylcarbodiimide (DCCD) inhibits the ATP synthesis that is coupled to

a redox potential of approximately –430 mV. Therefore, we need only assume that in the Bcd–EtfAB complex the electron flow from NADH through FADH₂ is bifurcated such that the oxidation of FADH₂ to FADH is coupled to the reduction of the clostridial ferredoxin (E′ equals –420 mV) and the oxidation of FADH to FAD is coupled to the reduction of crotonyl-CoA (E′ equals –10 mV)²⁵,²⁶ (FIG. 3). This proposed mechanism is analogous to that of electron bifurcation in the cytochrome bc₁ complex that catalyses the oxidation of reduced ubiquinone with cytochrome c of the respiratory chain¹⁰⁰. Evidently, in addition to ubiquinone-based electron bifurcation, flavin-based electron bifurcation also occurs.

**Evidence for electron bifurcation in methanogens.** Based on the mechanism of ferredoxin and crotonyl-CoA reduction with NADH that is shown in FIG. 3, we propose that that the cytoplasmic MvhADG–HdrABC complex catalyses the reduction of the heterodisulphide of coenzyme M (HS-CoM) and coenzyme B (HS-CoB) (E′ equals –140 mV) with H₂ (E′ equals –414 mV) and couples this exergonic reaction with the

Methanobacteriales  
Methanococcales  
Methanopyrales  
Methanomicrobiales  
(all without cytochromes)  

$ 4 \mathrm{H}_{2} + \mathrm{CO}_{2} \rightarrow \mathrm{CH}_{4} + 2 \mathrm{H}_{2} \mathrm{O} $

$ 2 \mathrm{Na}^{+} \rightarrow $

Eha and/or  
Ehb complex

$\begin{aligned}
& \text { Fd }_{\text {red }}^{2-} \\
& \text { H }_{2} + \text { Fd }_{\text {ox }} \\
& \text { Fd }_{\text {red }}^{2-}
\end{aligned}$

$\begin{aligned}
& \text { CO }_{2} \\
& \text { CHO-MFR }
\end{aligned}$

$\begin{aligned}
& \text { CH }=\mathrm{H}_{4} \text { MPT }^{+} \\
& \text { CH }_{2}=\mathrm{H}_{4} \text { MPT } \\
& \text { CH }_{3}-\mathrm{H}_{4} \text { MPT }
\end{aligned}$

$\begin{aligned}
& \text { HS-CoB } \\
& \text { CoM-S-S-CoB } \\
& \text { MvhADG-- } \\
& \text { HdrABC }
\end{aligned}$

$\begin{aligned}
& \text { CH }_{3}-\text { S-CoM } \\
& \text { MtrA-H }
\end{aligned}$

$\begin{aligned}
& \text { ATPase } \\
& \text { A }_{1} \text { A }_{0}
\end{aligned}$

$\begin{aligned}
& \text { ADP } \\
& \text { ATP }
\end{aligned}$

$\begin{aligned}
& \text { CH }_{4}
\end{aligned}$

$\begin{aligned}
& \text { Fd }_{\text {ox }} \\
& \text { Fd }_{\text {red }}^{2-}
\end{aligned}$

$\begin{aligned}
& \text { H }_{2} \rightarrow \mathrm{F}_{420} \mathrm{H}_{2} \\
& \text { H }_{2} \rightarrow \mathrm{F}_{420} \mathrm{H}_{2}
\end{aligned}$

$\begin{aligned}
& \text { 2 H }^{+} \\
& \text { 1 Na }^{+}
\end{aligned}$

$\begin{aligned}
& \text { 2 Na }^{+} \\
& \text { 4 Na }^{+}
\end{aligned}$

$\begin{aligned}
& -500 \, \mathrm{mV} \\
& -414 \, \mathrm{mV} \\
& -140 \, \mathrm{mV}
\end{aligned}$

---

Figure 5 | The coupling sites that are proposed to be involved in energy conservation in methanogens without cytochromes growing on $\mathrm{CO}_{2}$ and $\mathrm{H}_{2}$. The numbers in bold correspond to the reaction numbers in BOX 1. The first and last steps are coupled by flavin-based electron bifurcation. The ATP gain (moles of ATP per mole of methane) is assumed to be 0.5. The redox potentials are standard potentials at pH 7.0. The $E^{\prime \prime}$ of ferredoxin was set at $-500 \, \mathrm{mV}$, which is the $E^{\prime \prime}$ of the $\mathrm{CO}_{2} / \mathrm{CHO}-\mathrm{MFR}$ couple (discussed in the main text). The reaction that is catalysed by the cytoplasmic MvhADG–HdrABC complex (reaction 15) is delineated by a thicker grey arrow. $C_{1}$ units are highlighted in red. Fd, ferredoxin; $\mathrm{H}_{4} \mathrm{MPT}$, tetrahydromethanopterin; HS-CoB, coenzyme B; HS-CoM, coenzyme M; MFR, methanofuran.

---

together with its $A_{1} A_{0}$-ATP synthase, are probably coupled through the sodium ion motive force, as the $A_{1} A_{0}$-ATP synthase from *P. furiosus* has been shown to be sodium ion translocating ${ }^{89}$. As mentioned earlier, some of the subunits of Eha and Ehb show sequence similarity to subunits of complex I in the respiratory chain of bacteria, and it is therefore of interest that this complex can also be either proton translocating or sodium ion translocating ${ }^{112}$.

How can the function of the Eha or Ehb hydrogenase be explained by the scheme shown by FIG. 5? The reduced ferredoxin that is generated in reaction 9 (BOX 1) is required for autotrophic $\mathrm{CO}_{2}$ fixation — for example, the reduction of $\mathrm{CO}_{2}$ to $\mathrm{CO}\left(E^{\prime \prime}\right.$ equals $-520 \, \mathrm{mV})$, the reduction of acetyl-CoA and $\mathrm{CO}_{2}$ to pyruvate ($E^{\prime \prime}$ equals $-500 \, \mathrm{mV}$) and the reduction of succinyl-CoA and $\mathrm{CO}_{2}$ to 2-oxoglutarate ($E^{\prime \prime}$ equals $-500 \, \mathrm{mV}$) ${ }^{113,114}$. The reduced ferredoxin is also required for $\mathrm{CO}_{2}$ reduction to methane if the coupling of ferredoxin and CoM-S-S-CoB reduction with $\mathrm{H}_{2}$ by the cytoplasmic MvhADG–HdrABC complex is not tight. In the absence of an energy-converting hydrogenase, any uncoupling of ferredoxin and CoM-S-S-CoB reduction with $\mathrm{H}_{2}$ would eventually stop $\mathrm{CO}_{2}$ reduction to methane. Consistent with these functions is the fact that an Eha and/or Ehb hydrogenase complex is found in all methanogens without cytochromes and that the specific activities of Eha and Ehb in cell extracts are much lower than would be predicted if they were directly involved in $\mathrm{CO}_{2}$ reduction to methane ${ }^{95,113}$.

What is the function of the $\mathrm{Na}^{+} / \mathrm{H}^{+}$ antiporter (Nha) in FIG. 5? Inhibitors of $\mathrm{Na}^{+} / \mathrm{H}^{+}$ antiporters and artificial $\mathrm{Na}^{+} / \mathrm{H}^{+}$ antiporters, such as monensin, exert various effects on methanogenesis and ATP synthesis in *M. marburgensis* ${ }^{106,115,116}$ and *M. stadtmanae* ${ }^{67}$, which can be explained, at least in part, by the fact that the antiporter has a function in pH homeostasis.

The scheme in FIG. 5 can also explain why the ATP gain in methanogens without cytochromes does not exceed 0.5 and is thus much lower than the ATP gain of 1.5 that is observed in methanogens with cytochromes (FIG. 2). The finding that the ATP gain in methanogens without cytochromes is frequently lower than 0.5 indicates that coupling of ferredoxin and CoM-S-S-CoB reduction with $\mathrm{H}_{2}$ is not always tight, as would be expected from flavin-based electron bifurcation.

---

methanogenesis from methanol and $\mathrm{H}_{2}$ in *M. stadtmanae* only at low sodium ion concentrations, which indicates that the reaction of the $A_{1} A_{0}$-ATP synthase with DCCD is prevented by sodium ions ${ }^{67,89}$. In fact, from the primary structure of the $A_{0}$ subunits, it has been deduced that the $A_{1} A_{0}$-ATP synthase from all methanogens should be sodium ion translocating ${ }^{88,89}$. However, no sodium ion dependence was observed for the *Methanosarcina mazei* enzyme ${ }^{80}$, which is why in methanogens with cytochromes (FIG. 2), ADP phosphorylation is assumed to be driven by the proton motive force. Notably, the $\mathrm{F}_{1} \mathrm{~F}_{0}$ syntheses of bacteria can also be either sodium ion or proton coupled, with one sodium ion being equivalent to one proton ${ }^{110}$.

Another difference between the schemes in FIG. 2 and FIG. 5 is that in methanogens without cytochromes the energy-converting hydrogenase complex Eha and/or Ehb (reaction 9, see BOX 1) is assumed to be sodium ion translocating. This is indicated by the observation that the oxidation of formaldehyde (which reacts spontaneously with $\mathrm{H}_{4} \mathrm{MPT}$ to produce methylene-$\mathrm{H}_{4} \mathrm{MPT}$) to $\mathrm{CO}_{2}$ and $2 \mathrm{H}_{2}$ (reactions 2–5 and reaction 9 in reverse, see BOX 1) in *M. marburgensis* is sodium ion dependent ${ }^{76}$. This assumption is also supported by the finding that the Eha- and Ehb-type hydrogenase that is present in *Pyrococcus furiosus* ${ }^{111}$,

---

**A test case: M. stadtmanae**

The metabolic scheme in FIG. 5 indicates that during $\mathrm{CO}_{2}$ reduction to methane, methanogens without cytochromes conserve energy only in reaction 7 (BOX 1), which is catalysed by the membrane-associated methyl-$\mathrm{H}_{4}$ MPT-coenzyme M methyltransferase complex (MtrA–H). The sodium ion motive force that is generated in this reaction is subsequently used to drive reverse electron transport (reaction 9, BOX 1), ATP synthesis (reaction 13, BOX 1) and $\mathrm{Na}^{+} / \mathrm{H}^{+}$ antiport (reaction 14, BOX 1). If this scheme is correct, then how is energy conserved in *M. stadtmanae*, a methanogen without cytochromes that can only reduce methanol and not $\mathrm{CO}_{2}$ with $\mathrm{H}_{2}$ to methane

M.stadtmanae  
(without cytochromes)  
$\mathrm{CH}_{3} \mathrm{OH} + \mathrm{H}_{2} \rightarrow \mathrm{CH}_{4} + \mathrm{H}_{2} \mathrm{O}$  

$2 \mathrm{Na}^{+}$  

Ehb  
complex  

$\mathrm{Fd}_{\text {ox }} + \mathrm{H}_{2}$  

$\mathrm{Fd}_{\text {red }}$  
$-500 \mathrm{mV}$  

$\mathrm{CH}_{3} \mathrm{OH}$  

$16$  

$\mathrm{CH}_{3}-\mathrm{S}-\mathrm{CoM}$  

$2 \mathrm{e}^{-}$  
$-414 \mathrm{mV}$  
$-140 \mathrm{mV}$  

$2 \mathrm{H}_{2}$  

$15$  

$\mathrm{HS}-\mathrm{CoB}$  

$8$  

$\mathrm{CoM}-\mathrm{S}-\mathrm{S}-\mathrm{CoB}$  

$\mathrm{MvhADG}-$  
$\mathrm{HdrABC}$  

$\mathrm{CH}_{4}$  

$14$  

$2 \mathrm{H}^{+}$  

$1 \mathrm{Na}^{+}$  

$\mathrm{ADP}$  

$13$  

$\mathrm{A}_{1} \mathrm{~A}_{0}$  
$\mathrm{ATPase}$  

$4 \mathrm{Na}^{+}$  

$\mathrm{ATP}$  

---

Figure 6 | Proposed energy conservation by the Ehb complex in Methanosphaera stadtmanae growing on methanol and $\mathrm{H}_{2}$. The numbers in bold correspond to the reaction numbers in BOX 1. Reactions 9 and 15 are coupled by flavin-based electron bifurcation. The redox potentials are standard potentials at pH 7.0 ($E^{0^{\prime}}$). The $E^{0^{\prime}}$ of ferredoxin was set at $-500 \mathrm{mV}$ (discussed in the main text). The scheme can explain the described effects of dicyclohexylcarbodiimide, protonophores and sodium ionophores at high and low sodium ion concentrations ${ }^{67}$ if the presence of an active electrogenic $\mathrm{Na}^{+} / 2 \mathrm{H}^{+}$ antiporter is taken into account. The reaction that is catalysed by the cytoplasmic MvhADG-HdrABC complex (reaction 15) is delineated by a thicker grey arrow. $\mathrm{C}_{1}$ units are highlighted in red. Fd, ferredoxin; HS-CoB, coenzyme B; HS-CoM, coenzyme M.

---

$(\mathrm{CH}_{3} \mathrm{OH} + \mathrm{H}_{2} \rightarrow \mathrm{CH}_{4} + \mathrm{H}_{2} \mathrm{O} ; \Delta G^{0}$ equals $-112.5 \mathrm{~kJ}$ per mole) and that lacks a functional methyltransferase coupling site ${ }^{37}$? This is explained in the scheme shown in FIG. 6.

The first step in methanol metabolism is the formation of methyl-coenzyme M from methanol and coenzyme M, which is catalysed by the cytoplasmic enzyme complex MtaABC (reaction 16, BOX 1). Methyl-coenzyme M is subsequently reduced to methane by reactions 8 and 15 (BOX 1), which are catalysed by the cytoplasmic enzyme complexes methyl-coenzyme M reductase (MrtABG) and MvhADG-HdrABC, respectively. The reduced ferredoxin that is generated by reaction 15 (BOX 1) is finally reoxidized with protons to yield $\mathrm{H}_{2}$ (reaction 9, BOX 1). This reaction is catalysed by an energy-converting hydrogenase (Ehb) and coupled to the build-up of a sodium ion motive force which, in turn, can drive the synthesis of ATP (reaction 13, BOX 1).

As for $P$. furiosus that is growing on maltose, in $M$. stadtmanae that is growing on $\mathrm{H}_{2}$ and methanol the Ehb complex is proposed to catalyse the formation of $\mathrm{H}_{2}$ (REF. 111) (reaction 9, BOX 1). This allows the reduction of methanol with $\mathrm{H}_{2}$ to methane to be coupled to energy conservation. In this energy metabolism, $\mathrm{H}_{2}$ is both consumed (reaction 15, BOX 1) and formed (reaction 9, see BOX 1) (FIG. 6). This situation is not without precedent; the best known example is the proton motive Q cycle, which is catalysed by the $b c_{1}$ complex of the respiratory chain and in which ubiquinone is both oxidized and reduced ${ }^{25,26}$.

---

Proton motive Q cycle  
A cycle that is catalysed by the $b c_{1}$ complex (complex III) of the respiratory chain and that mediates the oxidation of ubiquinol with cytochrome $c$ and couples this reaction with the electrogenic translocation of four protons in a cyclic process.

---

An exception?  
Rice Cluster I (RC-I), a methanogen with cytochromes that can only grow using $\mathrm{CO}_{2}$ and $\mathrm{H}_{2}$ or formate as energy sources, was recently found in paddy-field sediments. The $\mathrm{H}_{2}$ threshold concentration shown by this archaeon was low (<10 Pa) ${ }^{117}$, indicating a low ATP gain. RC-I therefore behaves like a methanogen without cytochromes.

An explanation for this behaviour comes from an analysis of the genome sequence of this organism ${ }^{118}$, in which putative genes that encode two enzyme complexes that were thought to be characteristic for methanogens without cytochromes — an MvhADG-HdrABC complex and an $F_{420}$-dependent formate dehydrogenase — were found. As discussed above, methanogens with cytochromes are phylogenetically younger than those without cytochromes. The presence of characteristic genes from methanogens without cytochromes in methanogens with cytochromes is therefore not surprising. Interestingly, in the genome of RC-I, not all of the genes that are required for the synthesis of VhoACG (reaction 11, see BOX 1) and HdrDE (reaction 12, see BOX 1) are present, which indicates that this methanogen with cytochromes is dependent on the MvhADG-HdrABC complex (reaction 15, see BOX 1) for methanogenesis.

The results from RC-I therefore support rather than contradict the hypothesis that energy conservation is less effective in methanogens without cytochromes that are growing on $\mathrm{CO}_{2}$ and $\mathrm{H}_{2}$ than in methanogens with cytochromes.

---

### Conclusions

The differences in the physiological properties of methanogens with and without cytochromes are manifold. The most important differences are the higher growth yields and $\mathrm{H}_{2}$ threshold concentrations that are observed in methanogens with cytochromes. These differences have been explained in this Review mainly by differences in the coupling of ferredoxin and CoM-S-S-CoB reduction with $\mathrm{H}_{2}$: chemiosmotic coupling on the one hand and coupling by flavin-based electron bifurcation on the other. Our understanding of the mechanism of coupling by flavin-based electron bifurcation in methanogens without cytochromes is still at the level of a hypothesis; however, it is a hypothesis for which more and more evidence is accumulating. It should be noted that a mechanism cannot be proven but only disproved. This is the basis for planned future experiments.

Genes that putatively encode an MvhADG-HdrABC-like complex (FIG. 4) are also present in the genome of the sulphate-reducing delta-proteobacterium *Desulfovibrio vulgaris* ${ }^{119,120}$ and are expressed during growth on ethanol and sulphate ${ }^{119,120}$. This indicates that flavin-based electron bifurcation involving the flavoprotein HdrA might also have a role in the energy metabolism of non-methanogenic organisms.

Finally, there are other anaerobic microorganisms for which energy-conservation mechanisms are unclear; for example, the coupling site that allows acetogenic bacteria to grow on $\mathrm{CO}_{2}$ with $\mathrm{H}_{2}$ remains to be identified (REFS 121,122) (FIG. 1). When analysing the genome sequences of these bacteria for a possible coupling site, flavin-based electron bifurcation should be kept in mind.

1. Boone, D. R., Whitman, W. B. & Rouvière, P. in Methanogenesis (ed. Ferry, J. G.) 35–80 (Chapman & Hall, New York & London, 1993).
2. Garcia, J. L., Patel, B. K. C. & Ollivier, B. Taxonomic, phylogenetic and ecological diversity of methanogenic archaea. Anaerobe **6**, 205–226 (2000).
3. Ferry, J. G. & Kastead, K. A. in Archaea: Molecular and Cellular Biology (ed. Cavicchioli, R.) 288–314 (ASM, Washington DC, 2007).
4. Thauer, R. K. Biochemistry of methanogenesis: a tribute to Marjory Stephenson. Microbiology **144**, 2377–2406 (1998).
5. Grahame, D. A. & Gencic, S. in Encyclopedia of Microbiology (ed. Lederberg, J.) 188–198 (Academic, New York, 2000).
6. McInerney, M. J. *et al.* The genome of Syntrophus aciditrophicus: life at the thermodynamic limit of microbial growth. Proc. Natl Acad. Sci. USA **104**, 7600–7605 (2007).
7. Schink, B. in Molecular Basis of Symbiosis (ed. Overman, J.) 1–19 (Springer, Berlin, 2006).
8. Thauer, R. K., Jungermann, K. & Decker, K. Energy conservation in chemotrophic anaerobic bacteria. Bacterial Rev. **41**, 100–180 (1977).
9. Conrad, M. E., Templeton, A. S., Daley, P. F. & Alvarez-Cohen, L. Seasonally-induced fluctuations in microbial production and consumption of methane during bioremediation of aged subsurface refinery contamination. Environ. Sci. Technol. **33**, 4061–4068 (1999).
10. Hoehler, T. M., Alperin, M. J., Albert, D. B. & Martens, C. S. Thermodynamic control on hydrogen concentrations in anoxic sediments. Geochim. Cosmochim. Acta **62**, 1745–1756 (1998).
11. Kotsyurbenko, O. R., Glagolev, M. V., Nozhevnikova, A. N. & Conrad, R. Competition between homoacetogenic bacteria and methanogenic archaea for hydrogen at low temperature. FEMS Microbiol. Ecol. **38**, 153–159 (2001).
12. Amend, J. P. & Shock, E. L. Energetics of overall metabolic reactions of thermophilic and hyperthermophilic archaea and bacteria. FEMS Microbiol. Rev. **25**, 175–243 (2001).
13. Zinder, S. H. in Methanogenesis (ed. Ferry, J. G.) 128–206 (Chapman & Hall, New York & London, 1993).
14. Kamra, D. N. Rumen microbial ecosystem. Curr. Sci. **89**, 124–135 (2005).
15. Pester, M. & Brune, A. Hydrogen is the central free intermediate during lignocellulose degradation by termite gut symbionts. ISME J. **1**, 551–565 (2007).
16. Reeburgh, W. S. Oceanic methane biogeochemistry. Chem. Rev. **107**, 486–513 (2007).
17. Raghoebarsing, A. A. *et al.* A microbial consortium couples anaerobic methane oxidation to denitrification. Nature **440**, 918–921 (2006).
18. Hallam, S. J., Girguis, P. R., Preston, C. M., Richardson, P. M. & DeLong, E. F. Identification of methyl coenzyme M reductase A (*mcrA*) genes associated with methane-oxidizing archaea. Appl. Environ. Microbiol. **69**, 5483–5491 (2003).
19. Krüger, M. *et al.* A conspicuous nickel protein in microbial mats that oxidize methane anaerobically. Nature **426**, 878–881 (2003). Describes the purification and characterization of methyl-coenzyme M reductase from microbial mats that catalyses the anaerobic oxidation of methane.
20. Shima, S. & Thauer, R. K. Methyl-coenzyme M reductase and the anaerobic oxidation of methane in methanotrophic archaea. Curr. Opin. Microbiol. **8**, 643–648 (2005).
21. Shima, S. & Thauer, R. K. Anaerobic methane oxidation by archaea: a biochemical approach. Biosci. Ind. **64**, 23–26 (2006).
22. Thauer, R. K. & Shima, S. Methane and microbes. Nature **440**, 878–879 (2006).
23. Thauer, R. K. & Shima, S. in Archaea: Evolution, Physiology and Molecular Biology (eds. Garrett, R. & Klene, H.-P.) 275–283 (Blackwell Publishing, Malden, Massachusetts, 2007).
24. Thauer, R. K. & Shima, S. Methane as fuel for anaerobic microorganisms. Ann. NY Acad. Sci. **1125**, 158–170 (2008).
25. Herrmann, G., Jayamani, E., Mai, G. & Buckel, W. Energy conservation via electron-transferring flavoprotein in anaerobic bacteria. J. Bacteriol. **190**, 784–791 (2008). A mini-review in which the hypothesis of flavin-based electron bifurcation was first formulated.
26. Li, F. *et al.* Coupled ferredoxin and crotonyl coenzyme A (CoA) reduction with NADH catalyzed by the butyryl-
27. CoA dehydrogenase/Etf complex from Clostridium kluyveri. J. Bacteriol. **190**, 843–850 (2008). Showed that the cytoplasmic Bcd-EtfAB complex from *C. kluyveri* couples the endergonic reduction of ferredoxin with NADH to the exergonic reduction of crotonyl-CoA with NADH.
28. Seedorf, H. *et al.* The genome of Clostridium kluyveri, a strict anaerobe with unique metabolic features. Proc. Natl Acad. Sci. USA **105**, 2128–2133 (2008). Identified the genes which encode the proteins that are involved in ethanol acetate fermentation, including those that catalyse H₂ formation from NADH.
29. Jussowie, A. & Gottschalk, G. Further studies on the distribution of cytochromes in methanogenic bacteria. FEMS Microbiol. Lett. **37**, 15–18 (1986).
30. Kühn, W. *et al.* Distribution of cytochromes in methanogenic bacteria. FEMS Microbiol. Lett. **20**, 407–410 (1983).
31. Kühn, W., Fiebig, K., Walther, R. & Gottschalk, G. Presence of a cytochrome b559 in Methanosarcina barkeri. FEBS Lett. **105**, 271–274 (1979). First report of a cytochrome in a methanogenic archaeon.
32. Kühn, W. & Gottschalk, G. Characterization of the cytochromes occurring in Methanosarcina species. Eur. J. Biochem. **135**, 89–94 (1983).
33. Abken, H. J. *et al.* Isolation and characterization of methanophenazine and function of phenazines in membrane-bound electron transport of Methanosarcina mazei Gö1. J. Bacteriol. **180**, 2027–2032 (1998). Reported the discovery of methanophenazine in methanogens with cytochromes.
34. Beifuss, U., Tietze, M., Bäumer, S. & Deppenmeier, U. Methanophenazine: structure, total synthesis, and function of a new cofactor from methanogenic archaea. Angewandte Chemie-International Edition **39**, 2470–2472 (2000).
35. Tietze, M. *et al.* Redox potentials of methanophenazine and CoB-S-S-CoM, factors involved in electron transport in methanogenic archaea. Chembiochem **4**, 333–335 (2003).
36. Kendall, M. M. & Boone, D. R. in The Prokaryotes (eds Dworkin, M., Falkow, S., Rosenberg, E., Schleifer, K.-H. & Stackebrandt, E.) 244–256 (Springer, New York, 2006).
37. Bonin, A. S. & Boone, D. R. in The Prokaryotes (eds Dworkin, M., Falkow, S., Rosenberg, E., Schleifer, K.-H. & Stackebrandt, E.) 231–243 (Springer, New York, 2006).
38. Fricke, W. F. *et al.* The genome sequence of Methanosphaera stadtmanae reveals why this human intestinal archaeon is restricted to methanol and H₂ for methane formation and ATP synthesis. J. Bacteriol. **188**, 642–658 (2006).
39. Hallam, S. J., Girguis, P. R., Preston, C. M., Richardson, P. M. & DeLong, E. F. Identification of methyl coenzyme M reductase A (*mcrA*) genes associated with methane-oxidizing archaea. Appl. Environ. Microbiol. **69**, 5483–5491 (2003).
40. Weimer, P. J. & Zeikus, J. G. One carbon metabolism in methanogenic bacteria — cellular characterization and growth of Methanosarcina barkeri. Arch. Microbiol. **119**, 49–57 (1978).
41. Morii, H., Koga, Y. & Nagai, S. Energetic analysis of the growth of Methanobrevibacter arboriphilus A2 in hydrogen limited continuous cultures. Biotechnol. Bioeng. **29**, 310–315 (1987).
42. Karadagli, F. & Rittmann, B. E. Kinetic characterization of Methanobacterium bryantii M.o.H. Environ. Sci. Technol. **39**, 4900–4905 (2005).
43. de Poorter, L. M., Geerts, W. G., Theuvenet, A. P. & Keltjens, J. T. Bioenergetics of the formyl-methanofuran dehydrogenase and heterodisulfide reductase reactions in Methanothermobacter thermoautotrophicus. Eur. J. Biochem. **270**, 66–75 (2003).
44. Schönhheit, P., Moll, J. & Thauer, R. K. Growth parameters (Kₛ, μₘₐₓ, Yₛ) of Methanobacterium thermoautotrophicum. Arch. Microbiol. **127**, 59–65 (1980).
45. Wennerhold, J. Heterodisulfid-Reduktasen des Methanothermobacter Typs in Methanosarcina barkeri und Untersuchungen zur Rolle des H₂: Heterodisulfid-Oxidoreduktase-Komplex bei der Energiekonservierung in Methanothermobacter marburgensis. Thesis, Philipps Univ., Germany (2004).
46. Heijnen, J. J. & van Dijken, J. P. In search of a thermodynamic description of biomass yields for the chemotrophic growth of microorganisms. Biotechnol. Bioeng. **39**, 833–858 (1992).
47. Stouthamer, A. H. In Search of a Correlation Between Theoretical and Experimental Growth Yields (ed. Quayle, J. R.) (University Park Press, Baltimore, 1979).
48. Pirt, S. J. The maintenance energy of bacteria in growing cultures. Proc. R. Soc. Lond. Ser. B **163**, 224–231 (1965).
49. Conrad, R. Contribution of hydrogen to methane production and control of hydrogen concentrations in methanogenic soils and sediments. FEMS Microbiol. Ecol. **28**, 193–202 (1999).
50. Conrad, R. & Wetter, B. Influence of temperature on energetics of hydrogen metabolism in homoacetogenic, methanogenic and other anaerobic bacteria. Arch. Microbiol. **155**, 94–98 (1990).
51. Cord-Ruwisch, R., Seitz, H. D. & Conrad, R. The capacity of hydrogenotrophic anaerobic bacteria to compete for traces of hydrogen depends on the redox potential of the terminal electron acceptor. Arch. Microbiol. **149**, 350–357 (1988).
52. Hoehler, T. M., Bebout, B. M. & Des Marais, D. J. The role of microbial mats in the production of reduced gases on the early earth. Nature **412**, 324–327 (2001).
53. Lovley, D. R. Minimum threshold for hydrogen metabolism in methanogenic bacteria. Appl. Environ. Microbiol. **49**, 1530–1531 (1985).
54. Seitz, H. J., Schink, B., Pfennig, N. & Conrad, R. Energetics of syntrophic ethanol oxidation in defined chemostat cocultures. 1. Energy requirement for H₂ production and H₂ oxidation. Arch. Microbiol. **155**, 82–88 (1990).
55. Karadagli, F. & Rittmann, B. E. Thermodynamic and kinetic analysis of the H₂ threshold for Methanobacterium bryantii M.o.H. Biodegradation **18**, 439–452 (2007).
56. Karadagli, F. & Rittmann, B. E. A mathematical model for the kinetics of Methanobacterium bryantii M.o.H. considering hydrogen thresholds. Biodegradation **18**, 453–464 (2007).
57. Hendrickson, E. L., Haydock, A. K., Moore, B. C., Whitman, W. B. & Leigh, J. A. Functionally distinct genes regulated by hydrogen limitation and growth rate in methanogenic archaea. Proc. Natl Acad. Sci. USA **104**, 8930–8934 (2007).
58. Galagan, J. E. *et al.* The genome of *M. acetivorans* reveals extensive metabolic and physiological diversity. Genome Res. **12**, 532–542 (2002).
59. Guss, A. M., Mukhopadhyay, B., Zhang, J. K. & Metcalf, W. W. Genetic analysis of *mch* mutants in two Methanosarcina species demonstrates multiple roles for the methanopterin-dependent C₁ oxidation/ reduction pathway and differences in H₂ metabolism between closely related species. Mol. Microbiol. **55**, 1671–1680 (2005).
60. Li, L. Y. *et al.* Quantitative proteomic and microarray analysis of the archaeon Methanosarcina acetivorans grown with acetate versus methanol. J. Proteome Res. **6**, 759–771 (2007).
61. Wood, G. E., Haydock, A. K. & Leigh, J. A. Function and regulation of the formate dehydrogenase genes of the methanogenic archaeon Methanococcus maripaludis. J. Bacteriol. **185**, 2548–2554 (2003).
62. Finke, N., Hoehler, T. M. & Jørgensen, B. B. Hydrogen ‘leakage’ during methanogenesis from methanol and methylamine: implications for anaerobic carbon degradation pathways in aquatic sediments. Environ. Microbiol. **9**, 1060–1071 (2007).
63. Müller, V., Blaut, M. & Gottschalk, G. Utilization of methanol plus hydrogen by Methanosarcina barkeri for methanogenesis and growth. Appl. Environ. Microbiol. **52**, 269–274 (1986).
64. Perski, H. J., Moll, J. & Thauer, R. K. Sodium dependence of growth and methane formation in Methanobacterium thermoautotrophicum. Arch. Microbiol. **130**, 319–321 (1981).
65. Perski, H. J., Schönheit, P. & Thauer, R. K. Sodium dependence of methane formation in methanogenic bacteria. FEBS Lett. **143**, 323–326 (1982).
66. Gottschalk, G. & Thauer, R. K. The Na⁺-translocating methyltransferase complex from methanogenic archaea. Biochim. Biophys. Acta **1505**, 28–36 (2001).
67. Blaut, M. & Gottschalk, G. Coupling of ATP synthesis and methane formation from methanol and molecular hydrogen in Methanosarcina barkeri. Eur. J. Biochem. **141**, 217–222 (1984).

67. Sparling, R., Blaut, M. & Gottschalk, G. Bioenergetic studies of Methanosphaera stadtmanae, an obligate H₂-methanol utilizing methanogen. Can. J. Microbiol. 39, 742–748 (1993).

68. Becher, B., Müller, V. & Gottschalk, G. N⁵-methyl-tetrahydromethanopterin-coenzyme M methyltransferase of Methanosarcina strain Gö1 is an Na⁺-translocating membrane protein. J. Bacteriol. 174, 7656–7660 (1992).

69. Meuer, J., Kuettner, H. C., Zhang, J. K., Hedderich, R. & Metcalf, W. W. Genetic analysis of the archaeon Methanosarcina barkeri Fusaro reveals a central role for Ech hydrogenase and ferredoxin in methanogenesis and carbon fixation. Proc. Natl Acad. Sci. USA 99, 5632–5637 (2002). Genetic analysis of the energy metabolism of a methanogen with cytochromes growing on H₂ and CO₂ that showed the involvement of the energy-converting hydrogenase Ech in CO₂ reduction.

70. Stojanovic, A. & Hedderich, R. CO₂ reduction to the level of formylmethanofuran in Methanosarcina barkeri is non-energy driven when CO is the electron donor. FEMS Microbiol. Lett. 235, 163–167 (2004).

71. Künkel, A., Vorholt, J. A., Thauer, R. K. & Hedderich, R. An Escherichia coli hydrogenase-3-type hydrogenase in methanogenic archaea. Eur. J. Biochem. 252, 467–476 (1998).

72. Meuer, J., Bartoschek, S., Koch, J., Künkel, A. & Hedderich, R. Purification and catalytic properties of Ech hydrogenase from Methanosarcina barkeri. Eur. J. Biochem. 265, 325–335 (1999).

73. Bott, M. & Thauer, R. K. Proton-motive-force-driven formation of CO from CO₂ and H₂ in methanogenic bacteria. Eur. J. Biochem. 168, 407–412 (1987).

74. Bott, M. & Thauer, R. K. Proton translocation coupled to the oxidation of carbon monoxide to CO₂ and H₂ in Methanosarcina barkeri. Eur. J. Biochem. 179, 469–472 (1989).

75. Peinemann, S., Muller, V., Blaut, M. & Gottschalk, G. Bioenergetics of methanogenesis from acetate by Methanosarcina barkeri. J. Bacteriol. 170, 1369–1372 (1988).

76. Kaesler, B. & Schönheit, P. The role of sodium ions in methanogenesis. Formaldehyde oxidation to CO₂ and 2H₂ in methanogenic bacteria is coupled with primary electrogenic Na⁺ translocation at a stoichiometry of 2–3 Na⁺/CO₂. Eur. J. Biochem. 184, 223–232 (1989).

77. Kaesler, B. & Schönheit, P. The sodium cycle in methanogenesis. CO₂ reduction to the formaldehyde level in methanogenic bacteria is driven by a primary electrochemical potential of Na⁺ generated by formaldehyde reduction to CH₄. Eur. J. Biochem. 186, 309–316 (1989).

78. Ide, T., Bäumer, S. & Deppenmeier, U. Energy conservation by the H₂:heterodisulfide oxidoreductase from Methanosarcina mazei Gö1: identification of two proton-translocating segments. J. Bacteriol. 181, 4076–4080 (1999).

79. Murakami, E., Deppenmeier, U. & Ragsdale, S. W. Characterization of the intramolecular electron transfer pathway from 2-hydroxyphenazine to the heterodisulfide reductase from Methanosarcina thermophila. J. Biol. Chem. 276, 2432–2439 (2001).

80. Pisa, K. Y., Weidner, C., Maischak, H., Kavermann, H. & Müller, V. The coupling ion in the methanoarchaeal ATP synthases: H⁺ vs. Na⁺ in the A₁A₀ ATP synthase from the archaeon Methanosarcina mazei Gö1. FEMS Microbiol. Lett. 277, 56–63 (2007).

81. Deppenmeier, U. Different structure and expression of the operons encoding the membrane-bound hydrogenases from Methanosarcina mazei Gö1. Arch. Microbiol. 164, 370–376 (1995).

82. Hamann, N. *et al.* Cysteine-rich CCG domain contains a novel [4Fe–4S]-cluster binding motif as deduced from studies with subunit B of heterodisulfide reductase from Methanothermobacter marburgensis. Biochemistry 46, 12875–12885 (2007).

83. Hedderich, R., Hamann, N. & Bennati, M. Heterodisulfide reductase from methanogenic archaea: a new catalytic role for an iron–sulfur cluster. Biol. Chem. 386, 961–970 (2005).

84. Heiden, S., Hedderich, R., Setzke, E. & Thauer, R. K. Purification of a cytochrome b containing H₂: heterodisulfide oxidoreductase complex from membranes of Methanosarcina barkeri. Eur. J. Biochem. 213, 529–535 (1993).

85. Heiden, S., Hedderich, R., Setzke, E. & Thauer, R. K. Purification of a two-subunit cytochrome b-containing heterodisulfide reductase from methanol grown Methanosarcina barkeri. Eur. J. Biochem. 221, 855–861 (1994).

86. Peinemann, S., Hedderich, R., Blaut, M., Thauer, R. K. & Gottschalk, G. ATP synthesis coupled to electron transfer from H₂ to the heterodisulfide of 2-mercaptoethanesulfonate and 7-mercaptoheptanoylthreonine phosphate in vesicle preparations of the methanogenic bacterium strain Gö1. FEBS Lett. 263, 57–60 (1990).

87. Lewalter, K. & Müller, V. Bioenergetics of archaea: ancient energy conserving mechanisms developed in the early history of life. Biochim. Biophys. Acta 1757, 437–445 (2006).

88. Müller, V. *et al.* Bioenergetics of archaea: ATP synthesis under harsh environmental conditions. J. Mol. Microbiol. Biotechnol. 10, 167–180 (2005).

89. Pisa, K. Y., Huber, H., Thomm, M. & Müller, V. A sodium ion-dependent A₁A₀ ATP synthase from the hyperthermophilic archaeon Pyrococcus furiosus. FEBS J. 274, 3928–3938 (2007).

90. Vinothkumar, K. R., Smits, S. H. J. & Kuhlbrandt, W. pH-induced structural change in a sodium/proton antiporter from Methanococcus jannaschii. EMBO J. 24, 2720–2729 (2005).

91. Taglicht, D., Padan, E. & Schuldiner, S. Proton sodium stoichiometry of NhaA, an electrogenic antiporter from Escherichia coli. J. Biol. Chem. 268, 5382–5387 (1993).

92. Padan, E. & Schuldiner, S. Molecular physiology of the Na⁺/H⁺ antiporter in Escherichia coli. J. Exp. Biol. 196, 443–456 (1994).

93. Ferry, J. G. Enzymology of one-carbon metabolism in methanogenic pathways. FEMS Microbiol. Rev. 23, 13–38 (1999).

94. Bäumer, S. *et al.* The F₄₂₀H₂ dehydrogenase from Methanosarcina mazei is a redox-driven proton pump closely related to NADH dehydrogenases. J. Biol. Chem. 275, 17968–17973 (2000).

95. Tersteegen, A. & Hedderich, R. Methanobacterium thermoautotrophicum encodes two multisubunit membrane-bound [NiFe] hydrogenases. Transcription of the operons and sequence analysis of the deduced proteins. Eur. J. Biochem. 264, 930–943 (1999).

96. Hedderich, R. Energy-converting [NiFe] hydrogenases from archaea and extremophiles: ancestors of complex I. J. Bioenerg. Biomembr. 36, 65–75 (2004).

97. Setzke, E., Hedderich, R., Heiden, S. & Thauer, R. K. H₂:heterodisulfide oxidoreductase complex from Methanobacterium thermoautotrophicum: composition and properties. Eur. J. Biochem. 220, 139–148 (1994). Found that MvhADG and HdrABC from a methanogen without cytochromes form a complex that catalyses the reduction of CoM-S-S-CoB with H₂.

98. Stojanovic, A., Mander, G. J., Duin, E. C. & Hedderich, R. Physiological role of the F₄₂₀-non-reducing hydrogenase (Mvh) from Methanothermobacter marburgensis. Arch. Microbiol. 180, 194–203 (2003).

99. Alagaratnam, S. *et al.* A crystallographic study of Cys69Ala flavodoxin II from Azotobacter vinelandii: structural determinants of redox potential. Protein Sci. 14, 2284–2295 (2005).

100. Xia, D., Esser, L., Yu, L. & Yu, C. A. Structural basis for the mechanism of electron bifurcation at the quinol oxidation site of the cytochrome bc₁ complex. Photosyn. Res. 92, 17–34 (2007). Ubiquinone-based electron bifurcation in the bc₁ complex was used as a model for flavin-based electron bifurcation.

101. Gunsalus, R. P. & Wolfe, R. S. Stimulation of CO₂ reduction to methane by methyl-coenzyme M in extracts of Methanobacterium. Biochem. Biophys. Res. Commun. 76, 790–795 (1977).

102. Rouvière, P. E. & Wolfe, R. S. Novel biochemistry of methanogenesis. J. Biol. Chem. 263, 7913–7916 (1988). Mini-review that contained a figure showing the hypothetical chemical coupling of the first and last steps in methanogenesis from H₂ and CO₂ in methanogens without cytochromes.

103. Bobik, T. A. & Wolfe, R. S. Physiological importance of the heterodisulfide of coenzyme M and 7-mercaptoheptanoylthreonine phosphate in the reduction of carbon dioxide to methane in Methanobacterium. Proc. Natl Acad. Sci. USA 85, 60–63 (1988). Provided evidence that the ferredoxin-dependent reduction of CO₂ with H₂ to formyl-methanofuran in cell extracts of a methanogen without cytochromes is CoM-S-S-CoB dependent, which is an indication that ferredoxin reduction with H₂ is also CoM-S-S-CoB dependent.

104. Bobik, T. A. & Wolfe, R. S. Activation of formylmethanofuran synthesis in cell extracts of Methanobacterium thermoautotrophicum. J. Bacterial. 171, 1423–1427 (1989). Reported that the reduction of metronidazole with H₂ in cell extracts of a methanogen without cytochromes is CoM-S-S-CoB dependent, which is an indication that CoM-S-S-CoB reduction with H₂ is coupled with ferredoxin reduction with H₂.

105. Schönheit, P. & Perski, H. J. ATP synthesis driven by a potassium diffusion potential in Methanobacterium thermoautotrophicum is stimulated by sodium. FEMS Microbiol. Lett. 20, 263–267 (1983).

106. Schönheit, P. & Beimborn, D. B. Presence of a Na⁺/H⁺ antiporter in Methanobacterium thermoautotrophicum and its role in Na⁺ dependent methanogenesis. Arch. Microbiol. 142, 354–361 (1985).

107. Kaesler, B. & Schönheit, P. Methanogenesis and ATP synthesis in methanogenic bacteria at low electrochemical proton potentials — an explanation for the apparent uncoupler insensitivity of ATP synthesis. Eur. J. Biochem. 174, 189–197 (1988).

108. Mountfort, D. O., Mörschel, E., Beimborn, D. B. & Schönheit, P. Methanogenesis and ATP synthesis in a protoplast system of Methanobacterium thermoautotrophicum. J. Bacteriol. 168, 892–900 (1986).

109. Schönheit, P. & Beimborn, D. B. ATP synthesis in Methanobacterium thermoautotrophicum coupled to CH₄ formation from H₂ and CO₂ in the apparent absence of an electrochemical proton potential across the cytoplasmic membrane. Eur. J. Biochem. 148, 545–550 (1985).

110. Dimroth, P. & Cook, G. M. Bacterial Na⁺- or H⁺-coupled ATP syntheses operating at low electrochemical potential. Adv. Microb. Physiol. 49, 175–218 (2004).

111. Sapra, R., Bagramyan, K. & Adams, M. W. A simple energy-conserving system: proton reduction coupled to proton translocation. Proc. Natl Acad. Sci. USA 100, 7545–7550 (2003).

112. Vgenopoulou, I., Gemperli, A. C. & Steuber, J. Specific modification of a Na⁺ binding site in NADH:quinone oxidoreductase from Klebsiella pneumoniae with dicyclohexylcarbodiimide. J. Bacteriol. 188, 3264–3272 (2006).

113. Porat, I. *et al.* Disruption of the operon encoding Ehb hydrogenase limits anabolic CO₂ assimilation in the archaeon Methanococcus maripaludis. J. Bacteriol. 188, 1373–1380 (2006).

114. Thauer, R. K. A fifth pathway of carbon fixation. Science 318, 1732–1733 (2007).

115. Schönheit, P. & Beimborn, D. B. Monensin and gramicidin stimulate CH₄ formation from H₂ and CO₂ in Methanobacterium thermoautotrophicum at low external Na⁺ concentration. Arch. Microbiol. 146, 181–185 (1986).

116. Surin, S. *et al.* Isolation and characterization of an amiloride-resistant mutant of Methanothermobacter thermoautotrophicus possessing a defective Na⁺/H⁺ antiport. FEMS Microbiol. Lett. 269, 301–308 (2007).

117. Sakai, S. *et al.* Isolation of key methanogens for global methane emission from rice paddy fields: a novel isolate affiliated with the clone cluster rice cluster I. Appl. Environ. Microbiol. 73, 4326–4331 (2007).

118. Erkel, C., Kube, M., Reinhardt, R. & Liesack, W. Genome of rice cluster I archaea — the key methane producers in the rice rhizosphere. Science 313, 370–372 (2006).

119. Haveman, S. A. *et al.* Gene expression analysis of energy metabolism mutants of Desulfovibrio vulgaris Hildenborough indicates an important role for alcohol dehydrogenase. J. Bacteriol. 185, 4345–4353 (2003).

120. Zhang, W. *et al.* A proteomic view of Desulfovibrio vulgaris metabolism as determined by liquid chromatography coupled with tandem mass spectrometry. Proteomics 6, 4286–4299 (2006).

121. Imkamp, F., Biegel, E., Jayamani, E., Buckel, W. & Müller, V. Dissection of the caffeate respiratory chain in the acetogen Acetobacterium woodii: identification of an Rnf-type NADH dehydrogenase as a potential coupling site. J. Bacteriol. 189, 8145–8153 (2007).

122. Müller, V. Energy conservation in acetogenic bacteria. Appl. Environ. Microbiol. 69, 6345–6353 (2003).

123. Krüger, M., Treude, T., Wolters, H., Nauhaus, K. & Boetius, A. Microbial methane turnover in different marine habitats. *Palaeogeogr. Palaeoclimatol. Palaeoecol.* **227**, 6–17 (2005).

124. Lee, S. Y. & Holder, G. D. Methane hydrates potential as a future energy source. *Fuel Process. Technol.* **71**, 181–186 (2001).

125. Lelieveld, J., Crutzen, P. J. & Dentener, F. J. Changing concentration, lifetime and climate forcing of atmospheric methane. *Tellus B Chem. Phys. Meteorol.* **50**, 128–150 (1998).

**Acknowledgements**

This work was supported by the Max Planck Society, the Deutsche Forschungsgemeinschaft and the Fonds der Chemischen Industrie.

---

**DATABASES**

Entrez Genome Project: [http://wwwwlm.nih.gov/entrez/query.fcgi?db=genomepr](http://wwwwlm.nih.gov/entrez/query.fcgi?db=genomepr)

*Clostridium kluyveri* | *Desulfovibrio vulgaris* | *Escherichia coli* | *Methanocaldococcus jannaschii* | *Methanococcus maripaludis* | *Methanopyrus kandleri* | *Methanosarcina acetivorans* | *Methanosarcina barkeri* | *Methanosarcina mazei* | *Methanosphaera stadtmanae* | *Methanothermobacter thermotautotrophicus* | *Pyrococcus furiosus*

**FURTHER INFORMATION**

Rudolf K. Thauer’s homepage: [http://www.mpi-marburg.mpg.de/thauer/](http://www.mpi-marburg.mpg.de/thauer/)

Genomes OnLine Database: [http://www.genomesonline.org](http://www.genomesonline.org)

ALL LINKS ARE ACTIVE IN THE ONLINE PDF

---

NATURE REVIEWS | MICROBIOLOGY

© 2008 Macmillan Publishers Limited. All rights reserved.
