
ALCOHOLISM: CLINICAL AND EXPERIMENTAL RESEARCH

Emerging Role of Epigenetics in the Actions of Alcohol

Shivendra D. Shukla, Jose Velazquez, Samuel W. French, Shelly C. Lu, Maharaj K. Ticku†, and Samir Zakhari

This review deals with the recent developments on the epigenetic effects of ethanol. A large body of data have come from studies in liver and in neuronal systems and involve post-translational modifications in histones and methylations in DNA. Ethanol causes site selective acetylation, methylation, and phosphorylation in histone. With respect to methylations the methyl group donating system involving S-adenosyl methionine appears to play a central role. There is contrasting effect of acetylation versus methylation on the same site of histone, as it relates to the transcriptional activation. Epigenetic memory also appears to correlate with liver pathology and Mallory body formation. Experimental evidence supports transcriptional regulation of genes in the CNS by DNA methylations. These studies are contributing towards a better understanding of a novel epigenetic regulation of gene expression in the context of alcohol. The critical steps and the enzymes (e.g., histone acetyltransferase, histone deacetylase, DNA methyltransferase) responsible for the epigenetic modifications are prime targets for intense investigation. The emerging data are also beginning to offer novel insight towards defining the molecular actions of ethanol and may contribute to potential therapeutic targets at the nucleosomal level. These epigenetic studies have opened up a new avenue of investigation in the alcohol field.

Key Words: Acetylation, DNA, Epigenetics, Ethanol, Histone, Methylation.

THE PAST FEW years have seen a surge in the epigenetic research. Epigenetics refers to changes in gene expression that do not involve DNA coding sequence modifications. The concept of epigenetics started in 1940s (Waddington, 1942). Thus, identical DNA sequences can result in different gene expression profiles due to epigenetics. Epigenetic mechanisms principally include DNA methylations and a variety of histone modifications. These modifications open up the DNA template and render it accessible for transcription (Fig. 1). There are also nonprotein coding RNA transcripts (small RNA) that also contribute to the maintenance of the epigenome. The interaction of these processes determines the epigenetic status of the cells. Such modifications have been linked to a number of diseases including cancer, autoimmune disease and age-related and neurological disorders (Moss and Wallrath, 2007; Rodenhiser and Mann, 2006). Paradoxically, epigenetic marks can be transitory (short lived) and also can be transmitted transgenerational (Anway et al., 2005). The latter implies a heritable change. The mechanism by which an epigenetic signature becomes heritable is not understood. Epigenetic alterations can be time and tissue/organ specific and hence can be quite diverse in an organism or individual. Environmental factors, toxic agents, and drugs can induce modifications in DNA and histones. Ethanol is emerging as one of the agents that cause these modifications (Fig. 2). In this review we provide a brief introduction to the topic of epigenetics, followed by a critical update on ethanol-induced epigenetic changes.

From the Department of Medical Pharmacology & Physiology, University of Missouri Columbia (SDS), Missouri; Division of Metabolism & Health Effects, NIAAA (JV, SZ), Bethesda, Maryland; Department of Pathology, Harbor-UCLA Medical Center (SWF), Torrance, California; Keck School of Medicine USC (SCL), Los Angeles, California; and Department of Pharmacology, UTHSC (MKT), San Antonio, Texas.

Received for publication November 21, 2007; accepted May 8, 2008.
Reprint requests: Shivendra D. Shukla, PhD, Department of Medical Pharmacology & Physiology, University of Missouri School of Medicine, Columbia, MO 65212; Fax: 573-884-4558; E-mail: shuklasd@missouri.edu
† This review is dedicated to the memory Dr. Maharaj Ticku, whose sudden passing shocked us all.
Copyright © 2008 by the Research Society on Alcoholism.
DOI:10.1111/j.1530-0277.2008.00729.x

Alcohol Clin Exp Res, Vol 32, No 9, 2008: pp 1525–1534

INTRODUCTION TO EPIGENETICS

Chromosomes are composed of chromatin structures. Chromatin is made up of nucleosomes which consist of DNA strand wrapped around bead like histone octamers (Fig. 1). These high ordered beaded structures provide about 200,000-fold DNA compaction. Heterochromatins are, in general, silenced (condensed) whereas euchromatin are active (open) portions of the chromatin in terms of gene expression activity. There are 4 core histones (H2A, H2B, H3, and H4) and a linker histone H1. Furthermore, there are histone variants of different types e.g., H3.1, H3.2 & H3.3; H2A1-6, H2A.7. The histones are arranged in an octamer structure in the nucleosome that is composed of H3-H4 tetramers with H2A-H2B dimer surrounding them. The difference in variants H3.1 and

Chromosome
Nucleosome
Recruitment of activator and repressor complexes
Epigenetic Modification
Consequence

Histone
DNA

Co-activator
Repressor
HMT
HDAC
CO-Activator
H2A H2B
H4 H3
HAT
TF
DNMT
MBD
Activator

Post-translation
Histone modification
(Acetylation, Methylation, etc.)
Altered
Transcription
DNA Methylation

Fig. 1. A general overview of epigenetic modification in relation to transcription. This simplified figure depicts the DNA strands wrapped around the “bead structured” histone octamers in the nucleosomes. The enlarged view diagrammatically shows the recruitment and the formation of complex among different components involving enzymes, activator, repressors and transcription factors, etc. This results in modifications in the histone at N-terminal or in DNA leading to altered transcription. DNMT, DNA methyltransferase; HAT, histone acetyl transferase; HDAC, histone deacetylase; HMT, histone methyl transferase; MBD, methyl binding domain protein; TF, transcription factor.

Ethanol
(and or its metabolites)

MAP Kinase signaling

Histone Modifications
Acetylation — HAT, HDAC
Methylation — HMT, HDM
Phosphorylation — Kinases and phosphatases

DNA Modifications
DNDM — Hypomethylation
SAMe
DNMT — Hypermethylation

lysine are Lys4, Lys9, Lys27, Lys36, Lys79 on H3, and Lys 20 on H4. Lysine on histone can be mon-, di-, or tri-methylated. The trimethylation of lysine has been considered to be involved in long-term epigenetic memory. Methylations of histone lysines H3 K9, H3 K27, H4 K20 show relationship to gene silencing whereas, H3 K4, K3-K36, H3-K 79 methylation correlate with activated genes (Cheung and Lau, 2005; Jenuwein, 2006; Lohrum et al., 2007). Histone methylations are mediated by histone methyltransferases where the methyl group is donated by S-adenosyl-methionine (SAMe). Influence of SAMe on ethanol-induced epigenetic changes have recently been investigated as discussed in the sections below.

It is apparent that in some cases the same lysine residue e.g., Lys 4 or Lys 9 are susceptible to both acetylation and methylation. This will create a set of complex spectrum of different modifications. It has been proposed that these types of changes represent “histone code” for various cellular processes (Jenuwein and Allis, 2001; Strahl and Allis, 2000). This complexity increases when one considers the modifications in the variants in each subtype of histone e.g., H3.1, H3.2, and H3.3. This added complexity is the basis for the “histone barcode hypothesis” in which distinct post-translational signatures may create different chromosomal domains, which in turn, influence epigenetic states during differentiations and development (Hake and Allis, 2006). Whether such codes exist in an organized and discreet pattern for specific cellular responses or conditions, have yet to be experimentally proven.

Histones are acetylated by histone acetyltransferases (HATs) (Roth et al., 2001; Shahbazian and Grunstein, 2007) and deacetylated by histone deacetylases (HDACs) (Smith, 2007; Yang, 2007). There are 5 families of HATs (GNATs, MYST, p300/CBP Transcription factor HATs, and nuclear hormone-related HATs) and 4 classes of HDACs (class I to IV). Each family or class has additional members. Examples of the common HATs include GCN5 (general control non-derepressible 5) and PCAF (p300-CBP associated factor). CBP is c-AMP-response element binding (CREB) protein. The Class III HDACs include Sir 2 (silent information

Transcriptional regulation of
gene expression

Change in protein expression

Altered cellular functions

Fig. 2. Pathways and enzymes involved in the ethanol-induced epigenetic changes and the role of SAMe. The diagram shows the various types of modifications in histone and DNA and enzymes responsible for these modifications leading to the expression of genes. In the case of methylations, ethanol also effects SAMe, which plays a key role in both histone and DNA methylations. DNDM, DNA demethylases; DNMT, DNA methyltransferase; HAT, histone acetyl transferase; HDAC, histone deacetylase; HDM, histone demethylase; HMT, histone methyl transferase; MAP kinases, mitochondria activated protein kinases; SAMe, s-adenosyl methionine.

H3.2 is due to one amino acid. Recent work demonstrates a potential requirement of the H3.3 variant for the persistence of some of the active gene states (Ng and Gourdon, 2008).

Histones can be modified by, at least, 6 different pathways i.e., acetylation, methylation, phosphorylation, ubiquitinylation, ADP-ribosylation, and sumoylation. N-terminal tails of histone proteins can be modified by acetylation at lysine; methylation at lysine & arginine; phosphorylation at serine & threonine; and ubiquitinylation and sumoylation at lysine. The major sites of acetylation in histone H3 are Lys 4, Lys 9, Lys14 & Lys28. The major sites of histone methylation at

regulators 2) homologs. These enzymes exhibit some degree of site specificity in histones. The activation of HATs or HDACs involves recruitment and formation of complexes with multiple components. HATs are known to complex with co-activators and co-repressors together with interactions with the histones (Fig. 1). Likewise, HDACs also form complexes with components including repressors. Histone modifying enzymes are not specific for specific gene transcriptions. It is therefore likely that other components of the complex impart this type of specificity (Lee and Workman, 2007). A molecular understanding of the process of recruitment and assembly of these complexes together with signaling pathways that control them are therefore important topics for future investigations in this field.

During a switch in differentiation or in metabolism, large changes in gene expression occur as a consequence of complex modifications of chromosomal components including heterochromatin formation, gene silencing and activation which are orchestrated by global epigenetic changes in acetylation and methylation of histones and methylation of DNA (Lam zuolo and Orlando, 2007). As a result new cellular memory is established which is a consequence of stabilization and heritability of transcription programs of epigenetic gene regulation. By this mechanism, tissue transdifferentiation and regeneration occur. The result of the changes in the epigenome provides plasticity and preservation of cellular phenotypes (Lam zuolo and Orlando, 2007). There is a balanced reciprocal change in methylation of H3K9, H3K4 and H3K79 and acetylation of H3K9 (Zhang et al., 2004). When H3K4 and H3K79 are methylated, H3K9 is acetylated and H3K9 is demethylated to increase gene expression. When H3K9 is trimethylated, H3K4 and H3K79 are demethylated and H3K9 is deacetylated to silence gene expression and increase heterochromatin formation.

Modifications in DNA are by methylations that specifically occur at C5 position of cytosine associated with CpG dinucleotides. It is estimated that as much as 80% of all CpG dinucleotides in mammalian genome are methylated. The remaining unmethylated CpG residues are largely localized in the promoter regions of constitutively active genes and are referred to as CpG islands (Esteller, 2007). Most of the DNA methylation occurring in the promoter region modify gene expressions. Methylations can also occur in nonpromoter regions but their significance remains undecided. DNA is methylated by DNA methyltransferase (DNMT) and this process is reversed by demethylases (DDM). Four mammalian DNA methyltransferases have been identified; DNMT1, DNMT2, DNMT3a, and DNMT3b. However, the characteristics and significance of DNA demethylases are not well studied. DNMTs are also known to interact with HDACs. By analogy, HATs may interact with DDMs.

Both histone methylation and DNA methylation act in concert (Adcock et al., 2006). It is viewed by some researchers that DNA methylations may not be as important as histone modifications as an epigenetic marker. This is based on the argument that in some models (e.g., yeast, nematode) there is

little DNA methylation (see Szyf, 2007). Also, up to 60% reduction in the DNA methylation capacity in some systems had no influence on the gene expression (Skipper, 2008). Interactions between histone modifications and DNA methylations also exist. For example, in some cases, H3-K9 methylation is required for DNA methylation and the latter can trigger H3-K9 methylation. Increased histone acetylation and decreased DNA methylations are associated with increased gene transcription, indicating a reciprocal relationship between these 2 modifications.

This review first summarizes new developments on histone acetylations and methylations. This is followed by the role played by SAMe, the methyl group donor and its therapeutic usefulness in the context of the epigenetics. The review also discusses the evidence for epigenetic memory, reprogramming in vivo by drug treatment through histone or DNA methylation, and the role of DNA methylation in the genetic expression of NMDA-NR2B gene.

ROLE OF ETHANOL-INDUCED HISTONE ACETYLATION, METHYLATION, AND PHOSPHORYLATION

Ethanol causes selective acetylation of histone H3 at lys 9 (or H3AcK9) in primary culture of rat hepatocytes (Park et al., 2003). This was also observed in hepatic stellate cells (Kim and Shukla, 2005) and in rats acutely treated in vivo with ethanol (Kim and Shukla, 2006) where histone was also acetylated in lung, spleen, and testes. Other lysine residues on H3 (i.e., lys 14, 18, and 23) were not acetylated. Interestingly, this acetylation by ethanol was dependent on ethanol metabolism since pyrazole (inhibitor of alcohol dehydrogenase, ADH) and methyl cyanamide (inhibitor of aldehyde dehydrogenase, ALD), both reduced this response. Acetaldehyde and acetate, products of ethanol metabolism, both caused H3K9 acetylation. However, at this stage it appears that induction of histone acetylation by either ethanol or acetate may not be via the same mechanisms since ethanol effect was inhibited by MAP kinase inhibitors but not that of the acetate. Ethanol was shown to increase histone acetyl transferase activity (Park et al., 2005). In a related development it was observed that surrogate alcohols e.g., propanol, isopentanol etc. (found as contaminants in alcoholic beverages) also increased histone H3 acetylation. These alcohols modulate H3lys 9 acetylation via increasing HAT activity and this was dependent on their metabolism. Furthermore, their metabolites (e.g., acetate, propionate, butyrate etc.) also increased this acetylation but, in contrast, they caused both HAT activation and HDAC inhibition. These studies highlight mechanistic differences in alcohols and their metabolites in the context of increasing histone acetylation (Choudhury and Shukla, 2008). A significant aspect of these developments is the observation that increased H3K9 acetylation by ethanol correlated with transcriptional increase in the ADH1 gene expression, as examined by chromatin immunoprecipitation (CHIP) assays (Park et al., 2005). If ethanol and/or acetaldehyde increased the acetylation of

histones and this results in augmented ADH1 gene expression; then this, in turn, will increase ethanol metabolism and thereby lead to an “amplification cycle” (Shukla et al., 2007). It is therefore likely that the metabolism of ethanol in the liver has an impact on its epigenetic responses.

Acetylation and methylations of histone have been shown to exhibit intricate and sometimes opposing behaviors. This was also evident in the case of ethanol effect on these 2 types of modifications. In hepatocytes histone H3 methylation at lys 4 was associated with up-regulation of genes, while lys 9 methylation was associated with down-regulated genes (Pal-Bhadra et al., 2007). On one hand, ethanol caused an increase in H3K9 acetylation but decreased the methylation at the same lysine residue. Concurrently, methylation at H3 lysine 4 increased. Through a series of CHIP experiments, it became clear that H3K4 methylation by ethanol was associated with up-regulation of genes (e.g., Adh, glutathione-S-transferase-yc2) whereas methylation at H3K9 was associated with genes (L-serine dehydratase, cytP4502c11) down regulated by ethanol. Thus methylations at different lysine residues in histone H3 have opposing transcriptional responses (Pal-Bhadra et al., 2007). It may be noted that epigenetic silencing of tumor necrosis factor α (TNFα) gene promoter was shown to be mediated via altered histone H3K9 methylation and may be involved in endotoxin-mediated inflammatory processes (Gazzar et al., 2007).

Ethanol is well documented to alter MAP kinases (Aroor and Shukla, 2004). In this context, recent studies have addressed whether histone proteins are targets of these kinases. It was shown that ethanol and its metabolite acetaldehyde, both increased phosphorylation of histone H3 at ser-10 and ser-28 residues. Through the use of pharmacological manipulations it was established that ethanol-induced histone H3 phosphorylations were mediated via p38MAP kinase but not by p42/44MAP kinase or JNK kinases. Furthermore, ethanol caused a similar level of phosphorylation of histone H3 at serine-10 and ser-28 but, in contrast, the level of phosphorylation induced by acetaldehyde at ser-28 was greater than that at ser-10 (Lee and Shukla, 2007). These site selective differences further highlight the varied effects of ethanol and its metabolites on epigenetic changes (Table 1). The histone H3 phosphorylation in hepatocytes by ethanol or acetaldehyde had no influence on apoptosis (Lee and Shukla, 2007). Significance of this phosphorylation on transcriptional events remains to be ascertained.

It is clear that ethanol and its metabolites cause different site specific modifications in histones. Future studies should exploit these specific changes for the development of therapeutic modalities. In this respect studies with SAMe offer some new insight.

### ROLE OF SAMe AS METHYL DONOR AND ITS EPIGENETIC IMPORTANCE IN ALCOHOLIC LIVER DISEASE

The methylation of histones or DNA can be influenced and regulated by the levels of SAMe and the metabolic pathways related to SAMe. The liver is the main source of SAMe (also known as SAM and AdoMet) biosynthesis and consumption, turning over nearly 8 g/d in a normal adult (Mudd et al., 1980). SAMe is the link to 3 key metabolic pathways: polyamine synthesis, transmethylation, and transsulfuration (Fig. 3). In polyamine synthesis, SAMe is decarboxylated and the remaining propylamino moiety attached to its sulfonium ion is donated to putrescine to form spermidine and methylthioadenosine (MTA) and to spermidine to form spermine and a second molecule of MTA. In transmethylation, SAMe donates its methyl group to a large variety of acceptor molecules in reactions catalyzed by methyltransferase. S-adenosylhomocysteine (SAH) is generated as a product of transmethylation and is hydrolyzed to form homocysteine (Hcy) and adenosine through a reversible reaction catalyzed by SAH hydrolase. SAH is a potent competitive inhibitor of methylation reactions and

![Figure 3](#fig3)

**Fig. 3. Metabolic pathways involved in SAMe production and its role as methyl donor. The figure shows the metabolic pathways and enzymatic steps involved in the generation of SAMe. Ethanol has effects on SAMe, as described in the text. The level of SAMe production is important in the methylation of histone and DNA mediated by methyltransferases in which SAMe plays a pivotal role as the methyl group donor. GSH, glutathione; Hcy, homocysteine; MAT, methionine adenyl transferase; Met, methionine; MTA, methylthioadenosine; SAH, S-adenosyl homocysteine; SAMe, S-adenosyl methionine; MTHF, methyl tetrahydrofolate.**

| Type of modification | Ethanol effect | Relationship to transcription |
|----------------------|----------------|------------------------------|
| Histone acetylation  |                |                              |
| H3K9                 | Increase      | Increase in transcription    |
| H3K14 or K18 or K23  | None detected |                              |
| Histone methylation  |                |                              |
| H3K4                 | Increase      | Association with ethanol upregulated genes |
| H3K9                 | Increase      | Association with ethanol downregulated genes |
| Histone phosphorylation |             |                              |
| H3-ser-10            | Increase      | Not known                    |
| H3-ser-28            | Increase      | Not known                    |
| DNA methylation      |                |                              |
| CpG island           | Decrease      | Increase in transcription    |

The site-specific modifications in histone or in DNA and their relationship to the transcriptions are shown.

prompt removal of adenosine and Hcy is required to prevent accumulation of SAH. Hcy can be remethylated to form methionine via methionine synthase (MS), which requires folate and vitamin B₁₂ and betaine homocysteine methyltransferase (BHMT), which requires betaine, a metabolite of choline. Remethylation of homocysteine via MS requires 5-methyltetrahydrofolate (5-MTHF), which is derived from 5,10-methylenetetrahydrofolate (5,10-MTHF) in a reaction catalyzed by methylenetetrahydrofolate reductase (MTHFR). 5-MTHF is then converted to tetrahydrofolate (THF) as it donates its methyl group and THF is converted to 5,10-MTHF to complete the folate cycle. Hcy can also undergo the transsulfuration pathway to form cysteine (the rate-limiting precursor for GSH) via a two-step enzymatic process catalyzed by cystathionine γ-synthase (CBS) and cystathionase, both requiring vitamin B₆. Methionine adenosyltransferases (MAT) are necessary components in this cascade. All mammalian tissues express MAT and MS, whereas BHMT expression is limited to the liver and kidney. In the liver, SAMe plays a regulatory role on its metabolism by inhibiting MTHFR and MS and activating CBS. Thus, when SAMe is depleted, homocysteine is channeled to remethylation to regenerate SAMe; whereas when SAMe level is high, homocysteine is channeled to the trans-sulfuration pathway. Patients with liver disease including alcohol have multiple abnormalities in this pathway, resulting in lower SAMe and GSH levels and higher homocysteine level (Mato and Lu, 2007).

The MAT is a critical cellular enzyme because it is the only enzyme that can catalyze the biosynthesis of SAMe. The MAT gene is one of 482 genes required for survival of an organism (Mato et al., 2002). In mammals, 2 different genes, MAT1A and MAT2A, encode for 2 homologous MAT catalytic subunits, α1 and α2 (Kotb et al., 1997). MAT1A is expressed mostly in the liver and it encodes the α1 subunit found in 2 native MAT isoenzymes, MAT III (dimer) and MAT I (tetramer) (Kotb et al., 1997). MAT2A encodes for a catalytic subunit (α2) found in a native MAT isoenzyme (MAT II), which is widely distributed (Horikawa and Tsukada, 1992; Kotb et al., 1997). MAT2A and its gene product also predominate in the fetal liver and are progressively replaced by MAT1A during development (Gil et al., 1996; Horikawa et al., 1993). There is a regulatory subunit (β) that is associated only with MAT II (Kotb et al., 1997; Mato et al., 2002). The β subunit is encoded by the gene MAT2β that is expressed in extrahepatic tissues but not in normal liver (LeGros et al., 2000; Martínez-Chantar et al., 2003).

Decreased hepatic SAMe biosynthesis in alcoholic liver disease also effects DNA methylation. Chronic alcoholics have hypermethioninemia and impaired methionine clearance. This is due to decreased hepatic MAT activity as a result of both decreased MAT1A expression and inactivation of MAT I/III isoenzymes, leading to decreased SAMe biosynthesis, which can impact on methylation and anti-oxidant defense (Tsukamoto and Lu, 2001). This has been demonstrated in rats fed intragastric ethanol for 9 weeks where hepatic levels

of methionine, SAMe, GSH, and DNA methylation fell by ~40% (Lu et al., 2000). DNA hypomethylation can affect gene expression and chromatin structure, rendering affected regions more accessible to DNA-damaging agents (Pogribny et al., 1995). Indeed hypomethylation and increased expression of c-myc, and increased genome-wide DNA strand break accumulation have been observed (Lu et al., 2000). These changes may predispose the organ to malignant degeneration. Indeed, Mat1a knockout mice which exhibit chronic hepatic SAMe deficiency, develop spontaneous hepatocellular carcinoma (Martínez-Chantar et al., 2002). As discussed later in this review, ethanol also caused increase in the methylation of neuronal NR2B DNA in the 5'UTR region and CpG island. Influence of SAMe on this DNA methylation in neuronal system and its consequences, in the context of ethanol, remain poorly understood.

Therapeutic Implications of Epigenetic Mechanisms in Attenuation of ALD by SAMe

The SAMe has been used therapeutically in alcoholic liver disease but its mechanism of action remains unclear (Mato et al., 1999; Rambaldi and Gluud, 2001). SAMe reduced the expression of pro-inflammatory cytokines such as TNFα, which plays a key role in the development of many liver injuries including that caused by alcohol (Chawla et al., 1998; Veal et al., 2004). SAMe and its metabolite MTA can block lipopolysaccharide (LPS)-induced TNFα expression in RAW cells and primary cultures of Kupffer cells at the transcriptional level without affecting NFκB nuclear binding (Veal et al., 2004). SAMe's effect was not mediated via a change in TNFα promoter methylation status. Furthermore, SAMe's inhibitory effect was not prevented by overexpression of p65 or/and its co-activator p300, or enhanced by overexpression of coactivator-associated arginine methyltransferase 1, an enzyme that methylates p300 and inhibits a p65-p300 interaction (Veal et al., 2004). New data indicate that at therapeutic doses, SAMe exerts its effect largely via MTA. This is because SAMe is unstable and readily converts to MTA spontaneously. In contrast to SAMe, MTA inhibits methylation and was shown to reduce histone 3 lysine 4 (H3K4) methylation (Fish et al., 2005; Huang et al., 2006), which is linked to transcriptional activation. Using chromatin immunoprecipitation assay, it was observed that LPS treatment increased binding of trimethylated H3K4 to the TNFα promoter while both SAMe and MTA blocked this increase. Similar findings were observed with inducible nitric oxide synthase (iNOS), where SAMe and MTA also blocked the LPS-induced increase in transcription and binding of trimethylated H3K4 to the promoter (Iglesias et al., 2008). Thus, therapeutic use of SAMe can impact on gene expression via MTA-mediated inhibition of histone methylation. This highlights a new role for SAMe related to its therapeutic effects. Indeed, MTA can recapitulate many of SAMe's effect on growth, apoptosis, and gene expression and may be the responsible mediator for many of SAMe's actions (Mudd et al., 1980).

EVIDENCE FOR EPIGENETIC MEMORY AND REPROGRAMMING BY ETHANOL AND DRUG THROUGH METHYLATION OF HISTONE

Alcohol and drug ingestion both sensitize the liver to non-specific injury. This phenomenon is sometimes referred to as the “second hit hypothesis” (Day and James, 1998). The concept is that alcohol or a drug can induce a state which predisposes the liver to exaggerated injury when the liver is exposed to toxic compounds other than alcohol or drug. This is also referred to as “drug primed injury” (Bardag-Gorce et al., 2006). The mechanism of the “priming” effect of alcohol or the drug remains unknown. In the case of alcohol, rats fed ethanol chronically develop an exaggerated response to the injury when challenged with endotoxin (Bhagwandeet al., 1987). An example of “drug primed” injury is the DDC (diethyl-1,4-dihydro-2,4,6-trimethyl-3,5, pyridine decarboxylate) fed mouse model of Mallory body (MB) formation (French et al., 2001). When mice are fed DDC for 10 weeks, followed by drug withdrawal for 1 month, they respond to a large variety of liver toxins by forming MBs in 3 to 7 days. DDC primed mice liver cells in tissue culture also spontaneously develop MBs in 3 to 6 days (Wu et al., 2005). This is due to the abrupt change by introduction of foreign chemical which changes the liver homeostatic milieu. The drug-primed model was associated with a change in the expression of a large number of genes (649) in the liver (French et al., 2005). Likewise, chronic ethanol feeding in rats induces a change in the expression of several genes (1300) in the liver when the blood alcohol levels are high (Bardag-Gorce et al., 2006). A similar response to acute high ethanol levels was also noted in HepG2 cells which overexpressed CYP2E1 in vitro. Importance of epigenetic mechanisms in the altered gene expression caused by ethanol and DDC in rats and mice fed ethanol chronically, have been examined.

The increases in H3K9 acetylation, HAT activity, and p300 HAT protein were associated with chronic ethanol feeding and high blood alcohol levels (Bardag-Gorce et al., 2007b). In the case of the drug-primed mouse model, the changes in gene expression observed correlated with both an increase in H3K9 acetylation and a decrease in H3K9 methylation. There was also a correlation with the methylation of DNA as determined by immunohistochemistry using an antibody to 5-methyl cytosine. The changes in gene expression and the induction in MB formation were blocked by SAMe in vivo and in vitro (Li et al., 2007). Thus, increased acetylation of H3K9, decrease in trimethylation of H3K9 and methylation of DNA showed relationship to the changes in the gene expression and MB formation in drug-primed mouse. The gene expression changes, the MB formation and SAMe blocking phenomenon persist up to 4 months after withdrawal of the drug. This most likely represents an epigenetic memory phenomenon of gene expression.

Epigenetic memory explains phenotypic changes which occur in adult differentiated cells like hepatocytes. Sometimes the changes in phenotype are obvious as in intestinal metaplasia or in the squamous metaplasia that occurs in respiratory epithelium. The most obvious phenotypic change that occurs in liver cells is bile duct metaplasia which occurs after chronic injury due to viral hepatitis, autoimmune hepatitis or ASH (alcoholic steatohepatitis) and NASH, non-alcoholic steatohepatitis, (Ray et al., 1993). The epigenetic memory change in drug-primed hepatocytes can be recognized by immune histochemistry staining with the antibody to ubiquitin, alpha fetoprotein, GST mu2, p62 or UB-D (diubiquitin or Fat-10) and by their response to injury because they replicate much faster than the normal neighboring hepatocytes (growth advantage) and their propensity to form MBs and liver tumors (Bardag-Gorce et al., 2007a; Nan et al., 2004, 2006a,b; Roomi et al., 2006).

ROLE OF ETHANOL-INDUCED DNA METHYLATION IN NMDA-NR2B GENE

One of the important pharmacological targets of ethanol in the CNS is NMDA receptor (reviewed by Kumari and Ticku, 2000). Ethanol interferes differentially with the expression of NMDAR subunits. The NR2B subunit may be an obligatory component of an NMDA isoreceptor sensitive to ethanol. Earlier work has demonstrated that chronic exposure to ethanol results in a compensatory increase in NMDAR binding density (Grant et al., 1990; Gulya et al., 1991), and an elevated mRNA and protein expression of NR2B subunit in the CNS (Bao et al., 2001; Chandler et al., 1999; Follesa and Ticku, 1995, 1996; Hardy et al., 1999; Kalluri et al., 1998). As a result of increased NMDAR expression, it has been proposed that altered NMDAR-mediated responses contribute to the hyperexcitability and excitotoxicity associated with ethanol-withdrawal seizures (Thomas and Morrisett, 2000). The cellular and molecular mechanisms underlying alcohol tolerance, dependence and sensitivity, however, remain undiscovered. Recent work has implicated epigenetic modifications as important regulatory mechanism for transcription of NR2B gene in cultured cortical neurons of mice.

To investigate whether the changes of methylation status might play a direct role in activating NR2B gene transcription, cultured neuronal cells were treated with 5-azacytidine, a DNA methyltransferase inhibitor and an increase in expression of NR2B mRNA was observed. This suggests that the methylation modification in NR2B gene may be involved in the up-regulation of NR2B gene transcription and may be relevant to alcoholism. Effect of ethanol on the methylation of 5'UTR region of NR2B gene including a CpG island (Klein et al., 1998) has been suggested. This was examined by Southern blotting combined with methyl-sensitive restriction endonucleases, and it was demonstrated that chronic ethanol exposure caused a demethylation in the regions of Intron 1 and 2 (Marutha Ravindran and Ticku, 2004, 2005), suggesting that epigenetic DNA methylation may be involved in ethanol-induced alterations of NR2B gene expression.

ROLE OF EPIGENETICS IN THE ACTIONS OF ALCOHOL

Transcription regulation has been involved in the mechanisms of drugs of abuse and ethanol-induced adaptive alterations (for reviews, see Fitzgerald and Nestler, 1995; Pandey et al., 2005). The transcription factors, neuron-restrictive silencer factor (NRSF, Qiang et al., 2005), AP-1 (Qiang and Ticku, 2005), and CREB (Rani et al., 2005) have been reported in the regulation of NR2B gene transcription following chronic ethanol treatment. NR2B gene promoter and 5'-UTR region structure was cloned and characterized by Sasner and Buonanno (1996) and Klein et al. (1998). To identify the functional element in the 5'-flanking region of the NR2B gene, a 5' deletion analysis was performed by constructing a series of luciferase reporter gene plasmids containing NR2B promoter region in different length that extended from -5319 to +30bp relative to the reported transcription start site of the mouse NR2B gene. The results suggested the presence of enhancing element(s) within the -1224 to -400bp region, and silencing element(s) in the region between -3004 and -1224bp. In the above mentioned experiment, after ethanol treatment for 5 consecutive days, significant increase of luciferase activity was detected in most promoter regions except where -100 to -400bp compared with their corresponding controls. This ethanol-induced increase in promoter activities suggests the possible existence of ethanol-sensitive positive or negative epigenetically regulated elements in this region (Qiang and Ticku, 2005; Qiang et al., 2005).

Further, the roles of transcription factors NRSF, CREB, and AP-1 in regulating NR2B gene transcription as well as in mediating ethanol-induced neuroplasticity were identified. Neuron-restrictive silencer factor (NRSF, also called repressor element-1) a transcriptional repressor, has been identified in many genes. The role of NRSF in NR2B promoter activity and the molecular mechanisms of ethanol-induced NR2B up-regulation in cultured cortical neurons have been studied (Qiang et al., 2005). Role of transcription factors CREB and AP-1 has been implicated in drug addiction. In order to study whether AP-1 and CREB proteins bound to NR2B promoter in vitro and in vivo, EMSA and ChIP assays were performed with the antibodies broadly reactive with c-Fos, c-Jun, FosB, and CREB family proteins. Results indicated that both in vitro and in vivo, c-Fos, FosB, c-Jun, and p-CREB proteins in primary cultured neurons bind to the particular site of NR2B promoter, and may promote NR2B gene transcription in vivo (Qiang and Ticku, 2005). These experiments also indicate that CREB and AP-1 mediate chronic ethanol-induced NR2B up-regulation. DNA methylation is particularly stable, and patterns of methylation, control many specific features such as learning and memory (Razin, 1998). Changes in DNA methylation and histone acetylation have been reported recently in alcoholic patients (Bonsch et al., 2005) and in animals in vivo (Kim and Shukla, 2006). Recent studies on the epigenetic changes in synuclein (Bonsch et al., 2005), a protein that may be associated with craving (Foroud et al., 2007), are also noteworthy. The amount of DNA methylation at a promoter site correlates with the extent of gene inactivation. Thus epigenetic modification, involving DNA methylation may provide a novel mechanism for the long-term effects in CIE-induced NR2B gene up-regulation.

FUTURE DIRECTION

An appreciation for the complex network of epigenetic modifications comes from the fact there are about 15 HATs and about 10 HDACs known thus far. The 4 DNMTs and the possible involvement of DNA demethylases add to the diversity of epigenetic regulations (Fig. 2). Future studies are needed to determine which methylation and demethylation enzymes are involved in the methylation of histone and DNA. Several enzymes are potentially involved in the methylation and demethylation of H3K9 (Volkel and Angrand, 2007). Trimethylation of H3K9 increases methylation of DNA (Volkel and Angrand, 2007). Different methyltransferase mediated methylation of histones or DNA are responsible for epigenetic memory transmitted through mitosis (Dodd et al., 2007). The same is true for H3K9 acetylation (Mehnert and Kelly, 2007) where acetylation increases and deacetylation decreases cell proliferation, differentiation and angiogenesis. Modifications in histone H4, H2A and H2B or in the histone variants e.g., H3.3 by ethanol, are yet to be evaluated. Does ethanol action in cells create a distinctive “epigenetic signature” as it relates to histone or DNA modifications? This may serve as a diagnostic marker. HDAC inhibitors (which increase histone acetylation) are being developed as anti cancer agents. How does then an increase in histone acetylation by ethanol influence, for example, development of hepatic carcinoma? Is it possible that histone modifications by ethanol may also have protective effect? Are there differences in the effects of ethanol and its metabolites on individual types of HATs or HDACs? What is the role of cell signaling network in the control of histone and DNA modifications? These provocative questions remain to be answered.

Alcoholic liver disease in humans and animal models result in hepatic S-Adenosylmethionine (SAMe) depletion. In animal models, this has been shown to lead to DNA hypomethylation and increased DNA strand break. Which genes can be affected by this mechanism remains poorly characterized and whether SAMe supplementation can prevent these remains unknown. In addition to DNA hypomethylation, SAMe depletion may also affect histone methylation and remains to be investigated. Regarding SAMe as a therapeutic agent, whether its ability to inhibit proinflammatory mediators such as TNFα and iNOS is due to MTA mediated inhibition of H3K4 methylation will require confirmation. How these agents affect other histone methylations, such as H3K9, which is associated with transcriptional silencing, remain unexplored. Also, whether they also affect histone acetylation is worthy of study as these agents may increase the binding of histone deacetylases to the promoter of BHMT (Ou et al., 2007). Both SAMe depletion and SAMe therapy are expected to have important influences on epigenetics, via changes in DNA and histone methylation, and possibly histone acetylation. These are all fruitful areas of future investigation, which will broaden our understanding of

how SAMe depletion leads to liver injury and how therapeutic SAMe works via epigenetic mechanisms.

The precise regulatory mechanisms by which ethanol alters DNA methylation and transcription factors regulation that consequently lead to gene expression, are not clear. The future studies are needed to map the ethanol-induced changes in methylation status of individual CpG and determine the functional links between ethanol-induced demethylation in specific CpG sites and transcription factors accessibility to DNA promoter in the regulation of gene expression. Histone methylations are upstream of DNA methylations. DNMTs are also found in protein complexes with HDACs and other proteins. It will be therefore necessary to find out whether ethanol disrupts or promotes such inter-protein interactions leading to altered epigenetic responses.

Based on the preceding discussions it is apparent that investigations into the epigenetic consequences of ethanol have just begun. The relevance of these changes, whether transient or permanent, may have important implications in the pathophysiology of alcohol actions in a variety of cell and organ systems. Importance of epigenetic alterations in cell growth and development, fetal alcohol syndrome, alcohol addiction, gastrointestinal disease etc. have to be ascertained. Future awaits these exciting new developments.

ACKNOWLEDGMENT

SDS is thankful to Mr. Daniel Jackson for editorial help.

REFERENCES

Adcock IM, Ford P, Ito K, Barnes PJ (2006) Epigenetic and airways disease. BioMed Central-Respir Res 7: 21.

Anway MD, Cupp AS, Uzumcu M, Skinner MK (2005) Epigenetic transgenerational actions of endocrine disruptors and male fertility. Science 308:1466–1469.

Aroor AR, Shukla SD (2004) MAP Kinase signaling in diverse effects of ethanol. Life Sci 74:2339–2364.

Bao X, Hui D, Naassila M, Michaelis EK (2001) Chronic ethanol exposure increases gene transcription of subunits of an N-methyl-D-aspartate receptor-like complex in cortical neurons in culture. Neurosci Lett 315: 5–8.

Bardag-Gorce F, Dedes J, French BA, Oliva J, Li J, French SW (2007a) Mallory body formation is associated with epigenetic phenotypic changes in hepatocytes *in vivo*. Exp Mol Pathol 83:160–168.

Bardag-Gorce F, French BA, Dedes J, Li J, French SW (2006) Gene expression patterns of the liver in response to alcohol: *in vivo* and *in vitro* models compared. Exp Mol Pathol 80:241–251.

Bardag-Gorce F, French BA, Joyce M, Baires M, Montgomery R, Li J, French SW (2007b) Histone acetyltransferase p300 modulates in an epigenetic manner at high blood alcohol levels. Exp Mol Pathol 82:197–202.

Bhagwandeenv BS, Apte M, Manwarring L, Dickens J (1987) Endotoxin induced hepatic necrosis in rats on alcohol diet. J Pathol 152:47–53.

Bonsch D, Lenz B, Kornhuber J, Bleich S (2005) DNA hypermethylation of the alpha synuclein promoter in patients with alcoholism. Neuroreport 16:167–170.

Chandler LJ, Norwood D, Sutton G (1999) Chronic ethanol upregulates NMDA and AMPA, but not kainate receptor subunit proteins in rat primary cortical cultures. Alcohol Clin Exp Res 23:363–370.

Chawla RK, Watson WH, Eastin CE, Lee EY, Schmidt J, McClain CJ (1998) S-adenosylmethionine deficiency and TNF-α in lipopolysaccharide-induced hepatic injury. Am J Physiol 38:G125–G129.

Cheung P, Lau P (2005) Epigenetic regulation of histone demethylation and histone variants. Mol Endocrinol 19:563–573.

Choudhury M, Shukla SD (2008) Surrogate alcohols and their metabolites modify histone H3 acetylation: involvement of histone acetyl transferase and histone deacetylate. Alcohol Clin Exp Res 32:829–839.

Day CP, James OF (1998) Steatohepatitis: a tale of two “hits”. Gastroenterology 114:842–845.

Dodd IB, Michelsen MA, Sneppen K, Thon G (2007) Theoretical analysis of epigenetic cell memory by nucleosome modification. Cell 129:813–822.

Esteller M (2007) Cancer epigenomics: DNA methylomes and histone-modification maps. Nat Rev Genet 8:286–298.

Fish JE, Matouk CC, Rachlis A, Lin S, Tai SC, D'Abreo C, Marsden PA (2005) The expression of endothelial nitric-oxide synthase is controlled by a cell-specific histone code. J Biol Chem 280:24824–24838.

Fitzgerald LW, Nestler EJ (1995) Molecular and cellular adaptations in signal transduction pathways following ethanol exposure. Clin Neurosci 3:165–173.

Follesa P, Ticku MK (1995) Chronic ethanol treatment differentially regulates NMDA receptor subunit mRNA expression in rat brain. Mol Brain Res 29:99–106.

Follesa P, Ticku MK (1996) Chronic ethanol-mediated up-regulation of the N-methyl-D-aspartate receptor polypeptide subunits in mouse cortical neurons in culture. J Biol Chem 271:13297–13299.

Foroud T, Wetherill LF, Liang T, Dick DM, Hasselbrock V, Kramer J, Nurnberger J, Schuckit M, Carr L, Porjesz B, Xuei X, Edenberg HJ (2007) Association of alcohol craving with α-synuclein (SNCA). Alcohol Clin Exp Res 31:537–545.

French BA, Dedes J, Bardag-Gorce F, Li J, Wilson L, Fu P, Nan L, French SW (2005) The CYP2E1 inhibition enhances Mallory body formation. Exp Mol Pathol 78:207–211.

French BA, van Leeuwen F, Riley NE, Yuan QX, Bardag-Gorce F, Lue YH, Marceau N, French SW (2001) Aggresome formation in liver cells in response to different toxic mechanisms. Role of the ubiquitin-proteasome pathway and the frameshift mutant of ubiquitin. Exp Mol Pathol 71:241–246.

Gazzar ME, Yoza BK, Hu JY, Cousart SL, McCall CE (2007) Epigenetic silencing of tumor necrosis factor α during endotoxin tolerance. J Biol Chem 282:26857–26864.

Gil B, Casado M, Pajares M, Boscá L, Mato JM, Martín-Sanz P, Álvarez L (1996) Differential expression pattern of methionine adenosyltransferase isoenzymes during rat liver development. Hepatology 24:876–881.

Grant KA, Valverius P, Hudspith M, Tabakoff B (1990) Ethanol withdrawal seizures and the NMDA receptor complex. Eur J Pharmacol 176:289–296.

Gulya K, Grant KA, Valverius P, Hoffman PL, Tabakoff B (1991) Brain regional specificity and time-course of changes in the NMDA receptor-ionophore complex during ethanol withdrawal. Brain Res 547:129–134.

Hake SB, Allis CD (2006) Histone H3 variants and their potential role in indexing mammalian genome: the “H3 barcode hypothesis”. Proc Natl Acad Sci 103:6428–6435.

Hardy PA, Chen W, Wilce PA (1999) Chronic ethanol exposure and withdrawal influence NMDA receptor subunit and splice variant mRNA expression in the rat cerebral cortex. Brain Res 819:33–39.

Horikawa S, Ozasa H, Ota K, Tsukada K (1993) Immunohistochemical analysis of rat S-adenosylmethionine synthetase isozymes in developmental liver. FEBS Lett 330:307–311.

Horikawa S, Tsukada K (1992) Molecular cloning and developmental expression of a human kidney S-adenosylmethionine synthetase. FEBS Lett 312:37–41.

Huang J, Kent JR, Placek B, Whelan KA, Hollow CM, Zeng PY, Fraser NW, Berger SL (2006) Trimethylation of histone H3 lysine 4 by Set1 in the lytic infection of human herpes simplex virus 1. J Virol 80:5740–5746.

Iglesias AA, Xia M, Ramani K, Mato JM, Lu SC (2008) S-adenosylmethionine inhibits lipopolysaccharide-induced gene expression via modulation of histone methylation. Hepatology 47:1655–1666.

Jenuwein T (2006) The epigenetic magic of histone lysine methylation. FEBS J 273:3121–3135.

Jenuwein T, Allis CD (2001) Translating the histone code. Science 293:1074–1080.
Kalluri HS, Mehta AK, Ticku MK (1998) Up-regulation of NMDA receptor subunits in rat brain following chronic ethanol treatment. Mol Brain Res 58:221–224.
Kim JS, Shukla SD (2005) Histone H3 modifications in rat hepatic stellate cells by ethanol. Alcohol Alcohol 40:367–372.
Kim JS, Shukla SD (2006) Acute in vivo effect of ethanol (binge drinking) on histone H3 modifications in rat tissues. Alcohol Alcohol 41:126–132.
Klein M, Pieri I, Uhlmann F, Pfizenmaier K, Eisel U (1998) Cloning and characterization of promoter and 5'-UTR of the NMDA receptor subunit epsilon 2: evidence for alternative splicing of 5'-non-coding exon. Gene 208:259–269.
Kotb M, Mudd SH, Mato JM, Geller AM, Kredich NM, Chou JY, Cantoni GL (1997) Consensus nomenclature for the mammalian methionine adenosyltransferase genes and gene products. Trends Genet 13:51–52.
Kumari M, Ticku MK (2000) Regulation of NMDA receptors by ethanol. Prog Drug Res 54:152–189.
Lamzuolo C, Orlando V (2007) The function of the epigenome in cell programming. Cell Mol Life Sci 64:1043–1062.
Lee YJ, Shukla SD (2007) Histone H3 phosphorylation at serine 10 and serine 28 is mediated by p38MAPK in rat hepatocytes exposed to ethanol and acetaldehyde. Eur J Pharmacol 573:29–38.
Lee KK, Workman JL (2007) Histone acetyltransferase complexes: one size doesn't fit all. Nat Rev Mol Cell Biol 8:284–295.
LeGros HL Jr, Halim AB, Geller AM, Kotb M (2000) Cloning, expression, and functional characterization of the β regulatory subunit of human methionine adenosyltransferase (MAT II). J Biol Chem 275:2359–2366.
Li J, Bardag-Gorce F, Dedes J, French BA, Amidi F, Oliva J, French SW (2008). S-adenosylmethionine prevents Mallory Denk body formation in drug-primed mice by inhibiting the epigenetic memory. Hepatology 47:613–624.
Lohrum M, Stunnenburg HG, Logie C (2007) The new frontier in cancer research: deciphering cancer epigenetics. Int J Biochem Cell Biol 39:1450–1461.
Lu SC, Huang ZZ, Yang HP, Mato JM, Avila MA, Tsukamoto H (2000) Changes in methionine adenosyltransferase and S-adenosylmethionine homeostasis in the alcoholic rat liver. Am J Physiol 279:G178–G185.
Martínez-Chantar ML, Corrales FJ, Martínez-Cruz A, García-Trevijano ER, Huang ZZ, Chen LX, Kanel G, Avila MA, Mato JM, Lu SC (2002) Spontaneous oxidative stress and liver tumors in mice lacking methionine adenosyltransferase 1A. FASEB J 16:1292–1294.
Martínez-Chantar ML, Garcia-Trevijano ER, Latasa MU, Martin-Duce A, Fortes P, Caballeria J, Avila MA, Mato JM (2003) Methionine adenosyltransferase IIβ subunit gene expression provides a proliferative advantage in human hepatoma. Gastroenterology 124:940–948.
Marutha Ravindran CR, Ticku MK (2004) Changes in methylation pattern of NMDA receptor NR2B gene in cortical neurons after chronic ethanol treatment in mice. Mol Brain Res 121:19–27.
Marutha Ravindran CR, Ticku MK (2005) Role of CpG islands in the up-regulation of NMDA receptor NR2B gene expression following chronic ethanol treatment of cultured cortical neurons of mice. Neurochem Int 46:313–327.
Mato JM, Cámara J, Ortiz P, Rodés J, Spanish Collaborative Group for the Study of Alcoholic Liver Cirrhosis (1999) S-adenosylmethionine in the treatment of alcoholic liver cirrhosis: a randomized, placebo-controlled, double-blind multicentre clinical trial. J Hepatol 30:1081–1089.
Mato JM, Corrales FJ, Lu SC, Avila MA (2002) S-Adenosylmethionine: a control switch that regulates liver function. FASEB J 16:15–26.
Mato JM, Lu SC (2007) Role of S-adenosyl-L-methionine in liver health and injury. Hepatology 45:1306–1312.
Mehnert JM, Kelly WK (2007) Histone deacetylase inhibition: biology and mechanism of action. Cancer J 13:23–29.
Moss TJ, Wallrath LL (2007) Connections between epigenetic silencing and human disease. Mutat Res 618:163–174.

Mudd SH, Ebert MH, Scriver CR (1980) Labile methyl group balances in the human: the role of sarcosine. Metabolism 29:707–720.
Nan L, Bardag-Gorce F, Wu Y, Li S, French BA, French SW (2006a) Malignant body forming cells express the preneoplastic hepatocyte phenotype. Exp Mol Pathol 80:109–118.
Nan L, Dedes J, French BA, Bardag-Gorce F, Li J, Wu Y, French SW (2006b) Mallory body (cytokeratin aggresomes) formation is prevented in vitro by p38 inhibitor. Exp Mol Pathol 80:228–290.
Nan L, Wu Y, Bardag-Gorce F, Li J, French BA, Fu AN, Francis T, Vu J, French SW (2004) p62 is involved in the mechanisms of Mallory body formation. Exp Mol Pathol 77:168–175.
Ng RK, Gourdon JB (2008) Epigenetic memory of an active gene state depends on histone H3.3 incorporation into chromatin in the absence of transcription. Nat Cell Biol 10:102–109.
Ou XP, Yang HP, Ramani K, Iglesias-Ara A, Chen H, Mato JM, Lu SC (2007) Inhibition of human betaine homocysteine methyltransferase expression by S-adenosylmethionine and methylthioadenosine. Biochem J 401:87–96.
Pal-Bhadra M, Bhadra U, Jackson DJ, Mamatha L, Park P, Shukla SD (2007) Distinct methylation patterns in histone H3 at lys-4 and lys-9 correlated with up-and down-regulation of genes by ethanol in hepatocytes. Life Sci 81:979–987.
Pandey SC, Chartoff EH, Carlezon WA Jr, Zou J, Zhang H, Kreibich AS, Blendy JA, Crews FT (2005) CREB gene transcription factors: role in molecular mechanisms of alcohol and drug addiction. Alcohol Clin Exp Res 29:176–184.
Park P-H, Lim RW, Shukla SD (2005) Involvement of histone acetylation (AAT) in ethanol-induced acetylation of histone H3 in hepatocytes: potential mechanism for gene expression. Am J Physiol Gastrointest Liver Physiol 289:1124–1136.
Park PH, Miller R, Shukla SD (2003) Acetylation of histone H3 at lysine 9 by ethanol in rat hepatocytes. Biochem Biophys Res Commun 306:501–504.
Pogribny IP, Basnakian AG, Miller BJ, Lopatina NG, Poirier LA, James SJ (1995) Breaks in genomic DNA and within the p53 gene are associated with hypomethylation in livers of folate/methyl-deficient rats. Cancer Res 55:1894–1901.
Qiang M, Rani CS, Ticku MK (2005) Neuron-restrictive silencer factor regulates the N-methyl-D-aspartate receptor 2B subunit gene in basal and ethanol-induced gene expression in fetal cortical neurons. Mol Pharmacol 67:2115–2125.
Qiang M, Ticku MK (2005) Role of AP-1 in ethanol-induced N-methyl-D-aspartate receptor 2B subunit gene up-regulation in mouse cortical neurons. J Neurochem 95:1332–1341.
Rambaldi A, Gluud C (2001) S-Adenosyl-L-methionine for alcoholic liver disease. Cochrane Database Syst Rev 4:CD002235.
Rani CS, Qiang M, Ticku MK (2005) Potential role of cAMP response element-binding protein in ethanol-induced N-methyl-D-aspartate receptor 2B subunit gene transcription in fetal mouse cortical cells. Mol Pharmacol 67:2126–2136.
Ray MB, Mendenhall CL, French SW, Gartside PS (1993) The nature of bile duct changes in alcoholic liver disease. Liver 13:36–45.
Razin A (1998) CpG methylation, chromatin structure and gene silencing—a three-way connection. EMBO J 17:4905–4908.
Rodenhiser D, Mann M (2006) Epigenetics and human disease: translating basic biology into clinical applications. Can Med Assoc J 174:341–348.
Roomi MW, Gaal K, Yuan QX, French BA, Fu P, Bardag-Gorce F, French SW (2006) Preneoplastic liver cell foci induced by thioacetamide toxicity in drug-primed liver. Exp Mol Pathol 81:8–14.
Roth SY, Denu JM, Allis CD (2001) Histone acetyl transferases. Annu Rev Biochem 70:81–120.
Sasner M, Buonanno A (1996) Distinct N-methyl-D-aspartate receptor 2B subunit gene sequences confer neural and developmental specific expression. J Biol Chem 271:21316–21322.
Shahbazian MD, Grunstein M (2007) Functions of site-specific histone acetylation and deacetylation. Annu Rev Biochem 76:75–100.
Shukla SD, Lee YJ, Park P, Aroor AR (2007) Acetaldehyde alters MAP kinase signaling and epigenetic histone modifications in hepatocytes, in

Acetaldehyde Related Pathology-Bridging the Trans-Disciplinary Divide (Chadwick DJ, Goode J eds), pp 217–228. John Wiley & Sons, NJ.

Skipper M (2008) Helping cells to remember. Nat Rev Genet 9:2.

Smith CL (2007) A shifting paradigm: histone deacetylases and transcriptional activation. Bioessays 30:15–24.

Strahl BD, Allis CD (2000) The language of covalent histone modifications. Nature 403:41–45.

Szyf M (2007) The dynamic epigenome and its implications in toxicology. Toxicol Sci 100:7–23.

Thomas MP, Morrisett RA (2000) Dynamics of NMDAR-mediated neurotoxicity during chronic ethanol exposure and withdrawal. Neuropharmacology 39:218–226.

Tsukamoto HC, Lu SC (2001) Current concepts in the pathogenesis of alcoholic liver injury. FASEB J 15:1335–1349.

Veal N, Hsieh CL, Xiong S, Lu SC, Tsukamoto H (2004) Inhibition of lipopolysaccharide-stimulated TNFα promoter activity by S-adenosylmethionine and 5'-methylthioadenosine. Am J Physiol 287:G352–G362.

Volkel P, Angrand P-O (2007) The control of histone lysine methylation in epigenetic regulation. Biochimie 89:1–20.

Waddington CH (1942) Canalization of development and inheritance of acquired characters. Nature 150:563–565.

Wu Y, Nan L, Bardag-Gorce F, Li J, French BA, Wilson L-T, Debes J, French SW (2005) The role of laminin-integrin signaling in triggering (MB) formation: an in vivo and in vitro study. Exp Mol Pathol 78:1–8.

Yang X (2007) HATs and HDACs: from structure, function and regulation to novel strategies for therapy and prevention. Oncogene 26:5310–5318.

Zhang K, Silino JS, Jones PR, Yau PM, Bradbury EM (2004) A mass spectrometric “Western blot” to evaluate the correlations between histone methylation and histone acetylation. Proteomics 4:3769–3775.
