
REVIEW ARTICLE

DNA damage-induced gene expression in *Saccharomyces cerevisiae*

Yu Fu, Landon Pastushok & Wei Xiao

Department of Microbiology and Immunology, University of Saskatchewan, Saskatoon, SK, Canada

Correspondence: Wei Xiao, Department of Microbiology and Immunology, University of Saskatchewan, 107 Wiggins Road, Saskatoon, SK, Canada S7N 5E5. Tel.: +1 306 966 4308; fax: +1 306 966 4311; e-mail: wei.xiao@usask.ca

Received 15 February 2008; revised 27 May 2008; accepted 28 May 2008. First published online 8 July 2008.

DOI:10.1111/j.1574-6976.2008.00126.x

Editor: Martin Kupiec

Keywords
* Saccharomyces cerevisiae;
* DNA damage;
* transcriptional regulation;
* SOS response;
* cell-cycle checkpoint;
* signal transduction.

Abstract
After exposure to DNA-damaging agents, both prokaryotic and eukaryotic cells activate stress responses that result in specific alterations in patterns of gene expression. Bacteria such as *Escherichia coli* possess both lesion-specific responses as well as an SOS response to general DNA damage, and the molecular mechanisms of these responses are well studied. Mechanisms of DNA damage response in lower eukaryotes such as *Saccharomyces cerevisiae* are apparently different from those in bacteria. It becomes clear that many DNA damage-inducible genes are coregulated by the cell-cycle checkpoint, a signal transduction cascade that coordinates replication, repair, transcription and cell-cycle progression. On the other hand, among several well-characterized yeast DNA damage-inducible genes, their effectors and mechanisms of transcriptional regulation are rather different. This review attempts to summarize the current state of knowledge on the molecular mechanisms of DNA damage-induced transcriptional regulation in this model lower eukaryotic microorganism.

Introduction
DNA is the carrier of genetic information in most organisms. Any damage to the molecular structure of DNA has the potential to cause genomic instability, mutagenesis or even cell death. Unfortunately, DNA damage is unavoidable; DNA is continually exposed to insults resulting from exogenous and endogenous DNA-damaging agents as well as challenges posed by DNA replication. Therefore, it is not surprising that living organisms have developed numerous pathways to deal with DNA damage. In response to DNA damage, cells can alter many of their intracellular processes, including DNA repair, metabolism and cell-cycle progression, for cell survival and maintenance of genomic stability (Friedberg *et al.*, 2006). Such dynamic responses can be achieved through a variety of molecular mechanisms such as protein localization, protein degradation, posttranslational modification and genetic regulation. Generally speaking, altering gene expression is perhaps one of the most fundamental responses and it is reasonable to suggest that most, if not all, cellular processes are affected at the level of transcriptional regulation. In the field of DNA repair and mutagenesis, the notion that DNA damage causes an alteration in the expression profile of damage responsive genes has been an important area of research for many years. It is expected that the expression profile of a cell following DNA damage will provide us with critical information on how a cell protects itself from such stress.

Transcriptional regulation in response to DNA damage has been studied extensively in model bacteria such as *Escherichia coli*. In contrast, such information is relatively scarce in eukaryotic organisms, from simple lower unicellular eukaryotes such as *Saccharomyces cerevisiae* to humans. In this review, we attempt to summarize recent findings related to gene regulation in response to DNA damage in lower eukaryotes, particularly the budding yeast *S. cerevisiae*, and to compare this response with the well-studied bacterial SOS response.

Bacterial transcriptional responses to DNA damage

The SOS response in *E. coli*
When *E. coli* cells are subjected to DNA damage, about 48 unlinked genes are coordinately induced through a complex SOS regulatory network (Courcelle *et al.*, 2001). The increased expression of these SOS regulon genes results in the elaboration of a set of physiological responses such as an enhanced capacity for recombination repair and excision
DNA damage-induced transcription in budding yeast

repair, enhanced mutagenesis (due to error-prone translesion DNA synthesis mediated by PolIV and PolV) and inhibition of cell division. These responses have been collectively termed the SOS response (Radman, 1975; Witkin, 1976; Gottesman, 1981; Little & Mount, 1982).

### Current model for transcriptional control of the SOS response

The SOS regulatory network is mainly controlled by two proteins: RecA and LexA (Radman, 1975; Little & Mount, 1982; Walker, 1984). LexA is a transcriptional repressor that binds to SOS boxes located near or inside the operator site of the SOS-induced genes. SOS boxes are often palindromic structures with a high degree of homology in nucleotide sequence. An ideal symmetrical consensus sequence 5′-TACTGTATATATACAGTA-3′ was derived from the analysis of a pool of SOS boxes (Berg, 1988). The sequence distinction in SOS boxes allows the LexA repressor to bind to operators with different strengths (Lewis et al., 1994). The LexA occupancy prevents accessibility to RNA polymerase so as to inhibit the initiation of transcription. LexA is likely to interact with DNA via its N-terminal domain, and the C-terminus of LexA is required for its dimerization. The dimerization of LexA is essential for its ability to repress SOS-regulated genes *in vivo*. Meanwhile, LexA is able to undergo a slow intramolecular self-cleavage termed autodigestion, whose rate significantly increases upon interaction with RecA (Little, 1984, 1991, 1993).

The RecA protein of *E. coli* has at least three functions in the SOS response. It not only plays a role in the transcriptional regulation in response to DNA damage but also directly participates in translesion synthesis and homologous recombination. Following DNA damage, DNA synthesis becomes discontinuous and single-strand DNA (ssDNA) is produced by failed attempts to replicate damaged DNA. In the presence of ATP, RecA binds to the ssDNA region and forms helical RecA-ssDNA nucleoprotein filaments. LexA then diffuses to deep grooves in the RecA-ssDNA filaments and interacts with them in a manner that results in autocatalytic cleavage of LexA at a scissile peptide bond located between Ala84 and Gly85. Cleavage of LexA inactivates its ability as a repressor, thus releasing the SOS regulon genes from transcriptional repression. As cells begin to recover from the inducing treatment by various DNA repair and tolerance processes, the regions of ssDNA disappear, and thus the inducing signal is diminished. Without the cleavage stimulated by RecA-ssDNA filaments, the pool of LexA is boosted, which leads to repression of the transcription of SOS regulon genes and a return to the uninduced state (Walker, 1984; Friedberg et al., 2006).

---

### Fine tuning in the induction of the SOS response

The SOS regulatory system provides *E. coli* with a rapid transcriptional response to the presence of DNA damage. Furthermore, during SOS induction, the timing, the duration and the level of induction are diverse for different LexA-regulated genes, suggesting a fine-tuning mechanism in the SOS induction. The fine tuning is possibly determined by at least four parameters: (1) the binding affinity of LexA for the SOS box in the operator region; (2) the number of SOS boxes in the operator region; (3) the location of the SOS box relative to the promoter; and (4) the strength of the promoter.

After exposing *E. coli* cells to DNA-damaging agents, genes with operators that bind LexA relatively weakly are the first to turn on fully. For example, *uvrA*, *uvrB*, *ruvA*, *ruvB*, *recN* and *sulA* are induced within 5 min after 40 J m⁻² UV radiation (Courcelle et al., 2001). *uvrA* and *uvrB* encode proteins involved in nucleotide excision repair (NER); *recN*, *ruvA* and *ruvB* encode proteins used for recombination repair; while the protein product of the *sulA* gene can temporarily arrest cell division to allow bacteria time to complete the repair of damaged DNA. After the activation of loosely controlled SOS genes, if the damage cannot be fully repaired by NER and homologous recombination, genes with operators that are tightly controlled by LexA will be turned on. For example, the full induction of *umuC* and *umuD* is not observed until 20 min after 40 J m⁻² UV irradiation (Courcelle et al., 2001). Similar to LexA, the protein encoded by *umuD* also has a latent ability to autodigest, and the auto-digestion is strongly stimulated by the interaction between the RecA/ssDNA nucleoprotein filament and UmuD (Nohmi et al., 1988). UmuC and a posttranslationally processed form of UmuD (UmuD′) serve as a mutagenic lesion-bypass DNA polymerase (PolV). This last response allows the survival of *E. coli* after severe DNA damage, but at the expense of introducing errors into the genome.

In the SOS regulatory network, the transcript level of key regulatory genes *recA* and *lexA* is also regulated by LexA, thus forming a delicately controlled circuit (Brent & Ptashne, 1980; Little et al., 1981; Brent, 1982). *recA* has one SOS box positioned between the –35 and –10 regions of the promoter, and a tandem pair of SOS boxes is located in the *lexA* promoter region (Brent & Ptashne, 1981; Little et al., 1981; Schnarr et al., 1991). LexA can bind to these SOS boxes to prevent the initiation of transcription. Relative binding affinity experiments reveal that LexA binds to the *recA* operator more strongly than to the *lexA* operator and many other operators of SOS regulon genes such as *uvrA*, *uvrB* and *uvrD* (Brent & Ptashne, 1981; Peterson & Mount, 1987; Schnarr et al., 1991). Accordingly, it allows an intermediate inducible state, bridging an uninduced state and a

FEMS Microbiol Rev **32** (2008) 908–926

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reservedfully induced state (Little, 1983). A low amount of inducing signal can thus lead to the activation of some of the SOS functions, such as $u v r^{+}$-dependent NER, without substantial amplification of the RecA protein. When the inducing signal continues to accumulate, $r e c A$ will be induced, resulting in a full induction of SOS regulon genes. Meanwhile, due to the repression of $l e x A$ by LexA itself, the SOS response system is robust to prevent substantial induction caused by very small amounts of inducing signal. Furthermore, the strong repression of $r e c A$ by LexA and the constant expression of LexA in SOS response ensure a fast return to the uninduced state once the level of the inducing signal begins to decrease (Walker, 1984).

In addition to the two central regulators RecA and LexA, recent studies reveal that some other proteins are also involved in the subtleties of SOS induction. All these proteins appear to affect SOS regulation by modulating RecA-ssDNA stability. RecX can block the assembly of the RecA-ssDNA filament while not affecting the disassembly through capping the assembly ends (Drees et al., 2004). As a result, it strongly inhibits RecA-mediated DNA strand exchange, ATPase and coprotease activities. The $r e c X$ gene is located 76-bp downstream of $r e c A$, and these two genes belong to the same operon. Although $r e c X$ is cotranscribed with $r e c A$, $r e c X$ transcription is downregulated with respect to $r e c A$ by an intrinsic transcription terminator that is located between the $r e c A$ and $r e c X$ coding sequences. Despite the presence of this terminator, a $r e c A-r e c X$ message resulting from transcriptional read-through is detected at a level of 5–10% of the $r e c A$ message (Pages et al., 2003). RecX is barely detected during vegetative growth, but robust expression of $r e c X$ is observed after treating cells with DNA-damaging agents (Stohl et al., 2003). The maximal $r e c X$ expression is observed at a later time than maximal expression of RecA after UV irradiation (Courcelle et al., 2001). All these observations suggest that RecX is likely involved in the subtle regulation that helps shut off the SOS response.

Early studies have reported that DinI can destabilize the RecA-ssDNA filament when its concentration is 50–100-fold above the natural RecA concentration, and it inhibits all activities of RecA (Yasuda et al., 1998; Voloshin et al., 2001). Therefore, DinI was initially thought to aid the return of SOS-induced cells to a steady state. In contrast, recent research has shown that DinI is also able to stabilize the RecA-ssDNA filament to prevent its disassembly when present at concentrations that are stoichiometric with or somewhat greater than those of RecA (Lusetti et al., 2004). Furthermore, DinI-mediated stabilization affects RecA-mediated UmuD cleavage rather than RecA-mediated ATP hydrolysis and LexA coprotease activities (Lusetti et al., 2004), indicating that DinI could have a biological role in fine-tuning the activity of UmuD in order to limit SOS mutagenesis.

### Role of DNA helicases and nucleases in SOS induction

A critical step in the SOS response is the production of a RecA-ssDNA filament, and this step requires DNA helicases and nucleases. Either the RecFOR or the RecBCD pathway is necessary for the SOS response after UV irradiation (Ivancic-Bace et al., 2006). In $E$. coli, SOS induction immediately after UV irradiation is dependent on the RecFOR pathway. The RecFOR pathway includes the DNA helicase RecQ, the nuclease RecJ and the RecFOR complex, which facilitates RecA loading (Lusetti et al., 2006). It has been suggested that the pathway may aid RecA binding to ssDNA gaps. RecQ appears to be needed for fast degradation of the LexA repressor (Hishida et al., 2004). This observation leads to a model in which RecQ unwinds the template duplex in front of a stalled fork on the leading strand, and then switches over to the lagging strand to generate ssDNA on the leading strand template, allowing formation of the RecA filament in the $5^{\prime}-3^{\prime}$ direction for SOS induction (Heyer, 2004; Hishida et al., 2004).

RecBCD, also known as Exonuclease V, contains helicase, $5^{\prime}-3^{\prime}$ exonuclease, and RecA-loading activities. It can directly load RecA on the processed double-strand break (DSB) ends (Singleton et al., 2004). SOS induction after UV irradiation in recFOR mutants is not completely eliminated but is delayed, and this induction is dependent on the RecBCD enzyme (Thoms & Wackernagel, 1987; Hegde et al., 1995; Whitby & Lloyd, 1995; Renzette et al., 2005). Accordingly, it has been proposed that SOS induction requires RecBCD when the DSB ends appear later because of NER and replication fork collapse after UV irradiation (Ivancic-Bace et al., 2006).

### SOS-independent DNA damage induction in bacteria

Bacteria also possess damage-inducible systems that are independent of the SOS response. For example, alkylating or oxidative agents induce not only the SOS response but also other more specific DNA damage responses.

Following exposure to a sublethal dose of an alkylating agent (e.g. N-methyl-$N^{\prime}$-nitro-$N$-nitrosoguanidine, MNNG), bacteria such as $E$. coli manifest a pronounced resistance to both lethal and mutagenic effects caused by a much higher dose of the same or a similar alkylating agent (Jeggo et al., 1977). The resistance is due to the induction of one or more genes in response to low levels of the alkylating agents; the protein products of these genes can directly remove the alkylated adducts in DNA (Samson & Cairns, 1977). Therefore, this regulatory pathway is called the adaptive response to alkylation damage.

In $E$. coli, the Ada protein possesses dual functions and plays a pivotal role in the adaptive response: it is a

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved

FEMS Microbiol Rev **32** (2008) 908–926DNA damage-induced transcription in budding yeast

methyltransferase that removes alkyl groups from the methylated base and a strong transcriptional activator of several genes. The 39-kDa Ada protein is comprised of two functional domains linked by a hinge region. The C-terminal 19-kDa domain is capable of acquiring the methyl group from a mutagenic $O^{6}$-methylguanine or $O^{4}$-methylthymine to its Cys321 residue. The N-terminal 20-kDa domain specifically catalyzes the transfer of methyl groups from methyl phosphotriester lesions to its Cys38 residue in a direct and irreversible way (Demple *et al.*, 1985; Nakabeppu & Sekiguchi, 1986; Teo *et al.*, 1986; Lindahl *et al.*, 1988; Myers *et al.*, 1993b; Takinowaki *et al.*, 2006). Methylation of the Cys38 triggers a conformational change in the Ada protein, thereby dramatically (up to 1000-fold) enhancing the sequence-specific DNA-binding affinity of the Ada protein to the promoter regions of its own gene, *ada*, and other alkylation resistance genes including *alkA*, *aidB* and *alkB* (Teo *et al.*, 1986; Sakumi & Sekiguchi, 1989; Akimaru *et al.*, 1990; Myers *et al.*, 1993a; Sakashita *et al.*, 1993; Takinowaki *et al.*, 2006). The transcriptional regulatory element (Ada box) in the *ada* gene to which methylated Ada binds has been defined precisely, with the sequence 5′-AAAGCGCA-3′. Methylated Ada also displays binding affinity for the sequences 5′-AAANNAAGCGCA-3′ and 5′-AAT(N)6GCAA-3′ in the *alkA* and *aidB* promoter regions, respectively (Teo *et al.*, 1986; Nakamura *et al.*, 1988; Landini & Volkert, 1995).

Because the alkylation of Ada is irreversible, how is the adaptive response switched off? Several mechanisms have been proposed. One suggests that in the absence of alkylation adducts, the activated Ada is simply diluted by cell divisions (Lindahl *et al.*, 1988). Research has revealed that physiologically relevant high concentrations of unmethylated Ada are able to inhibit the activation of *ada* transcription by methylated Ada, both *in vitro* and *in vivo* (Saget & Walker, 1994). It is also possible that activated Ada is proteolytically degraded. The proteolytic cleavage of activated Ada in the hinge region linking the N-terminal to C-terminal domain of the protein might downregulate the expression of *ada*. Consistent with this model, a methylated 20 kDa Ada can bind to the *ada* promoter; however, it does not facilitate further binding of RNA polymerase to the promoter nor does it promote *ada* transcription *in vitro* (Akimaru *et al.*, 1990).

The expression of genes responding to oxidative DNA damage can be induced by the presence of reactive oxygen species (ROS). Two regulatory responses induced by ROS have been identified in *E. coli* – one controlled by *soxRS* and the other by *oxyR*. In the *soxRS* regulatory system, SoxR serves both as a sensor and as an activator. In the presence of ROS, SoxR forms 2Fe–2S centers, which convert SoxR to an active state. The activated SoxR induces transcription of *soxS*, a positive regulator that stimulates transcription of superoxide-responsive genes. Upon relief of oxidative stress, SoxR is rapidly converted to its transcriptionally inactive form, thus turning off the response (Wu & Weiss, 1992; Hidalgo & Demple, 1994; Hidalgo *et al.*, 1995; Ding *et al.*, 1996; Gaudu *et al.*, 1997; Volkert & Landini, 2001). The *oxyR* regulatory system is in response to hydrogen peroxide ($H_{2}O_{2}$). $H_{2}O_{2}$ activates the transcriptional activity of OxyR through formation of a disulfide bond between two of its cysteine residues. Activated OxyR induces the transcription of several oxidative stress genes (Zheng *et al.*, 1998; Aslund *et al.*, 1999; Volkert & Landini, 2001).

# Transcriptional responses to DNA damage in *S. cerevisiae*

From the above analyses, it appears that bacteria possess transcriptional responses to specific types of DNA lesions as well as general DNA damage, and that these regulatory pathways are fairly well understood. In contrast, DNA damage response in eukaryotes is apparently more complex than in prokaryotes. Rather than direct cleavage of transcription repressors, eukaryotes most likely use posttranslational modifications, such as phosphorylation and ubiquitination, and a sophisticated signal transduction cascade to achieve gene regulation in response to DNA damage. Here, we attempt to summarize the current knowledge of mechanisms of DNA damage response in *S. cerevisiae*, a unicellular lower eukaryotic model organism. Following a historic overview of identification and characterization of genes whose transcript levels are altered following treatment of cells with DNA-damaging agents, this review focuses on regulatory responses that are either common or unique to some well-studied genes.

# Genome-wide DNA damage-induced expression studies in yeast

Following work in prokaryotes revealing that DNA damage causes alterations in the transcriptional program of a cell, an important initiative was to identify a comparable response in the lower model eukaryote, *S. cerevisiae*. Although some of these experiments were performed one gene at a time, a precedent for systemic studies of transcription in bacteria (Kenyon & Walker, 1980) and the emergence of high-throughput technologies spurred the use of large-scale screens for such studies in yeast. To originally identify some of the genes that might be induced in response to DNA damage in *S. cerevisiae*, two early studies utilizing different genomic screens for DNA damage-induced genes are most notable. The first was based on a transcriptional fusion technique that is still widely used today for routine expression analysis. Using an experimental approach used previously in *E. coli* to screen for DNA damage-inducible genes (Kenyon & Walker, 1980), Szostak and colleagues tested

FEMS Microbiol Rev **32** (2008) 908–926

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reservedyeast genes as a random pool of *lacZ* fusions and screened for coordinately regulated DNA damage-inducible genes over various DNA damage treatments. From *c.* 8000 independent clones, a total of six damage-inducible (*DIN*) genes were identified, including one gene from a previous screen (Ruby *et al.*, 1983; Ruby & Szostak, 1985). Based on different expression profiles of *DIN* genes to specific DNA-damaging agents, the authors proposed at least two *DIN* classes. Such grouping of inducible genes is analogous to the clustering of large-scale data sets that is often necessary to manage and interpret modern DNA microarray experiments. Meanwhile, McClanahan & McEntee (1984) used a differential plaque hybridization strategy to identify genes whose transcript levels are altered after DNA damage treatment. Radiolabeled cDNA probes were generated from poly(A) mRNA harvested from control and treated cells, and subsequently used for differential plaque hybridization. By comparing the intensity of labeled probe bound between the control and the treated samples, differential expression of genes could be evaluated. Altogether, *c.* 9000 genomic clones were screened and four DNA damage-responsive (*DDR*) genes were identified. The experimental approach also enabled the identification of another set of genes, with decreased expression in response to DNA damage treatment. This latter finding testifies to the complexity of the DNA damage response in eukaryotes, because no such deactivation of genes was ever found in the *E. coli* SOS response (Walker, 1984).

These two early screens were significant to the field of transcriptional response to DNA damage in yeast. Not only were they among the first to demonstrate altered levels of transcripts in *S. cerevisiae* after DNA damage insult but also they were carried out in a genome-wide manner and foreshadowed some of the benefits and complications associated with analogous DNA microarray experiments that would follow. The powerful yet straightforward experimental global approaches also paid dividends by considerably expanding the number of genes known to be induced by DNA damage at the time. Together, 10 new DNA damage-inducible (*DIN/DDR*) and four DNA damage-repressive genes were identified. These studies allowed estimation of total DNA damage-responsive genes. Because the genomic library of Ruby and Szostak likely contained only *c.* 500 yeast genes representing about 8% of the genome, extrapolation led to an estimation of roughly 80 DNA damage-inducible genes in yeast. Significantly, the two independent screens yielded nonoverlapping data sets, indicating that neither screen was saturated and that the number of DNA damage-inducible genes was probably underestimated. Thus, even in the earliest stages of the studies of yeast DNA damage-induced gene expression, it was obvious that *S. cerevisiae* would have a much larger repertoire of DNA damage-induced genes than *E. coli*.

With the exception of these studies, the identification of DNA damage-inducible genes in the premicroarray era was performed on a gene-by-gene basis, and the entire complement of DNA damage-inducible genes grew very slowly. A comprehensive list of these genes, including relevant references, has been compiled and tabulated (Friedberg *et al.*, 1995) with references herein (McDonald *et al.*, 1997; Basrai *et al.*, 1999; Bennett, 1999; Brusky *et al.*, 2000; Fry *et al.*, 2003; Zaim *et al.*, 2005; Fu & Xiao, 2006).

### DNA microarray analyses in *S. cerevisiae*

Despite the above large-scale experimental approaches, a plausible high-throughput and truly global approach was not available until the emergence of genomic technologies. In particular, DNA microarrays have provided the majority of damage-induced gene expression data in budding yeast to date. This wealth of expression data provided by DNA microarrays has revealed a new realm for conceptualizing regulatory networks in a global manner. In the following paragraphs, we highlight only the first DNA microarray studies of DNA damage-induced expression in yeast, and note that there are numerous other significant studies that cannot be feasibly covered in this review. Table 1 is provided for an overview of genomic expression studies in *S. cerevisiae* focusing on DNA damage treatments or related gene mutants. We refer the reader to the Yeast Functional Genomics Database (http://yfgdb.princeton.edu/) for a navigable compilation of these and related studies, 93 of which are currently listed as genomic expression studies involving stress.

The premiere microarray study on DNA damage-induced expression in *S. cerevisiae* was published in 1999 (Jelinsky & Samson, 1999). Representative sequences of 6218 yeast ORFs were arrayed and cRNA probes were generated from untreated cultures or cultures exposed to the DNA-alkylating agent methyl methanesulfonate (MMS). As evidenced by the overlap with the previous independent data mentioned above, the results validated the use of DNA microarrays for damage-induced expression analysis. Of the genes already known to be induced by DNA-damaging agents by all other methods combined, 86% (18 out of 21) were identified in the DNA microarray experiment. The use of a single MMS dose and a fourfold minimum cut-off likely explain the omission of the three other previously identified inducible genes. Altogether, an astonishing 325 yeast genes (*c.* 5% of entire yeast genes) were found to be induced by MMS treatment, with a significant portion (112) representing uncharacterized ORFs. In addition, 76 ORFs had reduced expression upon MMS exposure. The reduced expression of some genes after DNA damage treatment is reminiscent of the early *DDR* screen (McClanahan & McEntee, 1984). Altogether, a single DNA microarray

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved  
DNA damage-induced transcription in budding yeast

Table 1. Genomic DNA microarray studies of transcriptional response to DNA-damaging agents and DNA repair-associated mutations in Saccharomyces cerevisiae

| References                     | Cell type | Mutant    | Treatment                                                                 |
|--------------------------------|-----------|------------|-------------------------------------------------------------------------|
| Jelinsky & Samson (1999)       | Haploid   |            | MMS                                                                     |
| Jelinsky et al. (2000)         | Haploid   |            | MMS, MNNG, BCNU, t-BuOOH, 4-NQO, IR, MMC                                   |
|                                | Haploid   | rpn4       | MMS                                                                     |
| Gasch et al. (2000)            | Haploid   |            | H₂O₂                                                                    |
| Gasch et al. (2001)            | Haploid   |            | MMS, IR, H₂O₂                                                           |
|                                | Haploid   | dun1       | MMS                                                                     |
|                                | Haploid   | mec1       | MMS, IR                                                                 |
|                                | Haploid   | crt1       | NA                                                                      |
| De Sanctis et al. (2001)       | Haploid   | rad53, rad6| IR                                                                      |
| Oshiro et al. (2002)           | Haploid   |            | MMS, HU, UV                                                             |
| Ostapenko & Solomon (2003)     | Haploid   | ctk1       | HU                                                                      |
| Green & Johnson (2004)         | Haploid   | tup1       | NA                                                                      |
| van Attikum et al. (2004)      | Haploid   | ino80, arp8| MMS                                                                     |
| Keller-Seitz et al. (2004)     | Diploid   | mec1       | Aflatoxin B₁ (N⁷-guanine DNA adducts)                                     |
| Mercier et al. (2005)          | All types |            | IR                                                                      |
| John et al. (2005)             | Haploid   |            | Cigarette smoke extract                                                 |
| Benton et al. (2006)           | Haploid   |            | MMS, IR                                                                 |
| Guo et al. (2006)              | Diploid   |            | Aflatoxin B₁                                                            |
| Kelly et al. (2006)            | Haploid   |            | Azinomycin B (interstrand crosslinks and alkylating agent)                 |
| Marques et al. (2006)          | Haploid   |            | H₂O₂                                                                    |
| Shenton et al. (2006)          | Haploid   |            | H₂O₂                                                                    |
| Workman et al. (2006)          | Haploid   | 30 TFs     | MMS                                                                     |
| Kugou et al. (2007)            | Diploid   | mre11, rad50| NA                                                                      |
| Fu et al. (2008)               | Haploid   | rad6, rad18| MMS                                                                     |

MMS, methyl methanesulfonate; MNNG, N-methyl-N′-nitro-N-nitrosoguanidine; BCNU, 1,3-bis(2-chloroethyl)-1-nitrosourea; t-BuOOH, tert-butyl hydroperoxide; 4-NQO, 4-nitroquinoline-*n*-oxide; IR, ionizing radiation; MMC, mitomycin C; HU, hydroxyurea; UV, ultraviolet irradiation. NA, not applicable.

experiment augmented the complement of known DNA damage-induced genes in *S. cerevisiae* by over 20-fold. Despite this wealth of new data, however, meaningful conclusions regarding transcriptional regulation were not possible with such an abundance of raw, unorganized data.

A follow-up study (Jelinsky et al., 2000) provided a more comprehensive analysis of transcriptional response to DNA damage and demonstrated how large amounts of microarray data can be manageably interpreted. This enabled the use of a large dataset to address a specific hypothesis, namely that

MAG1 belongs to a novel regulatory network. The DNA microarray data were computationally assembled into self-organizing maps (SOMs) (Tamayo et al., 1999) that group genes with similar expression profiles. Using SOMs, 18 different groups of genes were coregulated across various experimental conditions. Of the genes in the SOM containing MAG1, it was noted that most contained an upstream repressor sequence 2 element (URS2) previously identified upstream of several genes involved in DNA repair and metabolism (Xiao et al., 1993; Singh & Samson, 1995). It was thus hypothesized and resolved that the genes within the MAG1 SOM were regulated at the transcriptional level by the same repressor element and constitute a regulatory network. This study identified Rpn4 as a putative transcriptional regulator of a large number of genes in response to DNA damage. The fact that Rpn4 serves as a 26S proteasome-associated protein as well as a transcriptional factor that regulates at least 26 out of 32 proteasomal genes (Mannhaupt et al., 1999) points to a possibility that ubiquitin-mediated proteolysis may play a role in the transcriptional response to DNA damage. Other studies on the transcriptional response to DNA damage in yeast using DNA microarray experiments have successfully applied similar methods of data management to test certain hypotheses (see Table 1).

### DNA damage checkpoint pathways and transcriptional regulation

Both individual gene analyses and global transcriptional studies in budding yeast indicate that many genes are coordinately regulated in response to DNA damage. Although transcriptional responses to specific damaging agents result in distinct and signature profiles, the microarray data point to a general stress response pathway that controls transcription. These observations are consistent with previous reports based on individual gene analyses that DNA damage and replication checkpoints are involved in a transcriptional response to DNA damage.

The DNA damage checkpoint mutants were first isolated by their failure to delay cell-cycle progression into mitosis after irradiation with X-rays (Weinert & Hartwell, 1988; Rowley et al., 1992; Weinert et al., 1994). These mutants displayed increased radiation sensitivity, which could be largely negated through reimposing an artificial arrest before the M phase by an antitubulin agent. Hence, DNA damage checkpoint was initially defined as a nonessential and reversible response that slows down or arrests cell-cycle progression in response to DNA damage, allowing time for DNA repair (Hartwell & Weinert, 1989). At present, about 20 genes in budding yeast have been identified or anticipated to be involved in DNA damage checkpoints (Table 2) (Elledge, 1996; Nyberg et al., 2002; Friedberg et al., 2006).

Table 2. DNA damage sensors and signal transducers in Saccharomyces cerevisiae and their Schizosaccharomyces pombe and mammalian orthologs

| Category          | Protein function           | S. cerevisiae   | S. pombe    | Mammals       |
|-------------------|----------------------------|-----------------|-------------|---------------|
| Damage sensors    | RFC-like clamp loader      | Rad 24          | Rad 17      | RAD 17        |
|                   | Clamp loader               | Rfc 2-5         | Rfc 2-5     | RFC 2-5       |
|                   | PCNA-like clamp            | Ddc 1           | Rad 9       | RAD 9         |
|                   |                            | Rad 17          | Rad 1       | RAD 1         |
|                   |                            | Mec 3           | Hus 1       | HUS 1         |
| S-phase sensors   | Pol ε                     | Pol 2/Dun 2     | Cdc 20      | Pol ε        |
|                   |                            | Dpb 11          | Cut 5/Rad 4 | TopBP 1       |
|                   |                            | Drc 1/Sld 2     | Drc 1       | Unknown       |
|                   | DNA helicase               | Sgs 1           | Rqh 1       | WRN, BLM, RTS |
| Sensor/transducer | PI3K-like kinases          | Mec 1           | Rad 3       | ATR           |
|                   | ATR partner                | Lcd 1/Ddc 2     | Rad 26      | ATRIP         |
| Transducers       | PI3K-like kinases          | Tel 1           | Tel 1       | ATM           |
|                   | Damage mediator            | Rad 9           | Crb 2       | BRCA 1, MDC 1, 53BP 1 |
|                   | S-phase mediator           | Mrc 1           | Mrc 1       | Claspin       |
|                   |                            | Tof 1           | Swi 1       | Unknown       |
|                   |                            | Csm 3           | Swi 3       | Unknown       |
| Kinase            |                            | Rad 53          | Cds 1       | CHK 2         |
|                   |                            | Chk 1           | Chk 1       | CHK 1         |
|                   |                            | Pds 1           | Cut 2       | PTTG          |
|                   | Effector kinase            | Dun 1           | Unknown     | Unknown       |

These genes are required for different phases of the cell cycle and for different types of lesions. Because inactivation of damage checkpoint genes affects the transcriptional response to DNA damage as well as other cell fates, it has been well accepted that checkpoint pathways play critical roles in addition to cell-cycle arrest (Zhou & Elledge, 2000).

It is now clear that the checkpoint pathway comprises a subroutine integrated into the larger DNA damage response that regulates multifaceted responses (Zhou & Elledge, 2000). Besides arresting cell-cycle progression, the damage checkpoint pathway has been shown to promote DNA repair (Mills et al., 1999), control telomere length (Ritchie et al., 1999), activate transcription (Elledge, 1996) and trigger apoptosis in metazoan cells (Roos & Kaina, 2006). In a broad sense, the damage checkpoint coordinately regulates DNA damage responses including transcriptional regulation.

Based on the data from previous research, a putative model is proposed for the signal transduction pathways in response to DNA damage, consisting of sensors, transducers (including mediators) and effectors (Bachant & Elledge, 1998). The sensors are proteins that initially sense the damaged DNA and initiate the signaling response. Transducers can be activated by the DNA damage signal passed down from the sensor's and then amplify and relay the signal to the downstream effectors. The ultimate effectors are defined as proteins that execute the cellular response. In the case of transcriptional regulation, the effectors are likely to be transcription factors that recognize cis-acting elements in the promoter of DNA damage-inducible genes.

DNA damage sensors in S. cerevisiae

Because of the complexity of the source and type of DNA damage, as well as the different cell-cycle stages at which damage may occur, DNA damage sensors and their intrinsic interactions have not been well established. Much of our knowledge comes from studies with damage and replication checkpoints.

The budding yeast proliferating cell nuclear antigen (PCNA)-like 9-1-1 complex composed of Rad 17, Ddc 1 and Mec 3 is loaded on the DNA damage site with the assistance of the clamp loader Rad 24-Rfc complex, which leads to activation of the main damage response kinase Mec 1 (Kondo et al., 1999; Rouse & Jackson, 2002; Majka & Burgers, 2003; Majka et al., 2006). Therefore, Rad 24-Rfc and the 9-1-1 complexes together have been speculated to be sensors. Mec 1 forms a stable complex with Lcd 1/Ddc 2, and this complex itself has been anticipated to be a sensor due to the ability of Ddc 2 to bind to ends of single- and double-stranded oligonucleotides in vitro and to recruit Mec 1 to double-strand break (DSB) sites in vivo in a manner independent of the 9-1-1 complex (Kondo et al., 2001; Melo et al., 2001; Rouse & Jackson, 2002; Niida & Nakanishi, 2006). Another sensor candidate is Rad 9. Rad 9 is required for the activation of DNA damage checkpoint pathways in budding yeast, is phosphorylated after DNA damage in a Mec 1-and Tel 1-dependent manner and subsequently interacts with the downstream kinase Rad 53 (Naiki et al., 2004). Rad 9 displays the ability to associate with DSBs and controls the DNA damage-specific induction of some repair,
DNA damage-induced transcription in budding yeast

replication and recombination genes (Aboussekhra *et al.*, 1996; Naiki *et al.*, 2004). Furthermore, Rad9 is thought to act in a pathway distinct from that of Rad24/9-1-1 for damage checkpoint (Lydall & Weinert, 1995) and transcriptional response (de la Torre-Ruiz *et al.*, 1998), suggesting that Rad9 may serve as an alternative DNA damage sensor. The above candidate sensors primarily respond to DNA damage caused by UV, ionizing radiations and endogenous DSBs that induce G1/S and G2/M checkpoints. They also respond to excessive telomere ssDNA in the *cdc13-1* mutant (Lydall & Weinert, 1995) and the accumulation of DSBs due to a defective Cdc9 DNA ligase (Weinert & Hartwell, 1993), both inducing the G2/M checkpoint.

The intra-S checkpoint is experimentally activated by modest doses of alkylating agents, such as MMS, that cause replication-blocking lesions. In addition, inhibiting replication by means other than DNA damage, such as treatment with the ribonucleotide reductase (Rnr) inhibitor hydroxyurea, can activate similar cellular responses. Allele-specific mutants in the catalytic subunit of DNA polymerase ε (Pol2) are defective in MMS- and hydroxyurea-induced *RNR3* (Navas *et al.*, 1995) and *MAG1* (Zhu & Xiao, 1998) expression. Hence, Pol2 and its interacting partners Dpb11 and Drc1 may serve as S-phase sensors. It is of great interest to note that Dpb11 has been reported to interact with the Ddc1 subunit of 9-1-1 (Wang & Elledge, 2002), and that physical interactions between Polε and the human homologs of Rad24/9-1-1 were also reported (Makiniemi *et al.*, 2001; Post *et al.*, 2003), indicating that the damage signal may be relayed between sensors. Recently, the helicase Sgs1, which is a member of the *E. coli* RecQ helicase subfamily that includes mammalian homologs of human BLM, WRN and RTS, responsible for the Bloom, Werner and a subset of Rothmund–Thomson syndromes, respectively, was proposed as a candidate sensor. Sgs1 possesses ATPase activity and preferentially binds to branched DNA substrates (Bennett *et al.*, 1998, 1999). Sgs1 might be involved in the creation of the signal for checkpoint activation, perhaps by resolving aberrantly paired double helices (Cobb *et al.*, 2002). It functions in the same epistasis group as Polε to activate Rad53 in the presence of hydroxyurea, and this signaling pathway acts in parallel to that of Rad17 and Rad24 (Navas *et al.*, 1995; Frei & Gasser, 2000). However, it remains unclear whether putative S-phase-specific sensors like Polε and Sgs1 recognize a subset of DNA damage or their activities lead to common DNA damage intermediates such as ssDNA (like RecA in the SOS response).

The above checkpoint proteins are placed in the sensor class primarily based on genetic and biochemical properties or inference, as little direct biochemical evidence yet exists to support their sensor role. In principle, checkpoint sensors are capable of directly recognizing DNA lesions or interacting with pre-existing DNA damage repair proteins specific for individual lesions. In this context, DNA repair proteins may serve as damage sensors for both checkpoint and/or transcriptional responses. A good example is the interaction between NER and DNA damage checkpoint in response to UV treatment. A genetic screen for mutants defective in the activation of checkpoint following UV lesions but not other types of damage resulted in the identification of *RAD14*, which is absolutely required for the UV-induced damage checkpoint. Interestingly, the checkpoint activation requires processing of UV lesions by the NER complex instead of mere lesion recognition by Rad14, and Rad14 physically interacts with Ddc1 (Giannattasio *et al.*, 2004). Similarly, the Mre11/Rad50/Nbs1 (MRN) DSB-binding complex is required for the activation of ATM kinase in mammalian cells (Lee & Paull, 2005), although a similar function has not been reported for the corresponding Mre11/Rad50/Xrs2 (MRX) complex in budding yeast. In both the above cases, processing of UV-induced lesions by NER and DSBs by MRN/MRX results in ssDNA regions that may serve as an ultimate damage signal.

### DNA damage transducers in *S. cerevisiae*

Considerable efforts have been made to understand signal transducers in the damage and replication checkpoint pathways, and the same signal transduction cascade likely functions in the transcriptional response to DNA damage. Most of the transducers are protein kinases that, once activated by the DNA damage signal passed down from the sensors, amplify and relay the signal to the downstream effectors. In *S. cerevisiae*, checkpoint protein kinases Mec1, Rad53 and Dun1 are necessary for the transcriptional response of most, if not all, genes to DNA damage, and they appear to be central transducers in the regulation network (Zhou & Elledge, 1993; Allen *et al.*, 1994; Kiser & Weinert, 1996; Gasch *et al.*, 2001). The initiator of this signal transduction kinase cascade in budding yeast appears to be Mec1 and its regulatory subunit Lcd1/Ddc2, which is a serine/threonine protein kinase belonging to the phosphatidylinositol-3-kinase (PI3K) family. Mec1 is required for the activation of Rad53 through phosphorylation of consensus PI3K sites within Rad53 (Pellicioli & Foiani, 2005; Ma *et al.*, 2006). However, efficient and direct phosphorylation of Rad53 by Mec1 is only observed in the presence of Rad9 *in vitro*, and the stimulatory activity of Rad9 requires both phospho- and FHA-dependent interaction with Rad53, which allows Rad53 to be recognized as a substrate for Mec1 (Sweeney *et al.*, 2005). Hence, Rad9 is defined as a checkpoint mediator (or adaptor) that serves to facilitate and amplify signals. Rad9 functions primarily at G1/S and G2/M, while Mrc1 and its mammalian homolog claspin has been proposed to function as an S-phase or a replication checkpoint mediator. Mrc1 and Tof1 form a stable replication-pausing complex at the stalled replication fork,

FEMS Microbiol Rev **32** (2008) 908–926

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved
which is required to anchor subsequent DNA repair events (Katou *et al.*, 2003).

Activation of Rad53 is an essential intermediary step in yeast DNA damage responses that include delaying cell-cycle progression, promoting repair processes, stabilizing stalled replication forks and regulating transcription (Branzei & Foiani, 2006). The kinase Dun1 is one of the identified targets of Rad53, and the activation of Dun1 requires phosphorylation by Rad53 (Chen *et al.*, 2007). Dun1 was originally identified as a DNA damage-uninducible (*dun*) mutant defective in the transcriptional activation of genes encoding Rnr in response to DNA damage (Zhou & Elledge, 1993), and was subsequently shown to be required for the induction of other DNA damage-inducible genes (Basrai *et al.*, 1999; Zhu & Xiao, 2001; Fu & Xiao, 2006). Genomic expression research showed that deletion of *DUN1* affected the expression of > 1000 genes in response to MMS, and the response in the *dun1Δ* mutant is largely the same as the response seen in the *mec1* mutant (Gasch *et al.*, 2001), suggesting that most of the Mec1-dependent effects on genomic expression are mediated by the downstream Dun1 kinase. The mechanism used by Dun1 for DNA damage induction appears to be rather diverse and requires further investigation.

In response to DNA damage, the primary mechanism for transcriptional regulation is likely to be at the stage of transcription initiation. The downstream effectors are therefore expected to be transcription factors that directly influence transcription initiation. However, unlike damage checkpoint transducers that appear to be aligned in a linear cascade and control most if not all DNA damage-inducible genes, the downstream effectors are clearly diverse and dedicated to a particular gene or a group of genes (Fig. 1). In the next section, selected examples are presented to provide insights into the molecular mechanisms of DNA damage-induced transcriptional response and their interaction with the checkpoint cascade.

### Mechanisms of transcriptional regulation of yeast genes in response to DNA damage

#### RNR genes

Regulation of RNR genes is the best-known example of a eukaryotic transcriptional response to DNA damage to date. Rnr is an enzyme that converts nucleoside diphosphates (NDPs) into deoxynucleotide diphosphates (dNDPs), which represents the rate-limiting step in the production of the four dNTPs for DNA synthesis and repair (Reichard, 1988; Elledge *et al.*, 1993; Jordan & Reichard, 1998). Altered levels or an imbalance of dNTP pools can lead to a higher rate of spontaneous mutagenesis or cell death. Rnr is an

![Diagram](attachment:DNA_damage_induced_transcription_model.png)

**Fig. 1.** A working model of DNA damage-induced transcription in budding yeast. In budding yeast, DNA damage-induced transcription is controlled by signal transduction pathways including sensors, transducers and effectors. Mec1, Rad53 and Dun1 form the central kinase cascade to regulate the DNA damage-induced transcription of most genes examined. The phosphorylation of *trans*-acting factors (effectors) may change their affinity for the corresponding *cis*-acting elements in the promoter region of target genes, thus activating their transcription. Only three sets of well-characterized genes are depicted. Solid ovals indicate protein kinases. Dotted lines and question mark indicate that the molecular mechanism of this step(s) remains unknown. Note that the signal-sensor-transducer cascades are based on studies with cell-cycle checkpoints and have not been extensively examined with transcriptional regulation.

α₂β₂ tetramer, and four genes, *RNR1-4*, encode the subunits of budding yeast Rnr: *RNR1* and *RNR3* encode α (large) subunits, while *RNR2* and *RNR4* encode β (small) subunits. Expression of all four genes is inducible at the transcriptional level by a variety of DNA-damaging agents (Elledge & Davis, 1987, 1989, 1990; Huang & Elledge, 1997), among which *RNR3* exhibits the highest level of induction, and its transcriptional regulation has been examined most extensively.

The transcriptional level of the *RNR3* gene is very low under normal conditions. However, when treating yeast cells with DNA-damaging agents such as UV, MMS or 4-nitroquinoline-1-oxide (4-NQO), the transcript level of *RNR3* can be increased 100–500-fold (Elledge & Davis, 1990). In order to determine the mechanisms of induction, a series of mutants have been isolated that cause constitutive expression of *RNR3* (*crt* mutants) (Zhou & Elledge, 1992). These negative regulators of *RNR3* expression are divided into two groups: indirect regulators that result in endogenous DNA damage or a state of metabolic stress such as nucleotide depletion that results in the upregulation of *RNR3*, and

DNA damage-induced transcription in budding yeast

![Diagram](#)

Fig. 2. Summary of the transcriptional regulation of *RNR3*. In response to DNA damage or stalled replication, activation of the Mec1-Rad53-Dun1 kinase cascade results in phosphorylation of the repressor protein Crt1, and phosphorylated Crt1 loses its binding affinity for the *RNR3* promoter. With the assistance of Wtm1, Rpd3 and Hos2, the transcription of *RNR3* is highly activated. The transcription of *CRT1* is negatively regulated by itself and positively regulated by Crt10. Ccr4 influences Crt1 protein abundance by controlling the *CRT1* mRNA stability.

direct regulators involved in the regulatory pathway, including transcription factors. The *CRT1*, *TUP1* (*CRT4*) and *SSN6* (*CRT8*) genes encode negative regulators that bind to the *RNR3* promoter. A second screen was carried out for the mutations that disrupt the ability of DNA damage to induce transcription of *RNR3*, and the corresponding genes were designated *DUN* for DNA damage uninducible (Zhou & Elledge, 1993). The nonessential serine/threonine protein kinase Dun1 was isolated through this screen. Genetic analysis of *crt1*, *tup1* and *ssn6* showed that these mutations were epistatic to *dun1*, providing a strong genetic verification that *CRT1*, *TUP1* and *SSN6* function downstream of *DUN1* (Huang et al., 1998). Combined with observations that the Dun1 upstream kinases Mec1 and Rad53 are also essential for the DNA damage-induced transcription of *RNR3* (Huang et al., 1998), the signal transduction pathway for *RNR3* becomes conceivable (Fig. 2).

It is now clear that in response to DNA damage or replication blocks, the Rad53 protein kinase is activated in a Mec1-dependent manner, and activated Rad53 further phosphorylates the protein kinase Dun1. The Mec1-Rad53-Dun1 kinase cascade culminates in the phosphorylation of Crt1. Crt1 is a DNA-binding protein that recognizes a 13-bp consensus sequence termed the X box. Multiple X box-related sequences of different strength can be found in the promoter region of all four *RNR* genes. Crt1 is able to recruit the Tup1–Ssn6 general repressor complex to suppress the transcription of *RNR* genes. In the presence of DNA damage or replication blocks, Crt1 is phosphorylated in a Dun1-dependent manner and this phosphorylation abolishes its ability to bind X boxes, leading to the transcriptional induction of *RNR* genes (Huang et al., 1998).

Surprisingly, multiple X boxes were also identified in the *CRT1* promoter. Data from mutated X boxes showed that they confer *CRT1*-dependent repression on *CRT1* itself (Huang et al., 1998). The expression level of *CRT1* is very low under normal growth conditions but is inducible by DNA-damaging agents. Interestingly, the X boxes in the *CRT1* promoter consist of one with a weak affinity for Crt1 and one with a strong affinity. Thus, it is speculated that a weak induction of *CRT1* occurs immediately following DNA damage, which provides a buffer against spurious transcriptional activation of the pathway. Delaying full activation may ensure a rapid restoration of the basal repressed state when the DNA damage is repaired (Huang et al., 1998), which may serve as an autonomous regulatory circuit (Fig. 2). This mechanism is reminiscent of the autonomous regulation of the LexA repressor in the *E. coli* SOS response. Crt1 mediates repression by recruiting the general repressor Tup1–Ssn6 complex to *RNR* promoters via its

FEMS Microbiol Rev 32 (2008) 908–926

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved
N-terminus (Huang *et al.*, 1998; Li & Reese, 2000). Tup1-Ssn6 recruitment establishes a nucleosomal array over the promoter of *RNR3* with a positioned nucleosome occupying the TATA box to block access by the general transcriptional machinery (Li & Reese, 2000, 2001; Sharma *et al.*, 2003). The derepression of *RNR3* correlates with the disruption of the nucleosome position. In response to DNA damage signals, hyperphosphorylated Crt1 loses the ability to bind to the *RNR3* promoter, and the Tup1–Ssn6 complex is not recruited to the promoter region. Consequently, the chromatin structure is remodeled and thus increases the accessibility of DNA to transcription factors. The chromatin remodeling at the *RNR3* promoter requires a number of general transcriptional factors, such as TBP-associated factors (TAF~II~), and RNA polymerase II. Furthermore, the remodeling is also dependent on the SWI/SNF complex, which possesses a DNA-stimulated ATPase activity and can destabilize histone–DNA interactions in an ATP-dependent manner (Sharma *et al.*, 2003). The preinitiation complex components TFIID and RNA polymerase II aid in recruitment and retention of the SWI/SNF complex to the *RNR3* promoter.

Recently, it was reported that the expression of *CRT1* is also controlled by the novel regulator Crt10 (Fu & Xiao, 2006) (Fig. 2). *CRT10* was identified through screening of the *S. cerevisiae* deletion strains for hydroxyurea resistance. Epistatic analysis indicates that *CRT10* belongs to the *CRT1* pathway. Deletion of *CRT10* does not affect the transcript level of *TUP1* or *SSN6* regardless of hydroxyurea treatment, but significantly reduces the basal level as well as hydroxyurea-induced expression of *CRT1*; thus, *CRT10* appears to be a positive regulator of *CRT1* transcription (Fu & Xiao, 2006). Furthermore, the *dun1* mutation is epistatic to *crt10* with respect to both hydroxyurea sensitivity and *RNR* gene expression. Interestingly, the expression of *CRT10* itself is induced by DNA-damaging agents and this induction requires *DUN1* (Fu & Xiao, 2006). The increased Crt10 activity may be required to bring Rnr activity back to a normal level through increasing the expression of repressor Crt1 once the DNA damage is repaired. Like *CRT1*, the induction of *CRT10* itself depends on *DUN1*, suggesting that Crt10 functions downstream of Dun1 and forms another component of the autoregulatory circuit to regulate the expression of *RNR* genes (Fu & Xiao, 2006).

CCR4 encodes a component of the major cytoplasmic deadenylase, which is involved in mRNA poly(A) tail shortening (Tucker *et al.*, 2001). Cells defective in *CCR4* display particular sensitivity to the Rnr inhibitor hydroxyurea (Woolstencroft *et al.*, 2006). The *ccr4 dun1* double mutants exhibit synergistic sensitivity to hydroxyurea, and simultaneous overexpression of *RNR2, RNR3* and *RNR4* partially rescues the hydroxyurea hypersensitivity of a *ccr4 dun1* strain, implying that *CCR4* and *DUN1* function in different pathways to regulate the activity of Rnr. Deletion of *CRT1*

suppresses hydroxyurea sensitivity of the *ccr4* mutant, and overexpression of *CRT1* hypersensitizes *ccr4* to hydroxyurea. These observations lead to the conclusion that Ccr4 regulates *CRT1* mRNA poly(A) tail length and may thus subtly influence the Crt1 protein abundance (Woolstencroft *et al.*, 2006) (Fig. 2).

More regulatory factors were found to be involved in the transcriptional regulation of *RNR* genes. Wtm1 and Wtm2 have been reported to modulate the expression of *RNR3* (Tringe *et al.*, 2006). Moderate overexpression of both genes or high-level expression of *WTM2* alone upregulates *RNR3-lacZ* in the absence of DNA damage. In response to hydroxyurea and γ-rays, the expression level of *RNR3* attenuated 45% in *wtm2Δ* mutants, but not in *wtm1* mutants. Wtm2 was found to associate directly with the *RNR3* promoter, and the association correlates with its ability to increase constitutive *RNR3* expression. It remains unknown how Wtm2 increases *RNR3* transcription, although some observations hint that Wtm2 might enhance *RNR3* transcription by participating in chromatin remodeling (Tringe *et al.*, 2006). Furthermore, Wtm1 associates with and anchors the Rnr2/Rnr4 complex in the nucleus in untreated cells. In response to DNA damage or replication inhibition, the interaction between Wtm1 and Rnr2/Rnr4 is disrupted and this complex is released from the nucleus to the cytoplasm (Lee & Elledge, 2006; Zhang *et al.*, 2006). A recent report shows that two histone deacetylases (HDAC), Rpd3 and Hos2, are required for the transcriptional activation of *RNR3* in response to DNA damage (Sharma *et al.*, 2007) (Fig. 2). Although a direct association between Hos2 and the *RNR3* promoter has not been observed, Rpd3 was found to specifically bind to X boxes in the *RNR3* promoter region in a Tup1- or a Crt1-independent manner. The HDAC activity of Rpd3 and Hos2 is essential for the activation of *RNR3*. The observation that the recruitment of RNA polymerase II is dramatically reduced in the *rpd3 hos2* mutant indicates that Rpd3 and Hos2 activate the transcription by regulating the assembly of the preinitiation complex or inducing multiple rounds of RNA polymerase recruitment (Sharma *et al.*, 2007).

### PHR1

PHR1 encodes a photolyase that specifically and exclusively repairs pyrimidine dimers, which are the most abundant lesions found in DNA following UV irradiation (Sancar, 1985). Interestingly, the transcription of *PHR1* is induced by various DNA-damaging agents such as MMS, MNNG, UV irradiation, 4-NQO or γ-ray (Robinson *et al.*, 1986; Sebastian *et al.*, 1990) despite the fact that Phr1 only reverses UV-induced pyrimidine dimers but not lesions induced by other DNA-damaging agents. Three transcriptional regulatory elements have been defined within the *PHR1* promoterDNA damage-induced transcription in budding yeast

region: an upstream activation sequence (UAS), an upstream repression sequence (URS) and an upstream essential sequence (UES) (Sancar *et al.*, 1995). A 22-bp interrupted palindrome comprises UAS${}_{PHR1}$, and it is responsible for 80–90% of basal and induced expression. It alone can activate transcription of a CYC1 minimal promoter but does not confer damage responsiveness (Sancar *et al.*, 1995). URS${}_{PHR1}$ is defined to a 39-bp region that includes a 22-bp palindrome. Deletions or specific mutations within URS${}_{PHR1}$ increase basal-level expression but decrease the induction ratio. It functions as a strong URS and confers a low-level damage inducibility when placed in the context of a heterologous gene (Sancar *et al.*, 1995). The UES${}_{PHR1}$ is required for efficient derepression when URS${}_{PHR1}$ is present. Deletion of URS${}_{PHR1}$ also eliminates the requirement for UES${}_{PHR1}$ for transcriptional activation (Sancar *et al.*, 1995).

Three proteins have been identified that regulate the expression of *PHR1* by binding to the upstream regulatory elements. Ume6 is a bifunctional transcriptional regulator involved in several metabolic pathways (Strich *et al.*, 1994). It is a positive regulator of *PHR1* transcription and binds specifically to the UAS${}_{PHR1}$ (Sweet *et al.*, 1997). Multiple copies of Ume6 enhance the expression of *PHR1*; however, deletion of *UME6* reduces the expression of *PHR1* during vegetative growth, but only at a distinct cell-cycle phase (Sweet *et al.*, 1997; Sancar, 2000). Rph1 and Gis1, which are 35% identical to each other at the amino acid sequence level, are two DNA damage-responsive repressors of *PHR1* transcription (Jang *et al.*, 1999; Sancar, 2000). Both Rph1 and Gis1 contain two putative zinc fingers that are > 90% identical overall and identical in the DNA-binding loop, and they regulate the transcriptional response of *PHR1* through binding to URS${}_{PHR1}$. Deletion of both *RPH1* and *GIS1* is required to fully derepress *PHR1* in the absence of damage, suggesting that they are functionally redundant. *In vitro* footprinting and binding competition studies indicate that the sequence AG${}_{4}$ (C${}_{4}$T) within the URS${}_{PHR1}$ is the binding site for Rph1 and Gis1 (Jang *et al.*, 1999).

Induction of *PHR1* is controlled by the DNA damage signal transduction pathway. The serine and threonine residues of Rph1 can be phosphorylated, and the phosphorylation of Rph1 is increased in response to DNA damage. The DNA damage-induced Rph1 phosphorylation requires DNA damage checkpoint proteins Rad9, Rad17, Mec1 and Rad53, indicating that the phosphorylation of Rph1 is under the control of the Mec1-Rad53 DNA damage checkpoint pathway (Kim *et al.*, 2002). On the other hand, deletion of *DUN1*, *TEL1* or *CHK1* does not affect the phosphorylation of Rph1, indicating that *PHR1* is regulated by a potentially novel damage checkpoint that is distinct from the Mec1-Rad53-Dun1 protein kinase cascade. Based on the results of a coimmunoprecipitation assay, Rad53 does not appear to interact physically with Rph1, indicating the existence of a yet unknown kinase(s) in the Mec1-Rad53-Rph1 pathway (Kim *et al.*, 2002).

### MAG1 and DDI1

MAG1 encodes a 3-methyladenine DNA glycosylase that initiates the base excision repair pathway by removing lethal lesions such as 3-methyladenine (Chen *et al.*, 1989). MAG1 is not only induced by DNA-alkylating agents such as MMS but also by UV, 4-NQO or hydroxyurea (Chen *et al.*, 1990; Chen & Samson, 1991; Xiao *et al.*, 1993). By analyzing the DNA sequence immediately upstream of *MAG1*, another DNA damage-inducible gene, named *DDI1* for DNA Damage Inducible, was identified (Liu & Xiao, 1997). Similar to *MAG1*, *DDI1* is also induced by MMS, UV, 4-NQO and hydroxyurea. Furthermore, both genes require a similar dosage for maximum induction, and the induction profile is similar (Liu & Xiao, 1997).

*MAG1* and *DDI1* lie in a head-to-head configuration and are transcribed divergently. The expression of both *MAG1* and *DDI1* is controlled by common as well as distinct UAS and URS elements, possibly through antagonistic mechanisms (Liu & Xiao, 1997). A UAS${}_{MAG1}$ and a URS${}_{MAG1}$ have been identified in the promoter region of *MAG1* (Xiao *et al.*, 1993). Interestingly, fine mapping of the UAS${}_{MAG1}$ sequence reveals that it is located within the *DDI1* protein coding region, and the electrophoretic mobility shift assay using labeled UAS${}_{MAG1}$ as a probe detected sequence-specific binding protein(s) (Liu & Xiao, 1997), although the identity of the UAS${}_{MAG1}$-binding protein has not been reported. The expression of *DDI1* is negatively regulated by a URS${}_{DDI1}$ in its promoter region (Liu & Xiao, 1997). The intergenic region between *MAG1* and *DDI1* also contains a *cis*-acting element that coregulates the expression of both genes. In this shared promoter region, UAS${}_{DM}$, which contains two 8-bp tandem repeat sequences 5′-GGTGGCGA-3′, is required for the bidirectional expression of *MAG1* and *DDI1*; deletion or point mutations of this tandem repeat result in a reduced basal level expression and significant reduction of damage-induced expression (Liu & Xiao, 1997). Furthermore, UAS${}_{DM}$ alone is able to confer a DNA damage response when fused to a *CYC1* promoter (Liu & Xiao, 1997), indicating that this *cis*-acting element independently interacts with a transcriptional factor(s). With a yeast one-hybrid screen using UAS${}_{DM}$ as a bait, a transcriptional activator called Pdr3 was isolated (Zhu & Xiao, 2004). Pdr3 binds to UAS${}_{DM}$ *in vivo* and *in vitro*, and deletion of *PDR3* reduced both the basal level and the DNA damage-induced expression of *MAG1* and *DDI1*. In addition, deletion of *PDR3* does not further affect *MAG1* and *DDI1* expression if UAS${}_{DM}$ is deleted, indicating that UAS${}_{DM}$ is indeed the target for Pdr3 activation (Zhu & Xiao, 2004). Another transcriptional activator, Rpn4, was shown to be required for *MAG1*

FEMS Microbiol Rev **32** (2008) 908–926

© 2008 Federation of European Microbiological Societies Published by Blackwell Publishing Ltd. All rights reserved(Jelinsky *et al.*, 2000) and *DDI1* (Zhu & Xiao, 2004) expression; however, Rpn4 does not appear to bind UAS${}_{DM}$. Moreover, deletion of *RPN4* does not alter *MAG1* and *DDI1* expression in the *pdr3* mutant cells, suggesting that *RPN4* acts upstream of *PDR3* (Zhu & Xiao, 2004). Deletion of *PDR3* and *RPN4* has no effect on the basal level or the DNA damage-induced expression of *PHR1*, *RNR2* or *RNR3*. Meanwhile, Crt1 and Tup1/Ssn6, the repressors of the *RNR* genes, and Rph1 and Gis1, the repressors of *PHR1*, are not involved in the control of *MAG1* expression (Zhu & Xiao, 2001). Hence, it becomes apparent that all three sets of well-studied yeast damage-inducible genes (*RNR*, *PHR1* and *MAG1-DDI1*) have distinct regulators and that the regulatory mechanisms are also different from each other.

The expression of *MAG1* and *DDI1* is also controlled by the DNA damage checkpoint. Mutation of *POL2*, *MEC1* or *DUN1* reduces the DNA damage response of *MAG1* (Zhu & Xiao, 1998, 2001), suggesting that *MAG1* is regulated by the *POL2-MEC1-RAD53-DUN1* checkpoint pathway. In contrast, *DDI1* remains inducible in *sad1-1* (*rad53*), *dun1* or *mec1* mutants, but its induction is diminished in *pds1 sad1-1* (*rad53*) or *pds1 dun1* double mutants (Zhu & Xiao, 2001). *PDS1* encodes an anaphase inhibitor and functions downstream of *CHK1*; *CHK1-PDS1* and *RAD53-DUN1* may form two parallel branches in the DNA damage checkpoints (Fig. 1) (Gardner *et al.*, 1999; Schollaert *et al.*, 2004). This suggests that the *CHK1-PDS1* and *MEC1-RAD53-DUN1* checkpoint pathways may function redundantly in the control of *DDI1* expression (Zhu & Xiao, 2001).

### A putative eukaryotic SOS response in budding yeast?

Although cell-cycle checkpoints have been regarded as a eukaryotic SOS response, their similarity to the bacterial SOS response is limited. In budding yeast, the Rad51 protein is a sequence homolog of *E. coli* RecA and possesses ssDNA-binding and ATPase activities (Shinohara *et al.*, 1992). However, the only Rad51 enzymatic activity reported to date is ATP- and ssDNA-dependent recombinase, and it is involved in homologous recombination and homology-mediated DNA repair and damage tolerance pathways (Sung, 1994; Paques & Haber, 1999; Gangavarapu *et al.*, 2007). Although *RAD51* itself is DNA damage inducible (Shinohara *et al.*, 1992), deletion of *RAD51* does not affect the expression of the DNA damage-inducible genes examined (Y. Fu & W. Xiao, unpublished results). Nevertheless, it remains possible that there is a functional homolog of RecA in yeast with respect to a transcriptional response but it shares no sequence homology and enzymatic activity with RecA.

In a broad sense, *E. coli* RecA controls three important cellular responses to DNA damage, namely homologous recombination via RecBCD and RecFOR, TLS by working

with PolIV and PolV and the SOS response, which collectively provides a survival mechanism when cells encounter replication blocks and/or contain excessive ssDNA. The above functions are reminiscent of the Rad6–Rad18 complex in budding yeast. Rad6–Rad18 forms a stable complex that is essential for the DNA postreplication repair (PRR) pathway (Broomfield *et al.*, 2001). The Rad6–Rad18 complex possesses ssDNA-binding and ATPase activities (Bailly *et al.*, 1997) and assumes most if not all RecA functions to coordinate broad cellular responses to DNA damage. Firstly, Rad6–Rad18 as an E2–E3 ubiquitination complex monoubiquitates PCNA to promote Polζ- and Polη-mediated TLS (Hoege *et al.*, 2002; Stelter & Ulrich, 2003). Secondly, this mono-Ub-PCNA is required for PCNA poly-ubiquitination via a Lys63 chain linkage for an error-free mode of DNA damage tolerance (Hoege *et al.*, 2002) reminiscent of the RecFOR activity in *E. coli* (Chow & Courcelle, 2004). Finally, although Rad6–Rad18 does not have homologous recombinase activity like RecA, it may compete with the Ubc9–Siz1 complex that sumoylates PCNA at the same K164 residue; sumoylated PCNA recruits the DNA helicase Srs2, which inhibits the recombinase activity of the yeast RecA homolog Rad51 (Papouli *et al.*, 2005; Pfander *et al.*, 2005). Hence, it would be of great interest to investigate whether the Rad6–Rad18 complex also plays a role in an SOS response and, if so, how it interacts with the damage checkpoints.

## Conclusions

In summary, transcriptional response to DNA damage is an important process for cell survival of microorganisms under such life-threatening stress. In bacteria such as *E. coli*, the DNA damage-induced transcriptional regulation is centrally controlled by an SOS response. All SOS-inducible genes contain SOS boxes in their operator regions, and their expression is repressed by the interaction between the transcriptional repressor LexA and SOS boxes under noninduced conditions. The DNA damage induction is achieved through enhanced self-proteolysis of LexA assisted by ssDNA-activated RecA. In addition, bacteria also possess regulatory mechanisms in response to specific DNA damage.

DNA damage-induced transcriptional response in eukaryotic microorganisms is apparently more complicated and less understood. Nevertheless, several conclusions can be drawn from data analyses as presented in this review. Firstly, the idea that eukaryotic microorganisms possess an SOS response structurally conserved from bacteria has been effectively ruled out. Secondly, in budding yeast, a large number of yeast genes (more than 5%) are induced after DNA damage. Almost all DNA damage-induced genes individually investigated to date respond to a broad range

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved   FEMS Microbiol Rev **32** (2008) 908–926
of DNA-damaging agents regardless of whether the gene function provides a survival value to the particular damage (Birrell *et al.*, 2002). This observation supports the hypothesis that there may exist a coordinated signal transduction pathway to regulate transcriptional response to DNA damage. Thirdly, DNA damage checkpoint pathways variably affect transcriptional response to DNA damage in budding yeast and have been regarded as a functional analog of the bacterial SOS response. Fourthly, unlike the SOS-inducible genes in bacteria, few DNA damage-inducible genes in budding yeast share a common promoter sequence, suggesting that the downstream effectors are diversified. This is consistent with reports that all three well-characterized sets of damage-inducible genes in budding yeast are controlled by different transcriptional regulators and furthermore, their modes of regulation are different: while RNRs and *PHR1* are induced by derepression, *MAG1* and *DDI1* are induced by activation. Fifthly, the search for a novel mechanism(s) in eukaryotic microorganisms that may better resume the RecA-mediated SOS response is underway and may result in exciting insights into the eukaryotic SOS response. Finally, although transcriptional response to DNA damage is a convenient and useful indicator for the gene function, transcriptional levels do not necessarily reflect cellular protein levels, and this appears to be the case for DNA damage response in budding yeast (Lee *et al.*, 2007). Hence, other posttranscriptional regulatory mechanisms have to be taken into account when examining a particular gene function and its product activity.

### Acknowledgements

The authors wish to thank the laboratory members for helpful discussion and Michelle Hanna for proofreading the manuscript. This work is supported by the Natural Sciences and Engineering Research Council of Canada discovery grant OGP0138338 to W.X. Y.F. is a recipient of the Arthur Smyth Memorial Scholarship from College of Medicine, University of Saskatchewan.

### References

Aboussekhra A, Vialard JE, Morrison DE, de la Torre-Ruiz MA, Cernakova L, Fabre F & Lowndes NF (1996) A novel role for the budding yeast *RAD9* checkpoint gene in DNA damage-dependent transcription. *EMBO J* **15**: 3912–3922.

Akimaru H, Sakumi K, Yoshikai T, Anai M & Sekiguchi M (1990) Positive and negative regulation of transcription by a cleavage product of Ada protein. *J Mol Biol* **216**: 261–273.

Allen JB, Zhou Z, Siede W, Friedberg EC & Elledge SJ (1994) The *SAD1/RAD53* protein kinase controls multiple checkpoints and DNA damage-induced transcription in yeast. *Genes Dev* **8**: 2401–2415.

Aslund F, Zheng M, Beckwith J & Storz G (1999) Regulation of the OxyR transcription factor by hydrogen peroxide and the cellular thiol-disulfide status. *Proc Natl Acad Sci USA* **96**: 6161–6165.

Bachant JB & Elledge SJ (1998) Regulatory networks that control DNA damage-inducible gene in *Saccharomyces cerevisiae*. *DNA Damage and Repair*, Vol. 1: *DNA Repair in Prokaryotes and Lower Eukaryotes* (Nikoloff JA & Hoekstra MF, eds), pp. 383–410. Humana Press, Totowa, NJ.

Bailly V, Lauder S, Prakash S & Prakash L (1997) Yeast DNA repair proteins Rad6 and Rad18 form a heterodimer that has ubiquitin conjugating, DNA binding, and ATP hydrolytic activities. *J Biol Chem* **272**: 23360–23365.

Basrai MA, Velculescu VE, Kinzler KW & Hieter P (1999) NORF5/HUG1 is a component of the MEC1-mediated checkpoint response to DNA damage and replication arrest in *Saccharomyces cerevisiae*. *Mol Cell Biol* **19**: 7041–7049.

Bennett RA (1999) The *Saccharomyces cerevisiae ETH1* gene, an inducible homolog of exonuclease III that provides resistance to DNA-damaging agents and limits spontaneous mutagenesis. *Mol Cell Biol* **19**: 1800–1809.

Bennett RJ, Sharp JA & Wang JC (1998) Purification and characterization of the Sgs1 DNA helicase activity of *Saccharomyces cerevisiae*. *J Biol Chem* **273**: 9644–9650.

Bennett RJ, Keck JL & Wang JC (1999) Binding specificity determines polarity of DNA unwinding by the Sgs1 protein of *S. cerevisiae*. *J Mol Biol* **289**: 235–248.

Benton MG, Somasundaram S, Glasner JD & Palecek SP (2006) Analyzing the dose-dependence of the *Saccharomyces cerevisiae* global transcriptional response to methyl methanesulfonate and ionizing radiation. *BMC Genom* **7**: 305.

Berg OG (1988) Selection of DNA binding sites by regulatory proteins: the LexA protein and the arginine repressor use different strategies for functional specificity. *Nucleic Acids Res* **16**: 5089–5105.

Birrell GW, Brown JA, Wu HI, Giaever G, Chu AM, Davis RW & Brown JM (2002) Transcriptional response of *Saccharomyces cerevisiae* to DNA-damaging agents does not identify the genes that protect against these agents. *Proc Natl Acad Sci USA* **99**: 8778–8783.

Branzei D & Foiani M (2006) The Rad53 signal transduction pathway: replication fork stabilization, DNA repair, and adaptation. *Exp Cell Res* **312**: 2654–2659.

Brent R (1982) Regulation and autoregulation by lexA protein. *Biochimie* **64**: 565–569.

Brent R & Ptashne M (1980) The *lexA* gene product represses its own promoter. *Proc Natl Acad Sci USA* **77**: 1932–1936.

Brent R & Ptashne M (1981) Mechanism of action of the *lexA* gene product. *Proc Natl Acad Sci USA* **78**: 4204–4208.

Broomfield S, Hryciw T & Xiao W (2001) DNA postreplication repair and mutagenesis in *Saccharomyces cerevisiae*. *Mutat Res* **486**: 167–184.

Brusky J, Zhu Y & Xiao W (2000) *UBC13*, a DNA-damage-inducible gene, is a member of the error-free postreplication

repair pathway in *Saccharomyces cerevisiae*. Curr Genet 37: 168–174.

Chen J & Samson L (1991) Induction of S. *cerevisiae* MAG 3-methyladenine DNA glycosylase transcript levels in response to DNA damage. Nucleic Acids Res 19: 6427–6432.

Chen J, Derfler B, Maskati A & Samson L (1989) Cloning a eukaryotic DNA glycosylase repair gene by the suppression of a DNA repair defect in *Escherichia coli*. Proc Natl Acad Sci USA 86: 7961–7965.

Chen J, Derfler B & Samson L (1990) *Saccharomyces cerevisiae* 3-methyladenine DNA glycosylase has homology to the AlkA glycosylase of *E. coli* and is induced in response to DNA alkylation damage. EMBO J 9: 4569–4575.

Chen SH, Smolka MB & Zhou H (2007) Mechanism of Dun1 activation by Rad53 phosphorylation in *Saccharomyces cerevisiae*. J Biol Chem 282: 986–995.

Chow KH & Courcelle J (2004) RecO acts with RecF and RecR to protect and maintain replication forks blocked by UV-induced DNA damage in *Escherichia coli*. J Biol Chem 279: 3492–3496.

Cobb JA, Bjergbaek L & Gasser SM (2002) RecQ helicases: at the heart of genetic stability. FEBS Lett 529: 43–48.

Courcelle J, Khodursky A, Peter B, Brown PO & Hanawalt PC (2001) Comparative gene expression profiles following UV exposure in wild-type and SOS-deficient *Escherichia coli*. Genetics 158: 41–64.

de la Torre-Ruiz MA, Green CM & Lowndes NF (1998) RAD9 and RAD24 define two additive, interacting branches of the DNA damage checkpoint pathway in budding yeast normally required for Rad53 modification and activation. EMBO J 17: 2687–2698.

Demple B, Sedgwick B, Robins P, Totty N, Waterfield MD & Lindahl T (1985) Active site and complete sequence of the suicidal methyltransferase that counters alkylation mutagenesis. Proc Natl Acad Sci USA 82: 2688–2692.

De Sanctis V, Bertozzi C, Costanzo G, Di Mauro E & Negri R (2001) Cell cycle arrest determines the intensity of the global transcriptional response of *Saccharomyces cerevisiae* to ionizing radiation. Radiat Res 156: 379–387.

Ding H, Hidalgo E & Demple B (1996) The redox state of the [2Fe–2S] clusters in SoxR protein regulates its activity as a transcription factor. J Biol Chem 271: 33173–33175.

Drees JC, Lusetti SL, Chitteti-Pattu S, Inman RB & Cox MM (2004) A RecA filament capping mechanism for RecX protein. Mol Cell 15: 789–798.

Elledge SJ (1996) Cell cycle checkpoints: preventing an identity crisis. Science 274: 1664–1672.

Elledge SJ & Davis RW (1987) Identification and isolation of the gene encoding the small subunit of ribonucleotide reductase from *Saccharomyces cerevisiae*: DNA damage-inducible gene required for mitotic viability. Mol Cell Biol 7: 2783–2793.

Elledge SJ & Davis RW (1989) DNA damage induction of ribonucleotide reductase. Mol Cell Biol 9: 4932–4940.

Elledge SJ & Davis RW (1990) Two genes differentially regulated in the cell cycle and by DNA-damaging agents encode alternative regulatory subunits of ribonucleotide reductase. Genes Dev 4: 740–751.

Elledge SJ, Zhou Z, Allen JB & Navas TA (1993) DNA damage and cell cycle regulation of ribonucleotide reductase. Bioessays 15: 333–339.

Frei C & Gasser SM (2000) The yeast Sgs1p helicase acts upstream of Rad53p in the DNA replication checkpoint and colocalizes with Rad53p in S-phase-specific foci. Genes Dev 14: 81–96.

Friedberg E, Walker G & Wolfram S (1995) *DNA Repair and Mutagenesis*. ASM Press, Washington, DC.

Friedberg E, Walker G, Siede W, Wood R, Schultz R & Ellenberger T (2006) *DNA Repair and Mutagenesis*. 2nd edn. ASM Press, Washington, DC.

Fry RC, Sambandan TG & Rha C (2003) DNA damage and stress transcripts in *Saccharomyces cerevisiae* mutant sgs1. Mech Ageing Dev 124: 839–846.

Fu Y & Xiao W (2006) Identification and characterization of CRT10 as a novel regulator of *Saccharomyces cerevisiae* ribonucleotide reductase genes. Nucleic Acids Res 34: 1876–1883.

Fu Y, Zhu Y, Zhang K, Yeung M, Durocher D & Xiao W (2008) Rad6-Rad18 mediates a eukaryotic SOS response by ubiquitinating the 9-1-1 checkpoint clamp. Cell 133: 601–611.

Gangavarapu V, Prakash S & Prakash L (2007) Requirement of RAD52 group genes for postreplication repair of UV-damaged DNA in *Saccharomyces cerevisiae*. Mol Cell Biol 27: 7758–7764.

Gardner R, Putnam CW & Weinert T (1999) RAD53, DUN1 and PDS1 define two parallel G2/M checkpoint pathways in budding yeast. EMBO J 18: 3173–3185.

Gasch AP, Spellman PT, Kao CM et al. (2000) Genomic expression programs in the response of yeast cells to environmental changes. Mol Biol Cell 11: 4241–4257.

Gasch AP, Huang M, Metzner S, Botstein D, Elledge SJ & Brown PO (2001) Genomic expression responses to DNA-damaging agents and the regulatory role of the yeast ATR homolog Mec1p. Mol Biol Cell 12: 2987–3003.

Gaudu P, Moon N & Weiss B (1997) Regulation of the soxRS oxidative stress regulon. Reversible oxidation of the Fe–S centers of SoxR in vivo. J Biol Chem 272: 5082–5086.

Giannattasio M, Lazzaro F, Longhese MP, Plevani P & Muzi-Falconi M (2004) Physical and functional interactions between nucleotide excision repair and DNA damage checkpoint. EMBO J 23: 429–438.

Gottesman S (1981) Genetic control of the SOS system in *E. coli*. Cell 23: 1–2.

Green SR & Johnson AD (2004) Promoter-dependent roles for the Srb10 cyclin-dependent kinase and the Hda1 deacetylase in Tup1-mediated repression in *Saccharomyces cerevisiae*. Mol Biol Cell 15: 4191–4202.

Guo Y, Breeden LL, Fan W, Zhao LP, Eaton DL & Zarbl H (2006) Analysis of cellular responses to aflatoxin B(1) in yeast expressing human cytochrome P450 1A2 using cDNA microarrays. Mutat Res 593: 121–142.

Hartwell LH & Weinert TA (1989) Checkpoints: controls that ensure the order of cell cycle events. Science 246: 629–634.

DNA damage-induced transcription in budding yeast

Hegde S, Sandler SJ, Clark AJ & Madiraju MV (1995) recO and recR mutations delay induction of the SOS response in Escherichia coli. Mol Gen Genet 246: 254–258.

Heyer WD (2004) Damage signaling: RecQ sends an SOS to you. Curr Biol 14: R895–R897.

Hidalgo E & Demple B (1994) An iron-sulfur center essential for transcriptional activation by the redox-sensing SoxR protein. EMBO J 13: 138–146.

Hidalgo E, Bollinger JM Jr, Bradley TM, Walsh CT & Demple B (1995) Binuclear [2Fe-2S] clusters in the Escherichia coli SoxR protein and role of the metal centers in transcription. J Biol Chem 270: 20908–20914.

Hishida T, Han YW, Shibata T, Kubota Y, Ishino Y, Iwasaki H & Shinagawa H (2004) Role of the Escherichia coli RecQ DNA helicase in SOS signaling and genome stabilization at stalled replication forks. Genes Dev 18: 1886–1897.

Hoege C, Pfander B, Moldovan GL, Pyrowolakis G & Jentsch S (2002) RAD6-dependent DNA repair is linked to modification of PCNA by ubiquitin and SUMO. Nature 419: 135–141.

Huang M & Elledge SJ (1997) Identification of RNR4, encoding a second essential small subunit of ribonucleotide reductase in Saccharomyces cerevisiae. Mol Cell Biol 17: 6105–6113.

Huang M, Zhou Z & Elledge SJ (1998) The DNA replication and damage checkpoint pathways induce transcription by inhibition of the Crt1 repressor. Cell 94: 595–605.

Ivancic-Bace I, Vlasic I, Salaj-Smic E & Brcic-Kostic K (2006) Genetic evidence for the requirement of RecA loading activity in SOS induction after UV irradiation in Escherichia coli. J Bacteriol 188: 5024–5032.

Jang YK, Wang L & Sancar GB (1999) RPH1 and GIS1 are damage-responsive repressors of PHR1. Mol Cell Biol 19: 7630–7638.

Jeggo P, Defais TM, Samson L & Schendel P (1977) An adaptive response of E. coli to low levels of alkylating agent: comparison with previously characterised DNA repair pathways. Mol Gen Genet 157: 1–9.

Jelinsky SA & Samson LD (1999) Global response of Saccharomyces cerevisiae to an alkylating agent. Proc Natl Acad Sci USA 96: 1486–1491.

Jelinsky SA, Estep P, Church GM & Samson LD (2000) Regulatory networks revealed by transcriptional profiling of damaged Saccharomyces cerevisiae cells: Rpn4 links base excision repair with proteasomes. Mol Cell Biol 20: 8157–8167.

John L, Sharma G, Chaudhuri SP & Pillai B (2005) Cigarette smoke extract induces changes in growth and gene expression of Saccharomyces cerevisiae. Biochem Biophys Res Commun 338: 1578–1586.

Jordan A & Reichard P (1998) Ribonucleotide reductases. Annu Rev Biochem 67: 71–98.

Katou Y, Kanoh Y, Bando M *et al.* (2003) S-phase checkpoint proteins Tof1 and Mcr1 form a stable replication-pausing complex. Nature 424: 1078–1083.

Keller-Seitz MU, Certa U, Sengstag C, Wurgler FE, Sun M & Fasullo M (2004) Transcriptional response of yeast to aflatoxin


B1: recombinational repair involving RAD51 and RAD1. Mol Biol Cell 15: 4321–4336.

Kelly GT, Liu C, Smith R III, Coleman RS & Watanabe CM (2006) Cellular effects induced by the antitumor agent azinomycin B. Chem Biol 13: 485–492.

Kenyon CJ & Walker GC (1980) DNA-damaging agents stimulate gene expression at specific loci in Escherichia coli. Proc Natl Acad Sci USA 77: 2819–2823.

Kim EM, Jang YK & Park SD (2002) Phosphorylation of Rph1, a damage-responsive repressor of PHR1 in Saccharomyces cerevisiae, is dependent upon Rad53 kinase. Nucleic Acids Res 30: 643–648.

Kiser GL & Weinert TA (1996) Distinct roles of yeast MEC and RAD checkpoint genes in transcriptional induction after DNA damage and implications for function. Mol Biol Cell 7: 703–718.

Kondo T, Matsumoto K & Sugimoto K (1999) Role of a complex containing Rad17, Mec3, and Ddc1 in the yeast DNA damage checkpoint pathway. Mol Cell Biol 19: 1136–1143.

Kondo T, Wakayama T, Naiki T, Matsumoto K & Sugimoto K (2001) Recruitment of Mec1 and Ddc1 checkpoint proteins to double-strand breaks through distinct mechanisms. Science 294: 867–870.

Kugou K, Sasanuma H, Matsumoto K, Shirahige K & Ohta K (2007) Mre11 mediates gene regulation in yeast spore development. Genes Genet Syst 82: 21–33.

Landini P & Volkert MR (1995) Transcriptional activation of the Escherichia coli adaptive response gene aidB is mediated by binding of methylated Ada protein. Evidence for a new consensus sequence for Ada-binding sites. J Biol Chem 270: 8285–8289.

Lee JH & Paull TT (2005) ATM activation by DNA double-strand breaks through the Mre11-Rad50-Nbs1 complex. Science 308: 551–554.

Lee MW, Kim BJ, Choi HK *et al.* (2007) Global protein expression profiling of budding yeast in response to DNA damage. Yeast 24: 145–154.

Lee YD & Elledge SJ (2006) Control of ribonucleotide reductase localization through an anchoring mechanism involving Wtm1. Genes Dev 20: 334–344.

Lewis LK, Harlow GR, Gregg-Jolly LA & Mount DW (1994) Identification of high affinity binding sites for LexA which define new DNA damage-inducible genes in Escherichia coli. J Mol Biol 241: 507–523.

Li B & Reese JC (2000) Derepression of DNA damage-regulated genes requires yeast TAF(II)s. EMBO J 19: 4091–4100.

Li B & Reese JC (2001) Ssn6-Tup1 regulates RNR3 by positioning nucleosomes and affecting the chromatin structure at the upstream repression sequence. J Biol Chem 276: 33788–33797.

Lindahl T, Sedgwick B, Sekiguchi M & Nakabeppu Y (1988) Regulation and expression of the adaptive response to alkylating agents. Annu Rev Biochem 57: 133–157.

Little JW (1983) The SOS regulatory system: control of its state by the level of RecA protease. J Mol Biol 167: 791–808.

Little JW (1984) Autodigestion of lexA and phage λ repressors. *Proc Natl Acad Sci USA* 81: 1375–1379.

Little JW (1991) Mechanism of specific LexA cleavage: autodigestion and the role of RecA coprotease. *Biochimie* 73: 411–421.

Little JW (1993) LexA cleavage and other self-processing reactions. *J Bacteriol* 175: 4943–4950.

Little JW & Mount DW (1982) The SOS regulatory system of *Escherichia coli*. *Cell* 29: 11–22.

Little JW, Mount DW & Yanisch-Perron CR (1981) Purified lexA protein is a repressor of the recA and lexA genes. *Proc Natl Acad Sci USA* 78: 4199–4203.

Liu Y & Xiao W (1997) Bidirectional regulation of two DNA-damage-inducible genes, MAG1 and DDI1, from *Saccharomyces cerevisiae*. *Mol Microbiol* 23: 777–789.

Lusetti SL, Voloshin ON, Inman RB, Camerini-Otero RD & Cox MM (2004) The DinI protein stabilizes RecA protein filaments. *J Biol Chem* 279: 30037–30046.

Lusetti SL, Hobbs MD, Stohl EA, Chittenden S, Inman RB, Seifert HS & Cox MM (2006) The RecF protein antagonizes RecX function via direct interaction. *Mol Cell* 21: 41–50.

Lydall D & Weinert T (1995) Yeast checkpoint genes in DNA damage processing: implications for repair and arrest. *Science* 270: 1488–1491.

Ma JL, Lee SJ, Duong JK & Stern DF (2006) Activation of the checkpoint kinase Rad53 by the phosphatidylinositol kinase-like kinase Mec1. *J Biol Chem* 281: 3954–3963.

Majka J & Burgers PM (2003) Yeast Rad17/Mec3/Ddc1: a sliding clamp for the DNA damage checkpoint. *Proc Natl Acad Sci USA* 100: 2249–2254.

Majka J, Niedziela-Majka A & Burgers PM (2006) The checkpoint clamp activates Mec1 kinase during initiation of the DNA damage checkpoint. *Mol Cell* 24: 891–901.

Makiniemi M, Hillukkala T, Tuusa J *et al.* (2001) BRCT domain-containing protein TopBP1 functions in DNA replication and damage response. *J Biol Chem* 276: 30399–30406.

Mannhaupt G, Schnall R, Karpov V, Vetter I & Feldmann H (1999) Rpn4p acts as a transcription factor by binding to PACE, a nonamer box found upstream of 26S proteasomal and other genes in yeast. *FEBS Lett* 450: 27–34.

Marques M, Mojzita D, Amorim MA, Almeida T, Hohmann S, Moradas-Ferreira P & Costa V (2006) The Pep4p vacuolar proteinase contributes to the turnover of oxidized proteins but PEP4 overexpression is not sufficient to increase chronological lifespan in *Saccharomyces cerevisiae*. *Microbiology* 152: 3595–3605.

McClanahan T & McEntee K (1984) Specific transcripts are elevated in *Saccharomyces cerevisiae* in response to DNA damage. *Mol Cell Biol* 4: 2356–2363.

McDonald JP, Levine AS & Woodgate R (1997) The *Saccharomyces cerevisiae* RAD30 gene, a homologue of *Escherichia coli dinB* and *umuC*, is DNA damage inducible and functions in a novel error-free postreplication repair mechanism. *Genetics* 147: 1557–1568.


Melo JA, Cohen J & Toczyski DP (2001) Two checkpoint complexes are independently recruited to sites of DNA damage *in vivo*. *Genes Dev* 15: 2809–2821.

Mercier G, Berthault N, Touleimat N, Kepes F, Fourel G, Gilson E & Dutreix M (2005) A haploid-specific transcriptional response to irradiation in *Saccharomyces cerevisiae*. *Nucleic Acids Res* 33: 6635–6643.

Mills KD, Sinclair DA & Guarente L (1999) MEC1-dependent redistribution of the Sir3 silencing protein from telomeres to DNA double-strand breaks. *Cell* 97: 609–620.

Myers LC, Terranova MP, Ferentz AE, Wagner G & Verdine GL (1993a) Repair of DNA methylphosphotriesters through a metalloactivated cysteine nucleophile. *Science* 261: 1164–1167.

Myers LC, Verdine GL & Wagner G (1993b) Solution structure of the DNA methyl phosphotriester repair domain of *Escherichia coli* Ada. *Biochemistry* 32: 14089–14094.

Naiki T, Wakayama T, Nakada D, Matsumoto K & Sugimoto K (2004) Association of Rad9 with double-strand breaks through a Mec1-dependent mechanism. *Mol Cell Biol* 24: 3277–3285.

Nakabeppu Y & Sekiguchi M (1986) Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents: ada protein acts as a transcriptional regulator. *Proc Natl Acad Sci USA* 83: 6297–6301.

Nakamura T, Tokumoto Y, Sakumi K, Koike G, Nakabeppu Y & Sekiguchi M (1988) Expression of the *ada* gene of *Escherichia coli* in response to alkylating agents. Identification of transcriptional regulatory elements. *J Mol Biol* 202: 483–494.

Navas TA, Zhou Z & Elledge SJ (1995) DNA polymerase epsilon links the DNA replication machinery to the S phase checkpoint. *Cell* 80: 29–39.

Niida H & Nakanishi M (2006) DNA damage checkpoints in mammals. *Mutagenesis* 21: 3–9.

Nohmi T, Battista JR, Dodson LA & Walker GC (1988) RecA-mediated cleavage activates UmuD for mutagenesis: mechanistic relationship between transcriptional derepression and posttranslational activation. *Proc Natl Acad Sci USA* 85: 1816–1820.

Nyberg KA, Michelson RJ, Putnam CW & Weinert TA (2002) Toward maintaining the genome: DNA damage and replication checkpoints. *Annu Rev Genet* 36: 617–656.

Oshiro G, Wodicka LM, Washburn MP, Yates JR III, Lockhart DJ & Winzeler EA (2002) Parallel identification of new genes in *Saccharomyces cerevisiae*. *Genome Res* 12: 1210–1220.

Ostapenko D & Solomon MJ (2003) Budding yeast CTDK-I is required for DNA damage-induced transcription. *Eukaryot Cell* 2: 274–283.

Pages V, Koffel-Schwartz N & Fuchs RP (2003) *recX*, a new SOS gene that is co-transcribed with the *recA* gene in *Escherichia coli*. *DNA Repair (Amsterdam)* 2: 273–284.

Papouli E, Chen S, Davies AA, Huttner D, Krejci L, Sung P & Ulrich HD (2005) Crosstalk between SUMO and ubiquitin on PCNA is mediated by recruitment of the helicase Srs2p. *Mol Cell* 19: 123–133.

DNA damage-induced transcription in budding yeast

Paques F & Haber JE (1999) Multiple pathways of recombination induced by double-strand breaks in *Saccharomyces cerevisiae*. *Microbiol Mol Biol Rev* **63**: 349–404.

Pellicioli A & Foiani M (2005) Signal transduction: how rad53 kinase is activated. *Curr Biol* **15**: R769–R771.

Peterson KR & Mount DW (1987) Differential repression of SOS genes by unstable lexA41 (tsl-1) protein causes a “split- phenotype” in *Escherichia coli* K-12. *J Mol Biol* **193**: 27–40.

Pfander B, Moldovan GL, Sacher M, Hoege C & Jentsch S (2005) SUMO-modified PCNA recruits Srs2 to prevent recombination during S phase. *Nature* **436**: 428–433.

Post SM, Tomkinson AE & Lee EY (2003) The human checkpoint Rad protein Rad17 is chromatin-associated throughout the cell cycle, localizes to DNA replication sites, and interacts with DNA polymerase ε. *Nucleic Acids Res* **31**: 5568–5575.

Radman M (1975) SOS repair hypothesis: phenomenology of an inducible DNA repair which is accompanied by mutagenesis. *Basic Life Sci* **5A**: 355–367.

Reichard P (1988) Interactions between deoxyribonucleotide and DNA synthesis. *Annu Rev Biochem* **57**: 349–374.

Renzette N, Gumlaw N, Nordman JT *et al.* (2005) Localization of RecA in *Escherichia coli* K-12 using RecA-GFP. *Mol Microbiol* **57**: 1074–1085.

Ritchie KB, Mallory JC & Petes TD (1999) Interactions of *TLC1* (which encodes the RNA subunit of telomerase), *TEL1*, and *MEC1* in regulating telomere length in the yeast *Saccharomyces cerevisiae*. *Mol Cell Biol* **19**: 6065–6075.

Robinson GW, Nicolet CM, Kalainov D & Friedberg EC (1986) A yeast excision-repair gene is inducible by DNA damaging agents. *Proc Natl Acad Sci USA* **83**: 1842–1846.

Roos WP & Kaina B (2006) DNA damage-induced cell death by apoptosis. *Trends Mol Med* **12**: 440–450.

Rouse J & Jackson SP (2002) Lcd1p recruits Mec1p to DNA lesions *in vitro* and *in vivo*. *Mol Cell* **9**: 857–869.

Rowley R, Hudson J & Young PG (1992) The weel protein kinase is required for radiation-induced mitotic delay. *Nature* **356**: 353–355.

Ruby SW & Szostak JW (1985) Specific *Saccharomyces cerevisiae* genes are expressed in response to DNA-damaging agents. *Mol Cell Biol* **5**: 75–84.

Ruby SW, Szostak JW & Murray AW (1983) Cloning regulated yeast genes from a pool of *lacZ* fusions. *Methods Enzymol* **101**: 253–269.

Saget BM & Walker GC (1994) The Ada protein acts as both a positive and a negative modulator of *Escherichia coli’s* response to methylating agents. *Proc Natl Acad Sci USA* **91**: 9730–9734.

Sakashita H, Sakuma T, Ohkubo T, Kainosho M, Sakumi K, Sekiguchi M & Morikawa K (1993) Folding topology and DNA binding of the N-terminal fragment of Ada protein. *FEBS Lett* **323**: 252–256.

Sakumi K & Sekiguchi M (1989) Regulation of expression of the *ada* gene controlling the adaptive response. Interactions with the *ada* promoter of the Ada protein and RNA polymerase. *J Mol Biol* **205**: 373–385.


Samson L & Cairns J (1977) A new pathway for DNA repair in *Escherichia coli*. *Nature* **267**: 281–283.

Sancar GB (1985) Sequence of the *Saccharomyces cerevisiae PHR1* gene and homology of the *PHR1* photolyase to *E. coli* photolyase. *Nucleic Acids Res* **13**: 8231–8246.

Sancar GB (2000) Enzymatic photoreactivation: 50 years and counting. *Mutat Res* **451**: 25–37.

Sancar GB, Ferris R, Smith FW & Vandeberg B (1995) Promoter elements of the *PHR1* gene of *Saccharomyces cerevisiae* and their roles in the response to DNA damage. *Nucleic Acids Res* **23**: 4320–4328.

Schnarr M, Oertel-Buchheit P, Kazmaier M & Granger-Schnarr M (1991) DNA binding properties of the LexA repressor. *Biochimie* **73**: 423–431.

Schollaert KL, Poisson JM, Searle JS, Schwanekamp JA, Tomlinson CR & Sanchez Y (2004) A role for *Saccharomyces cerevisiae* Chk1p in the response to replication blocks. *Mol Biol Cell* **15**: 4051–4063.

Sebastian J, Kraus B & Sancar GB (1990) Expression of the yeast *PHR1* gene is induced by DNA-damaging agents. *Mol Cell Biol* **10**: 4630–4637.

Sharma VM, Li B & Reese JC (2003) SWI/SNF-dependent chromatin remodeling of *RNR3* requires TAF(II)s and the general transcription machinery. *Genes Dev* **17**: 502–515.

Sharma VM, Tomar RS, Dempsey AE & Reese JC (2007) Histone deacetylases RPD3 and HOS2 regulate the transcriptional activation of DNA damage-inducible genes. *Mol Cell Biol* **27**: 3199–3210.

Shenton D, Smirnova JB, Selley JN *et al.* (2006) Global translational responses to oxidative stress impact upon multiple levels of protein synthesis. *J Biol Chem* **281**: 29011–29021.

Shinohara A, Ogawa H & Ogawa T (1992) Rad51 protein involved in repair and recombination in *S. cerevisiae* is a RecA-like protein. *Cell* **69**: 457–470.

Singh KK & Samson L (1995) Replication protein A binds to regulatory elements in yeast DNA repair and DNA metabolism genes. *Proc Natl Acad Sci USA* **92**: 4907–4911.

Singleton MR, Dillingham MS, Gaudier M, Kowalczykowski SC & Wigley DB (2004) Crystal structure of RecBCD enzyme reveals a machine for processing DNA breaks. *Nature* **432**: 187–193.

Stelter P & Ulrich HD (2003) Control of spontaneous and damage-induced mutagenesis by SUMO and ubiquitin conjugation. *Nature* **425**: 188–191.

Stohl EA, Brockman JP, Burke KL, Morimatsu K, Kowalczykowski SC & Seifert HS (2003) *Escherichia coli* RecX inhibits RecA recombinase and coprotease activities *in vitro* and *in vivo*. *J Biol Chem* **278**: 2278–2285.

Strich R, Surosky RT, Steber C, Dubois E, Messenguy F & Esposito RE (1994) UME6 is a key regulator of nitrogen repression and meiotic development. *Genes Dev* **8**: 796–810.

Sung P (1994) Catalysis of ATP-dependent homologous DNA pairing and strand exchange by yeast RAD51 protein. *Science* **265**: 1241–1243.

Sweeney FD, Yang F, Chi A, Shabanowitz J, Hunt DF & Durocher D (2005) *Saccharomyces cerevisiae* Rad9 acts as a Mec1 adaptor to allow Rad53 activation. *Curr Biol* 15: 1364–1375.

Sweet DH, Jang YK & Sancar GB (1997) Role of UME6 in transcriptional regulation of a DNA repair gene in *Saccharomyces cerevisiae*. *Mol Cell Biol* 17: 6223–6235.

Takinowaki H, Matsuda Y, Yoshida T, Kobayashi Y & Ohkubo T (2006) The solution structure of the methylated form of the N-terminal 16-kDa domain of *Escherichia coli* Ada protein. *Protein Sci* 15: 487–497.

Tamayo P, Slonim D, Mesirov J *et al.* (1999) Interpreting patterns of gene expression with self-organizing maps: methods and application to hematopoietic differentiation. *Proc Natl Acad Sci USA* 96: 2907–2912.

Teo I, Sedgwick B, Kilpatrick MW, McCarthy TV & Lindahl T (1986) The intracellular signal for induction of resistance to alkylating agents in *E. coli*. *Cell* 45: 315–324.

Thoms B & Wackernagel W (1987) Regulatory role of *recF* in the SOS response of *Escherichia coli*: impaired induction of SOS genes by UV irradiation and nalidixic acid in a *recF* mutant. *J Bacteriol* 169: 1731–1736.

Tringe SG, Willis J, Liberatore KL & Ruby SW (2006) The WTM genes in budding yeast amplify expression of the stress-inducible gene RNR3. *Genetics* 174: 1215–1228.

Tucker M, Valencia-Sanchez MA, Staples RR, Chen J, Denis CL & Parker R (2001) The transcription factor associated Ccr4 and Caf1 proteins are components of the major cytoplasmic mRNA deadenylase in *Saccharomyces cerevisiae*. *Cell* 104: 377–386.

van Attikum H, Fritsch O, Hohn B & Gasser SM (2004) Recruitment of the INO80 complex by H2A phosphorylation links ATP-dependent chromatin remodeling with DNA double-strand break repair. *Cell* 119: 777–788.

Volkert MR & Landini P (2001) Transcriptional responses to DNA damage. *Curr Opin Microbiol* 4: 178–185.

Voloshin ON, Ramirez BE, Bax A & Camerini-Otero RD (2001) A model for the abrogation of the SOS response by an SOS protein: a negatively charged helix in DinI mimics DNA in its interaction with RecA. *Genes Dev* 15: 415–427.

Walker GC (1984) Mutagenesis and inducible responses to deoxyribonucleic acid damage in *Escherichia coli*. *Microbiol Rev* 48: 60–93.

Wang H & Elledge SJ (2002) Genetic and physical interactions between *DPB11* and *DDC1* in the yeast DNA damage response pathway. *Genetics* 160: 1295–1304.

Weinert TA & Hartwell LH (1988) The *RAD9* gene controls the cell cycle response to DNA damage in *Saccharomyces cerevisiae*. *Science* 241: 317–322.

Weinert TA & Hartwell LH (1993) Cell cycle arrest of *cdc* mutants and specificity of the *RAD9* checkpoint. *Genetics* 134: 63–80.

Weinert TA, Kiser GL & Hartwell LH (1994) Mitotic checkpoint genes in budding yeast and the dependence of mitosis on DNA replication and repair. *Genes Dev* 8: 652–665.


Whitby MC & Lloyd RG (1995) Altered SOS induction associated with mutations in *recF*, *recO* and *recR*. *Mol Gen Genet* 246: 174–179.

Witkin EM (1976) Ultraviolet mutagenesis and inducible DNA repair in *Escherichia coli*. *Bacteriol Rev* 40: 869–907.

Woolstencroft RN, Beilharz TH, Cook MA, Preiss T, Durocher D & Tyers M (2006) Ccr4 contributes to tolerance of replication stress through control of CRT1 mRNA poly(A) tail length. *J Cell Sci* 119: 5178–5192.

Workman CT, Mak HC, McCuine S *et al.* (2006) A systems approach to mapping DNA damage response pathways. *Science* 312: 1054–1059.

Wu J & Weiss B (1992) Two-stage induction of the *soxRS* (superoxide response) regulon of *Escherichia coli*. *J Bacteriol* 174: 3915–3920.

Xiao W, Singh KK, Chen B & Samson L (1993) A common element involved in transcriptional regulation of two DNA alkylation repair genes (*MAG* and *MGT1*) of *Saccharomyces cerevisiae*. *Mol Cell Biol* 13: 7213–7221.

Yasuda T, Morimatsu K, Horii T, Nagata T & Ohmori H (1998) Inhibition of *Escherichia coli* RecA coprotease activities by DinI. *EMBO J* 17: 3207–3216.

Zaim J, Speina E & Kierzek AM (2005) Identification of new genes regulated by the Crt1 transcription factor, an effector of the DNA damage checkpoint pathway in *Saccharomyces cerevisiae*. *J Biol Chem* 280: 28–37.

Zhang Z, An X, Yang K *et al.* (2006) Nuclear localization of the *Saccharomyces cerevisiae* ribonucleotide reductase small subunit requires a karyopherin and a WD40 repeat protein. *Proc Natl Acad Sci USA* 103: 1422–1427.

Zheng M, Aslund F & Storz G (1998) Activation of the OxyR transcription factor by reversible disulfide bond formation. *Science* 279: 1718–1721.

Zhou BB & Elledge SJ (2000) The DNA damage response: putting checkpoints in perspective. *Nature* 408: 433–439.

Zhou Z & Elledge SJ (1992) Isolation of *crt* mutants constitutive for transcription of the DNA damage inducible gene *RNR3* in *Saccharomyces cerevisiae*. *Genetics* 131: 851–866.

Zhou Z & Elledge SJ (1993) *DUN1* encodes a protein kinase that controls the DNA damage response in yeast. *Cell* 75: 1119–1127.

Zhu Y & Xiao W (1998) Differential regulation of two closely clustered yeast genes, *MAG1* and *DDI1*, by cell-cycle checkpoints. *Nucleic Acids Res* 26: 5402–5408.

Zhu Y & Xiao W (2001) Two alternative cell cycle checkpoint pathways differentially control DNA damage-dependent induction of *MAG1* and *DDI1* expression in yeast. *Mol Genet Genomics* 266: 436–444.

Zhu Y & Xiao W (2004) Pdr3 is required for DNA damage induction of *MAG1* and *DDI1* via a bi-directional promoter element. *Nucleic Acids Res* 32: 5066–5075.
