
Identification of the Wax Ester Synthase/Acyl-Coenzyme A:Diacylglycerol Acyltransferase WSD1 Required for Stem Wax Ester Biosynthesis in Arabidopsis${}^{1,2}$[W][OA]

Fengling Li, Xuemin Wu, Patricia Lam, David Bird${}^{3}$, Huanquan Zheng${}^{4}$, Lacey Samuels, Reinhard Jetter, and Ljerka Kunst*

Department of Botany, University of British Columbia, Vancouver, British Columbia, Canada V6T 1Z4 (F.L., X.W., P.L., D.B., H.Z., L.S., R.J., L.K.); and Department of Chemistry, University of British Columbia, Vancouver, British Columbia, Canada V6T 1Z1 (R.J.)

Wax esters are neutral lipids composed of aliphatic alcohols and acids, with both moieties usually long-chain ($\mathrm{C}_{16}$ and $\mathrm{C}_{18}$) or very-long-chain ($\mathrm{C}_{20}$ and longer) carbon structures. They have diverse biological functions in bacteria, insects, mammals, and terrestrial plants and are also important substrates for a variety of industrial applications. In plants, wax esters are mostly found in the cuticles coating the primary shoot surfaces, but they also accumulate to high concentrations in the seed oils of a few plant species, including jojoba (*Simmondsia chinensis*), a desert shrub that is the major commercial source of these compounds. Here, we report the identification and characterization of *WSD1*, a member of the bifunctional wax ester synthase/diacylglycerol acyltransferase gene family, which plays a key role in wax ester synthesis in *Arabidopsis* (*Arabidopsis thaliana*) stems, as first evidenced by severely reduced wax ester levels of in the stem wax of *wsd1* mutants. In vitro assays using protein extracts from *Escherichia coli* expressing WSD1 showed that this enzyme has a high level of wax synthase activity and approximately 10-fold lower level of diacylglycerol acyltransferase activity. Expression of the *WSD1* gene in *Saccharomyces cerevisiae* resulted in the accumulation of wax esters, but not triacylglycerol, indicating that WSD1 predominantly functions as a wax synthase. Analyses of *WSD1* expression revealed that this gene is transcribed in flowers, top parts of stems, and leaves. Fully functional yellow fluorescent protein-tagged WSD1 protein was localized to the endoplasmic reticulum, demonstrating that biosynthesis of wax esters, the final products of the alcohol-forming pathway, occurs in this subcellular compartment.

The cuticle is a thin hydrophobic layer that covers the outermost surface of the primary aerial tissues of terrestrial plants. It controls nonstomatal water loss (Riederer and Schreiber, 2001), protects plants from UV radiation (Krauss et al. 1997; Solovchenko and Merzlyak, 2003; Pfündel et al., 2006), minimizes deposition of dust, pollen, and pathogen spores (Barthlott and Neinhuis, 1997), and mediates a variety of plant interactions with insects (Eigenbrode and Espelie,

---

${}^{1}$ This work was supported by the Natural Sciences and Engineering Research Council of Canada (Special Research Opportunity grant to R.J., L.S., and L.K.). Additional funding was provided by the Canadian Foundation for Innovation and the Canada Research Chairs Program (to R.J.).

${}^{2}$ This work is part of ICON, a European Community Seventh Framework Programme.

${}^{3}$ Present address: Department of Biological Sciences, University of Manitoba, Winnipeg Manitoba, Canada R3T 2N2.

${}^{4}$ Present address: Department of Biology, McGill University, Montreal, Quebec, Canada H3A 1B1.

* Corresponding author; e-mail kunst@interchange.ubc.ca.

The author responsible for distribution of materials integral to the findings presented in this article in accordance with the policy described in the Instructions for Authors (www.plantphysiol.org) is: Ljerka Kunst (kunst@interchange.ubc.ca).

[W] The online version of this article contains Web-only data.

[OA] Open Access articles can be viewed online without a subscription.

www.plantphysiol.org/cgi/doi/10.1104/pp.108.123471

Plant Physiology, September 2008, Vol. 148, pp. 97–107, www.plantphysiol.org © 2008 American Society of Plant Biologists 97 Downloaded from www.plantphysiol.org on February 23, 2015 - Published by www.plant.org Copyright © 2008 American Society of Plant Biologists. All rights reserved.

Li et al.

A

![Diagram](https://i.imgur.com/1234567.png)

B

![Image](https://i.imgur.com/8901234.png)

Figure 1. Structure of *WSD1* gene and transcript levels in mutant alleles of *WSD1*. A, Exon-intron structure of the *WSD1* gene, with white boxes representing exons. Arrows indicate T-DNA insertion sites. B, Semiquantitative RT-PCR analysis of steady-state *WSD1* transcript leaves in stems of *wsd1-1* and *wsd1-2* mutants and wild-type plants. The bottom image shows expression of *GAPC* as a loading control.

review, see Kunst et al., 2006), but the biochemical activities of these proteins are presently unknown. Only the enzyme catalyzing the last two enzymatic steps of this pathway leading from alkanes to secondary alcohols and ketones have been identified through a reverse genetic approach and characterized as a

![Graph A](https://i.imgur.com/5678901.png)

species. For example, Arabidopsis leaf and stem cuticular waxes contain only 0.1% to 0.2% and 0.7% to 2.9% of wax esters, respectively (Jenks et al., 1995), whereas esters accumulate to 85% in leaf wax of the carnauba palm (*Copernicia cerifera*; Kolattukudy, 1976). Wax esters also serve as seed storage reserves in jojoba (*Simmondsia chinensis*), where they constitute approximately 97% of the seed oil (Benzioni and Forti, 1989). Carnauba and jojoba are the main commercial sources of plant wax esters, which have diverse pharmaceutical, cosmetic, and food applications, in addition to their traditional use as lubricants. Wax esters are not only found in cuticles of many plant species, but are also widely distributed in insects (ubiquitously in the exoskeleton, but also in beeswax) and mammals (among others in humans and whales), as well as in bacteria (storage lipids in *Acinetobacter* spp.; Kolattukudy, 1976; Ishige et al., 2003; Wältermann et al., 2006).

Even though major steps in plant cuticular wax biosynthesis have been proposed, our knowledge of the enzymes involved and the type of biochemical reactions that they catalyze is incomplete. Forward genetic approaches using a collection of wax-deficient Arabidopsis *eceriferum* (*cer*) mutants and maize (*Zea mays*) glossy mutants resulted in the identification of several genes encoding wax biosynthetic enzymes, including *CER6*, *CER10*, *GL8A*, and *GL8B* (Millar et al., 1999; Fiebig et al., 2000; Hooker et al., 2002; Xu et al., 1997; Dietrich et al., 2005; Zheng et al., 2005) required for fatty acid elongation. Several other mutants, specifically *cer1*, *cer2*, *wax2/cer3*, *gl1*, and *gl2*, were also useful for the identification of proteins required for the synthesis of wax components by the alkane pathway (for

![Graph B](https://i.imgur.com/2468013.png)

Figure 2. Cuticular wax composition on the stems of Arabidopsis. A, Compound classes in wild-type and *wsd1* mutant stem wax. B, Chain length distributions within compound classes generated by the alcohol (acyl reduction) pathway. Each value is the mean of the wax load on five individual plants. Error bars indicate SDs. C, GC analysis of ester fractions within wax mixtures from stems of wild-type and mutant *wsd1-2* plants. Wax esters were first concentrated and purified via TLC and then characterized by GC-MS.

Plant Physiol. Vol. 148, 2008
P450-dependent midchain alkane hydroxylase (MAH1; Greer et al., 2007).

For the alcohol-forming pathway, only the first of the two steps, the conversion of VLCFAs to primary alcohols by the fatty acyl-CoA reductase CER4, has been investigated at the molecular level (Rowland et al., 2006). Furthermore, compositional analysis of wax esters and free primary alcohols from Arabidopsis wild type and *cer* mutants demonstrated that the primary alcohols formed by the CER4 enzyme serve as substrates for biosynthesis of the alkyl esters found in the stem wax mixture (Lai et al., 2007). It was also reported that the composition of stem wax esters in Arabidopsis is dominated by C₁₆ acyl groups and C₂₆ to C₃₀ alkyl groups, suggesting that the wax synthase (WS) enzyme may have a preference for these substrate chain lengths. However, the WS enzyme involved in wax ester production has not been identified.

In an attempt to clone and characterize the gene/enzyme responsible for wax ester biosynthesis in Arabidopsis stem epidermis, we focused our attention on gene families exhibiting high sequence homology with the three types of WS enzymes known to catalyze wax ester formation in other organisms. The first type, the mammalian WS enzymes, have the highest activity with acyl-CoAs between C₁₂ and C₁₆ in length and efficiently use alcohols shorter than C₂₀ (Cheng and Russell, 2004). These enzymes have no obvious orthologs in plants, but do share sequence similarity with the Arabidopsis acyl-CoA:diacylglycerol acyltransferase 2 (DGAT2) protein. The second type, the jojoba WS, uses a wide range of saturated and unsaturated acyl-CoAs ranging from C₁₄ to C₂₄, with C₂₀:₁ as the preferred substrate, and exhibits the highest activity with C₁₈:₁ alcohol (Lardizabal et al., 2000). This protein has sequence homology with a family of 12 predicted proteins in Arabidopsis, which have not been studied

Arabidopsis Wax Ester Synthase WSD1

to date. The third type is the *Acinetobacter calcoaceticus* WS enzyme, which exhibits both WS activity and DGAT activity. This bifunctional protein is unrelated to the mammalian or the jojoba WS enzymes, and shows a preference for C₁₄ and C₁₆ acyl-CoA together with C₁₄ to C₁₈ alcohols (Kalscheuer and Steinbüchel, 2003; Stöveken et al., 2005). In Arabidopsis, there are 11 annotated WS/DGAT genes whose functions are currently unknown. However, recently a WS/DGAT bifunctional enzyme has been shown to be involved in the synthesis of wax esters composed of a VLCFA esterified to a short- to medium-chain alcohol (≥C₁₂) in the petunia (*Petunia hybrida*) petal (King et al., 2007).

To narrow down the number of candidate genes, we decided to concentrate on the jojoba-type WS and WS/DGAT genes up-regulated in the stem epidermis during active wax synthesis (Suh et al., 2005). We identified a WS/DGAT gene (At5g37300) with the highest levels of expression in the upper, rapidly expanding part of the stem, and a high ratio of epidermis-to-total stem expression as the prime candidate for functional analysis using reverse genetics. In this study, we report on the characterization of this candidate gene, named WSD1. We show that the protein it encodes is indeed involved in stem wax ester formation and is the major enzyme catalyzing this last step of the alcohol-forming pathway of wax biosynthesis.

### RESULTS

Among the Arabidopsis genes annotated as WS/DGAT, WSD1 showed the highest expression level in the stem epidermis and a high expression ratio of epidermis to total stem in a microarray experiment (Suh et al., 2005). To determine whether this gene is involved in wax ester formation in Arabidopsis stems, we obtained mutant lines *wsd1-1* (SALK_067714) and *wsd1-2* (SALK_118165; Alonso et al., 2003) from the Arabidopsis Biological Resource Center (ABRC) and analyzed them for disruption of the WSD1 gene using PCR. The *wsd1-1* allele had a T-DNA insertion in the second exon of the WSD1 gene, 643 bp downstream from the predicted translation start site, and *wsd1-2* carried a T-DNA insertion in the 5′-untranslated region of the WSD1 gene, 180 bp upstream of the predicted translation start site (Fig. 1A). Semiquantitative reverse transcription (RT)-PCR analyses conducted to assess steady-state levels of WSD1 gene transcript in mutant alleles revealed no detectable WSD1 transcript in *wsd1-1* and reduced transcript accumulation in *wsd1-2* (Fig. 1B).

#### Stem Cuticular Wax Phenotypes of *wsd1* Mutants

Insertions in the WSD1 gene did not result in a glossy stem phenotype or altered appearance and density of epicuticular wax crystals on the stem surface (scanning electron micrographs not shown), characteristic of a number of wax-deficient mutants studied to date. To further investigate the effect of WSD1 disruption

---

**Table I. Chain length distribution of wax esters**

Relative compositions (%) are given.

| Ester Chain Length | Wild Type | *wsd1-1* | *wsd1-2* |
|---------------------|-----------|----------|----------|
| C₃₈                | 1.7       | 12.9     | 16.1     |
| C₃₉                | 0.2       | nd       | nd       |
| C₄₀                | 6.6       | 28.2     | 23.2     |
| C₄₁                | 0.4       | nd       | tr       |
| C₄₂                | 20.4      | 27.4     | 18.2     |
| C₄₃                | 0.7       | nd       | nd       |
| C₄₄                | 29.5      | 13.6     | 19.0     |
| C₄₅                | 1.4       | nd       | nd       |
| C₄₆                | 21.4      | 7.4      | 12.3     |
| C₄₇                | 0.5       | nd       | nd       |
| C₄₈                | 10.6      | 10.4     | 11.1     |
| C₄₉                | nd        | nd       | nd       |
| C₅₀                | 3.8       | nd       | tr       |
| C₅₁                | tr        | nd       | nd       |
| C₅₂                | 1.6       | nd       | tr       |
| C₅₃                | nd        | nd       | nd       |
| C₅₄                | 1.2       | tr       | tr       |

*nd*, Not detected.  
*tr*, Trace amount (i.e. <0.1%).

Plant Physiol. Vol. 148, 2008

Downloaded from www.plantphysiol.org on February 23, 2015 - Published by www.plant.org  
Copyright © 2008 American Society of Plant Biologists. All rights reserved.
Li et al.

ruption on wax accumulation and specifically wax ester formation, we therefore extracted total stem waxes from the *wsd1-1* and *wsd1-2* mutants and wild-type plants and analyzed them by gas chromatography (GC). No differences in the absolute amounts and the relative proportions of most wax components, including aldehydes, alkanes, secondary alcohols, ketones, fatty acids, primary alcohols, phenyl-ethyl esters, and terpenoids, were found in mutant lines in comparison to the wild type (Fig. 2A). The only compound class affected by *wsd1* mutations was the alkyl esters, which were severely reduced in both mutants (Fig. 2). The alkyl ester profile detected in the wild-type wax included chain lengths ranging from C₃₈ to C₄₈ (Fig. 2B). None of these wax esters was detectable in the total wax extracts from the mutant stems. Therefore, a more detailed analysis of wax esters from wild-type and mutant stem cuticular wax mixtures was carried out after separation of wax compound classes by thin-layer chromatography (TLC). In the wild-type wax, ester chain lengths varied between C₃₈ and C₅₄, with a predominance of even-numbered homologs and a maximum at C₄₄ (Fig. 1; Table I). Only trace amounts of even-numbered wax esters and no odd-numbered wax esters were detected in either *wsd1-1* or *wsd1-2* mutant. The chain length distribution of mutant esters was shifted relative to the wild type, resulting in a predominance of the C₄₀ homolog.

### In Vitro Characterization of WSD1 Using Heterologous Expression in Escherichia coli

Because WSD1 has significant sequence similarity to the WS/DGAT from *A. calcoaceticus* ADP1, the question arises as to whether the WSD1 protein has both WS and DGAT activity. To test this, we heterologously expressed the WSD1 cDNA in *E. coli* BL21 (DE3). WS assays were carried out with crude protein extract using radiolabeled [1-¹⁴C]palmitoyl-CoA as the acyl donor and 1-octadecanol (C₁₈ alcohol) as acyl acceptor, whereas [1-¹⁴C]palmitoyl-CoA and 1,2-dipalmitoyl-glycerol were used as substrates in DGAT activity assays. The reaction products were detected by autoradiography following TLC separation and the specific activities of WSD1 were determined by scintillation counting. Our results showed that WSD1 is a bifunctional enzyme, which has high WS activity (84.4 ± 5.5 pmol/mg min) and substantially lower DGAT activity (7.7 ± 1.0 pmol/mg min; Fig. 3). The ratio of 10.9 between WS and DGAT activity of WSD1 was similar to the ratio of these activities (11.1) determined for the *A. calcoaceticus* ADP1 bifunctional enzyme (Kalscheuer and Steinbüchel, 2003).

#### Figure 3. WS and DGAT activity assays using crude extracts of recombinant *E. coli* BL21 (DE3). Cells of recombinant *E. coli* strain were cultivated for 6 h at 37°C in Luria-Bertani medium containing 1 mM isopropyl-β-D-thiogalactopyranoside to induce WSD1 expression. Crude protein extracts were used with 1-[C¹⁴]palmitoyl-CoA and octadecanol (C₁₈ alcohol) as substrates for WS activity assays, and with [1-C¹⁴]palmitoyl-CoA and 1,2-dipalmitoylglycerol as substrates for DGAT activity assays. Data are mean values of three experiments ± SD.

#### Figure 4. TLC analysis of neutral lipids synthesized in vivo in recombinant yeast H1246. Lane 1, Reference substances: wax esters from jojoba seeds and TAG; lane 2, yeast H1246 (pESC-URA [empty vector control]) fed with 0.1% (w/v) palmitic acid (C₁₆ acid); lane 3, yeast H1246 (pESC-URA:WSD1) fed with 0.1% (w/v) palmitic acid; lane 4, yeast H1246 (pESC-URA:WSD1) fed with 0.1% (w/v) palmitic acid and 0.1% (w/v) octadecanol (C₁₈ alcohol); lane 5, yeast H1246 (pESC-URA:WSD1) fed with 0.1% (w/v) palmitic acid and 0.1% (w/v) tetracosanol (C₂₄ alcohol); lane 6, yeast H1246 (pESC-URA:WSD1) fed with palmitic acid and 0.1% (w/v) octacosanol (C₂₈ alcohol).

### In Vivo Characterization of WSD1 Using Heterologous Expression in Yeast

To verify that WSD1 is indeed a bifunctional enzyme with WS and DGAT activity, we introduced the WSD1 cDNA into the yeast (*Saccharomyces cerevisiae*) mutant H1246 deficient in storage lipid biosynthesis (Sandager et al., 2002). TLC analysis of the lipids extracted from the recombinant yeast cells expressing the WSD1

Plant Physiol. Vol. 148, 2008
Downloaded from www.plantphysiol.org on February 23, 2015 - Published by www.plant.org
Copyright © 2008 American Society of Plant Biologists. All rights reserved.
revealed that these cells did not accumulate wax esters or triacylglycerols (TAGs) when grown in inducible medium alone (Fig. 4, lane 2) or when supplemented with palmitic acid ($\mathrm{C}_{16:0}$; Fig. 4, lane 3). In contrast, cultivation in inducible medium containing palmitic acid together with octadecanol ($\mathrm{C}_{18}$ alcohol) led to the formation of wax esters (Fig. 4, lane 4). Similarly, when yeast cells were fed with palmitic acid together with tetracosanol ($\mathrm{C}_{24}$ alcohol) or octacosanol ($\mathrm{C}_{28}$ alcohol), both very-long-chain fatty alcohols were incorporated into wax esters (Fig. 4, lanes 5 and 6). The wax esters generated in these three experiments were purified by preparative TLC and their exact structures determined by GC-mass spectrometry (MS) analysis as octadecyl ($\mathrm{C}_{18}$), tetracosyl ($\mathrm{C}_{24}$), and octacosyl ($\mathrm{C}_{28}$) esters of palmitic ($\mathrm{C}_{16}$) acid, respectively (Fig. 5). Unlike wax ester biosynthesis, heterologous expression of WSD1 in yeast did not yield detectable amounts of TAG under any condition tested (Fig. 4).

### Organ- and Tissue-Specific Expression of WSD1

To determine the organ-specific expression of the WSD1 gene, quantitative PCR (qPCR) and RT-PCR were performed. In both experiments, the highest WSD1 mRNA accumulation was detected in flowers, followed by stem tops and leaves. Very low levels of transcript were found in roots and seeds only by qPCR (Fig. 6, A and B). To confirm these results and more precisely define the tissue specificity of WSD1 expression, the 1,978-kb genomic fragment upstream of the WSD1 coding region was fused to the GUS reporter gene ($P_{WSD1}:GUS$). Consistent with qPCR and RT-PCR

results, histochemical analysis of GUS activity in transgenic plants harboring $P_{WSD1}:GUS$ showed a high level of expression in the top 3 cm of stems (Fig. 7A). Cross sections indicated WSD1 expression in all cell types (Fig. 7B), and this result was verified by in situ hybridization (Fig. 7C; Supplemental Fig. S1). Furthermore, GUS activity was apparent in the hydathodes and veins of 20-d-old cauline leaves (Fig. 7D), filaments of stamens (Fig. 7E), and siliques (Fig. 7F), but not in seeds, roots, and young seedlings (data not shown).

To investigate the subcellular localization of the WSD1 enzyme in Arabidopsis, the WSD1 cDNA was fused with the yellow fluorescent protein (YFP) coding sequence at its 5′ end and expressed in the *wsd1-1* mutant under the control of the 35S promoter. Expression of the $P_{35S}:YFP:WSD1$ construct complemented the wax ester deficiency of the *wsd1-1* mutant (Fig. 8) by restoring stem wax ester levels to 30% to 50% of the wild type, thus confirming that the mutant phenotype was caused by the T-DNA insertion in the WSD1 gene. At the same time, the complementation of the *wsd1* phenotype by the YFP-tagged WSD1 demonstrated that the addition of YFP did not interfere with WSD1 folding or function and allowed us to localize the functional protein within epidermal cells. Protein sequence analysis using the DAS transmembrane prediction server (http://www.sbc.su.se/~miklos/DAS) classified WSD1 as an integral membrane protein containing at least four transmembrane domains (Supplemental Fig. S2). This predicted membrane association was experimentally verified by examination of the $P_{35S}:YFP:WSD1$ transgenic plants in which YFP fluorescence uniformly labeled a reticulate network typi-

---

**Figure 5.** GC analysis of wax esters produced by recombinant yeast H1246 expressing the Arabidopsis WSD1 cDNA. A, C, and E, GC traces showing wax esters produced by yeast cells fed palmitic acid ($\mathrm{C}_{16}$ acid) and octadecanol ($\mathrm{C}_{18}$ alcohol; A), palmitic acid and tetracosanol ($\mathrm{C}_{24}$ alcohol; C), or palmitic acid and octacosanol ($\mathrm{C}_{28}$ alcohol; E); arrows indicate the peaks characterized by the corresponding mass spectra on the right. B, D, and F, Mass spectra of octadecyl ($\mathrm{C}_{18}$) palmitate (molecular ion $m/z = 508$, corresponding to $\mathrm{C}_{34}$ wax ester; B); tetracosyl ($\mathrm{C}_{24}$) palmitate (molecular ion $m/z = 592$, corresponding to $\mathrm{C}_{40}$ wax ester; D); and octacosyl ($\mathrm{C}_{28}$) palmitate (molecular ion $m/z = 648$, corresponding to $\mathrm{C}_{44}$ wax ester; F).

Plant Physiol. Vol. 148, 2008  
Downloaded from www.plantphysiol.org on February 23, 2015 - Published by www.plant.org  
Copyright © 2008 American Society of Plant Biologists. All rights reserved.
stem. These wax esters consist mainly of C₁₆ acyl and C₂₆ to C₃₀ alkyl moieties (Lai et al., 2007), but the identity of the WS that catalyzes their formation was previously unknown. Based on sequence similarities to enzymes that mediate wax ester biosynthesis in other organisms and high levels of expression at the time of maximal wax deposition (Suh et al., 2005), we predicted that WSD1, encoded by a WS/DGAT-type gene At5g37300, is the WS involved in stem wax ester production. Several lines of evidence discussed below demonstrate that this is indeed the case.

Phenotypic analysis of two independent *wsd1* mutants with T-DNA insertions in the *WSD1* gene revealed deficiencies in the accumulation of wax esters in both lines. These compounds were not detectable in the total wax extracts from either the loss-of-function allele *wsd1-1* or the partial-loss-of-function line *wsd1-2* (Fig. 2). Cuticular wax ester deficiencies in *wsd1* mutants were successfully complemented by the introduction of the wild-type *WSD1* (Fig. 8), indicating that inactivation of this gene was responsible for the observed mutant phenotypes. Collectively, these results support the conclusion that the WSD1 enzyme is required for wax ester biosynthesis in Arabidopsis stem.

In contrast to the jojoba WS, which could not be functionally expressed in microorganisms (Lardizabal et al., 2000), we were successful in expressing active WSD1 in *E. coli* and yeast. Introduction of WSD1 into *E. coli* conferred upon these recombinant cells the ability to synthesize both wax esters and TAGs, indicating that WSD1 is a bifunctional WS/DGAT enzyme (Fig. 3). Both WS and DGAT activities measured in vitro for WSD1 were comparable to those of the *A. calcoaceticus* ADP1 enzyme (Kalscheuer and Steinbüchel, 2003), with the WS activity approximately 10-fold greater than the DGAT activity. When WSD1 was expressed in yeast, wax ester synthesis could only be accomplished when the growth medium was supplemented with both the acyl and the alkyl substrates required for their production. In agreement with the established composition of

Figure 6. Organ-specific expression patterns of *WSD1* detected by quantitative real-time PCR (A) and RT-PCR (B). A, *WSD1* expression level is normalized to GAPC expression. B, Expression of GAPC was used as a loading control. Developing seeds were collected 8 d after flowering.

cal of the endoplasmic reticulum (ER; Fig. 9A). The YFP fluorescence signal colocalized with the signal from hexyl rhodamine B, a dye that labels the ER in plants (Boevink et al., 1996, Fig. 9, B–D).

DISCUSSION

Alkyl esters are the final products of the alcohol pathway of cuticular wax biosynthesis and account for up to 3% of total wax on Arabidopsis inflorescence

Figure 7. Organ- and tissue-specific expression patterns of *WSD1* detected in *WSD1* promoter:GUS lines and by in situ localization. A, Stained tissues expressing *WSD1* promoter::GUS reporter gene in stem. B and D to F, Cross sections of the stem (B), leaf (D), flower (E), and silique (F). C, Expression of *WSD1* as detected by in situ hybridization in the stem. Bars: A = 5 mm; B and C = 50 μm; D to F = 1 mm.

Arabidopsis Wax Ester Synthase WSD1

stricted to the epidermal cells like most other genes involved in wax deposition (Xia et al., 1997; Millar et al., 1999; Kurata et al., 2003; Pighin and Zheng et al., 2004; Rowland et al., 2006; Greer et al., 2007), but detected in all cell types by GUS activity assays and by in situ RNA hybridization. The significance of WSD1 expression in tissues other than the epidermis is unclear at present.

The subcellular localization of the active YFP-WSD1 by confocal microscopy demonstrated that this enzyme resides in the ER membranes (Fig. 9). Because the biosynthesis of fatty acyl and alkyl substrates for wax ester production also takes place at this site (Lessire et al., 1985; Kunst and Samuels, 2003; Zheng et al., 2005; Rowland et al., 2006; Samuels et al., 2008), this result implies that the whole alcohol pathway of wax biosynthesis is associated with the ER. Recently, the midchain hydroxylase that catalyzes the final steps of the alkane pathway, the formation of secondary alcohols and ketones, was also found to be present in the ER (Greer et al., 2007). Thus, the ER is the cellular compartment where most, if not all, of the cuticular wax components are generated. These wax components then need to be delivered from the ER to the ATP-binding cassette transporters in the plasma membrane (Pighin and Zheng et al., 2004; Bird et al., 2007) for export to the cuticle.

Based on all the evidence discussed above, we conclude that WSD1 plays a major role in the formation of wax esters in the stem of Arabidopsis. This enzyme is involved in cuticle formation, and in this physiological function acts as WS by using primary alcohols as acyl acceptors. This raises the question of whether wax ester formation is the only function of WSD1 or whether the enzyme can also use diacylglycerols as acyl acceptors and thus function as a DGAT. Results from three of our experiments address this question.

First, unlike the ADP1 *A. calcoaceticus* enzyme, which restored TAG biosynthesis when expressed in the quadruple mutant of yeast H1246 deficient in storage lipid production (Kalscheuer et al., 2004), introduction of WSD1 failed to restore TAG biosynthesis in this yeast mutant, suggesting that WSD1 does not have DGAT activity in vivo (Fig. 4). Because this result is in contrast with our cell-free assays using WSD1 heterologously expressed in *E. coli*, it further suggests that WSD1 may have DGAT activity only in vitro, but not in vivo, or that the DAG substrate is not accessible to the enzyme. Alternatively, the WSD1 enzyme may show different substrate specificities depending on whether *E. coli* or yeast was used as host for heterologous expression, as reported for the *A. calcoaceticus* WS/DGAT (Kalscheuer et al., 2004). The lack of in vivo DGAT activity for the members of the WS/DGAT family has also been reported previously for the PhWS1 from petunia when expressed in yeast (King et al., 2007). Similarly, the majority of the 15 WS/DGAT homologs from *Mycobacterium tuberculosis* H37Rv showed no DGAT or WS activity (Wältermann et al., 2006) in yeast. Second, WSD1 transcript was virtually undetectable in seeds of Arabi-

Figure 8. TLC analysis of wax mixtures from stems of wild type (Col-0), *wsd1-1* mutant, and transgenic plants carrying the *35S:YFP:WSD1* transgene. Wax mixtures were extracted from wild-type, mutant, and transgenic plants, respectively, and separated on a Silica gel 60 plate using the hexane:ethyl ether:acetic acid (90:7.5:1) solvent system.

Arabidopsis stem wax esters (Lai et al., 2007), WSD1 readily utilized C<sub>16</sub> acyl and C<sub>24</sub> or C<sub>28</sub> alkyl substrates. Feeding of palmitic acid (C<sub>16</sub> acid) with C<sub>18</sub> alcohol also resulted in wax ester formation (Fig. 4), indicating that WSD1 uses a relatively broad spectrum of acyl acceptors. However, esters containing short-chain alcohols could not be detected in yeast H1246 expressing WSD1, even though potential acyl acceptors, such as ethanol and isoamyl alcohol, were likely present in our yeast strain and were accessible to the heterologous enzyme. This assumption is based on previous findings that yeast expressing the *A. calcoaceticus* WS/DGAT enzyme or petunia wax synthase PhWS1 can utilize endogenous short-chain alcohols like ethanol and/or isoamyl alcohol and incorporate them into esters (Kalscheuer et al., 2004; King et al., 2007).

WSD1 expression was detected in Arabidopsis shoots, most prominently in flowers, in the top 3 cm of inflorescence stems and in leaves, consistent with a role for this gene in cuticular wax production (Figs. 6 and 7). Surprisingly, WSD1 expression was not re-

Li et al.

Figure 9. Subcellular localization of YFP:WSD1 expression in transgenic 35S:YFP:WSD1 Arabidopsis lines. A, YFP-WSD1 is localized in the ER in the *wsd1-1* leaf epidermal cells. B to D, Localization of the ER network stained by hexyl rhodamine B (B) and colocalization with YFP-WSD1 labeled network (C); D shows merging images of B and C. One centimeter = 10 μm.

![](https://example.com/image1.png)

dopsis wild type (Fig. 6). Third, *wsd1-1* and *wsd1-2* mutants did not exhibit reduced seed oil content in comparison to wild-type seeds (Supplemental Fig. S3). Based on these results, it seems unlikely that WSD1 is involved in storage TAG formation in a physiological context.

A large family of 11 sequences, including WSD1, has been tentatively annotated as WS/DGATs in the Arabidopsis genome (Fig. 10A; Supplemental Fig. S4), so it is conceivable that at least some of these predicted WSD enzymes can work as DGATs. They have similar sequence (16.0%–20.5% amino acid sequence identity over their entire length) and size, ranging from 476 to 523 residues in length with short N-terminal or C-terminal extensions in a few cases. In addition, like the *A. calcoaceticus* WS/DGAT, the WSD1-WSD11 sequences contain a highly conserved condensing domain with a proposed active-site motif (*146HHXXXDG152*) in their N-terminal region (Fig. 10B; Supplemental Fig. S4) suggested to be essential for catalytic activity in the acyl-CoA acyltransferase reactions involved in wax ester and TAG formation (Kalscheuer and Steinbüchel, 2003). Whether any of these proteins actually have WS or DGAT activity remains to be determined. An interesting

Figure 10. Phylogenetic tree of the 11 Arabidopsis proteins related to the bifunctional WS/DGAT ADP1 of *A. calcoaceticus*. A, Phylogenetic analysis by MEGA2.0 was performed using the neighbor-joining tree with 1,000 replicates; the handling gap option was pairwise deletion. Sequence alignments were assembled by the ClustalW algorithm version 1.83. B, Protein sequence alignment of a region of the Arabidopsis WSD1 and WS/DGAT from *A. calcoaceticus* strain ADP1, which includes a putative active site motif (boxed).

![](https://example.com/image2.png)

Plant Physiol. Vol. 148, 2008

Downloaded from www.plantphysiol.org on February 23, 2015 - Published by www.plant.org  
Copyright © 2008 American Society of Plant Biologists. All rights reserved.
candidate is WSD9 (At5g12420), which shows high expression levels in seeds (Schmid et al., 2005) and may, therefore, be involved in seed TAG synthesis. Other WSDs may act as acyltransferases in the formation of other lipid esters, possibly even in the oligomerization or polymerization reactions leading to cutin or suberin production.

In summary, we have shown that WSD1, a member of the Arabidopsis WS/DGAT enzyme family, is the WS that catalyzes the biosynthesis of wax esters, dimeric cuticular wax components found on Arabidopsis shoots. This enzyme resides in the ER and uses long-chain and very-long-chain primary alcohols with C₁₆ fatty acid for wax ester production.

Arabidopsis Wax Ester Synthase WSD1

dividing by the surface area determined for the corresponding sample. Stem surface areas were calculated by measuring the projected stem areas in photographs and multiplying by π.

In TLC analysis, wax mixtures from stems of wild-type, mutant, and transgenic plants were separated on silica gel 60 (EMD Chemicals) using hexane-diethylether-acetic acid (90:7.5:1 [v/v/v]) and visualized under UV light after spraying with primuline (Sigma).

### Analysis of Seed Oil Fatty Acid Composition

To determine the fatty acid composition of the seed oil, fatty acid methyl esters were prepared by refluxing the seed samples in 2 mL of 1 N methanolic HCl for 90 min at 80°C. After cooling, 2 mL of 0.9% NaCl solution and 150 μL of hexane were added and the mixture was vortexed vigorously. The fatty acid methyl esters in the hexane phase were analyzed by GC-liquid chromatography as described previously (Kunst et al., 1992). The identity of fatty acids in the samples was determined by comparing retention times with those of standards (Sigma).

### MATERIALS AND METHODS

#### Plant Material and Growth Conditions

T-DNA insertion mutant lines SALK_067714 and SALK_118165 were obtained from the ABRC (www.arabidopsis.org). Seeds were stratified for 3 to 4 d at 4°C and plants were grown on Sunshine Mix 5 (Sun Gro Horticulture) under continuous white fluorescent light (80–100 μE m⁻² s⁻¹). The temperature was set at 20°C to 22°C.

#### Real-Time PCR and RT-PCR

Total RNA was extracted from each sample by using the RNeasy mini kit (Qiagen) and treated with RNase-free DNase (Promega). Total RNA (2.5 μg) was used for reverse transcriptase reactions. First-strand cDNA synthesis was performed using the SuperScript III platinum two-step qRT-PCR kit with SYBR green (Invitrogen). Glyceraldehyde-3-P dehydrogenase C subunit (GAPC) was used as a reference. Primers for WSD1 (forward 5′-CTGATGTTTGATCAGGGTTATG-3′ and reverse 5′-TGCCAAACTGTTCATAAAAACACC-3′) and primers for GAPC (forward 5′-ACTCGAGAAAAGCTGCTAC-3′ and reverse 5′-ATTCGTTGTCGTACCATG-3′) were used for real-time PCR. The real-time PCR reaction included an initial 5-min denaturation at 95°C for 2 min, followed by 40 cycles of 95°C for 20 s, 55°C for 10 s, 72°C for 10 s, and a final 5-min extension at 72°C. The quantitative real-time PCR was performed using a Bio-Rad miniOpticon real-time PCR detection system (Bio-Rad) according to the manufacturer’s directions.

RT-PCR was carried out in an MJ Research PTC 200 thermocycler with WSD1 primers (forward 5′-ATGAAAGCGGAAAAAGTTATGG-3′ and reverse 5′-TATTTCCCCTGTGTGGCAGA-3′) and GAPC primers (forward 5′-ACTCGAGAAAGCTGCTAC-3′ and reverse 5′-ATTCGTTGTCGTACCATG-3′) using the following amplification conditions: initial denaturation 94°C for 5 min, followed by 28 cycles of 94°C for 30 s, 56°C for 30 s, and 72°C for 30 s, and a final 5-min extension at 72°C.

#### Constructs and Plant Transformation

For the GUS activity assays, the 1,978-bp fragment immediately upstream of the WSD1 coding region was amplified by PCR from genomic Arabidopsis DNA using forward 5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTAA-TAAATTCCCTCACCATCC-3′ and reverse 5′-GGGGACCACTTTGTACAA-GAAAGCTGGGTCTTTTCCGCTTTCATAGTCAG-3′ primers. The amplified fragment was introduced into a donor vector pDONR221 (Invitrogen), resulting in pDONR221-WSD1 promoter constructs, and then the WSD1 promoter was cloned into a binary vector pGWB3 (Nakagawa et al., 2007) to create pGWB3-WSD1 promoter:GUS construct.

To generate an N-terminal YFP fusion to WSD1, the WSD1 coding region was amplified from wild-type cDNA using gene-specific primers (forward 5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTCATGAAAAGCGGAAAAA-GTTATGG-3′ and reverse 5′-GGGGACCACTTTGTACAAGAAAAGCTGGGTCT-ATTCCCCCTTGTGTGGCAGA-3′). The PCR product was cloned into the pEARLEYGATE104 vector (Earley et al., 2006) behind the constitutive cauliflower mosaic virus 35S promoter using Gateway cloning kits (Invitrogen) to create the pEARLEYGATE104-YFP:WSD1 construct. The resulting construct was transformed into wild type (Col-0) and mutant *wsd1-1* using the *Agrobacterium*-mediated floral-dip plant transformation method (Clough and Bent, 1998).

#### GUS Activity Analysis

The tissues collected from transgenic plants harboring the WSD1 promoter:GUS construct were incubated in GUS assay buffer as described previously (Jefferson, 1987). After staining, samples were cleared by incubation in 95% (v/v) ethanol overnight. For tissue sectioning, GUS-stained stems were fixed and embedded in Spurr’s resin (Rowland et al., 2006). Ten- to 20-μm sections were prepared on a Leica Ultracut microtome (Reichert-Jung) and mounted on glass slides. The commonly observed staining patterns were recorded using a dissecting microscope fitted with a digital camera (SPOT Diagnostics).

#### Laser-Scanning Confocal Microscopy

Tissues from 10-d-old transgenic seedlings harboring the 35S:YFP:WSD1 were immersed in 1.6 mM hexyl rhodamine B solution (Molecular Probes) for 10 to 30 min, then mounted in distilled water and observed in a Zeiss LSM 5 Pascal confocal laser-scanning microscope (Carl Zeiss; http://www.zeiss.com). YFP fluorescence was detected using excitation of 514 nm with a 535- to 580-nm band-pass emission filter. Hexyl rhodamine B was excited with a 543-nm argon ion laser line with a 600- to 650-nm band-pass emission filter. All confocal images obtained were processed with LSM 5 Image Browser (Carl Zeiss) and Adobe Photoshop 5.0 software.

#### In Situ Hybridization

Tissue fixation, sectioning, hybridization, signal detection, and probe synthesis were performed as previously described (Hooker et al., 2002; Hepworth et al., 2005). The probes used for hybridization were obtained by PCR amplification from WSD1 cDNA using primers incorporating the T7 RNA polymerase-binding site. For the antisense probe, the primers used were (forward, 5′-CAGGATCCGATGGAGAGAG-3′ and reverse, 5′-GTAATAC-

#### Wax Analysis

Caulete leaves, flowers, and siliques were removed from the inflorescence stems and cuticular waxes extracted by immersing the stems two times for 30 s into chloroform at room temperature. Both chloroform extracts were combined and C₂₄ alkane was added as internal standard. The solvent was removed under a gentle stream of nitrogen and the wax mixtures were treated with bis-N,N-(trimethylsilyl)trifluoroacetamide (Sigma) in pyridine for 1 h at 70°C to transform all hydroxyl-containing compounds into the corresponding trimethylsilyl derivatives. The wax composition was determined with capillary GC (5890N; Agilent) and a mass spectrometric detector (5973N; Agilent). GC was carried out with temperature-programmed on-column injection at 53°C, oven 2 min at 50°C, raised by 40°C min⁻¹ to 200°C, held for 2 min at 200°C, raised by 3°C min⁻¹ to 320°C, and held for 30 min at 320°C, and He carrier gas inlet pressure was programmed for constant flow of 1.4 mL min⁻¹. The quantitative analyses of wax mixtures were carried out by capillary GC with flame ionization detector (FID) under the same GC conditions as above, but with H₂ carrier gas at a constant flow of 2 mL min⁻¹. Wax loads were determined by comparing GC-FID peak areas against internal standard and

Plant Physiol. Vol. 148, 2008
Downloaded from www.plantphysiol.org on February 23, 2015 - Published by www.plant.org
Copyright © 2008 American Society of Plant Biologists. All rights reserved.
Li et al.

GACTCACTATAGGGCGGTGATTTTGTATGCC-3'), and for the sense probe primers (forward, 5'-GTAATACGACTCACTATAGGGCCAGGATCC-GATGGAGAGAG-3' and reverse, 5'-GGTGATCTTTTGTATIGCCC-3') were used. Sections were visualized with a Zeiss Axioskop 2 light microscope (Carl Zeiss) equipped with a digital camera (SPOT Diagnostics).

### Heterologous Expression of WSD1 in *Escherichia coli* and Yeast

For expression in *E. coli*, full-length WSD1 cDNA was cloned into the expression vector pGEX4T2 (GE Healthcare) using *Bam*HI and *Xho*I restriction sites. The resulting plasmid pGEX4T2:WSD1 was transformed into *E. coli* BL21 (DE3). Recombinant *E. coli* was grown in Luria-Bertani medium for 6 h at 37°C in the presence of 1 mM isopropyl-β-D-thiogalactopyranoside. Cells were then harvested, washed, resuspended in 125 mM sodium phosphate buffer (pH 7.4), and disrupted by glass beads.

For expression in yeast (*Saccharomyces cerevisiae*), full-length WSD1 cDNA was amplified by PCR using forward and reverse primers 5'-AGAGGATC-CATGAAAGCGGAAAAAGTTATGG-3' and 5'-AGACTCGAGTCAAACCTTC-CGTTTTGTGAAA-3', respectively. The PCR product was cloned into the *Bam*HI-*Xho*I-restricted vector pESC-URA (Stratagene) collinear with the Gal-inducible *GAL1* promoter. The generated construct was transformed into yeast strain H1246 defective in storage lipid accumulation (Sandager et al., 2002). Recombinant yeast cells were cultivated in synthetic minimal dropout medium lacking uracil and containing 2% (w/v) Gal for 48 h at 30°C. Palmitic acid (C₁₆:₀) and various primary alcohols (C₁₈, C₂₄, and C₂₈) were dissolved in hot ethanol and added to the culture medium to a final concentration of 0.1% (w/v), where indicated.

### Determination of Enzyme Activities in Vitro

Enzyme activities were determined according to the method reported by Kalscheuer et al. (2004). Briefly, 3.75 mM octadecanol (C₁₈ alcohol) and 4.72 μM [1-C¹⁴]palmitoyl-CoA were used as substrates for WS activity, and 4.72 μM [1-C¹⁴]palmitoyl-CoA and 3.75 mM 1,2-dipalmitylglycerol were used as substrates for DGAT activity. One hundred micrograms of crude extract protein from recombinant *E. coli* was used in each assay. The assay mixtures were incubated for 30 min at 35°C and the reactions terminated by adding 500 μL of chloroform. Lipids were extracted from the chloroform phase and separated by TLC as described above. Radioactivity of reaction products was measured by scintillation counting.

### Yeast Lipid Analysis

For extraction of total lipids, yeast cells were harvested by centrifugation and 2 mL of methanol were added to each sample. After 5 min, 4 mL of chloroform and 1 mL of 0.9% NaCl were added, the two-phase system was mixed by vortexing, and the chloroform phase transferred to a new tube and concentrated under a stream of nitrogen. Wax esters were purified by preparative TLC and analyzed by GC-MS and GC-FID as described above.

Sequence data from this article can be found in the GenBank/EMBL data libraries under accession number NP_568547.

### Supplemental Data

The following materials are available in the online version of this article.

- **Supplemental Figure S1.** In situ hybridization analysis of WSD1 expression in Arabidopsis stem.
- **Supplemental Figure S2.** Transmembrane domain prediction for WSD1.
- **Supplemental Figure S3.** GC analyses of fatty acid methyl esters prepared from mature Arabidopsis seeds.
- **Supplemental Figure S4.** Sequence alignment of the WS/DGAT (WSD) family from Arabidopsis.

### ACKNOWLEDGMENTS

We thank the Genomic Analysis Laboratory of the Salk Institute for providing sequence indexed Arabidopsis T-DNA insertion mutants, Tsuyoshi Nakagawa for the pGWB3 vector, and Keith Earley for the pEARLEYGATE104 vector.

Received May 23, 2008; accepted July 9, 2008; published July 11, 2008.

### LITERATURE CITED

- Alonso JM, Stepanova AN, Leisse TJ, Kim CJ, Chen H, Shinn P, Stevenson DK, Zimmerman J, Barajas P, Cheuk R, et al (2003) Genome-wide insertional mutagenesis of Arabidopsis. Science 301: 653–657
- Barthlott W, Neinhuis C (1997) Purity of the sacred lotus, or escape from contamination in biological surfaces. Planta 202: 1–8
- Benzioni A, Forti M (1989) Jojoba. In G Robbelen, RK Downey, A Ashri, eds, Oil Crops of the World. McGraw-Hill Publishing Company, New York, pp 448–461
- Bird D, Beisson F, Brigham A, Shin J, Greer S, Jetter R, Kunst L, Wu X, Yephremov A, Samuels L (2007) Characterization of Arabidopsis WBC11/ABC11, an ATP binding cassette (ABC) transporter that is required for cuticular lipid secretion. Plant J 52: 485–498
- Boevink P, Cruz S, Hawes C, Harris N, Oparka KJ (1996) Virus-mediated delivery of the green fluorescent protein to the endoplasmic reticulum of plant cells. Plant J 10: 935–941
- Carver TLW, Gurr SJ (2006) Filamentous fungi on plant surfaces. In M Riederer, C Müller, eds, Annual Plant Reviews 23: Biology of the Plant Cuticle. Blackwell, Oxford, pp 368–397
- Cheng JB, Russell DW (2004) Mammalian wax biosynthesis. 2. Expression cloning of wax synthase cDNAs encoding a member of the acyltransferase enzyme family. J Biol Chem 279: 37798–37807
- Clough SJ, Bent AF (1998) Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis. Plant J 16: 735–743
- Dietrich CR, Perera MA, D Yandeau-Nelson M, Meeley RB, Nikolau BJ, Schnable PS (2005) Characterization of two GL8 paralogs reveals that the 3-ketoacyl reductase component of fatty acid elongase is essential for maize (*Zea mays* L.) development. Plant J 42: 844–861
- Earley KW, Haag JR, Pontes O, Opper K, Juehne T, Song K, Pikaard CS (2006) Gateway-compatible vectors for plant functional genomics and proteomics. Plant J 45: 616–629
- Eigenbrode SD, Espelie KE (1995) Effects of plant epicuticular lipids on insect herbivores. Annu Rev Entomol 40: 171–194
- Fiebig A, Mayfield JA, Miley NL, Chau S, Fischer RL, Preuss D (2000) Alterations in *CER6*, a gene identical to *CUT1*, differentially affect long-chain lipid content on the surface of pollen and stems. Plant Cell 12: 2001–2008
- Greer S, Wen M, Bird D, Wu X, Samuels AL, Kunst L, Jetter R (2007) The cytochrome P450 enzyme CYP96A15 is the midchain alkane hydroxylase responsible for formation of secondary alcohols and ketones in stem cuticular wax of Arabidopsis. Plant Physiol 145: 653–667
- Hepworth SR, Zhang Y, McKim S, Li X, Haughn GW (2005) BLADE-ON-PETIOLE-dependent signaling controls leaf and floral patterning in *Arabidopsis*. Plant Cell 17: 1434–1448
- Heredia A (2003) Biophysical and biochemical characteristics of cutin, a plant barrier biopolymer. Biochim Biophys Acta 1620: 1–7
- Hooker TS, Millar AA, Kunst L (2002) Significance of the expression of the *CER6* condensing enzyme for cuticular wax production in Arabidopsis. Plant Physiol 129: 1568–1580
- Ishige T, Tani A, Sakai Y, Kato N (2003) Wax ester production by bacteria. Curr Opin Microbiol 6: 244–250
- Jefferson RA (1987) Assaying chimeric gene in plants: the GUS gene fusion system. Plant Mol Biol Rep 5: 387–405
- Jenks MA, Tuttle HA, Eigenbrode SD, Feldmann KA (1995) Leaf epicuticular waxes of the *eceriferum* mutants in Arabidopsis. Plant Physiol 108: 369–377
- Jetter R, Kunst L, Samuels AL (2006) Composition of plant cuticular waxes. In M Riederer, C Müller, eds, Annual Plant Reviews 23: Biology of the Plant Cuticle. Blackwell, Oxford, pp 145–181
- Kalscheuer R, Luftmann H, Steinbüchel A (2004) Synthesis of novel lipids in *Saccharomyces cerevisiae* by heterologous expression of an unspecific bacterial acyltransferase. Appl Environ Microbiol 70: 7119–7125
- Kalscheuer R, Steinbüchel A (2003) A novel bifunctional wax ester synthase/acyl-CoA:diacylglycerol acyltransferase mediates wax ester and triacylglycerol biosynthesis in *Acinetobacter calcoaceticus* ADP1. J Biol Chem 278: 8075–8082

King A, Nam JW, Han J, Hilliard J, Jaworski JG (2007) Cuticular wax biosynthesis in petunia petals: cloning and characterization of an alcohol-acyltransferase that synthesizes wax-esters. Planta 226: 381–394

Kolattukudy PE (1976) Chemistry and Biochemistry of Natural Waxes. Elsevier, Amsterdam

Krauss P, Markstädter C, Riederer M (1997) Attenuation of UV radiation by plant cuticles from woody species. Plant Cell Environ 20: 1079–1085

Kunst L, Jetter R, Samuels AL (2006) Biosynthesis and transport of plant cuticular waxes. In M Riederer, C Müller, eds, Annual Plant Reviews 23: Biology of the Plant Cuticle. Blackwell, Oxford, pp 182–215

Kunst L, Samuels AL (2003) Biosynthesis and secretion of plant cuticular wax. Prog Lipid Res 42: 51–80

Kunst L, Taylor DC, Underhill EW (1992) Fatty acid elongation in developing seeds of Arabidopsis. Plant Physiol Biochem 30: 425–434

Kurata T, Kawabata-Awai C, Sakuradani E, Shimizu S, Okada K, Wada T (2003) The YORE-YORE gene regulates multiple aspects of epidermal cell differentiation in Arabidopsis. Plant J 36: 55–66

Lai C, Kunst L, Jetter R (2007) Composition of alkyl esters in the cuticular wax on inflorescence stems of Arabidopsis cer mutants. Plant J 50: 189–196

Lardizabal KD, Metz JG, Sakamoto T, Hutton WC, Pollard MR, Lassner MW (2000) Purification of a jojoba embryo wax synthase, cloning of its cDNA, and production of high levels of wax in seeds of transgenic Arabidopsis. Plant Physiol 122: 645–656

Lessire R, Juguelin H, Moreau P, Cassagne C (1985) Elongation of acyl-CoAs by microsomes from etiolated leek seedlings. Phytochemistry 24: 1187–1192

Leveau JHJ (2006) Microbial communities in the phyllosphere. In M Riederer, C Müller, eds, Annual Plant Reviews 23: Biology of the Plant Cuticle. Blackwell, Oxford, pp 334–367

Lolle SJ, Hsu W, Pruitt RE (1998) Genetic analysis of organ fusion in Arabidopsis. Genetics 149: 607–619

Millar AA, Clemens S, Zachgo S, Giblin EM, Taylor DC, Kunst L (1999) CUT1, an Arabidopsis gene required for cuticular wax biosynthesis and pollen fertility, encodes a very-long-chain fatty acid condensing enzyme. Plant Cell 11: 825–838

Müller C (2006) Plant-insect interactions on cuticular surfaces. In M Riederer, C Müller, eds, Annual Plant Reviews 23: Biology of the Plant Cuticle. Blackwell, Oxford, pp 398–422

Nakagawa T, Kurose T, Hino T, Tanaka K, Kawamukai M, Niwa Y, Toyooka K, Matsuoka K, Jinbo T, Kimura T (2007) Development of the series of gateway binary vectors, pGWB, for realizing efficient construction of fusion genes for plant transformation. J Biosci Bioeng 104: 34–41

Pfündel EE, Agati G, Cerovic ZG (2006) Optical properties of plant surfaces. In M Riederer, C Müller, eds, Annual Plant Reviews 23: Biology of the Plant Cuticle. Blackwell, Oxford, pp 216–249

Pighin JA, Zheng H, Balakshin LJ, Goodman IP, Western TL, Jetter R, Kunst L, Samuels AL (2004) Plant cuticular lipid export requires an ABC transporter. Science 306: 702–704

Riederer M, Schreiber L (2001) Protecting against water loss: analysis of the barrier properties of plant cuticles. J Exp Bot 52: 2023–2032

Rowland O, Zheng H, Hepworth SR, Lam P, Jetter R, Kunst L (2006) CER4 encodes an alcohol-forming fatty acyl-Coenzyme A reductase involved in cuticular wax production in Arabidopsis. Plant Physiol 142: 866–877

Samuels L, Kunst L, Jetter R (2008) Sealing plant surfaces: cuticular wax formation by epidermal cells. Annu Rev Plant Biol 59: 683–707

Sandager L, Gustavsson MH, Ståhl U, Dahlqvist A, Wiberg E, Banas A, Lenman M, Ronne H, Stymne S (2002) Storage lipid synthesis is nonessential in yeast. J Biol Chem 277: 6478–6482

Schmid M, Davison TS, Henz SR, Pape UJ, Demar M, Vingron M, Scholkopf B, Weigel D, Lohmann JU (2005) A gene expression map of Arabidopsis development. Nat Genet 37: 501–506

Sieber P, Schorderet M, Ryser U, Buchala A, Kolattukudy P, Metraux JP, Nawrath C (2000) Transgenic Arabidopsis plants expressing a fungal cutinase show alterations in the structure and properties of the cuticle and postgenital organ fusions. Plant Cell 12: 721–737

Solovchenko A, Merzlyak M (2003) Optical properties and contribution of cuticle to UV protection in plants: experiments with apple fruit. Photochem Photobiol Sci 2: 861–866

Stöveken T, Kalscheuer R, Malkus U, Reichelt R, Steinbüchel A (2005) The wax ester synthase/acyl Coenzyme A:diacylglycerol acyltransferase from Acinetobacter sp. strain ADP1: characterization of a novel type of acyltransferase. J Bacteriol 187: 1369–1376

Suh MC, Samuels AL, Jetter R, Kunst L, Pollard M, Ohlrogge J, Beisson F (2005) Cuticular lipid composition, surface structure, and gene expression in Arabidopsis stem epidermis. Plant Physiol 139: 1649–1665

Wältermann M, Stöveken T, Steinbüchel A (2006) Key enzymes for biosynthesis of neutral lipid storage compounds in prokaryotes: properties, function and occurrence of wax ester synthases/acyl-CoA: diacylglycerol acyltransferases. Biochimie 89: 230–242

Xia YJ, Nikolau BJ, Schnable PS (1997) Developmental and hormonal regulation of the Arabidopsis CER2 gene that codes for a nuclear-localized protein required for the normal accumulation of cuticular waxes. Plant Physiol 115: 925–937

Xu XJ, Dietrich CR, Delledonne M, Xia YJ, Wen TJ, Robertson DS, Nikolau BJ, Schnable PS (1997) Sequence analysis of the cloned glossy8 gene of maize suggests that it may code for a β-ketoacyl reductase required for the biosynthesis of cuticular waxes. Plant Physiol 115: 501–510

Zheng H, Rowland O, Kunst L (2005) Disruptions of the Arabidopsis enoyl-CoA reductase gene reveal an essential role for very-long-chain fatty acid synthesis in cell expansion during plant morphogenesis. Plant Cell 17: 1467–1481

Supplemental Figure 1. In situ hybridization analysis of WSD1 expression in Arabidopsis stem. Cross section hybridized with a digoxigenin-labeled anti-sense probe (A) and sense probe (negative control) (B).

"DAS" TM-segment prediction

![Graph](attachment:graph.png)

Supplemental Figure 2. Transmembrane domain prediction for WSD1. Protein sequence analysis was performed using the DAS transmembrane prediction server (http://www.sbc.su.se/~miklos/DAS/).

Supplemental Figure 3. Gas chromatographic analyses of fatty acid methyl esters prepared from mature Arabidopsis seeds. Data are presented as mean percentages ± SE (n=3).

ADP1-WS/DGAT
At5g37300 (WSD1) 1 ----------------MRPLHPIDIFLSLEKRQQPMHVGGLFLFQI-PDNADTFIQDLVNDIRISKSIPVPFNN--KLNGL
At1g72110 (WSD2) 1 ----------------MAIERQVTEAEEPVSPEARLFSLPGLDVFINVTIGCKTEGN-ASTIEGIKNTLINHPRFSSILVTGHGEHKGAARWI
At2g38995 (WSD3) 1 ----------------MAKEEOETAAIEPLSPVSQLFVSPSLYCFIIFTLGFQTRCN-PSTIEEGVKNTWIKLPRFSSKVEIKKN---GKASWV
At3g49190 (WSD4) 1 ----------------MEIETRPHISGDEKEEEQPLSPAARLFHAPEFNCNIISVIGFKSKLD-PCVFRGFKESFIRHPRFSSKLVTD----ENGQNQR
At3g49200 (WSD5) 1 EIKIRRRRGQIAETTVKKEVEEEEQPLSPAARLFHAPEFNCYIISVVGCLKNKEE-PDMIEGIKQTLMRHPRFSSKLVNN--CNNNRQEQK
At3g49210 (WSD6) 1 EIKTRRDTSETSVRKDDEEEVEEEQPLSPAARVFHAPEFNCYVISVIGIKKKID-PDVEIGLKQTLLRHPRFSSKMVSTSVGNKKRQTQS
At5g12420 (WSD7) 1 ----------------MTYGEEEPVSPMARVFQSPGIDLCAVTIMGFKTKIN-PDVVDALKQNVSKHPRFSSTLSDN-----GAKWI
At5g16350 (WSD8) 1 ----------------MKNEEEEPLSPMARVFQSPDIDLCAIINIGFKTKIN-PDVVDALKQNVYKHPRFSSKLSEN-----GEKWI
At5g22490 (WSD9) 1 ----------------MEKKMKEEEEEPLSPMARAFQEPSIDCGIVIKFGCKTKIN-PDIVDSLKLNVFKHPRFCSSLDDDG-----TKWLR
At5g53380 (WSD10) 1 ----------------MTKEEEVEEEPLSPMARLFQSPGIENCIIITMIGFKAKINPDIILDDLKHNVSKHPRFCSKLVIATHTNYDGERWM
At5g53390 (WSD11) 1 ----------------MGEDKKTARETVEEEPLSPCSRLENSPDFNCAIIVTMGSKVKGDTPAIIHGLEHTLVNHPRFSSILEMNINGKK---PRWW

HHXXXDG

ADP1-WS/DGAT
At5g37300 (WSD1) 65 FWEDEEEFDLDHHFRHTAL---PHPGRIELLIYISQEHOSTLLDRAKPLWTCCNIIEGIEGNRFAMYF-KIHHAMVDGVAGMRLIEKSLHDV
At1g72110 (WSD2) 79 PVKLRVDDHIIVPDLEYSN---IQNPDFVEDYTTSNIANIPMDMSKPLWEFHLLNMKTSAESELAIVKIHHSIGDGMSLMSLLACSKIS
At2g38995 (WSD3) 77 PTKINVEEEHVIVPDDIDPN--IENPDFLEDYTSNMAISPMDMSKPLWEFHLLKLKTSHAEAVTVARFHHSLGDGMSLMSLLACTKTC
At3g49190 (WSD4) 74 PVSVRVEDHVVPDLDYSN---IENPDFIEDYTTSKLANTPDMDSRPLWELHLLNIKTSNAESLATGKFFHHSGLDGMSLISLLASSKTS
At3g49200 (WSD5) 79 WVRTNVVVEDHFIVPKIKPQNIENSNFLEDYVSDFMKIPLDTSRPLWEVLHLLDLKTSDAENVAVLKIHHSVGDGMSIMSLLVACMKTS
At3g49210 (WSD6) 88 WVRTNVVVEDHVIIPKIQTQHIENANADFLESYVSDLTTIPLDTSKPLWEVHLLDLKTSDAENVAVLRIHHSLGDGMSMMSLLVACTKTS
At5g12420 (WSD7) 90 WRTNVVVTDHVIVSDIOTQNIENGNADFLETYVSNLTTVPLDTSKPLWQLHLLDLKTSDAENVAVLKFHHSGLDGMSMALVLACMKTS
At5g16350 (WSD8) 66 ETEVNVEDHVIVPYIDAEE---IGEGGQFIDDYMSSLTMIPLDRSRPLWDIHLNVKTSEEAEVGFIRSHHSLGDGMSLISMLLACTKTS
At5g22490 (WSD9) 66 ETEVNVEDHVIVPYIDPED---ICEGGQFVDDYISRLLIPLDRSRPLWDIHLNVKTSYAEAVGVIRFNHALADGMSFISLVLACTKTS
At5g53380 (WSD10) 71 TDVVNVEEHVFVPDIDPKL---TEEDVEFVEDYISSITMIPLDRTKPLWEVHILNAKTSDAAEICVIRCHHALGDGVSILLSLILASTKTS
At5g53390 (WSD11) 75 KTKVNVEDHVFPVDIDLQE---INKDGDGFVDDYVSRLTLSPLDKSKPLWDIHLINVKTSDAAEVGVMRCHHSLADGMSLMSLLVACTRKT
At5g53390 (WSD11) 78 RTKVKEEHVIVPDDPD---IENPDQYLEDYISKLTITIPMDLSKPLWEMHLLGVKTSNAESYAILKIHHSLGDGMSLMSLLACTRKT

ADP1-WS/DGAT
At5g37300 (WSD1) 151 TEKSIIVPPWCVEGKRAKRLREPKTGKIKKIMSGIKSQLQATPTVIQELSQTVFKDIRNPDHVSSFQAPCSILNQRVSSSRRFAAQSFDDL
At1g72110 (WSD2) 165 DPDALVSNTTATAKKP-ADSMA--------WWLVFGFWFMIRVFTTIVEFSKLMTVCFLEDTKNPLMGNPSDG-FQSWKWVHRRIISFE
At2g38995 (WSD3) 162 DPEAFFTFVAPKKNK-AKNVC--------FSLVAWFIVRLMFHTCVEVIKSIFICRASDTSAHIMGKPGA--LSANKFIHRIISLD
At3g49190 (WSD4) 160 DPDALPTTAATRKHAASSNKK--------WWLVGRFWFMIRIIFTTVVELFKYLTLCFMRDTKTPLMGKTGDA-IRSFKVIHRIVSFD
At3g49200 (WSD5) 167 NPDELPSLPYQYRSSSSRSSLLTTGSRSDSRLLWLVKVIWTAVILGLNTVCDALEFITTLFVKDTETPIKGDFLSTKSKQLRLVHRTVSLD
At3g49210 (WSD6) 178 NPNELPSLPYQNRPSSGSSSLKTSSRCYSRFFWLVMVLSAALLVLNTVCDALEFITALFLKDTETPIKGDFKLASKGKRMCMVHRTVSLD
At5g12420 (WSD7) 180 NPDELPSLPNQRSSSSRSLMAGSRGDSRFLWLMVIVSATMLVLNTVCDALEFITTMFLKDTETPIKGDFRSKSKRMCLVHRTVSLD
At5g16350 (WSD8) 153 DPDMFSNAIPSMKRRATMSHSLK---TKGWFLRSIFTIGSTMRLLLWNTTIDMLLLLLTVLFKDTKTPLKAGADVRSNPKR-FYHRIISLD
At5g22490 (WSD9) 153 NPDMLSTAIPSVKRRSTVSHSLK---KTGWFLTAIFTIGSTMRLIWNTLVDMFLFTMLFLKDTKTPLKGGANVRSNPKT-FYHRNISLD
At5g53380 (WSD10) 158 EPEAFSTPVPKCR---ESYNHR---RGFSFFRLVLVVCSTVRLIWNTLVDSFLCMTIFFLKDTDTPLKG---KPGAIIKK-FSHRIVSLD
At5g53390 (WSD11) 163 SNLESFPTIPAIKRREQMMSHRFG--NKGWYSRSINAVERYAVRLWNТИVDLLLLWATSLFFKDTETPISEGIGSGNNARR-FYHRTVSL
At5g53390 (WSD11) 164 SDPEALPTVAVHKKRFGPSCNSGF---FNKIWWLVVGLWFILELLFNTFVDILMFALTIFVLRDTEPPLIAKPGSE-LIPKRFVHRIIS

ADP1-WS/DGAT
At5g37300 (WSD1) 241 RFRNIAKSLNVTTINDVVLAVCSGALAAYLMSHNSLPSKPLIAMVPASIRNDDSD-------------------------VSNRI
At1g72110 (WSD2) 243 DVKLIKDTMNMKVNDVLLGMTQAGLRYLSSKYDGS-----------------------TAEEKKILEKLRRVRGAVAINLRPATKIEDLADMMAKGSKRWGNFI
At2g38995 (WSD3) 240 DVKMKANAMMTVNDVLFGMVQAGLRYLNQRYDELT----------------------SSKSRK-NLHNIGLHGWWFNLRPNRNIEDLAKMMAKGSKRWGNSI
At3g49190 (WSD4) 239 DVKLVKNNDMKVNDVLLGMTQAGLRYLSKYVDED----------------------MVVEKKKNLEKIRLRGTVFVNLRADTKLEDLANMMAKGSKRWGNFV
At3g49200 (WSD5) 257 DIKLTKNAMMTINDVVLGVTQAGLRYLARRYGEE----------------------ETKNRKQKKLPKRIIRLRSALLVNLRPTTGIQDLADMMMEKGSKRWGNWF
At3g49210 (WSD6) 268 DIKLIKNAMKMTVNDVVLGVSQAGLQYLKRRYGEQ--------------------EESKRNSSNIPKGIRLRAALLVNLRPTTGIQDLADMMTKGSKRWGNWI
At5g12420 (WSD7) 270 DIKLIKNMTKVNDVVLGVSQAGLQYLDRRYGEKKKKVGEDQDSKRKATDMPKRIRLRSALLVNLRPNTGIQDLADMMAKGSTRWGNWI
At5g16350 (WSD8) 239 DIKLIKNAMMTINDVLFGITQASLHYLNROYGTKK---EEDGALTSYRNNLPDIGIRFVACTVNLRSDIGFKPLADMMVKDSKRWGNYF
At5g22490 (WSD9) 239 DIKLIKNAMMTINDVLLGITQAALSYLNNRYEHENN-NEEDGVLTSYTNNLPDRIRFRAGCTVNLRSDIGFKPLAEMMVKDSKRWGNYF
At5g53380 (WSD10) 238 DIKLIKNAMETINDVLLGVTEAALRYLHQSYDKTN--EEAGTSLTPNRQDLLDRIRLRSILVNLRPPTGSQSIAADMMAKGSKCWGNYIS
At5g53390 (WSD11) 250 DDIKLIKNAMKMTINDVLLGVQTQDALSRYLNQRYGDKN--GEGVTTTSNLNNLPGKIRIRAGVAVNLRQDTGIQPLEDMLAKDSKCRWG
At5g53390 (WSD11) 249 FDDVKLVKNAKMVTNDVLLLGVTQAGLSRYLSRKYDQEA----------TPKSKE-SMRRIRLRSAIMINLRPNAGTEALADMMAKKSKCRW

ADP1-WS/DGAT
At5g37300 (WSD1) 299 TMILANATHKDDPLQRLEIIRRSVQNSKQRFKRMTSDOILNYSAVVYGPAGLIISGMMPKRQAFNLVISNVPGPREPLYWNGAKLDALY
At1g72110 (WSD2) 323 GTVIFPLWKSEKDPLEYIRRAKATMDRKKISLEAFFFFYGIKFTLKFFGGKAEAFGKRRIFGH--TSLAFSNVKGPDEEISFFHHPISYI
At2g38995 (WSD3) 321 GYLIPLGMKPQDDFEEYVRQAKTIMDGKKHSLEPLFSYGGLKVTMEVFGLRGKTLVKRIFGS--TTMIFSNVVGPDDEEISFFGHRIAyi
At3g49190 (WSD4) 321 GVIIIFPLWVRSFDDPLEYVRRAKSTMDIKKLSIESLICYGLIKLTRKILGGKVETLVRRRLFDH--TTLTFSNVVMGPDEDISFFDHPMSYV
At3g49200 (WSD5) 340 GYVVFPFSIALRDDPLEHLETAQKTISRKKNSYGAMLTYIFCRIIVKLLGIKVASIINRMVSN--TTMTFSNMVGPVEEVSFYGHPITYF
At3g49210 (WSD6) 351 GYIIFPFSSIALCDDPLKHLRRAKSTIDRKKNSLEAVLTFVVGKILLNTLGVQRANVLNRALSN--TTMSFSNLVGVPVEEISFYGHTVTYI
At5g12420 (WSD7) 360 GYIVFPFSIGLRDDPLQHLRRAKRITDRKKNSLEAALTFVAGKFIILKTFGVQVAKIINRALSN--TTMSFSNLIGPIEEISFYGHPITYM
At5g16350 (WSD8) 326 SFIILPLSISLETDPLVYLNKSKAMMARTKHSYQAALTYFLIKISLKVLGAKATSLFNOHLMN--TTTCVSNVIGPMEEISFRGHPVSYI
At5g22490 (WSD9) 328 SFIILPLSISLETDPLVYLNKSKAMMARTKHSYQAALTYFLIKISLKVLGAKATSLFNOHLMN--TTTCVSNVIGPMEEISFRGHPVSYI
At5g53380 (WSD10) 326 VILFPFTIALQSDPLVYLSNVKSMIDRKKNSLITYIIYTFSEFVIKAFGINVAAFQRKIMLN--TTMCISNLPGPTEEVSFHGHPIAYFA
At5g53390 (WSD11) 337 NYDSLVFVPFSISLETDPLVPLLKAKSIMDRKKHSLVAPMHYSIIEFIINTFG---TKVFNRTCSN--TTTLSNIVGPVEEVSLHGNC
At5g53390 (WSD11) 330 GNLFGYILLPSVGLTEDPLEYVRQAKATIDRKKHSLAEVFSMAFFKLIKVLGLKASVVLVRKVIHS--TTLSFSNVVGPKEEITFHGH

ADP1-WS/DGAT
At5g37300 (WSD1) 389 PASIVLDGQALNTMTSYLDKLVGLIACRNAIPRMQNLLTHLEEIQLEFGVIAKQEDIKTAN-------------------------
At1g72110 (WSD2) 411 AGSALVGA-QALNIHFISYVDKVINLAVDTTTIQDPNRLCDDMVEALEIKSATQGEIFHKTEV-------------------------
At2g38995 (WSD3) 409 AASTFGVPQALNICIQSYVDKLINIGVDVDVIPDPHHLCDLIIIEALRMNNSAAPKKVFHASKV-------------------------
At3g49190 (WSD4) 409 AASALGGP-QALIHHYVTYVNKVINLAVDTSVIRDPHLLCDDLVESLDIITKLAAMEKGVHKMEV-------------------------
At3g49200 (WSD5) 428 ASSAYGHP-HALTIHCSYMNKTITLIVDPTVISDPHRLCDDWEEELRSIKAAVTKRESLRLWDYLRILHNRVAWLLYQIALLYRM
At3g49210 (WSD6) 439 APSVYGHP-HALTMHFQSYMNTISLTVDPTVISDPHRLLDDEWEKSLQSIKAAVQERDSRSLD-------------------------
At5g12420 (WSD7) 448 APSVYGHP-HALTMHFQSYMNTISLTVDPTVISDPHRLLDDEWEKSLQSIKAAVQERDSRSLD-------------------------
At5g16350 (WSD8) 414 APSSYGHS-HALLIHLMSYADKTISLAYDPPTVISDPHKICDDMEESLKTMAVLWERGLLEAYKV-------------------------
At5g22490 (WSD9) 416 SPSSYGHS-HALLIHYTSYAGEITITVDPTVIPDPHKICDDMEESLKTMAVLWERGLLEAYKV-------------------------
At5g53380 (WSD10) 414 PSIYGLP-QALTTHYLSYANKMISVAVDP-MIIDAHKLCDLEEESLKNMKLAILEKGLPNHVN-------------------------
At5g53390 (WSD11) 421 ITYIALTGYGHS-QALMIHFISYANKMIIITIAVDPAVIPDPHNICDEMEKSLKAMKDTLSGKSD-------------------------
At5g53390 (WSD11) 418 PLNYISPCVFGHP-HALTIHFQTYANKVIISVTADPTVIPDPHKMCDDLVESLKMIIKAAVLERGLYEIEV-------------------------

Supplemental Figure 4. Sequence alignment of the WS/DGAT (WSD) family from Arabidopsis.

coding sequences for the eleven deduced WSD polypeptides from Arabidopsis and Acinetobacter calcoaceticus ADP1 were obtained from annotated gene databases. Identical residues are highlighted on black background; similar amino acids are shown on a gray background. The boxed region indicates a highly conserved region among all the WSD enzymes proposed to contain a putative active site (Kalscheuer and Steinbüchel, 2003).
