
A complete inventory of all enzymes in the eukaryotic methionine salvage pathway

Ivan Pirkov, Joakim Norbeck, Lena Gustafsson and Eva Albers

Department of Chemical and Biological Engineering – Molecular Biotechnology, Chalmers University of Technology, Göteborg, Sweden

Keywords
4-methylthio-2-oxobutyrate transaminases; 5′-methylthioribulose-1-phosphate dehydratase; Mde1p; Mri1p; Saccharomyces cerevisiae

Correspondence
E. Albers, Department of Chemical and Biological Engineering – Molecular Biotechnology, Chalmers University of Technology, SE-412 96 Göteborg, Sweden
Fax: +46 31 772 3801
Tel: +46 31 772 3838
E-mail: albers@chalmers.se

(Received 15 April 2008, revised 9 June 2008, accepted 13 June 2008)

doi:10.1111/j.1742-4658.2008.06552.x

The methionine salvage pathway is universally used to regenerate methionine from 5′-methylthioadenosine, a byproduct of certain reactions involving S-adenosylmethionine. We identified and verified the genes encoding the enzymes of all steps in this cycle in a commonly used eukaryotic model system: the yeast *Saccharomyces cerevisiae*. The genes encoding 5′-methylthioribose-1-phosphate isomerase and 5′-methylthioribulose-1-phosphate dehydratase are herein named *MRII* and *MDEI*, respectively. The 5′-methylthioadenosine phosphorylase was verified as Meu1p, the 2,3-dioxomethiopentane-1-phosphate enolase/phosphatase as Utr4p and the aci-reductone dioxygenase as Adi1p. The homologue of the enolase/phosphatase gene, *YNL010w*, was excluded from its candidate role in the cycle. The methodology used involved auxotrophic growth tests and analysis of intracellular 5′-methylthioadenosine in deletion mutants. The last step, a transamination of 4-methylthio-2-oxobutyrate to yield methionine, was found to be a highly redundant step. It was catalysed by amino acid transaminases, mainly coupled with aromatic and branched chain amino acids as amino donors, but also with proline, lysine and glutamate/glutamine. The aromatic amino acid transaminases, Aro8p and Aro9p, and the branched chain amino acid transaminases, Bat1p and Bat2p, seemed to be the main enzymes exhibiting 4-methylthio-2-oxobutyrate transaminase activity. Bat2p was found to be less specific and used proline, lysine, tyrosine and glutamate as amino donors in addition to the branched chain amino acids. Thus, for the first time, all enzymes of the methionine salvage pathway were identified in a eukaryote.

Sulfur metabolism has been studied extensively and is, to a large degree, universal for eukaryotes [1–4]. However, there are still predicted pathways in which all enzymes/genes have not been identified. Sulfur is needed in the form of the amino acids cysteine and methionine, as well as S-adenosylmethionine, the adenosylated form of methionine. S-adenosylmethionine has many different functions in the cell, including as a starting point for the synthesis of polyamines: putrescine, spermine and spermidine. Polyamines are important for growth, and yeast has an absolute requirement for putrescine and spermidine, as studied in deletion mutants of biosynthetic enzymes [5,6]. During the formation of spermidine and spermine, the metabolite 5′-methylthioadenosine (MTA) is formed, which contains the sulfur atom of methionine. As the assimilation of sulfur is strongly energy consuming in the form of redox equivalents [2], most organisms from bacteria to mammals and plants have evolved recycling pathways to reuse sulfur and regenerate methionine: the ‘methionine salvage pathway’ (MTA cycle) [7–11]. The cycle of mammals and yeast consists of six enzymatic steps and one spontaneous reaction. The active enzymes are MTA phosphorylase

Abbreviations
MOB, 4-methylthio-2-oxobutyrate; MTA, 5′-methylthioadenosine; SGD, Saccharomyces Genome Database (http://www.yeastgenome.org).

FEBS Journal 275 (2008) 4111–4120 © 2008 The Authors Journal compilation © 2008 FEBS

Enzymes of methionine salvage pathway

(EC 2.4.2.28), 5′-methylthioribose-1-phosphate isomerase (EC 5.3.1.23), 5′-methylthioribulose-1-phosphate dehydratase (EC 4.2.1.109), 2,3-dioxomethiopentane-1-phosphate enolase/phosphatase (EC 3.1.3.77), aci-reductone dioxygenase (EC 1.13.11.54) and 4-methylthio-2-oxobutyrate (MOB) transaminase as the final step to yield methionine. MOB transamination is catalysed by transaminases also active in amino acid synthesis, and the donated amino group is preferably taken from branched chain, aromatic amino acids and/or glutamate depending on the organism [12–15].

The function of all enzymes, except for the dehydratase, has been verified experimentally for the bacterium *Klebsiella* [16–18], which, to date, is the most extensively studied organism. However, in eukaryotic cells, enzymes of the MTA cycle have so far only sporadically been verified, and no methodical studies have been performed. For such cells, most information is available for the yeast *Saccharomyces cerevisiae*. The pathway in this organism has been shown to be present by the identification of intermediate metabolites from MTA and their importance for methionine metabolism [7,19,20]. Meu1p, Ypr118wp and Ymr009wp (Adilp) have been verified as enzymes active in the cycle by growth defects of deletion strains on MTA media, enzyme activity measurements and metabolite formation [1,21–23]. There are indications that Yel038wp (Utr4p) functions in the cycle [23] and, on the basis of homology, Yjr024cp has been suggested as a cycle enzyme [8]. Furthermore, an aspartate transaminase and the two aromatic amino acid transaminases, Aro8p and Aro9p, have been found to have some MOB transaminase activity [24,25]. However, a systematic study of the MOB transamination step has not been performed in yeast.

The aim of this study was to identify all enzymes of the methionine salvage pathway and the substrate specificity of the transaminase step in a single organism. *S. cerevisiae* was chosen as it has been used for a long time as a model organism for several cellular processes occurring in eukaryotic cells. Candidates for enzymes of the methionine salvage pathway were identified by homology comparison, and found to be Meu1p, Ypr118wp (here named Mrilp), Yjr024cp (here named Mdelp), Utr4p, Ynl010wp and Adilp; predicted functions were verified for all but Ynl010wp. As amino acid synthesis is well characterized in yeast [26], nine candidate MOB transaminases were identified, with Aro8p, Aro9p, Bat1p and Bat2p exhibiting the major part of the activity using mainly aromatic and branched chain amino acids as amino group donors.


Results

Identification of the first five enzymes of the methionine salvage pathway

We wished to identify and verify the function of the genes in the methionine salvage pathway encoding the enzymes that had not been defined unambiguously, i.e. the genes for the dehydratase and enolase/phosphatase. Several candidates for these genes and the genes encoding verified enzymes were identified using homology BLAST searches of the yeast genome compared with known genes of the cycle from several origins. Candidates for the enzymes were obtained as Yjr024cp for the dehydratase and Utr4p (Yel038wp) and Ynl010wp for the enolase/phosphatase. The known enzymes were also found in the BLAST search, namely Meu1p (Ylr017wp) for the MTA phosphorylase, Ypr118wp for the isomerase and Adilp (Ymr009wp) for the dioxygenase.

The function of these enzymes was tested in deletion mutants in the BY background with an auxotrophic requirement of methionine. As the methionine salvage pathway converts MTA to methionine, MTA could be used to fulfil the auxotrophic requirement. If such mutants are deleted in an enzyme of the pathway and a large part of the activity is therefore absent, a growth defect for the deletion strain will be seen on medium supplemented with MTA instead of methionine. Indeed, significantly decreased growth rate ratios (with Student’s t-test; \(P < 0.001\); degree of freedom, 6) were found for *meu1Δ*, *mri1Δ*, *mde1Δ*, *utr4Δ* and

![Figure 1](#fig1)

Fig. 1. Relative growth rates during growth on MTA versus methionine. The wild-type strain and mutants deleted for enzymes of the pathway were grown in triplicate in minimal medium with either 5 mM MTA or methionine as auxotrophic supplement. The growth rates on MTA for each strain were normalized with the corresponding growth rate on methionine.

I. Pirkov *et al.*

Enzymes of methionine salvage pathway

adilΔ (Fig. 1). Ynl010wp did not show a growth defect and therefore appeared not to have a biosynthetic function in the pathway, at least under basal conditions. Thus, all steps of the pathway were assigned with a single isoenzyme. In contrast with yeast strains used in previous studies [1], the BY strains used here can take up sufficient MTA when present at a sufficiently high extracellular level, i.e. 5 mM. However, our results were verified by using other mutants as well, constructed as described in ref. [1], with a higher affinity for the uptake of MTA. For these, the growth defect on MTA medium for deletion strains was checked and verified for Meulp, Mdelp, Utr4p, Adilp and Ynl010wp (data not shown).

To further elucidate the function of our candidate enzymes, the level of intracellular MTA during glucose growth was measured. As shown in Fig. 2, MTA was accumulated in the mutants to the highest level in the meulΔ strain, and then in a descending manner in deletion mutants for reaction steps further along the pathway. If the equilibrium constants of the reactions are rather close to unity, i.e. not shifted much to either side (for the isomerase reaction of the Bacillus subtilis enzyme, the constant has been determined to be six [27]), the accumulated MTA will be equally distributed along the part preceding the blocked reaction. For example, one-half of the amount of MTA accumulated in a mutant blocked in the first step will, in a mutant blocked in the second step, be converted to the first metabolite. Thus, making deletions along a pathway will result in amounts of the first metabolite as one-half, one-third, one-quarter and one-fifth. Indeed, our data followed this pattern, with 49%, 36%, 25% and 14% of the amount of MTA in meulΔ found in mri1Δ, mde1Δ, utr4Δ and adilΔ, respectively.

On the basis of the results, Ypr118wp and Yjr024cp were herein named Mrilp (methylthioribose-1-phosphate isomerase) and Mdelp (methylthioribulose-1-phosphate dehydratase), respectively.

### Specificity of amino group donors for the transamination step

In order to enable the prediction of the relevant amino acid transaminases responsible for MOB transamination, possible amino donors for the transaminase reaction were identified in the wild-type strains. The transaminase reaction was performed by adding crude extracts to buffered reaction solution in the presence of a mixture of 18 amino acids, and the consumption of each amino acid and the formation of methionine were determined (data not shown). A similar consumption pattern was found for the two wild-type strains, and the most preferable amino acids for transamination were aromatic and branched chain amino acids. In addition, lysine, proline and glutamate/glutamine were used to some extent, whereas alanine, arginine, glycine, histidine and probably aspartate/asparagine were not used. Serine and threonine may also function as donors, but at a low level.

The total amount of amino acids used in the amino acid screen was larger than the amount of methionine formed (242% and 117% of methionine formed for strains BY4742 and YPH501, respectively). Thus, deaminases, or non-enzymatic deamination, were active. This was also confirmed by the formation of ammonium during the reaction in the crude extract (40% and 12% of methionine formed, respectively). Serine and threonine deaminases are present in Saccharomyces [28–30]. Therefore, the ammonium release accompanying MOB transaminase activity was determined in the wild-type strains in the presence of either serine or threonine alone. Ammonium was released at similar levels at 5–8 μmol·min⁻¹·g⁻¹ protein for both of these amino acids. However, when only lysine or proline was the sole amino donor, no ammonium was released (< 0.6 μmol·min⁻¹·g⁻¹ protein). Thus, the consumption of serine and threonine in the amino acid screen is exhibited by deaminases and not by MOB transaminases.

To confirm the preference for amino donor, MOB activity was determined coupled to single amino acids (tyrosine, phenylalanine, glutamate, leucine, tryptophan, aspartate and histidine) in the wild-type strains. No significant differences could be found between the two strains (Table 1). The highest activity was found for

![Figure 2](https://i.imgur.com/yourimage.png)

**Fig. 2.** Accumulated intracellular levels of MTA. Comparison of accumulated levels in deletion mutants of cycle enzymes. The level of the wild-type was subtracted before normalizing with the amount of protein for each strain. Cells were cultivated in duplicate and the mean standard deviation, estimated using all data, is given as error bars. The amount accumulated is significant for all strains, except adilΔ, at \(P < 0.01\) with a degree of freedom of six using a two-tailed Student’s t-test.

Enzymes of methionine salvage pathway

Table 1. MOB transaminase activity with single amino acids as amino donors. The transamination was performed in crude extracts from two to four cultivations of wild-type strains (BY4742 and YPH501 diploid strain) with the indicated amino acids. The standard deviations (SD, %) were estimated using all data for each amino acid. nm, not measured.

| Amino donor | MOB transaminase activity (µmol·min⁻¹·g⁻¹ protein) |
|-------------|----------------------------------------------------|
|             | BY4742 | YPH501 | SD (%) |
| Tyrosine    | 43.7   | 48.2   | 9      |
| Phenylalanine | 12.1   | nm     | 16     |
| Glutamate   | 11.6   | 13.4   | 20     |
| Leucine     | 8.8    | 8.3    | 37     |
| Tryptophan  | 2.0    | nm     | 51     |
| Aspartate   | 1.0    | 3.7    | 73     |
| Histidine   | 0.38   | nm     | 15     |

tyrosine, but substantial activity was also found with phenylalanine, leucine and glutamate. The low ability of aspartate and histidine to act as amino donors was confirmed, with very low activities (Table 1).

### Identification of enzymes for MOB transamination

As the transamination of MOB is performed by amino acid transaminases, candidates for MOB transaminases in yeast were chosen by identifying the amino acid transaminases which catalyse the transfer of the amino group at the α-position (see Saccharomyces Genome Database (http://www.yeastgenome.org) (SGD) and ref. [26]). Other transaminases, such as Ugalp, His5p, Car2p, Arg8p and Bio3p, were disregarded because of the incorrect stereochemistry when the 2-oxo group of the substrates is transaminated. The nine candidates for MOB transaminases are aromatic amino acid transaminases (Aro8p and Aro9p), aspartate transaminases (Aat1p and Aat2p), branched chain amino acid transaminases (Bat1p and Bat2p), putative alanine transaminases (Alt1p and Alt2p) and phosphoserine transaminase (Ser1p). However, as alanine seems to be a less used amino acid, alanine transaminases are unlikely to be MOB transaminases.

As the main amino group donors were aromatic and branched chain amino acids, the main candidates for MOB transaminases are the aromatic amino acid transaminases, Aro8p and Aro9p, and the branched chain amino acid transaminases, Bat1p and Bat2p. The activity of MOB transamination coupled with single amino acids was studied in crude extracts of deletion mutants in ARO and BAT genes. It was found that Aro8p exhibited the main transaminase activity for tyrosine, phenylalanine and

Fig. 3. MOB transaminase activity in *aroΔ* strains. MOB transamination coupled with single aromatic amino acids in strains with a BY background: tyrosine (open column), phenylalanine (black column) and tryptophan (grey column). n.d., not detectable. The data were normalized to the wild-type level. The difference relative to the wild-type level was evaluated statistically using a two-tailed Student's t-test. Significant differences at ***P < 0.01** and **P < 0.05** are indicated with degrees of freedom of 18 for tyrosine and four for phenylalanine and tryptophan.

tryptophan (Fig. 3). Aro9p also showed some transaminase activity as the decrease in activity for the double mutant compared with *aro8Δ* was significant for tyrosine and phenylalanine (*P < 0.05*; Student's t-test; degrees of freedom, 18 and 4, respectively). In the double *aro8Δaro9Δ* strain, almost no MOB transaminase activity coupled with any of the aromatic amino acids was found. In addition, Bat2p seemed to have some tyrosine-coupled activity, because, in the *bat2Δ* strain, a significantly lower level was found compared with the wild-type (*P < 0.05*, data not shown).

Leucine-coupled transaminase activity could be attributed to Bat1p, Bat2p and another transaminase as, in the double *bat1Δbat2Δ* strain, a residual activity was present (Fig. 4). Such residual activity has also been reported for the ‘normal’ transaminase function of amino acid synthesis [31]. We did not have access to a method measuring transaminase activity solely coupled to either isoleucine or valine. Instead, a mixture of the branched chain amino acids plus proline and lysine as amino donors was used (data not shown). In contrast with leucine, both isoleucine and valine were only coupled to the MOB transaminase activity of Bat1p and Bat2p as, in the double *bat1Δbat2Δ* strain, neither isoleucine nor valine was used for methionine formation. Methionine formation was subsequently decreased in the *bat1Δbat2Δ* strain to 8 µmol·min⁻¹·g⁻¹ protein, compared with the other strains for which the formation of methionine was three times higher (data

I. Pirkov et al.

Enzymes of methionine salvage pathway

Discussion

In this study, we have investigated the methionine salvage pathway (MTA cycle), which recycles the sulfur atom of MTA from polyamine biosynthesis to yield methionine. All enzymes of this pathway have now been identified, as we have found that Mdelp is the 5'-methylthioribulose-1-phosphate dehydratase and Utr4p is the only 2,3-dioxomethiopentane-1-phosphate enolase/phosphatase. Our results can be summarized as shown in Fig. 5. The enzymes of the cycle have been identified by measurement of the enzyme activities, growth characteristics in deletion mutants of candidate enzymes and metabolite analysis. The reactions of the first part of the cycle were each catalysed by a single isoenzyme, whereas the last transaminase reaction was catalysed by a range of more or less nonspecific isoenzymes, giving a highly redundant step. MOB transamination was mainly coupled to aromatic and branched chain amino acids as amino donors, which is also the case in many other organisms [12–15]. Activity measurements indicated that the major part of MOB transaminase activity was exhibited by the aromatic amino acid transaminases, Aro8p and Aro9p, and the branched chain amino acid transaminases, Bat1p and Bat2p. It has been noted previously that the Bat proteins have functions in addition to transaminases in the biosynthesis of branched chain amino acids [31], and it is probable that at least one of these functions is as MOB transaminase in the methionine salvage pathway. The fact that the aro8Δaro9Δ mutant did not display a major growth effect strongly indicates that the Bat system can serve as a sufficient back-up under the tested conditions. However, although this would fit nicely with enzyme activity data, creating the quadruple mutant (aro8Δaro9Δbat1Δbat2Δ) needed to test this hypothesis was not possible, despite several attempts.

Polyamine biosynthesis is a cytosolic process, and thus it may be expected that the MTA cycle will be localized in the cytosol. Indeed, Meulp, Mril, Mdelp, Utr4p and Adilp (also Ynl010w) are found in the cytosol (see SGD). Most of the candidates of the transaminases are dual isoenzymes, one cytosolic and one mitochondrial. Most of the transaminases identified to have some activity in the cycle are cytosolic (Aro8p, Aro9p and Bat2p), except for Bat1p which is mitochondrial. It can therefore be concluded that the methionine cycle is a cytosolic event. Considering the localization of the two isoenzymes Bat1p and Bat2p, it may then be expected that the cytosolic Bat2p is the major enzyme of the cycle.


Fig. 4. MOB transaminase activity in batΔ strains. MOB transamination coupled with branched chain amino acids in strains with a YPH501 background. Activity coupled with leucine only. The data were normalized to the wild-type level (haploid strain). The mean standard deviation is given as error bars and was estimated using all data. The difference relative to the wild-type level was evaluated statistically using a two-tailed Student's t-test. A significant difference at ***P < 0.01 and *P < 0.05 is indicated with a degree of freedom of 12.

not shown). Thus, both Bat1p and Bat2p seem to be important for MOB transamination coupled with branched chain amino acids.

Other amino group donors found in the initial screen were lysine, proline and glutamate. The lysine-coupled activity could be attributed to Bat2p, as deletion of this protein resulted in an almost absent use of lysine in MOB transamination (data not shown). For proline-coupled activity, the same situation was seen, with no detectable consumption of proline when introducing bat2Δ both alone and in combination with bat1Δ. However, Ser1p may also have proline-coupled MOB activity as, in the ser1Δ strain, no proline was used from a mixture of 18 amino acids for the formation of methionine (data not shown). Glutamate-coupled activity could be attributed to Aat1p, Aat2p and Bat2p as, in deletion strains of these enzymes, glutamate-coupled MOB activity was found to be significantly reduced to about 40% of the wild-type level for all of the aat1Δ, aat2Δ and bat2Δ mutants.

In a similar manner to the other enzymes of the pathway, a growth defect may be found if a substantial part of the transaminase activity disappears when using MOB as the auxotrophic requirement for methionine mutants. However, no large growth defects were found for any of the mutants tested (aro8Δ, aro9Δ, aat1Δ, aat2Δ, alt1Δ, alt2Δ, bat2Δ, ser1Δ, aro8Δaro9Δ, aat1Δaat2Δ, alt1Δalt2Δ, aro8Δbat2Δ, aro8Δaat2Δ; data not shown).

Enzymes of methionine salvage pathway

Dioxygenase  
Adi1p (Ymr009wp)  

4-Methylthio-  
2-oxobutyrate  
MOB  

1,2-Dihydroxy 3-oxo  
methiopentene  

HCOOH  
O₂  

Enolase/phosphatase  
Utr4p (Yel038wp)  

Pᵢ  

2,3-Dioxomethio  
pentane-1P  

Methylthioribulose-1P dehydratase  
Mde1p (Yjr024cp)  

H₂O  

5′-Methylthio  
ribulose-1P  

Methylthioribose-1P isomerase  
Mr1p (Ypr118wp)  

5′-Methylthio  
ribose-1P  

Adenine  

MTA phosphorylase  
Meu1p (Ylr017wp)  

Pᵢ  

5′-Methylthio  
adenosine  
MTA  

2-Oxoglutaric  
acid  
Glu  

Tyr, Phe, Trp,  
Leu, Ile, Val,  
Pro, Lys, Glx  

2-Oxo  
acids  

MOB transaminase  
Aro8p, Aro9p,  
Bat1p, Bat2p  

Methionine  

ATP  

Pᵢ PPᵢ  

S-adenosylmethionine  

CO₂  

Spe2p  

S-adenosylmethioninamine  

Spe3p  

Putrescine  

Spe4p  

Spermidine  

Spermine  

POLYAMINES  

Spe1p  

Ornithine  

CO₂  

GLYCOLYSIS  

Pyruvate  

Fig. 5. Methionine salvage pathway in Saccharomyces cerevisiae. The EC numbers for the enzymes in the pathway are as follows: MTA phosphorylase, EC 2.4.2.28; 5′-methylthioribose-1-phosphate isomerase, EC 5.3.1.23; 5′-methylthioribulose-1-phosphate dehydratase, EC 4.2.1.109; 2,3-dioxomethiopentane-1-phosphate enolase/phosphatase, EC 3.1.3.77; aci-reductone dioxygenase, EC 1.13.11.54.

The interaction of the methionine salvage pathway with the rest of metabolism is an interesting issue. Nevertheless, the cycle seems to be an isolated pathway as its intermediates have so far not been found to participate in other reactions in the cell. However, as three of the metabolites are phosphorylated, it may be expected that nonspecific phosphatases will cleave off the phosphate group. Another side reaction depending on available metal ions (Fe²⁺ and Ni²⁺) is exhibited by at least bacterial dioxygenases, by which 3-methylthiopropionate is formed instead of MOB [18,32].

The pathway has received much interest previously because of the finding that some cancer cells lack phosphorylase, and in patients with liver cirrhosis phosphorylase activity is significantly reduced [33–35]. The lack of this enzyme seems to affect, via cycle intermediates, the regulation of ornithine decarboxylase, the first enzyme of polyamine synthesis, which is increasingly expressed and thereby causes an increased cancer risk [23]. On the basis of this property of cancer cells, attempts have been made to reduce the toxicity of chemotherapeutic agents by co-administration with MTA analogues [36]. MTA analogues have also been studied for the treatment of malarial and trypanosome parasites, and have been found to have a promising effect [9,37]. MTA has been shown to influence other cellular processes, that of most interest being apoptosis [38]. The role of the MTA cycle in apoptotic processes may also be substantial, as MOB has also been found to induce apoptosis [39]. Thus, a knowledge of the methionine salvage pathway will be important for several parts of medical research and drug development.

Finally, the role of amino acid transaminases in MOB transamination in yeast seems to be an intriguing question, as different isoenzymes may have different roles in different conditions, resulting in a very robust step of the cycle. We have clearly demonstrated that the aromatic and branched chain amino acid transaminases are the main enzymes for this step, similar to the situation in bacteria [12–15]. Thus, all of the enzymes in the methionine salvage pathway have been identified in a eukaryotic organism. Accordingly, we now have a eukaryotic cell with all enzymes known which will be a genetically tractable model for use in possible studies for therapy.

## Materials and methods

### Strains

The strains used were from the single deletion collection (Euroscarf, Frankfurt, Germany) made in the BY background (wild-type strains: BY4741 MATa and BY4742 MATα). These have the basic deletions of ura3Δ0his3Δ1leu2Δ0, and both mating types were used (MATa is additionally met15Δ0 and MATα is lys2Δ0). The branched chain transaminases were studied in strains in the YPH501 background [31] (haploid wild-type sporulated in this study).

I. Pirkov *et al.*                                                                                          Enzymes of methionine salvage pathway

Table 2. List of yeast strains constructed for this study.

| Strain                | Additional genotypes                                      |
|-----------------------|-----------------------------------------------------------|
| YPH501 haploid        | $MATa$                                                  |
| $alt1\Delta alt2\Delta$ | $ylr089c::kanMX ydr111c::kanMX$                         |
| $aat1\Delta aat2\Delta$ | $ykl106w::kanMX ylr027c::kanMX$                         |
| $aro8\Delta aro9\Delta$ | $ygl202w::kanMX yhr137w::natMX$                         |
| $aro8\Delta bat2\Delta$ | $ygl202w::kanMX yjr148w::kanMX$                         |
| $aat2\Delta aro8\Delta$ | $ylr027c::kanMX ygl202w::kanMX$                         |
| $aat2\Delta alt1\Delta$ | $ylr027c::kanMX ylr089c::kanMX$                         |

from the exponential phase of the growth curves. Inoculum for the screen (2 mL, cells taken from yeast peptone dextrose agar plates, 20 g·L⁻¹ peptone, 10 g·L⁻¹ yeast extract, 20 g·L⁻¹ glucose and 20 g·L⁻¹ agar) was grown for 1 day in YNB medium supplemented with at least 120 mg·L⁻¹ of methionine, and others as specified below. Cells were reinoculated in fresh medium without methionine at an absorbance at 610 nm of 0.5, and further incubated overnight to allow the cells to use residual methionine, both external and internal. Cells were transferred to water and diluted with water to an absorbance at 610 nm of unity to be used for the inoculation of microplates. Growth rates were determined with supplementation of either methionine or a substrate for the methionine salvage pathway (MOB or MTA; Sigma-Aldrich Chemie GmbH, Steinheim, Germany), and a lack of growth was checked by omitting both methionine and the substrate. In the screen for transaminase activity, YNB was additionally supplemented with a mixture of amino acids. Methionine and MOB were used at a concentration of 1 mM. In the screen for the activity of the other enzymes in the pathway, the YNB medium was supplemented with uracil, histidine and leucine, and instead buffered with 100 mM 2-(N-morpholino)ethanesulfonic acid (pH 6.2). Methionine and MTA were used at a concentration of 5 mM.

Double deletion mutants of the BY strains were made by crossing the a and α strains of the appropriate single mutants with subsequent selection on agar plates. For the construction of the $aro8\Delta aro9\Delta$ mutant, the $ARO9$ gene was replaced by natMX giving resistance to nourseothricin (Werner BioAgents, Jena, Germany) [40] in the $aro8\Delta$ strains (both mating types). The presence of the correct deletion was checked by PCR in all strains. The strains constructed for this study are listed in Table 2.

### Media, culture conditions and growth screens

The cells were cultured in minimal medium [1.7 g·L⁻¹ yeast nitrogen base (YNB) without amino acids and ammonium; Difco, Becton Dickinson and Company, Sparks, MD, USA] with 20 g·L⁻¹ glucose as carbon and energy source and 5 g·L⁻¹ $(NH_4)_2SO_4$ as nitrogen source buffered with 10 g·L⁻¹ of succinate at pH 5.8. Auxotrophic requirements were fulfilled by using a mixture of amino acids and bases (final concentrations: 0.1 g·L⁻¹ leucine, 0.5 g·L⁻¹ serine, 0.2 g·L⁻¹ aspartate, 0.02 g·L⁻¹ tyrosine, 0.17 g·L⁻¹ tryptophan, 0.12 g·L⁻¹ adenine and 0.05 g·L⁻¹ histidine, uracil, lysine, alanine, phenylalanine, tyrosine, isoleucine and valine) or dropout medium (CSM, BIO101 Inc., Carlsbad, CA, USA). In some cases, only basic histidine, uracil and leucine, with either methionine or lysine, were added at 120 mg·L⁻¹ to fulfil the auxotrophic requirements.

Cells were grown at 30 °C in both liquid cultures and on agar plates. Liquid cultivations were performed on a rotary shaker in flasks or 15 mL Falcon tubes, depending on the volume of the cultures.

Screening of growth using methionine auxotrophic strains was performed in 350 μL cultures with a starting absorbance at 610 nm of 0.04–0.06, and followed for 36 or 48 h (Bioscreen C, Thermic Labsystems Oy, Helsinki, Finland) as described in ref. [41]. To obtain absorbance values, the raw data of the Bioscreen (BS) were recalculated to adjust for nonlinearity, because of higher cell densities, according to absorbance = (BS – 0.067) + 0.8324057(BS – 0.067)³. The maximal specific growth rates were calculated by linear regression of the natural logarithm of absorbance values

### Crude cell extraction

The minimal medium was supplemented with the amino acid mixture for the growth of the BY strains with lysine requirement (i.e. the $MAT \alpha$ strains) and the YPH501, $bat1\Delta$, $bat2\Delta$ and $bat1\Delta bat2\Delta$ strains. The whole inocula (2 mL in 15 mL Falcon tubes, grown for at least 7 h) were used to start the main cultures (50 mL in total). The cultures were grown to an absorbance at 610 nm of 1.5–3 in the glucose phase. Glucose still left was checked with Diabur-Test® 5000 test-strips (Roche Diagnostics Scandinavia AB, Bromma, Sweden). Crude cell extracts were prepared from 50 mL of culture in KH₂PO₄ buffer, pH 7.5, as described previously [42], but with protease inhibitors Complete, EDTA-free (Roche Diagnostics GmbH, Mannheim, Germany). Protein was determined by the method of Lowry [43].

### Enzyme activities

The amino acid donors were screened by the addition of 100 μL of crude extract, appropriately diluted (giving a final concentration of 0.2–0.4 g protein·L⁻¹), to 700 μL of reagent, resulting in a reaction mixture containing 50 mM KH₂PO₄ (pH 7.5), 50 μM pyridoxal-5-phosphate, 1 mM Na-MOB and a mixture of amino acids (ADEFGHIKL-NPQRSTVWY), each at 2 mM, except for tyrosine which was present at 1 mM. This basic reaction buffer was also used as a base for the assay of individual amino acids, as

Enzymes of methionine salvage pathway

described later in this section. Samples were taken at the start and end of incubation (30 °C, 30 min), and reactions were stopped by the addition of 100 μL of reaction mixture to 200 μL of sulfosalicylic acid solution (30 g·L⁻¹ with 0.5 mM norleucine). The samples were mixed, frozen and analysed for amino acids and ammonium by an amino acid analyser at the Amino Acid Laboratory of Uppsala University (Uppsala, Sweden), as described previously [44]. In the same way, a mixture was used to study branched chain amino acid transaminases containing only leucine, isoleucine, valine, proline at 2 mM and lysine at 0.5 mM.

The measurement of the activity of single amino acids was performed in basic reaction buffer containing 2 mM of the single amino acid (1 mM of tyrosine), incubated at 30 °C for appropriate periods of time. NADH-coupled assays were measured on microtitre plates with a Fluostar spectrophotometric plate reader (BMG Labtechnologies, Offenburg, Germany) equipped with an internal reagent pump for the addition of starting substrate (amino acid and MOB). Enzyme activity (μmol·min⁻¹) was determined from the difference in slope of NADH absorbance (340 nm, e = 6.3 mM⁻¹·cm⁻¹) before and after addition of the substrate. For glutamate-coupled activity, the final reaction mixture also contained 50 mM NH₄Cl, 0.2 mM NADH, 1 mM NaN₃ and 10 μmol·min⁻¹·mL⁻¹ glutamate dehydrogenase [NAD(P)-dependent, type III bovine liver, Sigma-Aldrich Chemie GmbH], and, for aspartate-coupled activity, 0.2 mM NADH and 3.75 μmol·min⁻¹·mL⁻¹ malate dehydrogenase (from pig heart, Roche Diagnostics GmbH).

Tyrosine-coupled activity was assayed using a modified Diamondstone assay [15,45] in basic reaction buffer (1 mM of tyrosine).

Phenylalanine-, tryptophan- and histidine-coupled activities were assayed as their enol-borate complexes [46,47] with slight modifications. The reaction buffer was basic reaction buffer, except for histidine-coupled activity when the buffer also contained 23 mM Na₂EDTA. After incubation at 30 °C, the reaction was stopped in 1 mL of reaction mixture by the addition of 300 μL of 200 g·L⁻¹ *meta*-phosphoric acid. After centrifugation, two samples of supernatant (160 μL) were added to either 960 μL of borate/Na-arsenate (1 M/2 M, pH 6.5) or 960 μL of Na-arsenate (2 M, pH 6.5). The mixed samples were incubated at room temperature for 15 min before absorbance measurements (histidine, 293 nm, e = 12.0 mM⁻¹·cm⁻¹; phenylalanine, 300 nm, e = 9.15 mM⁻¹·cm⁻¹; tryptophan, 332 nm, e = 12.7 mM⁻¹·cm⁻¹). The absorbance of the borate-arsenate sample was corrected by the corresponding arsenate sample.

The leucine-coupled activity was measured (based on ref. [48]) as follows. A sample of crude extract was mixed with the reagent mixture. The reaction was started by the addition of Na-MOB, and samples were taken at the start and end of incubation. Samples (300 μL) were added to 300 μL of 100 g·L⁻¹ trichloroacetic acid, vortexed and centrifuged. The supernatant (500 μL) was transferred to a glass tube and 1 mL of 2,4-dinitrophenylhydrazine (3 g·L⁻¹ in 2 M HCl) was added. After incubation for 5 min at room temperature, 2.5 mL of cyclohexane (selective for 2-oxoisocaproate [49]) was added and the tubes were mixed vigorously for 2 min. The bottom aqueous layer was removed by a pipette, 2.5 mL of 0.5 M HCl was added and the tubes were mixed for another 1 min. A 1 mL sample of the organic phase was transferred to a new glass tube, 1 mL of 100 g·L⁻¹ Na₂CO₃ was added and the tube was mixed. After the separation of the phases, a 600 μL sample of the aqueous phase was taken and mixed with 300 μL of 2 M NaOH. The absorbance was measured at 440 nm (e = 14 mM⁻¹·cm⁻¹).

Deaminase activity was determined by mixing a sample of crude extract with the reagent mixture containing serine or threonine (as a control, lysine or proline was used instead). The reaction was started by the addition of Na-MOB and samples were taken at the start and end of incubation: volumes 1 : 1 with 100 g·L⁻¹ trichloroacetic acid. After centrifugation, the supernatant was neutralized with KOH and the released ammonium was determined by an enzymatic kit assay (Ammonia, E1112732, R-Biopharm GmbH, Darmstadt, Germany).

### Intracellular MTA analysis

The methionine-requiring strains of deletion mutants were grown on methionine-containing medium, and crude extractions of cells were performed as described above. The extracts were diluted to a final concentration of approximately 2.0 g protein·L⁻¹ with buffer (40 mM KH₂PO₄, 1 mM MgCl₂, pH 7.5) and spiked with 0.25 mM of MTA. Proteins were precipitated with two volumes of cold methanol (500 μL with 1 mL). The mixed samples were allowed to stand on ice for some time before centrifugation (20 000 g, 10 min) and removal of precipitated proteins. Methanol was evaporated in a DNA SpeedVac 120 (Savant Instruments Inc., Holbrook, NY, USA) and the sample was diluted to 1 mL with water. Storage was performed at –20 °C before analysis by ion-pairing reverse phase HPLC. The samples were run on a Waters HPLC system (Waters Sverige AB, Sollentuna, Sweden): Pump 510, Autosampler 717 plus, temperature control module, absorbance detector 486 and MILLENIUM³² (version 3.05.01) software. The analysis of a 25 μL sample was performed using a Hypersil BDS C18 5 μm column (Thermo Fisher Scientific Inc., Waltham, MA, USA), C18 ODS precolumn (Phenomenex, Torrance, CA, USA), an eluent containing 10 mM potassium acetate, pH 4.0, 5 mM sodium 1-heptane sulfonate and 9% (v/v) acetonitrile, at 25 °C, a flow rate of 0.7 mL·min⁻¹ with detection at 254 nm. The chromatograms were recorded for 30 min. Quantification was performed using MTA standards between 2 and 150 μM, dissolved in water. The level of the wild-type was subtracted before normalizing with the amount of protein for each strain.

Acknowledgements

The Swedish Governmental Agency for Innovation Systems (Vinnova) is acknowledged for financial support. We are grateful to Professor Roland Lill, Philipps-University, Marburg, Germany, for providing the *bat4* and corresponding wild-type strains. We also wish to thank Professor Claes Niklasson, Department of Chemical Reaction Engineering, Chalmers University of Technology, Göteborg, Sweden, for lending us the HPLC equipment.

References

1. Thomas D, Becker A & Surdin-Kerjan Y (2000) Reverse methionine biosynthesis from S-adenosylmethionine in eukaryotic cells. *J Biol Chem* 275, 40718–40724.
2. Thomas D & Surdin-Kerjan Y (1997) Metabolism of sulfur amino acids in *Saccharomyces cerevisiae*. *Microbiol Mol Biol Rev* 61, 503–532.
3. Lu SC (2000) S-Adenosylmethionine. *Int J Biochem Cell Biol* 32, 391–395.
4. Marzluf GA (1997) Molecular genetics of sulfur assimilation in filamentous fungi and yeast. *Annu Rev Microbiol* 51, 73–96.
5. Chattopadhyay MK, Tabor CW & Tabor H (2003) Spermidine but not spermine is essential for hypusine biosynthesis and growth in *Saccharomyces cerevisiae*: spermine is converted to spermidine *in vivo* by the FMS1-amine oxidase. *Proc Natl Acad Sci USA* 100, 13869–13874.
6. Tabor CW (1981) Mutants of *Saccharomyces cerevisiae* deficient in polyamine biosynthesis: studies on the regulation of ornithine decarboxylase. *Med Biol* 59, 272–278.
7. Marchitto KS & Ferro AJ (1985) The metabolism of 5′-methylthioadenosine and 5-methylthioribose 1-phosphate in *Saccharomyces cerevisiae*. *J Gen Microbiol* 131, 2153–2164.
8. Sekowska A, Dénervaud V, Ashida H, Michoud K, Haas D, Yokota A & Danchin A (2004) Bacterial variations on the methionine salvage pathway. *BMC Microbiol* 4, 9.
9. Sufrin JR, Meshnick SR, Spiess AJ, Garofalo-Hannan J, Pan XQ & Bacchi CJ (1995) Methionine recycling pathways and antimalarial drug design. *Antimicrob Agents Chemother* 39, 2511–2515.
10. Backlund PS Jr & Smith RA (1981) Methionine synthesis from 5′-methylthioadenosine in rat liver. *J Biol Chem* 256, 1533–1535.
11. Yang SF & Hoffman NE (1984) Ethylene biosynthesis and its regulation in higher plants. *Annu Rev Plant Physiol* 35, 155–189.
12. Berger BJ, English S, Chan G & Knodel MH (2003) Methionine regeneration and aminotransferases in *Bacillus subtilis*, *Bacillus cereus*, and *Bacillus anthracis*. *J Bacteriol* 185, 2418–2431.
13. Berger BJ, Dai W-W & Wilson J (1998) Methionine formation from α-ketomethiobutyrate in the trypanosomatid *Crithidia fasciculata*. *FEMS Microbiol Lett* 165, 305–312.
14. Berger LC, Wilson J, Wood P & Berger BJ (2001) Methionine regeneration and aspartate aminotransferase in parasitic protozoa. *J Bacteriol* 183, 4421–4434.
15. Heilbronn J, Wilson J & Berger BJ (1999) Tyrosine aminotransferase catalyzes the final step of methionine recycling in *Klebsiella pneumoniae*. *J Bacteriol* 181, 1739–1747.
16. Myers RW, Wray JW, Fish S & Abeles RH (1993) Purification and characterization of an enzyme involved in oxidative carbon-carbon bond cleavage reactions in the methionine salvage pathway of *Klebsiella pneumoniae*. *J Biol Chem* 268, 24785–24791.
17. Trackman PC & Abeles RH (1983) Methionine synthesis from 5′-S-methylthioadenosine. Resolution of enzyme activities and identification of 1-phospho-5-S-methylthioribulose. *J Biol Chem* 258, 6717–6720.
18. Dai Y, Wensink PC & Abeles RH (1999) One protein, two enzymes. *J Biol Chem* 274, 1193–1195.
19. Cone MC, Marchitto K, Zehfus B & Ferro AJ (1982) Utilization by *Saccharomyces cerevisiae* of 5′-methylthioadenosine as a source of both purine and methionine. *J Bacteriol* 151, 510–515.
20. Chattopadhyay MK, Tabor CW & Tabor H (2006) Methylthioadenosine and polyamine biosynthesis in a *Saccharomyces cerevisiae meu1Δ* mutant. *Biochem Biophys Res Commun* 343, 203–207.
21. Bumann M, Djafarzadeh S, Oberholzer AE, Bigler P, Altmann M, Trachsel H & Baumann U (2004) Crystal structure of yeast Ypr118w, a methylthioribose-1-phosphate isomerase related to regulatory eIF2B subunits. *J Biol Chem* 279, 37087–37094.
22. Hirano W, Gotoh I, Uekita T & Seiki M (2005) Membrane-type 1 matrix metalloproteinase cytoplasmic tail binding protein-1 (MTCBP-1) acts as an eukaryotic acireductone dioxygenase (ARD) in the methionine salvage pathway. *Genes Cells* 10, 565–574.
23. Subhi AL, Diegelman P, Porter CW, Tang B, Lu ZJ, Markham GD & Kruger WD (2003) Methylthioadenosine phosphorylase regulates ornithine decarboxylase by production of downstream metabolites. *J Biol Chem* 278, 49868–49873.
24. Yagi T, Kagamiyama H & Nozaki M (1982) Aspartate: 2-oxoglutarate aminotransferase from bakers'yeast: crystallization and characterization. *J Biochem* 92, 35–43.
25. Urrestarazu A, Vissers S, Iraqui I & Grenson M (1998) Phenylalanine- and tyrosine-auxotrophic mutants of *Saccharomyces cerevisiae* impaired in transamination. *Mol Gen Genet* 257, 230–237.
26. Jones EW & Fink GR (1982) Regulation of amino acid and nucleotide biosynthesis in yeast. In *The Molecular*

Enzymes of methionine salvage pathway

I. Pirkov *et al.*

Biology of the Yeast Saccharomyces cerevisiae (Strathern JN, Jones EW & Broach JR, eds), pp. 181–299. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

27 Saito Y, Ashida H, Kojima C, Tamura H, Matsumura H, Kai Y & Yokota A (2007) Enzymatic characterization of 5-methylthioribose 1-phosphate isomerase from Bacillus subtilis. Biosci Biotechnol Biochem 71, 2021–2028.

28 Petersen JGL, Kielland-Brandt MC, Nilsson-Tillgren T, Bornaes C & Holmberg S (1988) Molecular genetics of serine and threonine catabolism in Saccharomyces cerevisiae. Genetics 119, 527–534.

29 Ramos F & Wiame JM (1982) Occurrence of a catalytic L-serine (L-threonine) deaminase in Saccharomyces cerevisiae. Eur J Biochem 123, 571–576.

30 Pedersen JØ, Rodriguez MA, Praetorius-Ibba M, Nilsson-Tillgren T, Calderón IL & Holmberg S (1997) Locus-specific suppression of *ilv1* in Saccharomyces cerevisiae by deregulation of *CHA1* transcription. Mol Gen Genet 255, 561–569.

31 Kispal G, Steiner H, Court DA, Rolinski B & Lill R (1996) Mitochondrial and cytosolic branched-chain amino acid transaminases from yeast, homologs of the *myc* oncogene-regulated Eca39 protein. J Biol Chem 271, 24458–24464.

32 Ashida H, Saito Y, Kojima C, Kobayashi K, Ogasawara N & Yokota A (2003) A functional link between RuBisCO-like protein of *Bacillus* and photosynthetic RuBisCO. Science 302, 286–290.

33 Tisdale MJ (1983) Methionine synthesis from 5′-methylthioadenosine by tumour cells. Biochem Pharmacol 32, 2915–2920.

34 Berasain C, Hevia H, Fernandez-Irigoyen J, Larrea E, Caballeria J, Mato JM, Prieto J, Corrales FJ, Garcia-Trevijano ER & Avila MA (2004) Methylthioadenosine phosphorylase gene expression is impaired in human liver cirrhosis and hepatocarcinoma. Biochim Biophys Acta 1690, 276–284.

35 García-Castellano JM, Villanueva A, Healey JH, Sowers R, Cordon-Cardo C, Huvos A, Bertino JR, Meyers P & Gorlick R (2002) Methylthioadenosine phosphorylase gene deletions are common in osteosarcoma. Clin Cancer Res 8, 782–787.

36 Kung PP, Zehnder LR, Meng JJ, Kupchinsky SW, Skalitzky DJ, Johnson MC, Maegley KA, Ekker A, Kuhn LA, Rose PW *et al.* (2005) Design, synthesis, and biological evaluation of novel human 5′-deoxy-5′-methylthioadenosine phosphorylase (MTAP) substrates. Bioorg Med Chem Lett 15, 2829–2833.

37 Bacchi CJ, Goldberg B, Rattendi D, Gorrell TE, Spiess AJ & Sufrin JR (1999) Metabolic effects of a methylthioadenosine phosphorylase substrate analog on African trypanosomes. Biochem Pharmacol 57, 89–96.

38 Avila MA, García-Trevijano ER, Lu SC, Corrales FJ & Mato JM (2004) Methylthioadenosine. Int J Biochem Cell Biol 36, 2125–2130.

39 Tang B, Kadariya Y, Murphy ME & Kruger WD (2006) The methionine salvage pathway compound 4-methylthio-2-oxobutanate causes apoptosis independent of down-regulation of ornithine decarboxylase. Biochem Pharmacol 72, 806–815.

40 Goldstein AL & McCusker JH (1999) Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae. Yeast 15, 1541–1553.

41 Warringer J & Blomberg A (2003) Automated screening in environmental arrays allows analysis of quantitative phenotypic profiles in Saccharomyces cerevisiae. Yeast 20, 53–67.

42 Albers E, Laizé V, Blomberg A, Hohmann S & Gustafsson L (2003) Ser3p (Yer081wp) and Ser33p (Yil074cp) are phosphoglycerate dehydrogenases in Saccharomyces cerevisiae. J Biol Chem 278, 10264–10272.

43 Lowry OH, Rosebrough NJ, Farr AL & Randall RJ (1951) Protein measurement with the folin phenol reagent. J Biol Chem 193, 265–275.

44 Albers E, Larsson C, Lidén G, Niklasson C & Gustafsson L (1996) Influence of the nitrogen source on Saccharomyces cerevisiae anaerobic growth and product formation. Appl Environ Microbiol 62, 3187–3195.

45 Diamondstone TI (1966) Assay of tyrosine transaminase activity by conversion of p-hydroxyphenylpyruvate to p-hydroxybenzaldehyde. Anal Biochem 16, 395–401.

46 Fujioka M, Morino Y & Wada H (1970) Metabolism of phenylalanine (*Achromobacter eurydice*). Methods Enzymol 17, 585–596.

47 Lin ECC, Pitt BM, Civen M & Knox WE (1958) The assay of aromatic amino acid transaminations and keto acid oxidation by the enol-borate tautomerase method. J Biol Chem 233, 668–673.

48 Prohl C, Kispal G & Lill R (2000) Branched-chain amino-acid transaminases of yeast Saccharomyces cerevisiae. Methods Enzymol 324, 365–375.

49 Taylor RT & Jenkins WT (1966) Leucine aminotransferase. I. Colorimetric assays. J Biol Chem 241, 4391–4395.
