
Chemistry & Biology

Article

Amide N-Glycosylation by Asm25,  
an N-Glycosyltransferase of Ansamitocins

Peiji Zhao,$^{1,5}$ Linquan Bai,$^{2,5,*}$ Juan Ma,$^{1}$ Ying Zeng,$^{1}$ Lei Li,$^{2}$ Yirong Zhang,$^{2}$ Chunhua Lu,$^{3}$ Huanqin Dai,$^{1}$ Zhaoxian Wu,$^{3}$ Yaoyao Li,$^{3}$ Xuan Wu,$^{3}$ Gang Chen,$^{1}$ Xiaojiang Hao,$^{1}$ Yuemao Shen,$^{1,3,*}$ Zixin Deng,$^{2}$ and Heinz G. Floss$^{4}$

$^{1}$State Key Laboratory of Phytochemistry and Plant Resources in West China, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650204, China  
$^{2}$Laboratory of Microbial Metabolism, College of Life Science and Biotechnology, Shanghai Jiaotong University, Shanghai 200240, China  
$^{3}$Key Laboratory of the Ministry of Education for Cell Biology and Tumor Cell Engineering, School of Life Sciences, Xiamen University, Xiamen, Fujian 361005, China  
$^{4}$Department of Chemistry, Box 351700, University of Washington, Seattle, WA 98195-1700, USA  
$^{5}$These authors contributed equally to this work.  
*Correspondence: bailq@sjtu.edu.cn (L.B.), yshen@xmu.edu.cn (Y.S.)  
DOI 10.1016/j.chembiol.2008.06.007

---

### SUMMARY

Ansamitocins are potent antitumor maytansinoids produced by *Actinosynnema pretiosum*. Their biosynthesis involves the initial assembly of a macrolactam polyketide, followed by a series of postpolyketide synthase (PKS) modifications. Three ansamitocin glycosides were isolated from *A. pretiosum* and fully characterized structurally as novel ansamitocin derivatives, carrying a β-D-glucosyl group attached to the macrolactam amide nitrogen in place of the N-methyl group. By gene inactivation and complementation, asm25 was identified as the N-glycosyltransferase gene responsible for the macrolactam amide N-glycosylation of ansamitocins. Soluble, enzymatically active Asm25 protein was obtained from asm25-expressing *E. coli* by solubilization from inclusion bodies. Its optimal reaction conditions, including temperature, pH, metal ion requirement, and $K_{m}/K_{cat}$, were determined. Asm25 also showed broad substrate specificity toward other ansamycins and synthetic indolin-2-ones. To the best of our knowledge, this represents the first in vitro characterization of a purified antibiotic N-glycosyltransferase.

---

### INTRODUCTION

Ansamitocins are maytansinoids of microbial origin, produced by *Actinosynnema pretiosum* ssp. *auranticum* ATCC 31565. They differ structurally from their higher-plant congeners, such as the parent compound maytansine, only by carrying simpler acyl groups at C-3 (*Figure 1A*; *Higashide et al.*, 1977). Like the plant maytansinoids, ansamitocins show extremely potent cytotoxicity against various tumor cells (*Ootsu et al.*, 1980). Structurally, the maytansinoids belong to the ansamycin family of polyketide macrolactams (*Rinehart and Shield*, 1976), exemplified by the antitubercular agent rifamycin and the antitumor agent geldanamycin. Their molecular architecture is characterized by an aromatic chromophore with an aliphatic chain (ansa chain) connected back to a nonadjacent position through an amide linkage. Despite carrying either benzenic (ansamitocins, geldanamycin) or naphthalenic (rifamycin, naphthomycin) chromophores, genetic and feeding experiments have revealed that all the ansamycins share the same polyketide starter unit, 3-amino-5-hydroxybenzoic acid (AHBA) (*Ghisalba and Nuesch*, 1981; *Hatano et al.*, 1982; *Kibby et al.*, 1980). The ansamitocin biosynthetic genes were cloned from a cosmid library of *A. pretiosum* genomic DNA using the AHBA synthase gene *rifK* (*Kim et al.*, 1998) from the rifamycin biosynthetic gene cluster as probe (*Yu et al.*, 2002). Notably, two AHBA synthase genes were found, and sequencing of the surrounding DNA localized the genes expected to be required for ansamitocin biosynthesis in two clusters separated by 30 kb of nonessential DNA (*Figure 1B*). Bioinformatic analysis, gene inactivation, and expression experiments demonstrated that cluster I contains most of the ansamitocin biosynthetic genes, whereas cluster II carries only four of the seven genes required for AHBA formation (*Figure 1B*; *Yu et al.*, 2002). AsmA-D are four large open-reading frames (ORFs) encoding the loading domain and seven chain-elongation modules of a type I multifunctional polyketide synthase (PKS). Gene *asm9*, located immediately downstream of *asmD*, encodes the amide synthase responsible for macrolactamization of the nascent polyketide chain and release of proansamitocin from the PKS (*Kato et al.*, 2002; *Yu et al.*, 2002).

As in other microbial secondary metabolites, post-PKS modifications are important for the biological activity of the ansamitocins (*Cassady et al.*, 2004). Six modifying reactions are required to turn proansamitocin into ansamitocins, and cluster I contains candidate genes for each of these. Their functions and the order in which they act were determined by in vivo gene inactivation, feeding experiments with accumulated compounds, and in vitro enzymatic analysis of heterologously expressed gene products. This defined a predominant pathway from proansamitocin to ansamitocins, consisting of chlorination (Asm12), carbamoylation (Asm21), O-methylation (Asm7), acylation (Asm19), epoxidation (Asm11), and N-methylation (Asm10) (*Moss et al.*, 2002; *Spiteller et al.*, 2003). Notably, the only ansamitocin products of the *asm10* deletion mutant are macrolactam amide

Chemistry & Biology

Amide N-Glycosylation

A

R = CH₃          AP-1
R = CH₂CH₃       AP-2
R = CH(CH₃)₂     AP-3
R = CH₂CH(CH₃)₂  AP-4

PND-1
PND-2
PND-3
PND-4

AGP-1
AGP-2
AGP-3
AGP-4

B

Cluster I

C

D

0kb

PKS
AHBA
Modification
MM-ACP
Regulation
Transporter

Cluster II

0kb

12kb

Figure 1. The Post-PKS Modifications of N-Desmethylansamitocins and the Ansamitocin Biosynthetic Gene Cluster

(A) Chemical structures of ansamitocins (APs), N-desmethylansamitocins (PNDs), and ansamitocosides (AGPs). The putative scheme for the conversion of PNDs to APs and AGPs by Asm10 and Asm25, respectively.

(B) The ansamitocin biosynthetic gene cluster. The N-glycosyltransferase gene, asm25, and the N-methyltransferase gene, asm10, are indicated by a circle in Cluster I. AHBA: 3-amino-5-hydroxybenzoic acid; MM-ACP: methoxy-malonyl-ACP.

N-desmethylansamitocins (PNDs) (Figure 1A), indicating that N-methylation is the terminal step in the post-PKS modifications of ansamitocin biosynthesis (Spiteller et al., 2003).

Structural variations of the ansamitocins reside mainly in the C-3 acyl moiety. Other minor components with hydroxyl functions at C-15 or the C-14 methyl group, or lacking the N-methyl group, were also found in A. pretiosum (Izawa et al., 1981). Recently, from the cultures of A. pretiosum on solid ISP2 medium, we have isolated a series of novel N-glycosylated ansamitocins with a β-D-glucosyl residue attached to the macrolactam amide nitrogen instead of the N-methyl group (Figure 1A). Ansamitocosides P-1 and P-2 (AGP-1, AGP-2), the main components, were fully characterized structurally (Lu et al., 2004; Ma et al., 2007). In the present work, we show through gene inactivation and complementation the gene encoding the unique amide N-glycosyltransferase (N-Gtf) to be asm25, and we report the overexpression, purification, and characterization of Asm25. Soluble active Asm25 protein was purified from inclusion bodies formed in asm25-expressing E. coli, which is unprecedented in studies of antibiotic N-Gtfs. Additionally, the substrate range of

Asm25 was probed with other ansamycins and synthetic lactams (indolin-2-one derivatives).

RESULTS

Isolation and Structure Elucidation of Ansamitocoside P-3

Previously, we reported the isolation of two unique ansamitocin N-glycosides, macrolactam amide N-desmethyl-N-β-D-glucopyranosylansamitocins P-1 and P-2, named as ansamitocosides P-1 (AGP-1) and P-2 (AGP-2) from A. pretiosum ATCC 31565 (Lu et al., 2004; Ma et al., 2007), and detected additional ansamitocosides by LC-MS. In the present work, one more glycoside of a macrolactam amide N-desmethylansamitocin, namely ansamitocoside P-3 (AGP-3), was isolated from the ethyl acetate extract of A. pretiosum cultivated on solid ISP2 medium by improved preparative TLC methods (Ma et al., 2007). The molecular formula of AGP-3 was determined to be C₃₇H₅₁N₂O₁₄ClNa (m/z 805.2917 [M + Na]⁺, calculated: 805.2926) based on HR-ESI-MS data. The ¹³C-NMR and DEPT

864 Chemistry & Biology 15, 863–874, August 25, 2008 ©2008 Elsevier Ltd All rights reserved

Chemistry & Biology

Amide N-Glycosylation

A

![Diagram A](image1.png)


B

![Diagram B](image2.png)


C

![Diagram C](image3.png)


Figure 2. Inactivation of asm25 and Complementation of the Mutant

(A) Schematic representation of the 465 bp deletion within asm25 from the A. pretiosum genome. In shuttle plasmid pJTU478, a truncated asm25 was generated by linking the 2.7 kb and 3.4 kb genomic fragments originally flanking the deleted 465 bp region. S, Sacl; P, Pstl.

(B) PCR analysis of wild-type A. pretiosum and mutant BLQ11. Whereas wild-type A. pretiosum gave a 1.60 kb PCR-amplified product, mutant BLQ11 yielded a 1.10 kb product as expected.

(C) ESI-MS profiles of the ansamitocin products of the wild-type, asm25 deletion mutant BLQ11, and asm25-complemented strain BLQ15 cultured on solid ISP2 medium.

spectra of AGP-3 show 37 carbon signals, including 7 methyls, 4 methylenes, 16 methines, and 10 quaternary carbons (see Table S1A available online). But in the ${}^{1}$H-NMR spectra, the proton signal attributed to $\mathrm{CH}_{3} \mathrm{~N}$ at $\delta$ 3.18~3.22 in ansamitocin P-3 (AP-3) (Kupchan et al., 1977) was missing in AGP-3, indicating the absence of the $N$-methyl group in AGP-3. In addition, one more six-carbon unit ($\delta$ 85.3, 71.6, 80.0, 71.5, 79.5, 63.3) was assigned to be a hexosyl moiety. The sugar was determined to be a $\beta$-D-glucopyranosyl moiety based on the unambiguous NMR assignments. Particularly, the $J_{\mathrm{H}-1^{\prime \prime},-2^{\prime \prime}}=9.4 \mathrm{~Hz}$ indicated a $\beta$-glucosidic linkage. The HMBC correlations between the anomic proton at $\delta 5.55(\mathrm{H}-1^{\prime \prime})$ and the carbons at $\delta 172.6(\mathrm{C}-1)$ and $\delta 138.3(\mathrm{C}-18)$ revealed the linkage of the glucose unit to the aglycone, $N$-desmethylansamitocin P-3 (PND-3), via the macrolactam amide nitrogen. Therefore, AGP-3 was determined to be $N$-desmethyl-$N$-$\beta$-D-glucopyranosylansamitocin P-3 (Figure 1A).

**asm25 Is Involved in the Formation of Ansamitocosides**

Early work had established methylation of the macrolactam amide nitrogen as the terminal step in post-PKS modifications and identified PND-3 as the last intermediate in ansamitocin biosynthesis (Spiteller et al., 2003). The glycosylation of the same amide nitrogen could be an alternative step to the methylation catalyzed by the $N$-methyltransferase Asm10. Sequence analysis of the asm gene cluster had identified one gene of unknown function, asm25, with high sequence homology to glycosyltransferase (Gtf) genes (Walsh et al., 2003), which at the time had been speculated to be part of a glycosylation/deglycosylation excretion system (Figure 1B; Yu et al., 2002). To investigate whether asm25 is responsible for the $N$-glycosylation of PNDs, this gene was inactivated through a 465 bp in-frame deletion from 429 bp to 894 bp (Figure 2A). The resulting mutant strain BLQ11 was confirmed by PCR amplification using the total genomic DNA as template. On the electrophoretic gel, BLQ11 gave the 1.1 kb expected product, whereas the wild-type showed a 1.6 kb product (Figure 2B). The extracts of both BLQ11 and wild-type strains grown on solid ISP2 medium were analyzed by LC-ESI-MS, monitoring at the quasimolecular ion peaks of ansamitocins (APs), ansamitocosides (AGPs), and $N$-desmethylansamitocins (PNDs), respectively. The production of AGP-1, AGP-2, AGP-3, and AGP-4 (at $m / z$ 755.3 [M + H]${}^{+}$, 769.3 [M + H]${}^{+}$, 783.3 [M + H]${}^{+}$, and 797.3 [M + H]${}^{+}$, respectively) was observed in the wild-type but abolished in BLQ11 (Figure 2C).

Further support for the involvement of asm25 in the $N$-glycosylation at the macrolactam amide nitrogen came from

Chemistry & Biology 15, 863–874, August 25, 2008 ©2008 Elsevier Ltd All rights reserved 865

Chemistry & Biology

Amide N-Glycosylation

Figure 3. Properties of the Renatured Asm25

(A) A representative Coomassie brilliant blue-stained SDS-PAGE gel loaded with molecular weight markers (lane 1), renatured Asm25 (lane 2), inclusion bodies (lane 3), soluble protein (lane 4), and cell-free extract (lane 5) fractions.

(B) Temperature optimum for the macrolactam amide N-glycosylation of PND-3 catalyzed by Asm25.

(C) Determination of pH optimum for the macrolactam amide N-glycosylation of PND-3 catalyzed by Asm25. Asm25 was assayed in 50 mM Na-acetate/citric acid buffer ranging from pH 4.0 to 7.0 (open column), and 50 mM Tris-HCl buffer ranging from pH 7.0 to 9.0 (solid column).

(D) Effect of metal ions on the activity of Asm25. Asm25 was assayed in the absence (0 mM, green) or presence of various metal ions at 0.1 (blue), 1.0 (pink), and 10.0 mM (black) concentrations except for the concentration of Mg²⁺ used at 10 (green and blue), 50 (pink), and 100 mM (black). The productions of AGP-3 were quantitated by LC-MS with authentic AGP-3 as external standard (see Supplemental Data).

complementation of mutant BLQ11. An intact copy of asm25 under the control of the PermE* promoter was introduced into BLQ11 by conjugation using an integrative vector pJTU813, which generated a derivative named as BLQ15. LC-MS analysis of the extract of BLQ15 showed a chromatogram nearly identical to that of the wild-type; that is, the production of AGPs was fully restored by expression of the cloned asm25 (Figure 2C).

### In Vitro N-Glycosylation of PNDs by Asm25

To verify that Asm25 indeed catalyzes the glycosylation at the macrolactam amide nitrogen of ansamitocins, the asm25 gene was amplified by PCR from the *A. pretiosum* genome and cloned into plasmid pRSET-B for heterologous expression. The expression plasmid pJTU812 was transformed into *E. coli* BL21(DE3)-plysE and induced at different temperatures and/or with different concentrations of isopropyl-β-D-thiogalactopyranoside (IPTG). Asm25 was successfully overexpressed as an N-terminally His₆-tagged fusion protein as detected by SDS-PAGE. The estimated molecular weight of Asm25 recombinant protein is 45 kDa from the SDS-PAGE (Figure 3A), and its theoretical molecular weight is 45679.85 Da.

The yield of recombinant protein was highest when the cells were grown at 37°C at different concentrations of IPTG, but as shown in Figure 3A, almost all of it was present as inclusion bodies. In contrast, cells growing at 16°C or 20°C induced with 0.8 mM IPTG produced only minute amounts of soluble Asm25, which showed weak catalytic activity when incubated with PND-3 and UDP-Glc (data not shown). After many unsuccessful attempts to purify the soluble recombinant Asm25 by metal ion affinity chromatography, the inclusion bodies were chosen for the preparation of Asm25. Soluble His₆-Asm25 protein with the best enzymatic activity was obtained from the inclusion bodies by washing with 2.0 M urea and 0.5% Triton, solubilization in 8.0 M urea, and finally dialysis against renaturation buffer IV with glycerol and glycine (see Experimental Procedures). The solution of Asm25 (1.6 μM) in buffer IV with 30% glycerol and 1% bovine serum albumin (BSA) was used for the enzyme assays.

Based on the structures of the ansamitocinosides and the known pathway of ansamitocin biosynthesis (Spiteller et al., 2003), it was assumed that PNDs are the aglycone substrates for Asm25. Using as substrate PND-3 isolated from the asm10 mutant grown on solid ISP2 medium, the enzymatic properties of Asm25 were examined. The reactions were performed with 10 μl PND-3 (20 mM in DMSO), 10 μl UDP-Glc (40 mM; Sigma, St. Louis, MO), and 0.1 ml basal reaction solution (1.6 μM Asm25, 50 mM Tris-HCl [pH 8.0], 30% glycerol, 1% BSA, 0.1 M NaCl, 0.1 M glycine, 0.5 mM EDTA, 1.0 mM DDT, and 10 mM MgCl₂). The reaction product was identified as AGP-3 by MS (Figure S2B), NMR, and HPLC comparison (Figures S2H–M and Table S1B), and quantitated by LC-MS with authentic AGP-3. Based on the reaction temperature profile from 0°C to 50°C, Asm25 purified from the inclusion bodies was optimally active at 37°C and exhibited more than 70% of its maximal activity between 35°C and 42°C (Figure 3B). The pH profile was determined at 37°C in 50 mM Na-acetate/citric acid buffer (pH 4.0–7.0) and 50 mM Tris-HCl buffer (pH 7.0–9.0). Asm25 showed optimal activity at pH 7.5 and more than 70% of maximal activity in the pH range of 6.0–8.0 (Figure 3C).

Because many enzymes that bind dinucleotide substrates require divalent, oxophilic cations, such as Mg²⁺ or Mn²⁺, we examined the effects of metal ions on the activity of Asm25. The reactions were performed in the basal reaction solution containing 1.54 mM PND-3 and 3.08 mM UDP-Glc for 4 h at 37°C. Enzymatic activity improved with the addition of Li⁺, Zn²⁺, or Mn²⁺ at a concentration of 0.1 mM. Negligible effects on the catalytic activity were observed as the concentrations of Li⁺ and Mn²⁺ increased to 10 mM. However, the activity was partially inhibited by 0.1 mM Hg²⁺ or Cu²⁺, and almost completely inhibited by 1 mM Hg²⁺, Zn²⁺, or Cu²⁺ (Figure 3D).

Chemistry & Biology

Amide N-Glycosylation

**Figure 4. Kinetic Characterization of Asm25**

(A) Initial reaction rates were determined with PND-3 at 20, 30, 40, and 50 μM and UDP-Glc at 0.2, 0.5, 0.75, 1.0, and 2.0 mM, respectively. Mean values of three independent experiments with SD indicated by error bars.

(B) Secondary plot of intercept versus 1/[PND-3] and slope versus 1/[PND-3].

**Substrate Specificity of Asm25**

The aglycone specificity of Asm25 was evaluated using six maytansinoids (see the LC-MS profiles in Figure S1), including PND-1 to -4, N-desmethyl-deseoxymaytansinol (DDM) and N-desmethyl-deseoxyansamitocin P-1 (DDP-1) to react with UDP-Glc, respectively. The glycosylated products, AGP-1, -2, -3, -4, corresponding to the four PNDs, were detected by LC-MS (Figure 5A, panels 1–4) and HPLC (Figure S3). By comparison of the relative signal intensities of the product ions, LC-MS analysis indicated that the glycosylation of PND-2 to AGP-2 gave the highest yield. This is consistent with the competition reaction using equimolar amounts of all four PNDs, which showed that among these PND-2 was the preferred substrate of the enzyme (Figure 5A, panel 5). In contrast, no conversion of DDM to its corresponding glycoside was detected (data not shown). A glycosylation product of DDP-1 was detected, but the conversion efficiency was extremely low (data not shown). Therefore, among the substrates tested, PND-2 was the best sugar acceptor for the glycosylation reaction catalyzed by Asm25.

The sugar specificity of Asm25 was probed using PND-3 and four UDP-sugars, UDP-Glc, UDP-galactose, UDP-N-acetyl-glucosamine and UDP-glucuronic acid, and ADP- and GDP-Glc as substrates. A glycosylated product could only be detected by LC-MS in the assay with UDP-Glc as sugar donor (Figure 5B and Figure S4). Because no glycosylated products were formed in the other five reactions, UDP-galactose, UDP-N-acetyl-glucosamine, UDP-glucuronic acid, ADP- or GDP-Glc cannot replace UDP-Glc in the Asm25-catalyzed glycosylation of PND-3. Therefore, we conclude that Asm25 catalyzes the in vitro glycosylation of PNDs at the macrolactam amide nitrogen using UDP-Glc as the sole sugar donor.

**Kinetics of Renatured Asm25 with UDP-Glc and PND-3 as Cosubstrates**

Kinetic studies of the reaction catalyzed by the refolded Asm25 were performed using UDP-Glc as sugar donor and PND-3 as sugar acceptor. When the initial rate was measured by varying the concentrations of PND-3 at different fixed concentrations of UDP-Glc, a set of intersecting lines were obtained by regression analysis of the data at each UDP-Glc concentration (Figure 4A). Double reciprocal plots of the initial rate data are shown in Figure 4A. The lines in the double reciprocal plots converged, though not to a single point, implying that Asm25 obeys a ternary complex mechanism in which both substrates bind prior to product release. In general, two basic types of mechanisms, ternary complex and “ping-pong” mechanism, can be deduced from kinetic data based on the lines in double reciprocal plots. Lines converging to a single point indicate a ternary complex mechanism, whereas parallel lines suggest a ping-pong mechanism. The lines in Figure 4A do not converge to a single point; however, they show the tendency to converge, which differs from the parallel pattern. Therefore, a ternary complex mechanism can be concluded for Asm25. This was also seen in the Arabidopsis indole-3-acetic acid Gtf, which has a mechanism similar to Asm25 (Jackson et al., 2001). The slope replot does not pass through the origin, indicating that the glycosylation is not of a rapid equilibrium-ordered mechanism (Figure 4B). The reaction thus proceeds through a ternary enzyme-substrate complex (Lairson et al., 2008; Marangoni, 2003; Price and Stevens, 1982). The recombinant Asm25 exhibited Michaelis-Menten kinetics. The V_max and K_m values were determined by classical initial rate experiments and calculated from Lineweaver-Burk plots (Price and Stevens, 1982). The K_m for PND-3 and UDP-Glc were 17.8 μM and 87.3 μM, respectively, with k_cat of 1.3 · 10⁻² · s⁻¹.

**In Vitro Glycosylation of Other Ansamycins and Synthetic Lactams by Asm25**

To evaluate the substrate range of Asm25 further, five ansamycins—geldanamycin-derived lebstatin and 17-O-desmethylllebstatin (17-DMLB) (Figure 6; Stead et al., 2000), rifamycin A (Sensi et al., 1959), rifampicin (Morisaki et al., 1995), naphthomycin A (Lu and Shen, 2007)—and two synthetic lactams (indolin-2-one derivatives HC-14 and HC-52; see Supplemental Data) were used as sugar acceptors at 8.33 mM in standard assays with UDP-Glc. Rifamycin A, rifampicin, and naphthomycin A were not converted to the corresponding glycosides (data not shown).

Chemistry & Biology
Amide N-Glycosylation

A
Relative Abundance
1.62E5
100
60
20
0
2.10E5
100
60
20
0
1.29E5
100
60
20
0
1.15E6
100
60
20
0
9.42E4
100
60
20
0
m/z
Panel-1
Panel-2
Panel-3
Panel-4
Panel-5
PND-1
PND-2
PND-3
PND-4
AGP-1
AGP-2
AGP-3
AGP-4
593.2
615.4
575.2
693.3
737.3
777.4
769.3
791.4
607.2
667.2
707.3
643.4
621.2
783.3
805.4
765.3
635.3
657.4
779.3
819.4
621.2
643.4
607.2
635.2
673.4
687.4
751.3
737.3
755.3
791.5
805.4
769.3
797.3
B
Relative Abundance
1.14E5
100
60
20
0
1.38E5
100
60
20
0
1.83E5
100
60
20
0
1.96E5
100
60
20
0
m/z
UDP-glucose
UDP-galactose
UDP-N-acetyl
glucosamine
UDP-glucuronic
acid
PND-3
643.4
621.2
603.3
765.3
783.3
805.4
621.3
643.4
603.2
720.3
621.3
643.4
603.3
658.4
742.2
621.3
643.4
603.4
705.2
600
650
700
750
800
850

Figure 5. Substrate Specificity of Asm25 for Different Ansamitocin-Derived Aglycones and Sugar Donors

(A) ESI-MS spectra of the reaction products of Asm25 with different ansamitocin-derived aglycones. Panels 1–4: Incubations with different PNDs. Panel 5: Incubation with equimolar mixture of four PNDs. All reactions were carried out in standard assays, at least in duplicate (see Supplemental Data).

(B) ESI-MS spectra of the reaction products of Asm25 with different UDP-sugar donors. For assaying different sugar donors, 10 μl of sugar nucleotide (0.15 mM final concentration) and 10 μl PND-3 (0.04 mM final concentration) were added to 0.1 ml basal reaction solution for a 0.12 ml final volume. The reactions were terminated after 2 h at 37°C and the products were detected by LC-MS. Each assay was carried out at least in duplicate.

However, lebstatin, 17-DMLB, HC-14, and HC-52 were glycosylated by Asm25. In the LC-MS spectra, quasimolecular ion peaks were seen for lebstatin at m/z 571.4 [M + Na]+ and for the expected glycosylated product at m/z 733.4 [M + Na]+. Similarly, in the LC-MS spectra of the incubation with 17-DMLB, the glycosylated product was detected at m/z 719.4 [M + Na]+. Moreover,

868 Chemistry & Biology 15, 863–874, August 25, 2008 © 2008 Elsevier Ltd All rights reserved

Chemistry & Biology

Amide N-Glycosylation

Figure 6. ESI-MS Spectra of the Glycosylation on Selected Nonansamitocin Aglycones by Asm25

17-DMLB, lebstatin, HC-14, and HC-52 were used as sugar acceptors at 8.33 mM in standard assays with UDP-Glc, respectively (see Supplemental Data).

Chemistry & Biology 15, 863–874, August 25, 2008 © 2008 Elsevier Ltd All rights reserved 869

Chemistry & Biology

Amide N-Glycosylation

Figure 7. Antifungal and Antitumor Activities of AP-3, PND-3, and AGP-3

(A) Antifungal activity against *F. uniguttulatum* IFO 0699 of AP-3, PND-3, and AGP-3.

(B) The antitumor activities of AP-3, PND-3, and AGP-3 against HepG2 cells. The IC₅₀ of AP-3, PND-3, and AGP-3 were calculated to be 0.2, 0.1, and 2.0 μg/ml, respectively.

ansamitocins are limited mainly to the C-3 acyl moieties and to the omission of one or two of the post-PKS modification steps. Our isolation from *A. pretiosum* of three novel ansamitocin amide N-glycosides, AGP-1, AGP-2, and AGP-3 (Figure 1A), revealed a novel post-PKS modification reaction not heretofore encountered in ansamitocin biosynthesis (Lu et al., 2004; Ma et al., 2007).

Bioinformatic analysis of the asm gene cluster had revealed the presence of asm25 encoding a putative Gtf, but its function was obscure (Yu et al., 2002). Asm25 showed highest sequence similarity to CalG1 in the calicheamicin biosynthetic gene cluster cloned from *Micromonospora echinospora* ssp. calichensis (27% identity, 42% similarity) (Ahler et al., 2002). In the present work, the Asm25 protein has now been characterized as a dedicated tailoring Gtf for ansamitocins. Gtfs (EC 2.4.x.y) constitute a large family of enzymes that are involved in the biosynthesis of oligosaccharides, polysaccharides, glycoconjugates, and other natural products. Glycosyltransfers are nucleophilic replacement reactions between C-1 of nucleotide-activated sugars and aglycones that usually carry nucleophilic hydroxyl substituents. However, less frequently, amines and nucleophilic carbons can also serve as sugar acceptors, such as in plant xenobiotic metabolism by the bifunctional O- and N-Gtf UGT72B1 (Brazier-Hicks et al., 2007), and in the biosynthesis of antibiotics—e.g., rebeccamycin (Sanchez et al., 2002), urdamycin (Hoffmeister et al., 2000), and enterobactin (Fischbach et al., 2005). Many N-glycosides are found among the indolocarbazole antibiotics (Sanchez et al., 2006). The first N-Gtf-encoding gene (*ngt/rebG*) was cloned from *Lechevalieria aerocolonigenes* ATCC 39243 (Hyun et al., 2003; Nishizawa et al., 2005; Ohuchi et al., 2000; Onaka et al., 2003a, 2003b; Sanchez et al., 2002). It is involved in the biosynthesis of the indolocarbazole alkaloid rebeccamycin. Later, three more N-Gtf genes, *staG* (Onaka et al., 2002; Salas et al., 2005), *atmG* (Gao et al., 2006), and *inkG* (Kim et al., 2007), were identified.

So far, the biochemical characteristics and/or substrate specificity profiles of the four known N-Gtfs involved in N-glycosylations of indolocarbazole antibiotics have been investigated only by in vivo experiments due to the difficulty of obtaining soluble active proteins (Sanchez et al., 2005; Zhang et al., 2006). The expression of the *rebG* gene in *E. coli* and in *Streptomyces lividans* led to RebG overproduction as inclusion bodies. All

the corresponding glycosylated products of HC-14 and HC-52 were observed at *m/z* 366.5 [M + H]⁺ and 582.9 [M + Na]⁺, respectively (Figure 6).

**Antifungal and Antitumor Activities of PND-3, AP-3, and AGP-3**

AP-3, AGP-3, and PND-3 were tested for antifungal activity against yeast *Filobasidium uniguttulatum* IFO 0699. AP-3 and PND-3 showed remarkable activity against *F. uniguttulatum* at 50–200 ng/disc, but AGP-3 showed similar activity only at higher concentration (8–10 μg/disc; Figure 7A). This assay indicated that AP-3 possesses the strongest antifungal activity, followed by PND-3 and AGP-3.

The antitumor activities of AP-3, AGP-3, and PND-3 against the HepG2 cell line were assayed by the MTT method (Mosmann, 1983). AP-3 and PND-3 showed similar activities with IC₅₀ of 0.2 μg/ml and 0.1 μg/ml, respectively, which is consistent with values reported for maytansinoids (Sneden and Beemsterboer, 1980). AGP-3 was 10 times less active than AP-3 (Figure 7B). However, the in vivo antitumor activity of AGP-3 would still be worth testing because some maytansinoids have been reported to have low activity during in vitro test systems but a dramatic in vivo antitumor activity (Cassady et al., 2004).

**DISCUSSION**

The post-PKS modification reactions in the biosynthesis of ansamitocins are diverse, including chlorination, carbamoylation, O-methylation, acylation, epoxidation, and N-methylation. Nevertheless, the structural variations among the naturally occurring

efforts to obtain soluble and functional RebG were unsuccessful. As the best outcome, RebG was partially solublized by fusion with maltose binding protein or by coexpression with DnaK/DnaJ chaperones; however, this soluble RebG was inactive during in vitro assays (Zhang et al., 2006). Just as encountered in the heterologous expression of RebG, StaG, and AtmG, the overproduction of Asm25 in E. coli was successful, but produced inclusion bodies. Because in the previous work many strategies had been attempted without any success (Zhang et al., 2006), we decided to focus on refolding Asm25 after purification from the inclusion bodies under denaturing conditions, as this seemed the most straightforward approach.

Our extensive efforts in refolding Asm25 from denatured protein eventually produced soluble active enzyme in reasonable yield (~4.38 mg/l of induced cells). When inclusion bodies were purified, 2 M urea was added in renaturation buffer IIa to further purify Asm25 before solubilizing, and which also removed non-specifically adsorbed proteins. To refold Asm25, we used a procedure similar to that described previously (Yang et al., 2004), except that glycerol, which has long been known to increase protein stability (Gekko and Timasheff, 1981), and glycine were added to renaturation buffer IV. After adding 0.1 M glycine, the activity of Asm25 was increased dramatically (data not shown). The use of low molecular-weight additives such as L-arginine, sugars, glycerol, and sarcosine during the refolding process often helps in improving the yield of active proteins from inclusion bodies (Clark, 1998). These additives influence both the solubility and stability of the unfolded protein, folding intermediates, and the fully folded protein. With the soluble Asm25 available, we characterized the N-glycosylation reaction in vitro.

As anticipated, we found that UDP-Glc is the sugar donor and PNDs are the sugar acceptors for the Asm25-catalyzed glycosylation to generate AGPs (Figure 1A). Using UDP-Glc and PND-3 as cosubstrates, we investigated the kinetics of renatured Asm25 and found that the enzyme catalyzes the N-glycosylation of PND-3 via the formation of a ternary complex (Marangoni, 2003; Price and Stevens, 1982). Ternary complex formation is entirely in agreement with the theory—proposed on the basis of structural and sequence homology studies of Gtfs—that UDP-Glc transferases possess two major functional domains (Mackenzie, 1990). Each of these domains is thought to contain a binding site for one of the cosubstrates, enabling both substrate molecules to bind to the enzyme simultaneously. This ternary complex sequential mechanism is in agreement with the limited studies carried out on other Gtfs (Jackson et al., 2001; Mulichak et al., 2001).

The evaluation of the substrate specificity of Asm25 for various sugar acceptors showed that all four PNDs tested (Figure 1A) are used as substrates, but that the structure of the C-3 acyl group modulates the efficiency of conversion. Both in individual incubations and in a competition experiment with all four compounds, PND-2 was found to be the preferred substrate. This observation suggests that the C-3 acyl group may serve as a binding site in a hydrophobic interaction between PNDs and the aglycone binding pocket of Asm25. Additionally, this result is consistent with the fact that AGP-2 was the most abundant product in the fermentation of A. pretiosum on solid ISP2 medium (data not shown). Two N-desmethylansamitocins lacking the 4,5-epoxy function, DDM and DDP-1 (Figure S1), were

extremely poor substrates, if at all. This further supports the speculation that the macrolactam amide N-methylation and/or N-glycosylation is the final step in the post-PKS modifications of ansamitocins (Spiteller et al., 2003). However, it is rather surprising given the finding that two other benzenic ansamycins, the geldanamycin derivatives lebstatin and 17-DMLB (Figure 5A), do serve as substrates. The site of glycosylation has not been determined with these substrates, but is assumed to be the macrolactam amide nitrogen. Three naphthalenic ansamycins, rifamycin A, rifampicin, and naphthomycin A, however, were not glycosylated by Asm25, although two synthetic lactams, HC-14 and HC-52 (Figure 6), were converted to glycosides with modest efficiency. Though these results do not provide a clear picture of the aglycone binding site, they do indicate considerable promiscuity of the enzyme toward sugar acceptor substrates.

The relative promiscuity of Asm25 with respect to the aglycone substrate contrasts with the rather high specificity of the enzyme for just UDP-Glc as the sugar donor. The UDP derivatives of galactose, glucuronic acid, and N-acetylglucosamine, and ADP- and GDP-Glc, were not used as substrates. Similarly, it has been reported that no sugar moieties other than D-glucose are utilized for the glycosylations catalyzed by RebG (Onaka et al., 2003a, 2003b; Sanchez et al., 2002, 2005; Zhang et al., 2006). Conversely, StaG accepts a variety of sugar moieties, including L-rhamnose, L-olivose, L-digitoxose, and D-olivose during in vivo biotransformations (Salas et al., 2005). An examination of the structures in the PDB database shows that the conformations of UDP-hexose substrates bound to enzymes can vary significantly. The variations occur largely around the diphosphate linkage, which has shallow rotational barriers and the ability to adopt a large number of isoenergetic conformations. Thus, the nucleoside and hexose portions of the substrate can be presented in different orientations depending on the enzyme, reflecting large differences in the substrate binding pockets.

SIGNIFICANCE

Ansamitocins are the microbial versions of natural maytansinoids, a family of 19-membered macrocyclic lactams with extraordinary cytotoxic and antineoplastic activities, which are currently undergoing clinical evaluation as antibody-conjugated antitumor drugs (Widdison et al., 2006). Their biosynthesis, catalyzed by the products of the asm biosynthetic gene cluster, involves the assembly of an initial polyketide macrolactam, followed by a series of post-PKS modifications introducing a chlorine, two methyl groups, a cyclic carbamate, an ester side chain, and an epoxide function to form the ansamitocins. The unexpected isolation of ansamitocosides from A. pretiosum cultivated on solid ISP2 medium implies the further potentials for post-PKS modifications of maytansinoids and calls for alternative strategies for natural product mining. The identification of Asm25 as the N-Gtf adding glucose moiety to the macrolactam amide nitrogen enriches the antibiotic glycosyltransferase, especially the N-Gtf, toolbox. Notably, heterologous expression of asm25 in E. coli and solubilization of the Asm25 protein from the inclusion bodies provided for the first time a recombinant antibiotic N-glycosyltransferase in soluble, enzymatically active form. This allowed the in vitro
characterization of Asm25 to examine the properties of this enzyme and the catalytic mechanism of this unusual C-N bond forming glycosylation. The broad substrate range of the enzyme for different aglycones suggests the possibility of using it to generate N-glucosylated derivatives of ansamycins and other compounds for evaluation as improved bioactive agents or potential prodrugs.

### EXPERIMENTAL PROCEDURES

#### Materials, General Methods, and Instrumentation
Optical rotations were measured with a JASCO DIP-370 digital polarimeter in MeOH solution. Mass spectra were measured on an API Qstar Pulsar spectrometer. NMR spectra were recorded on Bruker AM-400 and DRX-500 NMR spectrometers with TMS as internal standard. UDP-Glc, rifamycin, and rifampicin were purchased from Sigma-Aldrich. Naphthomycin A was isolated from the endophytic Streptomyces sp. CS (Lu and Shen, 2007). The preparation of PND-1-4, DDM, lebstatin, 17-DMLB, HC-14, and HC-52 is described in the Supplemental Data. UDP-galactose, UDP-N-acetyl-glucosamine, UDP-glucuronic acid, ADP-Glc, and GDP-Glc were obtained as a gift from Taifo Mahmud of the Department of Pharmaceutical Sciences, Oregon State University, Corvallis. LC-MS for qualitative and quantitative analyses was carried out using a Waters series HPLC 2695 instrument (Waters Corp.) with a Thermo Finnigan LCQ Advantage (Thermo Finnigan) electrospray ionization mass detector (ion trap). For methods of LC-MS, see Supplemental Data.

#### Strains, Plasmids, Culture Techniques, and Media
Actinosynnema pretiosum ssp. aurantium ATCC 31565 and its derivatives, and Streptomyces hygroscopicus XM201 were grown either on solid or in liquid ISP2 medium (containing 0.4% yeast extract, 1% malt extract, and 0.4% glucose [pH 7.3]) or liquid TSBY medium [30 g/l tryptic soy broth (LabM; Topley House), 10 g/l yeast extract, and 103 g/l sucrose]. Fermentations were carried out for 7 days at 28°C. Subsequent extractions and purifications were performed as described in the Supplemental Data. *E. coli* DH10B (Invitrogen), ET12567(pUZ8002) (MacNeil et al., 1992), and BL21(DE3)pLysE (Invitrogen) were used as hosts for plasmid construction, *E. coli*-Actinosynnema biparental conjugation, and protein overexpression, respectively. The yeast *Filobasidium unigutulatum*IFO0699 was used as indicator strain for ansamitocin and ansamitocoside bioassay, and cytotoxicity assays were performed on a panel of HepG2 cells. pBluescript KS(−) and pET26a were used for plasmid construction. pHGF9053 (Minagawa et al., 2007) was the vector used for the gene inactivation of asm25 and pOJ260′, which is a derivative of pOJ260 (Bierman et al., 1992) with the original KpnI site removed, was used for asm10 inactivation. Integrative plasmid pLB139 (Wilkinson et al., 2002) was used for the complementation of the asm25 mutant. pRSET-B (Invitrogen) was used as vector for protein overexpression in *E. coli*. For the complete list of strains and plasmids used in this article, see Table S5.

#### Isolation and Identification of Ansamitocinosides
The strain A. pretiosum ATCC 31565 was inoculated on a slope of solid ISP2 medium in a test tube and cultivated for 5 days at 28°C to allow seed culture. Solid-state fermentation was performed with solid ISP2 medium (3 L, ~150 petri dishes) for 7 days at 28°C. The culture was extracted five times with EtOAc-MeOH-AcOH (80:15:5, v/v/v) to afford a crude extract of 21.0 g. By following the isolation procedure described previously (Ma et al., 2007), three ansamitocin glucosides, AGP-1 (5.0 mg), AGP-2 (8.0 mg), and AGP-3 (3.0 mg), were obtained. The NMR and MS data of AGP-1 and AGP-2 were identical with the literature (Ma et al., 2007). The ESI-MS and NMR (^1H, ^13C, DEPT, HSQC and HMBC) spectra (Figures S2A–L), HPLC comparison profile (Figure S2M), and NMR assignments (Tables S1A and S1B) for in vivo and in vitro AGP-3 are given in the Supplemental Data.

#### Inactivation of Glycosyltransferase Gene asm25
A 6646 bp Sacl fragment containing asm25 was cleaved from cosmid 19F11 (Yu et al., 2002) of the ATCC 31565 genomic library and cloned into pBluescript KS(−) digested with Sacl, resulting in pJTU476. The second plasmid, pJTU477, was constructed through the ligation of the 2742 bp Sacl-PvuII and 3439 bp PvuII-Sacl fragments of pJTU476 with Sacl-digested pET26a. Transfer of the 6181 bp Sacl fragment from pJTU477 to Sacl-digested pHGF9053 generated pJTU478 for the subsequent inactivation of asm25. pJTU478 was introduced into ATCC 31565 via *E. coli*-Actinosynnema biparental conjugation as described (Kieser et al., 2000). Exconjugants were selected with 15 μg/ml thiostrepton. For the screening for double-crossover mutants, an exconjugant was inoculated into liquid TSBY medium and cultivated for three rounds without the presence of thiostrepton. Then the thiostrepton-sensitive derivatives were analyzed through PCR amplification with a pair of primers (Asm25-Det-F: 5′-CCCCAGCACGGAGGAAGA-3′ and Asm25-Det-R: 5′-AGCGGAGGAGGAGACCCA-3′). The PCR amplification was done in a thermocycler (Thermo, MBS Satellite 0.2) under the following conditions: 35 cycles of 30 s at 94°C, 30 s at 50.5°C, and 90 s at 72°C. The asm25 mutant, named BLQ11, was also analyzed by fermentation and LC-MS analysis.

#### Complementation of Mutant BLQ11 with Cloned asm25
A 1.55 kb DNA fragment containing the asm25 gene was cleaved from pJTU812 by Ndel and EcoRI and cloned into plasmid pLB139 to obtain pJTU813, which was introduced from *E. coli* into BLQ11 through conjugation. Confirmation was carried out through plasmid isolation from the exconjugants, plasmid transformation of *E. coli*, and comparison between the newly purified pJTU813 and the original one by restriction enzyme digestions. An exconjugant carrying the correct plasmid was named as BLQ15 and characterized by fermentation and LC-MS analysis.

#### Cloning and Heterologous Expression of Recombinant His<sub>6</sub>-tagged Asm25
Gene asm25 was amplified from the ATCC 31565 genome by PCR using primers asm25-F (5′-GGATCCACATATGCGCGTTCTGTTCACC-3′, engineered BamHI and Ndel sites underlined) and asm25-R (5′-GAATTCACAC ACCCGCGACGCCTC-3′, engineered EcoRI site underlined). The amplified gene fragment was digested with BamHI and EcoRI and ligated with pRSET-B digested with the same enzymes to generate pJTU812. The sequence of the cloned asm25 was confirmed by DNA sequencing. pJTU812 was transformed into *E. coli* BL21(DE3)pLysE, and transformants were grown in LB medium supplemented with ampicillin (100 μg/ml) and chloramphenicol (17.5 μg/ml) at 37°C and 230 rpm for 12 h, then diluted 1:10 with fresh LB medium. The diluted cultures were grown to OD<sub>600</sub> = 0.4~0.6, then induced with 0.1~0.8 mM IPTG and incubated for 5 h at 37°C, 230 rpm.

#### Purification and Renaturation of Asm25 from Inclusion Bodies
*E. coli* cells (200 ml) induced for 5 h at 37°C were centrifuged at 8,000 g for 15 min and the pellet was resuspended in 3 ml renaturation buffer I (50 mM Tris-HCl [pH 8.0], 300 mM NaCl, 10 mM MgCl<sub>2</sub>). Cells were then frozen in liquid nitrogen, rapidly thawed in 37°C water for three cycles, and then sonicated on ice six times for 15 s each at 15 s intervals. After centrifugation of the lysate at 4°C and 30,000 g for 20 min, the inclusion body pellet was resuspended in 3.0 ml renaturation buffer IIa (50 mM Tris-HCl [pH 8.0], 300 mM NaCl, 10 mM MgCl<sub>2</sub>, 0.5% Triton, 2.0 M urea) for 5 min at room temperature and recovered by centrifugation at 4°C and 15,000 g for 15 min. The same procedure was repeated twice in 3.0 ml renaturation buffer IIb (50 mM Tris-HCl [pH 8.0], 300 mM NaCl, 10 mM MgCl<sub>2</sub>, 0.5% Triton). The purified Asm25 inclusion bodies were suspended in 4.0 ml renaturation buffer III (50 mM Tris-HCl [pH 8.0], 300 mM NaCl, 10 mM MgCl<sub>2</sub>, 8.0 M urea) and centrifuged at 8,000 g for 15 min; the pellet was then discarded. The supernatant was dialyzed against renaturation buffer IV (50 mM Tris-HCl [pH 8.0], 10% glycerol, 100 mM NaCl, 0.1 M glycine, 0.5 mM EDTA, 1.0 mM DDT, 10 mM MgCl<sub>2</sub>) for 24 h at 4°C. After centrifugation for 15 min at 4°C and 20,000 g, the supernatant (12.0 ml) was collected and used for subsequent enzyme assays. Protein purity and molecular mass were examined by SDS-PAGE followed by Coomassie Brilliant Blue staining of the gel. For SDS-PAGE, the protein solution was mixed with Laemmli sample buffer containing β-mercaptoethanol (5%) and incubated at 40°C for 30 min. The protein concentration was measured to be 1.6 μM by the Bradford method or by staining intensity compared with BSA concentration standards. The yield of Asm25 was calculated to be ~4.38 mg/l of induced cells. To make a basal reaction solution for activity assay and storage at 4°C, glycerol and

Chemistry & Biology

Amide N-Glycosylation

BSA were added to the enzyme solution in buffer IV at final concentrations of 30% and 1%, respectively.

**SUPPLEMENTAL DATA**

Supplemental Data include five figures, five tables, Supplemental Experimental Procedures, and Supplemental References, and can be found with this article online at http://www.chembiol.com/cgi/content/full/15/8/863/DC1/.

**ACKNOWLEDGMENTS**

We thank Taifo Mahmud at Oregon State University, Corvallis, for providing UDP-galactose, UDP-N-acetyl-glucosamine, UDP-glucuronic acid, ADP-Glc, and GDP-Glc. This work was partially supported by the National Natural Science Fund for Distinguished Young Scholars to Y. Shen (30325044), the Natural Science Foundation of China (30600005, 30570019), and the Ministry of Science and Technology (973 and 863 Programs).

Received: February 14, 2008  
Revised: June 10, 2008  
Accepted: June 13, 2008  
Published: August 22, 2008

**REFERENCES**

Ahler, J., Shepard, E., Lomovskaya, N., Zazopulos, E., Staffa, A., Bachmann, B.O., Huang, K., Fonstein, L., Czisny, A., Whitwam, R.E., et al. (2002). The calicheamicin gene cluster and its iterative type I enediyne PKS. Science 297, 1173–1176.

Bierman, M., Logan, R., O'Brien, K., Seno, E.T., Rao, R.N., and Schoner, B.E. (1992). Plasmid cloning vectors for the conjugal transfer of DNA from *Escherichia coli* to *Streptomyces* spp. Gene 116, 43–49.

Brazier-Hicks, M., Offen, W.A., Gershater, M.C., Revett, T.J., Lim, E.K., Bowles, D.J., Davies, G.J., and Edwards, R. (2007). Characterization and engineering of the bifunctional N- and O-glucosyltransferase involved in xenobiotic metabolism in plants. Proc. Natl. Acad. Sci. USA 104, 20238–20243.

Cassady, J.M., Chan, K.K., Floss, H.G., and Leistner, E. (2004). Recent developments in the maytansinoid antitumor agents. Chem. Pharm. Bull. (Tokyo) 52, 1–26.

Clark, E.D.B. (1998). Refolding of recombinant proteins. Curr. Opin. Biotechnol. 9, 157–163.

Fischbach, M.A., Lin, H., Liu, D.R., and Walsh, C.T. (2005). *In vitro* characterization of IroB, a pathogen-associated C-glycosyltransferase. Proc. Natl. Acad. Sci. USA 102, 571–576.

Gao, Q., Zhang, C., Blanchard, S., and Thorson, J.S. (2006). Deciphering indolocarbazole and enediyne aminodideoxypentose biosynthesis through comparative genomics: insights from the AT2433 biosynthetic locus. Chem. Biol. 13, 733–743.

Gekko, K., and Timasheff, S.N. (1981). Mechanism of protein stabilization by glycerol: preferential hydration in glycerol-water mixtures. Biochemistry 20, 4667–4676.

Ghisalba, O., and Nuesch, J. (1981). A genetic approach to the biosynthesis of the rifamycin-chromophore in *Nocardia mediterranei*. IV. Identification of 3-amino-5-hydroxybenzoic acid as a direct precursor of the seven-carbon amino starter-unit. J. Antibiot. (Tokyo) 34, 64–71.

Hatano, K., Akiyama, S., Asai, M., and Rickards, R.W. (1982). Biosynthetic origin of aminobenzoid nucleus (C7N-unit) of ansamitocin, a group of novel maytansinoid antibiotics. J. Antibiot. (Tokyo) 35, 1415–1417.

Higashide, E., Asai, M., Ootsu, K., Tanida, S., Kozai, Y., Hasegawa, T., Kishi, T., Sugino, Y., and Yoneda, M. (1977). Ansamitocin, a group of novel maytansinoid antibiotics with antitumour properties from *Nocardia*. Nature 270, 721–722.

Hoffmeister, D., Ichinose, K., Domann, S., Faust, B., Trefzer, A., Drager, G., Kirschning, A., Fischer, C., Kunzel, E., Bearden, D., et al. (2000). The NDP-sugar co-substrate concentration and the enzyme expression level influence

the substrate specificity of glycosyltransferases: cloning and characterization of deoxysugar biosynthetic genes of the urdamycin biosynthetic gene cluster. Chem. Biol. 7, 821–831.

Hyun, C.G., Billilign, T., Liao, J., and Thorson, J.S. (2003). The biosynthesis of indolocarbazoles in a heterologous *E. coli* host. Chembiochem 4, 114–117.

Izawa, M., Tanida, S., and Asai, M. (1981). Ansamitocin analogs from a mutant strain of *Nocardia*. II. Isolation and structure. J. Antibiot. (Tokyo) 34, 496–506.

Jackson, R.G., Lim, E.K., Li, Y., Kowalczyk, M., Sandberg, G., Hoggett, J., Ashford, D.A., and Bowles, D.J. (2001). Identification and biochemical characterization of an *Arabidopsis* indole-3-acetic acid glucosyltransferase. J. Biol. Chem. 276, 4350–4356.

Kato, Y., Bai, L., Xue, Q., Revill, W.P., Yu, T.W., and Floss, H.G. (2002). Functional expression of genes involved in the biosynthesis of the novel polyketide chain extension unit, methoxymalonyl-acyl carrier protein, and engineered biosynthesis of 2-desmethyl-2-methoxy-6-deoxyerythronolide B. J. Am. Chem. Soc. 124, 5268–5269.

Kibby, J.J., McDonald, I.A., and Rickards, R.W. (1980). 3-Amino-5-hydroxybenzoic acid as a key intermediate in ansamycin and maytansinoid biosynthesis. J. Chem. Soc. Chem. Commun., 768–769.

Kieser, T., Bibb, M.J., Buttner, M.J., Chater, K.F., and Hopwood, D.A. (2000). Practical Streptomycetes Genetics (Norwich, U.K.: John Innes Foundation Press).

Kim, C.G., Yu, T.W., Fryhle, C.B., Handa, S., and Floss, H.G. (1998). 3-Amino-5-hydroxybenzoic acid synthase, the terminal enzyme in the formation of the precursor of mC7N units in rifamycin and related antibiotics. J. Biol. Chem. 273, 6030–6040.

Kim, S.Y., Park, J.S., Chae, C.S., Hyun, C.G., Choi, B.W., Shin, J., and Oh, K.B. (2007). Genetic organization of the biosynthetic gene cluster for the indolocarbazole K-252a in *Nonomuraea longicatena* JCM 11136. Appl. Microbiol. Biotechnol. 75, 1119–1126.

Kupchan, S.M., Komoda, Y., Branfman, A.R., Sneden, A.T., Court, W.A., Thomas, G.J., Hintz, H.P., Smith, R.M., Karim, A., Howie, G.A., et al. (1977). The maytansinoids. Isolation, structural elucidation, and chemical interrelation of novel ansa macrolides. J. Org. Chem. 42, 2349–2357.

Lairson, L.L., Henrissat, B., Davies, G.J., and Withers, S.G. (2008). Glycosyltransferases: structures, functions, and mechanisms. Annu. Rev. Biochem. 77, 521–555.

Lu, C., and Shen, Y. (2007). A novel ansamycin, naphthomycin K from *Streptomyces* sp. J. Antibiot. (Tokyo) 60, 649–653.

Lu, C., Bai, L., and Shen, Y. (2004). A novel amide N-glycoside of ansamitocins from *Actinosynnema pretiosum*. J. Antibiot. (Tokyo) 57, 348–350.

Ma, J., Zhao, P.J., and Shen, Y.M. (2007). New amide N-glycosides of ansamitocins identified from *Actinosynnema pretiosum*. Arch. Pharm. Res. 30, 670–673.

Mackenzie, P.I. (1990). Expression of chimeric cDNAs in cell culture defines a region of UDP-glucuronosyltransferase involved in substrate selection. J. Biol. Chem. 265, 3432–3435.

MacNeil, D.J., Gewain, K.M., Ruby, C.L., Dezeny, G., Gibbons, P.H., and MacNeil, T. (1992). Analysis of *Streptomyces avermitilis* genes required for avermectin biosynthesis utilizing a novel integration vector. Gene 111, 61–68.

Marangoni, A.G. (2003). Enzyme Kinetics: A Modern Approach (Hoboken, NJ: John Wiley).

Minagawa, K., Zhang, Y., Ito, T., Bai, L., Deng, Z., and Mahmud, T. (2007). ValC, a new type of C7-Cyclitol kinase involved in the biosynthesis of the antifungal agent validamycin A. Chembiochem 8, 632–641.

Morisaki, N., Kobayashi, H., Iwasaki, S., Furihata, K., Dabbs, E.R., Yazawa, K., and Mikami, Y. (1995). Structure determination of ribosylated rifampicin and its derivative: new inactivated metabolites of rifampicin by mycobacterial strains. J. Antibiot. (Tokyo) 48, 1299–1303.

Mosmann, T. (1983). Rapid colorimetric assay for cellular growth and survival: application to proliferation and cytotoxicity assays. J. Immunol. Methods 65, 55–63.

Moss, S.J., Bai, L., Toelzer, S., Carroll, B.J., Mahmud, T., Yu, T.W., and Floss, H.G. (2002). Identification of *asm19* as an acyltransferase attaching the

biologically essential ester side chain of ansamitocins using N-desmethyl-4, 5-desepoxymaytansinol, not maytansinol, as its substrate. J. Am. Chem. Soc. 124, 6544–6545.

Mulichak, A.M., Losey, H.C., Walsh, C.T., and Garavito, R.M. (2001). Structure of the UDP-glucosyltransferase GtfB that modifies the heptapeptide aglycone in the biosynthesis of vancomycin group antibiotics. Structure 9, 547–557.

Nishizawa, T., Aldrich, C.C., and Sherman, D.H. (2005). Molecular analysis of the rebeccamycin L-amino acid oxidase from *Lechevalieria aerocolonigenes* ATCC 39243. J. Bacteriol. 187, 2084–2092.

Ohuchi, T., Ikeda-Araki, A., Watanabe-Sakamoto, A., Kojiri, K., Nagashima, M., Okanishi, M., and Suda, H. (2000). Cloning and expression of a gene encoding N-glycosyltransferase (ngt) from *Saccarothrix aerocolonigenes* ATCC39243. J. Antibiot. (Tokyo) 53, 393–403.

Onaka, H., Taniguchi, S., Igarashi, Y., and Furumai, T. (2002). Cloning of the staurosporine biosynthetic gene cluster from *Streptomyces sp.* TP-A0274 and its heterologous expression in *Streptomyces lividans*. J. Antibiot. (Tokyo) 55, 1063–1071.

Onaka, H., Taniguchi, S., Igarashi, Y., and Furumai, T. (2003a). Characterization of the biosynthetic gene cluster of rebeccamycin from *Lechevalieria aerocolonigenes* ATCC 39243. Biosci. Biotechnol. Biochem. 67, 127–138.

Onaka, H., Taniguchi, S., Ikeda, H., Igarashi, Y., and Furumai, T. (2003b). pTOYAMAcos, pTYM18, and pTYM19, actinomycete-*Escherichia coli* integrating vectors for heterologous gene expression. J. Antibiot. (Tokyo) 56, 950–956.

Ootsu, K., Kozai, Y., Takeuchi, M., Ikeyama, S., Igarashi, K., Tsukamoto, K., Sugino, Y., Tashira, T., Tsukagoshi, S., and Sakurai, Y. (1980). Effects of new antimitotic antibiotics, ansamitocins, on the growth of murine tumors in vivo and on the assembly of microtubules in vitro. Cancer Res. 40, 1707–1717.

Price, N.C., and Stevens, L. (1982). Fundamental of Enzymology (Oxford: Oxford University Press).

Rinehart, K.L., Jr., and Shield, L.S. (1976). Chemistry of the ansamycin antibiotics. Fortschr. Chem. Naturst. 33, 231–307.

Salas, A.P., Zhu, L., Sanchez, C., Brana, A.F., Rohr, J., Mendez, C., and Salas, J.A. (2005). Deciphering the late steps in the biosynthesis of the anti-tumour indolocarbazole staurosporine: sugar donor substrate flexibility of the StaG glycosyltransferase. Mol. Microbiol. 58, 17–27.

Sanchez, C., Butovich, I.A., Brana, A.F., Rohr, J., Mendez, C., and Salas, J.A. (2002). The biosynthetic gene cluster for the antitumor rebeccamycin: characterization and generation of indolocarbazole derivatives. Chem. Biol. 9, 519–531.

Sanchez, C., Zhu, L., Brana, A.F., Salas, A.P., Rohr, J., Mendez, C., and Salas, J.A. (2005). Combinatorial biosynthesis of antitumor indolocarbazole compounds. Proc. Natl. Acad. Sci. USA 102, 461–466.

Sanchez, C., Mendez, C., and Salas, J.A. (2006). Indolocarbazole natural products: occurrence, biosynthesis, and biological activity. Nat. Prod. Rep. 23, 1007–1045.

Sensi, P., Margalith, P., and Timbal, M.T. (1959). Rifomycin, a new antibiotic: preliminary report. Farmaco 14, 146–147.

Sneden, A.T., and Beemsterboer, G.L. (1980). Normaytansine, a new antileukemic ansa macrolide from *Maytenus buchananii*. J. Nat. Prod. 43, 637–640.

Spiteller, P., Bai, L., Shang, G., Carroll, B.J., Yu, T.W., and Floss, H.G. (2003). The post-polyketide synthase modification steps in the biosynthesis of the antitumor agent ansamitocin by *Actinosynnema pretiosum*. J. Am. Chem. Soc. 125, 14236–14237.

Stead, P., Latif, S., Blackaby, A.P., Sidebottom, P.J., Deakin, A., Taylor, N.L., Life, P., Spaul, J., Burrell, F., Jones, R., et al. (2000). Discovery of novel ansamycins possessing potent inhibitory activity in a cell-based oncostatin M signalling assay. J. Antibiot. (Tokyo) 53, 657–663.

Walsh, C., Freel Meyers, C.L., and Losey, H.C. (2003). Antibiotic glycosyltransferases: antibiotic maturation and prospects for reprogramming. J. Med. Chem. 46, 3425–3436.

Widdison, W.C., Wilhelm, S.D., Cavanagh, E.E., Whiteman, K.R., Leece, B.A., Kovtun, Y., Goldmacher, V.S., Xie, H., Steeves, R.M., Lutz, R.J., et al. (2006). Semisynthetic maytansine analogues for the targeted treatment of cancer. J. Med. Chem. 49, 4392–4408.

Wilkinson, C.J., Hughes-Thomas, Z.A., Martin, C.J., Bohm, I., Mironenko, T., Deacon, M., Wheatcroft, M., Wirtz, G., Staunton, J., and Leadlay, P.F. (2002). Increasing the efficiency of heterologous promoters in actinomycetes. J. Mol. Microbiol. Biotechnol. 4, 417–426.

Yang, X.A., Dong, X.Y., Li, Y., Wang, Y.D., and Chen, W.F. (2004). Purification and refolding of a novel cancer/testis antigen BJ-HCC-2 expressed in the inclusion bodies of *Escherichia coli*. Protein Expr. Purif. 33, 332–338.

Yu, T.W., Bai, L., Clade, D., Hoffmann, D., Toelzer, S., Trinh, K.Q., Xu, J., Moss, S.J., Leistner, E., and Floss, H.G. (2002). The biosynthetic gene cluster of the maytansinoid antitumor agent ansamitocin from *Actinosynnema pretiosum*. Proc. Natl. Acad. Sci. USA 99, 7968–7973.

Zhang, C., Albermann, C., Fu, X., Peters, N.R., Chisholm, J.D., Zhang, G., Gilbert, E.J., Wang, P.G., Van Vranken, D.L., and Thorson, J.S. (2006). RebG- and RebM-catalyzed indolocarbazole diversification. Chembiochem 7, 795–804.
