
doi:10.1016/j.jmb.2008.08.010

JMB Available online at www.sciencedirect.com ScienceDirect ELSEVIER

**C₄-Dicarboxylates Sensing Mechanism Revealed by the Crystal Structures of DctB Sensor Domain**

Yan-Feng Zhou$^{1\dagger}$, Beiyuan Nan$^{1\dagger}$, Jie Nan$^{1}$, Qingjun Ma$^{2}$, Santosh Panjikar$^{2}$, Yu-He Liang$^{1}$, Yiping Wang$^{1*}$ and Xiao-Dong Su$^{1*}$

$^{1}$National Laboratory of Protein Engineering and Plant Genetic Engineering, College of Life Sciences, Peking University, Beijing 100871, China

$^{2}$EMBL Hamburg Outstation, c/o Deutsches Elektronen Synchrotron, Notkestrasse 85, D-22603 Hamburg, Germany

Received 23 March 2008; received in revised form 2 August 2008; accepted 6 August 2008 Available online 12 August 2008

Edited by R. Huber

C₄-dicarboxylates are the major carbon and energy sources during the symbiotic growth of rhizobia. Responses to C₄-dicarboxylates depend on typical two-component systems (TCS) consisting of a transmembrane sensor histidine kinase and a cytoplasmic response regulator. The DctB–DctD system is the first identified TCS for C₄-dicarboxylates sensing. Direct ligand binding to the sensor domain of DctB is believed to be the first step of the sensing events. In this report, the water-soluble periplasmic sensor domain of Sinorhizobium meliloti DctB (DctBp) was studied, and three crystal structures were solved: the apo protein, a complex with C₄ succinate, and a complex with C₃ malonate. Different from the two structurally known CitA family of carboxylate sensor proteins CitA and DcuS, the structure of DctBp consists of two tandem Per–Arnt–Sim (PAS) domains and one N-terminal helical region. Only the membrane-distal PAS domain was found to bind the ligands, whereas the proximal PAS domain was empty. Comparison of DctB, CitA, and DcuS suggests a detailed stereochemistry of C₄-dicarboxylates ligand perception. The structures of the different ligand binding states of DctBp also revealed a series of conformational changes initiated upon ligand binding and propagated to the N-terminal domain responsible for dimerization, providing insights into understanding the detailed mechanism of the signal transduction of TCS histidine kinases.

© 2008 Elsevier Ltd. All rights reserved.

Keywords: C₄-dicarboxylates; signal perception; histidine kinase; DctB; crystal structure

---

**Introduction**

Cell survival and development depend on quick responses to environmental changes. A common mode of prokaryotic stimuli responses is through two-component systems (TCS) comprising a sensory histidine kinase and a response regulator, which interact in tandem.¹ Periplasmic sensing kinases, which feature an N-terminal periplasmic ligand-binding domain and two transmembrane helices, comprise the largest group of TCS sensors.² An external stimulus is typically detected in the periplasmic space by a sensory domain of the kinase, accompanied by conformational changes upon sensing. The signal will then be transmitted across the membrane to the cytoplasmic kinase domain, followed by conserved His-Asp phosphotransfer between the sensor and the regulator.¹

C₄-dicarboxylates [succinate (SIN), L-malate, fumarate, and so on] are utilized as specialized energy sources during the symbiotic growth of rhizobia. The C₄-dicarboxylate transport (Dct) system is the first identified regulatory system to monitor the external concentration of C₄-dicarboxylic molecules.³⁻⁵ The Dct system contains three proteins coded by the *dct* regulon: the *dctA* gene encodes a putative C₄-dicarboxylates carrier, while the sensor-regulator pair DctB–DctD is expressed from the *dctBD* operon.⁶

*Corresponding authors. E-mail addresses:* wangyp@pku.edu.cn; xdsu@pku.edu.cn.

† Y.-F.Z. and B.N. contributed equally to this work.

Abbreviations used: TCS, two-component system(s); DctBp, periplasmic sensor domain of Sinorhizobium meliloti DctB; PAS, Per–Arnt–Sim; SIN, succinate; Dct, dicarboxylate transport; MLA, malonate; DctBpp, membrane-proximal domain of DctBp; DctBpd, membrane-distal domain of DctBp; SeMet, selenomethionine; NCS, noncrystallographic symmetry; PDB, Protein Data Bank.

The DctB–DctD TCS can be activated by the presence of C₄-dicarboxylates in the periplasm, resulting in phosphorylated DctD protein. The regulator DctD consists of an N-terminal phosphoryl receiver domain, a C-terminal DNA-binding domain, and a central ATPase domain. [7–9] Phosphorylation of DctD regulates the activation of the σ⁵⁴-dependent RNA polymerase holoenzyme (Eσ⁵⁴), which controls the transcription of *dctA*. [10,11]

DctB is a typical transmembrane sensory histidine kinase containing two transmembrane helices (W27-A45 and L321-L338); the N-terminal extracellular region between the two transmembrane helices defines a 280-residue periplasmic domain, which is supposed to interact with and to sense the C₄-dicarboxylate ligands. The cytoplasmic part of the DctB following the C-terminal transmembrane helix contains the phosphoryl transfer region, including the kinase domain and the conserved histidine residue. DctB shows different selectivities for various dicarboxylates. C₄-dicarboxylates such as SIN are well-known inducers of DctB. [6] In contrast, the C₃-dicarboxylate malonate (MLA) does not induce the Dct system *in vivo*. Symmetric ligand binding and dimerization of the sensory histidine kinases are proposed as the key processes in TCS signaling. [12] Genetic studies revealed that trans autophosphorylation of DctB can be induced by the presence of C₄-dicarboxylates. [13] However, many aspects of the molecular details of the DctB–ligand interaction remain obscure. Until now, little is known on how DctB interacts specifically with C₄-dicarboxylates and on the details of conformational changes after ligand binding.

Three sensors for C₄-dicarboxylates—Tar, DcuS, and CitA—are now structurally known. The structure of the aspartate sensor Tar from *Salmonella typhimurium* is a four-helical bundle, [14,15] which is distinguishable from the other two sensors from the CitA family. The structure of the ligand-binding domain of CitA from *Klebsiella pneumoniae* (CitAp), complexed with citrate, has been solved by X-ray crystallographic method. [16] The structure of the ligand-binding domain of DcuS from *Escherichia coli* (DcuSp) has been determined in solution by NMR spectroscopy. [17] The periplasmic domain of CitA or DcuS is about 140 residues long and has been shown to adopt a Per–Arnt–Sim (PAS) fold of the photoactive yellow protein from *Halorhodospira halophila*. [18] The complex structure of CitA–citrate shows that CitA binds to citrate molecule directly; however, without a comparison with an apo structure or other CitA–ligand complex structures, mechanistic details are hard to conclude. Further functional studies of DcuS reveal conserved ligand-binding sites of DcuS and CitA, [19,20] but a similar problem remains for the DcuS studies, with only one set of structural coordinates available. For DctB, it has been shown that the sensory kinase could sense C₄-dicarboxylates directly as well. [21] However, the periplasmic region of DctB is much larger than that of DcuS or CitA—almost twice the length. Moreover, it is difficult to predict with high accuracy whether

DctB is a one-PAS-domain-containing or a two-PAS-domain-containing protein based on amino acid sequence alignments; thus, a homology model of DctB is impossible to build for the prediction and comparison of C₄-dicarboxylates/DctB binding based on available structural information.

To understand the underlying mechanism of DctB signal perception and transduction, we have solved three structures of the periplasmic sensor domain of *Sinorhizobium meliloti* DctB (DctBp; residues 42–311), including the apo structure and the structures of two complexes: one with C₄ SIN (DctBp/SIN) and the other with C₃ MLA (DctBp/MLA). The dimeric structures with different ligand bindings provided us with stereochemical details on the ligand perception and signal transduction of DctB.

## Results

### Overall description of the structures

Three kinds of DctBp crystals with different ligand binding states were obtained in this work (Fig. 1). Crystals of the DctBp/SIN complex belong to the C2 space group. Two chemically identical monomers, referred to as monomers A and B, were observed in the asymmetric unit of DctBp/SIN. Crystal packing analyses reveal two similar back-to-back dimers, designated as A–A′ and B–B′, where the primes indicate 2-fold crystallographic-axis-related monomers. Based on other structurally known histidine kinases, [16,22–24] the back-to-back dimers are most likely to be functionally relevant, with the 2-fold dimer axis perpendicular to the putative membrane (Figs. 1 and 2c). Crystals of apoDctBp and DctBp/MLA complex are isomorphous, both belonging to the P2₁ space group with two molecules in one asymmetric unit. No dicarboxylate existed under the crystallization condition of apoDctBp, while 200 mM MLA was present in the crystallization buffer of DctBp/MLA. Similar back-to-back dimers were observed in the asymmetric unit of apoDctB and DctBp/MLA complex (Fig. 1). Superposition of the Cα atoms of the two dimers gives a root mean standard deviation (RMSD) value of 0.4 Å, indicating that they are very similar. The structures of apoDctB and DctBp/MLA complex further suggested that the DctB periplasmic region can form dimers in the back-to-back arrangement, standing on the inner membrane with both N-terminal and C-terminal regions connecting to the transmembrane helices (Fig. 2b).

Structural elements of DctBp are shown in Fig. 2. In general, DctBp contains two similar domains with mixed α/β folds, each comprising an anti-parallel β sheet and connecting helices. The structure begins with three N-terminal helices α1–α3 (residues 54–118) located on the backside of the two central β sheets. The two domains are linked by helix α5, which is also involved in apoDctB dimerization. In each of the two domains, a long flexible loop and a

Crystal Structures of DctB Sensor Domain

| spacegroup | C2          | P2₁         | P2₁        |
|------------|-------------|-------------|------------|
| bound      | succinate   | malonate    | none       |
| molecule   |             |             |            |
| dimer      | ![Image 1](image1.png) | ![Image 2](image2.png) | ![Image 3](image3.png) |
| crystal    | ![Image 4](image4.png) | ![Image 5](image5.png) | ![Image 6](image6.png) |

Fig. 1. Crystals and dimeric structures of DctBp. Bound molecules are selectively shown as space-filling models in the monomers presented as ribbons. The dimer of DctBp/SIN was generated by symmetric operation of a monomer in the asymmetric unit. The dimer of apoDctBp or DctBp/MLA was observed in one asymmetric unit.

helix link the second and the third β strands (β2 and β3 for the first β sheet; β7 and β8 for the second β sheet).

The two tandem folds that DctBp adopts have shown similarity with previously reported PAS folds in other histidine kinases such as CitA and DcuS. PAS domains are widely distributed in many organisms, and over thousands of proteins are annotated to contain PAS domains by sequence analyses. However, due to the low sequence identities among the PAS-fold-containing proteins, it is often hard to predict accurately whether a protein has a PAS domain, unless structural information is available. Herein, the two PAS domains of DctBp are DctBpp (membrane-proximal domain of DctBp; residues 119–199) and DctBpd (membrane-distal domain of DctBp; residues 211–308), relative to the membrane. As shown in Table 1, despite the low sequence identities between CitAp, DcuSp, DctBpd, and DctBpp (<20%), superposition of the Cα atoms of those molecules yields RMSD values ranging from 2 to 2.8 Å. If the flexible loops extending over the β sheets are removed, superposition of the central β sheets yields much lower RMSD values, ranging from 1.4 to 2.4 Å over an average of 50 Cα atoms.

Ligand-binding pocket in the membrane-distal PAS domain

The SIN molecule was observed to only bind to the membrane-distal PAS domain of DctBp (Fig. 2). The ligand-binding pocket is mainly formed by the central β sheet and the loops between β2 and α4 and between β3 and β4 (Fig. 2). Residues holding SIN ligand are highly conserved among DctB proteins from different species (Fig. 2a). Upon SIN binding, the side chains of residues R152, Y154, T171, S173, Y179, and K197 form tight hydrogen-bonding networks and salt bridges with the two carboxyl groups of the ligand (Figs. 3 and 4) compared with the apoDctBp. Since SIN is a symmetric molecule, we define the carboxylic group near the R152 side chain as C1 position, and the other carboxylic group at the K197 side chain as C4 position (Fig. 3). For other carboxylate molecules such as citrate and MLA, the carboxylic group heading to the R152 side is referred to as C1 as well (Fig. 3c). Similar to other carboxylate-binding proteins, it is clear that negatively charged residues play essential roles in SIN binding of DctB. Under crystallization conditions (pH 6.0), tyrosine residues (pKa ~10) prefer to form hydrogen bonds with C1 (Y154) and C4 (Y179) carboxylic groups. Meanwhile, hydrophobic interactions also exist between the backbone carbon atoms of the SIN and the aromatic rings of F144 and Y149, with about a 3.7-Å distance from the rings to the C3 carbon atom of SIN (Fig. 3).

Conformation of the ligand-binding pocket changes remarkably upon SIN binding (Figs. 3 and 4). During SIN binding and sensing, the loops β2–α5, β3–β4 (residues G170-S173), and R152 are pulled towards the SIN molecule as if the whole pocket is “tightened” by the C4 ligand (Fig. 4d). The movement of loop β2–α5 is mainly caused by

(a)

| 17380400 SINME | LAGLQGQSRLIDASLKASLLRAVVBRQRALFILVLAADDAAITR |
| --- | --- |
| 118395 RHILE | EAALBEBQARMDVTLNALLRTVLEKYRALPFBVLSCQDTALAARLVGN.DAGTFB |
| 86358574 RHIET | EAALBEBQARTDARMNVALLRTVLEKYRALPFBVLSCQDTALAARLVGN.DAGTFB |
| 24982867 PSEPU | MEDDAKRASQQLGLYANALHTLIDRYRALPEAVLLADPELIAAALRGPVDKVQN |
| 15891636 AGRTU | IRTAESEASTDAELKIALNVALEKPRALPDLVLSGDPDLAALDGGSPPALD |
| 83592772 RHORU | LSHARDAAARSAAVLQSEIRKQSLPIJVLASDPDLGRALESR.DPAHTIA |
| 89359004 XANAU | LAQVESSAATAALARVALRSEIEKQRSLPLALAQDPDVRRALEIG.NAARID |
| 20804187 MESLO | EYALRDRAFAAFPPLAADSLKEIEKQPMIVLVARDDGAVERLSP.STANEA |
| 70732127 PSEFL | LVBEAARANERLGGLYANSHTLIBERYRALPAVLLADPELRALLQGFVATETQD |
| 67158578 AZOVI | WSELDIEARGQLELYAOSLRILVKERRFRALPADTEIQALLOESSDPSARE |

---

**N-term helical dimerization domain**

| 17380400 SINME | QPTSFVGNDTYFRDYTRLAVRDGMAEBFAMGTVSNRPGLYISRRVDGPGG.P |
| --- | --- |
| 118395 RHILE | REPTSFVGNDYRFREYEFGAVERGQAEHFALGTVSKKPKGLYISQRISSGSNG.LLGVVVVKEEF |
| 86358574 RHIET | REPTSFVGNDYRFREYEFGAVERGQAEHFALGTVSKKPKGLYISERISSGSDG.LLGVVVVKEEF |
| 24982867 PSEPU | RLPSSYVGSNYERPEBKQTRSQGSGRFYAVGVTSGVPGYFLASAVNDEHGRF |
| 15891636 AGRTU | RQDDSFVGSNYAFOREYGAMKTGSMSEHYALGNVSRRPGLYISRRIDASDGRPL |
| 83592772 RHORU | RAADSFVGSDYFFPERYLALD3RATEHFALGTVSHRPKGYYIARRIDGARG.P |
| 89359004 XANAU | RTPESEFLNSDYGFRPYFRLALAQGRAEHFAEGTVSRRPGLYLAQRVEDAGV.P |
| 20804187 MESLO | GEPTSFVGSDYRFRHYETEAMADGAAMQYALGTVSARPGLYLSRVRDGPPEG.P |
| 70732127 PSEFL | RLPSSYVGHNYGFRPFYFKQTRSQQSGRFYAVGVTSGITPGYFLASAVKADNQDF |
| 67158573 AZOVI | REDGNFVGGNYAFRPYBQGALR3GSARYFGVGATTGVPGYFLSRAVRDAGGRTL |

---

**Membrane-distal PAS domain**

| 17380400 SINME | LITSLPSTMTTKPIAEDRLAPIRESLQFGDAAPLLPPLPFRIKEARPDGSSSTLDALLPGDS.S.TAAFFLRVETMJPSTNWRELQGSP |
| --- | --- |
| 118395 RHILE | LITSLPSTMTIGRIAEDRLTAIRESLQFGAAPLQPPLPDMVRNLGB3LDVVEIVMPGDAKGKTRFLDVATSVPATGWHLQHLVAL |
| 86358574 RHIET | LITSIPSTMTIGRIAEDRLTAIRESLQFGDAFLLQPPLPDIIRHLGDGLAVVEIVTFGSGAKTSEFLNVGMPIPATGWHLQHLVAL |
| 24982867 PSEPU | FIANQDGWRYRELPQPLSGADRADLAETROQDKQFLVPLHHQVLTRFAPYSTLSRVQGPEGS...TEYLWESLPEGENWTLLHLRKPF |
| 15891636 AGRTU | DEMRFKTVVAPIDEARRKIIESLQFGNETLATLPFRVARSSSDGVLIHVDGPARA.RGDAYLQLEVPTTSWTHYLQPV |
| 83592772 RHORU | LITSLPSTFLAKIPKSAAESARIRESLFEKAPLTPLPIRRADVGTVVVALPTSRADENERPFLVEAGPQPATAWTLHVLAAPA |
| 89359004 XANAU | LISSVPQFRTTRPLDKVEREAIRESLQFGDSPLELLDLLPTTAADPAVKRVRLECGQAKERED...FLESSLFVFTTDWTLSVLAPT |
| 20804187 MESLO | LATSVPOWRFGALSPLSAKEKQTARDRLQLPGVTFRPVPLSRR...GDNLVTATPERSRSAFVAVSQDLGKAJP..GWRMSILLIPA |
| 70732127 PSEFL | FIANQPGWRYRHLQPLSSNDLEBTIKLTRQYDKQPLTPLQHOTLQRFDDNSSLARVEGPEGR...ASYLWESLPLAAEGWTLLHLRKPF |
| 67158573 AZOVI | ILASRPAWFRLLQEPAAAEEFFRPVDARKYGIREEVRPLALQPLRPLDGGGEELLRIDGPEGR...RDYLRLSQPLADEGWTELLEA |

---

**Membrane-proximal PAS domain**

(b)  

(c)  

Fig. 2. Overall structure of the DctBp. (a) Structure-based multiple sequence alignment of the DctBp family by CLUSTAL W [25] and ESPript. [26] Helices and strands are shown by helical curves and arrows, respectively. The highly conserved residues are shown against a red background, and the less conserved ones are shown in red letters. Dimerization helices, membrane-distal PAS domains, and membrane-proximal PAS domains are indicated by colored lines (green, magenta, and orange, respectively), and crucial residues for ligand binding are labeled with asterisks. Residues involved only in the dimer interfaces of DctBp/MLA are shown by magenta dots (●), and residues only in the dimer of DctBp/SIN are represented by red squares (■). Residues appearing in both of the dimers are indicated by green diamonds (◆). Conserved hydrophobic residues in the shared dimer interface are shown with orange bars on top of the aligned sequences (residues 81–86 and 114). The sequences aligned are the following (gi numbers are also shown before organisms in the figure): SINME, Sinorhizobium meliloti; RHILE, Rhizobium leguminosarum; RHIET, Rhizobium etli; AGRTU, Agrobacterium tumefaciens; RHORU, Rhodospirillum rubrum; XANAU, Xanthobacter autotrophicus; MSELO, Mesorhizobium loti; PSEFL, Pseudomonas fluorescens; AZOVI, Azotobacter vinelandii. (b) Ribbon diagrams showing the overall structure of one monomer, with one SIN bound to the distal PAS domain. The ligand is shown by space-filling models, with green carbon and red oxygen atoms. The dimerization helices, membrane-distal PAS domains, and membrane-proximal PAS domains are in green, magenta, and orange, respectively. (c) Dimer of DctBp/SIN. Only one SIN molecule is shown as a space-filling model in the monomer presented as ribbon. The 2-fold crystallographic axis and putative membrane are shown.

hydrophobic interactions; for instance, the phenyl ring of F144 moves 1.5 Å to the C3 carbon atom of SIN. Similar with CitA ligand binding, main-chain nitrogen atoms on loop β3–β4 stabilize the ligand via hydrogen bonding with an oxygen atom of the

C4-position carboxylic group. The main-chain Cα atom (T172) shows 2.2 Å movement compared with the apoDctBp, forming a compact ligand-binding pocket (Fig. 4d). In addition, the guanidyl group of R152 side chain reorients as large as 5 Å closer to the

Crystal Structures of DctB Sensor Domain

Table 1. Summary of crystallographic analyses

| Data collection | PDB code 3E4O (SeMet, DctB/SIN) | PDB code 3E4P (native, DctB/MLA) | PDB code 3E4Q (refolded, apoDctBp) |
|-----------------|----------------------------------|----------------------------------|------------------------------------|
| Space group     | $C2$                             | $P2_1$                           | $P2_1$                            |
| Cell dimensions | $a, b, c (\AA)$                  | $101.90, 66.60, 93.60$           | $57.98, 39.10, 111.54$            |
|                 | $\alpha, \beta, \gamma (^{\circ})$ | $90, 94.71, 90$                  | $90, 94.77, 90$                   |
| Peak            |                                  |                                  |                                   |
| Inflection      |                                  |                                  |                                   |
| Remote          |                                  |                                  |                                   |
| Wavelength      | $0.9796$                         | $0.9799$                         | $0.9760$                          | $0.9790$                         | $1.0$                              |
| Resolution ($\AA$) | $2.3 (2.38–2.30)^{a}$         | $2.3 (2.38–2.30)$                | $2.3 (2.38–2.30)$                 | $2.3 (2.40–2.30)$                | $2.75 (2.85–2.75)$                 |
| $R_{\text{sym}}$ or $R_{\text{merge}}$ | $9.3 (28.9)$                      | $9.2 (27.9)$                      | $9.1 (25.0)$                     | $8.4 (47.6)$                      | $6.8 (36.9)$                      |
| $I/\sigma I$    | $18.2 (5.5)$                     | $18.6 (5.6)$                     | $18.7 (7.1)$                      | $12.1 (2.7)$                     | $15.0 (2.3)$                      |
| Completeness (%) | $99.8 (98.5)$                    | $99.8 (99.4)$                    | $100 (99.8)$                      | $97.5 (94.7)$                    | $95.9 (97.7)$                     |
| Redundancy      | $7.5 (5.9)$                      | $7.5 (5.8)$                      | $7.6 (6.6)$                       | $2.9 (2.8)$                      | $2.7 (2.6)$                       |

| Refinement       | PDB code 3E4O (SeMet, DctB/SIN) | PDB code 3E4P (native, DctB/MLA) | PDB code 3E4Q (refolded, apoDctBp) |
|------------------|----------------------------------|----------------------------------|------------------------------------|
| Resolution ($\AA$) | $2.3$                           | $2.3$                           | $2.75$                            |
| Number of reflections | $209,855$                      | $64,895$                        | $33,562$                          |
| $R$-factor/$R_{\text{free}}$ | $0.21/0.25$               | $0.20/0.26$                     | $0.20/0.27$                       |
| Number of atoms   |                                  |                                  |                                   |
| Protein          | $3890$                          | $3854$                          | $3890$                            |
| Ligand           | $16$                            | $14$                            | —                                 |
| Water            | $199$                           | $114$                           | $10$                              |
| $B$-factors      |                                  |                                  |                                   |
| Protein          | $36.1$                          | $32.8$                          | $52.1$                            |
| Ligand           | $25.0$                          | $25.7$                          | —                                 |
| Water            | $37.4$                          | $31.0$                          | $34.5$                            |
| RMSD             |                                  |                                  |                                   |
| Bond lengths ($\AA$) | $0.014$                       | $0.015$                        | $0.021$                           |
| Bond angles ($^{\circ}$) | $1.61$                      | $1.73$                         | $2.17$                            |

$a$ The highest-resolution shell is shown in parentheses.

---

ligand-binding site, which separates the bound ligand from the solvent environment at the C1 carboxylate side. In contrast, residues located at the central $\beta$ sheet, such as K197 and Y179, do not show obvious conformational change. The function of these residues is likely to form a rigid and specific scaffold pocket for ligand binding, whereas for loop $\beta 3-\beta 4$ and residue R152, they are more flexible and likely to function as adaptable arms to accommodate the specific ligands.

The ligand-binding pocket shows strong specificity to $C_4$-dicarboxylates. When SIN is bound, the binding pocket is tightly closed to exclude water access (Fig. 4e and f). The bound SIN is held so tight and deep that it is not visible from the surface. However, when a smaller $C_3$ dicarboxylate such as MLA gets into the binding pocket, as shown in the DctBp/MLA structure, the binding pocket becomes quite relaxed, adopting a conformation very similar to that of the apo structure (Fig. 4d). Comparison of the structures of apoDctBp and DctBp/MLA reveals no obvious conformational change before and after MLA binding. In addition, the C1-position carboxylic group of MLA is only loosely stabilized by one positively charged residue R152 with a shift of about 5 Å compared with the SIN-bound structure (Fig. 4d), while two water molecules have been built into the extra electron densities at the C4 (counting as in the case of SIN) end of the ligand-binding pocket in the DctBp/MLA structure. The water molecules mediate hydrogen-bonding networks between the MLA ligand and the C4 end of the

binding pocket. One water molecule ($B$-factor = 29.5), Wat1 in Fig. 4b, links the N${}^\delta$ of K197, the O${}^\gamma$ of S173, and the C3 carboxyl oxygen of the MLA, forming a triangular pyramid; the other water molecule Wat2 ($B$-factor = 26.7) inserts into the space between the extending loop $\beta 3-\beta 4$ and the ligand, loosely bridging the C3-position carboxyl group of MLA and the main-chain nitrogen atoms of V172 and S173. We can thus conclude that the $C_3$-dicarboxylates such as molanate will not bind into the pocket as tight as the $C_4$-dicarboxylates, and particularly will not induce big-enough conformation changes to trigger subsequent signal transduction events.

**Differences between apo-bound and SIN-bound dimers**

The dimer arrangement of DctBp changes considerably when SIN binds to the sensory domain. In the structure of the complex DctBp/SIN, the N-terminal helices $\alpha 2$ and $\alpha 3$ and the top half of $\alpha 1$ comprise most of the dimerization interface. When DctBp is ligand-free, as illustrated by the structure of apoDctBp, helix $\alpha 5$ and the N-terminal half of helix $\alpha 1$ also participate in the dimerization (Figs. 1 and 2a). As a result, the dimerization surface area per monomer of the apoDctBp dimer is 1130 Å${}^2$, whereas the corresponding area in the DctBp/SIN dimer is reduced by about half to 540 Å${}^2$, which is somewhat too small for the formation of a tight dimer in solution. ${}^{30,31}$ The different

Fig. 3. (a) Stereo view of the SIN-binding pocket of DctBp. (b) LIGPLOT colored by atom type: carbon, black; oxygen, red; nitrogen, blue. Hydrogen bonds and salt bridges are depicted as dashed lines, with the distances labeled. Backbone carbon atoms of SIN are numbered C1–C4, with the C1-position carboxylic group interacting with R152. (c) Structure of some $C_4$-dicarboxylic acids, citrate, and MLA. From top to bottom: first row—SIN, fumarate, and maleate; second row—aspartate and orotate; bottom—citrate and MLA. The positions of the backbone carbon atoms of citrate and MLA are numbered similarly to that of SIN, with the same C1 orientations.

dimerization surfaces observed in this study indicate that divergent dimer stabilities under a physiological environment may also occur. When functioning *in vivo*, the sensor DctB is subjected to change its dimer arrangements according to the ligand binding states and then to transfer the chemical signals to the downstream components across the membrane.

## Discussion

### Comparison of the sensor domains of DctB, CitA, and DcuS

In the three known structures of $C_4$-dicarboxylates sensors, Tar is clearly different from CitA and DcuS.

Crystal Structures of DctB Sensor Domain

Fig. 4. The DctB ligand-binding pocket. (a) Ligand-binding pocket in the structure of DctBp/SIN, with $2F_{o}-F_{c}$ density contoured at $1.5\sigma$. Hydrogen bonds and salt links in the binding pocket are indicated by dashed lines, with distances labeled ($\AA$). The oxygen, nitrogen, and carbon atoms are in red, blue, and green, respectively. Main chains of residues 171–173 are shown as sticks. (b) Ligand-binding pocket in the structure of DctBp/MLA, with $2F_{o}-F_{c}$ density contoured at $1.5\sigma$. Two water molecules bridging the MLA-protein interaction are shown as Wat1 and Wat2. Carbon atoms in the MLA-presented structure are in salmon. (c) Empty ligand-binding pocket revealed by the apoDctBp structure. A few water molecules, randomly located in the two ligand-binding pockets of the dimer, are not shown. (d) Superposition of the ligand-binding pockets in the structures of apoDctBp, DctBp/MLA, and DctBp/SIN. On the right, F144 and the $\beta 2-\alpha 5$ loop are shown. Movements caused by SIN binding are indicated by arrows, with moved distances labeled. Atoms are colored as in (a–c). (e and f) Charged surfaces of the ligand-binding pockets when SIN or MLA is bound, with bound molecules removed when preparing those surfaces. The exposed MLA molecule is shown by stick and spheres, while the SIN is deeply and tightly buried inside the pocket and is not visible. All ligands were removed when the charged surfaces were prepared.

by forming a four-helical-bundle structure in the periplasmic domain. ${}^{15}$ The structures of DcuSp and CitAp have shown that the periplasmic sensor domains resemble the PAS domain, similar to the fold of photoactive yellow protein. The structures determined in this work show that DctBp contains two tandem PAS folds, referred to as membrane-distal and membrane-proximal. The membrane-distal PAS domain of DctBp, where the ligand-binding pocket is located, is structurally similar to those of CitA and DcuS, despite their sequence dissimilarity with identities of 18% and 17%, respectively (Fig. 5a).

(a)

| SmDctBDistal | 120 | AAVIYLIDRSGVAVAASNWQEPTSFVGNDYAFRDYFRLA-VR-DG--------- |
| KpCitA       | 97  | ATYITVGDASGQRLYHVNP-----DEIGKSMEGGDS-DEALIN-AK--------- |
| EcDcuS       | 95  | LLFIVVTDMQSLRYSHPEA-----QRIGQPFKGDDILK-A-LNGE---------- |
| SmDctBProximal | 211 | GKPAYVTDRRGIVLITSLP-----SWRFMTTKPIAEDRLAPIR-ESLQFGDAPLL |

(b3) → (b4) → (b5) → (b6) → (b7)
α4
α6

(b)

(c)

(d)

(e)

(f)

(g)

Fig. 5. (a) Alignment of the DctBp membrane-distal PAS domain to the membrane-proximal PAS domain and to the sensor domains of CitA and DcuS. The secondary structure elements of DctBp distal and proximal PAS domains are labeled above and below the alignment, respectively. KpCitA stands for CitA from *K. pneumoniae*, and EcDcuS stands for DcuS from *E. coli*. R152<sup>DctB</sup>, Y154<sup>DctB</sup>, K197<sup>DctB</sup>, and Y179<sup>DctB</sup>, which are involved in the ligand binding of DctB via hydrogen bonds and salt bridges, are in red. The unique basic residue in the proximal PAS domain of DctBp, R290<sup>DctB</sup>, is also in red. Y124<sup>DctB</sup>, F144<sup>DctB</sup>, Y149<sup>DctB</sup>, Y99<sup>CitA</sup>, M122<sup>CitA</sup>, F97<sup>DcuS</sup>, F120<sup>DcuS</sup>, and Y215<sup>DctB</sup> in the proximal PAS domain of DctBp, which are involved in hydrophobic interactions, are in purple. Conserved ligand binding residues in CitA and DcuS are in blue. The β3–β4 loops (β2–β3 in DcuS) are in orange. (b) Ligand-binding pocket of DctBp, with SIN bound. Hydrophobic interactions between F144/Y149 and the C3 carbon atom of SIN are shown by black lines. Hydrogen bonds and salt bridges are shown by yellow lines. (c) Ligand-binding pocket of CitAp, with citrate bound. (d) Superimposition of DctBp-distal domain and CitAp. (e) Putative ligand-binding pocket of DcuS. (f) Superimposition of the distal domain and proximal domain of DctBp. Green, distal PAS domain; purple, proximal PAS domain. (g) The putative hydrophobic ligand-binding pocket in the proximal PAS domain of DctBp.

DctBp is clearly using a ligand-binding and recognition strategy different from those used by CitA family members, albeit sharing the overall structural similarities. C<sub>4</sub>-dicarboxylates sensors are shown to bind ligands as dianions.<sup>19,32</sup> The two carboxylic groups of SIN are essential for specific dicarboxylates perception. Structural and functional analyses of CitA and DcuS suggested that positively charged amino acids are necessary for the binding of the anionic dicarboxylates.<sup>16,17,19,20</sup> In the CitA–citrate complex structure, H112<sup>CitA</sup>, R109<sup>CitA</sup>, R150<sup>CitA</sup>, and K152<sup>CitA</sup> are proved to be essential residues involved in interactions with carboxyl and hydroxyl groups (Fig. 5c). According to the NMR structure and molecular genetics data of DcuS, H110<sup>DcuS</sup>, R107<sup>DcuS</sup>, R147<sup>DcuS</sup>, and F149<sup>DcuS</sup> serve important

roles in the fumarate binding and citrate binding of DcuS (Fig. 5e). The histidine residue and the two arginine residues that dominate ligand binding in DcuS and CitA are universally conserved in CitA family, whereas F149<sup>DcuS</sup>-K152<sup>CitA</sup> is believed to be a homologous replacement responsible for subtype specificity recognition.<sup>20</sup>

The three-residue loop at the bottom of the ligand-binding pocket (loop β3–β4 in CitA and DctB; loop β2–β3 in DcuS) is required for specific recognition (Fig. 5). It has been shown that drastic conformational changes in the β3–β4 loop will occur on SIN binding (Fig. 3). Correspondingly, S144<sup>CitA</sup> and L145<sup>CitA</sup> on β3–β4 loop serve essential roles in citrate binding in CitA (Fig. 5c). In addition, mutation of F141<sup>DcuS</sup>, which locates on a position similar to those of T171<sup>DctB</sup> and S144<sup>CitA</sup>, changes the subtype specificities of DcuS (Fig. 5e).<sup>20</sup>

In the structure of DctBp/SIN, it is clear that R152<sup>DctB</sup> and K197<sup>DctB</sup> are necessary for SIN and all other ligand bindings. There is a corresponding residue R141<sup>CitA</sup> in the CitA structure (Fig. 5c). However, R141<sup>CitA</sup> is about 4.8 Å away from the C3-position hydroxyl group of citrate, suggesting that this arginine is not directly involved in stabilizing the bound citrate. Furthermore, no homologous residue of R141<sup>CitA</sup> exists in DcuS. On the other side of the SIN of the DctBp/SIN structure, the C4-position carboxylic group interacts directly with K197<sup>DctB</sup> and with the main-chain nitrogen atoms on the β3–β4 loop. In the binding pockets of CitA and DcuS, the K197<sup>DctB</sup> at the N-terminal of the PAS domain disappears, and another basic residue (H112<sup>CitA</sup> and H110<sup>DcuS</sup>) at the C-terminal conserved in the CitA family plays the role of lysine (K197<sup>DctB</sup>) (Fig. 5d).

Different from the high specificity of DctB for C<sub>4</sub>-dicarboxylates, CitA binds C<sub>6</sub>-tricarboxylates with high affinity, and DcuS senses both C<sub>4</sub>-dicarboxylates and citrate. Correspondingly, besides the negatively charged residues discussed above, the ligand-binding pockets of these three sensors have many other differences. The conserved R107<sup>DcuS</sup> and R109<sup>CitA</sup> are responsible for the C6-position carboxylic group of citrate in CitA family, whereas there is no other basic residue except for R152<sup>DctB</sup> or K197<sup>DctB</sup> in the ligand-binding pocket of DctB, consistent with the formula of C<sub>4</sub>-dicarboxylates. The C2 and C3 positions of the C<sub>4</sub> ligands are held by several aromatic rings in DctB, including Y124<sup>DctB</sup>, F144<sup>DctB</sup>, and Y149<sup>DctB</sup> (Fig. 3a). Hydrophobic residues—such as M122<sup>CitA</sup>, F120<sup>DcuS</sup>, and F141<sup>DcuS</sup> on flexible loops, and Y99<sup>CitA</sup>, F97<sup>DcuS</sup>, and F149<sup>DcuS</sup> located in the central β sheet—also exist in the ligand-binding pocket of CitA and DcuS (Fig. 5). Alignment of CitA, DcuS, and DctB revealed that M122<sup>CitA</sup> and F120<sup>DcuS</sup> have positions similar to that of F144<sup>DctB</sup>/Y149<sup>DctB</sup>. Moreover, F120A<sup>DcuS</sup> mutation greatly impaired DcuS activity.<sup>19</sup> However, the phenyl ring of F120<sup>DcuS</sup> extends into the solvent region in the solution structure, and M122<sup>CitA</sup> is not involved in interaction with citrate, suggesting that the flexible loops where these two residues

locate may experience large conformational changes during subtype-specific recognition. Since the essential arginine R152<sup>DctB</sup> responsible for interacting with the C1-position carboxylic group has been moved from the central β sheet in CitA/DcuS to the N-terminus of the edge helix α4 in DctBp (Fig. 5), flexibilities of the long loop linking the β-sheet core and the following edge helix are limited by R152 in the case of DctBp. The rigidity of the flexible loop covering the binding pocket of DctB determines its specificity for dicarboxylates that have four carbon atoms forming the backbones. It has been shown that mutations altering the size of the ligand-binding site of DcuS will generate different citrate or fumarate specificities.<sup>20</sup> The variations in the carboxylic groups and backbone lengths require different ligand-binding pockets for specific recognition, and that is why DctB has evolved into a different sensory pocket from the CitA family proteins.

### C<sub>4</sub> Specificities of DctB

Various C<sub>4</sub>-dicarboxylates have different substitutions of hydroxyl, amino, and related groups at positions C2 and C3 (Fig. 3). To keep the selectivity of some special C<sub>4</sub>-dicarboxylic acids such as SIN, DctB utilizes a few aromatic residues to form the ligand-binding pocket, which is especially responsible for the C2 and C3 positions of the ligands (Fig. 3). The SIN-binding pocket in DctBpd can be defined as 10 Å in width, 6.5 Å in height, and 5.5 Å in depth. When the saturated C<sub>4</sub>-dicarboxylate SIN is in a stretched conformation, the distance between the C atoms of the carboxylic groups is about 3.8 Å. In the structure of DctBp/SIN, this distance is about 3.2 Å.

For other C<sub>4</sub>-dicarboxylates such as malate, aspartate, tartrate, oxaloacetate, and fumarate, the stretched distance of the carboxyl groups is also about 3.8 Å. Different from SIN, fumarate, and its *cis* form, maleate is frozen into one configuration by the double bond. It is known that DctA does not accept maleate for its folded conformation, but maleate ranks as the strongest inducer for the Dct system.<sup>6</sup> The structure of DctBp/SIN suggests that DctB prefers compounds with a shape similar to that of maleate. On the other hand, considering rigidity, trans-fumarate may not be a good candidate for DctB, although it can be accepted by DctA. Malate, aspartate, and oxaloacetate are widely known as good inducers for the Dct system. The structure of DctBp/SIN shows that substitution of –OH, =O, or –NH<sub>2</sub> at the C2 position of SIN will not cause steric clash. It is thus rational to predict that malate, aspartate, and oxaloacetate can be accepted by DctB in the same manner as SIN. Orotate, a cyclic monocarboxylate, has been shown to be able to induce DctA, but the substrate is unable to induce the Dct system.<sup>33,34</sup> From the structure of DctBp/SIN, we can understand that an orotate molecule must be too large to be accommodated by the binding pocket, explaining why orotate cannot function as an inducer for DctB.

The membrane-proximal PAS domain

The membrane-proximal PAS domain of DctB is clearly the most different point between DctB and the other sensory proteins of the CitA family. The membrane-distal and membrane-proximal PAS domains of DctBp showed quite different characteristics. First, in relation to function, no ligand binding occurs in the proximal PAS domain. As indicated in Fig. 2a, among the different DctB family members aligned, the residues in the distal PAS domain are well-conserved, while the residues in the proximal PAS domain are much less conserved. The sequence identity between the membrane-distal and membrane-proximal PAS domains of DctBp is below 20%. Superposition of those two domains yields an RMSD of 2.5 Å for all the Cα atoms compared. Those two values indicate the sequence and structure dissimilarities between the membrane-distal and the membrane-proximal PAS domains. Second, the pocket in the proximal PAS domain corresponding to the ligand-binding site of the distal PAS domain has a surprisingly hydrophobic environment (Fig. 5g). Only one basic residue, R290, located similarly to R150CitA and R147DcuS exists in the putative ligand-binding pocket of the proximal PAS domain of DctBp. Moreover, helix α6 in the proximal PAS domain adopts a dramatically different orientation with the corresponding helix α4 in the distal PAS domain (Figs. 2 and 5f). The loop after helix 6 is very long. F263, similar in orientation to Y149/F144, extends into the putative ligand-binding pocket. The β9–β10 loop, which is the counterpart of the β3–β4 loop in the distal PAS domain of DctBp, is much longer than the normal three-residue-long carboxylate-binding loop (Fig. 5a and f). Under the experimental conditions tested in this work, the membrane-proximal PAS domain will not bind SIN even when the ligand concentration is as high as 600 μM. Taken together, the PAS fold structure of the DctBpp features a function different from that of a C₄-dicarboxylate sensor (e.g., it may be involved in the DctA–DctB interaction).

### Signaling model of DctB

Based on the overall conformational changes of the DctBp dimers before and after ligand binding, we propose a signal perception model that can be extended to the related sensory histidine kinases in general. For the unliganded DctBp, a stable dimer is formed depending on the N-terminal helices; upon binding of SIN or other C₄-dicarboxylate ligands to the membrane-distal PAS domain, drastic conformational changes occur in the ligand-binding pockets. Those conformational changes could be transmitted to the dimer interface via hydrophobic interactions between the N-terminal helices and the back of the central β sheet. The intradimeric movement will reduce the interacting surface. Through the transmembrane helices, the movement can be further transmitted to the intracellular coiled-coil regions, which propagate the upstream signal to the kinase domains, causing the signal of C₄-dicarboxylate ligand binding to trigger the phosphorylation events.

---

### Materials and Methods

#### Protein preparation and crystallization

DctBp (residues 42–311) was expressed in the *E. coli* strain BL21(DE3) as a 33-kDa N-terminal (His)₆-tagged protein, as previously described. Selenomethionine (SeMet)-labeled DctBp was produced as described. Protein refolding was performed on a nickel column at 277 K in accordance with the literature. The supernatant of a cell lysate expressing DctBp was loaded onto a 5-ml nickel-charged affinity column [equilibrated in buffer containing 20 mM Tris (pH 7.5) and 500 mM NaCl]. The column was washed with the equilibration buffer containing 50 mM imidazole and slowly washed with 100 ml of equilibration buffer containing 8.0 M urea. Target protein was refolded by slowly washing with 100 ml of equilibration buffer, and then eluted with equilibration buffer containing 300 mM imidazole. The refolded protein was further purified by a 120-ml Superdex 75 column (GE Healthcare, USA) in 20 mM Tris (pH 7.5) and 200 mM NaCl. For crystallization, hanging-drop method was performed at 289 K. Protein sample and reservoir solution were mixed 1:1 and equilibrated against 400 μl of reservoir solution. For the crystals of DctBp/MLA belonging to the space group P2₁, the protein was crystallized at 0.2 M sodium MLA (pH 6.0), 13–15% wt/vol polyethylene glycol 3350, and 5–10 mM strontium chloride; the crystals of apoDctBp were obtained under conditions of 0.1 M Hepes (pH 7.5), 15–20% polyethylene glycol 3350, and 0.04 M calcium chloride, whereas the crystals of DctBp/SIN were grown from 0.6 M sodium SIN (pH 6.0) with 3% vol/vol trifluoroethanol (Fig. 1). It normally took 4–5 weeks for the crystals to reach the full size suitable for data collection.

#### Data collection and processing

The crystals of SeMet-labeled complex DctBp/SIN were cryoprotected by 20% vol/vol dimethyl sulfoxide before being flash-cooled in a stream of cold nitrogen gas at 100 K. Three wavelength multiwavelength anomalous diffraction experiments were performed near the selenium edge at beamline BW7A of EMBL Hamburg at the DORIS storage ring of Deutsches Elektronen Synchrotron (Hamburg, Germany). Data collection was started at the peak wavelength (0.9796 Å), then followed at the inflection wavelength (0.9799 Å) and remote wavelength (0.9760 Å). Seven hundred thirty frames were collected at each wavelength with 0.5° oscillation. The diffraction data were processed using the HKL program package. For the crystals of apoDctBp and DctBp/MLA, data sets were collected on the BSRF beamline 3W1A at 100 K using a Mar165 CCD detector. All crystals were flash-frozen in liquid nitrogen and cryoprotected by 20% dimethyl sulfoxide before crystal mounting. The data set of the crystal of DctBp/MLA was processed using the programs XDS and XSCALE, and the apoDctBp data were processed using the HKL package. Data collection and processing statistics are summarized in Table 1.

# Crystal Structures of DctB Sensor Domain

## Structure determination and refinement

The SIN-bound structure was solved at 2.3 Å resolution using the multiwavelength anomalous diffraction protocol of Auto-Rickshaw, an automated crystal structure determination platform [44] in the space group C2. With this procedure, the structure factors were calculated from the measured intensity by employing the CCP4 [45] program, and all eight selenium atoms were located using SHELXD. [46] The positions of these sites were refined using the program MLPHARE. [45] Density modification and 2-fold noncrystallographic symmetry (NCS) averaging were performed using DM [47] in the correct enantiomorph. The resultant phases were used for automated model building in ARP/wARP, [48] which could only build 57% of the two molecules in the asymmetric unit. The partial model and the substructure were then used in the program OASIS-2004 [49] for dual-space fragment phasing using the peak data. Density modification and NCS averaging were repeated on the resultant phases from OASIS-2004. Map quality improved significantly, and the program ARP/wARP built 89% of the structure. Further model building was completed with manual adjustments using the program Coot, [50] alternating with additional REFMAC5 refinement cycles, which included bulk solvent correction, anisotropic scaling, and NCS restraints, and with each molecule defined as a TLS group in the modeling of anisotropy in the program REFMAC5. [51,52] The final structure was refined to an R-factor of 20.6% and an \(R_{\text{free}}\) of 25.1% using the data from 20 to 2.3 Å, with 98.6% of all residues in favored regions and with no outliers in the Ramachandran plot. [53]

A monomer from the structure of DctBp/SIN was used as the search model for the molecular replacement methods [54] to determine the structure of DctBp/MLA, using the MOLREP program. [55] Clear solutions were found in rotation and translation searches, and a back-to-back dimer was obtained by molecular replacement phasing. Simulated annealing in the program suite CNS [56] was used in the beginning of structure refinement, and several rounds of refinement were then performed in Coot and REFMAC5. When the \(R_{\text{free}}\) dropped below 30%, NCS restraint was released and TLS refinement was carried out. Meanwhile, one MLA molecule was built into the \(F_{o}-F_{c}\) density in the ligand-binding site in each monomer. The MLA-bound structure was refined to an R-factor of 19.9% and an \(R_{\text{free}}\) of 26.9% using the data from 30 to 2.3 Å, with 95.8% of the modeled residues in favored regions and 0.8% of the modeled residues in outliers.

The apoDctBp structure was solved by MOLREP. One monomer of the DctBp/MLA structure was used as the search model. Rigid-body refinement was performed first, and further refinement was carried out subsequently. After several rounds of refinement in Coot and REFMAC5, the apo structure was refined to an R-factor of 20.0% and an \(R_{\text{free}}\) of 27.5% using the data from 30 to 2.75 Å. No density suitable for C₄-dicarboxylate or C₃-dicarboxylate was observed in the ligand-binding pocket, and few water molecules were built into the pocket. All refinement statistics are also presented in Table 1.

## Figure preparation and accession codes

All figures of the structures were prepared using the program PyMOL. [‡] The coordinates and structure factors have been deposited in the Protein Data Bank (PDB) with accession codes 3E4O (C2; DctBp/SIN), 3E4P (\(P2_1\); DctBp/MLA), and 3E4Q (\(P2_1\); apoDctBp).

---

## Acknowledgements

We thank Drs. Yuhui Dong and Peng Liu for help with data collection at BSRF. This work was supported by the 973 National Key Basic Research Program on Nitrogen Fixation in China (no. 2001CB108900 to Y.W., chief scientist of the program), the National Natural Science Foundation of China (no. 30470940), and the Program of Introducing Talents of Discipline to Universities (no. B06001). Y.W. and X.-D.S. are recipients of the National Science Fund for Distinguished Young Scholars (National Natural Science Foundation of China, nos. 39925017 and 30325012, respectively).

---

## References

1. Stock, A. M., Robinson, V. L. & Goudreau, P. N. (2000). Two-component signal transduction. *Annu. Rev. Biochem.* **69**, 183–215.
2. Mascher, T., Helmann, J. D. & Unden, G. (2006). Stimulus perception in bacterial signal-transducing histidine kinases. *Microbiol. Mol. Biol. Rev.* **70**, 910–938.
3. Ronson, C. W., Astwood, P. M. & Downie, J. A. (1984). Molecular cloning and genetic organization of C₄-dicarboxylate transport genes from *Rhizobium leguminosarum*. *J. Bacteriol.* **160**, 903–909.
4. Wang, Y. P., Birkenhead, K., Boosten, B., Manian, S. & O’Gara, F. (1989). Genetic analysis and regulation of the *Rhizobium meliloti* genes controlling C₄-dicarboxylic acid transport. *Gene*, **85**, 135–144.
5. Yarosh, O. K., Charles, T. C. & Finan, T. M. (1989). Analysis of C₄-dicarboxylate transport genes in *Rhizobium meliloti*. *Mol. Microbiol.* **3**, 813–823.
6. Yurgel, S. N. & Kahn, M. L. (2004). Dicarboxylate transport by rhizobia. *FEMS Microbiol. Rev.* **28**, 489–501.
7. Gu, B., Lee, J. H., Hoover, T. R., Scholl, D. & Nixon, B. T. (1994). *Rhizobium meliloti* DctD, a sigma 54-dependent transcriptional activator, may be negatively controlled by a subdomain in the C-terminal end of its two-component receiver module. *Mol. Microbiol.* **13**, 51–66.
8. Huala, E., Stigter, J. & Ausubel, F. M. (1992). The central domain of *Rhizobium leguminosarum* DctD functions independently to activate transcription. *J. Bacteriol.* **174**, 1428–1431.
9. Lee, J. H., Scholl, D., Nixon, B. T. & Hoover, T. R. (1994). Constitutive ATP hydrolysis and transcription activation by a stable, truncated form of *Rhizobium meliloti* DCTD, a sigma 54-dependent transcriptional activator. *J. Biol. Chem.* **269**, 20401–20409.
10. Ledebur, H., Gu, B., Sojda, J., 3rd & Nixon, B. T. (1990). *Rhizobium meliloti* and *Rhizobium leguminosarum* dctD gene products bind to tandem sites in an activation sequence located upstream of sigma 54-dependent dctA promoters. *J. Bacteriol.* **172**, 3888–3897.
11. Scholl, D. & Nixon, B. T. (1996). Cooperative binding of DctD to the dctA upstream activation sequence of *Rhizobium meliloti* is enhanced in a constit-

[‡] <http://www.pymol.org>

tively active truncated mutant. *J. Biol. Chem.* 271, 26435–26442.

12. Hendrickson, W. A. (2005). Transduction of biochemical signals across cell membranes. *Q. Rev. Biophys.* 38, 321–330.

13. Giblin, L., Boesten, B., Turk, S., Hooykaas, P. & O’Gara, F. (1995). Signal transduction in the Rhizobium meliloti dicarboxylic acid transport system. *FEMS Microbiol. Lett.* 126, 25–30.

14. Yeh, J. I., Biemann, H. P., Pandit, J., Koshland, D. E. & Kim, S. H. (1993). The three-dimensional structure of the ligand-binding domain of a wild-type bacterial chemotaxis receptor. Structural comparison to the cross-linked mutant forms and conformational changes upon ligand binding. *J. Biol. Chem.* 268, 9787–9792.

15. Yeh, J. I., Biemann, H. P., Prive, G. G., Pandit, J., Koshland, D. E., Jr & Kim, S. H. (1996). High-resolution structures of the ligand binding domain of the wild-type bacterial aspartate receptor. *J. Mol. Biol.* 262, 186–201.

16. Reinelt, S., Hofmann, E., Gerharz, T., Bott, M. & Madden, D. R. (2003). The structure of the periplasmic ligand-binding domain of the sensor kinase CitA reveals the first extracellular PAS domain. *J. Biol. Chem.* 278, 39189–39196.

17. Pappalardo, L., Janausch, I. G., Vijayan, V., Zientz, E., Junker, J., Peti, W. *et al.* (2003). The NMR structure of the sensory domain of the membranous two-component fumarate sensor (histidine protein kinase) DcuS of Escherichia coli. *J. Biol. Chem.* 278, 39185–39188.

18. Borgstahl, G. E., Williams, D. R. & Getzoff, E. D. (1995). 1.4 Å structure of photoactive yellow protein, a cytosolic photoreceptor: unusual fold, active site, and chromophore. *Biochemistry*, 34, 6278–6287.

19. Kneuper, H., Janausch, I. G., Vijayan, V., Zweckstetter, M., Bock, V., Griesinger, C. & Unden, G. (2005). The nature of the stimulus and of the fumarate binding site of the fumarate sensor DcuS of Escherichia coli. *J. Biol. Chem.* 280, 20596–20603.

20. Kramer, J., Fischer, J. D., Zientz, E., Vijayan, V., Griesinger, C., Lupas, A. & Unden, G. (2007). Citrate sensing by the C₄-dicarboxylate/citrate sensor kinase DcuS of Escherichia coli: binding site and conversion of DcuS to a C₄-dicarboxylate- or citrate-specific sensor. *J. Bacteriol.* 189, 4290–4298.

21. Reid, C.J. & Poole, P.S. (1998). Roles of DctA and DctB in signal detection by the dicarboxylic acid transport system of Rhizobium leguminosarum. *J. Bacteriol.* 180, 2660–2669.

22. Neiditch, M. B., Federle, M. J., Pompeani, A. J., Kelly, R. C., Swem, D. L., Jeffrey, P. D. *et al.* (2006). Ligand-induced asymmetry in histidine sensor kinase complex regulates quorum sensing. *Cell*, 126, 1095–1108.

23. Bader, M. W., Sanowar, S., Daley, M. E., Schneider, A. R., Cho, U., Xu, W. *et al.* (2005). Recognition of antimicrobial peptides by a bacterial sensor kinase. *Cell*, 122, 461–472.

24. Cheung, J., Bingman, C. A., Reyngold, M., Hendrickson, W. A. & Waldburger, C. D. (2008). Crystal structure of a functional dimer of the PhoQ sensor domain. *J. Biol. Chem.* 283, 13762–13770.

25. Thompson, J. D., Higgins, D. G. & Gibson, T. J. (1994). CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. *Nucleic Acids Res.* 22, 4673–4680.

26. Gouet, P., Courcelle, E., Stuart, D. I. & Metoz, F. (1999). ESPript: analysis of multiple sequence alignments in PostScript. *Bioinformatics*, 15, 305–308.

27. Wallace, A. C., Laskowski, R. A. & Thornton, J. M. (1995). LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions. *Protein Eng.* 8, 127–134.

28. Lancaster, C. R., Gross, R. & Simon, J. (2001). A third crystal form of Wolinella succinogenes quinol:fumarate reductase reveals domain closure at the site of fumarate reduction. *Eur. J. Biochem.* 268, 1820–1827.

29. Iverson, T. M., Luna-Chavez, C., Cecchini, G. & Rees, D. C. (1999). Structure of the Escherichia coli fumarate reductase respiratory complex. *Science*, 284, 1961–1966.

30. Krissinel, E. & Henrick, K. (2007). Inference of macro-molecular assemblies from crystalline state. *J. Mol. Biol.* 372, 774–797.

31. Lo Conte, L., Chothia, C. & Janin, J. (1999). The atomic structure of protein-protein recognition sites. *J. Mol. Biol.* 285, 2177–2198.

32. Kaspar, S. & Bott, M. (2002). The sensor kinase CitA (DpiB) of Escherichia coli functions as a high-affinity citrate receptor. *Arch. Microbiol.* 177, 313–321.

33. Baker, K. E., Ditullio, K. P., Neuhard, J. & Kelln, R. A. (1996). Utilization of orotate as a pyrimidine source by Salmonella typhimurium and Escherichia coli requires the dicarboxylate transport protein encoded by dctA. *J. Bacteriol.* 178, 7099–7105.

34. Yurgel, S., Mortimer, M. W., Rogers, K. N. & Kahn, M. L. (2000). New substrates for the dicarboxylate transport system of Sinorhizobium meliloti. *J. Bacteriol.* 182, 4216–4221.

35. Gan, H. H., Perlow, R. A., Roy, S., Ko, J., Wu, M., Huang, J. *et al.* (2002). Analysis of protein sequence structure similarity relationships. *Biophys. J.* 83, 2781–2791.

36. Gordeliy, V. I., Labahn, J., Moukhametzianov, R., Efremov, R., Granzin, J., Schlesinger, R. *et al.* (2002). Molecular basis of transmembrane signalling by sensory rhodopsin II-transducer complex. *Nature*, 419, 484–487.

37. Hulko, M., Berndt, F., Gruber, M., Linder, J. U., Truffault, V., Schultz, A. *et al.* (2006). The HAMP domain structure implies helix rotation in transmembrane signaling. *Cell*, 126, 929–940.

38. Inouye, M. (2006). Signaling by transmembrane proteins shifts gears. *Cell*, 126, 829–831.

39. Nan, B., Zhou, Y., Liang, Y. H., Wen, J., Ma, Q., Zhang, S. *et al.* (2006). Purification and preliminary X-ray crystallographic analysis of the ligand-binding domain of Sinorhizobium meliloti DctB. *Biochim. Biophys. Acta*, 1764, 839–841.

40. Van Duyne, G. D., Standaert, R. F., Karplus, P. A., Schreiber, S. L. & Clardy, J. (1993). Atomic structures of the human immunophilin FKBP-12 complexes with FK506 and rapamycin. *J. Mol. Biol.* 229, 105–124.

41. Kristensen, O. & Laurberg, M. (2002). Expression, refolding and crystallization of Aquifex aeolicus elongation factor P. *Acta Crystallogr. Sect. D,* 58, 1039–1041.

42. Otwinowski, Z. & Minor, W. (1997). Processing of X-ray diffraction data collected in oscillation mode. *Methods Enzymol.* 276, 307–326.

43. Kabsch, W. (1993). Automatic processing of rotation diffraction data from crystals of internally unknown symmetry and cell constants. *J. Appl. Crystallogr.* 26, 795–800.

44. Panjikar, S., Parthasarathy, V., Lamzin, V. S., Weiss, M. S. & Tucker, P. A. (2005). Auto-Rickshaw: an automated crystal structure determination platform as an efficient tool for the validation of an X-ray diffraction experiment. *Acta Crystallogr. Sect. D,* 61, 449–457.

45. CCP4. (1994). The CCP4 suite: programs for protein crystallography. *Acta Crystallogr. Sect. D*, 50, 760–763.

46. Schneider, T. R. & Sheldrick, G. M. (2002). Substructure solution with SHELXD. *Acta Crystallogr. Sect. D*, 58, 1772–1779.

47. Cowtan, K. (1994). DM: an automated procedure for phase improvement by density modification. *CCP4/ESF-EACBM Newsl. Protein Crystallogr.*, 31, 34–38.

48. Perrakis, A., Morris, R. & Lamzin, V. S. (1999). Automated protein model building combined with iterative structure refinement. *Nat. Struct. Biol.*, 6, 458–463.

49. Wang, J. W., Chen, J. R., Gu, Y. X., Zheng, C. D. & Fan, H. F. (2004). Direct-method SAD phasing with partial-structure iteration: towards automation. *Acta Crystallogr. Sect. D*, 60, 1991–1996.

50. Emsley, P. & Cowtan, K. (2004). Coot: model-building tools for molecular graphics. *Acta Crystallogr. Sect. D*, 60, 2126–2132.

51. Murshudov, G. N., Vagin, A. A. & Dodson, E. J. (1997). Refinement of macromolecular structures by the maximum-likelihood method. *Acta Crystallogr. Sect. D*, 53, 240–255.

52. Winn, M. D., Isupov, M. N. & Murshudov, G. N. (2001). Use of TLS parameters to model anisotropic displacements in macromolecular refinement. *Acta Crystallogr. Sect. D*, 57, 122–133.

53. Davis, I. W., Murray, L. W., Richardson, J. S. & Richardson, D. C. (2004). MOLPROBITY: structure validation and all-atom contact analysis for nucleic acids and their complexes. *Nucleic Acids Res.*, 32, W615–W619.

54. Rossmann, M. G. & Blow, D. M. (1962). The detection of sub-units within the crystallographic asymmetric unit. *Acta Crystallogr.*, 15, 24–31.

55. Vagin, A. & Teplyakov, A. (1997). MOLREP: an automated program for molecular replacement. *J. Appl. Crystallogr.*, 30, 1022–1025.

56. Brunger, A. T., Adams, P. D., Clore, G. M., DeLano, W. L., Gros, P., Grosse-Kunstleve, R. W. *et al.* (1998). Crystallography and NMR system: a new software suite for macromolecular structure determination. *Acta Crystallogr. Sect. D*, 54, 905–921.
