
Cell. Mol. Life Sci. 65 (2008) 3981–3999  
1420-682X/08/243981-19  
DOI 10.1007/s00018-008-8224-x  
© Birkhäuser Verlag, Basel, 2008  

---

**Review**

**Roles of p53, Myc and HIF-1 in Regulating Glycolysis – the Seventh Hallmark of Cancer**

S. J. Yeung${}^{a,b,*}$, J. Pan${}^{d}$ and M.-H. Lee${}^{c}$

${}^{a}$ Department of General Internal Medicine, Ambulatory Treatment and Emergency Care, The University of Texas M. D. Anderson Cancer Center, 1515 Holcombe Blvd., Unit 1465, Houston, Texas 77030 (USA), Fax: 713-563-4491, e-mail: syeung@mdanderson.org  
${}^{b}$ Department of Endocrine Neoplasia and Hormonal Disorders, The University of Texas M. D. Anderson Cancer Center, 1515 Holcombe Blvd., Unit 1461, Houston, Texas 77030 (USA)  
${}^{c}$ Molecular and Cellular Oncology, The University of Texas M. D. Anderson Cancer Center, 1515 Holcombe Blvd., Unit 79, Houston, Texas 77030 (USA)  
${}^{d}$ Department of Pathophysiology, Sun Yat-Sen University Medical School, Guangzhou, Guangzhou (China)

Received 24 April 2008; received after revision 16 June 2008; accepted 25 July 2008  
Online First 3 September 2008

**Abstract.** Despite diversity in genetic events in oncogenesis, cancer cells exhibit a common set of functional characteristics. Otto Warburg discovered that cancer cells have consistently higher rates of glycolysis than normal cells. The underlying mechanisms leading to the Warburg phenomenon include mitochondrial changes, upregulation of rate-limiting enzymes/proteins in glycolysis and intracellular pH regulation, hypoxia-induced switch to anaerobic metabolism, and metabolic reprogramming after loss of p53 function. The regulation of energy metabolism can be traced to a “triad” of transcription factors: c-MYC, HIF-1 and p53. Oncogenic changes involve a nonrandom set of gene deletions, amplifications and mutations, and many oncogenes and tumor suppressor genes cluster along the signaling pathways that regulate c-MYC, HIF-1 and p53. Glycolysis in cancer cells has clinical implications in cancer diagnosis, treatment and interaction with diabetes mellitus. Many drugs targeting energy metabolism are in development. Future advances in technology may bring about transcriptome and metabolome-guided chemotherapy.

**Keywords.** Oncogenes, tumor suppressors, signaling pathways, mTOR, MYC, p53, HIF-1, glycolysis, Warburg phenomenon.

**Introduction**

Oncogenesis is a multi-step process with a wide variety of genetic or epigenetic changes in the malignant cells leading to six functional characteristics of cancer: 1) persistent growth signals, 2) evasion of apoptosis, 3) insensitivity to anti-growth signals, 4) unlimited replicative potential, 5) angiogenesis, and 6) invasion and metastasis [1]. This convergence from widely different combinations in genotypes to common phenotypic characteristics allows pathologists and clinicians to diagnose cancer based on radiological, cytological and histological evidence of unregulated growth, invasion and metastasis. Yet the switch from oxidative phosphorylation to glycolytic metabolism is another consistent characteristic of malignant cells that has allowed diagnosis and detection of metastasis by

* Corresponding author.

clinicians using F<sup>18</sup>-fluorodeoxyglucose positron emission tomography (PET). Genes in glycolysis, together with cell cycle checkpoint II and PLK3 pathways, are systematically upregulated in many types of cancer, and form a significant part of a cancer type-independent transcriptome signature that can identify malignancies [2]. Glycolytic metabolism has been discussed as a potential seventh hallmark or sign of cancer [3] pending understanding why and “how tumors inherently switch to glycolysis to meet energy needs”. Although the key factors/pathways behind the cancer metabolic phenotype still remain to be elucidated, the current literature shows that c-MYC, p53, and HIF-1 are crucial for the tumor cells’ aberrant metabolic behavior. In this review, we present a synthesis of the current literature to support glycolytic metabolism as the seventh sign, the roles of c-MYC, p53, and HIF-1 as key regulators as well as a discussion of the clinical implications.

### The Warburg Phenomenon

A very common characteristic of cancer cells described by Otto Warburg is that the cancer cell exhibits increased glycolytic metabolism compared with normal cells [4–6]. Using the dbEST database for expression of genes and expressed sequence tags [U.S. NCBI, National Institutes of Health [7]], it was found that genes involved in glycolysis are overexpressed in 24 different kinds of cancers representing more than 70% of human cancer cases [8]. Such transcriptomic data provide a clear overview of the extent of this phenomenon and a systems biology approach to reveal the key regulators behind coordinated changes in a large number of proteins in glycolysis. The cellular energy level as indicated by the nucleotide triphosphate/inorganic phosphate (NTP/Pi) ratio of malignant tumors is not affected by hypoxia and is decreased when glucose is deprived [9]. This shift from oxidative phosphorylation to glycolysis may not be a requirement for malignant transformation as Warburg hypothesized, but it is an epiphenomenon of transformation associated with high metastatic potential and survival advantage. In recent years, research has revealed how the AKT (protein kinase B) signaling pathway promotes continued cell growth and coordinated the necessary metabolic changes to support cell growth by increasing glucose uptake, glycolysis and ATP production. Recent discoveries in the regulation of enzymes, scaffolding protein and transporters involved in energy metabolism by oncogenes and tumor suppressor genes provide new insight into how a wide variation in the combination of oncogenes and tumor suppressor gene defects would lead to a common shift in energy metabolism from oxidative phosphorylation to glycolysis – the Warburg phenomenon.

#### Metabolism in Cancer Cells

Although there are several counterexamples of cell lines that can grow in culture media containing 5 mM galactose or 0.5 mM glucose without increased glycolysis [10, 11], most cancer cells fulfill their energy need primarily via glycolysis. Cancer cell proliferation can only proceed as metabolites accumulate to ensure an ample supply of building blocks for DNA, RNA, protein, lipid, and complex carbohydrate to prepare for mitosis [12]. Glycolysis, glutaminolysis, and de novo lipid biosynthesis form a stereotypic platform supporting cancer cell proliferation [13]. Even when the metabolic characteristics of a breast cancer cell line is compared with fast growing nonmalignant cells growing at the same rate, cancer cells have higher glucose, lactate, and glutamine fluxes per unit area of cell membrane and higher pentose phosphate pathway activity than the nonmalignant cells [14]. NAD(P)H derived by glutaminolysis and the pentose phosphate pathway are relevant providers of energy for anabolism. Fatty acid synthase (FASN), a key regulator of de novo lipid biosynthesis and a gene highly expressed in most carcinomas, is regulated by glucose via the carbohydrate responsive element binding protein (ChREBP), by glucocorticoids via sterol regulatory element binding protein-1 (SREBP-1), and by AKT/hypoxia-induced factor-1 (HIF-1) signaling inducing the SREBP-1 gene [15, 16]. Synthesis of palmitate uses one acetyl-CoA, seven malonyl-CoA, and seven NADPH [15], and this de novo fatty acid synthesis pathway depends on reductive power from NADPH and replenishment of oxaloacetate for the tricarboxylic acid (TCA) cycle, both of which can be provided by glutaminolysis [17].

In this review, we shall focus on glycolysis without detail coverage of fatty acid synthesis, glutaminolysis and the pentose phosphate pathway. High rates of glycolysis are consistently observed in cancers compared with benign tissues, and are due to upregulation of enzymes in glycolysis [e.g., hexokinase 2 (HK2), glyceraldehyde-3-phosphate dehydrogenase (GAPDH), 6-phosphofructo-1-kinase (PFK1), triose-phosphate isomerase (TPI), phosphoglycerate kinase 1 (PGK1), and enolase 1 (ENO1), and pyruvate kinase (PK) [8, 18–22]] and a coordinated decrease in some enzymes in gluconeogenesis and mitochondrial respiration [20]. The cancer transcriptome is characterized by high expression of genes involved in glycolysis, glutaminolysis and generation of phosphometabolites, providing a high throughput of supply materials and bio-energy from glucose for biosynthetic processes and cellular functions [23]. Both glycolysis and glutaminolysis generate

Cell. Mol. Life Sci. Vol. 65, 2008

Figure 1. Glycolysis Can Promote Resistance to Cancer Therapy. Glycolysis provides the metabolites and energy for DNA repair and chemotherapy drug inactivation/detoxification. Glycolysis can provide ATP/NAD⁺ (consumed by PARP) for DNA repair, and glycolysis, pentose phosphate pathway and glutaminolysis can provide NADPH, a universal reductant, for chemotherapy drug detoxification. These mechanisms can potentially contribute to resistance of the cancer to therapy.

metabolic intermediates for biosynthesis: e.g., glucose-6-phosphate for the formation of ribose-5-phosphate via the pentose phosphate pathway and glutamine for the formation of ammonia and aspartate which are used in the synthesis of purine and pyrimidine nucleotides. The relative activities of glycolysis and glutaminolysis may shift in solid tumors [24], and a shift to glycolysis is observed as solid tumors increase in size [25]. Pyruvate kinase type M2 (PKM2) is highly active in tetrameric form and less active in dimeric form. The tetramer-to-dimer shift of PKM2 controls relative activity of glycolysis and glutaminolysis; when PKM2 is in its dimeric form, glutaminolysis dominates [23]. This flexibility also depends on the level of expression of shuttle enzymes such as cytosolic glycerol 3-phosphate dehydrogenase and malate dehydrogenase (isoform with pI 7.8) [26], and some cancer cells (e.g., MCF-7 breast cancer cells) are completely dependent on glucose for growth and survival.

Glycolytic metabolism also offers the cancer cells survival advantages: independence from oxygen supply, ability to detoxify chemotherapy drugs and ability to repair DNA damage. The glycolytic pathway is linked to generating NAD(P)H to catalyze other redox reactions [27]. NADPH is also important for the cancer cells to survive chemotherapy by drug detoxification using the cytochrome P-450 system. In the drug detoxification process, phase I drug metabolizing enzymes such as NADPH-cytochrome P450 reductase (P450R), require a supply of NADPH which can be supplied by the pentose phosphate pathway [28] (Fig. 1). In addition to allowing glycolysis to continue, NAD⁺ regeneration is very important to cancer cells because they are required for repair of genotoxic damages. For DNA repair, poly(ADP-ribose) polymerase (PARP-1) is activated at the sites of DNA damage. PARP-1 breaks NAD⁺ into nicotinamide and ADP-ribose, polymerizes ADP-ribose, and transfers ADP-ribose moieties to carboxyl groups of nuclear proteins, consuming a large quantity of NAD⁺/ATP (Fig. 1). PARP-1 activation leads to rapid depletion of the cytosolic NAD⁺ pool and renders the cells unable to utilize glucose as a metabolic substrate [29]. The tumor metabolome (the complete set of metabolic intermediates, hormones and other signaling molecules, and secondary metabolites) also suggests a vulnerability of the cancer cells to a reduction of NAD⁺ after DNA damage [23].

Regulation of Glycolytic Flux in Cancer Cells. Regulation of glycolysis in cancer cells is different from non-malignant cells not only in the upregulation of enzymes mediating the pathway but also in the expression of different isoforms of several key enzymes (e.g., expression of HK2 and PKM2). Normally, the glycolytic flux rate is down-regulated by ATP produced by mitochondrial respiration. There are several layers of control over glycolysis. Glycolysis is stimulated by fructose 1,6-bisphosphate and fructose 2,6-bisphosphate [30]. The main control of glycolytic flux by mitochondrial respiration is mediated by the inhibition of PFK-1 activity by ATP. The tetramer-dimer ratio of PKM2, which controls the kinase activity, is regulated by ATP, amino acids synthesized from glycolytic intermediates (e.g., L-serine and L-alanine), fatty acids and fructose 1,6-bisphosphate [31, 32]. HK2 binds to mitochondrial porins, and catalyzes the first step in glycolysis without inhibition by glucose 6-phosphate. Due to high activity of mitochondria-bound HK2 in the presence of sufficient glucose, all phosphometabolites above pyruvate kinase accumulate until the levels of fructose 1,6-bisphosphate and fructose 2,6-bisphosphate are high enough to activate PFK-1 and to shift PKM2 to the active tetrameric state [12]. Moreover, different isoforms of the four genes (PFKFB1–4) encoding 6-phosphofructo-2-kinase/fructose 2,6-bisphosphatase (PFK-2/FBPase-2) [33] regulate the level of fructose 2,6-bisphosphate which stimulates

glycolysis by a potent positive allosteric effect on PFK-1 and inhibits gluconeogenesis by blocking fructose-1,6-bisphosphatase (FBPase-1) [33].

### Causes of Increased Glycolytic Metabolism in Malignant Cells

Several mechanisms have been proposed and probably all contribute in different degrees in different cancers to bring about a common phenotype of glycolytic metabolism – the Warburg phenomenon. These mechanisms include 1) mitochondrial changes, 2) hypoxia-induced switch from oxidative mitochondrial respiration to glycolysis, 3) changes in the metabolome or metabolite pools that facilitate glycolytic flux, and 4) coordinated regulation of proteins that control glycolytic flux. Behind these mechanisms are the genetic and epigenetic changes in proto-oncogenes and tumor suppressor genes in the multi-step process of carcinogenesis.

#### Mitochondrial Changes in Cancer

Cancer cells often have reductions in mitochondrial DNA, lower transcription rate for the mitochondrial genome overall, and accumulation of mitochondrial genomic mutations and deletions. Decrease in oxidative phosphorylation can be due to genetic changes in the mitochondria. Mutations in certain mitochondrial genes can disrupt the electron transport chains and oxidative phosphorylation. The mitochondrial genome is particularly susceptible to mutation for several reasons. Electrons may escape or leak from electron transport complexes, mainly at complexes I and III, to react with molecular oxygen, forming superoxide radicals ($\mathrm{O}_2^-$) and other reactive oxygen species that damage mitochondrial DNA. Mitochondrial DNA is supercoiled, circular and prone to breakage. Moreover, there are few repair mechanisms for mitochondrial DNA.

Mitochondrial DNA defects in cancer have been reviewed [34] and summarized in Table 1. Two general features of mitochondrial DNA mutations are seen in cancers irrespective of tissue types. First, the majority of mutations are base transitions (T to C and G to A). Second, while mutations occur in many mitochondrial genes, the D-loop region of the mitochondrial genome contains the frequent sites of somatic mutations. However, changes in the mitochondrial DNA differ among different cancers [35], and are not consistent enough to entirely account for the Warburg phenomenon.

In some cancers, mitochondrial germline mutations can facilitate carcinogenesis [36], but in most cases such mutations are acquired during or after carcinogenesis. The accumulation of defects in the mitochondrial genome may disable ATP generation via oxidative phosphorylation [36, 37], necessitating a switch to glycolytic metabolism. Loss of mitochondrial respiration increases NADH which inactivates PTEN (Phosphatase and Tensin homolog) through a redox modification mechanism, resulting in AKT activation [38]. Inactivating mutations of mitochondrial succinate dehydrogenase (SDH subunits B, C or D) and fumarate dehydrogenase can lead to pheochromocytoma (SDH mutations) and leiomyoma, leiomyosarcoma or renal carcinoma (fumarate dehydrogenase). The consequent accumulation of fumarate and succinate inhibits prolyl hydroxylases (the enzymes that control HIF-1α stability), leading to resistance to apoptosis and HIF-1-mediated reprogramming of the metabolism towards aerobic glycolysis [39].

A nuclear gene that regulates mitochondrial respiration is the tumor suppressor p53 (TP53, also known as p53)-inducible gene synthesis of cytochrome c oxidase-2 (SCO2) [40]. SCO2 is critical for regulating the cytochrome c oxidase (COX) complex, the major site of oxygen utilization in oxidative phosphorylation. SCO2 couples p53 to mitochondrial respiration and provides a possible explanation for the Warburg phenomenon [40]. Another possible mechanism for decreased oxidative phosphorylation is overexpression of the ATPase inhibitor protein (IF1) [41]. Decrease in $\mathrm{H}^+$-ATP synthase ($\beta$-F1-ATPase) is a proteomic signature of decreased oxidative phosphorylation and characteristic of cancer cell bioenergetics, which can predict the prognosis of colon, lung, and breast cancer. The level of this protein inversely correlates with the glycolytic rate in cancer cells [42]. When oxidative phosphorylation becomes inefficient or defective, loss of mitochondrial ATP removes the inhibition on glycolysis, which provides a compensatory mechanism to generate ATP. Despite the loss of oxidative phosphorylation, mitochondria remain essential in the processing of intermediate metabolites for various pathways involving carbohydrates, amino acids, and fatty acids. Cancer cells depleted of mitochondrial DNA by treatment with ethidium bromide ($\rho 0$ cells) [43] continue to maintain mitochondrial mass.

#### Hypoxia-Induced Switch from Oxidative Mitochondrial Respiration to Glycolysis

The solid tumor microenvironment is characterized by a disorganized microvasculature [44], increased interstitial pressure [45] and the presence of hypoxic zones [46]. Hypoxia or low oxygen tension is a characteristic of this microenvironment, and is also a characteristic of bone marrow from which hematologic malignancies develop. Malignant cells may be forced to adapt to low

Table 1. Mitochondrial genetic changes in cancers.

| Cancer Type                     | Mitochondrial Changes                                                                                          | Percentage of Cases |
|--------------------------------|----------------------------------------------------------------------------------------------------------------|---------------------|
| Breast cancer                  | Mutations in the D-loop region of mitochondrial genome 60%<br>16S rRNA, ND1, ND2, ND4, ND5, Cytochrome b, and ATPase 6 | <15%                |
| Ovarian cancer                 | D-loop, 12S rRNA, 16S rRNA, and cytochrome b mutations. Most are T → C or G → A transitions.                      | 60%                 |
| Colorectal cancer              | 12S rRNA, 16S rRNA, ND1, ND4L, ND5, Cytochrome b, COXI, COXII, and COXIII genes. Most are T → C or G → A transitions. | 70%                 |
| Gastric cancer                 | Deletion of mtDNA<br>Insertions/deletions in the D-loop region or transitions in ND1, ND5, and COXI                   | 54%<br>44%          |
| Hepatocarcinoma                | Mutations in the D-loop                                                                                       | Frequent            |
| Esophageal adenocarcinomas or  | D-loop alterations                                                                                            | 40%                 |
| Barrett's esophagus            |                                                                                                                |                     |
| Esophageal carcinoma           | D-loop mutations                                                                                              | 5%                  |
| Pancreatic cancer cell lines   | 12S rRNA, 16S rRNA, ND1, ND2, COXI, COXII, ATPase 6, COXIII, ND4, ND4L, ND5, ND6, Cytochrome b, as well as the non-coding D-loop region. Also 6-fold to 8-fold increase in the mtDNA mass. | 100%                |
| Renal cell carcinoma           | A 264-bp deletion of the ND1;<br>Loss of mtDNA and mRNA coding for subunit the ND3 gene<br>Loss of ATP synthase activity in Complex V | 100%<br>100%        |
| Prostate cancer                | D-loop region, 16S rRNA, and NADH subunits                                                                     | 18.75%              |
| Brain tumors                   | mtDNA highly amplified                                                                                         | 87%                 |
| Thyroid cancer                 | mtDNA alterations in the genes coding for Complex I and Complex IV of the respiratory chain<br>The majority of the mutations that were found occurred in the genes coding for subunits of complex I of the respiratory chain; the mtDNA common deletion was identified in 100% of Hurthle cell tumors, 33.3% of adenomas, and in 18.8% of non-Hurthle cell papillary carcinomas. | —                    |
| Hematologic malignancies       | Described mutations in cytochrome b, cytochrome c oxidases I and II and ATPase 8; increased mutations in the mitochondrially-encoded COXI and COXII genes | —                    |

ND – mitochondrially encoded NADH dehydrogenase; COX I-III – cytochrome oxidase subunit I-III; mtDNA – mitochondrial DNA; rRNA – ribosomal RNA; NADH – reduced form of nicotinamide adenine dinucleotide.

oxygen tension as they grow further away from the existing blood supply. Lack of oxygen shuts down oxidative mitochondrial respiration, the cancer cells must switch on glycolytic metabolism for bioenergy. The lack of mitochondrial ATP will remove the inhibition of PFK-1 and PKM2. PFKFB1-4 genes are responsive to hypoxia in vivo, indicating a physiological role of glycolysis in the adaptation to hypoxia [47]. There is also evidence for overexpression of a specific splice isoform of PFKFB-4 mRNA in cancer cells under hypoxic conditions [48]. Overall, hypoxia activates HIF to induce adaptive responses including angiogenesis, glycolysis, and pH regulation [49]. The hypoxic microenvironment in which the cancer cells thrive may constitute a selection pressure to select for tumor cell clones with high glycolytic

metabolism as the cells evolve through the carcinogenic process [50].

Metabolomic Changes Facilitating Glycolysis. As discussed above, fructose 2,6-bisphosphate is an important metabolite that stimulates glycolysis by a potent positive allosteric effect on PFK-1 and inhibits gluconeogenesis by blocking fructose 1,6-bisphosphatase (FBPase-1) [33]. A p53-inducible enzyme TIGAR (TP53-induced glycolysis and apoptosis regulator) is a FBPase-2 which functions to lower fructose 2,6-bisphosphate levels, and thereby inhibit glycolysis by decreasing the activity of PFK-1 and enhancing the activity of FBPase-1 [51]. Since FBPase-1 activity is reduced in many tumor cells (often due to loss of p53 function and the resultant downregulation of TIGAR

ATP/AMP → AMPK  
↓  
TSC1  
/TSC2  
↓  
Rheb  
↓  
GSK-3  
↓  
β-Catenin  
↓  
TCF4  
↓  
MYC  
←  
AKT  
↓  
mTOR  
↓  
eIF-4E  
↓  
HIF  
↓  
p53  
←  
MDM2  
↓  
GLUT1  
↓  
HK  
↓  
Glucose  
↓  
Glycolysis  
↓  
Pyruvate  
↓  
Lactate  
↓  
Ox Phos  
↓  
TCA  
↓  
Ac CoA  
↓  
FA Synthesis  
↓  
FASN  
↓  
ACL  
↓  
CPT1  
↓  
β-oxidation  

Figure 2. The PI3K/AKT/mTOR Signaling Pathway Regulates Cancer Cell Metabolism. See details in the text. Arrows represent stimulation/activation, and ends represent inhibition.

expression [51, 52]), the fructose 1,6-bisphosphate level remains persistently elevated, and the “brake” on glycolysis is removed. Yet the levels of TIGAR expression in various cancer types have not been examined. Nevertheless, fructose 1,6-bisphosphate may be the key metabolite that increases the activity of PKM2, leading to a drastic increase in forward flux through glycolysis.

Three other points in the metabolome restricting glycolysis are accumulation of pyruvate, accumulation of reduced hydrogen and depletion of NAD⁺ as glycolysis proceeds. Increased glycolysis alters the glycerol-3-phosphate and malate-aspartate shuttles, reducing transport of H⁺ into the mitochondrial intermembrane space, requiring the cancer cells to oxidize NADH to regenerate NAD⁺ in the cytosol by lactate dehydrogenase (LDH). c-MYC induces increase in LDH type A (LDH-A) expression [53], and LDH-A which converts pyruvate to lactate plays a key role in carcinogenesis [54]. Reduction in LDH-A using short hairpin RNAs stimulates oxidative phosphorylation and decreases mitochondrial membrane potential. The tumorigenicity and ability to proliferate under hypoxia are decreased in LDH-A-deficient cells [54].

The reduction in transport of H⁺ into the mitochondrial intermembrane space in cancer cells exhibiting the Warburg phenomenon also presents a challenge in intracellular pH (pHi) regulation. In normal cells, the Na⁺-driven Cl⁻/HCO₃⁻ exchanger (NHE1) and the Na⁺-independent Cl⁻/HCO₃⁻ exchanger are primarily responsible for maintenance of pHi. In cancer cells, high NHE1 activity increases pHi and acidifies the extracellular space. The increased pHi facilitates glycolysis, and the resulting lactate is transported out of the cancer cells via the H⁺/lactate cotransporter [55]. Other proteins involved in pHi regulation include the monocarboxylate carriers that transport bicarbonate anions and carbonic anhydrase. Carbonic anhydrase IX is a hypoxia-inducible transmembrane protein and the only tumor-associated carbonic anhydrase isoenzyme known [56, 57] and it is needed for growth and survival of cancer cells under both normoxia and hypoxia [58].

### Coordinated Regulation of Proteins that Control the Glycolytic Flux

#### Regulation of Cancer Cell Metabolism by the AKT Signaling Pathway
Activation of the AKT signaling may be sufficient to bring about the switch to glycolytic metabolism in cancer [59] (Fig. 2). AKT signaling regulates the transcription [60] and translation [through mammalian target of rapamycin (mTOR) and eukaryotic translation initiation factor 4E binding protein 1 (4E-BP1)] [61] of glucose transporter 1 (GLUT1). AKT activates HK2 association with the mitochondria, which promotes phosphorylation of glucose to glucose 6-phosphate to be metabolized via glycolysis or the pentose phosphate pathway, and the mitochondria-associated HK2 is involved in inhibition of apoptosis [62, 63]. AKT also regulates de novo fatty acid synthesis and usage of fatty acid for β-oxidation. It phosphorylates ATP citrate lyase (ACL), stimulating cleavage of citrate to oxaloacetate and acetyl-coenzyme A (Ac-CoA) to supply downstream de novo fatty acid synthesis [64]. Phosphoinositide 3-kinase (PI3K) and

Cell. Mol. Life Sci. Vol. 65, 2008 Review Article 3987

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 3. c-MYC, HIF-1 and p53 Regulates Glycolytic Metabolism. The Warburg phenomenon is at least in part due to upregulation of genes coding for glucose transporters and glycolytic and regulatory enzymes mediated by increased activity of the transcription factors c-MYC and HIF-1 in cancer cells, and a coordinated loss of regulatory proteins due to loss of p53 function. Loss of p53 function also leads to activation of GLUT-3 transcription via NFκB. The genes controlled by p53 are in blue fonts; by MYC in green fonts; by HIF-1 in cyan fonts; by both HIF-1 and MYC in red fonts. Arrows represent stimulation/activation, and ends represent inhibition. + indicates synergism. HK2 – hexokinase type 2; GPI – glucose phosphate isomerase; PFK1 – phosphofructokinase 1; PFK2 – phosphofructokinase 2; ALDA – aldolase A; TPI – triose phosphate isomerase; GAPDH – glyceraldehyde 3-phosphate dehydrogenase; PGK1 – phosphoglycerate kinase 1; PGM – phosphoglycerate mutase; ENO1 – enolase 1; PKM2 – pyruvate kinase type M2; LDH-A – lactate dehydrogenase type A; PDK1 – pyruvate dehydrogenase kinase-1; TIGAR – TP53-induced glycolysis and apoptosis regulator; SCO2 – synthesis of cytochrome c oxidase-2; IKK – I-kappa-B kinase; NFκB – nuclear factor-kappa-B; GLUT – glucose transporter.

AKT suppresses expression of the β-oxidation enzyme carnitine palmitoyltransferase 1A (CPT1A), and modulation of CPT1A expression by PI3K/AKT signaling is the mechanism to suppress β-oxidation during cell growth [65].

mTOR is downstream of the PI3K/AKT pathway and is regulated by AMP-activated protein kinase (AMPK) (the cellular energy sensor), the tuberous sclerosis 1 & 2 (TSC1-TSC2) complex, and Ras homolog enriched in brain (RHEB) [66]. mTOR is situated in the crossroads of signaling pathways and is an integration center of the signals to bring coordinated regulation of nutrient uptake, energy metabolism, cell growth, proliferation, and cell survival [67, 68]. Most importantly, mTOR is an upstream activator of HIF-1α in cancer cells [69], which is a subunit of a transcription factor that upregulates the expression of nearly all the genes involved in the glycolytic pathway [70]. Under hypoxic conditions, glucose supply also regulates HIF-1α activity. Hypoxia combined with glucose deprivation activates AMPK (detected by phosphorylation of AMPK) [71], and this combination inhibits the accumulation of HIF-1α by suppressing translation rather than changing transcription or proteasomal degradation [72].

**Regulation of Glycolysis by a Triad of Transcription Factors.** Three transcription factors, c-MYC, HIF-1 [i.e., the active complex of HIF-1α and HIF-1β (also known as aryl hydrocarbon receptor nuclear translocator, ARNT)] and p53, regulate the flux of glucose through the glycolytic pathway (Fig. 3). The transport of glucose into the cancer cells is controlled by glucose transporters including GLUT-1, which are regulated by HIF-1. Hexokinases are important enzymes that regulate glycolysis, and HK2 is the isoform expressed specifically in skeletal muscle, adipocytes and cancer cells. HK2 regulates the first step in glycolysis [73], and it is regulated by p53 as well as HIF-1. The upstream regulatory element of the HK2 gene con-

contains a carbohydrate response element (ChoRE) and response elements for protein kinase A, protein kinase C, HIF-1, and p53 [74, 75]. In cancer cells, the HK2 gene is amplified, activated, and induced by multiple signal transduction cascades, and the over-expressed HK2 binds to the outer membrane of mitochondria. HIF-1 is the major transcription factor regulating the transcription of the majority of the enzymes in the glycolytic pathway, all the way from glucose down to lactate [70] (Fig. 3). The promoter regions of the genes of these enzymes have been shown to have HIF-1 regulatory elements [74, 75]. Pyruvate may also regulate the levels of glycolytic enzymes by preventing the oxygen-induced degradation of HIF-1α protein and thus activating HIF-1 [76]. The enzyme that is very important in regulating pyruvate level is LDH-A, and the cis-acting elements of its gene promoter resemble the core of the ChoRE and E-box (5′-CAGGTG-3′), and they overlap with the consensus binding sites for both c-MYC and HIF-1. Increased activities of HIF-1 and/or c-MYC upregulate glycolytic enzyme genes to increase the glycolytic capacity in cancer cells [74]. Another enzyme regulated by HIF-1 and c-MYC is pyruvate dehydrogenase kinase-1 (PDK1) which inhibits pyruvate dehydrogenase by phosphorylation, stopping conversion of pyruvate to acetyl-CoA and thus depleting the fuel supply for oxidative phosphorylation [77]. The importance of HIF-1 in the Warburg phenomenon is supported by the fact that aerobic glycolysis is inhibited when HIF-1α level is decreased in renal carcinoma cells [78]. p53, one of the most frequently mutated genes in cancers, controls the balance between oxidative respiration and glycolysis through two important p53-inducible genes (TIGAR [51] and SCO2 [40]) as discussed above. p53 represses the transcriptional activity of the GLUT1 and GLUT4 gene promoters by direct DNA binding leading to decrease in glucose uptake [79]. Another recent report has demonstrated that the inhibitory effect of p53 on I-kappa-B kinase (IKK) dampens the positive feedback loop between glycolysis and IKK-nuclear factor-kappa-B (NFκB) signaling [80], and loss of p53 will activate NFκB to transcriptionally activate the expression of GLUT3 and to increase in the rate of aerobic glycolysis [80]. p53 also induces ubiquitination and degradation of phosphoglycerate mutase (PGM), and loss of p53 results in an increase in PGM protein level and enhanced glycolysis [81]. Therefore, it has become clear that this triad of transcription factors, HIF-1, c-MYC and p53, are responsible for a coordinated shift in cancer cell metabolism from oxidative phosphorylation to glycolysis.

Interaction of Signaling Pathways Regulating HIF-1, c-MYC and p53. In the cell survival response to hypoxia, c-MYC interacts with hypoxia-induced factors by various mechanisms [82]: 1) HIF-1α counteracts c-MYC activities through c-MYC displacement at the promoters of cell cycle and DNA repair genes; 2) HIF-2α, in contrast to HIF-1α, stimulates MAX binding with c-MYC to enhance c-MYC/MAX transcription activities; 3) HIF-1α inhibits c-MYC activity by binding to MAX and induces MAX interacting protein 1 (MXI1) expression which binds to MAX to repress expression of MYC target genes; 4) HIF-1α cooperates with c-MYC to enhance expression of common target genes regulating metabolism (e.g., HK2, PDK1) [82] (Fig. 4). c-MYC is controlled by β-catenin which is controlled by adenomatous polyposis coli tumor suppressor (APC) and further upstream signals from wingless-type MMTV integration site family (WNT) and the Frizzled receptor for WNT ligands. The non-receptor tyrosine kinase c-SRC and Abelson murine leukemia viral oncogene homolog 1 (ABL1) are also able to regulate the transcription of the c-MYC. HIF-1α competes with T-cell factor-4 for direct binding to β-catenin, and β-catenin can enhance HIF-1-mediated transcription by interacting with HIF-1α at the promoter region of HIF-1-regulated genes [83]. In carcinogenesis, c-MYC is deregulated frequently due to chromosomal translocations, leading to unregulated overexpression c-MYC. c-MYC activation collaborates with HIF to confer metabolic advantages to cancer cells (the Warburg phenomenon) to thrive in a hypoxic microenvironment [77, 82, 84, 85].

HIF-1α can bind to and stabilize p53 [86], and there are two p53-binding sites within the HIF-1α oxygen-dependent degradation (ODD) domain [87]. Low level of p53 attenuates HIF-1 transactivation by competing for p300, but high level of p53 degrades HIF-1α protein [88]. Overall, HIF-1 stimulates angiogenesis and induces adaptation to hypoxia whereas p53 mediates hypoxia-induced apoptosis [89]. In the absence of functional wild-type p53, HIF-1 activity is not attenuated. Mouse double minute 2 homolog (MDM2) is a ubiquitin-ligase that regulates the level of p53, and it can also interact with HIF-1α. Over-expression of MDM2 increases HIF-1α protein in hypoxic cells and increased HIF-1 transcriptional activity, perhaps in a p53-independent manner [90]. The functions of p53 and c-MYC are linked through ubiquitination by several E3 ubiquitin ligases, their regulator alternative reading frame (ARF), and ARF-binding protein 1 (ARF-BP1, also known as HectH9), constituting an intricate network balancing growth and apoptosis [91]. ARF-BP1 is activated to turn on c-MYC activity, but can also turn off p53 through ubiquitination.

Cell. Mol. Life Sci. Vol. 65, 2008 Review Article 3989

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 4. MYC, HIF-1α and p53 Interplay to Coordinate Regulation of Cancer Cell Metabolism. HIF-1α inhibits c-MYC activity, and induces MXI1 expression which binds to MAX to repress expression of MYC target genes, but for certain genes regulating metabolism (e.g., HK2, PDK1), HIF-1α cooperates with c-MYC to enhance expression. HIF-1α can bind to and stabilize p53. High level of p53 degrades HIF-1α but induces MDM2. MDM2 decreases p53 but increases HIF-1α. ARF-BP1/HectH9 is activated to turn on c-MYC activity, but can also turn off p53 through ubiquitination. c-MYC induces ARF, which suppresses c-MYC by inactivating ARF-BP1/HectH9, and activates p53 by suppressing MDM2 and ARF-BP1/HectH9. Arrows represent stimulation/activation, and ⊥ ends represent inhibition.

When c-MYC is overactive, it induces ARF which suppresses c-MYC action, inactivates ARF-BP1, and activates p53 by suppressing the E3 ligase activities of both MDM2 and ARF-BP1. In carcinogenesis, this regulatory network is often perturbed such that c-MYC activity is high while p53 activity is lost.

Convergence from a Constellation of Different Changes in Oncogenes and Tumor Suppressor Genes to a Common Phenotype of Glycolytic Metabolism. Other than HIF-1, p53 and c-MYC, some oncogenes and tumor suppressor genes have been studied in the context of energy metabolism in cancer cells. In clear cell renal carcinomas, deficiency of the wild-type von Hippel-Lindau tumor suppressor (VHL) protein (a component of HIF-1α ubiquitin ligase) is one of the factors responsible for downregulation of the biogenesis of oxidative phosphorylation complexes [92]. For many other oncogenes and tumor suppressor genes, they are situated along the signaling pathways regulating HIF-1, p53 and c-MYC, and their role potential impact on glycolytic metabolism often can be traced to their impact on HIF-1, p53 and c-MYC (Fig. 3).

It is also clear that glycolysis and several functional characteristics of cancer [persistent growth signals, evasion of apoptosis, insensitivity to anti-growth signals, angiogenesis] are linked. Suppression of the intrinsic apoptotic program may be achieved through mechanisms that directly lead to the Warburg phenotype [93, 94]. Persistent growth signaling through the AKT/mTOR signaling pathway will lead to the same metabolic phenotype [94]. PI3K/AKT signaling can lead to translocation of HK2 to the mitochondrial membrane and bind to the voltage-dependent anion channel (VDAC), negatively modulating truncated BH3-interacting domain death agonist (tBID) and perhaps BCL2 antagonist of cell death (BAD) to inhibit apoptosis [95]. HIF-1, the transcription factor for vascular endothelial growth factor (VEGF), links angiogenesis with glycolytic metabolism. A nuclear form of glyceraldehyde 3-phosphate dehydrogenase is part of the coactivator complex that binds to the transcription factor OCT-1 to control S-phase dependent histone expression, thereby linking the cellular metabolic state (redox status and NAD⁺ availability) to cell cycle regulators and synthesis of structural components in cell proliferation [96]. c-MYC, other than control of the energy metabolism through LDHA, PDK1, etc., is also controlling cell cycle progression, apoptosis, and overcome anti-growth signals.

Typically, a series of genetic (e.g., mutation, translocation, deletion, and amplification) or epigenetic changes (e.g., hypermethylation) to several of these genes are required before a normal cell transforms into a malignant cell. The Wellcome Trust Sanger Institute Cancer Genome Project found that mutations in more than 1% of genes contribute to cancer [97]. An updated cancer gene census is posted at http://www.sanger.ac.uk/genetics/CGP/Census. Among the 387 genes currently listed, there are at least 10% of these genes involved in the signaling pathways that regulate at least one member of the triad of transcription factors controlling glycolytic metabolism (i.e., KIT, MET, RET, EGFR, ERBB2, NTRK1, NTRK3, PDGFRA, PDGFRB, FGFR1, FGFR2, FGFR3, FLT3 are tyrosine kinase growth receptors which can activate RAS and AKT signaling to regulate HIF-1 and c-MYC; NF1, NRAS, HRAS, KRAS, BRAF, GRAF, MAP2K4 are in the RAS signaling pathway; TSC1, TSC2, STK11, PIK3CA, PTEN, are in the AKT signaling pathway; VHL and ARNT regulate HIF-1; ABL1, ABL2, APC, CDH1, CTNNB1, CDKN2A-p14ARF, FBXW7, MYC, MYCL1, MYCN are in the pathway regulating c-MYC; CDKN2A-p14ARF, ATM, TP53 are in the pathway regulating p53). Three additional important genes involved in carcinogenesis and regulate MYC, HIF-1α and p53 that are not included in the cancer gene census are SRC, MDM2 and 14-3-3σ. High resolution comparative genomic hybridization analysis of breast cancer samples reveals non-random regions where DNA copy number is commonly gained or lost [98]. Similar results were obtained in 24 lung adenocarcinoma samples [99], suggesting genes within these regions are critical to the malignant phenotype. The oncogenes and tumor suppressor genes that

can impact glycolytic metabolism also fall into these regions such that the oncogenes gain and tumor suppressor genes lose copy numbers, partially explaining changes in signaling through these pathways in oncogenesis (Fig. 5). Six of seven cancer genes that have amplification and 11 of 28 cancer genes that have deletions in the study by Futreal et al. [97] are expected to result in promotion of glycolytic metabolism[97]. Genome-wide analysis in 11 breast and 11 colorectal cancers showed that each cancer may accumulate about 11 cancer causing mutations per tumor [100]. More stringent statistical analyses identified the following genes with mutation rates significantly higher than background: p53 in breast cancer and APC, KRAS, TP53, SMAD4, and FBXW7 in colorectal cancer [101, 102]. Therefore, mutations (activating mutations in oncogenes and loss of function mutations in tumor suppressor genes) and non-random genomic copy number changes (gain in copies of oncogenes and loss in copies of tumor suppressor genes) can potentially explain why the Warburg phenomenon is a convergent phenotype in cancers resulting from diverse oncogenic events.

### Clinical Significance of the glycolytic metabolism in cancer cells

#### Cancer Imaging. The most significant clinical exploitation of the Warburg phenomenon is the development of ${}^{18}$F-fluorodeoxyglucose positron emission tomography (${}^{18}$F-FDG-PET). The avidity in glucose uptake and metabolism by cancer cells has made ${}^{18}$F-FDG-PET a very useful tool in the diagnosis, staging and prognosis of cancer. The retention index obtained from dual-phase ${}^{18}$F-FDG-PET correlates well with HK2 expression while the specific uptake value (SUV) at 1 hour correlates with GLUT-1 expression in pancreatic cancer [103]. In another study, the SUV in FDG-PET scan is demonstrated to be a function of microvasculature, GLUT-1, HK2, number of tumor cells/volume, proliferation rate, and HIF-1$\alpha$ [104]. Since SUV reflects these underlying biomarkers and biological processes associated with aggressive cancers, it is not surprising that SUV has been demonstrated to have prognostic value [105, 106].

#### Expression of Genes/Proteins Involved in Energy Metabolism as Predictive Biomarkers of Prognosis and Response to a Specific Therapy. Increase in proteins involved in glycolysis (e.g., HK2) [107] and decrease in proteins involved in oxidative phosphorylation (e.g., $\beta$-F1-ATPase) [42, 108] are biomarkers that can predict prognosis. Combined analysis of protein and mRNA data revealed 11 components of

the glycolysis pathway as associated with poor survival of patients with lung adenocarcinoma [109]. Transcriptomic profiling has identified subtypes of cancer based on consistent patterns of gene expression, leading to improved prognostic predictions. To match specific therapies to molecular targets for cancer therapy, 3-bromopyruvic acid (3-BrPA, a glycolysis inhibitor) selectively killed breast cancer cells expressing the mitochondria and wound signatures [110]. Bortezomib (a proteasome inhibitor) abrogates wound signature expression and selectively killed breast cells expressing the wound signature [110].

#### Interaction with Comorbidity. Diabetes mellitus type 2 (DM2) is associated with an elevated risk of pancreatic, liver, colon, gastric, breast, and endometrial cancer [111–116]. Extensive epidemiologic data suggest important roles of diabetes in carcinogenesis [111–116] and cancer survival [117]. The strongest association is perhaps with pancreatic cancer [118–122], and up to about 80% of pancreatic cancer patients have overt diabetes or impaired glucose tolerance [123]. Three major mechanisms have been postulated to explain the possible promoting impact of DM2 on cancer: hyperglycemia, activation of the insulin signaling pathway, activation of the insulin-like-growth-factor signaling pathway. Hyperglycemia per se may increase delivery of glucose to cancer cells for consumption, and may confer a growth promoting effect as well as resistance to therapy by chemotherapy or radiation. Both growth promotion and chemoresistance induced by glucose concentrations up to 400 mg/dL can be observed in cultures of pancreatic cancer cell lines and leukemia cell lines (our unpublished data). Insulin and insulin-like growth factor-1 (IGF-1) induced transcription of their target genes through activation of HIF-1 transcription activity [124], and the effect of insulin/IGF-1 on HIF-1 is mediated through the PI3K/AKT/mTOR pathway [125]. Hyperinsulinemia and high circulating levels of IGF-1 in DM2 (except late in the natural history of DM2 when pancreatic $\beta$-cell function has declined) can promote the glycolytic phenotype in premalignant and malignant cells.

#### Cancer Therapy. Just like the other 6 signs of malignancy, the Warburg phenomenon or cancer-specific bioenergetics can also be exploited for development of chemotherapy to benefit the patients:

#### mTOR Inhibitors. Since the AKT/mTOR pathway regulates the genes responsible for aerobic glycolysis, disruption of signaling through this pathway may switch the source of metabolic energy from glycolysis

Cell. Mol. Life Sci. Vol. 65, 2008 Review Article 3991

**Figure 5. Comparative Genomic Hybridization Revealed Non-random Gains and Losses of Genomic DNA in Breast Cancer and Lung Cancer.** The original data for chromosome 1 in breast cancer [98] is shown in (A). Regions colored in red denote areas of recurrent loss, with the minimum value of -100% representing loss of that area in all samples; similarly, regions colored in green denote areas of recurrent gain, with the maximum value of 100%. The original data for chromosome 1 in lung cancer [99] is shown in (B). Regions colored in green denote areas of recurrent loss, with the maximum value of -1 representing loss of that area in all samples; similarly, regions colored in red denote areas of recurrent gain. Whole genome frequency plots of breast cancer and lung cancer samples are merged (C). The black color shows the areas of overlap of the two whole genome frequency plots. As labeled, the location of tumor suppressor genes (blue) and oncogenes (cyan) relevant to the signaling pathways regulating glycolysis are indicated. Many tumor suppressor genes are located in chromosomal regions that lose DNA copies while many oncogenes are located in chromosomal regions that gain DNA copies. MYC and SRC are located in chromosomal regions that gain copies in about 70% of cases of breast and lung cancers. MYCL1 (MYC-related gene from lung cancer), AKT1 (protein kinase B, alpha), AKT2 (protein kinase B, beta), AKT3 (protein kinase B, gamma), IRS2 (insulin receptor substrate 2), HER2 (human EGF receptor type 2), HRAS (Harvey murine sarcoma virus oncogene) and HIF1B (hypoxia-inducible factor 1, beta subunit) are located in chromosomal regions that gain copies in about 50% of cases. FBXW7 (F-Box WD40 domain protein 7), CTNNB1 (β-catenin) and p14ARF (alternative reading frame) are located in chromosomal regions that lose copies in about 50% of cases.

to oxidative phosphorylation, and remove the survival advantage of cancer cells in the hypoxic, poorly perfused tumor microenvironment. Therefore, one method to impact cancer cell energy metabolism is to target mTOR. mTOR inhibitors (e.g., RAD001,

temsirolimus (CCI-779), AP-23573) are in phase I/II trials with promising results [126–129].

**HIF-1α Inhibitors.** mTOR is upstream to HIF-1, and mTOR inhibitors are expected to suppress HIF-1α

expression as rapamycin has been shown to do so [130]. Since HIF-1 is such an important drug target, small molecule inhibitors of HIF-1 have been identified by screening (topotecan, NSC 644221, YC-1 and PX-478). Topotecan, a topoisomerase I poison, has been found to inhibit HIF-1 transcriptional activity and HIF-1α protein accumulation in hypoxia-treated glioma cell, and the mechanism is dependent on topoisomerase I but not DNA damage [131]. NSC 644221 inhibited HIF-1-dependent transcription, and topoisomerase II is required for this inhibition [132]. YC-1 inhibits HIF-1-mediated erythropoietin production and angiogenesis [133]. The amino acid 720–780 region of HIF-1α is required for YC-1-induced degradation [134]. PX-478 decreases HIF-1α mRNA, protein level and transcription activity in cancer cell lines, but the primary inhibitory mechanism of PX-478 may be inhibition of translation [135]. PX-478 also has potent antineoplastic activity against human cancer xenografts [136].

**Glycolysis Inhibitors.** The increased dependence of cancer cells on glycolytic generation of ATP provides the basis for preferential killing of cancer cells by pharmacological inhibition of glycolysis. Several glycolysis inhibitors have exhibited antitumor effects. Lonidamine, a derivative of indazole-3-carboxylic acid, alters mitochondrial glycerol 3-phosphate and malate respiration and leads to a release of the bound hexokinase from mitochondria [137], and also inhibits electron transport in tumor mitochondria at the dehydrogenase-coenzyme level [138]. It exerts a powerful inhibitory effect on oxygen consumption, aerobic glycolysis and lactate transport in cancer cells [139]. Although phase II-III trials for the treatment of advanced breast, ovarian and lung cancer are encouraging and lonidamine can modulate responses to anthracycline and platinum compounds, its role in cancer chemotherapy remains to be established [140]. 3-BrPA is a hexokinase inhibitor [141, 142]. 3-BrPA effectively kills colon cancer cells and lymphoma cells in a hypoxic environment in which the cancer cells exhibit high glycolytic activity and decreased sensitivity to common anticancer agents [143]. 2-deoxy-D-glucose inhibits glycolytic enzymes and has also produced encouraging antineoplastic results in vitro and in vivo [141, 143–145] as well as sensitizing cancer cells to radiation [146, 147] particularly in cancer cells with high rates of glycolysis [148].

**Glutaminolysis Inhibitors.** As discussed above, glutaminolysis is an alternate metabolic pathway for generate energy and building blocks for proliferation in cancer cells. Several compounds that inhibit glutaminolysis have been evaluated for cancer therapy.

Phenylacetate inhibits glutaminolysis because it is readily condensed with the γ-amino group of glutamine, thereby inhibiting glutamine consumption, and it has antineoplastic activity against glioma and prostate cancer in animal models [149, 150]. Amino-oxyacetate is an inhibitor of glutamate oxaloacetate transaminase [151], and it inhibits the effect of glutamine and asparagine on glucose metabolism has been studied in ascites tumor cells. 6-Diazoy-5-oxo-L-norleucine (a glutamine analogue) have been shown to possess cytotoxic activity against a wide variety of animal and human xenografted solid tumors [152], as well as in combination with other metabolic inhibitors such as 2-deoxy-D-glucose may be explored [152].

**NAD Analogues.** Another therapeutic approach is to inhibit both glycolysis and glutaminolysis by interfering with NAD metabolism. This can be achieved either by use of AMP analogs such as 4-methoxy- and 4-amino-8-(3-D-ribofuranosylamino)-pyrimidi-[5,4-d]pyrimidine, which inhibit NAD synthesis or by 6-aminonicotinamide (6-AN) which is incorporated into NAD and NADP, forming 6-amino-NAD and 6-amino-NADP [153]. The accumulating 6-amino-NADP preferentially inhibits 6-phosphogluconate dehydrogenase. Thus, 6-phosphogluconate accumulates, which inhibits glucose 6-phosphate-isomerase and glycolysis [153]. Indeed, this therapeutic strategy appears to be promising in animal models [139, 154–156].

**ATP and Pyrimidine Depletors.** 6-methylmercaptopurine riboside (MMPR), known to inhibit *de novo* purine biosynthesis and thereby limit adenine supplies for ATP production. In high dosage, MMPR also decreases pyrimidine ribonucleotide concentrations. *De novo* pyrimidine synthesis inhibitor, N-(phosphonacetyl)-L-aspartate (PALA). A phase II multi-institutional trial evaluated the efficacy and toxicity of 5-FU in combination with PALA and leucovorin in patients with advanced pancreatic cancer [157], and found that the response rate was similar to other single agents in pancreatic cancer and resulted in some long term survival while having relatively mild toxicity. A triple-drug combination of MMPR, PALA and 6-AN (see the section on NAD analogues above) has been designed to deplete cellular energy in tumor cells [154]. Combining this triple combination with doxorubicin increases antineoplastic activity over that produced by either doxorubicin alone or the triple-drug combination [156]. The same triple combination plus 5-fluorouracil also exhibited antineoplastic activity against breast cancers in murine models [154].

**AMP Analogue for Pharmacologic Mimicry of Low Energy State.** 5-aminoimidazole-4-carboxamide ribo-

Cell. Mol. Life Sci. Vol. 65, 2008

Figure 6. Genetic Changes in Cancer Lead to Coordinated Changes in Specific Regulators of Metabolism to Manifest the Cancer Phenotype. Mutations and epigenetic changes lead to changes in the function of oncogenes and tumor suppressor genes. Genomic instability causes further changes that upset the balance of oncogenes and tumor suppressor genes as the carcinogenic process progresses. These events very often lead to changes in the function of 3 transcription factors: activation of HIF-1 and MYC and loss of p53 (TP53) function. The changes in these transcription factors cause a coordinated change in the enzymes, transporters, regulators and metabolites as well as changes in mitochondrial function to bring about a characteristic metabolic signature of cancer cells. These metabolic changes provide growth and survival advantages for the cancer cells in the tumor microenvironment. FA – fatty acid; TCA – tricarboxylic acid cycle; PPS – pentose phosphate shunt pathway.

Oncogenes $\bullet\bullet\bullet\bullet\bullet$ Tumor Suppressor Genes

Genetic  
Mutation, translocation, amplification, deletion  

Epigenetic  
DNA methylation, microRNA, alternative splicing  

Genomic instability  

HIF-1  
MYC TP53  

Enzymes Transporters Metabolites Mitochondrial Changes  

Glucose → Glycolysis → PPS → Purines  
Lactate ← ↑ Pyrimidines  

TCA  
Glutamine  

FA synthesis  
Fatty Acid  

↓  

Growth & Survival Advantages  

side (AICAR) is a cell-permeable nucleoside that is metabolized to increase the intracellular levels of AICA ribotide, an AMP analogue, to mimic a low energy state of the cell. AICAR inhibits lipogenesis, protein translation, and DNA synthesis in cancer cells, and it exhibited antineoplastic activity both in cell culture and in a nude mouse xenograft model [158]. A multitude of pharmacologic tools are becoming available to interfere with cancer energy metabolism at various vulnerable points. It is conceivable that each tumor reacts individually to the various drugs that interfere with energy metabolism. Clinicians are interested to know which patients will most benefit from which therapy. Therapy can be improved if we learn how expression of these metabolic enzymes modulates the response to different drugs by examining the transcriptome and metabolome. As discussed above, transcriptomic profiling can identify subtypes of cancer to match specific therapies to molecular targets for cancer therapy: e.g., 3-BrPA for breast cancer cells expressing the mitochondria and wound signatures [110]. The future may hold promise for cancer chemotherapy guided by the transcriptome and metabolome.

Concluding Remarks

A large body of evidence has accumulated and demonstrated that cancer cells display an increased glycolytic metabolism compared with normal cells. Rapid uninhibited cellular division is a hallmark of

cancer, and the shift from oxidative phosphorylation  
to glycolytic metabolism is not just an adaption  
subservient to rapid cell proliferation, and comparison  
between cancerous and normal breast epithelial cells  
growing at the same fast rates reveals differences in  
metabolism that support the “microenvironment  
evolution” model of cancer progression [14]. The  
current routine use of ${}^{18}$F-FDG-PET scan in clinical  
oncology unequivocally solidifies the position of avid  
glycolytic metabolism as a hallmark of cancer.  
This shift from oxidative phosphorylation to glycolysis  
may not be a cause of malignant transformation as  
Warburg hypothesized but an epiphenomenon of  
transformation associated with a higher metastatic  
potential and survival advantage. A prevailing theory  
of carcinogenesis is the somatic mutation theory  
which postulates that cancer begins with a single  
mutation in a somatic cell followed by successive  
mutations, but it has many unresolved paradoxes  
[159]. Carcinogenesis has been viewed as an evolu-  
tionary process at the cellular level [160], and a recent  
interesting model of carcinogenesis integrated neo-  
Darwinian evolution with cell-environment interac-  
tions [161]. The basic principle in neo-Darwinism is  
that phenotypic properties are retained or lost based  
on their contribution to fitness for survival. This  
theory was applied most convincingly to explain the  
Warburg phenomenon by Gatenby and Gillies [161].  
As cancer progression proceeds, the mutations in  
tumor cells increase, and traits that are found in  
invasive cancers (including the stereotypic metabolic  
platform of cancers) arise as adaptive mechanisms to  
environmental proliferative constraints.  
The alteration in glycolytic metabolism is regulated by  
oncogenes and tumor suppressor genes which control  
enzymes, scaffolding protein and transporters that  
coordinate the shift of energy metabolism from  
oxidative phosphorylation. The regulation of glycol-  
ysis and cancer metabolism by HIF-1 and c-MYC has  
been established by a large body of work contributed  
by many researchers over the past 15 years or so. A  
recent review summarized the roles of the (PI3K)/  
Akt/mTOR signaling pathway, HIF-1, and c-MYC  
[13]. An explosion of publications has firmly estab-  
lished the role of loss of p53 function in mediating this  
glycolytic phenotype. This review has integrated the  
role of loss of p53 function and focused on changes in  
the functions of this triad of transcription factors (c-  
MYC, HIF-1 and p53) and their interplay as the  
crucial molecular mechanisms underlying the cancer  
metabolic phenotype (Fig. 6).  
A mechanistic understanding may lead to novel  
therapeutics, and conversely, development of small  
molecule-approach to disturb energy metabolism  
pathway might provide new insight for cancer biology.


Clinical trials with the potential antineoplastic agents  
discussed above that interfere with the Warburg  
phenomenon or cancer-specific bioenergetics (e.g.,  
mTOR inhibitors, HIF-1$\alpha$ inhibitors, glycolysis inhib-  
itors, glutaminolysis inhibitors, and AMP analogues)  
may add new weapons against cancer. Correlative  
studies examining the impact of these agents on cancer  
metabolism may validate cancer-specific bioenerget-  
ics as a target for antineoplastic therapy. With the  
rapid technologic advances, the future may hold  
promise for custom designed antineoplastic regimens  
based on individual patient’s cancer transcriptome  
and metabolome.

Acknowledgments. This work was supported in part by the  
NIHRO1CA (089266 Lee M H), U.S. Department of Defense  
Breast Cancer Research Program of the Office of the Congress-  
ionally Directed Medical Research Programs (DOD SIDA  
BC062166 Yeung SJ & Lee MH) and Cancer Center Core  
Grant (CA16672).

1. Hanahan D. and Weinberg R. A. (2000) The hallmarks of  
   cancer. Cell 100, 57–70.
2. Lu Y., Yi Y., Liu P., Wen W., James M., Wang D. and You M.  
   (2007) Common human cancer genes discovered by integrat-  
   ed gene-expression analysis. PLoS ONE 2, e1149.
3. Garber K. (2006) Energy deregulation: licensing tumors to  
   grow. Science 312, 1158–1159.
4. Warburg O. (1930) On metabolism of tumors. London:  
   Constable.
5. Warburg O. (1956) On respiratory impairment in cancer cells.  
   Science 124, 269–270.
6. Warburg O. (1956) On the origin of cancer cells. Science 123,  
   309–314.
7. Boguski M. S., Lowe T. M. and Tolstoshev C. M. (1993)  
   dbEST-database for "expressed sequence tags". Nat. Genet.  
   4, 332–333.
8. Altenberg B. and Greulich K. O. (2004) Genes of glycolysis  
   are ubiquitously overexpressed in 24 cancer classes. Genomics  
   84, 1014–1020.
9. Eskey C. J., Koretsky A. P., Domach M. M. and Jain R. K.  
   (1993) Role of oxygen vs. glucose in energy metabolism in a  
   mammary carcinoma perfused ex vivo: direct measurement  
   by 31P NMR. Proc. Natl. Acad. Sci. USA 90, 2646–2650.
10. Reitzer L. J., Wice B. M. and Kennell D. (1979) Evidence that  
    glutamine, not sugar, is the major energy source for cultured  
    HeLa cells. J. Biol. Chem. 254, 2669–2676.
11. Mazurek S., Michel A. and Eigenbrodt E. (1997) Effect of  
    extracellular AMP on cell proliferation and metabolism of  
    breast cancer cell lines with high and low glycolytic rates. J.  
    Biol. Chem. 272, 4941–4952.
12. Eigenbrodt E., Reinacher M., Scheefers-Borchel U., Schee-  
    fers H. and Friis R. (1992) Double role for pyruvate kinase  
    type M2 in the expansion of phosphometabolite pools found  
    in tumor cells. Crit. Rev. Oncog. 3, 91–115.
13. DeBerardinis R. J., Lum J. J., Hatzivassiliou G. and Thomp-  
    son C. B. (2008) The biology of cancer: metabolic reprogram-  
    ming fuels cell growth and proliferation. Cell Metab. 7, 11–20.
14. Meadows A. L., Kong B., Berdichevsky M., Roy S., Rosiva  
    R., Blanch H. W. and Clark D. S. (2008) Metabolic and  
    Morphological Differences between Rapidly Proliferating  
    Cancerous and Normal Breast Epithelial Cells. Biotechnol.  
    Prog. 24, 334–341.
15. Kuhajda F. P. (2006) Fatty acid synthase and cancer: new  
    application of an old pathway. Cancer Res. 66, 5977–5980.

Cell. Mol. Life Sci. Vol. 65, 2008

16 Furuta E., Pai S. K., Zhan R., Bandyopadhyay S., Watabe M., Mo Y. Y., Hirota S., Hosobe S., Tsukada T., Miura K., Kamada S., Saito K., Iizumi M., Liu W., Ericsson J. and Watabe K. (2008) Fatty acid synthase gene is up-regulated by hypoxia via activation of Akt and sterol regulatory element binding protein-1. Cancer Res. 68, 1003–1011.

17 DeBerardinis R. J., Mancuso A., Daikhin E., Nissim I., Yudkoff M., Wehrli S. and Thompson C. B. (2007) Beyond aerobic glycolysis: transformed cells can engage in glutamine metabolism that exceeds the requirement for protein and nucleotide synthesis. Proc. Natl. Acad. Sci. USA 104, 19345–19350.

18 Newsholme E. A. and Board M. (1991) Application of metabolic-control logic to fuel utilization and its significance in tumor cells. Adv. Enzyme Regul. 31, 225–246.

19 Mazurek S., Eigenbrodt E., Failing K. and Steinberg P. (1999) Alterations in the glycolytic and glutaminolytic pathways after malignant transformation of rat liver oval cells. J. Cell. Physiol. 181, 136–146.

20 Unwin R. D., Craven R. A., Harnden P., Hanrahan S., Totty N., Knowles M., Eardley I., Selby P. J. and Banks R. E. (2003) Proteomic changes in renal cancer and co-ordinate demonstration of both the glycolytic and mitochondrial aspects of the Warburg effect. Proteomics 3, 1620–1632.

21 Zhang D., Tai L. K., Wong L. L., Chiu L. L., Sethi S. K. and Koay E. S. (2005) Proteomic study reveals that proteins involved in metabolic and detoxification pathways are highly expressed in HER-2/neu-positive breast cancer. Mol. Cell. Proteomics 4, 1686–1696.

22 Mikuriya K., Kuramitsu Y., Ryozawa S., Fujimoto M., Mori S., Oka M., Hamano K., Okita K., Sakaida I. and Nakamura K. (2007) Expression of glycolytic enzymes is increased in pancreatic cancerous tissues as evidenced by proteomic profiling by two-dimensional electrophoresis and liquid chromatography-mass spectrometry/mass spectrometry. Int. J. Oncol. 30, 849–855.

23 Mazurek S. and Eigenbrodt E. (2003) The tumor metabolome. Anticancer Res. 23, 1149–1154.

24 Medina M. A., Sanchez-Jimenez F., Marquez F. J., Perez-Rodriguez J., Quesada A. R. and Nunez de Castro I. (1988) Glutamine and glucose as energy substrates for Ehrlich ascites tumour cells. Biochem Int. 16, 339–347.

25 Eigenbrodt E., Kallinowski F., Ott M., Mazurek S. and Vaupel P. (1998) Pyruvate kinase and the interaction of amino acid and carbohydrate metabolism in solid tumors. Anticancer Res. 18, 3267–3274.

26 Mazurek S., Grimm H., Wilker S., Leib S. and Eigenbrodt E. (1998) Metabolic characteristics of different malignant cancer cell lines. Anticancer Res. 18, 3275–3282.

27 Mazurek S., BoscheK C. B. and Eigenbrodt E. (1997) The role of phosphometabolites in cell proliferation, energy metabolism, and tumor therapy. J. Bioenerg. Biomembr. 29, 315–330.

28 Iyanagi T. (2007) Molecular mechanism of phase I and phase II drug-metabolizing enzymes: implications for detoxification. Int. Rev. Cytol. 260, 35–112.

29 Ying W., Alano C. C., Garnier P. and Swanson R. A. (2005) NAD+ as a metabolic link between DNA damage and cell death. J. Neurosci. Res. 79, 216–223.

30 Hue L. and Rider M. H. (1987) Role of fructose 2,6-bisphosphate in the control of glycolysis in mammalian tissues. Biochem. J. 245, 313–324.

31 Mazurek S., Grimm H., BoscheK C. B., Vaupel P. and Eigenbrodt E. (2002) Pyruvate kinase type M2, a crossroad in the tumor metabolome. Br. J. Nutr. 87 Suppl 1, S23–29.

32 Mazurek S., BoscheK C. B., Hugo F. and Eigenbrodt E. (2005) Pyruvate kinase type M2 and its role in tumor growth and spreading. Semin. Cancer Biol. 15, 300–308.

33 Okar D. A., Manzano A., Navarro-Sabate A., Riera L., Bartrons R. and Lange A. J. (2001) PFK-2/FBPase-2, maker and breaker of the essential biofactor fructose-2,6-bisphosphate. Trends Biochem. Sci. 26, 30–35.

34 Carew J. S. and Huang P. (2002) Mitochondrial defects in cancer. Mol. Cancer 1, 9.

35 Mambo E., Chatterjee A., Xing M., Tallini G., Haugen B. R., Yeung S. C., Sukumar S. and Sidransky D. (2005) Tumor-specific changes in mtDNA content in human cancer. Int. J. Cancer 116, 920–924.

36 Brandon M., Baldi P. and Wallace D. C. (2006) Mitochondrial mutations in cancer. Oncogene 25, 4647–4662.

37 Chatterjee A., Mambo E. and Sidransky D. (2006) Mitochondrial DNA mutations in human cancer. Oncogene 25, 4663–4674.

38 Pelicano H., Xu R. H., Du M., Feng L., Sasaki R., Carew J. S., Hu Y., Ramdas L., Hu L., Keating M. J., Zhang W., Plunkett W. and Huang P. (2006) Mitochondrial respiration defects in cancer cells cause activation of Akt survival pathway through a redox-mediated mechanism. J. Cell Biol. 175, 913–923.

39 King A., Selak M. A. and Gottlieb E. (2006) Succinate dehydrogenase and fumarate hydratase: linking mitochondrial dysfunction and cancer. Oncogene 25, 4675–4682.

40 Matoba S., Kang J. G., Patino W. D., Wragg A., Boehm M., Gavrilova O., Hurley P. J., Bunz F. and Hwang P.M. (2006) p53 regulates mitochondrial respiration. Science 312, 1650–1653.

41 Capuano F., Guerrieri F. and Papa S. (1997) Oxidative phosphorylation enzymes in normal and neoplastic cell growth. J. Bioenerg. Biomembr. 29, 379–384.

42 Lopez-Rios F., Sanchez-Arago M., Garcia-Garcia E., Ortega A. D., Berrendero J. R., Pozo-Rodriguez F., Lopez-Encuentra A., Ballestin C. and Cuezva J. M. (2007) Loss of the mitochondrial bioenergetic capacity underlies the glucose avidity of carcinomas. Cancer Res. 67, 9013–9017.

43 Cavalli L. R., Varella-Garcia M. and Liang B. C. (1997) Diminished tumorigenic phenotype after depletion of mitochondrial DNA. Cell Growth Differ. 8, 1189–1198.

44 Secomb T. W., Hsu R., Dewhirst M. W., Klitzman B. and Gross J. F. (1993) Analysis of oxygen transport to tumor tissue by microvascular networks. Int. J. Radiat. Oncol. Biol. Phys. 25, 481–489.

45 Heldin C. H., Rubin K., Pietras K. and Ostman A. (2004) High interstitial fluid pressure—an obstacle in cancer therapy. Nat. Rev. Cancer 4, 806–813.

46 Vaupel P., Fortmeyer H. P., Runkel S. and Kallinowski F. (1987) Blood flow, oxygen consumption, and tissue oxygenation of human breast cancer xenografts in nude rats. Cancer Res. 47, 3496–3503.

47 Minchenko O., Opentanova I. and Caro J. (2003) Hypoxic regulation of the 6-phosphofructo-2-kinase/fructose-2,6-bisphosphatase gene family (PFKFB-1-4) expression in vivo. FEBS Lett. 554, 264–270.

48 Minchenko O. H., Ogura T., Opentanova I. L., Minchenko D. O. and Esumi H. (2005) Splice isoform of 6-phosphofructo-2-kinase/fructose-2,6-bisphosphatase-4, expression and hypoxic regulation. Mol. Cell. Biochem. 280, 227–234.

49 Acker T. and Plate K. H. (2002) A role for hypoxia and hypoxia-inducible transcription factors in tumor physiology. J. Mol. Med. 80, 562–575.

50 Semenza G. L. (2000) Hypoxia, clonal selection, and the role of HIF-1 in tumor progression. Crit. Rev. Biochem. Mol. Biol. 35, 71–103.

51 Bensaad K., Tsuruta A., Selak M. A., Vidal M. N., Nakano K., Bartrons R., Gottlieb E. and Vousden K. H. (2006) TIGAR, a p53-inducible regulator of glycolysis and apoptosis. Cell 126, 107–120.

52 Green D. R. and Chipuk J. E. (2006) p53 and metabolism: Inside the TIGAR. Cell 126, 30–32.

53 Shim H., Dolde C., Lewis B. C., Wu C. S., Dang G., Jungmann R. A., Dalla-Favera R. and Dang C. V. (1997) c-Myc transactivation of LDH-A: implications for tumor metabolism and growth. Proc. Natl. Acad. Sci. USA 94, 6658–6663.

54 Fantin V. R., St-Pierre J. and Leder P. (2006) Attenuation of LDH-A expression uncovers a link between glycolysis, mitochondrial physiology, and tumor maintenance. Cancer Cell 9, 425–434.

55 Cardone R. A., Casavola V. and Reshkin S. J. (2005) The role of disturbed pH dynamics and the Na+/H+ exchanger in metastasis. Nat. Rev. Cancer 5, 786–795.

56 Opavsky R., Pastorekova S., Zelnik V., Gibadulina A., Stanbridge E. J., Zavada J., Kettmann R. and Pastorek J. (1996) Human MN/CA9 gene, a novel member of the carbonic anhydrase family: structure and exon to protein domain relationships. Genomics 33, 480–487.

57 Ivanov S., Liao S. Y., Ivanova A., Danilkovitch-Miagkova A., Tarasova N., Weirich G., Merrill M. J., Proescholdt M. A., Oldfield E. H., Lee J., Zavada J., Waheed A., Sly W., Lerman M. I. and Stanbridge E. J. (2001) Expression of hypoxia-inducible cell-surface transmembrane carbonic anhydrases in human cancer. Am. J. Pathol. 158, 905–919.

58 Robertson N., Potter C. and Harris A. L. (2004) Role of carbonic anhydrase IX in human tumor cell growth, survival, and invasion. Cancer Res. 64, 6160–6165.

59 Elstrom R. L., Bauer D. E., Buzzai M., Karnauskas R., Harris M. H., Plas D. R., Zhuang H., Cinalli R. M., Alavi A., Rudin C. M. and Thompson C. B. (2004) Akt stimulates aerobic glycolysis in cancer cells. Cancer Res. 64, 3892–3899.

60 Barthel A., Okino S. T., Liao J., Nakatani K., Li J., Whitlock J. P., Jr. and Roth R. A. (1999) Regulation of GLUT1 gene transcription by the serine/threonine kinase Akt1. J. Biol. Chem. 274, 20281–20286.

61 Taha C., Liu Z., Jin J., Al-Hasani H., Sonenberg N. and Klip A. (1999) Opposite translational control of GLUT1 and GLUT4 glucose transporter mRNAs in response to insulin. Role of mammalian target of rapamycin, protein kinase b, and phosphatidylinositol 3-kinase in GLUT1 mRNA translation. J. Biol. Chem. 274, 33085–33091.

62 Majewski N., Nogueira V., Bhaskar P., Coy P. E., Skeen J. E., Gottlob K., Chandel N. S., Thompson C. B., Robey R. B. and Hay N. (2004) Hexokinase-mitochondria interaction mediated by Akt is required to inhibit apoptosis in the presence or absence of Bax and Bak. Mol. Cell 16, 819–830.

63 Majewski N., Nogueira V., Robey R. B. and Hay N. (2004) Akt inhibits apoptosis downstream of BID cleavage via a glucose-dependent mechanism involving mitochondrial hexokinases. Mol. Cell. Biol. 24, 730–740.

64 Bauer D. E., Hatzivassiliou G., Zhao F., Andreadis C. and Thompson C. B. (2005) ATP citrate lyase is an important component of cell growth and transformation. Oncogene 24, 6314–6322.

65 Deberardinis R. J., Lum J. J. and Thompson C. B. (2006) Phosphatidylinositol 3-kinase-dependent modulation of carnitine palmitoyltransferase 1A expression regulates lipid metabolism during hematopoietic cell growth. J. Biol. Chem. 281, 37372–37380.

66 Martin D. E. and Hall M. N. (2005) The expanding TOR signaling network. Curr. Opin. Cell Biol. 17, 158–166.

67 Albanell J., Dalmases A., Rovira A. and Rojo F. (2007) mTOR signalling in human cancer. Clin. Transl. Oncol. 9, 484–493.

68 Chiang G. G. and Abraham R. T. (2007) Targeting the mTOR signaling network in cancer. Trends Mol. Med. 13, 433–442.

69 Hudson C. C., Liu M., Chiang G. G., Otterness D. M., Loomis D. C., Kaper F., Giaccia A. J. and Abraham R. T. (2002) Regulation of hypoxia-inducible factor 1alpha expression and function by the mammalian target of rapamycin. Mol. Cell. Biol. 22, 7004–7014.

70 Semenza G. L., Roth P. H., Fang H. M. and Wang G. L. (1994) Transcriptional regulation of genes encoding glycolytic enzymes by hypoxia-inducible factor 1. J. Biol. Chem. 269, 23757–23763.

71 Laderoute K. R., Amin K., Calaoagan J. M., Knapp M., Le T., Orduna J., Foretz M. and Viollet B. (2006) 5′-AMP-activated protein kinase (AMPK) is induced by low-oxygen and glucose deprivation conditions found in solid-tumor microenvironments. Mol. Cell. Biol. 26, 5336–5347.

72 Kwon S. J. and Lee Y. J. (2005) Effect of low glutamine/glucose on hypoxia-induced elevation of hypoxia-inducible

factor-1alpha in human pancreatic cancer MiaPaCa-2 and human prostatic cancer DU-145 cells. Clin. Cancer Res. 11, 4694–4700.

73 Mathupala S. P., Rempel A. and Pedersen P. L. (1997) Aberrant glycolytic metabolism of cancer cells: a remarkable coordination of genetic, transcriptional, post-translational, and mutational events that lead to a critical role for type II hexokinase. J. Bioenerg. Biomembr. 29, 339–343.

74 Dang C. V., Lewis B. C., Dolde C., Dang G. and Shim H. (1997) Oncogenes in tumor metabolism, tumorigenesis, and apoptosis. J. Bioenerg. Biomembr. 29, 345–354.

75 Dang C. V. and Semenza G. L. (1999) Oncogenic alterations of metabolism. Trends Biochem. Sci. 24, 68–72.

76 Lu H., Forbes R. A. and Verma A. (2002) Hypoxia-inducible factor 1 activation by aerobic glycolysis implicates the Warburg effect in carcinogenesis. J. Biol. Chem. 277, 23111–23115.

77 Kim J. W., Gao P., Liu Y. C., Semenza G. L. and Dang C. V. (2007) Hypoxia-inducible factor 1 and dysregulated c-Myc cooperatively induce vascular endothelial growth factor and metabolic switches hexokinase 2 and pyruvate dehydrogenase kinase 1. Mol. Cell. Biol. 27, 7381–7393.

78 Robey I. F., Lien A. D., Welsh S. J., Baggett B. K. and Gillies R. J. (2005) Hypoxia-inducible factor-1alpha and the glycolytic phenotype in tumors. Neoplasia 7, 324–330.

79 Schwartzentruber-Bar-Yoseph F., Armoni M. and Karnieli E. (2004) The tumor suppressor p53 down-regulates glucose transporters GLUT1 and GLUT4 gene expression. Cancer Res. 64, 2627–2633.

80 Kawauchi K., Araki K., Tobiume K. and Tanaka N. (2008) p53 regulates glucose metabolism through an IKK-NF-kappaB pathway and inhibits cell transformation. Nat. Cell Biol. 10, 611–618.

81 Kondoh H., Leonart M. E., Gil J., Wang J., Degan P., Peters G., Martinez D., Carnero A. and Beach D. (2005) Glycolytic enzymes can modulate cellular life span. Cancer Res. 65, 177–185.

82 Huang L. E. (2008) Carrot and stick: HIF-alpha engages c-Myc in hypoxic adaptation. Cell Death Differ. 15, 672–677.

83 Kaidi A., Williams A. C. and Paraskeva C. (2007) Interaction between beta-catenin and HIF-1 promotes cellular adaptation to hypoxia. Nat. Cell Biol. 9, 210–217.

84 Dang C. V., Kim J. W., Gao P. and Yustein J. (2008) The interplay between MYC and HIF in cancer. Nat. Rev. Cancer 8, 51–56.

85 Gordon J. D., Thompson C. B. and Simon M. C. (2007) HIF and c-Myc: sibling rivals for control of cancer cell metabolism and proliferation. Cancer Cell 12, 108–113.

86 An W. G., Kanekal M., Simon M. C., Maltepe E., Blagosklonny M. V. and Neckers L. M. (1998) Stabilization of wild-type p53 by hypoxia-inducible factor 1alpha. Nature 392, 405–408.

87 Hansson L. O., Friedler A., Freund S., Rudiger S. and Fersht A. R. (2002) Two sequence motifs from HIF-1alpha bind to the DNA-binding site of p53. Proc. Natl. Acad. Sci. USA 99, 10305–10309.

88 Schmid T., Zhou J., Kohl R. and Brune B. (2004) p300 relieves p53-evoked transcriptional repression of hypoxia-inducible factor-1 (HIF-1). Biochem. J. 380, 289–295.

89 Fels D. R. and Koumenis C. (2005) HIF-1alpha and p53, the ODD couple? Trends Biochem. Sci. 30, 426–429.

90 Nieminen A. L., Qanungo S., Schneider E. A., Jiang B. H. and Agani F. H. (2005) Mdm2 and HIF-1alpha interaction in tumor cells during hypoxia. J. Cell. Physiol. 204, 364–369.

91 Dai M. S., Jin Y., Gallegos J. R. and Lu H. (2006) Balance of Yin and Yang: ubiquitylation-mediated regulation of p53 and c-Myc. Neoplasia 8, 630–644.

92 Hervouet E., Demont J., Pecina P., Vojtiskova A., Houstek J., Simonnet H. and Godinot C. (2005) A new role for the von Hippel-Lindau tumor suppressor protein: stimulation of mitochondrial oxidative phosphorylation complex biogenesis. Carcinogenesis 26, 531–539.

Cell. Mol. Life Sci. Vol. 65, 2008

93 Kroemer G. (2006) Mitochondria in cancer. Oncogene 25, 4630–4632.

94 Robey R. B. and Hay N. (2006) Mitochondrial hexokinases, novel mediators of the antiapoptotic effects of growth factors and Akt. Oncogene 25, 4683–4696.

95 Kim J. W. and Dang C. V. (2005) Multifaceted roles of glycolytic enzymes. Trends Biochem. Sci. 30, 142–150.

96 Zheng L., Roeder R. G. and Luo Y. (2003) S phase activation of the histone H2B promoter by OCA-S, a coactivator complex that contains GAPDH as a key component. Cell 114, 255–266.

97 Futreal P. A., Coin L., Marshall M., Down T., Hubbard T., Wooster R., Rahman N. and Stratton M. R. (2004) A census of human cancer genes. Nat. Rev. Cancer 4, 177–183.

98 Naylor T. L., Greshock J., Wang Y., Colligon T., Yu Q. C., Clemmer V., Zaks T. Z. and Weber B. L. (2005) High resolution genomic analysis of sporadic breast cancer using array-based comparative genomic hybridization. Breast Cancer Res. 7, R1186–1198.

99 Chari R., Lockwood W. W., Coe B. P., Chu A., Macey D., Thomson A., Davies J. J., MacAulay C. and Lam W. L. (2006) SIGMA: a system for integrative genomic microarray analysis of cancer genomes. BMC Genomics 7, 324.

100 Sjoblom T., Jones S., Wood L. D., Parsons D. W., Lin J., Barber T. D., Mandelker D., Leary R. J., Ptak J., Silliman N., Szabo S., Buckhaults P., Farrell C., Meeh P., Markowitz S. D., Willis J., Dawson D., Willson J. K., Gazdar A. F., Hartigan J., Wu L., Liu C., Parmigiani G., Park B. H., Bachman K. E., Papadopoulos N., Vogelstein B., Kinzler K. W. and Velculescu V. E. (2006) The consensus coding sequences of human breast and colorectal cancers. Science 314, 268–274.

101 Rubin A. F. and Green P. (2007) Comment on "The consensus coding sequences of human breast and colorectal cancers". Science 317, 1500.

102 Forrest W. F. and Cavet G. (2007) Comment on "The consensus coding sequences of human breast and colorectal cancers". Science 317, 1500; author reply 1500.

103 Higashi T., Saga T., Nakamoto Y., Ishimori T., Mamede M. H., Wada M., Doi R., Hosotani R., Imamura M. and Konishi J. (2002) Relationship between retention index in dual-phase (18)F-FDG PET, and hexokinase-II and glucose transporter-1 expression in pancreatic cancer. J. Nucl. Med. 43, 173–180.

104 Bos R., van Der Hoeven J. J., van Der Wall E., van Der Groep P., van Diest P. J., Comans E. F., Joshi U., Semenza G. L., Hoekstra O. S., Lammertsma A. A. and Molthoff C. F. (2002) Biologic correlates of (18)fluorodeoxyglucose uptake in human breast cancer measured by positron emission tomography. J. Clin. Oncol. 20, 379–387.

105 van Westreenen H. L., Plukker J. T., Cobben D. C., Verhoogt C. J., Groen H. and Jager P. L. (2005) Prognostic value of the standardized uptake value in esophageal cancer. AJR Am. J. Roentgenol. 185, 436–440.

106 Guillem J. G., Moore H. G., Akhurst T., Klimstra D. S., Ruo L., Mazumdar M., Minsky B. D., Saltz L., Wong W. D. and Larson S. (2004) Sequential preoperative fluorodeoxyglucose-positron emission tomography assessment of response to preoperative chemoradiation: a means for determining long-term outcomes of rectal cancer. J. Am. Coll. Surg. 199, 1–7.

107 Rho M., Kim J., Jee C. D., Lee Y. M., Lee H. E., Kim M. A., Lee H. S. and Kim W. H. (2007) Expression of type 2 hexokinase and mitochondria-related genes in gastric carcinoma tissues and cell lines. Anticancer Res. 27, 251–258.

108 Isidoro A., Casado E., Redondo A., Acebo P., Espinosa E., Alonso A. M., Cejas P., Hardisson D., Fresno Vara J. A., Belda-Iniesta C., Gonzalez-Baron M. and Cuezva J. M. (2005) Breast carcinomas fulfill the Warburg hypothesis and provide metabolic markers of cancer prognosis. Carcinogenesis 26, 2095–2104.

109 Chen G., Gharib T. G., Wang H., Huang C. C., Kuick R., Thomas D. G., Shedden K. A., Misek D. E., Taylor J. M., Giordano T. J., Kardia S. L., Iannettoni M. D., Yee J., Hogg P. J., Orringer M. B., Hanash S. M. and Beer D. G. (2003) Protein profiles associated with survival in lung adenocarcinoma. Proc. Natl. Acad. Sci. USA 100, 13537–13542.

110 Wong D. J., Nuyten D. S., Regev A., Lin M., Adler A. S., Segal E., van de Vijver M. J. and Chang H. Y. (2008) Revealing targeted therapy for human cancer by gene module maps. Cancer Res. 68, 369–378.

111 Nilsen T. I. and Vatten L. J. (2001) Prospective study of colorectal cancer risk and physical activity, diabetes, blood glucose and BMI: exploring the hyperinsulinaemia hypothesis. Br. J. Cancer 84, 417–422.

112 Muti P., Quattrin T., Grant B. J., Krogh V., Micheli A., Schunemann H. J., Ram M., Freudenheim J. L., Sieri S., Trevisan M. and Berrino F. (2002) Fasting glucose is a risk factor for breast cancer: a prospective study. Cancer Epidemiol. Biomarkers Prev. 11, 1361–1368.

113 Verlato G., Zoppini G., Bonora E. and Muggeo M. (2003) Mortality from site-specific malignancies in type 2 diabetic patients from Verona. Diabetes Care 26, 1047–1051.

114 Richardson L. C. and Pollack L. A. (2005) Therapy insight: Influence of type 2 diabetes on the development, treatment and outcomes of cancer. Nat. Clin. Pract. Oncol. 2, 48–53.

115 Coughlin S. S., Calle E. E., Teras L. R., Petrelli J. and Thun M. J. (2004) Diabetes mellitus as a predictor of cancer mortality in a large cohort of US adults. Am. J. Epidemiol. 159, 1160–1167.

116 Yamagata H., Kiyohara Y., Nakamura S., Kubo M., Tanizaki Y., Matsumoto T., Tanaka K., Kato I., Shirota T. and Iida M. (2005) Impact of fasting plasma glucose levels on gastric cancer incidence in a general Japanese population: the Hisayama study. Diabetes Care 28, 789–794.

117 van de Poll-Franse L. V., Houterman S., Janssen-Heijnen M. L., Dercksen M. W., Coebergh J. W. and Haak H. R. (2007) Less aggressive treatment and worse overall survival in cancer patients with diabetes: A large population based analysis. Int. J. Cancer 120, 1986–1992.

118 Lu X. H., Wang L., Li H., Qian J. M., Deng R. X. and Zhou L. (2006) Establishment of risk model for pancreatic cancer in Chinese Han population. World J. Gastroenterol. 12, 2229–2234.

119 Kuriki K., Hirose K. and Tajima K. (2007) Diabetes and cancer risk for all and specific sites among Japanese men and women. Eur. J. Cancer Prev. 16, 83–89.

120 Fisher W. E. (2001) Diabetes: risk factor for the development of pancreatic cancer or manifestation of the disease? World J. Surg. 25, 503–508.

121 Stolzenberg-Solomon R. Z., Graubard B. I., Chari S., Limburg P., Taylor P. R., Virtamo J. and Albanes D. (2005) Insulin, glucose, insulin resistance, and pancreatic cancer in male smokers. JAMA 294, 2872–2878.

122 Everhart J. and Wright D. (1995) Diabetes mellitus as a risk factor for pancreatic cancer. A meta-analysis. JAMA 273, 1605–1609.

123 Permert J., Ihse I., Jorfeldt L., von Schenck H., Arnqvist H. J. and Larsson J. (1993) Pancreatic cancer is associated with impaired glucose metabolism. Eur. J. Surg. 159, 101–107.

124 Zelzer E., Levy Y., Kahana C., Shilo B. Z., Rubinstein M. and Cohen B. (1998) Insulin induces transcription of target genes through the hypoxia-inducible factor HIF-1alpha/ARNT. EMBO J. 17, 5085–5094.

125 Treins C., Giorgetti-Peraldi S., Murdaca J., Semenza G. L. and Van Obberghen E. (2002) Insulin stimulates hypoxia-inducible factor 1 through a phosphatidylinositol 3-kinase/target of rapamycin-dependent signaling pathway. J. Biol. Chem. 277, 27975–27981.

126 Elit L. (2006) Drug evaluation: AP-23573—an mTOR inhibitor for the treatment of cancer. IDrugs 9, 636–644.

127 Yee K. W., Zeng Z., Konopleva M., Verstovsek S., Ravandi F., Ferrajoli A., Thomas D., Wierda W., Apostolidou E., Albitar M., O’Brien S., Andreeff M. and Giles F. J. (2006) Phase I/II study of the mammalian target of rapamycin inhibitor everolimus (RAD001) in patients with relapsed or

refractory hematologic malignancies. Clin. Cancer Res. 12, 5165–5173.

128 Chan S., Scheulen M. E., Johnston S., Mross K., Cardoso F., Dittrich C., Eiermann W., Hess D., Morant R., Semiglazov V., Bonner M., Salzberg M., Ostapenko V., Illiger H. J., Behringer D., Bardy-Bouxin N., Boni J., Kong S., Cincotta M. and Moore L. (2005) Phase II study of temsirolimus (CCI-779), a novel inhibitor of mTOR, in heavily pretreated patients with locally advanced or metastatic breast cancer. J. Clin. Oncol. 23, 5314–5322.

129 Ma W. W. and Jimeno A. (2007) Temsirolimus. Drugs Today (Barc) 43, 659–669.

130 Wang Y., Zhao Q., Ma S., Yang F., Gong Y. and Ke C. (2007) Sirolimus inhibits human pancreatic carcinoma cell proliferation by a mechanism linked to the targeting of mTOR/HIF-1 alpha/VEGF signaling. IUBMB Life 59, 717–721.

131 Rapisarda A., Uranchimeg B., Sordet O., Pommier Y., Shoemaker R. H. and Melillo G. (2004) Topoisomerase I-mediated inhibition of hypoxia-inducible factor 1, mechanism and therapeutic implications. Cancer Res. 64, 1475–1482.

132 Creighton-Gutteridge M., Cardellina J. H., 2nd, Stephen A. G., Rapisarda A., Uranchimeg B., Hite K., Denny W. A., Shoemaker R. H. and Melillo G. (2007) Cell type-specific, topoisomerase II-dependent inhibition of hypoxia-inducible factor-1alpha protein accumulation by NSC 644221. Clin. Cancer Res. 13, 1010–1018.

133 Chun Y. S., Yeo E. J., Choi E., Teng C. M., Bae J. M., Kim M. S. and Park J. W. (2001) Inhibitory effect of YC-1 on the hypoxic induction of erythropoietin and vascular endothelial growth factor in Hep3B cells. Biochem. Pharmacol. 61, 947–954.

134 Kim H. L., Yeo E. J., Chun Y. S. and Park J. W. (2006) A domain responsible for HIF-1alpha degradation by YC-1, a novel anticancer agent. Int. J. Oncol. 29, 255–260.

135 Koh M. Y., Spivak-Kroizman T., Venturini S., Welsh S., Williams R. R., Kirkpatrick D. L. and Powis G. (2008) Molecular mechanisms for the activity of PX-478, an anti-tumor inhibitor of the hypoxia-inducible factor-1{alpha}. Mol. Cancer Ther. 7, 90–100.

136 Welsh S., Williams R., Kirkpatrick L., Paine-Murrieta G. and Powis G. (2004) Antitumor activity and pharmacodynamic properties of PX-478, an inhibitor of hypoxia-inducible factor-1alpha. Mol. Cancer Ther. 3, 233–244.

137 Floridi A., Paggi M. G., D’Atri S., De Martino C., Marcante M. L., Silvestrini B. and Caputo A. (1981) Effect of lonidamine on the energy metabolism of Ehrlich ascites tumor cells. Cancer Res. 41, 4661–4666.

138 Floridi A. and Lehninger A. L. (1983) Action of the antitumor and antispermatogenic agent lonidamine on electron transport in Ehrlich ascites tumor mitochondria. Arch. Biochem. Biophys. 226, 73–83.

139 Ben-Horin H., Tassini M., Vivi A., Navon G. and Kaplan O. (1995) Mechanism of action of the antineoplastic drug lonidamine: 31P and 13C nuclear magnetic resonance studies. Cancer Res. 55, 2814–2821.

140 Di Cosimo S., Ferretti G., Papaldo P., Carlini P., Fabi A. and Cognetti F. (2003) Lonidamine: efficacy and safety in clinical trials for the treatment of solid tumors. Drugs Today (Barc) 39, 157–174.

141 Geschwind J. F., Georgiades C. S., Ko Y. H. and Pedersen P. L. (2004) Recently elucidated energy catabolism pathways provide opportunities for novel treatments in hepatocellular carcinoma. Expert Rev. Anticancer Ther. 4, 449–457.

142 Pelicano H., Martin D. S., Xu R. H. and Huang P. (2006) Glycolysis inhibition for anticancer treatment. Oncogene 25, 4633–4646.

143 Xu R. H., Pelicano H., Zhou Y., Carew J. S., Feng L., Bhalla K. N., Keating M. J. and Huang P. (2005) Inhibition of glycolysis in cancer cells: a novel strategy to overcome drug resistance associated with mitochondrial respiratory defect and hypoxia. Cancer Res 65, 613–621.


144 Maschek G., Savaraj N., Priebe W., Braunschweiger P., Hamilton K., Tidmarsh G. F., De Young L. R. and Lampidis T. J. (2004) 2-deoxy-D-glucose increases the efficacy of adriamycin and paclitaxel in human osteosarcoma and non-small cell lung cancers in vivo. Cancer Res. 64, 31–34.

145 Maher J. C., Krishan A. and Lampidis T. J. (2004) Greater cell cycle inhibition and cytotoxicity induced by 2-deoxy-D-glucose in tumor cells treated under hypoxic vs aerobic conditions. Cancer Chemother. Pharmacol. 53, 116–122.

146 Gomathinayagam R. and Damodaran C. (2006) The radiosensitization effect of 2-deoxy-D-glucose on human glioma cells. Cancer Biol. Ther. 5, 1152–1153.

147 Heminger K., Jain V., Kadakia M., Dwarakanath B. and Berberich S. J. (2006) Altered gene expression induced by ionizing radiation and glycolytic inhibitor 2-deoxy-glucose in a human glioma cell line: implications for radio sensitization. Cancer Biol. Ther. 5, 815–823.

148 Dwarakanath B. S., Zolzer F., Chandana S., Bauch T., Adhikari J. S., Muller W. U., Streffer C. and Jain V. (2001) Heterogeneity in 2-deoxy-D-glucose-induced modifications in energetics and radiation responses of human tumor cell lines. Int. J. Radiat. Oncol. Biol. Phys. 50, 1051–1061.

149 Samid D., Ram Z., Hudgins W. R., Shack S., Liu L., Walbridge S., Oldfield E. H. and Myers C. E. (1994) Selective activity of phenylacetate against malignant gliomas: resemblance to fetal brain damage in phenylketonuria. Cancer Res. 54, 891–895.

150 Samid D., Shack S. and Myers C. E. (1993) Selective growth arrest and phenotypic reversion of prostate cancer cells in vitro by nontoxic pharmacological concentrations of phenylacetate. J. Clin. Invest. 91, 2288–2295.

151 Gonzalez-Mateos F., Gomez M. E., Garcia-Salguero L., Sanchez V. and Aragon J. J. (1993) Inhibition of glycolysis by amino acids in ascites tumor cells. Specificity and mechanism. J. Biol. Chem. 268, 7809–7817.

152 Griffiths M., Keast D., Patrick G., Crawford M. and Palmer T. N. (1993) The role of glutamine and glucose analogues in metabolic inhibition of human myeloid leukaemia in vitro. Int. J. Biochem. Cell Biol. 25, 1749–1755.

153 Street J. C., Mahmood U., Ballon D., Alfieri A. A. and Koutcher J. A. (1996) 13C and 31P NMR investigation of effect of 6-aminonicotinamide on metabolism of RIF-1 tumor cells in vitro. J. Biol. Chem. 271, 4113–4119.

154 Stolfi R. L., Colofiore J. R., Nord L. D., Koutcher J. A. and Martin D. S. (1992) Biochemical modulation of tumor cell energy: regression of advanced spontaneous murine breast tumors with a 5-fluorouracil-containing drug combination. Cancer Res. 52, 4074–4081.

155 Nord L. D., Stolfi R. L., Colofiore J. R. and Martin D. S. (1996) Correlation of retention of tumor methylmercaptopurine riboside-5′-phosphate with effectiveness in CD8F1 murine mammary tumor regression. Biochem. Pharmacol. 51, 621–627.

156 Martin D. S., Stolfi R. L., Colofiore J. R., Nord L. D. and Sternberg S. (1994) Biochemical modulation of tumor cell energy in vivo: II. A lower dose of adriamycin is required and a greater antitumor activity is induced when cellular energy is depressed. Cancer Invest. 12, 296–307.

157 Whitehead R. P., Benedetti J. K., Abbruzzese J. L., Ardalan B., Goodwin J. W., Balcerzak S. P., Samlowski W. E., Lenz H. J. and Macdonald J. S. (2004) A phase II study of high-dose 24 hour continuous infusion 5-FU and leucovorin and low-dose PALA for patients with advanced pancreatic adenocarcinoma: a Southwest Oncology Group Study. Invest. New Drugs 22, 335–341.

158 Swinnen J. V., Beckers A., Brusselmans K., Organe S., Segers J., Timmermans L., Vanderhoydonc F., Deboel L., Derua R., Waelkens E., De Schrijver E., Van de Sande T., Noel A., Foulfelle F. and Verhoeven G. (2005) Mimicry of a cellular low energy status blocks tumor cell anabolism and suppresses the malignant phenotype. Cancer Res. 65, 2441–2448.

Cell. Mol. Life Sci. Vol. 65, 2008

159 Baker S. G. and Kramer B. S. (2007) Paradoxes in carcinogenesis: new opportunities for research directions. BMC Cancer 7, 151.

160 Vineis P. (2003) Cancer as an evolutionary process at the cell level: an epidemiological perspective. Carcinogenesis 24, 1–6.

161 Gatenby R. A. and Gillies R. J. (2004) Why do cancers have high aerobic glycolysis? Nat. Rev. Cancer 4, 891–899.

---

To access this journal online:

http://www.birkhauser.ch/CMLS
