
CHAPTER 14

Vancomycin Resistance VanS/VanR  
Two-Component Systems  

Hee-Jeon Hong, Matthew I. Hutchings and Mark J. Buttner*

Abstract  
Vancomycin is a member of the glycopeptide class of antibiotics. Vancomycin resistance (*van*) gene clusters are found in human pathogens such as *Enterococcus faecalis*, *Enterococcus faecium* and *Staphylococcus aureus*, glycopeptide-producing actinomycetes such as *Amycolotopsis orientalis*, *Actinoplanes teichomyceticus* and *Streptomyces toyocaensis* and the nonglycopeptide producing actinomycete *Streptomyces coelicolor*. Expression of the *van* genes is activated by the VanS/VanR two-component system in response to extracellular glycopeptide antibiotic. Two major types of inducible vancomycin resistance are found in pathogenic bacteria; VanA strains are resistant to vancomycin itself and also to the lipidated glycopeptide teicoplanin, while VanB strains are resistant to vancomycin but sensitive to teicoplanin. Here we discuss the enzymes the *van* genes encode, the range of different VanS/VanR two-component systems, the biochemistry of VanS/VanR, the nature of the effector ligand(s) recognised by VanS and the evolution of the *van* cluster.

Introduction  
Vancomycin is clinically important for treating enterococcal infections arising after abdominal surgery and is vital as the only widely effective treatment for infections caused by methicillin-resistant *Staphylococcus aureus* (MRSA), a major killer in hospital-acquired infections. Vancomycin and other glycopeptide antibiotics inhibit cell wall biosynthesis by binding to the D-alanyl-D-alanine (D-Ala-D-Ala) terminus of lipid-attached peptidoglycan precursors on the outside of the cytoplasmic membrane (Fig. 1A).<sup>1,2</sup> This interaction blocks formation of mature peptidoglycan, principally by denying transpeptidase access to its substrate, thereby preventing formation of the peptide crosslinks between polysaccharide strands that give the cell wall its rigidity.

The first clinical isolates of vancomycin-resistant strains of pathogenic *Enterococcus faecalis* and *Enterococcus faecium* (VRE) appeared in the late 1980s and were shown to reprogramme cell wall biosynthesis such that the ‘stem’ pentapeptide of peptidoglycan precursors terminated in D-alanyl-D-lactate (D-Ala-D-Lac), rather than in D-Ala-D-Ala (Fig. 1B).<sup>3–6</sup> The affinity of vancomycin for precursors terminating in D-Ala-D-Lac is ~1000-fold lower than for precursors terminating in D-Ala-D-Ala, rendering the modified bacteria resistant.<sup>3</sup> Because vancomycin is the front-line therapy for treating problematic infections caused by MRSA, the spread of vancomycin resistance through bacterial populations is an acute public health issue, highlighted by the recent emergence of vancomycin-resistant, methicillin-resistant *Staphylococcus aureus* (VRSA) in hospitals.<sup>7–10</sup>

*Corresponding Author: Mark J. Buttner—Department Molecular Microbiology, John Innes Centre, Norwich Research Park, Colney, Norwich NR4 7UH, UK.  
Email: mark.buttner@bbsrc.ac.uk*

Bacterial Signal Transduction: Networks and Drug Targets, edited by Ryutaro Utsumi.  
©2008 Landes Bioscience and Springer Science+Business Media.

Vancomycin Resistance VanS/VanR Two-Component Systems

![Diagram](attachment:diagram.png)

Figure 1. Transpeptidase and the mode of action of vancomycin. A) Transpeptidase recognises the sequence D-alanyl-D-alanine (D-Ala-D-Ala) at the end of the pentapeptide chain, cleaves off the terminal alanine and joins the remainder to the branch of a stem peptide from an adjacent polysaccharide chain. Vancomycin and other glycopeptide antibiotics inhibit cell wall biosynthesis by binding to the D-Ala-D-Ala terminus of lipid-attached peptidoglycan precursors on the outside of the cytoplasmic membrane. This interaction blocks formation of mature peptidoglycan, principally by denying transpeptidase access to its substrate. B) Vancomycin resistant bacteria reprogramme cell wall biosynthesis such that the stem pentapeptide terminates in D-alanyl-D-lactate (D-Ala-D-Lac). The affinity of vancomycin for D-Ala-D-Lac is ~1000-fold lower than for D-Ala-D-Ala, allowing transpeptidation to occur. Note that the peptidoglycan precursor shown is the one present in Streptomyces, but the exact nature of the precursor varies from genus to genus; in Streptomyces it has LL-diaminopimelic acid (LL-dpm) at position 3 of the stem pentapeptide and the branch is a single glycine.

A Range of Different VanS/VanR Systems

In the context of this book, the key point about the vancomycin resistance (van) genes is that they are expressed only in the presence of extracellular glycopeptides and that signal transduction is mediated by a two-component system consisting of a sensor kinase (VanS) and a response regulator (VanR). Five different VanS/VanR two-component systems have been examined, albeit to very differing extents. Most effort has been concentrated on the VanS/VanR systems associated with the clinically important VanA (VanS<sub>A</sub>/VanR<sub>A</sub>) and VanB (VanS<sub>B</sub>/VanR<sub>B</sub>) enterococcal strains that first appeared in hospitals in the late 1980s.<sup>11</sup> More recently, there has been analysis of VanS/VanR systems found in actinomycetes, the order of bacteria that make all of the known glycopeptides. In every case where it has been studied, the vanS/vanR genes are themselves under VanS/VanR control, creating an auto-amplification loop in the presence of inducer (Fig. 2).

Enterococcal VanA and VanB Strains

Enterococcal VanA strains are resistant to vancomycin itself and also to the lipidated glycopeptide teicoplanin (Fig. 3), while VanB strains are resistant to vancomycin but sensitive to teicoplanin. The van genes in VanA strains are carried on the transposon Tn1546 and there is very little sequence variation between the van genes in VanA isolates. The first isolates of the new S. aureus hospital 'superbug,' VRSA, arose from intergeneric transfer of Tn1546 from a co-isolate of E. faecalis.<sup>7,10</sup> The van genes in VanB strains are chromosomally encoded and are more diverse in sequence than their VanA equivalents.

In contrast to these VanA and VanB resistant strains, the comparatively rare VanC, VanE and VanG isolates of Enterococci have a D-alanyl-D-serine (D-Ala-D-Ser) ligase instead of a D-Ala-D-Lac ligase.<sup>6,12</sup> The substitution of D-Ser for D-Ala results in a ~6-fold decrease in affinity for vancomycin and therefore low-level resistance.<sup>13</sup> The D-Ser-based systems will not be considered further here.

Glycopeptide-Resistant Actinomycetes

Glycopeptide resistance has been explored in three different actinomycetes: Streptomyces coelicolor, which does not make a glycopeptide and Streptomyces toyocaensis and Actinoplanes teichomyceticus, which do make glycopeptides. Glycopeptide resistance has an additional significance

![Figure 2](attachment://image.png)

Figure 2. Organisation and regulation of the vancomycin resistance (van) gene cluster of S. coelicolor. The genes are organized into four transcription units, vanS/vanR, vanJ, vanK and vanHAX and these transcripts are induced by vancomycin in a vanR-dependent manner. Reproduced with permission from reference 23.

Vancomycin Resistance VanS/VanR Two-Component Systems

Figure 3. Structures of the glycopeptide antibiotics vancomycin, teicoplanin and A47934 and the cell wall-specific, nonglycopeptide antibiotic, moenomycin A.

in glycopeptide producers, where activation of the resistance genes by the endogenously produced antibiotic prevents suicide (auto-toxicity).

*S. coelicolor* is genetically the model species of a genus of Gram-positive, mycelial soil bacteria responsible for the production of two-thirds of the commercially important antibiotics. Like most other nonpathogenic actinomycetes, *S. coelicolor* lives in the soil and it seems likely that it encounters glycopeptide producers such that the *van* gene cluster (Figs. 2 and 4) confers a selective advantage. Further, it is widely believed that all glycopeptide resistance genes are ultimately derived from actinomycete glycopeptide producers.¹⁴ Consistent with this idea, the *S. coelicolor* resistance genes are clearly associated with a laterally acquired DNA element (G. Chandra and H.-J. Hong, unpublished).

*S. toyocaensis* is the producer of the ‘sugarless’ glycopeptide A47934 (Fig. 3) and the *van* resistance genes in this organism (Fig. 4) are associated with the A47934 biosynthetic cluster.¹⁵ *S. toyocaensis* is resistant to A47934 but sensitive to both vancomycin and teicoplanin.

*A. teichomyceticus* is the producer of teicoplanin (Fig. 3) and carries a *van* cluster (Fig. 4), including *vanS/vanR*, associated with the teicoplanin biosynthetic genes.¹⁶,¹⁷ *A. teichomyceticus* is resistant to all glycopeptides tested, but it now seems clear that this ‘pan-glycopeptide resistance’ does not arise from pan-glycopeptide induction of the *van* genes but rather because the *van* genes are expressed constitutively, even in the absence of antibiotic.¹⁸ The cause of the constitutive expression of the *van* genes is unknown, but one possibility is that the VanS sensor kinase is locked in the ‘on’ state in this organism.

## What Do the *van* Genes Encode?

The number of genes present in the *van* cluster varies (Fig. 4), but the ‘core’ cluster consists of five genes—*vanS/vanR*, plus a *vanHAX* operon encoding the three enzymes required for remodelling cell wall precursors: VanH, which converts pyruvate to D-lactate; VanA, a D-Ala-D-Lac ligase; and VanX, a D-Ala-D-Ala dipeptidase that cleaves any residual D-Ala-D-Ala dipeptide, ensuring

VanA
![Diagram of VanA](#)
VanB
![Diagram of VanB](#)
S. coelicolor
![Diagram of S. coelicolor](#)
S. toyocaensis
![Diagram of S. toyocaensis](#)
A. teichomyceticus
![Diagram of A. teichomyceticus](#)

Figure 4. Comparison of the van gene clusters from enterococcal VanA and VanB strains, *S. coelicolor*, *S. toyocaensis* and *A. teichomyceticus*.

that peptidoglycan precursors terminate uniformly in D-Ala-D-Lac. In addition to this minimal set, other genes are sometimes present. The VanA transposon Tn1546 encodes two accessory proteins, VanY and VanZ, which are not required for, but can contribute to, high level resistance to vancomycin and teicoplanin. VanY is a D,D-carboxypeptidase that can cleave the C-terminal D-Ala of peptidoglycan precursors (but has no activity against free D-Ala-D-Ala dipeptide, the VanX substrate).<sup>19,20</sup> VanZ confers low level teicoplanin resistance in the absence of the other resistance proteins by an unknown mechanism.<sup>21</sup>

The *S. coelicolor* cluster consists of seven genes, *vanSRJKHAX*, divided into four transcription units and carries two genes, *vanJ* and *vanK*, not found in enterococcal VanA and VanB strains (Figs. 2 and 4).<sup>22</sup> *vanJ*, encoding a predicted membrane protein of unknown function, is not required for vancomycin resistance, but *vanK* is essential for resistance.<sup>22</sup> VanK is a member of the Fem family of enzymes, which add the ‘branch’ amino acid(s) to the stem pentapeptide of peptidoglycan precursors. In *S. coelicolor*, the branch is a single glycine residue (Fig. 1) and, in the absence of vancomycin, this residue is added by an enzyme called FemX.<sup>23</sup> However, the constitutive FemX activity of *S. coelicolor* can recognise only precursors that terminate in D-Ala-D-Ala as a substrate. VanK is required for vancomycin resistance because it is the only enzyme that can add the Gly branch to precursors terminating in D-Ala-D-Lac (production of precursors lacking the Gly branch is lethal in *Streptomyces* because it prevents cross-linking of the peptidoglycan by transpeptidase, leading to cell lysis).<sup>23</sup> The absence of orthologues of *vanK* in the vancomycin-resistance gene clusters of pathogenic enterococci implies that FemX of enterococci can recognise precursors terminating in either D-Ala-D-Lac or D-Ala-D-Ala.

### VanS/VanR Biochemistry

VanS/VanR systems from enterococci and *S. coelicolor* have been characterised in vitro. Wright et al<sup>24</sup> demonstrated that a fusion protein consisting of maltose binding protein (MBP) and the cytosolic domain of enterococcal VanS<sub>A</sub> could catalyse both autophosphorylation and rapid phosphotransfer to purified VanR<sub>A</sub>. Incubation of MBP-VanS<sub>A</sub> with phosphorylated VanR<sub>A</sub> (VanR<sub>A</sub>~P) increased its dephosphorylation approximately 6-fold, suggesting that VanS can also act as a VanR<sub>A</sub>-specific phosphatase.<sup>24</sup> Similar experiments showed that enterococcal VanS<sub>B</sub> also possesses both VanR<sub>B</sub> kinase and phosphatase activity.<sup>25</sup> Further, Hutchings et al<sup>26</sup> showed that the cytosolic domain of *S. coelicolor* VanS can autophosphorylate and catalyse both phosphorylation and dephosphorylation of *S. coelicolor* VanR in vitro. Thus, the in vitro

biochemical evidence suggests that VanS is a bifunctional protein that can switch between kinase and phosphatase activities.

Using gel shift assays and a DNA fragment carrying the *vanH*<sub>A</sub> promoter region, Holman et al<sup>27</sup> showed that phosphorylation of enterococcal VanR<sub>A</sub> results in a 500-fold increase in DNA-binding activity. Similarly, VanR<sub>B</sub> ~ P was found to bind target promoters more tightly than unphosphorylated VanR<sub>B</sub> and to be more efficient in promoting open complex formation by RNA polymerase.<sup>28</sup> DNaseI footprinting experiments suggested that phosphorylation of VanR<sub>A</sub> resulted in oligomerisation of the protein at the *vanH*<sub>A</sub> promoter. Unphosphorylated VanR<sub>A</sub>, or a D53A variant which cannot be phosphorylated, exhibited lower DNA binding-affinity and a smaller footprint at the *vanH*<sub>A</sub> promoter.<sup>27</sup> Investigation of VanR<sub>B</sub> oligomerization using gel filtration suggests that enterococcal VanR<sub>B</sub> is converted from monomer to dimer on phosphorylation.<sup>28</sup> The intrinsic in vitro stability of phosphorylated response regulators varies widely, perhaps reflecting their physiological roles, with isolated proteins displaying half-lives ranging from 23 s for CheY ~ P, involved in chemotaxis, to 180 min for Spo0F ~ P, involved in *Bacillus* sporulation.<sup>25</sup> The half-life of VanR<sub>B</sub> ~ P is ~150 min.<sup>25</sup>

### VanS/VanR and Acetyl Phosphate

In many two-component systems, loss of the sensor kinase or loss of the response regulator leads to the same phenotype—loss of expression of the target genes. However, in both *S. coelicolor* and enterococci, deletion of *vanS* results in constitutive expression of the vancomycin resistance genes, suggesting that VanS negatively regulates VanR function in the absence of antibiotic. In other words, VanR ~ P can be generated in a VanS-independent manner and VanS acts as a VanR ~ P phosphatase in the absence of vancomycin. In *S. coelicolor*, VanS-independent synthesis of VanR ~ P appears to arise because VanR can be activated in vivo by the small molecule phosphodonor acetyl phosphate. Deletion of *vanS* in *S. coelicolor* results in constitutive expression of the *van* genes but a *vanS pta ackA* triple mutant, which should not be able to synthesise acetyl phosphate, fails to express the *van* genes, whereas a *pta ackA* double mutant shows wild-type, regulated induction of the *van* genes.<sup>26</sup> These results suggest that in the absence of vancomycin, acetyl phosphate phosphorylates VanR and VanS acts as a phosphatase to suppress the levels of VanR ~ P. On exposure to vancomycin, VanS activity switches from a phosphatase to a kinase and vancomycin resistance is induced (Fig. 5). It should be noted that transcription of the *S. coelicolor vanS/vanR* operon is itself under VanS/VanR control (Fig. 2)<sup>22</sup> and so there will be very little VanR or VanS protein in *S. coelicolor* growing in the absence of vancomycin. Thus, the ‘futile cycle’ of VanR phosphorylation and dephosphorylation shown to occur in the absence of vancomycin in Fig. 5 will occur at a significant level only after the organism has been transiently exposed to the antibiotic. Similar results have been obtained in enterococcal VanA strains. Arthur et al<sup>29</sup> showed that the *van* promoters of an *E. faecium* VanA strain were constitutively activated by VanR<sub>A</sub> in the absence of VanS<sub>A</sub> and concluded that VanS<sub>A</sub> negatively controls VanR<sub>A</sub> in the absence of glycopeptide inducer, presumably by dephosphorylation. Further, Haldimann et al<sup>30</sup> introduced a *vanH*<sub>A</sub>*-lacZ* fusion into an *ackA* strain of *E. coli*, which overproduces acetyl phosphate. Heterologous expression of enterococcal VanR<sub>A</sub> in this strain stimulated high levels of β-galactosidase production, suggesting that acetyl phosphate could act as an in vivo phosphodonor to the *E. faecium* VanR<sub>A</sub> protein in *E. coli*.

### ‘Crosstalk’ with Other Two-Component Systems

In an elegant study using flow cytometry, Baptista et al<sup>31</sup> took advantage of a *vanY*<sub>B</sub>*-gfp* transcriptional fusion to examine induction of *van* gene expression in single cells of an enterococcal VanB strain. In enterococcal VanB strains, null mutations in *vanS*<sub>B</sub> lead to a phenotype termed ‘heterogeneous,’ in which, in the absence of antibiotic, only a minority of the bacteria express the *van* genes.<sup>31</sup> Further, addition of antibiotic leads to uniform induction of the whole population, rather than selection of the subpopulation initially expressing resistance under non-inducing conditions. They concluded that a heterologous kinase activated VanR<sub>B</sub> in the absence of VanS<sub>B</sub>. Interestingly, this putative kinase was stimulated by vancomycin, teicoplanin and the nonglycopeptide cell wall inhibitor moenomycin (Fig. 3), suggesting that it might respond to the same indirect signal as

no drug                                      inducer

VanS                                        VanS
phosphatase                                 kinase

P                                          P
VanR                                       VanR
acetyl phosphate                           expression
of van genes

Figure 5. A model for the function of the vancomycin resistance VanS/VanR two-component signal transduction system in *S. coelicolor*. In the absence of antibiotic, acetyl phosphate phosphorylates D51 of VanR and VanS acts as a phosphatase to suppress the levels of VanR~P. In the presence of antibiotic, VanS is converted from a phosphatase into a kinase, leading to the accumulation of VanR~P and activation of the four promoters of the *van* gene cluster. Transcription of the *vanS/vanR* operon is itself under VanS/VanR control and so there will be very little VanR or VanS protein in *S. coelicolor* growing in the absence of vancomycin. Thus, the 'futile cycle' of VanR phosphorylation and dephosphorylation shown in the absence of vancomycin will occur at a significant level only after the organism has been transiently exposed to the antibiotic.

VanS<sub>A</sub> from VanA-type enterococci (see below). Presumably, in wild-type enterococcal VanB strains, the phosphatase activity of VanS<sub>B</sub> keeps VanR<sub>B</sub> in the unphosphorylated state in the presence of teicoplanin and moenomycin, preventing the putative heterologous kinase from activating *van* gene expression. The putative heterologous kinase has not been identified but a possible candidate is CroS, since it is known to be induced by vancomycin, teicoplanin and moenomycin A.<sup>32</sup> CroS is required for intrinsic β-lactam resistance in *E. faecalis* but the target genes of the CroRS two-component system involved in this resistance have not been identified.<sup>32</sup>

### Relationships Between VanS Proteins of Different Origin

In considering the nature of the effector ligand(s) that activate VanS, it is important to keep in mind the relationships between VanS proteins of different origin. First, the differences in the sizes of the extracytoplasmic sensor domains are striking. The putative extracytoplasmic sensor domain of VanS<sub>A</sub> is 103 amino acids long, the equivalent domain of VanS<sub>B</sub> consists of 37 amino acids, whereas the putative extracytoplasmic sensor domains of the three actinomycete VanS proteins contain only 26-27 amino acids. These sensor domains are sufficiently small for the actinomycete VanS proteins to have been included in a review of 'intramembrane-sensing' sensor kinases.<sup>33</sup> The VanS proteins from enterococcal VanA and VanB strains are only distantly related (16% overall identity) and the putative VanS<sub>A</sub> and VanS<sub>B</sub> sensor domains are not related in amino acid sequence. The enterococcal VanS<sub>A</sub> and VanS<sub>B</sub> proteins are also very diverged from their actinomycete equivalents (~15% overall identity in pairwise comparisons). In contrast, the VanS proteins from the three actinomycetes strongly resemble each other (65-77% overall identity in pairwise comparisons). However, in comparing VanS from *S. coelicolor* and *S. toyocaensis*, it is clear that this high similarity breaks down in the 26-27-residue stretch between the two predicted transmembrane helices, corresponding to the putative VanS sensor domain (Fig. 6). It now seems clear that the *van* genes

Vancomycin Resistance VanS/VanR Two-Component Systems

---

**extracytoplasmic sensor domain**

| TM helix 1 | TM helix 2 |
|------------|------------|
| VanSsc:    | M--DRRPGLSVRLKLTLSSYAGFLTLAGVLLLVAVGVELLDQGWLLTNERGAVRATPG-TVELRSFAPTAAWVMALLVFGLVG : 80 |
| VanSst:    | MRVHRAAPGMSVRLKFTLSSYAGFVMLAGTLMMLVTWVYLLRG--LPDAASSPNGLVVPVRAIVLHAYAPATAVVLLALLVFGLLG : 81 |
|            | *          |
| VanSsc:    | GWELAGRMLAPLDRITEARTTAATGSLSHRIRLPGRRDEYRELADAFDEMLARLEAHVAEQRFFAANASHELRTPLAVSKAIL : 163 |
| VanSst:    | GWLLAGRMLAPLTRITDARTTAANGSLSHRIELEGRKDEFRELADEIMLARLEAHVAEQRFFAATASHELRTPLAITKALL : 164 |
| VanSsc:    | DVARTDPHQDPGEIIDRLHAVNTRAILTEALLLISRAQRSFTREQVDMSLLAEEATETLLPFEAKHGVTLLETRGHVTMALG : 246 |
| VanSst:    | EVTINGPGRDLGELIGRLHNVARAILTEALLLISRAIDORSFTREPVDSLINVVEAVEETLLPLAEKHGVAIEASGDITPAMG : 247 |
| VanSsc:    | SPALLLQLTENLVHNAIVHNLPGRGRVWHTAAGPRTRLLVENTGDLISP HQASTLTEEPFORGTERIHTDHPGVGLGLAIVN : 329 |
| VanSst:    | SRALLLQLTVNLIHNAIVHNLEGRGIVRVTTGIESGSVTLSVENTGEVLP AOLVSTLTEPFRRGTERIRQDHAGVGLGLAI VK : 330 |
| VanSsc:    | TITQAHDGTLLTPRHSGGLRVTVELPAAAPHET--GR : 364 |
| VanSst:    | SITEAHDGTLLTIVERPFGGLQVTVRLLPAAPPPETLNR : 367 |

---

Figure 6. Alignment of the VanS proteins from S. coelicolor (VanSsc) and S. toyocaensis (VanSst). The two predicted transmembrane helices and the 26-27-amino acid putative extracytoplasmic sensor domain lying in between are highlighted. The histidine that is the putative site of autophosphorylation is marked with an asterisk.

of *A. teichomyceticus* are not inducible but constitutively active,¹⁸ and so it is hard to know if *A.* teichomyceticus VanS binds effector molecule(s). Nevertheless, it is interesting to note that the putative 26-residue extracytoplasmic sensor domain of *A. teichomyceticus* VanS differs from that of *S. toyocaensis* at only 4 residues.

# What Is the Effector Ligand Recognised by VanS?

The nature of the direct molecular ligand that activates VanS has not been determined for any VanS/VanR signal transduction system. Two distinct models exist: direct induction, in which the sensor kinase is activated by direct binding of antibiotic to the sensor domain and indirect induction, in which the sensor kinase is activated by binding an intermediate in cell wall biosynthesis or degradation that accumulates as a result of antibiotic action. These two models are not mutually exclusively, since a further possibility is that the VanS inducer is the antibiotic bound to a D-Ala-D-Ala-containing cell wall precursor, such as lipid II. Given that the sensor domains of VanS proteins are not homologous, it is possible or even likely that some VanS proteins respond directly to the antibiotic while others respond indirectly. A summary of the genetic evidence that addresses this question is presented below, but it seems unlikely that genetics alone can identify the nature of the inducer and that biochemical (in vitro reconstitution studies; in vivo cross-linking) or structural studies will be required to provide a definitive answer.

## Induction in Enterococcal VanA Strains

Screens for inducers of VanS<sub>A</sub> have been established by coupling a promoter under the control of VanS/VanR to suitable reporter genes,³⁴⁻³⁸ by assaying VanX activity in cell extracts,³⁹ by monitoring induction of Lac-containing precursors,⁴⁰ or by looking for induced vancomycin resistance in pretreated cultures.⁴⁰,⁴¹ All these reports agree that VanA strains are induced by vancomycin and teicoplanin. However, the most interesting results from these papers concern the potential for nonglycopeptide cell wall-specific antibiotics to induce VanS. All reports agree that VanA strains are inducible by the nonglycopeptide moenomycin A.³⁶⁻⁴¹ The experiments of Baptista et al³⁹ are particularly compelling since they assayed induction of VanX enzymatic activity in cell extracts and did not rely on multicopy plasmids or reporter genes. Since moenomycin A is not structurally related to glycopeptides it seems unlikely that the sensor domain of VanS<sub>A</sub> could bind both glycopeptides and moenomycin directly. The general conclusion has therefore been that VanS<sub>A</sub> must be activated by an intermediate in cell wall biosynthesis that accumulates in response to both glycopeptides and moenomycin A. Because moenomycin A inhibits transglycosylase,⁴²⁻⁴³ both glycopeptides and moenomycin A are likely to lead to accumulation of lipid II (a lipid-anchored cell wall precursor) on the external face of the cytoplasmic membrane and it has been speculated that lipid II might be the direct effector ligand of VanS<sub>A</sub>.³¹

## Induction in Enterococcal VanB Strains

Induction of VanB strains has also been addressed.³¹,³⁹,⁴⁴ In contrast to VanA strains, all the nonglycopeptides tested, including moenomycin A, failed to induce VanS<sub>B</sub>. Since all VanS<sub>B</sub> inducers identified are structurally related glycopeptides, the simplest interpretation of the data is that VanS<sub>B</sub> is likely to be induced directly by the drug itself.³¹ VanB strains are sensitive to the lipitated antibiotic teicoplanin because the VanS<sub>B</sub>/VanR<sub>B</sub> signal transduction system is not induced by teicoplanin.³¹,³⁹,⁴⁴ In further experiments, Baptista et al³¹,³⁹,⁴⁴ isolated teicoplanin-resistant mutants of VanB strains, six of which showed induction of the *van* genes by teicoplanin (but not by the nonglycopeptide moenomycin A). These six mutants all carried single amino acid substitutions in the N-terminal half of VanS<sub>B</sub>. How to interpret these gain-of-function mutations is not clear. Two were in the predicted extracytoplasmic sensor domain where they could potentially directly improve interaction with an extracellular ligand, such as teicoplanin. However, the remaining four were in the cytoplasmic linker domain that connects the sensor and kinase domains. It is possible that wild-type VanS<sub>B</sub> binds teicoplanin unproductively and that these four amino acid substitutions affect propagation of the induction signal through the membrane such that signal transduction now occurs. However, it should be noted that in the case examined in detail (an A167S substitution in the linker domain),

the substitution conferring teicoplanin inducibility also conferred hyper-inducibility by vancomycin. Again, it is possible that wild-type VanS<sub>B</sub> binds teicoplanin unproductively and that the A167S mutation makes VanS<sub>B</sub> hypersensitive to inducers. Thus, the teicoplanin-inducible VanS<sub>B</sub> mutations may be qualitative and involve a change in induction specificity, or they may be quantitative and involve an increase in the sensitivity of the protein to inducers.<sup>31,39,44</sup>

### Induction in Actinomycete Species

Inducers of VanS in *S. coelicolor* were identified using a bioassay. *S. coelicolor femX* null mutants are viable only in the presence of compounds that activate the VanS/VanR signal transduction system, because they rely on expression of VanK for survival. Hutchings et al<sup>26</sup> took advantage of this antibiotic-dependent phenotype to create a simple bioassay for inducers of the *van* genes in *S. coelicolor*. The structurally closely related glycopeptide antibiotics vancomycin, ristocetin, chloroeremomycin and A47934 all acted as inducers of the VanS/VanR system, but the lipidated glycopeptide teicoplanin and the nonglycopeptide moenomycin A did not.

To address the effector ligand issue further, Hutchings et al<sup>26</sup> carried out a “VanS/VanR swap” experiment between two glycopeptide-resistant *Streptomyces* species with differing spectra of inducer molecules, to see if inducer specificity was determined by VanS/VanR itself or by the host background. In *S. coelicolor*, the *van* genes are induced by both A47934 and vancomycin, while in *S. toyocaensis*, resistance is induced by A47934 but not by vancomycin.<sup>45</sup> Introduction of the *S. toyocaensis* VanS/VanR signal transduction system into an *S. coelicolor vanS/vanR* null mutant switched inducer specificity to that of *S. toyocaensis*. Thus, inducer specificity is determined by the origin of VanS/VanR. There are two potential explanations for this observation. If *Streptomyces* VanS is activated by accumulation of a cell wall intermediate, vancomycin must induce a radically different spectrum of cell wall intermediates in *S. coelicolor* and *S. toyocaensis*, which seems unlikely. The more likely alternative is that *Streptomyces* VanS is directly activated by binding antibiotic (or possibly antibiotic bound to D-Ala-D-Ala-containing cell wall precursors, such as lipid II) and that *S. toyocaensis* VanS interacts productively with A47934 but not with vancomycin, whereas *S. coelicolor* VanS interacts productively with both antibiotics. This would also be consistent with the fact that the nonglycopeptide moenomycin is not an inducer of VanS in *Streptomyces*.

Whether the *Streptomyces* VanS effector ligand is a cell wall intermediate or the antibiotic itself, the ability to respond differentially to vancomycin and A47934 must reside in differences between the sensor domains of the *S. toyocaensis* and *S. coelicolor* VanS proteins. The VanS proteins from these two species are very similar, with 65% identity overall (Fig. 6). However, as noted above, it is striking that this high level of identity breaks down in the 26-27-residue putative sensor domain lying between the two predicted transmembrane helices (Fig. 6).

### Functional Differences between Vancomycin and Teicoplanin

Enterococcal VanA strains are resistant to teicoplanin, whereas VanB strains are sensitive because teicoplanin fails to induce VanS<sub>B</sub>. Vancomycin and teicoplanin (Fig. 3) differ in the structure of the aglycone (the peptide part of the molecule), the glycosylation pattern and in the presence of a fatty-acid chain attached to teicoplanin that is absent in vancomycin. Through the chemo-enzymatic synthesis of a spectrum of vancomycin and teicoplanin derivatives, Kahne and colleagues showed definitively that the key functional difference between these two antibiotics is the presence or absence of the lipid: removal of the lipid from teicoplanin prevents it from killing VanB strains and addition of a lipid to vancomycin makes it an effective antibiotic against VanB strains.<sup>46</sup> Taking this a stage further, using the same range of vancomycin and teicoplanin derivatives, it has been shown that it is the presence or absence of the lipid and not the differences in aglycone structure or glycosylation pattern, that is the key difference between the two antibiotics in determining *van* gene inducer activity in *S. coelicolor* (M. Oberthür, H.-J. Hong, C. Leimkuhler, B. Falcone, C. Walsh, M. Buttner and D. Kahne unpublished). These observations raise interesting questions about the evolution of teicoplanin. Perhaps addition of the lipid was selected during evolution

of the producing organism, *A. teichomyceticus*, at least in part because it prevents competing soil bacteria like *S. coelicolor* from sensing the antibiotic and generating a resistance response.

The lipid moiety can serve to anchor teicoplanin in the membrane,^{47-49} and advocates of direct induction of VanS<sub>B</sub> have proposed that membrane anchoring prevents teicoplanin from interacting productively with the VanS<sub>B</sub> sensor domain. However, it should be noted that this lipid moiety is relatively short and that teicoplanin is water soluble, implying that capture of teicoplanin by the membrane would not be as complete as for other molecules carrying longer lipid tails such as, for example, lipoproteins. A further issue concerns the mode of action of these drugs. Both teicoplanin and vancomycin bind to D-Ala-D-Ala and inhibit both transpeptidation and transglycosylation. However, vancomycin exerts its major effect on transpeptidation whereas lipidated glycopeptides inhibit transglycosylation more strongly.^{43,50-52} These observations suggest that the actions of vancomycin and teicoplanin will lead to the accumulation of somewhat different spectra of cell wall intermediates, leaving open the possibility that the enterococcal VanB phenotype could be accounted for through an indirect induction mechanism.

## Evolution of the *van* Cluster

An important unanswered question is whether there is selective pressure against the evolution of constitutive expression of the *van* genes. Such pressure might arise from the relative thermodynamic instability of the D-Ala-D-Lac ester linkage. The D-Ala-D-Ala peptide bond is more stable than the D-Ala-D-Lac ester linkage and spontaneous hydrolysis of lactate from cell wall precursors would yield molecules incapable of supporting cell wall crosslinking (because the D-Ala-D-Lac or D-Ala-D-Ala bond is cleaved during transpeptidation and the energy of the bond is conserved to form the peptide crosslink with the pendant peptide of an adjacent polysaccharide chain; Fig. 1). Potentially consistent with this logic, *vanS* null mutants of *S. coelicolor*, which express D-Ala-D-Lac precursors constitutively, suffer, for unknown reasons, a growth rate disadvantage relative to the wild type expressing D-Ala-D-Ala precursors (Hong, Hutchings and Buttner, unpublished). However, constitutive expression of D-Ala-D-Lac precursors does not seem to cause a growth rate disadvantage in enterococci (Michel Arthur, pers. comm.) and, most importantly, members of the genera *Leuconostoc, Lactobacillus* and *Pediococcus* are all naturally resistant to vancomycin because they constitutively express cell wall precursors terminating in D-Ala-D-Lac.^{53-56}

Finally, it should be noted that expression of vancomycin resistance can bring into play other selective pressures that have very important clinical consequences. For example, enterococcal strains expressing vancomycin resistance become sensitive to third generation cephalosporins, like ceftriaxone, because the ceftriaxone-resistant penicillin-binding protein (called PBP5) cannot recognise cell wall precursors terminating in D-Ala-D-Lac as substrates.^{57} Likewise, the recent intergeneric transfer of Tn1546 from enterococci into *S. aureus* to create VRSA strains has similar interesting consequences: VRSA strains expressing vancomycin resistance become sensitive to β-lactams, because PBP2A (encoded by *mecA*), which confers β-lactam resistance, cannot recognise cell wall precursors terminating in D-Ala-D-Lac as substrates.^{58} Thus, while VRSA strains are highly resistant to vancomycin (MIC = 512 μg/ml) or β-lactams such oxacillin (MIC = 800 μg/ml) when applied individually, they are very effectively killed by low concentrations of these two drugs in combination (for example 40 μg/ml oxacillin with 12 μg/ml vancomycin is lethal).^{58,59}

## Acknowledgements

We are very grateful to Michel Arthur, Daniel Kahne, Chris Walsh, David Hopwood and Flavia Marinelli for helpful discussion, communication of unpublished data and comments on the manuscript. The authors’ work on vancomycin resistance in *Streptomyces* was funded by the Biotechnology and Biological Sciences Research Council (BBSRC) of the United Kingdom.

References

1. Williams DH, Williamson MP, Butcher DW et al. Detailed binding sites of the antibiotics vancomycin and ristocetin A: determination of intermolecular distances in antibiotic/substrate complexes by use of the time-dependent NOE. J Am Chem Soc 1983; 105:1332-1339.
2. Barna JCJ, Williams DH. The structure and mode of action of glycopeptide antibiotics of the vancomycin group. Annu Rev Microbiol 1984; 38:339-57.
3. Bugg TDH, Wright GD, Dutka-Malen S et al. Molecular basis for vancomycin resistance in Enterococcus faecium BM4147: biosynthesis of a depsipeptide peptidoglycan precursor by vancomycin resistance proteins VanH and VanA. Biochemistry 1991; 30:10408-10415.
4. Healy VL, Lessard IA, Roper DI et al. Vancomycin resistance in enterococci: reprogramming of the D-Ala-D-Ala ligases in bacterial peptidoglycan biosynthesis. Chem Biol 2000; 7:R109-119.
5. Walsh CT, Fisher SL, Park IS et al. Bacterial resistance to vancomycin: five genes and one missing hydrogen bond tell the story. Chem Biol 1996; 3:21-8.
6. Pootoolal J, Neu J, Wright GD. Glycopeptide antibiotic resistance. Annu Rev Pharmacol Toxicol 2002; 42:381-408.
7. Weigel LM, Clewell DB, Gill SR et al. Genetic analysis of a high-level vancomycin-resistant isolate of Staphylococcus aureus. Science 2003; 28:1569-1571.
8. Pearson H. 'Superbug' hurdles key drug barrier. Nature 2002; 418:469-470.
9. Chang S, Sievert DM, Hageman JC et al. Infection with vancomycin-resistant Staphylococcus aureus containing the vanA resistance gene. N Engl J Med 2003; 348:1342-1347.
10. Tenover FC, Weigel LM, Appelbaum PC et al. Vancomycin-resistant Staphylococcus aureus isolate from a patient in Pennsylvania. Antimicrob Agents Chemother 2004; 48:275-280.
11. Arthur M, Quintiliani R. Regulation of VanA- and VanB-type glycopeptide resistance in enterococci. Antimicrob Agents Chemother 2001; 45:375-81.
12. Reynolds PE, Courvalin P. Vancomycin resistance in enterococci due to synthesis of precursors terminating in D-alanyl-D-serine. Antimicrob Agents Chemother 2005; 49:21-5.
13. Billot-Klein D, Blanot D, Gutmann L et al. Association constants for the binding of vancomycin and teicoplanin to N-acetyl-D-alanyl-D-alanine and N-acetyl-D-alanyl-D-serine. Biochem J 1994; 304:1021-1022.
14. Davies J. Inactivation of antibiotics and the dissemination of resistance genes. Science 1994; 264:375-382.
15. Pootoolal J, Thomas MG, Marshall CG et al. Assembling the glycopeptide antibiotic scaffold: the biosynthesis of A47934 from Streptomyces toyocaensis. Proc Natl Acad Sci USA 2002; 99:8962-8967.
16. Sosio M, Kloosterman H, Bianchi A et al. Organization of the teicoplanin gene cluster in Actinoplanes teichomyceticus. Microbiology 2004; 150:95-102.
17. Serina S, Radice F, Maffioli S et al. Glycopeptide resistance determinants from the teicoplanin producer Actinoplanes teichomyceticus. FEMS Microbiol Lett 2004; 240:69-74.
18. Beltrametti F, Consolandi A, Carrano L et al. Resistance to glycopeptide antibiotics in the teicoplanin producer is mediated by van-gene homologue expression directing the synthesis of a modified cell wall peptidoglycan. Antimicrob Agents Chemother 2007; 51:1135-41.
19. Arthur M, Depardieu F, Cabanie L et al. Requirement of the VanY and VanX D,D-peptidases for glycopeptide resistance in enterococci. Mol Microbiol 1998; 30:819-30.
20. Wright GD, Molinas C, Arthur M et al. Characterization of VanY, a DD-carboxypeptidase from vancomycin-resistant Enterococcus faecium BM4147. Antimicrob Agents Chemother 1992; 36:1514-8.
21. Arthur M, Depardieu F, Molinas C et al. The vanZ gene of Tn1546 from Enterococcus faecium BM4147 confers resistance to teicoplanin. Gene 1995; 154:87-92.
22. Hong H-J, Hutchings MI, Neu JM et al. Characterisation of an inducible vancomycin resistance system in Streptomyces coelicolor reveals a novel gene (vanK) required for drug resistance. Mol Microbiol 2004; 52:1107-1121.
23. Hong H-J, Hutchings MI, Hill LM et al. The role of the novel Fem protein VanK in vancomycin resistance in Streptomyces coelicolor. J Biol Chem 2005; 280:13055-13061.
24. Wright GD, Holman TR, Walsh CT. Purification and characterization of VanR and the cytosolic domain of VanS: a two-component regulatory system required for vancomycin resistance in Enterococcus faecium BM4147. Biochemistry 1993; 32:5057-63.
25. Depardieu F, Courvalin P, Msadek T. A six amino acid deletion, partially overlapping the VanS<sub>B</sub> G2 ATP-binding motif, leads to constitutive glycopeptide resistance in VanB-type Enterococcus faecium. Mol Microbiol 2003; 50:1069-1083.
26. Hutchings MI, Hong H-J, Buttner MJ. The vancomycin resistance VanS/VanR two-component signal transduction system of Streptomyces coelicolor. Mol Microbiol 2006; 59:923-935.

27. Holman TR, Wu Z, Wanner BL et al. Identification of the DNA-binding site for the phosphorylated VanR protein required for vancomycin resistance in Enterococcus faecium Biochemistry 1994; 33:4625-31.

28. Depardieu F, Courvalin P, Kolb A. Binding sites of VanR<sub>B</sub> and σ<sup>70</sup> RNA polymerase in the vanB vancomycin resistance operon of Enterococcus faecium BM4524. Mol Microbiol 2005; 57:550-64.

29. Arthur M, Depardieu F, Gerbaud G et al. The VanS sensor negatively controls VanR-mediated transcriptional activation of glycopeptide resistance genes of Tn1546 and related elements in the absence of induction. J Bacteriol 1997; 179:97-106.

30. Haldimann A, Fisher SL, Daniels LL et al. Transcriptional regulation of the Enterococcus faecium BM4147 vancomycin resistance gene cluster by the VanS-VanR two-component regulatory system in Escherichia coli K-12. J Bacteriol 1997; 179:5903-5913.

31. Baptista M, Rodrigues P, Depardieu F et al. Single-cell analysis of glycopeptide resistance gene expression in teicoplanin-resistant mutants of a VanB-type Enterococcus faecalis. Mol Microbiol 1999; 32:17-28.

32. Comenge Y, Quintiliani R Jr, Li L et al. The CroRS two component regulatory system is required for intrinsic β-lactam resistance in Enterococcus faecalis. J Bacteriol 2003; 185:7184-92.

33. Mascher T. Intramembrane-sensing histidine kinases: a new family of bacterial cell envelope stress sensors. FEMS Microbiol Lett 2006; 264:133-144.

34. Ulijasz AT, Grenader A, Weisblum B. A vancomycin-inducible lacZ reporter system in Bacillus subtilis: induction by antibiotics that inhibit cell wall synthesis and by lysozyme. J Bacteriol 1996; 178:6305-6309.

35. Arthur M, Depardieu F, Courvalin P. Regulated interactions between partner and nonpartner sensors and response regulators that control glycopeptide resistance gene expression in enterococci. Microbiology 1999; 145:1849-58.

36. Lai MH, Kirsch DR. Induction signals for vancomycin resistance encoded by the vanA gene cluster in Enterococcus faecium. Antimicrob Agents Chemother 1996; 40:1645-1648.

37. Mani N, Sancheti P, Jiang ZD et al. Screening systems for detecting inhibitors of cell wall transglycosylation in Enterococcus. Cell wall transglycosylation inhibitors in Enterococcus. J Antibiot 1998; 51:471-479.

38. Grissom-Arnold J, Alborn WE, Nicas TI et al. Induction of VanA vancomycin resistance genes in Enterococcus faecalis: use of a promoter fusion to evaluate glycopeptide and nonglycopeptide induction signals. Microbial Drug Resistance 1997; 3:53-64.

39. Baptista M, Depardieu F, Courvalin P et al. Specificity of induction of glycopeptide resistance genes in Enterococcus faecalis. Antimicrob Agents Chemother 1996; 40:2291-2295.

40. Allen NE, Hobbs JN. Induction of vancomycin resistance in Enterococcus faecium by nonglycopeptide antibiotics. FEMS Microbiol Lett 1995; 132:107-114.

41. Handwerger S, Kolokathis A. Induction of vancomycin resistance in Enterococcus faecium by inhibition of transglycosylation. FEMS Microbiol Lett 1990; 58:167-170.

42. van Heijenoort J. Formation of the glycan chains in the synthesis of bacterial peptidoglycan. Glycobiology 2001; 11:25R-36R.

43. Chen L, Walker D, Sun B. et al. Vancomycin analogues active against vanA-resistant strains inhibit bacterial transglycosylase without binding substrate. Proc Natl Acad Sci USA 2003; 100:5658-5663.

44. Baptista M, Depardieu F, Reynolds P et al. Mutations leading to increased levels of resistance to glycopeptide antibiotics in VanB-type enterococci. Mol Microbiol 1997; 25:93-105.

45. Neu JM, Wright GD. Inhibition of sporulation, glycopeptide antibiotic production and resistance in Streptomyces toyocaensis NRRL 15009 by protein kinase inhibitors. FEMS Microbiol Lett 2001; 199:15-20.

46. Dong SD, Oberthur M, Losey HC et al. The structural basis for induction of VanB resistance. J Am Chem Soc 2002; 124:9064-5.

47. Sharman GJ, Try AC, Dancer RJ et al. The roles of dimerization and membrane anchoring in activity of glycopeptide antibiotics against vancomycin-resistant bacteria. J Am Chem Soc 1997; 119:12041-12047.

48. Cooper MA, Williams DH. Binding of glycopeptide antibiotics to a model of a vancomycin-resistant bacterium. Chem Biol 1999; 6:891-899.

49. Beauregard DA, Williams DH, Gwynn MN et al. Dimerization and membrane anchors in extracellular targeting of vancomycin group antibiotics. Antimicrob Agents Chemother 1995; 39:781-785.

50. Ge M, Chem Z, Onishi HR et al. Vancomycin derivatives that inhibit peptidoglycan biosynthesis without binding D-Ala-D-Ala. Science 1999; 284:507-511.

51. Sinha Roy R, Yang P, Kodali S et al. Direct interaction of a vancomycin derivative with bacterial enzymes involved in cell wall biosynthesis. Chem Biol 2001; 8:1095-1106.

52. Kerns, R. Dong, S.D. Fukuzawa S et al. The Role of hydrophobic substituents in the biological activity of glycopeptide antibiotics J Am Chem Soc 2000; 122:12608-12609.

53. Billot-Klein D, Gutmann L, Sable S et al. Modification of peptidoglycan precursors is a common feature of the low-level vancomycin-resistant VANB-type Enterococcus D366 and of the naturally glycopeptide-resistant species Lactobacillus casei, Pediococcus pentosaceus, Leuconostoc mesenteroides and Enterococcus gallinarum. J Bacteriol 1994; 176:2398-405.

54. Billot-Klein D, Legrand R, Schoot B et al. Peptidoglycan structure of Lactobacillus casei, a species highly resistant to glycopeptide antibiotics. J Bacteriol 1997; 179:6208-12.

55. Handwerger S, Pucci MJ, Volk KJ et al. Vancomycin-resistant Leuconostoc mesenteroides and Lactobacillus casei synthesize cytoplasmic peptidoglycan precursors that terminate in lactate. J Bacteriol 1994; 176:260-4.

56. Nicas TI, Cole CT, Preston DA et al. Activity of glycopeptides against vancomycin-resistant Gram-positive bacteria. Antimicrob Agents Chemother 1989; 33:1477-81.

57. Gutmann L, al-Obeid S, Billot-Klein D et al. Synergy and resistance to synergy between β-lactam antibiotics and glycopeptides against glycopeptide-resistant strains of Enterococcus faecium. Antimicrob Agents Chemother 1994; 38:824-9.

58. Severin A, Wu SW, Tabei K et al. Penicillin-binding protein 2 is essential for expression of high-level vancomycin resistance and cell wall synthesis in vancomycin-resistant Staphylococcus aureus carrying the enterococcal vanA gene complex. Antimicrob Agents Chemother 2004; 48:4566-73.

59. Severin A, Tabei K, Tenover F et al. High level oxacillin and vancomycin resistance and altered cell wall composition in Staphylococcus aureus carrying the staphylococcal mecA and the enterococcal vanA gene complex. J Biol Chem 2004; 279:3398-3407.
