
An Alternative Menaquinone Biosynthetic Pathway Operating in Microorganisms

Tomoshige Hiratsuka et al.

Science 321, 1670 (2008);

DOI: 10.1126/science.1160446

---

This copy is for your personal, non-commercial use only.

---

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by clicking here.

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines here.

The following resources related to this article are available online at www.sciencemag.org (this information is current as of January 15, 2013):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:
http://www.sciencemag.org/content/321/5896/1670.full.html

Supporting Online Material can be found at:
http://www.sciencemag.org/content/suppl/2008/09/18/321.5896.1670.DC1.html

A list of selected additional articles on the Science Web sites related to this article can be found at:
http://www.sciencemag.org/content/321/5896/1670.full.html#related

This article cites 21 articles, 5 of which can be accessed free:
http://www.sciencemag.org/content/321/5896/1670.full.html#ref-list-1

This article has been cited by 15 article(s) on the ISI Web of Science

This article has been cited by 11 articles hosted by HighWire Press; see:
http://www.sciencemag.org/content/321/5896/1670.full.html#related-urls

This article appears in the following subject collections:

Microbiology
http://www.sciencemag.org/cgi/collection/microbio

---

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2008 by the American Association for the Advancement of Science; all rights reserved. The title Science is a registered trademark of AAAS.

RESEARCH

6. G. E. Marcus, W. R. Neuman, M. Mackuen, *Affective Intelligence and Political Judgment* (Univ. of Chicago Press, Chicago, 2000).

7. R. McDermott, *Perspect. Polit.* **2**, 691 (2004).

8. D. Westen, *The Political Brain* (Public Affairs, New York, 2007).

9. M. J. Landau *et al.*, *Pers. Soc. Psychol. Bull.* **30**, 1136 (2004).

10. S. Fahmy, S. Cho, W. Wanta, Y. Song, *Vis. Commun. Q.* **13**, 3 (2006).

11. J. Faulkner, M. Schaller, J. H. Park, L. A. Duncan, *Group Process. Intergroup Relat.* **7**, 333 (2004).

12. C. D. Navarrete, D. M. T. Fessler, *Evol. Hum. Behav.* **27**, 270 (2006).

13. D. M. Amodio, J. T. Jost, S. L. Master, C. M. Lee, *Nat. Neurosci.* **10**, 1246 (2007).

14. J. T. Jost, J. Glaser, A. W. Kruglanski, F. J. Sulloway, *Psychol. Bull.* **129**, 339 (2003).

15. J. T. Jost, *Am. Psychol.* **61**, 651 (2006).

16. L. Huddy, S. Feldman, C. Taber, G. Lahav, *Am. J. Pol. Sci.* **49**, 593 (2005).

17. S. Feldman, *Pol. Psychol.* **24**, 593 (2003).

18. K. Stenner, *The Authoritarian Dynamic* (Cambridge Univ. Press, New York, 2005).

19. F. Pratto, J. Sidanius, L. M. Stallworth, B. F. Malle, *J. Pers. Soc. Psychol.* **67**, 741 (1994).

20. W. B. Cannon, *Bodily Changes in Pain, Hunger, Fear, and Rage* (Appleton, New York, 1915).

21. M. M. Bradley, P. J. Lang, in *Handbook of Psychophysiology*, J. T. Cacioppo, L. G. Tassinary, G. G. Berntson, Eds. (Cambridge Univ. Press, New York, 2007).

22. M. E. Thase, R. H. Howland, in *Handbook of Depression*, E. E. Beckham and W. R. Leber, Eds. (Guilford, New York, 1995).

23. E. Lemche *et al.*, *Hum. Brain Mapp.* **27**, 623 (2006).

24. G. H. Grosser, H. Wechser, M. Greenblatt, *The Threat of Impending Disaster* (MIT Press, Cambridge, MA, 1971).

25. Materials and methods are described in the SOM.

26. M. E. Dawson, A. M. Shell, D. L. Filion, in *Handbook of Psychophysiology*, J. T. Cacioppo, L. G. Tassinary, G. G. Berntson, Eds. (Cambridge Univ. Press, New York, 2007).

27. A. Miller, J. Long, in *Developmental Psychophysiology*, L. A. Schmidt, S. J. Segalowitz, Eds. (Cambridge Univ. Press, New York, 2007).

28. P. J. Lang, M. M. Bradley, B. N. Cuthbert, *Psychol. Rev.* **97**, 377 (1990).

29. G. D. Wilson, J. R. Patterson, *Br. J. Soc. Clin. Psychol.* **7**, 264 (1968).

30. S. Anders, M. Lotze, M. Erb, W. Grodd, *Hum. Brain Mapp.* **23**, 200 (2004).

31. C. L. Larson *et al.*, *Biol. Psychiatry* **60**, 410 (2006).

32. D. A. Fitzgerald, M. Angstad, L. M. Jelsone, P. J. Nathan, K. L. Phan, *Neuroimage* **30**, 1441 (2006).

33. N. G. Martin *et al.*, *Proc. Natl. Acad. Sci. U.S.A.* **83**, 4364 (1986).

34. L. Eaves *et al.*, *Twin Res.* **2**, 62 (1999).

35. J. R. Alford, C. L. Funk, J. R. Hibbing, *Am. Polit. Sci. Rev.* **99**, 153 (2005).

36. J. H. Fowler, L. A. Baker, C. T. Dawes, *Am. Polit. Sci. Rev.* **102**, 233 (2008).

37. Z. F. Mainen, *Nat. Neurosci.* **10**, 1511 (2007).

38. H. Bracha, D. Yoshioka, N. Masakawa, D. Stockman, *J. Affect. Disord.* **88**, 119 (2005).

39. C. A. Ponder *et al.*, T. C. Gilliam, A. A. Palmer, *Genes Brain Behav.* **6**, 736 (2007).

40. A. R. Hariri *et al.*, *Science* **297**, 400 (2002).

41. We thank E. Whitaker, C. Jacobs, B. Sexton, K. A. Espy, J. Brehm, D. Bulling, and the James Long Company for their invaluable assistance. Financial support was provided by the NSF (SES-0721378 and SES-0721707), the ManTech Corporation, and the University of Nebraska–Lincoln’s Strategic Research Cluster Grant program.

Supporting Online Material  
www.sciencemag.org/cgi/content/full/321/5896/1667/DC1  
Materials and Methods  
SOM Text  
Fig. S1  
Tables S1 to S6  
Appendix 1  

11 March 2008; accepted 18 August 2008  
10.1126/science.1157627


---

**An Alternative Menaquinone Biosynthetic Pathway Operating in Microorganisms**

Tomoshige Hiratsuka,$^{1}$ Kazuo Furihata,$^{2}$ Jun Ishikawa,$^{3}$ Haruyuki Yamashita,$^{4}$ Nobuya Itoh,$^{1}$ Haruo Seto,$^{5}$ Tohru Dairi$^{1\ast}$

In microorganisms, menaquinone is an obligatory component of the electron-transfer pathway. It is derived from chorismate by seven enzymes in *Escherichia coli*. However, a bioinformatic analysis of whole genome sequences has suggested that some microorganisms, including pathogenic species such as *Helicobacter pylori* and *Campylobacter jejuni*, do not have orthologs of the *men* genes, even though they synthesize menaquinone. We deduced the outline of this alternative pathway in a nonpathogenic strain of *Streptomyces* by bioinformatic screening, gene knockouts, shotgun cloning with isolated mutants, and in vitro studies with recombinant enzymes. As humans and commensal intestinal bacteria, including lactobacilli, lack this pathway, it represents an attractive target for the development of chemotherapeutics.

In prokaryotes, ubiquinone and menaquinone (MK) are lipid-soluble molecules that shuttle electrons between the membrane-bound protein complexes in the electron-transport chain (1, 2). For example, the facultative anaerobe *Escherichia coli* uses ubiquinone (CoQ-8) under aerobic conditions but uses MK 8 when it is grown anaerobically. By contrast, many Gram-positive aerobes such as *Bacillus subtilis* contain only MKs. MK biosynthesis is therefore essential for the survival of these strains. In mammalian cells, ubiquinone plays a role in the electron-transport chain in the inner mitochondrial membrane, and MK functions as an essential vitamin for the biological activation of a family of proteins involved in blood coagulation (3), bone metabolism (4), and cell-cycle regulation (5). The biosynthesis of MK had been mainly studied in *E. coli*. In this organism, chorismate, which is derived from the shikimate pathway, is converted into MK by seven enzymes (MenA to MenG, Fig. 1). Although humans lack this pathway, essential amounts of MK are normally supplied in the diet.

There is no trace of *menF*, *menD*, *menC*, *menE*, and *menB* gene orthologs in the genome of *Streptomyces coelicolor* A3(2) (6–8), even though it produces MKs. Similarly, some pathogens that synthesize MK, including *Helicobacter pylori* and *Campylobacter jejuni*, have also been reported to lack *men* gene homologs (9–12). We performed a tracer experiment with *S. coelicolor* A3(2) and [U-¹³C₆]glucose to test whether an alternative pathway to MK operated in the strain. We found that the labeling patterns of MK differed from those of the classical pathway and that 1,4-naphthoquinone-6-carboxylic acid (Fig. 1) (or its reduced form, 1,4-dihydroxy-6-naphthoate) was an intermediate, which suggests that MK is indeed biosynthesized by an alternative route in this species (13). We have identified the genes, enzymes, and biosynthetic intermediates responsible for this alternative pathway, which we have named the futalosine pathway.

We started our investigations by screening genome databases, as well as mutants that require MK for growth. We found that some microorganisms in the epsilon categories of Proteobacteria, Actinobacteria, and the *Deinococcus-Thermus* bacteria groups lacked *men* gene orthologs despite the fact that most of these strains are known to synthesize MKs. From among these, we selected four microorganisms for further analysis: *H. pylori*, *C. jejuni*, *Thermus thermophilus* (14) and *S. coelicolor*. We also made comparisons with microorganisms in which the known MK biosynthetic pathway operates, including *E. coli* (15), *Bacillus subtilis* (16), *Corynebacterium glutamicum* (17), and *Mycobacterium tuberculosis* (18). To find candidate genes, we first estimated orthologous genes as reciprocal best-hit pairs using the BLAST (Basic Local Alignment Search Tool) program (19) with a cutoff *e* value < 10⁻¹⁰ and then searched for candidate genes present in the *men* negative group but absent in the *men* positive group. We eventually identified ~50 candidate genes in *S. coelicolor* A3(2). Putative transcriptional regulators and membrane proteins such as adenosine triphosphate–binding cassette transporters that are known to transport a

---

1 Biotechnology Research Center, Toyama Prefectural University, Toyama 939-0398, Japan. 2 Division of Agriculture and Agricultural Life Science, The University of Tokyo, Bunkyo-ku, Tokyo 113-8657 Japan. 3 Department of Bioactive Molecules, National Institute of Infectious Diseases, Shinjuku-ku, Tokyo 162-8640, Japan. 4 Advanced Materials Research and Development Laboratory, ADEKA Corporation, Arakawa-ku, Tokyo 116-8553, Japan. 5 Faculty of Applied Bioscience, Tokyo University of Agriculture, Setagaya-ku, Tokyo 156-8502, Japan.

*To whom correspondence should be addressed. E-mail: dairi@pu-toyama.ac.jp

wide variety of metabolic compounds were excluded, leaving four candidates: SCO4326, SCO4327, SCO4506, and SCO4550. The products of these genes had been annotated as hypothetical proteins.

In a series of disruption experiments in *S. coelicolor* A3(2), each candidate gene was replaced with a thiostrepton-resistance gene by double-crossover homologous recombination (20). We knocked out all of the candidate genes with agar plates containing MK 4 (having a C20 prenyl side chain), which was commercially available and was substituted for the MK 8 (having a C40 prenyl side chain) usually found in *Streptomyces* strains. Disruption of each of the mutants was confirmed by polymerase chain reaction (PCR) analysis (Fig. 2 and fig. S1). The mutants disrupted at SCO4326, SCO4327, SCO4506, and SCO4550 required MK 4 for their growth. The MKs purified from the mutants were confirmed as MK 4 by high-performance liquid chromatography (HPLC) (Fig. 3 and fig. S2). We also confirmed that the phenotypes of the mutants did not reflect polar effects, a phenomenon in which a disruption of the gene located upstream affected the expression of the downstream gene when these genes constituted an operon (21).

To identify the complete set of genes in the alternative pathway, we used N-methyl-N-nitro-N-nitrosoguanidine mutagenesis to obtain mutants that required MK 4 for their growth. Twenty-six mutants were obtained, of which 14 were complemented by plasmids carrying genes we had identified from bioinformatic screening. Shotgun cloning experiments were performed with the other 12 mutants as hosts. Eventually, we succeeded in complementing the mutants by a plasmid carrying the SCO1494, SCO1495, and SCO1496 genes, which encode 3-dehydroquinate synthase, shikimate kinase I, and chorismate synthase, respectively. The mutants were then grown on medium containing shikimate, chorismate, or o-succinylbenzoate (an intermediate in the classical pathway) instead of MK 4, but growth was seen only on chorismate (Fig. 4), suggesting that they lacked functional chorismate synthase or shikimate kinase. Because the reaction catalyzed by chorismate synthase is irreversible (22), it appeared that the alternative pathway branches at chorismate.

Cocultivation revealed that each of the four mutants could grow together with the others, even in the absence of MK 4, suggesting that mutagenesis had blocked different points in the MK biosynthetic pathway and that intermediates had accumulated in the culture broth that allowed

![Figure 1](https://www.science.org/cms/asset/0a0f8d1c-2e0b-4f8a-bf8a-0a0f8d1c2e0b/112611671.fig1.jpg)

**Fig. 1.** MK biosynthetic pathways. (A) Classical pathway. Chorismate, which is derived from the shikimate pathway, is initially converted into isochorismate by MenF, isochorismate synthase, and then into 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate by MenD, which is a thiamine-dependent enzyme. This compound is dehydrated by MenC to give an aromatic compound, o-succinylbenzoate, followed by the attachment of coenzyme A by MenE. o-Succinylbenzoyl-CoA is then converted into 1,4-dihydroxy-2-naphthoate by MenB. In the last two steps of the pathway, MK is synthesized by MenA and MenG, which catalyze prenylation and S-adenosylmethionine-dependent methylation, respectively. Red and black bold lines show carbons originated from erythrose-4-phosphate and phosphoenolpyruvate, respectively. (B) Alternative pathway. Green and blue bold lines indicate two carbon units derived from C-5 and C-6 of glucose via different metabolic pathways. Based on the annotation of the open reading frames of *S. coelicolor* A3(2), we presumed that SCO4491 (prenylation), SCO4556 (methylation), SCO4490 (decarboxylation), and SCO4492 (decarboxylation) would be involved in the late step of the MK biosynthetic pathway, although we did not have any direct evidence.

the mutants to compensate. We then cultivated each mutant in the presence of MK 4, the broth was centrifuged, and the supernatant was concentrated in vacuo. After MK 4 was removed by ethyl acetate extraction, the concentrated aqueous layer was added to an agar plate, and growth of the other mutants on this medium was examined (fig. S3). Our results indicated that the blocked points in the MK biosynthesis of the mutants occurred in the following order: SCO4506, SCO4327, SCO4550, and SCO4326.

To purify the intermediates, we first used the SCO4506 and SCO4327 mutants as an intermediate converter and a secretor, respectively. The intermediate was extracted as above, and nuclear magnetic resonance (NMR) and mass spectrometry (MS) (figs. S4 and S5) confirmed the compound as futalosine (Fig. 1). This compound has previously been isolated from the culture broth of a Streptomyces strain (23). Because the SCO4327 protein has weak similarities to some nucleosidases, we expected it to release hypoxanthine from futalosine. We prepared a SCO4327 recombinant for an in vitro assay; however, no products were formed. It was unclear why the recombinant lacked enzymatic activity, but we assumed that it might be unstable. Hence, we prepared an ortholog of SCO4327 in the extreme thermophile *T. thermophilus* HB8 (TTHA0556 recombinant) (fig. S6), expecting it to be highly thermostable. When the recombinant TTHA0556 was incubated with futalosine, two reaction products were generated. One (fig. S7, peak B) was confirmed as hypoxanthine by HPLC mass spectral analysis (LC-MS). The other (fig. S7, peak C) was purified by reverse-phase preparative HPLC, and its structure was determined by infrared (IR) spectral, MS, and NMR analyses (figs. S8 to S11) as dehypoxanthine (dehypoxanthyl) futalosine (DHFL) (Fig. 1). This was confirmed as an intermediate in the alternative pathway by a bioassay using the

**Fig. 2. PCR analyses of genomic DNAs of the SCO4506-disrupted, SCO4326-disrupted, SCO46327-disrupted, and SCO4550-disrupted strains.** Genomic DNAs were prepared from the wild-type strain and the disruptants and used for templates for PCR analyses. The amplified fragments were analyzed by agarose gel electrophoresis as follows: molecular marker (lanes 1 and 10); wild-type (lanes 3, 5, 7, and 9); SCO4326-disruptant (lane 2); SCO4327-disruptant (lane 4); SCO4306-disruptant (lane 6); and SCO4550-disruptant (lane 8). The sizes of amplified fragments are schematically shown in fig. S1.

**Fig. 3. HPLC analyses of MKs extracted from disruptants.** The disruptants were cultivated in the presence of MK4 (having a C20 prenyl side chain). After cultivation, the MKs were extracted with ethyl acetate and evaporated in vacuo. The obtained materials were analyzed by reverse-phase HPLC. The wild-type (A) and the SCO4506-disruptant (B) are shown as examples. The SCO4506-disruptant contained only MK4 added into the medium, in contrast to the wild type that had MK8 usually found in *S. coelicolor*. HPLC analyses of MKs extracted from the other disruptants were shown in fig. S2.

**Fig. 4. One of 26 mutants obtained by mutagenic treatment required chorismate for its growth.** The mutant was spread onto a yeast extract–malt extract (YEME) plate and a YEME plate containing MK 4, chorismate, shikimate, or o-succinylbenzoate. After cultivation at 30°C for 1 week, photographic images were taken of each plate. The mutant could grow only in the presence of chorismate, which suggests that it lacked functional chorismate synthase or shikimate kinase. Because the reaction catalyzed by the former enzyme is irreversible, the alternative pathway branches at chorismate.

SCO4506- and SCO4327-disruptants as converters (fig. S12).

In the next step, we isolated an intermediate from the culture broth of the SCO4326 mutant, which was purified as above and its structure determined as cyclic DHFL (Fig. 1) (figs. S13 to S17).

As a small amount of DHFL accumulated in the culture broth of the SCO4550-disruptant, we predicted that it catalyzed the reaction adjacent to SCO4327. An in vitro enzyme assay was prepared using the recombinant TTHA1092, which contained an ortholog of SCO4550 in *T. thermophilus* HB8. However, we did not detect the formation of cyclic DHFL. Additional enzymes and/or cofactors, and optimized assay conditions, may be necessary.

Recombinant TTHA1568 was prepared to provide an ortholog of SCO4326 in *T. thermophilus* HB8 to convert cyclic DHFL into the next intermediate. A product was obtained and was confirmed to be 1,4-dihydroxy-6-naphthoate by LC-MS (fig. S18); this was previously demonstrated to be an intermediate in the alternative pathway (13).

Because the alternative pathway appeared to branch at chorismate with the SCO4506 enzyme, and an examination of the structure of futilosine indicates that the nucleoside moiety is derived from inosine, plus the fact that a tracer experiment (13) has implicated a C2 unit from pyruvate or phosphoenolpyruvate between chorismate and inosine forming the C6' and C7' positions of futilosine, we tested the ability of the recombinant enzyme TTHA0803 (an ortholog of SCO4506 in *T. thermophilus* HB8) to catalyze the formation of futilosine. However, we did not detect futilosine, although 3-(1-carboxyvinyl oxy) benzoate and m-

hydroxybenzoate were formed from chorismate in the absence and presence of flavin mononucleotide (FMN), respectively.

We have succeeded in outlining an alternative pathway for the biosynthesis of MK in microorganisms by a combination of bioinformatics and biochemical experiments. We confirmed the occurrence of the alternative pathway among bacteria and found it (see SOM Text) in several Gram negatives and Archaea, notably in chlamydia and spirochetes. The alternative pathway was distributed only in prokaryotes and was absent in eukaryotes, including lower ones such as fungi, yeasts, and protists. Because humans and commensals, such as lactobacilli, lack this alternative pathway, it is an attractive target for the development of chemotherapeutics. We also searched microorganisms that have both the alternative pathway and the classical pathway or that have both the ubiquinone pathway and the alternative pathway. However, we were not able to find such bacteria among microorganisms whose genome analysis has been completed.

### References and Notes

1. R. Bentley, R. Maganathan, *Microbiol. Rev.* **46**, 241 (1982).
2. R. Meganathan, *Vitam. Horm.* **61**, 173 (2001).
3. E. C. Cranenburg, L. J. Schurgers, C. Vermeer, *Thromb. Haemost.* **98**, 120 (2007).
4. S. M. Plaza, D. W. Lamson, *Altern. Med. Rev.* **10**, 24 (2005).
5. D. W. Lamson, S. M. Plaza, *Altern. Med. Rev.* **8**, 303 (2003).
6. S. D. Bentley et al., *Nature* **417**, 141 (2002).
7. I. Borodina, P. Krabben, J. Nielsen, *Genome Res.* **15**, 820 (2005).
8. M. D. Collins et al., *J. Gen. Microbiol.* **100**, 221 (1977).
9. J. F. Tomb et al., *Nature* **388**, 539 (1997).
10. J. Parkhill et al., *Nature* **403**, 665 (2000).
11. S. W. Marcelli et al., *FEMS Microbiol. Lett.* **138**, 59 (1996).
12. C. W. Moss et al., *J. Clin. Microbiol.* **28**, 395 (1990).
13. H. Seto et al., *J. Am. Chem. Soc.* **130**, 5614 (2008).
14. A. Henne et al., *Nat. Biotechnol.* **22**, 547 (2004).
15. F. R. Blattner et al., *Science* **277**, 1453 (1997).
16. F. Kunst et al., *Nature* **390**, 249 (1997).
17. M. Ikeda, S. Nakagawa, *Appl. Microbiol. Biotechnol.* **62**, 99 (2003).
18. S. T. Cole et al., *Nature* **393**, 537 (1998).
19. S. F. Altschul et al., *J. Mol. Biol.* **215**, 403 (1990).
20. D. A. Hopwood et al., Gene Manipulation of Streptomyces, a Laboratory Manual (John Innes Foundation, Norwich, UK, 1985).
21. Materials and methods are available as supporting material on Science Online.
22. P. Macheroux et al., *Planta* **207**, 325 (1999).
23. N. Hosokawa et al., *Chem. Pharm. Bull. (Tokyo)* **47**, 1032 (1999).
24. This work was supported in part by a Grant-in-Aid for Scientific Research (B) to S.H. and by a Grant-in-Aid for Scientific Research on Priority Areas “Applied Genomics” from the Ministry of Education, Culture, Sports, Science and Technology of Japan, the Urakami Foundation, and the Skylark Food Science Institute to T.D. Author contributions: T.H., H.S., and T.D. conceived and designed the experiments; T.H., K.F., J.I., H.S., and T.D. performed the experiments; T.H., K.F., H.Y., N.I., H.S., and T.D. analyzed the data; and H.S. and T.D. wrote the paper. The DNA sequences determined and characterized in this study have been deposited in the DDBJ, EMBL, and GenBank data banks as follows: SCO4506, AB447888; SCO4327, AB447889; SCO4550, AB447890; SCO4326, AB447891; TTHA0556, AB447892; TTHA1568, AB447893.

---

**Supporting Online Material**

www.sciencemag.org/cgi/content/full/321/5896/1670/DC1

Materials and Methods  
SOM Text  
Figs. S1 to S18  
Tables S1 to S4  

13 May 2008; accepted 24 July 2008  
10.1126/science.1160446

---

### An Inhibitor of FtsZ with Potent and Selective Anti-Staphylococcal Activity

David J. Haydon,$^{1}$ Neil R. Stokes,$^{1\ast}$ Rebecca Ure,$^{1}$ Greta Galbraith,$^{1}$ James M. Bennett,$^{1}$ David R. Brown,$^{1}$ Patrick J. Baker,$^{2}$ Vladimir V. Barynin,$^{2}$ David W. Rice,$^{2}$ Sveta E. Sedelnikova,$^{2}$ Jonathan R. Heal,$^{3}$ Joseph M. Sheridan,$^{3}$ Sachin T. Aiwale,$^{4}$ Pramod K. Chauhan,$^{4}$ Anil Srivastava,$^{4}$ Amit Taneja,$^{4}$ Ian Collins,$^{1}$ Jeff Errington,$^{1,5}$ Lloyd G. Czaplewski$^{1}$

FtsZ is an essential bacterial guanosine triphosphatase and homolog of mammalian β-tubulin that polymerizes and assembles into a ring to initiate cell division. We have created a class of small synthetic antibacterials, exemplified by PC190723, which inhibits FtsZ and prevents cell division. PC190723 has potent and selective in vitro bactericidal activity against staphylococci, including methicillin- and multi-drug-resistant *Staphylococcus aureus*. The putative inhibitor-binding site of PC190723 was mapped to a region of FtsZ that is analogous to the Taxol-binding site of tubulin. PC190723 was efficacious in an in vivo model of infection, curing mice infected with a lethal dose of *S. aureus*. The data validate FtsZ as a target for antibacterial intervention and identify PC190723 as suitable for optimization into a new anti-staphylococcal therapy.

Bacterial cell-division proteins, in particular FtsZ, offer unexploited and attractive targets for antibacterial drug discovery because they are essential and widely conserved throughout bacteria (1, 2). During cell division,

FtsZ undergoes guanosine 5′-triphosphate (GTP)-dependent polymerization to form the Z ring at the mid-cell. FtsZ recruits other proteins that together drive cell division and the formation of new cell poles (3). FtsZ is a distant structural and functional relative of mammalian β-tubulin (4–6), which is a well-exploited target for anticancer therapy (6). This suggests that the protein superfamily is amenable to inhibitor development and may provide structural insights to assist the rational design of FtsZ ligands. Several compounds that block bacterial cell division and/or inhibit the biochemical activity of the FtsZ protein in vitro (7–13) have been reported; however, so far none has demonstrated efficacy in models of infection or has entered clinical evaluation or use.

3-Methoxybenzamide (3-MBA) (C₈H₉NO₂, relative molecular mass of 151.16) (fig. S1) is a small ligand that has been reported to target FtsZ and inhibit cell division in the Gram-positive bacterium *Bacillus subtilis*, resulting in a filamentous morphology (7). Although 3-MBA has

1Prolysis, Begbroke Science Park, Oxfordshire OX5 1PF, UK.  
2Department of Molecular Biology and Biotechnology, University of Sheffield, Sheffield S10 2TN, UK.  
3Prosarix, Newton Hall, Cambridge CB22 7ZE, UK.  
4Jubilant Chemsys, B-34, Sector-58, Noida 201301, India.  
5Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne NE2 4HH, UK.

*To whom correspondence should be addressed. E-mail: neil.stokes@prolysis.com
