
PAPER

**The neomycin biosynthetic gene cluster of *Streptomyces fradiae* NCIMB 8233: genetic and biochemical evidence for the roles of two glycosyltransferases and a deacetylase†**

Qingzhi Fan,$^{a,b}$ Fanglu Huang,$^{b}$ Peter F. Leadlay*$^{a}$* and Jonathan B. Spencer‡$^{b}$

Received 23rd May 2008, Accepted 20th June 2008  
First published as an Advance Article on the web 25th July 2008  
DOI:10.1039/b808734b

An efficient protocol has been developed for the genetic manipulation of *Streptomyces fradiae* NCIMB 8233, which produces the 2-deoxystreptamine (2-DOS)-containing aminoglycoside antibiotic neomycin. This has allowed the in vivo analysis of the respective roles of the glycosyltransferases Neo8 and Neo15, and of the deacetylase Neo16 in neomycin biosynthesis. Specific deletion of each of the neo8, neo15 and neo16 genes confirmed that they are all essential for neomycin biosynthesis. The pattern of metabolites produced by feeding putative pathway intermediates to these mutants provided unambiguous support for a scheme in which Neo8 and Neo15, whose three-dimensional structures are predicted to be highly similar, have distinct roles: Neo8 catalyses transfer of N-acetylglucosamine to 2-DOS early in the pathway, while Neo15 catalyses transfer of the same aminosugar to ribostamycin later in the pathway. The in vitro substrate specificity of Neo15, purified from recombinant *Escherichia coli*, was fully consistent with these findings. The in vitro activity of Neo16, the only deacetylase so far recognised in the neo gene cluster, showed that it is capable of acting in tandem with both Neo8 and Neo15 as previously proposed. However, the deacetylation of N-acetylglucosaminylribostamycin was still observed in a strain deleted of the neo16 gene and fed with suitable pathway precursors, providing evidence for the existence of a second enzyme in *S. fradiae* with this activity.

---

### Introduction

2-Deoxystreptamine (2-DOS)-containing antibiotics¹² comprise the largest subgroup of the aminoglycoside class of antimicrobial compounds, which includes the clinically-useful gentamicin and neomycin. Neomycin, first isolated from *Streptomyces fradiae* and *Streptomyces albus* in 1949,³ is a broad-spectrum antibiotic active against both Gram-positive and Gram-negative bacteria. Like most aminoglycosides, it targets the bacterial 30S ribosomal subunit and inhibits protein synthesis by inducing codon misreading and interfering with initiation and translocation.⁴⁵ The aminoglycosides are significantly nephrotoxic and ototoxic and are therefore normally reserved for treatment of deadly infections such as those caused by *Mycobacterium tuberculosis* and methicillin-resistant *Staphylococcus aureus* (MRSA). More recent work has indicated the potential of neomycin and its analogues as a new class of anti-HIV agents,⁶⁷ and the demonstration of their in vitro anti-angiogenesis activity has led to their evaluation in anticancer drug development.⁸

As with other broad-spectrum antibiotics, the emergence among human pathogens of antibiotic resistance to aminoglycosides has blunted their clinical effectiveness. Encouragingly, chemical modification of the structure has in several cases produced valuable semisynthetic aminoglycosides such as amikacin⁹ that circumvent such resistance mechanisms. This has prompted increasing interest in the possibility of producing novel aminoglycoside antibiotics by engineering of the biosynthetic pathway. For this approach to be applied effectively, we will require a much more detailed understanding of the genes and enzymes involved.

Neomycin obtained from fermentation comprises a mixture of three structurally-related compounds: neomycin A (usually referred to as neamine), neomycin B and neomycin C (Fig. 1). Neamine consists of 2,6-diamino-2,6-dideoxy-D-glucose (neosamine C, ring II) attached by a glycosidic linkage to the aminocyclitol aglycone 2-deoxystreptamine (2-DOS, ring I). Neomycin B has a neamine core to which 2,6-diamino-2,6-dideoxy-L-idose (neosamine B, ring IV) and D-ribose (ring III) are attached via ring I. Neomycin C and neomycin B are identical except for the different configuration of the aminomethyl group at C5" in ring IV. The paromomycins, which are produced by

![Figure 1](#fig1)

**Fig. 1** Structure and ring numbering of neomycins and paromomycins.

---

$a$ Department of Biochemistry, University of Cambridge, 80 Tennis Court Road, Cambridge, CB2 1GA, United Kingdom. E-mail: pfl10@mole.bio.cam.ac.uk; Fax: +44 1223 766002; Tel: +44 1223 766041  
$b$ University Chemical Laboratory, University Chemical Laboratory, Lensfield Road, Cambridge, CB2 1EW, United Kingdom  
† Electronic supplementary information (ESI) available: Fig. S1–S4; Table S1. See DOI: 10.1039/b808734b  
‡ Deceased 6th April 2008.

Streptomyces rimosus, ${}^{10}$ have been reported as minor components of the fermentation in $S$. fradiae. ${}^{11}$

Extensive early studies on the biosynthesis of the 2-DOS-derived aminoglycosides, including the isolation of compounds from blocked mutants of $S$. fradiae, and the feeding of labelled putative precursors to 2-DOS idiotrophs, have strongly implicated paromamine, neamine and ribostamycin as intermediates in the formation of neomycin, and provided the first evidence for the order of assembly. ${}^{12}$ More recently, the biosynthetic gene cluster for neomycin has been cloned, by four research groups independently, from $S$. fradiae NCIMB 8233 (or the equivalent strain deposited elsewhere under other reference numbers) and sequenced (accession numbers: AJ843080, ${}^{12}$ AJ629247, AJ786317 and AB211959 ${}^{13}$). This information, together with comparison of the gene organisation with that of biosynthetic gene clusters for other aminoglycosides, has allowed a detailed biosynthetic pathway to be proposed. ${}^{12,13}$ The enzymes responsible for formation of the aglycone 2-DOS, in four steps from D-glucose-6-phosphate, have been fully characterised *in vitro*. ${}^{12-14}$ In contrast, many of the details of the proposed pathway from 2-DOS to neomycin (Scheme 1) remain to be clarified. For example, Neo16 (NeoL), which shows significant sequence similarity to BtrD in the butirosin gene cluster, was initially suggested to provide NDP-glucose/NDP-glucosamine sugar donors for glycosylation of 2-DOS to form paromamine. ${}^{15}$ However, as summarised in Scheme 1, Neo16 is now thought instead to act as a deacetylase acting to remove the acetyl group on $N$-acetylglucosamine moieties of aminoglycoside intermediates, based on its significant sequence similarity to BtrD which has also been recently unveiled as a deacetylase. ${}^{16}$ Given this insight, it would appear that neomycin biosynthesis from 2-DOS requires two distinct cycles of transfer of an $N$-acetyl-$D$-glucosamine (GlcNAc) from UDP-$N$-acetylglucosamine (UDP-GlcNAc) (to 2-DOS in cycle 1 and to ribostamycin in cycle 2) followed by deacetylation, and C6 modification of the newly-introduced sugar (Scheme 1). This raises the intriguing question of whether different enzymes carry out the comparable step in each cycle, or whether a single enzyme has the requisite dual specificity. Recombinant Neo8 and Neo15 were recently reported to catalyse *in vitro* the specific transfer of GlcNAc to 2-DOS and to ribostamycin respectively, while Neo16 *in vitro* was reported to be active in deacetylation of both $N$-acetylparomamine and 2″-$N$-acetyl-6″-hydroxyneomycin ($N'$-acetylglucosaminylribostamycin). ${}^{17}$ We report here our own independent findings on the specificity of Neo15 and Neo16 *in vitro*, as well as the development of an efficient method for genetic manipulation of the neomycin-producing $S$. fradiae. This has allowed us to demonstrate by an *in vivo* approach that the two glycosyltransferases Neo8 and Neo15 must act specifically and in that order, at different points in the pathway. In contrast, Neo16 is sufficient *in vitro* to accomplish deacetylation in both cycles, but it is not essential *in vivo* for the second cycle of deacetylation. This provides the first

![Diagram](attachment://diagram.png)

**Scheme 1** Proposed biosynthetic pathway of neomycin C. PRPP: 5-phosphoribosyl-1-diphosphate.

evidence for overlapping enzymatic activities in aminoglycoside biosynthesis.

# Results and discussion

## Attachment of N-acetyl-D-glucosamine to 2-deoxystreptamine

Since the early observation¹⁸ that a mutant of *S. fradiae* could be isolated which only produced ribostamycin, it has been considered likely that the two glycosylation steps in neomycin biosynthesis require two independent glycosyltransferases. However, initial inspection of the proteins encoded by the neomycin gene cluster only readily identified Neo8 (NeoD) as a candidate glycosyltransferase catalysing one or both of the required transfers of an aminoheose in neomycin biosynthesis.¹²,¹³ Neo8 has homologues in many aminoglycoside biosynthetic gene clusters that share high percentage amino acid sequence identities: most strikingly RibM (ribostamycin, 90%); but also LivM (lividomycin, 72%), ParM (paromomycin, 69%), TobM1 (tobramycin, 64%), KanM1 (kanamycin, 63%), AprM (apramycin, 61%) and GenM1 (gentamicin, 54%). All of these aminoglycoside pathways require attachment of at least one hexosamine sugar.

The *neo8* gene of *S. fradiae* was knocked out by targeted in-frame deletion, which removed 840 bp of the gene (out of a total of 1185 bp). The identity of the mutant was confirmed by PCR screening, followed by sequencing of the PCR products (Fig. S1-b). The mutant, Δ*neo8*, failed to produce neomycin. However, Δ*neo8* did produce neomycin when the medium was supplemented with neamine (0.1 mg mL⁻¹) or ribostamycin (0.2 mg mL⁻¹) (Fig. S1-d and Table S1) but not when the medium was supplemented with 2-DOS (0.1 mg mL⁻¹). This provides direct evidence that Neo8 is essential for the first transfer of UDP-GlcNAc but not for the second. Feeding of paromomycin (0.2 mg mL⁻¹) led, interestingly, to the production of paromomycin in good yield (Fig. S1-e and Table S1), accompanied by minor amounts of neomycin. This unusual production of paromomycin likely reflects a broad substrate specificity of the downstream enzymes, especially those involved in ribosylation. In agreement with this, it has previously been shown that BtrL, the homologue of Neo17 in butirosin biosynthesis, acts as a phosphoribose transferase *in vitro* on both neamine and paromamine substrates.¹⁹ In the *S. fradiae* Δ*neo8* mutant, Neo17 apparently competes effectively for the added paromamine against the dehydrogenase and aminotransferase enzymes Neo11 and Neo18, which normally convert paromamine to neamine. As a control, ribostamycin and paromamine were also fed to wild-type *S. fradiae*, and in each case there was an approximately three-fold decrease in neomycin production. Production of paromomycin was again observed with the addition of paromamine.

The Δ*neo8* mutant was complemented by introduction of a plasmid (pQZ53) in which *neo8* is under the control of the constitutively-expressed *ermE* promoter; or alternatively by introduction of a plasmid (pQZ60) in which *neo8* is under the control of its natural promoter. In either case, production of neomycin was restored (Fig. S1-c), albeit at a reduced level compared to wild-type, confirming that the loss of neomycin production in Δ*neo8* was not caused by an unintended polar effect on the expression of any gene downstream of *neo8*.

Heterologous expression of the Neo8 protein was also attempted, in order to test its substrate specificity *in vitro*. Neo8 was expressed using the vector pET28a(+) in *E. coli* and with plasmid pCJW93²⁰ in *Streptomyces lividans*. The protein was expressed poorly, and was found to be insoluble under a wide range of conditions tried. While this manuscript was in preparation, it was reported that Neo8 (NeoD) could be successfully co-expressed with *E. coli* chaperone proteins GroES and GroEL.¹⁷ Although the enzyme could not be purified, its activity in the *E. coli* lysate revealed an apparent specificity for 2-DOS as the acceptor of a GlcNAc group, while ribostamycin, neamine and paromamine were not substrates.¹⁷ Thus, *in vitro* and *in vivo* data are in agreement in suggesting that a distinct, second glycosyltransferase is required to transfer a GlcNAc group to ribostamycin.

## Attachment of N-acetyl-D-glucosamine to ribostamycin

Further, extensive BLAST searches against the open reading frames in the neomycin biosynthetic cluster¹² revealed Neo15 as a second candidate hexosaminyltransferase. Its closest three potential homologues in the available gene clusters of aminoglycosides are RibF (ribostamycin, 77%), LivF (lividomycin, 65%) and ParF (paromomycin, 60%). Both lividomycin and paromomycin possess the same ring IV as neomycin, a further hint that Neo15 could be the glycosyltransferase responsible for its addition. The *neo15* gene was knocked out by an in-frame deletion, which removed 759 bp of the gene (out of a total of 1101 bp). The mutation was confirmed by PCR screening and by sequencing of the PCR products (Fig. S2-c). Neomycin production was abolished in Δ*neo15*. Instead, ribostamycin was detected in the culture supernatant (Fig. S2-a), indicating that the biosynthetic pathway has been disrupted at the stage of the addition of the last sugar (ring IV). This result agrees with the glycosyltransferase activity proposed for Neo15. Complementation of the Δ*neo15* mutant was carried out in three alternative ways: first, by using a plasmid (pQZ54) containing *neo15* under the control of the *ermE* promoter; secondly, using a plasmid (pQZ61) housing *neo15* under the control of its natural promoter together with the other genes in the operon (*neo12-neo16*); or, finally, using a plasmid (pQZ62) in which only *neo15* is placed downstream of its natural promoter. In each case, neomycin production was restored (Fig. S2-b), consistent with Neo15 catalysing the specific transfer of the GlcNAc moiety from UDP-GlcNAc to ribostamycin.

To confirm the role of the *neo15* gene product in neomycin biosynthesis, the Neo15 protein was expressed with an N-terminal His₆-tag in *E. coli* BL21 (DE3) and purified by nickel chelate-affinity chromatography in a yield of 0.9 mg L⁻¹. The mass of the purified protein was determined to be 41295 Da (Fig. S2-d) (including the His₆-tag, and with loss of N-terminal methionine) by LC-ESI-MS (calculated, 41289 Da). A peak (41466 Da) corresponding to the α-N-gluconoylated²¹ Neo15 (calculated, 41467 Da) was also observed. The purified His₆-tagged Neo15 was tested for its glycosyltransferase activity *in vitro*. Incubation of Neo15 with UDP-GlcNAc and ribostamycin overnight at 30°C converted all ribostamycin to N-acetylglucosaminylribostamycin ([M+H]⁺ = 658) as observed by LC-ESI-MS (Fig. 2). The same conversion was detected using a cell-free extract of *E. coli* carrying the Neo15 expression plasmid (pQZ39), but not with an extract of *E. coli* containing an empty pET28a(+) vector. Neo15 was found to be relatively stable, retaining significant

Fig. 2 LC–MS analysis of Neo15 glycosyltransferase activity in vitro. (a) Control with the substrate ribostamycin but without Neo15. (b) Conversion of ribostamycin to N-acetylglycosaminylribostamycin by Neo15.

enzymatic activity in Tris-HCl buffer, pH 8.4 at 4 °C even after 37 days. The in vitro identification of Neo15 (NeoK) as the glycosyltransferase required to transfer GlcNAc to ribostamycin has also recently been independently reported by Eguchi and colleagues.¹⁷

Neo15 appears to have a narrow specificity for its ribostamycin substrate. Incubation of Neo15 with 2-DOS under the same conditions did not produce any N-acetylparomamine, confirming that Neo15 and Neo8 act independently. Replacing UDP-GlcNAc with UDP-D-glucosamine (UDP-GlcN) or TDP-glucosamine (TDP-GlcN) as alternative sugar donors did not yield any product. Neo15 also failed to catalyse glycosylation of butirosin, which differs from ribostamycin by having an (S)-4-amino-2-hydroxybutyrate substituent at the C1 amine of 2-DOS (data not shown).

### Structural comparisons between glycosyltransferases Neo8 and Neo15

Neo8 and Neo15 share only modest amino acid sequence identity (23.5%) but their three-dimensional structures are likely to be more similar than this might suggest. Glycosyltransferases are currently classified into 91 distinct sequence families²² (CAZy; carbohydrate-active enzymes database at http://www.cazy.org/) but almost all display only two different three-dimensional folds, referred to as GT-A and GT-B.²³ Both Neo8 and Neo15 are classified by CAZy in Family 4, the second largest described family (7868 entries at time of writing), all of which are retaining glycosyltransferases and have the GT-B fold.

Both Neo8 and Neo15 show significant sequence matches to four GT-B glycosyltransferases from Family 4 whose crystal structures have been determined. WaaG, also known as RfaG, is an α-1,3-glycosyltransferase responsible for transferring glucose from UDP-D-glucose onto L-glycerol-D-manno-heptose II in the biosynthesis of the core structure of lipopolysaccharide (LPS).²³ AviGT4 is one of several glycosyltransferases required for the biosynthesis of the polyketide antibiotic avilamycin A. WaaG and AviGT4 display only 16% sequence identity, but their GT-B fold (two distinct domains each with a “Rossmann-like” (β/α/β) fold) and their conserved catalytic residues reveal their close structural resemblance.²³ PimA, the third Family 4 glycosyltransferase, is an essential phosphatidylinositol mannosyltransferase from *Mycobacterium tuberculosis*²⁴ and MshA, the fourth, is involved in mycothiol biosynthesis in *Corynebacterium glutamicum*.²⁵ Although they all utilise different NDP-hexose substrates, the C-terminal domain housing the donor binding site is particularly highly conserved, and contains a signature sequence which is universally found in the retaining glycosyltransferases of CAZy Family 4.²⁶ This sequence contains two conserved glutamate residues (Fig. 3), spaced eight residues apart, and it is also present in the C-terminal region of both Neo8 and Neo15. Interestingly, MshA and the following enzyme in the mycothiol biosynthetic pathway (MshB) together also catalyse transfer of a GlcNAc group from UDP-GlcNAc to an acceptor, followed by deacetylation of the product.²⁷

WaaG 236 .QDKPRKFEAL AEK LG VRSNVHFFSGR...NDVS EL MAA AA DLLLHP. A YQE AAGI VILLEA IT AGL PVL TT
MshA 268 PNATPDTYRHM AEE LG VEKRIRFLDPRPP.SELV AV YRAA DIVAVP. S FNE SFGI VAMEA QAS GT PV I AA
Neo8 245 .GPQRERMREA VEA AG LADRFVFTGFLPH.DAVP SVM TAL DVLVMP. SAHE ELGG SALEAM VC GT PV A GY
Neo15 218 ..RGREARELV RAH QG AGRRVEVLDAYDIRTDWE RV FFGA SLFLLPS V FHETYNF AAAEAV VQL GV PV A AL
AviGT4 204 .DEITRRYGST VEP IG EVGGERRLDLLASAHAVL AMS QAV TGPWGG. I WCE PGAT VVSEA AV SGT PV VG T
PimA 251 .........EDEL REQ AG DLAGHLRFLGQVDDATKA SAM RSA DVYCAPH L GGE SFGI VLVEAM AA GT AVVAS

Fig. 3 Conserved active site motif in Neo8, Neo15 and closely-related glycosyltransferases of known structure. Fully conserved residues are in white text with a red background. Similar (at least 70%) residues are in red text and framed in blue. The conserved glutamate residues are marked with wedges (▲).

### Neo16 as a dual-function N-deacetylase in neomycin biosynthesis

Since both Neo8 and Neo15 use the same sugar donor, UDP-GlcNAc, deacetylation of the glucosamine moiety must take place twice during neomycin biosynthesis. We have previously characterised BtrD as a deacetylase responsible for converting N-acetylparomamine to paromamine in butirosin biosynthesis, where only one deacetylation reaction is required. Therefore Neo16 (NeoL), a potential homologue of BtrD (19% identity) in the neo cluster, was a candidate to fulfil at least one, and possibly both, of these roles.

The Neo16 protein was expressed with an N-terminal His₆-tag in *E. coli* BL21 (DE3) and purified by nickel-chelate affinity chromatography in a yield of 0.7 mg L⁻¹ of culture. The mass of the purified protein was determined to be 32138 Da (Fig. S3-b) (including the His₆-tag, and with loss of N-terminal methionine) by LC-ESI-MS (calculated, 32141 Da). A peak (32313 Da) corresponding to the α-N-gluconoylated Neo16 (calculated, 32319 Da) was also observed. Purified Neo16 was found to lose its activity within a few days, so fresh samples of Neo16 were used for all in vitro assays. Neo16 was found to function as a deacetylase in vitro with broad substrate specificity. Incubation of Neo16 with N-acetylparomamine overnight at 30 °C led to the formation of paromamine ([M + H]⁺ = 324) (Fig. 4b) as observed by LC-ESI-MS. Under the same conditions, incubation of Neo16 together with Neo15, ribostamycin and UDP-GlcNAc led to the formation of glucosaminylribostamycin ([M + H]⁺ = 616) (Fig. 4c). The outcome was the same if Neo16 was added after the completion of the reaction of Neo15 and removal of Neo15 protein by chloroform precipitation. A competitive assay between N-acetylparomamine and N-acetylglucosaminylribostamycin revealed that Neo16 does not have a clear preference for either substrate (data not shown). These results demonstrate that Neo16 is a deacetylase fully capable in vitro of accepting either N-acetylparomamine or N-acetylglucosaminylribostamycin, as also shown independently by Eguchi and colleagues.

In view of these results, we tested the ability of purified BtrD to deacetylate N-acetylglucosaminylribostamycin. Given its normal activity in the formation of butirosin, BtrD was expected to show selectivity for N-acetylparomamine as a substrate. Surprisingly, it acted on N-acetylglucosaminylribostamycin as effectively as did Neo16 to form glucosaminylribostamycin (Fig. S4). Moreover, like Neo16 it showed no substrate preference in a competitive assay between N-acetylparomamine and N-acetylglucosaminylribostamycin (data not shown). This suggests that BtrD and Neo16 have probably evolved to accommodate multiple substrates and their actions in vivo are controlled by other factors such as substrate availability or upstream regulators.

To complement the in vitro experiments, the *neo16* gene was disrupted by the insertion of a thiostrepton resistance gene (*tsr*) at a unique *BsaXI* site internal to the gene. An in-frame deletion was not required because the location of *neo16* means that there is no possibility of an unwanted polar effect on a downstream gene. The mutant was verified by PCR screening and Southern blot (Fig. S3-a & c). Neomycin production was abolished in the Δneo16 mutant and no neomycin was detected when 2-DOS was fed to the mutant. However when neamine (0.2 mg mL⁻¹) was added to the medium, neomycin production was restored (Fig. S3-d and Table S1). Paromycin was again detected in the culture supplemented with paromamine (0.2 mg mL⁻¹) (Fig. S3-e). Ribostamycin feeding (0.2 mg mL⁻¹) led to only small amounts of neomycin (Fig. S3-f), which is surprising when contrasted with the ability of ribostamycin to complement the Δneo8 mutant with high efficiency. However, if Neo16 were the only enzyme competent to carry out both of the deacetylation steps in neomycin biosynthesis then none of these feeding experiments should have given rise to neomycin or paromycin. These data taken together strongly suggest the presence of a second deacetylase in *S. fradiae* capable of the deacetylation of N-acetylglucosaminylribostamycin. Therefore, even though Neo16 shows the requisite broad specificity in vitro, and therefore is likely to have a dual specificity in vivo too, another deacetylase enzyme, perhaps located outside the boundaries of the currently-recognised *neo* gene cluster, cannot be excluded from a role in neomycin biosynthesis.

### Conclusion

Evidence has been obtained for the respective roles of two specific glycosyltransferases and a deacetylase in the biosynthetic pathway to neomycin from *S. fradiae*. A convenient system for genetic manipulation of the neomycin producer was developed and this allowed the role of Neo8 to be studied in vivo. A specific Δneo8 mutant was created and found to incorporate paromamine and later intermediates but not 2-DOS. These results complement the in vitro data that suggest Neo8 specifically adds N-acetylglucosamine to the aglycone 2-DOS. Meanwhile, a Δneo15 mutant accumulated ribostamycin in its supernatant, and recombinant Neo15 was purified and shown specifically to transfer GlcNAc from UDP-GlcNAc to ribostamycin to form N-acetylglucosaminylribostamycin. Both Neo16, the sole deacetylase to be recognised in the gene cluster, and BtrD, its counterpart in butirosin biosynthesis, show broad deacetylase activity for neomycin biosynthetic intermediates in vitro, consistent with a dual specificity for Neo16. Nevertheless, such dual specificity is not essential, for a second enzyme with the ability to substitute for Neo16 in deacetylation of N-acetylglucosaminylribostamycin appears to be present in *S. fradiae*. There is a precedent for this finding in mycothiol biosynthesis, where the deacetylase *mshB* was

Fig. 4 LC–MS analysis of Neo16 deacetylase activity in vitro. (a) Control assay with the substrate N-acetylparomamine but without Neo16. (b) Conversion of N-acetylparomamine to paromamine by Neo16. (c) Conversion of N-acetylglucosaminylribostamycin to glucosaminylribostamycin by Neo16.

found not to be essential, because the *mshB* mutant still produced approximately 20% of the wild-type level of mycothiol.²⁸

### Experimental

#### Bacterial strains and culture conditions

*S. fradiae* NCIMB 8233 was maintained on SFM solid medium²⁹ or in tryptic soy broth (TSB) (Difco). The Δneo16 mutant was maintained on SFM plates containing thiostrepton (25 μg mL⁻¹), and the complemented Δneo8 and Δneo15 mutants on plates containing apramycin (25 μg mL⁻¹). For fermentation, *S. fradiae* and its mutants were grown in TSB (50 mL) without any antibiotics

in a flask (250 mL) containing a spring (to aid dispersal of mycelial growth) at 30 °C in a rotary shaker (200 rpm) for seven days. For feeding studies, filter-sterilised compounds were added to TSB just before inoculation. *Escherichia coli* DH10B strain was used for general cloning; ET12567/pUZ8002 strain,³⁰ as a donor strain for conjugation with *S. fradiae*; BL21 (DE3) strain, as a host for protein expression.

#### Construction and verification of mutants

The *neo8* deletion plasmid, pQZ27, was constructed based on an *E. coli*–*Streptomyces* shuttle vector, pQZ21, derived from pYH7³¹ by HindIII and MscI digestion followed by blunting the ends and
self-ligation of an 8615 bp fragment. A *SapI* (3613 bp) fragment from the cosmid FR3F7² of the neomycin gene cluster, containing *neo8*, was first cloned into the *SmaI* site of pUC18 vector. The resulting plasmid, pQZ25, was digested with *EcoRI* and *BamHI* to recover a 3637 bp fragment, which was further digested by *BspHI* and *NcoI*. Two fragments, *EcoRI-BspHI* (1243 bp) and *NcoI-BamHI* (1554 bp), were ligated into *EcoRI* and *BamHI* digested pUC18 to form pQZ26. Finally, a blunt-ended *EcoRI-BamHI* fragment (2797 bp) from pQZ26 was cloned into the blunt-ended *BamHI* site of pQZ21 to form pQZ27.

The *neo15* deletion plasmid, pQZ37, was also constructed based on pQZ21. An *FspI* fragment (4592 bp) from the cosmid FR3F7, containing *neo15*, was first cloned into the *SmaI* site of pUC18. The resulting plasmid, pQZ35, was linearised by *AfeI* digestion before it was used as a PCR template for the construction of pQZ36. The following primers were used: 5′-GACGCCCTGGGTGGACC-3′ and 5′-GGACCGGTAGAACACTCCG-3′. The PCR reaction was performed using Phusion High-Fidelity Master Mix with GC buffer (New England Biolabs) under the following conditions: 98 °C, 2 min; 35 cycles of 10 s at 98 °C, 20 s at 60 °C and 3 min at 72 °C, and a final extension at 72 °C for 5 min. A typical reaction mixture contained *AfeI*-digested pQZ35 (1 μL), primers (1 μL each, stock concentration at 50 μM), DMSO (3 μL), MgCl₂ (1.5 μL, 50 mM), the master mix (25 μL) in a final volume of 50 μL. The product (6519 bp) was phosphorylated by T4 polynucleotide kinase (New England Biolabs) before self-ligation to form pQZ36. Finally, a blunt-ended *KpnI-PvuII* fragment (3465 bp) from pQZ36 was cloned into the blunt-ended *BamHI* site of pQZ21 to form pQZ37. During this work, a mistake was noted in the publicly deposited *neo15* sequence (AJ843080) at nucleotide 520, which should be G not C. This correction changes an aspartic acid into a histidine residue.

The *neo16* deletion plasmid, pQZ22, was also based on pQZ21. A *SacI* fragment (4242 bp) from the cosmid FR3F7, containing *neo16*, was cloned into the *SacI* site of pUC18 to form pQZ15. This plasmid was modified to remove the *BsaXI* site by *TfI* digestion, followed by recovery of a 6788 bp fragment which then self-ligated to form pQZ15′. Digestion of pQZ15′ with *BsaXI* and blunting of the ends allowed the insertion of a thiostrepton resistance marker (*tsr*), an *SfoI* fragment (1461 bp) from pJTU412³² to form pQZ16. A blunt-ended *SacI* fragment (5670 bp) was then ligated into the blunt-ended *BamHI* site of pQZ21 to form pQZ22.

The plasmids pQZ27 (*neo8*), pQZ37 (*neo15*) and pQZ22 (*neo16*) were introduced into *S. fradiae* through conjugation from the *E. coli* ET12567/pUZ8002 strain transformed with the relevant plasmids. Conjugation SFM plates (20 mL) were incubated at 30 °C for 16 hours before being overlaid with water (1 mL each) containing nalidixic acid (500 μg), and additionally apramycin (500 μg) for all except for the pQZ22 plates for which thiostrepton (500 μg) was added. Exconjugants (usually >10² per plate) were picked after four days and were verified on corresponding SFM plates with nalidixic acid (25 μg mL⁻¹), and additionally apramycin (25 μg mL⁻¹) or thiostrepton (25 μg mL⁻¹). Confirmed colonies were then propagated on SFM plates without antibiotics. Single colonies of the putative Δ*neo8* and Δ*neo15* mutants from the non-selective plates were patched onto paired SFM plates, one of them containing apramycin (25 μg mL⁻¹). Patches showing sensitivity to apramycin were screened by PCR. Single colonies of the putative Δ*neo16* mutant from the non-selective plates were

patched onto paired SFM plates, one with apramycin (25 μg mL⁻¹) and the other with thiostrepton (25 μg mL⁻¹). The patches with resistance to thiostrepton and sensitivity to apramycin were screened by PCR. The following primers were used for PCR mutant screening: (Δ*neo8*) 5′-TCTCCCCGTGGAGTCCCC-3′ and 5′-CCACGTTCACCACCAGATAGATG-3′; (Δ*neo15*) 5′-TCGTCAACATCCTCAACCG-3′ and 5′-TCACCCACCGTGCTCCT-3′; (Δ*neo16*) 5′-TCCTCGTAGAACAGGGTCAGC-3′ and 5′-GGACGGGAGGAGGAGGCAC-3′. PCR reactions were performed using BioMix™ Red (Bioline) under the following conditions: 94 °C, 5 min; 35 cycles of 30 s at 94 °C, 30 s at 55 °C and 1 min at 72 °C, and a final extension at 72 °C for 7 min. A typical reaction mixture contained genomic or plasmid DNA template (1 μL), primers (1 μL each, stock concentration at 50 μM), DMSO (1.5 μL), BioMix™ Red (15 μL) in a final volume of 30 μL. An 865 bp PCR product (Δ*neo8*) (Fig. S1-b) and a 486 bp PCR product (Δ*neo15*) (Fig. S2-c) were recovered from 0.7% agarose gel for sequencing to confirm the in-frame deletion. The Δ*neo16* mutant was also checked by Southern blot using *AgeI*-digested pQZ16 (2606 bp and 2040 bp fragments) as a probe (Fig. S3-c). The genomic DNA of the mutants and the wild-type, and the plasmid controls were all digested with *AgeI* and *SacI* separately.

### Metabolite purification and identification

After seven days of fermentation in TSB, the culture was harvested by centrifugation. The supernatant was collected and its pH was adjusted to 6 with HCl (2 M). The supernatant was then filtered through a GF/A glass fibre filter before being loaded onto a column (2 g) of DOWEX® 50 WX 8–200 ion-exchange resins, that had been pre-washed with acetonitrile (50 mL) followed by water (50 mL), and equilibrated with sodium phosphate buffer (50 mM, pH 5.4). The column was washed with sodium phosphate buffer (60 mL, 50 mM) and then with ammonium hydroxide solution (1 mL, 1 N). Target compounds were eluted with ammonium hydroxide solution (10 mL, 1 N). The eluate was freeze-dried before being redissolved in water (1 mL). It was washed twice with an equal volume of chloroform. The aqueous layer was analysed by LC-ESI-MS on a Hewlett-Packard HPLC 1100 series system (Agilent), coupled to a Finnigan LCQ (Thermo Finnigan) mass spectrometer. The sample was eluted from a 2.0 × 250 mm Gemini 5 μC18 column (Phenomenex) using a linear gradient of 15% to 100% acetonitrile (+0.1% pentafluoropropionic acid) in water (+0.1% pentafluoropropionic acid) over 30 minutes, at a flow rate of 0.15 mL min⁻¹. MS/MS was carried out with 20% relative collision energy (helium as collision gas). Fragmentation patterns of the products were compared to those of commercial or synthesised standards when available.

### Complementation of the Δ*neo8* and Δ*neo15* mutants

a) Plasmid pIB139-based complementation. The *neo8* gene was amplified using the following primers: 5′-CGTCGGGCA-TATGCAGGTGCAGATCCT-3′ (*NdeI*) and 5′-ACAGCGCC-GCAGCGAATTACGG-3′ (*EcoRI*). The *neo15* gene was amplified using the primers for its protein expression. The products were digested with *NdeI* and *EcoRI*, before being ligated into *NdeI*-and *EcoRI*-digested integrative vector pIB139³ to give pQZ53 (*neo8*) and pQZ54 (*neo15*). These complementation plasmids wereintroduced into the Δneo8 and Δneo15 mutants respectively by  
conjugation from *E. coli* ET12567/pUZ8002. As a control, the  
empty vector pIB139 was used for conjugation with the mutants.  
Exconjugants were verified based on their apramycin resistance.  
Complemented mutants were confirmed by PCR from their  
genomic DNA using the original mutants as a negative control,  
and the wild-type DNA and the plasmids as positive controls.  

b) Plasmid pSET152-based complementation. The *neo8* gene  
was excised from pQZ25 with *FspI*. The 2890 bp fragment  
was inserted into *PvuII*-digested pSET152<sup>34</sup> to form pQZ60.  
A 4914 bp fragment containing the genes *neo12* to *neo16* was  
cut out from pQZ35 with *PvuII*, and then ligated into *PvuII*-  
digested pSET152 to form pQZ61. This plasmid was digested  
with *AgeI* and a 9839 bp fragment was recovered to be used  
as a template for a PCR reaction, which amplified the 6537 bp  
vector flanking the genes *neo12* to *neo16*. The PCR reaction was  
carried out with 3 min extension times using the following primers:  
5′-CCCGGCCCTCGTGAGGCACGGC-3′ and 5′-GGCGTGT-  
CTCCTCGTTCGGGACGCTG-3′. Meanwhile, the *neo15* gene  
was amplified using the following primers: 5′-GTGGCTGAGG-  
CGCCTGCCGGG-3′ and 5′-TCACCCACC GTGCTCCTCC-  
TCCCGT-3′. The *neo15* product was phosphorylated before being  
ligated into the 6537 bp vector to form pQZ62 (forward insertion  
selected only). The sequence of the plasmid was checked by DNA  
sequencing. The complementation plasmids were again introduced  
into the Δneo8 (pQZ60) and Δneo15 (pQZ61 and pQZ62) mutants  
by conjugation from *E. coli* ET12567/pUZ8002. As a control, the  
empty vector pSET152 was used. The complemented mutants were  
again checked by PCR from their genomic DNA.  

Cloning, overexpression and purification of proteins  
The *neo15* gene was amplified by PCR from the cosmid FR3F7  
using the following primers: 5′-CCGCCGGAGGTGCCATAT-  
GGCTGAG-3′ (*NdeI*) and 5′-TCCGCCGCCTCCGAATTC-  
GGCTC-3′ (*EcoRI*). The *neo16* gene was amplified with  
the primers: 5′-CGGGAGGAGGAGCCATATGGGTGAGCC-  
3′ (*NdeI*) and 5′-ACGAGGCCGAATTCACCGGGCACCC-3′  
(*EcoRI*). The PCR reactions were performed under the same  
conditions as for the construction of pQZ36 except that 1 min  
extension time at 72°C was used instead of 3 min. The products  
were separately digested and ligated into the appropriate  
restriction sites of pET28a(+) (Novagen) expression vector to form  
plasmids pQZ39 (*Neo15*) and pQZ30 (*Neo16*), which were used  
to transform *E. coli* BL21 (DE3) competent cells. The sequences  
of *neo15* in pQZ39 and *neo16* in pQZ30 were verified by DNA  
sequencing.  
The *E. coli* cells harbouring the recombinant plasmids were  
grown in LB medium (1 L) containing kanamycin (50 μg mL<sup>-1</sup>)  
with shaking (250 rpm) at 30°C until the *A*<sub>600</sub> reached 0.6 to 0.9.  
Protein overexpression was induced by the addition of isopropyl β-  
D-thiogalactopyranoside (0.2 mM), and the culture was incubated  
further with shaking (200 rpm) at 18°C overnight. The cells were  
harvested by centrifugation and the pellet was resuspended in  
binding buffer (30 mL, containing 5 mM imidazole, 0.5 M NaCl,  
20 mM Tris-HCl, pH 7.9). The cells were lysed by sonication,  
followed by centrifugation. His<sub>6</sub>-Neo15 and His<sub>6</sub>-Neo16 were  
purified on a Ni<sup>2+</sup>-NTA affinity column (Novagen) at 4°C (Fig.  
S2-d and S3-b). After elution, the protein solution was exchanged  

into a storage buffer (100 mM NaCl, 20 mM Tris-HCl, pH 7.5)  
by ultrafiltration. The sample was applied on a 2.0 × 250 mm  
Jupiter 5 μC 4 column (Phenomenex). The method used a linear  
gradient of 35% to 75% acetonitrile (+0.1% trifluoroacetic acid)  
in water (+0.1% trifluoroacetic acid) over 25 min with a flow rate  
of 0.3 mL min<sup>-1</sup>.  
Protein expression of BtrD and *Providencia stuartii* 2′-N-  
acetyltransferase (AAC(2′)-Ia)<sup>35</sup> was carried out as described  
previously.<sup>16</sup>  

Chemicals  
Paromamine and neamine were synthesised by acid methanolysis  
of paromomycin and neomycin (Sigma) using the method of  
Dutcher and Donin.<sup>36</sup> N-Acetylparomamine was synthesised  
with AAC(2′)-Ia.<sup>16</sup> UDP-GlcN and TDP-GlcN were made as  
previously described.<sup>37</sup> Ribostamycin and UDP-GlcNAc were  
purchased from Sigma.  

Enzymatic assays  
A typical reaction mixture for assay of glycosyltransferase activity  
(50 μL) contained Tris-HCl (25 mM, pH 8.4), ribostamycin  
(1 mM) or butirosin (1 mM) or 2-DOS (1 mM), UDP-GlcNAc  
(2 mM) and Neo15 (0.15 mg mL<sup>-1</sup>) and was incubated at 30°C  
overnight. For assay of deacetylase activity, Neo16 (1 mg mL<sup>-1</sup>)  
or BtrD (1 mg mL<sup>-1</sup>) was included in the reaction mixture.  
The assay for the first deacetylation reaction contained Tris-  
HCl (25 mM, pH 8.4) and *N*-acetylparomamine (0.15 mM).  
The competitive assays included *N*-acetylparomamine (0.5 mM)  
and *N*-acetylglucosaminylribostamycin (0.5 mM) as substrates,  
which were obtained from quenched completed reactions of  
Neo15 and AAC(2′)-Ia. The reaction goes to completion overnight  
with ribostamycin concentration up to 10 mM for Neo15 and  
paromamine concentration up to 5 mM for AAC(2′)-Ia. All  
control reactions omitted either the substrate(s) or the protein(s).  
All reactions were quenched with chloroform (50 μL), followed  
by vortexing and centrifugation. The aqueous layer was analysed  
by LC-ESI-MS using the same method as for the knockout  
experiments.  

Acknowledgements  
We are grateful to Andrew W. Truman for the synthesis of UDP-  
GlcN and TDP-GlcN, and Dr Nicholas M. Llewellyn for the  
synthesis of paromamine, as well as for their helpful advice on  
preparation of this manuscript. This work was supported by a  
project grant from the BBSRC to J.B.S. and F.H. Q.F. thanks  
Trinity College, the Cambridge Overseas Trust and the ORS  
scheme for studentship support.  

References  
1 C. J. R. Pearce and K. L. Rinehart, Jr., in *Antibiotics*, vol. IV, ed  
J. W. Corcoran, Springer-Verlag, Berlin/Heidelberg/New York, 1981,  
pp. 74–100.  
2 N. M. Llewellyn and J. B. Spencer, *Nat. Prod. Rep.*, 2006, **23**, 864–874.  
3 S. A. Waksman and H. A. Lechevalier, *Science*, 1949, **109**, 305–307.  
4 P. Edelmann and J. Gallant, *Cell (Cambridge, Mass.)*, 1977, **10**, 131–  
137.  
5 A. P. Carter, W. M. Clemons, D. E. Brodersen, R. J. Morgan-Warren,  
B. T. Wimberly and V. Ramakrishnan, *Nature*, 2000, **407**, 340–348.  
6 J. B. H. Tok, L. J. Dunn and R. C. Des Jean, *Bioorg. Med. Chem. Lett.*, 2001, **11**, 1127–1132.

7 A. Litovchick, A. Lapidot, M. Eisenstein, A. Kalinkovich and G. Borkow, *Biochemistry*, 2001, **40**, 15612–15623.

8 S. Hirukawa, K. A. Olson, T. Tsuji and G. F. Hu, *Clin. Cancer Res.*, 2005, **11**, 8745–8752.

9 M. J. Cron, J. G. Keil, M. V. Ruggeri and D. Walker, *Chem. Commun.*, 1979, 266–267.

10 G. L. Coffey, L. E. Anderson, M. W. Fisher, M. M. Galbraith, A. B. Hillegas, D. L. Kohberger, P. E. Thompson, J. Weston and K. Ehrlich, *Antibiot. Chemother.*, 1959, **9**, 730–738.

11 E. J. Hessler, H. K. Jahneke, J. H. Robertson, K. Tsuji, K. J. Rinehart Jr. and W. T. Shier, *J. Antibiot. (Tokyo)*, 1970, **23**, 464–466.

12 F. Huang, S. F. Haydock, T. Mironenko, D. Spiteller, Y. Li and J. B. Spencer, *Org. Biomol. Chem.*, 2005, **3**, 1410–1418.

13 F. Kudo, Y. Yamamoto, K. Yokoyama, T. Eguchi and K. Kakinuma, *J. Antibiot.*, 2005, **58**, 766–774.

14 F. Huang, Y. Li, J. Yu and J. B. Spencer, *Chem. Commun.*, 2002, 2860–2861.

15 F. Kudo, K. Kawabe, H. Kuriki, T. Eguchi and K. Kakinuma, *J. Am. Chem. Soc.*, 2005, **127**, 1711–1718.

16 A. W. Truman, F. Huang, N. M. Llewellyn and J. B. Spencer, *Angew. Chem., Int. Ed. Engl.*, 2007, **46**, 1462–1464.

17 K. Yokoyama, Y. Yamamoto, F. Kudo and T. Eguchi, *ChemBioChem*, 2008, **9**, 865–869.

18 H. Baud, A. Betencourt, M. Peyre and L. Penasse, *J. Antibiot.*, 1977, **30**, 720–723.

19 F. Kudo, T. Fujii, S. Kinoshita and T. Eguchi, *Bioorg. Med. Chem.*, 2007, **15**, 4360–4368.

20 C. J. Rowe, J. Cortés, S. Gaisser, J. Staunton and P. F. Leadlay, *Gene*, 1998, **216**, 215–223.

21 K. F. Geoghegan, H. B. F. Dixon, P. J. Rosner, L. R. Hoth, A. J. Lanzetti, K. A. Borzilleri, E. S. Marr, L. H. Pezzullo, L. B. Martin, P. K. LeMotte, A. S. McColl, A. V. Karnath and J. G. Stroh, *Anal. Biochem.*, 1999, **267**, 169–184.

22 P. M. Coutinho, E. DeLeury, G. J. Davies and B. Henrissat, *J. Mol. Biol.*, 2003, **328**, 307–317.

23 Y. Bourne and B. Henrissat, *Curr. Opin. Struct. Biol.*, 2001, **11**, 593–600.

24 M. Guerin, J. Kordulakova, F. Schaeffer, A. Svetlikova, A. Buschiazzo, D. Giganti, B. Gicquel, K. Mikusova, M. Jackson and P. M. Alzari, *J. Biol. Chem.*, 2007, **282**, 20705–20714.

25 M. W. Vetting, P. A. Frantom and J. S. Blanchard, *J. Biol. Chem.*, 2008, **283**, 15834–15844.

26 R. A. Geremia, E. A. Petroni, L. Ielpi and B. Henrissat, *Biochem. J.*, 1996, **318**, 133–138.

27 G. L. Newton, P. Ta, K. P. Bzymek and R. C. Fahey, *J. Biol. Chem.*, 2006, **281**, 33910–33920.

28 N. A. Buchmeier, G. L. Newton, T. Koledin and R. C. Fahey, *Mol. Microbiol.*, 2003, **47**, 1723–1732.

29 T. Kieser, M. J. Bibb, M. J. Buttner, K. F. Chater and D. A. Hopwood, *Practical Streptomyces Genetics*, John Innes Foundation, Norwich, 2000.

30 D. J. MacNeil, J. L. Occi, K. M. Gewain, T. MacNeil, P. H. Gibbons, C. L. Ruby and S. J. Danis, *Gene*, 1992, **115**, 119–125.

31 Y. Sun, M. Samborskyy, T. Mironenko, P. F. Leadlay and S. F. Haydock, *Microbiology*, 2006, **152**, 3507–3515.

32 Y. Sun, X. Zhou, J. Liu, K. Bao, G. Zhang, G. Tu, T. Kieser and Z. Deng, *Microbiology*, 2002, **148**, 361–371.

33 C. J. Rowe, I. U. Böhm, B. Wilkinson, B. A. M. Rudd, G. Foster, A. P. Blackaby, P. J. Sidebottom, Y. Roddis, A. D. Buss, J. Staunton and P. F. Leadlay, *Chem. Biol.*, 2001, **8**, 475–485.

34 M. Bierman, R. Logan, K. O'Brien, E. T. Seno, R. N. Rao and B. E. Schoner, *Gene*, 1992, **116**, 43–49.

35 K. Franklin and A. J. Clarke, *Antimicrob. Agents Chemother.*, 2001, **45**, 2238–2244.

36 J. D. Dutcher and M. N. Donin, *J. Am. Chem. Soc.*, 1952, **74**, 3420–3422.

37 A. W. Truman, L. Robinson and J. B. Spencer, *ChemBioChem*, 2006, **7**, 1670–1675.
