
MicroRNAs in Cancer

Yong Sun Lee and Anindya Dutta

Department of Biochemistry and Molecular Genetics, University of Virginia, Charlottesville, Virginia 22908; email: ad8q@virginia.edu

Key Words

tumorigenesis, oncogene, tumor suppressor, microRNA expression profile, diagnosis and prognosis, therapy

Abstract

Within the past few years, studies on microRNA (miRNA) and cancer have burst onto the scene. Profiling of the miRNome (global miRNA expression levels) has become prevalent, and abundant miRNome data are currently available for various cancers. The pattern of miRNA expression can be correlated with cancer type, stage, and other clinical variables, so miRNA profiling can be used as a tool for cancer diagnosis and prognosis. miRNA expression analyses also suggest oncogenic (or tumor-suppressive) roles of miRNAs. miRNAs play roles in almost all aspects of cancer biology, such as proliferation, apoptosis, invasion/metastasis, and angiogenesis. Given that many miRNAs are deregulated in cancers but have not yet been further studied, it is expected that more miRNAs will emerge as players in the etiology and progression of cancer. Here we also discuss miRNAs as a tool for cancer therapy.

**SYNOPSIS**

Within the past decade, there was a major advance in biology: the discovery of small RNAs, including microRNA (miRNA) and small interfering RNA (siRNA), as highlighted in the December 2002 issue of *Science* (1). Since the discovery of RNA interference (RNAi) in nematodes (2), siRNA has provided a technical breakthrough for short-term genetics in mammalian systems. The big impact of small RNAs was well celebrated by the 2006 Nobel Prize awarded to the two scientists who discovered RNAi.

Moreover, miRNAs shed new light on the posttranscriptional regulation of gene expression. miRNAs were first discovered in worms (3, 4), and they were later observed in a number of animals, plants, and viruses. During the past several years the miRNA field has expanded, with many recent publications implicating miRNAs in diverse cellular processes.

Cancer is a complex genetic disease caused by the accumulation of mutations, which lead to deregulation of gene expression and uncontrolled cell proliferation. Given the wide impact of miRNAs on gene expression, it is not surprising that a number of miRNAs have been implicated in cancer. In this review, we present a comprehensive discussion of the links between miRNA and cancer.

**INTRODUCTION**

MicroRNAs (miRNAs) are small noncoding regulatory RNAs ranging in size from 17 to 25 nucleotides (see miRBase, http://microrna.sanger.ac.uk/). The definition of miRNAs is based on their generation by the action of Dicer, an RNase that processes hairpin-structured precursors (known as pre-miRNA) into mature miRNAs (5). miRNAs posttranscriptionally repress gene expression by recognizing complementary target sites in the 3′ untranslated region (UTR) of target messenger RNAs (mRNAs).

Eight years after the first miRNA was reported by Ambros’s and Ruvkun’s groups (3, 4) in 1993, the miRNA era began in earnest when

three groups identified tens of small RNAs from *Caenorhabditis elegans*, *Drosophila*, and human (6–8). During the past seven years, the number of miRNAs identified, as well as the number of publications describing them, has expanded enormously.

**Genes, Sequences, and Genomic Organization**

Currently, more than 5000 miRNAs from over 50 organisms are registered in the miRBase database (8a). Five hundred thirty-three human miRNAs are now known, but this number may increase, as bioinformatics studies predict that up to 1000 miRNAs exist (9, 10). miRNAs are named as miR- plus numbers (e.g., miR-125), although there are a few exceptions. miRNAs of similar sequence are usually distinguished by an additional letter following the miRNA number (e.g., miR-125b). A miRNA of identical mature sequence may appear at several genomic loci with different precursor sequences. In such a case, the different miRNA genes would be distinguished by the addition of another number at the end of the sequence (e.g., miR-125b-1).

In humans, approximately one-third of miRNAs are organized into clusters. A given cluster is likely to be a single transcriptional unit, suggesting a coordinated regulation of miRNAs in the cluster. In silico analysis has revealed that more than half of the clusters contain two or more miRNAs of similar sequence (11). However, it is very rare for miRNAs of an identical mature sequence to be duplicated in a cluster. This genomic organization confers simultaneous expression of similar miRNAs, possibly leading to combinatorial diversity and synergy in the biological effects. However, all the miRNAs from a single transcriptional cluster are not expressed at equal levels, suggesting that miRNAs are also regulated posttranscriptionally. Also, a significant portion of miRNAs are located in the intronic region of protein-coding or -noncoding transcription units (12), whereas a minor subset of miRNAs are mapped to repetitive sequences such as long interspersed nuclear elements (13).

# Expression

Many miRNAs exhibit characteristic expression patterns. Some miRNAs are differentially expressed in developmental stages, for instance the two founding members *lin-4* and *let-7* in *C. elegans*. For this reason, they were once known as small temporal RNAs because they are expressed in specific temporal phases of development and regulate developmental timing. Many miRNAs are expressed in a tissue-specific manner. In the few cases where a forward genetic approach in worms or flies led to identification of a miRNA, the function of the miRNA was inferred from the phenotype of the mutant. In mammalian cells, however, a miRNA expression profile is usually the first clue to its possible role. Analogous to mRNA expression, miRNA expression is determined by intrinsic cellular factors as well as diverse environmental variables.

Expression of a miRNA can be measured by molecular biology techniques, such as Northern blot, RNase protection assay, and primer extension assay. The small size of miRNAs initially hampered polymerase chain reaction (PCR)-based methods. However, since the development of adaptor-mediated quantitative real-time PCR (qRT-PCR; reviewed in Reference 14), PCR-based techniques have become very popular due to their high sensitivity. Microarray techniques (reviewed in Reference 15) are widely used to comprehensively assay the entire miRNome (the global miRNA expression profile) in tissues or in cell lines (Table 1). In addition to microarray and qRT-PCR, both in situ hybridization (16) and serial analysis of gene expression (SAGE) adapted for small RNAs (17) have been used to obtain miRNomEs. Interest in the SAGE approach was stimulated by recent innovations in ultrahigh-throughput sequencing that provide a powerful tool for various genomics studies. Overall, these technical improvements are expected to greatly widen the repertoire of miRNAs in a variety of biological systems.

# Biogenesis

Biogenesis of a miRNA begins with the synthesis of a long transcript known as a pri-miRNA (Figure 1). In general, pri-miRNAs are transcribed by RNA polymerase II and retain mRNA features such as 5′ cap structure and 3′ poly(A) tail (18, 19). However, other pathways generate in a minor set of miRNAs, especially from genomic repeats. For example, RNA polymerase III is responsible for transcription of miRNAs in Alu repeats (20).

In the nucleus, pri-miRNA is processed to pre-miRNA by RNase III enzyme Drosha and its interacting partner DGCR8 (DiGeorge syndrome critical region gene 8) (21–23). DGCR8 recognizes the stem and the flanking single-stranded RNA (ssRNA) and serves as a ruler for Drosha to cut the stem approximately 11 nucleotides away from the stem-ssRNA junction to release the hairpin-shaped pre-miRNA (24). A subset of miRNAs known as miRtrons bypass the Drosha requirement by taking an alternative pathway, where pre-miRNAs are derived as a byproduct of a splicing event (25–27). The pre-miRNA is exported by exportin-5 to the cytoplasm (28, 29) and is subsequently converted to mature duplex miRNA by another RNase III enzyme, Dicer (30). The two strands of the duplex are separated by a RNA helicase (31) during the specific assembly of a ribonucleoprotein complex containing miRNA (miRNP). As with the strand selection of a siRNA (32, 33), the strand whose 5′ end forms the more unstable duplex with its partner seems to preferentially survive as the miRNA in the miRNP (24, 32, 33). However, the detailed molecular events that take place during the miRNP assembly and strand selection are not yet clear.

## Mechanisms of Action

miRNAs posttranscriptionally suppress the target mRNA expression, mostly through interaction with the 3′ UTR (Figure 1). However, the exact mechanism for miRNAs on their targets

| Tumor type          | Profiling method$^a$ | miRNAs deregulated in tumors | Notes                                                                 | Reference |
|---------------------|----------------------|-------------------------------|---------------------------------------------------------------------|-----------|
|                     |                      | Upregulated                   | Downregulated                                                      |           |
| Breast cancer       | Bead                 |                               | Classification of breast tumors into five subtypes                  | 93        |
|                     | qRT-PCR, array       | miR-21                        |                                                                     | 165       |
|                     | Array                |                               | Classification according to ErbB 2/ER status                       | 158       |
|                     |                      | miR-21                        | miR-125b, -145                                                       | 185       |
| Prostate cancer     | Array                |                               | let-7c, miR-125b, miR-145                                             | 186       |
|                     |                      |                               |                                                                     | 158       |
|                     | Membrane             |                               | Classification according to androgen dependency                    | 66        |
| Pancreatic cancer   | Array                | miR-21, -221, -222, -181a, -181b, -181d, -155 |                                                                     | 187       |
|                     |                      | miR-196a                      | miR-217                                                             | 160       |
|                     |                      |                               | Pancreatic ductal adenocarcinoma versus chronic pancreatitis and normal tissues |           |
|                     | qRT-PCR of pre-miRNA | let-7i, miR-100, -155, -221, -301, -21, -181a, -125b, -212, -376a | miR-375                                                             | 188       |
|                     |                      |                               | Pancreatic adenocarcinoma, paired benign tissue, normal pancreas, chronic pancreatitis |           |
| Pancreatic tumor    | Array                | miR-103, -107, -23a, -26b, -342, -192, -204, -211, -21 | miR-155                                                             | 189       |
|                     |                      |                               | Endocrine tumors versus acinar cell carcinoma                        |           |
| Lung cancer         | Array                | miR-21, -205                  | miR-126*                                                            | 190       |
|                     |                      |                               | Clinicopathological features                                       |           |
| Ovarian cancer      | Array                | miR-200a, -141                | miR-199a, -140, -145, -125b-1                                        | 191       |
|                     |                      |                               | Clinicopathological features                                       |           |
| Cervical cancer     | Array                |                               | Versus Drosha overexpression                                         | 95        |
| Uterine leiomyoma   | Array                | let-7, miR-21, -23b           |                                                                     | 192       |
|                     |                      |                               | Clinicopathological features (tumor size, race, etc.)               |           |
| Hepatocellular cancer | Array               | miR-21                        |                                                                     | 134       |
|                     |                      | miR-221                       | let-7a, miR-122a                                                     | 193       |
|                     |                      |                               | Versus liver cirrhosis                                              |           |
|                     |                      | miR-224, -18                  | miR-199a, -199a*, -200a                                              | 194       |
| Thyroid cancer      | Array                |                               | miR-30d, -125b, -26a, and -30a-5p                                     | 67        |
|                     |                      |                               | Anaplastic thyroid carcinomas versus normal tissues                  |           |
|                     |                      | miR-197, -346in FTC relative to FA |                                                                     | 195       |
|                     |                      |                               | FTC versus FA                                                       |           |
|                     |                      | miR-221, -222, -181b           |                                                                     | 159       |
|                     |                      |                               | Papillary thyroid carcinoma versus normal tissues                    |           |
|                     |                      | miR-221, -222, -146b           |                                                                     | 100       |
|                     |                      |                               | Papillary thyroid carcinoma versus normal tissues                    |           |

| Tumor type         | Profiling method | miRNAs deregulated in tumors | Notes                                      | Reference |
|--------------------|-----------------|-------------------------------|--------------------------------------------|-----------|
|                    |                 | Upregulated                  | Downregulated                             |           |
| Colorectal cancer  | Array           | miR-25, -92inMSS relative to MSI-H |                                          | Microsatellite stability versus high microsatellite instability | 196 |
|                    | qRT-PCR         | miR-31, -96, -135b, -183      | miR-133b, -145                             | miR-31 according to cancer stage | 197 |
|                    | Cloning miSAGE  |                              |                                            |                                      | 17  |
| Pituitary adenomas | Array           | miR-26a, -149                | miR-21, -141, -144                         | Clinicopathological features       | 198 |
| Neuroblastoma      | qRT-PCR         |                              |                                            | Neuroblastoma subtypes             | 69  |
| Glioblastoma       | Array           | miR-221, -10b                | miR-128, -181b, -181a                       |                                      | 199 |
| Leukemia           | Bead            | miR-128a, -128bin ALL compared with AML | let-7b, miR-223in ALL compared with AML | ALL versus AML                      | 200 |
|                    | Cloning         | miR-21, -150, -155           | miR-92, -222                               | Various CLLs                        | 201 |
|                    | Array           |                             | miR-29b, -181bin aggressive CLL with 11q deletion | Three subtypes of CLL               | 202 |
|                    |                 |                             | miR-16-1, -15a                             | CLL according to ZAP-70 and IgVH    | 203 |
| Leukemia           | Array           |                             |                                            | B cell CLL                          | 65  |
| Various tumors     | Array, cloning  |                             |                                            | Various sarcomas                   | 204 |
|                    | Array           |                             |                                            | Solid tumors (lung, breast, stomach, prostate, colon, and pancreatic) | 122 |
|                    | Bead            |                             |                                            | Comprehensive collection of various cancers and normal tissues | 64  |

(miRNA gene)
(miRNA gene)

PolIII
(e.g., miR-571)
↓
(pri-miRNA)

Transcription
PolII

Cap (A)n

Drosha
-DGCR8/Pasha
↓
(pre-miRNA)

Exon Exon
Splicing
(e.g., miRtrons in
flies and worms)

Transport
Exportin-5

Nucleus

Cytoplasm

Dicing
Dicer

Strand selection and RISC assembly
Dicer, TRBP
Ago, PACT

miRNP
(Ago, etc)

Target mRNA
5'UTR ORF 3'UTR

Translational inhibition
(initiation/elongation step)

Cytoplasmic P-body
(RNA degradation)

Canonical pathway
Minor pathway

Figure 1

Pathways of microRNA (miRNA) biogenesis and action. The canonical pathway (red arrows) and minor pathways (light blue arrows) are depicted. Abbreviations: miRNP, ribonucleoprotein complex containing miRNA; ORF, open reading frame; P-body, mRNA-processing body; PACT, interferon-inducible double-stranded RNA-dependent activator; RISC, RNA-induced silencing complex; TRBP, human immunodeficiency virus–transactivating RNA-binding protein; UTR, untranslated region of mRNA.

degradation by a miRNA, which is distinguished from siRNA-mediated mRNA cleavage (39, 40). Plausibly, miRNAs inhibit translation of target mRNAs, which are then sequestered to P-bodies and become subject to degradation. This model fits well for both mechanisms, although miRNAs have also been proposed to lead to the degradation of the target mRNAs without sequestration to P-bodies (41, 42). In some cases where mRNA translation is inhibited by a miRNA but where mRNA levels remain similar, sequestration to P-bodies might be inefficient. Nonetheless, it is possible that different mechanisms apply to individual miRNA–mRNA interactions.

**Target Prediction and Identification**

Prediction of miRNA targets (reviewed in Reference 46) is important, given that miRNAs exert their function by regulating target mRNAs. The specificity of miRNA–mRNA interaction is mainly conferred by the first eight nucleotides of a miRNA (known as a seed sequence) (43). The likelihood that a predicted target is a bona fide target is influenced not only by seed pairing but also by other factors such as the number of target sites, the context of surrounding sequence in mRNA (44), and the occlusion of target sites by RNA-binding proteins (45). Currently, several computational algorithms (reviewed in Reference 46) can predict the target mRNA(s), but they are far from perfect. The gold standard is experimental demonstration that (a) a luciferase reporter fused to the 3′ UTR of the predicted target is repressed by overexpression of the miRNA and (b) this repression is abrogated by point mutation in the target sequence(s) in the 3′ UTR. Many targets are predicted by in silico analyses, but not all of them are confirmed as real targets in this biological assay.

The in silico predictions have been complemented by experimental screenings for targets of miRNAs. Because miRNAs destabilize

mRNAs, miRNA depletion by a knockdown of miRNA-processing machinery is expected to result in upregulation of target mRNA, which can be assayed by microarray analysis. This approach identified HMGA2 (high-mobility group AT-hook 2) as a target of let-7 (38). Microarrays can also identify mRNAs that are decreased upon transfection of a miRNA, and screening these mRNAs for computationally predicted sites targeted by the miRNA yields some bona fide targets (37). A proteomics approach to identify proteins decreased by miRNAs led to the identification of tropomyosin I as a miR-21 target (47). Another screen employed a library of miRNA-expressing vectors and a sensor plasmid containing green fluorescent protein fused with the 3′ UTR of the gene of interest. p27(Kip1) was found to be targeted by miR-221 and -222 by this approach (48). Another approach is to isolate the miRNA–target mRNA complex by immunoprecipitation of a component of RNA-induced silencing complex (RISC) followed by microarray hybridization of the precipitated mRNAs (49).

Although a growing number of miRNA–target pairs are being identified (Table 2), the fraction of validated pairs is still small, given that 30% of mRNAs have been predicted to be miRNA targets (50, 51). New target identification will be facilitated by the accumulation of validated miRNA–target pairs, which can be utilized to develop a better prediction algorithm. In addition, experimental tools for high-throughput target screening need to be improved.

**Biological Roles**

miRNAs play important roles in cell fate determination, proliferation, and cell death. In addition to these vital processes, miRNAs are implicated in diverse cellular activities, such as immune response (reviewed in References 52 and 53), insulin secretion (54), neurotransmitter synthesis (55), circadian rhythm (56), and viral replication (57). This list will undoubtedly expand as experimental data accumulate.

| miRNA           | Target gene                          | Note                                      | Reference |
|------------------|-------------------------------------|-------------------------------------------|-----------|
| let-7            | RAS                                 | Lung cancer                              | 98        |
| let-7            | Ccnd2, Cdk6, Cdc25A                 | Cancer cell lines (A549, HepG2, HeLa)    | 59        |
| let-7            | HMGA2                               | Ovarian cancer                           | 132       |
| let-7c           | c-Myc                               | Liver tumors                             | 83        |
| let-7            | HMGA2                               | Lung cancer cell lines                   | 38        |
| let-7g           | c-Myc, k-RAS                        |                                           | 70        |
| let-7            | HMGA2                               |                                           | 106       |
| let-7            | NF2                                 | Cholangiocarcinoma cell lines            | 206       |
| miR-9, -125a, -125b | trkC                                | Neuroblastoma                            | 107       |
| miR-10b          | HOXD10                              | Breast cancer                            | 146       |
| miR-16-1, -15a   | Bcl2                                | Chronic lymphocytic leukemia             | 140       |
| miR-17-5p        | AIB1                                | Breast cancer                            | 162       |
| miR-17-5p, -20   | TβRII                               |                                           | 74        |
| miR-18           | CTGF                                | Colon cancer model of angiogenesis       | 147       |
| miR-19           | THBS1                               | Colon cancer model of angiogenesis       | 147       |
| miR-20a          | E2F1,2,3                            |                                           | 126       |
| miR-21           | Pdcd4                               | Colorectal cancer                        | 145       |
| miR-21           | PTEN                                | HCC                                      | 134       |
| miR-21           | TPM1                                |                                           | 47        |
| miR-21           | PTEN                                | Cholangiocarcinoma                       | 176       |
| miR-27b          | CYP1B1                              | Breast cancer                            | 207       |
| miR-29a, b, c    | DNMT-3A, DNMT-3B                   | Non-small cell lung cancer               | 82        |
| miR-29s          | Mcl-1                               | Cholangiocarcinoma cell line             | 84        |
| miR-29b, -181b   | TCL1                                | Chronic lymphocytic leukemia             | 202       |
| miR-34a          | E2F3                                | Neuroblastoma                            | 76        |
| miR-34a          | Ccne2, MET                          |                                           | 62        |
| miR-34a, -34b, -34c | Bcl2                                | Non-small cell lung cancer               | 60        |
| miR-106a cluster | Mylip, Hipk3, Rbp1-like             | T cell leukemia                          | 116       |
| miR-122a         | Cyclin G1                           | HCC versus liver cirrhosis               | 193       |
| miR-124a         | Cdk6                                | Colon cancer, lung cancer                | 80        |
| miR-125a, -125b  | ERBB2andERBB3                      | Breast cancer                            | 144       |
| miR-125b         | Bak-1                               | Prostate cancer                         | 205       |
| miR-127          | Bcl6                                | Bladder (and prostate) cancer            | 208       |
| miR-206          | ERα                                | Breast cancer                            | 105       |
| miR-221, -222    | p27(Kip)                           | Glioblastoma                             | 135       |
| miR-221, -222    | p27(Kip)                           | Glioblastoma                             | 48        |
| miR-221, -222    | p27(Kip)                           | Prostate cancer                         | 136       |
| miR-372, -373    | LATS2                              | Testicular germ cell tumor              | 209       |
| miR-378          | Sufu, Fus-1                        | U87 glioblastoma cell line               | 148       |

Table 2 (Continued)

| miRNA | Target gene | Note | Reference |
|-------|-------------|------|-----------|
| BART 16, 17-5p, 1-5p (EBV-encoded miRNAs) | LMP 1 (EBV-encoded oncogene) | Nasopharyngeal carcinoma | 111 |
| KSHV miRNAs miR-K12-1, -K12-3-3p, -K12-6-3p, -K12-11 | THBS1 |  | 112 |

Listed for each miRNA is its target gene(s). A directly regulatory connection was demonstrated between each miRNA and target a by a luciferase assay using the 3' untranslated region (UTR) of the mRNA.

Abbreviations: AIB1, amplified in breast cancer 1; BART, BamH1A rightward transcript; Ccnd2, cyclin D2; Ccne2, cyclin E2; CTGF, connective tissue growth factor; DNMT, DNA methyltransferase; EBV, Epstein-Barr virus; ERBB, v-erb-b2 erythroblastic leukemia viral oncogene homolog; E2F, E2F transcription factor; HCC, hepatocellular cancer; Hipk3, homeodomain-interacting protein kinase 3; HMGA, high-mobility group AT-hook 2; KSHV, Kaposi sarcoma-associated herpesvirus; LATs2, large tumor suppressor, homolog 2; MET, hepatocyte growth factor receptor; Mylip, myosin regulatory light chain-interacting protein; PTEN, phosphatase and tensin homolog; Rbp1-like, retinoblastoma-binding protein 1-like; THBS1, thrombospondin 1; TPM1, tropomyosin 1; trkC, tropomyosin-related kinase C; TβRII, transforming growth factor beta-receptor type II.

miRNA transfection results in up-and downregulation of a number of mRNAs. In several cases, a set of genes belonging to a particular cellular pathway are enriched in the genes whose expression was altered (35, 58–62). The length of the miRNA seed sequence is comparable to that of the consensus sequence of transcription factor-binding elements. Thus, although miRNAs are expected to directly regulate a large set of genes simultaneously, random chance is unlikely to explain the enrichment of genes in a particular pathway in the list of putative targets.

### MicroRNA AND CANCER

#### Profiling of MicroRNA Expression in Cancers

miRNAs are aberrantly expressed in a variety of cancers. This was first observed in miR-15a and -16-1, which are clustered at chromosome 13q14, a frequently deleted region in B cell chronic lymphocytic leukemia (CLL) and other cancers (63). Concomitantly, reduction of these two miRNAs was observed in the cancer samples relative to the normal tissues.

miRNome analyses have become easier with the introduction of microarray techniques (Table 1). Many miRNAs are found to be up-or downregulated in the cancer samples

Although some miRNAs are increased, most are repressed in cancers relative to normal tissue counterparts (64, 66–69). In agreement with these observations, global depletion of miRNAs by knockdown of the miRNA-processing machinery stimulates cell transformation and tumorigenesis in vivo (70). This implies that the miRNA alteration is not simply an end result of tumorigenesis but that it actively contributes to cancer development. Despite the general reduction of miRNAs in cancers, several miRNAs are upregulated, some of which undoubtedly play oncogenic roles.

---

**Mechanism for MicroRNA Deregulation in Cancers**

As described in the previous section, many miRNAs are deregulated in cancers. Aberrant expression of miRNAs can arise through a number of different mechanisms (Figure 2).

---

**Genomic abnormalities.** As exemplified by miR-15a and -16-1, chromosomal abnormality is one reason for miRNA deregulation in cancers. Tumorigenesis is often accompanied by chromosomal aberrations such as deletion,

---

**miRNA gene**  
**Epigenetic regulation**  
• CpG methylation  
• Histone modification  

**Transcription factor**  
Myc, E2Fs, p53, BCR-ABL, Twist,  
NF-κB, ER-α, AR, STAT3, ...

**Pri-miRNA** **5' cap**  
**(A)n**  

**Drosha processing**  
• Level of Drosha  
• Interacting proteins  

**Pre-miRNA**  

**Dicer processing**  
• Level of Dicer  

**Mature miRNA**  

**miRNP**  
(Ago, etc.)  

**Target mRNA**  
**5'UTR** **ORF** **3'UTR**  

**Sequence variation in target site**  
• SNP  
• Alternative splicing (trkC)  

**Separation of 3'UTR from ORF**  
• Chromosomal translocation (HMGA2)

---

**Figure 2**  
Various mechanisms for deregulation of microRNA (miRNA) expression. Abbreviations: miRNP, ribonucleoprotein complex containing miRNA; ORF, open reading frame; SNP, single nucleotide polymorphism; UTR, untranslated region.

amplification, and translocation. In silico analysis has revealed that a significant fraction of miRNAs are mapped to these cancer-associated genomic regions or fragile sites in human (71) and in mouse (72). In many cases, miRNA levels are correlated with changes in copy number of the genomic loci, as demonstrated by comparison between array comparative genomic hybridization data and miRNA expression data (66, 73–78).

**Epigenetic factors.** Epigenetic factors can also affect miRNA expression. In many cancers, hypermethylation of CpG islands in promoter regions results in heritable transcriptional silencing of tumor-suppressor genes. Gene silencing by DNA methylation is closely related to histone modification. In silico analyses have indicated CpG islands near dozens of miRNAs (79). In addition, some miRNAs are upregulated (a) upon the exposure of cells to the demethylating agent 5-aza-2′-deoxycytidine (79), (b) upon mutation of DNA methyltransferases (DNMTs) (80), and (c) upon histone deacetylase inhibitor treatment (81). These studies have identified some miRNAs that are repressed by CpG hypermethylation in cancers relative to normal tissue. Representative examples include miR-9-1 in breast cancer

(79) and miR-124a in colorectal tumors (80). In the case of miR-124a, hypermethylation is tumor type specific, as no methylation is seen in neuroblastoma. Also, epigenetic silencing of a miRNA may be a reflection of tissue specificity. For example, miR-124a is normally highly expressed in neuronal tissues, so its epigenetic repression in colorectal tumors is not surprising. miRNAs may counteract CpG methylation. For example, miR-29 directly targets DNMT-3A and -3B. In agreement with this observation, ectopic expression of miR-29 results in a global reduction of DNA methylation, subsequently leading to a depression of tumor-suppressor genes that had been silenced by promoter methylation in cancer cells (82).

Transcriptional regulation. Transcription factors may induce miRNAs by activating the transcription of pri-miRNAs. This mechanism has been well documented in several cases where tissue-specific miRNAs are turned on by transcription factors during differentiation. Given the wide impact of transcription factors on fundamental cellular processes, it is not surprising that many oncogenes or tumor suppressors are transcription factors. Many miRNA-transcription factor relationships have been discovered in cancers (listed in Table 3).

Table 3 MicroRNAs (miRNAs) regulated by transcription factors

| Transcription factor | Target miRNA | Note | Reference |
|----------------------|--------------|------|-----------|
| AR                   | miR-125b     | Prostate cancer | 205      |
| MYCN                 | miR-17-5p, -92, -320 | Neuroblastoma | 124      |
| Twist                | miR-10b      | Breast cancer   | 146      |
| p53                  | miR-34a, miR-34b-34c cluster | Cell lines | 60       |
| p53                  | miR-34a      | Cell lines      | 137      |
| p53                  | miR-34a, miR-34b-34c cluster | Cell lines and in vivo | 62       |
| Stat 3               | miR-21       | Myeloma cell lines | 91       |
| HIF                  | miR-26, -210 |          | 150      |
| E2F3                 | miR-17-92 cluster |          | 125      |
| E2F1, 2, 3           | miR-17-92 cluster |          | 126      |
| Myc                  | miR-17-92 cluster | Angiogenesis in colon cancer model | 147      |

Listed are miRNAs whose promoter regions were analyzed by a luciferase reporter assay and were shown to bind to the transcription factor by chromatin immunoprecipitation assay.

Abbreviations: AR, androgen receptor; E2F, E2F transcription factor; HIF, hypoxia-inducing factor; MYCN, Myc myelocytomatosis viral-related oncogene, neuroblastoma derived; Stat 3, signal transducer and activator of transcription 3.
We discuss three of these—p53, c-Myc, and E2F—in detail below.

**Regulation at microRNA processing steps.**

The steady-state level of a mature miRNA is determined not only by the transcription rate of pri-miRNA but also by the processing efficiency of its precursors and by its stability. miRNAs often exhibit a discrepancy in expression of the mature form relative to that of the precursor (83–88). Although miRNAs in a genomic cluster are usually expressed from a common pri-miRNA, the levels of individual miRNAs in the cluster are not necessarily coordinated (89, 90). A time-course experiment after induction of pri-miR-21 revealed delayed kinetics in accumulation of mature miR-21 (91). Collectively, these observations indicate that miRNA processing and stability are important factors that determine miRNA expression level.

This mechanism has been confirmed in cancers by a comprehensive analysis of expression data (92). As stated above, miRNAs are generally reduced in cancers relative to the normal tissues. When a miRNA resides within a gene, the host gene can be regarded as the pri-miRNA. Comparison of microarray data between mRNA and miRNA revealed that the miRNA reduction in cancers is poorly correlated with a reduction of the host gene expression. Aberrant expression of miRNAs during tumorigenesis is presumably often due to alterations at posttranscriptional steps.

The expression levels of Dicer or Drosha are altered in a number of cancers (70, 93–96). Drosha upregulation is seen in more than half of cervical squamous cell carcinoma (SCC) specimens and is likely due to the copy number gain at chromosome 5p, where the *Drosha* gene is located (95). Hierarchical clustering of miRNA expression data successfully classified cervical SCC samples into two groups according to Drosha overexpression. Notably, some miRNAs were reduced upon Drosha overexpression, indicating that individual miRNAs respond differentially to an elevation of the miRNA processing machinery. Interestingly,

Drosha was reported to interact with an oncogenic fusion protein derived from a chromosomal translocation in some leukemias (97). This interaction affects pri-miRNA selection of Drosha and, as a result, influences miRNA expression patterns.

**Consequence of Aberrant MicroRNA Expression in Cancers**

miRNAs regulate the expression of their target mRNAs, so over- or underexpression of miRNAs is expected to result in down- or up-regulation, respectively, of the protein product of the target mRNAs. It is easy to implicate a miRNA in a cancer if a direct target of a miRNA is an oncogene or a tumor suppressor. Since the publication of a study showing that let-7 miRNA directly regulates *RAS* oncogene (98), a number of other miRNA-target pairs have been studied. However, the number of experimentally validated pairs is still small relative to (*a*) the number of miRNAs shown to be aberrantly expressed in tumors and (*b*) the number of in silico predicted pairs, mostly because current target prediction algorithms are not very accurate. A current list of miRNA-target pairs implicated in various cancers is given in Table 2, and important examples are discussed below.

If we compare global gene expression profiles in cancers and normal tissues, we find that many miRNAs and mRNAs are deregulated. Therefore, it is plausible that tumorigenesis (or at least the progression of a cancer) results from changes in the entire miRNome, rather than from the change of a single miRNA that regulates an oncogenic (or tumor-suppressive) target gene.

Some miRNAs appear to be deregulated in cancers much more frequently than others (Supplemental Table 1). These miRNAs may play key roles during tumorigenesis. For example, the miR-17-92 cluster and miR-155 have been shown experimentally to be bona fide oncogenes, as their ectopic expression accelerates tumor development. These two miRNAs are discussed in detail in separate sections.
# Aberrant Action of MicroRNAs Without Alteration of MicroRNA Expression Level

The function of protein-coding genes is rendered abnormal by point mutations, which either transform proto-oncogenes to oncogenes or abrogate functions of tumor-suppressor genes. In theory, the same mechanism of activation/inactivation may apply to miRNAs. However, mutation in mature miRNA sequence seems to be a rare event (72, 99–101), presumably due to the small size of the mature miRNA. In addition, miRNAs are thought to be more tolerant of point mutations or of single nucleotide insertion/deletion mutations than of protein-coding genes.

In contrast, sequence variation in miRNA target sites may occur and may play a role in cancer. In silico analyses of expressed sequence tag and single nucleotide polymorphism (SNP) databases indicate different allele frequencies of miRNA-binding sites in cancers versus normal tissues (102). Several experiments have shown that sequence polymorphisms in miRNA target sites affect miRNA-mRNA interaction and are implicated in disease phenotypes (100, 103–105) (Figure 2).

An interesting illustration of this mechanism can be found in *let-7* and its direct target oncogene, *HMGA2* (38, 106). Chromosomal rearrangements at the *HMGA2* locus in several tumors separate the open reading frame (ORF) from the 3′ UTR that contains *let-7* target sites. As a result, *HMGA2* escapes from suppression by *let-7*, is overexpressed, and promotes tumorigenesis (Figure 2).

An alternative splicing event may result in a different 3′ UTR that displays different miRNA target sites, as exemplified in the targeting of tropomyosin-related kinase C by miR-9, -125a, and -125b. One mRNA isoform encodes a truncated ORF that is functionally dominant negative to the intact protein. In this isoform, the 3′ UTR contains the target sites of these miRNAs. In contrast, the target sites are absent in another isoform encoding the intact ORF; only the former isoform was repressed by the

miRNAs (107). Although the stop codon is usually located in the last exon, generation of different 3′ UTRs by multiple polyadenylation sites or alternative splicing has been known to occur in a small but significant fraction of genes (108). Thus, variation of 3′ UTR and of attendant miRNA target sites is expected to be a mechanism for oncogene activation or tumor-suppressor inactivation (Figure 2).

Recently, Steitz and colleagues (109) reported that miRNAs activate the translation of the target mRNA in cells arrested at the G₀/G₁ stage. In addition to aberrant miRNA expression, the switch from repression to activation should be considered in studying the role of miRNAs in differentiation or tumorigenesis, as the same miRNA may exert opposite effects in resting cells and proliferating cancer cells in a given tissue.

## Viruses and MicroRNAs in Tumorigenesis

Several viruses regulate tumorigenesis by expressing viral oncogenes or by activating cellular oncogenes through integration of viral DNA into genomic loci. Both mechanisms are applicable to oncogenic miRNAs. For example, Epstein-Barr virus (EBV)-encoded miRNAs (110) directly target a viral oncogene, *LMP1*, whose overexpression is deleterious to host cells. EBV-encoded miRNAs are thought to enhance EBV-mediated cellular transformation by adjusting LMP1 level to a sublethal dose (111). Another study showed that a viral miRNA repressed a host tumor-suppressor gene. In this study, miRNAs from Kaposi sarcoma-associated herpesvirus (KSHV) directly targeted an antiangiogenic factor, thrombospondin-1 (*Tsp-1*) (112). Thus viral miRNAs act as viral oncogenes.

Viral integration near miRNA loci may lead to aberrant expression of miRNAs. Indeed, the *BIC* locus harboring miR-155 was originally described as a frequent integration site in virally induced lymphomas (113). Viral integration sites are often mapped to miRNA loci (114) such as the miR-17-92 cluster (115), the

**KSHV:** Kaposi sarcoma–associated herpesvirus

miR-106a-363 cluster (116), the miR-29a-29b-1 cluster (117), and the miR-106a cluster (118).

**Various Cancer Types**

Comprehensive miRNA profiles have been reported in clinical specimens from various cancers (Table 1). Consistent with the notion that miRNAs are expressed in a tissue-specific manner, miRNAs differ among cancers from various tissue origins. Also, a few miRNAs appear to be frequently deregulated in many cancers (Supplemental Table 1), suggesting that these miRNAs regulate fundamental processes such as cell proliferation and apoptosis.

**Oncogenic or Tumor-Suppressive MicroRNAs**

**miR-17-92 cluster.** The miR-17-92 cluster, located at chromosome 13q31.3 in humans, is composed of six miRNAs (mir-17, -18a, -19a, -20a, -19b-1, and -92a-1). There exists a similar cluster at chromosome 10 known as miR-106a-363, which also contains six miRNAs (mir-106a, -18b, -20b, -19b-2, -92a-2, and -363). The miR-17-92 cluster contains the first miRNAs demonstrated to be oncogenic. The development of B cell lymphoma is significantly accelerated by the forced expression of the miR-17-92 cluster in transgenic mice overexpressing the *c-myc* oncogene (119).

Consistent with its oncogenic role, the miR-17-92 cluster is upregulated in a variety of cancers including lymphomas (74, 75, 119, 120), lung cancers (77, 121), and others (122). There appear to be two mechanisms for upregulation of this cluster in cancers (Figure 3). One is the amplification of the chromosome 13q31 locus in several lymphomas and other cancers (74, 77, 78, 120). The other is transcriptional activation of the pri-miRNA, wherein an oncogenic transcription factor c-Myc binds the genomic locus upstream of the miR-17-92 cluster and activates its expression (123). In neuroblastoma cells, MYCN, a protein highly homologous to c-Myc, appears to activate this cluster instead of c-Myc (124). E2Fs also activate this cluster (125, 126) (Table 3).

Given that the E2Fs are direct targets of miR-17 and -20a, the miR-17-92 cluster constitutes a complex regulatory network with c-Myc and E2Fs. E2F1 and c-Myc are known to activate each other to form a positive feedback loop

![Figure 3](#fig3)

The miR-17-92 pathway.
(Figure 3). All of the E2Fs (E2F1, -2, and -3) are capable of activating the miR-17-92 cluster and are subject to repression by miR-17 and -20a from this cluster (Table 2). However, degrees of activation and repression vary between individual pairs of miRNA and E2F family members. In general, E2Fs are thought to be proliferative, but E2F1 is also proapoptotic. Although this network is too complicated to describe in a straightforward manner, the miR-17-92 cluster is clearly responsible for fine-tuning the regulatory network. Upon a proliferative cue, c-Myc and E2Fs turn on the miR-17-92 cluster. This cluster represses E2Fs, thereby preventing the uncontrolled amplification of the positive feedback loop between E2Fs and c-Myc. In addition, repression of E2F1 by the miRNAs may help minimize the proapoptotic potential of E2F1. This network is depicted in Figure 3.

**miR-21.** miR-21 is upregulated in almost all kinds of cancers (Supplemental Table 1). miR-21 is transcriptionally activated by signal transducer and activator of transcription 3 (Stat3) in the interleukin-6 signaling pathway (91) (Table 3). miR-21's role in invasion and metastasis is well characterized and is discussed further below.

**miR-155.** As with the miR-17-92 cluster, the oncogenic potential of miR-155 has been demonstrated by transgenic mice expressing miR-155 in B cells (127). The primary transcript for miR-155 is the *BIC* gene, which was originally known as a common viral integration site in lymphomas in chicken. High expression of miR-155 (*BIC*) is reported in various B cell malignancies (128–130) (Supplemental Table 1). miR-155 seems to be regulated at multiple levels. miR-155 (*BIC*) appears to be regulated by nuclear factor-kappa B (NF-κB) through the B cell receptor-mediated signaling pathway (85) or through the Toll-like receptor-activated signaling pathway (131). However, a discrepancy between the levels of miR-155 and *BIC* suggests that miR-155 may also be regulated during processing steps

(129). For example, the induction of *BIC* does not lead to upregulation of miR-155 in several Burkitt lymphoma cell lines. In contrast, miR-155 is concordantly increased upon the induction of *BIC* in Raji, an EBV-latency type III-positive Burkitt lymphoma cell line (85).

**let-7.** let-7 is one of the earliest-discovered miRNAs. In human, there are 12 paralogous let-7s. let-7 is almost absent during embryonic stages or tissues, whereas high expression of let-7 is seen in most differentiated tissues. The reduction of let-7 in cancers is reminiscent of let-7 expression during development in that it is most decreased in less-differentiated, advanced stages of cancer cells with mesenchymal characters (132).

let-7 is probably more abundant than any other miRNA, so reduction of let-7 may have a prominent effect on cell physiology compared to other miRNAs. To support this hypothesis, enhanced cellular transformation by global miRNA depletion was largely recapitulated by inhibition of let-7 alone (70). Well-known oncogenes such as *RAS*, *c-Myc*, and *HMGA2* are validated as direct targets of let-7 (Table 2). Hence, let-7 is an important tumor suppressor.

**miR-34s (miR-34a and miR-34b/-34c clusters).** The miR-34s recently acquired notoriety because they are induced by p53 (Figure 4). Consistent with p53's role as a tumor suppressor, miR-34s are downregulated in several tumors such as non-small cell lung cancers (60) and pancreatic cancers (61) (Supplemental Table 1). However, reduction of the miR-34s is not always correlated with p53 loss, suggesting a p53-independent mechanism of miR-34 reduction in some cancers. In fact, miR-34a is located in 1p36, a locus that is frequently deleted in a number of cancers (76). We return to our discussion of p53 regulation and miR-34s' roles below.

Although the miR-34s are thought to be tumor suppressors, they have been found to be upregulated in several cancers including renal cell carcinoma (133), colon cancer (58), and
Other miRNAs:
miR-30c, -103, -26a,
-107, -182, -129

Other miRNAs?

Genotoxic stress

Apoptosis
Cell-cycle arrest
Senescence

Figure 4

The p53 and miR-34 pathway. Abbreviation: MET, hepatocyte growth factor receptor.

hepatocellular carcinoma (134) (Supplemental Table 1). The role of high miR-34 and p53 status in these tumors awaits investigation.

### Aspects of Cancer Biology Regulated by MicroRNAs

#### Cell cycle.
Cell-cycle regulators often act as oncogenes or as tumor suppressors. The best-characterized example is the cell-cycle inhibitor p27(Kip1). p27(Kip1) is a tumor suppressor, as indicated by its low levels in some cancers. In addition, $p27(Kip1)$ mutation predisposes cells to tumorigenesis upon exposure to carcinogens. p27(Kip1) binds to Cdk2-cyclin E and prevents G1-to-S transition. $p27(Kip1)$ is a direct target of miR-221 and -222 in glioblastomas (48, 135) and in prostate cancer cells (136). In these cancer cells, $p27(Kip1)$ is anticorrelated with miR-221 and -222. Targeting $p27(Kip1)$ is responsible for the proproliferative role of these miRNAs, as artificial knockdown of $p27(Kip1)$ mirrors the phenotype of these miRNAs. miR-221 and -222 are overexpressed in other cancers (see Supplemental Table 1), suggesting that they play a role in a wide range of cancers.

In addition to regulating p27(Kip1), miRNAs regulate other cell-cycle proteins including Cdk6; Cdc25A; Ccnd2 (cyclin D2) (59); Cdk4 (62), a Rb-family protein (89); and p180 of DNA polymerase α (37). A number of miRNAs have been shown to perturb normal cell cycle when overexpressed or inhibited (37, 48, 62, 70, 137–139) (Supplemental Table 2). However, oscillation of a miRNA during the normal cell cycle has not yet been reported.

#### Programmed cell death.
Apoptosis is an active process controlled by a gene expression program that varies depending on the biological context. Because a balance between proliferation and apoptosis is essential for tissue homeostasis and proper differentiation, aberrant apoptosis may give rise to tumors. miRNAs participate in tumorigenesis by directly targeting antiapoptotic genes. Representative examples include the repression of antiapoptotic genes $Mcl-1$ and $Bcl-2$ by miR-29b (84) and by miR-34s (60), -15a, and -16 (140), respectively (Table 2). The loss of these miRNAs due to mutation of p53 or deletion of chromosome 13q14 leads to an increase in
the antia apoptotic gene expression and persistence of tumor cells that would normally have been removed by apoptosis. It is very likely that miRNAs target other genes in the apoptotic pathway, as transfection or expression of a number of miRNAs is associated with apoptosis (69, 76, 121) (Supplemental Table 2).

**p53.** p53, a sequence-specific transcription factor, is known as the guardian of the genome owing to its critical role in regulation of the cell cycle and apoptosis in the face of genomic damage. Genotoxic stress and oncogene activation activate p53 to modulate the transcription of several target genes. p53 is the most extensively studied tumor suppressor and its importance is underscored by mutation of p53 in almost 50% of human cancers.

miRNA profiling after p53 induction indicated miR-34a, -34b, and -34c (collectively, the miR-34s) as the most upregulated miRNAs (61, 62, 137, 141) (Figure 4). These miRNAs are induced after genotoxic stress in a p53-dependent manner in vitro and in vivo (62, 141). miR-34b and -34c are clustered at chromosome 11, whereas miR-34a is located in a separate genomic locus. Both pri-miRNAs are directly activated by p53. The miR-34s seem to be critical downstream effectors of p53, as ectopic expression of the miR-34s recapitulate the phenotype of p53 activation. The miR-34s promote cell-cycle arrest, apoptosis, and senescence (58, 60–62, 137, 141) (Supplemental Table 2). These effects are explained by the repression of several direct targets of the miR-34s such as *Bcl-2* (60), *Cdk4*, and *hepatocyte growth factor receptor* (62) (Table 2).

In addition to the miR-34s, other miRNAs may be important in the p53 pathway. miR-30c, -103, -26a, -107, and -182 were induced clearly, albeit less robustly, upon DNA damage in a p53-dependent manner (61). miR-26a expression was also shown to be dependent on p53 (142). In another approach, searching for p53-binding elements in DNA sequence near miRNAs identified miR-129 as a good candidate for regulation by p53 (143).

**Invasion and metastasis.** Features of malignant tumors, distinct from benign tumors, include invasion and metastasis. Malignant tumors are fatal, mostly due to their capacity to invade neighboring tissues and metastasize through the bloodstream to distant organs. An effect of miRNAs on invasion and migration has been reported (Supplemental Table 2). Ectopic expression of miR-125 impairs cell motility and invasion in a breast cancer cell line (144), and reduction of global miRNA expression enhances migration of cells (70).

These sporadic in vitro observations were followed by more detailed studies on miR-10b and miR-21. miR-21 is one of the most frequently upregulated miRNAs in cancers (Supplemental Table 1). It promotes cell motility and invasion by directly targeting *PTEN* (*phosphatase and tensin homolog*), a tumor suppressor known to inhibit cell invasion by blocking the expression of several matrix metalloproteases (134). Another pathway was recently reported in colorectal cancers, where miR-21 promotes invasion, intravasation, and metastasis by downregulating *Pdc*d4 (145).

miR-10b is the other miRNA implicated in metastasis. In metastatic breast cancer cells, miR-10b is upregulated, presumably as a result of transcriptional activation by a transcription factor, Twist. Ectopic expression of miR-10b promotes invasion, intravasation, and metastasis in otherwise noninvasive or nonmetastatic breast cancer cell lines. miR-10b directly targets *HOXD10*, whose reduction induces the expression of a well-characterized prometastatic gene, *RhoC* (146).

**Angiogenesis.** Recruiting blood vasculature is crucial to the survival of neoplastic cells. So far, the miR-17-92 cluster has been characterized in this process. The stimulation of neovascularization by c-Myc involves a downregulation of antiangiogenic factor *Tsp-1*. c-Myc represses *Tsp-1* and a related protein, *connective tissue growth factor (CTGF)* by activating the miR-17-92 cluster. *Tsp-1* and *CTGF* appear to be direct targets of miR-19 and -18, respectively. Ectopic expression of the miR-17-92 cluster is sufficient

www.annualreviews.org • MicroRNAs in Cancer 215
for promoting angiogenesis (147). A recent observation indicates that other miRNAs—miR-378 and -27a—may play a role in angiogenesis (139, 148) (Supplemental Table 2). Viral miRNAs may also play a role in angiogenesis, as *Tsp-1* has been shown to be a direct target of KSHV miRNAs (112).

**Others.** The microenvironment inside a solid tumor is usually hypoxic. Tumor cells under hypoxia tend to be resistant to therapies and to have a poor prognosis. Upregulation of certain miRNAs under hypoxia (149, 150) (Table 3) suggests that these miRNAs may influence the phenotype of hypoxic tumor cells. Cancers are predisposed by other external factors, such as genotoxic stress, a folate-deficient diet, and exposure to arsenic. There are a few reports of miRNA profiles under these conditions (151, 152), but whether miRNA changes are responsible for the cancer predisposition remains to be investigated.

### Clinical Applications

#### Diagnostic tools.
miRNAs may be used as diagnostic or prognostic tools, as miRNA expression profiles reflect tumor origin, stage, and other pathological variables. miRNAs can function as accurate molecular markers because they are relatively stable and resistant to RNase degradation—probably because of their small size (35, 153, 154). Researchers have shown that miRNAs can be isolated and quantitated from formalin-fixed paraffin-embedded (FFPE) specimens. qRT-PCR and microarray data were reliably and reproducibly obtained from FFPE samples that had been routinely processed and stored for 10 years. The data from FFPE samples are consistent with those from frozen samples (155, 156).

The development of qRT-PCR methods has improved the sensitivity of miRNA detection down to a few nanograms of total RNA (68, 157, 158). This amount can easily be obtained by fine-needle aspiration biopsies (FNABs); in fact, there has been a report of successful

miRNA measurement by qRT-PCR on FNAB samples (159).

miRNA markers that could be used for cancer diagnosis are becoming available. For example, miR-196a is high in pancreatic ductal adenocarcinoma (PDAC) but low in normal tissues and chronic pancreatitis. miR-217 exhibits the opposite expression pattern (Supplemental Table 1). Thus, the ratio of miR-196a to miR-217, calculated by qRT-PCR measurement of the two miRNAs from a tiny amount of total RNA, indicates whether the sample contains PDAC (160). Once reliable indicator miRNAs are chosen, they will likely yield easy and accurate tools for cancer diagnosis.

### Cancer therapeutic tool.
A number of miRNAs affect the growth of cancer cells in vitro and in vivo when overexpressed or inhibited. Therefore, cancer cell growth can be controlled by manipulating miRNAs. Overexpression or inhibition of miRNAs can be achieved in several ways. Synthetic miRNA mimics include siRNA-like oligoribonucleotide duplex (161) and chemically modified oligoribonucleotide (162). Conversely, miRNAs can be inhibited by variously modified antisense oligonucleotides such as 2′-O-methyl antisense oligonucleotide, antagomirs, and so on (reviewed in Reference 163). As the first successful tool for knockdown of a miRNA in vivo, antagomirs are of special interest (164). Antagomirs appear to be delivered to all tissues (except brain) after tail-vein injections into mice. The therapeutic value of an antagomir would be greatly enhanced by technical improvements for selective tumor-specific or tissue-specific delivery.

Synthetic oligonucleotides are effective in vivo for at most a couple of weeks, as has been demonstrated by experiments involving cancer cells engrafted in mice (165) and tail-vein injection to mice (166). To circumvent this limitation, miRNAs can be stably expressed through transcription of hairpin RNA from plasmid vector (reviewed in Reference 167). Recently, artificial overexpression of a miRNA

target sequence was shown to inhibit the miRNA function, presumably by titrating the miRNA away from endogenous targets (168, 169). Thus it should be possible to apply such competitive inhibitors for long-term sequestration of a miRNA.

Nonspecific side effects are as important as effectiveness and duration of miRNA expression (or inhibition). Some investigators have argued that miRNA mimics or inhibitors are specific enough to distinguish between similar miRNAs (170, 171). However, cross-reactivity between miRNAs of similar sequence is likely to be unavoidable at high doses of antagonists or agonists. Another possible side effect is that high expression of miRNA mimics may interfere with endogenous miRNA action by saturating the cellular machinery for miRNA processing or action. This may result in a change in expression of other miRNAs, leading to a deleterious effect in the cells. Indeed, a fatal side effect as a result of saturation of miRNA pathway has been reported (172). To minimize undesirable side effects, the expression or knockdown of a miRNA should be improved so that it is more accurate and controllable. An alternate approach to improving specificity is to target the pre-miRNAs with antisense or siRNA strategies (173, 174).

Some chemical compounds alter expression of a group of miRNAs (81); therefore, it may be possible to screen for drugs that could shift the miRNome in a cancer cell toward that of the normal tissue. NCI-60 cancer cell lines may serve as a platform for this screening, as their miRNA expression profiles have been published (68, 175). By modulating multiple miRNAs simultaneously, such a miRNome-modifying approach may be much more effective for therapy than strategies that aim to regulate a single miRNA.

miRNAs affect the drug sensitivity of a cell (165, 176) (Supplemental Table 2). Expression or inhibition of a miRNA can therefore be combined with treatment of a drug or other cytotoxic therapy. One example is miR-21 inhibition together with a secreted form of tumor necrosis factor–related apoptosis-inducing ligand, which results in a complete eradication of glioblastoma cells (177).

Collectively, preliminary results suggest that miRNAs could be useful for cancer therapy. However, there is still a significant gap between basic research on miRNAs and clinical application. Extensive preclinical and translational research is necessary to increase the efficacy and decrease the side effects of miRNAs in vivo. In parallel, we need to expand our knowledge of the interactions between miRNAs and gene expression programs and how these interactions are altered in tumorigenesis.

**CONCLUDING REMARKS**

The discovery of miRNAs has added another dimension to the study of the regulation of gene expression. In addition to transcriptional regulation, posttranscriptional repression by miRNAs may act like rheostats for fine-tuning of gene expression. miRNAs at end stages of differentiation may contribute to keeping cells differentiated by suppressing a large number of genes simultaneously.

In recent years, there has been an explosion of publications on miRNAs. Many of these have emphasized the role of miRNAs in cancer biology. miRNAs play a role in almost all aspects of cancer biology, including proliferation, apoptosis, invasion/metastasis, and angiogenesis. New technical developments are enabling investigators to describe a variety of cancer miRNomEs. Therefore, more miRNAs are expected to be identified as oncogenes or as tumor suppressors. Characterization of individual miRNA pathways will be greatly facilitated by an improvement of target prediction algorithms.

miRNA expression is associated with clinical variables of cancers, so miRNAs can be readily used as tools for cancer diagnosis and prognosis. For therapeutic benefit, the methods to manipulate miRNAs in vivo should become more robust.

SUMMARY POINTS

1. miRNAs exhibit an expression pattern characteristic of tumor type, stage, or other clinical variables. Thus miRNAs can be used for cancer diagnosis and prognosis.
2. miRNAs exert an active role in the etiology or progression of cancers by regulating their direct-target oncogenes or tumor-suppressor genes.
3. miRNAs play roles in diverse aspects of cancer such as proliferation, apoptosis, invasion/metastasis, and angiogenesis.
4. Artificial expression/inhibition of a miRNA can be therapeutic.

APPENDIX

**Other Small RNAs?**

In addition to siRNAs and miRNAs, other types of small RNA have been reported in yeast, *Drosophila*, *C. elegans*, and plants. These small RNAs include tiny noncoding RNAs, repeat associated small interfering RNAs, scan RNAs, and others (reviewed in Reference 178). These RNAs are thought to be related to miRNA and siRNA, but are distinct from them. Recently, Piwi-interacting RNA, another small RNA that is slightly larger than miRNA, was discovered from mammalian testes. Other classes of small RNA may exist but have yet to be discovered, given that a significant portion of transcripts are noncoding RNA. Whereas protein-coding genes comprise only 1–2% of the human genome, a significant fraction (~15%) of the human genome is transcribed; these transcripts remain the “dark matter” of the genome (179, 180). The discovery of noncoding RNAs is expected to be greatly accelerated by the widespread adaptation of high-throughput deep-sequencing techniques.

**Other Diseases**

miRNAs (or miRNA machinery) have been implicated in diseases other than cancer (182), for example Tourette’s syndrome, fragile X syndrome (181), DiGeorge syndrome (23), myotonic dystrophy, spinocerebellar ataxia type 3 (182), and schizophrenia and schizoaffective disorder (183). Of particular interest is myotonic dystrophy type 1, which is characterized by expansion of CTG repeat in the 3′ UTR of the *dmpk* gene. Tandem CAG sequences are found in some miRNA seed sequences and thus are able to interact with the CTG repeats, raising the possibility that these miRNAs may be involved in DM1 pathogenesis (184).

DISCLOSURE STATEMENT

The authors are not aware of any biases that might be perceived as affecting the objectivity of this review.

ACKNOWLEDGMENTS

Our work on microRNAs was supported by grants from the U.S. Department of Defense (PC050114) and the National Institutes of Health (AR053948).

LITERATURE CITED

1. Kennedy D. 2002. Breakthrough of the year. *Science* 298:2283
2. Fire A, Xu S, Montgomery MK, Kostas SA, Driver SE, Mello CC. 1998. Potent and specific genetic interference by double-stranded RNA in *Caenorhabditis elegans*. *Nature* 391:806–11
3. Lee RC, Feinbaum RL, Ambros V. 1993. The *C. elegans* heterochronic gene *lin-4* encodes small RNAs with antisense complementarity to *lin-14*. *Cell* 75:843–54
4. Wightman B, Ha I, Ruvkun G. 1993. Posttranscriptional regulation of the heterochronic gene *lin-14* by *lin-4* mediates temporal pattern formation in *C. elegans*. *Cell* 75:855–62
5. Ambros V, Bartel B, Bartel DP, Burge CB, Carrington JC, et al. 2003. A uniform system for microRNA annotation. *RNA* 9:277–79
6. Lagos-Quintana M, Rauhut R, Lendeckel W, Tuschl T. 2001. Identification of novel genes coding for small expressed RNAs. *Science* 294:853–58
7. Lee RC, Ambros V. 2001. An extensive class of small RNAs in *Caenorhabditis elegans*. *Science* 294:862–64
8. Lau NC, Lim LP, Weinstein EG, Bartel DP. 2001. An abundant class of tiny RNAs with probable regulatory roles in *Caenorhabditis elegans*. *Science* 294:858–62
8a. miRBase Release 10.0. 2007. Cambridge, UK: Wellcome Sanger Trust Inst.
9. Bentwich I, Avniel A, Karov Y, Aharonov R, Gilad S, et al. 2005. Identification of hundreds of conserved and nonconserved human microRNAs. *Nat. Genet.* 37:766–70
10. Berezikov E, Guryev V, van de Belt J, Wienholds E, Plasterk RH, Cuppen E. 2005. Phylogenetic shadowing and computational identification of human microRNA genes. *Cell* 120:21–24
11. Yu J, Wang F, Yang GH, Wang FL, Ma YN, et al. 2006. Human microRNA clusters: genomic organization and expression profile in leukemia cell lines. *Biochem. Biophys. Res. Commun.* 349:59–68
12. Rodriguez A, Griffiths-Jones S, Ashurst JL, Bradley A. 2004. Identification of mammalian microRNA host genes and transcription units. *Genome Res.* 14:1902–10
13. Smalheiser NR, Torvik VI. 2005. Mammalian microRNAs derived from genomic repeats. *Trends Genet.* 21:322–26
14. Schmittgen TD, Lee EJ, Jiang J, Sarkar A, Yang L, et al. 2008. Real-time PCR quantification of precursor and mature microRNA. *Methods* 44:31–38
15. Liu CG, Spizzo R, Calin GA, Croce CM. 2008. Expression profiling of microRNA using oligo DNA arrays. *Methods* 44:22–30
16. Wienholds E, Kloosterman WP, Miska E, Alvarez-Saavedra E, Berezikov E, et al. 2005. MicroRNA expression in zebrafish embryonic development. *Science* 309:310–11
17. Cummins JM, He Y, Leary RJ, Pagliarini R, Diaz LA Jr, et al. 2006. The colorectal microRNAome. *Proc. Natl. Acad. Sci. USA* 103:3687–92
18. Cai X, Hagedorn CH, Cullen BR. 2004. Human microRNAs are processed from capped, polyadenylated transcripts that can also function as mRNAs. *RNA* 10:1957–66
19. Lee Y, Kim M, Han J, Yeom KH, Lee S, et al. 2004. MicroRNA genes are transcribed by RNA polymerase II. *EMBO J.* 23:4051–60
20. Borchert GM, Lanier W, Davidson BL. 2006. RNA polymerase III transcribes human microRNAs. *Nat. Struct. Mol. Biol.* 13:1097–101
21. Lee Y, Ahn C, Han J, Choi H, Kim J, et al. 2003. The nuclear RNase III Drosha initiates microRNA processing. *Nature* 425:415–19
22. Denli AM, Tops BB, Plasterk RH, Ketting RF, Hannon GJ. 2004. Processing of primary microRNAs by the Microprocessor complex. *Nature* 432:231–35
23. Gregory RI, Yan KP, Amuthan G, Chendrimada T, Doratotaj B, et al. 2004. The Microprocessor complex mediates the genesis of microRNAs. *Nature* 432:235–40
24. Han J, Lee Y, Yeom KH, Nam JW, Heo I, et al. 2006. Molecular basis for the recognition of primary microRNAs by the Drosha-DGCR8 complex. *Cell* 125:887–901
25. Okamura K, Hagen JW, Duan H, Tyler DM, Lai EC. 2007. The mirtron pathway generates microRNA-class regulatory RNAs in *Drosophila*. *Cell* 130:89–100
26. Ruby JG, Jan CH, Bartel DP. 2007. Intronic microRNA precursors that bypass Drosha processing. *Nature* 448:83–86

27. Berezikov E, Chung WJ, Willis J, Cuppen E, Lai EC. 2007. Mammalian mirtron genes. *Mol. Cell* 28:328–36

28. Yi R, Qin Y, Macara IG, Cullen BR. 2003. Exportin-5 mediates the nuclear export of premicroRNAs and short hairpin RNAs. *Genes Dev.* 17:3011–16

29. Lund E, Guttinger S, Calado A, Dahlberg JE, Kutay U. 2004. Nuclear export of microRNA precursors. *Science* 303:95–98

30. Hutvagner G, McLachlan J, Pasquinelli AE, Balint E, Tuschl T, Zamore PD. 2001. A cellular function for the RNA-interference enzyme Dicer in the maturation of the let-7 small temporal RNA. *Science* 293:834–38

31. Salzman DW, Shubert-Coleman J, Furneaux H. 2007. P68 RNA helicase unwinds the human let-7 microRNA precursor duplex and is required for let-7-directed silencing of gene expression. *J. Biol. Chem.* 282:32773–79

32. Schwarz DS, Hutvagner G, Du T, Xu Z, Aronin N, Zamore PD. 2003. Asymmetry in the assembly of the RNAi enzyme complex. *Cell* 115:199–208

33. Khvorova A, Reynolds A, Jayasena SD. 2003. Functional siRNAs and miRNAs exhibit strand bias. *Cell* 115:209–16

34. Filipowicz W, Bhattacharyya SN, Sonenberg N. 2008. Mechanisms of post-transcriptional regulation by microRNAs: Are the answers in sight? *Nat. Rev. Genet.* 9:102–14

35. Lim LP, Lau NC, Garrett-Engele P, Grimson A, Schelter JM, et al. 2005. Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs. *Nature* 433:769–73

36. Bagga S, Bracht J, Hunter S, Massirer K, Holtz J, et al. 2005. Regulation by let-7 and lin-4 miRNAs results in target mRNA degradation. *Cell* 122:553–63

37. Kim HK, Lee YS, Sivaprasad U, Malhotra A, Dutta A. 2006. Muscle-specific microRNA miR-206 promotes muscle differentiation. *J. Cell Biol.* 174:677–87

38. Lee YS, Dutta A. 2007. The tumor suppressor microRNA let-7 represses the HMGA2 oncogene. *Genes Dev.* 21:1025–30

39. Liu J, Valencia-Sanchez MA, Hannon GJ, Parker R. 2005. MicroRNA-dependent localization of targeted mRNAs to mammalian P-bodies. *Nat. Cell Biol.* 7:719–23

40. Sen GL, Blau HM. 2005. Argonaute 2/RISC resides in sites of mammalian mRNA decay known as cytoplasmic bodies. *Nat. Cell Biol.* 7:633–36

41. Eulalio A, Behm-Ansmant I, Schweizer D, Izaurralde E. 2007. P-body formation is a consequence, not the cause, of RNA-mediated gene silencing. *Mol. Cell Biol.* 27:3970–81

42. Chu CY, Rana TM. 2006. Translation repression in human cells by microRNA-induced gene silencing requires RCK/p54. *PLoS Biol.* 4:e210

43. Lewis BP, Shih IH, Jones-Rhoades MW, Bartel DP, Burge CB. 2003. Prediction of mammalian microRNA targets. *Cell* 115:787–98

44. Grimson A, Farh KK, Johnston WK, Garrett-Engele P, Lim LP, Bartel DP. 2007. MicroRNA-targeting specificity in mammals: determinants beyond seed pairing. *Mol. Cell* 27:91–105

45. Kedde M, Strasser MJ, Boldajipour B, Vrielink JA, Slanchev K, et al. 2007. RNA-binding protein Dnd1 inhibits microRNA access to target mRNA. *Cell* 131:1273–86

46. Maziere P, Enright AJ. 2007. Prediction of microRNA targets. *Drug Discov. Today* 12:452–58

47. Zhu S, Si ML, Wu H, Mo YY. 2007. MicroRNA-21 targets the tumor suppressor gene tropomyosin 1 (TPM1). *J. Biol. Chem.* 282:14328–36

48. Le Sage C, Nagel R, Egan DA, Schrier M, Mesman E, et al. 2007. Regulation of the p27 (Kip1) tumor suppressor by miR-221 and miR-222 promotes cancer cell proliferation. *EMBO J.* 26:3699–708

49. Karginov FV, Conaco C, Xuan Z, Schmidt BH, Parker JS, et al. 2007. A biochemical approach to identifying microRNA targets. *Proc. Natl. Acad. Sci. USA* 104:19291–96

50. Lewis BP, Burge CB, Bartel DP. 2005. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets. *Cell* 120:15–20

51. Xie X, Lu J, Kulbokas EJ, Golub TR, Mootha V, et al. 2005. Systematic discovery of regulatory motifs in human promoters and 3′ UTRs by comparison of several mammals. *Nature* 434:338–45

52. Gantier MP, Sadler AJ, Williams BR. 2007. Fine-tuning of the innate immune response by microRNAs. *Immunol. Cell Biol.* 85:458–62

53. Calame K. 2007. MicroRNA-155 function in B cells. *Immunity* 27:825–27

54. Poy MN, Eliasson L, Krutzfeldt J, Kuwajima S, Ma X, et al. 2004. A pancreatic islet-specific microRNA regulates insulin secretion. *Nature* 432:226–30

55. Greco SJ, Rameshwar P. 2007. MicroRNAs regulate synthesis of the neurotransmitter substance P in human mesenchymal stem cell-derived neuronal cells. *Proc. Natl. Acad. Sci. USA* 104:15484–89

56. Cheng HY, Papp JW, Varlamova O, Dziema H, Russell B, et al. 2007. MicroRNA modulation of circadian-clock period and entrainment. *Neuron* 54:813–29

57. Jopling CL, Yi M, Lancaster AM, Lemon SM, Sarnow P. 2005. Modulation of hepatitis C virus RNA abundance by a liver-specific microRNA. *Science* 309:1577–81

58. Tazawa H, Tsuchiya N, Izumiya M, Nakagama H. 2007. Tumor-suppressive miR-34a induces senescence-like growth arrest through modulation of the E2F pathway in human colon cancer cells. *Proc. Natl. Acad. Sci. USA* 104:15472–77

59. Johnson CD, Esquela-Kerscher A, Stefani G, Byrom M, Kelnar K, et al. 2007. The let-7 microRNA represses cell proliferation pathways in human cells. *Cancer Res.* 67:7713–22

60. Bommer GT, Gerin I, Feng Y, Kaczorowski AJ, Kuick R, et al. 2007. p53-mediated activation of miRNA34 candidate tumor-suppressor genes. *Curr. Biol.* 17:1298–307

61. Chang TC, Wentzel EA, Kent OA, Ramachandran K, Mullendore M, et al. 2007. Transactivation of miR-34a by p53 broadly influences gene expression and promotes apoptosis. *Mol. Cell* 26:745–52

62. He L, He X, Lim LP, de Stanchina E, Xuan Z, et al. 2007. A microRNA component of the p53 tumour suppressor network. *Nature* 447:1130–34

63. Calin GA, Dumitru CD, Shimizu M, Bichi R, Zupo S, et al. 2002. Frequent deletions and downregulation of microRNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia. *Proc. Natl. Acad. Sci. USA* 99:15524–29

64. Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, et al. 2005. MicroRNA expression profiles classify human cancers. *Nature* 435:834–38

65. Calin GA, Liu CG, Sevignani C, Ferracin M, Felli N, et al. 2004. MicroRNA profiling reveals distinct signatures in B cell chronic lymphocytic leukemias. *Proc. Natl. Acad. Sci. USA* 101:11755–60

66. Porkka KP, Pfeiffer MJ, Waltering KK, Vessella RL, Tammela TL, Visakorpi T. 2007. MicroRNA expression profiling in prostate cancer. *Cancer Res.* 67:6130–35

67. Visone R, Pallante P, Vecchione A, Ciombella R, Ferracin M, et al. 2007. Specific microRNAs are downregulated in human thyroid anaplastic carcinomas. *Oncogene* 26:7590–95

68. Gaur A, Jewell DA, Liang Y, Ridzon D, Moore JH, et al. 2007. Characterization of microRNA expression levels and their biological correlates in human cancer cell lines. *Cancer Res.* 67:2456–68

69. Chen Y, Stallings RL. 2007. Differential patterns of microRNA expression in neuroblastoma are correlated with prognosis, differentiation, and apoptosis. *Cancer Res.* 67:976–83

70. Kumar MS, Lu J, Mercer KL, Golub TR, Jacks T. 2007. Impaired microRNA processing enhances cellular transformation and tumorigenesis. *Nat. Genet.* 39:673–77

71. Calin GA, Sevignani C, Dumitru CD, Hyslop T, Noch E, et al. 2004. Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers. *Proc. Natl. Acad. Sci. USA* 101:2999–3004

72. Sevignani C, Calin GA, Nnadi SC, Shimizu M, Davuluri RV, et al. 2007. MicroRNA genes are frequently located near mouse cancer susceptibility loci. *Proc. Natl. Acad. Sci. USA* 104:8017–22

73. Zhang L, Huang J, Yang N, Greshock J, Megraw MS, et al. 2006. MicroRNAs exhibit high-frequency genomic alterations in human cancer. *Proc. Natl. Acad. Sci. USA* 103:9136–41

74. Tagawa H, Karube K, Tsuzuki S, Ohshima K, Seto M. 2007. Synergistic action of the microRNA-17 polycistron and Myc in aggressive cancer development. *Cancer Sci.* 98:1482–90

75. Rinaldi A, Poretti G, Kwee I, Zucca E, Catapano CV, et al. 2007. Concomitant MYC and microRNA cluster miR-17-92 (C13orf25) amplification in human mantle cell lymphoma. *Leuk. Lymphoma* 48:410–12

76. Welch C, Chen Y, Stallings RL. 2007. MicroRNA-34a functions as a potential tumor suppressor by inducing apoptosis in neuroblastoma cells. *Oncogene* 26:5017–22

77. Hayashita Y, Osada H, Tatematsu Y, Yamada H, Yanagisawa K, et al. 2005. A polycistronic microRNA cluster, miR-17-92, is overexpressed in human lung cancers and enhances cell proliferation. *Cancer Res.* 65:9628–32

78. Tagawa H, Seto M. 2005. A microRNA cluster as a target of genomic amplification in malignant lymphoma. *Leukemia* 19:2013–16

79. Lehmann U, Hasemeier B, Christgen M, Muller M, Romermann D, et al. 2007. Epigenetic inactivation of microRNA gene hsa-miR-9-1 in human breast cancer. *J. Pathol.* 214:17–24

80. Lujambio A, Ropero S, Ballestar E, Fraga MF, Cerrato C, et al. 2007. Genetic unmasking of an epigenetically silenced microRNA in human cancer cells. *Cancer Res.* 67:1424–29

81. Scott GK, Mattie MD, Berger CE, Benz SC, Benz CC. 2006. Rapid alteration of microRNA levels by histone deacetylase inhibition. *Cancer Res.* 66:1277–81

82. Fabbri M, Garzon R, Cimmino A, Liu Z, Zanesi N, et al. 2007. MicroRNA-29 family reverts aberrant methylation in lung cancer by targeting DNA methyltransferases 3A and 3B. *Proc. Natl. Acad. Sci. USA* 104:15805–10

83. Shah YM, Morimura K, Yang Q, Tanabe T, Takagi M, Gonzalez FJ. 2007. Peroxisome proliferator-activated receptor α regulates a microRNA-mediated signaling cascade responsible for hepatocellular proliferation. *Mol. Cell Biol.* 27:4238–47

84. Mott JL, Kobayashi S, Bronk SF, Gores GJ. 2007. MiR-29 regulates Mcl-1 protein expression and apoptosis. *Oncogene* 26:6133–40

85. Kluiver J, Van Den Berg A, de Jong D, Blokzijl T, Harms G, et al. 2007. Regulation of premicroRNA BIC transcription and processing in Burkitt lymphoma. *Oncogene* 26:3769–76

86. Lee EJ, Baek M, Gusev Y, Brackett DJ, Nuovo GJ, Schmittgen TD. 2007. Systematic evaluation of microRNA processing patterns in tissues, cell lines, and tumors. *RNA* 14:35–42

87. Obernosterer G, Leuschner PJ, Alenius M, Martinez J. 2006. Post-transcriptional regulation of microRNA expression. *RNA* 12:1161–67

88. Michael MZ, O'Connor SM, van Holst Pellekaan NG, Young GP, James RJ. 2003. Reduced accumulation of specific microRNAs in colorectal neoplasia. *Mol. Cancer Res.* 1:882–91

89. Lu Y, Thomson JM, Wong HY, Hammond SM, Hogan BL. 2007. Transgenic overexpression of the microRNA miR-17-92 cluster promotes proliferation and inhibits differentiation of lung epithelial progenitor cells. *Dev. Biol.* 310:442–53

90. Mineno J, Okamoto S, Ando T, Sato M, Chono H, et al. 2006. The expression profile of microRNAs in mouse embryos. *Nucleic Acids Res.* 34:1765–71

91. Loffler D, Brocke-Heidrich K, Pfeifer G, Stocsits C, Hackermuller J, et al. 2007. Interleukin-6-dependent survival of multiple myeloma cells involves the Stat3-mediated induction of microRNA-21 through a highly conserved enhancer. *Blood* 110:1330–33

92. Thomson JM, Newman M, Parker JS, Morin-Kensicki EM, Wright T, Hammond SM. 2006. Extensive post-transcriptional regulation of microRNAs and its implications for cancer. *Genes Dev.* 20:2202–7

93. Blenkiron C, Goldstein LD, Thorne NP, Spiteri I, Chin SF, et al. 2007. MicroRNA expression profiling of human breast cancer identifies new markers of tumour subtype. *Genome Biol.* 8:R214

94. Chiosea S, Jelezcova E, Chandran U, Luo J, Mantha G, et al. 2007. Overexpression of Dicer in precursor lesions of lung adenocarcinoma. *Cancer Res.* 67:2345–50

95. Muralidhar B, Goldstein LD, Ng G, Winder DM, Palmer RD, et al. 2007. Global microRNA profiles in cervical squamous cell carcinoma depend on Drosha expression levels. *J. Pathol.* 212:368–77

96. Karube Y, Tanaka H, Osada H, Tomida S, Tatematsu Y, et al. 2005. Reduced expression of Dicer associated with poor prognosis in lung cancer patients. *Cancer Sci.* 96:111–15

97. Nakamura T, Canaani E, Croce CM. 2007. Oncogenic All1 fusion proteins target Drosha-mediated microRNA processing. *Proc. Natl. Acad. Sci. USA* 104:10980–85

98. Johnson SM, Grosshans H, Shingara J, Byrom M, Jarvis R, et al. 2005. RAS is regulated by the let-7 microRNA family. *Cell* 120:635–47

99. Yang J, Zhou F, Xu T, Deng H, Ge YY, et al. 2007. Analysis of sequence variations in 59 microRNAs in hepatocellular carcinomas. *Mutat. Res.* 638:205–9

100. He H, Jazdzewski K, Li W, Liyanarachchi S, Nagy R, et al. 2005. The role of microRNA genes in papillary thyroid carcinoma. *Proc. Natl. Acad. Sci. USA* 102:19075–80

101. Diederichs S, Haber DA. 2006. Sequence variations of microRNAs in human cancer: Alterations in predicted secondary structure do not affect processing. *Cancer Res.* 66:6097–104

102. Yu Z, Li Z, Jolicoeur N, Zhang L, Fortin Y, et al. 2007. Aberrant allele frequencies of the SNPs located in microRNA target sites are potentially associated with human cancers. *Nucleic Acids Res.* 35:4535–41

103. Abelson JF, Kwan KY, O’Roak BJ, Baek DY, Stillman AA, et al. 2005. Sequence variants in SLITRK1 are associated with Tourette’s syndrome. *Science* 310:317–20

104. Clop A, Marcq F, Takeda H, Pirottin D, Tordoir X, et al. 2006. A mutation creating a potential illegitimate microRNA target site in the myostatin gene affects muscularity in sheep. *Nat. Genet.* 38:813–18

105. Adams BD, Furneaux H, White BA. 2007. The microribonucleic acid (miRNA) miR-206 targets the human estrogen receptor-α (ER α) and represses ER α messenger RNA and protein expression in breast cancer cell lines. *Mol. Endocrinol.* 21:1132–47

106. Mayr C, Hemann MT, Bartel DP. 2007. Disrupting the pairing between let-7 and Hmga2 enhances oncogenic transformation. *Science* 315:1576–79

107. Laneve P, Di Marcotullio L, Gioia U, Fiori ME, Ferretti E, et al. 2007. The interplay between microRNAs and the neurotrophin receptor tropomyosin-related kinase C controls proliferation of human neuroblastoma cells. *Proc. Natl. Acad. Sci. USA* 104:7957–62

108. Nagy E, Maquat LE. 1998. A rule for termination-codon position within intron-containing genes: when nonsense affects RNA abundance. *Trends Biochem. Sci.* 23:198–99

109. Vasudevan S, Tong Y, Steitz JA. 2007. Switching from repression to activation: MicroRNAs can up-regulate translation. *Science* 318:1931–34

110. Pfeffer S, Zavolan M, Grasser FA, Chien M, Russo JJ, et al. 2004. Identification of virus-encoded microRNAs. *Science* 304:734–36

111. Lo AK, To KF, Lo KW, Lung RW, Hui JW, et al. 2007. Modulation of LMP1 protein expression by EBV-encoded microRNAs. *Proc. Natl. Acad. Sci. USA* 104:16164–69

112. Samols MA, Skalsky RL, Maldonado AM, Riva A, Lopez MC, et al. 2007. Identification of cellular genes targeted by KSHV-encoded microRNAs. *PLoS Pathog.* 3:e65

113. Clurman BE, Hayward WS. 1989. Multiple proto-oncogene activations in avian leukosis virus-induced lymphomas: evidence for stage-specific events. *Mol. Cell Biol.* 9:2657–64

114. Makunin IV, Pheasant M, Simons C, Mattick JS. 2007. Orthologous microRNA genes are located in cancer-associated genomic regions in human and mouse. *PLoS ONE* 2:e1133

115. Cui JW, Li YJ, Sarkar A, Brown J, Tan YH, et al. 2007. Retroviral insertional activation of the Fli-3 locus in erythroleukemias encoding a cluster of microRNAs that convert Epo-induced differentiation to proliferation. *Blood* 110:2631–40

116. Landais S, Landry S, Legault P, Rassart E. 2007. Oncogenic potential of the miR-106–363 cluster and its implication in human T cell leukemia. *Cancer Res.* 67:5699–707

117. Slape C, Hartung H, Lin YW, Bies J, Wolff L, Aplan PD. 2007. Retroviral insertional mutagenesis identifies genes that collaborate with NUP98–HOXD13 during leukemic transformation. *Cancer Res.* 67:5148–55

118. Lum AM, Wang BB, Li L, Channa N, Bartha G, Wabl M. 2007. Retroviral activation of the miR-106a microRNA cistron in T lymphoma. *Retrovirology* 4:5

119. He L, Thomson JM, Hemann MT, Hernando-Monge E, Mu D, et al. 2005. A microRNA polycistron as a potential human oncogene. *Nature* 435:828–33

120. Ota A, Tagawa H, Karnan S, Tsuzuki S, Karpas A, et al. 2004. Identification and characterization of a novel gene, C13orf25, as a target for 13q31-q32 amplification in malignant lymphoma. *Cancer Res.* 64:3087–95

121. Matsubara H, Takeuchi T, Nishikawa E, Yanagisawa K, Hayashita Y, et al. 2007. Apoptosis induction by antisense oligonucleotides against miR-17-5p and miR-20a in lung cancers overexpressing miR-17-92. *Oncogene* 26:6099–105

122. Volinia S, Calin GA, Liu CG, Ambs S, Croce CM, et al. 2006. A microRNA expression signature of human solid tumors defines cancer gene targets. *Proc. Natl. Acad. Sci. USA* 103:2257–61

123. O’Donnell KA, Wentzel EA, Zeller KI, Dang CV, Mendell JT. 2005. c-Myc-regulated microRNAs modulate E2F1 expression. *Nature* 435:839–43

124. Schulte JH, Horn S, Otto T, Samans B, Heukamp LC, et al. 2007. MYCN regulates oncogenic microRNAs in neuroblastoma. *Int. J. Cancer* 122:699–704

125. Woods K, Thomson JM, Hammond SM. 2007. Direct regulation of an oncogenic micro-RNA cluster by E2F transcription factors. *J. Biol. Chem.* 282:2130–34

126. Sylvestre Y, De Guire V, Querido E, Mukhopadhyay UK, Bourdeau V, et al. 2007. An E2F/miR-20a autoregulatory feedback loop. *J. Biol. Chem.* 282:2135–43

127. Costinean S, Zanesi N, Pekarsky Y, Tili E, Volinia S, et al. 2006. Pre-B cell proliferation and lymphoblastic leukemia/high-grade lymphoma in E(mu)-miR155 transgenic mice. *Proc. Natl. Acad. Sci. USA* 103:7024–29

128. Kluiver J, Poppema S, de Jong D, Blokzijl T, Harms G, et al. 2005. BIC and miR-155 are highly expressed in Hodgkin, primary mediastinal and diffuse large B cell lymphomas. *J. Pathol.* 207:243–49

129. Eis PS, Tam W, Sun L, Chadburn A, Li Z, et al. 2005. Accumulation of miR-155 and BIC RNA in human B cell lymphomas. *Proc. Natl. Acad. Sci. USA* 102:3627–32

130. Metzler M, Wilda M, Busch K, Viehmann S, Borkhardt A. 2004. High expression of precursor microRNA-55/BIC RNA in children with Burkitt lymphoma. *Genes Chromosom. Cancer* 39:167–69

131. O'Connell RM, Taganov KD, Boldin MP, Cheng G, Baltimore D. 2007. MicroRNA-155 is induced during the macrophage inflammatory response. *Proc. Natl. Acad. Sci. USA* 104:1604–9

132. Shell S, Park SM, Radjabi AR, Schickel R, Kistner EO, et al. 2007. Let-7 expression defines two differentiation stages of cancer. *Proc. Natl. Acad. Sci. USA* 104:11400–5

133. Dutta KK, Zhong Y, Liu YT, Yamada T, Akatsuka S, et al. 2007. Association of microRNA-34a overexpression with proliferation is cell type-dependent. *Cancer Sci.* 98:1845–52

134. Meng F, Henson R, Wehbe-Janek H, Ghoshal K, Jacob ST, Patel T. 2007. MicroRNA-21 regulates expression of the PTEN tumor suppressor gene in human hepatocellular cancer. *Gastroenterology* 133:647–58

135. Gillies JK, Lorimer IA. 2007. Regulation of p27Kip1 by miRNA 221/222 in glioblastoma. *Cell Cycle* 6:2005–9

136. Galardi S, Mercatelli N, Giorda E, Massalini S, Frasson GV, et al. 2007. miR-221 and miR-222 expression affects the proliferation potential of human prostate carcinoma cell lines by targeting p27Kip1. *J. Biol. Chem.* 282:23716–24

137. Tarasov V, Jung P, Verdoodt B, Lodygin D, Epanchintsev A, et al. 2007. Differential regulation of microRNAs by p53 revealed by massively parallel sequencing: MiR-34a is a p53 target that induces apoptosis and G1-arrest. *Cell Cycle* 6:1586–93

138. Linsley PS, Schelter J, Burchard J, Kibukawa M, Martin MM, et al. 2007. Transcripts targeted by the microRNA-16 family cooperatively regulate cell cycle progression. *Mol. Cell Biol.* 27:2240–52

139. Mertens-Talcott SU, Chinthalapalli S, Li X, Safe S. 2007. The oncogenic microRNA-27a targets genes that regulate specificity protein transcription factors and the G2-M checkpoint in MDA-MB-231 breast cancer cells. *Cancer Res.* 67:11001–11

140. Cimmino A, Calin GA, Fabbri M, Iorio MV, Ferracin M, et al. 2005. MiR-15 and miR-16 induce apoptosis by targeting BCL2. *Proc. Natl. Acad. Sci. USA* 102:13944–49

141. Raver-Shapira N, Marciano E, Meiri E, Spector Y, Rosenfeld N, et al. 2007. Transcriptional activation of miR-34a contributes to p53-mediated apoptosis. *Mol. Cell* 26:731–43

142. Xi Y, Shalgi R, Fodstad O, Pilpel Y, Ju J. 2006. Differentially regulated microRNAs and actively translated messenger RNA transcripts by tumor suppressor p53 in colon cancer. *Clin. Cancer Res.* 12:2014–24

143. Corney DC, Flecken-Nikitin A, Godwin AK, Wang W, Nikitin AY. 2007. MicroRNA-34b and microRNA-34c are targets of p53 and cooperate in control of cell proliferation and adhesion-independent growth. *Cancer Res.* 67:8433–38

144. Scott GK, Goga A, Bhaumik D, Berger CE, Sullivan CS, Benz CC. 2007. Coordinate suppression of ERBB2 and ERBB3 by enforced expression of microRNA miR-125a or miR-125b. *J. Biol. Chem.* 282:1479–86

145. Asangani IA, Rasheed SA, Nikolova DA, Leupold JH, Colburn NH, et al. 2007. MicroRNA-21 (miR-21) post-transcriptionally downregulates tumor suppressor Pdcd4 and stimulates invasion, intravasation and metastasis in colorectal cancer. *Oncogene* 27:2128–36

146. Ma L, Teruya-Feldstein J, Weinberg RA. 2007. Tumour invasion and metastasis initiated by microRNA-10b in breast cancer. *Nature* 449:682–88

146. First paper showing that a miRNA regulates tumor invasion and metastasis, concurrently identifying both the upstream regulator and the downstream target gene of the miRNA.

147. Dewe M, Homayouni A, Yu D, Murphy D, Sevignani C, et al. 2006. Augmentation of tumor angiogenesis by a Myc-activated microRNA cluster. *Nat. Genet.* 38:1060–65

148. Lee DY, Deng Z, Wang CH, Yang BB. 2007. MicroRNA-378 promotes cell survival, tumor growth, and angiogenesis by targeting SuFu and Fus-1 expression. *Proc. Natl. Acad. Sci. USA* 104:20350–55

149. Hebert C, Norris K, Scheper MA, Nikitakis N, Sauk JJ. 2007. High mobility group A2 is a target for miRNA-98 in head and neck squamous cell carcinoma. *Mol. Cancer* 6:5

150. Kulshreshtha R, Ferracin M, Wojcik SE, Garzon R, Alder H, et al. 2007. A microRNA signature of hypoxia. *Mol. Cell Biol.* 27:1859–67

151. Marsit CJ, Eddy K, Kelsey KT. 2006. MicroRNA responses to cellular stress. *Cancer Res.* 66:10843–48

152. Kutay H, Bai S, Datta J, Motiwala T, Pogribny I, et al. 2006. Downregulation of miR-122 in the rodent and human hepatocellular carcinomas. *J. Cell Biochem.* 99:671–78

153. Tang F, Hajkova P, Barton SC, Lao K, Surani MA. 2006. MicroRNA expression profiling of single whole embryonic stem cells. *Nucleic Acids Res.* 34:e9

154. Kim do N, Chae HS, Oh ST, Kang JH, Park CH, et al. 2007. Expression of viral microRNAs in Epstein-Barr virus-associated gastric carcinoma. *J. Virol.* 81:1033–36

155. Xi Y, Nakajima G, Gavin E, Morris CG, Kudo K, et al. 2007. Systematic analysis of microRNA expression of RNA extracted from fresh frozen and formalin-fixed paraffin-embedded samples. *RNA* 13:1668–74

156. Lawrie CH, Soneji S, Marafioti T, Cooper CD, Palazzo S, et al. 2007. MicroRNA expression distinguishes between germinal center B cell-like and activated B cell-like subtypes of diffuse large B cell lymphoma. *Int. J. Cancer* 121:1156–61

157. Zhang HH, Wang XJ, Li GX, Yang E, Yang NM. 2007. Detection of let-7a microRNA by real-time PCR in gastric carcinoma. *World J. Gastroenterol.* 13:2883–88

158. Mattie MD, Benz CC, Bowers J, Sensinger K, Wong L, et al. 2006. Optimized high-throughput microRNA expression profiling provides novel biomarker assessment of clinical prostate and breast cancer biopsies. *Mol. Cancer* 5:24

159. Pallante P, Visone R, Ferracin M, Ferraro A, Berlingieri MT, et al. 2006. MicroRNA deregulation in human thyroid papillary carcinomas. *Endocr. Relat. Cancer* 13:497–508

160. Szafranska AE, Davison TS, John J, Cannon T, Sipos B, et al. 2007. MicroRNA expression alterations are linked to tumorigenesis and non-neoplastic processes in pancreatic ductal adenocarcinoma. *Oncogene* 26:4442–52

161. Hutvagner G, Zamore PD. 2002. A microRNA in a multiple-turnover RNAi enzyme complex. *Science* 297:2056–60

162. Hossain A, Kuo MT, Saunders GF. 2006. Mir-17-5p regulates breast cancer cell proliferation by inhibiting translation of AIB1 mRNA. *Mol. Cell Biol.* 26:8191–201

163. Weiler J, Hunziker J, Hall J. 2006. Anti-miRNA oligonucleotides (AMOs): ammunition to target miRNAs implicated in human disease? *Gene. Ther.* 13:496–502

164. Krutzfeldt J, Rajewsky N, Braich R, Rajeev KG, Tuschl T, et al. 2005. Silencing of microRNAs in vivo with “antagomirs.” *Nature* 438:685–89

165. Si ML, Zhu S, Wu H, Lu Z, Wu F, Mo YY. 2007. MiR-21-mediated tumor growth. *Oncogene* 26:2799–803

166. Elmen J, Lindow M, Silahtaroglu A, Bak M, Christensen M, et al. 2007. Antagonism of microRNA-122 in mice by systemically administered LNA-antimiR leads to up-regulation of a large set of predicted target mRNAs in the liver. *Nucleic Acids Res.* 36:1153–62

167. Cullen BR. 2005. RNAi the natural way. *Nat. Genet.* 37:1163–65

168. Ebert MS, Neilson JR, Sharp PA. 2007. MicroRNA sponges: competitive inhibitors of small RNAs in mammalian cells. *Nat. Methods* 4:721–26

169. Franco-Zorrilla JM, Valli A, Todesco M, Mateos I, Puga MI, et al. 2007. Target mimicry provides a new mechanism for regulation of microRNA activity. *Nat. Genet.* 39:1033–37

170. Orom UA, Kauppinen S, Lund AH. 2006. LNA-modified oligonucleotides mediate specific inhibition of microRNA function. *Gene* 372:137–41

171. Krutzfeldt J, Kuwajima S, Braich R, Rajeev KG, Pena J, et al. 2007. Specificity, duplex degradation and subcellular localization of antagomirs. *Nucleic Acids Res.* 35:2885–92

172. Grimm D, Streetz KL, Jopling CL, Storm TA, Pandey K, et al. 2006. Fatality in mice due to oversaturation of cellular microRNA/short hairpin RNA pathways. *Nature* 441:537–41

173. Lee YS, Kim HK, Chung S, Kim KS, Dutta A. 2005. Depletion of human microRNA miR-125b reveals that it is critical for the proliferation of differentiated cells but not for the down-regulation of putative targets during differentiation. *J. Biol. Chem.* 280:16635–41

174. Jing Q, Huang S, Guth S, Zarubin T, Motoyama A, et al. 2005. Involvement of microRNA in AU-rich element-mediated mRNA instability. *Cell* 120:623–34

175. Blower PE, Verducci JS, Lin S, Zhou J, Chung JH, et al. 2007. MicroRNA expression profiles for the NCI-60 cancer cell panel. *Mol. Cancer Ther.* 6:1483–91

176. Meng F, Henson R, Lang M, Wehbe H, Maheshwari S, et al. 2006. Involvement of human microRNA in growth and response to chemotherapy in human cholangiocarcinoma cell lines. *Gastroenterology* 130:2113–29

177. Corsten MF, Miranda R, Kasmiieh R, Krichevsky AM, Weissleder R, Shah K. 2007. MicroRNA-21 knockdown disrupts glioma growth in vivo and displays synergistic cytotoxicity with neural precursor cell delivered S-TRAIL in human gliomas. *Cancer Res.* 67:8994–9000

178. Lau NC. 2008. Analysis of small endogenous RNAs. In *Current Protocols in Molecular Biology*, chapter 26.7. New York: Wiley

179. Kapranov P, Willingham AT, Gingeras TR. 2007. Genome-wide transcription and the implications for genomic organization. *Nat. Rev. Genet.* 8:413–23

180. Kapranov P, Cheng J, Dike S, Nix DA, Duttagupta R, et al. 2007. RNA maps reveal new RNA classes and a possible function for pervasive transcription. *Science* 316:1484–88

181. Caudy AA, Myers M, Hannon GJ, Hammond SM. 2002. Fragile X-related protein and VIG associate with the RNA interference machinery. *Genes Dev.* 16:2491–96

182. Bilen J, Liu N, Bonini NM. 2006. A new role for microRNA pathways: modulation of degeneration induced by pathogenic human disease proteins. *Cell Cycle* 5:2835–38

183. Perkins DO, Jeffries CD, Jarskog LF, Thomson JM, Woods K, et al. 2007. microRNA expression in the prefrontal cortex of individuals with schizophrenia and schizoaffective disorder. *Genome Biol.* 8:R27

184. Hon LS, Zhang Z. 2007. The roles of binding site arrangement and combinatorial targeting in microRNA repression of gene expression. *Genome Biol.* 8:R166

185. Iorio MV, Ferracin M, Liu CG, Veronese A, Spizzo R, et al. 2005. MicroRNA gene expression deregulation in human breast cancer. *Cancer Res.* 65:7065–70

186. Ozen M, Creighton CJ, Ozdemir M, Ittmann M. 2007. Widespread deregulation of microRNA expression in human prostate cancer. *Oncogene* 27:1788–93

187. Bloomston M, Frankel WL, Petrocca F, Volinia S, Alder H, et al. 2007. MicroRNA expression patterns to differentiate pancreatic adenocarcinoma from normal pancreas and chronic pancreatitis. *JAMA* 297:1901–8

188. Lee EJ, Gusev Y, Jiang J, Nuovo GJ, Lerner MR, et al. 2007. Expression profiling identifies microRNA signature in pancreatic cancer. *Int. J. Cancer* 120:1046–54

189. Roldo C, Missiaglia E, Hagan JP, Falconi M, Capelli P, et al. 2006. MicroRNA expression abnormalities in pancreatic endocrine and acinar tumors are associated with distinctive pathologic features and clinical behavior. *J. Clin. Oncol.* 24:4677–84

190. Yanaihara N, Caplen N, Bowman E, Seike M, Kumamoto K, et al. 2006. Unique microRNA molecular profiles in lung cancer diagnosis and prognosis. *Cancer Cell* 9:189–98

191. Iorio MV, Visone R, Di Leva G, Donati V, Petrocca F, et al. 2007. MicroRNA signatures in human ovarian cancer. *Cancer Res.* 67:8699–707

192. Wang T, Zhang X, Obijuru L, Laser J, Aris V, et al. 2007. A micro-RNA signature associated with race, tumor size, and target gene activity in human uterine leiomyomas. *Genes Chromosom. Cancer* 46:336–47

193. Gramantieri L, Ferracin M, Fornari F, Veronese A, Sabbioni S, et al. 2007. Cyclin G1 is a target of miR-122a, a microRNA frequently down-regulated in human hepatocellular carcinoma. *Cancer Res.* 67:6092–99

194. Murakami Y, Yasuda T, Saigo K, Urashima T, Toyoda H, et al. 2006. Comprehensive analysis of microRNA expression patterns in hepatocellular carcinoma and nontumorous tissues. *Oncogene* 25:2537–45

195. Weber F, Teresi RE, Broelsch CE, Frilling A, Eng C. 2006. A limited set of human microRNA is deregulated in follicular thyroid carcinoma. *J. Clin. Endocrinol. Metab.* 91:3584–91

196. Lanza G, Ferracin M, Gafa R, Veronese A, Spizzo R, et al. 2007. mRNA/microRNA gene expression profile in microsatellite unstable colorectal cancer. *Mol. Cancer* 6:54

197. Bandres E, Cubedo E, Agirre X, Malumbres R, Zarate R, et al. 2006. Identification by real-time PCR of 13 mature microRNAs differentially expressed in colorectal cancer and nontumoral tissues. *Mol. Cancer* 5:29

198. Bottoni A, Zatelli MC, Ferracin M, Tagliati F, Piccin D, et al. 2007. Identification of differentially expressed microRNAs by microarray: a possible role for microRNA genes in pituitary adenomas. *J. Cell Physiol.* 210:370–77

199. Ciafre SA, Galardi S, Mangiola A, Ferracin M, Liu CG, et al. 2005. Extensive modulation of a set of microRNAs in primary glioblastoma. *Biochem. Biophys. Res. Commun.* 334:1351–58

200. Mi S, Lu J, Sun M, Li Z, Zhang H, et al. 2007. MicroRNA expression signatures accurately discriminate acute lymphoblastic leukemia from acute myeloid leukemia. *Proc. Natl. Acad. Sci. USA* 104:19971–76

201. Fulci V, Chiaretti S, Goldoni M, Azzalin G, Carucci N, et al. 2007. Quantitative technologies establish a novel microRNA profile of chronic lymphocytic leukemia. *Blood* 109:4944–51

202. Pekarsky Y, Santanam U, Cimmino A, Palamarchuk A, Efanov A, et al. 2006. Tcl1 expression in chronic lymphocytic leukemia is regulated by miR-29 and miR-181. *Cancer Res.* 66:11590–93

203. Calin GA, Ferracin M, Cimmino A, Di Leva G, Shimizu M, et al. 2005. A microRNA signature associated with prognosis and progression in chronic lymphocytic leukemia. *N. Engl. J. Med.* 353:1793–801

204. Subramanian S, Lui WO, Lee CH, Espinosa I, Nielsen TO, et al. 2007. MicroRNA expression signature of human sarcomas. *Oncogene* 27:2015–26

205. Shi XB, Xue L, Yang J, Ma AH, Zhao J, et al. 2007. An androgen-regulated miRNA suppresses Bak1 expression and induces androgen-independent growth of prostate cancer cells. *Proc. Natl. Acad. Sci. USA* 104:19983–88

206. Meng F, Henson R, Wehbe-Janek H, Smith H, Ueno Y, Patel T. 2007. The microRNA let-7a modulates interleukin-6-dependent STAT-3 survival signaling in malignant human cholangiocytes. *J. Biol. Chem.* 282:8256–64

207. Tsuchiya Y, Nakajima M, Takagi S, Taniya T, Yokoi T. 2006. MicroRNA regulates the expression of human cytochrome P450 1B1. *Cancer Res.* 66:9090–98

208. Saito Y, Liang G, Egger G, Friedman JM, Chuang JC, et al. 2006. Specific activation of microRNA-127 with downregulation of the proto-oncogene BCL6 by chromatin-modifying drugs in human cancer cells. *Cancer Cell* 9:435–43

209. Voorhoeve PM, le Sage C, Schrier M, Gillis AJ, Stoop H, et al. 2006. A genetic screen implicates miRNA-372 and miRNA-373 as oncogenes in testicular germ cell tumors. *Cell* 124:1169–81

Contents

The First Fifty Years in Research  
**Peter A. Ward** . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Drosophila Models of Neurodegenerative Diseases  
Bingwei Lu and Hannes Vogel  . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .