
Molecular Cell
Short Article

# RNAIII-Independent Target Gene Control by the agr Quorum-Sensing System: Insight into the Evolution of Virulence Regulation in Staphylococcus aureus

Shu Y. Queck,$^{1,3}$ Max Jameson-Lee,$^{1}$ Amer E. Villaruz,$^{1}$ Thanh-Huy L. Bach,$^{1}$ Burhan A. Khan,$^{1}$ Daniel E. Sturdevant,$^{2}$ Stacey M. Ricklefs,$^{2}$ Min Li,$^{1,4}$ and Michael Otto$^{1,*}$

$^{1}$Laboratory of Human Bacterial Pathogenesis  
$^{2}$Research Technologies Branch, Genomics Unit  
Rocky Mountain Laboratories, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Hamilton, MT 59840, USA  
$^{3}$Present address: Silliker Pte Ltd., Block Q Unit #05-08, 180 Ang Mo Kio Avenue 8, Singapore 569830, Singapore  
$^{4}$Present address: Center of Laboratory Medicine, Huashan Hospital, Shanghai Medical College, Fudan University, Shanghai 200040, China  

*Correspondence: motto@niaid.nih.gov  
DOI 10.1016/j.molcel.2008.08.005

## SUMMARY

S. aureus is a major human pathogen and among the most common causes of bacterial infections in the community and the hospital. Many strains are resistant to a wide spectrum of antibiotics including methicillin (methicillin-resistant S. aureus, MRSA), making therapeutic intervention often extremely difficult. Moreover, recent outbreaks of MRSA in the community form a major challenge for the public health system ([Diep and Otto, 2008](https://doi.org/10.1016/j.jclinepi.2007.08.005)). This situation has sparked considerable interest in the analysis of staphylococcal molecular biology with the goal to find drug targets. Particularly, global regulators of virulence such as quorum-sensing regulators have been studied intensively, based on the notions that interfering with such systems would eliminate production of many virulence factors at a time ([Alksne and Projan, 2000](https://doi.org/10.1016/S0952-7915(00)00096-8)) and drugs interfering with virulence would not lead to resistance at the same speed as drugs that kill bacteria ([Clatworthy et al., 2007](https://doi.org/10.1038/nrmicro1738)).

The agr (accessory gene regulator) system represents a prototype of quorum-sensing regulators in Gram-positive bacteria ([Kleerebezem et al., 1997](https://doi.org/10.1073/pnas.94.18.9677)). It has been recognized early as a pivotal regulator of virulence factor expression ([Recsei et al., 1986](https://doi.org/10.1128/jb.166.3.1097-1102.1986)) and a potential target for therapeutic intervention ([Otto, 2004](https://doi.org/10.1016/j.tim.2004.04.002)). The agr system consists of two adjacent transcriptional units, RNAII and RNAIII, which are transcribed in opposite directions (Figure 1). RNAII comprises the messages for the AgrD precursor of the extracellular peptide-based quorum-sensing signal and its maturation and export protein AgrB. Furthermore, it encodes a classical bacterial two-component signal transduction system, composed of the sensor histidine kinase AgrC and its cognate response regulator AgrA. Upon binding of the quorum-sensing signal to AgrC, AgrA is activated and binds to the promoter regions in the agr system (P2 for RNAII and P3 for RNAIII). This positive feedback loop, typical of quorum-sensing systems, ascertains that target gene expression is altered swiftly at a certain threshold level of bacterial cell density ([Novick, 2003](https://doi.org/10.1016/S0968-0004(03)00100-5)).

According to current knowledge, modulation of target gene expression by agr is achieved via RNAIII ([Novick et al., 1993](https://doi.org/10.1073/pnas.90.18.8357)), which represents a paradigmatic example of virulence factor control via a small regulatory RNA (sRNA) ([Romby et al., 2006](https://doi.org/10.1016/j.mib.2006.04.002)) and the central role of sRNAs in quorum sensing ([Toledo-Araña et al., 2007](https://doi.org/10.1016/j.mib.2007.03.001)). Recent findings indicate that RNAIII generally acts

## INTRODUCTION

Many bacteria undergo substantial physiological rearrangements with increasing population density ([Camilli and Bassler, 2006](https://doi.org/10.1038/nature04843)). The corresponding gene-regulatory changes are controlled by quorum-sensing systems. In pathogenic bacteria, these systems have first been described as virulence regulators ([Passador et al., 1993](https://doi.org/10.1128/jb.175.15.4697-4708.1993); [Recsei et al., 1986](https://doi.org/10.1128/jb.166.3.1097-1102.1986)), whereas it has been shown more recently that quorum-sensing control is also aimed to respond to changing environmental conditions via metabolic adaptations ([Cassat et al., 2006](https://doi.org/10.1128/JB.01746-05); [Dunman et al., 2001](https://doi.org/10.1128/JB.183.17.5119-5128.2001); [Schuster et al., 2003](https://doi.org/10.1016/S0968-0004(03)00130-0); [Yao et al., 2006](https://doi.org/10.1128/JB.188.18.6518-6527.2006)).

Molecular Cell

Quorum Sensing in Staphylococcus aureus

# Quorum-sensing circuit

- **signal transduction**
  - AgrC
    - His-Pi
    - Two component system
  - AgrA
    - Asp-Pi
- **AIP maturation and export**
  - AIP
    - C=O
    - S
  - AgrB
  - AgrD

## Target gene control

### RNAIII-independent control
- **binding of AgrA to promoter**
  - mechanism unknown
  - up
    - PSMs
  - down
    - carbohydrate metabolism (several PTS systems, hexose transport, sialic acid and gluconate utilization)
    - amino acid metabolism (arginine biosynthesis, histidine degradation)
    - staphyloxanthin biosynthesis

### RNAIII-dependent control
- **evolution of RNAIII around hld gene**
  - hld (delta-toxin)
  - P2
  - P3
  - RNAIII
  - agrA
  - agrC
  - agrD
  - agrB
- **direct base pairing with target mRNA, or via base pairing to rot (or other sarA paralog?) mRNA**
  - up
    - exoproteases
    - lipase
    - urease
    - enterotoxin
    - leukocidins
    - alpha-toxin
    - methicillin resistance (mecA)
  - down
    - protein A
    - SarH1
    - oligopeptide permease

---

**Figure 1. Model of Target Gene Control by agr**

The quorum-sensing circuit is shown at the top. The autoinducing peptide (AIP) with its characteristic thiolactone structure is produced from the AgrD precursor, modified, and exported by AgrB. It activates the AgrC/AgrA two-component system. Target gene regulation, as determined by comparative quantitative real-time PCR and genome-wide transcriptional profiling between S. aureus MW2 wild-type, agr deletion, and RNAIII deletion strains, is shown at the bottom. Targets regulated independently of RNAIII are shown at the left, which include several metabolic and psm genes (psmα and psmβ operons). The latter are regulated by AgrA binding to the respective psm promoter regions. RNAIII-dependent gene regulation is shown at the right. AgrA binds to the P2 and P3 promoters, activating the agr quorum-sensing feedback mechanism via RNAII expression, and RNAIII expression together with that of hld. Evolution of RNAIII around the hld mRNA likely has linked the RNAIII-dependent and -independent parts of agr regulation. Many RNAIII-dependent agr targets contain a series of key virulence factors such as proteases and toxins. While regulation of many agr targets in an RNAIII-dependent fashion was shown in laboratory strains to occur via the SarA paralog Rot, the only SarA paralog regulated by agr in our study was SarH1.

by an antisense base pairing mechanism (Boisset et al., 2007) and regulates many target genes via control of a repressor protein gene called rot, a member of the sarA family of transcriptional regulators (Boisset et al., 2007; McNamara et al., 2000; Said-Salim et al., 2003).

The recently discovered phenol-soluble modulins (PSMs) play a key role in S. aureus immune evasion owing to their considerable leukocidal activity, and they significantly impact the outcome of infections by S. aureus, especially community-associated (CA-) MRSA (Wang et al., 2007). There are seven

PSM peptide genes in S. aureus, four forming the psmα operon encoding the PSMα peptides PSMα1–PSMα4; two forming the psmβ operon encoding the PSMβ peptides PSMβ1 and PSMβ2; and the hld gene, which encodes the δ-toxin and is embedded within RNAIII (Wang et al., 2007) (Figure 1). Most likely, δ-toxin is not involved in the RNAIII-dependent regulation of agr target genes, although this has been a matter of debate (Janzon and Arvidson, 1990; Novick et al., 1995). While the psm genes are present in all S. aureus strains whose genome has been sequenced, their expression may differ significantly (Wang
et al., 2007). Among the PSM peptides, the PSM$\alpha$ peptides have the most pronounced chemotactic, proinflammatory, and leukolytic activity (Wang et al., 2007). PSM peptides are secreted by a yet-unknown mechanism as the direct translational product with the N-terminal formylmethionine modification (Wang et al., 2007), which is typical for bacteria. However, due to the activity of bacterial N-formylmethionine deformylase, a certain percentage of PSMs are secreted in a N-deformylated form dependent on media and growth phase (Somerville et al., 2003). It is not known how N-terminal formylation impacts biological activity of the PSM peptides.

Intriguingly, production of PSM peptides is under exceptionally strict control by agr (Wang et al., 2007), but the molecular details of PSM regulation are unknown. Moreover, it is not understood why the hld gene encoding the PSM $\delta$-toxin is embedded within RNAIII. Therefore, here we re-evaluated the role of RNAIII within the agr regulatory system, with a focus on PSM regulation. By demonstrating RNAIII-independent regulation of a considerable portion of the agr regulon, our findings establish that there are two levels of target gene control by quorum sensing in S. aureus and give insight into the evolution of quorum-sensing systems with regard to the connection of metabolism and virulence gene regulation.

## RESULTS

### Agr-Dependent Gene Regulation in the CA-MRSA Strain MW2

Our knowledge of agr target genes is predominantly based on studies with the laboratory strain RN6390, which has a mutation in the rsbU global regulatory gene. The limited predictive value of those studies for clinically important strains has recently become apparent (Blevins et al., 2002). Thus, we chose the S. aureus MW2 strain for the current study, because it represents a community isolate with major clinical significance and has an intact rsbU gene (Baba et al., 2002; CDC, 1999). This strain is the prototype CA-MRSA strain and was reported to cause fatal CA infections in children (CDC, 1999). In accordance with its success as a pathogen in the community, it has revealed hyper-virulence in animal infection models (Voyich et al., 2005). Most importantly for the current study, strain MW2 has an active agr system (Wang et al., 2007). Here, to reevaluate the mechanism of target gene regulation by agr, we first determined the agr regulon of strain MW2. To that end, we performed genome-wide transcriptional profiling analyses with microarrays specific for the MW2 strain (Li et al., 2007), comparing gene expression in the MW2 wild-type and isogenic agr deletion strains during post-exponential growth phase (at maximal expression of agr).

In accordance with the classical notion of agr-dependent gene regulation (Novick, 2003), the microarray results demonstrated that agr in strain MW2 upregulates a series of exotoxin genes, such as those coding for several leukocidins, including the Pantoon-Valentine leukocidin (PVL), enterotoxins, and $\alpha$-toxin. Furthermore, genes encoding degradative exoenzymes (serine and cysteine proteases, lipase), and the methicillin resistance gene mecA, are under positive regulation of agr (Figure 1, see Table S1 available online). The most strongly downregulated transcript was that coding for the immune evasion molecule protein A. Interestingly, other downregulated agr target genes in MW2 mostly code for metabolic enzymes, possibly indicating that the upregulation of toxins occurs at the price of reduced metabolic activity (Figure 1, Table S1).

We were particularly interested in agr regulation of psm genes. Because the newly identified psm$\alpha$ genes are too small to meet the threshold length previously used for gene annotation in S. aureus genomes, including that of MW2 (Baba et al., 2002), the MW2 chip that we used only contains probes for the larger psm$\beta$, but not the smaller psm$\alpha$ genes. Thus, we used quantitative real-time PCR to measure expression of the psm$\alpha$ operon, in addition to confirmatory analysis of the psm$\beta$ operon (Figure 2A). Together, the microarray and quantitative real-time PCR data showed that the psm$\alpha$ and psm$\beta$ operons are under very strong positive regulation by agr.

The agr regulon of the CA-MRSA clinical isolate MW2 revealed several discrepancies to findings achieved using the laboratory strain RN6390. First, the current model of agr regulation comprises downregulation of surface-binding proteins by agr. It is believed that they are specifically needed during the establishment of an infection, while increased agr activity downregulates their expression later during infection (Novick, 2003). In contrast, these proteins were not regulated by agr in strain MW2, except for protein A, which does not have a role in host matrix protein binding but in immune evasion (Forsgren and Nordstrom, 1974; Gomez et al., 2004). Second, the SarA paralog Rot has been attributed a central role in the regulation of agr target genes (Boisset et al., 2007; McNamara et al., 2000; Said-Salim et al., 2003). However, the only sarA paralog to be regulated by agr in strain MW2 was sarH1, also termed sarS gene. It is tempting to speculate that sarH1 might functionally substitute for rot in strain MW2.

### Many Genes Are under RNAIII-Independent Control by agr

To gain a better understanding about the role of RNAIII in the agr regulon, we produced an allelic replacement mutant of RNAIII and compared it to the isogenic MW2 wild-type and agr deletion strains using genome-wide transcriptional profiling with the MW2 microarray and quantitative real-time PCR analysis (Figures 1 and 2A, Table S1). Remarkably, we found that many genes were regulated by agr independently of RNAIII. This group comprised many agr-downregulated genes involved in carbohydrate and amino acid metabolism and biosynthesis of the yellow pigment staphyloxanthin (Figure 1, Table S1). Quantitative real-time PCR confirmed this for the most strongly downregulated hexose phosphate transporter gene MW0197 (Figure 2A).

Most strikingly, genes upregulated by agr independently of RNAIII almost exclusively comprised the psm$\alpha$ and psm$\beta$ genes (Figures 1 and 2A, Table S1). Differential expression values of only two additional genes in this group were low (Table S1). Additionally, there was differential expression of a gene locus, apparently forming an operon, composed of the MW0370/MW0372 genes, which code for proteins of unknown function present only in the MW2, MSSA476, and RF122 strains among the 13 sequenced S. aureus genomes. Interestingly, the quantitative real-time PCR data revealed RNAIII-dependent downregulation of the psm$\alpha$, psm$\beta$, and MW00370/0372
Molecular Cell

Quorum Sensing in Staphylococcus aureus

A

![Image Description](image1)

B

![Image Description](image2)

Figure 2. Agr- and RNAIII-Dependent Gene Regulation and PSM Secretion

(A) Shown are quantitative real-time PCR of selected agr-controlled target genes in S. aureus MW2 wild-type, and isogenic agr and RNAIII deletion strains. cDNA samples used were the same as for the microarray analyses, prepared from RNA isolated from cells grown to postexponential growth phase (4 hr). Probes were designed to align within the specific genes (MW0197) or within parts of the respective operons: MW0370 for the MW0370/MW0372 operon, psmβ1 for the psmβ operon, and parts of the psmα1 gene and the intergenic region between psmα1 and psmα2 for the psmα operon (see Table S3 for probe sequences). Values are from three independent samples ± SEM ***p < 0.001; **p < 0.01; *p < 0.05.

(B) PSM concentration in culture filtrates of isogenic wild-type, agr, and RNAIII deletion strains. PSM amounts in 16 hr culture filtrates were determined by RP-HPLC/ESI-MS of three independent samples. Values are ± SEM, relative to the parental MW2 (100%). Asterisk indicates no detectable amounts of any PSM. 0, below detection limit.

The Dramatic Impact of agr on PSM Expression Is Not Mediated by RNAIII

To analyze the impact of RNAIII on PSM production at the translational level, we performed reverse-phase high-pressure liquid chromatography/electrospray mass spectrometry (RP-HPLC/ESI-MS) of culture filtrates. There was no detectable PSMα or PSMβ expression in the agr mutant strain, in contrast to considerable PSMα and PSMβ quantities in the wild-type and, importantly, the RNAIII deletion strain (Figure 2B). As mentioned above, the δ-toxin is not produced in the agr or RNAIII deletion strains, because the hld gene is embedded within RNAIII and thus deleted in both these strains. Slight differences that we observed in PSM levels between the wild-type and RNAIII deletion strains were not consistent between the PSMs of the same operon and may be explained by posttranslational effects and our observation that δ-toxin is needed for the coelution of more hydrophobic PSM peptides. Together, these results showed on the transcriptional and translational level that RNAIII has no considerable impact on PSM expression that would explain the dramatic overall influence of agr on PSM production.

Regulation of psm Gene Transcription Occurs by Direct Binding of AgrA to the psm Promoter Regions

We hypothesized that direct interaction of AgrA with the psmα and psmβ promoters is the most likely mechanism to account for RNAIII-independent control and the exceptionally strong influence of agr on the expression of the psmα and psmβ genes. To evaluate this hypothesis, we used (1) electrophoretic mobility shift assays (EMSAs) with natural and mutated AgrA consensus binding sequences, (2) DNase footprinting analyses, (3) expression studies with vector-expressed psm genes including promoter fragments of different lengths, and (4) measurement of PSM production with constitutive or inducible expression of vector-expressed agrA versus RNAIII genes in agr mutant strains. For the first and second experiments, we purified the AgrA protein of strain MW2 using expression in E. coli (Figure S1).

Sequence similarity to the agr P2 and P3 promoters (Koenig et al., 2004) indicated the presence of three possible AgrA binding sites in both the psmα and psmβ promoter regions (Figure 3A). Using EMSAs with specific probes, we detected AgrA binding to the psmα and psmβ promoter regions and determined the binding region to encompass two of the three putative binding sites in each promoter (fragments αS2 and βS1, Figures 3A and 3B). We observed multiple shifts very similar to those previously shown for the P2/P3 promoters (Koenig et al., 2004). In comparison, absence of shifts with the very similar fragments αS1 and βS2 (Figures 3A and 3B) indicated specificity of AgrA binding. Sequences similar to the binding sites in the psmα and psmβ promoter regions were found upstream of the MW0370/MW0372 locus (data not shown), suggesting that this locus is also under direct control by AgrA.

To confirm specific binding of AgrA, we exchanged one base in both of the two determined binding sequences of the psmα or psmβ promoters (Figure 3A). Binding of AgrA was reduced by a factor of ~2.5 to 5 for the psmα and ~1.5 to 2 for the psmβ promoter (Figure 3C). Binding to the psmβ promoter fragment was overall stronger, which may explain the smaller effect of the introduced 1 bp changes compared to the psmα promoter.

Molecular Cell

Quorum Sensing in Staphylococcus aureus

A

$psm\alpha$ promoter region

$\begin{aligned}
& \text { prom1 } \\
& \text { entire intergenic region (246 bp) } \\
& \text { prom2 } \\
& \text { ATGAGCTTAACCTCTATTTAAACCAATGATTGTAAAGATTT } \\
& \text { TACTCGAATGGAGATAATTTGGTTACTAACATTCTAAAA } \\
& \text { prom3 } \\
& \text { -35 } \\
& \text { -10 } \\
& \text { GTAAATGCACCTGTACAGTTAGGCAGTATTTCCCGTCTTTAAAATAAAAAATTCGCGAGTTATGATCATAAACAATTCAAGTTAGGAAAAAAAAATCAATTACGCACAAAGATAACTATGTACA } \\
& \text { CATTACGTGGACATGTCAATCCTCATAAAGGGCAGGAAAATTTTATTTTTTAAGCGTCAATACTAATATTGTTAAGTTCGAATCO } \\
& \text { SD } \\
& \text { ATGAAGTTTACTCATAAAGCAAAGGAGGTAATCTTAATG } \\
& \text { psm\(\alpha\) genes } \\
& \text { (psm\(\alpha\)1) } \\
& \text { TACTTCAAATGAGTATTCGTTTCCCTCATTAGAATTAC }
\end{aligned}$

$psm\beta$ promoter region

$\begin{aligned}
& \text { \(\beta S 1\) } \\
& \text { G/C \(\rightarrow\) A/T } \\
& \text { -10 } \\
& \text { AAATACGAATTCAGGCAAATTAATTGTGTTAAATACAGTTTTGAATCCTACTGTATTTCTTTTCTCTTTAAATACAGTTAAATACATTATAAGATGTTGTGCCGATAAACAAACTAT } \\
& \text { TTTATGCTTAAGTCCGTTGAATTAATGTCAAAACGGATTGACATAAAGAAAAAGAGAAATTTATGTCAATTOATGTAATATCTACAACAGGCCTATTTGTTTGATTA } \\
& \text { * } \\
& \text { SD } \\
& \text { TGCAATCAATTTATTTTTAAAAATAACAAACACAAAACGTTAAGCGAATAACATTCGGTGATTTAAAGCTACGCACGTTTTTGTATCTCTAAATTTAAATTTTAAGGAGTGTCTCAATG } \\
& \text { ACGTAGTTTAAATAAAATTTTATTGTTGTGTTTTGCAATTCGCTTATTGTAAGCCACTAAATTTTCGATGCGTGCAAAAACAATAGAAGTTTAAATTTAAATTCCTCACAAAAGTTAC } \\
& \text { psm\(\beta\) genes } \\
& \text { (psm\(\beta\)1) }
\end{aligned}$

B

$\begin{array}{cccccc} \text { AgrA } & - & + & - & + & - \\ \hline \text { psma promoter } & & & & & \\ \end{array}$

$\begin{array}{c} \text { 2nd } \\ \text { 1st } \\ \text { unbound } \end{array}$

control $\quad \alpha S 1 \quad \alpha S 2$

$\begin{array}{cccccc} \text { AgrA } & - & + & - & + & - \\ \hline \text { psmb promoter } & & & & & \\ \end{array}$

$\begin{array}{c} \text { 2nd } \\ \text { 1st } \\ \text { unbound } \end{array}$

control $\quad \beta S 2 \quad \beta S 1$

C

$psm\alpha$ promoter

$\begin{array}{c} \text { 2nd } \\ \text { 1st } \\ \text { unbound } \end{array}$

$\alpha S 2$

$psm\beta$ promoter

$\begin{array}{c} \text { 2nd } \\ \text { 1st } \\ \text { unbound } \end{array}$

$\alpha S 2-\text { mut. }$

$\begin{array}{c} \text { 2nd } \\ \text { 1st } \\ \text { unbound } \end{array}$

$\beta S 1-\text { mut. }$

Figure 3. Binding of AgrA to $psm$ Promoter Regions

(A) Promoter sequences of the $psm\alpha$ and $psm\beta$ loci. The ATG start codon of the first gene ($psm\alpha 1$, $psm\beta 1$) of the respective operon is marked in italic font and shaded in gray. Putative ribosomal binding sites (Shine-Dalgarno sequences, “SD”) are also shaded in gray. Putative AgrA-binding sites in the $psm\alpha$ and $psm\beta$ promoter regions, according to comparison with the agr P2 and P3 promoters, are boxed, and those found to be important for AgrA binding by EMSAs (B and C) are in blue. Fragments used for EMSAs are indicated by bars above the sequences (designated $\alpha S 1$, $\alpha S 2$, $\beta S 1$, $\beta S 2$, dark blue for fragments giving signals in the EMSAs, light blue for those without signal). Bases exchanged for mutated binding site analyses (C) are marked by a blue arrow and “G/C → A/T,” describing the nature of the introduced mutation. The AgrA-binding region, according to DNase footprinting analyses (Figure S2), is shaded in light red. Start points of fragments used for cloning in $psm\alpha$ expression vectors (Figure 4A) are marked prom1, prom2, and prom3 with an arrow (in purple). Transcription start sites were determined experimentally by 5′-RACE (in green, *). –10 and –35 regions, labeled in green, are deduced from the transcription start sites.

(B) EMSAs of different fragments (S1, S2, see [A]) of $psm\alpha$ and $psm\beta$ promoters (25 pM) with AgrA (10 nM). The control fragment was amplified with the same oligonucleotides as used by Koenig et al. (Koenig et al., 2004).

(C) EMSAs with natural and mutated binding sites of the $psm\alpha$-S2 and the $psm\beta$-S1 fragments, to which AgrA binding was demonstrated in (B), under corresponding experimental conditions. Shifts are marked. Acetyl phosphate (50 mM) was added to all EMSAs.

Molecular Cell

Quorum Sensing in Staphylococcus aureus

Additionally, EMSAs with $psm\alpha$ promoter-mutated binding sites showed a low shift that was also observed by Koenig et al. when acetyl phosphate was omitted (Koenig et al., 2004), most likely indicative of AgrA binding as an inactive monomer. Moreover, to determine the exact AgrA-binding location, we performed DNase footprinting analyses, which pinpointed the AgrA-binding sites to regions overlaying ($psm\beta$ promoter) or immediately adjacent to ($psm\alpha$ promoter) the consensus sites derived from the comparison with the P2/P3 promoters and the EMSA experiments (Figure 3A, Figure S2). These experiments substantiated that AgrA binds specifically to the $psm\alpha$ and $psm\beta$ promoters.

To further confirm direct control of $psm$ gene expression by AgrA on a translational level, we focused on the $psm\alpha$ genes as those having the greatest biological importance (Wang et al., 2007). We transformed a $psm\alpha$ deletion strain of MW2 with $psm\alpha$ expression vectors containing different lengths of the $psm\alpha$ promoter region (Figure 3A). As expected, we observed PSM$\alpha$ production only with the constructs containing the region to which we have demonstrated AgrA binding (Figure 4A). Interestingly, our results also indicated that the $psm\alpha$ genes are under minor negative control of additional regulators that act on the intergenic region upstream of the AgrA-binding site. These findings are in accordance with those from our previous study indicating that, while agr appears to be the major regulator controlling PSM levels, PSM expression in various S. aureus strains is not entirely correlated with agr expression and might thus be subject to additional regulation other than by agr (Wang et al., 2007).

To substantiate that AgrA rather than RNAIII impacts PSM expression, we cloned the agrA gene or RNAIII-encoding region in plasmids with inducible or constitutive control of expression and transformed the agr-negative strain S. aureus 601055 mut6 (Vuong et al., 2000) with these plasmids. PSM expression was only achieved when agrA, but not RNAIII, was induced or constitutively expressed (Figure 4B). Values for the $\delta$-toxin and PSM$\alpha$3 peptides are shown as examples, while the remaining PSM peptide values changed accordingly (data not shown). These results confirmed that $psm$ genes are under the control of AgrA, but not RNAIII. By using a different strain background, these experiments also ruled out that the regulatory mechanism we describe here is strain specific.

### PSM Peptides Are Not Involved in Gene Regulation

While AgrA binding to the $psm$ promoter regions explained RNAIII-independent upregulation, the mechanism by which downregulation by agr in an RNAIII-independent manner occurs remained unclear. However, no reasonable number of consensus sequences was found in the S. aureus MW2 genome, possibly indicating that yet-unidentified, additional structures are important for AgrA binding or that the respective genes are not regulated by direct binding of AgrA.

As an alternative explanation, we hypothesized that the RNAIII-independent downregulation of agr target genes may occur indirectly via a regulatory effect of the PSMs or the MW0370/MW0372 proteins. In fact, gene-regulatory effects of S. aureus toxins (Vojtov et al., 2002) and contrasting reports on a potential regulatory role of $\delta$-toxin have been published (Janzon and Arvidson, 1990; Novick et al., 1995). However,

---

**Figure 4. Control of $psm$ Expression by AgrA—Translational Analyses**

(A) PSM$\alpha$ concentrations in culture filtrates of the MW2 $psm\alpha$ deletion strain complemented with the $psm\alpha$ locus under control of different lengths of the $psm\alpha$ promoter regions (see Figure 3A). PSM amounts in 16 hr culture filtrates were determined by RP-HPLC/ESI-MS of three independent samples. Values are ± SEM. Maximal values obtained with the prom2 fragment were set to 100%. Asterisk indicates no detectable amounts of any PSM$\alpha$.

(B) Complementation of PSM expression in an agr mutant strain by agrA versus RNAIII. RNAIII or agrA was cloned in pTX15 (Peschel et al., 1996) for xylose-inducible control or pTX$_{\Delta}$ (Wang et al., 2007) for strong constitutive expression and transformed in S. aureus 601055 mut6 (“agr”), which has a nonfunctional agr system. PSM production in the control strains S. aureus 601055 (pTX16) or S. aureus 601055 (pTX$_{\Delta}$) was set to 100% (WT). Values for PSM$\alpha$3 and $\delta$-toxin are shown as examples. All pTX15 derivative strains were grown in TSB without glucose and with (“+”) or without (“−”) 0.5% xylose. All pTX$_{\Delta}$ derivative strains were grown in TSB. Expression of $\delta$-toxin from the plasmid-encoded RNAIII is not shown. Plasmid-encoded is distinguishable from the chromosomally encoded $\delta$-toxin in strain 601055 by its mass, because the plasmid-encoded MW2 RNAIII encodes a $\delta$-toxin with a glycine to serine substitution at position 10 (Baba et al., 2002).

comparative transcriptional profiling with MW2 and its isogenic psm (Wang et al., 2007) and MW0370/MW0372 gene deletion strains clearly ruled out a regulatory function of those proteins, notably including δ-toxin (GenBank accession number GSE10165).

**DISCUSSION**

Quorum-sensing regulation in staphylococci is assumed to be accomplished exclusively by the agr system. An alleged role of the TRAP protein in staphylococcal quorum sensing was found to be due to an experimental artifact (Adhikari et al., 2007), and a quorum-sensing role of the ubiquitous luxS system in staphylococci is being debated but is unlikely (Doherty et al., 2006; Li et al., 2008). Here we show that there are two distinct subsets of agr target gene regulation: an RNAIII independent that is primarily in charge of controlling metabolic genes, and an RNAIII dependent controlling virulence (Figure 1). Additionally, we demonstrate that agr-dependent regulation of the PSM gene family is achieved by direct binding of the AgrA response regulator protein.

Our data indicate that control of virulence factors by RNAIII and quorum-sensing control of metabolism were once separate regulatory circuits. The connection of RNAIII-dependent with quorum-sensing regulation likely evolved to allow for a quick cell-density-dependent change of virulence factor expression during infection, because regulation by an RNA rather than a protein is believed to lead to a shorter response time (Bejerano-Sagie and Xavier, 2007). The inclusion of the PSM toxins—in contrast to other virulence determinants—in the primary core set of agr-regulated genes is most likely due to their presumed original function in the noninfectious lifestyle of S. aureus, i.e., in biofilm development (Kong et al., 2006). While it is difficult to dissect which regulatory circuit existed first, the architecture of the system indicates secondary connection of virulence factor control to quorum-sensing regulation. Formation of the RNAIII-encoding region around the hld gene coding for δ-toxin represents a much more likely event than change of a part of RNAIII to contain a structural gene and simultaneous addition of an AgrA-dependent promoter. Of note, this model is in accordance with the facts that most staphylococcal species have no or only limited pathogenicity and that the evolution of virulent species such as S. aureus is believed to be a phylogenetically more recent event.

In addition to giving insight into the mechanism and evolution of target gene regulation by agr, our study determined the agr regulon of a CA-MRSA strain. The molecular basis of the exceptional virulence of CA-MRSA strains is poorly understood but believed to contribute significantly to the CA-MRSA pandemic (Diep and Otto, 2008). Notably, the only two factors demonstrated until now to have a major influence on the virulence of CA-MRSA in infection models, α-toxin and α-type PSMs (Wang et al., 2007; Wardenburg et al., 2007), are both regulated by agr (Recsei et al., 1986; Wang et al., 2007). These findings suggest that agr plays a key role in determining the virulence potential of CA-MRSA. In this regard, it is important to stress that our data indicate very strict control of PSM, protease, and overall toxin expression by agr in CA-MRSA. Additionally, we show agr-

dependent regulation of the PVL, which has been suggested to play a role in CA-MRSA infection based on epidemiological data (Gillet et al., 2002) but for which a clear role in CA-MRSA infection models could not be established (Voyich et al., 2006; Wardenburg et al., 2007). Furthermore, in contrast to previous reports on agr regulation, agr targets in the CA-MRSA strain MW2 did not comprise surface-binding proteins and the SarA paralog Rot. Additional studies will be needed to analyze whether these features are characteristic for clinically important, especially CA-MRSA, strains.

In conclusion, our study reveals an additional level of quorum-sensing target gene control in the major human pathogen S. aureus and exemplifies how gene control by an sRNA may be intertwined during evolution with another global regulatory circuit, adding virulence factor control to a quorum-sensing regulon originally in charge of primarily basic cell physiology.

**EXPERIMENTAL PROCEDURES**

**PSM Measurement by RP-HPLC/ESI-MS**

Determination of PSM production in S. aureus culture filtrates was performed using RP-HPLC/ESI-MS essentially as described (Wang et al., 2007). Samples were taken from cultures inoculated from precultures and grown for 16 hr. PSMs accumulate in the culture filtrate after agr expression starts in postexponential growth phase. Afterwards, PSM concentration in culture filtrates is relatively stable over several hours during stationary growth phase (Wang et al., 2007). PSM concentrations were determined by computing extracted ion chromatograms (EICs) of the most abundant m/z peak for every peptide and integration of the EIC peaks at the elution times corresponding to the respective peptide, using Agilent QuantAnalysis Software (see Figure S4). Values obtained from the peaks representing the N-formylated and the N-deformylated forms of a specific PSM were added.

**Purification of AgrA**

The AgrA expression clone was constructed by PCR amplification of the agrA gene from chromosomal DNA of strain MW2 and cloning into the expression vector pET9a (New England Biolabs). Purification of AgrA was performed as described (Koenig et al., 2004), except that SOURCE Q material (GE Healthcare) was used for the first chromatographic step and a Resource S 1 ml column (GE Healthcare) for the second, with gradient elution over 20 column volumes. Obtained AgrA was analyzed by SDS-PAGE (Figure S1), N-terminal sequencing, and analytical RP-HPLC/ESI-MS and found to be correct and pure.

**Electrophoretic Mobility Shift Assays**

EMSAs were performed as previously described (Koenig et al., 2004) with the following modifications. Final DNA concentrations were 25 pM. Binding reactions were performed in 20 μl, incubated at room temperature for 30 min, and run on an 8% nondenaturing 0.5 × TBE gel. Acetyl phosphate was added to a final concentration of 50 mM.

**DNase Footprinting**

DNase footprinting analysis was performed using the method described by Yindeeyoungyeon and Schell (Yindeeyoungyeon and Schell, 2000). For details and further methods, see the Supplemental Data.

**SUPPLEMENTAL DATA**

The Supplemental Data include Supplemental Experimental Procedures, three tables, four figures, and Supplemental References and can be found with this article online at http://www.molecule.org/cgi/content/full/32/1/150/DC1/.

156 Molecular Cell 32, 150–158, October 10, 2008 ©2008 Elsevier Inc.

Molecular Cell

Quorum Sensing in Staphylococcus aureus

**ACKNOWLEDGMENTS**

The authors thank Larye Parkins for genome pattern searches, Frank Gherardini and Julie Boylan for helpful advice, and Frank DeLeo for critically reading the manuscript. This study was supported by the Intramural Research Program of the National Institute of Allergy and Infectious Diseases, National Institutes of Health.

Received: February 4, 2008  
Revised: June 11, 2008  
Accepted: August 1, 2008  
Published: October 9, 2008  

**REFERENCES**

Adhikari, R.P., Arvidson, S., and Novick, R.P. (2007). A nonsense mutation in *agrA* accounts for the defect in *agr* expression and the avirulence of *Staphylococcus aureus* 8325–4 *traP:kan*. Infect. Immun. 75, 4534–4540.

Alksne, L.E., and Projan, S.J. (2000). Bacterial virulence as a target for antimicrobial chemotherapy. Curr. Opin. Biotechnol. 11, 625–636.

Baba, T., Takeuchi, F., Kuroda, M., Yuzawa, H., Aoki, K., Oguchi, A., Nagai, Y., Iwama, N., Asano, K., Naimi, T., et al. (2002). Genome and virulence determinants of high virulence community-acquired MRSA. Lancet 359, 1819–1827.

Bejerano-Sagie, M., and Xavier, K.B. (2007). The role of small RNAs in quorum sensing. Curr. Opin. Microbiol. 10, 189–198.

Blevins, J.S., Beenken, K.E., Elasri, M.O., Hurlburt, B.K., and Smeltzer, M.S. (2002). Strain-dependent differences in the regulatory roles of *sarA* and *agr* in *Staphylococcus aureus*. Infect. Immun. 70, 470–480.

Boisset, S., Geissmann, T., Huntzinger, E., Fechter, P., Bendridi, N., Possedko, M., Chevalier, C., Helfer, A.C., Benito, Y., Jacquier, A., et al. (2007). *Staphylococcus aureus* RNAIII coordinately represses the synthesis of virulence factors and the transcription regulator Rot by an antisense mechanism. Genes Dev. 21, 1353–1366.

Camilli, A., and Bassler, B.L. (2006). Bacterial small-molecule signaling pathways. Science 311, 1113–1116.

Cassat, J., Dunman, P.M., Murphy, E., Projan, S.J., Beenken, K.E., Palm, K.J., Yang, S.J., Rice, K.C., Bayles, K.W., and Smeltzer, M.S. (2006). Transcriptional profiling of a *Staphylococcus aureus* clinical isolate and its isogenic *agr* and *sarA* mutants reveals global differences in comparison to the laboratory strain RN6390. Microbiology 152, 3075–3090.

CDC (Centers for Disease Control and Prevention). (1999). Four pediatric deaths from community-acquired methicillin-resistant *Staphylococcus aureus*—Minnesota and North Dakota, 1997–1999. JAMA 282, 1123–1125.

Clatworthy, A.E., Pierson, E., and Hung, D.T. (2007). Targeting virulence: a new paradigm for antimicrobial therapy. Nat. Chem. Biol. 3, 541–548.

Diep, B.A., and Otto, M. (2008). The role of virulence determinants in community-associated MRSA pathogenesis. Trends Microbiol. 16, 361–369. Published online June 26, 2008. 10.1016/j.tim.2008.05.002.

Doherty, N., Holden, M.T., Qazi, S.N., Williams, P., and Winzer, K. (2006). Functional analysis of *luxS* in *Staphylococcus aureus* reveals a role in metabolism but not quorum sensing. J. Bacteriol. 188, 2885–2897.

Dunman, P.M., Murphy, E., Haney, S., Palacios, D., Tucker-Kellogg, G., Wu, S., Brown, E.L., Zagursky, R.J., Shlaes, D., and Projan, S.J. (2001). Transcription profiling-based identification of *Staphylococcus aureus* genes regulated by the *agr* and/or *sarA* loci. J. Bacteriol. 183, 7341–7353.

Forsgren, A., and Nordstrom, K. (1974). Protein A from *Staphylococcus aureus*: the biological significance of its reaction with IgG. Ann. N Y Acad. Sci. 236, 252–266.

Gillet, Y., Issartel, B., Vanhems, P., Fournet, J.C., Lina, G., Bes, M., Vandenesch, F., Piemont, Y., Brousse, N., Floret, D., and Etienne, J. (2002). Association between *Staphylococcus aureus* strains carrying gene for Panton-Valentine leukocidin and highly lethal necrotising pneumonia in young immunocompetent patients. Lancet 359, 753–759.

Gomez, M.I., Lee, A., Reddy, B., Muir, A., Soong, G., Pitt, A., Cheung, A., and Prince, A. (2004). *Staphylococcus aureus* protein A induces airway epithelial inflammatory responses by activating TNFR1. Nat. Med. 10, 842–848.

Janzon, L., and Arvidson, S. (1990). The role of the delta-lysin gene (*hld*) in the regulation of virulence genes by the accessory gene regulator (*agr*) in *Staphylococcus aureus*. EMBO J. 9, 1391–1399.

Kleerebezem, M., Quadri, L.E., Kuipers, O.P., and de Vos, W.M. (1997). Quorum sensing by peptide pheromones and two-component signal-transduction systems in Gram-positive bacteria. Mol. Microbiol. 24, 895–904.

Koenig, R.L., Ray, J.L., Maleki, S.J., Smeltzer, M.S., and Hurlburt, B.K. (2004). *Staphylococcus aureus* AgrA binding to the RNAIII-*agr* regulatory region. J. Bacteriol. 186, 7549–7555.

Kong, K.F., Vuong, C., and Otto, M. (2006). *Staphylococcus* quorum sensing in biofilm formation and infection. Int. J. Med. Microbiol. 296, 133–139.

Li, M., Cha, D.J., Lai, Y., Villaruz, A.E., Sturdevant, D.E., and Otto, M. (2007). The antimicrobial peptide-sensing system *aps* of *Staphylococcus aureus*. Mol. Microbiol. 66, 1136–1147.

Li, M., Villaruz, A.E., Vadyvaloo, V., Sturdevant, D.E., and Otto, M. (2008). Al-2-dependent gene regulation in *Staphylococcus epidermidis*. BMC Microbiol. 8, 4.

McNamara, P.J., Milligan-Monroe, K.C., Khalili, S., and Proctor, R.A. (2000). Identification, cloning, and initial characterization of *rot*, a locus encoding a regulator of virulence factor expression in *Staphylococcus aureus*. J. Bacteriol. 182, 3197–3203.

Novick, R.P. (2003). Autoinduction and signal transduction in the regulation of staphylococcal virulence. Mol. Microbiol. 48, 1429–1449.

Novick, R.P., Ross, H.F., Projan, S.J., Kornblum, J., Kreiswirth, B., and Moghazeh, S. (1993). Synthesis of staphylococcal virulence factors is controlled by a regulatory RNA molecule. EMBO J. 12, 3967–3975.

Novick, R.P., Projan, S.J., Kornblum, J., Ross, H.F., Ji, G., Kreiswirth, B., Vandenesch, F., and Moghazeh, S. (1995). The *agr* P2 operon: an autocatalytic sensory transduction system in *Staphylococcus aureus*. Mol. Gen. Genet. 248, 446–458.

Otto, M. (2004). Quorum-sensing control in *Staphylococci*—a target for antimicrobial drug therapy? FEMS Microbiol. Lett. 241, 135–141.

Passador, L., Cook, J.M., Gambello, M.J., Rust, L., and Iglewski, B.H. (1993). Expression of *Pseudomonas aeruginosa* virulence genes requires cell-to-cell communication. Science 260, 1127–1130.

Peschel, A., Ottenwalder, B., and Gotz, F. (1996). Inducible production and cellular location of the epidermin biosynthetic enzyme EpiB using an improved staphylococcal expression system. FEMS Microbiol. Lett. 137, 279–284.

Recsei, P., Kreiswirth, B., O’Reilly, M., Schlievert, P., Gruss, A., and Novick, R.P. (1986). Regulation of exoprotein gene expression in *Staphylococcus aureus* by *agr*. Mol. Gen. Genet. 202, 58–61.

Romby, P., Vandenesch, F., and Wagner, E.G. (2006). The role of RNAs in the regulation of virulence-gene expression. Curr. Opin. Microbiol. 9, 229–236.

Said-Salim, B., Dunman, P.M., McAleese, F.M., Macapagal, D., Murphy, E., McNamara, P.J., Arvidson, S., Foster, T.J., Projan, S.J., and Kreiswirth, B.N. (2003). Global regulation of *Staphylococcus aureus* genes by Rot. J. Bacteriol. 185, 610–619.

Schuster, M., Lostroh, C.P., Ogi, T., and Greenberg, E.P. (2003). Identification, timing, and signal specificity of *Pseudomonas aeruginosa* quorum-controlled genes: a transcriptome analysis. J. Bacteriol. 185, 2066–2079.

Somerville, G.A., Cockayne, A., Durr, M., Peschel, A., Otto, M., and Musser, J.M. (2003). Synthesis and deformylation of *Staphylococcus aureus* delta-toxin are linked to tricarboxylic acid cycle activity. J. Bacteriol. 185, 6686–6694.

Toledo-Arana, A., Repoila, F., and Cossart, P. (2007). Small noncoding RNAs controlling pathogenesis. Curr. Opin. Microbiol. 10, 182–188.

Vojtov, N., Ross, H.F., and Novick, R.P. (2002). Global repression of exotoxin synthesis by staphylococcal superantigens. Proc. Natl. Acad. Sci. USA 99, 10102–10107.

Voyich, J.M., Braughton, K.R., Sturdevant, D.E., Whitney, A.R., Said-Salim, B., Porcella, S.F., Long, R.D., Dorward, D.W., Gardner, D.J., Kreiswirth, B.N., et al. (2005). Insights into mechanisms used by *Staphylococcus aureus* to avoid destruction by human neutrophils. J. Immunol. 175, 3907–3919.

Voyich, J.M., Otto, M., Mathema, B., Braughton, K.R., Whitney, A.R., Welty, D., Long, R.D., Dorward, D.W., Gardner, D.J., Lina, G., et al. (2006). Is Panton-Valentine leukocidin the major virulence determinant in community-associated methicillin-resistant *Staphylococcus aureus* disease? J. Infect. Dis. 194, 1761–1770.

Vuong, C., Saenz, H.L., Gotz, F., and Otto, M. (2000). Impact of the agr quorum-sensing system on adherence to polystyrene in *Staphylococcus aureus*. J. Infect. Dis. 182, 1688–1693.

Wang, R., Braughton, K.R., Kretschmer, D., Bach, T.H., Queck, S.Y., Li, M., Kennedy, A.D., Dorward, D.W., Klebanoff, S.J., Peschel, A., et al. (2007).

Identification of novel cytolytic peptides as key virulence determinants for community-associated MRSA. Nat. Med. 13, 1510–1514.

Wardenburg, J.B., Bae, T., Otto, M., Deleo, F.R., and Schneewind, O. (2007). Poring over pores: alpha-hemolysin and Panton-Valentine leukocidin in *Staphylococcus aureus* pneumonia. Nat. Med. 13, 1405–1406.

Yao, Y., Vuong, C., Kocianova, S., Villaruz, A.E., Lai, Y., Sturdevant, D.E., and Otto, M. (2006). Characterization of the *Staphylococcus epidermidis* accessory-gene regulator response: quorum-sensing regulation of resistance to human innate host defense. J. Infect. Dis. 193, 841–848.

Yindeeyoungyeon, W., and Schell, M.A. (2000). Footprinting with an automated capillary DNA sequencer. Biotechniques 29, 1034–1036, 1038, 1040–1031.
