
Sulfur Metabolism in Phototrophic Sulfur Bacteria

Niels-Ulrik Frigaard¹ and Christiane Dahl²

¹Copenhagen Biocenter, Department of Biology, University of Copenhagen, Ole Maaløes Vej 5,  
2200 Copenhagen N, Denmark  

²Institut für Mikrobiologie & Biotechnologie, Rheinische Friedrich-Wilhelms-Universität Bonn,  
Meckenheimer Allee 168, D–53115 Bonn, Germany  

ABSTRACT

Phototrophic sulfur bacteria are characterized by oxidizing various  
inorganic sulfur compounds for use as electron donors in carbon dioxide  
fixation during anoxygenic photosynthetic growth. These bacteria are  
divided into the purple sulfur bacteria (PSB) and the green sulfur bacteria  
(GSB). They utilize various combinations of sulfide, elemental sulfur, and  
thiosulfate and sometimes also ferrous iron and hydrogen as electron  
donors. This review focuses on the dissimilatory and assimilatory  
metabolism of inorganic sulfur compounds in these bacteria and also briefly  
discusses these metabolisms in other types of anoxygenic phototrophic  
bacteria. The biochemistry and genetics of sulfur compound oxidation in  
PSB and GSB are described in detail. A variety of enzymes catalyzing sulfur  
oxidation reactions have been isolated from GSB and PSB (especially  
Allochromatium vinosum, a representative of the Chromatiaceae), and many  
are well characterized also on a molecular genetic level. Complete genome  
sequence data are currently available for 10 strains of GSB and for one  
strain of PSB. We present here a genome-based survey of the distribution  
and phylogenies of genes involved in oxidation of sulfur compounds in these  
strains. It is evident from biochemical and genetic analyses that the  
dissimilatory sulfur metabolism of these organisms is very complex and  
incompletely understood. This metabolism is modular in the sense that

ADVANCES IN MICROBIAL PHYSIOLOGY, VOL. 54  
ISBN 978-0-12-374323-7  
DOI: 10.1016/S0065-2911(08)00002-7  
Copyright © 2009 by Elsevier Ltd.  
All rights reserved

individual steps in the metabolism may be performed by different enzymes
in different organisms. Despite the distant evolutionary relationship between
GSB and PSB, their photosynthetic nature and their dependency on
oxidation of sulfur compounds resulted in similar ecological roles in the
sulfur cycle as important anaerobic oxidizers of sulfur compounds.

Abbreviations ..... 105
1. Introduction ..... 106
2. Anoxygenic phototrophic bacteria: physiology and taxonomy ..... 107
   2.1. Purple Sulfur Bacteria ..... 109
   2.2. Green Sulfur Bacteria ..... 112
   2.3. Purple Non-Sulfur Bacteria ..... 113
   2.4. Aerobic Bacteriochlorophyll-Containing Bacteria ..... 114
   2.5. Filamentous Anoxygenic Phototrophs ..... 114
   2.6. Heliobacteria ..... 115
3. Anoxygenic phototrophic sulfur bacteria: ecology ..... 116
4. Transformations of sulfur compounds ..... 118
   4.1. Purple Sulfur Bacteria ..... 118
   4.2. Green Sulfur Bacteria ..... 123
   4.3. Purple Non-Sulfur Bacteria ..... 123
   4.4. Aerobic Bacteriochlorophyll-Containing Bacteria ..... 124
   4.5. Filamentous Anoxygenic Phototrophs ..... 125
   4.6. Heliobacteria ..... 126
5. Oxidative sulfur metabolism: enzymes and genes ..... 126
   5.1. Oxidation of H₂S ..... 130
   5.2. Oxidation of Polysulfides ..... 134
   5.3. Uptake of External Sulfur ..... 135
   5.4. Properties of Sulfur Globules ..... 138
   5.5. Oxidation of Stored Sulfur to Sulfite ..... 143
   5.6. Oxidation of Sulfite to Sulfate ..... 148
   5.7. Oxidation of Thiosulfate ..... 155
   5.8. Other Enzymes Related to Sulfur Compound Oxidation ..... 162
6. Evolution of sulfur metabolism ..... 165
7. Sulfate assimilation in anoxygenic phototrophic bacteria ..... 167
   7.1. Historical Aspects and General Outline of the Pathway ..... 167
   7.2. Occurrence of Sulfate Assimilation in Anoxygenic Phototrophic
        Bacteria ..... 169
   7.3. Uptake of Sulfate ..... 174
   7.4. Activation of Sulfate ..... 174
   7.5. Reduction to Sulfite ..... 175
   7.6. Reduction of Sulfite to Sulfide ..... 176
   7.7. Incorporation of Sulfide ..... 177
8. Conclusions ..... 177
Acknowledgements ..... 179
References ..... 179

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 105

ABBREVIATIONS

ABC bacteria    aerobic bacteriochlorophyll-containing bacteria  
Alc.            Allochromatium  
APAT            Adenylylsulfate phosphoadenylyl transferase  
APS             adenosine-5'-phosphosulfate  
Atb.            Acidithiobacillus  
BChl            bacteriochlorophyll  
Cba.            Chlorobaculum  
Chl.            Chlorobium  
Cfl.            Chloroflexus  
Chp.            Chloroherpeton  
DMSP           dimethylsuloniopropionate  
Dsv.            Desulfobivrio  
Ect.            Ectothiorhodospira  
EPS             extracellular polymeric substances  
FAP bacteria   filamentous anoxygenic phototrophic bacteria  
GSB             green sulfur bacteria  
Hlr.            Halorhodospira  
LGT             lateral gene transfer  
LH complex      light-harvesting complex  
MPT             molybdopterin  
PAPS            3'-phosphoadenosine-5'-phosphosulfate  
Pcs.            Paracoccus  
PSB             purple sulfur bacteria  
PSR             polysulfide reductase  
Rba.            Rhodobacter  
RC              (photosynthetic) reaction center  
RLP             RuBisCO-like protein  
Rpl.            Rhodoplanes  
rRNA            ribosomal RNA  
SGP             sulfur globule protein  
SQR             sulfide:quinone reductase  
Tca.            Thiocapsa  
Thb.            Thiobacillus  
Tms.            Thiomicrospira  
Wol.            Wolinella

106 NIELS-ULRIK FRIGAARD AND CHRISTIANE DAHL

# 1. INTRODUCTION

The most important common property of phototrophic prokaryotes is the possession of tetrapyrrole pigments and a photosynthetic apparatus enabling light-dependent electron transfer and energy conservation processes. Only a few groups, all within the domain of *Bacteria*, possess these properties. Fundamental physiological differences exist between these groups: (a) the oxygenic phototrophic bacteria (cyanobacteria) that use water as photosynthetic electron donor and thus produce molecular oxygen as a product; (b) the anoxygenic phototrophic bacteria (groups of purple and green bacteria and heliobacteria) that use electron donors other than water, such as reduced sulfur compounds, organic compounds, hydrogen or ferrous iron; and (c) the aerobic anoxygenic bacteriochlorophyll-containing (ABC) bacteria, a group of poorly characterized *Proteobacteria* with a predominantly chemoheterotrophic physiology but with the potential to produce photosynthetic pigment–protein complexes and to perform light-mediated electron transport (Imhoff, 2008).

The utilization of reduced sulfur compounds, most notably sulfide, as photosynthetic electron donors is – although to a different extent – common among almost all groups of phototrophic bacteria. Classical in this respect are the purple sulfur bacteria (PSB; families *Chromatiaceae* and *Ectothiorhodospiraceae*) and green sulfur bacteria (GSB; family *Chlorobiaceae*) all of which utilize reduced sulfur compounds as electron donors and which collectively are known as phototrophic sulfur bacteria. A number of purple “non-sulfur” bacteria, some members of the filamentous anoxygenic phototrophs (FAP; relatives of *Chloroflexus*, previously known as green gliding bacteria or green non-sulfur bacteria), and a few representatives of the strictly anaerobic heliobacteria are also able to oxidize reduced sulfur compounds during phototrophic growth. Even certain species of cyanobacteria can perform anoxygenic photosynthesis at the expense of sulfide as an electron donor (Cohen et al., 1975). Photoautotrophic growth with sulfur compounds has so far not been described for any of the ABC bacteria.

We will first introduce the physiology, taxonomy, and ecology of anoxygenic phototrophic bacteria. Then the sulfur-oxidizing capabilities of the various groups of anoxygenic phototrophic bacteria will be described. It should be emphasized that some older reviews still serve as a valuable source of information especially regarding sulfur oxidation patterns by phototrophic sulfur bacteria (Brune, 1989, 1995b). This review focuses on sulfur compound oxidation in the PSB (*Chromatiaceae* and *Ectothiorhodospiraceae*) and in the GSB (*Chlorobiaceae*) with special attention on the current

knowledge of the biochemical details of the metabolic pathways employed in these groups. Within the past few years, much useful information on this subject has been derived from comparative analyses of the various available genome sequences from GSB and purple bacteria (Frigaard and Bryant, 2008a; Sander and Dahl, 2008) and this will surely continue to be the case as more genome sequences become available. We also give a brief overview of the assimilatory sulfate reduction metabolism of anoxygenic phototrophic bacteria. Regarding the metabolism of organosulfur compounds in anoxygenic phototrophic bacteria, we refer the reader to recent articles by Cook and coworkers (Baldock *et al.*, 2007; Denger *et al.*, 2004, 2006).

## 2. ANOXYGENIC PHOTOTROPHIC BACTERIA: PHYSIOLOGY AND TAXONOMY

The major groups of phototrophic bacteria are well-distinguished on the basis of fundamental structural, molecular, ecological, and physiological properties (Table 1). While the oxygenic cyanobacteria represent a separate large phylogenetic lineage not discussed further here, the anoxygenic phototrophic bacteria are found in only five different major phylogenetic lineages: (1) the FAP bacteria (*Chloroflexus* and relatives), (2) the GSB (comprising the family *Chlorobiaceae*), (3) the heliobacteria (comprising the family *Heliobacteriaceae* whose members are related to Gram-positive bacteria), (4) the purple bacteria (divided into purple sulfur and purple non-sulfur species, all of which belong to either the Alpha-, Beta- or Gammaproteobacteria and the ABC bacteria that also belong to the Alpha-, Beta- and Gammaproteobacteria, and (5) the phototrophic acidobacteria. The latter group is a recently discovered lineage of phototrophic bacteria of which only a single organism in enriched culture is known (Bryant *et al.*, 2007). This bacterium, “*Candidatus Chloroacidobacterium thermophilum,*” belongs to the poorly characterized phylum Acidobacteria and was derived from microbial mats of alkaline siliceous hot springs. Like GSB, it synthesizes BChl *a* and *c*, chlorosomes, FMO protein, and type I reaction centers, but unlike GSB it grows under aerobic conditions. It is not yet available in pure culture and it is not known to oxidize sulfur compounds for growth.

The variations in photosynthetic reaction centers (RCs), light-harvesting structures, and different CO₂ fixation pathways among these groups are quite remarkable (Table 1). All known photosynthetic RCs can be divided into two main groups: type I, in which a Fe–S cluster is the terminal electron

| Group | Purple sulfur bacteria (PSB) | Aerobic bacteriochlorophyll-containing (ABC) bacteria | Green sulfur bacteria (GSB) | Filamentous anoxygenic phototrophic (FAP) bacteria | Heliobacteria |
| --- | --- | --- | --- | --- | --- |
| Phylogenetic affiliation | Gammaproteobacteria | Alpha- and Betaproteobacteria | Chlorobiaceae (phylum Chlorobi) | Chloroflexaceae (phylum Firmicutes, order Clostridiales) |  |
| Type of bacteriochlorophyll | BChl $a$, $b$ | BChl $a$, Zn-BChl $a$ | BChl $c$, $d$, $e$ (a minor component in all) | BChl $a$. Some also have BChl $c$, $d$ | BChl $g$ |
| Reaction center type | Pheophytin/quinone (type II) | Pheophytin/quinone (type II) | Fe-S (type I) | Fe-S (type I) | Fe-S (type I) |
| Antenna complexes | LHI and LHII in intracytoplasmic membranes | Most have FMO protein, no LHII | Organic chlorosomes | None<sup>a</sup> |  |
| Preferred electron donor for phototrophic growth | Reduced sulfur compounds, hydrogen, (organic compounds) | Organic compounds, hydrogen, (sulfide, ferrous iron in some) | Sulfide, hydrogen, (ferrous iron) | Organic compounds, (sulfide) |  |
| Chemotrophic growth | $+/-$ | + | - | $-/$+ | + |
| Pathway of autotrophic CO₂ fixation | Calvin cycle | If present: Calvin cycle. No photoautotrophic growth | Reductive TCA cycle | Hydroxypro-pionate pathway (Chloroflexus), Calvin cycle (Oscillochloris) | None |
| Major habitat | Anoxic marine and freshwater environments | Eutrophic aquatic environments | Anoxic marine and freshwater | Hot springs, meta- and hypolimnion of stratified lakes, shallow freshwater pools | Soil |

<sup>a</sup>Heliobacteria contain no other antenna chlorophylls than those associated with the photosynthetic reaction center in the cytoplasmic membrane.

acceptor, and type II, in which a quinone is the terminal electron acceptor. Photosystem II of oxygenic phototrophic bacteria and eukaryotes and the purple bacterial RC are members of the type II group, whereas photosystem I of oxygenic phototrophic bacteria and eukaryotes and the homodimeric RCs of GSB and heliobacteria are members of the type I group. A combination of the two types of photosystems as found in oxygenic phototrophs (all cyanobacteria and all eukaryotic phototrophs) is required for the thermodynamically unfavorable utilization of water as an electron donor for photosynthesis. Due to the simpler architecture of their photosynthetic apparatus, all anoxygenic phototrophic bacteria depend on electron donors with standard potentials more negative than that of water (e.g., reduced sulfur compounds, hydrogen, and acetate). This results in comparatively narrower ecological niches for these bacteria than for cyanobacteria in extant ecosystems (Overmann, 2008).

In the following sections, the properties of the various groups of anoxygenic phototrophic bacteria will briefly be described. Details on the transformations of sulfur compounds are described in Section 4.

## 2.1. Purple Sulfur Bacteria

The PSB belong to the Gammaproteobacteria and fall into two families, the Chromatiaceae and the Ectothiorhodospiraceae. Both form coherent groups on the basis of their 16S ribosomal RNA (rRNA) sequences. The most important and easily recognized and distinguishing feature among these two families is the formation of sulfur globules within the confines of the cell wall in Chromatiaceae during growth on sulfide, polysulfides, thiosulfate, or elemental sulfur (Fig. 1A), while sulfur globules accumulate extracellularly in members of the Ectothiorhodospiraceae (Brune, 1995b; Dahl, 2008). The Ectothiorhodospiraceae member Thiorhodospira sibirica is an exception as it forms both extra- and intracellular sulfur globules (Bryantseva et al., 1999). All PSB fix CO₂ using the Calvin cycle and their photosynthetic RC is type II.

Very little genome sequence information is available for the phototrophic members of this group despite their diversity and ecological significance. The genome sequence of Halorhodospira halophila SL1 (DSMZ 244ᵀ) has been determined (www.jgi.doe.gov). The genome of Thermochromatium tepidum MC (DSMZ 3771ᵀ) has been sequenced by a commercial enterprise and is not publicly available. However, the genome of the important model organism Allochromatium vinosum DSMZ 180ᵀ is in the process of being sequenced by the JGI.

110 NIELS-ULRIK FRIGAARD AND CHRISTIANE DAHL

![](image.png)

Figure 1 Phototrophic sulfur bacteria with sulfur globules. (A) *Thiocapsa* sp. with intracellular sulfur globules, light microscopy, magnification × 2000; (B) *Chlorobium phaeovibrioides* strain 9130 with extracellular sulfur globules, magnification × 2000; (C) *Thiocystis violacea* DSMZ 212 with intracellular sulfur globules. The vesicular intracytoplasmic membranes (“chromatophores”) are associated with the sulfur globules in a highly organized manner, magnification × 22,000. (D) Part of a *Thiocapsa roseopersicina* (strain 8811) cell, the sulfur globule envelope is clearly visible, magnification × 179,550. As a result of the preparation of the cells for electron microscopy, the localization of sulfur globules is visible as “holes” in the electron micrographs. All photos kindly provided by Hans G. Trüper, Bonn.

### 2.1.1. Chromatiaceae

Most species have the vesicular type of intracytoplasmic membranes (Fig. 1C and D). Many species are motile by means of flagella, and about one-third of

the species of the Chromatiaceae have gas vesicles. The gas-vesicle-bearing Chromatiaceae typically colonize low-light stratified environments.

Two major physiological groups of Chromatiaceae can be distinguished: the metabolically versatile species and the metabolically specialized species. The specialized species (among them many large-celled species, e.g., Chromatium okenii, Allochromatium warmingii, and Isochromatium buderi) depend on strictly anaerobic conditions and are obligately phototrophic. Sulfide is required whereas thiosulfate and hydrogen are not used as electron donors. Only acetate and pyruvate are photoassimilated in the presence of CO₂ and sulfide. Assimilatory sulfate reduction does not occur.

The versatile species (among them many small-celled species, e.g., Allochromatium vinosum and Allochromatium minutissimum) use thiosulfate in addition to sulfide and elemental sulfur as electron donors (Imhoff, 2005a). Most of these species are capable of assimilatory sulfate reduction, can grow photoorganoheterotrophically in the absence of reduced sulfur compounds, and can grow as chemolithotrophs on reduced sulfur compounds (Gorlenko, 1974; Kämpf and Pfennig, 1980; Kondrat’eva et al., 1976). Some species can even grow as chemoorganotrophs in which case the addition of sulfide or thiosulfate as a sulfur source is required because the assimilation of sulfate is repressed under aerobic conditions (Kondrat’eva et al., 1981).

### 2.1.2. Ectothiorhodospiraceae

The Ectothiorhodospiraceae are distinguished from the Chromatiaceae by lamellar intracytoplasmic membrane structures, by significant differences in polar lipid composition, and by the dependence on saline and alkaline growth conditions. All species are motile by polar flagella. Ect. vacuolata harbors gas vesicles.

Four phototrophic genera of Ectothiorhodospiraceae are validly described: Ectothiorhodospira, Halorhodospira, Thiorhodospira, and Ectothiorhodosinus. Formally, the genus Ectothiorhodosinus (Gorlenko et al., 2004) has no standing in the nomenclature. All species of the genus Ectothiorhodospira grow well under anoxic conditions in the light with reduced sulfur compounds as photosynthetic electron donors and in the presence of organic carbon sources and inorganic carbonate. Several species of the genus Ectothiorhodospira are also able to grow chemolithotrophically on sulfur compounds (e.g., Ectothiorhodospira haloalkaliphila and Ectothiorhodospira shaposhnikovii) (Imhoff, 2005b).

Non-phototrophic genera of Ectothiorhodospiraceae include Alkalilimnicola (Yakimov et al., 2001), Alkalilspirillum (Rijkenberg et al., 2001), and

Arhodomonas (Adkins *et al.*, 1993). Alkalilimnicola ehrlichii is capable of anaerobic chemoautotrophic growth using hydrogen, sulfide, or thiosulfate as electron donor and nitrate as electron acceptor (Hoeft *et al.*, 2007). Other non-phototrophic members of Ectothiorhodospiraceae are also able to oxidize sulfur compounds, including species of Thioalkalivibrio and Thioalkalispira (Sorokin and Kuenen, 2005; Sorokin *et al.*, 2001).

### 2.2. Green Sulfur Bacteria

The GSB form a cluster of closely related genera united in a single family, Chlorobiaceae, which represents the only members of the phylum Chlorobi that have been isolated in pure culture (Garrity and Holt, 2001a). The taxonomy and nomenclature of GSB have recently been extensively modified to reflect recent progress in understanding their phylogeny (Imhoff, 2003). BChl $c$, $d$, and $e$ are the major light-harvesting pigments. These pigments are organized into large organelles known as chlorosomes in all GSB characterized to date (Frigaard and Bryant, 2006). Chlorosomes allow a highly efficient capture of light energy and, as a result of this, growth at remarkably low light intensities. The only other organisms in which chlorosomes have been found are the “green FAPs” (see Section 2.5) and the phototrophic acidobacteria (see introduction to Section 2). BChl $a$ is always present in the chlorosomes, the peripheral Fenna–Matthews–Olson (FMO) antenna protein, and the photosynthetic RC. In addition, GSB contain various carotenoids of the chlorobactene and isorenieratene series. Intracytoplasmic membranes are not formed. All characterized GSB have similar metabolic properties: they are strictly anaerobic, obligately phototrophic, and obligately autotrophic. They can grow with CO₂ as sole carbon source but they will photoassimilate acetate if CO₂ is present. In contrast to the purple bacteria, CO₂ is fixed via the reductive tricarboxylic acid cycle (also known as the reverse citric acid cycle). Most strains use electrons derived from oxidation of sulfide, thiosulfate, elemental sulfur, and H₂, but a few characterized strains can also oxidize Fe²⁺ (Heising *et al.*, 1999). Sulfur is deposited outside of the cells (Fig. 1B). Some GSB form gas vesicles. One benthic species, Chloroherpeton thalassium, is motile by gliding. All known pelagic isolates of GSB are non-flagellated, although some have acquired motility by forming highly specific symbioses with motile chemoheterotrophic bacteria (Overmann and Schubert, 2002).

GSB are commonly found in anoxic and sulfide-rich freshwater and estuarine environments, either in the water column, in sediments, or within microbial mats. They have also recently been found in the anoxic zone 100 m

below the surface of the Black Sea (Manske *et al.*, 2005; Overmann *et al.*, 1992a), on deep-sea hydrothermal vents in the Pacific Ocean (Beatty *et al.*, 2005), and in the microbial mats of Octopus and Mushroom Springs in Yellowstone National Park (Ward *et al.*, 1998).

Compared to other anoxygenic phototrophic bacteria, a large amount of genome sequence information is available for the GSB. Currently, the genome sequences of 10 strains of GSB are publicly available ([www.ncbi.nlm.nih.gov](http://www.ncbi.nlm.nih.gov)): *Cba.* *tepidum* TLS (DSMZ 12025<sup>T</sup>), *Chl.* *chlorochromatii* CaD3, *Chl.* *clathratiforme* BU1 (DSMZ 5477<sup>T</sup>), *Chl.* *ferrooxidans* DSMZ 13031<sup>T</sup>, *Chl.* *limicola* DSMZ 245<sup>T</sup>, *Chl.* *luteolum* DSMZ 273<sup>T</sup>, *Chl.* *phaeobacteroides* DSMZ 266<sup>T</sup>, *Chl.* *phaeovibrioides* DSMZ 265, *Prosthecochloris* sp. BS1, *Ptc.* *aestuarii* SK413 (DSMZ 271<sup>T</sup>). The genomes of *Cba.* *parvum* NCIMB 8327 (DSMZ 263<sup>T</sup>) and *Chp.* *thalassium* ATCC 35110<sup>T</sup> are in the process of being sequenced.

## 2.3. Purple Non-Sulfur Bacteria

The group of purple non-sulfur bacteria is by far the most diverse group of the photosynthetic purple bacteria. Representatives are found within the Alpha- and the Betaproteobacteria (Imhoff *et al.*, 2005). Their diversity is reflected by highly variable cell morphology, internal membrane structure, carotenoid composition, utilization of carbon sources, and by the kind of electron donors used for photosynthesis. The intracytoplasmic membranes that harbor the photosynthetic apparatus can be small finger-like intrusions, vesicles, or various types of lamellae. In general, purple non-sulfur bacteria grow preferentially photoheterotrophically under anoxic conditions in the light. Mixotrophic growth is common during which fermentation products of other bacteria are used as the preferred carbon source. Some species even lack the ability to grow photolithoautotrophically (Imhoff *et al.*, 2005). Most purple non-sulfur bacteria are metabolically very flexible and can switch between anoxygenic photosynthesis, fermentation, and aerobic or anaerobic respiration using nitrate, dimethyl sulfoxide, or trimethylamine-N-oxide as electron acceptor. Many species can grow photoautotrophically with hydrogen or sulfide as electron donor. Many of these organisms do not oxidize sulfide completely to sulfate but form sulfur globules as the terminal sulfur product. Purple non-sulfur bacteria are widely distributed in nature as indicated by their isolation from very diverse sources including pond water, marine coastal sediments, sewage, swine waste lagoons, and even earthworm droppings (Larimer *et al.*, 2004). The photosynthetic RC of purple non-sulfur bacteria is type II and, if present, CO<sub>2</sub> fixation occurs by the Calvin cycle.

to interest in their physiological diversity and their use as model organisms, genome sequence information has been generated for several purple non-sulfur bacteria, including strains of *Rhodobacter capsulatus* (SB1003), *Rhodobacter sphaeroides* (2.4.1, ATCC 17025, ATCC 17029), *Rhodospirillum rubrum* (ATCC 11170<sup>T</sup>), *Rubrivivax gelatinosus* (IL144, PM1), and *Rhodopseudomonas palustris* (BisA53, BisB18, BisB5, CGA009, HaA2, TIE-1).

### 2.4. Aerobic Bacteriochlorophyll-Containing Bacteria

The ABC bacteria are a relatively recently discovered group of phototrophic purple bacteria whose physiology and ecology is not completely understood (*Hiraishi and Shimada*, 2001). All ABC bacteria are capable of synthesizing BChl *a* under aerobic conditions and appear to be capable of photosynthetic growth only under aerobic conditions. The content of BChl in the cells is generally low and the BChl synthesis is strongly inhibited by light. The BChl found in most of these bacteria is BChl *a* esterified with phytol as in other purple bacteria. However, *Acidiphilium* species produce zinc-chelated BChl *a* as the major component. All ABC bacteria studied so far contain an active photosynthetic RC and LH1 light-harvesting complexes with BChl *a*. Peripheral LH2-like antennae are absent in most species. All ABC bacteria share aerobic chemoorganotrophy as the preferred mode of growth (*Yurkov*, 2006). While they are able to gain energy from their photosynthetic machinery, they do not grow solely at the expense of light energy. None of the characterized ABC bacteria are able to grow photolithotrophically with sulfur compounds as electron donors.

Until recently, all known species of ABC bacteria were believed to belong to the Alphaproteobacteria, except *Roseotales depolymerans* which belongs to the Betaproteobacteria. ABC bacteria do not form a homogeneous cluster within the Alphaproteobacteria but are closely interspersed with phototrophic and non-phototrophic species (*Imhoff and Hiraishi*, 2005). The work of *Fuchs et al.* (2007) was the first to unequivocally prove an ABC bacterium (*Congregibacter litoralis* KT71) among the Gammaproteobacteria. A few strains of ABC bacteria belonging to the *Roseobacter* and *Erythrobacter* genera have had their genome sequenced. In addition, the genome sequence of *Congregibacter litoralis* KT71 is nearly complete (*Fuchs et al.*, 2007). These sequences may help explain the physiology of this group of bacteria.

### 2.5. Filamentous Anoxygenic Phototrophs

The FAP bacteria belong to one of the deepest branching bacterial phyla (*Chloroflexi*) of the *Bacteria*. This phylum contains both phototrophic and
non-phototrophic members, characterized by a filamentous morphology and gliding motility. Most but not all FAP bacteria are facultatively aerobic and preferentially utilize organic substrates in their phototrophic or chemotrophic metabolism. Chloroflexus, Chloronema, Oscillochloris, Chlorothrix, Roseiflexus, and Heliothrix are well-described genera, although no species of Chloronema, Chlorothrix, and Heliothrix have been characterized in pure culture (Hanada and Pierson, 2002). The FAP bacteria have taxonomically been organized in the family Chloroflexaceae (Garrity and Holt, 2001b) although it has been suggested that all the above-mentioned genera other than Chloroflexus should be organized into new families (Tourova et al., 2006). All FAP bacteria contain BChl $a$ as a component of the light-harvesting antenna and a type II photosynthetic RC. In addition, four of the genera Chloroflexus, Chloronema, Oscillochloris, and Chlorothrix contain chlorosomes that often cause the cells to appear green (thus, the name green filamentous bacteria or “green FAPs” has been used to describe these organisms). Chlorosomes are large light-harvesting antenna structures that are attached to the cytoplasmic membrane and contain BChl $c$ or $d$ (Frigaard and Bryant, 2006). The only other organisms that contain chlorosomes are GSB and the phototrophic acidobacteria (see introduction to Section 2). FAP bacteria that do not contain chlorosomes have an orange, red, or pink coloration, and hence sometimes are called “red FAPs.” In Chloroflexus species, $\mathrm{CO}_{2}$ is fixed by the 3-hydroxypropionate cycle (Ivanovskii et al., 1993) while the Calvin cycle appears to be used in Oscillochloris trichoides DG-6 (Ivanovsky et al., 1999). The genome sequences of the following FAP strains have been, or are in the process of being, determined: Chloroflexus aggregans DSMZ 9485<sup>T</sup>, Chloroflexus aurantiacus J-10-fl (DSMZ 635<sup>T</sup>), Chloronema giganteum UdG9001, Oscillochloris sp. UdG9002, Roseiflexus castenholzii DSMZ 13941<sup>T</sup>, Roseiflexus sp. RS–1, Heliothrix oregonensis, and “Candidatus Chlorothrix halophila” (www.genomesonline.org).

## 2.6. Heliobacteria

Heliobacteria occur in nature primarily in soil and in this regard they differ significantly in their ecology from purple and green phototrophs. Cells of heliobacteria stain Gram-negative, although the phylogenetic position of the group is together with typical Gram-positive bacteria. They belong to the Firmicutes phylum and group within the Bacillus/Clostridium lineage. Spores are formed, gas vesicles are not present, and their metabolism is strictly anaerobic. Heliobacteria can grow both photo- and chemotrophically. Photoheterotrophic growth occurs on a very limited number of organic
compounds as carbon sources. Chemotrophic growth in the dark occurs by fermentation of pyruvate. Photoautotrophic growth on CO₂/H₂ or CO₂/H₂S has not been demonstrated with any species of heliobacteria. However, some strains oxidize sulfide to elemental sulfur during photoheterotrophic growth (Bryantseva *et al.*, 2000).

Heliobacteria contain a unique photosynthetic pigment called BChl g, which contains a vinyl group on ring I and an ethylidene group on ring II. This BChl distinguishes heliobacteria from all other phototrophic bacteria (Madigan, 2001) and gives them their characteristic brown color (Gest and Favinger, 1983). The pigment system of heliobacteria is simple: it consists of a single pigment-protein complex that is embedded in the cytoplasmic membrane and contains both the type I reaction center and all BChls that have an antenna function. Heliobacteria lack differentiated photosynthetic internal membranes, such as the membrane vesicles or lamellae of purple bacteria or the chlorosomes of green bacteria (Neerken and Amesz, 2001). The only heliobacterium for which genome sequence information is available is *Heliobacterium modesticaldum* Ice1 (ATCC 51547) ([http://genomes.tgen.org/helio.html](http://genomes.tgen.org/helio.html)).

## 3. ANOXYGENIC PHOTOTROPHIC SULFUR BACTERIA: ECOLOGY

The various photosynthetic tetrapyrrole and carotenoid pigments in phototrophic bacteria lead to a distinct coloration of the cells ranging from green, brown, and red to purple. Older reports about colored waters or sediments occurring in various natural environments (Bavendamm, 1924; Engelmann, 1882, 1888; van Niel, 1931; Winogradsky, 1887) can nowadays be explained by massive blooms of anoxygenic phototrophic sulfur bacteria forming in the water column or in aquatic sediments under appropriate conditions when both sulfide and light are available and oxygen is deficient or lacking (Overmann, 2008). We know now that these phototrophic sulfur bacteria comprise the PSB and the GSB whose ecology and sulfur metabolism share many similarities (Brune, 1989, 1995b; Overmann, 2008; van Gemerden and Mas, 1995), although their phylogeny is rather different (see Section 2).

In natural environments light and sulfide occur in opposing gradients. Therefore, growth of phototrophic sulfur bacteria is restricted to a narrow zone. In open waters, like lakes or lagoons, stratification of oxic and anoxic water layers is maintained by density differences. In benthic environments,

gradients of sulfide are much steeper due to the higher rates of sulfate reduction and a lower frequency of turbulent mixing (Overmann, 2008). In some habitats of phototrophic sulfur bacteria, particularly in intertidal sediments, redox conditions change rapidly within hours. Certain species of PSB have adapted to these conditions and can switch to an aerobic chemolithotrophic growth mode and oxidize sulfide or thiosulfate or both with molecular oxygen. Under these conditions, the synthesis of pigments and pigment-binding proteins is repressed and the cells become colorless. In contrast, all GSB are obligately anaerobic and obligately phototrophic. These properties enforce a greater constraint on the possible niches available to GSB.

Phototrophic sulfur bacteria typically occur in non-thermal aquatic ecosystems, although a few moderately thermophilic members of these groups have been described from hot spring mats, for example, the GSB *Chlorobaculum tepidum* and the PSB *Thermochromatium tepidum* (Castenholz et al., 1990; Madigan, 1986). Among the PSB, most members of the *Chromatiaceae* are typically found in freshwater and marine environments, whereas the *Ectothiorhodospiraceae* inhabit hypersaline waters. Some members of the GSB have also been isolated from a hypersaline athallasso-haline lake in Spain (Vila et al., 2002). In coastal and most lacustrine waters, metabolically specialized *Chromatiaceae* dominate (van Gemerden and Mas, 1995). These organisms are obligately photolithotrophic, lack assimilatory sulfate reduction, cannot reduce nitrate, and assimilate only a few carbon sources. This is to be expected since metabolically versatile species of the *Chromatiaceae* have no strong selective advantage in most pelagic habitats (Overmann, 2008). In some lakes, the absorption of ultraviolet and blue light by humic substances is so significant that green-colored species of GSB have a selective advantage over their brown-colored counterparts and over PSB (Parkin and Brock, 1980).

In most phototrophic organisms, the response to very low light intensities is an increase in the size of the photosynthetic unit (Drews and Golecki, 1995; Overmann, 2008): In purple bacteria, the size of the photosynthetic antenna is in the range of 20–2000 BChl $a$ molecules per reaction center (Zuber and Cogdell, 1995). However, the specialized photosynthetic antenna organelles of GSB, the chlorosomes, are significantly larger than those of any other phototroph resulting in a ratio of about 5000–8000 BChl $c$ molecules per RC (Frigaard et al., 2003). The theoretical quantum requirement for the CO₂ fixation in PSB is 8–10.5 mol quanta/(mol CO₂), but only 3.5–4.5 mol quanta/(mol CO₂) for GSB (Brune, 1989). In general, GSB exhibit lower maintenance energy requirements and higher sulfide tolerance than PSB and are thereby especially well adapted to low-light habitats (Overmann, 2008).

118 NIELS-ULRIK FRIGAARD AND CHRISTIANE DAHL

# 4. TRANSFORMATIONS OF SULFUR COMPOUNDS

In the following section the sulfur oxidation capabilities of the various groups of anoxygenic phototrophic bacteria will be briefly described. PSB and GSB preferentially use reduced sulfur compounds as electron donors during photolithoautotrophic growth. Their sulfur-metabolizing capabilities are summarized in Table 2. Sulfur oxidation capabilities in the ABC bacteria, the FAP bacteria, and the heliobacteria are rather limited. Information about the enzymes involved in the bacteria is in most cases not available.

## 4.1. Purple Sulfur Bacteria

Although both PSB and GSB photooxidize sulfide, PSB prefer lower sulfide concentrations than GSB in general. However, the tolerance towards high sulfide concentrations is strain-specific. For example, among the PSB the ability to tolerate sulfide extends to a concentration of up to 11 mM, as has been shown for *Thiorhodococcus drewsii* (Zaar et al., 2003).

### 4.1.1. Chromatiaceae

Here we focus on those members of the Chromatiaceae that grow obligately or facultatively photoautotrophically. Closely related organisms may be non-phototrophic (e.g., *Arsukibacterium ikkense* (Schmidt et al., 2007) or members of the genus *Rheinheimera* (Brettar et al., 2002; Romanenko et al., 2003)). All phototrophic members of the Chromatiaceae use sulfide and sulfur of the oxidation state zero as photosynthetic electron donors (Table 2). Specialized species (e.g., *Allochromatium warmingii*, *Isochromatium buderi*, and *Thiospirillum jenense*) are unable to use thiosulfate, hydrogen, or organic compounds as electron donors (Imhoff et al., 1998). However, some of these species have never been tested for their ability to use thiosulfate and sulfite. A range of versatile species uses several different sulfur compounds including thiosulfate or sulfite. Those members of the Chromatiaceae that have been studied with respect to the utilization of externally added polysulfides (e.g., *Alc. vinosum* and *Tca. roseopersicina*) readily use these compounds as photosynthetic electron donors (Steudel et al., 1990; van Gemerden, 1987; Visscher et al., 1990). Utilization of polysulfides is not surprising because polysulfides are found in the growth medium of *Alc. vinosum* as intermediates in the oxidation of sulfide to internally stored sulfur (Prange et al., 2004).

| Genus or species | Sulfur substrates used | Intermediates on | End product | Sulfate | Chemoautotrophic |
| --- | --- | --- | --- | --- | --- |
| Green Sulfur Bacteria (Chlorobiaceae) |  |  |  | assimilation | growth |
| Aneclochloris perflieviia,b | Sulfide | Sulfur, EC | Sulfate |  |  |
| Chlorobaculum chlorovibrioides | Sulfide, sulfur | Sulfur, EC | Sulfate |  |  |
| Chlorobaculum limnaeum | Sulfide, sulfur, thiosulfate in strain 1549 | Sulfur, EC | Sulfate |  |  |
| Chlorobaculum parvum | Sulfide, thiosulfate, sulfur, (tetrathionate in strain NCIB 8346) | Sulfur, EC | Sulfate |  |  |
| Chlorobaculum tepidum | Sulfide, thiosulfate, sulfur | Sulfur, EC | Sulfate |  |  |
| Chlorobaculum chlorofatiphilum | Sulfide, thiosulfate, sulfur | Sulfur, EC | Sulfate |  |  |
| Chlorobium chlorochromatii | Sulfide | nd | nd | + |  |
| Chlorobium clathratiforme | Sulfide, thiosulfate in strain DSM 5477T, sulfur | Sulfur, EC | Sulfate |  |  |
| Chlorobium ferrooxidans | Sulfide, thiosulfate only in strains 1630, 9330 and DSM 257, sulfur | Sulfur, EC | Sulfate |  |  |
| Chlorobium limicola | Sulfide, sulfur | Sulfur, EC | Sulfate |  |  |
| Chlorobium luteolum | Sulfide, sulfur | Sulfur, EC | Sulfate |  |  |
| Chlorobium phaeobacteroides | Sulfide, sulfur, thiosulfate in strain DSM 265 | Sulfur, EC | Sulfate |  |  |
| Chloroherpeton thalassium | Sulfide | Sulfur, EC | Sulfate |  |  |
| Prosthecochloris aestuarii | Sulfide, sulfur | Sulfur, EC | Sulfate |  |  |
| Prosthecochloris vibrioformis | Sulfide, sulfur | Sulfur, EC | Sulfate |  |  |

| Genus or species | Sulfur substrates used | Intermediates on sulfide | End product | Sulfate assimilation | Chemoautotrophic growth |
| --- | --- | --- | --- | --- | --- |
| Purple Sulfur Bacteria<br>Chromatiaceae |  |  |  |  |  |
| Allochromatium | Sulfide, sulfur, thiosulfate, sulfite (latter two not in *Alc. warmingii*) | Sulfur, IC | Sulfate | + (not in *Alc.* warmingii) | Some species |
| Chromatium | Sulfide, sulfur | Sulfur, IC | Sulfate | — | + (sulfide, thiosulfate) |
| Halochromatium | Sulfide, thiosulfate, sulfur, sulfite | Sulfur, IC | Sulfate | — | +/— |
| Isochromatium | Sulfide, sulfur | Sulfur, IC | Sulfate | — | +/— |
| Lamprocystis | Sulfide, thiosulfate, sulfur | Sulfur, IC | Sulfate | $S^0$ and sulfate | +/— |
| Marichromatium | Sulfide, thiosulfate, sulfur, sulfite (only *Mch. gracile*) | Sulfur, IC | Sulfate | —/nd | +/— |
| Rhodochromatium | Sulfide, thiosulfate, sulfur | Sulfur, IC | Sulfate | nd | — |
| Thermochromatium | Sulfide, sulfur | Sulfur, IC | Sulfate | nd | — |
| Thioalkalivoccus | Sulfide, sulfur | Sulfur, IC | Sulfate | nd | — |
| Thiobaca | Sulfide | Sulfur, IC | Sulfate | nd | nd |
| Thiocapsa | Sulfide, thiosulfate, sulfur, sulfite (only *Tca. litoralis* and *Tca. pendens*) | Sulfur, IC | Sulfate | +/— | +/— |
| Thiococcus | Sulfide, sulfur | Sulfur, IC | Sulfate | — | — |
| Thiocystis | Sulfide, thiosulfate (not in *Tcs. gelatinosa*), sulfur, sulfite in some strains | Sulfur, IC | Sulfate | + in some strains | + |

| Thiodictyon<sup>a</sup> | Sulfide, sulfur | Sulfur, IC | Sulfate |
| --- | --- | --- | --- |
| Thiolfaviococcus | Sulfide, sulfur | Sulfur, IC | Sulfate |
| Thiohalocapsa | Sulfide, thiosulfate, sulfur, sulfite | Sulfur, IC | Sulfate |
| Thiolamprovum | Sulfide, thiosulfate, sulfur | Sulfur, IC | Sulfate |
| Thiopedia<sup>a</sup> | Sulfide, sulfur | Sulfur, IC | Sulfate |
| Thorhodococcus | Sulfide, thiosulfate, sulfur | Sulfur, IC | Sulfate |
| Thorhodovibrio | Sulfide, sulfur | Sulfur, IC | Sulfate |
| Thiospirillum<sup>a</sup> | Sulfide, sulfur | Sulfur, IC | Sulfate |
| Ectothiorhodospiraceae |  |  |  |
| Ectothiorhodospinus<sup>d</sup> | Sulfide, thiosulfate | Sulfur, EC | Sulfate |
| Ectothiorhodospira | Sulfide, thiosulfate (not in Ect. marismortui), sulfur, sulfite (nd for some species) | Polysulfide, sulfur, EC | Sulfate |
| Halorhodospira | Sulfide, thiosulfate, sulfur only in Hlr. halophila | Sulfur, EC | Sulfur or sulfate + in Hlr. halochloris |
| Thorhodospira | Sulfide, sulfur | Sulfur, EC & IC | Sulfate |

The tabulated data were mostly taken from Overmann (2001); Imhoff (2003, 2005a, 2005b, 2008). Additional information was taken from Raymond and Sistrom (1969), Khanna and Nicholas (1982), Rees et al. (2003); Zaar et al. (2002); Gorlenko et al. (2004); Arunasi et al. (2005); Vogl et al. (2006).

IC, intracellular; EC, extracellular; nd, not determined.

<sup>a</sup> These genera have no clear phylogenetic standing because 16S rDNA sequences are not available.

<sup>b</sup> Type strain not available.

<sup>c</sup> Sulfate assimilation genes are present in strain DSMZ 273 but sulfate assimilation has not been investigated.

<sup>d</sup> The genus Ectothiorhodospinus has no standing in nomenclature.

Some organic sulfur compounds can also serve as electron donors for photosynthetic growth of Chromatiaceae: *Thiocapsa roseopersicina* splits mercaptomalate to fumarate and sulfide and mercaptopropionate to acrylate and sulfide and then uses the formed sulfide as electron donor (Visscher and Taylor, 1993). This organism furthermore oxidizes dimethyl sulfide to dimethyl sulfoxide (Visscher and van Gemerden, 1991). During fermentative dark metabolism of Chromatiaceae, sulfur compounds (elemental sulfur) can serve as acceptors of electrons liberated by the oxidation of stored carbon compounds (polyhydroxyalcanoic acid).

### 4.1.2. Ectothiorhodospiraceae

When PSB of the family Ectothiorhodospiraceae grow on sulfide as electron donor, extracellular sulfur globules are usually formed as intermediates (biphasic growth). Under the alkaline growth conditions which are optimal for Ectothiorhodospiraceae species, polysulfides are stable intermediates during oxidation of sulfide to sulfur (Then and Trüper, 1983, 1984).

Ectothiorhodospira species are generally able to oxidize sulfide and sulfur (Table 2). When grown on elemental sulfur, *Ect.* halochloris does not oxidize this compound to sulfate, but reduces it to sulfide and polysulfide (Then and Trüper, 1984). With the exception of *Ect.* marismortui (Oren et al., 1989), thiosulfate is used by all species of the genus Ectothiorhodospira. On thiosulfate, sulfur globules are not detectable as an intermediate in all cases. In *Ect.* shaposhnikovii, for instance, sulfur is only formed from sulfide but not with thiosulfate as electron donor (Kusche, 1985). In other cases the ability to form sulfur globules from thiosulfate is not explicitly mentioned, but also not excluded (Trüper, 1968). In this context it is interesting to note that, in *Ect.* shaposhnikovii, thiosulfate oxidation does not stimulate growth, but is accompanied by the formation of tetrathionate and, later, higher polythionates (Gogtova and Vainstein, 1981). The ability to grow on sulfite has not been determined for all species, but appears to be widespread in the genus Ectothiorhodospira (Imhoff, 2005b).

Members of the genus Halorhodospira oxidize sulfide to sulfur which is further oxidized to sulfate by some species. Thiosulfate is only used by *Hlr.* halophila (Raymond and Sistrom, 1969) and poorly by *Halorhodospira neutrophila* (Hirschler-Rea et al., 2003). Sulfur can also be used by some species (e.g., *Hlr.* halophila). The most restricted sulfur-oxidation capacities are observed in *Hlr.* halochloris and *Hlr.* abdelmalekii. Both species do not appear to be able to oxidize sulfur to sulfate (Then and Trüper, 1983, 1984). *Thiorhodospira sibirica* uses sulfide and sulfur, but not thiosulfate as electron donors for photoautotrophic growth (Bryantseva et al., 1999). The final

oxidation product is sulfate (Table 2). *Ectothiorhodospinus mongolicum* grows preferentially as a photoorganoheterotroph, but is able to grow photolithotrophically with sulfide. Thiosulfate is only used in the presence of organic matter. The final oxidation product is sulfate (*Gorlenko et al.*, 2004).

## 4.2. Green Sulfur Bacteria

Five genera with a total of 16 species of GSB are currently recognized (*Imhoff*, 2003, 2008; *Vogl et al.*, 2006). GSB exhibit little variation in their ability to oxidize sulfur compounds and almost all GSB are capable of oxidizing sulfide and elemental sulfur to sulfate (Table 2). One exception is *Chlorobium ferrooxidans* for which only Fe²⁺ and H₂ have been described as photosynthetic electron donors (*Heising et al.*, 1999). *Chl. ferrooxidans* does not grow on sulfide, thiosulfate, or elemental sulfur, and is even inhibited by 1 mM sulfide when grown on H₂. However, in general, GSB have a high affinity for sulfide, and sulfide is usually the preferred substrate even if other sulfur substrates are available (*Brune*, 1989, 1995b). Initially, sulfide is typically incompletely oxidized to elemental sulfur, which is deposited extracellularly as sulfur globules (Fig. 1B). These sulfur globules are oxidized completely to sulfate when the sulfide has been consumed.

Some strains belonging to the genera *Chlorobium* and *Chlorobaculum* oxidize thiosulfate for growth (*Imhoff*, 2003). No *Prosthecochloris* species have been reported to utilize thiosulfate. *Chlorobaculum parvum* NCIB 8346 (formerly *Chlorobium vibrioforme* subsp. *thiosulfatophilum*; *Imhoff*, 2003) and a strain described by Helge Larsen as *Chlorobium thiosulfatophilum* can use tetrathionate as electron donor (*Khanna and Nicholas*, 1982; *Larsen*, 1952). Sulfite utilization has not yet been described for any GSB.

## 4.3. Purple Non-Sulfur Bacteria

In general, purple non-sulfur bacteria are to a much lesser extent capable of tolerating and using toxic sulfur compounds such as sulfide than the PSB. However, a number of purple non-sulfur bacteria are able to grow photolithoautotrophically with reduced sulfur compounds (*Brune*, 1995b; *Hansen and van Gemerden*, 1972; *Sander and Dahl*, 2008).

Phototrophic Betaproteobacteria (found among the Rhodocyclales and the Burkholderiales) have not been reported to use reduced sulfur compounds as electron donors and sulfide inhibits growth even at low concentrations

(Bruné, 1995b; Imhoff *et al.*, 2005). Nevertheless, *sox* genes are present in the genome of *Rubrivivax gelatinosus* (Sander and Dahl, 2008) indicating a potential to oxidize thiosulfate to sulfate.

Within all three taxonomic groups of the alphaproteobacterial purple non-sulfur bacteria (*Rhodospirillales*, *Rhizobiales*, *Rhodobacterales*), species are found that are able to use reduced sulfur compounds. These organisms vary considerably with respect to intermediates and final products of sulfur compound oxidation. In many species (e.g., *Rhodobacter capsulatus*, *Rhodospirillum rubrum*) sulfur formed outside the cells is the end product of sulfide oxidation. However, in others, among them the species of the genera *Rhodovulum*, *Rhodopseudomonas palustris*, or *Blastochloris sulfoviridis*, sulfate is the end product of sulfide oxidation (reviewed in Bruné, 1995b; Imhoff *et al.*, 2005; Sander and Dahl, 2008). *Rhodomicrobium vanniellii* oxidizes sulfide to tetrathionate while *Rhodobium marinum* oxidizes it to sulfur and thiosulfate when grown photomixotrophically (Imhoff, 1983). Thiosulfate is also used by many species and oxidized either to tetrathionate (*Rhodopila globiformis*; Then and Trüper, 1981) or completely to sulfate (e.g. *Rhodovulum* species; Appia-Ayme *et al.*, 2001; Bruné, 1995b; Imhoff *et al.*, 2005). Some species, for example, *Rhodoplanes roseus* and *Rpl. elegans*, use only thiosulfate but not sulfide as electron donors (Hiraishi and Ueda, 1994). Sulfur is also used as a substrate by some species, for example, *Rhodopseudomonas julia* and most species of the genus *Rhodovulum* (Imhoff *et al.*, 2005). None of the purple non-sulfur bacteria are able to use sulfite as the sole electron donor. For more detailed information on sulfur metabolism in purple non-sulfur bacteria, the reader is referred to a recent review on sulfur oxidation in purple bacteria (Sander and Dahl, 2008).

### 4.4. Aerobic Bacteriochlorophyll-Containing Bacteria

In general, ABC bacteria cannot grow photolithoautotrophically on reduced sulfur compounds. However, several representatives of this group can oxidize inorganic sulfur compounds. One example is *Roseinatronobacter thiooxidans*, a strictly aerobic, obligately heterotrophic alkaliphile that can oxidize sulfide, thiosulfate, sulfite, and elemental sulfur to sulfate in the presence of organic compounds. This bacterium appears to be a sulfur-oxidizing lithoheterotroph as the efficiency of organic carbon utilization increases notably in the presence of thiosulfate (Sorokin *et al.*, 2000). Thiosulfate-oxidizing activity was also shown in *Erythromicrobium hydrolyticum*, strain E4(1), and *Roseococcus thiosulfatophilus*, strain RB-7, in the presence of organic carbon substrate and aeration (Yurkov *et al.*, 1994).

genomes of all currently genome-sequenced ABC bacteria contain the genes $soxB$, $soxAX$, $soxYZ$, and $soxCD$ encoding for a multienzyme complex that has been well characterized biochemically from *Paracoccus pantotrophus* (Friedrich et al., 2005). In *Pcs. pantotrophus* the encoded Sox proteins catalyze the oxidation of thiosulfate to sulfate. Taken together these findings indicate that many ABC bacteria are able to oxidize thiosulfate (Sander and Dahl, 2008). While virtually nothing is known about metabolism of inorganic sulfur compounds in these organisms, it is well established that members of the *Roseobacter* group are able to degrade dimethylsulfonio-propionate (DMSP), an organic sulfur compound produced in abundance by marine algae. Some isolates have been reported to carry out the major DMSP transformations that have been observed in natural bacterial communities. Among these are two competing pathways for DMSP degradation and a pathway for incorporation of the sulfur moiety of DMSP into bacterial protein (Gonzalez et al., 2003; Moran et al., 2003). In contrast to sulfide and thiosulfate, DMSP and dimethyl sulfide are abundant in aerobic marine environments (Liss, 1999).

On a biochemical level, sulfur-dependent chemolithotrophy and oxidative sulfur metabolism have best been described in *Acidiphilium acidophilum* (formerly *Thiobacillus acidophilus*) (Hiraishi et al., 1995; Meulenberg et al., 1992b; Pronk et al., 1990). In this organism, the utilization of thiosulfate is initiated by the oxidative condensation of two molecules of thiosulfate yielding tetrathionate. This step is catalyzed by the periplasmic thiosulfate:cytochrome $c$ oxidoreductase (Meulenberg et al., 1993). Furthermore, evidence was obtained that tetrathionate oxidation also takes place in the periplasm (Meulenberg et al., 1993). In addition, a tetrathionate hydrolase (de Jong et al., 1997), a trithionate hydrolase (Meulenberg et al., 1992a) and a sulfite:cytochrome $c$ oxidoreductase (de Jong et al., 2000) have been characterized from the organism.

### 4.5. Filamentous Anoxygenic Phototrophs

Slow photoautotrophic growth with hydrogen or sulfide as electron donor has been observed in some strains of *Chloroflexus aurantiacus* (Holo and Sirevag, 1986; Madigan and Brock, 1975, 1977) and *Oscillochloris trichoides* (Keppen et al., 2000), but not in *Chloroflexus aggregans* (Hanada et al., 1995). When sulfide is oxidized by *Chloroflexus* or *Oscillochloris* cultures, it is deposited as elemental sulfur outside the cells, often affixed to the cells. Sulfide-dependent photoautotrophic growth has also been observed in natural populations of *Chloroflexus* (Giovannoni et al., 1987; Ward et al.,

126 NIELS-ULRIK FRIGAARD AND CHRISTIANE DAHL

1998). A survey of four strains of *Oscillochloris trichoides* showed that the preferred mode of growth was photolithoheterotrophic, although they also grew photoautotrophically with sulfide as electron donor (Keppen *et al.*, 2000). All of the *Oscillochloris trichoides* stains tolerated sulfide up to about 4 mM and oxidized sulfide only to elemental sulfur. Thus, although sulfide is not the preferred substrate for photosynthetic cultivation of FAPs under laboratory conditions (Garrity and Holt, 2001b), sulfide oxidation could be important in certain natural environments. The natural habitats of *Chloronema*, *Heliothrix*, and *Roseiflexus* species contain no or very low concentrations of sulfide (Garrity and Holt, 2001b; Pierson *et al.*, 1985).

### 4.6. Heliobacteria

Heliobacteria cannot grow phototrophically using sulfide as the sole electron donor, but if added, sulfide is usually oxidized to elemental sulfur (Bryantseva *et al.*, 2000; Madigan, 2001). *Heliobacterium sulfidophilum* and *Heliobacterium undosum* are especially tolerant to sulfide (up to 2 mM at pH 7.5) (Bryantseva *et al.*, 2000), whereas the sulfate-fixing *Heliophilum fasciatum* is inhibited by sulfide concentrations above 0.1 mM (Ormerod *et al.*, 1996). Some strains perform assimilatory sulfate reduction, whereas others require a reduced sulfur compound (e.g., thiosulfate, sulfide, methionine, or cysteine) for biosynthetic purposes. The only heliobacterium for which genome sequence information is available, *Heliobacterium modesticaldum* Ice1 (ATCC 51547<sup>T</sup>) ([http://genomes.tgen.org/helio.html](http://genomes.tgen.org/helio.html)), cannot grow on sulfate as the sole sulfur source but requires thiosulfate, sulfide, methionine, or cysteine (Kimble *et al.*, 1995).

## 5. OXIDATIVE SULFUR METABOLISM: ENZYMES AND GENES

Here, we attempt to describe the enzymes or multienzyme systems involved in sulfur compound oxidation in phototrophic sulfur bacteria and summarize information on their occurrence. On a molecular genetic and biochemical level, sulfur oxidation is best characterized in the PSB *Alc. vinosum* and in the GSB *Cba. tepidum*. However, our knowledge of these processes is still rather incomplete. An overview of the currently proposed metabolic network is shown in Fig. 2. This figure is based on a combination of biochemical evidence and genome sequence analyses.

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 127

Formation of sulfur globules occurs  
in the periplasm or outside the  
cell depending on the organism  

![Diagram](attachment:diagram.png)

Figure 2 Overview of proposed pathways of thiotrophic sulfur transformations in phototrophic sulfur bacteria (PSB and GSB). Probably no single organism has all the reactions shown here. In the periplasm, the polysulfur chains are probably very short ($n$ probably around 3 or 4), whereas the polysulfur chains in the sulfur globules can be very long ($n > 3$ and possibly up to $n > 10^5$ as for polymeric sulfur (Prange et al., 2002a; Dahl and Prange, 2006)). Transport into the periplasm of sulfite formed by cytoplasmic Dsr proteins and oxidation of sulfite to sulfate in the periplasm as previously suggested (Dahl and Prange, 2006; Dahl, 2008) cannot be completely excluded.

Many enzymes potentially involved in sulfur metabolism can readily be identified in the genome sequences by sequence homology with known enzymes. Tables 3 and 4 show a detailed survey of genes known to be involved in or potentially involved in the oxidative sulfur metabolism in the genome-sequenced strains of phototrophic sulfur bacteria. These tables show that PSB and GSB share a number of genes potentially involved in the oxidation of reduced sulfur compounds: For example, genes for the sulfide-oxidizing enzyme flavocytochrome $c$ (FccAB) and sulfide:quinone oxidoreductase (SQR) occur in most PSB and GSB. With only two exceptions, the $dsr$ genes including $dsrAB$ for dissimilatory sulfite reductase are present. The Sox system is involved in oxidation of thiosulfate and can account for this activity observed in all strains capable of oxidizing thiosulfate.

Table 3 Occurrence of genes related to dissimilatory sulfur metabolism in genome-sequenced phototrophic sulfur bacteria${}^{a}$

| Organism | $sqr$ | $fccAB$ | $sox$ | $sgp$ | $dsr$ | $sat$ | $aprBA$ | $qmo$ | $sorAB$ |
|----------|-------|---------|-------|-------|-------|-------|---------|-------|---------|
| **Green Sulfur Bacteria** |  |  |  |  |  |  |  |  |  |
| Chlorobaculum parvum DSMZ 263T | + | + | + | + | + | + | + | + | + |
| Chlorobaculum tepidum TLS (DSMZ 12025T) | + | + | + | + | + | + | + | + | + |
| Chlorobium chlorochromatii CaD3 | + | + | + | + | + | + | + | + | + |
| Chlorobium clathratiforme BU-1 (DSMZ 5477T) | + | + | + | + | + | + | + | + | + |
| Chlorobium ferrooxidans DSM 13031T | + | + | + | + | + | + | + | + | + |
| Chlorobium limicola DSMZ 245T | + | + | + | + | + | + | + | + | + |
| Chlorobium luteolum DSMZ 273 | + | + | + | + | + | + | + | + | + |
| Chlorobium phaeobacteroides DSMZ 266T | + | + | + | + | + | + | + | + | + |
| Chloroherpeton thalassium ATCC 35110T | + | + | + | + | + | + | + | + | + |
| Prosthecochloris aestuarii DSMZ 271T | + | + | + | + | + | + | + | + | + |
| Prosthecochloris sp. BS1 | + | + | + | + | + | + | + | + | + |
| **Purple Sulfur Bacteria** |  |  |  |  |  |  |  |  |  |
| Ectothiorhodospiraceae |  |  |  |  |  |  |  |  |  |
| Halorhodospira halophila SL1 (DSMZ 244T) | ? | 3 copies${}^{b}$ | + | - | + | + | + | + | ? |
| Chromatiaceae |  |  |  |  |  |  |  |  |  |
| Allochromatium vinosum DSMZ 180T | Enzyme activity | + | ? | + | + | + | + | + | - |

---

The following abbreviations designate occurrence of a core set of several genes: $dsr$: $dsrABC EFHLN MKJOP$, $qmo$: $qmoABC$, $sgp$: $sgpABC$, $sox$: $soxB X A Y Z$.

${}^{a}$ This survey is based on Frigaard and Bryant (2008) and additional genome sequence analyses, except for *Alc. vinosum* for which only limited sequence information is available. Genomes were analyzed by BLAST searches using the resources provided by Integrated Microbial Genomes (DOE Joint Genomes Institute, http://img.jgi.doe.gov) and GenBank (http://www.ncbi.nlm.nih.gov). Baits: FccAB from *Alc. vinosum* (AAA23316 and AAB86576); $SQR$ from *Rhodobacter capsulatus* (AAF64400 and AAF64401); Dsr proteins from *Alc. vinosum* (U84760), Sox proteins from *Paracoccus denitrificans* (CAA55824: SoxB, CAB94380: SoxY, CAB94381: SoxZ, CAA55827: SoxA, CAB94379: SoxxX); APS reductase (AprMBA) from *Alc. vinosum* (U84759), Qmo proteins from *Cba. tepidum* (QmoA: CT0866, QmoB: CT0867, QmoC: CT0868).

${}^{b}$ Relationships of the encoded proteins to FccAB higher in all cases than that to SoxEF, however similarity to SoxEF is still significant.

Table 4 Occurrence of genes possibly related to transformation of sulfur compounds and hydrogen in genome-sequenced phototrophic sulfur bacteria

| Organism | PSRLC1 | PSRLC2 | PSRLC3 | hdr-hyd | hup | rlp |
| --- | --- | --- | --- | --- | --- | --- |
| Green Sulfur Bacteria |  |  |  |  |  |  |
| Chlorobaculum parvum DSMZ 263<sup>T</sup> | + | + | + | + | + | + |
| Chlorobaculum tepidum TLS (DSMZ 12025<sup>T</sup>) | + | + | + | + | + | + |
| Chlorobium chlorochromatii CaD3 | + | + | + | + | + | + |
| Chlorobium clathratiforme BU-1 (DSMZ 5477<sup>T</sup>) | + | + | + | + | + | + |
| Chlorobium ferrooxidans DSMZ 13031<sup>T</sup> | + | + | + | + | + | + |
| Chlorobium limicola DSMZ 245<sup>T</sup> | + | + | + | + | + | + |
| Chlorobium luteolum DSMZ 273<sup>T</sup> | + | + | + | + | + | + |
| Chlorobium phaeo vibroides DSMZ 266<sup>T</sup> | + | + | + | + | + | + |
| Chloroherpeton thalassium ATCC 35110<sup>T</sup> | + | + | + | + | + | + |
| Prosthecochloris aestuarii DSMZ 271<sup>T</sup> | + | + | + | + | + | + |
| Purple Sulfur Bacteria |  |  |  |  |  |  |
| Ectothiorhodospiraceae |  |  |  |  |  |  |
| Halorhodospira halophila SL1 (DSMZ 244<sup>T</sup>) | – | b | – | b | – | – |

a The following abbreviations designate occurrence of a core set of several genes: hdr-hyd: hdrDA-orf1247-orf1248-hydB2G2, hup, hupSLCD.

b *Hlr*. halophila has three polysulfide-reductase-like complexes that are rather distantly related to the PSRLC of GSB and of which one (encoded by Hhal_1934, 1935 and 1936) presumably is cytoplasmic as is the PSRLC3 of GSB.

adenosine-5′-phosphosulfate reductase (Apr), sulfate adenylyltransferase (Sat), and either the quinone-interacting membrane-bound oxidoreductase (Qmo) complex or AprM probably constitute a sulfite-oxidizing system. A putative alternative sulfite-oxidizing system has been identified in the GSB strains that do not have the Apr, Sat, and Qmo (or AprM) enzymes.

Homologs of sulfhydrogenase and heterodisulfide reductase are also found in GSB (Table 4). Homologs of polysulfide reductase are present in GSB as well as in the genome-sequenced PSB *Hlr. halophila* (Table 4; Frigaard and Bryant, 2008a). However, functions cannot easily be assigned to these proteins in phototrophic sulfur bacteria because (1) they are too distantly related to characterized enzymes; (2) they are not distributed among the organisms in a manner that obviously correlates with known physiological traits; and (3) the sulfur compound and hydrogen oxidation properties of analyzed strains can be accounted for by other enzymes.

We will now first describe the oxidation of sulfide. The following sections are dedicated to our current knowledge on the properties of sulfur deposited by PSB and GSB and to the proteins involved in its further oxidation to sulfite. We will then describe the current state of affairs regarding sulfite oxidation to sulfate and finally describe the pathway of thiosulfate oxidation.

## 5.1. Oxidation of H₂S

Different enzymes are candidates for sulfide oxidation: sulfide:quinone oxidoreductase (SQR) (Schütz et al., 1997), a flavocytochrome *c* sulfide dehydrogenase (FccAB) (Brune, 1995b; Chen et al., 1994; Kostanjevecki et al., 2000; Meyer and Cusanovich, 2003), and also the Sox system (Appia-Ayme et al., 2001). The occurrence of the respective genes in GSB and PSB genomes is summarized in Table 3.

### 5.1.1. Flavocytochrome *c*

Flavocytochrome *c* is usually a soluble, periplasmic enzyme consisting of a large sulfide-binding FccB flavoprotein subunit (41–47 kDa) and a smaller FccA cytochrome *c* subunit (Brune, 1995b). FccA from *Alc. vinosum* is 21 kDa and binds two heme *c* (van Beeumen et al., 1991) while in the GSB FccA is only 10 kDa and binds a single heme. In GSB, the FccB subunit has high sequence similarity (approximately 50% amino acid sequence identity) to a flavoprotein (SoxJ) encoded in the *sox* gene cluster (Section 5.7.1). In *Ect. valuolata* and also in *Cba. thiosulfatiphilum* DSMZ 249 (formerly *Chl.*

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 131

limicola; Imhoff, 2003) the protein is membrane-bound (Kostanjevecki et al., 2000; Verté et al., 2002). In the latter strain, membrane attachment is possibly due to an unusual signal peptide in the FccA subunit that is not cleaved but supposedly anchors the protein in the cytoplasmic membrane. The predicted signal peptide in FccA from GSB is followed by a highly variable, 15- to 25-residue sequence, which is rich in alanine and proline and which is suggested to act as a flexible arm (Verté et al., 2002). FccAB is constitutively expressed in Alc. vinosum (Bartsch, 1978) and also in Cba. thiosulfatiphilum DSMZ 249 (Verté et al., 2002).

In vitro, flavocytochromes can efficiently catalyze electron transfer from sulfide to a variety of small $c$-type cytochromes (e.g., cytochrome $c_{550}$ from Alc. vinosum; Bosshard et al., 1986; Davidson et al., 1985) that may then donate electrons to the photosynthetic reaction center. However, the in vivo role of flavocytochrome $c$ is unclear for a number of reasons. (1) Although many sulfide-utilizing organisms produce flavocytochrome $c$, some sulfide-utilizing GSB and PSB do not, which clearly demonstrates that flavocytochrome $c$ is not essential for sulfide oxidation (Brune, 1995b). (2) Mutants of Alc. vinosum (Reinartz et al., 1998) and Cba. tepidum (S.O. Barbas and N.-U. Frigaard, unpublished) deficient in flavocytochrome $c$ exhibit sulfide oxidation rates similar to those of the wild type. (3) The pattern of sulfide oxidation and the concomitant formation of elemental sulfur that is subsequently oxidized to sulfate upon sulfide depletion, is similar in Chl. luteolum DSMZ 273${}^{\mathrm{T}}$, which does not contain flavocytochrome $c$, and in Chl. limicola DSMZ 245${}^{\mathrm{T}}$, which does contain flavocytochrome $c$ (Steinmetz and Fischer, 1982). If indeed the FccAB flavocytochrome $c$ oxidizes sulfide in vivo, both GSB and PSB apparently have alternative sulfide-oxidizing enzyme systems, possibly sulfide:quinone reductase (Section 5.1.2) that may be quantitatively more important. However, it is also possible that flavocytochrome $c$ is advantageous under certain growth conditions and that such conditions have not yet been identified in these bacteria. Based on the large difference of redox potential between flavocytochrome $c$ and the photosynthetic reaction center, Brune (1995b) suggested that flavocytochrome $c$ may represent a high affinity system for sulfide oxidation that might be of advantage for the cells especially at very low sulfide concentrations.

### 5.1.2. Sulfide:Quinone Oxidoreductase

As an alternative to sulfide oxidation via flavocytochrome $c$, the transfer of electrons from sulfide primarily into the quinone pool was proposed based on energetic considerations as well as on the inhibitory effect of rotenone,

CCCP, and antimycin A on NAD photoreduction by sulfide (Brune, 1989; Brune and Trüper, 1986). Indeed, sulfide:quinone reductase (SQR; EC 1.8.5.-) catalyzing oxidation of sulfide with an isoprenoid quinone as the electron acceptor has been found in both chemotrophic and phototrophic prokaryotes as well as in some mitochondria (Griesbeck *et al.*, 2000; Theissen *et al.*, 2003).

Membrane-bound SQR activity has been biochemically demonstrated in GSB and also PSB and presumably feeds electrons into the photosynthetic electron transfer chain via a quinol-oxidizing Rieske iron-sulfur protein/cytochrome *b* complex (Reinartz *et al.*, 1998; Shahak *et al.*, 1992). The genome sequences of all GSB strains encode one (CT0117 in *Cba.* tepidum TLS) or two homologs of the biochemically characterized SQRs from *Rhodobacter capsulatus* (CAA66112) and *Oscillatoria limnetica* (AAF72962) (Table 3). This includes *Chl.* ferrooxidans (ZP_01385816), which cannot grow on sulfide as the sole electron donor (Heising *et al.*, 1999). This organism may benefit from SQR activity as a supplement to its energy metabolism. It could also use SQR as a protective mechanism to remove sulfide, which prevents growth when present in high concentrations. The SQR homologs of GSB are flavoproteins with predicted masses of about 53 kDa, and each contains all three conserved cysteine residues that are essential for sulfide oxidation in *Rhodobacter capsulatus* SQR (Griesbeck *et al.*, 2002). The significance of SQR in GSB is not yet clear. An *sqr* mutant of *Cba.* tepidum exhibits a lower sulfide oxidation rate than the wild type, which demonstrates that although SQR participates in sulfide oxidation in this organism, an alternative sulfide oxidation mechanism exists (Ocón Barbas and Frigaard, unpublished).

Although *Alc.* vinosum membranes exhibit SQR activity, it has so far not been possible to detect a *sqr*-related gene via Southern hybridization with heterologous probes or heterologous PCR nor could the protein be detected with antibodies directed against the *Rba.* *capsulatus* protein (M. Reinartz and C. Dahl, unpublished). The enzyme from *Alc.* vinosum and possibly other PSB may have properties distinct from those of characterized SQRs. In general, the sequences of SQR proteins are not well conserved and it is possible that a high degree of sequence variation could account for the lack of detection. In accordance, the *Hlr.* halophila genome contains one only distantly related homolog (Hhal_1665) of the biochemically well-characterized SQR from *Rhodobacter capsulatus* (Griesbeck *et al.*, 2002; Schütz *et al.*, 1999).

In *Rba.* *capsulatus*, SQR is absolutely essential for the oxidation of sulfide (Schütz *et al.*, 1999). The enzyme is a peripherally membrane-bound flavoprotein with its active site located in the periplasm (Schütz *et al.*, 1997, 1999). Interestingly, a classical N-terminal signal peptide is missing and

co-transport with the products of neighboring genes was excluded by gene deletion. However, nuclease treatment showed that 38 carboxy-terminal amino acids are necessary for the translocation, indicating a hitherto unknown transport mechanism (Schütz et al., 1999).

The primary product of the SQR reaction is soluble polysulfide whereas elemental sulfur does not appear to be formed in vitro (Griesbeck et al., 2002). Very probably, disulfide (or possibly a longer chain polysulfide) is the initial product of sulfide oxidation, which is released from the enzyme. Polysulfide anions of different chain lengths are in equilibrium with each other and longer-chain polysulfides can be formed by disproportionation reactions from the initial disulfide (Steudel, 1996b). When whole cells of Rba. capsulatus grow with sulfide, elemental sulfur is formed as the final product. In principle, elemental sulfur can form spontaneously from polysulfides (Steudel, 1996b).

In experiments using isolated spheroplasts from Chl. vibrioforme and Alc. minutissimum, soluble polysulfides have also been detected as the product of sulfide oxidation (Blöthe and Fischer, 2000). Polysulfides were also detected as primary products of sulfide oxidation by whole cells of Alc. vinosum (Prange et al., 2004) and have been reported as intermediates of the oxidation of sulfide to extracellular sulfur by species of the PSB family Ectothiorhodospiraceae (Then and Trüper, 1983; Trüper, 1978). While transient formation of polysulfide by the latter organism species had originally been attributed to chemical reaction between H₂S and elemental sulfur promoted by the alkaline culture medium (Trüper, 1978), it now appears more likely that they present enzymatically generated intermediates.

Some GSB additionally contain distantly related homologs of SQR with no assigned or obvious function (here denoted SQRLP1 and SQRLP2 for SQR-like proteins type 1 and type 2). SQRLP1 is present in Cba. tepidum TLS (CT1087) and in three other GSB. Among the GSB, SQRLP2 is only present in Cba. tepidum TLS (CT0876). Interestingly, SQRLP2 clusters in phylogenetic analyses with proteins of unknown function from various archaea. In addition, SQRLP2 from Cba. tepidum TLS shares 54% amino acid sequence identity with a protein from Sulfitobacter strain NAS14.1, which is a marine Roseobacter-like strain that grows aerobically on dimethylsulfoniopropionate (https://research.veterinstitute.org/moore/). Otherwise, SQRLP2 has few homologs in the databases.

### 5.1.3. Sox Proteins and Sulfide Oxidation

In Rhodovulum sulfidophilum, a member of the purple non-sulfur bacterial family Rhodobacteraceae, the Sox enzyme system catalyzing the oxidation of

thiosulfate to sulfate (see Section 5.7.1), is also indispensable for the oxidation of sulfide *in vivo* (Appia-Ayme *et al.*, 2001). The same might well be the case for other purple non-sulfur bacteria containing *sox* genes. However, in *Alc. vinosum* mutants deficient of either flavocytochrome *c* (Reinartz *et al.*, 1998), *sox* genes or both (D. Hensen, B. Franz and C. Dahl, unpublished), sulfide oxidation proceeds with wild-type rates indicating that SQR plays the main role in sulfide oxidation in this organism. Occurrence of *sox* genes in GSB and PSB indicates a tight correlation with the ability to oxidize thiosulfate (Table 3).

### 5.1.4. Other Proposed Sulfide-Oxidizing Enzymes

In *Alc. vinosum*, a dissimilatory sulfite reductase (DsrAB) operating in reverse, that is, in the direction of sulfite formation, has also been discussed to be involved in sulfide oxidation (Schedel *et al.*, 1979). However, this protein is clearly not essential for sulfide oxidation but rather absolutely required for oxidation of intracellularly stored sulfur (Pott and Dahl, 1998). This finding is consistent with the occurrence of *dsr* genes exclusively in sulfur-storing PSB but not in purple non-sulfur bacteria (Sander and Dahl, 2008).

It should be noted that cytochromes without flavin groups have also been proposed to mediate electron transfer from sulfide to the reaction center in some PSB (Brune, 1989; Fischer, 1984; Leguijt, 1993).

### 5.2. Oxidation of Polysulfides

As outlined above, polysulfides appear to be the primary product of the oxidation of sulfide in a number of purple and green bacteria. Utilization of externally added polysulfides has been studied in *Alc. vinosum* and *Tca. roseopersicina*. Both organisms readily used these compounds as photosynthetic electron donors (Steudel *et al.*, 1990; van Gemerden, 1987; Visscher *et al.*, 1990). It is currently unknown how polysulfides are converted into sulfur globules. Theoretically this could be a purely chemical, spontaneous process as longer polysulfides are in equilibrium with elemental sulfur (Steudel *et al.*, 1990). However, we have shown that *Alc. vinosum* sulfur globules do not contain major amounts of sulfur rings but probably consist of long-chains of sulfur with organic residues at one or both ends (Prange *et al.*, 1999, 2002a). Such organylsulfanes must eventually be formed by an unknown (enzymatic) mechanism.

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 135

## 5.3. Uptake of External Sulfur

Very many GSB and PSB including *Alc. vinosum* are able to oxidize externally supplied solid, virtually insoluble, elemental sulfur. This is a challenging biochemical problem in that elemental sulfur is extremely hydrophobic and virtually insoluble in water. In addition, elemental sulfur cannot be attacked via oxygenation reactions in the absence of molecular oxygen under anoxic conditions. In PSB like *Alc. vinosum*, elemental sulfur is first taken up by the cells and intracellular sulfur globules are formed as an obligate intermediate before the sulfur stored in the globules is oxidized to the final product sulfate (Brune, 1995b; Dahl, 2008; Franz et al., 2007). In contrast, GSB and PSB of the family Ectothiorhodospiraceae do not form intracellular sulfur globules. Very little is known on the attachment and utilization of solid elemental sulfur by phototrophic sulfur bacteria. Neither uptake of elemental sulfur nor the transformations necessary for its oxidation under anoxic conditions are well understood. Enzymes catalyzing the uptake and oxidation of externally added $S^0$ have not yet been isolated from any phototrophs.

Utilization of solid elemental sulfur must include binding and/or activation of the sulfur as well as transport inside the cells. As pointed out by Chan et al. (2008), two different strategies would in principle be possible, analogous to the reduction of insoluble electron acceptors (e.g., metals) (Lloyd, 2003): physical contact of the cells to their insoluble substrate and direct electron transfer from the cell envelope to the substrate via outer membrane proteins (Myers and Myers, 2001) or excretion of reducing substances that can act on substrate distant from the cells (Hernandez et al., 2004). Conclusive evidence for one of these models has not yet been obtained for phototrophic sulfur bacteria.

### 5.3.1. Properties of Elemental Sulfur

“Elemental sulfur” (“$S^0$”) has the formal valency zero. It tends to catenate and form chains with various lengths ($S_\infty$ or $S_\mu$) and ring sizes ($S_n$) (Steudel, 2000; Steudel and Eckert, 2003). All sulfur allotropes are hydrophobic, not wetted by water and they hardly dissolve in water. Cyclic, orthorhombic $\alpha$-sulfur ($\alpha$-$S_8$) is the most stable form of elemental sulfur at ambient pressure and temperature (Roy and Trudinger, 1970; Steudel, 2000). Polymeric sulfur consists mainly of chain-like macromolecules (Steudel and Eckert, 2003). Customary in trade, typical elemental sulfur (“flowers of sulfur”) mainly consists of $S_8$ rings and traces of $S_7$ rings which are responsible for the yellow color (Steudel and
Holz, 1988). In addition, elemental sulfur sublimed at ambient pressure always contains some polymeric sulfur (Steudel and Eckert, 2003).

### 5.3.2. Adhesion to Elemental Sulfur

Some information is available about adhesion to extracellular sulfur by chemotrophic sulfur oxidizers and by GSB. It is well established that acidophilic chemotrophic sulfur-oxidizing bacteria like *Acidithiobacillus thiooxidans* or *Acidithiobacillus ferrooxidans* attach to sulfur by glycocalyx-like extracellular polymeric substances (EPS) (Espejo and Romero, 1987; Vogler and Umbreit, 1941), specifically lipopolysaccharides (Bryant et al., 1984; Gehrke et al., 1998). In general, EPS are a complex mixture of high molecular weight polymers (MW > 10,000). They significantly influence the surface physicochemical properties, which are of considerable importance in governing bacterial adhesion. The origin of EPS is very complex, and their components and content heavily depend on many factors such as the substrate. In phototrophic sulfur bacteria, the presence, compositions, and structures of EPS have not been systematically elucidated. Structures attached to the cell wall (the so-called “spinae”) have been postulated to mediate adhesion of the GSB *Chlorobaculum limnaeum* UdG 6038 (formerly *Chlorobium limicola*; Imhoff, 2003) to extracellularly deposited sulfur (Pibernat and Abella, 1996). The presence of a large capsule in *Cba. limnaeum* UdG 6038 concomitant with a high number of spinae was interpreted as the spinae acting as the frame for capsule stabilization. Concomitant with the frequent observation of large numbers of sulfur globules per cell by phase-contrast microscopy, a role for both spinae and capsule in the metabolism of external sulfur globules has been suggested (Pibernat and Abella, 1996). In *Atb. ferrooxidans*, Rojas et al. (1995) have demonstrated the role of a capsule in retaining sulfur colloids during pyrite oxidation as temporary energy reservoirs. In spined GSB, the presence of spinae together with the capsule favors the retention of the external sulfur globules physically attached to the outer membrane of the cell wall. Moreover, it has recently been shown for the PSB *Alc. vinosum* that intimate physical cell-sulfur contact is a prerequisite for uptake of elemental sulfur (Franz et al., 2007).

The surface properties of bacteria that have been found to affect adhesion to insoluble substrates like sulfur are cell surface hydrophobicity (Devasia et al., 1993; Gehrke et al., 1998; Takeuchi and Suzuki, 1997) and
electrokinetic potential (Blake et al., 1994). Atb. ferrooxidans cells grown on sulfur or pyrite exhibited greater hydrophobicity than cells grown on soluble ferrous iron (Devasia et al., 1993). In a different study, sulfur-grown cells of Atb. ferrooxidans were found to exhibit purely hydrophobic surface properties and did not attach to charged particles such as pyrite. EPS from sulfur-grown cells contained more lipids, more free fatty acids, and a much higher ratio of uncharged (glucose) versus charged sugars (glucuronic acid) than EPS from pyrite-grown cells. It was concluded that attachment to hydrophobic substrates such as sulfur is dominated by van der Waal’s attraction forces (Gehrke et al., 1998). Measurements of zeta potentials of Atb. ferrooxidans cells showed that net surface charge was such as to minimize charge repulsive forces in the organism’s interaction with any substrate surface. When grown on elemental sulfur, washed cells were close to their isoelectric point (with a slight tendency to the negative), while they were negatively charged when grown on pyrite or ferrous iron (Blake et al., 1994). Besides hydrophobic interactions, hydrophilic interactions have also been implicated in cell-sulfur contact of Atb. ferrooxidans (Ohmura et al., 1993; Takeuchi and Suzuki, 1997).

### 5.3.3. Activation and Uptake of Elemental Sulfur

In all cases so far, a reaction activating elemental sulfur prior to its oxidation is postulated, due to the stability and low water solubility of the substrate. In the case of cyclo-octasulfur, this activation reaction could be an opening of the S₈ ring by nucleophilic reagents, resulting in the formation of linear inorganic or organic polysulfanes. In addition, the reduction of elemental sulfur to water-soluble sulfide has been discussed. Both reactions could be carried out by thiol groups of cysteine residues. Along this line, it was proposed for Acidithiobacillus and Acidiphilium that extracellular elemental sulfur is mobilized by thiol groups of special outer membrane proteins and transported into the periplasmic space as persulfide sulfur (Rohwerder and Sand, 2003). Experimental evidence for the existence of two different outer membrane proteins (OMPs) involved in cell-sulfur adhesion and sulfur uptake in Acidithiobacillus has been obtained by Buonfiglio et al. (1999) and Ramírez et al. (2004). In this respect it might be interesting to note that the complete genome sequences of 10 GSB able to oxidize externally added elemental sulfur contain a gene related to that for the sulfur-induced OMP of Atb. ferrooxidans identified by Ramírez et al. (2004) (CT2230 in Cba. tepidum). In several of these GSB the gene is located in the
vicinity of other genes encoding enzymes of sulfur metabolism (subunits of potential polysulfide reductases, dissimilatory sulfite reductase ($dsr$) operon). In *Chlorobium clathratiforme* BU1 (DSMZ 5477<sup>T</sup>) the gene is even part of the $dsr$ operon. Most conspicuously, the gene is absent in *Chlorobium ferrooxidans* that cannot oxidize reduced sulfur compounds. A closely related gene is not present in the genome of *Hlr.* *halophila*. However, a different gene encoding a potential outer membrane porin (Hhal_1947) is found in the sulfur gene cluster of *Hlr.* *halophila* (*Dahl*, 2008) where it is situated immediately upstream of genes encoding a potential flavocytochrome $c$.

### 5.3.4. Which Species of Elemental Sulfur are Metabolized?

The bonding energy between S–S bonds in polymeric sulfur is 2.4 kJ/mol weaker than in cyclo-octasulfur (*Steudel*, 1985, 1996a; *Steudel and Eckert*, 2003). Therefore, chain-like sulfur (polymeric sulfur) might be the more easily accessible species of “elemental sulfur” for microorganisms. Recently, it was demonstrated that *Alc.* *vinosum* indeed uses only, or at least strongly prefers, the polymeric sulfur (sulfur chains) fraction of commercially available elemental sulfur and is probably unable to take up and form sulfur globules from cyclo-octasulfur (*Franz et al.*, 2007). Evidence for the formation of intermediates like sulfide or polysulfides during uptake of elemental sulfur was not obtained. It may be speculated that “sulfur chains” rather than the more stable “sulfur rings” are the microbiologically preferred form of elemental sulfur also for other sulfur-oxidizing bacteria. This hypothesis gains some support in the recent study of *Urich et al.* (2006) who investigated the influence of different sulfur species on enzyme functions in the sulfur oxygenase reductase from *Aquifex aeolicus*. Theoretical considerations on the basis of the crystal structure of this enzyme led to the hypothesis that linear sulfur but not cyclic sulfur species can serve as a substrate for this enzyme.

### 5.4. Properties of Sulfur Globules

In anoxygenic phototrophic sulfur bacteria, sulfur appears to be generally deposited outside the cytoplasm. GSB and PSB of the family *Ectothiorhodospiraceae* form extracellular sulfur globules, while the globules are located in the periplasmic space in members of the family *Chromatiaceae* (*Pattaragulwanit et al.*, 1998).
SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 139

### 5.4.1. Intracellular versus Extracellular Sulfur Deposition

Sulfur deposited inside the cell is unattainable for other individuals. It might be concluded that deposition of sulfur outside the cells would make this sulfur available to all individuals in the culture. However, using sulfide-limited continuous cultures of *Cba. thiosulfatiphilum* DSMZ 249, it was shown that elemental sulfur produced by the GSB, although being deposited extracellularly, is not easily available for other individuals, and apparently remains (in part) attached to the cells (*van Gemerden*, 1986). It was even concluded that functionally the elemental sulfur formed by GSB is not extracellular and that, in essence, not even a GSB cell can profit from the extracellular sulfur produced by another GSB cell. Microscopic observations indeed show that most sulfur is attached to the cells and very little is free floating in the medium (*Trüper*, 1984; *Trüper and Genovese*, 1968). The mechanistic explanation of this phenomenon could be the production of extracellular capsules of, for example, polysaccharides or the presence of tubes of pili outside the cell (see Section 5.3.2). The latter were observed in cultures of *Chl. limicola* strain DSMZ 257 and *Cba. thiosulfatiphilum* DSMZ 249 (*Cohen-Bazire*, 1963) and also *Cba. limnaeum* UdG 6038 (*Pibernat and Abella*, 1996).

### 5.4.2. Speciation of Sulfur in Sulfur Deposits

Various techniques and approaches have been used to determine the exact chemical nature of the “elemental sulfur” in phototrophic sulfur bacteria. Different modifications of sulfur were detected or claimed and several models were proposed. Originally, orthorhombic sulfur crystals in *Thiocystis violacea* and other phototrophic bacteria were proposed by *Trüper and Hathaway* (1967). However, microscopic investigations (polarized microscopy; freeze etch microscopy) yielded first evidence that the elemental sulfur in the globules is not crystalline orthorhombic sulfur (*Hageage et al.*, 1970; *Remsen*, 1978; *Remsen and Trüper*, 1973). Using X-ray diffraction, *Hageage et al.* (1970) determined the intracellular sulfur of *Chromatium okenii* as “spherically symmetrical aggregates of radially arranged arrays of S₈ molecules that were in a metastable, liquid modification.” By determining and calculating the density of non-sulfur-containing bacteria and two sulfur-globule-containing *Chromatium* spp. in comparison with the density of elemental sulfur, *Guerrero et al.* (1984) postulated that the intracellularly deposited sulfur is complexed with another low-dense component and might be “hydrated sulfur.” Raman spectroscopic investigations showed that the sulfur globules of *Alc. vinosum* and *Hlr. abdelmalekii* contain only small

amounts of S₈ rings (Steudel, 1985; Then, 1984). Steudel and coworkers suggested that the sulfur in the sulfur globules consists of a nucleus of S₈ rings surrounded by water and long-chain sulfur species like polysulfides act as amphiphilic interface (Steudel, 1989; Steudel *et al.*, 1990). As another model for the modification of sulfur in sulfur bacteria, sulfur-sols (small milky-colored emulsions of elemental sulfur (sulfur rings) in water) were proposed (Steudel, 1996b, 2003; Steudel and Albertsen, 1999).

Finally, X-ray absorption near-edge structure (XANES) spectroscopy at the sulfur K-edge using synchrotron radiation was used as an *in situ* approach to investigate the sulfur speciation in intact phototrophic bacterial cells (Pickering *et al.*, 2001; Prange *et al.*, 1999, 2002a). In almost all earlier investigations, the sulfur globules were extracted from the cells prior to analysis (e.g., X-ray diffraction) causing changes in the chemical structure of the sulfur (Prange *et al.*, 2002a). In contrast, XANES is not only a sensitive but also a non-destructive method (Prange and Modrow, 2002). XANES revealed only one main form of sulfur in sulfur globules of GSB and PSB (Prange *et al.*, 2002a). Despite the different site of deposition (outside or inside the confines of the cell) the sulfur mainly consists of long sulfur chains very probably terminated by organic residues (mono-/bis-organyl polysulfanes) in all investigated members of the Chromatiaceae, Ectothiorhodospiraceae, and Chlorobiaceae. The organic residue present at the end of the sulfur chains appears to be glutathione or very similar to glutathione (Prange *et al.*, 2002a) and could be responsible for keeping the sulfur in a “liquid” state at ambient pressure and temperature conditions. Furthermore, this finding supports earlier speculations that reduced glutathione (probably in its amidated form) could act as a carrier molecule of sulfur to and from the globules (Bartsch *et al.*, 1996; Pott and Dahl, 1998) (see also Section 5.5.2).

XANES spectroscopy also yielded evidence that the sulfur chains in globules of *Alc. vinosum* are gradually shortened during oxidation of intracellularly stored sulfur to sulfate (Prange *et al.*, 2002a). It should be mentioned that some controversy has arisen about the interpretation of data acquired by XANES spectroscopy. Investigations of phototrophic sulfur bacteria by two different groups (Pickering *et al.*, 2001; Prange *et al.*, 2002a) yielded partly comparable experimental data but were interpreted in quite a different way. Pickering *et al.* (2001) concluded on the basis of theoretical considerations that the sulfur is “simply solid S₈.” The discrepancies are mainly based on the measurement mode (George *et al.*, 2002; Prange *et al.*, 2002b). The model for the sulfur globules of *Alc. vinosum* that corresponds best with the available experimental data consists of long sulfur chains terminated by organic groups as was suggested by Kleinjan *et al.* (2003).

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 141

Sulfur from sulfur globules isolated in the presence of oxygen from anaerobically grown *Alc. vinosum* was found as S₈ rings (Prange *et al.*, 2002a), indicating the influence of oxygen and the necessity of *in situ* methods like XANES spectroscopy that can be applied without destruction of the original sulfur environment.

### 5.4.3. Intracellular Localization of Sulfur Deposits

As already pointed out, the intracellular sulfur globules of *Chromatiaceae* reside in the periplasm (Pattaragulwanit *et al.*, 1998). While these sulfur globules appear to be more or less evenly distributed in many species (see also Fig. 1C), they can have very special and conspicuous localizations in other species. In *Allochromatium vinosum*, for example, globules are predominantly located at the two poles of the cell. Dividing cells form additional sulfur globules near the central division plane. In *Lamprobacter modestohalophilus* the sulfur globules appear in the center of cells, while they are found in the peripheral part of the cells that is free of gas vesicles in species of the genera *Lamprocystis* and *Thiodictyon*. Sulfur globules are also found in the cell periphery in *Thiopedia rosea* (Imhoff, 2005a). For *Thiorhodovibrio winogradsky* a formation of up to 10 small sulfur globules in a row along the long cell axis has been reported (Overmann *et al.*, 1992b). The specialized arrangement of sulfur inclusions suggests an important structure function relationship.

### 5.4.4. Sulfur Globule Envelopes in Chromatiaceae

Like many chemotrophic sulfur-oxidizing bacteria that form intracellular sulfur globules, the sulfur globules in the *Chromatiaceae* are enclosed by a protein envelope (Fig. 1D) (Brune, 1995a; Dahl, 1999; Dahl and Prange, 2006). In *Alc. vinosum*, this envelope is a monolayer of 2–5 nm consisting of three different hydrophobic “sulfur globule proteins” (SGPs) of 10.5 kDa, 10.6 kDa (*SgpA* and *SgpB*), and 8.5 kDa (*SgpC*). In *Tca. roseopersicina* strain 6311 (DSM 219) the envelope contains only two proteins of 10.7 kDa and 8.7 kDa (Brune, 1995a). The two larger sulfur globule proteins (*SgpA* and *SgpB*) of *Alc. vinosum* are homologous to each other and to the larger protein of *Tca. roseopersicina*. The smaller sulfur globule proteins (*SgpC*) in *Alc. vinosum* and *Tca. roseopersicina* are also homologous, indicating that these proteins are highly conserved between different species of the family *Chromatiaceae*. The proteins are targeted to the bacterial periplasm by Sec-dependent signal peptides (Pattaragulwanit *et al.*, 1998). The targeting process was experimentally verified with *phoA* fusions in *E. coli*
(Pattaragulwanit *et al.*, 1998) and also in *Alc. vinosum* (Prange *et al.*, 2004). Electron micrographs of two other species of the family Chromatiaceae (*Thiocystis violacea* and *Tca. roseopersicina*) also demonstrated an extracytoplasmic localization of the sulfur globules (Pattaragulwanit *et al.*, 1998).

All three sulfur globule proteins are rich in glycine and aromatic amino acids, particularly tyrosine. The amino acid sequences contain tandem repeats typically found in cytoskeletal keratin or plant cell wall proteins suggesting that they are structural proteins rather than enzymes involved in sulfur metabolism (Brune, 1995a). A direct/covalent attachment of chains of stored sulfur to the proteins enclosing the globules is unlikely as none of the Sgp proteins sequenced so far contains cysteine residues. In *Alc. vinosum* SgpC appears to play an important role in sulfur globule expansion (Prange *et al.*, 2004). SgpA and SgpB are in part, but not fully, competent to replace each other (Pattaragulwanit *et al.*, 1998; Prange *et al.*, 2004). The construction of *Alc. vinosum* mutants lacking SgpA and SgpC or all three Sgp proteins was not possible, leading to the conclusion that a basic level of sulfur globule proteins is obligatory for cell survival even under conditions that do not allow sulfur globule formation (Prange *et al.*, 2004). Experiments with a *sgpBC* double mutant clearly showed that an envelope is indispensable for the formation and deposition of intracellular sulfur (Prange *et al.* 2004). Each of the three *sgp* genes of *A. vinosum* forms a separate transcriptional unit (Pattaragulwanit *et al.*, 1998). All are constitutively expressed; however, the expression of *sgpB* and *sgpC* is significantly enhanced under photolithoautotrophic compared to photo-organoheterotrophic conditions. Interestingly, *sgpB* is expressed 10 times less than *sgpA* and *sgpC* implying that SgpA and SgpC are the “main proteins” of the sulfur globule envelope (Prange *et al.*, 2004).

Proteinaceous envelopes have never been reported for extracellular sulfur globules. Neither the complete genome sequence of *Hlr. halophila* nor those of GSB (Frigaard and Bryant, 2008a) contain potential *sgp* genes (Table 3). As outlined above, the sulfur speciation in sulfur globules of anoxygenic phototrophic bacteria is nearly identical, irrespective of whether it is accumulated in globules inside or outside the cells. It therefore appears that the Sgp proteins themselves are not responsible for keeping the sulfur in a certain chemical structure.

### 5.4.5. Reduction of Sulfur Deposits

Sulfur globules can also serve as an electron-acceptor reserve that allows a rudimentary anaerobic respiration with sulfur. Under anoxic conditions in
the absence of light, both PSB and GSB can reduce stored sulfur back to sulfide (Kondrat'eva, 1979; Paschinger *et al.*, 1974; Trüper, 1978; van Gemerden, 1968). Some GSB strains photochemically disproportionate elemental sulfur into sulfide and thiosulfate but in the absence of CO₂ and growth (Trüper *et al.*, 1988). Sulfide production from sulfur has also been observed in heliobacteria that are transferred from the light to the dark (Starynin and Gorlenko, 1993). Van Gemerden (1968) showed that the dark reduction of stored sulfur to H₂S by *Alc. vinosum* was coupled to oxidation of glycogen to poly-β-hydroxybutyrate and suggested that sulfur reduction was used to dispose of excess reducing equivalents formed during anaerobic breakdown of glucose. Given that the sulfur globules are not directly accessible to cytoplasmic enzymes, the involvement of quinone-reactive membrane-bound enzymes resembling polysulfide reductase PsrABC from *Wolinella succinogenes* (Krafft *et al.*, 1992, 1995) would be a likely possibility. The molybdopterin-containing active site subunit PsrA of this enzyme is located in the periplasm. Genes encoding homologous proteins that are predicted to reside in the periplasm are found in the genomes of many GSB and also in *Hlr. halophila* (see Section 5.8.2 and Table 4).

### 5.5. Oxidation of Stored Sulfur to Sulfite

The oxidative degradation of sulfur deposits in PSB and GSB is a main subject of current research on sulfur metabolism in phototrophic bacteria but is still not completely understood. In the case of extracellularly deposited sulfur, this process does not only involve oxidation of the sulfur but must include binding, activation and transport into the cells (see above Section 5.3).

#### 5.5.1. Occurrence and Arrangement of *dsr* Genes

The only gene region known so far to be essential for oxidation of stored sulfur was localized by interposon mutagenesis in *Alc. vinosum* (Dahl *et al.*, 2005; Pott and Dahl, 1998). A total of 15 open reading frames, designated *dsrABEFHCMKLJOPNRS*, were identified (Fig. 3). The first two of these genes encode the reverse dissimilatory sulfite reductase (DsrAB) of *Alc. vinosum* (Hipp *et al.*, 1997; Schedel *et al.*, 1979). Very similar gene clusters are also found in *Hlr. halophila* and GSB (Table 3 and Fig. 3). In *Hlr. halophila* the *dsr* gene cluster in addition contains genes encoding putative regulatory proteins and proteins possibly involved in sulfate

Allochromatium vinosum DSMZ 180
$\xrightarrow{dsrA} \xrightarrow{dsrB} \xrightarrow{dsrE} \xrightarrow{dsrF} \xrightarrow{dsrI} \xrightarrow{dsrC} \xrightarrow{dsrM} \xrightarrow{dsrK} \xrightarrow{dsrL} \xrightarrow{dsrJ} \xrightarrow{dsrQ} \xrightarrow{dsrP} \xrightarrow{dsrN} \xrightarrow{dsrR} \xrightarrow{dsrS} \xrightarrow{uvB}$

Halorhodospira halophila SL1
$\xleftarrow{11} \xleftarrow{10} \xrightarrow{dsrA} \xrightarrow{dsrB} \xrightarrow{dsrE} \xrightarrow{dsrF} \xrightarrow{dsrH} \xrightarrow{dsrC} \xrightarrow{dsrM} \xrightarrow{dsrK} \xrightarrow{dsrL} \xrightarrow{dsrT} \xrightarrow{dsrQ} \xrightarrow{dsrP} \xrightarrow{dsrN} \xrightarrow{12} \xrightarrow{13} \xrightarrow{14} \xrightarrow{sul}$

Chlorobium phaeovibrioides DSMZ 265
$\xrightarrow{7} \xrightarrow{PSRLC3BC} \xrightarrow{PCRLC3A} \xrightarrow{1} \xrightarrow{2} \xrightarrow{3} \xrightarrow{4} \xrightarrow{5} \xrightarrow{dsrN} \xrightarrow{dsrC} \xrightarrow{dsrA} \xrightarrow{dsrI} \xrightarrow{dsrB} \xrightarrow{6} \xrightarrow{dsrE} \xrightarrow{dsrF} \xrightarrow{dsrH} \xrightarrow{dsrT} \xrightarrow{dsrM} \xrightarrow{dsrK} \xrightarrow{dsrJ} \xrightarrow{dsrQ} \xrightarrow{dsrP} \xrightarrow{8} \xrightarrow{9}$

Chlorobaculum tepidum TLS
$\xrightarrow{dsrN} \xrightarrow{dsrG} \xrightarrow{dsrA} \xrightarrow{dsrB} \xrightarrow{dsrL} \xrightarrow{6} \xrightarrow{dsrT} \xrightarrow{dsrM} \xrightarrow{dsrK} \xrightarrow{dsJ} \xrightarrow{dsrQ} \xrightarrow{dsrP} \xrightarrow{8} \xrightarrow{9}$
$\xrightarrow{dsrC} \xrightarrow{dsrA} \xrightarrow{dsrB} \xrightarrow{dsrL} \xrightarrow{dsrE} \xrightarrow{dsrF} \xrightarrow{dsrH} \xrightarrow{1} \xrightarrow{2} \xrightarrow{3} \xrightarrow{sat} \xrightarrow{aprB} \xrightarrow{aprA} \xrightarrow{qmoA} \xrightarrow{qmoB} \xrightarrow{qmoC}$

Figure 3 Schematic overview of the gene organization around the $dsr$ genes in the PSB Alc. vinosum DSMZ 180<sup>T</sup> and Hlr. halophila SL1 and the GSB Chl. phaeovibrioides DSMZ 265 and Cba. tepidum TLS. The numbers indicate genes of unknown function; genes with the same number are homologous.

transport downstream of $dsrN$ (Dahl, 2008). GSB contain a cluster, $dsrNCABLEFHTMKJOP$, the only difference to Alc. vinosum being the absence of $dsrRS$ and the presence of $dsrT$. This cluster is present in all GSB, except Chl. ferrooxidans and Chp. thalassium, and it most likely encodes the same function as in Alc. vinosum. In Cba. tepidum TLS the $dsr$ genes are split into two clusters, and three functional $dsr$ genes are duplicated ($dsrA$, $dsrC$, and $dsrL$) (Fig. 3). This may be due to a frameshift mutation in the $dsrB$ gene in a recent ancestor of the TLS strain that rendered the gene non-functional. This could have been selected for a duplication, rearrangement, and subsequent frameshift mutation of a small segment of the genome, which restored a functional $dsrB$ gene but also resulted in a duplication of the $dsrCABL$ gene cluster. The two regions that contain a $dsrCABL$ cluster in Cba. tepidum TLS are 99.4% identical at the nucleotide level. From the currently available data, it appears that the $dsr$ genes only occur as a single cluster in all other genome-sequenced GSB.

The absence of $dsr$ genes in Chl. ferrooxidans is consistent with the observation that this bacterium is incapable of growth on elemental sulfur and sulfide. Because this strain appears to have ancestors that are sulfur- and sulfide-oxidizing GSB that contain the $dsr$ genes (Frigaard and Bryant, 2008a, b), it seems highly likely that Chl. ferrooxidans has lost the $dsr$ genes as a consequence of adapting to growth on Fe<sup>2+</sup>. The absence of $dsr$ genes in Chp. thalassium is especially interesting for two reasons: first, this organism is a very early diverging GSB, and second, this organism grows poorly on elemental sulfur. Like other GSB, Chp. thalassium grows well on sulfide and forms extracellular sulfur globules as an oxidation product (Gibson et al., 1984). However, this elemental sulfur is only very slowly oxidized; and this behavior could be due to the absence of the Dsr system. It is at present

unclear what might constitute an alternative sulfur-oxidizing system in *Chp. thalassium* (Table 4). Such a system might somehow involve the RuBisCO-like protein (RLP), which is present in all GSB including *Chp. thalassium* and which has been shown to be involved in growth on elemental sulfur in *Cba. tepidum* TLS (Hanson and Tabita, 2001, 2003). It is an interesting possibility that it might have been the acquisition of the Dsr-dependent system, which seems to be involved in efficient and complete oxidation of elemental sulfur, that led to the relatively recent, explosive radiation of the lineages of GSB that are not closely related to *Chloroherpeton* (Frigaard and Bryant, 2008a, b).

Most of the *dsr* genes are widespread not only in phototrophic but also in chemotrophic sulfur oxidizers; in addition they occur in sulfate-reducing bacteria (Grimm et al., 2008; Mussmann et al., 2005; Sander et al., 2006). In sulfate-reducing prokaryotes, sulfite reductase catalyzes the reduction of sulfite to sulfide as the final step of sulfate respiration (Matias et al., 2005). The genes *dsrABCNMKJOP* represent a core unit occurring in both sulfur-oxidizing and sulfate-reducing organisms. In contrast, other *dsr* genes appear to be specific for one of the physiological groups. The gene *dsrD* has so far only been detected in sulfate-/sulfite-reducing prokaryotes, whereas the genes *dsrEFH* appear to be typical for sulfur oxidizers (Sander et al., 2006). DsrL homologs were not only detected in sulfur oxidizers, but also in a few sulfite-respiring bacteria such as *Desulfitobacterium hafniense* and *Moorella thermoacetica* (Mussmann et al., 2005).

In several cases phylogenetic analysis of the common Dsr proteins yielded two separate clusters consisting of proteins from sulfate reducers on the one hand and of proteins from sulfur oxidizers on the other (Sander et al., 2006). Within the GSB, DsrA and other Dsr proteins constitute a monophyletic group. However, the *dsr* genes have experienced lateral gene transfer (LGT) within the GSB phylum; for example, DsrA from *Prosthecochloris aestuarii DSMZ 271**T*** is located within the *Chlorobium/Chlorobaculum* cluster (Frigaard and Bryant, 2008a). In contrast to DsrAB sulfite reductase and other cytoplasmic Dsr proteins, the components of the membrane-bound DsrMKJOP complex of GSB do not cluster with the proteins of other sulfur oxidizers but affiliate with the sulfate-/sulfite-reducing prokaryotes. This phenomenon suggests a horizontal gene transfer, which is also supported by the presence of *dsrT* (Mussmann et al., 2005) in GSB, a gene otherwise only found in sulfate-/sulfite-reducing prokaryotes (Sander et al., 2006). The Dsr system in GSB is therefore considered to have an intriguing chimeric nature possibly generated by lateral gene transfer of *dsrTMKJOP* from a sulfate-reducing prokaryote to a common ancestor of GSB.

5.5.2. Dsr Gene Products and their Proposed Functions

In *Alc. vinosum*, the *dsrAB* products form the cytoplasmic $\alpha_2\beta_2$-structured sulfite reductase. This protein is closely related to the dissimilatory sulfite reductases from sulfate-reducing bacteria and archaea (Hipp *et al.*, 1997). The prosthetic group of DsrAB is siroamide-[Fe₄S₄] with siroamide being an amidated form of the classical siroheme. The *dsrN*-encoded protein resembles cobyrinic acid $a,c$ diamide synthases and probably catalyzes the glutamine-dependent amidation of siroheme (Lübbe *et al.*, 2006). An involvement of DsrR in biosynthesis of siroamide is possible based on the finding that the genes *dsrN* and *dsrR* are fused in the chemotrophic sulfur-oxidizing bacterial endosymbiont (*Candidatus Ruthenia magnifica*) of the bivalve *Calyptogena magnifica* (Newton *et al.*, 2007). An *Alc. vinosum ΔdsrN* mutant showed a reduced sulfur oxidation rate. *Alc. vinosum* is apparently able to incorporate siroheme instead of siroamide into sulfite reductase, thereby retaining some function of the enzyme (Lübbe *et al.*, 2006).

Adjacent to *dsrAB*, the *dsrEFH* genes are located. The products of these three genes show significant similarity to each other (Pott and Dahl, 1998) and in *Alc. vinosum* they form a soluble cytoplasmic $\alpha_2\beta_2\gamma_2$-structured 75 kDa holoprotein (Dahl *et al.*, 2005). DsrC is a small soluble cytoplasmic protein with a highly conserved C-terminus including two conserved cysteine residues. Proteins closely related to DsrEFH and DsrC have recently been shown to act as parts of a sulfur relay system involved in thiouridine biosynthesis at tRNA wobble positions in *E. coli* (Ikeuchi *et al.*, 2006; Numata *et al.*, 2006). The *dsrM*-encoded protein is predicted to be a membrane-bound $b$-type cytochrome and shows similarities to a subunit of heterodisulfide reductases from methanogenic archaea. The iron–sulfur protein DsrK exhibits relevant similarity to the catalytic subunit of heterodisulfide reductases (Dahl *et al.*, 1999; Pott and Dahl, 1998). DsrK is predicted to reside in the cytoplasm. DsrP is another integral membrane protein. The periplasmic proteins DsrJ and DsrO are a triheme $c$-type cytochrome and an iron–sulfur protein, respectively. DsrKJO were co-purified from membranes of *Alc. vinosum* indicating the presence of a transmembrane electron-transporting complex consisting of DsrKMJOP (Dahl *et al.*, 2005). In *Alc. vinosum*, individual in-frame deletions of the *dsrMKJOP* genes lead to the complete inability of the mutants to oxidize stored sulfur (Sander *et al.*, 2006). DsrL is a cytoplasmic iron–sulfur flavoprotein with NADH:acceptor oxidoreductase activity (Y. Lübbe and C. Dahl, unpublished). In frame deletion of *dsrL* completely inhibited the oxidation of stored sulfur (Lübbe *et al.*, 2006). DsrS is a soluble cytoplasmic protein of unknown function.

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 147

In *Alc. vinosum*, the *dsr* genes, with the exception of the constitutively expressed *dsrC*, are expressed and the encoded proteins are formed at a low basic level even in the absence of sulfur compounds. An increased production of all Dsr proteins is induced by sulfide and/or stored sulfur (Dahl et al., 2005).

Since the proteins encoded at the *dsr* locus are either cytoplasmic or membrane-bound they cannot act directly on the extracytoplasmic sulfur globules. As already mentioned above, dissimilatory sulfite reductase catalyzes the six electron reduction of sulfite to sulfide in sulfate-reducing bacteria. Along this line, it has been proposed that sulfur stored in periplasmic or extracellular sulfur globules in PSB and GSB is reductively activated, transported to, and further oxidized in the cytoplasm by sulfite reductase operating in reverse. Different models have been suggested to explain the roles of the *dsr*-encoded proteins in such a scenario (Dahl et al., 2005; Pott and Dahl, 1998; Cort et al., 2008). In the model shown in Fig. 2 the NADH:acceptor oxidoreductase activity of DsrL is taken into account. The protein carries a thioredoxin motif CysXXCys immediately preceding the carboxy-terminal iron–sulfur cluster-binding sites. This indicates a potential disulfide reductase activity. Therefore, the possibility exists that DsrL uses NADH as electron donor for reduction of a di- or persulfidic compound. It is suggested that DsrL is involved in the reductive release of sulfide from a perthiolic organic carrier molecule transporting sulfur from the periplasmic sulfur globules to the cytoplasm (Dahl et al., 2005).

Glutathione amide is a likely candidate for the carrier molecule, as it bears an amide group at the glycyl moiety of glutathione and is especially resistant to autoxidation. It was found mainly as a persulfide in cells of *Alc. vinosum* grown photoautotrophically on sulfide (Bartsch et al., 1996). Recently, transporters have been characterized in *E. coli* mediating export (Pittman et al., 2005) and import (Suzuki et al., 2005) of glutathione. Shuttling of glutathione amide between cytoplasm and periplasm in phototrophic sulfur bacteria like *Alc. vinosum*, therefore, also appears feasible. From *Alc. vinosum*, DsrL is co-purified with sulfite reductase (Y. Lübbe and C. Dahl, unpublished). Sulfide released from the perthiol by the action of DsrL is possibly directly passed to the *dsrAB*-encoded sulfite reductase thereby reducing losses caused by evaporation of gaseous H₂S. On the other hand, sulfite reductase specifically interacts also with the membrane-bound Dsr proteins and DsrEFHC (Dahl et al., 2005). Electrons released from the oxidation of sulfide by sulfite reductase may therefore be fed into photosynthetic electron transport via DsrC and DsrKMJOP, which would be analogous to the pathway postulated for sulfate reducers (Pires et al., 2006), operating in the reverse direction. DsrM could operate as a quinone reductase, DsrP as a quinol oxidase and

finally the $c$-type cytochrome DsrJ would be reduced (Dahl *et al.*, 2005). From here, electrons could be transferred to HiPIP, the primary electron donor to the photosynthetic reaction center (Vermeglio *et al.*, 2002). The function of DsrEFH remains unclear but, as it occurs exclusively in sulfur oxidizers and shows interaction with DsrC (Cort *et al.*, 2008), it may be important for the pathway to operate in the sulfide oxidizing direction. On the other hand, sulfur-transfer reactions as performed by the related TusBCD and TusE proteins in *E. coli* could also be important for the Dsr-catalyzed sulfite-formation pathway (Ikeuchi *et al.*, 2006). Indeed, the similarity of DsrEFH to TusBCD not only on the amino acid sequence but also on a structural level points to the possibility that DsrEFH also has sulfur transferase activity (Dahl *et al.*, 2007; C. Dahl, A. Schulte, F. Grimm, J. Sander, U. Selan, C. Hong and D.H. Shin, unpublished).

### 5.6. Oxidation of Sulfite to Sulfate

While GSB are generally unable to oxidize externally supplied sulfite, some PSB can use sulfite as photosynthetic electron donor. In addition, sulfite appears to be generated in the cytoplasm of all phototrophic sulfur bacteria as the product of the Dsr system (Fig. 2).

Currently, two different pathways for sulfite oxidation are known in chemotrophic and phototrophic sulfur oxidizers (Kappler and Dahl, 2001): (a) direct oxidation by sulfite dehydrogenase (EC 1.8.2.1), typically a molybdenum-containing enzyme; and (b) indirect, AMP-dependent oxidation via adenosine-5′-phosphosulfate (adenylylsulfate, APS). While the APS reductase pathway has been shown in several but not all GSB and Chromatiaceae, it has never been detected in PSB of the family Ectothiorhodospiraceae. In accordance, potential APS reductase genes (*aprBA*, see below) are not found in the genome of *Hlr. halophila* (Table 3). Enzymatic evidence for the simultaneous occurrence of both pathways in the same organism has been provided for several PSB and GSB (Brune, 1995b; Kappler and Dahl, 2001; Trüper and Fischer, 1982). The occurrence of one or both sulfite-oxidation pathways can vary between genera of the same family and even between different strains of the same genus (Frigaard and Bryant, 2008a; Kappler and Dahl, 2001).

#### 5.6.1. Indirect Pathway via Adenylylsulfate (APS)

When the indirect pathway is employed, sulfite is oxidized by adenosine-5′-phosphosulfate (APS) reductase (also called adenylylsulfate reductase, EC

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 149

1.8.99.2) in a reaction that consumes sulfite and AMP and generates APS and reducing equivalents. Theoretically, APS could then be hydrolyzed to AMP and sulfate by adenylylsulfatase (EC 3.6.2.1) but there is no evidence for this enzyme in GSB or PSB. Alternatively, the energy of the phosphosulfate anhydride bond in APS can be conserved by the action of ATP:sulfate adenyltransferase (also called ATP sulfurase; EC 2.7.7.4). ATP sulfurase generates ATP and sulfate from APS and pyrophosphate. Alternatively, adenylylsulfate:phosphate adenyltransferase (APAT, formerly ADP sulfurase (EC 2.7.7.5) (Brüser *et al.*, 2000)) generates ADP and sulfate from APS and phosphate. Since ADP can be converted to ATP and AMP by adenylate kinase, both sulfate-liberating enzymes catalyze substrate phosphorylations, which have been proposed to be of energetic importance, especially in chemolithoautotrophic bacteria (Peck, 1968). The APS pathway is also known from sulfate reduction, where it operates in the reverse direction serving assimilatory and dissimilatory purposes. The dissimilatory APS reductases are highly conserved among sulfur-oxidizing and sulfate-reducing prokaryotes (Hipp *et al.*, 1997; Meyer and Kuever, 2007). APS reductases functioning in assimilatory sulfate reduction are completely different enzymes related to 3′-phosphoadenosine-5′-phosphosulfate (PAPS) reductases (Bick *et al.*, 2000; Kopriva *et al.*, 2001) (see Section 7).

Indirect AMP-dependent oxidation of sulfite to sulfate via APS (Fig. 2) occurs in the bacterial cytoplasm with APS reductase being membrane-bound (e.g., in many Chromatiaceae) or soluble, and ATP sulfurase and APAT being soluble enzymes (Brune, 1995a; Brüser *et al.*, 2000). In *Alc. vinosum*, the genes for ATP sulfurase (*sat*) and APS reductase (*aprMBA*, with *aprM* encoding a putative membrane anchor) form an operon (Hipp *et al.*, 1997; A. Wynen, H.G. Trüper and C. Dahl, unpublished, GenBank No. U84759). In the genomes of four GSB the genes for ATP sulfurase and APS reductase are located directly adjacent to each other (Frigaard and Bryant, 2008a). Genes related to *aprM* are not present. Instead, the APS reductase and ATP sulfurase genes in GSB are always clustered with genes encoding a so-called Qmo complex (*qmoABC*). Despite the absence of a recognizable APS reductase in its genome sequence, an APS reductase activity has been purified from *Chl. limicola* DSMZ 245<sup>T</sup> and biochemically characterized although not sequenced (Kirchhoff and Trüper, 1974). This APS reductase from *Chl. limicola* DSMZ 245<sup>T</sup> was reported to have a molecular mass of about 200 kDa and to contain one flavin per molecule and non-heme iron.

Apart from potential uncharacterized membrane subunits (e.g., AprM in *Alc. vinosum*), all known dissimilatory APS reductases form heterodimers
with one $\alpha$-subunit (AprA) of 70–80 kDa (1 FAD) and one $\beta$-subunit (AprB) of 18–23 kDa (2 [4Fe-4S] centers) ([Fritz et al., 2000; Hipp et al., 1997; Lampreia et al., 1994; Molitor et al., 1998; Speich et al., 1994]). A catalytic mechanism has been proposed in which sulfite initially forms a complex with the flavin ([Brune, 1995b, and references therein]). This then reacts with AMP to yield APS, releasing two electrons that are transferred via the flavin to the iron–sulfur centers.

The proteins that mediate the electron transport between the cytoplasmic AprAB APS reductase and the quinol/quinone pool in the membrane are still poorly characterized. However, experimental evidence is accumulating that the membrane-bound redox complex QmoABC (quinone-interacting membrane-bound oxidoreductase) is involved ([Pires et al., 2003]). This complex shares homology with subunits of heterodisulfide reductases. In *Cba. tepidum* TLS and three other GSB strains as well as in the sulfate reducers *Desulfovibrio desulfuricans* and *Archaeoglobus fulgidus*, QmoA and QmoB are cytoplasmic, nucleotide- and iron–sulfur-cluster-binding subunits and QmoC is a membrane-bound, heme-$b$-binding subunit that exchanges electrons with the isoprenoid quinone pool ([Frigaard and Bryant, 2008a; Pires et al., 2003]). The Qmo complex from *Desulfovibrio desulfuricans* was biochemically characterized and shown to have quinol-oxidizing activity ([Pires et al., 2003]). As in the four GSB, the genes $aprBA-qmoABC$ are clustered in *Dsv. desulfuricans* and other *Desulfovibrio*-like strains. In summary, the enzymes encoded by the $sat-aprBA-qmoABC$ operon in GSB, in principle, allow oxidation of sulfite to sulfate via an APS intermediate with concomitant reduction of membrane-bound quinones. We propose that the membrane protein AprM serves a function analogous to that of QmoABC in *Alc. vinosum*. Interestingly, the Apr proteins of sulfur-oxidizing phototrophic bacteria and sulfate-reducing prokaryotes diverged into two phylogenetic lineages, with the GSB affiliated with the sulfate reducers while *Alc. vinosum*-related sequences form a distinct group. This phylogenetic separation is exactly reflected in the differing presence of the putative proteins functionally associated with Apr, that is, the QmoABC complex and AprM ([Meyer and Kuever, 2007]).

With regard to release of sulfate from APS via ATP sulfurylase and/or APAT, activity in crude cell extracts for both enzymes has been described for some strains of PSB ([Dahl and Trüper, 1989; Trüper and Fischer, 1982]). In other strains, one or the other has been reported. Likewise, some GSB strains have been reported biochemically to contain either ATP sulfurylase activity (strains DSMZ 249 and DSMZ 257) or ADP sulfurylase activity (strains DSMZ 263<sup>T</sup> and NCIMB 8346) but not both activities ([Bias and Trüper, 1987; Khanna and Nicholas, 1983]). The genome of none of these
strains has been sequenced. It should be noted that specific “ADP sulfurylase” activities in many reported cases were below 100 mU/(mg protein) and that such activities may be due to other enzymes not specifically involved in oxidative sulfur metabolism (Brüser et al., 2000).

The best characterized ATP sulfurylase (Sat) from any sulfur-oxidizing bacterium is the enzyme from the endosymbiont of the hydrothermal vent worm *Riftia pachyptila* (Beynon et al., 2001; Renosto et al., 1991). Like all other ATP sulfurylases, the enzyme is strictly Mg²⁺-dependent. The $V_{\text{max}}$ of ATP synthesis is seven times higher than that of molybdolysis, the assay used for measuring the APS-producing reaction. The *Riftia* symbiont enzyme also has a higher $k_{\text{cat}}$ for the ATP synthesis direction (257/s compared to 64/s for the assimilatory enzyme from *Penicillium chrysogenum* that works in the sulfate activating direction (Renosto et al., 1991)). The native enzyme appears to be a dimer (MW 90 kDa) composed of identical size subunits (396 residues). The ATP sulfurylase from *Alc. vinosum* is isolated as a monomer with an apparent molecular mass of 45 kDa (A. Wynen, C. Dahl and H.G. Trüper, unpublished). Four GSB genomes encode highly similar, sat-derived ATP-sulfurylases (Table 3; CT0862 in *Cba. tepidum* TLS).

More information is available for ATP sulfurylases from sulfate-assimilating or sulfate-reducing organisms in which the activation of the chemically inert sulfate by adenylylation is the relevant reaction. Two completely different, unrelated types of ATP sulfurylase can be distinguished: the heterodimeric CysDN type occurs exclusively in sulfate-assimilating prokaryotes, for example, *E. coli* (Leyh, 1993). The other ATP sulfurylases characterized in sufficient detail are monomers or homo-oligomers of 41–69 kDa (Gavel et al., 1998; Sperling et al., 1998; Yu et al., 2006). Size variations are due to APS kinase or PAPS-binding allosteric domains residing on the same polypeptide in some cases. Five highly conserved regions are present, two of which are rich in basic amino acids, suggesting that they may participate in binding of MgATP²⁻ and SO₄²⁻. The homo-oligomeric ATP sulfurylases are related with the sat-encoded dissimilatory ATP sulfurylases from sulfur oxidizers on the amino acid sequence level.

The *in vivo* role of APAT is especially difficult to assign because all phototrophic bacteria with significant APAT activity (>100 mU/mg in crude extracts, see above and cf. Brüser et al., 2000) also contain ATP sulfurylase. It has been hypothesized that APAT may serve to ensure a high turnover of APS under pyrophosphate-limiting conditions as this enzyme is independent of the energy-rich pyrophosphate molecule (Brüser et al., 2000). In fact, the existence of APAT as an independent entity has been questioned

for a long time. Only in 2000 the enzyme was purified from the chemotrophic sulfur oxidizer *Thiobacillus denitrificans* (Brüser *et al.*, 2000). This enzyme is a homodimer of 41.4 kDa subunits. The $K_{\mathrm{M}}$ values for APS and phosphate are 300 μM and 12 mM, respectively. The pH optimum is 8.5–9.0. Catalysis is strictly unidirectional and occurs by a Ping-Pong mechanism with a covalently bound AMP as intermediate. Histidine modification suggested a histidine as the nucleotide-binding residue. APAT from *Tb*. *denitrificans* belongs to the histidine triad (HIT) superfamily of nucleotide hydrolases and transferases and among this to the GalT family which contains three known subfamilies of enzymes (Brenner, 2002; Mccoy *et al.*, 2006): the GalT-like UDP-hexose:hexose-1-P-uridylyltransferases, APAT and diadenosine $5^{\prime}, 5^{\prime \prime}-P^{1}, P^{4}$-tetraphosphate ($\mathrm{Ap}_{4} \mathrm{~A}$) phosphorylase (Booth and Guidotti, 1995). $\mathrm{Ap}_{4} \mathrm{~A}$ phosphorylase from yeast also has APAT activity while APAT from *Tb*. *denitrificans* does not exhibit $\mathrm{Ap}_{4} \mathrm{~A}$ phosphorylase activity. The *in vivo* function of the latter enzyme may therefore indeed be the formation of ADP and sulfate from phosphate and APS. However, genetic evidence for this assumption is currently missing.

### 5.6.2. Direct Pathway

While sulfite oxidases can transfer electrons to oxygen, ferricyanide, and sometime cytochrome $c$, the sulfite dehydrogenases can use one or both of the latter electron acceptors but not oxygen (Kappler, 2008; Kappler and Dahl, 2001). The oxygen-dependent sulfite oxidases are not relevant in anoxygenic phototrophic bacteria. The sulfite dehydrogenases belong to the sulfite oxidase family of molybdoenzymes. This family comprises established sulfite-oxidizing enzymes and proteins related to these as well as assimilatory nitrate reductases from plants (Hille, 1996). All these enzymes contain a single molybdopterin cofactor; other redox active centers may be present in addition.

The best characterized sulfite-oxidizing enzymes from the sulfite oxidase family are those from avian and mammalian sources (Kisker *et al.*, 1997). These proteins are homodimers containing heme $b$ and molybdenum coordinated via an MPT-type molybdenum pterin cofactor. The periplasmic SorAB protein from *Starkeya novella* is currently the best characterized bacterial sulfite-oxidizing enzyme (Doonan *et al.*, 2006; Feng *et al.*, 2003; Kappler and Bailey, 2005; Kappler *et al.*, 2000; Raitsimring *et al.*, 2005). It is a heterodimer of a large MoCo-dimer domain (40.2 kDa) and a small cytochrome $c$ subunit (8.8 kDa). Its molybdenum pterin cofactor is of the MPT-type with a 1:1 ratio between Mo and MPT. During catalysis, electrons are sequentially transferred to a single heme $c_{552}\left(E_{m 8.0}=+280 \mathrm{mV}\right)$ located

on the smaller subunit and passed on from there to a cytochrome $c_{550}$ from the same organism, thought to be the enzyme’s natural electron acceptor. Related proteins also appear to be localized in the periplasm and to contain a heme $c$-binding subunit (Myers and Kelly, 2005). Genes closely related to *sorAB* do not occur in the currently available genomes of PSB and GSB (Table 3).

Biochemical studies and, most importantly, the sequencing of a large number of bacterial genomes in the past few years revealed that many bacterial genes exist that encode proteins belonging to the sulfite oxidase family (Kappler, 2008). While the well-characterized bacterial sulfite dehydrogenases are soluble proteins, membrane-bound bacterial sulfite-oxidizing enzymes have also been reported in the literature (reviewed in Kappler, 2008; Kappler and Dahl, 2001). Most of the established or predicted soluble members of the sulfite oxidase family are periplasmic enzymes; however, some of the proteins belonging to this group (however, without a biochemically characterized function) are predicted to reside in the bacterial cytoplasm (Kappler, 2008). Direct oxidation of sulfite to sulfate in the bacterial cytoplasm can, therefore, not generally be excluded.

### 5.6.3. Sulfite Oxidation in Phototrophic Sulfur Bacteria: Unsolved Questions

Although enzymes participating in the indirect sulfite oxidation pathway in PSB have been studied for more than 30 years (Trüper and Rogers, 1971), their role *in vivo* is still questionable. As outlined above, the oxidative APS reductase pathway occurs exclusively in members of the Chromatiaceae and in some but not all GSB. In addition, APS reductase is clearly dispensable in *Alc. vinosum* (Dahl, 1996). The growth rates of the wild type and an APS-reductase-deficient *Alc. vinosum* mutant show little differences under light-limiting conditions. A difference is observed only at saturating irradiances. Under these conditions, the wild type grows considerably faster, indicating that the presence of a second pathway of sulfite oxidation allows a higher rate of supply of reducing power (Sanchez et al., 2001). Further experiments with *Alc. vinosum* indicated the involvement of a molybdoenzyme in sulfite oxidation. However, so far, all attempts have failed to prove the existence of *sorAB*-related genes or the respective protein (U. Kappler and C. Dahl, unpublished). As already pointed out above, none of the sequenced PSB or GSB contain genes homologous to those encoding proteins of the sulfite oxidase family. As *Hlr. halophila* and several of the GSB do not possess genes encoding for the APS pathway, different means for sulfite oxidation must be present.

As is apparent from Table 3, only four of 12 genome-sequenced GSB strains contain a Sat-AprBA-QmoABC enzyme system that potentially oxidizes sulfite. The *sat-aprBA-qmoABC* genes in *Cba. tepidum* TLS are immediately downstream of the *dsr* gene cluster, which corroborates the suggestion that they are involved in the biochemistry of the DSR system. An alternative enzyme that could oxidize the sulfite produced by the DSR system is the so-called PSRLC3 complex (see Section 5.8.2). This system is homologous with the polysulfide reductase (PSR) from *Wolinella succinogenes* (*Krafft et al.*, 1992). In *Wol. succinogenes*, PSR and a hydrogenase allow respiration on polysulfide using H₂ as the electron donor. However, homologs of PSR are also involved in metabolizing thiosulfate, tetra-thionate, and other inorganic and organic compounds. PSRLC3 in GSB is encoded by a homolog of the *Wol. succinogenes psrA* gene (ABL64190 in *Chl. phaeobacteroides DSMZ 266*) and fusion of *psrB-* and *psrC*-homologous genes (ABL64189 in *Chl. phaeobacteroides DSMZ 266*). Sequence analysis of the PsrBC-like subunit of PSRLC3 suggests that the PsrC-like domain has an orientation in the cytoplasmic membrane that is opposite that of the PsrC subunit of *Wol. succinogenes* PSR, such that the PsrB-like domain of PSRLC3 is in the cytoplasm. In addition, the PsrA-like subunit of PSRLC3 does not have any obvious signal sequence. Thus, the catalytic PsrA-like catalytic subunit and the PsrB-like domain of PSRLC3 are probably located in the cytoplasm. In contrast, PsrA and PsrB are established periplasmic proteins in *Wol. succinogenes*. Interestingly, the genes in GSB encoding PSRLC3 are immediately upstream of the *dsr* gene cluster (Fig. 3). In addition, three related genes (*Hhal_1934, 1935,* and *1936*) are found in the sulfur gene cluster of *Hlr. halophila*. In accordance with the situation in the GSB, the molybdopterin-binding putative active-site-bearing subunit (PsrA) as well as PsrB are predicted to be localized in the cytoplasm. In summary, it is an attractive possibility that PSRLC3 in GSB and a similar PSR-like enzyme in PSB are involved in cytoplasmic oxidation of the sulfite produced by the Dsr system (Fig. 2) (Section 5.5). If so, these cytoplasmic PSR-like enzymes could provide all of the Dsr-containing green and purple bacterial organisms that lack the putative Sat-Apr sulfite oxidation system (Table 3) with a means to oxidize sulfite (except *Chl. parvum DSMZ 263* for which no putative sulfite-oxidizing system is known). PSRLC3 is not widespread among other organisms, but a homologous complex with an overall amino acid sequence identity of approximate 50% is found in *Chloroflexus aurantiacus, Roseiflexus sp.* RS-1, and a few members of the high-GC Firmicutes.

In view of the current lack of experimental evidence for the presented suggestion, we would like to point out one further possibility. Analysis of a

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 155

soxY-deficient mutant of *Alc. vinosum* showed that this strain is severely impaired in the oxidation of sulfite (Hensen, Franz, and Dahl, unpublished). The Sox system is also a likely candidate for sulfite oxidation in purple Alpha- and Betaproteobacteria as sulfite is accepted *in vitro* as a substrate of the reconstituted Sox system of *Pcs. denitrificans* (Friedrich *et al.*, 2001; Sander and Dahl, 2008). Clearly, the question of sulfite oxidation in phototrophic sulfur bacteria will require special attention in the future.

## 5.7. Oxidation of Thiosulfate

Thiosulfate ($\text{S}_2\text{O}_3^{2-}$) is a rather stable and environmentally abundant sulfur compound of intermediate oxidation state. It fulfills an important role in the natural sulfur cycle and is used by many phototrophic and chemotrophic sulfur oxidizers (Jørgensen, 1990; Sorokin *et al.*, 1999). Two completely different pathways of thiosulfate oxidation appear to exist in PSB. In the first, thiosulfate is completely oxidized to sulfate via multiple steps. In the second, tetrathionate is produced by oxidation of two thiosulfate anions via thiosulfate dehydrogenase (thiosulfate:acceptor oxidoreductase, EC 1.8.2.2).

### 5.7.1. Oxidation of Thiosulfate to Sulfate: The Sox Multienzyme System

Many PSB and GSB can oxidize thiosulfate completely to sulfate (Table 2). In batch cultures of PSB growing on thiosulfate, the formation of sulfur globules is sometimes — but not always — observed. Sulfur globule formation on thiosulfate-containing medium has also been observed for at least some GSB (Brune, 1989; Steinmetz and Fischer, 1982) including *Cba. tepidum* TLS and *Cba. parvum* DSMZ 263<sup>T</sup> (Steinmetz and Fischer, 1982). Experiments with thiosulfate labeled with ${}^{35}\text{S}$ either in the sulfane (${}^{35}\text{SSO}_3^{2-}$) or sulfone ($\text{S}^{35}\text{SO}_3^{2-}$) position furthermore showed that *Cba. thiosulfatiphilum* rapidly released the sulfone sulfur as sulfate, while the sulfane sulfur was retained (probably as elemental sulfur) with the bacterial cells upon filtration (Khanna and Nicholas, 1982). Elemental sulfur may be an intermediate of thiosulfate oxidation in other GSB too, although it may not always be detectable due to a high turnover rate (Brune, 1989; Dahl, 2008). The kinetics of elemental sulfur formation and degradation are probably complex and depend on growth rate; for example, rapidly growing batch cultures of *Cba. tepidum* TLS under high light form much higher transient concentrations of elemental sulfur from thiosulfate than slowly growing cultures under low light (N.-U. Frigaard, unpublished). Polysulfides have also been suggested as intermediates occurring in the periplasm of GSB

during thiosulfate oxidation (Frigaard and Bryant, 2008a). Despite the obvious lack of sulfur globule accumulation under certain growth conditions or in certain strains, the formation of sulfur globules is an obligatory step during the oxidation of thiosulfate to sulfate in *Alc. vinosum* and probably also in other PSB. Two independent lines of evidence lead to this conclusion: (1) an *Alc. vinosum* mutant unable to form sulfur globules due to the lack of sulfur globule proteins cannot grow on thiosulfate (Prange *et al.*, 2004) and (2) *Alc. vinosum dsr* mutants blocked in sulfur oxidation form intracellular sulfur globules from thiosulfate as a dead end product (Pott and Dahl, 1998). In addition, studies with radioactively labeled thiosulfate demonstrated very clearly that the more reduced sulfane and the more oxidized sulfone sulfur atoms are processed differently in PSB (Smith and Lascelles, 1966; Trüper and Pfennig, 1966). Only the sulfane sulfur accumulates as stored sulfur \[S⁰\] before further oxidation, whereas the sulfone sulfur is rapidly converted into sulfate and excreted. The formation of sulfur as an intermediate in PSB is different from the thiosulfate-oxidizing pathway (Sox pathway) that occurs in a wide range of facultatively chemotrophic or photolithotrophic bacteria like *Pcs. pantotrophus* or *Rhodovulum sulfidophilum* (Appia-Ayme *et al.*, 2001; Friedrich *et al.*, 2001). In the latter, both sulfur atoms of thiosulfate are oxidized to sulfate without the appearance of sulfur deposits as intermediates.

The Sox complex, a periplasmic thiosulfate-oxidizing multienzyme complex was first found and characterized in *Pcs. versutus* (Lu *et al.*, 1985) and *Pcs. pantotrophus* (Friedrich *et al.*, 2001; Rother *et al.*, 2001). In *Pcs. pantotrophus* the Sox complex is essential for thiosulfate oxidation *in vivo* and catalyzes reduction of cytochrome *c* coupled to the oxidation of thiosulfate, sulfide, sulfite, and elemental sulfur *in vitro*. The *sox* gene cluster of *Pcs. pantotrophus* comprises 15 genes (*soxRSVWXYZABCDEFGH*). SoxR is a repressor protein of the ArsR family. SoxS is a periplasmic thioredoxin and essential for the full expression of the *sox* gene cluster (Rother *et al.*, 2005). SoxV is a membrane protein and SoxW is another periplasmic thioredoxin. SoxF is a monomeric flavoprotein with sulfide dehydrogenase activity (Bardischewsky *et al.*, 2006). The following seven genes (*soxX* to *soxD*) encode four periplasmic proteins (SoxXA, SoxYZ, SoxB, and Sox(CD)₂) which constitute the core Sox system (Fig. 4A): SoxXA is composed of two *c*-type cytochromes, the diheme SoxA and the monoheme SoxX. The heme group in the SoxX subunit has a His/Met axial ligation. One of the hemes in SoxA (heme 1) has a classical His/Cys coordination. This heme is redox inactive. Heme 2 of SoxA is also His/Cys coordinated, albeit the cysteine is modified into a cysteine persulfide ligand resulting in an unusually low midpoint redox potential of −432 ± 15 mV

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 157

A                                                                 B

SO₄²⁻ + H⁺                          S₂O₃²⁻                      S₂O₃²⁻
          SoxYZ-SH                  2e⁻ + H⁺                     2e⁻ + H⁺
                                      SoxAX                        SoxAX
                                      SoxYZ-S-S-SO₃²⁻             SoxYZ-SH
                                      H₂O                          SoxYZ-S-S-SO₃²⁻
                                      SoxB                         H₂O
                                      SoxYZ-S-SH                   SoxB
                                      SO₄²⁻ + H⁺                   R-Sₙ₊₁H or
                                      3 H₂O                        H-Sₙ₊₁⁻
                                      SoxCD                        ?
                                      R-SₙH or                     SoxYZ-S-SH
                                      H-Sₙ⁻                        SO₄²⁻ + H⁺


Figure 4 Proposed pathway of thiosulfate oxidation catalyzed by Sox systems in (A) organisms that do not form sulfur globules *en route* to sulfate and (B) organisms that form sulfur globules as intermediates. All reactions take place in the periplasm. Cycle A reflects the model as proposed by Friedrich *et al.* (2001, 2005) for the aerobic chemolithotrophic *Paracoccus denitrificans*. It involves SoxCD acting as a sulfur dehydrogenase. Cycle B is suggested to operate in PSB and GSB lacking SoxCD. The boxed names indicate Sox proteins catalyzing transformations of sulfur compounds bound to SoxYZ. The sulfur compound binding protein SoxYZ is circled. Sulfur compounds are bound by a conserved SoxY cysteine residue (Cys138 in *Pcs. pantotrophus*). The sulfur atom of this residue is indicated by a capital letter S. Two proposed sulfur acceptor substrates are shown as “RSₙH” and “HSₙ⁻.” Final deposition of sulfur in sulfur globules is proposed to proceed as indicated in Fig. 2. This figure is adapted from Sander and Dahl, 2008 with permission.

(Bamford *et al.*, 2002; Cheesman *et al.*, 2001; Dambe *et al.*, 2005; Kappler *et al.*, 2005; Reijerse *et al.*, 2007). The latter heme is the one believed to participate in catalysis. SoxYZ is free of cofactors and able to covalently bind sulfur compounds of various oxidation states to a conserved cysteine residue (Quentmeier and Friedrich, 2001). The monomeric SoxB has been shown to interact with the SoxYZ complex (Quentmeier *et al.*, 2003) and contains a dinuclear manganese cluster (Epel *et al.*, 2005). The protein is proposed to function as a sulfate thiohydrolase. Sox(CD)₂ is composed of the molybdoprotein SoxC and the diheme c-type cytochrome SoxD (Quentmeier *et al.*, 2000). The proposed mechanism for thiosulfate oxidation requires four different proteins: SoxB, SoxXA, SoxYZ, and SoxCD (Friedrich *et al.*, 2001) (Fig. 4A). SoxXA is proposed to fuse the sulfur substrate (thiosulfate) to the sulfhydryl Cys138 near the carboxy-terminus of SoxY to initiate the reaction cycle. SoxB is believed to act as a sulfate thiol esterase and to be responsible for hydrolytic cleavage of a sulfate group from the bound sulfur substrate. Sox(CD)₂ then oxidizes the remaining sulfane sulfur, acting as a sulfur dehydrogenase. Further action of

SoxB releases a second sulfate molecule and thereby restores SoxYZ. As pointed out by Sauvé *et al.* (2007), SoxYZ is the pivotal component of the Sox system since it participates in every reaction of the pathway. The crystal structure of *Pcs. denitrificans* SoxYZ revealed that the conserved carrier cysteine is located on a peptide swinging arm probably allowing the cysteine-bound intermediates to access and orient themselves within the active sites of multiple reaction partners (Sauvé *et al.*, 2007).

Since the identification of *sox* genes in *Paracoccus*, *sox* gene clusters have been found by cloning or genome-sequencing approaches not only in a great number of chemotrophic but also in many phototrophic sulfur-oxidizing bacteria, including GSB as well as purple sulfur and non-sulfur bacteria (Friedrich *et al.*, 2005; Frigaard and Bryant, 2008a; Hensen *et al.*, 2006; Sander and Dahl, 2008) (see also Table 3).

In all GSB that have *sox* genes, they occur in a conserved cluster of eight genes, *soxJXYZAKBW* (Frigaard and Bryant, 2008a). Only one GSB strain that contains this *sox* cluster has not been demonstrated to grow on thiosulfate (*Chl. chlorochromatii* CaD3), whereas all strains that grow on thiosulfate and that have been examined for *sox* genes indeed contain the *sox* cluster. This strongly indicates that oxidation of thiosulfate in GSB is dependent on *sox* genes. The gene *soxJ* (*CT1015* in *Cba. tepidum* TLS) encodes a putative FAD-containing dehydrogenase related to the sulfide-binding FccB flavoprotein subunit of flavocytochrome *c*. The gene *soxK* (*CT1020* in *Cba. tepidum* TLS) encodes a hypothetical 11 kDa protein with a signal peptide and with a homolog encoded in the *sox* cluster of the PSB *Alc. vinosum* (ORF9/ABE01362) (see below). However, a homolog is not present in the *Hlr. halophila* sulfur gene cluster (Dahl, 2008). SoxA in GSB and some other bacteria such as *Alc. vinosum* and *Starkeya novella* have only one heme-binding motif corresponding to the heme 2 in *Pcs. pantotrophus*. This difference is reflected in a phylogenetic sequence analysis, in which SoxA sequences from GSB and *Alc. vinosum* group separately from the SoxA sequences of *Pcs. denitrificans* and *Rhodobacter sphaeroides* that have two heme-binding motifs. The *soxCD* genes, which are essential components of the Sox system in *Pcs. pantotrophus*, do not occur in the genome sequences of GSB. This observation suggests (1) that the persulfide-form of the SoxYZ carrier protein (SoxYZ-S-SH) is transformed back to the unmodified form (SoxYZ-SH) differently in GSB and in *Pcs. pantotrophus* and (2) that the sulfane moiety from the thiosulfate molecules that become attached to the SoxYZ carrier protein is not completely oxidized to sulfate in the GSB Sox system as they are in the *Pcs. pantotrophus* Sox system. In this process, a net electron gain in the GSB Sox system is only accomplished by the SoxAX-dependent oxidation. The conservation of the *soxJ* and *soxK* genes in the

GSB *sox* gene cluster suggests that SoxJ and SoxK may substitute for SoxCD in GSB in the process that regenerates the SoxYZ complex (Fig. 4). *Alc. vinosum* also contains *sox* genes. Gene inactivation and complementation studies clearly showed that the *soxBXA* and *soxYZ* genes, located in two independent gene regions, are essential for thiosulfate oxidation in *Alc. vinosum* (*Hansen et al.*, 2006). Three periplasmic Sox proteins were purified from *Alc. vinosum*: the heterodimeric *c*-type cytochrome SoxXA (SoxX 11 kDa, SoxA 29 kDa; one covalently bound heme is present in each subunit), the heterodimeric SoxYZ (SoxY 12.7 kDa, SoxZ 11.2 kDa), and the monomeric SoxB (62 kDa, predicted to bind two manganese atoms) (*Hansen et al.*, 2006).

In *Alc. vinosum* the genes *soxB* and *soxXA* are transcribed divergently. Upstream of *soxB* a gene encoding a potential regulator protein is located and immediately downstream of *soxA* two further interesting genes are found. The first (ORF9) is homologous to *soxK* and encodes a hypothetical 12.2 kDa (9.2 kDa after processing) protein with a signal peptide. The second (*rhd*) encodes a putative periplasmic protein (22.2 kDa after processing) containing a conserved domain typical for rhodaneses. A homologous gene is not found close to either *sox* genes of GSB or in *Hlr. halophila*. In vitro, rhodaneses (thiosulfate sulfur transferases) can catalyze the transfer of the sulfane sulfur atom of thiosulfate to cyanide yielding thiocyanate (rhodanide, SCN⁻) and sulfite. This is, however, not the physiological role in most cases. In the past, the detection of rhodanese and thiosulfate reductase activity in phototrophic sulfur bacteria led to the assumption that thiosulfate would be cleaved into sulfite and sulfide in the presence of suitable reduced thiol acceptors like glutathione and dihydrolipoic acid, and that the H₂S formed during the proposed reaction would be immediately oxidized to sulfur stored in sulfur globules (*Brune*, 1989, 1995b; *Dahl*, 1999). However, gene inactivation showed that the *Alc. vinosum rhd* product does not play such a vital role and is dispensable for thiosulfate oxidation (*Hansen et al.*, 2006). The physiological role of the *rhd*-encoded protein remains to be elucidated. The deduced properties of other genes encoded in immediate vicinity of the *Alc. vinosum sox* genes were described in detail by *Hansen et al.* (2006). A function in oxidative sulfur metabolism for these hypothetical proteins is not obvious.

In *Hlr. halophila* putative *sox* genes are clustered but not organized in a single operon (*Dahl*, 2008). *Hlr. halophila* contains a gene homologous to *soxH*. A related gene is neither present in GSB nor located close to the sequenced *sox* genes in *Alc. vinosum*. The *soxH* gene is not required for lithotrophic growth on thiosulfate in *Pcs. pantotrophus* (*Rother et al.*, 2001). In *Hlr. halophila*, *soxBHYZ* appear to be co-transcribed. They are separated

from a gene encoding a fusion of SoxXA by a divergently oriented cluster of four genes, among them $fccAB$ possibly encoding a flavocytochrome $c$ (sulfide dehydrogenase). The derived FccB polypeptide also shows similarity to SoxF, an important though not essential component of the $Pcs$. pantotrophus Sox system (Bardischewsky *et al.*, 2006). However, the similarity is significantly lower than that to the flavoprotein subunit FccB of $Alc$. vinosum flavocytochrome $c$ (Dolata *et al.*, 1993; Reinartz *et al.*, 1998). The gene immediately upstream of $fccB$ in $Hlr$. halophila is clearly related to $fccA$ encoding the cytochrome $c$ subunit of $Alc$. vinosum flavocytochrome $c$ while similarity to $soxE$ from $Pcs$. pantotrophus is below detection limits in searches using the BLAST algorithm (Altschul *et al.*, 1990).

On the basis of these results and the model suggested by Friedrich *et al.* (2001) for non-sulfur-storing bacteria (Fig. 4A), a model for thiosulfate oxidation in sulfur-storing organisms is proposed (Fig. 4B). The initial oxidation and covalent binding of thiosulfate to SoxYZ is catalyzed by SoxXA and sulfate is then hydrolytically released by SoxB. Due to the lack of the “sulfur dehydrogenase” SoxCD, the sulfane sulfur atom still hooked up to SoxY cannot be directly further oxidized in organisms like $Alc$. vinosum. Probably, the sulfur is instead transferred to growing sulfur globules. Such a suggestion is feasible as the sulfur globules in $Alc$. vinosum and in many if not all other organisms forming intracellular sulfur deposits reside in the bacterial periplasm (Dahl and Prange, 2006; Pattaragulwanit *et al.*, 1998) and therefore in the same cellular compartment as the Sox proteins. How the transfer of SoxY-bound sulfur to the sulfur globules is achieved is currently unclear as the lack of the potential sulfur transferase encoded immediately adjacent to $soxXA$ in $Alc$. vinosum did not lead to a detectable phenotype. Possibly, other sulfur transferases present in the cells function as a back up system (Hensen *et al.*, 2006). In GSB and members of the Ectothiorhodospiraceae like $Hlr$. halophila, additional steps may be necessary for formation of extracellular sulfur globules, that is, sulfur deposited beyond the outer membrane.

As outlined above, the Sox system appears to occur in two different versions in two physiologically different groups of bacteria that oxidize thiosulfate completely to sulfate: those that form sulfur as an intermediate and those that perform a direct oxidation without accumulation of sulfur (Hensen *et al.*, 2006). A survey of currently sequenced purple bacterial genomes supported the presence of two subgroups of organisms containing sox gene clusters (Sander and Dahl, 2008): those containing genes for the potential sulfur dehydrogenase SoxCD and those lacking these genes. This conspicuous difference is well related to the intermediates formed in the organisms. A complete sox gene cluster occurs in organisms not forming

deposits of elemental sulfur. These include the thiosulfate-oxidizing purple non-sulfur bacteria. The *soxCD* genes are not present in most organisms forming sulfur deposits. These include the GSB and PSB. However, it should be pointed out that the chemotrophic sulfur-oxidizing epsilonproteobacteria *Thiomicrospira crunogena* and *Sulfurimonas denitrificans* (Scott et al., 2006; Sievert et al., 2008) and the marine gammaproteobacterium *Neptuniibacter caesariensis* MED92 (Arahal et al., 2007) have *soxCD*, although their other *sox* genes are more closely related to those of *Cba. tepidum*, *Alc. vinosum*, and *Thb. denitrificans*, which do not have *soxCD* and produce sulfur globules. Interestingly, *Tms. crunogena* is able to form sulfur from thiosulfate (Javor et al., 1990) while elemental sulfur formation by *S. denitrificans* has not been reported. Sievert et al. (2008) speculated that *Tms. crunogena soxCD* is regulated and turned on and off depending on environmental conditions. Furthermore, these authors proposed that SoxCD may even function differently in *S. denitrificans* and *Tms. crunogena* and that it might exhibit sulfite dehydrogenase activity in these organisms.

### 5.7.2. Oxidation of Thiosulfate to Tetrathionate: Thiosulfate Dehydrogenase

The formation of tetrathionate from thiosulfate has been mainly studied in chemoorganotrophic bacteria that use thiosulfate as a supplemental but not as the sole energy source (Jørgensen, 1990; Podgorsek and Imhoff, 1999; Sorokin et al., 1999). The pathway occurs only in a few PSB including *Alc. vinosum* (Hansen et al., 2006; Smith and Lascelles, 1966).

In *Alc. vinosum* the ratio between tetrathionate and sulfate formed from thiosulfate is strongly pH-dependent with more tetrathionate as the product under slightly acidic conditions (Smith, 1966). In *Alc. vinosum* thiosulfate dehydrogenase is a periplasmic 30 kDa monomer with an isoelectric point of 4.2. The enzyme contains heme *c* and is reduced by thiosulfate at pH 5.0 but not at pH 7.0. In accordance, the pH optimum of the enzyme was determined to be 4.25 (Hansen et al., 2006). An examination of the kinetic properties of *Alc. vinosum* thiosulfate dehydrogenase with ferricyanide as artificial electron acceptor was initiated but interpretation of experimental results is complicated by the fact that enzymes that use two molecules of the same substrate do not follow regular Michaelis–Menten kinetics. However, some important constants could be estimated: the limiting *V*<sub>max</sub> is about 34,000 units/(mg protein) (corresponding to a *k*<sub>cat</sub> of 1.7 × 10<sup>4</sup>/s) and the [S]<sub>0.5</sub> for ferricyanide is about 0.5 mM. [S]<sub>0.5</sub> is the substrate concentration that yields half maximal velocity. It is important to note that it is not identical to *K*<sub>m</sub> as a *K*<sub>m</sub> cannot be given for reactions not following Michaelis–Menten kinetics (Segel, 1993). While thiosulfate did not display

strong substrate inhibition at any of the experimental ferricyanide levels, ferricyanide did show substrate inhibition on *Alc. vinosum* thiosulfate dehydrogenase (Hansen *et al.*, 2006). Furthermore, the enzyme was significantly inhibited by sulfite (50% inhibition at 80 μM of sulfite). Under optimized assay conditions cytochrome *c* from yeast is used as electron acceptor instead of ferricyanide by the enzyme, whereas horse heart cytochrome *c* is not accepted.

The properties of *Alc. vinosum* thiosulfate dehydrogenase described by Hansen *et al.* (2006) are compatible with older data presented by Smith (1966) and Fukumori and Yamanaka (1979). In both reports a tetrathionate-forming activity with a pH optimum in the acidic range was described. The presence of a tetrathionate-forming enzyme operating at pH 8.0 in *Alc. vinosum* (Knobloch *et al.*, 1981; Schmitt *et al.*, 1981) could not be confirmed by Hansen *et al.* (2006). Thiosulfate dehydrogenases from other bacterial sources show remarkable heterogeneity with respect to structural properties and catalytic characteristics (Brune, 1989; Kusai and Yamanaka, 1973; Then and Trüper, 1981; Visser *et al.*, 1996) which has been interpreted as indicating convergent rather than divergent evolution (Visser *et al.*, 1996). A gene sequence encoding a heme-containing thiosulfate dehydrogenase has not yet been reported. A Blast search with the amino-terminal sequence of the enzyme from *Alc. vinosum* yielded only one significantly related sequence, a hypothetical *c*-type cytochrome from *Cupriavidus* (*Ralstonia*, *Wautersia*) *metallidurans* (Hansen *et al.*, 2006).

## 5.8. Other Enzymes Related to Sulfur Compound Oxidation

### 5.8.1. RuBisCO-Like Protein

The enzyme RuBisCO (ribulose 1,5-bisphosphate carboxylase/oxygenase) catalyzes the key step in the Calvin–Benson–Bassham CO₂ fixation pathway in many phototrophic and chemotrophic organisms (Tabita, 1999). Genome sequencing revealed that some bacteria and archaea contain homologs of RuBisCO, called RuBisCO-like proteins (RLPs), which do not have the same enzymatic activity as bonafide RuBisCO. For example, in *Bacillus subtilis* RLP functions as a 2,3-diketomethylthiopentyl-1-phosphate enolase in the methionine salvage pathway of this organism (Ashida *et al.*, 2003). All GSB genomes sequenced to date contain an RLP, as does *Alc. vinosum* (Q93UZ0) but not *Hlr. halophila*. GSB do not have other recognizable genes

for a methionine salvage pathway and thus the RLP probably does not function in methionine metabolism. A mutant of *Cba. tepidum* TLS lacking RLP (CT1772) has a pleiotropic phenotype with increased levels of oxidative stress proteins and defects in photopigment content, photoautotrophic growth rate, carbon fixation rates, and the ability to oxidize thiosulfate and elemental sulfur (Hanson and Tabita, 2001, 2003). Notably, sulfide oxidation is not affected in the *rlp* mutant of *Cba. tepidum* TLS. Hanson and Tabita subsequently suggested that RLP is involved in the biosynthesis of a low-molecular-weight thiol, which is essential for oxidation of thiosulfate and elemental sulfur. The possible role of such a hypothetical thiol as a carrier of sulfane sulfur is illustrated in Fig. 2. However, the function of GSB RLP is probably not limited to oxidation of inorganic sulfur compounds because *Chl. ferrooxidans* contains an RLP very similar to the RLP in other GSB, even though this organism cannot grow on inorganic sulfur compounds and does not contain genes thought to be involved in oxidation of thiosulfate (*sox* genes) and elemental sulfur (*dsr* genes) (Table 4).

### 5.8.2. Polysulfide-Reductase-Like Complexes

The well-characterized polysulfide reductase (PSR) in the chemotrophic *Wolinella succinogenes* allows respiration on polysulfide by producing sulfide (Krafft et al., 1992). The *Wol. succinogenes* PSR is encoded by the *psrABC* genes and consists of two periplasmic subunits, a molybdopterin-containing PsrA subunit and a [4Fe-4S]-cluster-binding PsrB subunit, and a membrane-anchoring PsrC subunit that binds an isoprenoid quinone and exchanges electrons with PsrB. The PsrA subunit is translocated to the periplasm by the twin-arginine transport (Tat) system. Several homologs of PSR are found in the genome sequences of phototrophic sulfur bacteria but because PSR homologs have been implicated in metabolism of thiosulfate, tetra(thionate), and other inorganic and organic compounds, the function of the PSR-like complexes in GSB and PSB cannot easily be established from sequence analysis alone.

Three types of complexes, here denoted polysulfide-reductase-like complex 1, 2, and 3 (PSRLC1, PSRLC2, and PSRLC3), with sequence similarity to the characterized PSR in *Wol. succinogenes*, are found in the genome sequences of GSB (Table 4). Similar to the case of *Wol. succinogenes* PSR, PSRLC1 (comprising CT0494, CT0495, and CT0496 in *Cba. tepidum* TLS) and PSRLC2 (comprising Cpha266_2562, 2563, and 2564 in *Chl. phaeobacteroides* DSMZ 266) are encoded by three genes. For both PSRLC1 and PSRLC2, the PsrA-like subunits with the catalytic site have a Tat signal

sequence and thus should be translocated into the periplasm. PSRLC3 is encoded by two genes similar to *psrA* and a fusion of *psrBC*. The PsrA-like subunit of this complex does not have a Tat signal and presumably is located in the cytoplasm. The PSRLC3 has been implicated in sulfite oxidation as discussed in Section 5.6.3. Homologs of PSRLC1 and PSRLC2 are found in many other organisms; for example, *Carboxydotermus hydrogenoformans* has a PSRLC1 homolog that has an overall amino acid sequence identity of approximate 50% with the PSRLC1 of *Cba. tepidum TLS.*

*Hlr. halophila* also has three types of PSR-like complexes with unknown function, each of which is encoded by three genes. The genes of one of the PSRLC (Hhal_1164, 1165, 1166) are located immediately adjacent, but divergently transcribed, to genes encoding an FccAB flavocytochrome *c* homolog (Hhal_1162, 1163). The PsrA-like subunit of this PSRLC and one of the other PSRLC (Hhal_0353, 0354, 0355) contain a Tat signal sequence. However, the PsrA-like subunit of the third PSRLC in *Hlr. halophila* (Hhal_1936, 1935, 1934) does not have a Tat signal sequence and the genes encoding this complex cluster with genes related to sulfur metabolism. Thus, it is possible that a PSR-like complex in the GSB and *Hlr. halophila* that do not have an APS reductase (AprBA) functions in sulfite oxidation (see Section 5.6.3).

### 5.8.3. Sulfhydrogenase-Like and Heterodisulfide-Reductase-Like Complexes

A putative cytoplasmic αβγδ-heterotetrameric, bi-directional hydrogenase, which resembles *Pyrococcus furiosus* hydrogenase II that catalyzes H₂ production, H₂ oxidation, as well as the reduction of elemental sulfur and polysulfide to sulfide (*Ma et al.*, 2000), is present in all sequenced GSB genomes, except those of *Chl. phaeovibrioides DSMZ 265* and *Chl. luteolum DSMZ 273**T*** (Table 4). The respective genes are not present in the PSB *Hlr. halophila*. The genes encoding this putative sulfhydrogenase form a conserved *hyd1* cluster, *hydB1G1DA* (CT1891–CT1894 in *Cba. tepidum TLS*), except in *Chl. chlorochromatii CaD3* in which the genes are split into two clusters, *hydB1G1* and *hydDA*. Since *Cba. tepidum TLS* is unable to grow on H₂, these genes apparently do not confer the ability to oxidize large amounts of H₂. Likewise, the presence of this enzyme in *Chl. ferrooxidans DSMZ 13031**T*** and its absence from *Chl. phaeovibrioides DSMZ 265*, *Chl. luteolum DSMZ 273**T***, and *Hlr. halophila* suggests that its primary role is not related to elemental sulfur or polysulfide metabolism.

There are two types of complexes with sequence homology to heterodisulfide reductases encoded in the genomes of the sequenced GSB. One is the Qmo complex, which is probably involved in intracellular sulfite oxidation as

discussed above (Section 5.6.1). The other is encoded by genes that form a conserved cluster with genes encoding a putative hydrogenase. This hdr-hyd2 gene cluster, hdrD-hdrA-orf1247-orf1248-hydB2-hydG2, is conserved in seven of the sequenced strains. The hdrD gene in GSB is a fusion of the hdrC and hdrB genes found in other organisms. As with the Hyd1 complex mentioned above, the presence of Hdr-Hyd2 in Chl. ferrooxidans DSMZ 13031<sup>T</sup> and its absence from two other GSB strains and also from Hlr. halophila, suggests that its presence is not essential in elemental sulfur or polysulfide metabolism.

# 6. EVOLUTION OF SULFUR METABOLISM

The dissimilatory oxidation of sulfur compounds as a means to supply phototrophic sulfur bacteria with electrons for thiotrophic growth clearly is a complex metabolism. Nevertheless, increasing biochemical and genetic information reveal that many similarities exist in the enzyme systems that transform sulfur compounds in thiotrophic organisms and that these similarities in many cases clearly are due to lateral gene transfer events. However, it is also apparent that the exact genetic composition in a given organism is optimized for the particular lifestyle of the organism.

As described above (Section 5.5), the Dsr system appears to allow efficient oxidation of elemental sulfur in both PSB and GSB. However, the genetic history of the Dsr system is somewhat different in these two types of bacteria, since parts of the Dsr system in GSB (DsrAB and other proteins) are related to the Dsr system in PSB, whereas other parts (DsrMKJOPT) are related to the Dsr system in sulfate-reducing bacteria (Sander *et al.*, 2006). In fact, the observation that the earliest diverging GSB, *Chp*. thalassium, does not possess the Dsr system, suggests that the common ancestor of all currently known GSB did not possess the Dsr system. Thus, it is an interesting possibility that it might have been the acquisition of a (chimeric) Dsr system that has led to the relatively recent, explosive radiation of the lineages of GSB that are not closely related to *Chloroherpeton*. Genetic evidence also suggests that the GSB that contain the Dsr system may oxidize the sulfite generated by the Dsr enzymes in different manners, some using the Sat-Apr-Qmo system (Section 5.6.1), while others may use the putative PSRLC3 complex (Sections 5.6.3 and 5.8.2). The former system may present an advantage over the latter system by apparently conserving a high-energy phosphate anhydride bond (Fig. 2). One GSB strain, *Chl*. ferrooxidans, that grows on ferrous iron has lost the Dsr system and other sulfur-oxidizing enzymes and consequently the ability to grow on sulfur compounds altogether.

Thiosulfate oxidation is found only in certain GSB strains and is dependent on a modified Sox system (Section 5.7.1). It differs conspicuously from the well-characterized Sox system of certain chemotrophic bacteria by not including the SoxCD enzyme (Fig. 4). The absence of SoxCD disables GSB from oxidizing thiosulfate completely to sulfate in the periplasm using the Sox system. As a consequence of not having SoxCD, these organisms probably transiently accumulate polysulfide species, and possibly elemental sulfur, which subsequently is oxidized by the Dsr system. The same is true for the PSB *Alc. vinosum* that has a lifestyle similar to GSB. The question then arises as to why these bacteria do not oxidize thiosulfate (and possibly other sulfur compounds) completely to sulfate using the Sox system? One possible explanation is that the absence of SoxCD forces the oxidation of sulfur to occur via the Dsr system, thus possibly conserving energy by generating a transmembrane proton gradient and high-energy phosphate anhydride bonds (Fig. 2), which do not occur if the sulfur is oxidized completely to sulfate in the periplasm (L.H. Gregersen and N.-U. Frigaard, unpublished). This metabolic “trick” could be a hallmark of phototrophic sulfur bacteria since all known Sox-containing PSB and GSB contain the Dsr system and not SoxCD, whereas other bacteria that contain a closely related Sox system (e.g., the marine gammaproteobacteria *Neptuniibacter caesariensis* MED92 (*Arahal et al.*, 2007) and *Congregibacter litoralis* KT71 (*Fuchs et al.*, 2007)) contain SoxCD but not the Dsr system.

Thiosulfate utilization by the Sox system in GSB presents an interesting case study of lateral transfer. The Sox proteins are only present in some GSB strains and have phylogenies that are incongruent with that for ribosomal RNAs (*Frigaard and Bryant*, 2008a). This indicates that the *sox* genes in GSB were not inherited vertically from a common ancestor. However, the Sox proteins from GSB form a monophyletic cluster in phylogenetic analyses, which strongly implies that the currently known *sox* genes in GSB have only been laterally exchanged within the GSB. In addition, all eight genes in the *sox* gene cluster of GSB have congruent phylogenies. This suggests that all eight *sox* genes were transferred simultaneously as one conserved cluster to each recipient strain. How might such a transfer have occurred? Two GSB strains, DSMZ 273<sup>T</sup> and DSMZ 265, which are very closely related in terms of ribosomal RNA phylogeny and genome organization, differ in one important respect: the latter strain contains the *sox* gene cluster whereas the former strain does not (Table 3). Analysis of the genome sequences reveals that the *sox* cluster in strain DSMZ 265 resides on an 11 kbp island that contains four additional genes. This island appears to have been inserted into a region of the genome in a recent

ancestor that was not involved in sulfur metabolism. This ancestor was likely to be similar to strain DSMZ 273<sup>T</sup> and unable to use thiosulfate. The genes on the island include a transposase, an integrase, and an RNA-directed DNA polymerase, all of which are indicative of a mobile element. Such a mobile element may be carried by a plasmid or a virus. The observation that the *sox* genes only occur in the *Chlorobium/Chlorobaculum* lineage, and not other GSB lineages, suggests that the putative mobile element only transfers within this lineage of closely related organisms—a trait observed with many plasmids and viruses. To our knowledge, no virus of any kind that infects GSB has yet been isolated but there is no reason to believe such viruses do not exist.

# 7. SULFATE ASSIMILATION IN ANOXYGENIC PHOTOTROPHIC BACTERIA

Assimilatory metabolism of sulfate in anoxygenic phototrophic bacteria has not been covered in the reviews by Brune (1989, 1995b). We therefore take this opportunity to summarize the currently available information for all groups of these organisms and include not only GSB and PSB but also purple non-sulfur bacteria, FAP bacteria, aerobic anoxygenic bacteria, and one representative of the Heliobacteria.

## 7.1. Historical Aspects and General Outline of the Pathway

Assimilatory reduction of sulfate in phototrophic organisms (mostly plants and algae) has been investigated mostly from a nutritional point of view. Sulfate was believed generally not to be limiting for growth (Mothes and Specht, 1934).

For Enterobacteria, a pathway (Fig. 5) was described (reviewed in Kredich, 1996) which was believed to be operating in phototrophic bacteria as well: (1) activation of sulfate to form a sulfonucleotide (adenosine-5′-phosphosulfate); (2) followed by a second activation step with ATP to form 3′-phospho-adenosine-5′-phosphosulfate (PAPS); (3) a thioredoxin-dependent PAPS reductase then forms free sulfite; (4) sulfite reductase uses electrons from NAD(P)H or ferredoxin, respectively, and free sulfide is released that is finally (5) integrated into cysteine. It should be noted that activated sulfate in the form of PAPS is generally very important for sulfate transfer reactions to numerous substrates yielding an array of sulfonated

Sulfate<sub>out</sub>
$cysPTWA$ or $sul1$
↓
Sulfate<sub>in</sub>
| ATP
ATP sulfurylase
$cysDN$ or $sat$
↓ → PPi
APS
| 2 e⁻
APS reductase
$cysH$
↓ → AMP
Sulfite
| 6 e⁻ + 6 H⁺
Sulfite reductase
$cysI$ or $cysIJ$
↓ → 3 H₂O
Sulfide
O-acetylserine (thiol)-lyase A or B
$cysK$ or $cysM$
↓
Cysteine

APS kinase
$cysC$
↓
PAPS
| 2 e⁻
PAPS reductase
$cysH$
← PAP
Sulfotransferases
→ Sulfonated products
e.g. sulfolipids

Serine + Acetyl-CoA
Serine acetyltransferase
$cysE$
↓
O-Acetylserine

Figure 5 Assimilatory sulfate reduction and cysteine biosynthesis. Alternative pathways and enzymes occurring in anoxygenic phototrophic bacteria are shown. Direct reduction of APS appears to be more common than direct reduction of PAPS (cf. Table 5). Gene names are boxed. Except for the membrane-bound transporters, all enzymes involved are localized in the cytoplasm.

compounds, for example, sulfolipids (Negishi *et al.*, 2001). Such biosyntheses can also occur in organisms that are not able to perform the reductive steps, that is, the capacity for activation of sulfate is not necessarily combined with the capacity for assimilatory sulfate reduction.

In the 1970s, the simple transfer of the enterobacterial pathway was challenged for cyanobacteria, unicellular green algae, as well as for *Euglena* (reviewed by Schmidt and Jäger, 1992): It was especially questioned whether PAPS was indeed the substrate for the first reduction step or whether APS was reduced without further activation. Evidence was presented for both alternatives (Abrams and Schiff, 1973; Brunold and Schiff, 1976; Kanno *et al.*, 1996; Li and Schiff, 1991; Schmidt, 1973; Schwenn, 1989). The true nature of the reaction mechanism remained elusive (reviewed in Hell, 1997). For plants, the final solution was approached with the proof of APS reductase in *Arabidopsis thaliana* and biochemical identification of the enzyme from *Lemna* (Gutierrez-Marcos *et al.*, 1996; Setya *et al.*, 1996; Suter *et al.*, 2000). PAPS turned out to be the primary substrate for reduction in diverse eubacteria (including enterobacteria), archaea, and fungi, but not in plants, algae, and cyanobacteria. In the latter groups, APS is the major substrate for reduction and this also appears to hold true for most anoxygenic phototrophic bacteria (Sander and Dahl, 2008).

In bacteria including anoxygenic phototrophic bacteria, sulfate reduction pathways are clearly induced under sulfur-limited conditions. Today it is generally accepted that sulfate uptake, reduction and fixation of sulfide are the most regulated steps in this process (Haverkamp and Schwenn, 1999; Kredich, 1996; Neumann *et al.*, 2000; Sander and Dahl, 2008).

### 7.2. Occurrence of Sulfate Assimilation in Anoxygenic Phototrophic Bacteria

The genes related to assimilatory sulfate reduction in the genomes of anoxygenic phototrophic bacteria currently available are tabulated in Table 5. It appears that the enzymes involved in sulfate assimilation in these organisms are not uniform but may vary even within the same family (compare *Alc. vinosum* and *Tcs. roseopersicina*).

Some anoxygenic phototrophic bacteria are very much specialized for living in habitats with reduced sulfur compounds and such bacteria may completely lack a sulfate reduction pathway. The GSB are a good example for this specialization. Most members of the Chlorobiaceae lack genes for the enzymes catalyzing sulfate assimilation (in Table 5 *Cba. tepidum* is given as an example). Within the Chromatiaceae and Ectothiorhodospiraceae the

Table 5 Occurrence of genes related to assimilatory sulfate reduction in anoxygenic phototrophic bacteria

| Organism | $c y s P T W A$ or $s u l L$ | sat | $c y s D N$ | $c y s H$ | $c y s J$ | $c y s I$ |
| --- | --- | --- | --- | --- | --- | --- |
| Alpha proteobacteria |  |  |  | $[4 F e-4 S]^{2+}$ |  | no $[4 F e-4 S]^{2+}$ |
| Bradyrhizobiaceae |  |  |  |  |  |  |
| Rhodopseudomonas palustris BisA 53 | $c y s P T W A$ | — |  | + | — | + |
| Rhodopseudomonas palustris BisB 18 | $c y s P T W A$ | — | 2 copies (both + APS kinase) | + | — | + |
| Rhodopseudomonas palustris BisB 5 | $c y s P T W A$ | — | 2 copies (both + APS kinase) | + | — | + |
| Rhodopseudomonas palustris | $c y s P T W A$ | — | + (+ APS kinase) | + | — | + |
| CGA 009 (ATCC BAA-98) |  |  |  |  |  |  |
| Rhodopseudomonas palustris HaA 2 | $c y s P T W A$ | — | + (+ APS kinase) | + | — | + |
| (ATCC BAA-1122) |  |  |  |  |  |  |
| Rhodobacteraceae |  |  |  |  |  |  |
| Roseovarius sp. 217<sup>a</sup> | $s u l L$ | — |  | + | + | + |
| Roseovarius nubinhibens ISM | no $c y s P$, no $s u l L$ | — |  | + (+ APS kinase) | + | + |
| (DSMZ 15170)<sup>a</sup> |  |  |  |  |  |  |
| Roseobacter sp. MED193<sup>a</sup> | $s u l L$ | — |  | + (+ APS kinase) | + | + |
| OCh 114 | $s u l L$ | — |  | + (+ APS kinase) | + | + |
| Rhodobacter sphaeroides | 3 copies $c y s P T W A$ | + (+ APS kinase) |  | + | + | + |
| ATCC 17025<sup>a</sup> |  |  |  |  |  |  |
| Rhodobacter sphaeroides | 2 copies $c y s P T W A$ | + (+ APS kinase) |  | + | + | + |
| ATCC 17029<sup>a</sup> |  |  |  |  |  |  |
| Rhodobacter sphaeroides 2.4.1 | $c y s P T W A$ | + (+ APS kinase) |  | + | + | + |
| (ATCC 55304) |  |  |  |  |  |  |
| Rhodobacterales bacterium HTCC2654<sup>a</sup> | $s u l L$ | — |  | + (+ APS kinase) | + | + |
| Dinoroseobacter shibae DFL_12<sup>a</sup> | $s u l L$ | — |  | + | + | + |

+ (APS kinase)
+ (APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+ (+ APS kinase)
+
Table 5 (Continued)

| Organism | $cysPTWA$ or $sulI$ | sat | $cysDN$ | $cysH$ | $[4Fe-4S]_{2+}$ | $cysJ$ |
|----------|---------------------|-----|---------|--------|------------------|--------|
| Chloroflexus aurantiacus J-10-f (DSMZ 635<sup>T</sup>) | $sulI$ | $+(+APS\ kinase)$ | $-$ | $+$ | $-$ | $+$ |
| Roseiflexus castenholzii DSMZ 13941<sup>Ta</sup> | $sulI$ | $+(+APS\ kinase)$ | $-$ | $+$ | $-$ | $-$ |
| Roseiflexus sp. RS-1 | $sulI$ | $+(+APS\ kinase)$ | $-$ | $+$ | $-$ | $-$ |
| Heliobacteria |  |  |  |  |  |  |
| Heliobacterium modesticaldum ATCC 51547<sup>Ta</sup> | $cysPTWA$ | $-$ | $+$ | $-$ | $-$ | $-$ |

This survey is based on complete or draft genome sequence data, except for *Alc. vinosum* and *Tca. roseopersicina* for which only limited sequence information is available. The following purple bacterial genera are ABC bacteria: *Roseovarius*, *Roseobacter*, *Dinoroseobacter*, *Acidiphilium*, and *Congregibacter*. Four non-phototrophic species (underlined) that do not contain genes encoding reaction center polypeptides are included for comparative reasons. *Cba. tepidum* is shown as an example of those GSB that are incapable of assimilatory sulfate reduction.

a Draft genomes at the time of compilation of this table. Genomes were analyzed by BLAST searches using the resources provided by Integrated Microbial Genomes (DOE Joint Genomes Institute, http://img.jgi.doe.gov) or by the Phototrophic Prokaryotes Sequencing Project (http://genomes.tgen.org).

b Neumann et al. (2000).

c Haverkamp and Schwenn (1999).

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 173

specialized species are also unable to grow photoorganotrophically and accordingly none of these species assimilates sulfate as a sulfur source (Imhoff, 2005a, 2005b). On the other hand, very many versatile purple and even a few green bacteria are able to assimilate and reduce sulfate in the absence of a source of reduced sulfur. Chl. ferrooxidans DSMZ 13031<sup>T</sup> is one example for a GSB that grows with sulfate as the sole sulfur source and cannot utilize sulfide, thiosulfate, or elemental sulfur as electron donors (Heising et al., 1999). In agreement with this observation, the Chl. ferrooxidans genome encodes a single gene cluster that includes the assimilatory sulfate reduction genes *cysIHDNCG* and the sulfate permease genes *cysPTWA*, which are transcribed in opposite directions. These assimilatory sulfate reduction genes share a high degree of sequence similarity with those of the clostridia *Clostridium thermocellum* and *Desulfitobacterium hafniense*. An identical *cys* gene cluster is observed in Chl. luteolum DSMZ 273<sup>T</sup> raising the possibility that this strain also is capable of assimilatory sulfate reduction and growth in the absence of reduced sulfur compounds using electron donors such as H<sub>2</sub> and Fe<sup>2+</sup>.

The GSB Chl. phaeovibrioides DSMZ 265 and Ptc. aestuarii DSMZ 271<sup>T</sup> also contain *cysC* and *cysN* homologs. However, in these two organisms the genes occur in a cluster that appears to be involved in another aspect of sulfur metabolism. This cluster also contains a *cysQ* homolog that may encode a phosphatase acting on APS or PAPS (Neuwald et al., 1992) and a homolog of the ArsB/NhaD superfamily of permeases that translocates Na<sup>+</sup> and various anions such as sulfate across the cytoplasmic membrane (500231320 and 500231330, respectively, in Chl. phaeovibrioides DSMZ 265). It is therefore possible that Chl. phaeovibrioides DSMZ 265 and Ptc. aestuarii DSMZ 271<sup>T</sup> possess a system that processes APS or sulfite (or both) differently than in other GSB and that actively excretes sulfate.

Among the FAP bacteria, the ability to assimilate sulfate may or may not be present, exemplified by the sequenced *Chloroflexus aggregans*, *Chloroflexus aurantiacus*, and *Roseiflexus* strains. While *Cfl.* aurantiacus harbors a set of genes the products of which can catalyze the complete activation and reduction of sulfate, *Cfl.* aggregans and *Roseiflexus* strains are predicted to be able to take up and activate, but not to be capable of reducing sulfate (Table 5). There is currently only one genome-sequenced representative of the Heliobacteria, *Heliobacterium modesticaldum* Icel (ATCC 51547<sup>T</sup>) (http://genomes.tgen.org/helio.html). This organism cannot grow on sulfate as the sole sulfur source. Accordingly, genes for proteins involved in reduction of (P)APS and sulfite are not present.

7.3. Uptake of Sulfate

Assimilatory sulfate reduction commences with the uptake of extracellular sulfate (Fig. 5). In *Escherichia coli* the genes *cysTWA* encode three membrane-bound components of a periplasmic substrate-binding transport system (Hryniewicz and Kredich, 1991). The preceding gene *cysP* has been shown to encode a periplasmic thiosulfate binding protein distinct from the sulfate binding protein of *Salmonella typhimurium*. In plants as well as many lower eukaryotes, sulfate transport is mediated by both high- and low-affinity systems. Both act as thermodynamic active processes which are thought to be driven by proton motive force, through a H⁺/SO₄²⁻ co-transport (Mendoza-Cózatl et al., 2005). The characterized high-affinity transporters Sul1 from *Saccharomyces cerevisiae* (Cherest et al., 1997) and Hst1At from *Arabidopsis thaliana* (Vidmar et al., 2000) are prominent members of the MFS superfamily of sulfate permeases and related transporters (COG0659). This family also comprises a number of bacterial proteins.

Most anoxygenic phototrophic bacteria encode genes homologous to *cysPTWA* indicating that they use a sulfate transport system related to that of Enterobacteria (Table 5). In some anoxygenic phototrophic bacteria, including *Hlr. halophila*, related genes do not appear to be present. Instead, a gene is found encoding for a potential sulfate permease related to the high-affinity sulfate transporter of *Saccharomyces cerevisiae*. However, in *Hlr. halophila*, *Rhodobacterales* bacterium HTCC2654, and *Rosoevari us* sp. 217, this gene is located in immediate vicinity of clusters of genes involved in oxidative sulfur metabolism (Sander and Dahl, 2008). Therefore, the encoded transporter may also be involved in export of sulfate produced in the cytoplasm as an end product of the oxidation of reduced sulfur compounds.

7.4. Activation of Sulfate

Once inside of the cell, the low reactivity of sulfate must first be overcome by the formation of a phosphate–sulfate anhydride bond in the compound adenosine-5′-phosphosulfate (adenylylsulfate, APS). This reaction is catalyzed by the enzyme ATP sulfurylase (EC 2.7.7.4) (Leustek and Saito, 1999). Due to instability of APS and the unfavorable reaction equilibrium (∆G′ = +45 kJ/mol) the backward reaction was believed to be prevented by cleavage of pyrophosphate by abundant pyrophosphatase activity and further activation by APS kinase to PAPS (Leustek et al., 2000). As
elaborated above, PAPS is indeed very important for sulfate transfer reactions to numerous substrates (Negishi *et al.*, 2001).

Assimilatory ATP sulfurylases occur in two different forms: a heterodimeric CysDN type from *E. coli* (Leyh, 1993) and an unrelated homo-oligomeric type found in bacteria, plants, and fungi (Foster *et al.*, 1994; MacRae *et al.*, 2001). The latter polypeptides are related to the *sat*-encoded dissimilatory ATP sulfurylases of sulfate-reducing and chemotrophic sulfur-oxidizing prokaryotes (Beynon *et al.*, 2001; Sperling *et al.*, 1998). In both, heterodimeric and homo-oligomeric, ATP sulfurylases carboxy-terminal APS kinase domains may be present.

In anoxygenic phototrophic bacteria both types of ATP sulfurylases can be present. Genes encoding the Sat-type of ATP sulfurylase are found in *Rhodobacteraceae* and FAP bacteria while the other analyzed genomes contain *cysDN* genes. In most of these cases, *cysN* is fused with an APS kinase gene.

### 7.5. Reduction to Sulfite

For incorporation of reduced sulfur into biomolecules, for example amino acids, sulfate in APS is reduced to sulfite and finally into sulfide. This process may occur differently, depending on the organism (Fig. 5). Similar to the situation in *E. coli* APS may first be phosphorylated by an APS kinase (EC 2.7.1.25, gene *cysC*) using ATP to produce PAPS and ADP. In the following reaction, PAPS reductase (EC 1.8.99, gene *cysH*) generates free sulfite using reduced thioredoxin as electron donor. The other pathway involves the direct reduction of APS by APS reductase (EC 1.8.99.x, also *cysH*) (Bick *et al.*, 2000).

Assimilatory APS reductases and PAPS reductases are related on a sequence level. The signature of the plant, algae and cyanobacteria type APS reductase was originally seen in a two domain structure consisting of a reductase domain with an \[4Fe-4S\]$^{2+}$ cluster and a C-terminal thioredoxin/glutaredoxin domain, using glutathione as electron donor. In contrast, PAPS reductase was believed to have only the reductase domain without an iron–sulfur cluster and to use thioredoxin as electron donor (Kopriva *et al.*, 2002). Today, the situation indicates even more variants of assimilatory sulfonucleotide reductases: (1) the *E. coli* CysH-like enzymes reduce PAPS using thioredoxin or glutaredoxin as electron donor, have no \[4Fe-4S\]$^{2+}$ cluster, and are found in many eubacteria, archaea, and fungi (Berendt *et al.*, 1995); (2) another CysH-like group reduces APS rather than PAPS, uses thioredoxin as electron donor, has an \[4Fe-4S\]$^{2+}$ cluster, and was found
in *Pseudomonas aeruginosa* (Bick et al., 2000); (3) yet another protein with strong homology to CysH is from *Bacillus subtilis* and is able to reduce APS and PAPS (Berndt et al., 2004); (4) the plant-type sulfonucleotide reductase (e.g. *Arabidopsis thaliana*, *Enteromorpha intestinalis*) uses APS, carries an [4Fe-4S]$^{2+}$ cluster, and contains a C-terminal glutaredoxin-like domain (Bick et al., 1998; Gao et al., 2000); (5) a reductase from the moss *Physcomitrella patens* that shares high similarity with CysH-like proteins, has no iron–sulfur cluster but prefers APS rather than PAPS (Kopriva et al., 2007). This functional variability seems independent of oxygenic or phototrophic lifestyles and reflects the complex evolutionary development of assimilatory sulfate reduction (Kopriva, 2008).

Table 5 shows that almost all of the *cysH* genes present in anoxygenic phototrophs encode proteins predicted to bind an iron–sulfur cluster. Based on the above considerations it appears most likely that these proteins reduce APS rather than PAPS. *Alkalimnicola ehrlichii* is an exception as it contains a *bonafide* PAPS reductase gene albeit in addition to a gene encoding an assimilatory APS reductase. For *Tcs. roseopersicina* strain M1 the presence of a PAPS reductase gene and a product dedicated to the reduction of PAPS has been shown biochemically (Haverkamp and Schwenn, 1999).

### 7.6. Reduction of Sulfite to Sulfide

Sulfite is finally reduced to sulfide by an assimilatory sulfite reductase (EC 1.8.7.1). Again, different types can be differentiated: The NADPH-dependent assimilatory sulfite reductases of enterobacteria are composed of two different subunits: a siroheme-[4Fe-4S]-containing protein (CysI) and a flavoprotein (CysJ) (Murphy et al., 1973; Ostrowski et al., 1989; Siegel et al., 1973). In contrast the ferredoxin-dependent sulfite reductases from cyanobacteria, algae, and higher plants are much simpler homo-oligomers of just a siroheme-[4Fe-4S]-containing protein. These enzymes are only very distantly related to their dissimilatory counterparts (Dhillon et al., 2005).

The data presented in Table 5 indicate that the ferredoxin-dependent sulfite reductase is more common in purple bacteria than the NADPH-dependent enzyme consisting of two different polypeptides. *Tca. roseopersicina* strain M1 is currently the only purple bacterium for which the existence of a *cysIJ*-encoded sulfite reductase has been firmly established (Haverkamp and Schwenn, 1999).

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 177

## 7.7. Incorporation of Sulfide

The final step in cysteine biosynthesis is catalyzed by either $O$-acetylserine (thiol)-lyase A or $O$-acetylserine (thiol)-lyase B, encoded by the genes $cysK$ and $cysM$, respectively (Kredich, 1992, 1996). The CysK and CysM proteins from *E. coli* are 43% identical. CysK synthesizes cysteine from $O$-acetylserine and sulfide, while the CysM protein differs in that it can utilize thiosulfate in addition to sulfide. The reaction between $O$-acetylserine and thiosulfate produces S-sulfocysteine, which is converted into cysteine by an as yet uncharacterized mechanism. On a biochemical level, cysteine and S-sulfocysteine biosynthesis in anoxygenic phototrophic bacteria were studied by Hensel and Trüper (1976, 1981). Both activities were found in several species of Chromatiaceae and Ectothiorhodospiraceae, and in *Rhodocyclus tenuis* and *Rubrivivax gelatinosus*, both purple non-sulfur bacteria belonging to the Betaproteobacteria. The tested alphaproteobacterial purple non-sulfur bacteria exhibited only $O$-acetylserine (thiol)-lyase A activity, that is, they did not catalyze reaction of $O$-acetylserine with thiosulfate. The two different $O$-acetylserine (thiol)-lyases were purified from *Alc. vinosum* (Hensel and Trüper, 1981).

Sulfide for cysteine biosynthesis cannot only be obtained from the transport and reduction of inorganic sulfate as described but a number of bacteria can also derive it from organic sulfonate compounds such as taurine (van der Ploeg et al., 1996, 1998, 2001).

## 8. CONCLUSIONS

The combination of available biochemical information and the survey of genome sequence data presented here allows some general conclusions to be made about the sulfur compound oxidation enzymes in GSB and PSB. Sulfide:quinone oxidoreductase (encoded by $sqr$) appears to be especially important for the oxidation of sulfide. Flavocytochrome $c$ is less widespread. Sox genes (SoxXABYZ) occur in both GSB and PSB and are responsible for the oxidation of thiosulfate. In many cases, elemental sulfur appears as an intermediate of sulfide and thiosulfate oxidation. All thiosulfate-utilizing GSB strains have an identical $sox$ gene cluster ($soxJXYZAKBW$). The $soxCD$ genes found in certain other thiosulfate-utilizing organisms like *Pcs. pantotrophus* are absent from all investigated GSB and PSB. A putative complex denoted SoyYZ, related to the thiosulfate-binding SoxYZ complex, could be involved in the processing of an as yet unidentified sulfur compound in GSB (Frigaard and Bryant, 2008a).
Siroamide-sulfite reductase (DsrAB) and proteins encoded by several other *dsr* genes are essential for the oxidation of sulfur formed as an intermediate during sulfide or thiosulfate oxidation. Most GSB and PSB utilizing sulfide or elemental sulfur contain the dissimilatory sulfite reductase *dsrNCABLEFHTMKJOP* genes. Although *Chp. thalassium* appears to have an (probably less efficient) alternative enzyme system for elemental sulfur oxidation, the *dsr* genes appear to be involved in elemental sulfur utilization in all other cases. Sulfite is the product of this process. The question of sulfite oxidation in purple bacteria will require special attention in the future, as the APS reductase pathway is restricted to only some members of the *Chromatiaceae* and *Chlorobiaceae*. It has even been shown that this pathway is not essential in *Alc. vinosum*. In addition, genes for a classical sulfite:acceptor oxidoreductase are present neither in GSB nor in PSB.

Analyses indicate that, although the phylogenies of some enzymes (e.g., the DsrA protein) are congruent with the organismal phylogeny at the phylum level, the phylogenies of other enzymes are not (e.g., the SQR protein). Some enzyme systems that are only present in some strains exhibit a phylogeny incongruent with the cellular core phylogeny (e.g., the Sox and the Sat-Apr-Qmo systems). Thus, these systems appear to result from LGT rather than gene elimination. In the case of the *sox* gene cluster, evidence for phage-mediated LGT by a mobile genetic element was identified in GSB.

As a final point, although the GSB are closely related, with the exception of *Chp. thalassium*, genomic analyses clearly show that gene elimination and LGT substantially influence the distribution of sulfur metabolism genes both within the GSB and among prokaryotes from other phyla. These observations illustrate the dynamic structures of prokaryotic genomes and in addition demonstrate that even organisms that superficially appear to be very closely related on the basis of their cellular core machinery nevertheless can have unexpected differences in physiology and life style.

Assimilatory sulfate reduction occurs in many anoxygenic phototrophic bacteria. Two different types of sulfate transporters appear to be used. While sulfate is activated to adenosine-5′-phosphosulfate via a hetero-dimeric *cysDN*-encoded ATP sulfurylase in members of the *Bradyrhizobia-ceae*, *Acetobacteraceae*, *Rhodospirillaceae*, *Comamonadaceae*, a few GSB, and *Heliobacterium modesticaldum*, the homo-oligomeric *sat*-encoded type of this enzyme occurs in *Rhodobacteraceae* and is also present in the PSB *Alc. vinosum* as well as in FAP bacteria. In most sulfate-assimilating anoxygenic phototrophic bacteria, APS appears to be immediately reduced without further phosphorylation to PAPS. The generated sulfite is reduced further to sulfide by assimilatory sulfite reductase. This enzyme appears to be ferredoxin-dependent in almost all cases, a flavoprotein subunit able to
SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 179

use NADPH as an electron donor – as in *E. coli* – is not encoded in the sequenced genomes of anoxygenic phototrophic bacteria.

# ACKNOWLEDGEMENTS

Support by the Deutsche Forschungsgemeinschaft to C.D. is gratefully acknowledged. Birgitt Hüttig provided excellent technical assistance. N.-U.F. gratefully acknowledges support from the Danish Natural Science Research Council (grant 21-04-0463). We thank Hans G. Trüper for the light and electron micrographs presented in Fig. 1.

# REFERENCES

Abrams, W.R. and Schiff, J.A. (1973) Studies on sulfate utilization by algae. 11. An enzyme-bound intermediate in the reduction of adenosine-5′-phosphosulfate (APS) by cell-free extracts of wild-type *Chlorella* and mutants blocked for sulfate reduction. *Archiv für Mikrobiologie* **94**, 1–10.

Adkins, J.P., Madigan, M.T., Mandelco, L., Woese, C.R. and Tanner, R.S. (1993) *Arhodomonas aquaeolei* gen. nov., sp. nov., an aerobic, halophilic bacterium isolated from a subterranean brine. *Int. J. Syst. Bacteriol.* **43**, 514–520.

Altschul, S.F., Gish, W., Miller, W., Myers, E.W. and Lipman, D.J. (1990) Basic local alignment search tool. *J. Mol. Biol.* **215**, 403–410.

Appia-Ayme, C., Little, P.J., Matsumoto, Y., Leech, A.P. and Berks, B.C. (2001) Cytochrome complex essential for photosynthetic oxidation of both thiosulfate and sulfide in *Rhodovulum sulfidophilum*. *J. Bacteriol.* **183**, 6107–6118.

Arahal, D.R., Lekunberri, I., Gonzalez, J.M., Pascual, J., Pujalte, M.J., Pedros-Alio, C. and Pinhassi, J. (2007) *Neptuniibacter caesariensis* gen. nov., sp. nov., a novel marine genome-sequenced gammaproteobacterium. *Int. J. Syst. Evol. Microbiol.* **57**, 1000–1006.

Arunasi, K., Sasikala, C., Ramana, C.V., Süling, J. and Imhoff, J.F. (2005) *Marichromatium indicum* sp. nov., a novel purple sulfur gammaproteobacterium from mangrove soil of Goa, India. *Int. J. Syst. Evol. Microbiol.* **55**, 673–679.

Ashida, H., Saito, Y., Kojima, C., Kobayashi, K., Ogasawara, N. and Yokota, A. (2003) A functional link between RuBisCo-like protein of *Bacillus* and photosynthetic RuBisCO. *Science* **302**, 286–290.

Baldock, M.I., Denger, K., Smits, T.H.M. and Cook, A.M. (2007) *Roseovarius* sp. strain 217: aerobic taurine dissimilation via acetate kinase and acetate-CoA ligase. *FEMS Microbiol. Lett.* **271**, 202–206.

Bamford, V.A., Bruno, S., Rasmussen, T., Appia-Ayme, C., Cheesman, M.R., Berks, B.C. and Hemmings, A.M. (2002) Structural basis for the oxidation of thiosulfate by a sulfur cycle enzyme. *EMBO J.* **21**, 5599–5610.

Barbas, S.O. and Frigaard, N.-U. (Unpublished).

Bardischewsky, F., Quentmeier, A. and Friedrich, C.G. (2006) The flavoprotein SoxF functions in chemotrophic thiosulfate oxidation of *Paracoccus pantotrophus* in vivo and in vitro. *FEMS Microbiol. Lett.* 258, 121–126.

Bartsch, R.G. (1978) Cytochromes. In: *The Photosynthetic Bacteria* (R.K. Clayton and W.R. Sistrom, eds), pp. 249–279. Plenum Press, New York.

Bartsch, R.G., Newton, G.L., Sherrill, C. and Fahey, R.C. (1996) Glutathione amide and its perthiol in anaerobic sulfur bacteria. *J. Bacteriol.* 178, 4742–4746.

Bavendamm, W. (1924) Die farblosen und roten Schwefelbakterien des Süß- und Salzwassers. In: *Pflanzenforschung (2)* (R. Kolkwitz, ed.), pp. 1–156. Verlag G. Fischer, Jena.

Beatty, J.T., Overmann, J., Lince, M.T., Manske, A.K., Lang, A.S., Blankenship, R.E., Van Dover, C.L., Martinson, T.A. and Plumley, F.G. (2005) An obligately photosynthetic bacterial anaerobe from a deep-sea hydrothermal vent. *Proc. Natl. Acad. Sci. U.S.A.* 102, 9306–9310.

Berendt, U., Haverkamp, T. and Schwenn, J.D. (1995) Reaction mechanism of thioredoxin: 3′-phospho-adenylylsulfate reductase investigated by site-directed mutagenesis. *Eur. J. Biochem.* 233, 347–356.

Berndt, C., Lillig, C.H., Wollenberg, M., Bill, E., Mansilla, M.C., de Mendoza, D., Seidler, A. and Schwenn, J.D. (2004) Characterization and reconstitution of a 4Fe-4S adenylyl sulfate/phosphoadenylyl sulfate reductase from *Bacillus subtilis*. *J. Biol. Chem.* 279, 7850–7855.

Beynon, J.D., MacRae, I.J., Huston, S.L., Nelson, D.C., Segel, I.H. and Fisher, A.J. (2001) Crystal structure of ATP sulfurylase from the bacterial symbiont of the hydrothermal vent tubeworm *Riftia pachyptila*. *Biochemistry* 40, 14509–14517.

Bias, U. and Trüper, H.G. (1987) Species specific release of sulfate from adenylylsulfate by ATP sulfurylase or ADP sulfurylase in the green sulfur bacteria *Chlorobium limicola* and *Chlorobium vibrioforme*. *Arch. Microbiol.* 147, 406–410.

Bick, J.A., Aslund, F., Chen, Y.C. and Leustek, T. (1998) Glutaredoxin function for the carboxyl-terminal domain of the plant-type 5′-adenylylsulfate reductase. *Proc. Natl. Acad. Sci. U.S.A.* 95, 8404–8409.

Bick, J.A., Dennis, J.J., Zylstra, G.J., Nowack, J. and Leustek, T. (2000) Identification of a new class of 5′-adenylylsulfate (APS) reductases from sulfate-assimilating bacteria. *J. Bacteriol.* 182, 135–142.

Blake, R.C., Shute, E.A. and Howard, G.T. (1994) Solubilization of minerals by bacteria: electrophoretic mobility of *Thiobacillus ferrooxidans* in the presence of iron, pyrite, and sulfur. *Appl. Environ. Microbiol.* 60, 3349–3357.

Blöthe, M. and Fischer, U. (2000) New insights in sulfur metabolism of purple and green phototrophic sulfur bacteria and their spheroplasts. *BIOSpektrum, Special Edition 1st Joint Congress DGHM, ÖGHMP and VAAM: “Microbiology 2000”, Munich*, p. 62.

Booth, J.W. and Guidotti, G. (1995) An alleged yeast polyphosphate kinase is actually diadenosine-5′,5″-P1,P4-tetraphosphate α,β-phosphorylase. *J. Biol. Chem.* 270, 19377–19382.

Bosshard, H.R., Davidson, M.W., Knaff, D.B. and Millett, F. (1986) Complex formation and electron transfer between mitochondrial cytochrome c and flavocytochrome c552 from *Chromatium vinosum*. *J. Biol. Chem.* 261, 190–193.

Brenner, C. (2002) Hint, Fhit, and GalT: function, structure, evolution, and mechanism of three branches of the histidine triad superfamily of nucleotide hydrolases and transferases. *Biochemistry* 41, 9003–9014.

Brettar, I., Christen, R. and Höfle, M.G. (2002) *Rheinheimera baltica* gen. nov., sp. nov., a blue-coloured bacterium isolated from the central Baltic Sea. *Int. J. Syst. Evol. Microbiol.* **52**, 1851–1857.

Brune, D.C. (1989) Sulfur oxidation by phototrophic bacteria. *Biochim. Biophys. Acta* **975**, 189–221.

Brune, D.C. (1995a) Isolation and characterization of sulfur globule proteins from *Chromatium vinosum* and *Thiocapsa roseopersicina*. *Arch. Microbiol.* **163**, 391–399.

Brune, D.C. (1995b) Sulfur compounds as photosynthetic electron donors. In: *Anoxygenic Photosynthetic Bacteria* (R.E. Blankenship, M.T. Madigan and C.E. Bauer, eds), Advances in Photosynthesis, Vol. 2, pp. 847–870. Kluwer Academic Publishers, Dordrecht.

Brune, D.C. and Trüper, H.G. (1986) Noncyclic electron transport in chromatophores from photolithoautotrophically grown *Rhodobacter sulfidophilus*. *Arch. Microbiol.* **145**, 295–301.

Brunold, C. and Schiff, J.A. (1976) Studies of sulfate utilization by algae. 15. Enzymes of assimilatory sulfate reduction in *Euglena* and their cellular localization. *Plant Physiol.* **57**, 430–436.

Brüser, T., Selmer, T. and Dahl, C. (2000) “ADP sulfurylase” from *Thiobacillus denitrificans* is an adenylylsulfate:phosphate adenylyltransferase and belongs to a new family of nucleotidytransferases. *J. Biol. Chem.* **275**, 1691–1698.

Bryant, D.A., Costas, A.M.G., Maresca, J.A., Chew, A.G.M., Klatt, C.G., Bateson, M.M., Tallon, L.J., Hostetler, J., Nelson, W.C., Heidelberg, J.F. and Ward, D.M. (2007) *Candidatus Chloracidobacterium thermophilum*: An aerobic phototrophic acidobacterium. *Science* **317**, 523–526.

Bryant, R.D., Costerton, J.W. and Laishley, E.J. (1984) The role of *Thiobacillus albertis* glycocalyx in the adhesion of cells to elemental sulfur. *Can. J. Microbiol.* **30**, 81–90.

Bryantseva, I.A., Gorlenko, V.M., Kompan’tseva, E.I., Imhoff, J.F., Süling, J. and Mityushina, L. (1999) *Thiorhodospira sibirica* gen. nov., sp. nov., a new alkaliphilic purple sulfur bacterium from a Siberian Soda lake. *Int. J. Syst. Bacteriol.* **49**, 697–703.

Bryantseva, I.A., Gorlenko, V.M., Tourova, T.P., Kuznetsov, B.B., Lysenko, A.M., Bykova, S.A., Galchenko, V.F., Mityushina, L.L. and Osipov, G.A. (2000) *Heliobacterium sulfidophilum* sp. nov. and *Heliobacterium undosum* sp. nov.: Sulfide-oxidizing heliobacteria from thermal sulfidic springs. *Microbiology* **69**, 325–334.

Buonfiglio, V., Polidoro, M., Soyer, F., Valenti, P. and Shively, J. (1999) A novel gene encoding a sulfur-regulated outer membrane protein in *Thiobacillus ferrooxidans*. *J. Biotechnol.* **72**, 85–93.

Castenholz, R.W., Bauld, J. and Jørgensen, B.B. (1990) Anoxygenic microbial mats of hot springs: Thermophilic *Chlorobium* sp. *FEMS Microbiol. Ecol.* **74**, 325–336.

Chan, L.-K., Morgan-Kiss, R. and Hanson, T.E. (2008) Genetic and proteomic studies of sulfur oxidation in *Chlorobium tepidum* (syn. *Chlorobaculum tepidum*). In: *Sulfur Metabolism in Phototrophic Organisms* (R. Hell, C. Dahl, D.B. Knaff and T. Leustek, eds), Advances in Photosynthesis and Respiration, Vol. 27, pp. 357–373. Springer, Dordrecht.

Cheesman, M.R., Little, P.J. and Berks, B.C. (2001) Novel heme ligation in a *c*-type cytochrome involved in thiosulfate oxidation: EPR and MCD of SoxAX from *Rhodovulum sulfidophilum*. *Biochemistry* **40**, 10562–10569.

Chen, Z.W., Koh, M., van Driessche, G., van Beeumen, J.J., Bartsch, R.G., Meyer, T.E., Cusanovich, M.A. and Mathews, F.S. (1994) The structure of flavocytochrome c sulfide dehydrogenase from a purple phototrophic bacterium. *Science* 266, 430–432.

Cherest, H., Davidian, J.C., Thomas, D., Benes, V., Ansorge, W. and Surdin-Kerjan, Y. (1997) Molecular characterization of two high affinity sulfate transporters in *Saccharomyces cerevisiae*. *Genetics* 145, 627–635.

Cohen-Bazire, G. (1963) Some observations on the organization of the photosynthetic apparatus in purple and green bacteria. In: *Bacterial Photosynthesis* (H. Gest, A. San Pietro and L.P. Vernon, eds), pp. 89–110. The Antioch Press, Yellow Springs, Ohio.

Cohen, Y., Jørgensen, B.B., Padan, E. and Shilo, M. (1975) Sulphide-dependent anoxygenic photosynthesis in the cyanobacterium *Oscillatoria limnetica*. *Nature* 257, 489–491.

Cort, J.R., Selan, U., Schulte, A., Grimm, F., Kennedy, M.A. and Dahl, C. (2008) *Allochromatium vinosum* DsrC: solution-state NMR structure, redox properties and interaction with DsrEFH, a protein essential for purple bacterial sulfur oxidation. *J. Mol. Biol.* 382, 692–707.

Dahl, C. (1996) Insertional gene inactivation in a phototrophic sulfur bacterium: APS-reductase-deficient mutants of *Chromatium vinosum*. *Microbiology* 142, 3363–3372.

Dahl, C. (1999) Deposition and oxidation of polymeric sulfur in prokaryotes. In: *Biochemical Principles and Mechanisms of Biosynthesis and Biodegradation of Polymers* (A. Steinbüchel, ed.), pp. 27–34. Wiley-VCH, Weinheim.

Dahl, C. (2008) Inorganic sulfur compounds as electron donors in purple sulfur bacteria. In: *Sulfur in Phototrophic Organisms* (R. Hell, C. Dahl, T. Leustek and D.B. Knaff, eds), *Advances in Photosynthesis and Respiration*, Vol. 27, pp. 289–315. Springer, Dordrecht.

Dahl, C., Engels, S., Pott-Sperling, A.S., Schulte, A., Sander, J., Lübbe, Y., Deuster, O. and Brune, D.C. (2005) Novel genes of the *dsr* gene cluster and evidence for close interaction of Dsr proteins during sulfur oxidation in the phototrophic sulfur bacterium *Allochromatium vinosum*. *J. Bacteriol.* 187, 1392–1404.

Dahl, C. and Prange, A. (2006) Bacterial sulfur globules: occurrence, structure and metabolism. In: *Inclusions in Prokaryotes* (J.M. Shively, ed.), *Microbiology Monographs*, Vol. 1, pp. 21–51. Springer, Heidelberg.

Dahl, C., Rakhely, G., Pott-Sperling, A.S., Fodor, B., Takacs, M., Toth, A., Kräling, M., Györfi, K., Kovacs, A., Tusz, J. and Kovacs, K. (1999) Genes involved in hydrogen and sulfur metabolism in phototrophic sulfur bacteria. *FEMS Microbiol. Lett.* 180, 317–324.

Dahl, C., Schulte, A., Grimm, F., Sander, J., Selan, U., Hong, C., and Shin, D.H. (Unpublished).

Dahl, C., Schulte, A. and Shin, D.H. (2007) Cloning, expression, purification, crystallization and preliminary X-ray diffraction analysis of DsrEFH from *Allochromatium vinosum*. *Acta Crystallographica* F63, 890–892.

Dahl, C. and Trüper, H.G. (1989) Comparative enzymology of sulfite oxidation in *Thiocapsa roseopersicina* strains 6311, M1 and BBS under chemotrophic and phototrophic conditions. *Z. Naturforsch.* 44c, 617–622.

Dambe, T., Quentmeier, A., Rother, D., Friedrich, C. and Scheidig, A.J. (2005) Structure of the cytochrome complex SoxXA of *Paracoccus pantotrophus*, a heme enzyme initiating chemotrophic sulfur oxidation. *J. Struct. Biol.* 152, 229–234.

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 183

Davidson, M.W., Gray, G.O. and Knaff, D.B. (1985) Interaction of *Chromatium vinosum* flavocytochrome *c*-552 with cytochromes *c* studied by affinity chromatography. *FEBS Lett.* **187**, 155–159.

de Jong, G.A.H., Hazeu, W., Bos, P. and Kuenen, J.G. (1997) Isolation of the tetrathionate hydrolase from *Thiobacillus acidophilus*. *Eur. J. Biochem.* **243**, 678–683.

de Jong, G.A.H., Tang, J.A., Bos, P., de Vries, S. and Kuenen, G.J. (2000) Purification and characterization of a sulfite: cytochrome *c* oxidoreductase from *Thiobacillus acidophilus*. *J. Mol. Catal. B* **8**, 61–67.

Denger, K., Smits, T.H.M. and Cook, A.M. (2006) Genome-enabled analysis of the utilization of taurine as sole source of carbon or nitrogen by *Rhodobacter sphaeroides* 2.4.1. *Microbiology* **152**, 3197–3206.

Denger, K., Weimitschke, S., Hollemeyer, K. and Cook, A.M. (2004) Sulfoacetate generated by *Rhodopseudomonas palustris* from taurine. *Arch. Microbiol.* **182**, 154–158.

Devasia, P., Natarajan, K.A., Sathyanarayana, D.N. and Rao, G.R. (1993) Surface chemistry of *Thiobacillus ferrooxidans* relevant to adhesion on mineral surfaces. *Appl. Environ. Microbiol.* **59**, 4051–4055.

Dhillon, A., Goswami, S., Riley, M., Teske, A. and Sogin, M. (2005) Domain evolution and functional diversification of sulfite reductases. *Astrobiology* **5**, 18–29.

Dolata, M.M., van Beeumen, J.J., Ambler, R.P., Meyer, T.E. and Cusanovich, M.A. (1993) Nucleotide sequence of the heme subunit of flavocytochrome *c* from the purple phototrophic bacterium, *Chromatium vinosum*. A 2.6-kilobase pair DNA fragment contains two multiheme cytochromes, a flavoprotein, and a homolog of human ankyrin. *J. Biol. Chem.* **268**, 14426–14431.

Doonan, C.J., Kappler, U. and George, G.N. (2006) Structure of the active site of sulfite dehydrogenase from *Starkeya novella*. *Inorg. Chem.* **45**, 7488–7492.

Drews, G. and Golecki, J.R. (1995) Structure, molecular organization and biosynthesis of membranes of purple bacteria. In: *Anoxygenic Photosynthetic Bacteria* (R. Blankenship, M.T. Madigan and C.E. Bauer, eds), *Advances in Photosynthesis*, Vol. 2, pp. 231–257. Kluwer Academic Publishers, Dordrecht.

Engelmann, T.W. (1882) Ueber Licht- und Farbenperception niederster Organismen. *Pflügers Archiv für Physiologie* **29**, 387–400.

Engelmann, T.W. (1888) Die Purpurbakterien und ihre Beziehung zum Licht. *Bot. Ztg.* **46**, 661–669.

Epel, B., Schäfer, K.O., Quentmeier, A., Friedrich, C. and Lubitz, W. (2005) Multifrequency EPR analysis of the dimanganese cluster of the putative sulfate thiohydrolase SoxB of *Paracoccus pantotrophus*. *J. Biol. Inorg. Chem.* **10**, 636–642.

Espejo, R.T. and Romero, P. (1987) Growth of *Thiobacillus ferrooxidans* on elemental sulfur. *Appl. Environ. Microbiol.* **53**, 1907–1912.

Feng, C., Kappler, U., Tollin, G. and Enemark, J.H. (2003) Intramolecular electron transfer in a bacterial sulfite dehydrogenase. *J. Am. Chem. Soc.* **125**, 14696–14697.

Fischer, U. (1984) Cytochromes and iron sulfur proteins in sulfur metabolism of phototrophic sulfur bacteria. In: *Sulfur, Its Significance for Chemistry, for the Geo-, Bio- and Cosmosphere and Technology* (A. Müller and B. Krebs, eds), pp. 383–407. Elsevier Science Publishers B.V., Amsterdam.

Foster, B.A., Thomas, S.M., Mahr, J.A., Renosto, F., Patel, H.C. and Segel, I.H. (1994) Cloning and sequencing of ATP sulfurylase from *Penicillium chrysogenum*: Identification of a likely allosteric domain. *J. Biol. Chem.* **269**, 19777–19786.

Franz, B., Lichtenberg, H., Holmes, J., Modrow, H., Dahl, C. and Prange, A. (2007) Utilization of solid “elemental” sulfur by the phototrophic purple sulfur bacterium *Allochromatium vinosum*: a sulfur K-edge XANES spectroscopy study. *Microbiology* **153**, 1268–1274.

Friedrich, C.G., Bardischewsky, F., Rother, D., Quentmeier, A. and Fischer, J. (2005) Prokaryotic sulfur oxidation. *Curr. Opin. Microbiol.* **8**, 253–259.

Friedrich, C.G., Rother, D., Bardischewsky, F., Quentmeier, A. and Fischer, J. (2001) Oxidation of reduced inorganic sulfur compounds by bacteria: emergence of a common mechanism? *Appl. Environ. Microbiol.* **67**, 2873–2882.

Frigaard, N.-U. (Unpublished).

Frigaard, N.U. and Bryant, D.A. (2006) Chlorosomes: antenna organelles in photosynthetic green bacteria. In: *Complex Intracellular Structures in Prokaryotes* (J.M. Shively, ed), *Microbiology Monographs*, Vol. 2, pp. 79–114. Springer, Berlin.

Frigaard, N.U. and Bryant, D.A. (2008a) Genomic insights into the sulfur metabolism of phototrophic green sulfur bacteria. In: *Sulfur Metabolism in Phototrophic Organisms* (R. Hell, C. Dahl, D.B. Knaff and T. Leustek, eds), *Advances in Photosynthesis and Respiration*, Vol. 27, pp. 337–355. Springer, Dordrecht.

Frigaard, N.U. and Bryant, D. (2008b) Genomic and evolutionary perspectives on sulfur metabolism in green sulfur bacteria. In: *Microbial Sulfur Metabolism* (C. Dahl and C. G. Friedrich, eds), pp. 60–76. Springer, Heidelberg, Berlin.

Frigaard, N.U., Chew, A.G.M., Li, H., Maresca, J.A. and Bryant, D.A. (2003) *Chlorobium tepidum*: Insights into the structure, physiology, and metabolism of a green sulfur bacterium derived from the complete genome sequence. *Photosyn. Res.* **78**, 93–117.

Fritz, G., Buchert, T., Huber, H., Stetter, K.O. and Kroneck, P.M.H. (2000) Adenylylsulfate reductases from archaea and bacteria are 1:1 alpha beta-heterodimeric iron-sulfur flavoenzymes – high similarity of molecular properties emphasizes their central role in sulfur metabolism. *FEBS Lett.* **473**, 63–66.

Fuchs, B.M., Spring, S., Teeling, H., Quast, C., Wulf, J., Schattenhofer, M., Yan, S., Ferreira, S., Johnson, J., Glöckner, F.O. and Amann, R. (2007) Characterization of a marine gammaproteobacterium capable of aerobic anoxygenic photosynthesis. *Proc. Natl. Acad. Sci. U.S.A.* **104**, 2891–2896.

Fukumori, Y. and Yamanaka, T. (1979) A high-potential nonheme iron protein (HiPIP)-linked, thiosulfate-oxidizing enzyme derived from *Chromatium vinosum*. *Curr. Microbiol.* **3**, 117–120.

Gao, Y., Schofield, O.M.E. and Leustek, T. (2000) Characterization of sulfate assimilation in marine algae focusing on the enzyme 5′-adenylylsulfate reductase. *Plant Physiol.* **123**, 1087–1096.

Garrity, G.M. and Holt, J.G. (2001a) Chlorobi phy. nov. In: *Bergey’s Manual of Systematic Bacteriology* (D.R. Boone, R.W. Castenholz and G.M. Garrity, eds), Vol. 1, pp. 601–623. Springer, New York.

Garrity, G. and Holt, J.G. (2001b) Phylum BVI. Chloroflexi phy. nov. In: *Bergey’s Manual of Systematic Bacteriology* (D.R. Boone, R.W. Castenholz and G. Garrity, eds), Vol. 1, p. 427. Springer, New York.

Gavel, O.Y., Bursakov, S.A., Calvete, J.J., George, G.N., Moura, J.J.G. and Moura, I. (1998) ATP sulfurylases from sulfate-reducing bacteria of the genus *Desulfovibrio*. A novel metalloprotein containing cobalt and zinc. *Biochemistry* **37**, 16225–16232.

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 185

Gehrke, T., Telegdi, J., Thierry, D. and Sand, W. (1998) Importance of extracellular polymeric substances from *Thiobacillus ferrooxidans* for bioleaching. *Appl. Environ. Microbiol.* **64**, 2743–2747.

George, G.N., Pickering, I.J., Yu, E.Y. and Prince, R.C. (2002) X-ray absorption spectroscopy of bacterial sulfur globules. *Microbiology* **148**, 2267–2268.

Gest, H. and Favinger, J.F. (1983) *Heliotacterium chlorum*, an anoxygenic brownish-green bacterium containing a “new” form of bacteriochlorophyll. *Arch. Microbiol.* **136**, 11–16.

Gibson, J., Pfennig, N. and Waterbury, J.B. (1984) *Chloroherpeton thalassium* gen. nov. et spec. nov., a non-filamentous, flexing and gliding green sulfur bacterium. *Arch. Microbiol.* **138**, 96–101.

Giovannoni, S.J., Reysbech, N.P., Ward, D. and Castenholz, R.W. (1987) Obligately phototrophic *Chloroflexus*: primary production in anaerobic hot spring microbial mats. *Arch. Microbiol.* **147**, 80–87.

Gogtova, G.I. and Vainstein, M.B. (1981) Thiosulfate oxidation in the dark by the purple sulfur bacterium *Ectothiorhodospira shaposhnikovii*. *Mikrobiologiya* **55**, 960–963.

Gonzalez, J.M., Covert, J.S., Whitman, W.B., Henriksen, J.R., Mayer, F., Scharf, B., Schmitt, R., Buchan, A., Fuhrman, J.A., Kiene, R.P. and Moran, M.A. (2003) *Silicibacter pomeroyi* sp. nov. and *Roseovarius nubinhibens* sp. nov., dimethylsulfooniopropionate-demethylating bacteria from marine environments. *Int. J. Syst. Evol. Microbiol.* **53**, 1261–1269.

Gorlenko, V.M. (1974) Oxidation of thiosulphate by *Amoebobacter roseus* in darkness under micoaerobic conditions. *Microbiologiya* **40**, 1035–1039.

Gorlenko, V.M., Bryantseva, I.A., Panteleeva, E.E., Tourova, T.P., Kolganova, T.V., Makhneva, Z.K. and Moskalenko, A.A. (2004) *Ectothiorhodospinus mongolicum* gen. nov., sp. nov., a new purple bacterium from a soda lake in Mongolia. *Microbiology* **73**, 66–73.

Gregersen, L.H. and Figgaard, N.-U. (Unpublished).

Griesbeck, C., Hauska, G. and Schütz, M. (2000) Biological sulfide oxidation: sulfide:quinone reductase (SQR), the primary reaction. In: *Recent Research Developments in Microbiology* (S.G. Pandalai, ed.), Vol. 4, pp. 179–204. Research Signpost, Trivandrum, India.

Griesbeck, C., Schütz, M., Schödl, T., Bathe, S., Nausch, L., Mederer, N., Vielreicher, M. and Hauska, G. (2002) Mechanism of sulfide-quinone oxidoreductase investigated using site-directed mutagenesis and sulfur analysis. *Biochemistry* **41**, 11552–11565.

Grimm, F., Franz, B., Dahl, C. (2008) Thiosulfate and sulfur oxidation in purple sulfur bacteria. In: *Microbial Sulfur Metabolism* (C. Dahl, C.G. Friedrich, eds), pp. 101–116. Springer, Heidelberg, Berlin.

Guerrero, R., Mas, J. and Pedros-Alio, C. (1984) Boyant density changes due to intracellular content of sulfur in *Chromatium warmingii* and *Chromatium vinosum*. *Arch. Microbiol.* **137**, 350–356.

Gutierrez-Marcos, J.F., Roberts, M.A., Campbell, E.I. and Wray, J.L. (1996) Three members of a novel small gene-family from *Arabidopsis thaliana* able to complement functionally an *Escherichia coli* mutant defective in PAPS reductase activity encode proteins with a thioredoxin-like domain and “APS reductase” activity. *Proc. Natl. Acad. Sci. U.S.A.* **93**, 13377–13382.

Hageage, G.J., Jr., Eanes, E.D. and Gherna, R.L. (1970) X-ray diffraction studies of the sulfur globules accumulated by *Chromatium* species. *J. Bacteriol.* **101**, 464–469.

Hanada, S., Hiraishi, A., Shimada, K. and Matsuura, K. (1995) *Chloroflexus aggregans* sp. nov., a filamentous phototrophic bacterium which forms dense cell aggregates by active gliding movement. *Int. J. Syst. Evol. Microbiol.* **45**, 676–681.

Hanada, S. and Pierson, B. K. (2002) The family Chloroflexaceae. In: *The Prokaryotes: an Evolving Electronic Resource for the Microbiological Community* (M. Dworkin, ed), http://link.springer-ny.com/link/service/books/10125/. Springer, Berlin, Heidelberg, New York.

Hansen, T.A. and van Gemerden, H. (1972) Sulfide utilization by purple sulfur bacteria. *Arch. Mikrobiol.* **86**, 49–56.

Hanson, T.E. and Tabita, F.R. (2001) A ribulose-1,5-bisphosphate carboxylase/oxygenase (RubisCO)-like protein from *Chlorobium tepidum* that is involved with sulfur metabolism and the response to oxidative stress. *Proc. Natl. Acad. Sci. U.S.A.* **98**, 4397–4402.

Hanson, T.E. and Tabita, F.R. (2003) Insights into the stress response and sulfur metabolism revealed by proteome analysis of a *Chlorobium tepidum* mutant lacking the Rubisco-like protein. *Photosyn. Res.* 231–248.

Haverkamp, T. and Schwenn, J.D. (1999) Structure and function of a *cysBJIH* gene cluster in the purple sulfur bacterium *Thiocapsa roseopersicina*. *Microbiology* **145**, 115–125.

Heising, S., Richter, L., Ludwig, W. and Schink, B. (1999) *Chlorobium ferrooxidans* sp. nov., a phototrophic green sulfur bacterium that oxidizes ferrous iron in coculture with a “Geospirillum” sp. strain. *Arch. Microbiol.* **172**, 116–124.

Hell, R. (1997) Molecular physiology of plant sulfur metabolism. *Planta* **202**, 138–148.

Hensel, G. and Trüper, H.G. (1976) Cysteine and S-sulfocysteine biosynthesis in phototrophic bacteria. *Arch. Microbiol.* **109**, 101–103.

Hensel, G. and Trüper, H.G. (1981) O-acetylserine-sulfhydrolase and S-sulfocysteine synthase activities of *Chromatium vinosum*. *Arch. Microbiol.* **130**, 228–233.

Hensen, D., Franz, B. and Dahl, C. (Unpublished).

Hensen, D., Sperling, D., Trüper, H.G., Brune, D.C. and Dahl, C. (2006) Thiosulfate oxidation in the phototrophic sulfur bacterium *Allochromatium vinosum*. *Mol. Microbiol.* **62**, 794–810.

Hernandez, M.E., Kappler, A. and Newman, D.K. (2004) Phenazines and other redox-active antibiotics promote microbial mineral reduction. *Appl. Environ. Microbiol.* **70**, 921–928.

Hille, R. (1996) The mononuclear molybdenum enzymes. *Chem. Rev.* **96**, 2757–2816.

Hipp, W.M., Pott, A.S., Thum-Schmitz, N., Faath, I., Dahl, C. and Trüper, H.G. (1997) Towards the phylogeny of APS reductases and sirohaem sulfite reductases in sulfate-reducing and sulfur-oxidizing prokaryotes. *Microbiology* **143**, 2891–2902.

Hiraishi, A. and Shimada, K. (2001) Aerobic anoxygenic photosynthetic bacteria with zinc-bacteriochlorophyll. *J. Gen. Appl. Microbiol.* **47**, 161–180.

Hiraishi, A. and Ueda, Y. (1994) *Rhodoplanes* gen. nov., a new genus of phototrophic bacteria including *Rhodopseudomonas rosea* as *Rhodoplanes roseus* comb. nov. and *Rhodoplanes elegans* sp. nov. *Int. J. Syst. Bacteriol.* **44**, 665–673.

Hiraishi, A., Urata, K. and Satoh, T. (1995) A new genus of marine budding phototrophic bacteria, *Rhodobium* gen. nov., which includes *Rhodobium orientis* sp. nov. and *Rhodobium marinum* comb. nov. *Int. J. Syst. Bacteriol.* **45**, 226–234.

Hirschler-Rea, A., Matheron, R., Riffaud, C., Moune, S., Eatock, C., Herbert, R.A., Willison, J.C. and Caumette, P. (2003) Isolation and characterization of spirilloid purple phototrophic bacteria forming red layers in microbial mats of

Mediterranean salterns: Description of *Halorhodospira neutrophila* sp. nov. and emendation of the genus *Halorhodospira*. *Int. J. Syst. Evol. Microbiol.* 53, 153–163.

Hoeft, S.E., Blum, J.S., Stolz, J.F., Tabita, F.R., Witte, B., King, G.M., Santini, J.M. and Oremland, R.S. (2007) *Alkalilimnicola ehrlichii* sp. nov., a novel, arsenite-oxidizing haloalkaliphilic gammaproteobacterium capable of chemoautotrophic or heterotrophic growth with nitrate or oxygen as the electron acceptor. *Int. J. Syst. Evol. Microbiol.* 57, 504–512.

Holo, H. and Sirevag, R. (1986) Autotrophic growth and CO₂ fixation of *Chloroflexus aurantiacus*. *Arch. Microbiol.* 159, 257–264.

Hryniewicz, M. and Kredich, N.M. (1991) The *cysP* promoter of *Salmonella typhimurium*: characterization of two binding sites for CysB protein, studies of in vivo transcription initiation and demonstration of the anti-inducer effects of thiosulfate. *J. Bacteriol.* 173, 5876–5886.

Ikeuchi, Y., Shigi, N., Kato, J., Nishimura, A. and Suzuki, T. (2006) Mechanistic insights into sulfur relay by multiple sulfur mediators involved in thiouridine biosynthesis at tRNA wobble positions. *Mol. Cell* 21, 97–108.

Imhoff, J.F. (1983) *Rhodopseudomonas marina* sp. nov., a new marine phototropic purple bacterium. *Syst. Appl. Microbiol.* 4, 512–521.

Imhoff, J.F. (2003) Phylogenetic taxonomy of the family *Chlorobiaceae* on the basis of 16S rRNA and *fmo* (Fenna-Matthews-Olson protein) gene sequences. *Int. J. Syst. Evol. Microbiol.* 53, 941–951.

Imhoff, J.F. (2005a) Family I. *Chromatiaceae* Bavendamm 1924, 125<sup>AL</sup> emend. Imhoff 1984b, 339. In: *Bergey's Manual of Systematic Bacteriology* (D.J. Brenner, N.R. Krieg, J.T. Staley and G.M. Garrity, eds), Vol. 2, Part B, pp. 3–40. Springer, New York.

Imhoff, J.F. (2005b) Family II. *Ectothiorhodospiraceae* Imhoff 1984b, 339<sup>VP</sup>. In: *Bergey's Manual of Systematic Bacteriology* (D.J. Brenner, N.R. Krieg, J.T. Staley and G.M. Garrity, eds), Vol. 2, Part B, pp. 41–57. Springer, New York.

Imhoff, J.F. (2008) Systematics of anoxygenic phototrophic bacteria. In: *Sulfur Metabolism in Phototrophic Organisms* (R. Hell, C. Dahl, D.B. Knaff and T. Leustek, eds), *Advances in Photosynthesis and Respiration*, Vol. 27, pp. 269–287. Springer, New York.

Imhoff, J.F. and Hiraishi, A. (2005) Aerobic bacteria containing bacteriochlorophyll and belonging to the *Alphaproteobacteria*. In: *Bergey's Manual of Systematic Bacteriology* (D.J. Brenner, N.R. Krieg, J.T. Staley and G.M. Garrity, eds), Vol. 2, Part A, pp. 133–135. Springer, Dordrecht.

Imhoff, J.F., Hiraishi, A. and Süling, J. (2005) Anoxygenic phototrophic purple bacteria. In: *Bergey's Manual of Systematic Bacteriology* (D.J. Brenner, N.R. Krieg, J.T. Staley and G.M. Garrity, eds), Vol. 2, part A, pp. 119–132. Springer, New York.

Imhoff, J.F., Süling, J. and Petri, R. (1998) Phylogenetic relationships among the *Chromatiaceae*, their taxonomic reclassification and description of the new genera *Allochromatium*, *Halochromatium*, *Isochromatium*, *Marichromatium*, *Thiococcus*, *Thiohalocapsa*, and *Thermochromatium*. *Int. J. Syst. Bacteriol.* 48, 1129–1143.

Ivanovskii, R.N., Krasil'nikova, E.N. and Fal, Y.I. (1993) A pathway of the autotrophic CO₂ fixation in *Chloroflexus aurantiacus*. *Arch. Microbiol.* 159, 257–264.

Ivanovsky, R.N., Fal, Y.I., Berg, I.A., Ugolkova, N.V., Krasilnikova, E.N., Keppen, O.I., Zakharchuc, L.M. and Zyakun, A.M. (1999) Evidence for the presence of
the reductive pentose phosphate cycle in a filamentous anoxygenic phototrophic bacterium, *Oscillochloris trichoides* strain DG-6. *Microbiology* 145, 1743–1748.

Javor, B.J., Wilmot, D.B. and Vetter, R.D. (1990) pH-dependent metabolism of thiosulfate and sulfur globules in the chemolithotrophic marine bacterium *Thiomicrospira crunogena*. *Arch. Microbiol.* 154, 231–238.

Jørgensen, B.B. (1990) The sulfur cycle of freshwater sediments: role of thiosulfate. *Limnol. Oceanogr.* 35, 1329–1342.

Kämpf, C. and Pfennig, N. (1980) Capacity of Chromatiaceae for chemotrophic growth. Specific respiration rates of *Thiocystis violacea* and *Chromatium vinosum*. *Arch. Microbiol.* 127, 125–135.

Kanno, N., Nagahisa, E., Sato, M. and Sato, Y. (1996) Adenosine 5′-phosphosulfate sulfotransferase from the marine macroalga *Porphyra yezoensis* Ueda (Rhodophyta): stabilization, purification, and properties. *Planta* 198, 440–446.

Kappler, U. (2008) Bacterial sulfite-oxidizing enzymes – enzymes for chemolithotrophy only? In: *Microbial Sulfur Metabolism* (C. Dahl, C.G. Friedrich, eds), pp. 151–169. Springer, Heidelberg.

Kappler, U. and Bailey, S. (2005) Molecular basis of intramolecular electron transfer in sulfite-oxidizing enzymes is revealed by high resolution structure of a heterodimeric complex of the catalytic molybdopterin subunit and a *c*-type cytochrome subunit. *J. Biol. Chem.* 280, 24999–245007.

Kappler, U., Bennett, B., Rethmeier, J., Schwarz, G., Deutzmann, R., McEwan, A.G. and Dahl, C. (2000) Sulfite: cytochrome *c* oxidoreductase from *Thiobacillus novellus* – Purification, characterization, and molecular biology of a heterodimeric member of the sulfite oxidase family. *J. Biol. Chem.* 275, 13202–13212.

Kappler, U. and Dahl, C. (Unpublished).

Kappler, U. and Dahl, C. (2001) Enzymology and molecular biology of prokaryotic sulfite oxidation (minireview). *FEMS Microbiol. Lett.* 203, 1–9.

Kappler, U., Hanson, G.R., Jones, A. and McEwan, A.G. (2005) A recombinant diheme SoxAX cytochrome – Implications for the relationship between EPR signals and modified heme-ligands. *FEBS Lett.* 579, 2491–2498.

Keppen, O.I., Tourova, T.P., Kuznetsov, B.B., Ivanovsky, R.N. and Gorlenko, V.M. (2000) Proposal of *Oscillochloridaceae* fam. nov., on the basis of a phylogenetic analysis of the filamentous anoxygenic phototrophic bacteria, and emended description of *Oscillochloris* and *Oscillochloris trichoides* in comparison with further new isolates. *Int. J. Syst. Evol. Microbiol.* 50, 1529–1537.

Khanna, S. and Nicholas, D.J.D. (1982) Utilization of tetrathionate and ${}^{35}$S-labelled thiosulphate by washed cells of *Chlorobium vibrioforme* f. sp. *thiosulfatophilum*. *J. Gen. Microbiol.* 128, 1027–1034.

Khanna, S. and Nicholas, D.J.D. (1983) Substrate phosphorylation in *Chlorobium vibrioforme* f. sp. *thiosulfatophilum*. *J. Gen. Microbiol.* 129, 1365–1370.

Kimble, L.K., Mandelco, L., Woese, C.R. and Madigan, M.T. (1995) *Heliobacterium modesticladum*, sp. nov., a thermophilic heliobacterium of hot springs and volcanic soils. *Arch. Microbiol.* 163, 259–267.

Kirchhoff, J. and Trüper, H.G. (1974) Adenylylsulfate reductase of *Chlorobium limicola*. *Arch. Microbiol.* 100, 115–120.

Kisker, C., Schindelin, H. and Rees, D.C. (1997) Molybdenum-cofactor-containing enzymes: structure and mechanism. *Annu. Rev. Biochem.* 66, 233–267.
Kleinjan, W.E., de Keizer, A. and Janssen, A.J.H. (2003) Biologically produced sulfur. In: *Elemental Sulfur and Sulfur-Rich Compounds I* (R. Steudel, ed.), pp. 167–187. Springer, Berlin.

Knobloch, K., Schmitt, W., Schleifer, G., Appelt, N. and Müller, H. (1981) On the enzymatic system thiosulfate-cytochrome c-oxidoreductase. In: *Biology of Inorganic Nitrogen and Sulfur* (H. Bothe and A. Trebst, eds), pp. 359–365. Springer, Berlin.

Kondrat'eva, E.N. (1979) Interrelation between modes of carbon dioxide assimilation and energy production in phototrophic purple and green bacteria. In: *Microbial Biochemistry* (J.R. Quayle, ed), *International Review of Biochemistry*, Vol. 21, pp. 117–175. University Park, Baltimore.

Kondrat'eva, E.N., Zhukov, V.G., Ivanovskii, R.N., Petushkova, Y.P. and Monosov, E.Z. (1976) The capacity of the phototrophic sulfur bacterium *Thiocapsa roseopersicina* for chemosynthesis. *Arch. Microbiol.* **108**, 287–292.

Kondrat'eva, E.N., Zhukov, V.G., Ivanovskii, R.N., Petushkova, Y.P. and Monosov, E.Z. (1981) Light and dark metabolism in purple sulfur bacteria. *Sov. Sci. Rev.* **2**, 325–364.

Kopriva, S. (2008) Phylogenetic analysis of sulfate assimilation and cysteine biosynthesis in phototrophic organisms. In: *Sulfur Metabolism in Phototrophic Organisms* (R. Hell, C. Dahl, D.B. Knaff and T. Leustek, eds), *Advances in Photosynthesis and Respiration*, Vol. 27, pp. 31–58. Springer, Dordrecht.

Kopriva, S., Büchert, T., Fritz, G., Suter, M., Benda, R., Schünemann, V., Koprivova, A., Schürmann, P., Trautwein, A.X., Kroneck, P.M.H. and Brunold, C. (2002) The presence of an iron-sulfur cluster in adenosine 5′-phosphosulfate reductase separates organisms utilizing adenosine 5′-phosphosulfate and phosphoadenosine 5′-phosphosulfate for sulfate assimilation. *J. Biol. Chem.* **277**, 21786–21791.

Kopriva, S., Büchert, T., Fritz, G., Suter, M., Weber, M., Benda, R., Schaller, J., Feller, U., Schürmann, P., Schünemann, V., Trautwein, A.X., Kroneck, P.M.H. and Brunold, C. (2001) Plant adenosine 5′-phosphosulfate reductase is a novel iron-sulfur protein. *J. Biol. Chem.* **276**, 42881–42886.

Kopriva, S., Fritzemeier, K., Wiedemann, G. and Reski, R. (2007) The putative moss 3′-phosphoadenosine phosphosulfate reductase is a novel form of adenosine 5′-phosphosulfate reductase without iron sulfur cluster. *J. Biol. Chem.* **282**, 22930–22938.

Kostanjevecki, V., Brige, A., Meyer, T.E., Cusanovich, M.A., Guisez, Y. and van Beeumen, J. (2000) A membrane-bound flavocytochrome c-sulfide dehydrogenase from the purple phototrophic sulfur bacterium *Ectothiorhodospira vacuolata*. *J. Bacteriol.* **182**, 3097–3103.

Krafft, T., Bokranz, M., Klimmek, O., Schröder, I., Fahrenholz, F., Kojro, E. and Kröger, A. (1992) Cloning and nucleotide sequence of the *psrA* gene of *Wolinella succinogenes* polysulfide reductase. *Eur. J. Biochem.* **206**, 503–510.

Krafft, T., Gross, R. and Kröger, A. (1995) The function of *Wolinella succinogenes psr* genes in electron transport with polysulfide as the terminal electron acceptor. *Eur. J. Biochem.* **230**, 601–606.

Kredich, N.M. (1992) The molecular basis for positive regulation of *cys* promoters in *Salmonella typhimurium* and *Escherichia coli*. *Mol. Microbiol.* **6**, 2747–2753.

Kredich, N.M. (1996) Biosynthesis of cysteine. In: *Escherichia coli and Salmonella typhimurium. Cellular and Molecular Biology* (F.C. Neidhardt, ed.), pp. 514–527. American Society for Microbiology, Washington D.C.

Kusai, K. and Yamanaka, T. (1973) The oxidation mechanisms of thiosulphate and sulphide in *Chlorobium thiosulfatophilum*: roles of cytochrome *c*-551 and cytochrome *c*-553. *Biochim. Biophys. Acta* **325**, 304–314.

Kusche, W.H. (1985) *Untersuchungen an Elektronentransportproteinen und zum Schwefelstoffwechsel in Ectothiorhodospiraceae*. University of Bonn, Germany: PhD Dissertation.

Lampreia, J., Pereira, A.S. and Moura, J.J.G. (1994) Adenylylsulfate reductases from sulfate-reducing bacteria. *Meth. Enzymol.* **243**, 241–260.

Larimer, F.W., Chain, P., Hauser, L., Lamerdin, J., Malfatti, S., Do, L., Land, M.L., Pelletier, D.A., Beatty, J.T., Lang, A.S., Tabita, F.R., Gibson, J.L., Hanson, T.E., Bobst, C., Torres y Torres, J.L., Peres, C., Harrison, F.H., Gibson, J. and Harwood, C.S. (2004) Complete genome sequence of the metabolically versatile photosynthetic bacterium *Rhodopseudomoans palustris*. *Nat. Biotechnol.* **22**, 55–61.

Larsen, H. (1952) On the culture and general physiology of the green sulfur bacteria. *J. Bacteriol.* **64**, 187–196.

Leguijt, T. (1993) *Photosynthetic Electron Transfer in Ectothiorhodospira*. University of Amsterdam, Netherlands: PhD Dissertation.

Leustek, T., Martin, M.N., Bick, J.A. and Davies, J.P. (2000) Pathways and regulation of sulfur metabolism revealed through molecular and genetic studies. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **51**, 141–165.

Leustek, T. and Saito, K. (1999) Sulfate transport and assimilation in plants. *Plant Physiol.* **120**, 637–643.

Leyh, T.S. (1993) The physical biochemistry and molecular genetics of sulfate activation. *Crit. Rev. Biochem. Mol. Biol.* **28**, 515–542.

Li, J. and Schiff, J.A. (1991) Purification and properties of adenosine 5′-phosphosulfate sulfotransferase from *Euglena*. *Biochem. J.* **274**, 355–360.

Liss, P. (1999) Biogeochemistry: Take the shuttle – from marine algae to atmospheric chemistry. *Science* **285**, 1217–1218.

Lloyd, J.R. (2003) Microbial reduction of metals and radionuclides. *FEMS Microbiol. Rev.* **27**, 411–425.

Lu, W.-P., Swoboda, E.P. and Kelly, D.P. (1985) Properties of the thiosulfate-oxidizing multi-enzyme system from *Thiobacillus versutus*. *Biochim. Biophys. Acta* **828**, 116–122.

Lübbe, Y. and Dahl, C. (Unpublished).

Lübbe, Y.J., Youn, H.-S., Timkovich, R. and Dahl, C. (2006) Siro(haem)amide in *Allochromatium vinosum* and relevance of DsrL and DsrN, a homolog of cobyrinic acid *a,c* diamide synthase for sulfur oxidation. *FEMS Microbiol. Lett.* **261**, 194–202.

Ma, K.S., Weiss, R. and Adams, M.W.W. (2000) Characterization of hydrogenase II from the hyperthermophilic archaeon *Pyrococcus furiosus* and assessment of its role in sulfur reduction. *J. Bacteriol.* **182**, 1864–1871.

MacRae, I.J., Segel, I.H. and Fisher, A.J. (2001) Crystal structure of ATP sulfurylase from *Penicillium chrysogenum*: Insights into the allosteric regulation of sulfate assimilation. *Biochemistry* **40**, 6795–6804.

Madigan, M.T. (1986) *Chromatium tepidum* sp. nov., a thermophilic photosynthetic bacterium of the family Chromatiaceae. *Int. J. Syst. Bacteriol.* **36**, 222–227.

Madigan, M.T. (2001) The Family Heliobacteriaceae. In: *The Prokaryotes: An Evolving Electronic Resource for the Microbiological Community* (M. Dworkin, ed), [http://link.springer-ny.com/link/service/books/10125/](http://link.springer-ny.com/link/service/books/10125/). Springer, New York.

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 191

Madigan, M.T. and Brock, T.D. (1975) Photosynthetic sulfide oxidation by Chloroflexus aurantiacus, a filamentous, photosynthetic, gliding bacterium. *J. Bacteriol.* **122**, 782–784.

Madigan, M.T. and Brock, T.D. (1977) CO₂ fixation in photosynthetically-grown Chloroflexus auranticus. *FEMS Microbiol. Lett.* **1**, 301–304.

Manske, A.K., Glaeser, J., Kuypers, M.A.M. and Overmann, J. (2005) Physiology and phylogeny of green sulfur bacteria forming a monospecific phototrophic assemblage at a depth of 100 meters in the Black Sea. *Appl. Environ. Microbiol.* **71**, 8049–8060.

Matias, P.M., Pereira, I.A.C., Soares, C.M. and Carrondo, M.A. (2005) Sulphate respiration from hydrogen in Desulfovibrio bacteria: a structural biology overview. *Prog. Biophys. Mol. Biol.* **89**, 292–329.

McCoy, J.G., Arabshahi, A., Bitto, E., Bingman, C.A., Ruzicka, F.J., Frey, P.A. and Phillips, G.N. (2006) Structure and mechanism of an ADP-glucose phosphorylase from Arabidopsis thaliana. *Biochemistry* **45**, 3154–3162.

Mendoza-Cózatl, D., Loza-Tavera, H., Hernández-Navarro, A. and Moreno-Sánchez, R. (2005) Sulfur assimilation and glutathione metabolism under cadmium stress in yeast, protists and plants. *FEMS Microbiol. Rev.* **29**, 653–671.

Meulenberg, R., Pronk, J.T., Frank, J., Hazeu, W., Bos, P. and Kuenen, J.G. (1992a) Purification and partial characterization of a thermostable trithionate hydrolase from the acidophilic sulphur oxidizer Thiobacillus acidophilus. *Eur. J. Biochem.* **209**, 367–374.

Meulenberg, R., Pronk, J.T., Hazeu, W., Bos, P. and Kuenen, J.G. (1992b) Oxidation of reduced sulphur compounds by intact cells of Thiobacillus acidophilus. *Arch. Microbiol.* **157**, 161–168.

Meulenberg, R., Pronk, J.T., Hazen, W., van Dijken, J.P., Frank, J., Bos, P. and Kuenen, J.G. (1993) Purification and partial characterization of thiosulphate dehydrogenase from Thiobacillus acidophilus. *J. Gen. Microbiol.* **139**, 2033–2039.

Meyer, B. and Kuever, J. (2007) Phylogeny of the alpha and beta subunits of the dissimilatory adenosine-5′-phosphosulfate (APS) reductase from sulfate-reducing prokaryotes – origin and evolution of the dissimilatory sulfate-reduction pathway. *Microbiology* **153**, 2026–2044.

Meyer, T.E. and Cusanovich, M.A. (2003) Discovery and characterization of electron transfer proteins in the photosynthetic bacteria. *Photosyn. Res.* **76**, 111–126.

Molitor, M., Dahl, C., Molitor, I., Schäfer, U., Speich, N., Huber, R., Deutzmann, R. and Trüper, H.G. (1998) A dissimilatory sirohaem-sulfite reductase-type protein from the hyperthermophilic archaeon Pyrobaculum islandicum. *Microbiology* **144**, 529–541.

Moran, M.A., Gonzalez, J.M. and Kiene, R.P. (2003) Linking a bacterial taxon to sulfur cycling in the sea: studies of the marine Roseobacter group. *Geomicrobiol. J.* **20**, 375–388.

Mothes, K. and Specht, W. (1934) Über den Schwefelstoffwechsel der Pflanzen. *Planta* **22**, 800–803.

Murphy, M.J., Siegel, L.M., Kamin, H. and Rosenthal, D. (1973) Reduced nicotinamide adenine dinucleotide phosphate-sulfite reductase of enterobacteria. II. Identification of a new class of heme prosthetic group: an iron-tetrahydro-porphyrin (isobacteriochlorin type) with eight carboxylic acid groups. *J. Biol. Chem.* **248**, 2801–2814.

Mussmann, M., Richter, M., Lombardot, T., Meyerdierks, A., Kuever, J., Kube, M., Glöckner, F.O. and Amann, R. (2005) Clustered genes related to sulfate respiration in uncultured prokaryotes support the theory of their concomitant horizontal transfer. *J. Bacteriol.* 187, 7126–7137.

Myers, J.D. and Kelly, D.J. (2005) A sulphite respiration system in the chemoheterotrophic human pathogen *Campylobacter jejuni*. *Microbiology* 151, 233–242.

Myers, J.M. and Myers, C.R. (2001) Role for outer membrane cytochromes OmcA and OmcB of *Shewanella putrefaciens* MR-1 in reduction of manganese dioxide. *Appl. Environ. Microbiol.* 67, 260–269.

Neerken, S. and Amesz, J. (2001) The antenna reaction center complex of heliobacteria: Composition, energy conversion and electron transfer. *Biochim. Biophys. Acta* 1507, 278–290.

Negishi, M., Pedersen, L.G., Petrotchenko, E., Shevtsov, S., Gorokhov, A., Kakuta, Y. and Pedersen, L.C. (2001) Structure and function of sulfotransferases. *Arch. Biochem. Biophys.* 390, 149–157.

Neumann, S., Wynen, A., Trüper, H.G. and Dahl, C. (2000) Characterization of the *cys* gene locus from *Allochromatium vinosum* indicates an unusual sulfate assimilation pathway. *Mol. Biol. Rep.* 27, 27–33.

Newwald, A.F., Krishnan, B.R., Brikun, I., Kulakauskas, S., Suziedelis, K., Tomcsanyi, T., Leyh, T.S. and Berg, D.E. (1992) *cysQ*, a gene needed for cysteine synthesis in *Escherichia coli* K-12 only during aerobic growth. *J. Bacteriol.* 174, 415–425.

Newton, I.L.G., Woyke, T., Auchtung, T.A., Dilly, G.F., Dutton, R.J., Fisher, M.C., Fontanez, K.M., Lau, E., Stewart, F.J., Richardson, P.M., Barry, K.W., Saunders, E., Detter, J.C., Wu, D., Eisen, J.A. and Cavanaugh, C.M. (2007) The *Calyptogena magnifica* chemoautotrophic symbiont genome. *Science* 315, 998–1000.

Numata, T., Fukai, S., Ikeuchi, Y., Suzuki, T. and Nureki, O. (2006) Structural basis for sulfur relay to RNA mediated by heterohexameric TusBCD complex. *Structure* 14, 357–366.

Ohmura, N., Kitamura, K. and Saiki, H. (1993) Selective adhesion of *Thiobacillus ferrooxidans* to pyrite. *Appl. Environ. Microbiol.* 59, 4044–4050.

Oren, A., Kessel, M. and Stackebrandt, E. (1989) *Ectothiorhodospira marismortui* sp. nov., an obligately anaerobic, moderately halophilic purple sulfur bacterium from a hypersaline sulfur spring on the shore of the Dead Sea. *Arch. Microbiol.* 151, 524–529.

Ormerod, J.G., Kimble, L.K., Nesbakken, T., Torgersen, Y.A., Woese, C.R. and Madigan, M.T. (1996) *Heliophilum fasciatum* gen. nov., sp. nov. and *Heliobacterium gestii* sp. nov.: endospore-forming heliobacteria from rice field soils. *Arch. Microbiol.* 165, 226–234.

Ostrowski, J., Wu, J.-Y., Rueger, D.C., Miller, B.E., Siegel, L.M. and Kredich, N.M. (1989) Characterization of the *cysJIH* regions of *Salmonella typhimurium* and *Escherichia coli* B. *J. Biol. Chem.* 264, 15726–15737.

Overmann, J. (2001) Green sulfur bacteria. In: *Bergey’s Manual of Systematic Bacteriology* (G.M. Garrity, ed.), Vol. 1, pp. 601–623. Williams and Wilkins, Baltimore.

Overmann, J. (2008) Ecology of phototrophic sulfur bacteria. In: *Sulfur Metabolism in Phototrophic Organisms* (R. Hell, C. Dahl, D.B. Knaff and T. Leustek, eds), *Advances in Photosynthesis and Respiration*, pp. 375–396. Springer, Dordrecht.

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 193

Overmann, J., Cypionka, H. and Pfennig, N. (1992a) An extremely low-light-adapted phototrophic sulfur bacterium from the Black Sea. *Limnol. Oceanogr.* **37**, 150–155.

Overmann, J., Fischer, U. and Pfennig, N. (1992b) A new purple sulfur bacterium from saline littoral sediments, *Thiorhodovibrio winogradskyi* gen. nov. and sp. nov. *Arch. Microbiol.* **157**, 329–335.

Overmann, J. and Schubert, K. (2002) Phototrophic consortia: model systems for symbiotic interrelations between prokaryotes. *Arch. Microbiol.* **177**, 201–208.

Parkin, T.B. and Brock, T.D. (1980) The effects of light quality on the growth of phototrophic bacteria in lakes. *Arch. Microbiol.* **125**, 19–27.

Paschinger, H., Paschinger, J. and Gaffron, H. (1974) Photochemical disproportionation of sulfur into sulfide and sulfate by *Chlorobium limicola* forma *thiosulfatophilum*. *Arch. Microbiol.* **96**, 341–351.

Pattaragulwanit, K., Brune, D.C., Trüper, H.G. and Dahl, C. (1998) Molecular genetic evidence for extracytoplasmic localization of sulfur globules in *Chromatium vinosum*. *Arch. Microbiol.* **169**, 434–444.

Peck, H.D., Jr. (1968) Energy-coupling mechanisms in chemolithotrophic bacteria. *Annu. Rev. Microbiol.* **22**, 489–518.

Pibernat, I.V. and Abella, C.A. (1996) Sulfide pulsing as the controlling factor of spinae production in *Chlorobium limicola* strain UdG 6038. *Arch. Microbiol.* **165**, 272–278.

Pickering, I.J., George, G.N., Yu, E.Y., Brune, D.C., Tuschak, C., Overmann, J., Beatty, J.T. and Prince, R.C. (2001) Analysis of sulfur biochemistry of sulfur bacteria using X-ray absorption spectroscopy. *Biochemistry* **40**, 8138–8145.

Pierson, B.K., Giovannoni, S.J., Stahl, D.A. and Castenholz, R.W. (1985) *Heliothrix oregonensis*, gen. nov., sp. nov., a phototrophic filamentous gliding bacterium containing bacteriochlorophyll *a*. *Arch. Microbiol.* **142**, 164–167.

Pires, R.H., Lourenço, A.I., Morais, F., Teixeira, M., Xavier, A.V., Saraiva, L.M. and Pereira, I.A.C. (2003) A novel membrane-bound respiratory complex from *Desulfovibrio desulfuricans* ATCC 27774. *Biochim. Biophys. Acta* **1605**, 67–82.

Pires, R.H., Venceslau, S.S., Morais, F., Teixeira, M., Xavier, A.V. and Pereira, I.A.C. (2006) Characterization of the *Desulfovibrio desulfuricans* ATCC 27774 DsrMKJOP complex—a membrane-bound redox complex involved in the sulfate respiratory pathway. *Biochemistry* **45**, 249–262.

Pittman, M.S., Robinson, H.C. and Poole, R.K. (2005) A bacterial glutathione transporter (*Escherichia coli* CydDC) exports reductant to the periplasm. *J. Biol. Chem.* **280**, 32254–32261.

Podgorsek, L. and Imhoff, J.F. (1999) Tetrathionate production by sulfur oxidizing bacteria and the role of tetrathionate in the sulfur cycle of Baltic Sea sediments. *Aquat. Microb. Ecol.* **17**, 255–265.

Pott, A.S. and Dahl, C. (1998) Sirohaem-sulfite reductase and other proteins encoded in the *dsr* locus of *Chromatium vinosum* are involved in the oxidation of intracellular sulfur. *Microbiology* **144**, 1881–1894.

Prange, A., Arzberger, I., Engemann, C., Modrow, H., Schumann, O., Trüper, H.G., Steudel, R., Dahl, C. and Holmes, J. (1999) *In situ* analysis of sulfur in the sulfur globules of phototrophic sulfur bacteria by X-ray absorption near edge spectroscopy. *Biochim. Biophys. Acta* **1428**, 446–454.

Prange, A., Chauvistre, R., Modrow, H., Holmes, J., Trüper, H.G. and Dahl, C. (2002a) Quantitative speciation of sulfur in bacterial sulfur globules: X-ray

absorption spectroscopy reveals at least three different speciations of sulfur. *Microbiology* **148**, 267–276.

Prange, A., Dahl, C., Trüper, H.G., Chauvistre, R., Modrow, H. and Hormes, J. (2002b) X-ray absorption spectroscopy of bacterial sulfur globules: a detailed reply. *Microbiology* **148**, 2268–2270.

Prange, A., Engelhardt, H., Trüper, H.G. and Dahl, C. (2004) The role of the sulfur globule proteins of *Allochromatium vinosum*: mutagenesis of the sulfur globule protein genes and expression studies by real-time RT PCR. *Arch. Microbiol.* **182**, 165–174.

Prange, A. and Modrow, H. (2002) X-ray absorption spectroscopy and its application in biological, agricultural and environmental research. *Rev. Environ. Sci. Biotechnol.* **1**, 259–276.

Pronk, J.T., Meulenberg, R., Hazeu, W., Bos, P. and Kuenen, J.G. (1990) Oxidation of reduced inorganic sulphur compounds by acidophilic thiobacilli. *FEMS Microbiol. Rev.* **75**, 293–306.

Quentmeier, A., Hellwig, P., Bardischewsky, F., Grelle, G., Kraft, R. and Friedrich, C.G. (2003) Sulfur oxidation in *Paracoccus pantotrophus*: interaction of the sulfur-binding protein SoxYZ with the dimanganese SoxB protein. *Biochem. Biophys. Res. Commun.* **312**, 1011–1018.

Quentmeier, A., Kraft, R., Kostka, S., Klockenkamper, R. and Friedrich, C.G. (2000) Characterization of a new type of sulfite dehydrogenase from *Paracoccus pantotrophus* GB17. *Arch. Microbiol.* **173**, 117–125.

Quentmeier, A. and Friedrich, C.G. (2001) The cysteine residue of the SoxY protein as the active site of protein-bound sulfur oxidation of *Paracoccus pantotrophus* GB17. *FEBS Lett.* **503**, 168–172.

Raitsimring, A.M., Kappler, U., Feng, C.J., Astashkin, A.V. and Enemark, J.H. (2005) Pulsed EPR studies of a bacterial sulfite-oxidizing enzyme with pH-invariant hyperfine interactions from exchangeable protons. *Inorg. Chem.* **44**, 7283–7285.

Ramírez, P., Guiliani, N., Valenzuela, L., Beard, S. and Jerez, C.A. (2004) Differential protein expression during growth of *Acidithiobacillus ferrooxidans* on ferrous iron, sulfur compounds, or metal sulfides. *Appl. Environ. Microbiol.* **70**, 4491–4498.

Raymond, J.C. and Sistrom, W.R. (1969) *Ectothiorhodospira halophila* – a new species of the genus *Ectothiorhodospira*. *Arch. Mikrobiol.* **69**, 121–126.

Rees, G.N., Harfoot, C.G., Janssen, P.H., Schoenborn, L., Kuever, J. and Lunsdorf, H. (2002) *Thiobaca trueperi* gen. nov., sp. nov., a phototrophic purple sulfur bacterium isolated from freshwater lake sediment. *Int. J. Syst. Evol. Microbiol.* **52**, 671–678.

Reijerse, E.J., Sommerhalter, M., Hellwig, P., Quentmeier, A., Rother, D., Laurich, C., Bothe, E., Lubitz, W. and Friedrich, C.G. (2007) The unusual redox centers of SoxXA, a novel c-type heme-enzyme essential for chemotrophic sulfur-oxidation of *Paracoccus pantotrophus*. *Biochemistry* **46**, 7804–7810.

Reinartz, M. and Dahl, C. (Unpublished).

Reinartz, M., Tschäpe, J., Brüser, T., Trüper, H.G. and Dahl, C. (1998) Sulfide oxidation in the phototrophic sulfur bacterium *Chromatium vinosum*. *Arch. Microbiol.* **170**, 59–68.

Remsen, C.C. (1978) Comparative subcellular architecture of photosynthetic bacteria. In: *The Photosynthetic Bacteria* (R.K. Clayton and W.R. Sistrom, eds), pp. 31–62. Plenum Press, New York.

Remsen, C.C. and Trüper, H.G. (1973) The fine structure of *Chromatium buderi*. *Arch. Mikrobiol.* **90**, 269–280.

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 195

Renosto, F., Martin, R.L., Borrell, J.L., Nelson, D.C. and Segel, I.H. (1991) ATP sulfurylase from trophosome tissue of *Riftia pachyptila* (hydrothermal vent tube worm). *Arch. Biochem. Biophys.* **290**, 66–78.

Rijkenberg, M.J.A., Kort, R. and Hellingwerf, K.J. (2001) *Alkalispirillum mobile* gen. nov., spec. nov., an alkaliphilic non-phototrophic member of the Ectothiorhodospiraceae. *Arch. Microbiol.* **175**, 369–375.

Rohwerder, T. and Sand, W. (2003) The sulfane sulfur of persulfides is the actual substrate of the sulfur-oxidizing enzymes from *Acidithiobacillus* and *Acidiphilium* spp. *Microbiology* **149**, 1699–1709.

Rojas, J., Gierig, M. and Tributsch, H. (1995) Sulfur colloids as temporary energy reservoirs for *Thiobacillus ferrooxidans* during pyrite oxidation. *Arch. Microbiol.* **163**, 352–356.

Romanenko, L.A., Uchino, M., Falsen, E., Zhukova, N.V., Mikhailov, V.V. and Uchimura, T. (2003) *Rheinheimera pacifica* sp. nov., a novel halotolerant bacterium isolated from deep sea water of the Pacific. *Int. J. Syst. Evol. Microbiol.* **53**, 1973–1977.

Rother, D., Heinrich, H.J., Quentmeier, A., Bardischewsky, F. and Friedrich, C.G. (2001) Novel genes of the *sox* gene cluster, mutagenesis of the flavoprotein SoxF, and evidence for a general sulfur-oxidizing system in *Paracoccus pantotrophus* GB17. *J. Bacteriol.* **183**, 4499–4508.

Rother, D., Orawski, G., Bardischewsky, F. and Friedrich, C.G. (2005) SoxRS-mediated regulation of chemotrophic sulfur oxidation in *Paracoccus pantotrophus*. *Microbiology* **151**, 1707–1716.

Roy, A.B. and Trudinger, P.A. (1970) *The Biochemistry of Inorganic Compounds of Sulfur*. Cambridge University Press, London.

Sanchez, O., Ferrera, I., Dahl, C. and Mas, J. (2001) In vivo role of APS reductase in the purple sulfur bacterium *Allochromatium vinosum*. *Arch. Microbiol.* **176**, 301–305.

Sander, J. and Dahl, C. (2008) Metabolism of inorganic sulfur compounds in purple bacteria. In: *Purple Bacteria* (C.N. Hunter, F. Daldal, M.C. Thurnauer and J.T. Beatty, eds), *Advances in Photosynthesis and Respiration*, Vol. 28, pp. 595–622. Springer, Dordrecht.

Sander, J., Engels-Schwarzlose, S. and Dahl, C. (2006) Importance of the DsrMKJOP complex for sulfur oxidation in *Allochromatium vinosum* and phylogenetic analysis of related complexes in other prokaryotes. *Arch. Microbiol.* **186**, 357–366.

Sauvé, V., Bruno, S., Berks, B.C. and Hemmings, A.M. (2007) The SoxYZ complex carries sulfur cycle intermediates on a peptide swinging arm. *J. Biol. Chem.* **282**, 23194–23204.

Schedel, M., Vanselow, M. and Trüper, H.G. (1979) Siroheme sulfite reductase from *Chromatium vinosum*. Purification and investigation of some of its molecular and catalytic properties. *Arch. Microbiol.* **121**, 29–36.

Schmidt, A. (1973) Sulfate reduction in a cell-free system of *Chlorella*. The ferredoxin-dependent reduction of a protein-bound intermediate by a thiosulfonate reductase. *Arch. Mikrobiol.* **93**, 29–52.

Schmidt, A. and Jäger, K. (1992) Open questions about sulfur metabolism in plants. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **43**, 325–349.

Schmidt, M., Priemé, A. and Stougaard, P. (2007) *Arsukibacterium ikkense* gen. nov., sp. nov., a novel alkaliphilic, enzyme-producing gamma-Proteobacterium isolated from a cold and alkaline environment in Greenland. *Syst. Appl. Microbiol.* **30**, 197–201.

Schmitt, W., Schleifer, G. and Knobloch, K. (1981) The enzymatic system thiosulfate: cytochrome *c* oxidoreductase from photolithoautotrophically grown *Chromatium vinosum*. *Arch. Microbiol.* **130**, 334–338.

Schütz, M., Maldener, I., Griesbeck, C. and Hauska, G. (1999) Sulfide-quinone reductase from *Rhodobacter capsulatus*: requirement for growth, periplasmic localization, and extension of gene sequence analysis. *J. Bacteriol.* **181**, 6516–6523.

Schütz, M., Shahak, Y., Padan, E. and Hauska, G. (1997) Sulfide-quinone reductase from *Rhodobacter capsulatus*. *J. Biol. Chem.* **272**, 9890–9894.

Schwenn, J.D. (1989) Sulphate assimilation in higher plants – a thioredoxin-dependent PAPS-reductase from spinach leaves. *Z. Naturforsch.* **C42**, 93–102.

Scott, K.M., Sievert, S.M., Abirl, F.N., Ball, L.A., Barrett, C.J., Blake, R.A., Boller, A.J., Chain, P.S.G., Clark, J.A., Davis, C.R., Detter, C., Do, K.F., Dobrinski, K.P., Faza, B.I., Fitzpatrick, K.A., Freyermuth, S.K., Harmer, T.L., Hauser, L.J., Hügler, M., Kerfeld, C.A., Klotz, M.G., Kong, W.W., Land, M., Lapidus, A., Larimer, F.W., Longo, D.L., Lucas, S., Malfatti, S.A., Massey, S.E., Martin, D.D., McCuddin, Z., Meyer, F., Moore, J.L., Ocampo, L.H., Paul, J.H., Paulsen, I.T., Reep, D.K., Ren, Q.H., Ross, R.L., Sato, P.Y., Thomas, P., Tinkham, L.E. and Zeruth, G.T. (2006) The genome of deep-sea vent chemolithoautotroph *Thiomicrospira crunogena* XCL-2. *PLoS Biology* **4**, 2196–2212.

Segel, I.H. (1993) *Enzyme Kinetics: Behaviour and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems*. Wiley-Interscience, New York.

Setya, A., Murillo, M. and Leustek, T. (1996) Sulfate reduction in higher plants: molecular evidence for a novel 5′-adenylsulfate reductase. *Proc. Natl. Acad. Sci. U.S.A.* **93**, 13383–13388.

Shahak, Y., Arieli, B., Padan, E. and Hauska, G. (1992) Sulfide quinone reductase (SQR) activity in *Chlorobium*. *FEBS Lett.* **299**, 127–130.

Siegel, L.M., Murphy, M.J. and Kamin, H. (1973) Reduced nicotinamide adenine dinucleotide phosphate-sulfite reductase of Enterobacteria. I. The *Escherichia coli* hemoflavoprotein: molecular parameters and prosthetic groups. *J. Biol. Chem.*, **248**, 251–264.

Sievert, S.M., Hügler, M., Taylor, C.D., Wirsén, C.O. (2008) Sulfur oxidation at deep sea hydrothermal vents. In: *Microbial Sulfur Metabolism* (C. Dahl, C.G. Friedrich, eds), pp. 238–258. Springer, Berlin, Heidelberg.

Smith, A.J. (1966) The role of tetrathionate in the oxidation of thiosulfate by *Chromatium* sp. strain D. *J. Gen. Microbiol.* **42**, 371–380.

Smith, A.J. and Lascelles, J. (1966) Thiosulfate metabolism and rhodanese in *Chromatium* sp. strain D. *J. Gen. Microbiol.* **42**, 357–370.

Sorokin, D.Y. and Kuenen, J.G. (2005) Haloalkaliphilic sulfur-oxidizing bacteria in soda lakes. *FEMS Microbiol. Rev.* **29**, 685–702.

Sorokin, D.Y., Kuenen, J.G. and Jetten, M.S.M. (2001) Denitrification at extremely high pH values by the alkaliphilic, obligately chemolithoautotrophic, sulfur oxidizing bacterium *Thioalkalivibrio denitrificans* strain ALJD. *Arch. Microbiol.* **175**, 94–101.

Sorokin, D.Y., Teske, A., Robertson, L.A. and Kuenen, J.G. (1999) Anaerobic oxidation of thiosulfate to tetrathionate by obligately heterotrophic bacteria, belonging to the *Pseudomonas stutzeri* group. *FEMS Microbiol. Ecol.* **30**, 113–123.

Sorokin, D.Y., Tourova, T.P., Kuznetsov, B.B., Bryantseva, I.A. and Gorlenko, V.M. (2000) *Roseinatronobacter thiooxidans* gen. nov., sp. nov., a new alkaliphilic

aerobic bacteriochlorophyll a-containing bacterium isolated from a soda lake. *Microbiology* **69**, 75–82.

Speich, N., Dahl, C., Heisig, P., Klein, A., Lottspeich, F., Stetter, K.O. and Trüper, H.G. (1994) Adenylylsulphate reductase from the sulphate-reducing archaeon *Archaeoglobus fulgidus*: cloning and characterisation of the genes and comparison of the enzyme with other iron-sulphur flavoproteins. *Microbiology* **140**, 1273–1284.

Sperling, D., Kappler, U., Wynen, A., Dahl, C. and Trüper, H.G. (1998) Dissimilatory ATP sulfurylase from the hyperthermophilic sulfate reducer *Archaeoglobus fulgidus* belongs to the group of homo-oligomeric ATP sulfurylases. *FEMS Microbiol. Lett.* **162**, 257–264.

Starynin, D.A. and Gorlenko, V.M. (1993) Sulfide-oxidizing, spore-forming heliobacteria isolated from a thermal sulfide spring. *Microbiology (English Translation of Mikrobiologiya)* **62**, 343–347.

Steinmetz, M.A. and Fischer, U. (1982) Cytochromes of the green sulfur bacterium *Chlorobium vibrioforme f. thiosulfatophilum*. Purification, characterization and sulfur metabolism. *Arch. Microbiol.* **19**, 26.

Steudel, R. (1985) Neue Entwicklungen in der Chemie des Schwefels und des Selens. *Nova Acta Leopoldina* **264**, 231–246.

Steudel, R. (1989) On the nature of the “elemental sulfur” (S⁰) produced by sulfur-oxidizing bacteria— a model for S⁰ globules. In: *Autotrophic Bacteria* (H.G. Schlegel and B. Bowien, eds), pp. 289–303. Sciene Tech Publishers, Madison, WI.

Steudel, R. (1996a) Das gelbe Element und seine erstaunliche Vielseitigkeit. *Chemie in unserer Zeit* **30**, 226–234.

Steudel, R. (1996b) Mechanism for the formation of elemental sulfur from aqueous sulfide in chemical and microbiological desulfurization processes. *Ind. Eng. Chem. Res.* **35**, 1417–1423.

Steudel, R. (2000) The chemical sulfur cycle. In: *Environmental Technologies to Treat Sulfur Pollution* (P. Lens and W. Hulshoff Pol, eds), pp. 1–31. IWA Publishing, London.

Steudel, R. (2003) Aqueous sulfur sols. In: *Elemental Sulfur and Sulfur-Rich Compounds I* (R. Steudel, ed.), pp. 153–166. Springer, Berlin.

Steudel, R. and Albertsen, A. (1999) The chemistry of aqueous sulfur sols—models for bacterial sulfur globules? In: *Biochemical Principles and Mechanisms of Biosynthesis and Biodegradation of Polymers* (A. Steinbüchel, ed.), pp. 17–26. Wiley-VCH, Weinheim.

Steudel, R. and Eckert, B. (2003) Solid sulfur allotropes. In: *Elemental Sulfur and Sulfur-Rich Compounds* (R. Steudel, ed.), pp. 1–79. Springer, Berlin.

Steudel, R., Holdt, G., Visscher, P.T. and van Gemerden, H. (1990) Search for polythionates in cultures of *Chromatium vinosum* after sulfide incubation. *Arch. Microbiol.* **155**, 432–437.

Steudel, R. and Holz, B. (1988) Detection of reactive sulfur molecules (S₆, S₇, S₉, S₁₀) in commercial sulfur, in sulfur minerals, and in sulfur metals slowly cooled to 20°C. *Z. Naturforsch.* **B43**, 581–589.

Suter, M., von Ballmoos, P., Kopriva, S., den Camp, R.O., Schaller, J., Kuhlemeier, C., Schürmann, P. and Brunold, C. (2000) Adenosine 5′-phosphosulfate sulfotransferase and adenosine 5′-phosphosulfate reductase are identical enzymes. *J. Biol. Chem.* **275**, 930–936.

Suzuki, H., Koyanagi, T., Izuka, S., Onishi, A. and Kumagai, H. (2005) The *yliA*, *-B*, *-C*, and *-D* genes of *Escherichia coli* K-12 encode a novel glutathione importer with an ATP-binding cassette. *J. Bacteriol.* 187, 5861–5867.

Tabita, F.R. (1999) Microbial ribulose 1,5-bisphosphate carboxylase/oxygenase: A different perspective. *Photosyn. Res.* 60, 1–28.

Takeuchi, T.L. and Suzuki, I. (1997) Cell hydrophobicity and sulfur adhesion of *Thiobacillus thiooxidans*. *Appl. Environ. Microbiol.* 63, 2058–2061.

Theissen, U., Hoffmeister, M., Grieshaber, M. and Martin, W. (2003) Single eubacterial origin of eukaryotic sulfide:quinone oxidoreductase, a mitochondrial enzyme conserved from the early evolution of eukaryotes during anoxic and sulfidic times. *Mol. Biol. Evol.* 20, 1564–1574.

Then, J. (1984) *Beiträge zur Sulfidoxidation durch Ectothiorhodospira abdelmalekii und Ectothiorhodospira halochloris*. University of Bonn, Germany: PhD Dissertation.

Then, J. and Trüper, H.G. (1981) The role of thiosulfate in sulfur metabolism of *Rhodopseudomonas globiformis*. *Arch. Microbiol.* 130, 143–146.

Then, J. and Trüper, H.G. (1983) Sulfide oxidation in *Ectothiorhodospira abdelmalekii*. Evidence for the catalytic role of cytochrome c-551. *Arch. Microbiol.* 135, 254–258.

Then, J. and Trüper, H.G. (1984) Utilization of sulfide and elemental sulfur by *Ectothiorhodospira halochloris*. *Arch. Microbiol.* 139, 295–298.

Tourova, T.P., Spiridonova, E.M., Slobodova, N.V., Boulygina, E.S., Keppen, O.I., Kuznetsov, B.B. and Ivanovsky, R.N. (2006) Phylogeny of anoxygenic filamentous phototrophic bacteria of the family Oscillochloridaceae analyses as inferred from comparative analyses of the *rrs*, *cbbL*, and *nifH* genes. *Microbiology* 75, 192–200.

Trüper, H.G. (1968) *Ectothiorhodospira mobilis* Pelsh, a photosynthetic sulfur bacterium depositing sulfur outside the cells. *J. Bacteriol.* 95, 1910–1920.

Trüper, H.G. (1978) Sulfur metabolism. In: *The Photosynthetic Bacteria* (R.K. Clayton and W.R. Sistrom, eds), pp. 677–690. Plenum, New York.

Trüper, H.G. (1984) Phototrophic bacteria and their sulfur metabolism. In: *Sulfur, Its Significance for Chemistry, for the Geo-, Bio-, and Cosmosphere and Technology* (A. Müller and B. Krebs, eds), pp. 367–382. Elsevier Science Publishers B.V., Amsterdam.

Trüper, H.G. and Fischer, U. (1982) Anaerobic oxidation of sulphur compounds as electron donors for bacterial photosynthesis. *Philos. Trans. R. Soc. Lond. B* 298, 529–542.

Trüper, H.G. and Genovese, S. (1968) Characterization of photosynthetic sulfur bacteria causing red water in Lake Faro (Messina, Italy). *Limnol. Oceanogr.* 13, 225–232.

Trüper, H.G. and Hathaway, J.C. (1967) Orthorhombic sulphur formed by photosynthetic sulphur bacteria. *Nature* 215, 435–436.

Trüper, H.G., Lorenz, C., Schedel, M. and Steinmetz, M. (1988) Metabolism of thiosulfate in *Chlorobium*. In: *Green Photosynthetic Bacteria* (J.M. Olson, J.G. Ormerod, J. Amesz, E. Stackebrandt and H.G. Trüper, eds), pp. 189–200. Plenum Press, New York.

Trüper, H.G. and Pfennig, N. (1966) Sulphur metabolism in Thiorhodaceae. III. Storage and turnover of thiosulphate sulphur in *Thiocapsa floridana* and *Chromatium* species. *Antonie van Leeuwenhoek Int. J. Mol. Microbiol.* 32, 261–276.

SULFUR METABOLISM IN PHOTOTROPHIC SULFUR BACTERIA 199

Trüper, H.G. and Rogers, L.A. (1971) Purification and properties of adenylyl sulfate reductase from the phototrophic sulfur bacterium, *Thiocapsa roseopersicina*. *J. Bacteriol.* **108**, 1112–1121.

Urich, T., Gomes, C.M., Kletzin, A. and Frazao, C. (2006) X-ray structure of a self-compartmentalizing sulfur cycle metalloenzyme. *Science* **311**, 996–1000.

van Beeumen, J.J., Demol, H., Samyn, B., Bartsch, R.G., Meyer, T.E., Dolata, M.M. and Cusanovich, M.A. (1991) Covalent structure of the diheme cytochrome subunit and amino-terminal sequence of the flavoprotein subunit of flavocytochrome *c* from *Chromatium vinosum*. *J. Biol. Chem.* **266**, 12921–12931.

van der Ploeg, J.R., Weiss, M.A., Saller, E., Nashimoto, H., Saito, N., Kertesz, M.A. and Leisinger, T. (1996) Identification of sulfate starvation-regulated genes in *Escherichia coli*: a gene cluster involved in the utilization of taurine as a sulfur source. *J. Bacteriol.* **178**, 5438–5446.

van der Ploeg, J.R., Cummings, N.J., Leisinger, T. and Connerton, I.F. (1998) *Bacillus subtilis* genes for the utilization of sulfur from aliphatic sulfonates. *Microbiology* **144**, 2555–2561.

van der Ploeg, J.R., Eichhorn, E. and Leisinger, T. (2001) Sulfonate-sulfur metabolism and its regulation in *Escherichia coli*. *Arch. Microbiol.* **176**, 1–8.

van Gemerden, H. (1968) On the ATP generation by *Chromatium* in the dark. *Arch. Mikrobiol.* **64**, 118–124.

van Gemerden, H. (1986) Production of elemental sulfur by green and purple sulfur bacteria. *Arch. Microbiol.* **146**, 52–56.

van Gemerden, H. (1987) Competition between purple sulfur bacteria and green sulfur bacteria: role of sulfide, sulfur and polysulfides. In: *Ecology of Photosynthetic Prokaryotes with Special Reference to Meromictic Lakes and Coastal Lagoons* (T. Lindholm, ed.), pp. 13–27. Abo Academy Press, Abo.

van Gemerden, H. and Mas, J. (1995) Ecology of purple sulfur bacteria. In: *Anoxygenic Photosynthetic Bacteria*. (R.E. Blankenship, M.T., Madigan, C.E. Bauer, eds), *Advances in Photosynthesis*. Vol. 2, pp. 49–85. Kluwer Academic Publishers, Dordrecht.

van Niel, C.B. (1931) On the morphology and physiology of the purple and green sulfur bacteria. *Arch. Mikrobiol.* **3**, 1–112.

Vermeglio, A., Li, J., Schoepp-Cothenet, B., Pratt, N. and Knaff, D.B. (2002) The role of high-potential iron protein and cytochrome c(8) as alternative electron donors to the reaction center of *Chromatium vinosum*. *Biochemistry* **41**, 8868–8875.

Verté, F., Kostanjevecki, V., de Smet, L., Meyer, T.E., Cusanovich, M.A. and van Beeumen, J.J. (2002) Identification of a thiosulfate utilization gene cluster from the green phototrophic bacterium *Chlorobium limicola*. *Biochemistry* **41**, 2932–2945.

Vidmar, J.J., Tagmount, A., Cathala, N., Touraine, B. and Davidian, J.C.E. (2000) Cloning and characterization of a root specific high-affinity sulfate transporter from *Arabidopsis thaliana*. *FEBS Lett.* **475**, 65–69.

Vila, X., Guyoneaud, R., Cristina, X.P., Figueras, J.B. and Abella, C.A. (2002) Green sulfur bacteria from hypersaline Chiprana Lake (Monegros, Spain): habitat description and phylogenetic relationship of isolated strains. *Photosyn. Res.* **71**, 165–172.

Visscher, P.T. and Taylor, B.F. (1993) Organic thiols as organolithotrophic substrates for growth of phototrophic bacteria. *Appl. Environ. Microbiol.* **59**, 93–96.

Visscher, P.T., Nijburg, J.W. and van Gemerden, H. (1990) Polysulfide utilization by *Thiocapsa roseopersicina*. Arch. Microbiol. **155**, 75–81.

Visscher, P.T. and van Gemerden, H. (1991) Photoautotrophic growth of *Thiocapsa roseopersicina* on dimethyl sulfide. FEMS Microbiol. Lett. **81**, 247–250.

Visser, J.M., de Jong, G.A.H., Robertson, L.A. and Kuenen, J.G. (1996) Purification and characterization of a periplasmic thiosulfate dehydrogenase from the obligately autotrophic *Thiobacillus sp.* W5. Arch. Microbiol. **166**, 372–378.

Vogl, K., Glaeser, J., Pfannes, K.R., Wanner, G. and Overmann, R. (2006) *Chlorobium chlorochromatii* sp. nov., a symbiotic green sulfur bacterium isolated from the phototrophic consortium “*Chlorochromatium aggregatum*”. Arch. Microbiol. **185**, 363–372.

Vogler, K.G. and Umbreit, W.W. (1941) The necessity for direct contact in sulphur oxidation by *Thiobacillus thiooxidans*. Soil Sci. **51**, 331–337.

Ward, D.M., Ferris, M.J., Nold, S.C. and Bateson, M.M. (1998) A natural view of microbial biodiversity within hot spring cyanobacterial mat communities. Mol. Biol. Rev. **62**, 1353–1370.

Winogradsky, S.N. (1887) Über Schwefelbakterien. Bot. Ztg. **45**, 489–508.

Wynen, A., Dahl, C. and Trüper, H.G. (Unpublished).

Wynen, A., Trüper, H.G. and Dahl, C. (Unpublished).

Yakimov, M.M., Guiliano, L., Chernikova, T.N., Gentile, G., Abraham, W.R., Lunsdorf, H., Timmis, K.N. and Golyshin, P.N. (2001) *Alcalilimnicola halodurans* gen. nov., sp. nov., an alkaliphilic, moderately halophilic and extremely halotolerant bacterium, isolated from sediments of soda-depositing Lake Natron, East Africa Rift Valley. Int. J. Syst. Evol. Microbiol. **51**, 2133–2143.

Yu, Z., Lansdon, E.B., Segel, I.H. and Fisher, A.J. (2006) Crystal structure of the bifunctional ATP sulfurylase-APS kinase from the chemolithotrophic thermophile *Aquifex aeolicus*. J. Mol. Biol. **365**, 732–743.

Yurkov, V.V. (2006) Aerobic phototrophic proteobacteria. In: *The Prokaryotes* (M. Dworkin, S. Falkow, E. Rosenberg, K.-H. Schleifer and E. Stackebrandt, eds), Vol. 5, pp. 562–584. Springer, New York.

Yurkov, V.V., Krasil’nikova, E.N. and Gorlenko, V.M. (1994) Thiosulfate metabolism in the aerobic bacteriochlorophyll-a-containing bacteria *Erythromicrobium hydrolyticum* and *Roseococcus thiosulfatophilus*. Microbiology **63**, 91–94.

Zaar, A., Fuchs, G., Golecki, J.R. and Overmann, J. (2003) A new purple sulfur bacterium isolated from a littoral microbial mat, *Thiorhodococcus drewsii* sp. nov. Arch. Microbiol. **179**, 174–183.

Zuber, H. and Cogdell, R.J. (1995) Structure and organization of purple bacterial antenna complexes. In: *Anoxygenic Photosynthetic Bacteria* (R.E. Blankenship, M.T. Madigan and C.E. Bauer, eds), Advances in Photosynthesis, Vol. 2, pp. 315–348. Kluwer Academic Publishers, Dordrecht.
