
Mechanism for Coordinated RNA Packaging and Genome Replication by Rotavirus Polymerase VP1

Xiaohui Lu,$^{1,5}$ Sarah M. McDonald,$^{2}$ M. Alejandra Tortorici,$^{2,6}$ Yizhi Jane Tao,$^{1,7}$ Rodrigo Vasquez-Del Carpio,$^{2,8}$ Max L. Nibert,$^{3}$ John T. Patton,$^{2}$ and Stephen C. Harrison$^{1,4,*}$

$^{1}$Laboratory of Molecular Medicine, Children’s Hospital and Harvard Medical School, Boston, MA 02115, USA  
$^{2}$Laboratory of Infectious Diseases, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892, USA  
$^{3}$Department of Microbiology and Molecular Genetics, Harvard Medical School, Boston, MA 02115, USA  
$^{4}$Howard Hughes Medical Institute, Children’s Hospital, Boston, MA 02115, USA  
$^{5}$Present address: Optimal Decisions Group, Cambridge, MA 02142, USA  
$^{6}$Present address: Unité de Virologie Structurale, Département de Virologie, Institut Pasteur, 75724 Paris Cedex 15, France  
$^{7}$Present address: Department of Biochemistry and Cell Biology, Rice University, Houston, TX 77251, USA  
$^{8}$Present address: Department of Molecular Physiology and Biophysics, Mount Sinai School of Medicine, New York, NY 10029, USA  

*Correspondence: harrison@crystal.harvard.edu  
DOI 10.1016/j.str.2008.09.006


**SUMMARY**

Rotavirus RNA-dependent RNA polymerase VP1 catalyzes RNA synthesis within a subviral particle. This activity depends on core shell protein VP2. A conserved sequence at the $3'$ end of plus-strand RNA templates is important for polymerase association and genome replication. We have determined the structure of VP1 at 2.9 Å resolution, as apoenzyme and in complex with RNA. The cage-like enzyme is similar to reovirus λ3, with four tunnels leading to or from a central, catalytic cavity. A distinguishing characteristic of VP1 is specific recognition, by conserved features of the template-entry channel, of four bases, UGUG, in the conserved $3'$ sequence. Well-defined interactions with these bases position the RNA so that its $3'$ end overshoots the initiating register, producing a stable but catalytically inactive complex. We propose that specific $3'$ end recognition selects rotavirus RNA for packaging and that VP2 activates the autoinhibited VP1/RNA complex to coordinate packaging and genome replication.

**INTRODUCTION**

Viral RNA-dependent RNA polymerases (RdRPs) share a common catalytic mechanism and exhibit highly conserved architectures, sequence motifs, and catalytic residues (O’Reilly and Kao, 1998). Despite their similarities, individual viral RdRPs differ in the ways they engage RNA templates, prime RNA synthesis, and coordinate their activities with other processes in the infected cell. RdRPs of segmented double-stranded RNA (dsRNA) viruses of the Reoviridae family catalyze RNA synthesis while contained within subviral particles (Lawton et al., 2000). The structures and functions of Reoviridae RdRPs are expected to reflect this particle-associated RNA synthesis mechanism (Butcher et al., 2001; Tao et al., 2002).

Rotaviruses, members of the Reoviridae family, are nonenveloped particles with three concentric protein layers surrounding 11 segments of dsRNA (Estes and Kapikian, 2007; Prasad et al., 1996). The outermost layer is shed during cell entry, resulting in a transcriptionally active, double-layered particle (DLP). The core of the DLP is built from 60 asymmetric dimers of shell protein VP2, which forms the DLP inner layer (Lawton et al., 1997; Prasad et al., 1996; Yeager et al., 1990). Copies of VP1, the viral RdRP, and VP3, the RNA capping enzyme, are tethered within the core, near each fivefold axis (Chen et al., 1999; Liu et al., 1992; Pizarro et al., 1991; Valenzuela et al., 1991). Using dsRNA genome segments as templates, the tethered VP1-VP3 complexes synthesize capped, nonpolyadenylated, plus-strand (+RNA) molecules that are extruded from the DLP into the cytoplasm of the infected cell (Lawton et al., 1997, 2000; Prasad et al., 1996). There, the +RNAs function as mRNAs for translation and, after association with VP1 and incorporation into cores (see below), as templates for minus-strand (−RNA) synthesis and formation of genomic dsRNA.

Studies of rotavirus replication intermediates (RI) suggest that core assembly is initiated by the interaction of VP1 and VP3 with +RNA, forming a pre-core RI that lacks polymerase activity (Gallegos and Patton, 1989; Patton, 1996). VP2 multimers then interact with the pre-core RI to form a core RI, which can initiate −RNA synthesis to produce dsRNA (Gallegos and Patton, 1989; Patton and Gallegos, 1990). The VP2-dependent activity of VP1 coordinates genome replication and RNA packaging (Estes and Kapikian, 2007), preventing the formation of naked dsRNAs in the infected cell (Labbe et al., 1994; Patton, 1996; Zeng et al., 1998).

Experiments with recombinant proteins have characterized the requirements for initiation of rotavirus dsRNA synthesis. Formation of a −RNA initiation complex in vitro is a salt-sensitive process requiring VP1, VP2, GTP, Mg²⁺, and a +RNA template (Chen and Patton, 2000; Tortorici et al., 2003). VP1 interacts with +RNA in the absence of VP2, demonstrating that the role

Structure

Rotavirus RNA Polymerase VP1

of VP2 extends beyond assisting in recruitment of template RNA to the polymerase (Chen and Patton, 2000; Patton, 1996; Tortorici et al., 2003). A cis-acting +RNA element, the 3′ consensus sequence (3′CS+: 5′-UGUGACC-3′), is critical for template recognition by VP1. The UGUG tetranucleotide of the 3′CS+ is a polymerase recognition signal that mediates high-affinity interactions with VP1 (Chen et al., 2001; Tortorici et al., 2003). The terminal ACC segment of the 3′CS+ is important for initiating minus-strand synthesis, but it is dispensable for specific VP1 binding.

Once dsRNA synthesis is complete, the polymerase switches to a transcriptional mode, with the –RNA as template. During transcription, VP1 initiates +RNA synthesis templated from the 3′ end of –RNA (3′CS–: 5′-[A/U]7CC-3′); thus, the RdRP can clearly function on non-UGUG-containing templates. Whereas high-affinity binding of VP1 to the 3′CS+ UGUG nucleotides is apparently critical for pre-core RI formation and for initiation of dsRNA synthesis, it may not be so critical during initiation of transcription, for which polymerase and template are both sequestered within the confines of the core.

How does VP1 engage the UGUG nucleotides of the 3′CS+ and what is the significance of the resulting interactions during dsRNA synthesis? We have determined the atomic structures of the VP1 apoenzyme and of its complexes with various RNA oligonucleotides. The structures of the VP1/RNA complexes show that VP1 has sequence-specific interactions with the UGUG bases of the 3′CS+ and that these contacts stabilize the template in the catalytic site, one nucleotide past the initiation register. This autoinhibited VP1/+RNA complex must undergo a structural reorganization following engagement by the core shell protein VP2, leading to activation of the RdRP. The results presented here thus provide a mechanism for coordination of rotavirus RNA packaging and genome replication during infection.

## RESULTS

### Structure of the VP1 Apoenzyme

Recombinant VP1 with a hexahistidine (His) tag fused at either its N or C terminus (nVP1 and cVP1, respectively) was purified to >95% homogeneity as a soluble, monomeric, 125 kDa protein. Whereas nVP1 catalyzed only low levels of dsRNA synthesis in vitro, the activity of cVP1 was like that of untagged recombinant VP1 with an incorporation rate of ~180 nt/min (see Figure S1 available online). Because of this robust activity, cVP1 was used for structural analyses. We obtained crystals of cVP1 (hereafter referred to as VP1) using low ionic strength crystallization conditions, similar to those used in assaying VP1 in vitro for polymerase activity. The space group of the apoenzyme crystals was P2₁2₁2₁, a = 77.7, b = 112.0, c = 142.2, with one molecule per asymmetric unit. The VP1 structure was determined from data extending to a minimum Bragg spacing of 2.9 Å. Experimental phases were obtained from multiwavelength anomalous dispersion (MAD) using crystals of selenomethionine (SeMet)-substituted VP1 (Table 1). Experimental maps, improved by density modification, allowed unambiguous placement of 90% of the VP1 polypeptide chain. The final, refined atomic model of VP1 contains 1073 out of 1095 amino acid residues. The missing regions include the N-terminal methionine, two flexible loops (amino acids [aa] 18–22 and 346–358), and the C-terminal His tag.

VP1 is a compact, globular protein, ~70 Å in diameter. There are three distinct domains: (1) an N-terminal domain (aa 1–332), (2) a polymerase domain comprising fingers, palm, and thumb subdomains (aa 333–778), and (3) a C-terminal “bracelet” domain (aa 779–1089) (Figure 1). Together, the N- and C-terminal domains sandwich most of the polymerase domain, creating a cage structure with the catalytic region located within a largely hollow center. The structure of VP1 is very similar to that of the reovirus RdRP λ3 (Tao et al., 2002). Comparison of VP1 and λ3 reveals shared features, as well as features unique to VP1 (Figure S2). As in λ3, four tunnels lead to the VP1 catalytic center (Figures 1B and 1C); comparison with the λ3 elongation complexes allows the functions of these tunnels to be assigned (Figure 1C).

The VP1 polymerase domain includes the six canonical motifs A–F shared by RdRPs (Figure S3B) and resembles the polymerase domains of HIV reverse transcriptase (Kohlstaedt et al., 1992), hepatitis C virus NS5B (Ago et al., 1999; Bressanelli et al., 1999; Lesburg et al., 1999), and bacteriophage φ6 P2 (Butcher et al., 2001), in addition to reovirus λ3 (Figure S2) (Tao et al., 2002). The palm subdomain (aa 489–523, 596–685) is a four-stranded, antiparallel β sheet supported by three α helices (Figure 1D). It includes the conserved acidic residues of the active site (Asp520 and Asp525 of motif A; Asp631 and Asp632 of motif C) (Doublie and Ellenberger, 1998; Joyce and Steitz, 1994). At the junction of the palm and thumb subdomains, a small sheet with a β turn forms a substructure found in other polymerases and known as the “primer grip” (Jacob-Molina et al., 1993) (Figures S2C and S2D). There is also, as in λ3 (aa 557–567), a so-called priming loop, a flexible structural element that binds the triphosphates of the priming nucleotide during initiation. The priming loop is an exception to the similarities of the palm subdomains of polymerases from many different kinds of viruses. In VP1, the putative priming loop (aa 489–499) bends away from the nucleotide-binding site by about 90° (Figures S2E and S2F), leaving it in a retracted (down) state incapable of supporting an initiation nucleotide in the priming (P) site.

The VP1 fingers subdomain (aa 333–488, 524–595) lacks a four-stranded β sheet that protrudes from the surface of the λ3 fingers (λ3 aa 400–419, 617–643) (Figure S2D). Moreover, the structural elements of the VP1 fingers that comprise one side of the template entry tunnel (VP1 aa 395–420; λ3 aa 455–491) are somewhat different in their three-dimensional arrangement from those in λ3, consistent with the likely role of these segments in UGUG recognition (see below). The VP1 thumb subdomain (aa 686–778), like that of λ3, includes a β strand followed by three α helices. A loop at the tip of the VP1 thumb (aa 697–700) interacts with the tip of the fingers, enclosing the catalytic site on the palm and creating a constriction in the template entry tunnel (Figure 1D). The fingers-thumb contact appears to maintain VP1 in a closed conformation even in its resting state. A β ribbon of the fingers subdomain (aa 445–466) reaches beyond the thumb interface. This structure, representing motif F, includes three arginine residues (Arg452, Arg457, and Arg460) that project toward the binding site for the incoming nucleotide (N site) (Figure S2E). In the case of λ3, similar motif F arginine residues act with conserved aspartic acid residues of the palm subdomain, and with residues of the extended priming

Table 1. Data Collection Statistics for Native Structure Determination

| Data Set | Native | SeMet Derivative 1 | SeMet Derivative 2 | SeMet Derivative 3 |
| --- | --- | --- | --- | --- |
|  |  | Peak<sup>a</sup> | Inflection | Remote | Peak | Inflection | Remote |
| Wavelength (Å) | 1.1808 | 0.9794 | 0.9797 | 0.9794 | 0.9797 | 0.9641 | 0.9794 | 0.9641 |
| Number of unique reflections | 27,999 | 33,959 | 25,264 | 30,517 | 29,663 | 24,300 | 22,928 | 22,768 |
| Completeness (%)<sup>b</sup> | 99.9 (99.6) | 84.3 (70.6) | 88.5 (78.3) | 92.8 (71.8) | 89.8 (61.5) | 94.0 (81.0) | 65.1 (53.7) | 73.3 (70.0) | 67.3 (54.0) |
| Resolution (Å) | 30–2.9 | 50–3.2 | 50–3.6 | 50–3.4 | 50–3.4 |
| I/σ<sup>b</sup> | 25.0 (3.5) | 7.9 (2.2) | 6.7 (2.4) | 4.8 (1.2) | 5.1 (1.1) | 5.5 (1.5) | 4.8 (1.3) | 5.1 (1.5) | 4.8 (1.3) |
| R<sub>merge</sub><sup>b,c</sup> | 0.055 (41.0) | 0.083 (.39) | 0.098 (.36) | 0.167 (.64) | 0.161 (.67) | 0.141 (.55) | 0.125 (.51) | 0.135 (.52) | 0.130 (.50) |
| Isomorphous R<sub>cullis</sub><sup>d</sup> (centric/acentric) | 0.76/0.75 | - | 0.95/0.94 | 0.96/0.91 | 0.93/0.90 | 1.02/0.98 | 0.92/0.91 | 0.92/0.91 | 0.91/0.90 |
| Anomalous R<sub>cullis</sub><sup>d</sup> (acentric)<sup>d</sup> | - | 0.79 | 0.96 | 0.87 | 0.97 | 0.96 | 0.93 | 0.99 | 0.96 |
| Isomorphous phasing power<sup>e</sup> (centric/acentric) | 0.82/0.88 | - | 0.29/0.31 | - | 0.33/0.33 | 0.24/0.26 | - | 0.21/0.24 | 0.22/0.25 |
| Anomalous phasing power<sup>e</sup> (acentric) | - | 1.15 | 0.5 | 0.73 | 0.41 | 0.44 | 0.58 | 0.32 | 0.36 |
| Figure of merit (centric/acentric) |  |  |  |  |  |  | 0.20/0.28 |
| Refinement statistics |  |  |  |  |  |  | 50–2.9 |
| R |  |  |  |  |  |  | 23.3% for 24,785 reflections |
| R<sub>free</sub> |  |  |  |  |  |  | 28.6% for 2,136 reflections |
| Rms bond length, bond angles |  |  |  |  |  |  | 0.0067 Å, 1.14° |

<sup>a</sup> This data set was used as the reference data set in phase refinement using SHARP.

<sup>b</sup> Numbers in parentheses are for the highest-resolution bins.

<sup>c</sup> $R_{\text{merge}} = \sum_h \sum_i |I_{hi}| - \langle I_h \rangle / \sum_h \sum_i |I_{hi}|$.

<sup>d</sup> $R_{\text{cullis}} = \sum_h ||F_P(h) - F_{PH}(h)| - |F_H^{\text{calc}}(h)|| / \sum_h |F_P(h) - F_{PH}(h)|$, where $F_P$, $F_{PH}$, and $F_H$ are the amplitudes of the native, SeMet derivatives, and Se atoms, respectively.

<sup>e</sup> Phasing power = rms($F_H/E$), where $E$ is the lack of closure error.

Structure

Rotavirus RNA Polymerase VP1

Figure 1. Structure of the VP1 Apoenzyme

(A and B) Ribbon diagram (A) and surface rendering (B) of the complete VP1 polypeptide chain. The N-terminal domain is in yellow, the C-terminal (bracelet) domain is in pink, and the C-terminal plug is in cyan. The conventionally designated subdomains of the polymerase domain are in light blue (fingers), red (palm), and green (thumb).

(C) Sagittal cutaway of the image in (B), after rotation to the left by 90°, showing the four tunnels extending into the central cavity.

(D–F) Ribbon diagrams of the VP1 domains: polymerase, N-terminal, and C-terminal domains.

density in the template entry tunnel, and we could position individual nucleotides unambiguously (Figure 3; Figure S4). A network of ten hydrogen bonds to the phosphates and ribose rings and eight hydrogen bonds to the bases holds the oligonucleotide in position (Figure 4A). VP1 residues contributing to this network are predominantly those of the fingers subdomain, but residues of the N-terminal domain also contribute. As in the various RNA oligonucleotide complexes of reovirus λ3, RNA binding produces only subtle conformational shifts in VP1. A loop of the fingers subdomain (aa 397–404) retracts from the central cavity, and Ser398 and Ser401 of the retracted loop form hydrogen bonds with the backbone of the RNA (Figure 4C). In addition, an antiparallel β hairpin (Pβ1–2: aa 403–417) and an α helix (Pα6: aa 420–430) connected to the loop has moved away from the active site. Extensive interactions between U7 of the 3′CS+ and Phe416 fix the position of the β hairpin (Figure 4C).

Superposition of the VP1/3′CS+ complex and the λ3 initiation complex aligns C2 and A3 of the rotavirus 3′CS+ oligonucleotide with C1 and G2 of the reovirus oligonucleotide 5′-AUUAGC-3′ (Figure 5). Thus, C1 of the 3′CS+ overshoots the register for initiation by one nucleotide, presenting C2 and A3 in positions opposite the P and N sites, respectively. Peptide carbonyls of VP1 residues Gly592, Glu593, and Lys594 accept hydrogen bonds from the 2′-OH groups of nucleotides C2 and A3, consistent with the preference of this enzyme for RNA templates

Structure 16, 1678–1688, November 12, 2008 ©2008 Elsevier Ltd All rights reserved 1681

Structure

Rotavirus RNA Polymerase VP1

A
![Image Description](image1)

B
![Image Description](image2)

C
![Image Description](image3)

D
![Image Description](image4)

Figure 2. Polymerase Activity of VP1 Plug Mutants

(A) Sagittal cutaway of a surface rendering of VP1, showing the positions of the C-terminal plug in the –RNA/dsRNA exit tunnel. The plugs for the cVP1 apoenzyme (orange), the cVP1/3′CS+ complex (green), and the nVP1 apoenzyme (cyan) are superimposed. A 3′CS+ oligonucleotide is shown in the template entry tunnel, and surfaces of motif F (purple) and the priming loop (gold) are colored for reference.

(B) Same image as in (A), but showing only the locations of the cVP1 plug and the connecting α helix in the C-terminal bracelet (dark blue). The region deleted in each VP1 mutant is indicated by color: 1079 (turquoise), 1074 (orange), and 1059 (dark blue).

(C) VP1 proteins were purified using differential centrifugation and analyzed for relative purity by SDS-PAGE and Coomassie blue staining. Wild-type, His-tagged VP1 (cVP1), and untagged (VP1) served as controls for assaying the untagged plug mutants (1059, 1074, and 1079).

(D) In vitro dsRNA synthesis. Reaction mixtures contained either wild-type or mutant VP1 and gene 8 +RNA, in the presence or absence of VP2. Radiolabeled dsRNA products were resolved by SDS-PAGE and detected by autoradiography.

forms stacking interactions with the phenyl ring of Phe416. Such well-defined, base-specific interactions distinguish the rotavirus VP1/3′CS+ complex from those of other RdRPs studied to date.

To compare the interaction of VP1 with the 3′ end of its +RNA to that of its –RNA, VP1 apoenzyme crystals were soaked with an oligonucleotide representing the 3′CS– (5′-AAAAGCC-3′) (Figure 3A; Figure S4). A network of nine hydrogen bonds, all mediated by residues of the fingers subdomain, holds this oligonucleotide in place (Figure 4B). Seven of these same residues (Ser398, Ser401, Lys419, Lys420, Gly592, Lys594, and Lys597) are also involved in anchoring the 3′CS+ into the template entry tunnel. Unlike the UGUG portion of the 3′CS+, the AAAA portion of the 3′CS– is not recognized specifically by the polymerase. The A4 and A5 residues are the only ordered components of the AAAA stretch, and they adopt orientations different from the corresponding bases (G4 and U5) of the 3′CS+ (Figure 4B; Figure S4). Moreover, the only potentially base-specific interaction links the ε-amino group of Lys597 and O² of the C2 base. Difference maps show that, like the 3′CS+, the 3′CS– oligonucleotide is in a one-nucleotide overshot register (Figure 3A; Figure S4). Results from soaking other oligonucleotides (e.g., 5′-GGCUUU-3′) into VP1 crystals show that they bind in-register (Figure S4), excluding the possibility that all RNAs that move into the template entry channel are stabilized in an overshot position.

The 3′CS+ sequences of group A rotaviruses (5′-UGUGACC-3′) and group C rotaviruses (5′-UGUGGCU-3′) share the UGUG motif recognized by SA11 VP1. The VP1 amino acid residues that interact with the 3′CS in the template channel are conserved not only in VP1s of essentially all group A viruses but also in VP1s

of group C, which otherwise have only about 50% sequence identity with those of group A. Genomic segments from group B rotaviruses have a different four-base consensus sequence (5′-ACCC-3′) at the 3′ termini of their positive strands; the corresponding recognition residues of group B VP1s are also less conserved, perhaps corresponding to an alternative specificity. These patterns of conservation are consistent with our conclusion that UGUG recognition defines a mechanistically significant state of the polymerase-template complex.

UGUG Recognition by VP1 Defines Template Overshoot

To examine directly the contribution of UGUG recognition in the template entry channel to the overshoot register of the 3′CS+, apoenzyme crystals were soaked with either (1) an oligonucleotide containing an insertion of an adenine between C2 and A3 (5′-UGUGAAC-3′) or (2) an oligonucleotide with a deletion of A3 (5′-UGUGC-3′) (Table 2). The A-inserted oligonucleotide reflects the 3′ end of some rotavirus gene 5 +RNAs (strain SA11); this variant 3′CS+ is therefore biologically functional. Density difference maps show that the UGUGA portions of the A-inserted oligonucleotide (5′-UGUGAAC-3′) and the 3′CS+ oligonucleotide are closely aligned in the template entry tunnel, revealing a similar pattern of hydrogen bonding with both templates (Figure 3B; Figure S4). In this arrangement, the extra A and the C2 residues of the A-inserted oligonucleotide end up occupying the same positions as do the C1 and C2 residues of the 3′CS+ oligonucleotide, thereby leaving the 3′ end of the longer template fragment displaced by two positions beyond the initiation register. Difference maps show that the longer oligonucleotide has an

1682 Structure 16, 1678–1688, November 12, 2008 ©2008 Elsevier Ltd All rights reserved

Structure

Rotavirus RNA Polymerase VP1

Table 2. Data Collection Statistics for Structure Determination of RNA Oligonucleotide/VP1 Complexes

| Experiment | Apoenzyme | 0.5 mM 3′CS+ and | 0.5 mM 3′CS+ | 0.5 mM 3′CS+ | 0.5 mM 5′CS+ | 0.5 mM 3′CS− |
| --- | --- | --- | --- | --- | --- | --- |
| RNA sequence 5′-3′ | UGUGACC | UGUGACC | UGUGCC | UGUGAACC | GGCUUU | AAAAGCC |
| Soak time (hr) | 24 | 4 | 24 | 24 | 24 | 24 |
| Space group | P2₁2₁2₁ | P2₁2₁2₁ | P2₁ | P2₁2₁2₁ | P2₁2₁2₁ | P2₁2₁2₁ |
| Unit cell dimensions | a = 76.38 | a = 76.35 | a = 76.0 | a = 77.12 | a = 76.65 | a = 76.74 |
|  | b = 112.47 | b = 112.75 | b = 143.766 | b = 112.66 | b = 112.53 | b = 112.31 |
|  | c = 143.05 | c = 143.79 | c = 112.833 | c = 144.84 | c = 143.58 | c = 143.72 |
|  | α = β = γ = 90 | α = β = γ = 90 | α = γ = 90, β = 90.65 | α = β = γ = 90 | α = β = γ = 90 | α = β = γ = 90 |
| Resolution (Å) | 30–2.9 | 30–2.6 | 30–2.8 | 30–3.2 | 30–3.0 | 30–3.0 |
| Completeness (%)a | 99.9 (99.6) | 90.1 (45.1) | 81.9 (62.2) | 88.9 (85.7) | 91.8 (60.3) | 92.4 (40.9) |
| I/σa | 25.0 (3.5) | 18.1 (1.9) | 17.2 (2.7) | 13.7 (3.5) | 16.1 (1.7) | 18.0 (1.5) |
| Rmerge (%)a | 5.5 (41.0) | 7.3 (39.7) | 5.1 (33.9) | 9.8 (40.4) | 7.6 (58.2) | 6.5 (53.7) |
| Total observed reflections | 109,200 | 124,070 | 138,830 | 59,470 | 106,010 | 125,720 |
| Unique reflections | 28,000 | 35,449 | 49,581 | 18,536 | 23,558 | 23,905 |

Refinement Statistics

| R (%) | 23.32 | 23.7 | 22.5 | 21.53 | 23.5 | 22.8 | 23.4 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Rfree (%) | 28.56 | 28.7 | 27.85 | 28.18 | 28.1 | 27.9 | 28.9 |
| Rms bond length | 0.007 | 0.009 | 0.007 | 0.008 | 0.008 | 0.009 | 0.009 |
| Rms bond angles | 1.14 | 1.28 | 1.25 | 1.24 | 1.31 | 1.46 | 1.41 |

a The numbers in parentheses refer to the highest-resolution shell.

extra bubble of density just beyond its C2 position, possibly accounting for the undefined C1 nucleotide (Figure S4). Despite the doubly overshot position of the 3′ end, +RNAs with a 3′CS+ that includes the inserted A (5′-UGUGAACC-3′) are efficient templates for dsRNA synthesis in vitro (Figure 3D).

Analysis of VP1 complexes formed with the A3-deleted oligonucleotide (5′-UGUGCC-3′) shows that VP1 and the UGUG segment interact just as in the complexes with wild-type (5′-UGUGACC-3′) and A-inserted (5′-UGUGAACC-3′) 3′CS+ (Figure 3C). Because of the A3 deletion, however, C1 aligns in

Figure 3. Structure of VP1 in Complex with RNA Oligonucleotides

(A–C) The wild-type 3′CS+ oligonucleotide (magenta) in the template entry tunnel with its 3′C1 residue in a one-nucleotide overshot position. In each panel, one other oligonucleotide is superimposed (green): (A) AAAAGCC, (B) UGUGAACC, or (C) UGUGCC. The C1 residue could not be resolved in the VP1/UGUGAACC complex. The priming loop (turquoise; sticks indicate Ser495 and Ser497), motif F (light blue), and catalytic residues Asp520, Asp631, and Asp632 (red) are shown.

(D) In vitro dsRNA synthesis. Template activities of gene 8 +RNAs ending in the conventional 3′CS+ sequence, UGUGCC, or UGUGAACC were assayed in reaction mixtures containing cVP1, in the presence or absence of VP2. Radiolabeled dsRNA products were resolved by SDS-PAGE and detected by autoradiography.

Structure

Rotavirus RNA Polymerase VP1

Figure 4. VP1-Template Interactions

(A and B) Identification of VP1 residues that engage (A) the 3′CS+ (UGUGACC) or (B) the 3′CS− (AAAAGCC). The relevant amino acids and nucleotides are labeled, and hydrogen bonds are indicated as red lines. Labeled residues are in the fingers subdomain, except for Arg186, Lys188, and Arg190, which are in the N-terminal domain. Both complexes are stabilized, in part, by stacking of the C1 and C2 bases on each other. The VP1/3′CS+ complex is further stabilized by the stacking of G4 and U5 bases on each other, the interaction of the U7 base with Phe416, and the interaction of the G6 base with Arg190 and Arg701.

(C) Merged structures of the VP1 apoenzyme (pink) and the VP1/3′CS+ complex (green), showing a shift of a loop-sheet element in the fingers subdomain that accompanies RNA binding. Amino acid residues and nucleotides are labeled, and hydrogen bonds are indicated as red lines.

VP2 in promoting initiation complex formation extends beyond simply moving the 3′ end of the template into register for initiation.

Attempts to recapitulate an initiation complex by soaking VP1 apoenzyme crystals with the 3′CS+ oligonucleotide, Mg²⁺, and GTP were not successful. Although density maps revealed the presence of the 3′CS+ in the entry tunnel, Mg²⁺ could not be detected, and only poorly resolved additional density was noted in the catalytic pocket, presumably representing weakly bound GTP. Moreover, the C1 position of the 3′CS+ oligonucleotide remained overshot by one position and the putative priming loop remained in the retracted position. The failure of these soaks to produce functional initiation complexes is consistent with reconstitution assays, which have shown that VP2 is necessary for forming VP1/+RNA complexes that can support dsRNA synthesis. Thus, the response to VP2-directed activation of VP1 is likely to include a shift of the priming loop from a retracted to an extended conformation, a back translocation of the 3′CS+ into initiation register, and perhaps further concerted changes. The RdRP of the dsRNA bacteriophage φ6 also binds template RNA in an overshot register, determined by a pocket that accepts the 3′ nucleotide (Butcher et al., 2001). In that case, however, addition of the complementary nucleoside triphosphates is sufficient to reposition the template and initiate phosphodiester bond formation.

Cap Binding

The surface of reovirus λ3 has a binding site for the N7-methyl-GpppG cap on +RNA; this site also binds GTP (Tao et al., 2002). We soaked the VP1 crystals in GTP and detected a bound

the initiation register. Weak difference density at the location occupied by C1 of the undeleted template suggests that a small percentage of the A3-deleted oligonucleotide may have aligned in the overshot register (Figure S4), and we propose that the template can shift between the initiation register and the overshot register within the entry tunnel. Recognition of the UGUG bases clearly shifts the equilibrium toward the initiation register in the case of the A3-deleted template.

VP1 recognition of the UGUG bases of +RNAs thus appears to create a stable autoinhibited polymerase-template complex. We carried out in vitro replication assays using +RNA with an A3-deleted 3′ end (5′-UGUGCC-3′) as template for dsRNA synthesis, to determine whether a correct register is sufficient to generate an active complex. The template activity of A3-deleted +RNA remains VP2 dependent (Figure 3D). We conclude that the role of

1684 Structure 16, 1678–1688, November 12, 2008 © 2008 Elsevier Ltd All rights reserved

Structure

Rotavirus RNA Polymerase VP1

Figure 5. Comparison of the VP1/3′CS+ and the λ3 Initiation Complexes

Elements of the rotavirus VP1/3′CS+ complex are superimposed on a ribbon representation of the reovirus λ3 initiation complex (light gray). The priming (P) and incoming (N) nucleotide positions are indicated, and Mg²⁺ ions are shown in olive. The template RNAs of rotavirus (UGUGACC, magenta) and reovirus (UAGC, green) and the incoming nucleotides (orange) are shown. The priming loop (λ3, yellow; VP1, turquoise) and motif F (λ3, pink; VP1, light blue) are colored, and the location of several key residues, including the catalytic aspartates (λ3, dark blue; VP1, red) and motif F arginines are indicated.

nucleotide with its purine ring in roughly the location corresponding to the unmethylated purine site on λ3 (data not shown). An uncapped oligonucleotide with the 5′ sequence of the +RNA, 5′-GGCUUU-3′, binds in the template tunnel (Figure S4D). Thus, the uncapped sequence alone is insufficient to dictate binding proximal to the cap site, and the cap (not present in the soaked oligonucleotide) appears to be the primary element by which VP1 grasps and recognizes the 5′ end of a plus strand.

DISCUSSION

The rotavirus polymerase domain resembles closely that of other viral RdRPs, and its N- and C-terminal domains are very similar to those of reovirus λ3. Moreover, both the conformation of the bound template and the way in which the templating bases are wedged against a segment of the fingers domain are conserved among all polymerases with a right-handed core for which the structure of a catalytic complex is known. Sequence-specific recognition of an RNA template sequence has not previously been documented, however, and the binding of template in an overshot register is also unusual. These similarities and differences allow us to correlate structural features with various aspects of VP1 activity, both in its enzymatic mechanism and in its broader biological function.

A model summarizing events in dsRNA synthesis by VP1 can be developed from comparisons of our crystallographic results with those obtained earlier for λ3 initiation and elongation complexes (Figure 6). The rotavirus polymerase specifically recognizes its +RNA template by forming hydrogen bonds with the bases of the 3′CS+ UGUG residues. These interactions, combined with additional contacts to the sugar-phosphate backbone, an-

Structure

Rotavirus RNA Polymerase VP1

Figure 6. Proposed Model of RNA Synthesis by VP1

Superimposed on the VP1 structure are elements from the reovirus λ3 (A) initiation complex, PDB ID code 1MWH, (B) short elongation complex, PDB ID code 1N38, and (C and D) long elongation complex, PDB ID code 1N35. The reovirus template is shown in green, the incoming nucleotides in orange, the Mg²⁺ ions in olive, and the extended λ3 priming loop in yellow. The priming (P) and incoming (N) nucleotide positions are indicated in (A). Important VP1 structural elements are shown, including motif F (light blue), the priming loop (turquoise), and the catalytic aspartates (red).

(A) During initiation of RNA synthesis, an extended priming loop is proposed to support the priming nucleotide, as seen for λ3.

(B) Following initiation, translocation of the dinucleotide requires retraction of the priming loop to prevent a clash between the elongating product (beige) and the extended priming loop.

(C) During elongation, a dsRNA product would emerge through the large exit tunnel when the VP1 C-terminal plug is assumed to be displaced.

(D) An inserted C-terminal plug (purple) would conflict with passage of the dsRNA product (affected nucleotides in beige), requiring repositioning of the plug during replication and/or redirection of the +RNA product during transcription.


During initiation,
priming loop is up


Following initiation,
priming loop must retract


During elongation,
plug is displaced


Plug would
clash with
dsRNA


–RNA near the template entry channel and favor its selective insertion.

All viruses of the many Reoviridae genera have similarly structured cores (see, for example, Lawton et al., 2000), with a tightly packed shell composed of 60 asymmetric dimers of VP2 or a homologous core shell protein (e.g., λ1 in orthoreoviruses and VP3 in orbiviruses). The shell is an icosahedrally symmetric array of 12 core shell protein decamers, each centered on a fivefold axis and each associated with a single (internal) polymerase molecule. The structures described in this paper and the properties of the viral cores suggest a simple mechanism for specific, +RNA packaging in a rotavirus infection. (1) Recognition of the 3′CS+ establishes stable, but inactive, VP1/+RNA complexes. (2) Association of this initial complex with a decamer of VP2 (or VP2 homolog) creates the basic assembly unit of the core. In all Reoviridae family members, one or more copies of an enzyme required for capping the transcript (e.g., VP3 of rotaviruses) must also be recruited. Association of the polymerase with the core shell protein subunits could also precede RNA recognition. (3) Twelve such basic assembly units (e.g., VP1-VP3-RNA-[VP2]₁₀ for rotaviruses) come together to form a functional core. (4) Additional proteins (e.g., VP6, which forms the second “layer” of the rotavirus DLP) then stabilize the core and mediate interaction with outer-shell components (e.g., rotaviruses VP4 and VP7). Recombinant polymerases from other Reoviridae family members (e.g., reovirus and bluetongue virus) have modest activity in vitro in the absence of the core shell protein. Whether the purified forms of the reovirus and bluetongue virus polymerases represent the structural and functional equivalents of the VP1 apoenzyme, or possibly a later activated form, remains to be determined.

EXPERIMENTAL PROCEDURES

Expression and Purification of Proteins

Recombinant baculoviruses expressing untagged VP1 and VP2 and His-tagged VP1 proteins (cVP1 or nVP1) were generated as described elsewhere (Patton et al., 1997; Tortorici et al., 2006). To prepare untagged C-truncated VP1, cDNA encoding residues 1–1059, 1–1074, or 1–1079 were subcloned into pCR-Bac vectors. Recombinant baculoviruses were formed with the Bac-N-Blue baculovirus expression system (Invitrogen). His-tagged proteins were recovered from clarified lysates by ammonium sulfate precipitation and cobalt-affinity (Talon) column chromatography (Tortorici et al., 2006). Untagged VP1 and VP2 were purified as described earlier (Patton et al., 1997).

Expression of SeMet-Substituted VP1

We adapted the methionine deprivation method (Bellizzi et al., 1999) to obtain selenomethionine (SeMet) derivatives of VP1. Because high concentrations of the analog was inhibitory to VP1 overexpression, SeMet was introduced in two steps, initially at a concentration of 5 mg/ml of culture, and then, 13 hr later, in amounts sufficient to bring the final SeMet concentration to 50 mg/ml. The incorporation of SeMet, as determined by mass spectrometry, was about 85%.

Crystallization and Data Collection

Crystals were obtained by starting with 1 μl of VP1 at 10 mg/ml in 25 mM Na HEPES (pH 7.8), 100 mM NaCl mixed with 2 μl of crystallization buffer

1686 Structure 16, 1678–1688, November 12, 2008 ©2008 Elsevier Ltd All rights reserved

Structure

Rotavirus RNA Polymerase VP1

(25 mM Na MES [pH 6.5], 1.5% [w/v] PEG 3350) and allowing the drop to equilibrate at 12°C by hanging-drop vapor diffusion with a well solution identical in composition to the drop except for the protein. With microseeding, thin, plate-like crystals appeared after 1 day and grew to full size (~15 × 200 × 200 μm³) in about 2 weeks. Crystals were transferred into cryoprotectant (25% [v/v] ethylene glycol, 8.3 mM Na HEPES [pH 7.8], 16.7 mM Na MES [pH 6.5]) and flash-frozen in liquid nitrogen. X-ray diffraction data were collected at the Advanced Light Source stations 7.2.1 and 7.2.2 and at the Advanced Photon Source beamlines ID-19 and ID-24. Data were reduced and scaled using HKL2000 (Table 1) (Otvinnoski and Minor, 1997).

### Structure Determination

Multiwavelength anomalous dispersion (MAD) data sets were collected at three wavelengths, corresponding to peak, inflection, and remote points of the Se K-edge profile. Direct methods (Shake-and-Bake; Weeks and Miller, 1999) were used to locate Se positions. A run with a 3.2 Å resolution peak-wavelength data set yielded 12 Se positions, and the remaining 21 positions were located in difference maps phased from the initial 12 sites using CNS (Brünger et al., 1998) and validated using Shake-and-Bake. The Se coordinates were refined with SHARP (Bricogne et al., 2003), using data from three SeMet-substituted crystals and a native (unsubstituted) crystal (Table 1). The electron density map calculated with experimental phases was improved by solvent flattening, histogram matching, and phase resolution extension, using the program DM (Cowtan and Main, 1998). Comparison with the structure of reovirus λ3 allowed 90% of the polypeptide chain to be traced unambiguously, using the program O (Jones et al., 1991). Most of the remaining residues were added during iterative rounds of refinement in CNS and rebuilding in O. The final model as analyzed with PROCHECK (Laskowski et al., 1998) has over 87% of the residues in the most favored regions of the Ramachandran plot and only four (0.4%), all in poorly defined loops, in disallowed regions. The final R_work and R_free (the latter based on a 7.5% test set) are 23.3% and 28.6%, respectively.

### Soaking Experiments

Crystals of apoenzyme were transferred to a stabilizing solution (1% [w/v] PEG 3350, 8.3 mM Na HEPES [pH 7.8], 16.7 mM Na MES [pH 6.5], 25% [v/v] ethylene glycol) containing 0.5 mM RNA oligonucleotide and incubated for 24 hr before freezing (Table 2). The structure of the apoenzyme was used as a phasing model. Initial rigid-body refinement yielded maps with interpretable density for the oligonucleotide, which was built with O; the structures of the various complexes were refined with CNS.

### In Vitro dsRNA Synthesis

Polymerase chain reaction was used to prepare gene 8 cDNAs with a 5′-terminal T7 promoter and a wild-type or mutated 3′ terminus (Tortorici et al., 2006). Gene 8 +RNAs were transcribed from the cDNAs using the Ambion T7 MEGA-script system. The reaction mixtures for replication assays (20 μl) included 4 pmol of VP1, 16 pmol of VP2, 8 pmol of gene 8 +RNA, 50 mM Tris-HCl (pH 7.1), 1.5% PEG 8000, 2 mM dithiothreitol, 1.5 U RNasin, 10 mM magnesium acetate, 2 mM MnCl₂, 1.25 mM each CTP, UTP, and ATP, 5 mM GTP, and 10 μCi of [α-³²P]UTP. Reaction mixtures were incubated for 5 hr at 37°C, and ³²P-labeled dsRNA products were analyzed by SDS-PAGE and autoradiography.

### Preparation of Figures

Figures were prepared with MolScript (Kraulis, 1991), POVScript+ (Fenn et al., 2003), Povray (http://www.povray.org/), UCSF Chimera (Pettersen et al., 2004), Raster3D (Merritt and Bacon, 1997), Spock (J.A. Christopher, http://quorum.tamu.edu/spock/), and ALSCRIPT (Barton, 1993).

### ACCESSION NUMBERS

Coordinates and structure factors have been deposited in the Protein Data Bank. ID codes are as follows: cVP1 (apo), **2R7Q**; nVP1 (apo), **2R7O**; cVP1 (UGUGACC), 2R7R; cVP1 (UGUGCC), **2R7S**; cVP1 (UGUGAAC), **2R7T**; cVP1 (AAAAGCC), **2R7U**; cVP1 (GGCUUU), **2R7V**; cVP1 (mRNA 5′ cap), **2R7W**; cVP1 (UGUGACC + GTP), **2R7X**.

---

SUPPLEMENTAL DATA

Supplemental Data include four figures and can be found with this article online at http://www.structure.org/cgi/content/full/16/11/1678/DC1/.

---

ACKNOWLEDGMENTS

We thank Teresa Broering, Sophia Rits-Volloch, Rachelle Gaudet, Kelly Arnett, Hongyan Yang, Harish Ramanathan, and Tamara Bar-Magen for technical assistance, and Philip Dormitzer, Ethan Settembre, and Piotr Sliz for advice. We also express our appreciation to the staff at ALS (beamlines 7.2.1 and 7.2.2) and APS (beamlines ID-19 and ID-24) for help in data collection. This work was supported by NIH grants CA13202 (to S.C.H.) and AI47904 (to M.L.N.), and by the Intramural Research Program of the NIH, National Institutes of Allergy and Infectious Diseases (J.T.P., S.M.M., M.A.T., and R.V.-D.C.). S.C.H. is an Investigator in the Howard Hughes Medical Institute.

Received: July 17, 2008  
Revised: September 6, 2008  
Accepted: September 17, 2008  
Published: November 11, 2008  

---

REFERENCES

Ago, H., Adachi, T., Yoshida, A., Yamamoto, M., Habuka, N., Yatsunami, K., and Miyano, M. (1999). Crystal structure of the RNA-dependent RNA polymerase of hepatitis C virus. Structure 7, 1417–1426.

Barton, G.J. (1993). ALSCRIPT—a tool to format multiple sequence alignment. Protein Eng. 6, 37–40.

Bellizzi, J.J., Widom, J., Kemp, C.W., and Clardy, J. (1999). Producing selenomethionine-labeled proteins with a baculovirus expression vector system. Structure 7, R263–R267.

Bressanelli, S., Tomei, L., Roussel, A., Incitti, I., Vitale, R.L., Mathieu, M., De Francesco, R., and Rey, F.A. (1999). Crystal structure of the RNA-dependent RNA polymerase of hepatitis C virus. Proc. Natl. Acad. Sci. USA 96, 13034–13039.

Bricogne, G., Vonrhein, C., Flensburg, C., Schiltz, M., and Paciorek, W. (2003). Generation, representation and flow of phase information in structure determination: recent developments in and around SHARP 2.0. Acta Crystallogr. D Biol. Crystallogr. 59, 2023–2030.

Brünger, A.T., Adams, P.D., Clore, G.M., DeLano, W.L., Gros, P., Grosse-Kunstleve, R.W., Jiang, J.S., Kuszewski, J., Nilges, M., Pannu, N.S., et al. (1998). Crystallography & NMR system: a new software suite for macromolecular structure determination. Acta Crystallogr. D Biol. Crystallogr. 54, 905–921.

Butcher, S.J., Grimes, J.M., Makeyev, E.V., Bamford, D.H., and Stuart, D.I. (2001). A mechanism for initiating RNA-dependent RNA polymerization. Nature 410, 235–240.

Chen, D., and Patton, J.T. (2000). De novo synthesis of minus strand RNA by the rotavirus RNA polymerase in a cell-free system involves a novel mechanism of initiation. RNA 6, 1455–1467.

Chen, D., Luongo, C.L., Nibert, M.L., and Patton, J.T. (1999). Rotavirus open cores catalyze 5′-capping and methylation of exogenous RNA: evidence that VP3 is a methyltransferase. Virology 265, 120–130.

Chen, D., Barros, M., Spencer, E., and Patton, J.T. (2001). Features of the 3′-consensus sequence of rotavirus mRNAs critical to minus strand synthesis. Virology 282, 221–229.

Cowtan, K., and Main, P. (1998). Miscellaneous algorithms for density modification. Acta Crystallogr. D Biol. Crystallogr. 54, 487–493.

Doublie, S., and Ellenberger, T. (1998). The mechanism of action of T7 DNA polymerase. Curr. Opin. Struct. Biol. 8, 704–712.

Estes, M.K., and Kapikian, A.Z. (2007). Rotaviruses. In Fields Virology, Fifth Edition, D.M. Knipe and P.M. Howley, eds. (Philadelphia: Lippincott Williams & Wilkins), pp. 1917–1974.

Fenn, T.D., Ringe, D., and Petsko, G.A. (2003). POVScript+: a program for model and data visualization using persistence of vision ray-tracing. J. Appl. Crystallogr. 36, 944–947.

Gallegos, C.O., and Patton, J.T. (1989). Characterization of rotavirus replication intermediates: a model for the assembly of single-shelled particles. Virology 172, 616–627.

Jacobo-Molina, A., Ding, J., Nanni, R.G., Clark, A.D., Jr., Lu, X., Tantillo, C., Williams, R.L., Kamer, G., Ferris, A.L., Clark, P., et al. (1993). Crystal structure of human immunodeficiency virus type 1 reverse transcriptase complexed with double-stranded DNA at 3.0 Å resolution shows bent DNA. Proc. Natl. Acad. Sci. USA 90, 6320–6324.

Jones, T.A., Zou, J.Y., Cowan, S.W., and Kjeldgaard, M. (1991). Improved methods for building protein models in electron density maps and the location of errors in these models. Acta Crystallogr. A 47, 110–119.

Joyce, C.M., and Steitz, T.A. (1994). Function and structure relationships in DNA polymerases. Annu. Rev. Biochem. 63, 777–822.

Kohlstaedt, L.A., Wang, J., Friedman, J.M., Rice, P.A., and Steitz, T.A. (1992). Crystal structure at 3.5 Å resolution of HIV-1 reverse transcriptase complexed with an inhibitor. Science 256, 1783–1790.

Kraulis, P.J. (1991). MolScript: a program to produce both detailed and schematic plots of protein structures. J. Appl. Crystallogr. 24, 946–950.

Labbe, M., Baudoux, P., Charpilienne, A., Poncet, D., and Cohen, J. (1994). Identification of the nucleic acid binding domain of the rotavirus VP2 protein. J. Gen. Virol. 75, 3423–3430.

Laskowski, R.A., MacArthur, M.W., and Thornton, J.M. (1998). Validation of protein models derived from experiment. Curr. Opin. Struct. Biol. 8, 631–639.

Lawton, J.A., Estes, M.K., and Prasad, B.V. (1997). Three-dimensional visualization of mRNA release from actively transcribing rotavirus particles. Nat. Struct. Biol. 4, 118–121.

Lawton, J.A., Estes, M.K., and Prasad, B.V. (2000). Mechanism of genome transcription in segmented dsRNA viruses. Adv. Virus Res. 55, 185–229.

Lesburg, C.A., Cable, M.B., Ferrari, E., Hong, Z., Mannarino, A.F., and Weber, P.C. (1999). Crystal structure of the RNA-dependent RNA polymerase from hepatitis C virus reveals a fully encircled active site. Nat. Struct. Biol. 6, 937–943.

Liu, M., Mattion, N.M., and Estes, M.K. (1992). Rotavirus VP3 expressed in insect cells possesses guanylyltransferase activity. Virology 188, 77–84.

Merritt, E.A., and Bacon, D.J. (1997). Raster3D: photorealistic molecular graphics. Methods Enzymol. 277, 505–524.

O’Reilly, E.K., and Kao, C.C. (1998). Analysis of RNA-dependent RNA polymerase structure and function as guided by known polymerase structures and computer predictions of secondary structure. Virology 252, 287–303.

Otwinowski, Z., and Minor, W. (1997). Processing of X-ray diffraction data collected in oscillation mode. Methods Enzymol. 276, 307–326.

Patton, J.T. (1996). Rotavirus VP1 alone specifically binds to the 3′ end of viral mRNA, but the interaction is not sufficient to initiate minus-strand synthesis. J. Virol. 70, 7940–7947.

Patton, J.T., and Gallegos, C.O. (1990). Rotavirus RNA replication: single-stranded RNA extends from the replicase particle. J. Gen. Virol. 71, 1087–1094.

Patton, J.T., Jones, M.T., Kalbach, A.N., He, Y.W., and Xiaobo, J. (1997). Rotavirus RNA polymerase requires the core shell protein to synthesize the double-stranded RNA genome. J. Virol. 71, 9618–9626.

Pettersen, E.F., Goddard, T.D., Huang, C.C., Couch, G.S., Greenblatt, D.M., Meng, E.C., and Ferrin, T.E. (2004). UCSF Chimera: a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612.

Pizarro, J.L., Sandino, A.M., Pizarro, J.M., Fernandez, J., and Spencer, E. (1991). Characterization of rotavirus guanylyltransferase activity associated with polypeptide VP3. J. Gen. Virol. 72, 325–332.

Prasad, B.V., Rothnagel, R., Zeng, C.Q., Jakana, J., Lawton, J.A., Chiu, W., and Estes, M.K. (1996). Visualization of ordered genomic RNA and localization of transcriptional complexes in rotavirus. Nature 382, 471–473.

Tao, Y., Farsetta, D.L., Nibert, M.L., and Harrison, S.C. (2002). RNA synthesis in a cage—structural studies of reovirus polymerase λ3. Cell 111, 733–745.

Tortorici, M.A., Broering, T.J., Nibert, M.L., and Patton, J.T. (2003). Template recognition and formation of initiation complexes by the replicase of a segmented double-stranded RNA virus. J. Biol. Chem. 278, 32673–32682.

Tortorici, M.A., Shapiro, B.A., and Patton, J.T. (2006). A base-specific recognition signal in the 5′ consensus sequence of rotavirus plus-strand RNAs promotes replication of the double-stranded RNA genome segments. RNA 12, 133–146.

Valenzuela, S., Pizarro, J., Sandino, A.M., Vasquez, M., Fernandez, J., Hernandez, O., Patton, J., and Spencer, E. (1991). Photoaffinity labeling of rotavirus VP1 with 8-azido-ATP: identification of the viral RNA polymerase. J. Virol. 65, 3964–3967.

Weeks, C.M., and Miller, R. (1999). Optimizing Shake-and-Bake for proteins. Acta Crystallogr. D Biol. Crystallogr. 55, 492–500.

Yeager, M., Dryden, K.A., Olson, N.H., Greenberg, H.B., and Baker, T.S. (1990). Three-dimensional structure of rhesus rotavirus by cryoelectron microscopy and image reconstruction. J. Cell Biol. 110, 2133–2144.

Zeng, C.Q., Estes, M.K., Charpilienne, A., and Cohen, J. (1998). The N terminus of rotavirus VP2 is necessary for encapsidation of VP1 and VP3. J. Virol. 72, 201–208.
