
Biochimica et Biophysica Acta 1794 (2009) 769–781

Contents lists available at ScienceDirect

Biochimica et Biophysica Acta

journal homepage: www.elsevier.com/locate/bbapap

Review

Mechanisms of RND multidrug efflux pumps

Hiroshi Nikaido *, Yumiko Takatsuka

Department of Molecular and Cell Biology, Barker Hall, University of California, Berkeley, CA 94720-3202, USA

ARTICLE INFO

Article history:

Received 12 September 2008

Received in revised form 18 October 2008

Accepted 21 October 2008

Available online 3 November 2008

Keywords:

AcrB

AcrD

TolC

AcrA

Reconstitution

Disulfide cross-linking

Proton relay network

ABSTRACT

RND (Resistance-Nodulation-Division) family transporters are widespread especially among Gram-negative bacteria, and catalyze the active efflux of many antibiotics and chemotherapeutic agents. They have very large periplasmic domains, and form tripartite complexes with outer membrane channels and periplasmic adaptor proteins. AcrAB-TolC complex of *Escherichia coli*, which pumps out a very wide range of drugs, has been studied most intensively. Early studies showed that the transporter captures even those substrates that cannot permeate across the cytoplasmic membrane, such as dianionic β-lactams, suggesting that the capture can occur from the periplasm. It was also suggested that the capture occurs from the cytoplasmic membrane/periplasm interface, because most substrates contain a sizable hydrophobic domain; however, this may simply be a reflection of the nature of the binding site within AcrB. Genetic studies of chimeric transporters showed that much of the substrate specificity is determined by their periplasmic domains. Biochemical studies with intact cells recently led to the determination of the kinetic constants of AcrB for some β-lactams, and the result confirms the old prediction that AcrB is a rather slow pump. Reconstitution of purified AcrB and its relatives showed that the pump is a drug/proton antiporter, that AcrA strongly stimulates the activity of the pump, and that AcrB seems to have a highest affinity for conjugated bile salts. Structural study with mutants of the network of charged residues in the transmembrane domain showed that protonation here produced a far-reaching conformational change, which was found to be present in one of the protomers in the asymmetric crystal structure of the wild-type AcrB. The functional rotary hypothesis then predicts that the drug bound in the periplasmic domain is extruded through this conformational change initiated by the protonation of one of the residues in the aforementioned network, an idea that was recently supported by disulfide cross-linking as well as by the behavior of linked AcrB protomers.

© 2008 Elsevier B.V. All rights reserved.

1. Introduction

Efflux pumps of the RND (Resistance-Nodulation-Division) superfamily (such as AcrB of *Escherichia coli* and MexB of *Pseudomonas aeruginosa*) play an important role in producing multidrug resistance (both intrinsic and elevated) in Gram-negative bacteria. This is because these pumps become associated with two other classes of proteins, the outer membrane channel such as TolC of *E. coli* and OprM of *P. aeruginosa*, belonging to the OMF (outer membrane factor) family of proteins [1], and the periplasmic “adaptor” protein such as AcrA of *E. coli* and MexA of *P. aeruginosa*, classified into the MFP (membrane fusion protein) family [2]. Importantly, each of these three component proteins is essential for drug efflux, and the absence of even one component makes the entire complex totally nonfunctional [3,4]. The construction of this tripartite complex suggested that the drugs are here exported directly into the external medium, rather than into the periplasm [5]. This is a huge advantage for bacterial cells, because once exported into the external space, drug molecules must traverse the outer membrane barrier to reenter the cells. (An early conceptual diagram is shown in Fig. 1). Thus these pumps work synergistically with the outer membrane barrier. Wild-type strains of most Gram-negative bacteria are resistant to most lipophilic antibiotics (for *E. coli* they include penicillin G, oxacillin, cloxacillin, nafcillin, macrolides, novobiocin, linezolid, and fusidic acid), and this “intrinsic resistance” was often thought to be caused by the exclusion of drugs by the outer membrane barrier. Indeed breaching the outer membrane barrier does sensitize *E. coli* cells to the drugs just mentioned [6]. However, the inactivation of the major and constitutively expressed RND pump AcrB makes the bacteria almost completely susceptible to these agents (the minimal inhibitory concentration [MIC] of a lipophilic penicillin, cloxacillin, goes down from 512 μg/ml in the wild type to only 2 μg/ml [7]) even in the presence of the intact outer membrane barrier. Thus the characteristic intrinsic resistance of gram-negative bacteria owes as much to the RND pumps as to the outer membrane barrier.

2. Substrate range of AcrB and the periplasmic substrate capture

We noted early [5] that AcrB of *E. coli* can handle a very wide range of compounds. These include cationic dyes such as acriflavine, crystal violet, ethidium bromide, and rhodamine 6G; antibiotics such as

* Corresponding author. Tel.: +1 510 642 2027; fax: +1 510 643-6334.
E-mail address: nhiroshi@berkeley.edu (H. Nikaido).

1570-9639/$ – see front matter © 2008 Elsevier B.V. All rights reserved.
doi:10.1016/j.bbapap.2008.10.004

H. Nikaido, Y. Takatsuka / Biochimica et Biophysica Acta 1794 (2009) 769–781

In this connection, it is interesting that the general inhibitor of *P. aeruginosa* and *E. coli* RND pumps, MC-207,110 (phenylalanyl-arginyl-β-naphthylamine) [12], which apparently is a favored substrate of the pumps, affected the carbenicillin MIC only to a small extent [12]. Thus the addition of this inhibitor decreased the MICs of drugs such as erythromycin and levofloxacin to almost exactly the same degree as the genetic inactivation of the MexB-MexA-OprM pump. In contrast the inhibitor produced a decrease of only fourfold in carbenicillin MIC, although the genetic inactivation decreased it 512-fold. A similar discrepancy was also found for the MICs of tetracycline and ethidium bromide [12]. Since we know that carbenicillin must be captured exclusively from the periplasm, these results might suggest that this particular inhibitor acts more effectively for drugs that are predominantly captured in the cytosol. However, the results may be caused by the binding of various substrates to different subdomains of the large binding pocket of the transporter.

The RND pumps have been shown to act in synergy with the “simple” pumps that extrude drugs only into the periplasm [13]. For example, in *P. aeruginosa*, the expression of a simple tetracycline pump TetA produces only a very modest degree of resistance (MIC = 32 μg/ml) in the absence of the main tripartite pump MexB-MexA-OprM, but MIC is raised very strongly to 512 μg/ml if the tripartite pump in addition is expressed at the normal, constitutive level. That the high resistance is not due to the pumping of tetracycline by the tripartite pump alone is seen from the fact that the tripartite pump, in the absence of TetA, produces an MIC of only 4 μg/ml. Thus these two types of pumps appear to act truly in a synergistic manner. The most likely molecular explanation of these results is that the tetracycline pumped out into the periplasm by the TetA pump is then captured by the MexB pump, and is extruded into the outside medium through the tripartite structure [13]. This concept then explains why the simple, or the single-component transporters such as TetA [14] or *S. aureus* QacA [15] confer significant resistance to *E. coli* cells. It also emphasizes the importance of periplasmic substrate capture mechanism in the function of RND pumps.

### 3. Kinetic constants of the AcrB pump

In order to understand the behavior of the pump in intact cells, it is obviously essential to know its kinetic constants. The first hint on the binding affinity of various substrates was obtained by using them as competitive inhibitors in the reconstitution assay of AcrB [16], which will be described in detail later in this review. In this assay, which depended on the export of fluorescent phospholipids by the purified AcrB protein, taurocholate, a conjugated bile salt, inhibited the reaction most strongly (presumably by competing as substrates), the 50% inhibition occurring at 15 μM. In contrast, antibiotics were less efficient inhibitors, and cl oxacin and erythromycin caused 50% inhibition only around 100 μM. These results are important in showing that the conjugated bile salts are the preferred, and most likely the natural, substrates of the AcrB pump of *E. coli*, which must live in an environment full of these membrane-disrupting detergents. However, this is not a direct assay of substrate binding, and we do not know the true affinity of each substrate to the pump. (We should also mention that running similar assays in intact cells is difficult because both the substrate and the competitive inhibitor must diffuse across the outer membrane before binding to the pump.)

There was a recent attempt to measure the binding of substrates to purified AcrB protein by the use of fluorescence polarization [17]. Because binding of fluorescent substrates to a large AcrB trimer is expected to slow the tumbling of these molecules, this ingenious method offers advantages. The apparent dissociation constants reported ranged from 5.5 μM for rhodamine 6G to 74 μM for ciprofloxacin. However, the method does not give reliable values of binding stoichiometry, and because the authors did not use mutant AcrB proteins that are altered in substrate-binding sites, it is unclear
whether the binding occurred to the true binding sites, or to some extraneous pockets that bind lipophilic molecules in a non-specific manner.

Several laboratories followed the efflux of fluorescent dyes from cells after their energization. Typically, cells are preloaded with dyes that become fluorescent only within the membrane. The preloading requires inactivation of the RND pump by proton-conductor such as CCCP (carbonyl cyanide *m*-chlorophenylhydrazone), and re-energization of the cells is done by adding compounds such as glucose or formate. If everything goes perfectly, the time-dependent decrease in fluorescence intensity should follow the integrated form of the Michaelis–Menten equation, which should give us the kinetic constants $K_{\mathrm{m}}$ and $V_{\max }$. In the first paper reporting on the use of these probes [18], the results were not convincing because the efflux rates from *P*. aeruginosa strains either producing or not producing the MexB–MexA–OrpM efflux complex showed little difference, presumably because deenergization with cyanide could not be reversed rapidly. However, efflux assays were successfully performed since then as a qualitative assay, for example with NPN (*N*-phenylnaphthylamine) and *E*. coli [12], and with DASPEI (2-[4-dimethylamino]styryl-*N*-ethylpyridinium iodide) and *E*. coli [19]. No effort to use the efflux curve for quantitative analysis of the transport process was reported. We have tried to do this quantitative analysis with a fluorescent dye, Nile Red using *E*. coli (J. Bohnert and H. Nikaido, unpublished results). However, the system is extremely complex, because CCCP used in the dye-loading period may still persist in the cells at least in the beginning, and the full membrane energization with carbon sources will take a few seconds, thus making the analysis of the early phase of the efflux curve quite difficult.

Very recently we had the first success with the estimation of kinetic constants of the AcrB pump (K. Nagano and H. Nikaido, manuscript in preparation). With attempts using intact cells, the difficulty is always with the estimation of the substrate concentration within the periplasm, where it is interacting with the pump. We solved this problem by using β-lactams as substrates. Since β-lactams are hydrolyzed by the periplasmic β-lactamase, and since we know the $K_{\mathrm{m}}$ and $V_{\max }$ of the enzyme, we can calculate the periplasmic concentration of β-lactams from the Michaelis–Menten equation if we follow the hydrolysis of β-lactams by intact cells. Once the periplasmic concentration is known, the rate at which the β-lactams cross the outer membrane by diffusion through porin channels ($V_{\text {in }}$) can be calculated because it is proportional to the concentration difference of the β-lactam between the external medium and the periplasm. The efflux rate is the difference between the influx rate $V_{\text {in }}$ and the hydrolysis rate $V_{\text {hyd }}$, or $V_{\text {in }} - V_{\text {hyd }}$. This approach was successfully applied to the influx, efflux, and hydrolysis of a cephalosporin, nitrocefin, and we found that the efflux occurs with the Michaelis–Menten saturation kinetics, with the apparent $K_{\mathrm{m}}$ of 3–5 μM. There was little sign of positive cooperativity. However, positive cooperativity was evident with the efflux of other cephalosporins with apparently lower affinity to the pump, a result that is interesting in view of the functionally rotating mechanism of the pump (see below) and an earlier report on CzcA [20].

Very recently, binding of 24-, 25-, or 27-hydroxysterols to Niemann–Pick C1 protein (NPC1), a member of the RND family involved in cholesterol trafficking in vertebrate cells [21], was reported [22]. The binding took place in the presence of a high concentration (1%) of the detergent Nonidet P40, and the half-saturation was reached with about 0.1 μM of 25-hydroxysterol. 7-, 19, 20-Hydroxysterols did not bind to this protein. In comparison with bacterial RND pumps, the NPC1 protein has an N-terminal extension consisting of an extramembranous domain of about 240 residues and an additional transmembrane segment. This extramembranous domain is secreted from cultured cells as a soluble dimer, which bound 25-hydroxysterol with a strong affinity ($K_{\mathrm{D}} = 10 \, \mathrm{nM}$) [23]. Since this domain is uniquely present in NPC1, it is unclear whether these beautiful results would

help us in our effort to understand the ligand binding process in efflux transporters.

### 4. Suggestions from genetic studies

Genetic approaches have been able to provide valuable insights on the reaction mechanism of RND pumps, as described below.

#### 4.1. Use of chimeric genes

Replacing large segments of genes with sequences coming from another homolog has been most useful in identifying the domains responsible for functions of these pumps. This was first done in 2002 by Elkins and Nikaido [24] as well as Tikhonova et al. [25]. The first group used *E*. coli AcrB, which gives a strong resistance to ciprofloxacin, novobiocin, fusidic acid, erythromycin, and taurocholate, and AcrD, which generates no resistance to these agents except a modest one against taurocholate. Replacing both of the large periplasmic loops of AcrD with those from AcrB created a transporter pumping out all of the tested AcrB substrates efficiently. Replacing loops of AcrB with those from AcrD created a transporter that behaved like AcrD in terms of its substrate specificity. Finally, replacing the transmembrane regions of AcrB with corresponding sequences from AcrD did not alter the substrate specificity of the transporter. These results suggest that the substrate specificity, and presumably the critical binding of substrates, occur in the periplasmic domain, not in the transmembrane domain. The latter group used AcrB and *P*. aeruginosa MexB. The results suggested that the substrate specificity was determined largely by the second external loop of these transporters, a conclusion consistent with the location of the substrate-binding pocket identified in the asymmetric trimer crystal of AcrB (see Inference from structure, below). They also showed that replacing the region coding for the transmembrane helices 8 through 12 of AcrB with the MexB sequence produced a somewhat MexB-like pattern for a few drugs (lower efflux of ethidium bromide and lincomycin, and higher efflux of cinoxacin); it is difficult to explain these results in terms of structure. Finally, this group showed that the N-terminal periplasmic loop and the first part of the C-terminal loop essentially determine the interaction of the RND pumps with their cognate periplasmic adaptor proteins. In 2003, a study replacing the transmembrane strands of *P*. aeruginosa MexB with those of MexY, an aminoglycoside pump [26], was published. There was no change in substrate specificity. Although these are negative results, they are at least consistent with the hypothesis that the periplasmic domain plays a large role in determining the substrate specificity.

#### 4.2. Use of random mutants

In this approach pioneered by Lomovskaya's laboratory, spontaneous mutants of *P*. aeruginosa MexD that acquired the ability to extrude carbenicillin, which is not handled by the wild-type MexD, were isolated [27]. All mutants mapped to the periplasmic domain, none to the transmembrane domain, further supporting the conclusion mentioned above. Among the residues identified, Phe608 (corresponding to Phe610 of AcrB) is a part of the substrate-binding pocket more recently identified (see below). Some others (Gln34, Gln89 [corresponding to Glu 89 of AcrB], Asn673[Thr676]) are on the walls of the large lateral cleft of the periplasmic domain, which will be discussed later as a potential site of periplasmic entry for substrates. Bohnert et al. [28] used an *E*. coli strain that lacked both AcrB and AcrF and instead overproduced another RND pump, YhiV. A spontaneous mutant obtained after repeated exposure to levofloxacin was shown to owe its increased resistance to a change of an aliphatic Val610 into an aromatic Phe in YhiV. This mutant pump
has an altered specificity. It produces a stronger resistance to relatively small, aromatic compounds such as levofloxacin, linezolid and tetracycline, but the resistance to non-aromatic, bulky macro- lides becomes weaker than in the parent protein. This residue, which corresponds to Val612 of AcrB, is seen to be a part of the substrate-binding pocket in the asymmetric AcrB trimer structure that will be discussed later.

In a different approach, Middlemiss and Poole [29] carried out an in vitro random mutagenesis of the *mexB* gene from *P. aeruginosa*. This is a more comprehensive approach, but is expected to generate more “noise”. Indeed the group of mutants that decreased the level of resistance to most of drugs included the presumed proton relay mutants in Asp407 and Asp408 of the transmembrane domain (see below), or Gly220 mutant in the “peg” that is inserted into the periplasmic domain of the neighboring protomer. The mutants that are significantly altered in the substrate specificity, in contrast, often contained alterations in the periplasmic domain, as expected. Among these, alterations of Ala618 and Arg716 (corresponding to Arg717 of AcrB) occur on the opposing walls of the large lateral cleft. However, there were several mutants in the transmembrane domain, and their properties remain to be explained.

### 4.3. Site-directed mutagenesis

In 1999, inspection of amino acid sequence of an RND-family toxic ation efflux pump CzcA of *Ralstonia* sp. and other RND pumps showed that there are several conserved charged residues in the transmem- brane domain, including Asp402, Asp408, and Glu415 [20]. Changing these residues into non-acidic residues abolished the cation pumping activity. This was followed up a few years later by the site-directed mutagenesis study of MexB [30]. In MexB, the Asp402 and Asp408 of CzcA are replaced by a tandem pair of aspartate residues, Asp407 and Asp408 on transmembrane segment (TMS) 4, and both of them were found to be essential for the pump function. This study also identified an essential Lys939 located in TMS 10, and suggested that these three residues produce a proton-relay network. Finally, we have identified Thr978 of AcrB on TMS 11 as another, functionally essential component of this network [31]. (This study also identified Arg971, located close to the cytosolic end of TMS11, as an essential residue [31], although it is far away from the network of other charged residues.)

Site-directed mutagenesis was also applied to residues assumed to be involved in the binding or passage of substrates. When we reported on the crystal structures of AcrB with ligands in the central cavity of the transmembrane domain [32], we were aware of the problem that the ligand might be binding to any hydrophobic pocket of the protein, which has nothing to do with the normal pathway of the exported ligand. Thus we changed residues that appeared to be involved in the binding through site-directed mutagenesis, and we obtained an assuring result that conversion of Phe386 to alanine nearly totally abolished the resistance to ethidium, rhodamine 6G, and dequalinium [32]. However, this result was obtained by using a very high copy number vector of the pUC series, and we now are aware that this approach may produce misleading results possibly by making the cell membrane leaky through a strong overproduction of intrinsic membrane proteins. Indeed, Phe386Ala mutation hardly affected susceptibility of *E. coli* to most drugs, when tested by expression with the vector pSPORT1 [33]. Nevertheless, Hearn et al. [34], using a *Pseu- domonas fluorescens* homolog of AcrB, EmhB, expressed from a medium copy number plasmid, found that decreased efflux of dequalinium was produced by the same change (Phe386Ala) as well as the change of Asn99 to alanine. Furthermore, changing Asp101, which is located at the “ceiling” of the central cavity, into alanine decreased the efflux of most drugs as well as of polycyclic hydrocarbons.

With AcrB mutant Asn101Ala, which will be described later in the section Inference from structure, changing into alanine the residues Glu673, Phe664, or Phe666, all located in the right (looking into the center of molecule with the periplasmic domain up) wall or the bottom of the large periplasmic cleft, close to its entrance, decreased the efflux activity [33]. With the Cu<sup>+</sup> and Ag<sup>+</sup> efflux pump CusA of *E. coli*, changing into leucine the methionine residues 573, 623, and 672, all close to the entrance of the periplasmic large cleft in the AcrB model, strongly reduced its activity [35]. Together with the binding of substrates to this cleft observed in the Asn101Ala mutant AcrB [33], these results favor the assumption that the periplasmic uptake of substrates by these pumps involves the entry from the large periplasmic cleft.

In an interesting study, Dastidar et al. [36] introduced cysteine residues to various positions of *Haemophilus influenzae* AcrB, chosen on the basis of their effects on substrate specificity in MexD [27] (described above), and measured their accessibility to a relatively hydrophilic probe, fluorescein maleimide in the presence and absence of efflux substrates. Most substrates, except ethidium, inhibited strongly the modification of cysteine at position 288, which corresponds to Gly290 of AcrB that is close to the bottom of the substrate-binding pocket in the asymmetric trimer structure (described later in the review). Interestingly, the modification of cysteine at position 601, corresponding to Phe617 of AcrB, one of the residues lining the substrate-binding pocket, is inhibited by Triton X-100, but not by other substrates. These data suggest the large size (or the flexibility) of the binding pocket that would accommodate different ligands in different ways.

Murakami et al. [19] used cysteine scanning mutagenesis to examine the role of the central “pore”, which appears to be closed in the crystal structure of AcrB. Although alterations of many residues, including Asp101 just mentioned, were found to decrease the extrusion of drugs, the interpretation is complicated by the fact that the pore is made by a three-stranded coiled-coil structure composed of all three protomers of AcrB, so that inter-subunit disulfide bonds are formed in some cases.

## 5. Reconstitution studies

In the biochemical studies of most bacterial transporters, mem- brane vesicles, either right-side-out or inverted, have been most useful. However, with RND family multidrug efflux pumps, this approach was not successful, although it was vigorously pursued in several laboratories including our own. The likely reasons for this failure include the inherently slow kinetics of the pump, described above, and the hydrophobic nature of most substrates which allows a spontaneous equilibration of transported substrates across the membrane layer. Another possibility may be that the periplasmic adaptor protein, which seems to be necessary to activate the transporter (see below), becomes stripped off during the preparation of the membranes.

The first successful functional reconstitution of an RND pump was achieved by Zgurskaya and Nikaido [16] in 1999. Cells containing *acrAB* genes on a high-copy-number pUC plasmid were grown without induction, and the membrane proteins were solubilized with 5% Triton X-100 overnight in the cold. The native AcrB protein contains four histidine residues among the eight residues at its C-terminus, and the protein could be purified by adsorption to a Cu<sup>2+</sup>-chelate matrix followed by elution with imidazole, always in the presence of Triton X-100. Just before the reconstitution, AcrB was adsorbed again to the Cu<sup>2+</sup>-chelate matrix, and Triton X-100 was exchanged with octyl-β-d-glucoside by elution with a buffer containing the latter detergent as well as imidazole. Since AcrB formed visible aggregates in octylgluco- side within a few hours, it was immediately reconstituted into proteoliposomes by the octylglucoside dilution method.

Innovative approaches were required for assays of the transport activity of AcrB. Unlike many transporters that move hydrophilic ligands (see below), most of the substrates of AcrB are quite hydrophobic [5], and are expected to traverse across membrane bilayers through

spontaneous diffusion. Thus it was impossible to rely upon the quantitation of ligands accumulated within the intravesicular space. Inspired by the report that mammalian P-glycoprotein, which also transports hydrophobic compounds, has activity as a phospholipid flippase [37], we used a 7-nitrobenz-2-oxa-1,3-diazol-4-yl (NBD)-labeled fluorescent phospholipid as the substrate. Phospholipids,

A

![Diagram A]

Fig. 3. Proton flux assay of reconstituted AcrB. AcrB-containing vesicles were made in the presence of 1 mM pyranine, a fluorescent pH indicator, and the extravesicular pyranine was removed by gel filtration. The interior-acidic pH gradient was generated by diluting the vesicles containing 0.1 M KCl into 0.1 M NaCl and then by adding valinomycin (arrow). The acidic internal pH was maintained for several minutes in the absence of drug (curve 1). When erythromycin (curve 2), chloramphenicol (curve 3), cloxacillin (curve 4), glycocholate (curve 5), or taurocholate (curve 6) were present at 0.2 mM, the interior pH returned to its initial value more rapidly, presumably due to the proton efflux accompanying the influx of drugs by the AcrB antiporter. From [16].

however, are likely to become reinserted into the original membrane even when they are expelled from it by the AcrB pump. In order to minimize this possibility, an excess of “acceptor” liposomes, which did not contain AcrB, were used to “trap” the fluorescent phospholipids extruded. Finally, the amount of the NBD-labeled phospholipids remaining in the AcrB-containing “donor” proteoliposomes was estimated by initially quenching the NBD fluorescence through fluorescence energy transfer to rhodamine-labeled phospholipids (Fig. 2A).

Fig. 2B shows a representative result. Addition of Mg²⁺ activated the transport process, which is consistent with the observation that in intact cells, another RND-family multidrug efflux pump MexXY requires the presence of Mg²⁺ in the medium for its function [38]. The pumping was further accelerated by the addition of lipid-free AcrA. The latter result was interpreted as the result of AcrA protein connecting the donor and acceptor vesicles [16]. Alternatively, however, AcrA might have activated the pumping activity of AcrB directly, in view of the fact that AcrD, a close homolog of AcrB, is activated in a reconstituted system by AcrA [11] (see below).

To show more directly that AcrB functions as a proton/drug antiporter, the system was energized by using the valinomycin-induced flux of K⁺, which was converted into a proton gradient in the presence of KCl. By measuring the intravesicular pH with a fluorescent, membrane-impermeable pH indicator pyranine, we confirmed that proton efflux occurred accompanying the pH-gradient-driven influx of drugs (Fig. 3). When the number of protons moved per vesicle is calculated, it appears that the AcrB pump is

B

![Diagram B]

Fig. 2. Phospholipid extrusion assay of reconstituted AcrB. A. Principle of the assay. Donor proteoliposomes containing AcrB (blue rectangles) were mixed with acceptor liposomes not containing any protein. The NBD fluorescence from the NBD-labeled phosphatidylethanolamine (orange) in the donor vesicles is initially quenched by fluorescent emission energy transfer to the rhodamine-labeled phosphatidylethanolamine (red). However, following the export of phospholipids to the acceptor vesicles, the surface density of labeled phospholipids decreases, and the fluorescent emission from NBD increases. B. An example of data. When a transmembrane pH gradient was generated by diluting the vesicles (internal pH, 7.0) into a buffer of pH 6.0, donor vesicles not containing any AcrB (bottom trace) do not show much change in fluorescence. Nor do the AcrB-containing vesicles in the absence of pH gradient (middle trace). However, when AcrB-containing vesicles are exposed to pH gradient, there is the expected gradual increase in NBD fluorescence in the presence of Mg²⁺. Note that the rate of fluorescence increase is strongly accelerated if AcrA (15 μg/ml) is also added. From [16].

functioning extremely slowly, with a turnover rate of less than one per minute. We note that the assay was carried out without the addition of Mg²⁺ and AcrA, which were needed for the activation of AcrB according to the fluorescent phospholipid efflux assay. We also do not know if a fraction of the pump was inactivated during the purification and reconstitution.

The affinity of substrates for AcrB was estimated in two ways. First, the substrates were added as potential competitors in the fluorescent phospholipid efflux assay. This showed that conjugated bile acids, such as taurocholate, were the most effective inhibitors of phospholipid efflux (Fig. 4). Second, the potential substrates were added in the proton efflux assay, and again taurocholate was the most effective among the compounds tested (Fig. 3). These results fit the notion that the natural substrate for the AcrB pump are the bile salts, most of which exist in the conjugated form in the intestinal tract of higher animals.

Soon afterward, the successful reconstitution of another RND pump, CzcA protein of *Ralstonia*, was reported [20]. CzcA catalyzes the export of toxic divalent cations such as Zn²⁺, Co²⁺, and Cd²⁺. The PCR-amplified gene was inserted into a commercial vector pASK-IBA3, which supplies a C-terminal, eight-residue tag. The protein was purified by affinity chromatography with a modified streptavidin matrix that binds the C-terminal tag [39]. It may be significant that solubilization and purification were performed in the presence of phospholipids, as specified by [40]. The purified CzcA was mixed with Triton X-100-treated liposomes, and the proteoliposomes were formed by removal of the detergent with Bio-Beads. A pH gradient was imposed across the membrane by diluting proteoliposomes containing 100 mM Tris–Cl at pH 5.0 into 100 mM Tris–Cl at pH 7.0. Since the substrates do not diffuse spontaneously across the membrane bilayer, the pumping activity can be assessed by traditional methods, and indeed vesicles were shown to accumulate divalent cations such as Zn²⁺. Better results were obtained by acidifying the vesicle interior by the diffusion of NH₃ from proteoliposomes loaded with 0.5 M NH₄Cl. The data with Zn²⁺ were reported to show a sigmoidal kinetics with a Hill coefficient of 2, although only four concentrations of the substrate were used. The half-maximal rate was achieved at a very high concentration of 6.6 mM, which casts doubt on the physiological relevance of these data. We now know that a similar cation pump, CusA captures ions from the periplasm (see Coda below); yet these reconstitution experiments were done to detect the capture of extravesicular

Fig. 4. Inhibition of the phospholipid extrusion activity of reconstituted AcrB by known substrates. The initial rates of the intermembrane lipid transfer were measured in a setup similar to Fig. 2 in the presence of erythromycin (◊), cloxacillin (♦), glycochenodeoxycholate (▽), glycocholate (∇), and taurocholate (●). The vesicles were first equilibrated with the drug. Initial rates were measured after the addition of AcrA to 25 μg/ml. From [16].

Fig. 5. Proton flux assay of reconstituted AcrD with extravesicular substrates. AcrD-containing proteoliposomes were made in KCl, and an interior-acidic pH gradient was generated by diluting them into NaCl and by adding valinomycin at 30 s. The insets show the experimental setup. The gray object denotes AcrD, with its periplasmic domain shown as ellipses. The black rectangle denotes AcrA, and the substrates are shown as “S”. A. The requirement for AcrA. Curve 2 shows that, even when gentamicin (70 μM) was present in the extravesicular space, no increase in proton efflux was seen in comparison with the control vesicles not containing AcrD (curve 1). However, when AcrA (plus Mg²⁺) was included within the vesicles (curve 3), a strong stimulation of proton efflux (presumably accompanying gentamicin transport) was seen. B. The effect of gentamicin addition in a single batch of AcrD-containing proteoliposomes with AcrA inside the vesicles. Curve 1, no antibiotic added. Curve 3, gentamicin added to the external medium. Curve 2, gentamicin and anti-AcrD polyclonal antiserum added to the external medium. C. The effect of various aminoglycosides on proton efflux. To a single batch of AcrD-containing proteoliposomes with AcrA inside, 70 μM amikacin (curve 2), tobramycin (curve 3), or gentamicin (curve 4) was added to the extravesicular space. Curve 1 shows no drug addition control. From [11].
(corresponding to cytosolic) capture of cations, although we cannot exclude the possibility that the two systems are very different.

Reconstitution of another multidrug efflux RND pump AcrD was achieved in 2005 [11]. AcrD was of special interest, as some of its substrates, aminoglycosides, are very hydrophilic and are not expected to cross membrane bilayers spontaneously. AcrD, in a form with a C-terminal hexahistidine tag, was expressed from a high copy number plasmid with isopropyl-β-D-thiogalactoside (IPTG) induction, and was purified in dodecylmaltoside. The AcrD preparations in dodecylmaltoside were used directly for proteoliposome reconstitution by octylglucoside dilution method, without the prior removal of dodecylmaltoside. Routine assays relied on the proton efflux determined with the intravesicular pH indicator pyranine, as described earlier. It was soon discovered that no acceleration of proton flux occurred, with aminoglycosides (70 μM) either outside or inside vesicles, unless AcrA (with the lipidation site removed) was added at the time of reconstitution, and thus to the interior of the vesicles (Fig. 5A). It was known that AcrA was required for the activity of AcrD in intact cells [24], but this was expected because AcrA (or one of its homologs) would be needed for the construction of a tripartite efflux machinery. In the proteoliposomes containing AcrD, a tripartite structure is not needed for efflux, and the result strongly suggested that AcrA and its homologs of the MFP family had another important function in directly activating the pumping function of the cognate RND efflux transporter. A similar situation was found more recently with the MFP MacA for the activation of an ABC efflux transporter MacB [41]. When the aminoglycosides were added to the external medium, the drugs are occupying a compartment that is similar to the cytosol in intact cells. Under such a setup, we could show that kanamycin, tobramycin, gentamicin, and amikacin produced an increased efflux of proton (Fig. 5B, 5C), presumably because the aminoglycosides were being pumped into the vesicles by AcrD. Indeed, the intravesicular accumulation of an aminoglycoside could be demonstrated by using \[^3\text{H}]\)gentamicin [11]. When aminoglycosides were added into the intravesicular space by performing reconstitution in their presence, and the vesicle interior is acidified, we get a situation in which the drugs occupy an acidic space comparable to the periplasmic space. Under such conditions, gentamicin and amikacin both strongly accelerated the proton efflux, showing directly that these substrates of the RND pump can be captured from the periplasm (Fig. 6). Another interesting finding was that one aminoglycoside with the structure very different from all others, streptomycin, stimulated proton movement only when it was incorporated into the intravesicular space (corresponding to the periplasm in intact cells), and not when it was present in extravesicular medium (corresponding to cytosol) [11]. This suggests that there are two separate ways for capturing substrates in AcrD.

## 6. Inference from structure

Very important pieces of knowledge on the mechanisms of RND pump complex came from crystallography. Thus the structure of the outer membrane channel TolC was determined in 2000 [42] (shown in red in Fig. 7A), followed by the structure of its *P. aeruginosa* homolog, OprM [43]. These proteins exist as a tightly woven trimer, containing a single 12-stranded β-barrel traversing the outer membrane, and a remarkably long (close to 100 Å) periplasmic extension of the channel as a long α-helical bundle. Murakami et al. solved the structure of AcrB as a homotrimer with a three-fold rotational symmetry in 2002 [8], the first crystallographic structure for a proton-motive-force-driven transporter solved (green in Fig. 7A; more details are shown in Fig. 7B). The periplasmic portion is at least as large as the transmembrane portion as was predicted from the amino acid sequence, and the top of the periplasmic domain, TolC-binding domain, has the dimension that is similar to the tip of the α-helical bundle of TolC. This was followed by the crystallographic elucidation of the structures of the central portion (about two-thirds) of the adaptor proteins MexA [44,45] and AcrA [46] (blue in Fig. 7A). These are elongated proteins as shown earlier [47], and in crystals show a strong tendency to become packed side-by-side to form hexamers and heptamers. Thus the structures of all three component classes of the tripartite assembly are known, and it is possible to propose how these proteins are assembled together [48] (Fig. 7A). In such models, the coiled coil domain of the adaptor is assumed to interact with the outer membrane channel protein, a hypothesis that is supported by biochemical data [49,50]. Studies with chimeric constructs of AcrA homologs showed that the C-terminal domain, whose structure is not known from crystallography, is essential for interaction with AcrB [51]. Mutant studies also suggest that the domain close to this end of AcrA, containing the α+β structure, is the main site of its interaction with the periplasmic domain of RND pumps [52].

Earlier we mentioned that AcrB likely captures some of its substrates from the periplasm, and the prediction made in the early days that drugs may first partition into the membrane-periplasm interface (Fig. 1). Indeed the AcrB trimer structure showed that there was a small opening (vestibule) between subunits at the bottom of the periplasmic domain, close to the external surface of the membrane bilayer [8], and this led to the top of the large central cavity in the transmembrane domain (Fig. 7B). It was thus hypothesized that the drugs diffused through the vestibule into the central cavity, where it was captured. Indeed, co-crystallization of AcrB with various drugs produced crystals with drug molecules within this central cavity [53]. However, there was no proof that the bound drugs were on their correct path to extrusion, and the extrusion pathways were not clear in the crystal structure. Nevertheless it must be stated that in the AcrB–YajC co-crystal obtained accidentally from *E. coli* grown in ampicillin-containing medium [54] contains two ampicillin molecules per AcrB protomer at this position.

We obtained another set of data with one site-directed mutant of AcrB, Asn109Ala [33]. In the simplest hypothesis for drug extrusion, the drug molecules are captured at the ceiling of the central cavity, enter the central pore bordered by three helices (roughly corresponding to Asp99 through Leu117) from the three protomers, and pass through this pore to reach the top of the periplasmic domain. However, the pore in the crystal structure appears closed. Since the

![Figure 6](attachment:figure_6.png)

**Fig. 6.** Proton flux assay of reconstituted AcrD with an intravesicular substrate. Two AcrD-containing proteoliposome preparations were made side-by-side with entrapped AcrA. One preparation did not contain any substrate (curve 1), but the other was made with entrapped gentamicin (curve 2). After the generation of pH gradient (inside acidic), the vesicles with entrapped gentamicin are seen to show an enhanced proton efflux, presumably because the substrate is captured by AcrD and is released into the same intravesicular space. From [11].
bound not only to the ceiling of the central cavity (as seen earlier [53]) but also to the periplasmic domain, to one side of the large, deep, external cleft. Again, there was the problem that the ligands could have become bound to any lipophilic pocket which may not be on the proper pathway for export. Changing residues that appear to be interacting with the ligands into alanine through site-directed mutagenesis produced a drastic decrease in resistance to most drugs in the case of Glu673, and more modest yet significant decreases with Phe666 and Phe664. Even these data are not decisive, however, because they may inactivate the transporter by a mechanism unrelated to ligand binding. Nevertheless the more recent asymmetric AcrB structures (discussed later in this review) are consistent with the hypothesis that this is the portal for the periplasmic entry of substrates.

Because of these difficulties, we tried a different approach. We crystallized AcrB mutants in which one of the residues in the transmembrane domain putatively involved in proton translocation (see “Site-directed mutagenesis” above) was altered [31,55]. We assumed that during the normal function cycle of AcrB, a proton antiporter, one of these residues will be altered in its protonation state, and this may cause conformational changes of the whole protein which would be coupled to ligand export. If so, our mutant proteins would mimic this transient state, and would assume a conformation similar to that of a transient intermediate during the process of drug extrusion. In the mutants examined, Asp407, Asp408, or Lys940 was changed into Ala. Since inspection of the AcrB crystal structure showed that the side-chain of Thr978 also interacts tightly with the three amino acid residues just mentioned [31], we also included this residue for analysis. When the crystal structures (without ligands) of these mutant proteins were examined [55], we found an extensive conformational alteration in the transmembrane domain (Fig. 8). It was quite reassuring that the conformational alterations were similar in all four mutants, confirming that they were due to the disruption of the salt bridge/H-bonding network among these four residues, rather than to specific effects of individual mutations. Much of the changes involve the shortening or melting of helices, except that helix 8, one of the helices connecting the transmembrane domain with the periplasmic domain, becomes extended into the periplasmic domain (Fig. 8). When the structure of the periplasmic domain was examined, however, there were only very small changes [55], a disappointing result because we expected major conformational alterations in the periplasmic domain during substrate binding and extrusion (see above). The reason for this failure was suggested from the studies describing the asymmetric trimer structure of AcrB, described below.

At about the same time as our study of the proton-relay network mutants, three papers appeared describing the asymmetric trimer structure of AcrB [56–58]. In contrast to the symmetric structures seen before, in these structures each protomer takes a unique conformation. All groups identified one protomer in which a cavity is opened up in the periplasmic domain, a likely substrate-binding site, and in the study by the Murakami group [56], this cavity was indeed found with a bound substrate molecule (minocycline or doxorubicin) (Fig. 9). This protomer is called the “binding” protomer in the nomenclature of the Murakami group. Other protomers are called the “access” and “extrusion” protomers. In the extrusion protomer, which shows the largest conformational alteration, the binding pocket becomes collapsed, and there appears to be an opening of the pathway from the binding pocket to the top (central funnel) of the periplasmic domain. Inspection of the structures of the transmembrane domains shows that the tight association of Asp407–Asp408–Lys940–Thr978 becomes disrupted in the extrusion protomer, likely as a consequence of the protonation of Asp407 [56–58]; indeed the arrangement of these side-chains here is nearly identical with that found in our mutant protein [55]. This is consistent with our hypothesis that the disruption of the network among the four

side-chain of Asn109 appears to extend deeply into the center of the pore, we naively thought that changing it to alanine might weaken the interaction of helices from the three subunits, and “loosen up” the pore. The mutant AcrB functions almost normally as judged from the resistance levels to several inhibitors. Interestingly, when the mutant protein was crystallized in the presence of several ligands (ethidium, rhodamine 6G, ciprofloxacin, nafcillin, and the inhibitor Phe-Arg-β-naphthylamide [12]), ligands were found to be

(Phe136, 178, 610, 615, 617, and 628; Val139 and 612; Ile277 and 626; and Tyr327 [57]). This pocket was indeed found to contain drugs minocycline and doxorubicin [56], and its role is also supported by the previously described mutation data with the AcrB homolog YhiV [28]. The significance of this binding is also suggested by the observation that the pocket becomes collapsed in the extrusion protomer. Finally, its physiological significance is further supported by the comparison of its surface in AcrB and AcrD (Fig. 10): although the surface is entirely hydrophobic in AcrB, in the aminoglycoside pump AcrD it contains many oxygen atoms that may function in the binding of the hydrophilic, basic substrates. How do the substrates reach the binding pocket and how do they leave the pocket? All three groups agree that there is an open pathway between the binding pocket and the external cleft of the periplasmic domain [56–58] (see Fig. 11A). Although this cleft was earlier thought by some investigators to be the site of interaction between AcrA and AcrB, at this point the possibility of substrate entry from the lateral cleft is also supported by the results of random mutagenesis data and the results of site-directed mutagenesis already described. (The entry of the substrate from the cleft was advocated by Lomovskaya and Totrov [59], before the appearance of the asymmetric crystal structures.) The Murakami group suggests in addition that the pathway to the cleft is connected to the vestibule. If this is the case, it keeps open the possibility that the substrate may go through the initial part of the vestibule, and then go into the binding pocket. In the extrusion protomer, the binding pocket becomes narrower, and the pathway to the vestibule becomes closed (Fig. 11B). At the same time, a new pathway from the binding pocket to the funnel-like structure at top becomes open. These crystallographic observations suggest that the protomers within the trimeric AcrB goes through a cyclic conformational change, from the open, “access” conformation through the ligand-bound, “binding” one to finally the “extrusion” conformer, whose transmembrane domain shows signs of disrupted network among proton-translocating residues [56–58].

## 7. Back to biochemistry

Although the structure of the asymmetric AcrB trimer suggests strongly of the mechanism of ligand extrusion coupled to proton

residues mentioned alters the conformation of the periplasmic domain. However, the asymmetric structure also shows that extensive conformational changes in the periplasmic domain require complementary accommodation by the neighboring protomers [56–58]. This was not possible in our construct, in which all protomers corresponded to the mutant protein.

As mentioned already, the extrusion protomer is the subunit where the conformation of the periplasmic domain is altered most strongly from the conformation seen in the symmetric trimer. This conformational alteration is presumably driven by the extension of helix 8 toward the periplasm, this change in turn is caused by the disruption of the salt bridge/H-bonding network among Asp407, Asp408, Lys940, and Thr978, as seen in the structures of both the wild-type protein in the asymmetric model [56–58] and the structure of the proton-relay network mutants [55].

The possible route of substrate binding and extrusion is suggested by the structure of the asymmetric trimer [56–58]. In the binding protomer, the periplasmic domain contains an expanded binding pocket containing several aromatic and hydrophobic residues

H. Nikaido, Y. Takatsuka / Biochimica et Biophysica Acta 1794 (2009) 769–781

![Image Description](image1.png)

Fig. 10. The substrate-binding pocket of the binding protomer in AcrB (left) and homology-modeled AcrD (right). The residues that are close to the surface of the pocket (136–139, 176–178, 275–278, 327, and 610–620) are shown as solid surfaces, with coloring based on the element (oxygen, red; nitrogen, blue; and carbon, white). The bound minocycline is shown in the stick representation at the center. As seen, in AcrB (based on the PDB file 2DRD) the cavity surface is dominated by hydrophobic residues such as F178, V139, F610, F628, F615, V612, I278, and I277. In the hypothetical structure of AcrD (right), the surface has become much more hydrophilic as many of these residues are replaced with hydrophilic amino acids. The direction of view is similar to the Fig. 11, although here the subject was turned clockwise approximately by 30°. The figure was drawn by using UCSF Chimera package [69].

influx, this mechanism still remains a hypothesis. We set out to prove this hypothesis by carrying out biochemical studies of the pump. We note that the proposed cyclic change involves the opening and closing of the large external cleft in the periplasmic domain (Fig. 9). Thus if the opening of the cleft is prevented, we can predict that the pump will cease to function. We introduced pairs of cysteine residues at various positions on the opposing walls of the cleft [60]. Although single cysteine mutations produced little inhibition of transport, proteins containing double mutations (Asp566Cys and Thr678Cys; Phe666Cys and Thr678Cys; Phe666Cys and Gln830Cys) were strongly compromised in function. This is likely to be due to the forced closing of the cleft by disulfide bonds, but we cannot rule out the possibility that the trimeric assembly failed because of the premature formation of disulfide bonds between protomers. Indeed with cysteine pairs at other positions, there were data suggesting this interpretation. To avoid this problem, we expressed mutated AcrB in a host strain with a defective DsbA, a periplasmic disulfide oxidoreductase that plays a major role in the formation of disulfide bonds in this compartment. Indeed in this strain, the AcrB protein containing Phe666Cys and Thr678Cys (or Gln830Cys) largely retained its transport activity, presumably because disulfide bond formation did not occur in the absence of DsbA enzyme activity. We tried to see, by using these mutants, if the cross-linking and inactivation of AcrB can be observed in “real time” by using a fast-acting disulfide cross-linking agents based on the methanethiosulfonate groups. Indeed, addition of such cross-linkers to cells that are pumping out a fluorescent dye, ethidium, was shown to stop the function of the pump instantaneously (Fig. 12), thus providing biochemical support for the rotating mechanism

![Image Description](image2.png)

Fig. 11. Cut-out view of the binding protomer with the bound minocycline (in a ball-and-stick representation) (A) and the extruding protomer (B), both from PDB file 2DRD, drawn by using UCSF Chimera package [69]. The wide passageway from the external cleft to the binding site (dashed arrow) seen in A appears to be completely closed in B.

H. Nikaido, Y. Takatsuka / Biochimica et Biophysica Acta 1794 (2009) 769–781

binding pocket and the external leaflet of the bilayer [58]. Another issue is the precise way the substrate binds to the very large substrate-binding cavity. Already the two substrates that were found in the asymmetric trimers [56], minocycline and doxorubicin, are occupying the pocket in significantly different ways. Perhaps the large size of the cavity and many potential binding modes of substrates might explain the observation that Phe-Arg-β-naphthylamide inhibits the efflux of various substrates with very different efficiency. The rotary mechanism shows that the large external cleft of the periplasmic domain of AcrB becomes closed in the extrusion protomer, but this cleft is where AcrA is generally thought to bind to AcrB, and it is unknown what happens to the bound AcrA in this protomer. Nor do we have any idea on how AcrA stimulates the function of AcrB. Finally, some RND transporters, such as MdtBC of *E. coli* [62,63], contain two protomers of different sequences, and it is unknown how the functionally rotating mechanism would work in such a situation.

The functions of the MFP, AcrA and its homologs, are described by others in this series. However, it should be mentioned that the MFP of the Cus system of *E. coli*, CusB, is known to bind the substrate Cu²⁺ [64], and most excitingly, the direct transfer of metal ion from the periplasmic binding protein CusF to CusB was recently demonstrated [65]. Thus it seems quite possible that in this system the substrate ion flows from the periplasmic binding protein to the periplasmic adaptor protein, and then finally to the periplasmic domain of an RND transporter, CusA.

There are also unresolved questions on the function of tripartite RND pumps in intact cells. For example, *P. aeruginosa* strains overexpressing a tripartite pump MexCD-OprJ were often found to have reduced function of other tripartite pumps MexAB-OprM and MexXY-OprM although the latter pumps are expressed at the normal level [66]. Another curious observation is that TonB, a protein that functions in energizing the outer membrane siderophore receptors, is needed in *P. aeruginosa* for the function of the MexAB-OprM pump [67,68]. As far as the authors are aware, there is no explanation of this effect at the molecular level, but this reminds us that a small protein YajC was accidentally co-crystallized with AcrB [54]. Although the deletion of gene *yajC* had only a small effect on drug susceptibility, this observation certainly requires further follow-up study, especially because this co-crystal was obtained from cells expressing both YajC and AcrB at normal constitutive levels, without the lopsided, plasmid-driven overexpression of only AcrB.

## Acknowledgements

The studies in the authors' laboratory are supported by a U.S. Public Health Service grant, AI 09644. We thank V. Koronakis for supplying the high resolution version of Fig. 7A.

## References

[1] I.T. Paulsen, J.H. Park, P.S. Choi, M.H. Saier, A family of gram-negative bacterial outer membrane factors that function in the export of proteins, carbohydrates, drugs and heavy metals from gram-negative bacteria, FEMS Microbiol. Lett. 156 (1997) 1–8.

[2] T. Dinh, I.T. Paulsen, M.H. Saier, A family of extracytoplasmic proteins that allow transport of large molecules across the outer membranes of gram-negative bacteria, J. Bacteriol. 176 (1994) 3825–3831.

[3] D. Ma, D.N. Cook, M. Alberti, N.G. Pon, H. Nikaido, J.E. Hearst, Molecular cloning and characterization of *acrA* and *acrE* genes of *Escherichia coli*, J. Bacteriol. 175 (1993) 6299–6313.

[4] D. Ma, D.N. Cook, M. Alberti, N.G. Pon, H. Nikaido, J.E. Hearst, Genes *acrA* and *acrB* encode a stress-induced efflux system of *Escherichia coli*, Mol. Microbiol. 16 (1995) 45–55.

[5] H. Nikaido, Multidrug efflux pumps of gram-negative bacteria, J. Bacteriol. 178 (1996) 5853–5859.

[6] M. Vaara, Antibiotic-supersusceptible mutants of *Escherichia coli* and *Salmonella typhimurium*, Antimicrob. Agents Chemother. 37 (1993) 2255–2260.

[7] H. Nikaido, M. Basina, V. Nguyen, E.Y. Rosenberg, Multidrug efflux pump AcrAB of *Salmonella typhimurium* excretes only those beta-lactam antibiotics containing lipophilic side chains, J. Bacteriol. 180 (1998) 4686–4692.

[8] S. Murakami, R. Nakashima, E. Yamashita, A. Yamaguchi, Crystal structure of bacterial multidrug efflux transporter AcrB, Nature 419 (2002) 587–593.

[9] X.Z. Li, H. Nikaido, K. Poole, Role of mexA–mexB–oprM in antibiotic efflux in *Pseudomonas aeruginosa*, Antimicrob. Agents Chemother. 39 (1995) 1948–1953.

[10] X.Z. Li, D. Ma, D.M. Livermore, H. Nikaido, Role of efflux pump(s) in intrinsic resistance of *Pseudomonas aeruginosa*: active efflux as a contributing factor to beta-lactam resistance, Antimicrob. Agents Chemother. 38 (1994) 1742–1752.

[11] J.R. Aires, H. Nikaido, Aminoglycosides are captured from both periplasm and cytoplasm by the AcrD multidrug efflux transporter of *Escherichia coli*, J. Bacteriol. 187 (2005) 1923–1929.

[12] O. Lomovskaya, M.S. Warren, A. Lee, J. Galazzo, R. Fronko, M. Lee, J. Blais, D. Cho, S. Chamberland, T. Renau, R. Leger, S. Hecker, W. Watkins, K. Hoshino, H. Ishida, V.J. Lee, Identification and characterization of inhibitors of multidrug resistance efflux pumps in *Pseudomonas aeruginosa*: novel agents for combination therapy, Antimicrob. Agents Chemother. 45 (2001) 105–116.

[13] A. Lee, W. Mao, M.S. Warren, A. Mistry, K. Hoshino, R. Okumura, H. Ishida, O. Lomovskaya, Interplay between efflux pumps may provide either additive or multiplicative effects on drug resistance, J. Bacteriol. 182 (2000) 3142–3150.

[14] N. Tamura, S. Konishi, A. Yamaguchi, Mechanisms of drug/H⁺ antiport: complete cysteine-scanning mutagenesis and the protein engineering approach, Curr. Opin. Chem. Biol. 7 (2003) 570–579.

[15] I.T. Paulsen, M.H. Brown, T.G. Littlejohn, B.A. Mitchell, R.A. Skurray, Multidrug resistance proteins QacA and QacB from *Staphylococcus aureus*: membrane topology and identification of residues involved in substrate specificity, Proc. Natl. Acad. Sci. U. S. A. 93 (1996) 3630–3635.

[16] H.I. Zgurskaya, H. Nikaido, Bypassing the periplasm: reconstitution of the AcrAB multidrug efflux pump of *Escherichia coli*, Proc. Natl. Acad. Sci. U. S. A. 96 (1999) 7190–7195.

[17] C.C. Su, E.W. Yu, Ligand-transporter interaction in the AcrB multidrug efflux pump determined by fluorescence polarization assay, FEBS Lett. 581 (2007) 4972–4976.

[18] A. Ocaktan, H. Yoneyama, T. Nakae, Use of fluorescence probes to monitor function of the subunit proteins of the MexA–MexB–oprM drug extrusion machinery in *Pseudomonas aeruginosa*, J. Biol. Chem. 272 (1997) 21964–21969.

[19] S. Murakami, N. Tamura, A. Saito, T. Hirata, A. Yamaguchi, Extramembrane central pore of multidrug exporter AcrB in *Escherichia coli* plays an important role in drug transport, J. Biol. Chem. 279 (2004) 3743–3748.

[20] M. Goldberg, T. Pribyl, S. Juhnke, D.H. Nies, Energetics and topology of CzcA, a cation/proton antiporter of the resistance-nodulation-cell division protein family, J. Biol. Chem. 274 (1999) 26065–26070.

[21] T.Y. Chang, C.C. Chang, N. Ohgami, Y. Yamauchi, Cholesterol sensing, trafficking, and esterification, Annu. Rev. Cell Dev. Biol. 22 (2006) 129–157.

[22] R.E. Infante, L. Abi-Mosleh, A. Radhakrishnan, J.D. Dale, M.S. Brown, J.L. Goldstein, Purified NPC1 protein. I. Binding of cholesterol and oxysterols to a 1278-amino acid membrane protein, J. Biol. Chem. 283 (2008) 1052–1063.

[23] R.E. Infante, A. Radhakrishnan, L. Abi-Mosleh, L.N. Kinch, M.L. Wang, N.V. Grishin, J.L. Goldstein, M.S. Brown, Purified NPC1 protein: II. Localization of sterol binding to a 240-amino acid soluble luminal loop, J. Biol. Chem. 283 (2008) 1064–1075.

[24] C.A. Elkins, H. Nikaido, Substrate specificity of the RND-type multidrug efflux pumps AcrB and AcrD of *Escherichia coli* is determined predominantly by two large periplasmic loops, J. Bacteriol. 184 (2002) 6490–6498.

[25] E.B. Tikhonova, Q. Wang, H. Zgurskaya, Chimeric analysis of the multicomponent multidrug efflux transporters from Gram-negative bacteria, J. Bacteriol. 184 (2002) 6499–6507.

[26] S. Eda, H. Maseda, T. Nakae, An elegant means of self-protection in gram-negative bacteria by recognizing and extruding xenobiotics from the periplasmic space, J. Biol. Chem. 278 (2003) 2085–2088.

[27] W. Mao, M.S. Warren, D.S. Black, T. Satou, T. Murata, T. Nishino, N. Gotoh, O. Lomovskaya, On the mechanism of substrate specificity by resistance nodulation division (RND)-type multidrug resistance pumps: the large periplasmic loops of MexD from *Pseudomonas aeruginosa* are involved in substrate recognition, Mol. Microbiol. 46 (2002) 889–901.

[28] J.A. Bohnert, S. Schuster, E. Fähnrich, R. Trittlér, W.V. Kern, Altered spectrum of multidrug resistance associated with a single point mutation in the *Escherichia coli* RND-type MDR efflux pump YhiV (MdtF), J. Antimicrob. Chemother. 59 (2007) 1216–1222.

[29] J.K. Middlemiss, K. Poole, Differential impact of MexB mutations on substrate selectivity of the MexAB-OprM multidrug efflux pump of *Pseudomonas aeruginosa*, J. Bacteriol. 186 (2004) 1258–1269.

[30] L. Guan, T. Nakae, Identification of essential charged residues in transmembrane segments of the multidrug transporter MexB of *Pseudomonas aeruginosa*, J. Bacteriol. 183 (2001) 1734–1739.

[31] Y. Takatsuka, H. Nikaido, Threonine-978 in the transmembrane segment of the multidrug efflux pump AcrB of *Escherichia coli* is crucial for drug transport as a probable component of the proton relay network, J. Bacteriol. 188 (2006) 7284–7289.

[32] E.W. Yu, J.R. Aires, H. Nikaido, AcrB multidrug efflux pump of *Escherichia coli*: composite substrate-binding cavity of exceptional flexibility generates its extremely wide substrate specificity, J. Bacteriol. 185 (2003) 5657–5664.

[33] E.W. Yu, J.R. Aires, G. McDermott, H. Nikaido, A periplasmic drug-binding site of the AcrB multidrug efflux pump: a crystallographic and site-directed mutagenesis study, J. Bacteriol. 187 (2005) 6804–6815.

[34] E.M. Hearn, M.R. Gray, J.M. Foght, Mutations in the central cavity and periplasmic domain affect efflux activity of the resistance-nodulation-division pump EmhB from *Pseudomonas fluorescens* cLP6a, J. Bacteriol. 188 (2006) 115–123.

[35] S. Franke, G. Grass, C. Rensing, D.H. Nies, Molecular analysis of the copper-transporting efflux system CusCFBA of *Escherichia coli*, J. Bacteriol. 185 (2003) 3804–3812.

[36] V. Dastidar, W. Mao, O. Lomovskaya, H.I. Zgurskaya, Drug-induced conformational changes in multidrug efflux transporter AcrB from *Haemophilus influenzae*, J. Bacteriol. 189 (2007) 5550–5558.

[37] S. Ruetz, P. Gros, Phosphatidylcholine translocase: a physiological role for the *mdr2* gene, Cell 77 (1994) 1071–1081.

[38] W. Mao, M.S. Warren, A. Lee, A. Mistry, O. Lomovskaya, MexXY–OprM efflux pump is required for antagonism of aminoglycosides by divalent cations in *Pseudomonas aeruginosa*, Antimicrob. Agents Chemother. 45 (2001) 2001–2007.

[39] T.G. Schmidt, A. Skerra, The Strep-tag system for one-step purification and high-affinity detection or capturing of proteins, Nature Protocols 2 (2007) 1528–1535.

[40] K. Hanada, I. Yamato, Y. Anraku, Purification and reconstitution of *Escherichia coli* proline carrier using a site specifically cleavable fusion protein, J. Biol. Chem. 263 (1988) 7181–7185.

[41] E.B. Tikhonova, V.K. Devroy, S.Y. Lau, H.I. Zgurskaya, Reconstitution of the *Escherichia coli* macrolide transporter: the periplasmic membrane fusion protein MacA stimulates the ATPase activity of MacB, Mol. Microbiol. 63 (2007) 895–910.

[42] V. Koronakis, A. Sharff, E. Koronakis, B. Luisi, C. Hughes, Crystal structure of the bacterial membrane protein TolC central to multidrug efflux and protein export, Nature 405 (2000) 914–919.

[43] H. Akama, M. Kanemaki, M. Yoshimura, T. Tsukihara, T. Kashiwagi, H. Yoneyama, S. Narita, A. Nakagawa, T. Nakae, Crystal structure of the drug discharge outer membrane protein, OprM, of *Pseudomonas aeruginosa*: dual modes of membrane anchoring and occluded cavity end, J. Biol. Chem. 279 (2004) 52816–52819.

[44] H. Akama, T. Matsuura, S. Kashiwagi, H. Yoneyama, S. Narita, T. Tsukihara, A. Nakagawa, T. Nakae, Crystal structure of the membrane fusion protein, MexA, of the multidrug transporter in *Pseudomonas aeruginosa*, J. Biol. Chem. 279 (2004) 25939–25942.

[45] M.K. Higgins, E. Bokma, E. Koronakis, C. Hughes, V. Koronakis, Structure of the periplasmic component of a bacterial drug efflux pump, Proc. Natl. Acad. Sci. U. S. A. 101 (2004) 9994–9999.

[46] J. Mikolosko, K. Bobyk, H.I. Zgurskaya, P. Ghosh, Conformational flexibility in the multidrug efflux system protein AcrA, Structure 14 (2006) 577–587.

[47] H.I. Zgurskaya, H. Nikaido, AcrA is a highly asymmetric protein capable of spanning the periplasm, J. Mol. Biol. 285 (1999) 409–420.

[48] J. Eswaran, E. Koronakis, M.K. Higgins, C. Hughes, V. Koronakis, Three's company: component structures bring a closer view of tripartite drug efflux pumps, Curr. Opin. Struct. Biol. 14 (2004) 741–747.

[49] J.F. Stegmeier, G. Polleichtner, N. Brandes, C. Hotz, C. Andersen, Importance of the adaptor (membrane fusion) protein hairpin domain for the functionality of multidrug efflux pumps, Biochemistry 45 (2006) 10303–10312.

[50] S. Lobedanz, E. Bokma, M.F. Symmons, E. Koronakis, C. Hughes, V. Koronakis, A periplasmic coiled-coil interface underlying TolC recruitment and the assembly of bacterial drug efflux pumps, Proc. Natl. Acad. Sci. U. S. A. 104 (2007) 4612–4617.

[51] C.A. Elkins, H. Nikaido, Chimeric analysis of AcrA function reveals the importance of its C-terminal domain in its interaction with the AcrB multidrug efflux pump, J. Bacteriol. 185 (2003) 5349–5356.

[52] G. Krishnamoorthy, E.B. Tikhonova, H.I. Zgurskaya, Fitting periplasmic membrane fusion proteins to inner membrane transporters: mutations that enable *Escherichia coli* AcrA to function with *Pseudomonas aeruginosa* MexB, J. Bacteriol. 190 (2007) 691–698.

[53] E.W. Yu, G. McDermott, H.I. Zgurskaya, H. Nikaido, D.E. Koshland, Structural basis of multiple drug-binding capacity of the AcrB multidrug efflux pump, Science 300 (2003) 976–980.

[54] S. Törnroth-Horsefield, P. Gourdon, R. Horsefield, L. Brive, N. Yamamoto, H. Mori, A. Snijder, R. Neutze, Crystal structure of AcrB in complex with a single transmembrane subunit reveals another twist, Structure 15 (2007) 1663–1673.

[55] C.C. Su, M. Li, R. Gu, Y. Takatsuka, G. McDermott, H. Nikaido, E.W. Yu, Conformation of the AcrB multidrug efflux pump in mutants of the putative proton relay pathway, J. Bacteriol. 188 (2006) 7290–7296.

[56] S. Murakami, R. Nakashima, E. Yamashita, T. Matsumoto, A. Yamaguchi, Crystal structures of a multidrug transporter reveal a functionally rotating mechanism, Nature 443 (2006) 173–179.

[57] M.A. Seeger, A. Schiefner, T. Eicher, F. Verrey, K. Diederichs, K.M. Pos, Structural asymmetry of AcrB trimer suggests a peristaltic pump mechanism, Science 313 (2006) 1295–1298.

[58] G. Sennhauser, P. Amstutz, C. Briand, O. Storchenegger, M.G. Grutter, Drug export pathway of multidrug exporter AcrB revealed by DARPin inhibitors, PLoS Biol. 5 (2007) e7.

[59] O. Lomovskaya, M. Totrov, Vacuuming the periplasm, J. Bacteriol. 187 (2005) 1879–1883.

[60] Y. Takatsuka, H. Nikaido, Site-directed disulfide cross-linking shows that cleft flexibility in the periplasmic domain is needed for the multidrug efflux pump AcrB of *Escherichia coli*, J. Bacteriol. 189 (2007) 8677–8684.

[61] M.A. Seeger, C. von Ballmoos, T. Eicher, L. Brandstetter, F. Verrey, K. Diederichs, K.M. Pos, Engineered disulfide bonds support the functional rotation mechanism of multidrug efflux pump AcrB, Nat. Struct. Mol. Biol. 15 (2008) 199–205.

[62] N. Baranova, H. Nikaido, The BaeSR two-component regulatory system activates transcription of the *yegMNOB (mdtABCD)* transporter gene cluster in *Escherichia coli* and increases its resistance to novobiocin and deoxycholate, J. Bacteriol. 184 (2002) 4168–4176.

[63] S. Nagakubo, K. Nishino, T. Hirata, A. Yamaguchi, The putative response regulator BaeR stimulates multidrug resistance of *Escherichia coli* via a novel multidrug exporter system, MdtABC, J. Bacteriol. 184 (2002) 4161–4167.

[64] I. Bagai, W. Liu, C. Rensing, N.J. Blackburn, M.M. McEvoy, Substrate-linked conformational change in the periplasmic component of a Cu(I)/Ag(I) efflux system, J. Biol. Chem. 282 (2007) 35695–35702.

[65] I. Bagai, W. Liu, C. Rensing, N.J. Blackburn, M.M. McEvoy, Direct metal transfer between periplasmic proteins identifies a bacterial copper chaperone, Biochemistry (2008) e-published 10/11/2008.

[66] K. Jeannot, S. Elsen, T. Kohler, I. Attree, C. van Delden, P. Plesiat, Resistance and virulence of *Pseudomonas aeruginosa* clinical strains overproducing the MexCD-OprJ efflux pump, Antimicrob. Agents Chemother. 52 (2008) 2455–2462.

[67] Q. Zhao, X.Z. Li, A. Mistry, R. Srikumar, L. Zhang, O. Lomovskaya, K. Poole, Influence of the TonB energy-coupling protein on efflux-mediated multidrug resistance in *Pseudomonas aeruginosa*, Antimicrob. Agents Chemother. 42 (1998) 2225–2231.

[68] Q. Zhao, K. Poole, Differential effects of mutations in *tonB1* on intrinsic multidrug resistance and iron acquisition in *Pseudomonas aeruginosa*, J. Bacteriol. 184 (2002) 2045–2049.

[69] E.F. Pettersen, T.D. Goddard, C.C. Huang, G.S. Couch, D.M. Greenblatt, E.C. Meng, T.E. Ferrin, UCSF Chimera—a visualization system for exploratory research and analysis, J. Comput. Chem. 25 (2004) 1605–1612.
