
Cellular Microbiology (2009) 11(3), 442–460

doi:10.1111/j.1462-5822.2008.01266.x  
First published online 2 December 2008

**The inositol polyphosphate 5-phosphatase OCRL1 restricts intracellular growth of Legionella, localizes to the replicative vacuole and binds to the bacterial effector LpnE**

Stefan S. Weber, Curdin Ragaz and Hubert Hilbi*

Institute of Microbiology, ETH Zürich, Wolfgang-Pauli-Strasse 10, 8093 Zürich, Switzerland.

---

### Summary

*Legionella pneumophila*, the causative agent of Legionnaires’ disease, replicates within a specific vacuole in amoebae and macrophages. To form these ‘Legionella-containing vacuoles’ (LCVs), the bacteria employ the Icm/Dot type IV secretion system and effector proteins, some of which anchor to the LCV membrane via the host glycolipid phosphatidylinositol 4-phosphate [PtdIns(4)P]. Here we analysed the role of inositol polyphosphate 5-phosphatases (IP5Ps) during *L. pneumophila* infections. Bacterial replication and LCV formation occurred more efficiently in *Dictyostelium discoideum* amoebae lacking the IP5P Dd5P4, a homologue of human OCRL1 (Oculocerebrorenal syndrome of Lowe), implicated in retrograde endosome to Golgi trafficking. The phenotype was complemented by Dd5P4 but not the catalytically inactive 5-phosphatase. Ectopically expressed Dd5P4 or OCRL1 localized to LCVs in *D. discoideum* via an N-terminal domain previously not implicated in membrane targeting, and OCRL1 was also identified on LCVs in macrophages. Dd5P4 was catalytically active on LCVs and accumulated on LCVs harbouring wild-type but not ΔicmT mutant *L. pneumophila*. The N-terminal domain of OCRL1 bound *L. pneumophila* LpnE, a Sel1-like repeat protein involved in LCV formation, which localizes to LCVs and selectively binds PtdIns(3)P. Our results indicate that OCRL1 restricts intracellular growth of *L. pneumophila* and binds to LCVs in association with LpnE.

---

### Introduction

*Legionella pneumophila* is an environmental bacterium that parasitizes various species of predatory amoebae in water and soil, including the social amoeba *Dictyostelium discoideum* (Fields *et al.*, 2002; Steinert and Heuner, 2005). The bacteria are accidentally transmitted to the human lung via contaminated aerosols and thus may cause the severe pneumonia Legionnaires’ disease, or a milder illness termed Pontiac fever. *L. pneumophila* replicates in alveolar macrophages (Nash *et al.*, 1984), using a mechanism likely adopted through its longstanding evolutionary interaction with amoebae (Greub and Raoult, 2004; Molmeret *et al.*, 2005; Hilbi *et al.*, 2007). To transfer between host cells, *L. pneumophila* switches from a replicative to a transmissive phase, which involves a complex regulatory network including an apparent quorum sensing system (Spirig *et al.*, 2008; Tiaden *et al.*, 2007; 2008).

Within macrophages and amoebae, *L. pneumophila* forms a replicative compartment, the *Legionella*-containing vacuole (LCV). LCVs avoid fusion with lysosomes (Horwitz, 1983), instead intercept early secretory trafficking at endoplasmic reticulum (ER) exit sites (Kagan and Roy, 2002) and likely also communicate with other trafficking routes (Dorer *et al.*, 2006). In the course of their maturation, LCVs accumulate the v-SNARE Sec22b, as well as the small GTPases Arf1 and Rab1, the activity of which is required to form the replicative niche (Kagan and Roy, 2002; Derre and Isberg, 2004; Kagan *et al.*, 2004). At later stages, LCVs fuse with the ER and acquire the ER resident protein calnexin (Swanson and Isberg, 1995; Tilney *et al.*, 2001; Robinson and Roy, 2006), and calnexin-positive LCVs undergo a transition from ‘tight’ to ‘spacious’ vacuoles (Lu and Clarke, 2005).

Uptake of *L. pneumophila* and the formation of LCVs in macrophages and amoebae requires a type IV secretion system (T4SS), the Icm/Dot apparatus (Segal *et al.*, 1998; Vogel *et al.*, 1998; Segal and Shuman, 1999; Solomon *et al.*, 2000; Hilbi *et al.*, 2001; Watarai *et al.*, 2001; Otto *et al.*, 2004). More than 70 Icm/Dot substrates (‘effector proteins’) have been identified, some of which interfere with host cell processes such as apoptosis, GTP cycles,

Received 14 August, 2008; revised 7 November, 2008; accepted 10 November, 2008. *For correspondence. E-mail hilbi@micro.biol.ethz.ch; Tel. (+41) 44 632 4782; Fax (+41) 44 632 1137.*

© 2008 Blackwell Publishing Ltd

cellular microbiology
or phosphoinositide metabolism (Segal *et al.*, 2005; Brüggemann *et al.*, 2006; Hilbi, 2006; Ninio and Roy, 2007). The Icm/Dot substrate SidC and its paralogue SdcA localize to the LCV membrane (Luo and Isberg, 2004), where the proteins specifically anchor to phosphatidylinositol 4-phosphate [PtdIns(4)P] (Weber *et al.*, 2006). SidC is a bifunctional effector, which recruits ER vesicles to replication-permissive LCVs (Ragaz *et al.*, 2008). A putative effector protein apparently not secreted by the Icm/Dot T4SS is LpnE, which was identified by genomic subtractive hybridization of the *L. pneumophila* and *L. micdadei* genomes (Newton *et al.*, 2006; 2007). LpnE is a member of the Sel1-like repeat domain-containing EnhC enhanced entry protein family, implicated in the efficient infection of *A. castellani* and THP-1 macrophages as well as LCV trafficking.

Phosphoinositides are pivotal players in eukaryotic receptor-mediated signal transduction, actin remodeling and membrane dynamics (De Matteis and Godi, 2004; Di Paolo and De Camilli, 2006). The inositol headgroup of these glycolipids can be phosphorylated by kinases or dephosphorylated by phosphatases at three positions, leading to seven different molecules, which recruit specific effector proteins to distinct membranes. Given the crucial importance of phosphoinositides in membrane dynamics and vesicle trafficking, it is not surprising that a number of pathogenic bacteria subvert host cell phosphoinositide metabolism (Pizarro-Cerda and Cossart, 2004; Hilbi, 2006).

Phosphoinositide 3-kinases (PI3Ks) play a role in phagosome formation and maturation (Gillooly *et al.*, 2001), as well as in retrograde endosome to trans Golgi network (TGN) trafficking (Bonifacino and Rojas, 2006). While PI3Ks are dispensable for the uptake of *L. pneumophila* (Khelef *et al.*, 2001; Weber *et al.*, 2006), inhibition or deletion of PI3Ks promotes intracellular replication of *L. pneumophila* in *D. discoideum*, inhibits the transition from ‘tight’ to ‘spacious’ LCVs, increases the amount of SidC bound to LCVs and protects *L. pneumophila ΔicmT* mutants from degradation (Weber *et al.*, 2006).

Inositol polyphosphate 5-phosphatases (IP5Ps), such as the mammalian enzymes OCRL1 and INPP5B, are implicated in the retrograde pathway from the endosome to the TGN or from the ER-to-Golgi intermediate compartment (ERGIC) to ER respectively (Lowe, 2005; Williams *et al.*, 2007). Furthermore, as OCRL1 localizes to endosomes as well as to the TGN and TGN-derived clathrin vesicles, the enzyme might also promote secretory transport from the TGN to endosomes (Dressman *et al.*, 2000; Choudhury *et al.*, 2005). OCRL1 (but not INPP5B) associates with clathrin-coated transport intermediates and endocytic pits, and directly interacts with clathrin and the endocytic clathrin adaptor complex AP2 (Ungewickell *et al.*, 2004; Choudhury *et al.*, 2005; Erdmann *et al.*, 2007). OCRL1 also binds to the GTPases Arf1 (secretory pathway) and Arf6 (endosomal trafficking) (Lichter-Konecki *et al.*, 2006), as well as to Rab GTPases associated with the Golgi apparatus (Rab1, Rab6) or endosomes (Rab5) (Hyvola *et al.*, 2006) and to the Rab5 effector APPL1 (Erdmann *et al.*, 2007). OCRL1 is a type II IP5P that preferentially dephosphorylates the phosphoinositide lipid PtdIns(4,5)P₂ yielding PtdIns(4)P (Zhang *et al.*, 1995). Defects in OCRL1 cause **Oculo-cerebrorenal** syndrome of **Lowe** (Attree *et al.*, 1992), a rare X-linked disorder characterized by mental retardation, congenital cataracts and renal failure.

OCRL1 and INPP5B are 45% identical and share a catalytically active inositol polyphosphate 5-phosphatase domain and an inactive Rho GAP domain. This domain architecture is conserved in one out of four distinct *D. discoideum* IP5Ps (Dd5P1–4), namely Dd5P4 (Loovers *et al.*, 2003). While the deletion of either Dd5P1, -2 or -3 did not produce obvious phenotypes, the ablation of *D. discoideum* Dd5P4 caused developmental defects, lower growth rates in axenic medium as well as on bacterial lawns and impaired phagocytosis. OCRL1 is a close mammalian orthologue of Dd5P4 sharing 33% overall amino acid identity with the amoeba enzyme (Fig. 1). Upon expression of OCRL1 in *D. discoideum* lacking Dd5P4, phagocytosis, growth and developmental defects were partially restored, and thus, OCRL1 can functionally replace Dd5P4 (Loovers *et al.*, 2007).

Here we analyse the role of IP5Ps during phagocytosis and intracellular replication of *L. pneumophila*. We find that Dd5P4/OCRL1 attenuates intracellular replication of *L. pneumophila*, localizes to LCVs via its N-terminus and binds to the *L. pneumophila* PtdIns(3)P-binding effector protein LpnE.

### Results

#### Dd5P4 attenuates intracellular replication of wild-type *L. pneumophila*

IP5Ps play a role in membrane dynamics and vesicle trafficking in mammals as well as in *D. discoideum* (Lowe, 2005; Loovers *et al.*, 2007). To assess whether IP5Ps are involved in the uptake of *L. pneumophila* and formation of LCVs, we used *D. discoideum* mutant strains lacking individual IP5Ps (*ΔDd5P1*, -2, -3 or -4). Compared with wild-type *D. discoideum*, the uptake of wild-type *L. pneumophila* was reduced in *ΔDd5P1* or *ΔDd5P4* by approximately 40% and not affected in *ΔDd5P2* or *ΔDd5P3* (Fig. 2A). Phagocytosis of *L. pneumophila* lacking a functional Icm/Dot T4SS (*ΔicmT*) was not affected by the absence of any of the four IP5Ps. Moreover, as described previously for wild-type *D. discoideum*
444 S. S. Weber, C. Ragaz and H. Hilbi

| Dd5P4 | LVA | IP5P | ASH | Rho GAP |
| --- | --- | --- | --- | --- |
| 33 Identity (%) | 21 | 46 | 38 | 36 |
| OCRL1 | LVA | IP5P | ASH | Rho GAP |
|  | FEDNF | LIDLE |

Fig. 1. Domain architecture of the *D. discoideum* inositol polyphosphate 5-phosphatase Dd5P4 and the mammalian orthologue OCRL1. Highlighted are the domains (i) LVA (*Legionella* vacuole association), (ii) IP5P (inositol polyphosphate 5-phosphatase) including the catalytically essential aspartic acid residues (arrows), (iii) ASH (ASPM-SPD2-Hydin), and (iv) Rho GAP (GTPase activating protein) including the catalytically essential arginine residue (arrow), which is replaced by isoleucine or glutamine in Dd5P4 and OCRL1 respectively. The percentage identity of the full-length proteins and the individual domains are indicated. OCRL1 binds to vacuoles containing Icm/Dot-proficient *L. pneumophila* (LVA domain), to α-adaptin of the AP2 complex (FEDNF motif) (Ungewickell et al., 2004), to clathrin (LIDLE motif, amino acids 539–600) (Choudhury et al., 2005; Erdmann et al., 2007), to the GTPases Rab1, -5, -6 (amino acids 540–726) (Hyvola et al., 2006), Arf1 and Arf6 (Rho GAP domain) (Lichter-Konecki et al., 2006), as well as to the Rab5 effector APPL1 (ASH, Rho GAP) (Erdmann et al., 2007).

(Weber et al., 2006), amoebae lacking the IP5Ps phagocytosed approximately 10 times more wild-type *L. pneumophila* than Δ*icmT* mutant bacteria.

Interestingly, approximately 100- to 1000-fold more wild-type *L. pneumophila* were released from ΔDd5P4 compared with wild-type *D. discoideum* or compared with mutant amoebae lacking the other IP5Ps (Fig. 2B), indicating that *L. pneumophila* replicates much more efficiently in absence of Dd5P4. To quantify intracellular *L. pneumophila*, infected amoebae were lysed with saponin (Fig. 2C). Under these conditions, intracellular replication of *L. pneumophila* was found to initiate earlier and to proceed at a higher rate in ΔDd5P4 compared with wild-type *D. discoideum*. A more efficient intracellular replication in absence of Dd5P4 was also observed by flow cytometry using GFP-labelled *L. pneumophila* (Fig. 2D). The degradation of *L. pneumophila* Δ*icmT* by amoebae was not affected in the absence of Dd5P4 (Fig. 2D) or any other IP5P (data not shown).

In order to analyse complementation of the intracellular growth phenotype, GFP fusion constructs of Dd5P4 and fragments thereof were constitutively expressed in *D. discoideum* ΔDd5P4 under the control of the act15 promoter. The production of full-length Dd5P4<sub>1-787</sub>-GFP reduced intracellular growth of wild-type *L. pneumophila* in Dd5P4-deficient host cells almost to the level observed in wild-type *D. discoideum* (Fig. 2E). In contrast, upon production of the catalytically inactive mutant protein Dd5P4<sub>1-787(D319A)</sub>-GFP, *L. pneumophila* grew as efficiently as in ΔDd5P4. The production of Dd5P4<sub>1-465</sub>-GFP containing the catalytic 5-phosphatase domain in ΔDd5P4 suppressed the growth of wild-type *L. pneumophila* at early time points, while the production of Dd5P4<sub>466-787</sub>-GFP in Dd5P4-deficient host cells did not affect intracellular growth of the bacteria (Fig. 2F). These results indicate that catalytic activity of Dd5P4 is required to attenuate intracellular replication of *L. pneumophila*. The ectopic expression of OCRL1 in ΔDd5P4 did not result in reduced intracellular replication (data not shown). This lack of complementation might be due to less efficient production of the mammalian protein in *D. discoideum* amoebae. Flow cytometry analysis revealed that Dd5P4-GFP was produced by all amoebae (>99%) and at higher levels compared with OCRL1-GFP, which was produced by only a fraction of the amoebae (approximately 40%) and at lower levels (Fig. S1).

**Dd5P4 attenuates the maturation and dynamics of LCVs**

The maturation of LCVs involves the acquisition of the ER-resident protein calnexin (Swanson and Isberg, 1995; Tilney et al., 2001; Robinson and Roy, 2006) and a transition of calnexin-positive LCVs from ‘tight’ to ‘spacious’ vacuoles (Lu and Clarke, 2005). In wild-type *D. discoideum*, approximately 70% LCVs turned calnexin-positive within 1 h post infection (Fig. 3A). Throughout this time period, LCVs recruited calnexin-GFP more efficiently in absence of Dd5P4, indicating that Dd5P4 attenuates the interaction and fusion of ER with LCVs. Moreover, within 4.5 h approximately 25%-40% LCVs underwent a transition from tight to spacious vacuoles in wild-type *D. discoideum*, regardless of whether Dd5P4-GFP or a catalytically inactive mutant protein was ectopically expressed or not (Fig. 3B). In contrast, the LCV transition was almost completely blocked in strain ΔDd5P4, as in absence of Dd5P4 less than 10% spacious LCVs were observed 4.5 h post infection. The transition from tight to spacious LCVs was complemented by ectopically

© 2008 Blackwell Publishing Ltd, Cellular Microbiology, 11, 442–460

Fig. 2. Dd5P4 attenuates intracellular replication of wild-type *L. pneumophila*.

A. Phagocytosis of GFP-labelled wild-type (black bars) or ΔicmT mutant *L. pneumophila* (grey bars) by wild-type *D. discoideum* or ΔDd5P1–4 was determined by flow cytometry.

B. Release of wild-type *L. pneumophila* into the supernatant of infected wild-type *D. discoideum* (■), ΔDd5P1 (◆), ΔDd5P2 (▲), ΔDd5P3 (●) or ΔDd5P4 (□) was quantified by colony forming units (cfu).

C. Single round infections of wild-type *L. pneumophila* in wild-type *D. discoideum* (■) or ΔDd5P4 (□) were assessed by lysing the infected cells with 0.8% saponin and determining bacterial cfu.

D. Quantification by flow cytometry of intracellular growth of GFP-labelled wild-type *L. pneumophila* or degradation of GFP-labelled ΔicmT within wild-type *D. discoideum* or ΔDd5P4.

E. Release of wild-type *L. pneumophila* into the supernatant of infected wild-type *D. discoideum* (■), ΔDd5P4 (□), ΔDd5P4 producing Dd5P4<sub>1-787</sub>-GFP (Δ) or ΔDd5P4 producing phosphatase-inactive Dd5P4<sub>1-787(D319A)</sub>-GFP (○) was quantified by determining cfu.

F. Release of wild-type *L. pneumophila* into the supernatant of infected wild-type *D. discoideum* (■) or ΔDd5P4 producing Dd5P4<sub>1-787</sub>-GFP (Δ), phosphatase-inactive Dd5P4<sub>1-787(D319A)</sub>-GFP (○), Dd5P4<sub>1-465</sub>-GFP (▲) or Dd5P4<sub>466-787</sub>-GFP (●) was quantified by determining cfu.

*P* (two-tailed Student’s *t*-test) < 0.02 (A), < 0.002 (B), < 0.01 (C).

Fig. 3. Dd5P4 attenuates the maturation and dynamics of LCVs.

A. Calnexin–GFP-labelled wild-type *D. discoideum* (■) or ΔDd5P4 (□) were infected with DsRed-labelled wild-type *L. pneumophila* (moi of 50), and at the time points indicated, the percentage of calnexin–GFP-positive LCVs (of 100 LCVs total per time point) was scored by confocal laser fluorescence microscopy. Similar results were obtained in two independent experiments.

B. Wild-type *D. discoideum* (filled symbols) or ΔDd5P4 (open symbols) harbouring no plasmid (■, □), producing Dd5P4<sub>1-787</sub>–GFP (▲, △) or phosphatase-inactive Dd5P4<sub>1-787(D319A)</sub>–GFP (●, ○) were infected with DsRed-labelled wild-type *L. pneumophila*. At the time indicated, LCVs harbouring DsRed-labelled *L. pneumophila* (red) were visualized using an affinity purified anti-SidC antibody (green), and scored for ‘tight’ (inset: left panel) or ‘spacious’ (inset: right panel) vacuoles (bar, 2 μm). The data represent the average and standard deviation of two independent experiments.

C. Wild-type *D. discoideum* (filled symbols) or ΔDd5P4 (open symbols) harbouring no plasmid (■, □), or producing OCRL1<sub>1-893</sub> (◆, ◇) were infected with DsRed-labelled wild-type *L. pneumophila* and treated as in (B). The data represent the average and standard deviation of two independent experiments.

expressing functional Dd5P4–GFP but not by the catalytically inactive mutant protein Dd5P4<sub>1-787(D319A)</sub>–GFP (Fig. 3B). The defect in LCV transition was also partially complemented by ectopical expression of the human enzyme OCRL1<sub>1-893</sub> in ΔDd5P4 (Fig. 3C), indicating that OCRL1 at least partially substitutes for Dd5P4 with regard to LCV dynamics. The functional redundancy among the two enzymes is reflected by an identical domain organization and an overall identity of 33% (Fig. 1).

Dd5P4 binds to LCVs via an N-terminal domain and is catalytically active on LCVs

The effects of Dd5P4 on intracellular replication of *L. pneumophila* and on the maturation and dynamics of LCVs suggest that Dd5P4 might directly bind to LCVs. To test this hypothesis, we produced GFP fusion proteins of Dd5P4 and fragments thereof in wild-type and ΔDd5P4 *D. discoideum* (Fig. 4A). Flow cytometry analysis revealed that the different fusion proteins or GFP alone were produced by more than 95% of the amoebae (Fig. S1, data not shown). The GFP fusion proteins were distributed throughout the host cell cytoplasm in intact wild-type *D. discoideum* (Fig. S2) as well as in ΔDd5P4 amoebae (data not shown) infected with *L. pneumophila*. However, upon homogenization of infected *D. discoideum*, the wild-type fusion protein Dd5P4<sub>1-787</sub>–GFP and

Fig. 4. Dd5P4 binds to LCVs via an N-terminal domain and is catalytically active on LCVs.

A. Dd5P4– and OCRL1–GFP fusion constructs that were ectopically expressed in *D. discoideum* and tested for localization on LCVs.

B. Confocal laser scanning micrographs of paraformaldehyde-fixed lysates of wild-type *D. discoideum* producing the Dd5P4–GFP fusion constructs indicated (green) and infected at an moi of 100 for 1 h with DsRed-labelled wild-type *L. pneumophila* (red). LCVs in lysates of infected amoebae were visualized using an affinity purified anti-SidC antibody (blue).

C. Quantification of Dd5P4<sub>1-787</sub>–GFP on SidC-positive LCVs containing DsRed-labelled wild-type *L. pneumophila* in wild-type *D. discoideum* or ΔDd5P4 at 1 h post infection.

D. Dot plot of SidC fluorescence intensity (median and variance) on LCVs (*n* > 70) in calnexin–GFP-labelled wild-type *D. discoideum* or ΔDd5P4. The amoebae were infected with DsRed-labelled wild-type *L. pneumophila* and immunolabelled for SidC, which binds to PtdIns(4)P on LCVs. Similar results were obtained in three independent experiments.

A

| LVA | IP5P | Rho GAP | GFP |
| --- | --- | --- | --- |
|  |  | 787 | Dd5P4<sub>1-787</sub>-GFP (117.9 kDa) |
|  | D319A |  | Dd5P4<sub>1-787(D319A)</sub>-GFP (117.9 kDa) |
|  |  | 465 | Dd5P4<sub>1-465</sub>-GFP (80.8 kDa) |
|  |  | 466 | Dd5P4<sub>466-787</sub>-GFP (65.2 kDa) |
|  |  | 133 | Dd5P4<sub>133-465</sub>-GFP (65.8 kDa) |
|  |  | 132 | Dd5P4<sub>1-132</sub>-GFP (43.1 kDa) |
|  |  | 133 | Dd5P4<sub>133-787</sub>-GFP (102.9 kDa) |
|  |  | 893 | OCRL1<sub>1-893</sub>-GFP (131.3 kDa) |
|  |  | 236 | OCRL1<sub>1-236</sub>-GFP (55.3 kDa) |
|  |  |  | GFP (28.1 kDa) |

B

| SidC | Lpn | GFP | Merge | SidC | Lpn | GFP | Merge |
| --- | --- | --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |

C

| Dd5P4<sub>1-787</sub>-GFP positive LCVs (%) |
| --- |
| n = 313 |
| n = 218 |
| WT |
| ΔDd5P4 |
| Dd5P4<sub>1-787</sub>-GFP |
| Dd5P4<sub>1-787</sub>-GFP |

D

| SidC fluorescence intensity (INT*mm²) |
| --- |
| WT |
| ΔDd5P4 |

the catalytically inactive mutant protein Dd5P4${}_{1-787(D319A)}$—GFP were found to localize to LCVs (Fig. 4B), indicating that Dd5P4 indeed accumulates on LCVs, and the catalytic activity of the phosphatase is not required for this localization. Furthermore, all GFP fusion proteins containing the N-terminal 132 amino acids of Dd5P4 localized to LCVs, while fusion proteins lacking the N-terminus or GFP alone did not bind to LCVs. These results identify the 132 amino acid N-terminal fragment of Dd5P4 as an LCV-binding domain. This Dd5P4 fragment has not been previously implicated in membrane targeting, and thus, we term the domain ‘LVA’ (*Legionella* vacuole association). Identical results were obtained for the recruitment of Dd5P4 or fragments thereof to LCVs in lysates of the *D. discoideum ΔDd5P4* mutant strain (Fig. S3). In wild-type or ΔDd5P4 mutant *D. discoideum* approximately 80% of LCVs stained positive for ectopically produced Dd5P4${}_{1-787}$—GFP 1 h post infection (Fig. 4C). Furthermore, in all our experiments aimed at the quantification of Dd5P4 or fragments thereof on LCVs, we did not observe obvious differences in the amount of the proteins on either tight or spacious vacuoles (data not shown).

To test whether Dd5P4 not only binds to LCVs but is catalytically active on this subcellular compartment, we quantified on the vacuoles Ptdlns(4)P, putatively produced from Ptdlns(4,5)$P_{2}$ by the 5-phosphatase activity of Dd5P4. To this end, we determined by immunofluorescence microscopy the amount of the Ptdlns(4)P-binding Icm/Dot substrate SidC (Weber et al., 2006) on LCVs in wild-type or ΔDd5P4 *D. discoideum*. LCVs in wild-type *D. discoideum* (i.e. in presence of Dd5P4) accumulated 1.4 times more SidC ($P < 0.001$; Mann–Whitney $U$-test), compared with LCVs in the ΔDd5P4 mutant strain (Fig. 4D). This result is in agreement with the notion that Dd5P4 is catalytically active on LCVs and thus increases the local concentration of Ptdlns(4)P by using Ptdlns(4,5)$P_{2}$ as a substrate.

OCRL1 binds to LCVs via an N-terminal domain in *D. discoideum* and decorates LCVs in macrophages

The defective transition from tight to spacious LCVs observed in *D. discoideum ΔDd5P4* was partially complemented by ectopic expression of the human enzyme OCRL1 (Fig. 3C), suggesting that OCRL1 might localize to LCVs similar to Dd5P4. To address this question, we ectopically expressed full-length OCRL1${}_{1-893}$—GFP as well as OCRL1${}_{1-236}$—GFP in wild-type and ΔDd5P4 *D. discoideum*. Compared with the full-length fusion protein OCRL1${}_{1-893}$—GFP, the fragment OCRL1${}_{1-236}$—GFP was produced by more amoebae (approximately 76%) and at higher levels, yet both mammalian fusion proteins were less efficiently generated than the Dd5P4 fusion proteins (Fig. S1). In intact *D. discoideum* infected with *L. pneumophila*, ectopically expressed full-length OCRL1${}_{1-893}$—GFP yielded a punctate staining, while the N-terminal OCRL1${}_{1-236}$—GFP fragment localized throughout the host cell cytoplasm (Fig. 5A). However, upon homogenization of infected amoebae, full-length OCRL1—GFP, and even more pronounced OCRL1${}_{1-236}$—GFP, was found to accumulate on LCVs in wild-type *D. discoideum* (Fig. 5B), as well as in the ΔDd5P4 mutant strain (data not shown). These results indicate that, similar to Dd5P4, also OCRL1 localizes to LCVs via an N-terminal LVA domain.

In RAW264.7 macrophages infected with *L. pneumophila* the localization of endogenous OCRL1 was analysed by immunofluorescence microscopy. Similar to Dd5P4 in *D. discoideum*, immunolabelling of OCRL1 in intact macrophages yielded a punctate staining (data not shown), and upon homogenization of the infected macrophages, OCRL1 was identified on LCVs containing single or several bacteria (Fig. 5C). In summary, Dd5P4 and OCRL1 are recruited via their N-termini to LCVs in *D. discoideum* or macrophages respectively.

**Accumulation of Dd5P4${}_{1-132}$ on LCVs harbouring Icm/Dot-proficient *L. pneumophila***

Upon fixation of infected *D. discoideum* with paraformaldehyde, the recruitment of Dd5P4 to LCVs was not visible in intact cells (Fig. S2). Thus, we tried methanol fixation to visualize the association of Dd5P4 to LCVs in intact amoebae (Fig. 6A and B). Under these conditions, an accumulation of Dd5P4${}_{1-132}$—GFP on LCVs was clearly visible in intact infected cells, while longer fusion constructs were distributed throughout the cytoplasm similar to paraformaldehyde-fixed amoebae (Fig. S2, data not shown). To analyse the subcellular distribution of Dd5P4${}_{1-132}$—GFP in *D. discoideum* infected with *L. pneumophila* in more detail, we used the ER protein disulfide isomerase (PDI) and the endosome/lysosome protein common antigen 1 (CA1) as markers. One hour post infection, 62% or 50% LCVs harbouring wild-type *L. pneumophila* stained positive for Dd5P4${}_{1-132}$—GFP or double-positive for Dd5P4${}_{1-132}$—GFP and PDI respectively (Fig. 6A). Furthermore, 65% or 28% of LCVs harbouring wild-type *L. pneumophila* stained positive for Dd5P4${}_{1-132}$—GFP or double positive for Dd5P4${}_{1-132}$—GFP and CA1 respectively (Fig. 6B). In contrast, Dd5P4${}_{1-132}$—GFP was not detectable on LCVs harbouring ΔicmT mutant *L. pneumophila*, 13% of which stained positive for PDI (Fig. 6A) and 56% were positive for CA1 (Fig. 6B).

An analysis of the recruitment kinetics revealed that the percentage of Dd5P4${}_{1-132}$-positive LCVs harbouring wild-type *L. pneumophila* continuously increased over 2 h, at which time point close to 100% of the vacuoles stained positive for Dd5P4${}_{1-132}$—GFP (Fig. 6C). At 1 h post infection, approximately 80% of the LCVs stained positive for

© 2008 Blackwell Publishing Ltd, *Cellular Microbiology*, **11**, 442–460

Fig. 5. OCRL1 binds to LCVs via an N-terminal domain in *D. discoideum* and decorates LCVs in macrophages.

A and B. Confocal laser scanning micrographs of paraformaldehyde-fixed (A) wild-type *D. discoideum* expressing the full-length OCRL1<sub>1-893</sub>-GFP or N-terminal OCRL1<sub>1-236</sub>-GFP fusion construct (green), infected with DsRed-labelled *L. pneumophila* (red), or (B) lysates thereof. The LCVs were stained with an affinity-purified anti-SidC antibody (blue).

C. Paraformaldehyde-fixed lysates of murine RAW 264.7 macrophages, infected with DsRed-labelled *L. pneumophila* (red) were stained for LCVs and OCRL1 using antibodies against SidC (blue) and OCRL1 (green) respectively.

Dd5P4<sub>1-132</sub>-GFP, similar to the recruitment kinetics of full-length Dd5P4<sub>1-787</sub>-GFP (Fig. 4C). Thus, the recruitment kinetics of the N-terminal Dd5P4<sub>1-132</sub> fragment and full-length Dd5P4<sub>1-787</sub> are very similar. In contrast, LCVs harbouring ΔicmT mutant bacteria never acquired detectable amounts of Dd5P4<sub>1-132</sub>-GFP. In summary, these results indicate that Dd5P4 is selectively recruited via its N-terminal domain to LCVs harbouring wild-type but not icm/dot mutant *L. pneumophila*.

The N-terminal domains of Dd5P4 or OCRL1 have not been previously implicated in membrane targeting of these IP5Ps. To test whether Dd5P4<sub>1-132</sub> localizes to membranes by binding to phosphoinositides or other lipids, we performed a protein-lipid overlay assay using purified GST–Dd5P4<sub>1-132</sub> (41.3 kDa) and as a positive control GST–

SidC<sub>609-917</sub> (62.5 kDa). While GST–Dd5P4<sub>1-132</sub> did not bind to any of the lipids spotted onto nitrocellulose membranes (Fig. 6D), GST–SidC<sub>609-917</sub> specifically bound to PtdIns(4)P as described (Weber et al., 2006; Ragaz et al., 2008). Thus, Dd5P4<sub>1-132</sub> might bind to LCVs via a bacterial or a host cell protein receptor rather than via a lipid.

*L. pneumophila* LpnE binds to OCRL1<sub>1-236</sub>, LCV membranes and PtdIns(3)P

As Dd5P4<sub>1-132</sub> apparently does not bind to a lipid, we tested whether Dd5P4 or OCRL1 bind to a protein receptor on LCVs. To assay whether the N-terminal domain of OCRL1 binds to a *L. pneumophila* protein, OCRL1<sub>1-236</sub> was covalently coupled to Affigel-10 beads, incubated

© 2008 Blackwell Publishing Ltd, *Cellular Microbiology*, **11**, 442–460

450 S. S. Weber, C. Ragaz and H. Hilbi

Fig. 6. Accumulation of Dd5P4${}_{1-132}$ on LCVs harbouring Icm/Dot-proficient *L. pneumophila*. A and B. Confocal laser scanning micrographs of methanol-fixed wild-type *D. discoideum* producing Dd5P4${}_{1-132}$–GFP (green), infected with DsRed-labelled wild-type or $\Delta icmT$ *L. pneumophila* (red) for 1 h, and stained with an antibody (blue) against (A) PDI (protein disulfide isomerase) ($n = 132$ LCVs, wt; $92$ LCVs, $\Delta icmT$) or (B) CA1 (common antigen 1) ($n = 456$ LCVs, wt; $213$ LCVs, $\Delta icmT$). The quantification of colocalization is illustrated in a pie graph using the same colour code as for the confocal micrographs. Bar, $5 \mu$m.

C. Kinetics of Dd5P4${}_{1-132}$–GFP acquisition on LCVs containing DsRed-labelled wild-type (■) or $\Delta icmT$ (▲) *L. pneumophila* in wild-type *D. discoideum*.

D. Binding of affinity-purified GST fusion proteins of Dd5P4${}_{1-132}$ or SidC${}_{609-917}$ to different lipids (100 pmol) immobilized on nitrocellulose membranes was analysed by a protein-lipid overlay assay using an anti-GST antibody. LPA, lysophosphatidic acid; LPC, lysophosphocholine; PI, phosphatidylinositol; PE, phosphatidylethanolamine; PC, phosphatidylcholine; S1P, sphingosine-1-phosphate; PA, phosphatidic acid; PS, phosphatidylserine.

with *L. pneumophila* lysate, and bound proteins were identified by LC-ESI-MS/MS (Fig. 7A). Using this protocol, a prominent protein band specifically retained by OCRL1${}_{1-236}$ coupled to beads was identified as the 41.4 kDa *L. pneumophila* protein LpnE (Newton et al., 2006; 2007). The most prominent protein band eluting from OCRL1${}_{1-236}$ beads incubated with either *L. pneumophila* lysate or binding buffer corresponded to the 27.3 kDa OCRL1${}_{1-236}$ fragment.

To confirm the interaction between OCRL1${}_{1-236}$ and LpnE, purified GST–OCRL1${}_{1-236}$ was incubated with *L. pneumophila* lysate, precipitated with glutathione beads, and LpnE was identified by Western blot using an antibody against LpnE (Fig. 7B). In the reciprocal approach, purified GST–LpnE was incubated with lysate from *D. discoideum* producing OCRL1${}_{1-236}$–GFP, precipitated with glutathione beads, and binding of OCRL1${}_{1-236}$–GFP was visualized by Western blot using an antibody against GFP (Fig. 7C). These immunoprecipitation experiments confirmed the interaction between OCRL1${}_{1-236}$ and LpnE. In a similar experimental setup the interaction of LpnE with Dd5P4${}_{1-132}$ was analysed. However, neither did LpnE associate with GST–Dd5P4${}_{1-132}$, nor Dd5P4${}_{1-132}$–GFP with GST–LpnE. Perhaps, the additional N-terminal amino acids 1–122 of OCRL1 contribute to or stabilize the interaction with LpnE.

© 2008 Blackwell Publishing Ltd, Cellular Microbiology, 11, 442–460

OCRL1 restricts vacuolar growth 451

A
Lpn lysate Buffer
beads control OCRL1<sub>1-236</sub> control OCRL1<sub>1-236</sub>

B
Anti LpnE
beads GST GST-
lysate input - + - +
(41.4 kDa)

C
Anti GFP
beads GST GST-
lysate - - + +
input GST- GST-
LpnE LpnE
(55.3 kDa)

D
Brightfield Calnexin Lpn LpnE Merge
wt 1um
ΔicmT
ΔlpnE

Fig. 7. *L. pneumophila* LpnE binds to OCRL1<sub>1-236</sub> and LCV membranes.
A. For pulldown experiments, OCRL1<sub>1-236</sub> covalently linked to Affigel-10 beads or control beads were incubated with *L. pneumophila* lysate or buffer, and proteins bound to the washed beads were separated by SDS-PAGE, stained with Coomassie brilliant blue and identified by LC-ESI-MS/MS. A 41.4 kDa band was identified as *L. pneumophila* LpnE.
B. Purified GST–OCRL1<sub>1-236</sub> was incubated with *L. pneumophila* lysate or (C) purified GST–LpnE was incubated with lysate from *D. discoideum* producing OCRL1<sub>1-236</sub>–GFP, respectively, precipitated with glutathione beads, and the proteins bound to washed beads were visualized by Western blot using antibodies against LpnE or GFP respectively (composed images from one experiment; representative of three independent assays).
D. Confocal laser scanning micrographs of lysed macrophages infected with GFP-labelled wild-type, Δ*icmT* or Δ*lpnE* *L. pneumophila*. Calnexin and LpnE were visualized using antibodies.

If the interaction between OCRL1 and LpnE is relevant in vivo, LpnE should localize to LCV membranes. Indeed, LpnE was identified by immunofluorescence microscopy on LCVs in RAW264.7 macrophages infected with wild-type but not *icmT* mutant *L. pneumophila* (Fig. 7D). As expected, LCVs harbouring an *L. pneumophila* Δ*lpnE* mutant strain did not stain with the antibody. These results are in agreement with a function for LpnE in binding or stabilizing OCRL1 on LCVs. We also tried to visualize LpnE on LCVs harbouring wild-type *L. pneumophila* in *D. discoideum*. However, the affinity-purified anti-LpnE antibody used was not suitable in this case, as Western blot experiments revealed cross-reactions with *D. discoideum* proteins (data not shown).

Next, we analysed whether the *L. pneumophila* Δ*lpnE* mutant strain shows a phenotype regarding the recruitment of OCRL1/Dd5P4 in *D. discoideum*. At 1 h post infection LCVs harbouring the *L. pneumophila* Δ*lpnE* mutant strain stained positive for OCRL1<sub>1-236</sub>–GFP (Fig. 8A) or Dd5P4<sub>1-132</sub>–GFP (Fig. 8B) to a similar extent as LCVs harbouring wild-type bacteria. Furthermore, the acquisition kinetics of Dd5P4<sub>1-132</sub>–GFP were very similar regarding LCVs harbouring Δ*lpnE* or wild-type *L. pneumophila* (Fig. 8C). While approximately 90% of the LCVs containing Δ*lpnE* or wild-type *L. pneumophila* stained positive for Dd5P4<sub>1-132</sub>–GFP 2 h post infection, none of the LCVs containing a Δ*icmT* strain acquired the fusion protein during a 4 h infection period. We also tested

© 2008 Blackwell Publishing Ltd, Cellular Microbiology, 11, 442–460

452 S. S. Weber, C. Ragaz and H. Hilbi

**A**
OCRL1<sub>1-236</sub>-GFP  | Lpn  | Merge
--- | --- | ---
wt |  | 
2μm |  | 
ΔlpnE |  | 

**B**
Dd5P4<sub>1-132</sub>-GFP  | Lpn  | Merge
--- | --- | ---
wt |  | 
5μm |  | 
ΔlpnE |  | 

**C**
![Graph](https://i.imgur.com/placeholder.png)

**D**
LpnE
| LPA | S1P |
| --- | --- |
| LPC | PI(3,4)P₂ |
| PI | PI(3,5)P₂ |
| PI(3)P | PI(4,5)P₂ |
| PI(4)P | PI(3,5)P₂ |
| PI(5)P | PA |
| PE | PS |
| PC | blank |

SidC
| LPA | S1P |
| --- | --- |
| LPC | PI(3,4)P₂ |
| PI | PI(3,5)P₂ |
| PI(3)P | PI(4,5)P₂ |
| PI(4)P | PI(3,4,5)P₃ |
| PI(5)P | PA |
| PE | PS |
| PC | blank |

---

Fig. 8. *L. pneumophila* LpnE is dispensable for OCRL1 and Dd5P4 recruitment and binds to PtdIns(3)P.

A and B. Confocal laser scanning micrographs of (A) paraformaldehyde-fixed lysates of wild-type *D. discoideum* producing OCRL1<sub>1-236</sub>-GFP or (B) methanol-fixed wild-type *D. discoideum* producing Dd5P4<sub>1-132</sub>-GFP, infected with wild-type or ΔlpnE *L. pneumophila* (moi 50, 1 h) and stained with an antibody (red).

C. Kinetics of Dd5P4<sub>1-132</sub>-GFP acquisition on LCVs containing DsRed-labelled wild-type (■), ΔlpnE (●) or ΔicmT (▲) *L. pneumophila* in wild-type *D. discoideum* during a 4 h infection. The data is representative of 3 independent experiments.

D. Binding of affinity-purified GST fusion proteins of *L. pneumophila* LpnE or SidC to different lipids (100 pmol; left panels) or twofold serial dilutions of phosphoinositides (100–1.56 pmol; right panels) immobilized on nitrocellulose membranes was analysed by a protein-lipid overlay assay using an anti-GST antibody. LPA, lysophosphatidic acid; LPC, lysophosphocholine; PI, phosphatidylinositol; PE, phosphatidylethanolamine; PC, phosphatidylcholine; S1P, sphingosine-1-phosphate; PA, phosphatidic acid; PS, phosphatidylserine. The experiment was reproduced twice with similar results.

whether LpnE modulates the catalytic activity of Dd5P4. As shown above (Fig. 4D), more PtdIns(4)P-binding SidC accumulated on LCVs in presence of Dd5P4, indicating that the phosphatase is catalytically active. However, in absence or presence of *lpnE*, the amount of SidC on LCVs was similar, indicating that LpnE does not affect the activity of Dd5P4 (Fig. S4). Taken together, these results suggest that LpnE is apparently dispensable for

© 2008 Blackwell Publishing Ltd, Cellular Microbiology, 11, 442–460
the recruitment of OCRL1/Dd5P4 to LCVs, and the *L. pneumophila* protein does not modulate the catalytic activity of the phosphatase.

Finally, as LpnE localizes to the LCV membrane (Fig. 7D), we addressed the question whether the protein might bind to a lipid. Using a protein-lipid overlay assay, LpnE was found to specifically bind to Ptdlns(3)P but not to other phosphoinositides or lipids *in vitro* (Fig. 8D). This result suggests that LpnE anchors to LCVs via Ptdlns(3)P. SidC, used as a control, specifically bound to Ptdlns(4)P and less efficiently to Ptdlns(3)P, as observed previously (Weber et al., 2006).

### Discussion

An analysis of the role of IP5Ps during phagocytosis and intracellular replication of *L. pneumophila* revealed that *D. discoideum* Dd5P4 impairs intracellular growth of *L. pneumophila* and the interaction of LCVs with the ER (Figs 2 and 3). Furthermore, Dd5P4 and OCRL1 were found to be recruited to LCVs in *D. discoideum* and macrophages (Figs 4 and 5). Dd5P4 is the closest homologue of mammalian OCRL1, and both IP5Ps are classified as GAP domain-containing inositol phosphatases (Loovers et al., 2007) (Fig. 1). Dd5P4 and OCRL1 share a number of properties, including a catalytic activity as type II IP5Ps (Zhang et al., 1995; Loovers et al., 2003). Upon heterologous expression of OCRL1 in *D. discoideum ΔDd5P4*, growth and developmental defects of the mutant strain are fully complemented, and a phagocytosis defect is partially complemented (Loovers et al., 2003; 2007). Similarly, the impaired transition from tight to spacious LCVs observed in the ΔDd5P4 mutant strain is partially complemented upon heterologous expression of OCRL1 (Fig. 3C). Together, these results suggest that the orthologues Dd5P4 and OCRL1 share similar functions, and thus, *D. discoideum* Dd5P4 is a valid model to study OCRL1. Compared with Dd5P4–GFP lower amounts of OCRL1–GFP bound to LCVs (Figs 4B and 5B), corresponding to the finding that OCRL1–GFP was produced by fewer *D. discoideum* cells and in lower amounts (Fig. S1). Therefore, a less efficient production and LCV targeting of OCRL1 in *D. discoideum* likely accounts for the only partial complementation of phenotypes observed for strain ΔDd5P4 infected with *L. pneumophila*.

OCRL1 shares the domain architecture and 45% identity with INPP5B, another mammalian type II IP5P. While OCRL1 and INPP5B are structurally similar, some evidence supports a role for OCRL1 rather than INPP5B during intracellular replication of *L. pneumophila*: (i) ectopically expressed OCRL1 localizes to LCVs in *D. discoideum* (Fig. 5B); (ii) the anti-OCRL1 antibody used in our study was raised against the N-terminal amino acid residues 1–237 (Choudhury et al., 2005), which are only 18% identical with INPP5B, against which no specific antibody could be raised (Williams et al., 2007); and (iii) OCRL1 (but not INPP5B) binds clathrin (Ungewickell et al., 2004; Choudhury et al., 2005; Williams et al., 2007). Clathrin is a pivotal component of early endosome to TGN vesicle trafficking and was recently identified in the proteome of purified LCVs (Urwyler et al., 2008). INPP5P, on the other hand, is implicated in the retrograde pathway from the ERGIC to ER (Williams et al., 2007), which does not involve clathrin.

Dd5P4/OCRL1 directly affects the formation and maturation of LCVs, as in absence of Dd5P4 LCVs interact with the ER more rapidly, and the transition from tight to spacious vacuoles is impaired (Fig. 3). The significance of this morphological transition for *L. pneumophila* infection is not clear. The transition is inhibited due to the absence of different *D. discoideum* or *L. pneumophila* factors, yet the efficiency of intracellular bacterial replication is either increased (in *D. discoideum ΔDd5P4* or ΔPI3K1/2) (Fig. 2; Weber et al., 2006), decreased (in *D. discoideum ΔRtoA*) (Li et al., 2005) or not affected (by *L. pneumophila ΔsidC-sdcA*) (Ragaz et al., 2008). The spacious phagosomes likely do not represent compartments harbouring ‘sick’ bacteria that no longer form a replication-permissive vacuole, as the recruitment to LCVs of calnexin is dependent on the paralogous lcm/Dot substrates SidC and SdcA, and this process precedes the morphological transition (Ragaz et al., 2008). In absence of these effectors, the number of spacious vacuoles decreased by more than 50%, yet *L. pneumophila ΔsidC-sdcA* mutant bacteria replicate at wild-type rate (and not more efficiently).

OCRL1 regulates retrograde vesicle trafficking between endosomes and the TGN (Ungewickell et al., 2004; Choudhury et al., 2005; Williams et al., 2007), and therefore, a functional retrograde pathway might restrict intracellular replication of *L. pneumophila*. The bacteria also replicate more efficiently in *D. discoideum* in absence of PI3Ks (Weber et al., 2006), which have also been implicated in endosomal vesicle trafficking, including the retrograde endosome to TGN transport (Lowe, 2005; Bonifacino and Rojas, 2006). However, while in absence of Dd5P4 lower amounts of the Ptdlns(4)P-binding lcm/ Dot substrate SidC are bound to LCVs (Fig. 4D), higher amounts of SidC are bound to LCVs in absence of PI3Ks (Weber et al., 2006). Thus, binding of SidC to LCVs, and as a corollary the amount of Ptdlns(4)P on LCVs is apparently not the rate-limiting factor for intracellular growth of *L. pneumophila*.

Accumulating evidence implicates trafficking between the TGN and endosomes in the formation of LCVs. Recently, the secretory and endosomal small GTPases Rab1, Rab7, Rab8 and Rab14 were identified in the proteome of purified LCVs, and these GTPases were

© 2008 Blackwell Publishing Ltd, *Cellular Microbiology*, **11**, 442–460
recruited to vacuoles harbouring wild-type but not $\Delta icmT$ mutant $L$. *pneumophila* (Urwyler et al., 2008). Rab8 localizes to endosomes and regulates the late secretory pathway through endosomes by promoting fusion of TGN exit transport vesicles with endosomes (Henry and Sheff, 2008). Thus, Rab8 and OCRL1 share a subcellular localization and might accumulate on LCVs by direct fusion of the LCVs with endosomes at early retrograde transport exit sites and/or late secretary entry sites.

OCRL1 and Dd5P4 anchor to LCVs harbouring Icm/Dot-proficient bacteria via an N-terminal domain, which we term ‘LVA’ (*Legionella* vacuole association) to reflect its novel role in membrane binding of the IP5Ps. Except the N-terminal 21 amino acid residues of Dd5P4, the LVA domains of Dd5P4 and OCRL1 are fairly similar and share 21% identity (Fig. 1). However, the N-terminus of OCRL1 includes another 122 amino acids, the function of which is unknown. These additional amino acids might contribute to the interaction of OCRL1<sub>1–236</sub> with the $L$. *pneumophila* effector LpnE (Fig. 7), in agreement with the finding that the shorter N-terminus of Dd5P4 apparently does not interact with LpnE (data not shown). LpnE localizes to LCVs harbouring wild-type but not $\Delta icmT$ mutant $L$. *pneumophila* (Figs 7D and 8C), suggesting that the effector is an Icm/Dot substrate. Moreover, LpnE selectively binds to PtdIns(3)$P$ (Fig. 8D) and thus is the first member of a second family of phosphoinositide-binding $L$. *pneumophila* effectors, in addition to the PtdIns(4)$P$-binding Icm/Dot substrates represented by SidC (Weber et al., 2006; Ragaz et al., 2008).

OCRL1<sub>1–236</sub> and Dd5P4<sub>1–132</sub> are recruited to LCVs harbouring an $L$. *pneumophila* strain lacking *lpnE* (Fig. 8), indicating that LpnE is not required for the recruitment of the IP5Ps. LpnE might stabilize OCRL1 on LCVs, or other $L$. *pneumophila* Sel1-like repeat proteins might substitute for the protein. Many effector proteins of $L$. *pneumophila* form families of (functionally redundant) paralogues (Luo and Isberg, 2004). Indeed, the four $L$. *pneumophila* genomes sequenced to date encode the same 6 or 7 Sel1-like repeat proteins (Fig. S5), including LpnE (Newton et al., 2006; 2007), EnhC (Cirillo et al., 2000; Liu et al., 2008) and LidL (Conover et al., 2003). The four Sel1-like repeat proteins most closely related to LpnE are EnhC, LidL, Lpg1062 and Lpg1356, which share 28–37% identity with LpnE. Sel1-like repeats are implicated in protein–protein interactions, and the proteins may serve as adaptors for the assembly of macromolecular complexes (Mittl and Schneider-Brachert, 2007).

Binding of OCRL1/Dd5P4 on LCVs by LpnE or other Sel1-like repeat proteins might promote the restriction of intracellular growth of $L$. *pneumophila*. Thus, the interaction between LpnE and OCRL1 possibly represents a bacterial mechanism to downregulate intracellular replication, thereby sustaining a protective niche, which would be lost upon rapid killing of the host cell. Alternatively, the accumulation of OCRL1/Dd5P4 on LCVs harbouring Icm/Dot-proficient but not *icm/dot* mutant $L$. *pneumophila* (Fig. 6) suggests that OCRL1 and/or the retrograde transport pathway play a role in growth restriction of vacuolar pathogens by the host cell. Thereby, this pathway might contribute to containment and clearance of pathogenic intracellular microorganisms.

OCRL1 is recruited and activated by Rab GTPases of the Golgi apparatus (Rab1, Rab6) or early endosomes (Rab5) (Hyvola et al., 2006) (Fig. 1). The small GTPases Arf1 and Rab1 localize to LCVs and promote intracellular replication of $L$. *pneumophila* (Kagan and Roy, 2002; Nagai et al., 2002; Derre and Isberg, 2004; Kagan et al., 2004), and therefore, these GTPases might recruit and activate OCRL1 (Fig. 4D). The ASH and Rho GAP domains of OCRL1 have been proposed to orient the catalytic IP5P domain to the membrane (Erdmann et al., 2007). However, as the LVA domain of OCRL1/Dd5P4 is sufficient to bind the phosphatases to LCVs, this domain also contributes to or is even sufficient to properly position the catalytic IP5P domain on LCVs. In agreement with this hypothesis, an N-terminal Dd5P4 fragment including only the LVA and IP5P domains is recruited to LCVs (Fig. 4A and B) and partially complements the intracellular growth phenotype of $L$. *pneumophila* in the $\Delta Dd5P4$ mutant strain (Fig. 2F).

*Legionella pneumophila* is taken up by macrophages and amoebae by ‘efficient phagocytosis’, an Icm/Dot T4SS-dependent process resembling macropinocytosis (Hilbi et al., 2001; Watarai et al., 2001; Weber et al., 2006). While PI3Ks are dispensable for efficient phagocytosis of $L$. *pneumophila* (Weber et al., 2006), the absence of Dd5P1 and Dd5P4 reduced the number of infected *D*. *discoideum* by approximately 40% (Fig. 2A). These findings are in agreement with a role of Dd5P4 in promoting phagocytosis by removing PtdIns(4,5)$P_{2}$ at the plasma membrane (Loovers et al., 2007). Along the same line, at early time points post infection (< 30 min) already 20–45% of the LCVs stained positive for the N-terminal fragment Dd5P4<sub>1–132</sub> (Fig. 6C), again indicating a role for Dd5P4 in early steps of the uptake process. OCRL1 was found to translocate to membrane ruffles (resembling macropinocytosis) upon stimulation of quiescent cells with growth factors and activation of the small GTPase Rac (Faucherre et al., 2003; 2005). Furthermore, OCRL1 binds to the GTPases Rac, Cdc42 and Rab5 present in ruffles (Hyvola et al., 2006; Erdmann et al., 2007). Therefore, $L$. *pneumophila* possibly facilitates its uptake by recruiting OCRL1/Dd5P4 to the entry sites. An analogous mechanism has been reported for the enteropathogenic bacteria *Shigella flexneri* (Niebuhr et al., 2002) and *Salmonella enterica* (Terebiznik et al., 2002), which facilitate their entry by translocating into host cells the bacterial inositol

© 2008 Blackwell Publishing Ltd, *Cellular Microbiology*, **11**, 442–460

polyphosphate 4-phosphatase (IP4P) IpgD or SigD/SopB respectively. Hence, *L. pneumophila* might exploit the host cell IP5Ps OCRL1/Dd5P4 not only to modulate intracellular replication but also to promote its uptake.

### Experimental procedures

#### Bacteria, cells and growth conditions

The *L. pneumophila* wild-type strain JR32 (Sadosky *et al.*, 1993) and the isogenic Δ*icmT* mutant strain GS3011 (Segal *et al.*, 1998) were grown for 3 days on CYE agar plates. Liquid cultures were inoculated in AYE medium at an OD<sub>600</sub> of 0.1 and grown for 21 h at 37°C (Weber *et al.*, 2006). To maintain plasmids, chloramphenicol was added at 5 μg ml<sup>-1</sup>. As ‘input’ controls for phagocytosis experiments, 20 μl of a 10<sup>5</sup> ml<sup>-1</sup> bacterial solution was plated and counted after 3 days.

The *D. discoideum* wild-type strain Ax3 and mutant strains lacking individual IP5Ps (Δ*Dd5P1–4*) were obtained from P.J.M. van Haastert (University of Groningen) (Loovers *et al.*, 2003). The amoebae were grown axenically at 23°C in 75 cm<sup>2</sup> tissue culture flasks in HL5 liquid medium (10 g of glucose, 5 g of thiotone E peptone, 2.5 mM Na<sub>2</sub>HPO<sub>4</sub>, 2.5 mM KH<sub>2</sub>PO<sub>4</sub> in 1 l of H<sub>2</sub>O, pH 6.5), supplemented with 20 μg ml<sup>-1</sup> G418 or 10 μg ml<sup>-1</sup> blasticidin S when necessary, split twice a week and fed with fresh HL5 medium 24 h before use. Transfection of *D. discoideum* with expression plasmids for GFP fusion constructs was performed as described (Faix *et al.*, 2004). Briefly, exponentially growing cells were washed once in SorC (2 mM Na<sub>2</sub>HPO<sub>4</sub>, 15 mM KH<sub>2</sub>PO<sub>4</sub>, 50 μM CaCl<sub>2</sub>, pH 6.0) and once in electroporation buffer (EB; 50 mM sucrose, 10 mM KH<sub>2</sub>PO<sub>4</sub>, pH 6.1). The cell pellet was re-suspended in EB and the suspension gently mixed with ~25 μg plasmid DNA and transferred to a 4 mm electroporation cuvette. Electroporation conditions were 1 ms, at 1 kV and 10 μF applied twice 5 s apart using the Gene Pulser Xcell apparatus (Bio-Rad). After electroporation, the cells were transferred to a six-well plate and incubated for 15 min at 23°C. The suspension was adjusted to 2 mM CaCl<sub>2</sub> and 2 mM MgCl<sub>2</sub> and incubated for further 15 min at 23°C. Finally, 3 ml HL5 medium was added, and the cells were allowed to recover for 24 h. Then, the medium was replaced by HL5 containing 20 μg ml<sup>-1</sup> of G418 to select for transfectants. All transfected *D. discoideum* strains were analysed for the presence of the GFP fusion proteins by flow cytometry and Western blot using an anti-GFP antibody respectively.

Murine RAW264.7 macrophages were cultivated in RPMI1640 medium supplemented with 10% FCS and 2 mM L-glutamine at 37°C in a humidified atmosphere of 5% CO<sub>2</sub>.

#### Construction of plasmids and a chromosomal *lpnE* deletion mutant

All plasmids are listed in Table 1. pSW101 was constructed by ligating a dimer of oSW3 and oSW4 into the Sacl and Xbal sites of pVatM–GFP. To allow the expression of GFP fusion constructs in *D. discoideum*, *egfp* was amplified by PCR using pEGFP-N1 as a template and the primers listed in Table S1. The PCR fragment was cut with Xhol and Xbal and ligated into plasmid pSW101 yielding pSW102. Translational *gfp* fusions of *Dd5P4*, *Ocrl1* and several fragments thereof were constructed by PCR amplification of the gene fragments using pDd5P4–GFP, pDd5P4<sub>D319A</sub>–GFP or pSW105 as templates. The PCR fragments were cut with BamHI and Xhol and ligated into plasmid pSW102 yielding pSW103, pSW105, pSW106, pSW107, pSW108, pSW110, pSW111, pSW112, pSW113 and pSW114. All constructs were sequenced.

A translational *gst* fusion of *Dd5p4<sub>1–132</sub>* was constructed by PCR amplification of the gene fragment from pSW105. The PCR fragment was cut with BamHI/Sall and ligated into pGEX-4T-1 yielding plasmid pSW009. Translational *gst* fusions of *Ocrl1<sub>1–236</sub>* and *lpnE* were constructed by releasing the genes from pSW113 with BamHI and Xhol or from pSW014 with BamHI and Sall respectively. The *Ocrl1<sub>1–236</sub>* fragment was ligated into the BamHI and Sall sites of pGEX-6P-1 and pGEX-4T-1, yielding pSW010 and pSW011 respectively. The *lpnE* fragment was ligated into the BamHI and Sall site of pGEX-4T-1 yielding pSW013.

The chromosomal *lpnE* (*lpg2222*) deletion mutant strain SW01 (JR32 *lpnE::Km<sup>R</sup>*) was generated by allelic exchange using counter-selection on sucrose as described (Water *et al.*, 1994; Tiaden *et al.*, 2007). To this end, the downstream and upstream flanking sequences *lpnE‘down’* and *lpnE‘up’* were amplified by PCR using the primer pairs oSW28/29 and oSW30/31 respectively, and digested with BamHI. pUC4K was cut with BamHI to release the Km resistance cassette (*Km<sup>R</sup>*). The flanking sequences *lpnE‘down’* and *lpnE‘up’* as well as the *Km<sup>R</sup>* were ligated in a four-way ligation reaction into pGEM-T-Easy, yielding pSW015. This construct was analysed by restriction digestion and sequenced to test for correct inserts. The resulting *lpnE‘down’-Km<sup>R</sup>-lpnE‘up’* fragment was released from pSW015 by NotI and cloned into the pLAW344 suicide plasmid, yielding the allelic exchange vector pSW016. *L. pneumophila* JR32 was transformed with pSW016 by electroporation, Km<sup>R</sup> colonies were selected by growth at 30°C and counter-selected with 2% sucrose and Km. The chromosomal deletion of *lpnE* was confirmed by PCR and Western blot using an anti-LpnE antibody (Newton *et al.*, 2006; 2007). The complementation plasmid pSW014 was constructed by PCR amplification of *lpnE* from *L. pneumophila* genomic DNA. This fragment was subsequently cloned into the BamHI and Sall sites of pMMB207C-RBS-M45.

#### Analysis of phagocytosis and intracellular growth of *L. pneumophila*

Phagocytosis of *L. pneumophila* by *D. discoideum*, intracellular replication of wild-type and killing of Δ*icmT L. pneumophila* was analysed as described using GFP-labelled bacteria and flow cytometry gating on the amoebae (Weber *et al.*, 2006). Release of *L. pneumophila* from *D. discoideum* due to intracellular replication was also quantified by determining colony-forming units (cfu) in the supernatant as described (Solomon *et al.*, 2000; Otto *et al.*, 2004). Briefly, exponentially growing amoebae were washed with SorC and re-suspended in MB medium (7 g of yeast extract, 14 g of thiotone E peptone, 20 mM MES in 1 l H<sub>2</sub>O, pH 6.9). Approximately 10<sup>5</sup> *D. discoideum* per well were seeded onto a 96-well plate, let adhere for 1–2 h, and were infected at an multiplicity of infection (moi) of 1 with *L. pneumophila* grown on CYE plates for 3–4 days and re-suspended in MB medium. The infection was synchronized by centrifugation, and the infected amoebae were incubated at 25°C. At the time points

Table 1. Plasmids used in this study.

| Plasmid           | Characteristics                                                                 | PCR oligonucleotides (template)                     | Reference                |
|-------------------|---------------------------------------------------------------------------------|----------------------------------------------------|--------------------------|
| pCR2              | gst-sidC in pGEX-4T-1                                                          |                                                  | Weber et al. (2006)      |
| pCR44             | gst-sidC<sub>609–917</sub> in pGEX-4T-1                                         |                                                  | Ragaz et al. (2008)      |
| pCalnexin-GFP     | CalnexinA-RSSSKLK-gfp(S65T); P<sub>act15</sub>                                  |                                                  | Müller-Taubenberger et al. (2001) |
| pDd5P4-GFP        | Dd5P4-gfp in pDM6                                                              |                                                  | Loovers et al. (2007)    |
| pDd5P4<sub>D319A</sub>-GFP | Dd5p4<sub>D319A</sub>-gfp in pDM6                                        |                                                  | Loovers et al. (2007)    |
| pEGFP-N1          | Expression of EGFP in mammalian cells                                          |                                                  | Amersham                 |
| pGEX-4T-1         | Expression of N-terminal gst fusions; P<sub>tac</sub>                           |                                                  | Amersham                 |
| pGEX-6P-1         | Expression of N-terminal gst fusions; P<sub>tac</sub>, PreScission protease site |                                                  | Amersham                 |
| pUC4K             | pUC4K oriR (pBR322), MCS::Km                                                   |                                                  | Amersham                 |
| pGEM-T-Easy       | Cloning of PCR products                                                        |                                                  | Promega                  |
| pLAW344           | oriT (RK2), oriR (ColE1), sacB                                                 |                                                  | Wiater et al. (1994)     |
| pMMB207-Km14-gfp<sub>c</sub> | Gfp in pMMB207-Km14, ΔlacI<sup>q</sup>                                |                                                  | Mampel et al. (2006)     |
| pMMB207C-RBS-M45  | M45-MCS in pMMB207; P<sub>tac</sub>                                            |                                                  | Weber et al. (2006)      |
| pOCRL1            | OcrI1 in pMB74                                                                 |                                                  | Loovers et al. (2007)    |
| pSW001            | DsredExpress in pMMB207C, ΔlacI<sup>q</sup>                                     |                                                  | Mampel et al. (2006)     |
| pSW009            | gst-Dd5p4<sub>1–132</sub> in pGEX-4T-1                                         | oSW5/24 (pSW105)                                 | This study               |
| pSW010            | gst-OCRL1<sub>1–236</sub> in pGEX-6T-1                                         |                                                  | This study               |
| pSW011            | gst-OCRL1<sub>1–236</sub> in pGEX-4T-1                                         |                                                  | This study               |
| pSW013            | gst-lpnE in pGEX-4T-1                                                         |                                                  | This study               |
| pSW014            | M45-LpnE in pMMB207C; P<sub>tac</sub>                                          | oSW26/27 (L. pneumophila genomic DNA)             | This study               |
| pSW015            | Genomic region of lpnE in pGEM-T Easy                                          |                                                  | This study               |
| pSW016            | Genomic region of lpnE in pLAW344                                              |                                                  | This study               |
| pSW101            | MCS-m45 in pDXA                                                               | oSW3/4                                           | This study               |
| pSW102            | MCS-gfp in pDXA                                                               | oSW1/2 (pEGFP-N1)                                | This study               |
| pSW103            | OcrI1<sub>1–893</sub>-gfp in pDXA                                              | oSW7/8 (pOCRL1)                                  | This study               |
| pSW105            | Dd5p4<sub>1–787</sub>-gfp in pDXA                                              | oSW5/12 (pDd5P4-GFP)                            | This study               |
| pSW106            | Dd5p4<sub>1–787(D319A)</sub>-gfp in pDXA                                      | oSW5/12 (pDd5P4<sub>D319A</sub>-GFP)              | This study               |
| pSW107            | Dd5p4<sub>1–465</sub>-gfp in pDXA                                              | oSW5/17 (pDd5P4-GFP)                            | This study               |
| pSW108            | Dd5p4<sub>466–787</sub>-gfp in pDXA                                            | oSW18/12 (pDd5P4-GFP)                           | This study               |
| pSW111            | Dd5p4<sub>1–132</sub>-gfp in pDXA                                              | oSW5/19 (pSW105)                                | This study               |
| pSW112            | Dd5p4<sub>133–465</sub>-gfp in pDXA                                            | oSW20/17 (pSW105)                               | This study               |
| pSW113            | OcrI1<sub>1–236</sub>-gfp in pDXA                                              | oSW7/23 (pSW103)                                | This study               |
| pSW114            | Dd5p4<sub>133–787</sub>-gfp in pDXA                                            | oSW20/12 (pSW105)                               | This study               |
| pVatM-GFP         | VatM-gfp in pDXA                                                              |                                                  | Lu and Clarke (2005)     |

indicated, the number of bacteria released into the supernatant was quantified by plating aliquots of appropriate dilutions on CYE plates. Due to the relatively high moi of 1 used, L. pneumophila grew with moderate efficiency in wild-type D. discoideum, as observed in previous studies (Weber et al., 2006; Ragaz et al., 2008).

Intracellular bacterial growth before host cell lysis was quantified by counting cfu after selectively lysing infected D. discoideum with saponin ('single round replication') as described (Weber et al., 2006). Briefly, at the time points indicated, the MB medium was replaced by 100 μl of 0.8% saponin and incubated for 20 min. The cells were lysed by vigorously pipetting, and aliquots were plated in appropriate dilutions.

**Intracellular trafficking of L. pneumophila, and localization of Dd5P4, OCRL1 or LpnE on LCVs**

Dictyostelium discoideum or macrophages were processed for fluorescence microscopy as described (Albers et al., 2005; Weber et al., 2006). At the time points indicated, the infected phagocytes were washed 3 times with cold SorC buffer (D. discoideum) or PBS (macrophages) and fixed with 4% paraformaldehyde in PBS at 4°C or with methanol at –20°C for 30 min. The fixed cells were washed, permeabilized (0.1% Triton X-100, 10 min) and blocked with 2% normal human AB serum in SorC or PBS for 30 min. Alternatively, the infected phagocytes were washed with cold SorC or PBS, re-suspended in homogenization buffer (20 mM Hepes, 250 mM sucrose, 0.5 mM EGTA, pH 7.2) (Derre and Isberg, 2004) and lysed by 11 passages through a ball homogenizer (Isobiotech) using an exclusion size of 8 μm. The homogenate was centrifuged onto coverslips coated with poly-L-lysine, immobilized by 4% paraformaldehyde for 30 min at 4°C and blocked with 2% normal human AB serum in SorC or PBS for 30 min.

The coverslips with fixed intact cells or lysates were incubated for 1 h at room temperature on parafilm with 30 μl of primary antibody diluted in blocking buffer (affinity purified rabbit anti-SidC (Weber et al., 2006), 1:1000; affinity purified rabbit anti-LpnE (Newton et al., 2006; 2007), 1:30; affinity purified sheep anti-OCRL1<sub>1–237</sub> (Choudhury et al., 2005), 1:400; goat anti-calnexin (Santa Cruz, 1:100); anti-D. discoideum common antigen 1 (CA1), 1:1000; anti-D. discoideum protein disulfide isomerase (PDI) (Monnat et al., 1997), 1:10 000; anti-L. pneumophila (Santa Cruz, 1:100) and washed 3 times. Secondary antibodies (FITC-conjugated goat anti-rabbit IgG [H+L], Cy5-conjugated goat anti-rabbit IgG [H+L]; rhodamine-conjugated bovine anti-rabbit IgG; FITC-conjugated donkey anti-sheep IgG [H+L], Cy5-conjugated donkey anti-goat IgG [H+L], Cy5-conjugated goat anti-mouse IgG

© 2008 Blackwell Publishing Ltd, Cellular Microbiology, 11, 442–460
[H+L], or Cy3-conjugated goat anti-mouse IgG [H+L]; Jackson ImmunoResearch) were diluted in blocking buffer and incubated for 1 h at room temperature. Finally, the coverslips were washed and mounted using Vectashield (Vector Laboratories) supplemented with 1 μg ml⁻¹ DAPI to stain DNA.

The amount of SidC on LCVs harbouring DsRed-labelled *L. pneumophila* in wild-type or ΔDd5P4 *D. discoideum* was quantified by immunofluorescence using affinity purified anti-SidC and Cy5-conjugated secondary antibodies (Weber et al., 2006). The fluorescence intensity of an area identical for all samples and covering a single LCV was quantified using the ‘Quantity One’ software (Bio-Rad) after background correction (averaged intensity of 3 areas within the infected amoeba). ‘Tight’ and ‘spacious’ vacuoles were scored exactly as described (Weber et al., 2006; Ragaz et al., 2008). Briefly, ‘spacious’ vacuoles were defined as vacuoles where the SidC-labelled membrane surrounding *L. pneumophila* was clearly detached, leaving a space between the membrane and the bacterium. All other LCVs observed were scored as ‘tight’.

The samples were viewed with an inverted confocal microscope (Axiovert 200 M; Zeiss), equipped with a 100× oil phase contrast objective (Plan Neofluar; Zeiss), an ‘Ultraview’ confocal head (Perkin Elmer) and a krypton/argon laser (643-RYB-A01; Melles Griot). Data processing was performed with the ‘Volocity’ 2.6.1 software (Improvision).

### Binding of GST–Dd5P4 or GST–LpnE to phosphoinositides and other lipids in vitro

The GST–Dd5P4<sub>1–132</sub> or GST–LpnE fusion proteins were produced in *E. coli*/BL21(DE3) (Novagen) induced at a cell density of OD<sub>600</sub> 0.6 with 0.5 mM isopropyl-1-thio-β-D-galactopyranoside (IPTG) for 3 h at 30°C in LB medium. The soluble fusion proteins were purified from lysates prepared by French press using glutathione sepharose beads in a batch procedure according to the manufacturer’s recommendations (Amersham), and purity of the protein preparations was analysed by SDS-PAGE.

Direct binding of GST–Dd5P4<sub>1–132</sub> or GST–LpnE to phosphoinositides and other lipids was tested in a protein-lipid overlay assay as described (Weber et al., 2006; Dowler et al., 2002), using commercially available PIP-strips and PIP-arrays (Echelon) and the PtdIns(4)P-binding probes GST–SidC or GST–SidC<sub>609–917</sub> as a control.

### Pulldown experiments

The N-terminal fragment OCRL1<sub>1–236</sub> was purified from *E. coli* BL21(DE3) and covalently linked to Affigel-10 beads as described (Ragaz et al., 2008), incubated with 1 ml *L. pneumophila* lysate prepared by sonication, and washed 4 times with binding buffer (20 mM Hepes, 100 mM NaCl, 5 mM MgCl<sub>2</sub>, 1 mM DTT, pH 7.5). As a comparison, the *L. pneumophila* protein SidM covalently linked to Affigel-10 beads was used. Proteins bound to the washed beads were separated by 12% SDS-PAGE, stained with Coomassie brilliant blue and identified by LC-ESI-MS/MS. To confirm binding of *L. pneumophila* LpnE to OCRL1<sub>1–236</sub>, GST–OCRL1<sub>1–236</sub> or GST–LpnE was produced and purified as described above, incubated with lysates prepared by sonication of *L. pneumophila* or *D. discoideum* producing OCRL1<sub>1–236</sub>-GFP, respectively, and precipitated with glutathione beads. The proteins bound to beads washed with binding buffer were separated by 12% SDS-PAGE and visualized by Western blot using antibodies against LpnE (1:500; (Newton et al., 2006; 2007) or GFP (1:2500; Roche Diagnostics) respectively.

## Acknowledgements

We would like to thank P.J.M. van Haastert (University of Groningen) for the *D. discoideum* ΔDd5P1–4 mutant strains, as well as for Dd5P4 and OCRL1 expression vectors, M. Lowe (University of Manchester) for an antibody against human OCRL1 and OCRL1 expression vectors, E. Hartland (University of Melbourne) for an anti-LpnE antibody, and P. Cosson (University of Geneva) for antibodies against *D. discoideum* PDI and CA1. W.-D. Hardt and A. Oxenius (ETH Zürich) kindly provided access to confocal microscope and flow cytometer facilities respectively. This work was supported by grants from the Swiss National Science Foundation (631-065952, PP00A-112592), the ETH Zürich (TH 17/02-3, TH-26 06-2), the OPO Foundation and the Velux Foundation.

## References

Albers, U., Reus, K., Shuman, H.A., and Hilbi, H. (2005) The amoebae plate test implicates a parologue of *lpxB* in the interaction of *Legionella pneumophila* with *Acanthamoeba castellanii*. Microbiology **151**: 167–182.

Attree, O., Olivos, I.M., Okabe, I., Bailey, L.C., Nelson, D.L., Lewis, R.A., *et al.* (1992) The Lowe’s oculocerebrorenal syndrome gene encodes a protein highly homologous to inositol polyphosphate-5-phosphatase. Nature **358**: 239–242.

Bonifacino, J.S., and Rojas, R. (2006) Retrograde transport from endosomes to the trans-Golgi network. Nat Rev Mol Cell Biol **7**: 568–579.

Brüggemann, H., Cazalet, C., and Buchrieser, C. (2006) Adaptation of *Legionella pneumophila* to the host environment: role of protein secretion, effectors and eukaryotic-like proteins. Curr Opin Microbiol **9**: 86–94.

Choudhury, R., Diao, A., Zhang, F., Eisenberg, E., Saint-Pol, A., Williams, C., *et al.* (2005) Lowe syndrome protein OCRL1 interacts with clathrin and regulates protein trafficking between endosomes and the trans-Golgi network. Mol Biol Cell **16**: 3467–3479.

Cirillo, S.L., Lum, J., and Cirillo, J.D. (2000) Identification of novel loci involved in entry by *Legionella pneumophila*. Microbiology **146**: 1345–1359.

Conover, G.M., Derre, I., Vogel, J.P., and Isberg, R.R. (2003) The *Legionella pneumophila* LidA protein: a translocated substrate of the Dot/Icm system associated with maintenance of bacterial integrity. Mol Microbiol **48**: 305–321.

De Matteis, M.A., and Godi, A. (2004) PI-loting membrane traffic. Nat Cell Biol **6**: 487–492.

Derre, I., and Isberg, R.R. (2004) *Legionella pneumophila* replication vacuole formation involves rapid recruitment of proteins of the early secretory system. Infect Immun **72**: 3048–3053.

Di Paolo, G., and De Camilli, P. (2006) Phosphoinositides in cell regulation and membrane dynamics. Nature **443**: 651–657.

© 2008 Blackwell Publishing Ltd, Cellular Microbiology, **11**, 442–460
Dorer, M.S., Kirton, D., Bader, J.S., and Isberg, R.R. (2006) RNA interference analysis of *Legionella* in *Drosophila* cells: exploitation of early secretory apparatus dynamics. *PLoS Pathog* 2: e34.

Dowler, S., Kular, G., and Alessi, D.R. (2002) Protein lipid overlay assay. *Sci STKE* 2002: PL6.

Dressman, M.A., Olivos-Glander, I.M., Nussbaum, R.L., and Suchy, S.F. (2000) OCRL1, a Ptdlns(4,5)P(2) 5-phosphatase, is localized to the trans-Golgi network of fibroblasts and epithelial cells. *J Histochem Cytochem* 48: 179–190.

Erdmann, K.S., Mao, Y., McCrea, H.J., Zoncu, R., Lee, S., Paradise, S., *et al.* (2007) A role of the Lowe syndrome protein OCRL in early steps of the endocytic pathway. *Dev Cell* 13: 377–390.

Faix, J., Kreppel, L., Shaulsky, G., Schleicher, M., and Kimmel, A.R. (2004) A rapid and efficient method to generate multiple gene disruptions in *Dictyostelium discoideum* using a single selectable marker and the Cre-loxP system. *Nucleic Acids Res* 32: e143.

Faucherre, A., Desbois, P., Satre, V., Lunardi, J., Dorseuil, O., and Gacon, G. (2003) Lowe syndrome protein OCRL1 interacts with Rac GTPase in the trans-Golgi network. *Hum Mol Genet* 12: 2449–2456.

Faucherre, A., Desbois, P., Nagano, F., Satre, V., Lunardi, J., Gacon, G., and Dorseuil, O. (2005) Lowe syndrome protein OCRL1 is translocated to membrane ruffles upon Rac GTPase activation: a new perspective on Lowe syndrome pathophysiology. *Hum Mol Genet* 14: 1441–1448.

Fields, B.S., Benson, R.F., and Besser, R.E. (2002) *Legionella* and Legionnaires’ disease: 25 years of investigation. *Clin Microbiol Rev* 15: 506–526.

Gillooly, D.J., Simonsen, A., and Stenmark, H. (2001) Phosphoinositides and phagocytosis. *J Cell Biol* 155: 15–18.

Greub, G., and Raoult, D. (2004) Microorganisms resistant to free-living amoebae. *Clin Microbiol Rev* 17: 413–433.

Henry, L., and Sheff, D.R. (2008) Rab8 regulates basolateral secretion, but not recycling, traffic at the recycling endosome. *Mol Biol Cell* 19: 2059–2068.

Hilbi, H. (2006) Modulation of phosphoinositide metabolism by pathogenic bacteria. *Cell Microbiol* 8: 1697–1706.

Hilbi, H., Segal, G., and Shuman, H.A. (2001) Icm/Dot-dependent upregulation of phagocytosis by *Legionella pneumophila*. *Mol Microbiol* 42: 603–617.

Hilbi, H., Weber, S.S., Ragaz, C., Nyfeler, Y., and Urwyler, S. (2007) Environmental predators as models for bacterial pathogenesis. *Environ Microbiol* 9: 563–575.

Horwitz, M.A. (1983) The Legionnaires’ disease bacterium (*Legionella pneumophila*) inhibits lysosome-phagosome fusion in human monocytes. *J Exp Med* 158: 1319–1331.

Hyvola, N., Diao, A., McKenzie, E., Skippen, A., Cockcroft, S., and Lowe, M. (2006) Membrane targeting and activation of the Lowe syndrome protein OCRL1 by rab GTPases. *EMBO J* 25: 3750–3761.

Kagan, J.C., and Roy, C.R. (2002) *Legionella* phagosomes intercept vesicular traffic from endoplasmic reticulum exit sites. *Nat Cell Biol* 4: 945–954.

Kagan, J.C., Stein, M.P., Pypaert, M., and Roy, C.R. (2004) *Legionella* subvert the functions of Rab1 and Sec22b to create a replicative organelle. *J Exp Med* 199: 1201–1211.


Khelef, N., Shuman, H.A., and Maxfield, F.R. (2001) Phagocytosis of wild-type *Legionella pneumophila* occurs through a wortmannin-insensitive pathway. *Infect Immun* 69: 5157–5161.

Li, Z., Solomon, J.M., and Isberg, R.R. (2005) *Dictyostelium discoideum* strains lacking the RtoA protein are defective for maturation of the *Legionella pneumophila* replication vacuole. *Cell Microbiol* 7: 431–442.

Lichter-Konecki, U., Farber, L.W., Cronin, J.S., Suchy, S.F., and Nussbaum, R.L. (2006) The effect of missense mutations in the RhoGAP-homology domain on OCRL1 function. *Mol Genet Metab* 89: 121–128.

Liu, M., Conover, G.M., and Isberg, R.R. (2008) *Legionella pneumophila* EnhC is required for efficient replication in tumor necrosis factor alpha-stimulated macrophages. *Cell Microbiol* 10: 1906–1923.

Loovers, H.M., Veenstra, K., Snippe, H., Pesesse, X., Erneux, C., and van Haastert, P.J.M. (2003) A diverse family of inositol 5-phosphatases playing a role in growth and development in *Dictyostelium discoideum*. *J Biol Chem* 278: 5652–5658.

Loovers, H.M., Kortholt, A., de Groote, H., Whitty, L., Nussbaum, R.L., and van Haastert, P.J. (2007) Regulation of phagocytosis in *Dictyostelium* by the inositol 5-phosphatase OCRL homolog Dd5P4. *Traffic* 8: 618–628.

Lowe, M. (2005) Structure and function of the Lowe syndrome protein OCRL1. *Traffic* 6: 711–719.

Lu, H., and Clarke, M. (2005) Dynamic properties of *Legionella*-containing phagosomes in *Dictyostelium amoebae*. *Cell Microbiol* 7: 995–1007.

Luo, Z.-Q., and Isberg, R.R. (2004) Multiple substrates of the *Legionella pneumophila* Dot/Icm system identified by interbacterial protein transfer. *Proc Natl Acad Sci USA* 101: 841–846.

Mampel, J., Spirig, T., Weber, S.S., Haagensen, J.A., Molin, S., and Hilbi, H. (2006) Planktonic replication is essential for biofilm formation by *Legionella pneumophila* in a complex medium under static and dynamic flow conditions. *Appl Environ Microbiol* 72: 2885–2895.

Mittl, P.R., and Schneider-Brachert, W. (2007) Sel1-like repeat proteins in signal transduction. *Cell Signal* 19: 20–31.

Molmeret, M., Horn, M., Wagner, M., Santic, M., and Abu Kwaik, Y. (2005) Amoebae as training grounds for intracellular bacterial pathogens. *Appl Environ Microbiol* 71: 20–28.

Monnat, J., Hacker, U., Geissler, H., Rauchenberger, R., Neuhaus, E.M., Maniak, M., and Soldati, T. (1997) *Dictyostelium discoideum* protein disulfide isomerase, an endoplasmic reticulum resident enzyme lacking a KDEL-type retrieval signal. *FEBS Lett* 418: 357–362.

Müller-Taubenberger, A., Lupas, A.N., Li, H., Ecke, M., Simmeth, E., and Gerisch, G. (2001) Calreticulin and calnexin in the endoplasmic reticulum are important for phagocytosis. *EMBO J* 20: 6772–6782.

Nagai, H., Kagan, J.C., Zhu, X., Kahn, R.A., and Roy, C.R. (2002) A bacterial guanine nucleotide exchange factor activates ARF on *Legionella* phagosomes. *Science* 295: 679–682.

Nash, T.W., Libby, D.M., and Horwitz, M.A. (1984) Interaction between the Legionnaires’ disease bacterium (*Legionella*

pneumophila) and human alveolar macrophages. Influence of antibody, lymphokines, and hydrocortisone. *J Clin Invest* 74: 771–782.

Newton, H.J., Sansom, F.M., Bennett-Wood, V., and Hartland, E.L. (2006) Identification of *Legionella pneumophila*-specific genes by genomic subtractive hybridization with *Legionella micdadei* and identification of *lpnE*, a gene required for efficient host cell entry. *Infect Immun* 74: 1683–1691.

Newton, H.J., Sansom, F.M., Dao, J., McAlister, A.D., Sloan, J., Cianciotto, N.P., and Hartland, E.L. (2007) Sel1 repeat protein LpnE is a *Legionella pneumophila* virulence determinant that influences vacuolar trafficking. *Infect Immun* 75: 5575–5585.

Niebuhr, K., Giuriato, S., Pedron, T., Philpott, D.J., Gaits, F., Sable, J., et al. (2002) Conversion of PtdIns(4,5)P₂ into PtdIns(5)P by the *S.* flexneri effector IpgD reorganizes host cell morphology. *EMBO J* 21: 5069–5078.

Ninio, S., and Roy, C.R. (2007) Effector proteins translocated by *Legionella pneumophila*: strength in numbers. *Trends Microbiol* 15: 372–380.

Otto, G.P., Wu, M.Y., Clarke, M., Lu, H., Anderson, O.R., Hilbi, H., et al. (2004) Macroautophagy is dispensable for intracellular replication of *Legionella pneumophila* in *Dictyostelium discoideum*. *Mol Microbiol* 51: 63–72.

Pizarro-Cerda, J., and Cossart, P. (2004) Subversion of phosphoinositide metabolism by intracellular bacterial pathogens. *Nat Cell Biol* 6: 1026–1033.

Ragaz, C., Pietsch, H., Urwyler, S., Tiaden, A., Weber, S.S., and Hilbi, H. (2008) The *Legionella pneumophila* phosphatidylinositol-4 phosphate-binding type IV substrate SidC recruits endoplasmic reticulum vesicles to a replication-permissive vacuole. *Cell Microbiol* 10: 2416–2433.

Robinson, C.G., and Roy, C.R. (2006) Attachment and fusion of endoplasmic reticulum with vacuoles containing *Legionella pneumophila*. *Cell Microbiol* 8: 793–805.

Sadosky, A.B., Wiater, L.A., and Shuman, H.A. (1993) Identification of *Legionella pneumophila* genes required for growth within and killing of human macrophages. *Infect Immun* 61: 5361–5373.

Segal, G., and Shuman, H.A. (1999) *Legionella pneumophila* utilizes the same genes to multiply within *Acanthamoeba castellanii* and human macrophages. *Infect Immun* 67: 2117–2124.

Segal, G., Purcell, M., and Shuman, H.A. (1998) Host cell killing and bacterial conjugation require overlapping sets of genes within a 22-kb region of the *Legionella pneumophila* genome. *Proc Natl Acad Sci USA* 95: 1669–1674.

Segal, G., Feldman, M., and Zusman, T. (2005) The Icm/Dot type IV secretion systems of *Legionella pneumophila* and *Coxiella burnetii*. *FEMS Microbiol Rev* 29: 65–81.

Solomon, J.M., Rupper, A., Cardelli, J.A., and Isberg, R.R. (2000) Intracellular growth of *Legionella pneumophila* in *Dictyostelium discoideum*, a system for genetic analysis of host–pathogen interactions. *Infect Immun* 68: 2939–2947.

Spirig, T., Tiaden, A., Kiefer, P., Buchrieser, C., Vorholt, J.A., and Hilbi, H. (2008) The *Legionella* autoinducer synthase LqsA produces an α-hydroxyketone signaling molecule. *J Biol Chem* 283: 18113–18123.


Steinert, M., and Heuner, K. (2005) *Dictyostelium* as host model for pathogenesis. *Cell Microbiol* 7: 307–314.

Swanson, M.S., and Isberg, R.R. (1995) Association of *Legionella pneumophila* with the macrophage endoplasmic reticulum. *Infect Immun* 63: 3609–3620.

Terebiznik, M.R., Vieira, O.V., Marcus, S.L., Slade, A., Yip, C.M., Trimble, W.S., et al. (2002) Elimination of host cell PtdIns(4,5)P(2) by bacterial SigD promotes membrane fission during invasion by *Salmonella*. *Nat Cell Biol* 4: 766–773.

Tiaden, A., Spirig, T., Weber, S.S., Brüggemann, H., Bosshard, R., Buchrieser, C., and Hilbi, H. (2007) The *Legionella pneumophila* response regulator LqsR promotes host cell interactions as an element of the virulence regulatory network controlled by RpoS and LetA. *Cell Microbiol* 9: 2903–2920.

Tiaden, A., Spirig, T., Carranza, P., Brüggemann, H., Riedel, K., Eberl, L., et al. (2008) Synergistic contribution of the *Legionella pneumophila lqs* genes to pathogen–host interactions. *J Bacteriol* 190: 7532–7547.

Tilney, L.G., Harb, O.S., Connelly, P.S., Robinson, C.G., and Roy, C.R. (2001) How the parasitic bacterium *Legionella pneumophila* modifies its phagosome and transforms it into rough ER: implications for conversion of plasma membrane to the ER membrane. *J Cell Sci* 114: 4637–4650.

Ungewickell, A., Ward, M.E., Ungewickell, E., and Majerus, P.W. (2004) The inositol polyphosphate 5-phosphatase OCRL associates with endosomes that are partially coated with clathrin. *Proc Natl Acad Sci USA* 101: 13501–13506.

Urwyler, S., Nyfeler, Y., Ragaz, C., Lee, H., Mueller, L.N., Aebersold, R., and Hilbi, H. (2008) Proteome analysis of *Legionella* vacuoles purified by magnetic immuno-separation reveals secretory and endosomal GTPases. *Traffic* (Epub ahead of print). doi: 10.1111/j.1600-0854.2008.00851.x

Vogel, J.P., Andrews, H.L., Wong, S.K., and Isberg, R.R. (1998) Conjugative transfer by the virulence system of *Legionella pneumophila*. *Science* 279: 873–876.

Watarai, M., Derre, I., Kirby, J., Groyney, J.D., Dietrich, W.F., and Isberg, R.R. (2001) *Legionella pneumophila* is internalized by a macropinocytotic uptake pathway controlled by the Dot/Icm system and the mouse *Lgn1* locus. *J Exp Med* 194: 1081–1096.

Weber, S.S., Ragaz, C., Reus, K., Nyfeler, Y., and Hilbi, H. (2006) *Legionella pneumophila* exploits PI(4)P to anchor secreted effector proteins to the replicative vacuole. *PLoS Pathog* 2: e46.

Wiater, L.A., Sadosky, A.B., and Shuman, H.A. (1994) Mutagenesis of *Legionella pneumophila* using Tn903dll/lacZ: identification of a growth-phase-regulated pigment gene. *Mol Microbiol* 11: 641–653.

Williams, C., Choudhury, R., McKenzie, E., and Lowe, M. (2007) Targeting of the type II inositol polyphosphate 5-phosphatase INPP5B to the early secretory pathway. *J Cell Sci* 120: 3941–3951.

Zhang, X., Jefferson, A.B., Auethavekiat, V., and Majerus, P.W. (1995) The protein deficient in Lowe syndrome is a phosphatidylinositol-4,5-bisphosphate 5-phosphatase. *Proc Natl Acad Sci USA* 92: 4853–4856.

Supporting information

Additional Supporting Information may be found in the online version of this article:

Fig. S1. Ectopic production of GFP and fusion proteins of Dd5P4, OCRL1 or fragments in wild-type *D. discoideum*. Flow cytometry data of *D. discoideum* wild-type strain Ax3 producing (A) no ectopic protein, (B) GFP, (C) Dd5P4<sub>1–787</sub>–GFP, (D) Dd5P4<sub>1–132</sub>–GFP, (E) OCRL1<sub>1–893</sub>–GFP, or (F) OCRL1<sub>1–236</sub>–GFP. While all amoebae (> 99%) produced GFP or fusion proteins of Dd5P4 or Dd5P4<sub>1–132</sub> at high levels, only a portion of the amoebae (42–76%) produced GFP fusion proteins with mammalian OCRL1 or OCRL1<sub>1–236</sub> at lower levels.

Fig. S2. Distribution of ectopically produced GFP fusion proteins of Dd5P4 and fragments in *D. discoideum* wild-type. Confocal laser scanning micrographs of paraformaldehyde-fixed wild-type *D. discoideum* producing the Dd5P4–GFP fusion proteins indicated (green), infected with DsRed-labelled *L. pneumophila* (red) at an moi of 50 (1 h) and stained with an anti-SidC antibody (blue) for LCVs. DNA was visualized with DAPI. The Dd5P4–GFP fusion proteins localized throughout the host cell cytoplasm.

Fig. S3. Binding of GFP fusion proteins of Dd5P4 and fragments to LCVs in infected *D. discoideum ΔDd5P4*. Confocal laser scanning micrographs of *D. discoideum ΔDd5P4* expressing the Dd5P4–GFP fusion constructs indicated (green), and infected with DsRed-labelled wild-type *L. pneumophila* (red) at an moi of 100 (1 h). LCVs in lysates were visualized using an affinity purified anti-SidC antibody (blue). Only Dd5P4–GFP fusion constructs containing the N-terminal 132 amino acids of Dd5P4 localize to LCVs.

Fig. S4. The catalytic activity of Dd5P4 on LCVs is not affected by LpnE. Dot plot of SidC fluorescence intensity (median and variance) on LCVs in calnexin–GFP-labelled wild-type or ΔDd5P4 *D. discoideum*. The amoebae were infected with DsRed-labelled JR32 wild-type or ΔlpnE mutant *L. pneumophila* and immunolabelled for SidC, which binds to PtdIns(4)P on LCVs. In presence of the inositol polyphosphate 5-phosphatase Dd5P4, the median SidC fluorescence intensity was higher, indicating that the enzyme was catalytically active on the LCV membrane producing PtdIns(4)P. In absence of *lpnE*, Dd5P4 was still catalytically active, indicating that LpnE does not affect the activity of the phosphatase.

Fig. S5. Overview and alignment of *L. pneumophila* proteins containing Sel1-like repeats. (A) Overview of *L. pneumophila* Sel1-like repeat proteins. The genes encoding Sel1-like repeat proteins are present in the genomes of all *L. pneumophila* strains sequenced to date (Philadelphia, Paris, Lens, Corby) with the exception of *lpg1062*. LpnE (Lpg2222, Lpp2174, Lpl2147, Lpc1689), EnhC (Lpg2639, Lpp2692, Lpl2564, Lpc0501), LidL (Lpg1172, Lpp1174, Lpl1180, Lpc0638), Lpg1062 (Lpl1059, Lpc2212), Lpg1356 (Lpp1310, Lpl1307, Lpc0770), Lpg2485 (Lpp2549, Lpl2405, Lpc1993), Lpg0896 (Lpp0957, Lpl0927, Lpc2397). Lpc0165 is unique to *L. pneumophila* strain Corby. The following domains are highlighted: Sel1 (grey diamonds), signal peptide (black bar), TPR (dark grey diamonds), methyltransferase (grey bar), HNH nuclease (dark grey bar). (B) Alignment of the closest LpnE homologues: EnhC (30.1% identity), LidL (32.5%), Lpg1062 (28.4%) and Lpg1356 (37.3%). Additional amino acids present in EnhC or LidL that do not align with the other homologues are indicated in parentheses.

Table S1. Oligonucleotides used in this study.

Please note: Wiley-Blackwell are not responsible for the content or functionality of any supporting materials supplied by the authors. Any queries (other than missing material) should be directed to the corresponding author for the article.
