
Mitsutoshi Yoneyama
Takashi Fujita

RNA recognition and signal
transduction by RIG-I-like
receptors

Authors' addresses
Mitsutoshi Yoneyama${}^{1,2,3}$, Takashi Fujita${}^{1,2}$
${}^{1}$Laboratory of Molecular Genetics, Institute for Virus
Research, Kyoto University, Kyoto, Japan.
${}^{2}$Laboratory of Molecular Cell Biology, Graduate School of
Biostudies, Kyoto University, Kyoto, Japan.
${}^{3}$PRESTO, Japan Science and Technology Agency, Saitama,
Japan.

Correspondence to:
Takashi Fujita
Laboratory of Molecular Genetics
Institute for Virus Research, Kyoto University
Shogoinkawahara-cho, Sakyo-ku
Kyoto 606-8507
Japan
Tel.: +81 75 751 4031
Fax: +81 75 751 4031
e-mail: tfujita@virus.kyoto-u.ac.jp

Summary: Viral infection is detected by cellular sensor molecules as
foreign nucleic acids and initiates innate antiviral responses, including
the activation of proinflammatory cytokines and type I interferon (IFN).
Recent identification of cytoplasmic viral sensors, such as retinoic acid-
inducible gene-I-like receptors (RLRs), highlights their significance in
the induction of antiviral innate immunity. Moreover, it is intriguing to
understand how they can discriminate endogenous RNA from foreign
viral RNA and initiate signaling cascades leading to the induction of type
I IFNs. This review focuses on the current understanding of the molecular
machinery underlying RNA recognition and subsequent signal transduc-
tion by RLRs.

Keywords: antiviral innate immunity, type I interferon, RIG-I-like receptors

Introduction
To counteract the threat of microbial invasion, higher
eukaryotes have evolved self-defense mechanisms: innate and
adaptive immunity. Initially, host sensor molecules detect
specific pathogen-associated molecular patterns (PAMPs) and
activate the innate immune system through cytokine produc-
tion including pro-inflammatory cytokines and type I interfer-
ons (IFNs). Cytokine expression provokes swift eradication of
the invading pathogens and simultaneously regulates the
adaptive immune system. Therefore, understanding the role
of the molecular mechanisms underlying pathogen detection
and subsequent intracellular signaling cascades leading to
cytokine activation is extremely important for developing
novel therapeutic strategies against microbial infections.

The well-characterized sensor molecules include Toll-like
receptors (TLRs) (1, 2). TLRs, which are transmembrane
proteins, detect PAMPs derived from fungi, bacteria, parasites,
or viruses via extracellular leucine-rich repeat (LRR)
sequences, at the plasma membrane surface or in the endo-
somal compartment, and transmit signals through the
cytoplasmic Toll-interleukin (IL)-1 receptor (TIR) domain.
Endosomal TLRs, such as TLR3, TLR7/8, and TLR9, are
known to detect microbial nucleic acids and induce type I IFNs, which are critical regulators of antiviral innate immunity. C-type lectin receptors (CLRs), which detect fungal PAMPs (3), are another type of membrane-bound receptor. At least three cytoplasmic sensor families have been characterized. Nod-like receptors (NLRs), which have a characteristic C-terminal LRR and internal nucleotide-binding domain (NBD), are known to detect cytoplasmic PAMPs, leading to the activation of cytokine expression including IL-1β (4, 5). A cytoplasmic DNA sensor has been proposed to detect viral or bacterial B-form DNA and to activate type I IFNs (6, 7). Although DNA-dependent activator of IFN-regulatory factors (DAI) was identified as a possible DNA sensor (8), DAI knockout (KO) mice showed no abnormality in DNA-induced IFN production (9), suggesting that uncharacterized DNA sensor(s) could be functionally redundant. In addition, retinoic acid-inducible gene-I (RIG-I)-like receptors (RLRs) specifically detect viral RNA in cytoplasm and induce IFNs (10). Among these sensor molecules, endosomal TLRs, DNA sensors, and RLRs are specific sensors of virus-derived PAMPs, especially viral nucleic acids, and frequently activate type I IFNs for viral clearance (2). In particular, it is important to understand how RLRs strictly discriminate non-self-RNA from self-RNA, because endogenous self-RNA is expressed abundantly in the cytoplasm of the virus-infected cells. This review focuses on RLRs and discusses how these molecules detect viral RNA, discriminate endogenous self-RNA, and initiate the antiviral innate immune system.

### Type I IFN system

The type I IFN system is a critical cytokine for antiviral innate immunity (11). IFNs were originally identified as soluble factors expressed by virus-infected cells and involved in the inhibition of viral replication (12, 13). Subsequent identification of IFN-encoding genes demonstrated that IFN consists of single IFN-β and IFN-α families (>10 genes) (14, 15). The type I IFN system is regulated by two signaling cascades: a pathogen-induced IFN-producing signal and an IFN receptor-mediated secondary signal (16, 17). The first signal is initiated by the detection of viral infection via the above-mentioned sensor molecules, which frequently activate a series of transcription factors, such as activating transcription factor (ATF)/c-jun, nuclear factor κB (NF-κB), and IFN regulatory factor (IRF). These factors bind to the promoter region of the IFN-β gene (18–21), form a transcriptional complex termed the ‘enhanceosome’, and activate robust transcription through interaction with the RNA polymerase complex (22).

However, IFN-α genes are known to be regulated by only IRF (23), indicating an essential function of IRF(s) in the viral-induced activation of IFN genes. IRF-3 and IRF-7, key regulators of IFN gene activation (24–30), are specifically phosphorylated by a viral-induced signal, then translocate to the nucleus, and bind to the promoter sequences of IFN genes. Analysis of KO mice of these two genes clearly showed their essential role in the activation of antiviral immunity (31, 32).

The expressed IFNs transmit a secondary signal through an interaction with the cognate heterodimeric IFN receptors (IFNRs), which are broadly expressed in most cells, in an autocrine or paracrine manner. Interaction between IFNs and IFNRs activates the downstream Janus kinase (Jak)-Signal transducer activator of transcription (STAT) pathway. The resulting activation of transcription factor, IFN-stimulated gene factor 3 (ISGF3), which consists of STAT1, STAT2, and IRF-9 as a DNA-binding subunit, induces the expression of hundreds of IFN-inducible genes (ISGs), including double-stranded RNA (dsRNA)-dependent protein kinase (PKR), 2′–5′-oligoadenylatesynthetase, adenosine deaminase, and Mx guanosine triphosphatase (GTPase). Although the physiological significance of most of ISGs remains unclear, ISGs collectively establish strong antiviral activity in the cells. In addition to these cell-intrinsic effects in the establishment of the antiviral state, type I IFNs, especially TLR-induced IFNs in immunospecific cells, are known to function in the regulation of subsequent innate and adaptive immunity. For example, it has been demonstrated that IFNs are involved in the maturation of dendritic cells (DCs), including the cross-presentation of viral antigens for CD8⁺ T cells and the activation of natural killer cells (33).

#### Identification of RIG-I and its function

RIG-I was identified originally as a gene induced in retinoic acid-treated acute promyelocytic leukemia cells (34). Subsequent identification of a porcine homolog of RIG-I as a porcine reproductive and respiratory syndrome virus-induced gene suggested a possible connection between RIG-I and viral infection (35). In 2004, RIG-I was isolated functionally as a cDNA clone that can activate IRF-regulated reporter gene expression in response to transfection with synthetic dsRNA, polyinosinic-polycytidylic acid (polyI:C), which is classically identified as a strong IFN inducer (36).

RIG-I belongs to the DExD/H box-containing RNA helicase family and has two characteristic caspase recruitment domains (CARDs) at its N-terminus in addition to the C-terminal helicase domain. Mutant RIG-I containing the N-terminal
CARD can act as a constitutive activator capable of inducing IFN production without viral infection. Although full-length RIG-I showed no constitutive properties, a marked augmentation of IFN-inducing signaling was observed in response to viral infection. Moreover, a CARD-deficient mutant functioned as a dominant negative inhibitor. These results suggested strongly that CARD is responsible for the downstream signaling cascade, and the C-terminal region containing the helicase domain strictly regulates RIG-I function in a virus-dependent manner. Mutagenesis of full-length RIG-I in which the adenosine triphosphate (ATP)-binding lysine residue of the helicase domain was substituted to alanine generated a dominant negative inhibitor form of RIG-I, indicating that RIG-I requires ATP to activate signaling. The C-terminal portion [aa 735–925; repressor domain (RD)] is involved in inhibiting RIG-I activation in the absence of viral stimulation (37); however, recent reports clearly showed that the C-terminal domain (CTD) (aa 792–925), overlapping with RD, is responsible for the recognition of specific viral RNA (38, 39) (see below).

Analysis of RIG-I KO mice clearly showed that RIG-I plays an essential role in virus-induced type I IFN expression not only in fibroblasts but also in conventional DCs, which is critical for the regulation of adaptive immunity (40). By contrast, plasmacytoid DCs, known as robust IFN-α-producing cells, mainly utilize TLRs for virus detection, indicating the cell-type specific usage of virus sensors in the innate immune system. Interestingly, RIG-I KO mice were embryonic lethal because of liver degeneration, suggesting a possible developmental function of RIG-I other than in viral sensing (40). However, another RIG-I KO strain made by Wang et al. (41) is viable because of differential strategies in gene targeting, and this mouse develops a colitis-like phenotype associated with the downregulation of G protein subunit Gi2α and resultant abnormal T-cell activation. Although the precise mechanism underlying these observations is not fully understood, they suggest intrinsic function(s) of RIG-I other than viral sensing. Recently, negative regulation of granulocyte differentiation by RIG-I has also been demonstrated (42).

### RLR family members MDA5 and LGP2

The mammalian genome encodes two other helicase genes closely related to RIG-I. Melanoma differentiation-associated gene 5 (MDA5) was originally isolated as a gene induced by the IFN- and protein kinase C-activating compound mezerein in a melanoma cell line (43). MDA5 shows 23% and 35% amino acid similarity to RIG-I in the N-terminal tandem CARD and helicase domains, respectively; however, the C-terminal portion of MDA5 does not show an inhibitory function similar to that of RD (37, 44). Functional analyses of MDA5 using cell culture indicated that MDA5 functions as a positive regulator, similar to RIG-I. Furthermore, an essential role of MDA5 in virus-induced innate immunity was demonstrated by analysis of KO mice (45, 46). Interestingly, comparison of RIG-I KO and MDA5 KO mice revealed that these sensors detect different types of viruses (40, 45). This differential specificity is clearly explained by preferential RNA structure recognition by RIG-I or MDA5 (see below).

Functions of MDA5 other than virus sensing have been suggested. Growth inhibition and apoptosis of several tumor cells were observed after the ectopic expression of MDA5 (47). Moreover, helicard, a mouse homolog of MDA5, was reported to be involved in apoptosis via caspase-dependent cleavage (48). Although a similar cleavage of MDA5 was observed in poliovirus-infected cells (49), the physiological significance of the specific cleavage of MDA5 still remains unclear. Furthermore, genome-wide analysis demonstrated that the MDA5 gene was a strong candidate of a gene locus associated with type I diabetes, suggesting the involvement of MDA5 in autoimmune disorders (50).

Another RLR family member is laboratory of genetics and physiology 2 (LGP2), which was identified as the gene adjacent to the STAT3/5 locus (51). LGP2 protein shows 41% and 31% amino acid similarity to helicase domain of RIG-I and MDA5, respectively; however, LGP2 is completely devoid of an N-terminal CARD. In vitro experiments showed that LGP2 plays an inhibitory role in RIG-I/MDA5-mediated signaling (44, 52, 53). However, recent gene KO analysis indicated that LGP2 plays a positive role in encephalomyocarditis virus (EMCV) infection (54). Thus, further biological studies will be necessary to determine the physiological role of LGP2 in the innate immune system.

#### Viral RNA recognition by RLRs

As mentioned above, RIG-I and MDA5 are responsible for the detection of different types of viruses. Initially, Kato et al. (45) demonstrated that IFN production of RIG-I KO embryonic fibroblasts (MEFs) was defective in response to infection by Newcastle disease virus (NDV), Sendai virus (SeV) (Paramyxoviridae), vesicular stomatitis virus (VSV) (Rhabdoviridae), influenza A virus (Orthomyxoviridae), or Japanese encephalitis virus (JEV) (Flaviviridae), whereas no defect was observed in MDA5-deficient cells. By contrast, IFN production by Picornaviridae, such as EMCV, Theiler’s virus, and Mengo virus, was inhibited in MDA5 KO cells but not in RIG-I KO cells. Recent reports

further demonstrated the requirement for RIG-I or MDA5 in  
the detection of individual viruses (Fig. 1). Collectively, most  
single-stranded RNA (ssRNA) viruses, including positive- and  
negative-stranded viruses, are recognized by RIG-I, whereas  
Picornaviridae and murine norovirus (Calcioviridae) recognition is  
MDA5 dependent (55, 56). However, several viruses are  
differentially recognized. Double-stranded segmented RNA  
virus, Reoviridae, is detected by either RIG-I or MDA5 in a  
segment length-dependent manner (55, 57). Among the Flavi-  
viridae, although JEV and hepatitis C virus (HCV) are detected  
specifically by RIG-I, West Nile virus and Dengue virus are  
likely to be recognized redundantly by both RIG-I and MDA5  
(55, 58). Furthermore, IFN production in response to murine  
hepatitis virus (MHV), which belongs to the Coronaviridae, was  
impaired in MDA5 KO cells but not in RIG-I KO cells, suggesting  
the MDA5-dependent detection of MHV (59).

**RNA structure recognized by RLRs**

This differential viral recognition by RIG-I and MDA5 is based  
on distinct preferences for substrate RNA. This has been  
suggested by the observation that RIG-I is selectively activated  

by transfection with in vitro-transcribed RNA, whereas MDA5  
is selectively activated by transfection with polyI:C (45). In  
2006, two groups (60, 61) independently demonstrated that  
5′-triphosphate (5′-ppp)-containing viral RNA, which is  
observed in most types of RNA virus genome and in vitro-  
transcribed RNA, was recognized selectively by RIG-I as  
non-self foreign RNA. Because 5′-ppp moieties of most host  
transcripts are either masked by a 7-methyl-guanosine  
capping (mRNA) or removed by processing in the maturation  
process (tRNA and rRNA), these self-RNAs can be discrimi-  
nated from non-self viral RNA. The 5′-terminus of Picomaviridae  
and Calcioviridae, which are selectively recognized by MDA5,  
are commonly masked by covalent attachment of the viral  
peptide VPg, resulting in escape from RIG-I detection. It has  
also been reported that viral genome of Hantaan virus  
(HTNV), Crimean-Congo hemorrhagic fever virus (CCHFV)  
(Bunyaviridae), and Borna disease virus (BDV) (Bornaviridae)  
cannot trigger RIG-I signaling, because their 5′-ppp structure  
is removed in infected cells, suggesting another viral strategy  
to escape from RIG-I detection (62). However, it was revealed  
that relatively short (approximately 25 bp) dsRNA with at  
least a single phosphate at either the 5′- or 3′-ends could be  

![Diagram](attachment:diagram.png)

**Fig. 1. Detection of viral infection by retinoic acid-inducible gene-I (RIG-I)-like receptors (RLRs).** Most of the RNA viruses, which contain  
5′-ppp moiety and relatively short dsRNA, are recognized by RIG-I. Long dsRNA generated in the process of viral replication are detected by melanoma  
differentiation-associated gene 5 (MDA5). Hantaan virus, Crimean–Congo hemorrhagic fever virus, and Borna disease virus can escape from RIG-I  
detection, because of lacking 5′-ppp structure of their genome; however, it remains unclear whether these viruses are detected by MDA5 or laboratory  
of genetics and physiology 2 (LGP2).

recognized by RIG-I using an in vitro biochemical assay (39). This observation was confirmed by a report in which processed short polyI:C or short dsRNA segments of Reoviridae were able to activate RIG-I signaling (57). Together, these results indicate strongly that RIG-I selectively recognizes both 5′-ppp–ssRNA and short dsRNA as foreign RNA. It should be noted that a recent report suggested that RIG-I requires a polyuridine motif at the 3′-non-translated region of the HCV genome; however, it remains to be addressed whether this is the case for other viruses (63).

As polyI:C is the only defined synthetic RNA ligand for MDA5, it had been hypothesized that the dsRNA structure could be a signature for MDA5 ligands. Recently, Akira and colleagues (57) demonstrated that relatively long polyI:C (>1 kb) was selectively recognized by MDA5, whereas short polyI:C generated by enzyme digestion (<1 kb) was RIG-I dependent. Furthermore, they showed that long dsRNA produced by EMCV infection and long segments of Reovirus dsRNA genome can trigger MDA5-dependent IFN production. Indeed, the production of abundant double-stranded replication intermediates was observed in EMCV-infected cells (64). Thus, MDA5 selectively detects relatively long dsRNA, which are not observed in endogenous self-RNA.

Molecular basis of RNA recognition by RIG-I

To address the molecular machinery underlying substrate recognition and subsequent ATP-dependent activation of RIG-I, biochemical analysis was performed using recombinant proteins (39). In the binding assay, recombinant RIG-I specifically interacted with 5′-ppp–ssRNA and dsRNA, including polyI:C, in an ATP-independent manner. An in vitro helicase assay revealed that RIG-I unwound dsRNA containing a 3′-overhang (>5 nt), but not blunt-ended or 5′-overhanging RNA, in the presence of ATP. Interestingly, the reporter assay using cultured cells demonstrated that dsRNA resistant to the helicase activity efficiently induced IFN production, whereas 3′-overhang RNA, which are susceptible to unwinding, failed to activate IFN signaling. These observations strongly suggested that stable complex formation between RIG-I and substrate RNA was necessary for RIG-I activation, and the ATPase activity of RIG-I could be required for conformational changes rather than RNA unwinding. An attempt was then made to examine the stable RIG-I–RNA complex. A limited protease digestion assay revealed that a trypsin-resistant 17-kDa fragment was detected in the presence of substrate RNA but not polyI:C. This fragment corresponded to CTD of RIG-I (aa 792–925), which completely overlaps with RD.

binding assay clearly showed that CTD specifically recognized 5′-ppp–ssRNA and short dsRNA, indicating that CTD is responsible for RNA recognition of RIG-I.

A three-dimensional structure of CTD was determined by nuclear magnetic resonance (NMR) and X-ray crystallography (38, 39). Both results exhibited an almost identical structure, which resembled MSS4, a guanosine diphosphate (GDP)/guanosine triphosphate (GTP) exchange factor for small Rab-like GTPase, and incorporates a single zinc ion to stabilize its rigid structure. Interestingly, the electrostatic surface potential indicated that one side of CTD exhibited a cleft-like surface with positive-charged amino acids, and the opposite side was a convex acidic surface, suggesting that the former might be responsible for RNA binding. Indeed, the NMR spectra of residues on the basic surface were preferentially titrated out in the presence of either 5′-ppp–ssRNA or dsRNA. Furthermore, site-directed mutagenesis of full-length RIG-I clearly demonstrated that basic residues in the concave surface are essential for substrate recognition and subsequent signal transduction. By contrast, the function of the acidic convex surface remains unclear. Because it has been reported that RD interacts directly with CARD and the internal linker region of helicase domain (37), the convex surface might be involved in this intramolecular interaction, whereas no direct evidence has been provided.

Although the current biochemical and structural analysis unveiled RNA-binding property of RIG-I, the molecular mechanism of the subsequent ATP-dependent activation remains unclear. A possible model is that inactive RIG-I, whose CARD is masked by intramolecular binding with RD, detects viral RNA via CTD, activates intrinsic ATPase, leading to conformational changes to expose N-terminal CARD for interaction with downstream adapter molecules (Fig. 2). Alternatively, as Cui et al. (38) demonstrated that RD could form an intermolecular homodimer in response to 5′-ppp–RNA interaction, RNA-induced dimerization of RIG-I might be a critical step for the induction of ATPase activity to adopt the correct conformation. Moreover, the observation that polyI:C, despite its high-affinity binding to RIG-I and ATPase-inducing ability, does not activate RIG-I-mediated signaling may suggest the direct involvement of substrate RNA in the determination of RIG-I conformation. Further analysis will be required to address this issue.

RLR-mediated signal transduction

It has been proposed that CARD–CARD homotypic interactions are involved in the assembly of protein complexes to

Yoneyama & Fujita · RNA sensing and signaling by RLRs

Virus infection
↓
Viral RNA
5′-PPP 5′ ppp-ssRNA dsRNA (Short)
↓
+ATP
↓
SIGNAL

Fig. 2. Model of retinoic acid-inducible gene-I (RIG-I) activation. In the absence of viral infection, RIG-I is inactivated by intramolecular interaction between C-terminal repressor domain (RD) and caspase recruitment domain (CARD) or linker region of helicase domain. When viral RNA appears as a result of viral infection, RIG-I selectively detects these non-self-RNAs via basic cleft-like structure of C-terminal domain (CTD) and induces ATP-dependent conformational change to form a dimer or oligomer, which allows CARD to interact with the downstream adapter protein IPS-1. It is unknown how melanoma differentiation-associated gene 5 (MDA5) detects viral dsRNA and activates downstream signaling.

initiate specific biological activity, such as programmed cell death. In 2005, four groups independently identified CARD-containing adapter molecule for RLRs, interferon-β promoter stimulator 1 (IPS-1), also termed mitochondrial antiviral signaling (MAVS), virus-induced signaling adaptor (VISA), and CARD adapter inducing IFN-β (Cardif) (65–68). The IFN-inducing signal was significantly activated by the over-expression of IPS-1 (65–68) and conversely impaired in IPS-1 KO cells in response to either RIG-I-specific or MDA5-specific stimulation (69, 70), indicating that IPS-1 is indispensable for RLR-mediated signal transduction. IPS-1 consists of an N-terminal single CARD and a proline-rich region (PRR) in the middle of the protein (Fig. 3). Interestingly, IPS-1 encodes a transmembrane domain at its C-terminal end and was shown to be expressed on the outer membrane of mitochondria (67), suggesting a critical function of mitochondria as a platform for RLR-mediated signaling. Indeed, the enforced mislocalization of IPS-1 to the plasma membrane or endoplasmic reticulum (ER) or truncation of its C-terminal transmembrane domain resulted in the failure of signal activation. However, the physiological significance of mitochondrial expression is not fully understood.

Activated RIG-I and MDA5 are predicted to interact with IPS-1 via CARD-CARD interaction, which induces the recruitment of downstream signaling molecules (Fig. 3). It has been revealed that tumor necrosis factor (TNF) receptor-associated factor (TRAF) family members are involved in this signaling cascade. TRAF3, which is known to be a E3 ligase for Lys63-linked polyubiquitination, directly interacts with the TRAF-interacting motif (TIM) in the PRR of IPS-1 (71, 72). TRAF3 KO cells showed substantial impairment of the virus-induced signal, indicating its essential role in RLR signaling. Similarly, interactions between IPS-1 and TRAF2 or TRAF6 were demonstrated (66). Recruited TRAF family members transmit signals to downstream protein kinases, inhibitor of NF-κB (IkB) kinase (IKK) family members, which are essential for the activation of transcription factors IRF-3/7 and NF-κB. The canonical IKK complex, consisting of IKK-α, IKK-β, and the regulatory subunit NF-κB essential modulator (NEMO) (also termed IKK-γ), phosphorylates IkB, and subsequent proteasome-dependent degradation of IkB allows functional NF-κB to translocate to the nucleus (73). By contrast, non-canonical IKKs, TRAF family member-associated NF-κB activator (TANK)-binding kinase 1 (TBK1) and IKK-i (also termed IKKε), activate the signal-dependent phosphorylation of IRF3 and IRF7 to form a functional homodimer or heterodimer (74–78). An interaction between IPS-1 and TRAF3 is essential for the recruitment of both IKK complexes, whereas TRAF2/6 is likely to be responsible

Yoneyama & Fujita · RNA sensing and signaling by RLRs

Fig. 3. Retinoic acid-inducible gene-I (RIG-I)-like receptor (RLR)-mediated signaling and its regulation. Activated RIG-I and melanoma differentiation-associated gene 5 (MDA5) interact with interferon-β promoter stimulator 1 (IPS-1) and transmit signal through the indicated signaling molecules (details are discussed in the text). The black arrows and blue lines indicate positive or negative regulation of the signal, respectively, by cellular proteins. The red lines represent inhibition by viral inhibitors.

for NF-κB activation (66, 72). As a regulatory component, it has been reported that TANK and NAK-associated protein 1 (NAP1) are commonly involved in TBK-1/IKK-i-mediated activation of IRFs (79, 80). NEMO, which is known to be a regulatory subunit for the canonical IKK complex, plays a role even in TBK-1/IKK-i-mediated activation of IRFs (81), suggesting crosstalk between these two IKK complexes in RLR-mediated signaling.

The involvement of Fas-associated death domain (FADD) and receptor interacting protein 1 (RIP1), which are both death domain (DD)-containing molecules, in the IPS-1 signaling complex was demonstrated (65, 82). These molecules interact with the C-terminal portion of IPS-1 and activate NF-κB-activating axis via interaction and activation of caspase-8 and caspase-10 (65, 83). Furthermore, recent reports demonstrated that TNFR-associated DD (TRADD) forms a complex with IPS-1, TRAF3, TANK, FADD, and RIP1, and leads to the activation of both IRF3 and NF-κB (84). This observation is closely related to TNFR-mediated signaling, which is known to form a similar signaling complex (85); however, it remains unknown why IRF-3 is selectively activated by the IPS-1-containing signaling complex but not in TNFR-mediated signaling. A plausible explanation is that mitochondrial localization of the former may recruit a differential signaling complex. In this context, mediator of IRF-3 activation (MITA), also known as stimulator of interferon genes (STING), was identified as a critical scaffold protein for IRF-3 activation (86, 87). MITA, which is expressed on the outer membrane of mitochondria with its four transmembrane domains, interacts directly with IPS-1 and IRF-3 and positively regulates the virus-dependent recruitment of TBK1 to the IPS-1 complex on mitochondria, suggesting a possible branching factor for the IRF-3-activation pathway. However, another report (87) demonstrated that STING is predominantly expressed in ER via its transmembrane domains, interacted directly with RIG-I or the translocon-associated proteins but not MDA5 and involved in IFN activation, suggesting an alternative signaling pathway other than the mitochondria-mediated signaling. As the discrepancy between these two reports still remains unclear, further analysis will be necessary to evaluate a significance of this molecule in innate immune signaling.

Regulation of RLR signaling by cellular proteins

RIG-I and MDA5 are ubiquitously expressed in most types of cells and known to be strong ISGs, indicating positive feedback regulation. Indeed, viral infection or IFN treatment induces robust upregulation of both mRNA and protein levels of RIG-I and MDA5. This observation clearly explains the classical ‘priming’ effect, which is defined as accelerated IFN production in response to viral infection in IFN-pretreated cells. However, LGP2, also known as ISG, is involved in negative feedback regulation against some viral infections, as shown by the observation that the over-expression of LGP2 strongly inhibits virus-induced IFN production (44, 52, 53). This inhibitory effect of LGP2 is likely to be exerted by multiple steps. One possible mechanism is the sequestration of ligand RNA from recognition by RIG-I or MDA5, because LGP2 can recognize RNA ligands with high affinity (44). Alternatively, the C-terminal portion of LGP2, which can function as RD, may directly inhibit the self-dimerization of activated RIG-I (37). Furthermore, it has been shown that LGP2 can interact directly with IPS-1 and competes for binding between IPS-1 and IKK-i (53).

Recent reports have provided new molecules that are responsible for the regulation of RLR-mediated signaling (88, 89) (Fig. 3). In particular, observations for post-translational modification of RLRs have been accumulating. Tripartite motif protein 25 (TRIM25), which is E3 ubiquitin ligase, specifically interacts with RIG-I CARD and conjugates Lys-63-linked ubiquitin at the Lys172 residue of RIG-I CARD (90). The data clearly indicated that the ubiquitination of RIG-I CARD by TRIM25 is essential for the interaction with IPS-1, as well as subsequent signal activation. Interestingly, TRIM25 does not induce the ubiquitination of MDA5, suggesting a differential regulation of CARDs in RLRs, whilst it is unknown whether MDA5 requires ubiquitination for its interaction with IPS-1. By contrast, ubiquitination of RIG-I by the E3 ligase ring finger 125 (RNF125) had the opposite effect (91). Lys-48-linked ubiquitination in the N-terminal portion of RIG-I, and also presumably MDA5, induces proteosomal degradation, suggesting the negative regulation of RIG-I-mediated signaling by RNF125. Furthermore, the conjugation of ubiquitin-like ISG15 proteins to RIG-I has also been described for the negative regulation of RIG-I-mediated signaling (92, 93).

De-ubiquitinating enzyme A (DUBA), which is characteristic of ovarian tumor domain (OTU), interacted directly with TRAF3 and catalyzed the cleavage of Lys-63-linked ubiquitin chains of TRAF3. This de-ubiquitination by DUBA induces the dissociation of TBK1 from TRAF3, resulting in the

sequestration of RLR-mediated signaling (94). Furthermore, another OTU-containing DUB, A20, which is known to be involved in NF-kB activation, shows a negative regulatory effect on RLR signaling (95–97). In this case, the C-terminal ubiquitin ligase domain of A20, rather than the N-terminal de-ubiquitination domain, is critical for its inhibitory activity (97), suggesting that protein degradation of uncharacterized target molecules is involved in the signaling cascade. Another OTU-containing DUB protein, CYLD (cylindromatosis), has also been reported to be a negative regulator of RIG-I-mediated signaling (98, 99). CYLD physically interacts with RIG-I or IPS-1 and inactivates signaling via dissociation of Lys-63-linked ubiquitin moieties of RIG-I, suggesting a counteraction to TRAM25-mediated ubiquitination of RIG-I. Collectively, although the recent reports strongly suggested positive and negative regulation of RLRs and related signaling molecules via post-translational modifications, the precise regulatory mechanisms remain to be elucidated.

Several molecules that directly interact with RLRs or related molecules and attenuate their signaling activities have been described (Fig. 3). Suppressor of IKKε (SIKE) forms a complex with TBK1 and/or IKK-i (IKKε) in the absence of viral stimulation and inhibits these kinases from interacting with RIG-I or IRF3 (100). Upon viral infection, SIKE dissociates from the TBK1/IKK-i complex via an unknown mechanism and the released kinases can be incorporated into the signaling complex. Atg5–Atg12 conjugate, an essential regulator of autophagic machinery, directly associates with the CARD of RIG-I and IPS-1 and inhibits the interaction between them (101). Interestingly, a member of NLR family protein, NLRX1, has also been identified as a negative regulator of RLR-mediated antiviral signal (102). NLRX1 is ubiquitously expressed on the outer membrane of mitochondria (103) and negatively regulates IPS-1 activity via a direct interaction between IPS-1 CARD and NLRX1 NBD. This observation is intriguing in understanding the relationship between the two cytoplasmic sensor families. It has been reported that SH2 domain-containing phosphatase-1 (SHP-1) can activate RIG-I-mediated IFN production by directly binding to and inhibiting the activation of interleukin-1 receptor-associated kinase 1 (IRAK1) (104); however, the molecular mechanism of the inhibitory effect by IRAK1 remains to be characterized. As a MDA5-specific negative regulator, dihydroacetone kinase (DAK) has been identified as a MDA5-interacting protein (105). The data indicated that DAK specifically associates with MDA5, but not with RIG-I, and inhibits polyI:C-induced IFN production. Collectively, the RLR-mediated signaling pathway is regulated by a complicated mode of actions with multiple

molecules. Notably, most of them are negative regulations, presumably because unwanted or prolonged activation of antiviral signaling is harmful for host survival.

Interaction between RLRs and viral proteins

It has been documented that viruses have a strategy to interfere with host immune responses to assist their survival. Information about viral inhibitors targeting RLRs and IPS-1 has also been accumulating.

The simplest strategy for viruses to resist antiviral defense machinery is to escape from the RLR detection system. Several instances have been documented in recent reports. As mentioned above, RNA viruses, including HTNV, CCHFV, and BDV, did not induce IFN, because they do not contain a 5′-ppp moiety in their own genome in infected cells, and thus escape from RIG-I detection (62). Moreover, Picornaviridae and Calcioviridae encode a VPg protein, which is covalently attached at the 5′-terminal, and hide from RIG-I (106). However, these two species are detected by MDA5, which suggests that hosts might develop alternative sensing machinery to counter the viral escape strategy.

Another strategy is the development of viral proteins that directly inhibit the function of RLRs or IPS-1. V proteins of Paramyxoviridae, including NDV and SeV, specifically interact with MDA5 and interfere with its activity to transmit signals (44, 107–109). As Paramyxoviridae are sensed by RIG-I, the biological meaning of this specific inhibition of MDA5 is not obvious. However, a recent report demonstrated that the defective interfering (DI) particle of SeV, which contains double-stranded species as a result of abnormal copy-back genome synthesis, could be detected by MDA5 (110), explaining the probable connection between the V protein of Paramyxoviridae and MDA5. Nonstructural protein 1 (NS1) of influenza A virus was reported to interact with RIG-I and inhibit its function (61, 111–113). Whilst the inhibition of IFN production by NS1 has been predicted to be due to its RNA-binding activity, thereby sequestering viral dsRNA from RLR detection, this observation suggests a differential mode of inhibitory action on NS1. By contrast, the protease activity of HCV NS3/4A is known to be an inhibitor of RIG-I-

mediated IRF3 activation (114). This inhibition is obtained by the specific cleavage at Cys508 of IPS-1 by NS3/4A to disrupt its mitochondrial localization (68, 115, 116). Identical processing is observed in GB virus B (Flaviviridae), which is phylogenetically related to HCV (117). Furthermore, it was reported that 3ABC protease of hepatitis A virus, a member of the Picomaviridae, cleaved IPS-1 at residue Q428 (118), which suggests that the specific cleavage of IPS-1 is a common strategy for viruses to attenuate host antiviral activity. Moreover, reports of viral inhibitors that disrupt the RLR-mediated antiviral signaling have been accumulating (88, 119). This finding suggests that the balance between host defense system and viral inhibitory effect could be critical for both viral growth and eradication in infected cells.

Concluding remarks and future perspectives

This review summarizes the current knowledge about the molecular mechanisms in antiviral innate immunity, including recognition of non-self-RNA by RLRs and the regulation of subsequent signal transduction leading to activation of type I IFNs. Recent studies have greatly advanced the understanding of self/non-self-discrimination and interaction between host and pathogen in innate immune signaling. In the recognition of viral RNA, molecular patterns of viral nucleotides for detection by RLRs were clarified, i.e. RIG-I recognizes 5′-ppp or short dsRNA and MDA5 detects long dsRNA. It was also demonstrated that CTD and CARD, which detect foreign RNA and transmit signals, respectively, are essential for RIG-I. However, the molecular basis underlying subsequent conformational changes of RLRs, including the activation of ATPase for triggering biological signals, is still unclear. It also still remains undetermined how RLRs access the viral ribonucleoprotein complex in infected cells. Although molecules involved in RLR-mediated signaling are characterized extensively, the precise regulation of these factors in immunogenic signaling cascades has not been fully elucidated. Further exploration of these issues will provide new insights into the antiviral immune system and may help us to discover new therapeutic approaches targeting RLR-mediated signaling for some viral infectious diseases.

References

1. Takeda K, Kaisho T, Akira S. Toll-like receptors. Annu Rev Immunol 2003;21:335–376.
2. Akira S, Uematsu S, Takeuchi O. Pathogen recognition and innate immunity. Cell 2006;124:783–801.
3. Willment JA, Brown GD. C-type lectin receptors in antifungal immunity. Trends Microbiol 2008;16:27–32.

4. Martinon F, Gaide O, Petrilli V, Mayor A, Tschopp J. NALP inflammasomes: a central role in innate immunity. Semin Immunopathol 2007;29:213–229.
5. Inohara N, Chamaillard M, McDonald C, Nunez G. NOD-LRR proteins: role in host-microbial interactions and inflammatory disease. Annu Rev Biochem 2005;74: 355–383.
6. Stetson DB, Medzhitov R. Recognition of cytosolic DNA activates an IRF3-dependent innate immune response. Immunity 2006;24:93–103.
7. Ishii KJ, et al. A Toll-like receptor-independent antiviral response induced by double-stranded B-form DNA. Nat Immunol 2006;7:40–48.
8. Takaoka A, et al. DAI (DLM-1/ZBP1) is a cytosolic DNA sensor and an activator of innate immune response. Nature 2007;448:501–505.
9. Ishii KJ, et al. TANK-binding kinase-1 delineates innate and adaptive immune responses to DNA vaccines. Nature 2008;451:725–729.
10. Yoneyama M, Onomoto K, Fujita T. Cytoplasmic recognition of RNA. Adv Drug Deliv Rev 2008;60:841–846.
11. Samuel CE. Antiviral actions of interferons. Clin Microbiol Rev 2001;14: 778–809.
12. Isaacs A, Lindenmann J. Virus interference. I. The interferon. Proc R Soc Lond B Biol Sci 1957;147:258–267.
13. Nagano Y, Kojima Y. Immunizing property of vaccinia virus inactivated by ultraviolets rays. C R Seances Soc Biol Fil 1954;148:1700–1702.
14. Nagata S, Mantei N, Weissmann C. The structure of one of the eight or more distinct chromosomal genes for human interferon-α. Nature 1980;287:401–408.
15. Taniguchi T, Fujii-Kuriyama Y, Muramatsu M. Molecular cloning of human interferon cDNA. Proc Natl Acad Sci USA 1980;77:4003–4006.
16. Honda K, Takaoka A, Taniguchi T. Type I interferon gene induction by the interferon regulatory factor family of transcription factors. Immunity 2006;25:349–360.
17. van Boxel-Dezaire AH, Rani MR, Stark GR. Complex modulation of cell type-specific signaling in response to type I interferons. Immunity 2006;25:361–372.
18. Du W, Maniatis T. An ATF/CREB binding site is required for virus induction of the human interferon beta gene. Proc Natl Acad Sci USA 1992;89:2150–2154.
19. Fujita T, Miyamoto M, Kimura Y, Hammer J, Taniguchi T. Involvement of a cis-element that binds an H2TF-1/NFκB like factor(s) in the virus-induced interferon-β gene expression. Nucleic Acids Res 1989;17: 3335–3346.
20. Lenardo MJ, Fan CM, Maniatis T, Baltimore D. The involvement of NF-kappa B in beta-interferon gene regulation reveals its role as widely inducible mediator of signal transduction. Cell 1989;57:287–294.
21. Miyamoto M, et al. Regulated expression of a gene encoding a nuclear factor, IRF-1, that specifically binds to IFN-β gene regulatory elements. Cell 1988;54: 903–913.
22. Carey M. The enhanceosome and transcriptional synergy. Cell 1998;92:5–8.
23. Ryals J, Dierks P, Ragg H, Weissmann C. A 46-nucleotide promoter segment from an IFN-α gene renders an unrelated promoter inducible by virus. Cell 1985;41:497–507.
24. Yoneyama M, Suhara W, Fukuhara Y, Fukuda M, Nishida E, Fujita T. Direct triggering of the type I interferon system by virus infection: activation of a transcription factor complex containing IRF-3 and CBP/p300. EMBO J 1998;17:1087–1095.
25. Lin R, Heylbroeck C, Pitha PM, Hiscott J. Virus-dependent phosphorylation of the IRF-3 transcription factor regulates nuclear translocation, transactivation potential, and proteasome-mediated degradation. Mol Cell Biol 1998;18:2986–2996.
26. Weaver BK, Kumar KP, Reich NC. Interferon regulatory factor 3 and CREB-binding protein/p300 are subunits of double-stranded RNA-activated transcription factor DRAF1. Mol Cell Biol 1998;18: 1359–1368.
27. Sato M, Tanaka N, Hata N, Oda E, Taniguchi T. Involvement of the IRF family transcription factor IRF-3 in virus-induced activation of the IFN-beta gene. FEBS Lett 1998;425:112–116.
28. Sato M, Hata N, Asagiri M, Nakaya T, Taniguchi T, Tanaka N. Positive feedback regulation of type I IFN genes by the IFN-inducible transcription factor IRF-7. FEBS Lett 1998;441:106–110.
29. Wathelet MG, Lin CH, Parekh BS, Ronco LV, Howley PM, Maniatis T. Virus infection induces the assembly of coordinately activated transcription factors on the IFN-β enhancer in vivo. Mol Cell 1998;1: 507–518.
30. Marie I, Durbin JE, Levy DE. Differential viral induction of distinct interferon-alpha genes by positive feedback through interferon regulatory factor-7. EMBO J 1998;17:6660–6669.
31. Sato M, et al. Distinct and essential roles of transcription factors IRF-3 and IRF-7 in response to viruses for IFN-alpha/beta gene induction. Immunity 2000;13: 539–548.
32. Honda K, et al. IRF-7 is the master regulator of type-I interferon-dependent immune responses. Nature 2005;434:772–777.
33. Stetson DB, Medzhitov R. Type I interferons in host defense. Immunity 2006;25: 373–381.
34. Sun YW. RIG-I, a human homolog gene of RNA helicase, is induced by retinoic acid during the differentiation of acute promyelocytic leukemia cell. Thesis, Shanghai Second Medical University, 1997.
35. Zhang X, Wang C, Schook LB, Hawken RJ, Rutherford MS. An RNA helicase, RHIV-1, induced by porcine reproductive and respiratory syndrome virus (PRRSV) is mapped on porcine chromosome 10q13. Microb Pathog 2000;28:267–278.
36. Yoneyama M, et al. The RNA helicase RIG-I has an essential function in double-stranded RNA-induced innate antiviral responses. Nat Immunol 2004;5:730–737.
37. Saito T, et al. Regulation of innate antiviral defenses through a shared repressor domain in RIG-I and LGP2. Proc Natl Acad Sci USA 2007;104:582–587.
38. Cui S, et al. The C-terminal regulatory domain is the RNA 5′-triphosphate sensor of RIG-I. Mol Cell 2008;29:169–179.
39. Takahasi K, et al. Nonself RNA-sensing mechanism of RIG-I helicase and activation of antiviral immune responses. Mol Cell 2008;29:428–440.
40. Kato H, et al. Cell type-specific involvement of RIG-I in antiviral response. Immunity 2005;23:19–28.
41. Wang Y, et al. Rig-I−/− mice develop colitis associated with downregulation of Gox2. Cell Res 2007;17:858–868.
42. Zhang NN, et al. RIG-I plays a critical role in negatively regulating granulocytic proliferation. Proc Natl Acad Sci USA 2008;105:10553–10558.
43. Kang DC, Gopalkrishnan RV, Wu Q, Jankowsky E, Pyle AM, Fisher PB. mda-5: an interferon-inducible putative RNA helicase with double-stranded RNA-dependent ATPase activity and melanoma growth-suppressive properties. Proc Natl Acad Sci USA 2002;99:637–642.
44. Yoneyama M, et al. Shared and unique functions of the DExD/H-box helicases RIG-I, MDA5, and LGP2 in antiviral innate immunity. J Immunol 2005;175: 2851–2858.
45. Kato H, et al. Differential roles of MDA5 and RIG-I helicases in the recognition of RNA viruses. Nature 2006;441: 101–105.
46. Gitlin L, et al. Essential role of mda-5 in type I IFN responses to polyriboinosinic:polyribocytidylic acid and encephalomyocarditis picornavirus. Proc Natl Acad Sci USA 2006;103:8459–8464.

47. Kang DC, et al. Expression analysis and genomic characterization of human melanoma differentiation associated gene-5, mda-5: a novel type I interferon-responsive apoptosis-inducing gene. Oncogene 2004;23:1789–1800.

48. Kovacsovics M, Martinon F, Micheau O, Bodmer JL, Hofmann K, Tschopp J. Overexpression of Helicard, a CARD-containing helicase cleaved during apoptosis, accelerates DNA degradation. Curr Biol 2002; 12:838–843.

49. Barral PM, et al. MDA-5 is cleaved in poliovirus-infected cells. J Virol 2007; 81:3677–3684.

50. Smyth DJ, et al. A genome-wide association study of nonsynonymous SNPs identifies a type 1 diabetes locus in the interferon-induced helicase (IFIH1) region. Nat Genet 2006;38:617–619.

51. Cui Y, et al. The Stat3/5 locus encodes novel endoplasmic reticulum and helicase-like proteins that are preferentially expressed in normal and neoplastic mammary tissue. Genomics 2001;78:129–134.

52. Rothenfusser S, et al. The RNA helicase Lgp2 inhibits TLR-independent sensing of viral replication by retinoic acid-inducible gene-I. J Immunol 2005;175:5260–5268.

53. Komuro A, Horvath CM. RNA- and virus-independent inhibition of antiviral signaling by RNA helicase LGP2. J Virol 2006;80:12332–12342.

54. Venkataraman T, et al. Loss of DExD/H box RNA helicase LGP2 manifests disparate antiviral responses. J Immunol 2007; 178:6444–6455.

55. Loo YM, et al. Distinct RIG-I and MDA5 signaling by RNA viruses in innate immunity. J Virol 2008;82:335–345.

56. McCartney SA, Thackray LB, Gitlin L, Gilfillan S, Virgin Iv HW, Colonna M. MDA-5 recognition of a murine norovirus. PLoS Pathog 2008;4:e1000108.

57. Kato H, et al. Length-dependent recognition of double-stranded ribonucleic acids by retinoic acid-inducible gene-I and melanoma differentiation-associated gene 5. J Exp Med 2008;205:1601–1610.

58. Fredericksen BL, Gale M Jr. West Nile virus evades activation of interferon regulatory factor 3 through RIG-I-dependent and -independent pathways without antagonizing host defense signaling. J Virol 2006;80:2913–2923.

59. Roth-Cross JK, Bender SJ, Weiss SR. Murine Coronavirus mouse hepatitis virus (MHV) is recognized by MDA5 and induces Type I IFN in brain macrophages/microglia. J Virol 2008;82:9829–9838.

60. Hornung V, et al. 5′-Triphosphate RNA is the ligand for RIG-I. Science 2006;314:994–997.

61. Pichlmair A, et al. RIG-I-mediated antiviral responses to single-stranded RNA bearing 5′-phosphates. Science 2006;314: 997–1001.

62. Habjan M, et al. Processing of genome 5′ termini as a strategy of negative-strand RNA viruses to avoid RIG-I-dependent interferon induction. PLoS ONE 2008;3:e2032.

63. Saito T, Owen DM, Jiang F, Marcotrigiano J, Gale M Jr. Innate immunity induced by composition-dependent RIG-I recognition of hepatitis C virus RNA. Nature 2008;454:523–527.

64. Weber F, Wagner V, Rasmussen SB, Hartmann R, Paludan SR. Double-stranded RNA is produced by positive-strand RNA viruses and DNA viruses but not in detectable amounts by negative-strand RNA viruses. J Virol 2006;80:5059–5064.

65. Kawai T, et al. IPS-1, an adaptor triggering RIG-I- and Mda5-mediated type I interferon induction. Nat Immunol 2005;6:981–988.

66. Xu LG, Wang YY, Han KJ, Li LY, Zhai Z, Shu HB. VISA is an adapter protein required for virus-triggered IFN-beta signaling. Mol Cell 2005;19:727–740.

67. Seth RB, Sun L, Ea CK, Chen ZJ. Identification and characterization of MAVS, a mitochondrial antiviral signaling protein that activates NF-kB and IRF 3. Cell 2005; 122:669–682.

68. Meylan E, et al. Cardif is an adaptor protein in the RIG-I antiviral pathway and is targeted by hepatitis C virus. Nature 2005;437:1167–1172.

69. Sun Q, et al. The specific and essential role of MAVS in antiviral innate immune responses. Immunity 2006;24:633–642.

70. Kumar H, et al. Essential role of IPS-1 in innate immune responses against RNA viruses. J Exp Med 2006;203:1795–1803.

71. Oganesyan G, et al. Critical role of TRAF3 in the Toll-like receptor-dependent and -independent antiviral response. Nature 2006;439:208–211.

72. Saha SK, et al. Regulation of antiviral responses by a direct and specific interaction between TRAF3 and Cardif. EMBO J 2006;25:3257–3263.

73. Karin M, Ben-Neriah Y. Phosphorylation meets ubiquitination: the control of NF-[kappa]B activity. Annu Rev Immunol 2000;18:621–663.

74. Fitzgerald KA, et al. IKKε and TBK1 are essential components of the IRF3 signaling pathway. Nat Immunol 2003;4:491–496.

75. Sharma S, tenOever BR, Grandvaux N, Zhou GP, Lin R, Hiscott J. Triggering the interferon antiviral response through an IKK-related pathway. Science 2003;300:1148–1151.

76. Hemmi H, et al. The roles of two IkappaB kinase-related kinases in lipopolysaccharide and double stranded RNA signaling and viral infection. J Exp Med 2004;199: 1641–1650.

77. McWhirter SM, Fitzgerald KA, Rosains J, Rowe DC, Golenbock DT, Maniatis T. IFN-regulatory factor 3-dependent gene expression is defective in Tbk1-deficient mouse embryonic fibroblasts. Proc Natl Acad Sci USA 2004;101:233–238.

78. Perry AK, Chow EK, Goodnough JB, Yeh WC, Cheng G. Differential requirement for TANK-binding kinase-1 in type I interferon responses to toll-like receptor activation and viral infection. J Exp Med 2004;199:1651–1658.

79. Guo B, Cheng G. Modulation of the interferon antiviral response by the TBK1/IKKi adaptor protein TANK. J Biol Chem 2007;282:11817–11826.

80. Sasai M, et al. NAK-associated protein 1 participates in both the TLR3 and the cytoplasmic pathways in type I IFN induction. J Immunol 2006;177:8676–8683.

81. Zhao T, et al. The NEMO adaptor bridges the nuclear factor-kB and interferon regulatory factor signaling pathways. Nat Immunol 2007;8:592–600.

82. Balachandran S, Thomas E, Barber GN. A FADD-dependent innate immune mechanism in mammalian cells. Nature 2004;432:401–405.

83. Takahashi K, Kawai T, Kumar H, Sato S, Yonehara S, Akira S. Roles of caspase-8 and caspase-10 in innate immune responses to double-stranded RNA. J Immunol 2006;176:4520–4524.

84. Michallet MC, et al. TRADD protein is an essential component of the RIG-like helicase antiviral pathway. Immunity 2008;28:651–661.

85. Micheau O, Tschopp J. Induction of TNF receptor I-mediated apoptosis via two sequential signaling complexes. Cell 2003;114:181–190.

86. Zhong B, et al. The adaptor protein MITA links virus-sensing receptors to IRF3 transcription factor activation. Immunity 2008;29:538–550.

87. Ishikawa H, Barber GN. STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling. Nature 2008;455:674–678.

88. Komuro A, Bamming D, Horvath CM. Negative regulation of cytoplasmic RNA-mediated antiviral signaling. Cytokine 2008;43:350–358.

89. Moore CB, Ting JP. Regulation of mitochondrial antiviral signaling pathways. Immunity 2008;28:735–739.

90. Gack MU, et al. TRIM25 RING-finger E3 ubiquitin ligase is essential for RIG-I-medi-

ated antiviral activity. Nature 2007; 446:916–920.

91. Arimoto K, Takahashi H, Hishiki T, Konishi H, Fujita T, Shimotohno K. Negative regulation of the RIG-I signaling by the ubiquitin ligase RNF125. Proc Natl Acad Sci USA 2007;104:7500–7505.

92. Zhao C, Denison C, Huibregtse JM, Gygi S, Krug RM. Human ISG15 conjugation targets both IFN-induced and constitutively expressed proteins functioning in diverse cellular pathways. Proc Natl Acad Sci USA 2005;102:10200–10205.

93. Kim MJ, Hwang SY, Imaizumi T, Yoo JY. Negative feedback regulation of RIG-I-mediated antiviral signaling by interferon-induced ISG15 conjugation. J Virol 2008;82:1474–1483.

94. Kayagaki N, et al. DUBA: a deubiquitinating enzyme that regulates type I interferon production. Science 2007;318:1628–1632.

95. Saitoh T, et al. A20 is a negative regulator of IFN regulatory factor 3 signaling. J Immunol 2005;174:1507–1512.

96. Wang YY, Li L, Han KJ, Zhai Z, Shu HB. A20 is a potent inhibitor of TLR3- and Sendai virus-induced activation of NF-kappaB and ISRE and IFN-beta promoter. FEBS Lett 2004;576:86–90.

97. Lin R, et al. Negative regulation of the retinoic acid-inducible gene I-induced antiviral state by the ubiquitin-editing protein A20. J Biol Chem 2006;281:2095–2103.

98. Friedman CS, et al. The tumour suppressor CYLD is a negative regulator of RIG-I-mediated antiviral response. EMBO Rep 2008;9:930–936.

99. Zhang M, et al. Regulation of IkappaB kinase-related kinases and antiviral responses by tumor suppressor CYLD. J Biol Chem 2008;283:18621–18626.

100. Huang J, Liu T, Xu LG, Chen D, Zhai Z, Shu HB. SIKE is an IKK epsilon/TBK1-associated

suppressor of TLR3- and virus-triggered IRF-3 activation pathways. EMBO J 2005;24:4018–4028.

101. Jounai N, et al. The Atg5 Atg12 conjugate associates with innate antiviral immune responses. Proc Natl Acad Sci USA 2007;104:14050–14055.

102. Moore CB, et al. NLRX1 is a regulator of mitochondrial antiviral immunity. Nature 2008;451:573–577.

103. Tattoli I, et al. NLRX1 is a mitochondrial NOD-like receptor that amplifies NF-kB and JNK pathways by inducing reactive oxygen species production. EMBO Rep 2008;9:293–300.

104. An H, et al. Phosphatase SHP-1 promotes TLR- and RIG-I-activated production of type I interferon by inhibiting the kinase IRAK1. Nat Immunol 2008;9:542–550.

105. Dia F, et al. Negative regulation of MDA5-but not RIG-I-mediated innate antiviral signaling by the dihydroxyacetone kinase. Proc Natl Acad Sci USA 2007;104:11706–11711.

106. Paul AV. Possible unifying mechanism of Picornavirus genome replication. In: Semler B, Wimmer E, eds. Molecular Biology of Picornaviruses. Washington, DC: ASM Press, 2002:227–246.

107. Andrejeva J, et al. The V proteins of paramyxoviruses bind the IFN-inducible RNA helicase, mda-5, and inhibit its activation of the IFN-beta promoter. Proc Natl Acad Sci USA 2004;101:17264–17269.

108. Childs K, et al. mda-5, but not RIG-I, is a common target for paramyxovirus V proteins. Virology 2007;359:190–200.

109. Komatsu T, Takeuchi K, Gotoh B. Bovine parainfluenza virus type 3 accessory proteins that suppress beta interferon production. Microbes Infect 2007;9:954–962.

110. Yount JS, Gitlin L, Moran TM, Lopez CB. MDA5 participates in the detection of paramyxovirus infection and is essential for the

early activation of dendritic cells in response to Sendai Virus defective interfering particles. J Immunol 2008;180:4910–4918.

111. Mibayashi M, Martinez-Sobrido L, Loo YM, Cardenas WB, Gale M Jr, Garcia-Sastre A. Inhibition of retinoic acid-inducible gene I-mediated induction of beta interferon by the NS1 protein of influenza A virus. J Virol 2007;81:514–524.

112. Opitz B, et al. IFNβ induction by influenza A virus is mediated by RIG-I which is regulated by the viral NS1 protein. Cell Microbiol 2007;9:930–938.

113. Guo Z, et al. NS1 protein of influenza A virus inhibits the function of intracytoplasmic pathogen sensor, RIG-I. Am J Respir Cell Mol Biol 2007;36:263–269.

114. Foy E, et al. Control of antiviral defenses through hepatitis C virus disruption of retinoic acid-inducible gene-I signaling. Proc Natl Acad Sci USA 2005;102:2986–2991.

115. Loo YM, et al. Viral and therapeutic control of IFN-beta promoter stimulator 1 during hepatitis C virus infection. Proc Natl Acad Sci USA 2006;103:6001–6006.

116. Lin R, et al. Dissociation of a MAVS/IPS-1/VISA/Cardif-IKKε molecular complex from the mitochondrial outer membrane by hepatitis C virus NS3-4A proteolytic cleavage. J Virol 2006;80:6072–6083.

117. Chen Z, et al. GB virus B disrupts RIG-I signaling by NS3/4A-mediated cleavage of the adaptor protein MAVS. J Virol 2007;81:964–976.

118. Yang Y, et al. Disruption of innate immunity due to mitochondrial targeting of a picornaviral protease precursor. Proc Natl Acad Sci USA 2007;104:7253–7258.

119. Yoneyama M, Fujita T. Function of RIG-I-like receptors in antiviral innate immunity. J Biol Chem 2007;282:15315–15318.
