
Biochemistry 2009, 48, 1025–1035

Identification and Characterization of the *Staphylococcus aureus* Gene Cluster Coding for Staphyloferrin A†

Jennifer L. Cotton, Jianshi Tao, and Carl J. Balibar*

Department of Infectious Diseases, Novartis Institutes for BioMedical Research, 500 Technology Square, Cambridge, Massachusetts 02139

Received September 29, 2008; Revised Manuscript Received December 10, 2008

ABSTRACT: Siderophores are key virulence factors that allow bacteria to grow in iron-restricted environments. The Gram-positive pathogen *Staphylococcus aureus* is known to produce four siderophores for which genetic and/or structural data are unknown. Here we characterize the gene cluster responsible for producing the prevalent siderophore staphyloferrin A. In addition to expressing the cluster in the heterologous host *Escherichia coli*, which confers the ability to synthesize the siderophore, we reconstituted staphyloferrin A biosynthesis *in vitro* by expressing and purifying two key enzymes in the pathway. As with other polycarboxylate siderophores, staphyloferrin A is biosynthesized using the recently described nonribosomal peptide synthetase independent siderophore (NIS) biosynthetic pathway. Two NIS synthetases condense two molecules of citric acid to D-ornithine in a stepwise ordered process with SfnaD using the δ-amine as a nucleophile to form the first amide followed by SfnaB utilizing the α-amine to complete staphyloferrin A synthesis.

The facultatively aerobic Gram-positive bacterium *Staphylococcus aureus* is a natural inhabitant of the skin and mucous membranes of mammals. With its ability to cause disease in humans ranging from mild wound infections to more serious ailments including septic shock, endocarditis, and osteomyelitis (1), *S. aureus* has gained a great deal of notoriety. Although most prevalent in hospital settings, the emergence of multidrug-resistant *S. aureus* (MRSA¹) in community acquired infections is concerning (2), especially considering that the number of deaths in the United States attributed to MRSA has doubled in just 5 years and now surpasses the number of deaths caused by AIDS (3).

As with most other microorganisms, iron is an essential factor for growth of *S. aureus*. Despite its high abundance on earth, the bioavailability of iron is relatively low because in aerobic environments ferric iron forms insoluble (oxy)-hydroxide precipitates. Within higher eukaryotes, iron limitation is overcome with high affinity solubilizing glycoproteins such as transferrin and lactoferrin (4). However, this stringent control of iron transport in eukaryotes results in further restriction of extracellular iron, presenting major challenges to invading pathogens which must assimilate iron in order to grow and establish an infection. One of the most common strategies employed by pathogenic bacteria to circumvent iron limitation is to release and then reabsorb low-molecular-weight high-affinity iron chelators termed siderophores (5). These ferric-siderophore systems have been demonstrated to be critical virulence factors in several pathogens including *Escherichia coli* (6), *Erwinia chrysanthemi* (7), *Vibrio cholerae* (8), *Bacillus anthracis* (9), and *Pseudomonas aeruginosa* (10).

The best characterized enzymes for constructing siderophores are nonribosomal peptide synthetases (NRPS). These multimodular enzyme complexes utilize a thiotemplated mechanism to condense amino acid and carboxylate substrates into a myriad of chemically diverse iron chelators that include enterobactin, pyoverdine, vibriobactin, and mycobactin (11). A second system for siderophore biosynthesis, exemplified by the molecule aerobactin (12), utilizes NRPS-independent siderophore (NIS) enzymes to condense dicarboxylic acids with diamine or amino alcohol building blocks (13).

To date, *S. aureus* has been demonstrated to produce four siderophores: staphyloferrin A (14, 15), staphyloferrin B (16, 17), staphylobactin (18), and aureochelin (19). Of these, only the structures of staphyloferrin A and B have been elucidated. A gene cluster has been proposed for staphylobactin, and only a mass is known for aureochelin. Studies demonstrated that 33 of 34 tested staphylococci produced staphyloferrin A whereas only 17 of 34 strains produced staphyloferrin B (15), indicating that staphyloferrin A may be the more prevalent siderophore. Despite its importance, little is known concerning the biosynthesis of staphyloferrin A. Herein, we clone a gene cluster containing four genes, termed *sfnaA*, *sfnaB*, *sfnaC*, and *sfnaD*, that when expressed in the heterologous host *E. coli* confers the ability to biosynthesize staphyloferrin A. The 2 NIS synthases,

† This research was supported by the Novartis Institutes for BioMedical Research.
* Corresponding author. E-mail: carl.balibar@novartis.com. Phone: 617-871-5745. Fax: 617-871-5791.
¹ Abbreviations: MRSA, multidrug-resistant *Staphylococcus aureus*; NRPS, nonribosomal peptide synthetase; NIS, NRPS independent siderophore; SDS-PAGE, sodium dodecyl sulfate polyacrylamide gel electrophoresis; BSA, bovine serum albumin; CAS, chrome azurol S; HDTMA, hexadecyltrimethylamine; HPLC, high-performance liquid chromatography; LC/MS, liquid chromatography mass spectrometry; TBA, tetrabutylammonium; PCR, polymerase chain reaction; IPTG, isopropyl β-D-1-thiogalactopyranoside; PLP, pyridoxal-5′-phosphate; Fur, ferric uptake region.

10.1021/bi801844c CCC: $40.75 © 2009 American Chemical Society
Published on Web 01/12/2009

1026 Biochemistry, Vol. 48, No. 5, 2009

Cotton et al.

**FIGURE 1:** (A) Structure of staphyloferrin A. (B) Gene cluster coding for staphyloferrin A with 2 Fur boxes found between divergently transcribed SfnaD and SfnaA shown. (C) 4–15% SDS-PAGE gel of the two NIS synthetases from the *sfna* cluster. Ladder is Bio-Rad precision plus protein dual color prestained standard.

SfnaB and SfnaD, were purified and used to reconstitute the full biosynthetic route to staphyloferrin A in vitro, which allowed for elucidation of reaction order and biosynthetic intermediates.

---

### EXPERIMENTAL PROCEDURES

#### Materials and General Methods
Standard recombinant DNA, molecular cloning, and microbiological procedures were performed as described (20). Competent TOP10 and BL21 (DE3) *E. coli* strains were from Invitrogen. Oligonucleotide primers were from Integrated DNA Technologies. Phusion DNA polymerase, restriction enzymes, and T4 DNA ligase were from New England Biolabs. Plasmids pET28b and pETBlue-2 were from Novagen. DNA sequencing to verify PCR fidelity was performed on double-stranded DNA by the Agencourt Bioscience Corporation (Beverly, MA). Plasmid DNA preparation was performed using the Qiaprep kit from QIAGEN. Gel extraction of DNA fragments and restriction endonuclease cleanup were done using the Qiaquick kits from QIAGEN. TALON metal affinity resin was from Clontech Laboratories, Inc. SDS-PAGE gels were from BioRad (21). Protein samples were concentrated using a 10KMWCO Amicon Ultra device from Millipore and final protein concentrations were calculated using either the method of Bradford, with BSA as a standard. Sodium citrate, D-ornithine, ATP, 2,5-<sup>13</sup>C-citric acid, HEPES, Tris, kanamycin, ampicillin, chrome azurol S (CAS), hexadecyltrimethylamine (HDTMA), PIPES, and iron(III) chloride were from Sigma. Staphyloferrin A standard was obtained from the Novartis natural products unit (NPU, Basel, Switzerland).

Ion-pair HPLC was performed on an Agilent 1200 series system equipped with a Phenomenex Gemini C18 110A column (150 × 2.00 mm, 4 μm). Analytical LC/MS was performed on an Agilent 1200 series system equipped with a Phenomenex Jupiter Proteo 90A column (150 × 1.00 mm, 4 μm) in line with an Applied Biosystems 4000 Q Trap LC/MS/MS system. High resolution mass spectrometry was performed on a Waters LCT premier ESI-TOF mass spec-

**Cloning, Expression, and Purification of *sfna* Cluster and *SfnaB* and *SfnaD*.** All constructs were obtained from PCR amplification of genomic DNA from *S.* aureus RN4220. The following primers were used (underlined, restriction site): *sfna* cluster, 5′-AATCAATGGGATCCTTAATTATTTTCTCGATACAAAG-3′ and 5′-AATCAATGCGGCCGCTTAATAGTGGTTTACCATAGCAG-3′; *sfnaB*, 5′-AATCAATGCTAGCATGGTATATCTTGAAATGGGCAAAG-3′ and 5′-AATCAATCTCGAGACTTGTGAAC-3′; *sfnaD*, 5′-AATCAATGCTAGCATGAACCTTGAAC-3′ and 5′-AATCAATCTCGAGTTAATTTTTCTCGATACAAAG-3′. The PCR product from *sfna* was digested with BamHI and NotI and ligated into a similarly digested pETBlue-2 vector, yielding the plasmid psfna. The PCR products from *sfnaB* and *sfnaD* were digested with NheI and XhoI and ligated into similarly digested pET28b vector.

The described expression plasmids for *sfnaB* and *sfnaD* were transformed into *E. coli* BL21 (DE3) competent cells. Both constructs were grown in Luria-Bertani media supplemented with 50 μg/mL kanamycin at 30 °C until an OD<sub>600</sub> of 0.7, at which point the temperature was dropped to 20 °C and the cells were induced with 100 μM IPTG. Growth was allowed to continue for an additional 18 h. One liter of cells was harvested by centrifugation at 5000g, resuspended in 20 mL of buffer (20 mM Tris, pH 8.0, 500 mM NaCl) and lysed with two passes through a French press at 1100 psi. The lysate was cleared by centrifugation at 50000g and then incubated with 1 mL of TALON metal affinity resin for 1 h at 4 °C. The resin was then transferred to a column and protein was eluted with an imidazole buffer step gradient using steps of 25 mL of 0 mM imidazole, 15 mL of 5 mM imidazole, 10 mL of 25 mM imidazole, and 5 mL of 400 mM imidazole. After running an SDS-PAGE gel to confirm protein-containing fractions, the 25 mM and 400 mM imidazole fractions were combined and dialyzed twice against 20 mM Tris, pH 8.0, 50 mM NaCl, 2 mM MgCl<sub>2</sub>, and 10% glycerol. Protein was flash frozen in liquid nitrogen and stored at −80 °C.

**Siderophore Production Assays.** TOP10 cells transformed with psfna or pETBlue-2 were grown overnight in Luria-Bertani broth supplemented with 100 μg/mL ampicillin. Cells plated directly onto CAS minimal media plates were spotted in volumes of 2, 5, or 10 μL of overnight culture and allowed to dry. In the O-CAS assay, cells were streaked out onto LB plates and allowed to grow overnight at 37 °C until colonies developed. These plates were then overlaid with 20 mL of the O-CAS reagent. Time required for detection of positive color change for both the CAS assay and O-CAS assay varied between 12 and 72 h at room temperature. CAS plates were made as previously described (22, 23), and O-CAS media was made as previously described (24).

**Identification of Staphyloferrin A from *E. coli* Culture Media.** TOP10 cells transformed with psfna or pETBlue-2 were grown overnight in Luria-Bertani broth supplemented with 100 μg/mL ampicillin. Ten mL of culture was centrifuged at 9000 g and then formic acid was added to the supernatant at a final concentration of 2% (w/v). After

Staphyloferrin A Biosynthesis

**A**

| TOP10 pETBlue-2 | TOP10 psfna |
| --- | --- |
| 2 μL |  |
| 5 μL |  |
| 10 μL |  |

CAS

O-CAS

**B**

Intensity (cps * 10⁴)

staphyloferrin A  
psfna  
pETBlue-2  

Time (min)

FIGURE 2: (A) Siderophore production assay for TOP10 E. coli expressing either empty pETBlue-2 vector or the sfna gene cluster. On the left, 2, 5, or 10 μL of overnight culture are plated on CAS minimal media plates. On the right, cells are streaked onto LB and then overlaid with O-CAS reagent. Color change from blue to orange/yellow indicates siderophore production. (B) LC/ESI-MS analysis of culture extracts from TOP10 cells expressing either empty pETBlue-2 vector or psfna compared to staphyloferrin A standard. Product was monitored at m/z = 479 [M – H]⁻.

lyophilization the remaining solid was resuspended in 500 μL of 75/25 methanol/water prewarmed to 70 °C. The suspension was diluted 10-fold in 0.1% w/v formic acid, and 10 μL was injected on the LC/MS and run at 80 μL/min using a step gradient consisting of a 4 min hold at 0% B, a 4 min ramp to 15% B, and a 4 min ramp then 4 min hold at 90% B. Solvent A consisted of 0.2% formic acid in water and solvent B consisted of 0.2% formic acid in a 95/5% mixture of acetonitrile/water.

Enzymatic Reactions. In vitro enzymatic reactions of 100 μL total volume contained 1 mM Na-citrate, 1 mM D-ornithine, 5 mM ATP, 0.5 mM MgCl₂, 50 mM HEPES, pH 7.3, 5 μM SfnaB, and 5 μM SfnaD. The reaction occurred for 2 h at room temperature before being quenched. In reactions where protein was omitted, the corresponding volume was replaced with 20 mM Tris, pH 8.0. Reactions for turnover of the citryl-D-ornithine intermediate by SfnaB were performed as above, but D-ornithine was replaced with the intermediate and no SfnaD was included.

For ion-pair HPLC analysis, reactions were quenched with 200 μL of methanol, centrifuged at 21000g for 10 min to pellet protein, transferred to a new 1.7 mL tube, evaporated to the initial 100 μL volume in a speedvac, and supplemented

with 3 mM Fe³⁺Cl₃. Thirty microliters of this quenched reaction mixture was injected on the HPLC and run at 0.3 mL/min using a step gradient consisting of a 2 min hold at 15% B, a 1 min ramp then 4 min hold at 28% B, a 1 min ramp then 3.5 min hold at 34% B, a 1 min ramp then 3.5 min hold at 40% B, and a 1 min ramp then 3 min hold at 80% B. Solvent A consisted of 10 mM tetrabutylammonium-hydrogen sulfate in water, pH 7.3, and solvent B was acetonitrile. Peaks were monitored at 340 nm.

For LC/MS analysis, reactions were quenched with 1% formic acid and centrifuged at 21000g for 10 min to pellet protein. Ten microliters of a 1/10 dilution of the quenched reaction was injected on the LC/MS and run at 80 μL/min using a step gradient consisting of a 4 min hold at 0% B, a 4 min ramp to 15% B, and a 4 min ramp then 4 min hold at 90% B. Solvent A consisted of 0.2% formic acid in water, and solvent B consisted of 0.2% formic acid in a 95/5% mixture of acetonitrile/water.

Large scale purification of the citryl-D-ornithine intermediate was conducted in a total volume of 5.66 mL and contained 1 mM Na-citrate, 1 mM D-ornithine, 5 mM ATP, 0.5 mM MgCl₂, 50 mM HEPES, pH 7.3, 4.8 mM Tris, pH 8.0, and 5 μM SfnaD. After overnight incubation, the reaction

1028 Biochemistry, Vol. 48, No. 5, 2009

![Figure](https://i.imgur.com/yourimage.png)

FIGURE 3: Ion-pair HPLC analysis of reactions for formation of staphyloferrin A in vitro using 5 μM SfnaB, 5 μM SfnaD, 1 mM citrate, 1 mM D-ornithine, and 5 mM ATP. Components omitted from the reaction are indicated on the specified trace. Product was monitored at 340 nm.

was lyophilized to dryness, rediluted in 600 μL D₂O and analyzed by NMR.

**RESULTS**

*Cloning and Analysis of the sfna Gene Cluster.* Given its structural similarity to the siderophores aerobactin (12), rhizobactin (25), and vibrioferin (26), it seemed likely that staphyloferrin A would be assembled by NIS biosynthetic machinery. Bioinformatics performed using a tBLASTn search of the *S. aureus* NCTC8325 genome with IucA and IucC from the aerobactin gene cluster as a query revealed two clusters containing proteins with homology to NIS synthetases. The first was the *sbn* cluster previously identified as being responsible for staphylobactin biosynthesis (18). The second was a small cluster of four genes containing a PLP-dependent amino acid racemase and two NIS synthetases, enzymes which would be sufficient to biosynthesize the D-ornithine backbone and two citryl amides, respectively, that constitute staphyloferrin A (Figure 1A). Further implicating this cluster in siderophore biosynthesis was the presence of two ferric uptake regulator regions found upstream of the divergent promoters driving transcription of the cluster (Figure 1B) and its localization adjacent to three iron complex transporters. This cluster was tentatively named *sfna* for staphyloferrin A, and the genes labeled A through D.

In order to interrogate whether this cluster was responsible for siderophore production, we amplified it, cloned it into the high copy number pETBlue-2 vector, and transformed it into *E. coli* TOP10 cells. Since the genes within the cluster are transcribed divergently from centrally located promoters, cloning into pETBlue-2 did not place the cluster under control of the T7 promoter present on the plasmid, as the genes were cloned in reverse orientation relative to the promoter. Instead, transcription of the genes remained under the control of endogenous *S. aureus* promoters. Using the universal CAS assay for siderophore detection (22, 23), *E. coli* harboring psfna or the control pETBlue-2 vector were grown either directly on CAS minimal media plates or on LB plates followed by overlay with O-CAS media. Growth of bacteria on iron-deficient media often induces upregulation of siderophore biosynthetic genes, therefore direct growth on CAS minimal media should induce transcription of the *sfna* cluster. A slight confounding effect of this assay is that siderophores endogenous to the heterologous host *E. coli* could also be produced in response to iron restricted growth. To resolve this issue, growth was repeated on iron sufficient LB media, which should keep endogenous siderophores at minimal levels. However, because psfna is present at such high copy numbers, transcriptional regulation should be bypassed and siderophore production from psfna should still be high. The produced siderophore can then be detected by overlaying the plate with O-CAS media (24).

As demonstrated in Figure 2A, *E. coli* containing control pETBlue-2 vector form small yellow halos when grown on CAS minimal media plates, indicating the production of small amounts of endogenous siderophore that remove iron from the blue CAS-Fe³⁺-HDTMA complex. Noticeably, *E. coli* containing psfna form very large yellow halos when grown on the same plates, indicating the overproduction of siderophore not present in the control strain. Verifying this result, *E. coli* containing psfna again form yellow halos when grown on LB and overlaid with O-CAS, demonstrating siderophore production even in iron replete media, whereas *E. coli* containing pETBlue-2 control vector are deficient in siderophore production under the same conditions. These results imply that the *sfna* cluster is able to confer production of large amounts of a siderophore in the heterologous host *E. coli.*

To determine the identity of this siderophore we analyzed the culture supernatant of *E. coli* harboring psfnA. As can be seen in Figure 2B, culture supernatants from *E. coli* harboring psfnA, but not empty pETBlue-2 vector, contain a compound that coelutes and has the same mass as staphyloferrin A, indicating that the *sfna* gene cluster confers the ability to synthesize staphyloferrin A.

*Cloning and Expression of SfnaB and SfnaD.* Although the *sfna* gene cluster was demonstrated to be capable of

Staphyloferrin A Biosynthesis

**A**

| Intensity (Cps * 10⁴) |
| --- |
| 4500 |
| staphyloferrin A |
| Full Reaction |
| No SfnaB |
| No SfnaD |
| No D-ornithine |
| No Citrate |
| No ATP |
| No Enzyme |

| Time (min) |
| --- |
| 0 | 2 | 4 | 6 | 8 | 10 | 12 | 14 |

| Intensity (cps) |
| --- |
| 3.5*10⁶ |
| 3.0*10⁶ |
| 2.5*10⁶ |
| 2.0*10⁶ |
| 1.5*10⁶ |
| 1.0*10⁶ |
| 5.0*10⁵ |

| m/z |
| --- |
| 100 | 150 | 200 | 250 | 300 | 350 | 400 | 450 | 500 | 550 |

**B**

| Intensity (Cps * 10⁴) |
| --- |
| 2500 |
| Full Reaction |
| No SfnaB |
| No SfnaD |
| No D-ornithine |
| No Citrate |
| No ATP |
| No Enzyme |

| Time (min) |
| --- |
| 0 | 2 | 4 | 6 | 8 | 10 | 12 | 14 |

| Intensity (cps) |
| --- |
| 5.5*10⁶ |
| 5.0*10⁶ |
| 4.5*10⁶ |
| 4.0*10⁶ |
| 3.5*10⁶ |
| 3.0*10⁶ |
| 2.5*10⁶ |
| 2.0*10⁶ |
| 1.5*10⁶ |
| 1.0*10⁶ |
| 5.0*10⁵ |

| m/z |
| --- |
| 100 | 150 | 200 | 250 | 300 | 350 | 400 | 450 | 500 | 550 |

FIGURE 4: LC/ESI-MS analysis of reactions for formation of staphyloferrin A (A) or the citryl-D-ornithine intermediate (B) in vitro using 5 μM SfnaB, 5 μM SfnaD, 1 mM citrate, 1 mM D-ornithine, and 5 mM ATP. Components omitted from the reaction are indicated on the specified trace. Mass spectra of the major peak are shown to the left of the traces. Product was monitored at \( m/z = 479 \) [M – H]⁻ (A) or 307 [M + H]⁺ (B).

producing the siderophore staphyloferrin A, the order and logic of the biosynthetic steps was still unknown. To address this question, we chose to characterize this system in vitro. We focused our attention on the two NIS synthetases, SfnaB and SfnaD because the proposed PLP-dependent amino acid racemase, SfnaC, belongs to an enzyme family that has been rigorously characterized previously (27) and use of its proposed product, D-ornithine, should bypass the need for its activity. SfnaB and SfnaD were cloned as N-terminal His₆-tagged proteins using the vector pET28b and expressed in E. coli BL21 at 20 °C, yielding 25 mg/L of protein for each construct. The 67.2 kDa SfnaB and 76.1 kDa SfnaD proteins

1030 Biochemistry, Vol. 48, No. 5, 2009

Cotton et al.

A

| Intensity (cps * 10⁴) |
| --- |
| 3500 | Full Reaction |
| 3000 | No SfnaB |
| 2500 | No SfnaD |
| 2000 | No D-ornithine |
| 1500 | No Citrate |
| 1000 | No ATP |
| 500 | No Enzyme |
| 0 |  |

| Time (min) |
| --- |
| 0 | 2 | 4 | 6 | 8 | 10 | 12 | 14 |

Intensity (cps)

| m/z |
| --- |
| 131.0 |  |
|  | 506.0 | 537.5 |
|  |  |

B

| Intensity (cps * 10⁴) |
| --- |
| 2500 | Full Reaction |
| 2000 | No SfnaB |
| 1500 | No SfnaD |
| 1000 | No D-ornithine |
| 500 | No Citrate |
| 0 | No ATP |
|  | No Enzyme |

| Time (min) |
| --- |
| 0 | 2 | 4 | 6 | 8 | 10 | 12 | 14 |

Intensity (cps)

| m/z |
| --- |
| 102.3 |  |
| 121.4 |  |
| 133.3 |  |
| 153.3 |  |
| 179.2 |  |
| 217.2 |  |
| 239.4 |  |
|  | 309.5 |

FIGURE 5: LC/ESI-MS analysis of reactions for formation of <sup>13</sup>C isotope-labeled staphyloferrin A (A) or the citryl-D-ornithine intermediate (B) in vitro using 5 μM SfnaB, 5 μM SfnaD, 1 mM 2,5-<sup>13</sup>C-citric acid, 1 mM D-ornithine, and 5 mM ATP. Components omitted from the reaction are indicated on the specified trace. Mass spectra of the major peak are shown to the right of the traces. Product was monitored at *m/z* = 483 [M – H]<sup>-</sup> (A) or 309 [M + H]<sup>+</sup> (B).

were purified to homogeneity using metal affinity chromatography (Figure 1C).

Characterization of SfnaB and SfnaD. Characterization of previous NIS synthetases demonstrated that they catalyze the ATP-dependent activation of carboxylate substrates via an acyl-adenylate, an intermediate which is subsequently captured by an amine substrate yielding overall condensation to an amide (28–30). Given this mechanism of catalysis, we sought to determine whether incubation of citrate, D-ornithine, and ATP with SfnaB and SfnaD was sufficient

Staphyloferrin A Biosynthesis

**Scheme 1: Overall Pathway to Staphyloferrin A Biosynthesis**

![Chemical Reaction Diagram](https://i.imgur.com/chemical_reaction.png)

to yield staphyloferrin A. As can be seen in Figure 3, reactions containing all the aforementioned reagents led to the formation of a new peak at 11.8 min, when analyzed by ion-pair HPLC; a peak which coeluted with authentic staphyloferrin A standard. When either both enzymes, SfnaD alone, ATP, citrate, or D-ornithine was omitted, no reaction product was observed. Substitution of L-ornithine for D-ornithine did not support staphyloferrin A formation (data not shown). Interestingly, when only SfnaB was omitted from the reaction, no new peak formed, but those peaks which are attributed to citrate disappeared. This may imply that SfnaD consumes citrate in formation of an initial citryl-D-ornithine intermediate on route to staphyloferrin A formation. This intermediate may not be detectable by ion pairing HPLC because it still contains a free positively charged amine that either prohibits it from binding iron which is required for detection at 340 nm or interferes with its interaction with the positively charged TBA ion-pairing reagent. Indeed, free D-ornithine with its two free amines cannot be detected by this method.

In order to verify our results and identify whether an intermediate was forming with SfnaD, thus demonstrating an order to the reaction, we repeated the analysis using LC/ESI-MS. As can be seen in Figure 4A, reactions containing SfnaB, SfnaD, ATP, citrate, and ornithine produced a peak at 11.5 min that coeluted with authentic staphyloferrin A when detecting at an \( m/z = 479 \) in negative mode, consistent with an \([M - H]^-\) assignment. High resolution mass spectrometry indicated that the product had a molecular formula consistent with C₁₇H₂₄N₂O₁₄, the same as staphyloferrin A (479.1153 observed, 479.1150 calculated, \([M - H]^-\)). When any of the reaction components were omitted, or when L-ornithine was substituted for D-ornithine (data not shown), no staphyloferrin A formed. To detect a possible citryl-D-ornithine intermediate, the reactions were analyzed in positive mode at an \( m/z = 307 \), consistent with an \([M + H]^+\) assignment. As can be seen in Figure 4B, such an intermediate is detectable at 2.1 min in both the full reaction and reactions containing only SfnaD. High resolution mass spectrometry indicated that the intermediate had a molecular formula consistent with C₁₁H₁₈N₂O₈, which is in line with a citryl-D-ornithine intermediate (307.1144 observed, 307.1141 calculated, \([M + H]^+\)). Again, this intermediate is only formed when citrate, ATP, and D-ornithine are present, and is not dependent on the presence of SfnaB.

Although formation of both staphyloferrin A and the citryl-D-ornithine intermediate simultaneously tracked with disappearance of citrate and D-ornithine, as determined by monitoring the reaction in negative mode at \( m/z = 191 \) (citrate: \([M - H]^-\)) or in positive mode at \( m/z = 133 \) (D-ornithine: \([M + H]^+\)) (data not shown), we wanted to corroborate that the newly observed peaks were in fact representing the condensation of citrate with D-ornithine. Therefore, all the reactions were repeated using 2,5-¹³C-citric acid. As can be seen in Figure 5, use of this doubly isotope labeled citrate resulted in a corresponding 2 Da mass shift in the citryl-D-ornithine intermediate and a 4 Da mass shift in staphyloferrin A, which incorporates 2 molecules of citrate. High resolution mass spectrometry also indicated a 2 Da mass shift in the citryl-D-ornithine intermediate (309.1213 observed, 309.1208 calculated, \([M + H]^+\)) and a 4 Da mass shift in staphyloferrin A (483.1295 observed, 483.1284 calculated, \([M - H]^-)\). These reactions confirmed that SfnaB and SfnaD are responsible for staphyloferrin A biosynthesis and perform catalysis in an ordered manner; SfnaD catalyzes the ATP-dependent condensation of a citryl-D-ornithine intermediate which is then converted to mature staphyloferrin A by SfnaB.

**Characterization of the Citryl-D-ornithine Intermediate.** Although SfnaD was shown to catalyze the first condensation between citrate and D-ornithine, it was unknown whether the α-amine or δ-amine of D-ornithine was the initial nucleophile. To address this, a large scale preparative reaction containing only SfnaD was used to obtain several milligrams of the citryl-D-ornithine intermediate, which was subsequently analyzed by a 2D homonuclear COSY NMR experiment. When the spectrum for D-ornithine was compared to the spectrum for the citryl-D-ornithine intermediate, it became apparent that there was a 0.2 ppm chemical shift difference for protons H4 and H5, whereas the chemical shifts for protons H2 and H3 remained essentially unchanged (Figure 6A). To confirm that the intermediate is the N5-citryl isomer, a 2D heteronuclear HMBC NMR experiment was performed. As can be seen in Figure 6B, a new correlation between H5 and a far downfield carbon is present in the intermediate that is not present in D-ornithine, and no new correlations are observed for H2. Figure 6C summarizes all key correlations observed in the COSY and HMBC spectra. Taken together, these data indicate that the terminal end of the D-ornithine side chain has undergone chemical modification and that the δ-amine is the first nucleophile to participate in amide bond formation with citrate catalyzed by SfnaD.

With the intermediate structure confirmed, it was important to demonstrate that this intermediate is in fact on pathway to staphyloferrin A formation and is utilized as a substrate by SfnaB. As can be seen in Figure 6D, SfnaB converts δ-citryl-D-ornithine to staphyloferrin A in a reaction which requires both citrate and ATP. Thus SfnaB, like SfnaD, catalyzes an ATP-dependent condensation reaction between citrate and a free amine.

1032 Biochemistry, Vol. 48, No. 5, 2009 Cotton et al.

A

H2  
H5  

H3  
H4  

H4  
H3  

H5  

H2  

B  

H2  
H5  

C4  
C3  

C1  

C  

HN  
NH₂  
OH  

HO  
O  
OH  

O  
OH  

D  

Intensity (cps * 10⁴)  
Full Reaction  
No Citrate  
No ATP  

Time (min)

FIGURE 6: (A) 2D homonuclear proton COSY NMR spectrum of the δ-citryl-D-ornithine reaction intermediate (red) overlaid on D-ornithine (blue). The large shift in the ${}^{3}J$ coupling between H4 and H5 is boxed in green. Spectra were acquired in D₂O on a 400 MHz Bruker instrument equipped with a cryo DUAL probe. (B) 2D heteronuclear HMBC NMR spectrum of the δ-citryl-D-ornithine reaction intermediate (black) overlaid on D-ornithine (blue). The additional correlation to H5 in the intermediate is boxed in green. (C) Structure of the δ-citryl-D-ornithine intermediate with observed COSY correlations in bold and key HMBC proton carbon correlations denoted with arrows. (D) LC/ESI-MS analysis of reactions for formation of staphyloferrin A from the δ-citryl-D-ornithine intermediate *in vitro* using 5 μM Sfnab, 1 mM citrate, 1 mM δ-citryl-D-ornithine, and 5 mM ATP. Components omitted from the reaction are indicated on the specified trace. Product was monitored at $m/z = 479 [M - H]^{-}$.

Staphyloferrin A Biosynthesis

DISCUSSION

The acquisition of iron is essential for growth of most microorganisms and is paramount to pathogenesis during vertebrate infection. Bacteria have evolved several mechanisms to overcome iron restriction, one of the most prominent involving the biosynthesis, excretion, and uptake of siderophores which compete with host iron binding proteins. Despite their critical role in virulence (31), little is known about the siderophores of *S. aureus*. Only a single molecular-genetic study has been conducted linking genes to the production of a siderophore in *S. aureus* (18). In that study, a nine gene operon containing genes *sbnA-I* was demonstrated to be responsible for the biosynthesis of staphylobactin, a siderophore for which no structure is known. As with siderophores in other microorganisms, production of staphylobactin was demonstrated to play a critical role in virulence, as an *sbnE* mutant of *S. aureus* was shown to be severely attenuated in a murine kidney abscess model.

Despite these data, several questions remain concerning siderophore biosynthesis in *S. aureus*. Most importantly, homologous collinear clusters containing genes *sbnA-sbnH* found in *Rolstonia solanacearum* and *Rolstonia metallidurans* were demonstrated to be responsible for the biosynthesis of staphyloferrin B (32, 33), a polycarboxylate siderophore produced by several species of staphylococci (16, 17). This similarity in biosynthetic enzymes yet difference in final siderophore produced remain to be reconciled, as a cluster coding for staphyloferrin B in *S. aureus* has not been identified. Like staphyloferrin B, a structure for the polycarboxylate siderophore staphyloferrin A is known (14, 15), but no gene cluster has been linked to its production. Finally, the fourth known siderophore produced by *S. aureus*, aureochelin, has neither a structure nor a gene cluster associated with it (19).

Herein we describe the first complete elucidation of siderophore production in *S. aureus* from identification of genes to final biosynthetic product. Staphyloferrin A, composed of two citrate molecules condensed via amides to D-ornithine, was likely to be synthesized by a NIS biosynthetic pathway because of its dicarboxylate/diamine structure. A bioinformatics search of the *S. aureus* genome revealed a cluster of four genes coding for two putative NIS synthetases, a transporter, and a PLP-dependent racemase, all genes which could be predicted to be involved in staphyloferrin A biosynthesis. This gene cluster was found to contain a putative ferric uptake regulator (Fur) box upstream of the divergent promoters for the genes *sfnaA* and *sfnaD* and was located adjacent to a cluster coding for iron complex transporters, further implying a connection to siderophore production. When this cluster was cloned and overexpressed in *E. coli* it conferred the ability to synthesize large amounts of a siderophore not made by control cells as determined by the universal CAS assay.

In order to elucidate the product of this gene cluster, we cloned, expressed, and purified the two encoded NIS synthetases, SfnAB and SfnAD. If this was indeed the cluster for staphyloferrin A biosynthesis, we could bypass the need for the predicted PLP-dependent amino acid racemase, SfnAC, by supplying the presumed product D-ornithine. Using both SfnAB and SfnAD, we were able to reconstitute staphyloferrin A biosynthesis *in vitro* using the substrates

FIGURE 7: Phylogenetic analysis of various NIS synthetases. In addition to SfnAB and SfnAD, ten enzymes from each type, as categorized previously (13), were used for analysis. SfnAB and SfnAD clearly cluster with type B NIS synthetases. Clustering was performed in the Vector NTI program using the neighbor-joining algorithm of Saitou and Nei (36).

citrate, D-ornithine and ATP. Product was observed and confirmed by coelution with standard staphyloferrin A by both ion pairing and reversed phase liquid chromatography. The reaction was shown to require ATP, was catalyzed in the defined order SfnAD followed by SfnAB, and formed the citryl-D-ornithine intermediate through an amide bond at the δ-amine (Scheme 1).

A categorization of NIS synthetases has been devised on the basis of phylogenetic analysis which divided the NIS superfamily into three groups termed type A, B, and C (13). With limited information and proposed catalytic functions, two models were devised to explain the types of NIS synthetases. In the first model, the type of NIS enzyme is dependent on the substrate activated with type A enzymes catalyzing activation of citrate, type B enzymes catalyzing activation of α-ketoglutarate, and type C enzymes catalyzing activation of monoamide or monoester derivatives of citrate or succinate. In the second model, the NIS synthetase exhibits a preference for either amine or alcohol nucleophiles to capture their activated substrate, with type A enzymes catalyzing amide bond formation between various dicarboxylate substrates and an amine nucleophile, type B enzymes catalyzing ester bond formation between citrate and an alcohol nucleophile, and type C enzymes catalyzing amide bond formation between monoamide or monoester derivatives of citrate or succinate and an amine nucleophile. There is limited *in vitro* experimental evidence confirming this bioinformatics analysis. Recently AsbA was demonstrated to be a type A NIS synthetase catalyzing condensation between citrate and spermidine in petrobactin biosynthesis (30), DesD was demonstrated to be a type C NIS synthetase catalyzing trimerization and macrocyclization of N-hydroxy-N-succinylcadaverine in desferrioxamine E biosynthesis (29), PubC was demonstrated to be a type C NIS synthetase

catalyzing dimerization and macrocyclization of N-hydroxy-N-succinyl-putrescine in putrebactin biosynthesis (28), AsbB was demonstrated to be a type C NIS synthetase catalyzing condensation of spermidine and N8-citryl-spermidine or N1(3,4-dihydroxybenzoyl)-N8-citryl-spermidine in petrobactin biosynthesis (34), and BibC was demonstrated to contain an adenylation domain capable of catalyzing cyclodimerization of N-hydroxy-N-succinylcadaverine in bisucaberin biosynthesis (35).

According to the proposed models, both SfnaD and SfnaB would likely be classified as type A NIS synthetases based on their activities. SfnaD activates citrate via an acyladenylate intermediate which is subsequently captured by specific nucleophilic attack of the δ-amine of D-ornithine. The remaining α-amine of this citryl-D-ornithine intermediate then performs nucleophilic attack on a second citrate-adenylate produced by SfnaB to complete staphyloferrin A biosynthesis. However, despite their A type-like chemistry of activating citrate, neither clusters with the A type NIS synthetases. Instead they cluster with the B type NIS synthetases, which given their activities renders both proposed models inadequate (Figure 7). Interestingly, there appears to be two distinct clades within the type B NIS synthetases. The lower clade contains AcsA, SbnC, and PvsB, all of which are from pathways that assemble α-ketoglutarate containing siderophores and are consistent with the proposed models. The upper clade contains SfnaD and SfnaB, which have A type-like specificity and do not conform to the models. Perhaps this group represents a novel subtype of NIS synthetase, a B<sub>A</sub> type, that cluster with type B NIS synthetases, perform A type-like chemistry, but are specific for D-amino acid substrates and so are separate from other A type NIS synthetases. This demonstrates that prediction of NIS synthetase activity is likely more complex than previously thought, as SfnaD and SfnaB represent the first biochemically characterized members of type B NIS synthetases.

By definitively linking a gene cluster to a known siderophore, this study sets the stage for future *in vivo* experimentation elucidating the importance of staphyloferrin A production to staphylococcal pathogenicity. With four known siderophores, and possibly others yet to be discovered, it is essential to understand the importance each plays in the iron-restricted growth of *S. aureus in vivo* and *in vitro*. Furthermore, SfnaB and SfnaD represent the first members of type B NIS synthetases to be biochemical characterized *in vitro*, adding understanding to the one A type and several C type NIS synthetases previously studied.

## ACKNOWLEDGMENT

We thank Gregg Chenail for help with HPLC and LC/MS analysis, Bing Wang and Karl Gunderson for help with acquiring NMR spectra, Guangxiang Wu and David Jiang for help with acquiring high resolution mass spectra, and Neil S. Ryder for a careful reading of the manuscript.

## REFERENCES

1. Archer, G. L. (1998) *Staphylococcus aureus*: a well-armed pathogen. *Clin. Infect. Dis.* 26, 1179–1181.
2. Klevens, R. M., Morrison, M. A., Fridkin, S. K., Reingold, A., Petit, S., Gershman, K., Ray, S., Harrison, L. H., Lynfield, R., Dumyati, G., Townes, J. M., Craig, A. S., Fosheim, G., McDougal, L. K., and Tenover, F. C. (2006) Community-associated methicillin-resistant *Staphylococcus aureus* and healthcare risk factors. *Emerg. Infect. Dis.* 12, 1991–1993.
3. Klevens, R. M., Morrison, M. A., Nadle, J., Petit, S., Gershman, K., Ray, S., Harrison, L. H., Lynfield, R., Dumyati, G., Townes, J. M., Craig, A. S., Zell, E. R., Fosheim, G. E., McDougal, L. K., Carey, R. B., and Fridkin, S. K. (2007) Invasive methicillin-resistant *Staphylococcus aureus* infections in the United States. *JAMA* 298, 1763–1771.
4. Weinberg, E. D. (1999) Iron loading and disease surveillance. *Emerg. Infect. Dis.* 5, 346–352.
5. Schaible, U. E., and Kaufmann, S. H. (2004) Iron and microbial infection. *Nat. Rev. Microbiol.* 2, 946–953.
6. Williams, P. H. (1979) Novel iron uptake system specified by ColV plasmids: an important component in the virulence of invasive strains of *Escherichia coli*. *Infect. Immun.* 26, 925–932.
7. Enard, C., Diolez, A., and Expert, D. (1988) Systemic virulence of *Erwinia chrysanthemi* 3937 requires a functional iron assimilation system. *J. Bacteriol.* 170, 2419–2426.
8. Henderson, D. P., and Payne, S. M. (1994) *Vibrio cholerae* iron transport systems: roles of heme and siderophore iron transport in virulence and identification of a gene associated with multiple iron transport systems. *Infect. Immun.* 62, 5120–5125.
9. Cendrowski, S., MacArthur, W., and Hanna, P. (2004) *Bacillus anthracis* requires siderophore biosynthesis for growth in macrophages and mouse virulence. *Mol. Microbiol.* 51, 407–417.
10. Meyer, J. M., Neely, A., Stintzi, A., Georges, C., and Holder, I. A. (1996) Pyoverdin is essential for virulence of *Pseudomonas aeruginosa*. *Infect. Immun.* 64, 518–523.
11. Crosa, J. H., and Walsh, C. T. (2002) Genetics and assembly line enzymology of siderophore biosynthesis in bacteria. *Microbiol. Mol. Biol. Rev.* 66, 223–249.
12. de Lorenzo, V., and Neilands, J. B. (1986) Characterization of iucA and iucC genes of the aerobactin system of plasmid ColV-K30 in *Escherichia coli*. *J. Bacteriol.* 167, 350–355.
13. Challis, G. L. (2005) A widely distributed bacterial pathway for siderophore biosynthesis independent of nonribosomal peptide synthetases. *ChemBioChem* 6, 601–611.
14. Konetschny-Rapp, S., Jung, G., Meiwes, J., and Zahner, H. (1990) Staphyloferrin A: a structurally new siderophore from staphylococci. *Eur. J. Biochem.* 191, 65–74.
15. Meiwes, J., Fiedler, H. P., Haag, H., Zahner, H., Konetschny-Rapp, S., and Jung, G. (1990) Isolation and characterization of staphyloferrin A, a compound with siderophore activity from *Staphylococcus hyicus* DSM 20459. *FEMS Microbiol. Lett.* 55, 201–205.
16. Drechsel, H., Freund, S., Nicholson, G., Haag, H., Jung, O., Zahner, H., and Jung, G. (1993) Purification and chemical characterization of staphyloferrin B, a hydrophilic siderophore from staphylococci. *Biometals* 6, 185–192.
17. Haag, H., Fiedler, H. P., Meiwes, J., Drechsel, H., Jung, G., and Zahner, H. (1994) Isolation and biological characterization of staphyloferrin B, a compound with siderophore activity from staphylococci. *FEMS Microbiol. Lett.* 115, 125–130.
18. Dale, S. E., Doherty-Kirby, A., Lajoie, G., and Heinrichs, D. E. (2004) Role of siderophore biosynthesis in virulence of *Staphylococcus aureus*: identification and characterization of genes involved in production of a siderophore. *Infect. Immun.* 72, 29–37.
19. Courcol, R. J., Trivier, D., Bissinger, M. C., Martin, G. R., and Brown, M. R. (1997) Siderophore production by *Staphylococcus aureus* and identification of iron-regulated proteins. *Infect. Immun.* 65, 1944–1948.
20. Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) *Molecular cloning: a laboratory manual*, 2nd ed., Cold Spring Harbor Press, Plainview, NY.
21. Bradford, M. M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal. Biochem.* 72, 248–254.
22. Payne, S. M. (1994) Detection, isolation, and characterization of siderophores. *Methods Enzymol.* 235, 329–344.
23. Schwyn, B., and Neilands, J. B. (1987) Universal chemical assay for the detection and determination of siderophores. *Anal. Biochem.* 160, 47–56.
24. Perez-Miranda, S., Cabirol, N., George-Tellez, R., Zamudio-Rivera, L. S., and Fernandez, F. J. (2007) O-CAS, a fast and universal method for siderophore detection. *J. Microbiol. Methods* 70, 127–131.
25. Lynch, D., O’Brien, J., Welch, T., Clarke, P., Cuiv, P. O., Crosa, J. H., and O’Connell, M. (2001) Genetic organization of the region

encoding regulation, biosynthesis, and transport of rhizobactin 1021, a siderophore produced by *Sinorhizobium meliloti*. *J. Bacteriol.* 183, 2576–2585.

26. Tanabe, T., Funahashi, T., Nakao, H., Miyoshi, S., Shinoda, S., and Yamamoto, S. (2003) Identification and characterization of genes required for biosynthesis and transport of the siderophore vibrioferrin in *Vibrio parahaemolyticus*. *J. Bacteriol.* 185, 6938–6949.

27. Amadasi, A., Bertoldi, M., Contestabile, R., Bettati, S., Cellini, B., di Salvo, M. L., Borri-Voltattorni, C., Bossa, F., and Mozzarelli, A. (2007) Pyridoxal 5′-phosphate enzymes as targets for therapeutic agents. *Curr. Med. Chem.* 14, 1291–1324.

28. Kadi, N., Arbache, S., Song, L., Oves-Costales, D., and Challis, G. L. (2008) Identification of a gene cluster that directs putrebactin biosynthesis in *Shewanella* species: PubC catalyzes cyclodimerization of N-hydroxy-N-succinylputrescine. *J. Am. Chem. Soc.* 130, 10458–10459.

29. Kadi, N., Oves-Costales, D., Barona-Gomez, F., and Challis, G. L. (2007) A new family of ATP-dependent oligomerization-macrocyclization biocatalysts. *Nat. Chem. Biol.* 3, 652–656.

30. Oves-Costales, D., Kadi, N., Fogg, M. J., Song, L., Wilson, K. S., and Challis, G. L. (2007) Enzymatic logic of anthrax stealth siderophore biosynthesis: AsbA catalyzes ATP-dependent condensation of citric acid and spermidine. *J. Am. Chem. Soc.* 129, 8416–8417.

31. Rozakska, B., Lisiecki, P., Sadowska, B., Mikucki, J., and Rudnicka, W. (1998) The virulence of *Staphylococcus aureus* isolates differing by siderophore production. *Acta Microbiol. Pol.* 47, 185–194.

32. Bhatt, G., and Denny, T. P. (2004) *Ralstonia solanacearum* iron scavenging by the siderophore staphyloferrin B is controlled by PhcA, the global virulence regulator. *J. Bacteriol.* 186, 7896–7904.

33. Gilis, A., Khan, M. A., Cornelis, P., Meyer, J. M., Mergeay, M., and van der Lelie, D. (1996) Siderophore-mediated iron uptake in *Alcaligenes eutrophus* CH34 and identification of aleB encoding the ferric iron-alcaligin E receptor. *J. Bacteriol.* 178, 5499–5507.

34. Oves-Costales, D., Kadi, N., Fogg, M. J., Song, L., Wilson, K. S., and Challis, G. L. (2008) Petrobactin biosynthesis: AsbB catalyzes condensation of spermidine with N8-citrlyl-spermidine and its N1-(3,4-dihydroxybenzoyl) derivative. *Chem. Commun.* 4034–4036.

35. Kadi, N., Song, L., and Challis, G. L. (2008) Bisucaberin biosynthesis: an adenylation domain of the BibC multi-enzyme catalyzes cyclodimerization of N-hydroxy-N-succinylcadaverine. *Chem. Commun.* 5119–5121.

36. Saitou, N., and Nei, M. (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. *Mol. Biol. Evol.* 4, 406–425.

BI801844C
