
FOCUS ON CD4⁺ T-CELL DIVERSITY

REVIEWS

Epigenetic control of T-helper-cell differentiation

Christopher B. Wilson, Emily Rowell and Masayuki Sekimata

Abstract | Naive CD4⁺ T cells give rise to T-helper-cell subsets with functions that are tailored to their respective roles in host defence. The specification of T-helper-cell subsets is controlled by networks of lineage-specifying transcription factors, which bind to regulatory elements in genes that encode cytokines and other transcription factors. The nuclear context in which these transcription factors act is affected by epigenetic processes, which allow programmes of gene expression to be inherited by progeny cells that at the same time retain the potential for change in response to altered environmental signals. In this Review, we describe these epigenetic processes and discuss how they collaborate to govern the fate and function of T helper cells.

---

**Epigenetic process**

A process that affects gene expression without altering the sequences of bases in the DNA. Epigenetic changes are potentially heritable in the absence of the factors that initially induced them, and some propose that this term be restricted to those that are demonstrably heritable (although the broader definition is used here). In mammals, epigenetic processes that affect gene transcription include methylation of cytosines in CpG dinucleotides, post-translational histone modifications and changes to higher-order chromatin structure.

**Chromatin**

DNA and the proteins with which it is associated in the nucleus.

Department of Immunology,  
University of Washington,  
Seattle, Washington 98195,  
USA.  
Correspondence to C.B.W.  
e-mail: cbwilson@u.washington.edu  
doi:10.1038/nri2487  
Published online 19 January 2009  

CD4⁺ T helper (Tₕ)-cell subsets were first described by Mossman and Coffman¹, who found that mouse T-cell clones segregated into two subsets they named Tₕ1 and Tₕ2 cells based on their mutually exclusive production of interferon-γ (IFNγ) or interleukin-4 (IL-4), IL-5 and IL-13, respectively. The relevance of this distinction was subsequently shown by Locksley and colleagues², who found that mice which mounted a Tₕ1-cell-dominant response resolved infection with Leishmania major, whereas mice that mounted a Tₕ2-cell-dominant response did not. Work by many groups has since established the importance of Tₕ1 cells in host defence to intracellular pathogens and of Tₕ2 cells in the protection against helminth infections. More recently, cells that produce IL-17A were shown to represent a distinct Tₕ-cell lineage — the Tₕ17-cell subset — that in addition to IL-17A produces IL-17F, IL-21, IL-22 and, in humans, IL-26. Tₕ17 cells contribute to host defence against extracellular bacteria and fungi, particularly at mucosal sites, although the full extent of their contribution to host defence is still being investigated³. In contrast to these protective functions of Tₕ cells, inappropriate Tₕ2-cell responses give rise to allergic diseases, whereas autoimmune diseases result from inappropriate Tₕ1- and Tₕ17-cell responses³⁻⁶.

For the development of distinct Tₕ-cell lineages, the instructions that are received by naive CD4⁺ T cells during initial encounters with antigen-presenting cells (APCs) must be converted into cell-intrinsic changes. These changes are passed on to progeny cells through multiple cell divisions and occur over time and in various environments where effector and memory T cells carry out their functions. These instructions are converted by responding T cells into changes in the abundance, interactions and locations of transcription factors, which in turn lead to changes in gene expression. The resulting information could in principle be propagated from one T-cell generation to the next solely through self-reinforcing transcription factor networks. In practice, more precise control of gene expression is achieved through epigenetic processes⁷, which facilitate heritable and stable programmes of gene expression, while preserving the potential for these programmes to be modified in response to environmental changes.

In this Review, we describe the epigenetic processes that help to regulate Tₕ1-, Tₕ2- and Tₕ17-cell lineage fate and function by affecting gene transcription. In recent years, technological advances have accelerated the pace of discovery in this field. As a result, progressively more comprehensive and higher resolution maps of gene regulatory elements and their cell-type-specific epigenetic marks, including DNA methylation, histone modifications and three-dimensional chromatin structure, are being derived from stem cells and other cell types, including T cells. These findings and their contribution to Tₕ-cell differentiation and function are also discussed in this Review. Epigenetic processes that influence mRNA splicing, stability and translation in the immune system, such as microRNAs, have been reviewed recently⁸⁻¹⁰ and are therefore not discussed here. Follicular Tₕ cells¹¹ and other proposed Tₕ-cell subsets that might represent distinct lineages but are not yet firmly established are also not discussed in this Review.

Regulating lineage choice

**Transcription factor networks**. Nuclear factor of activated T cells (NFAT) and other transcription factors that are activated in naive CD4⁺ T cells in response to signals through the T-cell receptor (TCR) and co-stimulatory molecules induce the production of IL-2, which leads to IL-2-induced activation of signal transducer and activator of transcription 5 (STAT5) and entry into the cell cycle. Thereafter, T<sub>H</sub>-cell-lineage choice is determined, for the most part, by the cytokine milieu and the network of transcription factors that are induced downstream of cytokine signalling pathways (FIG. 1).

T<sub>H</sub>1-cell development is initiated by STAT1, which is activated in response to IFNγ and IL-27 that are produced by natural killer (NK) cells and APCs, respectively. Together with TCR-induced transcription factors, STAT1 induces the transcription factor T-bet, a key (if not ‘master’) regulator of the T<sub>H</sub>1-cell lineage, which in turn induces the production of IFNγ, the activation of the transcription factors H2.0-like homeobox (HLX) and runt-related transcription factor 3 (RUNX3), and opposes the inhibitory effects of GATA-binding protein 3 (GATA3; see later) on T<sub>H</sub>1-cell differentiation¹²⁻¹⁷. The expression of IL-12 receptor-β2 (IL-12Rβ2) is also induced in this process; IL-12Rβ2 pairs with IL-12Rβ1 to form the IL-12 receptor, thereby allowing APC-derived IL-12 to activate STAT4. STAT4, T-bet, HLX and RUNX3 then bind to and activate *Ifng*, which reinforces T<sub>H</sub>1-cell commitment through the activation of STAT1 in a positive-feedback loop. Concomitantly, T-bet and RUNX3 bind to and repress *Il4* to inhibit T<sub>H</sub>2-cell differentiation.

T<sub>H</sub>2-cell differentiation is initiated by the activation of STAT6 by IL-4, which, together with TCR-induced transcription factors, binds to and activates *Gata3* (REF. 5). Alternatively, Notch signalling can induce *Gata3* in a STAT6-independent manner¹⁸,¹⁹. GATA3 induces the transcription factor MAF, which helps to activate *Il4*, and together GATA3 and STAT6 activate the transcription of *Il4, Il5* and *Il13*. T<sub>H</sub>2-lineage commitment is stabilized by the autoactivation of GATA3, the autocrine and paracrine activation of STAT6 by IL-4, and the STAT6- and GATA3-dependent antagonism of IFNγ expression and T<sub>H</sub>1-cell differentiation⁵.

The transcription factors that are involved in T<sub>H</sub>1- and T<sub>H</sub>2-cell differentiation are not required for T<sub>H</sub>17-cell differentiation and can antagonize this process²⁰,²¹. Transforming growth factor-β (TGFβ) inhibits T<sub>H</sub>1- and T<sub>H</sub>2-cell differentiation, and promotes regulatory T (T<sub>Reg</sub>)-cell and T<sub>H</sub>17-cell lineage commitment by inducing the expression of the transcription factors forkhead box P3 (FOXP3) and retinoic-acid-receptor-related orphan receptor-γt (RORγt; also known as RORC), which are required for T<sub>Reg</sub>- and T<sub>H</sub>17-cell lineage commitment, respectively²²⁻²³. In the absence of IL-6, FOXP3 inhibits RORγt and therefore blocks T<sub>H</sub>17-cell development, whereas in the presence of IL-6, STAT3 is activated, which inhibits the expression of FOXP3 and its interactions with RORγt. This results in an increase in the expression of RORγt (as well as of RORα, which has an ancillary role in T<sub>H</sub>17-cell induction)²⁴, and T<sub>H</sub>17-cell differentiation is favoured. Once T<sub>H</sub>17-cell differentiation has been initiated, the T cells produce IL-21, which activates STAT3 and induces the expression of IL-23R. This allows APC-derived IL-23 to activate STAT3, which dampens IL-10 production, drives IL-22 production and stabilizes T<sub>H</sub>17-cell differentiation and commitment²¹,²⁵⁻²⁸. The transcription factor aryl-hydrocarbon receptor (AHR) also influences the differentiation of T<sub>Reg</sub> and T<sub>H</sub>17 cells²⁹,³⁰. Although initial studies suggested otherwise, the cytokine and transcription factor networks involved in human and mouse T<sub>H</sub>17-cell differentiation seem to be similar²¹,³¹,³².

**Epigenetic processes**. The ability of transcription factors to bind to DNA at regulatory regions on genes is affected by their concentration, post-translational modifications and subcellular localization, as well as by the state of the chromatin and underlying DNA. The epigenetic context in which transcription factors function is provided by the position and compaction of nucleosomes, the interactions of nucleosomes with the DNA, post-translational histone modifications and the methylation status of the DNA⁶,³³⁻³⁶. Therefore, unlike genetic information, epigenetic information is not encoded by changes in the sequence of the DNA but by differential methylation of the DNA and modifications of chromatin, which affect whether, when and to what level specific genes are expressed in a given cell. Because the DNA sequence remains unchanged, epigenetic modifications and the information that they encode can be heritable but plastic — the potential to erase modifications and inscribe new ones is retained.

In mammals, DNA can be methylated on cytosines in CpG dinucleotides. At present, DNA methylation is the only proven mechanism by which epigenetic information is faithfully propagated from one cell generation to the next. Heritability is achieved through copying of the pattern of methylated cytosines from parental to progeny DNA strands by DNA methyltransferase 1 (DNMT1)³⁷. DNA methylation at gene promoters, and possibly at distal regulatory elements, can directly inhibit transcription³⁸,³⁹; by contrast, DNA methylation within transcribed sequences seems to have little effect on transcription, although demethylation within the transcribed sequences of *Il4* and *Ifng* correlates with high expression levels of these cytokines in T<sub>H</sub>2 and T<sub>H</sub>1 cells, respectively¹³,⁴⁰. Methylated DNA directly represses gene expression by blocking the binding of some transcription factors to the promoter and other regulatory elements, thereby inhibiting the recruitment of RNA polymerase II, and indirectly by providing docking sites for methyl-CpG-binding domain proteins (MBDs)⁴¹,⁴². Although plants contain enzymes that actively demethylate DNA, such enzymes have not been identified in mammals, in which the only established mechanism for DNA demethylation is passive — that is, the result of a failure to copy methylation patterns from the parental strand onto the daughter strand during DNA replication.

Nucleosome composition and histone modifications are diverse and dynamic. Variant forms of histones H2 and H3 and the linker histone H1 can be incorporated

Notch  
A transmembrane receptor that is involved in the pathway for direct cell–cell signalling through its association with a transmembrane ligand of the Delta or Jagged family on a neighbouring cell. The large intracellular domain of Notch is cleaved and travels to the nucleus to become a direct co-activator of the transcription factor recombinant-signal-binding protein for immunoglobulin-κ J region (RBPJ).

Nucleosome  
The basic structural subunit of chromatin, which consists of ~156 base pairs of DNA wrapped around an octamer of histones.

FOCUS ON CD4⁺ T-CELL DIVERSITY

### Figure 1 | Cytokines and transcription factor networks regulate T-helper-cell differentiation.

Activation-induced division of naive CD4⁺ T cells provides a context that allows their differentiation into one of several T helper (Tₕ)-cell lineages.

#### a | Tₕ₁-cell development
- **Initiation; proliferation**
- **Differentiation; stabilization**
- **Epigenetic remodelling**

#### b | Tₕ₂-cell development
- **Heritability**

#### c | Tₕ₁₇-cell development
- **Heritability**

**Figure 1 | Cytokines and transcription factor networks regulate T-helper-cell differentiation.** Activation-induced division of naive CD4⁺ T cells provides a context that allows their differentiation into one of several T helper (Tₕ)-cell lineages.
- **a | Tₕ₁-cell differentiation** is initiated by the activation of signal transducer and activator of transcription 1 (STAT1) by interferon-γ (IFNγ)- and/or interleukin-27 (IL-27), both of which upregulate T-bet. T-bet induces the expression of H2.0-like homeobox (HLX), and together they collaborate with transcription factors that are activated following T-cell receptor (TCR) signalling to activate *Ifng* transcription and to antagonize GATA-binding protein 3 (GATA3). These events result in the expression of IL-12 receptor (IL-12R), which binds IL-12 that is secreted by antigen-presenting cells, such as dendritic cells (DCs), and thereby mediates the activation of STAT4. T-bet also induces the activation of runt-related transcription factor 3 (RUNX3), and along with STAT4, these transcription factors drive Tₕ₁-cell differentiation. IL-2-induced STAT5 activation has a permissive role in the initial stages of Tₕ₁-cell differentiation.
- **b | T-cell differentiation to the Tₕ₂-cell lineage** involves the induction of GATA3. GATA3 activation is mediated by STAT6 and IL-4, which is activated by STAT5 and STAT6 and/or recombination-signal-binding protein for immunoglobulin-κ J region (RBPI). This establishes a positive-feedback loop that drives Tₕ₂-cell differentiation and the expression of IL-4, IL-5 and IL-13. IL-2-induced STAT5 activation has a permissive role in the initial stages of Tₕ₂-cell differentiation.
- **c | Tₕ₁₇-cell differentiation** is initiated by the activation of STAT3, which induces the expression of IL-21 and cooperates with transforming growth factor-β (TGFβ) signalling to induce the expression of retinoic-acid-receptor-related orphan receptor-γt (RORγt), IL-17 and IL-23R, and STAT3 activation is attenuated by IL-2-induced STAT5. IL-21 and IL-23 drive the production of IL-17 and IL-22 and Tₕ₁₇-cell differentiation. These pathways also induce epigenetic remodelling at genes that encode lineage-restricted transcription factors and cytokines to facilitate heritable patterns of gene expression and lineage commitment. IFNGR, IFNγ receptor; NK, natural killer.

---

© 2009 Macmillan Publishers Limited. All rights reserved.

Histone code
Post-translational
modifications of histone tails
that involve characteristic
clusters of modifications,
including acetylation,
phosphorylation,
ubiquitylation, methylation,
sumoylation and ADP-
ribosylation that combine
to create an epigenetic
mechanism for the regulation
of gene expression.

Heterochromatin
Highly compacted chromatin
that is transcriptionally
inactive. Includes structural
regions of the chromosome
that lack genes (for example,
centromeres; known as
constitutive heterochromatin)
as well as genes that are
silenced in a given cell type
(known as facultative
heterochromatin).

into nucleosomes or removed, and histones can be
modified by the enzyme-catalysed addition or removal
of acetyl, methyl, phosphate, ubiquitin, sumoyl or ADP-
ribose groups<sup>34</sup>. Acetylation alters histone charge, which
reduces the interactions between histones and DNA and
enhances nucleosome mobility<sup>36</sup>. In addition, incorpora-
tion of histone H1 condenses chromatin<sup>43</sup>, and post-
translational modifications of histones create or remove
binding sites for regulatory proteins that facilitate or
impede transcription.

The diversity of potential modifications led to the
‘histone code’ and later ‘nucleosome code’ hypothesis,
which posits that histone modifications combine to create
a code that is recognized by specific regulatory proteins
or complexes that are involved in transcription<sup>36</sup>. Genome-wide studies identified many different combinations of histone modifications that could allow for fine-tuning of transcription, but a few sets of core modifications seem to be sufficient to characterize genes and regulatory elements as active and accessible, inactive but poised, or silent<sup>44</sup> (FIG. 2). Specifically, the promoters and enhancers of active or recently transcribed genes can be

characterized by the presence of histones H3 and H4 that
are acetylated at various residues, by H3 lysine 4 (H3K4)
that is modified with one (monomethylated H3K4), two
(dimethylated H3K4) or three (trimethylated H3K4)
methyl groups (BOX 1; TABLE 1) and by an alternative
form of histone H2A, known as H2A.Z. These histone
modifications are absent from silenced genes, whereas
dimethylated and trimethylated H3K27, or dimethylated
and trimethylated H3K9 are present. Dimethylated and
trimethylated H3K27 are typically found throughout the
condensed, facultative heterochromatin of silenced tissue-
specific loci, and dimethylated and trimethylated H3K9
are found throughout constitutive heterochromatin or in
discrete sites within active genes, where they may inhibit
inappropriate transcription<sup>33</sup>. Promoters of genes that
are poised to be either activated or silenced generally
do not have any of these histone modifications or have a
bivalent modification pattern — that is, they have both
permissive (H3K4 dimethylation and trimethylation) and
repressive (H3K27 trimethylation) histone modifications<sup>33,44–46</sup>. Locus control regions (LCRs) and distal
enhancers (BOX 1) at an inactive locus that are bivalent or


**Figure 2 | Chromatin and chromatin modifications.**  
**a** | DNA is compacted through its association with histone proteins to form chromatin, the basic unit of which is the nucleosome. Nucleosomes consist of two copies of histones H2A, H2B, H3 and H4. Each nucleosome is encircled by approximately ~156 base pairs of DNA and interconnected by linker DNA. Wrapping of DNA around nucleosomes generates a 10 nm fibre that is typical of euchromatin, which can be further compacted into a 30 nm fibre that is typical of heterochromatin. **b** | Binding of transcriptional regulatory and chromatin remodelling proteins displaces nucleosomes, and the sites at which nucleosomes have been displaced can be detected as DNasel hypersensitive sites. **c** | Representative acetylation (Ac) and methylation modifications to the tails of histones H3 (red line) and H4 (blue line) at promoters and enhancers of genes that are silent, inactive but poised, or active and accessible. For simplicity, modifications are shown on only one of the two histone tails but may be present alone or in combination on one or both. H3K4 modified with one, two and/or three (me1, me2 and/or me3) methyl groups is permissive. H3K9 and H3K27 modified with two and/or three (me2 and/or me3) methyl groups are repressive.
only have methylated H3K4 may be ‘pioneer’ elements that are amenable to subsequent activation or silencing, at which time histone modifications and DNA methylation at these elements are altered accordingly ${ }^{41,42}$.

The epigenetic state of regulatory elements is also altered when transcription factors and RNA polymerase II that are bound recruit chromatin-remodelling complexes that displace or alter the conformation of nucleosomes. Such regions are accessible to cutting by DNaseI, allowing the regulatory elements to be detected as DNasel hypersensitive sites. Therefore, functional regulatory elements in a given cell can be experimentally identified by approaches that detect DNaseI hypersensitive sites, post-translational histone modifications and differential DNA methylation ${ }^{35}$.

### Epigenetics and T<sub>H</sub>-cell subsets

**Cause or consequence.** Transcription factors may directly transactivate (induce) or repress gene expression and may also affect transcription by recruiting proteins that modify the epigenetic state of genes to which they bind or by blocking the recruitment of these proteins. Epigenetic modifications may persist in the absence of the transcription factors that initially induced them, but do such modifications merely mark past events and report transcriptional competence or do they also contribute to differences in competence? In other words, in addition to the transcription factor networks described above, which initiate and help to sustain T<sub>H</sub>-cell subset differentiation, do epigenetic modifications also help to maintain these differentiated states?

Subsequently, studies showed that treatment of CD4<sup>+</sup> T cells with inhibitors of histone deacetylases (HDACs) enhanced the expression of both IFNγ and T<sub>H</sub>2-type cytokines ${ }^{49,50}$. Genetic evidence supported these findings: conditional ablation of DNMT1 or MBD2 — which recruit HDACs and chromatin-remodelling complexes to methylated DNA and induce a repressive chromatin state — led to increased expression of IFNγ and T<sub>H</sub>2-type cytokines and an inability of T<sub>H</sub>1 or T<sub>H</sub>2 cells to silence the expression of cytokine genes that are associated with the opposing lineage ${ }^{51-54}$. These studies also suggested that DNMT1 and MBD2 mediate gene silencing mostly, if not wholly, by directly affecting the loci that encode IFNγ and T<sub>H</sub>2-type cytokines. Thus, DNA methylation, MBDs and histone deacetylation dampen the expression of both T<sub>H</sub>1- and T<sub>H</sub>2-type cytokines, and help to restrict cytokine expression to the appropriate lineage.

A prediction of these findings is that lineage-specific transcription factors regulate T<sub>H</sub>1- and T<sub>H</sub>2-cell fate in part through epigenetic processes. Consistent with this possibility, chromatin-remodelling complexes that contain Brahma-related gene 1 (BRG1; also known as SMARCA4) displace nucleosomes and remodel chromatin at the Ifng promoter in mouse T<sub>H</sub>1 cells in a STAT4-dependent manner; these complexes are required for high Ifng expression ${ }^{55}$. In addition, mice that are haplo-insufficient for the H3K4 methyltransferase MLL have a defect in the maintenance but not the induction of Gata3, Il4, Il5 and Il13 expression, whereas T<sub>H</sub>1-cell differentiation is not affected ${ }^{56}$. Conversely, mice that lack expression of MEL18, a polycomb repressor complex 1 (PRC1) protein that binds to trimethylated H3K27, have impaired GATA3 expression and T<sub>H</sub>2-cell differentiation, although the cause of these defects is not known ${ }^{57}$. These findings, together with a large body of correlative data, suggest that epigenetic mechanisms are key determinants of T<sub>H</sub>-cell differentiation and function.

Early biochemical evidence that the epigenetic state of a gene has a causal role in transcriptional competence came from studies in which treatment with 5-aza-cytidine, an inhibitor of DNA methylation, resulted in the production of IL-2 (REF. 47) and IFNγ (REF. 48) by T-cell lines that could not previously produce these cytokines.

#### Box 1 | Promoters, other regulatory elements and their interactions

Promoters are located immediately upstream of the point where transcription starts. Mammalian promoters are typically several hundred base pairs in length and contain binding sites for transcription factors, which together with the position and orientation of their binding sites helps to determine the cells and the conditions under which that gene will be expressed, as well as the magnitude of its expression. Transcription factors that are bound to the DNA create a platform to recruit the basal transcriptional machinery, which is common to all cells and consists of RNA polymerases and their associated co-factors. Protein coding (and microRNA) genes recruit RNA polymerase II-containing complexes and associated co-factors that can displace or remodel nucleosomes, can phosphorylate RNA polymerase II and can add or remove acetyl, methyl, phosphate, ubiquitin, sumoyl or ADP-ribose groups to histones and transcription factors. The content and post-translational modifications of these RNA polymerase II-containing complexes are dynamic and determine whether binding leads to transcript initiation and elongation.

Promoters are sufficient for proper gene regulation in prokaryotes, but do so in concert with other regulatory elements to achieve proper gene regulation in mammalian cells. These regulatory elements may be located just upstream of the promoter, within introns or up to hundreds of kilobases upstream or downstream of the gene (or genes) they regulate, or even on other chromosomes. Enhancers augment transcription either actively or by promoting permissive epigenetic modifications, whereas silencers repress gene expression by promoting repressive epigenetic modifications. Unlike the function of promoters, which depends on their proximity to the transcription start site and their 5′ to 3′ orientation, the function of enhancers and silencers is independent of their orientation and location. Insulators create boundaries between genes or genetic loci, thereby allowing genes in these loci to be regulated independently of neighbouring regulatory elements, gene loci and chromosomal domains or territories. Locus control regions typically contain both enhancer and insulator activity and have been functionally defined by their ability to permit copy number-dependent expression of transgenes. Matrix attachment regions are found at the base of chromatin loops, which they tether to structures such as the nuclear matrix.

---

Locus control region  
A DNA sequence that is defined by its ability (in transgenic assays) to permit high-level, tissue-specific expression of a linked promoter at all integration sites.

Chromatin-remodelling complex  
An enzymatic complex that carries out the remodelling of DNA–nucleosomal architecture and determines transcriptional activity. The SWI–SNF (switching-defective–sucrose non-fermenting) ATPases are an example of complexes that remodel chromatin.

DNasel hypersensitive site  
A region of chromatin (usually less than a few hundred base pairs) that is ~100 times more sensitive to digestion by DNasel than bulk chromatin and corresponds to regions in which nucleosomes are depleted. Regulatory elements, including enhancers, promoters and insulators, which are functional in the cells being assayed, typically map to these sites.
Table 1 | Histone lysine modifications

| Modification          | Histone lysines modified                                                                 | Transferases that add modification*                                                                 | Deacetylases or demethylases that remove modification                                                                                          | Function of histone modification                                                                                                      | Effect on transcription |
|----------------------|---------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|------------------------|
| Acetylation          | H3K9, H3K14 and H3K18                                                                  | KAT2A (GCN5) and KAT2B (PCAF)                                                                   | HDAC1 and HDAC2 (in SIN3A, NURD and CoREST complexes)                                                                                      | Binds or recruits bromodomain-containing proteins (such as TAF1)                                                                      | Permissive             |
|                      | H4K5, H4K8, H4K12 and H4K16                                                            | KAT5 (TIP60)                                                                                    | HDAC1 and HDAC2 (in SIN3A, NURD and CoREST complexes)                                                                                      | Binds or recruits bromodomain-containing proteins (such as TAF1)                                                                      | Permissive             |
|                      | H2aK5, H2bK12, H2bK15, H3K14, H3K18, H4K5 and H4K8                                     | KAT3A (CBP) and KAT3B (p300)                                                                    | HDAC1 and HDAC2 (in SIN3A, NURD and CoREST complexes)                                                                                      | Binds or recruits bromodomain-containing proteins (such as TAF1)                                                                      | Permissive             |
| Methylation          | H3K4 monomethylation                                                                  | KMT7 (SET7 or SET9)                                                                             | KDM1 (LSD1) and KDM5B (JARID1B)                                                                                                            | Binds or recruits the WDR5 component of the H3K4 methyltransferase MLL complex                                                       | Permissive             |
|                      | H3K4 dimethylation and trimethylation                                                  | KMT2A–KMT2E (MLL1–MLL5)                                                                         | KDM1 (LSD1) and KDM5A–KDM5D (JARID1A–JARID1D)                                                                                             | Binds or recruits chromodomain, PHD- and Tudor-domain-containing proteins (such as TFIID, CHD1, BPTF and WDR5)                         | Permissive             |
|                      | Unmodified H3K4                                                                       |                                                                                                 | Binds or recruits NURD complexes and DNMT3a–DNMT3l                                                                                           |                                                                                                                                          | Repressive             |
|                      | H3K27 dimethylation and trimethylation                                                 | KMT6 (EZH2)                                                                                    | KDM6A and KDM6B (UTX and JMJD3)                                                                                                             | Binds or recruits PRC1 complex and DNA methyltransferases                                                                            | Repressive             |
|                      | H3K9 dimethylation and trimethylation                                                  | KMT1B (SUV39H) and KMT1C (G9a)                                                                  | KDM1 (LSD1) and KDM4A–KDM4D (JMJD2A–JMJD2D)                                                                                                | Binds or recruits CBX5 (HP1) and DNA methyltransferases                                                                              | Repressive             |

See REFS 34,36,44,141–143.*Alternative name is included in brackets. Also note that histone acetyltransferases are now referred to as KATs in recognition of their broader substrate specificity. BPTF, bromodomain and PHD finger transcription factor; CBP, CREB-binding protein; CBX5, chromobox homologue 5; CHD1, chromodomain-helicase-DNA-binding protein 1; CoREST, corepressor of REST; DNMT3a, DNA methyltransferase 3a; EZH2, enhancer of zeste homologue 2; GCN5, general control non-depressible 5; HDAC, histone deacetylase; HP1, heterochromatin protein 1; JARID1, jumonji AT-rich interactive domain 1; JMJD, jumonji-domain-containing protein histone demethylase; KAT, lysine acetyltransferase; KDM, lysine demethylase; KMT, lysine methyltransferase; LSD1, lysine-specific histone demethylase 1; NURD, nucleosome remodelling and histone deacetylation; PCAF, p300/CBP-associated factor; PHD, plant homeodomain; PRC1, polycomb repressive complex 1; SUV39H, suppressor of variegation 3–9 homologue; TAF1, TFIID subunit 1; TFIID, transcription factor IID; TIP60, Tat interactive protein, 60 kDa; UTX, ubiquitously transcribed X chromosome tetratricopeptide repeat; WDR5, WD-repeat-containing domain 5.

However, it is worth noting that whether the specific epigenetic modifications described below are causally related to differences in T-cell function or whether they merely report these differences has for the most part not been directly determined.

**The $T_{\mathrm{H}}$2-cytokine locus and lineage commitment.** Just over a decade ago, alterations to the chromatin structure of the $Il4$ and $Il13$ genes were shown to occur as naive T cells differentiated into $T_{\mathrm{H}}$2 or $T_{\mathrm{H}}$1 cells^{58,59}. These findings stimulated interest in the contribution of epigenetic processes to $T_{\mathrm{H}}$-cell differentiation and in the $T_{\mathrm{H}}$2-cytokine locus as a model system by which to address the coordinated regulation of clustered, functionally related genes. The accrual of additional information was greatly accelerated by the availability of complete genomic sequences for humans, mice and other species, and by improvements in the techniques used for assessing epigenetic modifications.

The mouse $T_{\mathrm{H}}$2-cytokine locus contains $Il4$, $Il5$, $Il13$ and the constitutively expressed $Rad50$ gene (FIG. 3), and is flanked by $Irf1$ and $Kif3A$. The gene composition of this locus, as well as the linear relationship of the genes within the locus, is conserved in the genomes of mammals, suggesting that these relationships are functionally important. Consistent with this possibility, the genes encoding $T_{\mathrm{H}}$2-type cytokines are regulated through their promoters and also by several additional regulatory elements, the locations of which have been experimentally determined by the detection of DNaseI hypersensitive sites, histone modifications and differential DNA methylation, as well as through computational identification of conserved non-coding sequences (CNS). Various approaches have been used to test the function of the elements that have been identified by these methods and their interactions with transcription factors^{35}.

In the mouse $T_{\mathrm{H}}$2-cytokine locus, the transcription of $Il4$ is enhanced by regulatory elements that map to DNaseI hypersensitive site I (HSI) and HSII in the second intron of $Il4$, to DNaseI hypersensitive site $V_{\mathrm{A}}$ (HSV$_{\mathrm{A}}$) and HSV located 3′ of $Il4$, to DNaseI hypersensitive site s1 (Hss1) and Hss2 located between $Il13$ and $Il4$, and to the $T_{\mathrm{H}}$2-cytokine LCR, which encompasses $Rad50$ hypersensitive site 4 (RHS4), RHS5, RHS6 and RHS7 (FIG. 3; see figure legend for the convention used to name hypersensitive sites in this locus and note that HSV maps to CNS2, and Hss1 and Hss2 map to CNS1)^{5}. The expression of $Il13$ is augmented by regulatory elements at CNS1, the $T_{\mathrm{H}}$2-cytokine LCR and HS1, which maps to the CG-rich element (CGRE) upstream of the $Il13$ promoter. Many of these enhancers and each of the $T_{\mathrm{H}}$2-cytokine promoters are direct targets of NFAT, other TCR-induced transcription factors and $T_{\mathrm{H}}$2-cytokine-promoting transcription factors. For example, STAT6 binds to the

FOCUS ON CD4⁺ T-CELL DIVERSITY

![Diagram](#)

Figure 3 | The T helper 2 cytokine locus in mouse T cells. Naive CD4⁺ T cells have DNaseI hypersensitive sites at hypersensitive site s3 (Hss3), HSIV, the 5′ end of the Rad50 gene at Rad50 hypersensitive site 2 (RHS2) and RHS3, and perhaps at RHS6 in the locus control region (LCR) of the T helper 2 (Tₕ₂)-cytokine locus. In addition, the locus has low levels of permissive histone modifications (light green blocks) at LCR and HSV and substantial levels at RHS2, whereas HSIV has a bivalent modification pattern with both permissive dimethylated and/or trimethylated H3K4 and repressive trimethylated H3K27 (yellow blocks) and low levels of repressive modifications at Hss3 (light blue blocks). In Tₕ₂ cells, DNaseI hypersensitive sites and substantial levels of permissive histone modifications (such as acetylated H3, acetylated H4 and dimethylated and/or trimethylated H3K4; dark green blocks) are acquired at the promoters and enhancers of Il4 (interleukin-4), Il13 and Il5, and repressive trimethylated H3K27 (blue blocks) is absent throughout the locus. DNA methylation is progressively reduced at these cytokine genes and their enhancers over time in Tₕ₂ cells (not shown). Conversely, in Tₕ₁ cells repressive trimethylated H3K27 spreads to encompass Il4, Il13 and a region that extends from conserved non-coding sequence 1 (CNS1) to CNS2 (to our knowledge, data for Rad50 and Il5 are not yet available). The function of specific elements, such as promoters (pro), enhancers (enh), silencers (sil) and the LCR are indicated. The binding sites for the lineage-restricted transcription factors MAF, CCCTC-binding factor (CTCF), GATA-binding protein 3 (GATA3), runt-related transcription factor 3 (RUNX3), signal transducer and activator of transcription 5 (STAT5), STAT6 and T-bet are also shown. Gene locations, intergenic CNS (that is, intergenic regions where there is ≥70% sequence conservation between humans and mice that extends ≥100 base pairs as identified at the VISTA web site) and the size of the region depicted are shown. DNaseI hypersensitive sites in the Tₕ₂-cytokine locus are referred by their commonly used names, in which sites in or near Il4 are indicated as HS followed by a roman numeral (for example, HSIV), sites between Il4 and Il13 are indicated as Hss followed by a number (for example, Hss1), sites in or upstream of Il13 are indicated as HS followed by a number (for example, HS1), and sites in or near Rad50 are indicated as RHS followed by a number (for example, RHS6). T-cell subset-specific patterns of DNA methylation in this locus are described in the text.

Il4 and Il13 promoters and to HSVₐ, as well as to RHS6 and RHS7; GATA3 binds to the Il5 and Il13 promoters, to the first intron of Il4, HSVₐ and RHS7, and to the Il13 HS1–CGRE region; and recombination-signal-binding protein for immunoglobulin-κ J region (RBPI), which is the DNA-binding component of the Notch pathway, binds to HSV⁶⁰,⁶¹. Perhaps surprisingly, none of these elements, including the Tₕ₂-cytokine LCR, appears to affect the expression of Il5, to which GATA3 binds⁵.

Naive CD4⁺ T cells express low levels of GATA3 and T-bet and produce small but detectable amounts of Il4, Il5 and Il13 mRNA before cell division when activated by TCR ligation⁶²,⁶³. In these cells, the Tₕ₂-cytokine locus is characterized by a paucity of DNaseI hypersensitive sites and histone modifications, although some are still present (FIG. 3). Specifically, DNaseI hypersensitive sites are found in the promoter of the constitutively expressed

Rad50 gene, at HSIV (the Il4 silencer) and Hss3 (one of three DNaseI hypersensitive sites that are located between Il4 and Il13, which is of unknown function), as well as at RHS6 in the Tₕ₂-cytokine LCR (although this was observed in one study⁶⁴ but not another⁶⁵). HSIV also has bivalent histone modifications (that is, both permissive H3K4 dimethylation and H3 acetylation, and repressive H3K27 trimethylation are detectable)⁶⁶,⁶⁷, whereas Hss3 is marked solely by repressive trimethylated H3K27. The 3′ Il4 enhancer, Il4 CNS2 and the Tₕ₂-cytokine LCR are marked weakly by dimethylated H3K4 and/or H3 acetylation⁶⁴,⁶⁷,⁶⁸.

The expression of Tₕ₂-type cytokines is probably restrained in naive T cells by the high degree of CpG methylation (~90%) at their promoters, CNS1, CNS2 and the Tₕ₂-cytokine LCR⁴⁰,⁵¹,⁵³,⁶⁴,⁶⁹. The Il4 promoter is less methylated (~60%) than the other promoters
and lacks CpG motifs in the 250 base pairs that are proximal to the start site<sup>53</sup>, which might facilitate the early low-level expression of IL-4. Therefore, in naive CD4<sup>+</sup> T cells, modest amounts of permissive and repressive epigenetic marks are focally targeted to a subset of the known regulatory elements in the T<sub>H</sub>2-cytokine locus. This bivalent epigenetic state may help poise this locus, permitting TCR-induced transcription factors to bind and induce early, low-level expression of T<sub>H</sub>2-type cytokines and providing the potential for this locus to convert to a fully permissive state during T<sub>H</sub>2-cell differentiation or to a silenced state during T<sub>H</sub>1- and T<sub>H</sub>17-cell differentiation<sup>5,6</sup>.

Permissive histone modifications are acquired in the T<sub>H</sub>2-cytokine locus in the first 24–48 hours following activation of naive CD4<sup>+</sup> T cells under both T<sub>H</sub>1- and T<sub>H</sub>2-cell-polarizing conditions through the actions of NFAT and other transcription factors that are induced by TCR signalling<sup>70,71</sup>. These transcription factors also induce the expression of IL-2, which activates STAT5, which in turn binds to and induces chromatin remodelling at intron 2 of *Il4* to promote T<sub>H</sub>2-cell differentiation<sup>72</sup>. However, the continued presence and progressive increase in these permissive histone modifications, and the subsequent acquisition of T<sub>H</sub>2-specific DNaseI hypersensitive sites and DNA demethylation at the T<sub>H</sub>2-cytokine locus are unique to T<sub>H</sub>2 cells<sup>20,64–68,70,72–74</sup>.

The active locus of effector T<sub>H</sub>2 cells contains the hypersensitive sites that are found in naive T cells as well as new sites at the *Il4*, *Il5* and *Il13* promoters, at each of the known enhancers and at the LCR, although the cells must be restimulated for HSV<sub>A</sub> to be detected. Permissive H3 and H4 acetylation and H3K4 dimethylation are acquired or become more prominent at these elements, and repressive H3K27 trimethylation is lost throughout the locus. These changes are evident in the first week of mouse T<sub>H</sub>2-cell differentiation in culture. By this time, demethylation of the DNaseI hypersensitive sites has commenced but it progresses slowly by a passive replication-dependent process. By contrast, RHS7 undergoes a more rapid and potentially active demethylation<sup>40,51,53,64,69,75</sup>. With the exception of the demethylation of RHS5 and RHS6, these epigenetic changes are not found in the T<sub>H</sub>2-cytokine locus of T<sub>H</sub>1 cells; instead, the locus is modified with repressive H3K27 trimethylation<sup>66</sup>.

GATA3 is necessary and apparently sufficient to induce most, if not all, of these T<sub>H</sub>2-cell-specific epigenetic modifications. It may do so through the direct or indirect recruitment of histone acetyltransferases (HATs) and histone H3K4 methyltransferases<sup>5,56</sup>, the displacement of MBD2 and associated HDAC-containing complexes<sup>51</sup>, the displacement of DNMT1 and inhibition of DNA methylation<sup>53,69</sup>, and the recruitment of chromatin-remodelling complexes to the mouse T<sub>H</sub>2-cytokine locus<sup>76</sup>. However, the molecular details of the mechanism by which GATA3 induces epigenetic modifications remain to be fully elucidated. In physiological situations, STAT6 and/or Notch signalling are activated before GATA3 and help to induce GATA3 expression by binding to, transactivating and recruiting HATs and

other chromatin modifiers to one or both of its promoters<sup>18,74,77</sup>. Once induced, GATA3 binds to its promoter, sustaining its own expression through direct activation and recruitment of the H3K4 methyltransferase MLL<sup>56</sup>. STAT6 also facilitates T<sub>H</sub>2-cell differentiation by binding to multiple sites on the T<sub>H</sub>2-cytokine locus. In addition to the STAT6-dependent pathway, T<sub>H</sub>2-cell differentiation can be induced independently of STAT6, partly through the conversion of RBPJ that is bound to the *Il4* CNS2 to a co-activator of *Il4* in a process that depends on Notch signalling<sup>61</sup>. Therefore, although T<sub>H</sub>2-cell differentiation can be initiated either through STAT6-dependent or Notch-dependent pathways, it is stabilized by the autoactivation of GATA3 and by the GATA3-mediated epigenetic modification of *Gata3* and the T<sub>H</sub>2-cytokine locus.

**IFNγ and T<sub>H</sub>1-lineage commitment.** IFNG is not clustered with other co-expressed cytokine genes. In all vertebrates except rodents, the nearest upstream neighbours of *IFNG* are *IL22* and *IL26*, which are mainly expressed by T<sub>H</sub>17 cells<sup>21,31,78</sup>, with the housekeeping gene *MDM1* located further upstream; the nearest downstream gene to *IFNG* is ~500 kilobases away. In mice and rats, complex structural rearrangements are evident >70 kilobases upstream of *Ifng*, and *Il26* is absent; a few remnants of sequences from *Il26* remain, which is consistent with this gene having been lost in rodents as a result of the structural rearrangements<sup>79</sup> (FIG. 4). Despite these differences, cell-type-specific patterns of IFNG expression are similar in rodents and humans, suggesting that the essential regulatory elements and their relationships are conserved and are proximal to these structural changes. Consistent with this possibility, multiple regulatory elements and CNS have been identified in a region that extends 60–70 kilobases upstream and downstream of the mouse *Ifng* locus. These include enhancers at CNS-34, CNS-22, CNS-6, CNS+18–20 and CNS+29, as well as a putative insulator (BOX 1) at CNS+46, although enhancer function has been confirmed *in vivo* for only CNS-22 (REF. 80) and a previously described enhancer in intron 1 (REF. 81). Recent genome-wide analyses of DNaseI hypersensitive sites and histone modifications in human CD4<sup>+</sup> T cells suggest that a similar set of regulatory elements is present in the human *IFNG* locus<sup>44,46,82</sup> (FIG. 4).

Activated naive CD4<sup>+</sup> T cells produce low levels of IFNγ, indicating that the *Ifng* locus is in a poised state. DNA at the mouse *Ifng* promoter and at CNS-34, CNS-22, CNS+29 and CNS+46 is demethylated in naive T cells, and CNS-34 and CNS-22 exhibit low levels of permissive H3K4 dimethylation and H4 acetylation<sup>38,79,80</sup>. Conversely, moderate levels of repressive H3K27 trimethylation are present between *Ifng* and CNS+18–20, from CNS+29 to CNS+46 and adjacent to CNS-22. Therefore, overall the *Ifng* locus has bivalent histone modifications, which makes it poised for either expression or silencing. T<sub>H</sub>1-cell differentiation *in vitro* and in response to infection *in vivo* leads to a marked increase in H3K4 dimethylation, H3 and/or H4 acetylation, the acquisition of DNaseI hypersensitive sites at regulatory elements within the *Ifng* locus and a complete loss of repressive
FOCUS ON CD4+ T-CELL DIVERSITY

![Diagram](#)

Figure 4 | The *Ifng* locus in mouse naive, T helper 1 and T helper 2 cells and human CD4+ T cells. Naive mouse CD4+ T cells have DNase I hypersensitive sites at conserved non-coding sequence -34 (CNS-34) and near CNS+46, low levels of permissive histone modifications (such as acetylated H3, acetylated H4 and dimethylated and/or trimethylated H3K4; light green blocks) at *Ifng* (interferon-γ) CNS-22 and CNS-34 and repressive trimethylated H3K27 (light blue blocks) at the 3′ end of the locus. DNA at CNS-34, CNS-22, the *Ifng* promoter (pro), CNS+29 and CNS+46 is demethylated. In T helper 1 (*T*~H~1) cells, hypersensitive site I (HSI), HSII and HSIII, DNase I hypersensitive sites at several CNS enhancers sites, and high levels of permissive histone modifications (dark green blocks) are acquired, whereas trimethylated H3K27 is lost. In addition, DNA demethylation occurs at *Ifng* CNS-54, CNS-6 and CNS+18 (not shown). The opposite occurs in *T*~H~2 cells, in which high levels of repressive trimethylated H3K27 (dark blue blocks) spreads throughout the locus. The elements that have permissive chromatin modifications in mouse *T*~H~1 cells are DNase I hypersensitive in total human CD4+ T cells<sup>82</sup> and in *T*~H~1 cells (C.B.W. and M.S., unpublished observations). DNase I hypersensitive sites in human CNS-4 and CNS+80 of the *IFNG* locus were detected only in *T*~H~1 cells, and for this reason are denoted by red arrows rather than black arrows (which denote sites that are also found in total human CD4+ T cells). The function of specific elements, such as promoters, enhancers (enh) and insulators (ins) are indicated, as are the binding sites for the lineage-restricted transcription factors CCCTC-binding factor (CTCF), GATA-binding protein 3 (GATA3), runt-related transcription factor 3 (RUNX3), signal transducer and activator of transcription 4 (STAT4), STAT5, STAT6 and T-bet. Gene locations and intergenic CNS (that is, intergenic regions where there is ≥70% sequence conservation between humans and mice that extends ≥100 base pairs as identified at the VISTA web site) are shown at the bottom. T-cell subset-specific patterns of DNA methylation at this locus are described in the text.

H3K27 trimethylation throughout the locus<sup>59,79,80,83–85</sup>. However, repressive H3K9 dimethylation is induced and retained at specific sites of the *Ifng* locus in *T*~H~1 cells<sup>86</sup>, in which it may help to prevent aberrant transcriptional initiation<sup>33</sup>. By contrast, when naive T cells differentiate into *T*~H~2 cells, permissive histone modifications are lost, repressive H3K27 trimethylation across the locus and CpG methylation are increased and NFAT loses the ability to bind to the *Ifng* promoter<sup>38,79,83,86</sup>.

Similarly to the *T*~H~2-cytokine locus, permissive histone modifications are initially acquired at the *Ifng* locus under both *T*~H~1- and *T*~H~2-cell-inducing conditions, but *T*~H~1-cell-specific changes are evident by 17 hours following stimulation<sup>70</sup>. STAT1, STAT4 and STAT5 all contribute to these modifications, although they are not essential for *T*~H~1-cell differentiation *in vivo*<sup>14</sup>. STAT1 probably promotes the transcription of *Ifng* through T-bet expression

and has not been shown to bind to the endogenous *Ifng* gene. As with *T*~H~2-cell differentiation, STAT5 directly promotes the transcription of *Ifng* by binding to CNS-6, the *Ifng* promoter and CNS+18–20, thereby facilitating histone acetylation, chromatin remodelling and T-bet binding to the *Ifng* promoter<sup>87</sup>. STAT4 binds to the *Ifng* promoter and many other elements, including CNS-22, leading to the induction of permissive epigenetic modifications and the activation of gene expression. In addition, STAT4 recruits BRG1-containing chromatin-remodelling complexes to the *Ifng* promoter, induces the acquisition of HSI and HSII and promotes permissive histone modifications at this site<sup>17,55</sup>.

Although STAT4 supports *Ifng* expression and *T*~H~1-cell differentiation synergistically with T-bet<sup>88</sup>, the expression of which is enhanced by STAT4 (REF. 89), T-bet can induce *T*~H~1-cell differentiation and the

T<sub>H</sub>2 cell

T<sub>H</sub>1 cell

Naive T cell

T<sub>H</sub>17 cell

Pkhdl

pro

RORγt STAT3

CNS1

CNS2

STAT3

Il17a

enh pro

Il17f

Mcm3

pro

200 kilobases

Permissive histone modifications CNS

Figure 5 | The Il17a–Il17f locus in mouse naive, T helper 1, T helper 2 and T helper 17 cells. Naive mouse CD4<sup>+</sup> T cells have weak permissive histone H3 acetylation (light green blocks) at conserved non-coding sequence 1 (CNS1), CNS5, CNS7 and CNS8, which is reduced in T helper 1 (T<sub>H</sub>1) and T<sub>H</sub>2 cells. By contrast, T<sub>H</sub>17 cells exhibit higher levels of H3 acetylation (dark green blocks) at these regions, at other CNS in this region and at the Il17a (interleukin-17a) and Il17f promoters<sup>101</sup>. Signal transducer and activator of transcription 3 (STAT3) binds to the Il17a and Il17f promoters in T<sub>H</sub>17 cells<sup>20,21,102</sup> and retinoic-acid-receptor-related orphan receptor-γt (RORγt) has been shown to bind to CNS2, at least when it is overexpressed<sup>24</sup>. DNasel hypersensitive sites have not to our knowledge been mapped in this locus in T<sub>H</sub>17 cells. Gene locations, intergenic CNS (intergenic regions where there is ≥70% sequence conservation between humans and mice that extends ≥100 base pairs as identified at VISTA web site) numbered as reported in REF. 101 and the size of the region depicted are shown at the bottom. enh, enhancer; Mcm3, minichromosome maintenance deficient 3; Pkhdl, polycystic kidney and hepatic disease 1; pro, promoter.

production of IFNγ in the absence of STAT4 when the expression of T-bet is forced<sup>71,90</sup>. However, under physiological conditions, STAT4 and IL-12 are required for T<sub>H</sub>1-cell differentiation, as their absence substantially reduces or abolishes this process<sup>91–93</sup>. T-bet directly transactivates Ifng transcription and has many additional effects. More specifically, it binds to the Ifng promoter and to many enhancers<sup>80,94,95</sup> (FIG. 4), it induces the expression of HLX and RUNX3, it binds with these transcription factors to the Ifng promoter and with RUNX3 to the Il4 silencer<sup>12,13,96</sup> and it inhibits GATA3 expression and function<sup>16,17</sup>. T-bet binds to the Ifng promoter even when its DNA is repressively methylated<sup>97</sup>, where it displaces HDAC-containing complexes and possibly recruits HATs<sup>98</sup>. Moreover, the Weinmann laboratory<sup>99</sup> recently showed that T-bet can directly recruit jumonji-domain-containing protein histone demethylase 3 (JMJD3) to remove repressive H3K27 trimethylation and the histone methyltransferases SET7 (also known as SET9 and KMT7) to induce H3K4 dimethylation, thereby creating a permissive chromatin state. Together, these functions probably explain how forced expression of T-bet can induce the expression of IFNγ even in committed T<sub>H</sub>2 cells<sup>100</sup>.

**T<sub>H</sub>17-cytokine loci and lineage commitment.** Much less is known about the regulatory mechanisms and epigenetic processes that control T<sub>H</sub>17-cell differentiation. IL-17A and IL-17F are typically co-expressed by T<sub>H</sub>17 cells, and the genes that encode them are co-localized in mammals, suggesting that they may be coordinately regulated by shared regulatory elements (similarly to the T<sub>H</sub>2-type cytokines). In mice, eight candidate regulatory elements have been described in the Il17 locus based on sequence conservation<sup>101</sup> (FIG. 5). In these eight elements, as well as the Il17a and Il17f promoters, permissive H3 acetylation is induced or increases solely or to a greater degree in naive CD4<sup>+</sup> T cells that are cultured under T<sub>H</sub>17-cell-inducing conditions than in those that are cultured under T<sub>H</sub>1- or T<sub>H</sub>2-cell-inducing conditions<sup>101</sup>. Consistent with its crucial role in T<sub>H</sub>17-cell differentiation, STAT3 binds to and induces H3 acetylation at the Il17a and Il17f promoters<sup>20,21,102</sup>. The T<sub>H</sub>17-lineage-specific transcription factors RORγt and RORα do not seem to bind to these promoters, but can bind to CNS2, which is a ROR-dependent enhancer that is located just upstream of Il17a<sup>24</sup>. The transcription factors that bind to the other CNS and their function, if any, are unknown. The genes that encode other T<sub>H</sub>17-type cytokines (IL-21, IL-22 and IL-26) are located on different chromosomes and, curiously, are close to genes that are expressed by T<sub>H</sub>1 cells but not by T<sub>H</sub>17 cells. Il21 is located next to Il2, and Il22 and Il26 (or its remnants in rodents) are located next to Ifng. The juxtaposition of T<sub>H</sub>1-type cytokine genes to T<sub>H</sub>17-type cytokine genes suggests that they may be regulated in part by competition for or alternative use of regulatory elements. However, these possibilities, and the epigenetic processes by which this might be achieved, have not been investigated.

**Structuring regulatory relationships**

The location of regulatory elements within the genome is linear and fixed, whereas the actual relationships of these elements to each other and to their cognate genes are nonlinear, mobile and dynamic. The three-dimensional structure of chromosomes changes during cell differentiation,
such that loops of chromatin containing relevant genes can be extended outside the chromatin territory. Such ‘chromatin looping’ can bring distal regulatory elements in proximity to one another and to the promoters of their target genes, thereby facilitating gene expression ${ }^{103-105}$.

Chromatin looping allows the $I l 4$, $I l 5$ and $I l 13$ promoters to come into proximity with each other in many cell types, which results in a basal conformation that is not specific to T cells. However, studies in mice have shown that the LCR is located close to these promoters only in T cells, apparently helping to poise this locus for subsequent activation ${ }^{106}$. STAT6, the $\mathrm{T}_{\mathrm{H}} 2$ LCR and probably GATA3 are required to establish this poised conformation during T-cell development, but once it has been established, the conformation is maintained and is similar in naive, $\mathrm{T}_{\mathrm{H}} 1$ and $\mathrm{T}_{\mathrm{H}} 2$ cells. $\mathrm{T}_{\mathrm{H}} 2$-cell-specific changes to this conformation are induced in response to cell activation in $\mathrm{T}_{\mathrm{H}} 2$-cell-inducing conditions. This triggers the expression of special AT-rich sequence binding protein 1 (SATB1; an architectural factor), which binds to CNS1, CNS2 and other sites across the locus and promotes the formation of additional loops and more intimate interactions between regulatory elements and the $I l 4$, $I l 5$ and $I l 13$ promoters ${ }^{107}$. In the absence of SATB1 these changes are lost and $\mathrm{T}_{\mathrm{H}} 2$-type cytokine expression is compromised. SATB1-induced chromatin looping extends to include the flanking $K i f 3 a$ gene, suggesting that the regulatory domain of the $\mathrm{T}_{\mathrm{H}} 2$-cytokine locus includes this gene.

Genome-wide studies suggest that CCCTC-binding factor (CTCF) may also be involved in the three-dimensional organization of the $\mathrm{T}_{\mathrm{H}} 2$-cytokine locus. CTCF is a self-interacting, insulator protein that can mediate chromatin looping to proximal elements within a locus and can insulate a locus from surrounding chromatin, nearby genes and regulatory elements ${ }^{105,108}$. CTCF co-localizes with cohesins, which contributes to the CTCF-dependent insulator function and perhaps to CTCF-mediated chromatin looping ${ }^{109}$. In mouse and human T cells, CTCF and cohesins have been shown to strongly bind to sites that flank the $\mathrm{T}_{\mathrm{H}} 2$-cytokine locus as well as to RHS2 and Hss3 within the locus ${ }^{46,109}$.

Similar mechanisms may be involved at the $I f n g$ and $I l 17$ loci, which are flanked by sites where CTCF and cohesins are bound ${ }^{46,109}$. CTCF binds to $I f n g$ in $\mathrm{T}_{\mathrm{H}} 1$ cells, but not $\mathrm{T}_{\mathrm{H}} 2$ cells ${ }^{109}$, and may help to induce a $\mathrm{T}_{\mathrm{H}} 1$-specific locus architecture (M.S. and C.B.W., unpublished observations), including $\mathrm{T}_{\mathrm{H}} 1$-specific chromatin looping that brings $I f n g$ CNS+18–20 close to the $I f n g$ promoter ${ }^{110}$. Interchromosomal interactions between the $I f n g$ and the $\mathrm{T}_{\mathrm{H}} 2$-cytokine locus in naive T cells, which are lost after $\mathrm{T}_{\mathrm{H}} 1$- and $\mathrm{T}_{\mathrm{H}} 2$-cell differentiation, have also been described ${ }^{110}$, but the molecular basis and functional importance of these interactions remain unclear.

$\mathrm{T}_{\mathrm{H}} 2$ cells did not express lineage-inappropriate cytokines or lose robust expression of lineage-appropriate cytokines when polarizing conditions were removed or switched. Commitment was attributed to heritable alterations to the cytokine genes themselves and/or to the lineage-specifying transcription factors GATA3 and T-bet. Recent studies have identified the mechanisms that are involved in lineage commitment.

**Silencing and remembering.** Silencing of $I f n g$ in $\mathrm{T}_{\mathrm{H}} 2$ cells occurs at many levels. GATA3 and STAT6 bind to the $I f n g$ promoter, and this is associated with binding of PRC1 and the H3K27 methyltransferase EZH2 to the $I f n g$ locus, as well as with increased acquisition of repressive H3K27 trimethylation, during $\mathrm{T}_{\mathrm{H}} 2$-cell differentiation ${ }^{86}$. As a result, IFNγ expression is repressed, although another report suggests that DNA binding by EZH2 occurs in both $\mathrm{T}_{\mathrm{H}} 1$ and $\mathrm{T}_{\mathrm{H}} 2$ cells and may have other effects ${ }^{111}$. GATA3 also interacts with T-bet following its tyrosine phosphorylation by IL-2-inducible T-cell kinase (ITK) and can inhibit T-bet function ${ }^{16,98}$; this may account for the contribution of ITK to $\mathrm{T}_{\mathrm{H}} 2$-cell differentiation ${ }^{112}$. Whether the expression of $T b x 21$ (the gene encoding T-bet) is directly inhibited by GATA3 is not known. GATA3 also binds HDACs ${ }^{98}$, which it may recruit to $I f n g$, and inhibits the expression of IL-12Rβ2 and STAT4 (REF. 17). $\mathrm{T}_{\mathrm{H}} 2$-cell differentiation ${ }^{38}$ and/or loss of STAT4 signalling ${ }^{113}$ result in the recruitment of DNMT3a and an associated increase in CpG methylation at the $I f n g$ promoter, perhaps as an indirect result of reduced T-bet-mediated H3K4 methylation and binding of DNMT3a–DNMT3l complexes to unmodified H3K4 (REF. 114). Finally, the transcriptional repressor B-lymphocyte-induced maturation protein 1 (BLIMP1) represses IFNγ expression in $\mathrm{T}_{\mathrm{H}} 2$ cells, in which it is highly expressed, possibly by binding to CNS-22 in the $I f n g$ locus (FIG. 4) and in many sites near or in the $T b x 21$ gene ${ }^{115}$.

In addition to silencing $I f n g$, GATA3 is essential for promoting the expression of $\mathrm{T}_{\mathrm{H}} 2$-type cytokines and consequently for inducing and, at least partly, maintaining the $\mathrm{T}_{\mathrm{H}} 2$-cell response ${ }^{68,116,117}$. Deletion of GATA3 in $\mathrm{T}_{\mathrm{H}} 2$ cells — that had been generated *in vivo* in response to infection with *Nippostrongylus brasiliensis* or *in vitro* by culturing naive CD4 + T cells for 4–5 weeks in $\mathrm{T}_{\mathrm{H}} 2$-inducing conditions — resulted in reduced numbers of IL-5- and IL-13-producing cells (but not IL-4-producing cells) and in a reduction in the production of IL-5 and IL-13, as well as a ~50% reduction in the amount of IL-4 produced per cell ${ }^{117}$. Similarly, silencing of $I f n g$ was markedly impaired when GATA3 was deleted at the start of $\mathrm{T}_{\mathrm{H}} 2$-cell differentiation. Interestingly, however, when IFNγ expression was evaluated after $G a t a 3$ had been deleted from established $\mathrm{T}_{\mathrm{H}} 2$ cells, silencing of $I f n g$ was found to be less impaired ${ }^{117}$. These findings suggest that GATA3 is not essential for but does help to maintain the permissive epigenetic state of the $\mathrm{T}_{\mathrm{H}} 2$-cytokine locus and the repressive state of the $I f n g$ locus. In addition, the data indicate that GATA3 has a modest role in transactivating $I l 4$ and a more important role in transactivating $I l 5$ and $I l 13$ in committed $\mathrm{T}_{\mathrm{H}} 2$ cells.

### Heritability, plasticity and diversity

The $\mathrm{T}_{\mathrm{H}} 1$- and $\mathrm{T}_{\mathrm{H}} 2$-cell paradigm arose from studies of long-term T-cell clones ${ }^{1}$. Commitment to a single lineage was later shown to be acquired after ≥4 cycles of cell division under T-cell-lineage inducing conditions *in vitro* ${ }^{62,63}$. Indeed, after 4 cell divisions $\mathrm{T}_{\mathrm{H}} 1$ or
Maintenance of this permissive epigenetic state and the expression of Gata3, Il4, Il5 and Il13 depends on the H3K4 methyltransferase MLL, which binds to Gata3 and the T<sub>H</sub>2-cytokine locus in memory T<sub>H</sub>2 cells, but not in naive T cells<sup>56</sup>.

The expression of T-bet seems to be crucial for the induction of T<sub>H</sub>1-cell differentiation, but is less important for maintaining the differentiation state<sup>13,118</sup>. A dominant-negative form of T-bet inhibits the expression of IFNγ and abolishes DNasel hypersensitive sites from the *Ifng* locus in developing T<sub>H</sub>1 cells, but not in long-term T<sub>H</sub>1-cell lines. The stability of T<sub>H</sub>1 clones correlates with marked DNA demethylation within the *Ifng* locus, which is acquired more slowly than permissive chromatin modifications and may contribute to or be a marker of heritable commitment. Similarly to *Il5* and *Il13* in T<sub>H</sub>2 cells, the maintenance of the expression of IL-12Rβ2 and HLX requires T-bet, as their expression was lost when T-bet function was blocked. Thus, the full programme of T<sub>H</sub>1- and T<sub>H</sub>2-cell-specific gene expression cannot be sustained without continued expression of T-bet and GATA3, respectively. And although the expression of *Ifng* in T<sub>H</sub>1 cells and *Il4* in T<sub>H</sub>2 cells can be partly sustained through heritable epigenetic modifications in the absence of these lineage-specifying transcription factors, in physiological contexts the expression of *Ifng* and *Il4* is collaboratively sustained by these transcription factors and by epigenetic processes.

T-bet has also been shown to interact with GATA3. Indeed, T-bet can bind to and inhibit GATA3 (REF. 16), but this ITK-dependent interaction is not required for silencing the T<sub>H</sub>2-cytokine locus<sup>112</sup>. T-bet also silences the expression of GATA3 in T<sub>H</sub>1 cells<sup>17</sup>, but how this is mediated and whether silencing of GATA3 and the T<sub>H</sub>2-cytokine locus is heritable and sustained in the absence of T-bet has not to our knowledge been determined. In addition, T-bet is thought to be required for the silencing of *Il4* in T<sub>H</sub>1 cells mainly, although not completely, by cooperatively binding with RUNX3 to the *Il4* silencer, which is located 3′ of *Il4* at HSIV<sup>12,96</sup>. RUNX proteins induce heritable silencing in other contexts<sup>119</sup>, but the mechanism by which they do so at the *Il4* silencer is not clear. The *Il4* silencer is crucial for silencing the expression of IL-4 in T<sub>H</sub>1 cells<sup>120</sup>, possibly through a switch that allows constitutively bound EZH2 to generate repressive trimethylated H3K27 across this locus<sup>67</sup>.

T<sub>H</sub>17-cell differentiation is strongly inhibited by IL-4, IL-27 and IFNγ, attenuated by IL-2, more readily achieved in T-bet-deficient cells and repressed by forced expression of T-bet<sup>20,21,121</sup>. The inhibitory effects of IL-2 and IL-27 on the differentiation of this lineage depend on STAT5 and STAT1, respectively. STAT1, STAT5 and STAT6 bind to the *Il17a* promoter, where they may compete with STAT3 for binding<sup>121</sup>. Inhibition by STAT5 may also be mediated by the induction of FOXP3, which binds to and inhibits RORγt<sup>22</sup>. Whether any of these STAT-dependent effects contribute directly to the silencing of IL-17 expression and the mechanisms by which T<sub>H</sub>17-cell differentiation is stably repressed are not known.


During the induction of T<sub>H</sub>17 cells, the expression of T<sub>H</sub>1- and T<sub>H</sub>2-type cytokines is repressed by TGFβ, which inhibits T-bet and GATA3 expression<sup>122,123</sup>. TGFβ-induced SMAD (mothers against decapentaplegic homologue) proteins can bind to the promoters and probably repress *Tbx21* and *Ifng* in T<sub>H</sub>17 cells<sup>124</sup>. However, there is currently no evidence suggesting that these effects are heritable in the absence of TGFβ, or that T<sub>H</sub>17-lineage commitment can be epigenetically maintained in the absence of STAT3, RORγt and RORα. There is some evidence in support of the contrary, which shows that T-bet, IFNγ, RORγt and IL-17 are often co-expressed by cells *in vivo* and that T-bet may help to induce the expression of IL-23R and be induced by IL-23 in T<sub>H</sub>17 cells<sup>125</sup>.

**Forgetting or disregarding.** A limitation of the T<sub>H</sub>1, T<sub>H</sub>2, T<sub>H</sub>17 paradigm and most of the studies cited above is that the systems that were used in each study were often designed for the purpose of showing what is possible rather than seeking to recapitulate what actually occurs *in vivo*. Although the relevance of these subsets to host defence, autoimmunity and allergy is clear, mutually exclusive, canonical phenotypes and irreversible commitment are often violated *in vivo*. CD4<sup>+</sup> T cells that express IFNγ plus IL-17 and/or IL-22 are commonly generated *in vivo*<sup>78,126–131</sup>. And although T<sub>H</sub>17 and T<sub>H</sub>1 cells typically express either CC-chemokine receptor 6 (CCR6) or CXC-chemokine receptor 3 (CXCR3), respectively, cells that produce both T<sub>H</sub>1- and T<sub>H</sub>17-type cytokines typically express both receptors<sup>31,78,128,132</sup>. Memory CD4<sup>+</sup> T cells that produce IFNγ in conjunction with IL-10, produce IL-17 and IL-10 or produce IFNγ in conjunction with IL-4 have been described in mice and humans<sup>133–139</sup>.

The *in vivo* induction of cells that co-express cytokines of mixed lineages and the lack of irreversible silencing of lineage-inappropriate cytokines in some T<sub>H</sub> cells probably reflect the more diverse and resource-limited nature of life *in vivo* as opposed to the homogeneous environments that are constructed *in vitro*<sup>140</sup>. Heterogeneity in the mix and abundance of cytokines, Notch ligands, peptide–MHC complexes and co-stimulatory molecules, asymmetric cell division and variation in the number of times cells have divided provide ample opportunities for diversification and retention of plasticity or heritability.

At present, little is known about the mechanisms by which this diversity is achieved and whether such diversity is itself heritable, reflecting a permanent détente that is achieved through the loss of cross-lineage counter-regulatory mechanisms. The complex network of epigenetic processes and feed-forward networks that are evident in canonical T<sub>H</sub>-cell subsets indicate that lineage fidelity could be tweaked at many points to achieve diversity and maintain a desirable degree of plasticity.


**Concluding remarks**

There is now a compelling body of evidence showing that the state of chromatin and DNA methylation at lineage-restricted cytokine and transcription factor genes, as well as their regulatory elements in T<sub>H</sub> cells, both reflects and affects their functions in transcription. There is also

increasing clarity regarding the specific epigenetic modifications that are associated with active and accessible, inactive but poised, and silenced loci, but the extent to which these modifications work in a combinatorial manner to encode and finely tune transcription is not yet completely clear.

A future challenge will be to determine with greater clarity how specific combinations of epigenetic modifications are established by networks of lineage-specifying transcription factors, and whether, when and how they can later be removed or selectively modified to achieve or alter T<sub>H</sub>-lineage specification. Such analyses should help to unravel the basis for the non-canonical patterns of cytokines that are expressed by T<sub>H</sub> cells *in vivo*, and whether these patterns reflect residual plasticity of cells within the main T<sub>H</sub> lineages, stable subsets within these lineages or a snapshot in time of cells in the process of shifting from one lineage to another. New, high-resolution and comprehensive approaches provide tools to probe for these mechanisms. Initial insights from such studies show that total human resting CD4<sup>+</sup> T cells exhibit molecular signatures of open chromatin at the *Ifng* locus (FIG. 4) that resemble those found in mouse and human T<sub>H</sub>1 cells<sup>82</sup>. This indicates that resting memory T cells maintain epigenetic signatures of previous events while retaining plasticity<sup>136</sup>. Whether these epigenetic marks are subject to further modulation in new environments or are derived solely from a subset of committed cells within a larger uncommitted population that lacks these marks is unclear. Development of ways to apply comprehensive, systems-based approaches to specific cell subsets as they arise *in vivo* will be important for our understanding of their inherent diversity and of processes by which this diversity is achieved.

**Note added in proof**

Two reports provide additional insights into T<sub>H</sub>-cell lineage commitment and plasticity. Lee *et al.*<sup>144</sup> showed that even after 4 weeks in polarizing culture conditions, T<sub>H</sub>17 cells produce IL-17 and IFNγ or IFNγ alone if they are not continually exposed to TGFβ. Wei *et al.*<sup>145</sup> mapped permissive H3K4 trimethylation and repressive H3K27 trimethylation in the genome of several CD4<sup>+</sup> T-cell subsets and showed that the promoters of the genes encoding T-bet and GATA3 have both of these chromatin markers in T<sub>H</sub>17 and T<sub>Reg</sub> cells. They also provided evidence suggesting that this bivalent epigenetic state may allow these cells to activate parts of the T<sub>H</sub>1 and T<sub>H</sub>2 transcriptional programme in response to alterations in the types of cytokine to which they are exposed.

1. Mosmann, T. R., Cherwinski, H., Bond, M. W., Giedlin, M. A. & Coffman, R. L. Two types of murine helper T cell clone. I. Definition according to profiles of lymphokine activities and secreted proteins. *J. Immunol.* **136**, 2348–2357 (1986).
2. Heinzel, F. P., Sadick, M. D., Holaday, B. J., Coffman, R. L. & Locksley, R. M. Reciprocal expression of interferon gamma or interleukin 4 during the resolution or progression of murine leishmaniasis. Evidence for expansion of distinct helper T cell subsets. *J. Exp. Med.* **169**, 59–72 (1989).
3. Ouyang, W., Kolls, J. K. & Zheng, Y. The biological functions of T helper 17 cell effector cytokines in inflammation. *Immunity* **28**, 454–467 (2008).
4. Murphy, K. M. & Reiner, S. L. The lineage decisions of helper T cells. *Nature Rev. Immunol.* **2**, 933–944 (2002).
5. Ansel, K. M., Djuretic, I., Tanasa, B. & Rao, A. Regulation of Th2 differentiation and *Il4* locus accessibility. *Annu. Rev. Immunol.* **24**, 607–656 (2006).
6. Lee, G. R., Kim, S. T., Spilianakis, C. G., Fields, P. E. & Flavell, R. A. T helper cell differentiation: regulation by *cis* elements and epigenetics. *Immunity* **24**, 369–379 (2006).
7. Ledford, H. Language: disputed definitions. *Nature* **455**, 1023–1028 (2008).
8. Merkenschlager, M. & Wilson, C. B. RNAi and chromatin in T cell development and function. *Curr. Opin. Immunol.* **20**, 131–138 (2008).
9. Anderson, P. Post-transcriptional control of cytokine production. *Nature Immunol.* **9**, 353–359 (2008).
10. Baltimore, D., Boldin, M. P., O’Connell, R. M., Rao, D. S. & Taganov, K. D. MicroRNAs: new regulators of immune cell development and function. *Nature Immunol.* **9**, 839–845 (2008).
11. Nurieva, R. I. *et al.* Generation of T follicular helper cells is mediated by interleukin-21 but independent of T helper 1, 2, or 17 cell lineages. *Immunity* **29**, 138–149 (2008).
12. Djuretic, I. M. *et al.* Transcription factors T-bet and Runx3 cooperate to activate *Ifng* and silence *Il4* in T helper type 1 cells. *Nature Immunol.* **8**, 145–153 (2007). This study shows that T-bet and RUNX3 together bind to the *Ifng* promoter to activate IFNγ expression and to the *Il4* silencer in the T<sub>H</sub>2-cytokine locus to silence IL-4 expression and, consequently, T<sub>H</sub>2-cell differentiation.
13. Mullen, A. C. *et al.* Hlx is induced by and genetically interacts with T-bet to promote heritable T<sub>H</sub>1 gene induction. *Nature Immunol.* **3**, 652–658 (2002).
14. Schoenborn, J. R. & Wilson, C. B. Regulation of interferon-γ during innate and adaptive immune responses. *Adv. Immunol.* **96**, 41–101 (2007).
15. Townsend, M. J. *et al.* T-bet regulates the terminal maturation and homeostasis of NK and Va14i NKT cells. *Immunity* **20**, 477–494 (2004).
16. Hwang, E. S., Szabo, S. J., Schwartzberg, P. L. & Glimcher, L. H. T helper cell fate specified by kinase-mediated interaction of T-bet with GATA-3. *Science* **307**, 430–433 (2005).
17. Usui, T. *et al.* T-bet regulates Th1 responses through essential effects on GATA-3 function rather than on *IFNG* gene acetylation and transcription. *J. Exp. Med.* **203**, 755–766 (2006).
18. Amsen, D. *et al.* Direct regulation of Gata3 expression determines the T helper differentiation potential of Notch. *Immunity* **27**, 89–99 (2007).
19. van Panhuys, N. *et al.* In vivo studies fail to reveal a role for IL-4 or STAT6 signaling in Th2 lymphocyte differentiation. *Proc. Natl Acad. Sci. USA* **105**, 12423–12428 (2008).
20. Ivanov, I. I., Zhou, L. & Littman, D. R. Transcriptional regulation of Th17 cell differentiation. *Semin. Immunol.* **19**, 409–417 (2007).
21. McGechy, M. J. & Cua, D. J. Th17 cell differentiation: the long and winding road. *Immunity* **28**, 445–453 (2008).
22. Zhou, L. *et al.* TGF-β-induced Foxp3 inhibits T<sub>H</sub>17 cell differentiation by antagonizing RORγt function. *Nature* **453**, 236–240 (2008).
23. Dong, C. T<sub>H</sub>17 cells in development: an updated view of their molecular identity and genetic programming. *Nature Rev. Immunol.* **8**, 337–348 (2008).
24. Yang, X. O. *et al.* T helper 17 lineage differentiation is programmed by orphan nuclear receptors RORα and RORγ. *Immunity* **28**, 29–39 (2008).
25. Korn, T. *et al.* IL-21 initiates an alternative pathway to induce proinflammatory T<sub>H</sub>17 cells. *Nature* **448**, 484–487 (2007).
26. Nurieva, R. *et al.* Essential autocrine regulation by IL-21 in the generation of inflammatory T cells. *Nature* **448**, 480–483 (2007).
27. Onoda, T. *et al.* Human CD4<sup>+</sup> central and effector memory T cells produce IL-21: effect on cytokine-driven proliferation of CD4<sup>+</sup> T cell subsets. *Int. Immunol.* **19**, 1191–1199 (2007).
28. Zhou, L. *et al.* IL-6 programs T<sub>H</sub>-17 cell differentiation by promoting sequential engagement of the IL-21 and IL-23 pathways. *Nature Immunol.* **8**, 967–974 (2007).
29. Quintana, F. J. *et al.* Control of T<sub>reg</sub> and T<sub>H</sub>17 cell differentiation by the aryl hydrocarbon receptor. *Nature* **453**, 65–71 (2008).
30. Veldhoen, M. *et al.* The aryl hydrocarbon receptor links T<sub>H</sub>17-cell-mediated autoimmunity to environmental toxins. *Nature* **453**, 106–109 (2008).
31. Manel, N., Unutmaz, D. & Littman, D. R. The differentiation of human T<sub>H</sub>-17 cells requires transforming growth factor-β and induction of the nuclear receptor RORγt. *Nature Immunol.* **9**, 641–649 (2008).
32. Volpe, E. *et al.* A critical function for transforming growth factor-β, interleukin 23 and proinflammatory cytokines in driving and modulating human T<sub>H</sub>-17 responses. *Nature Immunol.* **9**, 650–657 (2008).
33. Berger, S. L. The complex language of chromatin regulation during transcription. *Nature* **447**, 407–412 (2007).
34. Kouzarides, T. Chromatin modifications and their function. *Cell* **128**, 693–705 (2007).
35. Rowell, E., Merkenschlager, M. & Wilson, C. B. Long range regulation of cytokine gene expression.
36. Ruthenburg, A. J., Li, H., Patel, D. J. & Allis, C. D. Multivalent engagement of chromatin modifications by linked binding modules. *Nature Rev. Mol. Cell Biol.* **8**, 983–994 (2007).
37. Wilson, C. B., Makar, K. W., Shnyreva, M. & Fitzpatrick, D. R. DNA methylation and the expanding epigenetics of T cell lineage commitment. *Semin. Immunol.* **17**, 105–119 (2005).
38. Jones, B. & Chen, J. Inhibition of IFN-γ transcription by site-specific methylation during T helper cell development. *EMBO J.* **25**, 2443–2452 (2006).
39. Schubeler, D. *et al.* Genomic targeting of methylated DNA: influence of methylation on transcription, chromatin structure, and histone acetylation. *Mol. Cell. Biol.* **20**, 9103–9112 (2000).
40. Lee, D. U., Agarwal, S. & Rao, A. Th2 lineage commitment and efficient IL-4 production involves extended demethylation of the *IL-4* gene. *Immunity* **16**, 649–660 (2002).
41. Meissner, A. *et al.* Genome-scale DNA methylation maps of pluripotent and differentiated cells. *Nature* **454**, 766–770 (2008).

NATURE REVIEWS | IMMUNOLOGY

VOLUME 9 | FEBRUARY 2009 | 103

© 2009 Macmillan Publishers Limited. All rights reserved

42. Mohn, F. *et al.* Lineage-specific polycomb targets and *de novo* DNA methylation define restriction and potential of neuronal progenitors. *Mol. Cell* **30**, 755–766 (2008).

43. Trojer, P. & Reinberg, D. Facultative heterochromatin: is there a distinctive molecular signature? *Mol. Cell* **28**, 1–13 (2007).

44. Wang, Z. *et al.* Combinatorial patterns of histone acetylations and methylations in the human genome. *Nature Genet.* **40**, 897–903 (2008).

45. Azuara, V. *et al.* Chromatin signatures of pluripotent cell lines. *Nature Cell Biol.* **8**, 532–538 (2006).

46. Barski, A. *et al.* High-resolution profiling of histone methylations in the human genome. *Cell* **129**, 823–837 (2007).

References 44 and 46 provide a database of 39 different histone modifications, through which common sets of modifications that are typical of active and inactive promoters and enhancers can be deduced.

47. Ballas, Z. I. The use of 5-azacytidine to establish constitutive interleukin 2-producing clones of the EL4 thymoma. *J. Immunol.* **133**, 7–9 (1984).

48. Young, H. A. *et al.* Differentiation of the T helper phenotypes by analysis of the methylation state of the IFN-gamma gene. *J. Immunol.* **153**, 3603–3610 (1994).

49. Bird, J. J. *et al.* Helper T cell differentiation is controlled by the cell cycle. *Immunity* **9**, 229–237 (1998).

50. Valapour, M. *et al.* Histone deacetylation inhibits IL4 gene expression in T cells. *J. Allergy Clin. Immunol.* **109**, 238–245 (2002).

51. Hutchins, A. S. *et al.* Gene silencing quantitatively controls the function of a developmental trans-activator. *Mol. Cell* **10**, 81–91 (2002). This study shows that the MBD2 is present at the *Il4* gene in T~H~1 cells and can be displaced by enforced expression of GATA3. MBD2-deficient mice showed modestly increased expression of IFNγ and T~H~2-type cytokines in the appropriate T-cell lineages, but T~H~2 cells from these mice expressed IFNγ and T~H~1 cells expressed Th2-type cytokines.

52. Lee, P. P. *et al.* A critical role for Dnmt1 and DNA methylation in T cell development, function, and survival. *Immunity* **15**, 763–774 (2001).

53. Makar, K. W. *et al.* Active recruitment of DNA methyltransferases regulates interleukin 4 in thymocytes and T cells. *Nature Immunol.* **4**, 1183–1190 (2003).

54. Makar, K. W. & Wilson, C. B. DNA methylation is a non-redundant repressor of the Th2 effector program. *J. Immunol.* **173**, 4402–4406 (2004). References 53 and 54 show that the DNA methyltransferase DNMT1 is rapidly excluded from the *Il4* gene and CNS2 in T~H~2 cells. Conditional ablation of DNMT1 leads to modestly increased expression of IFNγ and T~H~2-type cytokines in the appropriate T-cell lineages, expression of IFNγ in T~H~2 cells and marked expression of T~H~2-type cytokines in T~H~1 and CD8+ T cells.

55. Zhang, F. & Boothby, M. T helper type 1-specific Brg1 recruitment and remodeling of nucleosomes positioned at the IFN-γ promoter are Stat4 dependent. *J. Exp. Med.* **203**, 1493–1505 (2006).

56. Yamashita, M. *et al.* Crucial role of MLL for the maintenance of memory T helper type 2 cell responses. *Immunity* **24**, 611–622 (2006). This study shows that the histone H3K4 methyltransferase MLL is recruited to *Gata3* and *Il4* in T~H~2 cells and is required to maintain but not to establish GATA3 and IL-4 expression and T~H~2-cell differentiation.

57. Kimura, M. *et al.* Regulation of Th2 cell differentiation by *mel-18*, a mammalian polycomb group gene. *Immunity* **15**, 275–287 (2001).

58. Takemoto, N. *et al.* Th2-specific DNase I-hypersensitivity sites in the murine IL-13 and IL-4 intergenic region. *Int. Immunol.* **10**, 1981–1985 (1998).

59. Agarwal, S. & Rao, A. Modulation of chromatin structure regulates cytokine gene expression during T cell differentiation. *Immunity* **9**, 765–775 (1998).

60. Amsen, D. *et al.* Instruction of distinct CD4 T helper cell fates by different Notch ligands on antigen-presenting cells. *Cell* **117**, 515–526 (2004).

61. Tanaka, S. *et al.* The interleukin-4 enhancer CNS-2 is regulated by Notch signals and controls initial expression in NKT cells and memory-type CD4 T cells. *Immunity* **24**, 689–701 (2006).

62. Grogan, J. L. *et al.* Early transcription and silencing of cytokine genes underlie polarization of T helper cell subsets. *Immunity* **14**, 205–215 (2001).

63. Mullen, A. C. *et al.* Cell cycle controlling the silencing and functioning of mammalian activators. *Curr. Biol.* **11**, 1695–1699 (2001).

64. Fields, P. E., Lee, G. R., Kim, S. T., Bartsevich, V. V. & Flavell, R. A. Th2-specific chromatin remodeling and enhancer activity in the Th2 cytokine locus control region. *Immunity* **21**, 865–876 (2004). This report shows that the 3′ end of *Rad50* contains clusters of DNasel hypersensitive sites, hyperacetylated histones and demethylated DNA in T~H~2 cells, and that these sites have enhancer activity *in vivo* and together are sufficient to function as an LCR.

65. Lee, D. U. & Rao, A. Molecular analysis of a locus control region in the T helper 2 cytokine gene cluster: a target for STAT6 but not GATA3. *Proc. Natl Acad. Sci. USA* **101**, 16010–16015 (2004). This report identifies DNasel hypersensitive sites in the T~H~2-cytokine LCR, although there are some differences regarding the specificity of the sites for distinct CD4+ T-cell subsets between this study and reference 64.

66. Koyanagi, M. *et al.* EZH2 and histone 3 trimethyl lysine 27 associated with *Il4* and *Il13* gene silencing in T~H~1 cells. *J. Biol. Chem.* **280**, 31470–31477 (2005).

67. Baguet, A. & Bix, M. Chromatin landscape dynamics of the *Il4–Il13* locus during T helper 1 and 2 development. *Proc. Natl Acad. Sci. USA* **101**, 11410–11415 (2004).

68. Yamashita, M. *et al.* Essential role of GATA3 for the maintenance of type 2 helper T (Th2) cytokine production and chromatin remodeling at the Th2 cytokine gene loci. *J. Biol. Chem.* **279**, 26983–26990 (2004).

69. Tykocinski, L. O. *et al.* A critical control element for interleukin-4 memory expression in T helper lymphocytes. *J. Biol. Chem.* **280**, 28177–28185 (2005).

70. Avni, O. *et al.* T~H~ cell differentiation is accompanied by dynamic changes in histone acetylation of cytokine genes. *Nature Immunol.* **3**, 643–651 (2002).

71. Fields, P. E., Kim, S. T. & Flavell, R. A. Cutting Edge: changes in histone acetylation at the *IL-4* and *IFN-γ* loci accompany Th1/Th2 differentiation. *J. Immunol.* **169**, 647–650 (2002).

72. Zhu, J., Cote-Sierra, J., Guo, L. & Paul, W. E. Stat5 activation plays a critical role in Th2 differentiation. *Immunity* **19**, 739–748 (2003).

73. Lee, H. J. *et al.* GATA-3 induces T helper cell type 2 (Th2) cytokine expression and chromatin remodeling in committed Th1 cells. *J. Exp. Med.* **192**, 105–115 (2000).

74. Ouyang, W. *et al.* Stat6-independent GATA-3 autoactivation directs IL-4-independent Th2 development and commitment. *Immunity* **12**, 27–37 (2000).

75. Kim, S. T., Fields, P. E. & Flavell, R. A. Demethylation of a specific hypersensitive site in the Th2 locus control region. *Proc. Natl Acad. Sci. USA* **104**, 17052–17057 (2007).

76. Wurster, A. L. & Pazin, M. J. BRG1-mediated chromatin remodeling regulates differentiation and gene expression of T helper cells. *Mol. Cell. Biol.* **28**, 7274–7285 (2008).

77. Asnagli, H., Afkarian, M. & Murphy, K. M. Cutting Edge: identification of an alternative GATA-3 promoter directing tissue-specific gene expression in mouse and human. *J. Immunol.* **168**, 4268–4271 (2002).

78. Acosta-Rodriguez, E. V. *et al.* Surface phenotype and antigenic specificity of human interleukin 17-producing T helper memory cells. *Nature Immunol.* **8**, 639–646 (2007).

79. Schoenborn, J. R. *et al.* Comprehensive epigenetic profiling identifies multiple distal regulatory elements directing transcription of the gene encoding interferon-γ. *Nature Immunol.* **8**, 732–742 (2007). This study maps DNasel hypersensitive sites, histone modifications and DNA methylation on the mouse *Ifng* locus and shows the structural divergence of part of this locus ~70 kilobases upstream of *Ifng* in rodents compared with humans and other mammals.

80. Hatton, R. D. *et al.* A distal conserved sequence element controls *Ifng* gene expression by T cells and NK cells. *Immunity* **25**, 717–729 (2006). This report identifies several distal regulatory elements upstream of *Ifng* based on sequence conservation and the presence of favourable histone modifications. It also shows that T-bet binds to several of these sites, one of which (CNS-22) is crucial for expression of *Ifng* from a BAC transgene.

81. Soutto, M., Zhou, W. & Aune, T. M. Cutting Edge: distal regulatory elements are required to achieve selective expression of IFN-γ in Th1/Tc1 effector cells. *J. Immunol.* **169**, 6664–6667 (2002).

82. Boyle, A. P. *et al.* High-resolution mapping and characterization of open chromatin across the genome. *Cell* **132**, 311–322 (2008). This study provides a comprehensive, genome-wide map of DNasel hypersensitive sites in resting human CD4+ T cells and compares the locations of these sites with the histone modifications that were identified by reference 46.

83. Agarwal, S. & Rao, A. Long-range transcriptional regulation of cytokine gene expression. *Curr. Opin. Immunol.* **10**, 345–352 (1998).

84. Lee, D. U., Avni, O., Chen, L. & Rao, A. A distal enhancer in the interferon-γ (IFN-γ) locus revealed by genome sequence comparison. *J. Biol. Chem.* **279**, 4802–4810 (2004).

85. Shnyreva, M. *et al.* Evolutionarily conserved sequence elements that positively regulate IFN-γ expression in T cells. *Proc. Natl Acad. Sci. USA* **101**, 12622–12627 (2004).

86. Chang, S. & Aune, T. M. Dynamic changes in histone-methylation ‘marks’ across the locus encoding interferon-γ during the differentiation of T helper type 2 cells. *Nature Immunol.* **8**, 723–731 (2007).

87. Shi, M., Lin, T. H., Appell, K. C. & Berg, L. J. Janus-kinase-3-dependent signals induce chromatin remodeling at the *Ifng* locus during T helper 1 cell differentiation. *Immunity* **28**, 763–773 (2008).

88. Thieu, V. T. *et al.* Signal transducer and activator of transcription 4 is required for the transcription factor Tbet to promote T helper 1 cell-fate determination. *Immunity* **29**, 679–690 (2008).

89. Yang, Y., Ochando, J. C., Bromberg, J. S. & Ding, Y. Identification of a distant T-bet enhancer responsive to IL-12/Stat4 and IFNγ/Stat1 signals. *Blood* **110**, 2494–2500 (2007).

90. Mullen, A. C. *et al.* Role of T-bet in commitment of T~H~1 cells before IL-12-dependent selection. *Science* **292**, 1907–1910 (2001).

91. Way, S. S., Havenar-Daughton, C., Kolumam, G. A., Orgun, N. N. & Murali-Krishna, K. IL-12 and type-I IFN synergize for IFN-γ production by CD4 T cells, whereas neither are required for IFN-γ production by CD8 T cells after *Listeria monocytogenes* infection. *J. Immunol.* **178**, 4498–4505 (2007).

92. Cai, G., Radzanowski, T., Villegas, E. N., Kastelein, R. & Hunter, C. A. Identification of STAT4-dependent and independent mechanisms of resistance to *Toxoplasma gondii*. *J. Immunol.* **165**, 2619–2627 (2000).

93. Kaplan, M. H., Sun, Y. L., Hoey, T. & Grusby, M. J. Impaired IL-12 responses and enhanced development of Th2 cells in Stat4-deficient mice. *Nature* **382**, 174–177 (1996).

94. Chang, S. & Aune, T. M. Histone hyperacetylated domains across the *Ifng* gene region in natural killer cells and T cells. *Proc. Natl Acad. Sci. USA* **102**, 17095–17100 (2005).

95. Beima, K. M. *et al.* T-bet binding to newly identified target gene promoters is cell type-independent but results in variable context-dependent functional effects. *J. Biol. Chem.* **281**, 11992–12000 (2006).

96. Naoe, Y. *et al.* Repression of interleukin-4 in T helper type 1 cells by Runx/Cbf β binding to the *Il4* silencer. *J. Exp. Med.* **204**, 1749–1755 (2007).

97. Tong, Y., Aune, T. & Boothby, M. T-bet antagonizes mSin3a recruitment and transactivates a fully methylated IFN-γ promoter via a conserved T-box half-site. *Proc. Natl Acad. Sci. USA* **102**, 2034–2039 (2005).

98. Chen, G. Y., Osada, H., Santamaria-Babi, L. F. & Kannagi, R. Interaction of GATA-3/T-bet transcription factors regulates expression of sialyl Lewis X homing receptors on Th1/Th2 lymphocytes. *Proc. Natl Acad. Sci. USA* **103**, 16894–16899 (2006).

99. Miller, S. A., Huang, A. C., Miazgowicz, M. M., Brassil, M. M. & Weinmann, A. S. Coordinated, but physically separable interaction with H3K27-demethylase and H3K4-methyltransferase activities are required for T-box protein-mediated activation of developmental gene expression. *Genes Dev.* **22**, 2980–2993 (2008). This study shows that T-bet interacts with an H3K27 demethylase and can recruit it to target genes to erase this repressive histone modification, and at the same time can recruit an H3K4 methyltransferase to add this permissive histone modification.

100. Szabo, S. J. *et al.* A novel transcription factor, Tbet, directs Th1 lineage commitment. *Cell* **100**, 655–669 (2000).

101. Akimzhanov, A. M., Yang, X. O. & Dong, C. Chromatin remodeling of interleukin-17 (IL-17)–IL-17F cytokine gene locus during inflammatory helper T cell differentiation. *J. Biol. Chem.* **282**, 5969–5972 (2007).

102. Wei, L., Laurence, A., Elias, K. M. & O’Shea J., J. IL-21 is produced by TH17 cells and drives IL-17 production in a STAT3-dependent manner. *J. Biol. Chem.* **322**, 34605–34610 (2007).

103. Decker, J. Gene regulation in the third dimension. *Science* **319**, 1793–1794 (2008).

104. Misteli, T. Beyond the sequence: cellular organization of genome function. *Cell* **128**, 787–800 (2007).

105. Williams, A. & Flavell, R. A. The role of CTCF in regulating nuclear organization. *J. Exp. Med.* **205**, 747–750 (2008).

106. Spilianakis, C. G. & Flavell, R. A. Long-range intrachromosomal interactions in the T helper type 2 cytokine locus. *Nature Immunol.* **5**, 1017–1027 (2004). This report shows for the first time the existence of long-range intrachromosomal interactions in the T<sub>H</sub>2-cytokine locus and the importance of STAT6 and GATA3 in the establishment of these interactions.

107. Cai, S., Lee, C. C. & Kohwi-Shigematsu, T. SATB1 packages densely looped, transcriptionally active chromatin for coordinated expression of cytokine genes. *Nature Genet.* **38**, 1278–1288 (2006). This study identifies many SATB1 binding sites in the T<sub>H</sub>2-cytokine locus in T<sub>H</sub>2 cells, and shows that activation-induced, SATB1-dependent chromatin looping of the T<sub>H</sub>2-cytokine locus is important for T<sub>H</sub>2-type cytokine expression.

108. Filippova, G. N. Genetics and epigenetics of the multifunctional protein CTCF. *Curr. Top. Dev. Biol.* **80**, 337–360 (2008).

109. Parelho, V. *et al.* Cohesins functionally associate with CTCF on mammalian chromosome arms. *Cell* **132**, 422–433 (2008). This study shows that cohesins colocalize throughout the genome with CTCF and contribute to the insulator function of CTCF in model systems. Cohesin and CTCF binding sites are found at the boundary and within the *Ifng* gene and do not to interfere with its transcription.

110. Spilianakis, C. G., Lalioti, M. D., Town, T., Lee, G. R. & Flavell, R. A. Interchromosomal associations between alternatively expressed loci. *Nature* **435**, 637–645 (2005). This report describes long-range interchromosomal interactions between the T<sub>H</sub>2-cytokine and *Ifng* loci in naive T cells and suggests that these interactions may poise these loci for expression during T<sub>H</sub>2- or T<sub>H</sub>1-cell differentiation.

111. Jacob, E., Hod-Dvorai, R., Schif-Zuck, S. & Avni, O. Unconventional association of the polycomb group proteins with cytokine genes in differentiated T helper cells. *J. Biol. Chem.* **283**, 13471–13481 (2008).

112. Miller, A. T., Wilcox, H. M., Lai, Z. & Berg, L. J. Signaling through Itk promotes T helper 2 differentiation via negative regulation of T-bet. *Immunity* **21**, 67–80 (2004).

113. Yu, Q., Thieu, V. T. & Kaplan, M. H. Stat4 limits DNA methyltransferase recruitment and DNA methylation of the IL-18Rα gene during Th1 differentiation. *EMBO J.* **26**, 2052–2060 (2007).

114. Ooi, S. K. *et al.* DNMT3L connects unmethylated lysine 4 of histone H3 to *de novo* methylation of DNA. *Nature* **448**, 714–717 (2007).

115. Cimmino, L. *et al.* Blimp-1 attenuates Th1 differentiation by repression of *Ifng*, *Tbx21*, and *Bcl6* gene expression. *J. Immunol.* **181**, 2338–2347 (2008).

116. Pai, S. Y., Truitt, M. L. & Ho, I. C. GATA-3 deficiency abrogates the development and maintenance of T helper type 2 cells. *Proc. Natl Acad. Sci. USA* **101**, 1993–1998 (2004).

117. Zhu, J. *et al.* Conditional deletion of Gata3 shows its essential function in T<sub>H</sub>1-T<sub>H</sub>2 responses. *Nature Immunol.* **5**, 1157–1165 (2004). This report shows that GATA3 is essential for the initiations of T<sub>H</sub>2-cell differentiation and contributes to but is not essential for its maintenance.

118. Martins, G. A., Hutchins, A. S. & Reiner, S. L. Transcriptional activators of helper T cell fate are required for establishment but not maintenance of signature cytokine expression. *J. Immunol.* **175**, 5981–5985 (2005).

119. Taniuchi, I. *et al.* Differential requirements for Runx proteins in CD4 repression and epigenetic silencing during T lymphocyte development. *Cell* **111**, 621–633 (2002).

120. Ansel, K. M. *et al.* Deletion of a conserved *Il4* silencer impairs T helper type 1-mediated immunity. *Nature Immunol.* **5**, 1251–1259 (2004).

121. Laurence, A. *et al.* Interleukin-2 signaling via STAT5 constrains T helper 17 cell generation. *Immunity* **26**, 371–381 (2007).

122. Gorelik, L., Constant, S. & Flavell, R. A. Mechanism of transforming growth factor β-induced inhibition of T helper type 1 differentiation. *J. Exp. Med.* **195**, 1499–1505 (2002).

123. Gorelik, L., Fields, P. E. & Flavell, R. A. Cutting Edge: TGF-β inhibits Th type 2 development through inhibition of GATA-3 expression. *J. Immunol.* **165**, 4773–4777 (2000).

124. Yu, J. *et al.* Pro- and antiinflammatory cytokine signaling: reciprocal antagonism regulates interferon-γ production by human natural killer cells. *Immunity* **24**, 575–590 (2006).

125. Chen, Y. *et al.* Anti-IL-23 therapy inhibits multiple inflammatory pathways and ameliorates autoimmune encephalomyelitis. *J. Clin. Invest.* **116**, 1317–1326 (2006).

126. Laurence, A. & O’Shea, J. J. T<sub>H</sub>-17 differentiation: of mice and men. *Nature Immunol.* **8**, 903–905 (2007).

127. Wilson, N. J. *et al.* Development, cytokine profile and function of human interleukin 17-producing helper T cells. *Nature Immunol.* **8**, 950–957 (2007).

128. Annunziato, F. *et al.* Phenotypic and functional features of human Th17 cells. *J. Exp. Med.* **204**, 1849–1861 (2007).

129. Zheng, Y. *et al.* Interleukin-22, a T<sub>H</sub>17 cytokine, mediates IL-23-induced dermal inflammation and acanthosis. *Nature* **445**, 648–651 (2007).

130. Harrington, L. E. *et al.* Interleukin 17-producing CD4<sup>+</sup> effector T cells develop via a lineage distinct from the T helper type 1 and 2 lineages. *Nature Immunol.* **6**, 1123–1132 (2005).

131. Ivanov, I. *et al.* The orphan nuclear receptor RORγt directs the differentiation program of proinflammatory IL-17<sup>+</sup> T helper cells. *Cell* **126**, 1121–1133 (2006).

132. Singh, S. P., Zhang, H. H., Foley, J. F., Hedrick, M. N. & Farber, J. M. Human T cells that are able to produce IL-17 express the chemokine receptor CCR6. *J. Immunol.* **180**, 214–221 (2008).

133. McGeachy, M. J. *et al.* TGF-β and IL-6 drive the production of IL-17 and IL-10 by T cells and restrain T<sub>H</sub>-17 cell-mediated pathology. *Nature Immunol.* **8**, 1390–1397 (2007).

134. O’Garra, A. & Vieira, P. T<sub>H</sub>1 cells control themselves by producing interleukin-10. *Nature Rev. Immunol.* **7**, 425–428 (2007).

135. Krawczyk, C. M., Shen, H. & Pearce, E. J. Functional plasticity in memory T helper cell responses. *J. Immunol.* **187**, 4080–4088 (2007).

136. Messi, M. *et al.* Memory and flexibility of cytokine gene expression as separable properties of human T<sub>H</sub>1 and T<sub>H</sub>2 lymphocytes. *Nature Immunol.* **4**, 78–86 (2003).

137. Sundrud, M. S. *et al.* Genetic reprogramming of primary human T cells reveals functional plasticity in Th cell differentiation. *J. Immunol.* **171**, 3542–3549 (2003).

138. Adeeku, E., Gudapati, P., Mendez-Fernandez, Y., Van Kaer, L. & Boothby, M. Flexibility accompanies commitment of memory CD4 lymphocytes derived from IL-4 locus-activated precursors. *Proc. Natl Acad. Sci. USA* **105**, 9307–9312 (2008).

139. Lohning, M. *et al.* Long-lived virus-reactive memory T cells generated from purified cytokine-secreting T helper type 1 and type 2 effectors. *J. Exp. Med.* **205**, 53–61 (2008).

140. Reiner, S. L., Sallusto, F. & Lanzavecchia, A. Division of labor with a workforce of one: challenges in specifying effector and memory T cell fate. *Science* **317**, 622–625 (2007).

141. Allis, C. D. *et al.* New nomenclature for chromatin-modifying enzymes. *Cell* **131**, 633–636 (2007).

142. Ruthenburg, A. J., Allis, C. D. & Wysocka, J. Methylation of lysine 4 on histone H3: intricacy of writing and reading a single epigenetic mark. *Mol. Cell* **25**, 15–30 (2007).

143. Yang, X. J. & Seto, E. The Rpd3/Hda1 family of lysine deacetylases: from bacteria and yeast to mice and men. *Nature Rev. Mol. Cell Biol.* **9**, 206–218 (2008).

144. Lee, Y. *et al.* Late developmental plasticity in the helper 17 lineage. *Immunity* 31 Dec 2008 (doi:10.1016/j.immuni.2008.11.005)

145. Wei, G. *et al.* Global mapping of histone H3 K4 and K27 trimethylation reveals specificity and plasticity in lineage fate determination of differentiating CD4<sup>+</sup> T cells. *Immunity* 15 Jan 2008 (doi:10.1016/j.immuni.2008.12.009)


**Acknowledgements**

We apologize to the colleagues whose work was not discussed owing to space limitations. Work from the authors’ laboratories was supported by the National Institutes of Health, USA (grant numbers R01-AI071272, R01-HD18184, N01-AI40069 and T32-AI07411).


**DATABASES**

Entrez Gene: [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gene](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gene)

AHR | GATA3 | HLX | IFNγ | IL-4 | IL-5 | IL-13 | IL-17A | IL-17F | Rad50 | RBPI | RUNX3 | RORγt | STAT4 | STAT5 | STAT6 | Tbx21 | TGFβ


**FURTHER INFORMATION**

Christopher B. Wilson’s homepage: [http://depts.washington.edu/immunweb/faculty/profiles/wilson.html](http://depts.washington.edu/immunweb/faculty/profiles/wilson.html)

The VISTA web site: [http://pipeline.lbl.gov/cgi-bin/gateway2](http://pipeline.lbl.gov/cgi-bin/gateway2)

ALL LINKS ARE ACTIVE IN THE ONLINE PDF
