
Available online at www.sciencedirect.com

**ScienceDirect**

**Current Opinion in Cell Biology**

**ATR/Mec1: coordinating fork stability and repair**

Anna M Friedel, Brietta L Pike and Susan M Gasser

During S phase, eukaryotic cells unwind and duplicate a tremendous amount of DNA, generating structures that are very sensitive to both endogenous and exogenous insults. The collision of DNA polymerases with damaged DNA or other obstructions to fork progression generates replication stress, which can evolve into fork collapse if the replisome components are not stabilized. To ensure genome integrity, stalled replication forks are recognized by a checkpoint, whose central player is the human kinase ATR or Mec1 in *S. cerevisiae*. This review will discuss recent findings revealing roles of the ATR/Mec1 kinase: both in stabilizing the replisome directly and in activating the checkpoint response to regulate origin firing, DNA repair, fork restart, and cell cycle progression.

**Address**

Friedrich Miescher Institute for Biomedical Research, Maulbeerstrasse 66, CH-4058 Basel, Switzerland

Corresponding author: Gasser, Susan M (susan.gasser@fmi.ch)

---

**Current Opinion in Cell Biology** 2009, **21**:237–244

This review comes from a themed issue on  
Cell regulation  
Edited by Brian Hemmings and Nikolas Tonks  

Available online 21st February 2009  

0955-0674/$ – see front matter  
© 2009 Elsevier Ltd. All rights reserved.

DOI **10.1016/j.ceb.2009.01.017**

---

**Introduction**

Maintaining genome integrity is crucial for all organisms. Eukaryotes have developed sophisticated control mechanisms called checkpoints that recognize DNA damage and act to slow progression through the cell cycle and coordinate repair. During S phase, cells are particularly sensitive to DNA insult, which can arise from genotoxic lesions (e.g., provoked by MMS) or by endogenous replication stress (e.g., fork delay), which can be exacerbated by treatment with hydroxyurea (HU). Consequently, additional checkpoint response mechanisms exist in S phase to deal with stalled replication forks and/or DNA damage. Central to checkpoint signal transduction pathways are the phosphoinositide 3-kinase (PI3K)-related protein kinases ATR and ATM, or their budding yeast homologs Mec1 and Tel1. While ATM/Tel1 is activated principally by DNA double-strand breaks (DSBs), ATR/Mec1 responds to a wide range of DNA damage, including replication stress, base adducts, UV-induced nucleotide damage, and DSBs [1]. This review will examine the roles of ATR/Mec1 as it interacts with fork components to stabilize the replisome, and as it activates the checkpoint signal transduction pathway that controls origin firing, DNA repair, and prevents early initiation of mitosis.

**Activation and recruitment of the checkpoint kinase ATR/Mec1**

The current model suggests that ATR/Mec1 does not recognize the primary lesion itself, but long stretches of single-stranded DNA (ssDNA), which are generated either by the functional uncoupling of replicative helicases and polymerases during fork stalling or by nucleolytic processing of DSBs (Figure 1) [2,3]. The regions of ssDNA are not naked but are coated by replication protein A (RPA). RPA itself is a target of ATR/Mec1, but also plays a central role in recruiting ATR/Mec1 via its co-factor, the ATR-interacting protein (ATRIP; Ddc2 in budding yeast, see Table 1) [4]. Visible ATRIP/Ddc2 foci demonstrate the accumulation of the complex at sites of repair [5,6]. However, RPA-coated ssDNA alone is not sufficient for ATR/Mec1 activation. It has been reported that primers created by Polymerase α (Polα)/primase are required for checkpoint activation in Xenopus egg extracts and that primed ssDNA with a free 5′ end, but not unprimed ssDNA, is needed to activate the downstream checkpoint kinase CHK1 [7–9].

Other factors have been implicated in activation of the ATR/Mec1 kinase. The 9-1-1 checkpoint clamp, a heterotrimer structurally related to PCNA but consisting of RAD9-RAD1-HUS1 in humans and Rad17-Mec3-Ddc1 in *S. cerevisiae*, is loaded onto primer-template junctions adjacent to RPA-coated ssDNA [1,10]. The clamp loader RAD17-RFC2-5 (human) or Rad24-Rfc2-5 (yeast) loads 9-1-1 independently of ATR-ATRIP, although recent data suggest that phosphorylation of RAD17 Ser635 and Ser645 by ATR affects the turnover of RAD9 molecules in foci and therefore contributes to 9-1-1 retention close to DNA lesions [11]. In humans, 9-1-1 then recruits the BRCT domain-containing topoisomerase-binding protein-1 (TOPBP1) that binds ATRIP and thus contributes to ATR activation [12,13]. By contrast, it has been reported that *S. cerevisiae* 9-1-1 can activate Mec1-Ddc2 directly in vitro [10]. Intriguingly, the co-localization of these sensors, 9-1-1 and Mec1/Ddc2, appears to be sufficient to activate the checkpoint even without induced damage [14].

Recently, two studies have shown that Dpb11 plays a role in checkpoint activation, acting like its human homolog TOPBP1 [15,16]. In this case however, the C-terminus

Figure 1

A moving replication fork

- Srs2
- Pol δ
- leading strand
- Pol ε
- PCNA
- RNA
- RPA
- lagging strand
- Pol α/prim
- Sgs1
- Csm3
- Tof1
- Mrc1
- MCM2-7, Cdc45, GINS, MCM10
- PCNA
- Pol δ

B stalled replication fork

- Srs2
- Pol δ
- PCNA
- RNA
- RPA
- Pol α/prim
- Sgs1
- Rad24-RFC
- RPA
- Mec1
- Ddc2
- Dpb11
- Chk1
- Rad53
- Exo1
- Pol ε
- PCNA
- Csm3
- Tof1
- Mrc1
- MCM2-7, Cdc45, GINS, MCM10

intra-S-phase checkpoint activation:
- slow down replication
- stabilize the replisome
- prevent late origins from firing
- prevent aberrant HR
- transcriptional regulation of damage-
- inducible genes

ATR roles at the replication fork Friedel, Pike and Gasser 239

Table 1

Conserved S. cerevisiae and H. sapiens checkpoint proteins and their functions.

| S. cerevisiae | H. sapiens | Function |
|---------------|------------|----------|
| RFA-ssDNA     | RPA-ssDNA  | Signal   |
| Rad24-RFC     | RAD17-RFC  | Sensor (RFC-like complex, clamp loader) |
| Ddc1-Rad17-Mec3 | RAD9-RAD1-HUS1 | Sensor (9-1-1, DNA damage checkpoint clamp) |
| Dpb11         | TOPBP1     | Sensor (required for ATR activation) |
| Mre11-Rad50-Xrs2 | MRE11-RAD50-NBS1 | Sensor (MRX/MRN) |
| Mec1/Ddc2     | ATR/ATRIP  | Sensor (signaling kinase) |
| Tel1          | ATM        | Sensor (signaling kinase) |
| Polε         | POLε       | Fork associated (leading strand polymerase) |
| Sgs1          | BLM, WRN   | Fork associated (RecQ helicase) |
| Tof1/Csm3     | TIM/TIPIN  | Fork associated |
| Ino80         | INO80      | Fork associated (chromatin remodeler) |
| Mrc1          | Claspin    | Mediator |
| Rad9          | BRCA1/53BP1 | Mediator |
| Rad53         | CHK2       | Effector (signaling kinase) |
| Chk1          | CHK1       | Effector (signaling kinase) |

of Dpb11, and not its BRCT repeats, interacts with Ddc2, leading to Mec1 activation. Moreover, Mordes *et al.* 2008 observed that Mec1-dependent phosphorylation of Dpb11 (Thr731) further enhances the ability of Dpb11 to amplify Mec1-Ddc2 activity. Additional research is needed to confirm exactly how Dpb11 is recruited to damage since it appears to colocalize with Polε during initiation, but not during elongation [17]. In one model, 9-1-1 and Mec1-Ddc2 are recruited independently of RPA-ssDNA, and Mec1 subsequently phosphorylates the Ddc1 subunit of 9-1-1. In an alternative model, 9-1-1 and Dpb11 act in parallel to activate Mec1-Ddc2. In this case Dpb11 could be recruited to RPA-ssDNA via its interaction with other proteins, for example, Polε or Sld2 and Sld3 [17–19]. Data from the Burgers laboratory has also suggested that Dpb11 and 9-1-1 can independently activate Mec1-Ddc2 *in vitro*, although synergism is observed when both activators are present [15].

Direct interaction of ATR/Mec1 with components of the replication fork

By studying mutations in the budding yeast Mec1, or in targets of Mec1, it is clear that ATR/Mec1 functions at stalled replication forks to keep replication polymerases engaged, and that this is largely separable from its role in activating downstream checkpoint kinases such as Rad53 [2,20] (see Table 2). For instance, even in the absence of exogenous genotoxic stress, *mec1* mutants accumulate gross spontaneous chromosomal rearrangements (GCRs)

at rates far higher than mutants lacking the downstream kinase Rad53, or in the adaptor proteins that function in the checkpoint response [21,22]. Spontaneous GCRs are thought to arise from inappropriate fork-associated recombination events. The direct role of Mec1 in maintaining DNA polymerases at forks has been demonstrated by Chromatin immunoprecipitation (ChIP) assays for both DNA Polα and Polε. Both polymerases are stably associated at forks stalled on HU for roughly an hour in a Mec1/Ddc2-dependent manner, whereas *rad53-11* or *rad53Δ* mutations had little or no effect on polymerase stability under identical conditions [20,23]. Moreover, ChIP has shown that Mec1/Ddc2 accumulates at stalled replication forks, which is not the case for the downstream kinase Rad53 [23]. A partial loss of function mutant, *mec1-100*, which compromises the S phase checkpoint but leaves the G2/M activation of Rad53 intact, provokes a partial loss of replicative polymerases at stalled forks, yet this mutation leads to a complete fork collapse when the RecQ helicase Sgs1 is deleted. The polymerase loss phenotype observed in *mec1-100 sgs1Δ* mutants correlates with compromised replication fork recovery and a dramatic increase in the rate of GCR. By contrast, cells lacking the Rad53 kinase show no loss of polymerases and no synergism with *sgs1Δ*, although there is significant displacement of the MCM helicase from stalled forks [23]. This latter may result from release of MCM from the replisome allowing extensive DNA unwinding and a subsequent collapse of forks in *rad53Δ* cells. Finally it

Model of a moving and stalled replication fork in S. cerevisiae. (a) At an unperturbed replication fork MCM helicases/Cdc45 unwind the parental DNA double strand. Polymerase ε (Polε) is responsible for leading strand synthesis, while polymerase α (Polα/prim) initiates Okazaki-fragment-synthesis at the lagging strand that is completed by polymerase δ (Polδ). (b) At a stalled replication fork single stranded DNA coated by RPA (RPA-ssDNA) accumulates owing to functional uncoupling of MCM/Cdc45 and the replicative polymerases. This triggers an S phase-specific checkpoint response, where 9-1-1 and the checkpoint kinase Mec1-Ddc2 are recruited independently to RPA-ssDNA. Subsequently, Mec1-Ddc2 is activated, leading to phosphorylation of various downstream targets including mediator proteins Mrc1 or Rad9, which contribute to the activation of the downstream effector kinases Rad53 and Chk1. Especially, Rad53 was shown to play a crucial role in stabilizing the replisome, preventing late origins from firing, preventing homologous recombination (HR), and mediating DNA repair. Furthermore, Mec1-Ddc2 was also shown to stabilize the replication fork by acting locally at the stalled fork.

Table 2

Contribution of Mec1/ATR versus Rad53/CHK1 to S phase checkpoint events.

| Mec1/ATR target | S phase checkpoint events | Rad53/CHK1 target |
|------------------|--------------------------|--------------------|
| Phosphorylation of Mrc1 | Slowed rate of replication | Downregulation of dNTP levels |
| Maintenance of polymerases and RPA | Stabilization of the replisome at stalled replication forks | Maintenance of the MCM complex |
| Sgs1/BLM (via Rad53) | Protection of stalled or damaged replication forks | Inhibition of Exo1 activity |
| (via Rad53) | Prevention/delay of late firing origins | Downregulation of Cdc7/Dbf4 activity |
| Locally: by MCM2 phosphorylation and Plx1 recruitment | Replication fork restart | Activation of Dun1 kinase |
|  |  | Globally: by Rad53-dephosphorylation |

has been shown that in a rad53Δ strain MMS-induced lethality, presumably due to fork collapse, can be suppressed by ablation of Exo1 that works together with Sgs1 to resect DSBs [24**,25,26], while mec1Δ defects cannot [27**]. Together, these observations provide compelling arguments that the ATR/Mec1 kinase has a crucial role in the maintenance of functional replication forks independent of its well characterized function in the activation of Rad53 (see Table 2).

In human cells, it has been shown not only that the RecQ Bloom's Syndrome helicase (BLM) is a target of ATR but also that ATR-dependent phosphorylation of residue Thr99 of BLM is required for efficient replication fork restart and suppression of new origin firing after aphidicolin treatment [28**]. Thus BLM helicase, like its yeast homolog, has both an anti-recombinase function to resolve deleterious strand invasion, in addition to a role as an ATR target in protecting stalled or damaged forks. It is unclear whether this also involves reversing fork regression, or possibly maintaining polymerases for fork restart. Finally, there is accumulating evidence that ATR and another human RecQ helicase, Werner’s syndrome helicase (WRN), work together to suppress fragile site instability [29]. The molecular mechanism remains unclear, but may involve recovery of stalled replication forks at fragile sites, which are prone to replisome pausing and/or strand breaks. However, unlike yeast, where deletion of the RecQ helicase is synergistic with mec1-100 [23], siRNA-mediated downregulation of human ATR and WRN suggests that these two activities are epistatic for fragile site suppression in the presence of low doses of aphidicolin [29].

The replication fork component Mrc1 is a key target of Mec1/Ddc2 and is important both for S phase checkpoint activation and for the stabilization of replication forks even in an unperturbed S phase. Mrc1 is not an essential protein, but several studies have shown that the rate of replication fork movement decreases significantly in mrc1Δ mutants in the absence of exogenous damage [30–32]. It was proposed that loss of Mrc1 results in uncoupling of the processive Cdc45/MCM helicase from the leading strand Pol ε [33**,34**], causing ssDNA to accumulate. Recently, a study from Petermann et al.

found that depletion of Claspin, the human Mrc1 homolog, similarly decreases replication fork progression [35]. Furthermore, Claspin-depletion and CHK1-depletion slow fork progression in an additive manner, suggesting that Claspin does not only mediate CHK1 activation but also stabilizes forks. A mutant form of the yeast protein in which all possible Mec1-target sites (SQ/TQ) have been changed to non-phosphorylatable AQ sites (mrc1AQ) has been used to distinguish the function of Mrc1 in replication from its role in Rad53 activation [31]. mrc1AQ mutants display normal rates of replication fork progression but fail to activate Rad53. This suggests that Mrc1 prevents fork collapse primarily through its presence at the fork and not through Rad53 activation [31].

Not only in the presence of HU but also in unperturbed cells, Mrc1 has been shown to interact via its N-terminal and C-terminal domains with the N-terminal and C-terminal domains of the Pol ε catalytic subunit [34**]. Phosphorylation, presumably by Mec1/Ddc2, disrupts the N-terminal interaction of Mrc1 with Pol ε, and this may be responsible for slowing replication fork progression in response to damage by binding and inhibiting other proteins at the fork, for example, the MCM helicase. Finally, the interaction of Mrc1’s C-terminus with the C-terminal domain of Pol ε is not affected during the S phase checkpoint, possibly providing a mechanism through which Mrc1 stabilizes the fork, that is, by anchoring Pol ε to DNA during HU-treatment. Indeed, the binding of yeast Pol ε to HU-stalled replication forks is decreased in mrc1Δ mutants and this again is highly synergistic with loss of Sgs1 [34**,36].

S phase checkpoint activation: the downstream effector kinases CHK1 and Rad53

Activated ATR/Mec1 gives rise to a global checkpoint response by phosphorylation and activation of numerous targets, including downstream effector kinases Chk1 and Rad53 in yeast, and CHK1 and CHK2 in humans. In particular, the kinases Rad53 in *S. cerevisiae* and human CHK1 are thought to be crucial for the S phase-specific checkpoint response. Mediator proteins like Rad9/53BP1 or Mrc1/Claspin contribute to the activation of downstream kinases, and the amplified signal leads to a

response that includes cell cycle arrest, activation of DNA repair, transcription of damage inducible genes, and S phase-specific mechanisms to prevent replication fork collapse on MMS and to prevent late origins from firing [37,38]. Importantly, replication fork stabilization enables the resumption of DNA synthesis once the stress is removed since *de novo* assembly of the pre-replication complex required for replication fork firing is not possible during S phase [1*,34**]. Phosphorylation of CHK1 by ATR leads to its activation and rapid release from chromatin, allowing amplification of the checkpoint signal through phosphorylation of multiple downstream targets. One is CDC25, which inhibits CDK activation to prevent entry into mitosis [1*]. Histone H3 Thr11 (H3-T11) is another target of chromatin-bound CHK1 as recently reported by Shimada *et al.*, and decreased phosphorylation after DNA damage correlates with transcriptional repression. They suggested the following mechanism for CHK1 function in damage-induced transcriptional repression: upon DNA damage CHK1 dissociates from chromatin and H3-T11 becomes hypophosphorylated, which impairs the recruitment of the histone acetyltransferase GCN5 and therefore leads to decreased histone H3K9 acetylation and thus reduced transcription [39].

Rad53 is the main effector of the *S. cerevisiae* S phase checkpoint response and its phosphorylation is sufficient and necessary for checkpoint activation [40]. It has been suggested that the primary role of Rad53 in response to MMS, UV, and IR is to prevent Exo1-dependent replication fork breakdown since it has been shown that deletion of *EXO1*, which encodes an exonuclease with a 5′-flap endonuclease activity, rescues the MMS, UV, and IR but not HU sensitivity of *rad53* mutants [27**]. This is especially interesting in connection with studies from three independent laboratories that report that Exo1 collaborates with Sgs1 to resect DNA DSBs [24**,25,26]. A two-step model for DSB end resection was proposed, in which Mre11-Rad50-Xrs2 (MRX) and Sae2 initiate 5′-strand resection and Sgs1 and/or Exo1 play a role in long-range resection to create ssDNA that serves as a substrate for Rad51 leading to homologous recombination. However, it is presently unknown how Exo1 contributes to replication fork integrity. It is conceivable that in the absence of Rad53, MMS treatment leads to excessive DNA unwinding due to MCM release or loss of replisome components and increased DNA breaks or DSB-like structures that are then processed by Exo1. Alternatively, the Lydall laboratory proposed that Rad53 may directly downregulate Exo1 activity [41], since Exo1 is phosphorylated in a Mec1/Rad53-dependent manner upon telomere uncapping or treatment with the break-inducing agents bleomycin or camptothecin. This suggests a negative feedback loop of Exo1 phosphorylation that limits ssDNA accumulation and checkpoint activation [41].

In addition to preventing replication fork collapse, another function of the S phase-specific checkpoint is to prevent late origin firing [42]. Indeed, DNA Pol α was shown to bind early but not late firing origins in wild-type cells treated with HU, while in the *rad53* mutant Pol α could be recovered efficiently at both [2,20]. A new study from Alvino *et al.* challenges the long standing model that Rad53 specifically inhibits late origin firing during a replication block [43*]. They suggest that late origin firing is not specifically impeded, but instead the entire S phase is delayed according to the speed of replication fork progression. Their model proposes that Rad53 downregulates the activity of the Cdc7/Dbf4 kinase that is required for replication initiation and thereby regulates origin firing by the rate of replication elongation while the order in which origins fire is maintained. Further research should reveal if this model is accurate, and if so how cells maintain the precise order of origin firing once the S phase checkpoint has been activated.

### Replication fork restart after DNA damage

Although S phase checkpoint activation has been studied intensively, not much is known about how cells inactivate the checkpoint once the replication stress is removed ('recovery') or how they downregulate the checkpoint when DNA repair fails ('adaptation'). Curiously, Trenz *et al.* recently demonstrated that ATR-dependent phosphorylation of MCM2 Ser92 in response to replication stress induces the binding of Polo-like kinase 1 (PLX1) to chromatin leading to origin firing in *Xenopus* egg extracts [44**]. This function was shown to be independent of PLX1-mediated phosphorylation of Claspin, which facilitates adaptation and the onset of mitosis after prolonged ATR activation. A model was suggested in which ATR-mediated MCM2 phosphorylation and subsequent PLX1 recruitment acts locally to activate dormant origins neighboring stalled replication forks. This mechanism disengages the global ATR/CHK1-dependent suppression of origin firing to complete replication in areas with replication fork stalling. This is consistent with the observation that PLX1-depletion leads to DSB accumulation when forks stall after aphidicolin treatment.

Consistent with Rad53 as the main S phase checkpoint effector, recovery and adaptation correlate with the disappearance of phosphorylated Rad53, largely due to the function of Ser/Thr phosphatases [40]. It has been proposed that Rad53 regulates fork restart and late origin firing by independent mechanisms. It is likely that different phosphatases could recognize specific Rad53 phosphorylation patterns and therefore control distinct checkpoint responses [45*,46]. New data suggest a mechanistic link between Rad53 deactivation by the Psy2-Pph3 phosphatase and replication fork restart after MMS treatment [45*,47]. Szyjka *et al.* also reported that MMS-treated *pph3Δ* cells display reduced fork progression and decreased Rad53 phosphorylation levels.
In a *pph3Δ* background, deletion of Ptc2, another Rad53 phosphatase that was previously shown to be necessary for DSB adaptation [48], results in a block of replication fork progression and lethality in MMS, which can be rescued by overexpressing a dominant negative kinase-dead *rad53* allele. Interestingly, neither Pph3, Ptc2 nor Ptc3 deletion alone or in combination is required for Rad53 deactivation after HU-induced replication stress, suggesting that another not yet identified phosphatase might be responsible [45*,46]. Thus, different phosphatases seem to be necessary to recognize and dephosphorylate specific Rad53 phosphorylation patterns that correspond to distinct genotoxic insults as well as to differential mechanisms to lead to fork restart or late origin firing. Further research on how phosphatases regulate Rad53 inactivation after replication stress is needed to help understand the complex mechanism of S phase checkpoint recovery and adaptation.

Interestingly, recent data suggest that the evolutionary conserved Ino80 chromatin remodeling complex is not only important for DSB repair but may also play a crucial role to restart HU-stalled forks [49,50,51**]. Ino80 is recruited to DSBs by Mec1/Tel1-dependent phosphorylation of histone H2A (γ-H2A) [49]. Shimada *et al.* observed enhanced Ino80 binding at stalled forks and at late firing origins after HU-treatment, and consistently replication fork collapse and increased rates of DSBs when important Ino80 subunits are deleted [51**]. The model they proposed hypothesizes that Ino80 promotes recovery by mobilizing histones ahead of the fork to mediate polymerase resumption. Furthermore, the Ino80 subunit Ies4 is phosphorylated by Mec1/Tel1 after DNA damage and surprisingly genetic data suggest redundant functions for phosphorylated Ies4 and the S phase-specific checkpoint regulator Tof1 [52]. It is of great interest to understand how exactly this chromatin remodeler contributes to replication fork restart.

### Conclusion

The conserved checkpoint kinase ATR or Mec1 serves multiple roles in the cellular response to endogenous replication stress and DNA damage. It functions directly at stalled forks (i.e. stabilizing replicative polymerases) and is also crucial for activation of a checkpoint response during S phase. The combination of strains mutated in both replication machinery components and checkpoint proteins demonstrate that the roles of ATR/Mec1 are complex. A major enigma is the mechanism by which ATR/Mec1 contributes to polymerase stabilization to allow resumption of replication and successful completion of DNA synthesis once the insult has been removed. Which target(s) of ATR/Mec1 stabilize replicative polymerases at the fork? Is this sufficient to suppress fork-associated recombination events? Although many potential targets of ATR/Mec1 at the

fork have emerged, including RecQ helicases, Claspin/Mrc1, and RPA, further work is required to answer these questions.

## Acknowledgements

The Gasser laboratory is supported by grants from the EU and the Swiss National Science Foundation, as well as the Novartis Research Foundation, and a HFSP fellowship to B Pike. We thank Kenji Shimada and Stephanie Kung for helpful comments.

## References and recommended reading

Papers of particular interest published within the period of review have been highlighted as:

- of special interest
- of outstanding interest

1. Cimprich KA, Cortez D: ATR: an essential regulator of genome integrity. *Nat Rev Mol Cell Biol* 2008, **9**:616-627.
   - Gives a comprehensive overview on ATR in DNA-damage signaling in comparison to ATM.
2. Aparicio OM, Stout AM, Bell SP: Differential assembly of Cdc45p and DNA polymerases at early and late origins of DNA replication. *Proc Natl Acad Sci USA* 1999, **96**:9130-9135.
3. Paulsen RD, Cimprich KA: The ATR pathway: fine-tuning the fork. *DNA Repair (Amst)* 2007, **6**:953-966.
4. Zou L, Elledge SJ: Sensing DNA damage through ATRIP recognition of RPA-ssDNA complexes. *Science* 2003, **300**:1542-1548.
5. Melo JA, Cohen J, Toczyski DP: Two checkpoint complexes are independently recruited to sites of DNA damage *in vivo*. *Genes Dev* 2001, **15**:2809-2821.
6. Rouse J, Jackson SP: Lcd1p recruits Mec1p to DNA lesions *in vitro* and *in vivo*. *Mol Cell* 2002, **9**:857-869.
7. Michael WM, Ott R, Fanning E, Newport J: Activation of the DNA replication checkpoint through RNA synthesis by primase. *Science* 2000, **289**:2133-2137.
8. Byun TS, Pacek M, Yee MC, Walter JC, Cimprich KA: Functional uncoupling of MCM helicase and DNA polymerase activities activates the ATR-dependent checkpoint. *Genes Dev* 2005, **19**:1040-1052.
9. MacDougall CA, Byun TS, Van C, Yee MC, Cimprich KA: The structural determinants of checkpoint activation. *Genes Dev* 2007, **21**:898-903.
10. Majka J, Niedziela-Majka A, Burgers PM: The checkpoint clamp activates Mec1 kinase during initiation of the DNA damage checkpoint. *Mol Cell* 2006, **24**:891-901.
11. Medhurst AL, Warmerdam DO, Akerman I, Verwayen EH, Kanaar R, Smits VA, Lakin ND: ATR and Rad17 collaborate in modulating Rad9 localisation at sites of DNA damage. *J Cell Sci* 2008, **121**:3933-3940.
12. Delacroix S, Wagner JM, Kobayashi M, Yamamoto K, Karnitz LM: The Rad9-Hus1-Rad1 (9-1-1) clamp activates checkpoint signaling via TopBP1. *Genes Dev* 2007, **21**:1472-1477.
13. Lee J, Kumagai A, Dunphy WG: The Rad9-Hus1-Rad1 checkpoint clamp regulates interaction of TopBP1 with ATR. *J Biol Chem* 2007, **282**:28036-28044.
14. Bonilla CY, Melo JA, Toczyski DP: Colocalization of sensors is sufficient to activate the DNA damage checkpoint in the absence of damage. *Mol Cell* 2008, **30**:267-276.
   - By using an artificial tethering system it was demonstrated that colocalization of Mec1-Ddc2 and 9-1-1 is sufficient to activate a checkpoint response in absence of DNA damage.
15. Navadgi-Patil VM, Burgers PM: Yeast DNA replication protein Dpb11 activates the Mec1/ATR checkpoint kinase. *J Biol Chem* 2008, **283**:35853-35859.
16. Mordes DA, Nam EA, Cortez D: Dpb11 activates the  
    Mec1-Ddc2 complex. *Proc Natl Acad Sci U S A* 2008,  
    105:18730-18734.  
    References [15] and [16] report that *S. cerevisiae* Dbp11 activates Mec1-  
    Ddc2 in vitro and therefore is a functional ortholog of human TopBP1.  

17. Masumoto H, Sugino A, Araki H: Dpb11 controls the association  
    between DNA polymerases alpha and epsilon and the  
    autonomously replicating sequence region of budding yeast.  
    *Mol Cell Biol* 2000, 20:2809-2817.  

18. Zegerman P, Diffley JF: Phosphorylation of Sld2 and Sld3 by  
    cyclin-dependent kinases promotes DNA replication in  
    budding yeast. *Nature* 2007, 445:281-285.  

19. Tanaka S, Umemori T, Hirai K, Muramatsu S, Kamimura Y,  
    Araki H: CDK-dependent phosphorylation of Sld2 and Sld3  
    initiates DNA replication in budding yeast. *Nature* 2007,  
    445:328-332.  

20. Cobb JA, Bjergbaek L, Shimada K, Frei C, Gasser SM: DNA  
    polymerase stabilization at stalled replication forks  
    requires Mec1 and the RecQ helicase Sgs1. *EMBO J* 2003,  
    22:4325-4336.  

21. Myung K, Datta A, Chen C, Kolodner RD: SGS1, the  
    Saccharomyces cerevisiae homologue of BLM and WRN,  
    suppresses genome instability and homeologous  
    recombination. *Nat Genet* 2001, 27:113-116.  

22. Myung K, Kolodner RD: Suppression of genome instability  
    by redundant S-phase checkpoint pathways in  
    Saccharomyces cerevisiae. *Proc Natl Acad Sci U S A* 2002,  
    99:4500-4507.  

23. Cobb JA, Schleker T, Rojas V, Bjergbaek L, Tercero JA,  
    Gasser SM: Replisome instability, fork collapse, and gross  
    chromosomal rearrangements arise synergistically from  
    Mec1 kinase and RecQ helicase mutations. *Genes Dev* 2005,  
    19:3055-3069.  

24. Zhu Z, Chung WH, Shim EY, Lee SE, Ira G: Sgs1 helicase and two  
    nucleases Dna2 and Exo1 resect DNA double-strand break  
    ends. *Cell* 2008, 134:981-994.  
    Report together with references [25] and [26] that the RecQ helicase Sgs1  
    collaborates with Exo1 in DSB end resection. A two-step model was  
    proposed in which MRX together with Sae2 initiates 5′ degradation, which  
    is further processed by Exo1/Sgs1 to generate long ssDNA tracts that  
    serve Rad51 as a substrate for homologous recombination.  

25. Mimitou EP, Symington LS: Sae2, Exo1 and Sgs1 collaborate in  
    DNA double-strand break processing. *Nature* 2008,  
    455:770-774.  

26. Gravel S, Chapman JR, Magill C, Jackson SP: DNA helicases  
    Sgs1 and BLM promote DNA double-strand break resection.  
    *Genes Dev* 2008, 22:2767-2772.  

27. Segurado M, Diffley JF: Separate roles for the DNA damage  
    checkpoint protein kinases in stabilizing DNA replication  
    forks. *Genes Dev* 2008, 22:1816-1827.  
    This work provides further evidence that Mec1 has separable functions in  
    replication fork stabilization from Rad53. The rad53Δ, but not mec1Δ,  
    lethality on MMS can be rescued by deletion of the 5′-3′ exonuclease and  
    flap endonuclease Exo1.  

28. Davies SL, North PS, Hickson ID: Role for BLM in replication-  
    fork restart and suppression of origin firing after replicative  
    stress. *Nat Struct Mol Biol* 2007, 14:677-679.  
    Using a dual-labeling chromosome fiber spreading technique it was  
    shown that ATR-catalyzed BLM phosphorylation on T99 is important  
    for replication fork restart after HU or aphidicolin treatment and for  
    suppression of new origin firing.  

29. Pirzio LM, Pichierri P, Bignami M, Franchitto A: Werner syndrome  
    helicase activity is essential in maintaining fragile site stability.  
    *J Cell Biol* 2008, 180:305-314.  

30. Hodgson B, Calzada A, Labib K: Mrc1 and Tof1 regulate DNA  
    replication forks in different ways during normal S phase. *Mol  
    Cell* 2007, 18:3894-3902.  

31. Szyjka SJ, Viggiani CJ, Aparicio OM: Mrc1 is required for normal  
    progression of replication forks throughout chromatin in S.  
    *cerevisiae*. *Mol Cell* 2005, 19:691-697.  

32. Tourriere H, Versini G, Cordon-Preciado V, Alabert C, Pasero P:  
    Mrc1 and Tof1 promote replication fork progression  
    and recovery independently of Rad53. *Mol Cell* 2005,  
    19:699-706.  

33. Pursell ZF, Isoz I, Lundstrom EB, Johansson E, Kunkel TA: Yeast  
    DNA polymerase epsilon participates in leading-strand DNA  
    replication. *Science* 2007, 317:127-130.  
    It is shown for the first time that polymerase (in *S. cerevisiae* is involved in  
    leading strand DNA synthesis.  

34. Lou H, Komata M, Katou Y, Guan Z, Reis CC, Budd M, Shirahige K,  
    Campbell JL: Mrc1 and DNA polymerase epsilon function  
    together in linking DNA replication and the S phase  
    checkpoint. *Mol Cell* 2008, 32:106-117.  
    Reports that Mrc1 interacts with the N-terminal and C-terminal half of  
    polymerase ε and that phosphorylation of Mrc1 during S phase check-  
    point activation prevents its interaction with N-terminal half of polymerase  
    ε. A model was suggested in which Mrc1 modulates coupling of DNA  
    unwinding and polymerization during replication.  

35. Petermann E, Helleday T, Caldecott KW: Claspin promotes  
    normal replication fork rates in human cells. *Mol Biol Cell* 2008,  
    19:2373-2378.  

36. Bjergbaek L, Cobb JA, Tsai-Pflugfelder M, Gasser SM:  
    Mechanistically distinct roles for Sgs1p in checkpoint  
    activation and replication fork maintenance. *EMBO J* 2005,  
    24:405-417.  

37. Zhang W, Durocher D: Dun1 counts on rad53 to be turned on.  
    *Mol Cell* 2008, 31:1-2.  

38. Matsuoka S, Ballif BA, Smogorzewska A, McDonald ER 3rd,  
    Hurov KE, Luo J, Bakalarski CE, Zhao Z, Solimini N, Lerenthal Y  
    et al.: ATM and ATR substrate analysis reveals extensive  
    protein networks responsive to DNA damage. *Science* 2007,  
    316:1160-1166.  

39. Shimada M, Niida H, Zineldeen DH, Tagami H, Tanaka M, Saito H,  
    Nakanishi M: Chk1 is a histone H3 threonine 11 kinase that  
    regulates DNA damage-induced transcriptional repression.  
    *Cell* 2008, 132:221-232.  

40. Heideker J, Lis ET, Romesberg FE: Phosphatases, DNA damage  
    checkpoints and checkpoint deactivation. *Cell Cycle* 2007,  
    6:3058-3064.  

41. Morin I, Ngo HP, Greenall A, Zubko MK, Morrice N, Lydall D:  
    Checkpoint-dependent phosphorylation of Exo1  
    modulates the DNA damage response. *EMBO J* 2008,  
    27:2400-2410.  

42. Branzei D, Foiani M: The Rad53 signal transduction pathway:  
    replication fork stabilization, DNA repair, and adaptation.  
    *Exp Cell Res* 2006, 312:2654-2659.  

43. Alvino GM, Collingwood D, Murphy JM, Delrow J, Brewer BJ,  
    Raghuraman MK: Replication in hydroxyurea: it’s a matter of  
    time. *Mol Cell Biol* 2007, 27:6396-6406.  
    This study questions the current model that Rad53 specifically inhibits  
    late origin firing after replication stress, but suggests a delay of the entire  
    S phase according to the rate of replication progression.  

44. Trenz K, Errico A, Costanzo V: Plx1 is required for chromosomal  
    DNA replication under stressful conditions. *EMBO J* 2008,  
    27:876-885.  
    This study shows that ATR-dependent phosphorylation of MCM2 recruits  
    Polo-like kinase to stalled replication forks leading to origin firing and  
    completion of replication.  

45. Szyjka SJ, Aparicio JG, Viggiani CJ, Knott S, Xu W, Tavare S,  
    Aparicio OM: Rad53 regulates replication fork restart after DNA  
    damage in *Saccharomyces cerevisiae*. *Genes Dev* 2008,  
    22:1906-1920.  
    This paper provides a mechanistic link between Rad53 dephosphoryla-  
    tion and replication fork restart. Cells lacking the the Psy2-Pph3 phos-  
    phatase show defects in Rad53 dephosphorylation and fork restart after  
    MMS treatment, while deactivation of Rad53 by overexpression of a  
    dominant-negative kinase-dead variant of Rad53 is sufficient to allow fork  
    restart during recovery.  

46. Travesa A, Duch A, Quintana DG: Distinct phosphatases  
    mediate the deactivation of the DNA damage checkpoint  
    kinase Rad53. *J Biol Chem* 2008, 283:17123-17130.

47. O'Neill BM, Szyjka SJ, Lis ET, Bailey AO, Yates JR 3rd, Aparicio OM, Romesberg FE: Pph3-Psy2 is a phosphatase complex required for Rad53 dephosphorylation and replication fork restart during recovery from DNA damage. Proc Natl Acad Sci USA 2007, 104:9290-9295.

48. Leroy C, Lee SE, Vaze MB, Ochsenbien F, Guerois R, Haber JE, Marsolier-Kergoat MC: PP2C phosphatases Ptc2 and Ptc3 are required for DNA checkpoint inactivation after a double-strand break. Mol Cell 2003, 11:827-835.

49. van Attikum H, Fritsch O, Gasser SM: Distinct roles for SWR1 and INO80 chromatin remodeling complexes at chromosomal double-strand breaks. EMBO J 2007, 26:4113-4125.

50. Papamichos-Chronakis M, Peterson CL: The Ino80 chromatin-remodeling enzyme regulates replisome function and stability. Nat Struct Mol Biol 2008, 15:338-345.

51. Shimada K, Oma Y, Schleker T, Kugou K, Ohta K, Harata M, Gasser SM: Ino80 chromatin remodeling complex promotes recovery of stalled replication forks. Curr Biol 2008, 18:566-575.
This study reports that Ino80 is enriched at stalled forks and shows, together with reference [50], that Ino80 stabilizes stalled replication forks and facilitates fork restart.

52. Morrison AJ, Kim JA, Person MD, Highland J, Xiao J, Wehr TS, Hensley S, Bao Y, Shen J, Collins SR et al.: Mec1/Tel1 phosphorylation of the INO80 chromatin remodeling complex influences DNA damage checkpoint responses. Cell 2007, 130:499-511.
