
JOURNAL OF VIROLOGY, May 2009, p. 4652–4669
0022-538X/09/$08.00+0 doi:10.1128/JVI.02408-08
Copyright © 2009, American Society for Microbiology. All Rights Reserved.

Epstein-Barr Virus Nuclear Antigen 3C Augments Mdm2-Mediated p53 Ubiquitination and Degradation by Deubiquitinating Mdm2

Abhik Saha, Masanao Murakami, Pankaj Kumar, Bharat Bajaj, Karen Sims, and Erle S. Robertson*

Department of Microbiology and Tumor Virology Program of the Abramson Comprehensive Cancer Center, University of Pennsylvania Medical School, 201E Johnson Pavilion, 3610 Hamilton Walk, Philadelphia, Pennsylvania 19104

Received 21 November 2008/Accepted 18 February 2009

Epstein-Barr virus (EBV) nuclear antigen 3C (EBNA3C) is one of the essential latent antigens for primary B-cell transformation. Previous studies established that EBNA3C facilitates degradation of several vital cell cycle regulators, including the retinoblastoma (pRb) and p27<sup>KIP</sup> proteins, by recruitment of the SCF<sup>Skp2</sup>E3 ubiquitin ligase complex. EBNA3C was also shown to be ubiquitinated at its N-terminal residues. Furthermore, EBNA3C can bind to and be degraded in vitro by purified 20S proteasomes. Surprisingly, in lymphoblastoid cell lines, EBNA3C is extremely stable, and the mechanism for this stability is unknown. In this report we show that EBNA3C can function as a deubiquitination enzyme capable of deubiquitinating itself in vitro as well as in vivo. Functional mapping using deletion and point mutational analysis showed that both the N- and C-terminal domains of EBNA3C contribute to the deubiquitination activity. We also show that EBNA3C efficiently deubiquitates Mdm2, an important cellular proto-oncogene, which is known to be overexpressed in several human cancers. The data presented here further demonstrate that the N-terminal domain of EBNA3C can bind to the acidic domain of Mdm2. Additionally, the N-terminal domain of EBNA3C strongly stabilizes Mdm2. Importantly, EBNA3C simultaneously binds to both Mdm2 and p53 and can form a stable ternary complex; however, in the presence of p53 the binding affinity of Mdm2 toward EBNA3C was significantly reduced, suggesting that p53 and Mdm2 might share a common overlapping domain of EBNA3C. We also showed that EBNA3C enhances the intrinsic ubiquitin ligase activity of Mdm2 toward p53, which in turn facilitated p53 ubiquitination and degradation. Thus, manipulation of the oncoprotein Mdm2 by EBNA3C potentially provides a favorable environment for transformation and proliferation of EBV-infected cells.

Epstein-Barr virus (EBV) is a ubiquitous human gammaherpesvirus that persists for the life of the host. EBV infects more than 90% of the adult population worldwide and efficiently immortalizes infected human primary B cells. This ability is likely to predispose the host to a variety of cancers, including endemic Burkitt's lymphoma, nasopharyngeal carcinoma, post-transplant lymphoproliferative disease, and some subtypes of Hodgkin's disease (61).

One of the biological hallmarks of EBV-cell interaction is establishment of latency. Three major types of latency have been described, each having its own distinct pattern of gene expression. Type I latency is typically reflected in Burkitt's lymphoma tumors (72). EBV nuclear antigen 1 (EBNA1) protein is the predominant viral antigen expressed in this form of latency (72). Type II latency is typically seen in nasopharyngeal carcinoma and Hodgkin's disease, where EBNA1, latent membrane protein 1 (LMP1), and LMP2A and LMP2B proteins are expressed (72). Type III latency, also referred to as the growth program (72), is seen in lymphoblastoid cell lines (LCLs) and results in the expression of nine viral latency proteins, including six nuclear proteins (EBNA1, EBNA2, EBNA3A, EBNA3B, EBNA3C, and EBNALP) and three latent membrane proteins (LMP1, LMP2A, and LMP2B), and the viral RNAs which include the EBERs and BARTs (34, 61). Four viral antigens, EBNA2, LMP1, EBNA3A, and EBNA3C, have been shown to be absolutely essential for EBV transformation of human B cells and establishment of latency in vitro (2, 32, 56, 69).

EBNA3C has been shown to play a complex regulatory role in the transcription of viral and cellular genes (27, 45, 63, 82). In addition to its transcriptional functions, EBNA3C has cell cycle-regulatory functions, mediated by direct protein-protein interactions with regulators of the cell cycle (35, 36, 37). Recently, we demonstrated that EBNA3C targets the SCF<sup>Skp2</sup>E3 ubiquitin (Ub) ligase complex and thereby destabilizes a number of important cell cycle components such as retinoblastoma protein (Rb) and p27<sup>KIP</sup> (36, 37). EBNA3C is also ubiquitinated at its N-terminal domain through interaction with the SCF<sup>Skp2</sup>E3 ligase complex (37). Studies have also shown that EBNA3C interacts with the α subunit of the 20S proteasome and is degraded in vitro by purified 20S proteasomes (75). Surprisingly, in actively proliferating LCLs, EBNA3C appears to be remarkably stable, with no indication of proteasome-mediated degradation (75). However, the mechanistic details of EBNA3C stabilization are yet to be elucidated.

Mdm2 was first described as one of the genes amplified on double-minute chromosomes present in the spontaneously transformed BALB/c/3T3 murine cell line 3T3DM (10). Subsequent analysis demonstrated that Mdm2 is overexpressed in 5 to 10% of human tumors (29, 53). The best-known biological function of the human version of Mdm2 (Hdm2) is to negatively regulate the activity of the tumor suppressor protein p53

* Corresponding author. Mailing address: Department of Microbiology and Tumor Virology Program of the Abramson Comprehensive Cancer Center, University of Pennsylvania Medical School, 201E Johnson Pavilion, 3610 Hamilton Walk, Philadelphia, PA 19104. Phone: (215) 746-0114. Fax: (215) 746-0115. E-mail: erle@mail.med.upenn.edu.
Published ahead of print on 25 February 2009.

(42, 54). Under conditions of cellular stress, p53 upregulates the transcription of an array of genes which are critically implicated in control of numerous cellular processes including the cell cycle, apoptosis, DNA repair, differentiation, and senescence (65). The N-terminal amino acid residues 1 to 120 of Mdm2 contain the interaction domain which binds to the transactivation domain of p53 (12, 57). The C-terminal region of Mdm2 encodes the Ub ligase activity specific for p53 (22, 39). Hence, the interaction of Mdm2 with p53 operates both to inhibit its transcriptional regulatory activity and to facilitate the degradation of p53 through ubiquitination (22, 39). Additionally, Mdm2 harbors a nuclear export signal that promotes export of the Mdm2-p53 complex from the nucleus to the cytoplasm (64). Mdm2 also regulates sumoylation (14), neddylation (80), and acetylation (38) of p53. It should be noted that in response to exposure to various types of stress, p53 is stabilized and activated. Several pathways leading to inhibition of Mdm2 action have been identified, including the phosphorylation of Mdm2 by DNA damage-induced kinases (33, 50, 51, 70) and the interaction of Mdm2 with other cellular proteins (9, 74), including ARF, which disrupts Mdm2 regulation of p53 by blocking Mdm2 Ub ligase activity (30, 58, 68, 81). Conversely, the *mdm2* gene is among those that are upregulated by p53; thus, not only is Mdm2 required for keeping p53 in check under nonstress conditions and releasing it when appropriate, but it is also part of an autoregulatory feedback loop (54).

Overexpression of Mdm2 abolishes p53-mediated cell cycle arrest and apoptosis (13). These studies are consistent with clinical observations that implicate a dysfunctional p53-Mdm2 system in close to 70% of all tumor samples (53). In about 7% of all cancers, wild-type p53 is present and the problem lies in a surplus of Mdm2 (53). For example, single nucleotide polymorphism 309 in the Mdm2 promoter, which leads to high levels of Mdm2, accelerates tumor formation in both hereditary and spontaneous cancers (6). In tumors with wild-type p53, an attractive route to therapy would be to diminish the inhibitory effects of Mdm2 by blocking its interaction with p53 (83). Interestingly, Mdm2 is an unstable protein which is ubiquitinated in an autocatalytic manner (19, 25). However, Mdm2 can differentiate between self-ubiquitination and targeting p53 for ubiquitination. It has been confirmed that EBV is associated with elevated levels of Mdm2 expression (71, 77). Thus, mechanisms that regulate the respective activities of Mdm2 in the context of EBV infections and transformation should provide information as to its contribution to EBV-associated cancers.

This study provides the first evidence that EBNA3C plays a dual role in the Ub-proteasome-mediated protein degradation pathway. We demonstrate here that, in addition to recruiting the E3 Ub ligase, SCF<sup>Skp2</sup> (37), EBNA3C can also function as a deubiquitinating enzyme, though it is unclear how these activities of EBNA3C, which have opposing effects on cell fate, are differentially regulated. The results presented here also emphasize that EBNA3C can suppress the function of the p53 tumor suppressor by stabilizing Mdm2 function through deubiquitination.

---

**MATERIALS AND METHODS**

Plasmids, antibodies, and cell lines. pA3M or pA3F-EBNA3C constructs express either full-length EBNA3C or different truncated versions of EBNA3C

with either a Myc tag or a Flag tag at the carboxy-terminal end, as described previously (3, 37). Glutathione S-transferase (GST)-EBNA3C truncation mutants have been previously described (3). The C143N point mutation in the EBNA3C gene was prepared by a standard PCR primer mutagenesis method. Constructs expressing either green fluorescent protein (GFP)-tagged full-length EBNA3C or different GFP-tagged truncated mutants were prepared by cloning PCR-amplified fragments into pEGFP-C1 vector (BD Biosciences Clontech) at EcoRI and SalI restriction sites. pCDNA3-HA-Ub was provided by George Mosialos (Alexander Fleming Biomedical Sciences Research Center, Vari, Greece) (76). pCDNA3-myc-HAUSP was provided by Aart G. Jochemsen (Leiden University Medical Center, The Netherlands) (52). pGEX-Mdm2 was a kind gift from Wafik S. El-Deiry (University of Pennsylvania). pRK5-Flag-Mdm2 constructs were provided by Xiaolu Yang (University of Pennsylvania) (71). pRK5-HA-Mdm2 was used as a template for preparing constructs expressing either full-length Mdm2 or different truncated versions with a carboxy-terminal Flag tag by cloning PCR-amplified fragments into pA3F vector at EcoRI and NotI restriction sites. All constructs and mutations were verified by DNA sequencing (University of Pennsylvania DNA sequencing facility).

Mouse monoclonal antibody reactive to Mdm2 (SMP14) was purchased from Santa Cruz Biotechnology, Inc. (Santa Cruz, CA). Mouse monoclonal antibody reactive to Flag epitope (M2) was purchased from Sigma-Aldrich Corp. (St. Louis, MO). The monoclonal antibodies mouse antihemagglutinin (anti-HA; 12CA5) and mouse anti-Myc (9E10) were prepared from the respective hybridoma cultures. A10 monoclonal and rabbit polyclonal antibodies reactive to EBNA3C have been previously described (16).

HEK 293 cells are human embryonic kidney cells transformed with sheared adenovirus type 5 DNA (20). HEK 293T cells are HEK 293 cells that stably express the simian virus 40 large-T antigen. Both HEK 293 and 293T cells were obtained from Jon Aster (Brigham and Women’s Hospital, Boston, MA). U2OS is a human osteosarcoma cell line (59). The p53-null cell line Saos-2 is also a human osteosarcoma cell line and was obtained from Jon Aster (Brigham and Women’s Hospital, Boston, MA). MEF p53<sup>−/−</sup> Mdm2<sup>−/−</sup> is a mouse embryonic fibroblast cell line null for both p53 and Mdm2 and was a kind gift from Xiaolu Yang (University of Pennsylvania) (71). HEK 293T, U2OS, Saos-2, and MEF cells were grown in Dulbecco’s modified Eagle’s medium (purchased from HyClone, Logan, UT) supplemented with 10% fetal bovine serum, 50 U/ml penicillin, 50 μg/ml streptomycin, and 2 mM L-glutamine. The Burkitt’s lymphoma cell line BJAB is derived from an EBV-negative African Burkitt’s lymphoma tumor (15) and was provided by Elliott Kieff (Harvard Medical School, Boston, MA). BJAB and the EBV-positive cell line LCL2 were maintained in RPMI 1640 medium (HyClone, Logan, UT) supplemented as described for Dulbecco’s modified Eagle’s medium above. EBNA3C-expressing BJAB cell lines have been described previously (62). All cultures were incubated at 37°C in a humidified environment supplemented with 5% CO<sub>2</sub>.

Transfection. HEK 293T cells were transfected by electroporation with a Bio-Rad Gene Pulser II electroporator. Briefly, 15 × 10<sup>6</sup> cells harvested in exponential phase were collected, washed in phosphate-buffered saline (PBS), and resuspended in 400 μl of the appropriate medium without serum containing DNA for transfection (3, 35). Resuspended cells were transferred to an 0.4-cm-gap cuvette, and electroporation was performed at 975 μF and 210 V for HEK 293T cells. Transfected cells were transferred to a 100-mm petri dish containing 10 ml of complete medium and incubated at 37°C. Unless indicated, transfected cells were harvested at 36 h, and sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) was performed with 5% of the total normalized protein lysate.

Immunoprecipitation and Western blotting. Transfected cells were harvested, washed with ice-cold PBS, and lysed in 0.5 ml ice-cold radioimmunoprecipitation (RIPA) buffer (1% Nonidet P-40 [NP-40], 10 mM Tris [pH 7.5], 2 mM EDTA, 150 mM NaCl, supplemented with protease inhibitors [1 mM phenylmethylsulfonyl fluoride, 1 μg/ml aprotinin, 1 μg/ml pepstatin, and 1 μg/ml leupeptin]). Cell debris was removed by centrifugation at 21,000 × g (10 min and 4°C), and the supernatant was transferred to a fresh microcentrifuge tube. Lysates were then precleared by end-over-end rotation with normal mouse serum and 30 μl of a 1:1 mixture of protein A-protein G-conjugated Sepharose beads (1 h, 4°C). Beads were spun out, and supernatant was transferred to a fresh microcentrifuge tube. The protein of interest was captured by rotating the remaining lysate with 1 μg of appropriate antibody overnight at 4°C. Immune complexes were captured with 30 μl of a 1:1 mixture of protein A and protein G Sepharose beads, pelleted, and washed five times with ice-cold RIPA buffer.

For Western blot assays, input lysates and immunoprecipitated (IP) complexes were boiled in Laemmli buffer (41), fractionated by SDS-PAGE, and transferred to a 0.45-μm nitrocellulose membrane. The membranes were then
probed with appropriate antibodies followed by incubation with appropriate infrared-tagged secondary antibodies and viewed on an Odyssey imager (LiCor Inc., Lincoln, NE).

**Purification of GST fusion proteins.** *Escherichia coli* BL21(DE3) cells were transformed with the plasmid constructs for each GST fusion protein. Single colonies were picked and grown overnight in 3 ml of Luria broth. One milliliter of the overnight culture was used to inoculate a 500-ml culture. The larger culture was incubated until the optical density at 600 nm was approximately 0.6, at which point it was induced with 1 mM isopropyl-β-D-thiogalactopyranoside (IPTG) for 5 h. The bacteria were pelleted, washed once with STE buffer (100 mM NaCl, 10 mM Tris, and 1 mM EDTA, pH 7.5), resuspended in 3 ml NETN buffer (0.5% NP-40, 100 mM NaCl, 20 mM Tris, 1 mM EDTA, pH 8.0), supplemented with protease inhibitors, and incubated on ice for 15 min. A volume of 150 μl of 1 M dithiothreitol (DTT) and 1.8 ml of a 10% solution of Sarkosyl in STE buffer was added, and the suspension was sonicated (for 3 min on ice) to solubilize the proteins. The lysate was centrifuged (12,000 × g, 10 min, 4°C) to separate the unsolubilized fraction. The clear supernatant was transferred to a fresh tube, to which 3 ml of 10% Triton X-100 in STE buffer and 200 μl of glutathione-Sepharose beads were added. The tube was rotated overnight at 4°C, after which the purified protein bound to glutathione was collected by centrifugation (2 min, 600 × g, 4°C) and washed five times with NETN buffer supplemented with protease inhibitors. The level of purification was determined by SDS-PAGE, and purified proteins were stored at 4°C.

**GST pull-down assays.** For pull-down assays from cell lysates, lysates were prepared in RIPA buffer (0.5% NP-40, 10 mM Tris [pH 7.5], 2 mM EDTA, 150 mM NaCl, supplemented with protease inhibitors). Lysates were precleared and then rotated with either GST control or the appropriate GST fusion protein bound to glutathione-Sepharose beads. For in vitro binding experiments, GST fusion proteins were incubated with ³⁵S-labeled in vitro-translated protein in binding buffer (1× PBS, 0.1% NP-40, 0.5 mM DTT, 10% glycerol, supplemented with protease inhibitors). In vitro translation was done with the T7-TNT Quick Coupled transcription-translation system (Promega Inc., Madison, WI) according to the manufacturer’s instructions.

**Immunofluorescence.** HEK 293T or U2OS cells were plated on 22-by 22-mm coverslips. Cells were transfected with appropriate plasmids as mentioned elsewhere in the paper using Lipofectamine 2000 (Invitrogen, Carlsbad, CA). At 36 h posttransfection, cells were fixed using 4% buffered formalin (20 min at room temperature [RT]) and washed three times with PBS. BJAB cells or BJAB cells stably expressing EBNA3C were air dried onto slides and fixed using a 1:1 mixture of acetone and methanol at −20°C for 10 min. Fixed cells were blocked and permeabilized with 0.2% fish skin gelatin in 1× PBS containing 0.1% Triton X-100 for 30 min at RT. Flag-tagged Mdm2 was detected using M2 antibody (1:1,000 dilution; Sigma-Aldrich Corp., St. Louis, MO), and EBNA3C was detected using EBNA3C-reactive rabbit polyclonal antibody (1:150 dilution). Endogenously expressed Mdm2 was detected using Mdm2-specific monoclonal antibody SMP14 (1:500 dilution; Santa Cruz Biotechnology, Inc., Santa Cruz, CA). Primary antibodies were diluted in blocking buffer and incubated with the cells for 1 h at RT. Cells were washed three times with blocking buffer and exposed to secondary antibodies. Goat anti-rabbit antibody conjugated to Alexa Fluor 488 and goat anti-mouse antibody conjugated to Alexa Fluor 594 were used to detect EBNA3C and Mdm2, respectively. Secondary antibodies were diluted in blocking buffer at 1:2,000 and incubated for 1 h at RT, followed by three washes with blocking buffer. The last wash contained 4′,6′-diamidino-2-phenylindole (DAPI; Promega, Madison, WI) to counterstain the nuclei.

**Stability assay.** Cells were transiently transfected using electroporation with appropriate plasmids expressing HA-tagged Mdm2, Myc-tagged p53, and/or different Flag-tagged EBNA3C species as indicated elsewhere in the paper. At 36 h after transfection, cells were treated with 40 μg/ml cycloheximide (CHX; CalBiochem, Gibbstown, NJ) and lysates were subjected to immunoblot analyses. Band intensities were quantitated using software provided with the Odyssey imager (LiCor Inc., Lincoln, NE).

**Real-time quantitative PCR.** Total RNA was isolated by using TRIzol reagent according to the instructions of the manufacturer (Invitrogen, Inc., Carlsbad, CA). cDNA was made by using a Superscript II reverse transcriptase kit (Invitrogen, Inc., Carlsbad, CA) according to the instructions of the manufacturer. Briefly, reverse transcriptase PCR was done in a total volume of 20 μl containing 10 mM Tris-HCl (pH 8.3), 1.5 mM MgCl₂, a 200 μM concentration of each deoxynucleotide triphosphate, a 2 μM concentration of each primer, 2 μl of randomly primed cDNA, and 0.5 unit of AmpliTaq (Applied Biosystems, Foster City, CA). The specific primers for *mdm2* were 5′-CCGGAATTCATGAGTGTTGGAACTAGTTTGC-3′ (sense) and 5′-ATAAGAATGCGGCCGCGGGGAATAAGTTAGC-3′ (antisense), yielding a 222-bp PCR product. The glyceraldehyde-3-phosphate dehydrogenase (GAPDH) gene was amplified by using the

primers 5′-TGCACCACCAACTGCTTAG-3′ (sense) and 5′-GATGCAGGGATGATGTTC-3′ (antisense), yielding a 185-bp PCR product. The target gene was amplified from cDNA by using the SYBR green real-time master mix (MJ Research Inc., Waltham, MA), 1 μM (each) primer, and 1 μl of the cDNA product in a total volume of 20 μl. Thirty-five cycles of 1 min at 94°C, 1 min at 55°C, and 1 min at 72°C, followed by 10 min at 72°C, were performed in an Opticon II thermocycler (MJ Research Inc., Waltham, MA). Each cycle was followed by two plate readings, with the first at 72°C and the second at 85°C. A melting-curve analysis was performed to verify the specificity of the products, and the values for the relative quantitation were calculated by the threshold cycle method. All assays were performed in triplicate.

**In vitro deubiquitination assays.** Myc-tagged EBNA3C proteins, either wild type or truncated, or full-length EBNA1 or HAUSP was IP from HEK293T cell lysates prepared with NP-40 lysis buffer (25 mM Tris-HCl, pH 8.0, 150 mM NaCl, 5 mM EDTA, 10% glycerol, 0.5% NP-40, supplemented with protease inhibitors) from cells that were pretreated for 4 h with 20 μM N-acetyl-Leu-Leu-Nle-CHO (Biomol International, L.P., Plymouth Meeting, PA). IP proteins were washed three times with NP-40 lysis buffer and three times with deubiquitination enzyme (DUB) buffer (50 mM Tris-HCl, pH 7.0, 1 mM DTT), followed by incubation with 2.5 μg of tetraubiquitin (Ub₄; Biomol International, L.P., Plymouth Meeting, PA) in 25 μl of DUB buffer overnight at 37°C. Supernatants were fractionated by 15% SDS-PAGE, followed by silver staining (Bio-Rad Laboratories, Hercules, CA) for Ub₄ degradation products. IP products were monitored by 10% SDS-PAGE, followed by immunoblotting with a monoclonal antibody against Myc tag epitope (9E10).

**In vivo ubiquitination/deubiquitination assay.** HEK 293T cells (15 × 10⁶) were transfected by electroporation (as described above) with appropriate plasmids expressing HA-Ub (5 μg), Flag-EBNA3C 1 to 365 (10 μg), and Flag-Mdm2 (10 μg). For the deubiquitination assay, cells were additionally transfected with either Myc-tagged EBNA3C (10 μg) or other EBNA3C truncated constructs or 10 μg of pA3M-EBNA1 or 10 μg of pCDNA3-*myc*-HAUSP as indicated. Cells were incubated for 36 h and pretreated for an additional 6 h with 20 μM MG132 (Biomol International) before harvesting. Flag-tagged N-terminal (residues 1 to 365) EBNA3C or Flag-tagged Mdm2 was IP with M2 antibody and resolved by SDS-PAGE. The extent of ubiquitination of Flag-tagged proteins was determined by Western blot analysis using the HA-specific antibody (12CA5).

**Labeling with HAUbVME probe.** Lysates (approximately 200 μg) prepared from either BJAB cells or BJAB cells stably expressing EBNA3C were incubated with HAUbVME probe (1 μg/μl) for 2 h at 37°C and subjected to immunoprecipitation with antibody against HA epitope (12CA5). Samples were resolved by SDS-PAGE and transferred to an 0.45-μm nitrocellulose membrane. The membrane was probed with anti-EBNA3C monoclonal antibody (A10). HEK 293T cells were transfected with either 15 μg of EBNA3C expression construct or empty vector. Cells were harvested at 36 h posttransfection and IP with 1.5 μg of M2 antibody against the Flag epitope. IP proteins were incubated with HAUbVME probe (1 μg/μl) for 2 h at 37°C in labeling buffer (50 mM Tris, pH 7.4, 5 mM MgCl₂, 250 mM sucrose, 1 mM DTT, 2 mM ATP). Labeled samples were divided into halves, resolved by 7% SDS-PAGE, and subjected to Western blotting with anti-HA antibody for detection of labeled proteins and A10 antibody for detection of EBNA3C. HAUbVME probe was kindly provided by Hidde Ploegh (Whitehead Institute, Boston, MA) (7).

---

**RESULTS**

**EBNA3C forms a complex with Mdm2 both in vivo and in vitro.** The human homologue of the mouse double-minute 2 (Mdm2, also known as Hdm2) oncogene is overexpressed in more than 40 different types of malignancies, including solid tumors, sarcomas, and leukemias (29, 53). Because of its prevalent expression and its interactions with p53 and other signaling molecules, Mdm2 plays a central role in cancer development and progression. Although there are some conflicting data, the overall evidence suggests that EBV infection can lead to an increase in Mdm2 expression (18).

In order to determine whether EBNA3C is able to form a complex with Mdm2, we performed binding assays using coimmunoprecipitation experiments. HEK 293T cells were cotransfected with expression constructs for Flag-tagged EBNA3C and Mdm2 tagged with an HA epitope. The results
Vol. 83, 2009                                                                 EBNA3C DEUBIQUITINATES Mdm2 4655

A)  
| Lysate | IP: αHA |
| --- | --- |
| HA-Mdm2 | - + - + |
| Flag-EBNA3C | + + + + |
| vector | + - + - |
| WB: αHA |  |
| 95- |  |
| WB: αFlag | 170- |
| 130- |  |

B)  
| Lysate | IP: αFlag |
| --- | --- |
| HA-Mdm2 | + + + + |
| Flag-EBNA3C | - + - + |
| vector | + - + - |
| WB: αFlag | 170- |
| 130- |  |
| WB: αHA | 95- |
|  |

C)  
| Lysate | Control serum | Lysate | Control serum |
| --- | --- | --- | --- |
| IP: αMdm2 |  | IP: αMdm2 |  |
| WB: αMdm2 |  |  |  |
| IP: αMdm2 |  |  |  |
| WB: αA10 |  |  |  |
| BJAB neo | LCL1 |  |  |
| Binding (%) |  |  |  |

D)  
| Lysate | GST-Mdm2 |
| --- | --- |
| WB: αA10 | 170- |
| 130- |  |
| BJAB neo |  |
| WB: αA10 | 170- |
| 130- |  |
| BJAB E3C #7 |  |
| WB: αA10 | 170- |
| 130- |  |
| BJAB E3C #10 |  |
| Coomassie staining |  |

FIG. 1. EBNA3C forms a complex with Mdm2 both in vivo and in vitro. (A and B) Fifteen million HEK 293T cells were cotransfected with Flag-tagged EBNA3C and HA-tagged Mdm2. In each case control samples were balanced with empty vector. Cells were harvested at 36 h posttransfection, approximately 5% of the lysed cells were saved as input, and the remainder were IP with 1.5 μg of appropriate antibody. Lysates and IP complexes were resolved by 7% SDS-PAGE and subjected to Western blotting (WB) with the indicated antibodies. The same blots were stripped and reprobed with appropriate antibodies. (C) Fifty million BJAB cells and LCL1 cells (EBV-transformed LCL) were collected and lysed in RIPA buffer, and protein complexes were IP with Mdm2-specific antibody (αMdm2). Samples were resolved by 7% SDS-PAGE, and Western blotting for the indicated proteins was done by stripping and reprobing the same membrane. (D) Either GST control or GST-Mdm2 beads were incubated with lysates prepared from 20 million BJAB cells and BJAB cells stably expressing EBNA3C (two clones, E3C 7 and E3C 10). EBNA3C was detected by Western blotting with the specific monoclonal antibody (A10). Coomassie blue staining of SDS-PAGE-resolved purified GST and GST-Mdm2 proteins is shown in the right panel. All panels are representative gels from similar repeated experiments. Numbers at left of each gel or blot are molecular masses in kilodaltons.

demonstrated that EBNA3C strongly associated with Mdm2 (Fig. 1A and B, lane 4). Empty vector was used as negative control (Fig. 1A and 1B, lane 3). To further determine whether this binding occurred under endogenous conditions, Mdm2 was IP from an EBV-negative Burkitt's lymphoma cell line, BJAB, and an EBV-transformed LCL, LCL1, expressing EBNA3C and coimmunoprecipitation of EBNA3C (~15% of total amount of EBNA3C expression [Fig. 1C]) was monitored by Western blot analysis using A10, an EBNA3C-specific monoclonal antibody (Fig. 1C). Analysis of the data from the ectopic expression system as well as cell lines endogenously expressing Mdm2 and EBNA3C at physiological levels strongly demonstrated an association of Mdm2 with EBNA3C. To further corroborate the association in human cells, an in vitro binding experiment was conducted, where bacterially expressed GST-fused Mdm2 was incubated with cell lysates prepared from either BJAB cells or BJAB cells stably expressing EBNA3C. EBNA3C interacted strongly with GST-Mdm2 but not with the GST control (Fig. 1D). Coomassie blue staining of a parallel gel showed the amount of GST and GST-Mdm2 proteins used in the binding assay (Fig. 1D, right panel). These results indicate that EBNA3C forms a stable complex with Mdm2.

The N-terminal domain of EBNA3C binds to Mdm2. To map the domain of EBNA3C that interacts with Mdm2, HEK 293T cells were transfected with expression constructs for HA-tagged Mdm2 and either full-length EBNA3C (residues 1 to

A)

| IP: αFlag | Lysate |
|------------|--------|
| HA-Mdm2    | +      | +     | +     | +     | +     | +     | +     | +     |
| Flag-vector | +      | -     | -     | -     | -     | +     | -     | -     | -     |
| Flag-EBNA3C | -      | +     | -     | -     | -     | -     | +     | -     | -     |
| Flag-E3C 1-365 | -      | -     | +     | -     | -     | -     | +     | -     | -     |
| Flag-E3C 366-620 | -      | -     | -     | +     | -     | -     | -     | +     | -     |
| Flag-E3C 621-992 | -      | -     | -     | -     | +     | -     | -     | -     | +     |

WB: αMyc

WB: αHA

WB: αGAPDH

B)

E3C 1-992
IVT Input
GST
GST-Mdm2

E3C 366-620
IVT Input
GST
GST-Mdm2

E3C 1-365

E3C 621-992

D)

GST-E3C
IVT Input
GST
90-130
130-160
160-190
Mdm2

Coomassie staining

E)

EBNA3C
1
JK LZ
Acidic domain
DEDED
PPP
Q
992
Cyclin A
P-rich domain
Q-rich domain
NLS
Mdm2
Binding

1
365
621
992
1
100
1
129
1
159
1
200
90—130
130—160
160—190

FIG. 2. The N-terminal domain of EBNA3C binds to Mdm2. (A) Fifteen million HEK 293T cells were cotransfected with pRK5-HA-Mdm2, encoding Mdm2 tagged with the HA epitope, or pA3F-EBNA3C proteins, encoding either full-length EBNA3C (1 to 992) or different truncated versions (residues 1 to 365, 366 to 620, and 621 to 992), as indicated. All EBNA3C proteins were tagged with Flag epitope. Cells were harvested at 36 h, 5% of the lysed cells were saved as input, and the remainder were IP with 1.5 μg anti-Flag antibody (M2). Samples were resolved by 10% SDS-PAGE and transferred to an 0.45-μm nitrocellulose membrane. The membrane was probed with anti-HA antibody (12CA5) followed by an infrared-tagged secondary antibody and scanned. The membrane was stripped and reprobed with M2 antibody followed by an infrared-tagged secondary antibody and rescanned. (B to D) Full-length or different truncated mutant constructs of EBNA3C (B and C) and Mdm2 protein (D) were in vitro translated (IVT) using a T7-TNT translation kit. All ³⁵S-radiolabeled in vitro-translated proteins in binding buffer were precleared by rotation with GST beads for 1 h at 4°C unless otherwise stated. Binding reactions were set up by incubating the in vitro-translated proteins with either GST control or GST fusion proteins (GST-Mdm2 [B and C] and GST-EBNA3C 90 to 130, GST-EBNA3C 130 to 160, and GST-EBNA3C 160 to 190 [D]). Reaction mixtures were resolved on appropriate polyacrylamide gels, exposed to phosphorimager plates, and scanned on a Storm 850 imaging system. Coomassie blue staining of SDS-PAGE-resolved purified GST-EBNA3C truncated proteins is shown at the bottom of panel D. All panels are representative gels from similar repeated experiments. (E) The schematic illustrates structural motifs of EBNA3C that potentially contribute to protein-protein interactions and summarizes the study of binding between different domains of EBNA3C and full-length Mdm2. ++, strong binding; +, moderate binding; –, no binding. NLS, nuclear localization signal. Numbers at left of gels and blots are molecular masses in kilodaltons. The asterisk indicates the immunoglobulin bands.

ments with bacterially expressed GST-Mdm2 showed strong interaction with residues 1 to 159 and 1 to 200 of EBNA3C (Fig. 2C, bottom panel, lanes 3 and 4, respectively) but not with EBNA3C residues 1 to 100 or 1 to 129 (Fig. 2C, bottom panel, lanes 1 and 2, respectively). All fragments of EBNA3C failed to interact with the GST control, indicating that the observed

binding was specific for Mdm2 (Fig. 2C, middle panel, lanes 1 to 4). Different GST-EBNA3C truncated constructs were subsequently used to further refine the association between Mdm2 and EBNA3C. In vitro-translated ³⁵S-radiolabeled full-length Mdm2 showed strong binding with GST-EBNA3C residues 130 to 160 and 160 to 190 (Fig. 2D, top panel, lanes 4 and 5,

respectively) but not with the GST control or GST-EBNA3C residues 90 to 130 (Fig. 2C, top panel, lanes 2 and 3, respectively). Coomassie blue staining of a parallel gel showed the levels of the GST proteins used in the binding assay (Fig. 2C, bottom panel). These results indicate that the domain of EBNA3C shown to interact with other cell cycle-regulatory components can also form a complex with Mdm2. This shows that this region of EBNA3C is of particular significance in deregulation of the cell cycle in EBV-infected cells.

The central acidic domain of Mdm2 binds to the N-terminal domain of EBNA3C. While little is known about the tertiary and quaternary structure of EBNA3C, Mdm2 is well understood structurally and importantly in the context of structure-function relationships. The full-length transcript of the *mdm2* gene encodes a protein of 491 amino acids with a predicted molecular mass of 56 kDa, while the SDS-PAGE molecular mass is around 95 kDa due to highly posttranslational modification of the acidic domain (12). This protein contains several conserved structural domains including an N-terminal p53 interaction domain, the structure of which has been solved using X-ray crystallography (40). The Mdm2 protein also contains a central acidic domain (residues 230 to 300). The phosphorylation of residues within this domain appears to be important for regulation of Mdm2 function (5). In addition, this region contains nuclear export and import signals that are essential for proper nuclear-cytoplasmic trafficking of Mdm2 (1). Another conserved domain within the central region of Mdm2 is a zinc finger domain, the function of which is poorly understood. Mdm2 also contains a C-terminal RING domain (residues 430 to 480), which confers E3 Ub ligase activity (60). In an attempt to gain insights into the functionality of the association between Mdm2 and EBNA3C, a number of truncated Mdm2 constructs were designed and tested for their ability to bind EBNA3C by using in vivo binding experiments. The results of the coimmunoprecipitation assay clearly showed that full-length Mdm2 and the central acidic domain strongly associate with EBNA3C (Fig. 3A, bottom panel, lanes 1 and 3, respectively), whereas no binding was observed with the other truncated versions of Mdm2 or with the vector control (Fig. 3A, bottom panel, lanes 2, 4, and 5).

EBNA3C colocalizes with Mdm2 in the same cellular compartment in vivo. To visualize the interaction of Mdm2 and EBNA3C in an in vivo scenario, colocalization experiments were performed. Flag-tagged Mdm2 and untagged EBNA3C were ectopically expressed in HEK293T and U2OS cell lines. The results of the immunofluorescence analysis of EBNA3C and Mdm2 showed that they were tightly nuclear and demonstrated a stippled, punctate pattern with the exclusion of nucleoli (Fig. 4A). Mdm2 colocalized with EBNA3C at a number of spots, as visualized by yellow fluorescence. This indicates that these two proteins exist in similar nuclear compartments (Fig. 4A, top and middle panels). To visualize the interaction of these proteins in a more physiological setting, an EBV-positive LCL (LCL1) was used. At these levels of expression in transformed primary B cells, EBNA3C and Mdm2 also displayed a distinct punctate pattern with various foci that colocalized as seen with yellow fluorescence (Fig. 4A, bottom panel). For additional proof, the colocalization study was extended using truncated domains of EBNA3C, the N-terminal domain (residues 1 to 365) and, as a negative control, the

middle part (residues 366 to 620), which did not interact with Mdm2 in GST-binding assays or in in vivo coimmunoprecipitation assays. Unlike full-length EBNA3C, the staining for the N-terminal domain, while concentrated in the nucleus, was more diffuse (Fig. 4B, top panel). However, Mdm2 also colocalized with this domain of EBNA3C (Fig. 4B, top panel). In contrast, the central domain of EBNA3C (residue 366 to 620) localized completely in the nucleolus region and showed negligible colocalization with Mdm2 (Fig. 4B, bottom panel). These data corroborate the findings of the studies of in vivo and in vitro association between Mdm2 and EBNA3C, showing that these molecules localized in part to similar compartments in the cell nucleus.

EBNA3C expression leads to stabilization of Mdm2. Mdm2 is known to be overexpressed in numerous human cancers (29, 53). Elevated levels of nuclear accumulation of Mdm2 protein were reported previously in EBV-induced lymphoproliferation in severe combined immunodeficient (SCID) mice (18). In an attempt to check the Mdm2 protein expression level in the context of EBV infection and its potent nuclear antigen EBNA3C, EBV-transformed B cells (LCL2) and a Burkitt lymphoma cell line, BJAB cells, stably expressing EBNA3C (two different clones, 7 and 10) were tested along with negative-control BJAB cells. As shown in Fig. 5A, the level of Mdm2 expression was significantly increased in EBV-transformed cell line LCL2 (Fig. 5A, middle panel, lane 4) as well as BJAB cells stably expressing EBNA3C (Fig. 5A, middle panel, lanes 2 and 3) compared to the BJAB control cells (Fig. 5A, middle panel, lane 1). The effect of EBNA3C on Mdm2 steady-state levels is not due to changes in transcription, as EBNA3C expression does not alter the level of *mdm2* mRNAs in these cells (data not shown). Real-time PCR was used to compare the mRNA levels of *mdm2* in BJAB cells, two independent clones of BJAB cells stably expressing EBNA3C (clones 7 and 10), and LCL2.

To determine whether the increase in Mdm2 level was due to stabilization by EBNA3C, transiently cotransfected cells were treated with the proteasome inhibitor MG132. The results showed that the treatment with MG132 (Fig. 5B, middle panel, compare lanes 1 and 3) or the presence of EBNA3C (Fig. 5B, middle panel, compare lanes 1 and 2) led to a significant accumulation (threefold) of Mdm2 compared to that in the lanes showing no treatment or vector control. Additionally, the Mdm2 level was dramatically increased (sevenfold) when Mdm2 was coexpressed with EBNA3C and when cells were treated with MG132 compared to the level in the vector control lane (Fig. 5B, middle panel, compare lanes 1 and 4). Therefore, the increased levels of Mdm2 observed in the presence of EBNA3C in the presence or absence of MG132 may be caused by stabilization of Mdm2 by this viral antigen (Fig. 5B, middle panel, compare lanes 1 and 2). Importantly, the level of EBNA3C in these experiments (with or without MG132) was not affected (Fig. 5B, top panel, lanes 1 and 4), which indicates that EBNA3C itself is stable.

To test directly the effect of EBNA3C expression on Mdm2 stability, HEK293T cells expressing Mdm2 alone or with EBNA3C were treated with the protein synthesis inhibitor CHX for 1 to 3 h at 36 h posttransfection. Mdm2 levels were determined by Western blot analysis. Immunoblotting against total Mdm2 showed that the stability of Mdm2 protein was

A)

|  | IP: αFlag | Lysate |
| --- | --- | --- |
| EBNA3C | + | + |
| Flag-vector | - | + |
| Flag-Mdm2 | + | - |
| Flag-Mdm2 1-282 | - | + |
| Flag-Mdm2 1-198 | - | - |
| Flag-Mdm2 298-491 | - | + |

![Image Description](#)

B)

Mdm2 1
P53 binding
Acidic domain
Zinc finger domain
RING domain
491

EBNA3C
Binding

1
491
+
1
282
+
1
198
-
298
491
-

FIG. 3. The central acidic domain of Mdm2 binds to the N-terminal domain of EBNA3C. (A) Fifteen million HEK 293T cells were cotransfected with pA3M-EBNA3C, encoding full-length EBNA3C tagged with the Myc epitope, and pA3F-Mdm2 proteins, encoding either full-length Mdm2 (residues 1 to 491) or different truncations (1 to 282, 1 to 198, and 298 to 491), as indicated. All Mdm2 proteins were tagged with Flag epitope. Cells were harvested at 36 h, 5% of the lysed cells were saved as input, and the remainder were IP with 1.5 μg anti-Flag antibody (M2). Samples were resolved by 12% SDS-PAGE and transferred to an 0.45-μm nitrocellulose membrane. The membrane was probed with anti-Myc antibody (9E10) for EBNA3C. The membrane was stripped and reprobed with M2 antibody for Mdm2 proteins. Numbers at left of panels are molecular masses in kilodaltons. (B) The schematic illustrates different domains of Mdm2 and reviews the study of binding between different domains of Mdm2 and EBNA3C. +, strong binding; –, no binding. The asterisk indicates the immunoglobulin bands.

significantly enhanced by EBNA3C coexpression (Fig. 5C, right middle panel), whereas in the absence of EBNA3C, Mdm2 was almost completely degraded 2 h after addition of CHX (Fig. 5C, left middle panel). Determination of the relative percentage of Mdm2 remaining showed that the half-life of Mdm2 was increased ~5-fold with EBNA3C expression (Fig. 5C, right middle panel). The results of these experiments suggest that expression of EBNA3C can stabilize Mdm2 possibly through regulation of its degradation.

The N-terminal domain of EBNA3C is sufficient for stabilization of Mdm2. Coimmunoprecipitation experiments and in vitro binding assays suggested that the amino terminus of EBNA3C binds to the central acidic domain of Mdm2. To further define the domain or domains of EBNA3C important for Mdm2 stability and also to determine if the binding domain of EBNA3C is essential for Mdm2 stability, the above-mentioned experiments were extended using different truncated domains of EBNA3C. Three EBNA3C truncated mutants were tested; the N-terminal binding region of EBNA3C (residues 1 to 365) showed the strongest ability to increase Mdm2 expression, and the nonbinding middle region of EBNA3C (residues 366 to 620) was the weakest (Fig. 6A, top panel, compare lanes 3, 4, and 5). Quantitation of the Mdm2 bands showed that Mdm2 expression was induced about sixfold by the N-terminal domain of EBNA3C (Fig. 6A, top panel, compare lanes 1 and 3), similar to that seen with wild-type EBNA3C (Fig. 6A, top panel, compare lanes 1 and 2), and twofold by the C-terminally truncated mutant (Fig. 6A, compare lanes 1 and 5). The central domain of EBNA3C had no significant effect on the expression/stabilization of Mdm2 (Fig. 6A, top panel, compare lanes 1 and 4). Incubation with MG132 resulted in further accumulation of Mdm2 in all cases. Moreover, the enhancement was comparable to that seen without MG132 (compare Fig. 6A and B, top panels). GAPDH Western blot analysis (Fig. 6A and B, middle panels) suggested that the changes in Mdm2 levels were not due to differences in total protein levels.

To further confirm that the increase in Mdm2 level with either full-length EBNA3C or the N-terminal region was due to an increase in stability, the half-life of Mdm2 was determined. The Mdm2 degradation rate in the presence of various EBNA3C truncated mutants was determined by CHX treatment as stated above. In the presence of vector control and the central domain of EBNA3C, Mdm2 was rapidly degraded after CHX treatment (Fig. 6C, I and IV, middle panels, respectively). Mdm2 induced by the C-terminal domain of EBNA3C was also rapidly degraded with a half-life of ~2.0 h, although its expression level was much higher (Fig. 6C, V, middle panel). In contrast, Mdm2 in cells expressing either wild-type EBNA3C or the N-terminal domain was significantly stabilized, with an estimated half-life of over 3 h (Fig. 6C, II and III, middle panels, respectively). These results indicated that the N-terminal binding domain of EBNA3C (residues 1 to 365) was necessary and also sufficient for stabilization of Mdm2.

EBNA3C is a DUB. Previous studies from our laboratory have shown that EBNA3C interacts with and increases the

cellular partners. Sequence homology analysis predicted that EBNA3C contains two sequence motifs that share similarity with the cysteine and histidine boxes found in the UBP Ub-specific protease subfamily of enzymes with deubiquitinating activity (76). The N-terminal domain of EBNA3C (residues 130 to 160) contains a consensus cysteine box motif, while the consensus histidine box motif is located in the C-terminal region (residues 680 to 710). First, to test whether this viral antigen in fact is a DUB, an in vitro DUB experiment was conducted using Ub₄ as a substrate. Wild-type Myc-tagged EBNA3C, EBNA1, and HAUSP (or USP-7) were expressed in HEK 293T cells, IP, and tested for their ability to cleave Ub₄ substrate. Both wild-type EBNA3C and HAUSP readily cleaved the tetrameric substrate to its monomer (Fig. 7B, top panel, lanes 2 and 4, respectively), whereas vector control and EBNA1 showed no activity (Fig. 7B, top panel, lanes 1 and 3, respectively), similar to the substrate-alone control (Fig. 7B, top panel, lane 5). As expected, commercially available purified isopeptidase T also cleaved the Ub₄ substrate to its monomeric form (Fig. 7B, top panel, lane 6). Previous studies showed that EBNA1 interacted with a DUB, HAUSP (24, 66). However, EBNA1 did not exhibit any activity in our assay system (Fig. 7B, top panel, lane 3), indicating that HAUSP was not coimmunoprecipitated with EBNA1 and served as a negative control in this experiment. On the other hand, EBNA3C showed DUB activity somewhat similar to those of both bona fide deubiquitinases, HAUSP and isopeptidase T (Fig. 7B, top panel, lane 2 compared with lanes 4 and 6).

To gain insights into the residues of EBNA3C responsible for DUB activity, we pursued similar in vitro DUB experiments using different truncated domains of EBNA3C (Fig. 7C). Both the N-terminal domain (residues 1 to 365) containing the consensus cysteine box motif and the C-terminal domain (residues 621 to 992) including the consensus histidine box motif showed DUB activity, though at a considerably lower level than that in wild-type EBNA3C (Fig. 7D, top panel, lanes 3 and 5 compared with lane 2). The middle region (residues 366 to 620) showed no detectable activity (Fig. 7D, top panel, lane 4), indicating that, to obtain full DUB activity, the structural integrity of the full-length EBNA3C is indispensable. These results strongly suggest either that EBNA3C can act as a DUB or that it may associate with DUBs.

A new chemistry-based proteomics approach has allowed demonstration of DUB activity with specific active-site-directed probes against DUBs (7). The active-site-directed probes, which contain an epitope-tagged Ub (HAUb), can covalently modify only active DUBs. To determine whether EBNA3C is a DUB and able to form a covalent complex with the DUB-specific probe, we used the HAUbVME probe, which has broad reactivity toward DUBs (7). Lysates prepared from either BJAB cells or BJAB cells stably expressing EBNA3C were incubated with the HAUbVME probe and subjected to immunoprecipitation with specific monoclonal antibody against the HA epitope. Coimmunoprecipitation of the EBNA3C band (Fig. 7E, compare lanes 3 and 6) in cells with stable expression of this viral antigen strongly supported the possibility that EBNA3C can function as a DUB. For further confirmation, EBNA3C tagged with the Flag epitope was exogenously expressed in HEK 293T cells, IP, and incubated with this active probe. Western blot analysis against HA epitope

stability of the c-Myc oncoprotein, although the mechanistic detail was unclear (3). Here we showed that expression of EBNA3C led to stabilization of another oncoprotein, Mdm2. It has also been reported that EBNA3C is remarkably stable in actively growing LCLs (75) and as shown by in vitro pulse-chase analysis (78). It is therefore conceivable that EBNA3C may itself act as a DUB, or it may associate with either cellular or viral DUBs, which provide stability for it and associated

FIG. 5. EBNA3C expression leads to stabilization of Mdm2. (A) Twenty-five million BJAB cells or BJAB cells stably expressing EBNA3C (E3C 7 and 10) and 50 million LCL (LCL2) cells were harvested and lysed in RIPA buffer. Samples were resolved by 10% SDS-PAGE and probed for indicated antibodies. (B) Fifteen million HEK 293T cells were cotransfected with 10 μg of pRK5-HA-Mdm2, encoding HA-tagged Mdm2, and 10 μg of either pA3M vector (lanes 1 and 3) or pA3M-EBNA3C (lanes 2 and 4). At 36 h posttransfection, samples were treated with either 40 μM MG132 (+ lanes) or dimethyl sulfoxide (- lanes) for 6 h. Total samples were normalized by the Bradford assay, resolved by 10% SDS-PAGE, and probed for indicated antibodies. (C) HEK 293T cells were transfected with expression plasmids for HA-tagged Mdm2 and Myc-tagged EBNA3C. At 36 h posttransfection, cells were treated with 40 μg/ml CHX for indicated lengths of time. Ten percent of lysates from each sample were resolved by 10% SDS-PAGE. GAPDH blotting was done for the loading control. Western blotting was done by stripping and reprobing the same membrane. Specific Mdm2 bands in each gel were scanned with Odyssey imager software. All panels are representative pictures from similar repeated experiments. Numbers at left of gels and blots are molecular masses in kilodaltons.

showed that EBNA3C forms a stable covalent complex with this DUB-specific probe (Fig. 7F, bottom panel, lane 4). No specific band was detected with vector control immunoprecipitation (Fig. 7F, bottom panel, lane 2), indicating the specificity of the experiment.

EBNA3C deubiquitinates itself in a cell-free system and also in human cells. We previously showed that EBNA3C is ubiquitinated at its N-terminal domain (37). Importantly, the protein turnover rate was not affected in actively proliferating LCLs (75). The deubiquitination activity of EBNA3C led us to test further whether full-length EBNA3C is able to deubiquitinate itself. We first determined whether EBNA3C is able to function as a DUB on itself as the substrate. HEK 293T cells were transfected with Flag-tagged N-terminal EBN3C (residues 1 to 365) along with HAUb. In a separate transfection, cells were transfected with either Myc-tagged full-length EBNA3C or different truncated mutants as indicated (Fig. 8A). Two other proteins, Myc-tagged EBNA1 and HAUSP, were used as negative and positive controls, respectively. Immunoprecipitations were done separately with appropriate antibodies, and IP complexes were mixed and incubated together. Western blot analysis showed, upon incubation with EBNA3C or HAUSP, a dramatic reduction in levels of polyubiquitination of the N-terminal domain of EBNA3C compared to the vector control (Fig. 8A, left top panel, compare lanes 3, 4, and 7). This effect was completely diminished when the enzymatically inactive mutant, i.e., residues 366 to 992 of the UBP consensus cysteine box-deleted mutant EBNA3C, was used instead of wild-type EBNA3C (Fig. 8A, lane 5). Similar to previous results, EBNA1 showed no activity on polyubiquitination of N-terminal EBNA3C, which provided additional evidence for the specificity of this activity. The expression levels of Myc-tagged proteins are shown in Fig. 8B.

To confirm this activity in an in vivo setting, the ubiquitinated Flag-tagged EBNA3C N-terminal domain was IP from HEK 293T cell lysates, expressed with either Myc-tagged full-length or different EBNA3C truncated mutants (Fig. 8C). Similarly, Myc-tagged EBNA1 and HAUSP were also transfected as controls. The in vivo deubiquitination data also showed results similar to the in vitro results above. In addition, the deubiquitination activity of EBNA3C was substantially reduced by replacing the conserved catalytic residue cysteine 143 with asparagine (Fig. 8D, left top panel, compare lanes 3 and 4). These results clearly demonstrated that wild-type EBNA3C expeditiously deubiquitinates polyubiquitinated EBNA3C as a substrate in a cell-free system and in vivo.

EBNA3C deubiquitinates and stabilizes Mdm2 in vivo. Mdm2 is a short-lived protein, and as a functional E3 ligase,

Vol. 83, 2009                                                                 EBNA3C DEUBIQUITINATES Mdm2 4661

A)

| WB: $\alpha H A$ | Vector | EBNA3C | E3C 1-365 | E3C 366-620 | E3C 621-992 |
| --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |
| WB: $\alpha G A P D H$ |  |  |  |  |  |
|  |  |  |  |  |  |
| WB: $\alpha M y c$ |  |  |  |  |  |
|  |  |  |  |  |  |

B)

| WB: $\alpha H A$ | Vector | EBNA3C | E3C 1-365 | E3C 366-620 | E3C 621-992 |
| --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |
| WB: $\alpha G A P D H$ |  |  |  |  |  |
|  |  |  |  |  |  |
| WB: $\alpha M y c$ |  |  |  |  |  |
|  |  |  |  |  |  |

C)

I.

| CHX (H) | 0 | 1 | 2 | 3 |
| --- | --- | --- | --- | --- |
| WB: $\alpha M y c$ |  |  |  |  |
| WB: $\alpha H A$ |  |  |  |  |
| WB: $\alpha G A P D H$ |  |  |  |  |

II.

|  | 0 | 1 | 2 | 3 |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |
|  |  |  |  |  |

III.

|  | 0 | 1 | 2 | 3 |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |
|  |  |  |  |  |

IV.

| CHX (H) | 0 | 1 | 2 | 3 |
| --- | --- | --- | --- | --- |
| WB: $\alpha M y c$ |  |  |  |  |
| WB: $\alpha H A$ |  |  |  |  |
| WB: $\alpha G A P D H$ |  |  |  |  |

V.

|  | 0 | 1 | 2 | 3 |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |
|  |  |  |  |  |

D)

| EBNA3C 1 | JK | LZ | Acidic domain | Cyclin A | DEDED | P-rich domain | Q-rich domain | NLS | 992 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |  |  |  |  |

Mdm2

Stability

|  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 |  |  |  |  |  |  |  |  | 992 |
|  |  |  |  |  |  |  |  |  | ++ |
| 1 |  |  |  |  |  |  |  |  | 365 |
|  |  |  |  |  |  |  |  |  | ++ |
|  |  |  |  |  |  |  |  |  | 620 |
|  |  |  |  |  |  |  |  |  | - |
|  |  |  |  |  |  |  |  |  | 992 |
|  |  |  |  |  |  |  |  |  | +/- |

FIG. 6. The N terminus of EBNA3C stabilizes Mdm2. (A and B) Fifteen million HEK 293T cells were cotransfected with 10 μg of pRK5-HA-Mdm2, encoding HA-tagged Mdm2, and 10 μg of either pA3M vector or pA3M-EBNA3C proteins (full-length and different truncated versions) as indicated. All EBNA3C proteins were tagged with a Myc epitope. At 36 h posttransfection, samples were treated with either 40 μM MG132 (B) or dimethyl sulfoxide control (A) for 6 h. Total samples were normalized by the Bradford assay, resolved by 10% SDS-PAGE, and probed for indicated antibodies. (C) HEK 293T cells were transfected with expression plasmids for HA-tagged Mdm2 and either vector control or different Myc-tagged EBNA3C constructs expressing either amino acids 1 to 365, amino acids 366 to 620, or amino acids 621 to 992. At 36 h posttransfection, cells were treated with 40 μg/ml CHX for indicated lengths of time. Samples were resolved by 10% SDS-PAGE. GAPDH blotting was done for loading control. Western blotting was done by stripping and reprobing the same membrane. All panels are representative pictures from similar repeated experiments. Specific Mdm2 bands in each gel were scanned with Odyssey imager software. Numbers at left of blots and gels are molecular masses in kilodaltons. (D) The schematic illustrates different domains of Mdm2 and summarizes the assay of stability of Mdm2 for different EBNA3C domains. ++, maximum stability; +/−, moderate stability; −, minimum stability. NLS, nuclear localization signal.

Mdm2 is capable of self-ubiquitination and degradation (19, 25). The enhanced stability of Mdm2 in the presence of EBNA3C prompted us to examine whether EBNA3C deubiquitinates Mdm2 to enhance its stability. An in vivo deubiquitination experiment was set up in which HEK 293T cells were cotransfected with expression constructs for HA-tagged

Ub, Flag-tagged Mdm2, and either Myc-tagged full-length EBNA3C or different EBNA3C truncated mutants as indicated (Fig. 9A). Similar to the previous results, full-length EBNA3C also exhibited efficient deubiquitination on Mdm2 (Fig. 9A, left top panel, compare lanes 2 and 3). Neither the truncated domain (residues 1 to 365 containing the consensus cysteine box),

FIG. 7. EBNA3C can function as a DUB. (A) Alignment of EBNA3C fragments (lower row) with the consensus sequences of Ub protease (UBP) family proteins (upper row) using the T-Coffee method (55). Highlighted residues indicate cysteine (C) and histidine (H) amino acids implicated in the catalytic mechanism. (B) Myc-tagged full-length EBNA3C, EBNA1, HAUSP, or vector (pA3M) was IP with the Myc-specific antibody (9E10) from lysates of transfected HEK 293T cells and tested for the ability to cleave Ub₄. After overnight incubation at 37°C, the reaction supernatants were analyzed by 15% SDS-PAGE and silver staining (top panel). Substrate control (Ub₄) and positive control (isopeptidase T [IsopT]) are shown in lanes 5 and 6 and in lane 6, respectively. Lines indicate the positions of tetrameric (Ub₄), trimeric (Ub₃), dimeric (Ub₂), and monomeric Ub. Specific mono-Ub bands were scanned with Odyssey imager software. The amount of IP Myc-tagged proteins was determined by immunoblotting with 9E10 (bottom panel). The asterisk indicates the immunoglobulin bands. (C) Cartoon diagrams of different EBNA3C truncated fragments. Squares show the positions of consensus cysteine and histidine domains in EBNA3C. (D) Myc-tagged full-length EBNA3C or different truncated fragments of EBNA3C as indicated in panel C were subjected to an in vitro deubiquitination assay as described for panel B. The reaction supernatants were analyzed by 15% SDS-PAGE followed by silver staining (top panel). Different forms of Ub are shown by lines on the right side of the gel. The bottom panel shows the Western blot analysis of the total amount of immunoprecipitated Myc-tagged EBNA3C proteins. The asterisk indicates the immunoglobulin bands. (E) Approximately 200 μg of lysates prepared from either BJAB cells or BJAB cells stably expressing EBNA3C was incubated with HAUbVME probe (1 μg/μl) for 2 h at 37°C and subjected to immunoprecipitation with specific monoclonal antibody against HA epitope (12CA5). Samples were resolved by 7% SDS-PAGE and transferred to an 0.45-μm nitrocellulose membrane. The membrane was probed with anti-EBNA3C antibody (A10). NS, nonspecific band. (F) HEK 293T cells were transfected with either 15 μg of full-length EBNA3C-expressing construct or empty vector. Cells were harvested at 36 h posttransfection, approximately 5% of the lysed cells were saved as input, and the remainder were IP with 1.5 μg of M2 antibody against Flag epitope. IP proteins were incubated with HAUbVME probe (1 μg/μl) for 2 h at 37°C in labeling buffer (see Materials and Methods). Samples were divided into halves, resolved by 7% SDS-PAGE, and subjected to Western blotting with anti-HA antibody (top panel) and A10 for detection of EBNA3C (bottom panel). Numbers at left of blots and gels are molecular masses in kilodaltons.

the central domain (residues 366 to 620), nor the C-terminal domain (residues 621 to 992, which include the consensus histidine box) of EBNA3C showed deubiquitinating activity toward Mdm2 polyubiquitination (Fig. 9A, left top panel, compare lanes 4, 5, and 6 with lane 2), indicating the specificity of this experiment. However, in the presence of the N-terminal domain of EBNA3C (residues 1 to 365), Mdm2 polyubiquitination is significantly increased (Fig. 9A, left top panel, compare lanes 2 and 4), which is not surprising, as the EBNA3C N-terminal binding domain is probably self-ubiquitinated and is coimmunoprecipitated with Mdm2. In a separate experiment, the catalytically inactive mutant of EBNA3C (C143N) showed no detectable activity (Fig. 9B, left top panel, compare lanes 2 and 4), providing additional supportive evidence.

Vol. 83, 2009                                                                 EBNA3C DEUBIQUITINATES Mdm2 4663

A)

| IP: $a_{myc}$ | myc-vector | myc-EBNA3C | myc-E3C 366–992 | myc-EBNA1 | myc-HAUSP |
| --- | --- | --- | --- | --- | --- |
| IP: $a_{flag}$ | + | + | + | + | + |
| Flag-E3C 1–365 | + | + | + | + | + |
| HA-Ub | - | + | + | + | + |

Incubate  
O/N at 37°C  

Lysate  
+ + + + + +  
- + + + + +  

WB: $a_{HA}$  
Poly-Ub-N-term  
*  

WB: $a_{flag}$  
55-  

WB: $a_{myc}$  
170-  
130-  
95-  
72-  

E3C 1–365  
E3C 1–992  
E3C 366–992  
EBNA1  

C)  

| IP: $a_{flag}$ | Flag-E3C 1–365 | HA-Ub | myc-EBNA3C | myc-E3C 1–365 | myc-E3C 366–620 | myc-E3C 621–992 | myc-E3C 366–992 | myc-EBNA1 | myc-HAUSP |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| + | + | + | + | + | + | + | + | + | + |
| - | + | + | + | + | + | + | + | + | + |
| - | - | + | - | - | - | - | - | - | - |
| - | - | + | - | - | - | - | - | - | - |
| - | - | - | + | - | - | - | - | - | - |
| - | - | - | - | + | - | - | - | - | - |
| - | - | - | - | - | + | - | - | - | - |
| - | - | - | - | - | - | + | - | - | - |
| - | - | - | - | - | - | - | + | - | - |
| - | - | - | - | - | - | - | - | + | - |

IP: $a_{Flag}$  
WB: $a_{HA}$  
Poly-Ub-N-term  
*  

IP: $a_{Flag}$  
WB: $a_{Flag}$  

IP: $a_{Flag}$  
WB: $a_{Myc}$  

E3C 1–365  

Total ubiquitination  

Lysate  
5%  

WB: $a_{HA}$  

WB: $a_{Flag}$  

WB: $a_{Myc}$  

B)  

| Lysate | myc-vector | myc-EBNA3C | myc-E3C 366–992 | myc-EBNA1 | myc-HAUSP |
| --- | --- | --- | --- | --- | --- |
| + | - | - | - | - | - |
| - | + | - | - | - | - |
| - | - | + | - | - | - |
| - | - | - | + | - | - |
| - | - | - | - | + | - |

WB: $a_{myc}$  

D)  

| IP: $a_{Flag}$ | Flag-E3C 1–365 | HA-Ub | EBNA3C | EBNA3C C143N |
| --- | --- | --- | --- | --- |
| + | + | + | + | + |
| - | + | + | + | + |
| - | - | + | - | + |
| - | - | - | + | - |

Lysate  
+ | + | + | + | +  

WB: $a_{HA}$  

Total Ubiquitination  

WB: $a_{Flag}$  

WB: $a_{A10}$  

FIG. 8. EBNA3C deubiquitates itself in a cell-free system as well as in vivo. DUBs (Myc-tagged proteins as indicated) and the ubiquitinated substrate (Flag-tagged EBNA3C 1 to 365 plus HA-tagged Ub) were expressed separately in HEK 293T cells. After immunoprecipitation with appropriate antibodies, products were washed and mixed together. The deubiquitination reaction was carried out in 100 μl of deubiquitination buffer (see Materials and Methods) at 37°C overnight (O/N). The same blots were reprobed with M2 and 9E10 for Flag-tagged and Myc-tagged proteins, respectively. The asterisks indicate the immunoglobulin bands, and arrowheads show the corresponding bands of Flag-tagged EBNA3C 1 to 365. (B) The amount of expressed Myc-tagged proteins was determined by immunoblotting with 9E10. (C) HEK 293T cells were transfected with expression vectors for HA-tagged Ub, Flag-tagged EBNA3C 1 to 365, Myc-tagged full-length EBNA3C and its various truncations as indicated, Myc-tagged EBNA1, and Myc-tagged HAUSP. Flag-tagged EBNA3C 1 to 365 proteins were IP with the M2 antibody from lysates of transfected cells prepared 36 h after transfection and analyzed by immunoblotting with HA-specific antibody (12CA5). Five percent of the whole-cell extracts of the transfected cells were subjected to SDS-PAGE and immunoblotting with 12CA5 to determine the overall extent of ubiquitination. The same blots were stripped and reprobed with M2 and 9E10 for Flag-tagged and Myc-tagged proteins, respectively. Asterisks indicate the immunoglobulin bands. IP proteins in whole-cell extracts are shown by arrowheads. (D) Ubiquitinated Flag-tagged EBNA3C 1 to 365 protein was IP similarly to what was described for panel C in the presence of either vector control (lanes 2), full-length EBNA3C (lanes 3), or catalytic mutant of EBNA3C (C143N; lanes 4) and probed for appropriate antibodies as indicated. All panels are representative pictures from similar repeated experiments. Numbers at left of blots or gels are molecular masses in kilodaltons transfected cells prepared 36 h after transfection and analyzed by immunoblotting with HA-specific antibody (12CA5). Five percent of the whole-cell extracts of the transfected cells were subjected to SDS-PAGE and immunoblotting with 12CA5 to determine the overall extent of ubiquitination. The same blots were stripped and reprobed with M2 and 9E10 for Flag-tagged and Myc-tagged proteins, respectively. Asterisks indicate the immunoglobulin bands. IP proteins in whole-cell extracts are shown by arrowheads. (D) Ubiquitinated Flag-tagged EBNA3C 1 to 365 protein was IP similarly to what was described for panel C in the presence of either vector control (lanes 2), full-length EBNA3C (lanes 3), or catalytic mutant of EBNA3C (C143N; lanes 4) and probed for appropriate antibodies as indicated. All panels are representative pictures from similar repeated experiments. Numbers at left of blots or gels are molecular masses in kilodaltons.

FIG. 9. EBNA3C deubiquitates Mdm2. (A and B) Fifteen million HEK 293T cells were transfected with expression vectors for HA-tagged Ub, Flag-tagged Mdm2, and Myc-tagged full-length EBNA3C and its various truncated mutants (A) or catalytically inactivated mutant EBNA3C C143N (B) as indicated. Flag-tagged Mdm2 was IP with M2 antibody against Flag epitope from lysates of transfected cells prepared 36 h after transfection and was analyzed by immunoblotting with HA-specific antibody, 12CA5 (A and B, top panels). Five percent of the whole-cell extracts of the transfected cells were subjected to SDS-PAGE and immunoblotting with 12CA5 to determine the overall extent of ubiquitination. The same blots were stripped and reprobed with M2 and 9E10 for Flag-tagged and Myc-tagged proteins, respectively. The asterisk in panel A indicates the immunoglobulin bands, and arrowheads show the corresponding bands of Flag-tagged Mdm2. Numbers at left of panels are molecular masses in kilodaltons. (C) U2OS cells were transfected with HA-tagged Ub, Flag-tagged Mdm2, and Myc-tagged EBNA3C expression constructs (wild type and C143N catalytic mutant). At 36 h posttransfection, cells were fixed with a 1:1 acetone and methanol mixture and an immunofluorescence assay was done with appropriate monoclonal antibodies and DAPI.

Previous studies have shown that ubiquitination of many nuclear proteins led to cytoplasmic relocalization (11). It is possible that Ub may serve as a unique and specific signal in the nucleocytoplasmic shuttling of these nuclear proteins. To visualize the deubiquitinating effect of EBNA3C on Mdm2 ubiquitination, a similar immunofluorescence staining strategy was utilized in which U2OS cells were transfected with appropriate constructs as shown in Fig. 9C. In the presence of Ub,

FIG. 10. EBNA3C forms a ternary complex with p53 and Mdm2 and facilitates p53 ubiquitination by Mdm2. (A and B) Fifteen million MEF cells (p53$^{-/-}$ Mdm2$^{-/-}$) were cotransfected with Flag-tagged EBNA3C, HA-tagged Mdm2, and Myc-tagged p53 expression constructs. In each case control samples were balanced with empty vector. Cells were harvested at 36 h posttransfection, approximately 5% of the lysed cells were saved as input, and the remainder were IP with 1.5 μg of M2 antibody. Lysates and IP complexes were resolved by 10% SDS-PAGE and subjected to Western blotting (WB) with the indicated antibodies. The same blots were stripped and reprobed with appropriate antibodies. (B) Fifty million BJAB cells and BJAB cells stably expressing EBNA3C (clone 10) were collected and lysed in RIPA buffer, and protein complexes were IP with p53-specific antibody (αp53). Samples were resolved by 8% SDS-PAGE, and Western blotting for the indicated proteins was done by stripping and reprobing the same membrane. (C) Saos-2 cells were transfected with expression plasmids for HA-tagged Mdm2, untagged EBNA3C, and Myc-tagged p53. At 36 h posttransfection, cells were treated with 40 μg/ml CHX for indicated lengths of time. Ten percent of lysates from each sample were resolved by 10% SDS-PAGE. GAPDH blotting was done for the loading control. Western blotting was done by stripping and reprobing the same membrane. Specific p53 bands in each gel were scanned with Odyssey imager software. (D) Saos-2 cells were transfected with expression vectors for HA-tagged Ub, Flag-tagged Mdm2, untagged EBNA3C, and Myc-tagged p53. Myc-tagged p53 was IP with 9E10 antibody from lysates of transfected cells prepared 36 h after transfection and analyzed by immunoblotting with HA-specific antibody. Five percent of the whole-cell extracts of the transfected cells were subjected to SDS-PAGE and immunoblotting with 12CA5 to determine the overall extent of ubiquitination. The same blots were stripped and reprobed with appropriate antibodies as indicated. All panels are representative gels from similar repeated experiments. Numbers at left of panels are molecular masses in kilodaltons.

immunostaining of Mdm2 showed diffuse cytoplasmic localization with a variable degree of nuclear localization (Fig. 9C, compare third and fourth rows). The nuclear localization pattern of EBNA3C in the presence of Ub was indistinguishable from the pattern where Ub was absent (Fig. 9C, compare first and second rows). Interestingly, coexpression of EBNA3C resulted in a total reduction in cytoplasmic immunostaining of Mdm2 in the presence of Ub (Fig. 9C, compare fourth and fifth rows), suggesting that EBNA3C might have a distinct role in Mdm2 deubiquitination and relocalization. However, coexpression of a catalytically inactive mutant of EBNA3C (C143N) showed no change in Mdm2 relocalization in the presence of Ub (Fig. 9C, compare fourth and sixth), further supporting the above results.

EBNA3C forms a ternary complex with p53 and Mdm2 and facilitates ubiquitination of p53 by Mdm2. To elucidate the functional relationship between Mdm2 and EBNA3C in regulating p53 activities, we first determined whether EBNA3C can form a ternary complex with Mdm2 and p53. Both p53- and Mdm2-deficient MEF cells (p53$^{-/-}$ Mdm2$^{-/-}$) were transfected with expression constructs for Flag-tagged EBNA3C,

HA-tagged Mdm2, and Myc-tagged p53 as indicated (Fig. 10A). EBNA3C distinctly coprecipitated p53 and Mdm2 individually (Fig. 10A, lane 2, left bottom panel, and lane 6, left middle panel, respectively). The result also showed that EBNA3C can form a ternary complex with p53 and Mdm2 (Fig. 10A, lane 4, left middle and bottom panels), while the EBNA3C-Mdm2 interaction was significantly reduced by exogenous expression of p53 (Fig. 10A, compare lanes 4 and 6, left middle panel), indicating that the association of EBNA3C with Mdm2 and that with p53 probably share an overlapping domain. In another separate experiment, the interaction among EBNA3C, Mdm2, and p53 was further validated by immunoprecipitation of p53 using BJAB cells and BJAB cells stably expressing EBNA3C (Fig. 10B). These results again suggested that EBNA3C can target both p53 and Mdm2 for deregulating the cellular functions of these proteins. This probably occurs through distinct mechanisms.

We have shown earlier that EBNA3C recruited the SCF${}^{Skp2}$ E3 ligase machinery to target the ubiquitination and degradation of two important cell cycle regulators, p27${}^{KIP}$ and pRb (36, 37). To test whether EBNA3C also regulates p53 ubiquitina-

meostasis in all eukaryotes are controlled not only by phosphorylation and dephosphorylation but also by ubiquitination and deubiquitination.

A prominent demonstration of this concept is the deubiquitination activity exhibited by EBNA3C, which is capable of efficiently deubiquitinating itself. It has also been earlier reported that EBNA3C has cell cycle-deregulatory functions, apparently mediated by direct protein-protein interactions (35, 36, 37). Recently, we demonstrated that EBNA3C targets numerous cell cycle-regulatory components such as Rb and p27 (36, 37) for their destabilization by recruiting the E3 Ub ligase complex machinery (SCF<sup>Skp2</sup>) and also by induction of self-ubiquitination at the N-terminal domain (37). EBNA3C has also been found to interact with and to be degraded in vitro by the purified 20S proteasome complex (75). However, no indication of proteasome-mediated degradation has been detected in actively growing LCLs (75). In another report, West also demonstrated similar results using pulse-chase analysis (78). The mechanistic reasons for the stability of this viral antigen after being ubiquitinated are still unclear. It is possible that in EBV-infected cells, EBNA3C forms a complex with cellular DUB, or it may itself act as a DUB that offers protection from its proteolysis. Despite the controversy regarding the different possibilities for EBNA3C stability, it is clear that in order to deregulate the entire mammalian cell cycle, EBNA3C potentially manipulates the Ub-proteasome machinery. We have shown recently that, in contrast to destabilizing proteins (36, 37), EBNA3C also stabilizes the cellular oncoprotein c-Myc, although the molecular mechanism by which this occurs is still not clear (3). Here we report that, in addition to deubiquitinating itself, EBNA3C can also efficiently deubiquitinate Mdm2, an oncoprotein which is known to be overexpressed in many cancers (29, 53). Mdm2 is also overexpressed in certain tumors with wild-type p53 (29), suggesting that it may contribute to tumor development by inactivation of p53. However, Mdm2 also has p53-independent activities that may play a role in malignant transformation (28, 47). In addition to deregulating p53 activity, Mdm2 has been shown to interact with and inactivate the retinoblastoma (Rb) tumor suppressor protein (79). Furthermore, it can also modulate the activity, stability, and apoptotic function of E2F1/DP1 transcription factors (4, 46, 49). Therefore, Mdm2 can enhance cellular transformation by both p53-dependent and -independent mechanisms.

Our studies show that EBNA3C forms a complex with Mdm2. In vitro binding assays showed that EBNA3C directly interacts with Mdm2, and it also forms a stable complex in vivo. EBNA3C associates with Mdm2 via the same domain, residues 130 to 190, that has been suggested to bind many components important to the cell cycle (3, 35, 37). Our results also demonstrate that EBNA3C forms a ternary complex with p53 and Mdm2 (Fig. 11), although in the presence of p53 the association of Mdm2 with EBNA3C was significantly reduced. This suggests that the association of EBNA3C with Mdm2 and p53 probably shares an overlapping domain and that the binding affinity of p53 toward EBNA3C is greater than the affinity for Mdm2.

Studies regarding the deubiquitinating activity of EBNA3C began with the observation that cotransfection with an EBNA3C-expressing construct significantly accelerated Mdm2 expression compared to the vector control. Further, our stud-

ies show that the binding domain of EBNA3C is sufficient to stabilize Mdm2. The participation of EBNA3C with Mdm2 and p53 in the ternary complex, combined with our finding that EBNA3C blocks proteasomal degradation of Mdm2, prompted us to investigate the possibility that EBNA3C is involved in recruiting Mdm2 E3 ligase activity for accelerated ubiquitination and subsequent turnover of p53. Our results indeed show that both ubiquitination and destabilization of p53 are accelerated in the presence of the EBNA3C-Mdm2 complex. Recent studies have shown, however, that the RING finger domain of Mdm2, where the Ub E3 ligase activity resides, is necessary but not sufficient for p53 ubiquitination (31). The central acidic domain acts as an additional essential contributor, apart from the RING finger domain of Mdm2 (31). The contribution of the Mdm2 acidic domain to the regulation of Mdm2-mediated p53 degradation has been extensively studied. Two tumor suppressors, ARF and Rb, interact with this acidic domain and stabilize p53 (26, 58, 67, 81). In contrast, binding of the transcriptional coactivator p300 to the acidic region stimulates Mdm2-mediated p53 degradation (21). There are other proteins that bind to the Mdm2 internal region and that could regulate p53 and Mdm2 turnover, including the ribosomal protein L5 (17, 48), TATA-binding protein (43), and the recently identified Mdm2 binding protein (8). In agreement with these studies, our results also show that EBNA3C binds to this acidic domain, which positively regulates Mdm2-mediated p53 ubiquitination.

Recently, a death-domain-associated protein (DAXX), together with HAUSP, has been demonstrated to be critically involved in reducing Mdm2 ubiquitination and thereby promoting enhancement of p53 ubiquitination and degradation in unstressed conditions (71). In EBV-infected cells EBNA3C may function in a similar manner, and this is currently under investigation in our lab. Interestingly, a number of recent studies have suggested a complicated role of HAUSP in regulation of both p53 and Mdm2 (44). HAUSP has been found to interact with p53, leading to its deubiquitination. However, the ablation of HAUSP expression resulted in p53 accumulation as well as a noticeable reduction in Mdm2 expression. This indicates that HAUSP has a key function in regulating Mdm2 stability (52). Studies have also shown that HAUSP is targeted by one of the EBV latent antigens, EBNA1, for lowering the levels of p53 (24, 66). It would be extremely interesting to investigate the further possibility of HAUSP involvement in the complex regulatory role of EBNA3C for Mdm2 and p53 stability in EBV-infected cells.

In summary, our data provide the first evidence that EBNA3C can function as a DUB which efficiently deubiquitates itself both in vitro and in vivo. Additionally, we also show that EBNA3C binds and deubiquitinates an important cellular proto-oncogene, Mdm2, which contributes to its stability and cellular accumulation. This facilitates p53 ubiquitination and degradation, although it remains to be clarified whether degradation of p53 is dependent on its binding to EBNA3C (Fig. 11). These findings lead us to speculate that the complex regulation of p53-Mdm2 functions by EBNA3C serves to augment the efficiency of EBV-mediated lymphomagenesis. This study is a significant step toward new insights in our understanding of the importance of EBNA3C in the development of EBV-associated cancers and effective therapies.


ACKNOWLEDGMENTS

We thank George Mosialos, Aart G. Jochemsen, Wafik S. El-Deiry, and Xiaolu Yang for generously providing reagents. We express our gratitude to Hidde Ploegh for providing the HAUbVME probe.

The project is supported by NCI grant CA137894-01 to E.S.R. E.S.R. is a scholar of the Leukemia and Lymphoma Society of America.


REFERENCES

1. Argentini, M., N. Barboule, and B. Wasylyk. 2001. The contribution of the acidic domain of MDM2 to p53 and MDM2 stability. Oncogene 20:1267–1275.
2. Arvanitakis, L., N. Yaseen, and S. Sharma. 1995. Latent membrane protein-1 induces cyclin D2 expression, pRb hyperphosphorylation, and loss of TGF-beta 1-mediated growth inhibition in EBV-positive B cells. J. Immunol. 155:1047–1056.
3. Bajaj, B. G., M. Murakami, Q. Cai, S. C. Verma, K. Lan, and E. S. Roberts. 2008. Epstein-Barr virus nuclear antigen 3C interacts with and enhances the stability of the c-Myc oncoprotein. J. Virol. 82:4082–4090.
4. Blattner, C., A. Sparks, and D. Lane. 1999. Transcription factor E2F-1 is upregulated in response to DNA damage in a manner analogous to that of p53. Mol. Cell. Biol. 19:3704–3713.
5. Blattner, C., T. Hay, D. W. Meek, and D. P. Lane. 2002. Hypophosphorylation of Mdm2 augments p53 stability. Mol. Cell. Biol. 22:6170–6182.
6. Bond, G. L., W. Hu, E. E. Bond, H. Robins, S. G. Lutzker, N. C. Arva, J. Bargonetti, F. Bartel, H. Taubert, P. Wuerl, K. Onel, L. Yip, S. J. Hwang, L. C. Strong, G. Lozano, and A. J. Levine. 2004. A single nucleotide polymorphism in the MDM2 promoter attenuates the p53 tumor suppressor pathway and accelerates tumor formation in humans. Cell 119:591–602.
7. Borodovsky, A., H. Ovaa, N. Kolli, T. Gan-Erdene, K. Wilkinson, H. Ploegh, and B. Kessler. 2002. Chemistry-based functional proteomics reveals novel members of the deubiquitinating enzyme family. Chem. Biol. 9:1149–1159.
8. Boyd, M. T., N. Vlatkovic, and D. S. Haines. 2000. A novel cellular protein (MTBP) binds to MDM2 and induces a G1 arrest that is suppressed by MDM2. J. Biol. Chem. 275:31883–31890.
9. Brooks, C. L., and W. Gu. 2006. p53 ubiquitination: Mdm2 and beyond. Mol. Cell 21:307–315.
10. Cahilly-Snyder, L., T. Yang-Feng, U. Francke, and D. L. George. 1987. Molecular analysis and chromosomal mapping of amplified genes isolated from a transformed mouse 3T3 cell line. Somatic Cell Mol. Genet. 13:235–244.
11. Carter, S., O. Bischof, A. Dejean, and K. H. Vousden. 2007. C-terminal modifications regulate MDM2 dissociation and nuclear export of p53. Nat. Cell Biol. 9:428–435.
12. Chen, J., V. Marechal, and A. J. Levine. 1993. Mapping of the p53 and mdm-2 interaction domains. Mol. Cell. Biol. 13:4107–4114.
13. Chen, J., X. Wu, J. Lin, and A. J. Levine. 1996. mdm-2 inhibits the G1 arrest and apoptosis functions of the p53 tumor suppressor protein. Mol. Cell. Biol. 16:2445–2452.
14. Chen, L., and J. Chen. 2003. MDM2-ARF complex regulates p53 sumoylation. Oncogene 22:5348–5357.
15. Clements, G. B., G. Klein, and S. Povey. 1975. Production by EBV infection of an EBNA-positive subline from an EBNA-negative human lymphoma cell line without detectable EBV DNA. Int. J. Cancer 16:125–133.
16. Cotter, M. A., II, and E. S. Robertson. 2000. Modulation of histone acetyltransferase activity through interaction of Epstein-Barr nuclear antigen 3C with prothymosin alpha. Mol. Cell. Biol. 20:5722–5735.
17. Elenbaas, B., M. Dobbelstein, J. Roth, T. Shenk, and A. J. Levine. 1996. The MDM2 oncoprotein binds specifically to RNA through its RING finger domain. Mol. Med. 2:439–451.
18. El Mansouri, S., A. Martin, A. Mercadier, C. Capoulade, V. Maréchal, J. Wiels, J. Feuillard, and M. Raphaël. 1999. High expression of MDM2 protein and low rate of p21(WAF1/CIP1) expression in SCID mice Epstein Barr virus-induced lymphoproliferation. J. Histochem. Cytochem. 47:1315–1322.
19. Fang, S., J. P. Jensen, R. L. Ludwig, K. H. Vousden, and A. M. Weissman. 2000. Mdm2 is a RING finger-dependent ubiquitin protein ligase for itself and p53. J. Biol. Chem. 275:8945–8951.
20. Graham, F. L., J. Smiley, W. C. Russell, and R. Nairn. 1977. Characteristics of a human cell line transformed by DNA from human adenovirus type 5. J. Gen. Virol. 36:59–74.
21. Grossman, S. R., M. Perez, A. L. Kung, M. Joseph, C. Mansur, Z. X. Xiao, S. Kumar, P. M. Howley, and D. M. Livingston. 1998. p300/MDM2 complexes participate in MDM2-mediated p53 degradation. Mol. Cell 2:405–415.
22. Haupt, Y., R. Maya, A. Kazaz, and M. Oren. 1997. Mdm2 promotes the rapid degradation of p53. Nature 387:296–299.
23. Hershko, A., and A. Ciechanover. 1998. The ubiquitin system. Annu. Rev. Biochem. 67:425–479.
24. Holowaty, M. N., M. Zeghouf, H. Wu, J. Tellam, V. Athanasopoulos, J. Greenblatt, and L. Frappier. 2003. Protein profiling with Epstein-Barr nu-

clear antigen-1 reveals an interaction with the herpesvirus-associated ubiquitin-specific protease HAUSP/USP7. J. Biol. Chem. 278:29987–29994.

25. Honda, R., and H. Yasuda. 2000. Activity of MDM2, a ubiquitin ligase, toward p53 or itself is dependent on the RING finger domain of the ligase. Oncogene 19:1473–1476.

26. Hsieh, J. K., F. S. Chan, D. J. O'Connor, S. Mitnacht, S. Zhong, and X. Lu. 1999. RB regulates the stability and the apoptotic function of p53 via MDM2. Mol. Cell 3:181–193.

27. Johannsen, E., C. L. Miller, S. R. Grossman, and E. Kieff. 1996. EBNA-2 and EBNA-3C extensively and mutually exclusively associate with RBPJk in Epstein-Barr virus-transformed B lymphocytes. J. Virol. 70:4179–4183.

28. Jones, S. N., A. R. Hancock, H. Vogel, L. A. Donehower, and A. Bradley. 1998. Overexpression of Mdm2 in mice reveals a p53-independent role for Mdm2 in tumorigenesis. Proc. Natl. Acad. Sci. USA 95:15608–15612.

29. Juven-Gershon, T., and M. Oren. 1999. Mdm2: the ups and downs. Mol. Med. 5:71–83.

30. Kamijo, T., J. D. Weber, G. Zambetti, F. Zindy, M. F. Roussel, and C. J. Sherr. 1998. Functional and physical interactions of the ARF tumor suppressor with p53 and Mdm2. Proc. Natl. Acad. Sci. USA 95:8292–8297.

31. Kawai, H., D. Wiederschain, and Z. M. Yuan. 2003. Critical contribution of the MDM2 acidic domain to p53 ubiquitination. Mol. Cell. Biol. 23:4939–4947.

32. Kempkes, B., D. Spitkovsky, P. Jansen-Durr, J. W. Ellwart, E. Kremmer, H. J. Delecluse, C. Rottenberger, G. W. Bornkamm, and W. Hammer-schmidt. 1995. B-cell proliferation and induction of early G1-regulating proteins by Epstein-Barr virus mutants conditional for EBNA2. EMBO J. 14:88–96.

33. Khosravi, R., R. Maya, T. Gottlieb, M. Oren, Y. Shiloh, and D. Shkedy. 1999. Rapid ATM-dependent phosphorylation of MDM2 precedes p53 accumulation in response to DNA damage. Proc. Natl. Acad. Sci. USA 96:14973–14977.

34. Kieff, E. 1996. Epstein-Barr virus and its replication, p. 2343–2396. In B. N. Fields, D. M. Knipe, and P. M. Howley (ed.), Fields virology, 3rd ed. Lippincott-Raven Publishers, Philadelphia, PA.

35. Knight, J. S., and E. S. Robertson. 2004. Epstein-Barr virus nuclear antigen 3C regulates cyclin A/p27 complexes and enhances cyclin A-dependent kinase activity. J. Virol. 78:1981–1991.

36. Knight, J. S., N. Sharma, and E. S. Robertson. 2005. Epstein-Barr virus latent antigen 3C can mediate the degradation of the retinoblastoma protein through an SCF cellular ubiquitin ligase. Proc. Natl. Acad. Sci. USA 102:18562–18566.

37. Knight, J. S., N. Sharma, and E. S. Robertson. 2005. SCF<sup>Skp2</sup> complex targeted by Epstein-Barr virus essential nuclear antigen. Mol. Cell. Biol. 25:1749–1763.

38. Kobet, E., X. Zeng, Y. Zhu, D. Keller, and H. Lu. 2000. MDM2 inhibits p300-mediated p53 acetylation and activation by forming a ternary complex with the two proteins. Proc. Natl. Acad. Sci. USA 97:12547–12552.

39. Kubbatat, M. H., S. N. Jones, and K. H. Vousden. 1997. Regulation of p53 stability by Mdm2. Nature 387:299–303.

40. Kussie, P. H., S. Gorina, V. Marechal, B. Elenbaas, J. Moreau, A. J. Levine, and N. P. Pavletich. 1996. Structure of the MDM2 oncoprotein bound to the p53 tumor suppressor transactivation domain. Science 274:948–953.

41. Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227:680–685.

42. Lane, D. P., and P. A. Hall. 1997. MDM2—arbiter of p53’s destruction. Trends Biochem. Sci. 22:372–374.

43. Léveillard, T., and B. Wasyluk. 1997. The MDM2 C-terminal region binds to TAFII250 and is required for MDM2 regulation of the cyclin A promoter. J. Biol. Chem. 272:30651–30661.

44. Li, M., C. L. Brooks, N. Kon, and W. Gu. 2004. A dynamic role of HAUSP in the p53-Mdm2 pathway. Mol. Cell 13:879–886.

45. Lin, J., E. Johannsen, E. Robertson, and E. Kieff. 2002. Epstein-Barr virus nuclear antigen 3C putative repression domain mediates coactivation of the LMP1 promoter with EBNA-2. J. Virol. 76:232–242.

46. Loughran, O., and N. B. La Thangue. 2000. Apoptotic and growth-promoting activity of E2F modulated by MDM2. Mol. Cell. Biol. 20:2186–2197.

47. Lundgren, K., R. Montes de Oca Luna, Y. B. McNeill, E. P. Emerick, B. Spencer, C. R. Barfield, G. Lozano, M. P. Rosenberg, and C. A. Finlay. 1997. Targeted expression of MDM2 uncouples S phase from mitosis and inhibits mammary gland development independent of p53. Genes Dev. 11:714–725.

48. Marechal, V., B. Elenbaas, J. Piette, J. C. Nicolas, and A. J. Levine. 1994. The ribosomal L5 protein is associated with mdm-2 and mdm-2–p53 complexes. Mol. Cell. Biol. 14:7414–7420.

49. Martin, K., D. Trouche, C. Hagemeier, T. S. Sorensen, N. B. La Thangue, and T. Kouzarides. 1995. Stimulation of E2F1/DP1 transcriptional activity by MDM2 oncoprotein. Nature 375:691–694.

50. Maya, R., M. Balass, S. T. Kim, D. Shkedy, J. F. Leal, O. Shifman, M. Moas, T. Buschmann, Z. Ronai, Y. Shiloh, M. B. Kastan, E. Katzir, and M. Oren. 2001. ATM-dependent phosphorylation of Mdm2 on serine 395: role in p53 activation by DNA damage. Genes Dev. 15:1067–1077.

51. Mayo, L. D., J. J. Turchi, and S. J. Berberich. 1997. Mdm-2 phosphorylation

by DNA-dependent protein kinase prevents interaction with p53. Cancer Res. 57:5013–5016.

52. Meulmeester, E., M. M. Maurice, C. Boutell, A. F. Teunisse, H. Ovaa, T. E. Abraham, R. W. Dirks, and A. G. Jochemsen. 2005. Loss of HAUSP-mediated deubiquitination contributes to DNA damage-induced destabilization of Hdmx and Hdm2. Mol. Cell 18:565–576.

53. Momand, J., D. Jung, S. Wilczynski, and J. Niland. 1998. The MDM2 gene amplification database. Nucleic Acids Res. 26:3453–3459.

54. Momand, J., H. H. Wu, and G. Dasgupta. 2000. MDM2—master regulator of the p53 tumor suppressor protein. Gene 242:15–29.

55. Notredame, C., D. G. Higgins, and J. Heringa. 2000. T-Coffee: a novel method for fast and accurate multiple sequence alignment. J. Mol. Biol. 302:205–217.

56. Peng, M., and E. Lundgren. 1992. Transient expression of the Epstein-Barr virus LMP1 gene in human primary B cells induces cellular activation and DNA synthesis. Oncogene 7:1775–1782.

57. Picksley, S. M., B. Vojtesek, A. Sparks, and D. P. Lane. 1994. Immunochemical analysis of the interaction of p53 with MDM2; fine mapping of the MDM2 binding site on p53 using synthetic peptides. Oncogene 9:2523–2529.

58. Pomerantz, J., N. Schreiber-Agus, N. J. Liégeois, A. Silverman, L. Alland, L. Chin, J. Potes, K. Chen, I. Orlow, H. W. Lee, C. Cordon-Cardo, and R. A. DePinho. 1998. The Ink4a tumor suppressor gene product, p19Arf, interacts with MDM2 and neutralizes MDM2’s inhibition of p53. Cell 92:713–723.

59. Pontén, J., and E. Saksela. 1967. Two established in vitro cell lines from human mesenchymal tumours. Int. J. Cancer 2:434–447.

60. Poyurovsky, M. V., C. Priest, A. Kentsis, K. L. Borden, Z. Q. Pan, N. Pavletich, and C. Prives. 2007. The Mdm2 RING domain C-terminus is required for supramolecular assembly and ubiquitin ligase activity. EMBO J. 26:90–101.

61. Rickinson, A. B., and E. Kieff. 2001. Epstein-Barr virus and its replication, p. 2511–2573. In D. M. Knipe, P. M. Howley, D. E. Griffin, R. A. Lamb, M. A. Martin, B. Roizman, and S. E. Straus (ed.), Fields virology, 4th ed., vol. 2. Lippincott Williams & Wilkins, Philadelphia, PA.

62. Robertson, E. S., S. Grossman, E. Johannsen, C. Miller, J. Lin, B. Tomkinson, and E. Kieff. 1995. Epstein-Barr virus nuclear protein 3C modulates transcription through interaction with the sequence-specific DNA-binding protein Jk. J. Virol. 69:3108–3116.

63. Rosendorff, A., D. Illanes, G. David, J. Lin, E. Kieff, and E. Johannsen. 2004. EBNA3C coactivation with EBNA2 requires a SUMO homology domain. J. Virol. 78:367–377.

64. Roth, J., M. Dobbelstein, D. A. Freedman, T. Shenk, and A. J. Levine. 1998. Nucleo-cytoplasmic shuttling of the hdm2 oncoprotein regulates the levels of the p53 protein via a pathway used by the human immunodeficiency virus rev protein. EMBO J. 17:554–564.

65. Ryan, K. M., A. C. Phillips, and K. H. Vousden. 2001. Regulation and function of the p53 tumor suppressor protein. Curr. Opin. Cell Biol. 13:332–337.

66. Saridakis, V., Y. Sheng, F. Sarkari, M. N. Holowaty, K. Shire, T. Nguyen, R. G. Zhang, J. Liao, W. Lee, A. M. Edwards, C. H. Arrowsmith, and L. Frappier. 2005. Structure of the p53 binding domain of HAUSP/USP7 bound to Epstein-Barr nuclear antigen 1 implications for EBV-mediated immortalization. Mol. Cell 18:25–36.

67. Sherr, C. J. 1998. Tumor surveillance via the ARF-p53 pathway. Genes Dev. 12:2984–2991.

68. Sherr, C. J., and J. D. Weber. 2000. The ARF/p53 pathway. Curr. Opin. Genet. Dev. 10:94–99.

69. Sinclair, A. J., I. Palmero, G. Peters, and P. J. Farrell. 1994. EBNA-2 and EBNA-LP cooperate to cause G0 to G1 transition during immortalization of resting human B lymphocytes by Epstein-Barr virus. EMBO J. 13:3321–3328.

70. Stommel, J. M., and G. M. Wahl. 2004. Accelerated MDM2 auto-degradation induced by DNA-damage kinases is required for p53 activation. EMBO J. 23:1547–1556.

71. Tang, J., L. K. Qu, J. Zhang, W. Wang, J. S. Michaelson, Y. Y. Degenhardt, W. S. El-Deiry, and X. Yang. 2006. Critical role for Daxx in regulating Mdm2. Nat. Cell Biol. 8:855–862.

72. Thorley-Lawson, D. A. 2001. Epstein-Barr virus: exploiting the immune system. Nat. Rev. Immunol. 1:75–82.

73. Thrower, J. S., L. Hoffman, M. Rechsteiner, and C. M. Pickart. 2000. Recognition of the polyubiquitin proteolytic signal. EMBO J. 19:94–102.

74. Toledo, F., and G. M. Wahl. 2006. Regulating the p53 pathway: in vivo veritas. Nat. Rev. Cancer 6:909–923.

75. Touitou, R., J. O’Nions, J. Heaney, and M. J. Allday. 2005. Epstein-Barr virus EBNA3 proteins bind to the C8/alpha7 subunit of the 20S proteasome and are degraded by 20S proteasomes in vitro, but are very stable in latently infected B cells. J. Gen. Virol. 86:1269–1277.

76. Trompouki, E., E. Hatziavassiliou, T. Tsichritzis, H. Farmer, A. Ashworth, and G. Mosialos. 2003. CYLD is a deubiquitinating enzyme that negatively regulates NF-kappaB activation by TNFR family members. Nature 424:793–796.

VOL. 83, 2009                                                                                             EBNA3C DEUBIQUITINATES Mdm2                      4669

77. Villuendas, R., F. Pezzella, K. Gatter, P. Algara, M. Sánchez-Beato, P. Martínez, J. C. Martínez, K. Muñoz, P. García, L. Sánchez, S. Kocialkowsky, E. Campo, J. L. Orradre, and M. A. Piris. 1997. p21WAF1/CIP1 and MDM2 expression in non-Hodgkin’s lymphoma and their relationship to p53 status: a p53+, MDM2-, p21-immunophenotype associated with missense p53 mutations. J. Pathol. 181:51–61.

78. West, M. J. 2006. Structure and function of the Epstein-Barr virus transcription factor, EBNA 3C. Curr. Protein Pept. Sci. 7:123–136.

79. Xiao, Z., J. Chen, A. J. Levine, N. Modjtahedi, J. Xing, W. R. Sellers, and D. M. Livingston. 1995. Interaction between the retinoblastoma protein and the oncoprotein MDM2. Nature 375:694–698.

80. Xirodimas, D. P., M. K. Saville, J. C. Bourdon, R. T. Hay, and D. P. Lane.

81. Zhang, Y., Y. Xiong, and W. G. Yarbrough. 1998. ARF promotes MDM2 degradation and stabilizes p53: ARF-INK4a locus deletion impairs both the Rb and p53 tumor suppression pathways. Cell 92:725–734.

82. Zhao, B., and C. E. Sample. 2000. Epstein-Barr virus nuclear antigen 3C activates the latent membrane protein 1 promoter in the presence of Epstein-Barr virus nuclear antigen 2 through sequences encompassing an Spi-1/Spi-B binding site. J. Virol. 74:5151–5160.

83. Zheleva, D. I., D. P. Lane, and P. M. Fischer. 2003. The p53-Mdm2 pathway: targets for the development of new anticancer therapeutics. Mini Rev. Med. Chem. 3:257–270.
