
REVIEW

# Principles of c-di-GMP signalling in bacteria

**Regine Hengge**

Abstract | On the stage of bacterial signal transduction and regulation, bis-(3′-5′)-cyclic dimeric guanosine monophosphate (c-di-GMP) has long played the part of Sleeping Beauty. c-di-GMP was first described in 1987, but only recently was it recognized that the enzymes that ‘make and break’ it are not only ubiquitous in the bacterial world, but are found in many species in huge numbers. As a key player in the decision between the motile planktonic and sedentary biofilm-associated bacterial ‘lifestyles’, c-di-GMP binds to an unprecedented range of effector components and controls diverse targets, including transcription, the activities of enzymes and larger cellular structures. This Review focuses on emerging principles of c-di-GMP signalling using selected systems in different bacteria as examples.

Bis-(3′-5′)-cyclic dimeric guanosine monophosphate (c-di-GMP) was discovered by Benziman and co-workers as a factor that allosterically activates the membrane-bound cellulose synthase of *Gluconacetobacter xylinus*¹². c-di-GMP is a soluble molecule that functions as a second messenger in bacteria. In general, c-di-GMP stimulates the biosynthesis of adhesins and exopolysaccharide matrix substances in biofilms and inhibits various forms of motility: it controls switching between the motile planktonic and sedentary biofilm-associated ‘lifestyles’ of bacteria (see REFS 3–8 for recent reviews) (FIG. 1). Moreover, c-di-GMP controls the virulence of animal and plant pathogens⁹⁻¹², progression through the cell cycle¹³, antibiotic production¹⁴ and other cellular functions. However, the mechanisms that underlie these molecular processes, and in particular the direct targets that are affected by c-di-GMP-binding effector components, are only beginning to emerge.

Interest in these c-di-GMP-controlling systems increased dramatically with the advent of whole genome sequencing, when it was recognized that GGDEF and EAL domains are not only ubiquitous in bacteria, but that many species encode a surprisingly large number of these proteins. Artificial manipulation of the cellular c-di-GMP content by the overproduction of GGDEF domain proteins strongly stimulated the synthesis of adhesins and biofilm matrix components and interfered with motility and acute virulence functions, whereas overproduction of EAL domain proteins produced the opposite phenotypes⁴,⁶⁻¹² (FIG. 1). Current studies are now assigning molecular functions to specific GGDEF, EAL and HD-GYP domain proteins and are starting to reveal how these systems are integrated into complex regulatory networks. Most recently, certain GGDEF and EAL domain proteins that lack DGC or PDE activity (in which key amino acids for enzymatic activity are not conserved) have been found to be biologically active; for example, in transcriptional regulation²⁴ and RNA degradation²⁵.

This Review will first define the basic c-di-GMP control module and its general signal-processing properties. The consequences, challenges and benefits of having multiple such modules in single bacterial species will then be discussed, which will lead to a concept of temporal, functional and/or spatial sequestration of some c-di-GMP control modules. Finally, enzymatically inactive ‘degenerate’ GGDEF and EAL domain proteins, which can function by direct macromolecular interactions, will be placed in this context. This Review does

Institut für Biologie — Mikrobiologie, Freie Universität Berlin, 14195 Berlin, Germany.  
e-mail: Rhenggea@zedat.fu-berlin.de  
doi:10.1038/nrmicro2109

NATURE REVIEWS | MICROBIOLOGY  
VOLUME 7 | APRIL 2009 | 263

© 2009 Macmillan Publishers Limited. All rights reserved.

REVIEW S

2 GTP
Diguanylate
cyclases
GGDEF

Phosphodiesterases
pGpG
EAL
HD-GYP
2 GMP

c-di-GMP

Motility
Expression and
activity of flagella,
swimming and
swarming

Virulence
Expression of
acute virulence
genes

Sessility and biofilm
formation
Synthesis of various
adhesins and
exopolysaccharides

Cell cycle progression
Proteolysis of a
replication inhibitor

Figure 1 | Structure and physiological functions of c-di-GMP. At the cellular level, bis-(3′-5′)-cyclic dimeric guanosine monophosphate (c-di-GMP) is controlled by diguanylate cyclases that carry GGDEF domains (red) and specific phosphodiesterases that carry EAL or HD-GYP domains (blue). c-di-GMP can reduce motility by downregulating flagellar expression (for example, in *Pseudomonas aeruginosa*) or assembly (for example, in *Caulobacter crescentus*) or interfering with flagellar motor function (for example, in *Escherichia coli* and *C. crescentus*; for a review, see REF. 8). Low c-di-GMP levels are required for the expression of acute virulence genes (for example, in *Vibrio cholerae*¹²). In all bacteria tested, high c-di-GMP levels stimulated various biofilm-associated functions, such as the formation of fimbriae and other adhesins and various matrix exopolysaccharides⁴,⁶. In *C. crescentus*, precisely timed and localized action of c-di-GMP is a key step in cell cycle progression¹³. pGpG, 5′-phosphoguanylyl-(3′-5′)-guanosine.

and PDEs, effectors and targets that, through a common pool of c-di-GMP, affects a common functional output. Different c-di-GMP control modules can integrate different environmental or cellular input signals (which affect expression or the activities of the DGCs and PDEs) and use different types of effectors and targets to control a plethora of cellular functions.

‘Making and breaking’ c-di-GMP. The active DGC is a dimer of two subunits with GGDEF domains. The active site (A site) is located at the interface between the two subunits, which each bind one molecule of GTP¹⁷,²⁹–³¹. The A site corresponds to the GGDEF motif, and any point mutation in this motif (except a D to E mutation) eliminates enzymatic activity¹⁶. The catalytic mechanism is similar to that of structurally related adenylate cyclases and polymerases²⁹,³¹. In addition, most DGCs are subject to allosteric product inhibition, which involves c-di-GMP binding to a secondary site (I site) characterized by an RXXD motif, which is separated from the A site by a linker composed of five amino acids. This feedback control avoids excessive GTP consumption, helps to set an upper limit for c-di-GMP accumulation and might buffer against stochastic variations in cellular c-di-GMP content³². Recent studies with WspR, a DGC from *Pseudomonas aeruginosa*, indicate that inactivation of the enzyme follows a cycle that includes transiently existing tetramers (which can be free of c-di-GMP or can possess c-di-GMP bound to the I sites) and an elongated dimer (which contains c-di-GMP at the I sites). This dimer can be reactivated by the action of PDEs³³.

An active EAL domain PDE is a monomeric enzyme that linearizes c-di-GMP to 5′-pGpG, which is then further degraded by nonspecific cellular PDEs (FIG. 1). The catalytic process requires either Mg²⁺ or Mn²⁺ and is inhibited by Ca²⁺ and Zn²⁺ (REFS 20,21,23,34). The second type of c-di-GMP-specific PDE is the HD-GYP domain proteins, which form a subfamily of the HD superfamily of metal-dependent phosphohydrolases and are unrelated to the EAL proteins³⁵. These enzymes break the phosphodiester bond in c-di-GMP to produce 5′-pGpG, and can further degrade 5′-pGpG to GMP²².

Although all three domains can occur separately, composite proteins in which a GGDEF domain is covalently linked to either EAL or HD-GYP domains also exist. Phenotypic studies *in vivo* and *in vitro* have shown that these fusion proteins usually have either DGC or PDE activity²,²⁰,²¹,³⁶–³⁸. A particularly interesting case is PdeA from *Caulobacter crescentus*, in which a GGDEF domain with a slightly degenerate A site motif (GEDEF) allosterically activates the PDE activity of the EAL domain after binding GTP, which it cannot convert to c-di-GMP²⁰. Bifunctional enzymes that can conditionally switch between the two activities also exist. For example, the full-size GGDEF-EAL protein BphG1 from *Rhodobacter sphaeroides* functions as a PDE, but experimental removal of the EAL domain resulted in DGC activity regulated by the amino-terminal bacteriophytochrome domain³⁹. Whether DGC or PDE activity prevails in the GGDEF-EAL protein ScrC, which inversely

not present a complete overview of all c-di-GMP signalling systems described to date, but rather uses a selection of these systems to illustrate the general principles of c-di-GMP signalling.

### The c-di-GMP control module

A second-messenger control module generally consists of four components: the two enzymes that in response to certain signals produce and degrade the second messenger, an effector molecule that binds and is allosterically regulated by the second messenger and a target component that in response to direct contact with the effector produces a molecular output. These components are genetically defined by a common phenotype of the corresponding mutations and clear epistatic relationships. The classic example in bacteria is cyclic AMP (cAMP) signalling in *Escherichia coli*. cAMP is synthesized by a single adenylate cyclase that is controlled by complex signal input and degraded by a specific PDE. cAMP allosterically controls a single effector, the transcription factor CRP, and therefore acts on multiple targets, all of which are similar cAMP-CRP-binding sites in transcriptional control regions²⁶–²⁸.

In the case of c-di-GMP, the basic principle is the same (FIG. 2), but owing to the multiplicity of all four principal components in a single bacterial species, the regulatory outcome is more complex and diverse. This multiplicity of components allows us to define functional c-di-GMP control modules. A module is the set of DGCs

coordinates swarming and adhesion in *Vibrio parahaemolyticus*, seems to depend on modulation by two accessory proteins, ScrA and ScrB⁴⁰. The GGDEF–EAL protein MSDGC1 of *Mycobacterium smegmatis*, which is essential for long-term starvation survival, exhibits both enzymatic activities simultaneously *in vitro*, whereas the isolated domains are inactive⁴¹.

**Signal inputs into the c-di-GMP control module.** Cellular c-di-GMP concentrations depend on the protein levels (discussed below) as well as the specific activities of DGCs and PDEs. Similarly to histidine sensor kinases of two-component systems, most GGDEF, EAL and HD-GYP domains are linked to various N-terminal sensory input domains, suggesting that numerous environmental and cellular signals are integrated into the c-di-GMP signalling network⁴²,⁴³. Many of these proteins contain one or more transmembrane helices, which in Gram-negative bacteria can place the sensory sites in the periplasm. Signals perceived include oxygen and redox conditions, light, starvation and various extracellular substances, such as antibiotics, polyamines or intercellular signalling molecules⁴,⁷. These signals are perceived, for example, by haem- or flavin-associated PAS domains¹⁹,⁴⁴⁻⁴⁶, the oxygen-binding haemerythrin domain⁴⁷, the blue-light-sensing FAD-associated BLUF domain⁴⁸,⁴⁹, red and far red-sensing bacteriophytochromes³⁹, GAF domains that have been implicated in small ligand binding and protein–protein interactions⁵⁰, and the CHASE, MASE1 and MASE2 domains⁴². Some GGDEF and EAL domain proteins contain several sensory domains, suggesting that complex signal integration occurs even in single proteins.

Moreover, c-di-GMP synthesis or degradation can be the output of two-component signalling pathways when GGDEF, EAL or HD-GYP domains are linked to two-component receiver (REC) domains¹⁵,¹⁷,²². In the *C. crescentus* REC–REC–GGDEF protein PleD, phosphorylation of one of the two N-terminal REC domains induces stable dimerization, which is essential for the formation of the enzymatically active site¹⁷,³¹. Phosphorylation of the N-terminal REC domain of the *Vibrio cholerae* EAL domain protein and transcription factor VieA by the sensor kinase VieS directly stimulates increased VieA-mediated transcription of the *vieSAB* operon (autoregulation) and thereby indirectly enhances the contribution of the EAL domain of VieA to total cellular PDE activity⁵¹. In addition, several reports indicate that REC domains can be crucial for the localization of GGDEF and EAL domain proteins to the cell poles¹³,³⁰,⁵²,⁵³.

**Signal transfer by c-di-GMP-binding effectors.** To exert its function, c-di-GMP has to bind to and allosterically alter the structure and output function of an effector component. c-di-GMP-sensing components are highly diverse, and therefore cannot be recognized by a single common domain or c-di-GMP-binding site (for a list of currently known families of effector components, including specific examples, see TABLE 1).

Four types of c-di-GMP effector proteins are currently known. The PilZ family of proteins (named

---

Input signal one
![Diagram](#)
Input signal two

Figure 2 | **The basic c-di-GMP signalling module.** A basic bis-(3′-5′)-cyclic dimeric guanosine monophosphate (c-di-GMP) signalling module consists of at least one diguanylate cyclase (DGC), which is characterized by its GGDEF domain, and one phosphodiesterase (PDE) of the EAL or HD-GYP domain type. DGCs and PDEs respond to different input signals through their amino-terminal sensory domains and balance the c-di-GMP level by their antagonistic activities. Binding of c-di-GMP to a secondary site (I site) that is present in most DGCs results in feedback product inhibition of these enzymes and contributes to a physiological upper limit to cellular accumulation of c-di-GMP. Specific effectors, which can be proteins or RNAs (riboswitches), can bind c-di-GMP and subsequently affect the regulatory output of a direct target molecule or structure. Targets can be promoter DNA (if the effector is a transcription factor), enzymes or complex cellular structures, such as the flagellar basal body or an exopolysaccharide synthesis and secretion apparatus. See the main text for details, including the consequences of the various \( K_d \), \( K_i \) and \( K_m \) constants. DGCs are shown in red, PDEs are shown in blue, effectors are shown in green and targets are shown in yellow.

after a type IV pilus control protein in *P. aeruginosa*) represents the best-studied class (members of this family are listed in TABLE 1). In some cases, the PilZ domain is directly attached to the carboxyl terminus of the GGDEF, EAL and/or HD-GYP domains⁵⁴, or is linked to a domain that generates a molecular output; for example, the production of cellulose⁵⁴ or alginate⁵⁵,⁵⁶. Structural analysis revealed that, as expected for an allosterically controlled protein, conformational changes occur during binding of a PilZ-like protein to c-di-GMP⁵⁷. The PilZ-like proteins studied so far seem to be activated by c-di-GMP and to function by protein–protein interactions (TABLE 1). Two other types of effector proteins that are not related to PilZ are represented by the c-di-GMP-binding transcription factor FleQ of *P. aeruginosa*, which as a repressor is inactivated by c-di-GMP⁵⁸, and PelD of *P. aeruginosa*, which is activated by binding c-di-GMP through a site that resembles the I site motif in GGDEF domain proteins⁵⁹.

REVIEW S

Table 1 | c-di-GMP-binding effector components

| Effector family* | Example | Species | Functions controlled† | Refs |
|------------------|---------|---------|-----------------------|------|
| Protein effectors |         |         |                       |      |
| PilZ (+)         | Alg44   | Pseudomonas aeruginosa | Alginate synthesis (+) | 55    |
| PilZ (+)         | BcsA    | Various Gram-negative bacteria | Cellulose synthesis (+) | 54,66 |
| PilZ (+)         | DgrA    | Caulobacter crescentus | Flagellar activity (−) | 67    |
| PilZ (+)         | PilZ    | P. aeruginosa | Twitching motility (−) | 115   |
| PilZ (+)         | Plz proteins | Vibrio cholerae | Virulence gene expression | 68    |
| PilZ (+)         | YcgR    | Escherichia coli and Salmonella spp. | Flagella activity (−) | 69    |
| FleQ (−)         | FleQ    | P. aeruginosa | Flagella expression (+) and Pel (part of the EPS) synthesis (−) | 58    |
| PelD (+)         | PelE    | P. aeruginosa | Pel (part of the EPS) synthesis (+) | 59    |
| I site effectors (+) | PopA | C. crescentus | Cell cycle progression (+) | 13    |
| RNA effectors    |         |         |                       |      |
| GEMM (+ and −)   | Vc1 (encoded by gbpA) | V. cholerae | Intestinal adhesion | 61    |
| GEMM (+ and −)   | Vc2 (encoded by VC1722) | V. cholerae | Biofilm formation and rugosity | 61    |
| GEMM (+ and −)   | Cd1     | Clostridium difficile | Flagella synthesis | 61    |

*A + indicates that the effector is activated by c-di-GMP and a – indicates that the activity of the effector is reduced by c-di-GMP. † A + indicates that the function is positively controlled by the effector and a – indicates that the function is negatively controlled by the effector. See the main text for information on the direct molecular targets contacted by the effector. c-di-GMP, bis-(3′-5′)-cyclic dimeric guanosine monophosphate; EPS, exopolysaccharide; GEMM, RNA element occurring in genes for the environment, membranes and motility.

The fourth type of effector is regulated through the I site. It was proposed that the c-di-GMP-binding I site in many GGDEF domains can also be an effector site and that dedicated effector proteins might have evolved from GGDEF domains in which the A site has degenerated^{4,32}. The first example was the C. crescentus PopA protein (previously named PdpA): when c-di-GMP binds to its I site, PopA sequesters the replication inhibitor and global cell cycle regulator CtrA to the differentiating cell pole, thereby targeting CtrA for degradation^{13}. CdgG from V. cholerae requires an intact I site motif, but not its degenerate active site, to control rugosity, biofilm formation and motility, suggesting that CdgG is another I site-dependent c-di-GMP-binding effector^{60}. Similar GGDEF domain proteins with an intact I site but a degenerate A site also exist in other bacteria, but their functions have yet to be analysed.

As predicted^{4}, c-di-GMP is also a ligand for a riboswitch^{61}: the effector is a conserved RNA domain (GEMM; RNA element occurring in genes for the environment, membranes and motility) that is present in the 5′-untranslated regions of different mRNAs. Although several organisms make extensive use of this specific riboswitch (for example, the Deltaproteobacterium Geobacter uraniumreducens), it is absent in most species, but it seems likely that additional c-di-GMP-dependent riboswitches will be discovered in the future.

The effective cellular c-di-GMP concentrations, together with the affinities for c-di-GMP of effector components, are crucial for triggering c-di-GMP-dependent outputs. As physiological cellular c-di-GMP levels are low (especially under motility-promoting conditions), precise experimental measurements of these levels in wild-type strains are not trivial^{62–65}. Moreover, total cellular c-di-GMP concentrations determined experimentally are not necessarily the same as the effective cellular concentrations, as a significant fraction of the c-di-GMP molecules seems to be bound^{66}, which may also result in a high local concentration that is different from the concentration of freely diffusible c-di-GMP (discussed below).

However, available information about the affinities of various c-di-GMP-binding sites allows an estimation of biologically effective cellular concentrations of c-di-GMP (FIG. 2). Most DGCs are subject to product inhibition through their I sites. K_{i} (dissociation constant of the inhibitor) varies between 0.5 μM and 1 μM^{31,32}, indicating that the physiological upper limit for freely diffusible c-di-GMP is in the low μM range. Consistent with these observations, the K_{m} (rate constant) for PDEs was between 60 nM and several μM^{20,21,23}. The K_{d} (dissociation constant) for various PilZ effector proteins was in the range of <50 nM–1 μM^{57,66–69}, and the I site effector protein PopA exhibited a K_{d} of approximately 2 μM^{13}. Taken together, these data suggest that switches regulated by c-di-GMP respond to concentrations that range from <50 nM to a few μM (given the dimensions of a typical post-exponentially growing Gram-negative cell, such as an E. coli cell, a cellular concentration of 1 nM corresponds to approximately 1 molecule per cell).

Strikingly, the recently described c-di-GMP riboswitch has a very high affinity (K_{d} of 1 nM)^{61}. The physiological relevance of this has not been determined, but an RNA aptamer with such an affinity could buffer stochastic variations or completely scavenge residual free c-di-GMP molecules under physiological conditions, in

Box 1 | c-di-GMP control modules in Escherichia coli

![Diagram](attachment:diagram.png)

During the transition from the post-exponential phase to the stationary phase in complex media, *E. coli* cells downregulate the expression of flagella and motility [70,108,109] (see the figure) and, when grown below 30 °C, induce adhesive curli fimbriae [110]. This transition is controlled by mutual inhibition of the FlhDC (motility) and σ<sup>S</sup> (adhesion) control cascades [70]. At the top levels of the cascades, mutual exclusion operates by the competition of sigma subunits of RNA polymerase (σ<sup>70</sup>, σ<sup>FliA</sup> and σ<sup>S</sup>) as well as by FliZ, which acts as an inhibitor of σ<sup>S</sup> function [70].

At a lower hierarchical level, bis-(3′-5′)-cyclic dimeric guanosine monophosphate (c-di-GMP) is crucial for this motile-to-sedentary ‘lifestyle’ switch. c-di-GMP is antagonistically controlled by the σ<sup>S</sup>-activated GGDEF proteins (YegE and YedQ), and the EAL-type phosphodiesterase (PDE) YhjH, which is indirectly dependent on the flagellar master regulator FlhDC. When flagellar expression stops during entry into stationary phase and ClpXP degrades the remaining FlhDC, YhjH-mediated PDE activity is overcome by YegE- and YedQ-mediated diguanylate cyclase (DGC) activity [70]. Consequently, flagellar activity is reduced through the c-di-GMP-binding effector protein YcgR [69,70,80]. Moreover, the transcription of *csgD*, which encodes an activator of the curli structural operon *csgBAC*, is under positive control of YegE [70], and therefore the YegE–YhjH module also represents a checkpoint that links cessation of flagellar expression to induction of curli expression. In addition, *csgD* transcription also requires the σ<sup>S</sup>-dependent DGC YdaM, and is negatively controlled by its antagonistic PDE YciR [38,70]. The YdaM–YciR system does not contribute to motility control [70], and therefore this system seems to act locally (see the main text). Finally, the DGC YaiC (known as AdrA in *Salmonella* spp.), which is expressed under CsgD control later during entry into stationary phase [38,63,84,111], is required for production of cellulose [112] and seems to be counteracted by the EAL domain protein YoaD [11], which is also under σ<sup>S</sup> control [84]. BBH, basal body hook. Figure is modified, with permission, from REF. 70 © (2008) Cold Spring Harbor Laboratory Press.

molecules could then stimulate a downstream target process. In *V. cholerae*, two proteins that are crucial for both host colonization and pathogenesis, the PDE VieA (which has a *K*<sub>m</sub> of 60 nM)<sup>23,65</sup> and GbpA (which contains the GEMM–Vc1 c-di-GMP riboswitch in its mRNA)<sup>61</sup>, might be an example of such a cooperation.

Targets of c-di-GMP signalling and cellular processes affected. Given the diversity of c-di-GMP-binding effectors, it is not surprising that target components or processes controlled by these effectors are equally diverse.

First, c-di-GMP can affect the transcription of target genes. The *P. aeruginosa* transcription factor FleQ, which activates the expression of flagellar genes and represses the biofilm-promoting *pel* operon, directly binds c-di-GMP, which antagonizes the activity of c-di-GMP as a repressor<sup>58</sup>. In *E. coli*, two separate DGC-PDE systems (YdaM–YciR and YegE–YhjH) converge to control the transcription of the *csgDEFG* operon, which encodes the transcriptional regulator CsgD and several factors involved in the expression and assembly of adhesive curli fimbriae<sup>38,70</sup> (BOX 1). In *Salmonella* spp., YciR and YegE have similar roles, but the other two components have been replaced by functionally similar enzymes<sup>36,63,64,71</sup>. The c-di-GMP-binding effector has not yet been identified unequivocally, but the MerR-like transcription factor MlrA<sup>72</sup> might be involved. Unlike other regulators that affect transcription of the *csgDEFG* operon (for example, outer membrane protein R (OmpR), integration host factor (IHF), CpxR and RcsB)<sup>73–76</sup>, MlrA acts highly specifically on the transcription of the *csgDEFG* operon and shows the same narrow spectrum of target genes as YdaM and YciR in microarray analyses<sup>38</sup> (H. Weber and R.H., unpublished observations). Owing to its role in CsgD expression, the DGC YdaM also indirectly controls YaiC, another DGC that is expressed in a CsgD-dependent manner at a late stage during entry into stationary phase<sup>38</sup>. A transcriptional cascade of DGCs is also present in *V. cholerae*, in which the GGDEF–EAL protein CdgC controls the expression of a set of transcriptional regulators, including the quorum-sensing regulator HapR, which in turn affects the expression of several GGDEF–EAL and HD-GYP domain-encoding genes<sup>77–79</sup>. Furthermore, transcription can also be the regulatory output of a c-di-GMP-binding riboswitch in the 5′-untranslated region of an mRNA if it affects a transcription anti-termination mechanism. Alternatively, riboswitches can control translational initiation<sup>61</sup>.

Some c-di-GMP-binding effectors directly affect enzymatic function. The classic example is the PilZ-like effector in cellulose biosynthesis, which is a regulatory domain of the cellulose synthase, BcsA<sup>1,54,66</sup>. The membrane-spanning protein Alg44 in *P. aeruginosa*, which features a cytoplasmic N-terminal PilZ domain linked to a C-terminal periplasmic domain that resembles membrane fusion proteins of multidrug efflux systems, cooperates with another integral membrane protein, the glycosyltransferase Alg8, and thereby affects synthesis and secretion of the exopolysaccharide alginate<sup>55,56</sup>.

REVIEW S

Box 2 | Localized GGDEF and EAL proteins in Caulobacter crescentus

| PopA swarmer pole |
| --- |
| PopA stalked pole |
| TipF |
| PleD |

![Diagram](attachment:diagram.png)

- Holdfast formation
- Flagellum ejection
- Stalk formation
- S phase entry
- CtrA degradation

PleD  
REC—REC—GGDEF  

PopA  
REC—REC—GEDEF  

Cell division in *C. crescentus* is asymmetric (see the figure), producing a flagellated motile swarmer cell and a sessile stalked cell that is attached to a surface via the holdfast. DNA replication is blocked in the swarmer cell by the global regulator CtrA. For proliferation, the swarmer cell must therefore differentiate into a stalked cell, which involves surface attachment through pili, holdfast formation and flagellum ejection. CtrA is then temporarily degraded, which occurs at the stalk pole and allows transition from the G1 to the S phase. During the cell cycle, several GGDEF and EAL domain proteins are specifically localized to different cell poles. The phosphodiesterase (PDE) TipF is localized at the ‘newborn’ and future swarmer pole, which later generates the low-bis-(3′-5′)-cyclic dimeric guanosine monophosphate (c-di-GMP) conditions required for flagellum assembly¹⁰⁴. At the differentiating (swarmer-to-stalked) pole, processes positively controlled by c-di-GMP take place: the diguanylate cyclase (DGC) PleD is recruited to this pole and is required for holdfast formation and flagellum ejection¹⁷,¹¹³ and the GGDEF and I site-type c-di-GMP effector protein PopA later sequesters CtrA to this pole, where CtrA is degraded by ClpXP, allowing initiation of DNA replication¹³. To retain PopA and sequester CtrA to the stalked pole, c-di-GMP binding to PopA is essential. However, the initial polar localization of PopA actually occurs earlier and to the swarmer pole before it differentiates into a stalked pole owing to the action of the polar localization factor PodJ¹³,¹¹⁴. In summary, the cell cycle and the spatial localization and action of GGDEF and EAL domain proteins are tightly intertwined: the DGC PleD and the PDE TipF are directed to, and are active at, different cell poles in a cell cycle-dependent manner and progression of the cell cycle is dependent on the GGDEF I site-type c-di-GMP effector protein PopA. Proteins with a GGDEF domain are shown in red, proteins with an EAL motif are shown in blue, effector proteins are shown in green and target proteins and their genes are shown in yellow. REC, receiver domain.

stationary phase⁷⁰,⁸⁰, when cells change from the motile to the adhesive lifestyle (BOX 1). In *C. crescentus*, DgrA-mediated downregulation of flagellar activity is probably an early step in the attachment of a swarmer cell to a surface, and is followed by flagellar-to-stalked cell pole differentiation (BOX 2).

Finally, a proteolytic targeting process that is crucial for the G1-to-S phase transition during the cell cycle of *C. crescentus*, a period when c-di-GMP levels peak⁸¹, was shown to be a direct target of the c-di-GMP I site-type effector protein PopA. c-di-GMP-bound PopA, which is located at the stalk pole (BOX 2), sequesters the DNA replication inhibitor and global cell cycle regulator CtrA to this pole through the mediator protein RcdA. CtrA is then degraded by the ClpXP protease, which allows progression into the S phase of the cell cycle¹³.

These examples show that the molecular mechanisms of direct c-di-GMP action are diverse. As there are several different types of c-di-GMP-binding effectors, it seems that virtually any kind of molecular mechanism in a cell can be controlled by c-di-GMP, which confers an unprecedented flexibility to regulation by this second-messenger molecule.

Multiplicity of GGDEF and EAL domain proteins

The large number of GGDEF and EAL domain proteins in single species has been puzzling. In general, Gram-negative bacteria have more of these proteins than Gram-positive bacteria, and Gram-negative bacteria, especially the Gammaproteobacteria, display a striking proliferation of GGDEF and EAL domain proteins. For example, *E. coli* has 29 GGDEF or EAL domain proteins, whereas *Salmonella enterica* subsp. *enterica* serovar Typhimurium has 19, *C. crescentus* has 14 and various *Vibrio* species encode more than 50 GGDEF or EAL domain proteins. By contrast, HD-GYP domain proteins are less common and even absent in some species (although in some species, such as *Thermotoga maritima*, they can account for all PDE activity in the cell)⁴²,⁸². The presence of these large sets of proteins, most of which function as DGCs and PDEs, in a single species generates a new flexibility of signalling that is not found in a simple second-messenger system that consists of a minimal set of components (for example, the cAMP system found in enteric bacteria), but also raises the question of signalling specificity.

The total cellular DGC and PDE activities in a cell depend on the actual cellular levels and specific activities of the GGDEF, EAL and HD-GYP domain proteins. This results in certain rates of c-di-GMP production and degradation that are equal at an equilibrium characterized by a certain c-di-GMP level. However, this equilibrium can be dynamic, especially when the actual cellular c-di-GMP level is higher than the \(K_m\) of the relevant PDEs and lower than the \(K_i\) of the relevant DGC: that is, in a range in which these antagonistic enzymes operate close to saturation (the cellular concentration of the DGC substrate GTP is always in the millimolar range⁸³). This system is therefore highly sensitive to changes in the levels and activities of the enzymes involved, such as environmentally controlled changes in the expression, proteolysis

Furthermore, the PelD effector protein in *P. aeruginosa* is attached to the cytoplasmic membrane and probably directly affects the production and secretion of the Pel exopolysaccharide⁵⁹.

Complex cellular structures, such as the flagellar basal body, are also regulated by c-di-GMP. The motility-controlling PilZ proteins in *E. coli* and *C. crescentus* (YcgR and DgrA, respectively) do not affect the expression of flagellar genes, but directly interfere with the motor function of fully assembled flagella, suggesting that these proteins directly interact with a cytoplasm-facing component of the flagellar basal body⁶⁷,⁷⁰,⁸⁰. In *E. coli*, this results in an altered flagellar rotational switching pattern and a reduction in swimming speed during entry into
and activities of these enzymes. Consequently, increased DGC or PDE activity drives c-di-GMP levels up or down, respectively, until a new steady state concentration, which depends on the $K_{i}$ and $K_{m}$ of the relevant DGCs and PDEs, respectively, is reached. This allows fine-tuned and hypersensitive signal input and integration, but at the same time raises a problem of robustness. How is such a hypersensitive system buffered against stochastic noise in expression and activity, which increases with an increase in the numbers of the different DGCs and PDEs involved?

In addition, if all DGCs and PDEs in a cell contribute to a common c-di-GMP pool, there would be a single common output to all of these systems (only differences in affinities for c-di-GMP between the effectors would provide some specificity of output). However, the versatility of c-di-GMP signalling would be greatly enhanced if the system could be modified to include non-converging pathways that operate in parallel.

### Sequestration of c-di-GMP control modules

A solution to the signalling specificity and noise problems that would also overcome the limitations of a convergent single-output system would be the sequestration of GGDEF and EAL domain proteins. This would mean that not all of these proteins are present and active at the same time and place. Sequestration could be temporal, meaning that cellular levels and activities of DGCs and PDEs would change in response to environmental or cellular alterations over time, or functional, meaning that entire c-di-GMP control modules would be present at the same time but would operate in physically separated entities, generating ‘local pools’ of c-di-GMP with separate regulatory outputs. This would also allow spatial sequestration, as such modules could be precisely localized to specific positions in the cell, where they would exert their specific output functions.

### Temporal sequestration: tight regulation of expression and proteolysis of c-di-GMP control modules

The few studies on the regulation of expression of GGDEF and EAL domain proteins, c-di-GMP-binding effectors and target components suggest that the components that constitute distinct c-di-GMP control modules are tightly controlled and co-regulated.

In *E. coli*, only a few GGDEF and EAL domain-encoding genes are expressed at high levels in growing cells relative to the other GGDEF and EAL domain-encoding genes${}^{84}$. These include the PDE-encoding *yhjH* gene, which is required for motility. YhjH and the c-di-GMP-binding effector YcgR, which can interfere with motility, are under indirect control of the flagellar master regulator FlhDC, and are therefore co-regulated with flagella${}^{70,80,85}$. Many GGDEF and EAL domain-encoding genes in *E. coli*, including those required for curli and cellulose synthesis, are under the control of $\sigma^{S}$, which indicates that they belong to the general stress response. Many of these genes are induced during entry into stationary phase. This regulation is crucial for the reduction of flagellar activity and induction of adhesive curli fimbriae during the transition from the post-exponential phase to the stationary phase${}^{38,70,84}$ (BOX 1). Furthermore, the direct target of c-di-GMP control in curli synthesis — transcription of the regulatory *csgD* gene — is under $\sigma^{S}$ control at several levels${}^{72,86}$. Two GGDEF domain proteins and putative DGCs in *E. coli*, YcdT and YdeH, are downregulated post-transcriptionally by the mRNA-binding protein CsrA${}^{87}$, a global regulator that inhibits synthesis of the poly-$N$-acetyl-glucosamine (PGA) exopolysaccharide and stimulates the expression of the flagellar master regulator FlhDC${}^{88,89}$. This regulation of FlhDC should also allow CsrA to upregulate the PDE YhjH.

In *V. cholerae*, biofilm formation is negatively regulated by cAMP–CRP, which controls the expression of several GGDEF and EAL domain-encoding genes${}^{90}$. The quorum-sensing regulator HapR represses the biofilm regulatory gene *vpsT* both directly and indirectly by controlling several GGDEF and EAL domain-encoding genes, which suggests that biofilm formation is also reduced by high cell density${}^{79,91}$. As quorum sensing and low c-di-GMP levels (generated by the PDE VieA) are crucial for virulence gene expression inside the host${}^{92}$, this underlines the importance of c-di-GMP for biofilm formation in the extra-host environment. Consistent with this, several GGDEF domain-encoding genes are induced during the mucosal escape response, a late step of infection${}^{93,94}$.

c-di-GMP synthesis can also be controlled by proteolysis. In *Yersinia pestis*, the GGDEF domain protein HmsT is degraded by the ClpXP and Lon proteases when cells are shifted from low temperatures (below 34 °C) to 37 °C${}^{95}$. Together with HmsH and HmsR (outer and inner membrane proteins, respectively, that are degraded in parallel), HmsT is required for biofilm-associated phenotypes, such as hemin and Congo red binding. These phenotypes can only be observed at low temperatures (such as those found in the flea, which serves as the environmental host in the transmission of plague) and are counteracted by the PDE–EAL domain protein HmsP${}^{95,96}$.

In summary, bacterial cells limit the kind and numbers of GGDEF and EAL domain proteins that are expressed under particular conditions. On the one hand, this reduces the negative effects of stochastic noise on c-di-GMP signalling, and therefore on the expression and activity of the different DGCs and PDEs, without sacrificing signal integration. On the other hand, changing sets of DGCs and PDEs also means changing sets of signalling input domains that can feed into c-di-GMP control: regulation of ‘regulatability’ by different input signals.

### Functional and spatial sequestration of c-di-GMP signalling

Several authors have proposed that some of the many c-di-GMP control systems in single species can operate in independent parallel pathways, such that they exhibit functional sequestration${}^{4,7,38,63,97}$. It should also be noted that certain mycobacterial and alphaproteobacterial species possess high numbers of adenylate cyclases${}^{82,98-100}$, suggesting they have similarly complex cAMP signalling. Moreover, in eukaryotic cells, adenylate cyclases have been shown to function in cAMP microdomains${}^{101}$.
REVIEWS

Functional sequestration implies that c-di-GMP signalling occurs in ‘microcompartments’: multiprotein complexes that comprise a specific DGC and/or PDE that are controlled by distinct input signals, as well as specific effector and target components, which should associate by specific protein–protein or protein–DNA interactions. There is increasing evidence that such interactions exist. For example, the HD-GYP-type PDE domain of RpfG in *Xanthomonas axonopodis* pv citri was found to interact directly with several GGDEF domain proteins<sup>102</sup>. Specific interactions were detected between the DGC HmsT, the PDE HmsP and the putative glycosyltransferase HmsR and its accessory factor HmsS of *Y. pestis*, all of which are attached to the inner membrane and are required for biofilm-associated synthesis and excretion of exopoly-saccharide matrix substances<sup>103</sup>. Small deletions in the C-terminal periplasmic domain of Alg44 were found to block Alg8-mediated alginate polymerization, which suggested that direct communication occurs between the c-di-GMP-binding PilZ-like effector domain of Alg44 and the alginate-producing glycosyltransferase Alg8 (both of which are located at the cytoplasmic face of the inner membrane)<sup>56</sup>. In other species, c-di-GMP-binding PilZ domains occur in covalent linkage with GGDEF, EAL and HD-GYP domains, or with putative target domains, suggesting that local confinement of partial or entire c-di-GMP control modules is not uncommon<sup>54</sup>. Because all relevant components are closely associated, the local c-di-GMP concentration may become high and therefore biologically effective without substantially contributing to the cellular c-di-GMP pool; alternatively, c-di-GMP made within such a complex may be sterically prevented from diffusing away.

Functional sequestration in locally acting units would allow the emergence of separate parallel-operating c-di-GMP control modules with different molecular regulatory outputs in a single cell. A criterion for identifying functional sequestration would therefore be that two DGCs or two PDEs that are present and active under the same conditions differentially affect distinct target processes. An example is the inverse coordination of motility and curli expression in *E. coli* during the transition from the post-exponential phase to the stationary phase (BOX 2). The DGC–PDE module YegE–YhjH controls motility and has a modulating effect on the transcription of the curli activator gene *csgD*, whereas the YdaM–YciR module strongly and exclusively affects *csgD* expression<sup>38,70</sup>. Some DGC–PDE modules may also control the freely diffusible cytoplasmic c-di-GMP (and thereby affect several output functions), whereas others may act locally and highly specifically through a single effector on a single target. The GGDEF domain proteins YedQ and YeaJ also contribute to the YegE–YhiH-regulated c-di-GMP pool (and therefore have a minor or conditional effect on motility), and this c-di-GMP serves at least two effectors, as the PilZ protein YcgR mediates the effect on motility, but is not required for the effect on *csgD* transcription<sup>70</sup>. This system therefore seems to control c-di-GMP that is freely diffusible in the cytoplasm. By contrast, the YdaM–YciR system exerts a strong and highly specific effect on *csgD* expression only, which is suggestive of a locally confined function<sup>38,70</sup>.

The observation that experimentally measurable manipulation of the cellular c-di-GMP concentration does not affect a particular target process that is affected by mutations in genes encoding distinct active DGCs or PDEs further indicates that the control and action of c-di-GMP can be localized. Such evidence has been presented for the regulation of the curli regulator CsgD in *Salmonella* spp. (which is similar, although not identical, to that of CsgD from *E. coli*)<sup>63</sup>, again suggesting that c-di-GMP-induced transcription of *csgD* is an example of local action of a c-di-GMP control module.

Importantly, functional sequestration of c-di-GMP signalling may also be confined to distinct spatial locations in the cell. This can be the cytoplasmic membrane, as exemplified by the Hms, Alg44 and Alg8 systems mentioned above<sup>56,103</sup>. In *P. aeruginosa*, the DGC WspR clusters in cellular foci when it is activated by phosphorylation of its REC domain, which seems to occur in response to a stimulus associated with surface growth<sup>52</sup>. However, the paradigm for precisely controlled and functionally essential spatial sequestration are several GGDEF and EAL domain proteins that have key roles during the cell cycle of *C. crescentus*. The cell cycle of *C. crescentus* is obligatorily coupled to differentiation from a flagellated and motile swarmer cell to a sedentary non-flagellated stalked cell, as only these stalked cells can replicate DNA and undergo cell division. During the cell cycle, the DGC PleD<sup>17</sup>, the PDE TipF<sup>104</sup> and the GGDEF I site effector protein PopA<sup>13</sup> are dynamically localized to different cell poles, which is intimately linked to their specific functions during the cell cycle and even promotes cell cycle progression (BOX 2).

### Enzymatically inactive GGDEF and EAL proteins

Many species possess GGDEF and EAL domain proteins in which the GGDEF and EAL motifs, as well as amino acids at other positions, that are essential for the enzymatic function of DGCs and PDEs are not conserved<sup>16,21,34</sup>. What are the functions of these ‘degenerate’ GGDEF and EAL domain proteins and how do they act at the molecular level? As outlined above, functional sequestration of c-di-GMP control modules requires direct interactions between the GGDEF and EAL domain proteins, effectors and targets involved. It is conceivable that some of these systems might have further evolved by ‘giving up’ synthesis and degradation, and therefore the use of the local second-messenger c-di-GMP (which would also eliminate potential ‘spill-over’ of c-di-GMP), and now rely on protein–protein interactions only.

This is illustrated by the YcgF–YcgE system in *E. coli*<sup>24</sup>. YcgF is an EAL domain protein with a blue-light-sensing N-terminal BLUF domain<sup>48,49</sup>. Its EAL domain lacks all four amino acids that are involved in c-di-GMP binding and also lacks an essential catalytic glutamate residue. As a result, YcgF does not bind or degrade c-di-GMP, irrespective of blue light irradiation<sup>24</sup>. However, purified YcgF directly interacts with the YcgE repressor and, in response to blue light, releases YcgE from cognate operator sites *in vitro*, which is reflected by blue light-induced derepression of the corresponding genes *in vivo*. This leads to induction of a distinct set of small proteins,
two of which (YmgA and YmgB) act through the RcsB response regulator to interfere with curli expression and stimulate the production of the matrix exopolysaccharide colanic acid²⁴. Interestingly, the regulator YcgE is a paralogue of MlrA, the transcription factor that cooperates with active DGC and PDE enzymes (YdaM and YciR, respectively) to control transcription of the curli regulatory gene, *csgD* (discussed above).

The *E. coli* protein YhdA (also known as CsrD) is degenerate in key residues in its GGDEF and EAL domains. *In vivo*, YhdA stimulates the degradation of two small regulatory RNAs, CsrB and CsrC, that can sequester the mRNA-binding protein CsrA. CsrA is a global regulator that promotes the expression of the flagellar master regulator FlhDC and interferes with expression of two GGDEF domain proteins, YcdT and YdeH, and biosynthesis of the PGA exopolysaccharide²⁵,⁸⁷–⁸⁹,¹⁰⁵. *In vitro*, YhdA directly binds to the CsrB and CsrC RNAs with high affinity (although not with strict specificity)²⁵. In addition, YhdA is also required for expression of the curli regulator CsgD, suggesting that regulation at the RNA level also occurs in curli control⁸⁴. A role for EAL domain proteins in RNA binding and/or turnover is not too surprising, as the PDE substrate, c-di-GMP, is a small RNA itself.

In summary, degenerate GGDEF and EAL domain proteins do not act by synthesizing or degrading c-di-GMP, but rather function through direct macromolecular interactions. It should be noted that enzymatically inactive GGDEF and EAL domains that still bind GTP or c-di-GMP to allosterically control the activity of a partner protein or domain could be evolutionary intermediates on the pathway to degeneration. For example, GTP binding to the slightly degenerate GGDEF domain of PdeA from *C. crescentus* controls PDE activity of the C-terminal EAL domain²⁰. On this pathway, GGDEF or EAL proteins could also become c-di-GMP effector proteins.

Interestingly, all degenerate GGDEF and EAL domain proteins studied so far still regulate motility and/or biofilm functions. This also seems to be true for less well-characterized cases in other species¹⁰⁶,¹⁰⁷. Degenerate GGDEF and EAL domain proteins can indirectly affect cellular c-di-GMP levels; for example, YhdA affects the expression

of several GGDEF and EAL domain proteins through the Csr system and CsgD. Care should therefore be taken to assign DGC or PDE activity only to those GGDEF and EAL domain proteins that have been purified and found to be active *in vitro*, or at least only to those proteins that feature all key residues for enzymatic function.

### Conclusions and perspectives

c-di-GMP has emerged as a bacterial second-messenger molecule that is universally involved in the molecular decision between planktonic motile and sedentary bacterial lifestyles. The control of its synthesis and degradation, and above all its action, is of unprecedented versatility. The striking multiplicity in single species of DGCs (GGDEF domain proteins), c-di-GMP PDEs (EAL or HD-GYP domain proteins) and c-di-GMP-binding effector components, especially in proteobacteria, allows a plethora of signals to be integrated and many target processes to be regulated. Sets of GGDEF and EAL domain proteins are not only large in single species but also highly diverse even in closely related bacteria. In addition, the multiplicity of components provides the basis for functional and spatial sequestration of some c-di-GMP control modules in separate pathways that can operate in parallel. Such local operation requires protein–protein interactions, which in turn seem to have allowed the evolution of systems with degenerate GGDEF and EAL domain proteins that no longer rely on c-di-GMP metabolism to function, but can act through macromolecular interactions alone. Yet these systems seem to remain ‘evolutionarily trapped’ in their old physiological context: that is, the control of motility and/or biofilm-related functions.

To appreciate fully the importance of c-di-GMP in the bacterial world, we will have to assign molecular and physiological functions to all GGDEF and EAL domain proteins in certain model species, identify all the effector molecules and target processes affected and find out how these c-di-GMP control modules are integrated with global regulatory circuits that control bacterial stress responses, development and behaviour. In other words, we will have to unravel the systems biology of c-di-GMP signalling.

1. Ross, P. *et al.* Regulation of cellulose synthesis in *Acetobacter xylinum* by cyclic diguanylate. *Nature* **325**, 279–281 (1987). The first report of c-di-GMP as an allosteric activator for an enzyme.
2. Tal, R. *et al.* Three *cdg* operons control cellular turnover of cyclic di-GMP in *Acetobacter xylinum*: genetic organization and occurrence of conserved domains in isoenzymes. *J. Bacteriol.* **180**, 4416–4425 (1998).
3. Jenal, U. Cyclic di-guanosine-monophosphate comes of age: a novel secondary messenger involved in modulating cell surface structures in bacteria? *Curr. Opin. Microbiol.* **7**, 185–191 (2004).
4. Jenal, U. & Malone, J. Mechanisms of cyclic-di-GMP signaling in bacteria. *Annu. Rev. Genet.* **40**, 385–407 (2006). An excellent and comprehensive review on both the mechanisms and physiological context of c-di-GMP signalling.
5. Römling, U., Gomelsky, M. & Galperin, M. Y. C-di-GMP: the dawning of a novel bacterial signalling system. *Mol. Microbiol.* **57**, 629–639 (2005).
6. Römling, U. & Amikam, D. Cyclic di-GMP as a second messenger. *Curr. Opin. Microbiol.* **9**, 1–11 (2006).
7. Ryan, R. P., Fouhy, Y., Lucey, F. & Dow, J. M. Cyclic di-GMP signaling in bacteria: recent advances and new puzzles. *J. Bacteriol.* **128**, 8327–8334 (2006).
8. Wolfe, A. J. & Visick, K. L. Get the message out: cyclic-di-GMP regulates multiple levels of flagellum-based motility. *J. Bacteriol.* **130**, 463–475 (2008).
9. Cotter, P. A. & Stibitz, S. c-di-GMP-mediated regulation of virulence and biofilm formation. *Curr. Opin. Microbiol.* **10**, 17–23 (2007).
10. Dow, J. M., Fouhy, Y., Lucey, J. F. & Ryan, R. P. The HD-GYP domain, cyclic di-GMP signaling, and bacterial virulence to plants. *Mol. Plant Microbe Interact.* **19**, 1378–1384 (2006).
11. Ryan, R. P. *et al.* Cyclic di-GMP signalling in the virulence and environmental adaptation of *Xanthomonas campestris*. *Mol. Microbiol.* **63**, 429–442 (2007).
12. Tamayo, R., Pratt, J. T. & Camilli, A. Roles of cyclic diguanylate in the regulation of bacterial pathogenesis. *Annu. Rev. Microbiol.* **61**, 131–148 (2007).
13. Duerig, A. *et al.* Second messenger-mediated spatiotemporal control of protein degradation regulates bacterial cell cycle progression. *Genes Dev.* **23**, 93–104 (2009). This paper not only reports the first GGDEF I site-type c-di-GMP effector protein, but also identifies the direct target — a cell cycle regulator that is targeted for proteolysis — thereby revealing that this process is essential for cell cycle progression in *C. crescentus*.
14. Fineran, P. C., Williamson, N. R., Lilley, K. S. & Salmond, G. P. Virulence and prodigiosin antibiotic biosynthesis in *Serratia* are regulated pleiotropically by the GGDEF/EAL domain protein, PigX. *J. Bacteriol.* **189**, 7653–7662 (2007).
15. Hickman, J. W., Tifrea, D. F. & Harwood, C. S. A chemosensory system that regulates biofilm formation through modulation of cyclic diguanylate levels. *Proc. Natl Acad. Sci. USA* **102**, 14422–14427 (2005).
16. Malone, J. *et al.* The structure–function relationship of WspR, a *Pseudomonas fluorescens* response regulator with a GGDEF output domain. *Microbiology* **153**, 980–994 (2007).
REVIEW S

17. Paul, R. *et al.* Cell cycle-dependent dynamic localization of a bacterial response regulator with a novel di-guanylate cyclase output domain. *Genes Dev.* 18, 715–727 (2004).

The first experimental demonstration that the GGDEF domain exhibits DGC activity and that a DGC (PleD) is localized to a distinct cellular site (a cell pole) at a specific phase of the cell cycle of *C. crescentus*.

18. Ryjenkov, D. A., Tarutina, M., Moskvin, O. V. & Gomelsky, M. Cyclic diguanylate is a ubiquitous signaling molecule in bacteria: insights into biochemistry of the GGDEF protein domain. *J. Bacteriol.* 187, 1792–1798 (2005).

19. Chang, A. L. *et al.* Phosphodiesterase A1, a regulator of cellulose synthesis in *Acetobacter xylinum*, is a heme-based sensor. *Biochemistry* 40, 3420–3426 (2001).

20. Christen, M., Christen, B., Folcher, M., Schauerte, A. & Jenal, U. Identification and characterization of a cyclic di-GMP-specific phosphodiesterase and its allosteric control by GTP. *J. Biol. Chem.* 280, 30829–30837 (2005).

21. Schmidt, A. J., Ryjenkov, D. A. & Gomelsky, M. The ubiquitous protein domain EAL is a cyclic diguanylate-specific phosphodiesterase: enzymatically active and inactive EAL domains. *J. Bacteriol.* 187, 4774–4781 (2005).

22. Ryan, R. P. *et al.* Cell–cell signaling in *Xanthomonas campestris* involves an HD-GYP domain protein that functions in cyclic di-GMP turnover. *Proc. Natl Acad. Sci. USA* 103, 6712–6717 (2006).

The first study to detect the PDE activity of an HD-GYP domain.

23. Tamayo, R., Tischler, A. D. & Camilli, A. The EAL domain protein VieA is a cyclic diguanylate phosphodiesterase. *J. Biol. Chem.* 280, 33324–33330 (2005).

24. Tschowri, N. & Hengge, R. The BLUF-EAL protein YcgF acts as a direct anti-repressor in a blue light response of *E.coli*. *Genes Dev.* 23, 522–534 (2009).

This is the first paper to describe the direct molecular function of a degenerate EAL protein that depends on a protein–protein interaction for its activity.

25. Suzuki, K., Babitzke, P., Kushner, S. R. & Romeo, T. Identification of a novel regulatory protein (CsrD) that targets the global regulatory RNAs CsrB and CsrC for degradation by RNase E. *Genes Dev.* 20, 2605–2617 (2006).

26. Botsford, J. L. & Harman, J. G. Cyclic AMP in prokaryotes. *Microbiol. Rev.* 56, 100–122 (1992).

27. Holland, K., Busby, S. J. & Lloyd, G. S. New targets for the cyclic AMP receptor protein in the *Escherichia coli* K-12 genome. *FEMS Microbiol. Lett.* 274, 89–94 (2007).

28. Imamura, R. *et al.* Identification of the *cpdA* gene encoding cyclic 3′, 5′-adenosine monophosphate phosphodiesterase in *Escherichia coli*. *J. Biol. Chem.* 271, 25423–25429 (1996).

29. Chan, C. *et al.* Structural basis of activity and allosteric control of diguanylate cyclase. *Proc. Natl Acad. Sci. USA* 101, 17084–17089 (2004).

30. Paul, R. *et al.* Activation of the diguanylate cyclase PleD by phosphorylation-mediated dimerization. *J. Biol. Chem.* 282, 29170–29177 (2007).

31. Wassmann, P. *et al.* Structure of BeF3-modified response regulator PleD: implications of diguanylate cyclase activation, catalysis, and feedback inhibition. *Structure (Camb.)* 15, 915–927 (2007).

32. Christen, B. *et al.* Allosteric control of cyclic di-GMP signaling. *J. Biol. Chem.* 281, 32015–32024 (2006).

The authors showed that most DGCs have a secondary binding site for c-di-GMP (the I site) that is involved in product feedback inhibition.

33. De, N. *et al.* Phosphorylation-independent regulation of the diguanylate cyclase WspR. *PLoS Biol.* 6, e67 (2008).

34. Rao, F., Yang, Y., Qi, Y. & Liang, Z. X. Catalytic mechanism of c-di-GMP specific phosphodiesterase: a study of the EAL domain-containing RocR from *Pseudomonas aeruginosa*. *J. Bacteriol.* 190, 3622–3631 (2008).

This paper provides the structure of distinct conserved amino acids of an EAL domain and assigns them specific functions.

35. Galperin, M. Y., Natale, D. A., Aravind, L. & Koonin, E. V. A specialized version of the HD hydrolase domain implicated in signal transduction. *J. Mol. Microbiol. Biotechnol.* 1, 303–305 (1999).

36. García, B. *et al.* Role of the GGDEF protein family in *Salmonella* cellulose biosynthesis and biofilm formation. *Mol. Microbiol.* 54, 264–277 (2004).

37. Kuchma, S. L. *et al.* BifA, a cyclic-di-GMP phosphodiesterase, inversely regulates biofilm formation and swarming motility by *Pseudomonas aeruginosa* PA14. *J. Bacteriol.* 189, 8165–8178 (2007).

38. Weber, H., Pesavento, C., Possling, A., Tischendorf, G. & Hengge, R. Cyclic-di-GMP-mediated signaling within the σS network of *Escherichia coli*. *Mol. Microbiol.* 62, 1014–1034 (2006).

This paper shows that a specific DGC–PDE pair (YdaM–YciR) is crucial for adhesive curli fimbriae formation in *E. coli* and is the first to link c-di-GMP signaling to the general stress response.

39. Tarutina, M., Ryjenkov, D. A. & Gomelsky, L. An unorthodox bacteriophytochrome from *Rhodobacter sphaeroides* involved in turnover of the second messenger c-di-GMP. *J. Biol. Chem.* 281, 34751–34758 (2006).

40. Ferreira, R. B. R., Antunes, L. C. M., Greenberg, E. P. & McCarter, L. L. *Vibrio parahaemolyticus* ScrC modulates cyclic dimeric GMP regulation of gene expression relevant to growth on surfaces. *J. Bacteriol.* 190, 851–860 (2008).

41. Kumar, M. & Chatterji, D. Cyclic-di-GMP: a second messenger required for long-term survival, but not for biofilm formation, in *Mycobacterium smegmatis*. *Microbiology* 154, 2942–2955 (2008).

42. Galperin, M. Y., Nikolskaya, A. N. & Koonin, E. V. Novel domains of the prokaryotic two-component signal transduction systems. *FEMS Microbiol. Lett.* 203, 11–21 (2001).

43. Galperin, M. Y. Bacterial signal transduction network in a genomic perspective. *Environ. Microbiol.* 6, 552–567 (2004).

44. Delgado-Nixon, V. M., Gonzalez, G. & Gilles-Gonzalez, M.-A. Dos, a heme-binding PAS protein from *Escherichia coli*, is a direct oxygen sensor. *Biochemistry* 39, 2685–2691 (2000).

45. Sasakura, Y. *et al.* Characterization of a direct oxygen sensor heme protein from *Escherichia coli*. Effects of the heme redox states and mutations at the heme-binding site on catalysis and structure. *J. Biol. Chem.* 277, 23821–23827 (2002).

46. Takahashi, H. & Shimizu, T. Phosphodiesterase activity of Ec DOS, a heme-regulated enzyme from *Escherichia coli*, toward 3′,5′-cyclic diguanilic acid is obviously enhanced by O₂ and Co binding. *Chem. Lett.* 35, 970–971 (2006).

47. French, C. E., Bell, J. M. & Ward, F. B. Diversity and distribution of hemerythrin-like proteins in prokaryotes. *FEMS Microbiol. Lett.* 279, 131–145 (2008).

48. Hasegawa, K., Masuda, S. & Ono, T. A. Light induced structural changes of a full-length protein and its BLUF domain in YcgF (BlrP), a blue-light sensing protein that uses FAD (BLUF). *Biochemistry* 45, 3785–3793 (2006).

49. Nakasone, Y., Ono, T. A., Ishii, A., Masuda, S. & Terazima, M. Transient dimerization and conformational change of a BLUF protein: YcgF. *J. Am. Chem. Soc.* 129, 7028–7035 (2007).

50. Hurley, J. H. GAF domains: cyclic nucleotides come full circle. *Sci. STKE* 164, pe1 (2003).

51. Martinez-Wilson, H. F., Tamayo, R., Tischler, A. D., Lazinski, D. W. & Camilli, A. The *Vibrio cholerae* hybrid sensor kinase VieS contributes to motility and biofilm regulation by altering cyclic diguanylate level. *J. Bacteriol.* 190, 6439–6447 (2008).

52. Güvener, Z. T. & Harwood, C. S. Subcellular location characteristics of the *Pseudomonas aeruginosa* GGDEF protein, WspR, indicate that it produces cyclic-di-GMP in response to growth on surfaces. *Mol. Microbiol.* 66, 1459–1473 (2007).

53. Kazmierczak, B. I., Lebron, M. B. & Murray, T. S. Analysis of FimX, a phosphodiesterase that governs twitching motility in *Pseudomonas aeruginosa*. *Mol. Microbiol.* 60, 1026–1043 (2006).

54. Amikam, D. & Galperin, M. Y. PilZ domain is part of the bacterial c-di-GMP binding protein. *Bioinformatics* 22, 3–6 (2006).

An insightful bioinformatic prediction of c-di-GMP-binding activity by PilZ domain proteins.

55. Merighi, M., Lee, V. T., Hyodo, M., Hayakawa, Y. & Lory, S. The second messenger bis-(3′-5′)-cyclic-GMP and its PilZ domain-containing receptor Alg44 are required for alginate biosynthesis in *Pseudomonas aeruginosa*. *Mol. Microbiol.* 65, 876–895 (2007).

56. Oglesby, L., Jain, S. & Ohman, D. E. Membrane topology and roles of *Pseudomonas aeruginosa* Alg8 and Alg44 in alginate polymerization. *Microbiology* 154, 1605–1615 (2008).

57. Benach, J. *et al.* The structural basis of cyclic diguanylate signal transduction by PilZ domains. *EMBO J.* 26, 5153–5166 (2007).

58. Hickman, J. W. & Harwood, C. S. Identification of FleQ from *Pseudomonas aeruginosa* as a c-di-GMP-responsive transcription factor. *Mol. Microbiol.* 69, 376–389 (2008).

59. Lee, V. T. *et al.* A cyclic-di-GMP receptor required for bacterial exopolysaccharide production. *Mol. Microbiol.* 65, 1474–1484 (2007).

60. Beyhan, S., Odell, L. S. & Yildiz, F. H. Identification and characterization of cyclic diguanylate signaling systems controlling rugosity in *Vibrio cholerae*. *J. Bacteriol.* 190, 7392–7405 (2008).

61. Sudarsan, N. *et al.* Riboswitches in eubacteria sense the second messenger cyclic di-GMP. *Science* 321, 411–413 (2008).

Identification in several species of a c-di-GMP-binding riboswitch that serves as a direct effector for the regulation by c-di-GMP in 5′-untranslated regions of mRNAs.

62. Beyhan, S. & Yildiz, F. H. Smooth to rugose phase variation in *Vibrio cholerae* can be mediated by a single nucleotide change that targets c-di-GMP signalling pathway. *Mol. Microbiol.* 63, 995–1007 (2007).

63. Kader, A., Simm, R., Gerstel, U., Morr, M. & Römling, U. Hierarchical involvement of various GGDEF domain proteins in rdar morphotype development of *Salmonella enterica* serovar typhimurium. *Mol. Microbiol.* 60, 602–616 (2006).

64. Simm, R., Morr, M., Kader, A., Nimtz, M. & Römling, U. GGDEF and EAL domains inversely regulate cyclic di-GMP levels and transition from sessility to motility. *Mol. Microbiol.* 53, 1123–1134 (2004).

65. Tischler, A. D. & Camilli, A. Cyclic diguanylate regulates *Vibrio cholerae* virulence gene expression. *Infect. Immun.* 73, 5873–5882 (2005).

A key paper that links c-di-GMP signalling to virulence gene regulation.

66. Weinhouse, H. *et al.* c-di-GMP-binding protein, a new factor regulating cellulose synthesis in *Acetobacter xylinum*. *FEBS Lett.* 416, 207–211 (1997).

67. Christen, M. *et al.* DgrA is a member of a new family of cyclic diguanosine monophosphate receptors and controls flagellar motor function in *Caulobacter crescentus*. *Proc. Natl Acad. Sci. USA* 104, 4112–4117 (2007).

68. Pratt, J. T., Tamayo, R., Tischler, A. D. & Camilli, A. PilZ domain proteins bind cyclic diguanylate and regulate diverse processes in *Vibrio cholerae*. *J. Biol. Chem.* 282, 12860–12870 (2007).

69. Ryjenkov, D. A., Simm, R., Römling, U. & Gomelsky, M. The PilZ domain is a receptor for the second messenger c-di-GMP: the PilZ protein YcgR controls motility in enterobacteria. *J. Biol. Chem.* 281, 30310–30314 (2006).

70. Pesavento, C. *et al.* Inverse regulatory coordination of motility and curli-mediated adhesion in *Escherichia coli*. *Genes Dev.* 22, 2434–2446 (2008).

This paper clarifies the molecular functioning of the complex switch mechanism that results in mutual exclusion of the FlhDC (motility) and σS (adhesion) transcriptional control cascades.

71. Simm, R., Lusch, A., Kader, A., Andersson, M. & Römling, U. Role of EAL-containing proteins in multicellular behavior of *Salmonella enterica* serovar typhimurium. *J. Bacteriol.* 189, 3613–3623 (2007).

72. Brown, P. K. *et al.* MrA, a novel regulator of curli (Agf) and extracellular matrix synthesis by *Escherichia coli* and *Salmonella enterica* serovar typhimurium. *Mol. Microbiol.* 41, 349–363 (2001).

73. Gerstel, U., Park, C. & Römling, U. Complex regulation of *csgD* promoter activity by global regulatory proteins. *Mol. Microbiol.* 49, 639–654 (2003).

74. Jubelin, G. *et al.* CpxR/OmpR interplay regulates curli gene expression in response to osmolarity in *Escherichia coli*. *J. Bacteriol.* 187, 2038–2049 (2005).

75. Prigent-Combaret, C. *et al.* Complex regulatory network controls initial adhesion and biofilm formation in *Escherichia coli* via regulation of the *csgD* gene. *J. Bacteriol.* 2001, 7213–7223 (2001).

76. Vianney, A. *et al.* *Escherichia coli tol* and *rcs* genes participate in the complex network affecting curli synthesis. *Microbiology* 151, 2487–2497 (2005).

77. Hammer, B. K. & Bassler, B. L. Distinct sensory pathways in *Vibrio cholerae* El Tor and classical biotypes modulated c-di-GMP levels to control biofilm formation. *J. Bacteriol.* **191**, 169–177 (2008).

78. Lim, B., Beyhan, S. & Yildiz, F. H. Regulation of *Vibrio* polysaccharide synthesis and virulence factor production by CdgC, a GGDEF-EAL domain protein, in *Vibrio cholerae*. *J. Bacteriol.* **189**, 717–729 (2007).

79. Waters, C. M., Lu, W., Rabinowitz, J. D. & Bassler, B. L. Quorum sensing controls biofilm formation in *Vibrio cholerae* through modulation of cyclic di-GMP levels and repression of *vpsT*. *J. Bacteriol.* **190**, 2527–2536 (2008).

80. Girgis, H. S., Liu, Y., Ryu, W. S. & Tavazoie, S. A comprehensive genetic characterization of bacterial motility. *PLoS Genet.* **3**, e154 (2007). Using novel techniques for genome-wide genetic analysis, 36 novel genes that are associated with motility, including genes involved in c-di-GMP control of motility, were identified and analysed.

81. Paul, R. *et al.* Allosteric regulation of histidine kinases by their cognate response regulator determines cell fate. *Cell* **133**, 452–461 (2008).

82. Galperin, M. Y. A census of membrane-bound and intracellular signal transduction proteins in bacteria: bacterial IQ, extroverts and introverts. *BMC Microbiol.* **5**, 35 (2005).

83. Buckstein, M. H., He, J. & Rubin, H. Characterization of nucleotide pools as a function of physiological state in *Escherichia coli*. *J. Bacteriol.* **190**, 718–726 (2008).

84. Sommerfeldt, N. *et al.* Gene expression patterns and differential input into curli fimbriae regulation of all GGDEF/EAL domain proteins in *Escherichia coli*. *Microbiology* **155**, 1318–1331 (2009).

85. Frye, J. *et al.* Identification of new flagellar genes of *Salmonella enterica* serovar typhimurium. *J. Bacteriol.* **188**, 2233–2243 (2006).

86. Römling, U., Sierralta, W. D., Eriksson, K. & Normark, S. Multicellular and aggregative behaviour of *Salmonella typhimurium* strains is controlled by mutations in the *agfD* promoter. *Mol. Microbiol.* **28**, 249–264 (1998).

87. Jonas, K. *et al.* The RNA binding protein CsrA controls c-di-GMP metabolism by directly regulating the expression of GGDEF proteins. *Mol. Microbiol.* **70**, 236–257 (2008).

88. Babitzke, P. & Romeo, T. CsrB sRNA family: sequestration of RNA-binding regulatory proteins. *Curr. Opin. Microbiol.* **10**, 156–163 (2007).

89. Wang, X. *et al.* CsrA post-transcriptionally represses *pgaABCD*, responsible for synthesis of a biofilm polysaccharide adhesin of *Escherichia coli*. *Mol. Microbiol.* **56**, 1648–1663 (2005).

90. Fong, J. C. N. & Yildiz, F. H. Interplay between cAMP-CRP and c-di-GMP in *Vibrio cholerae* biofilm formation. *J. Bacteriol.* **190**, 6646–6659 (2008).

91. Yildiz, F. H., Liu, X. S., Heydorn, A. & Schoolnick, G. K. Molecular analysis of rugosity in a *Vibrio cholerae* O1 El Tor phase variant. *Mol. Microbiol.* **53**, 497–515 (2004).

92. Camilli, A. & Bassler, B. L. Bacterial small-molecule signaling pathways. *Science* **311**, 1113–1116 (2006).

93. Schild, S. *et al.* Genes induced late in infection increase fitness of *Vibrio cholerae* after release into the environment. *Cell Host Microbe* **2**, 264–277 (2007).

94. Tamayo, R., Schild, S., Pratt, J. T. & Camilli, A. Role of cyclic di-GMP during El Tor biotype *Vibrio cholerae* infection: characterization of the *in vivo*-induced cyclic di-GMP phosphodiesterase CdpA. *Infect. Immun.* **76**, 1617–1627 (2008).

95. Kirillina, O., Fetherstone, J. D., Bobrov, A. G., Abney, J. & Perry, R. D. HmsP, a putative phosphodiesterase, and HmsT, a putative diguanylate cyclase, control Hms-dependent biofilm formation in *Yersinia pestis*. *Mol. Microbiol.* **54**, 75–88 (2004).

96. Perry, R. D. *et al.* Temperature regulation of the hemin storage (*Hms*⁺) phenotype of *Yersinia pestis* is posttranscriptional. *J. Bacteriol.* **186**, 1638–1647 (2004). The first report to show that regulated proteolysis of GGDEF proteins plays a key part in c-di-GMP signalling.

97. Kulasakara, H. *et al.* Analysis of *Pseudomonas aeruginosa* diguanylate cyclases and phosphodiesterase reveals a role for bis-(3′-5′)-cyclic-GMP in virulence. *Proc. Natl Acad. Sci. USA* **103**, 2839–2844 (2006).

98. Baker, D. A. & Kelly, J. M. Structure, function and evolution of microbial adenylyl and guanylyl cyclases. *Mol. Microbiol.* **52**, 1229–1242 (2004).

99. Shenoy, A. R. & Visweswariah, S. S. Class III nucleotide cyclases in bacteria and archaea: lineage-specific expansion of adenyl cyclases and a dearth of guanylyl cyclases. *FEBS Lett.* **561**, 11–21 (2004).

100. Shenoy, A. R. & Visweswariah, S. S. New messages from old messengers: cAMP and mycobacteria. *Trends Microbiol.* **14**, 543–550 (2006).

101. Willoughby, D. & Cooper, D. M. Organization and Ca²⁺ regulation of adenyl cyclases in cAMP microdomains. *Physiol. Rev.* **87**, 965–1010 (2007).

102. Andrade, M. O. *et al.* The HD-GYP domain of RpfG mediates a direct linkage between the Rpf quorum-sensing pathway and a subset of diguanylate cyclase proteins in the phytopathogen *Xanthomonas axonopodis pv citri*. *Mol. Microbiol.* **62**, 537–551 (2006).

103. Bobrov, A. G., Kirillina, O., Forman, S., Mack, D. & Perry, R. D. Insights into *Yersinia pestis* biofilm development: topology and co-interaction of Hms inner membrane proteins involved in exopolysaccharide production. *Environ. Microbiol.* **10**, 1419–1432 (2008).

104. Huitema, E., Pritchard, S., Matteson, D., Radhakrishnan, S. K. & Viollier, P. H. Bacterial birth scar proteins mark future flagellum assembly site. *Cell* **124**, 1025–1037 (2006). This paper describes the striking finding that marker proteins (TipN and TipF) are deposited at the new-born poles during cell division. One of these proteins was shown to be an EAL-type PDE that was essential for flagellum assembly at the incipient swarmer pole of *C. crescentus*.

105. Jonas, K., Tomenius, H., Römling, U., Georgellis, D. & Melefors, O. Identification of YhdA as a regulator of the *Escherichia coli* carbon storage regulation system. *FEMS Microbiol. Lett.* **264**, 232–237 (2006).

106. Holland, L. M. *et al.* A staphylococcal GGDEF domain protein regulates biofilm formation independently of c-di-GMP. *J. Bacteriol.* **190**, 5178–5189 (2008).

107. Thormann, K. M. *et al.* Control of formation and cellular detachment from *Shewanella oneidensis* MR-1 biofilms by cyclic di-GMP. *J. Bacteriol.* **188**, 2681–2691 (2006).

108. Adler, J. & Templeton, B. The effect of environmental conditions on the motility of *Escherichia coli*. *J. Gen. Microbiol.* **46**, 175–184 (1967).

109. Amsler, C. D., Cho, M. & Matsumura, P. Multiple factors underlying the maximum motility of *Escherichia coli* as cultures enter post-exponential growth. *J. Bacteriol.* **175**, 6238–6244 (1993).

110. Olsén, A., Wick, M. J., Mörgelin, M. & Björck, L. Curli fibrous surface proteins of *Escherichia coli* interact with major histocompatibility complex class I molecules. *Infect. Immun.* **66**, 944–949 (1998).

111. Brombacher, E., Baratto, A., Dorel, C. & Landini, P. Gene expression regulation by the curli activator CsgD protein: modulation of cellulose biosynthesis and control of negative determinants for microbial adhesion. *J. Bacteriol.* **188**, 2027–2037 (2006).

112. Zogaj, X., Nimtz, M., Rohde, M., Bokranz, W. & Römling, U. The multicellular morphotypes of *Salmonella typhimurium* and *Escherichia coli* produce cellulose as the second component of the extracellular matrix. *Mol. Microbiol.* **39**, 1452–1463 (2001). The first report of cellulose as an extracellular matrix polysaccharide in biofilms at a wet surface-air interface.

113. Aldridge, P., Paul, R., Goymre, P., Rainey, P. & Jenal, U. Role of the GGDEF regulator PleD in polar development of *Caulobacter crescentus*. *Mol. Microbiol.* **47**, 1695–1708 (2003).

114. Viollier, P. H., Sternheim, N. & Shapiro, L. Identification of a localization factor for the polar positioning of bacterial structural and regulatory proteins. *Proc. Natl Acad. Sci. USA* **99**, 13831–13836 (2002).

115. Alm, R. A., Bodero, A. J., Free, P. D. & Mattick, J. S. Identification of a novel gene, *pilZ*, essential for type 4 fimbril biogenesis in *Pseudomonas aeruginosa*. *J. Bacteriol.* **178**, 46–53 (1996).


**Acknowledgements**

The author thanks U. Jenal for providing the figure in BOX 2 and for helpful discussions, and C. Pesavento for providing input on the manuscript. Work from the author’s laboratory cited in this review has been supported by the Deutsche Forschungsgemeinschaft, the Fonds der Chemischen Industrie and the Dr Hans Messner Stiftung.


**DATABASES**

Entrez Genome Project: [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=genomeprj](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=genomeprj)

*Caulobacter crescentus* | *Escherichia coli* | *Geobacter uraniumreducens* | *Mycobacterium smegmatis* | *Pseudomonas aeruginosa* | *Rhodobacter sphaeroides* | *Thermotoga maritima* | *Vibrio cholerae* | *Vibrio parahaemolyticus* | *Yersinia pestis*


**FURTHER INFORMATION**

Regine Hengge’s homepage: [http://www.biologie.fu-berlin.de/en/microbio/mibi-hengge/index.html](http://www.biologie.fu-berlin.de/en/microbio/mibi-hengge/index.html)

ALL LINKS ARE ACTIVE IN THE ONLINE PDF
