
Marzenna Blonska  
Xin Lin  

CARMA1-mediated NF-κB and JNK activation in lymphocytes  

Authors' address  
Marzenna Blonska${}^{1}$, Xin Lin${}^{1}$  
${}^{1}$Department of Molecular and Cellular Oncology, University of Texas, M.D. Anderson Cancer Center, Houston, TX, USA.  

Correspondence to:  
Xin Lin  
Department of Molecular and Cellular Oncology  
University of Texas  
M.D. Anderson Cancer Center  
1515 Holcombe Boulevard  
Unit 108, Houston  
TX77030, USA  
Tel.: +1 713 792 8969  
Fax: +1 713 794 3270  
e-mail: xlin@mdanderson.org  

Acknowledgements  
This work was supported by grants from the National Institutes of Health (RO1GM065899 and RO1GM079451) to X. L. X. L. is a Scholar of Leukemia and Lymphoma Society, and a recipient of the Investigator Award of Cancer Research Institute, Inc. M. B. is a Special Fellow of Leukemia and Lymphoma Society.  

Summary: Activation of transcription factor nuclear factor-κB (NF-κB) and Jun N-terminal kinase (JNK) play the pivotal roles in regulation of lymphocyte activation and proliferation. Deregulation of these signaling pathways leads to inappropriate immune response and contributes to the development of leukemia/lymphoma. The scaffold protein CARMA1 [caspase-recruitment domain (CARD) membrane-associated guanylate kinase (MAGUK) protein 1] has a central role in regulation of NF-κB and the JNK2/c-Jun complex in both B and T lymphocytes. During last several years, tremendous work has been done to reveal the mechanism by which CARMA1 and its signaling partners, B cell CLL-lymphoma 10 and mucosa-associated lymphoid tissue 1, are activated and mediate NF-κB and JNK activation. In this review, we summarize our findings in revealing the roles of CARMA1 in the NF-κB and JNK signaling pathways in the context of recent advances in this field.  

Keywords: NF-κB, IKK, JNK, Jun, AP-1, CARMA1  

Introduction  
Lymphocyte activation is a key step in the adaptive immune response. Activated naive T cells undergo clonal expansion and acquire the capability to kill target cells infected with pathogens or to produce cytokines essential for regulating immune response. Deregulation of this process results in immunodeficiency, autoimmune diseases, or leukemia/lymphoma. T-cell activation and proliferation is initiated by the interaction of T-cell receptor (TCR) with antigen peptides presented in the context of a major histocompatibility complex (MHC) class II by antigen-presenting cells (APCs). A second, costimulatory signal is generated through interaction of B7 molecules on APCs and CD28 receptors on T cells (1). This combined stimulation (known as TCR/CD28 or CD3/CD28 costimulation) induces a series of signal transduction cascades leading to the activation of protein tyrosine kinases, various adapter proteins, small GTPases, effector enzymes such as serine/threonine kinases, and transcription factors, including nuclear factor κB (NF-κB), activator protein-1 (AP-1), and nuclear factor of activated T cells (NFAT) (reviewed in 2, 3). These transcription factors ultimately

© 2009 The Authors  
Journal compilation © 2009 Blackwell Munksgaard  
Immunological Reviews  
0105-2896  

© 2009 The Authors • Journal compilation © 2009 Blackwell Munksgaard • Immunological Reviews 228/2009 199

control the expression of cytokines and genes that regulate T-cell activation, differentiation, and proliferation (3, 4).

NF-κB is necessary for full T-cell activation and effective immune responses (3). In resting cells, NF-κB is sequestered in the cytoplasm through the inhibitory molecule, termed inhibitor of NF-κB (IκB), which masks the nuclear localization sequence of NF-κB. Stimulation of cells with various stimuli, including stimulation of TCR signaling pathway, leads to activation of the IκB kinase (IKK) complex that contains two catalytic subunits, IKKα and IKKβ, and one regulatory subunit, IKKγ/NF-κB essential modulator (NEMO) (5). The activated IKK phosphorylates IκB, triggering rapid ubiquitination and degradation of IκB by the 26S proteasome complex, which unmasks the nuclear localization sequence of NF-κB. Therefore, NF-κB can be rapidly translocated into the nucleus and initiate the transcription of its target genes (5).

During last several years, tremendous progress has been made in the identification of mechanism and components that regulate IKK activity in various cell types including lymphocytes (3, 6). Within these years, our research group has also participated in revealing the signaling cascade that mediates TCR-induced NF-κB activation (7–12). One of our key contributions is the demonstration of the essential role of CARMA1 [caspase-recruitment domain (CARD) membrane-associated guanylate kinase (MAGUK) protein 1], a scaffold protein, in TCR-induced NF-κB activation (7). In this article, we summarize our recent findings in CARMA1-mediated NF-κB activation as well as the research progress in this field. In addition, as we have recently identified the molecular mechanism by which CARMA1 controls c-Jun N-terminal kinase (JNK) activity and the c-Jun level (13), we also discuss the role of CARMA1 in TCR-induced JNK2 activation.

is recruited into the IS following the stimulation by APCs (17). By expressing dominant-negative mutants of PKCθ, we (18) and others (19) suggested that PKCθ might be involved in TCR-induced NF-κB activation. Indeed, gene-targeting studies provided the genetic evidence that PKCθ is required for TCR-induced NF-κB activation (20).

One of important questions at the time was how PKCθ links TCR signals to the IKK complex. To address this question, we took a somatic approach to identify signaling molecules that link PKCθ to the IKK complex, in which we generated a Jurkat T-cell line (JGFP1) that expresses green fluorescent protein (GFP) under the control of a NF-κB-dependent promoter. This cell line was subjected to random mutagenesis. The mutagenized cells were subjected to the selection of GFP-negative cells under the condition of TCR stimulation, and this pool of GFP-negative cells was further selected for GFP-positive cells following tumor necrosis factor α (TNFα) stimulation. Using this selection method, we cloned a mutant cell line, named JPM50.6 (Fig. 1), which is defective in CD3/CD28 costimulation but not TNFα-induced NF-κB activation (7). Subsequently, we have found that JPM50.6 cells are defective in the expression of CARMA1 protein, and expression of CARMA1 can effectively rescue the defect of TCR-induced NF-κB activation (7). Together, our study provides the genetic evidence that CARMA1 is required for TCR-mediated NF-κB activation. Consistent with our findings, two other groups independently demonstrate that CARMA1 plays an essential role in TCR-induced NF-κB activation by

The identification of CARMA1 as the essential component mediating antigen receptor-induced NF-κB activation

Stimulation of T cells by APCs induces the formation of a large multi-component complex at the contact area between the T cell and the APC, termed the supramolecular activation complex (SMAC) or immunological synapse (IS) (14). The SMAC/IS of T cell is highly enriched in cholesterol and glycosphingolipids, also termed as lipid rafts or microdomains (15). Some signaling molecules are constitutively associated with lipid rafts, while others are recruited into lipid rafts following CD3/CD28 costimulation (16). Early studies indicate that protein kinase Cθ (PKCθ), a Ca²⁺-independent PKC isoform,

JGFP1 (parental cell)
A Unstimulated
Counts
0 10 20 30 40 50
10⁰ 10¹ 10² 10³ 10⁴
B PMA-CD28
Counts
0 10 20 30 40 50
10⁰ 10¹ 10² 10³ 10⁴
Fluorescence
C TNF-α
Counts
0 10 20 30 40 50
10⁰ 10¹ 10² 10³ 10⁴
JPM50.6 (mutant cell)
D Unstimulated
Counts
0 10 20 30 40 50
10⁰ 10¹ 10² 10³ 10⁴
E PMA-CD28
Counts
0 10 20 30 40 50
10⁰ 10¹ 10² 10³ 10⁴
Fluorescence
F TNF-α
Counts
0 10 20 30 40 50
10⁰ 10¹ 10² 10³ 10⁴
Fluorescence

Fig. 1. Expression of a nuclear factor-κB (NF-κB)-dependent EGFP gene in a mutant Jurkat T cell line, JPM50.6. (A–C) Jurkat T-cell line (JGFP1) or (D–F) JPM50.6 cells were (A, D) unstimulated, (B, E) stimulated with phorbol 12-myristate 13-acetate (PMA) (10 ng/ml) plus CD28 monoclonal antibody (1 μg/ml), or (C, F) tumor necrosis factor α (TNFα) (20 ng/ml) for 12 h. The cells were collected and analyzed by flow cytometer (BD FACSCalibur). This figure is adapted from reference (7) with permission.
expressing dominant-negative mutants of CARMA1 (21) or by using CARMA1 RNA interference (RNAi) approach (22). Subsequently, the gene-targeting (23–25) and ENU mutagenesis (26) studies in mice further demonstrate that CARMA1 is required for B-cell receptor (BCR)- and TCR-induced NF-κB activation, which confirms the notion that CARMA1 is an essential signaling component mediating antigen receptor-induced NF-κB activation. Moreover, antigen-induced proliferation of T and B cells from carma1-null mice is defective (23–25). In addition, CARMA1 deficiency also selectively impairs the development of a subset of B cells, which results in reduced marginal-zone B cells and absence of peritoneal B1 subpopulation (24, 27). Altogether, these studies demonstrate that CARMA1 is an essential component mediating antigen receptor-induced NF-κB activation.

The CARMA family proteins

CARMA1 (also known as CARD11) was independently identified by two research groups using bioinformatics approaches based on its CARD (28, 29), and it belongs to the MAGUK family of proteins that function as molecular scaffolds to assist recruitment and assembly of signaling molecules in the cytoplasmic membrane (30). In addition to the N-terminal CARD, CARMA1 contains several protein-protein interaction domains including a coiled-coil domain (C-C), a PDZ domain, a Src homology 3 (SH3) domain, and a guanylate kinase (GuK)-like domain (28, 29) (Fig. 2). The structural arrangement of PDZ, SH3, and GUK domains is also called MAGUK domain. CARMA1 has two mammalian homologs, CARMA2 (also known as CARD14 or Bimp2) and CARMA3 (also known as CARD10 and Bimp1) that share high degree of sequence and structural homology (28, 31, 32) (Fig. 2). These three CARMA proteins exhibit a distinct tissue distribution pattern. Specifically, CARMA1 is predominantly expressed in hematopoietic tissues such as spleen, thymus, and peripheral blood leukocytes (PBLs); CARMA2 is expressed only in placenta; and CARMA3 is expressed in a broad range of tissues but not in hematopoietic cells (28, 31, 32). This distinct tissue distribution suggests that CARMA family members may have the same function and play similar roles in different tissues. Consistent with this hypothesis, ectopic expression of CARMA3 in CARMA1-deficient T cells effectively rescues the defect of TCR-induced NF-κB activation (10). Using a gene-targeting approach, we recently generated CARMA3-deficient mice and demonstrated that CARMA3 is a critical component in G protein-coupled receptor (GPCR) signaling pathways and mediates GPCR-induced NF-κB activation (33). However, it remains to be determined whether CARMA1 is also required for GPCR-induced NF-κB in hematopoietic cells and whether CARMA3 is involved in other signaling pathways besides GPCRs.

CARMA1, Bcl10, and MALT1 complex

Upon TCR stimulation, CARMA1 is redistributed to the lipid rafts (21) and recruits two signaling components, B-cell CLL-lymphoma 10 (Bcl10) and mucosa-associated lymphoid tissue lymphoma translocation gene 1 (MALT1), also known as Paracaspase (8, 9) (Fig. 2). This TCR-induced assembly of the CARMA1–Bcl10–MALT1 complex (known as CBM complex) subsequently interacts with and induces the activity of the IKK complex, leading to NF-κB activation (8, 11, 34).

Bcl10, a CARD-containing adapter protein, was identified by functional cloning from MALT cells (35, 36) and by bioinformatics searching for CARD-containing proteins (37–39). One of early key findings for antigen receptor-induced NF-κB activation was the demonstration of Bcl10 as an essential signaling component in TCR- and BCR-induced NF-κB activation and functions downstream of PKC (40). Like CARMA1-deficient mice, Bcl10 deficiency also leads to reduced marginal-zone B cells and absence of peritoneal B1 subpopulation.

MALT1, another MALT lymphoma-associated oncogene, was also identified using functional cloning from MALT lymphoma samples (41, 42) and bioinformatics approaches (43). Although gene-targeting studies demonstrate that MALT1 is required for TCR-induced NF-κB activation (44, 45), the initial analysis of BCR-induced NF-κB activation in Malt1-deficient mice led to two conflict conclusions. One study showed that Malt1 deficiency resulted in the defect of BCR-induced NF-κB activation (44), whereas another study suggested that BCR-induced NF-κB activation was not defective

Fig. 2. Structures of CARMA1–3 (CARD MAGUK protein 1–3), B-cell CLL-lymphoma 10 (Bcl10), and mucosa-associated lymphoid tissue lymphoma translocation gene 1 (MALT1). CARD, caspase-recruitment domain; C-C, coiled-coil domain; MAGUK, membrane-associated guanylate kinase (GuK)-like domain; S/T rich, Ser/Thr-rich domain; DD, death domain; Ig, immunoglobulin-like domain.

in Malt1-deficient mice (45). However, more recent studies suggest that this discrepancy might be due to that Malt1 is mainly involved in BCR-induced activation of c-Rel, a subunit of NF-kB (46). Therefore, based on the differential requirement of Malt1 in the activation of NF-kB subunits (46), it will be interesting to determine whether BCR stimulation also induces the same CBM complex as TCR stimulation does.

### Subcellular localization of CARMA1

Subcellular localization of CARMA1 appears to be critical for its function. CARMA1 is constitutively associated with the cytoplasmic membrane and recruited into IS following TCR stimulation (8, 10, 11, 21, 34). Two research groups independently cloned the cDNA encoding CARMA1, and named CARMA1 (28) or CARD11 (29). Although their studies showed that ectopic expression of both CARMA1 and CARD11 led to the potent activation of NF-kB (28, 29), we found that the expression vector encoding CARMA1 but not CARD11 could rescue the defect of CD3/CD28 costimulation-induced NF-kB activation in CARMA1-deficient Jurkat T cells (8). The alignment of CARMA1 and CARD11 sequence revealed a point mutation in CARD11 resulting in substitution of Leu808 with Pro in its SH3 domain. Our analysis of human genome sequence confirmed that wildtype CARMA1 cDNA should encode Leu at residue 808. The significance of proper CARMA1 localization for its function was demonstrated using CARMA1 with L808P mutation. Our studies have indicated that although CARMA1 is mainly associated with the cytoplasmic membrane, whereas the CARMA1 (L808P) is uniformly distributed in cytosol. Moreover, CARMA1 (L808P) is not recruited into the IS and does not mediate NF-kB activation upon CD3/CD28 costimulation (8), indicating that the membrane localization of CARMA1 is essential for TCR-induced NF-kB activation. However, CARMA1 contains neither a transmembrane domain nor a membrane-anchoring motif. Therefore, it is likely that CARMA1 associates with an unknown adapter protein that is responsible for anchoring CARMA1 to the membrane.

Besides its localization in the cytoplasmic membrane, CARMA1 is also inducibly recruited into the IS or lipid rafts upon TCR stimulation (8, 21). It is not clear whether the protein anchoring CARMA1 to cytoplasmic membrane is also involved in the recruitment of CARMA1 into the IS. Several proteins have been implicated in regulating CARMA1's subcellular localization. One study has reported that phosphoinositide-dependent kinase 1 (PDK1) physically associates with CARMA1 following CD3/CD28 costimulation and possibly recruits CARMA1 into lipid rafts (47). It is also implicated that PDK1 is required for CD3/CD28-induced NF-kB activation (47). Another study has identified the adhesion- and degranulation-promoting adapter protein (ADAP) as a CARMA1-interacting protein, which also contributes to CARMA1 translocation to lipid rafts. ADAP-deficient T cells have impaired CD3/CD28-induced NF-kB activation (48). However, ADAP is not constitutively associated with cytoplasmic membrane, suggesting that ADAP is likely not the protein constitutively linking CARMA1 to cytoplasmic membrane. Therefore, it remains to be determined which proteins link CARMA1 to the cytoplasmic membrane and whether they are required for recruiting CARMA1 to the IS.

#### Posttranslational modification of CARMA1

The deficiency of CARMA1 results in not only the defect of antigen-receptor-induced NF-kB but also the defect of NF-kB activation induced by phorbol 12-myristate 13-acetate (PMA) plus ionomycin, the pharmacological inducer of PKC, suggesting that CARMA1 functions downstream of PKC in antigen receptor signaling pathways. Earlier studies indicate that PKCθ (18–20) and PKCβ (49, 50) are required for TCR- and BCR-induced NF-kB activation, respectively. To determine whether CARMA1 is connected to PKCθ, we examined whether these two proteins form a complex. Our results demonstrated that CARMA1 indeed physically associated with PKCθ, and this interaction was dependent on the linker region (residues 432–671) between the C-C and PDZ domains of CARMA1 (8). Further, we found that overexpression of constitutively active mutant of PKCθ potently induced phosphorylation of CARMA1 (10). Therefore, we hypothesized that the CARMA1 linker region might contain one or more PKC phosphorylation sites. As the linker region is highly enriched in Ser and Thr residues, we combined the bioinformatics approach with peptide mapping and mutagenesis methods to identified PKC-specific phosphorylation sites. We found that the mutation of Ser552, Ser555, or Ser645 to Ala in the linker region of human CARMA1 impaired PKCθ-induced phosphorylation of CARMA1 in vitro, and CARMA1 mutants with substitutions of these residues failed to mediate TCR-induced NF-kB activation (10). Independently, David Rawlings’ group (51) also identified homologous Ser residues in mouse CARMA1 as phosphorylation sites of PKCβ and PKCθ. Consistent with these studies, Shinohara et al. (52, 53) has reported that PKCβ directly phosphorylate CARMA1 upon BCR stimulation in chicken DT40 B cells. Together, these studies support the model of PKCθ-dependent and PKCβ-dependent phosphorylation of CARMA1 in T cells and B cells, respectively.
Although Ser565 of CARMA1 is not predicted to be the putative PKC phosphorylation site, our mass spectra analysis also revealed Ser565 of CARMA1 was phosphorylated following CD3/CD28 costimulation (10). Importantly, the mutation of this residue also resulted in a functional defect of CARMA1-mediated NF-κB activation, suggesting that at least one additional kinase might phosphorylate CARMA1 and plays a critical role in TCR-induced NF-κB activation. Several kinases, such as PDK1, AKT, and TAK1 [transforming growth factor-β (TGFβ)-activated protein kinase], have been shown to associate with CARMA1 upon TCR stimulation (47, 52, 54, 55). However, it remains to be determined whether the linker region or other domains of CARMA1 serves as substrates for these kinases.

Signal-dependent phosphorylation of CARMA1 likely induces conformational changes that enable CARMA1 to associate with downstream components such as Bcl10 and MALT1 (10, 51, 53). Therefore, we (10) and others (51) propose that CARMA1 resides in an inactive form in unstimulated cells, but following antigen receptor stimulation and phosphorylation by PKC, CARMA1 switches to an active conformation and exposes an interaction site for Bcl10 binding, which further recruits the IKK complex, leading to NF-κB activation (Fig. 3). Consistent with this model, deletion of the CARMA1 linker region results in constitutive CARMA1 activity (51, 56).

In addition to the linker region-dependent phosphorylation, other kinases might contribute to CARMA1 phosphorylation outside the linker region. Ishiguro et al. have reported that calmodulin-dependent protein kinase II (CaMKII) is recruited to the IS following TCR stimulation and phosphorylates CARMA1 on Ser109 (Fig. 3). This phosphorylation may also

CARMA1
Linker
PDZ
SH3
GUK
C-C
CARD
Bcl10
CARD
S/T

PKC

CARMA1
S555
P
P
PDZ
SH3
GUK
S645
S552
C-C
CARD
S109
Bcl10
S/T

Fig. 3. Schematic presentation of the phosphorylation-dependent conformational change of CARMA1 [caspase-recruitment domain (CARD) membrane-associated guanylate kinase (MAGUK) protein 1]. Protein kinase C (PKC)-dependent phosphorylation of the linker region of CARMA1 induces conformational change and enables CARMA1 to interact with B-cell CLL-lymphoma 10 (Bcl10) (through their CARD domains).

facilitate the interaction between CARMA1 and its downstream component Bcl10 and enhances NF-κB activation (57). It will be interesting to determine whether other kinases that have been implicated in TCR-induced NF-κB activation are also involved in phosphorylation of CARMA1. Such studies will provide further insight into the mechanism by which CARMA1 mediates antigen receptor-induced NF-κB activation.

Posttranslational modification of Bcl10 positively and negatively regulates antigen receptor signaling

Bcl10 contains an N-terminal CARD domain and a C-terminal Ser/Thr-rich domain (Fig. 2). The CARD domain of Bcl10 is responsible for the interaction with CARMA1 and oligomerization following TCR stimulation (8, 58, 59), which is believed to be required for antigen receptor-induced NF-κB activation. More recently, it has been shown that Lys31 and Lys63 residues in the CARD domain of Bcl10 are required for signal-induced polyubiquitination (60). This polyubiquitination is linked through the Lys63 (K63) residue of ubiquitin, which functions as a protein–protein interaction motif rather than triggering the proteasome-mediated degradation. As the result, the polyubiquitination chain on Bcl10 can be recognized by NEMO subunit in the IKK complex, which leads to the efficient interaction between the IKK complex and the CBM complex (60).

It is well established that Bcl10 is phosphorylated in response to TCR activation, which regulates NF-κB activation both positively (61) and negatively (62–65). Although the function of the C-terminal Ser/Thr-rich domain of Bcl10 is not fully determined, several studies suggest that signal-dependent phosphorylation of these Ser/Thr residues plays a critical role for ubiquitination-mediated degradation of Bcl10 (62–65). Bcl10-deficient T cells reconstituted with Bcl10-S138A mutant have impaired TCR-induced phosphorylation, ubiquitination, and subsequently degradation of Bcl10, which leads to a prolonged NF-κB activation and enhanced IL-2 production (64). Of note, CaMKII has been shown to phosphorylate Ser138 in Bcl10 (63). In addition, it has also been shown that IKKβ is involved in the phosphorylation of multiple Ser residues in Bcl10 (62, 65).

Several ubiquitin E3 ligases, such as NEDD4, Itch, and β-TrCP, have been suggested to be involved in Bcl10 ubiquitination (65, 66). Scharschmidt et al. (66) suggest that NEDD4 and Itch are involved in Bcl10 ubiquitination, and the ubiquitinated Bcl10 is subjected to lysosomal-mediated degradation. By contrast, Lobry et al. (65) suggest that β-TrCP mediates the ubiquitination of Bcl10 following IKKβ-dependent
phosphorylation, which is then subjected to the proteasome-mediated degradation. Therefore, further studies are required to determine the contribution of individual E3 ligases. Of note, the K63-linked ubiquitination of Lys31 and Lys63 in the CARD domain of Bcl10 is likely different from these phosphorylation-dependent ubiquitination in the C-terminal Ser/Thr-rich domain. Therefore, it will also be interesting to determine which E3 ligase is involved in ubiquitination of Lys31 and Lys63 in Bcl10.

The roles of MALT1 in antigen receptor signaling pathway

MALT1 contains an N-terminal death domain (DD), followed by two immunoglobulin (Ig)-like domains, and a C-terminal caspase-like domain (Fig. 2). The mechanism by which MALT1 is involved in antigen receptor-induced NF-κB activation is somewhat controversial. One study suggests that MALT1 functions as an ubiquitin E3 ligase to induce K63-linked polyubiquitination of NEMO (67), whereas another study suggests that tumor necrosis factor receptor-associated factor 6 (TRAF6), an ubiquitin E3 ligase, induces K63-linked polyubiquitination of NEMO (68). More recently, another study suggests that TRAF6 inducibly associates with MALT1 and functions as an E3 ligase for MALT1. In this case, MALT1 itself undergoes the K63-linked polyubiquitination upon TCR stimulation, and this polyubiquitin chain on Malt1 provides a docking surface for binding of NEMO, leading to the recruitment of the IKK complex (69).

More recently, MALT1 has also been shown to have proteolytic activity upon T-cell stimulation (70, 71). Both Bcl10 and A20, a deubiquitinating enzyme, have been identified as the substrates for MALT1 (70, 71). The proteolytic activity of MALT1 is required for optimal NF-κB activation and enhances TCR-induced IL-2 production (70, 71). However, one study suggests that MALT1 directly regulates Caspase-8 activation upon TCR stimulation, which is required for NF-κB activation and IL-2 production (72). This study may partially explain the earlier observation that Caspase-8 is associated with the CBM complex and involved in TCR-induced NF-κB activation (73).

with its substrate IKKβ (52, 68). However, other studies propose that the CBM complex serves as a molecular platform to recruit signaling components responsible for K63-linked polyubiquitination of NEMO (67). The study from our laboratory supports the second model. We have recently found that although signal-induced IKKβ phosphorylation is not defective in CARMA1- or Bcl10-deficient cells, the IKK kinase activity is completely defective, which is likely due to the defect of signal-induced NEMO polyubiquitination (11). Therefore, our conclusion is that TCR-induced IKK phosphorylation is independent on the CBM complex, whereas TCR-induced NEMO polyubiquitination is dependent on the CBM complex (11).

The mechanism by which the CBM complex regulates NEMO polyubiquitination is not fully defined. Earlier studies suggest that TRAF6 functions as the NEMO E3 ligase and induces K63-linked polyubiquitination of NEMO following TCR stimulation (68). However, TCR-induced NF-κB activation is not defective in TRAF6-deficient T cells (74), suggesting that other E3 ligases can compensate the defect of TRAF6 in the TCR signaling pathway. It has been shown that NEMO is ubiquitinated at its Lys399 (Lys392 in mice) residue through a MALT1-dependent mechanism in lymphocytes stimulated with PMA/ionomycin, and a NEMO variant with K399R mutation was shown to interfere with NF-κB activation (67). Surprisingly, generation of mice harboring an ubiquitination-defective form of NEMO (with K392R point mutation) did not confirm the critical role of Lys392 in the antigen receptor signaling (75). Consistent with this study, our recent studies also show that NEMO-deficient Jurkat T cells reconstituted with NEMO-K399R mutant exhibit only partially reduced NF-κB activation in comparison with the cells reconstituted with wildtype NEMO (11). Therefore, it remains to be determined which E3 ligases are involved in TCR-induced polyubiquitination of the IKK complex and which residue of NEMO is ubiquitinated following TCR stimulation.

IKK phosphorylation following TCR and BCR stimulation

Activation of the IKK complex is dependent on phosphorylation. IKKβ is phosphorylated on two residues within the activation loop of its kinase domain, Ser177 and Ser181, and phosphorylation of these residues is required for NF-κB activation (76). Our recent findings that TCR stimulation can still induce IKKα/β phosphorylation in the absence of CARMA1 or Bcl10 suggest that the IKK complex receives a CARMA1/Bcl10-independent signal for phosphorylation of

The regulation of IKK activation by the CBM complex

It is well established that the formation of CBM complex is critical for antigen receptor-induced IKK activation. However, the molecular mechanism by which the CBM complex regulates IKK activity is not fully determined. Some studies suggest that the CBM complex brings the upstream kinase, which is responsible for IKKβ phosphorylation, into close proximity

IKKα/β (11). Interestingly, we have also found that two distinct signaling pathways mediate GPCR-induced NF-κB activation. In this case, we found that although GPCR-induced IKK kinase activity is completely defective in Carma3⁻/⁻ cells, the signal-induced IKK phosphorylation is intact (33). Thus, deficiency of CARMA1 in lymphocytes, as well as its homolog CARMA3 in non-hematopoietic cells, impairs IKK activation without affecting the signal-induced IKKβ phosphorylation. This finding proposes a novel model for activation of the IKK complex, in which the signal-induced phosphorylation and polyubiquitination of the IKK complex is regulated through two signaling events (Fig. 4) instead of a linear signaling cascade.

Early studies have demonstrated that phosphorylation of Ser residues in the activation loop of IKKα and IKKβ is required for their kinase activity (76, 77). TAK1 was originally found to phosphorylate and activate IKK in the interleukin-1 (IL-1) signaling pathway (78, 79). More recent genetic studies demonstrate that TAK1 controls IKK activation upon stimulation with IL-1, TNFα, Toll-like receptor (TLR) family members, and receptor activator of NF-κB (RANK) ligand (80, 81). Several lines of evidence support the role of TAK1 in the regulation of IKKα/β phosphorylation in the antigen receptor signaling. First, TAK1 is inducibly associated with IKK, and TCR-activated TAK1 can phosphorylate IKKβ (11). Second, a TAK1-specific inhibitor (5Z-7-oxozeaenol) or TAK1 knock-down can effectively block TCR-induced IKKα/β phosphorylation and NF-κB activation in Jurkat T cells (11, 68). Finally, BCR-induced IKKα/β phosphorylation is significantly impaired in chicken TAK1-deficient DT40 B cells (11, 52). However, recent studies, using mice with conditionally deleted TAK1 gene in T or B cells, lead to the conclusions that TAK1 is differentially required for TCR- and BCR-induced NF-κB activation (80, 82–84). Sato et al. (80) found that deletion of TAK1 in B cells affected JNK activation but did not affect BCR-induced NF-κB activation. By contrast, deletion of TAK1 in T cells prevented thymocyte development to mature T cells and impaired TCR-induced NF-κB activation in mature thymocytes (82–84). The reason for the differential requirement of TAK1 in BCR and TCR signaling is unclear but can be partially explained by inefficient deletion of floxed TAK1 by Cre recombinase in B cells. The residual TAK1 in these cells could account for the observed NF-κB activation (82, 85). However, the functional redundancy of TAK1 and other

![Diagram](attachment:diagram.png)

Fig. 4. Schematic model of T-cell receptor-induced nuclear factor-κB (NF-κB) activation. Stimulation of the antigen receptor initiates several proximal signaling events that lead to activation of protein kinase Cθ (PKCθ). The PKCθ phosphorylates CARMA1 [caspase-recruitment domain (CARD) membrane-associated guanylate kinase (MAGUK) protein 1], which enables CARMA1 to associate with the downstream signaling components, B-cell CLL-lymphoma 10 (Bcl10) and mucosa-associated lymphoid tissue lymphoma translocation gene 1 (MALT1) (the CBM complex). CBM cooperates with tumor necrosis factor receptor-associated factor 6 (TRAF6) to mediate ubiquitination of NF-κB essential modulator (NEMO). PKCθ also induces activation of TAK1 [transforming growth factor-β (TGFβ)-activated protein kinase], in a CARMA1-independent manner, which in turn phosphorylates IkB kinase (IKK) α/β. The combined phosphorylation of IKKα/β and ubiquitination of NEMO activate the IKK complex, leading to phosphorylation and degradation of IκBα, and subsequent activation of NF-κB.

kinases needs to be considered. It is possible that alternative TAK1-independent pathway exists in B cells, and can compensate the defective TAK1-dependent NF-κB activation. Thus, further studies are required to definitively establish the role of TAK1 in BCR-induced NF-κB activation.

### JNK activation in lymphocytes

JNK activation is another key signal event induced by TCR engagement (86, 87). The JNK family of kinases is encoded by three different genes: Jnk1, Jnk2, and Jnk3. JNK1 and JNK2 proteins are ubiquitously expressed, while the expression of JNK3 is restricted to the brain, heart, and testis (88). Interestingly, JNKs are expressed as 46 kDa (JNKp46) and 54 kDa (JNKp54) isoforms due to the alternative splicing on their last exons, which results in an extra 43-residue tail at the C-terminus of p54 isoforms of JNK1 and JNK2 (88, 89). The functional significance of these splicing variants remains to be determined.

It has been shown that activation of both JNK1 and JNK2 plays important roles in T-cell activation, differentiation, and proliferation (87, 90, 91). TCR/CD28 costimulation as well as treatment with phorbol esters plus ionophore effectively activates JNKs in T cells (86). Mice deficient in either JNK1 or JNK2 exhibit severe defects in T-cell mediated immune responses (90, 92). However, the molecular mechanism by which antigen receptor induces JNK activation is not fully defined.

The additional regulatory mechanisms also play a role in maintaining proper control of JNK-mediated signals that are necessary for T-cell activation and differentiation (93). For example, unlike in most of other cell types, JNK expression is induced upon T-cell activation (93). Although JNK1 and JNK2 share a high degree of sequence homology, they differ in their binding affinity to transcription factors (88). JNK2 preferentially binds to c-Jun (88, 89) and targets c-Jun for ubiquitination and degradation (94, 95), whereas JNK1 phosphorylates c-Jun and induces its transcriptional activity (96).

---

Our initial investigation on the effect of CARMA1 deficiency in JNK activation found that JNKs immunoprecipitated from CARMA1-deficient T cells effectively phosphorylated c-Jun. Thus, we concluded that CARMA1 does not have a significant impact in TCR-induced JNK activation (7). However, studies using Jurkat T cells expressing dominant-negative mutant of CARMA1 or lymphocytes from CARMA1-deficient mice showed that CARMA1 is involved in TCR-induced JNK activation (21, 23). The difference between our studies and other studies might be due to that we used JNK1-specific antibodies to immunoprecipitate JNK from CARMA1-deficient Jurkat T cells and then used the in vitro kinase assay to determine JNK kinase activity (7). Therefore, we might only have examined the JNK1 activity in the initial characterization of CARMA1-deficient Jurkat T cells. To address this discrepancy, we decided to use phospho-JNK antibodies to detect TCR-induced JNK activation in CARMA1-deficient Jurkat T cells. We surprisingly found that CARMA1 deficiency selectively impaired the signal-induced phosphorylation of p54, but not p46 isoform of JNKs (13) (Fig. 5A). As majority of JNK2 is expressed as p54 isoform whereas JNK1 is mainly expressed as p46 isoform in Jurkat T cells, we concluded that CARMA1 selectively regulates JNK2 activation (13). As the expression of JNK1-p54 in Jurkat T cells is very low, we cannot rule out the possibility that JNK1-p54 is also regulated by CARMA1-dependent pathway. Similarly, we found that JNK2 activation was also defective in the primary T and B cells from *Carm1*^−/−^ mice, whereas TCR-induced JNK1 activation was not.

![Figure 5](#fig-5)

**Fig. 5.** CARMA1 [caspase-recruitment domain (CARD) membrane-associated guanylate kinase (MAGUK) protein 1] -deficient JPM50.6 cells are defective in T-cell receptor (TCR)-induced Jun N-terminal kinase (JNK)2 phosphorylation and c-Jun accumulation. (A, B) Jurkat or JPM50.6 (CARMA1-) cells (6 × 10⁶/sample) were stimulated with or without anti-CD3 plus anti-CD28 antibodies (6 and 3 μg/ml, respectively) for various time points. Whole cell lysates were subjected to SDS-PAGE and analyzed by immunoblotting with antibodies against phospho-JNK, JNK, CARMA1, phospho-extracellular signal-regulated kinase (ERK), or ERK (A) and c-Jun or tubulin (B). This figure is adapted from reference (13) with permission.

significantly impaired in the absence of CARMA1 (13). Together, our studies provide the genetic and biochemical evidence that different isoforms of JNKs can be differentially regulated through distinct signaling cascades (13).

As described above, both JNK1 and JNK2 are expressed as 46 kDa and 54 kDa isoforms. These isoforms are different in their C-terminal tails (88, 89). The result that CARMA1 deficiency specifically affects the activation of p54, but not p46 isoform of JNKs led us to ask whether CARMA1 or its downstream components such as Bcl10 interacts with the C-terminal tail of p54 isoforms. Indeed, we found that Bcl10 selectively interacted with JNK2-p54, but not with JNK2-p46, through the C-terminal 43-residues. Moreover, CARMA1-induced Bcl10 oligomerization was required for Bcl10 to interact with JNK2-p54. Therefore, we proposed that the oligomerized Bcl10 functions as a scaffold for assembling JNK2 with its upstream kinases (13).

It is well established that JNK is activated by sequential protein phosphorylation through a mitogen-activated protein kinase (MAPK) module that includes a MAPK, a MAPK kinase (MAP2K), and a MAP2K kinase (MAP3K) (97). Although several MAP3Ks, including MEKK1, MEKK2, MEKK3, MLK, ASK, Tpl2, and TAK1, have been implicated to play a role in JNK activation (97, 98), recent studies indicate that TAK1 plays a critical and non-redundant role in antigen receptor-induced JNK activation (82, 84). Consistent with the role of TAK1 in antigen receptor-induced JNK activation, we found that Bcl10 inducibly associated with TAK1 (13).

The kinase activity of JNK is achieved through dual phosphorylation on threonine and tyrosine residues by MAP2K. Although both MAP kinase kinase 7 (MKK7) and MKK4 function as MAP2Ks for JNK (97), MKK7 is likely a kinase directly phosphorylating JNK in activated T cells (91). Indeed, following CD3/CD28 costimulation, JNK2 is complexed with MKK7, TAK1, and Bcl10 (13). Based on these data, we propose a model for CARMA1-mediated JNK2 activation, in which activated PKCs phosphorylate CARMA1. The phosphorylated CARMA1, in turn, induces Bcl10 oligomerization, which functions as a scaffold molecule to recruit TAK1 and MKK7 for JNK2 activation (Fig. 6).

CARMA1-dependent signaling controls the expression level of c-Jun

Stimulation of TCR and BCR leads to activation of several transcription factors, including the AP-1, which regulate cell proliferation and survival (99). AP-1 is a dimer containing members of Jun family (c-Jun, JunB, and JunD) with Fos or ATF family proteins (100). c-Jun is the best characterized AP-1 family member that positively regulates cells proliferation (101) and is frequently overexpressed in several cancers including lymphomas (102–104). Stimulation of the antigen receptor or mitogen treatment significantly induces c-Jun gene expression in normal PBLs and Jurkat T cells (105). Interestingly, c-Jun is further regulated at the transcription level by its own gene product through a positive feedback loop (106).

![Diagram](attachment:diagram.png)

Fig. 6. The model of T-cell receptor (TCR)-induced Jun N-terminal kinase (JNK) phosphorylation. TCR and CD28 ligation activates various proximal signaling components leading to activation of protein kinase Cθ (PKCθ) and CARMA1 [caspase-recruitment domain (CARD) membrane-associated guanylate kinase (MAGUK) protein 1], which, in turn, recruits B-cell CLL-lymphoma 10 (Bcl10) and induces its oligomerization. The oligomerized Bcl10 functions as a scaffold molecule for linking JNK2 and its upstream kinases TAK1 [transforming growth factor-β (TGFβ)-activated protein kinase], and MKK7. The activated JNK2 and independently activated JNK1 lead to accumulation of c-Jun and activation of AP-1, respectively.

Blonska & Lin · CARMA1 for NF-κB and JNK activation

Our recent study has revealed that c-Jun protein is rapidly accumulated in Jurkat T cells upon CD3/CD28 costimulation, but this accumulation is defective in the absence of CARMA1 (Fig. 5B). In addition, the basal level of c-Jun is significantly lower in CARMA1-deficient cells (13). How CARMA1 controls the c-Jun level is not clear. It has been shown that the steady state level of c-Jun is regulated by its ubiquitination and degradation (107). Under normal growth conditions, JNK2 is tightly bound to c-Jun targeting this protein for ubiquitination and degradation (94, 95, 107, 108). Upon stimulation, activated JNK2 dissociates from c-Jun allowing its phosphorylation by JNK1, which subsequently activates and stabilizes c-Jun (108–110). Therefore, JNK2 deficiency results in elevated c-Jun stability (108). By contrast, defective JNK2 activation may result in constitutive c-Jun ubiquitination and degradation in CARMA1-deficient cells. Our latest study confirms that c-Jun is highly ubiquitinated in the absence of CARMA1 (M. Blonska and X. Lin, unpublished data).

However, one study by Gao et al. (111) suggests that the activated JNK phosphorylates Itch, an E3 ligase, and enhances c-Jun ubiquitination and degradation following TCR stimulation. This finding is somehow contradicted to the current model that JNK-mediated phosphorylation of c-Jun decreases its ubiquitination and increases its stability (108–110). This discrepancy could be due to the differential effect of multiple E3 ligases that regulate c-Jun ubiquitination. Itch-mediated ubiquitination may represent a late effect on c-Jun stability following TCR stimulation. The constitutive degradation of c-Jun in unstimulated cells is controlled by another E3 ligase, Fbxw7, which is constitutively associated with c-Jun and induces c-Jun ubiquitination and degradation (112). Therefore, it will be interesting to examine whether CARMA1-dependent JNK2 activation affects the activity of all known c-Jun E3 ligases, such as Fbxw7 and Itch.

Although activation of JNK contributes to the activation of c-Jun and other AP-1 transcription factor members (113, 114), we do not observe AP-1 defect in CARMA1-deficient cells (13). Earlier studies also demonstrated that CARMA1 deficiency or deactivating mutation did not affect the AP-1 activation in Jurkat T cells (7, 22) and primary T cells from CARMA1 null mice (23). Similarly, lymphocytes from Bcl10 null mice have intact AP-1 activity following TCR and BCR stimulation (40). A possible explanation is that JNK1 is still activated in the absence of CARMA1 and Bcl10, which can phosphorylate c-Jun. In addition, other AP-1 components, like JunD, are not regulated by JNK2, and can be activated in CARMA1-deficient cells upon CD3/CD28 costimulation (M. Blonska and X. Lin, unpublished data), which may compensate the defect of c-Jun accumulation in CARMA1-deficient cells. Consistent with this possibility, another study shows that deletion of c-Jun from T and B cells do not decreases AP-1 activation in these cells (115), indicating that c-Jun is not an essential AP-1 component and can be replaced by other Jun family members. However, it will be important to investigate whether the expression of some AP-1-dependent genes is defective in CARMA1-deficient cells following the stimulation of antigen receptors.

### Concluding remarks

Significant progress towards understanding the function of CARMA1 in the NF-κB and JNK signaling pathways has been achieved during the past few years. However, the molecular mechanism of this action is only beginning to be elucidated and many questions remain. In particular, it is not clear how CARMA1 is recruited to the IS upon TCR ligation. This event is critical for assembly of the CBM complex and subsequent NF-κB activation. Another open issue is to understand the mechanism by which CBM activates the IKK complex. Although CARMA1-dependent NEMO polyubiquitination seems to be important for IKK activity, it is still possible that CBM mediates some additional modification of the IKK complex members.

It is important to note that CARMA1 upregulation leads to constitutively active NF-κB and aberrant NF-κB activity is observed in several cancers. Although the oncogenic function of CARMA1 has yet to be elucidated, several lines of evidence suggest that upregulated CARMA1 could play an important role in the development of leukemia/lymphoma. CARMA1 overexpression was found in adult T-cell leukemia (116), primary gastric B-cell lymphoma (117), and diffuse large B-cell lymphoma (DLBCL) (118). Interestingly, recent study has demonstrated that the activated B-cell-like (ABC) subtype of DLBCL is dependent on CARMA1 to survive, whereas CARMA1 knockdown in the germinal center B-cell (GBC) like DLBC does not affect cell viability (118). The latest study also demonstrates that CARMA1 is mutated in about 10% ABC-DLBCL patient samples (119). Thus, CARMA1 deregulation contributes to the development of lymphoma; however, there is no direct evidence that mutation of CARMA1 is sufficient to drive lymphoma. Further studies on transgenic mice expressing constitutively active CARMA1 in B or T cells are required to address this important question. Nevertheless, CARMA1 appears to be an attractive therapeutic target not only for CARMA1-dependent leukemia/lymphoma but also for diseases with deregulated lymphocyte proliferation.

References

1. Samelson LE. Signal transduction mediated by the T cell antigen receptor: the role of adapter proteins. Annu Rev Immunol 2002;20:371–394.
2. Clements JL, Boerth NJ, Lee JR, Koretzky GA. Integration of T cell receptor-dependent signaling pathways by adapter proteins. Annu Rev Immunol 1999;17:89–108.
3. Schulze-Luehrmann J, Ghosh S. Antigen-receptor signaling to nuclear factor kappa B. Immunity 2006;25:701–715.
4. Karin M, Lin A. NF-kappaB at the crossroads of life and death. Nat Immunol 2002;3: 221–227.
5. Karin M, Ben-Neriah Y. Phosphorylation meets ubiquitination: the control of NF-[kappa]B activity. Annu Rev Immunol 2000;18:621–663.
6. Hayden MS, Ghosh S. Shared principles in NF-kappaB signaling. Cell 2008;132: 344–362.
7. Wang D, et al. A requirement for CARMA1 in TCR-induced NF-kappa B activation. Nat Immunol 2002;3:830–835.
8. Wang D, et al. CD3/CD28 costimulation- induced NF-kappaB activation is mediated by recruitment of protein kinase C-theta, Bcl10, and IkappaB kinase beta to the immunological synapse through CARMA1. Mol Cell Biol 2004;24:164–171.
9. Che T, You Y, Wang D, Tanner MJ, Dixit VM, Lin X. MALT1/paracaspase is a signaling component downstream of CARMA1 and mediates T cell receptor-induced NF-kappaB activation. J Biol Chem 2004; 279:15870–15876.
10. Matsumoto R, et al. Phosphorylation of CARMA1 plays a critical role in T cell receptor-mediated NF-kappaB activation. Immunity 2005;23:575–585.
11. Shambharkar PB, et al. Phosphorylation and ubiquitination of the IkappaB kinase complex by two distinct signaling pathways. EMBO J 2007;26:1794–1805.
12. Tanner MJ, Hanel W, Gaffen SL, Lin X. CARMA1 coiled-coil domain is involved in the oligomerization and subcellular localization of CARMA1 and is required for T cell receptor-induced NF-kappaB activation. J Biol Chem 2007;282:17141–17147.
13. Blonska M, et al. The CARMA1-Bcl10 signaling complex selectively regulates JNK2 kinase in the T cell receptor-signaling pathway. Immunity 2007;26:55–66.
14. Dustin ML, et al. A novel adaptor protein orchestrates receptor patterning and cytoskeletal polarity in T-cell contacts. Cell 1998;94:667–677.
15. Xavier R, Seed B. Membrane compartmentation and the response to antigen. Curr Opin Immunol 1999;11:265–269.
16. Bromley SK, et al. The immunological synapse. Annu Rev Immunol 2001;19: 375–396.
17. Monks CR, Kupfer H, Tamir I, Barlow A, Kupfer A. Selective modulation of protein kinase C-theta during T-cell activation. Nature 1997;385:83–86.
18. Lin X, O’Mahony A, Mu Y, Geleziunas R, Greene WC. Protein kinase C-theta participates in NF-kappaB activation induced by CD3–CD28 costimulation through selective activation of IkappaB kinase beta. Mol Cell Biol 2000;20:2933–2940.
19. Coudronniere N, Villalba M, Englund N, Altman A. NF-kappa B activation induced by T cell receptor/CD28 costimulation is mediated by protein kinase C-theta. Proc Natl Acad Sci USA 2000;97:3394–3399.
20. Sun Z, et al. PKC-theta is required for TCR-induced NF-kappaB activation in mature but not immature T lymphocytes. Nature 2000;404:402–407.
21. Gaide O, et al. CARMA1 is a critical lipid raft-associated regulator of TCR-induced NF-kappa B activation. Nat Immunol 2002;3:836–843.
22. Pomerantz JL, Denny EM, Baltimore D. CARD11 mediates factor-specific activation of NF-kappaB by the T cell receptor complex. EMBO J 2002;21:5184–5194.
23. Hara H, et al. The MAGUK family protein CARD11 is essential for lymphocyte activation. Immunity 2003;18:763–775.
24. Egawa T, et al. Requirement for CARMA1 in antigen receptor-induced NF-kappa B activation and lymphocyte proliferation. Curr Biol 2003;13:1252–1258.
25. Newton K, Dixit V. Mice lacking the CARD of CARMA1 exhibit defective B lymphocyte development and impaired proliferation of their B and T lymphocytes. Curr Biol 2003;13:1247–1251.
26. Jun J, et al. Identifying the MAGUK protein Carma-1 as a central regulator of humoral immune responses and atopy by genome-wide mouse mutagenesis. Immunity 2003;18:751–762.
27. Pappu BP, Lin X. Potential role of CARMA1 in CD40-induced splenic B cell proliferation and marginal zone B cell maturation. Eur J Immunol 2006;36:3033–3043.
28. Bertin J, et al. CARD11 and CARD14 are novel caspase recruitment domain (CARD)/membrane-associated guanylate kinase (MAGUK) family members that interact with BCL10 and activate NF-kappa B. J Biol Chem 2001;276:11877–11882.
29. Gaide O, Martinon F, Micheau O, Bonnet D, Thome M, Tschopp J. Carmal, a CARD-containing binding partner of Bcl10, induces
30. Dimitratos SD, Woods DF, Stathakis DG, Bryant PJ. Signaling pathways are focused at specialized regions of the plasma membrane by scaffolding proteins of the MAGUK family. Bioessays 1999;21:912–921.
31. Wang L, et al. Card10 is a novel caspase recruitment domain/membrane-associated guanylate kinase family member that interacts with BCL10 and activates NF-kappa B. J Biol Chem 2001;276:21405–21409.
32. McAllister-Lucas LM, et al. Bimpl, a MAGUK family member linking protein kinase C activation to Bcl10-mediated NF-kappaB induction. J Biol Chem 2001;276:30589–30597.
33. Grabiner BC, et al. CARMA3 deficiency abrogates G protein-coupled receptor-induced NF-{kappa}B activation. Genes Dev 2007;21:984–996.
34. Hara H, et al. The molecular adapter Carma1 controls entry of IkappaB kinase into the central immune synapse. J Exp Med 2004;200:1167–1177.
35. Willis TG, et al. Bcl10 is involved in t(1;14)(p22;q32) of MALT B cell lymphoma and mutated in multiple tumor types. Cell 1999;96:35–45.
36. Zhang Q, et al. Inactivating mutations and overexpression of BCL10, a caspase recruitment domain-containing gene, in MALT lymphoma with t(1;14)(p22;q32). Nat Genet 1999;22:63–68.
37. Koseki T, et al. CIPER, a novel NF kappaB-activating protein containing a caspase recruitment domain with homology to Herpesvirus-2 protein E10. J Biol Chem 1999;274:9955–9961.
38. Thome M, et al. Equine herpesvirus-2 E10 gene product, but not its cellular homologue, activates NF-kappaB transcription factor and c-Jun N-terminal kinase. J Biol Chem 1999;274:9962–9968.
39. Yan M, Lee J, Schilbach S, Goddard A, Dixit V. mE10, a novel caspase recruitment domain-containing proapoptotic molecule. J Biol Chem 1999;274:10287–10292.
40. Ruland J, et al. Bcl10 is a positive regulator of antigen receptor-induced activation of NF-kappaB and neural tube closure. Cell 2001;104:33–42.
41. Dierlamm J, et al. The apoptosis inhibitor gene API2 and a novel 18q gene, MLT, are recurrently rearranged in the t(11;18)(q21;q21) associated with mucosa-associated lymphoid tissue lymphomas. Blood 1999;93:3601–3609.
42. Akagi T, et al. A novel gene, MALT1 at 18q21, is involved in t(11;18) (q21;q21) found in low-grade B-cell lymphoma of

mucosa-associated lymphoid tissue. Oncogene 1999;18:5785–5794.

43. Uren AG, et al. Identification of paracaspases and metacaspases: two ancient families of caspase-like proteins, one of which plays a key role in MALT lymphoma. Mol Cell 2000;6:961–967.

44. Ruefli-Brasse AA, French DM, Dixit VM. Regulation of NF-kappaB-dependent lymphocyte activation and development by paracaspase. Science 2003;302:1581–1584.

45. Ruland J, Duncan GS, Wakeham A, Mak TW. Differential requirement for Malt1 in T and B cell antigen receptor signaling. Immunity 2003;19:749–758.

46. Ferch U, et al. MALT1 directs B cell receptor-induced canonical nuclear factor-kappaB signaling selectively to the c-Rel subunit. Nat Immunol 2007;8:984–991.

47. Lee KY, D'Acquisto F, Hayden MS, Shim JH, Ghosh S. PDK1 nucleates T cell receptor-induced signaling complex for NF-kappaB activation. Science 2005;308:114–118.

48. Medeiros RB, et al. Regulation of NF-kappaB activation in T cells via association of the adapter proteins ADAP and CARMA1. Science 2007;316:754–758.

49. Su TT, et al. PKC-beta controls I kappa B kinase lipid raft recruitment and activation in response to BCR signaling. Nat Immunol 2002;3:780–786.

50. Saijo K, Mecklenbrauker I, Schmedt C, Tarakhovsky A. B cell immunity regulated by the protein kinase C family. Ann N Y Acad Sci 2003;987:125–134.

51. Sommer K, et al. Phosphorylation of the CARMA1 linker controls NF-kappaB activation. Immunity 2005;23:561–574.

52. Shinohara H, et al. PKC beta regulates BCR-mediated IKK activation by facilitating the interaction between TAK1 and CARMA1. J Exp Med 2005;202:1423–1431.

53. Shinohara H, Maeda S, Watarai H, Kurosaki T. IkappaB kinase beta-induced phosphorylation of CARMA1 contributes to CARMA1 Bcl10 MALT1 complex formation in B cells. J Exp Med 2007;204:3285–3293.

54. Narayan P, Holt B, Tosti R, Kane LP. CARMA1 is required for Akt-mediated NF-kappaB activation in T cells. Mol Cell Biol 2006;26:2327–2336.

55. Qiao G, et al. T-cell receptor-induced NF-kappaB activation is negatively regulated by E3 ubiquitin ligase Cbl-b. Mol Cell Biol 2008;28:2470–2480.

56. McCully RR, Pomerantz JL. The protein kinase C-responsive inhibitory domain of CARD11 functions in NF-kappaB activation to regulate the association of multiple signaling cofactors that differentially depend on Bcl10 and MALT1 for association. Mol Cell Biol 2008;28:5668–5686.

57. Ishiguro K, et al. Ca2+/calmodulin-dependent protein kinase II is a modulator of CARMA1-mediated NF-kappaB activation. Mol Cell Biol 2006;26:5497–5508.

58. Schaefer BC, Kappler JW, Kupfer A, Marrack P. Complex and dynamic redistribution of NF-kappaB signaling intermediates in response to T cell receptor stimulation. Proc Natl Acad Sci USA 2004;101:1004–1009.

59. Rossman JS, Stoicheva NG, Langel FD, Patterson GH, Lippincott-Schwartz J, Schaefer BC. POIKADOTS are foci of functional interactions in T-Cell receptor-mediated signaling to NF-kappaB. Mol Biol Cell 2006;17:2166–2176.

60. Wu CJ, Ashwell JD. NEMO recognition of ubiquitinated Bcl10 is required for T cell receptor-mediated NF-kappaB activation. Proc Natl Acad Sci USA 2008;105:3023–3028.

61. Ruefli-Brasse AA, Lee WP, Hurst S, Dixit VM. Rip2 participates in Bcl10 signaling and T-cell receptor-mediated NF-kappaB activation. J Biol Chem 2004;279:1570–1574.

62. Wegener E, et al. Essential role for IkappaB kinase beta in remodeling Carma1-Bcl10-Malt1 complexes upon T cell activation. Mol Cell 2006;23:13–23.

63. Ishiguro K, Ando T, Goto H, Xavier R. Bcl10 is phosphorylated on Ser138 by Ca2+/calmodulin-dependent protein kinase II. Mol Immunol 2007;44:2095–2100.

64. Zeng H, et al. Phosphorylation of Bcl10 negatively regulates T-cell receptor-mediated NF-kappaB activation. Mol Cell Biol 2007;27:5235–5245.

65. Lobry C, Lopez T, Israel A, Weil R. Negative feedback loop in T cell activation through IkappaB kinase-induced phosphorylation and degradation of Bcl10. Proc Natl Acad Sci USA 2007;104:908–913.

66. Scharschmidt E, Wegener E, Heissmeyer V, Rao A, Krappmann D. Degradation of Bcl10 induced by T-cell activation negatively regulates NF-kappa B signaling. Mol Cell Biol 2004;24:3860–3873.

67. Zhou H, et al. Bcl10 activates the NF-kappaB pathway through ubiquitination of NEMO. Nature 2004;427:167–171.

68. Sun L, Deng L, Ea CK, Xia ZP, Chen ZJ. The TRAF6 ubiquitin ligase and TAK1 kinase mediate IKK activation by BCL10 and MALT1 in T lymphocytes. Mol Cell 2004;14:289–301.

69. Oeckinghaus A, et al. Malt1 ubiquitination triggers NF-kappaB signaling upon T-cell activation. EMBO J 2007;26:4634–4645.

70. Rebeaud F, et al. The proteolytic activity of the paracaspase MALT1 is key in T cell activation. Nat Immunol 2008;9:272–281.

71. Coornaert B, et al. T cell antigen receptor stimulation induces MALT1 paracaspase-

mediated cleavage of the NF-kappaB inhibitor A20. Nat Immunol 2008;9:263–271.

72. Kawadler H, Gantz MA, Riley JL, Yang X. The paracaspase MALT1 controls caspase-8 activation during lymphocyte proliferation. Mol Cell 2008;31:415–421.

73. Su H, et al. Requirement for caspase-8 in NF-kappaB activation by antigen receptor. Science 2005;307:1465–1468.

74. King CG, et al. TRAF6 is a T cell-intrinsic negative regulator required for the maintenance of immune homeostasis. Nat Med 2006;12:1088–1092.

75. Ni CY, et al. Cutting edge: K63-linked polyubiquitination of NEMO modulates TLR signaling and inflammation in vivo. J Immunol 2008;180:7107–7111.

76. Delhase M, Hayakawa M, Chen Y, Karin M. Positive and negative regulation of IkappaB kinase activity through IKKbeta subunit phosphorylation. Science 1999;284:309–313.

77. Ling L, Cao Z, Goeddel DV. NF-kappaB-inducing kinase activates IKK-alpha by phosphorylation of Ser-176. Proc Natl Acad Sci USA 1998;95:3792–3797.

78. Ninomiya-Tsuji J, Kishimoto K, Hiyama A, Inoue J, Cao Z, Matsumoto K. The kinase TAK1 can activate the NIK-I kappaB as well as the MAP kinase cascade in the IL-1 signaling pathway. Nature 1999;398:252–256.

79. Wang C, Deng L, Hong M, Akkaraju GR, Inoue J, Chen ZJ. TAK1 is a ubiquitin-dependent kinase of MKK and IKK. Nature 2001;412:346–351.

80. Sato S, et al. Essential function for the kinase TAK1 in innate and adaptive immune responses. Nat Immunol 2005;6:1087–1095.

81. Shim JH, et al. TAK1, but not TAB1 or TAB2, plays an essential role in multiple signaling pathways in vivo. Genes Dev 2005;19:2668–2681.

82. Liu HH, Xie M, Schneider MD, Chen ZJ. Essential role of TAK1 in thymocyte development and activation. Proc Natl Acad Sci USA 2006;103:11677–11682.

83. Sato S, et al. TAK1 is indispensable for development of T cells and prevention of colitis by the generation of regulatory T cells. Int Immunol 2006;18:1405–1411.

84. Wan YY, Chi H, Xie M, Schneider MD, Flavell RA. The kinase TAK1 integrates antigen and cytokine receptor signaling for T cell development, survival and function. Nat Immunol 2006;7:851–858.

85. Chang L, et al. The E3 ubiquitin ligase itch couples JNK activation to TNFalpha-induced cell death by inducing c-FLIP(L) turnover. Cell 2006;124:601–613.

86. Su B, Jacinto E, Hibi M, Kallunki T, Karin M, Ben-Neriah Y. JNK is involved in signal

integration during costimulation of T lymphocytes. Cell 1994;77:727–736.

87. Dong C, Davis RJ, Flavell RA. MAP kinases in the immune response. Annu Rev Immunol 2002;20:55–72.

88. Gupta S, et al. Selective interaction of JNK protein kinase isoforms with transcription factors. EMBO J 1996;15:2760–2770.

89. Kallunki T, et al. JNK2 contains a specificity-determining region responsible for efficient c-Jun binding and phosphorylation. Genes Dev 1994;8:2996–3007.

90. Dong C, Yang DD, Wysk M, Whitmarsh AJ, Davis RJ, Flavell RA. Defective T cell differentiation in the absence of Jnk1. Science 1998;282:2092–2095.

91. Dong C, et al. JNK is required for effector T-cell function but not for T-cell activation. Nature 2000;405:91–94.

92. Yang DD, et al. Differentiation of CD4+ T cells to Th1 cells requires MAP kinase JNK2. Immunity 1998;9:575–585.

93. Weiss L, Whitmarsh AJ, Yang DD, Rincon M, Davis RJ, Flavell RA. Regulation of c-Jun NH(2)-terminal kinase (Jnk) gene expression during T cell activation. J Exp Med 2000;191:139–146.

94. Fuchs SY, Dolan L, Davis RJ, Ronai Z. Phosphorylation-dependent targeting of c-Jun ubiquitination by Jun N-kinase. Oncogene 1996;13:1531–1535.

95. Fuchs SY, Xie B, Adler V, Fried VA, Davis RJ, Ronai Z. c-Jun NH2-terminal kinases target the ubiquitination of their associated transcription factors. J Biol Chem 1997;272:32163–32168.

96. Derijard B, et al. JNK1: a protein kinase stimulated by UV light and Ha-Ras that binds and phosphorylates the c-Jun activation domain. Cell 1994;76:1025–1037.

97. Davis RJ. Signal transduction by the JNK group of MAP kinases. Cell 2000;103:239–252.

98. Cuevas BD, Abell AN, Johnson GL. Role of mitogen-activated protein kinase kinases in signal integration. Oncogene 2007;26:3159–3171.

99. Shaulian E, Karin M. AP-1 as a regulator of cell life and death. Nat Cell Biol 2002;4:E131–E136.

100. Eferl R, Wagner EF. AP-1: a double-edged sword in tumorigenesis. Nat Rev Cancer 2003;3:859–868.

101. Shaulian E, Karin M. AP-1 in cell proliferation and survival. Oncogene 2001;20:2390–2400.

102. Mathas S, et al. Aberrantly expressed c-Jun and JunB are a hallmark of Hodgkin lymphoma cells, stimulate proliferation and synergize with NF-kappa B. EMBO J 2002;21:4104–4113.

103. Troen G, et al. Constitutive expression of the AP-1 transcription factors c-jun, junD, junB, and c-fos and the marginal zone B-cell transcription factor Notch2 in splenic marginal zone lymphoma. J Mol Diagn 2004;6:297–307.

104. Leventaki V, et al. NPM-ALK oncogenic kinase promotes cell-cycle progression through activation of JNK/cJun signaling in anaplastic large-cell lymphoma. Blood 2007;110:1621–1630.

105. Chauhan D, et al. Regulation of c-jun gene expression in human T lymphocytes. Blood 1993;81:1540–1548.

106. Angel P, Hattori K, Smeal T, Karin M. The jun proto-oncogene is positively autoregulated by its product, Jun/AP-1. Cell 1988;55:875–885.

107. Laine A, Ronai Z. Ubiquitin chains in the ladder of MAPK signaling. Sci STKE 2005;2005:re5.

108. Sabapathy K, Hochedlinger K, Nam SY, Bauer A, Karin M, Wagner EF. Distinct roles for JNK1 and JNK2 in regulating JNK activity and c-Jun-dependent cell proliferation. Mol Cell 2004;15:713–725.

109. Musti AM, Treier M, Bohmann D. Reduced ubiquitin-dependent degradation of c-Jun after phosphorylation by MAP kinases. Science 1997;275:400–402.

110. Sabapathy K, Wagner EF. JNK2: a negative regulator of cellular proliferation. Cell Cycle 2004;3:1520–1523.

111. Gao M, et al. Jun turnover is controlled through JNK-dependent phosphorylation of the E3 ligase Itch. Science 2004;306:271–275.

112. Welcker M, Clurman BE. FBW7 ubiquitin ligase: a tumour suppressor at the crossroads of cell division, growth and differentiation. Nat Rev Cancer 2008;8:83–93.

113. Lamb JA, Ventura JJ, Hess P, Flavell RA, Davis RJ. JunD mediates survival signaling by the JNK signal transduction pathway. Mol Cell 2003;11:1479–1489.

114. Yazgan O, Pfarr CM. Regulation of two JunD isoforms by Jun N-terminal kinases. J Biol Chem 2002;277:29710–29718.

115. Chen J, Stewart V, Spyrou G, Hilberg F, Wagner EF, Alt FW. Generation of normal T and B lymphocytes by c-jun deficient embryonic stem cells. Immunity 1994;1:65–72.

116. Oshiro A, et al. Identification of subtype-specific genomic alterations in aggressive adult T-cell leukemia/lymphoma. Blood 2006;107:4500–4507.

117. Nakamura S, et al. Overexpression of caspase recruitment domain (CARD) membrane-associated guanylate kinase 1 (CARMA1) and CARD9 in primary gastric B-cell lymphoma. Cancer 2005;104:1885–1893.

118. Ngo VN, et al. A loss-of-function RNA interference screen for molecular targets in cancer. Nature 2006;441:106–110.

119. Lenz G, et al. Oncogenic CARD11 mutations in human diffuse large B cell lymphoma. Science 2008;319:1676–1679.
