
Available online at www.sciencedirect.com

**Regulation of the cytosolic DNA-sensing system in innate immunity: a current view**

Hideyuki Yanai, David Savitsky, Tomohiko Tamura and Tadatsugu Taniguchi

The sensing of nucleic acids by pattern recognition receptors is a central feature of innate immunity and is mediated by two types of receptors, Toll-like receptors and cytosolic receptors. Indeed, DNA, be it pathogen-derived or self-derived, can potently trigger the innate immune system; and much attention has been focused on the regulation of the DNA-sensing system(s) in the context of protective and pathological immune responses. An accumulating body of evidence has demonstrated that in addition to the membrane-bound type DNA-sensing receptor TLR9, there are cytosolic DNA receptors that can also evoke these responses. In particular, DAI (DLM-1/ZBP1), Trex1, and other regulators of the cytosolic DNA-sensing system have recently been identified and characterized. Here, we summarize our current understanding of how cytosolic DNA receptors contribute to the regulation of innate immune responses and discuss the complexity of the cytosolic DNA-sensing system as well as its future prospects.

**Address**

Department of Immunology, Graduate School of Medicine and Faculty of Medicine, the University of Tokyo, Hongo 7-3-1, Bunkyo-ku, Tokyo 113-0033, Japan

Corresponding author: Taniguchi, Tadatsugu ([tada@m.u-tokyo.ac.jp](mailto:tada@m.u-tokyo.ac.jp))

---

**Current Opinion in Immunology** 2009, 21:17–22

This review comes from a themed issue on  
Innate Immunity  
Edited by Siamon Gordon and Giorgio Trinchieri  

0952-7915/$ – see front matter  
© 2009 Elsevier Ltd. All rights reserved.  

DOI [10.1016/j.coi.2009.01.005](https://doi.org/10.1016/j.coi.2009.01.005)

---

**Introduction**

The detection of microbial components by pattern recognition receptors (PRRs) is one of the earliest defense mechanisms that triggers innate immune responses against infections [1,2]. Of the many classes of molecules detected by cells as pathogen-associated molecular patterns (PAMPs), nucleic acids are the most potent and broadly recognized [3,4*,5–9]. To sense nucleic acids, the immune system employs several classes of receptors, including RNA helicases that can respond to cytosolic double-stranded (ds)RNA (retinoic acid-inducible gene I (RIG-I)/melanoma differentiation associated gene 5 (MDA5) [9,10]), and Toll-like receptors (TLRs) that can recognize endosomal dsRNA (TLR3 [11]), single-stranded (ss)RNA (TLR7/8 [12–14]), and hypomethylated DNA (TLR9 [15]). As these proteins and their signaling pathways have been reviewed elsewhere, we primarily focus our attention on the system that is responsible for the detection of cytosolic DNA.

Since the discovery of TLR9, there has been a growing body of evidence that DNA derived from microbial and host cells can be recognized via a TLR9-independent mechanism. For example, infection of TLR9-deficient cells by DNA viruses, such as herpes simplex virus 1 (HSV1), or intracellular bacteria that have a cytoplasmic phase in their life cycle, such as *Listeria monocytogenes*, still evoke DNA-dependent type I IFN responses [7,16]. Moreover, in DNase II-deficient macrophages, which are defective in the clearance of DNA from engulfed apoptotic cells, accumulated DNA activates type I IFN gene expression in a TLR9-independent manner [17,18]. Finally, the transfection of cells with synthetic double-stranded DNA (dsDNA) activates TLR9-independent type I IFN responses and other innate immune responses [4*,7]. These observations indicate the presence of a non-TLR, cytosolic DNA sensor(s) that can initiate innate immune responses, especially the induction of type I IFN genes.

Recently, a DNA sensor and activator of innate immune responses has been identified and termed DNA-dependent activator of IFN-regulatory factors (DAI and also known as DLM-1 and ZBP1) [19**]. Subsequent studies have shown the presence of an additional, redundant mechanism(s) for DNA-sensing and activation of the innate immune system, as well as a mechanism of negative regulation of cytosolic DNA-mediated immune responses [20,21].

---

**Identification of DAI as a cytosolic DNA-sensor**

As described above, it was known that DNA from invading microbes or host cells exposed in the cytosol evoke immune responses. Moreover, type I IFN signaling, almost paradoxically, was known to be required to achieve a robust immune response to cytosolic DNA, including production of type I IFN [7,18,22**]. This raised the possibility that a putative, type I IFN-activating DNA sensor itself is IFN-inducible, a situation analogous to the RIG-I and MDA5 RNA sensors [9,10]. Consequently, DAI was discovered as a candidate cytosolic DNA sensor

from a cDNA microarray analysis carried out to screen for IFN-inducible genes that contain DNA binding domains. DAI mRNA expression is, indeed, type I IFN-inducible and, as was shown previously, its protein contains two DNA binding domains in the N-terminal region, termed Zα and Zβ (Figure 1) [23]. It has also been shown that DAI resides mainly in the cytosol and that its overexpression in L929 cells, a murine fibroblast cell line, enhanced the levels of IFN-α/β, IL-6, and CXCL10 in response to poly(dA-dT)·poly(dT-dA) (B-DNA) stimulation [4*,19**]. On the contrary, the knockdown of DAI protein in these cells by RNA interference (RNAi) inhibited the B-DNA-mediated activation of these genes. Furthermore, DAI knockdown in L929 cells challenged with pathogen-derived DNA, for example, from bacterial or viral sources, also resulted in the reduction of innate immune responses, suggesting that DAI broadly responds to DNA-mediated immune responses.

In characterizing DAI’s recognition of B-DNA and interferon stimulatory DNA (ISD) [7], a new DNA-binding region was identified and termed as the D3 region (Figure 1) [19**]. It was shown that mutants lacking either the D3, Zα, or Zβ region failed to evoke type I IFN response upon B-DNA stimulation, indicating that these three DNA binding regions all participate in the DNA-mediated immune responses [21]. Furthermore, it was shown that the artificially induced dimerization of DAI resulted in the DNA-independent activation of type I IFN genes [21]. Collectively, these observations further

---

**Figure 1**

![Diagram](attachment:diagram.png)

---

Cytosolic DNA-sensing system in innate immunity. DAI (DLM1/ZBP-1) interacts with double-stranded (ds)DNA such as B-DNA through its D3 region. Subsequently, a tighter DNA-DAI complex is formed via the activity of the Zα and Zβ domains of DAI. The formation of this complex results in the multimerization of DAI, which in turn recruits TBK1 and IRF3 and the induction of type I IFN genes. Receptor-interacting protein kinase 1 (RIP1) and RIP3 interact with DAI to activate the NF-κB pathway. The inflammasome may detect dsDNA to induce IL-1β secretion through the adaptor protein apoptosis-associated speck-like protein containing a CARD (ASC). RIG-I recognizes dsDNA in human cells. Stimulator of interferon genes (STING)/mediator of IRF3 activation (MITA) that localizes to the mitochondria or endoplasmic reticulum, functions as an adaptor that links IPS-1/MAVS/VISA/CARDIF to TBK1 and IRF3. 3′ repair exonuclease 1 (TREX1), adenosine deaminase acting on RNA 1 (ADAR1), and vaccinia virus encoded E3L protein negatively regulate cytosolic DNA-mediated immune responses.

underscore the notion that DAI indeed functions as a cytosolic DNA sensor and activator of innate immune responses.

# Signaling pathways downstream of DAI

Recent studies have revealed that the interferon regulatory factor (IRF) family of transcription factors functions as crucial regulators for the PRR-mediated induction of type I IFN and pro-inflammatory cytokine genes [24–28]. In the case of a cytosolic DNA-mediated gene regulation program, IFN-β and IFN-α4 mRNA induction was found to be strongly inhibited in IRF3-deficient mouse embryonic fibroblasts (MEFs) [7,19**]. Moreover, IFN-α4 mRNA expression was reduced in IRF7-deficient MEFs, although a normal induction of IFN-β mRNA was observed. These findings suggest that IRF3 is crucial to the expression of type I IFN genes in response to cytosolic DNA, whereas IRF7 partially cooperates with IRF3 in the induction of at least the IFN-α4 subtype. Interestingly, the relative contribution of IRF3 and IRF7 to DNA-mediated type I IFN gene expression depends on the signaling pathway that is activated. The cytosolic DNA-sensing pathway and the TLR9-mediated pathway, activated by extracellular DNA, requires, in the former, predominantly IRF3 and, in the latter, exclusively IRF7 [24,29].

IRF3 activation in response to B-DNA stimulation occurs via its co-recruitment with one of two serine/threonine kinases, TANK-binding kinase 1 (TBK1) and inducible IkB kinase (IKKe/i), to the C-terminal region of DAI [4*,19**,20]. This interaction is DNA-dependent as DAI mutants that lack DNA binding domains neither recruited TBK1 nor activated IRF3 [19**]. Interestingly, DAI mutants lacking either the Zα or Zβ domains still bound to B-DNA, emphasizing the predominant role the D3 region has in binding DNA [21]. Activation of DAI by B-DNA is accompanied by DAI oligomerization. This step appears to be crucial, since experimentally induced DAI dimer formation in the absence of DNA stimulation activated type I IFN genes [21]. From these and other data, a model was proposed in which the initial binding of DAI to DNA is mediated by the D3 region with subsequent stabilization provided by the Zα and Zβ domains (Figure 1). Because a DNA length dependence was observed in the activation of type I IFN, it is presumed that DAI forms a tandem array along the length of the DNA, which then provide scaffolding on to which TBK1 and IRF3 are recruited and activated.

There are data to suggest that cytosolic DNA-induced signaling is transmitted through the mitochondria. While the TBK1 adaptor protein TIR domain containing adaptor-inducing IFN (TRIF, also known as TICAM1) is dispensable to activation of the IFN-β promoter in response to B-DNA, IFN-β promoter stimulator-1 (IPS-1), also known as mitochondrial anti-viral signaling protein (MAVS) [30], virus-induced signaling adaptor (VISA) [31], or CARD adaptor inducing IFN-β (CARDIF) [32])-deficient MEFs were partially defective, indicating it is involved in the signaling pathway [4*,7,33,34]. Moreover, a new adaptor protein potentially involved in this pathway, termed stimulator of interferon genes (STING)/mediator of IRF3 activation (MITA) was recently reported [35**,36]. STING/MITA contains multiple transmembrane domains and forms a complex with IPS-1, TBK1 [35**,36], and IRF3 [36]. B-DNA-mediated and ISD-mediated IFN-β production was found to be markedly impaired in STING/MITA-deficient MEFs [35**]. These observations indicate that STING/MITA functions as an adaptor that links IPS-1 to TBK1 and the activation of IRF3 following cytosolic DNA-mediated signaling. However, the cellular localization of STING/MITA remains controversial. One group observed that STING/MITA localizes to the endoplasmic reticulum (ER) where it associates with the translocon-associated protein (TRAP) complex [35**]. STING/MITA is thought to then link to TBK1 via the interaction between the TRAP complex and another protein complex known as the exocyst, which interacts with TBK1 [37]. The second group observed that STING/MITA localizes to the mitochondrial outer membrane where it interacts with IPS-1 and TBK1 [30–32,38]. Further work is required to clarify the subcellular localization of STING/MITA and determine whether DAI signals through this pathway.

In addition to IRF3, knockdown experiments performed in L929 cells demonstrated that DAI is also crucially involved in the B-DNA-mediated activation of NF-κB [19**]. The precise signaling pathway by which DAI activates NF-κB is unclear, but there is evidence to suggest it involves several established players. For example, receptor-interacting protein kinase 1 (RIP1), a serine/threonine protein kinase family member related to IL-1 receptor-associated kinases, interacts with DAI through a RIP homotypic interaction motif (RHIM) [39]. RIP1 is essential for the activation of NF-κB downstream of several PRRs, such as TLR3, TLR4 and RIG-I/MDA5 [32,38,40,41]. Moreover, knockdown experiments have demonstrated that RIP1 is required for NF-κB activation by B-DNA stimulation [39]. DAI contains three domains with sequence elements that resemble RHIM, termed RHIM-like repeats RLR-A, RLR-B, and RLR-C, of which RLR-A was found to be crucial for the protein-protein interaction with RIP1 [39]. Substitution of conserved amino acids in the RLR-A motif, located within the D3 region of DAI, to alanine disrupted the interaction of DAI with RIP1 and abrogated the activation of an NF-κB reporter gene. DAI also interacts with RIP3, an RHIM-containing protein closely related to RIP1 that has been shown to potentiate DAI-mediated NF-κB reporter activation [39]. These data strongly suggest that DAI's activation of the NF-κB pathway requires its association with RIP1. Moreover, the activation of

NF-κB in response to B-DNA was shown to be STING/ MITA-dependent, but TBK1-independent and IKKε/i- independent [4*,35**,42]. Finally, it is interesting to note that unlike B-DNA, ISD stimulation does not cause detectable activation of either NF-κB or MAP kinases [7]. Therefore, it seems possible that these ligands are recognized by distinct receptors (see below).

**Operation of redundant mechanisms in cytosolic DNA-sensing system**

Is DAI the sole cytosolic DNA sensor? There are several lines of evidence that indicate the presence of a redundant cytosolic DNA-sensing receptor(s). For instance, upon DAI protein knockdown by RNAi, the induction of type I IFN by B-DNA was potently inhibited in L929 and a murine macrophage cell line, RAW264.7 cells, while such inhibition was very modest in MEFs [19**,21]. In addition, ISD induced expression of type I IFN genes strongly in MEFs, but only modestly in L929 cells [21]. Related to these findings, DNA-mediated activation of innate immune responses *in vitro* and *in vivo* was observed to be almost normally in mice deficient in the *Dai* gene [20] (HY, unpublished data). Although one study has shown that RIG-I, a cytosolic RNA sensor, can engage in B-DNA-mediated type I IFN production under certain conditions [33], RIG-I-deficient MEFs show normal responses to B-DNA stimulation [4*]. Thus, these observations indicate the presence of not only redundant proteins in the cytosolic DNA-triggered innate immune response but also cell type-specific and, possibly, species-specific mechanisms too.

It may be that not all cytoplasmic DNA sensors function redundantly. NACHT-leucine-rich repeat-PYD containing protein 3 (NALP3) also known as cryopyrin, and its adaptor protein apoptosis-associated speck-like protein containing a CARD (ASC), regulate secretion of interleukin (IL)-1β in response to a DNA virus (adenovirus) infection [43*,44]. Inflammasome activation also occurs upon cytosolic exposure of DNA, though in this case DNA-sensing was shown to be dependent on ASC and not NALP3 [43*]. It remains unclear, however, as to whether the NALP3/ASC inflammasome can directly detect cytosolic DNA or if the response is triggered by another protein.

**Negative regulation of cytosolic DNA- mediated innate immune responses**

“The master said, ‘To go beyond is the same as falling short.’” (The Confucian Analects). Excessive or uncontrolled activation of immune responses is self-destructive. Earlier reports have shown that the loss of functional DNase I results in the development of SLE-like symptoms in human and mouse [45,46]. More recently, the autoimmune disease polyarthritis has been shown to develop in mice carrying a null mutation in the *Dnase2a* gene [47], encoding an endonuclease essential for the degradation of

self-DNA derived from apoptotic cells in the lysosome of phagocytic macrophages [18]. Therefore, mechanisms that limit or negatively regulate the duration and/or magnitude of cytosolic DNA-induced immune activation are predicted to exist. Of particular interest is the recently identified 3′ repair exonuclease 1 (Trex1), the most abundant 3′ → 5′ DNA exonuclease in the cell [22**]. Trex1 was discovered owing to it being an ISD-inducible and ISD-binding protein. It was also shown that single-stranded DNA derived from endogenous retroelements can serve as a substrate for Trex1. Trex1-deficient mice develop lethal autoimmunity accompanied with a high production of type I IFN and autoantibodies [22**]. Interestingly, these symptoms in Trex1-deficient mice were suppressed by the genetic ablation of the IRF3 or type I IFN receptor genes. The importance of Trex1 in the negative regulation of DNA-induced immune activation is underscored further by the fact that mutations in human *TREX1* gene cause Aicardi-Goutieres syndrome and chilblain lupus; these are autoimmune diseases known to involve elevated type I IFN production [48]. Thus, Trex1 is required for the prevention of autoimmunity that would otherwise develop during the activation of the cytosolic DNA pathway by cell-intrinsic substrates.

Another negative regulator of the DNA-mediated innate immune response, adenosine deaminase acting on RNA 1 (ADAR1), has also been reported [21]. ADAR1 is an IFN inducible protein that possesses DNA-binding domains similar to those found in DAI [49]. ADAR1-deficient MEFs show a markedly elevated induction level of IFN-β upon B-DNA stimulation or HSV1 infection [21]. Presumably, ADAR1 inhibits cytosolic DNA-mediated immune responses by competing with cytosolic DNA sensors for their ligands. Finally, vaccinia DNA virus’ E3L protein perhaps highlights the importance of this system. The E3L protein harbors a DAI-like DNA binding domain, is known to block IFN responses [50], and inhibits B-DNA-mediated induction of IFN-β in MEFs [21]. The speculation is that vaccinia virus employs E3L to inhibit the cytosolic DNA-sensing system, thereby enabling it to evade the host’s anti-viral immune responses.

**Concluding remarks**

In recent years, the nature of the cytosolic DNA recognition system has gradually been clarified. It appears, however, that the system is more complex than was previously anticipated, wherein redundant and possibly cell type-specific and species-specific mechanisms operate. While the identification of additional cytosolic DNA sensors is clearly a key issue to address in the future, fully understanding the complex nature of the signaling networks that underlie DNA mediated innate immune responses is likely to be a complicated challenge. Yet, in deciphering these networks we may address a fundamental issue in immunology, which is how the immune

Regulation of the cytosolic DNA-sensing system in innate immunity Yanai et al. 21

viruses by Toll-like receptor 7. Proc Natl Acad Sci USA 2004, 101:5598-5603.

Acknowledgements
This work was supported by Grant-In-Aid for Scientific Research on Priority Area ‘Integrative Research Toward the Conquest of Cancer’, ‘Dynamics of extracellular environments’ and Global Center of Excellence Program ‘Integrative Life Science Based on the Study of Biosignaling Mechanism’ from the Ministry of Education, Culture, Sports, Science for the Promotion of Science.

References and recommended reading
Papers of particular interest, published within the period of review, have been highlighted as:

• of special interest
• of outstanding interest

1. Akira S, Uematsu S, Takeuchi O: Pathogen recognition and innate immunity. Cell 2006, 124:783-801.
2. Janeway CA Jr, Medzhitov R: Innate immune recognition. Annu Rev Immunol 2002, 20:197-216.
3. Isaacs A, Cox RA, Rotem Z: Foreign nucleic acids as the stimulus to make interferon. Lancet 1963, 2:113-116.
4. Ishii KJ, Coban C, Kato H, Takahashi K, Torii Y, Takeshita F, Ludwig H, Sutter G, Suzuki K, Hemmi H et al.: A Toll-like receptor-independent antiviral response induced by double-stranded B-form DNA. Nat Immunol 2006, 7:40-48.
   This report, together with Ref. [7], provided the evidence that cytosolic DNA activates TLR9-independent, TBK1-IRF3-mediated innate immune responses.
5. Kato H, Takeuchi O, Sato S, Yoneyama M, Yamamoto M, Matsui K, Uematsu S, Jung A, Kawai T, Ishii K et al.: Differential roles of MDA5 and RIG-I helicases in the recognition of RNA viruses. Nature 2006, 441:101-105.
6. Kawai T, Akira S: Pathogen recognition with Toll-like receptors. Curr Opin Immunol 2005, 17:338-344.
7. Stetson DB, Medzhitov R: Recognition of cytosolic DNA activates an IRF3-dependent innate immune response. Immunity 2006, 24:93-103.
8. Sumpter R Jr, Loo YM, Foy E, Li K, Yoneyama M, Fujita T, Lemon SM, Gale M Jr: Regulating intracellular antiviral defense and permissiveness to hepatitis C virus RNA replication through a cellular RNA helicase, RIG-I. J Virol 2005, 79:2689-2699.
9. Yoneyama M, Kikuchi M, Matsumoto K, Imazumi T, Miyagishi M, Taira K, Foy E, Loo YM, Gale M Jr, Akira S et al.: Shared and unique functions of the DExD/H-box helicases RIG-I, MDA5, and LGP2 in antiviral innate immunity. J Immunol 2005, 175:2851-2858.
10. Yoneyama M, Kikuchi M, Natsukawa T, Shinobu N, Imazumi T, Miyagishi M, Taira K, Akira S, Fujita T: The RNA helicase RIG-I has an essential function in double-stranded RNA-induced innate antiviral responses. Nat Immunol 2004, 5:730-737.
11. Alexopoulou L, Holt AC, Medzhitov R, Flavell RA: Recognition of double-stranded RNA and activation of NF-κB by Toll-like receptor 3. Nature 2001, 413:732-738.
12. Diebold SS, Kaisho T, Hemmi H, Akira S, Reis e Sousa C: Innate antiviral responses by means of TLR7-mediated recognition of single-stranded RNA. Science 2004, 303:1529-1531.
13. Heil F, Hemmi H, Hochrein H, Ampenberger F, Kirschning C, Akira S, Lipford G, Wagner H, Bauer S: Species-specific recognition of single-stranded RNA via toll-like receptor 7 and 8. Science 2004, 303:1526-1529.
14. Lund JM, Alexopoulou L, Sato A, Karow M, Adams NC, Gale NW, Iwasaki A, Flavell RA: Recognition of single-stranded RNA

15. Hemmi H, Takeuchi O, Kawai T, Kaisho T, Sato S, Sanjo H, Matsumoto M, Hoshino K, Wagner H, Takeda K et al.: A Toll-like receptor recognizes bacterial DNA. Nature 2000, 408:740-745.
16. Hochrein H, Schlatter B, O'Keeffe M, Wagner C, Schmitz F, Schiemann M, Bauer S, Suter M, Wagner H: Herpes simplex virus type-1 induces IFN-α production via Toll-like receptor 9-dependent and -independent pathways. Proc Natl Acad Sci USA 2004, 101:11416-11421.
17. Okabe Y, Kawane K, Akira S, Taniguchi T, Nagata S: Toll-like receptor-independent gene induction program activated by mammalian DNA escaped from apoptotic DNA degradation. J Exp Med 2005, 202:1333-1339.
18. Yoshida H, Okabe Y, Kawane K, Fukuyama H, Nagata S: Lethal anemia caused by interferon-beta produced in mouse embryos carrying undigested DNA. Nat Immunol 2005, 6:49-56.
19. Takaoka A, Wang Z, Choi MK, Yanai H, Negishi H, Ban T, Lu Y, Miyagishi M, Kodama T, Honda K et al.: DAI (DLM-1/ZBP1) is a cytosolic DNA sensor and an activator of innate immune response. Nature 2007, 448:501-505.
   This report demonstrated that DAI (DLM1/ZBP-1) is a cytosolic DNA sensor to activate innate immune responses.
20. Ishii KJ, Kawagoe T, Koyama S, Matsui K, Kumar H, Kawai T, Uematsu S, Takeuchi O, Takeshita F, Coban C et al.: TANK-binding kinase-1 delineates innate and adaptive immune responses to DNA vaccines. Nature 2008, 451:725-729.
21. Wang Z, Choi MK, Ban T, Yanai H, Negishi H, Lu Y, Tamura T, Takaoka A, Nishikura K, Taniguchi T: Regulation of innate immune responses by DAI (DLM-1/ZBP1) and other DNA-sensing molecules. Proc Natl Acad Sci USA 2008, 105:5477-5482.
22. Stetson DB, Ko JS, Heidmann T, Medzhitov R: Trex1 prevents cell-intrinsic initiation of autoimmunity. Cell 2008, 134:587-598.
   This report showed that Trex1 is an essential negative regulator of the cytosolic DNA pathway and prevents cell-intrinsic initiation of autoimmunity triggered by single-stranded DNA derived from endogenous retro-elements.
23. Fu Y, Comella N, Tognazzi K, Brown LF, Dvorak HF, Kocher O: Cloning of DLM-1, a novel gene that is up-regulated in activated macrophages, using RNA differential display. Gene 1999, 240:157-163.
24. Honda K, Taniguchi T: IRFs: master regulators of signalling by Toll-like receptors and cytosolic pattern-recognition receptors. Nat Rev Immunol 2006, 6:644-658.
25. Mamane Y, Heylbroeck C, Genin P, Algarte M, Servant MJ, LePage C, DeLuca C, Kwon H, Lin R, Hiscott J: Interferon regulatory factors: the next generation. Gene 1999, 237:1-14.
26. Paun A, Pitha PM: The IRF family, revisited. Biochimie 2007, 89:744-753.
27. Tamura T, Yanai H, Savitsky D, Taniguchi T: The IRF family transcription factors in immunity and oncogenesis. Annu Rev Immunol 2008, 26:535-584.
28. Taniguchi T, Ogasawara K, Takaoka A, Tanaka N: IRF family of transcription factors as regulators of host defense. Annu Rev Immunol 2001, 19:623-655.
29. Honda K, Takaoka A, Taniguchi T: Type I interferon gene induction by the interferon regulatory factor family of transcription factors. Immunity 2006, 25:349-360.
30. Seth RB, Sun L, Ea CK, Chen ZJ: Identification and characterization of MAVS, a mitochondrial antiviral signaling protein that activates NF-κB and IRF3. Cell 2005, 122:669-682.
31. Xu LG, Wang YY, Han KJ, Li LY, Zhai Z, Shu HB: VISA is an adapter protein required for virus-triggered IFN-β signaling. Mol Cell 2005, 19:727-740.
32. Meylan E, Curran J, Hofmann K, Moradpour D, Binder M, Bartenschlager R, Tschopp J: Cardif is an adaptor protein in the RIG-I antiviral pathway and is targeted by hepatitis C virus. Nature 2005, 437:1167-1172.

33. Cheng G, Zhong J, Chung J, Chisari FV: Double-stranded DNA and double-stranded RNA induce a common antiviral signaling pathway in human cells. *Proc Natl Acad Sci USA* 2007, **104**:9035-9040.

34. Kumar H, Kawai T, Kato H, Sato S, Takahashi K, Coban C, Yamamoto M, Uematsu S, Ishii KJ, Takeuchi O et al.: Essential role of IPS-1 in innate immune responses against RNA viruses. *J Exp Med* 2006, **203**:1795-1803.

35. Ishikawa H, Barber GN: STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling. *Nature* 2008, **455**:674-678.
This report, together with Ref. [36], showed that STING/MITA is essential for IPS-1-mediated innate immune responses.

36. Zhong B, Yang Y, Li S, Wang YY, Li Y, Diao F, Lei C, He X, Zhang L, Tien P et al.: The adaptor protein MITA links virus-sensing receptors to IRF3 transcription factor activation. *Immunity* 2008, **29**:538-550.

37. Chien Y, Kim S, Bumeister R, Loo YM, Kwon SW, Johnson CL, Balakireva MG, Romeo Y, Kopelovich L, Gale M Jr et al.: RalB GTPase-mediated activation of the IkB family kinase TBK1 couples innate immune signaling to tumor cell survival. *Cell* 2006, **127**:157-170.

38. Kawai T, Takahashi K, Sato S, Coban C, Kumar H, Kato H, Ishii KJ, Takeuchi O, Akira S: IPS-1, an adaptor triggering RIG-I- and Mda5-mediated type I interferon induction. *Nat Immunol* 2005, **6**:981-988.

39. Kaiser WJ, Upton JW, Mocarski ES: Receptor-interacting protein homotypic interaction motif-dependent control of NF-kB activation via the DNA-dependent activator of IFN regulatory factors. *J Immunol* 2008, **181**:6427-6434.

40. Balachandran S, Thomas E, Barber GN: A FADD-dependent innate immune mechanism in mammalian cells. *Nature* 2004, **432**:401-405.

41. Michallet MC, Meylan E, Ermolaeva MA, Vazquez J, Rebsamen M, Curran J, Poeck H, Bscheider M, Hartmann G, König M et al.: 

42. Ishii KJ, Akira S: Innate immune recognition of, and regulation by, DNA. *Trends Immunol* 2006, **27**:525-532.

43. Muruve DA, Petrilli V, Zaiss AK, White LR, Clark SA, Ross PJ, Parks RJ, Tschopp J: The inflammasome recognizes cytosolic microbial and host DNA and triggers an innate immune response. *Nature* 2008, **452**:103-107.
This report demonstrated that dsDNA introduced into the cytosol activates the inflammasome by triggering ASC-dependent caspase-1 activation.

44. Ye Z, Ting JP: NLR, the nucleotide-binding domain leucine-rich repeat containing gene family. *Curr Opin Immunol* 2008, **20**:3-9.

45. Napirei M, Karsunky H, Zevnik B, Stephan H, Mannherz HG, Moroy T: Features of systemic lupus erythematosus in Dnase1-deficient mice. *Nat Genet* 2000, **25**:177-181.

46. Yasutomo K, Horiuchi T, Kagami S, Tsukamoto H, Hashimura C, Urushihara M, Kuroda Y: Mutation of DNASE1 in people with systemic lupus erythematosus. *Nat Genet* 2001, **28**:313-314.

47. Kawane K, Ohtani M, Miwa K, Kizawa T, Kanbara Y, Yoshioka Y, Yoshikawa H, Nagata S: Chronic polyarthritis caused by mammalian DNA that escapes from degradation in macrophages. *Nature* 2006, **443**:998-1002.

48. Crow YJ, Hayward BE, Parmar R, Robins P, Leitch A, Ali M, Black DN, van Bokhoven H, Brunner HG, Hamel BC et al.: Mutations in the gene encoding the 3'-5' DNA exonuclease TREX1 cause Aicardi-Goutieres syndrome at the AGS1 locus. *Nat Genet* 2006, **38**:917-920.

49. Herbert A, Alfken J, Kim YG, Mian IS, Nishikura K, Rich A: AZ-DNA binding domain present in the human editing enzyme, double-stranded RNA adenosine deaminase. *Proc Natl Acad Sci USA* 1997, **94**:8421-8426.

50. Chang HW, Watson JC, Jacobs BL: The E3L gene of vaccinia virus encodes an inhibitor of the interferon-induced, double-stranded RNA-dependent protein kinase. *Proc Natl Acad Sci USA* 1992, **89**:4825-4829.
