
CHAPTER SIX

THE DEBS PARADIGM FOR TYPE I  
MODULAR POLYKETIDE SYNTHASES  
AND BEYOND  

Leonard Katz  

---

### Contents

1. Introduction  
   114  

2. DEBS and the Concept of a Module  
   114  

   2.1. Generalizability of the DEBS paradigm  
   118  

3. Beyond the DEBS Paradigm  
   119  

   3.1. Specificity of the AT domains  
   122  

   3.2. Novel loading modules  
   123  

   3.3. Methylation domains  
   126  

   3.4. Trans PKS activities  
   127  

   3.5. Unusual modular organization  
   130  

   3.6. Unusual module functions  
   133  

   3.7. Intermodular interactions  
   135  

4. Conclusion  
   136  

References  
136  

---

### Abstract

Polyketides are natural products that form the basis of numerous human and veterinary drugs. The biosynthesis of complex polyketides is carried out by polyketide synthases (PKSs), enzymes composed of multifunctional polypeptides that are assembled into large protein complexes. Nucleotide sequencing revealed that the PKS that produces the polyketide backbone of the antibiotic erythromycin, DEBS (for 6-deoxyerythronolide B synthase), contains a discrete domain for every enzymatic step of the corresponding biochemical pathway, that the domains are organized into modules each corresponding to a single extension (condensation and β-carbonyl processing) step in the biochemical pathway, that the organization of the domains is consistent from module to module, that faithful production of the polyketide 6-dEB requires that the domains are always used and never bypassed, that the PKS does not contain

Synthetic Biology Engineering Research Center, University of California, Berkeley Emeryville, California, USA  

*Methods in Enzymology*, Volume 459  
© 2009 Elsevier Inc.  
ISSN 0076-6879, DOI: 10.1016/S0076-6879(09)04606-0  
All rights reserved.

additional domains that are not used, and that the domains are organized in a linear array in the order of use in the biosynthesis of 6-dEB. Taken together, these properties are often referred to as the DEBS paradigm. In this chapter, the biosyntheses of numerous polyketides will be described to highlight the generalizability of the DEBS paradigm, but also to illustrate the range of deviations from the paradigm so far found in nature that contribute to product versatility.

### 1. INTRODUCTION

The *eryAI-III* genes encoding deoxyerythronolide B synthase (DEBS), the polyketide synthase (PKS) responsible for the biosynthesis of 6-deoxyerythronolide B (6-dEB), the polyketide backbone of the macrolide antibiotic erythromycin (*Fig. 6.1A*) in the producing organism *Saccharopolyspora erythraea*, were the first PKS genes to be sequenced in their entirety (*Caffrey et al.*, 1992; *Cortes et al.*, 1990; *Donadio and Katz*, 1992; *Donadio et al.*, 1991). The sequence revealed the organization and architecture that became the paradigm for type I PKSs—modularity—taken to mean that the structure of each of the building blocks that are assembled into the polyketide is overtly programmed by a specific set of functional domains sequestered in a large polypeptide, and that each module could specify its biochemical characteristics independently of the other modules contained within the PKS. In the 18 years since the original description of DEBS, numerous type I PKS systems have been sequenced and characterized, and many examples of overall architecture that differ from that described for DEBS have been uncovered. This chapter presents an overview of the organization and architecture of a variety of PKS systems, highlighting the differences found from the organization and architecture of DEBS. This treatment will not include every PKS system currently known, nor will it present an historical account of the differences observed. The goal is to demonstrate how numerous complicated structures of polyketides can be produced from a short list of enzymatic activities organized in a variety of ways.

### 2. DEBS AND THE CONCEPT OF A MODULE

A PKS module is defined as a set of activities (domains) responsible for incorporating a particular precursor at a designated point in the nascent acyl chain and for determining its structure in the final polyketide produced. A two-dimensional representation of the modular organization of DEBS, along with the growing structure of the corresponding acyl chain into 6-DEB, is shown in *Fig. 6.1A*. In addition to the loading module, which

Load
Module 1
Module 2
Module 3
Module 4
Module 5
Module 6
End

DEBS 1
DEBS 2
DEBS 3

AT ACP KS AT ACP KS AT ACP KR ER DH KR ACP KS AT ACP KS AT ACP TE

O
|
S—S
|
O
|
HO
|
CH₃

O
|
S—S
|
O
|
HO
|
HO
|
CH₃

O
|
S—S
|
O
|
HO
|
HO
|
HO
|
CH₃

O
|
S—S
|
O
|
HO
|
HO
|
HO
|
HO
|
CH₃

O
|
S—S
|
O
|
HO
|
HO
|
HO
|
HO
|
HO
|
CH₃

O
|
S—S
|
O
|
HO
|
HO
|
HO
|
HO
|
HO
|
HO
|
CH₃

(1) HO
     |
(2) O
     |
(3) OH
     |
(4) HO
     |
(5) O
     |
(6) OH

Erythromycin A

Figure 6.1 (Continued)

determines the starter acyl ester employed in the synthesis, DEBS has six extension modules that extend and cyclize the acyl chain into the 14-membered macrolactone, 6-dEB. The modules are numbered in the order in which they are used in nascent acyl chain synthesis. All DEBS extension modules have three common domains: KS, AT and ACP. The KS domain conducts the decarboxylative Claisen-like condensation between the resident growing chain attached via a thioester to a Cys residue in the active site and the extender unit. An additional potential function of the KS domain, although not apparent from the sequence, is known as gate-keeping. Studies on DEBS KS2 employing the feeding of diketide N-acetylcysteamine thioesters five carbons in length that mimic the structure of nascent chain predicted to be located on ACP1 at the end of the first condensation cycle showed that only the diketide with methyl and hydroxyl groups in the 2S,3R-configuration could be elongated beyond the diketide form (*Luo et al.*, 1996; *Weissman et al.*, 1998a). In later work performed at Kosan Biosciences, approximately half of the 154 hybrid PKSs constructed that were composed of the DEBS loading domain and two extender modules from a variety of naturally occurring PKSs and the DEBS TE domain produced detectable quantities of the triketide predicted by the construct (*Menzella et al.*, 2005). More than half of the PKSs that originally failed to produce polyketide made the predicted compound when the hybrid module KS2 domain was replaced with the KS domain in the module that was immediately downstream in the naturally occurring PKS of the ACP domain present in module 1 of the hybrid (*Chandran et al.*, 2006), lending credence to the argument that the KS domain plays a gate-keeping role in polyketide synthesis.

The AT domain determines which precursor is incorporated at the given extension step; in the case of 6-DEB, each of the six extension AT domains incorporates 2-[S]-methylmalonyl CoA (*Marsden et al.*, 1994). The ACP domain carries the extended chain *via* a thioester linkage to a phosphopantetheine group linked to a resident Ser residue and presents it to the reductive domains for subsequent reduction, dehydration, and so on.

---

**Figure 6.1** Organization of the 6-deoxyerythronolide B synthase (DEBS) and the structure of erythromycin A. Stepwise growth of the polyketide chain is shown attached to the ACP domain of each module after full elaboration by the modular functions. Modules are labeled Load 1 through 6 and are delineated by the *solid lines*. The modules contained within each protein are shown by the *block arrows*. The polyketide backbone of erythromycin A is shown in *black*; all other atoms shown in *gray* originate from post-polyketide modifications, represented by *dashed arrows*. The circled numbers represent the condensation number and the corresponding atoms introduced into the polyketide chain. The numbers are placed at the position corresponding to the β-carbon or α-side-chain introduced at each condensation. ACP, acyl carrier protein; AT, acyltransferase; DH, dehydratase; ER, enoylreductase; KR, β-ketoreductase; KS, β-acyl ACP synthase; TE, thioesterase.
Module 3 of DEBS carries functional KS, AT and ACP domains (as well as a mutated, nonfunctional KR domain); the unreduced $\beta$-keto group generated at the third incorporation cycle persists through the remainder of the synthesis and becomes the 9-ketone of 6-DEB. Modules 1, 2, 5, and 6 have an additional KR domain that reduces the $\beta$-carbonyl generated by condensation into the corresponding hydroxyls present at C13, C11, C5, and C3, respectively, in 6-DEB. (The C13 OH is the lactone oxygen.) Module 4 has a full set of reductive domains, KR, DH, and ER, to yield the fully reduced methylene center at C7 of 6-DEB. The linear domain organization KS-AT-(ER-DH)-KR-ACP is preserved in each of the modules. Each module is contained within a single polypeptide and each polypeptide contains two modules except DEBS1, which has the additional loading module, composed of an AT and ACP domain. The AT domain specifies the incorporation of propionyl CoA as the starting acyl unit (Weissman *et al.*, 1998b). The thioesterase TE domain is appropriately located at the C terminus of module 6, and acts after nascent chain synthesis is completed to release the polyketide from DEBS and cyclize it into the 14-membered macrolactone.

The stereochemistries of the methyl and hydroxyl groups in 6-dEB are programmed in DEBS. Attack of the KR domain on the $\beta$-ketone can take place from either side of the acyl chain, resulting in formation of the OH carrying the D or L configuration; the stereospecificity of the KR is determined by its structure, and knowledge of the sequence can allow prediction of the OH stereochemistry (Castonguay *et al.*, 2007; O'Hare *et al.*, 2006; Reid *et al.*, 2003). Less is understood about how the configuration of the methyl side chain at each of the even-numbered positions of the 6-dEB ring is determined. Because all AT domains use the common precursor 2-[S]-methylmalonyl CoA, the stereochemistry of the methyl groups at C12, C8, and C6 of 6-dEB indicates that epimerization had occurred at some point in the synthesis employing modules 1, 3, and 4, respectively. It is currently thought that methyl stereochemistry is programmed in the KS domain but both the precise segment involved in epimerization, and whether epimerization takes place before or after the condensation event are not yet known (Holzbaur *et al.*, 1999; Lau *et al.*, 1999; Weissman *et al.*, 1997).

In addition to the functional domains present within each DEBS module, the proteins contain docking domains at the N- and/or C-termini to enable interaction with the correct modules to ensure the fidelity of polyketide biosynthesis. The docking domains are composed of short $\alpha$-helical segments that form bundles when associated with their cognate partner module (Keatinge-Clay, 2008; Weissman, 2006); it is not yet possible to determine the partner relationships among docking domains by reading their sequences.

The protein products of the $eryAI$-$III$ genes are not capable of polyketide production without additional enzymatic activities. Fully functional DEBS
requires the activity of a phosphopantetheinyl transferase (PPTase) for activation of the ACP domains, enabling them to form thioester linkages with the extender precursors. Sequences corresponding to PPTase activity were not found in either the DEBS genes or within the adjacent erythromycin biosynthetic gene cluster. A discrete gene encoding the PPtase SePptII was recently identified from the genomic sequence of *S. erythraea* and shown to catalyze phosphopantetheinylation of one of the DEBS ACP domains (*Weissman et al.*, 2004). Interestingly, a second PPTase in *S. erythraea*, SePptI was identified within a PKS module, whose cognate polyketide has not yet been identified. SepPptI did not activate one of the DEBS ACPs tested.

Though not absolutely required for synthesis, efficient, high-level 6-dEB production in *S. erythraea* requires a short-chain thioesterase designated TEII, whose gene is located in the *ery* biosynthetic cluster. Deletion of TEII resulted in high-level incorporation of an acetate, rather than propionate, as the starter (*Hu et al.*, 2003). Substrate specificity of AT domains is discussed further below.

Recent structural work on components of DEBS (reviewed in (*Smith and Tsai*, 2007)) indicate that the PKS complex contains homodimers of each of the three DEBS proteins and that the homodimers themselves possess a head-to-head arrangement, with tight interactions within the middle of the modules (the segment between the AT and KR domains) and very little interaction between the KS domains, AT domains, and ACP domains.

## 2.1. Generalizability of the DEBS paradigm

The general paradigm for the structure of PKSs for polyketide biosynthesis that emerged from the understanding of the organization and architecture of DEBS has two fundamental components. First, each step in the biosynthesis of a complex polyketide is programmed by a discrete module which contains all the enzymatic activities required for the chemical events that take place (condensation, reduction, determination of the correct stereochemistry of the α- and β-side chains). Second, the fidelity of the synthesis is maintained by (1) the specificity of the AT domains in each module for the correct extender unit, (2) the faithful utilization of the domain activities in each module so that the required chemistries are never bypassed, and (3) the proper juxtaposition of the modules in the PKS complex so that the growing acyl chain is always passed to the correct module for the next step in the synthesis. Diversity of structures in the polyketide family can be achieved by (1) variation of the use of starter and extender units (programmed by AT domains), (2) variation in the degree of reduction from module to module, and (3) variation in the number of modules employed in the synthesis, all in keeping with the basic paradigm developed for DEBS.

The PKS sequences of the 16-membered macrolactones tylactone backbone of the macrolide antibiotic tylosin and platenolid (spiramycin,

carbomycin, etc.) were among the first to appear after publication of the DEBS sequence (DeHoff *et al.*, 1996; Kuhstoss *et al.*, 1996). The sequence for the 16-membered macrolide chalcomycin followed some time later (Ward *et al.*, 2004). As can be seen in Fig. 6.2, they consist of seven modules and contain AT domains for the specific incorporation of malonyl CoA, methylmalonyl CoA, or ethylmalonyl CoA, but the overall organization is remarkably similar to that of DEBS. Subsequent characterization of the PKSs for the biosynthesis of the 14-membered macrolactones oleandolid and narbonolide also revealed their similarity to DEBS, including the maintenance of the stereochemistry at equivalent positions of the macro- lactone rings (reviewed in Katz and Ashley, 2005). (The pik PKS also produces the 12-membered macrolactone 10-deoxymethynolide, the back- bone of methymycin as described in Section 3.7.2.) In general, colinearity between the arrangement of the modules at the level of DNA and their order of use in the biosynthesis of the corresponding polyketide was preserved in polyketides classified as macrolides, including the antibiotics shown in Fig. 6.1, as well as the antifungal compounds nystatin (18 modules), candi- cidin (18 modules), and oligomycin (16 modules) (reviewed in Ikeda and Omura, 2002). As PKS sequences from nonmacrolide polyketides from nonactinomycete hosts became available, however, differences from the DEBS organization began to appear. Nonetheless, most of the more than 40 PKS systems currently sequenced maintained a great deal of similarity to the DEBS paradigm and led to the question of whether knowledge of a PKS sequence would enable the prediction of the structure of the corresponding polyketide. As the number of sequences of AT domains increased, it was possible to establish a correspondence between AT domain sequence and the incorporation of a particular extender CoA (malonyl, methylmalonyl, ethylmalonyl, or methoxymalonyl) during synthesis (Haydock *et al.*, 1995). On the other hand, translating PKS sequence into polyketide structure would also depend either on the preservation of colinearity between the order of the modules at the DNA level and their use in polyketide synthesis, or the ability to determine the docking specificities of the modules from examination of their docking sequences. Many systems have been uncovered in which colinearity is not preserved, nor is there yet a full understanding of modular docking specificities.

---

### 3. BEYOND THE DEBS PARADIGM

The remainder of this chapter deals with the major differences from the modular organization seen in DEBS in PKS systems discovered in the past 17 years. One or two examples of systems will be described that highlight the following changes or differences from DEBS: additional

A
Load → Module 1 → Module 2 → Module 3 → Module 4 → Module 5 → Module 6 → End
KS⁰ AT ACP KS AT ACP KS AT ACP KS AT ACP DH KR ER KR KR TE

Oleandomycin

Oleandrolide

B
Load → Module 1 → Module 2 → Module 3 → Module 4 → Module 5 → Module 6 → End
KS⁰ AT ACP KS AT ACP KS AT ACP DH KR ER KR KR TE

10-Deoxymethynolide

Methymycin

Narbomycin

Pikromycin

Narbonolide

C
Load → Module 1 → Module 2 → Module 3 → Module 4 → Module 5 → Module 6 → Module 7 → End
KS⁰ AT ACP KS AT ACP KS AT ACP DH KR ER DH KR KR TE

Tylactone

Tylosin

Rosaramicin

D
Load → Module 1 → Module 2 → Module 3 → Module 4 → Module 5 → Module 6 → Module 7 → End
KS⁰ AT ACP KS AT ACP RS AT ACP KS AT ACP DH ER DH KR KR TE

Platenolide

Spiramycin

Carbomycin

Niddamycin

Platenomycin

Leucomycin

Midecamycin

etc.

E
Load → Module 1 → Module 2 → Module 3 → Module 4 → Module 5 → Module 6 → Module 7 → End
KS⁰ AT ACP KS AT ACP KS AT ACP DH ER DH KR KR TE

Chalcolactone

Chalcomycin

Chalcoside

Figure 6.2 Organization of PKSs of macrolide antibiotics and structure of the corresponding PKS macrolactone products A. The PKS protein components are shown as block arrows. Names of the antibiotics produced after post-polyketide modification are shown on the far right. The structure of chalcomycin is shown. KS⁰, KS domain with active site Cys replaced with Gln residue. All other abbreviations, naming, numbering, and shading conventions duplicate those in Fig. 6.1.

domains, loss of specificity of AT domains, absence of domains for required functions, novel domain function and domain organization, and novel modular function. Discussion is limited to the functions that directly act on the carbon backbone of the polyketide chain. Functions that modify hydroxyl, methyl, or ethyl groups incorporated during polyketide biosynthesis will not be discussed. Unless otherwise indicated, the structures of the compounds described in this chapter are shown in Fig. 6.3.

![Chemical Structure 1](image1.png)
Z,Z,E-premonensin A: R = 3 CH  
Z,Z,E-premonensin B: R = H  

![Chemical Structure 2](image2.png)
Epothilone C: R = H  
Epothilone D: R = CH₃  

![Chemical Structure 3](image3.png)  
Oleandrose  

AvermectinB1a: R =  
AvermectinB1b: R =  
Doramectin: R =  

![Chemical Structure 4](image4.png)  
Geldanamycin  

![Chemical Structure 5](image5.png)  
Rapamycin  

![Chemical Structure 6](image6.png)  
Rifamycin  

![Chemical Structure 7](image7.png)  
Phoslactomycin B  

![Chemical Structure 8](image8.png)  
Curacin A  

![Chemical Structure 9](image9.png)  
Myxovirescin A  

![Chemical Structure 10](image10.png)  
Rhizoxin D3  

![Chemical Structure 11](image11.png)  
Pederin  

![Chemical Structure 12](image12.png)  
Myxothiazole A  

![Chemical Structure 13](image13.png)  
Bryostatin 1  

![Chemical Structure 14](image14.png)  
Stigmatellin A  

![Chemical Structure 15](image15.png)  
Bacillaene  

![Chemical Structure 16](image16.png)  
Jamaicamide A  

![Chemical Structure 17](image17.png)  
Leinamycin  

Figure 6.3 Continued.

Disorazole A₁

Virginiamycin M

Mupirocin

Soraphen A

Myxalamid B

Ambruticin VS3

Borrelidin

Etnangien

Figure 6.3 Structures of polyketides. Number and shading conventions duplicate those in Fig.6.1.

### 3.1. Specificity of the AT domains

#### 3.1.1. Extension modules

Fidelity of the structure of erythromycin is due in large part to the high specificity of the DEBS AT domains for the substrate 2-methylmalonyl CoA. Naturally occurring desmethyl analogues from unmutated *S. erythraea* strains have not been reported and could be made only by substitution of a DEBS AT domain with a malonyl CoA-specifying AT domain (*Oliynyk et al.*, 1996; *Ruan et al.*, 1997). Similar high specificity of the AT domains was found in all other PKS systems with a few notable exceptions. A single set of PKS genes was shown to produce both *Z,Z,E*-premonensins A and B, which differ in the side-chains at C16: A, ethyl; and B, methyl (*Oliynyk et al.*, 2003). The AT domain of module 5, which resembles the methylmalonyl-specifying consensus AT sequences, can introduce either ethylmalonyl CoA or methylmalonyl CoA into the nascent polyketide chain. Similarly, the epo PKS genes were determined to produce both epothilones C and D, which differ by the single methyl group at C12; it was predicted, and subsequently confirmed, that the AT of module 4 could incorporate either malonyl CoA or methylmalonyl CoA (*Molnar et al.*, 2000; *Petkovic et al.*, 2008; *Tang et al.*, 2000). To date, this is the only published example of an AT domain that specifies incorporation of malonyl- or methylmalonyl-CoA.

3.1.2. Loading modules

The AT loading domains (AT<sub>L</sub>) have been found to have more relaxed substrate specificity than the AT domains of extender modules. The DEBS AT<sub>L</sub> domain displayed high specificity for propionyl CoA but could use acetyl CoA only when the producing host was depleted of propionyl CoA (*Kao et al.*, 1994). The loading AT<sub>L</sub> domain of the avermectin PKS from *Streptomyces avermitilis* normally can use either isobutyryl CoA or 2-methylbutyryl CoA as the starter unit, leading to the generation of a family of avermectins (*Ikeda et al.*, 1999) (Fig. 6.3). The ave PKS AT<sub>L</sub> domain can also utilize a variety of linear or cyclic fatty acids (presumably after the host converted them to their respective CoAs), including cyclohexanecarboxylate, which yields the commercial product doramectin (*Denoya et al.*, 1995). Replacement of the loading domain of DEBS with the loading domain from the avermectin (ave) PKS resulted in the generation of triketides (using a construct LD-Mod1-Mod2-TE) containing branched side-chains (*Marsden et al.*, 1998). Interestingly, exogenously fed branched or cyclic fatty acids at high levels could also be incorporated into the biosynthesis of erythromycin analogues by conversion to their respective CoAs and subsequent utilization as starter units by the DEBS AT loading domain (*Pacey et al.*, 1998), suggesting a somewhat relaxed specificity of the DEBS AT loading domain, as well as an absence of a gate-keeping role for KS1.

3.2. Novel loading modules

3.2.1. KS<sup>Q</sup> domain

Of the many sequenced type I PKS systems that use an AT domain to load the starter unit, most were found to contain a sequence immediately upstream of AT<sub>L</sub> that resembles a KS domain but in which the Cys residue at the active site is replaced by Gln (in streptomycete systems), or Ser or Tyr (in myxobacterial systems). The absence of the Cys residue renders the KS<sup>Q</sup> incapable of forming thioester linkages with acyl chains but they are still capable of the decarboxylation activity. Examination of the sequences of the adjacent AT<sub>L</sub> domains indicates that they load either malonyl or methylmalonyl CoA. After transfer to the adjacent ACP<sub>L</sub> domain, they are decarboxylated by the KS<sup>Q</sup> domain (and subsequently reduced) before movement to the KS domain of module 1 (*Bisang et al.*, 1999; *Witkowski et al.*, 1999). The ethyl side chain at C13 of the 14-membered macrolactones 6-DEB and narbonolide arise, therefore, by two distinct mechanisms. DEBS (Fig. 6.1) incorporates propionyl CoA directly (and transfers it to result in propionyl ACP<sub>L</sub>), whereas the pik PKS (Fig. 6.2) incorporates methylmalonyl CoA, then transfers it and decarboxylates it to propionyl ACP<sub>L</sub>. Because of the inherently tight specificities of extender domains in

general, in contrast to the somewhat relaxed specificities of AT<sub>L</sub> domains that use straight, branched, or cyclic fatty acids (e.g. DEBS, ave PKS), the KS<sup>Q</sup>-AT-ACP loading domain organization most likely ensures the faithful start of polyketide synthesis with a single starter unit.

### 3.2.2. AMP-ligase loading domain

A different mechanism of activating and loading starter units was first revealed in the sequence of the rapamycin (Fig. 6.3) biosynthesis gene cluster (*Schwecke et al.*, 1995) and later shown in a number of other PKS systems. In place of the (KS<sup>Q</sup>)-AT-ACP loading module, the loading modules of these systems contain a domain that activates a carboxylate through the formation of an AMP-adduct by a domain that resembles the activation (A) domains of nonribosomal peptide synthetases (NRPS), which ligate AMP with the carboxyl end of amino acids. The adenylylation loading domains have been given various names: they are referred to here as A domains. The canonical loading module consists of two domains, A and ACP, but individual loading modules may contain additional domains to modify the starter unit. In modules containing an A domain, the designation of the ACP domain is T, for thiolation, following the nomenclature used for NRPS systems, where PCP (peptidyl carrier protein) is used in place of ACP. In general, PKS systems containing the AMP-ligase–containing domain use cyclohexanecarboxylate units that are partially or fully oxidized (aromatic) and are usually modified with hydroxyl or amino side chains. The rifamycin (rif) and geldanamycin (gdm), PKS loading modules consist of A and T domains, and load 3-amino-5-hydroxybenzoate, as do the loading modules of the herbimycin, mitomycin (mtm), and ansamitocin PKSs (structures not shown) (*August et al.*, 1998; *Mao et al.*, 1999; *Rascher et al.*, 2003, 2005; *Yu et al.*, 2002).

Cyclohexanecarboxylate is produced via a multistep pathway in *Streptomyces collinus* and other organisms from shikimic acid (3,4,5-trihydroxycyclohexenecarboxylate) (reviewed in *Moore and Hertweck*, 2002). One of the intermediates in the pathway, 3,4-dihydroxycyclohexenecarboxylate, is used as the starter for the synthesis of rapamycin, ascomycin and FK506, but is reduced to 3,4-dihydroxycyclohexanecarboxyl-ACP by an ER domain present in the loading module (A-ER-T) of these PKS systems (*Lowden et al.*, 2001; *Schwecke et al.*, 1995; *Wu et al.*, 2000). Cyclohexanecarboxylate, cyclohexenecarboxylate, and cycloheptanecarboxylate, all chemically synthesized analogues of the natural starter, could also be incorporated into the synthesis of rapamycin analogues by exogenous feeding of the free acids to the producing culture, *Streptomyces hygroscopicus* (*Lowden et al.*, 2004). It is interesting to note that cyclohexanecarboxylate is a natural starter of phoslactomycin biosynthesis, which employs a PKS containing a starter AT<sub>L</sub>-ACP loading module. Cyclohexanecarboxylate is also the starter unit

for the synthesis of the polyketide ansatrienin A (structure not shown), but the sequence of the loading module of the corresponding PKS system has not been reported.

Though the AT<sub>L</sub> or A loading domains show relaxed specificities for substrates, it is likely that they exhibit preferences for the most commonly used starter unit, hence the general appearance of either a single compound, or a mixture in which one is in high excess, when the producing host is cultivated under standard laboratory conditions. In addition, pathways that determine the levels of the cognate starter units are generally well regulated and the levels of alternative starters are minimized in the host. It is generally only when the culture conditions or the genetic background of the host are changed, such that the most preferred substrate is no longer present or that the levels of alternative substrates are raised, that mixtures of compounds begin to appear.

### 3.2.3. GNAT Domain

The loading modules of the PKSs for the cyanobacterial polyketide curacin A ([Chang *et al.*, 2004]), the myxobacterial polyketide myxovirescin A ([Simunovic *et al.*, 2006]) and rhizoxin, a polyketide from a filamentous fungus ([Brendel *et al.*, 2007]), all contain the following organization: an N-terminal AR (adapter region), a linker region of variable length, a GNAT domain, and an ACP domain. GNAT represents a superfamily of *N*-acyltransferase enzymes that catalyze acyl transfer to a primary amine. The pederin PKS contains a similar GNAT<sub>L</sub>-ACP<sub>L</sub> loading module but lacks the AR domain ([Piel, 2002]). These three compounds appear to use acetyl CoA as the starter for polyketide biosynthesis. Extensive biochemical studies on the cloned and expressed curacin loading module revealed that the GNAT domain binds and decarboxylates malonyl CoA prior to its transfer to the ACP<sub>L</sub> domain. Both comparative biochemical studies between the GNAT<sub>L</sub>-ACP<sub>L</sub> proteins carrying or lacking the AR domain, and structural studies of the loading module, indicated that the AR domain aids transfer of the acyl group to the ACP by directing transfer of the pantotheine moiety of the ACP domain into the active site of the GNAT<sub>L</sub> domain ([Gu *et al.*, 2007]). Decarboxylation of the starter unit prior to thioesterification is in contrast to the strategy used by KS<sup>Q</sup>-containing modules, wherein the KS<sup>Q</sup> domain most likely decarboxylates the malonyl-ACP unit.

### 3.2.4. Leinamycin loading module

The first two steps in the biosynthesis of leinamycin, a hybrid nonribosomal peptide/polyketide, consists of condensation of the starter D-alanine with L-cysteine, followed by six successive condensations employing malonyl CoA as the extender unit, using six PKS modules ([Tang *et al.*, 2004a]). The gene product LnmQ consists of the A–PCP (peptidyl carrier protein) didomain. The peptide LnmP is a discrete PCP. Recent biochemical

analysis has demonstrated that LnmQ binds D-alanine and then transfers it to LnmP, from which it undergoes condensation with the cysteinyl residue attached to the PCP of the NRPS module in LmnI (Tang *et al.*, 2007). In the LNM system, LnmQ, the loading module is designated module 1 and the NRPS module, the first extender module, is designated module 2.

### 3.3. Methylation domains

#### 3.3.1. O-Methylation
O-methylations of hydroxyl groups in polyketide backbones usually take place after completion of acyl chain synthesis and are catalyzed by cytochrome P450 enzymes encoded by discrete genes that lie within the biosynthesis clusters. Notable exceptions occur in the biosynthesis of myxothiazol and stigmatellin A, polyketides produced by the myxobacterium *Stigmatella aurantiaca*. Myxothiazol contains two methoxy groups at positions corresponding to the 6th and 7th condensation cycles of biosynthesis and the corresponding modules of the stigmatellin PKS each contain the domain organization KS–AT–OM–KR–ACP (OM = O-methyltransferase), suggesting that the O-methylations take place during, rather than after, nascent chain synthesis (Silakowski *et al.*, 1999). Similarly, the modules implicated in the fourth and fifth cycles of stigmatellin A biosynthesis also contain OM domains (Gaitatzis *et al.*, 2002). An OM domain was also identified in the sixth module of the jamaicamide PKS cluster and corresponded to the activity required for the presence of the 25-O-methyl group on the compound (Edwards *et al.*, 2004).

#### 3.3.2. C-Methylation: α-carbon
Epothilone contains a geminal dimethyl group at C4. Sequencing of the epo PKS revealed a functional domain for C-methylation (which uses the methyl donor S-adenosylmethionine) in module 8 (Molnar *et al.*, 2000; Tang *et al.*, 2000). The AT8 sequence resembles the consensus methylmalonyl-specifying AT cluster, so the geminal dimethyl is produced by incorporation of methylmalonyl CoA, followed by single C-methylation of the α-carbon during the eighth condensation cycle. The CM domain is positioned between the AT and ACP domains (module 8 lacks reductive domains); hence the general KS-AT-…-ACP organization of DEBS is preserved. Geminal dimethyls are also present at C-8 and C-18 of bryostatin but, as described below, because malonyl CoA is incorporated in each cycle of bryostatin biosynthesis, the geminal dimethyl groups are produced by C-dimethylation of the α-carbons produced in the fourth and ninth condensation cycles, catalyzed by the corresponding CM domains present in modules 4 and 9 of the bryostatin PKS (Sudek *et al.*, 2007). Similarly, the geminal dimethyl group in pederin is introduced via dimethylation by the MT domain in module 2 (Piel, 2002). The methyl groups at C6 of

leinamycin, C13 of bacillaene, C9 of jamaicamide, and C15 and C15′ of disorazole originate through single C-methylations of the α-carbons produced during the sixth, seventh, second, and second condensation cycles, respectively, each catalyzed by an MT domain present in their corresponding modules, as shown in Fig. 6.3 (Butcher *et al.*, 2007; Carvalho *et al.*, 2005; Edwards *et al.*, 2004; Tang *et al.*, 2004a).

## 3.4. Trans PKS activities

Trans activities described here refer to discrete enzymes that are not covalently linked to PKS modules and act at specific points during synthesis of nascent acyl chains. These are distinguished from the “decorating” or modifying activities (O- or C-methylation, hydroxylation, glycosylation, etc.) that act after completion of polyketide chain biosynthesis.

### 3.4.1. AT

First described for the PKS systems involved in the synthesis of pederin (Piel, 2002) and leinamycin (Cheng *et al.*, 2003), a number of PKSs, including virginiamycin M (Pulsawat *et al.*, 2007a), bryostatin (Sudek *et al.*, 2007), myxovirescin A (Simunovic *et al.*, 2006), disorazole (Carvalho *et al.*, 2005), and rhizoxin (Brendel *et al.*, 2007), that incorporate malonyl CoA exclusively at all positions of the nascent acyl chain, contain modules that lack the corresponding AT domains but are otherwise organized normally (KS-(DH-ER-KR)(MT)-ACP). Discrete ORFs that encode putative acyltransferase activities have been identified adjacent to the PKS gene cluster in most of these AT-less PKS systems and, in the bryostatin system, trans-acyltransferase activity has been confirmed *in vivo* and *in vitro* (Lopanik *et al.*, 2008). Trans-acylation requires interaction between the ACP domain of the AT-less PKS module and the discrete acyltransferase, but the identities of the segments of the modules that undergo these interactions have not yet been determined.

### 3.4.2. KR

Chalcomycin (Fig. 6.2) contains a 2,3-trans double bond in its polyketide backbone but module 7 of the chm PKS was shown to lack both the required KR and DH domains. Cloning of the chm PKS from the chalcomycin producer *Streptomyces bikiniensis* into *Streptomyces fradiae* resulted in the production of the expected 3-keto 16-membered macrolide, chalomnolide (Fig. 6.2), indicating that 2,3-cis double-bond formation could either be bypassed during acyl chain synthesis, or was a postpolyketide tailoring event. A potential KR-determining ORF, designated *chmU*, was found to map about 4 kb downstream of the chm PKS but was not tested

for function. A DH-like domain was not uncovered in the chalcomycin biosynthesis cluster (Ward *et al.*, 2004).

### 3.4.3. DH

Phoslactomycin (PLM) contains 3 *cis*-double bonds: 14,15-, 12,13-, and the 2,3-bond in the 6-membered lactone. Modules 1 and 2 of the PLM PKS, which introduce the 14,15- and 12,13-atoms, respectively, contain the canonical KR-DH didomain (the KR domains of these modules are believed to produce the corresponding L-3-hydroxylacyl product required to yield the *cis*-double bond). On the other hand, module 7 of PLM, responsible for incorporation of the 2,3 atoms of the completed molecule, does not have a DH domain (Palaniappan *et al.*, 2003). Bioinformatic analysis of both the PLM and fostreicin biosynthetic gene clusters uncovered a pair of homologous ORFs (*plmT*2, ORF4, respectively) belonging to a family of NAD-dependent epimerases/dehydratases. Knockout of *plmT*2 resulted in the appearance of a PLM analogue containing a saturated lactone, indicating its involvement in the generation of the 2,3-*cis* double bond, likely as a post-PKS tailoring enzyme (Palaniappan *et al.*, 2008).

### 3.4.3. HCS cassette: β-carbon methylation

A number of polyketides, including virginiamycin, myxovirescin A, bafilomycin, curacin A, mupirocin, pederin, bryostatin, and jamaicamide, contain unmodified or modified methyl side chains at odd-numbered positions on their polyketide backbones, corresponding to the positions of β-carbon atoms generated after condensation. β-Methyl side chains cannot be generated through incorporation of methylmalonyl units during condensation cycles; rather, they are produced though a three-step pathway illustrated in Fig. 6.4. The first step involves condensation of acetyl-ACP with the unreduced β-carbon of the nascent polyketide chain (tethered to the corresponding ACP domain) resulting in production of

![Figure 6.4](attachment://image.png)

**Figure 6.4** Scheme for the introduction of β-methyl side chains. Only the ACP domains of the modules that hold the nascent polyketide chains are shown. HCS, hydroxymethylglutaryl CoA synthase; ECH, enoyl CoA hydratase.
the $\beta$-OH-$\beta$-carboxymethyl intermediate. This step resembles condensation of the nucleophile donor, acetyl CoA, and the electrophile acceptor, acetoacetyl CoA, to produce hydroxymethylglutaryl CoA in the mevalonate biosynthetic pathway (Theisen *et al.*, 2004), catalyzed by the enzyme hydroxymethylglutaryl CoA synthase; hence, the designation HCS for the enzymes that carry out this process in polyketide biosynthesis. In curacin A biosynthesis, HCS-mediated condensation takes place after the first condensation, acetoacetyl-ACP is the substrate for the HCS activity associated with the ORF designated CurD, and hydroxymethylglutaryl-ACP is produced. (In the other compounds listed above, $\beta$-carbonyl condensation takes place at later stages of polyketide synthesis.) Where a methyl side chain is the end-product of the reaction scheme, the $\beta$-OH-$\beta$-carboxymethyl intermediate then undergoes concerted dehydration catalyzed by an enoyl CoA-hydratase-like ORF designated ECH$_1$ and decarboxylation, catalyzed by ECH$_2$ to produce the $\beta$-methyl side and the corresponding adjacent double bond (Geders *et al.*, 2007). (The order of the two reactions has not been determined.) In the case of pederin, the double bond is displaced to become an exomethylene in the formation of the 6-membered cyclic hemiacetal ring (Piel, 2002). In all instances except for bryostatin, an HCS–ECH$_1$–ECH$_2$ cassette consisting of three distinct ORFs was identified in the corresponding PKS cluster and, in at least two cases, was found to map immediately downstream of the module that carried the acyl chain substrate for carboxymethylation (Butcher *et al.*, 2007; Pulsawat *et al.*, 2007b; Simunovic and Muller, 2007; Wu *et al.*, 2008). In jamaicamide biosynthesis, a yet to be identified halogenase acts at some stage of the process to produce the chloromethyl side chain (Edwards *et al.*, 2004). A single HCS activity, BryR, was identified in the bryostatin PKS cluster but ECH activities were not found, correlating with the presence of (methyl)carboxymethylene side-chains at C13 and C21 (Sudek *et al.*, 2007). Myxovirescin A contains modified $\beta$-methyl side-chains at C13 and C17. The methyl group at C17 is O-methylated, which probably arises via P450-mediated hydroxylation and O-methylation following HCS-directed $\beta$-methylation. The ethyl side-chain present at C13 was proposed to arise by employment of propionyl-ACP as the nucleophilic donor (Calderone *et al.*, 2007). Two discrete HCS proteins, TaC and TacF, were identified in the PKS cluster, but only a single set of ECH1 and ECH2 activities, TaX and TaY, were found (Simunovic *et al.*, 2006).

Biochemical confirmation of HCS, ECH$_1$ and ECH$_2$ activities *in vitro* was obtained by Calderone *et al.* for the respective proteins of the bacillaene system, employing acetyl-AcpK as the nucleophile donor and a cloned segment of PksL containing the tandem ACP domains (PksL') that was esterified on the panthetheine arm of the second ACP domain with an acetoacetyl moiety and the electrophile acceptor. Employinghigh resolution mass spectrometry for detection, they found that the HCS (PksG) catalyzed the formation of hydroxymethylglutaryl-PksL', and that the addition of the two ECH proteins, PksH and PksI, resulted in the formation of the predicted isopentenyl-PKSL' product ([Calderone *et al.*, 2006](https://doi.org/10.1074/jbc.M513189200)). Analogous *in vitro* work demonstrated HCS activities for the two proteins TaC and TaF and ECH activities for TaX and TaY ([Calderone *et al.*, 2007](https://doi.org/10.1074/jbc.M611368200)). In the three instances cited, the substrate acetoacetyl-ACP was used successfully as the acceptor in HCS-mediated carboxymethylation, although the authentic substrate in each of these cases is a much larger molecule. Furthermore, acetyl-ACP is also not thought to be the donor for C-16 ethylation in myxovirescin synthesis.

### 3.5. Unusual modular organization

#### 3.5.1. Multiple AT domains

Soraphen A contains a phenyl side-chain adjacent to the lactone oxygen, indicating that benzoic acid is used as the starter unit. Sequencing of the cluster revealed the unusual loading module–module 1 organization ACP<sub>L</sub>-KS1-AT1a-AT1b-KR1-ACP1 ([Ligon *et al.*, 2002](https://doi.org/10.1074/jbc.M111.287973); [Wilkinson *et al.*, 2001](https://doi.org/10.1074/jbc.M105818200)). A proposal advanced by Wilkinson *et al.* ([Wilkinson *et al.*, 2001](https://doi.org/10.1074/jbc.M105818200)) suggested the intermingling of domains of the loading module and module 1 where the starter unit, benzoyl CoA, binds to AT1a, and then is back-transferred to ACP<sub>L</sub>, followed by transfer to KS1, where it undergoes condensation with malonyl-ACP, which is transferred from AT1b to ACP1. Similar loading module–module 1 organizations (ACP<sub>L</sub>-KS1-AT<sub>L</sub>-AT1-DH-KR1-ACP1) were found in the myxathiiazole and myxalamid PKS clusters, where AT<sub>L</sub> was proposed to load the starter, isobutyryl CoA, and transfer it to ACP<sub>L</sub> ([Silakowski *et al.*, 1999, 2001](https://doi.org/10.1074/jbc.274.36.25581)). Tandem AT domains were also discovered in MmpC, a component of the mupirocin PKS, but a prediction of the role of MmpC in the biosynthesis of the polyketide could not be made ([Wu *et al.*, 2008](https://doi.org/10.1074/jbc.M708876200)).

#### 3.5.2. Multiple ACP domains

In the PKS clusters that produce bacillaene, mupirocin, virginiamycin M, and pederin, the modules that carry the nascent acyl chains that serve as acceptors for β-carboxymethylation contain two tandem ACP domains. The corresponding modules in the curacin A and jamaicamide clusters carry three tandem ACP domains. (The corresponding modules in the myxovirescin and bryostatin PKS systems were not identified.) The role of the tandem ACP domains in β-alkylation is not understood, nor has it been determined whether more than a single ACP domain is necessary for the three-step process to take place. Tandem ACP domains were also observed in module 8 of the ambruticin PKS cluster, and β-carboxymethylation was

proposed as an intermediate step, followed by a Favorski rearrangement to result in the loss of a carbon atom from the polyketide chain (Julien *et al.*, 2006). The canonical HCS-ECH<sub>1</sub>-ECH<sub>2</sub> β-methylation cassette was not detected in the cluster, however. Tandem ACP domains were observed in module 2 of the disorazole PKS but the second ACP domain was found not to contain the active-site serine residue and is, therefore, inactive (Carvalho *et al.*, 2005). Two ACP domains were also found in module 6 of the “AT-less” leinamycin PKS but were separated by the CM domain that is believed to introduce the α-methyl side chain at C-9. The corresponding module, module 6, has the domain organization KS-KR-ACP<sub>1</sub>-CM-ACP<sub>2</sub> (Cheng *et al.*, 2003). Site-directed mutagenesis indicated that either of the two ACP domains could independently support synthesis of leinamycin, but ACP2 was found to be fourfold more efficient at LmnG (AT)-catalyzed loading of malonate than ACP1 (Tang *et al.*, 2006).

### 3.5.3. “Broken” modules

Type I PKS modules typically reside within a single polypeptide chain, but departures from this paradigm have been observed. Seven examples are shown in Fig. 6.5, each illustrating separation between distinct modular domains. The seventh module of the myxalamid PKS is divided between MyxB<sub>1</sub>, which contains the KS and AT domains, and MyxB<sub>2</sub>, which carries the DH KR and ACP domains, as well as a nonfunctional ER domain designated ER° (Silakowski *et al.*, 2001). In the bacillaene PKS system, modules 4 and 8 are each split between two multimodular poly-peptides: module 4 between PksJ and PksL, and module 8 between PksL


![Diagram](attachment:diagram.png)


Figure 6.5 Organization of PKSs showing “broken” modules. Abbreviations, naming, numbering, and shading conventions duplicate those in Fig. 6.1.

and PksM (Butcher *et al.*, 2007). In each case, the separations are between the KS and adjacent DH domains. (The bacillaene PKS is “AT-less.”) In the disorazole PKS system, module 4 is split between the DH domain in DsxA and the KR domain in DsxB (Carvalho *et al.*, 2005). In the rhizoxin system, the 3rd PKS module is split between the KR and CM domains in proteins RsxB and RsxC, respectively (Brendel *et al.*, 2007), and in the ambruticin PKS system, the seventh module is divided between AmbE and AmbF, between the KR and ACP domains. In the etnangien PKS, which is composed of 20 modules distributed on six proteins, modules 3, 7, 10 and 14 are split between two proteins, and each split is located in a different interdomain region of the corresponding module (Menche *et al.*, 2008). The most unusual case is in the lankacidin (Fig. 6.6B) PKS. Module 2 appears to be divided into three proteins, LkcA, LkcB, and LkcC, although it is possible that the KS domain at the terminus of LkcC is part of module 2 (Tatsuno *et al.*, 2007). In each of these cases, it is not

![Figure 6.6](#)

Figure 6.6 Iterative use of modules (stuttering). (A) Stigmatellin PKS. (B) Lankacidin PKS. (C) A model for *cis*-double bond formation by the epothilone PKS. Details are described in the text. Abbreviations, naming, numbering, and shading conventions duplicate those in Fig.6.1.
understood how the split module is held together so that the normal intramodular interactions, (AT–ACP, KS–ACP, and DH-, ER-, KR-, and CM-ACP) take place.

# 3.6. Unusual module functions

## 3.6.1. “Skipping”

Skipping refers to the bypass of the utilization of a domain or an entire module in the production of a polyketide. For type I PKS systems, skipping is a rare event, but a small number of examples of this phenomenon have been reported. It is usually associated with the presence of additional functional domains or modules in a PKS for which there is no obvious need in the biosynthesis of the corresponding polyketide. Truncated derivatives of a number of polyketides, including the avermectin analogue nemadectin and epothilone, were observed in fermentation broths, but these molecules usually appeared at very low levels ([Carter et al., 1988](https://doi.org/10.1074/jbc.263.35.18598); [Hardt et al., 2001](https://doi.org/10.1074/jbc.M101428200)). The subsequent discoveries of their respective PKS systems indicated that the truncated compounds could be produced by bypass of one or other module during biosynthesis, but the low levels detected underscored the fidelity of module-to-module chain passage. In the etnangien PKS, in which the organization of the modules appears to be colinear with the growth of the nascent polyketide chain, module 14, consisting of a KS and an ACP domain, does not appear to be used in the synthesis of the acyl chain ([Menche et al., 2008](https://doi.org/10.1074/jbc.M802653200)). Interestingly, module 14 is split between the two domains between EtnG and EtnH, possibly allowing close physical contact between the two proteins to permit direct acyl chain transfer from ACP13 to KS15, thereby bypassing module 14 completely. In the disorazole PKS, a module consisting of KS and ACP domains—the PKS is AT-less—and contained entirely within but at the C-terminus of DszB, follows the seventh module and is not predicted to be used in the synthesis of the polyketide ([Carvalho et al., 2005](https://doi.org/10.1074/jbc.M502615200)). Similarly, two tandem KS–ACP modules appear in DszC, downstream from NRPS module 8. These two modules are not predicted to be used in the biosynthesis of disorazole, but the TE domain, which lies downstream from the ACP domain of the second KS–ACP module, is predicted to be used in the formation of the head-to-tail polyketide dimer. It appears, therefore, that three modules are skipped in disorazole synthesis. A similar KS–ACP didomain module, also predicted not to be used in the synthesis of mupirocin, was found between modules 4 and 5 of the mupirocin PKS. Disruption of this module resulted in the loss of mupirocin production, indicating a yet-to-be-determined role in mupirocin biosynthesis ([El-Sayed et al., 2003](https://doi.org/10.1074/jbc.M302876200)).

An artificial PKS composed of DEBS Load-DEBS Module 1-Rapamycin Module 2-DEBS Module 2-TE was found to produce the predicted tetraketide but, surprisingly, only as a minor component of amixture that was largely composed of triketides that were the same as those produced from a PKS composed of DEBS Load-Module 1-Module 2-TE (Rowe *et al.*, 2001). Subsequent mutagenesis work indicated that the rap ACP domain but not the rap KS domain was required for acyl chain skipping, indicating that the chain was transferred directly from DEBS ACP1 to rap ACP2 without forming the thioacyl intermediate on rap KS2, although it is possible that rap KS2 participated in the ACP1-to-ACP2 thiotransfer (Thomas *et al.*, 2002).

### 3.6.2. Stuttering

Stuttering is the term that is used to describe the iterative use of a module in polyketide chain growth and is usually implicated when the number of modules present in a PKS is fewer than the number of condensation events required for complete biosynthesis of the corresponding polyketide. It is usually possible to identify the iterative module by comparing the proposed segment of the completed polyketide to the structure of the segment predicted from the use of the module. Stigmatellin is built through ten condensation cycles from a PKS composed of nine modules. As shown in Fig. 6.6A, the last three condensations yield identical segments of the nascent chain that can be programmed by StiH or StiI, both of which are composed of KS-AT-ACP domains. Either StiH or StiI is used in two successive condensations in the growth of the polyketide chain (Gaitatzis *et al.*, 2002). (It is also formally possible that StiH is bypassed and that StiJ is used three times.) Borrelidin is produced from eight condensations by a PKS composed of six extension modules (Olano *et al.*, 2004). Module 5 (BorA5) of the borellidin PKS (KS –AT-DH-ER-ACP) is thought to be used in three successive chain extensions to introduce nine carbon atoms into the compound.

The most unusual example of iterative use of a PKS module takes place during the biosynthesis of lankacidin C, a 17-membered macrolide, produced from eight condensations using from a PS/PKS composed of five modules (Tatsuno *et al.*, 2007). The segment containing the domains KS-DH-CM-ACP1-ACP1-KS-AT, spread over four proteins, is thought to be used for the first five condensations, but employing a different set of reduction domains in each condensation cycle to yield to introduce ten carbon atoms into the polyketide backbone, reduced to varying extents (Fig. 6.6B).

The biochemical basis of stuttering is not yet understood. Stuttering requires passage of the nascent acyl chain from the ACP domain to the KS domain of the same module. The ACP domain is then loaded with a second extender unit which undergoes condensation with the acyl chain tethered to the KS domain. Where the module used iteratively is contained within a multidomain polypeptide, it is most likely that the nascent chain is passed from the ACP to the KS domain on the same protein (intramodular)
passage). On the other hand, if the module undergoing iterative use is present on a discrete protein, as in borrelidin or stigmatellin, it is possible that two or more proteins can form part of the PKS complex, thereby allowing forward, intermodular passage of the nascent acyl chain.

## 3.7. Intermodular interactions

### 3.7.1. cis-Double bond formation by the epo PKS

The products of the epo PKS, epothilones C and D, contain a *cis* double bond at C12–C13 (Fig. 6.3), which would be expected to be introduced during the 4th condensation cycle of nascent chain growth and controlled by module 4, but module 4 was found not to contain the required DH domain normally required for the dehydration reaction (*Molnar et al.*, 2000; *Tang et al.*, 2000). Module 4 does have the required KR domain to generate the proposed L-3-hydroxyacyl chain (which would subsequently undergo *syn* elimination to generate the *cis*-double bond). Removal of the DH5 function resulted in the production of a compound lacking the 12,13-*cis* double bond, indicating that the DH5 domain is required to for *cis*-double bond formation (*Tang et al.*, 2004b). The work reported left a number of still-to-be-resolved possibilities for the mechanism: (1) the *cis*-double bond is introduced during the fourth condensation cycle with the nascent chain attached to ACP4—the DH5 domain thus “reaches back” into module 4 to cause the 2,3-dehydration; (2) the chain is transferred from ACP4 to the ACP domain of module 5 without elongation (skipping), where it undergoes 2,3 dehydration by DH5 to generate the *cis*-double bond, then is transferred back to KS5 for the normal fifth condensation cycle (stuttering) that includes the generation of a *trans* double bond, which undergoes subsequent reduction (Fig. 6.6C); or (3) the chain is elongated normally in module 5, where it undergoes two DH5-mediated dehydrations, 2,3- and 4,5- to generate the ultimate 10,11-*trans* double bond and 12,13-*cis* double bond, respectively. In all of these mechanisms, DH5 is proposed to act at two points in polyketide chain growth, to mediate the formation of two subsequent double bonds.

### 3.7.2. Chain termination in the pik PKS

Sequencing of the macrolide biosynthetic cluster in *Streptomyces venezuelae* revealed that a single set of PKS-determining genes containing six modules was responsible for the synthesis of both the 12-membered macrolactone 10-deoxymethynolide and the 14-membered macrolactone narbonolide (*Xue et al.*, 1998). The former could be made by the first five modules, the latter required all six modules, but the lactonizations to produce both products required the TE domain that was positioned at the C-terminus of module 6. Early work on this system suggested that production of the 12-membered lactone involved thiotransfer of the nascent acyl from
ACP5 to ACP6 without chain growth (skipping), where it could undergo TE-mediated release from the PKS and lactonization, and that a functional KS6 domain was required for thiotransfer (Beck *et al.*, 2002, 2003). Recent work, however, has demonstrated that skipping from ACP5 to ACP6 does not take place but that the TE domain can interact with the nascent chain tethered to ACP5 to produce the 12-membered macrolactone or with the nascent chain attached to ACP6 to produce the 14-membered lactone. Hence, the TE domain can form interactions with ACP5 and ACP6. For the former, the full domain set of module 6 is required to allow proper docking between PikAIII and PikAIV to allow the TE domain to produce the 12-membered macrolactone (Kittendorf and Sherman, 2008).

### 4. CONCLUSION

The basic tenets of the DEBS paradigm, that each step in the biosynthesis of a complex polyketide is programmed by a discrete module which contains all the enzymatic activities required for the chemical events that take place, and that structural diversity can be achieved through variation in the functional modular domains and though the employment of different numbers of modules, has held up well for the more than 50 PKS systems that have been characterized. The most pronounced departure has been the uncovering of AT-less PKS modules together with the *trans* AT domains that specify the use of malonyl CoA as the common extender in all nascent chain extensions, but the presence of $\alpha$-C methylation domains in modules has enabled growing polyketides to acquire methyl side chains. Changes in the nature of the loading modules from the DEBS AT-ACP didomain have uncovered some loss of specificity of the use of starter units. $\beta$-Methyl side chains in polyketides introduced by the HCS–ECH$_{1}$–ECH$_{2}$ cassette is a departure from the DEBS paradigm, but can be viewed as *trans* modification to a nascent chain, similar to *trans* O- or $\alpha$-$C$-methylation. Modules that undergo skipping and stuttering also fall outside the DEBS paradigm, but these represent a small number of cases. In the main, the wide variety of new modules and new activities uncovered after the original description of DEBS has strengthened the belief that a wide variety of useful novel polyketides will be produced by directed or combinatorial module assembly.

## REFERENCES

August, P. R., Tang, L., Yoon, Y. J., Ning, S., Müller, R., Yu, T. W., Taylor, M., Hoffmann, D., Kim, C. G., Zhang, X., Hutchinson, C. R., and Floss, H. G. (1998). Biosynthesis of the ansamycin antibiotic rifamycin: Deductions from the molecular
analysis of the rif biosynthetic gene cluster of *Amycolatopsis mediterranei* S699. *Chem. Biol.* 5, 69–79.

Beck, B. J., Aldrich, C. C., Fecik, R. A., Reynolds, K. A., and Sherman, D. H. (2003). Iterative chain elongation by a pikromycin monomodular polyketide synthase. *J. Am. Chem. Soc.* 125, 4682–4683.

Beck, B. J., Yoon, Y. J., Reynolds, K. A., and Sherman, D. H. (2002). The hidden steps of domain skipping: Macrolactone ring size determination in the pikromycin modular polyketide synthase. *Chem. Biol.* 9, 575–583.

Bisang, C., Long, P. F., Cortés, J., Westcott, J., Crosby, J., Matharu, A. L., Cox, R. J., Simpson, T. J., Staunton, J., and Leadlay, P. F. (1999). A chain initiation factor common to both modular and aromatic polyketide synthases. *Nature* 401, 502–505.

Brendel, N., Partida-Martinez, L. P., Scherlach, K., and Hertweck, C. (2007). A cryptic PKS-NRPS gene locus in the plant commensal *Pseudomonas fluorescens* Pf-5 codes for the biosynthesis of an antimitotic rhizoxin complex. *Org. Biomol. Chem.* 5, 2211–2213.

Butcher, R. A., Schroeder, F. C., Fischbach, M. A., Straight, P. D., Kolter, R., Walsh, C. T., and Clardy, J. (2007). The identification of bacillaene, the product of the PksX mega-complex in *Bacillus subtilis*. *Proc. Natl. Acad. Sci. USA* 104, 1506–1509.

Caffrey, P., Bevitt, D. J., Staunton, J., and Leadlay, P. F. (1992). Identification of DEBS 1, DEBS 2 and DEBS 3, the multienzyme polypeptides of the erythromycin-producing polyketide synthase from *Saccharopolyspora erythraea*. *FEBS Lett.* 304, 225–228.

Calderone, C. T., Iwig, D. F., Dorrestein, P. C., Kelleher, N. L., and Walsh, C. T. (2007). Incorporation of nonmethyl branches by isoprenoid-like logic: Multiple beta-alkylation events in the biosynthesis of myxovirescin A1. *Chem. Biol.* 14, 835–846.

Calderone, C. T., Kowtoniuk, W. E., Kelleher, N. L., Walsh, C. T., and Dorrestein, P. C. (2006). Convergence of isoprene and polyketide biosynthetic machinery: Isoprenyl-S-carrier proteins in the pksX pathway of *Bacillus subtilis*. *Proc. Natl. Acad. Sci. USA* 103, 8977–8982.

Carter, G. T., Nietsche, J. A., Hertz, M. R., Williams, D. R., Siegel, M. M., Morton, G. O., James, J. C., and Borders, D. B. (1988). LL-F28249 antibiotic complex: A new family of antiparasitic macrocyclic lactones. Isolation, characterization and structures of LL-F28249 alpha, beta, gamma, lambda. *J. Antibiot. (Tokyo)* 41, 519–529.

Carvalho, R., Reid, R., Viswanathan, N., Gramajo, H., and Julien, B. (2005). The biosynthetic genes for disorazoles, potent cytotoxic compounds that disrupt microtubule formation. *Gene* 359, 91–98.

Castonguay, R., He, W., Chen, A. Y., Khosla, C., and Cane, D. E. (2007). Stereospecificity of ketoreductase domains of the 6-deoxyerythronolide B synthase. *J. Am. Chem. Soc.* 129, 13758–13769.

Chandran, S. S., Menzella, H. G., Carney, J. R., and Santi, D. V. (2006). Activating hybrid modular interfaces in synthetic polyketide synthases by cassette replacement of ketosynthase domains. *Chem. Biol.* 13, 469–474.

Chang, Z., Sitachitta, N., Rossi, J. V., Roberts, M. A., Flatt, P. M., Jia, J., Sherman, D. H., and Gerwick, W. H. (2004). Biosynthetic pathway and gene cluster analysis of curacin A, an antitubulin natural product from the tropical marine cyanobacterium *Lyngbya majuscula*. *J. Nat. Prod.* 67, 1356–1367.

Cheng, Y. Q., Tang, G. L., and Shen, B. (2003). Type I polyketide synthase requiring a discrete acyltransferase for polyketide biosynthesis. *Proc. Natl. Acad. Sci. USA* 100, 3149–3154.

Cortes, J., Haydock, S. F., Roberts, G. A., Bevitt, D. J., and Leadlay, P. F. (1990). An unusually large multifunctional polypeptide in the erythromycin-producing polyketide synthase of *Saccharopolyspora erythraea*. *Nature* 348, 176–178.

DeHoff, B. S., Sutton, K. L., and Rosteck, P. R. Jr. (1996). Sequence of *Streptomyces fradiae* tylactone synthase gene tylG. GenBank Accession No. U78289.

Denoya, C. D., Fedechko, R. W., Hafner, E. W., McArthur, H. A., Morgenstern, M. R., Skinner, D. D., Stutzman-Engwall, K., Wax, R. G., and Wernau, W. C. (1995). A second branched-chain alpha-keto acid dehydrogenase gene cluster (bkdFGH) from *Streptomyces avermitilis*: Its relationship to avermectin biosynthesis and the construction of a bkdF mutant suitable for the production of novel antiparasitic avermectins. *J. Bacteriol.* 177, 3504–3511.

Donadio, S., and Katz, L. (1992). Organization of the enzymatic domains in the multifunctional polyketide synthase involved in erythromycin formation in *Saccharopolyspora erythraea*. *Gene* 111, 51–60.

Donadio, S., Staver, M. J., McAlpine, J. B., Swanson, S. J., and Katz, L. (1991). Modular organization of genes required for complex polyketide biosynthesis. *Science* 252, 675–679.

Edwards, D. J., Marquez, B. L., Nogle, L. M., McPhail, K., Goeger, D. E., Roberts, M. A., and Gerwick, W. H. (2004). Structure and biosynthesis of the jamaicamides, new mixed polyketide-peptide neurotoxins from the marine cyanobacterium *Lyngbya majuscula*. *Chem. Biol.* 11, 817–833.

El-Sayed, A. K., Hothersall, J., Cooper, S. M., Stephens, E., Simpson, T. J., and Thomas, C. M. (2003). Characterization of the mupirocin biosynthesis gene cluster from *Pseudomonas fluorescens* NCIMB 10586. *Chem. Biol.* 10, 419–430.

Gaitatzis, N., Silakowski, B., Kunze, B., Nordsiek, G., Blöcker, H., Höfle, G., and Müller, R. (2002). The biosynthesis of the aromatic myxobacterial electron transport inhibitor stigmatellin is directed by a novel type of modular polyketide synthase. *J. Biol. Chem.* 277, 13082–13090.

Geders, T. W., Gu, L., Mowers, J. C., Liu, H., Gerwick, W. H., Häkansson, K., Sherman, D. H., and Smith, J. L. (2007). Crystal structure of the ECH2 catalytic domain of CurF from *Lyngbya majuscula*. Insights into a decarboxylase involved in polyketide chain beta-branching. *J. Biol. Chem.* 282, 35954–35963.

Gu, L., Geders, T. W., Wang, B., Gerwick, W. H., Häkansson, K., Smith, J. L., and Sherman, D. H. (2007). GNAT-like strategy for polyketide chain initiation. *Science* 318, 970–974.

Hardt, I. H., Steinmetz, H., Gerth, K., Sasse, F., Reichenbach, H., and Höfle, G. (2001). New natural epothilones from *Sorangium cellulosum*, strains So ce90/B2 and So ce90/D13: Isolation, structure elucidation, and SAR studies. *J. Nat. Prod.* 64, 847–856.

Haydock, S. F., Aparicio, J. F., Molnár, I., Schwecke, T., Khaw, L. E., König, A., Marsden, A. F., Galloway, I. S., Staunton, J., and Leadlay, P. F. (1995). Divergent sequence motifs correlated with the substrate specificity of (methyl)malonyl-CoA:acyl carrier protein transacylase domains in modular polyketide synthases. *FEBS Lett.* 374, 246–248.

Holzbaur, I. E., Ranganathan, A., Thomas, I. P., Kearney, D. J., Reather, J. A., Rudd, B. A., Staunton, J., and Leadlay, P. F. (1999). Molecular basis of Celmer’s rules: The role of two ketoreductase domains in the control of chirality by the erythromycin modular polyketide synthase. *Chem. Biol.* 6, 189–195.

Hu, Z., Pfeifer, B. A., Chao, E., Murli, S., Kealey, J., Carney, J. R., Ashley, G., Khosla, C., and Hutchinson, C. R. (2003). A specific role of the *Saccharopolyspora erythraea* thioesterase II gene in the function of modular polyketide synthases. *Microbiology* 149, 2213–2225.

Ikeda, H., Nonomiya, T., Usami, M., Ohta, T., and Omura, S. (1999). Organization of the biosynthetic gene cluster for the polyketide anthelmintic macrolide avermectin in *Streptomyces avermitilis*. *Proc. Natl. Acad. Sci. USA* 96, 9509–9514.

Ikeda, H., and Omura, S. (2002). Biosynthesis, regulation, and genetics of macrolide production. In “Macrolide Antibiotics: Chemistry, Biology, and Practice.” (S. Omura, ed.), pp. 285–326. Academic Press, New York.

Julien, B., Tian, Z. Q., Reid, R., and Reeves, C. D. (2006). Analysis of the ambruticin and jerangolid gene clusters of *Sorangium cellulosum* reveals unusual mechanisms of polyketide biosynthesis. *Chem. Biol.* **13**, 1277–1286.

Kao, C. M., Katz, L., and Khosla, C. (1994). Engineered biosynthesis of a triketide lactone from an incomplete modular polyketide synthase. *J. Am. Chem. Soc.* **116**, 11612–11613.

Katz, L., and Ashley, G. W. (2005). Translation and protein synthesis: Macrolides. *Chem. Rev.* **105**, 499–528.

Keatinge-Clay, A. (2008). Crystal structure of the erythromycin polyketide synthase dehydratase. *J. Mol. Biol.* **384**, 941–953.

Kittendorf, J. D., and Sherman, D. H. (2008). The methymycin/pikromycin pathway: A model for metabolic diversity in natural product biosynthesis. *Bioorg. Med. Chem.* (epub).

Kuhstoss, S., Huber, M., Turner, J. R., Paschal, J. W., and Rao, R. N. (1996). Production of a novel polyketide through the construction of a hybrid polyketide synthase. *Gene* **183**, 231–236.

Lau, J., Fu, H., Cane, D. E., and Khosla, C. (1999). Dissecting the role of acyltransferase domains of modular polyketide synthases in the choice and stereochemical fate of extender units. *Biochemistry* **38**, 1643–1651.

Ligon, J., Hill, S., Beck, J., Zirkle, R., Molnár, I., Zawodny, J., Money, S., and Schupp, T. (2002). Characterization of the biosynthetic gene cluster for the antifungal polyketide soraphen A from *Sorangium cellulosum* So ce26. *Gene* **285**, 257–267.

Lopanik, N. B., Shields, J. A., Buchholz, T. J., Rath, C. M., Hothersall, J., Haygood, M. G., Håkansson, K., Thomas, C. M., and Sherman, D. H. (2008). In vivo and in vitro transacylation by BryP, the putative bryostatin pathway acyltransferase derived from an uncultured marine symbiont. *Chem. Biol.* **15**, 1175–1186.

Lowden, P. A., Böhm, G. A., Metcalfe, S., Staunton, J., and Leadlay, P. F. (2004). New rapamycin derivatives by precursor-directed biosynthesis. *ChemBioChem* **5**, 535–538.

Lowden, P. A., Wilkinson, B., Böhm, G. A., Handa, S., Floss, H. G., Leadlay, P. F., and Staunton, J. (2001). Origin and true nature of the starter unit for the rapamycin polyketide synthase. *Angew. Chem. Int. Ed. Engl.* **40**, 777–779.

Luo, G., Pieper, R., Rosa, A., Khosla, C., and Cane, D. E. (1996). Erythromycin biosynthesis: Exploiting the catalytic versatility of the modular polyketide synthase. *Bioorg. Med. Chem.* **4**, 995–999.

Mao, Y., Varoglu, M., and Sherman, D. H. (1999). Molecular characterization and analysis of the biosynthetic gene cluster for the antitumor antibiotic mitomycin C from *Streptomyces lavendulae* NRRL 2564. *Chem. Biol.* **6**, 251–263.

Marsden, A. F., Caffrey, P., Aparicio, J. F., Loughran, M. S., Staunton, J., and Leadlay, P. F. (1994). Stereospecific acyl transfers on the erythromycin-producing polyketide synthase. *Science* **263**, 378–380.

Marsden, A. F., Wilkinson, B., Cortés, J., Dunster, N. J., Staunton, J., and Leadlay, P. F. (1998). Engineering broader specificity into an antibiotic-producing polyketide synthase. *Science* **279**, 199–202.

Menche, D., Arikan, F., Perlova, O., Horstmann, N., Ahlbrecht, W., Wenzel, S. C., Jansen, R., Irschik, H., and Müller, R. (2008). Stereochemical determination and complex biosynthetic assembly of etnangien, a highly potent RNA polymerase inhibitor from the myxobacterium *Sorangium cellulosum*. *J. Am. Chem. Soc.* **130**, 14234–14243.

Menzella, H. G., Reid, R., Carney, J. R., Chandran, S. S., Reisinger, S. J., Patel, K. G., Hopwood, D. A., and Santi, D. V. (2005). Combinatorial polyketide biosynthesis by *de novo* design and rearrangement of modular polyketide synthase genes. *Nat. Biotechnol.* **23**, 1171–1176.

Molnár, I., Schupp, T., Ono, M., Zirkle, R., Milnamow, M., Nowak-Thompson, B., Engel, N., Toupet, C., Stratmann, A., Cyr, D. D., Gorlach, J., Mayo, J. M., *et al.* (2000).
The biosynthetic gene cluster for the microtubule-stabilizing agents epothilones A and B from *Sorangium cellulosum* So ce90. *Chem. Biol.* **7**, 97–109.

Moore, B. S., and Hertweck, C. (2002). Biosynthesis and attachment of novel bacterial polyketide synthase starter units. *Nat. Prod. Rep.* **19**, 70–99.

O’Hare, H. M., Baerga-Ortiz, A., Popovic, B., Spencer, J. B., and Leadlay, P. F. (2006). High-throughput mutagenesis to evaluate models of stereochemical control in ketoreductase domains from the erythromycin polyketide synthase. *Chem. Biol.* **13**, 287–296.

Olano, C., Moss, S. J., Braña, A. F., Sheridan, R. M., Math, V., Weston, A. J., Méndez, C., Leadlay, P. F., Wilkinson, B., and Salas, J. A. (2004). Biosynthesis of the angiogenesis inhibitor borrelidin by *Streptomyces parvulus* Tu4055: Cluster analysis and assignment of functions. *Chem. Biol.* **11**, 87–97.

Oliynyk, M., Brown, M. J., Cortés, J., Staunton, J., and Leadlay, P. F. (1996). A hybrid modular polyketide synthase obtained by domain swapping. *Chem. Biol.* **3**, 833–889.

Oliynyk, M., Stark, C. B., Bhatt, A., Jones, M. A., Hughes-Thomas, Z. A., Wilkinson, C., Oliynyk, Z., Demydchuk, Y., Staunton, J., and Leadlay, P. F. (2003). Analysis of the biosynthetic gene cluster for the polyether antibiotic monensin in *Streptomyces cinnamonensis* and evidence for the role of monB and monC genes in oxidative cyclization. *Mol. Microbiol.* **49**, 1179–1190.

Pacey, M. S., Dirlam, J. P., Geldart, R. W., Leadlay, P. F., McArthur, H. A., McCormick, E. L., Monday, R. A., O’Connell, T. N., Staunton, J., and Winchester, T. J. (1998). Novel erythromycins from a recombinant *Saccharopolyspora erythraea* strain NRRL 2338 pIG1. I. Fermentation, isolation and biological activity. *J. Antibiot. (Tokyo)* **51**, 1029–1034.

Palaniappan, N., Alhamadsheh, M. M., and Reynolds, K. A. (2008). cis-Delta(2,3)–double bond of phoslactomycins is generated by a post-PKS tailoring enzyme. *J. Am. Chem. Soc.* **130**, 12236–12237.

Palaniappan, N., Kim, B. S., Sekiyama, Y., Osada, H., and Reynolds, K. A. (2003). Enhancement and selective production of phoslactomycin B, a protein phosphatase IIa inhibitor, through identification and engineering of the corresponding biosynthetic gene cluster. *J. Biol. Chem.* **278**, 35552–35557.

Petković, H., Sandmann, A., Challis, I. R., Hecht, H. J., Silakowski, B., Low, L., Beeston, N., Kuscer, E., Garcia-Bernardo, J., Leadlay, P. F., Kendrew, S. G., Wilkinson, B., *et al.* (2008). Substrate specificity of the acyl transferase domains of EpoC from the epothilone polyketide synthase. *Org. Biomol. Chem.* **6**, 500–506.

Piel, J. (2002). A polyketide synthase-peptide synthetase gene cluster from an uncultured bacterial symbiont of Paederus beetles. *Proc. Natl. Acad. Sci. USA* **99**, 14002–14007.

Pulsawat, N., Kitani, S., Kinoshita, H., Lee, C. K., and Nihira, T. (2007a). Identification of the bkdAB gene cluster, a plausible source of the starter-unit for virginiamycin M production in *Streptomyces virginiae*. *Arch. Microbiol.* **187**, 459–466.

Pulsawat, N., Kitani, S., and Nihira, T. (2007b). Characterization of biosynthetic gene cluster for the production of virginiamycin M, a streptogramin type A antibiotic, in *Streptomyces virginiae*. *Gene* **393**, 31–42.

Rascher, A., Hu, Z., Buchanan, G. O., Reid, R., and Hutchinson, C. R. (2005). Insights into the biosynthesis of the benzoquinone ansamycins geldanamycin and herbimycin, obtained by gene sequencing and disruption. *Appl. Environ. Microbiol.* **71**, 4862–4871.

Rascher, A., Hu, Z., Viswanathan, N., Schirmer, A., Reid, R., Nierman, W. C., Lewis, M., and Hutchinson, C. R. (2003). Cloning and characterization of a gene cluster for geldanamycin production in *Streptomyces hygroscopicus* NRRL 3602. *FEMS Microbiol. Lett.* **218**, 223–230.

Reid, R., Piagentini, M., Rodriguez, E., Ashley, G., Viswanathan, N., Carney, J., Santi, D. V., Hutchinson, C. R., and McDaniel, R. (2003). A model of structure and
catalysis for ketoreductase domains in modular polyketide synthases. *Biochemistry* **42**, 72–79.

Rowe, C. J., Böhm, I. U., Thomas, I. P., Wilkinson, B., Rudd, B. A., Foster, G., Blackaby, A. P., Sidebottom, P. J., Roddis, Y., Buss, A. D., Staunton, J., and Leadlay, P. F. (2001). Engineering a polyketide with a longer chain by insertion of an extra module into the erythromycin-producing polyketide synthase. *Chem. Biol.* **8**, 475–485.

Ruan, X., Pereda, A., Stassi, D. L., Zeidner, D., Summers, R. G., Jackson, M., Shivakumar, A., Kakavas, S., Staver, M. J., Donadio, S., and Katz, L. (1997). Acyltransferase domain substitutions in erythromycin polyketide synthase yield novel erythromycin derivatives. *J. Bacteriol.* **179**, 6416–6425.

Schwecke, T., Aparicio, J. F., Molnár, I., König, A., Khaw, L. E., Haydock, S. F., Oliynyk, M., Caffrey, P., Cortés, J., Lester, J. B., Bohm, G. T., Staunton, J., *et al.* (1995). The biosynthetic gene cluster for the polyketide immunosuppressant rapamycin. *Proc. Natl. Acad. Sci. USA* **92**, 7839–7843.

Silakowski, B., Nordsiek, G., Kunze, B., Blöcker, H., and Müller, R. (2001). Novel features in a combined polyketide synthase/non-ribosomal peptide synthetase: The myxalamid biosynthetic gene cluster of the myxobacterium *Stigmatella aurantiaca* Sga15. *Chem. Biol.* **8**, 59–69.

Silakowski, B., Schairer, H. U., Ehret, H., Kunze, B., Weinig, S., Nordsiek, G., Brandt, P., Blöcker, H., Höfle, G., Beyer, S., and Müller, R. (1999). New lessons for combinatorial biosynthesis from myxobacteria. The myxothiazol biosynthetic gene cluster of *Stigmatella aurantiaca* DW4/3-1. *J. Biol. Chem.* **274**, 37391–37399.

Simunovic, V., and Müller, R. (2007). 3-hydroxy-3-methylglutaryl-CoA-like syntheses direct the formation of methyl and ethyl side groups in the biosynthesis of the antibiotic myxovirescin A. *ChemBioChem* **8**, 497–500.

Simunovic, V., Zapp, J., Rachid, S., Krug, D., Meiser, P., and Müller, R. (2006). Myxovirescin A biosynthesis is directed by hybrid polyketide synthases/nonribosomal peptide synthetase, 3-hydroxy-3-methylglutaryl-CoA synthases, and trans-acting acyltransferases. *ChemBioChem* **7**, 1206–1220.

Smith, S., and Tsai, S. C. (2007). The type I fatty acid and polyketide synthases: A tale of two megasynthases. *Nat. Prod. Rep.* **24**, 1041–1072.

Sudek, S., Lopanik, N. B., Waggoner, L. E., Hildebrand, M., Anderson, C., Liu, H., Patel, A., Sherman, D. H., and Haygood, M. G. (2007). Identification of the putative bryostatin polyketide synthase gene cluster from “Candidatus Endobugula serula,” the uncultivated microbial symbiont of the marine bryozoan *Bugula neritina*. *J. Nat. Prod.* **70**, 67–74.

Tang, G. L., Cheng, Y. Q., and Shen, B. (2004a). Leinamycin biosynthesis revealing unprecedented architectural complexity for a hybrid polyketide synthase and nonribosomal peptide synthetase. *Chem. Biol.* **11**, 33–45.

Tang, G. L., Cheng, Y. Q., and Shen, B. (2006). Polyketide chain skipping mechanism in the biosynthesis of the hybrid nonribosomal peptide-polyketide antitumor antibiotic leinamycin in *Streptomyces atroolivaceus* S-140. *J. Nat. Prod.* **69**, 387–393.

Tang, G. L., Cheng, Y. Q., and Shen, B. (2007). Chain initiation in the leinamycin-producing hybrid nonribosomal peptide/polyketide synthetase from *Streptomyces atroolivaceus* S-140. Discrete, monofunctional adenylation enzyme and peptidyl carrier protein that directly load D-alanine. *J. Biol. Chem.* **282**, 20273–20282.

Tang, L., Shah, S., Chung, L., Carney, J., Katz, L., Khosla, C., and Julien, B. (2000). Cloning and heterologous expression of the epothilone gene cluster. *Science* **287**, 640–642.

Tang, L., Ward, S., Chung, L., Carney, J. R., Li, Y., Reid, R., and Katz, L. (2004b). Elucidating the mechanism of cis double bond formation in epothilone biosynthesis. *J. Am. Chem. Soc.* **126**, 46–47.

Tatsuno, S., Arakawa, K., and Kinashi, H. (2007). Analysis of modular-iterative mixed biosynthesis of lankacidin by heterologous expression and gene fusion. *J. Antibiot. (Tokyo)* **60**, 700–708.

Theisen, M. J., Misra, I., Saadat, D., Campobasso, N., Miziorko, H. M., and Harrison, D. H.. (2004). 3-hydroxy-3-methylglutaryl-CoA synthase intermediate complex observed in “real-time.” *Proc. Natl. Acad. Sci. USA* **101**, 16442–16447.

Thomas, I., Martin, C. J., Wilkinson, C. J., Staunton, J., and Leadlay, P. F. (2002). Skipping in a hybrid polyketide synthase. Evidence for ACP-to-ACP chain transfer. *Chem. Biol.* **9**, 781–787.

Ward, S. L., Hu, Z., Schirmer, A., Reid, R., Revill, W. P., Reeves, C. D., Petrakovsky, O. V., Dong, S. D., and Katz, L. (2004). The chalcomycin biosynthesis gene cluster from *Streptomyces bikiniensis*: Novel features of an unusual ketolide through expression of the chm PKS in *Streptomyces fradiae*. *Antimicrob. Agents Chemother.* **48**, 4703–4712.

Weissman, K. J. (2006). The structural basis for docking in modular polyketide biosynthesis. *ChemBioChem* **7**, 485–494.

Weissman, K. J., Bycroft, M., Cutter, A. L., Hanefeld, U., Frost, E. J., Timoney, M. C., Harris, R., Handa, S., Roddis, M., Staunton, J., and Leadlay, P. F. (1998a). Evaluating precursor-directed biosynthesis towards novel erythromycins through *in vitro* studies on a bimodular polyketide synthase. *Chem. Biol.* **5**, 743–754.

Weissman, K. J., Bycroft, M., Staunton, J., and Leadlay, P. F. (1998b). Origin of starter units for erythromycin biosynthesis. *Biochemistry* **37**, 11012–11017.

Weissman, K. J., Hong, H., Oliynyk, M., Siskos, A. P., and Leadlay, P. F. (2004). Identification of a phosphopantetheinyl transferase for erythromycin biosynthesis in *Saccharopolyspora erythraea*. *ChemBioChem* **5**, 116–125.

Weissman, K. J., Timoney, M., Bycroft, M., Grice, P., Hanefeld, U., Staunton, J., and Leadlay, P. F. (1997). The molecular basis of Celmer’s rules: The stereochemistry of the condensation step in chain extension on the erythromycin polyketide synthase. *Biochemistry* **36**, 13849–13855.

Wilkinson, C. J., Frost, E. J., Staunton, J., and Leadlay, P. F. (2001). Chain initiation on the soraphen-producing modular polyketide synthase from *Sorangium cellulosum*. *Chem. Biol.* **8**, 1197–1208.

Witkowski, A., Joshi, A. K., Lindqvist, Y., and Smith, S. (1999). Conversion of a beta-ketoacyl synthase to a malonyl decarboxylase by replacement of the active-site cysteine with glutamine. *Biochemistry* **38**, 11643–11650.

Wu, J., Hothorsall, J., Mazzetti, C., O’Connell, Y., Shields, J. A., Rahman, A. S., Cox, R. J., Crosby, J., Simpson, T. J., Thomas, C. M., and Willis, C. L. (2008). *In vivo* mutational analysis of the mupirocin gene cluster reveals labile points in the biosynthetic pathway: The “leaky hosepipe” mechanism. *ChemBioChem* **9**, 1500–1508.

Wu, K., Chung, L., Revill, W. P., Katz, L., and Reeves, C. D. (2000). The FK520 gene cluster of *Streptomyces hygroscopicus* var. *ascomyceticus* (ATCC 14891) contains genes for biosynthesis of unusual polyketide extender units. *Gene* **251**, 81–90.

Xue, Y., Zhao, L., Liu, H. W., and Sherman, D. H. (1998). A gene cluster for macrolide antibiotic biosynthesis in *Streptomyces venezuelae*: Architecture of metabolic diversity. *Proc. Natl. Acad. Sci. USA* **95**, 12111–12116.

Yu, T. W., Bai, L., Clade, D., Hoffmann, D., Toelzer, S., Trinh, K. Q., Xu, J., Moss, S. J., Leistner, E., and Floss, H. G. (2002). The biosynthetic gene cluster of the maytansinoid antitumor agent ansamitocin from *Actinosynnema pretiosum*. *Proc. Natl. Acad. Sci. USA* **99**, 7968–7973.
