
Available online at www.sciencedirect.com

**ELSEVIER**

**ScienceDirect**

**Current Opinion in Immunology**

---

**How are T<sub>H</sub>1 and T<sub>H</sub>2 effector cells made?**

Derk Amsen<sup>1</sup>, Charalampos G Spilianakis<sup>2</sup> and Richard A Flavell<sup>3,4</sup>

Differentiation of T<sub>H</sub>1 and T<sub>H</sub>2 effector cells proceeds through several phases: First, naive CD4<sup>+</sup> precursor cells are instructed to differentiate as appropriate to optimally fight the infectious threat encountered. This process is governed by the IL12 and IL4 cytokines, as well as by signaling through the Notch receptor. In response to these signals, transcription is initiated of lineage specific cytokine genes including the *Ifnγ* and *Il4* genes as well as of genes encoding transcriptional regulators, such as T-bet and Gata3. The respective differentiation programs are reinforced by both positive and negative feedback mechanisms. Furthermore, epigenetic modifications of the lineage specific genes result in the emergence of regulatory elements, which control high level lineage restricted expression by both intrachromosomal and interchromosomal associations. Together, these mechanisms ensure stable inheritance of the differentiated fate in the numerous progeny of the original naive CD4<sup>+</sup> T cells.

**Addresses**

1. Department of Cell Biology and Histology, Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands
2. Institute of Molecular Biology and Biotechnology, Foundation of Research and Technology, Vassiliki Vouton, GR 71110, Crete, Greece
3. Department of Immunobiology, Yale University School of Medicine, New Haven, CT 06520, USA
4. Howard Hughes Medical Institute, Yale University School of Medicine, New Haven, CT 06520, USA

Corresponding author: Flavell, Richard A (richard.flavell@yale.edu)

---

**IFNγ and responsible for orchestrating immunity to intracellular microorganisms, and the T<sub>H</sub>2-cell lineage, producing IL4, IL5, and IL13, which directs responses to parasites [1].**

**Cytokine induced T<sub>H</sub>1/T<sub>H</sub>2 induction**

Activation of naive CD4<sup>+</sup> T cells *in vitro* in the presence of exogenously added IL12 skews their differentiation to the T<sub>H</sub>1-cell lineage (Figure 1). The major effector of IL12 receptor signaling is the transcription factor STAT4, which promotes expression of multiple T<sub>H</sub>1-cell genes, including the *Ifnγ* gene, at least partly by inducing distinct chromatin modifications [3]. STAT4 collaborates in this with the transcription factor T-bet [3]. Expression of T-bet is induced by TCR signaling and strongly elevated by activation of the STAT1 transcription factor, which occurs in a positive feedback loop in response to autocrine produced IFNγ [4]. One of the genes induced by T-bet encodes Runx3 [5,6] and together Runx3 and T-bet bind to several enhancers and the promoter of the *Ifnγ* gene, further promoting its transcription. Runx3 and T-bet also bind to a silencer in the *Il4* gene, resulting in transcriptional repression of this T<sub>H</sub>2 gene [5,6]. Finally, T-bet promotes expression of the IL12 receptor β chain [4], increasing responsiveness to IL12. Among the many functions T-bet performs, the crucial function may be to antagonize Gata3 [7], a transcription factor controlling T<sub>H</sub>2 differentiation.

**Addition of exogenous IL4 to *in vitro* differentiation cultures promotes T<sub>H</sub>2 differentiation (Figure 1) [8]. This depends on activation of the transcription factor STAT6, which induces expression of Gata3 and may directly transactivate the *Il4* gene [8]. An important function of Gata3 is to reorganize chromatin structure in the so called T<sub>H</sub>2 locus (see below), encompassing the IL4, IL5 and IL13 genes [8]. In addition, Gata3 opposes T<sub>H</sub>1 differentiation by inhibiting expression of the IL12 receptor β chain and of STAT4. STAT6 signaling also induces expression of c-Maf, a transcription factor necessary for high-level expression of the *Il4* gene [8].**

**Notch ligands are alternative signals for T<sub>H</sub>1/T<sub>H</sub>2 differentiation**

Not all T<sub>H</sub>1 responses require IL12. T<sub>H</sub>1 responses to certain viruses are not dependent on IL12, for example [9–11]. Also, many T<sub>H</sub>2 responses *in vivo* are largely independent of IL4 receptor signaling, including those to parasites [12–14,15]. Alternative T<sub>H</sub>1 and T<sub>H</sub>2 instructing signals must therefore exist. Ligands for the Notch pathway are candidate-instructing signals for both lineages.

---

**Introduction**

Immunity to different classes of microorganisms is orchestrated by separate lineages of effector T helper (T<sub>H</sub>) cells [1], which differentiate from naive CD4<sup>+</sup> precursor cells in response to cues provided by antigen presenting cells (APC) [2]. In this review, we will discuss the molecular mechanisms that control induction of the effector lineage as well as the genetic and epigenetic events controlling long term stable expression of the lineage specific cytokine genes. Although multiple different T<sub>H</sub>-cell lineages have been identified, we will focus here on the T<sub>H</sub>1-cell lineage, characterized by production of

---

**Current Opinion in Immunology 2009, 21:153–160**

This review comes from a themed issue on Lymphocyte Development Edited by Casey Weaver and Alexander Rudensky

0952-7915/$ – see front matter © 2009 Elsevier Ltd. All rights reserved.

DOI 10.1016/j.coi.2009.03.010

www.sciencedirect.com Current Opinion in Immunology 2009, 21:153–160

154 Lymphocyte Development

Figure 1

![Diagram](attachment:diagram.png)

IL12 and IL4 driven T helper differentiation. T<sub>H</sub>1 induction by IL12: Initial TCR activation induces low-grade expression of the *Ifng* and the *Tbx21* genes (1). Signaling through the IL12 receptor results in STAT4 mediated promotion of *Ifnγ* expression (2). Binding of the IFNγ receptor by low initial auto/paracrine produced IFNγ activates STAT1 (3), which strongly promotes expression of the *Tbx21* gene (4). T-bet then enhances the transcriptional competence of the *Ifnγ* gene (5) leading to increased production of this cytokine (6). In addition, T-bet prevents T<sub>H</sub>2 differentiation by inhibiting Gata3 (7). Finally, T-bet promotes expression of the IL12 receptor β2 chain (8), resulting in greater IL12 responsiveness (9) and yet further elevated production of IFNγ (10). T<sub>H</sub>2 induction by IL4: Initial TCR signaling induces low-level expression of the *Il4* and *Gata3* genes (11). IL4 receptor signaling strongly promotes expression of these two genes (12). Gata3 reorganizes chromatin structure in the T<sub>H</sub>2 locus, encompassing the *Il4*, *Il5*, and *Il13* genes, enhancing their transcription competence (13). Increased IL4 production further enhances T<sub>H</sub>2-cell differentiation in a feed forward loop (14). Finally, Gata3 prevents the T<sub>H</sub>1 differentiation program by inhibiting expression of the IL12 receptor β2 chain (15) and of the *Stat4* gene (not depicted). Primary events are indicated with black arrows, secondary events with red arrows, and tertiary events with blue arrows.

The heterodimeric cell surface receptor Notch consists of an extracellular domain, which is non-covalently associated with a transmembrane polypeptide (Figure 2) [16]. In the canonical signaling pathway, ligand binding by Notch leads to cleavage, which allows the intracellular domain (NICD) to translocate to the nucleus and convert the DNA binding protein RBP-J (also known as CSL) into a transcriptional activator (Figure 2) [16]. Two conserved families of ligands exist, called Jagged and Delta-like (DLL) (Figure 2), which are structurally quite distinct [16]. Jagged and DLL are both capable of inducing the same canonical Notch signaling pathway. However, the different ligands do possess unique functions, although the underlying mechanisms are not understood [17].

**Induction of T<sub>H</sub>1-cell differentiation by Notch**

Expression of DLL ligands (1 or 4) is induced on APC by stimulation with microbial products that promote the ability of APC to induce T<sub>H</sub>1-cell differentiation (Figure 2) [18,19,20*,21–23]. Furthermore, when overexpressed on APC or when crosslinked as fusion proteins, DLL ligands promote T<sub>H</sub>1-cell differentiation [18,20*,22,24–26]. Although genetic evidence supporting a requirement for the Notch pathway in T<sub>H</sub>1 responses is currently lacking [18,27,28], blockade of DLL ligands did reduce T<sub>H</sub>1 responses *in vivo* [20*,24,25].

T cell specific RBP-J deficiency did not block T<sub>H</sub>1-cell responses in several studies [18,27,28,29*]. However, since these studies employed IL12 dependent T<sub>H</sub>1 models, whereas Notch may be required specifically in the absence of IL12 [20*], no definitive conclusion can currently be drawn about the involvement of RBP-J in T<sub>H</sub>1-cell induction. Consistent with a role for RBP-J in this, Notch activated expression of the *Tbx21* gene (which encodes T-bet) and RBP-J associated with the *Tbx21* gene *in vivo* in a T cell hybridoma (Figure 2) [30], although this could not be confirmed in primary T cells [31*]. A potentially RBP-J-independent mechanism involves the ability of Notch to prolong the nuclear retention of the NFκB family proteins P50 and P65 (Figure 2) [32]. However, since at least P50 has also been reported to induce expression of the T<sub>H</sub>2 factor Gata3 [33], it is not obvious how these effects on NFκB proteins would specifically translate into selective T<sub>H</sub>1-cell differentiation. Finally, DLL may inhibit T<sub>H</sub>2-cell differentiation by interfering with IL4 receptor signaling (Figure 2) [22].

How are T<sub>H</sub>1 and T<sub>H</sub>2 effector cells made? Amsen, Spilianakis and Flavell 155

Figure 2

| Bacteria/viruses/TLR-ligands | Parasites/allergens |
| --- | --- |
| RSV<br/>Acinetobacter lwoffii<br/>Lactococcus lactis<br/>LPS, CpG | Schistosoma mansoni eggs<br/>toxin of Vibrio cholerae<br/>dust mite extract<br/>Prostaglandin E2 |

![Diagram](attachment:diagram.png)

Current Opinion in Immunology

Notch mediated T<sub>H</sub>-cell differentiation. Notch signaling: Two families of Notch ligands exist, called DLL and Jagged. Notch is present as a heterodimer on the surface of the T cells. Binding of ligand to the extracellular domain of Notch results in γ-secretase mediated cleavage of the transmembrane portion of Notch, allowing the intracellular domain (NICD) to translocate to the nucleus, where it forms a trimolecular complex with the DNA binding protein RBP-J and the transcriptional coactivator Mastermind-like (MAM). T<sub>H</sub>1 (left): Members of the DLL family of Notch ligands are expressed on APC in response to microbial stimuli that promote T<sub>H</sub>1-cell induction by APC. Notch may promote T<sub>H</sub>1-cell differentiation by direct transactivation of the *Tbx21* gene (1), leading to expression of T-bet, transactivation of the *Ifnγ* gene via binding to NFκB family members like p50 (2), and indirectly through inhibition of IL4 receptor signaling (3). T<sub>H</sub>2: Expression of Jagged family members is induced on APC by T<sub>H</sub>2 promoting microbial and pro-inflammatory stimuli. Notch directly promotes activity of the upstream Gata3 promoter resulting in transcription of the *Gata3* gene (4). Furthermore, Notch regulates transcription of the *Il4* gene by binding to the HS5 enhancer (5). Please note that crucial events, such as TCR signaling have been omitted for clarity.

---

### Induction of T<sub>H</sub>2-cell differentiation by Notch

Interestingly, Notch has also been implicated in T<sub>H</sub>2-cell differentiation. In fact, genetic loss of function studies have shown that Notch is essential for T<sub>H</sub>2-cell responses under physiologically relevant conditions, such as during parasite infections [18,27,28,29*,34]. The consequence of Notch ligation (T<sub>H</sub>1 or T<sub>H</sub>2) may depend on the ligand used to activate Notch. Whereas expression of DLL ligands is associated with T<sub>H</sub>1 differentiation, expression on APC of Jagged ligands is associated with T<sub>H</sub>2 responses (Figure 2), and ectopically expressed Jagged1 or crosslinked Jagged1-Fc fusion proteins can promote T<sub>H</sub>2-cell differentiation [18,25,34–36]. A requirement for Jagged ligands in T<sub>H</sub>2-cell responses has, however, not (yet) been demonstrated [35,37].

T<sub>H</sub>2-cell induction by Notch depends on the canonical signaling pathway involving RBP-J. T<sub>H</sub>2-cell responses are severely diminished in the absence of T cell specific expression of RBP-J or of Notch1 and Notch2, as well as

in transgenic mice expressing a dominant negative version of the Notch cofactor MAML1 (Figure 2) in T cells [18,27,28,29*]. Several studies indicated that Notch regulates transcription of crucial T<sub>H</sub>2-cell genes. Thus, the upstream (but not the downstream) promoter of the *Gata3* gene responds to Notch signaling and both RBP-J and Notch1 associate with a conserved site in this promoter *in vivo* (Figure 2) [29*,31*]. Also, Notch directly activates transcription of the *Il4* gene via conserved RBP-J binding sites in the HS5 enhancer (Figure 2) [18,38]. The fact that IL4 is genetically downstream of Notch explains why exogenously added IL4, such as in traditional cytokine mediated skewing experiments, overcomes the requirement for Notch in T<sub>H</sub>2-cell differentiation [30,39].

The available data are consistent with the following model (Figure 2): Since Gata3 is necessary for Notch induced T<sub>H</sub>2-cell differentiation [29*,31*], Notch may promote expression of the *Gata3* gene first. Gata3 protein

www.sciencedirect.com Current Opinion in Immunology 2009, 21:153–160

then remodels the T<sub>H</sub>2 cytokine locus, rendering the HS5 enhancer accessible to Notch. Ensuing auto/paracrine production of IL4 further enhances T<sub>H</sub>2 differentiation. In a further positive feedback mechanism, Gata3 protein reinforces expression of the Gata3 gene, and this effect from Gata3 is synergistically enhanced by Notch [31]. Crucial elements of this model still need to be proven definitively. One important experiment would be to delete the Notch responsive element in the *Gata3* gene to formally determine whether transactivation of *Gata3* by Notch is indeed necessary for T<sub>H</sub>2 differentiation.

**The emergence of regulatory elements**

Although the *Ifnγ* and *Il4* genes are transcribed in naive T cells within 3–24 h after initial activation [40], only upon terminal differentiation several days later does strong lineage specific expression occur. This elevated and lineage restricted expression requires increased expression of positively or negatively acting lineage specific transcription factors. These factors bind to the gene promoters as well as lineage specific regulatory regions, which are induced during differentiation by chromatin modifying enzymes associated with factors such as T-bet and Gata3.

Regulatory elements have been identified by a combination of bioinformatics and biochemistry. Conserved (between species) non-coding sequences (CNS) indicated the existence of important regulatory elements, and hypersensitivity to digestion by DNase I revealed the binding of protein factors typically found at regulatory elements. During the differentiation process, many epigenetic modifications are made to these regulatory regions. The histones in the chromatin of cytokine loci of naive T cells are hypo-acetylated. TCR stimulation induces rapid acetylation on histones H3 and H4 in the cytokine gene promoters, irrespective of polarizing conditions. The lineage specific maintenance of acetylation depends on cytokine dependent STAT signaling and lineage specific transcriptional activators T-bet and GATA3 [41].

**Cis-regulatory elements in the *Ifnγ* cytokine locus**

Expression of the *Ifnγ* gene is regulated by its *cis* acting promoter and by two intronic regulatory elements (Figure 3) [42]. These elements are not by themselves sufficient to achieve T<sub>H</sub>1 specific expression. Indeed, additional regulatory elements have been characterized. Two of these, *Ifnγ*-CNS1 and *Ifnγ*-CNS2, are located 5 kb upstream and 18 kb downstream of the initiation codon of the murine *Ifnγ* gene, respectively (Figure 3) [43,44]. Furthermore, studies utilizing bacterial artificial chromosomes containing a human *Ifnγ* transgene suggested that for proper expression yet additional elements are required, which are dispersed in an extended region of over 100 kb around the gene (Figure 3) [41].

In T<sub>H</sub>1 cells, the CNSs in the *Ifnγ* locus contain dimethylations of histone H3 lysine 4 (H3K4), a stable mark associated with poised or actively transcribed chromatin. Furthermore, H3K9 methylation occurs after T cell activation and is maintained throughout T<sub>H</sub>1-cell differentiation [45]. By contrast, this modification is diminished in T<sub>H</sub>2 cells in a GATA3 dependent manner, and both STAT6 and GATA3 are recruited to the *Ifnγ* locus in T<sub>H</sub>2 cells [46]. Also recruited is the polycomb group methyltransferase enzyme EZH2, which initiates repressive H3K27 methylation. Correspondingly, trimethyl H3K27 is broadly distributed throughout the locus in T<sub>H</sub>2 cells. Thus, multiple regulatory regions have now been identified in the *Ifnγ* locus, which undergo dynamic modifications during cellular differentiation.

**Cis-regulatory elements in the T<sub>H</sub>2 cytokine locus**

The T<sub>H</sub>2 cytokine locus comprises the *Il4*, *Il5* and *Il13* cytokine genes and stretches out over 120 kb. Between the *Il5* and *Il13* genes resides the *Rad50* gene, which is constitutively expressed and not coregulated with the T<sub>H</sub>2 cytokine genes. Multiple DNase I hypersensitive sites have been identified in the T<sub>H</sub>2 locus. Some of these flank the *Il4* and *Il13* genes (Figure 3) and together controlled expression of reporter transgenes in a T<sub>H</sub>2-specific fashion [47,48]. Among these is the 3′ Notch responsive enhancer HS5 [18,38]. Additionally, seven hypersensitive sites exist more distally. Four of these (RHS4-7) contain the so called T<sub>H</sub>2 locus control region (LCR) at the 3′ of the *Rad50* gene (Figure 3). This was shown by the ability of this region to support high level tissue (T<sub>H</sub>2)-specific expression of linked reporter genes, irrespective of any flanking sequences [49,50], consistent with the operational definition of LCRs.

**Long range intrachromosomal and interchromosomal interactions**

As discussed, many of the regulatory regions are located far from the cytokine genes. How then do such elements act on their target genes? Using the so called Chromosome Conformation Capture (3C) assay [51], it was shown that the *Ifnγ* locus adopts a secondary chromatin configuration where distant regulatory regions come in close proximity with the promoter of the *Ifnγ* gene, depending on the expression profile of the cell (Figure 4) [52]. Similarly, the coordinated regulation of the T<sub>H</sub>2 cytokine genes involves the generation of chromatin loops that bring together the promoters of the genes, their enhancers and the T<sub>H</sub>2 LCR (Figure 4) [53]. This LCR regulates the cytokine genes without affecting the constitutively expressed *Rad50* gene, which is looped out of the active chromatin configuration (Figure 4).

Quite unexpectedly, long-range chromatin interactions are not restricted to intrachromosomal regions. The T<sub>H</sub>2

How are T<sub>H</sub>1 and T<sub>H</sub>2 effector cells made? Amsen, Spilianakis and Flavell 157

Figure 3

(a)

![Diagram](TH2 LCR)

(b)

Genomic organization of the T<sub>H</sub>2 cytokine locus on mouse chromosome 11 and the *Ifnγ* locus on mouse chromosome 10. (a) Blue bold arrows indicate constitutive (present in naive CD4<sup>+</sup>, T<sub>H</sub>1, and T<sub>H</sub>2 cells) DNase I hypersensitive sites and black arrows indicate DNase I hypersensitive sites specific in T<sub>H</sub>2 cells (for the T<sub>H</sub>2 locus) or T<sub>H</sub>1 cells (for the *Ifnγ* locus). Vertical boxes on the loci indicate gene exons. (b) Evolutionary conserved regions in the IFNγ locus. Cross species conservation analysis of at least 100 bp and 70% DNA homology between mouse and human species for the genomic regions displaying the *Ifnγ* locus on mouse chromosome 10 using the ECR browser ([http://ecrbrowser.dcode.org](http://ecrbrowser.dcode.org)) (Colors depicting the conserved regions: blue-coding regions, yellow-untranslated regions, red-intergenic region, green-transposons and simple repeats, pink-intronic regions). Arrows on top depict DNase I hypersensitive sites in naive CD4<sup>+</sup> cells (black), T<sub>H</sub>1 cells (blue), and T<sub>H</sub>2 cells (red).

---

LCR and other regulatory elements in the T<sub>H</sub>2 locus, localized on mouse chromosome 11, colocalize and come in close physical proximity with the *Ifnγ* gene locus, located on mouse chromosome 10 [54]. Analysis utilizing genetically modified animals suggests a direct mechanism for transcriptional gene regulation of genes by regulatory elements located on different chromosomes. A genetic deletion of the RHS7 DNase I hypersensitive site of the T<sub>H</sub>2 LCR that predominantly regulates the expression of the T<sub>H</sub>2 cytokine gene loci on chromosome 11 could affect the expression profile of the *Ifnγ* gene on mouse chromosome 10 [54]. Interchromosomal interactions opened a completely new avenue in our understanding of gene regulation. This phenomenon now appears to be a general mechanism employed in multiple genetic systems [40].

Subnuclear localization of loci in lymphocytes

Given that regulatory elements can control genes on different chromosomes, another layer of control over gene expression involves the subnuclear localization of genes. Genes tend to relocalize from the nuclear periphery to the inner part of the nucleus upon their activation [55]. Silent genes in developing B and T cells are repositioned in the nucleus to pericentromeric heterochromatin showing a direct link between the subnuclear localization of genes and differentiation [56,57]. Regulatory elements such as LCRs, enhancers, or insulators also act by repositioning specific genetic loci to regions with active or silent transcription. Furthermore, sequence-specific DNA binding proteins may act by directly repositioning loci to relevant chromatin compartments [58,59].

158 Lymphocyte Development

Figure 4

Dynamic interplay of chromosomal interactions in the nucleus of T cells. (a) Arrows on top of the TH2 cytokine locus represent the intrachromosomal interactions between the RHS7 DNase I hypersensitive site of the TH2 LCR and several other genomic regions of the TH2 locus in TH2 cells. Arrows on top of the Ifnγ locus depict the intrachromosomal as well as the interchromosomal interactions captured for the Ifnγ–TH2 cytokine loci. The arrow with the discontinuous line represents the TH1 specific intrachromosomal interaction between two regions of the Ifnγ locus. (b) Schematic hypothetical representation of the territories for mouse chromosome 10 and 11. The TH2 cytokine locus is located on mouse chromosome 11 and the Ifnγ locus is located on mouse chromosome 10. We hypothesize that the interchromosomal interactions between the TH2 cytokine locus and the Ifnγ locus are taking place in the interchromosome domain compartment in naive CD4+ cells potentially keeping the two loci poised and at the same time transcriptionally silent for rapid expression upon TCR stimulation. During differentiation of naive CD4+ T cells to TH1 cells the Ifnγ gene is mainly regulated through intrachromosomal interactions with regulatory elements in the locus, and in TH2 cells intrachromosomal interactions between the TH2 LCR and the cytokine gene promoters upon the action of TH2 cell lineage specific factors regulate transcription.

Reportedly, the transcriptionally silent Il4 (in TH1 cells) and Ifnγ genes (in TH2 cells) are associated with centromeric domains [60], although this was not confirmed by another group. Instead, it was found that during TH1 differentiation, the GATA3 and c-maf loci were progressively repositioned to centromeric heterochromatin and/or the nuclear periphery, compartments associated with transcriptional repression. Therefore, the loci of the transcriptional regulators of the cytokine genes may reposition in the cell nucleus during T cell differentiation rather than the cytokine gene loci themselves [61].

Conclusion

Differentiation of naive CD4+ T helper cells into the TH1-cell and TH2-cell lineages occurs in response to specific information from APC and cytokines. In addition to the cytokines IL4 and IL12, ligands for the Notch pathway are also involved in this process.

TCR stimulation in combination with instructional differentiation signals results in the activation of specific transcription factors that initiate cytokine gene expression and modify chromatin structure of regulatory regions.

These control strong lineage specific expression of the cytokine genes both by acting on genes located within the same chromosome, as well as with genes located on other chromosomes. How such intrachromosomal and inter-chromosomal interactions control gene transcription is an important question for the future.

### Acknowledgements

The authors would like to thank Andrey Antov for help making Figures and Fran Manzo for help with manuscript preparation. DA is supported by an AMC fellowship and a fellowship from the Landsteiner Foundation for Blood research. BS is supported by a Cancer Research Institute Investigator award and a Marie Curie International Reintegration Grant (IRG). RAF is an investigator of the Howard Hughes Medical Institute.

### References and recommended reading

Papers of particular interest published within the period of review have been highlighted as:

- of special interest
- of outstanding interest

1. Abbas AK, Murphy KM, Sher A: Functional diversity of helper T lymphocytes. *Nature* 1996, **383**:787-793.
2. Kapsenberg ML: Dendritic-cell control of pathogen-driven T-cell polarization. *Nat Rev Immunol* 2003, **3**:984-993.
3. Thieu VT, Yu Q, Chang HC, Yeh N, Nguyen ET, Sehra S, Kaplan MH: Signal transducer and activator of transcription 4 is required for the transcription factor T-bet to promote T helper 1 cell-fate determination. *Immunity* 2008, **29**:679-690.
   - Rather than having a linear relationship, STAT4 and T-bet are shown to have complementary roles in regulation of the T<sub>H</sub>1 transcriptional program, with some genes dependent on one and other genes dependent on both transcription factors.
4. Weaver CT, Hatton RD, Mangan PR, Harrington LE: IL-17 family cytokines and the expanding diversity of effector T cell lineages. *Annu Rev Immunol* 2007, **25**:821-852.
5. Djuretic IM, Levanon D, Negreanu V, Groner Y, Rao A, Ansel KM: Transcription factors T-bet and Runx3 cooperate to activate Ifng and silence Il4 in T helper type 1 cells. *Nat Immunol* 2007, **8**:145-153.
6. Naoe Y, Setoguchi R, Akiyama K, Muroi S, Kuroda M, Hatam F, Littman DR, Taniuchi I: Repression of interleukin-4 in T helper type 1 cells by Runx/Cbf beta binding to the Il4 silencer. *J Exp Med* 2007, **204**:1749-1755.
   - References [4] and [6] show both feed forward and negative feedback mechanisms from T-bet and Runx in the T<sub>H</sub>1 differentiation program.
7. Usui T, Preiss JC, Kanno Y, Yao ZJ, Bream JH, O'Shea JJ, Strober W: T-bet regulates Th1 responses through essential effects on GATA-3 function rather than on IFNG gene acetylation and transcription. *J Exp Med* 2006, **203**:755-766.
8. Ansel KM, Djuretic I, Tanasa B, Rao A: Regulation of Th2 differentiation and Il4 locus accessibility. *Annu Rev Immunol* 2006, **24**:607-656.
9. de Wit MC, Horzinek MC, Haagmans BL, Schijns VE: Host-dependent type 1 cytokine responses driven by inactivated viruses may fail to default in the absence of IL-12 or IFN-alpha/beta. *J Gen Virol* 2004, **85**:795-803.
10. Oxenius A, Karrer U, Zinkernagel RM, Hengartner H: IL-12 is not required for induction of type 1 cytokine responses in viral infections. *J Immunol* 1999, **162**:965-973.
11. Schijns VE, Haagmans BL, Wierda CM, Kruithof B, Heijnen IA, Alber G, Horzinek MC: Mice lacking IL-12 develop polarized Th1 cells during viral infection. *J Immunol* 1998, **160**:3958-3964.
12. Finkelman FD, Morris SC, Orekhova T, Mori M, Donaldson D, Reiner SL, Reilly NL, Schopf L, Urban JF Jr: Stat6 regulation of in vivo IL-4 responses. *J Immunol* 2000, **164**:2303-2310.

13. Jankovic D, Kullberg MC, Noben-Trauth N, Caspar P, Paul WE, Sher A: Single cell analysis reveals that IL-4 receptor/Stat6 signaling is not required for the in vivo or in vitro development of CD4+ lymphocytes with a Th2 cytokine profile. *J Immunol* 2000, **164**:3047-3055.
14. Voehringer D, Reese TA, Huang X, Shinkai K, Locksley RM: Type 2 immunity is controlled by IL-4/IL-13 expression in hematopoietic non-eosinophil cells of the innate immune system. *J Exp Med* 2006, **203**:1435-1446.
15. King SB, Knorn AM, Ohnmacht C, Voehringer D: Accumulation of effector CD4 T cells during type 2 immune responses is negatively regulated by Stat6. *J Immunol* 2008, **180**:754-763.
   - This article shows that STAT6-deficient mice generate normal absolute numbers of parasite specific T<sub>H</sub>2 cells. However, large numbers of non-T<sub>H</sub>2 cells are also generated in these mice, which have caused previous studies (measuring only relative frequencies) to underestimate the magnitude of the T<sub>H</sub>2 responses.
16. Bray SJ: Notch signalling: a simple pathway becomes complex. *Nat Rev Mol Cell Biol* 2006, **7**:678-689.
17. D'Souza B, Miyamoto A, Weinmaster G: The many facets of Notch ligands. *Oncogene* 2008, **27**:5148-5167.
18. Amsen D, Blander JM, Lee GR, Tanigaki K, Honjo T, Flavell RA: Instruction of distinct CD4 T helper cell fates by different notch ligands on antigen-presenting cells. *Cell* 2004, **117**:515-526.
19. Napolitani G, Rinaldi A, Bertoni F, Sallusto F, Lanzavecchia A: Selected Toll-like receptor agonist combinations synergistically trigger a T helper type 1-polarizing program in dendritic cells. *Nat Immunol* 2005, **6**:769-776.
20. Skokos D, Nussenzwieg MC: CD8- DCs induce IL-12-independent Th1 differentiation through Delta 4 Notch-like ligand in response to bacterial LPS. *J Exp Med* 2007, **204**:1525-1531.
   - This article shows that Notch predominantly functions in IL12-independent T<sub>H</sub>1-cell responses.
21. Rudd BD, Schaller MA, Smit JJ, Kunkel SL, Neupane R, Kelley L, Berlin AA, Lukacs NW: MyD88-mediated instructive signals in dendritic cells regulate pulmonary immune responses during respiratory virus infection. *J Immunol* 2007, **178**:5820-5827.
22. Sun J, Krawczyk CJ, Pearce EJ: Suppression of Th2 cell development by Notch ligands Delta 1 and Delta 4. *J Immunol* 2008, **180**:1655-1661.
23. Debarry J, Garn H, Hanuszkiewicz A, Dickgreber N, Blumer N, von Mutius E, Bufe A, Gatermann S, Renz H, Holst O *et al.*: Acinetobacter lwoffii and Lactococcus lactis strains isolated from farm cowsheds possess strong allergy-protective properties. *J Allergy Clin Immunol* 2007, **119**:1514-1521.
24. Maekawa Y, Tsukumo S, Chiba S, Hirai H, Hayashi Y, Okada H, Kishihara K, Yasutomo K: Delta1-Notch3 interactions bias the functional differentiation of activated CD4+ T cells. *Immunity* 2003, **19**:549-559.
25. Elyaman W, Bradshaw EM, Wang Y, Oukka M, Kivisakk P, Chiba S, Yagita H, Khoury SJ: JAGGED1 and delta 1 differentially regulate the outcome of experimental autoimmune encephalomyelitis. *J Immunol* 2007, **179**:5990-5998.
26. Okamoto M, Takeda K, Joetham A, Ohnishi H, Matsuda H, Swasey CH, Swanson BJ, Yasutomo K, Dakhama A, Gelfand EW: Essential role of Notch signaling in effector memory CD8+ T cell-mediated airway hyperresponsiveness and inflammation. *J Exp Med* 2008, **205**:1087-1097.
27. Tanigaki K, Tsuji M, Yamamoto N, Han H, Tsukada J, Inoue H, Kubo M, Honjo T: Regulation of alphabeta/gammadelta T cell lineage commitment and peripheral T cell responses by Notch/RBP-J signaling. *Immunity* 2004, **20**:611-622.
28. Tu L, Fang TC, Artis D, Shestova O, Pross SE, Maillard I, Pear WS: Notch signaling is an important regulator of type 2 immunity. *J Exp Med* 2005, **202**:1037-1042.
29. Amsen D, Antov A, Jankovic D, Sher A, Radtke F, Souabni A, Busslinger M, McCright B, Gridley T, Flavell RA: Direct regulation of Gata3 expression determines the T helper differentiation potential of Notch. *Immunity* 2007, **27**:89-99.

Together with reference [31], this article shows that Notch directly regulates expression of the Gata3 gene, providing a mechanism for initiation of T<sub>H</sub>2-cell differentiation.

30. Minter LM, Turley DM, Das P, Shin HM, Joshi I, Lawlor RG, Cho OH, Palaga T, Gottipati S, Telfer JC et al.: Inhibitors of gamma-secretase block in vivo and in vitro T helper type 1 polarization by preventing Notch upregulation of Tbx21. Nat Immunol 2005, 6:680-688.

31. Fang TC, Yashiro-Ohtani Y, Del Bianco C, Knoblock DM, Blacklow SC, Pear WS: Notch directly regulates Gata3 expression during T helper 2 cell differentiation. Immunity 2007, 27:100-110.

Together with reference [29], this article shows that Notch directly regulates expression of the Gata3 gene, providing a mechanism for initiation of T<sub>H</sub>2-cell differentiation.

32. Shin HM, Minter LM, Cho OH, Gottipati S, Fauq AH, Golde TE, Sonenshein GE, Osborne BA: Notch1 augments NF-kappaB activity by facilitating its nuclear retention. EMBO J 2006, 25:129-138.

33. Das J, Chen CH, Yang L, Cohn L, Ray P, Ray A: A critical role for NF-kappa B in GATA 3 expression and TH2 differentiation in allergic airway inflammation. Nat Immunol 2001, 2:45-50.

34. Liotta F, Frosali F, Querci V, Mantei A, Fili L, Maggi L, Mazzinghi B, Angeli R, Ronconi E, Santarlasci V et al.: Human immature myeloid dendritic cells trigger a TH2-polarizing program via Jagged-1/Notch interaction. J Allergy Clin Immunol 2008, 121:1000-1005 e1008.

35. Krawczyk CM, Sun J, Pearce EJ: Th2 differentiation is unaffected by Jagged2 expression on dendritic cells. J Immunol 2008, 180:7931-7937.

36. Krishnamoorthy N, Oriss TB, Paglia M, Fei M, Yarlagadda M, Vanhaesebroeck B, Ray A, Ray P: Activation of c-Kit in dendritic cells regulates T helper cell differentiation and allergic asthma. Nat Med 2008, 14:565-573.

37. Worsley AG, Leibundgut-Landmann S, Slack E, Phng LK, Gerhardt H, Sousa CR, Macdonald AS: Dendritic cell expression of the Notch ligand jagged2 is not essential for Th2 response induction in vivo. Eur J Immunol 2008, 38:1043-1049.

38. Tanaka S, Tsukada J, Suzuki W, Hayashi K, Tanigaki K, Tsuji M, Inoue H, Honjo T, Kubo M: The interleukin-4 enhancer CNS-2 is regulated by Notch signals and controls initial expression in NKT cells and memory-type CD4 T cells. Immunity 2006, 24:689-701.

39. Ong CT, Sedy JR, Murphy KM, Kopan R: Notch and presenilin regulate cellular expansion and cytokine secretion but cannot instruct Th1/Th2 fate acquisition. PLoS ONE 2008, 3:e2823.

40. Lee GR, Kim ST, Spilianakis CG, Fields PE, Flavell RA: T helper cell differentiation: regulation by cis elements and epigenetics. Immunity 2006, 24:369-379.

41. Rowell E, Merkenschlager M, Wilson CB: Long-range regulation of cytokine gene expression. Curr Opin Immunol 2008, 20:272-280.

42. Agarwal S, Rao A: Modulation of chromatin structure regulates cytokine gene expression during T cell differentiation. Immunity 1998, 9:765-775.

43. Lee DU, Avni O, Chen L, Rao A: A distal enhancer in the IFN-γ locus revealed by genome sequence comparison. J Biol Chem 2004, 279:4802-4810.

44. Shnyreva M, Weaver WM, Blanchette M, Taylor SL, Tompa M, Fitzpatrick DR, Wilson CB: Evolutionarily conserved sequence elements that positively regulate IFN-α expression in T cells. Proc Natl Acad Sci USA 2004, 101:12622-12627.

45. Schoenborn JR, Dorschner MO, Sekimata M, Santer DM, Shnyreva M, Fitzpatrick DR, Stamatoyannopoulos JA, Wilson CB:

Comprehensive epigenetic profiling identifies multiple distal regulatory elements directing transcription of the gene encoding interferon-gamma. Nat Immunol 2007, 8:732-742.

In this article, a comprehensive epigenetic analysis was made of the extended Ifnγ locus.

46. Chang S, Aune TM: Dynamic changes in histone-methylation 'marks' across the locus encoding interferon-gamma during the differentiation of T helper type 2 cells. Nat Immunol 2007, 8:723-731.

47. Lee GR, Fields PE, Flavell RA: Regulation of IL-4 gene expression by distal regulatory elements and GATA-3 at the chromatin level. Immunity 2001, 14:447-459.

48. Takemoto N, Koyano-Nakagawa N, Yokota T, Arai N, Miyatake S, Arai K: TH2-specific DNase I-hypersensitive sites in the murine IL-13 and IL-4 intergenic region. Int Immunol 1998, 10:1981-1985.

49. Fields PE, Lee GR, Kim ST, Bartsevich V, Flavell RA: Th2-specific chromatin remodeling and enhancer activity in the Th2 cytokine locus control region. Immunity 2004, 21:865-876.

50. Lee GR, Fields PE IV, Griffin TJ, Flavell RA: Regulation of the Th2 cytokine locus by a Locus Control Region. Immunity 2003, 19:145-153.

This article describes the identification of a Locus control region in the T<sub>H</sub>2 cytokine locus.

51. Simonis M, Kooren J, de Laat W: An evaluation of 3C-based methods to capture DNA interactions. Nat Methods 2007, 4:895-901.

Comparison of the different 3C-based methods.

52. Eviazova ER, Aune TM: Dynamic alterations in the conformation of the Ifng gene region during T helper cell differentiation. Proc Natl Acad Sci USA 2004, 101:251-256.

53. Spilianakis C, Flavell RA: Long-range intrachromosomal interactions in the T helper type 2 cytokine locus. Nat Immunol 2004, 5:1017-1027.

54. Spilianakis C, Lalioti M, Town T, Lee GR, Flavell RA: Interchromosomal associations between alternatively expressed loci. Nature 2005, 435:637-645.

This article provides a demonstration of the functional interchromosomal association between the Ifnγ gene with regulatory elements of the T<sub>H</sub>2 cytokine locus.

55. Kosak ST, Groudine M: Form follows function: the genomic organization of cellular differentiation. Genes Dev 2004, 18:1371-1384.

56. Brown KE, Baxter J, Graf D, Merkenschlager M, Fisher AG: Dynamic repositioning of genes in the nucleus of lymphocytes preparing for cell division. Mol Cell 1999, 3:207-217.

57. Brown KE, Guest SS, Smale ST, Hahm K, Merkenschlager M, Fisher AG: Association of transcriptionally silent genes with Ikaros complexes at centromeric heterochromatin. Cell 1997, 91:845-854.

58. Fraser P, Bickmore W: Nuclear organization of the genome and the potential for gene regulation. Nature 2007, 447:413-417.

59. Krueger C, Osborne CS: Raising the curtains on interchromosomal interactions. Trends Genet 2006, 12:637-639.

60. Grogan JL, Mohrs M, Harmon B, Lacy DA, Sedat JW, Locksley RM: Early transcription and silencing of cytokine genes underlie polarization of T helper cell subsets. Immunity 2001, 14:205-215.

61. Hewitt SL, High FA, Reiner SL, Fisher AG, Merkenschlager M: Nuclear repositioning marks the selective exclusion of lineage-inappropriate transcription factor loci during T helper cell differentiation. Eur J Immunol 2004, 34:3604-3613.
