
Cellular Microbiology (2009) 11(6), 911–926

doi:10.1111/j.1462-5822.2009.01301.x  
First published online 18 March 2009

**Targeting eEF1A by a *Legionella pneumophila* effector leads to inhibition of protein synthesis and induction of host stress response**

Xihui Shen,$^{1\dagger}$ Simran Banga,$^{1\dagger}$ Yancheng Liu,$^{1}$ Li Xu,$^{1}$ Ping Gao,$^{1}$ Ilya Shamovsky,$^{2}$ Evgeny Nudler$^{2}$ and Zhao-Qing Luo$^{1*}$

$^{1}$ Department of Biological Sciences, Purdue University, 915 West State Street, West Lafayette, IN 47907, USA.  
$^{2}$ Department of Biochemistry, New York University School of Medicine, New York, New York 10016, USA.

---

### Summary

The *Legionella pneumophila* Dot/Icm type IV secretion system is essential for the biogenesis of a phagosome that supports bacterial multiplication, most likely via the functions of its protein substrates. Recent studies indicate that fundamental cellular processes, such as vesicle trafficking, stress response, autophagy and cell death, are modulated by these effectors. However, how each translocated protein contributes to the modulation of these pathways is largely unknown. In a screen to search substrates of the Dot/Icm transporter that can cause host cell death, we identified a gene whose product is lethal to yeast and mammalian cells. We demonstrate that this protein, called SidI, is a substrate of the Dot/Icm type IV protein transporter that targets the host protein translation process. Our results indicate that SidI specifically interacts with eEF1A and eEF1Bγ, two components of the eukaryotic protein translation elongation machinery and such interactions leads to inhibition of host protein synthesis. Furthermore, we have isolated two SidI substitution mutants that retain the target binding activity but have lost toxicity to eukaryotic cells, suggesting potential biochemical effect of SidI on eEF1A and eEF1Bγ. We also show that infection by *L. pneumophila* leads to eEF1A-mediated activation of the heat shock regulatory protein HSF1 in a virulence-dependent manner and deletion of *sidI* affects such activation.

---

Moreover, similar response occurred in cells transiently transfected to express SidI. Thus, inhibition of host protein synthesis by specific effectors contributes to the induction of stress response in *L. pneumophila*-infected cells.

### Introduction

*Legionella pneumophila* is a facultative intracellular bacterial pathogen capable of multiplying within fresh water amoeba and mammalian cells. Upon being phagocytosed, the bacterium initiates a unique trafficking route that bypasses the endocytic pathways to establish a *Legionella*-containing vacuole (LCV) with characteristics of the host rough endoplasmic reticulum (ER) compartment (Tilney *et al.*, 2001). After 18–24 h of replication, the vacuole is lysed and a large number of bacteria are released to initiate a new round of infection.

Essential to intracellular replication of *L. pneumophila* is the Dot/Icm type IV secretion system that translocates bacterial effector proteins into host cells during infection (Ninio and Roy, 2007). A large number of substrates of the Dot/Icm transporter have been identified, but only a few of them have been assigned specific biochemical activities and roles in the biogenesis of the LCVs. It is well established that LCVs actively intercept vesicles originating from the ER to transform the bacterial vacuole into an ER-like compartment. Several bacterial effectors involved in this process have been characterized. Among them, RalF functions to recruit and activate Arf1 (Nagai *et al.*, 2002) whereas SidM/DrrA along with LepB modulates the activity of Rab1 (Machner and Isberg, 2006; Murata *et al.*, 2006; Ingmundson *et al.*, 2007; Machner and Isberg, 2007). Another effector, SidJ, is important for efficient recruitment of ER proteins to the bacterial vacuole, but its mechanism of action is unknown (Liu and Luo, 2007).

Reverse genetics and analysis of global gene expression profiles of infected cells revealed that in addition to vesicle trafficking, the Dot/Icm system modulates more cellular processes than previously anticipated (Dorer *et al.*, 2006; Losick and Isberg, 2006). These observations may explain the large number of effectors transferred by the system. Depletion of components of host proteasome

Received 2 January, 2009; revised 12 February, 2009; accepted 13 February, 2009. *For correspondence. E-mail* luoz@purdue.edu; Tel. (+1) 765 496 6697; Fax (+1) 765 494 0876.  
$^\dagger$These authors contributed equally to this work.

© 2009 Blackwell Publishing Ltd

cellular microbiology

912 X. Shen et al.

degradation machinery by siRNA leads to inhibition of *L. pneumophila* replication (Dorer *et al.*, 2006). As anticipated, proteins predicted to be involved in ubiquitination pathways have been identified in *L. pneumophila* genomes and one such protein has been shown to possess ubiquitin ligase activity (de Felipe *et al.*, 2005; Kubori *et al.*, 2008).

Mammalian cells infected by *L. pneumophila* in its early stage exhibited strong resistance to exogenous apoptotic stimuli; this may be due to activities of cell death inhibitory effectors such as SdhA and SidF (Laguna *et al.*, 2006; Banga *et al.*, 2007) and by the virulence-dependent induction of many anti-apoptotic genes (Losick and Isberg, 2006; Abu-Zant *et al.*, 2007). Mutations in certain Dot/Icm substrate genes resulted in host cell death, suggesting the existence of bacterial factors detrimental to the host (Laguna *et al.*, 2006; Banga *et al.*, 2007). In this study, we describe one such effector Sidl, which functions as an inhibitor of host protein synthesis by targeting eEF1A and eEF1Bγ, two components of the elongation machinery of eukaryotic protein synthesis (Browne and Proud, 2002). We also show that inhibition of protein synthesis by Sidl leads to the induction of the host stress shock response, possibly by making eEF1A available to activate the heat shock response regulator HSF1.

### Results

#### Identification of a *L. pneumophila* protein exhibits toxicity to eukaryotic cells

The availability of various genetic tools makes the yeast *Saccharomyces cerevisiae* a very useful model in the study of bacterial effectors (Siggers and Lesser, 2008). Several *L. pneumophila* effectors have been identified by their ability to kill yeast cells (Campodonico *et al.*, 2005) or to interfere with its vesicle trafficking process (Shohdy *et al.*, 2005; Heidtman *et al.*, 2009). Furthermore, because some mammalian pro-apoptotic proteins exhibit lethality to yeast (Tao *et al.*, 1997), the toxic phenotypes may also lead to the identification of cell death-inducing *L. pneumophila* effectors. The use of random libraries may not allow one to identify all such genes, as many genes require a full-length protein for their activity. Thus, we cloned full-length hypothetical *L. pneumophila* genes into a yeast vector and the resulting plasmids were individually introduced into a yeast strain. Transformation that yielded few colonies suggested the identification of a potentially toxic protein (Fig. S1). Several proteins exhibiting lethality to yeast were identified from the first 96 candidates (Table S1). Here we described one of such genes [lpg2504, ceg32 (Zusman *et al.*, 2007)] that codes for a protein of 942 amino acids with a predicted molecular weight of approximately 110 kDa, which we referred to as *sidl*.

To confirm the lethality phenotype, we expressed Sidl from the yeast galactose-inducible promoter and examined the inducer-dependent toxicity. Whereas all yeast strains grew normally on medium containing glucose, the strain harbouring the chromosome-encoded P<sub>gal</sub>::*sidl* failed to grow on medium containing galactose (Fig. 1A). These data are consistent with an earlier study showing that *sidl* cannot be expressed in yeast (Heidtman *et al.*, 2009). Overexpression of foreign proteins sometimes leads to non-specific toxicity to yeast, so we performed two sets of experiments to examine the possibility that the observed Sidl-mediated lethality was caused by a specific activity. First, we screened for Sidl mutants that no longer were toxic to yeast. Plasmid DNA carrying *sidl* was mutagenized with hydroxylamine and the treated DNA was transformed into yeast. Surviving yeast strains were screened by immunoblotting for their ability to code for a full-size Sidl protein (Fig. 1B and Fig. S2A). Three such mutants were identified from about 400 candidates. In the first mutant, Glu482 was changed to lysine and the mutant was designated Sidl*. In the other two mutants, Arg453 was substituted with either Leu or Pro. In this study, we will focus on the R453P mutant, which we referred to as Sidl**. Consistent with their original non-toxic phenotypes, yeast strains carrying these alleles of *sidl* expressed from a galactose-inducible promoter gained the ability to grow on inducing medium (Fig. 1A lower panel). The loss of toxicity was not due to decreased protein stability because these mutants (Sidl* and Sidl**) encoded stable proteins in both yeast and *Escherichia coli* (Fig. 1B and Fig. S2A and B). Second, we transiently expressed Sidl in mammalian cells of 80% confluency and found that cell survival significantly decreased in 24 h (Fig. 1C left panel and D). In similar experiments with the mutants, although about 80% of the cells were transfected (expressing the GFP signal), the cells continued to proliferate to confluency (Fig. 1B right panel and data not shown). Furthermore, GFP fusions of these proteins can be detected in these cells (Fig. S2C). These data indicate that these mutants have lost the cell-killing activity. Taken together, our screening has identified a *L. pneumophila* protein toxic to eukaryotic cells, likely by targeting an essential host process(es) conserved among a wide spectrum of eukaryotes.

#### Sidl is a substrate of the Dot/Icm system that is induced at exponential growth phase

*sidl* (ceg32) is one of genes identified to be regulated by PmrA, a protein that controls expression of many substrates of the Dot/Icm system (Zusman *et al.*, 2007). We employed three independent methods, the interbacterial protein transfer assay (Luo and Isberg, 2004), the SidCΔC100 fusion assay (VanRheenen *et al.*, 2006) and

© 2009 Blackwell Publishing Ltd, *Cellular Microbiology*, **11**, 911–926

Fig. 1. Toxicity of Sidl to eukaryotic cells.

A. Sidl-mediated toxicity to yeast. Growth of yeast cells expressing *sidl*, *sidl*<sup>*</sup> or *sidl*<sup>**</sup> from the galactose-inducible promoter on glucose medium (upper panel) or on galactose medium (lower panel). A yeast strain harbouring an empty vector was used as a control (vector). Serially diluted yeast cultures grown in glucose medium were spotted followed by incubation at 30°C for 2 days (glucose) or 3 days (galactose).

B. Untagged *sidl*<sup>*</sup> and *sidl*<sup>**</sup> were expressed in yeast. Total lysates of yeast cells grown in 2% galactose medium were probed with an anti-Sidl antibody.

C. Inhibition of mammalian cell proliferation by Sidl. 293T cells on coverslips with about 80% confluence were transfected to express Sidl (left panel) or the Sidl<sup>*</sup> mutant (right panel) (transfection efficiency was about 80%); images were acquired from fixed samples 24 h post transfection. Bar: 50 μm.

D. Relative viable cell number of samples transfected to express different alleles of *sidl*. 293T cells transfected to express the indicated proteins for 24 h were lifted and viable cells were enumerated. Data were expressed in relative to samples transfected with the vector set at 100%. Similar results were obtained from at least three independent experiments each done in triplicate.
**P < 0.004.**

the Cya fusion assay (Bardill *et al.*, 2005), to determine whether Sidl is an *L. pneumophila* effector delivered into host cells by the Dot/Icm system. Dot/Icm-dependent transfer of Sidl was readily detected by all three assays (Fig. 2A–E), indicating that this protein is a substrate of this type IV protein transfer system. We also used the saponin fractionation method (VanRheenen *et al.*, 2006) to further examine the translocation of Sidl into infected cells during infection. Sidl was detected in soluble fraction from cells infected by the wild-type *L. pneumophila* but not by the *dotA* mutant or the *sidl* deletion mutant (Fig. 2F first lane), indicating that Sidl is delivered into host cytosol by the bacterium in a Dot/Icm-dependent manner.

We examined the expression pattern of *sidl* in relation to bacterial growth phase. Interestingly, unlike most characterized substrates of the Dot/Icm transporter whose expression is induced at postexponential phase, expression of *sidl* appears to be induced in cells of exponential

growth phase (Fig. 2G and H), suggesting the transferring of this protein into host cell when bacteria are actively replicating (Liu *et al.*, 2008).

In order to determine whether this gene is required for intracellular replication of *L. pneumophila*, we constructed an in-frame deletion mutant of *sidl* (Fig. 2H last lane). The mutant did not display detectable growth defect in mouse bone marrow-derived macrophages, *Dictyostelium discoideum* or the U937 cell line (Fig. S3 and data not shown), indicating that similar to many other substrates of the Dot/Icm transporter, *sidl* is not essential for intracellular multiplication of *L. pneumophila* in these infection models.

Sidl interacts with components of the host translation elongation machinery

As Sidl shares no detectable homology to proteins in the database nor does it contain motifs suggestive of known

© 2009 Blackwell Publishing Ltd, *Cellular Microbiology*, 11, 911–926

914 X. Shen et al.

biochemical activities, we attempted to determine its biochemical functions by identifying its targets in host cell. To this end, we incubated Affigel beads (Bio-Rad) coated with His₆-Sidl or GST with lysates of U937 cells. After washing with PBS buffer, proteins retained on the beads separated by SDS-PAGE were visualized by silver staining. Several proteins with molecular weights ranging from 40 to 130 kDa were retained by beads coated with His₆-Sidl, but not by beads coated with GST (Fig. 3A lanes 1 and 3). These proteins were identified by mass spectrometry analysis as components of the mammalian protein translation elongation machinery, including eEF1A, eEF1Bα, eEF1Bγ and the valyl tRNA synthetase (Fig. 3A lane 3). As these protein translation factors often are purified as a stable multisubunit complex (Bec et al., 1989), we attempted to identify the direct binding targets of Sidl by incubating purified GST-Sidl with rabbit reticulocyte lysates (RRL) enriched in protein synthesis components and retrieved the tagged protein with glutathione beads. After washing with a more stringent buffer (Experimental procedures), two components of the protein synthesis elongation complex, eEF1A and eEF1Bγ, were copurified with GST-Sidl but not with the control GST-SidF, suggesting that these two proteins interact directly with Sidl (Fig. 3B).

© 2009 Blackwell Publishing Ltd, Cellular Microbiology, 11, 911–926

Fig. 2. Dot/Icm-mediated translocation and growth phase-dependent expression of Sidl.

A. Transfer of Sidl by the Dot/Icm system between bacterial cells. Protein transfer was performed by using either Lp02(Dot/Icm⁺) or Lp03(dotA⁻) expressing a Cre::Sidl fusion as the donor strains and a strain containing a floxed reporter as a recipient (Luo and Isberg, 2004). The transfer of Cre mediated by the effector SidF (Banga et al., 2007) was used as a positive control. Solid bars, donor strain *L.* *pneumophila* Lp02(Dot/Icm⁺); striped bars, donor strain *L.* *pneumophila* Lp03(dotA⁻).

B–D. Sidl-mediated translocation of the transfer-deficient SidCΔC100 into macrophages. Mouse macrophages were infected with bacteria expressing SidC, SidCΔC100 or SidCΔC100::Sidl for 1 h; *L.* *pneumophila* and SidC were differently labelled with distinctive antibodies followed by appropriate secondary antibodies as described (VanRheenen et al., 2006). At least 150 vacuoles in triplet samples were scored in each experiment. Representative images of vacuoles harbouring *sidCΔC100*(C) or *sidCΔC100::sidl* (D); bacteria were labelled in red and SidC was stained in green.

E. Dot/Icm-mediated transfer of Sidl measured by using the pertusis toxin Cya as a reporter. U937 infected with Lp02(Dot/Icm⁺) or Lp03 (dotA⁻) containing *cya-sidl* fusion plasmid were analysed for cAMP production. The translocated effector SidJ (Liu and Luo, 2007) was used as positive control. Solid bars, *L.* *pneumophila* Lp02(Dot/Icm⁺); striped bars, *L.* *pneumophila* Lp03 (dotA⁻).

F. Sidl was translocated into host cells by the Dot/Icm system during infection. U937 macrophages were infected with relevant bacterial strains at an moi of 5 for 6 h and the collected cells were lysed with 0.2% saponin. Soluble proteins precipitated with methanol were separated by SDS-PAGE, transferred to nitrocellulose membranes. Sidl in insoluble fraction was also probed (low panel). The cytosolic protein isocitrate dehydrogenase was probed as a control for bacterial cell lysis. Lanes: 1, wild-type strain Lp02(Dot/Icm⁺); 2, Lp03(dotA⁻); 3, Lp02Δsidl(Dot/Icm⁺); 4, total cell lysate of Lp02(Dot/Icm⁺).

G–H. Expression of *sidI* is induced when *L.* *pneumophila* was grown to exponential phase. *L.* *pneumophila* cultures established by diluting stationary phase cells at 1:20 (starting OD₆₀₀ = 0.2) in fresh AYE broth were grown at 37°C with vigorous shaking. The growth of the bacteria was monitored by measuring the values of OD₆₀₀ at a 2 h interval (G). At indicated time points (hours), equal amounts of cells (estimated by OD₆₀₀ values) were withdrawn and lysates of the samples were separated by SDS-PAGE; proteins transferred to nitrocellulose membranes were probed with anti-Sidl antibodies (H); the *sidl* deletion mutant grown at OD₆₀₀ = 1.8 was loaded as a control (H last lane). The bacterial isocitrate dehydrogenase (ICDH) was detected as loading controls.

---

We further determined the direct binding of Sidl to eEF1A and eEF1Bγ with individually purified proteins. We produced GST fusion proteins of eEF1A, eEF1Bα and eEF1Bγ and each of these proteins was incubated with His₆-Sidl. Glutathione beads were used to capture the GST fusion protein. After extensive wash with the TEN buffer (*Experimental procedures*), we found that Sidl was retained by GST-eEF1A or by GST-eEF1Bγ (Fig. 3C). No

![Figure 3](#fig-3)

Fig. 3. Sidl interacts with components of the mammalian translation elongation machinery.

A. Components of the host protein synthesis machinery were retained by Sidl. Affigel beads coated with GST (lane 1) or His₆-Sidl (lane 3) were incubated with mammalian cell lysates. His₆-Sidl-coated beads incubated with cell lysis buffer (lane 2) was used as another control. After washing with lysis buffer, proteins separated by SDS-PAGE were visualized by silver staining; bands only retained by the His₆-Sidl-coated beads were identified by matrix-assisted laser desorption/ionization/mass spectrometry analysis. Relevant protein size markers (in kDa) were indicated.

B. Copurification of eEF1A and eEF1Bγ with GST-Sidl from RRL. Each GST-tagged protein was incubated with RRL and glutathione-coated beads were used to retrieve GST-Sidl or GST-SidF. After removing unbound proteins by extensive washing, the retained proteins resolved by SDS-PAGE were detected by Coomassie bright blue staining and identified by mass/spectrometry analysis. Relevant markers (in kDa) were indicated.

C. Direct binding of Sidl to eEF1A and eEF1Bγ. His₆-Sidl was incubated with GST-tagged eEF1A, eEF1Bγ or eEF1Bβ, and the protein complex was captured with glutathione beads; retained Sidl was detected by immunoblotting.

D. Sidl* formed complexes with eEF1A or eEF1Bγ in mammalian cells. Lysates of 293T cells transfected to express GFP-Sidl* or GFP-SidF were subjected to immunoprecipitation with an anti-GFP antibody, the precipitates resolved by SDS-PAGE were detected for eEF1A and eEF1Bγ using specific antibodies. 5% (50 μg) of total protein was probed as input controls (lanes 1 and 2). TCL, total cell lysates.

© 2009 Blackwell Publishing Ltd, *Cellular Microbiology*, **11**, 911–926

916 X. Shen et al.

interaction between Sidl and eEF1Bα or eEF1Bβ was detected (Fig. 3C and data not shown). These data indicate that eEF1A and eEF1Bγ are the direct targets of Sidl.

We also determined that the Sidl* still interacts with both eEF1A and eEF1Bγ (see below and data not shown), indicating that the loss of lethality in this mutant is not due to the loss of target binding activity. Thus, we used Sidl* to examine *in vivo* binding of Sidl to its targets. Lysates of 293T cells transfected to express GFP::Sidl* were subjected to immunoprecipitation with an anti-GFP antibody and coprecipitated proteins were detected with antibodies against eEF1A or eEF1Bγ. Although Sidl* was expressed at a low level (Fig. 3D bottom panel first lane), eEF1A and eEF1Bγ was detected in the precipitates (Fig. 3D lane 3). Collectively, these data establish that Sidl specifically binds eEF1A and eEF1Bγ of the translation elongation complex.

---

**Sidl inhibits mammalian protein synthesis**

Our observations that Sidl interacts with components of the eukaryotic translation elongation machinery suggest that this protein modulates host protein synthesis. Thus, we examined the effect of Sidl on protein synthesis of mammalian cells using the RRL system that allows *in vitro* transcription/translation analysis (Promega, L1171). Adding a plasmid containing the *sidF* gene (Banga et al., 2007) led to robust production of SidF. However, inclusion of 180 ng of Sidl completely inhibited the synthesis of SidF (Fig. 4A). The inhibitory effect was dependent upon the activity of Sidl as heat-treated protein failed to block protein synthesis (Fig. 4A). Furthermore, as little as 30 ng of Sidl was able to completely inhibit SidF synthesis (Fig. 4B). Such inhibitory effects were also observed when presynthesized mRNA encoding luciferase was used for analysis (Fig. 4C). The mRNA directed the syn-

---

**Fig. 4. Inhibition of protein synthesis by Sidl.**

A. Sidl abolished *in vitro* synthesis of SidF from plasmid pcDNA3-sidF. Sidl or heat-inactivated Sidl was added in an *in vitro* protein synthesis system (Promega) and the production of SidF was measured by immunoblotting. Lanes: 1, markers (kDa); 2, H₂O; 3, 180 ng Sidl; 4, 540 ng Sidl; 5, 540 ng heat-inactivated Sidl.

B. Dose-dependent inhibition of SidF production. Different amounts of Sidl were added to RRL containing pcDNA3-sidF and the production of SidF was detected by Western blot. The levels of relevant proteins were monitored by immunoblotting with the appropriate antibodies. Actin was detected as a loading control (lowest panel).

C. Inhibition of translation of luciferase mRNA. Fifty nanograms of GST (1), His₆-Sidl (2), heat-inactivated His₆-Sidl (3), His₆-Sidl** (4) or His₆-Sidl* (5) was added to RRL containing luciferase mRNA. Production of light was detected after adding the luciferase assay reagent.

D. Dose-dependent inhibition of protein synthesis by Sidl and its mutants. His₆-Sidl or indicated mutants were added to RRL containing luciferase mRNA and the synthesis of luciferase was monitored by the production of light.

E. Inhibition of *in vivo* protein synthesis by Sidl. Seven hours after transfection, ³⁵S-methionine was added to 293T cells for 3 h and the incorporation of S³⁵ was measured. 1, vector pEGFP-C1; 2, wild-type *sidl*; 3, *sidl*⁸. Similar results were obtained from at least three independent experiments and the data shown are from one representative experiment done in triplicate. ***P < 0.0002; **P < 0.007; *P < 0.05.

© 2009 Blackwell Publishing Ltd, Cellular Microbiology, 11, 911–926
thesis of luciferase in the RRL system, but no light production was detected when active Sidl but not when heat-inactivated Sidl was added. When added at 50 ng, the mutants Sidl* and Sidl** still exhibited partial inhibitory activity (Fig. 4C fourth and fifth bars).

As both Sidl* and Sidl** still exert considerable inhibitory effect, we further analysed their activity in a dose-response experiment. Under our experimental conditions, 1 ng of wild-type Sidl is sufficient to exert 50% inhibition (Fig. 4D). On the other hand, 100 ng of Sidl* (E482K) or 25 ng of Sidl** (R453P) is needed to achieve similar level of inhibition (Fig. 4D). These data indicate that these mutations severely impaired but did not abolish the protein synthesis inhibition activity of Sidl.

Transfection of mammalian cells with a *sidl* expressing plasmid within 10 h did not cause significant cell death (Fig. S4), we thus examined the effect of Sidl on *in vivo* protein synthesis by the ${}^{35}$S-methionine incorporation assay (Belyi et al., 2006). Under our experimental conditions where transfection efficiency was about 80% for the GFP vector or the plasmid carrying *sidl* (data not shown), cells similarly transfected with the *sidl* construct incorporate ${}^{35}$S-methionine at about 35% of those transfected to express GFP (Fig. 4E). Sidl* also detectably affects the incorporation of ${}^{35}$S-methionine, consistent with the partial inhibitory activity observed in *in vitro* experiments (Fig. 4E). Taken together, our data indicate that the direct consequence of binding of Sidl to eEF1A and eEF1Bγ is the inhibition of host protein synthesis. Given the high degree of similarity among protein translation machinery from different eukaryotes, it is clear that the toxicity of Sidl to yeast is due to a similar mechanism. The protein synthesis inhibition activity of Sidl resembles the Lgt proteins that abolish eEF1A activity by glucosylating the G protein (Belyi et al., 2006; 2008). To examine whether Sidl and Lgt1 are functionally redundant, we constructed a *L. pneumophila* mutant lacking both genes, but no growth defect was detected for this double mutant in several hosts examined (Fig. S5).

### Sidl interacts with eEF1A independent of its nucleotide binding status

Despite the apparently similar effect of Sidl and Lgt1 on protein synthesis, extensive mass/spectrometry analysis of eEF1A copurified with active Sidl (Fig. 3B) did not reveal any novel post-translational modification of this protein (data not shown). Like many monomeric G proteins, eEF1A transits between active and inactive forms; depending upon the nucleotides (GTP or GDP) it binds. GTPase-activating proteins or guanine nucleotide exchange factors (GEF) for classic small GTPases often exhibit higher binding affinity to the G protein associated with a specific nucleotide. To determine the biochemical

© 2009 Blackwell Publishing Ltd, *Cellular Microbiology*, 11, 911–926

---

### Fig. 5. Sidl binds eEF1A independent of guanine nucleotide it associates.
A. Sidl has a lower affinity for nucleotide free eEF1A. GST-eEF1A (lane 1), GST-eEF1A that had been dialysed against 5 mM EDTA for 20 min (lane 2) or GST (lane 3) was incubated with His${}_{6}$-Sidl, and the protein complex was captured with glutathione beads. Retained His${}_{6}$-Sidl was detected by immunoblotting.
B. Sidl binds eEF1A independent of the guanine nucleotides. Nucleotide-free GST-eEF1A loaded with non-hydrolysable GDPβS or (lane 1) or GTPγS (lane 2) was incubated with His${}_{6}$-Sidl and the protein complexes were captured and detected as described in (A). The GST protein was used as a control (lane 3). In each case, 5% of the input was detected as controls.
C. Mutants Sidl*(E482K) and Sidl**(R453P) bind eEF1A with affinities comparable to that of wild-type protein. The indicated amounts of His${}_{6}$-Sidl or its mutants were added to wells of immulon-2 plates coated with eEF1A. Sidl or the mutant proteins retained by eEF1A was detected by ELISA.

activity of Sidl, we first tested its interactions with eEF1A associated with different nucleotides. In a GST pull down assay, Sidl exhibits higher affinity for untreated recombinant eEF1A than for its nucleotide free form (Fig. 5A). However, association with GDP or GTP does not affect the binding affinity of eEF1A to Sidl (Fig. 5B). These results indicate that although Sidl has a higher affinity for nucleotide-bound form of eEF1A, it does not preferentially bind eEF1A associated with a specific guanine nucleotide. This property is similar to that of eEF1Bα, the cognate GEF of eEF1A, whose binding constant to the G protein is independent of guanine nucleotides (Olarewaju et al.,
2004). However, in our experiments to examine whether Sidl modulates the nucleotide binding cycle of eEF1A, we did not detect any effect of Sidl on the loading of GTP to eEF1A or the dissociation of GDP from the protein, nor did we detect any significant effect of Sidl on the intrinsic GTPase activity of eEF1A (data not shown). Sidl also does not possess detectable intrinsic GTPase activity because incubation of the protein with \[^{32}\text{P}-\gamma\]-GTP did not lead to detectable release of \({}^{32}\text{PO}_4\) (data not shown).

To determine whether binding to eEF1A by Sidl is sufficient for the protein synthesis inhibition activity, we examined whether Sidl* and Sidl** exhibit any decrease in their binding affinity to eEF1A by an ELISA assay. Similar amounts of Sidl and the mutants were required to saturate-immobilized eEF1A (Fig. 5C), all with a \(K_d\) of about 10 nM, indicating that the mutations did not affect the binding of Sidl to eEF1A. Taken together, these data suggest that physical binding is not sufficient for the inhibitory effect of Sidl on eEF1A. Other activities such as subtle but effective modulation of eEF1A GTP hydrolysis cycle or yet undetected post-translational modifications of the target proteins may be responsible for its function.

### Sidl contributes to the induction of host stress response during *L. pneumophila* infection

Recent studies indicate that in addition to its canonical function in protein synthesis, eEF1A is involved in the regulation of diverse cellular processes including cell death, cytoskeleton structure and stress response (Gross and Kinzy, 2005; Borradaille et al., 2006; Shamovsky et al., 2006). Infection of wild-type *L. pneumophila* in both amoeba and mammalian hosts results in a robust induction of stress response genes (Farbrother et al., 2006; Losick and Isberg, 2006). Moreover, a recent study indicates that eEF1A is essential for the activation of heat shock factor 1 (HSF1), the major stress response regulatory protein in mammalian cells (Shamovsky et al., 2006). Therefore, we examined whether HSF1 is activated during *L. pneumophila* infection and, if so, whether the activity of Sidl contributes to such activation. The activation of HSF1 was monitored by the formation of an HSF1/eEF1A protein complex detected by coimmunoprecipitation (Shamovsky et al., 2006). A large amount of the HSF1/eEF1A complex was detected in cells subjected to heat shock at 42°C for 20 min (Fig. 6A lane 5). Basal-level association between these two proteins was also detected in resting cells or cells infected with a Dot/Icm-deficient *L. pneumophila* strain (Fig. 6A lanes 1 and 2). The lack of HSF1 activation in infections using the dotA mutant was not due to degradation of internalized bacteria because these bacteria were viable by the time the cells were lysed (Fig. S6). Importantly, a high level of HSF1/eEF1A complex was detected in cells infected with wild-type bacteria (Fig. 6A lane 3). Furthermore, the amount of the protein complex was reproducibly decreased in cells infected with the *sidl* mutant but not the *lgt1* mutant (Fig. 6A lanes 4 and 6). Expression of Sidl from a plasmid partially restored the reduction (Fig. 6A lane 7). Interestingly, the formation of the eEF1A/HSF1 complex also occurred in cells transfected to transiently express Sidl but not the Sidl* mutant (Fig. 6B lanes 2 and 3).

Under heat shock conditions, the activation of HSF1 by eEF1A and a non-coding RNA molecule enhances the binding of HSF1 to the heat shock element (HSE), thus leading to induction of target gene expression (Shamovsky et al., 2006). To determine whether infection by *L. pneumophila* has similar effect on HSF1, we examined the DNA binding activity of HSF1 from cells infected with relevant *L. pneumophila* strains by the electrophoretic mobility shift assay. No shift was detected when lysates of uninfected cells or cells infected by the Dot/Icm-deficient strain were used for binding (Fig. 6C lanes 1 and 3). However, retardation of the probe was detected in reactions using lysates from cells infected by wild-type bacteria or cells treated by heat shock (Fig. 6C lanes 2 and 4). Importantly, infection with the *sidl* deletion mutant resulted in no detectable shift (Fig. 6C lane 5), a defect that can be restored partially by expressing Sidl from a plasmid (Fig. 6C lane 6 arrow). The binding of HSF1 to the HSE was specific because the addition of 5× non-labelled probe abolished the shift (Fig. 6C). These results further point to a role of Sidl in the activation of HSF1.

To determine whether the activation of HSF1 leads to induction of *hsp* genes, we examined mRNA levels of *hsp70*, one of the target genes of HSF1, by real-time PCR analysis. Consistent with the observed HSF1 activation, heat shock treatment led to about 18-fold induction of *hsp70* transcription (Fig. 6D). Similarly, expression of the *hsp70* gene was induced about 12-fold in cells infected with wild-type *L. pneumophila* for 8 h, but no induction was observed in cells infected with a *dot/icm*-deficient strain. Such induction detectably decreased in cells infected by the *sidl* deletion mutant (Fig. 6D and fourth bar). Interestingly, transient expression of wild-type Sidl, but not the non-active mutant Sidl*, led to about 15-fold induction of *hsp70* (Fig. 6D), indicating that Sidl-mediated inhibition of protein synthesis led to increased expression of *hsp70*. Unexpectedly, we did not detect significant increase of the Hsp70 protein in cells infected by *L. pneumophila* throughout the infection cycle (Fig. S7). HSF1 is known to regulate the expression of a large number of genes and the gene list is still expanding (Westerheide and Morimoto, 2005); it is possible that proteins encoded by some of these genes are elevated at detectable levels in *L. pneumophila*-infected cells. Taken together, our data suggest that HSF1 is activated by *L. pneumophila* infection and that Sidl contributes to such activation.

Legionella infection and host stress response 919

A
![Image Description](image1.png)

B
![Image Description](image2.png)

C
DNA binding Competition
![Image Description](image3.png)

D
![Image Description](image4.png)

Fig. 6. Sidl contributes to the activation of HSF1 during *L. pneumophila* infection.

A. *L. pneumophila* infection-mediated activation of HSF1. Upper panel, cell lysates of U937 macrophages infected with the relevant bacterial strains at an moi of 5 were subjected to immunoprecipitation with an anti-eEF1A antibody and the precipitated HSF1 was detected after SDS-PAGE. Lanes: 1, uninfected; 2, Lp03(*dot/icm*⁻); 3, Lp02(*dot/icm*⁺); 4, Lp02Δsidl; 5, heat shock; 6, Lp02Δlgt1; 7, Lp02Δsidl/psidl. Middle two panels: eEF1A and HSF1 was probed, respectively, to detect cellular levels of these proteins. Lower panel, quantification of band intensity of upper panel relative to that of the heat shock treated cells set at 100%.

B. Activation of HSF1 by Sidl but not by the Sidl* mutant. Immunoprecipitates were obtained with an eEF1A-specific antibody from lysates of 293T cells transfected for 6 h with the appropriate plasmids. HSF1 was detected as described in (A). Lanes: 1, GFP; 2, Sidl; 3, Sidl*; 4, heat shock.

C. Infection of *L. pneumophila* increases the DNA binding activity of HSF1. Lysates prepared from U937 macrophages infected with relevant bacterial strains were used for assaying the binding of HSF1 to HSE. The probe was detected with an antibody specific for digoxigenin used to label the binding target. Lanes: 1, uninfected; 2, Lp02(*dot/icm*⁺); 3, Lp03(*dot/icm*⁻); 4, heat shock; 5, Lp02Δsidl; 6, Lp02Δsidl/pSidl. Competition experiments received five times more unlabelled probe were performed to determine the specificity of the binding.

D. Induction of *hsp70*. qPCR analysis for *hsp70* was performed with cDNA derived from total RNA of U937 cells infected for 8 h (infection) or 293T cells transfected to express indicated genes for 6 h (transfection). The levels of expression were normalized with mRNA specific for the GAPDH gene. Relative gene expression represents the normalized values of the indicated samples against cDNA of uninfected cells or cells transfected to express GFP. Lanes: 1, uninfected; 2, Lp03(*dot/icm*⁻); 3, Lp02(*dot/icm*⁺); 5, heat shock; 6, Lp02Δlgt1; 7, Lp02Δsidl/psidl; 8, vector expressing GFP; 9, GFP::Sidl; 10, GFP::Sidl*; 11, heat shock. Data shown are representatives of more than three independent experiments.

Discussion

Infection by *L. pneumophila* leads to significant inhibition of host protein synthesis (McCusker et al., 1991). Here we show that Sidl, a substrate of the Dot/Icm transporter, may contribute to such inhibition in a process that results in the induction of the host stress response.

It is well established that activated eEF1A is responsible for loading charged amino acids to the A site of the ribosome during protein synthesis, whereas eEF1Bγ, along with two other components eEF1Bα and eEF1Bβ, is the GEF responsible for activating eEF1A by loading GTP

to this G protein (Browne and Proud, 2002). There is no detectable homology between eEF1A and eEF1Bγ, suggesting that these proteins bind to different regions of Sidl. The observation that the two non-toxic Sidl substitution mutants still bind eEF1A in a manner similar to that of wild-type protein indicates that simple physical association is insufficient for Sidl to exert its full inhibitory activity. Whether the residual inhibitory activity of Sidl* and Sidl** (Fig. 4C–D) results from their interactions with eEF1Bγ remains to be studied. Sidl could modulate the GTP hydrolysis cycle of eEF1A; our inability to detect such effect may result from the extremely low intrinsic GTPase

© 2009 Blackwell Publishing Ltd, *Cellular Microbiology*, 11, 911–926

activity of purified eEF1A (Crechet and Parmeggiani, 1986). Equally possible is that Sidl biochemically modifies eEF1A or eEF1Bγ at post-translational level. However, despite extensive effort, we did not detect novel post-translational modification on eEF1A associated with Sidl (Fig. 3B). The identification of novel post-translational modifications on eEF1A is complicated by its natural extensive and dynamic modifications (Lamberti et al., 2004).

Given its toxicity to cells when overexpressed, Sidl could contribute to the demise of host cells during *L. pneumophila* infection, although active killing of the host cells apparently would be detrimental to an intracellular pathogen. Three lines of evidence suggest that Sidl plays a role in the induction of the host heat shock genes associated with *L. pneumophila* infection. First, deletion of *sidl* but not *lgt1* led to a reduction in the activation of HSF1 in infected cells (Fig. 6). Second, transient expression of Sidl induced HSF1 activation (Fig. 6B). Although the binding of HSF1 to HSE was not detected in cells infected by the *sidl* mutant (Fig. 6C), active HSF1 clearly existed in these cells (Fig. 6A and D). These discrepancies may result from the sensitivity of the assays employed in these experiments. According to a current model, one of the most direct effect of heat shock or other stress is a drastic decrease in protein synthesis rate that leads to the release of eEF1A for the activation of HSF1 (Shamovsky et al., 2006). Interestingly, perturbations that interfere with protein synthesis often lead to stress resistance. For example, in yeast mutations in gene coding for eEF1Bγ result in constitutive resistance to oxidative stress (Olawewaju et al., 2004). The protein synthesis inhibitory activity of Sidl could exert similar effect to make eEF1A available for the activation of HSF1. Finally, the expression of *sidl* is induced when the bacterial cells reach exponential growth phase, concomitant with the occurrence of *hsp* induction during infection (Fig. 2H; Losick and Isberg, 2006), further suggesting a role of Sidl in this process. Deletion of *sidl* does not completely abolish the induction of stress response in *L. pneumophila*-infected cells (Fig. 6), suggesting the presence of other effectors in this process. In agreement with this notion, two additional proteins targeting eEF1A were recently reported (Belyi et al., 2008).

Stress shock response is a fundamental mechanism necessary for cell survival under a variety of unfavourable conditions, such as cellular stress induced by physical, chemical or biological agents. Accordingly, the induction of *hsp* genes is very likely beneficial to *L. pneumophila* replication. For example, the chaperone activity of Hsps may be important for the proper folding of relevant host proteins or *L. pneumophila* effectors delivered by the Dot/Icm transporter. Alternatively, as Hsps also are involved in important cellular processes such as cell death and resis-

tance to ER stress, it is possible that Hsps-mediated modulation of these pathways is important for intracellular growth of *L. pneumophila* (Losick and Isberg, 2006). Taken together, we postulate that protein synthesis inhibition mediated by Sidl plays a role in induction of the host stress response, possibly to provide a cellular environment more conducive for bacterial replication. The induction of a host heat stress response has been documented in infections caused by a variety of pathogens and its roles in infections can be either beneficial or detrimental to the infecting agents (Glotzer et al., 2000; Singh and Aballay, 2006). Clearly, different pathogens have evolved different strategies to modulate this cellular pathway.

The common activities of Lgt proteins and Sidl in inhibiting host protein synthesis indicates that *L. pneumophila* had evolved more than one mechanism to modulate the activity of eEF1A, which may represent one example of the functional redundancy among virulence factors of *L. pneumophila* (Luo and Isberg, 2004). Recent studies indicate that eEF1A is involved in several non-canonic cellular processes, including the activation of HSF1 (Shamovsky et al., 2006), the TCTP-mediated tumour proliferation (Cans et al., 2003) and the resistance to cell death induced by lipotoxin or ER stress (Talapatra et al., 2002; Borradaille et al., 2006). It is possible that regulation of eEF1A activity by Sidl and other bacterial factors also alters other host cellular processes necessary for bacterial intracellular growth. That a mutant lacking both Lgt1 and Sidl did not detectably affect intracellular replication of *L. pneumophila* suggests the existence of more *L. pneumophila* effectors dedicated to modulate cellular pathways regulated by this multifunctional protein.

## Experimental procedures

### Bacterial, yeast strains and plasmid construction

All *L. pneumophila* strains used in this study were derivatives of the Philadelphia 1 strain Lp02 (Berger and Isberg, 1993). Bacteria were grown and maintained on CYE medium as previously described (Conover et al., 2003). In complementation experiments, the vector used for expressing the gene of interest was introduced into the wild-type strain or mutants and the resulting strains were used for infection. *E. coli* strains were grown and maintained on LB agar or LB broth, when necessary antibiotics were included as described (Conover et al., 2003). Following a standard procedure (Dumenil and Isberg, 2001), we generated the *sidl* in-frame deletion mutant ZL07 with plasmid pZLΔsidl, which was constructed by ligating into Sacl/Sall-digested pSR47s (Berger and Isberg, 1993) two PCR products amplified from Lp02 genomic DNA using Sidlup5′Sall/Sidlup3′BamHI (Table S3), Sidld5′BglII/Sidld3′Sacl (digested with BglII/Sacl). To complement the mutation, the coding region of *sidl* was amplified using SidlCom5′Sacl/Sidl3′Sall and the PCR product was inserted into pJB908 (Bardill et al., 2005) as a Sacl/Sall fragment. The plasmid used for deleting Lgt1 was constructed similarly with primer pairs PL301/PL302 and PL303/PL304.

infections were conducted with bacterial cultures grown to the postexponential phase as judged by optical density of the cultures (OD<sub>600</sub> = 3.3–3.8) as well as an increase of bacterial motility. Sidl and its derivatives were cloned into pEGFPC1 (Clontech) for expression in mammalian cells. cDNA clones of used mammalian genes were amplified from a human kidney cDNA library (Clontech) or from clones purchased from ATCC. All genes were verified by sequencing analysis. The sequences of all primers used are in Table S3.

### Yeast manipulation and screening of L. pneumophila proteins toxic to eukaryotic cells

Yeast strains were PJ69-4A (James *et al.*, 1996) or W303 (Fan *et al.*, 1996). Yeast was grown in YPD medium or in appropriate amino acid dropout minimal media. Full-length genes larger than 350 amino acids of *L. pneumophila* were individually cloned into pGBKT7 (BD Biosciences) as a BamHI/Sall or BgIII/Sall fragment and the resulting plasmids was introduced into yeast strain PJ69-4A (James *et al.*, 1996) by a standard transformation protocol (Gietz *et al.*, 1995). After 3 days of incubation at 30°C, plasmids that resulted in few or no transformants were retained for further analysis. For the determination of expression-dependent toxicity, Sidl or its derivatives Sidl* and Sidl** was cloned into pSB157 (Fazzio and Tsukiyama, 2003) (courtesy of Sue Biggins, Fred Hutchinson Cancer Research Center, Seattle, WA) and the resulting plasmids were linearized with an appropriate restriction enzyme and then transformed into yeast strain W303 (Fan *et al.*, 1996). Several transformants were analysed for galactose-dependent toxicity by spotting serially diluted cultures grown in glucose medium. To prepare cell lysates for protein analysis, cells from 5 ml overnight cultures were first lysed with a cracking buffer [40 mM Tris-Cl (pH 6.8), 5% SDS, 0.1 mM EDTA, 8 M urea, bromothymol Blue 0.4 mg ml<sup>-1</sup>] with glass beads. Samples were resolved by SDS-PAGE after adding Laemmli buffer.

### Random mutagenesis of sidl

DNA of plasmid pSB3 (Table S2) was treated with hydroxylamine essentially as described (Luo and Farrand, 1999). Five micrograms of plasmid DNA was incubated in a 100 μl reaction mixture containing 0.5 M hydroxylamine, 0.5 mM Na<sub>2</sub>EDTA and 5 mM Tris-HCl (pH 6.0) either for 10 h at 37°C or for 1 h at 65°C. Treated DNA was purified with an affinity column (Zymo), dissolved in ddH<sub>2</sub>O and was introduced into the yeast strain PJ69-4A (James *et al.*, 1996). Transformants were plated onto Trp dropout plates and surviving colonies were purified. Protein samples prepared from 5 ml culture were resolved by SDS-PAGE, transferred to nitrocellulose membranes and detected by immunoblotting with an anti-Sidl antibody. Clones expressing full-size Sidl were further verified by cloning the mutated alleles of *sidl* into new vector. The mutations were identified by double-strand DNA sequencing analysis.

### Protein purification

Native eEF1A was purified from rat livers as described (Shamovsky *et al.*, 2006). To express and purify GST-Sidl and His<sub>6</sub>

Sidl, the open reading frame of this gene was amplified by PCR using SidIP5′BamHI/SidIF3′Sall and the resulting DNA fragment was inserted into pGEX-6P-1 (GE Healthcare) or pQE30 (Qiagen). GST-Lgt1, GST-eEF1A, GST-eEF1Bα, GST-eEF1Bβ and GST-eEF1Bγ were expressed similarly from pGEX-6P-1 (GE Healthcare). For protein production, bacteria were grown at 37°C in LB medium (100 mg ml<sup>-1</sup> Ampicillin) to an OD<sub>600</sub> of 0.5, shifted to 22°C and then induced with 0.2 mM isopropylthio-D-galactopyranoside, and cultivated for an additional 15 h at 22°C. Harvested cells were lysed in a French press at 1500 psi, and the soluble fraction obtained by centrifugation at 6000 g for 10 min at 4°C was incubated with glutathione Sepharose resin or Ni<sup>2+</sup> resin (Amersham Pharmacia) equilibrated with PBS in order to purify recombinant affinity-tagged proteins. We washed the resin with 20 times of the column volume with PBS; for Ni<sup>2+</sup> resin, imidazole was included at 20 mM in the washing buffer to facilitate the removal of non-specifically bound proteins. The GST tag was removed by incubation with PreScission Protease (100 U) (Amersham Pharmacia) for 18 h at 4°C, and recombinant proteins were eluted from the glutathione resin with PBS. Bound His<sub>6</sub>-tagged proteins were eluted with PBS buffer containing 200 mM imidazole. After dialysing against PBS to remove imidazole, the protein was further purified by gel filtration with an FPLC system using a Superdex 200 10/300 GL column (GE Healthcare). TBST buffer (50 mM Tris-Cl, 150 mM NaCl, 0.1% Triton-X100, pH 7.4) was used as eluent and the flow rate was set at 0.4 ml min<sup>-1</sup>. The single peak corresponding to the protein was collected, dialysed in the appropriate buffer for subsequent use. Protein concentrations were determined by the Bradford assay; the purity of all proteins was more than 95% as assessed by SDS-PAGE followed by Coomassie bright blue staining (Fig. S2B).

### Cell culture and transfection

U937 cells were cultured in RPMI medium supplemented with 10% foetal bovine/calf serum and 5 mM glutamate. 293T cells were cultured in Dulbecco’s modified minimum Eagle’s medium (DMEM) supplemented with 10% foetal bovine/calf serum. When necessary, U937 cells were differentiated into macrophages by incubating at 37°C with 5% CO<sub>2</sub> for 36–48 h with 50 ng ml<sup>-1</sup> phorbol myristic acid. Cells grown to about 80% confluence were transfected with Lipofectamine 2000 (Invitrogen) following manufacturer’s instructions. Mouse macrophages were prepared from bone marrow of female A/J mice of 6–10 weeks of age following published protocols (Swanson and Isberg, 1995). For infection, macrophages were plated into 24-well plates at 2 × 10<sup>5</sup> cell per well. Infection was carried out at the indicated multiplicity of infection (moi) as required by the particular experiments. In all cases, 1 h after adding bacteria to cultured cells, infection was synchronized by washing the infected cells three times with warm PBS buffer.

### Affinity chromatography from cell lysates and in vitro protein binding

We coated Affigel beads using purified His<sub>6</sub>-Sidl with protocols supplied by the manufacturer (Bio-Rad). U937 cells collected from 500 ml culture were lysed in a glass homogenizer Dounce (Wheaton) with 0.5 ml PBS supplemented with 5 mM DTT and
© 2009 Blackwell Publishing Ltd, *Cellular Microbiology*, **11**, 911–926

protease inhibitor cocktail (Roche). After removing unbroken cells and nuclei by centrifugation at 10 000 g for 10 min at 4°C, the postnuclear supernatant was added to beads and incubated for 14 h at 4°C. Beads were washed five times with PBS and bound proteins were treated with SDS sample buffer. Proteins resolved by SDS-PAGE were visualized by silver staining (Bio-Rad). Individual protein gel bands were excised, digested with trypsin and analysed by matrix-assisted laser desorption/ionization/mass spectrometry (Taplin Biological Mass Spectrometry Facility, Harvard Medical School).

For GST pull down experiments from RRL, 5 μg purified GST fusion protein was mixed with 200 μl RRL on a rotator for 3 h at 4°C and 100 μl pre-washed glutathione beads were added to the reactions. After another 2 h of incubation at 4°C, the beads were washed three times with TEN buffer [100 mM Tris-Cl (pH 8.0), 10 mM EDTA, 200 mM NaCl]. Proteins associated with beads were solubilized with SDS sample buffer and were separated by SDS-PAGE. Proteins were detected by coomassie bright blue staining.

To analyse protein-protein interactions with purified proteins, 2 μg purified GST-fusion protein was mixed with 2 μg His-tagged protein in PBS for 1 h at 4°C. After adding 50 μl of pre-washed glutathione beads, binding was allowed to proceed for 1 h. The beads were then washed five times with TEN buffer containing 400 mM NaCl. Retained proteins were detected by immunoblot after SDS-PAGE.

### ELISA

To compare the binding affinity between wild-type Sidl protein and its point mutant forms, His-tagged proteins were purified from bacteria first using Ni-NTA affinity column, and then by passing through gel filtration column (Superdex 200 10/300 GL). For ELISA, immulon-2 96-well plate was coated overnight with purified GST-EF1A (75 ng well⁻¹) in bi-carbonate buffer (pH 9.6). The wells were subsequently blocked with 5% BSA for 1 h at 30°C. Purified Sidl proteins were then added at different concentrations ranging from 0.01 to 2000 nM to the wells and the plate incubated at 25°C for 90 min to allow binding of proteins. Wells were then washed 5× with PBS containing 0.1% Tween 20 and incubated with affinity-purified Sidl antibody (1:1000) at 37°C for 1 h. The plate was again thoroughly washed 5× with above PBS wash buffer followed by addition of 1:10 000 dilution of secondary antibody conjugated with horseradish peroxidase. After incubation at 37°C for 1 h, the wells are washed 5× and developed with 1-Step Ultra-TMP-ELISA (Pierce) substrate. The reactions were terminated with 2 M sulphuric acid as per manufacturer’s instructions and quantified by measuring absorbance at 450 nm. For the controls, wells coated with BSA protein were subjected to binding with Sidl protein.

### Cell viability assay

Viability of 293T cells transfected to express Sidl, Sidl* or GFP was measured using the trypan blue dye according to a published protocol (Freshneyy, 2007). Briefly, 10 h after transfection, cells were resuspended in a concentration of about 10⁶ cells ml⁻¹ and were diluted 1:1 with a 0.4% trypan blue solution. The number of stained cells and total number of cells were determined using a hemocytometer. The percentage of viable cells (unstained) was calculated.

---

In vitro protein synthesis

For transcription/translation analysis, we added 0.1 μg DNA of a plasmid carrying sidF (Banga et al., 2007) to 10 μl of the TNT system (Promega, L1171). For protein translation from pre-made mRNA, mRNA coding for the firefly luciferase was added to the RRL system (Promega, L4960) at a final concentration of 0.03 μg μl⁻¹. In each case, the reactions were allowed to proceed for 2 h at 30°C. Purified Sidl or its mutants were added to the reactions at indicated concentrations. The production of proteins was detected by Western blot or by measuring production of light with the luciferase assay reagent (Promega) substrate following by detection with a luminometer (Thermo Scientific).

### In vivo protein synthesis

To measure the effect of Sidl on de novo protein synthesis, we followed a published protocol (Belyi et al., 2006). Briefly, we transfected 293T cells seeded in 24-well plates with plasmids carrying gfp, sidl or sidl* with Lipofectamine 2000 according to instructions of the manufacturer (Invitrogen). After incubating at 37°C for 6 h, cells were washed with DMEM-Met (Invitrogen) and were incubated for another 1 h. The protein synthesis was then chase-labelled with ³⁵S-methionine (0.5 Ci well⁻¹) (Perkin Elmer) for 3 h in DMEM-Met medium. Cells were then washed with PBS for five times and were lysed by 0.1% SDS supplemented with 0.2 mg ml⁻¹ BSA. Proteins were precipitated by 10% trichloroacetic acid, filtered onto nitrocellulose membranes. The incorporated ³⁵S-methionine was measured by liquid scintillation counting.

### Immunoprecipitation

We seeded 293T cells on 100 mm plates at 5 × 10⁵ cells per plate 1 day before transfection, and then transiently transfected cells with indicated plasmids. At the time points required by the experiments, we washed the cells with ice-cold PBS and lysed them with a solution containing Tris (20 mM, pH 8.0), NaCl (150 mM), 0.2% NP-40 (w/v), 10% glycerol (w/v), NaF (100 mM), PMSF (1 mM), sodium orthovanadate (Na₃VO₄)(1 mM), aprotinin (20 μg ml⁻¹) and leupeptin (40 μg ml⁻¹). After incubation on ice for 30 min, cell lysates were centrifuged at 12 000 g for 15 min at 4°C. To prepare immunoprecipitates, we incubated cell lysates with an appropriate antibody and protein G agarose (Roche) for 7–12 h at 4°C. We then washed the agarose beads five times with lysis buffer, and solubilized adsorbed proteins by boiling the washed beads for 5 min in SDS-PAGE loading buffer. Proteins of interest were detected by Western blot after SDS-PAGE. To detect the second protein, membranes were treated with a stripping buffer (2% SDS, 100 mM β-mercaptoethanol, 62.5 mM Tris-Cl, pH 6.8) for 30 min at 70°C and the proteins were detected by appropriate antibodies. For coimmunoprecipitation using lysates from infected cells, 5 × 10⁷ U937 macrophages seeded in 100 mm Petri dish were infected at an moi of 5 for 8 h with indicated L. pneumophila strains. Cell lysates and precipitates were prepared as described above.

### Antibodies, immunostaining and Western blot

To prepare anti-Legionella antisera, we first fixed bacterial cells of strain Lp03 with 0.5% paraformaldehyde in PBS for 30 min at

© 2009 Blackwell Publishing Ltd, Cellular Microbiology, 11, 911–926
37°C. After extensively washing the cells with PBS (10 times with 50× volume of the pellet), the cells were used to immunize rat or rabbit following standard protocols (Pocono Rabbit Farm and Laboratory, Canadensis, PA). Sidl specific antibodies were prepared by injecting rabbits with purified proteins with a similar procedure by the same service provider. When necessary, antibodies were affinity-purified against the same proteins covalently coupled to an Affigel matrix (Bio-Rad) using standard protocols (Dumenil and Isberg, 2001). Monoclonal and polyclonal antibodies against GFP, eEF1A, eEF1Bγ were purchased from Sigma, Millipore and Bethyl Laboratories (Montgomery, TX) respectively. For immunostaining, anti-Legionella antisera were used at 1:10 000; purified anti-SidC was used at 1:500. Cell fixation, permeabilization and immunostaining were performed as described (Conover *et al.*, 2003). Intracellular bacteria were distinguished from extracellular bacteria by differential immunostaining with anti-*L*. *pneumophila* antibodies.

For Western blots, samples resolved by SDS-PAGE were transferred onto nitrocellulose membranes. After blocking with 4% milk, membranes were incubated with the appropriate primary antibody: anti-Sidl, 1:1000; anti-eEF1A, 1:500; anti-GFP, 1:50 000; anti-eEF1Bγ, 1:2000; anti-actin, 1:1000. Serum specific for *Bacillus subtilis* isocitrate dehydrogenase was generously provided by A. L. Sonenshein, Tufts University Medical School, Boston, MA and was used at 1:10 000. Horseradish peroxidase conjugated secondary antibodies and enhanced bioluminescence reagents were used to detect the signals (Pierce, Rockford, IL). Alternatively, membranes were incubated with an appropriate IRDye infrared secondary antibody (Li-Cor’s Biosciences Lincoln, Nebraska, USA) and the signals were detected, and if necessary, the intensity of the bands is quantified by using the Odyssey infrared imaging system.

### Immunofluorescence microscopy

Immunofluorescence microscopy studies on BMMs were performed and processed as described (Swanson and Isberg, 1995). SidC (Luo and Isberg, 2004) were labelled by using affinity-purified polyclonal antibody followed by FITC-conjugated goat anti-rabbit antibody (Jackson LAB). Rat anti-Legionella antibodies generated by immunizing a rat with fixed *L*. *pneumophila* (Pocono Rabbit Farm and Laboratory) were used to label the bacterium. Samples were inspected and scored with an Olympus IX-81 fluorescence microscope.

### Guanine nucleotide-dependent protein binding assay

GST-EF1A was purified from bacterial lysate using glutathione affinity column. The purified protein was dialysed in 5 mM EDTA for 20 min on ice to generate nucleotide-free eEF1A. To determine the affinity of Sidl for the nucleotide-free and bound form of eEF1A, the EDTA-treated or untreated GST-eEF1A (5 μg each) was incubated with 10 μg of gel filtration column (Superdex 200 10/300 GL) purified His~6~-Sidl at 30°C for 30 min. Washed GST-resin was then added and the tubes were rotated at 4°C for 90 min. The GST beads were washed five times in wash buffer (PBS containing 1% Triton) and solubilized in Laemmli buffer. For control purified GST protein was incubated with purified His-Sidl. To detect binding preference of Sidl between GTP- and GDP-bound eEF1A, the dialysed GST-eEF1A (5 μg) was first loaded with 100 μM non-hydrolysable GTPγS or GDPβS in 20 mM MgCl₂ at 4°C for 30 min followed by 10 min at 30°C. The nucleotide-bound GST-EF1A was then incubated with 10 μg of gel filtration column purified His~6~-Sidl as described above. The samples were run on 10% SDS-PAGE gel for Western blot analysis.

### Electrophoretic mobility shift assay

Differentiated U937 macrophages were infected with wild-type Lp02, the *dotA* mutant Lp03 or the *sidl* mutant at an moi of 5 for 8 h. Infected cells were collected, frozen immediately and stored at –80°C. The cell pellet was resuspended in a buffer containing 20 mM Hepes (pH 7.9), 25% glycerol, 0.42 M NaCl, 1.5 mM MgCl₂, 0.2 mM EDTA, 0.5 mM PMSF and 0.2 mM DTT, kept on ice for 20 min with intermittent mixing and centrifuged at max speed for 10 min at 4°C. The total protein content of the cell lysates was measured using Bradford assay and was adjusted to a concentration of 1.0 μg μl⁻¹ and 1.0 μg protein from each treatment was mixed with 0.8 ng digoxigenin-labelled probe (obtained by annealing 5′-GATCTCGGCTGGAATATCCCCGACCTGGCA GCCGA-3′ and 5′-GATCTCGGCTGCCAG GTCGGGAATATTCC AGCCGA-3′). The volume of the binding reactions was brought to 20 μl with deionized water and binding was allowed to proceed for 15 min at 25°C. For competition experiments, five times more unlabelled probe was added to the same reactions. Gel loading, electrophoresis and signal detection were performed in accordance with protocols of the DIG Gel Shift Kit (Roche). Uninfected and heat-shocked cells were used as appropriate controls and treated with the same procedures.

### Real-time PCR

We infected differentiated U937 with wild type, the *dotA* mutant and the *sidl* deletion mutant at an moi of 5 for 8 h. For transfection samples, we transfected 80% confluent 293T cells with plasmids harbouring *gfp*, *gfp-sidl* or *gfp-sidl*⁺ for 6 h. In all experiments, resting cells and cells that had been heat-treated at 42°C for 15 min were used as controls. After washing collected cells with PBS, we isolated total RNA using the RNAqueous-4PCR (Ambion) following instructions supplied by the manufacturer. One microgram of RNA was used to synthesize cDNA with the RETROscript kit following instructions of the manufacturer (Ambion). Two microlitres of the products was used for qRT-PCR with the Absolute QPCR SYBR Green ROX mix (Thermo Scientific). To measure the *hsp70* mRNA levels, primers 5′-GCCAACAAAGATCACCATCAC-3′ and 5′-TTTGTACTTCTCCGC CTCCT-3′, designed to amplify a 150 bp DNA fragment, were used and 70 nmol of the primers was used for each reaction. The GAPDH genes were used for internal control with primers 5′-TTGCCATCAATGACCCCTTCA-3′ and 5′-CGCCCCACTTG ATTTTGGAA-3′.

### Data collection and statistic analyses

All experiments involved in quantitative data were performed in triplicate for at least three independent times; statistic analysis was carried out with Student’s *t*-test.

### Gene accession number

The gene described in this manuscript is lpg2504 with an accession number of YP_096511 in the Genebank.

© 2009 Blackwell Publishing Ltd, *Cellular Microbiology*, **11**, 911–926
Author contributions

X.S., S.B. and Z.-Q.L. designed research; X.S., S.B., L.X., P.G., Y.L. and Z.-Q.L. performed research; X.S., S.B. and Z.-Q.L. analysed data; I.S. and E.N. contributed novel reagents and analytical tools and Z.-Q.L. wrote the paper.

Acknowledgements

We thank Drs Ruben Aguilar (Purdue University, IN), Sue Biggins and Marion Dorer (Fred Hutchinson Cancer Research Center, Seattle, WA) for yeast strains, plasmids and helpful discussion for yeast work. We also thank Dr Arthur Aronson for critical reading of the manuscript. This work was supported by Grant 0535451Z from the American Heart Association and National Institute of Health Grants R01AI069344, R03AI073326 (Z.-Q.L.) and R01GM069800 (E.N.)

References

Abu-Zant, A., Jones, S., Asare, R., Sutelles, J., Price, C., Graham, J., and Kwaik, Y.A. (2007) Anti-apoptotic signalling by the Dot/Icm secretion system of *L. pneumophila*. Cell Microbiol 9: 246–264.

Banga, S., Gao, P., Shen, X., Fiscus, V., Zong, W.X., Chen, L., and Luo, Z.Q. (2007) *Legionella pneumophila* inhibits macrophage apoptosis by targeting pro-death members of the Bcl2 protein family. Proc Natl Acad Sci USA 104: 5121–5126.

Bardill, J.P., Miller, J.L., and Vogel, J.P. (2005) IcmS-dependent translocation of SdeA into macrophages by the *Legionella pneumophila* type IV secretion system. Mol Microbiol 56: 90–103.

Bec, G., Kerjan, P., Zha, X.D., and Waller, J.P. (1989) Valyl-tRNA synthetase from rabbit liver. I. Purification as a heterotypic complex in association with elongation factor 1. J Biol Chem 264: 21131–21137.

Belyi, Y., Niggeweg, R., Opitz, B., Vogelsgesang, M., Hippenstiel, S., Wilm, M., and Aktories, K. (2006) *Legionella pneumophila* glucosyltransferase inhibits host elongation factor 1A. Proc Natl Acad Sci USA 103: 16953–16958.

Belyi, Y., Tabakova, I., Stahl, M., and Aktories, K. (2008) Lgt: a family of cytotoxic glucosyltransferases produced by *Legionella pneumophila*. J Bacteriol 190: 3026–3035.

Berger, K.H., and Isberg, R.R. (1993) Two distinct defects in intracellular growth complemented by a single genetic locus in *Legionella pneumophila*. Mol Microbiol 7: 7–19.

Borradaille, N.M., Buhman, K.K., Listenberger, L.L., Magee, C.J., Morimoto, E.T., Ory, D.S., and Schaffer, J.E. (2006) A critical role for eukaryotic elongation factor 1A-1 in lipotoxic cell death. Mol Biol Cell 17: 770–778.

Browne, G.J., and Proud, C.G. (2002) Regulation of peptide-chain elongation in mammalian cells. Eur J Biochem 269: 5360–5368.

Campodonico, E.M., Chesnel, L., and Roy, C.R. (2005) A yeast genetic system for the identification and characterization of substrate proteins transferred into host cells by the *Legionella pneumophila* Dot/Icm system. Mol Microbiol 56: 918–933.

Cans, C., Passer, B.J., Shalak, V., Nancy-Portebois, V., Crible, V., Amzallag, N., et al. (2003) Translationally controlled tumor protein acts as a guanine nucleotide dissociation inhibitor on the translation elongation factor eEF1A. Proc Natl Acad Sci USA 100: 13892–13897.

Conover, G.M., Derre, I., Vogel, J.P., and Isberg, R.R. (2003) The *Legionella pneumophila* LidA protein: a translocated substrate of the Dot/Icm system associated with maintenance of bacterial integrity. Mol Microbiol 48: 305–321.

Crechet, J.B., and Parmeggiani, A. (1986) Characterization of the elongation factors from calf brain. 3. Properties of the GTPase activity of EF-1 alpha and mode of action of kirromycin. Eur J Biochem 161: 655–660.

Dorer, M.S., Kirton, D., Bader, J.S., and Isberg, R.R. (2006) RNA interference analysis of *Legionella* in Drosophila cells: exploitation of early secretory apparatus dynamics. PLoS Pathog 2: e34.

Dumenil, G., and Isberg, R.R. (2001) The *Legionella pneumophila* IcmR protein exhibits chaperone activity for IcmQ by preventing its participation in high-molecular-weight complexes. Mol Microbiol 40: 1113–1127.

Fan, H.Y., Cheng, K.K., and Klein, H.L. (1996) Mutations in the RNA polymerase II transcription machinery suppress the hyperrecombination mutant hpr1 delta of *Saccharomyces cerevisiae*. Genetics 142: 749–759.

Farbrother, P., Wagner, C., Na, J., Tunggal, B., Morio, T., Urushihara, H., et al. (2006) Dictyostelium transcriptional host cell response upon infection with *Legionella*. Cell Microbiol 8: 438–456.

Fazzio, T.G., and Tsukiyama, T. (2003) Chromatin remodeling in vivo: evidence for a nucleosome sliding mechanism. Mol Cell 12: 1333–1340.

de Felipe, K.S., Pampou, S., Jovanovic, O.S., Pericone, C.D., Ye, S.F., Kalachikov, S., and Shuman, H.A. (2005) Evidence for acquisition of *Legionella* type IV secretion substrates via interdomain horizontal gene transfer. J Bacteriol 187: 7716–7726.

Freshneyy, R.I. (2007) Culture of Animal Cells: A Manual of Basic Technique. New York: Wiley-Liss.

Gietz, R.D., Schiestl, R.H., Willems, A.R., and Woods, R.A. (1995) Studies on the transformation of intact yeast cells by the LiAc/SS-DNA/PEG procedure. Yeast 11: 355–360.

Glotzer, J.B., Saltik, M., Chiocca, S., Michou, A.I., Moseley, P., and Cotten, M. (2000) Activation of heat-shock response by an adenovirus is essential for virus replication. Nature 407: 207–211.

Gross, S.R., and Kinzy, T.G. (2005) Translation elongation factor 1A is essential for regulation of the actin cytoskeleton and cell morphology. Nat Struct Mol Biol 12: 772–778.

Heidtman, M., Chen, E.J., Moy, M.Y., and Isberg, R.R. (2009) Large-scale identification of *Legionella pneumophila* Dot/Icm substrates that modulate host cell vesicle trafficking pathways. Cell Microbiol 11: 230–248.

Ingmundson, A., Delprato, A., Lambright, D.G., and Roy, C.R. (2007) *Legionella pneumophila* proteins that regulate Rab1 membrane cycling. Nature 450: 365–369.

James, P., Halladay, J., and Craig, E.A. (1996) Genomic libraries and a host strain designed for highly efficient two-hybrid selection in yeast. Genetics 144: 1425–1436.

Kubori, T., Hyakutake, A., and Nagai, H. (2008) *Legionella* translocates an E3 ubiquitin ligase that has multiple

U-boxes with distinct functions. Mol Microbiol 67: 1307–1319.

Laguna, R.K., Creasey, E.A., Li, Z., Valtz, N., and Isberg, R.R. (2006) A Legionella pneumophila-translocated substrate that is required for growth within macrophages and protection from host cell death. Proc Natl Acad Sci USA 103: 18745–18750.

Lamberti, A., Caraglia, M., Longo, O., Marra, M., Abbruzzese, A., and Arcari, P. (2004) The translation elongation factor 1A in tumorigenesis, signal transduction and apoptosis: review article. Amino Acids 26: 443–448.

Liu, Y., and Luo, Z.Q. (2007) The Legionella pneumophila effector SidJ is required for efficient recruitment of endoplasmic reticulum proteins to the bacterial phagosome. Infect Immun 75: 592–603.

Liu, Y., Gao, P., Banga, S., and Luo, Z.Q. (2008) An in vivo gene deletion system for determining temporal requirement of bacterial virulence factors. Proc Natl Acad Sci USA 105: 9385–9390.

Losick, V.P., and Isberg, R.R. (2006) NF-kappaB translocation prevents host cell death after low-dose challenge by Legionella pneumophila. J Exp Med 203: 2177–2189.

Luo, Z.Q., and Farrand, S.K. (1999) Signal-dependent DNA binding and functional domains of the quorum-sensing activator TraR as identified by repressor activity. Proc Natl Acad Sci USA 96: 9009–9014.

Luo, Z.Q., and Isberg, R.R. (2004) Multiple substrates of the Legionella pneumophila Dot/lcm system identified by interbacterial protein transfer. Proc Natl Acad Sci USA 101: 841–846.

McCusker, K.T., Braaten, B.A., Cho, M.W., and Low, D.A. (1991) Legionella pneumophila inhibits protein synthesis in Chinese hamster ovary cells. Infect Immun 59: 240–246.

Machner, M.P., and Isberg, R.R. (2006) Targeting of host Rab GTPase function by the intravacuolar pathogen Legionella pneumophila. Dev Cell 11: 47–56.

Machner, M.P., and Isberg, R.R. (2007) A bifunctional bacterial protein links GDI displacement to Rab1 activation. Science 318: 974–977.

Murata, T., Delprato, A., Ingmundson, A., Toomre, D.K., Lambright, D.G., and Roy, C.R. (2006) The Legionella pneumophila effector protein DrrA is a Rab1 guanine nucleotide-exchange factor. Nat Cell Biol 8: 971–977.

Nagai, H., Kagan, J.C., Zhu, X., Kahn, R.A., and Roy, C.R. (2002) A bacterial guanine nucleotide exchange factor activates ARF on Legionella phagosomes. Science 295: 679–682.

Ninio, S., and Roy, C.R. (2007) Effector proteins translocated by Legionella pneumophila: strength in numbers. Trends Microbiol 15: 372–380.

Olawewaju, O., Ortiz, P.A., Chowdhury, W.Q., Chatterjee, I., and Kinzy, T.G. (2004) The translation elongation factor eEF1B plays a role in the oxidative stress response pathway. RNA Biol 1: 89–94.

Shamovsky, I., Ivannikov, M., Kandel, E.S., Gershon, D., and Nudler, E. (2006) RNA-mediated response to heat shock in mammalian cells. Nature 440: 556–560.

Shohdy, N., Efe, J.A., Emr, S.D., and Shuman, H.A. (2005) Pathogen effector protein screening in yeast identifies Legionella factors that interfere with membrane trafficking. Proc Natl Acad Sci USA 102: 4866–4871.


Siggers, K.A., and Lesser, C.F. (2008) The Yeast Saccharomyces cerevisiae: a versatile model system for the identification and characterization of bacterial virulence proteins. Cell Host Microbe 4: 8–15.

Singh, V., and Aballay, A. (2006) Heat shock and genetic activation of HSF-1 enhance immunity to bacteria. Cell Cycle 5: 2443–2446.

Swanson, M.S., and Isberg, R.R. (1995) Association of Legionella pneumophila with the macrophage endoplasmic reticulum. Infect Immun 63: 3609–3620.

Talapatra, S., Wagner, J.D., and Thompson, C.B. (2002) Elongation factor-1 alpha is a selective regulator of growth factor withdrawal and ER stress-induced apoptosis. Cell Death Differ 9: 856–861.

Tao, W., Kurschner, C., and Morgan, J.I. (1997) Modulation of cell death in yeast by the Bcl-2 family of proteins. J Biol Chem 272: 15547–15552.

Tilney, L.G., Harb, O.S., Connelly, P.S., Robinson, C.G., and Roy, C.R. (2001) How the parasitic bacterium Legionella pneumophila modifies its phagosome and transforms it into rough ER: implications for conversion of plasma membrane to the ER membrane. J Cell Sci 114: 4637–4650.

VanRheenen, S.M., Luo, Z.Q., O’Connor, T., and Isberg, R.R. (2006) Members of a Legionella pneumophila family of proteins with ExoU (phospholipase A) active sites are translocated to target cells. Infect Immun 74: 3597–3606.

Westerheide, S.D., and Morimoto, R.I. (2005) Heat shock response modulators as therapeutic tools for diseases of protein conformation. J Biol Chem 280: 33097–33100.

Zusman, T., Aloni, G., Halperin, E., Kotzer, H., Degtyar, E., Feldman, M., and Segal, G. (2007) The response regulator PmrA is a major regulator of the icm/dot type IV secretion system in Legionella pneumophila and Coxiella burnetii. Mol Microbiol 63: 1508–1523.


Supporting information

Additional Supporting Information may be found in the online version of this article:

Fig. S1. Screening of *L. pneumophila* proteins toxic to yeast. Full-length genes of *L. pneumophila* were cloned into yeast vector pGBKT7 and the resulting plasmids were individually transformed into yeast strain PJ69-4A and transformants were plated onto tryptophan dropout plate. Image shown is one plate that led to the identification of a protein (B6) potentially toxic to yeast.

Fig. S2. The SidI E482K mutant (*sidI*) codes for a stable protein.

A. Yeast cells harbouring a plasmid carrying SidI* produced a stable protein. Lysates if yeast cells harbouring pGBKT7::SidI* grown in Trp dropout medium was resolved by SDS-PAGE and proteins transferred to nitrocellulose membranes were probed with an antibody against the Myc epitope fused to the protein.

B. SidI, SidI* and SidI** purified from *E. coli*. Cell pellets prepared from *E. coli* strains harbouring constructs for expression each of these three proteins were used to purify the proteins in parallel. Proteins eluted by 200 mM imidazole were further purified with an FPLC system with the Superdex 200 GL 10/30 column (GE Healthcare). Samples separated by SDS-PAGE were stained with Coomassie bright blue. Lane 1: total cell lysates; lane 2: purified proteins.

C. Expression of SidI* and SidIR453P, SidIR453L in mammalian cells. Constructs harbouring the mutant genes fused to EGFP on pEGFPC1 were used to transfect 293T cells, lysates prepared 24 h after transfection were subjected to SDS-PAGE separation and expression of the proteins was analysed by immunoblotting using a GFP specific antibody. Note that the amounts of samples (total protein) loaded in this gel were about eight times more than those shown in Fig. 3D lanes 1 and 2.

Fig. S3. Intracellular growth of the *sidl* mutant in mouse bone marrow-derived macrophages (A) and *Dictyostelium discoideum* (B). The indicated bacterial strains grown in AYE broth to late exponential phase (*OD*~600~ = 3.4–3.8) were used to infect host cell monolayer at an moi of 0.05. Extracellular bacteria were removed by washing infected cells with warm PBS for three times. At indicated time points, cells were lysed with 0.02% saponin and the number of viable bacterial was determined by plating appropriate dilutions onto CYE plates. Similar results were obtained from three or more independent experiments and data shown were from one representative experiment done in triplicates. Bacterial strains: Lp02: wild type; Lp03: *dotA*⁻; dSidI: Lp2Δsidl; dsidl-SidI/o: complementation strain.

Fig. S4. SidI did not cause cell death within 10 h of transfection. 293T cells were transfected with plasmid DNA carrying indicated proteins with Lipofectamine 2000. Ten hours after transfection, cells were lifted and were subjected to stain with 0.2% trypan blue for 2 min. Viable (unstained) and total cell numbers were enumerated using a hemocytometer. The percentage of viable cells (unstained) was calculated.

Fig. S5. A double *sidl* and *lgt1* mutant is not defective in intracellular growth. Intracellular growth of the *sidl* or the *sidl/lgt1* double mutant in mouse bone marrow-derived macrophages (A) and *Dictyostelium discoideum* (B). Indicated bacterial strains grown in AYE broth to late exponential phase (*OD*~600~ = 3.4–3.8) were used to infect host cell monolayer at an moi of 0.05. Extracellular bacteria were removed by washing infected cells with

warm PBS for three times. At indicated time points, cells were lysed with 0.02% saponin and the number of viable bacterial was determined by plating appropriate dilutions onto CYE plates. Similar results were obtained from three or more independent experiments and data shown were from one representative experiment done in triplicates. Bacterial strains: Lp02, wild type; Lp03, *dotA*⁻; dSidI, Lp2Δsidl.

Fig. S6. The *dotA* mutant Lp03 was not killed by macrophages in 10 h. Bacteria grown to postexponential phase were used to infect U937 macrophages (1) or mouse bone marrow derived macrophages (2). Extracellular bacteria were removed by washing three times with warm PBS 1 h after uptake. Cells were lysed with 0.2% saponin at 1 and 10 h after uptake respectively. Viable bacteria were determined by plating the lysates onto CYE plates. The percentage of viable bacteria at 10 h was obtained by dividing the bacterial numbers at 10 h by those of 1 h. Data shown are averages of two experiments done in triplicate.

Fig. S7. Hsp proteins levels in infected cells. A. Lysates of U937 cells infected with indicated strains of *L. pneumophila* for 8 h (upper panel) or 14 h (middle panel) at an moi of 5 were prepared, adjusted to similar protein concentrations. Hsp70 was detected after SDS-PAGE separation. Cells subjected to heat shock for 15 min at 42°C and recovered for 10 h were used as a positive control. In both cases, the tubulin protein was probed as a loading control (lower panel).

Table S1. *L. pneumophila* proteins that are toxic or inhibitory to yeast growth identified from 96 hypothetical proteins.

Table S2. Bacterial, yeast strains and plasmids used in this study.

Table S3. Primers used in this study.

Please note: Wiley-Blackwell are not responsible for the content or functionality of any supporting materials supplied by the authors. Any queries (other than missing material) should be directed to the corresponding author for the article.
