
EXPERIMENTAL CELL RESEARCH 315 (2009) 2399–2409

available at www.sciencedirect.com

ScienceDirect

www.elsevier.com/locate/yexcr

**Research Article**

# The renal cell carcinoma-associated oncogenic fusion protein PRCCTFE3 provokes p21<sup>WAF1/CIP1</sup>-mediated cell cycle delay

**Klaas Medendorp, Jan J.M. van Groningen, Lilian Vreede, Lisette Hetterschijt, Linda Brugmans, Wilhelmina H. van den Hurk, Ad Geurts van Kessel\***

Department of Human Genetics, Radboud University Nijmegen Medical Centre, Nijmegen Centre for Molecular Life Sciences, Nijmegen, The Netherlands

---

## ARTICLE INFORMATION

- **Article Chronology:**
  - Received 26 November 2008
  - Revised version received 23 April 2009
  - Accepted 23 April 2009
  - Available online 5 May 2009

- **Keywords:**
  - Renal cell carcinoma
  - PRCCTFE3
  - Cell cycle
  - p21<sup>WAF1/CIP1</sup>

---

## ABSTRACT

Previously, we found that in t(X;1)(p11;q21)-positive renal cell carcinomas the bHLH-LZ transcription factor TFE3 is fused to a novel protein designated PRCC. In addition, we found that the PRCCTFE3 fusion protein, which has retained all known functional domains of TFE3, acts as a more potent transcriptional activator than wild type TFE3. We also found that PRCCTFE3 expression confers *in vitro* and *in vivo* transformation onto various cell types, including those of the kidney. Here we show that *de novo* expression of the PRCCTFE3 fusion protein provokes cell cycle delay. This delay, which is mediated by induction of the cyclin-dependent kinase inhibitor p21<sup>WAF1/CIP1</sup>, affects both the G1/S and the G2/M phases of the cell cycle and prevents the cells from undergoing polyploidization. We also show that the PRCCTFE3 fusion protein binds directly to the p21<sup>WAF1/CIP1</sup> promoter and that the PRCCTFE3-induced up-regulation of p21<sup>WAF1/CIP1</sup> leads to activation of the pRB pathway. Finally, we show that in t(X;1)(p11;q21)-positive renal tumor cells several processes that link PRCCTFE3 expression to p21<sup>WAF1/CIP1</sup>-mediated cell cycle delay are abrogated. Our data suggest a scenario in which, during the course of renal cell carcinoma development, an initial PRCCTFE3-induced cell cycle delay must be numbed, thus permitting continued proliferation and progression towards full-blown malignancy.

© 2009 Elsevier Inc. All rights reserved.

---

## Introduction

Oncogenic stimulation can induce cell cycle delay and senescence to ensure that damaged cells stop to proliferate [1–5]. Recently, it was shown that such a delay may play a critical role in the *in vivo* protection from tumor development [6–10], and may be mediated by cyclin-dependent kinase inhibitors such as p16<sup>INK4a</sup> and p21<sup>WAF1/CIP1</sup> [11,12]. Subsequent elimination of this protective mechanism by suppression of the p53 and/or p16<sup>INK4a</sup> pathways has been shown to permit continued proliferation of primary cells in

---

* Corresponding author. Department of Human Genetics 855, Radboud University Nijmegen Medical Centre, PO Box 9101, 6500 HB Nijmegen, The Netherlands. Fax: +31 24 3668752.
E-mail address: a.geurtsvankessel@antrg.umcn.nl (A. Geurts van Kessel).

0014-4827/$ – see front matter © 2009 Elsevier Inc. All rights reserved.
doi:10.1016/j.yexcr.2009.04.022

MiT protein TFE3 to a novel protein, designated PRCC [18,19]. TFE3 contains several archetypal transcription factor-associated functional domains, among which are two transactivation domains, a basic domain, a helix–loop–helix domain, and a leucine-zipper domain [20]. PRCC neither contains domains indicative for its function nor exhibits homology to any known protein [18,19]. It has been shown, however, that PRCC can associate with pre-mRNA splicing factors such as SC35, PRL1 and CDC5 and that, after fusion with TFE3, this association is weakened [21]. By employing transactivation assays, we and others found that the PRCCTFE3 fusion protein, which has retained all the functional domains of TFE3, acts as a significantly more potent transcriptional activator than wild type TFE3 [21,22]. Moreover, we found that PRCCTFE3 is able to confer malignant transformation onto various cell types, including those of the kidney, in both *in vitro* and *in vivo* assays [23]. In addition, it has recently been shown that the *MET* tyrosine kinase receptor gene can act as a downstream transcriptional target of several TFE3 fusion proteins, including PRCCTFE3 [24–26] [own unpublished data], thereby providing a functional rationale for the oncogenic nature of these fusion proteins.

Here, we report that *de novo* expression of the oncogenic PRCCTFE3 fusion protein provokes cell cycle delay. This delay is mediated through induction of the cyclin-dependent kinase inhibitor p21<sup>WAF1/CIP1</sup> and a concomitant activation of the pRB pathway. In addition, we report that in t(X;1)(p11;q21)-positive renal tumor cells this PRCCTFE3-induced cell cycle delay is numbed, thus allowing the development of a full-blown malignancy.

---

### Materials and methods

#### Cell culture

Hela, U2OS, Rat-1 and Phoenix cells were cultured in DMEM, supplemented with 10% FCS, penicillin (100 U/ml) and streptomycin (100 μg/ml). The renal cell carcinoma-derived cell line Cl89-12117 [27] was cultured in RPMI1640, supplemented with 10% FCS, penicillin (100 U/ml) and streptomycin (100 μg/ml). HEK293/T-REx/PRCCTFE3 [27] and similarly obtained HEK293/T-REx/PRCC cells were cultured in DMEM, supplemented with 10% FCS, zeocin (500 μg/ml), blasticidin (5 μg/ml), penicillin (100 U/ml) and streptomycin (100 μg/ml). For induction of PRCCTFE3 and/or PRCC expression, tetracyclin was added to the culture medium (24 h; 1 μg/ml, unless stated otherwise). HEK293/PRCCTFE3-TY1 was cultured in DMEM, supplemented with 10% FCS, penicillin (100 U/ml), streptomycin (100 μg/ml), G418 (Gibco; 0.5 μg/ml) and tetracycline (1 μg/ml). For induction of TY1-tagged PRCCTFE3, tetracyclin was washed away and cells were grown on complete medium without tetracycline. For an arrest at the metaphase/anaphase transition of the cell cycle, cells were treated with nocodazole (Sigma Aldrich; 16 h, 250 ng/ml). Cells were subsequently released from the nocodazole-induced arrest in pre-warmed complete medium after two washes in pre-warmed PBS. To inhibit translation of proteins, cells were treated with cycloheximide (CHX; Sigma Aldrich, 16 h, 10 μg/ml). For the establishment of growth curves cells were trypsinized, resuspended in complete medium and counted using a Bürker-Türk chamber (W. Schreck). Counts from duplicate cultures were averaged for each time point and plotted on a log scale. Cell population doubling times were calculated from the linear phase of the growth curve. For retroviral infection, supernatants were produced through transfection of Phoenix packaging cells with a shRNA construct, pRS-p21 [28], using Lipofectamine 2000 (Invitrogen). 48 h post transfection, the tissue culture medium was filtered through a 0.45 μm millipore filter and the resulting viral supernatant was used for a 6 h infection of HEK293/T-REx/PRCCTFE3 cells through the addition of 4 μg/ml polybrene (Sigma Aldrich). Subsequently, the cells were allowed to recover for 24 h in fresh medium after which infected cells were selected using 2 μg/ml puromycin (Sigma Aldrich).

#### Western blot analysis

For western blot analysis, proteins were extracted using a lysis buffer (50 mM Tris–HCl pH 8.0, 150 mM NaCl, 1% NP40, 0.1% SDS, 0.5% DOC). Subsequently, cell debris was removed by centrifugation and the cleared samples were incubated for 5 min in sample buffer before loading (20 μg per lane) onto 4–12% NuPage bis–tris gels (Invitrogen). After electrophoresis, the gels were blotted onto nitrocellulose membranes (Protran; Schleicher and Schuell). The resulting blots were blocked in 5% non-fat dry milk/Tris buffered saline containing 0.05% Tween-20 (TBST), and incubated overnight with primary antibody and, subsequently, with a peroxidase conjugated secondary antibody (Zymed) or with a fluorescent conjugated secondary antibody (Molecular Probes). Signals were visualized using autoradiographic exposure to Kodak X-Omat films or scanned and analyzed using the fluorescence-scanning Odyssey system and associated software (Li-Cor). For quantification of the signals the ImageJ software tool was used ([http://rsb.info.nih.gov/ij/](http://rsb.info.nih.gov/ij/)). The area under the curve (AUC) of a specific signal was corrected by the AUC of the loading control. The measurements of the untransfected or uninduced samples were arbitrarily set at 1 (or 100%) and the other conditions were recalculated correspondingly, thereby allowing ratio comparisons.

#### Antibodies

A polyclonal anti-PRCC antibody was used as described before [27]. Anti-p21<sup>WAF1/CIP1</sup> (Sigma Aldrich), anti-p53 (Sigma Aldrich), anti-Ran (BD Transduction Laboratories), anti-γ-tubulin (Santa Cruz), anti-CDK2 (Abcam) and anti-pRB (phospho S807 and S811; Abcam) antibodies were used according to the instructions of the manufacturers.

#### Cell cycle analysis using flow cytometry

Replicative DNA synthesis and DNA content were analyzed by bivariate flow cytometry using a fluorescence activated cell sorter (FACS) as described before [27]. For analysis of the progression of cells through the S phase, and the subsequent G2/M and G0/G1 phases of the cell cycle, cells were pulse labeled for 10 min at 37 °C with 10 μM BrdU and harvested at different time intervals by trypsinization followed by overnight fixation in 70% ethanol at 4 °C. After adding 10 ml PBS, cells were spun down and treated with 0.1 M HCl containing 0.4 mg/ml pepsin for 20 min at room temperature. Subsequently, cells were permeabilized (0.5% Tween-20, 0.1% BSA in PBS) and incubated for 12 min at 37 °C in 2 M HCl, followed by the addition of 0.05 M borate buffer (pH 8.5).

Cells were washed with 0.5% Tween-20, 0.1% BSA in PBS and incubated with anti-BrdU-FITC antibody (Becton Dickinson; 1 h, 4 °C). Finally, cells were counterstained with 40 μg/ml propidium iodide supplemented with 250 μg/ml RNase A and analyzed by FACS. Apoptosis assays were performed using an Annexin V-FITC apoptosis detection kit I (BD Biosciences), according to the instructions of the manufacturer.

### Chromatin immunoprecipitation analysis

Chromatin immunoprecipitations were performed essentially as described before [29]. Protein–DNA complexes were immunoprecipitated using an antibody directed against the TY1-tag (BB2; a gift from H. Stunnenberg) and a mouse pre-immune serum. The recovered genomic DNAs were analyzed by PCR using primer pairs spanning the p21<sup>WAF1/CIP1</sup> promoter [30].

---

### Results

#### PRCCTFE3 expression results in cell cycle delay

In order to assess the effect of *de novo* PRCCTFE3 expression on cell cycle progression, we performed BrdU pulse-chase experiments on both PRCCTFE3 expressing and non-expressing, i.e., tetracycline-induced and non-induced, HEK293/T-REx/PRCCTFE3 cells (see Materials and methods and Fig. S1). After pulse labeling, cells were harvested at various time points, followed by cell cycle profile analyses using FACS. Through such analyses, BrdU-positive cells can be followed as they pass through the S, G2/M, and G0/G1 phases of the cell cycle, respectively.

Immediately after pulse labeling (*t* = 0) all BrdU-positive cells were in the S phase, as expected (Fig. 1). Of the non-induced (−tet) cells 37.1% were in the S phase (BrdU-positive), 41.5% in the G0/G1 (2 N) phase and 16.4% in the G2/M (4 N) phase of the cell cycle (BrdU-negative). Similarly, of the induced, PRCCTFE3 expressing, cells (+tet) 19.4% were in the S phase, 55.9% in the G0/G1 phase and 19.4% in the G2/M phase of the cell cycle, respectively. Notably, the percentage of BrdU-positive (S phase) cells was significantly lower (19.4% versus 37.1%) in the PRCCTFE3 expressing cells than in the non-expressing cells (+tet and −tet, respectively). In reverse, the percentage of G0/G1 (BrdU-negative) cells was significantly higher (55.9% versus 41.5%) in the PRCCTFE3 expressing cells than in the non-expressing cells. These results suggest that during pulse labeling less PRCCTFE3 expressing cells had entered the S phase and, thus, that these cells were delayed in the G1 to S transition of the cell cycle.

A less significant difference was observed for the respective induced and non-induced G2/M phase cells (19.4% versus 16.4%). After a 4 h chase (Fig. 1), 69.5% of the non-induced BrdU-positive cells had entered the G2/M phase, whereas 15.8% of these cells had entered the G0/G1 phase of the cell cycle. In the simultaneously induced, PRCCTFE3 expressing, BrdU-positive cells 74.6% had entered the G2/M phase, whereas only 6.8% of these cells had entered the G0/G1 phase. After a 6 h chase, 37.5% of the non-induced BrdU-positive cells had progressed through the M phase into the G0/G1 phase. In contrast, only 14.6% of the induced, PRCCTFE3 expressing, BrdU-positive cells had entered the G0/G1 phase, whereas 71.8% of these latter cells were still in the G2/M phase of the cell cycle. After a 12 h chase 69.6% of the non-induced BrdU-positive cells had re-entered the G0/G1 phase, whereas the induced, PRCCTFE3 expressing, BrdU-positive cells were predominantly present in the G2/M phase of the cell cycle (53.6%) and, thus, were delayed.

The cell cycle data also revealed a relatively slow passage of the PRCCTFE3 expressing cells through the S phase (Fig. 1; +tet). From these results we conclude that exogenous PRCCTFE3 expression in HEK293 cells results in a delay of the G1/S, G2/M, and S phases of the cell cycle.

In order to independently confirm these data, we established growth curves of PRCCTFE3 expressing and non-expressing HEK293/T-REx cells (Fig. S2). Cells from duplicate cultures were counted at days 0, 2, 5, 7, 9 and 12, and averaged for each time point. Cell population doubling times were calculated from the linear phases of the growth curves. By doing so, we found that PRCCTFE3 non-expressing cells (−tet) exhibited a doubling time of ~24 h. Based on this doubling time, subsequent FACS analysis indicated that the cells remained 14.5 h in the G0/G1 phase, 9 h in the S phase and 30 min in the G2/M phase of the cell cycle (not shown). In contrast, we found that PRCCTFE3 expressing cells (+tet) only doubled once during the first 5 days and stopped growing from that point on. Subsequent FACS analysis (day 1–5) indicated that the cells were almost evenly distributed over G1/G0 and G2/M phases of the cell cycle, with few cells in S phase (not shown).

#### PRCCTFE3 expression abrogates cell cycle recovery after nocodazole treatment

Our observation that PRCCTFE3 expression results in a G2/M phase delay, prompted us to assess the effect of the microtubule destabilizing drug nocodazole, which normally arrests cells at the metaphase/anaphase transition of the cell cycle [31], on this delay. In order to do so, we again pulse labeled both non-induced and induced HEK293/T-REx/PRCCTFE3 cells with BrdU, followed by a 16 h treatment of these cells with nocodazole and a subsequent release from this treatment during various time intervals.

Of the non-induced cells 35.3% were in the S phase (BrdU-positive), 37.8% in the G0/G1 phase, and 21.5% in the G2/M phase of the cell cycle (Fig. S3). After PRCCTFE3 induction 49.9% of the cells were in the G0/G1 phase, only 22.8% in the S phase, and 19.8% in the G2/M phase of the cell cycle (Fig. S3), which is in full agreement with the above notion that PRCCTFE3 expressing cells are delayed in the G1 to S phase transition. Subsequently, the BrdU-positive cells were chased for different time intervals after a 16 h nocodazole treatment and subjected to FACS analysis (Fig. 2).

Immediately after nocodazole treatment (0 h) most cells (78.4–81.2%) were arrested in the G2/M phase of the cell cycle, as expected. After a 2 h release from nocodazole 25.1% of the non-induced cells had exited the M phase and re-entered the G0/G1 phase, whereas of the PRCCTFE3 expressing cells only 14.6% had re-entered the G0/G1 phase of the cell cycle. After an 8 h release from nocodazole 72.7% of the PRCCTFE3 expressing cells were still present in the G2/M phase, whereas 29.2% of the non-induced cells had re-entered the G0/G1 phase.

Notably, 16.5% of the BrdU-positive non-induced cells failed to re-enter the G0/G1 phase of the cell cycle but, instead, became polyploid. Subsequent microscopic analysis revealed that these polyploid cells are mononuclear in nature, thus indicating that

Fig. 1 – Cell cycle delay after exogenous PRCCTFE3 expression. HEK293/T-REx/PRCCTFE3 cells, incubated with or without tetracyclin (+ or – tet), were pulse labeled with BrdU. Subsequently, cell cycle profiles were obtained by FACS analysis. In the upper boxes dot plots representing BrdU fluorescence (y-axis) versus DNA content (x-axis) are shown. In the lower boxes DNA histograms of all cells in the above dot plots are shown. At time point 0 h, cells in the G0/G1 (2 N), S and G2/M (4 N) phases of the cell cycle, and polyploid cells (>4 N) are marked by boxes 1, 2, 3 and 4, respectively. The percentages of cells in each of these cell cycle phases are listed (1–4). At later time points (4–12 h), cell cycle profiles are calculated based on BrdU-labeled cells, which are represented in the upper square of each dot plot. Percentages of cells in the G0/G1 (2 N), S and G2/M (4 N) phases of the cell cycle, and polyploid cells (>4 N) are listed below the dot plots. Non-induced and induced, PRCCTFE3 expressing, cells are indicated by – or + tet, respectively.

they result from endoreduplication. By combining these data, we conclude that PRCCTFE3 expression leads to an abrogation of cell cycle recovery after nocodazole treatment.

In order to independently confirm these data, we calculated the cell doubling times in PRCCTFE3 expressing and non-expressing HEK293/T-REx cells after nocodazole treatment (Fig. S4). By doing so, we found that the non-expressing cells exhibited a doubling time of ~46 h during the first 4 days after release from nocodazole. After this time point, a doubling time of ~24 h was measured, which compares to that of non-treated cells (see above). Conversely, the number of PRCCTFE3 expressing cells only increased 6% during the first day and stopped growing after this time point, thus confirming an abrogation of cell cycle recovery after nocodazole treatment upon PRCCTFE3 expression.

### PRCCTFE3 expression prevents polyploidy in the presence of nocodazole

Since we observed polyploidy in nocodazole treated non-PRCCTFE3 expressing cells, but not in similarly treated PRCCTFE3 expressing cells, we decided to assess the role of the PRCCTFE3 fusion protein in polyploidization in further detail. Therefore, we again BrdU-pulse labeled non-induced and induced HEK293/T-REx/PRCCTFE3 cells, and assayed by FACS the ploidy status of the BrdU-positive and BrdU-negative cells after incubation during several time intervals in medium containing nocodazole.

After a 16 h incubation in the presence of nocodazole, most non-induced cells were arrested at the G2/M phase of the cell cycle, as expected (Fig. 3; –tet, 4N). Completely in line with the above results, again a significant proportion of the induced,

0 hrs          2 hrs          4 hrs          6 hrs          8 hrs
10⁴           10⁴           10⁴           10⁴           10⁴
10³           10³           10³           10³           10³
10²           10²           10²           10²           10²
10¹           10¹           10¹           10¹           10¹
10⁰           10⁰           10⁰           10⁰           10⁰

G1/G0:        6.2%          G1/G0:        25.1%         G1/G0:        27.5%         G1/G0:        34.4%         G1/G0:        29.2%
S:            5.1%          S:            10.3%         S:            2.8%          S:            4.0%          S:            5.2%
G2/M:         81.2%         G2/M:         56.9%         G2/M:         61.5%         G2/M:         50.7%         G2/M:         49.1%
>4N:          7.5%          >4N:          7.7%          >4N:          8.2%          >4N:          10.9%         >4N:          16.5%

2N 4N         2N 4N         2N 4N         2N 4N         2N 4N

0 hrs          2 hrs          4 hrs          6 hrs          8 hrs
10⁴           10⁴           10⁴           10⁴           10⁴
10³           10³           10³           10³           10³
10²           10²           10²           10²           10²
10¹           10¹           10¹           10¹           10¹
10⁰           10⁰           10⁰           10⁰           10⁰

G1/G0:        3.1%          G1/G0:        14.6%         G1/G0:        17.0%         G1/G0:        14.4%         G1/G0:        14.0%
S:            6.3%          S:            5.0%          S:            4.8%          S:            5.8%          S:            5.6%
G2/M:         79.4%         G2/M:         72.3%         G2/M:         73.8%         G2/M:         70.3%         G2/M:         72.7%
>4N:          11.2%         >4N:          8.1%          >4N:          4.4%          >4N:          9.5%          >4N:          7.7%

2N 4N         2N 4N         2N 4N         2N 4N         2N 4N

Fig. 2 – Delayed recovery of PRCCTFE3 expressing cells from nocodazole treatment. HEK293/T-REx/PRCCTFE3 cells, incubated with or without tetracyclin (+ or – tet), were pulse labeled with BrdU and, subsequently, subjected to cell cycle profiling as indicated in Fig. 1. In the upper boxes dot plots representing BrdU fluorescence (y-axis) versus DNA content (x-axis) are shown. In the lower boxes DNA histograms of the BrdU-positive cells, which are represented in the upper square of each dot plot, are shown. After pulse labeling, cells were grown overnight in the presence of nocodazole, and harvested at different time points (0–8 h) after nocodazole removal. The percentages of cells in each of the cell cycle phases are listed.

PRCCTFE3-expressing, BrdU-negative cells were delayed in the G0/ G1 phase of the cell cycle (Fig. 3; +tet, 2N). After a 24 h incubation in nocodazole 19.0% of the non-induced BrdU-positive cells were polyploid (>4N), whereas the induced, PRCCTFE3 expressing, BrdU-positive cells were predominantly residing in the G2/M phase of the cell cycle (76.3%), i.e., they were diploid (4N). After a 48 h incubation in the presence of nocodazole 60.4% of the non- induced cells were polyploid. Further extension of the incubation in nocodazole (72 and 96 h, respectively) resulted in a further increase in the percentage of polyploid non-induced cells (up to 66.8%). In reverse, however, even after a 96 h exposure to nocodazole of the induced, PRCCTFE3 expressing, cells the percentage of polyploid cells remained low (6.2%).

In order to assess the presence of putative apoptotic effects elicited by PRCCTFE3 expression induction in the cell system used, we performed apoptosis assays on nocodazole treated cells, with or without PRCCTFE3 expression induction. By doing so, we observed very similar apoptotic levels under the various tissue
culture conditions tested (Fig. S5) indicating that, next to nocodazole, PRCCTFE3 expression induction does not have any effect.

From our results we conclude that exogenous PRCCTFE3 expression prevents polyploidization in the presence of the microtubule destabilizing drug nocodazole. In contrast, polyploidization was readily observed in similarly treated t(X;1)(p11;q21)-positive Cl89-12117 and Cl89-17872 renal cell carcinoma-derived cells which endogenously express PRCCTFE3 (19.4% and 46.0% polyploid cells after 96 h in nocodazole versus 6.1% and 6.7% in non-treated cells, respectively). This latter result indicates that in these tumor cells the PRCCTFE3-mediated mechanism that prevents polyploidization is abrogated.

Since degradation of cyclin B1 is a prerequisite for progression of cells through mitosis, inhibition of this process is expected to result in a concomitant inhibition of cyclin B1 degradation [32]. Indeed, after PRCCTFE3 expression induction, the levels of cyclin B1 remained constant even after release of the cells from a nocodazole-induced mitotic arrest (not shown).

**PRCCTFE3 induces p21<sup>WAF1/CIP1</sup> expression and directly binds to its promoter**

Since G1/S and G2/M cell cycle arrest has been shown to be mediated by cyclin-dependent kinase inhibitors such as p21<sup>WAF1/CIP1</sup>

[12,33,34] we decided to assess the expression of this protein in our PRCCTFE3 induction system. Therefore, HEK293/T-REx/PRCCTFE3 cells were incubated for 24 h with tetracyclin and subsequently assayed by western blot analysis for expression of the PRCCTFE3 and p21<sup>WAF1/CIP1</sup> proteins (Fig. 4A). As an internal loading control the small GTPase Ran was used. By doing so, we found that the PRCCTFE3 protein was abundantly expressed after tetracyclin induction, as expected. Concomitantly, we found that the p21<sup>WAF1/CIP1</sup> protein exhibited a clear up-regulation in expression. No overt changes in expression were observed for the endogenous PRCC (not shown), p53 and (control) Ran proteins.

In order to correlate PRCCTFE3 expression induction with the observed p21<sup>WAF1/CIP1</sup> up-regulation, HEK293/T-REx/PRCCTFE3 cells were incubated in different concentrations of tetracyclin and, again, analyzed by western blotting (Fig. S1A). Whereas at low concentrations (0–1 ng/ml) only a minor induction of PRCCTFE3 expression was observed, this induction was apparent at concentrations of 10 ng/ml and up. Simultaneously, up-regulation of p21<sup>WAF1/CIP1</sup> was observed. This up-regulation was not observed after induction of control empty vector HEK293/T-REx cells with tetracyclin (Fig. S1A; vector). In order to exclude PRCCTFE3 induction unrelated effects, non-induced and induced HEK293/T-REx/PRCCTFE3 cells were analyzed for p21<sup>WAF1/CIP1</sup> protein levels at different time intervals. By doing so, we found that as early as 2–4 h after PRCCTFE3 expression induction the p21<sup>WAF1/CIP1</sup> protein was

![Figure](https://i.imgur.com/yourimageurl.png)

Fig. 3 – PRCCTFE3 expression prevents polyploidy in the presence of nocodazole. HEK293/T-REx/PRCCTFE3 cells incubated with or without tetracyclin (+ or − tet) were pulse labeled with BrdU and grown in the presence of nocodazole during different time intervals (16–96 h). Subsequently, cell cycle profiles were generated as indicated in Fig. 1. The upper boxes represent dot plots of BrdU fluorescence (y-axis) versus DNA content (x-axis), whereas the lower boxes represent DNA histograms of the BrdU-positive cells in the dot plots.
up-regulated (Fig. S1B). This up-regulation was further increased after 8 h and matched closely with the concomitant up-regulation of the PRCCTFE3 protein. Taken together, we conclude that *de novo* PRCCTFE3 expression up-regulates p21<sup>WAF1/CIP1</sup> expression.

In order to assess whether the observed PRCCTFE3-mediated up-regulation of p21<sup>WAF1/CIP1</sup> results from *de novo* protein synthesis, we cultured both tetracyclin-induced and -uninduced HEK293/T-REx/PRCCTFE3 cells and control HEK293/T-REx/PRCC cells in the presence or absence of the mRNA translation inhibitor cycloheximide (CHX; Fig. 4B). As expected, no p21<sup>WAF1/CIP1</sup> expression was observed under the various culture conditions applied to the control HEK293/T-REx/PRCC cells. In the HEK293/T-REx/PRCCTFE3 cells, however, p21<sup>WAF1/CIP1</sup> protein up-regulation was readily observed after PRCCTFE3 induction in the absence of cycloheximide but not in its presence, thus indicating that the above-observed PRCCTFE3-induced up-regulation of p21<sup>WAF1/CIP1</sup> indeed reflects *de novo* protein synthesis. As in Fig. 4A, no overt changes were observed in p53 expression.

Since the PRCCTFE3 fusion protein can act as a transcriptional activator [22], we set out to assess whether the observed p21<sup>WAF1/CIP1</sup> up-regulation results from direct binding of the PRCCTFE3 protein to its gene promoter. To this end, we performed chromatin immunoprecipitation (ChIP) experiments in HEK293 cells expressing a TY-1 tagged (see Materials and methods) full-length PRCCTFE3 protein. The recovered ChIP DNAs were analyzed for the enrichment of p21<sup>WAF1/CIP1</sup> promoter sequences. By doing so, we indeed found that the anti-TY1 ChIP DNAs were enriched for these sequences (Fig. 4C). From these results we conclude that the PRCCTFE3 protein activates the p21<sup>WAF1/CIP1</sup> promoter through direct binding.

In order to exclude HEK293-specific effects three independent cell lines, Hela, U2OS and Rat-1, were transiently transfected with either empty vectors or PRCCTFE3 expression constructs and analyzed for p21<sup>WAF1/CIP1</sup> expression by western blotting (Fig. 4D). As in HEK293 cells, also in Hela and U2OS cells *de novo* expression of PRCCTFE3 elicited an increase in p21<sup>WAF1/CIP1</sup> protein expression levels. As expected, the p21<sup>WAF1/CIP1</sup>-deficient Rat-1 cells [35–37] failed to show such an induction.

Exogenous PRCCTFE3 expression does not activate the pRB pathway in t(X;1)(p11;q21)-positive tumor cells

Since p21<sup>WAF1/CIP1</sup> is known to act within the pRB cell cycle control pathway [12], we set out to investigate whether the p21<sup>WAF1/CIP1</sup> downstream effectors CDK2 and/or pRB are affected by *de novo* PRCCTFE3 expression. To this end, HEK293/T-REx/PRCCTFE3 cells were again incubated with tetracyclin and, subsequently, assayed by western blot analysis for CDK2 protein expression and pRB phosphorylation levels (Fig. 5). After *de novo* PRCCTFE3 expression induction, we observed a clear decrease in the CDK2 protein level (Fig. 5A). Since CDK2 is known to catalyze phosphorylation of pRB at serine 807, which in turn disrupts its binding to E2F and enables cell cycle progression [38–41], we used an anti-pRB phospho S807 antibody (see Materials and methods) to assess the pRB phosphorylation status. By doing so, we observed a clear decrease in the level of phosphorylated pRB protein after *de novo* PRCCTFE3 expression induction (Fig. 5B). Together, these observations support a scenario in which the PRCCTFE3-induced cell cycle delay is brought about through a p21<sup>WAF1/CIP1</sup>-mediated activation of the pRB cell cycle control pathway.

In order to further substantiate this notion, we stably knocked down p21<sup>WAF1/CIP1</sup> expression in our HEK293/T-REx/PRCCTFE3 inducible cell line using a retroviral shRNA construct (Fig. S6A). Next, we determined the doubling times of the shRNA engineered PRCCTFE3-expressing and non-expressing cells. By doing so, no

Fig. 4 – PRCCTFE3 induces p21<sup>WAF1/CIP1</sup> expression. (A) Proteins were extracted from HEK293/T-REx/PRCCTFE3 cells incubated 24 h with or without tetracyclin (+ or − tet) and, subsequently, analyzed by western blotting using anti-PRCC, anti-p53, anti-p21<sup>WAF1/CIP1</sup> and anti-Ran (control) antibodies. (B) HEK293/T-REx/PRCC and HEK293/T-REx/PRCCTFE3 cells were incubated with or without tetracyclin (+ or − tet) and/or cycloheximide (+ or − CHX), and analyzed by western blotting using anti-p53 and anti-p21<sup>WAF1/CIP1</sup> antibodies. (C) Protein–DNA complexes were immunoprecipitated from HEK293/PRCCTFE3-TY1 cells using an anti-tag antibody (αTY1) and a mouse pre-immune serum (IgG). The recovered genomic DNAs, as well as the input chromatin, were analyzed by PCR using primer pairs spanning the p21<sup>WAF1/CIP1</sup> gene promoter. (D) Hela, U2OS and Rat-1 cells were either not transfected (−) or transiently transfected with an empty vector or a PRCCTFE3 expression construct. Cells were lysed 24 h after transfection and analyzed by western blotting using anti-PRCC, anti-p21<sup>WAF1/CIP1</sup> and anti-Ran (control) antibodies. Protein bands were derived from single membranes and band intensities of at least 3 independent experiments were quantified and averaged using the ImageJ software tool. The amount of protein was corrected for by the loading control. The non-transfected condition (−) was arbitrarily set at 100%.

Fig. 5 – PRCCTFE3 expression activates the pRB signaling pathway. Proteins were extracted from HEK293/T-REx/PRCCTFE3 cells incubated 0 and 48 h with tetracycline and, subsequently, analyzed by western blotting using anti-CDK2 (A), anti-phospho-pRB (B) and anti-Ran (control) antibodies. Protein bands were derived from single membranes and quantified using the ImageJ software tool. The amount of protein was corrected for by using the loading control. The non-induced condition $(t=0)$ was arbitrarily set at 1.

overt differences in cell doubling times were observed in the respective $p21^{WAF1/CIP1}$ knockdown cells (both ~24 h; Fig. S6B), which fully supports our notion that the PRCCTFE3-induced cell cycle delay is mediated by $p21^{WAF1/CIP1}$.

In order to subsequently assess whether exogenous PRCCTFE3 expression also affects $p21^{WAF1/CIP1}$ protein levels in t(X;1)(p11;q21)-positive renal tumor cells, we transfected the Cl89-12117 cell line (see above) with a PRCCTFE3 expression construct, resulting in a marked increase in the PRCCTFE3 protein level (Fig. 6A). In contrast to the above obtained results, however, we failed to observe any concurrent effect on the $p21^{WAF1/CIP1}$ expression level (Fig. 6A). In order to assess whether other components of the pRB pathway, i.e., CDK2 and/or pRB, might still be affected by the exogenous PRCCTFE3 expression in renal tumor cells, we again performed western blot analyses on non-transfected and transfected Cl89-12117 cells using anti-CDK2 and anti-pRB phospho S807 antibodies. By doing so, we failed to detect any changes in CDK2 protein expression (Fig. 6B) and/or pRB phosphorylation (Fig. 6C) levels. These data are completely in line with a model in which in t(X;1)(p11;q21)-positive renal cell carcinomas an initial PRCCTFE3-induced cell cycle delay is numbed, thus allowing a progression towards full-blown malignancy.

including the $p21^{WAF1/CIP1}$-mediated S phase delay in PRCCTFE3 expressing cells [42,43].

We also found that, after exposure of cells to the microtubule destabilizing drug nocodazole, PRCCTFE3 expression prevents polyploidization. This finding is in line with, and substantiates, the observed PRCCTFE3-induced G2/M and G1/S cell cycle delay, thus preventing mitotically compromised cells from undergoing endoreduplication. The observed effects were not due to putative apoptotic effects elicited by PRCCTFE3 expression induction. Others have found that induction of $p21^{WAF1/CIP1}$ is required for a G1/S cell cycle arrest after activation of the mitotic spindle checkpoint, and that this arrest prevents endoreduplication [44–47]. Subsequently, Vogel et al. [48] found that in normal cell cycle control the prevention of polyploidization is based on cross-talk between the mitotic spindle assembly checkpoint and the G2/M and G1/S checkpoints. Our observation that PRCCTFE3 is capable of inducing $p21^{WAF1/CIP1}$ expression and, concomitantly, G2/M and G1/S cell cycle delay is in agreement with these data.

More interestingly we found that, in contrast to HEK293, Hela, and U2OS cells, exogenous expression of PRCCTFE3 in t(X;1)(p11;q21)-positive renal tumor cells failed to affect $p21^{WAF1/CIP1}$ expression levels and/or pRB pathway activation. This latter result suggests that the processes that link PRCCTFE3 expression to $p21^{WAF1/CIP1}$-mediated cell cycle delay are uncoupled in these tumor cells. Such an uncoupling is completely in line with our observation that in t(X;1)(p11;q21)-positive renal tumor cells the prevention of nocodazole-induced polyploidization is abrogated.

Our observations are also in accordance with the notion [49] that an oncogenic chromosome translocation by itself may not be sufficient to induce malignant transformation, and that complementary alterations are required to provoke transformation. Such a step-wise mechanism is compatible with recent observations indicating that oncogenic stimulation may induce cell cycle arrest and premature senescence [1–9], and that a subsequent elimination of this protective mechanism may permit continued proliferation in the presence of the initial oncogenic event, thereby leading to a full-blown malignancy [13]. As we observed $p21^{WAF1/CIP1}$ expression in t(X;1)(p11;q21)-positive renal tumor cells, we

Discussion

We found that *de novo* expression of the renal cell carcinoma-associated oncogenic fusion protein PRCCTFE3 provokes cell cycle delay. This delay was mediated by direct induction of the cyclin-dependent kinase inhibitor $p21^{WAF1/CIP1}$ and affected the G1/S, G2/M and S phases of the cell cycle. In addition, we found that the PRCCTFE3-induced $p21^{WAF1/CIP1}$ up-regulation was associated with an activation of the pRB pathway, i.e., with CDK2 protein and pRB phosphorylation down-regulation. The PRCCTFE3-induced cell cycle delay could be rescued by knocking down $p21^{WAF1/CIP1}$ expression. No PRCCTFE3-mediated induction of the cyclin-dependent kinase inhibitor $p16^{INK4a}$ was observed (not shown). Our results are in accordance with literature data [12,33,34],

any mutations (not shown), which is in line with the above notion that in these cells the escape from cell cycle delay may be brought about through an alternative mechanism(s).

In conclusion, our data indicate that exogenous expression of the renal cell carcinoma-associated oncogenic fusion protein PRCCTFE3 provokes a p21<sup>WAF1/CIP1</sup>-mediated cell cycle delay. Since p21<sup>WAF1/CIP1</sup> is also expressed in primary PRCCTFE3-positive renal tumor cells, and since we found that in these tumor cells p21<sup>WAF1/CIP1</sup> levels are no longer affected by exogenous PRCCTFE3, we suggest that during the course of tumor development the initial p21<sup>WAF1/CIP1</sup>-induced cell cycle delay must be numbed, thus allowing a step-wise progression towards full-blown malignancy. This step-wise progression may include a PRCCTFE3-mediated acquisition of resistance to anti-proliferative signals, as was recently observed after TFE3 over-expression in p21<sup>WAF1/CIP1</sup>-deficient Rat-1 cells [51].

---

### Acknowledgments

The authors thank Marion Lohrum, Reuven Agami, Eric Schoenmakers, Diederik de Bruijn, Piet van Erp and Frank van Leeuwen for advice and support and Henk Stunnenberg for providing the BB2 antibody. This work was supported by the Dutch Cancer Society.

---

### Appendix A. Supplementary data

Supplementary data associated with this article can be found, in the online version, at [doi:10.1016/j.yexcr.2009.04.022](doi:10.1016/j.yexcr.2009.04.022).

---

### REFERENCES

[1] M. Serrano, A.W. Lin, M.E. McCurrach, D. Beach, S.W. Lowe, Oncogenic ras provokes premature cell senescence associated with accumulation of p53 and p16INK4a, Cell 88 (1997) 593–602.

[2] M. Braig, C.A. Schmitt, Oncogene-induced senescence: putting the brakes on tumor development, Cancer Res. 66 (2006) 2881–2884.

[3] M. Tsuda, T. Watanabe, T. Seki, T. Kimura, H. Sawa, A. Minami, T. Akagi, K. Isobe, K. Nagashima, S. Tanaka, Induction of p21<sup>(WAF1/CIP1)</sup> by human synovial sarcoma-associated chimeric oncoprotein SYT-SSX1, Oncogene 24 (2005) 7984–7990.

[4] M. Collado, M. Serrano, The power and the promise of oncogene-induced senescence markers, Nat. Rev. Cancer 6 (2006) 472–476.

[5] W.J. Mooi, D.S. Peeper, Oncogene-induced cell senescence—halting on the road to cancer, N. Engl. J. Med. 355 (2006) 1037–1046.

[6] M. Braig, S. Lee, C. Loddonkemper, C. Rudolph, A.H. Peters, B. Schlegelberger, H. Stein, B. Dorken, T. Jenuwein, C.A. Schmitt, Oncogene-induced senescence as an initial barrier in lymphoma development, Nature 436 (2005) 660–665.

[7] Z. Chen, L.C. Trotman, D. Shaffer, H.K. Lin, Z.A. Dotan, M. Niki, J.A. Koutcher, H.I. Scher, T. Ludwig, W. Gerald, C. Cordon-Cardo, P.P. Pandolfi, Crucial role of p53-dependent cellular senescence in suppression of Pten-deficient tumorigenesis, Nature 436 (2005) 725–730.

[8] M. Collado, J. Gil, A. Efeyan, C. Guerra, A.J. Schuhmacher, M. Barradas, A. Benguria, A. Zaballos, J.M. Flores, M. Barbacid, D. Beach, M. Serrano, Tumour biology: senescence in premalignant tumours, Nature 436 (2005) 642.

[9] C. Michaloglou, L.C. Vredeveld, M.S. Soengas, C. Denoyelle, T. Kuilman, C.M. van der Horst, D.M. Majoor, J.W. Shay, W.J. Mooi,

D.S. Peeper, BRAFE600-associated senescence-like cell cycle arrest of human naevi, Nature 436 (2005) 720–724.

[10] A.L. Fridman, M.A. Tainsky, Critical pathways in cellular senescence and immortalization revealed by gene expression profiling, Oncogene 27 (2008) 5975–5987.

[11] E. Hara, R. Smith, D. Parry, H. Tahara, S. Stone, G. Peters, Regulation of p16CDKN2 expression and its implications for cell immortalization and senescence, Mol. Cell Biol. 16 (1996) 859–867.

[12] F. Bunz, A. Dutriaux, C. Lengauer, T. Waldman, S. Zhou, J.P. Brown, J.M. Sedivy, K.W. Kinzler, B. Vogelstein, Requirement for p53 and p21 to sustain G2 arrest after DNA damage, Science 282 (1998) 1497–1501.

[13] P.M. Voorhoeve, R. Agami, Unraveling human tumor suppressor pathways: a tale of the INK4A locus, Cell Cycle 3 (2004) 616–620.

[14] P.M. Voorhoeve, C.le Sage, M. Schrier, A.J. Gillis, H. Stoop, R. Nagel, Y.P. Liu, J.van Duijse, J. Drost, A. Griekspoor, E. Zlotorynski, N. Yabuta, G. De Vita, H. Nojima, L.H. Looijenga, R. Agami, A genetic screen implicates miRNA-372 and miRNA-373 as oncogenes in testicular germ cell tumors, Cell 124 (2006) 1169–1181.

[15] J.N. Eble, G. Sauter, J.I. Epstein, I.A. Sesterhenn, Tumors of the kidney, World Health Organization Classification of Tumors. Pathology and Genetics of Tumours of the Urinary System and Male Genital Organs, IARC Press, Lyon, France, 2004, pp. 37–38.

[16] K. Medendorp, J.J.M. van Groningen, M. Schepens, L. Vreede, J. Thijssen, E.F. Schoenmakers, W.H. van den Hurk, A. Geurts van Kessel, R.P. Kuiper, Molecular mechanisms underlying the MiT translocation subgroup of renal cell carcinomas, Cytogenet. Genome Res. 118 (2007) 157–165.

[17] R. Rampal, A. Pappo, M. Zielenska, R. Grant, B.Y. Ngan, Pediatric renal cell carcinoma: clinical, pathologic, and molecular abnormalities associated with the members of the MiT transcription factor family, Am. J. Clin. Pathol. 126 (2006) 349–364.

[18] M.A. Weterman, M. Wilbrink, A. Geurts van Kessel, Fusion of the transcription factor TFE3 gene to a novel gene, PRCC, in t(X;1) (p11;q21)-positive papillary renal cell carcinomas, Proc. Natl. Acad. Sci. U. S. A. 93 (1996) 15294–15298.

[19] S.K. Sidhar, J. Clark, S. Gill, R. Hamoudi, A.J. Crew, R. Gwilliam, M. Ross, W.M. Linehan, S. Birdsall, J. Shipley, C.S. Cooper, The t(X;1)(p11.2;q21.2) translocation in papillary renal cell carcinoma fuses a novel gene PRCC to the TFE3 transcription factor gene, Hum. Mol. Genet. 5 (1996) 1333–1338.

[20] H. Beckmann, L.K. Su, T. Kadesch, TFE3: a helix-loop-helix protein that activates transcription through the immunoglobulin enhancer muE3 motif, Genes Dev. 4 (1990) 167–179.

[21] Y.M. Skalsky, P.M. Ajuh, C. Parker, A.I. Lamond, G. Goodwin, C.S. Cooper, PRCC, the commonest TFE3 fusion partner in papillary renal carcinoma is associated with pre-mRNA splicing factors, Oncogene 20 (2001) 178–187.

[22] M.A. Weterman, J.J.M. van Groningen, A. Jansen, A. Geurts van Kessel, Nuclear localization and transactivating capacities of the papillary renal cell carcinoma-associated TFE3 and PRCC (fusion) proteins, Oncogene 19 (2000) 69–74.

[23] M.A. Weterman, J.J.M. van Groningen, A. den Hartog, A. Geurts van Kessel, Transformation capacities of the papillary renal cell carcinoma-associated PRCCTFE3 and TFE3PRCC fusion genes, Oncogene 20 (2001) 1414–1424.

[24] Y. Sagara, Y. Miyata, K. Nomata, K. Abe, J. Eguchi, T. Hayashi, H. Sakai, S. Kanda, H. Kanetake, TFE3-ren al carcinoma in an adult patient: a case with strong expression of phosphorylated hepatocyte growth factor (HGFR)/Met, Pathol. Res. Pract. 205 (2009) 57–61.

[25] M. Tsuda, I.J. Davis, P. Argani, N. Shukla, G.G. McGill, M. Nagai, T. Saito, M. Lae, D.E. Fisher, M. Ladanyi, TFE3 fusions activate MET signaling by transcriptional up-regulation, defining another class

of tumors as candidates for therapeutic MET inhibition, Cancer Res. 67 (2007) 919–929.

[26] P. Argani, M. Tsuda, M.-S. Chen, M. Edgar, M. Landanyi, Overexpression of the MET receptor tyrosine kinase in tumors with TFE3 gene fusions, Mod. Pathol. 19 (suppl. 1) (2006) 297A Ref Type: Abstract.

[27] M.A. Weterman, J.J.M. van Groningen, L. Tertoolen, A. Geurts van Kessel, Impairment of MAD2B–PRCC interaction in mitotic checkpoint defective t(X;1)-positive renal cell carcinomas, Proc. Natl. Acad. Sci. U. S. A. 98 (2001) 13808–13813.

[28] P.M. Voorhoeve, R. Agami, The tumor-suppressive functions of the human INK4A locus, Cancer Cell 4 (2003) 311–319.

[29] D.R.de Bruijn, S.V. Allander, A.H. van Dijk, M.P. Willemse, J. Thijssen, J.J.M. van Groningen, P.S. Meltzer, A. Geurts van Kessel, The synovial-sarcoma-associated SS18-SSX2 fusion protein induces epigenetic gene (de)regulation, Cancer Res. 66 (2006) 9474–9482.

[30] S. Carreira, J. Goodall, I. Aksan, S.A. La Rocca, M.D. Galibert, L. Denat, L. Larue, C.R. Goding, Mitf cooperates with Rb1 and activates p21Cip1 expression to regulate cell cycle progression, Nature 433 (2005) 764–769.

[31] D.M. Scolnick, T.D. Halazonetis, Chfr defines a mitotic stress checkpoint that delays entry into metaphase, Nature 406 (2000) 430–435.

[32] D.A. Brito, C.L. Rieder, Mitotic checkpoint slippage in humans occurs via cyclin B destruction in the presence of an active checkpoint, Curr. Biol. 16 (2006) 1194–1200.

[33] V. Dulic, G.H. Stein, D.F. Far, S.I. Reed, Nuclear accumulation of p21Cip1 at the onset of mitosis: a role at the G2/M-phase transition, Mol. Cell Biol. 18 (1998) 546–557.

[34] T.A. Chan, P.M. Hwang, H. Hermeking, K.W. Kinzler, B. Vogelstein, Cooperative effects of genes controlling the G(2)/M checkpoint, Genes Dev. 14 (2000) 1584–1588.

[35] A.K. Godwin, M.W. Lieberman, Early and late responses to induction of rasT24 expression in Rat-1 cells, Oncogene 5 (1990) 1231–1241.

[36] X. Lu, D.P. Lane, Differential induction of transcriptionally active p53 following UV or ionizing radiation: defects in chromosome instability syndromes? Cell 75 (1993) 765–778.

[37] L.A. Allan, T. Duhig, M. Read, M. Fried, The p21(WAF1/CIP1) promoter is methylated in Rat-1 cells: stable restoration of p53-dependent p21(WAF1/CIP1) expression after transfection of a genomic clone containing the p21(WAF1/CIP1) gene, Mol. Cell Biol. 20 (2000) 1291–1298.

[38] R.A. Weinberg, The retinoblastoma protein and cell cycle control, Cell 81 (1995) 323–330.

[39] C.J. Sherr, J.M. Roberts, Inhibitors of mammalian G1 cyclin-dependent kinases, Genes Dev. 9 (1995) 1149–1163.

[40] W. Du, M. Vidal, J.E. Xie, N. Dyson, RBF, a novel RB-related gene that regulates E2F activity and interacts with cyclin E in Drosophila, Genes Dev. 10 (1996) 1206–1218.

[41] D.J. Riley, E.Y. Lee, W.H. Lee, The retinoblastoma protein: more than a tumor suppressor, Annu. Rev. Cell Biol. 10 (1994) 1–29.

[42] S. Waga, G.J. Hannon, D. Beach, B. Stillman, The p21 inhibitor of cyclin-dependent kinases controls DNA replication by interaction with PCNA, Nature 369 (1994) 574–578.

[43] V.V. Ogryzko, P. Wong, B.H. Howard, WAF1 retards S-phase progression primarily by inhibition of cyclin-dependent kinases, Mol. Cell Biol. 17 (1997) 4877–4882.

[44] S.H. Khan, G.M. Wahl, p53 and pRb prevent rereplication in response to microtubule inhibitors by mediating a reversible G1 arrest, Cancer Res. 58 (1998) 396–401.

[45] J.S. Lanni, T. Jacks, Characterization of the p53-dependent postmitotic checkpoint following spindle disruption, Mol. Cell Biol. 18 (1998) 1055–1064.

[46] Z.A. Stewart, S.D. Leach, J.A. Pietenpol, p21(Waf1/Cip1) inhibition of cyclin E/Cdk2 activity prevents endoreduplication after mitotic spindle disruption, Mol.Cell Biol. 19 (1999) 205–215.

[47] M. Motwani, X. Li, G.K. Schwartz, Flavopiridol, a cyclin-dependent kinase inhibitor, prevents spindle inhibitor-induced endoreduplication in human cancer cells, Clin. Cancer Res. 6 (2000) 924–932.

[48] C. Vogel, A. Kienitz, I. Hofmann, R. Muller, H. Bastians, Crosstalk of the mitotic spindle assembly checkpoint with p53 to prevent polyploidy, Oncogene 23 (2004) 6845–6853.

[49] P.D. Aplan, Causes of oncogenic chromosomal translocation, Trends Genet. 22 (2006) 46–55.

[50] S.L. Lessnick, C.S. Dacwag, T.R. Golub, The Ewing's sarcoma oncoprotein EWS/FLI induces a p53-dependent growth arrest in primary human fibroblasts, Cancer Cell 1 (2002) 393–401.

[51] S.M. Nijman, E.M. Hijmans, S. El Messaoudi, M.M. van Dongen, C. Sardet, R. Bernards, A functional genetic screen identifies TFE3 as a gene that confers resistance to the anti-proliferative effects of the retinoblastoma protein and transforming growth factor-beta, J. Biol. Chem. 281 (2006) 21582–21587.
