
CHEMBIOCHEM

DOI: 10.1002/cbic.200900122

# Identification of OxyE as an Ancillary Oxygenase during Tetracycline Biosynthesis

Peng Wang,${}^{[a,b]}$ Wenjun Zhang,${}^{[b]}$ Jixun Zhan,${}^{[b,c]}$ and Yi Tang*${}^{[b]}$

---

The double hydroxylation of 6-pretetramid to 4-keto-anhydro-tetracycline is a key tailoring reaction during the biosynthesis of the broad-spectrum antibiotic tetracyclines. It has been shown previously by heterologous reconstitution that OxyL is a dioxygenase and is the only enzyme required to catalyze the insertion of oxygen atoms at the C-12a and C-4 positions. We report here that OxyE, a flavin adenine dinucleotide (FAD)-dependent hydroxylase homologue, is an ancillary mono-oxygenase for OxyL during oxytetracycline biosynthesis in *Streptomyces rimosus*. By using both gene disruption and heterologous reconstitution approaches, we demonstrated that OxyE plays a nonessential, but important role in oxytetracycline biosynthesis by serving as a more efficient C-4 hydroxylase. In addition, we demonstrated that partially oxidized biosynthetic intermediates can undergo various glycosylation modifications in *S. rimosus*. Our results indicate that the synergistic actions of OxyE and OxyL in the double hydroxylation step prevent accumulation of shunt products during oxytetracycline biosynthesis in *S. rimosus*.

---

## Introduction

Tetracyclines, such as oxytetracycline 1, have been used as broad-spectrum antibiotics for over a half century.[1] By targeting the 30S ribosomal subunit, tetracyclines inhibit bacterial protein synthesis with relatively few side effects. The core structure of tetracyclines is richly oxygenated, with at least six oxygen atoms contributing strong binding interactions to the target 16S RNA.[2,3] For example, the keto-enol configuration at positions C-11, C-12 and C-11a, is crucial for the biological activities of tetracyclines. Equally important are the substitutions on ring A, which include C-12a hydroxyl and critical C-4 dimethylamine moiety, which originated from reductive amination of a C-4 ketone group.[4] On the other hand, some of the oxygen atoms on the opposite periphery of the molecules have been shown to be nonessential for the biological activity. The 6-deoxytetracyclines, such as doxycycline and minocycline, exhibit improved pharmacological properties.[5] Therefore, from a biosynthetic perspective, the mechanisms of oxygenating the aromatic scaffold of tetracyclines are of great interest. The identification and the characterization of involved oxygenases can contribute to the engineered biosynthesis of tetracycline derivatives.

Tetracyclines are aromatic polyketides that are synthesized by type II polyketide synthases (PKSs) in soil-borne *Streptomyces* bacteria.[6–9] The oxytetracycline (oxy) biosynthetic gene cluster from *Streptomyces rimosus* has been sequenced recently,[10] and a minimum set of pathway enzymes that are required for the biosynthesis of anhydrotetracycline (ATC) has been identified and reconstituted in the engineered heterologous host *Streptomyces coelicolor* CH999.[4] The biosynthetic pathway for 1 starts with the assembly of the amidated decaketide backbone catalyzed by the extended minimal PKS (OxyABCD), which is then transformed into the fully aromatic intermediate 6-methylpretetramid 2 by the actions of immediate downstream tailoring enzymes OxyJKNF.[11] Subsequent tailoring steps, including hydroxylations at C-12a, C-4 (which is reductively aminated), C-5, and C-6 yield the final product 1. A total of five oxygenases are present in the oxy gene cluster (oxy-EGLRS), of which oxyE, oxyL, and oxyS encode flavin adenine dinucleotide (FAD)-dependent hydroxylases based on conserved protein sequences. OxyS (OtcC) was previously identified as the ATC oxygenase that hydroxylates C-6,[12,13] and OxyL has been confirmed to be a dioxygenase that can hydroxylate 2 at both the C-12a and C-4 positions to yield 4-keto-ATC 3 (Scheme 1).[4] The functions of the OxyE, OxyG (small quinone-forming oxygenase) and OxyR (pyridoxamine 5-phosphate oxidase) have not been resolved.

Here we aim to functionally assign the role of OxyE in the biosynthesis of 1. OxyE displays 51% sequence identity to MtmOl, an oxygenase encoded in the mithramycin biosynthetic gene cluster with no apparent function as indicated by gene inactivation.[14] In the oxy biosynthetic pathway, OxyE was originally thought to be a redundant enzyme because its deletion did not affect the production of oxytetracycline.[15]

---

[a] P. Wang,${}^{+}$  
Laboratory of Microbial Metabolism and School of Life Science and Biotechnology  
Shanghai Jiaotong University  
Shanghai 200030 (China)

[b] P. Wang${}^{+}$ W. Zhang,${}^{+}$ Dr. J. Zhan, Prof. Y. Tang  
Department of Chemical and Biomolecular Engineering  
University of California, Los Angeles  
420 Westwood Plaza, Los Angeles, CA 90095 (USA)  
Fax: (+1) 310-206-4107  
E-mail: yitang@ucla.edu

[c] Dr. J. Zhan  
Present address: Department of Biological and Irrigation Engineering  
Utah State University  
105 Old Main Hill Logan, UT 84322-4105(USA)

[+] These authors contributed equally to this work.

Tetracycline Biosynthesis

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

Scheme 1. Biosynthetic pathway of oxytetracycline. Chlorotetracycline 5, which was isolated from *S. aureofaciens* is shown for comparison purposes. Oxidation of 2 to 3 is a critical tailoring step in the biosynthetic pathway.

nally assigned to catalyze the C-5 oxidation of 5a,11a-dehydro-tetracycline (DHTC) to yield 5a, 11a-dehydrooxytetracycline (DHOTC). [10] However, this hypothesis came in doubt when an OxyE homologue was also found in the chlorotetracycline 5 biosynthetic gene cluster from *Streptomyces aureofaciens*, [15] while 5 lacked the corresponding C-5 hydroxyl. In this work, we elucidated the function of OxyE through gene disruption, as well as heterologous reconstitution studies. Through structural characterization of several novel metabolites produced by *S. rimosus* mutant strains and the *S. coelicolor* heterologous reconstitution host, we characterized the function of OxyE as a C-4 monooxygenase serving an ancillary function to OxyL during the biosynthesis of oxytetracycline in *S. rimosus*.

Results

Inactivation of *oxyE* reduced the yield of oxytetracycline and led to the emergence of a new metabolite PW1

The gene disruption experiment was first performed in *S. rimosus* (ATCC 10970) to test the necessity of OxyE in the biosynthesis of oxytetracycline. WP1, a mutant strain of *S. rimosus*, was constructed with an in-frame deletion of *oxyE* (Figure 1 A). Both the Δ*oxyE* mutant strain WP1 and wild-type strain were grown on R5 agar for eight days, followed by HPLC and LC–MS analyses in parallel. Oxytetracycline was detected from the organic extracts of the Δ*oxyE* mutant strain WP1 agar culture, however, with a yield of only ~50% compared to the wild-type strain under identical growth conditions (Figure 2A). In addition, a new metabolite PW1 7 (*m/z* 558 \[M+H\]+) emerged (Figure 2B). The UV spectrum of 7 was nearly identical to that of WJ119 4, a previously isolated, spontaneously oxidized product of 2. [11] This indicated that both 7 and 4 might share a similar chromophore and aromatic scaffold. HR-ESIMS suggested the molecular formula of C₂₆H₂₃NO₁₃ for 7 (*m/z*: calcd: 580.1067; found: 580.1066 \[M+Na\]+), and the HRMS MS/MS analysis of 7 further yielded a major MS fragment (*m/z*: calcd: 404.0746;

found: 404.0758 \[M+Na\]+), which corresponds to the same molecular formula (C₂₀H₁₅NO₇) as 4. All of the HRMS analyses indicated that 7 might be a derivative of 2, with a linked hexose moiety. By using a combination of gel filtration chromatography and reversed-phase HPLC, 7 was purified to homogeneity as a red powder at a titer of ~5 mg L⁻¹. The structure of 7 was further elucidated by using a combination of 1D and 2D NMR (Table 1) (COSY, NOESY, HMQC, HMBC) spectroscopic analyses and is shown in Scheme 2.

The anomeric carbon signal at δ<sub>C-1′</sub> 103.5 ppm (δ<sub>H-1′</sub> 4.69 ppm) and the presence of several oxymethine protons confirmed the presence of an O-glycoside in the structure of 7. The COSY and HMBC correlations observed at δ<sub>H-2′</sub> 3.47 ppm (δ<sub>C-2′</sub> 73.8 ppm), δ<sub>H-3′</sub> 3.27 ppm (δ<sub>C-3′</sub> 75.6 ppm), δ<sub>H-4′</sub> 3.44 ppm (δ<sub>C-4′</sub> 71.6 ppm), δ<sub>H-5′</sub> 3.54 ppm (δ<sub>C-5′</sub> 75.8 ppm),

![Figure 1](https://i.imgur.com/figure_1.png)

Figure 1. Inactivation of *oxyE* and *oxyL* in *S. rimosus*. A) In-frame deletion of *oxyE* by homologous recombination to generate WP1 in *S. rimosus*. B) Replacement of *oxyL* by the apramycin-resistance gene cassette *aac(3)IV* by using PCR-directed mutagenesis and homologous recombination to generate WP2.

CHEMBIOCHEM

A) 360 nm
B) 430 nm

$\Delta_{oxyL}$

$\Delta_{oxyE}$

S. rimosus wild-type

$t / \text{min}$
6   8   10  12  14  16  18  20  22

$\Delta_{oxyL}$

$\Delta_{oxyE}$

S. rimosus wild-type

$t / \text{min}$
6   8   10  12  14  16  18  20  22

Figure 2. HPLC analyses A) 360 nm and B) 430 nm of organic extracts from S. rimosus wild-type, $\Delta_{oxyE}$, and $\Delta_{oxyL}$ respectively. Compound 1 was produced by wild-type S. rimosus. The titer of 1 decreased by ~50% in $\Delta_{oxyE}$ with the emergence of a new metabolite 7. Compound 1 production was abolished in $\Delta_{oxyL}$ with the production of a major metabolite 8. Note that 8 was also produced by the wild type strain. The 430 nm trace for $\Delta_{oxyE}$ is not drawn to scale to visualize 7.

Table 1. $^1$H NMR (500 MHz, [D$_6$]DMSO) and $^{13}$C NMR (125 MHz, [D$_6$]DMSO) data for 7, 9, 10.

| No. | 7                                                                 | 9                                                                 | 10                                                                |
|-----|-------------------------------------------------------------------|-------------------------------------------------------------------|-------------------------------------------------------------------|
|     | $^{13}$C NMR $\delta$ [ppm]                                      | $^{13}$C NMR $\delta$ [ppm]                                      | $^{13}$C NMR $\delta$ [ppm]                                      |
|     | $^1$H NMR $\delta$ [ppm]                                         | $^1$H NMR $\delta$ [ppm]                                         | $^1$H NMR $\delta$ [ppm]                                         |
|     | (area, m, $J_{\text{HH}}$ [Hz])                                  | (area, m, $J_{\text{HH}}$ [Hz])                                  | (area, m, $J_{\text{HH}}$ [Hz])                                  |
| 1   | 168.5                                                             | 179.8                                                             | 187.3                                                             |
| 2   | 99.3                                                              | 97.6                                                              | 101.9                                                             |
| 3   | 166.1                                                             | 163.4                                                             | 162.8                                                             |
| 4   | 95.6                                                              | 7.39 (1H, s)                                                     | 140.6                                                             |
| 4a  | 130.2                                                             |                                                                   | 125.2                                                             |
| 5   | 136.3                                                             | 105.2                                                             | 7.21(1H, s)                                                      |
| 5a  | 137.2                                                             | 158.4                                                             | 114.9                                                             |
| 6   | 33.1                                                              | 5.11 (1H, q, 6.9)                                                 | 76.4                                                              |
| 6a  | 148.0                                                             |                                                                   | 145.8                                                             |
| 7   | 118.5                                                             | 6.82 (1H, d, 7.4)                                                 | 116.0                                                             |
| 8   | 135.7                                                             | 7.44 (1H, t, 7.7)                                                 | 7.49 (1H, t, 7.8)                                                 |
| 9   | 114.7                                                             | 6.77 (1H, d, 8.0)                                                 | 6.79 (1H, d, 8.0)                                                 |
| 10  | 161.3                                                             | 162.1                                                             | 116.1                                                             |
| 10a | 115.1                                                             |                                                                   | 114.4                                                             |
| 11  | 189.2                                                             | 185.5                                                             | 187.2                                                             |
| 11a | 105.1                                                             | 107.2                                                             | 114.4                                                             |
| 12  | 163.7                                                             | 169.7                                                             | 175.7                                                             |
| 12a | 108.2                                                             | 108.3                                                             | 120.9                                                             |
| 13  | 172.6                                                             | 172.1                                                             | 172.9                                                             |
| 14  | 28.3                                                              | 1.29 (3H, d, 6.9)                                                 | 37.9                                                              |
| 1'  | 103.5                                                             | 4.69 (1H, d, 8.0)                                                 | 5.61 (1H, d, 3.8)                                                 |
| 2'  | 73.8                                                              | 3.47 (1H, t, 8.5)                                                 | 69.7                                                              |
| 3'  | 75.6                                                              | 3.27 (1H, t, 9.0)                                                 | 65.5                                                              |
| 4'  | 71.6                                                              | 3.44 (1H, t, 9.0)                                                 | 112.0                                                             |
| 5'  | 75.8                                                              | 3.54 (1H, d, 9.5)                                                 | 140.4                                                             |
| 6'  | 170.2                                                             |                                                                   | 173.7                                                             |
| 7'  |                                                                   | 48.6                                                              |                                                                   |
| 10-OH |                                                                   | 14.0 (1H, s)                                                      |                                                                   |
| 12-OH |                                                                   |                                                                   | 14.21 (1H, s)                                                    |
| 13-NH$_2$ | 8.09 (1H, brs), 8.98 (1H, brs)                              | 7.49 (1H, brs), 9.49 (1H, brs)                                   | 9.51 (1H, brs), 9.63 (1H, brs)                                   |

and the $^1$H,$^{13}$C HMBC correlations that were observed between the anomic carbon and H-2' and H-5', as well as those between an carboxylic acid carbon signal ($\delta_{\text{C-6'}}$ 170.2 ppm) and H-4' and H-5' established that the O-glycoside moiety was a glucuronic acid. The identity and the configuration of this O-glycoside moiety were further confirmed by the NOESY experiment. In addition to the NMR spectroscopic signals of the glu-curonic acid, there were 20 carbon and 8 proton signals in $^{13}$C

and $^1$H NMR spectra, respectively. Two of the proton signals were broad and corresponded to the free amide protons ($\delta_{\text{H}}$ 8.98 and 8.09 ppm). The remaining signals were highly similar to those of 4 except for the following differences: loss of one aromatic proton (C-5) and one sp$^3$ quaternary carbon (C-6), gain of one sp$^2$ quaternary aromatic carbon ($\delta_{\text{C-5}}$ 136.3 ppm) and one sp$^3$ methine ($\delta_{\text{C-6}}$ 33.1 ppm) attached to the C-6 methyl group. The $^1$H,$^{13}$C HMBC correlation between the

Tetracycline Biosynthesis

OxyL (slow)

![Chemical Structure 1](chemical_structure_1.png)  
6-methylpretetramid 2  

$\xrightarrow{\text{OxyE}}$  

![Chemical Structure 2](chemical_structure_2.png)  
4-hydroxyl, 6-methylpretetramid 6  

$\xrightarrow{\text{OxyL (slow)}}$  

![Chemical Structure 3](chemical_structure_3.png)  
4-keto-ATC 3  

$\downarrow \Delta oxyE$  
In S. rimosus  

$\downarrow \Delta oxyL$  
In S. rimosus  

$\downarrow$ in S. coelicolor CH999  

![Chemical Structure 4](chemical_structure_4.png)  
PW1 7  

![Chemical Structure 5](chemical_structure_5.png)  
PW2 8  

![Chemical Structure 6](chemical_structure_6.png)  
JX11 10  

![Chemical Structure 7](chemical_structure_7.png)  
PW3 9  

---

**Scheme 2. Proposed biosynthesis for compounds 6, 7, 8, 9, and 10. Compound 2 can be hydroxylated at the C-4 position by both OxyE and OxyL to yield intermediate 6. However, the action of OxyL alone in S. rimosus is likely inefficient in catalyzing this step, as ΔoxyE led to the accumulation of 7, which is a shunt product that accumulates as a result of lack of C-4 hydroxylation. OxyE therefore serves an ancillary role to ensure the C-4 hydroxylation is complete, as no trace of compound 7 can be found in strains contain a functional copy of oxyE. OxyL also catalyzes the C-12a hydroxylation of 6 to yield the key intermediate 3. In the absence of OxyL in S. rimosus, the intermediate 6 is glycosylated and air oxidized to yield the shunt product 8. Compound 8 is also observed in wild-type S. rimosus, likely as a result of incomplete conversion of 6 to 3. The role of OxyE as a C-4 hydroxylase was conclusively demonstrated in the heterologous host CH999 through the biosynthesis of 6, which was recovered as the spontaneously oxidized product 10.**

anomeric proton and C-5 suggested that C-5 is hydroxylated, and the resulting hydroxyl serves as the O-linkage between the glucuronic acid and the aglycon.

The detection of 1 from the ΔoxyE mutant strain WP1 demonstrated that OxyE is not essential for the biosynthesis of 1. However, the yield of 1 was attenuated compared to that of the wild-type strain, thus indicating that OxyE likely exerted auxiliary functions. Accumulation of the shunt product 7, which is a derivative of 2, suggested that OxyE might be directly involved in the tailoring of 2. Therefore we reasoned that removal of OxyE might result in the incomplete oxidation of 2 to 3, which can lead to the shunt oxidation of 2 at C-5 in S. rimosus. The C-5 oxidation could be spontaneous, or most likely catalyzed by endogenous oxygenases in S. rimosus and prevented the air oxidation of 2 to 4. The C-5-oxidized aglycon is subsequently not recognized by the downstream oxy biosynthetic enzymes and is glucuronidated by endogenous glucuronosyltransferase yielding the shunt product 7.

Inactivation of oxyL abolished the production of oxytetracycline and increased the titer of PW2

OxyL has been previously identified as the dioxygenase that can hydroxylate 2 at both the C-12a and C-4 positions to yield 3.[4] Therefore, it appears that OxyL and OxyE might both use 2 as the substrate. To elucidate the exact function of OxyE, we disrupted the oxyL gene in the wild-type S. rimosus strain to remove the competing OxyL. The oxyL gene was removed by insertion of the acc(3)/IV cassette to afford the mutant WP2, which was confirmed by PCR (Figure 1 B). HPLC analysis of the organic extract of WP2 culture revealed the complete disappearance of oxytetracycline, with the concomitant appearance of a predominant metabolite PW2 8 (Figure 2). No trace of 7 was detected in this strain; this confirms that the presence of OxyE suppresses oxidation of C-5. LC–MS analysis showed that 8 (m/z 556 [M+H]+) displayed a UV absorption pattern similar to that of both 7 and 4, again suggesting retention of the chromophore and aromatic scaffold. The decrease in mass by 2 compared to that of 7 hinted that there might be an additional degree of unsaturation in 8. The mutant strain PW2 was

---

**ChemBioChem 2009, 10, 1544–1550**  
© 2009 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim  
www.chembiochem.org 1547

further cultured on a large scale to isolate 8 in quantities sufficient for complete structural characterization. However, 8 was highly unstable and rapidly degraded during purification. We noticed that when it was placed in a solvent containing methanol, 8 readily converted into a new compound PW3 9, which was significantly more stable. Compound 9 exhibited an identical UV/Vis spectrum as 8. HR-ESIMS revealed a molecular formula of C₂₇H₂₃NO₁₃ for 9 (m/z: calcd: 592.1067; found: 592.1056 [M+Na]⁺), consistent with the addition of one methyl group to 8. Due to its improved stability, we were able to purify 9 by reversed-phase HPLC to homogeneity. Compound 9 was obtained as a red powder with a yield of ~20 mg L⁻¹.

The NMR spectroscopic characterization of 9 is shown in Table 1, and the elucidated structure is shown in Scheme 2. The ¹³C NMR spectrum of 9 contained a total of 27 carbon signals, most of which resemble those that are observed in 4. The presence of an anomeric proton at δ<sub>H-1′</sub> 5.61 ppm (δ<sub>C-1′</sub> 100.7 ppm) indicated the existence of an O-glycoside, and the analysis of the carbon and proton signals suggested that this O-glycoside moiety in 9 was an unusual 4′,5′-anhydroglucuronic acid. This was established through the ¹H,¹³C HMBC correlations between the *sp*² methine proton δ<sub>H-4′</sub> 6.06 ppm and two non-protonated carbons (δ<sub>C-5′</sub> 140.4 ppm and δ<sub>C-6′</sub> 173.7 ppm). Further COSY and ¹H,¹³C HMBC correlations were observed among δ<sub>H-2′</sub> 4.01 ppm (δ<sub>C-2′</sub> 69.7 ppm), δ<sub>H-3′</sub> 4.02 ppm (δ<sub>C-3′</sub> 65.5 ppm), δ<sub>H-4′</sub> 6.06 ppm (δ<sub>C-4′</sub> 112.0 ppm) and the anomeric proton confirmed the identity of 4-deoxy-α-L-threo-hex-4-enopyranuronate, which is highly consistent with previously reported NMR spectra of this 4,5-anhydroglucuronic acid.[16–18] The ¹³C NMR spectrum further revealed the presence of a low-field methyl moiety δ<sub>H-7′</sub> 3.14 ppm, which showed a HMBC correlation to C-6′. This suggested that the 4′,5′-anhydroglucuronic acid was esterified by methanol and is consistent with our experimental workup.

A comparison of the remaining ¹³C and ¹H NMR spectra between 9 and 4 revealed that the aromatic proton at C-4 and the corresponding carbon signal that were observed in 4 were absent in 9. Instead, the C-4 carbon signal was shifted to δ<sub>C-4</sub> 140.6 ppm as confirmed by ¹H,¹³C HMBC correlations to the key aromatic proton signal (δ<sub>H-5</sub> 7.21 ppm), indicating the C-4 position is most likely hydroxylated. The O-linkage at C-4 was unequivocally confirmed by the ¹H,¹³C HMBC correlation of the anomeric proton to C-4. All of the carbon and proton signals summarized in Table 1 are consistent with the structure of 9 shown in Scheme 2. The molecular structure of 8 was then deduced to be the free acid derivative of 9 as shown in Scheme 2.

The abolition of oxytetracycline from the S. rimosus ΔoxyL mutant strain confirmed the vital role of OxyL, and the characterization of the major metabolite 8 revealed the catalytic role of OxyE. We reasoned that OxyE likely catalyzes the C-4 hydroxylation of 2 to yield an intermediate 4-hydroxy-6-methylpretetramid 6, which was previously isolated in a blocked mutant of S. aureofaciens.[19] In wild-type S. rimosus, 6 is subsequently hydroxylated by OxyL to yield 3. However, in the absence of OxyL, 6 can be readily glycosylated at the newly installed C-4 hydroxyl and spontaneously air oxidized at ring C to afford 8. The transfer of a 4′,5′-anhydroglucuronic acid to the C-4 hydroxyl was unexpected, and the exact reason for dehydration of the glucuronic acid is unknown. The same moiety has been reported previously in S. rimosus, in which an angiotensin-converting-enzyme inhibitor dimethyl-A58365A was modified by 4-deoxy-α-L-threo-hex-4-enopyranuronate in high yield during feeding.[16] It is notable that 8 was also present in the organic extract of the wild-type strain as indicated by HPLC and LC-MS analyses, albeit with much lower yield (Figure 2B). This further hints that the OxyL-catalyzed conversion of either 2 to 3, or 6 to 3, might be relatively slow (Scheme 2). As a result, the biosynthetic intermediates can be subjected to actions of endogenous enzymes that are not associated with the pathway and lead to formation of shunt products such as 7 and 8. OxyE thus serves as the ancillary enzyme to assist in the hydroxylation of C-4.

Heterologous reconstitution confirmed the C-4 hydroxylation activity of OxyE

Biosynthesis of 7 and 8 by ΔoxyE and ΔoxyL mutants, respectively, strongly indicated that OxyE catalyzes the hydroxylation of 2 at C-4. To directly confirm the function of OxyE, we performed heterologous expression studies. Our previous study had identified that OxyA, -B, -C, -D, -J, -K, -N, and -F were essential and sufficient to produce 2 as a major product with good yield in the heterologous host S. coelicolor CH999/pWJ119, and 2 was spontaneously oxidized to yield 4.[11] A new shuttle vector was therefore constructed in which the gene encoding OxyE was added to pWJ119 to yield pJX11. As expected, the organic extract of CH999/pJX11 contained <1% of 4, which was previously observed in CH999/pWJ119, with the accompanying appearance of a new major metabolite JX11 (10; m/z 394 [M−H]⁻). Compound 10 was isolated as an orange powder at a yield of ~25 mg L⁻¹ after reversed-phase HPLC purification. The molecular structure of 10 was then determined by 1D and 2D NMR spectroscopy as shown in Scheme 2.

The ¹³C NMR spectrum of 10 showed 20 signals, comprised of 4 aromatic CH residues, 1 methyl group, and 15 quaternary carbons; this suggests a completely aromatized scaffold similar to that of 4. Further comparison demonstrated that there were loss of one aromatic proton and one downfield hydroxyl proton, and the gain of two downfield carbon signals (δ<sub>C-4</sub> 179.0 and δ<sub>C-1</sub> 187.3 ppm) indicative of the presence of a quinone moiety in 10. The new carbonyl signals were assigned based on the ¹H,¹³C HMBC correlation between δ<sub>H-5</sub> 8.05 ppm and δ<sub>C-4</sub> 179.0 ppm. Analysis of the remaining ¹H, ¹³C, and ¹H,¹³C HMBC signals (Table 1) confirmed that 10 was a ring-A-oxidized derivative of 6. Biosynthesis of the new major metabolite 10 from CH999/pJX11 further proved that OxyE catalyzes the C-4 hydroxylation of 2 to yield 6. It is also evident from the structure of 10 that S. coelicolor lacks the glucuronosyltransferase that is present in S. rimosus that can decorate 6 into 8.

Tetracycline Biosynthesis

Antimicrobial bioassays of compounds PW1 and PW3 showed no antibiotic activities.

The antibiotic activities of 7 and 9 towards *Salmonella enterica* serovar Cerro 87, and *Escherichia coli* DH10B were tested by an agar diffusion assay.[20] The antibiotic oxytetracycline and the solvent DMSO were used as positive and negative controls, respectively. Addition of 7 and 9 did not lead to any inhibition zone up to the concentration of 100 µg mL⁻¹; this indicates that these compounds do not have antibiotic activities towards the tested strains.

## Discussion

The double hydroxylation step that converts 2 to 3 is a key tailoring step in tetracycline biosynthesis. The reaction transforms the fully aromatic ring A into a cyclohexenone that is essential for tetracycline biological activities. Our previous heterologous reconstitution and in vitro studies have demonstrated that OxyL is capable of catalyzing the C-4 and C-12a double hydroxylation reactions,[4] which is further confirmed in this work. We further demonstrated that OxyE, a FAD-dependent hydroxylase encoded in the oxy biosynthetic gene cluster, is also capable of hydroxylating C-4 of 2, a transformation that is shared by the activities of OxyL. This is supported by the following findings: 1) biosynthesis of 6 (in the form of 10) was observed upon heterologous coexpression of OxyE with enzymes responsible for biosynthesis of 2; 2) disruption of *oxyL* in *S. rimosus* led to the accumulation of the shunt product 8 retaining the C-4 hydroxyl; and 3) inactivation of *oxyE* in *S. rimosus* resulted in incomplete C-4 hydroxylation that led to the production of a new shunt metabolite 7. These results revealed the surprising finding that OxyE and OxyL have redundant catalytic activities in the tetracycline biosynthetic pathway. We propose that OxyE is encoded in the gene cluster as an ancillary C-4 hydroxylase, largely because the reaction catalyzed by OxyL alone in *S. rimosus* is incomplete, either due to low protein expression levels and/or the slow reaction kinetics. Our previous heterologous reconstitution study in *S. coelicolor* CH999 failed to identify the function of OxyE,[4] probably due to the strong expression of OxyL on the SCP2* based shuttle vector, as well as the lack of competing shunt glycosylation pathways. The synergistic actions of both OxyL and OxyE can therefore facilitate the complete turnover of 2 to 3, preventing irreversible shunt pathways in *S. rimosus* such as glucuronidation and spontaneous oxidation. On the basis of these results, we have provided a revised oxidative tailoring pathway of 2 as shown in Scheme 2. The observed ancillary function of OxyE to OxyL as a C-4 hydroxylase may be the mechanism by which multi-oxygenation takes place in other aromatic polyketides biosynthesis, such as in mithramycin synthesis. We propose that MtmOI might also catalyze C-4 hydroxylation and serve as an ancillary oxygenase to MtmOII, which is known to catalyze the C-4 and C-12a double hydroxylations.

Both shunt metabolites 7 and 8 isolated from *S. rimosus* mutant strains are O-glycosylated with glucuronic acid and 4-deoxy-α-L-threo-hex-4-enopyranuronate, respectively. These modifications are proposed to be catalyzed by the glucuronosyltransferase, which transfers activated precursors, probably UDP-glucuronic acid, to the biosynthetic intermediates. Similar glucuronidation modification has been reported previously in *S. rimosus*,[16] indicating the presence of promiscuous glycosyltransferase with broad substrate specificities. The mechanism governing the installation of different acids on 7 and 8 is not known, and it might be possible that the glucuronic acid is dehydrated after being glycosylated to 6 to yield the 4′,5′-anhydroglucuronic acid attached to 8. Interestingly, although glucuronosyltransferases are commonly observed in mammalian metabolism,[21,22] few have been described in prokaryotic organisms.[23,24] To our knowledge, this is the first isolation of glucuronidated tetracycline biosynthetic intermediates, although glucosylated, truncated tetracycline intermediates have been reported to be produced by a *S. rimosus* mutant.[25]

In summary, by using gene disruption and heterologous reconstitution experiments, we have characterized the function of OxyE as a C-4 monooxygenase during oxytetracycline biosynthesis in *S. rimosus*. Our results demonstrated the power of combining gene knockout and heterologous reconstitution techniques in deciphering the functional role of enzymes in natural product biosynthesis, and provided new insight into the oxidation tailoring reactions that take place during tetracycline biosynthesis.

## Experimental Section

### General

1D and 2D NMR spectra were recorded in [D₆]DMSO on a Bruker DRX 500 instrument (500 MHz for ¹H NMR and 125 MHz for ¹³C NMR). LC–MS analyses were conducted with a Shimadzu 2010 EV Liquid Chromatography Mass Spectrometer by using both positive and negative electrospray ionization and a Phenomenex Luna reverse-phase C18 column (5 µm 2.0×100 mm). High-resolution ESIMS were measured on an IonSpec Ultima 7T FTICR instrument. Compound isolations were performed on a Beckman-Coulter Gold HPLC instrument.

### Culture techniques and genetic manipulations

*S. coelicolor* CH999 was used as the host for heterologous expression for all pRM5-derived plasmids. Protoplast preparation and PEG-mediated transformation were performed as described previously.[26] The transformants were grown on R5 agar that was supplemented with thiostrepton (50 µg mL⁻¹) at 28 °C for 8 d. *E. coli* Topo 10 (Invitrogen) and XL-1 Blue (Stratagene) strains were used for subcloning and plasmid manipulations. Growth of *E. coli* was on Luria-Bertani (LB) agar and in LB liquid medium with kanamycin (35 µg mL⁻¹) or ampicillin (100 µg mL⁻¹) at 37 °C. In vivo generation of targeted mutations in *S. rimosus* was achieved by conjugation between *E. coli* ET12567 and Streptomyces strains according to Kieser et al.[26]

### Construction of pJX11

PCR was performed with Platinum Pfx DNA polymerase (Invitrogen). The *oxyE* gene with the ribosomal binding site flanked by Xbal/Nhel was amplified through PCR by using the cosmid pYT264 as the template.[10] The PCR product was cloned into pCR-Blunt vector (Invitrogen) for subcloning and *oxyE* gene was then introduced into pWJ119[11] to yield pJX11 by using compatible cohesive ends Xbal/Nhel.

### Targeted disruption of *oxyE* and *oxyL*

The vector pPW104 which harbored *oxyE* was constructed by the recovery of an approximate-

ly 4.1 kb DNA fragment after KpnI and BglII digestion of pYT264 for insertion into the KpnI and BamHI sites of pJTU1278. This vector was digested with Apal then self-ligated to yield pPW105. A construct that harbored oxyL to be used to generate a gene replacement vector was constructed by using the ReDirect technology.[27] A ca. 8.6 kb PmlI fragment that contained oxyL was recovered from cosmid pYT264 and inserted into PCR Blunt vector to generate pPW164. This plasmid was introduced into E. coli BW25113/ plJ790 by electroporation. The acc(3)IV cassette was amplified by PCR from plJ773 with the primers oxyL-KO-F: 5′-CCC TGG CCC TCG CCA AGC GCT TCG CGG AGG CCC CGC ATG ATT CCG GGG ATC CGT CGA CC-3′ and oxyL-KO-R: 5′-CGC CGT TGA TCG CTG TCA TAC GCG TGT CCT TGG TCA TGT AGG CTG GAG CTG CTT C-3′. The cassette was used to replace a 1674 bp portion of oxyL in pPW164, to give pPW165. Digestion with EcoRI yielded a product that was cloned into the same site of pJTU1278 to give pPW166. Both pPW105 and pPW166 DNA were introduced to S. rimosus by conjugation from E. coli ET12567 to S. rimosus. Putative double-crossover strains were confirmed by PCR with the primers of oxyE-T1: 5′-TGC TGA CCG ATC TGC TGG AG-3′, oxyE-T2: 5′-ATC CGG CTC TGC ATC ACC TG-3′, oxyL-T1: 5′-ATA CGG GAC ACC GGG GAC GG-3′ and oxyL-T2: 5′-GCC CTC CTC GGA CAG CTG CC-3′.

Isolation of 7: The large-scale culturing experiment of strain WP1 was performed in R5 agar (4 L) over 8 d. The following separation procedures were used to isolate the pure compound. The cultures (4 L) were homogenized and extracted three times with EtOAc/ MeOH/AcOH (89:10:1, 3 L). The extracts were dried with anhydrous Na₂SO₄ and evaporated to dryness. These extracts were first loaded into a Sephadex LH-20 column and eluted with MeOH. The fractions containing the target compound were then further separated on an XTerra preparative MSC18 column (5 μm, 50×19 mm) with isocratic elution (MeCN/H₂O, 40%, 0.1% TFA) at a flow rate of 2.5 mL min⁻¹, yielding pure 7 (t_R=12 min) with a yield of ~5 mg L⁻¹.

Isolation of 9: The large-scale culturing experiment of strain WP2 was performed in R5 agar (4 L) over 8 d. The extracts were fractionated on a Sephadex LH-20 column and eluted with MeOH. The fractions containing 9 were then further separated on an XTerra preparative MSC18 column (5 μm, 50×19 mm) with isocratic elution (MeOH/H₂O, 70%, 0.1% TFA) at a flow rate of 3 mL min⁻¹, yielding pure 9 (t_R=8 min) with a yield of ~20 mg L⁻¹.

Isolation of 10: The strain CH999/pJX11 was grown on R5 agar (1 L) with thiostrepton (50 μg mL⁻¹) at 30 °C for 10 d. The culture was minced and extracted with EtOAc/MeOH/AcOH (89:10:1, 1.5 L) three times. The extract was dried with anhydrous Na₂SO₄ and evaporated to dryness. The residue was dissolved in DMSO (2.5 mL) and separated on an Alltima semi-preparative C18 column (5 μm, 250×10 mm), with a gradient elution from 5% to 95% MeCN/H₂O (0.1% TFA) over 30 min at a flow rate of 1 mL min⁻¹, to give pure 10 (t_R=20 min) with a yield of ~25 mg L⁻¹

Acknowledgements

This work is funded by NSF CBET 0545860. Peng Wang is supported by China Scholarship Council and Wenjun Zhang is partially supported by Nell I. Mondy Fellowship.

[1] I. Chopra, P. M. Hawkey, M. Hinton, J. Antimicrob. Chemother. 1992, 29, 245–277.
[2] D. E. Brodersen, W. M. Clemons, Jr., A. P. Carter, R. J. Morgan-Warren, B. T. Wimberly, V. Ramakrishnan, Cell 2000, 103, 1143–1154.
[3] M. Pioletti, F. Schluenzen, J. Harms, R. Zarivach, M. Gluhmann, H. Avila, A. Bashan, H. Bartels, T. Auerbach, C. Jacobi, T. Hartsch, A. Yonath, F. Franceschi, Embo J. 2001, 20, 1829–1839.
[4] W. Zhang, K. Watanabe, X. Cai, M. E. Jung, Y. Tang, J. Zhan, J. Am. Chem. Soc. 2008, 130, 6068–6069.
[5] M. G. Charest, C. D. Lerner, J. D. Brubaker, D. R. Siegel, A. G. Myers, Science 2005, 308, 395–398.
[6] B. J. Rawlings, Nat. Prod. Rep. 1999, 16, 425–484.
[7] E. S. Kim, M. J. Bibb, M. J. Butler, D. A. Hopwood, D. H. Sherman, Gene 1994, 141, 141–142.
[8] A. Thamchaipenet, L.-H. Zhou, D. Hranueli, P. Raspor, P. G. Waterman, I. S. Hunter, J. Biol. Chem. 1999, 274, 32829–32834.
[9] I. S. Hunter in Microbial Secondary Metabolites: Biosynthesis, Genetics and Regulation (Eds.: F. Fierro, J. F. Martin), Research Signpost, Lucknow, 2002, pp. 141–166.
[10] W. Zhang, B. D. Ames, S. C. Tsai, Y. Tang, Appl. Environ. Microbiol. 2006, 72, 2573–2580.
[11] W. Zhang, K. Watanabe, C. C. Wang, Y. Tang, J. Biol. Chem. 2007, 282, 25717–25725.
[12] C. Binnie, M. Warren, M. J. Butler, J. Bacteriol. 1989, 171, 887–895.
[13] N. A. Peric-Concha, B. Borovicka, P. F. Long, D. Hrnaueli, P. G. Waterman, I. S. Hunter, J. Biol. Chem. 2005, 280, 37455–37460.
[14] L. Prado, F. Lombó, A. F. Brana, C. Mendez, J. Rohr, J. A. Salas, Mol. Gen. Genet. 1999, 261, 216–225.
[15] “Cloning of the Biosynthetic Pathway for Chlorotetracycline and Tetracycline Formation and Cosmids Useful Therein”, M. J. Ryan, J. A. Lotvin, N. Strathy, S. F. Fantini, US Patent 5866410, 1999.
[16] J. S. Mynderse, D. S. Fukuda, A. H. Hunt, J. Antibiot. 1995, 48, 425–427.
[17] C. Delattre, P. Michaud, J. M. Lion, B. Courtois, J. Biotechnol. 2005, 118, 448–457.
[18] J. Alföldie, R. Palovcik, C. Peciar, J. Hirsch, P. Kovac, Carbohydr. Res. 1975, 44, 133–137.
[19] J. R. D. McCormick, U. H. Joachim, E. R. Jensen, S. Johnson, N. O. Sjoland, J. Am. Chem. Soc. 1965, 87, 1793–1794.
[20] I. Wiegand, K. Hilpert, R. E. Hancock, Nat. Protoc. 2008, 3, 163–175.
[21] A. Radominska-Pandya, J. M. Little, P. J. Czernik, Curr. Drug Metab. 2001, 2, 283–298.
[22] U. A. Argikar, R. P. Remmel, Drug Metab. Dispos. 2009, 37, 229–236.
[23] M. Barreras, S. R. Salinas, P. L. Abidian, M. A. Kampel, L. Ielpi, J. Biol. Chem. 2008, 283, 25027–25035.
[24] S. J. Gould, J. Guo, J. Bacteriol. 1994, 176, 1282–1286.
[25] M. A. Deseo, S. Hunter, P. G. Waterman, J. Antibiot. 2005, 58, 822–827.
[26] M. J. B. T. Kieser, M. J. Buttner, K. F. Charter, D. A. Hopwood, Practical Streptomyces Genetics, The John Innes Foundation, Norwich, 2000.
[27] B. Gust, G. L. Challis, K. Fowler, T. Kieser, K. F. Chater, Proc. Natl. Acad. Sci. USA 2003, 100, 1541–1546.

Received: March 7, 2009
Published online on May 26, 2009

1550 www.chembiochem.org Ⓒ 2009 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim ChemBioChem 2009, 10, 1544–1550
