
The Role of Ubiquitin in NF-κB Regulatory Pathways

Brian Skaug,$^{1}$ Xiao-Mo Jiang,$^{1}$ and Zhijian J. Chen$^{1,2}$

$^{1}$Department of Molecular Biology, $^{2}$Howard Hughes Medical Institute, University of Texas Southwestern Medical Center, Dallas, Texas 75390-9148; email: zhijian.chen@utsouthwestern.edu

### Key Words
apoptosis, deubiquitination, IKK, TAK1

### Abstract
Nuclear factor kappa enhancer binding protein (NF-κB) regulates diverse biological processes including immunity, inflammation, and apoptosis. A vast array of cellular stimuli converges on NF-κB, and ubiquitination plays an essential role in the coordination of these signals to regulate NF-κB activity. At least three steps in NF-κB activation directly involve ubiquitination: proteasomal degradation of inhibitor of NF-κB (IκB), processing of NF-κB precursors, and activation of the transforming growth factor (TGF)-β-activated kinase (TAK1) and IκB kinase (IKK) complexes. In this review, we discuss recent advances in the identification and characterization of ubiquitination and deubiquitination machinery that regulate NF-κB. Particular emphasis is given to proteasome-independent functions of ubiquitin, specifically its role in the activation of protein kinase complexes and in coordination of cell survival and apoptosis signals downstream of tumor necrosis factor α (TNFα).

INTRODUCTION

As a central regulator of cell survival and the response to microbial invasion and inflammatory cytokines, nuclear factor kappa enhancer binding protein (NF-κB) has naturally become the focus of intense research since its discovery over 20 years ago (1, 2). Likewise, ubiquitination has become a prevalent topic in many fields of biology research, owing to its role in regulation of numerous cellular processes (3, 4). Studies aimed at understanding NF-κB regulation have led to the identification and characterization of ubiquitin ligases, ubiquitin-binding proteins, and deubiquitination enzymes (DUBs) as central players in many signaling pathways converging on NF-κB (5). Some of these studies have led to insights into unique biochemical processes including proteasomal processing of NF-κB precursors and activation of protein kinases by ubiquitination.

Cell-free, biochemical approaches have been key to identifying the molecular machinery involved and the means by which protein ubiquitination and deubiquitination contribute to NF-κB regulation. Subsequent data from cell culture and animal models have largely validated the importance of ubiquitination in IkB degradation, NF-κB precursor processing, and kinase activation. However, mechanistic details have yet to emerge of how ubiquitination and deubiquitination are regulated, how ubiquitination promotes kinase activation, and how dysfunction of ubiquitination and deubiquitination machinery contributes to the pathogenesis of diseases associated with dysregulation of NF-κB.

### Ubiquitination Enzymes

Ubiquitin is a 76-amino acid polypeptide, which is covalently attached to proteins in a three-step enzymatic cascade (3). First, the ubiquitin-activating enzyme (E1) forms a thiol ester bond between the C-terminal glycine of ubiquitin and the active-site cysteine of E1 in an ATP-dependent reaction. The activated ubiquitin is transferred to a cysteine of

a ubiquitin-conjugating enzyme (E2 or UBC), forming an E2-ubiquitin thioester. Finally, in the presence of a ubiquitin-protein ligase (E3), ubiquitin is transferred to a target protein, forming an isopeptide bond between the C-terminal glycine of ubiquitin and the ε-amino group of a lysine residue on the target protein. Ubiquitin contains seven lysine residues, each of which can be covalently attached to another ubiquitin, resulting in the formation of a polyubiquitin chain. The linkage of the polyubiquitin chain, e.g., lysine 48 (K48) versus lysine 63 (K63), can influence the fate of the ubiquitinated substrate (4).

E3 enzymes can be broadly categorized into two families: the really interesting new gene (RING) type and homologous to the E6-associated protein C terminus (HECT) type (6, 7). The RING-finger domain contains a characteristic series of histidine and cysteine residues whose spacing allows coordination of two zinc ions. RING-type E3 ligases can be further subdivided into two groups. In single-protein E3 ligases, such as Casitas B-lineage lymphoma (Cbl), the RING-finger and substrate-recruiting domains are part of the same protein, whereas in multisubunit E3 ligases, the RING-finger domain is part of a multiprotein complex. For example, in the Skp1-Cullin-F-box protein (SCF) E3 enzymes, the RING-finger domain protein RING-box-1 (RBX1), also known as ROC1, binds to ubiquitin-loaded E2 and forms a complex with scaffold proteins SKP1 and CUL1 along with an F-box protein. F-box proteins normally contain domains that recognize specific protein substrates. The HECT-type E3 ligases contain a conserved cysteine residue within the HECT domain that forms a thiol-ester bond with ubiquitin. The ubiquitin on the HECT domain can be transferred directly to protein substrates (8).

substrates to the execution of cellular processes (9, 10). A combination of biochemical and bioinformatics approaches has led to the identification of close to 20 different types of UBDs, which are quite diverse in terms of size (20–150 amino acids), structure, and the function of UBD-containing proteins.

Although many UBDs were identified on the basis of their ability to bind monoubiquitin, some UBDs prefer to bind polyubiquitin chains and actually have higher affinities for polyubiquitin chains of a specific linkage. In most cases, binding affinities between ubiquitin and UBDs are relatively low (typically 10–500 μM). Nonetheless, mutations in UBDs that inhibit ubiquitin binding disrupt protein function in vivo, suggesting that these low-affinity interactions are physiologically significant (11–13).

**Inflammatory cytokines:** secreted proteins that promote inflammatory and immune responses. Examples include TNFα, IL-1β, and IL-6

**NF-κB:** nuclear factor kappa enhancer binding protein

**DUB:** deubiquitination enzyme

**IkB:** inhibitor of NF-κB

**UBD:** ubiquitin-binding domain

### Proteasome

The 26S proteasome contains a 20S core particle and a 19S regulatory complex (14). The core particle has two heptameric rings of β-subunits sandwiched between two heptameric rings of α-subunits, forming a barrel-shaped structure. The proteolytic sites are within the two innermost rings. Polyubiquitinated proteins are recognized by the regulatory complex, unfolded in an ATP-dependent process, deubiquitinated, and translocated through a narrow channel into the core particle. Within the core particle, hydrolysis of peptide bonds leads to the formation of peptides varying from 3 to 23 amino acids in length. A chain of at least four ubiquitin molecules appears to be required for efficient binding to the proteasome. Ubiquitinated substrates are recruited to the proteasome both by ubiquitin receptors intrinsic to the proteasome and proteasome accessory proteins that contain UBDs.

### Ubiquitin-Binding Domains

Just as phosphorylation is detected by specialized phosphopeptide-binding domains, conserved ubiquitin-binding domains (UBDs) exist to transduce signals from ubiquitinated

### Deubiquitination Enzymes

Much like other covalent modifications of proteins, ubiquitination is a reversible process. Indeed, according to one estimate there are close to 100 DUBs in the human genome

(15, 16). Most DUBs are cysteine proteases, whereas a few are metalloproteases containing the JAB1/MPN/Mov34 (JAMM) domain. The cysteine proteases can be further categorized on the basis of their protease domains into ubiquitin-specific proteases (USP), ubiquitin C-terminal hydrolases (UCH), ovarian tumor (OTU)-type proteases, and Machado-Joseph disease proteases (MJD).

A combination of genetic and biochemical evidence points to a crucial role for deubiquitination in mammalian physiology. In the context of NF-κB pathways, the familial cylindromatosis tumor suppressor (CYLD) and A20 both negatively regulate IKK activity through deubiquitination of upstream signaling proteins, and dysfunction of these enzymes has deleterious consequences (discussed in detail below).

**THE NF-κB SIGNALING PATHWAYS**

The NF-κB family of transcription factors consists of p50, p52, p65 (also known as REL-A), c-REL, and REL-B (2). p50 and p52 are produced by proteasomal processing of the precursors p105 and p100, respectively. All of the NF-κB family members contain an N-terminal REL-homology domain (RHD), which is responsible for homo- and heterodimerization, DNA binding, nuclear translocation, and interaction with IkB proteins. NF-κB subunits form dimers that can shuttle between the nucleus and cytoplasm. Their activation is regulated by IkB proteins, which are bound to NF-κB dimers under basal conditions and retain NF-κB dimers within the cytoplasm. Stimulation of cells with any of a multitude of agonists leads to a rapid cascade of IkB phosphorylation, polyubiquitination, and proteasomal degradation. Spared from degradation and now free from their IkB binding partners, NF-κB dimers can translocate to the nucleus and regulate transcription of target genes.

The IkB proteins consist of IkBa, IkBβ, and IkBe, which inhibit NF-κB, as well as the atypical IkBg and Bcl-3, which function as NF-κB coactivators. All of the IkBs contain multiple ankyrin repeat domains that bind to NF-κB dimers. p105 and p100 also contain ankyrin repeats, which are degraded upon maturation to p50 and p52, respectively.

**Canonical NF-κB Pathways**

Stimulation of cells with agonists such as tumor necrosis factor (TNF)-α, interleukin-1β (IL-1β), or Toll-like receptor (TLR) ligands activates the canonical pathway of NF-κB activation (Figure 1). The signals transduced by the receptors for each of these ligands converge on the IkB kinase (IKK) complex, consisting of two catalytic subunits, IKKa and IKKβ, and the regulatory subunit NF-κB essential modulator (NEMO, also known as IKKy). IKK phosphorylates IkBa at serines 32 and 36, leading to polyubiquitination of lysines 21 and 22 and subsequent degradation by the 26S proteasome (5). Analyses of IKKβ- and IKKa-deficient cells have shown that IKKβ makes a more significant contribution to canonical NF-κB activation than does IKKa. NEMO appears to be essential for canonical NF-κB activation, as NEMO-deficient cells fail to activate NF-κB in response to IL-1β, TNFα, and lipopolysaccharides (LPS) (17).

The NF-κB dimer p50/p65 is primarily bound to IkB proteins under basal conditions, but rapid degradation of IkBs following stimulation allows p50/p65 to accumulate in the nucleus and activate gene transcription. One of the genes potently upregulated by NF-κB is IkBa itself, which can enter the nucleus, displace NF-κB from DNA, and transport it back to the cytoplasm, forming a negative feedback loop. Therefore, without persistent upstream activation signals, NF-κB is rapidly resequestered by IkBa. The traditional IkB proteins IkBa, IkBβ, and IkBe have similar but not entirely overlapping functions. Their differences likely result from temporal differences in their degradation and resynthesis (18).

ANRV378-BI78-27 ARI 29 April 2009 19:34

![Diagram](#)

Figure 1

Two pathways leading to NF-κB activation. In canonical NF-κB activation (left), stimulation of the TNF receptor (TNFR), IL-1 receptor (IL-1R), and Toll-like receptors (TLRs) leads to activation of the TAK1 complex through TRAF proteins. TAK1 then activates IKK, which in turn phosphorylates IkB proteins and targets them for polyubiquitination by the SCF-βTrCP E3 ligase complex. Ubiquitinated IkB is degraded by the proteasome, allowing the p50/p65 NF-κB dimer to enter the nucleus and activate gene transcription. In noncanonical NF-κB activation (right), stimulation of a subset of receptors, including the BAFF receptor, leads to the stabilization of the kinase NIK, followed by activation of IKKα. IKKα phosphorylates p100, leading to its ubiquitination by the SCF-βTrCP complex. Ubiquitinated p100 is targeted for proteasomal processing to p52. The p52/REL-B dimer then translocates into the nucleus to activate gene transcription.

## Noncanonical NF-κB Pathways

Stimulation of a subset of TNF receptor (TNFR) superfamily members, including those for BAFF, CD40 ligand, and lymphotoxin-β, induces NF-κB activation through another proteasome-dependent mechanism, in this case dependent upon NF-κB-inducing kinase (NIK) and the IKKα subunit (19, 20). Following activation of NIK and IKKα, p100 is phosphorylated by IKKα, then targeted for polyubiquitination and subsequent proteasomal processing to p52 (Figure 1).

Proteasomal processing of p105 occurs cotranslationally and posttranslationally, and can result in its complete degradation or formation of p50. The prevalence of each type of processing, as well as the role of ubiquitination in these processes, has been controversial. Processing of p105 to p50 was reconstituted in vitro and required ubiquitination and the proteasome (21). Another group demonstrated that p50 is produced cotranslationally in a proteasome-dependent process (22). However, expression of a dominant-negative UBCH5 (an E2) inhibited

www.annualreviews.org • Ubiquitin in NF-κB Regulatory Pathways 773

IKKβ-dependent processing, but not basal processing (23), and it was recently shown that the 20S proteasome can process full-length p105 to p50 in a ubiquitin-independent process (24). Thus, it appears that basal processing of p105 to p50 does not require ubiquitination, whereas signal-induced processing does.

How does the proteasome distinguish between different regions of proteins, such that the C termini of p105 and p100 are degraded while the N termini remain intact? Jentsch and colleagues (25) proposed that proteasomal processing begins at a hairpin-like loop region within the protein, then proceeds in both the C-terminal and N-terminal directions. Both p100 and p105 contain a glycine-rich region, which appears to function as a processing signal (26). While degradation proceeds through the entire C terminus, N-terminal processing is halted by a region of stable tertiary structure and/or the presence of another associated protein, i.e., the other NF-κB subunit of a dimer. Indeed, initiation of proteasomal processing at an internal site was recently demonstrated using p105 and the yeast transcription factors Spt23 and Mga2 (24, 27).

**UBIQUITIN-MEDIATED ACTIVATION OF TAK1 AND IKK**

In both canonical and noncanonical NF-κB pathways, stimulation of cell surface receptors, such as TNFR, IL-1R, TLR, and CD40, leads to the recruitment of members of the TNF receptor-associated factors (TRAFs), such as TRAF2 and TRAF6, to the receptors (28). With the exception of TRAF1, all other TRAF proteins (TRAF2–7) contain a highly conserved N-terminal RING domain, and biochemical experiments demonstrate that these TRAF proteins are ubiquitin ligases (29). In an effort to understand the mechanism of IKK activation by TRAF6, a cell-free system was established by adding recombinant TRAF6 protein to cytosolic extract, which led to IKK activation (29). Fractionation of cell extract led to the identification of two distinct factors that are required for this activity: TRAF6-regulated IKK activator 1 (TRIKA1) and TRIKA2. TRIKA1 was identified as the dimeric ubiquitin-conjugating enzyme complex UBC13/UEV1A. TRAF6 and UBC13/UEV1A facilitated synthesis of K63-, but not K48-, linked polyubiquitin chains. Moreover, using ubiquitin mutants in which lysines were mutated to arginine individually or in combination, it was shown that TRAF6-dependent IKK activation required K63, but not K48, of ubiquitin.

TRIKA2 was later identified as the transforming growth factor-β-activated kinase-1 (TAK1) complex consisting of TAK1 and the regulatory subunits TAK1-binding protein 1 (TAB1) and TAB2 (30). Immunopurified TAK1 complex phosphorylated IKK and the MAP kinase kinase MKK6 after incubation of the TAK1 complex with TRAF6, UBC13/UEV1A, and ubiquitin. K63, but not K48, of ubiquitin was required for this activity. These results suggest that K63-linked polyubiquitination has an important proteasome-independent regulatory function in stress kinase pathways (Figure 2).

**Role of TRAF6-Catalyzed K63 Polyubiquitination in the IL-1R/TLR Pathways**

The creation of TRAF6-deficient mice revealed the importance of TRAF6 in diverse physiological processes. These mice have dysregulated bone metabolism leading to osteopetrosis. In addition, TRAF6-deficient cells have a severe deficiency in NF-κB activation in response to IL-1β, CD40, and several TLR ligands such as LPS (31, 32).

Despite the evidence that TRAF6 is a ubiquitin ligase required for NF-κB activation by the stimuli listed above, the requirement of the TRAF6 RING domain for activation of NF-κB is still not entirely clear. In overexpression studies, a RING-domain deletion mutant was partially defective in NF-κB activation compared to full-length TRAF6, whereas further deletion of several zinc fingers caused severe impairment (33). In complementation studies using embryonic fibroblasts (MEFs) from TRAF6-deficient

IL-1β

IL-1R/TLRs

MyD88
IRAK
TRAF6
TAK1
TAB2
NEMO
IKKβ
IKKα

CYLD
A20

Proteasome

Phosphorylation
K63-linked ubiquitination
K48-linked ubiquitination

p65
p50

Figure 2

Ubiquitin-mediated activation of transforming growth factor-β-activated kinase-1 (TAK1) and IkB kinase (IKK) in the IL-1R/Toll-like receptor (TLR) pathway. Binding of interleukin-1β (IL-1β) to IL-1R or LPS to TLR4 causes recruitment of MyD88, IRAK, and TRAF6 to the receptor. TRAF6, an E3 ubiquitin ligase, catalyzes synthesis of K63-linked polyubiquitin chains, some of which are conjugated to TRAF6 itself. The polyubiquitin chains function as a scaffold to recruit the TAK1 and IKK complexes through binding to the regulatory subunits, TAB2 and NEMO, respectively. Recruitment of the kinase complexes facilitates autophosphorylation of TAK1 and subsequent phosphorylation of IKKβ by TAK1, leading to IkB degradation and subsequent activation of NF-κB (represented by the p50/p65 dimer). K63-linked polyubiquitin chains can be disassembled by deubiquitination enzymes including CYLD and A20, which inhibit IKK activation.

mice, full-length TRAF6 as well as deletion mutants lacking the RING domain restored IL-1β- and LPS-induced NF-κB activation, despite the inability of these mutants to restore activation of TAK1 (34). By contrast, a point mutation within the RING domain of TRAF6 (C70A), which abrogated its ubiquitination activity, severely impaired its ability to induce NF-κB activation in an overexpression experiment, as well as its ability to restore IL-1β- and

RANKL-induced IKK activation in TRAF6-deficient cells (35, 36). Further studies are required to resolve these apparent discrepancies and to better understand the function of the TRAF6 RING domain in vivo.

In addition to catalyzing the synthesis of K63-linked polyubiquitin chains, TRAF6 is also one of the “targets” to which ubiquitin is attached. A specific lysine, K124, of TRAF6 has been identified as the key acceptor site
for ubiquitination, and a mutation of K124  
abrogated its ability to restore IL-1β-induced  
IKK activation in TRAF6-deficient MEFs (35).  
However, TRAF6 and UBC13/UEV1A catalyze synthesis primarily of free polyubiquitin chains, rather than TRAF6-conjugated chains,  
in vitro (37, 38). It has also been proposed  
that K63 polyubiquitination of IL-1β receptor-associated kinase 1 (IRAK1) is important for  
IKK activation in the IL-1 pathway (38, 39).  
Therefore, further studies are required to clarify the physiological role of polyubiquitination  
of TRAF6, IRAK1, and other potential targets (e.g., NEMO, see below) in the IL-1/TLR  
pathways.  

How does TRAF6 catalyze K63-linked  
polyubiquitin chain synthesis? Binding of  
IL-1β to its receptor causes recruitment of the  
proteins MyD88, IRAK1, IRAK4, and TRAF6.  
The C-terminal TRAF domain of TRAF6  
mediates interactions with upstream signaling components, the function of which may  
be to induce oligomerization of TRAF6 (5).  
This model is consistent with the fact that a  
TRAF6 mutant, in which the TRAF domain  
is replaced by bacterial gyrase B domain, can  
support TRAF6 autoubiquitination and IKK  
activation upon forced dimerization with the  
small-molecule coumermycin (30). Nonetheless,  
it is still not clear how dimerization or  
oligomerization of TRAF6 activates its ubiquitin E3 activity. Biochemical and structural  
information will be required to clearly understand the mechanism of polyubiquitin chain  
elongation by TRAF6 and other E3 ligases.  

The importance of UBC13 for IKK activation has been assessed in studies using conditional *Ubc13* knockout and haploinsufficient (*Ubc13*+/−) mice because homozygous deletion is embryonic lethal. Bone marrow-derived macrophages (BMDMs) from mice with conditional *Ubc13* deletion demonstrated reduced cytokine production in response to various TLR ligands. These cells, as well as *Ubc13*-deficient B cells and fibroblasts, showed defective MAPK activation but little or no impairment in IκBα degradation in response to various TLR ligands, IL-1β, and TNFα (40). Polyubiquiti-  

nation of NEMO but not TRAF6 was impaired in *Ubc13*-deficient MEFs treated with  
TNFα or IL-1β. T cell-specific *Ubc13* knockout caused a deficiency in peripheral T cell development, and thymocytes from these mice proliferated less than those from wild-type mice in response to anti-CD3/CD28 stimulation or PMA/ionophore stimulation (41). NF-κB and MAPK activation was severely defective in response to stimulation with these ligands, likely due to reduced TAK1 activation. *Ubc13*+/− mice were healthy and had normal numbers of immune cells but demonstrated blunted responses to inflammatory stimuli (42). They were protected from LPS challenge compared to wild-type mice, likely owing to impaired degradation of IκBα and MAPK activation leading to reduced cytokine secretion. Taken together, these studies demonstrate that UBC13 has a critical role in the development and homeostasis of the immune system. In the context of T cell development and activation, this function likely involves ubiquitination-dependent activation of TAK1 leading to MAPK and IKK activation. However, the requirement of UBC13 for NF-κB activation appears to vary among different cell types. It is not clear why deletion of *Ubc13* did not abolish IKK activation in some cell types. One possibility is that other E2s may compensate for the loss of *Ubc13* in certain cell types. Consistent with this hypothesis, UBC5 is known to catalyze the synthesis of polyubiquitin chains of various linkages. In fact, UBC5 is capable of activating IKK in vitro (43).  

Multiple lines of evidence strongly support the essential role of TAK1 in NF-κB and MAPK activation in response to numerous inflammatory stimuli: TAK1 directly phosphorylates and activates IKKβ and MKK6 in a ubiquitin-dependent manner (30); RNAi knockdown and pharmacological inhibition of TAK1 reduce IL-1β- and TNFα-induced IKK and c-Jun N-terminal kinase (JNK) activation (12, 44, 45); mutations of *Tak1* in *Drosophila melanogaster*, or silencing of *Tak1* in a *Drosophila* cell line, impair IKK and JNK activation in response to bacterial challenge (46–48); and deletion of *Tak1* in multiple mouse cell types  
abrogates IKK and MAPK activation by a variety of NF-κB agonists (49–52). Although TAB2-deficient cells exhibit normal IL-1β-induced NF-κB activation (53), RNAi against TAB2 and its homolog TAB3 reduces IL-1β-and TNFα-induced IKK activation (12, 54), indicating some redundancy between these two regulatory subunits. By contrast, TAB1 appears to be required for TAK1 activation by osmotic stress but not by proinflammatory cytokines (55).

**Role of TRAF2 and RIP1 in the TNFR Pathway**

Binding of TNFα to its receptor (TNFR) leads to the receptor recruitment of several signaling proteins, including TNFR1-associated death domain protein (TRADD), TRAF2, and the receptor-interacting protein kinase (RIP1) (56). Like TRAF6, TRAF2 overexpression is sufficient to induce activation of the canonical NF-κB activation pathway, and this activity is blocked by a catalytically inactive mutant of UBC13 (29). Overexpressed TRAF2 is polyubiquitinated by K63-linked ubiquitin chains in a manner that depends on UBC13 (57). Although TRAF2-deficient MEFs can still activate NF-κB in response to TNFα (58, 59), MEFs lacking both TRAF2 and TRAF5 are deficient in IKK activation by TNFα, suggesting a certain degree of functional redundancy between TRAF2 and TRAF5 (60). Like TRAF6, TRAF2 activity is likely promoted by oligomerization, because an FKBP12-TRAF2 N-terminal chimera induces IKK activation in the presence of the small molecule FK1012, which induces oligomerization of FKBP12 (61).

RIP1 has emerged as an important target for polyubiquitination in the TNFα receptor pathway (Figure 3). RIP1 is required for IKK activation by TNFα and is polyubiquitinated with K63-linked chains within minutes of TNFα stimulation (11, 62–64). Complementation experiments using RIP1-deficient MEFs revealed that K377 is the key acceptor residue, and a K377R mutant did not restore TNFα-induced

IKK activation, whereas other lysine mutants did. The function of the polyubiquitin chain on RIP1 appears to be recruitment of the TAK1 and IKK complexes to TNFR, because the K377R mutant abolished TAK1 and IKK recruitment to the receptor, but not recruitment of the adaptor protein TRADD or TRAF2 (11). Recruitment of the TAK1 and IKK complexes is mediated through binding between K63-linked polyubiquitinated RIP1 and TAB2 (or TAB3) and NEMO, respectively (11–13). TAB2, TAB3, and NEMO all contain UBDs that bind preferentially to K63-linked polyubiquitin chains. Mutations within these UBDs prevent the activation of TAK1 and IKK (discussed in detail below).

**Role of Deubiquitination in Downregulation of NF-κB Signaling Cascades**

In support of K63-linked polyubiquitination as an important signaling event for activation of IKK, a multitude of reports have shown a critical function for deubiquitination (DUB) enzymes in negative regulation (65). These reports have taken our understanding of IKK regulation to a new level by identifying the machinery responsible for inactivating signaling proteins upstream of IKK and elucidating the role of ubiquitin binding and editing for this purpose.

CYLD is the first DUB known to inhibit IKK activation (66–68). Mutations in CYLD have been identified in patients with familial cylindromatosis in which affected individuals develop multiple benign tumors of the skin appendages (69). The protein has three cytoskeletal-associated protein-glycine-conserved (CAP-GLY) domains, a proline-rich repeat, and a domain with homology to ubiquitin-specific protease. CYLD cleaves tetraubiquitin to tri-, di-, and monoubiquitin, but C-terminally truncated forms of CYLD that are associated with familial cylindromatosis are defective in cleaving tetraubiquitin. In addition, CYLD interacts with NEMO and TRAF2 and causes reduced ubiquitination of these

TNFα

TNFR

Complex I
TRADD
TRAF2
cIAPs
RIP1
Ub

Complex II
RIP1
FADD
Caspase 8
TRADD
TRAF2
FADD
Caspase 8

Proteasome
cFLIP
Ub
ITCH
Tax1BP1
A20
RIP1
TAK1 complex
JNK
IKK complex
p65 p50
Phosphorylation
K63-linked ubiquitination
K48-linked ubiquitination
Apoptosis
cFLIP
cIAP
A20
p65 p50

Figure 3

Role of ubiquitin in coordinating NF-κB activation and cell death in the tumor necrosis factor (TNF) pathway. Binding of TNFα to the TNF receptor (TNFR) leads to the rapid formation of a receptor-associated complex (complex I) composed of the adaptor protein TRADD, the receptor-interacting protein kinase (RIP1), TRAF2, cIAP1, and cIAP2. TRAF2 and cIAPs mediate K63 polyubiquitination of RIP1, which recruits and activates the TAK1 kinase complex. TAK1 phosphorylates and activates IKK, resulting in the activation of NF-κB (represented by the p50/p65 dimer). NF-κB turns on the transcription of many target genes, including the cell death inhibitors c-FLIP, A20, and cIAPs. c-FLIP inhibits caspase 8, which forms a cytoplasmic complex with FADD, TRAF2, and TRADD (complex II) subsequent to the formation of the complex I. In the absence of cIAPs, RIP1 is deubiquitinated and released from the receptor to form another complex with FADD and caspase 8, which is not inhibited by c-FLIP and causes apoptosis even when NF-κB is active. RIP1 is deubiquitinated and targeted for degradation by a ubiquitin-editing complex consisting of the deubiquitination enzyme A20, ubiquitin-binding protein TAX1BP1, and ubiquitin ligase ITCH. TNFα also activates c-Jun N-terminal kinase (JNK), which phosphorylates and activates ITCH. ITCH ubiquitinates c-FLIP, which is then degraded by the proteasome, thereby promoting apoptosis.

proteins as well as TRAF6. Overexpression of CYLD, but not mutants defective in DUB activity, reduces NF-κB activation, whereas siRNA against CYLD enhances NF-κB activation by multiple stimuli. Knockdown of CYLD expression protects cells against TNFα-induced apoptosis, suggesting a molecular basis for the tumor-suppressing activity of CYLD. A recent report demonstrates that Cyld-deficient mice are highly susceptible to

Apoptosis:
programmed cell death mediated by a series of biochemical events leading to caspase activation and DNA fragmentation

chemically induced skin tumors. This report shows that the atypical IkB protein BCL-3 is another target of CYLD (70). BCL-3 shuttles between the nucleus and cytoplasm; while in the nucleus, it associates with the NF-κB subunits p50 and p52 and promotes cell proliferation and oncogenesis through activation of the cyclin D1 promoter. CYLD negatively regulates BCL-3/p50 and BCL-3/p52-mediated cyclin D1 expression by removing K63-linked

polyubiquitin chains from BCL-3 and inhibiting its nuclear accumulation. The DUB activity of CYLD is apparently necessary in this process, as a point mutation in the DUB domain renders CYLD unable to reduce BCL-3 ubiquitination or reduce its nuclear accumulation. Therefore, CYLD represses tumor formation by regulating cyclin D1-induced cell proliferation. This pathway may operate in parallel to CYLD's regulation of IKK activation and NF-κB-induced cell survival, although the relative importance of each pathway is not yet clear. It is also not clear why *Cylc* mutations preferentially cause cylindromatosis. One possibility is that the loss of function of Cyld in many tissues can be compensated for by other NF-κB inhibitors, such as A20 (see below), whereas in skin appendages, such as sweat glands, *Cyld* functions as a dominant tumor suppressor.

Three other recent reports highlight additional physiological functions of CYLD. *Cylc*-deficient mice are highly susceptible to chemically induced colitis and colon tumorigenesis (71). Cells from these mice had enhanced NF-κB activation in response to T cell receptor (TCR) engagement, CD40 ligand, and LPS but, surprisingly, not TNFα. NEMO ubiquitination was enhanced in *Cylc*-deficient cells following TCR stimulation. Reiley et al. (72) observed no apparent enhancement in LPS- and TNFα-induced activation of IKK or JNK in *Cylc*-deficient BMDMs. The predominant phenotype in this report was a defect in T cell development, apparently due to enhanced ubiquitination and degradation of LCK, a kinase important in T cell receptor signaling. Subsequent studies by the same group showed that CYLD knockout mice develop spontaneous inflammation of the colon and that *Cylc*-deficient T cells are hyperresponsive to TCR stimulation and have constitutive activation of TAK1, IKKβ, and JNK (73). Thus, CYLD regulates both T cell development and activation, in part through inhibition of signaling intermediates upstream of NF-κB.

A20 is a TNFα-inducible protein that contains an N-terminal OTU-type DUB domain and seven C-terminal zinc fingers (74). A20

knockout mice die at an early age with inflammation and damage in numerous tissues, including liver, kidney, intestine, joints, and bone marrow (75). The mice are hypersensitive to the inflammatory stimuli LPS and TNFα, implicating A20 as a crucial negative regulator of the inflammatory response. More recently, multiple polymorphisms of the A20 gene have been linked to the human autoimmune disease systemic lupus erythematosus (76), suggesting a role for A20 in suppression of autoimmune disease in humans.

Interestingly, the N and C termini of A20 have distinct ubiquitin-editing functions, both of which target RIP1 (77). The A20 N-terminal OTU domain catalyzes deubiquitination of K63-linked polyubiquitin chains on RIP1, whereas the A20 C-terminal zinc-finger domains promote K48-linked polyubiquitination and proteasomal degradation of RIP1. These results lead to a model in which, following TNFα stimulation, A20 removes K63-linked polyubiquitin chains and then adds K48-linked chains to target RIP1 for degradation. TRAF6 may also be a target of A20; recombinant A20, but not an OTU domain mutant (C103A), promotes deubiquitination of TRAF6, suggesting a mechanism by which A20 exerts its suppressive activity downstream of TLRs (78). However, overexpression of an A20 mutant lacking DUB activity causes a similar degree of inhibition of IL-1β- and TNFα-induced NF-κB activation compared to wild-type A20 (79). It is worth noting that this experiment was done in cells that have some basal expression of A20, whereas Dixit and colleagues (77) utilized fibroblasts from A20−/− mice in their study. It is possible that overexpression of A20 mutants lacking DUB activity can function in conjunction with endogenous A20 to inhibit IKK. Further studies, including engineering knockin mice carrying point mutations that abrogate A20’s DUB or E3 activity within the endogenous locus, should clarify the enzymatic functions of A20 in IKK inhibition.

It is not clear how DUB substrate specificity is determined. Two groups recently reported the crystal structure of the A20 N-terminal

Inflammation: a response to noxious stimuli such as pathogens and toxins, involving extravasation of leukocytes and lymphocytes from vasculature to damaged tissues
OTU domain (80, 81). However, this structural information about the catalytic region of A20 does not immediately suggest an explanation as to how A20 distinguishes between ubiquitinated substrates. In fact, it appears that the OTU domain preferentially cleaves K48-linked chains over K63-linked chains (80), which is surprising in light of the aforementioned data about A20. Interestingly, mixing of A20 with polyubiquitinated TRAF6 results in the appearance of free polyubiquitin chains but not monoubiquitin, suggesting that TRAF6 polyubiquitin chains are cleaved at their proximal end (81). This suggests that A20 substrate specificity may involve direct interaction with TRAF6; however, a clear understanding of how A20 substrate specificity is determined will require further studies.

Tax1-binding protein 1 (TAX1BP1), also known as TRAF6-binding protein (T6BP), and the immune modulatory protein ITCH were also recently demonstrated as important negative regulators that function together with A20 to inhibit IKK (82, 83). Two groups used different strategies to create TAX1BP1-deficient mice, producing different phenotypes. Shembade et al. (84) used a gene-trapping strategy to produce TAX1BP1-deficient mice, causing embryonic lethality with hemorrhaging and cardiac defects. Iha et al. (85) produced TAX1BP1-deficient mice using a more traditional exon deletion strategy, and these mice survived past birth but died at an early age with inflammatory cardiac valvulitis. TAX1BP1-deficient mice were hypersensitive to IL-1β and TNFα challenge, and TAX1BP1-deficient MEFs exhibited persistent IKK activation in response to TNF-α, IL-1β, and LPS. TAX1BP1 promoted association of A20 with RIP1 and TRAF6, leading to reduced ubiquitination following TNFα or IL-1β treatment, respectively. TAX1BP1 has three coiled-coil (CC) domains and two zinc fingers. The second zinc finger bound to K63- and K48-linked polyubiquitin chains, and point mutations in this zinc finger abrogated this binding. These point mutations impaired the ability of TAX1BP1 to restore downregulation of IKK in TAX1BP1-deficient MEFs (85).

Shembade et al. (82) also established an intriguing link between TAX1BP1, A20, and the HECT-family E3 ligase ITCH (82). ITCH-deficient, or *itchy*, mice have spontaneous skin inflammation (86). Shembade et al. found that ITCH-deficient cells have persistent NF-κB activation in response to TNFα. Furthermore, ITCH promotes RIP1 degradation, and this is dependent upon both TAX1BP1 and A20. ITCH-TAX1BP1 interaction depends on C-terminal Pro-Pro-X-Tyr motifs in TAX1BP1, which are recognized by WW domains present in HECT-family E3 ligases. Thus TAX1BP1, ITCH, and A20 appear to operate as a ubiquitin-editing complex to inactivate RIP1 and modulate the cell’s response to TNFα.

This report also revealed a striking example of a viral pathogen hijacking the cell’s ubiquitination machinery (82). TAX1BP1’s namesake, the human T cell leukemia virus (HTLV-1) protein TAX, induces persistent NF-κB activation, leading to transformation of T cells (87). TAX expression abrogated the interaction of A20 and ITCH with TAX1BP1 as well as recruitment of A20 and ITCH to RIP1. These results suggest the possibility that TAX promotes persistent NF-κB activation by blocking negative regulation of RIP1 by the TAX1BP1-A20-ITCH ubiquitin-editing complex. Consistent with a role for K63-linked polyubiquitination in this pathogen-host interaction, TAX’s ability to induce NF-κB activation is lost in the absence of UBC13 (88).

### Pervasive Role of Ubiquitination in Diverse Signaling Pathways

In addition to the TNFR and IL-1R/TLR signaling pathways, many other signaling cascades converge on IKK to promote NF-κB activation. Studies aimed at identifying and characterizing the molecular machinery involved in these pathways have revealed an important role for K63 polyubiquitination (89). Specifically, activation of the T cell receptor (TCR), cytosolic antibacterial and antiviral receptors, and the DNA-damage response all lead to
ubiquitination of NF-κB regulatory proteins (Table 1).

Upon engagement of the TCR, a tyrosine kinase cascade leads to activation of the serine/threonine kinase PKCθ. Subsequently, the IKK complex is activated in a mechanism involving the formation of a complex of CARMA1, BCL10, and MALT1 (CBM) (90, 91). A series of recent reports focused on elucidating the mechanism(s) coupling CBM formation to IKK activation have identified K63-linked polyubiquitination as an important event in this process. Two separate reports showed UBC13-dependent ubiquitination of NEMO, although separate conclusions were reached as to the E3 ligase involved. One report demonstrated that the C terminus of MALT1 has ubiquitin ligase activity and proposed that BCL10 activates MALT1 to promote NEMO ubiquitination (92). In the other report, BCL10-induced IKK activation was recapitulated in a cell-free system by adding recombinant BCL10 to cytosolic extract (93). This activity was found to be dependent on UBC13 and TRAF6. Although RNAi knockdown of TRAF6 expression only mildly impaired TCR-induced IKK activation and interleukin-2 (IL-2) production in Jurkat cells, RNAi knockdown of TRAF6 and TRAF2 caused a more severe impairment, indicating some redundancy of these two factors in this pathway. In addition, RNAi knockdown of TAK1 also impaired TCR-induced IKK activation. K399 within the C-terminal zinc-finger domain of NEMO appears to be a key acceptor site for ubiquitination because a K399R mutant was not ubiquitinated upon co-expression with BCL10 (92). The K399R mutant of NEMO was only able to partially restore PMA/ionomycin-induced NF-κB activation in a NEMO-deficient cell line, suggesting that NEMO ubiquitination plays a role in TCR-induced NF-κB activation. However, this conclusion was recently challenged by a genetic experiment, which showed that primary mouse cells containing the K392R mutation of mouse NEMO (equivalent to K399 in human) were largely normal in NF-κB and MAPK activation in response to TCR and TLR stimulation

(94). As Ubc13-deficient T cells are defective in the activation of NF-κB and MAPK by TCR (41), K63 polyubiquitination of proteins other than NEMO may be important in the TCR pathway. Indeed, recent studies have identified MALT1 and BCL10 as targets of polyubiquitination that are important for TCR signaling. One report demonstrated that TRAF6 stimulates ubiquitination of MALT1, promoting recruitment of the TAK1 and IKK complexes to MALT1 following TCR engagement (95). Multiple C-terminal lysines on MALT1 are acceptor sites for ubiquitin, and mutations of these Lys residues impaired TCR-induced NF-κB activation and IL-2 production. Another report showed that K63-linked ubiquitination of BCL10 promoted recruitment of IKK to the CBM (96). Mutations of K31 and K63 to Arg impaired BCL10 ubiquitination and reduced the ability of BCL-10 to restore TCR-induced NF-κB activation in Jurkat cells in which expression of endogenous BCL-10 was stably reduced by shRNA.

Retinoic acid-inducible gene-I (RIG-I) is a cytoplasmic sensor for RNA and is essential for interferon production in response to RNA virus infection (97, 98). Binding of RNA to RIG-I causes activation of interferon regulatory factor 3 (IRF3) and NF-κB via the kinases TBK1/IKKε and IKKα/β, respectively, through a signaling cascade dependent on mitochondrial antiviral signaling protein (MAVS; also known as IPS-1, VISA, or Cardif). The activation of NF-κB and IRF3 leads to the induction of type I interferons (IFN-I) to suppress viral infection. Tripartite motif-25 (TRIM-25), a RING-domain protein previously implicated in antiviral defense, promotes K63-linked polyubiquitination of K172 on RIG-I (99). Ubiquitination of RIG-I appears to facilitate the interaction of RIG-I with MAVS because a RIG-I K172R mutant was defective in interaction with MAVS and failed to activate IRF3 and NF-κB. *Trim*²⁵⁻/⁻ MEFs are deficient in IFNβ production and defense against Sendai virus infection, suggesting that RIG-I ubiquitination is an important part of the antiviral response.

Table 1 Representative proteins involved in ubiquitin-mediated regulation of IKK${}^{a,b}$

| Protein category | Protein name or Domain architecture |
|------------------|------------------------------------|
| E2              | UBC 5/UBC 13                       |
|                 | TRAF                              |
|                 | RING ZnF CC TRAF                  |
| E3              | TRIM 25                           |
|                 | RING B box/CCD SPRY               |
|                 | c-IAP                             |
|                 | BIR CARD RING                     |
|                 | ITCH                              |
|                 | C2 WW HECT                        |
| Ub target       | TRAF                              |
|                 | See above                         |
|                 | RIP 1                             |
|                 | Kinase CC DD                      |
|                 | RIP 2                             |
|                 | Kinase CARD                       |
|                 | RIG-I                             |
|                 | CARD Helicase                     |
|                 | NEMO                              |
|                 | IKK binding CC1 CC2 LZ ZnF        |
|                 | UBD                               |
|                 | IRAK 1                            |
|                 | DD Kinase                         |
| E 3 regulator / | MALT 1                            |
| Ub target       | DD IgG Caspase like               |
|                 | BCL 10                            |
|                 | CARD                              |
| Ub receptor     | NEMO                              |
|                 | See above                         |
|                 | TAB 2/3                           |
|                 | CUE ZnF                           |
| Deubiquitination | CYLD                              |
| enzyme(DUB)     | CAP-GLY USP                       |
|                 | A20                               |
|                 | OTU ZnF                           |
|                 | DUBA                              |
|                 | OTU UIM                           |

* ${}^{a}$ Protein name abbreviations: UBC, ubiquitin-conjugating enzyme; TRAF, tumor necrosis factor receptor-associated factor; TRIM 25, tripartite motif-25; c-IAP, cellular inhibitor of apoptosis; RIP, receptor-interacting protein; RIG-I, retinoic acid-inducible gene-I; NEMO, NF-$\kappa$B essential modulator; IRAK 1, IL-1R-associated serine/threonine kinase 1; MALT 1, mucosal associated lymphoid tissue 1; TAB, transforming growth factor $\beta$-associated kinase-binding protein; CYLD, cylindromatosis suppressor; DUBA, deubiquitinating enzyme A.
* ${}^{b}$ Domain abbreviations: UBC, ubiquitin-conjugating enzyme; RING, really interesting new gene; ZnF, zinc finger; CC, coiled coil; BIR, baculoviral inhibition of apoptosis protein repeat; CARD, caspase activation recruitment domain; C2, protein kinase C-conserved region 2; HECT, homologous to the E6-associated protein C terminus; DD, death domain; UBD, ubiquitin-binding domain; LZ, leucine zipper; IgG, immunoglobulin; CUE, Cue 1 homologous; CAP-GLY, cytoskeletal-associated protein-glycine conserved; USP, ubiquitin-specific protease; OTU, ovarian tumor-type cysteine protease; UIM, ubiquitin-interacting motif.

782 Skaug · Jiang · Chen
The cytoplasmic portion of MAVS contains several TRAF-binding sites, including that of TRAF3. TRAF3-deficient cells have an impaired ability to induce IFN-I (100, 101). An intact RING domain of TRAF3 is required for IFN-I induction, suggesting that TRAF3 activates TBK1 through its ability to function as a ubiquitin ligase (102). Further supporting the role of ubiquitination in TRAF3 signaling, a recent study identified a new DUB, DUBA, as a negative regulator of the IRF3 pathway (103). DUBA contains an OTU domain as well as a ubiquitin-interacting motif (UIM)-type UBD, and it was shown to catalyze the specific removal of K63-linked polyubiquitin chains from TRAF3 to suppress TBK1 and IRF3 activation.

Nucleotide-binding oligomerization domain (NOD) proteins NOD1 and NOD2 are cytosolic receptors that detect intracellular bacterial peptidoglycans (104). NOD1 detects γ-D-glutamyl-meso-diaminopimelic acid (iE-DAP) derived from most gram-negative bacteria and certain gram-positive bacteria, whereas NOD2 detects muramyl dipeptide (MDP) from almost all bacteria. Mutations in the gene for NOD2 have been identified in patients with Crohn’s disease, an inflammatory bowel disease. Binding of MDP to NOD2 activates IKK via RIP2, a RIP1-like kinase (also known as RICK), leading to NF-κB activation, but mutations in NOD2 associated with Crohn’s disease hinder the ability of NOD2 to activate NF-κB in response to MDP. NOD2 was shown to induce RIP2-dependent ubiquitination of NEMO at K285, and NEMO ubiquitination was impaired by NOD2 mutations associated with Crohn’s disease (105, 106).

Subsequent studies have revealed that NF-κB activation by NOD1 and NOD2 requires K63-linked ubiquitination of RIP2; the key ubiquitination acceptor site was mapped to K209 (107, 108). Furthermore, A20 deubiquitates RIP2, blocking NF-κB activation by MDP (109). A20⁻/⁻ mice secrete significantly more IL-6 than wild-type mice in response to MDP, indicating that A20 is an important negative regulator of the MDP-induced inflammatory response in vivo.

DNA damage also activates NF-κB through IKK. How the cytosolic IKK complex is activated by a signal originating in the nucleus was an intriguing mystery, but the molecular events coordinating this process have begun to be uncovered (110, 111). DNA damage induces sumoylation of a small fraction of NEMO at K277 and K309 by the SUMO ligase PIASy, promoting the accumulation of NEMO in the nucleus (112). NEMO is then phosphorylated by the kinase ATM. Phosphorylated NEMO is ubiquitinated also at K277 and K309, and then it enters the cytosol together with ATM to activate IKK, leading to NF-κB activation. The mechanism by which ubiquitinated NEMO activates IKK in the cytosol remains to be elucidated.

### Mechanism of Protein Kinase Activation by Ubiquitin

Studies to assess the role of ubiquitin in kinase activation have led to the identification of UBDs in TAB2 and NEMO as well as the revelation that ubiquitin binding is required for activation of the TAK1 and IKK complexes, respectively (11–13). TAB2 and TAB3 have two predicted UBDs: an N-terminal Cuel-homologous (CUE) domain and a C-terminal novel zinc-finger (NZF) domain. The NZF domain was shown to bind preferentially to K63-linked polyubiquitin chains. Deletion of the NZF domain abrogated IKK activation in vitro and in TNFα-stimulated cells, but substitution of a heterologous UBD restored activation (12). Therefore, ubiquitin binding by TAB2 and TAB3 is an essential step for inflammatory stimulus-induced IKK activation (Figure 2).

Mutations in NEMO were identified as the cause of the rare X-linked recessive disorder anhidrotic ectodermal dysplasia with immunodeficiency (EDA-ID) and EDA-ID with osteopetrosis and lymphoedema (OL-EDA-ID) (113). These diseases are characterized by a host of developmental disorders, as well as increased susceptibility to bacterial and fungal infections, despite normal levels of T and B lymphocytes. The mutations identified in EDA-ID are

**Inflammatory bowel disease:** an autoimmune disease causing diarrhea and abdominal pain; characterized by inflammatory lesions of the gastrointestinal tract

www.annualreviews.org • Ubiquitin in NF-κB Regulatory Pathways 783
heterogenous in location and include each of  
the two CC domains, a linker region between  
CC2 and the leucine zipper (LZ), and the  
zinc-finger domain.

The ubiquitin-binding region of NEMO  
has been pinpointed to an area encompass-  
ing the CC2 and LZ domains, termed NEMO  
ubiquitin-binding (NUB) domain (11, 13).  
Point mutants in this region that reduce  
NEMO’s ability to bind to polyubiquitin chains,  
including a D311N mutation found in a pa-  
tient with EDA-ID, impair its ability to restore  
TNFα-induced IKK activation in a NEMO-  
deficient cell line. As in the case of CYLD, this  
combination of genetic and biochemical evi-  
dence strongly suggests that ubiquitin-media-  
ted regulation of IKK activation has a critical  
role in human physiology and that its dysregu-  
lation can have devastating consequences.

Although these reports emphasize the  
prevalence of ubiquitin binding in kinase reg-  
ulation and identify specific regions and even  
individual amino acids important for these sig-  
naling events, the intriguing question of how a  
kinase is activated by ubiquitin binding remains.  
Structural studies of diubiquitin molecules  
have provided some clues as to how different  
ubiquitin linkages lead to different functional  
consequences. K48-linked diubiquitin adopts a  
closed conformation with numerous interubiq-  
uitin contacts between hydrophobic residues,  
whereas K63-linked diubiquitin has very little  
interubiquitin interaction and thus a more  
“open” conformation (114). Nevertheless, no  
definitive model has emerged to explain how  
ubiquitin binding enhances kinase activity.  
Do polyubiquitin chains function as a scaffold  
to organize these kinase complexes in an  
orientation and/or higher order structure  
favorable for substrate binding and phosphory-  
lation? Alternatively, does polyubiquitin chain  
binding induce an allosteric change in the  
structure of these kinase complexes, thereby  
promoting activation and/or recruitment to  
substrates? Structural information about TAB2  
and NEMO, particularly in the presence of  
K63-linked ubiquitin chains, will be needed to  
address these questions.


Adding another level of complexity to the  
mechanism(s) of IKK complex activation is that  
NEMO is also a target for ubiquitination in a  
variety of signaling cascades. TNFα stimula-  
tion induces NEMO ubiquitination, which is  
dependent upon an intact zinc-finger domain.  
A C417R mutation, identified in a patient with  
EDA-ID, impaired TNFα-induced ubiquiti-  
nation and IKK activation (115). As discussed  
above, ubiquitination of NEMO also occurs in  
the TCR, NOD, and DNA-damage signaling  
pathways. However, distinct lysines on NEMO  
have been proposed as the ubiquitination sites  
in different NF-κB pathways; whether ubiquiti-  
nation of NEMO at any of these sites is impor-  
tant for IKK activation requires further study.  
In particular, caution needs to be taken in in-  
terpreting the results of mutating certain lysine  
residues; some mutations may have unintended  
effects such as altering the structure of NEMO.

Ubiquitin binding and ubiquitination of a  
protein are often coupled. Similarly, mutations  
that affect NEMO ubiquitin binding also af-  
fect its ubiquitination, making it difficult to de-  
termine whether ubiquitin binding or ubiqui-  
tination of NEMO is important for IKK ac-  
tivation. However, a recent study identified a  
NEMO mutant, K270A, which renders IKK  
constitutively active, bypassing the require-  
ment for ubiquitin binding (116). This muta-  
tion enhances the ability of NEMO (including  
ubiquitin-binding-deficient NEMO mutants)  
to be ubiquitinated after TRAF6 overexpres-  
sion. Thus the authors propose that the func-  
tion of ubiquitin binding by NEMO is to induce  
the formation of an “activated state” that occurs  
upstream of NEMO ubiquitination.

**UBIQUITIN-MEDIATED  
DEGRADATION OF SIGNALING  
PROTEINS IN THE NF-κB  
PATHWAYS**

**Ubiquitination of IkB, p105, and p100  
by SCF-βTrCP**

Following IKK activation by various stim-  
uli, IkB proteins are rapidly phosphorylated,  
polyubiquitinated, and then degraded by the

proteasome. Phosphorylation of IkB creates a
degron, DpSGXXpS (where p stands for phospho-
ryl group), which binds to the F-box protein
βTrCP. βTrCP contains C-terminal WD40 re-
peats, which bind to phosphorylated IkB, and
an N-terminal F-box, which binds to Skp1 to
form an SCF complex (117–119). There are
two βTrCP proteins in mammals, and there ap-
pears to be some redundancy in their function.
IkBα degradation is only mildly delayed in the
absence of βTrCP1, but RNAi knockdown of
βTrCP1 and βTrCP2 expression severely im-
pairs IkBα degradation (120).

The SCF-βTrCP complex is also respon-
sible for the ubiquitination of p100 and p105
following their phosphorylation by IKK (23,
121, 122). In addition, there are many sub-
strates of SCF-βTrCP that function outside of
the NF-κB pathway, including those involved
in hedgehog and Wnt signaling, cell cycle con-
trol, and circadian rhythm (123). In almost all
cases, phosphorylation of the substrates is a pre-
requisite for recognition by βTrCP. It should
be noted that in most cases the ubiquitination
of protein substrates by the βTrCP complex
and their degradation by the proteasome have
not been reconstituted in vitro using purified
proteins, leaving open the possibility that addi-
tional components and regulatory mechanisms
remain to be discovered in order to fully under-
stand signal-dependent degradation of βTrCP
substrates, including IκBs.

### Ubiquitination and Degradation
of NIK

The noncanonical NF-κB signaling pathway is
particularly important in regulation of B cell
survival, maturation, and homeostasis (124).
In the absence of NIK or IKKα, B cell sur-
vival and maturation are severely impaired. Re-
cent studies revealed ubiquitination-mediated
regulation of NIK activity as a critical com-
ponent in this noncanonical pathway of NF-
κB activation (Figure 4). The BAFF receptor
binds the cytoplasmic protein TRAF3 through
a TRAF-binding motif (125). BAFF induces
proteolysis of TRAF3, leading to stabilization
of NIK (126). Overexpression of TRAF3 pro-
motes proteasomal degradation of NIK. Con-
versely, loss of TRAF3 results in constitutive
processing of p100 to p52 in B cells, and
TRAF3-null lethality is rescued by loss of p100
or NIK, indicating that noncanonical NF-κB
activation is the key factor causing death in the
absence of TRAF3 (127).

Despite the ability of TRAF3 to promote
NIK degradation, the role of the RING finger
of TRAF3 in NIK ubiquitination is controver-
sial. He et al. (128) showed that an intact RING
domain of TRAF3 is required to inhibit p100
processing in complementation experiments in
TRAF3-deficient MEF cells. In contrast, Val-
labhapurapu et al. (129) showed that TRAF3
lacking the RING domain could still pro-
mote NIK degradation in overexpression ex-
periments in HEK293 cells. Further studies by
genetic complementation of TRAF3-deficient
B cells may be required to resolve this issue.

TRAF2 also acts as a negative regulator of
the noncanonical NF-κB pathway (130, 131).
TRAF2- and TRAF3-deficient B cells both
have enhanced survival in vivo and in vitro, even
in the absence of BAFF. Recent studies suggest
that TRAF2 and TRAF3 act cooperatively and
nonredundantly to suppress the noncanonical
NF-κB pathway (129, 132). However, neither
TRAF2 nor TRAF3 has been shown to catalyze
NIK ubiquitination in vitro (126). Recent stud-
ies provide strong evidence that the cellular in-
hibitors of apoptosis, cIAP1 and cIAP2, func-
tion as the E3 for NIK (129, 133, 134). cIAPs
are RING-finger E3 ligases initially discovered
through their association with TRAF2 (135,
136). cIAP deficiency or degradation results in
stabilization of NIK and processing of p100 to
p52 (133, 134). Biochemical studies suggest that
TRAF3 acts as an adaptor protein to coordinate
the assembly of a TRAF2-cIAP1-cIAP2 E3 lig-
ase complex to induce constitutive NIK degra-
dation to maintain a low level of NIK under
basal conditions. Stimulation of B cells through
the BAFF receptor or CD40 leads to activa-
tion of TRAF2, which catalyzes K63-linked
polyubiquitination of cIAPs, causing cIAPs to
ubiquitinate TRAF3. TRAF3 is then degraded

Unstimulated state

Stimulated state

CD40L

CD40/BAFF-R

SMAC-mimetic

clAPs
TRAF2
Ub
clAPs
TRAF3
Ub

Proteasome

NIK

IKKa

p100 Rel-B

p52 Rel-B

p52 Rel-B

Phosphorylation
K63-linked ubiquitination
K48-linked ubiquitination

Figure 4

Role of ubiquitin in the noncanonical pathway of NF-κB activation. In unstimulated cells, the kinase NIK is constitutively ubiquitinated by an E3 complex consisting of TRAF3, TRAF2, and cIAPs. This polyubiquitination targets NIK for degradation by the proteasome. Upon stimulation of B cells through CD40 and BAFF receptor (BAFFR), TRAF2 is activated to catalyze K63 polyubiquitination of cIAPs, which promotes the ability of cIAPs to catalyze K48 polyubiquitination of TRAF3. TRAF3 is then degraded by the proteasome, hindering the ability of cIAPs to target NIK degradation. The accumulated NIK phosphorylates IKKα, which in turn phosphorylates p100, leading to its proteasomal processing to p52. p52 forms a complex with Rel-B to regulate gene expression in B cells. Small-molecule mimetics of SMAC can also activate NIK by triggering the autoubiquitination and degradation of cIAPs.

by the proteasome, leading to the stabilization of NIK (129). Whether or not NIK needs to be further activated after stabilization requires additional investigation.

**ROLE OF UBIQUITIN IN COORDINATING NF-κB ACTIVATION AND CELL DEATH**

**Ubiquitination of RIP1 by TRAFs and cIAPs**

The TNF pathway provides a paradigm for understanding the delicate balance between cell survival and death (137, 138). Binding of TNFα to TNFR1 induces the sequential formation of two protein complexes: first, a receptor-associated complex (complex I) containing TRADD, RIP1, TRAF2, cIAP1, and cIAP2. Formation of complex I leads to K63 polyubiquitination of RIP1 and activation of IKK and NF-κB. Then, in a less well-understood process, this complex dissociates from the receptor, and a second, cell death-promoting complex (complex II) is formed, which includes FADD and caspase 8 (Figure 3). However, the activation of caspase 8 in this complex is normally

Caspases: cysteine-aspartic acid proteases that play an essential function in apoptosis, including the terminal biochemical events leading to DNA fragmentation

Skaug • Jiang • Chen

prevented because the activation of NF-κB upregulates antiapoptotic proteins, such as cIAP1/2 and cFLIP, which inhibit caspase 8 activation.

IAPs are thought to inhibit apoptosis through their ability to bind and inhibit caspases (139). However, recent studies suggest that not all cIAPs inhibit caspases directly. Instead, the ubiquitin ligase activity of cIAP1 and cIAP2 plays a key role in preventing apoptosis. cIAP1 and cIAP2 undergo autoubiquitination and degradation upon binding to the IAP antagonist second mitochondrial activator of caspases (SMAC)/direct IAP-binding protein with low pI (DIABLO), which is released from the mitochondria (140, 141).

Paradoxically, recent studies using small-molecule mimetics of SMAC have found that these IAP antagonists induce both apoptosis and NF-κB activation, despite the ability of NF-κB to promote cell survival (133, 134, 142). As discussed above, the loss of cIAPs leads to the stabilization of NIK and the subsequent activation of the noncanonical NF-κB pathway, which induces TNFα. TNFα leads to the polyubiquitination of RIP1, which normally leads to NF-κB activation. When cIAPs are depleted by genetic or chemical means, RIP1 is deubiquitinated by CYLD, dissociates from the receptor complex, and forms the death-inducing complex with FADD and caspase 8 (143). For reasons that are still not well understood, the caspase 8 in this complex is apparently resistant to inhibition by c-FLIP. Thus, in the presence of SMAC mimetics and TNFα, cells undergo apoptosis despite NF-κB activation.

These findings suggest that ubiquitination and deubiquitination of RIP1 serves as a switch that determines a cell’s life or death in the TNF pathway. TRAF2 and cIAPs are likely candidates as the E3 for RIP1. TRAF2-deficient cells have increased TNFα-induced apoptosis despite normal NF-κB activation (58, 59). In addition, TRAF2 or UBC13 dominant-negative mutants cause enhanced TNFα-induced apoptosis; this effect is reduced in RIP1-deficient cells, indicating that TRAF2 deficiency poten-

tiates apoptosis in a RIP1-dependent manner (144). Nevertheless, it has not been shown that TRAF2 can directly ubiquitinate RIP1 in vitro. By contrast, recent studies suggest that cIAP1 and cIAP2 may catalyze ubiquitination of RIP1 (145). However, it is still puzzling why so many RING-domain E3s are needed for RIP1 ubiquitination in the TNF pathway. In this regard, it will be informative to determine whether the RING domain of TRAF2 or cIAPs is important for RIP1 ubiquitination under physiological conditions.

**Second mitochondrial activator of caspase (SMAC) mimetics:** small molecules resembling the N-terminal peptide of SMAC that bind to IAPs with high affinity and specificity

---

### **Ubiquitination of cFLIP by ITCH**

TNFα also promotes apoptosis through activation of JNK (146). Prolonged activation of JNK enhances TNFα-induced apoptosis, in part by phosphorylation and activation of the E3 ligase ITCH, which in turn leads to ubiquitination and proteasomal degradation of cFLIP (147). Consistent with this model, *Itch* deficiency protects cells from TNFα-induced apoptosis, and *Itch* mutant mice (itchy mice) are protected from TNFα-induced lethality. NF-κB activation suppresses prolonged JNK activation through upregulation of Gadd45b and/or XIAP (148, 149) or by inducing expression of antioxidant proteins (150). These findings suggest a mechanism by which JNK and NF-κB regulatory pathways interact to control cell survival.

---

### **CONCLUSIONS AND PERSPECTIVES**

Over two decades of intensive research on NF-κB regulation have resulted in tremendous gains in our understanding of this transcription factor and its associated signaling pathways. Equally important are the insights into fundamental principles governing protein ubiquitination, processing and degradation of ubiquitinated substrates, kinase regulation, and apoptosis.

Each year numerous reports reveal new features of NF-κB regulatory pathways, including the identification of proteins with previously

unknown roles in ubiquitin editing and kinase regulation, as well as biochemical and structural characterization of NF-κB regulatory factors. K63-linked polyubiquitination of multiple proteins in these signaling pathways is becoming a common theme, adding breadth to the understanding of ubiquitin’s regulatory role. A deeper mechanistic understanding of ubiquitination and DUBs and kinase regulation by ubiquitin remains as an important challenge for this field. In addition, much remains to be learned about how substrate binding and activity of ubiquitination and DUBs are regulated.

Research into NF-κB regulatory pathways will inevitably continue to lead to a better understanding of the complex processes of immunity, inflammation, and apoptosis. In addition, the use of NF-κB regulatory pathways as a model system will likely lead to the discovery of new and unexpected biochemical processes generally relevant to many other signaling pathways.

---

### SUMMARY POINTS

1. In canonical NF-κB activation, inflammatory stimuli such as TLR ligands, IL-1β, and TNFα induce activation of the IKK complex leading to IκB phosphorylation, followed by ubiquitination by the SCF-βTrCP E3 ligase complex. Ubiquitinated IκB is degraded by the proteasome, allowing NF-κB to enter the nucleus and activate gene transcription.

2. In the noncanonical pathway, IKKα is activated in a NIK-dependent process, leading to p100 phosphorylation, ubiquitination, and processing to p52. This process is subject to regulation of NIK degradation, with cIAPs, TRAF2, and TRAF3 suppressing NIK activation by promoting its degradation.

3. p105 and p100 are processed by the proteasome into the p50 and p52 subunits, respectively. Processing of p105 and the yeast transcription factors Spt23 and Mga2 initiates at an internal site within the protein.

4. K63-linked polyubiquitination by the E2 complex UBC13/UEV1A and the RING domain E3 ligase, TRAF6, activates the TAK1 kinase complex in a proteasome-independent mechanism. This signaling event is important for activation of the canonical NF-κB pathway in cells stimulated with IL-1β and TLR ligands.

5. Ubiquitin binding and ubiquitination of NEMO play an important role in activation of NF-κB by diverse signaling pathways, including TNFR, IL-1R/TLR, TCR, NOD proteins, and DNA damage.

6. DUBs, such as CYLD and A20, deubiquitinate signaling intermediates upstream of IKK, including TRAF6, TRAF2, and RIP1. Dysfunction or genetic ablation of these DUBs leads to persistent NF-κB activation and severe pathologic sequelae. A20 acts as part of a complex with ITCH and TAX1BP1 to deubiquitinate RIP1 and thus inhibit persistent IKK activation; the HTLV-1 oncoprotein TAX disrupts this inhibitory complex and in this way promotes persistent NF-κB activation.

7. The ubiquitination status of RIP1 is critical in determining whether TNFα leads to a prosurvival or proapoptotic response. TRAF2 and cIAP1/2 appear to be likely candidate E3 ligases for RIP1.

FUTURE ISSUES

1. The mechanism by which K63-linked polyubiquitination promotes activation of the TAK1 and IKK complexes is not yet clear. Structural information will likely be necessary to understand what, if any, allosteric changes and organization of ubiquitin-binding proteins into higher-order structures are induced by ubiquitin binding. Not only will this information be useful for understanding these two kinase complexes, it could shed light on more general principles of inducible kinase activation.

2. The mechanism by which DUBs, such as CYLD and A20, bind to and cleave K63-linked polyubiquitin chains from ubiquitinated substrates, e.g., TRAF6 and RIP1, remains unclear. Specifically, it remains to be clearly established whether or not K63-linked polyubiquitin chains are cleaved from their proximal or distal ends; also, reconciliation of the apparent preference of A20 for K48-linked polyubiquitin chains in vitro but K63-linked polyubiquitin chains in vivo will require additional studies.

3. Although NF-κB-inducible upregulation of A20 expression helps to explain its ability to inhibit persistent IKK activation, additional regulatory mechanisms clearly exist to promote recruitment of A20 to its substrates and to inhibit IKK activation despite persistent upstream stimulation. The revelation that A20 promotes RIP1 inactivation and degradation as part of a complex with TAX1BP1 and ITCH is a step in the right direction toward understanding regulation of DUB localization and activity in cells.

4. Dysfunction of NOD2 promotes inflammatory bowel disease, but it is still not clear how diminished activation of NF-κB associated with the NOD2 mutations found in Crohn’s disease leads to excessive inflammation. Whether NOD proteins bind directly to bacterial peptidoglycans or there are additional receptors is another important issue that needs further investigation.

5. The ability to intervene pharmacologically in NF-κB regulatory pathways would no doubt create tremendous potential for therapies for inflammatory and neoplastic diseases. E3 ligases in particular are appealing targets because of their relative specificity to certain pathways, e.g., TRAF6 in the TLR/IL-1R pathway versus TRAF2/5 in the TNFR pathway. Thus far, no pharmacological inhibitors of E3 ligases in NF-κB regulatory pathways have been described. Inhibition of the binding between K63 polyubiquitin chains and ubiquitin receptors, such as TAB2 and NEMO, represents another avenue of therapeutic intervention. It is likely that a better biochemical and structural understanding of the activity and regulation of ubiquitin ligases and ubiquitin receptors will help pave the way for pharmacological inhibition.

6. RIP1 has emerged as a key regulator of a cell’s life and death, but how RIP1 is regulated at the cell surface and how it controls apoptosis (and necrosis) require intense research in the future.


DISCLOSURE STATEMENT

The authors are not aware of any affiliations, memberships, funding, or financial holdings that might be perceived as affecting the objectivity of this review.

# ACKNOWLEDGMENTS

Research in our laboratory is supported by grants from NIH (RO1-GM63692 and RO1-AI60919) and the Welch Foundation (I-1389). Z.J.C. is an Investigator of Howard Hughes Medical Institute.

# LITERATURE CITED

1. Sen R, Baltimore D. 1986. Multiple nuclear factors interact with the immunoglobulin enhancer sequences. *Cell* 46:705–16
2. Hayden MS, Ghosh S. 2008. Shared principles in NF-kappaB signaling. *Cell* 132:344–62
3. Hershko A. 1983. Ubiquitin: roles in protein modification and breakdown. *Cell* 34:11–12
4. Pickart CM. 2004. Back to the future with ubiquitin. *Cell* 116:181–90
5. Chen ZJ. 2005. Ubiquitin signalling in the NF-kappaB pathway. *Nat. Cell Biol.* 7:758–65
6. Bernassola F, Karin M, Ciechanover A, Melino G. 2008. The HECT family of E3 ubiquitin ligases: multiple players in cancer development. *Cancer Cell* 14:10–21
7. Petroski MD, Deshaies RJ. 2005. Function and regulation of cullin-RING ubiquitin ligases. *Nat. Rev. Mol. Cell Biol.* 6:9–20
8. Pickart CM. 2001. Mechanisms underlying ubiquitination. *Annu. Rev. Biochem.* 70:503–33
9. Hicke L, Schubert HL, Hill CP. 2005. Ubiquitin-binding domains. *Nat. Rev. Mol. Cell Biol.* 6:610–21
10. Hurley JH, Lee S, Prag G. 2006. Ubiquitin-binding domains. *Biochem. J.* 399:361–72
11. Ea CK, Deng L, Xia ZP, Pineda G, Chen ZJ. 2006. Activation of IKK by TNFalpha requires site-specific ubiquitination of RIP1 and polyubiquitin binding by NEMO. *Mol. Cell* 22:245–57
12. Kanayama A, Seth RB, Sun L, Ea CK, Hong M, et al. 2004. TAB2 and TAB3 activate the NF-kappaB pathway through binding to polyubiquitin chains. *Mol. Cell* 15:535–48
13. Wu CJ, Conze DB, Li T, Srinivasula SM, Ashwell JD. 2006. Sensing of Lys 63-linked polyubiquitination by NEMO is a key event in NF-kappaB activation [corrected]. *Nat. Cell Biol.* 8:398–406
14. Elsasser S, Finley D. 2005. Delivery of ubiquitinated substrates to protein-unfolding machines. *Nat. Cell Biol.* 7:742–49
15. Nijman SM, Luna-Vargas MP, Velds A, Brummelkamp TR, Dirac AM, et al. 2005. A genomic and functional inventory of deubiquitinating enzymes. *Cell* 123:773–86
16. Amerik AY, Hochstrasser M. 2004. Mechanism and function of deubiquitinating enzymes. *Biochim. Biophys. Acta* 1695:189–207
17. Israel A. 2006. NF-kappaB activation: Nondegradative ubiquitination implicates NEMO. *Trends Immunol.* 27:395–97
18. Hoffmann A, Baltimore D. 2006. Circuitry of nuclear factor kappaB signaling. *Immunol. Rev.* 210:171–86
19. Senftleben U, Cao Y, Xiao G, Greten FR, Krahn G, et al. 2001. Activation by IKKalpha of a second, evolutionary conserved, NF-kappa B signaling pathway. *Science* 293:1495–99
20. Xiao G, Harhaj EW, Sun SC. 2001. NF-kappaB-inducing kinase regulates the processing of NF-kappaB 2 p100. *Mol. Cell* 7:401–9
21. Palombella VJ, Rando OJ, Goldberg AL, Maniatis T. 1994. The ubiquitin-proteasome pathway is required for processing the NF-kappa B1 precursor protein and the activation of NF-kappa B. *Cell* 78:773–85
22. Lin L, DeMartino GN, Greene WC. 1998. Cotranslational biogenesis of NF-kappaB p50 by the 26S proteasome. *Cell* 92:819–28
23. Cohen S, Achbert-Weiner H, Ciechanover A. 2004. Dual effects of IkappaB kinase beta-mediated phosphorylation on p105 Fate: SCF(beta-TrCP)-dependent degradation and SCF(beta-TrCP)-independent processing. *Mol. Cell. Biol.* 24:475–86
24. Moorthy AK, Savinova OV, Ho JQ, Wang VY, Vu D, Ghosh G. 2006. The 20S proteasome processes NF-kappaB1 p105 into p50 in a translation-independent manner. *EMBO J.* 25:1945–56
25. Rape M, Jentsch S. 2004. Productive RUPture: activation of transcription factors by proteasomal processing. *Biochim. Biophys. Acta* 1695:209–13
26. Lin L, Ghosh S. 1996. A glycine-rich region in NF-kappaB p105 functions as a processing signal for the generation of the p50 subunit. *Mol. Cell. Biol.* 16:2248–54

27. Piwko W, Jentsch S. 2006. Proteasome-mediated protein processing by bidirectional degradation initiated from an internal site. *Nat. Struct. Mol. Biol.* 13:691–97

28. Chung JY, Park YC, Ye H, Wu H. 2002. All TRAFs are not created equal: common and distinct molecular mechanisms of TRAF-mediated signal transduction. *J. Cell Sci.* 115:679–88

29. Deng L, Wang C, Spencer E, Yang L, Braun A, et al. 2000. Activation of the IkappaB kinase complex by TRAF6 requires a dimeric ubiquitin-conjugating enzyme complex and a unique polyubiquitin chain. *Cell* 103:351–61

30. Wang C, Deng L, Hong M, Akkaraju GR, Inoue J, Chen ZJ. 2001. TAK1 is a ubiquitin-dependent kinase of MKK and IKK. *Nature* 412:346–51

31. Lomaga MA, Yeh WC, Sarosi I, Duncan GS, Furlonger C, et al. 1999. TRAF6 deficiency results in osteopetrosis and defective interleukin-1, CD40, and LPS signaling. *Genes Dev.* 13:1015–24

32. Naito A, Azuma S, Tanaka S, Miyazaki T, Takaki S, et al. 1999. Severe osteopetrosis, defective interleukin-1 signalling and lymph node organogenesis in TRAF6-deficient mice. *Genes Cells* 4:353–62

33. Cao Z, Xiong J, Takeuchi M, Kurama T, Goeddel DV. 1996. TRAF6 is a signal transducer for interleukin-1. *Nature* 383:443–46

34. Kobayashi N, Kadono Y, Naito A, Matsumoto K, Yamamoto T, et al. 2001. Segregation of TRAF6-mediated signaling pathways clarifies its role in osteoclastogenesis. *EMBO J.* 20:1271–80

35. Lamothe B, Besse A, Campos AD, Webster WK, Wu H, Darnay BG. 2007. Site-specific Lys-63-linked tumor necrosis factor receptor-associated factor 6 auto-ubiquitination is a critical determinant of IkB kinase activation. *J. Biol. Chem.* 282:4102–12

36. Lamothe B, Webster WK, Gopinathan A, Besse A, Campos AD, Darnay BG. 2007. TRAF6 ubiquitin ligase is essential for RANKL signaling and osteoclast differentiation. *Biochem. Biophys. Res. Commun.* 359:1044–49

37. Petroski MD, Zhou X, Dong G, Daniel-Issakani S, Payan DG, Huang J. 2007. Substrate modification with lysine 63-linked ubiquitin chains through the UBC13-UEV1A ubiquitin-conjugating enzyme. *J. Biol. Chem.* 282:29936–45

38. Windheim M, Stafford M, Peggie M, Cohen P. 2008. Interleukin-1 (IL-1) induces the Lys63-linked polyubiquitination of IL-1 receptor-associated kinase 1 to facilitate NEMO binding and the activation of IkappaBalpha kinase. *Mol. Cell. Biol.* 28:1783–91

39. Conze DB, Wu CJ, Thomas JA, Landstrom A, Ashwell JD. 2008. Lys63-linked polyubiquitination of IRAK-1 is required for interleukin-1 receptor- and toll-like receptor-mediated NF-kappaB activation. *Mol. Cell. Biol.* 28:3538–47

40. Yamamoto M, Okamoto T, Takeda K, Sato S, Sanjo H, et al. 2006. Key function for the Ubc13 E2 ubiquitin-conjugating enzyme in immune receptor signaling. *Nat. Immunol.* 7:962–70

41. Yamamoto M, Sato S, Saitoh T, Sakurai H, Uematsu S, et al. 2006. Cutting edge: pivotal function of Ubc13 in thymocyte TCR signaling. *J. Immunol.* 177:7520–24

42. Fukushima T, Matsuzawa S, Kress CL, Bruey JM, Krajewska M, et al. 2007. Ubiquitin-conjugating enzyme Ubc13 is a critical component of TNF receptor-associated factor (TRAF)-mediated inflammatory responses. *Proc. Natl. Acad. Sci. USA* 104:6371–76

43. Chen ZJ, Parent L, Maniatis T. 1996. Site-specific phosphorylation of IkappaBalpha by a novel ubiquitination-dependent protein kinase activity. *Cell* 84:853–62

44. Ninomiya-Tsuji J, Kajino T, Ono K, Ohtomo T, Matsumoto M, et al. 2003. A resorcylic acid lactone, 5Z-7-oxozeaenol, prevents inflammation by inhibiting the catalytic activity of TAK1 MAPK kinase kinase. *J. Biol. Chem.* 278:18485–90

45. Takaesu G, Surabhi RM, Park KJ, Ninomiya-Tsuji J, Matsumoto K, Gaynor RB. 2003. TAK1 is critical for IkappaB kinase-mediated activation of the NF-kappaB pathway. *J. Mol. Biol.* 326:105–15

46. Chen W, White MA, Cobb MH. 2002. Stimulus-specific requirements for MAP3 kinases in activating the JNK pathway. *J. Biol. Chem.* 277:49105–10

47. Silverman N, Zhou R, Erlich RL, Hunter M, Bernstein E, et al. 2003. Immune activation of NF-kappaB and JNK requires *Drosophila* TAK1. *J. Biol. Chem.* 278:48928–34

48. Vidal S, Khush RS, Leulier F, Tzou P, Nakamura M, Lemaitre B. 2001. Mutations in the *Drosophila* dTAK1 gene reveal a conserved function for MAPKKKKs in the control of rel/NF-kappaB-dependent innate immune responses. *Genes Dev.* 15:1900–12

49. Liu HH, Xie M, Schneider MD, Chen ZJ. 2006. Essential role of TAK1 in thymocyte development and activation. *Proc. Natl. Acad. Sci. USA* 103:11677–82

50. Sato S, Sanjo H, Takeda K, Ninomiya-Tsuji J, Yamamoto M, et al. 2005. Essential function for the kinase TAK1 in innate and adaptive immune responses. *Nat. Immunol.* 6:1087–95

51. Shim JH, Xiao C, Paschal AE, Bailey ST, Rao P, et al. 2005. TAK1, but not TAB1 or TAB2, plays an essential role in multiple signaling pathways in vivo. *Genes Dev.* 19:2668–81

52. Wan YY, Chi H, Xie M, Schneider MD, Flavell RA. 2006. The kinase TAK1 integrates antigen and cytokine receptor signaling for T cell development, survival and function. *Nat. Immunol.* 7:851–58

53. Sanjo H, Takeda K, Tsujimura T, Ninomiya-Tsuji J, Matsumoto K, Akira S. 2003. TAB2 is essential for prevention of apoptosis in fetal liver but not for interleukin-1 signaling. *Mol. Cell. Biol.* 23:1231–38

54. Ishitani T, Takaesu G, Ninomiya-Tsuji J, Shibuya H, Gaynor RB, Matsumoto K. 2003. Role of the TAB2-related protein TAB3 in IL-1 and TNF signaling. *EMBO J.* 22:6277–88

55. Inagaki M, Omori E, Kim JY, Komatsu Y, Scott G, et al. 2008. TAK1-binding protein 1, TAB1, mediates osmotic stress-induced TAK1 activation but is dispensable for TAK1-mediated cytokine signaling. *J. Biol. Chem.* 283:33080–86

56. Chen G, Goeddel DV. 2002. TNF-R1 signaling: a beautiful pathway. *Science* 296:1634–35

57. Shi CS, Kehrl JH. 2003. Tumor necrosis factor (TNF)-induced germinal center kinase-related (GCKR) and stress-activated protein kinase (SAPK) activation depends upon the E2/E3 complex Ubc13-Uev1A/TNF receptor-associated factor 2 (TRAF2). *J. Biol. Chem.* 278:15429–34

58. Lee SY, Reichlin A, Santana A, Sokol KA, Nussenzweig MC, Choi Y. 1997. TRAF2 is essential for JNK but not NF-kappaB activation and regulates lymphocyte proliferation and survival. *Immunity* 7:703–13

59. Yeh WC, Shahinian A, Speiser D, Kraanus J, Billia F, et al. 1997. Early lethality, functional NF-kappaB activation, and increased sensitivity to TNF-induced cell death in TRAF2-deficient mice. *Immunity* 7:715–25

60. Tada K, Okazaki T, Sakon S, KobaraI T, Kurosawa K, et al. 2001. Critical roles of TRAF2 and TRAF5 in tumor necrosis factor-induced NF-kappaB activation and protection from cell death. *J. Biol. Chem.* 276:36530–34

61. Baud V, Liu ZG, Bennett B, Suzuki N, Xia Y, Karin M. 1999. Signaling by proinflammatory cytokines: oligomerization of TRAF2 and TRAF6 is sufficient for JNK and IKK activation and target gene induction via an amino-terminal effector domain. *Genes Dev.* 13:1297–308

62. Li H, Kobayashi M, Blonska M, You Y, Lin X. 2006. Ubiquitination of RIP is required for tumor necrosis factor alpha-induced NF-kappaB activation. *J. Biol. Chem.* 281:13636–43

63. Legler DF, Micheau O, Doucey MA, Tschopp J, Bron C. 2003. Recruitment of TNF receptor 1 to lipid rafts is essential for TNFalpha-mediated NF-kappaB activation. *Immunity* 18:655–64

64. Zhang SQ, Kovalenko A, Cantarella G, Wallach D. 2000. Recruitment of the IKK signalosome to the p55 TNF receptor: RIP and A20 bind to NEMO (IKKgamma) upon receptor stimulation. *Immunity* 12:301–11

65. Sun SC. 2008. Deubiquitylation and regulation of the immune response. *Nat. Rev. Immunol.* 8:501–11

66. Brummelkamp TR, Nijman SM, Dirac AM, Bernards R. 2003. Loss of the cylindromatosis tumour suppressor inhibits apoptosis by activating NF-kappaB. *Nature* 424:797–801

67. Kovalenko A, Chable-Bessia C, Cantarella G, Israel A, Wallach D, Courtois G. 2003. The tumour suppressor CYLD negatively regulates NF-kappaB signalling by deubiquitination. *Nature* 424:801–5

68. Trompouki E, Hatzivassiliou E, Tsichritzis T, Farmer H, Ashworth A, Mosialos G. 2003. CYLD is a deubiquitinating enzyme that negatively regulates NF-kappaB activation by TNFR family members. *Nature* 424:793–96

69. Bignell GR, Warren W, Seal S, Takahashi M, Rapley E, et al. 2000. Identification of the familial cylindromatosis tumour-suppressor gene. *Nat. Genet.* 25:160–65

70. Massoumi R, Chmielarska K, Hennecke K, Pfeifer A, Fassler R. 2006. Cyld inhibits tumor cell proliferation by blocking Bcl-3-dependent NF-kappaB signaling. *Cell* 125:665–77

71. Zhang J, Stirling B, Temmerman ST, Ma CA, Fuss IJ, et al. 2006. Impaired regulation of NF-kappaB and increased susceptibility to colitis-associated tumorigenesis in CYLD-deficient mice. *J. Clin. Investig.* 116:3042–49

72. Reiley WW, Zhang M, Jin W, Losiewicz M, Donohue KB, et al. 2006. Regulation of T cell development by the deubiquitinating enzyme CYLD. *Nat. Immunol.* 7:411–17

73. Reiley WW, Jin W, Lee AJ, Wright A, Wu X, et al. 2007. Deubiquitinating enzyme CYLD negatively regulates the ubiquitin-dependent kinase Tak1 and prevents abnormal T cell responses. *J. Exp. Med.* 204:1475–85

74. Krikos A, Laherty CD, Dixit VM. 1992. Transcriptional activation of the tumor necrosis factor alpha-inducible zinc finger protein, A20, is mediated by kappa B elements. *J. Biol. Chem.* 267:17971–76

75. Lee EG, Boone DL, Chai S, Libby SL, Chien M, et al. 2000. Failure to regulate TNF-induced NF-kappaB and cell death responses in A20-deficient mice. *Science* 289:2350–54

76. Musone SL, Taylor KE, Lu TT, Nititham J, Ferreira RC, et al. 2008. Multiple polymorphisms in the TNFAIP3 region are independently associated with systemic lupus erythematosus. *Nat. Genet.* 40:1062–64

77. Wertz IE, O’Rourke KM, Zhou H, Eby M, Aravind L, et al. 2004. De-ubiquitination and ubiquitin ligase domains of A20 downregulate NF-kappaB signalling. *Nature* 430:694–99

78. Boone DL, Turer EE, Lee EG, Ahmad RC, Wheeler MT, et al. 2004. The ubiquitin-modifying enzyme A20 is required for termination of Toll-like receptor responses. *Nat. Immunol.* 5:1052–60

79. Evans PC, Ovaa H, Hamon M, Kilshaw PJ, Hamm S, et al. 2004. Zinc-finger protein A20, a regulator of inflammation and cell survival, has de-ubiquitinating activity. *Biochem. J.* 378:727–34

80. Komander D, Barford D. 2008. Structure of the A20 OTU domain and mechanistic insights into deubiquitination. *Biochem. J.* 409:77–85

81. Lin SC, Chung JY, Lamothe B, Rajashankar K, Lu M, et al. 2008. Molecular basis for the unique deubiquitinating activity of the NF-kappaB inhibitor A20. *J. Mol. Biol.* 376:526–40

82. Shembade N, Harhaj NS, Parvatiyar K, Copeland NG, Jenkins NA, et al. 2008. The E3 ligase Itch negatively regulates inflammatory signaling pathways by controlling the function of the ubiquitin-editing enzyme A20. *Nat. Immunol.* 9:254–62

83. Ling L, Goeddel DV. 2000. T6BP, a TRAF6-interacting protein involved in IL-1 signaling. *Proc. Natl. Acad. Sci. USA* 97:9567–72

84. Shembade N, Harhaj NS, Liebl DJ, Harhaj EW. 2007. Essential role for TAX1BP1 in the termination of TNF-alpha-, IL-1- and LPS-mediated NF-kappaB and JNK signaling. *EMBO J.* 26:3910–22

85. Iha H, Peloponese JM, Verstrepen L, Zapart G, Ikeda F, et al. 2008. Inflammatory cardiac valvulitis in TAX1BP1-deficient mice through selective NF-kappaB activation. *EMBO J.* 27:629–41

86. Perry WL, Hustad CM, Swing DA, O’Sullivan TN, Jenkins NA, Copeland NG. 1998. The itchy locus encodes a novel ubiquitin protein ligase that is disrupted in a18H mice. *Nat. Genet.* 18:143–46

87. Sun SC, Ballard DW. 1999. Persistent activation of NF-kappaB by the tax transforming protein of HTLV-1: hijacking cellular IkappaB kinases. *Oncogene* 18:6948–58

88. Shembade N, Harhaj NS, Yamamoto M, Akira S, Harhaj EW. 2007. The human T-cell leukemia virus type 1 Tax oncoprotein requires the ubiquitin-conjugating enzyme Ubc13 for NF-kappaB activation. *J. Virol.* 81:13735–42

89. Krappmann D, Scheidereit C. 2005. A pervasive role of ubiquitin conjugation in activation and termination of IkappaB kinase pathways. *EMBO Rep.* 6:321–26

90. Rawlings DJ, Sommer K, Moreno-Garcia ME. 2006. The CARMA1 signalosome links the signalling machinery of adaptive and innate immunity in lymphocytes. *Nat. Rev. Immunol.* 6:799–812

91. van Oers NS, Chen ZJ. 2005. Cell biology. Kinasing and clipping down the NF-kappa B trail. *Science* 308:65–66

92. Zhou H, Wertz I, O’Rourke K, Ultsch M, Seshagiri S, et al. 2004. Bcl10 activates the NF-kappaB pathway through ubiquitination of NEMO. *Nature* 427:167–71

93. Sun L, Deng L, Ea CK, Xia ZP, Chen ZJ. 2004. The TRAF6 ubiquitin ligase and TAK1 kinase mediate IKK activation by BCL10 and MALT1 in T lymphocytes. *Mol. Cell* 14:289–301

94. Ni CY, Wu ZH, Florence WC, Parekh VV, Arrate MP, et al. 2008. Cutting edge: K63-linked polyubiquitination of NEMO modulates TLR signaling and inflammation in vivo. *J. Immunol.* 180:7107–11

95. Oeckinghaus A, Wegener E, Welteke V, Ferch U, Arslan SC, et al. 2007. Malt1 ubiquitination triggers NF-kappaB signaling upon T-cell activation. *EMBO J.* 26:4634–45

96. Wu CJ, Ashwell JD. 2008. NEMO recognition of ubiquitinated Bcl10 is required for T cell receptor-mediated NF-kappaB activation. *Proc. Natl. Acad. Sci. USA* 105:3023–28

97. Pichlmair A, Reis e Sousa C. 2007. Innate recognition of viruses. *Immunity* 27:370–83

98. Seth RB, Sun L, Chen ZJ. 2006. Antiviral innate immunity pathways. *Cell Res.* 16:141–47

99. Gack MU, Shin YC, Joo CH, Urano T, Liang C, et al. 2007. TRIM25 RING-finger E3 ubiquitin ligase is essential for RIG-I-mediated antiviral activity. *Nature* 446:916–20

100. Häcker H, Redecke V, Blagoev B, Kratchmarova I, Hsu LC, et al. 2006. Specificity in Toll-like receptor signalling through distinct effector functions of TRAF3 and TRAF6. *Nature* 439:204–7

101. Oganesyan G, Saha SK, Guo B, He JQ, Shahangian A, et al. 2005. Critical role of TRAF3 in the Toll-like receptor-dependent and -independent antiviral response. *Nature* 439:208–11

102. Saha SK, Pietras EM, He JQ, Kang JR, Liu SY, et al. 2006. Regulation of antiviral responses by a direct and specific interaction between TRAF3 and Cardif. *EMBO J.* 25:3257–63

103. Kayagaki N, Phung Q, Chan S, Chaudhari R, Quan C, et al. 2007. DUBA: a deubiquitinase that regulates type I interferon production. *Science* 318:1628–32

104. Kanneganti TD, Lamkanfi M, Nunez G. 2007. Intracellular NOD-like receptors in host defense and disease. *Immunity* 27:549–59

105. Abbott DW, Wilkins A, Asara JM, Cantley LC. 2004. The Crohn’s disease protein, NOD2, requires RIP2 in order to induce ubiquitinylation of a novel site on NEMO. *Curr. Biol.* 14:2217–27

106. Abbott DW, Yang Y, Hutti JE, Madhavarapu S, Kelliher MA, Cantley LC. 2007. Coordinated regulation of Toll-like receptor and NOD2 signaling by K63-linked polyubiquitin chains. *Mol. Cell. Biol.* 27:6012–25

107. Hasegawa M, Fujimoto Y, Lucas PC, Nakano H, Fukase K, et al. 2008. A critical role of RICK/RIP2 polyubiquitination in Nod-induced NF-kappaB activation. *EMBO J.* 27:373–83

108. Yang Y, Yin C, Pandey A, Abbott D, Sassetti C, Kelliher MA. 2007. NOD2 pathway activation by MDP or *Mycobacterium tuberculosis* infection involves the stable polyubiquitination of Rip2. *J. Biol. Chem.* 282:36223–29

109. Hitotsumatsu O, Ahmad RC, Tavares R, Wang M, Philpott D, et al. 2008. The ubiquitin-editing enzyme A20 restricts nucleotide-binding oligomerization domain containing 2-triggered signals. *Immunity* 28:381–90

110. Huang TT, Wuerzberger-Davis SM, Wu ZH, Miyamoto S. 2003. Sequential modification of NEMO/IKKgamma by SUMO-1 and ubiquitin mediates NF-kappaB activation by genotoxic stress. *Cell* 115:565–76

111. Wu ZH, Shi Y, Tibbetts RS, Miyamoto S. 2006. Molecular linkage between the kinase ATM and NF-kappaB signaling in response to genotoxic stimuli. *Science* 311:1141–46

112. Mabb AM, Wuerzberger-Davis SM, Miyamoto S. 2006. PIASy mediates NEMO sumoylation and NF-kappaB activation in response to genotoxic stress. *Nat. Cell Biol.* 8:986–93

113. Doffinger R, Smahi A, Bessia C, Geissmann F, Feinberg J, et al. 2001. X-linked anhidrotic ectodermal dysplasia with immunodeficiency is caused by impaired NF-kappaB signaling. *Nat. Genet.* 27:277–85

114. Varadan R, Assfalg M, Haririnia A, Raasi S, Pickart C, Fushman D. 2004. Solution conformation of Lys63-linked di-ubiquitin chain provides clues to functional diversity of polyubiquitin signaling. *J. Biol. Chem.* 279:7055–63

115. Tang ED, Wang CY, Xiong Y, Guan KL. 2003. A role for NF-kappaB essential modifier/IkappaB kinase-gamma (NEMO/IKKgamma) ubiquitination in the activation of the IkappaB kinase complex by tumor necrosis factor-alpha. *J. Biol. Chem.* 278:37297–305

116. Bloor S, Ryzhakov G, Wagner S, Butler PJ, Smith DL, et al. 2008. Signal processing by its coil zipper domain activates IKK gamma. *Proc. Natl. Acad. Sci. USA* 105:1279–84

117. Spencer E, Jiang J, Chen ZJ. 1999. Signal-induced ubiquitination of IkappaBalpha by the F-box protein Slimb/beta-TrCP. *Genes Dev.* 13:284–94

118. Winston JT, Strack P, Beer-Romero P, Chu CY, Elledge SJ, Harper JW. 1999. The SCFbeta-TRCP-ubiquitin ligase complex associates specifically with phosphorylated destruction motifs in IkappaBalpha and beta-catenin and stimulates IkappaBalpha ubiquitination in vitro. *Genes Dev.* 13:270–83

119. Yaron A, Hatzubai A, Davis M, Lavon I, Amit S, et al. 1998. Identification of the receptor component of the IkappaBalpha-ubiquitin ligase. *Nature* 396:590–94

120. Guardavaccaro D, Kudo Y, Boulaire J, Barchi M, Busino L, et al. 2003. Control of meiotic and mitotic progression by the F box protein beta-Trcp1 in vivo. *Dev. Cell* 4:799–812

121. Ciechanover A, Gonen H, Bercovich B, Cohen S, Fajerman I, et al. 2001. Mechanisms of ubiquitin-mediated, limited processing of the NF-kappaB1 precursor protein p105. *Biochimie* 83:341–49

122. Fong A, Sun SC. 2002. Genetic evidence for the essential role of beta-transducin repeat-containing protein in the inducible processing of NF-kappa B2/p100. *J. Biol. Chem.* 277:22111–14

123. Frescas D, Pagano M. 2008. Deregulated proteolysis by the F-box proteins SKP2 and beta-TrCP: tipping the scales of cancer. *Nat. Rev. Cancer* 8:438–49

124. Pomerantz JL, Baltimore D. 2002. Two pathways to NF-kappaB. *Mol. Cell* 10:693–95

125. Morrison MD, Reiley W, Zhang M, Sun SC. 2005. An atypical tumor necrosis factor (TNF) receptor-associated factor-binding motif of B cell-activating factor belonging to the TNF family (BAFF) receptor mediates induction of the noncanonical NF-kappaB signaling pathway. *J. Biol. Chem.* 280:10018–24

126. Liao G, Zhang M, Harhaj EW, Sun SC. 2004. Regulation of the NF-kappaB-inducing kinase by tumor necrosis factor receptor-associated factor 3-induced degradation. *J. Biol. Chem.* 279:26243–50

127. He JQ, Zarnegar B, Oganesyan G, Saha SK, Yamazaki S, et al. 2006. Rescue of TRAF3-null mice by p100 NF-kappa B deficiency. *J. Exp. Med.* 203:2413–18

128. He JQ, Saha SK, Kang JR, Zarnegar B, Cheng G. 2007. Specificity of TRAF3 in its negative regulation of the noncanonical NF-kappa B pathway. *J. Biol. Chem.* 282:3688–94

129. Vallabhapurapu S, Matsuzawa A, Zhang W, Tseng PH, Keats JJ, et al. 2008. Nonredundant and complementary functions of TRAF2 and TRAF3 in a ubiquitination cascade that activates NIK-dependent alternative NF-kappaB signaling. *Nat. Immunol.* 9:1364–70

130. Grech AP, Amesbury M, Chan T, Gardam S, Basten A, Brink R. 2004. TRAF2 differentially regulates the canonical and noncanonical pathways of NF-kappaB activation in mature B cells. *Immunity* 21:629–42

131. Xia ZP, Chen ZJ. 2005. TRAF2: a double-edged sword? *Sci. STKE* 2005:pe7

132. Zarnegar BJ, Wang Y, Mahoney DJ, Dempsey PW, Cheung HH, et al. 2008. Noncanonical NF-kappaB activation requires coordinated assembly of a regulatory complex of the adaptors cIAP1, cIAP2, TRAF2 and TRAF3 and the kinase NIK. *Nat. Immunol.* 9:1371–78

133. Varfolomeev E, Blankenship JW, Wayson SM, Fedorova AV, Kayagaki N, et al. 2007. IAP antagonists induce autoubiquitination of c-IAPs, NF-kappaB activation, and TNFalpha-dependent apoptosis. *Cell* 131:669–81

134. Vince JE, Wong WW, Khan N, Feltham R, Chau D, et al. 2007. IAP antagonists target cIAP1 to induce TNFalpha-dependent apoptosis. *Cell* 131:682–93

135. Rothe M, Pan MG, Henzel WJ, Ayres TM, Goeddel DV. 1995. The TNFR2-TRAF signaling complex contains two novel proteins related to baculoviral inhibitor of apoptosis proteins. *Cell* 83:1243–52

136. Yang Y, Fang S, Jensen JP, Weissman AM, Ashwell JD. 2000. Ubiquitin protein ligase activity of IAPs and their degradation in proteasomes in response to apoptotic stimuli. *Science* 288:874–77

137. Hsu H, Shu HB, Pan MG, Goeddel DV. 1996. TRADD-TRAF2 and TRADD-FADD interactions define two distinct TNF receptor 1 signal transduction pathways. *Cell* 84:299–308

138. Micheau O, Tschopp J. 2003. Induction of TNF receptor I-mediated apoptosis via two sequential signaling complexes. *Cell* 114:181–90

139. Srinivasula SM, Ashwell JD. 2008. IAPs: What’s in a name? *Mol. Cell* 30:123–35

140. Du C, Fang M, Li Y, Li L, Wang X. 2000. Smac, a mitochondrial protein that promotes cytochrome c-dependent caspase activation by eliminating IAP inhibition. *Cell* 102:33–42

141. Yang QH, Du C. 2004. SMAC/DIABLO selectively reduces the levels of c-IAP1 and c-IAP2 but not that of XIAP and livin in HeLa cells. *J. Biol. Chem.* 279:16963–70

142. Petersen SL, Wang L, Yalcin-Chin A, Li L, Peyton M, et al. 2007. Autocrine TNFalpha signaling renders human cancer cells susceptible to Smac-mimetic-induced apoptosis. *Cancer Cell* 12:445–56

143. Wang L, Du F, Wang X. 2008. TNF-alpha induces two distinct caspase-8 activation pathways. *Cell* 133:693–703

144. O’Donnell MA, Legarda-Addison D, Skountzos P, Yeh WC, Ting AT. 2007. Ubiquitination of RIP1 regulates an NF-kappaB-independent cell-death switch in TNF signaling. *Curr. Biol.* 17:418–24

145. Bertrand MJ, Milutinic S, Dickson KM, Ho WC, Boudreault A, et al. 2008. cIAP1 and cIAP2 facilitate cancer cell survival by functioning as E3 ligases that promote RIP1 ubiquitination. *Mol. Cell* 30:689–700

146. Papa S, Bubici C, Zazzeroni F, Pham CG, Kuntzen C, et al. 2006. The NF-kappaB-mediated control of the JNK cascade in the antagonism of programmed cell death in health and disease. *Cell Death Differ.* 13:712–29

147. Chang L, Kamata H, Solinas G, Luo JL, Maeda S, et al. 2006. The E3 ubiquitin ligase itch couples JNK activation to TNFalpha-induced cell death by inducing c-FLIP(L) turnover. *Cell* 124:601–13

148. Tang G, Minemoto Y, Dibling B, Purcell NH, Li Z, et al. 2001. Inhibition of JNK activation through NF-kappaB target genes. *Nature* 414:313–17

149. De Smaele E, Zazzeroni F, Papa S, Nguyen DU, Jin R, et al. 2001. Induction of gadd45beta by NF-kappaB downregulates pro-apoptotic JNK signalling. *Nature* 414:308–13

150. Kamata H, Honda S, Maeda S, Chang L, Hirata H, Karin M. 2005. Reactive oxygen species promote TNFalpha-induced death and sustained JNK activation by inhibiting MAP kinase phosphatases. *Cell* 120:649–61

Contents

Preface . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Recognition and Processing of Ubiquitin-Protein Conjugates by the Proteasome  
Daniel Finley ……………………………………………………………… 477  

Degradation of Activated Protein Kinases by Ubiquitination  
Zhimin Lu and Tony Hunter ………………………………………… 435  

The Role of Ubiquitin in NF-κB Regulatory Pathways  
Brian Skaug, Xiaomo Jiang, and Zhijian J. Chen …………………… 769  

Biological and Chemical Approaches to Diseases of Proteostasis Deficiency  
Evan T. Powers, Richard I. Morimoto, Andrew Dillin, Jeffery W. Kelly,  
and William E. Balch ………………………………………………… 959  

Gene Expression  

RNA Polymerase Active Center: The Molecular Engine of Transcription  
Evgeny Nudler ………………………………………………………… 335  

Genome-Wide Views of Chromatin Structure  
Oliver J. Rando and Howard Y. Chang ……………………………… 245  

The Biology of Chromatin Remodeling Complexes  
Cedric R. Clapier and Bradley R. Cairns …………………………… 273  

The Structural and Functional Diversity of Metabolite-Binding Riboswitches  
Adam Roth and Ronald R. Breaker …………………………………… 305  

Lipid and Membrane Biogenesis  

Genetic and Biochemical Analysis of Non-Vesicular Lipid Traffic  
Dennis R. Voelker …………………………………………………… 827  

Cholesterol 24-Hydroxylase: An Enzyme of Cholesterol Turnover in the Brain  
David W. Russell, Rebekkah W. Halford, Denise M.O. Ramirez, Rahul Shah,  
and Tiina Kotti ……………………………………………………… 1017  

Lipid-Dependent Membrane Protein Topogenesis  
William Dowhan and Mikhail Bogdanov ……………………………… 515  

Single-Molecule Studies of the Neuronal SNARE Fusion Machinery  
Axel T. Brunger, Keith Weninger, Mark Bowen, and Steven Chu ………… 903  

Mechanisms of Endocytosis  
Gary J. Doherty and Harvey T. McMahon …………………………… 857  

Recent Advances in Biochemistry

Motors, Switches, and Contacts in the Replisome  
Samir M. Hamdan and Charles C. Richardson ………… 205  

Large-Scale Structural Biology of the Human Proteome  
Aled Edwards ………… 541  

Collagen Structure and Stability  
Matthew D. Shoulders and Ronald T. Raines ………… 929  

The Structural and Biochemical Foundations of Thiamin Biosynthesis  
Christopher T. Jurgenson, Tadhg P. Begley, and Steven E. Ealick ………… 569  

Proton-Coupled Electron Transfer in Biology: Results from  
Synergistic Studies in Natural and Model Systems  
Steven Y. Reece and Daniel G. Nocera ………… 673  

Mechanism of Mo-Dependent Nitrogenase  
Lance C. Seefeldt, Brian M. Hoffman, and Dennis R. Dean ………… 701  

Inorganic Polyphosphate: Essential for Growth and Survival  
Narayana N. Rao, María R. Gómez-García, and Arthur Kornberg ………… 605  

Essentials for ATP Synthesis by F₁F₀ ATP Synthases  
Christoph von Ballmoos, Alexander Wiedenmann, and Peter Dimroth ………… 649  

The Chemical Biology of Protein Phosphorylation  
Mary Katherine Tarrant and Philip A. Cole ………… 797  

Sphingosine 1-Phosphate Receptor Signaling  
Hugh Rosen, Pedro J. Gonzalez-Cabrera, M. Germana Sanna, and Steven Brown ………… 743  

The Advent of Near-Atomic Resolution in Single-Particle Electron  
Microscopy  
Yifan Cheng and Thomas Walz ………… 723  

Super-Resolution Fluorescence Microscopy  
Bo Huang, Mark Bates, and Xiaowei Zhuang ………… 993  

Indexes  

Cumulative Index of Contributing Authors, Volumes 74–78 ………… 1041  
Cumulative Index of Chapter Titles, Volumes 74–78 ………… 1045  

Errata  

An online log of corrections to *Annual Review of Biochemistry* articles may be found at  
http://biochem.annualreviews.org/errata.shtml
