
PROGRESS

Prokaryotic ubiquitin-like protein (Pup), proteasomes and pathogenesis

K. Heran Darwin

Abstract | Proteasomes are ATP-dependent, multisubunit proteases that are found in all eukaryotes and archaea and some bacteria. In eukaryotes, the small protein ubiquitin is covalently attached in a post-translational manner to proteins that are targeted for proteasomal degradation. Despite the presence of proteasomes in many prokaryotes, ubiquitin or other post-translational protein modifiers were presumed to be absent from these organisms. Recently a prokaryotic ubiquitin-like protein, Pup, was found to target proteins for proteolysis by the Mycobacterium tuberculosis proteasome. The discovery of this ubiquitin-like modifier opens up the possibility that other bacteria may also have small post-translational protein tagging systems, with the ability to affect cellular processes.

Proteasomes regulate a multitude of functions in eukaryotes and are essential for life. The eukaryotic proteasome core (20S) is composed of four rings: two hetero-heptameric rings of β-subunits sandwiched between two hetero-heptameric rings of α-subunits that restrict access to the catalytic core¹ (FIG. 1). Eukaryotic proteasome cores are highly complex, with three of the seven different β-subunits having catalytic activity (TABLE 1). Entry of substrates into the proteasome core, where they are hydrolysed, usually requires the activity of a hexameric ring of regulatory particle ATPases that cap the ends of the proteasome cores. In addition to the ATPases, numerous accessory factors, including regulatory particle non-ATPases and deubiquitinases (DUBs), participate in the recognition, unfolding and degradation of substrates that are post-translationally tagged with chains of ubiquitin².

Almost all proteins that are targeted for proteasomal degradation in eukaryotes are tagged with ubiquitin. Immature ubiquitin polypeptides are processed by proteases, including ubiquitin-specific proteases and ubiquitin carboxy-terminal hydrolases, resulting in mature ubiquitin molecules that end in a Gly-Gly motif, a feature common to all conjugatable ubiquitin-like (UBL) modifiers³⁻⁵. The newly exposed C-terminal glycine is subjected to a series of reactions that result in the conjugation of ubiquitin to lysine residues in target proteins²,⁶,⁷ (FIG. 1). Activating enzymes use ATP to adenylate the C-terminal glycine of ubiquitin, which is then passed to an active-site cysteine in the E1 enzyme. From here, ubiquitin is transferred to a ubiquitin-conjugating enzyme (E2) that delivers ubiquitin to a protein ligase (E3), which catalyses the formation of an isopeptide bond with a lysine on the target substrate. E3 ligases are numerous in eukaryotes and contain various substrate-binding activities that provide specificity to the ubiquitin–proteasome system by determining which substrates are conjugated with ubiquitin. In general, proteins that are targeted to the proteasome have additional ubiquitin molecules that are successively attached to Lys48 on other ubiquitin molecules, forming polyubiquitin chains²,⁸,⁹. Polyubiquitin chains can be recognized by the regulatory complex of the proteasome, then removed and recycled by DUBs for further ubiquitylation reactions (FIG. 1).

Regulated proteolysis has profound implications for eukaryotic physiology, controlling numerous processes, including gene expression and cell division. In addition to targeting proteins for degradation, ubiquitin also forms scaffolds on which other proteins can interact. Ubiquitin has the potential to form polyubiquitin chains using any one of its seven lysine residues, or even its amino terminus. For example, the nuclear factor-κB (NF-κB) pathway uses at least two types of ubiquitin linkage to activate gene expression¹⁰. NF-κB is a transcription factor that is bound by the inhibitor IkB in the eukaryotic cytosol, where it is unable to activate gene expression. In the interleukin-1 (IL-1) and Toll-like receptor (TLR) pathways of NF-κB activation, the E3 ligase TNF receptor-associated factor 6 (TRAF6) is activated to form Lys63-linked ubiquitin chains on the regulatory protein, NEMO, as well as on TRAF6 itself. The Lys63 chains are thought to form scaffolds that recruit additional factors that ultimately activate the IkB kinase, IKK. Activation of IKK leads to the phosphorylation of IkB, which results in Lys48-linked polyubiquitylation of IkB and its subsequent degradation. Degradation of IkB releases NF-κB, allowing it to translocate into the nucleus where it can activate gene expression. Thus, two different ubiquitin linkages have crucial roles in a single important pathway.

In addition to polyubiquitylation, monoubiquitylation is important for numerous functions, including the regulation of enzyme activity, DNA repair and subcellular protein targeting¹¹. Other UBLs, such as small ubiquitin-related modifier (SUMO) and NEDD8, also affect various important cellular processes⁶ (TABLE 2).

Unlike their eukaryotic counterparts, the roles of proteasomes in prokaryotic physiology are largely unknown¹². Furthermore, despite the presence of bacterial and archaeal proteasomes, ubiquitin and UBLs had never been successfully identified in prokaryotes, leading to the conclusion that they were absent from this domain of life. This presumption was recently overturned with the report of a small protein modifier in bacteria that targets proteins for degradation by a bacterial proteasome¹³. The discovery of a post-translational tagging system in bacteria has opened up the possibility that small protein modifiers, like

NATURE REVIEWS | MICROBIOLOGY

© 2009 Macmillan Publishers Limited. All rights reserved

PROGRESS

![Diagram](#)

Figure 1 | Overview of the eukaryotic ubiquitin–proteasome system. Ubiquitin (Ub) is encoded by four different loci in yeast as part of a larger polypeptide⁴⁹. Processing proteases expose carboxy-terminal Gly-Gly motifs that are activated by adenylation with an E1 enzyme. The E1 enzyme subsequently transfers Ub to an E2 enzyme, where a thioester bond is formed. The E2 then transfers Ub to any number of E3 ligases. The E3 ligase family can be subdivided into HECT (homologous to the E6-APC terminus) and RING (really interesting gene) domain ligases. RING ligases hold both the E2 and substrate, and facilitate the direct transfer of Ub from the E2 to the substrate; by contrast, HECT ligases form a thioester bond with Ub before transfer to a substrate lysine. E3 ligases dictate the type of Ub linkage that is formed. Proteins with Lys48-linked chains are usually targeted for degradation by the 26S proteasome. Other types of Ub linkage (mono- and poly-Lys63 and others) can result in degradation but generally serve other functions (see the main text for details). DUB, deubiquitinase; NF-κB, nuclear factor-κB; RPN, regulatory particle non-ATPase; RPT, regulatory particle ATPase; UCH, Ub C-terminal hydrolase; USP, Ub-specific protease.

have proteasomes¹⁹,²²⁻²⁴. Multisubunit regulatory complexes that are similar to those in eukaryotes have not been identified in prokaryotes, suggesting that the mechanisms by which proteins are targeted for degradation are different, or that regulatory complex interactions with cores are transient or weak. So far, the best evidence that a proteasome-type ATPase can interact with a prokaryotic proteasome was reported by Smith and co-workers, who showed that the archaeal proteasome-activating nucleotidase (PAN) from *Methanococcus jannaschii* could interact with *Thermoplasma acidophilum* proteasome cores to stimulate the degradation of an unfolded, non-native protein²⁵. However, no one has ever demonstrated the degradation of polypeptides by bacterial proteasomes *in vitro*.

### Proteasomes and pathogenesis

The first known function associated with any prokaryotic proteasome was discovered in the bacterial pathogen *M. tuberculosis*. *M. tuberculosis* is the third highest cause of death in the world, and new and improved therapies are desperately needed. One approach to develop new antimicrobial drugs is to find compounds that attack the pathways that normally protect bacteria against host immunity. Work from the Nathan laboratory demonstrated that the production of nitric oxide (NO) by inducible nitric oxide synthase (iNOS) in macrophages is essential to control *M. tuberculosis* growth in mice²⁶. NO has numerous activities that restrict microbial growth, including the ability to damage nucleic acids, proteins and lipids²⁷. Although studies have shown that mice which are wild type for iNOS survived much longer after *M. tuberculosis* infection than iNOS-deficient mice, bacteria were never completely sterilized from the animals. This suggested that *M. tuberculosis* had mechanisms to resist eradication by NO. In an effort to find new targets for antimycobacterial drug development, a screen for *M. tuberculosis* mutants that were sensitized to NO was performed. Mutations in genes encoding the putative proteasome accessory factors Mpa (*Mycobacterium* proteasome-associated ATPase) and PafA (proteasome accessory factor A) sensitized *M. tuberculosis* to NO *in vitro* and, importantly, severely attenuated *M. tuberculosis* growth in mice²².

Mpa and PafA were postulated to participate in proteasome function because they are encoded near proteasome core genes

those in eukaryotes, could have far-reaching implications for prokaryotic physiology and even pathogenesis. Here, I discuss what is currently known about bacterial proteasomes and UBL biology, focusing on the pathogen *Mycobacterium tuberculosis*.

### Prokaryotic proteasomes

Any one bacterial species usually has a collection of ATP-dependent proteases, such as ClpP, HslV, Lon or FtsH¹⁴. In addition to these proteases, some bacteria and all archaea have proteasomes. Like their eukaryotic counterparts, prokaryotic proteasome 20S cores are self-compartmentalized proteases composed of 14 α-subunits and 14 β-subunits, with the N-terminal threonines of the β-subunits providing the protease activity¹⁵. In contrast to eukaryotic proteasomes, core particles from archaea and bacteria are far simpler structures with homo-heptameric rings of catalytic β-subunits flanked by homo-heptameric rings of α-subunits¹⁶⁻²¹ (FIG. 2; TABLE 1). To date, only bacteria found in the class Actinomycetes are known to

© 2009 Macmillan Publishers Limited. All rights reserved

Table 1 | Features of eukaryotic and prokaryotic proteasome systems*

| Eukarya | Bacteria or archaea |
| --- | --- |
| **Core structure** | Two rings of seven different α-subunits; two rings of seven different β-subunits | Two rings of seven identical α-subunits; two rings of seven identical β-subunits |
| **Active site (or sites)** | Three β-subunit amino-terminal threonines; three activities: tryptic, chymotryptic and post-acidic | All β-subunit amino-terminal threonines: chymotryptic; *Mycobacterium tuberculosis* proteasome has additional activities |
| **Accessory factors** | Regulatory complex composed of numerous subunits, including ubiquitin-binding proteins, deubiquitylases; hetero-hexamer of AAA ATPases | Homo-hexamers of AAA ATPases; PafA: required for pupylation in *M. tuberculosis* |
| **Substrates** | Numerous, most require ubiquitin conjugation prior to proteolysis; some substrates do not require ubiquitin (such as ornithine decarboxylase) | Mycobacteria: FabD, Ino1, Mpa, PanB, SodA (all require Pup) <br> Archaea: unknown |
| **Cellular pathways** | Includes protein turnover, signal transduction, nuclear factor-κB regulation, endocytosis, DNA repair, autophagy, cell cycle and transcription | Mycobacteria: protein turnover <br> Archaea: unknown |

*References and additional details are in the main text.

and are usually only found in proteasome-bearing bacteria²⁸. Mpa has sequence similarity to eukaryotic proteasomal ATPases and is proposed to bind, unfold and deliver degradation substrates into the proteasome core. PafA did not resemble any protein of known function but nonetheless seemed to be in the same pathway as Mpa and the proteasome; *pafA* mutants were similarly sensitive to NO *in vitro*, and had the same attenuated phenotype in mice²²,²⁹. Unlike most eukaryotic proteasomal accessory factors, Mpa and PafA are not essential for growth under normal conditions. Chemical inhibition or genetic repression of proteasome protease activity also sensitized *M. tuberculosis* to NO, linking proteasome activity to Mpa and PafA²²,³⁰. However, *M. tuberculosis* proteasome protease activity was required for normal growth under non-stressed conditions, suggesting that the proteasome core has functions that are independent of both Mpa and PafA.

### Bacterial ‘ubiquitin’

One of the mysteries of the bacterial proteasome system was how proteins were targeted for degradation in the apparent absence of ubiquitin or UBLs. To begin to answer this question, natural substrates of the *M. tuberculosis* proteasome were first identified. Two biosynthetic enzymes, FabD (malonyl CoA-acyl carrier protein) and PanB (ketopantoate hydroxymethyltransferase), were identified as degradation substrates³¹. Both proteins accumulated in

proteasome-bearing bacteria, were necessary to facilitate degradation. The development of a new two-hybrid system allowed this hypothesis to be tested by looking for interactions between proteasome components and degradation substrates in mycobacteria³². A positive interaction was detected between the substrate FabD and Rv2111c¹³. Surprisingly, after validation of this result, epitope-tagged FabD and Rv2111c co-immunoprecipitated from mycobacterial lysates as a covalently linked complex, and not as separate proteins. Mass spectrometry revealed that Rv2111c formed an isopeptide bond between its C terminus and the ε-amino group of a specific lysine (Lys173) in FabD.

Although Rv2111c is not predicted to have a ubiquitin fold, a feature common among almost all UBLs, it nonetheless has a penultimate C-terminal Gly-Gly motif. The C terminus of Rv2111c is Gly-Gly-Gln, and therefore it was predicted that the glutamine might be removed in a manner similar to the proteolytic processing of UBLs. However, high-resolution tandem mass spectrometry revealed that not only was the terminal glutamine retained, but it was converted to a glutamic acid. When unconjugated Rv2111c that had been purified from mycobacteria was analysed by mass spectrometry nearly all molecules were deamidated¹³. This result suggested that deamidation preceded covalent attachment to substrate proteins.

Despite the identification of endogenous *M. tuberculosis* substrates, attempts to degrade FabD with recombinant proteasomes and Mpa *in vitro* were unsuccessful, suggesting that other cofactors were required for full proteasome function. To address the hypothesis that Mpa needed to interact with other degradation cofactors, a bacterial two-hybrid screen in *Escherichia coli* was performed with Mpa as bait. The screen resulted in the identification of Rv2111c, a protein of unknown function encoded with the proteasome core genes of *M. tuberculosis*¹³. Purified Rv2111c interacted non-covalently with Mpa but did not promote FabD degradation by proteasomes and Mpa *in vitro*.

At this point, numerous presumed players in the bacterial proteasome system had been identified, but how they interacted with each other remained ambiguous. Only circumstantial evidence suggested that there were contacts between several of the proteins, and no stable interactions between any bacterial proteasome and cognate ATPases had been reported. It was possible that additional proteins, specific to

In another study, Burns and co-workers also noticed the Gly-Gly motif in Pup³³. Using epitope-tagged Pup, they purified and identified from *Mycobacterium smegmatis* two covalently linked proteins, superoxide dismutase (SodA) and myo-inositol-1-phosphate synthase (Ino1) with the same Pup~substrate Gly-Gly-Glu~Lys linkage. In addition, the group showed that the deletion of the C-terminal glutamine abrogated pupylation. Consistent with the study by Pearce and co-workers, Burns and co-workers showed that pupylated proteins were more stable in a proteasome-defective mutant than in wild-type *M. smegmatis*.

PROGRESS

Table 2 | Comparison of the biochemistry of ubiquitin and ubiquitin-like modifiers

| Modifier          | Processing enzymes                | Conjugating enzymes*                     | Linkage   |
|-------------------|----------------------------------|------------------------------------------|-----------|
| **Eukaryotes**    |                                  |                                          |           |
| Ubiquitin         | DUBs, USPs, UCHs                 | E1 (2)⁵⁰; E2 (>30); E3 (~600)            | GG~K      |
| SUMO1–4          | SENPs or ULPs                    | E1 (1, heterodimer); E2 (1); E3 (3)      | GG~K      |
| NEDD8             | COP9, YUH1                       | E1 (1, heterodimer); E2 (1); E3 (1)      | GG~K      |
| ISG15             | UBP43                            | E1 (1); E2 (1); E3 (2)                   | GG~K      |
| URM1              | Unknown                          | E1 (1)                                  | GG~K      |
| UBD (also known as FAT10) | Unknown                      | E1 (1)⁵¹                                | GG~K      |
| **Prokaryotes**   |                                  |                                          |           |
| Pup               | Deamidase and/or de-pupylase?    | PafA or PafD?                           | GGE~K     |

*Based on estimates for humans (the number is provided in parentheses). DUB, deubiquitinase; Paf, proteasome accessory factor; Pup, prokaryotic ubiquitin-like protein; SENP, sentrin-specific protease; SUMO, small ubiquitin-related modifier; UBD, ubiquitin D; UCH, ubiquitin carboxy-terminal hydrolase; ULP, ubiquitin-like-specific protease; URM1, ubiquitin-related modifier 1; USP, ubiquitin-specific protease.

Taken together these results revealed, for the first time, that the post-translational modification of a polypeptide by a small protein modifier can occur in bacteria in a similar manner to UBL modification in eukaryotes. So far, it seems that, unlike ubiquitin, Pup does not form chains, but it is still possible that certain substrates are poly-pupylated. It is likely that Pup targets proteins for degradation in all proteosome-bearing bacteria, but it is important to remember that eukaryotic ubiquitin has functions in addition to regulating protein stability (TABLE 1). Thus, we cannot rule out that Pup, too, may have degradation-independent functions.

### A Pup ligase?

Currently the only protein known to be essential for pupylation is PafA. A disruption mutation in *pafA* resulted in failed substrate pupylation in *M. tuberculosis*¹³. PafA has no homology with eukaryotic E1, E2 or E3 enzymes, suggesting that the activity of PafA is different from canonical UBL systems. Structural analysis using HHpred prediction software³⁴ revealed that PafA is similar to an *E. coli* protein of unknown function, YbdK, that is homologous to glutamine synthetases with γ-glutamate cysteine ligase activity³⁵ (S. Hubbard, personal communication). The structural analysis by Lehmann and co-workers suggested that YbdK does not bind ammonia like other glutamine synthetases because it lacks several amino acids that are involved in glutamine synthetase catalysis. They demonstrated that YbdK had ligase activity between glutamate and L-cysteine, but they did not know if cysteine was the normal biological substrate. No ligase activity was detected with the other 19 amino acids or ammonia.

Using a bioinformatic approach, Iyer and co-workers reported similar predictions and proposed a model in which PafA uses ATP to phosphorylate the C-terminal glutamic acid γ-carboxylate of Pup, which then reacts with the target lysine amino group to form an isopeptide bond³⁶. However, it is not yet established whether the glutamic acid γ-carboxylate of Pup is indeed the site of substrate attachment: the C-terminal glutamic acid has two carboxylate groups that have the potential to be attached to substrate lysine (FIG. 2). This model also presumes that Pup is already deamidated, and does not suggest that PafA is involved in this process.

YbdK was shown to form dimers³⁵, and Iyer and co-workers predicted that PafA might also form dimers with itself or with its homologue PafD (Rv2112c)³⁶. This hypothesis about PafA structure and function remains to be tested.

### Pupylation and pathogenesis

The failure to degrade pupylated proteins results in NO sensitivity and attenuated virulence. However, it is not yet known which degradation substrates are linked to these phenotypes. There are several possible reasons why proteasome function is protective, none of which is mutually exclusive from the others. One hypothesis is that the bacterial proteasome is required to degrade NO-damaged proteins that would otherwise be toxic to the cell. If this were true, it would be expected that damaged proteins would accumulate in proteasome-defective bacteria treated with NO. Preliminary data show that protein oxidation increases in *M. tuberculosis* treated with acidified nitrite, a source of NO, but the amount and number of oxidized protein species do not appear to be different between wild-type and Mpa-deficient *M. tuberculosis* (K.H.D., unpublished observations). This result may not be surprising given that the *M. tuberculosis* proteasome is not required for protection against several other stresses that are expected to result in protein oxidation or misfolding. Interestingly, *mpa*, *pafA* and proteasome core mutants are more resistant to hydrogen peroxide, further suggesting that the proteasome is not needed to combat all oxidant stresses²²,³⁰.

Another hypothesis is that a limited number of proteins, or even a single protein, become particularly toxic following exposure to NO. These may include metal-binding proteins, or proteins that tend to aggregate when misfolded. The accumulation of these substrates in a proteasome-defective strain under normal growth conditions might not have a deleterious effect. However, in the presence of NO, metal ions could be displaced or protein misfolding could occur, resulting in cell death.

Yet another alternative explanation that links proteasome function with pathogenesis is that the *M. tuberculosis* proteasome regulates the expression of antioxidant or virulence genes. All ATP-dependent proteases, including ClpP, Lon, FtsH and HslV, degrade transcription factors in numerous bacterial species¹⁴. In a simple scenario, it is possible that the proteasome degrades transcription factors that derepress antioxidant genes, or other genes required for *M. tuberculosis* pathogenesis. Indirect changes in gene expression owing to a lack of proteasome function could also have an impact on the ability of *M. tuberculosis* to resist NO or cause disease.

It is possible that more than one of the above scenarios explains why *M. tuberculosis* strains that are defective for protein degradation are attenuated *in vivo*. The lack of protein turnover in any organism could render it less competitive for growth under stressful conditions. Thus, targeting the proteasome pathway, including putative post-translational modification enzymes such as PafA, could be an effective approach for battling various infectious diseases.

### Other bacteria with Pup

Pup, PafA and Mpa are found in numerous genera of pathogenic and non-pathogenic actinobacteria, and almost always in strains that encode proteasomes (exceptions include

Figure 2 | Proposed model of the Pup–proteasome pathway in *Mycobacterium tuberculosis*. Unlike ubiquitin, prokaryotic ubiquitin-like protein (Pup) is not processed proteolytically from a larger precursor protein. Pup seems to be deamidated at the carboxy-terminal glutamine. It has been proposed that PafA then phosphorylates the γ-carboxylate of the C terminus of Pup, but this has not been confirmed. The attachment of Pup to the substrate lysine can potentially be through either the α- or γ-carboxylate. It is not known if polypupylation occurs, nor is it known if Pup is removed by a de-pupylase (DPup) before degradation and recycled similarly to ubiquitin.

Bifidobacterium spp. and Corynebacterium spp.) (FIG. 3). All sequenced Corynebacterium strains have a Pup homologue based on BLAST (basic local alignment and search tool) analysis³⁷, but it was previously noted that they do not encode proteasomes³⁸. Inspection of the *Corynebacterium glutamicum pup* region reveals that *mpa*, *pup* and *pafA* are conserved and *pup* is likely to be transcriptionally and translationally coupled with *pafA*, an organization that is not observed in proteasome-bearing bacteria (FIG. 3). The different organization of this region compared with the same locus in *M. tuberculosis* suggests that a deletion of the proteasome core genes occurred that left the remaining proteasome accessory factor genes intact. Interestingly, the corynebacterial Mpa homologues lack a C-terminal extension with a penultimate tyrosine that is essential for function³¹,³⁹. Site-directed mutagenesis of the penultimate tyrosine of *M. tuberculosis* Mpa resulted in failed substrate degradation and reduced protection against NO³⁹. Furthermore, a transposon mutation that deleted the last two amino acids of Mpa (tyrosine and leucine) attenuated *M. tuberculosis* virulence in mice as much as a null mutation in *mpa*³¹. The archaeal ATPase PAN also has a conserved penultimate tyrosine that is required to open proteasome cores for protein degradation²⁵. These observations suggest that the Mpa C terminus probably interacts with the proteasome core to activate degradation. Because *Corynebacterium* spp. do not appear to have proteasomes or have proteases that have dramatically diverged from proteasomes, it is possible that *Corynebacterium* Mpa homologues have evolved different C termini. In any case, it remains to be determined if *mpa*, *pafA* and *pup* are expressed in this genus or other bacterial genera.

Are there other Pup-like proteins in *M. tuberculosis*? Based on homology searches there do not seem to be additional Pup-like proteins in mycobacteria. However, as discussed earlier, genetic or chemical inhibition of proteasome protease activity results in a severe growth defect under normal growth conditions, a phenotype that is not observed with a pupylation-defective (*pafA*) mutant¹³,²²,³⁰. This suggests that the proteasome can degrade proteins that are targeted in a Pup-independent manner. It is possible that other tags can be processed from larger proteins in a manner akin to ubiquitin. Degradation signals may also be inherent to the substrate, or include other types of modifications, such as phosphorylation.

Other bacterial UBLs?

ThiS and MoaD have a ubiquitin fold, are conserved in most bacteria, and are involved in thiamine and molybdopterin cofactor biosynthesis, respectively⁴⁰. ThiS and MoaD have C-terminal Gly-Gly motifs that undergo a series of chemistries that closely resemble ubiquitin activation; however, these proteins transfer sulphur rather than conjugate to other proteins. Recently eukaryotic ubiquitin-related modifier 1 (Urm1), which is similar to ThiS and MoaD, was shown to have a sulphur carrier function, in addition to its known ability to conjugate to proteins⁴¹⁻⁴³. Although Urm1 is similar to ThiS and MoaD, its function is more related to the *E. coli* tRNA sulphuration pathway⁴⁴,⁴⁵, and not thiamine or molybdopterin metabolism. One of the proteins involved in the *E. coli* tRNA sulphuration is TusA, which is proposed to activate the desulphurase activity of IscS. TusA is thought to accept a sulphur from IscS on one of its two cysteines to form a disulphide bond⁴⁴. Intriguingly, although TusA is not predicted to have UBL properties, it has a C-terminal Gly-Gly motif. Perhaps TusA, like ThiS or MoaD, forms an adenylated intermediate to accept sulphur for the bacterial tRNA sulphuration pathway. More intriguingly, it is possible that these bacterial sulphur-transfer proteins can also covalently conjugate to proteins.

Examination of bacterial genome sequences has revealed that several species encode proteins that are highly similar to ubiquitin. For example, the gut commensal organism *Bacteroides fragilis* encodes a protein that is nearly identical to human ubiquitin⁴⁶. Curiously, it does not have a Gly-Gly motif, so it is not clear if it could have a true ubiquitin-like function. Alternatively, *Bacteroides* spp. might have a conjugation mechanism that does not require Gly-Gly. In addition to *Bacteroides* spp., the gastrointestinal pathogen *Helicobacter pylori* encodes fragments of ubiquitin⁴⁷. It is not known if either *Bacteroides* spp. or *H. pylori* make these proteins. An intriguing possibility is that these commensal bacteria acquired ubiquitin genes from their hosts. It is tempting to propose that they could be used for

NATURE REVIEWS | MICROBIOLOGY

© 2009 Macmillan Publishers Limited. All rights reserved

PROGRESS

a

| Mycobacterium tuberculosis |
| --- |
|  | mpa |
|  | pafD |
|  | pup |
|  | prcB |
|  | prcA |
|  | lppK |
|  | helZ |
|  | pafA |
|  | pafB |
|  | pafC |
|  | tatA |
|  | helY |
|  | pepE |
|  | PPE36 |
|  | PE22 |
|  | pknJ |
|  | tatA |

| Corynebacterium glutamicum |
| --- |
|  |  |

| Nocardia farcinica |
| --- |
|  |  |

| Rhodococcus sp. |
| --- |
|  |  |

| Streptomyces coelicolor |
| --- |
|  |  |

b

| Bifidobacterium longum |
| --- |
|  | pafD |
|  | pafA |
|  | fmt |
|  | priA |
|  | serB |
|  | hup |
|  | purB |

| Rhodopirellula baltica |
| --- |
|  | pafD |
|  | tgt |
|  | rpoA |

Figure 3 | Comparison of the pup regions of bacteria with and without proteasomes. a | Bacteria containing prokaryotic ubiquitin-like protein (pup). In *Mycobacterium tuberculosis*, the following genes are shown: *pup* (red); proteasome core genes (*β*-subunit gene (*prcB*; pink) and *α*-subunit gene (*prcA*; orange)); proteasome accessory factor A (*pafA*; green); and *Mycobacterium* proteasomal ATPase (*mpa*; blue). Homologues in other bacteria are shaded the same colour. The *pafB* and *pafC* genes do not seem to be involved in pupylation or degradation in *M. tuberculosis*. *pafD* (yellow) is 40% identical and 60% similar to *pafA* (*e*-value = 10<sup>−73</sup>) but its role in proteasome function or pupylation has not been established. b | Bacteria that have *paf* homologues but no apparent *pup* or proteasome genes. Data were collected from the Microbial Genome Database (see Further information).

intra-bacterial purposes, or even to subvert normal mammalian cell functions. The idea that bacterial UBLs can be translocated into eukaryotes might not be so radical, as several groups have shown that bacteria inject enzymes with DUB and E3 ligase activity into host cells to disrupt normal cell signalling.

**Prospects**

The question of why it took so long to identify a bacterial UBL is often asked. It is perhaps not surprising that homology searches using ubiquitin or UBLs did not find Pup. Pup is only ‘ubiquitin-like’ in that it has a Gly-Gly motif, attaches to lysines and targets proteins to a proteasome for degradation. Furthermore, it does not have a predicted ubiquitin fold like canonical UBLs. Another reason why it might be difficult to identify bacterial UBLs is that the UBL-modified form of a protein can be transient or rapidly turned over. In *M. tuberculosis*, the pupylated form of a proteasome substrate is by far the least abundant species and is not observable unless affinity purified.

The discovery of Pup has opened up the possibility that small protein modifiers may be present in other bacteria, including those without proteasomes. The function of these modifiers does not have to be limited to proteolysis. Much like eukaryotes, prokaryotes presumably need signals that target proteins for secretion or subcellular sorting. Now that we know bacteria can have UBLs, investigators can look more closely for new post-translational modifications.

**Note added in proof**

A paper describing the discovery of a Pup deamidase, Dop, has recently been published. Dop is the same protein we call PafD. This work also showed that PafA was sufficient to ligate deamidated Pup to the substrate FabD in vitro.

K. Heran Darwin is at the Department of Microbiology, New York University School of Medicine, 550 First Avenue MSB 236, New York, New York 10016, USA.  
e-mail: heran.darwin@med.nyu.edu  
doi:10.1038/nrmicro2148  
Published online 1 June 2009

1. Pickart, C. M. & Cohen, R. E. Proteasomes and their kin: proteases in the machine age. *Nature Rev. Mol. Cell Biol.* **5**, 177–187 (2004).
2. Hershko, A. & Ciechanover, A. The ubiquitin system. *Annu. Rev. Biochem.* **67**, 425–479 (1998).
3. Ventii, K. H. & Wilkinson, K. D. Protein partners of deubiquitinating enzymes. *Biochem. J.* **414**, 161–175 (2008).
4. Wilkinson, K. D. Regulation of ubiquitin-dependent processes by deubiquitinating enzymes. *FASEB J.* **11**, 1245–1256 (1997).
5. Marfany, G. & Denuc, A. To ubiquitinate or to deubiquitinate: it all depends on the partners. *Biochem. Soc. Trans.* **36**, 833–838 (2008).
6. Kerscher, O., Felberbaum, R. & Hochstrasser, M. Modification of proteins by ubiquitin and ubiquitin-like proteins. *Annu. Rev. Cell Dev. Biol.* **22**, 159–180 (2006).
7. Johnson, E. S. Protein modification by SUMO. *Annu. Rev. Biochem.* **73**, 355–382 (2004).
8. Chau, V. *et al.* A multiubiquitin chain is confined to specific lysine in a targeted short-lived protein. *Science* **243**, 1576–1583 (1989).
9. Hough, R. & Rechsteiner, M. Ubiquitin-lysozyme conjugates. Purification and susceptibility to proteolysis. *J. Biol. Chem.* **261**, 2391–2399 (1986).
10. Chen, Z. J. Ubiquitin signalling in the NF-*κ*B pathway. *Nature Cell Biol.* **7**, 758–765 (2005).
11. Haglund, K. & Dikic, I. Ubiquitylation and cell signaling. *EMBO J.* **24**, 3353–3359 (2005).
12. De Mot, R., Nagy, I., Walz, J. & Baumeister, W. Proteasomes and other self-compartmentalizing proteases in prokaryotes. *Trends Microbiol.* **7**, 88–92 (1999).
13. Pearce, M. J., Mintseris, J., Ferreyra, J., Gygi, S. P. & Darwin, K. H. Ubiquitin-like protein involved in the proteasome pathway of *Mycobacterium tuberculosis*. *Science* **322**, 1104–1107 (2008).
14. Butler, S. M., Festa, R. F., Pearce, M. J. & Darwin, K. H. Self-compartmentalized bacterial proteases and pathogenesis. *Mol. Microbiol.* **60**, 553–562 (2006).
15. Zwickl, P., Baumeister, W. & Goldberg, A. L. in *Proteasomes: The World of Regulatory Proteolysis* (eds Hilt, W. & Wolf, D. H.) 8–18 (Eurekah.com, Georgetown, Texas, 2000).
16. Lin, G. *et al.* *Mycobacterium tuberculosis prcBA* genes encode a gated proteasome. *Mol. Microbiol.* **59**, 1405–1416 (2006).
17. Hu, G. *et al.* Structure of the *Mycobacterium tuberculosis* proteasome and mechanism of inhibition by a peptidyl boronate. *Mol. Microbiol.* **59**, 1417–1428 (2006).
18. Löwe, J. *et al.* Crystal structure of the 20S proteasome from the archaeon *T. acidophilum* at 3.4 Å resolution. *Science* **268**, 533–539 (1995).
19. Nagy, I., Tamura, T., Vanderleyden, J., Baumeister, W. & De Mot, R. The 20S proteasome of *Streptomyces coelicolor*. *J. Bacteriol.* **180**, 5448–5453 (1998).
20. Kwon, Y. D., Nagy, I., Adams, P. D., Baumeister, W. & Jap, B. K. Crystal structures of the *Rhodococcus* proteasome with and without its pro-peptides: implications for the role of the pro-peptide in proteasome assembly. *J. Mol. Biol.* **335**, 233–245 (2004).
21. Tamura, T. *et al.* The first characterization of a eubacterial proteasome: the 20S complex of *Rhodococcus*. *Curr. Biol.* **5**, 766–774 (1995).
22. Darwin, K. H., Ehrt, S., Weich, N., Gutierrez-Ramos, J.-C. & Nathan, C. F. The proteasome of *Mycobacterium tuberculosis* is required for resistance to nitric oxide. *Science* **302**, 1963–1966 (2003).
23. Pouch, M.-N., Cournoyer, B. & Baumeister, W. Characterization of the 20S proteasome from the actinomycete *Frankia*. *Mol. Microbiol.* **35**, 368–377 (2000).
24. Knipfer, N. & Shrader, T. E. Inactivation of the 20S proteasome in *Mycobacterium smegmatis*. *Mol. Microbiol.* **25**, 375–383 (1997).
25. Smith, D. M. *et al.* ATP binding to PAN or the 26S ATPases causes association with the 20S proteasome, gate opening, and translocation of unfolded proteins. *Mol. Cell* **20**, 687–698 (2005).
26. MacMicking, J. D. *et al.* Identification of nitric oxide synthase as a protective locus against tuberculosis. *Proc. Natl Acad. Sci. USA* **94**, 5243–5248 (1997).
27. Shiloh, M. U. & Nathan, C. F. Reactive nitrogen intermediates and the pathogenesis of *Salmonella* and mycobacteria. *Curr. Opin. Microbiol.* **3**, 35–42 (2000).
28. Nagy, I., Geert, S., Vanderleyden, J. & De Mot, R. Further sequence analysis of the DNA regions with the *Rhodococcus* 20S proteasome structural genes reveals extensive homology with *Mycobacterium leprae*. *DNA Seq.* **7**, 225–228 (1997).

29. Festa, R. A., Pearce, M. J. & Darwin, K. H. Characterization of the proteasome accessory factor (*paf*) operon in *Mycobacterium tuberculosis*. *J. Bacteriol.* **189**, 3044–3050 (2007).

30. Gandotra, S., Schnappinger, D., Monteleone, M., Hillen, W. & Ehrt, S. *In vivo* gene silencing identifies the *Mycobacterium tuberculosis* proteasome as essential for the bacteria to persist in mice. *Nature Med.* **13**, 1515–1520 (2007).

31. Pearce, M. J. *et al.* Identification of substrates of the *Mycobacterium tuberculosis* proteasome. *EMBO J.* **25**, 5423–5432 (2006).

32. Singh, A., Mai, D., Kumar, A. & Steyn, A. J. Dissecting virulence pathways of *Mycobacterium tuberculosis* through protein–protein association. *Proc. Natl Acad. Sci USA* **103**, 11346–11351 (2006).

33. Burns, K. E., Liu, W. T., Boshoff, H. I., Dorrestein, P. C. & Barry, C. E. 3rd. Proteasomal protein degradation in mycobacteria is dependent upon a prokaryotic ubiquitin-like protein. *J. Biol. Chem.* **284**, 3069–3075 (2008).

34. Soding, J., Biegert, A. & Lupas, A. N. The HHpred interactive server for protein homology detection and structure prediction. *Nucleic Acids Res.* **33**, W244–W248 (2005).

35. Lehmann, C. *et al.* YbdK is a carboxylate-amine ligase with a γ-glutamyl:cysteine ligase activity: crystal structure and enzymatic assays. *Proteins* **56**, 376–383 (2004).

36. Iyer, L. M., Burroughs, A. M. & Aravind, L. Unraveling the biochemistry and provenance of pupylation: a prokaryotic analog of ubiquitination. *Biol. Direct* **3**, 45 (2008).

37. Altschul, S. F. *et al.* Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. *Nucleic Acids Res.* **25**, 3389–3402 (1997).

38. Zhang, X. *et al.* The N-terminal coiled coil of the *Rhodococcus erythropolis* ARC AAA ATPase is neither

39. necessary for oligomerization nor nucleotide hydrolysis. *J. Struct. Biol.* **146**, 155–165 (2004).

40. Darwin, K. H., Lin, G., Chen, Z., Li, H. & Nathan, C. Characterization of a *Mycobacterium tuberculosis* proteasomal ATPase homologue. *Mol. Microbiol.* **55**, 561–571 (2005).

41. Kessler, D. Enzymatic activation of sulfur for incorporation into biomolecules in prokaryotes. *FEMS Microbiol. Rev.* **30**, 825–840 (2006).

42. Pedrioli, P. G., Leidel, S. & Hofmann, K. Urm1 at the crossroad of modifications. ‘Protein Modifications: Beyond the Usual Suspects’ Review Series. *EMBO Rep.* **9**, 1196–1202 (2008).

43. Leidel, S. *et al.* Ubiquitin-related modifier Urm1 acts as a sulphur carrier in thiolation of eukaryotic transfer RNA. *Nature* **458**, 228–232 (2009).

44. Schlieker, C. D., Van der Veen, A. G., Damon, J. R., Spooner, E. & Ploegh, H. L. A functional proteomics approach links the ubiquitin-related modifier Urm1 to a tRNA modification pathway. *Proc. Natl Acad. Sci. USA* **105**, 18255–18260 (2008).

45. Ikeuchi, Y., Shigi, N., Kato, J., Nishimura, A. & Suzuki, T. Mechanistic insights into sulfur relay by multiple sulfur mediators involved in thioridine biosynthesis at tRNA wobble positions. *Mol. Cell* **21**, 97–108 (2006).

46. Numata, T., Ikeuchi, Y., Fukai, S., Suzuki, T. & Nureki, O. Snapshots of tRNA sulphuration via an adenylated intermediate. *Nature* **442**, 419–424 (2006).

47. Cerdeno-Tarraga, A. M. *et al.* Extensive DNA inversions in the *B. fragilis* genome control variable gene expression. *Science* **307**, 1463–1465 (2005).

48. Giannakis, M., Chen, S. L., Karam, S. M., Engstrand, L. & Gordon, J. I. *Helicobacter pylori* evolution during progression from chronic atrophic gastritis to gastric cancer and its impact on gastric stem cells. *Proc. Natl Acad. Sci. USA* **105**, 4358–4363 (2008).

49. Rytkonen, A. & Holden, D. W. Bacterial interference of ubiquitination and deubiquitination. *Cell Host Microbe* **1**, 13–22 (2007).

50. Ozkaynak, E., Finley, D., Solomon, M. J. & Varshovsky, A. The yeast ubiquitin genes: a family of natural gene fusions. *EMBO J.* **6**, 1429–1439 (1987).

51. Jin, J., Li, X., Gygi, S. P. & Harper, J. W. Dual E1 activation systems for ubiquitin differentially regulate E2 enzyme charging. *Nature* **447**, 1135–1138 (2007).

52. Chiu, Y. H., Sun, O. & Chen, Z. J. E1–L2 activates both ubiquitin and FAT10. *Mol. Cell* **27**, 1014–1023 (2007).

53. Striebel, F. *et al.* Bacterial ubiquitin-like modifier Pup is deamidated and conjugated to substrates by distinct but homologous enzymes. *Nature Struct. Mol. Biol.* 17 May 2009 (doi:10.1038/nsmb.1597).

**Acknowledgements**

I am grateful to A. Darwin, I. Mohr and H. Ovaa for critical review of this manuscript. I thank the anonymous referees for excellent suggestions and important corrections. I am supported by National Institutes of Health (NIH) grants AI065437 and HL092774 and a Center For AIDS Research Pilot Project grant (NIH S P30 A1027742-17).

**DATABASES**

Entrez Genome Project: <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=genomeprj>

*Bacteroides fragilis* | *Corynebacterium glutamicum* | *Escherichia coli* | *Helicobacter pylori* | *Methanococcus jannaschii* | *Mycobacterium smegmatis* | *Mycobacterium tuberculosis* | *Thermoplasma acidophilum*

UniProtKB: <http://www.uniprot.org>

FabD | Ino1 | MoaD | Mpa | NEDD8 | NEMO | PafA | PanB | Pup | SodA | ThiS | TRAF6

**FURTHER INFORMATION**

Microbial Genome Database: <http://mbgd.genome.ad.jp/>

ALL LINKS ARE ACTIVE IN THE ONLINE PDF
