
REVIEWS

---

**Dynamics and diversity in autophagy mechanisms: lessons from yeast**

**Hitoshi Nakatogawa**, **Kuninori Suzuki**, **Yoshiaki Kamada** and **Yoshinori Ohsumi**

Abstract | Autophagy is a fundamental function of eukaryotic cells and is well conserved from yeast to humans. The most remarkable feature of autophagy is the synthesis of double membrane-bound compartments that sequester materials to be degraded in lytic compartments, a process that seems to be mechanistically distinct from conventional membrane traffic. The discovery of autophagy in yeast and the genetic tractability of this organism have allowed us to identify genes that are responsible for this process, which has led to the explosive growth of this research field seen today. Analyses of autophagy-related (Atg) proteins have unveiled dynamic and diverse aspects of mechanisms that underlie membrane formation during autophagy.

---

**Macroautophagy**  
The sequestration of cytosolic components in autophagosomes and their subsequent degradation when autophagosomes fuse with lysosomes.

**Autophagosome**  
A double membrane-bound vesicle that is formed during autophagy. The vesicle sequesters materials to be degraded and delivers them to the lysosome in mammals or the vacuole in yeasts and plants.

---

Autophagy (or macroautophagy) was defined in mammalian cells more than 50 years ago as a system that delivers the cytoplasmic components and organelles of a cell to lysosomes for degradation ${}^{1-3}$. The most crucial event in autophagy is the sequestration of these materials by forming a new compartment. Induction of autophagy leads to *de novo* formation of cup-shaped membranes in the cytoplasm called isolation membranes (or phagophores), which expand while becoming spherical and eventually seal to become double membrane-bound structures called autophagosomes (FIG. 1). As a natural consequence of this process, a portion of the cytoplasm is confined in the autophagosome. The outer membrane of the autophagosome is subsequently fused with the lysosomal membrane to allow the degradation of the contents together with the inner membrane. Therefore, in contrast to the ubiquitin–26S proteasome system, autophagy mediates primarily non-selective and bulk degradation of many intracellular proteins in one swoop.

Whereas autophagy is drastically induced in response to a shortage of nutrients, it is also regulated by various physiological signals, such as hormones, growth factors and pathogen infection, and occurs constitutively at a basal level ${}^{2,3}$. Cytoplasmic components are, in principle, nonspecifically engulfed by autophagosomes under starvation conditions. By contrast, recent studies have revealed that autophagy can also be selective in other situations, in which specific ‘cargoes’, including disease-related inclusions, superfluous or damaged organelles, and even invasive bacteria, are enwrapped by autophagosome-like membranes. Autophagy is now used as a collective term for these related phenomena. Reflecting these diversities, in addition to its essential role for cell survival under nutrient-deprived conditions, autophagy is involved in a wide range of physiological and pathological processes in eukaryotic organisms ${}^{3,4}$. However, the molecular mechanisms that underlie autophagosomal membrane formation and the selective incorporation of cargoes into these membranes remain largely unknown.

Autophagy was beyond the limits of molecular dissection for a long time because it could only be detected by electron microscopy and biochemical analysis of lysosomes was technically difficult. The budding yeast *Saccharomyces cerevisiae* therefore proved to be an ideal organism to use to gain insights into genes that are essential for autophagy. Yeast autophagy was discovered by observing vacuolar proteinase-deficient cells with a light microscope ${}^{5}$. Within a few hours of shifting the cells to nutrient starvation media, vacuoles were filled with vesicles containing cytoplasmic components, and these vesicles were termed autophagic bodies. It was subsequently shown that the autophagic body is derived from the autophagosome by its fusion with the vacuolar membrane ${}^{6,7}$. This process proved to be essentially the same as macroautophagy, which had been described in mammalian and plant cells. Detailed analyses of autophagosomal membranes by electron microscopy showed that these membranes seem thin compared with other organelle membranes ${}^{6}$. Moreover, freeze-fracture electron microscopy indicated that there are asymmetric compositions between the outer and inner membranes of the autophagosome: whereas

---

*Integrated Research Institute, Tokyo Institute of Technology, Yokohama 226-8503, Japan.*

*PRESTO, Japan Science and Technology Agency, Saitama 332-0012, Japan.*

*Department of Cell Biology, National Institute for Basic Biology, Okazaki 444-8585, Japan.*

Correspondence to Y.O.  
e-mail:  
*yohsumi@iri.titech.ac.jp*

doi:10.1038/nrm2708  
Published online xx June 2009

---

© 2009 Macmillan Publishers Limited. All rights reserved

This occurs through membrane dynamics that turned out to be similar to that of autophagy⁹. The group also isolated mutants of Cvt genes that are defective in this pathway¹⁰. Other genetic approaches identified genes involved in autophagy as well as pexophagy, an autophagic degradation pathway for peroxisomes in yeast¹¹⁻¹⁵.

These independent screens of mutants produced different gene names (APG, AUT, CVT, GSA, PAG, PAZ and PDD) that have since been unified to ATG (autophagy-related) to avoid confusion¹⁶. Although 31 ATG genes have been reported so far, 15 genes are commonly required for all of the above pathways (starvation-induced autophagy, the Cvt pathway and pexophagy). These genes, hereafter referred to as ‘core’ ATG genes, encode the fundamental machinery for the biogenesis of autophagy-related membranes (FIG. 2). Characterization of these 15 ATG gene products revealed that they consist of five subgroups — the Atg1 kinase and its regulators¹⁷, the autophagy-specific phosphatidylinositol (PtdIns) 3-kinase complex¹⁸, the Atg12 conjugation system¹⁹, the Atg8 conjugation system²⁰ and a subgroup of functionally unknown proteins that interact with each other²¹⁻²⁴ (TABLE 1). In addition to these core components, some Atg proteins are specifically required for each pathway (FIG. 2). In this Review, we describe the present knowledge on the mechanisms of autophagy that are obtained from yeast studies, especially focusing on starvation-induced autophagy, which is the most fundamental and evolutionarily conserved mode of autophagy (see below). More specifically, we first describe the identification of the pre-autophagosomal structure (PAS), an assembly of Atg proteins, then describe how Atg subgroups function to build the autophagosome and, finally, dynamic aspects of the PAS revealed by recent studies.

### Identification of the PAS

Immunoelectron microscopy of Atg8, one of the core Atg proteins, revealed that this protein localizes both to isolation membranes and to autophagosomes; thus, Atg8 serves as a marker for membrane dynamics during autophagy (see below)²⁵. However, fluorescence microscopy showed that one dot of fluorescently tagged Atg8 is usually observed in close proximity to the vacuole in each cell²⁶ (FIG. 3a). This dot was seen even in several mutants of ATG genes, suggesting that it represents neither an isolation membrane nor an autophagosome. Time-lapse microscopy of Atg8 fused with green fluorescent protein (GFP) in a temperature-sensitive ATG1 strain suggested that autophagosomes are generated at or around this dot²⁶, and, indeed, most Atg proteins are at least partly colocalized here²⁶. From these results, it is likely that the dot represents the assembly of the Atg proteins responsible for autophagosome formation, and thus the assembly was termed the PAS.

Further analyses, in which the localization of each Atg protein to the PAS under autophagy-inducing conditions was systematically examined in cells that lacked one of the other Atg proteins, showed that the Atg proteins organize the PAS according to hierarchical relationships among the subgroups²⁷,²⁸ (FIG. 3b). If one

---

**Figure 1 | Autophagy in yeast.** In starvation-induced (non-selective) autophagy, the isolation membrane mainly non-selectively engulfs cytosolic constituents and organelles to form the autophagosome. The inner membrane-bound structure of the autophagosome (the autophagic body) is released into the vacuolar lumen following fusion of the outer membrane with the vacuolar membrane, and is disintegrated to allow degradation of the contents by resident hydrolases. In selective autophagy, specific cargoes (protein complexes or organelles) are enveloped by membrane vesicles that are similar to autophagosomes, and are delivered to the vacuole for degradation. Although the Cvt (cytoplasm-to-vacuole targeting) pathway mediates the biosynthetic transport of vacuolar enzymes, its membrane dynamics and mechanism are almost the same as those of selective autophagy (see the main text).

---

**Ubiquitin–26S proteasome system**

The system that degrades selected proteins that are first marked with ubiquitin chains and then degraded by the multi-catalytic proteinase complex, the proteasome.

**Autophagic body**

The inner membrane-bound structure of the autophagosome that is released into the vacuolar lumen by fusion of the autophagosomal outer membrane with the vacuolar membrane.

REVIEW S

![Diagram](#)

Figure 2 | Classification of Atg proteins. Autophagy-related (Atg) proteins that are commonly required for three autophagy-related pathways — starvation-induced autophagy, the Cvt (cytoplasm-to-vacuole targeting) pathway and pexophagy (an autophagic degradation pathway for peroxisomes in yeast) — are classified as the core machinery for membrane formation. Proteins that are specific for each pathway, which include conductor proteins (see the main text), are also shown. It should be noted that Atg11 is involved in both the Cvt pathway and pexophagy.

Atg protein placed upstream in the hierarchical model is genetically deleted, the PAS localization of the downstream Atg proteins is significantly impaired. By contrast, in some cases, upstream Atg proteins accumulate at the PAS in the absence of the downstream protein (see below). Among Atg proteins that belong to the same subgroup, not only hierarchical relationships but also interdependent relationships are also seen. These observations suggest that the Atg proteins function in a coordinate manner to generate the autophagosome at the PAS while interacting with each other both in and among the subgroups. It seems that the hierarchy in the localization of Atg proteins to the PAS represents their order of action in autophagosome formation. Below, we overview the characteristics and functions of each Atg subgroup.

**Atg1 kinase and its regulators**

Autophagy in yeast is mainly a response to nutrient starvation⁵. Target of rapamycin (Tor), a master regulator of nutrient signalling, is involved in the induction of autophagy because the Tor inhibitor rapamycin mimics starvation and induces autophagy, even under nutrient-rich conditions²⁹. Although the Tor protein forms two distinct complexes, Tor complex 1 (TORC1) and TORC2 (REF. 30), only TORC1 function is sensitive to rapamycin, indicating that TORC1 is responsible for controlling autophagy. The addition of cyclic AMP suppresses induction of autophagy by nutrient starvation or rapamycin treatment, suggesting that cAMP-dependent protein kinase, PKA, also has an inhibitory role in autophagy²⁹,³¹,³².

The Atg1 kinase and its regulators — Atg13, Atg17, Atg29 and Atg31 — collaboratively function in the initial step of autophagosome formation, downstream of TORC1 (FIG. 4; TABLE 1). These proteins comprise the most upstream Atg subfamily in the hierarchy of the localization of Atg proteins to the PAS (FIG. 3). Atg1 is a Ser/Thr protein kinase, the activity of which is essential for autophagy³³ and is largely enhanced following nutrient starvation or the addition of rapamycin¹⁷. This regulation involves Atg13 (REF. 17). Although Atg13 is phosphorylated in a TORC1-dependent manner under nutrient-replete conditions, it is immediately dephosphorylated in response to starvation or rapamycin treatment¹⁷,³⁴ (FIG. 4). Dephosphorylated Atg13 associates with Atg1 and somehow leads to upregulation of the kinase activity of Atg1. As dephosphorylation of Atg13 occurs normally in the presence of any non-essential phosphatase mutant (T. Funakoshi and Y.O., unpublished observations), multiple phosphatases might dephosphorylate Atg13. Phosphorylation of one or more factors by Atg1 is expected to trigger a downstream event in autophagosome formation. Although several Atg proteins are phosphorylated in an Atg1-dependent manner both *in vivo* and *in vitro*, the physiological significance of the phosphorylation of these proteins — and thus an authentic substrate or substrates of Atg1 — still remain elusive (Y.K. and Y.O., unpublished observations).

Whereas Atg1 and Atg13 are among the core components of autophagosome formation, Atg17, Atg29 and Atg31 are specifically required for starvation-induced autophagy (FIG. 2) and form a ternary complex¹⁷,³⁵–³⁸ (FIG. 4) (Y. Kabeya and Y.O., unpublished observations). Although this ternary complex seems to be formed constitutively, it associates with the Atg1–Atg13 complex in response to nutrient starvation, which is important for the activation of Atg1 (REFS 35,39). This association is also a prerequisite for the recruitment of other core Atg proteins to the PAS in the absence of Atg11, suggesting that this subgroup functions as a trigger for autophagosome formation (see below)³⁵,³⁸–⁴⁰. The kinase activity of Atg1 is dispensable for both the formation of a complex of these five Atg proteins and the recruitment of a number of Atg proteins to the PAS³⁸,³⁹. In yeast that have the kinase-dead allele of ATG1, however, some Atg proteins abnormally accumulate at the PAS, and others, such as Atg2, become absent from the PAS (REFS 38,39,41). This suggests that the kinase activity of Atg1 is involved in the dynamics of the Atg proteins at the PAS, probably through the phosphorylation of one or more Atg proteins.

Although it was proposed that dephosphorylation of Atg13 is one of the initial events in autophagy, whether Atg13 is a direct target of TORC1 or whether dephosphorylation of Atg13 is sufficient for the induction of autophagy has remained unanswered⁴². These questions have recently been addressed to understand the mechanism by which TORC1 signalling regulates autophagy (Y.K. and Y.O., unpublished observations). It was shown that Atg13 is directly phosphorylated by TORC1 *in vitro* at multiple Ser residues. Expression of an unphosphorylatable Atg13 mutant can at least partially induce autophagy in non-starved cells, suggesting that dephosphorylation of Atg13 is sufficient for autophagy induction. As mammalian homologues of the Atg1 complex have been reported, the TORC1–Atg1 signalling module is thought to be conserved across most eukaryotes to regulate autophagy⁴³–⁴⁸.

Table 1 | Subgroups of Atg proteins required for starvation-induced autophagy

| Subgroup components | Known or putative function | References |
|---------------------|---------------------------|------------|
| **Atg1 kinase and its regulators** |  |  |
| Atg1 | Ser/Thr kinase | 17,33,38,39,57 |
| Atg13 | TORC1 substrate involved in the regulation of Atg1 activity | 17,34,38,39,57 |
| Atg17 | Forms a starvation-induced autophagy-specific complex with Atg29 and Atg31, which further associates with Atg1 and Atg13 | 17,35,38,39–41 |
| Atg29 | Component of the Atg17–Atg29–Atg31 complex | 36,38 |
| Atg31 | Component of the Atg17–Atg29–Atg31 complex | 37,38 |
| **Ptdlns 3-kinase complex** |  |  |
| Vps34 | Ptdlns 3-kinase | 18,52 |
| Vps15 | Ser/Thr kinase required for Vps34 activity | 18,52 |
| Vps30 (also known as Atg6) | Component of unknown function | 18,52 |
| Atg14 | Recruits the Ptdlns 3-kinase complex to the PAS | 18,52 |
| **Atg12 conjugation system** |  |  |
| Atg12 | Ubiquitin-like protein that is conjugated to Atg5 | 19,72,73,76,77 |
| Atg5 | A target of Atg12 and interacts with Atg16 | 19,72,73,76,77 |
| Atg7 | Common E1 enzyme for Atg12–Atg5 and Atg8–PE formation | 19 |
| Atg10 | Specific E2 enzyme for Atg12–Atg5 formation | 19 |
| Atg16 | Required for the PAS localization of Atg12–Atg5 | 72,73 |
| **Atg8 conjugation system** |  |  |
| Atg8 | Ubiquitin-like protein conjugated to PE | 20,25,62–64,68,69 |
| Atg3 | Specific E2 enzyme for Atg8–PE formation | 20,63,68 |
| Atg4 | Removes the carboxy-terminal Arg and conjugated PE from Atg8 | 62–64 |
| Atg7 | Common E1 enzyme for Atg12–Atg5 and Atg8–PE formation | 20,68 |
| **Atg2–Atg18 complex** and **Atg9** |  |  |
| Atg2 | Forms a complex with Atg18, which is involved in Atg9 dynamics at the PAS | 21,22,50 |
| Atg18 | Binds to Ptdlns3P | 50,54,55 |
| Atg9 | Integral membrane protein of unknown function | 23,41,57,60 |

*Trigger and regulate PAS assembly of other proteins. †Produces Ptdlns3P at the PAS. §Stimulates and determines the location of Atg8 lipidation. ‡Controls membrane tethering and hemifusion. ¶The function of this complex is unknown. PAS, pre-autophagosomal structure; PE, phosphatidylethanolamine; Ptdlns, phosphatidylinositol; TORC1, target of rapamycin complex 1; Vps, vacuolar protein sorting.

---

### The Ptdlns 3-kinase complex

One Atg subgroup forms a complex with the PtdIns 3-kinase that phosphorylates the D3 position of the inositol ring in PtdIns to produce PtdIns3P (TABLE 1). In addition to this complex, we describe another subgroup that comprises Atg18, a possible effector for PtdIns3P, Atg2 and Atg9.

The activity of PtdIns 3-kinase is essential for autophagy. Vps34 is the sole PtdIns 3-kinase in S. cerevisiae and forms two distinct complexes, I and II, that have essential roles in autophagy and the vacuolar protein sorting (Vps) pathway, respectively. Complex I is composed of Vps34, Vps15, Vps30 (also known as Atg6) and Atg14, whereas complex II contains Vps38 instead of Atg14 (FIG. 5). The specific presence of Atg14 or Vps38 directs the localization of complexes I and II to the PAS and the endosomal membrane, respectively, in addition to the vacuolar membrane (FIG. 5).

---

### Vacuolar protein sorting

A pathway that mediates the selective transport of a subset of proteins from the late Golgi compartment to the vacuole through the endosome.

REVIEW S

with this, it has recently been shown that Atg9 physically interacts with Atg17, a component of the Atgl subgroup [41]. It has also been shown that Atg9 self-associates, which is important for its localization to the PAS [60].

## Two ubiquitin-like conjugation systems

There are two protein conjugation systems among the Atg subgroups, each composed of two ubiquitin-like proteins (Atg8 and Atg12) and three enzymes (Atg3, Atg7 and Atg10) that are required for their conjugation reactions (FIG. 6a,b; TABLE 1). Atg12 forms a conjugate with Atg5 (REF. 19), whereas Atg8 is conjugated to phosphatidylethanolamine (PE) [20], a major component of various biological membranes. Both conjugates localize to autophagy-related membranes [25,61] (FIG. 6c), suggesting their direct involvement in the biogenesis of these membranes. Consistent with this notion, these conjugates are positioned downstream in the hierarchy of the localization of Atg proteins to the PAS (FIG. 3). Recent *in vitro* studies have substantially advanced our understanding of the functions of these ubiquitin-like protein conjugates.

### Conjugation of Atg8

Atg8 is synthesized as a precursor with an additional sequence in its carboxyl terminus (a single Arg residue in S. cerevisiae Atg8). This is immediately cleaved by the Cys protease Atg4 to expose the Gly residue that is essential for subsequent reactions [62,63] (FIG. 6a). Formation of the Atg8–PE conjugate is mediated by Atg7 and Atg3, which are the E1 and E2 enzymes in the ubiquitylation reaction, respectively [20]. The carboxyl group of the exposed Gly of Atg8 is activated by Atg7, in a reaction that requires ATP, to form a thioester intermediate with the active Cys residue of Atg7. The Gly residue of Atg8 is then transferred to the active Cys residue of Atg3 and eventually forms an amide bond with the amino group in PE (FIG. 6a). Atg8, probably in this lipidated form, localizes to the isolation membrane and the autophagosome [25].

---

**Figure 3 | The PAS as the assembly of Atg proteins.**

a | Fluorescence microscopy images of cells that express both cyan fluorescent protein (CFP)-tagged autophagy-related 8 (Atg8) and yellow fluorescent protein (YFP)-tagged Atg5, as well as the merged image. The arrow indicates the pre-autophagosomal structure (PAS). The vacuole is indicated by the letter V. Scale bar, 5 μm.

b | A hierarchical model of the localization of Atg proteins to the PAS in starvation-induced autophagy. If an upstream Atg protein is genetically removed, the localization of downstream Atg proteins is lost. The Atg1–Atg13–Atg17–Atg29–Atg31 complex, which is formed in response to nutrient starvation, has an essential role in PAS organization in starvation-induced autophagy. This complex is involved not only in the recruitment of other Atg proteins to the PAS but also in their dissociation from the PAS. Mechanisms by which upstream proteins recruit downstream proteins are still largely unknown. It is suggested that phosphatidylinositol 3-phosphate (PtdIns3P) produced by Ptdlns 3-kinase complex I at the PAS recruits the PtdIns3P-binding protein Atg18 and its associated protein Atg2. Vps, vacuolar protein sorting. Figure part a is modified, with permission, from *EMBO J.* REF. 26 © (2001) Macmillan Publishers Ltd. All rights reserved.

Although the precise location in which complex I produces PtdIns3P is still unknown, recent studies show that PtdIns3P is enriched on isolation membranes and autophagosomal membranes and is eventually transported to the vacuole [56].

Of the core Atg proteins, Atg9 is the sole integral membrane protein. Thus, this protein has been extensively analysed to obtain insights into a membrane source of the autophagosome [57–59]. In addition to its localization to the PAS [27], Atg9–GFP is also observed as small dots that move in the cytoplasm [57]. Whereas disruption of actin filaments or microtubules does not disturb this movement, energy depletion by the addition of sodium azide does [41]. However, it is still unclear whether these cytosolic dots of Atg9 are relevant to autophagosome formation. Atg9 accumulates at the PAS at the non-permissive temperature in an ATG1 temperative-sensitive strain [57]. It was therefore proposed that Atg9 shuttles between the PAS and the cytoplasmic pool during autophagosome formation. Similarly, deletion of ATG2 and ATG18 also causes the accumulation of Atg9 at the PAS [27]. The Atg2–Atg18 complex and the function of Atg1 might be involved in the dynamics of Atg9 at the PAS (see below). In the hierarchical model of the localization of Atg proteins to the PAS (FIG. 3), Atg9 is located just after the Atg1 subgroup. Consistent

---

**E1**  
An enzyme that activates ubiquitin and ubiquitin-like proteins using ATP and transfers them to E2 enzymes.

**E2**  
An enzyme that receives ubiquitin and ubiquitin-like proteins from E1 enzymes and conjugates them to target molecules.

---

**Figure 4 | The Atg1 subfamily.** When target of rapamycin complex 1 (TORC1) is inactivated following nutrient depletion or rapamycin treatment, autophagy-related 13 (Atg13) is dephosphorylated. This allows the association of Atg1 subfamily proteins with Atg13, followed by the upregulation of the Atg1 kinase activity and recruitment of other core Atg proteins to the pre-autophagosomal structure (PAS) to initiate autophagosome formation. These events are immediately reversed on the addition of nutrients.

Complex I
![Diagram of Complex I](#)

Complex II
![Diagram of Complex II](#)

Figure 5 | Two PI3K complexes. Phosphatidylinositol (PtdIns) 3-kinase complexes I and II are composed of common subunits — the PtdIns 3-kinase vacuolar protein sorting 34 (Vps34), Vps30 (also known as autophagy-related 6 (Atg6)) and Vps15 — and specialized subunits, Atg14 and Vps38, respectively. Complex I localizes to the pre-autophagosomal structure (PAS) and the vacuole, its downstream factors include Atg2–Atg18 and it functions in autophagosome formation. By contrast, complex II resides in endosomes and the vacuole, its downstream factors include the Fab1–Fig4–Vac14 complex, Vac7 and Atg18, and it has a role in the regulation of vacuole morphology.

under autophagy-inducing conditions, even in cells that are deficient for autophagosome formation²⁷. Atg8–PE might accumulate on precursors of autophagosomal membranes in these cells. Identification and characterization of these structures containing Atg8–PE, including the elucidation of their components, morphology and formation process, will provide us with crucial information on molecular mechanisms of, and a source of lipid supply for, autophagosome formation.

Conjugation of Atg12. Similar to the Atg8–PE system, the conjugation reaction of Atg12 is catalysed by the common E1 enzyme Atg7 and the specific E2 enzyme Atg10. The C-terminal Gly residue of Atg12 forms an isopeptide bond with the specific Lys residue of Atg5 (REF. 19) (FIG. 6b). It seems that neither a processing enzyme nor a deconjugation enzyme exists in the Atg12–Atg5 system, and that formation of this conjugate occurs constitutively. The Atg12–Atg5 conjugate further interacts with Atg16 and forms a complex of ~350 kDa (~800 kDa in mammals) by virtue of the oligomerization ability of Atg16 (REFS 72–74) (FIG. 6b). Immunoelectron microscopic analyses of mammalian cells showed that while LC3 (a mammalian Atg8 homologue) is present on both surfaces of the isolation membrane⁶⁵, the ATG12–ATG5–ATG16L (a mammalian Atg16 homologue) complex predominantly localizes on the outer surface of the membrane⁶¹,⁷⁴ (FIG. 6c). In addition, fluorescence microscopy showed that GFP-labelled ATG5 dissociates from the membrane immediately before or after completion of the autophagosome⁶¹. Although these observations seem to imply that the ATG12–ATG5–ATG16 complex functions as a coat protein, as observed in secretory vesicle formation, it has recently been estimated that the number of complex molecules that participate in autophagosome formation is too low to assemble a coat that surrounds the membrane⁷⁵.

Crosstalk between the conjugation systems. The involvement of the Atg12–Atg5–Atg16 complex in the Atg8–PE system has been implied genetically, as mutations that abolish the complex significantly decrease Atg8–PE production²⁶,⁷⁶. It has also been shown that purified Atg12–Atg5 conjugates drastically stimulate the formation of Atg8–PE in vitro⁷⁷,⁷⁸. Atg12–Atg5 directly interacts with the E2 enzyme Atg3 and enhances its activity (FIG. 6a,b). Thus, Atg12–Atg5 exerts an E3-like function in the lipidation of Atg8. Atg8 can be conjugated to phosphatidylserine (PS) as efficiently as to PE in the in vitro reaction, although PE was identified as the sole target of Atg8 in vivo⁶⁸,⁷⁷,⁷⁹. Unlike E3 enzymes in the ubiquitin system, Atg12–Atg5 is not involved in determining substrate specificity in lipidation of Atg8 (REF. 77). Instead, it has been proposed that Atg3 itself has an ability to discriminate PE from PS under physiological conditions⁸⁰.

Atg16 is dispensable for the E3-like function of Atg12–Atg5 in vitro, in spite of its requirement for Atg8–PE formation in vivo²⁶. This apparent contradiction could be explained by considering the spatial regulation of the Atg12–Atg5–Atg16 complex. In yeast, Atg16 is required for the localization of Atg12–Atg5 to the PAS²⁷.

Deconjugation enzyme  
An enzyme that cleaves the isopeptide bond (the amide bond between autophagy-related protein 8 (Atg8) and phosphatidylethanolamine) formed in ubiquitin and ubiquitin-like protein conjugates.

Hemifusion  
Fusion between outer leaflets of membranes while inner leaflets remain intact. This is regarded as a common intermediate state in biological membrane fusion events.

E3  
An enzyme that stimulates the conjugation reaction by E2 enzymes and is also involved in the selection of target molecules.

In addition, forced localization of ATG16L to the plasma membrane in mammalian cells causes lipidation of LC3 on that membrane<sup>81</sup>. These results indicate that ATG16 is involved in specification of the site of ATG8 lipidation. The localization of the Atg12–Atg5–Atg16 complex to the PAS might imply that Atg8–PE is produced at the PAS. Lipidation of Atg8 on the isolation membrane is also conceivable. As mentioned above, Atg8–PE, however, accumulates even in mutant cells in which the isolation membrane is not formed. Therefore, it is possible that Atg8–PE is formed elsewhere and then somehow transferred to the isolation membrane, or that lipidation of Atg8 occurs at both sites. Thus, this long-standing question requires further investigation.

The role of Atg8 in selective autophagy. In addition to its function in autophagosomal membrane formation, Atg8 is involved in efficient incorporation of cargoes into autophagosomes in selective types of autophagy. Although the Cvt pathway exists only in yeast and serves as a biosynthetic pathway, extensive studies on this pathway have established a conceptual framework to understand the mechanism of selective autophagy (see below). In this pathway, the cargo Ape1 self-assembles into an aggregate-like structure and interacts with the receptor protein Atg19 (REFS 82,83). Atg19 also interacts with Atg8, and this interaction is thought to link the cargo–receptor complex to the forming Cvt vesicle<sup>84–86</sup>. In mammalian cells, p62 is responsible for selective degradation of ubiquitin-positive protein inclusions through autophagy and also binds to mammalian homologues of Atg8 (REFS 87–90). Therefore, although Atg19 and p62 are unrelated to each other in their entire sequences, these proteins probably function in similar ways in selective incorporation of the cargoes into vesicles. Interestingly, recent structural studies revealed a common interaction between these receptors and Atg8 homologues: the Trp-X-X-Leu motif in the receptors binds to the highly conserved, hydrophobic pocket in the Atg8 homologues in a similar manner<sup>86,91</sup>. A similar interaction might work broadly in the recognition of various cargoes in selective autophagy.

Dynamic features of the PAS

In contrast to our earlier view that the PAS is a static and stoichiometric structure<sup>26</sup>, recent studies have uncovered dynamic aspects of the PAS, which can be versatile in its composition depending on the physiological situations. Here, we define the PAS as a dynamic assembly of the core Atg proteins, which function as membrane-forming machinery, and of ‘conductor’ proteins that spatio-temporally regulate the core proteins to determine the site for PAS organization and the mode of membrane formation. On this basis, we describe our present view on the PAS.

As mentioned above, yeast cells have the Cvt pathway, in which small autophagosome-like vesicles called Cvt vesicles are formed to deliver vacuolar enzymes, such as Ape1, to the vacuole. The assembly of the Atg proteins, which has also been called the PAS, is involved in the formation of the Cvt vesicle as well as that of the autophagosome<sup>92</sup>. In contrast to autophagy, the Cvt pathway is active under nutrient-rich conditions and, consistently, the PAS that mediates Cvt vesicle formation is observed under such conditions — this made it difficult to analyse Atg protein dynamics in response to induction of autophagy. The problem has been circumvented by analysing cells that lack Atg11 (REFS 38,39); Atg11 is specifically required for the Cvt pathway, and is responsible for organization

Figure 6 | Two ubiquitin-like conjugation systems. **a** | The conjugation system of autophagy-related 8 (Atg8). First, Atg4 cleaves the carboxy-terminal Arg (R) residue of Atg8 to expose Gly (G) at the new C terminus. Atg8 is then activated by Atg7 (an E1 enzyme), transferred to Atg3 (an E2 enzyme) and eventually conjugated to phosphatidylethanolamine (PE). The active site Cys (C) residues of Atg7 and Atg3 are indicated. Atg4 also cleaves the amide bond between Atg8 and PE to release the protein from membranes. **b** | The conjugation system of Atg12. Atg12 is conjugated to the specific Lys (K) residue of Atg5 in a similar manner to the conjugation reaction of Atg8, except that Atg10 functions as the E2 enzyme in this system instead of Atg3. An E3 enzyme for the Atg12 conjugation reaction has not been reported. The Atg12–Atg5 conjugate interacts with Atg16 and forms an oligomer. The Atg12–Atg5–Atg16 complex then exerts an E3 enzyme-like function on the Atg8 conjugation reaction; the transfer reaction of Atg8 from Atg3 to PE is stimulated by this complex. Atg12–Atg5–Atg16 is also suggested to determine the site of the production of Atg8–PE (see the main text for details). **c** | Localization of ubiquitin-like protein conjugates on autophagy-related membranes. Atg8–PE is present on both surfaces of the isolation membrane, and part of the conjugate is left inside the autophagosome, delivered to the vacuole and degraded. Atg12–Atg5–Atg16 preferentially localizes on the outer surface of the membrane and dissociates from the membrane upon completion of the autophagosome. **d** | Membrane tethering and hemifusion functions of Atg8. On conjugation to PE on liposomes, Atg8 oligomerizes and tethers the liposomes, leading to hemifusion of the membranes. Only lipids in outer leaflets interdiffuse in hemifused membranes.

In contrast to PAS assembly in starvation-induced autophagy, it is remarkable that PAS assembly in the Cvt pathway depends on a large complex composed of the cargo Ape1 and the receptor Atg19 (REF. 94) (FIG. 7). It is thought that the cargo–receptor complex serves as a scaffold for the recruitment of Atg11 (REF. 95). Atg11 then recruits the core Atg proteins to allow the formation of the Cvt vesicle around the cargo–receptor complex. Thus, similar to the Atg17–Atg29–Atg31 complex in starvation-induced autophagy, Atg11 behaves as a conductor together with the cargo–receptor complex in the organization of the PAS in the Cvt pathway (FIG. 7). In addition, it was reported that Atg11 is also involved in pexophagy and ‘mitophagy’, the autophagic degradation of mitochondria, in which Atg11 is suggested to function in a similar manner but in cooperation with Atg30 and probably an unidentified factor, respectively, instead of Atg19 (REFS 96,97).

In this way, conductor Atg proteins seem to regulate the core Atg proteins spatio-temporally and determine the mode of the PAS. The results obtained in studies on starvation-induced autophagy and the Cvt pathway clearly show that two factors — nutrient conditions and the existence of the cargo — can regulate the mode of the PAS and thus the site of vesicle formation and the size of the vesicle. These factors could independently or cooperatively affect the mode of the PAS. In addition, unidentified conductors might exist that respond to different environmental signals or recognize specific cargo complexes. This creates the functional diversity of the roles of autophagy that is rapidly emerging in higher eukaryotes. Although the PAS has not been described in mammalian cells, foci that might correspond to the PAS have recently been reported in slime moulds and higher plants^{98,99}.

The PAS has only been observed as a dot under a fluorescence microscope. We defined the PAS as an assembly of the Atg proteins that can exist without the isolation membrane^{26}. However, certain Atg proteins, such as Atg8 and the Atg12–Atg5–Atg16 complex, localize to both the PAS and the isolation membrane. In addition, intriguingly, Atg1, but neither Atg13 nor the Atg17–Atg29–Atg31 complex, is transported to the vacuole, even though these five Atg proteins are interdependently assembled onto the PAS^{26}. Therefore, it is possible that a number of Atg proteins transit from the PAS to the isolation membrane as the membrane grows. Further analyses, including detailed immuno-electron microscopy and fluorescence imaging for protein interactions in living cells, will allow us to discuss the PAS more definitively as the spatial configuration of the Atg proteins to the isolation membrane and the cargo complex.

**Concluding remarks**

For unicellular organisms such as yeast, depletion of nutrients must be the most frequent and crucial stress in nature. Therefore, a starvation-induced mode of autophagy, which is essential for the maintenance of a pool of metabolites such as amino acids, would have been established first and has been conserved during

Figure 7 | Model for the role of conductor proteins in PAS organization. Under nutrient-rich conditions, autophagy-related 11 (Atg11) serves as a conductor, together with a large assembly of aminopeptidase 1 (Ape1; a cargo protein) and Atg19 (a receptor), to recruit other Atg proteins, including core machinery components and Cvt (cytoplasm-to-vacuole targeting)-specific proteins. This recruitment organizes the pre-autophagosomal structure (PAS) to mediate Cvt vesicle formation around the cargo (left panel). Although the Atg17–Atg29–Atg31 complex, which is specifically required for starvation-induced autophagy, also localizes to the PAS under these conditions, this complex is dispensable for the Cvt pathway. By contrast, under nutrient-deprived conditions, Atg17–Atg29–Atg31, together with Atg1 and Atg13, acts as a conductor in a manner that is independent of Atg11. By these means the PAS is reorganized to a mode for autophagosome formation (right panel). The precise interactions among these proteins under each condition are elusive.

REVIEW S

evolution. In fact, all of the core ATG genes are conserved in mammals and plants. Once the basic system for membrane formation had been established, additional molecules that endow cargo selectivity might have been acquired, which have developed various functions of autophagy.

The machinery for starvation-induced autophagy is likely to utilize specialized factors for selective autophagy in some cases. Certain cargoes for selective autophagy are more efficiently transported to lytic compartments by autophagy under starvation conditions in a manner that is dependent on such factors, in which the cargoes should be incorporated into autophagosomes together with other cytoplasmic components ${ }^{9,100,101}$. In addition, cytoplasmic acetaldehyde dehydrogenase 6 (Ald6) and ribosomes are known to be preferentially degraded through autophagy under nitrogen starvation conditions, even though they seem to be completely dispersed in the cytoplasm, suggesting a novel mechanism for these cases ${ }^{102,103}$. Thus, the way in which materials that are to be degraded in autophagy are sequestered also varies. Finally, it should also be noted that Leu amino-peptidase 3 (Lap3) has recently proved to be selectively transported to the vacuole for degradation in glycerol-grown (non-starved) yeast cells ${ }^{101}$ (T. Kageyama and Y.O., unpublished observations). This process depends on Atg11 and Atg19, which are specifically required for the Cvt pathway, indicating that the Cvt pathway also functions as a selective degradation system.

Recent studies, especially in higher eukaryotes such as mammals and plants, have rapidly unveiled the diversity and complexity of autophagy. An in-depth knowledge of the diverse modes of autophagy is quite important for accurate understanding of not only the mechanism but also the significance of autophagy in each physiological or pathological situation.

1. Deter, R. L., Baudhuin, P. & De Duve, C. Participation of lysosomes in cellular autophagy induced in rat liver by glucagon. J. Cell Biol. 35, C11–C16 (1967).
2. Klionsky, D. J. Autophagy: from phenomenology to molecular understanding in less than a decade. Nature Rev. Mol. Cell Biol. 8, 931–937 (2007).
3. Mizushima, N. Autophagy: process and function. Genes Dev. 21, 2861–2873 (2007).
4. Mizushima, N., Levine, B., Cuervo, A. M. & Klionsky, D. J. Autophagy fights disease through cellular self-digestion. Nature 451, 1069–1075 (2008).
5. Takeshige, K., Baba, M., Tsuboi, S., Noda, T. & Ohsumi, Y. Autophagy in yeast demonstrated with proteinase-deficient mutants and conditions for its induction. J. Cell Biol. 119, 301–311 (1992). Reports the discovery of autophagy in yeast.
6. Baba, M., Takeshige, K., Baba, N. & Ohsumi, Y. Ultrastructural analysis of the autophagic process in yeast: detection of autophagosomes and their characterization. J. Cell Biol. 124, 903–913 (1994).
7. Baba, M., Osumi, M. & Ohsumi, Y. Analysis of the membrane structures involved in autophagy in yeast by freeze-replica method. Cell Struct. Funct. 20, 465–471 (1995). References 6 and 7 report the detailed morphological characterization of yeast autophagy.
8. Tsukada, M. & Ohsumi, Y. Isolation and characterization of autophagy-defective mutants of Saccharomyces cerevisiae. FEBS Lett. 333, 169–174 (1993). First report on autophagy-defective yeast mutants.
9. Baba, M., Osumi, M., Scott, S. V., Klionsky, D. J. & Ohsumi, Y. Two distinct pathways for targeting proteins from the cytoplasm to the vacuole/lysosome. J. Cell Biol. 139, 1687–1695 (1997).
10. Harding, T. M., Hefner-Gravink, A., Thumm, M. & Klionsky, D. J. Genetic and phenotypic overlap between autophagy and the cytoplasm to vacuole protein targeting pathway. J. Biol. Chem. 271, 17621–17624 (1996).
11. Thumm, M. et al. Isolation of autophagocytosis mutants of Saccharomyces cerevisiae. FEBS Lett. 349, 275–280 (1994).
12. Yuan, W., Tuttle, D. L., Shi, Y. J., Ralph, G. S. & Dunn, W. A. Jr. Glucose-induced microautophagy in Pichia pastoris requires the α-subunit of phosphofructokinase. J. Cell Sci. 110, 1935–1945 (1997).
13. Sakai, Y., Koller, A., Rangell, L. K., Keller, G. A. & Subramani, S. Peroxisome degradation by microautophagy in Pichia pastoris: identification of specific steps and morphological intermediates. J. Cell Biol. 141, 625–636 (1998).
14. Mukaiyama, H. et al. Paz2 and 13 other PAZ gene products regulate vacuolar engulfment of peroxisomes during micropexophagy. Genes Cells 7, 75–90 (2002).
15. Titorenko, V. I., Keizer, I., Harder, W. & Veenhuis, M. Isolation and characterization of mutants impaired in the selective degradation of peroxisomes in the yeast

Hansenula polymorpha. J. Bacteriol. 177, 357–363 (1995).

16. Klionsky, D. J. et al. A unified nomenclature for yeast autophagy-related genes. Dev. Cell 5, 539–545 (2003).

17. Kamada, Y. et al. Tor-mediated induction of autophagy via an Apg1 protein kinase complex. J. Cell Biol. 150, 1507–1513 (2000). Shows dephosphorylation of Atg13 in response to nutrient starvation, which allows Atg13 to interact with Atg1, leading to stimulation of the kinase activity of Atg1.

18. Kihara, A., Noda, T., Ishihara, N. & Ohsumi, Y. Two distinct Vps34 phosphatidylinositol 3-kinase complexes function in autophagy and carboxypeptidase Y sorting in Saccharomyces cerevisiae. J. Cell Biol. 152, 519–530 (2001). Identifies two distinct PtdIns 3-kinase complexes, one of which contains Atg14 as a specific component and is required for autophagy.

19. Mizushima, N. et al. A protein conjugation system essential for autophagy. Nature 395, 395–398 (1998).

20. Ichimura, Y. et al. A ubiquitin-like system mediates protein lipidation. Nature 408, 488–492 (2000). References 19 and 20 report the discovery of two ubiquitin-like protein conjugates required for autophagy, Atg12–Atg5 and Atg8–PE, respectively.

21. Shintani, T., Suzuki, K., Kamada, Y., Noda, T. & Ohsumi, Y. Apg2p functions in autophagosome formation on the perivacuolar structure. J. Biol. Chem. 276, 30452–30460 (2001).

22. Wang, C. W. et al. Apg2 is a novel protein required for the cytoplasm to vacuole targeting, autophagy, and pexophagy pathways. J. Biol. Chem. 276, 30442–30451 (2001).

23. Noda, T. et al. Apg9p/Cvt7p is an integral membrane protein required for transport vesicle formation in the Cvt and autophagy pathways. J. Cell Biol. 148, 465–480 (2000).

24. Barth, H., Meiling-Wesse, K., Epple, U. D. & Thumm, M. Autophagy and the cytoplasm to vacuole targeting pathway both require Aut10p. FEBS Lett. 508, 23–28 (2001).

25. Kirisako, T. et al. Formation process of autophagosome is traced with Apg8/Aut7p in yeast. J. Cell Biol. 147, 435–446 (1999). Reports the identification of Atg8 as a marker protein for the autophagosome and the isolation membrane.

26. Suzuki, K. et al. The pre-autophagosomal structure organized by concerted functions of APG genes is essential for autophagosome formation. EMBO J. 20, 5971–5981 (2001). Reports the identification of the PAS.

27. Suzuki, K., Kubota, Y., Sekito, T. & Ohsumi, Y. Hierarchy of Atg proteins in pre-autophagosomal structure organization. Genes Cells 12, 209–218 (2007). Reveals that Atg proteins organize the PAS in a hierarchical manner.

28. Cao, Y., Cheong, H., Song, H. & Klionsky, D. J. In vivo reconstitution of autophagy in Saccharomyces cerevisiae. J. Cell Biol. 182, 703–713 (2008).

29. Noda, T. & Ohsumi, Y. Tor, a phosphatidylinositol kinase homologue, controls autophagy in yeast. J. Biol. Chem. 273, 3963–3966 (1998).

30. Wullschleger, S., Loewith, R. & Hall, M. N. TOR signaling in growth and metabolism. Cell 124, 471–484 (2006).

31. Yorimitsu, T., Zaman, S., Broach, J. R. & Klionsky, D. J. Protein kinase A and Sch9 cooperatively regulate induction of autophagy in Saccharomyces cerevisiae. Mol. Biol. Cell 18, 4180–4189 (2007).

32. Budovskaya, Y. V., Stephan, J. S., Reggiori, F., Klionsky, D. J. & Herman, P. K. The Ras/cAMP-dependent protein kinase signaling pathway regulates an early step of the autophagy process in Saccharomyces cerevisiae. J. Biol. Chem. 279, 20663–20671 (2004).

33. Matsuura, A., Tsukada, M., Wada, Y. & Ohsumi, Y. Apg1p, a novel protein kinase required for the autophagic process in Saccharomyces cerevisiae. Gene 192, 245–250 (1997).

34. Funakoshi, T., Matsuura, A., Noda, T. & Ohsumi, Y. Analyses of APG13 gene involved in autophagy in yeast, Saccharomyces cerevisiae. Gene 192, 207–213 (1997).

35. Kabeya, Y. et al. Atg17 functions in cooperation with Atg1 and Atg13 in yeast autophagy. Mol. Biol. Cell 16, 2544–2553 (2005).

36. Kawamata, T. et al. Characterization of a novel autophagy-specific gene, ATG29. Biochem. Biophys. Res. Commun. 338, 1884–1889 (2005).

37. Kabeya, Y., Kawamata, T., Suzuki, K. & Ohsumi, Y. Cis1/Atg31 is required for autophagosome formation in Saccharomyces cerevisiae. Biochem. Biophys. Res. Commun. 356, 405–410 (2007).

38. Kawamata, T., Kamada, Y., Kabeya, Y., Sekito, T. & Ohsumi, Y. Organization of the pre-autophagosomal structure responsible for autophagosome formation. Mol. Biol. Cell 19, 2039–2050 (2008).

39. Cheong, H., Nair, U., Geng, J. & Klionsky, D. J. The Atg1 kinase complex is involved in the regulation of protein recruitment to initiate sequestering vesicle formation for nonspecific autophagy in Saccharomyces cerevisiae. Mol. Biol. Cell 19, 668–681 (2008). References 38 and 39 show that the Atg1 kinase and its regulators collaboratively function to organize, or reorganize, the PAS in response to nutrient starvation.

40. Cheong, H. et al. Atg17 regulates the magnitude of the autophagic response. Mol. Biol. Cell 16, 3438–3453 (2005).

41. Sekito, T., Kawamata, T., Ichikawa, R., Suzuki, K. & Ohsumi, Y. Atg17 recruits Atg9 to organize the pre-autophagosomal structure. Genes Cells 14, 525–538 (2009).

42. Klionsky, D. J. The molecular machinery of autophagy: unanswered questions. J. Cell Sci. 118, 7–18 (2005).

43. Hara, T. *et al.* FIP200, a ULK-interacting protein, is required for autophagosome formation in mammalian cells. *J. Cell Biol.* **181**, 497–510 (2008).

44. Chan, E. Y., Longatti, A., McKnight, N. C. & Tooze, S. A. Kinase-inactivated ULK proteins inhibit autophagy via their conserved C-terminal domains using an Atg13-independent mechanism. *Mol. Cell. Biol.* **29**, 157–171 (2009).

45. Hosokawa, N. *et al.* Nutrient-dependent mTORC1 association with the ULK1–Atg13–FIP200 complex required for autophagy. *Mol. Biol. Cell* **20**, 1981–1991 (2009).

46. Chang, Y. Y. & Neufeld, T. P. An Atg1/Atg13 complex with multiple roles in TOR-mediated autophagy regulation. *Mol. Biol. Cell* **20**, 2004–2014 (2009).

47. Jung. *et al.* ULK–Atg13–FIP200 complexes mediate mTOR signaling to the autophagy machinery. *Mol. Biol. Cell* **20**, 1992–2003 (2009).

48. Ganley. *et al.* ULK1•ATG13•FIP200 complex mediates mTOR signaling and is essential for autophagy. *J. Biol. Chem.* **284**, 12297–12305.

49. Petiot, A., Ogier-Denis, E., Blommaart, E. F., Meijer, A. J. & Codogno, P. Distinct classes of phosphatidylinositol 3′-kinases are involved in signaling pathways that control macroautophagy in HT-29 cells. *J. Biol. Chem.* **275**, 992–998 (2000).

50. Obara, K., Sekito, T., Niimi, K. & Ohsumi, Y. The Atg18–Atg2 complex is recruited to autophagic membranes via phosphatidylinositol 3-phosphate and exerts an essential function. *J. Biol. Chem.* **283**, 23972–23980 (2008).

51. Schu, P. V. *et al.* Phosphatidylinositol 3-kinase encoded by yeast VPS34 gene essential for protein sorting. *Science* **260**, 88–91 (1993).

52. Obara, K., Sekito, T. & Ohsumi, Y. Assortment of phosphatidylinositol 3-kinase complexes—Atg14p directs association of complex I to the pre-autophagosomal structure in *Saccharomyces cerevisiae*. *Mol. Biol. Cell* **17**, 1527–1539 (2006).

53. Dove, S. K. *et al.* Svp1p defines a family of phosphatidylinositol 3,5-bisphosphate effectors. *EMBO J.* **23**, 1922–1933 (2004).

54. Stromhaug, P. E., Reggiori, F., Guan, J., Wang, C. W. & Klionsky, D. J. Atg21 is a phosphoinositide binding protein required for efficient lipidation and localization of Atg8 during uptake of amino-peptidase I by selective autophagy. *Mol. Biol. Cell* **15**, 3553–3566 (2004).

55. Efe, J. A., Botelho, R. J. & Emr, S. D. Atg18 regulates organelle morphology and Fab1 kinase activity independent of its membrane recruitment by phosphatidylinositol 3,5-bisphosphate. *Mol. Biol. Cell* **18**, 4232–4244 (2007).

56. Obara, K., Noda, T., Niimi, K. & Ohsumi, Y. Transport of phosphatidylinositol 3-phosphate into the vacuole via autophagic membranes in *S. cerevisiae*. *Genes Cells*, **13**, 537–547 (2008).

57. Reggiori, F., Tucker, K. A., Stromhaug, P. E. & Klionsky, D. J. The Atg1–Atg13 complex regulates Atg9 and Atg23 retrieval transport from the pre-autophagosomal structure. *Dev. Cell* **6**, 79–90 (2004).

58. Young, A. R. *et al.* Starvation and ULK1-dependent cycling of mammalian Atg9 between the TGN and endosomes. *J. Cell Sci.* **119**, 3888–3900 (2006).

59. Xie, Z. & Klionsky, D. J. Autophagosome formation: core machinery and adaptations. *Nature Cell Biol.* **9**, 1102–1109 (2007).

60. He, C., Baba, M., Cao, Y. & Klionsky, D. J. Self-interaction is critical for Atg9 transport and function at the phagophore assembly site during autophagy. *Mol. Biol. Cell* **19**, 5506–5516 (2008).

61. Mizushima, N. *et al.* Dissection of autophagosome formation using Apg5-deficient mouse embryonic stem cells. *J. Cell Biol.* **152**, 657–668 (2001). Demonstration of Atg5 localization on the isolation membrane and real-time visualization of the process of autophagosome formation in living cells using GFP-tagged Atg5.

62. Kirisako, T. *et al.* The reversible modification regulates the membrane-binding state of Apg8/Aut7 essential for autophagy and the cytoplasm to vacuole targeting pathway. *J. Cell Biol.* **151**, 263–276 (2000).

63. Kim, J., Huang, W. P. & Klionsky, D. J. Membrane recruitment of Aut7p in the autophagy and cytoplasm to vacuole targeting pathways requires Aut1p, Aut2p, and the autophagy conjugation complex. *J. Cell Biol.* **152**, 51–64 (2001).

64. Nakatogawa, H., Ichimura, Y. & Ohsumi, Y. Atg8, a ubiquitin-like protein required for autophagosome formation, mediates membrane tethering and hemifusion. *Cell* **130**, 165–178 (2007).

65. Kabeya, Y. *et al.* LC3, a mammalian homologue of yeast Apg8p, is localized in autophagosome membranes after processing. *EMBO J.* **19**, 5720–5728 (2000).

66. Yoshimoto, K. *et al.* Processing of ATG8s, ubiquitin-like proteins, and their deconjugation by ATG4s are essential for plant autophagy. *Plant Cell* **16**, 2967–2983 (2004).

67. Klionsky, D. J. *et al.* Guidelines for the use and interpretation of assays for monitoring autophagy in higher eukaryotes. *Autophagy* **4**, 151–175 (2008).

68. Ichimura, Y. *et al.* In vivo and in vitro reconstitution of Atg8 conjugation essential for autophagy. *J. Biol. Chem.* **279**, 40584–40592 (2004).

69. Xie, Z., Nair, U. & Klionsky, D. J. Atg8 controls phagophore expansion during autophagosome formation. *Mol. Biol. Cell* **19**, 3290–3298 (2008).

70. Fujita, N. *et al.* An Atg4B mutant hampers the lipidation of LC3 paralogues and causes defects in autophagosome closure. *Mol. Biol. Cell* **19**, 4651–4659 (2008).

71. Sou, Y. S. *et al.* The Atg8 conjugation system is indispensable for proper development of autophagic isolation membranes in mice. *Mol. Biol. Cell* **19**, 4762–4775 (2008).

72. Mizushima, N., Noda, T. & Ohsumi, Y. Apg16p is required for the function of the Apg12–Apg5p conjugate in the yeast autophagy pathway. *EMBO J.* **18**, 3888–3896 (1999).

73. Kuma, A., Mizushima, N., Ishihara, N. & Ohsumi, Y. Formation of the approximately 350-kDa Apg12–Apg5•Apg16 multimeric complex, mediated by Apg16 oligomerization, is essential for autophagy in yeast. *J. Biol. Chem.* **277**, 18619–18625 (2002).

74. Mizushima, N. *et al.* Mouse Apg16L, a novel WD-repeat protein, targets to the autophagic isolation membrane with the Apg12–Apg5 conjugate. *J. Cell Sci.* **116**, 1679–1688 (2003).

75. Geng, J., Baba, M., Nair, U. & Klionsky, D. J. Quantitative analysis of autophagy-related protein stoichiometry by fluorescence microscopy. *J. Cell Biol.* **182**, 129–140 (2008).

76. Hanada, T. & Ohsumi, Y. Structure–function relationship of Atg12, a ubiquitin-like modifier essential for autophagy. *Autophagy* **1**, 110–118 (2005).

77. Hanada, T. *et al.* The Atg12–Atg5 conjugate has a novel E3-like activity for protein lipidation in autophagy. *J. Biol. Chem.* **282**, 37298–37302 (2007). Shows that the ubiquitin-like protein conjugate Atg12–Atg5 directly stimulates the conjugation reaction of the other ubiquitin-like protein Atg8 to PE.

78. Fujioka, Y. *et al.* In vitro reconstitution of plant Atg8 and Atg12 conjugation systems essential for autophagy. *J. Biol. Chem.* **283**, 1921–1928 (2008).

79. Sou, Y. S., Tanida, I., Komatsu, M., Ueno, T. & Kominami, E. Phosphatidylserine in addition to phosphatidylethanolamine is an in vitro target of the mammalian Atg8 modifiers, LC3, GABARAP, and GATE-16. *J. Biol. Chem.* **281**, 3017–3024 (2006).

80. Oh-oka, K., Nakatogawa, H. & Ohsumi, Y. Physiological pH and acidic phospholipids contribute to substrate specificity in lipidation of Atg8. *J. Biol. Chem.* **283**, 21847–21852 (2008).

81. Fujita, N. *et al.* The Atg16L complex specifies the site of LC3 lipidation for membrane biogenesis in autophagy. *Mol. Biol. Cell* **19**, 2092–2100 (2008).

82. Kim, J., Scott, S. V., Oda, M. N. & Klionsky, D. J. Transport of a large oligomeric protein by the cytoplasm to vacuole protein targeting pathway. *J. Cell Biol.* **137**, 609–618 (1997).

83. Scott, S. V., Guan, J., Hutchins, M. U., Kim, J. & Klionsky, D. J. Cvt19 is a receptor for the cytoplasm-to-vacuole targeting pathway. *Mol. Cell* **7**, 1131–1141 (2001).

84. Shintani, T., Huang, W. P., Stromhaug, P. E. & Klionsky, D. J. Mechanism of cargo selection in the cytoplasm to vacuole targeting pathway. *Dev. Cell* **3**, 825–837 (2002).

85. Chang, C. Y. & Huang, W. P. Atg19 mediates a dual interaction cargo sorting mechanism in selective autophagy. *Mol. Biol. Cell* **18**, 919–929 (2007).

86. Noda, N. N. *et al.* Structural basis of target recognition by Atg8/LC3 during selective autophagy. *Genes Cells* **13**, 1211–1218 (2008).

87. Bjorkoy, G. *et al.* p62/SQSTM1 forms protein aggregates degraded by autophagy and has a protective effect on huntingtin-induced cell death. *J. Cell Biol.* **171**, 603–614 (2005).

88. Pankiv, S. *et al.* p62/SQSTM1 binds directly to Atg8/LC3 to facilitate degradation of ubiquitinated protein aggregates by autophagy. *J. Biol. Chem.* **282**, 24131–24145 (2007).

89. Komatsu, M. *et al.* Homeostatic levels of p62 control cytoplasmic inclusion body formation in autophagy-deficient mice. *Cell* **131**, 1149–1163 (2007).

90. Shvets, E., Fass, E., Scherz-Shouval, R. & Elazar, Z. The N-terminus and Phe52 residue of LC3 recruit p62/SQSTM1 into autophagosomes. *J. Cell Sci.* **121**, 2685–2695 (2008).

91. Ichimura, Y. *et al.* Structural basis for sorting mechanism of p62 in selective autophagy. *J. Biol. Chem.* **283**, 22847–22857 (2008). References 86–91 establish the molecular framework for p62-mediated degradation of disease-related protein inclusions through autophagy and its pathological significance.

92. Nice, D. C., Sato, T. K., Stromhaug, P. E., Emr, S. D. & Klionsky, D. J. Cooperative binding of the cytoplasm to vacuole targeting pathway proteins, Cvt13 and Cvt20, to phosphatidylinositol 3-phosphate at the pre-autophagosomal structure is required for selective autophagy. *J. Biol. Chem.* **277**, 30198–30207 (2002).

93. Kim, J. *et al.* Cvt9/Gsa9 functions in sequestering selective cytosolic cargo destined for the vacuole. *J. Cell Biol.* **153**, 381–396 (2001).

94. Shintani, T. & Klionsky, D. J. Cargo proteins facilitate the formation of transport vesicles in the cytoplasm to vacuole targeting pathway. *J. Biol. Chem.* **279**, 29889–29894 (2004). Shows that PAS organization under nutrient-rich conditions, and thus Cvt vesicle formation, largely depends on the existence of the cargo molecule Ape1 in addition to the Cvt-specific proteins Atg11 and Atg19.

95. Yorimitsu, T. & Klionsky, D. J. Atg11 links cargo to the vesicle-forming machinery in the cytoplasm to vacuole targeting pathway. *Mol. Biol. Cell* **16**, 1593–1605 (2005).

96. Farre, J. C., Manjithaya, R., Mathewson, R. D. & Subramani, S. PpAtg30 tags peroxisomes for turnover by selective autophagy. *Dev. Cell* **14**, 365–376 (2008).

97. Kanki, T. & Klionsky, D. J. Mitophagy in yeast occurs through a selective mechanism. *J. Biol. Chem.* **283**, 32386–32393 (2008).

98. Otto, G. P., Wu, M. Y., Kazgan, N., Anderson, O. R. & Kessin, R. H. *Dictyostelium* macroautophagy mutants vary in the severity of their developmental defects. *J. Biol. Chem.* **279**, 15621–15629 (2004).

99. Fujiki, Y. *et al.* An *Arabidopsis* homolog of yeast ATG6/VPS30 is essential for pollen germination. *Plant Physiol.* **143**, 1132–1139 (2007).

100. Ravikumar, B. *et al.* Inhibition of mTOR induces autophagy and reduces toxicity of polyglutamine expansions in fly and mouse models of Huntington disease. *Nature Genet.* **36**, 585–595 (2004).

101. Kageyama, T., Suzuki, K. & Ohsumi, Y. Lap3 is a selective target of autophagy in yeast, *Saccharomyces cerevisiae*. *Biochem. Biophys. Res. Commun.* **378**, 551–557 (2009).

102. Onodera, J. & Ohsumi, Y. Ald6p is a preferred target for autophagy in yeast, *Saccharomyces cerevisiae*. *J. Biol. Chem.* **279**, 16071–16076 (2004).

103. Kraft, C., Deplazes, A., Sohrmann, M. & Peter, M. Mature ribosomes are selectively degraded upon starvation by an autophagy pathway requiring the Ubp3p/Bre5p ubiquitin protease. *Nature Cell Biol.* **10**, 602–610 (2008). References 102 and 103 report that the cytoplasmic protein Ald6 and ribosomal proteins, respectively, are preferentially degraded in yeast by mechanisms that seem to be distinct from those previously suggested for other selective types of autophagy.


**DATABASES**

UniProtKB: http://www.uniprot.org  
Ape1 | Atg1 | Atg2 | Atg3 | Atg5 | Atg7 | Atg8 | Atg9 | Atg10 | Atg11 | Atg12 | Atg13 | Atg14 | Atg16 | Atg17 | Atg18 | Atg29 | Atg31 | Tor | Vps15 | Vps30 | Vps34 | Vps38  

ALL LINKS ARE ACTIVE IN THE ONLINE PDF
