
Stem Cell Rev and Rep (2009) 5:204–223  
DOI 10.1007/s12015-009-9077-x  

# The Molecular Mechanism of Induced Pluripotency: A Two-Stage Switch

**Wouter Scheper · Sjef Copray**

Published online: 24 June 2009  
© Humana Press 2009  

---

### Abstract  
Pluripotent stem cells are basic cells with an indefinite self-renewal capacity and the potential to generate all the cell types of the three germinal layers. So far, the major source for pluripotent stem cells is the inner cell mass of the blastocysts: embryonic stem (ES) cells. Potential clinical application of ES cells is faced with many practical and ethical concerns. So, a major breakthrough was achieved in 2006, when it was shown that pluripotent stem cells could be obtained by transducing mouse embryonic and adult fibroblasts with a limited set of defined transcription factors. These reprogrammed cells, named induced pluripotent stem (iPS) cells, resembled ES cells in many of their characteristics. Since this initial study, iPS cell research has taken an incredible flight, and to date iPS cells have been generated from cells from several species using different sets of reprogramming factors. Given the potential to generate patient-specific cell populations without the need for human embryonic cells, iPS cell technology has been received with great excitement by research and medical communities. However, many questions regarding the actual molecular process of induced reprogramming remain unanswered and need to be addressed before iPS cells can go to the clinic. In this review, we start by summarizing recent advances in iPS cell research and inventory the hurdles that still need to be taken before safe clinical application. Our major aim, however, is to review the available data on the molecular processes underlying pluripotency reprogramming and present a two-stage switch model.

---

### Keywords  
iPS cells · Epigenetics · Embryonic stem cells · Transformation · Dedifferentiation

---

### Introduction  

In a landmark study, Takahashi and Yamanaka [1] showed that ectopic expression of defined transcription factors was sufficient to reprogram mouse fibroblasts back to a pluripotent state [1]. In this study, different combinations of 24 candidate reprogramming genes were retrovirally transduced into mouse fibroblasts derived from transgenic mice containing a neomycin resistance gene knocked into the Fbx15 locus. Fbx15 is exclusively expressed in ES cells [2], and drug resistance due to reactivation of the reporter gene was used as a marker for induced pluripotency. They found that the combined introduction of the four factors Oct4, Sox2, c-Myc and Klf4 yielded multiple colonies of cells that resembled ES cells in morphology and proliferative characteristics. Moreover, these cells were positive for the ES cell markers alkaline phosphatase (AP) and SSEA1, differentiated into all three germ layers *in vitro*, and were able to form teratomas when injected into nude mice. Nonetheless, these induced pluripotent stem (iPS) cells differed from ES cells in a number of ways. For example, a substantial number of genes were differentially expressed in ES cells and iPS cells, and when transplanted into blastocysts, iPS cells were unable to give rise to live chimeras. In addition, although these cells showed features of a pluripotent state, this state was dependent on the continuous expression of exogenous Oct4 and Sox2, while the promoters of endogenous Oct4 and Nanog genes—genes that are crucial for pluripotency of ES cells [3, 4]—remained repressed by DNA methylation. Thus, although they resembled ES cells in many features, these iPS cells were clearly not identical to ES cells.

---

W. Scheper · S. Copray (✉)  
Department of Neuroscience,  
University Medical Centre Groningen,  
A.Deusinglaan 1,  
9713 AV Groningen, The Netherlands  
e-mail: j.c.v.m.copray@med.umcg.nl

A major improvement was reported less than one year later, when three studies demonstrated that selection for endogenous Nanog or Oct4 expression instead of Fbx15 yielded iPS cells that were fully reprogrammed to a pluripotent state [5–7]. Nanog and Oct4 were expected to be better markers of complete reprogramming because they are crucial to the pluripotent state of ES cells [3, 4], while Fbx15 is dispensable for pluripotency [2]. In contrast to Fbx15 selection, Nanog- and Oct4-selected iPS cells contributed to live chimeras when injected into blastocysts. Moreover, one study reported that these iPS cells were transmitted through the germline to offspring [6], which is generally considered one of the most stringent criteria for assessing pluripotency. In addition, Nanog- and Oct4-selected iPS cells were highly similar to ES cells in gene expression profiles, as well as in global DNA methylation and histone modification patterns [5–7]. Finally, in contrast to Fbx15-selected iPS cells, the pluripotent state was dependent on the reactivation of endogenous Oct4 and Nanog loci, as the promoters of these genes were observed to be demethylated in reprogrammed cells while retroviral transgenes were silenced by *de novo* DNA methylation.

These first steps in iPS cell technology have been received with great excitement in the medical world, since it promised the generation of patient-derived pluripotent stem cells as sources for cell replacement therapies for a variety of disorders. For iPS cell technology to live up to its therapeutic promise, it was of great importance to demonstrate that reprogramming by defined factors could also be achieved in human somatic cells. Shortly after the work discussed above, three groups independently demonstrated that the principle of reprogramming by defined factors is also applicable to human cells [8–10]. In these studies, iPS cells were derived from genetically unmodified human fibroblasts by the retroviral introduction of different sets of transcription factors. Two groups used the same reprogramming cocktail that was used in mouse [8, 10]. One group however, found that a different set of factors (Oct4, Sox2, Nanog and Lin28) could be used to reprogram human somatic cells as well, suggesting that different factors may be capable of inducing pluripotency [9]. In each of these studies, human iPS cells closely resembled ES cells in many characteristics, including gene expression profiles, morphology, expression of telomerase, and patterns of DNA methylation and histone modifications. Viral transgenes were found to be silenced, and Oct4 and Nanog were expressed from endogenous loci. Also, human iPS cells were able to give rise to all three germ layers *in vitro* and they formed mature teratomas after injection into nude mice. Since these initial results, numerous studies have reported the generation of human iPS cells, confirming the reproducibility of the reprogramming process in human cells [11–20].

---

Potential Clinical Applications of iPS Cells

The actual potential clinical significance of iPS cells was demonstrated by a number of different groups. In a breakthrough study, Hanna and colleagues provided proof of principle for the use of iPS cells in cell transplantation therapy in mouse [21]. They showed that transgenic mice that were engineered to suffer from human sickle cell anaemia could be successfully treated with hematopoietic progenitor cells that were produced from autologous iPS cells. To achieve this, adult fibroblasts were isolated from diseased animals and reprogrammed into iPS cells. Subsequently, the genetic defect in these iPS cells was repaired by homologous recombination, and these cells were differentiated into hematopoietic progenitor cells *in vitro*. Importantly, when these cells were transplanted back into the diseased host mice, animals were rescued from all haematological and systemic symptoms associated with sickle cell anaemia. After this initial study, the same group went on to report that iPS cell technology may hold great value for the treatment of Parkinson’s disease as well [22]. They showed that reprogrammed fibroblasts efficiently differentiated into neuronal and glial cell types *in vitro*. When transplanted into the fetal mouse brain, these cells functionally integrated into various brain regions. Moreover, they were able to generate iPS cell-derived dopaminergic neurons which, after transplantation into the adult brain, were able to restore dopaminergic function and improve behavioural symptoms in a rat model of Parkinson’s disease.

Since then, more studies have reported the derivation of therapeutically relevant cell types from iPS cells, including human insulin-secreting cells [20, 23], mouse and human functional cardiomyocytes [24, 25], and mouse endothelial cells that proved successful in treating a murine model of the coagulation disorder hemophilia A [26]. A number of recent studies demonstrated that iPS cells indeed can be generated directly from human patients suffering from Parkinson’s disease, type 1 diabetes mellitus, severe combined immunodeficiency, muscular dystrophy and Down syndrome [16]. iPS cells obtained from amyotrophic lateral sclerosis [15] and from spinal muscular atrophy [27] patients could be terminally differentiated into motor neurons. Apart from serving as a potential source for cell replacement therapies, iPS cells from patients with a genetic disorder may offer a unique tool to study how specific genetic aberrations give rise to the pathological development of specific cell types.

---

Clearing Obstacles on the Road to Clinical Application

Before iPS cells can be applied to treat human diseases, several technical obstacles need to be cleared. First,

selection for reprogrammed cells in culture generally requires the insertion of a drug-resistance marker gene into the genome of human cells, which is undesirable if these cells are to be used in the clinic. Interestingly, one of the earlier iPS cell studies already demonstrated that iPS cells could be selected from genetically unmodified human fibroblasts using cell morphology alone [5]. Alternatively, another study showed that culturing transduced fibroblasts in serum-free medium was sufficient to select for reprogrammed cells, a strategy based on the observation that ES cells tolerate knockout serum replacement (KSR), while mouse embryonic fibroblasts do not [28].

A second hurdle is the use of retroviruses for the transduction of reprogramming factors, since viral-mediated integration of transgenes into the host genome carries the risk of tumor formation by insertional mutagenesis. Most experiments in iPS cell research to date have used this method to reprogram somatic cells. Sommer and colleagues improved the reprogramming method by demonstrating that iPS cells can be produced from mouse fibroblasts using a single lentiviral vector expressing all four factors Oct4, Sox2, Klf4 and c-Myc [29]. In contrast to previous iPS cell experiments with separate viral vectors for each reprogramming factor, this method results in only a single retroviral insertion into the genome. The authors propose that this method may yield safer iPS cells, since the single viral copy may be removed from the iPS cell genome after reprogramming (e.g., by loxP/Cre technology). Indeed, successful excision of integrated transgenes in the generation of iPS cells has recently been shown using loxP/Cre technology [30]. It should be noted however, that a long terminal repeat (LTR) will remain behind after loxP/Cre excision and could disrupt or activate endogenous genes. Carey and colleagues subsequently demonstrated that the same method is also effective in reprogramming human somatic cells [31]. Recent studies demonstrate that iPS cells may be successfully obtained even without the use of integrating viruses. In one study, Stadtfeld and colleagues were able to produce iPS cells from adult mouse hepatocytes using non-integrating adenoviral vectors [32]. They demonstrated that this method allowed for transient expression of the exogenous reprogramming factors without genomic integration, thereby eliminating the risk of insertional mutagenesis. Okita et al. [33] and recently Kaji et al. [34] were able to successfully generate iPS cells without any viral vectors. They produced iPS cells by transfecting mouse fibroblasts with plasmids containing Oct4, Sox2, Klf4 and c-Myc, demonstrating convincingly that iPS cells can be produced by virus-free methods. Interestingly, Yamanaka recently suggested that replacing viral transfection methods may not be necessary at all. He proposes that viral integration sites could be determined by genome sequencing, and that safe iPS cell clones may be

obtained by excluding clones that show dangerous retroviral integration [35]. Anyway, the latest developments show that gene transfection may not even be needed at all for iPS generation, and that intracellular introduction of (recombinant) proteins of the 4 reprogramming factors could suffice for that: Zhou et al. designed and fused a poly-arginine (i.e. 11R) protein transduction domain to the C-terminus of Oct4, Sox2, Klf4 and c-Myc and were able to generate iPS cells from mouse embryonic fibroblasts [36]. Taken together, these results mark important steps towards the generation of safer iPS cells.

The use of oncogenes in the generation of iPS cells is another problem that hinders the therapeutic application of these cells. c-Myc is a well-established oncogene [37], and overexpression or potential reactivation of the transgene poses an important danger for future iPS cell therapies. Indeed, tumor formation has been observed in iPS cell-derived mice, possibly due to reactivation of c-Myc [6, 38]. As a major step towards solving this issue, several studies have demonstrated that mouse and human iPS cells can be derived without c-Myc, although its absence significantly reduced reprogramming efficiency [9, 14, 39]. Chimeric mice, generated from c-Myc-free iPS cells remained tumor-free after birth, while ~15% of the animals derived from four-factor iPS cells developed tumors [14]. As an alternative, Belloch and colleagues found that substituting c-Myc with n-Myc, a less tumorigenic member of the Myc family, yielded iPS cells with efficiency comparable to the original set of factors [28]. Unfortunately however, they did not investigate whether the use of n-Myc results in reduced tumor formation when compared to c-Myc. Although the oncogenic potential of c-Myc receives most attention in literature, Oct4, Sox2 and Klf4 are associated with multiple types of cancer as well [40–42]. To resolve this issue, a recent trend is to try and avoid the transduction of these oncogenes. One way to achieve this is by reprogramming somatic host cells that endogenously express sufficient levels of a reprogramming factor, thereby eliminating the need to introduce that factor. For example, several studies have taken advantage of the fact that neural progenitor cells (NPCs) endogenously express high levels of Sox2, and found that these cells can be reprogrammed by introduction of Oct4, Klf4 and c-Myc alone [43, 44], Oct4 and Klf4 alone [45–47], or even Oct4 alone [48], albeit with reduced efficiency. Similarly, meningiocytes and keratinocytes appear to be particularly susceptible to reprogramming, and this is attributed to the relatively high expression levels of Sox2, and c-Myc and Klf4, respectively [17, 18, 49]. A second way to avoid the use of oncogenes is by their replacement with small molecules. When Shi and coworkers generated iPS cells by transduction of Oct4 and Klf4 alone, they found that the reduced efficiency could be rescued when transduced NPCs were simultaneously treated
with the small molecule BIX-01294 (BIX), an inhibitor of the G9a histone methyltransferase [45]. In addition, they found that BIX enabled NPCs to be reprogrammed in the absence of exogenous Oct4, although in that case transduction of all three other factors (Sox2, Klf4 and c-Myc) was required. They concluded that BIX was able to functionally replace either Oct4 or c-Myc and Sox2, providing an exciting new venue for the production of safer iPS cells. Subsequent studies have confirmed the general principle of replacing oncogenes with small molecules. It was shown that efficient reprogramming can be achieved without c-Myc [50, 51], without both c-Myc and Sox2 [52], or without both Klf4 and c-Myc [19] when transfected cells were treated with the histone deacetylase inhibitor valproic acid, the DNA methyltransferase inhibitor 5-aza-cytidine, the Wnt signaling component WNT3a, or the L-channel calcium channel agonist Bayk8644 (BayK). In another study, combined inhibition of mitogen-activated protein (MAP) kinase signaling and the Wnt signaling component glycogen synthase kinase-3 (GSK3) also allowed for reprogramming in the absence of Sox2 and c-Myc [47]. Finally, small molecules such as 5-aza-cytidine may also be useful to increase the efficiency of reprogramming when using the ‘traditional’ four transcription factors [53]. Taken together, these studies demonstrate that small molecules are able to replace one or more reprogramming factors, providing promising new venues for the production of safer iPS cells.

and die after the blastocyst stage [3]. Moreover, pluripotent stem cells need to maintain fairly precise Oct4 protein levels to maintain their stem cell identity: reduced Oct4 expression in both mouse and human ES cells leads to the spontaneous differentiation into trophoblast lineages, while only a modest increase in Oct4 levels induces differentiation into primitive endoderm and mesoderm [56–58].

Additional roles for Oct4 have been reported as well. For example, Oct4 also functions to promote the survival of mammalian germline cells, as disruption of Oct4 expression in these cells leads to apoptotic death rather than differentiation [59]. In addition, Oct4 is a known oncogene, and its aberrant expression is implicated in dysplastic growth and the formation of several types of cancer (for review see [41] or [60]). Also, it has been reported that Oct4 levels influence the oncogenic capability of ES cells *in vitro*, as increased levels of Oct4 induced the cancerous transformation of nontumorigenic cells [56].

Sox2 Sox2, an SRY-related HMG-box (Sox) transcription factor, is highly expressed in the pluripotent lineage of the early embryo and, like Oct4, plays an important role in maintaining pluripotency [61]. In the early embryo, expression of Sox2 is restricted to the ICM, germ cells, and epiblast. In contrast to Oct4, however, Sox2 is also expressed in the extraembryonic ectoderm and in precursor cells in the early and adult nervous system [61, 62]. In ES cells, Sox2 has been found to form heterodimers with Oct4 to synergistically control the expression of ES cell-specific genes such as UTF1, Fgf4, and Fbx15 [2, 63, 64]. Furthermore, Sox2 is reported to regulate expression of Oct4 itself, and this is suggested to explain, at least in part, the importance of Sox2 in stem cell pluripotency [65].

Sox2 knockout embryos fail to develop epiblast and die at the implantation stage [61]. When cultured *in vitro*, Sox2-null blastocysts are incapable of proliferating and differentiate mainly into cells of the trophectoderm lineage. In addition, reduced levels of Sox2 in ES cells is associated with a loss of pluripotency and a tendency to differentiate [65, 66]. In the adult brain, Sox2 expression is required for the viability of neural stem cells, as reduced Sox2 levels lead to impaired neurogenesis and neurodegeneration [67].

Mutations of Sox2 genes are associated with a number of human pathologies. For example, Sox2 mutations are reported to be involved in anophthalmia-esophageal-genital (AEG) syndrome [68], and in abnormalities in the hypothalamo-pituitary axis [69]. In addition, Sox2 is implicated in the formation of a number of different cancers, including breast cancer, small-cell lung cancer, and prostate cancer [69, 70].

Molecular Mechanism of Induced Reprogramming

Although incredible “technical” progress towards safe iPS induction has been made over the last two years, as indicated above, much of the actual cellular and molecular mechanisms underlying induced pluripotency reprogramming still remains enigmatic. Combining existing knowledge about the four reprogramming factors and stem cell pluripotency with the analysis of the specific outcome of various iPS cell induction approaches, provides valuable mechanistic insights into the steps of the reprogramming process.

The Magical Four

Oct4 Oct4 (also known as Oct3, OTF3/4 or POU5F1) is a member of the octamer-binding (Oct) family of transcription factors and is a key regulator of stem cell pluripotency [54]. In both the developing mouse and human embryo, expression of Oct4 is confined to the inner cell mass (ICM), while in the adult expression is maintained in germ cells but is downregulated in differentiated somatic cells [54, 55].

The essential role that Oct4 plays in normal mammalian development has been demonstrated in Oct4 knockout embryos, as these embryos fail to develop a pluripotent ICM

Klf4 Klf4 is a member of the Krüppel-like factor (KLF) family of transcription factors, and is involved in a wide
range of cellular processes, including development, proliferation, differentiation, and apoptosis [72]. In the developing mouse embryo, expression of Klf4 follows a dynamic pattern, being first expressed in extraembryonic tissues, and subsequently in the gastrointestinal (GI) tract and in the developing skin layers of the embryo [73–75]. In adult tissues, Klf4 is mainly expressed in terminally differentiated epithelial cells of the GI tract and skin [74, 76]. Interestingly, Klf4 has been found to be highly expressed in non-dividing cells, while virtually no Klf4 can be detected in cells that actively proliferate [77].

Klf4 knockout embryos develop properly, but die shortly after birth due to a defect in skin barrier function [74]. Reduced Klf4 protein levels in ES cells do not induce an obvious phenotype, but simultaneous reduction of multiple Klf proteins causes differentiation, suggesting that other Klf proteins are able to compensate for Klf4 function [78, 79]. Additionally, Klf4 plays important roles in the regulation of proliferation, as induced expression of Klf4 in cultured cells leads to inhibition of DNA synthesis and cell cycle progression [77, 80].

Unlike Oct4 and Sox2, Klf4 has been identified as both a putative oncogene and a tumor suppressor gene [81], and as such, is particularly relevant in cancer formation. Indeed, Klf4 has been associated with various types of human cancer, including colorectal, breast, and esophageal cancers [82–86].

c-Myc The c-Myc oncogene is a multidomain transcription factor with important functions in cellular processes such as proliferation, differentiation, and cell growth [71, 87]. Global analyses showed that c-Myc is an important transcription factor, being involved in the transcriptional regulation of ~10% of the genome [71, 88]. Accordingly, estimates suggest that there may be up to 25,000 c-Myc binding sites in the human genome [89], many more than would be expected based on the c-Myc binding consensus sequence CACA/GTG. In addition to regulating the transcription of protein-coding genes, c-Myc is also implicated in the regulation of non-coding microRNA genes [90, 91].

c-Myc knockout embryos develop normally in the early stages of development. In later stages, however, c-Myc deficiency causes abnormalities in a variety of tissues, including the heart, neural tube, vasculature and blood cells, resulting in the death of c-Myc knockout embryos ~10 days after gestation [92, 93]. Paradoxically, c-Myc knockout ES cells exhibit normal proliferation and self-renewal *in vitro* [92]. Finally, elevated c-Myc expression is observed in ~70% of human tumors, making it one of the most frequently deregulated oncogenes in human cancer [71].

Stem Cell Pluripotency: Autoregulatory Loop of Oct4, Sox2 and Nanog

Over recent years, substantial evidence has emerged that suggests that pluripotency is governed by a trinity of Oct4, Sox2, and a third nuclear regulator, Nanog (for review see [94]). Nanog is a homeobox transcription factor that was recently discovered based on its ability to maintain ES cell self-renewal in the absence of leukemia inhibitor factor (LIF), the factor commonly used by researchers to maintain cultured ES cells in an undifferentiated state [4, 95]. Since its discovery, several studies have shown that Nanog is critically involved in the maintenance of pluripotency. For instance, Nanog-null mouse embryos fail to maintain a pluripotent epiblast, while disruption of Nanog *in vitro* results in an increased tendency of ES cells to differentiate spontaneously [4, 96]. In addition, decreased Nanog levels in ES cells cause the upregulation of transcription factors associated with trophectoderm differentiation such as GATA4, GATA6 and Cdx2, suggesting that Nanog may help sustain the pluripotent state by repressing these factors [97, 98]. Finally, Nanog is known to act as a strong activator of Oct4 [99].

An important breakthrough in ES cell research was achieved by two studies that used chromatin immunoprecipitation (ChIP) and genome-wide location analyses in both human and mouse ES cells to identify genes occupied by Oct4, Sox2 and Nanog [100, 101]. Both studies demonstrated that the three key regulators together targeted the promoters of their own genes, as well as those of each other. This suggested that Oct4, Sox2 and Nanog are able to form an interconnected autoregulatory loop to maintain their own expression (Fig. 1). Autoregulatory circuits are believed to enhance the stability of gene expression and

![Diagram](attachment://diagram.png)

Fig. 1 Oct4, Sox2 and Nanog form an interconnected autoregulatory circuit, by targeting both their own promoters and those of each other. In this way, the three key transcription factors are able to maintain their own expression, thereby contributing to the stable maintenance of pluripotency. Proteins are depicted as circles, gene promoters as rectangles. Adapted from [98]

reduce response time to external stimuli [102–104], and these properties may provide valuable mechanisms through which pluripotency can be stably maintained while allowing stem cells to respond to developmental signals [101]. Since the initial proposition, several studies provided strong empirical evidence for the existence of this autoregulatory loop and its importance in maintaining ES cell pluripotency [65, 66, 105–107].

Stem Cell Pluripotency: Transcriptional Network of Oct4, Sox2 and Nanog

In addition to binding their own promoters, Oct4, Sox2 and Nanog have been identified to target up to several hundreds of other genes in ES cells [100, 101]. In many cases, the three key regulators of pluripotency are found to co-occupy these genes, further suggesting that they act cooperatively to regulate stem cell pluripotency. Intriguingly, Oct4, Sox2 and Nanog were found to target both transcriptionally active and inactive genes (Fig. 2). Actively transcribed genes included various transcription factors (e.g. STAT3, ZIC3, REST, and SKIL), and chromatin-modifying enzymes (e.g. SMARCAD1, MYST3, and SET). In addition, genes of signal transduction components that are instrumental to ES cells (see below) were actively transcribed, including components of the TGF-β pathway (e.g. TDGF1, and Lefty2/EBAF) and Wnt pathway (e.g. DKK1,

and FRAT2). Importantly, these are all factors with known roles in maintaining ES cell self-renewal and pluripotency [100, 101]. Conversely, roughly 50% of genes targeted by Oct4, Sox2 and Nanog were genes that are inactive in ES cells, and whose expression is associated with lineage commitment [100, 101]. For a large part, these included transcription factors involved in differentiation and developmental processes (e.g. HOXB1, PAX6, MYF5, and MEIS1), suggesting that loss of Oct4, Sox2 and Nanog upon differentiation leads to upregulation of genes that are required for development [101]. In addition, the three factors are linked to several epigenetic complexes that are associated with transcriptional repression, including components of the NuRD histone deacetylase complex (P66b and HDAC2), proteins of the Polycomb group, and SWI/SNF chromatin remodeling complexes [108]. Taken together, the Oct4, Sox2 and Nanog trio appear to be master regulators of a transcriptional network that contributes to ES cell pluripotency by enhancing transcription of pluripotency genes, while at the same time repressing genes that are associated with differentiation and development (Fig. 2).

Given their importance in maintaining stem cell pluripotency, it can be expected that both the interconnected autoregulatory loop and the transcriptional network need to be revived in reprogrammed somatic cells. It will be discussed below that reactivation of these two systems

Fig. 2 The Oct4, Sox2 and Nanog trio contribute to ES cell pluripotency by repressing genes linked to lineage commitment, and activating genes involved in pluripotency. Active genes include, besides Oct4, Sox2 and Nanog themselves, ES cell-specific transcription factors, components of ES cell signaling cascades and epigenetic enzymes. In contrast, repressed genes comprise regulators of differentiation into the three germ layers and extra-embryonic development. Proteins are depicted as circles, gene promoters as rectangles. Adapted from [97, 98, 105]

![Diagram](https://i.imgur.com/placeholder.png)

Key pluripotency genes  
ES cell transcription factors  
ES cell signaling  
Epigenetic regulators  
Inhibitor of neurogenesis  
Telomere-associated protein  

Ectoderm development  
Mesoderm development  
Endoderm development  
Extra-embryonic development

indeed represents a crucial event in the process of induced reprogramming.

Stem Cell Pluripotency: Epigenetic Signature

Embryonic stem cells carry several epigenetic characteristics that distinguish them from differentiated cells and that are thought to contribute to stem cell identity. For example, key pluripotency genes such as Oct4 and Nanog are free of DNA methylation in ES cells and can be actively transcribed, while lineage commitment results in silencing of these genes by *de novo* DNA methylation [109]. Interestingly, as will be discussed later on, these methylation marks are removed during the reprogramming process, allowing transcription of these genes to be reactivated.

In addition to DNA methylation, histone modification patterns also differ between ES cells and differentiated cells. For example, the repression of developmental genes in ES cells is regulated in a remarkable way. Paradoxically, both activating H3K4me3 and repressive H3K27me3 histone modifications are found at these genes [110, 111]. The H3K4me3 marks of these ‘bivalent’ domains allow transcription to be initiated at these genes, but transcription elongation is obstructed by proteins of the Polycomb group (PcG)—an evolutionarily conserved family of proteins that regulate expression of developmental genes through gene silencing—that bind to the repressive H3K27me3 mark [112]. Intriguingly, most silenced developmental genes that are regulated by Oct4, Sox2 and Nanog in ES cells are also targeted by PcG proteins, indicating an important role for PcG proteins in ES cell pluripotency [110, 113]. Thus, genes involved in lineage commitment can be viewed as ‘cued’ for rapid activation during cellular differentiation, upon which repressive H3K27me3 marks are lost and transcription can be initiated. In this way, bivalent domains are believed to be instrumental to the transcriptional flexibility of ES cells by allowing developmental genes to be stably repressed without irrevocably silencing them [114]. Given that bivalent domains are virtually exclusive to ES cells and that they are an important characteristic of the pluripotent state, it is tempting to assume that the restoration of bivalent domains represents a key step in the reprogramming of somatic cells into iPS cells. In this way, lineage-associated genes that have been activated during differentiation and have lost repressive H3K27me3 marks may be re-silenced when these bivalent domains are restored. Interestingly, it has been reported that the chromatin of fully reprogrammed iPS cells carries bivalent histone marks that are virtually identical to ES cells [5, 7, 8, 53], indicating that the bivalent profile is indeed restored during reprogramming.


Stem Cell Pluripotency: Self-renewal vs. Lineage Commitment

ES cells are capable of responding to a large variety of signal molecules, and these signals may contribute to either the maintenance or loss of stem cell identity. For example, it was previously mentioned that LIF, a member of the interleukin-6 (IL6) family, can promote the undifferentiated state of ES cells in culture [115]. Signaling through LIF involves the recruitment of JAK kinases and activation of the transcription factor STAT3, which then induces activation of pluripotency genes [116]. Other signaling pathways, such as the Wnt/β-catenin and the TGFβ/activin/nodal pathways, are important contributors to ES cell pluripotency as well (for review see [117]). In contrast, other molecules have been identified that cause ES cells to differentiate. For instance, BMP4 is known to induce differentiation of human ES cells along the trophoblast lineage, a process that depends on members of the SMAD family of nuclear proteins and that leads to downregulation of Oct4 and Nanog [118]. Notably, BMP4 has the exact opposite effect in mouse ES cells, where it supports self-renewal [119], suggesting that mechanisms involved in the maintenance of pluripotency may differ considerably between species. ES cell signalling cascades have come to be relevant in the context of reprogramming, as it has been found that the Wnt signalling component Wnt3a is able to functionally replace c-Myc from the reprogramming cocktail [51]. Possibly, other components of the Wnt pathway, or components of other signalling pathways, may soon prove similarly useful to the production of iPS cells.

Inducing Stem Cell Pluripotency: Reprogramming Factors

Although initial iPS cell studies used the traditional quartet of Oct4, Sox2, Klf4 and c-Myc to generate iPS cells from somatic cells, research efforts over the past two years have turned up an increasing variety of additional factors that are able to contribute to the reprogramming process (see Clearing Obstacles on the Road to Clinical Application). Analysis of the specific functions and downstream pathways of each single factor and combinations of them provides valuable insight into the molecular process of induced reprogramming.

Oct4 and Sox2 The interconnected autoregulatory circuit of Oct4, Sox2 and Nanog and the core transcriptional network of downstream proteins described above may provide an appealing explanation for how iPS cells are produced from differentiated cells. When exogenous Oct4 and Sox2, together with other factors, are introduced into somatic cells, they may directly induce the expression of endogenous Oct4, Sox2 and Nanog through the autoregulatory

circuit, after which these factors contribute to the induction of pluripotency by both maintaining their own expression and activating the pluripotent transcriptional network ([120]; Fig. 3). In this way, the autoregulatory circuit and the transcriptional network that are inactivated in somatic cells, are ‘resuscitated’ during reprogramming to establish the pluripotent state. Experimental evidence for this model is provided by studies demonstrating that continuous retroviral transgene expression is not required for the maintenance of pluripotency in iPS cells, but is only essential for the induction of pluripotency [5–7, 121]. Indeed, numerous studies demonstrate that retroviral gene

![Diagram](attachment:diagram.png)

Fig. 3 Exogenous Oct4 and Sox2 resuscitate the interconnected autoregulatory loop during reprogramming. In infected fibroblasts, endogenous Oct4, Sox2 and Nanog are reactivated by ectopic expression of Oct4, Sox2 and other reprogramming factors. Progression of the reprogramming process results in increased self-sustainability of the endogenous interconnected loop. Meanwhile, transgene expression is gradually silenced by *de novo* DNA methylation, indicating that exogenous factors are required only for the induction of pluripotency. The interconnected loop is now resuscitated and is able to stably maintain the pluripotent state. Proteins are depicted as circles, gene promoters as rectangles. Adapted from [117]

expression, although high in infected fibroblasts, becomes robustly silenced in fully reprogrammed iPS cells by *de novo* DNA methylation (for review see [122]). This implies that somewhere along the reprogramming process, endogenous expression of Oct4, Sox2, Nanog and other pluripotency factors has been sufficiently reactivated to ensure maintenance of the pluripotent state, independent of transgene expression.

The fact that Nanog is not an essential reprogramming factor is surprising, since it is a core component of the interconnected autoregulatory circuit in pluripotent stem cells. One possible explanation is that ectopic overexpression of Oct4 and Sox2 is sufficient to induce expression of endogenous Nanog through the autoregulatory loop [100, 101]. Besides Oct4 and Sox2, other reprogramming factors, including Klf4, may also directly contribute to the expression of endogenous Nanog (see below).

$c$-Myc In contrast to the direct relationship between Oct4 and Sox2 and the pluripotent state, the role of other factors, including $c$-Myc, in induced reprogramming is more enigmatic. $c$-Myc is known to fulfil a variety of roles in different cellular processes that may prove relevant in the context of reprogramming. For example, $c$-Myc is a key component of active chromatin and associates with several histone acetyltransferase (HAT) complexes, including p300, CREB binding protein, and TRRAP, a component of the GCN5 and TIP60 HAT complexes [123, 124]. Consequently, it is thought that an important reprogramming function of $c$-Myc is to facilitate an open chromatin conformation through global histone acetylation, thereby providing Oct4 and Sox2 access to their target loci [46, 125]. Another model posits that $c$-Myc, as a well-known oncogene, induces a cancer-like transformation in somatic cells [126]. $c$-Myc is known to induce expression of proliferation genes such as p15, p21 and cyclin E, which may accelerate cell cycling [127, 128]. This may be a key event in the reprogramming process, as cellular division may provide cells with a valuable window of opportunity to re-establish transcriptional programmes (for review see [129]). For example, the displacement of transcriptional regulators (such as transcription factors, chromatin remodelling enzymes and Polycomb proteins) from their target loci and disruption of the spatial localization of chromosomes within the nucleus during mitosis allow cells to establish genome-wide changes in gene expression. Indeed, the generation of iPS cells appears to be more efficient with rapidly dividing cells [10]. In addition, it has recently been found that $c$-Myc enhances DNA synthesis, independent of its transcription factor activity, by causing increased replication origin activity [130]. Thus, $c$-Myc-induced DNA replication and cell cycle progression may provide dividing cells with opportunities to reset their epigenome,
aiding in the reprogramming to pluripotency. Finally, it has been observed that c-Myc directly upregulates expression of TERT, the gene encoding the enzymatic subunit of the telomerase [131]. Somatic cells that have been completely reprogrammed to a pluripotent state show elevated telomerase activity similar to ES cells [1, 9], and this may be a consequence of c-Myc activity. Interestingly, it was shown that the addition of TERT to the traditional four factor cocktail increased efficiency of reprogramming [10], suggesting that TERT activation, possibly through c-Myc, is important in the reprogramming process.

Klf4 Several possible roles of Klf4 in induced reprogramming have been suggested. First, Klf4's role may be to counteract the apoptotic effects of c-Myc. Sustained expression of c-Myc induces p53-dependent apoptosis, and Klf4 may be required to prevent this process by suppressing p53 [132]. However, Klf4 activity in turn causes suppressed proliferation through the activation of p21 [133], and c-Myc may inhibit this anti-proliferative effect of Klf4 by suppressing p21 [134]. Therefore, it is believed that the balance between c-Myc and Klf4 activity may play an important role in reprogramming [126]. A second role for Klf4 in reprogramming may be as a cofactor of Oct4 and Sox2. It is known that in differentiated cells, Oct4 and Sox2 together are unable to induce expression of ES cell-specific genes (such as Lefty1), while in the presence of Klf4 expression of these genes does occur [78]. This indicates that Klf4 may help Oct4 and Sox2 to activate pluripotency genes in differentiated cells undergoing reprogramming. Furthermore, Klf4 has been implicated in the activation of Nanog, although in this case Klf4 functions through p53 [132]. p53 has been reported to repress Nanog expression during differentiation [135], and Klf4 may thus function to activate endogenous Nanog expression by inhibiting p53 activity. Finally, similar to c-Myc, Klf4 associates with the p300 histone acetyltransferase [136], and may facilitate the reprogramming process by modulating accessibility of the Oct4 and Sox2 target loci.

Importantly, Klf4 and c-Myc have been found to be dispensable for reprogramming, although reprogramming efficiency has been observed to be substantially lower in their absence [9, 10, 14, 19, 39, 45]. This has led researchers to believe that Oct4 and Sox2 may be considered to be the true 'reprogrammers', and that c-Myc and Klf4 may act as enhancers of efficiency by facilitating activation of crucial pluripotency genes. Since it is known that forced expression of Oct4 can eliminate the need for Sox2 in maintaining pluripotency [62], some researchers even believe that Oct4 is in fact the only essential factor in the reprogramming of somatic cells [48]. However, this presumption has become disputable, as the aforementioned

study by Shi and colleagues has shown that iPS cells can be produced even in the absence of Oct4 [45].

Lin28 As was mentioned in Potential Clinical Applications of iPS Cells, human primary fibroblasts can be reprogrammed into iPS cells by substituting c-Myc and Klf4 in the original cocktail by Nanog and Lin28 [9]. The ways in which Nanog may contribute to the reprogramming process have been discussed above. Much less, however, is known about the possible role of Lin28 in the induction of pluripotency. Lin28 possesses multiple RNA-binding domains and localizes both to the nucleus and to the cytoplasm, where it is associated with sites of mRNA and miRNA regulation called processing bodies, or P-bodies [137]. It has been suggested that Lin28 may facilitate the reprogramming process by stabilizing mRNA transcripts from genes that are required to establish the pluripotent transcriptional network [128]. In addition, it has recently been found that in ES cells, Lin28 prevents the processing of let-7 miRNAs, a family of miRNAs that is known to stimulate differentiation of cancer cells [138, 139]. This suggests that Lin28 may promote reprogramming by preventing let-7-dependent differentiation in transfected somatic cells.

Small compounds in reprogramming A variety of small compounds have been found to contribute to the reprogramming of somatic cells, most of them interfering with the activities of repressive epigenetic enzymes. For example, the small compound BIX-01294 (BIX), an inhibitor of the G9a histone methyltransferase, was found to facilitate reprogramming in neural progenitor cells transfected with only Oct4 and Klf4 [45]. As endogenous expression of Sox2 in NPCs makes viral transduction of this factor dispensable for reprogramming [43, 44], it can be concluded that BIX is able to functionally replace c-Myc in generating iPS cells. Importantly, BIX is known to reduce levels of H3K9me2 at genes that are targeted by G9a, including pluripotency genes such as Oct4 [109]. Since H3K9 methylation of pluripotency genes is associated with their epigenetic silencing genes during differentiation, the G9a-inhibitor BIX may aid in reprogramming by promoting a permissive chromatin state and active transcription of these key pluripotency genes [45].

Similar to BIX, the histone deacetylase inhibitor valproic acid (VPA) was also found to rescue reprogramming efficiency when c-Myc (together with Klf4) was omitted from the classic reprogramming cocktail [19, 50]. In fact, application VPA to fibroblasts transfected with only Oct4 and Sox2 yielded efficiencies that were superior to cells transduced with all four classic factors. Interestingly, the authors found by microarray analysis that treating fibroblasts with VPA alone, i.e. without any reprogramming factors, already resulted in the upregulation of embryonicgenes and downregulation of fibroblast-specific genes [50]. As inhibition of histone deacetylation results in a more open chromatin conformation, it is believed that a role of VPA in reprogramming may be to increase chromatin accessibility to transcriptional regulator proteins. Since one of the presumed roles of c-Myc in reprogramming is to increase accessibility of pluripotency genes by opening up chromatin structure, BIX and valproic acid may act by functionally replacing c-Myc.

As a third modulator of epigenetic regulation, the DNA methyltransferase inhibitor 5-aza-cytidine was found to enhance reprogramming efficiency as well [53]. In transfected fibroblasts, 5-aza-cytidine treatment was found to increase reprogramming efficiency, but only when treatment was started no earlier than 8 days post-transfection. Application of the compound prior to that time point resulted in increased cell death instead of enhanced efficiency. The authors suggest that this effect may be caused by the fact that a large proportion of transfected cells still resemble differentiated cells more than reprogrammed cells at this early time point, and global hypomethylation in differentiated cells is known to induce apoptosis [140].

Finally, it has recently been shown that ectopic expression of c-Myc can be replaced by culturing transfected cells in media containing Wnt3a, a component of the Wnt/β-catenin pathway [51]. The efficiency-boosting effect of Wnt3a was abolished in the presence of ICG-001, a compound that selectively represses genes downstream the Wnt/β-catenin pathway by disrupting the interaction between β-catenin and CREB-binding protein (CBP). Marson and co-workers provide several possible mechanisms by which stimulation of Wnt signaling may promote the generation of iPS cells. For example, Tcf3, a transcription factor downstream of the Wnt pathway in ES cells, is known to regulate the promoters of key pluripotency genes, including Oct4, Sox2 and Nanog. In reprogramming, the Wnt3a-mediated Tcf3 activity may thus potentiate the effect of exogenous Oct4, Sox2 and Klf4 in reviving the pluripotency network. In addition, it is well-known that c-Myc is a downstream target of the Wnt/β-catenin pathway in cancer cells, and treating transfected cells with Wnt3a may induce activation of endogenous c-Myc to facilitate reprogramming.

All these results suggest strongly that an important reprogramming role of the presently known small compounds is the promotion of an epigenetic state that, similar to that in ES cells, is permissive for active transcription of pluripotency genes. Excitingly, this would predict that other small compounds that have similar effects on the epigenome (e.g. activators of histone acetyltransferases) can be exploited in efforts to increase reprogramming efficiency and replace oncogenic transgenes.

Inducing Stem Cell Pluripotency: Epigenetic Changes

In *Stem Cell Pluripotency: Epigenetic Signature*, it was discussed that a key event in reprogramming may be the restoration of repressive bivalent domains (i.e. chromatin regions bearing both activating and repressive histone modifications) at loci containing lineage-associated genes. In contrast, chromatin of pluripotency genes must undergo different modifications, since these genes must be reactivated rather than silenced. It is thought that repressive H3K9me3 marks that were deposited in a G9a histone methyltransferase-dependent manner on Oct4 (but not Nanog and Sox2) upon lineage commitment [109], are removed from the Oct4 locus during reprogramming. Although the underlying mechanism is unclear, it has been shown that Oct4 regulates transcription of the histone H3K9 demethylases Jmjd1a and Jmjd2c [141]. Possibly, exogenous Oct4 is able to induce expression of these demethylases, that in turn remove the repressive H3K9me3 mark from endogenous Oct4 loci.

Additionally, it has been shown that in ES cells the autoregulatory circuit is linked to (repressive) epigenetic enzymes, including the nucleosome remodelling and deacetylase (NuRD) complex, Polycomb proteins and SWI/SNF chromatin remodelling complexes [108]. It was suggested that these repressive enzymes contribute to the maintenance of pluripotency in ES cells by preventing transcription of genes associated with differentiation. In the context of reprogramming, the revived autoregulatory loop of Oct4, Sox2 and Nanog may activate these enzymes to repress lineage-associated genes in transfected somatic cells, thereby facilitating their reprogramming to iPSCs.

Induced reprogramming is associated with alterations in DNA methylation profiles as well. Pluripotency genes such as Oct4, Sox2 and Nanog are repressed by *de novo* DNA methylation upon differentiation, and these methylation marks are removed during the reprogramming process [1, 5–10]. In fact, the degree of demethylation at these loci may indicate the progression and faithfulness of reprogramming, as incompletely reprogrammed Fbx15-selected iPSCs display only partial demethylation of the Oct4 promoter [1]. Nevertheless, the mechanism by which DNA methylation is lost at these key genes remains elusive. Given the slow dynamics and the requirement for multiple rounds of cellular division in the reprogramming process [120], it is thought that a lack of methylation maintenance on newly synthesized DNA strands is responsible for the loss of these marks. Such progressive loss of DNA methylation over multiple rounds of cellular division would then allow reactivation of endogenous pluripotency genes to levels that are sufficient to regenerate and maintain the autoregulatory loop. Although the exact mechanism responsible for this methylation loss is unknown, it perhapsinvolves the inhibition of the maintenance DNA methyl-
transferase DNMT1. Consistent with this concept, it was
found that specific knockdown of DNMT1 by RNA
interference in transfected cells facilitates the transition to
pluripotency [53].

Inducing Stem Cell Pluripotency: An Inefficient
and Gradual Process

Already in their breakthrough study in 2006, Takahashi and
Yamanaka observed that only very few cells expressing all
four transduced factors became iPS cells [1]. This low
reprogramming efficiency was corroborated by subsequent
studies [6, 8, 46, 142]. Using the original protocol of
transfecting four transcription factors, overall reprogram-
ming efficiency is typically observed to be less than 0.1%,
which is lower than when cells are reprogrammed using ES
cell fusion or nuclear transfer [143]. Several explanations
have been suggested for this low efficiency. First, the
generation of iPS cells may require specific, narrow-range
expression levels of the transduced factors, and by chance,
only a small proportion of transfected cells may acquire
these precise levels. Variation in expression levels may be
the result of their random retroviral integration into
unselected sites or it may be caused by differences in copy
numbers of viral insertions [144]. In ES cells, it is known
that pluripotency depends on tightly regulated levels of
pluripotency genes. For example, a 50% change in Oct4
levels causes ES cells to differentiate [57]. This implies that
Oct4 levels should be regulated within narrow ranges
during reprogramming, as premature silencing of retroviral
transgenes could allow partially reprogrammed iPS cells to
revert back to the differentiated state of the target cell,
while uncontrolled expression would induce the immediate
differentiation of reprogrammed iPS cells [122]. When
using the classic set of four reprogramming factors, specific
levels of Klf4 and c-Myc may be important as well because,
as was discussed above, the delicate balance between these
factors is thought to be important in reprogramming. Thus,
appropriate control of transgene expression may be a rare
event, contributing to the low efficiency of the reprogram-
ming process.

A second explanation may be that retroviral integration
may cause activation of additional, unknown reprogram-
ming genes by insertional mutagenesis. iPS cells carry
multiple viral integrations at random sites, and very few
cells may carry insertions at the appropriate sites to activate
these additional genes. However, two recent studies argue
against this view, as they demonstrate that the successful
generation of iPS cells does not require viral transgene
integration into specific sites [38, 145]. Further compelling
evidence against this model has been provided by recent
studies showing that reprogramming of somatic cells by

defined factors can be achieved by non-integrating meth-
ods, such as plasmid transfection [33] or non-integrating
adenoviruses [32].

Yet another explanation for low reprogramming effi-
ciency may lie in the fact that fibroblast populations are
heterogeneous and contain cells that are more susceptible
to reprogramming (for example because they, by chance,
possess a relatively open chromatin organization that
facilitates reprogramming). However, this is unlikely to
be a universal cause for low reprogramming efficiency, as
it has been shown that other cell types, such as pancreatic
β cells and cells from stomach and liver, can be
reprogrammed at frequencies comparable to fibroblasts
[38, 142].

The activation of stress-associated and anti-proliferative
genes as a consequence of the reprogramming process may
also put a constraint on reprogramming efficiency. For
instance, expression of Cdkn1a and Cdkn2a, two cyclin-
dependent kinase inhibitors that are involved in various
differentiation and anti-proliferative pathways, was ob-
served to be substantially upregulated in transfected
fibroblasts [53]. This may be a direct consequence of the
introduction of exogenous transcription factors, as Cdkn1a
is targeted by Klf4 [39] and Cdkn2a is activated by aberrant
c-Myc expression [146]. These data imply that the ectopic
expression of reprogramming factors induces intrinsic
safety mechanisms that serve to suppress uncontrolled
proliferation [53]. Thus, opposing forces may be at play
in transfected cells, and chance may dictate that only a
small percentage of cells are able to overcome the anti-
proliferative response and achieve the pluripotent state.

Finally, an explanation for the divergent efficiencies of
reprogramming by defined factors, ES cell fusion and
nuclear transfer may lie in technical differences [129]. In
ES cell fusion and nuclear transfer, the somatic cell or
nucleus is exposed to all components of the pluripotent
transcriptional network, whereas in transcription factor-
induced reprogramming cells are exposed to only a small
subset of these factors. These factors must reactivate the
transcriptional network ‘from scratch’, and this process may
be more susceptible to chance variation than the reprog-
ramming process in ES cell fusion or nuclear transfer.

It can be concluded that transcription factor-induced
reprogramming is a inefficient and gradual process, and that
the process likely depends on rare, stochastic events.
Indeed, the importance of chance variation in reprogram-
ming is corroborated by several findings. For example,
partially reprogrammed cells are commonly observed to be
rather heterogeneous in features such as global expression
profile and morphology [5, 6, 11]. Moreover, iPS cells
originating from the same parental cell reactivate endoge-
nous Oct4 expression at different time points during the
reprogramming process [121], demonstrating that generat-
ing iPS cells is a gradual process that involves stochastic changes.

Inducing Stem Cell Pluripotency: Distinct, Sequential Events

The importance of stochasticity in reprogramming has led researchers to question whether restoration of the pluripotent state takes place as a random process or is guided through a set of specific sequential events. To address this issue, two recent studies used inducible lentiviral vectors to study intermediate stages of the reprogramming process [147, 148]. In both studies it was found that the generation of iPS cells occurs through a predictable succession of events. The first observed changes in transfected fibroblasts (~3 days after induction of viral expression) were the downregulation of fibroblast-specific genes such as Thyl and upregulation of the ES cell-specific markers alkaline phosphatase and Fbx15 (Fig. 4). Subsequently, after 5–9 days, these cells displayed a gradual upregulation of the ES cell marker gene SSEA1. Modest endogenous expression of the pluripotency genes Oct4, Sox2 and Nanog was observable as early as 10–16 days after transgene induction (Fig. 4). Interestingly, upregulation of these pluripotency genes closely correlated with other changes, such as reactivation of telomerase and the silenced X chromosome [148]. In addition, at this stage in the reprogramming process cells became increasingly independent of viral transgene expression. However, transfected cells were not yet fully reprogrammed at this timepoint, as downregulation of transgene expression resulted in reversion to the fibroblast state [148]. Brambrink and colleagues propose that the reprogramming process may be initiated in distinct cells at different timepoints, or that cells require different times to complete the reprogramming process. Prolonged expression of viral vectors may therefore allow a larger portion of cells to go through the stochastic changes to reach a pluripotent state [147]. Ultimately, viral vectors are completely silenced in fully reprogrammed cells, most likely by *de novo* DNA methylation. Taken together, these data suggest that reprogramming by defined factors is a gradual, stochastic process with distinct intermediate cell populations.

The observation that reactivation of endogenous Oct4, Sox2 and Nanog expression is a relatively late reprogramming event is intriguing, as it renders the previously discussed ‘resuscitation model’ (i.e. reactivation of the autoregulatory circuit by ectopic expression of Oct, Sox2 and other factors) too simplistic. Although Oct4, Sox2 and Nanog are able to bind to their own promoters and upregulate each others expression [100, 101], their activation only late in the reprogramming process demonstrates that this does not occur in the earliest stages of the reprogramming process. For these endogenous loci to be reactivated, demethylation of their promoters is a prerequisite [5, 7–10]. As discussed above, the removal of DNA methylation marks requires multiple rounds of cell division, and this may explain why reactivation of endogenous Oct4,

![Diagram](attachment:diagram.png)

Fig. 4 Transcription factor-induced reprogramming takes place through a predictable sequence of steps. The first observable event is represented by the downregulation of lineage-specific genes. Concomitantly, ES cell genes such as alkaline phosphatase and Fbx15 are gradually upregulated, followed by SSEA1. Next, endogenous Oct4 and Nanog are upregulated, and this event closely correlates with reactivation of both telomerase and the inactive X chromosome. The upregulation of endogenous Oct4 and Nanog marks an increasing independence on transgene expression, and ultimately leads to the complete silencing of retroviral vectors. Adapted from [144, 145]

Sox2 and Nanog only occurs at a progressed stage in the reprogramming process.

A Two-stage Model of Induced Pluripotency Reprogramming

Based on the data discussed in the previous sections, we can propose that the process of reprogramming by defined factors consists of two broad stages (Fig. 5). The broadness of the two stages may be considered somewhat arbitrary as there are several sub-events embedded in each proposed stage. In brief, the first stage serves to repress lineage-associated genes and, moreover, to reset the epigenome of the target cell to a ES cell-like ground state. In the second stage, this chromatin state will allow exogenous reprogramming factors to reactivate both the endogenous interconnected autoregulatory loop and the core transcriptional network that underlie pluripotency.

One of the earliest events in the first stage is the repression of lineage-associated genes, and several mechanisms are likely to be involved. For example, developmental genes that are targeted and inactivated by Oct4, Sox2 and Nanog (e.g. Pax6, Meis1, HoxB1, Esx1l), may be directly repressed by exogenous Oct4 and Sox2. In addition, expression of repressive epigenetic enzymes (such as the NuRD and the Setdb1 complexes) that are under the control of Oct4, Sox2 and Nanog, may be induced by exogenous Oct4 and Sox2 as well. These enzymes may then restore bivalent histone marks on lineage-associated genes, that subsequently recruit repressive Polycomb proteins whose expression is also induced by Oct4 and Sox2. Concomitant with the silencing of lineage genes is the upregulation of a subset of ES cell-specific genes, such as Fbx15, alkaline phosphatase and SSEA1. The reason that these genes can be activated this early in the reprogramming process is unclear, but it may be that, in contrast to pluripotency genes such as Oct4, Sox2 and Nanog, these genes possess a relatively accessible chromatin state, even in differentiated cells. Possibly, other ES cell-specific genes with permissive chromatin conformations may also be gradually upregulated by exogenous Oct4 and Sox2, such as some transcription factors and components of signal transduction pathways. It may be hypothesized that the combined effect of upregulation of these ES cell genes and repression of lineage-associated genes is already sufficient to produce the ‘quasi-pluripotent’ phenotype that is commonly observed in reprogramming experiments: transfected cells display ES cell characteristics (e.g. ES cell-like morphology and colony formation), but quickly regress back to the differentiated state when transgene expression is stopped. Finally, a key function of the first stage is to establish a permissive chromatin state to allow transcription

---

**Somatic cell**  
![Image Description](#)  
**Partially reprogrammed cell**  
![Image Description](#)  
**iPS cell**  
![Image Description](#)

**STAGE 1**  
- Downregulation of lineage genes  
  - Direct repression  
  - Restoration of bivalent domains  

- Activation of specific ES cell genes  
  - E.g. SSEA1, Fbx15 and AP  

- Chromatin remodelling at pluripotency genes  
  - Unfolding of condensed chromatin  
  - Removal of repressive chromatin marks  

**STAGE 2**  
- Resuscitation of autoregulatory loop  
- Full reactivation of ES cell transcriptional network  
  - Reactivation of telomerase, ES cell signal cascades, etc.  
- Completion of transgene silencing  

---

Fig. 5 Reprogramming by defined factors may be viewed as a two-stage process. In the first stage, exogenous Oct4 and Sox2 cause the downregulation of lineage-associated genes and upregulation of a subset of ES cell-specific genes. In addition, the first stage of reprogramming comprises widespread epigenetic remodelling: epigenetic enzymes that are most likely activated by reprogramming factors induce a global unfolding of chromatin and catalyze the removal of repressive chromatin modifications from key pluripotency genes. This allows these pluripotency genes to be targeted and activated by exogenous Oct4 and Sox2, resulting in the revival of the interconnected autoregulatory loop and reactivation of the ES cell transcriptional network. At the same time, transgene silencing that was initiated during the first reprogramming stage reaches completion, leading the fully re-established pluripotent state to be independent from continuous transgene expression.
of the full array of ES cell-specific genes. At a global level, the open chromatin conformation is likely the result of reprogramming factors such as c-Myc, Klf4 or small compounds. Also, histone modification enzymes such as the histone acetyltransferase Myst3 and the chromatin remodelling protein Smarcad1 may be activated directly by exogenous Oct4 and Sox2 to aid in the general unfolding of chromatin. At the same time, multiple rounds of cellular division lead to the progressive loss of DNA methylation marks at promoters of key pluripotency genes, including Oct4, Sox2 and Nanog. This process possibly involves inhibition of DNMT1 by unknown factors that are activated by exogenous Oct4 and Sox2. In addition to DNA methylation, repressive H3K9me3 histone marks need to be removed from pluripotency genes, and histone demethylases (such as Jmjd1a and Jmjd2c) that are activated by transduced Oct4 may be involved in this process.

Once completed, the removal of repressive epigenetic marks from key pluripotency genes allows for the full reactivation of the interconnected autoregulatory loop and core transcriptional network in the second stage of induced reprogramming. Exogenous Oct4 and Sox2 are now able to target and activate endogenous Oct4, Sox2 and Nanog loci, and expression of these pluripotency regulators will be self-maintained through the autoregulatory loop. In addition, the completion of epigenetic remodelling at other pluripotency genes results in the gradual revival of the full transcriptional network that is characteristic of the pluripotent state. Components of this network and their downstream targets will then achieve full-blown pluripotency, exemplified by reactivation of telomerase, the inactivated X chromosome and ES cell signalling cascades. Finally, transgene silencing that was initiated at the end of the first stage comes to completion, and the pluripotent state has become completely dependent on the endogenous autoregulatory circuit.

Discussion

Since iPS cells were first generated in 2006, reprogramming by defined factors has been heralded as the technology that would revolutionize human medicine: now, pluripotent cells could be derived from each individual patient and be used for autologous cell therapy, making tissue rejection and the use of human embryos a thing of the past. However, iPS cell technology has its own hurdles, and these need to be overcome if iPS cells are to be applied in human therapy.

Some of these hurdles have already been—or are close to being—solved, while others remain a problem (see Clearing Obstacles on the Road to Clinical Application). For example, Oct4 is observed in most studies to be indispensable in reprogramming experiments, but is also a known oncogene. Ideally, Oct4 should be deleted from the reprogramming cocktail. Shi and colleagues [45] already found that the G9a histone methyltransferase inhibitor BIX may substitute Oct4. However, the transduction of the three other oncogenes—Sox2, Klf4 and c-Myc—was still required to successfully reprogram somatic cells. Another recent study reported that inhibition of DNA methylation and histone acetylation could induce the reactivation of several endogenous pluripotency genes, including Oct4 [149]. As the need for exogenous Sox2 may be obviated by using target cell types that endogenously express this reprogramming factor (e.g. neural progenitor cells or meningiocytes), and Klf4 and c-Myc are dispensable for reprogramming, it may soon be possible to generate iPS cells in the absence of oncogenic reprogramming factors.

Additionally, generating patient-specific cells for transplantation therapy is expected to be extremely expensive and time-consuming, as the entire process is estimated to encompass roughly two years [150]. This would not only render iPS cell technology too expensive to be applied on the basis of individual patients, but it would also make the technology useless for disorders that demand rapid treatment, such as spinal cord injuries. To solve these issues, efforts are currently being undertaken to establish libraries of a collection of iPS cell lines with known genotype. Although these cells would not be patient-specific, careful selection of genotypes is expected to yield cells with a close genetic match for the largest part of the population at considerably reduced costs.

Although the potential use of iPS cells for human cell replacement therapy receives most attention, other valuable applications should not be overlooked. As indicated before, iPS cells derived from patients suffering from genetic disorders may revolutionize research on these disorders, since they may provide a tool to study in detail the development and mechanism of specific cell pathologies. Alternatively, iPS cell libraries may be used by the pharmaceutical industry to greatly facilitate drug development. Specific cell types that are relevant to drug efficacy and metabolism could be derived from these iPS cell lines and be used to test drug effects and toxicity early in the process of drug development. Cancer research may benefit from iPS cell technology as well, as a thorough understanding of the reprogramming process may provide important insights into processes of de-differentiation known to be involved in carcinogenesis. Moreover, iPS cells may be generated from tumour tissue to study the involvement of specific genetic traits in the process of cancer formation.

iPS cell research has taken an astonishing, jubilant flight. However, it is crucial to take various criticisms into serious consideration. One of the earliest point of debate has been the actual origin of iPS cells. The low efficiency of the
reprogramming process led Takahashi and Yamanaka to question whether their iPS cells truly originated from differentiated fibroblasts, rather than from rare tissue stem or progenitor cells that coexisted in the fibroblast cultures [1]. They discarded this possibility, arguing that “the four factors induced iPS cells with comparably low efficiency even from bone marrow stroma, which should be more enriched in mesenchymal stem cells and other multipotent cells”. Not fully satisfied with this explanation, subsequent studies addressed this issue. Aoi and colleagues applied a lineage-tracing strategy and confirmed that hepatocyte-derived iPS cells indeed originated from cells expressing the hepatic albumin gene [38]. However, in a subsequent study, Hanna and colleagues challenged the validity of this work, arguing that albumin expression is not exclusive to hepatocytes but also observed in liver cell populations such as oval cells [151]. In an effort to put an end to the debate, they used mature B lymphocytes to generate iPS cells carrying the genetic rearrangements associated with B cell terminal differentiation. In another study, Stadtfeld and colleagues contributed to resolving the debate by generating iPS cells from pancreatic β cells [142]. As only terminally differentiated β cells produce insulin, these cells could easily be identified and therefore, like B lymphocytes, are considered ideal to determine the origin of iPS cells. These studies have conclusively demonstrated that iPS cells can be derived from terminally differentiated cells.

Another critical issue in iPS research concerns the question whether iPS cells are really identical to ES cells, since small but clear differences between ES cells and iPS cells have been reported. Patterns of DNA methylation on Oct4 and Nanog, for instance, differ between ES cells and iPS cells [1, 5–7]. However, the relevance of these small differences to stem cell identity is debatable, as it is has been shown that it is not the exact DNA methylation pattern that determines gene activity, but rather the overall density of methylation marks [152, 153]. Additionally, differences in viability are observed between animals derived from ES cells and iPS cells, as iPS cell-derived mice display increased risk of tumor formation in comparison to mice generated from ES cells [6]. It remains to be determined whether this discrepancy is a result of the experimental procedure of induced reprogramming, or that it reflects genuine differences between iPS cells and ES cells.

Many mechanistic questions regarding the molecular process of transcription factor-induced reprogramming remain still unanswered. Although the role of Oct4 and Sox2 as revivers of the pluripotent circuitry is widely acknowledged, the precise role of other reprogramming factors, such as Klf4 and c-Myc, remains unclear. These factors seem to generally contribute to the reactivation of pluripotency by creating an epigenetic environment that allows global transcriptional reorganization. However, these

factors are also linked to cellular functions other than epigenetic remodeling and thus may play additional roles in the reprogramming process. For example, c-Myc’s function in enhancing proliferation and the ability of Klf4 to induce expression of Nanog may clearly contribute to the reprogramming process. Interestingly, a recent study on the contribution of the reprogramming factors to the induction of pluripotency found that c-Myc is particularly involved in the repression of lineage-associated genes in the early stages of the process [154]. Indeed, it was observed in the same study that c-Myc transgene expression was required only for the first few days of the process. Although not essential to reprogramming, this function of c-Myc may greatly enhance the earliest steps of the reprogramming process by laying the framework for efficient repression of lineage genes [154]. Finally, the mechanisms governing the widespread epigenetic remodeling during reprogramming remain largely elusive. It is unknown, for instance, what process causes the progressive loss of DNA methylation at the promoters of Oct4, Sox2 and Nanog and the demethylation of H3K9 at key loci such as Oct4. Moreover, reprogramming involves activating epigenetic modifications at some genomic locations (e.g. pluripotency genes such as Oct4, Sox2 and Nanog), while repressive modifications are deposited at other regions (e.g. lineage genes). The intricate mechanisms involved in targeting these opposing forces to their respective loci are still unknown. Interestingly, Soldner and colleagues found that gene expression profiles of human iPS cells in which viral transgenes were excised using loxP/Cre technology resembled those of ES cells more closely than the parental iPS cell that still contained viral insertions [30]. Although silencing of transgenes is commonly observed in iPS cells, some residual transgene expression has been reported in several studies [7, 15, 16, 27]. This suggests that residual transgene expression of virus-carrying iPS cells can affect the characteristics of the cells, and may explain the differences between ES cells and iPS cells such as those described above.

The obstacles and questions described above form the major challenges for IPS researchers in the coming years in the road towards the clinic.

## References

1. Takahashi, K., & Yamanaka, S. (2006). Induction of pluripotent stem cells from mouse embryonic and adult fibroblast cultures by defined factors. *Cell*, 126, 663–676. doi:[10.1016/j.cell.2006.07.024](https://doi.org/10.1016/j.cell.2006.07.024).
2. Tokuzawa, Y., Kaiho, E., Maruyama, M., et al. (2003). Fbx15 is a novel target of Oct3/4 but is dispensable for embryonic stem cell self-renewal and mouse development. *Molecular and Cellular Biology*, 23, 2699–2708. doi:[10.1128/MCB.23.8.2699-2708.2003](https://doi.org/10.1128/MCB.23.8.2699-2708.2003).

Humana Press

3. Nichols, J., Zevnik, B., Anastassiadis, K., et al. (1998). Formation of pluripotent stem cells in the mammalian embryo depends on the POU transcription factor Oct4. *Cell*, 95, 379–391. doi:[10.1016/S0092-8674(00)81769-9](https://doi.org/10.1016/S0092-8674(00)81769-9).

4. Mitsui, K., Tokuzawa, Y., Itoh, H., et al. (2003). The homeoprotein Nanog is required for maintenance of pluripotency in mouse epiblast and ES cells. *Cell*, 113, 631–642. doi:[10.1016/S0092-8674(03)00393-3](https://doi.org/10.1016/S0092-8674(03)00393-3).

5. Maherali, N., Sridharan, R., Xie, W., et al. (2007). Directly reprogrammed fibroblasts show global epigenetic remodeling and widespread tissue contribution. *Cell Stem Cell*, 1, 55–70. doi:[10.1016/j.stem.2007.05.014](https://doi.org/10.1016/j.stem.2007.05.014).

6. Okita, K., Ichisaka, T., & Yamanaka, S. (2007). Generation of germline-competent induced pluripotent stem cells. *Nature*, 448, 313–317. doi:[10.1038/nature05934](https://doi.org/10.1038/nature05934).

7. Wernig, M., Meissner, A., Foreman, R., et al. (2007). In vitro reprogramming of fibroblasts into a pluripotent ES-cell-like state. *Nature*, 448, 318–324. doi:[10.1038/nature05944](https://doi.org/10.1038/nature05944).

8. Takahashi, K., Tanabe, K., Ohnuki, M., et al. (2007). Induction of pluripotent stem cells from adult human fibroblasts by defined factors. *Cell*, 131, 861–872. doi:[10.1016/j.cell.2007.11.019](https://doi.org/10.1016/j.cell.2007.11.019).

9. Yu, J., Vodyanik, M. A., Smuga-Otto, K., et al. (2007). Induced pluripotent stem cell lines derived from human somatic cells. *Science*, 318, 1917–1920. doi:[10.1126/science.1151526](https://doi.org/10.1126/science.1151526).

10. Park, I. H., Zhao, R., West, J. A., et al. (2008). Reprogramming of human somatic cells to pluripotency with defined factors. *Nature*, 451, 141–146. doi:[10.1038/nature06534](https://doi.org/10.1038/nature06534).

11. Mali, P., Ye, Z., Hommond, H. H., et al. (2008). Improved efficiency and pace of generating induced pluripotent stem cells from human adult and fetal fibroblasts. *Stem Cells*, 26, 1998–2005. doi:[10.1634/stemcells.2008-0346](https://doi.org/10.1634/stemcells.2008-0346).

12. Lowry, W. E., Richter, L., Yachechko, R., et al. (2008). Generation of human induced pluripotent stem cells from dermal fibroblasts. *Proceedings of the National Academy of Sciences of the United States of America*, 105, 2883–2888. doi:[10.1073/pnas.0711983105](https://doi.org/10.1073/pnas.0711983105).

13. Liao, J., Wu, Z., Wang, Y., et al. (2008). Enhanced efficiency of generating induced pluripotent stem (iPS) cells from human somatic cells by a combination of six transcription factors. *Cell Research*, 18, 600–603. doi:[10.1038/cr.2008.51](https://doi.org/10.1038/cr.2008.51).

14. Nakagawa, M., Koyanagi, M., Tanabe, K., et al. (2008). Generation of induced pluripotent stem cells without Myc from mouse and human fibroblasts. *Nature Biotechnology*, 26, 101–106. doi:[10.1038/nbt1374](https://doi.org/10.1038/nbt1374).

15. Dimos, J., Rodolfa, K., Niakan, K., et al. (2008). Induced pluripotent stem cells generated from patients with ALS can be differentiated into motor neurons. *Science*, 321, 1218–1221. doi:[10.1126/science.1158799](https://doi.org/10.1126/science.1158799).

16. Park, I. H., Arora, N., Huo, H., et al. (2008). Disease-specific induced pluripotent stem cells. *Cell*, 134, 877–886. doi:[10.1016/j.cell.2008.07.041](https://doi.org/10.1016/j.cell.2008.07.041).

17. Maherali, N., Ahfeldt, T., Rigamonti, A., Utikal, J., Cowan, C., & Hochedlinger, K. (2008). A high-efficiency system for the generation and study of human induced pluripotent stem cells. *Cell Stem Cell*, 3, 340–345. doi:[10.1016/j.stem.2008.08.003](https://doi.org/10.1016/j.stem.2008.08.003).

18. Aasen, T., Raya, A., Barrero, M., et al. (2008). Efficient and rapid generation of induced pluripotent stem cells from human keratinocytes. *Nature Biotechnology*, 26, 1276–1284. doi:[10.1038/nbt.1503](https://doi.org/10.1038/nbt.1503).

19. Huangfu, D., Osafune, K., Maehr, R., et al. (2008). Induction of pluripotent stem cells from primary human fibroblasts with only Oct4 and Sox2. *Nature Biotechnology*, 26, 1269–1275. doi:[10.1038/nbt.1502](https://doi.org/10.1038/nbt.1502).

20. Tateishi, K., He, J., Taranova, O., Liang, G., D'Alessio, A., & Zhang, Y. (2008). Generation of insulin-secreting islet-like clusters from human skin fibroblasts. *Journal of Biological Chemistry*, 283, 31601–31607. doi:[10.1074/jbc.M806597200](https://doi.org/10.1074/jbc.M806597200).

21. Hanna, J., Wernig, M., Markoulaki, S., et al. (2007). Treatment of sickle cell anemia mouse model with iPS cells generated from autologous skin. *Science*, 318, 1920–1923. doi:[10.1126/science.1152092](https://doi.org/10.1126/science.1152092).

22. Wernig, M., Zhao, J. P., Pruszak, J., et al. (2008). Neurons derived from reprogrammed fibroblasts functionally integrate into the fetal brain and improve symptoms of rats with Parkinson's disease. *Proceedings of the National Academy of Sciences of the United States of America*, 105, 5856–5861. doi:[10.1073/pnas.0801677105](https://doi.org/10.1073/pnas.0801677105).

23. Zhang D, Jiang W, Liu M, et al. Highly efficient differentiation of human ES cells and iPS cells into mature pancreatic insulin-producing cells. *Cell Research* 2009; In press.

24. Zhang, J., Wilson, G. F., Soerens, A. G., et al. (2009). Functional cardiomyocytes derived from human induced pluripotent stem cells. *Circulation Research*, 104, e30–e41. doi:[10.1161/CIRCRESAHA.108.192237](https://doi.org/10.1161/CIRCRESAHA.108.192237).

25. Narazaki, G., Uosaki, H., Teranishi, M., et al. (2008). Directed and systematic differentiation of cardiovascular cells from mouse induced pluripotent stem cells. *Circulation*, 118, 498–506. doi:[10.1161/CIRCULATIONAHA.108.769562](https://doi.org/10.1161/CIRCULATIONAHA.108.769562).

26. Xu, D., Alipio, Z., Fink, L. M., et al. (2009). Phenotypic correction of murine hemophilia A using an iPS cell-based therapy. *Proceedings of the National Academy of Sciences of the United States of America*, 106, 808–813. doi:[10.1073/pnas.0812090106](https://doi.org/10.1073/pnas.0812090106).

27. Ebert, A. D., Yu, J., Rose, F. F., Jr., Mattis, V. B., Lorson, C. L., Thomson, J. A., et al. (2009). Induced pluripotent stem cells from a spinal muscular atrophy patient. *Nature*, 457, 277–280. doi:[10.1038/nature07677](https://doi.org/10.1038/nature07677).

28. Blelloch, R., Venere, M., Yen, J., & Ramalho-Santos, M. (2007). Generation of induced pluripotent stem cells in the absence of drug selection. *Cell Stem Cell*, 1, 245–247. doi:[10.1016/j.stem.2007.08.008](https://doi.org/10.1016/j.stem.2007.08.008).

29. Sommer CA, Stadtfeld M, Murphy GJ, Hochedlinger K, Kotton DN, Mostoslavsky G. iPS Cell Generation Using a Single Lentiviral Stem Cell Cassette. *Stem Cells*. 2008; In press.

30. Soldner, F., Hockemeyer, D., Beard, C., et al. (2009). Parkinson's disease patient-derived induced pluripotent stem cells free of viral reprogramming factors. *Cell*, 136, 964–977. doi:[10.1016/j.cell.2009.02.013](https://doi.org/10.1016/j.cell.2009.02.013).

31. Carey, B. W., Markoulaki, S., Hanna, J., et al. (2009). Reprogramming of murine and human somatic cells using a single polycistronic vector. *Proceedings of the National Academy of Sciences of the United States of America*, 106, 157–162. doi:[10.1073/pnas.0811426106](https://doi.org/10.1073/pnas.0811426106).

32. Stadtfeld, M., Nagaya, M., Utikal, J., Weir, G., & Hochedlinger, K. (2008). Induced pluripotent stem cells generated without viral integration. *Science*, 322, 945–949. doi:[10.1126/science.1162494](https://doi.org/10.1126/science.1162494).

33. Okita, K., Nakagawa, M., Hyenjong, H., Ichisaka, T., & Yamanaka, S. (2008). Generation of mouse induced pluripotent stem cells without viral vectors. *Science*, 322, 949–953. doi:[10.1126/science.1164270](https://doi.org/10.1126/science.1164270).

34. Kaji K, Norrby K, Paca A, Mileikovsky M, Mohseni P, Woltjen K. Virus-free induction of pluripotency and subsequent excision of reprogramming factors. *Nature online* 2009.

35. Yamanaka, S. (2009). A fresh look at iPS cells. *Cell*, 137, 13–17. doi:[10.1016/j.cell.2009.03.034](https://doi.org/10.1016/j.cell.2009.03.034).

36. Zhou, H., Wu, S., Joo, J. Y., et al. (2009). Generation of induced pluripotent stem cells using recombinant proteins. *Cell Stem Cell*, 4, 381–384. doi:[10.1016/j.stem.2009.04.005](https://doi.org/10.1016/j.stem.2009.04.005).

37. Ponzielli, R., Katz, S., Barsyte-Lovejoy, D., & Penn, L. Z. (2005). Cancer therapeutics: targeting the dark side of Myc.

38. Aoi, T., Yae, K., Nakagawa, M., et al. (2008). Generation of pluripotent stem cells from adult mouse liver and stomach cells. *Science*, 321, 699–702. doi:[10.1126/science.1154884](https://doi.org/10.1126/science.1154884).

39. Wernig, M., Meissner, A., Cassady, J. P., & Jaenisch, R. (2008). c-Myc is dispensable for direct reprogramming of mouse fibroblasts. *Cell Stem Cell*, 2, 10–12. doi:[10.1016/j.stem.2007.12.001](https://doi.org/10.1016/j.stem.2007.12.001).

40. Chen, Y., Shi, L., Zhang, L., et al. (2008). The molecular mechanism governing the oncogenic potential of SOX2 in breast cancer. *Journal of Biological Chemistry*, 283, 17969–17978. doi:[10.1074/jbc.M802917200](https://doi.org/10.1074/jbc.M802917200).

41. Trosko, J. E. (2006). From adult stem cells to cancer stem cells: Oct-4 Gene, cell-cell communication, and hormones during tumor promotion. *Annals of the New York Academy of Sciences*, 1089, 36–58. doi:[10.1196/annals.1386.018](https://doi.org/10.1196/annals.1386.018).

42. Rowland, B. D., & Peiper, D. S. (2006). KLF4, p21 and context-dependent opposing forces in cancer. *Nature Reviews Cancer*, 6, 11–23. doi:[10.1038/nrc1780](https://doi.org/10.1038/nrc1780).

43. Duinsbergen, D., Eriksson, M., 't Hoen, P. A., Frisén, J., & Mikkers, H. (2008). Induced pluripotency with endogenous and inducible genes. *Experimental Cell Research*, 314, 3255–3263. doi:[10.1016/j.yexcr.2008.06.024](https://doi.org/10.1016/j.yexcr.2008.06.024).

44. Eminli, S., Utikal, J., Arnold, K., Jaenisch, R., & Hochedlinger, K. (2008). Reprogramming of neural progenitor cells into induced pluripotent stem cells in the absence of exogenous Sox2 expression. *Stem Cells*, 26, 2467–2474. doi:[10.1634/stemcells.2008-0317](https://doi.org/10.1634/stemcells.2008-0317).

45. Shi, Y., Do, J. T., Desponts, C., Hahm, H. S., Schöler, H. R., & Ding, S. (2008). A combined chemical and genetic approach for the generation of induced pluripotent stem cells. *Cell Stem Cell*, 2, 525–528. doi:[10.1016/j.stem.2008.05.011](https://doi.org/10.1016/j.stem.2008.05.011).

46. Kim, J. B., Zaehres, H., Wu, G., et al. (2008). Pluripotent stem cells induced from adult neural stem cells by reprogramming with two factors. *Nature*, 454, 646–650. doi:[10.1038/nature07061](https://doi.org/10.1038/nature07061).

47. Silva, J., Barrandon, O., Nichols, J., Kawaguchi, J., Theunissen, T., & Smith, A. (2008). Promotion of reprogramming to ground state pluripotency by signal inhibition. *PLoS Biology*, 6, 2237–2247. doi:[10.1371/journal.pbio.0060253](https://doi.org/10.1371/journal.pbio.0060253).

48. Kim, J. B., Sebastiano, V., Wu, G., et al. (2009). Oct4-induced pluripotency in adult neural stem cells. *Cell*, 136, 411–419. doi:[10.1016/j.cell.2009.01.023](https://doi.org/10.1016/j.cell.2009.01.023).

49. Qin, D., Gan, Y., Shao, K., et al. (2008). Mouse meningiocytes express sox2 and yield high efficiency of chimeras after nuclear reprogramming with exogenous factors. *Journal of Biological Chemistry*, 283, 33730–33735. doi:[10.1074/jbc.M806788200](https://doi.org/10.1074/jbc.M806788200).

50. Huangfu, D., Maehr, R., Guo, W., et al. (2008). Induction of pluripotent stem cells by defined factors is greatly improved by small-molecule compounds. *Nature Biotechnology*, 26, 795–797. doi:[10.1038/nbt1418](https://doi.org/10.1038/nbt1418).

51. Marson, A., Foreman, R., Chevalier, B., et al. (2008). Wnt signaling promotes reprogramming of somatic cells to pluripotency. *Cell Stem Cell*, 3, 132–135. doi:[10.1016/j.stem.2008.06.019](https://doi.org/10.1016/j.stem.2008.06.019).

52. Shi, Y., Desponts, C., Do, J. T., Hahm, H. S., Schöler, H. R., & Ding, S. (2008). Induction of pluripotent stem cells from mouse embryonic fibroblasts by Oct4 and Klf4 with small-molecule compounds. *Cell Stem Cell*, 3, 568–574. doi:[10.1016/j.stem.2008.10.004](https://doi.org/10.1016/j.stem.2008.10.004).

53. Mikkelsen, T. S., Hanna, J., Zhang, X., et al. (2008). Dissecting direct reprogramming through integrative genomic analysis. *Nature*, 454, 49–55. doi:[10.1038/nature07056](https://doi.org/10.1038/nature07056).

54. Pesce, M., & Scholer, H. R. (2001). Oct-4: gatekeeper in the beginnings of mammalian development. *Stem Cells*, 19, 271–27. doi:[10.1634/stemcells.19-4-271](https://doi.org/10.1634/stemcells.19-4-271).

55. Pesce, M., & Scholer, H. (2000). Oct4: control of totipotency and germline determination. *Molecular Reproduction and Development*, 55, 452–457. doi:[10.1002/(SICI)1098-2795(200004)55:4<452::AID-MRD14>3.0.CO;2-S](https://doi.org/10.1002/(SICI)1098-2795(200004)55:4<452::AID-MRD14>3.0.CO;2-S).

56. Gidekel, S., Pizov, G., Bergman, Y., & Pikarsky, E. (2003). Oct-3/4 is a dosedependent oncogenic fate determinant. *Cancer Cell*, 4, 361–370. doi:[10.1016/S1535-6108(03)00270-8](https://doi.org/10.1016/S1535-6108(03)00270-8).

57. Niwa, H., Miyazaki, J., & Smith, A. G. (2000). Quantitative expression of Oct-3/4 defines differentiation, dedifferentiation or self-renewal of ES cells. *Nature Genetics*, 24, 372–376. doi:[10.1038/74199](https://doi.org/10.1038/74199).

58. Zaehres, H., Lensch, M. W., Daheron, L., Stewart, S. A., Itskovitz-Eldor, J., & Daley, G. Q. (2005). High-efficiency RNA interference in human embryonic stem cells. *Stem Cells*, 23, 299–305. doi:[10.1634/stemcells.2004-0252](https://doi.org/10.1634/stemcells.2004-0252).

59. Kehler, J., Tolkunova, E., Koschorz, B., et al. (2004). Oct4 is required for primordial germ cell survival. *EMBO Reports*, 5, 1078–1083. doi:[10.1038/sj.embor.7400279](https://doi.org/10.1038/sj.embor.7400279).

60. Cheng, L., Sung, M. T., Cossu-Rocca, P., et al. (2007). OCT4: biological functions and clinical applications as a marker of germ cell neoplasia. *Journal of Pathology*, 211, 1–9. doi:[10.1002/path.2105](https://doi.org/10.1002/path.2105).

61. Avilion, A. A., Nicolis, S. K., Pevny, L. H., Perez, L., Vivian, N., & Lovell-Badge, R. (2003). Multipotent cell lineages in early mouse development depend on SOX2 function. *Genes and Development*, 17, 126–140. doi:[10.1101/gad.224503](https://doi.org/10.1101/gad.224503).

62. Miyagi, S., Saito, T., Mizutani, K., et al. (2004). The Sox-2 regulatory regions display their activities in two distinct types of multipotent stem cells. *Molecular and Cellular Biology*, 24, 4207–4220. doi:[10.1128/MCB.24.10.4207-4220.2004](https://doi.org/10.1128/MCB.24.10.4207-4220.2004).

63. Yuan, H., Corbi, N., Basilico, C., & Dailey, L. (1995). Developmental-specific activity of the FGF-4 enhancer requires the synergistic action of Sox2 and Oct-3. *Genes and Development*, 9, 2635–2645. doi:[10.1101/gad.9.21.2635](https://doi.org/10.1101/gad.9.21.2635).

64. Nishimoto, M., Miyagi, S., Katayanagi, T., Tomioka, M., Muramatsu, M., & Okuda, A. (2003). The embryonic Octamer factor 3/4 displays distinct DNA binding specificity from those of other Octamer factors. *Biochemical and Biophysical Research Communications*, 302, 581–586. doi:[10.1016/S0006-291X(03)00218-3](https://doi.org/10.1016/S0006-291X(03)00218-3).

65. Masui, S., Nakatake, Y., Toyooka, Y., et al. (2007). Pluripotency governed by Sox2 via regulation of Oct3/4 expression in mouse embryonic stem cells. *Nature Cell Biology*, 9, 625–635. doi:[10.1038/ncb1589](https://doi.org/10.1038/ncb1589).

66. Chew, J. L., Loh, Y. H., Wensheng, Z., et al. (2005). Reciprocal transcriptional regulation of Pou5f1 and Sox2 via the Oct4/Sox2 complex in embryonic stem cells. *Molecular and Cellular Biology*, 25, 6031–6046.

67. Ferri, A. L., Cavallaro, M., Braida, D., et al. (2004). Sox2 deficiency causes neurodegeneration and impaired neurogenesis in the adult mouse brain. *Development*, 131, 3805–3819. doi:[10.1242/dev.01204](https://doi.org/10.1242/dev.01204).

68. Williamson, K. A., Hever, A. M., Rainger, J., et al. (2006). Mutations in SOX2 cause anophthalmia-esophageal-genital (AEG) syndrome. *Human Molecular Genetics*, 15, 1413–1422. doi:[10.1093/hmg/ddl064](https://doi.org/10.1093/hmg/ddl064).

69. Kelberman, D., Rizzoti, K., Avilion, A., et al. (2006). Mutations within Sox2/SOX2 are associated with abnormalities in the hypothalamo-pituitarygonadal axis in mice and humans. *Journal of Clinical Investigation*, 116, 2442–2455.

70. Dong, C., Wilhelm, D., & Koopman, P. (2004). Sox genes and cancer. *Cytogenet Genome Research*, 105, 442–447. doi:[10.1159/000078217](https://doi.org/10.1159/000078217).

71. Dang, C. V., O’Donnell, K. A., Zeller, K. I., Nguyen, T., Osthus, R. C., & Li, F. (2006). The c-Myc target gene network. *Seminars in Cancer Biology*, 16, 253–264. doi:[10.1016/j.semcancer.2006.07.014](https://doi.org/10.1016/j.semcancer.2006.07.014).

72. Dang, D. T., Pevsner, J., & Yang, V. W. (2000). The biology of the mammalian Krüppel-like family of transcription factors. *International Journal of Biochemistry and Cell Biology*, 32, 1103–1121. doi:[10.1016/S1357-2725(00)00059-5](https://doi.org/10.1016/S1357-2725(00)00059-5).

73. Wei, D., Kanai, M., Huang, S., & Xie, K. (2006). Emerging role of KLF4 in human gastrointestinal cancer. *Carcinogenesis*, 27, 23–31. doi:[10.1093/carcin/bgi243](https://doi.org/10.1093/carcin/bgi243).

74. Segre, J. A., Bauer, C., & Fuchs, E. (1999). Klf4 is a transcription factor required for establishing the barrier function of the skin. *Nature Genetics*, 22, 356–360. doi:[10.1038/11926](https://doi.org/10.1038/11926).

75. Conkright, M. D., Wani, M. A., Anderson, K. P., & Lingrel, J. B. (1999). A gene encoding an intestinal-enriched member of the Krüppel-like factor family expressed in intestinal epithelial cells. *Nucleic Acids Research*, 27, 1263–1270. doi:[10.1093/nar/27.5.1263](https://doi.org/10.1093/nar/27.5.1263).

76. Garrett-Sinha, L. A., Eberspaecher, H., Seldin, M. F., & de Crombrugge, B. (1996). A gene for a novel zinc-finger protein expressed in differentiated epithelial cells and transiently in certain mesenchymal cells. *Journal of Biological Chemistry*, 271, 31384–31390. doi:[10.1074/jbc.271.49.31384](https://doi.org/10.1074/jbc.271.49.31384).

77. Shields, J. M., Christy, R. J., & Yang, V. W. (1996). Identification and characterization of a gene encoding a gut-enriched Krüppel-like factor expressed during growth arrest. *Journal of Biological Chemistry*, 271, 20009–20017. doi:[10.1074/jbc.271.33.20009](https://doi.org/10.1074/jbc.271.33.20009).

78. Nakatake, Y., Fukui, N., Iwamatsu, Y., et al. (2006). Klf4 cooperates with Oct3/4 and Sox2 to activate the Lefty1 core promoter in embryonic stem cells. *Molecular and Cellular Biology*, 26, 72–7782. doi:[10.1128/MCB.00468-06](https://doi.org/10.1128/MCB.00468-06).

79. Jiang, J., Chan, Y. S., Loh, Y. H., et al. (2008). A core Klf circuitry regulates self-renewal of embryonic stem cells. *Nature Cell Biology*, 10, 353–360. doi:[10.1038/ncb1698](https://doi.org/10.1038/ncb1698).

80. Chen, X., Johns, D. C., Geiman, D. E., et al. (2001). Krüppel-like factor 4 (gut-enriched Krüppel-like factor) inhibits cell proliferation by blocking G1/S progression of the cell cycle. *Journal of Biological Chemistry*, 276, 30423–30428. doi:[10.1074/jbc.M101194200](https://doi.org/10.1074/jbc.M101194200).

81. McConnell, B. B., Ghaleb, A. M., Nandan, M. O., & Yang, V. W. (2007). The diverse functions of Krüppel-like factors 4 and 5 in epithelial biology and pathobiology. *Bioessays*, 29, 549–557. doi:[10.1002/bies.20581](https://doi.org/10.1002/bies.20581).

82. Ohnishi, S., Ohnami, S., Laub, F., et al. (2003). Downregulation and growth inhibitory effect of epithelial-type Krüppel-like transcription factor KLF4, but not KLF5, in bladder cancer. *Biochemical and Biophysical Research Communications*, 308, 251–256. doi:[10.1016/S0006-291X(03)01356-1](https://doi.org/10.1016/S0006-291X(03)01356-1).

83. Katz, J. P., Perreault, N., Goldstein, B. G., et al. (2005). Loss of Klf4 in mice causes altered proliferation and differentiation and precancerous changes in the adult stomach. *Gastroenterology*, 128, 935–945. doi:[10.1053/j.gastro.2005.02.022](https://doi.org/10.1053/j.gastro.2005.02.022).

84. Foster, K. W., Frost, A. R., McKie-Bell, P., et al. (2000). Increase of GKLF messenger RNA and protein expression during progression of breast cancer. *Cancer Research*, 60, 6488–6495.

85. Wang, N., Liu, Z. H., Ding, F., et al. (2002). Downregulation of gut-enriched Krüppel-like factor expression in esophageal cancer. *World Journal of Gastroenterology*, 8, 966–970.

86. Zhao, W., Hisamuddin, I. M., Nandan, M. O., et al. (2004). Identification of Krüppel-like factor 4 as a potential tumor suppressor gene in colorectal cancer. *Oncogene*, 23, 395–402. doi:[10.1038/sj.onc.1207067](https://doi.org/10.1038/sj.onc.1207067).

87. Lebofsky, R., & Walter, J. C. (2007). New Myc-anisms for DNA replication and tumorigenesis? *Cancer Cell*, 12, 102–103. doi:[10.1016/j.ccr.2007.07.013](https://doi.org/10.1016/j.ccr.2007.07.013).

88. Patel, J. H., Loboda, A. P., Showe, M. K., Showe, L. C., & McMahon, S. B. (2004). Analysis of genomic targets reveals

complex functions of MYC. *Nature Reviews Cancer*, 4, 562–568. doi:[10.1038/nrc1393](https://doi.org/10.1038/nrc1393).

89. Cawley, S., Bekiranov, S., Ng, H. H., et al. (2004). Unbiased mapping of transcription factor binding sites along human chromosomes 21 and 22 points to widespread regulation of noncoding RNAs. *Cell*, 116, 499–509. doi:[10.1016/S0092-8674(04)00127-8](https://doi.org/10.1016/S0092-8674(04)00127-8).

90. Cowling, V. H., & Cole, M. D. (2006). Mechanism of transcriptional activation by the Myc oncoproteins. *Seminars in Cancer Biology*, 16, 242–252. doi:[10.1016/j.semcancer.2006.08.001](https://doi.org/10.1016/j.semcancer.2006.08.001).

91. Chang, T. C., Yu, D., Lee, Y. S., et al. (2008). Widespread microRNA repression by Myc contributes to tumorigenesis. *Nature Genetics*, 40, 43–50. doi:[10.1038/ng.2007.30](https://doi.org/10.1038/ng.2007.30).

92. Davis, A. C., Wims, M., Spotts, G. D., Hann, S. R., & Bradley, A. (1993). A null c-myc mutation causes lethality before 10.5 days of gestation in homozygotes and reduced fertility in heterozygous female mice. *Genes and Development*, 7, 671–682. doi:[10.1101/gad.7.4.671](https://doi.org/10.1101/gad.7.4.671).

93. Baudino, T. A., McKay, C., Pendeville-Samain, H., et al. (2002). c-Myc is essential for vasculogenesis and angiogenesis during development and tumor progression. *Genes and Development*, 16, 2530–2543. doi:[10.1101/gad.1024602](https://doi.org/10.1101/gad.1024602).

94. Niwa, H. (2007). How is pluripotency determined and maintained? *Development*, 134, 635–646. doi:[10.1242/dev.02787](https://doi.org/10.1242/dev.02787).

95. Chambers, I., Colby, D., Robertson, M., et al. (2003). Functional expression cloning of Nanog, a pluripotency sustaining factor in embryonic stem cells. *Cell*, 113, 643–655. doi:[10.1016/S0092-8674(03)00392-1](https://doi.org/10.1016/S0092-8674(03)00392-1).

96. Chambers, I., Silva, J., Colby, D., et al. (2007). Nanog safeguards pluripotency and mediates germline development. *Nature*, 450, 1230–1234. doi:[10.1038/nature06403](https://doi.org/10.1038/nature06403).

97. Hyslop, L., Stojkovic, M., Armstrong, L., et al. (2005). Down-regulation of nanog induces differentiation of human embryonic stem cells to extraembryonic lineages. *Stem Cells*, 23, 1035–1043. doi:[10.1634/stemcells.2005-0080](https://doi.org/10.1634/stemcells.2005-0080).

98. Bhattacharya, B., Miura, T., Brandenberger, R., et al. (2004). Gene expression in human embryonic stem cell lines: unique molecular signature. *Blood*, 103, 2956–2964. doi:[10.1182/blood-2003-09-3314](https://doi.org/10.1182/blood-2003-09-3314).

99. Pan, G., Li, J., Zhou, Y., Zheng, H., & Pei, D. (2006). A negative feedback loop of transcription factors that controls stem cell pluripotency and self-renewal. *FASEB Journal*, 20, 1730–1732. doi:[10.1096/fj.05-5543fje](https://doi.org/10.1096/fj.05-5543fje).

100. Loh, Y. H., Wu, Q., Chew, J. L., et al. (2006). The Oct4 and Nanog transcription network regulates pluripotency in mouse embryonic stem cells. *Nature Genetics*, 38, 431–440. doi:[10.1038/ng1760](https://doi.org/10.1038/ng1760).

101. Boyer, L. A., Lee, T. I., Cole, M. F., et al. (2005). Core transcriptional regulatory circuitry in human embryonic stem cells. *Cell*, 122, 947–956. doi:[10.1016/j.cell.2005.08.020](https://doi.org/10.1016/j.cell.2005.08.020).

102. Alon, U. (2007). Network motifs: theory and experimental approaches. *Nature Reviews Genetics.*, 8, 450–461. doi:[10.1038/nrg2102](https://doi.org/10.1038/nrg2102).

103. McAdams, H. H., & Arkin, A. (1997). Stochastic mechanisms in gene expression. *Proceedings of the National Academy of Sciences of the United States of America*, 94, 814–819. doi:[10.1073/pnas.94.3.814](https://doi.org/10.1073/pnas.94.3.814).

104. Rosenfeld, N., Elowitz, M. B., & Alon, U. (2002). Negative autoregulation speeds the response times of transcription networks. *Journal of Molecular Biology*, 323, 785–793. doi:[10.1016/S0022-2836(02)00994-4](https://doi.org/10.1016/S0022-2836(02)00994-4).

105. Kuroda, T., Tada, M., Kubota, H., et al. (2005). Octamer and Sox elements are required for transcriptional cis regulation of Nanog gene expression. *Molecular and Cellular Biology*, 25, 2475–2485. doi:[10.1128/MCB.25.6.2475-2485.2005](https://doi.org/10.1128/MCB.25.6.2475-2485.2005).

106. Okumura-Nakanishi, S., Saito, M., Niwa, H., & Ishikawa, F. (2005). Oct-3/4 and Sox2 regulate Oct-3/4 gene in embryonic stem cells. *Journal of Biological Chemistry*, 280, 5307–5317. doi:[10.1074/jbc.M410015200](https://doi.org/10.1074/jbc.M410015200).

107. Rodda, D. J., Chew, J. L., Lim, L. H., et al. (2005). Transcriptional regulation of Nanog by OCT4 and SOX2. *Journal of Biological Chemistry*, 280, 24731–24737. doi:[10.1074/jbc.M502573200](https://doi.org/10.1074/jbc.M502573200).

108. Wang, J., Rao, S., Chu, J., et al. (2006). A protein interaction network for pluripotency of embryonic stem cells. *Nature*, 444, 364–368. doi:[10.1038/nature05284](https://doi.org/10.1038/nature05284).

109. Feldman, N., Gerson, A., Fang, J., et al. (2006). G9a-mediated irreversible epigenetic inactivation of Oct-3/4 during early embryogenesis. *Nature Cell Biology*, 8, 188–194. doi:[10.1038/ncb1353](https://doi.org/10.1038/ncb1353).

110. Bernstein, B. E., Mikkelsen, T. S., Xie, X., et al. (2006). A bivalent chromatin structure marks key developmental genes in embryonic stem cells. *Cell*, 125, 315–326. doi:[10.1016/j.cell.2006.02.041](https://doi.org/10.1016/j.cell.2006.02.041).

111. Azuara, V., Perry, P., Sauer, S., et al. (2006). Chromatin signatures of pluripotent cell lines. *Nature Cell Biology*, 8, 532–538. doi:[10.1038/ncb1403](https://doi.org/10.1038/ncb1403).

112. Guenther, M. G., Levine, S. S., Boyer, L. A., Jaenisch, R., & Young, R. A. (2007). A chromatin landmark and transcription initiation at most promoters in human cells. *Cell*, 130, 77–88. doi:[10.1016/j.cell.2007.05.042](https://doi.org/10.1016/j.cell.2007.05.042).

113. Boyer, L. A., Plath, K., Zeitlinger, J., et al. (2006). Polycomb complexes repress developmental regulators in murine embryonic stem cells. *Nature*, 441, 349–353. doi:[10.1038/nature04733](https://doi.org/10.1038/nature04733).

114. Silva, J., & Smith, A. (2008). Capturing pluripotency. *Cell*, 132, 532–536. doi:[10.1016/j.cell.2008.02.006](https://doi.org/10.1016/j.cell.2008.02.006).

115. Smith, A. G., Heath, J. K., Donaldson, D. D., et al. (1988). Inhibition of pluripotential embryonic stem cell differentiation by purified polypeptides. *Nature*, 336, 688–690. doi:[10.1038/336688a0](https://doi.org/10.1038/336688a0).

116. Niwa, H., Burdon, T., Chambers, I., & Smith, A. (1998). Self-renewal of pluripotent embryonic stem cells is mediated via activation of STAT3. *Genes and Development*, 12, 2048–2060. doi:[10.1101/gad.12.13.2048](https://doi.org/10.1101/gad.12.13.2048).

117. Liu, Y., Ji, L., Ten, Y., Wang, Y., & Pei, X. (2007). The molecular mechanism of embryonic stem cell pluripotency and self-renewal. *Science in China Series C Life Sciences*, 50, 619–623. doi:[10.1007/s11427-007-0074-5](https://doi.org/10.1007/s11427-007-0074-5).

118. Xu, R. H., Chen, X., Li, D. S., et al. (2002). BMP4 initiates human embryonic stem cell differentiation to trophoblast. *Nature Biotechnology*, 20, 1261–1264. doi:[10.1038/nbt761](https://doi.org/10.1038/nbt761).

119. Qi, X., Li, T. G., Hao, J., et al. (2004). BMP4 supports self-renewal of embryonic stem cells by inhibiting mitogen-activated protein kinase pathways. *Proceedings of the National Academy of Sciences of the United States of America*, 101, 6027–6032. doi:[10.1073/pnas.0401367101](https://doi.org/10.1073/pnas.0401367101).

120. Jaenisch, R., & Young, R. (2008). Stem cells, the molecular circuitry of pluripotency and nuclear reprogramming. *Cell*, 132, 567–582. doi:[10.1016/j.cell.2008.01.015](https://doi.org/10.1016/j.cell.2008.01.015).

121. Meissner, A., Wernig, M., & Jaenisch, R. (2007). Direct reprogramming of genetically unmodified fibroblasts into pluripotent stem cells. *Nature Biotechnology*, 25, 1177–1181. doi:[10.1038/nbt1335](https://doi.org/10.1038/nbt1335).

122. Hotta, A., & Ellis, J. (2008). Retroviral vector silencing during iPSC cell induction: an epigenetic beacon that signals distinct pluripotent states. *Journal of Cellular Biochemistry*, 105, 940–948. doi:[10.1002/jcb.21912](https://doi.org/10.1002/jcb.21912).

123. McMahon, S. B., Van Buskirk, H. A., Dugan, K. A., Copeland, T. D., & Cole, M. D. (1998). The novel ATM-related protein TRRAP is an essential cofactor for the c-Myc and E2F oncoproteins. *Cell*, 94, 363–374. doi:[10.1016/S0092-8674(00)81479-8](https://doi.org/10.1016/S0092-8674(00)81479-8).

124. Vervoorts, J., Luscher-Firzlaff, J. M., Rottmann, S., et al. (2003). Stimulation of c-MYC transcriptional activity and acetylation by recruitment of the cofactor CBP. *EMBO Reports*, 4, 484–490. doi:[10.1038/sj.embor.embor821](https://doi.org/10.1038/sj.embor.embor821).

125. Knoepfler, P. S., Zhang, X. Y., Cheng, P. F., et al. (2006). Myc influences global chromatin structure. *EMBO Journal*, 25, 2723–2734. doi:[10.1038/sj.emboj.7601152](https://doi.org/10.1038/sj.emboj.7601152).

126. Yamanaka, S. (2007). Strategies and new developments in the generation of patient-specific pluripotent stem cells. *Cell Stem Cell*, 1, 39–49. doi:[10.1016/j.stem.2007.05.012](https://doi.org/10.1016/j.stem.2007.05.012).

127. Hooker, C. W., & Hurlin, P. J. (2006). Of Myc and Mnt. *Journal of Cell Science*, 119, 208–216. doi:[10.1242/jcs.02815](https://doi.org/10.1242/jcs.02815).

128. Welstead, G. G., Schorderet, P., & Boyer, L. A. (2008). The reprogramming language of pluripotency. *Current Opinion in Genetics and Development*, 18, 123–129. doi:[10.1016/j.gde.2008.01.013](https://doi.org/10.1016/j.gde.2008.01.013).

129. Egli, D., Birkhoff, G., & Eggan, K. (2008). Mediators of reprogramming: transcription factors and transitions through mitosis. *Nature Reviews Molecular Cell Biology*, 9, 505–516. doi:[10.1038/nrm2439](https://doi.org/10.1038/nrm2439).

130. Dominguez-Sola, D., Ying, C. Y., Grandori, C., et al. (2007). Nontranscriptional control of DNA replication by c-Myc. *Nature*, 448, 445–451. doi:[10.1038/nature05953](https://doi.org/10.1038/nature05953).

131. Wu, K. J., Grandori, C., Amacker, M., et al. (1999). Direct activation of TERT transcription by c-MYC. *Nature Genetics*, 21, 220–224. doi:[10.1038/6010](https://doi.org/10.1038/6010).

132. Rowland, B. D., Bernards, R., & Peeper, D. S. (2005). The KLF4 tumour suppressor is a transcriptional repressor of p53 that acts as a context-dependent oncogene. *Nature Cell Biology*, 7, 1074–1082. doi:[10.1038/ncb1314](https://doi.org/10.1038/ncb1314).

133. Zhang, W., Geiman, D. E., Shields, J. M., et al. (2000). The gut-enriched Kruppel-like factor (Kruppel-like factor 4) mediates the transactivating effect of p53 on the p21WAF1/Cipl promoter. *Journal of Biological Chemistry*, 275, 18391–18398. doi:[10.1074/jbc.C000062200](https://doi.org/10.1074/jbc.C000062200).

134. Seoane, J., Le, H. V., & Massague, J. (2002). Myc suppression of the p21(Cipl) Cdk inhibitor influences the outcome of the p53 response to DNA damage. *Nature*, 419, 729–734. doi:[10.1038/nature01119](https://doi.org/10.1038/nature01119).

135. Lin, T., Chao, C., Saito, S., et al. (2005). p53 induces differentiation of mouse embryonic stem cells by suppressing Nanog expression. *Nature Cell Biology*, 7, 165–171. doi:[10.1038/ncb1211](https://doi.org/10.1038/ncb1211).

136. Evans, P. M., Zhang, W., Chen, X., et al. (2007). Kruppel-like factor 4 is acetylated by p300 and regulates gene transcription via modulation of histone acetylation. *Journal of Biological Chemistry*, 282, 33994–34002. doi:[10.1074/jbc.M701847200](https://doi.org/10.1074/jbc.M701847200).

137. Balzer, E., & Moss, E. G. (2007). Localization of the developmental timing regulator Lin28 to mRNP complexes, P-bodies and stress granules. *RNA Biology*, 4, 16–25.

138. Yu, F., Yao, H., Zhu, P., et al. (2007). let-7 regulates self renewal and tumorigenicity of breast cancer cells. *Cell*, 131, 1109–1123. doi:[10.1016/j.cell.2007.10.054](https://doi.org/10.1016/j.cell.2007.10.054).

139. Kumar, M. S., Erkeland, S. J., Pester, R. E., et al. (2008). Suppression of non-small cell lung tumor development by the let-7 microRNA family. *Proceedings of the National Academy of Sciences of the United States of America*, 105, 3903–3908. doi:[10.1073/pnas.0712321105](https://doi.org/10.1073/pnas.0712321105).

140. Jackson-Grusby, L., Beard, C., Possemato, R., et al. (2001). Loss of genomic methylation causes p53-dependent apoptosis and epigenetic deregulation. *Nature Genetics*, 27, 31–39. doi:[10.1038/83730](https://doi.org/10.1038/83730).

141. Loh, Y. H., Zhang, W., Chen, X., George, J., & Ng, H. H. (2007). Jmjd1a and Jmjd2c histone H3 Lys 9 demethylases regulate self-renewal in embryonic stem cells. *Genes and Development*, 21, 2545–2557. doi:[10.1101/gad.1588207](https://doi.org/10.1101/gad.1588207).

142. Stadtfeld, M., Brennand, K., & Hochedlinger, K. (2008). Reprogramming of pancreatic β cells into induced pluripotent

stem cells. *Current Biology*, 18, 890–894. doi:[10.1016/j.cub.2008.05.010](https://doi.org/10.1016/j.cub.2008.05.010).

143. Verfaillie, C. (2008). The undoing of differentiation by four defined factors: a big step forward towards generating patient specific pluripotent stem cells. *Journal of Hepatology*, 49, 876–878. doi:[10.1016/j.jhep.2008.08.007](https://doi.org/10.1016/j.jhep.2008.08.007).

144. Wilmut, I. (2007). The first direct reprogramming of adult human fibroblasts. *Cell Stem Cell*, 1, 3–594. doi:[10.1016/j.stem.2007.11.013](https://doi.org/10.1016/j.stem.2007.11.013).

145. Varas F, Stadtfeld M, De Andres-Aguayo L, et al. Fibroblast derived induced pluripotent stem cells show no common retroviral vector insertions. *Stem Cells*. 2008; In press.

146. Gregory, M. A., Qi, Y., & Hann, S. R. (2005). The ARF tumor suppressor: keeping Myc on a leash. *Cell Cycle*, 4, 249–252.

147. Brambrink, T., Foreman, R., Welstead, G. G., et al. (2008). Sequential expression of pluripotency markers during direct reprogramming of mouse somatic cells. *Cell Stem Cell*, 2, 151–159. doi:[10.1016/j.stem.2008.01.004](https://doi.org/10.1016/j.stem.2008.01.004).

148. Stadtfeld, M., Maherali, N., Breault, D., & Hochedlinger, K. (2008). Defining Molecular Cornerstones during Fibroblast to iPS Cell Reprogramming in Mouse. *Cell Stem Cell*, 2, 230–240. doi:[10.1016/j.stem.2008.02.001](https://doi.org/10.1016/j.stem.2008.02.001).

149. Ruau, D., Ensenat-Waser, R., Dinger, T. C., et al. (2008). Pluripotency associated genes are reactivated by chromatin-modifying agents in neurosphere cells. *Stem Cells*, 26, 920–926. doi:[10.1634/stemcells.2007-0649](https://doi.org/10.1634/stemcells.2007-0649).

150. Cyranoski, D. (2008). 5 things to know before jumping on the iPS bandwagon. *Nature*, 452, 406–408. doi:[10.1038/452406a](https://doi.org/10.1038/452406a).

151. Hanna, J., Markoulaki, S., Schorderet, P., et al. (2008). Direct reprogramming of terminally differentiated mature B lymphocytes to pluripotency. *Cell*, 133, 250–264. doi:[10.1016/j.cell.2008.03.028](https://doi.org/10.1016/j.cell.2008.03.028).

152. Boyes, J., & Bird, A. (1992). Repression of genes by DNA methylation depends on CpG density and promoter strength: evidence for involvement of a methyl-CpG binding protein. *EMBO Journal*, 11, 327–333.

153. Lorincz, M. C., Schubeler, D., Hutchinson, S. R., Dickerson, D. R., & Groudine, M. (2002). DNA methylation density influences the stability of an epigenetic imprint and Dnmt3a/b-independent de novo methylation. *Molecular and Cellular Biology*, 22, 7572–7580. doi:[10.1128/MCB.22.21.7572-7580.2002](https://doi.org/10.1128/MCB.22.21.7572-7580.2002).

154. Sridharan, R., Tchieu, J., Mason, M. J., et al. (2009). Role of the murine reprogramming factors in the induction of pluripotency. *Cell*, 136, 364–77. doi:[10.1016/j.cell.2009.01.001](https://doi.org/10.1016/j.cell.2009.01.001).
