
PERSPECTIVE

INFLAMMATION

**AlMing 2 Detect Foreign DNA**

Arthur M. Krieg

Published 30 June 2009; Volume 2 Issue 77 pe39

Foreign genomic DNA can be detected by immune cells in the cytoplasm, triggering reactions such as cell death and secretion of proinflammatory cytokines, including interleukin 1β (IL-1β). Several studies have now elucidated the mechanism for this response. The HIN-200 (hematopoietic interferon-inducible nuclear proteins with a 200–amino acid repeat) family member AIM2 (absent in melanoma 2) binds and oligomerizes on cytoplasmic DNA through a HIN domain. The oligomerized AIM2 recruits the adaptor ASC (apoptosis-associated specklike protein containing a CARD) through homotypic pyrin domain interactions and binds caspase-1, forming an inflammasome that generates IL-1β. Together with previous studies, this work demonstrates that immune detection of foreign nucleic acids occurs through several distinct signaling pathways, leading to diverse immune outcomes.

To detect infection quickly, the immune system relies on several families of receptors that bind molecules that are either unique to pathogens (such as lipopolysaccharide), or that are sequestered in intracellular compartments distinct from those in which the receptors are located (such as nucleic acids). These receptor families include Toll-like receptors (TLRs; 10 members identified in humans), retinoic acid–inducible gene I–like receptors (RLRs), and nucleotide-binding oligomerization domain–like receptors (NLRs) (1). Certain TLRs can detect single-stranded (ss) or double-stranded (ds) RNA or DNA in the endosomes; the RLRs can detect cytoplasmic ssRNA or dsRNA; and one of the NLRs, NALP3 (Nacht domain-, leucine-rich repeat-, and PYD-containing protein 3), can be activated by cytoplasmic nucleic acids (1) (Fig. 1). For more than 15 years, cytoplasmic dsDNA has been known to induce macrophage cell death (2), but a receptor for cytoplasmic DNA had not been identified. Approaching this question from different directions, four independent research groups have now identified AIM2 as a cytoplasmic dsDNA receptor that induces the formation of a specific inflammasome and the secretion of IL-1β (3–6).

Considering that all forms of life use RNA and DNA, it may seem surprising that the immune system uses receptors specific for these universal macromolecules to detect pathogens. However, in 1984, bacterial DNA was shown to activate immune

---

Pfizer Research Technology Center, 620 Memorial Drive, Cambridge, MA 02139, USA.  
E-mail, arthur.krieg@pfizer.com

IFN and that this response requires TBK1 (TANK-binding kinase-1) and IKKi (inhibitor of nuclear factor κB kinase, inducible), but not TLR9 (13). This response seems to be biologically relevant because DNA that enters the cytoplasm during the course of *Listeria* infection triggers an IFN response (14). A putative candidate receptor for the cytoplasmic DNA-induced IFN-β response, DAI (DNA-dependent activator of IFN-regulatory factors), is associated with both TBK1 and IRF-3 (interferon regulatory transcription factor 3) (15), yet mice deficient in DAI have normal responses to cytoplasmic DNA, indicating that this factor is redundant (16).

In 2008, a second type of TLR9-independent response to cytoplasmic DNA was described: production of IL-1β by an “inflammasome,” a multiprotein complex that activates caspase-1, with the subsequent cleavage and release of IL-1β, IL-18, and IL-22 (17). This inflammasome response was triggered by adenoviral or herpes viral infection, nonreplicating adenoviral vectors, and transfected purified adenoviral DNA (17). The viral response required a specific member of the cytoplasmic NLRs, NALP3, but the response to purified transfected DNA was independent of NALP3 (17). Both responses required the adaptor protein ASC (apoptosis-associated specklike protein containing a CARD) and caspase-1 (17).

Neither NALP3 nor ASC is required for IRF-3 activation in response to transfected DNA (17), so cytoplasmic DNA triggers at least two separate and parallel responses: (i) an inflammasome response leading to IL-1β secretion (17), and (ii) a TBK1- and IRF-3–dependent response leading to type I IFN secretion (13, 16). Viral exposure activates a third pathway, the NALP3 inflammasome (17), but it is unclear whether cytoplasmic viral DNA is the key signal for this.

Four independent research groups have now elucidated the mechanism by which cytoplasmic DNA activates the inflammasome: AIM2, a member of the HIN-200 family of IFN-inducible proteins, binds the DNA and recruits ASC, triggering inflammasome assembly and pro-IL-1β cleavage (3–6). These studies establish that AIM2 binds dsDNA through its C-terminal oligo or oligosaccharide binding domain (HIN-200 domain) (5, 6), and colocalizes with DNA in the cytoplasm (as detected by fluorescence resonance energy transfer) in a HIN-dependent manner (5) (Fig. 2). Confocal

www.SCIENCESIGNALING.org 30 June 2009 Vol 2 Issue 77 pe39 1

PERSPECTIVE

![Image Description](image.png)

Fig. 1. Immune detection of nucleic acids. Free nucleic acids, bacteria, and viruses normally enter cells through endocytosis, by which they can activate endosomal TLRs, inducing NF-κB (nuclear factor κB) and pathways involving IRFs that lead to IFN secretion. Endosomal dsRNA can activate pathways dependent on TRIF [Toll IL-1 receptor (TIR) domain-containing adaptor-inducing IFN-β], whereas endosomal ssRNA and CpG DNA can activate pathways dependent on MyD88 (myeloid differentiation primary response gene 88). Unless the endosomes are disrupted, entry of the nucleic acids into the cytoplasm appears to be relatively slow. Although it remains possible that free extracellular nucleic acids can also be taken up directly into the cytoplasm, no such pathway has been demonstrated. Cytoplasmic RNA of appropriate length and structure can activate RLRs such as MDA5 (melanoma differentiation associated gene 5) and RIG-I (retinoic acid-inducible protein I), activating signaling pathways with downstream effects similar to those of TLRs. Cytoplasmic DNA, by contrast, can oligomerize AIM2, triggering the formation of an inflammasome that releases IL-1β. Dashed lines represent multistep pathways, whereas dotted lines represent pathways in which the hypothetical steps are not well understood.

Microscopy of cells loaded with Cy3-labeled plasmid DNA and tagged AIM2 established that AIM2 not only binds DNA, but actually oligomerizes onto the DNA in a NALP3-independent manner (6).

The N-terminal pyrin domains (PYDs) of the oligomerized AIM2 recruit ASC (5, 6). The PYD of AIM2 is unique among those of several proteins tested in its ability to bind ASC (5). Furthermore, among the PYD proteins tested, only PYDs of NALP3 and AIM2 induced pro-IL-1β maturation when ASC and caspase-1 were co-expressed (5). In macrophages transfected with DNA, ASC and caspase-1 are required along with AIM2 for activation of IL-1β and caspase 1 and apoptosis; NALP3 is not (5, 6). Stable expression of AIM2 in human embryonic kidney 293 cells is sufficient to confer responsiveness to cytoplasmic dsDNA (6). The IFN-β response to cytoplasmic dsDNA requires neither AIM2 nor ASC, and indeed, this response is significantly higher in their absence, confirming that the inflammasome response represents a signaling pathway distinct from the IFN-β response (5).

PERSPECTIVE

Several different inflammasomes have been described, but none contained HIN-200 family members. Among HIN-200 proteins, the pyrin domain of AIM2 is the most similar to the inflammasome-related pyrin domains (18). AIM2 is also the only family member to have a clear human homolog.

Together with earlier work, these findings provide a clear picture of the importance of nucleic acids in innate immune defense—both RNA and DNA are detected in both the endosome and cytoplasm, in both double-stranded and single-stranded forms. The different receptors trigger quite different patterns of immune defenses, which may be simply due to chance, or might provide clues to the evolutionary forces that have shaped these defenses. The endosomal receptor TLR9 also evolved to detect DNA, but in contrast to AIM2, the selectivity of TLR9s for unmethylated CpG motifs distinguishes viral or bacterial DNA from self DNA. Unlike TLR9, the DNA source is unimportant to AIM2—the presence of DNA in the cytoplasm, where AIM2 is constitutively localized, is sufficient to trigger the response. Other differences in these two DNA detection pathways are the length and strandedness of the nucleic acids required for stimulation: Single-stranded CpG DNA as short as 6 bases can trigger the activation of TLR9, whereas AIM2 is most effectively activated by dsDNA at least 44 base pairs or longer. Presumably, AIM2’s requirement for longer DNA relates to its need to oligomerize onto the DNA, in order to recruit ASC and other proteins into an inflammasome (Fig. 2). The two pathways have markedly different consequences—TLR9 induction results in minimal, if any, IL-1β secretion, instead strongly inducing secretion of type I IFN from plasmacytoid dendritic cells and B cell maturation and differentiation into antibody-secreting cells, especially in the presence of specific antigen. Like endosomal CpG DNA, cytoplasmic DNA also induces a strong IFN response, but this requires TBK1 instead of TLR9, and the proximal receptor remains to be identified. The evolution of so many different receptors and pathways to detect nucleic acids is evidence of the utility of this as a general strategy for pathogen detection, but there is an open question as to why inflammasome activation is limited to the presence of cytoplasmic DNA, instead of also occurring in response to endosomal DNA. Perhaps the latter is more “dangerous,” necessitating the affected cell to take the extreme step of activating a pathway that will likely result in cell death. A related question is why evolution has limited the TLR9 detection pathway for CpG motifs to the endosome; why not also have pathways for the specific recognition of CpG motifs in the cytoplasm? Perhaps the cytoplasmic pathways are needed to respond to endogenous DNA—for example, from apoptotic cells. Alternatively, a sensor for DNA that does not require CpG motifs would enable the immune system to detect viruses that have strongly suppressed CpG motifs in their genomes: Many parvoviruses, retroviruses, and papilloma viruses have <20% of the expected frequency of CpG motifs (19) and might activate AIM2 more effectively than TLR9.

For every proinflammatory pathway, there are generally counterregulatory pathways, and the AIM2 inflammasome may be no exception: A different member of the HIN-200 family of IFN-inducible proteins, p202, binds to dsDNA in a manner similar to that of AIM2, but appears to antagonize AIM2 (3). Because it lacks a pyrin domain, p202 cannot

---

**Plasma membrane**

**Endosome**

**HIN domain**
**Pyrin domain**
**CARD domain**
**Caspase**

**dsDNA**

**AIM2**

**ASC**

**Caspase-1**

**Conversion of pro-IL-1β to IL-1β**

---

**Fig. 2.** The AIM2 inflammasome. (A) dsDNA enters the cytoplasm. (B) AIM2 oligomerizes on dsDNA. (C) ASC is recruited to the AIM2 oligomer. (D) Caspase-1 is recruited and activated, generating IL-1β.

form an inflammasome upon binding DNA. Such binding may be competitive or otherwise antagonistic to AIM2, because RNA interference (RNAi)-mediated inhibition of p202 enhances the dsDNA-induced caspase activation (3). It is interesting that p202 abundance is higher in bone marrow-derived macrophages from autoimmune mice than from normal control strains (3) and that the gene encoding p202 has been identified as a lupus susceptibility gene in mice (20). However, there is no known human homolog for p202, so any connection to human autoimmune disease remains speculative at this time.

Another unanswered question is the mechanism by which the dsDNA gets into the cytoplasm to activate AIM2. DNA is actively taken up by cells into endosomes in vitro and in vivo (21, 22), but the mechanism of escape from endosomes into the cytoplasm is poorly understood, except in the case of certain bacteria such as *Listeria*, which actively mediate endosomal escape (14). As with other immune defenses, it seems inevitable that we will find ways in which pathogens have subverted or co-opted the AIM2 DNA recognition mechanism.

The studies on AIM2 may be relevant in the clinical development of novel therapies in which cytoplasmic DNA is directed into cells. For example, the plasmids used for DNA vaccines contain CpG motifs, yet do not require TLR9 to be recognized (23). DNA vaccines do require TBK1, which is also essential for the cytoplasmic detection of dsDNA (16). It will be interesting to determine whether the AIM2 inflammasome participates in the generation of immune responses to DNA vaccines. Could a better understanding of this pathway improve the performance of DNA vaccines in primates, which so far has been disappointing? Regardless, this quartet of papers has greatly advanced our understanding of how the immune system perceives nucleic acids.

### References

1. T. Kawai, S. Akira, The roles of TLRs, RLRs and NLRs in pathogen recognition. *Int. Immunol.* **21**, 317–337 (2009).
2. K. J. Stacey, I. L. Ross, D. A. Hume, Electroporation and DNA-dependent cell death in murine macrophages. *Immunol. Cell Biol.* **71**, 75–85 (1993).
3. T. L. Roberts, A. Idris, J. A. Dunn, G. M. Kelly, C. M. Burnton, S. Hodgson, L. L. Hardy, V. Garceau, M. J. Sweet, I. L. Ross, D. A. Hume, K. J. Stacey, HIN-200 proteins regulate caspase activation in response to foreign cytoplasmic DNA. *Science* **323**, 1057–1060 (2009).
4. T. Burckstummer, C. Baumann, S. Bluml, E. Dixit, G. Durnberger, H. Jahn, M. Planyavsky, M. Bilban, J. Colinge, K. L. Bennett, G. Superti-Furga, An orthogonal proteomic-genomic screen identifies AIM2 as a cytoplasmic DNA sensor for the inflammasome. *Nat. Immunol.* **10**, 266–272 (2009).
5. V. Hornung, A. Ablasser, M. Charrel-Dennis, F. Bauernfeind, G. Horvath, D. R. Caffrey, E. Latz, K. A. Fitzgerald, AIM2 recognizes cytosolic dsDNA and forms a caspase-1-activating inflammasome with ASC. *Nature* **458**, 514–518 (2009).
6. T. Fernandes-Alnemri, J. W. Yu, P. Datta, J. Wu, E. S. Alnemri, AIM2 activates the inflammasome and cell death in response to cytoplasmic DNA. *Nature* **458**, 509–513 (2009).
7. T. Tokunaga, H. Yamamoto, S. Shimada, H. Abe, T. Fukuda, Y. Fujisawa, Y. Furutani, O. Yano, T. Kataoka, T. Sudo, Antitumor activity of deoxyribonucleic acid fraction from Mycobacterium bovis BCG. I. Isolation, physicochemical characterization, and antitumor activity. *J. Natl. Cancer Inst.* **72**, 955–962 (1984).
8. A. M. Krieg, A. K. Yi, S. Matson, T. J. Waldschmidt, G. A. Bishop, R. Teasdale, G. A. Koretzky, D. M. Klinman, CpG motifs in bacterial DNA trigger direct B-cell activation. *Nature* **374**, 546–549 (1995).
9. H. Hemmi, O. Takeuchi, T. Kawai, T. Kaisho, S. Sato, H. Sanjo, M. Matsumoto, K. Hoshino, H. Wagner, K. Takeda, S. Akira, A Toll-like receptor recognizes bacterial DNA. *Nature* **408**, 740–745 (2000).
10. E. Latz, A. Schoenemeyer, A. Visintin, K. A. Fitzgerald, B. G. Monks, C. F. Knetter, E. Lien, N. J. Nilsen, T. Espevik, D. T. Golenbock, TLR9 signals after translocating from the ER to CpG DNA in the lysosome. *Nat. Immunol.* **5**, 190–198 (2004).
11. K. Suzuki, A. Mori, K. J. Ishii, J. Saito, D. S. Singer, D. M. Klinman, P. R. Krause, L. D. Kohn, Activation of target-tissue immune-recognition molecules by double-stranded polynucleotides. *Proc. Natl. Acad. Sci. U.S.A.* **96**, 2285–2290 (1999).
12. H. Yoshida, Y. Okabe, K. Kawane, H. Fukuyama, S. Nagata, Lethal anemia caused by interferon-beta produced in mouse embryos carrying undigested DNA. *Nat. Immunol.* **6**, 49–56 (2005).
13. K. J. Ishii, C. Coban, H. Kato, K. Takahashi, Y. Torii, F. Takeshita, H. Ludwig, G. Sutter, K. Suzuki, H. Hemmi, S. Sato, M. Yamamoto, S. Uematsu, T. Kawai, O. Takeuchi, S. Akira, A Toll-like receptor-independent antiviral response induced by double-stranded B-form DNA. *Nat. Immunol.* **7**, 40–48 (2006).
14. D. B. Stetson, R. Medzhitov, Recognition of cytosolic DNA activates an IRF3-dependent innate immune response. *Immunity* **24**, 93–103 (2006).
15. A. Takaoka, Z. Wang, M. K. Choi, H. Yanai, H. Negishi, T. Ban, Y. Lu, M. Miyagishi, T. Kodama, K. Honda, Y. Ohba, T. Taniguchi, DAI (DLM-1/ZBP1) is a cytosolic DNA sensor and an activator of innate immune response. *Nature* **448**, 501–505 (2007).
16. K. J. Ishii, T. Kawagoe, S. Koyama, K. Matsui, H. Kumar, T. Kawai, S. Uematsu, O. Takeuchi, F. Takeshita, C. Coban, S. Akira, TANK-binding kinase-1 delineates innate and adaptive immune responses to DNA vaccines. *Nature* **451**, 725–729 (2008).
17. D. A. Muruve, V. Petrilli, A. K. Zaiss, L. R. White, S. A. Clark, P. J. Ross, R. J. Parks, J. Tschopp, The inflammasome recognizes cytosolic microbial and host DNA and triggers an innate immune response. *Nature* **452**, 103–107 (2008).
18. T. Liu, A. Rojas, Y. Ye, A. Godzik, Homology modeling provides insights into the binding mode of the PAAD/DAPIN/pyrin domain, a fourth member of the CARD/DD/DED domain family. *Protein Sci.* **12**, 1872–1881 (2003).
19. S. Karlin, W. Doerfler, L. R. Cardon, Why is CpG suppressed in the genomes of virtually all small eukaryotic viruses but not in those of large eukaryotic viruses? *J. Virol.* **68**, 2889–2897 (1994).
20. D. Choubey, R. Panchanathan, Interferon-inducible Ifi200-family genes in systemic lupus erythematosus. *Immunol. Lett.* **119**, 32–41 (2008).
21. A. R. Thierry, E. Vives, J. P. Richard, P. Prevot, C. Martinand-Mari, I. Robbins, B. Lebleu, Cellular uptake and intracellular fate of antisense oligonucleotides. *Curr. Opin. Mol. Ther.* **5**, 133–138 (2003).
22. J. A. Wolff, V. Budker, The mechanism of naked DNA uptake and expression. *Adv. Genet.* **54**, 3–20 (2005).
23. B. Spies, H. Hochrein, M. Vabulas, K. Huster, D. H. Busch, F. Schmitz, A. Heit, H. Wagner, Vaccination with plasmid DNA activates dendritic cells via Toll-like receptor 9 (TLR9) but functions in TLR9-deficient mice. *J. Immunol.* **171**, 5908–5912 (2003).

Citation: A. M. Krieg, AlMing 2 detect foreign DNA. *Sci. Signal.* **2**, pe39 (2009).

Science Signaling

AIMing 2 Detect Foreign DNA  
Arthur M. Krieg (June 23, 2009)  
*Science Signaling* **2** (77), pe39. [doi:10.1126/scisignal.277pe39]

---

The following resources related to this article are available online at http://stke.sciencemag.org.  
This information is current as of June 10, 2015.

---

### Article Tools
Visit the online version of this article to access the personalization and article tools:  
http://stke.sciencemag.org/content/2/77/pe39

### Related Content
The editors suggest related resources on *Science*'s sites:  
http://stke.sciencemag.org/content  
http://www.sciencemag.org/content/sci/323/5917/1057.full.html  
http://stke.sciencemag.org/content  
http://stke.sciencemag.org/content/sigtrans/3/150/pe45.full.html  
http://stke.sciencemag.org/content  
http://stke.sciencemag.org/content  
http://stke.sciencemag.org/content  
http://stke.sciencemag.org/content  
http://stke.sciencemag.org/content  
http://stke.sciencemag.org/content/sigtrans/2/78/pe40.full.html  
http://stke.sciencemag.org/content  
http://stke.sciencemag.org/content  

### References
This article cites 23 articles, 5 of which you can access for free at:  
http://stke.sciencemag.org/content/2/77/pe39#BIBL  

### Glossary
Look up definitions for abbreviations and terms found in this article:  
http://stke.sciencemag.org/cgi/glossarylookup  

### Permissions
Obtain information about reproducing this article:  
http://www.sciencemag.org/about/permissions.dtl  

---

Science Signaling (ISSN 1937-9145) is published weekly, except the last December, by the American Association for the Advancement of Science, 1200 New York Avenue, NW, Washington, DC 20005.  
Copyright 2015 by the American Association for the Advancement of Science; all rights reserved.
