
Progress in Lipid Research 48 (2009) 307–343

**Contents lists available at ScienceDirect**

**Progress in Lipid Research**

journal homepage: www.elsevier.com/locate/plipres

---

**Review**

# Mammalian phosphoinositide kinases and phosphatases

Takehiko Sasaki${}^{a,b,*}$, Shunsuke Takasuga${}^{a,b}$, Junko Sasaki${}^{a,b}$, Satoshi Kofuji${}^{a,b}$, Satoshi Eguchi${}^{a,b}$, Masakazu Yamazaki${}^{b}$, Akira Suzuki${}^{b,c}$

* ${}^{a}$ Department of Pathology and Immunology, Akita University, Graduate School of Medicine, Akita 010-8543, Japan
* ${}^{b}$ Global Center of Excellence Program, Akita University, Graduate School of Medicine, Akita 010-8543, Japan
* ${}^{c}$ Division of Embryonic and Genetic Engineering, Medical Institute of Bioregulation, Kyushu University, Fukuoka 812-8582, Japan

---

## ARTICLE INFO

### Keywords:
Phosphoinositide  
Kinase  
Phosphatase  

---

## ABSTRACT

Phosphoinositides are lipids that are present in the cytoplasmic leaflet of a cell’s plasma and internal membranes and play pivotal roles in the regulation of a wide variety of cellular processes. Phosphoinositides are molecularly diverse due to variable phosphorylation of the hydroxyl groups of their inositol rings. The rapid and reversible configuration of the seven known phosphoinositide species is controlled by a battery of phosphoinositide kinases and phosphoinositide phosphatases, which are thus critical for phosphoinositide isomer-specific localization and functions. Significantly, a given phosphoinositide generated by different isoforms of these phosphoinositide kinases and phosphatases can have different biological effects. In mammals, close to 50 genes encode the phosphoinositide kinases and phosphoinositide phosphatases that regulate phosphoinositide metabolism and thus allow cells to respond rapidly and effectively to ever-changing environmental cues. Understanding the distinct and overlapping functions of these phosphoinositide-metabolizing enzymes is important for our knowledge of both normal human physiology and the growing list of human diseases whose etiologies involve these proteins. This review summarizes the structural and biological properties of all the known mammalian phosphoinositide kinases and phosphoinositide phosphatases, as well as their associations with human disorders.

© 2009 Elsevier Ltd. All rights reserved.

---

**Abbreviations:** Ambra1, activating molecule in Beclin-1 regulated autophagy; AR, androgen receptor; ARF, ADP-ribosylation factor; ASH, ASPM-SPD2-hydin; BAR, Bin-Amphiphysin-Rvs; BCR, B cell receptor; CFD, Francois-Neetens Fleck Corneal Dystrophy; Ci-VSP, *C. intestinalis* voltage-sensor-containing phosphatase; CMT, Charcot-Marie-Tooth; COP, coatomer complex coat protein; CR, cysteine-rich; CR3, complement receptor 3; DENN, differentially expressed in normal versus neoplastic; DEP, Dishevelled Egl-10 and pleckstrin; DN, dominant-negative; EBP50, 4.1-ezrin-radixin-moesin (ERM)-binding phosphoprotein 50; EGFR, epidermal growth factor receptor; EH, Eps15 homology; ER, endoplasmic reticulum; ERGIC, ER-to-Golgi intermediate compartment; ES, embryonic stem; FA, focal adhesion; FAPP, four-phosphatase-adaptor protein; FERM, band 4.1/ezrin/radixin/moesin; FYVE, Fab1p, YOTB, Vac1p and EEA1; G-CSF, granulocyte colony-stimulating factor; HAUSP, herpesvirus-associated ubiquitin-specific protease; Hdac2, histone deacetylase-2; IGF-1, insulin-like growth factor; ING2, inhibitor of growth protein-2; INPP, inositol polyphosphate phosphatase; Ins(1,4,5)P₃, inositol 1,4,5-trisphosphate; IRS, insulin receptor substrate; ITAM, immunoreceptor tyrosine activation motif; ITIM, immunoreceptor tyrosine inhibitory motif; KD, kinase-dead; LCCS, lethal congenital contractual syndrome; LDD, Lhermitte-Ducros disease; LKU, lipid kinase unique; LPA, lysophosphatidic acid; LRP6, LDL receptor-related protein-6; M-CSF, monocyte colony-stimulating factor; MDCK, Madin-Darby canine kidney; MEF, mouse embryonic fibroblast; MMAC1, mutated in multiple advanced cancers-1; MTM, myotubularin; NASH, non-alcoholic steatohepatitis; NGF, nerve growth factor; NLS, nuclear localization sequence; NPF, asparagine-proline-phenylalanine; OCRL1, oculocerebrorenal syndrome of Lowe-1; PA, phosphatidic acid; PDGF, platelet-derived growth factor; PDK1, phosphoinositide-dependent kinase-1; PEST, proline, glutamine, serine, threonine; PH, pleckstrin homology; PHD, plant homeodomain; PH-GRAM, pleckstrin homology-glucosyltransferases Rab-like GTPase activators and myotubularins; PHTS, PTEN hamartoma tumor syndrome; PI3Ks, phosphoinositide 3-kinases; PI4Ks, phosphatidylinositol 4-kinases; PIKfyve, phosphoinositide kinase for five position containing a fyve finger; PIPK, phosphatidylinositol phosphate kinase; PIPKI, type I PIPK; PIPKII, type II PIPK; PIPKIII, type III PIPK; PKD, protein kinase D; PLIP, PTEN-like lipid phosphatase; PR, proline-rich; PTB, phosphotyrosine-binding; PtdIns, phosphatidylinositol; PTEN, phosphatase and tensin homologue deleted on chromosome 10; PTPMT1, protein tyrosine phosphatase localized to the mitochondrion-1; PX, phox; RANKL, receptor activator of nuclear factor-κB ligand; RhoGDI, guanine nucleotide dissociation inhibitor of Rho; ROCK, Rho kinase; ROS, reactive oxygen species; S6K, S6 kinase; Sac, suppressor of actin; SAM, sterile α motif; SBF1, SET binding factor 1; SCF, stem cell factor; SDF1, stromal cell-derived factor-1; SET, Suvar3-9 Enhancer-of-zeste Trithorax; SH2, Src homology 2; SHIP, SH2-containing inositol phosphatase; SNPs, single nucleotide polymorphisms; SPOP, speckle-type POZ domain protein; TCP-1, t-complex polypeptide-1; TCR, T cell receptor; TGN, trans-Golgi network; TIRFM, total internal reflection fluorescent microscopy; TM, transmembrane; TNFα, tumor necrosis factor-α; TPIP, TPTE and PTEN homologous inositol lipid phosphatase; UVRAG, UV irradiation resistance associated gene; vps, vacuolar protein sorting; VSV-G, vesicular stomatitis virus glycoprotein; WT, Wild-type; XLMTM, X-linked myotubular myopathy.

* Corresponding author. Address: Department of Pathology and Immunology, Akita University, Graduate School of Medicine, Akita 010-8543, Japan. Tel.: +81 18 884 6080; fax: +81 836 2607.
* E-mail address: tsasaki@med.akita-u.ac.jp (T. Sasaki).

0163-7827/$ - see front matter © 2009 Elsevier Ltd. All rights reserved.

doi:10.1016/j.plipres.2009.06.001

Contents

1. Introduction ..... 309
2. Phosphoinositide 3-kinases (PI3Ks) ..... 311
   2.1. Classification ..... 311
   2.2. Class I PI3Ks ..... 312
      2.2.1. Structure ..... 312
      2.2.2. Expression ..... 312
      2.2.3. Biological functions of p110α and p110β ..... 312
      2.2.4. Biological functions of p110γ and p110δ ..... 313
      2.2.5. Disease association ..... 313
   2.3. Class II PI3Ks ..... 314
      2.3.1. Structure ..... 314
      2.3.2. Expression ..... 314
      2.3.3. Biological functions ..... 314
   2.4. Class III PI3K ..... 314
      2.4.1. Structure ..... 314
      2.4.2. Expression ..... 315
      2.4.3. Biological functions ..... 315
      2.4.4. Regulation ..... 315
      2.4.5. Disease association ..... 315
3. Phosphatidylinositol-4 kinases (PI4Ks) ..... 315
   3.1. Classification ..... 315
   3.2. Type II PI4Ks ..... 316
      3.2.1. Structure ..... 316
      3.2.2. Expression ..... 316
      3.2.3. Biological functions ..... 316
   3.3. Type III PI4Ks ..... 316
      3.3.1. Structure ..... 316
      3.3.2. Expression ..... 316
      3.3.3. Biological functions ..... 316
4. Phosphatidylinositol phosphate kinases (PIP Ks) ..... 317
   4.1. Classification ..... 317
   4.2. Type I PIP Ks ..... 317
      4.2.1. Structure ..... 317
      4.2.2. Expression ..... 317
      4.2.3. Biological functions ..... 317
      4.2.4. Regulatory factors ..... 319
      4.2.5. Disease association ..... 319
   4.3. Type II PIP Ks ..... 319
      4.3.1. Structure ..... 319
      4.3.2. Expression ..... 319
      4.3.3. Biological functions ..... 319
      4.3.4. Regulation ..... 320
      4.3.5. Disease association ..... 320
   4.4. Type III PIP K ..... 320
      4.4.1. Structure ..... 320
      4.4.2. Expression ..... 320
      4.4.3. Biological functions ..... 321
      4.4.4. Regulation ..... 321
      4.4.5. Disease association ..... 321
5. Phosphoinositide 3-phosphatases ..... 321
   5.1. Classification ..... 321
   5.2. PTEN ..... 321
      5.2.1. Structure ..... 321
      5.2.2. Expression ..... 322
      5.2.3. Biological functions ..... 322
      5.2.4. Regulation ..... 323
   5.3. TPIP ..... 323
      5.3.1. Structure ..... 323
      5.3.2. Expression ..... 323
      5.3.3. Biological functions ..... 323
   5.4. Myotubularins ..... 323
      5.4.1. Classes and general structure ..... 323
      5.4.2. Myotubularin/MTM1 ..... 324
      5.4.3. MTMR1 ..... 324
      5.4.4. MTMR2 ..... 324
      5.4.5. MTMR3 and MTMR4 ..... 325
      5.4.6. MTMR6, MTMR7 and MTMR8 ..... 325
      5.4.7. MTMR9 ..... 325
      5.4.8. MTMR14 ..... 325
      5.4.9. Other MTMR pseudophosphatases ..... 325

6. Phosphoinositide 4-phosphatases ..... 325
   6.1. Classification ..... 325
   6.2. INPP4s ..... 326
      6.2.1. Structure ..... 326
      6.2.2. Expression ..... 326
      6.2.3. Biological functions ..... 326
      6.2.4. Disease association ..... 326
   6.3. TMEM55s ..... 326
      6.3.1. Structure ..... 326
      6.3.2. Expression ..... 326
      6.3.3. Biological functions ..... 326
7. Phosphoinositide 5-phosphatases ..... 326
   7.1. Classification ..... 326
   7.2. Synaptojanins ..... 327
      7.2.1. Structure ..... 327
      7.2.2. Expression ..... 327
      7.2.3. Biological functions ..... 327
      7.2.4. Regulation ..... 327
   7.3. OCRL 1 ..... 328
      7.3.1. Structure ..... 328
      7.3.2. Expression ..... 329
      7.3.3. Biological functions ..... 329
      7.3.4. Regulation ..... 329
      7.3.5. Disease association ..... 329
   7.4. INPP5B ..... 329
      7.4.1. Structure ..... 329
      7.4.2. Expression ..... 329
      7.4.3. Biological functions ..... 329
   7.5. INPP5J ..... 330
      7.5.1. Structure ..... 330
      7.5.2. Expression ..... 330
      7.5.3. Biological functions ..... 330
   7.6. SKIP ..... 330
      7.6.1. Structure ..... 330
      7.6.2. Expression ..... 330
      7.6.3. Biological functions ..... 330
   7.7. SHIPs ..... 330
      7.7.1. Structure ..... 330
      7.7.2. Expression ..... 330
      7.7.3. Biological functions of SHIP 1 ..... 330
      7.7.4. Biological functions of SHIP 2 ..... 331
   7.8. INPP5E ..... 331
      7.8.1. Structure ..... 331
      7.8.2. Expression ..... 331
      7.8.3. Biological functions ..... 331
   7.9. PLIP. ..... 332
      7.9.1. Structure ..... 332
      7.9.2. Expression ..... 332
      7.9.3. Biological functions ..... 332
   7.10. Sac family phosphatases ..... 332
      7.10.1. Structure ..... 332
      7.10.2. Expression ..... 332
      7.10.3. Biological functions ..... 333
      7.10.4. Disease association ..... 333
8. Perspectives ..... 333
Acknowledgements ..... 334
References ..... 334

1. Introduction
Mammalian cell membranes contain the lipid phosphatidylinositol (PtdIns), which can be phosphorylated on several of the hydroxyl residues of its inositol ring to give rise to seven other phosphoinositide species (Fig. 1). Recent examinations of a variety of phosphoinositide-binding proteins have renewed interest in the possible functions of phosphoinositides as direct regulators of diverse cellular processes [1–5]. To date, the list of molecules that can bind to a phosphoinositide includes protein kinases, phospholipases, ion channel proteins, scaffold proteins, cytoskeletal proteins, and regulators of membrane trafficking [6–8]. However, specific phosphoinositides are produced and degraded in one or more well-defined membrane compartments, helping to confine or release a relevant phosphoinositide-binding molecule at the appropriate site for its action. In view of the pleiotropic cellular processes controlled by phosphoinositides, and the distinct molecules targeted by each, the interconversion of phosphoinositides must be spatio-temporally organized in a context-dependent manner. In mammals, the 18 phosphoinositide interconversion reactions identified thus far are mediated by as many as 47 genes encoding 19 phosphoinositide kinases and 28 phosphoinositide phosphatases

T. Sasaki et al./Progress in Lipid Research 48 (2009) 307–343

Fig. 1. Structures of phosphatidylinositol and its phosphoinositide derivatives. Phosphatidylinositol consists of a glycerol backbone that is esterified at its sn-1 and sn-2 positions to two fatty acids, and linked via phosphate at its sn-3 position to the D1 position of a D-myo-inositol head group. In mammals, the hydroxyl groups at the D3, 4 and 5 positions of the myo-inositol head group can be phosphorylated to yield the indicated mono-phosphate, bis-phosphate and tris-phosphate phosphoinositide derivatives. The phosphorylation of the 2- and 6-hydroxyl groups has yet to be documented.

Table 1  
Mammalian phosphoinositide kinases$^a$

| Class/type | Protein | Reaction$^b$ | Gene symbol | Chromosomal locus | Alternative names |
|------------|---------|--------------|-------------|--------------------|-------------------|
|            |         |              |             | Human             | Mouse             |
| Phosphoinositide 3-kinases | | | | | |
| Class I A PI3K | p110α | L(A,E)$^c$ | PIK3CA | 3q26 | 3B | PI3Kα/PIK3CA |
| | p110β | L(A,E) | PIK3CB | 3q22 | 9E4 | PI3Kβ/PIK3CB |
| | p110δ | L(A,E) | PIK3CD | 1q36 | 4E2 | PI3Kδ/PIK3CD |
| Class I B PI3K | p110γ | L(A,E) | PIK3CG | 7q22 | 12B | PI3Kγ/PIK3CG |
| Class II PI3K | PI3K-C2α | A(E) | PIK3C2A | 11p15 | 7F1 | PIK3C2α/Cpk-m |
| | PI3K-C2β | A(E) | PIK3C2B | 1q32 | 1E4 | PIK3C2β |
| | PI3K-C2γ | A(E) | PIK3C2G | 12p12 | 6G2 | PIK3C2γ |
| Class III PI3K | Vps34 | A | PIK3C3 | 18q12 | 18B1 | PIK3C3 |
| Phosphatidylinositol 4-kinases$^d$ | | | | | |
| Type II PI4Ks | PI4KIIα | B | PI4K2A | 10q24 | 19C3 | PI4K2A |
| | PI4KIIβ | B | PI4K2B | 4p15 | 5C1 | PI4K2B |
| Type III PI4Ks | PI4KIIIα | B | PIK4CA | 22q11 | 16A3 | PIK4CA/PI4KA |
| | PI4KIIIβ | B | PIK4CB | 1q21 | 3F2.1 | PIK4CB/PI4KB |
| Phosphatidylinositol phosphate kinases | | | | | |
| Type I PIPKs | PIPKIα | H | PIP5K1A | 1q22 | 19C1 | PIP5KIαPIP5K1A (mouse PIPKIβ/PIP5K1B) |
| | PIPKIβ | H | PIP5K1B | 9q13 | 3F2.1 | PIP5KIβ/PIP5K1B (mouse PIPKIα/PIP5K1A) |
| | PIPKIγ | H | PIP5K1C | 19p13 | 10C1 | PIP5KIγ/PIP5K1C |
| Type II PIPKs | PIPKIIα | I | PIP4K2A | 10p12 | 2A3 | PIP4K2A/PIP5K2A/PIP5KIIα |
| | PIPKIIβ | I | PIP4K2B | 17q12 | 11D | PIP4K2B/PIP5K2B/PIP5KIIβ |
| | PIPKIIγ | I | PIP4K2C | 12q13 | 10D3 | PIP4K2C/PIP5K2C/PIP5KIIγ |
| Type III PIPK | PIPKIII | F(C) | PIP5K3 | 2q35 | 1C2 | PIKFYVE/p235/PIP5K3 |

$a$ Regulatory subunits are not listed.  
$b$ Uppercase letters correspond to kinase reactions illustrated in Fig. 2.  
$c$ Parentheses indicate reactions that are secondary in nature for this enzyme.  
$d$ The original type I PI4K enzyme was later found to be a PI3K enzyme.

(Tables 1 and 2, respectively, and Fig. 2). In addition, multiple iso-  
zymes may exist for a given phosphoinositide kinase or phosphatase.  
These proteins often show overlapping tissue distributions, and an  
individual cell can express several isozymes. To shed light on the  
non-redundant roles of each phosphoinositide-metabolizing  
enzyme in vivo, researchers have turned to gene knockout studies

Table 2  
Mammalian phosphoinositide phosphatases$^a$

| Class/type | Protein | Reaction$^b$ | Gene symbol | Chromosomal locus | Alternative names |
|------------|---------|--------------|-------------|-------------------|-------------------|
|            |         |              |             | Human             | Mouse             |
| Phosphoinositide 3-phosphatases | PTEN | l(a)$^c$ | PTEN | 10q23 | 19C1 | MMAC1/TEP1 |
|            | TPIP    | g, l         | TPTE2      | 13q12             | 8A2               | TPTE2 (mouse PTEN2) |
| Myotubularins | MTM1   | a, g         | MTM1       | Xq28              | XA7.2             | CNM/MTMX/XLMTM |
|            | MTMR1   | a, g         | MTMR1      | Xq28              | XA7.2             | |
|            | MTMR2   | a, g         | MTMR2      | 11q22             | 9                 | CMT4B; CMT4B1 |
|            | MTMR3   | a, g         | MTMR3      | 22q12             | 11A1              | FYVE-DSP1/ZFYVE10 |
|            | MTMR4   | a, g         | MTMR4      | 17q22             | 11C               | FYVE-DSP2/ZFYVE11 |
|            | MTMR6   | a(g)         | MTMR6      | 13q12             | 14D1              | |
|            | MTMR7   | a            | MTMR7      | 8p22              | 8B1.2             | |
|            | MTMR8   | (a)          | MTMR8      | Xq12              | 14D1              | (mouse MTMR9) |
|            | MTMR14  | a, g         | MTMR14     | 3p25              | 6E3               | hJUMPY/hEDTP |
| Phosphoinositide 4-phosphatases | INPP4 | INPP4A    | d                 | INPP4A            | 2q11             | 1B                  | INPP4 type I/L-PIPase |
|            |         | INPP4B       | d          | INPP4B            | 4q31              | 8C2               | INPP4 type II/P-PIPase |
|            | TMEM55  | TMEM55A      | i          | TMEM55A           | 14q11             | 4A1-A2            | |
|            |         | TMEM55B      | i          | TMEM55B           | 8q21              | 14C1              | |
| Phosphoinositide 5-phosphatases | Type II INPP5s$^d$ | SYNJ1 | a, b, f, h, i, j | SYNJ1 | 21q22 | 16C3-C4 | synaptojanin 1/INPP5G |
|            |         | SYNJ2        | a, b, f, h, i, j | SYNJ2 | 6q25 | 17A2-A3 | synaptojanin 2/INPP5H |
|            |         | OCRL1        | h, j       | OCRL              | Xq26              | XA4               | OCRL/LOCR/NPHL2 |
|            |         | INPP5B       | h, j       | INPP5B            | 1p34              | 4D2               | 5PTase |
|            |         | INPP5J       | h, j       | INPP5J            | 22q11             | 11A1              | PIPP/PIB5PA |
|            |         | SKIP         | j,(h)      | SKIP              | 17p13             | 11B5              | PPS/INPP5K |
|            | Type III INPP5s | SHIP1 | j          | INPP5D            | 2q36              | 1C5               | INPP5D/SHIP/hp51CN/SIP-145 |
|            |         | SHIP2        | j          | INPPL1            | 11q23             | 7                 | INPPL1 |
|            | Type IV INPP5 | INPP5E | f, h, j    | INPP5E            | 9q34              | 2A3               | Pharbin/inositol polyphosphate-5-phosphatase, 72 kDa |
| Others     | PLIP    | c            | PTPMT1     | 11q11             | 2E1               | PTPMT1/MOSP |
|            | Sac     | SAC1         | a, b(f)    | SACM1L            | 3p21              | 9F                | SACM1L |
|            |         | SAC2         | h, j       | INPP5F            | 10q26             | 7F3               | INPP5F |
|            |         | SAC3         | f          | FIG4              | 6q21              | 10B1              | FIG4 |

$a$ Regulatory subunits are not listed.  
$b$ Lowercase letters correspond to phosphatase reactions illustrated in Fig. 2.  
$c$ Parentheses indicate reactions that are secondary in nature for this enzyme.  
$d$ Type I inositol polyphosphate 5-phosphatases do not hydrolyze phosphoinositides.

Kinase Reactions

Phosphatase Reactions

Fig. 2. Interconversion of phosphoinositides. Known reactions catalyzed by phosphoinositide kinases (left) and phosphoinositide phosphatases (right) are depicted. Each reaction arrow labeled with an uppercase letter is catalyzed by a phosphoinositide kinase as listed in Table 1. Each reaction arrow labeled with a lowercase letter is catalyzed by a phosphoinositide phosphatase as listed in Table 2. Reactions in grey are not yet documented.

T. Sasaki et al./Progress in Lipid Research 48 (2009) 307–343

Class I
p110α/β/δ
p110γ

| HR4 | HR3 | HR2 | HR1 |
| --- | --- | --- | --- |
| p85 binding | Ras binding | C2 | helical | Kinase |
|  |  |  |  | α: 1068 aa |
|  |  |  |  | β: 1070 aa |
|  |  |  |  | δ: 1044 aa |
| p101/p84/p87 binding |  |  |  | 1102 aa |

Class II
PI3K-C2α
PI3K-C2β
PI3K-C2γ

| PR | Ras binding | C2 | helical | Kinase | PX | C2 |
| --- | --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  | 1686 aa |
|  |  |  |  |  |  | 1634 aa |
|  |  |  |  |  |  | 1448 aa |

Class III
Vps34

| C2 | helical | Kinase |
| --- | --- | --- |
|  |  | 887 aa |

Fig. 3. Structural features of mammalian phosphoinositide 3-kinases. Domains present in the eight known mammalian PI3K catalytic subunits are shown; regulatory subunits are not shown. All class I and II PI3 Ks share four homology regions (HR 1–4) as indicated, with HR4 being a Ras-binding domain; HR3, a protein kinase C homology domain 2 (C2); HR2, a helical domain; and HR1, a kinase domain. The p85 binding and p101/p84/p87 binding domains facilitate interaction with the PI3K regulatory subunits. Class II, but not class I or III, PI3Ks also contain a proline-rich region (PR) and a phox homology domain (PX). The sole class III enzyme, Vps34, contains the C2, helical and kinase domains but lacks a Ras-binding domain. Numbers of amino acids in the human proteins are indicated.

generate PtdIns3P, PtdIns(3,4)P₂, and PtdIns(3,4,5)P₃, respectively. The eight members of mammalian PI3K family fall into three classes based on their structures and the phosphoinositides they preferentially utilize as substrates (Fig. 3 and Table 1) [5,13]. The class I PI3Ks are heterodimeric enzymes that preferentially phosphorylate PtdIns(4,5)P₂ to produce PtdIns(3,4,5)P₃. These enzymes are further subdivided into two groups, class IA and class IB, depending on their associated regulatory proteins (see below). The class II PI3Ks are monomeric proteins that predominantly phosphorylate PtdIns to give PtdIns3P. The only mammalian class III PI3K identified to date is Vps34, a homolog of Saccharomyces cerevisiae Vps34p (vacuolar protein sorting 34). Mammalian Vps34 almost exclusively synthesizes PtdIns3P from PtdIns.

### 2.2. Class I PI3Ks

#### 2.2.1. Structure

Class IA PI3Ks are heterodimeric proteins composed of a catalytic subunit and a regulatory subunit. The three class IA catalytic subunits are p110α, p110β, and p110δ. Each of these subunits contains a Ras-binding domain, a C2 domain, a helical domain, and a lipid kinase domain (refer to Fig. 3). Each catalytic subunit binds to one of five regulatory subunits called p85α, p85β, p55γ, p55α, and p50α [14]. These regulatory subunits contain Src homology 2 (SH2) domains that recognize the phosphorylated tyrosine in the YXXM motifs present in intracellular proteins such as receptors and adaptor molecules [15]. This protein-protein interaction increases the lipid kinase activity of class IA PI3Ks and leads to their recruitment from the cytosol to the membrane where the relevant substrates exist.

The sole class IB catalytic subunit is p110γ. This protein contains a p101/p84/p87 binding domain instead of a p85-binding domain but otherwise features the same collection of domains as the class IA PI3Ks. The p110γ subunit associates with either the p101, p87, or p84 regulatory subunit to form a heterodimer that is activated through interaction with the βγ subunits of heterotrimeric GTP-binding proteins [16–18].

The domains of the PI3Ks govern their regulation and functions. Both class IA and class IB PI3K catalytic subunits bind directly to and are activated by GTP-bound Ras through the Ras-binding domain [19,20]. The C2 domain of PI3Ks is thought to participate in membrane interaction. The helical domain is central to interdomain packing in which the surfaces formed by the helices interact with the catalytic domain, the C2 domain, and the Ras-binding domain [21].

#### 2.2.2. Expression

The p110α and p110β PI3Ks are expressed ubiquitously, whereas p110δ and p110γ are found almost exclusively in immune system cells. In leukocytes, p110γ accounts for almost all of the PI3K activity required for PtdIns(3,4,5)P₃ synthesis induced downstream of G-protein coupled receptor engagement [22–24]. In cell types that do not express detectable levels of p110γ, this function is taken over primarily by p110β [25–27].

#### 2.2.3. Biological functions of p110α and p110

##### 2.2.3.1. Development

Complete inactivation of either p110α or p110β in knockout mice results in embryonic lethality at embryonic day 10.5 (E10.5) or E3.5, respectively, demonstrating that both these isozymes are essential for embryogenesis [28,29]. The death of p110α knockout mice (p110α⁻/⁻ mice) can be attributed to failed developmental angiogenesis (see below), but the mechanism underlying the embryonic death observed in the absence of p110β has not been elucidated.

##### 2.2.3.2. Metabolic regulation

Activation of class IA PI3Ks predominantly generates PtdIns(3,4,5)P₃, which is a crucial activator of phosphoinositide-dependent kinase-1 (PDK1) and thus Akt. Akt is a kinase with critical roles in cell survival and metabolism. The class IA PI3Ks have been implicated in insulin-mediated effects such as inhibition of hepatic gluconeogenesis and glucose uptake into muscle and fat [30,31]. Indeed, upon stimulation of the insulin receptor, insulin receptor substrate (IRS) proteins become phosphorylated and activate class IA PI3Ks. In humans, a polymorphism in the p85α regulatory subunit gene is associated with increased risk for type 2 diabetes [32].

With respect to p110α, Foukas et al. have reported that mice heterozygous for a substitution mutation D933A that inactivates p110α kinase activity are abnormally small and show increased plasma insulin levels after fasting [33]. These animals, termed
p110α<sup>D933A/WT</sup> mice (WT, wild-type), are also resistant to the glucose-lowering effects of insulin and display impaired glucose clearance. The same effects can be achieved by giving mice a chemical inhibitor of p110α activity but not an inhibitor of p110β activity [34]. In p110α<sup>D933A/WT</sup> mice, IRS-1/2-associated PI3K activity and Akt phosphorylation are reduced in the liver, muscle and fat. Other intriguing metabolic phenotypes associated with genetic or chemical inhibition of p110α activity in mice include hyperphagia with increased plasma leptin levels and impaired insulin-stimulated glucose uptake in adipocytes [33,34].

Metabolic phenotypes have also been reported in mice with a conditional p110β deficiency in the liver [25], and in p110β<sup>K805R/K805R</sup> mice that express a kinase-dead (KD) p110β protein [27]. Like p110α<sup>D933A/WT</sup> mice, these mutants exhibit insulin resistance, glucose tolerance, hyperinsulinaemia and hyperleptinaemia. However, p110β is not activated by insulin, since Akt phosphorylation is normal both in p110β-deficient liver subjected to insulin challenge and in p110β<sup>K805R/K805R</sup> mouse embryonic fibroblasts (MEFs) stimulated with insulin or insulin-like growth factor (IGF-1). Mice with a double heterozygous loss of p110α and p110β show hyperinsulinaemia and glucose intolerance [35]. Thus, p110α and p110β operate via distinct mechanisms that cooperate to maintain glucose homeostasis.

### 2.2.3.3. Angiogenesis

Graupera et al. have reported a selective requirement for p110α in angiogenesis [36]. Mice homozygous for the D933A mutation that inactivates p110α kinase activity (p110α<sup>D933A/D933A</sup> mice) show growth retardation from E9.5 onward and die *in utero* by mid-gestation. Although the initial stages of vascular development occur normally in p110α<sup>D933A/D933A</sup> embryos, angiogenic sprouting and vascular remodeling are severely impaired. Conditional homozygous deletion of p110α specifically in endothelial and endocardial cells causes similar vascular defects. Heterozygous p110α<sup>D933A/WT</sup> mice show normal proliferation and survival of cardiac endothelial cells, but reduced microvessel outgrowth in aortic ring explants. In contrast, mice with a conditional deletion of p110β specifically in endothelial cells show normal angiogenesis and are viable and fertile. Thus, p110α plays a unique role in developmental angiogenesis and is required in a cell-autonomous manner in endothelial cells to promote vascular remodeling.

### 2.2.4. Biological functions of p110γ and p110δ

The p110γ (class IB) and p110δ (class IA) PI3Ks are expressed predominantly in hematopoietic cells. These isoforms are dispensable for embryogenesis because p110γ<sup>−/−</sup> and p110δ<sup>−/−</sup> mice, as well as mice with KD versions of these proteins, are all viable [22,37–39]. However, p110γ and p110δ do play important roles in specific hematopoietic and some non-hematopoietic cell types.

#### 2.2.4.1. Leukocytes

In leukocytes, p110γ is indispensable for the PtdIns(3,4,5)P<sub>3</sub> synthesis induced by the engagement of G protein-coupled receptors for chemokines, leukotrienes and bacterial chemoattractants [22–24,40]. This PI3K isoform is therefore important for leukocyte chemotaxis, superoxide production and extravasation into inflamed tissues [41]. p110δ has also been implicated in leukocyte chemotaxis [42].

#### 2.2.4.2. Mast cells

Both p110γ and p110δ are critical for the mast cell degranulation that drives allergic and anaphylactic reactions. FcεRI engagement on the mast cell surface by IgE-antigen complexes results in direct activation of p110δ via this isoform's SH2-containing regulatory subunit p85. *In vitro*, both human mast cells treated with a p110δ-specific inhibitor, and bone marrow-derived mast cells from p110δ<sup>KD/KD</sup> mice, exhibit a defect in FcεRI-induced degranulation [43]. *In vivo*, genetic inactivation or pharmacological inhibition of p110δ in mice significantly reduces anaphylaxis. Although p110γ lacks the structure that allows direct activation in response to FcεRI binding to IgE-antigen, p110γ<sup>−/−</sup> mice are also resistant to anaphylaxis due to the failure of their activated mast cells to respond to adenosine [44]. Adenosine potentiates mast cell degranulation in response to IgE-allergen and this costimulatory signal is totally dependent on p110γ activity.

#### 2.2.4.3. Lymphocytes

p110δ is critical for T cell receptor (TCR) and B cell receptor (BCR) signaling and is responsible for the PI3K activity required at the interface of a T cell and its antigen-presenting cell [45]. Genetic inactivation of p110δ attenuates the proliferation of both T and B cells, whereas p110γ<sup>−/−</sup> mice have defects in thymocyte survival and T cell proliferation but not in B cell proliferation [22,37,39,46]. Double mutant mice lacking both p110γ and p110δ (either p110γ<sup>−/−</sup>p110δ<sup>−/−</sup> or p110γ<sup>−/−</sup>p110δ<sup>KD/KD</sup> animals) exhibit much more severe T cell defects than do single mutants lacking either p110γ or p110δ activity [47,48]. These defects include: decreased thymocyte development, increased thymocyte apoptosis, reduced numbers of peripheral T cells, a strikingly Th2-skewed profile, and impaired regulatory T cell functions. p110γ<sup>−/−</sup>p110δ<sup>KD/KD</sup> mice also develop inflammation in multiple organs. Thus, although p110γ and p110δ both generate PtdIns(3,4,5)P<sub>3</sub>, these isoforms appear to participate in distinct signaling pathways that act synergistically to regulate T cell responses. The underlying molecular mechanisms are not yet understood.

#### 2.2.4.4. Non-hematopoietic cells

In cardiomyocytes, p110γ acts as a negative regulator of cardiac contractility [38,49]. In vascular endothelial cells, both p110γ and p110δ modulate the proadhesive state of the endothelium, and thereby influence the exit of neutrophils from the vascular system into the tissues [50,51]. In renal tubular epithelial cells, p110γ reduces apoptosis and plays a role in the maintenance of renal function in cisplatin-induced acute kidney injury [52].

### 2.2.5. Disease association

#### 2.2.5.1. Cancer

Phosphoinositide 3-kinase activity was first described in association with viral oncoproteins or proto-oncogene products [53,54]. It is now evident that the class IA PI3K isoforms play critical roles in human cancers because they produce the PtdIns(3,4,5)P<sub>3</sub> that drives Akt activation and promotes tumor cell survival and proliferation. However, Samuels et al. screened 35 human colon cancers for somatic mutations in the genes encoding the eight PI3K family members and detected such mutations only in PIK3CA, which encodes p110α [55]. To date, 18 point mutations causing single amino acid substitutions at 12 different sites in p110α have been found in human cancers, as has amplification of the chromosomal region containing PIK3CA [56,57]. Curiously, although p110α is ubiquitously expressed, p110α mutations occur with high frequency only in breast and colorectal cancers. Three mutation hotspots have been identified in the human p110α protein: E542, E545 and H1047. E542 and E545 lie in the p110α helical domain whereas H1047 is in the lipid kinase domain. With respect to the human enzymes, p110α proteins bearing an E542K, E545K or H1047R mutation all show elevated PI3K activity *in vitro*, drive Akt-S6 kinase (S6K) signaling, and induce oncogenic transformation when overexpressed in fibroblasts. Interestingly, PIK3CA mutations and mutations of the key tumor suppressor PTEN (phosphatase and tensin homologue deleted on chromosome 10) occur in breast cancers in a mutually exclusive fashion [58].

Wortmannin and LY294002 are chemical inhibitors that block the activities of all PI3K enzymes and induce cell cycle arrest and cell death. These inhibitors have been used extensively to evaluate PI3K functions. In addition, new isoform-specific, “small molecule” inhibitors have been developed to target cancer cells drivenby excessive PI3K activation. For example, Fan et al. used an array of ten such PI3K inhibitors to investigate which PI3K isoform was critical for the proliferation of human glioma xenografts *in vivo* [59]. PIK-90, which selectively inhibits p110α, arrested proliferation when used in concert with the mTOR inhibitor rapamycin; this cooperative efficacy was not observed when selective inhibitors of p110β were used. Another inhibitor PI-103 blocks p110α to the same degree as PIK-90 but is also effective against mTOR; this inhibitor blocked glioma proliferation when used as a single agent. These studies show that the growth of gliomas depends on both mTOR and PI3K activities, and that p110α is the major isoform driving the proliferation of this tumor cell type.

The *PIK3CB* gene, which encodes the p110β catalytic subunit, has not been found to be mutated in human cancers [55]. However, MEFs lacking p110β are totally resistant to transformation by oncogenic H-Ras and the epidermal growth factor receptor (EGFR) [25]. In addition, RNAi-mediated depletion of p110β leads to inhibition of tumor growth and PI3K signaling in human cancer cell lines harboring null *PTEN* alleles [60]. Lastly, in a murine prostate cancer model induced by *Pten* loss, conditional ablation of p110β completely inhibited tumor formation with a concomitant suppression of Akt phosphorylation [25]. Loss of p110α in the same cancer model had no impact on tumor formation or on Akt phosphorylation. In line with these observations, Ciraulo et al. have reported that mice expressing a catalytically inactive form of p110β are protected from breast cancer development triggered by ERBB2, an oncogene known to signal through PI3K [27]. Thus, different class IA PI3K isoforms appear to play dominant roles in different cancers in different genetic contexts.

### 2.2.5.2. Inflammatory disorders

At least in mouse models, the level of p110γ activity determines the severity of tissue damage associated with acute or chronic inflammatory diseases [61–63]. This involvement is consistent with the biological function of this isoform in leukocyte chemotaxis and extravasation. Indeed, p110γ-deficient mice show ameliorated symptoms in models of atherosclerosis [64], myocardial ischemia/reperfusion injury [65], rheumatoid arthritis [66], systemic lupus erythematosus [67] and asthma [68]. Accordingly, small molecule inhibitors of p110γ and p110δ have been developed for the treatment of human inflammatory diseases [69] and phase I/II clinical trials are currently underway [70].

## 2.3. Class II PI3Ks

### 2.3.1. Structure

The class II PI3Ks preferentially phosphorylate PtdIns to generate PtdIns3P but can also phosphorylate PtdIns4P to yield PtdIns(3,4)P₂. There are three class II PI3Ks, called PI3K-C2α, PI3K-C2β and PI3K-C2γ, that are monomeric enzymes (~170 kDa) with structural similarity to the class I PI3Ks (refer to Fig. 3). The shared domains include the Ras-binding domain, the C2 domain, the helical domain and the lipid kinase domain. All three class II PI3Ks also possess a phox (PX) domain and a C2 domain at their C-termini. The N-terminal sequences are unique to each isoform but all three classes contain at least two proline-rich (PR) regions. PI3K-C2α also has a clathrin-binding site [71].

### 2.3.2. Expression

PI3K-C2α is expressed in many mammalian tissues and is localized primarily in the trans-Golgi network (TGN) with some expression in the nucleus [72,73]. PI3K-C2β is also widely expressed in mammalian tissues, as determined by Northern blotting [74]. Endogenous PI3K-C2β is distributed across the cytoplasm and localized around the nucleus in a punctate manner [75]. PI3K-C2γ is expressed in the liver, breast and prostate in humans [76], in the liver in rats [77], and in the liver and kidney in mice [78]. In COS-7 cells, PI3K-C2γ is detected mainly in the Golgi [77].

### 2.3.3. Biological functions

#### 2.3.3.1. PI3K-C2α

In vitro, PI3K-C2α catalyzes the phosphorylation of PtdIns and PtdIns4P to give rise to PtdIns3P and PtdIns(3,4)P₂, respectively [79]. The N-terminus of PI3K-C2α interacts with clathrin in a way that increases PI3K-C2α's lipid kinase activity [80]. This interaction also has the effect of localizing PI3K-C2α to clathrin-coated pits in the plasma membrane and the TGN, suggesting that PI3K-C2α regulates clathrin-mediated membrane trafficking [80,81]. Indeed, Zhao et al. have reported that such traffic is modulated by an association between PI3K-C2α and the microtubule motor machinery [82]. Although PI3K-C2α has also been detected in the nucleus, its role here is not well understood [73].

Other in vitro studies have indicated that PI3K-C2α is involved in several additional functions, including neurosecretion [83], apoptotic cell death [84,85], and Ca²⁺-induced, Rho-dependent vascular smooth muscle contraction [86]. It has also been suggested that extracellular signaling mediated by EGF [87], insulin [88], chemokines [89], tumor necrosis factor-α (TNFα) or leptin [90] can activate PI3K-C2α in cell cultures. This hypothesis has been bolstered by the finding that PI3K-C2α is involved in the insulin-induced translocation of the GLUT4 glucose transporter to the plasma membrane [91]. Despite this wealth of in vitro data, however, the physiological roles of PI3K-C2α remain largely unknown.

#### 2.3.3.2. PI3K-C2β

A variety of functions have been reported for PI3K-C2β. In vitro, PI3K-C2β phosphorylates PtdIns and PtdIns4P but prefers PtdIns [92]. Maffucci et al. have shown that PI3K-C2β is involved in lysosphosphatidic acid (LPA)-induced cell migration [93], whereas two other groups have found that PI3K-C2β regulates cell migration through its product PtdIns3P and a Rac-dependent signaling pathway [94,95]. It is also thought that PI3K-C2β may play a role in growth factor-related vesicle traffic because certain growth factors, including EGF and platelet-derived growth factor (PDGF), stimulate the association of PI3K-C2β with the EGF receptor (EGFR) and PDGFR [87], and PI3K-C2β can bind to clathrin [75]. Clathrin-mediated endocytosis is known to be responsible for the internalization of many receptors, including EGFR. Other factors, such as stem cell factor (SCF) [96] and various integrins [97], also activate PI3K-C2β. Finally, there is evidence that PI3K-C2β-mediated activation of Akt signaling regulates neuronal cell survival through interaction with intersectin [98]. Again, however, despite these in vitro findings, the physiological functions of PI3K-C2β are largely unclear. Harada et al. have generated transgenic mice overexpressing PI3K-C2β in epidermal tissues, as well as mice lacking PI3K-C2β in all tissues [99], but both strains are born alive at the expected Mendelian ratio, survive to adulthood, and display no overt abnormalities.

#### 2.3.3.3. PI3K-C2γ

Although PI3K-C2γ is highly expressed in liver, and Ono et al. have suggested that PI3K-C2γ may play a role in liver regeneration in the rat [77], the precise biological function of this enzyme remains to be elucidated.

## 2.4. Class III PI3K

The sole mammalian class III PI3K is Vps34. This molecule is a homolog of the protein encoded by the *Saccharomyces cerevisiae* gene Vps34 that was first identified in a screen for genes involved in vacuolar trafficking [100].

### 2.4.1. Structure

Mammalian Vps34 contains a C2 domain, a helical domain, and a lipid kinase domain. Each of these domains is approx. 30% homol-ogous to the class IB PI3K/p110γ (refer to Fig. 3). However, Vps34 lacks all other domains shared by the class I or II PI3Ks. Although the helical domain has a regulatory function in the class I PI3Ks, the function of the Vps34 helical domain is unclear. Vps34 also lacks the multiple basic residues that are conserved in the substrate recognition loop of all class I PI3Ks, perhaps accounting for Vps34’s very limited substrate specificity: only the phosphorylation of PtdIns to generate PtdIns3P is catalyzed by Vps34 (refer to Fig. 2 and Table 1). Budovskaya et al. have reported that the extreme C-terminal 11 residues of yeast Vps34p are required for its lipid kinase activity, and that these residues are conserved in Vps34 homologs from yeast to humans [101]. In vitro, Volinia et al. have demonstrated that human Vps34 activity is higher in the presence of Mn²⁺ than in the presence of Mg²⁺, distinguishing Vps34 from the class I PI3Ks [102].

### 2.4.2. Expression

The human Vps34 mRNA is expressed ubiquitously but is present in its highest levels in heart and skeletal muscle [102]. The subcellular localization of endogenous Vps34 remains uncertain. Analysis of RFP-fusion protein expression in HEK-293 cells has revealed that Vps34 exists in a cytosolic and a vesicular form [103]. The Vps34 vesicles are in close proximity to the ER and correspond to LAMP2-positive late endosomes/lysosomes.

### 2.4.3. Biological functions

In mammals, Vps34 is important both for vesicular trafficking in the endosomal/lysosomal system and for autophagy. In HeLa cells expressing a KD form of Vps34, the maturation of procathepsin D and its trafficking to the lysosomes is blocked [104]. Related to this observation, injection of anti-Vps34 antibodies into polarized WIF-B hepatic cells induces the formation of vacuoles that contain both lysosomal membrane proteins and apical (but not basolateral) plasma membrane proteins, suggesting that Vps34 specifically regulates the dynamics of apical plasma membrane proteins [105]. In U-251 glioblastoma cells, siRNA knockdown of Vps34 causes an accumulation of similar vacuoles resembling late endosomes/lysosomes, and inhibits proliferation [106]. Vps34’s function in the regulation of autophagy was originally demonstrated in yeast [107] and in *Drosophila* [108] by genetic analyses. In mammals, studies using 3-methyl adenine, a Vps34 inhibitor, have shown that the activity of this enzyme is required for starvation-induced autophagy [109].

Vps34 also plays important roles in innate immune responses to pathogens, functioning in the recognition of the threat [110], the process of phagocytosis [111,112], and the formation of superoxide that destroys the invaders [113,114].

### 2.4.4. Regulation

#### 2.4.4.1. Interacting proteins

Several proteins interact with mammalian Vps34 and affect its biological activity. Vps15 was first identified in *S. cerevisiae* [115] and its ortholog p150 is conserved in *Drosophila melanogaster*, mice and humans [116]. Structurally, p150 resembles a serine/threonine kinase but it is unclear whether p150 actually has this activity. The association of p150 with Vps34 increases Vps34 activity in vitro [116] and siRNA knockdown of p150 decreases protein levels of Vps34 [117], suggesting that p150 is important for both the activity and stability of Vps34.

Five proteins are involved in Vps34-associated autophagy: Beclin-1, Bcl2, Ambra1, UVRAG (UV irradiation resistance associated gene) and Bif-1. Beclin-1, which was first identified as a Bcl2-interacting tumor suppressor [118] and is conserved in all eukaryotes, binds directly to the C2 domain of Vps34 [119]. The yeast ortholog of Beclin-1 is ATG6, a member of the autophagy-related Atg gene family [120]. Like yeast strains with ATG6 mutations, Beclin-1-deficient mouse embryonic stem (ES) cells are defective in their

autophagic response to nutrient deprivation [121]. Ambra1 (activating molecule in Beclin-1 regulated autophagy) is an adaptor protein that binds to Beclin-1 [122]. siRNA knockdown of Ambra1 inhibits Beclin-1-dependent autophagy and the binding of Beclin-1 to Vps34. Similarly, overexpression of Bcl2 inhibits Beclin-1-dependent autophagy and decreases the binding of Beclin-1 to Vps34. Thus, Ambra1 and Bcl2 may control the onset and resolution of autophagy by their opposing regulatory effects on Beclin-1-associated Vps34 activity.

UVRAG is a component of the Beclin-1–Bcl2–Vps34 complex required for autophagy [123]. UVRAG contains a PR region, a C2 domain and a coiled-coil domain, with the latter mediating UVRAG’s binding to Beclin-1. siRNA knockdown of UVRAG inhibits autophagy. Overexpression of UVRAG enhances Vps34 activity when Beclin-1 and Vps34 are co-overexpressed. The binding of UVRAG to Beclin-1 is not affected by amino acid starvation.

The PR region of UVRAG allows it to bind directly to the SH3 domain of Bif-1 [124], a protein that also has a BAR (Bin-Amphiphysin-Rvs) domain that binds to membranes and shapes their curvature [125]. Bif1⁻/⁻ MEFs exhibit impaired autophagy and decreased Vps34 activity under both nutrient-rich and nutrient-poor conditions [124]. Bif1⁻/⁻ mice develop normally but have a higher incidence of malignancies such as lymphomas and carcinomas of the liver and lung.

#### 2.4.4.2. Amino acids

Controversy exists over whether amino acids themselves increase or decrease Vps34 activity. On one hand, amino acid depletion promotes autophagy and Vps34 activation as described above. However, Byfield et al. have reported that Vps34 is a critical component of the nutrient-induced signaling pathway that leads to S6K1 activation [126]. Furthermore, Nobukuni et al. have demonstrated that amino acids activate mTOR through Vps34 [127]. Since mTOR activity is known to inhibit autophagy, additional studies are needed to resolve this issue.

### 2.4.5. Disease association

A candidate schizophrenia gene is located within the human chromosomal region D18S450-D18S487, a region that also includes the human *PIK3C3* gene that encodes the Vps34 protein [128]. Indeed, Stopkova et al. have demonstrated an association between the –432C > T polymorphism of the promoter region of the *PIK3C3* gene and schizophrenia in a cohort of Israeli patients [129]. In another study of schizophrenia, preferential transmission of the –432C *PIK3C3* allele was detected in a group of male patients but not in a group of female patients [130].

## 3. Phosphatidylinositol-4 kinases (PI4Ks)

### 3.1. Classification

All phosphatidylinositol 4-kinases (PI4Ks) phosphorylate the D4 position of the inositol ring of PtdIns and produce PtdIns4P. This reaction is critical for the synthesis of derivatives with a greater degree of phosphorylation, including PtdIns(4,5)P₂ and PtdIns(3,4,5)P₃. The first chromatographic purifications of phosphoinositide kinase activity from bovine brain and cultured murine fibroblasts resulted in the isolation of three fractions (I, II and III) deemed to contain PI4K activities [131,132]. However, it was subsequently discovered that, although the type II and type III fractions did indeed contain distinct PI4K enzymes [133], the type I fraction actually contained a PI3K enzyme [53].

To date, based on their domain structures and sensitivity to inhibitors, four distinct mammalian PI4Ks have been identified and are classified as follows: the type II isozymes, PI4KIIα and PI4KIIβ, and the type III isozymes, PI4KIIIα and PI4KIIIβ [134]. Type III
PI4Ks have a C-terminal lipid kinase domain similar to that of PI3Ks and are sensitive to PI3K inhibitors such as wortmannin [135]. In contrast, the catalytic region of type II PI4Ks differs substantially from that of PI3Ks and type III PI4Ks, and is insensitive to wortmannin [135]. As is described below, each PI4K isozyme localizes to a specific cellular compartment in a specific tissue, suggesting that each isozyme produces the PtdIns4P required for a different cellular function.

### 3.2. Type II PI4Ks

#### 3.2.1. Structure
Both PI4KIIα and PI4KIIβ feature a lipid kinase domain split in two by a long insert (Fig. 4). The N-terminal half of the kinase domain contains a cysteine-rich (CR) domain that is palmitoylated [136]. This palmitoylation likely facilitates the membrane targeting of these enzymes [137]. In addition to its palmitoylation, membrane-associated PI4KIIβ is phosphorylated but the function of this modification is not clear.

The major differences between the type II PI4K isozymes are found in the N-terminal region. PI4KIIα has a PR region, whereas PI4KIIβ contains a highly acidic region [138]. The N-terminus of PI4KIIα is required for its targeting to specific membranes, including to cellugyrin-positive GLUT4 vesicles [139]. The organelle targeting mediated by the N-terminus of PI4KIIβ remains to be elucidated.

#### 3.2.2. Expression
Both PI4KIIα and PI4KIIβ appear to be ubiquitously expressed in human tissues. Northern blotting of human tissues has shown that the abundance of PI4KIIβ mRNA is greatest in liver, and that PI4KIIα mRNA levels are higher than those of PI4KIIβ mRNA in brain [138,140].

At the subcellular level, immunocytochemical analysis has demonstrated that palmitoylated PI4KIIα is localized to the membranes of the endoplasmic reticulum (ER), TGN and endosomes [141,142]. Palmitoylated PI4KIIβ is also primarily associated with the membranes of the ER and Golgi but, unlike PI4KIIα, can be recruited from the cytosol to the plasma membrane in a Rac1-dependent manner when cells are stimulated with PDGF [138,143].

#### 3.2.3. Biological functions
PI4KIIα is thought to function in vesicle trafficking. In the TGN, the local production of PtdIns4P by this enzyme promotes PtdIns4P binding by the clathrin adaptor AP-1 and the recruitment of these AP-1 complexes to the membrane. Accordingly, RNAi-mediated silencing of the PI4KIIα decreases the amount of PtdIns4P in the Golgi and blocks AP-1 complex recruitment [142]. PI4KIIα is also reportedly required for the trafficking of EGFR to the late endosomes, where this protein is degraded [144]. Lastly, Guo et al. have described the association of PI4KIIα with synaptic vesicles, indicating that the PtdIns4P produced by this isozyme may function in neurotransmission [145]. Compared to PI4KIIα, PI4KIIβ may be more directly involved in intracellular signaling since PI4KIIβ has been found to associate with the TCR [146]. Both PI4KIIα and PI4KIIβ may function in phagocytosis [147].

### 3.3. Type III PI4Ks

#### 3.3.1. Structure
Two type III PI4Ks exist: PI4KIIIα (~230 kDa) and PI4KIIIβ (~92 kDa) (refer to Fig. 4). Both contain a continuous C-terminal lipid kinase domain similar to that of the PI3Ks, and both are sensitive to wortmannin. Unlike type II PI4Ks, type III PI4Ks contain a “lipid kinase unique” (LKU) domain that is homologous to the helical domain in the PI3Ks [134,148]. PI4KIIIα and PI4KIIIβ differ in that PI4KIIIα contains a bipartite nuclear localization sequence (NLS) and a pleckstrin homology (PH) domain that may be responsible for the association of this isozyme with membranes. In contrast, PI4KIIIβ features a Rab-binding region located between its LKU and its lipid kinase domain. PI4KIIIβ-Rab association may regulate membrane transport from the Golgi complex to the plasma membrane [149].

#### 3.3.2. Expression
In the rat, PI4KIIIα mRNA is detected primarily in the brain but is also found at low levels in many other organs [150]. Electron microscopic analysis of rat neurons has revealed that PI4KIIIα is localized to membranes of various organelles, such as the ER, mitochondria, and multivesicular bodies [151]. PI4KIIIα can also be detected in the nucleus of rat neuronal cells [152].

Unlike PI4KIIIα, PI4KIIIβ is ubiquitously expressed in rat tissues [150]. Electron microscopy of neurons and analysis of GFP-fusion protein expression in Madin-Darby canine kidney (MDCK) cells has revealed that PI4KIIIβ is present in early Golgi compartments [151,153]. The presence of PI4KIIIβ in the nucleus has also been reported [154]. This variable localization appears to depend on differential phosphorylation of the PI4KIIIβ protein, which has eight potential phosphorylation sites. For example, PI4KIIIβ phosphorylated on Ser294 is targeted exclusively to the Golgi, whereas phosphorylation of the enzyme on Ser496 or Thr504 results in its detection in the nucleus [155].

#### 3.3.3. Biological functions
The wortmannin-sensitive PI4Ks regulate hormone-sensitive PtdIns(4,5)P₂ pools and thus govern intracellular concentrations of cytoplasmic inositol 1,4,5-trisphosphate (Ins(1,4,5)P₃) and Ca²⁺ [156]. A recent pharmacological study has indicated that PI4KIIIα, but not of PI4KIIIβ or PI4KIIα, is indispensable for the production of PtdIns4P and PtdIns(4,5)P₂ at the plasma membrane, as well as for Ca²⁺ signaling [157]. The specific functions of PI4KIIIα in the ER, nucleus and mitochondria have yet to be elucidated.

PI4KIIIβ is recruited to the Golgi by binding to the small GTPase ADP-ribosylation factor-1 (ARF1). In the Golgi, PI4KIIIβ is phosphorylated on Ser294 by protein kinase D1 (PKD1) or PKD2 but not by PKD3. This phosphorylation increases the lipid kinase activity of PI4KIIIβ [158], resulting in vigorous production of PtdIns4P. That PtdIns4P production by PI4KIIIβ at the Golgi is required for efficient TGN-to-plasma membrane transport has been shown in experiments examining the transport of vesicular stomatitis virus

glycoprotein (VSV-G) to the plasma membrane. Overexpression of WT PI4KIIIβ increased VSV-G transport, whereas the expression of a PKD1 KD mutant reduced VSV-G transport [158].

Additional evidence supporting a role for PI4KIIIβ in membrane transport comes from studies of the four-phosphatase-adaptor proteins 1 and 2 (FAPP1 and FAPP2). FAPP1 and FAPP2 bind to ARF1 and PtdIns4P [159,160] and are normally localized to the TGN. Knockdown of FAPP expression inhibits the transport of VSV-G to the plasma membrane, suggesting that ARF1, PtdIns4P and FAPPs regulate the traffic of proteins traveling from the TGN to the plasma membrane. Furthermore, PI4KIIIβ interacts with the GTP-bound form of the small GTPase Rab11, and inhibition of this interaction abolishes Rab11 localization to the Golgi and blocks VSV-G transport to the plasma membrane [149]. Thus, an association between PI4KIIIβ and Rab11 is important for transport from the Golgi to the plasma membrane.

Although PI4KIIIβ is also localized in the nucleus, its function here is not understood.

---

### 4. Phosphatidylinositol phosphate kinases (PIPKs)

#### 4.1. Classification

Phosphatidylinositol phosphate kinases (PIPKs) do not share significant homology with any other known lipid or protein kinases. Three distinct classes whose members exhibit significant sequence similarity in their catalytic regions have been identified. The type I PIPKs and type II PIPKs take two different routes to produce PtdIns(4,5)P₂, whereas the sole type III PIPK predominantly produces PtdIns(3,5)P₂. Type I PIPKs (PIPKIs) are PtdIns4P 5-kinases that catalyze the phosphorylation of PtdIns4P at the D5 position of the inositol ring (refer to Fig. 2 and Table 1) [161–163]. Type II PIPKs (PIPPIIs) are PtdIns5P 4-kinases that phosphorylate PtdIns5P at the D4 position [164–168]. The type III PIPK (PIPKIII) phosphorylates PtdIns and PtdIns3P at the D5 position to produce PtdIns5P and PtdIns(3,5)P₂, respectively [169].

---

#### 4.2. Type I PIPKs

##### 4.2.1. Structure

Three mammalian PIPKI isoforms, α, β, γ, have been identified [161–163]. In addition, the mRNA encoding the PIPKIγ isoform can be alternatively spliced to generate the PIPKI γ635, γ661 and γ687 isoforms [163,170]. The nomenclature of PIPKIα and β is reversed between human and mouse; mouse PIPKIα is equivalent to human PIPKIβ, and vice versa. In this review, we use the mouse PIPKI nomenclature throughout.

All PIPKI isoforms share a central kinase core (PIPKc) domain (Fig. 5) that shows approximately 80% amino acid identity among these proteins [163]. In the C-terminal part of the PIPKc domain, there is a 25 amino acid activation loop that topologically corresponds to the activation loop of protein kinases [171] and is critical for both the substrate specificity and subcellular targeting of PIPKs [172,173]. In contrast to the conserved PIPKc sequence, the N-terminal and C-terminal regions of the PIPKI isoforms differ considerably in length and amino acid sequence, except for a C-terminal region of partial homology (approximately 40% identity) between PIPKIβ and PIPKIγ [163]. The alternative splicing of PIPKIγ results in the PIPKIγ635 protein, which contains 635 amino acids; the PIPKIγ661 protein, which possesses an additional 26 amino acids tacked onto the C-terminus of PIPKIγ635; and the PIPKIγ687 protein, which contains an additional 26 amino acids tacked onto the C-terminus of PIPKIγ661 [163,170].

##### 4.2.2. Expression

Northern blot analyses of multiple mouse and human tissues have demonstrated that PIPKIα and β are both widely expressed but have differing patterns of tissue distribution. For example, although the α isoform is barely detectable in skeletal muscle, this tissue is the site of the highest expression of the β isoform [161,162]. PIPKIγ is preferentially expressed in brain, based on Northern and Western blot analyses [163,174].

##### 4.2.3. Biological functions

PtdIns(4,5)P₂ is involved in a wide variety of biological processes, including development, endocytosis, actin dynamics, chemotaxis, and Ins(1,4,5)P₃ formation. The pools of PtdIns(4,5)P₂ supporting these processes are produced in large part by the five known PIPKI isoforms. The distinct subcellular localizations of these proteins indicate that specific PIPKIs interact with particular components in a given subcellular compartment and produce function-specific pools of PtdIns(4,5)P₂.

###### 4.2.3.1. Development

PIPKIα⁻/⁻ mice are born at the anticipated Mendelian ratio, are healthy and fertile, and show no developmental abnormalities [175]. Although PIPKIβ⁻/⁻ mice also appear grossly normal and survive to adulthood, both PIPKIβ⁺/⁻ and PIPKIβ⁻/⁻ mice are born at less than the expected ratio when heterozygous PIPKIβ⁺/⁻ mice are intercrossed [176]. This decrease in fertility is even more striking if the PIPKIβ⁺/⁻ mice are of the PIPKIα⁻/⁻ background [177].

The role of PIPKIγ in embryonic development has yet to be clarified. Di Paolo et al. originally reported on a strain of PIPKIγ⁻/⁻ mice lacking PIPKIγ exons 2–6, which encode most of the catalytic region. These mutants were born alive but died within 24 h [178]. In contrast, PIPKIγ⁻/⁻ mice generated by Wang et al. using a gene trap strategy exhibit embryonic lethality at E11.5 that is accompanied by a cardiovascular defect [179]. The PIPKIγ⁻/⁻ embryos in this case also show impaired neural tube closure that is associated with defects in PtdIns(4,5)P₂ production, adherens junction formation and neural cell migration. The discrepancy in the phenotypes of these two strains of PIPKIγ-deficient mice may be due to differences in genetic background or to the degree of deletion of the PIPKIγ gene.

Recently, Pan et al. used an siRNA-based screening method in cell cultures to examine the Wnt signaling pathway important for embryonic development. These researchers found that PIPKIs are required for the phosphorylation of LDL receptor-related protein-6 (LRP6) induced by Wnt3a [180].

###### 4.2.3.2. Endocytosis

PtdIns(4,5)P₂ binds directly to several endocytic proteins and thereby regulates their functions [181–183]. To date, all three PIPKI isoforms have been linked to clathrin-mediated endocytosis, although in different pathways. PIPKIα, but not PIPKIβ or PIPKIγ, regulates routine constitutive endocytosis, as evidenced by overexpression and RNAi knockdown experiments [184]. In contrast, PIPKIβ and PIPKIγ are active in stimulation-induced endocytosis pathways in non-neuronal cells and neurons, respectively [174,185,186]. For example, in NR6 cells, overexpression of PIPKIβ results in increased endocytosis of EGFR, whereas a KD mutant of PIPKIβ inhibits this endocytosis [186]. PIPKIγ mainly contributes to the pool of PtdIns(4,5)P₂ found in synaptic nerve terminals, and analyses of PIPKIγ⁻/⁻ mice have demonstrated that PIPKIγ plays essential roles in both the exocytosis and endocytosis of synaptic vesicles [174,178].

It has been reported that the μ subunits of both the AP2 and AP1B complexes interact with PIPKIγ661 through a tyrosine-based motif in its C-terminal region [187,188]. However, Krauss et al. found that this μ2 binding site is present within the PIPKc domain of all PIPKI isoforms [189]. It may be that PIPKIγ661 contains two

T. Sasaki et al./Progress in Lipid Research 48 (2009) 307–343

| PIPKI α | PIPKc | 539 aa |
| --- | --- | --- |
| PIPKI β |  | 546 aa |
| PIPKI γ 635 |  | 635 aa |
| PIPKI γ 661 |  | 661 aa |
| PIPKI γ 687 |  | 687 aa |

| PIPKII α | PIPKc | 406 aa |
| --- | --- | --- |
| PIPKII β |  | 416aa |
| PIPKII γ |  | 421aa |

| PIPKIII /PIKfyve | FYVE | DEP | TCP-1 | PIPKc | 2052 aa |
| --- | --- | --- | --- | --- | --- |

Fig. 5. Structural features of mammalian phosphatidylinositol phosphate kinases. The only recognizable domain present in all PIPKs is the highly conserved kinase core domain (PIPKc). The PIPKc domain is bisected by a non-conserved sequence in the type I and II PIPKs (not shown). PIPKIII differs from the PIPKIs and PIPKIIIs in containing a FYVE (Fab1p, YOTB, Vac1p and EEA1) domain, a DEP (dishevelled, Egl-10 and pleckstrin) domain and a TCP-1 (t-complex polypeptide-1) domain. Numbers of amino acids in the human proteins are indicated.

μ2 binding sites that show different binding properties under different experimental conditions [188]. In addition, Nakano-Kobayashi et al. have shown that the β2 subunit of the AP2 complex binds directly to the C-terminal tail of PIPKIγ661 and potently enhances the kinase activity of this isoform [190]. Importantly, β2-PIPKIγ661 interaction depends on the phosphorylation status of PIPKIγ661: in neurons, the dephosphorylation of PIPKIγ661 is triggered by depolarization, and only this form of the enzyme can interact with the β2 subunit.

[201]. These results suggest a feedback loop model that dynamically modulates the interaction of PIPKIγ661 with talin.

### 4.2.3.4. Chemotaxis

Lacalle et al. have found that PIPKIα, but not PIPKIβ or PIPKIγ635, asymmetrically localizes at the uropod of differentiated HL60 cells, and that siRNA knockdown of PIPKIα impairs cellular polarization and directional movement during chemotaxis [202]. Significantly, the localization of PIPKIα to the uropod requires the C-terminal tail of PIPKIα, which has no homology to any other PIPKI. Moreover, this region of the PIPKIα C-terminus interacts with a uropod protein EBP50 (4.1-ezrin-radixin-moesin (ERM)-binding phosphoprotein 50), which further recruits ERM and RhoGDI (guanine nucleotide dissociation inhibitor of Rho) to the PIPKIα-EBP50 complex [202]. PIPKIγ661 also localizes at the uropod of chemotaxing neutrophils [203]. Expression of a KD form of PIPKIγ in these neutrophils does not significantly affect the speed of leading edge movement in a gradient of the chemoattractant C5a, but does drastically impair uropod retraction due to decreased PtdIns(4,5)P₂ production at the uropod [203]. These data support a specific role for PtdIns(4,5)P₂ generated by PIPKIα or PIPKIγ661 in uropod functions.

Other work has implied that different stimuli may rely on the activation of distinct PIPKI isozymes to induce directional cell migration. For example, PIPKIγ661, but not PIPKIβ, contributes to EGF-induced migration [204]. In this context, PIPKIγ661 is directly phosphorylated on Tyr634 by EGFR, and this event is required for EGF-induced migration. Interestingly, knockdown of PIPKIγ661 does not affect cell migration in response to other chemoattractants such as LPA or stromal cell-derived factor-1α (SDF1α), which are ligands for G protein-coupled receptors.

### 4.2.3.5. Ins(1,4,5)P₃ formation

Koreh et al. have demonstrated that cells possess at least two different PtdIns(4,5)P₂ pools: the agonist-sensitive pool and the agonist-insensitive pool [205]. Several groups have attempted to determine which PIPKI isoform supplies PtdIns(4,5)P₂ for each of these pools. Wang et al. used siRNA knockdown to examine the roles of PIPKIα, β, γ635 and γ661 in Ins(1,4,5)P₃-mediated Ca²⁺ signaling in HeLa cells. They found that depletion of PIPKIγ635, but not that of any other PIPKI, decreases histamine-induced Ins(1,4,5)P₃ production and Ca²⁺ flux by 70% [206]. In contrast, platelets derived from PIPKIβ⁻/⁻ mice show impaired production of PtdIns(4,5)P₂ and Ins(1,4,5)P₃ following

thrombin stimulation [177]. Although loss of PIPKIα alone has no apparent effect on thrombin-induced Ins(1,4,5)P₃ production, platelets lacking both PIPKIα and PIPKIβ fail to produce Ins(1,4,5)P₃ in response to thrombin even in the presence of the PIPKIγ isoform, which is the most abundant PIPKI enzyme in mouse platelets [177]. Thus, although PIPKIγ may be important for the Ins(1,4,5)P₃ response in HeLa cells, PIPKIγ does not contribute to Ins(1,4,5)P₃ formation in thrombin-stimulated platelets. Additional analyses will be necessary to understand why the main contributors of the PtdIns(4,5)P₂ needed to support Ins(1,4,5)P₃ formation differ in different cell types.

### 4.2.3.6. Nuclear function
PIPKIβ localizes at nuclear speckles, suggesting a role for PIPKIβ in the nucleus [207]. However, the molecular mechanism by which PIPKIβ might regulate nuclear functions remains unclear. Mellman et al. have demonstrated that PIPKIβ associated with nuclear speckles interacts with a newly identified non-canonical poly(A) polymerase called Star-PAP [208]. Star-PAP and PIPKIβ cooperate to control the expression of select mRNAs, including transcripts encoding haemoxygenase-1 and other oxidative stress response genes.

### 4.2.4. Regulatory factors
It has been difficult to determine which PIPKI isoform is responsible for generating the pool of PtdIns(4,5)P₂ implicated in a given cellular process. To resolve this issue, much effort has been expended on searching for molecules that specifically interact with and activate each PIPKI isoform.

All PIPKI isoforms are activated *in vitro* by phosphatidic acid (PA) [161,163,209,210], although the physiological relevance of PA activation remains unclear. Two PA-producing enzymes, phospholipase D and diacylglycerol kinase, are thought to be involved in *in vivo* activation of PIPKIs [192,211,212].

Historically, endogenous PIPKI in the membrane fraction of a variety of tissues was activated *in vitro* by addition of GTPγS [213–215]. Subsequently, Chong et al. and Hartwig et al. demonstrated that the small GTPases RhoA and Rac1 could stimulate PIPKI activity in mouse fibroblast lysates and permeabilized platelets, respectively [216,217]. However, although RhoA and Rac1 can interact with all three PIPKI isozymes in a guanine nucleotide-independent manner, there is no concrete evidence thus far that these small GTPases can directly activate the PIPKIs. Indeed, Honda et al. showed that neither RhoA nor Rac1 could activate PIPKI in a reconstituted system involving purified recombinant proteins [192], suggesting that any activation of PIPKIs by these small GTPases is indirect. In the case of RhoA-mediated PIPKI stimulation, the RhoA effector Rho kinase (ROCK) appears to be responsible for PIPKI activation [218–220]. It remains unclear how Rac1 and ROCK regulate PIPKI activity.

Several groups have used either a reconstituted system or permeabilized HL-60 cells to identify ARF small GTPases as direct activators of PIPKIs [159,192,221]. In both types of studies, ARF proteins cooperate with PA in triggering PIPKI activity important in several biological contexts [192,222]. Of the six known mammalian ARF proteins (ARF1-6), only the functional relationship between ARF6 and PIPKI has been studied intensively to date. Although the other ARF proteins can apparently activate all PIPKI isoforms, the spatio-temporal factors governing which ARF protein activates which PIPKI, and when, remain to be determined.

Several other proteins have been identified as potential PIPKI binding partners. (1) Saito et al. have demonstrated that the Tec family tyrosine kinase Btk associates with PIPKIβ and targets this isoform to the plasma membrane to facilitate local PtdIns(4,5)P₂ production [223]. This activation results in a supply of PtdIns(4,5)P₂ at a place appropriate for phospholipase C and PI3K activation. (2) The Ajba protein binds directly to and activates PIP-

KIβ during lamellipodia formation [224]. (3) β-arrestin interacts with PIPKIβ in an isoproterenol stimulation-dependent manner [225].

#### 4.2.5. Disease association
Lethal congenital contractural syndrome (LCCS) is an autosomal recessive form of arthrogryposis [226]. So far, two forms of the disease, LCCS and LCCS2, have been identified and chromosomally mapped [227,228]. Narkis et al. have reported a third form of LCCS, LCCS3, that is caused by a mutation in PIPKIγ [229]. This mutation causes asparagine to be substituted for aspartic acid at residue 253 of the PIPKIγ protein, thereby abolishing its kinase activity.

### 4.3. Type II PIPKs

#### 4.3.1. Structure
PIPKIIs produce PtdIns(4,5)P₂ by phosphorylating PtdIns5P (refer to Fig. 2 and Table 1). In mammals, three PIPKII isoforms, α, β, γ have been identified [165–168]. All three are highly similar in amino acid sequence: the α isoform shares 77% and 61% sequence identity with the β and γ isoforms, respectively, and the β and γ isoforms are 63% identical [168]. All three PIPKII isoforms contain the conserved PIPKc kinase domain bisected in the center by a non-conserved inserted sequence (refer to Fig. 5). In the C-terminal part of the kinase domain, all three PIPKIIs feature the ~25 amino acid activation loop that also appears in PIPKIs.

#### 4.3.2. Expression
PIPKIIα and PIPKIIβ are ubiquitously expressed but differentially enriched in different tissues. The α isoform is abundant in brain, whereas the β isoform is highly expressed in heart, placenta, kidney and pancreas [165–167]. PIPKIIγ expression is much more restricted, with PIPKIIγ mRNA being highly expressed in the cortex and outer medulla of the kidney [168,230].

Studies of the subcellular localization of PIPKII isoforms have sometimes yielded conflicting results. Although Boronenkov et al. reported that both endogenous and epitope-tagged PIPKIIα are localized in the nucleus [207], two more recent studies have demonstrated that GFP-tagged PIPKIIα appears predominantly in the cytosol and only rarely in the nucleus [231,232]. In contrast, PIPKIIβ is localized mainly in the nucleus, with only a small fraction present in the cytosol [207,231,233]. Although PIPKIIβ has no obvious NLS, the acidic α-helix present in its kinase insert region can mediate the nuclear targeting of this isoform [231,234].

PIPKIIγ was originally detected in the ER by immunochemistry and subcellular fractionation [168]. However, Clarke et al. have recently reported that, in kidney cell lines overexpressing GFP-tagged PIPKIIγ, this isoform appears in the cis-Golgi and unidentified vesicular compartments [230].

#### 4.3.3. Biological functions
Although both PIPKIs and PIPKIIs synthesize PtdIns(4,5)P₂, these two enzyme classes have distinct physiological roles. Indeed, it has been proposed that, while the main function of the PIPKIs is to produce PtdIns(4,5)P₂, the main function of PIPKIIs may be to eliminate PtdIns5P by converting it to PtdIns(4,5)P₂. Unlike PIPKIs, PIPKIIs do not regulate actin dynamics in mammalian cells and fail to rescue the lethality of the mss4 (yeast PIPKI) mutation in S. cerevisiae [163,235]. In fact, PIPKIIs do not even exist in yeast but are found only in metazoa, suggesting that PIPKIIs might play roles specific to the needs of multicellular organisms. Although the detailed molecular mechanisms by which PIPKIIs regulate cell functions are still poorly understood, evidence is emerging that PIPKIIs are involved in the control of processes as diverse as proliferation, secretion, ion channel function, stress responses and insulin sensitivity [236,237]. PIPKIIα-deficient mice appear to have no

obvious phenotype [232]. No knockout mice have been reported to date for PIPKIIγ, but PIPKIIβ mice are hypersensitive to insulin in vivo (see below).

### 4.3.3.1. Control of PtdIns5P levels.
Cellular PtdIns5P levels increase in response to various stimuli [238–241]. Wilcox et al. have demonstrated that enhanced tyrosine phosphorylation induced by pervanadate treatment increases PtdIns5P levels [242]. RNAi knockdown of PIPKIIα, but not of PIPKIIβ or PIPKIIγ, significantly enhances pervanadate-stimulated PtdIns5P production, implying that PIPKIIα is responsible for regulating intracellular PtdIns5P levels in a subcellular compartment that is sensitive to pervanadate treatment [242].

As well as at the plasma membrane, PtdIns5P can be detected in the nucleus at levels that vary throughout the cell cycle, and increase in response to UV irradiation or oxidative stress [241,243]. Bunce et al. have found that PIPKIIβ interacts with speckle-type POZ domain protein (SPOP), a nuclear speckle-associated protein that recruits substrates to Cul3-based ubiquitin ligases [234]. SPOP then mediates the ubiquitination of PIPKIIβ by these ligases. Nuclear PtdIns5P may also stimulate Cul3-SPOP ubiquitin ligase activity.

### 4.3.3.2. Regulation of Akt activation.
PIPKIIβ⁻/⁻ mice are viable but hypersensitive to insulin, resulting in reduced body weight [244]. The leptin levels and food intake of these mutants are normal but they remain lean even on a high-fat diet. It is unclear why PIPKIIβ deficiency confers hypersensitivity to insulin, but it appears that Akt activation induced by insulin is enhanced in the skeletal muscles of PIPKIIβ⁻/⁻ mice [244]. Several other studies have supported a model in which PIPKIIβ controls the activation of the PI3K/Akt signaling pathway by regulating PtdIns5P levels within a cell. Carricaburu et al. have shown that overexpression of PIPKIIβ is associated with reduced levels of PtdIns(3,4,5)P₃ in cells stimulated with insulin, or in cells expressing activated PI3K [245]. Conversely, elevation of PtdIns5P due to overexpression of lpgD, a Shigella flexneri PtdIns(4,5)P₂ 4-phosphatase, induces Akt activation [245]. These results suggest that PtdIns5P can positively regulate the PI3K-mediated pathway that leads to Akt activation. Another report supports this molecular link between PtdIns5P and PI3K/Akt, in that PtdIns5P is rapidly produced and co-localizes with Akt at sites of Shigella flexneri attack [246].

### 4.3.4. Regulation

The localization of PIPKIIα appears to be regulated by PIPKIs. Hinchliffe et al. have demonstrated that PIPKIIα co-immunoprecipitates with any one of the three PIPKI isozymes when these proteins are co-expressed in mammalian cells, and that these interactions result in translocation of PIPKIIα from the cytosol to the plasma membrane [247]. In terms of regulating PIPKIIα activity, this isozyme’s function is inhibited when the Thr376 residue present in its activation loop is phosphorylated by PKD/PKCμ [248]. However, Huang et al. have reported that tyrosine phosphorylation of PIPKIIα increases its enzymatic activity in bovine photoreceptor rod outer segments [249].

PIPKIIβ activity is regulated by p38 MAP kinase. In response to UV irradiation, p38 MAP kinase directly phosphorylates PIPKIIβ on Ser326, leading to inactivation of this isozyme and an increase in nuclear PtdIns5P levels [241]. This PIPKIIβ-mediated change in nuclear PtdIns5P alters the localization of “inhibitor of growth protein-2” (ING2), a nuclear adaptor that interacts directly with PtdIns5P via ING2’s plant homeodomain (PHD). ING2 that has bound PtdIns5P regulates p53 and histone acetylation in response to cellular stresses [250].

Itoh et al. have found that the PIPKIIγ protein acquires activity when it is overexpressed in and immunoprecipitated from mammalian cells [168]. However, it has been difficult thus far to assess the regulation of PIPKIIγ activity because this isozyme has no activity when expressed in bacterial cells [232]. These observations suggest that PIPKIIγ activity requires some post-translational modification or heterodimerization with other PIPKIs to become fully active.

### 4.3.5. Disease association

In humans, PIPKIIα is considered to be a candidate gene for the development of schizophrenia and bipolar affective disorders, and several single nucleotide polymorphisms (SNPs) of PIPKIIα are found at increased frequency in patients with these diseases [251,252]. In a Xenopus system, expression of a PIPKIIα protein bearing the schizophrenia-associated mutation N251S results in impaired kinase activity and reduced KCNQ potassium channel function [253]. KCNQ channels are required to suppress the basal activity of dopaminergic neurons and dopaminergic firing, and schizophrenic patients lose dopaminergic control [2053]. It is therefore possible that dysregulation of KCNQ channels associated with the PIPKIIα (N251S) mutation may contribute to the schizophrenia phenotype.

### 4.4. Type III PIPK

#### 4.4.1. Structure

Type III phosphatidylinositol phosphate kinase (PIPKIII) is the large protein product (2052 aa) of a single-copy gene that is evolutionarily conserved from yeast to mammals. In mammals, PIPKIII was originally known as PIKfyve (phosphoinositide kinase for five position containing a fyve finger) and contains four domains of interest: FYVE, DEP, TCP-1 and PIPKc (refer to Fig. 5). The N-terminal FYVE domain (Fab1p, YOTB, Vac1p and EEA1) is 100% conserved between mice and humans and is responsible for membrane localization. The DEP domain (dishevelled, Egl-10 and pleckstrin) has not been functionally characterized. Although the mouse and human DEP domains are 99% conserved, this domain does not appear in Fab1p and PPK-3, the PIPKIII orthologs in yeast and nematode, respectively. The TCP-1 (t-complex polypeptide-1) domain present in the middle of the PIPKIII protein is conserved from yeast to mammals and is 99% identical in mouse and human. In yeast, the TCP-1 domain is involved in the binding of the Fab1p protein to several regulatory proteins [254]. The C-terminus of the mammalian PIPKIII protein contains the catalytic PIPKc domain, which is similar in sequence to the lipid kinase domains of the mammalian PIPKI and PIPKII enzymes. In mouse PIPKIII, there is a predicted ATP-binding Lys residue at position 1831 within the PIPKc domain, and a PIPKIII protein bearing a K1831E mutation has no kinase activity [255]. Indeed, when overexpressed, the PIPKIII (K1831E) mutant exhibits a dominant-negative (DN) effect on endogenous PIPKIII activity [256].

In vitro studies with immunopurified mouse PIPKIII have demonstrated that this enzyme phosphorylates the D5 position of the inositol ring in PtdIns and PtdIns3P to generate PtdIns5P and PtdIns(3,5)P₂, respectively [169]. PIPKIII can also synthesize these lipid products in intact cells [257]. In mammalian cells, overexpression of the DN PIPKIII (K1831E) mutant, or RNAi knockdown of PIPKIII, results in decreased intracellular PtdIns(3,5)P₂ content. A stable transformant of HEK293 cells overexpressing WT PIPKIII exhibits increased levels of PtdIns5P, whereas a transformant expressing the PIPKIII (K1831E) mutant shows decreased levels of PtdIns5P.

#### 4.4.2. Expression

PIPKIII is highly expressed in muscle and fat tissues [258]. The subcellular localization of endogenous PIPKIII remains uncertain, but it is suspected that the FYVE domain facilitates PtdIns3P-dependent localization of PIPKIII to early endosomes [259,260].
was shown early on in mice that PIPKIII proteins with a truncated FYVE domain cannot associate with membranes [258].

### 4.4.3. Biological functions

#### 4.4.3.1. Cellular membrane dynamics
Loss-of-function of PIPKIII in mammalian cells causes a significant perturbation of cellular membrane dynamics that leads to vacuolation and enlarged early and late endosomes. For example, overexpression of the DN PIPKIII (K1831E) mutant in COS, HEK293 or CHO cells results in the generation of numerous cytoplasmic vacuoles [256]. Similarly, RNAi-mediated depletion of endogenous PIPKIII in HeLa cells causes severe vacuolation [259]. Ikonov et al. have made two useful mouse PIPKIII mutants, one expressing a mutated PIPKIII (K1999E) protein, and the other expressing a mutated PIPKIII (K2000E) protein [261]. The PIPKIII (K1999E)-expressing mouse is selectively impaired in its synthesis of PtdIns(3,5)P₂, whereas the PIPKIII (K2000E)-expressing mutant cannot produce PtdIns5P. Only the cells of the PIPKIII (K1999E) mutant possess vacuoles identical to those observed in cells transfected with the PIPKIII (K1831E) mutant. Furthermore, microinjection of PtdIns(3,5)P₂, but not PtdIns5P, into PIPKIII (K1831E)-expressing cells can correct the cell vacuolation phenotype, suggesting that vacuolation caused by loss of PIPKIII function is due to the resulting decrease in PtdIns(3,5)P₂ production. Marker studies have revealed that the vacuoles induced by loss of PIPKIII function are derived from the membranes of early and late endosomes.

Despite their drastic morphological changes, PtdIns(3,5)P₂-depleted cells are largely normal in most cellular functions, including receptor internalization, cell surface recycling, lysosomal degradation, and protein biosynthesis [262]. However, overexpression of the DN PIPKIII (K1831E) mutant inhibits fluid phase endocytosis in HEK293 cells [260], and RNAi knockdown of PIPKIII in HeLa cells delays retrograde traffic from the early endosomes to the TGN [259].

#### 4.4.3.2. Insulin action
Shisheva et al. have shown that mouse PIPKIII transcripts are enriched in insulin-sensitive tissues and in fat and skeletal muscle, and increase during the differentiation of fibroblasts into adipocytes [258]. Two additional studies have suggested a positive role for PIPKIII in insulin signaling. Both overexpression of the DN PIPKIII (K1831E) mutant and RNAi-mediated PIPKIII depletion can block insulin-induced glucose uptake and GLUT4 translocation to the cell surface in 3T3-L1 adipocytes [263,264]. However, another study has suggested that PIPKIII activation can inhibit insulin action. In insulin-stimulated cells, PIPKIII becomes phosphorylated at Ser318 by Akt, and this phosphorylation activates PIPKIII's intrinsic PtdIns3P 5-kinase activity. Overexpression of a phosphorylation-resistant mutant called PIPKIII (S318A) has been found to enhance insulin-stimulated GLUT4 translocation [265], suggesting that PIPKIII negatively regulates GLUT4 trafficking.

One means of investigating the above discrepancy may be to use selective PIPKIII inhibitors. PIPKIII kinase activity can be blocked by curcumin, one of the polyphenols in the curry spice turmeric [263]. However, as curcumin also potently inhibits PKC, EGFR tyrosine kinase, and IkB kinase, this inhibition is not terribly specific. It has recently been reported that a pyridofuropyrimidine compound called YM201636 is a highly potent and selective inhibitor of PIPKIII. In NIH-3T3 cells, YM201636 treatment causes an 80% decrease in PtdIns(3,5)P₂ production [266]. This compound should be a valuable tool for carrying out a reversible block of PtdIns(3,5)P₂ synthesis and thus clarifying the selective functions of PIPKIII.

If insulin activates PIPKIII via phosphorylation or some other modification, the levels of PtdIns(3,5)P₂ and PtdIns5P should increase in insulin-treated cells. Insulin stimulation of some cell lines, including 3T3-L1 adipocytes, results in elevated intracellular PtdIns5P [239]. However, it has not been directly demonstrated that insulin increases PtdIns(3,5)P₂ levels in intact cells. Animal models such as tissue-specific PIPKIII knockout mice are awaited for further elucidation of the roles of PIPKIII in insulin signaling in vivo.

### 4.4.4. Regulation
Biochemical and genetic studies have identified the Vac14p and Fig4p proteins as activators of Fab1p in yeast [254]. The TCP-1 domain of Fab1p is particularly important for the interaction of this PIPKIII enzyme with regulatory proteins and for PtdIns(3,5)P₂ synthesis [254,267]. It has been proposed that mouse PIPKIII also forms a ternary complex with the mammalian equivalents of the yeast regulatory proteins, namely Vac14 (ArPIKfyve) and Sac3 (Fig. 4) [268], and that this complex is important for PIPKIII activity in mammalian cells. For example, fibroblasts from “pale tremor” mice (which lack Sac3 activity; see Section 7.10.3), and those from Vac14-deficient mice show only low levels of intracellular PtdIns(3,5)P₂ [269,270]. In addition, in both pale tremor mice and Vac14-deficient mice, neurons undergo a form of cell death characterized by the formation of numerous intracellular vacuoles. This intriguing phenotype is reminiscent of that of yeast lacking Fab1p [271].

### 4.4.5. Disease association
Francois-Neetens Fleck Corneal Dystrophy (CFD, MIM 121850) is an autosomal dominant corneal dystrophy characterized by many small white flecks scattered in the corneal stroma. Linkage and sequencing analyses have revealed mutations in the PIPKIII gene in 8 of 10 families with CFD [272]. These missense, frameshift, and/or protein-truncating mutations cause loss-of-function of the enzyme. The histological findings associated with CFD are consistent with the role of PIPKIII in membrane trafficking, since the corneal flecks appear to be abnormally swollen keratinocytes containing numerous cytoplasmic vacuoles. However, given that PIPKIII is widely expressed in human tissues [273], it is not understood why these morphological abnormalities are observed in only certain cell types in CFD patients.

## 5. Phosphoinositide 3-phosphatases

### 5.1. Classification
The phosphoinositide 3-phosphatases include PTEN, TPIP (TPTE and PTEN homologous inositol lipid phosphatase), and multiple members of the myotubularin family (see Table 2). All phosphoinositide 3-phosphatases contain a phosphatase domain containing a catalytic motif called CX₅R in their active sites (Fig. 6). Although PTEN does have protein phosphatase activity, it primarily dephosphorylates PtdIns(3,4,5)P₃ specifically at the D3 position on the inositol ring. TPIP also mainly dephosphorylates PtdIns(3,4,5)P₃, whereas various myotubularin phosphatases favor PtdIns3P as their substrate. The dephosphorylation of PtdIns3P can also be carried out by Sac1, a CX₅R-containing Sac phosphatase that will be described in Section 7.10.

### 5.2. PTEN

#### 5.2.1. Structure
Analysis of the crystal structure of the tumor suppressor PTEN has revealed that the catalytic cleft of this (primarily) lipid phosphatase is wider than the clefts of most CX₅R-containing protein tyrosine phosphatases, allowing key catalytic residues access to the bulky PtdIns(3,4,5)P₃ headgroup [274]. The catalytic core
T. Sasaki et al./Progress in Lipid Research 48 (2009) 307–343

| PTEN | Ptase C2 // PDZ-binding | PEST | CX5R motif |
| --- | --- | --- | --- |
|  |  |  | 403aa AIH CKAGKG R TG |

| TPIPα |  |  | 445aa AIH CKGGKG R TG |
| --- | --- | --- | --- |

| TPIPβ | TM |  | 326aa AIH CKGGKG R TG |
| --- | --- | --- | --- |

| TIPγ |  |  | 522aa AIH CKGGKG R TG |
| --- | --- | --- | --- |

| MTM1 | PHG Ptase SID coiled-coil domain |  | 603aa LVH C SDGWDR TA |
| --- | --- | --- | --- |

| MTMR1 |  |  | 665aa VVH C SDGWDR TA |
| --- | --- | --- | --- |

| MTMR2 |  |  | 643aa VVH C SDGWDR TA |
| --- | --- | --- | --- |

| MTMR3 | FYVE |  | 1198aa LVH C SDGWDR TP |
| --- | --- | --- | --- |

| MTMR4 |  |  | 1195aa LVH C SDGWDR TP |
| --- | --- | --- | --- |

| MTMR6 |  |  | 621aa VVH C SDGWDR TS |
| --- | --- | --- | --- |

| MTMR7 |  |  | 660aa LVH C SDGWDR TA |
| --- | --- | --- | --- |

| MTMR8 |  |  | 704aa LVH C SDGWDR TA |
| --- | --- | --- | --- |

| MTMR14 | CX5R motif |  | 650aa VVH CISGWD R TP |
| --- | --- | --- | --- |

Fig. 6. Structural features of mammalian phosphoinositide 3-phosphatases. All members of this family (except MTMR14) contain a conserved phosphatase (Ptase) domain with a CX5R motif. The Ptase domains of the myotubularins (MTMs) encompass two regions previously proposed to be distinct domains: the SET-interaction domain (SID) and the Rac-induced recruitment domain (not shown). TPIPγ is structurally related to PTEN but contains four transmembrane (TM)-spanning regions as predicted by the TMHMM 2.0 webware of the Center for Biological Sequence Analysis (http://www.cbs.dtu.dk/index.shtml). Both PTEN and TPIPγ have a C2 domain but no MTMs do. PTEN and the first MTM subgroup (bracket) containing MTM1, MTMR1 and MTMR2 contain a PDZ-binding domain. All MTMs (except MTMR14) contain a PH-GRAM (PHG) domain and a coiled-coil domain as predicted by the NPSA webware of the Pôle BioInformatique Lyonnais (http://pbil.ibcp.fr/htm/index.php). MTMs of the second subgroup also have a FYVE domain. Numbers of amino acids in the human proteins are indicated.

sequence CKAGKGR in the PTEN phosphatase domain is located at amino acids 124–130. Most PTEN mutations found in cancer patients [e.g., PTEN (C124S)] inactivate phosphatase activity toward both lipid and protein substrates [275]. However, some PTEN mutations [e.g., PTEN (G129E)] abrogate PTEN phosphatase activity toward PtdIns(3,4,5)P₃ but not toward protein substrates [276]. No PTEN mutation that selectively affects protein phosphatase activity has been found in human cancers, providing strong evidence that it is the loss of PTEN's lipid phosphatase activity that drives tumorigenesis.

As well as its catalytic core, PTEN contains a lipid-binding C2 domain that is essential for PTEN stability and its recruitment to the plasma membrane and centrosomes. PTEN also contains two PEST (proline, glutamine, serine, threonine) sequences that enhance proteolytic sensitivity, and a PDZ domain that is thought to be important for maintaining PTEN stability and for protein-protein interactions (refer to Fig. 6). The PEST sequences and the PDZ domain are located at the C-terminal end of the PTEN protein.

### 5.2.2. Expression

Immunohistochemical studies have shown that PTEN protein is localized in both the cytosol and the nucleus [277] [278]. The N-terminal 16 amino acid polybasic region and the C2 domain contribute to the translocation of PTEN from the cytosol to the plasma membrane. The C-terminal tail of PTEN may negatively regulate PTEN membrane binding, because this tail can interact intramolecularly with the C2 domain [279]. With respect to the nucleus, although PTEN lacks a typical NLS, mono-ubiquitination of PTEN residues K13 and K287 can enhance the import of the protein into the nucleus [280]. PTEN (K287E) and PTEN (K13E) mutants are defective in mono-ubiquitination and nuclear import and are associated with cancer development, suggesting that nuclear localization of PTEN may be important for its tumor-suppressive function.

#### 5.2.3. Biological functions

PTEN’s most important function is to negatively regulate the activation of the key cell survival kinase Akt in the cytoplasm. By dephosphorylating the PtdIns(3,4,5)P₃ produced by PI3Ks, PTEN dampens down PDK1 activation and consequently Akt activation. The growth-promoting effects of uncontrolled Akt activation are thus reined in. PTEN can bind to each of p53 [281], the androgen receptor (AR) [282], E2F-1, CENP-C (an integral component of the kinetochore) [283], and MSP58 (important for cell transformation) [284], and regulate the nuclear translocation, degradation and/or functions of these proteins. PTEN also dephosphorylates PtdIns(3,4,5)P₃ within the nucleus but the effects of this dephosphorylation are not yet clear [285].

##### 5.2.3.1. Tumor suppression

PTEN is also known as MMAC1 (mutated in multiple advanced cancers-1) because it is the second most frequently mutated, deleted or silenced tumor suppressor gene in human cancers. Germline mutations of *PTEN* result in hereditary disorders such as Cowden disease, Bannayan-Riley-Ruvalcaba syndrome, Lhermitte-Ducros disease (LDD), Proteus syndrome, and Proteus-like syndrome. These syndromes are all characterized by the development of multiple hamartomas and an increased risk of tumorigenesis, such that these diseases have been collectively classified as types of “PTEN hamartoma tumor syndrome” (PHTS) [286]. Two-thirds of the germline *PTEN* mutations associated with Cowden disease are localized in exons 5, 7
and 8. More than 40% of Cowden disease mutations occur in exon 5, which contains the phosphatase CX₅R motif [286]. Hereditary mutations and deletions have also been reported in the promoter region of the *PTEN* gene [287].

Sporadic *PTEN* mutations are found in malignancies as varied as glioblastomas and endometrial and prostatic cancers [288]. However, in many sporadic tumors, PTEN protein expression is frequently impaired in the absence of any detectable alteration to the *PTEN* gene. Indeed, PTEN protein expression is reduced or lost in half of sporadic cancers, making *PTEN* as important a tumor suppressor gene as p53.

Loss of PTEN protein function leads to enhancements of cell proliferation, survival, migration, stem cell renewal/maintenance, cell size, centromeric instability, and DNA double strand breaks [289–292,283,293]. These defects render affected cells susceptible to carcinogens, increasing the occurrence of secondary genetic or epigenetic alteration(s) that culminate in the initiation and progression of cancers. Pten haplo-insufficiency in both mice and humans clearly promotes the development of various types of tumors, despite the retention of a WT *PTEN* allele [294,295]. Thus, cells that experience even a partial loss of PTEN may gain a growth and/or survival advantage. In mouse models, heterozygous deletion of *Pten* specifically in endothelial cells [296] or in NKT cells [297] results in the increased growth of human cancer xenografts. These observations suggest that, with respect to the human situation, Cowden disease patients (who have inherited a mutated *PTEN* allele) are not only at risk for the onset of cancers, but may also experience accelerated growth of any incipient tumors due to enhanced angiogenesis and impaired NKT-mediated anti-tumor surveillance.

### 5.2.3.2. Additional functions

Homozygosity for a null mutation of *Pten* in mice results in early embryonic lethality [295,298,299]. Thus, mouse models based on a conditional gene-targeting approach have been developed to examine the physiological functions of PTEN in specific cell types. Collectively, analyses of these conditional PTEN mutants have revealed that PTEN deficiency can cause: autoimmune disease [300,301], heart failure [49], non-alcoholic steatohepatitis (NASH) [302], insulin hypersensitivity [302,303], macroencephaly [304], defects in immunoglobulin class switch recombination [305], increased bone mineral density [306], respiratory distress syndrome [289], hyperkeratosis and shaggy hair [307], and resistance to hair graying [308]. All these studies point to the importance of PTEN for the maintenance of a variety of physiological functions in various tissues. For more detailed descriptions of conditional PTEN mutant mice and their phenotypes, the reader is referred to two excellent recent reviews [309,310].

### 5.2.4. Regulation

PTEN can be regulated both transcriptionally and post-translationally. At the transcriptional level, the *PTEN* gene generates naturally-occurring splice variants, and reduced expression of full-length PTEN transcripts is associated with differential expression of these splice variants in Cowden syndrome and Bannayan-Riley-Ruvalcaba syndrome [311]. It is speculated that dysregulated PTEN splicing may be the cause of these syndromes, especially in cases where no mutation of the *PTEN* gene itself can be detected. In human cancers, levels of PTEN transcripts are often dramatically decreased due to silencing of the *PTEN* promoter by DNA methylation [291] [312].

At the post-translational level, modifications such as phosphorylation by casein kinase 2 can affect PTEN protein expression levels, subcellular localization, and intrinsic enzymatic activity [313,314]. Other PTEN modifications with similar effects include phosphorylation by GSK3β [315] or Srk family kinases [316]; acetylation by PCAF [317]; oxidation by reactive oxygen species (ROS)

[318]; and ubiquitination by NEDD4-1 [319] or herpesvirus-associated ubiquitin-specific protease (HAUSP/USP7) [320].

## 5.3. TPIP

### 5.3.1. Structure

Human TPIP exists in three splice variants: TPIPα, β and γ [321,322]. All three isoforms share a phosphatase domain that exhibits an extremely high degree of homology with the human PTEN phosphatase domain (refer to Fig. 6). All three isoforms also share a C2 domain that is moderately homologous to the human PTEN C2 domain. From computational predictions ([http://www.cbs.dtu.dk/services/TMHMM/](http://www.cbs.dtu.dk/services/TMHMM/)), TPIPα and TPIPγ appear to have unique N-termini that contain three or four transmembrane (TM) segments, respectively, whereas TPIPβ has no TM segment. The phosphatase domain of TPIPγ probably lies in the cytoplasm, whereas that of TPIPα extends into the extracellular environment or lumen of endosomes. The TM segments in TPIPγ display sequence homology to the voltage-sensor domains of the conventional voltage-gated channel proteins and to Ci-VSP (*C. intestinalis* voltage-sensor-containing phosphatase) [323,324].

The mouse ortholog of TPIP is called mTPTE. Three splice variants of mTPTE have been identified, all of which contain the phosphatase and C2 domains and four TM segments [322]. A distinct human gene product called TPTE (hTPTE) has been reported that is also highly homologous to PTEN but is not a phosphoinositide phosphatase [325]. In the hTPTE sequence, the lysine and glycine residues in the CX₅R motif conserved among PTEN, TPIP and mTPTE are replaced with a threonine and an asparaginic acid residue, respectively [326,327].

### 5.3.2. Expression

TPIP is expressed in the testes, brain and stomach [321]. The subcellular localization of endogenous TPIP has yet to be characterized. TPIPα that is ectopically expressed in COS-7 cells localizes in the ER, whereas TPIPβ that is ectopically expressed in HEK293 cells is diffusely distributed in the cytoplasm [321].

### 5.3.3. Biological functions

Thus far, little is known about the biological functions of TPIP. Like PTEN, TPIP dephosphorylates PtdIns(3,4,5)P₃ specifically at the D3 position. Walker et al. reported that overexpression of either TPIPα or TPIPβ in PTEN-deficient U87MG cells had no effect on Akt phosphorylation [321]. However, Murata et al. have found that when the Ci-VSP protein is stimulated by depolarization, this enzyme dephosphorylates PtdIns(3,4,5)P₃ and PI(4,5)P₂, thereby regulating the activity of neighboring phosphoinositide-sensitive ion channels [323,328]. These observations have prompted speculation that at least some TPIP proteins may regulate cellular responses triggered by changes in membrane potential. If true, TPIP would then exert functions that are fundamentally distinct from those of PTEN despite the structural similarities of these enzymes.

## 5.4. Myotubularins

### 5.4.1. Classes and general structure

The myotubularin (MTM) family consists of 15 members containing a characteristic phosphatase (Ptase) domain (refer to Fig. 6). Nine of these family members possess a conserved active site motif (CXXGWDR) and hydrolyze the D3 position of PtdIns3P and/or PtdIns(3,5)P₂. The remaining six MTM proteins are catalytically inactive ‘pseudophosphatases’ that may regulate the activities of other MTM phosphatases. The catalytically active MTM proteins are divided into four subgroups based on structure. The MTM1 subgroup consists of MTM1, MTMR1 and MTMR2; the MTMR3 subgroup consists of MTMR3 and MTMR4; the MTMR6
subgroup consists of MTMR6, MTMR7 and MTMR8; and MTMR14 is the sole member of the fourth subgroup. The proteins belonging to the MTM1, MTMR3 and MTMR6 subgroups have a PH-GRAM (pleckstrin homology-glucosyltransferases, Rab-like GTPase activators and myotubularins) domain at the N-terminus [329] and a coiled-coil domain close to the C-terminus. In addition, MTM1 subgroup members possess a C-terminal PDZ-binding domain, whereas MTMR3 subgroup members have a C-terminal FYVE domain. MTMR14 contains no significant domains other than the Ptase domain. The Ptase domain of certain MTM molecules contains a SID sequence [SET (Suvar3-9, Enhancer-of-zeste, Trithorax)-interacting domain] whose function is to mediate protein-protein interactions [330].

The six pseudophosphatases are known as MTMR5, MTMR9, MTMR10, MTMR11, MTMR12 and MTMR13. All contain a central Ptase-like domain that is catalytically inactive as well as an N-terminal PH-GRAM domain and a C-terminal coiled-coil domain. In addition, the MTMR5 and MTMR13 pseudophosphatases have a C-terminal PH domain and show a unique extension of the N-terminal region that contains a DENN (differentially expressed in normal versus neoplastic) domain. These domains are not present in any active MTMs.

### 5.4.2. Myotubularin/MTM1

#### 5.4.2.1. Structure and substrates.
MTM1 was originally identified by positional cloning as the gene responsible for X-linked myotubular myopathy (XLMTM) [331]. *In vitro*, recombinant GST-tagged MTM1 specifically dephosphorylates PtdIns3P. In HEK293 cells, overexpression of a catalytically inactive MTM1 mutant (C375S) known to be associated with severe myotubular myopathy dramatically decreases MTM1 lipid phosphatase activity and elevates PtdIns3P levels twofold compared with expression of WT MTM1 [332]. This study was the first to confirm that MTM1 is the PtdIns3P 3-phosphatase mutated in myotubular myopathy. However, MTM1 activity is not restricted to PtdIns3P as a substrate, as MTM1 can also dephosphorylate the D3 position of PtdIns(3,5)P₂ [240]. Accordingly, overexpression of MTM1 in intact Jurkat leukemic T cells results in elevated PtdIns5P production [240].

#### 5.4.2.2. MTM1-deficient mice.
MTM1⁻/⁻ mice die either shortly before or after birth [333]. Mutants that are born rapidly develop progressive muscle weakness characterized by centronuclear myopathy and the accumulation of central nuclei in muscle fibers. Histopathological analyses of these mutants have revealed that MTM1 plays a role in muscle maintenance rather than in myogenesis. Analyses of conditional MTM1-deficient mice have shown that a lack of MTM1 specifically in skeletal muscle is sufficient and necessary to induce the mouse equivalent of XLMTM [333].

#### 5.4.2.3. Interaction with MTMR12.
MTMR12 was originally identified as a 3-phosphatase adaptor subunit called 3-PAP that shares sequence similarity with MTM1 [334]. It was subsequently shown that MTM1 physically interacts with 3-PAP/MTMR12, and that the SID of MTMR12 is essential for this interaction [335]. Although endogenous MTM1 localization remains to be elucidated, exogenously expressed MTM1 localizes to the plasma membrane in COS-7 cells and induces extensive filopodial projections. However, co-expression of MTMR12 with MTM1 completely prevents MTM1 localization to plasma membrane as well as the MTM1-induced membrane remodeling phenotype. Thus, MTM1 localization and function are modulated by interaction with the adaptor MTMR12.

### 5.4.3. MTMR1
The MTMR1 gene arose following an ancient duplication of the MTM1 gene and sits adjacent to it on the X chromosome in both humans and mice [336]. The substrate specificity of MTMR1 is comparable to that of MTM1 [240]. Analyses of MTMR1 transcripts have revealed the presence of four alternatively spliced mRNA transcripts whose production is differentially regulated during the development of mouse and human muscle cells. However, the proteins derived from these variant transcripts have very similar substrate specificities and subcellular localizations. The role of MTMR1 in myogenesis remains to be elucidated.

### 5.4.4. MTMR2

#### 5.4.4.1. Structure and substrates.
MTMR2 is one of the best characterized MTM molecules. Studies of human MTMR2 co-crystallized with a water-soluble analog of PtdIns3P were the first to reveal that the Ptase domain of MTMR2 is larger than the catalytic domains present in other types of phosphoinositide 3-phosphatases due to the presence of the SID, which is highly conserved among MTMR family members (refer to Fig. 6) [337]. Examination of the crystal structure of MTMR2 has also demonstrated that the structure of the N-terminal PH-GRAM domain is highly similar to that of a conventional PH domain. Consistent with its PH domain-like structure, the MTMR2 PH-GRAM domain binds to a range of phosphoinositides, with a preference for PtdIns(3,5)P₂ [338]. Like other active MTMR family members, MTMR2 removes the phosphate from the D3 position of PtdIns3P and PtdIns(3,5)P₂ [339]. Crystallographic and deuterium-exchange mass spectrometric analyses of MTMR2 protein complexed to PtdIns3P or PtdIns(3,5)P₂ have revealed the precise mechanism by which MTMR2 recognizes its substrates [340].

#### 5.4.4.2. Charcot-Marie-Tooth disease type 4B1 (CMT4B1).
Charcot-Marie-Tooth disease type 4B1 is a severe demyelinating peripheral neuropathy caused by recessive mutations in the MTMR2 gene [341]. Most MTMR2 mutations found in CMT4B1 patients lead to dramatically reduced dephosphorylation of PtdIns3P and PtdIns(3,5)P₂ [342]. Dysregulation of these phosphoinositides is thought to disrupt membrane trafficking in the Schwann cells of CMT4B1 patients.

#### 5.4.4.3. MTMR2-deficient mice.
Two groups have reported that MTMR2-deficient mice develop a phenotype very similar to that of CMT4B1 patients [343,344]. These mouse mutants display a peripheral neuropathy with myelin outfoldings, as well as impaired spermatogenesis. Selective deletion of MTMR2 in the Schwann cells (but not motor neurons) of conditional mutants reproduces the myelin alterations seen in MTMR2 null mutant mice [345].

#### 5.4.4.4. Regulation by MTMR5 and MTMR13.
Two of the inactive MTMR pseudophosphatases, MTMR5 (also known as SBF1; Set binding factor 1) and MTMR13 (SBF2), bind selectively to MTMR2 and are thought to regulate its subcellular localization and enzymatic activity. The coiled-coil domains of both MTMR2 and MTMR5 are necessary for this interaction [346], and MTMR5⁻/⁻ mice exhibit defective spermatogenesis in the absence of peripheral neuropathy [347]. *In vitro*, the 3-phosphatase activity of MTMR2 can also be stimulated by MTMR13 [342]. In humans, mutations of MTMR13 result in Charcot-Marie-Tooth disease type 4B2 (CMT4B2), which is associated with defects very similar to those characteristic of the CMT4B1 disease caused by MTMR2 mutations [348,349]. Consistent with these observations, targeted disruption of mouse MTMR13 induces myelin outfoldings and infoldings in peripheral nerves that recapitulate the key features of CMT4B2 [350,351]. It has been speculated that an MTMR2-MTMR5 complex may be essential for spermatogenesis, whereas an MTMR2-MTMR13 complex may be a key regulator of myelin homeostasis in Schwann cells [352].

T. Sasaki et al./Progress in Lipid Research 48 (2009) 307–343

5.4.5. MTMR3 and MTMR4

The active phosphatases MTMR3 and MTMR4 are highly similar in their intron/exon organization and domain structures and are the only MTM family members to possess an extended C-terminus containing a FYVE domain [353,354]. RT-PCR analysis has shown that both proteins are widely distributed in human tissues. With respect to function, only MTMR3 has been examined in any detail. Like other MTMR phosphatases, MTMR3 hydrolyzes PtdIns3P and PtdIns(3,5)P₂ [355]. Yeast cells expressing human MTMR3 exhibit aberrantly large vacuoles, reminiscent of those present in Fab1-deficient mutants. In addition, this heterologous expression of MTMR3, but not expression of an inactive form of MTMR3 bearing a C413S mutation, results in intracellular production of PtdIns5P, suggesting that MTMR3 is capable of dephosphorylating PtdIns(3,5)P₂ at least in yeast.

In terms of domain functions, the FYVE domain of MTMR3 neither provides endosomal localization nor binds to PtdIns3P [356]. In contrast, the PH-GRAM domain of MTMR3 binds primarily to PtdIns5P, and deletion of this domain reduces PtdIns3P and PtdIns(3,5)P₂ in vitro. This interaction between the PH-GRAM domain and PtdIns5P may be essential for the allosteric activation of MTMR3 by PtdIns5P [339].

5.4.6. MTMR6, MTMR7 and MTMR8

Like MTM1 and MTMR2, MTMR6 dephosphorylates the D3 position of PtdIns3P and PtdIns(3,5)P₂ [339]. Introduction of a C336S mutation into the active site of MTMR6 completely abolishes its phosphatase activity toward both these substrates. MTMR6 is unique among MTM family members in its ability to negatively regulate the calcium-activated potassium channel Kca3.1 [357]. The coiled-coil domains present in the potassium channel protein and the MTMR6 protein mediate the interaction between them. Overexpression of MTMR6 results in Kca3.1 inhibition that depends on the phosphatase activity of MTMR6 and is rescued by the addition of PtdIns3P, but not by the addition of any other phosphoinositide [358].

Very little is known about MTMR7 and MTMR8, which resemble MTMR6 in structure but not in substrate specificity or function. MTMR7 is the only MTM family member that can dephosphorylate the water-soluble substrate inositol 1,3-bisphosphate in addition to PtdIns3P [359]. Northern blot and immunoblot analyses have demonstrated that mammalian MTMR7 is expressed mainly in the brain. Unlike overexpression of MTMR6, overexpression of MTMR8 does not inhibit Kca3.1 function [358].

5.4.7. MTMR9

MTMR9 was originally identified as an MTMR7-binding protein by tandem mass spectrometry of neuroblastoma cells [359]. The coiled-coil domain of MTMR9 is sufficient for its interaction with MTMR7. MTMR9 binding activates the phosphatase activity of MTMR7 but does not change its localization, unlike the binding of MTMR5 or MTMR12 to MTMR2. Yeast two-hybrid studies have shown that MTMR9 also interacts with MTMR6 [360]. This interaction results in increased enzymatic activity and stability of MTMR6 [361]. A SNP in the human MTMR9 gene is associated with obesity, suggesting that this phosphatase may be involved in appetite regulation [362].

5.4.8. MTMR14

Tosch et al. originally identified MTMR14 through the similarity of its catalytic motif to that of MTM1, and reported that two missense mutations, R336Q and Y462C, occur in the MTMR14 gene of centronuclear myopathy patients [363]. Compared with the WT MTMR14 enzyme, a recombinant MTMR14 protein bearing these mutations shows decreased phosphatase activity toward PtdIns3P and PtdIns(3,5)P₂ in vitro, and has little effect on PtdIns3P-positive structures detected by a GST-2xFYVE probe in COS-1 cells. The expression of murine MTMR14 increases with the differentiation of C2C12 myoblasts [363], implying that MTMR14 may play a physiological role in muscle function.

5.4.9. Other MTMR pseudophosphatases

MTMR10 and MTMR11 are inactive members of the MTMR family but little is known about their functions. Systematic analysis of heteromeric interactions of MTMR family members has revealed that MTMR10 interacts with MTM1 and MTMR2 [360]. In contrast, yeast two-hybrid screening has shown that MTMR11 is the sole MTMR protein with no heteromeric binding partner. Heterolo-gously-expressed, EGFP-tagged MTMR10 and MTMR11 localize mainly in the cytosol of HeLa cells but their functions here remain to be determined.

6. Phosphoinositide 4-phosphatases

6.1. Classification

Whereas the phosphoinositide 3- and 5-phosphatases comprise two large families of enzymes, only four phosphoinositide 4-phosphatases have been identified in mammals: two approx. 100 kDa inositol polyphosphate 4-phosphatases called INPP4A and INPP4B; and two 30 kDa transmembrane proteins called TMEM55A and TMEM55B. INPP4s dephosphorylate the D4 position of

| INPP4A | C2 | PEST | Ptase | CKSAKDR | 978aa |
| --- | --- | --- | --- | --- | --- |
| INPP4B |  |  | Ptase | CKSAKDR | 924aa |
| TMEM55A |  | Ptase | TM | CKDTSRR | 257aa |
| TMEM55B |  |  | TM | CKVTSQR | 277aa |

Fig. 7. Structural features of mammalian phosphoinositide 4-phosphatases. All members of this family contain a CX₅R motif (black bar; amino acid sequences indicated) in the phosphatase (Ptase) domain. Both INPP4s contain a C2 domain, and INPP4A has a PEST domain. The TMEM55s contain two TM-spanning regions as predicted by the TMHMM 2.0 webware of the Center for Biological Sequence Analysis (http://www.cbs.dtu.dk/index.shtml). Numbers of amino acids in the human proteins are indicated.

PtdIns(3,4)P₂, whereas the TMEM55 proteins dephosphorylate the D4 position of PI(4,5)P₂. In addition, among its other activities (see Section 7.10), the Sac family phosphatase Sac1 can dephosphorylate PtdIns4P. No PtdIns(3,4,5)P₃ 4-phosphatase has been identified thus far.

### 6.2. INPP4s

#### 6.2.1. Structure
INPP4A (also called type I INPP4) and INPP4B (type II INPP4) show 37% identity in amino acid sequence. Remarkably, the active site motif in the Ptase domains of these isozymes is identical (CKSAKDR) (Fig. 7). Both isozymes have an N-terminal C2 domain but the binding properties of these domains differ: the INPP4A C2 domain binds to PtdIns(3,4)P₂, PtdIns3P, phosphatidylserine and calcium [364] [365], whereas the C2 domain of INPP4B binds to PA and PtdIns(3,4,5)P₃ [366]. *In vitro*, deletion of the C2 domain of INPP4A abrogates PtdIns(3,4)P₂ dephosphorylation and impairs endosomal localization of the INPP4A enzyme. A specific role for the INPP4B C2 domain has yet to be established. INPP4A, but not INPP4B, contains two PEST sequences that serve as a cleavage motif recognized by the calcium-dependent protease calpain [367]. INPP4A can be inactivated by calpain-mediated proteolysis both *in vitro* and in platelets stimulated with thrombin plus calcium ionophore [368].

#### 6.2.2. Expression
INPP4A is highly expressed in brain and to a lesser extent in heart, skeletal muscle, spleen, kidney and lung [367]. INPP4B is highly expressed in heart and skeletal muscle, and to a lesser extent in brain, placenta and pancreas [369]. Overexpression studies have shown that INPP4A is localized in recycling and early endosomes in quiescent cells, and at the plasma membrane in serum-stimulated cells [364]. INPP4B appears to be diffusely distributed in the cytoplasm.

#### 6.2.3. Biological functions
In line with its endosomal localization, INPP4A is thought to influence membrane trafficking. INPP4A can be activated by Rab5, a critical regulator of endocytic transport [370]. Knockdown of INPP4A decreases transferrin uptake, suggesting that INPP4A promotes Rab5-dependent receptor-mediated endocytosis. The actions of INPP4A in the endosomal system are likely mediated by its lipid product PtdIns3P because INPP4A overexpression restores the dilation of early endosomes caused by a PtdIns3P deficiency [364]. In addition, Rab5 interacts with and activates phosphoinositide-metabolizing enzymes such as p110β and INPP5E (see Section 7.8), both of which may produce INPP4A’s substrate PtdIns(3,4)P₂. Another effect of INPP4A overexpression is to suppress the proliferation of megakaryocytes and fibroblasts [371]. This suppression may be attributable to the decrease in PtdIns(3,4)P₂ levels associated with increased INPP4A activity. PtdIns(3,4)P₂ is known to bind to and activate some effectors of PtdIns(3,4,5)P₃, which is clearly involved in cell growth. *In vitro* studies of INPP4B functions have yet to be reported.

Little work has been done on the *in vivo* functions of INPP4s. A spontaneous mutant mouse strain called weebie exhibits deletion of a single nucleotide in *Inpp4A* exon 10 and displays defective cerebellar development [372]. The frameshift generated by the weebie mutation is predicted to create a stop codon that truncates the INPP4A protein such that most of its C-terminus, including the phosphatase domain, is lost. The phenotypes of knockout mice lacking either *Inpp4A* or *Inpp4B* have yet to be reported.

#### 6.2.4. Disease association
A polymorphism present in the PEST sequence of INPP4A is associated with severe atopic asthma in humans [373]. Loss of INPP4B function may promote tumorigenesis, as transcription of *INPP4B* is reportedly downregulated in several human myeloid, erythroid and lymphoid leukemic cell lines [374].

### 6.3. TMEM55s

#### 6.3.1. Structure
Thus far, only two papers have been published on the enzymatic properties and biological functions of the TMEM55 proteins. TMEM55A and TMEM55B were originally cloned based on their homology to the *Burkholderia pseudomallei* virulence factor BopB, a putative phosphatase that contains a CX₅R phosphatase motif [375]. The amino acid sequences of human TMEM55A and TMEM55B are 51% identical to each other. Both isozymes contain a CX₅R motif in their central phosphatase domain and two putative TM domains at their C-termini. Catalytically, recombinant TMEM55 proteins are capable of specifically hydrolyzing the D4 phosphate of PtdIns(4,5)P₂ but do not hydrolyze any other phosphoinositides or soluble inositol phosphates.

#### 6.3.2. Expression
Both TMEM55A and TMEM55B are ubiquitously expressed in human tissues, and both isozymes show cytosolic and late endosomal membrane localization [375].

#### 6.3.3. Biological functions
In response to DNA damage, TMEM55B translocates from the cytosol to the nucleus [376]. This translocation is accompanied by an increase in PtdIns5P, a phosphoinositide species that binds to the PHD motif of ING2 and thus is implicated in p53-induced apoptosis [250]. Overexpression of TMEM55B (but not TMEM55A) enhances ING2-dependent p53 acetylation and cell death induced by genotoxic agents. Conversely, siRNA knockdown of TMEM55B leads to decreased stability of p53. Significantly, TMEM55B-dependent p53 acetylation and apoptosis are inhibited by PIPKIIB, a kinase that converts PtdIns5P to PtdIns(4,5)P₂ (see Section 4.3). It is therefore believed that TMEM55B controls nuclear levels of PtdIns5P and thus p53-dependent apoptosis.

The physiological function of TMEM55A is unclear but when this isozyme is overexpressed in HeLa cells, EGFR degradation induced by EGF stimulation is enhanced [375].

### 7. Phosphoinositide 5-phosphatases

#### 7.1. Classification
Inositol polyphosphate 5-phosphatases (INPP5s) are defined by the presence of an inositol 5-phosphatase (5-Ptase) domain that contains two consensus motifs, (F/I)WXGDXN(F/Y)R and (R/K)X(R/N)XP(S/A)(W/Y)(C/T)DR(I/V)L(W/Y), that are critical for substrate binding and catalysis (Fig. 8). There are four classes of INPP5s, types I–IV, but only types II–IV hydrolyze phosphoinositide substrates as well as soluble inositol phosphates. The type II INPP5 enzymes include the synaptojanins, OCRL1, INPP5B, INPP5J and SKIP. The type III INPP5 enzymes are SHIP1 and SHIP2, and the sole type IV enzyme is INPP5E.

Several other enzymes can act as phosphoinositide 5-phosphatases. PLIP does not have the two consensus motifs of the INPP5s but instead contains a phosphatase domain with the CX₅R motif CKAGRSR, which is identical to that in PTEN. The Sac family phosphatases Sac1, Sac2 and Sac3 can also hydrolyze 5-phosphate residues (described in Section 7.10).

T. Sasaki et al./Progress in Lipid Research 48 (2009) 307–343

| Type II | OCRL1 | V | I | W | L | G | D | L | N | Y | R |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SYNJ1 | V | F | W | C | G | D | F | N | Y | R |
| INPP5J | V | F | W | F | G | D | L | N | F | R |

| Type III | SHIP1 | L | F | W | F | G | D | L | N | Y | R |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SHIP2 | L | F | W | F | G | D | L | N | Y | R |

| Type IV | INPP5E | V | F | W | F | G | D | F | N | F | R |

Consensus 1

Consensus 2

K C R V P A W C D R I L W R  
K V R T P A W T D R V L W R  
K K R K P A W T D R I L W K  
K Y R L P S W C D R V L W W  
R T N V P S W C D R I L W K  
K Q R T P S Y T D R V L Y K  

Fig. 8. Consensus amino acid sequences present in the 5-phosphatase domains of type II, III and IV phosphoinositide 5-phosphatases. Sequences present in OCRL1, SYNJ1 and INPP5J (type II), SHIP1 and SHIP2 (type III), and INPP5E (type IV) are shown. Amino acids conserved in all of these 5-phosphatases are boxed. See Fig. 9 for the complete domain structures of these proteins.

### 7.2. Synaptojanins

#### 7.2.1. Structure

Two mammalian synaptojanin isozyms, SYNJ1 and SYNJ2, have been identified. SYNJ1 was first isolated as a protein capable of binding to the SH3 domain of Grb2 [377,378]. The SYNJ1 protein contains an N-terminal Sac domain that has inositol phosphatase activity (see Section 7.10), a central 5-Ptase domain, and a C-terminal PR region [379] (Fig. 9). SYNJ2 has a similar domain structure. In the rat, the SYNJ1 and SYNJ2 proteins share 57.2% amino acid sequence identity in their Sac1-like domains, and 53.8% identity in their 5-Ptase domains [380].

Multiple splice variants of both SYNJ1 and SYNJ2 exist (refer to Fig. 9) [379,380,381,382–384]. For SYNJ1, the major splice variants are SYNJ1-145 and SYNJ1-170 [385]. Compared to SYNJ1-145, the SYNJ1-170 protein has an additional 30 kDa extension at its C-terminus that contains three asparagine-proline-phenylalanine (NPF) repeats. The NPF repeat region is essential for the binding of SYNJ1-170 to the endocytic protein Eps15 [386]. The major splice variants of SYNJ2 are SYNJ2A, SYNJ2B1 and SYNJ2B2, which differ structurally in their C-terminal regions (refer to Fig. 9).

With respect to substrate specificity, all SYNJ1 and SYNJ2 enzymes use their central 5-Ptase domains to dephosphorylate Ins(1,4,5)P₃, Ins(1,3,4,5)P₄, PtdIns(4,5)P₂ and PtdIns(3,4,5)P₃ at the D5 position of the inositol ring [379,387]. The N-terminal Sac domain is thought to allow SYNJ1 and SYNJ2 proteins to act as 3- and 4-phosphatases and remove the D3 and D4 phosphates of PtdIns3P, PtdIns4P and PtdIns(3,5)P₂ [388].

#### 7.2.2. Expression

SYNJ1-145 and SYNJ1-170 are both expressed in a wide variety of tissues, but only SYNJ1-145 is present at high levels in brain [378,380,385]. Immunofluorescence and electron microscopy studies using an antibody that recognizes SYNJ1 isoforms have revealed that SYNJ1 is concentrated in nerve terminals, especially in clathrin-coated endocytic intermediates [377,386].

The multiple splice variants of SYNJ2 are also expressed in a range of tissues [380–382,384]. SYNJ2A is ubiquitously expressed but SYNJ2B1 and SYNJ2B2 are abundantly expressed in testis and brain [384]. At the subcellular level, SYNJ2A is recruited to mitochondria via interaction with mitochondrial outer membrane protein OMP25 [383]. In contrast, an immunofluorescence study using an antibody recognizing both SYNJ2B1 and 2 revealed that SYNJ2B localizes at nerve terminals in brain and at the spermatid manchette in testis [384]. Overexpressed SYNJ2B is associated with membrane ruffles [389].

#### 7.2.3. Biological functions

##### 7.2.3.1. SYNJ1

SYNJ1⁻/⁻ mice exhibit several neurological defects and die shortly after birth [390]. In neurons from these mutants, PtdIns(4,5)P₂ levels are increased, and abnormal accumulation of clathrin-coated vesicles in nerve terminals and increased synaptic depression are observed. SYNJ1⁻/⁻ neurons are also impaired in the re-entry of recycling vesicles into the fusion-competent synaptic vesicle pool [391]. These results indicate that SYNJ1 plays an essential role in the post-endocytic uncoating of vesicles. Mani et al. have performed quantitative synaptic vesicle recycling assays in SYNJ1⁻/⁻ neurons and have found that, in addition to vesicle uncoating, SYNJ1 is critical for the endocytosis step itself [392]; specifically, the catalytic activities of both phosphatase domains are required. Potential functional differences between the SYNJ1-145 and SYNJ1-170 isoforms in endocytosis have been revealed by live-imaging experiments using total internal reflection fluorescence microscopy (TIRFM) [393]. SYNJ1-145 is rapidly recruited at a late stage of clathrin-coated pit formation, whereas SYNJ1-170 is present at all stages of clathrin-coated pit formation.

In accordance with its functions in endocytosis and synaptic vesicle recycling, SYNJ1 interacts with and cooperates with many proteins involved in these processes. In particular, the PR regions in the C-termini of both SYNJ1-145 and SYNJ1-170 bind to the SH3 domains of endocytic proteins such as amphiphysin, endophilin, syndapin/pacsin and intersectin/Dap160 [379,394–397]. Furthermore, SYNJ1-170 uses its unique NPF-containing C-terminus to bind to the EH (Eps15 homology) domains of Eps15, to the ear domain of the α-adaptin component of the AP2 complex, and to the N-terminal domain of the clathrin heavy chain [386,398].

##### 7.2.3.2. SYNJ2

SYNJ2 knockout mice have yet to be described. In vitro work has suggested that SYNJ2 has a variety of functions, depending on the biological context. (1) Like SYNJ1, SYNJ2 is involved in endocytosis; siRNA-mediated depletion of SYNJ2 in lung carcinoma cells causes a defect in clathrin-mediated receptor endocytosis [399], and SYNJ2 interacts with the small GTPase Rac1 in a GTP-dependent manner and thereby regulates clathrin-mediated endocytosis [389]. In addition, the SYNJ2B1 and SYNJ2B2 splice variants use their PR domains to interact with the same collection of SH3-containing endocytic proteins as are bound by SYNJ1 proteins [384]. (2) In the context of cell migration, RNAi-mediated inhibition of SYNJ2 expression prevents the formation of lamellipodia and invadopodia, resulting in defective cell movement [400]. (3) As mentioned above, the C-terminal motif present in SYNJ2A binds to OMP25 and recruits SYNJ2A to these organelles [383]. Overexpression of OMP25 in CHO cells induces perinuclear clustering of the mitochondria, and this phenotype is phenocopied by overexpression of mitochondria-tagged SYNJ2A. SYNJ2 may therefore play a role in the intracellular distribution of mitochondria.

#### 7.2.4. Regulation

Like other synaptic proteins, SYNJ1 is constitutively phosphorylated in nerve terminals and undergoes dephosphorylation by cal-

Type II
- SYNJ1-145: Sac domain, 5-Ptase, PR, NPF (1292 aa)
- SYNJ1-170: Sac domain, 5-Ptase, PR, NPF (1558 aa)
- SYNJ2A: Sac domain, 5-Ptase (1248 aa)
- SYNJ2B1: Sac domain, 5-Ptase, PR (1451 aa)
- SYNJ2B2: Sac domain, 5-Ptase, PR (1496 aa)
- OCRL1: ASH, RhoGAP (901 aa)
- INPP5B: 5-Ptase, CAAX (993 aa)
- INPP5J: SKICH (1006 aa)
- SKIP (448 aa)

Type III
- SHIP1α: SH2, 5-Ptase, C2, PR, NPXY (1189 aa)
- SHIP1β: SH2, 5-Ptase, C2, PR (1128 aa)
- SHIP1δ: SH2, 5-Ptase, C2 (959 aa)
- s-SHIP1: 5-Ptase, C2, PR (928 aa)
- SHIP2: SH2, 5-Ptase, C2, PR, SAM (1258 aa)

Type IV
- INPP5E: PR, 5-Ptase, CAAX (644 aa)

Others
- PLIP: Ptase (201 aa)
- SAC1: Sac domain (587 aa)
- SAC2: Sac domain (1132 aa)
- SAC3: Sac domain (907 aa)

Fig. 9. Structural features of the mammalian phosphoinositide 5-phosphatases. The 5-phosphatase (5-Ptase) domain in the type II, III and IV enzymes shown contains the two consensus motifs illustrated in Fig. 8. A CX₅R motif is present in the phosphatase (Ptase) domain of PLIP. The Sac domains of the synaptojanins and Sac phosphatases also contain a CX₅R motif. Several type II, III and IV enzymes contain a PR, and the type III enzymes contain a C2 domain. Additional domains as indicated are: NPF, asparagine-proline-phenylalanine repeats; GAP, GTPase-activating domain; CAAX, cysteine-two aliphatic amino acids-any amino acid box; SKICH, SKIP carboxyl homology domain; SH2, Src homology domain; NPXY, asparagine-proline-any amino acid-tyrosine motif; SAM, sterile α motif. Numbers of amino acids in the human proteins are indicated.

cineurin in response to depolarization [377]. The phosphorylation status of SYNJ1 affects its 5-phosphatase activity and interaction with partner proteins. For example, the phosphorylation of SYNJ1 on Ser1144 by cdk5 inhibits both SYNJ1 catalytic activity and SYNJ1's interaction with endophilin-1 and amphiphysin [401]. In addition, the phosphorylation of tyrosines in the SYNJ1 PR region by the Eph2 receptor suppresses SYNJ1 catalytic activity and prevents SYNJ1 interaction with endophilin-1, but does not impede SYNJ1 binding to amphiphysin [402]. Little is known of the regulation of SYNJ2 proteins.

7.3. OCRL1

7.3.1. Structure
OCRL1 (oculocerebrorenal syndrome of Lowe-1) is closely related in structure to INPP5B (see Section 7.4). OCRL1 and INPP5B are the only known RhoGAP-like domain-containing 5-phosphatases in humans and mice [403–406]. These enzymes contain the conserved central 5-Ptase domain, followed by an ASH (ASPM, SPD2, hydin) domain [407] and the RhoGAP-like domain [408–410] (refer to Fig. 9). In the RhoGAP-like domains of OCRL1
and INPP5B, the arginine residue in the critical catalytic region, which is conserved in other RhoGAP-containing proteins, has been changed to a glutamine. As a result, the RhoGAP-like domains of these proteins appear to lack catalytic activity [409]. Consistent with its unique structure, the RhoGAP-like domain of OCRL1 interacts with the Rho family small GTPases Rac1 and Cdc42, even when these proteins are present in their GDP-bound forms [410–412]. A recent study has also suggested that the RhoGAP-like domain of OCRL1 can interact with ARF1 and ARF6 [413].

Two alternative splice variants of OCRL1 have been reported: OCRL1a and OCRL1b [414]. Compared to OCRL1b, OCRL1a contains an eight amino acid insertion in the RhoGAP domain that differentially affects both its localization and its binding to partner proteins (see below).

With respect to substrate specificity, OCRL1 primarily removes the 5-phosphates of PtdIns(4,5)P₂, PtdIns(3,4,5)P₃, Ins(1,4,5)P₃ and Ins(1,3,4,5)P₄ [387,404,415,416].

### 7.3.2. Expression

OCRL1a and OCRL1b are both expressed in almost all tissues but only OCRL1a is expressed in brain [414].

At the subcellular level, at least in cell lines, OCRL1 is localized in the TGN, early endosomes, membrane ruffles and clathrin-coated trafficking intermediates [412,417–421]. Lysosomal localization of OCRL1 was reported in a kidney tubule cell line [416] but has been not confirmed in other studies [418,419]. Pertinent to its localization, OCRL1 contains a LIDLE clathrin-binding motif that allows it to directly interact with the terminal domain of the clathrin heavy chain [412,419,420]. In OCRL1a, the LIDLE motif is immediately adjacent to the eight amino acid insertion present in the RhoGAP-like domain. Interestingly, OCRL1a binds clathrin with higher affinity than does OCRL1b, and thus is more enriched in clathrin-coated pits [413]. A second clathrin-binding site has been identified in the N-terminal non-catalytic regions of both OCRL1a and b. OCRL1 also interacts with the α-adaptin subunit of AP2 via its FEDNF motif in the N-terminus [419,420]. In addition, OCRL1 can bind directly to the endosomal protein Rab5 and the Rab5 effector APPL1. Each of these interactions contributes independently to the recruitment of OCRL1 to endosomal compartments [412,422,423].

### 7.3.3. Biological functions

Overexpression studies and RNAi knockdown experiments have demonstrated that OCRL1 regulates vesicle trafficking between the endosomes and the TGN [420], consistent with OCRL1’s localization. As noted above, in response to EGF-induced Rac1 activation, OCRL1 translocates to the membrane ruffle. In addition, overexpression of the RhoGAP-like domain of OCRL1 perturbs Rac-dependent actin reorganization [421]. OCRL1-deficient fibroblasts prepared from patients with Lowe’s syndrome (see Section 7.3.5) show decreased actin stress fibers and increased actin puncta [424]. This abnormal cytoskeletal reorganization is associated not only with dysregulation of RhoGAP-like domain functions but also with inhibition of the modulation of PtdIns(4,5)P₂ levels normally mediated by OCRL1 5-phosphatase activity.

OCRL1⁻/⁻ mice show no gross defects or signs of a Lowe’s syndrome phenotype. However, knockout of both OCRL1 and INPP5B results in embryonic lethality, suggesting that the absence of abnormal phenotypes in OCRL1⁻/⁻ mice may be due to partial functional redundancy with INPP5B [425].

### 7.3.4. Regulation

Rab small GTPases regulate the membrane targeting and activation of OCRL1 [422]. Specifically, OCRL1 can interact with Golgi-associated Rab1 and Rab6, and with endosomal Rab5. Importantly,

mutated OCRL1 proteins that cannot bind to Rabs fail to translocate to the TGN and endosomes, indicating that this interaction with Rabs is indispensable for OCRL1 localization. *In vitro* reconstitution experiments using recombinant OCRL1 and Rab proteins have shown that Rab5 and Rab6 directly stimulate the 5-phosphatase activity of OCRL1.

#### 7.3.5. Disease association

Mutations of the human OCRL1 gene are responsible for Lowe’s syndrome, a rare X-linked disorder characterized by congenital cataracts, mental retardation, and renal Fanconi syndrome [408,426,406,427]. Mutations in this gene are also involved in Dent disease, a condition characterized by defective kidney reabsorption [428]. The OCRL1 mutations linked to Lowe’s syndrome and Dent disease are varied and include nonsense mutations, frameshift mutations and missense mutations ([http://research.nhgri.nih.gov/lowe](http://research.nhgri.nih.gov/lowe)). The missense mutations are most often found in the OCRL1 catalytic domain, indicating the importance of OCRL1’s 5-phosphatase activity. A small number of missense mutations are found in the ASH domain or the RhoGAP-like domain ([http://research.nhgri.nih.gov/lowe/](http://research.nhgri.nih.gov/lowe/)) [429–431]. However, the interaction of OCRL1 with APPL1 that requires OCRL1’s ASH and RhoGAP-like domains is abolished by all known disease mutations in these regions, suggesting the importance of these domains in the pathogenesis [412,423].

### 7.4. INPP5B

#### 7.4.1. Structure

INPP5B displays much of the same domain organization as OCRL1 and is 45% identical to it in amino acid sequence [403–405] (refer to Fig. 9). However, the gene encoding INPP5B is present only in higher eukaryotes like humans and mice. In addition to the 5-Ptase, ASH and RhoGAP-like domains, INPP5B contains a C-terminal CAAX motif that is prenylated on the cysteine residue [404]. Three INPP5B isoforms of molecular weights 87 kDa, 105 kDa and 115 kDa are produced by tissue-specific and developmentally-regulated alternative splicing [405].

With respect to substrate specificity, like OCRL1, INPP5B primarily removes the 5-phosphates of PtdIns(4,5)P₂, PtdIns(3,4,5)P₃, Ins(1,4,5)P₃ and Ins(1,3,4,5)P₄.

#### 7.4.2. Expression

INPP5B is ubiquitously expressed and highly enriched in the kidney, testis and lung [405].

At the subcellular level, INPP5B was initially reported to occur in both the mitochondria and cytosol [403]. However, the overexpression of a constitutively active form of Rab5 results in the localization of INPP5B in enlarged endosomes and translocation of the enzyme to the membrane ruffles when the cells are stimulated with serum [370,403]. Williams et al. have refined INPP5B localization further, showing that this phosphatase can be found not only in the Golgi apparatus but also in the ER-to-Golgi intermediate compartment (ERGIC) [432]. These data implicate INPP5B in the early secretory pathway. Interaction of INPP5B with Rab proteins has also been reported to affect the membrane localization and enzymatic activation of INPP5B [370,432]. Unlike OCRL1, however, INPP5B is not present in clathrin-coated vesicles, probably due to the failure of INPP5B to interact with clathrin and α-adaptin [432].

#### 7.4.3. Biological functions

As mentioned above, given the localization of INPP5B in the Golgi and ERGIC, this phosphatase is thought to operate in the early secretory pathway. Consistent with this hypothesis, when HeLa cells overexpressing INPP5B are treated with brefeldin A,the cycling protein ERGIC53 is redistributed from the ER to the ERGIC [432], an event required for secretion.

INPP5B$^{-/-}$ mice appear normal but, unlike OCRL1$^{-/-}$ mice, have a defect in sperm functions [433]. Sperm from INPP5B$^{-/-}$ males show reduced motility and impaired ability to fertilize eggs. These phenotypes are due to decreased proteolytic processing of fertilin-β, which is a sperm surface protein implicated in sperm–egg membrane binding. In testicular Sertoli cells of INPP5B$^{-/-}$ mice, apical endocytosis is disturbed, resulting in the appearance of abnormal swollen vacuoles that contain intact adherence junction components [433]. INPP5B may therefore play important roles in endocytosis and plasma membrane recycling in Sertoli cells.

### 7.5. INPP5J

#### 7.5.1. Structure

INPP5J was originally cloned based on the presence of the two consensus motifs characteristic of inositol polyphosphate 5-phosphatases [434] (refer to Fig. 8). Defined domains and motifs in the INPP5J protein include an N-terminal PR region containing a putative SH3-binding motif (PRSPSR), six 14-3-3 binding motifs, and a SKICH domain [435] (see Section 7.6.1 and refer to Fig. 9). With respect to catalysis, INPP5J removes the D5 phosphate of PtdIns(4,5)P₂, PtdIns(3,4,5)P₃, Ins(1,4,5)P₃ and Ins(1,3,4,5)P₄.

#### 7.5.2. Expression

INPP5J mRNA has been detected in brain, heart, kidney, stomach, small intestine and lung [434]. In terms of subcellular localization, INPP5J is found at the plasma membrane [434].

#### 7.5.3. Biological functions

Thus far, little is known about the biological functions of INPP5J. Ooms et al. reported that overexpression of INPP5J in PC12 cells decreases PtdIns(3,4,5)P₃ accumulation, Akt phosphorylation, and neurite elongation [436]. Conversely, siRNA-mediated depletion of endogenous INPP5J led to increased PtdIns(3,4,5)P₃ accumulation, enhanced Akt phosphorylation at the neurite growth cone, and neurite hyperelongation.

### 7.6. SKIP

#### 7.6.1. Structure

SKIP (skeletal muscle- and kidney-enriched inositol polyphosphate phosphatase) also contains the two consensus inositol polyphosphate 5-phosphatase motifs and a SKIP carboxyl homology domain (SKICH) domain [435] (refer to Fig. 8 and Fig. 9). Two SKIP isoforms of 51 kDa and 43 kDa exist as a result of alternative splicing. In vitro, SKIP strongly prefers PtdIns(3,4,5)P₃ as a substrate, although it can also act on PtdIns(4,5)P₂, Ins(1,4,5)P₃ and Ins(1,3,4,5)P₄ [437].

#### 7.6.2. Expression

SKIP is expressed ubiquitously in human tissues but its expression is especially high in heart, skeletal muscle and kidney [438]. SKIP is found primarily in the TGN and ER of resting cells. Following insulin stimulation, SKIP is recruited to the plasma membrane via its SKICH domain.

#### 7.6.3. Biological functions

The primary function of SKIP is to negatively regulate PI3K-dependent cellular processes at the plasma membrane. Both in vitro and in vivo results indicate that SKIP is a major suppressor of insulin action in skeletal muscle. Ectopic expression of SKIP decreases insulin-induced intracellular PtdIns(3,4,5)P₃, reduces GLUT4 translocation to the plasma membrane, and inhibits glucose uptake and glycogen synthesis in a manner dependent on SKIP's

### 7.7. SHIPs

#### 7.7.1. Structure

The SHIP (SH2-containing inositol phosphatase) family comprises SHIP1 (also known as INPP5D) and SHIP2 (also known as INPP5L1). Alternative splicing of the transcript encoding full-length SHIP1 (SHIP1α) creates three shorter isoforms called SHIP1β, SHIP1δ and s-SHIP1 [440,441].

The prototype enzyme SHIP1α was originally identified based on its ability to bind to adaptor proteins such as Shc, Grb2 and Dok, and to the immunoreceptor FcγRIIB [442]. As shown in Fig. 9, all SHIP1 isoforms and SHIP2 contain an N-terminal SH2 domain and C-terminal PR regions that facilitate binding to a plethora of kinases, adaptor proteins and membrane receptors with ITIM (immunoreceptor tyrosine inhibitory motif) or ITAM (immunoreceptor tyrosine activation motif) sequences. The C-terminal tails of all SHIP enzymes contain tyrosines within NPXY motifs. When phosphorylated, these tyrosines provide binding sites for the phosphotyrosine-binding (PTB) domains and SH2 domains present in partner proteins. Unlike SHIP1, SHIP2 has a unique C-terminal SAM (sterile α motif) domain that is a putative protein–protein interaction module.

Although the 5-phosphatase domains are very similar among SHIP1 isoforms and SHIP2 (65% sequence identity), the SHIP1 isoforms hydrolyze the 5-phosphate of both PtdIns(3,4,5)P₃ and Ins(1,3,4,5)P₄ whereas SHIP2 activity appears to be more specific for PtdIns(3,4,5)P₃ [443]. The catalytic activity of SHIP1 can be allosterically enhanced by the binding of its lipid product PtdIns(3,4)P₂ to its C2 domain [444]. However, the intrinsic phosphatase activity of SHIP1 does not appear to be stimulated by protein–protein interaction [445].

#### 7.7.2. Expression

SHIP1α, SHIP1β and SHIP1δ are exclusively expressed in cells of hematopoietic and spermatogenic origin whereas SHIP2 is expressed ubiquitously [446]. s-SHIP1 is expressed in primitive cell populations but not in any lineage-committed cells [441]. SHIP1 is recruited to the plasma membrane from the cytosol following membrane receptor stimulation. For example, SHIP1 is recruited to the phagocytic cup in response to Fcγ receptor and complement receptor 3 activation in macrophages [447]. In osteoclasts, SHIP1 is localized to podosomes under the influence of c-Src tyrosine kinase [448].

#### 7.7.3. Biological functions of SHIP1

Genetic manipulations of Ship1 in mice have shown that this phosphatase plays important roles in myeloid cell homeostasis, chemotaxis, bone metabolism and mast activation.

##### 7.7.3.1. Myeloid cell homeostasis

SHIP1$^{-/-}$ mice are viable but have shortened lifespans [449,450]. These mutants suffer from progressive myeloid hyperplasia leading to splenomegaly, lymphadenopathy, and myeloid cell infiltration of vital organs. SHIP1$^{-/-}$ hematopoietic progenitors exhibit hyperproliferative responses to stimulation by monocyte colony-stimulating factor (M-CSF), granulocyte colony-stimulating factor (G-CSF), interleukin-3, or SCF [449]. In mature neutrophils and mast cells, PtdIns(3,4,5)P₃ levels and Akt activity are significantly elevated in the absence of SHIP1[450]. Accordingly, these cells exhibit enhanced resistance to cell death induced by various apoptotic stimuli or growth factor withdrawal. Although these findings suggest that SHIP1 could act as a tumor suppressor of myeloid leukemia, thus far, a loss-of-function mutation of the human SHIP1 gene has yet to be identified in patients with myeloid leukemia.

### 7.7.3.2. Chemotaxis
PI3K and PtdIns(3,4,5)P₃ are thought to be involved in establishing and maintaining cellular polarity by regulating the subcellular localization and activation of downstream effectors that are essential for proper chemotaxis. SHIP1⁻/⁻ neutrophils migrate much more slowly than do WT cells, and are defective in spatially-restricted F-actin assembly in response to chemoattractants [451]. PtdIns(3,4,5)P₃ is distributed broadly around the perimeter of SHIP1⁻/⁻ neutrophils and is not confined to a single membrane region. Thus, it is likely that SHIP1 has a critical role in restricting where PtdIns(3,4,5)P₃ and F-actin can accumulate in migrating neutrophils. The premature death of SHIP1⁻/⁻ mice may be partly due to a dysregulation of chemotaxis that leads to an uncontrolled accumulation of damaging inflammatory cells.

### 7.7.3.3. Osteoporosis
Osteoclasts are cells of monocyte-macrophage origin that resorb bone matrix. In vitro, the differentiation of precursor cells into mature osteoclasts is stimulated by M-CSF and RANKL (receptor activator of nuclear factor-κB ligand). Takeshita et al. showed that the in vitro osteoclastogenesis induced by M-CSF and RANKL is increased in the absence of SHIP1 [452]. Furthermore, SHIP1⁻/⁻ osteoclasts are resistant to cell death and show an increased capacity to resorb mineralized matrix.
In WT mouse osteoclasts, SHIP1 localizes at podosomes in a c-Src-dependent manner and negatively regulates bone resorption. Whereas c-Src⁻/⁻ mice are osteopetrotic, SHIP1⁻/⁻ mice exhibit severe osteoporosis due to the presence of numerous enlarged and hypernucleated osteoclasts. Interestingly, SHIP1⁻/⁻ osteoclasts are morphologically similar to those in patients with Paget disease, a late-onset disorder characterized by focal areas of increased bone turnover that contain enlarged hyperactive osteoclasts [452]. However, mutations of the SHIP1 gene have yet to found in patients with this disease.

### 7.7.3.4. Mast cell activation
Resting mast cells bear FcεRI molecules that are routinely engaged by free IgE antibodies. When several of these IgE-bearing FcεRI molecules are aggregated by polyvalent antigen, mast cells are activated and release proinflammatory mediators such as histamine and TNFα, and initiate allergic reactions. The β and γ subunits of the FcεRI multimer contain ITAMs that can bind to the SH2 domain of SHIP1. Studies of bone marrow-derived mast cells from SHIP1⁻/⁻ mice have revealed that SHIP1 is a negative regulator of FcεRI-mediated cellular responses. The binding of IgE to FcεRI present on SHIP1⁻/⁻ mast cells (without crosslinking by antigen) is sufficient to trigger PtdIns(3,4,5)P₃ accumulation and a degranulation response [450,453]. These data suggest that SHIP1 may act as a gatekeeper that prevents the degranulation signal from progressing unless it is overwhelmed by the massive signaling initiated by antigens.
The meroterpenoid pelorol is a SHIP1 agonist isolated from Dactylospongia elegans. Ong et al. have investigated SHIP1 activation as a means of inhibiting allergic reactions, and have developed two structural analogs of pelorol, AQX-016A and AQX-MN100, that can bind to the SHIP1 C2 domain and allosterically activate SHIP's phosphatase activity towards PtdIns(3,4,5)P₃ [444]. In WT hematopoietic cells, but not in those from SHIP1⁻/⁻ mice, these AQX compounds inhibit TNFα production and PtdIns(3,4,5)P₃-regulated signal transduction events (e.g., Akt activation). In a mouse cutaneous anaphylaxis model, topical application of the AQX compounds dramatically suppresses allergen-induced inflammation and edema. This inhibition of PtdIns(3,4,5)P₃-dependent processes by SHIP1-activating compounds may represent a powerful new strategy for treating diseases in which the PI3K/Akt pathway is aberrantly upregulated.

### 7.7.4. Biological functions of SHIP2
Several lines of evidence suggest that SHIP2 negatively regulates insulin-stimulated cellular actions. (1) In humans, polymorphisms in the SHIP2 gene that lead to decreased SHIP2 activity have been associated with type 2 diabetes mellitus [454]. (2) Overexpression of SHIP2 reduces insulin-induced PtdIns(3,4,5)P₃ accumulation, Akt activation, GSK3-β phosphorylation, 2-deoxyglucose uptake, and GLUT4 translocation [455]. (3) A Ship2 knockout strain that also lacks a neighboring gene, Phox2a, exhibits increased sensitivity to insulin in the form of severe neonatal hypoglycaemia and deregulated expression of gluconeogenesis genes; these animals die shortly after birth [457]. Adult heterozygotes of this strain show elevated glucose tolerance and heightened insulin sensitivity associated with enhanced recruitment of GLUT4 and increased glycogen synthesis in skeletal muscles. On the other hand, Sleeman et al. reported a SHIP2 knockout strain that are viable and show normal insulin and glucose tolerance [456]. Additional studies are needed to resolve these discrepancies.

### 7.8. INPP5E

#### 7.8.1. Structure
INPP5E is also called the type IV 5-phosphatase, the 72 kDa inositol polyphosphate 5-phosphatase, and Pharbin [458–460]. INPP5E possesses an N-terminal PR region, the characteristic central 5-Ptase domain, and a C-terminal CAAX motif (refer to Fig. 9). INPP5E hydrolyzes PtdIns(3,5)P₂, PtdIns(4,5)P₂, and PtdIns(3,4,5)P₃. Kisseleva et al. have reported that INPP5E has the highest affinity of any 5-phosphatase for PtdIns(3,4,5)P₃ [459]. Indeed, INPP5E has 10 times greater affinity for PtdIns(3,4,5)P₃ than does SHIP1, based on \(K_m\) values.

#### 7.8.2. Expression
INPP5E is highly expressed in testis, brain, and heart, and is localized predominantly in the plasma membrane and the Golgi [460].

#### 7.8.3. Biological functions
Horan et al. have reported that INPP5E overexpression in macrophages reduces PtdIns(3,4,5)P₃ to almost undetectable levels, and increases PtdIns(3,4)P₂ at the phagocytic cup when a macrophage engulfs an IgG-coated particle. These observations correlate with a significant decrease in macrophage-mediated phagocytosis [447]. Conversely, siRNA knockdown of INPP5E in macrophages results in amplified and prolonged PtdIns(3,4,5)P₃ accumulation at the cup. Interestingly, INPP5E is dynamically recruited to the phagocytic cup in response to the activation of FcγR but not complement receptor 3 (CR3), and INPP5E overexpression does not inhibit CR3-mediated phagocytosis. This restricted negative regulatory role for INPP5E stands in sharp contrast to that of SHIP1, which suppresses both FcγR- and CR3-mediated phagocytosis [461,462]. The available data suggest that INPP5E specifically regulates phagocytosis in response to FcγR engagement by hydrolyzing PtdIns(3,4,5)P₃ at the plasma membrane.
Contrary to its role in modulating PtdIns(3,4,5)P₃ and PtdIns(3,4)P₂ levels in macrophages, INPP5E overexpression has little effect on PtdIns(3,4,5)P₃ accumulation in insulin-stimulated
adipocytes [463]. Rather, in adipocytes, INPP5E generates PtdIns3P by hydrolyzing PtdIns(3,5)P₂. Ectopic expression of INPP5E in adipocytes promotes both the translocation and insertion of GLUT4 into the plasma membrane, consistent with the proposed role of PtdIns3P in regulating GLUT4 localization [464,465]. Thus, INPP5E can function as both a signal-terminating and a signal-initiating enzyme, and can influence the metabolism of different phosphoinositides depending on the cell type in question and its activation status. The *in vivo* significance of INPP5E’s ability to regulate macrophages and adipocytes has yet to be confirmed by genetic approaches.

### 7.9. PLIP

#### 7.9.1. Structure
PTEN-like lipid phosphatase (PLIP) was originally identified through computer database searches of the *Dictyostelium* genome using the sequence in the phosphatase active site region of PTEN as a query [466]. PLIP is a highly conserved phosphatase with over 60 orthologs in metazoans. The catalytic Ptase domain in PLIP is most closely related to the Ptase domain of PTEN and does not have the two consensus sequences present in the INPP5s. One of the most distinctive structural features of *Dictyostelium* PLIP is the presence of a putative helical TM domain at its N-terminus [466]. However, mammalian orthologs of PLIP do not contain a putative TM domain [467].

*In vitro*, PLIP has a unique preference for PtdIns5P as a substrate [467]. Kinetic analyses have shown that murine PLIP’s *K*ₘ value for PtdIns5P is 150-fold lower than its *K*ₘ value for the artificial phosphotyrosine analog pNPP. Moreover, the protein phosphatase activity of PLIP toward radiolabeled myelin basic protein is over 1000-fold lower than its activity toward PtdIns5P [467].

#### 7.9.2. Expression
RT-PCR analyses have revealed that the highest levels of PLIP mRNA expression occur in testis, with lower levels present in brain, liver, skeletal muscle and uterus [467]. Studies of the pancreatic insulinoma cell line INS-1 832/13 have indicated that PLIP resides almost exclusively in the mitochondria [468].

#### 7.9.3. Biological functions
Although PLIP acts on PtdIns5P *in vitro*, the physiological role of PLIP as a lipid phosphatase in intact cells remains unclear. The PLIP protein has also been identified as a protein tyrosine phosphatase called PTPMT1 (protein tyrosine phosphatase localized to the mitochondrion-1). Pagliarini et al. have shown that the mitochondrial localization of PLIP/PTPMT1 depends on its N-terminal signal sequences [468]. Contrary to *in vitro* studies, knockdown of PLIP/PTPMT1 in rat insulinoma cells alters their mitochondrial phosphoprotein profile but not their PtdIns5P levels. Further analysis is needed to clarify whether the endogenous substrates of PLIP/PTPMT1 are phosphoproteins or PtdIns5P or both.

### 7.10. Sac family phosphatases

#### 7.10.1. Structure
The Sac (suppressor of actin) domain was originally identified in a yeast protein, Sac1p, that was isolated in a suppressor screening of a temperature-sensitive actin gene mutation (act1-1) in *Saccharomyces cerevisiae* [469]. The 400 amino acids of the yeast Sac domain are organized into seven highly conserved motifs. The RXNCLDCLDRTN sequence within the sixth motif corresponds to the catalytic CX₅R motif common to many mammalian protein and phosphoinositide phosphatases. The N-terminal region of the yeast Sac domain contains a leucine zipper-like motif that may promote the oligomerization of Sac family phosphatases. In mammals, sequences homologous to the Sac domain have been found in the synaptojanins (see Section 7.2) as well as in the Sac family phosphatases Sac1, Sac2 and Sac3 (refer to Fig. 9).

With respect to catalysis, the Sac domain in various Sac family members mediates the dephosphorylation of the D3, D4 and/or D5 positions of various phosphoinositides. The preferred reactions carried out by each of these enzymes are detailed in Fig. 2 and Table 2.

##### 7.10.1.1. Sac1
Mammalian Sac1 was first identified in a rat brain cDNA library [470]. Rat Sac1 has the same substrate specificity as yeast Sac1p, in that it acts on PtdIns3P, PtdIns4P and PtdIns(3,5)P₂ but not on PtdIns(4,5)P₂. This specificity holds true both *in vivo* and *in vitro*. (It should be noted that the substrate specificity of the Sacs has most often been determined by *in vitro* assays of purified GST fused to the Sac domain derived from each enzyme.) Interestingly, although human Sac1 shows 95% amino acid identity to rat Sac1 [471], the GST-fused human Sac1 domain efficiently dephosphorylates only PtdIns4P and PtdIns3P.

##### 7.10.1.2. Sac2
Unlike Sac1 and Sac3, mammalian Sac2 has no yeast counterparts [472]. Unlike human Sac1, human Sac2 dephosphorylates only PtdIns(4,5)P₂ and PtdIns(3,4,5)P₃ at the D5 position and does not hydrolyze PtdIns3P, PtdIns4P or PtdIns(3,5)P₂.

##### 7.10.1.3. Sac3
Mammalian Sac3 has been reported by two groups to be the counterpart of yeast Fig4p, which is a highly specific PtdIns(3,5)P₂ 5-phosphatase [267,473]. However, the substrate specificity of mammalian Sac3 is not yet clear. Yuan et al. have shown *in vitro* that the GST-fused Sac domain of rat Sac3 preferentially dephosphorylates PtdIns3P, PtdIns4P and PtdIns(3,5)P₂, consistent with the substrate specificity of rat Sac1 [473]. On the other hand, Sbrissa et al. have demonstrated *in vitro* that immunoprecipitated Myc-tagged human Sac3 hydrolyzes PtdIns(3,5)P₂, PtdIns(3,4,5)P₃ and PtdIns(4,5)P₂ [268]. In intact HEK293 cells, siRNA knockdown of Sac3 results in a slight increase in [³²Pi]PtdIs(3,5)P₂ accumulation. Taken together, these studies indicate that mammalian Sac3 is most likely a PtdIns(3,5)P₂ 5-phosphatase like yeast Fig4p.

#### 7.10.2. Expression
Northern blotting and immunoblotting analyses have shown that rat Sac1 is ubiquitously expressed in adult tissues. Subcellular fractionation and immunostaining studies have localized this enzyme to the ER, an event that depends on Sac1’s C-terminal putative TM region. In contrast to rat Sac1, human Sac1 protein fused to GFP localizes to both the ER and the TGN [471], and the regulation of Golgi PtdIns4P levels by Sac1 has a direct role in growth-dependent secretion [474]. In human fibroblasts cultured under standard growth conditions, a large proportion of the intracellular Sac1 pool localizes to the ER in an event mediated by the coatamer complex coat protein (COP)-I. However, under starvation conditions, Sac1 interacts with COP-II and is induced to undergo oligomerization via its N-terminal leucine zipper motif. This oligomerization and binding to COP-II results in the accumulation of Sac1 in the TGN, an event that subsequently decreases PtdIns4P levels in the Golgi and inhibits constitutive secretion. The Sac1 oligomers dissociate and are redistributed back to the ER following the activation of p38 MAP kinase by growth factors [474].

Comparatively little is known about the expression of Sac2 and Sac3. The human Sac2 transcript is expressed ubiquitously but most highly in the brain, heart, skeletal muscle, kidney and placenta. Northern blotting and immunoblotting analyses have indicated that rat and human Sac3 are expressed almost ubiquitously.

T. Sasaki et al./Progress in Lipid Research 48 (2009) 307–343

### 7.10.3. Biological functions

#### 7.10.3.1. Sac1
Sac1-deficient mice have been generated using gene trap insertion of a targeting vector into Sac1 intron 1 [475]. Sac1-deficient embryos die prior to implantation, suggesting that Sac1 has an essential housekeeping function. However, Sac1 heterozygosity in either male and female mice does not affect fertility. RNAi-mediated ablation of Sac1 in HeLa cells results in disorganization of the TGN and decreased cell viability [475].

#### 7.10.3.2. Sac2
Trivedi et al. have reported that Sac2 is a negative regulator of the Akt pathway [476]. These researchers showed that histone deacetylase-2 (Hdac2) deficiency in mice attenuates cardiac hypertrophy, and that Sac2 is one of the genes upregulated in Hdac2-deficient hearts. Conversely, the Sac2 transcript is significantly downregulated in the heart of the Hdac2 transgenic mouse. Overexpression of Sac2 in cultured myocytes results in decreased phosphorylation of Akt, and siRNA knockdown of Sac2 enhances Akt phosphorylation [476]. It is therefore hypothesized that Sac2 modulates the Akt pathway downstream of Hdac2, possibly by dephosphorylating PtdIns(3,4,5)P₃.

#### 7.10.3.3. Sac3
Overexpression of rat Sac3 in differentiating PC12 neuronal cells promotes neurite outgrowth in a manner that depends on Sac3’s phosphoinositide phosphatase activity [473]. More extensive neurite outgrowth can be achieved by stimulating the Sac3-expressing PC12 cells with nerve growth factor (NGF). Conversely, the inhibition of Sac3 expression by the addition of an antisense oligonucleotide suppresses the neurite outgrowth of NGF-stimulated PC12 cells.

Additional insight into the function of Sac3 can be gained from studies of “pale tremor” mice. These natural mouse mutants have an insertion of early transposon 2β into Sac3 intron 18. Pale tremor mice exhibit severe tremor, abnormal gait, and diluted pigmentation [270]. Initially, the neurons in these mutants display enlarged cytoplasmic vacuoles. Subsequently, the neurons are lost in massive numbers in the thalamus, pons, medulla and deep cerebellar nuclei. In the skin, pigment-containing hair follicles are greatly reduced in number and clumped melanosomes are observed in pigmented hairs.

The ablation of Sac3 transcription in the brains of pale tremor mice has been confirmed by Northern blotting using a cDNA probe specific for Sac3 exons 8–15. Analysis of phosphoinositides in cultured fibroblasts derived from these mutants showed a 3-fold reduction in PtdIns(3,5)P₂ but no change in the level of any other phosphoinositide. This result is consistent with studies showing that Sac3 physically associates with and activates PIPKIII, the type III phosphatidylinositol phosphate kinase (see Section 4.4). Deletion of Sac3 therefore reduces, rather than increases, intracellular levels of PtdIns(3,5)P₂ despite Sac3’s intrinsic PtdIns(3,5)P₂ phosphatase activity. Pale tremor fibroblasts have enlarged vacuoles that are derived from late endosomes and resemble the vacuoles that appear in various cell lines overexpressing a dominant-negative form of PIPKIII [256].

### 7.10.4. Disease association
Chow et al. investigated Sac3 as a candidate gene for Charcot-Marie-Tooth (CMT) disorder in 95 individuals that had been diagnosed with this disease but lacked mutations in any known genes [270]. These researchers identified a protein truncation mutation within the Sac domain together with a missense mutation I41T in Sac3 exon 2 in four unrelated patients [270]. When a yeast Fig4p protein containing the corresponding mutation (I59T) was expressed in the Fig4-deficient yeast strain ΔFig4, lower levels of PtdIns(3,5)P₂ were detected under hyperosmotic conditions than when WT Fig4p was expressed in ΔFig4 yeast. These data suggest that the activation of yeast Fab1 kinase, which corresponds to human PIPKIII, was impaired. Thus, in humans, the I41T allele may cause a partial loss of Sac3 function that is pathogenic in combination with a Sac3 null allele. These data demonstrate that mutation of Sac3 is responsible for the peripheral neuropathy seen in CMT disorder patients. It has been proposed that this subtype of CMT disorder be designated “CMT4J” to indicate its recessive inheritance pattern.

---

### 8. Perspectives

Over the past decade, there has been enormous progress in our understanding of the physiological significance of phosphoinositide metabolism to intracellular signal transduction and cellular behavior. Phosphoinositides have turned up repeatedly in numerous studies in a broad range of disciplines, placing these molecules at the front lines of biological and medical research. Mutations of phosphoinositide-metabolizing enzymes such as PTEN, p110α, OCRL1 and various myotubularins have already been found in patients with cancers, myopathies and neurological disorders. Future studies will no doubt link mutations and polymorphisms in genes encoding other elements of phosphoinositide metabolism to more human diseases. In addition, it is becoming clear that some pathogenic bacteria highjack and utilize the phosphoinositide metabolism of the host to advance infection. Ongoing studies using mouse reverse genetics should continue to reveal the critical roles of phosphoinositide-metabolizing enzymes in fundamental normal physiology, host defense, and the development of pathology.

Animal studies have taught us that dysfunction in the control of phosphoinositide metabolism can result in widely diverse defects, making it more important than ever that we discover how as few as seven phosphoinositide isomers can control dozens of cellular processes. An extensive body of work now exists indicating that a single phosphoinositide species can play different roles in controlling different cellular proteins, depending on the phosphoinositide’s subcellular localization. Furthermore, biochemical evidence suggests that coincident interactions, in addition to the recognition of a particular phosphoinositide, are required to restrict the localization of a given phosphoinositide-binding protein and direct its function towards a specific membrane compartment. These interactions may constitute a mechanism to ensure that phosphoinositide effectors that bind to the same lipid, but which mediate different processes, are separately regulated. Conversely, a cell may express multiple isozymes of overlapping substrate specificities, but each isozyme appears to be responsible for carrying out the required reaction in a spatially discrete way. The substantiation of these notions will require the simultaneous analysis of the localization and activities of enzymes and lipid substrates and products, which in turn will require the continued development of novel probes for detecting proteins and lipids without altering the molecular geography of the membrane of interest. The use of a panel of genetically-modified mouse strains, each lacking a particular phosphoinositide-metabolizing isoform, would greatly increase our understanding of the specific and redundant functions of equivalent isozymes in cellular regulation.

It may not be enough to examine only the phosphorylation aspects of phosphoinositide metabolism. Studies in *S. cerevisiae* have shown that the fatty acid composition of phospholipids varies among the subcellular organelles of these organisms. In mammals, it has long been known that the fatty acyl residues in the majority of phosphoinositides are stearic acid at the sn-1 position and arachidonic acid at the sn-2 position [477,478]. However, progress in mass spectrometry techniques coupled with reverse phase liquid chromatography has allowed the detection in mammalian cells of at least 17 different phosphatidylinositol mono- and bis-phosphate molecules that vary in their fatty acyl residues.

[479, 480]. Notably, the acyltransferases catalyzing the incorporation of fatty acids into lysophospholipids have recently been identified as a family of phospholipid remodeling enzymes [481–483]. Considering that the diversity in fatty acyl residues of a phosphoinositide may influence its metabolism by the phosphoinositide-metabolizing enzymes, as well as the affinity of partner proteins binding to it, future studies should focus not only on the phosphorylation status of phosphoinositides but also on their fatty acid structure in the sn-1, 2 positions.

The phosphoinositides are regulators whose multifaceted physiology we are only just beginning to explore. Knowledge of their functions in specific contexts and how those functions are controlled will be essential to manipulating them to benefit the human condition in the future.

## Acknowledgements

This work was supported by grants-in-aid from Japan Science and Technology Corporation (JST); grants-in-aid and Global Center of Excellence Program from the Japanese Ministry of Education, Culture, Sports, Technology (MEXT); grant-in-Aid for Creative Scientific Research from Japan Society for the Promotion of Science (JSPS); Naito Foundation (to J.S.); and Toray Science Foundation (to T.S.).

## References

1. Vanhaesebroeck B, Leevers SJ, Ahmadi K, Timms J, Katso R, Driscoll PC, et al. Synthesis and function of 3-phosphorylated inositol lipids. Ann Rev Biochem 2001;70:535–602.
2. Cantley LC. The phosphoinositide 3-kinase pathway. Science 2002;296: 1655–7.
3. Itoh T, Takenawa T. Phosphoinositide-binding domains: functional units for temporal and spatial regulation of intracellular signalling. Cell Signal 2002;14:733–43.
4. Carlton JG, Cullen PJ. Coincidence detection in phosphoinositide signaling. Trends Cell Biol 2005;15:540–7.
5. Hawkins PT, Anderson KE, Davidson K, Stephens LR. Signalling through Class I PI3Ks in mammalian cells. Biochem Soc Trans 2006;34:647–62.
6. Di Paolo G, De Camilli P. Phosphoinositides in cell regulation and membrane dynamics. Nature 2006;443:651–7.
7. Sasaki T, Sasaki J, Sakai T, Takasuga S, Suzuki A. The physiology of phosphoinositides. Biol Pharm Bull 2007;30:1599–604.
8. Vicinanza M, D'Angelo G, Di Campli A, De Matteis MA. Function and dysfunction of the PI system in membrane trafficking. EMBO J 2008;27: 2457–70.
9. Wurmser AE, Gary JD, Emr SD. Phosphoinositide 3-kinases and their FYVE domain-containing effectors as regulators of vacuolar/lysosomal membrane trafficking pathways. J Biol Chem 1999;274:9129–32.
10. Suh PG, Park JI, Manzoli L, Cocco L, Peak JC, Katan M, et al. Multiple roles of phosphoinositide-specific phospholipase C isozymes. BMB Rep 2008;41: 415–34.
11. Balla T. Inositol-lipid binding motifs: signal integrators through protein-lipid and protein-protein interactions. J Cell Sci 2005;118:2093–104.
12. Roth MG. Phosphoinositides in constitutive membrane traffic. Physiol Rev 2004;84:699–730.
13. Domin J, Waterfield MD. Using structure to define the function of phosphoinositide 3-kinase family members. FEBS Lett 1997;410:91–5.
14. Deane JA, Fruman DA. Phosphoinositide 3-kinase: diverse roles in immune cell activation. Annu Rev Immunol 2004;22:563–98.
15. Backer JM, Myers Jr MG, Shoelson SE, Chin DJ, Sun XJ, Miralpeix M, et al. Phosphatidylinositol 3′-kinase is activated by association with IRS-1 during insulin stimulation. EMBO J 1992;11:3469–79.
16. Stephens LR, Eguinoa A, Erdjument-Bromage H, Lui M, Cooke F, Coadwell J, et al. The Gβγ sensitivity of a PI3K is dependent upon a tightly associated adaptor, p101. Cell 1997;89:105–14.
17. Suire S, Coadwell J, Ferguson GJ, Davidson K, Hawkins P, Stephens L. P84, a new Gβγ-activated regulatory subunit of the type IB phosphoinositide 3-kinase p110γ. Curr Biol 2005;15:566–70.
18. Voigt P, Brock C, Nurnberg B, Schaefer M. Assigning functional domains within the p101 regulatory subunit of phosphoinositide 3-kinase γ. J Biol Chem 2005;280:5121–7.
19. Rodriguez-Viciana P, Warne PH, Khwaja A, Marte BM, Pappin D, Das P, et al. Role of phosphoinositide 3-OH kinase in cell transformation and control of the actin cytoskeleton by Ras. Cell 1997;89:457–67.
20. Pacold ME, Suire S, Perisic O, Lara-Gonzalez S, Davis CT, Walker EH, et al. Crystal structure and functional analysis of Ras binding to its effector phosphoinositide 3-kinase γ. Cell 2000;103:931–43.
21. Walker EH, Perisic O, Ried C, Stephens L, Williams RL. Structural insights into phosphoinositide 3-kinase catalysis and signalling. Nature 1999;402: 313–20.
22. Sasaki T, Irie-Sasaki J, Jones RG, Oliveira-dos-Santos AJ, Stanford WL, Bolon B, et al. Function of PI3Kγ in thymocyte development, T cell activation, and neutrophil migration. Science 2000;287:1040–6.
23. Li Z, Jiang H, Xie W, Zhang Z, Smrcka AV, Wu D. Roles of PLC-β2 and -β3 and PI3Kγ in chemoattractant-mediated signal transduction. Science 2000;287: 1046–9.
24. Hirsch E, Katanaev VL, Garlanda C, Azzolino O, Pirola L, Silengo L, et al. Central role for G protein-coupled phosphoinositide 3-kinase γ in inflammation. Science 2000;287:1049–53.
25. Jia S, Liu Z, Zhang S, Liu P, Zhang L, Lee SH, et al. Essential roles of PI(3)K-p110β in cell growth, metabolism and tumorigenesis. Nature 2008;454:776–9.
26. Guillermet-Guibert J, Bjorklof K, Salpekar A, Gonella C, Ramadani F, Bilancio A, et al. The p110β isoform of phosphoinositide 3-kinase signals downstream of G protein-coupled receptors and is functionally redundant with p110γ. Proc Natl Acad Sci USA 2008;105:8292–7.
27. Ciraolo E, Iezzi M, Marone R, Marengo S, Curcio C, Costa C, et al. Phosphoinositide 3-kinase p110β activity: key role in metabolism and mammary gland cancer but not development. Sci Signal 2008;1:ra3.
28. Bi L, Okabe I, Bernard DJ, Wynshaw-Boris A, Nussbaum RL. Proliferative defect and embryonic lethality in mice homozygous for a deletion in the p110α subunit of phosphoinositide 3-kinase. J Biol Chem 1999;274:10963–8.
29. Bi L, Okabe I, Bernard DJ, Nussbaum RL. Early embryonic lethality in mice deficient in the p110β catalytic subunit of PI 3-kinase. Mamm Genome 2002;13:169–72.
30. Okada T, Kawano Y, Sakakibara T, Hazeki O, Ui M. Essential role of phosphatidylinositol 3-kinase in insulin-induced glucose transport and antilipolysis in rat adipocytes. Studies with a selective inhibitor wortmannin. J Biol Chem 1994;269:3568–73.
31. Kotani K, Carozzi AJ, Sakaue H, Hara K, Robinson LJ, Clark SF, et al. Requirement for phosphoinositide 3-kinase in insulin-stimulated GLUT4 translocation in 3T3-L1 adipocytes. Biochem Biophys Res Commun 1995;209:343–8.
32. Barroso I, Luan J, Middelberg RP, Harding AH, Franks PW, Jakes RW, et al. Candidate gene association study in type 2 diabetes indicates a role for genes involved in β-cell function as well as insulin action. PLoS Biol 2003;1:E20.
33. Foukas LC, Claret M, Pearce W, Okkenhaug K, Meek S, Peskett E, et al. Critical role for the p110α phosphoinositide-3-OH kinase in growth and metabolic regulation. Nature 2006;441:366–70.
34. Knight ZA, Gonzalez B, Feldman ME, Zunder ER, Goldenberg DD, Williams O, et al. A pharmacological map of the PI3-K family defines a role for p110α in insulin signaling. Cell 2006;125:733–47.
35. Brachmann SM, Ueki K, Engelman JA, Kahn RC, Cantley LC. Phosphoinositide 3-kinase catalytic subunit deletion and regulatory subunit deletion have opposite effects on insulin sensitivity in mice. Mol Cell Biol 2005;25: 1596–607.
36. Graupera M, Guillermet-Guibert J, Foukas LC, Phng LK, Cain RJ, Salpekar A, et al. Angiogenesis selectively requires the p110α isoform of PI3K to control endothelial cell migration. Nature 2008;453:662–6.
37. Clayton E, Bardi G, Bell SE, Chantry D, Downes CP, Gray A, et al. A crucial role for the p110δ subunit of phosphatidylinositol 3-kinase in B cell development and activation. J Exp Med 2002;196:753–63.
38. Patrucco E, Notte A, Barberis L, Selvetella G, Maffei A, Brancaccio M, et al. PI3Kγ modulates the cardiac response to chronic pressure overload by distinct kinase-dependent and -independent effects. Cell 2004;118: 375–87.
39. Okkenhaug K, Bilancio A, Farjot G, Priddle H, Sancho S, Peskett E, et al. Impaired B and T cell antigen receptor signaling in p110δ PI 3-kinase mutant mice. Science 2002;297:1031–4.
40. Ito N, Yokomizo T, Sasaki T, Kurosu H, Penninger J, Kanaho Y, et al. Requirement of phosphatidylinositol 3-kinase activation and calcium influx for leukotriene B4-induced enzyme release. J Biol Chem 2002;277: 44898–904.
41. Andrews S, Stephens LR, Hawkins PT. PI3K class IB pathway in neutrophils. Sci STKE 2007;2007:cm3.
42. Sadhu C, Masinovsky B, Dick K, Sowell CG, Staunton DE. Essential role of phosphoinositide 3-kinase δ in neutrophil directional movement. J Immunol 2003;170:2647–54.
43. Ali K, Bilancio A, Thomas M, Pearce W, Gilfillan AM, Tkaczyk C, et al. Essential role for the p110δ phosphoinositide 3-kinase in the allergic response. Nature 2004;431:1007–11.
44. Laffargue M, Calvez R, Finan P, Trifilieff A, Barbier M, Altruda F, et al. Phosphoinositide 3-kinase γ is an essential amplifier of mast cell function. Immunity 2002;16:441–51.
45. Garcon F, Patton DT, Emery JL, Hirsch E, Rottapel R, Sasaki T, et al. CD28 provides T-cell costimulation and enhances PI3K activity at the immune synapse independently of its capacity to interact with the p85/p110 heterodimer. Blood 2008;111:1464–71.
46. Alcazar I, Marques M, Kumar A, Hirsch E, Wymann M, Carrera AC, et al. Phosphoinositide 3-kinase γ participates in T cell receptor-induced T cell activation. J Exp Med 2007;204:2977–87.
47. Swat W, Montgrain V, Doggett TA, Douangpanya J, Puri K, Vermi W, et al. Essential role of PI3Kδ and PI3Kγ in thymocyte survival. Blood 2006;107: 2415–22.

[48] Ji H, Rintelen F, Waltzinger C, Bertschy Meier D, Bilancio A, Pearce W, et al. Inactivation of PI3Kγ and PI3Kδ distorts T-cell development and causes multiple organ inflammation. Blood 2007;110:2940–7.

[49] Crackower MA, Oudit GY, Kozieradzki I, Sarao R, Sun H, Sasaki T, et al. Regulation of myocardial contractility and cell size by distinct PI3K-PTEN signaling pathways. Cell 2002;110:737–49.

[50] Puri KD, Doggett TA, Douangpanya J, Hou Y, Tino WT, Wilson T, et al. Mechanisms and implications of phosphoinositide 3-kinase δ in promoting neutrophil trafficking into inflamed tissue. Blood 2004;103:3448–56.

[51] Puri KD, Doggett TA, Huang CY, Douangpanya J, Hayflick JS, Turner M, et al. The role of endothelial PI3Kγ activity in neutrophil trafficking. Blood 2005;106:150–7.

[52] Kuwana H, Terada Y, Kobayashi T, Okado T, Penninger JM, Irie-Sasaki J, et al. The phosphoinositide-3 kinase γ-Akt pathway mediates renal tubular injury in cisplatin nephrotoxicity. Kidney Int 2008;73:430–45.

[53] Whitman M, Downes CP, Keeler M, Keller T, Cantley L, Type I. Phosphatidylinositol kinase makes a novel inositol phospholipid, phosphatidylinositol-3-phosphate. Nature 1988;332:644–6.

[54] Cantley LC, Auger KR, Carpenter C, Duckworth B, Graziani A, Kapeller R, et al. Oncogenes and signal transduction. Cell 1991;64:281–302.

[55] Samuels Y, Wang Z, Bardelli A, Silliman N, Ptak J, Szabo S, et al. High frequency of mutations of the PIK3CA gene in human cancers. Science 2004;304:554.

[56] Vogt PK, Kang S, Elsiger MA, Gymnopoulos M. Cancer-specific mutations in phosphatidylinositol 3-kinase. Trends Biochem Sci 2007;32:342–9.

[57] Gymnopoulos M, Elsiger MA, Vogt PK. Rare cancer-specific mutations in PIK3CA show gain of function. Proc Natl Acad Sci USA 2007;104:5569–74.

[58] Saal LH, Holm K, Maurer M, Memeo L, Su T, Wang X, et al. PIK3CA mutations correlate with hormone receptors, node metastasis, and ERBB2, and are mutually exclusive with PTEN loss in human breast carcinoma. Cancer Res 2005;65:2554–9.

[59] Fan QW, Knight ZA, Goldenberg DD, Yu W, Mostov KE, Stokoe D, et al. A dual PI3 kinase/mTOR inhibitor reveals emergent efficacy in glioma. Cancer Cell 2006;9:341–9.

[60] Wee S, Wiederschain D, Maira SM, Loo A, Miller C, deBeaumont R, et al. PTEN-deficient cancers depend on PIK3CB. Proc Natl Acad Sci U S A 2008;105:13057–62.

[61] Sasaki T, Suzuki A, Sasaki J, Penninger JM. Phosphoinositide 3-kinases in immunity: lessons from knockout mice. J Biochem 2002;131:495–501.

[62] Wymann MP, Bjorklof K, Calvez R, Finan P, Thomast M, Trifilieff A, et al. Phosphoinositide 3-kinase γ: a key modulator in inflammation and allergy. Biochem Soc Trans 2003;31:275–80.

[63] Hawkins PT, Stephens LR. PI3Kγ is a key regulator of inflammatory responses and cardiovascular homeostasis. Science 2007;318:64–6.

[64] Chang JD, Sukhova GK, Libby P, Schwartz E, Lichtenstein AH, Field SJ, et al. Deletion of the phosphoinositide 3-kinase p110γ gene attenuates murine atherosclerosis. Proc Natl Acad Sci USA 2007;104:8077–82.

[65] Doukas J, Wrasidlo W, Noronha G, Dneprovskaja E, Fine R, Weis S, et al. Phosphoinositide 3-kinase γ/δ inhibition limits infarct size after myocardial ischemia/reperfusion injury. Proc Natl Acad Sci USA 2006;103:19866–71.

[66] Camps M, Ruckle T, Ji H, Ardisson V, Rintelen F, Shaw J, et al. Blockade of PI3Kγ suppresses joint inflammation and damage in mouse models of rheumatoid arthritis. Nat Med 2005;11:936–43.

[67] Barber DF, Bartolome A, Hernandez C, Flores JM, Redondo C, Fernandez-Arias C, et al. PI3Kγ inhibition blocks glomerulonephritis and extends lifespan in a mouse model of systemic lupus. Nat Med 2005;11:933–5.

[68] Takeda M, Ito W, Tanabe M, Ueki S, Kato H, Kihara J, et al. Allergic airway hyperresponsiveness, inflammation, and remodeling do not develop in phosphoinositide 3-kinase γ-deficient mice. J Allergy Clin Immunol 2009;123:805–12.

[69] Ruckle T, Schwarz MK, Rommel C. PI3Kγ inhibition: towards an ‘aspirin of the 21st century’? Nature reviews 2006;5:903–18.

[70] Marone R, Cmiljanovic V, Giese B, Wymann MP. Targeting phosphoinositide 3-kinase: moving towards therapy. Biochim Biophys Acta 2008;1784:159–85.

[71] Traer CJ, Foster FM, Abraham SM, Fry MJ. Are class II phosphoinositide 3-kinases potential targets for anticancer therapies? Bull Cancer 2006;93:E53–8.

[72] Domin J, Gaidarov I, Smith ME, Keen JH, Waterfield MD. The class II phosphoinositide 3-kinase PI3K-C2α is concentrated in the trans-Golgi network and present in clathrin-coated vesicles. J Biol Chem 2000;275:11943–50.

[73] Didichenko SA, Thelen M. Phosphatidylinositol 3-kinase c2α contains a nuclear localization sequence and associates with nuclear speckles. J Biol Chem 2001;276:48135–42.

[74] Brown RA, Ho LK, Weber-Hall SJ, Shipley JM, Fry MJ. Identification and cDNA cloning of a novel mammalian C2 domain-containing phosphoinositide 3-kinase, HsC2-PI3K. Biochem Biophys Res Commun 1997;233:537–44.

[75] Wheeler M, Domin J. The N-terminus of phosphoinositide 3-kinase-C2β regulates lipid kinase activity and binding to clathrin. J Cell Physiol 2006;206:586–93.

[76] Ho LK, Liu D, Rozyccka M, Brown RA, Fry MJ. Identification of four novel human phosphoinositide 3-kinases defines a multi-isoform subfamily. Biochem Biophys Res Commun 1997;235:130–7.

[77] Ono F, Nakagawa T, Saito S, Owada Y, Sakagami H, Goto K, et al. A novel class II phosphoinositide 3-kinase predominantly expressed in the liver and its enhanced expression during liver regeneration. J Biol Chem 1998;273:7731–6.

[78] Misawa H, Ohtsubo M, Copeland NG, Gilbert DJ, Jenkins NA, Yoshimura A. Cloning and characterization of a novel class II phosphoinositide 3-kinase containing C2 domain. Biochem Biophys Res Commun 1998;244:531–9.

[79] Domin J, Pages F, Volinia S, Rittenhouse SE, Zvelebil MJ, Stein RC, et al. Cloning of a human phosphoinositide 3-kinase with a C2 domain that displays reduced sensitivity to the inhibitor wortmannin. Biochem J 1997;326(Pt. 1):139–47.

[80] Gaidarov I, Smith ME, Domin J, Keen JH. The class II phosphoinositide 3-kinase C2α is activated by clathrin and regulates clathrin-mediated membrane trafficking. Mol Cell 2001;7:443–9.

[81] Gaidarov I, Zhao Y, Keen JH. Individual phosphoinositide 3-kinase C2α domain activities independently regulate clathrin function. J Biol Chem 2005;280:40766–72.

[82] Zhao Y, Gaidarov I, Keen JH. Phosphoinositide 3-kinase C2α links clathrin to microtubule-dependent movement. J Biol Chem 2007;282:1249–56.

[83] Meunier FA, Osborne SL, Hammond GR, Cooke FT, Parker PJ, Domin J, et al. Phosphatidylinositol 3-kinase C2α is essential for ATP-dependent priming of neurosecretory granule exocytosis. Mol Biol Cell 2005;16:4841–51.

[84] Kang S, Song J, Kang J, Kang H, Lee D, Lee Y, et al. Suppression of the α-isoform of class II phosphoinositide 3-kinase gene expression leads to apoptotic cell death. Biochem Biophys Res Commun 2005;329:6–10.

[85] Elis W, Triantafellow E, Wolters NM, Sian KR, Caponigro G, Borawski J, et al. Down-regulation of class II phosphoinositide 3-kinase α expression below a critical threshold induces apoptotic cell death. Mol Cancer Res 2008;6:614–23.

[86] Yoshioka K, Sugimoto N, Takuwa N, Takuwa Y. Essential role for class II phosphoinositide 3-kinase α-isoform in Ca²⁺-induced, Rho- and Rho kinase-dependent regulation of myosin phosphatase and contraction in isolated vascular smooth muscle cells. Mol Pharmacol 2007;71:912–20.

[87] Arcaro A, Zvelebil MJ, Wallasch C, Ullrich A, Waterfield MD, Domin J, et al. Phosphoinositide 3-kinases are downstream targets of activated polypeptide growth factor receptors. Mol Cell Biol 2000;20:3817–30.

[88] Brown RA, Domin J, Arcaro A, Waterfield MD, Shepherd PR. Insulin activates the α isoform of class II phosphoinositide 3-kinase. J Biol Chem 1999;274:14529–32.

[89] Turner SJ, Domin J, Waterfield MD, Ward SG, Westwick J, The CC. Chemokine monocyte chemotactic peptide-1 activates both the class I p85/p110 phosphatidylinositol 3-kinase and the class II PI3K-C2α. J Biol Chem 1998;273:25987–95.

[90] Ktori C, Shepherd PR, O'Rourke L. TNF-α and leptin activate the α-isoform of class II phosphoinositide 3-kinase. Biochem Biophys Res Commun 2003;306:139–43.

[91] Falasca M, Hughes WE, Dominguez V, Sala G, Fostira F, Fang MQ, et al. The role of phosphoinositide 3-kinase C2α in insulin signaling. J Biol Chem 2007;282:28226–36.

[92] Arcaro A, Volinia S, Zvelebil MJ, Stein R, Watton SJ, Layton MJ, et al. Human phosphoinositide 3-kinase C2β, the role of calcium and the C2 domain in enzyme activity. J Biol Chem 1998;273:33082–90.

[93] Maffucci T, Cooke FT, Foster FM, Traer CJ, Fry MJ, Falasca M, et al. Phosphoinositide 3-kinase defines a novel signaling pathway in cell migration. J Cell Biol 2005;169:789–99.

[94] Domin J, Harper L, Aubyn D, Wheeler M, Florey O, Haskard D, et al. The class II phosphoinositide 3-kinase PI3K-C2β regulates cell migration by a PtdIns3P dependent mechanism. J Cell Physiol 2005;205:452–62.

[95] Katso RM, Pardo OE, Palamidessi A, Franz CM, Marinov M, De Laurentiis A, et al. Phosphoinositide 3-Kinase C2β regulates cytoskeletal organization and cell migration via Rac-dependent mechanisms. Mol Biol Cell 2006;17:3729–44.

[96] Arcaro A, Khanzada UK, Vanhaesebroeck B, Tetley TD, Waterfield MD, Seckl MJ. Two distinct phosphoinositide 3-kinases mediate polypeptide growth factor-stimulated PKB activation. EMBO J 2002;21:5097–108.

[97] Zhang J, Banfic H, Straforini F, Tosi L, Volinia S, Rittenhouse SE. A type II phosphoinositide 3-kinase is stimulated via activated integrin in platelets. A source of phosphatidylinositol 3-phosphate. J Biol Chem 1998;273:14081–4.

[98] Das M, Scappini E, Martin NP, Wong KA, Dunn S, Chen YJ, et al. Regulation of neuron survival through an intersectin-phosphoinositide 3'-kinase C2β -AKT pathway. Mol Cell Biol 2007;27:7906–17.

[99] Harada K, Truong AB, Cai T, Khavari PA. The class II phosphoinositide 3-kinase C2β is not essential for epidermal differentiation. Mol Cell Biol 2005;25:11122–30.

[100] Herman PK, Emr SD. Characterization of VPS34, a gene required for vacuolar protein sorting and vacuole segregation in Saccharomyces cerevisiae. Mol Cell Biol 1990;10:6742–54.

[101] Budovskaya YV, Hama H, DeWald DB, Herman PK, The C. Terminus of the Vps34p phosphoinositide 3-kinase is necessary and sufficient for the interaction with the Vps15p protein kinase. J Biol Chem 2002;277:287–94.

[102] Volinia S, Dhand R, Vanhaesebroeck B, MacDougall LK, Stein R, Zvelebil MJ, et al. A human phosphatidylinositol 3-kinase complex related to the yeast Vps34p-Vps15p protein sorting system. EMBO J 1995;14:3339–48.

[103] Axe EL, Walker SA, Manifava M, Chandra P, Roderick HL, Habermann A, et al. Autophagosome formation from membrane compartments enriched in phosphatidylinositol 3-phosphate and dynamically connected to the endoplasmic reticulum. J Cell Biol 2008;182:685–701.

[104] Row PE, Reaves BJ, Domin J, Luzio JP, Davidson HW. Overexpression of a rat kinase-deficient phosphoinositide 3-kinase, Vps34p, inhibits cathepsin D maturation. Biochem J 2001;353:655–61.

[105] Tuma PL, Nyasae LK, Backer JM, Hubbard AL. Vps34p differentially regulates endocytosis from the apical and basolateral domains in polarized hepatic cells. J Cell Biol 2001;154:1197–208.

[106] Johnson EE, Overmeyer JH, Gunning WT, Maltese WA. Gene silencing reveals a specific function of hVps34 phosphatidylinositol 3-kinase in late versus early endosomes. J Cell Sci 2006;119:1219–32.

[107] Kihara A, Noda T, Ishihara N, Ohsumi Y. Two distinct Vps34 phosphatidylinositol 3-kinase complexes function in autophagy and carboxypeptidase Y sorting in Saccharomyces cerevisiae. J Cell Biol 2001;152:519–30.

[108] Juhasz G, Hill JH, Yan Y, Sass M, Baehrecke EH, Backer JM, et al. The class III PI(3)K Vps34 promotes autophagy and endocytosis but not TOR signaling in Drosophila. J Cell Biol 2008;181:655–66.

[109] Petiot A, Ogier-Denis E, Blommaart EF, Meijer AJ, Codogno P. Distinct classes of phosphatidylinositol 3'-kinases are involved in signaling pathways that control macroautophagy in HT-29 cells. J Biol Chem 2000;275:992–8.

[110] Kuo CC, Lin WT, Liang CM, Liang SM, Class III I. Phosphatidylinositol 3'-kinase play distinct roles in TLR signaling pathway. J Immunol 2006;176:5943–9.

[111] Vieira OV, Botelho RJ, Rameh L, Brachmann SM, Matsuo T, Davidson HW, et al. Distinct roles of class I and class III phosphatidylinositol 3-kinases in phagosome formation and maturation. J Cell Biol 2001;155:19–25.

[112] Fratti RA, Backer JM, Gruenberg J, Corvera S, Deretic V. Role of phosphatidylinositol 3-kinase and Rab5 effectors in phagosomal biogenesis and mycobacterial phagosome maturation arrest. J Cell Biol 2001;154:631–44.

[113] Ellson CD, Davidson K, Ferguson GJ, O'Connor R, Stephens LR, Hawkins PT. Neutrophils from p40phox−/− mice exhibit severe defects in NADPH oxidase regulation and oxidant-dependent bacterial killing. J Exp Med 2006;203:1927–37.

[114] Anderson KE, Boyle KB, Davidson K, Chessa TA, Kulkarni S, Jarvis GE, et al. CD18-dependent activation of the neutrophil NADPH oxidase during phagocytosis of Escherichia coli or Staphylococcus aureus is regulated by class III but not class I or II PI3Ks. Blood 2008;112:5202–11.

[115] Herman PK, Stack JH, DeModena JA, Emr SD. A novel protein kinase homolog essential for protein sorting to the yeast lysosome-like vacuole. Cell 1991;64:425–37.

[116] Panaretou C, Domin J, Cockcroft S, Waterfield MD. Characterization of p150, an adaptor protein for the human phosphatidylinositol (PtdIns) 3-kinase. Substrate presentation by phosphatidylinositol transfer protein to the p150.Ptdins 3-kinase complex. J Biol Chem 1997;272:2477–85.

[117] Backer JM. The regulation and function of Class III PI3Ks: novel roles for Vps34. Biochem J 2008;410:1–17.

[118] Liang XH, Kleeman LK, Jiang HH, Gordon G, Goldman JE, Berry G, et al. Protection against fatal Sindbis virus encephalitis by beclin, a novel Bcl-2-interacting protein. J Virol 1998;72:8586–96.

[119] Furuya N, Yu J, Byfield M, Pattingre S, Levine B. The evolutionarily conserved domain of Beclin 1 is required for Vps34 binding, autophagy and tumor suppressor function. Autophagy 2005;1:46–52.

[120] Suzuki K, Ohsumi Y. Molecular machinery of autophagosome formation in yeast, Saccharomyces cerevisiae. FEBS Lett 2007;581:2156–61.

[121] Yue Z, Jin S, Yang C, Levine AJ, Heintz N. Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor. Proc Natl Acad Sci USA 2003;100:15077–82.

[122] Fimia GM, Stoykova A, Romagnoli A, Giunta L, Di Bartolomeo S, Nardacci R, et al. Ambra1 regulates autophagy and development of the nervous system. Nature 2007;447:1121–5.

[123] Liang C, Feng P, Ku B, Dotan I, Canaani D, Oh BH, et al. Autophagic and tumour suppressor activity of a novel Beclin1-binding protein UVRAG. Nat Cell Biol 2006;8:688–99.

[124] Takahashi Y, Coppola D, Matsushita N, Cualing HD, Sun M, Sato Y, et al. Bif-1 interacts with Beclin 1 through UVRAG and regulates autophagy and tumorigenesis. Nat Cell Biol 2007;9:1142–51.

[125] Itoh T, De Camilli P. BAR, F-BAR (EFC) and ENTH/ANTH domains in the regulation of membrane-cytosol interfaces and membrane curvature. Biochim Biophys Acta 2006;1761:897–912.

[126] Byfield MP, Murray JT, Backer JM. HVps34 is a nutrient-regulated lipid kinase required for activation of p70 S6 kinase. J Biol Chem 2005;280:33076–82.

[127] Nobukuni T, Joaquín M, Roccio M, Dann SG, Kim SY, Gulati P, et al. Amino acids mediate mTOR/raptor signaling through activation of class 3 phosphatidylinositol 3OH-kinase. Proc Natl Acad Sci USA 2005;102:14238–43.

[128] Williams NM, Rees MI, Holmans P, Norton N, Cardno AG, Jones LA, et al. A two-stage genome scan for schizophrenia susceptibility genes in 196 affected sibling pairs. Hum Mol Genet 1999;8:1729–39.

[129] Stopkova P, Saito T, Papolos DF, Vevera J, Pacult I, Zukov I, et al. Identification of PIK3C3 promoter variant associated with bipolar disorder and schizophrenia. Biol Psychiatry 2004;55:981–8.

[130] Duan S, Gao R, Xing Q, Du J, Liu Z, Chen Q, et al. A family-based association study of schizophrenia with polymorphisms at three candidate genes. Neurosci Lett 2005;379:32–6.

[131] Endemann G, Dunn SN, Cantley LC. Bovine brain contains two types of phosphatidylinositol kinase. Biochemistry 1987;26:6845–52.

[132] Whitman M, Kaplan D, Roberts T, Cantley L. Evidence for two distinct phosphatidylinositol kinases in fibroblasts. Implications for cellular regulation. Biochem J 1987;247:165–74.

[133] Pike LJ. Phosphatidylinositol 4-kinases and the role of polyphosphoinositides in cellular regulation. Endocr Rev 1992;13:692–706.

[134] Balla A, Balla T. Phosphatidylinositol 4-kinases: old enzymes with emerging functions. Trends Cell Biol 2006;16:351–61.

[135] Balla T. Phosphatidylinositol 4-kinases. Biochim Biophys Acta 1998;1436:69–85.

[136] Barylko B, Gerber SH, Binns DD, Grichine N, Khvotchev M, Sudhof TC, et al. A novel family of phosphatidylinositol 4-kinases conserved from yeast to humans. J Biol Chem 2001;276:7705–8.

[137] Jung G, Wang J, Wlodarski P, Barylko B, Binns DD, Shu H, et al. Molecular determinants of activation and membrane targeting of phosphoinositol 4-kinase IIβ. Biochem J 2008;409:501–9.

[138] Balla A, Tuymetova G, Barshishat M, Geiszt M, Balla T. Characterization of type II phosphatidylinositol 4-kinase isoforms reveals association of the enzymes with endosomal vesicular compartments. J Biol Chem 2002;277:20041–50.

[139] Xu Z, Huang G, Kandror KV. Phosphatidylinositol 4-kinase type IIα is targeted specifically to cellugyrin-positive glucose transporter 4 vesicles. Mol Endocrinol 2006;20:2890–7.

[140] Minogue S, Anderson JS, Waugh MG, dos Santos M, Corless S, Cramer R, et al. Cloning of a human type II phosphatidylinositol 4-kinase reveals a novel lipid kinase family. J Biol Chem 2001;276:16635–40.

[141] Waugh MG, Minogue S, Anderson JS, Balinger A, Blumenkrantz D, Calnan DP, et al. Localization of a highly active pool of type II phosphatidylinositol 4-kinase in a p97/valosin-containing-protein-rich fraction of the endoplasmic reticulum. Biochem J 2003;373:57–63.

[142] Wang YJ, Wang J, Sun HQ, Martinez M, Sun YX, Macia E, et al. Phosphatidylinositol 4 phosphate regulates targeting of clathrin adaptor AP-1 complexes to the Golgi. Cell 2003;114:299–310.

[143] Wei YJ, Sun HQ, Yamamoto M, Wlodarski P, Kunii K, Martinez M, et al. Type II phosphatidylinositol 4-kinase β is a cytosolic and peripheral membrane protein that is recruited to the plasma membrane and activated by Rac-GTP. J Biol Chem 2002;277:46586–93.

[144] Minogue S, Waugh MG, De Matteis MA, Stephens DJ, Berditchevski F, Hsuan JJ. Phosphatidylinositol 4-kinase is required for endosomal trafficking and degradation of the EGF receptor. J Cell Sci 2006;119:571–81.

[145] Guo J, Wenk MR, Pellegrini L, Onofri F, Benfenati F, De Camilli P. Phosphatidylinositol 4-kinase type IIα is responsible for the phosphatidylinositol 4-kinase activity associated with synaptic vesicles. Proc Natl Acad Sci USA 2003;100:3995–4000.

[146] Srivastava R, Sinha RK, Subrahmanyam G, Type II. Phosphatidylinositol 4-kinase β associates with TCR-CD3 ζ chain in Jurkat cells. Mol Immunol 2006;43:454–63.

[147] Pizarro-Cerda J, Payrastre B, Wang YJ, Veiga E, Yin HL, Cossart P, et al. Phosphatidylinositol 4-kinases promote Listeria monocytogenes entry into target cells. Cell Microbiol 2007;9:2381–90.

[148] Heilmeyer Jr LM, Vereb Jr G, Kakuk A, Szivak I. Mammalian phosphatidylinositol 4-kinases. IUBMB Life 2003;55:59–65.

[149] de Graaf P, Zwart WT, van Dijken RA, Deneka M, Schulz TK, Geijesen N, et al. Phosphatidylinositol 4-kinaseβ is critical for functional association of rab11 with the Golgi complex. Mol Biol Cell 2004;15:2038–47.

[150] Zolyomi A, Zhao X, Downing GJ, Balla T. Localization of two distinct type III phosphatidylinositol 4-kinase enzyme mRNAs in the rat. Am J Physiol Cell Physiol 2000;278:C914–20.

[151] Balla A, Vereb G, Gulkan H, Gehrmann T, Gergely P, Heilmeyer Jr LM, et al. Immunohistochemical localisation of two phosphatidylinositol 4-kinase isoforms, PI4K230 and PI4K92, in the central nervous system of rats. Exp Brain Res 2000;134:279–88.

[152] Kakuk A, Friedlander E, Vereb Jr G, Kasa A, Balla A, Balla T, et al. Nuclear localization of phosphatidylinositol 4-kinase PI4K230 in various mammalian cells. Cytometry A 2006;69:1174–83.

[153] Weixel KM, Blumental-Perry A, Watkins SC, Aridor M, Weisz OA. Distinct Golgi populations of phosphatidylinositol 4-phosphate regulated by phosphatidylinositol 4-kinases. J Biol Chem 2005;280:10501–8.

[154] de Graaf P, Klapisz EE, Schulz TK, Cremers AF, Verkleij AJ, Van Bergen en Henegouwen PM. Nuclear localization of phosphatidylinositol 4-kinase β. J Cell Sci 2002;115:1769–75.

[155] Szivak I, Lamb N, Heilmeyer LM. Subcellular localization and structural function of endogenous phosphorylated phosphatidylinositol 4-kinase (PI4K92). J Biol Chem 2006;281:16740–9.

[156] Nakanishi S, Catt KJ, Balla T. A wortmannin-sensitive phosphatidylinositol 4-kinase that regulates hormone-sensitive pools of inositolphospholipids. Proc Natl Acad Sci USA 1995;92:5317–21.

[157] Balla A, Kim YJ, Varnai P, Szentpetery Z, Knight Z, Shokat KM, et al. Maintenance of hormone-sensitive phosphoinositide pools in the plasma membrane requires phosphatidylinositol 4-kinase IIIα. Mol Biol Cell 2008;19:711–21.

[158] Hausser A, Storz P, Martens S, Link G, Toker A, Pfizenmaier K. Protein kinase D regulates vesicular transport by phosphorylating and activating

phosphatidylinositol-4 kinase IIβ at the Golgi complex. Nat Cell Biol 2005;7:880–6.

[159] Godi A, Pertile P, Meyers R, Marra P, Di Tullio G, Iurisci C, et al. ARF mediates recruitment of PtdIns-4-OH kinase-β and stimulates synthesis of PtdIns(4, 5)P₂ on the Golgi complex. Nat Cell Biol 1999;1:280–7.

[160] Godi A, Di Campli A, Konstantakopoulos A, Di Tullio G, Alessi DR, Kular GS, et al. FAPPs control Golgi-to-cell-surface membrane traffic by binding to ARF and PtdIns(4)P. Nat Cell Biol 2004;6:393–404.

[161] Ishihara H, Shibasaki Y, Kizuki N, Katagiri H, Yazaki Y, Asano T, et al. Cloning of cDNAs encoding two isoforms of 68-kDa type I phosphatidylinositol-4-phosphate 5-kinase. J Biol Chem 1996;271:23611–4.

[162] Loijens JC, Anderson RA. Type I. Phosphatidylinositol-4-phosphate 5-kinases are distinct members of this novel lipid kinase family. J Biol Chem 1996;271:32937–43.

[163] Ishihara H, Shibasaki Y, Kizuki N, Wada T, Yazaki Y, Asano T, et al. Type I phosphatidylinositol-4-phosphate 5-kinases. Cloning of the third isoform and deletion/substitution analysis of members of this novel lipid kinase family. J Biol Chem 1998;273:8741–8.

[164] Rameh LE, Tolias KF, Duckworth BC, Cantley LC. A new pathway for synthesis of phosphatidylinositol-4, 5-bisphosphate. Nature 1997;390:192–6.

[165] Boronenkov IV, Anderson RA. The sequence of phosphatidylinositol-4-phosphate 5-kinase defines a novel family of lipid kinases. J Biol Chem 1995;270:2881–4.

[166] Divecha N, Truong O, Hsuan JJ, Hinchliffe KA, Irvine RF. The cloning and sequence of the C isoform of PtdIns4P 5-kinase. Biochem J 1995;309(Pt 3):715–9.

[167] Castellino AM, Parker GJ, Boronenkov IV, Anderson RA, Chao MV. A novel interaction between the juxtamembrane region of the p55 tumor necrosis factor receptor and phosphatidylinositol-4-phosphate 5-kinase. J Biol Chem 1997;272:5861–70.

[168] Itoh T, Ijuin T, Takenawa T. A novel phosphatidylinositol-5-phosphate 4-kinase (phosphatidylinositol-phosphate kinase IIγ) is phosphorylated in the endoplasmic reticulum in response to mitogenic signals. J Biol Chem 1998;273:20292–9.

[169] Sbrissa D, Ikonomov O, Shisheva A. PIKfyve, a mammalian ortholog of yeast Fab1p lipid kinase, synthesizes 5-phosphoinositides. Effect of insulin. J Biol Chem 1999;274:21589–97.

[170] Giudici ML, Emson PC, Irvine RF. A novel neuronal-specific splice variant of Type I phosphatidylinositol 4-phosphate 5-kinase isoform γ. Biochem J 2004;379:489–96.

[171] Rao VD, Misra S, Boronenkov IV, Anderson RA, Hurley JH. Structure of type IIβ phosphatidylinositol phosphate kinase: a protein kinase fold flattened for interfacial phosphorylation. Cell 1998;94:829–39.

[172] Kunz J, Wilson MP, Kisseleva M, Hurley JH, Majerus PW, Anderson RA. The activation loop of phosphatidylinositol phosphate kinases determines signaling specificity. Mol Cell 2000;5:1–11.

[173] Kunz J, Fuelling A, Kolbe L, Anderson RA. Stereo-specific substrate recognition by phosphatidylinositol phosphate kinases is swapped by changing a single amino acid residue. J Biol Chem 2002;277:5611–9.

[174] Wenk MR, Pellegrini L, Klenchin VA, Di Paolo G, Chang S, Daniell L, et al. PIP kinase Iγ is the major PI(4, 5)P₂ synthesizing enzyme at the synapse. Neuron 2001;32:79–88.

[175] Sasaki J, Sasaki T, Yamazaki M, Matsuoka K, Taya C, Shitara H, et al. Regulation of anaphylactic responses by phosphatidylinositol phosphate kinase type I {α}. J Exp Med 2005;201:859–70.

[176] Wang Y, Litvinov RI, Chen X, Bach TL, Lian L, Petrich BG, et al. Loss of PIP5KIγ, unlike other PIP5KI isoforms, impairs the integrity of the membrane cytoskeleton in murine megakaryocytes. J Clin Invest 2008;118:812–9.

[177] Wang Y, Chen X, Lian L, Tang T, Stalker TJ, Sasaki T, et al. Loss of PIP5KIβ demonstrates that PIP5KI isoform-specific PIP₂ synthesis is required for IP₃ formation. Proc Natl Acad Sci USA 2008;105:14064–9.

[178] Di Paolo G, Moskowitz HS, Gipson K, Wenk MR, Voronov S, Obayashi M, et al. Impaired PtdIns(4,5)P₂ synthesis in nerve terminals produces defects in synaptic vesicle trafficking. Nature 2004;431:415–22.

[179] Wang Y, Lian L, Golden JA, Morrisey EE, Abrams CS. PIP5KI γ is required for cardiovascular and neuronal development. Proc Natl Acad Sci USA 2007;104:11748–53.

[180] Pan W, Choi SC, Wang H, Qin Y, Volpicelli-Daley L, Swan L, et al. Wnt3a-mediated formation of phosphatidylinositol 4, 5-bisphosphate regulates LRP6 phosphorylation. Science 2008;321:1350–3.

[181] Gaidarov I, Keen JH. Phosphoinositide-AP-2 interactions required for targeting to plasma membrane clathrin-coated pits. J Cell Biol 1999;146:755–64.

[182] Martin TF. PI(4, 5)P(2) regulation of surface membrane traffic. Curr Opin Cell Biol 2001;13:493–9.

[183] Takenawa T, Itoh T. Phosphoinositides, key molecules for regulation of actin cytoskeletal organization and membrane traffic from the plasma membrane. Biochim. Biophys Acta 2001;1533:190–206.

[184] Padron D, Wang YJ, Yamamoto M, Yin H, Roth MG. Phosphatidylinositol phosphate 5-kinase Iβ recruits AP-2 to the plasma membrane and regulates rates of constitutive endocytosis. J Cell Biol 2003;162:693–701.

[185] Davis JN, Rock CO, Cheng M, Watson JB, Ashmun RA, Kirk H, et al. Complementation of growth factor receptor-dependent mitogenic signaling by a truncated type I phosphatidylinositol 4-phosphate 5-kinase. Mol Cell Biol 1997;17:7398–406.


[186] Barbieri MA, Heath CM, Peters EM, Wells A, Davis JN, Stahl PD. Phosphatidylinositol-4-phosphate 5-kinase-1β is essential for epidermal growth factor receptor-mediated endocytosis. J Biol Chem 2001;276:47212–6.

[187] Bairstow SF, Ling K, Su X, Firestone AJ, Carbonara C, Anderson RA. Type Iγ/661 phosphatidylinositol phosphate kinase directly interacts with AP2 and regulates endocytosis. J Biol Chem 2006;281:20632–42.

[188] Ling K, Bairstow SF, Carbonara C, Turbin DA, Huntsman DG, Anderson RA. Type Iγ phosphatidylinositol phosphate kinase modulates adherens junction and E-cadherin trafficking via a direct interaction with mu 1B adaptin. J Cell Biol 2007;176:343–53.

[189] Krauss M, Kukhtina V, Pechstein A, Haucke V. Stimulation of phosphatidylinositol kinase type I-mediated phosphatidylinositol (4, 5)-bisphosphate synthesis by AP-2μ-cargo complexes. Proc Natl Acad Sci USA 2006;103:11934–9.

[190] Nakano-Kobayashi A, Yamazaki M, Unoki T, Hongu T, Murata C, Taguchi R, et al. Role of activation of PIP5Kγ661 by AP-2 complex in synaptic vesicle endocytosis. EMBO J 2007;26:1105–16.

[191] Shibasaki Y, Ishihara H, Kizuki N, Asano T, Oka Y, Yazaki Y. Massive actin polymerization induced by phosphatidylinositol-4-phosphate 5-kinase in vivo. J Biol Chem 1997;272:7578–81.

[192] Honda A, Nogami M, Yokozeki T, Yamazaki M, Nakamura H, Watanabe H, et al. Phosphatidylinositol 4-phosphate 5-kinase α is a downstream effector of the small G protein ARF6 in membrane ruffle formation. Cell 1999;99:521–32.

[193] Yamamoto M, Hilgemann DH, Feng S, Bito H, Ishihara H, Shibasaki Y, et al. Phosphatidylinositol 4,5-bisphosphate induces actin stress-fiber formation and inhibits membrane ruffling in CV1 cells. J Cell Biol 2001;152:867–76.

[194] van Horck FP, Lavazais E, Eickholt BJ, Moolenaar WH, Divecha N. Essential role of type I(α) phosphatidylinositol 4-phosphate 5-kinase in neurite remodeling. Curr Biol 2002;12:241–5.

[195] Doughman RL, Firestone AJ, Wojtasik ML, Bunce MW, Anderson RA. Membrane ruffling requires coordination between type Iα phosphatidylinositol phosphate kinase and Rac signaling. J Biol Chem 2003;278:23036–45.

[196] Emoto K, Inadome H, Kanaho Y, Narumiya S, Umeda M. Local change in phospholipid composition at the cleavage furrow is essential for completion of cytokinesis. J Biol Chem 2005;280:37901–7.

[197] Di Paolo G, Pellegrini L, Letinic K, Cestra G, Zoncu R, Voronov S, et al. Recruitment and regulation of phosphatidylinositol phosphate kinase type Iγ by the FERM domain of talin. Nature 2002;420:85–9.

[198] Ling K, Doughman RL, Firestone AJ, Bunce MW, Anderson RA. Type I. γ phosphatidylinositol phosphate kinase targets and regulates focal adhesions. Nature 2002;420:89–93.

[199] Ling K, Doughman RL, Iyer VV, Firestone AJ, Bairstow SF, Mosher DF, et al. Tyrosine phosphorylation of type Iγ phosphatidylinositol phosphate kinase by Src regulates an integrin-talin switch. J Cell Biol 2003;163:1339–49.

[200] Lee SY, Voronov S, Letinic K, Nairn AC, Di Paolo G, De Camilli P. Regulation of the interaction between PIPKI γ and talin by proline-directed protein kinases. J Cell Biol 2005;168:789–99.

[201] Bairstow SF, Ling K, Anderson RA. Phosphatidylinositol phosphate kinase type Iγ directly associates with and regulates Shp-1 tyrosine phosphatase. J Biol Chem 2005;280:23884–91.

[202] Lacalle RA, Peregil RM, Albar JP, Merino E, Martinez AC, Merida I, et al. Type I phosphatidylinositol 4-phosphate 5-kinase controls neutrophil polarity and directional movement. J Cell Biol 2007;179:1539–53.

[203] Lokuta MA, Senetar MA, Bennin DA, Nuzzi PA, Chan KT, Ott VL, et al. Type Iγ PIP kinase is a novel uropod component that regulates rear retraction during neutrophil chemotaxis. Mol Biol Cell 2007;18:5069–80.

[204] Sun Y, Ling K, Wagoner MP, Anderson RA, Type I. γ phosphatidylinositol phosphate kinase is required for EGF-stimulated directional cell migration. J Cell Biol 2007;178:297–308.

[205] Koreh K, Monaco ME. The relationship of hormone-sensitive and hormone-insensitive phosphatidylinositol to phosphatidylinositol 4,5-bisphosphate in the WRK-1 cell. J Biol Chem 1986;261:88–91.

[206] Wang YJ, Li WH, Wang J, Xu K, Dong P, Luo X, et al. Critical role of PIP5KI{γ}87 in InsP₃-mediated Ca(2+) signaling. J Cell Biol 2004;167:1005–10.

[207] Boronenkov IV, Loijens JC, Umeda M, Anderson RA. Phosphoinositide signaling pathways in nuclei are associated with nuclear speckles containing pre-mRNA processing factors. Mol Biol Cell 1998;9:3547–60.

[208] Mellman DL, Gonzales ML, Song C, Barlow CA, Wang P, Kendziorski C, et al. A PtdIns4,5P₂-regulated nuclear poly(A) polymerase controls expression of select mRNAs. Nature 2008;451:1013–7.

[209] Moritz A, De Graan PN, Gispen WH, Wirtz KW. Phosphatidic acid is a specific activator of phosphatidylinositol-4-phosphate kinase. J Biol Chem 1992;267:7207–10.

[210] Jenkins GH, Fisette PL, Anderson RA, Type I. Phosphatidylinositol 4-phosphate 5-kinase isoforms are specifically stimulated by phosphatidic acid. J Biol Chem 1994;269:11547–54.

[211] Tolias KF, Couvillon AD, Cantley LC, Carpenter CL. Characterization of a Rac1- and RhoGDI-associated lipid kinase signaling complex. Mol Cell Biol 1998;18:762–70.

[212] Divecha N, Roefs M, Halstead JR, D'Andrea S, Fernandez-Borga M, Oomen L, et al. Interaction of the type Iα PIKinase with phospholipase D: a role for the

local generation of phosphatidylinositol 4, 5-bisphosphate in the regulation
of PLD2 activity. EMBO J 2000;19:5440–9.
[213] Urumow T, Wieland OH. Stimulation of phosphatidylinositol 4-phosphate
phosphorylation in human placenta membranes by GTP γ S. FEBS Lett
1986;207:253–7.
[214] Smith CD, Chang KJ. Regulation of brain phosphatidylinositol-4-phosphate
kinase by GTP analogues. A potential role for guanine nucleotide regulatory
proteins. J Biol Chem 1989;264:3206–10.
[215] Urumow T, Wieland OH. Purification and partial characterization of
phosphatidylinositol-4-phosphate kinase from rat liver plasma membranes.
Further evidence for a stimulatory G-protein. Biochim Biophys Acta
1990;1052:152–8.
[216] Chong LD, Traynor-Kaplan A, Bokoch GM, Schwartz MA. The small GTP-
binding protein Rho regulates a phosphatidylinositol 4-phosphate 5-kinase
in mammalian cells. Cell 1994;79:507–13.
[217] Hartwig JH, Bokoch GM, Carpenter CL, Janmey PA, Taylor LA, Toker A, et al.
Thrombin receptor ligation and activated Rac uncap actin filament barbed
ends through phosphoinositide synthesis in permeabilized human platelets.
Cell 1995;82:643–53.
[218] Oude Weernink PA, Schulte P, Guo Y, Wetzel J, Amano M, Kaibuchi K, et al.
Stimulation of phosphatidylinositol-4-phosphate 5-kinase by Rho-kinase. J
Biol Chem 2000;275:10168–74.
[219] Yamazaki M, Miyazaki H, Watanabe H, Sasaki T, Maehama T, Frohman MA, et al.
Phosphatidylinositol 4-phosphate 5-kinase is essential for ROCK-
mediated neurite remodeling. J Biol Chem 2002;277:17226–30.
[220] Yang SA, Carpenter CL, Abrams CS. Rho and Rho-kinase mediate thrombin-
induced phosphatidylinositol 4-phosphate 5-kinase trafficking in platelets. J
Biol Chem 2004;279:42331–6.
[221] Jones DH, Morris JB, Morgan CP, Kondo H, Irvine RF, Cockcroft S, et al.
Phosphatidylinositol 4-phosphate 5-kinase directly interacts with ADP-
ribosylation factor 1 and is responsible for phosphatidylinositol 4, 5-
bisphosphate synthesis in the golgi compartment. J Biol Chem
2000;275:13962–6.
[222] Skippen A, Jones DH, Morgan CP, Li M, Cockcroft S. Mechanism of ADP-
ribosylation factor-stimulated phosphatidylinositol 4, 5-bisphosphate
synthesis in HL60 cells. J Biol Chem 2002;277:5823–31.
[223] Saito K, Tolias KF, Saci A, Koon HB, Humphries LA, Scharenberg A, et al. BTK
regulates PtdIns-4,5-P₂ synthesis: importance for calcium signaling and PI3K
activity. Immunity 2003;19:669–78.
[224] Kisseleva M, Feng Y, Ward M, Song C, Anderson RA, Longmore GD, et al.
Protein Ajuba regulates phosphatidylinositol 4, 5-bisphosphate levels in
migrating cells through an interaction with and activation of PIPKI α. Mol Cell
Biol 2005;25:3956–66.
[225] Nelson CD, Kovacs JJ, Nobles KN, Whalen EJ, Lefkowitz RJ. β-arrestin
scaffolding of phosphatidylinositol 4-phosphate 5-kinase Iα promotes
agonist-stimulated sequestration of the β2-adrenergic receptor. J Biol Chem
2008;283:21093–210101.
[226] Vuopala K, Herva R. Lethal congenital contracture syndrome: further
delineation and genetic aspects. J Med Genet 1994;31:521–7.
[227] Makela-Bengs P, Jarvinen N, Vuopala K, Suomalainen A, Ignatius J, Sipila M, et al. Assignment of the disease locus for lethal congenital
contracture syndrome to a restricted region of chromosome 9q34, by
genome scan using five affected individuals. Am J Hum Genet 1998;63:
506–16.
[228] Narkis G, Landau D, Manor E, Elbedour K, Tzemach A, Fishelson M, et al. Homozygosity mapping of lethal congenital contractual syndrome type 2
(LCCS2) to a 6 cM interval on chromosome 12q13. Am J Med Genet A
2004;130A:272–6.
[229] Narkis G, Ofir R, Landau D, Manor E, Volokita M, Hershkowitz R, et al. Lethal
contractural syndrome type 3 (LCCS3) is caused by a mutation in PIP5K1C,
which encodes PIPKI γ of the phophatidylinositol pathway. Am J Hum Genet
2007;81:530–9.
[230] Clarke JH, Emson PC, Irvine RF. Localization of phosphatidylinositol
phosphate kinase IIγ in kidney to a membrane trafficking compartment
within specialized cells of the nephron. Am J Physiol Renal Physiol
2008;295:F1422–1430.
[231] Ciruela A, Hinchliffe KA, Divecha N, Irvine RF. Nuclear targeting of the β
isoform of type II phosphatidylinositol
phosphate kinase (phosphatidylinositol 5-phosphate 4-kinase) by its α-helix 7. Biochem J
2000;346(Pt 3):587–91.
[232] Clarke JH, Richardson JP, Hinchliffe KA, Irvine RF. Type II PtdInsP kinases: location,
regulation and function. Biochem Soc Symp 2007:149–59.
[233] Richardson JP, Wang M, Clarke JH, Patel KJ, Irvine RF. Genomic
tagging of endogenous type IIβ phosphatidylinositol 5-phosphate 4-
kinase in DT40 cells reveals a nuclear localisation. Cell Signal 2007;19:
1309–14.
[234] Bunce MW, Boronenkov IV, Anderson RA. Coordinated activation of the
nuclear ubiquitin ligase Cul3-SPOP by the generation of phosphatidylinositol
5-phosphate. J Biol Chem 2008;283:8678–86.
[235] Homma K, Terui S, Minemura M, Qadota H, Anraku Y, Kanaho Y, et al.
Phosphatidylinositol-4-phosphate 5-kinase localized on the plasma
membrane is essential for yeast cell morphogenesis. J Biol Chem
1998;273:15779–86.
[236] Luoh SW, Venkatesan N, Tripathi R. Overexpression of the amplified Pip4k2β
gene from 17q11–12 in breast cancer cells confers proliferation advantage.
Oncogene 2004;23:1354–63.

[237] Rozenvayn N, Flaumenhaft R. Protein kinase C mediates translocation of type
II phosphatidylinositol 5-phosphate 4-kinase required for platelet α-granule
secretion. J Biol Chem 2003;278:8126–34.
[238] Morris JB, Hinchliffe KA, Ciruela A, Letcher AJ, Irvine RF. Thrombin
stimulation of platelets causes an increase in phosphatidylinositol 5-
phosphate revealed by mass assay. FEBS Lett 2000;475:57–60.
[239] Sbrissa D, Ikonomov O, Strakova J, Shisheva A. Role for a novel signaling
intermediate, phosphatidylinositol 5-phosphate, in insulin-regulated F-actin
stress fiber breakdown and GLUT4 translocation. Endocrinology
2004;145:4853–65.
[240] Tronchere H, Laporte J, Pendaries C, Chaussade C, Liaubet L, Pirola L, et al.
Production of phosphatidylinositol 5-phosphate by the phosphoinositide 3-
phosphatase myotubularin in mammalian cells. J Biol Chem
2004;279:7304–12.
[241] Jones DR, Bultsma Y, Keune WJ, Halstead JR, Elouarrat D, Mohammed S, et al.
Nuclear PtdIns5P as a transducer of stress signaling: an in vivo role for
PIP4KB. Mol Cell 2006;23:685–95.
[242] Wilcox A, Hinchliffe KA. Regulation of extranuclear PtdIns5P production
by phosphatidylinositol phosphate 4-kinase 2α. FEBS Lett 2008;582:
1391–4.
[243] Clarke JH, Letcher AJ, D'Santos CS, Halstead JR, Irvine RF, Divecha N. Inositol
lipids are regulated during cell cycle progression in the nuclei of murine
erythroleukaemia cells. Biochem J 2001;357:905–10.
[244] Lamia KA, Peroni OD, Kim YB, Rameh LE, Kahn BB, Cantley LC. Increased
insulin sensitivity and reduced adiposity in phosphati-
dylinositol 5-phosphate 4-kinase β⁻/⁻ mice. Mol Cell Biol 2004;24:
5080–7.
[245] Carricaburu V, Lamia KA, Lo E, Favreau L, Payrastre B, Cantley LC, et al. The
phosphatidylinositol (PI)-5-phosphate 4-kinase type II enzyme controls
insulin signaling by regulating PI-3, 4, 5-trisphosphate degradation. Proc
Natl Acad Sci USA 2003;100:9867–72.
[246] Pendaries C, Tronchere H, Arbibe L, Mounier J, Gozani O, Cantley L, et al.
PtdIns5P activates the host cell PI3-kinase/Akt pathway during Shigella
flexneri infection. EMBO J 2006;25:1024–34.
[247] Hinchliffe KA, Giudici ML, Letcher AJ, Irvine RF. Type IIα phosphatidylinositol
phosphate kinase associates with the plasma membrane via interaction with
type I isoforms. Biochem J 2002;363:563–70.
[248] Hinchliffe KA, Irvine RF. Regulation of type II PIP kinase by PKD
phosphorylation. Cell Signal 2006;18:1906–13.
[249] Huang Z, Guo XX, Chen SX, Alvarez KM, Bell MW, Anderson RE. Regulation of
type II phosphatidylinositol phosphate kinase by tyrosine phosphorylation in
bovine rod outer segments. Biochemistry 2001;40:4550–9.
[250] Gozani O, Karuman P, Jones DR, Ivanov D, Cha J, Lugovskoy AA, et al. The PHD
finger of the chromatin-associated protein ING2 functions as a nuclear
phosphoinositide receptor. Cell 2003;114:99–111.
[251] Stopkova P, Saito T, Fann CS, Papolos DF, Vevera J, Pacalt I, et al. Polymorphism
screening of PIP5K2A: a candidate gene for chromosome 10p-linked
psychiatric disorders. Am J Med Genet B Neuropsychiatr Genet
2003;123B:50–8.
[252] Schwab SG, Knapp M, Sklar P, Eckstein GN, Sewekow C, Borrmann-
Hassenbach M, et al. Evidence for association of DNA sequence variants in
the phosphatidylinositol-4-phosphate 5-kinase IIα gene (PIP5K2A) with
schizophrenia. Mol Psychiatr 2006;11:837–46.
[253] Fedorenko O, Strutz-Seebohm N, Henrion U, Ureche ON, Lang F, Seebohm G, et al. A schizophrenia-linked mutation in PIP5K2A fails to activate neuronal
M channels. Psychopharmacology (Berl) 2008;199:47–54.
[254] Botelho RJ, Efe JA, Teis D, Emr SD. Assembly of a Fab1 phosphoinositide kinase
signaling complex requires the Fig4 phosphoinositide phosphatase. Mol Biol
Cell 2008;19:4273–86.
[255] Sbrissa D, Ikonomov O, Shisheva A. PIKfyve lipid kinase is a protein kinase:
downregulation of 5′-phosphoinositide product formation by
autophosphorylation. Biochemistry 2000;39:15980–9.
[256] Ikonomov OC, Sbrissa D, Shisheva A. Mammalian cell morphology and
endocytic membrane homeostasis require enzymatically active phospho-
inositide 5-kinase PIKfyve. J Biol Chem 2001;276:26141–7.
[257] Sbrissa D, Ikonomov O, Deeb R, Shisheva A. Phosphatidylinositol 5-
phosphate biosynthesis is linked to PIKfyve and is involved in osmotic
response pathway in mammalian cells. J Biol Chem 2002;277:
47276–84.
[258] Shisheva A, Sbrissa D, Ikonomov O. Cloning, characterization, and expression
of a novel Zn2+-binding FYVE finger-containing phosphoinositide kinase in
insulin-sensitive cells. Mol Cell Biol 1999;19:623–34.
[259] Rutherford AC, Traer C, Wassmer T, Pattni K, Bujny MV, Carlton JG, et al. The
mammalian phosphatidylinositol 3-phosphate 5-kinase (PIKfyve)
regulates endosome-to-TGN retrograde transport. J Cell Sci 2006;119:
3944–57.
[260] Ikonomov O, Sbrissa D, Foti M, Carpentier J, Shisheva A. PIKfyve controls fluid
phase endocytosis but not recycling/degradation of endocytosed receptors or
sorting of procathepsin D by regulating multivesicular body morphogenesis.
Mol Biol Cell 2003;14:4581–91.
[261] Ikonomov O, Sbrissa D, Mlak K, Kanzaki M, Pessin J, Shisheva A. Functional
dissection of lipid and protein kinase signals of PIKfyve reveals the role of
PtdIns 3,5-P₂ production for endomembrane integrity. J Biol Chem
2002;277:9206–11.
[262] Shisheva A. PIKfyve: partners, significance, debates and paradoxes. Cell Biol
Int 2008;32:591–604.

[263] Ikonomov O, Sbrissa D, Mlak K, Shisheva A. Requirement for PIKfyve enzymatic activity in acute and long-term insulin cellular effects. Endocrinology 2002;143:4742–54.

[264] Ikonomov O, Sbrissa D, Dondapati R, Shisheva A. ArPIKfyve-PIKfyve interaction and role in insulin-regulated GLUT4 translocation and glucose transport in 3T3-L1 adipocytes. Exp Cell Res 2007;313:2404–16.

[265] Berwick D, Dell G, Welsh G, Heesom K, Hers I, Fletcher L, et al. Protein kinase B phosphorylation of PIKfyve regulates the trafficking of GLUT4 vesicles. J Cell Sci 2004;117:5985–93.

[266] Jefferies H, Cooke F, Jat P, Boucheron C, Koizumi T, Hayakawa M, et al. A selective PIKfyve inhibitor blocks PtdIns(3,5)P(2) production and disrupts endomembrane transport and retroviral budding. EMBO Rep 2008;9:164–70.

[267] Rudge S, Anderson D, Emr S. Vacuole size control: regulation of PtdIns(3,5)P₂ levels by the vacuole-associated Vac14-Fig4 complex, a PtdIns(3,5)P₂-specific phosphatase. Mol Biol Cell 2004;15:24–36.

[268] Sbrissa D, Ikonomov O, Fu Z, Ijuin T, Gruenberg J, Takenawa T, et al. Core protein machinery for mammalian phosphatidylinositol 3,5-bisphosphate synthesis and turnover that regulates the progression of endosomal transport. Novel Sac phosphatase joins the ArPIKfyve-PIKfyve complex. J Biol Chem 2007;282:23878–91.

[269] Zhang Y, Zolov S, Chow C, Slutsky S, Richardson S, Piper R, et al. Loss of Vac14, a regulator of the signaling lipid phosphatidylinositol 3,5-bisphosphate, results in neurodegeneration in mice. Proc Natl Acad Sci USA 2007;104:17518–23.

[270] Chow C, Zhang Y, Dowling J, Jin N, Adamska M, Shiga K, et al. Mutation of FIG4 causes neurodegeneration in the pale tremor mouse and patients with CMT4J. Nature 2007;448:68–72.

[271] Odorizzi G, Babst M, Emr SD. Fab1p PtdIns(3)P 5-kinase function essential for protein sorting in the multivesicular body. Cell 1998;95:847–58.

[272] Li S, Tiab L, Jiao X, Munier FL, Zografos L, Frueh BE, et al. Mutations in PIP5K3 are associated with Francois-Neetens mouchetee fleck corneal dystrophy. Am J Hum Genet 2005;77:54–63.

[273] Cabezas A, Pattni K, Stenmark H. Cloning and subcellular localization of a human phosphatidylinositol 3-phosphate 5-kinase, PIKfyve/Fab1. Gene 2006;371:34–41.

[274] Lee JO, Yang H, Georgescu MM, Di Cristofano A, Maehama T, Shi Y, et al. Crystal structure of the PTEN tumor suppressor: implications for its phosphoinositide phosphatase activity and membrane association. Cell 1999;99:323–34.

[275] Myers MP, Stolarov JP, Eng C, Li J, Wang SI, Wigler MH, et al. P-TEN, the tumor suppressor from human chromosome 10q23, is a dual-specificity phosphatase. Proc Natl Acad Sci USA 1997;94:9052–7.

[276] Myers MP, Pass I, Batty IH, Van der Kaay J, Stolarov JP, Hemmings BA, et al. The lipid phosphatase activity of PTEN is critical for its tumor suppressor function. Proc Natl Acad Sci USA 1998;95:13513–8.

[277] Gil A, Andres-Pons A, Pulido R. Nuclear PTEN: a tale of many tails. Cell Death Differ 2007;14:395–9.

[278] Baker SJ. PTEN enters the nuclear age. Cell 2007;128:25–8.

[279] Odriozola L, Singh G, Hoang T, Chan AM. Regulation of PTEN activity by its carboxyl-terminal autoinhibitory domain. J Biol Chem 2007;282:23306–15.

[280] Trotman LC, Wang X, Alimonti A, Chen Z, Teruya-Feldstein J, Yang H, et al. Ubiquitination regulates PTEN nuclear import and tumor suppression. Cell 2007;128:141–56.

[281] Freeman DJ, Li AG, Wei G, Li HH, Kertesz N, Lesche R, et al. PTEN tumor suppressor regulates p53 protein levels and activity through phosphatase-dependent and -independent mechanisms. Cancer Cell 2003;3:117–30.

[282] Lin HK, Hu YC, Lee DK, Chang C. Regulation of androgen receptor signaling by PTEN (phosphatase and tensin homolog deleted on chromosome 10) tumor suppressor through distinct mechanisms in prostate cancer cells. Mol Endocrinol 2004;18:2409–23.

[283] Shen WH, Balajee AS, Wang J, Wu H, Eng C, Pandolfi PP, et al. Essential role for nuclear PTEN in maintaining chromosomal integrity. Cell 2007;128:157–70.

[284] Okumura K, Zhao M, Depinho RA, Furnari FB, Cavenee WK. Cellular transformation by the MSP58 oncogene is inhibited by its physical interaction with the PTEN tumor suppressor. Proc Natl Acad Sci USA 2005;102:2703–6.

[285] Lindsay Y, McCoull D, Davidson L, Leslie NR, Fairservice A, Gray A, et al. Localization of agonist-sensitive PtdIns(3,4,5)P₃ reveals a nuclear pool that is insensitive to PTEN expression. J Cell Sci 2006;119:5160–8.

[286] Marsh DJ, Coulon V, Lunetta KL, Rocca-Serra P, Dahia PL, Zheng Z, et al. Mutation spectrum and genotype-phenotype analyses in Cowden disease and Bannayan-Zonana syndrome, two hamartoma syndromes with germline PTEN mutation. Hum Mol Genet 1998;7:507–15.

[287] Zhou XP, Waite KA, Pilarski R, Hampel H, Fernandez MJ, Bos C, et al. Germline PTEN promoter mutations and deletions in Cowden/Bannayan-Riley-Ruvalcaba syndrome result in aberrant PTEN protein and dysregulation of the phosphoinositol-3-kinase/Akt pathway. Am J Hum Genet 2003;73:404–11.

[288] Stokoe D. Pten. Curr Biol 2001;11:R502.

[289] Yanagi S, Kishimoto H, Kawahara K, Sasaki M, Nishio M, et al. Pten controls lung morphogenesis, bronchioalveolar stem cells, and onset of lung adenocarcinomas in mice. J Clin Invest 2007;117:2929–40.

[290] Kimura T, Suzuki A, Fujita Y, Yomogida K, Lomeli H, Asada N, et al. Conditional loss of PTEN leads to testicular teratoma and enhances embryonic germ cell production. Development 2003;130:1691–700.

[291] Whang YE, Wu X, Suzuki H, Reiter RE, Tran C, Vessella RL, et al. Inactivation of the tumor suppressor PTEN/MMAC1 in advanced human prostate cancer through loss of expression. Proc Natl Acad Sci USA 1998;95:5246–50.

[292] He XC, Yin T, Grindley JC, Tian Q, Sato T, Tao WA, et al. PTEN-deficient intestinal stem cells initiate intestinal polyposis. Nat Genet 2007;39:189–98.

[293] Groszer M, Erickson R, Scripture-Adams DD, Lesche R, Trumpp A, Zack JA, et al. Negative regulation of neural stem/progenitor cell proliferation by the Pten tumor suppressor gene in vivo. Science 2001;294:2186–9.

[294] Kwabi-Addo B, Giri D, Schmidt K, Podsypanina K, Parsons R, Greenberg N, et al. Haploinsufficiency of the Pten tumor suppressor gene promotes prostate cancer progression. Proc Natl Acad Sci USA 2001;98:11563–8.

[295] Suzuki A, de la Pompa JL, Stambolic V, Elia AJ, Sasaki T, del Barco Barrantes I, et al. High cancer susceptibility and embryonic lethality associated with mutation of the PTEN tumor suppressor gene in mice. Curr Biol 1998;8:1169–78.

[296] Hamada K, Sasaki T, Koni PA, Natsui M, Kishimoto H, Sasaki J, et al. The PTEN/PI3K pathway governs normal vascular development and tumor angiogenesis. Genes Dev 2005;19:2054–65.

[297] Kishimoto H, Ohteki T, Yajima N, Kawahara K, Natsui M, Kawarasaki S, et al. The Pten/PI3K pathway governs the homeostasis of Vα 14iNKT cells. Blood 2007;109:3316–24.

[298] Di Cristofano A, Pesce B, Cordon-Cardo C, Pandolfi PP. Pten is essential for embryonic development and tumour suppression. Nat Genet 1998;19:348–55.

[299] Stambolic V, Suzuki A, de la Pompa JL, Brothers GM, Mirtsos C, Sasaki T, et al. Negative regulation of PKB/Akt-dependent cell survival by the tumor suppressor PTEN. Cell 1998;95:29–39.

[300] Di Cristofano A, Kotsi P, Peng YF, Cordon-Cardo C, Elkou KB, Pandolfi PP. Impaired Fas response and autoimmunity in Pten+/− mice. Science 1999;285:2122–5.

[301] Suzuki A, Yamaguchi MT, Ohteki T, Sasaki T, Kaisho T, Kimura Y, et al. T cell-specific loss of Pten leads to defects in central and peripheral tolerance. Immunity 2001;14:523–34.

[302] Horie Y, Suzuki A, Kataoka E, Sasaki T, Hamada K, Sasaki J, et al. Hepatocyte-specific Pten deficiency results in steatohepatitis and hepatocellular carcinomas. J Clin Invest 2004;113:1774–83.

[303] Wijesekara N, Konrad D, Eweida M, Jefferies C, Liadis N, Giacca A, et al. Muscle-specific Pten deletion protects against insulin resistance and diabetes. Mol Cell Biol 2005;25:1135–45.

[304] Backman SA, Stambolic V, Suzuki A, Haight J, Elia A, Pretorius J, et al. Deletion of Pten in mouse brain causes seizures, ataxia and defects in soma size resembling Lhermitte-Duclos disease. Nat Genet 2001;29:396–403.

[305] Suzuki A, Kaisho T, Ohishi M, Tsukio-Yamaguchi M, Tsubata T, Koni PA, et al. Critical roles of Pten in B cell homeostasis and immunoglobulin class switch recombination. J Exp Med 2003;197:657–67.

[306] Liu X, Bruxvoort KJ, Zylstra CR, Liu J, Cichowski R, Faugere MC, et al. Lifelong accumulation of bone in mice lacking Pten in osteoblasts. Proc Natl Acad Sci USA 2007;104:2259–64.

[307] Suzuki A, Itami S, Ohishi M, Hamada K, Inoue T, Komazawa N, et al. Keratinocyte-specific Pten deficiency results in epidermal hyperplasia, accelerated hair follicle morphogenesis and tumor formation. Cancer Res 2003;63:674–81.

[308] Inoue-Narita T, Hamada K, Sasaki T, Hatakeyama S, Fujita S, Kawahara K, et al. Pten deficiency in melanocytes results in resistance to hair graying and susceptibility to carcinogen-induced melanomagenesis. Cancer Res 2008;68:5760–8.

[309] Knobbe CB, Lapin V, Suzuki A, Mak TW. The roles of PTEN in development, physiology and tumorigenesis in mouse models: a tissue-by-tissue survey. Oncogene 2008;27:5398–415.

[310] Suzuki A, Nakano T, Mak TW, Sasaki T. Portrait of PTEN: messages from mutant mice. Cancer Sci 2008;99:209–13.

[311] Sarquis MS, Agrawal S, Shen L, Pilarski R, Zhou XP, Eng C. Distinct expression profiles for PTEN transcript and its splice variants in Cowden syndrome and Bannayan-Riley-Ruvalcaba syndrome. Am J Hum Genet 2006;79:23–30.

[312] Mirmohammadsadegh A, Marini A, Nambiar S, Hassan M, Tannapfel A, Ruzicka T, et al. Epigenetic silencing of the PTEN gene in melanoma. Cancer Res 2006;66:6546–52.

[313] Vazquez F, Ramaswamy S, Nakamura N, Sellers WR. Phosphorylation of the PTEN tail regulates protein stability and function. Mol Cell Biol 2000;20:5010–8.

[314] Torres J, Pulido R. The tumor suppressor PTEN is phosphorylated by the protein kinase CK2 at its C terminus. Implications for PTEN stability to proteasome-mediated degradation. J Biol Chem 2001;276:993–8.

[315] Maccario H, Perera NM, Davidson L, Downes CP, Leslie NR. PTEN is destabilized by phosphorylation on Thr366. Biochem J 2007;405:439–44.

[316] Lu Y, Yu Q, Liu JH, Zhang J, Wang H, Koul D, et al. Src family protein-tyrosine kinases alter the function of PTEN to regulate phosphatidylinositol 3-kinase/AKT cascades. J Biol Chem 2003;278:40057–66.

[317] Okumura K, Mendoza M, Bachoo RM, DePinho RA, Cavenee WK, Furnari FB. PCAF modulates PTEN activity. J Biol Chem 2006;281:26562–8.

[318] Kwon J, Lee SR, Yang KS, Ahn Y, Kim YJ, Stadtman ER, et al. Reversible oxidation and inactivation of the tumor suppressor PTEN in cells

stimulated with peptide growth factors. Proc Natl Acad Sci USA 2004;101:16419–24.

[319] Wang X, Trotman LC, Koppie T, Alimonti A, Chen Z, Gao Z, et al. NEDD4-1 is a proto-oncogenic ubiquitin ligase for PTEN. Cell 2007;128:129–39.

[320] Song MS, Salmena L, Carracedo A, Egia A, Lo-Coco F, Teruya-Feldstein J, et al. The deubiquitinylation and localization of PTEN are regulated by a HAUSP-PML network. Nature 2008;455:813–7.

[321] Walker SM, Downes CP, Leslie NR. TPIP: a novel phosphoinositide 3-phosphatase. Biochem J 2001;360:277–83.

[322] Tapparel C, Reymond A, Girardet C, Guillou L, Lyle R, Lamon C, et al. The TPTE gene family: cellular expression, subcellular localization and alternative splicing. Gene 2003;323:189–99.

[323] Murata Y, Iwasaki H, Sasaki M, Inaba K, Okamura Y. Phosphoinositide phosphatase activity coupled to an intrinsic voltage sensor. Nature 2005;435:1239–43.

[324] Iwasaki H, Murata Y, Kim Y, Hossain MI, Worby CA, Dixon JE, et al. A voltage-sensing phosphatase, Ci-VSP, which shares sequence identity with PTEN, dephosphorylates phosphatidylinositol 4, 5-bisphosphate. Proc Natl Acad Sci USA 2008;105:7970–5.

[325] Chen H, Rossier C, Morris MA, Scott HS, Gos A, Bairoch A, et al. A testis-specific gene, TPTE, encodes a putative transmembrane tyrosine phosphatase and maps to the pericentromeric region of human chromosomes 21 and 13, and to chromosomes 15, 22, and Y. Hum Genet 1999;105:399–409.

[326] Worby CA, Dixon JE. Phosphoinositide phosphatases: emerging roles as voltage sensors? Mol Interv 2005;5:274–7.

[327] Leslie NR, Yang X, Downes CP, Weijer CJ. PtdIns(3,4,5)P(3)-dependent and -independent roles for PTEN in the control of cell migration. Curr Biol 2007;17:115–25.

[328] Murata Y, Okamura Y. Depolarization activates the phosphoinositide phosphatase Ci-VSP, as detected in Xenopus oocytes coexpressing sensors of PIP₂. J Physiol 2007;583:875–89.

[329] Doerks T, Strauss M, Brendel M, Bork P. GRAM, a novel domain in glucosyltransferases, myotubularins and other putative membrane-associated proteins. Trends Biochem Sci 2000;25:483–5.

[330] Cui X, De Vivo I, Slany R, Miyamoto A, Firestein R, Cleary ML. Association of SET domain and myotubularin-related proteins modulates growth control. Nat Genet 1998;18:331–7.

[331] Laporte J, Hu L, Kretz C, Mandel J, Kioschis P, Coy J, et al. A gene mutated in X-linked myotubular myopathy defines a new putative tyrosine phosphatase family conserved in yeast. Nat Genet 1996;13:175–82.

[332] Taylor G, Maehama T, Dixon J. Inaugural article: myotubularin, a protein tyrosine phosphatase mutated in myotubular myopathy, dephosphorylates the lipid second messenger, phosphatidylinositol 3-phosphate. Proc Natl Acad Sci USA 2000;97:8910–5.

[333] Buj-Bello A, Laugel V, Messaddeq N, Zahreddine H, Laporte J, Pellissier J, et al. The lipid phosphatase myotubularin is essential for skeletal muscle maintenance but not for myogenesis in mice. Proc Natl Acad Sci USA 2002;99:15060–5.

[334] Nandurkar H, Caldwell K, Whisstock J, Layton M, Gaudet E, Norris F, et al. Characterization of an adapter subunit to a phosphatidylinositol (3)P 3-phosphatase: identification of a myotubularin-related protein lacking catalytic activity. Proc Natl Acad Sci USA 2001;98:9499–504.

[335] Nandurkar H, Layton M, Laporte J, Selan C, Corcoran L, Caldwell K, et al. Identification of myotubularin as the lipid phosphatase catalytic subunit associated with the 3-phosphatase adapter protein, 3-PAP. Proc Natl Acad Sci USA 2003;100:8660–5.

[336] Kioschis P, Wiemann S, Heiss N, Francis F, Gotz C, Poustka A, et al. Genomic organization of a 225-kb region in Xq28 containing the gene for X-linked myotubular myopathy (MTM1) and a related gene (MTMR1). Genomics 1998;54:256–66.

[337] Begley M, Taylor G, Kim S, Veine D, Dixon J, Stuckey J. Crystal structure of a phosphoinositide phosphatase, MTMR2: insights into myotubular myopathy and Charcot-Marie-Tooth syndrome. Mol Cell 2003;12:1391–402.

[338] Tsujita K, Itoh T, Ijuin T, Yamamoto A, Shisheva A, Laporte J, et al. Myotubularin regulates the function of the late endosome through the gram domain-phosphatidylinositol 3,5-bisphosphate interaction. J Biol Chem 2004;279:13817–24.

[339] Schaetzky J, Dove SK, Short B, Lorenzo O, Clague MJ, Barr FA. Phosphatidylinositol-5-phosphate activation and conserved substrate specificity of the myotubularin phosphatidylinositol 3-phosphatases. Curr Biol 2003;13:504–9.

[340] Begley M, Taylor G, Brock M, Ghosh P, Woods V, Dixon J. Molecular basis for substrate recognition by MTMR2, a myotubularin family phosphoinositide phosphatase. Proc Natl Acad Sci USA 2006;103:927–32.

[341] Bolino A, Muglia M, Conforti F, LeGuern E, Salih M, Georgiou D, et al. Charcot-Marie-Tooth type 4B is caused by mutations in the gene encoding myotubularin-related protein-2. Nat Genet 2000;25:17–9.

[342] Berger P, Berger I, Schaffitzel C, Tersar K, Volkmer B, Suter U. Multi-level regulation of myotubularin-related protein-2 phosphatase activity by myotubularin-related protein-13/set-binding factor-2. Hum Mol Genet 2006;15:569–79.

[343] Bolino A, Bolis A, Previtali S, Dina G, Bussini S, Dati G, et al. Disruption of Mtmr2 produces CMT4B1-like neuropathy with myelin outfoldings and impaired spermatogenesis. J Cell Biol 2004;167:711–21.

[344] Bonneick S, Boentert M, Berger P, Atanasoski S, Mantei N, Wessig C, et al. An animal model for Charcot-Marie-Tooth disease type 4B1. Hum Mol Genet 2005;14:3685–95.

[345] Bolis A, Coviello S, Bussini S, Dina G, Pardini C, Previtali S, et al. Loss of Mtmr2 phosphatase in Schwann cells but not in motor neurons causes Charcot-Marie-Tooth type 4B1 neuropathy with myelin outfoldings. J Neurosci 2005;25:8567–77.

[346] Kim S, Vacratsis P, Firestein R, Cleary M, Dixon J. Regulation of myotubularin-related (MTMR)2 phosphatidylinositol phosphatase by MTMR5, a catalytically inactive phosphatase. Proc Natl Acad Sci USA 2003;100:4492–7.

[347] Firestein R, Nagy P, Daly M, Huie P, Conti M, Cleary M. Male infertility, impaired spermatogenesis, and azoospermia in mice deficient for the pseudophosphatase Sbf1. J Clin Invest 2002;109:1165–72.

[348] Senderek J, Bergmann C, Weber S, Ketelsen U, Schorle H, Rudnik-Schoneborn S, et al. Mutation of the SBF2 gene, encoding a novel member of the myotubularin family, in Charcot-Marie-Tooth neuropathy type 4B2/11p15. Hum Mol Genet 2003;12:349–56.

[349] Azzedine H, Bolino A, Taieb T, Birouk N, Di Duca M, Bouhouche A, et al. Mutations in MTMR13, a new pseudophosphatase homologue of MTMR2 and Sbf1, in two families with an autosomal recessive demyelinating form of Charcot-Marie-Tooth disease associated with early-onset glaucoma. Am J Hum Genet 2003;72:1141–53.

[350] Tersar K, Boentert M, Berger P, Bonneick S, Wessig C, Toyka K, et al. Mtmr13/Sbf2-deficient mice. an animal model for CMT4B2. Hum Mol Genet 2007;16:2991–3001.

[351] Robinson F, Niesman I, Beiswenger K, Dixon J. Loss of the inactive myotubularin-related phosphatase Mtmr13 leads to a Charcot-Marie-Tooth 4B2-like peripheral neuropathy in mice. Proc Natl Acad Sci USA 2008;105:4916–21.

[352] Robinson F, Dixon J. Myotubularin phosphatases: policing 3-phosphoinositides. Trends Cell Biol 2006;16:403–12.

[353] Zhao R, Qi Y, Zhao Z. FYVE-DSP1, a dual-specificity protein phosphatase containing an FYVE domain. Biochem Biophys Res Commun 2000;270:222–9.

[354] Zhao R, Qi Y, Chen J, Zhao Z. FYVE-DSP2, a FYVE domain-containing dual specificity protein phosphatase that dephosphorylates phosphatidylinositol 3-phosphate. Exp Cell Res 2001;265:329–38.

[355] Walker D, Urbe S, Dove S, Tenza D, Raposo G, Clague M. Characterization of MTMR3. An inositol lipid 3-phosphatase with novel substrate specificity. Curr Biol 2001;11:1600–5.

[356] Lorenzo O, Urbe S, Clague M. Analysis of phosphoinositide binding domain properties within the myotubularin-related protein MTMR3. J Cell Sci 2005;118:2005–12.

[357] Srivastava S, Li Z, Lin L, Liu G, Ko K, Coetzee W, et al. The phosphatidylinositol 3-phosphate phosphatase myotubularin-related protein 6 (MTMR6) is a negative regulator of the Ca²⁺-activated K⁺ channel KCa3.1. Mol Cell Biol 2005;25:3630–8.

[358] Choudhury P, Srivastava S, Li Z, Ko K, Albaqumi M, Narayan K, et al. Specificity of the myotubularin family of phosphatidylinositol-3-phosphatase is determined by the PH/GRAM domain. J Biol Chem 2006;281:31762–9.

[359] Mochizuki Y, Majerus P. Characterization of myotubularin-related protein 7 and its binding partner, myotubularin-related protein 9. Proc Natl Acad Sci USA 2003;100:9768–73.

[360] Lorenzo O, Urbe S, Clague M. Systematic analysis of myotubularins: heteromeric interactions, subcellular localisation and endosome related functions. J Cell Sci 2006;119:2953–9.

[361] Zou J, Chang SC, Marjanovic J, Majerus PW. MTMR9 increases MTMR6 enzyme activity, stability, and role in apoptosis. J Biol Chem 2009;284:2064–71.

[362] Yanagiya T, Tanabe A, Iida A, Saito S, Sekine A, Takahashi A, et al. Association of single-nucleotide polymorphisms in MTMR9 gene with obesity. Hum Mol Genet 2007;16:3017–26.

[363] Tosch V, Rohde H, Tronchere H, Zanoteli E, Monroy N, Kretz C, et al. A novel PtdIns3P and PtdIns(3,5)P₂ phosphatase with an inactivating variant in centronuclear myopathy. Hum Mol Genet 2006;15:3098–106.

[364] Ivetic I, Munday AD, Kisseleva MV, Zhang XM, Luff S, Tiganis T, et al. The type Iα inositol polyphosphate 4-phosphatase generates and terminates phosphoinositide 3-kinase signals on endosomes and the plasma membrane. Mol Biol Cell 2005;16:2218–33.

[365] Shearn CT, Norris FA. Biochemical characterization of the type I inositol polyphosphate 4-phosphatase C2 domain. Biochem Biophys Res Commun 2007;356:255–9.

[366] Ferron M, Vacher J. Characterization of the murine Inpp4b gene and identification of a novel isoform. Gene 2006;376:152–61.

[367] Norris FA, Auethavekiat V, Majerus PW. The isolation and characterization of cDNA encoding human and rat brain inositol polyphosphate 4-phosphatase. J Biol Chem 1995;270:16128–33.

[368] Norris FA, Atkins RC, Majerus PW. Inositol polyphosphate 4-phosphatase is inactivated by calpain-mediated proteolysis in stimulated human platelets. J Biol Chem 1997;272:10987–9.

[369] Norris FA, Atkins RC, Majerus PW. The cDNA cloning and characterization of inositol polyphosphate 4-phosphatase type II. Evidence for conserved alternative splicing in the 4-phosphatase family. J Biol Chem 1997;272:23859–64.

[370] Shin HW, Hayashi M, Christoforidis S, Lacas-Gervais S, Hoepfner S, Wenk MR, et al. An enzymatic cascade of Rab5 effectors regulates phosphoinositide turnover in the endocytic pathway. J Cell Biol 2005;170:607–18.

[371] Vyas P, Norris FA, Joseph R, Majerus PW, Orkin SH. Inositol polyphosphate 4-phosphatase type I regulates cell growth downstream of transcription factor GATA-1. Proc Natl Acad Sci USA 2000;97:13696–701.

[372] Nystuen A, Legare ME, Shultz LD, Frankel WN. A null mutation in inositol polyphosphate 4-phosphatase type I causes selective neuronal loss in weebler mutant mice. Neuron 2001;32:203–12.

[373] Sharma M, Batra J, Mabalirajan U, Sharma S, Nagarkatti R, Aich J, et al. A genetic variation in inositol polyphosphate 4 phosphatase a enhances susceptibility to asthma. Am J Respir Crit Care Med 2008;177:712–9.

[374] Barnache S, Le Scolan E, Kosmider O, Denis N, Moreau-Gachelin F. Phosphatidylinositol 4-phosphatase type II is an erythropoietin-responsive gene. Oncogene 2006;25:1420–3.

[375] Ungewickell A, Hugge C, Kisseleva M, Chang SC, Zou J, Feng Y, et al. The identification and characterization of two phosphatidylinositol-4, 5-bisphosphate 4-phosphatases. Proc Natl Acad Sci USA 2005;102:18854–9.

[376] Zou J, Marjanovic J, Kisseleva MV, Wilson M, Majerus PW, Type I. Phosphatidylinositol-4, 5-bisphosphate 4-phosphatase regulates stress-induced apoptosis. Proc Natl Acad Sci USA 2007;104:16834–9.

[377] McPherson PS, Czernik AJ, Chilcote TJ, Onofri F, Benfenati F, Greengard P, et al. Interaction of Grb2 via its Src homology 3 domains with synaptic proteins including synapsin I. Proc Natl Acad Sci USA 1994;91:6486–90.

[378] Sakisaka T, Itoh T, Miura K, Takenawa T. Phosphatidylinositol 4,5-bisphosphate phosphatase regulates the rearrangement of actin filaments. Mol Cell Biol 1997;17:3841–9.

[379] McPherson PS, Garcia EP, Slepnev VI, David C, Zhang X, Grabs D, et al. A presynaptic inositol-5-phosphatase. Nature 1996;379:353–7.

[380] Nemoto Y, Arribas M, Haffner C, DeCamilli P. Synaptojanin 2, a novel synaptojanin isoform with a distinct targeting domain and expression pattern. J Biol Chem 1997;272:30817–21.

[381] Khvotchev M, Sudhof TC. Developmentally regulated alternative splicing in a novel synaptojanin. J Biol Chem 1998;273:2306–11.

[382] Seet LF, Cho S, Hessel A, Dumont DJ. Molecular cloning of multiple isoforms of synaptojanin 2 and assignment of the gene to mouse chromosome 17A2–3.1. Biochem Biophys Res Commun 1998;247:116–22.

[383] Nemoto Y, De Camilli P. Recruitment of an alternatively spliced form of synaptojanin 2 to mitochondria by the interaction with the PDZ domain of a mitochondrial outer membrane protein. EMBO J 1999;18:2991–3006.

[384] Nemoto Y, Wenk MR, Watanabe M, Daniell L, Murakami T, Ringstad N, et al. Identification and characterization of a synaptojanin 2 splice isoform predominantly expressed in nerve terminals. J Biol Chem 2001;276:41133–42.

[385] Ramjaun AR, McPherson PS. Tissue-specific alternative splicing generates two synaptojanin isoforms with differential membrane binding properties. J Biol Chem 1996;271:24856–61.

[386] Haffner C, Takei K, Chen H, Ringstad N, Hudson A, Butler MH, et al. Synaptojanin 1: localization on coated endocytic intermediates in nerve terminals and interaction of its 170 kDa isoform with Eps15. FEBS Lett 1997;419:175–80.

[387] Schmid AC, Wise HM, Mitchell CA, Nussbaum R, Woscholski R, Type II. Phosphoinositide 5-phosphatases have unique sensitivities towards fatty acid composition and head group phosphorylation. FEBS Lett 2004;576:9–13.

[388] Guo S, Stolz LE, Lemrow SM, York JD. SAC1-like domains of yeast SAC1, INP52, and INP53 and of human synaptojanin encode polyphosphoinositide phosphatases. J Biol Chem 1999;274:12990–5.

[389] Malecz N, McCabe PC, Spaargaren C, Qiu R, Chuang Y, Symons M. Synaptojanin 2, a novel Rac1 effector that regulates clathrin-mediated endocytosis. Curr Biol 2000;10:1383–6.

[390] Cremona O, Di Paolo G, Wenk MR, Luthi A, Kim WT, Takei K, et al. Essential role of phosphoinositide metabolism in synaptic vesicle recycling. Cell 1999;99:179–88.

[391] Kim WT, Chang S, Daniell L, Cremona O, Di Paolo G, De Camilli P. Delayed reentry of recycling vesicles into the fusion-competent synaptic vesicle pool in synaptojanin 1 knockout mice. Proc Natl Acad Sci USA 2002;99:17143–8.

[392] Mani M, Lee SY, Lucast L, Cremona O, Di Paolo G, De Camilli P, et al. The dual phosphatase activity of synaptojanin1 is required for both efficient synaptic vesicle endocytosis and reavailability at nerve terminals. Neuron 2007;56:1004–18.

[393] Perera RM, Zoncu R, Lucast L, De Camilli P, Toomre D. Two synaptojanin 1 isoforms are recruited to clathrin-coated pits at different stages. Proc Natl Acad Sci USA 2006;103:19332–7.

[394] de Heuvel E, Bell AW, Ramjaun AR, Wong K, Sossin WS, McPherson PS. Identification of the major synaptojanin-binding proteins in brain. J Biol Chem 1997;272:8710–6.

[395] Ringstad N, Nemoto Y, De Camilli P. The SH3p4/Sh3p8/SH3p13 protein family: binding partners for synaptojanin and dynamin via a Grb2-like Src homology 3 domain. Proc Natl Acad Sci USA 1997;94:8569–74.

[396] Qualmann B, Roos J, DiGregorio PJ, Kelly RB, Syndapin I. A synaptic dynaminsbinding protein that associates with the neural Wiskott–Aldrich syndrome protein. Mol Biol Cell 1999;10:501–13.

[397] Roos J, Kelly RB. Dap160, a neural-specific Eps15 homology and multiple SH3 domain-containing protein that interacts with Drosophila dynamin. J Biol Chem 1998;273:19108–19.

[398] Haffner C, Di Paolo G, Rosenthal JA, de Camilli P. Direct interaction of the 170 kDa isoform of synaptojanin 1 with clathrin and with the clathrin adaptor AP-2. Curr Biol 2000;10:471–4.

[399] Rusk N, Le PU, Mariggio S, Guay G, Lurisci C, Nabi IR, et al. Synaptojanin 2 functions at an early step of clathrin-mediated endocytosis. Curr Biol 2003;13:659–63.

[400] Chuang YY, Tran NL, Rusk N, Nakada M, Berens ME, Symons M. Role of synaptojanin 2 in glioma cell migration and invasion. Cancer Res 2004;64:8271–5.

[401] Lee SY, Wenk MR, Kim Y, Nairn AC, De Camilli P. Regulation of synaptojanin 1 by cyclin-dependent kinase 5 at synapses. Proc Natl Acad Sci USA 2004;101:546–51.

[402] Irie F, Okuno M, Pasquale EB, Yamaguchi Y. EphrinB-EphB signalling regulates clathrin-mediated endocytosis through tyrosine phosphorylation of synaptojanin 1. Nat Cell Biol 2005;7:501–9.

[403] Speed CJ, Matzaris M, Bird PI, Mitchell CA. Tissue distribution and intracellular localisation of the 75-kDa inositol polyphosphate 5-phosphatase. Eur J Biochem 1995;234:216–24.

[404] Jefferson AB, Majerus PW. Properties of type II inositol polyphosphate 5-phosphatase. J Biol Chem 1995;270:9370–7.

[405] Matzaris M, O'Malley CJ, Badger A, Speed CJ, Bird PI, Mitchell CA. Distinct membrane and cytosolic forms of inositol polyphosphate 5-phosphatase II. Efficient membrane localization requires two discrete domains. J Biol Chem 1998;273:8256–67.

[406] Lowe M. Structure and function of the Lowe syndrome protein OCRL1. Traffic 2005;6:711–9.

[407] Ponting CP. A novel domain suggests a ciliary function for ASPM, a brain size determining gene. Bioinformatics 2006;22:1031–5.

[408] Attree O, Olivos IM, Okabe I, Bailey LC, Nelson DL, Lewis RA, et al. The Lowe's oculocerebrorenal syndrome gene encodes a protein highly homologous to inositol polyphosphate-5-phosphatase. Nature 1992;358:239–42.

[409] Peck J, Douglas Gt, Wu CH, Burbelo PD. Human RhoGAP domain-containing proteins: structure, function and evolutionary relationships. FEBS Lett 2002;528:27–34.

[410] Lichter-Konecki U, Farber LW, Cronin JS, Suchy SF, Nussbaum RL. The effect of missense mutations in the RhoGAP-homology domain on ocrl1 function. Mol Genet Metab 2006;89:121–8.

[411] Faucherre A, Desbois P, Satre V, Lunardi J, Dorseuil O, Gacon G. Lowe syndrome protein OCRL1 interacts with Rac GTPase in the trans-Golgi network. Hum Mol Genet 2003;12:2449–56.

[412] Erdmann KS, Mao Y, McCrea HJ, Zoncu R, Lee S, Paradise S, et al. A role of the Lowe syndrome protein OCRL in early steps of the endocytic pathway. Dev Cell 2007;13:377–90.

[413] Choudhury R, Noakes CJ, McKenzie E, Kox C, Lowe M. Differential clathrin binding and subcellular localization of OCRL1 splice isoforms. J Biol Chem 2009;284:9965–73.

[414] Johnson JM, Castle J, Garrett-Engele P, Kan Z, Loerch PM, Armour CD, et al. Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays. Science 2003;302:2141–4.

[415] Zhang X, Jefferson AB, Auethavekiat V, Majerus PW. The protein deficient in Lowe syndrome is a phosphatidylinositol-4,5-bisphosphate 5-phosphatase. Proc Natl Acad Sci USA 1995;92:4853–6.

[416] Zhang X, Hartz PA, Philip E, Racusen LC, Majerus PW. Cell lines from kidney proximal tubules of a patient with Lowe syndrome lack OCRL inositol polyphosphate 5-phosphatase and accumulate phosphatidylinositol 4,5-bisphosphate. J Biol Chem 1998;273:1574–82.

[417] Olivos-Glander IM, Janne PA, Nussbaum RL. The oculocerebrorenal syndrome gene product is a 105-kD protein localized to the Golgi complex. Am J Hum Genet 1995;57:817–23.

[418] Dressman MA, Olivos-Glander IM, Nussbaum RL, Suchy SF. Ocrl1, a PtdIns(4,5)P(2) 5-phosphatase, is localized to the trans-Golgi network of fibroblasts and epithelial cells. J Histochem Cytochem 2000;48:179–90.

[419] Ungewickell A, Ward ME, Ungewickell E, Majerus PW. The inositol polyphosphate 5-phosphatase Ocrl associates with endosomes that are partially coated with clathrin. Proc Natl Acad Sci USA 2004;101:13501–6.

[420] Choudhury R, Diao A, Zhang F, Eisenberg E, Saint-Pol A, Williams C, et al. Lowe syndrome protein OCRL1 interacts with clathrin and regulates protein trafficking between endosomes and the trans-Golgi network. Mol Biol Cell 2005;16:3467–79.

[421] Faucherre A, Desbois P, Nagano F, Satre V, Lunardi J, Gacon G, et al. Lowe syndrome protein Ocr1 is translocated to membrane ruffles upon Rac GTPase activation: a new perspective on Lowe syndrome pathophysiology. Hum Mol Genet 2005;14:1441–8.

[422] Hyvola N, Diao A, McKenzie E, Skippen A, Cockcroft S, Lowe M. Membrane targeting and activation of the Lowe syndrome protein OCRL1 by rab GTPases. EMBO J 2006;25:3750–61.

[423] McCrea HJ, Paradise S, Tomasini L, Addis M, Melis MA, De Matteis MA, et al. All known patient mutations in the ASH-RhoGAP domains of OCRL affect

targeting and APPL1 binding. Biochem Biophys Res Commun 2008;369: 493–9.

[424] Suchy SF, Nussbaum RL. The deficiency of PIP₂ 5-phosphatase in Lowe syndrome affects actin polymerization. Am J Hum Genet 2002;71: 1420–7.

[425] Janne PA, Suchy SF, Bernard D, MacDonald M, Crawley J, Grinberg A, et al. Functional overlap between murine Inpp5b and OcrI1 may explain why deficiency of the murine ortholog for OCRL1 does not cause Lowe syndrome in mice. J Clin Invest 1998;101:2042–53.

[426] Nussbaum RL, Orrison BM, Janne PA, Charnas L, Chinault AC. Physical mapping and genomic structure of the Lowe syndrome gene OCRL1. Hum Genet 1997;99:145–50.

[427] Astle MV, Horan KA, Ooms LM, Mitchell CA. The inositol polyphosphate 5-phosphatases: traffic controllers, waistline watchers and tumour suppressors? Biochem Soc Symp 2007:161–81.

[428] Hoopes Jr RR, Shrimpton AE, Knohl SJ, Hueber P, Hoppe B, Matyus J, et al. Dent Disease with mutations in OCRL1. Am J Hum Genet 2005;76:260–7.

[429] Lin T, Orrison BM, Leahey AM, Suchy SF, Bernard DJ, Lewis RA, et al. Spectrum of mutations in the OCRL1 gene in the Lowe oculocerebrorenal syndrome. Am J Hum Genet 1997;60:1384–8.

[430] Lin T, Orrison BM, Suchy SF, Lewis RA, Nussbaum RL. Mutations are not uniformly distributed throughout the OCRL1 gene in Lowe syndrome patients. Mol Genet Metab 1998;64:58–61.

[431] Kawano T, Indo Y, Nakazato H, Shimadzu M, Matsuda I. Oculocerebrorenal syndrome of Lowe: three mutations in the OCRL1 gene derived from three patients with different phenotypes. Am J Med Genet 1998;77: 348–55.

[432] Williams C, Choudhury R, McKenzie E, Lowe M. Targeting of the type II inositol polyphosphate 5-phosphatase INPP5B to the early secretory pathway. J Cell Sci 2007;120:3941–51.

[433] Hellsten E, Bernard DJ, Owens JW, Eckhaus M, Suchy SF, Nussbaum RL. Sertoli cell vacuolization and abnormal germ cell adhesion in mice deficient in an inositol polyphosphate 5-phosphatase. Biol Reprod 2002;66:1522–30.

[434] Mochizuki Y, Takenawa T. Novel inositol polyphosphate 5-phosphatase localizes at membrane ruffles. J Biol Chem 1999;274:36790–5.

[435] Gurung R, Tan A, Ooms LM, McGrath MJ, Huysmans RD, Munday AD, et al. Identification of a novel domain in two mammalian inositol-polyphosphate 5-phosphatases that mediates membrane ruffle localization. The inositol 5-phosphatase skip localizes to the endoplasmic reticulum and translocates to membrane ruffles following epidermal growth factor stimulation. J Biol Chem 2003;278:11376–85.

[436] Ooms LM, Fedele CG, Astle MV, Ivetak I, Cheung V, Pearson RB, et al. The inositol polyphosphate 5-phosphatase, PIPP, Is a novel regulator of phosphoinositide 3-kinase-dependent neurite elongation. Mol Biol Cell 2006;17:607–22.

[437] Ijuin T, Takenawa T. SKIP negatively regulates insulin-induced GLUT4 translocation and membrane ruffle formation. Mol Cell Biol 2003;23: 1209–20.

[438] Ijuin T, Mochizuki Y, Fukami K, Funaki M, Asano T, Takenawa T. Identification and characterization of a novel inositol polyphosphate 5-phosphatase. J Biol Chem 2000;275:10870–5.

[439] Ijuin T, Yu YE, Mizutani K, Pao A, Tateya S, Tamori Y, et al. Increased insulin action in SKIP heterozygous knockout mice. Mol Cell Biol 2008;28:5184–95.

[440] Lucas DM, Rohrschneider LR. A novel spliced form of SH2-containing inositol phosphatase is expressed during myeloid development. Blood 1999;93: 1922–33.

[441] Tu Z, Ninos JM, Ma Z, Wang JW, Lemos MP, Desponts C, et al. Embryonic and hematopoietic stem cells express a novel SH2-containing inositol 5′-phosphatase isoform that partners with the Grb2 adapter protein. Blood 2001;98:2028–38.

[442] Liu L, Damen JE, Cutler RL, Krystal G. Multiple cytokines stimulate the binding of a common 145-kilodalton protein to Shc at the Grb2 recognition site of Shc. Mol Cell Biol 1994;14:6926–35.

[443] Wisniewski D, Strife A, Swendeman S, Erdjument-Bromage H, Geromanos S, Kavanaugh WM, et al. A novel SH2-containing phosphatidylinositol 3, 4, 5-trisphosphate 5-phosphatase (SHIP2) is constitutively tyrosine phosphorylated and associated with src homologous and collagen gene (SHC) in chronic myelogenous leukemia progenitor cells. Blood 1999;93: 2707–20.

[444] Ong CJ, Ming-Lum A, Nodwell M, Ghanipour A, Yang L, Williams DE, et al. Small-molecule agonists of SHIP1 inhibit the phosphoinositide 3-kinase pathway in hematopoietic cells. Blood 2007;110:1942–9.

[445] Phee H, Jacob A, Coggeshall KM. Enzymatic activity of the Src homology 2 domain-containing inositol phosphatase is regulated by a plasma membrane location. J Biol Chem 2000;275:19090–7.

[446] Liu Q, Shalaby F, Jones J, Bouchard D, Dumont DJ. The SH2-containing inositol polyphosphate 5-phosphatase, ship, is expressed during hematopoiesis and spermatogenesis. Blood 1998;91:2753–9.

[447] Horan KA, Watanabe K, Kong AM, Bailey CG, Rasko JE, Sasaki T, et al. Regulation of FcγR-stimulated phagocytosis by the 72-kDa inositol polyphosphate 5-phosphatase: SHIP1, but not the 72-kDa 5-phosphatase, regulates complement receptor 3 mediated phagocytosis by differential recruitment of these 5-phosphatases to the phagocytic cup. Blood 2007;110:4480–91.

[448] Yogo K, Mizutamari M, Mishima K, Takenouchi H, Ishida-Kitagawa N, Sasaki T, et al. Src homology 2 (SH2)-containing 5′-inositol phosphatase localizes to podosomes, and the SH2 domain is implicated in the attenuation of bone resorption in osteoclasts. Endocrinology 2006;147:3307–17.

[449] Helgason CD, Damen JE, Rosten P, Grewal R, Sorensen P, Chappel SM, et al. Targeted disruption of SHIP leads to hemopoietic perturbations, lung pathology, and a shortened life span. Genes Dev 1998;12:1610–20.

[450] Liu Q, Sasaki T, Kozierezki I, Wakeham A, Itie A, Dumont DJ, et al. SHIP is a negative regulator of growth factor receptor-mediated PKB/Akt activation and myeloid cell survival. Genes Dev 1999;13:786–91.

[451] Nishio M, Watanabe K, Sasaki J, Taya C, Takasuga S, Izuzuka R, et al. Control of cell polarity and motility by the PtdIns(3, 4, 5)P₃ phosphatase SHIP1. Nat Cell Biol 2007;9:36–44.

[452] Takeshita S, Namba N, Zhao JJ, Jiang Y, Genant HK, Silva MJ, et al. SHIP-deficient mice are severely osteoporotic due to increased numbers of hyper-resorptive osteoclasts. Nat Med 2002;8:943–9.

[453] Huber M, Helgason CD, Damen JE, Liu L, Humphries RK, Krystal G. The src homology 2-containing inositol phosphatase (SHIP) is the gatekeeper of mast cell degranulation. Proc Natl Acad Sci USA 1998;95:11330–5.

[454] Kaisaki PJ, Delepine M, Woon PY, Sebag-Montefiore L, Wilder SP, Menzel S, et al. Polymorphisms in type II SH2 domain-containing inositol 5-phosphatase (INPPL1, SHIP2) are associated with physiological abnormalities of the metabolic syndrome. Diabetes 2004;53:1900–4.

[455] Wada T, Sasaoka T, Funaki M, Hori H, Murakami S, Ishiki M, et al. Overexpression of SH2-containing inositol phosphatase 2 results in negative regulation of insulin-induced metabolic actions in 3T3-L1 adipocytes via its 5′-phosphatase catalytic activity. Mol Cell Biol 2001;21: 1633–46.

[456] Sleeman MW, Wortley KE, Lai KM, Gowen LC, Kintner J, Kline WO, et al. Absence of the lipid phosphatase SHIP2 confers resistance to dietary obesity. Nat Med 2005;11:199–205.

[457] Clement S, Krause U, Desmedt F, Tanti JF, Behrends J, Pesesse X, et al. The lipid phosphatase SHIP2 controls insulin sensitivity. Nature 2001;409:92–7.

[458] Asano T, Mochizuki Y, Matsumoto K, Takenawa T, Endo T. Pharbin, a novel inositol polyphosphate 5-phosphatase, induces dendritic appearances in fibroblasts. Biochem Biophys Res Commun 1999;261:188–95.

[459] Kisseleva MV, Wilson MP, Majerus PW. The isolation and characterization of a cDNA encoding phospholipid-specific inositol 5-phosphatase. J Biol Chem 2000;275:20110–6.

[460] Kong AM, Speed CJ, O'Malley CJ, Layton MJ, Meehan T, Loveland KL, et al. Cloning and characterization of a 72-kDa inositol-polyphosphate 5-phosphatase localized to the Golgi network. J Biol Chem 2000;275: 24052–64.

[461] Kamen LA, Levinsohn J, Swanson JA. Differential association of phosphatidylinositol 3-kinase, SHIP-1, and PTEN with forming phagosomes. Mol Biol Cell 2007;18:2463–72.

[462] Cox D, Dale BM, Kashiwada M, Helgason CD, Greenberg S. A regulatory role for Src homology 2 domain-containing inositol 5′-phosphatase (SHIP) in phagocytosis mediated by Fc γ receptors and complement receptor 3 (α(M)β(2); CD11b/CD18). J Exp Med 2001;193:61–71.

[463] Kong AM, Horan KA, Sriratana A, Bailey CG, Collyer LJ, Nandurkar HH, et al. Phosphatidylinositol 3-phosphate [PtdIns3P] is generated at the plasma membrane by an inositol polyphosphate 5-phosphatase: endogenous PtdIns3P can promote GLUT4 translocation to the plasma membrane. Mol Cell Biol 2006;26:6065–81.

[464] Ishiki M, Randhawa VK, Poon V, Jebailey L, Klip A. Insulin regulates the membrane arrival, fusion, and C-terminal unmasking of glucose transporter-4 via distinct phosphoinositides. J Biol Chem 2005;280: 28792–802.

[465] Kanda H, Tamori Y, Shinoda H, Yoshikawa M, Sakaue M, Udagawa J, et al. Adipocytes from Munc18c-null mice show increased sensitivity to insulin-stimulated GLUT4 externalization. J Clin Invest 2005;115:291–301.

[466] Merlot S, Meili R, Pagliarini D, Maehama T, Dixon J, Firtel R. A PTEN-related 5-phosphatidylinositol phosphatase localized in the Golgi. J Biol Chem 2003;278:39866–73.

[467] Pagliarini D, Worby C, Dixon J. A PTEN-like phosphatase with a novel substrate specificity. J Biol Chem 2004;279:38590–6.

[468] Pagliarini D, Wiley S, Kimple M, Dixon J, Kelly P, Worby C, et al. Involvement of a mitochondrial phosphatase in the regulation of ATP production and insulin secretion in pancreatic β cells. Mol Cell 2005;19:197–207.

[469] Novick P, Osmond B, Botstein D. Suppressors of yeast actin mutations. Genetics 1989;121:659–74.

[470] Nemoto Y, Kearns B, Wenk M, Chen H, Mori K, Alb JJ, et al. Functional characterization of a mammalian Sac1 and mutants exhibiting substrate-specific defects in phosphoinositide phosphatase activity. J Biol Chem 2000;275:34293–305.

[471] Rohde H, Cheong F, Konrad G, Paiha K, Mayinger P, Boehmelt G. The human phosphatidylinositol phosphatase SAC1 interacts with the coatomer I complex. J Biol Chem 2003;278:52689–99.

[472] Minagawa T, Ijuin T, Mochizuki Y, Takenawa T. Identification and characterization of a sac domain-containing phosphoinositide 5-phosphatase. J Biol Chem 2001;276:22011–5.

[473] Yuan Y, Gao X, Guo N, Zhang H, Xie Z, Jin M, et al. rSac3, a novel Sac domain phosphoinositide phosphatase, promotes neurite outgrowth in PC12 cells. Cell Res 2007;17:919–32.

[474] Blagoveshchenskaya A, Cheong F, Rohde H, Glover G, Knodler A, Nicolson T, et al. Integration of Golgi trafficking and growth factor signaling by the lipid phosphatase SAC1. J Cell Biol 2008;180:803–12.

[475] Liu Y, Boukhelifa M, Tribble E, Morin-Kensicki E, Uetrecht A, Bear J, et al. The Sac1 phosphoinositide phosphatase regulates Golgi membrane morphology and mitotic spindle organization in mammals. Mol Biol Cell 2008;19: 3080–96.

[476] Trivedi C, Luo Y, Yin Z, Zhang M, Zhu W, Wang T, et al. Hdac2 regulates the cardiac hypertrophic response by modulating Gsk3 β activity. Nat Med 2007;13:324–31.

[477] Holub BJ, Kuksis A. Structural and metabolic interrelationships among glycerophosphatides of rat liver in vivo. Can J Biochem 1971;49: 1347–56.

[478] Baker RR, Thompson W. Positional distribution and turnover of fatty acids in phosphatidic acid, phosphinositides, phosphatidylcholine and phosphatidyl- ethanolamine in rat brain in vivo. Biochim Biophys Acta 1972;270: 489–503.

[479] Wenk MR, Lucast L, Di Paolo G, Romanelli AJ, Suchy SF, Nussbaum RL, et al. Phosphoinositide profiling in complex lipid mixtures using electrospray ionization mass spectrometry. Nat Biotechnol 2003;21: 813–7.

[480] Ogiso H, Taguchi R. Reversed-phase LC/MS method for polyphosphoinositide analyses: changes in molecular species levels during epidermal growth factor activation in A431 cells. Anal Chem 2008;80:9226–32.

[481] Hishikawa D, Shindou H, Kobayashi S, Nakanishi H, Taguchi R, Shimizu T. Discovery of a lysophospholipid acyltransferase family essential for membrane asymmetry and diversity. Proc Natl Acad Sci USA 2008;105: 2830–5.

[482] Lee HC, Inoue T, Imae R, Kono N, Shirae S, Matsuda S, et al. Caenorhabditis elegans mboa-7, a member of the MBOAT family, is required for selective incorporation of polyunsaturated fatty acids into phosphatidylinositol. Mol Biol Cell 2008;19:1174–84.

[483] Shindou H, Shimizu T. Acyl-CoA:lysophospholipid acyltransferases. J Biol Chem 2009;284:1–5.
