
Pharmacology & Therapeutics 124 (2009) 219–234

Contents lists available at ScienceDirect Pharmacology & Therapeutics journal homepage: www.elsevier.com/locate/pharmthera

Associate editor: S. Pestka

# Functions of the cytoplasmic RNA sensors RIG-I and MDA-5: Key regulators of innate immunity

Paola M. Barral ${ }^{a, e, *}$, Devanand Sarkar ${ }^{a, b, c}$, Zao-zhong Su ${ }^{a}$, Glen N. Barber ${ }^{d}$, Rob DeSalle ${ }^{f}$, Vincent R. Racaniello ${ }^{e}$, Paul B. Fisher ${ }^{a, b, c, e, *}$

${ }^{a}$ Department of Human and Molecular Genetics, Virginia Commonwealth University, School of Medicine, Richmond, VA  
${ }^{b}$ VCU Institute of Molecular Medicine, Virginia Commonwealth University, School of Medicine, Richmond, VA  
${ }^{c}$ VCU Massey Cancer Center, Virginia Commonwealth University, School of Medicine, Richmond, VA, USA  
${ }^{d}$ Department of Microbiology and Immunology, UM/Sylvester Comprehensive Cancer Center, University of Miami School of Medicine, Miami, FL, USA  
${ }^{e}$ Department of Microbiology, Columbia University College of Physicians and Surgeons, New York, NY, USA  
${ }^{f}$ American Museum of Natural History, Sackler Institute for Comparative Genomics, New York, NY, USA

## ARTICLE INFO

**Keywords:**  
Antiviral innate immunity  
MDA-5  
RIG-I  
Domain grafting  
Cell signaling  
Apoptosis  
Viral pathogenesis  

## ABSTRACT

The innate immune system responds within minutes of infection to produce type I interferons and pro-inflammatory cytokines. Interferons induce the synthesis of cell proteins with antiviral activity, and also shape the adaptive immune response by priming T cells. Despite the discovery of interferons over 50 years ago, only recently have we begun to understand how cells sense the presence of a virus infection. Two families of pattern recognition receptors have been shown to distinguish unique molecules present in pathogens, such as bacterial and fungal cell wall components, viral RNA and DNA, and lipoproteins. The first family includes the membrane-bound toll-like receptors (TLRs). Studies of the signaling pathways that lead from pattern recognition to cytokine induction have revealed extensive and overlapping cascades that involve protein–protein interactions and phosphorylation, and culminate in activation of transcription proteins that control the transcription of genes encoding interferons and other cytokines. A second family of pattern recognition receptors has recently been identified, which comprises the cytoplasmic sensors of viral nucleic acids, including MDA-5, RIG-I, and LGP2. In this review we summarize the discovery of these cytoplasmic sensors, how they recognize nucleic acids, the signaling pathways leading to cytokine synthesis, and viral countermeasures that have evolved to antagonize the functions of these proteins. We also consider the function of these cytoplasmic sensors in apoptosis, development and differentiation, and diabetes.

© 2009 Elsevier Inc. All rights reserved.

## Contents

1. Introduction . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
P.M. Barral et al. / Pharmacology & Therapeutics 124 (2009) 219–234

| 9. Role of retinoic acid-inducible gene-I in development and differentiation | 229 |
| --- | --- |
| 10. Melanoma differentiation associated gene-5 and type 1 diabetes | 229 |
| 11. Evolution | 230 |
| 12. Perspectives | 231 |
| Acknowledgments | 231 |
| References | 232 |

---

### 1. Introduction

The number of pathogens that we encounter daily is astronomical. Most are halted by an efficient defense system that has evolved over millions of years in the face of microbial infections. An important component of this defense arsenal is the innate immune system, which responds within minutes of infection to produce type I interferons (Isaacs & Lindenmann, 1957; Rubinstein et al., 1979; Friesen et al., 1981; Pestka & Baron, 1981). These antiviral proteins are produced by infected cells and lead to the synthesis of cell proteins, which halt viral replication, and also shape the adaptive immune response by priming T cells. Additionally, interferons also modulate a plethora of other important cellular functions including cell growth and differentiation, histocompatibility and tumor antigen expression, gene expression and anti-tumor effects (Fisher et al., 1983; Fisher & Grant, 1984; Giacomini et al., 1984; Grant et al., 1985; Greiner et al., 1984; Huang et al., 1999a,1999b; Jiang et al., 2000; Moulton et al., 1992; Sen & Sarkar, 2007; Su et al., 2008; Greiner et al., 1985; Greiner et al., 1987).

Despite the discovery of interferons over 50 years ago, only recently have we begun to understand how cells sense the presence of a virus infection and initiate cytokine synthesis. First insights came from the discovery of the toll-like receptors (TLRs) during a study of genes essential for the establishment of the dorsal-ventral axis in Drosophila (Nusslein-Volhard & Wieschaus, 1980). The TLRs were subsequently shown to be transmembrane proteins that distinguish unique molecules present in pathogens, such as bacterial and fungal cell wall components, viral RNA and DNA, and lipoproteins (reviewed in Medzhitov 2007). We now understand that pathogens are recognized as foreign by a family of host pattern recognition receptors. Examination of the signaling pathways that lead from pattern recognition to cytokine induction has revealed extensive and overlapping cascades that involve protein-protein interactions and phosphorylation. These culminate in activation of transcription proteins that control the transcription of genes encoding interferons and other cytokines (reviewed in Kawai & Akira 2008).

A second family of pattern recognition receptors has recently been identified, which comprises the cytoplasmic sensors of viral nucleic acids, including MDA-5, RIG-I, and LGP2. Here we review the discovery of these proteins, how they recognize nucleic acids, the signaling pathways leading to cytokine synthesis, and viral countermeasures that have evolved to antagonize the functions of these proteins. We also consider the role of these cytoplasmic sensors in apoptosis, development and differentiation, and diabetes.

---

### 2. Characterization of retinoic acid-inducible gene-I and melanoma differentiation associated gene-5

#### 2.1. Identification of retinoic acid-inducible gene-I and melanoma differentiation associated gene-5

Retinoic acid-inducible gene-I (RIG-I, also known as DDX58) and melanoma differentiation associated gene-5 (MDA-5, also known as Helicard or IFIH1) are virus sensors expressed ubiquitously in the cytoplasm. RIG-I was initially identified as a gene induced in acute promyelocytic leukemia cells after treatment with retinoic acid (Sun, 1997). A few years later its role as an antiviral protein was reported. In fact, screening an expression cDNA library obtained from IFN-β-

P.M. Barral et al. / Pharmacology & Therapeutics 124 (2009) 219–234

![Diagram](https://i.imgur.com/yourimage.png)

Fig. 1. Schematic representation of the primary structure and functional domains of MDA-5, RIG-I, RIG-I-SV (splice variant) and LGP2. CARD: caspase activating recruitment domain. RD: repressor domain. CTD: C-terminal domain.

conformational change, that in the presence of ATP causes uncovering of the CARDs, that are then able to interact with the adaptor protein IPS-1 and transduce the signal (Takahasi et al., 2008) (Fig. 2).

### 2.3. Subcellular distribution of retinoic acid-inducible gene-I and melanoma differentiation associated gene-5

Recent evidence suggests that RIG-I localizes to membrane ruffles in non-polarized epithelial cells where it associates with the F-actin cytoskeleton (Mukherjee et al., 2009). MDA-5, however, localizes to the cytoplasm with no appreciable co-localization with F-actin. The specific distribution of RIG-I to the membrane ruffles is dependent on RacGTPase activity (Mukherjee et al., 2009). Deletion analysis has shown that association of RIG-I with the cytoskeleton was dependent on CARDs.

In further studies it has been shown that RIG-I is able to induce cellular migration through its actin association (Mukherjee et al., 2009). RIG-I, unlike MDA-5, is also localized to apical junction complexes in polarized epithelial and endothelial cells. The biological consequences of the interaction of RIG-I with F-actin are significant. It appears that actin depolymerization results in RIG-I redistribution and activation of IRF3 and NF-κB and IFN-β promoter activity (Mukherjee et al., 2009).

### 3. rig-I and mda-5 promoter regulation

rig-I was first cloned as a retinoic acid (RA)-inducible gene (Sun, 1997). However, the molecular mechanism by which RA regulates rig-I expression remains to be determined. The promoter region of rig-I has not been mapped for RA-responsive elements and with the accrual of information on the role of rig-I in RA signaling, efforts have not been expended in analyzing RA-regulation of rig-I. On the other hand, both rig-I and mda-5 are type I IFN-inducible genes and analysis of the ~2-kb promoter region of human rig-I revealed interesting findings (Su et al., 2007). The promoter was more active in diverse normal cells compared to their malignant counterparts that correlated with higher rig-I mRNA expression in normal cells compared to cancer cells. In an experimental mouse brain tumor model, rig-I expression was abundantly detected in the surrounding normal brain while it was hardly detectable in the tumor tissue. In both normal and tumor cells rig-I promoter activity could be induced by IFN-β or by poly(I).poly(C). Serial deletion and mutation analysis identified an interferon regulatory factor 1 (IRF1)-binding site just proximal to the transcription start site that conferred the basal, IFN-β- and dsRNA-mediated induction of the rig-I promoter. As a corollary to rig-I, IRF1 expression was also higher in normal cells than in cancer cells and IRF1 induction by IFN-β followed a similar kinetics to that of rig-I. IRF1 is a tumor suppressor and the expression profile of rig-I together with its regulation by IRF1 indicates that rig-I might also have tumor suppressor properties. Although mda-5 was cloned as a type I IFN-inducible gene and mda-5 induction by type I IFN has been studied extensively (Kang et al., 2004; Kang et al., 2002), as yet no studies have been published analyzing in detail the promoter region of mda-5. It would be useful to have a comparative study between rig-I and mda-5 promoters, to check if the mda-5 promoter also displays normal cell specificity and whether IFN regulation of the mda-5 promoter is controlled by the classical IFN-stimulated regulatory elements (ISRE) or by other factors, such as IRF1. More importantly, these promoters might be screened for identification of small molecules with potential antiviral properties.

### 4. The retinoic acid-inducible gene-I and melanoma differentiation associated gene-5 signaling pathway

#### 4.1. Recognition of ribonucleic acid

A role for RIG-I as a cytoplasmic sensor of viral RNA was first discovered during a screen for molecules involved in intracellular dsRNA-induced expression of IFN (Yoneyama et al., 2004). Over-expression of RIG-I in mouse L929 cells led to increased activation of the IFN-β promoter after introduction of dsRNA into cells. Activation of this promoter was shown to be dependent upon the RIG-I helicase domain. RIG-I was found to bind poly(I:C) linked to agarose beads, but not single-stranded RNA (poly(A)) or dsDNA. Synthesis of RIG-I also augmented the IFN response of cells infected with Newcastle disease virus, and reduced the viral yield of vesicular stomatitis virus, a rhabdovirus, and encephalomyocarditis virus (EMCV), a picornavirus. Based on these observations, it was suggested that RIG-I specifically recognizes dsRNA produced during the replication of RNA viruses

![Diagram](https://i.imgur.com/yourimage2.png)

Fig. 2. Model for activation of RIG-I and MDA-5 by RNA. In the absence of RNA, the sensor molecule is folded in an inactive state, with the CARD domain occluded by the CTD. The latter also contains the binding sites for RNA. When RNAs are produced in virus-infected cells (either dsRNA or ssRNA with 5′-phosphates), these bind the CTD and cause a conformational change that exposes the CARD domain. ATP is required for the conformational change. The CARD domain then interacts with downstream signaling molecules, leading to IFN transcription.

(P.M. Barral et al., Pharmacology & Therapeutics 124 (2009) 219–234)

(Yoneyama et al., 2004). However, the interaction of RIG-I with its RNA ligand was clearly not sufficient for induction of IFN. This conclusion came from the observation that an amino acid change in the ATP binding domain of RIG-I blocked the induction of IFN, but did not affect the ability of the protein to bind dsRNA (Yoneyama et al., 2004).

The importance of RIG-I in sensing RNA virus infections was emphasized by the results of experiments with mice lacking the *rig-I* gene (Kato et al., 2005). The production of IFN in response to infection with Sendai and Newcastle disease viruses (two paramyxoviruses), and VSV was abrogated in cDCs from *rig-I⁻/⁻* mice. In contrast, IFN production by pDCs was not dependent upon RIG-I, but upon toll-like receptors. These observations indicated that sensing of viral RNA by RIG-I occurs in a cell-type dependent manner.

MDA-5 was originally implicated in the recognition of poly(I:C) and was shown to bind this synthetic nucleic acid, although with less affinity than RIG-I (Kang et al., 2002; Yoneyama et al., 2005). Overproduction of MDA-5 inhibited the growth of EMCV and VSV, as was also observed for RIG-I (Yoneyama et al., 2005). In another study, it was shown that MDA-5, RIG-I and TLR3 are activated during measles virus infection (Berghäll et al., 2006) and overexpression of MDA-5 but not RIG-I or TLR3 leads to enhanced IFN-β promoter activity in MV-infected A549 cells, suggesting that MDA-5 is involved in MV-induced expression of antiviral cytokines in human epithelial cells (Berghäll et al., 2006). It has also been demonstrated that RIG-I and MDA-5 expression, but not TLR3, resulted in the activation of the IFN-β promoter in influenza A virus-infected epithelial cells (Sirén et al., 2006), suggesting that RIG-I and MDA-5 mediate influenza A virus-induced IFN-β synthesis in epithelial cells (Sirén et al., 2006).

Gene disruptions provided evidence for differences in RNA sensing (Gitlin et al., 2006; Kato et al., 2006). Administration of poly(I:C) to wild type or *rig-I⁻/⁻* mice led to similar levels of cytokines, including IFNs, in sera. In contrast, *mda-5⁻/⁻* mice did not produce IFN or interleukins in response to poly(I:C) (Kato et al., 2006). Cultured bone-marrow-derived DCs isolated from *rig-I⁻/⁻* or wild type mice produced similar levels of IFN after delivery of poly(I:C) into the cytoplasm. In contrast, after stimulation with poly(I:C), production of IFN-α/β by bone-marrow DCs, macrophages, or fibroblasts derived from *mda-5⁻/⁻* mice was severely impaired. Fibroblasts from *mda-5⁻/⁻* mice responded normally to dsRNAs transcribed in vitro, while cells from *rig-I⁻/⁻* mice did not. These dsRNAs were produced by in vitro transcription of the mouse lamin A/C gene to produce (+) and (−) strand RNAs, which were subsequently annealed. Therefore, it was concluded that MDA-5 recognizes poly(I:C) in vivo and in vitro, while RIG-I can recognize poly(I:C) in vitro, but not in vivo; the latter protein principally recognizes RNA transcripts.

IFN production after infection of cultured cells with paramyxoviruses, VSV, influenza virus, and Japanese encephalitis virus (a flavivirus) was impaired in MEFs from *rig-I⁻/⁻* mice. In contrast, RIG-I was dispensable for production of IFN in cells infected with EMCV. GMCSF-DCs from *rig-I⁻/⁻* mice failed to produce IFN after transfection with VSV RNA, but not after transfection of EMCV RNA. The authors concluded that in mice, RIG-I recognizes dsRNA produced during RNA virus replication, and not poly(I:C) (Kato et al., 2006).

The conclusion that the ligand for RIG-I is dsRNA was not consistent with the fact that many negative-strand RNA viruses, such as members of the paramyxovirus and orthomyxovirus families, do not produce dsRNA during their replicative cycles. It was subsequently discovered that RIG-I also recognizes RNA molecules with a 5′-ppp (Hornung et al., 2006; Pichlmair et al., 2006; Plumet et al., 2007). The triphosphate is found at the 5′-end of in vitro transcribed RNAs, explaining previous results indicating that these molecules are sensed by RIG-I. In cells infected with measles virus, the leader RNA, a small, uncapped RNA with 5′-ppp, activates RIG-I (Hornung et al., 2006). Similarly, the influenza virus single-stranded genomic RNA, which bears 5′-phosphates, is also a ligand for RIG-I (Pichlmair et al., 2006). Removal of the 5′-phosphates

does not affect RNA binding to RIG-I, but abrogates IFN induction. Cellular RNAs do not display a 5′-triphosphate: this structure is either removed or replaced with a 5′-cap structure. Therefore, ‘self’ RNAs do not activate RIG-I.

Infection of mice lacking the gene encoding MDA-5 revealed that this protein is critical for detection of infection with picornaviruses, but not flaviviruses, influenza viruses, or paramyxoviruses (Gitlin et al., 2006; Kato et al., 2006). How this selective recognition was achieved was explained by the finding that RIG-I, but not MDA-5, recognizes RNA with 5′-ppp. The 5′-ends of the RNA genomes and RNA replication intermediates of picornaviruses are not phosphorylated, but are covalently linked to a small protein, VPg. Such RNA molecules therefore would not be recognized by RIG-I. Picornavirus infections are sensed by MDA-5 in mice lacking RIG-I. However, in the absence of MDA-5, picornavirus genomes are not sensed. It is not known why dsRNAs produced during the replicative cycles of picornaviruses are not recognized by RIG-I — perhaps they are too long (see below). Although poly(I:C) and EMCV viral RNA can activate MDA-5, the actual ligand in virus-infected cells has not been identified. Candidates include the highly structured regions of RNA in the 5′- and 3′-noncoding regions of the viral genome.

Although RIG-I contains a domain with homology to RNA helicases, it does not have an RNA-binding motif, and therefore how it interacts with RNA was initially not understood. The C-terminal domain (amino acids 792–925) of recombinant RIG-I binds dsRNA (including short RNAs, ~25-bp) and 5′-ppp RNA within this RNA (Takahasi et al., 2008). The amino acid sequence of RIG-I indicates that it is a ligand-dependent ATPase (Cui et al., 2008; Gee et al., 2008; Takahasi et al., 2008). When the ATP binding site at amino acid 270 is altered, RIG-I cannot induce IFN synthesis, but it can still bind dsRNA and 5′-ppp RNA (Takahasi et al., 2008). Although RIG-I has RNA helicase activity, it is not required for triggering IFN synthesis — consistent with the fact that single-stranded 5′-ppp RNA is a functional ligand for RIG-I. The precise role of ATP in RIG-I activation is not known; it has been suggested to play a role in the structural change of the RNA–RIG-I complex to an active form (Yoneyama & Fujita, 2008).

The C-terminal domain (CTD) of RIG-I contains a repressor domain (aa 723–925), which can bind the CARD and helicase domain and inhibit the RIG-I-mediated signaling (Saito et al., 2007). The repressor domain is believed to act in *cis* by masking the CARD domain. A region that overlaps with the repressor domain (aa 792–925) was identified by limited protease digestion of RIG-I bound to RNA (Takahasi et al., 2008). This fragment is sufficient to bind dsRNA or 5′-ppp RNA. The atomic structure of the CTD reveals similarity to the mammalian suppressor of Sec4 (Mss4), a guanine nucleotide exchange factor that is not an RNA-binding protein (Cui et al., 2008; Takahasi et al., 2008). A positively-charged cleft on one side of the molecule has been suggested to be the RNA recognition surface. Alteration of amino acids within this cleft reduces RNA binding. Based on the observation that the CTD binds 5′-ppp RNA as a dimer, it has been proposed that one dsRNA molecule can bind multiple CTDs. Precise understanding of the RIG-I — ligand interaction awaits structural determination of the complex.

Amino acid changes in the basic cleft that inactivate RNA recognition do not produce constitutively active RIG-I, suggesting that the surfaces that bind RNA and repressor functions do not overlap. This finding, together with the observation that RNA recognition does not require ATP binding or hydrolysis, has led to a model in which the repressor domain maintains RIG-I in a closed, inactive conformation (Fig. 2). Upon binding the RNA ligand on the concave surface of the CTD, a conformational change (possibly requiring ATP) occurs, exposing the CARD domain, and allowing the CARD domain to initiate a signal transduction pathway leading to IFN synthesis.

The mechanism for the discrimination of RNA ligands by RIG-I and MDA-5 is not known. Both proteins have a CTD with common residues, although there are differences in surface charge that may influence substrate recognition (Yoneyama & Fujita, 2008).
Commercially available poly(I:C), which has been frequently used to study RNA sensing, is at least 3-kb in length. By producing poly (I:C) of various lengths it was determined that short segments of the polymer (~300-bp) do not activate MDA-5, but are potent ligands for RIG-I (Kato et al., 2008). In contrast, longer segments of poly(I:C) preferentially activate MDA-5. Consistent with these findings is the observation that the short dsRNA segments of the reovirus genome (1.2–1.4-kbp) selectively activate RIG-I, while the longer RNA segments (3.4-kbp) activate MDA-5. The mechanism underlying the discrimination between long and short dsRNA of MDA-5 and RIG-I remains to be elucidated. One possibility is that long segments of dsRNA bind RIG-I but induce a conformation of the protein that cannot engage in downstream signaling (Takahasi et al., 2008). This hypothesis is supported by the observation that binding of short dsRNA and 5′-ppp RNA to RIG-I leads to the formation of 17- and 30-kDa protease resistant fragments; in contrast, binding of long dsRNA causes formation of a 60-kDa protease resistant fragment (Takahasi et al., 2008).

It is likely that there are exceptions to the simple RNA recognition rules described above for MDA-5 and RIG-I. For example, it has been reported that IFN production occurs during infection with dengue and West Nile viruses in the absence of either RIG-I or MDA-5 (Fredericksen et al., 2008; Kato et al., 2006; Loo et al., 2008; Sumpter et al., 2005). These results suggest that perhaps other sensors of viral RNA remain to be discovered.

Collectively, studies to date indicate that RIG-I and MDA-5 can recognize different types of viruses, although there is significant evidence to also indicate possible redundant roles for these two helicases, probably due to the fact that some viruses are able to inadvertently produce both long and short types of dsRNA species following infection. Both RIG-I and MDA-5 require IPS-1 to direct innate immune defenses and are cooperatively essential for mounting an effective response to positive- and negative-stranded virus infection.

### 4.2. Induction of cytokine signaling and the role of adaptors Fas associated death domain/receptor-interacting serine-threonine kinase 1, interferon β promoter stimulator 1 and a new player stimulator of interferon genes

#### 4.2.1. Fas associated death domain/receptor-interacting serine-threonine kinase 1

While it was apparent that RIG-I and MDA-5 were important for intracellular recognition of viral RNAs, it was less obvious which molecules resided downstream of these helicases and facilitated their ability to activate IRF3, IRF7 and NF-κB, required for efficient type I IFN transcription. At approximately the same time as RIG-I was reported to be important in TLR-independent signaling, another study also reported the discovery of a TLR-independent innate immune pathway that involved the death domain-containing proteins FADD and RIP1 (Balachandran et al., 2004). FADD (Fas associated death domain) was previously isolated by its ability to interact with the death receptor Fas, and to mediate caspase-8 dependent cell death in response to FasL (Chinnaiyan et al., 1995). In contrast, RIP1 (receptor-interacting serine-threonine kinase 1) has been reported to play a key role in facilitating tumor necrosis factor-alpha (TNF-α) signaling and is recruited by TRADD (TNFR1-associated death domain protein) to the TNF receptor to induce NF-κB activation (Hsu et al., 1996). Both FADD and RIP1 can interact through death domain association, with TRADD to mediate TNF-α-induced apoptosis (Kelliher et al., 1998; Micheau & Tschopp, 2003). In the absence of FADD or RIP1, cells were also found to be defective in their ability to produce type I IFN (Balachandran et al., 2004; Balachandran et al., 2007). Normal MEFs lacking these death domain-containing molecules were extremely sensitive to RNA virus infection and also exhibited defective IFN-mediated antiviral gene activity through, amongst other things, a lack of IRF7 production. Subsequent studies indicated that FADD and RIP1 are essential for RIG-I- and MDA-5-

mediated signaling and likely formed an ‘innateosome’ complex with these helicases to facilitate the activity of NF-κB and IRF3 and IRF7 (Balachandran et al., 2004; Balachandran et al., 2007) (Fig. 3). Of interest is that insect homologues, referred to as IMD (for Rip1) and dFADD (for FADD) were also shown to play a critical role in TLR-independent signaling in Drosophila and were essential for protecting flies against gram-negative bacteria (Lemaitre et al., 1995; Georgel et al., 2001; Leulier et al., 2002; Naitza et al., 2002; Ferrandon et al., 2004). Insects lacking IMD or dFADD fail to activate the Drosophila IKK complex which is essential for the production of appropriate stress responses to antibacterial genes (Aggarwal & Silverman, 2008). Thus, FADD and RIP1 may be evolutionarily conserved death domain-containing molecules important in host defense against pathogen infection.

#### 4.2.2. Interferon β promoter stimulator 1 (IPS-1)

In addition to the studies described above related to FADD/RIP1, a number of groups in 2005 reported the isolation of a new adaptor molecule that also appeared to mediate RIG-I/MDA-5 function. One group utilized an expression cloning strategy to identify molecules that activated the IFN-β promoter and isolated a gene that they referred to as IFN-β promoter stimulator 1 (IPS-1) (Kawai et al., 2005). A second group isolated the same protein by screening uncharacterized molecules known to be potent inducers of NF-κB. They termed the protein VISA (for virus-induced signaling adaptor) (Xu et al., 2005). In contrast two other groups performed profile searches in human protein databases to identify new CARD-domain-containing proteins, similar to domains evident in RIG-I/MDA-5 and called the products Cardif (CARD adaptor inducing IFN-β) and MAVS (mitochondrial antiviral signaling) (Meylan et al., 2005; Seth et al., 2005). Herein we will term the protein IPS-1.

IPS-1 appears to be expressed in a variety of tissues and is 540 amino acids in length, predicting a protein of 65-kDa. IPS-1 was found to activate type I IFN transcription, the NF-κB promoter as well IP-10, RANTES, and ISRE promoters, that are also regulated by the IRF family. IPS-1 was also found to associate with RIG-I and MDA-5, through CARD domain interactions, as well as with FADD and RIP1, which as mentioned earlier, are also required to facilitate RIG-I/MDA-5 activity. RNAi knockdown of IPS-1 reduced the ability of poly(IC) and RIG-I/MDA-5 to induce type I IFN, but not TRIF, indicating that IPS-1 did not play a significant role in the TLR pathway (although one group did report that IPS-1 associated with TRIF and TRAF6 to mediate bifurcation of the TLR3-triggered NF-κB and IRF3 activation pathways) (Xu et al., 2005). IPS-1 was unable to activate the IFN-α and IFN-β promoters in cells deficient in both TBK1 and IKKi, indicating that it lies upstream of these kinases (Kawai et al., 2005; Meylan et al., 2005; Sun et al., 2006; Xu et al., 2005). Interestingly, IPS-1 was found to localize in the mitochondrial membrane, although little is known about protein partners that may associate with IPS-1 in this organelle (Sun et al., 2006). It also remains to be determined how IPS-1 is able to associate with RIG-I/MDA-5 in the cell following virus infection.

Biochemical analysis has indicated that the CARD domain of IPS-1 is essential for signaling activity, although a transmembrane domain was reported as being required for IPS-1 dimerization, which was also deemed essential for activity. It was further found that NS3–4A, a multifunctional protein of hepatitis C virus (HCV) that has serine protease activity essential for the production of mature viral proteins, targeted IPS-1 to subvert interferon mediated antiviral activity (Meylan et al., 2005). Thus, HCV-mediated resistance to IFN action may involve IPS-1 deregulation (Meylan et al., 2005). IPS-1−/− mice appeared viable and a variety of cells derived from these animals, such as MEFs, macrophages and conventional dendritic cells had impaired type I IFN responses to both RIG-I-recognized RNA viruses such as SeV, NDV, and VSV, as well as MDA-5-recognized viruses such as EMCV (Kumar et al., 2006; Sun et al., 2006). However plasmacytoid dendritic cells did not appear affected by the loss of IPS-1 and were able to produce IFN in response to SeV infection.
P.M. Barral et al. / Pharmacology & Therapeutics 124 (2009) 219–234

# Viral RNA and DNA-Mediated Induction of IFNα/β Transcription: The TLR-Independent Pathway.

---

## Figure 3. The RIG-I and MDA-5 signaling pathway. Short dsRNAs or 5′-triphosphate ssRNAs activate RIG-I which associates with mitochondrial IPS-1 to trigger FADD/RIP1 (innateosome) controlled activation of the type I interferon genes. Longer RNAs generated from picornaviruses trigger MDA-5-mediated activation of type I IFN. The RIG-I/IPS-1 pathway also requires the assistance of STING to activate type I IFN, which resides in the endoplasmic reticulum (ER). STING may regulate the translocon composed of the TRAP complex and Sec61, which in turn may regulate exocyst-mediated stimulation of the TBK1 pathway. STING also appears pivotal in regulating DNA-mediated triggering of type I IFN production.

---

### Text Below Figure:

**IPS-1 deficient animals were severely impaired in their ability to produce type I IFN as well as other cytokines such as IL-6 and IP-10, following infection with EMCV. Loss of IPS-1 was not found to affect TLR signaling, for example in response to LPS or exogenous poly(IC). Accordingly, IPS-1 deficient animals were extremely sensitive to lethal VSV infection and slightly more sensitive to EMCV infection, indicating the importance of this molecule in both RIG-I and MDA-5-mediated innate immune signaling (Kumar et al., 2006; Sun et al., 2006). While one group indicated that IPS-1 was partially required to facilitate DNA virus mediated signaling (vaccinia), other reports indicated that IPS-1 was not required for IFN induction by cytosolic DNA or *Listeria monocytogenes* (Kumar et al., 2006; Sun et al., 2006).**

In summary, the experimental data indicate that IPS-1 is a potent inducer of type I IFN following oligomerization and largely resides in the membrane of mitochondria. IPS-1 facilitates both RIG-I and MDA-5 signaling through CARD domain association and is essential for in vivo innate immunity to RNA, but not DNA virus infection. While the exact signaling mechanisms remain to be determined, it has recently been reported that TRADD may be recruited to IPS-1 to orchestrate ‘innateosome’ complex formation with the E3 ubiquitin ligase TRAF3 and TANK, and with FADD and RIP1, which culminates in the activation of IRF3 and NF-κB (Michallet et al., 2008; Hacker et al., 2006; Oganesyan et al., 2006). Further work will undoubtedly elucidate this complex series of events.

### 4.2.3. Stimulator of interferon genes (STING)

While significant progress has been made in unraveling the molecular components of TLR-independent innate signaling in response to RNA viruses, less was known about innate signaling in response to DNA virus

infection. While plasmacytoid dendritic cells were known to utilize TLR9 in the recognition of CpG-containing DNA, it was apparent that important TLR-independent pathways also existed to recognize DNA in alternative tissues, the mechanisms of action of which remain to be determined (Takeuchi & Akira, 2007). However, in 2008, a molecule referred to as STING (stimulator of IFN genes) was isolated using an expression cloning strategy to identify molecules that activated the IFN-β promoter (Ishikawa & Barber, 2008). This molecule was predicted to be 379-aa in length and comprised several putative transmembrane regions in the amino terminal region. Two other groups also isolated STING around this time (referred to as MITA and MYPS) (Jin et al., 2008; Zhong et al., 2008).

Confocal analysis indicated that STING was ubiquitously expressed, resided in the endoplasmic reticulum (ER) and following over-expression potently induced the activation of both NF-κB as well as IRF3 to stimulate type I IFN production (Ishikawa & Barber, 2008). STING deficient animals were generated and were viable and MEFs lacking STING were found to be sensitive to VSV and SeV infection. However, loss of STING did not appear to dramatically affect poly(IC) signaling. Indeed, co-immunoprecipitation experiments indicated that STING appeared to associate with RIG-I, rather than MDA-5, which may explain sensitivity to VSV but not poly(IC), which are dependent on RIG-I and MDA-5, respectively. IPS-1 was also found to associate with STING in co-immunoprecipitation experiments but it was not clear if this was a direct or indirect (complex) association. Of further interest was that STING was found by two-hybrid screening to associate with SSR2/TRAP, a member of the TRAP complex comprising four subunits (α-D) that facilitates translocation of proteins into the ER following translation (Hartmann et al., 1993; Ishikawa & Barber, 2008; Lipschutz et al., 2003; Menetret et al., 2005).
The TRAP complex is known to associate with the ‘translocon’, a protein complex that forms an aqueous pore spanning the ER membrane, that consists of three copies of the SEC61 heterotrimer. Thus, STING is predominantly an ER resident protein that may link RIG-I mediated intracellular innate signaling to the ‘translocon’. Speculatively, RIG-I may detect translating viral RNAs at the intersection of ribosome/ER ‘translocon’ association and require STING to exert effective innate immune function. The mitochondrial network is also known to associate with the ER and such mitochondria-associated membranes (MAM) may provide direct physical contact between the ER and mitochondria, thus facilitating a role for IPS-1 in the ‘innateosome’ complex ([Bozidis et al., 2007](https://doi.org/10.1083/jcb.200609054); [Hartmann et al., 1993](https://doi.org/10.1083/jcb.123.6.1543); [Ishikawa & Barber, 2008](https://doi.org/10.1083/jcb.200804179); [Lipschutz et al., 2003](https://doi.org/10.1083/jcb.200304080); [Menetret et al., 2005](https://doi.org/10.1083/jcb.200504082)). Alternatively, STING may participate in mediating ER stress response pathways, but this remains to be verified.

Although it is not clear how signaling from the ‘translocon’ to IRF3/NF-κB occurs, it has recently been established that the ‘translocon’ physically associates with the ‘exocyst’ — the octameric Sec6–Sec8 complex that also associates with the ER and tethers secretory vesicles to membranes, and facilitates protein synthesis and secretion ([Guo & Novick, 2004](https://doi.org/10.1083/jcb.200404080); [Lipschutz et al., 2003](https://doi.org/10.1083/jcb.200304080)). Recently, the ‘exocyst’ complex was found to recruit and activate TBK1 and play a role in type I IFN-β induction ([Chien et al., 2006](https://doi.org/10.1083/jcb.200604080)). Preliminary analysis indicated that STING co-immunoprecipitated with TBK1, and that RNAi ablation of Sec5 also rendered cells defective in the STING function.

Importantly, the generation of STING deficient MEFs and macrophages yielded the surprising finding that STING was essential for recognizing non CpG DNA species, such as short double-stranded DNA oligonucleotides that are able to potently induce IFN (ISD), as well as bacterial DNA and poly d(AdT). Thus, STING may facilitate the detection of intracellular viral RNA and DNA species as well as B-form types of DNA in general, indicating convergence of these intracellular pathogen-associated molecular pattern (PAMP) recognition pathways.

### 4.2.4. RNaseL and the antiviral response

The activation of signaling pathways by double-stranded RNA molecules leads to the expression of IFNs and these, in turn, initiate expression of IFN-induced genes (ISGs). Double-stranded RNA is sensed by RIG-I and MDA-5. It had generally been considered that the RNA molecules arose as a by-product of viral infection; however, recent evidence suggests that cellular generated small RNAs can amplify the antiviral response ([Malathi et al., 2007](https://doi.org/10.1083/jcb.200704080)). Specifically, it has been shown that the antiviral endoribonuclease, RNaseL, is activated by 2′, 5′ oligoadenylate, and produces small RNA fragments from cellular RNA which contribute to IFN production and this involves RIG-I, MDA-5 and IFN stimulator protein 1 (IPS-1) and IRF3 ([Malathi et al., 2007](https://doi.org/10.1083/jcb.200704080)). Using normal and RNaseL deficient mice, viral infection produces appreciably more IFN-β if RNaseL is expressed ([Malathi et al., 2007](https://doi.org/10.1083/jcb.200704080)). Similarly the introduction of 2′–5′ oligoadenylate into normal mice, but not RNaseL null mice, caused a marked increase in levels of circulating IFN. It has previously been demonstrated that activation of RNaseL by 2′–5′ oligoadenylate increases the expression of a number of IFN-stimulated genes ([Malathi et al., 2005](https://doi.org/10.1083/jcb.200504080)). Although RNaseL is stimulated by cellular (self) RNAs, it is interesting to note that viral dsRNAs activate 2′–5′ oligoadenylate synthetase, which causes production of 2′–5′ oligoadenylate, activating RNaseL. RNaseL cleaves single-stranded RNA resulting in double-stranded (duplex) structures, which can then activate signaling by RIG-I and MDA-5, producing IFN.

## 5. Negative regulation of the signaling pathway

Limitation of IFN production is an essential physiological requisite necessary for the overall well-being of the organism. Following initiation of antiviral and antiproliferative responses or the activation of innate immunity, restriction of production of excess IFN must occur.

The fact that there exist multiple mechanisms to control IFN levels confirms the importance of counteracting deleterious effects of IFN, that include chronic cellular toxicity and the initiation of inflammatory or autoimmune diseases.

A number of mechanisms contribute to the regulation of the RIG-I signaling pathway to limit its action under steady-state conditions. In particular, a number of negative regulators of RIG-I signaling have been characterized, although significantly fewer have been identified for MDA-5. Fundamentally, some of these mechanisms contribute to a negative feedback loop whereas others regulate steady-state levels.

### 5.1. Inhibition of retinoic acid-inducible gene-I/melanoma differentiation associated gene-5 by laboratory of genetics and physiology 2

Of the recently characterized novel regulators of RIG-I and MDA-5 signaling, most appear to function to maintain a tight control of virus-initiated IFN production. One such negative regulator is a member of the RLH family, LGP2, unlike RIG-I and MDA-5, lacks the N-terminal CARD required for activating IPS-1-dependent signaling events (see below). It has also been observed that RIG-I-mediated production of IFN can lead to an increase in RIG-I transcription, initiating an IFN amplification loop.

Under normal physiological conditions, RIG-I exists in an auto-repressed state determined by its repressor domain (RD) located at the end of the C-terminus ([Saito et al., 2007](https://doi.org/10.1083/jcb.200704080)). Besides this autoinhibition, which occurs in the absence of stimuli and controls the basal activity of RIG-I, several proteins have been found to have an inhibitory role in the RIG-I-mediated signaling pathway. LGP2 was the first protein characterized as an inhibitor of the RIG-I/MDA-5 signaling pathways. It exerts this function in different ways. One mechanism is by sequestering ligands from RIG-I/MDA-5 recognition. In fact LGP2 is also a DExD/H RNA helicase, which has a high homology with RIG-I and MDA-5 in the C-terminal region, but is completely devoid of the N-terminal region containing the effector CARD domain. Therefore, it can bind to RNA through its CTD, competing with RIG-I and MDA-5, but, lacking the N-terminal CARD, cannot interact with the adaptor protein IPS-1, losing the ability to transmit the signal ([Rothénfusser et al., 2005](https://doi.org/10.1083/jcb.200504080); [Yoneyama et al., 2004](https://doi.org/10.1083/jcb.200404080)). Another way in which LGP2 exerts an inhibitory role on RIG-I is mediated by the presence of a RD located in the C-terminal region, analogous to that present in RIG-I. This domain associates with the CARD domain and with a linker region between the CARD and the RNA helicase domain and causes RIG-I dimerization that is necessary for its activation and binding with the downstream protein IPS-1. It was found that, through this analogous domain, LGP2 associates with RIG-I, acting in trans preventing RIG-I dimerization necessary for its activation ([Saito et al., 2007](https://doi.org/10.1083/jcb.200704080)). LGP2 was also found to interact with the C-terminal and the transmembrane regions of IPS-1 competing for binding of the downstream kinase Ikki/Ikke essential for IRF3 activation ([Komuro & Horvath, 2006](https://doi.org/10.1083/jcb.200604080)). Therefore, LGP2 acts at two levels in the signaling pathway attenuating the production of IFN. Interestingly LGP2 is also an IFN-inducible gene and it is induced by viral infection playing a negative feedback role in the antiviral signaling loop.

Interestingly LGP2⁻/⁻ mice are more resistant to VSV infection than wt animals, but are more susceptible to encephalomyocarditis (EMCV) infection ([Venkataraman et al., 2007](https://doi.org/10.1083/jcb.200704080)). VSV produces RNA intermediates during replication, which are specifically recognized by RIG-I, whereas EMCV infection causes the activation of MDA-5. It might therefore be postulated that LGP2 is a positive regulator of MDA-5 mediated signaling pathways.

### 5.2. Inhibition by RNF125

The RIG-I and MDA-5 signaling pathway is also negatively regulated by the RING finger E3 ubiquitin ligase RNF125 ([Arimoto et al., 2007](https://doi.org/10.1083/jcb.200704080)). This protein is IFN-inducible and by conjugating ubiquitin to RIG-I and MDA-5 causes their proteosomal degradation thereby decreasing their
cellular levels. By mutational analysis it was shown that the N-terminal region of RIG-I, which contains the CARD domain, is the main substrate for the RNF125 protein, with minimal ubiquitination occurring in the C-terminal region. Overexpression of RNF125 in virus-infected or dsRNA-stimulated cells decreases IFN-β production, whereas its reduction, mediated by siRNA knockdown, augments IFN-β expression. RNF125 also acts downstream of RIG-I and MDA-5, by targeting, for proteosomal degradation, the adaptor protein IPS-1. It is now clear that ubiquitination is not only a mechanism for targeting proteins for proteosomal degradation (usually through Lys 48-linked polyubiquitination) but can also play a role in promoting protein interactions or catalytic activation, regulating their functions (through Lys 63-linked ubiquitination) ([Liu et al., 2005](https://doi.org/10.1074/jbc.M509350200)). Ubiquitination is a reversible process in which the action of ubiquitin-conjugating enzymes is counteracted by de-ubiquitinating enzymes (DUB). In the case of RIG-I besides being ubiquitinated by RNF125 for proteosomal degradation, it is also ubiquitinated by TRIM25 causing activation.

### 5.3. Inhibition by de-ubiquitinating enzymes

Several de-ubiquitinating (DUB) enzymes have also been shown to regulate the level of ubiquitinated RIG-I, consequently acting as negative regulators of the RIG-I signaling pathway. The anti-apoptotic protein A20, a known inhibitor of NF-κB, blocks the RIG-I-mediated signaling ([Lin et al., 2006](https://doi.org/10.1074/jbc.M602538200)). A20 is a peculiar protein that contains an ovarian tumor (OTU) domain at its N-terminus that has a de-ubiquitinating function and a C-terminal ubiquitin ligase domain. Both of these regions are necessary for inhibition of NF-κB ([Wertz et al., 2004](https://doi.org/10.1038/nature02611)), but only the C-terminal region is responsible for inactivation of RIG-I signaling. Stable transfection of the constitutively active domain of RIG-I confers more resistance to VSV replication and this effect is reversed by the addition of A20. In other studies, it has been shown that the de-ubiquitinating enzyme A (DUBA) removes the lysine-63-linked polyubiquitin chain from the TNF receptor-associated factor 3 (TRAF3), resulting in the dissociation of TRAF3 from TANK-binding kinase 1 (TBK1) and consequently disrupting the signal ([Kayagaki et al., 2007](https://doi.org/10.1074/jbc.M702884200)). DUBA is another member of the ovarian tumor (OTU) family of de-ubiquitinating enzymes and was identified in a siRNA-based screen as an enzyme able to negatively regulate IFN production in the antiviral pathway. Knocking-down DUBA by siRNA leads to more IFN production and its over-expression has the opposite effect. The tumor suppressor CYLD (cylindromatosis), another OTU de-ubiquitinating enzyme that removes Lys 63-linked polyubiquitin chains, was also recently shown to negatively regulate RIG-I ([Friedman et al., 2008](https://doi.org/10.1074/jbc.M802775200); [Zhang, Wu, et al., 2008](https://doi.org/10.1074/jbc.M802775200)). CYLD negatively regulates the signaling function of RIG-I by direct interaction with RIG-I causing its de-ubiquitination. It also acts as a de-ubiquitinating enzyme of TBK1, the kinase that phosphorylates IRF3, leading to inactivation of IRF3 signaling pathway and IFN production. RIG-I is ubiquitinated in its CARD domain by the E3 ubiquitin ligase TRIM25, and this is necessary to render RIG-I active. CYLD acts as a de-conjugating enzyme, removing the K63 ubiquitin chain and inactivating the signal.

### 5.4. Inhibition by interferon-stimulated gene 15

Another protein that has an inhibitory effect on RIG-I signaling pathway is the ubiquitin-like IFN-stimulated gene 15 (ISG15). ISG15 was found to interact with RIG-I ([Zhao et al., 2005](https://doi.org/10.1074/jbc.M509350200)) and later it was shown that RIG-I ISGylation negatively regulates RIG-I ([Kim et al., 2008](https://doi.org/10.1074/jbc.M802775200)). ISG15 is a ubiquitin-like protein, which is strongly induced by IFNs during viral infection ([Narasimhan et al., 2005](https://doi.org/10.1074/jbc.M509350200)).

### 5.5. Inhibition of melanoma differentiation associated gene-5 by dihydroxyacetone kinase

Using a yeast two-hybrid screen the dihydroxyacetone kinase (DAK) was shown to interact with MDA-5 and to have a specific inhibitory effect on MDA-5-mediated IFN signaling ([Diao et al., 2007](https://doi.org/10.1074/jbc.M702884200)). It was observed that overexpression of DAK inhibits activation of IRF3 and the IFN-β promoter, but does not inhibit the NF-κB activation pathway. Although MDA-5 and RIG-I are very similar in domain structure and function, DAK does not interact with RIG-I and does not inhibit the RIG-I-mediated signaling pathway. The interaction of DAK with MDA-5 is disrupted during viral infection, suggesting that DAK is a physiological inhibitor of MDA-5.

### 5.6. Inhibition by the autophagy complex Atg12–Atg5

The RIG-I/MDA-5 signaling pathway is also inhibited by Atg12–Atg5, a protein conjugate, which plays a key role in regulating autophagy ([Jounai et al., 2007](https://doi.org/10.1074/jbc.M702884200)). Specifically, Atg-deficient embryonic mouse fibroblasts (MEFs) are more resistant to VSV infection and produce more IFN than wt MEFs. Regulation of IFN production by the Atg12–Atg5 complex depends on binding to the CARD of RIG-I, MDA-5 and IPS-1. In an analogous fashion, Atg7−/− MEFs are also resistant to VSV infection, indicating that Atg12–Atg5 complex formation, which requires Atg7, is essential for inhibition of signaling.

### 5.7. Inhibition by NLRX1

NLRX1 is a member of the nucleotide-binding domain and leucine-rich repeat containing (NLR) protein family ([Moore et al., 2008](https://doi.org/10.1074/jbc.M802775200)). NLRX1 is a widely expressed protein that is generally localized to the outer mitochondrial membrane and has recently been shown to bind to IPS-1. The CARD domain on IPS-1 determines this interaction through a proposed nucleotide-binding domain in NLRX1 and results in marked inhibition of RIG-I and MDA-5 mediated IFN release. Thus, NLRX1 can be viewed as a negative regulator of RIG-I and MDA-5 activity such that depletion of NLRX1 with siRNAs results in increased virus-induced IFN production ([Moore et al., 2008](https://doi.org/10.1074/jbc.M802775200)). NLRX1 contains a mitochondrial targeting sequence, and it is notable that deletion of the leucine-rich repeat results in loss of the IPS-1 inhibitory activity. It appears that NLRX1 competes with RIG-I for IPS-1 possibly through interaction between the CARDs. As the level of NLRX1 does not change in response to IFN treatment or MDA-5 and RIG-I expression it appears that it acts to regulate steady-state antiviral responses.

### 5.8. Inhibition by caspase-1

In a recent study it was demonstrated that caspase-1, which functions as an inflammatory caspase and whose expression is activated during viral infection, negatively regulates RIG-I signaling activity by promoting RIG-I secretion, consequently controlling its intracellular levels ([Kim & Yoo, 2008](https://doi.org/10.1074/jbc.M802775200)). It is not clear how RIG-I is secreted, but since RIG-I interacts with caspase-1 and it was found in the supernatant together with caspase-1, one hypothesis is that RIG-I secretion involves the ‘inflammasome’ complex. However, the physiological significance of RIG-I secretion is not yet clear.

Another recent report describes a RIG-I splicing variant (RIG-I SV) which was found to be induced during viral infection or upon IFN stimulation and shown to suppress RIG-I signaling ([Gack et al., 2008](https://doi.org/10.1074/jbc.M802775200)). The splicing variant lacks residues 36 to 80 located in the first CARD, therefore it cannot interact with TRIM25 or be ubiquitinated in the second CARD, losing its ability to bind the adaptor protein IPS-1 and consequently to carry out signal transduction. RIG-I-SV acts as a dominant inhibitor of the RIG-I-mediated antiviral response. In fact, RIG-I SV interacts with RIG-I WT inhibiting its multimerization and consequent interaction with IPS-1. This inhibition of virus-induced IFN signal transduction could potentially serve as a negative feedback mechanism.

6. Cell type specificity of retinoic acid-inducible gene-I and melanoma differentiation associated gene-5

In vivo evaluation of the importance of RIG-I and MDA-5 in recognizing virus infection came through the generation of viable knockout mice that lacked these genes. *rig-I*⁻/⁻ MEFs did not significantly produce type I IFN in response to a variety of negative-stranded RNA viruses, such as paramyxoviruses, rhabdoviruses and orthomyxoviruses, since the major mechanism of 5′-triphosphate viral RNA recognition was absent (Kato et al., 2005). Conventional dendritic cells (cDCs) lacking *rig-I* (but not MyD88⁻/⁻TRIF⁻/⁻ double knockout cDCs), induced from bone-marrow in the presence of GMCSF, were also defective in their ability to manufacture type I IFN following exposure to the paramyxovirus, Newcastle disease virus (NDV). However, plasmacytoid dendritic cells (pDCs) lacking RIG-I were able to normally generate type I IFN following NDV infection through activation of the TLR pathway since MyD88⁻/⁻TRIF⁻/⁻ defective pDCs had severely impaired type I IFN responses following similar virus exposure (Kato et al., 2005, 2006). Despite this, *rig-I*⁻/⁻ mice remained sensitive to lethal infection by the above viruses. In contrast, macrophages, cDCs and MEFs lacking MDA-5 were impaired in their ability to produce type I IFN in response to the positive-stranded virus EMCV (but not the above negative-stranded viruses) as well as poly(IC) (Kato et al., 2006; Gitlin et al., 2006). pDCs from MDA-5 deficient animals exhibited normal ability to induce type I IFN following exposure to EMCV, in contrast to pDCs lacking MyD88. Nevertheless, mice lacking MDA-5 remained highly susceptible to picornavirus infection. These data largely confirmed concomitant in vitro RNA-binding analysis performed on these helicase proteins (as described earlier) (Takeuchi & Akira, 2008; Takeuchi & Akira 2007). For example, EMCV generates longer RNA species that are replicate intermediates compared to negative-stranded viruses, which are recognized by MDA-5 rather than RIG-I. However, it is worth noting that some viruses, such as the flaviviruses, dengue virus and Japanese encephalitis virus, were able to activate both the MDA-5 and RIG-I pathways in MEFs, probably because these viruses generated both long and short viral dsRNA structures following infection (Loo et al., 2008). These studies indicate that RIG-I and MDA-5 play a pivotal role in recognizing virus infection in most cell types, except in pDCs, where the TLR (predominantly TLR 7) pathway appears to be the important mechanism in manifesting antiviral host defense, for reasons that remain unknown.

7. Viral countermeasures

The various defense mechanisms that operate in healthy hosts have evolved for millions of years, yet are imperfect because viral genomes encode gene products that block every step of host defense. The RNA sensing pathways described in this review are no exception. Viral gene products may antagonize nearly every step of the sensing of RNA by RIG-I and MDA-5 (Fig. 4). Understanding such viral countermeasures not only improves our understanding of innate sensing pathways, but may also suggest avenues for therapeutic intervention.

7.1. Interference with ribonucleic acid sensing

As described previously, host RNAs are not detected by RIG-I or MDA-5 because they do not contain 5′-phosphates. Many viral RNAs are also capped, either by cellular or viral capping enzymes, or, in the case of influenza viruses, when the viral polymerase primes the synthesis of mRNAs with capped fragments derived from the 5′-ends of host mRNAs (Plotch et al., 1981). A phosphatase is produced during infection with certain negative-strand RNA viruses (Borna disease virus, Crimean-Congo Hemorrhagic fever virus, Hantaan virus) that converts the 5′-ppp on viral mRNAs to a monophosphate (Habjan et al., 2008), thereby preventing detection by RIG-I. The genomic RNA of picornaviruses is covalently linked to a viral protein, VPg (Flanegan et al., 1977; Lee et al., 1977), which also prevents activation of RIG-I. These modifications of viral RNAs can all be viewed as strategies to avoid recognition by RIG-I.

Some viral genomes encode proteins that bind dsRNA and prevent their detection by RIG-I and MDA-5 in the cytoplasm. These include the vaccinia virus E3L protein (Smith et al., 2001; Xiang et al., 2002), Ebola virus VP35 (Haasnoot et al., 2007), and human cytomegalovirus proteins TRS1 and m142/m143 (Cassady, 2005; Child et al., 2004, 2006; Hakki & Geballe, 2005; Valchanova et al., 2006).

Infection of cells with members of the picornavirus family leads to degradation of MDA-5 and RIG-I. Cleavage of MDA-5 is carried out by cellular caspases, which are activated by virus infection (Barral et al., 2007; Rebsamen et al., 2008). Cleavage of MDA-5 can also be accomplished in the absence of virus infection by inducing apoptosis pharmacologically (Kovacsovics et al., 2002; Barral et al., 2007). In contrast, the viral 3C<sup>Pio</sup> protease cleaves RIG-I directly in cells infected with poliovirus, rhinovirus, and other picornaviruses (Barral et al., submitted). It is not known if sensor cleavage plays a role in attenuating IFN response during picornavirus infection. Cleavage of RIG-I is a somewhat puzzling event because it is assumed that picornavirus infections are sensed by MDA-5. Perhaps cleavage of RIG-I/MDA-5 antagonizes other functions of the sensor proteins.

Some viral proteins have been shown to bind RIG-I or MDA-5, thereby blocking the induction of IFN. The V proteins of paramyxoviruses bind MDA-5 and block IFN induction (Andrejeva et al., 2004; Childs et al., 2007). The C-terminal domain of V proteins binds a specific region of MDA-5, but not RIG-I; such binding prevents dsRNA binding, dimerization, and activation of MDA-5 (Childs et al., 2008). However, the significance of such antagonism is not known, because mice lacking the gene encoding *mda-5* are as susceptible to paramyxovirus infection as wild type mice, and mount similar IFN responses (Gitlin et al., 2006; Kato et al., 2006), and RIG-I appears to be the sensor for paramyxovirus infections. Perhaps the conserved MDA-5 binding ability of paramyxovirus V proteins targets a different activity of the protein. The NS1 protein of influenza A viruses binds RIG-I (Pichlmair et al., 2006; Mibayashi et al., 2007) and inhibits signaling and IFN induction (Guo et al., 2007; Opitz et al., 2007). As expected, viruses that do not produce the NS1 protein produce high levels of IFN in infected cells (Talon et al., 2000). The G transmembrane glycoprotein of human metapneumovirus, a paramyxovirus, also binds RIG-I and blocks IFN-β induction (Bao et al., 2008). There is some evidence that these viral proteins might not directly interact with RIG-I, and the mechanism of interference has not been elucidated.

7.2. Interference with signal transduction

The downstream adaptor protein IPS-1 is cleaved in cells infected with a variety of viruses. As a consequence, IRF3 phosphorylation does not occur and IFN transcription is attenuated. The 3C<sup>Pio</sup> protease of the picornavirus hepatitis A virus degrades IPS-1 only when part of the hydrophobic precursor protein 3ABC, which targets the protease to the mitochondrial membrane (Yang et al., 2007). The NS3/4A protease of hepatitis C virus cleaves within the transmembrane domain of IPS-1, causing its release from the mitochondrion (Meylan et al., 2005; Lin et al., 2006). These examples demonstrated that IPS-1 must be present in the mitochondrial membrane to function in the signaling pathway leading to IFN synthesis (Seth et al., 2005).

Members of the signaling pathway downstream of IPS-1 are also targets of viral antagonism. The cascades that begin with ligand-dependent activation of RIG-I and MDA-5 converge at the IKK family of proteins (Fig. 4) whose role is to phosphorylate NF-κB, IRF3, and IRF7. Members of the IKK protein family are targets of viral antagonism. The G1 protein of hantavirus disrupts the interaction of TRAF3 with TBK1, thereby interrupting signaling (Alff et al., 2008). The NS3/4A protease of hepatitis C virus and the N1L protein of vaccinia

P.M. Barral et al. / Pharmacology & Therapeutics 124 (2009) 219–234

![](image.png)

Fig. 4. Viral countermeasures in the RNA sensing and signaling pathway. The steps leading from detection of RNA by RIG-I and MDA-5 through signaling and transcription of IFN genes are depicted. Viral gene products that interfere at different steps of this pathway are listed. An arrow indicates stimulation, and a bar indicates inhibition of the pathways.

virus interact with TBK1, blocking phosphorylation of downstream transcription proteins ([DiPerna et al., 2004](https://doi.org/10.1073/pnas.0406846101); [Otsuka et al., 2005](https://doi.org/10.1038/nature03908)). The consequence of inhibiting TBK1 activity is failure to phosphorylate IRF3, which cannot enter the nucleus to stimulate IFN transcription. The vaccinia virus K7R protein also prevents the activation of IRF3 and IRF7 by the TBK1–IKKe ([Schroder et al., 2008](https://doi.org/10.1016/j.cell.2008.02.039)). The K7R protein binds the human DEAD box helicase DDX3, which is part of the TBK1–IKKe complex and is required for the activation of the IFN-β promoter ([Soulat et al., 2008](https://doi.org/10.1016/j.molcel.2008.02.017)).

### 7.3. Inhibition of interferon regulatory factors

The transcription proteins IRF3 and IRF7 are prime targets for inhibition by a variety of viral proteins. The papain-like protease of SARS virus binds IRF3 and prevents dimerization, thereby inhibiting nuclear translocation ([Devaraj et al., 2007](https://doi.org/10.1073/pnas.0700463104)). Other viral proteins act as IRF mimics. For example, the V proteins of paramyxoviruses bind TBK1–IKKe and compete with IRF3, thereby inhibiting its phosphorylation ([Lu et al., 2008](https://doi.org/10.1073/pnas.0711985105)). The genome of Kaposi's sarcoma-associated herpesvirus (KSHV) forms dimers with IRF7, blocking its ability to bind DNA ([Joo et al., 2007](https://doi.org/10.1073/pnas.0700463104)). Several human herpesvirus 8 proteins are dominant-negative inhibitors of IRF3, preventing its association with the co-activators CBP and p300 ([Lin et al., 2001](https://doi.org/10.1073/pnas.011596698)). The transcription

protein K-bZIP of KSHV blocks the activation of the IFN-β promoter by competing with IRF3 for binding sites in the promoter ([Lefort et al., 2007](https://doi.org/10.1073/pnas.0700463104)), while ICP0 of HSV binds IRF3 and localizes the protein in nuclear bodies where it cannot activate the IFN-β promoter ([Melroe et al., 2007](https://doi.org/10.1073/pnas.0700463104)).

Another strategy to prevent IRF mediated transcriptional activation is proteolytic degradation. The ICP0 protein from bovine herpesvirus degrades IRF3 instead of sequestering it like the HSV orthologue ([Saira et al., 2007](https://doi.org/10.1073/pnas.0700463104)). Degradation of IRF3 is also brought about by the Vpr and Vif proteins of HIV-1, the N protease of flaviviruses, and the NSP1 protein of rotaviruses; the latter also induces degradation of IRF7 ([Barro & Patton, 2007](https://doi.org/10.1073/pnas.0700463104); [Bauhofer et al., 2007](https://doi.org/10.1073/pnas.0700463104); [Okumura et al., 2008](https://doi.org/10.1073/pnas.0700463104)). Of great interest is the observation that the vaccinia virus E3L protein disables the cellular ISG15 protein ([Guerra et al., 2008](https://doi.org/10.1073/pnas.0700463104)). ISG15 is an ISG that is believed to counter virus-induced proteolysis of IRF3; for example, it can prevent degradation of IRF3 by paramyxovirus V protein ([Lu et al., 2006](https://doi.org/10.1073/pnas.0700463104)). The vaccinia virus E3L protein therefore disables the cell's attempt to block a viral antagonism of the innate immune response.

NF-κB function is also modulated during virus infection. This complex is not only involved in the production of IFN-β and chemokines, but also inhibits apoptosis and causes cell proliferation, which may benefit viral replication. Consequently, in some viral infections, NF-κB
activity is blocked early in the replicative cycle, to prevent IFN-mediated clearing of infection. The A238L protein of African swine fever virus acts as a homolog of IκBα; it binds NF-κB and prevents its entry into the nucleus (Tait et al., 2000). Later in infection, the viral A224L protein activates NF-κB, causing inhibition of apoptosis and cellular proliferation. Why cleavage of the p65-RelA subunit of NF-κB by poliovirus proteinase 3C<sup>pro</sup> at a late stage in infection would be beneficial to virus replication is not clear (Neznanov et al., 2005). Perhaps the concomitant stimulation of apoptosis helps stimulate virus release from cells (Jackson et al., 2005).

Other viral proteins regulate NF-κB by disrupting IKK. The B14R protein of vaccinia virus binds IKKβ and prevents its phosphorylation and therefore activation (Chen et al., 2008), while the K13 protein of KSHV binds the IKKα–IKKβ complex in a way that causes activation of NF-κB (Matta et al., 2007). Curiously, cleavage of IκBα by the 3C<sup>pro</sup> proteinase of the picornavirus coxsackivirus B3 limits NF-κB mediated signaling, but leads to reduced viral replication. The cleaved IκBα binds NF-κB, which enters the nucleus but cannot transactivate; therefore apoptosis increases, limiting viral replication (Zaragoza et al., 2006). The authors suggest that IκBα acts as a sensor for viral replication.

### 8. Apoptosis and growth suppression

mda-5 was cloned as a transcript induced during induction of terminal differentiation of human melanoma cells and initiates an irreversible growth arrest program in these cancer cells (Kang et al., 2002). Indeed, the first report on cloning and characterization of mda-5 revealed its growth suppressor properties (Kang et al., 2002). Follow-up studies demonstrated that adenovirus-mediated delivery of mda-5 induced apoptosis that could be inhibited by active Ras/Raf pathway (Kang et al., 2004; Lin et al., 2006). In genetically modified rodent fibroblasts as well as in human pancreatic and colorectal cancer cell lines, constitutively active Ras/Raf/MEK/ERK pathway precluded mda-5-induced apoptosis that could be reversed by antisense inhibition of Ras or chemical inhibition of the ERK pathway (Lin, Lacoste, et al., 2006; Lin, Su, et al., 2006; Lin, Yang, et al., 2006). Deletion analysis revealed that compared to the full-length mda-5, deletion of either the CARD or the helicase domain significantly abrogated the growth suppressor properties of mda-5 (Kang et al., 2004). This loss-of-function was more pronounced when the CARD domain was deleted compared to the deletion of the helicase domain indicating that CARD domain is the active domain of the molecule, an observation later confirmed for RIG-I.

The cloning of human mda-5 was subsequently followed by cloning of the murine mda-5 gene, named Helicard (Kovacsovics et al., 2002). Apoptotic stimuli, such as FasL, resulted in cleavage of MDA-5/Helicard by caspases with subsequent nuclear translocation of the helicase domain, the consequences of which remain to be determined. MDA-5/Helicard augmented FasL-mediated DNA degradation while a mutant, resistant to caspase-cleavage, and a second mutant, with loss of helicase activity, lost this activity. However, MDA-5/Helicard alone did not induce apoptosis in 293T cells. Caspase-mediated cleavage of human MDA-5 has also been observed in poliovirus-infected cells (Barral et al., 2007).

RH116, having 99.5% identity to mda-5, was cloned as a gene downregulated by Murabutide, an HIV-suppressive immunomodulator (Cocude et al., 2003). Overexpression of RH116 inhibited growth, but did not induce apoptosis and the mechanism of this growth inhibition was not analyzed. Interestingly, counterintuitive to the antiviral function of MDA-5, RH116 actually increased HIV-1 replication. HIV-1 treatment itself induced RH116 expression, which was localized predominantly in the nucleus. Thus, although it is accepted that MDA-5 inhibits growth, there is debate about induction of apoptosis by MDA-5 and possible differential effects of the human vs. the murine version of this gene. The lack of follow-up studies on Helicard or RH116 precludes resolution of this issue at the present time.

Similar to MDA-5, RIG-I also displays apoptosis-inducing properties. Cytosolic poly(I:C) or influenza A virus infection activates caspases 1 and 3 that results in pro-IL-18 processing (Rintahaka et al., 2008). Overexpression of RIG-I or its active form (only the CARD domains) or IPS-1 activated caspases, processed pro-IL-18 and induced apoptosis in HaCaT keratinocytes cells. The reverse experiment, i.e., inhibition of *rig-I* to block these phenomena, was not carried out to strengthen the direct link of RIG-I in virus-induced apoptosis. The apoptosis induction by a synthetic RA (CD437) was shown to be inhibited by the hepatitis C virus (HCV) non-structural (NS)3/4A protease, which is know to cleave RIG-I, thereby indirectly showing an involvement of RIG-I in apoptosis induction (Pan et al., 2008). Infection of cells with Sendai virus activates RIG-I and that leads to the activation of IRF3. IRF3 induces the expression of virus stress-inducible genes and causes host cell apoptosis by activating caspase-8 (Peters et al., 2008). A dominant-negative inhibitor of RIG-I, containing only the helicase domain, inhibits IRF3 activation, prevents apoptosis and allows persistent virus infection. A novel strategy was used to capitalize on specific properties of RIG-I as an anti-cancer therapy (Poeck et al., 2008). RIG-I, not MDA-5, is activated by 5′-triphosphates. An siRNA, with 5′-triphosphate ends, was generated against Bcl-2. The recognition of 5′-triphosphate by RIG-I activated innate immune cells, such as dendritic cells and directly induced expression of IFNs and apoptosis in tumor cells. These RIG-I-mediated effects synergized with Bcl-2 siRNA to induce apoptosis of tumor cells in lung metastasis in animal models. The observation that systemic administration of this bifunctional RNA molecule resulted in tumor-specific apoptosis offers promise of potential therapeutic use of this approach.

### 9. Role of retinoic acid-inducible gene-I in development and differentiation

As has been discussed above, the best-characterized role of RIG-I is in viral RNA-induced type 1 IFN generation. However, recent evidence has suggested that it can act, in the absence of viral infection, to regulate myeloid cell differentiation (Zhang, Shen, et al., 2008). Initial experiments suggested that treatment of retinoic acid-sensitive myeloid leukemia cell lines with all-trans-retinoic acid (ATRA) resulted in increased expression of RIG-I. As this is considered partially to mimic normal myelopoiesis it has been concluded that RIG-I is developmentally regulated during myeloid differentiation (Zhang, Shen, et al., 2008). A potential role for MDA-5 in differentiation was not investigated and is an area worth further exploration.

It had been shown that most *rig-I*<sup>−/−</sup> mice die before birth (Kato et al., 2005). However, in a later study, in which a different region of the RIG-I gene was deleted, mice grew to maturity. These animals developed granulocytosis, indicative of defects in myeloid development. In the same study it was noted that the granulocytosis in the *rig-I*<sup>−/−</sup> mice is attributable to reduction in expression of IFN consensus sequence-binding protein (Icsbp) which plays a central role in regulating myeloid differentiation (Zhang, Shen, et al., 2008).

### 10. Melanoma differentiation associated gene-5 and type 1 diabetes

In a large screen of single-nucleotide polymorphisms (SNPs) associated with type 1 diabetes, a number of mutations were observed in the IFN induced with helicase C domain 1 (IFIH1/MDA-5) linkage disequilibrium (LD) block on chromosome 2q. Indeed there was a strong correlation between SNPs in the MDA-5 gene itself and in the 3′ intergenic region and susceptibility to type 1 diabetes (Liu et al., 2009; Smyth et al., 2006; Todd et al., 2007). Furthermore, after screening a large panel of controls and type 1 diabetes patients for MDA-5 expression it was suggested that SNPs leading to increased expression of MDA-5 may be associated with increased risk of developing the
disease and that there might be an increase in MDA-5 expression overall in diabetes although this latter observation did not appear to be statistically significant in this cohort (Liu et al., 2009). These studies suggest an intriguing possibility that blocking MDA-5 expression could provide a protective mechanism in the context of diabetes.

### 11. Evolution

The origin of the MDA-5 and RIG-I proteins has been examined from a phylogenetic perspective (Sarkar et al., 2008). What at first might seem a simple task is complicated by the presence of four protein domains in MDA-5 and RIG-I (Fig. 5A). Because the proteins are made up of these four domains that are themselves members of their own large domain families, it is possible that the MDA-5 and RIG-I proteins have a diverse and chimeric evolutionary history. In addition, the proteins that are closely related to MDA-5 and RIG-I can be determined using phylogenetic approaches.

Three steps were used to unravel the evolutionary history of these proteins. The first step was to examine if any of the domains had ancient linkage to each other or to other domains in other proteins. The second step was to identify whether any of the four domains had similar evolutionary histories, thus supporting the hypothesis that fusion events of previously independent domains occurred. The third step was to examine the evolutionary patterns of these singular and/or linked domains.

It is well known that helicase and DEADH domains are found linked in the genomes of species from the organismal superdomain Archaea as well as throughout the genomes of Eukaryotes (Story et al., 2001; Abdelhaleem et al., 2003). There are no known linked helicase DEADH domains in the organismal superdomain Bacteria. Given the well accepted topology of the tree of life with respect to the three organismal superdomains (Bacteria, Eukarya, Archaea), these observations suggest that the helicase DEADH linkage occurred in the common ancestor of Archaea and Eukarya. However, the helicase DEADH domain linkage could be a more recent phenomenon due to secondary fusion of these domains. If the latter scenario is correct then the evolutionary histories of the two domains will be divergent and if the former scenario is correct the evolutionary histories of the two domains will be congruent. When we examined the congruence of helicase and DEADH domains with respect to their evolutionary histories using the Incongruence Length Difference (ILD; 47–49) test, we found that both have statistically significant congruent histories (*p* < 0.05; (Sarkar et al., 2008) allowing us to reject the hypothesis that these two domains have fused after the divergence of Archaea. This result strongly implies that the helicase DEADH domain linkage in RIG-I and MDA-5 is an ancient relationship that arose in the common ancestor of the Archaea and Eukarya.

![Figure 5](#fig-5)

**Fig. 5.** A. Schematic diagram showing the domain structure of RIG-I and MDA-5. The numbers above the cartoon indicate the size of the domains. Abbreviations: C1 = CARD domain 1; C2 = CARD domain 2; hel = helicase and DEADH = DEAD box domain. B. Phylogenetic tree showing the overall topology of the helicase/DEADH domains. C. Stepwise diagram showing the duplication events involved in the evolution of the helicase/DEADH linked domains leading to RIG-I and MDA-5. Each double-headed arrow represents a gene duplication event. D. Phylogenetic tree showing the topology for the CARD domains.

Understanding the ancient linkage of the CARD domains is more complex. These domains, most often associated with caspases, are found in a wide variety of proteins from a broad array of vertebrates (Inohara et al., 2005; Kersse et al., 2007). Because CARD domains appear in a large number of protein families and some of these protein families are not related, we suggest that the CARD domains may have experienced different evolutionary histories to the helicase DEAH domains. Most caspases (cysteine-dependent aspartate-specific proteases) have a single CARD domain, with the exceptions being MDA-5, RIG-I, and Nacht proteins. Because the Nacht CARD domains have diverged greatly from the MDA-5 and RIG-I CARD domains (see below), we can rule out an ancient linkage involving Nacht CARD domains. Furthermore, because the linked CARD1/CARD2 state in MDA-5 is found in all vertebrates, we conclude that the linkage of the two CARD domains in this protein occurred in the common ancestor of vertebrates.

The second step involved understanding the CARD1/CARD2 linked arrangement in RIG-I and MDA-5. The linkage of the CARD1/CARD2 domains could have arisen by two very different processes in the vertebrates. First, a single CARD domain could have simply duplicated to produce two linked domains. Second, two independently derived CARD domains could have fused to produce the linked state. Again the former scenario would result in the two CARD domains having congruent evolutionary history and in the latter the two domains would have incongruent histories. The domain comparisons for CARD1 and CARD2 revealed strong incongruence of CARD1 and CARD2 using the ILD test \((p = 0.68)\). In addition, when the congruence of CARD1 is tested relative to the helicase and DEAH domains there is also strong incongruence \((p = 0.45\) and \(p = 0.91\) respectively). On the other hand, the CARD2 domain shows strong congruence with both the helicase and DEAH domains (both \(p < 0.01\)). The ILD tests indicate that the evolutionary history of the CARD1 domain is divergent from the other three domains and suggests that the CARD1 domain was grafted onto an existing CARD2-helicase-DEAD/DEAH structure more than likely in the common ancestor of vertebrates.

The congruence analysis discussed above using the ILD test suggests that there are two independently evolving domain linkages that make up these proteins. Consequently, we constructed phylogenies for the domains in these proteins to examine the evolutionary history of each. We first examined the linked helicase/DEAD-DEAH domains and second, the genealogical relationships of the two CARD domains.

Fig. 5B shows a phylogenetic analysis of a large representation of helicase, DEAH domains proteins in the database. This phylogenetic tree can be directly translated into the diagram shown in Fig. 5C. This scenario suggests that the helicase DEAH proteins have undergone at least seven duplication events to produce new families of helicase proteins in the evolution of eukaryotes. The final two duplications produced the LGP2 helicases and then the MDA-5-RIG-I proteins. Because LGP2, MDA-5, and RIG-I are found only in vertebrates and DICER proteins are broadly distributed phylogenetically, we propose that the duplication event of MDA-5 and RIG-I occurred in the common ancestor of vertebrates.

The CARD domain phylogenetic analysis is consistent with our observations based on the ILD tests (Fig. 5D). The phylogenetic and incongruence results indicate that the CARD2 boxes were the first N-terminal elements to be grafted to the anciently duplicated helicase/DEAD domains. Next, the MDA-5 CARD1 domain was grafted to the CARD2-helicase/DEAD domains. Because RIG-I and MDA-5 genes are only found in vertebrates we propose that the CARD1 fusion occurred in the common ancestor of vertebrates.

The evolutionary history of the MDA-5 and RIG-I proteins that we propose here, demonstrates an intriguing and circuitous pathway that is consistent with intense selection pressure for the existence and maintenance of these genes. Gene duplication is generally considered the easiest way to construct proteins with similar domain structure. In other words, the most parsimonious route to RIG-I and MDA-5

proteins would have been to make one and then duplicate the entire assemblage. However, according to the phylogenetic analyses, there was independent fusion of CARD2 to the RIG-I or MDA-5 helicase/DEAD domains and then another independent fusion of CARD1 to the CARD2/helicase/ DEAD domain. This pathway rules out simple duplications as the only route to the eventual structure of the RIG-I and MDA-5 proteins. We suggest, from these data, that the MDA-5 and RIG-I domain structures are a case where the most parsimonious evolutionary path has not been taken, because the scenario the trees support has nearly twice as many steps in them than required to perform simple duplication events (see Sarkar et al., 2008).

### 12. Perspectives

In just 6 years we have progressed from the discovery of the cytoplasmic RNA sensors of the innate immune response, to developing a detailed understanding of RNA recognition and the signaling pathways that lead to interferon induction. The molecular mechanisms by which MDA-5 and RIG-I recognize specific RNA ligands remain to be determined; we expect that additional structural work will be required to answer this question. Specifically, it will be necessary to solve the atomic structures of full-length RIG-I and MDA-5 with and without an RNA ligand to provide a detailed understanding of the recognition process. The requirement for the RNA helicase activity is also enigmatic. Similarly, RNA unwinding is not needed for RNA recognition, although this activity may be important for the conformational change that leads to exposure of the CARD domains.

A relatively uncharted area of innate immunity is the recognition of viral DNA. There have been several reports of cytoplasmic DNA sensors, but precisely how these recognize nucleic acid and induce cytokine synthesis remains to be determined (Stetson & Medzhitov, 2006; Ishii et al., 2006; Burckstummer et al., 2009). STING is essential for interferon production in response to cytoplasmic dsDNA, but it is not a nucleic acid sensor (Ishikawa & Barber, 2008). It will be of interest to determine whether DNA sensing pathways overlap with those described here for detection of RNA.

Viral genomes encode proteins that block every step of the RNA sensing pathways described in this review. Viral gene products may also antagonize nearly every step of the sensing of RNA by RIG-I and MDA-5, including RNA recognition, signaling, and induction of transcription. Understanding such viral countermeasures not only improves our understanding of innate sensing pathways, but may also suggest avenues for therapeutic intervention.

An important question is whether we can use our understanding of RNA sensing pathways for health benefit. We know that polymorphisms in genes encoding TLRs may influence the outcome of virus infection (Awomoyi et al., 2007). Therefore it would be useful to identify changes in MDA-5 and RIG-I that might alter susceptibility to infection. Other approaches include induction of MDA-5 or RIG-I transcription with specific promoter activators, or restoring defective innate responses that occur during chronic diseases such as hepatitis C virus infection. It might also be possible to suppress the inappropriate activation of innate responses that take place during autoimmune and inflammatory disease. One approach to such therapy would be to design inhibitors of key components that are targeted by viral antagonism. Such approaches could bring unexpected health benefits beyond the prevention of infectious diseases.

### Acknowledgments

The present studies were supported in part by the National Institutes of Health grants GM068448 (PBF), AI079336 (GNB) and AI50754 and AI068017 (VRR). Support was also provided by the Samuel Waxman Cancer Research Foundation (SWCRF) (PBF). RD thanks the Sackler Institute for Comparative Genomics, the Lewis and Dorothy B Cullman Program in Molecular Systematics and the Korein Family Foundation all

at the American Museum of Natural History for their continued support. Support from the VCU Institute of Molecular Medicine to DS and PBF is acknowledged. DS is a Harrison Scholar in Cancer Research at the VCU Massey Cancer Center, VCU School of Medicine. PBF holds the Thelma Newmeyer Corman Chair in Cancer Research at the VCU Massey Cancer Center, VCU School of Medicine and is a SWCRF Investigator.

# References

Abdelhaleem, M., Maltais, L., & Wain, H. (2003). The human DDX and DHX gene families of putative RNA helicases. *Genomics* 81(6), 618–622.

Aggarwal, K., & Silverman, N. (2008). Positive and negative regulation of the *Drosophila* immune response. *BMB Rep* 41(4), 267–277.

Alff, P. J., Sen, N., Gorbunova, E., Gavrilovskaya, I. N., & Mackow, E. R. (2008). The NY-1 hantavirus Gn cytoplasmic tail coprecipitates TRAF3 and inhibits cellular interferon responses by disrupting TBK1–TRAF3 complex formation. *J Virol* 82(18), 9115–9122.

Andrejeva, J., Childs, K. S., Young, D. F., Carlos, T. S., Stock, N., Goodbourn, S., et al. (2004). The V proteins of paramyxoviruses bind the IFN-inducible RNA helicase, mda-5, and inhibit its activation of the IFN-beta promoter. *Proc Natl Acad Sci U S A* 101(49), 17264–17269.

Arimoto, K., Takahashi, H., Hishiki, T., Konishi, H., Fujita, T., & Shimotohno, K. (2007). Negative regulation of the RIG-I signaling by the ubiquitin ligase RNF125. *Proc Natl Acad Sci U S A* 104(18), 7500–7505.

Awomoyi, A. A., Rallabhandi, P., Pollin, T. I., Lorenz, E., Sztein, M. B., Boukhvalova, M. S., et al. (2007). Association of TLR4 polymorphisms with symptomatic respiratory syncytial virus infection in high-risk infants and young children. *J Immunol* 179(5), 3171–3177.

Balachandran, S., Thomas, E., & Barber, G. N. (2004). A FADD-dependent innate immune mechanism in mammalian cells. *Nature* 432(7015), 401–405.

Balachandran, S., Venkataraman, T., Fisher, P. B., & Barber, G. N. (2007). Fas-associated death domain-containing protein-mediated antiviral innate immune signaling involves the regulation of Irf7. *J Immunol* 178(4), 2429–2439.

Bao, X., Liu, T., Shan, Y., Li, K., Garofalo, R. P., & Casola, A. (2008). Human metapneumovirus glycoprotein G inhibits innate immune responses. *PLoS Pathog* 4(5), e1000077.

Barral, P. M., Morrison, J. M., Drahos, J., Gupta, P., Sarkar, D., Fisher, P. B., et al. (2007). MDA-5 is cleaved in poliovirus-infected cells. *J Virol* 81(8), 3677–3684.

Barro, M., & Patton, J. T. (2007). Rotavirus NSP1 inhibits expression of type I interferon by antagonizing the function of interferon regulatory factors IRF3, IRF5, and IRF7. *J Virol* 81(9), 4473–4481.

Bauhofer, O., Summerfield, A., Sakoda, Y., Tratschin, J. D., Hofmann, M. A., & Ruggli, N. (2007). Classical swine fever virus Npro interacts with interferon regulatory factor 3 and induces its proteosomal degradation. *J Virol* 81(7), 3087–3096.

Berghäll, H., Sirén, J., Sarkar, D., Julkunen, I., Fisher, P. B., Vainionpää, R., et al. (2006). The interferon-inducible RNA helicase, mda-5, is involved in measles virus-induced expression of antiviral cytokines. *Microbes Infection* 8(8), 2138–2144.

Bozidis, P., Williamson, C. D., & Colberg-Poley, A. M. (2007). Isolation of endoplasmic reticulum, mitochondria, and mitochondria-associated membrane fractions from transfected cells and from human cytomegalovirus-infected primary fibroblasts. *Curr Protoc Cell Biol*, 27 Chapter 3, Unit 3.

Burckstummer, T., Baumann, C., Bluml, S., Dixit, E., Durnberger, G., Jahn, H., et al. (2009). An orthogonal proteomic-genomic screen identifies AIM2 as a cytoplasmic DNA sensor for the inflammasome. *Nat Immunol* 10(3), 266–272.

Cassady, K. A. (2005). Human cytomegalovirus TRS1 and IRS1 gene products block the double-stranded-RNA-activated host protein shutoff response induced by herpes simplex virus type 1 infection. *J Virol* 79(14), 8707–8715.

Chen, R. A., Ryzhakov, G., Cooray, S., Randow, F., & Smith, G. L. (2008). Inhibition of IkappaB kinase by vaccinia virus virulence factor B14. *PLoS Pathog* 4(2), e22.

Chien, Y., Kim, S., Bumeister, R., Loo, Y. M., Kwon, S. W., Johnson, C. L., et al. (2006). RalB GTPase-mediated activation of the IkappaB family kinase TBK1 couples innate immune signaling to tumor cell survival. *Cell* 127(1), 157–170.

Child, S. J., Hakki, M., De Niro, K. L., & Geballe, A. P. (2004). Evasion of cellular antiviral responses by human cytomegalovirus TRS1 and IRS1. *J Virol* 78(1), 197–205.

Child, S. J., Hanson, L. K., Brown, C. E., Janzen, D. M., & Geballe, A. P. (2006). Double-stranded RNA binding by a heterodimeric complex of murine cytomegalovirus m142 and m143 proteins. *J Virol* 80(20), 10173–10180.

Childs, K., Stock, N., Ross, C., Andrejeva, J., Hilton, L., Skinner, M., et al. (2007). mda-5, but not RIG-I, is a common target for paramyxovirus V proteins. *Virology* 359(1), 190–200.

Childs, K. S., Andrejeva, J., Randall, R. E., & Goodbourn, S. (2008). Mechanism of mda-5 inhibition by paramyxovirus V proteins. *J Virol* 83(3), 1465–1473.

Chinnaiyan, A. M., O Rourke, K., Tewari, M., & Dixit, V. M. (1995). FADD, a novel death domain-containing protein, interacts with the death domain of Fas and initiates apoptosis. *Cell* 81(4), 505–512.

Cocude, C., Truong, M. J., Billaut-Mulot, O., Delsart, V., Darcissac, E., Capron, A., et al. (2003). A novel cellular RNA helicase, RH116, differentially regulates cell growth, programmed cell death and human immunodeficiency virus type 1 replication. *J Gen Virol* 84(Pt 12), 3215–3225.

Cui, S., Eisenacher, K., Kirchhofer, A., Brzozka, K., Lammens, A., Lammens, K., et al. (2008). The C-terminal regulatory domain is the RNA 5′-triphosphate sensor of RIG-I. *Mol Cell* 29(2), 169–179.

Devaraj, S. G., Wang, N., Chen, Z., Tseng, M., Barretto, N., Lin, R., et al. (2007). Regulation of IRF-3-dependent innate immunity by the papain-like protease domain of the severe acute respiratory syndrome coronavirus. *J Biol Chem* 282(44), 32208–32221.

Diao, F., Li, S., Tian, Y., Zhang, M., Xu, L. G., Zhang, Y., et al. (2007). Negative regulation of MDA5- but not RIG-I-mediated innate antiviral signaling by the dihydroxyacetone kinase. *Proc Natl Acad Sci U S A* 104(28), 11706–11711.

DiPerna, G., Stack, J., Bowie, A. G., Boyd, A., Kotwal, G., Zhang, Z., et al. (2004). Poxvirus protein N1L targets the I-kappaB kinase complex, inhibits signaling to NF-kappaB by the tumor necrosis factor superfamily of receptors, and inhibits NF-kappaB and IRF3 signaling by toll-like receptors. *J Biol Chem* 279(35), 36570–36578.

Ferrandon, D., Imler, J. L., & Hoffmann, J. A. (2004). Sensing infection in *Drosophila*: toll and beyond. *Semin Immunol* 16(1), 43–53.

Fisher, P. B., & Grant, S. (1984). Effects of interferon on differentiation in normal and tumor cells. *Pharmacol & Therapeut* 27, 143–166.

Fisher, P. B., Miranda, A. F., Babiss, L. E., Pestka, S., & Weinstein, I. B. (1983). Opposing effects of interferon produced in bacteria and of tumor promoters on myogenesis in human myoblast cultures. *Proc Natl Acad Sci USA* 80, 2961–2965.

Flanegan, J. B., Pettersson, R. F., Ambros, V., Hewlett, M. J., & Baltimore, D. (1977). Covalent linkage of a protein to a defined nucleotide sequence at the 5′-terminus of virion and replicative intermediate RNAs of poliovirus. *Proc. Natl. Acad. Sci. USA* 74, 961–965.

Fredericksen, B. L., Keller, B. C., Fornek, J., Katze, M. G., & Gale, M., Jr. (2008). Establishment and maintenance of the innate antiviral response to West Nile Virus involves both RIG-I and MDA5 signaling through IPS-1. *J Virol* 82(2), 609–616.

Friedman, C. S., O'Donnell, M. A., Legarda-Addison, D., Ng, A., Cardenas, W. B., Yount, J. S., et al. (2008). The tumour suppressor CYLD is a negative regulator of RIG-I-mediated antiviral response. *EMBO Rep* 9(9), 930–936.

Friesen, H. J., Stein, S., Evinger, M., Familletti, P. C., Moschera, J., Meienhofer, J., et al. (1981). Purification and molecular characterization of human fibroblast interferon. *Arch Biochemistry Biophys* 206(2), 432–450.

Gack, M. U., Kirchhofer, A., Shin, Y. C., Inn, K. S., Liang, C., Cui, S., et al. (2008). Roles of RIG-I N-terminal tandem CARD and splice variant in TRIM25-mediated antiviral signal transduction. *Proc Natl Acad Sci U S A* 105(43), 16743–16748.

Gee, P., Chua, P. K., Gevorkyan, J., Klumpp, K., Najera, I., Swinney, D. C., et al. (2008). Essential role of the N-terminal domain in the regulation of RIG-I ATPase activity. *J Biol Chem* 283(14), 9488–9496.

Georgel, P., Naitza, S., Kappler, C., Ferrandon, D., Zachary, D., Swimmer, C., et al. (2001). *Drosophila* immune deficiency (IMD) is a death domain protein that activates antibacterial defense and can promote apoptosis. *Dev Cell* 1(4), 503–514.

Giacomini, P., Aguzzi, A., Pestka, S., Fisher, P. B., & Ferrone, S. (1984). Modulation by recombinant DNA leukocyte (α) and fibroblast (β) interferons of the expression and shedding of HLA and tumor associated antigens by human melanoma cells. *J Immunol* 133, 1649–1655.

Grant, S., Bhalla, K., Weinstein, I. B., Pestka, S., Milano, M. D., & Fisher, P. B. (1985). Recombinant human interferon sensitizes resistant myeloid leukemic cells to induction of terminal differentiation. *Biochem Biophys Res Commun* 130, 379–388.

Greiner, J. W., Guadagni, F., Noguchi, P., Pestka, S., Colcher, D., Fisher, P. B., et al. (1987). Use of recombinant interferon to enhance monoclonal antibody-targeting of carcinoma lesions in vivo. *Science* 235, 895–898.

Greiner, J. W., Hand, P. H., Noguchi, P., Fisher, P. B., Pestka, S., & Schlom, J. (1984). Enhanced expression of surface tumor-associated antigens on human breast and colon tumor cells after recombinant leukocyte α-interferon treatment. *Cancer Res* 44, 3208–3214.

Greiner, J. W., Schlom, J., Pestka, S., Langer, J. A., Giacomini, P., Kusama, M., et al. (1985). Modulation of tumor associated antigen expression and shedding by recombinant human leukocyte and fibroblast interferons. *Pharmacol & Therapeut* 31, 209–236.

Gitlin, L., Barchet, W., Gilfillan, S., Cella, M., Beutler, B., Flavell, R. A., et al. (2006). Essential role of mda-5 in type I IFN responses to polyriboinosinic:polyribocytidylic acid and encephalomyocarditis picornavirus. *Proc Natl Acad Sci U S A* 103(22), 8459–8464.

Guerra, S., Caceres, A., Knobeloch, K. P., Horak, I., & Esteban, M. (2008). Vaccinia virus E3 protein prevents the antiviral action of ISG15. *PLoS Pathog* 4(7), e1000096.

Guo, W., & Novick, P. (2004). The exocyst meets the translocon: a regulatory circuit for secretion and protein synthesis? *Trends Cell Biol* 14(2), 61–63.

Guo, Z., Chen, L. M., Zeng, H., Gomez, J. A., Plowden, J., Fujita, T., et al. (2007). NS1 protein of influenza A virus inhibits the function of intracytoplasmic pathogen sensor, RIG-I. *Am J Respir Cell Mol Biol* 36(3), 263–269.

Haasnoot, J., de Vries, W., Geutjes, E. J., Prins, M., de Haan, P., & Berkhout, B. (2007). The Ebola virus VP35 protein is a suppressor of RNA silencing. *PLoS Pathog*. 3, 794–803.

Habjan, M., Andersson, I., Klingström, J., Schumann, M., Martin, A., Zimmermann, P., et al. (2008). Processing of genome 5′ termini as a strategy of negative-strand RNA viruses to avoid RIG-I-dependent interferon induction. *PLoS ONE* 3(4), e2032.

Hacker, H., Redecke, V., Blagoev, B., Krathmarova, I., Hsu, L. C., Wang, G. G., et al. (2006). Specificity in Toll-like signalling through distinct effector functions of TRAF3 and TRAF6. *Nature* 439, 204–207.

Hakki, M., & Geballe, A. P. (2005). Double-stranded RNA binding by human cytomegalovirus pTRS1. *J Virol* 79(12), 7311–7318.

Hartmann, E., Gorlich, D., Kostka, S., Otto, A., Kraft, R., Knespel, S., et al. (1993). A tetrameric complex of membrane proteins in the endoplasmic reticulum. *Eur J Biochem* 214(2), 375–381.

Hornung, V., Ellegast, J., Kim, S., Brzozka, K., Jung, A., Kato, H., et al. (2006). 5′-Triphosphate RNA is the ligand for RIG-I. *Science* 314(5801), 994–997.

Hsu, H., Huang, J., Shu, H. B., Baichwal, V., & Goeddel, D. V. (1996). TNF-dependent recruitment of the protein kinase RIP to the TNF receptor-1 signaling complex. *Immunity* 4(4), 387–396.

Huang, F., Adelman, J., Jiang, H., Goldstein, N. I., & Fisher, P. B. (1999a). Differentiation induction subtraction hybridization (DISH): an approach for cloning genes differentially expressed during growth arrest and terminal differentiation in human melanoma cells. *Gene* 236, 125–131.

Huang, F., Adelman, J., Jiang, H., Goldstein, N. I., & Fisher, P. B. (1999b). Identification and temporal expression pattern of genes modulated during irreversible growth

arrest and terminal differentiation in human melanoma cells. *Oncogene* 18, 3546–3552.

Inohara Chamaillard, M., McDonald, C., & Nunez, G. (2005). NOD-LRR proteins: role in host-microbial interactions and inflammatory disease. *Annu Rev Biochem* 74, 355–383.

Isaacs, A., & Lindenmann, J. (1957). Virus interference. I. The interferon. *Proc R Soc Lond B Biol Sci* 147(927), 258–267.

Ishikawa, H., & Barber, G. N. (2008). STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling. *Nature* 455(7213), 674–678.

Ishii, K. J., Coban, C., Kato, H., Takahashi, K., Torii, Y., Takeda, F., et al. (2006). A Toll-like receptor-independent antiviral response induced by double-stranded B-form DNA. *Nat. Immunol* 7, 40–48.

Jackson, W. T., Giddings, T. H., Jr., Taylor, M. P., Mulinyawe, S., Rabinovitch, M., Kopito, R. R., et al. (2005). Subversion of cellular autophagosomal machinery by RNA viruses. *PLoS Biol* 3(5), e156.

Jiang, H., & Fisher, P. B. (1993). Use of a sensitive and efficient subtraction hybridization protocol for the identification of genes differentially regulated during the induction of differentiation in human melanoma cells. *Mol Cell Differen* 1(3), 285–299.

Jiang, H., Kang, D. C., Alexandre, D., & Fisher, P. B. (2000). RaSH, A rapid subtraction hybridization approach for identifying and cloning differentially expressed genes. *Proc Natl Acad Sci USA* 97, 12684–12689.

Jin, L., Waterman, P. M., Jonscher, K. R., Short, C. M., Reisdorph, N. A., & Cambier, J. C. (2008). MPYS, a novel membrane tetraspanner, is associated with major histocompatibility complex class II and mediates transduction of apoptotic signals. 28(16), 5014–5026.

Joo, C. H., Shin, Y. C., Gack, M., Wu, L., Levy, D., & Jung, J. U. (2007). Inhibition of interferon regulatory factor 7 (IRF7)-mediated interferon signal transduction by the Kaposi's sarcoma-associated herpesvirus viral IRF homolog vIRF3. *J Virol* 81 (15), 8282–8292.

Jounai, N., Takeshita, F., Kobiyama, K., Sawano, A., Miyawaki, A., Xin, K. Q., et al. (2007). The Atg5 Atg12 conjugate associates with innate antiviral immune responses. *Proc Natl Acad Sci USA* 104(35), 14050–14055.

Kang, D. C., Gopalkrishnan, R. V., Lin, L., Randolph, A., Valerie, K., Pestka, S., et al. (2004). Expression analysis and genomic characterization of human melanoma differentiation-associated gene-5, *mda-5*: a novel type I interferon-responsive apoptosis-inducing gene. *Oncogene* 23(9), 1789–1800.

Kang, D. C., Gopalkrishnan, R. V., Wu, Q., Jankowsky, E., Pyle, A. M., & Fisher, P. B. (2002). *mda-5*: an interferon-inducible putative RNA helicase with double-stranded RNA-dependent ATPase activity and melanoma growth-suppressive properties. *Proc Natl Acad Sci USA* 99(2), 637–642.

Kato, H., Sato, S., Yoneyama, M., Yamamoto, M., Uematsu, S., Matsui, K., et al. (2005). Cell type-specific involvement of RIG-I in antiviral response. *Immunity* 23(1), 19–28.

Kato, H., Takeuchi, O., Mikamo-Satoh, E., Hirai, R., Kawai, T., Matsushita, K., et al. (2008). Length-dependent recognition of double-stranded ribonucleic acids by retinoic acid-inducible gene-I and melanoma differentiation-associated gene 5. *J Exp Med* 205(7), 1601–1610.

Kato, H., Takeuchi, O., Sato, S., Yoneyama, M., Yamamoto, M., Matsui, K., et al. (2006). Differential roles of MDA5 and RIG-I helicases in the recognition of RNA viruses. *Nature* 441(7089), 101–105.

Kawai, T., & Akira, S. (2008). Toll-like receptor and RIG-I-like receptor signaling. *Ann NY Acad Sci* 1143, 1–20.

Kawai, T., Takahashi, K., Sato, S., Coban, C., Kumar, H., Kato, H., et al. (2005). IPS-1, an adaptor triggering RIG-I- and Mda5-mediated type I interferon induction. *Nat Immunol* 6(10), 981–988.

Kayagaki, N., Phung, Q., Chan, S., Chaudhari, R., Quan, C., O'Rourke, K. M., et al. (2007). DUBA: a deubiquitinase that regulates type I interferon production. *Science* 318(5856), 1628–1632.

Kelliher, M. A., Grimm, S., Ishida, Y., Kuo, F., Stanger, B. Z., & Leder, P. (1998). The death domain kinase RIP mediates the TNF-induced NF-kappaB signal. *Immunity* 8(3), 297–303.

Kersse, K., Vanden Berghe, T., Lamkanfi, M., & Vandenabeele, P. (2007). A phylogenetic and functional overview of inflammatory caspases and caspase-1-related CARD-only proteins. *Biochem Soc Trans* 35(Pt 6), 1508–1511.

Kim, M. J., & Yoo, J. Y. (2008). Active caspase-1-mediated secretion of retinoic acid inducible gene-I. *J Immunol* 181(10), 7324–7331.

Kim, M. J., Hwang, S. Y., Imazumi, T., & Yoo, J. Y. (2008). Negative feedback regulation of RIG-I-mediated antiviral signaling by interferon-induced ISG15 conjugation. *J Virol* 82(3), 1474–1483.

Komuro, A., & Horvath, C. M. (2006). RNA- and virus-independent inhibition of antiviral signaling by RNA helicase LGP2. *J Virol* 80(24), 12332–12342.

Kovacsovics, M., Martinon, F., Micheau, O., Bodmer, J. L., Hofmann, K., & Tschopp, J. (2002). Overexpression of Helicard, a CARD-containing helicase cleaved during apoptosis, accelerates DNA degradation. *Curr Biol* 12(10), 838–843.

Kumar, H., Kawai, T., Kato, H., Sato, S., Takahashi, K., Coban, C., et al. (2006). Essential role of IPS-1 in innate immune responses against RNA viruses. *J Exp Med* 203(7), 1795–1803.

Lee, Y. F., Nomoto, A., Detjen, B. M., & Wimmer, E. (1977). A protein covalently linked to poliovirus genome RNA. *Proc. Natl. Acad. Sci. USA* 74, 59–63.

Lefort, S., Soucy-Faulkner, A., Grandvaux, N., & Flamand, L. (2007). Binding of Kaposi's sarcoma-associated herpesvirus K-bZIP to interferon-responsive factor 3 elements modulates antiviral gene expression. *J Virol* 81(20), 10950–10960.

Lemaitre, B., Kromer-Metzger, E., Michaut, L., Nicolas, E., Meister, M., Georgel, P., et al. (1995). A recessive mutation, immune deficiency (imd), defines two distinct control pathways in the *Drosophila* host defense. *Proc Natl Acad Sci USA* 92(21), 9465–9469.

Leulier, F., Vidal, S., Saigo, K., Ueda, R., & Lemaitre, B. (2002). Inducible expression of double-stranded RNA reveals a role for dFADD in the regulation of the antibacterial response in *Drosophila* adults. *Curr Biol* 12(12), 996–1000.


Lin, R., Genin, P., Mamane, Y., Sgarbanti, M., Battistini, A., Harrington, W. J., Jr., et al. (2001). HHV-8 encoded vIRF-1 represses the interferon antiviral response by blocking IRF-3 recruitment of the CBP/p300 coactivators. *Oncogene* 20(7), 800–811.

Lin, R., Lacoste, J., Nakhaei, P., Sun, Q., Yang, L., Paz, S., et al. (2006). Dissociation of a MAVS/IPS-1/VISA/Cardif-IKKepsilon molecular complex from the mitochondrial outer membrane by hepatitis C virus NS3-4A proteolytic cleavage. *J Virol* 80(12), 6072–6083.

Lin, L., Su, Z., Lebedeva, I. V., Gupta, P., Boukerche, H., Rai, T., et al. (2006). Activation of Ras/Raf protects cells from melanoma differentiation-associated gene-5-induced apoptosis. *Cell Death Differ* 13(11), 1982–1993.

Lin, R., Yang, L., Nakhaei, P., Sun, Q., Sharif-Askari, E., Julkunen, I., et al. (2006). Negative regulation of the retinoic acid-inducible gene I-induced antiviral state by the ubiquitin-editing protein A20. *J Biol Chem* 281(4), 2095–2103.

Lipschutz, J. H., Lingappa, V. R., & Mostov, K. E. (2003). The exocyst affects protein synthesis by acting on the translocation machinery of the endoplasmic reticulum. *J Biol Chem* 278(23), 20954–20960.

Liu, Y. C., Penninger, J., & Karin, M. (2005). Immunity by ubiquitylation: a reversible process of modification. *Nat Rev Immunol* 5(12), 941–952.

Liu, S., Wang, H., Jin, Y., Podolsky, R., Linga Reddy, M. V., Pedersen, J., et al. (2009). The IFIH1 polymorphisms are significantly associated with type 1 diabetes and IFIH1 gene expression in peripheral blood mononuclear cells. *Hum Mol Genet* 18(2), 358–365.

Loo, Y. M., Fornek, J., Crochet, N., Bajwa, G., Perwitasari, O., Martinez-Sobrido, L., et al. (2008). Distinct RIG-I and MDA5 signaling by RNA viruses in innate immunity. *J Virol* 82(1), 335–345.

Loo, Y. M., Owen, D. M., Li, K., Erickson, A. K., Johnson, C. L., Fish, P. M., et al. (2006). Viral and therapeutic control of IFN-beta promoter stimulator 1 during hepatitis C virus infection. *Proc Natl Acad Sci USA* 103(15), 6001–6006.

Lu, L. L., Puri, M., Horvath, C. M., & Sen, G. C. (2008). Select paramyxoviral V proteins inhibit IRF3 activation by acting as alternative substrates for inhibitor of kappaB kinase epsilon (IKKe)/TBK1. *J Biol Chem* 283(21), 14269–14276.

Lu, G., Reinert, J. T., Pitha-Rowe, I., Okumura, A., Kellum, M., Knobeloch, K. P., et al. (2006). ISG15 enhances the innate antiviral response by inhibition of IRF-3 degradation. *Cell Mol Biol (Noisy-le-grand)* 52(1), 29–41.

Malathi, K., Dong, B., Gale, M., Jr., & Silverman, R. H. (2007). Small self-RNA generated by RNase L amplifies antiviral innate immunity. *Nature* 448(7155), 816–819.

Malathi, K., Paranjape, J. M., Bulanova, E., Shim, M., Guenther-Johnson, J. M., Faber, P. W., et al. (2005). A transcriptional signaling pathway in the IFN system mediated by 2′-5′-oligoadenylate activation of RNase L. *Proc Natl Acad Sci USA* 102(41), 14533–14538.

Matta, H., Mazzacurati, L., Schamus, S., Yang, T., Sun, Q., & Chaudhary, P. M. (2007). Kaposi's sarcoma-associated herpesvirus (KSHV) oncoprotein K13 bypasses TRAFs and directly interacts with the IkappaB kinase complex to selectively activate NF-kappaB without JNK activation. *J Biol Chem* 282(34), 24858–24865.

Melroe, G. T., Silva, L., Schaffer, P. A., & Knipe, D. M. (2007). Recruitment of activated IRF-3 and CBP/p300 to herpes simplex virus ICP0 nuclear foci: potential role in blocking IFN-beta induction. *Virology* 360(2), 305–321.

Menetret, J. F., Hegde, R. S., Heinrich, S. U., Chandramouli, P., Ludtke, S. J., Rapoport, T. A., et al. (2005). Architecture of the ribosome-channel complex derived from native membranes. *J Mol Biol* 348(2), 445–457.

Meylan, E., Curran, J., Hofmann, K., Moradpour, D., Binder, M., Bartenschlager, R., et al. (2005). Cardif is an adaptor protein in the RIG-I antiviral pathway and is targeted by hepatitis C virus. *Nature* 437(7062), 1167–1172.

Medzhitov, R. (2007). Recognition of microorganisms and activation of the immune response. *Nature* 449(7164), 819–826.

Mibayashi, M., Martinez-Sobrido, L., Loo, Y. M., Cardenas, W. B., Gale, M., Jr., & Garcia-Sastre, A. (2007). Inhibition of retinoic acid-inducible gene I-mediated induction of beta interferon by the NS1 protein of influenza A virus. *J Virol* 81(2), 514–524.

Michallet, M. C., Meylan, E., Ermolaeva, M. A., Vazquez, J., Rebsamen, M., Curran, J., et al. (2008). TRADD protein is an essential component of the RIG-like helicase antiviral pathway. *Immunity* 28(5), 651–661.

Micheau, O., & Tschopp, J. (2003). Induction of TNF receptor I-mediated apoptosis via two sequential signaling complexes. *Cell* 114(2), 181–190.

Moore, C. B., Bergstralh, D. T., Duncan, J. A., Lei, Y., Morrison, T. E., Zimmermann, A. G., et al. (2008). NLRX1 is a regulator of mitochondrial antiviral immunity. *Nature* 451(7178), 573–577.

Moulton, T. A., Jiang, H., Guarini, L., Fetell, M. R., & Fisher, P. B. (1992). Induction of growth suppression and modification of gene expression in multidrug resistant human glioblastoma multiforme cells by recombinant human fibroblast and immune interferon. *Intl J Cancer* 51, 373–378.

Mukherjee, A., Morosky, S. A., Shen, L., Weber, C. R., Turner, J. R., Kim, K. S., et al. (2009). Retinoic Acid-induced Gene-1 (RIG-I) associates with the actin cytoskeleton via caspase activation and recruitment domain-dependent interactions. *J Biol Chem* 284(10), 6486–6494.

Naitza, S., Rosse, C., Kappler, C., Georgel, P., Belvin, M., Gubb, D., et al. (2002). The *Drosophila* immune defense against gram-negative infection requires the death protein dFADD. *Immunity* 17(5), 575–581.

Narasimhan, J., Wang, M., Fu, Z., Klein, J. M., Haas, A. L., & Kim, J. J. (2005). Crystal structure of the interferon-induced ubiquitin-like protein ISG15. *J Biol Chem* 280(29), 27356–27365.

Neznanov, N., Chumakov, K. M., Neznanova, L., Almasan, A., Banerjee, A. K., & Gudkov, A. V. (2005). Proteolytic cleavage of the p65-RelA subunit of NF-kappaB during poliovirus infection. *J Biol Chem* 280(25), 24153–24158.

Nusslein-Volhard, C., & Wieschaus, E. (1980). Mutations affecting segment number and polarity in *Drosophila*. *Nature* 287(5785), 795–801.

Okumura, A., Alce, T., Lubyova, B., Ezelle, H., Strebel, K., & Pitha, P. M. (2008). HIV-1 accessory proteins VPR and Vif modulate antiviral response by targeting IRF-3 for degradation. *Virology* 373(1), 85–97.

Opitz, B., Rejaibi, A., Dauber, B., Eckhard, J., Vinzing, M., Schmeck, B., et al. (2007). IFNbeta induction by influenza A virus is mediated by RIG-I which is regulated by the viral NS1 protein. *Cell Microbiol* 9(4), 930–938.

Organesyan, G., Saha, S. K., Guo, B., He, J. Q., Shahanigan, A., Zarnegar, B., et al. (2006). Critical role of TRAF3 in the Toll-like receptor-dependent and -independent antiviral response. *Nature* 439, 208–211.

Otsuka, M., Kato, N., Moriyama, M., Taniguchi, H., Wang, Y., Dharel, N., et al. (2005). Interaction between the HCV NS3 protein and the host TBK1 protein leads to inhibition of cellular antiviral responses. *Hepatology* 41(5), 1004–1012.

Pan, M., Geng, S., Xiao, S., Ren, J., Liu, Y., Li, X., et al. (2008). Apoptosis induced by synthetic retinoic acid CD437 on human melanoma A375 cells involves RIG-I pathway. *Arch Dermatol Res* 301(1), 15–20.

Pestka, S., & Baron, S. (1981). Definition and classification of the interferons. *Meth Enzymol* 78, 3–14.

Peters, K., Chattopadhyay, S., & Sen, G. C. (2008). IRF-3 activation by Sendai virus infection is required for cellular apoptosis and avoidance of persistence. *J Virol* 82(7), 3500–3508.

Pichlmair, A., Schulz, O., Tan, C. P., Naslund, T. I., Liljestrom, P., Weber, F., et al. (2006). RIG-I-mediated antiviral responses to single-stranded RNA bearing 5′-phosphates. *Science* 314(5801), 997–1001.

Plotch, S. J., Bouloy, M., Ulmanen, I., & Krug, R. M. (1981). A unique cap(m7GpppXm)-dependent influenza virion endonuclease cleaves capped RNAs to generate the primers that initiate viral RNA transcription. *Cell* 23(3), 847–858.

Plumet, S., Herschke, F., Bourhis, J. M., Valentin, H., Longhi, S., & Gerlier, D. (2007). Cytosolic 5′-triphosphate ended viral leader transcript of measles virus as activator of the RIG I-mediated interferon response. *PLoS ONE* 2(3), e279.

Poeck, H., Besch, R., Maihoefer, C., Renn, M., Tormo, D., Morskaya, S. S., et al. (2008). 5′-Triphosphate-siRNA: turning gene silencing and Rig-I activation against melanoma. *Nat Med* 14(11), 1256–1263.

Rebsamen, M., Meylan, E., Curran, J., & Tschopp, J. (2008). The antiviral adaptor proteins Cardif and Trif are processed and inactivated by caspases. *Cell Death Differ* 15(11), 1804–1811.

Rintahaka, J., Wiik, D., Kovanen, P. E., Alenius, H., & Matikainen, S. (2008). Cytosolic antiviral RNA recognition pathway activates caspases 1 and 3. *J Immunol* 180(3), 1749–1757.

Rothenfusser, S., Goutagny, N., DiPerna, G., Gong, M., Monks, B. G., Schoenemeyer, A., et al. (2005). The RNA helicase Lgp2 inhibits TLR-independent sensing of viral replication by retinoic acid-inducible gene-I. *J Immunol* 175(8), 5260–5268.

Rubinstein, M., Rubinstein, S., Familletti, P. C., Miller, R., Waldman, A. A., & Pestka, S. (1979). Human leukocyte interferon: production, purification to homogeneity, and initial characterization. *Proc Natl Acad Sci USA* 76(2), 640–644.

Saira, K., Zhou, Y., & Jones, C. (2007). The infected cell protein 0 encoded by bovine herpesvirus 1 (blCP0) induces degradation of interferon response factor 3 and, consequently, inhibits beta interferon promoter activity. *J Virol* 81(7), 3077–3086.

Saito, T., Hirai, R., Loo, Y. M., Owen, D., Johnson, C. L., Sinha, S. C., et al. (2007). Regulation of innate antiviral defenses through a shared repressor domain in RIG-I and LGP2. *Proc Natl Acad Sci U S A* 104(2), 582–587.

Sarkar, D., Desalle, R., & Fisher, P. B. (2008). Evolution of MDA-5/RIG-I-dependent innate immunity: independent evolution by domain grafting. *Proc Natl Acad Sci USA* 105(44), 17040–17045.

Schroder, M., Baran, M., & Bowie, A. G. (2008). Viral targeting of DEAD box protein 3 reveals its role in TBK1/IKKepsilon-mediated IRF activation. *EMBO J* 27(15), 2147–2157.

Sen, G. C., & Sarkar, S. N. (2007). The interferon-stimulated genes: targets of direct signaling by interferons, double-stranded RNA, and viruses. *Curr Top Microbiol Immunol* 316, 233–250.

Seth, R. B., Sun, L., Ea, C. K., & Chen, Z. J. (2005). Identification and characterization of MAVS, a mitochondrial antiviral signaling protein that activates NF-kappaB and IRF 3. *Cell* 122(5), 669–682.

Sirén, J., Imaizumi, T., Sarkar, D., Pietilä, T., Noah, D. L., Lin, R., et al. (2006). Retinoic acid inducible gene-I and mda-5 are involved in influenza A virus-induced expression of antiviral cytokines. *Microbes Infection* 8(8), 2013–2020.

Smith, E. J., Marie, I., Prakash, A., Garcia-Sastre, A., & Levy, D. E. (2001). IRF3 and IRF7 phosphorylation in virus-infected cells does not require double-stranded RNA-dependent protein kinase R or Ikappa B kinase but is blocked by Vaccinia virus E3L protein. *J Biol Chem* 276(12), 8951–8957.

Smyth, D. J., Cooper, J. D., Bailey, R., Field, S., Burren, O., Smink, L. J., et al. (2006). A genome-wide association study of nonsynonymous SNPs identifies a type 1 diabetes locus in the interferon-induced helicase (IFIH1) region. *Nat Genet* 38(6), 617–619.

Soulat, D., Burckstummer, T., Westermayer, S., Goncalves, A., Bauch, A., Stefanovic, A., et al. (2008). The DEAD-box helicase DDX3X is a critical component of the TANK-binding kinase 1-dependent innate immune response. *EMBO J* 27(15), 2135–2146.

Stetson, D. B., & Medzhitov, R. (2006). Recognition of cytosolic DNA activates an IRF3-dependent innate immune response. *Immunity* 24(1), 93–103.

Story, R. M., Li, H., & Abelson, J. N. (2001). Crystal structure of a DEAD box protein from the hyperthermophile Methanococcus jannaschii. *Proc Natl Acad Sci U S A* 98(4), 1465–1470.

Su, Z. -z., Lee, S. G., Emdad, L., Lebedeva, I. V., Gupta, P., Valerie, K., et al. (2008). Cloning and characterization of SARI (suppressor of AP-1, regulated by IFN). *Proc Natl Acad Sci USA* 105, 20906–20911.

Su, Z. Z., Sarkar, D., Emdad, L., Barral, P. M., & Fisher, P. B. (2007). Central role of interferon regulatory factor-1 (IRF-1) in controlling retinoic acid inducible gene-I (RIG-I) expression. *J Cell Physiol* 213(2), 502–510.

Sumpter, R., Jr., Loo, Y. M., Foy, E., Li, K., Yoneyama, M., Fujita, T., et al. (2005). Regulating intracellular antiviral defense and permissiveness to hepatitis C virus RNA replication through a cellular RNA helicase, RIG-I. *J Virol* 79(5), 2689–2699.

Sun, Y. W. (1997). *RIG-I, a Human Homolog Gene of RNA Helicase, is Induced by Retinoic Acid During the Differentiation of Acute Promyelocytic Leukemia Cell*. Shanghai Second Medical University, Shanghai.

Sun, Q., Sun, L., Liu, H. H., Chen, X., Seth, R. B., Forman, J., et al. (2006). The specific and essential role of MAVS in antiviral innate immune responses. *Immunity* 24(5), 633–642.

Tait, S. W., Reid, E. B., Greaves, D. R., Wileman, T. E., & Powell, P. P. (2000). Mechanism of inactivation of NF-kappa B by a viral homologue of I kappa b alpha. Signal-induced release of i kappa b alpha results in binding of the viral homologue to NF-kappa B. *J Biol Chem* 275(44), 34656–34664.

Takahasi, K., Yoneyama, M., Nishihori, T., Hirai, R., Kumeta, H., Narita, R., et al. (2008). Nonself RNA-sensing mechanism of RIG-I helicase and activation of antiviral immune responses. *Mol Cell* 29(4), 428–440.

Takeuchi, O., & Akira, S. (2007). Recognition of viruses by innate immunity. *Immunol Rev* 220, 214–224.

Takeuchi, O., & Akira, S. (2008). MDA5/RIG-I and virus recognition. *Curr Opin Immunol* 20(1), 17–22.

Talon, J., Horvath, C. M., Polley, R., Basler, C. F., Muster, T., Palese, P., et al. (2000). Activation of interferon regulatory factor 3 is inhibited by the influenza A virus NS1 protein. *J Virol* 74(17), 7989–7996.

Todd, J. A., Walker, N. M., Cooper, J. D., Smyth, D. J., Downes, K., Plagnol, V., et al. (2007). Robust associations of four new chromosome regions from genome-wide analyses of type 1 diabetes. *Nat Genet* 39(7), 857–864.

Valchanova, R. S., Picard-Maureau, M., Budt, M., & Brune, W. (2006). Murine cytomegalovirus m142 and m143 are both required to block protein kinase R-mediated shutdown of protein synthesis. *J Virol* 80(20), 10181–10190.

Venkataraman, T., Valdes, M., Elsby, R., Kakuta, S., Caceres, G., Saijo, S., et al. (2007). Loss of DExD/H box RNA helicase LGP2 manifests disparate antiviral responses. *J Immunol* 178(10), 6444–6455.

Wertz, I. E., O'Rourke, K. M., Zhou, H., Eby, M., Aravind, L., Seshagiri, S., et al. (2004). Deubiquitination and ubiquitin ligase domains of A20 downregulate NF-kappaB signalling. *Nature* 430(7000), 694–699.

Xiang, Y., Condit, R. C., Vijaysri, S., Jacobs, B., Williams, B. R., & Silverman, R. H. (2002). Blockade of interferon induction and action by the E3L double-stranded RNA binding proteins of vaccinia virus. *J Virol* 76(10), 5251–5259.

Xu, L. G., Wang, Y. Y., Han, K. J., Li, L. Y., Zhai, Z., & Shu, H. B. (2005). VISA is an adapter protein required for virus-triggered IFN-beta signaling. *Mol Cell* 19(6), 727–740.

Yang, Y., Liang, Y., Qu, L., Chen, Z., Yi, M., Li, K., et al. (2007). Disruption of innate immunity due to mitochondrial targeting of a picornaviral protease precursor. *Proc Natl Acad Sci U S A* 104(17), 7253–7258.

Yoneyama, M., & Fujita, T. (2008). Structural mechanism of RNA recognition by the RIG-I-like receptors. *Immunity* 29(2), 178–181.

Yoneyama, M., Kikuchi, M., Matsumoto, K., Imaizumi, T., Miyagishi, M., Taira, K., et al. (2005). Shared and unique functions of the DExD/H-box helicases RIG-I, MDA5, and LGP2 in antiviral innate immunity. *J Immunol* 175(5), 2851–2858.

Yoneyama, M., Kikuchi, M., Natsukawa, T., Shinobu, N., Imaizumi, T., Miyagishi, M., et al. (2004). The RNA helicase RIG-I has an essential function in double-stranded RNA-induced innate antiviral responses. *Nat Immunol* 5(7), 730–737.

Zaragoza, C., Saura, M., Padalko, E. Y., Lopez-Rivera, E., Lizarbe, T. R., Lamas, S., et al. (2006). Viral protease cleavage of inhibitor of kappaBalpha triggers host cell apoptosis. *Proc Natl Acad Sci U S A* 103(50), 19051–19056.

Zhang, M., Wu, X., Lee, A. J., Jin, W., Chang, M., Wright, A., et al. (2008). Regulation of IkappaB kinase-related kinases and antiviral responses by tumor suppressor CYLD. *J Biol Chem* 283(27), 18621–18626.

Zhang, N. N., Shen, S. H., Jiang, L. J., Zhang, W., Zhang, H. X., Sun, Y. P., et al. (2008). RIG-I plays a critical role in negatively regulating granulocytic proliferation. *Proc Natl Acad Sci U S A* 105(30), 10553–10558.

Zhao, C., Denison, C., Huibregtse, J. M., Gygi, S., & Krug, R. M. (2005). Human ISG15 conjugation targets both IFN-induced and constitutively expressed proteins functioning in diverse cellular pathways. *Proc Natl Acad Sci U S A* 102(29), 10200–10205.

Zhong, B., Yang, Y., Li, S., Wang, Y. Y., Li, Y., Diao, F., et al. (2008). The adaptor protein MITA links virus-sensing receptors to IRF3 transcription factor activation. *Immunity* 29(4), 538–550.
