
Atg32 Is a Mitochondrial Protein that Confers Selectivity during Mitophagy

Tomotake Kanki${}^{1,2,3,5}$ Ke Wang${}^{1,2,3}$ Yang Cao${}^{1,2,3}$ Misuzu Baba${}^{4}$ and Daniel J. Klionsky${}^{1,2,3,*}$

${}^{1}$Life Sciences Institute  
${}^{2}$Department of Molecular, Cellular, and Developmental Biology  
${}^{3}$Department of Biological Chemistry  
University of Michigan, Ann Arbor, MI 48109, USA  

${}^{4}$Department of Chemical and Biological Sciences, Faculty of Science, Japan Women’s University, Mejrodai, Tokyo 112-8681, Japan  

${}^{5}$Present address: Department of Clinical Chemistry and Laboratory Medicine, Kyushu University Graduate School of Medical Sciences, Fukuoka, 812-8582 Japan  

*Correspondence: klionsky@umich.edu  

DOI 10.1016/j.devcel.2009.06.014  

---

### SUMMARY

Mitochondrial quality control is important in maintaining proper cellular homeostasis. Although selective mitochondrial degradation by autophagy (mitophagy) is suggested to have an important role in quality control, and though there is evidence for a direct relation between mitophagy and neurodegenerative diseases, the molecular mechanism of mitophagy is poorly understood. Using a screen for mitophagy-deficient mutants, we found that YIL146C/ECM37 is essential for mitophagy. This gene is not required for other types of selective autophagy or for nonspecific macroautophagy. We designated this autophagy-related (ATG) gene as ATG32. The Atg32 protein localizes on mitochondria. Following the induction of mitophagy, Atg32 binds Atg11, an adaptor protein for selective types of autophagy, and is then recruited to and imported into the vacuole along with mitochondria. Therefore, Atg32 confers selectivity for mitochondrial sequestration as a cargo and is necessary for recruitment of this organelle by the autophagy machinery for mitophagy.

---

### INTRODUCTION

It is suggested that engulfment of mitochondria by autophagy occurs through a selective process, termed mitophagy, which might play an important role for mitochondrial quality control (Milijalica et al., 2007; Nowikovsky et al., 2007; Priault et al., 2005; Twig et al., 2008; Zhang et al., 2007). Direct evidence for mitophagy, however, has been limited. Mitophagy is now attracting considerable attention, because it is related with cellular differentiation and disease. For example, during erythroid cell maturation, mitochondria are eliminated by autophagy when Nix-dependent loss of membrane potential is induced (Sandoval et al., 2008; Schweers et al., 2007). Parkin is an E3 ubiquitin ligase encoded by the PARK gene, and a loss-of-function mutation in PARK2 represents the most common cause of Parkinson disease. Recently, Parkin is shown to be recruited selectively to impaired mitochondria and to promote their degradation via autophagy (Narendra et al., 2008). Despite the important role for mitophagy in development and disease, the molecular mechanism of mitophagy is not well understood.

Autophagy in yeast can be classified into several categories, and the best characterized of these are nonspecific and selective macroautophagy (Cao and Klionsky, 2007). Macroautophagy is the bulk degradation of cytoplasmic components that allows cells to respond to various types of stress and to adapt to changing nutrient conditions (Klionsky, 2005; Yorimitsu and Klionsky, 2007). Selective autophagy includes the cytoplasm to vacuole targeting (Cvt) pathway (Klionsky and Emr, 2000; Yorimitsu and Klionsky, 2005) and pexophagy (Dunn et al., 2005), which have the Cvt complex (aminopeptidase I [Ape1], along with receptor and adaptor proteins) and peroxisomes as specific cargo, respectively.

To date, 31 autophagy-related (ATG) genes have been identified, which function as the molecular machinery for autophagy. Most of the ATG genes are required for both nonspecific macroautophagy and selective autophagy, but some are required only for specific types of autophagy. For example, Atg19 functions as a receptor protein that binds the precursor form of Ape1 (prApe1). Atg19 also interacts with Atg11, an adaptor protein that is needed along with Atg19 to recruit the Cvt complex to the phagophore assembly site (PAS), where the sequestering cytosolic vesicles are generated (Shintani et al., 2002). Similarly, during pexophagy in *Pichia pastoris*, Atg30 localizes to peroxisomes. Atg11 binds Atg30 and recruits the peroxisomes to the

98 Developmental Cell 17, 98–109, July 21, 2009 ©2009 Elsevier Inc.

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

A
WT
Om45-GFP
DIC
atg32Δ
Om45-GFP
DIC

B
| WT | atg32Δ | atg1Δ | atg11Δ |
| --- | --- | --- | --- |
| 0 4 6 | 0 4 6 | 0 4 6 | 0 4 6 |

C
| pRS416: ATG32 | empty |
| --- | --- |
| SD-N (h): 0 6 | SD-N (h): 0 6 |
| Om45-GFP | Om45-GFP |
| kDa: 72 | kDa: 72 |
| 55 | 55 |
| 36 | 36 |
| GFP | 28 | 28 |

Figure 1. Atg32 Is Required for Mitophagy

(A) Wild-type (WT) and atg32Δ strains expressing Om45-GFP were cultured in YPL medium for 3 days. The localization of GFP was visualized by fluorescence microscopy. DIC, differential interference contrast. (B) Wild-type (WT), atg32Δ, atg1Δ and atg11Δ strains, or (C) the atg32Δ strain transformed with a vector expressing ATG32 (ATG32(416)) or the empty vector, and coexpressing Om45-GFP were cultured in YPL or SML medium to mid-log growth phase, then shifted to SD-N medium for 0, 4, and 6 hr. GFP processing was monitored by immunoblotting with anti-YFP antibody. The asterisk indicates a nonspecific band. The positions of molecular mass markers are indicated.

PAS (Farre et al., 2008). Atg11, Atg19, and Atg30 are not required for nonspecific macroautophagy. In both nonspecific and selective macroautophagy, the morphological hallmark is the sequestration of cargo within double-membrane cytosolic vesicles that deliver the contents to the vacuole. Fusion with the vacuole limiting membrane releases the inner vesicle into the lumen, where it is degraded, which allows access to the cargo. During selective types of autophagy, the membrane of the sequestering vesicle is closely apposed to the cargo, resulting in the exclusion of bulk cytoplasm. Thus, specific tags are recognized on the cargo to allow them to be selectively targeted. In the case of the Cvt pathway, this includes the propeptide of prApe1 (Shintani et al., 2002), whereas in pexophagy it is the peroxin Pex14 (Farre et al., 2008).

Most of the ATG genes are required for selective mitochondrial autophagy; however, because these genes are also needed for nonspecific macroautophagy, it is unclear whether there is a specific mechanism that selects mitochondria as a cargo. Recently, we established a sensitive method to monitor mitophagy that relies on the marker protein Om45-GFP and found that the cargo adaptor protein Atg11 is essential for mitophagy, suggesting that mitochondria are selectively imported into the vacuole (Kanki and Klionsky, 2008). Using the Om45-GFP marker, we screened a yeast knockout library for strains that are deficient in mitophagy. We found that YIL146C/ECM37 is essential for mitophagy, but is not required for other types of selective autophagy or for nonspecific macroautophagy. We designated this autophagy-related gene as ATG32 and characterized the Atg32 protein in the present study.

RESULTS

Screening and Identification of ATG32 as a Mitophagy-Related Gene

To monitor mitophagy, we tagged the C terminus of the mitochondrial outer membrane protein Om45 with green fluorescent protein (GFP), and induced mitophagy by culturing cells in medium with lactate as the sole carbon source (YPL) for more than 2 days (Kanki and Klionsky, 2008). When mitochondria are delivered to the vacuole in wild-type cells, GFP fluorescence can be observed within the lumen of this organelle (Figure 1A, WT). To screen for mutants that affect mitophagy, we transformed a DNA fragment into the yeast knockout library to tag GFP at the chromosomal locus for OM45, induced mitophagy, and monitored the presence or absence of the vacuolar GFP fluorescence. From this screen, we identified a strain with a deletion in the YIL146C/ECM37 gene, which was first characterized as part of a large-scale screen for mutants sensitive to calcofluor white (Lussier et al., 1997); the knockout strain was not previously known to affect mitochondrial degradation. Based on the delivery of Om45-GFP to the vacuole, however, we found that the yil146cΔ strain was completely blocked in mitophagy (Figure 1A, atg32Δ). Accordingly, we named this gene ATG32. The ATG32 gene encodes a 529 amino acid protein of predicted molecular mass 58,968 Da. It has one predicted transmembrane domain in the C-terminal fifth of the protein; there are no other predicted domains or functional motifs. ATG32 does not have obvious homologs in higher eukaryotes, but does have putative homologs in other fungi including Candida glabrata, Kluyveromyces lactis, and Eremothecium gossypii.

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

Figure 2. Atg32 Is a Mitophagy-Specific Protein

(A) Wild-type (WT), atg32Δ, atg1Δ, and atg11Δ strains expressing GFP-Atg8 were cultured in SMD medium to mid-log phase, and then starved in SD-N for up to 3 hr. Cells were collected at the indicated time points and cell lysates were subjected to immunoblot analysis with anti-YFP antibody.

(B) The wild-type (TN124), atg32Δ (TKYM131), and atg1Δ (HAY572) strains were grown in YPD and shifted to SD-N for 1.5 and 3 hr. Samples were collected and protein extracts assayed for Pho8Δ60 activity. The results represent the mean and standard deviation (SD) of three experiments.

(C) Wild-type, atg32Δ, atg1Δ and atg11Δ strains were cultured in YPD medium and analyzed for prApe1 maturation by immunoblotting to monitor the Cvt pathway during vegetative growth. The positions of precursor and mature Ape1 are indicated.

(D) Wild-type, atg32Δ, and atg1Δ strains were integrated with GFP at the PEX14 locus. Cells were grown in oleic acid-containing medium for 19 hr, then shifted to SD-N for the indicated times. Samples were collected and analyzed by immunoblot with antibodies to YFP.

Om45-GFP delivery to the vacuole. In this case, GFP-Atg8 is a marker for the phagophore, the initial sequestering compartment that forms the autophagosome, and a population of this protein remains associated with the completed vesicle. Wild-type cells showed the accumulation of free GFP from GFP-Atg8 after cells were shifted to medium lacking nitrogen to induce autophagy (Figure 2A). The atg1Δ mutant maintained the full-length fusion protein and did not generate any of the processed product. In contrast, the atg11Δ mutant was able to proteolytically process GFP-Atg8 essentially the same as the wild-type. Similarly, the atg32Δ strain displayed processed GFP-Atg8 and was indistinguishable from wild-type cells, suggesting that nonspecific autophagy was not defective in this strain. To confirm this observation, we relied on a second assay that provides a more quantitative measure of nonspecific autophagy, activation of the precursor form of Pho8Δ60 (Noda et al., 1995). The wild-type strain showed a time-dependent increase in Pho8Δ60-dependent alkaline phosphatase activity following a shift to medium lacking nitrogen, whereas the atg1Δ strain maintained a background level of activity (Figure 2B). As with GFP-Atg8 processing, the atg32Δ strain displayed a level of autophagy activity that was similar to the wild-type.

The best-characterized example of selective autophagy is the Cvt pathway, which is used for delivery of the resident hydrolase Ape1 to the vacuole (Klionsky and Emr, 2000). We examined processing of prApe1 in wild-type and control strains and compared it with that in the atg32Δ mutant. The wild-type strain showed

ATG32 Is a Mitophagy-Specific Gene

In addition to post-logarithmic phase growth in lactate medium, mitophagy can be induced when cells are shifted from lactate medium (YPL) to nitrogen-starvation medium (SD-N), and the level of mitophagy can be semiquantitatively monitored by measuring the amount of GFP processed from Om45-GFP in the vacuole (Kanki and Klionsky, 2008). Consistent with the microscopy observation, the atg32Δ mutation blocked mitophagy completely, similar to the result seen with the atg1Δ strain, which is defective in both selective and nonspecific autophagy, and the atg11Δ strain, which is blocked only in selective autophagy pathways (Figure 1B). The mitophagy deficiency of the atg32Δ strain was complemented by exogenously introducing ATG32, confirming that the defect is the direct result of the absence of the Atg32 protein (Figure 1C).

Next, we decided to determine whether the ATG32 gene product is required for nonspecific macroautophagy or other types of selective autophagy. The GFP-Atg8 processing assay (Shintani and Klionsky, 2004) is similar to that used for monitoring

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

essentially complete maturation of prApe1 under steady-state conditions, whereas the atg1Δ and atg11Δ strains were completely defective for processing (Figure 2C). The atg32Δ strain showed normal prApe1 maturation, suggesting that Atg32 is not needed for the Cvt pathway. Finally, we used one additional GFP processing assay to monitor another type of selective autophagy, pexophagy. In this case, the marker protein is Pex14 fused to GFP (Reggiori et al., 2005). The wild-type and atg1Δ strains again served as positive and negative controls, respectively (Figure 2D). The atg32Δ strain displayed processing of Pex14-GFP after shifting cells from medium with oleic acid to glucose minus nitrogen, similar to the wild-type. Thus, these results indicate that ATG32 is a mitophagy-specific gene that is not required for either nonspecific autophagy or other types of selective autophagy.

To further examine the phenotype associated with loss of ATG32, we monitored cell growth on a nonfermentable carbon source and observed no difference from the wild-type (see Figure S1 available online). Similarly, the level of a mitochondrial marker protein, F₁-β, and mitochondrial DNA were indistinguishable from the wild-type (Figure S1B and S1C). Next, we examined loss of viability in starvation conditions. Although the autophagy-deficient atg1Δ strain showed a complete loss of viability after 6 days of starvation, both the wild-type and atg32Δ strains grew to similar levels (Figure S1D), in agreement with our finding that ATG32 is not required for macroautophagy. Finally, we examined cellular production of ROS for cells grown in YPL or shifted to SD-N. The ROS production in the atg32Δ strain was similar to the wild-type in both conditions (Figure S1E). From our present results, we can only conclude that Atg32 plays a critical role in eliminating superfluous organelles (e.g., after cells have been shifted to glucose or reach stationary phase) for adaptation to changing environmental conditions.

Mitochondria are eliminated from mammalian cells when they are dysfunctional, for example following treatment with CCCP (Narendra et al., 2008); however, this treatment does not induce mitophagy in yeast (Kanki and Klionsky, 2008). To examine whether mitophagy in yeast in response to damaged mitochondria is Atg32 dependent, we relied on doxycycline-regulated depletion of Mdm38 (Kanki and Klionsky, 2008; Nowikovsky et al., 2007). When Mdm38 was depleted we detected a very weak mitophagy response (Figure S2), similar to our previous results (Kanki and Klionsky, 2008), which might reflect incomplete loss of the Mdm38 protein. Nonetheless, processing of Om45-GFP in response to the shutoff of MDM38 expression was completely blocked in the absence of Atg32, suggesting that the latter protein is involved in the elimination of damaged as well as superfluous mitochondria.

### Atg32 Localizes to Mitochondria and Is Imported into the Vacuole along with Mitochondria during Starvation

To further characterize Atg32, we chromosomally tagged the N terminus, which did not affect its function (Figure S3). To examine the localization of Atg32, we expressed GFP-Atg32 under the control of the regulatable GAL1 promoter in galactose medium (YPGal). First, we examined the localization of GFP-Atg32 by co-labeling the cells with the mitochondrial marker dye MitoFluor Red 589, and observed complete colocalization (Figure 3A). After nitrogen starvation for 2 hr, some of the GFP-Atg32 accumulated in the vacuole (Figure 3B). To determine whether this accumulation of GFP-Atg32 within the vacuole lumen is a result of mitochondrial vacuolar targeting, we tagged RFP on the C terminus of the mitochondrial outer membrane protein Om45 (Om45-RFP). In growing conditions, GFP-Atg32 showed a punctate and tubular morphology typical of yeast mitochondria and completely overlapped with Om45-RFP (Figure 3C, upper panel), again confirming the mitochondrial localization of Atg32. After nitrogen starvation, Om45-RFP accumulated in the vacuole along with GFP-Atg32, suggesting that GFP-Atg32 was targeted to the vacuole along with mitochondria (Figure 3C lower panel).

We further examined the mitochondrial localization of Atg32 using a biochemical approach. A strain expressing protein-A-tagged Atg32 (PA-Atg32), 3xHA-tagged Om45, and PA-tagged Tim23 (inner membrane marker) was fractionated by differential centrifugation. Om45-3HA and Tim23-PA were enriched in the mitochondrial (6500 x g) fraction, along with PA-Atg32 (Figure 3D), whereas the cytosolic marker Pgk1 was mostly in the supernatant fraction. Next, the isolated mitochondria were treated with proteinase K in the presence or absence of Triton X-100. Although Tim23-PA was protected from proteinase K in the absence of detergent, PA-Atg32 was degraded, suggesting that Atg32 localizes on the mitochondrial outer membrane (Figure 3E). Finally, we treated isolated mitochondria with 0.1 M sodium carbonate (pH 11) and separated the membrane (100,000 x g pellet) and supernatant fractions. The peripheral inner membrane β subunit of the F₁-ATPase was released into the supernatant, whereas PA-Atg32 was detected only in the membrane fraction along with the majority of Om45 and Tim23 (Figure 3F), suggesting that Atg32 is a transmembrane protein.

### Atg32 Is a Receptor for Recruiting Mitochondria to the Vacuole

Atg11 serves as an adaptor for the Cvt pathway and pexophagy by linking cargo to the autophagy machinery. Our previous result showing that Atg11 is essential for mitophagy (Kanki and Klionsky, 2008) led us to propose that Atg11 might interact with Atg32 as a receptor on mitochondria. To examine this possibility, a yeast two-hybrid assay was performed between Atg11 fused with the Gal4 DNA binding domain, BD-Atg11, and activation domain-fused Atg32, AD-Atg32. The cells expressing BD-Atg11 and AD-Atg32, but not those expressing either the activation or binding domains as empty vectors, could grow on selective plates (Figure 4A), suggesting that Atg11 can bind Atg32. To verify the physiological nature of this interaction, we expressed PA-Atg32 and HA-Atg11 in an atg32Δ strain and carried out protein A (PA) affinity isolation with immunoglobulin G (IgG)-Sepharose (Figure 4B). Consistent with the yeast two-hybrid result, PA-Atg32 coprecipitated HA-Atg11 in growing conditions (SMD), although the amount of coprecipitated HA-Atg11 was extremely minimal (Figure 4B, lane 3); however, a 1 hr shift to nitrogen starvation (SD-N) conditions dramatically increased the amount of coprecipitated HA-Atg11 (Figure 4B, lane 6), suggesting that nitrogen starvation induces Atg11-Atg32 binding. This result is consistent with the previous finding that nitrogen starvation can induce mitophagy (Kanki and Klionsky, 2008; Kiššová et al., 2007).

### Overexpression of Atg32 Enhances the Induction of Mitophagy

If Atg32 is a mitochondrial tag, we hypothesized that its overexpression would enhance the induction of mitophagy. To test this,

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

Figure 3. Atg32 Localizes to Mitochondria and Is Imported into the Vacuole along with Mitochondria during Starvation

(A) A strain expressing GFP-Atg32 under the control of the GAL1 promoter (TKYM136) was cultured in YPGal medium to mid-log phase and labeled with the mitochondrial marker MitoFluor Red 589. The localization of GFP and MitoFluor Red were visualized by fluorescence microscopy.

(B) Strain TKYM136 was cultured in YPGal medium to mid-log phase and the shifted to starvation medium (SD-N) for 2 hr. Cells were labeled with the vacuolar membrane marker, FM 4-64. The localization of GFP and FM 4-64 were visualized by fluorescence microscopy.

(C) A strain expressing GFP-Atg32 and Om45-RFP (TKYM145) was cultured in YPGal medium to mid-log phase and then shifted to starvation medium (SD-N) for 2 hr. The localization of GFP and RFP were visualized by fluorescence microscopy.

(D) Mitochondria were purified from atg11Δ cells expressing Om45-3HA, Tim23-PA and PA-Atg32 as described in Experimental Procedures. Equal amounts of the total cell homogenate (T), mitochondrial (M), and supernatant (S) fractions were loaded.

(E) Isolated mitochondria were treated with proteinase K (PK) with or without Triton X-100, or

(F) with 0.1 M sodium carbonate, pH 11.0 as described in Experimental Procedures. Samples were TCA precipitated and subjected to immunoblotting.

we monitored mitophagy in galactose medium with and without the overexpression of Atg32. In wild-type cells overexpressing the vector (pCu-PA), Om45-GFP processing could not be detected by Western blot (Figure 4C). In contrast, when PA-Atg32 was overexpressed (pCu-PA-Atg32), Om45-GFP was processed to generate free GFP. We noted that the overexpressed Atg32 was quickly degraded during starvation (Figure 4C); however, the majority of this degradation was pep4Δ-independent, indicating that it did not occur in the vacuole (data not shown). Nonetheless, Om45-GFP processing was blocked in atg1Δ and atg11Δ strains, indicating that it was dependent on autophagy (Figure S4).

A Portion of Mitochondria with Atg32 Can Target to the Vacuolar Surface

During the Cvt pathway, the cargo protein prApe1 forms a complex in the cytosol, and is targeted to the perivacuolar PAS. The Cvt complex is then enwrapped by a Cvt vesicle, which eventually fuses with the vacuole. To see whether mitochondria target to the vacuole through a similar pathway, we monitored GFP-Atg32 localization as a marker of mitochondria using atg1Δ, atg11Δ, and pep4Δ strains. In the atg11Δ strain, GFP-Atg32 showed a typical mitochondrial morphology that localizes largely to the inner side of the plasma membrane; it was difficult to detect any other cytosolic puncta in both growing and starvation conditions (~1% of the cells had puncta on the vacuolar rim in either condition) (Figure 5A). Presumably, the lack of the adaptor protein Atg11 prevented mitochondria from being recruited to the vacuole. In the atg1Δ strain, most of the Atg proteins and the Cvt complex are recruited to the PAS (Suzuki et al., 2007), but later steps in the autophagic process are blocked. GFP-Atg32 showed cytosolic puncta that localized near the vacuole membrane during starvation in the atg1Δ strain (35.6% of the cells had puncta on the vacuolar rim during starvation, compared to 0.6% in growing conditions; Figure 5B, arrowheads); this result was similar to the wild-type strain (33.3% and 0% of the cells with vacuolar rim puncta in starvation and growing conditions, respectively; Figure S5), except that in the wild-type strain a fluorescent signal appeared in the vacuole lumen during starvation. In a pep4Δ strain, which lacks the one of the main vacuolar hydrolases, GFP-Atg32 puncta accumulated within the vacuole during starvation (Figure 5C). The change in GFP-Atg32 localization could also be seen during a time-course analysis (Figure S6).

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

In the *atg1Δ* strain, prApe1 and Atg8 accumulate at a specific point near the vacuole surface that is thought to represent the PAS (Shintani et al., 2002). Most of the GFP-Atg32 puncta did not colocalize with CFP-Ape1 in the *atg1Δ* strain (Figure 5D) or a wild-type strain (data not shown), suggesting that mitophagy generally does not occur simultaneously at the same PAS that is being used for the Cvt pathway and nonspecific autophagy. Next, we coexpressed GFP-Atg32 and either CFP-Atg8 or CFP-Atg11, which also form puncta at the PAS with Ape1, and observed their localization in the *atg1Δ* strain. The majority of the GFP-Atg32 puncta on the vacuole rim overlapped with CFP-Atg8 and CFP-Atg11 puncta, although only approximately half of the CFP-Atg8 and CFP-Atg11 puncta overlapped with GFP-Atg32 (Figures 5E and 5F). A similar result was seen with the colocalization of these proteins in a wild-type strain (Figure S7). This finding is consistent with our observation that Atg11 binds Atg32, our previous report that Atg11 accumulates at the PAS for the Cvt pathway and macroautophagy (Yorimitsu and Klionsky, 2005), and with the known role of Atg8 in autophagic sequestration of cargo. Finally, we coexpressed GFP-Atg32 and the mitochondrial matrix protein Idh1-RFP. After starvation for 2 hr, the GFP-Atg32 puncta on the vacuole colocalized with Idh1-RFP (Figure S8), further verifying that Atg32 movement to the vacuole corresponds to mitochondria.

The GFP-Atg32 perivacuolar puncta appeared as a more intense fluorescent signal than other parts of the mitochondria. The intensity of the GFP-Atg32 perivacuolar puncta was 3.3 ± 1.2 times greater than the intensity of GFP-Atg32 on mitochondria when the GFP signal was standardized relative to Idh1-RFP (Figure S9). However, the GFP-Atg32 puncta in the vacuole in the *pep4Δ* strain was comparable with that at the PAS in the *atg1Δ* strain (Figures 5B and 5C), indicating that the GFP-Atg32 puncta, once targeted to the vacuole surface, are eventually delivered into the vacuole.

### Electron Microscopy Analysis of Mitophagy

To further examine the localization of Atg32 and its targeting to the vacuole associated with mitochondria, we used immunoelectron microscopy. The *pep4Δ* strain expressing GFP-Atg32 under the *GAL1* promoter was cultured in YPGal medium and then shifted to SD-N for 4 hr. Cells were immunostained with anti-YFP antibody and observed by electron microscopy. In YPGal, mitochondria were detected in the cytosol and Atg32 localized on their surface (Figures 6A and 6B), further confirming our biochemical observation that Atg32 localizes on the mitochondrial outer membrane. After starvation, mitochondria stained with Atg32 were delivered into the vacuole (Figures 6C and 6D).

In SD-N, the mitochondria were detected primarily within double-membrane vesicles. These results suggest that under

from cell lysates. The upper two panels show the immunoblot of total cell lysates (input, equivalent to $A_{600} = 0.08$ unit of cells were loaded) and the lower two panels show the IgG precipitates (IP, equivalent to $A_{600} = 2.50$ units of cells were loaded), which were probed with anti-HA and anti-PA antibody.

(C) A strain expressing Om45-GFP (TKYM22) transformed with a vector expressing protein A only (PA) or PA-Atg32 was cultured in SMGal medium to the mid-log growth phase, then shifted to SD-N medium for 0, 2, 4, and 6 hr. The level of GFP processing, and the amount of PA and PA-Atg32 were monitored by immunoblotting with anti-YFP antibody and anti-PA antibody, respectively. The positions of molecular mass markers are indicated.

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

A atgl1Δ, growing
GFP-Atg32 FM 4-64 overlay
atgl1Δ, SD-N (1.5 h)

B atglΔ, growing
GFP-Atg32 FM 4-64 overlay
atglΔ, SD-N (1.5 h)

C pep4Δ, SD-N (1.5 h)
GFP-Atg32 FM 4-64 overlay

D atglΔ, SD-N (2 h)
GFP-Atg32 CFP-Ape1 FM 4-64 overlay

E atglΔ, SD-N (2 h)
GFP-Atg32 CFP-Atg8 FM 4-64 overlay

F atglΔ, SD-N (2 h)
GFP-Atg32 HA-CFP-Atg11 FM 4-64 overlay

Figure 5. Mitochondria Target to the Vacuole through a Cvt-Pathway-Independent Mechanism

The (A) atgl1Δ, (B, D-F) atglΔ, and (C) pep4Δ strains transformed with a plasmid expressing (A–C) GFP-tagged Atg32 under the control of the CUP1 promoter (pCuGFP-Atg32) and (D) pCFP-Ape1, (E) pCFP-Atg8, or (F) pHAt-CFP-Atg11 were cultured in SMD medium, then shifted to starvation medium (SD-N) for 1.5 or 2 hr. Cells were labeled with the vacuolar marker FM 4-64 and analyzed by fluorescence microscopy. The arrowheads in (B) and (D) mark GFP-Atg32, and the arrows mark nonoverlapping CFP-Ape1 (D). The arrowheads and arrows in (E) and (F) mark overlapping and nonoverlapping GFP-Atg32 and CFP-Atg8 or CFP-Atg11 puncta, respectively.

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

Growing                                                                 SD-N (4 h)

![Image](image1.png) ![Image](image2.png)
![Image](image3.png) ![Image](image4.png)

Figure 6. Atg32 Localizes on Mitochondria and Targets to the Vacuole during Mitophagy

Cells expressing GFP-Atg32 under the GAL1 promoter were cultured in YPGal to mid-log phase (A and B), then shifted to SD-N for 4 hr (C and D), prepared for electron microscopy using freeze substitution and stained with anti-YFP antibody followed by immunogold as described in Experimental Procedures. The arrow marks an example of a mitochondrion that is contained within a double-membrane autophagosome. AB, autophagic body; AP, autophagosome; M, mitochondria; V, vacuole; N, nucleus. Scale bar, 500 nm.

the conditions we used to induce mitophagy, the majority of sequestration occurred by a macroautophagic process. In addition, the vesicles containing the mitochondria, mitophagosomes, appeared to be devoid of bulk cytosol, further indicating the selective nature of the process; during specific types of autophagy such as the Cvt pathway and pexophagy, the targeted cargo is closely apposed to the sequestering vesicle membrane and bulk cytosol is largely excluded.

DISCUSSION

The presence of mitochondria in autophagosomes was first reported in 1957 in mammalian cells (Clark, 1957). Because of connections with damaged mitochondria, aging, and disease, a major question has been whether mitochondrial engulfment by autophagosomes can occur in a selective manner. Recent studies suggest that mitochondrial autophagy can occur by a selective process (Elmore et al., 2001; Nowikovsky et al., 2007; Priault et al., 2005; Rodriguez-Enriquez et al., 2004; Twig et al., 2008). Because some ATG genes essential for both macroautophagy and the Cvt pathway are required for mitophagy (Kiššová et al., 2004, 2007; Tal et al., 2007; Zhang et al., 2007), the fundamental mechanism of mitophagy is believed to be similar to other types of autophagy. However, the mechanism of mitochondrial selection, which is essential for mitochondrial quality control, has not been known. This information might provide

Developmental Cell 17, 98–109, July 21, 2009 ©2009 Elsevier Inc. 105

important insight into certain neurodegenerative diseases, such as Parkinson disease, as well as mechanisms involved in cellular differentiation.

Based on the examples of other types of selective autophagy, including the Cvt pathway and pexophagy, we hypothesized that an organelle tag would be used to mark mitochondria and that this tag would be recognized by a component of the autophagy machinery. To identify such a tag, we performed a mitophagy screen and found that the *atg32Δ* strain is completely blocked in mitophagy (Figure 1). Atg32 is not required for nonspecific macroautophagy, the Cvt pathway, or pexophagy, suggesting that Atg32 is a mitophagy-specific protein (Figure 2). Two genes have been reported to be required for mitophagy, *AUP1* and *UTH1* (Kiššová et al., 2004; Tal et al., 2007). Aup1 is a mitochondrial protein phosphatase, and strains with null mutations in this gene show rapamycin sensitivity (Ruan et al., 2007). However, Uth1 is a SUN family protein, and in this case null strains show rapamycin resistance (Kiššová et al., 2004) and display a partial inhibition of mitophagy; these strains are blocked at an early stage of mitophagy (Kiššová et al., 2007). The functions of these proteins with regard to mitophagy have not been clarified. We examined these null strains for their affects on mitophagy; however, in our hands the corresponding knockout strains showed completely normal levels of mitophagy (Figure S10). Furthermore, the *aup1Δ* strain purchased from Research Genetics/Invitrogen (BY4742, *aup1Δ::KanR*) blocked mitophagy, whereas an *aup1Δ* strain that we constructed using the same (data not shown) and a second different genetic background did not block mitophagy. The difference between our results and a previous report showing a 40% reduction of mitophagy in the *uth1Δ* strain (Kiššová et al., 2007) might be due to the difference in strain background and the detection method. Thus, *atg32Δ* is the first identified mutant, to our knowledge, that shows a complete block only in specific mitophagy and not in the Cvt pathway or nonspecific macroautophagy. In addition, our data indicate that Atg32 can connect the Atg proteins and mitochondria directly, providing an explanation for the mechanism of selectivity during mitophagy.

We found that Atg32 localized on mitochondria and could interact with the adaptor protein Atg11, an interaction that was enhanced under mitophagy conditions (Figures 4 and 5). By interacting with Atg11, Atg32 is delivered to the vacuole surface and then delivered into the lumen. The last step of mitochondrial delivery into the vacuolar lumen is still unclear. Kiššová et al., (2007) reported that mitochondria are directly sequestered via microautophagy; however, the majority of our electron microscopy data suggest that uptake occurs via macroautophagy (Figure 6C, arrow), although we cannot rule out a microautophagic mechanism.

In yeast, both the Cvt pathway and nonspecific macroautophagy use the PAS, which is thought to be the site of Cvt vesicle and autophagosome formation, respectively. Most of the Atg proteins and prApe1, the principal cargo of the Cvt pathway, accumulate at the PAS. The autophagosome or Cvt vesicle is then formed, and eventually this compartment fuses with the vacuole. Because these Atg proteins and prApe1 accumulate at the same point on the vacuolar surface, it is thought that macroautophagy and the Cvt pathway are using a common PAS, although there is no experimental evidence to support this hypothesis. To see whether the Cvt pathway and mitophagy use the same PAS or a different pathway, we used the *atg1Δ* strain. This strain accumulates Atg proteins and prApe1 at the PAS, but blocks further steps in the sequestration process. We then monitored the localization of GFP-Atg32 as a marker of mitochondria, and CFP-Ape1 as a marker of the Cvt-dependent PAS. GFP-Atg32 formed strong puncta near the vacuole surface, but usually did not colocalize with the PAS that is marked by CFP-Ape1 puncta during starvation (Figure 5D). This might be expected based on the observation that the membranes that form during selective autophagy closely appose the particular cargo being enwrapped. Thus, engulfment of prApe1 likely occurs at a distinct site relative to sequestration of mitochondria. During starvation conditions, we infrequently observed CFP-Ape1 puncta that colocalized with GFP-Atg32 (data not shown); in this case both cargos were presumably present at a larger phagophore that was a precursor to an autophagosome.

We also monitored the localization of GFP-Atg32 and CFP-Atg8 or CFP-Atg11 in *atg1Δ* and wild-type strains. Atg8 determines the size of nonselective autophagosomes (Xie et al., 2008) and accumulates at the PAS during autophagy. In agreement with the requirement of Atg8 for mitophagy (Kanki and Klionsky, 2008; Zhang et al., 2007), CFP-Atg8 partially colocalized with GFP-Atg32 (Figures 5E and S7). Similarly, CFP-Atg11, which is required for the Cvt pathway and binds prApe1, formed multiple puncta on the vacuole surface and approximately half of them displayed colocalization with the GFP-Atg32 puncta (Figures 5F and S7). This finding is consistent with our data that Atg32 binds Atg11. Presumably, the remaining (i.e., nonoverlapping) CFP-Atg8 and CFP-Atg11 puncta localize at the PAS used for the Cvt pathway and nonspecific autophagy. Finally, we noticed that the GFP-Atg32 puncta observed near the vacuolar surface of the *atg1Δ* strain or in the vacuolar lumen of the *pep4Δ* strain generated a more intense fluorescent GFP signal compared with that on mitochondria localized just inside of the plasma membrane (Figures 5D–5F, S4, and S9).

Our current model for mitophagy relative to the Cvt pathway and pexophagy is summarized in Figure 7. All three processes require Atg11 as an adaptor protein. Atg11 interacts with the Cvt pathway receptor Atg19 to recruit the prApe1-Ams1-Atg19 complex to the PAS (Shintani et al., 2002), and a slightly modified PAS is used in common with nonspecific macroautophagy (Cheong et al., 2008). Following the induction of pexophagy in *Pichia pastoris*, PpAtg30 binds the peroxisomal protein PpPex14 and is phosphorylated. PpAtg30 further interacts with PpAtg11, allowing recruitment of the peroxisome to the PAS for both macropexophagy and micropexophagy (Farre et al., 2008). In *Saccharomyces cerevisiae*, the requirement of Atg11 for pexophagy has already been reported (Kim et al., 2001), but other steps similar to those found in *Pichia pastoris* have not been demonstrated. When mitophagy is induced, Atg11 binds the mitochondrial resident protein Atg32. Atg11 recruits mitochondria to the vacuole surface, where uptake might occur by a microautophagic process or by a type of macroautophagy that is separate from the PAS that is used for the Cvt pathway. Clearly additional experiments will be needed to elucidate the details of this selective method of organelle targeting.

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

been described previously (Shintani et al., 2002; Yorimitsu and Klionsky, 2005). To generate an Atg32-expressing plasmid [ATG32(416)], a DNA fragment encoding the ATG32 gene including sequences from the promoter and terminator regions was polymerase chain reaction (PCR) amplified from yeast genomic DNA with primers that included SpeI and EcoRI restriction sites, and ligated into the corresponding sites of the pRS416 vector. For the yeast two-hybrid vector pGAD-Atg32, the PA-tagged (pCuPA-ATG32[416]), and GFP-tagged (pCuGFP-ATG32[416]) Atg32 expression vectors, a DNA fragment encoding ATG32 was PCR amplified with BglII and SalI, or EcoRI and SalI restriction sites from yeast genomic DNA and ligated into the BamHI and SalI sites of pGAD-C1 (James et al., 1996), the EcoRI and SalI sites of pRS416-CuProtA (Kim et al., 2001), and the EcoRI and SalI sites of pCuGFP(416) (Kim et al., 2001), respectively. Monoclonal anti-YFP antibody clone JL8 (Clontech/Takara Bio Group, Mountain View, CA), monoclonal anti-HA antibody clone-HA7 (Sigma-Aldrich, St. Louis, MO), anti-Ape1 serum (Kim et al., 2001), anti-ATPase F1-β antiserum (Dr. Michael Douglas, Washington University, Saint Louis), and anti-Pgk1 antiserum (Dr. Jeremy Thorner, University of California, Berkeley) were used for immunoblotting.

### Mitophagy Screening

A yeast knockout strain library (BY4742 background) was purchased from Research Genetics/Invitrogen. To express Om45-GFP, a DNA fragment encoding GFP was integrated at the 3′ end of OM45 by a PCR-based integration method (Longtine et al., 1998). Cells grown on SMD plates were shifted to YPL medium and cultured for 3 days (60 ± 5 hr), then observed by fluorescence microscopy.

### Assays for Nonspecific Autophagy, Mitophagy, and Pexophagy

For monitoring nonspecific autophagy, the alkaline phosphatase activity of Pho8Δ60, and processing of the GFP-Atg8 chimera were carried out as described previously (Noda et al., 1995; Shintani and Klionsky, 2004). Om45-GFP processing to monitor mitophagy (Kanki and Klionsky, 2008) and Pex14-GFP processing to monitor pexophagy (Reggiori et al., 2005), have been described previously.

### Fluorescence Microscopy

Yeast cells expressing fluorescent protein-fused chimeras were grown to mid-log phase or starved in the indicated media. To label the vacuolar membrane or mitochondria, we incubated cells in medium containing 20 μg/ml N-(3-triethylammoniumpropyl)-4-(p-diethylaminophenylhexatrienyl) pyridinium dibromide (FM 4-64; Molecular Probes, Eugene, OR) or 1 μM MitoFluor Red 589 (Molecular Probes) at 30°C for 30 min, respectively. After being washed with medium, the cells were incubated in medium at 30°C for 30 to 60 min. Fluorescence microscopy observation was carried out as described previously (Monastyrska et al., 2008). To quantify the number of cells that have GFP-Atg32 puncta on the vacuole surface, we counted 80 to 100 cells for each strain.

### Protein A Affinity Isolation

Protein A affinity isolation was carried out essentially as described previously (Shintani et al., 2002).

### Subcellular Localization

We followed the protocol for cellular fractionation as described previously (Graham, 2001). Cells expressing Om45-3HA, Tim23-PA, and PA-Atg32 were converted to spheroplasts with Zymolyase, suspended in homogenization buffer (0.6 M mannitol, 20 mM HEPES (pH 7.4), and protease inhibitors) and homogenized in a Dounce homogenizer. The cell homogenate was centrifuged at 600 x g for 10 min to remove the nucleus and unbroken cells. The supernatant fraction was centrifuged at 6500 x g for 10 min. The pellet was collected as the mitochondrial fraction. Isolated mitochondria were suspended in ice-cold suspension medium (0.6 M mannitol and 20 mM HEPES [pH 7.4]) and treated with protease K (200 μg/ml) for 30 min on ice with/without 0.5% Triton X-100. The protease K reaction was stopped by adding 10% trichloroacetic acid (TCA). Isolated mitochondria were separately suspended in ice-cold 0.1 M sodium carbonate (pH 11.0), incubated for 30 min on ice, and then centrifuged at 100,000 x g for 30 min at 4°C. Proteins in the pellet and supernatant fractions were precipitated by adding 10% TCA. TCA precipitated proteins were washed with acetone and subjected to immunoblotting.

---

**Figure 7. Model of Selective Autophagy**

The cargos of the Cvt pathway, prApe1 and Ams1, form a complex with the receptor protein Atg19. Atg11 interacts with Atg19 to recruit the prApe1-Ams1-Atg19 complex to the PAS, the site of Cvt vesicle formation. When pexophagy is induced in *Pichia pastoris*, PpAtg30 binds the peroxisomal protein PpPex14. PpAtg30 further interacts with PpAtg11, allowing recruitment of the peroxisome to the PAS for both macropexophagy and micropexophagy. When mitophagy is induced, the mitochondrial protein Atg32 binds Atg11. Atg11 recruits mitochondria to the vacuolar surface, where uptake might occur via the PAS (through a macroautophagy-like process) or independent of this structure (through a microautophagy-like process).

---

### EXPERIMENTAL PROCEDURES

#### Strains, Media, and Enzymes

The yeast strains used in this study are listed in Table S1 (available online). Yeast cells were grown in the following media: rich (YPD; 1% yeast extract, 2% peptone, 2% glucose), lactate (YPL; 1% yeast extract, 2% peptone, 2% lactate), galactose (YPGal; 1% yeast extract, 2% peptone, 2% galactose), synthetic minimal with glucose (SMD; 0.67% yeast nitrogen base, 2% glucose, amino acids, and vitamins), synthetic minimal with lactate (SML; 0.67% yeast nitrogen base, 2% lactate, amino acids, and vitamins) or synthetic minimal with galactose (SMGal; 0.67% yeast nitrogen base, 2% galactose, amino acids, and vitamins). Starvation experiments were performed in synthetic minimal medium lacking nitrogen (SD-N; 0.17% yeast nitrogen base without amino acids, 2% glucose).

#### Plasmids and Antibodies

The plasmids to express hemagglutinin (HA)-tagged Atg11, CFP-Ape1, CFP-Atg8, HA-CFP-Atg11, and the yeast two-hybrid plasmid pGBDU-Atg11 have

Developmental Cell 17, 98–109, July 21, 2009 © 2009 Elsevier Inc. 107

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

**Electron Microscopy**

The *pep4Δ* strain expressing GFP-Atg32 under the *GAL1* promoter (TKYM151) was cultured in YPGal medium, then shifted to SD-N and cultured for 4 hr. Cells were frozen in a Balzer’s HPM010 high-pressure freezer (Bal-tec, Lichtenstein) or KF80 freezing device (Leica, Austria). Immunoelectron microscopy was performed according to the procedures described previously ([Baba, 2008](http://www.cell.com/developmental-cell/supplement/S1534-5807(09)00255-X)). Ultrathin sections were stained with anti-YFP antibody, followed by 0.8 nm colloidal-gold-conjugated goat anti-mouse IgG (Aurion, Wageningen, The Netherlands), and then visualized with silver enhancement.

**SUPPLEMENTAL DATA**

Supplemental Data include ten figures and a table and can be found with this article online at [http://www.cell.com/developmental-cell/supplement/S1534-5807(09)00255-X](http://www.cell.com/developmental-cell/supplement/S1534-5807(09)00255-X).

**ACKNOWLEDGMENTS**

This work was supported by National Institutes of Health Grant GM53396 (to D.J.K.) and Japan Society for the Promotion of Science Postdoctoral Fellowships for Research Abroad (to T.K.).

Received: December 29, 2008  
Revised: March 23, 2009  
Accepted: June 25, 2009  
Published: July 20, 2009

**REFERENCES**

Baba, M. (2008). Electron microscopy in yeast. Methods Enzymol. 451, 133–149.

Bogenhagen, D.F. (1999). Repair of mtDNA in vertebrates. Am. J. Hum. Genet. 64, 1276–1281.

Cao, Y., and Klionsky, D.J. (2007). Physiological functions of Atg6/Beclin 1: a unique autophagy-related protein. Cell Res. 17, 839–849.

Cheong, H., Nair, U., Geng, J., and Klionsky, D.J. (2008). The Atg1 kinase complex is involved in the regulation of protein recruitment to initiate sequestering vesicle formation for nonspecific autophagy in *Saccharomyces cerevisiae*. Mol. Biol. Cell 19, 668–681.

Clark, S.L., Jr. (1957). Cellular differentiation in the kidneys of newborn mice studied with the electron microscope. J. Biophys. Biochem. Cytol. 3, 349–362.

Dunn, W.A., Jr., Cregg, J.M., Kiel, J.A.K.W., van der Klei, I.J., Oku, M., Sakai, Y., Sibirny, A.A., Stasyk, O.V., and Veenhuis, M. (2005). Pexophagy: the selective autophagy of peroxisomes. Autophagy 1, 75–83.

Elmore, S.P., Qian, T., Grissom, S.F., and Lemasters, J.J. (2001). The mitochondrial permeability transition initiates autophagy in rat hepatocytes. FASEB J. 15, 2286–2287.

Farre, J.C., Manjithaya, R., Mathewson, R.D., and Subramani, S. (2008). PpAtg30 tags peroxisomes for turnover by selective autophagy. Dev. Cell 14, 365–376.

Graham, J.M. (2001). Isolation of mitochondria from tissues and cells by differential centrifugation. Curr. Protoc. Cell Biol. Chapter 3, Unit 3.3.

James, P., Halladay, J., and Craig, E.A. (1996). Genomic libraries and a host strain designed for highly efficient two-hybrid selection in yeast. Genetics 144, 1425–1436.

Kanki, T., and Klionsky, D.J. (2008). Mitophagy in yeast occurs through a selective mechanism. J. Biol. Chem. 283, 32386–32393.

Kim, J., Kamada, Y., Stromhaug, P.E., Guan, J., Hefner-Gravink, A., Baba, M., Scott, S.V., Ohsumi, Y., Dunn, W.A., Jr., and Klionsky, D.J. (2001). Cvt9/Gsa9 functions in sequestering selective cytosolic cargo destined for the vacuole. J. Cell Biol. 153, 381–396.

Kišsová, I., Deffieu, M., Manon, S., and Camougrand, N. (2004). Uth1p is involved in the autophagic degradation of mitochondria. J. Biol. Chem. 279, 39068–39074.

Kišsová, I., Salin, B., Schaeffer, J., Bhatia, S., Manon, S., and Camougrand, N. (2007). Selective and non-selective autophagic degradation of mitochondria in yeast. Autophagy 3, 329–336.

Klionsky, D.J. (2005). The molecular machinery of autophagy: unanswered questions. J. Cell Sci. 118, 7–18.

Klionsky, D.J., and Emr, S.D. (2000). Autophagy as a regulated pathway of cellular degradation. Science 290, 1717–1721.

Larsson, N.G., and Clayton, D.A. (1995). Molecular genetic aspects of human mitochondrial disorders. Annu. Rev. Genet. 29, 151–178.

Longtine, M.S., McKenzie, A., III, Demarini, D.J., Shah, N.G., Wach, A., Brachat, A., Philippsen, P., and Pringle, J.R. (1998). Additional modules for versatile and economical PCR-based gene deletion and modification in *Saccharomyces cerevisiae*. Yeast 14, 953–961.

Lussier, M., White, A.M., Sheraton, J., di Paolo, T., Treadwell, J., Southard, S.B., Horenshtein, C.I., Chen-Weiner, J., Ram, A.F., Kapteyn, J.C., et al. (1997). Large scale identification of genes involved in cell surface biosynthesis and architecture in *Saccharomyces cerevisiae*. Genetics 147, 435–450.

Mijaljica, D., Prescott, M., and Devenish, R.J. (2007). Different fates of mitochondria: alternative ways for degradation? Autophagy 3, 4–9.

Monastyrska, I., He, C., Geng, J., Hoppe, A.D., Li, Z., and Klionsky, D.J. (2008). Arp2 links autophagic machinery with the actin cytoskeleton. Mol. Biol. Cell 19, 1962–1975.

Narendra, D., Tanaka, A., Suen, D.F., and Youle, R.J. (2008). Parkin is recruited selectively to impaired mitochondria and promotes their autophagy. J. Cell Biol. 183, 795–803.

Noda, T., Matsuura, A., Wada, Y., and Ohsumi, Y. (1995). Novel system for monitoring autophagy in the yeast *Saccharomyces cerevisiae*. Biochem. Biophys. Res. Commun. 210, 126–132.

Nowikovsky, K., Reipert, S., Devenish, R.J., and Schwyen, R.J. (2007). Mdm38 protein depletion causes loss of mitochondrial K⁺/H⁺ exchange activity, osmotic swelling and mitophagy. Cell Death Differ. 14, 1647–1656.

Priault, M., Salin, B., Schaeffer, J., Vallette, F.M., di Rago, J.P., and Martinou, J.C. (2005). Impairing the bioenergetic status and the biogenesis of mitochondria triggers mitophagy in yeast. Cell Death Differ. 12, 1613–1621.

Reggiori, F., Monastyrska, I., Shintani, T., and Klionsky, D.J. (2005). The actin cytoskeleton is required for selective types of autophagy, but not nonspecific autophagy, in the yeast *Saccharomyces cerevisiae*. Mol. Biol. Cell 16, 5843–5856.

Rep, M., and Grivell, L.A. (1996). The role of protein degradation in mitochondrial function and biogenesis. Curr. Genet. 30, 367–380.

Rodriguez-Enriquez, S., He, L., and Lemasters, J.J. (2004). Role of mitochondrial permeability transition pores in mitochondrial autophagy. Int. J. Biochem. Cell Biol. 36, 2463–2472.

Ruan, H., Yan, Z., Sun, H., and Jiang, L. (2007). The *YCR079w* gene confers a rapamycin-resistant function and encodes the sixth type 2C protein phosphatase in *Saccharomyces cerevisiae*. FEMS Yeast Res. 7, 209–215.

Sandoval, H., Thiagarajan, P., Dasgupta, S.K., Schumacher, A., Prchal, J.T., Chen, M., and Wang, J. (2008). Essential role for Nix in autophagic maturation of erythroid cells. Nature 454, 232–235.

Schweers, R.L., Zhang, J., Randall, M.S., Lloyd, M.R., Li, W., Dorsey, F.C., Kundu, M., Opferman, J.T., Cleveland, J.L., Miller, J.L., et al. (2007). NIX is required for programmed mitochondrial clearance during reticulocyte maturation. Proc. Natl. Acad. Sci. USA 104, 19500–19505.

Shintani, T., Huang, W.-P., Stromhaug, P.E., and Klionsky, D.J. (2002). Mechanism of cargo selection in the cytoplasm to vacuole targeting pathway. Dev. Cell 3, 825–837.

Shintani, T., and Klionsky, D.J. (2004). Cargo proteins facilitate the formation of transport vesicles in the cytoplasm to vacuole targeting pathway. J. Biol. Chem. 279, 29889–29894.

Suzuki, K., Kubota, Y., Sekito, T., and Ohsumi, Y. (2007). Hierarchy of Atg proteins in pre-autophagosomal structure organization. Genes Cells 12, 209–218.

Developmental Cell

Atg32 Is an Organelle Tag for Mitophagy

Tal, R., Winter, G., Ecker, N., Klionsky, D.J., and Abeliovich, H. (2007). Aup1p, a yeast mitochondrial protein phosphatase homolog, is required for efficient stationary phase mitophagy and cell survival. J. Biol. Chem. 282, 5617–5624.

Twig, G., Elorza, A., Molina, A.J., Mohamed, H., Wikstrom, J.D., Walzer, G., Stiles, L., Haigh, S.E., Katz, S., Las, G., et al. (2008). Fission and selective fusion govern mitochondrial segregation and elimination by autophagy. EMBO J. 27, 433–446.

Wallace, D.C. (2005). A mitochondrial paradigm of metabolic and degenerative diseases, aging, and cancer: a dawn for evolutionary medicine. Annu. Rev. Genet. 39, 359–407.

Xie, Z., Nair, U., and Klionsky, D.J. (2008). Atg8 controls phagophore expansion during autophagosome formation. Mol. Biol. Cell 19, 3290–3298.

Yorimitsu, T., and Klionsky, D.J. (2005). Atg11 links cargo to the vesicle-forming machinery in the cytoplasm to vacuole targeting pathway. Mol. Biol. Cell 16, 1593–1605.

Yorimitsu, T., and Klionsky, D.J. (2007). Eating the endoplasmic reticulum: quality control by autophagy. Trends Cell Biol. 17, 279–285.

Zhang, Y., Qi, H., Taylor, R., Xu, W., Liu, L.F., and Jin, S. (2007). The role of autophagy in mitochondria maintenance: characterization of mitochondrial functions in autophagy-deficient S. cerevisiae strains. Autophagy 3, 337–346.
