
The Journal of Antibiotics (2009) 62, 471–481  
© 2009 Japan Antibiotics Research Association All rights reserved 0021-8820/09 $32.00  
www.nature.com/ja

---

**REVIEW ARTICLE**

---

# Biosynthetic genes for aminoglycoside antibiotics

Fumitaka Kudo¹ and Tadashi Eguchi²

Biosynthetic studies of aminoglycoside antibiotics have progressed remarkably during the last decade. Many biosynthetic gene clusters for aminoglycoside antibiotics including streptomycin, kanamycin, butirosin, neomycin and gentamicin have been identified to date. In addition, most butirosin and neomycin biosynthetic enzymes have been functionally characterized using recombinant proteins. Herein, we reanalyze biosynthetic genes for structurally related 2-deoxystreptamine (2DOS)-containing aminoglycosides, such as kanamycin, gentamicin and istamycin, based on genetic information including characterized biosynthetic enzymes in neomycin and butirosin biosynthetic pathways. These proposed enzymatic functions for uncharacterized enzymes are expected to support investigation of the complex biosynthetic pathways for this important class of antibiotics.

*The Journal of Antibiotics (2009) 62, 471–481; doi:10.1038/ja.2009.76; published online 31 July 2009*

Keywords: aminoglycoside antibiotics; aminocyclitol; biosynthesis; biosynthetic enzyme; biosynthetic gene

---

## INTRODUCTION

Aminoglycoside antibiotics are known as a classical but clinically important class of agents, especially against *Mycobacterium tuberculosis*.¹ In 1944, Selman Waksman found aminoglycoside antibiotic streptomycin from soil bacteria, which showed significant antibacterial activity, importantly anti-tuberculosis activity. This discovery led to the isolation of such bioactive compounds including kanamycin, found by Hamao Umezawa in Japan. These aminoglycoside antibiotics are structurally similar, possessing a core aminocyclitol moiety and various unusual sugars including aminosugars and deoxysugars (Figure 1). Combinations of sugar attachments and amino functionality are critical for antibacterial activity, which results from specific interaction with bacterial ribosomal RNAs to inhibit their protein synthesis. In fact, such specific interactions have been observed clearly through X-ray structural analysis.²

Emergence of bacteria that are resistant against such antibiotics remains a serious problem. Resistant enzymes in bacteria modify aminoglycosides through phosphorylation, adenylation and acetylation.³⁴ Modified aminoglycosides become unable to interact with bacterial rRNAs and therefore become ineffective. To prevent such modification reactions, deoxygenation and 4-amino-2S-hydroxybutyration of aminoglycoside have been developed as effective methods because resistant enzymes cannot reach or modify such antibiotics even though they can still interact with bacterial rRNAs.⁵

In spite of the ototoxic and nephrotoxic side effects of aminoglycosides, the highly specific RNA-binding ability has engendered the preparation of various analogous compounds to apply for human immunodeficiency virus infection and antiplasmid approaches.⁷⁸ Consequently, aminoglycoside remains an important material in medicinal chemistry.

---

Biosynthetic studies of aminoglycosides have indeed been performed for many years. Initially, feeding experiments with radioisotope and stable isotope-labeled primary metabolites such as glucose and glucosamine elucidated the origin of carbon atoms.⁹ Idiotrophic mutants were also constructed and used to postulate the biosynthetic pathway through the determination of structures of accumulated intermediates.¹⁰ Cosynthesis with blocked mutants was also effective to determine the order of biosynthetic steps.¹¹ However, these experiments yielded no direct evidence for enzymatic reaction mechanisms.

Enzymatic analyses were performed extensively by Walker,¹² yielding important information to elucidate the complex biosynthetic pathway at the enzyme reaction level. However, only limited information about the enzymes was available at that time because the purification of quite small amounts of secondary metabolic enzymes from a natural source was usually a difficult task. In the 1980s, genetic analyses for natural product biosynthesis were initiated to identify the responsible biosynthetic genes. Initially, complementary gene fragments for idiotrophic mutants to restore antibiotic production were screened to identify the desired biosynthetic genes. In addition, resistant genes were used as a target to identify secondary metabolite biosynthetic gene clusters. A partial streptomycin biosynthetic gene cluster was then identified in 1987.¹³ Subsequently, several biosynthetic genes for streptomycin were identified in association with the central gene cluster.¹⁴ However, only a few biosynthetic genes were heterologously expressed and investigated to characterize the function to date.¹⁵⁻¹⁸ Among those, the characterization of L-glutamine:syllo-inosamine aminotransferase StsC gave important information to elucidate the aminocyclitol-containing aminoglycosides biosynthetic pathway.¹⁵ Using the hybridization method, Piepersberg and co-workers¹⁵

---

¹Department of Chemistry, Tokyo Institute of Technology, Meguro-ku, Tokyo, Japan  
²Department of Chemistry and Materials Science, Tokyo Institute of Technology, O-okayama, Meguro-ku, Tokyo 151-8551, Japan  

Correspondence: Professor T Eguchi, Department of Chemistry and Materials Science, Tokyo Institute of Technology, O-okayama, Meguro-ku, Tokyo 151-8551, Japan.  
E-mail: eguchi@cms.titech.ac.jp  

Received 4 June 2009; revised 9 July 2009; accepted 13 July 2009; published online 31 July 2009

Biosynthetic genes for aminoglycoside antibiotics  
F Kudo and T Eguchi  

Figure 1 Aminoglycoside antibiotics whose biosynthetic genes are deposited in a public gene database such as DDBJ, EMBL and GenBank.  
(a) 2-deoxystreptamine-containing aminoglycoside antibiotics, (b) myo-inositol-derived aminocyclitol-containing aminoglycoside antibiotics.

showed that its homologous gene is conserved in the aminocyclitol antibiotic-producing microorganisms. In fact, StsC homologous amidotransferase genes are conserved completely in the genomes of aminocyclitol antibiotic-producing microorganisms as a specific feature.

In 1999, the 2-deoxy-scyllo-inosose (2DOI) synthase was isolated from butirosin-producing *Bacillus circulans* as a crucial enzyme at the first step in the 2-deoxystreptamine (2DOS) biosynthetic pathway. Its corresponding gene, *btrC*, was then cloned and the recombinant enzyme was found to have 2DOI synthase activity. In addition, a butirosin biosynthetic gene cluster (*btr*) was identified by chromosomal gene walking from the DOI synthase gene (Table 1). The majority of aminoglycosides contain 2DOS as a core unique aminocyclitol aglycon. Therefore, the discovery of the *btr* gene boosted the identification of several other 2DOS-containing aminoglycoside biosynthetic gene clusters (Figures 1 and 2 and Table 1). To date, biosynthetic gene clusters for butirosin, neomycin, ribostamycin,

paromomycin, lividomycin, kanamycin, tobramycin, gentamicin, sisomicin, istamycin, apramycin and hygromycin B have been identified, yielding important information to predict the biosynthetic pathway step by step, combined with previous mutant studies. It was readily speculated that conserved biosynthetic enzymes encoded in gene clusters could construct common structures of aminoglycosides. In accordance with that prediction, seven neamine-related biosynthetic genes and two unique ribostamycin-related biosynthetic genes were clearly classified. In addition, two neomycin-related enzymes and several butirosin-specific biosynthetic enzymes have been identified. Most enzymes in neomycin and butirosin biosynthesis have been functionally characterized with the recombinant proteins. In this review, we first summarize the neomycin and butirosin biosynthetic enzymes characterized to date; then, we bioinformatically reanalyze the function of the biosynthetic genes for other 2DOS-containing aminoglycoside antibiotics including kanamycin, tobramycin, gentamicin and istamycin (also fortimicin).

Table 1 Biosynthetic enzymes of 2-deoxystreptamine-containing aminoglycoside antibiotics

| Characterized function or putative function | Butirosin | Neomycin | Ribostamycin | Paromomycin | Lividomycin | Kanamycin | Tobramycin | Gentamicin | Istamycin | Fortimicin | Frankia | Apramycin | Hygromycin B |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1a | 2DOIA synthase | BtrC | NeoC | RibC | ParC | KanC | LivC | GenC | TobC | IstC | HygC | AprC | 3363 |
| 2a | L-Gln:2DOI aminotransferase | BtrS | NeoS | RibS | ParS | KanE | LivS | GenE | TobE | IstE | HygS | AprS | 3364 |
| 3a | 2DOIA dehydrogenase (NAD+ dependent) | BtrE | NeoE | RibE | ParE | KanM, M2 | LivM | GenM1, M2 | TobM1, M2 | IstM | HygE | AprE | 3362 |
| 4a | UDP-GlcNAc:2DOSGlcNAc transferase | BtrM | NeoM | RibM | ParM | KanN | LivD | GenD | TobN | IstD | HygD | AprM | 3367 |
| 5a | 2'-N-acetylparomamine deacetylase | BtrD | NeoD | RibD | ParD | KanQ | LivQ | GenQ | TobQ | IstQ | HygQ | AprN | 3360 |
| 6a | Paromamine 6'-dehydrogenase | BtrQ | NeoQ | RibQ | ParQ | KanB | LivB | GenB1, B2, | TobB | IstB | HygB | AprQ | 3369 |
| 7a | L-Gln:6'-oxoparomamine aminotransferase | BtrB | NeoB | RibB | ParB | KanL | LivL | B3, B4 |  |
| 8b | PRPP:neamine 5'-phosphoribosyltransferase | BtrL | NeoL | RibL | ParL | LivL |  |
| 9b | 5'-phosphoribosylneomycin phosphatase | BtrP | NeoP | RibP | ParP | LivP |  |
| 10c | UDP-GlcNAc:ribostamycin GlcNAc transferase | BtrO | NeoF | RibF | ParF | LivF |  |
| 11c | Neomycin 5"-epimerase? (radical SAM enzyme) | BtrH | NeoN | RibN | ParN | LivN |  |
| 12 | Acyl carrier protein (ACP) | BtrI |  |  |  |  |  |  |  |  |  |  |  |
| 13 | γ-L-Glu-ACP ligase | BtrJ |  |  |  |  |  |  |  |  |  |  |  |
| 14 | γ-L-Glu-ACP decarboxylase | BtrK |  |  |  |  |  |  |  |  |  |  |  |
| 15 | γ-L-Glu-GABA-ACP monooxygenase | BtrU |  |  |  |  |  |  |  |  |  |  |  |
| 16 | NAD(P):H/FMN oxidoreductase | BtrH |  |  |  |  |  |  |  |  |  |  |  |
| 17 | γ-L-Glu-AHBA-ACP:ribostamycin transferase | BtrH |  |  |  |  |  |  |  |  |  |  |  |
| 18 | γ-L-Glu-butirosin γ-L-glutamyl cyclotransferase | BtrG |  |  |  |  |  |  |  |  |  |  |  |
| 19 | Butirosin 3"-epimerase? | BtrF |  |  |  |  |  |  |  |  |  |  |  |
| 20 | 2DOIA dehydrogenase (radical SAM enzyme) | BtrN |  |  |  |  |  |  |  |  |  |  |  |
| 21 | Unknown | BtrA | NeoA | RibA | ParA | LivA |  |  |  |  |  |  |  |
| 22 | Hypothetical protein | BtrV | NeoX | RibX | ParX | LivX |  |  |  |  |  |  |  |
| 23 | ABC transporter | BtrT | NeoT | RibT | ParT | LivT |  |  |  |  |  |  |  |
| 24 | ABC transporter | BtrU | NeoU | RibU | ParU | LivU |  |  |  |  |  |  |  |
| 25 | Aminoglycoside N-acetyltransferase (resistant) | AacC8 | AacC8 |  |  |  |  |  |  |  |  |  |  |
| 26 | ABC transporter | BtrW |  |  |  |  |  |  |  |  |  |  |  |
| 27 | ABC transporter | BtrX |  |  |  |  |  |  |  |  |  |  |  |
| 28 | Transcriptional regulator | BtrR1 |  |  |  |  |  |  |  |  |  |  |  |
| 29 | Transcriptional regulator | BtrR2 |  |  |  |  |  |  |  |  |  |  |  |
| 30 | Hypothetical protein | BtrT |  |  |  |  |  |  |  |  |  |  |  |
| 31 | Putative regulator component | Neol | Ribl | Parl | Livl | Kanl |  |  |  |  |  |  |  |
| 32 | Putative regulator component | NeoH | RibH | ParH | LivH | KanH |  |  |  |  |  |  |  |
| 33 | Putative regulator component | NeoG | RibG | ParG | LivG | KanG |  |  |  |  |  |  |  |
| 34 | Aminoglycoside O-phosphotransferase | AphA | Rph | ParR(AphA) |  |  |  |  |  |  |  |  |  |
| 35 | Extracellular aminoglycoside phosphatase | Park | LivK |  |  |  |  |  |  |  |  |  |  |

Biosynthetic genes for aminoglycoside antibiotics  
F Kudo and T Eguchi

| Characterized function or putative function | Butirosin | Neomycin | Ribostamycin | Paramomycin | Lividomycin | Kanamycin | Tobramycin | Frankia sp. CcI3 | Apramycin Hygromycin B |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 36 | Unknown | LivO | LivV | LivW | LivY |  |  |  |  |
| 37 | Hypothetical protein | AprD4 | AprD3 |  |  |  |  |  |  |
| 38d | Putative Fe-S oxidoreductase | GenD3 |  |  |  |  |  |  |  |
| 39d | Aminoglycoside 6'-N-acetyltransferase |  |  |  |  |  |  |  |  |
| 40 | Transcriptional regulator | KanA |  |  |  |  |  |  |  |
| 41 |  | KanT | TobT |  |  |  |  |  |  |
| 42 | Putative transport protein | KanD2 | TobD2 | GenD2 |  |  |  |  |  |
| 43e | Putative dehydrogenase | KanJ |  |  |  |  |  |  |  |
| 44 | Hypothetical protein | Kmr | FmrB, O, P |  |  |  |  |  |  |
| 45 | Hypothetical protein | GmrA, B | 3372, 3382 |  |  |  |  |  |  |
| 46 | 16S RNA methylase |  |  |  |  |  |  |  |  |
| 47 | Putative carbamoyltransferase | TobZ |  |  |  |  |  |  |  |
| 48 | Putative carbamoyl-phosphate synthase | TobL |  |  |  |  |  |  |  |
| 49 | Hypothetical protein | TobX |  |  |  |  |  |  |  |
| 50f | Putative tRNA-ribosyltransferase | GenO | FosA | 3378 |  |  |  |  |  |
| 51g | Radical SAM protein | GenD1 | FosC | 3383 |  |  |  |  |  |
| 52f | GTP cyclohydrolase | GenW | FosG | 3370 |  |  |  |  |  |
| 53h | Putative radical SAM methyltransferase | GenK | ForK | (Fms7) |  |  |  |  |  |
| 54g | Hypothetical protein | GenX |  |  |  |  |  |  |  |
| 55g | Hypothetical protein | GenU |  |  |  |  |  |  |  |
| 56g | Transmembrane efflux protein | GenV |  |  |  |  |  |  |  |
| 57f | Membrane antipporter | GenY | ForY |  |  |  |  |  |  |
| 58f | Hypothetical protein | GenA | FosE | 3379 |  |  |  |  |  |
| 59f | Radical SAM protein | GenF | FosF | 3380 |  |  |  |  |  |
| 60h | Hypothetical protein | GenG | FosG | 3381 |  |  |  |  |  |
| 61i | Hypothetical protein | GenH, I | IstH, I, J | ForH, I, J, V |  |  |  |  |  |
| 62f | Putative methyltransferase | GenT | ForT | 3377 |  |  |  |  |  |
| 63g | Putative methyltransferase | GenN |  |  |  |  |  |  |  |
| 64j | Hypothetical protein | IstW | ForW | 3361 |  |  |  |  |  |
| 65j | Putative methyltransferase | IstO | ForO | 3365 |  |  |  |  |  |
| 66j | Radical SAM protein | IstL2 | ForL2 | 3366 |  |  |  |  |  |
| 67j | FAD-dependent oxidoreductase | IstZ | ForZ | 3371 |  |  |  |  |  |
| 68j | Carboxymuconolactone decarboxylase family protein | IstX | ForX | 3376 |  |  |  |  |  |
| 69k | Inositol 2-dehydrogenase | ForG | 3374 |  |  |  |  |  |  |
| 70j | Putative methyltransferase | IstN | ForN | 3373 |  |  |  |  |  |
| 71k | Inositol-phosphate phosphatase | ForA |  |  |  |  |  |  |  |
| 72j | Radical SAM protein | IstL, L3 | ForL | 3357, 3375 |  |  |  |  |  |
| 73 | 16S RNA methylase | ImrA | KamB |  |  |  |  |  |  |

| Characterized function or putative function | Butirosin | Neomycin | Ribostamycin | Paromomycin | Ribostamycin | Frankia sp. CcI 3 | Hygromycin B |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Hypothetical protein | IstA | AprA |  |  |  |  |  |
| Hypothetical protein | IstF | AprU |  |  |  |  |  |
| Putative methyltransferase | IstU | AprO |  |  |  |  |  |
| Sugar hydrolase |  | AprD1 |  |  |  |  |  |
| UDP-glucose 4-epimerase |  | AprF |  |  |  |  |  |
| Hypothetical protein |  | AprG |  |  |  |  |  |
| Hypothetical protein |  | AprH |  |  |  |  |  |
| Glycosyltransferase family 28 |  | AprI |  |  |  |  |  |
| Putative methyltransferase |  | AprJ |  |  |  |  |  |
| Phosphosugar mutase |  | AprK |  |  |  |  |  |
| Creatinine amidohydrolase |  | AprP |  |  |  |  |  |
| UDP-GlcNAcC6 dehydratase |  | AprD2 |  |  |  |  |  |
| Aminotransferase |  | AprL |  |  |  |  |  |
| UDP-glucose 4-epimerase |  | AprD5 |  |  |  |  |  |
| Hygromycin B kinase | HygA |  |  |  |  |  |  |
| Putative UDP-galactose 4-epimerase | HygK |  |  |  |  |  |  |
| Putative phosphatase | HygU |  |  |  |  |  |  |
| Glycosyltransferase group 1 | HygD |  |  |  |  |  |  |
| Galactose-1-phosphate uridylyltransferase | HygO |  |  |  |  |  |  |
| Sugar isomerase | HygP |  |  |  |  |  |  |
| Hydroxylase | HygX |  |  |  |  |  |  |
| aminotransferase | HygL |  |  |  |  |  |  |
| NAD-dependent epimerase | HygJ |  |  |  |  |  |  |
| Radical SAM protein | HygY |  |  |  |  |  |  |
| Putative glycosyltransferase | HygF |  |  |  |  |  |  |
| d-glycerol-manno-heptose | HygN |  |  |  |  |  |  |
| 7-phosphate kinase | HygM |  |  |  |  |  |  |
| Hypothetical protein | HygT |  |  |  |  |  |  |

Biosynthetic genes for aminoglycoside antibiotics

Butirosin biosynthetic enzymes (AB097196, Bacillus circulans SANK 72073), neomycin biosynthetic enzymes (AJ628955, S. rimosus subsp. paromomycinus NRRL B-11466), ribostamycin biosynthetic enzymes (AJ628422, S. kanamyceticus DSM 40500), tobramycin biosynthetic enzymes (AJ628149, Micromonospora echinospora DSM 43036), istamycin biosynthetic enzymes (AJ628421, M. olivasterospora DSM 43868), probable aminoglycoside-related biosynthetic enzymes in Frankia sp. CcI 3 (complete genome), apramycin biosynthetic enzymes (AJ845083, S. tenjimariensis ATCC 31603), fortimicin biosynthetic enzymes (AJ629123, Streptomyces sp. DSM 40477, AJ875019, Streptomyces hygroscopicus subsp. hygroscopicus DSM 40578). In the table, nos. 1–7 (a) are neamine-related biosynthetic enzymes; nos. 8, 9 (b) are ribosylation enzymes in ribostamycin-related compounds; nos. 10, 11 (c) are neomycin-related biosynthetic enzymes; nos. 12–20 are butirosin pathway-specific enzymes; nos. 38, 39 (d) are lividomycin and apramycin-specific enzymes; nos. 51, 54–56, 63 (g) are gentamicin- and sisomicin-specific enzymes; nos. 53, 60 (h) are gentamicin- and fortimicin-specific enzymes; nos. 69, 71 (k) are fortimicin- and istamycin-specific enzymes; no. 61 (i) is a gentamicin- and sisomicin-, fortimicin- and istamycin-specific enzyme; nos. 64–68, 70, 72 (j) are fortimicin- and istamycin-specific enzymes from myo-inositol 1-phosphate; Others are pathway-specific enzymes, transporters, resistant enzymes, transcriptional regulators, and unknown functional hypothetical proteins.

Biosynthetic genes for aminoglycoside antibiotics  
F Kudo and T Eguchi  

butirosin  
btrR1 U R2 T S N M L A B C D E F G H I J K O PV Q W X  

neomycin  
aphA neoG H I E S C M T U Q N P XFD L B A aac8 R  

ribostamycin  
rph ribG H I E S C M T U Q N PXF D L B A aac8  

paromomycin  
parE S C M T U Q N PXF D L B A K I R Y G H  

lividomycin  
livE S C M T U Q N PFD L B K I H G A XO V W Y Z  

kanamycin  
kanZ Y X W A GH I R ET B Q S1 CD2 M2N S2 J K kmr M1 LOP D3 U  

tobramycin  
tobX E T B Q Z S1 C D2 M2 NS2 M1 L U A D3 OR  

gentamicin  
genN D T I H GFA Y EV UX B2 K B3 P B4 WS2 D1 M2D2 C S1gmrAM1D3 Q B1 gmrB O  

istamycin  
imrA istA D P B J F U Z H W Q C L3 LI SNXM L2O E  

fortimicin  
forY H I J Q W V M S E OL2 DP B K Z X G N A L fmrR,O T fosGFEDC fmrPB A  

Frankia sp. CcI3  
3357 3358 3359 3360 3361 3363 3365 3367 3369 3370 3371 3373 3375 3377 3379 3381 3383  

apramycin  
aprX R2 R1 N W V U aprAkamB aprO D1 F G H I J KM C S EP D2 D3 Q D4 Z L D5  

hygromycin B  
hygV W A K U D O E PXL J YFN C MSTIH G  

a b c d e f g h i j k  

Figure 2 Biosynthetic gene clusters for aminoglycoside antibiotics. Characterized functions or putative functions of genes are showed in Table 1. Genes are classified according to the biosynthetic functions summarized in Table 1 (a)–(k).  

NEOMYCIN-RELATED BIOSYNTHETIC GENES  

We presented 2DOS biosynthetic genes in Table 1: many homologous proteins are conserved in the structurally related compounds (Figure 2). As many identical biosynthetic genes for aminoglycoside antibiotics are deposited as different symbols in the database, we use here the btr gene-based code names that are used systematically by Piepersberg et al.²³ Seven proteins (nos. 1–7 in Table 1), 2DOI synthase (BtrC, NeoC), L-glutamine:2DOI aminotransferase (BtrS, NeoS), NAD-dependent dehydrogenase (BtrE, NeoE), glycosyltransferase (BtrM, NeoM), deacetylase (BtrD, NeoD), flavin adenine dinucleotide (FAD)-dependent dehydrogenase (BtrQ, NeoQ) and another aminotransferase (BtrB, NeoB), are all conserved in butirosin, neomycin, ribostamycin, paromomycin, lividomycin, kanamycin, tobramycin, gentamicin and istamycin biosynthetic gene clusters, indicating that these are responsible for the construction of a common neamine-like structure. In fact, 2DOI formed from D-glucose-6-phosphate by 2DOI synthase is converted into 2-deoxy-scyllo-inosamine (2DOIA) by Gln:2DOI aminotransferase,²⁴,²⁵ which is a homologous protein to StsC in streptomycin biosynthesis. Next, an NAD-dependent dehydrogenase catalyzes the dehydrogenation at C-1 of 2DOIA to give 3-amino-2,3-dideoxy-scyllo-inosose (amino-DOI).²⁶ Amino-DOI is then converted to 2DOS by dual functional Gln:2DOI aminotransferase to complete the 2DOS biosynthesis.²⁷ Then, 2DOS is glycosylated using glycosyltransferase with UDP-N-acetyl-D-glucosamine (UDP-GlcNAc) as a glycosyl donor to give N-acetylparomamine, whose N-acetyl group is removed using a deacetylase to give paromamine.²⁸ Paromamine is believed to be a branching intermediate to neomycin-related aminoglycosides,

Biosynthetic genes for aminoglycoside antibiotics

F Kudo and T Eguchi

---

D-glucose-6-phosphate (G6P) → 2-deoxy-scyllo-inosose (2DOI) → 2-deoxy-scyllo-inosamine (2DOIA) → 3-amino-2,3-dideoxy-scyllo-inosose (amino-DOI) → 2-deoxystreptamine (2DOS)

UDP-GlcNAc → 2'-N-acetylparomamine → paromamine → neamine

PRPP → 2-O₃PO → ribostamycin → neomycin C → neomycin B

Figure 3 Neomycin biosynthetic pathway. Structurally related butirosin, paromycin and lividomycin are shown.

kanamycin and gentamicin. In neomycin biosynthesis, C-6′ of paromamine is oxidized to 6′-oxoparomamine by an FAD-dependent dehydrogenase.²⁹ 6′-Oxoparomamine is then converted to neamine by another aminotransferase.²⁹ All these seven enzymes derived from the neomycin biosynthetic gene cluster were functionally characterized using its recombinant proteins (Figure 3).

Neamine is then ribosylated by two unique enzymes (nos. 8 and 9) for ribostamycin-related aminoglycoside biosynthesis. To date, BtrL and BtrP encoded in the butirosin biosynthetic gene cluster were used to verify the function.³⁰ Subsequently, BtrL was found to catalyze the phosphoribosylation with 5-phosphoribosyl-1-diphosphate as the ribosyl donor. Then, phosphatase BtrP catalyzes the dephosphorylation to give ribostamycin.

Among neomycin-related gene clusters, two unique proteins (nos. 10 and 11) are conserved except for transcriptional regulators, transporters and resistant enzymes. One (NeoF) was found to be responsible for the glycosylation with UDP-GlcNAc to give a pseudo-tetrasaccharide.²⁸ The N-acetyl moiety of the product is then deacetylated by a dual functional deacetylase (NeoD), which also catalyzes the deacetylation of N-acetylparomamine.²⁸ This product can be further converted to neomycin C by a dual functional FAD-dependent dehydrogenase (NeoQ) and an aminotransferase (NeoB) in the neamine biosynthesis.²⁹ Finally, a remaining radical SAM protein is apparently responsible for the epimerization at C-5″ of neomycin C to give neomycin B, although its enzymatic activity has not been confirmed yet. Butirosin-specific enzymes (nos. 12–19) for the 4-amino-2S-hydroxybutyrate moiety were also characterized by the Cambridge group.³¹,³² Details of the enzymatic functions of the butirosin biosynthetic enzymes were described in a recent review.²²

Overall, most proteins encoded in the neomycin and butirosin biosynthetic gene cluster are apparently required for the construction

Biosynthetic genes for aminoglycoside antibiotics  
F Kudo and T Eguchi  

Figure 4 Kanamycin, tobramycin and gentamicin biosynthetic pathways. Structurally related sisomicin is also shown.

of aminoglycosides (Table 1). However, several proteins appear to be inactive and remain merely as evolutionary relics. For example, BtrE encoded in the butirosin gene cluster was presumed to be responsible for the dehydrogenation of 2DOIA in the 2DOS pathway, although it showed no expected activity.²⁶ A butirosin-specific radical SAM enzyme, BtrN, was then found to be responsible for dehydrogenation instead of BtrE.³³ In addition, it is noteworthy that C-6′ of paromomycin and lividomycin remains in the hydroxy group, even though a set of dehydrogenase and aminotransferase responsible for the neamine biosynthesis is conserved. Therefore, the dehydrogenase should not recognize paromamine toward the neamine formation.

In lividomycin biosynthesis, the hydroxy group at C-3′ of paromomycin is expected to be removed by certain enzymes. This deoxy structure is also apparent in tobramycin and apramycin, which are produced by two independent biosynthetic gene clusters in the same microorganisms. Two unique enzymes, LivW and LivY (nos. 38 and 39), are conserved in both the lividomycin and apramycin biosynthetic gene clusters. Consequently, these putative radical SAM oxidoreductase and dehydrogenase appear to be responsible for deoxygenation at C-3′ of the paromamine moiety. The LivY homologous protein is also encoded in the gentamicin biosynthetic gene cluster and might be involved in the deoxygenation reaction at the corresponding position of gentamicin. It is unclear what enzyme is responsible for the mannosylation at C-5‴ of lividomycin B leading to lividomycin A. A certain glycosyltransferase outside of the gene cluster might be responsible for the reaction.

### KANAMYCIN-RELATED BIOSYNTHETIC GENES

On the basis of the information described above for the characterized neomycin and butirosin biosynthetic enzymes, one can simply speculate about the functions of kanamycin biosynthetic enzymes (Figure 4). Paromamine biosynthesis is expected to be the same as the pathway in neomycin biosynthesis. Consequently, five conserved enzymes—(nos. 1–5) KanC, KanS1, KanE, KanM1 and KanN—might be responsible for paromamine formation. Recombinant KanM1 protein was reported to have only a weak glycosyltransferase activity with 2DOS and TDP-glucose and GDP-mannose.³⁴ In contrast, these five *kan* genes expressing *Streptomyces lividans* reportedly produce paromamine, indicating that our proposed enzymatic function is likely.³⁵ However, detailed enzymatic analysis is expected to be necessary to characterize the real function of KanM1.

One homologous glycosyltransferase, KanM2, is expected to attach glucose at the C-6 of paromamine with NDP-glucose. Then, oxidation and subsequent transamination can be catalyzed using a putative dehydrogenase, KanD2, and an aminotransferase, KanS2, respectively, to afford kanamycin C. These three proteins—KanM2, KanD2 and KanS2—are uniquely conserved in the kanamycin, tobramycin and gentamicin biosynthetic gene cluster (Table 1, nos. 4–2, 43, 2–2). The next oxidation and transamination can be catalyzed by an FAD-dependent oxidoreductase, KanQ, and a pyridoxal phosphate (PLP)-dependent aminotransferase, KanB, similar to neamine formation in the neomycin biosynthetic pathway.

Deamination and hydroxylation at C-2′ of kanamycin B to afford kanamycin A might be catalyzed by two unique unknown functional hypothetical proteins, KanJ and KanK, which are only unassigned open reading frames (ORFs), except for transcriptional regulators, transporters and resistance enzymes in the kanamycin biosynthetic gene cluster. Although a reverse scheme of oxidation and transamination in the conversion of a hydroxy to an amino group is simply inferred, KanJ and KanK show no homology to such enzymes. Alternatively, oxidation of the amino group to form an imine intermediate that is hydrolyzed nonenzymatically to be a ketone is the likely transformation. The presumed ketone intermediate might be reduced to be kanamycin A. Deoxygenation at C-3′ of kanamycin B leading to tobramycin can be catalyzed by a unique radical SAM oxidoreductase AprD4 and a dehydrogenase AprD3 in the apramycin gene cluster, as described above.

### GENTAMICIN-RELATED BIOSYNTHETIC GENES

The gentamicin biosynthetic pathway shares several similar enzymatic reaction steps with the kanamycin biosynthetic pathway (Figure 4). GenC, GenS1, GenE, GenM1 and GenD seem to be responsible for the

paromamine formation, similar to the neomycin biosynthetic pathway. The Gen-specific and Sis-specific<sup>36</sup> enzymes, GenD1, GenD2, GenX, GenU, GenV, GenN, GenS2 and GenM2, could then be involved in the biosynthesis of the garosamine moiety. A putative glycosyltransferase, GenM2, is expected to be a xylose transferase, which could use NDP-xylose as a glycosyl donor. A putative dehydrogenase, GenD2, and a putative aminotransferase, GenS2, might be responsible for the oxidation and transamination at C-3", similar to the kanamycin biosynthesis. A putative radical SAM C-methyltransferase, GenD1, can catalyze the C-methylation at C-6" and a putative N-methyltransferase, GenN, can catalyze N-methylation either of N-6' or N-3". The other proteins—GenX, GenU and GenV—might be involved in the other chemistry in the garosamine formation or unexpected reaction steps.

Gentamicin-specific genes compared with sisomicin biosynthetic genes are GenK, GenB2, GenB4 and GenG. GenK and GenG are also conserved in the fortimicin biosynthetic gene cluster. Consequently, a putative radical SAM C-methyltransferase, GenK, might catalyze the C-methylation of the purpuromamine moiety. Another hypothetical protein, GenG, might be involved in the reduction of C-4' and C-5'. In contrast, the putative PLP-dependent enzymes GenB2 and GenB4 might be responsible for the PLP-dependent epimerization at C-6' in the late stage of the biosynthesis.

In the gentamicin, sisomicin, fortimicin and istamycin biosynthetic gene clusters, GenH, GenI, GenQ and GenB1 (and/or B3 also) are conserved. They are apparently responsible for the biosynthesis of the common feature of the purpuromamine moiety. A putative FAD-dependent dehydrogenase, GenQ, and a putative aminotransferase, GenB1, are expected to be responsible for the oxidation and transamination at C-6', as is apparent in neamine biosynthesis. Moreover, other hypothetical proteins, GenH and GenI, are apparently responsible for the dideoxygenation of C-3' and C-4'. This enzymatic transformation can contain dehydration and subsequent reduction. Consequently, certain redox enzymes are apparently involved in this sequential reaction. GenO, GenW, GenY, GenA, GenF and GenT are conserved in fortimicin biosynthetic genes and might be involved in the dideoxygenation reaction, although, in that case, istamycin

biosynthesis requires extra ORFs outside of the gene cluster for the dideoxygenation. Among these, GenF is presumed to be a radical SAM oxidoreductase. It might therefore be involved in the reduction step in deoxygenation. The istamycin biosynthetic gene cluster contains an extra radical SAM enzyme, IstL3, which might be involved in the reduction chemistry. A putative methyltransferase, GenT, might catalyze N-methylation either of N-6' or N-3", and the other hypothetical proteins GenO, GenW, GenY and GenA might assist dehydration reactions in dideoxygenation as an acid/base catalyst or attach good leaving groups such as phosphate. Alternatively, these might be inactive for biosynthetic enzymatic reactions. A putative dehydrogenase, GenD3, might be involved in reduction during deoxygenation, as described in the section describing lividomycin/apramycin deoxygenation.

Heterologous expression of *genC*, *genS1*, *genE*, *genM1*, *genD* and *genM2* genes in *S*. venezuelae was attempted, resulting in gentamicin A2 formation.<sup>37</sup> This result clearly supports the above-proposed function of Gen enzymes. In addition, *genD1* gene disruption yielded accumulation of gentamicin A2, indicating that this putative radical SAM C-methyltransferase works after gentamicin A2 formation.<sup>38</sup> Detailed enzymatic analysis of GenD1 is necessary to determine the true substrate.

### FORTIMICIN-ISTAMYCIN BIOSYNTHETIC GENES

In istamycin biosynthesis, 2DOI synthase IstC and 2DOI aminotransferase IstS are expected to be used to synthesize the aminocyclitol skeleton (Figure 5). In contrast, in fortimicin biosynthesis, myo-inositol 1-phosphate can be used as the precursor, and a putative inositol phosphate phosphatase, ForA, and a putative inositol 2-dehydrogenase ForG, catalyze dephosphorylation and oxidation to give scyllo-inosose, which is converted to scyllo-inosamine by a putative aminotransferase, ForS. The following pathway is expected to be similar to the paromamine-neamine pathway, except for the regiospecificity of a putative NAD-dependent dehydrogenase, ForE, which must oxidize C-6 of the cyclitol moiety instead of C-1 after pseudosaccharide formation catalyzed using a putative glycosyltransferase ForM and a putative deacetylase ForD. A putative

![Chemical Structures](attachment:chemical_structures.png)

**Figure 5** Fortimicin biosynthetic pathway. Structurally related istamycin is also shown.

dual functional aminotransferase, ForS, is then presumed to catalyze transamination at C-6 of the cyclitol moiety.

In fortimicin and istamycin biosynthesis, dideoxygenation of a sugar moiety is also interesting to determine which enzymes are responsible. Although several homologous hypothetical proteins, ForH, ForI, ForJ and ForV, are conserved in the For, Ist, Gen and Sis gene clusters, the numbers of proteins differ. Moreover, these proteins would not be sufficient for the presumed two sets of dehydration and reduction to construct the dideoxy moiety, as discussed above. Consequently, several redox enzymes conserved in the For, Gen and Sis gene clusters might be involved in such dehydration and reduction. In istamycin biosynthesis, some extra radical SAM enzymes, IstL, IstL2 or IstL3, might be involved in the reduction. During fortimicin biosynthesis, two unique enzymes conserved in For and Gen might be responsible for the C-methylation and deoxygenation. A putative radical SAM enzyme, ForK, was in fact verified to be the C-methyltransferase by a gene complementation study.³⁹ Another hypothetical enzyme, ForG, might be involved in dideoxygenation, as described in the section of the gentamicin biosynthetic pathway.

Finally, the For-specific and Ist-specific enzymes are ForW, ForO, ForL, ForL2, ForZ, ForX and ForN, which are apparently involved in the fortamine moiety. The putative methyltransferases, ForO and ForN, can catalyze either O-methylation or N-methylation of the cyclitol moiety. Two putative radical SAM enzymes, ForL and ForL2, and/or an FAD-dependent oxidoreductase, ForZ, can be involved in redox reactions such as epimerization reactions at C-3 and C-4. The remaining two hypothetical proteins, ForW and ForX, might be involved in the glycosyltransfer reaction. The istamycin-specific methyltransferase, IstU, might be involved in N-methylation of the amino group at C-6'.

### OTHER 2DOS-CONTAINING AMINOGLYCOSIDE BIOSYNTHETIC GENES

The apramycin and hygromycin B biosynthetic enzymes are unique, even though several 2DOS and neamine-related biosynthetic enzymes are conserved. The apramycin biosynthetic gene cluster lacks one aminotransferase gene for amination at C-6' of paromamine (Table 1). Therefore, a possible aldehyde intermediate might be formed using a putative FAD-dependent dehydrogenase, AprQ, and then converted to the octose moiety, although the timing of the enzymatic reaction is unclear. Two putative dehydrogenase AprD3 and oxidoreductase AprD4 conserved in the Apr and Liv clusters are apparently involved in the deoxygenation, as described above. The remaining Apr-specific enzymes are expected to be involved in the construction of the octose moiety and attachment of the additional sugar moiety.

Hygromycin B biosynthetic gene cluster contains three 2DOS biosynthetic enzymes; the other ORFs appear to be unique for hygromycin B biosynthesis, as presented in Table 1. Several Hyg enzymes are conserved in the spectinomycin biosynthetic gene cluster⁴⁰ and are apparently responsible for a similar acetal structure.

### OTHER AMINOGLYCOSIDE BIOSYNTHETIC GENES

The streptomycin, bluensomycin, spectinomycin and kasugamycin biosynthetic genes are also known. Actually, the streptomycin biosynthetic system has historically been well studied at the genetic and enzymatic levels. In fact, its complex regulation system has been mostly established.⁴¹ However, not so many biosynthetic enzymes have been characterized even now because, probably, the biosynthetic pathway for streptomycin is too unique to predict appropriate functions compared with other aminoglycoside biosynthetic genes. Among

the streptomycin biosynthetic enzymes, Gln:scyllo-inosose aminotransferase StsC¹⁵ and Arg:inosamine-phosphatase amidinotransferase StrB¹⁶ are characterized biochemically. Characterization of StsC was important, as described in the Introduction section, and thus led to the characterization of Gln:2DOI aminotransferase BtrS in the 2DOS biosynthesis.²⁴ This characteristic inosose aminotransferase is conserved completely in the aminocyclitol-containing aminoglycoside antibiotic biosynthetic gene clusters (Table 1). Kasugamycin lacks the amino functionality in the cyclitol moiety and therefore does not possess the aminotransferase gene in the gene cluster.⁴²

Several biosynthetic enzymes for bluensomycin⁴³ and spectinomycin⁴⁴ have also been characterized, but those studies have not been performed systematically yet. Biochemical characterization of many other enzymes is necessary to understand complex biosynthetic pathways for these aminoglycosides on the enzymatic reaction level.

#### CONCLUDING REMARKS

A tremendous amount of genetic information is made available through genome projects of living systems, including those of microorganisms. Consequently, on the basis of its genomic information, one can predict that *Frankia* sp. CcI3 would produce fortimicin/istamycin-like aminoglycoside antibiotics (Figure 2 and Table 1). In addition, recombinant proteins expressed in heterologous host strains are now easily obtained for functional characterization. However, as we described above, so many uncharacterized proteins are encoded in small secondary metabolic biosynthetic gene clusters even now. Our classification of 2DOS aminoglycoside antibiotic biosynthetic enzymes would help to investigate the function, leading to the elucidation of whole biosynthetic pathways. Fundamentally, aminoglycoside biosynthesis contains several sets of oxidation/transamination for conversion of a hydroxy group to an amino group, glycosylation/deacetylation to attach a glucosamine, phosphoribosylation/dephosphorylation to attach a ribose group, glycosylation, probable dehydration/reduction to make a deoxysugar moiety, methylation and epimerization. Combinations of these enzymatic reactions and substrate specificity of the enzymes determine the final structure of aminoglycosides. Once we assimilate the insights of enzymatic reactions, including substrate specificities, and handle such enzymes through protein engineering, a rational combinatorially fashioned biosynthetic approach to create various structural aminoglycosides can be undertaken. It is therefore now important to verify the proposed function of the enzymes, including hypothetical proteins. Our proposed biosynthetic pathways suggest that many unknown functional hypothetical proteins might be involved in the biosynthesis of aminoglycosides. Characterization of such proteins would expand the number of functional enzymes and contribute greatly to the development of bioscience. We are therefore now struggling to characterize such aminoglycoside biosynthetic enzymes.

#### ACKNOWLEDGEMENTS

Our biosynthetic study has been supported by many grants from MEXT and nongovernment foundations. We gratefully acknowledge our numerous colleagues, especially the late Professor Katsumi Kainuma, who initiated our 2DOS-containing aminoglycoside biosynthetic studies 20 years ago. His contribution to this field is continuously and greatly appreciated.

---

1 Davies, J. in Aminoglycoside Antibiotics. In the beginning there was streptomycin (ed Arya, D.P.) 1–13 (John Wiley & Sons Inc., Hoboken, NJ, 2007).

2 Carter, A. P. *et al.* Functional insights from the structure of the 30S ribosomal subunit and its interactions with antibiotics. *Nature* **407**, 340–348 (2000).

Biosynthetic genes for aminoglycoside antibiotics

F Kudo and T Eguchi

3 Wright, G. D. Aminoglycoside-modifying enzymes. *Curr. Opin. Microbiol.* 2, 499–503 (1999).

4 Shakya, T. & Wright, G. D. Mechanisms of aminoglycoside antibiotic resistance. in *Aminoglycoside Antibiotics* (ed. Arya, D. P.) 119–140 (John Wiley & Sons Inc., Hoboken, NJ, 2007).

5 Kondo, S. & Hotta, K. Semisynthetic aminoglycoside antibiotics: development and enzymatic modifications. *J. Infect. Chemother.* 5, 1–9 (1999).

6 Elson-Schwab, L. & Tor, Y. Targeting HIV-1 RNA with Aminoglycoside antibiotics and their derivatives. in *Aminoglycoside Antibiotics* (ed. Arya, D. P.) 267–287 (John Wiley & Sons Inc., Hoboken, NJ, 2007).

7 Denap, J. C., Thomas, J. R., Musk, D. J. & Hergenrother, P. J. Combating drug-resistant bacteria: small molecule mimics of plasmid incompatibility as antiplasmid compounds. *J. Am. Chem. Soc.* 126, 15402–15404 (2004).

8 Thomas, J. R. & Hergenrother, P. J. Targeting RNA with small molecules. *Chem. Rev.* 108, 1171–1224 (2008).

9 Rinehart, Jr. K. L. & Stroshane, R. M. Biosynthesis of aminocyclitol antibiotics. *J. Antibiot.* 29, 319–353 (1976).

10 Rinehart, Jr. K. L. Biosynthesis and mutasynthesis of aminocyclitol antibiotics. *J. Antibiot.* 32, S32–S46 (1979).

11 Furumai, T., Takeda, K., Kinumaki, A., Ito, Y. & Okuda, T. Biosynthesis of butirosins. II. Biosynthetic pathway of butirosins elucidated from cosynthesis and feeding experiments. *J. Antibiot.* 32, 891–899 (1979).

12 Walker, J. B. Enzymic synthesis of aminocyclitol moieties of aminoglycoside antibiotics from inositol by *Streptomyces* spp.: detection of glutamine-aminocyclitol aminotransferase and diaminocyclitol aminotransferase activities in a spectinomycin producer. *J. Bacteriol.* 177, 818–822 (1995).

13 Distler, J., Braun, C., Ebert, A. & Piepersberg, W. Gene cluster for streptomycin biosynthesis in *Streptomyces griseus*: analysis of a central region including the major resistance gene. *Mol. Genet.* 208, 204–210 (1987).

14 Distler, J., Mansouri, K., Mayer, G., Stockmann, M. & Piepersberg, W. Streptomycin biosynthesis and its regulation in Streptomycetes. *Gene* 115, 105–111 (1992).

15 Ahlert, J., Distler, J., Mansouri, K. & Piepersberg, W. Identification of *stsC*, the gene encoding the L-glutamine:scyllo-inosose aminotransferase from streptomycin-producing Streptomycetes. *Arch. Microbiol.* 168, 102–113 (1997).

16 Fritsche, E., Bergner, A., Humm, A., Piepersberg, W. & Huber, R. Crystal structure of L-arginine:inosamine-phosphate amidinotransferase StrB1 from *Streptomyces griseus*: an enzyme involved in streptomycin biosynthesis. *Biochemistry* 37, 17664–17672 (1998).

17 Beyer, S., Mayer, G. & Piepersberg, W. The StrQ protein encoded in the gene cluster for 5′-hydroxystreptomycin of *Streptomyces glaucescens* GLA.O is a alpha-D-glucose-1-phosphate cytidylyltransferase (CDP-D-glucose synthase). *Eur. J. Biochem.* 258, 1059–1067 (1998).

18 Yamase, H., Zhao, L. & Liu, H.-w. Engineering a hybrid sugar biosynthetic pathway: production of L-rhamnose and its implication on dihydrostreptose biosynthesis. *J. Am. Chem. Soc.* 122, 12397–12398 (2000).

19 Kudo, F., Hosomi, Y., Tamegai, H. & Kakinuma, K. Purification and characterization of 2-deoxy-scyllo-inosose synthase derived from *Bacillus circulans*. A crucial carbocyclization enzyme in the biosynthesis of 2-deoxystreptamine-containing aminoglycoside antibiotics. *J. Antibiot.* 52, 81–88 (1999).

20 Kudo, F. *et al.* Molecular cloning of the gene for the key carbocycle-forming enzyme in the biosynthesis of 2-deoxystreptamine-containing aminocyclitol antibiotics and its comparison with dehydroquinate synthase. *J. Antibiot.* 52, 559–571 (1999).

21 Ota, Y. *et al.* Butirosin-biosynthetic gene cluster from *Bacillus circulans*. *J. Antibiot.* 53, 1158–1167 (2000).

22 Kudo, F. & Eguchi, T. Biosynthetic enzymes for the aminoglycosides butirosin and neomycin. *Methods Enzymol.* 459, 493–519 (2009).

23 Piepersberg, W., Aboshanab, K. M., Schmidt-Beissner, H. & Wehmeier, U. F. The biochemistry and genetics of aminoglycoside producers. in *Aminoglycoside Antibiotics* (ed. Arya, D. P.) 15–118 (John Wiley & Sons Inc., Hoboken, NJ, 2007).

24 Tamegai, H. *et al.* Identification of L-glutamine: 2-deoxy-scyllo-inosose aminotransferase required for the biosynthesis of butirosin in *Bacillus circulans*. *J. Antibiot.* 55, 707–714 (2002).

25 Huang, F., Li, Y., Yu, J. & Spencer, J. B. Biosynthesis of aminoglycoside antibiotics: cloning, expression and characterization of an aminotransferase involved in the pathway to 2-deoxystreptamine. *Chem. Commun.* 2860–2861 (2002).

26 Kudo, F., Yamamoto, Y., Yokoyama, K., Eguchi, T. & Kakinuma, K. Biosynthesis of 2-deoxystreptamine by three crucial enzymes in *Streptomyces fradiae* NBRC 12773. *J. Antibiot.* 58, 766–774 (2005).

27 Yokoyama, K. *et al.* Stereochemical recognition of doubly functional aminotransferase in 2-deoxystreptamine biosynthesis. *J. Am. Chem. Soc.* 127, 5869–5874 (2005).

28 Yokoyama, K., Yamamoto, Y., Kudo, F. & Eguchi, T. Involvement of two distinct N-acetylglucosaminyltransferases and a dual-function deacetylase in neomycin biosynthesis. *Chembiochem.* 9, 865–869 (2008).

29 Huang, F. *et al.* Elaboration of neosamine rings in the biosynthesis of neomycin and butirosin. *Chembiochem.* 8, 283–288 (2007).

30 Kudo, F., Fujii, T., Kinoshita, S. & Eguchi, T. Unique O-ribosylation in the biosynthesis of butirosin. *Bioorg. Med. Chem.* 15, 4360–4368 (2007).

31 Li, Y., Llewellyn, N. M., Giri, R., Huang, F. & Spencer, J. B. Biosynthesis of the unique amino acid side chain of butirosin: possible protective-group chemistry in an acyl carrier protein-mediated pathway. *Chem. Biol.* 12, 665–675 (2005).

32 Llewellyn, N. M., Li, Y. & Spencer, J. B. Biosynthesis of butirosin: transfer and deprotection of the unique amino acid side chain. *Chem. Biol.* 14, 379–386 (2007).

33 Yokoyama, K., Numakura, M., Kudo, F., Ohmori, D. & Eguchi, T. Characterization and mechanistic study of a radical SAM dehydrogenase in the biosynthesis of butirosin. *J. Am. Chem. Soc.* 129, 15147–15155 (2007).

34 Park, S. H. *et al.* Expanding substrate specificity of GT-B fold glycosyltransferase via domain swapping and high-throughput screening. *Biotechnol. Bioeng.* 102, 988–994 (2009).

35 Nepal, K. K., Oh, T. J. & Sohng, J. K. Heterologous production of paromamine in *Streptomyces lividans* TK24 using kanamycin biosynthetic genes from *Streptomyces kanamyceticus* ATCC12853. *Mol. Cells* 27, 601–608 (2009).

36 Hong, W. R. *et al.* Molecular cloning and sequence analysis of the sisomicin biosynthetic gene cluster from *Micromonospora inyoensis*. *Biotechnol. Lett.* 31, 449–455 (2009).

37 Park, J. W. *et al.* Genetic dissection of the biosynthetic route to gentamicin A2 by heterologous expression of its minimal gene set. *Proc. Natl Acad. Sci. USA* 105, 8399–8404 (2008).

38 Kim, J. Y. *et al.* Gene inactivation study of *gntE* reveals its role in the first step of pseudotrisaccharide modifications in gentamicin biosynthesis. *Biochem. Biophys. Res. Commun.* 372, 730–734 (2008).

39 Kuzuyama, T., Seki, T., Dairi, T., Hidaka, T. & Seto, H. Nucleotide sequence of fortimicin KL1 methyltransferase gene isolated from *Micromonospora olivasterospora*, and comparison of its deduced amino acid sequence with those of methyltransferases involved in the biosynthesis of bialaphos and fosfomycin. *J. Antibiot.* 48, 1191–1193 (1995).

40 Kim, K. R., Kim, T. J. & Suh, J. W. The gene cluster for spectinomycin biosynthesis and the aminoglycoside-resistance function of *spcM* in *Streptomyces spectabilis*. *Curr. Microbiol.* 57, 371–374 (2008).

41 Hara, H., Ohnishi, Y. & Horinouchi, S. DNA microarray analysis of global gene regulation by A-factor in *Streptomyces griseus*. *Microbiology* 155, 2197–2210 (2009).

42 Ikeno, S., Aoki, D., Hamada, M., Hori, M. & Tsuchiya, K. S. DNA sequencing and transcriptional analysis of the kasugamycin biosynthetic gene cluster from *Streptomyces kasugaensis* M338-M1. *J. Antibiot.* 59, 18–28 (2006).

43 Walker, J. B. Pathways of biosynthesis of the guanidinated inositol moieties of streptomycin and bluensomycin. *Methods Enzymol.* 43, 429–433 (1975).

44 Thapa, L. P., Oh, T. J., Liou, K. & Sohng, J. K. Biosynthesis of spectinomycin: heterologous production of spectinomycin and spectinamine in an aminoglycoside-deficient host, *Streptomyces venezuelae* YJ003. *J. Appl. Microbiol.* 105, 300–308 (2008).
