
Glycobiology vol. 19 no. 11 pp. 1197–1203, 2009  
doi:10.1093/glycob/cwp110  
Advanced Access publication on August 6, 2009  

# Dermatan 4-O-sulfotransferase 1 is pivotal in the formation of iduronic acid blocks in dermatan sulfate  

Benny Pacheco, Marco Maccarana, and Anders Malmström¹  

Department of Experimental Medical Science, BMC D12, Lund University, SE 221 84 Lund, Sweden  

Received on March 11, 2009; revised on June 18, 2009; accepted on July 16, 2009  

Chondroitin/dermatan sulfate is a highly complex linear polysaccharide ubiquitously found in the extracellular matrix and at the cell surface. Several of its functions, such as binding to growth factors, are mediated by domains composed of alternating iduronic acid and 4-O-sulfated N-acetylgalactosamine residues, named 4-O-sulfated iduronic acid blocks. These domains are generated by the action of two DS-epimerases, which convert D-glucuronic acid into its epimer L-iduronic acid, in close connection with 4-O-sulfation. In this study, dermatan sulfate structure was evaluated after downregulating or increasing dermatan 4-O-sulfotransferase 1 (D4ST-1) expression. siRNA-mediated downregulation of D4ST-1 in primary human lung fibroblasts led to a drastic specific reduction of iduronic acid blocks. No change of epimerase activity was found, indicating that the influence of D4ST-1 on epimerization is not due to an altered expression level of the DS-epimerases. Analysis of the dermatan sulfate chains showed that D4ST-1 is essential for the biosynthesis of the disulfated structure iduronic acid-2-O-sulfate-N-acetylgalactosamine-4-O-sulfate, thus confirmed to be strictly connected with the iduronic acid blocks. Also the biologically important residue hexuronic acid-N-acetylgalactosamine-4,6-O-disulfate considerably decreased after D4ST-1 downregulation. In conclusion, D4ST-1 is a key enzyme and is indispensable in the formation of important functional domains in dermatan sulfate and cannot be compensated by other 4-O-sulfotransferases.

**Keywords**: chondroitin sulfate/dermatan sulfate/epimerase/iduronic acid/sulfotransferase

## Introduction

Growth factor signaling is modulated by the presence of various factors in the extracellular matrix (ECM). Several proteoglycans (PGs) bind and sequester growth factors in the ECM and act as coreceptors at the cell surface, thereby regulating the availability and direct signaling, respectively. This is an important function during patterning of an embryo where PGs establish morphogen gradients (Hacker et al. 2005). PGs consist of a protein core to which one or several glycosaminoglycan (GAG) chains are covalently attached. One family of PGs is the chondroitin/dermatan sulfate (CS/DS) carrying proteoglycans that includes versican, decorin, and biglycan. Versican is a large PG, carrying multiple CS/DS chains while decorin and biglycan are small PGs with one and two CS/DS chains, respectively (Iozzo 1998).

CS/DS is a long linear polymer of alternating hexuronic acid (either D-glucuronic acid (GlcA) or L-iduronic acid (IdoA)) and N-acetylgalactosamine linked via β-(1 → 3) and β-(1 → 4) bonds, respectively, and ranges in length from 20 to 80 repeats of this disaccharide unit (Trowbridge and Gallo 2002). During assembly, the chain is extensively modified by epimerization and sulfation. Two epimerases, DS-epimerase 1 (Maccarana et al. 2006) and DS-epimerase 2 (Pacheco et al. 2009b) convert GlcA into IdoA to a variable extent. This results in the formation of various IdoA domains in CS/DS, i.e., IdoA blocks or alternating IdoA/GlcA residues. Both GalNAc and GlcA/IdoA can be modified by sulfation. GalNAc can be sulfated in the 4-O-position by the three major sulfotransferases C4ST-1, C4ST-2, and D4ST-1 which have different substrate specificities (Evers et al. 2001; Hiraoka et al. 2001; Mikami et al. 2003). In vitro, C4ST-1 preferably sulfates a GalNAc flanked by two GlcA residues while D4ST-1, in contrast, prefers two flanking IdoA residues. C4ST-2 on the other hand has an equal preference for both substrates. GalNAc are also sulfated in position 6 by C6ST-1 and C6ST-2 (Kae et al. 1998; Kitagawa et al. 2000) and GlcA/IdoA in position 2 by the 2-O-sulfotransferase, UST (Kobayashi et al. 1999). Binding of several growth factors to the CS/DS chain is strictly dependent on these modifications.

In dermatan sulfate from both tissue samples and cell cultures, IdoA blocks are commonly associated with 4-O-sulfation of GalNAc. Moreover, several studies have shown a close connection between 4-O-sulfation of GalNac and epimerization of GlcA. For example, reducing sulfation in cultured cells by either chlorate treatment or lowering the amount of sulfate in the growing medium, results in the formation of a low epimerized dermatan sulfate (Silbert et al. 1986; Greve et al. 1988). Further, concomitant sulfation is also required for an efficient epimerization in vitro (Malmstrom et al. 1975). The question which 4-O-sulfotransferase is working together with the DS-epimerase(s) to generate 4-O-sulfated IdoA blocks has remained open. However, it has been hypothesized that D4ST-1 is a likely candidate due to its substrate specificity (Mikami et al. 2003).

The aim of this study was to investigate the coupling between sulfation and epimerization, and specifically the involvement of D4ST-1 in this regard. We show, by using a siRNA approach, that loss of D4ST-1 leads to a substantial reduction in IdoA residues located in block structures without affecting epimerase activity. In addition, knockdown of D4ST-1 results in aberrant

---

¹ To whom correspondence should be addressed: Tel: +46-46-222-85-74; Fax: +46-46-211-34-17; e-mail: anders.malmstrom@med.lu.se

© The Author 2009. Published by Oxford University Press. All rights reserved. For permissions, please e-mail: journals.permissions@oxfordjournals.org 1197

B Pacheco et al.

![Figure 1](#fig1)
Fig. 1. Relative mRNA levels and sulfotransferase activity after siRNA-mediated knockdown of D4ST-1 in human lung fibroblasts. Two siRNAs targeting human D4ST-1 and one negative control siRNA were transfected into HFL-1 cells. Seventy-two hours post-transfection, mRNA levels for D4ST-1 were determined by RT-PCR (A) or cell lysates assayed for sulfotransferase activity toward dermatan (B). Sulfotransferase activity and D4ST-1 mRNA levels after knockdown are expressed as relative to the negative control and represent the mean of two independent experiments ± SD in (A) and four/two (siRNA 1/siRNA 2) independent experiments ± SD in (B), respectively.

4-O-sulfation of GalNAc indicating that C4ST-1 and C4ST-2 cannot compensate for the role of D4ST-1. The data in this study show a pivotal role for D4ST-1 in the biosynthesis and function of dermatan sulfate.

### Results

#### siRNA-mediated knockdown of D4ST-1 leads to a reduction in 4-O-sulfation of GalNAc

To address the function of the sulfotransferase D4ST-1 in dermatan sulfate biosynthesis, a siRNA approach was used to decrease the expression of this enzyme in human lung fibroblasts (HFL-1). Two siRNAs targeting D4ST-1 (siRNA 1 and siRNA 2) and one negative control siRNA (NC) were obtained and transfected into HFL-1 cells. Seventy-two hours post-transfection, a reduction in D4ST-1 mRNA of approximately 80% was observed for both siRNAs compared to control (Figure 1A). Sulfotransferase activity in cellular lysates was measured by the transfer of radiolabeled sulfate from the donor 3′-phosphoadenosine 5′-phosphosulfate (PAPS) into desulfated dermatan sulfate (dermatan). Compared to the negative control, there was only a

![Figure 2](#fig2)
Fig. 2. Isolation of CS/DS chains from proteoglycans after D4ST-1 siRNA treatment. HFL-1 cells were labeled with \[^{35}S\]sulfate after siRNA-mediated knockdown of D4ST-1, and proteoglycans released into the medium were isolated on a DE52-column. Isolated proteoglycans were thereafter separated by size on a Superose 6 column (A) to separate versican from biglycan/decorin. Biglycan/decorin fractions were pooled (as indicated with a straight line), and CS/DS released by beta-elimination. Subsequently, heparan sulfate was destroyed by deamination at pH 1.5 and CS/DS re-isolated on a Superose 6 column (B).

25 and 40% reduction in sulfotransferase activity for the two siRNAs, respectively (Figure 1B). The seemingly low knock-down is not due to inefficient siRNAs since the assay is not specific for D4ST-1, and several other sulfotransferases with activity against dermatan are also measured (e.g., C4ST-2). No apparent morphological changes were observed after knock-down of D4ST-1. However, a significant reduction in protein recovery (52% ± 3) was noted after lysis of cells transfected with D4ST-1 siRNA 2 compared to the negative control, due to an apparent decrease in the proliferation rate. This effect was not observed for D4ST-1 siRNA 1.

To investigate the effect of targeting D4ST-1 on the formation of dermatan sulfate, transfected cells were labeled with \[^{35}S\]-sulfate. Proteoglycans were isolated from the conditioned medium with ion-exchange chromatography. Subsequently, biglycan and decorin were separated from versican with gel filtration (Figure 2A). The GAG-chains of the small proteoglycans were subsequently released from the core-protein, treated with HNO₂ to remove HS, and subjected to gel filtration (Figure 2B). No significant changes in the relative amount of HS or any difference in dermatan sulfate chain-length was observed after siRNA treatment (Figure 2B).

Table I. Composition of sulfated disaccharides obtained after digestion of CS/DS chains from decorin/biglycan with chondroitinase ABC

| Total labeled sulfates (%) | Control | D4ST-1 siRNA 1 | D4ST-1 siRNA 2 |
|----------------------------|---------|----------------|----------------|
| ΔA units, ΔHexA-GalNAc4S   | 77      | 66             | 53             |
| ΔC units, ΔHexA-GalNAc6S   | 14      | 28             | 42             |
| ΔB units, ΔHexA2S-GalNAc4S | 4.8     | 1.4            | 0.8            |
| ΔD units, ΔHexA2S-GalNAc6S | 1.9     | 2.8            | 3.3            |
| ΔE units, ΔHexA-GalNAc4S6S | 2.4     | 1.5            | 1.4            |

The isolated DS chains were treated with chondroitinase ABC and analyzed with HPLC to determine the sulfation status. Knockdown of D4ST-1 resulted in a decrease in ΔA units (ΔHex-GalNAc4S) ranging from 77% of the total labeled sulfates in the control sample to 66 and 53% in the two siRNA samples, respectively (Table I). The greater decrease in siRNA 2 treated samples is in agreement with activity data (Figure 1B). The biologically relevant disulfated structures ΔB (ΔHex2S-GalNAc4S) and ΔE (ΔHex-GalNAc4S6S) amounted to 4.8 and 2.4% in the control and decreased upon siRNA treatment (Table I). This clearly indicates that D4ST-1 is involved in the formation of these structures. Interestingly, the reduction in the above sulfation was accompanied by a relative 2- to 3-fold increase in 6-O-mono-sulfated units (Table I). Also, the disulfated structure ΔD (ΔHex2S-GalNAc6S) increased. In summary, these data indicate that D4ST is involved in the biosynthesis of several biologically active structures in dermatan sulfate and its role cannot be compensated by the presence of C4ST-1 and C4ST-2.

---

**D4ST-1 regulates the IdoA content and distribution in dermatan sulfate**

The observed decrease in 4-O-sulfation and B structures (Table I) that in mammals have only been found in the iB-form (IdoA2S-GalNAc4S) prompted us to examine if knockdown of D4ST-1 affected epimerization. \[^{35}S\]-labeled DS from decorin/biglycan was therefore purified as above and subsequently treated with chondroitinase AC-I or chondroitinase B. Digestion of the GAG chains with chondroitinase AC-I, which only cleaves glucuronic acid containing structures, resulted in a substantial amount of larger oligosaccharides in the control, representing resistant IdoA-containing structures, i.e., iduronic acid blocks (Figure 3A). However, after knockdown of D4ST-1, the majority of the digested material was instead found in the disaccharide peak (Figure 3A). Interestingly, a specific loss was seen in consecutive IdoA residues (larger than octasaccharides after ACI treatment, i.e., iduronic acid blocks), while alternating IdoA/GlcA structures (≤ octasaccharides) were increased compared to the control (Figure 3A).

Digestion of the isolated chains with chondroitinase B, which only cleave IdoA linkages, confirmed the loss of IdoA containing structures with an increase of resistant material in samples treated with siRNA for D4ST-1 (Figure 3B). Altogether, a total decrease in the IdoA content of 48 and 70% (calculated after chondroitinase B treatment) was observed after knockdown of D4ST-1 with the two siRNAs, respectively (Figure 3C).

---

Fig. 3. Analysis of the epimerization pattern obtained after knockdown of D4ST-1. \[^{35}S\] sulfate-labeled CS/DS from HFL-1 cells was isolated and subjected to chondroitinase AC-I treatment (A) or digestion with chondroitinase B (B) before separation of split products on a Superdex peptide 10/30 column operated at 0.3 mL/min in 0.2 M NH₄HCO₃. Elution positions of oligosaccharide standards are indicated with arrows. (C) Quantification of the total iduronic acid content from the chondroitinase B split products' distribution, relative to the negative control.

---

Knockdown of D4ST-1 does not affect DS-epimerase activity

To examine if knockdown of D4ST-1 affects epimerase activity, lysates from siRNA-treated and control cells were assayed by measuring the release of tritium from the labeled substrate [5-³H]dK4 (Figure 4). No effect in tritium-release could be detected, indicating that D4ST-1 does not directly influence the total epimerase activity present in the cell, which could have explained the observed reduction in the iduronic acid content.

It has previously been shown that an interaction between the heparan sulfate (HS)-2-O-sulfotransferase and the HS-epimerase is necessary for proper localization of the enzyme to the Golgi (Pinhal et al. 2001). We therefore investigated if an interaction exists between D4ST-1 and any of the

B Pacheco et al.

Fig. 4. Measurement of epimerase activity in HFL-1 cells after downregulation of D4ST-1 by siRNA. Human lung fibroblasts were transfected with siRNA against D4ST-1 and 72 h post-transfection cells were lysed. Epimerase activity was measured as release of tritium from \([5-{ }^{3} \mathrm{H}] \mathrm{dK} 4\) and data are expressed relative to that of the negative control. Data represent the mean of four independent transfections \pm SD.

two DS-epimerases. D4ST-1 with a C-terminal TAP-tag was overexpressed together with either DS-epi1/His or FLAG/DS-epi2 in 293HEK cells and subsequently immunoprecipitated using an IgG-Sephrose gel. D4ST-1/CTAP could be readily purified. However, DS-epi1 and FLAG/DS-epi2 were not detected in the bound fractions when probed with an anti-DS-epi1 or anti-FLAG antibody in Western blot, respectively (data not shown).

In summary, D4ST-1 does not directly influence epimerase activity, and no physical interaction could be detected between this enzyme and either of the two DS-epimerases.

Overexpression of D4ST-1 in 293HEK-cells does not affect the structure of biglycan/decorin dermatan sulfate chains

To investigate whether overexpression of D4ST-1 would stimulate the formation of IdoA, epithelial 293HEK cells were stably transfected with a C-terminally TAP-tagged D4ST-1. Two D4ST-1/CTAP clones with a 3.6-fold (clone 1) and 5.7-fold (clone 2) increase in sulfotransferase activity toward dermatan were selected and labeled with \({ }^{[35} \mathrm{~S}\)-sulfate (Figure 5A). However, CS/DS chains of both versican (data not shown) and decorin/biglycan (Figure 5B) isolated from D4ST-1/CTAP overexpressing clones showed a similar pattern after chondroitinase AC-I treatment as control cells. Of note is the lower amount of both total IdoA content and the distribution of IdoA in decorin/biglycan from epithelial 293HEK-cells, which is mainly found in alternating IdoA/GlcA structures, compared to that found in fibroblasts, which is mainly located in iduronic acid blocks (Figures 3A and 5B). These data strongly support an important role for D4ST-1 in the regulation of both IdoA content and distribution in dermatan sulfate.

Discussion

A key feature of dermatan sulfate is its domain structure, i.e., the presence of IdoA blocks separated by blocks of GlcA or alternating GlcA/IdoA residues (Fransson and Roden 1967; Coster et al. 1975). In vivo, IdoA blocks are commonly associated with 4-O-sulfation of GalNAc. These domains thus arise from the action of two DS-epimerases (DS-epi1 and DS-epi2) (Pacheco

et al. 2009a, 2009b) with subsequent sulfation in the 4-O-position of GalNAc by D4ST-1 (Evers et al. 2001) when present in the sequence IdoAα1-3GalNAcβ1-4IdoA. 4-O-Sulfated IdoA blocks have an important role in mediating the biological functions of dermatan sulfate through binding to ligands such as fibroblast growth factor 2, hepatocyte growth factor, and heparin cofactor II (Maimone and Tollefsen 1990; Lyon et al. 1998; Taylor et al. 2005). In this study, we have demonstrated a key role for the 4-O-sulfotransferase D4ST-1 in the formation of such functional domains. In addition to a reduced 4-O-sulfation, siRNA-mediated knockdown of D4ST-1 significantly reduced the amount of IdoA residues located in blocks but not in alternating GlcA/IdoA structures (Figure 3). It is interesting to note that C4ST-2, which is present in HFL-1 cells (Tiedemann et al. 2005), does not seem to be able to compensate for the role of D4ST-1. This is somewhat surprising since its substrate preference in vitro is very similar to that of D4ST-1 (Mikami et al. 2003) and points to other mechanisms in vivo. It has previously been shown that also C4ST-1 deficiency cannot be rescued by expression of C4ST-2 (Uyama et al. 2006), and C4ST-1 knockout mice show a severe chondrodysplastic phenotype (Kluppel et al. 2005). Thus, the three main 4-O-sulfotransferases do not have redundant roles in dermatan sulfate biosynthesis. It is possible that C4ST-2, in contrast to D4ST-1 and C4ST-1, is more involved in sulfating mixed IdoA/GlcA structures in vivo,
which is consistent with its previously determined substrate specificity in vitro.

The reduction of the IdoA content after knockdown of D4ST-1 is in agreement with previous studies showing a coupling between the epimerase and sulfotransferase reactions in dermatan sulfate biosynthesis (Malmstrom et al. 1975; Silbert et al. 1986). Notably, the observed decrease in IdoA blocks is not due to a direct effect on epimerase activity since no change in total epimerase activity could be detected after knockdown of D4ST-1. It has long been suggested that the coupling between sulfation and epimerization might be facilitated by an interaction of a sulfotransferase and a DS-epimerase. Furthermore, the biosynthetic enzymes might exist in “Gagosomes” where the relative levels of the different enzymes determine the structure of the resulting chain (Esko and Selleck 2002). A physical interaction between biosynthetic enzymes in the heparan sulfate biosynthesis has also been shown to be important in targeting to the Golgi, i.e., the kin-recognition mechanism (Pinhal et al. 2001). A similar mechanism could be in place for D4ST-1 and either one, or both DS-epimerases. However, no direct interaction could be detected in our experimental setting. If an interaction exists it might be transient or too weak for detection in traditional immunoprecipitation experiments or alternatively could be mediated by other proteins that are titrated out in our overexpressing setup. Although conclusive evidence is missing, these data indicate that targeting of the DS-epimerases to the Golgi via D4ST-1 might not occur. A plausible link between the action of D4ST1 and DS-epimerases might instead be the “4-O-sulfation lock” of IdoA. This hypothesis was deduced from experiments in vitro showing that chondroitin/dermatan sulfate is not a substrate for epimerization (Malmstrom 1984) and that the addition of PAPS to epimerase assays, using a nonsolubilized enzyme source, shifts the equilibrium of epimerization to the formation of more IdoA by reducing “back epimerization” of IdoA (Malmstrom et al. 1975). The significant lower amount of IdoA observed after knockdown of D4ST-1 could therefore potentially be a result of back epimerization of IdoA into GlcA due to the lack of 4-O-sulfation but this remains to be elucidated. It also remains to be established if there is any difference in the coupling between D4ST-1 and DS-epi 1 and 2, which can potentially be addressed in knockout animals currently being developed in our laboratory.

A drastic reduction of ΔB (ΔHexA2S-GalNAc4S) units was observed in decorin/biglycan-derived DS-chains after siRNA treatment (Table I). The proportion of B units within the IdoA blocks was not experimentally determined. However, a similar decrease was also obtained after knockdown of DS-epi 1 or DS-epi 2 (data not shown), confirming that this unit is closely associated with epimerization (Maimone and Tollefsen 1990; Karamanos et al. 1995). This is consistent with the reported substrate specificity of the CS/DS-2-O-ST (UST) (Kobayashi et al. 1999; Xu et al. 2007). Moreover, a decrease was observed for ΔE units (ΔHexA-GalNAc4S6S) (Table I). We did not establish if, and in what proportion, E units are located within the IdoA blocks. These types of disulfated structures have been shown to constitute the binding site for a number of ligands, including heparin cofactor II and the neurotrophic factor midkine (Maimone and Tollefsen 1990; Deepa et al. 2002; Halldorsdottir et al. 2006).

Of interest is the fact that no increase in the iduronic acid content was observed after overexpression of D4ST-1 in 293HEK

D4ST-1 regulates the iduronic acid content and distribution epithelial cells. It has previously been shown that priming dermatan sulfate biosynthesis by the addition of increasing amounts of xylosides reduces the IdoA content in the produced DS, suggesting that the DS-epimerases are limiting factors (Coster et al. 1991). Furthermore, tissues with low epimerase activity also produce DS with fewer IdoA residues (Tiedemann et al. 2001). The inability to demonstrate an increase in the IdoA content after overexpression of D4ST-1 in epithelial cells could therefore potentially be explained by that insufficient epimerase activity is present to cope with increased D4ST-1 expression. It is interesting to note that when sufficient epimerase activity is present, the amount of D4ST-1 in the cell directly correlates with the level of IdoA blocks, as shown with two siRNAs with different efficiency. We have previously shown that TGF-β alone or together with EGF and PDGF regulates epimerase activity, and the expression of both C4ST-1 and D4ST-1 (Tiedemann et al. 2005). As indicated above, a vast array of growth factors binds dermatan sulfate via specific structures thereby influencing their function. Fine-tuning the expression of D4ST-1, and the two DS-epimerases, during development and pathology could therefore be a mechanism to, for example, regulate the availability of growth factors for signaling at the cell surface.

In conclusion, we suggest a pivotal role for D4ST-1 in the formation of 4-O-sulfated IdoA blocks as well as of the disulfated structures IdoA2S-GalNAc4S and HexA-GalNAc4S6S in dermatan sulfate. The multiplicity and the biological activities of these structures make D4ST-1 an interesting target for enzyme inhibitors of potential pharmacological interest.

### Material and methods

#### Materials
Human lung fibroblasts and 293HEK cells were obtained from ATCC and grown in MEM supplemented with PEST, glutamine, and 10% DCS or 10% FBS, respectively. [5-³H]-Defructosylated K4 (dK4) was prepared as previously described (Hannesson et al. 1996). A negative control siRNA was obtained from Qiagen (Solna, Sweden) and two siRNAs with the sequences 5′-GGCCUUUUGAGGUUGUGACUtt-3′ (D4ST-1 siRNA 1) and 5′-GGAUGUGUCUGCCUAAGUAUtt-3′ (D4ST-1 siRNA 2) were obtained from Ambion (Stockholm, Sweden).

Primers were obtained from DNA Technology A/S (Risskov, Denmark) and DS-epi1/HIS and FLAG/DS-epi2 expression vectors were prepared according to Maccarana et al. (2006) and Pacheco et al. (2009), respectively.

#### Overexpression of D4ST-1 in 293HEK
To generate a C-terminally TAP-tagged D4ST1, the open reading frame of human D4ST-1 was amplified with the primers 5′-gatcgttttaacatgtttccccgcccgctga-3′ (forward) introducing a *PmeI* restriction site and 5′-gatcttaattaactgctgacacgcctccttg-3′ (reverse) introducing a *PacI* restriction site. The amplified product was thereafter inserted into the *PmeI* and *PacI* sites in the pcDNA3/CTAP vector (a generous gift from Dr. Anne-Claude Gingras (Gingras et al. 2005)) and sequenced. The pcDNA3.1-D4ST-1/CTAP vector was transfected into 293HEK cells using lipofectamine 2000, used according to manufacturer instructionsB Pacheco et al.

(Invitrogen, Paisley, United Kingdom), and stably transfected clones obtained by selection with 1g/l of G418.

### siRNA-mediated knockdown of D4ST-1

HFL-1 were plated at a cell density of $2.8 \times 10^6$ cells in a $75 \mathrm{~cm}^2$ flask together with siRNA/lipofectamine mixture prepared by mixing $22.5 \mu \mathrm{L}$ siRNA (100 $\mu \mathrm{M}$ stock) and $37.5 \mu \mathrm{L}$ lipofectamine 2000 (Invitrogen, Paisley, United Kingdom), according to manufacturer instructions giving a final siRNA concentration of $100 \mathrm{nM}$. Cells were incubated for $48 \mathrm{~h}$ and the medium was thereafter changed to sulfate-deprived MEM, $10 \% \mathrm{DCS}, 1 \%$ glutamine, $1 \% \mathrm{PEST}$ containing $100 \mu \mathrm{Ci} / \mathrm{mL}$ of ${ }^{35} \mathrm{SO}_{4}$. After a $24 \mathrm{~h}$ incubation, the medium was collected and cells lysed in $1.2 \mathrm{~mL}$ of $50 \mathrm{mM}$ HEPES, pH 7.5, $10 \%$ glycerol, $100 \mathrm{mM} \mathrm{KCl}$, $2 \mathrm{mM} \mathrm{EDTA}, 0.1 \% \mathrm{NP}-40, 2 \mathrm{mM} \mathrm{DTT}$, and protease inhibitors.

### RT-PCR

HFL-1 cells were transfected as above with siRNAs targeting D4ST-1 and after a 72-h incubation total RNA was purified. cDNA was subsequently synthesized from $1 \mu \mathrm{g}$ of total RNA using the iScript cDNA synthesis kit, BIORAD. cDNA corresponding to an amount of $100 \mathrm{ng}$ was thereafter used for PCR on a ABI PRISM platform with iTAQ SYBR green supermix with ROX, BIORAD. Relative quantification was performed according to the delta delta Ct method using 18S as a housekeeping gene. Primers used in the PCR reaction were: 18S-RNA forward ($5^{\prime}-\operatorname{cgaacgtctgccctatcaac}-3^{\prime}$), 18S-RNA reverse ($5^{\prime}-\operatorname{tgccctccttggatgtggta}-3^{\prime}$), D4ST-1 forward ($5^{\prime}-\operatorname{aacgctatggggctgagata}-3^{\prime}$), D4ST-1 reverse ($5^{\prime}-\operatorname{atctcaggaactcggggaat}-3^{\prime}$).

### Sulfotransferase assay

Sulfotransferase activity using desulfated dermatan sulfate was performed in $50 \mathrm{mM}$ imidazole-HCl, pH 6.8, $12.5 \mu \mathrm{g}$ dermatan, $2 \mathrm{mM}$ dithiothreitol, $10 \mathrm{mM} \mathrm{MnCl}_{2}, 1 \%$ Triton X-100, and $1 \mu \mathrm{Ci}$ PAPS${ }^{35}$ for $1 \mathrm{~h}$ at $37^{\circ} \mathrm{C}$. The reaction was stopped by the addition of $95 \%$ ethanol/$1.3 \% \mathrm{NaAc}$ and $200 \mu \mathrm{g}$ of DS carrier. Unincorporated PAPS${ }^{35}$ was removed on a fast desalting column packed with Sephadex G-25 as described earlier (Mikami et al. 2003). Radioactive sulfate incorporation into dermatan was quantified by liquid scintillation counting.

### Epimerase assay

Epimerase activity was assayed by measuring ${ }^{3} \mathrm{H}$-release from [5-${ }^{3} \mathrm{H}] \mathrm{dK} 4$ as previously described (Maccarana et al. 2006). Briefly, cell lysates were desalted on Sephadex-G25 columns equilibrated with the desalting buffer, $20 \mathrm{mM} \mathrm{MES}$ (pH 5.5 at $37^{\circ} \mathrm{C}), 10 \%$ glycerol, $0.5 \mathrm{mM} \mathrm{EDTA}, 0.1 \%$ Triton X-100, $1 \mathrm{mM} \mathrm{DTT}$, protease inhibitors. Assays were performed in $100 \mu \mathrm{L}$ final volume of $0.8 \mathrm{X}$ desalting buffer, $0.5 \mathrm{mg} \mathrm{BSA}, 2 \mathrm{mM} \mathrm{MnCl}_{2}$, $0.5 \% \mathrm{NP} 40$, and $30,000 \mathrm{dpm}[5-{ }^{3} \mathrm{H}] \mathrm{dK} 4(200 \mu \mathrm{M} \mathrm{HexA})$ for $15 \mathrm{~h}$ at $37^{\circ} \mathrm{C}$, stopped by boiling and centrifuged. Quantification of released ${ }^{3} \mathrm{H}_{2} \mathrm{O}$ was performed after distillation of $90 \mu \mathrm{L}$ of the reaction mixture by liquid scintillation counting.

and protease inhibitors. The medium was subsequently applied on a $0.25 \mathrm{~mL}$ DE-52 column, washed with 50 volumes of $6 \mathrm{M}$ Urea/$50 \mathrm{mM} \mathrm{NaAc}$ (pH 5.5), $0.1 \%$ Triton X-100, $5 \mathrm{mM} \mathrm{EDTA}$, $1 \mathrm{mM} \mathrm{DTT}$, and protease inhibitors and bound material eluted with $1 \mathrm{~mL}$ of buffer C (4 M guanidine/$50 \mathrm{mM} \mathrm{NaAc}$ (pH 5.8), $0.1 \%$ Triton X-100, $5 \mathrm{mM} \mathrm{EDTA}, 1 \mathrm{mM} \mathrm{DTT}$, and protease inhibitors), and $50 \mu \mathrm{g}$ of dextran T500 was added to each sample as carrier. Isolated proteoglycans were thereafter size-separated on a Superose 6 column operated at $0.5 \mathrm{~mL} / \mathrm{min}$ with buffer C. Individual proteoglycan fractions were pooled, precipitated with four volumes of $95 \% \mathrm{EtOH} / 1.3 \% \mathrm{NaAc}$ overnight and centrifuged for $30 \mathrm{~min}$ at $20,000 \times g$. GAG chains were subsequently removed by beta-elimination for $16 \mathrm{~h}$ at $45^{\circ} \mathrm{C}$ with $50 \mathrm{mM} \mathrm{KOH}$ and $0.1 \mathrm{M} \mathrm{NaBH}_{4}$. Released GAGs were then reisolated on a DE-52 column and deaminated at pH 1.5 to destroy heparan sulfate. CS/DS chains were isolated from destroyed HS on a Superose 6 column run in $0.2 \mathrm{M} \mathrm{NH}_{4} \mathrm{HCO}_{3}$ operated as above and lyophilized.

### Chondroitinase ABC, AC-I, and B digestion

Dermatan sulfate isolated as above were digested with $0.25 \mathrm{mU}$ of chondroitinase B for $2 \mathrm{~h}$ at $37^{\circ} \mathrm{C}$ in $50 \mu \mathrm{L}$ of $20 \mathrm{mM}$ Tris-HCl (pH 7.5), $50 \mathrm{mM} \mathrm{NaCl}, 4 \mathrm{mM} \mathrm{CaCl}_{2}$, and $0.1 \mathrm{mg} / \mathrm{mL} \mathrm{BSA}$. Chondroitinase AC-I digestions were performed with $0.5 \mathrm{mU}$ of enzyme for $2 \mathrm{~h}$ at $37^{\circ} \mathrm{C}$ in $50 \mu \mathrm{L}$ of $50 \mathrm{mM}$ Tris-HCl (pH 7.5) and $50 \mathrm{mM} \mathrm{NaAc}$. Alternatively, for disaccharide fingerprinting dermatan sulfate was digested with $25 \mathrm{mU}$ of chondroitinase ABC for $3 \mathrm{~h}$ at $37^{\circ} \mathrm{C}$ in $50 \mu \mathrm{L}$ of $50 \mathrm{mM}$ Tris-HCl (pH 8.0) and $50 \mathrm{mM} \mathrm{NaAc}$. The reactions were stopped by boiling for $2 \mathrm{~min}$ and centrifuged. One hundred and fifty microliters of $0.2 \mathrm{M} \mathrm{NH}_{4} \mathrm{HCO}_{3}$ were thereafter added to each sample before application to a Superdex peptide column operated in $0.2 \mathrm{M} \mathrm{NH}_{4} \mathrm{HCO}_{3}$ with a flow rate of $0.3 \mathrm{~mL} / \mathrm{min}$.

### Disaccharide fingerprinting

To analyze sulfation pattern, chondroitinase ABC-digested samples were reduced with $25 \mathrm{mM} \mathrm{NaBH}_{4}$ for $15 \mathrm{~min}$ at $37^{\circ} \mathrm{C}$ and excess reagent destroyed by the addition of acetic acid to a final concentration of $250 \mathrm{mM}$. Reduced samples were applied to a Superdex peptide 10/30 column operated as above, and fractions containing reduced disaccharides were pooled and lyophilized. Samples were resuspended in $10 \mu \mathrm{L} \mathrm{H}_{2} \mathrm{O}$ and subsequently separated on a CarboPac PA1column operated at $1 \mathrm{~mL} / \mathrm{min}$ in $0.1 \mathrm{M} \mathrm{NaOH}$ with a $\mathrm{NaCl}$ gradient as described earlier (Midura et al. 1994).

### Funding

The Swedish Research Council; Medical Faculty of Lund University; Medical Faculty Program Tissue in Motion; Poly-sackaridforskning AB; the Johan and Greta Kock foundation; and the Österlund foundation.

### Conflict of interest statement

None declared.

### Abbreviations

CS, chondroitin sulfate; DS, dermatan sulfate; DS-epi1, dermatan sulfate epimerase 1; DS-epi2, dermatan sulfate epimerase

### Isolation of metabolically labeled GAG chains from the cellular medium

After labeling of cells as indicated above, the medium was collected and diluted 1:3 with $9 \mathrm{M}$ Urea/$50 \mathrm{mM} \mathrm{NaAc}$ (pH 5.5), $0.1 \%$ Triton X-100, $0.25 \mathrm{M} \mathrm{NaCl}, 5 \mathrm{mM} \mathrm{EDTA}, 1 \mathrm{mM} \mathrm{DTT}$, 1202
2; ECM, extracellular matrix; GAG, glycosaminoglycan; GalNAc, N-acetylgalactosamine; GlcA, glucuronic acid; IdoA, iduronic acid; HFL-1, human lung fibroblasts; HS, heparan sulfate; PAPS, 3′-phosphoadenosine 5′-phosphosulfate, PGs, proteoglycans.

**References**

Coster L, Hernnas J, Malmstrom A. 1991. Biosynthesis of dermatan sulphate proteoglycans. The effect of beta-D-xyloside addition on the polymer-modification process in fibroblast cultures. *Biochem J.* 276(Pt 2): 533–539.

Coster L, Malmstrom A, Sjoberg I, Fransson L. 1975. The co-polymeric structure of pig skin dermatan sulphate. Distribution of L-iduronic acid sulphate residues in co-polymeric chains. *Biochem J.* 145: 379–389.

Deepa SS, Umehara Y, Higashiyama S, Itoh N, Sugahara K. 2002. Specific molecular interactions of oversulfated chondroitin sulfate E with various heparin-binding growth factors. Implications as a physiological binding partner in the brain and other tissues. *J Biol Chem.* 277: 43707–43716.

Esko JD, Selleck SB. 2002. Order out of chaos: Assembly of ligand binding sites in heparan sulfate. *Annu Rev Biochem.* 71: 435–471.

Evers MR, Xia G, Kang H-G, Schachner M, Baenziger JU. 2001. Molecular cloning and characterization of a dermatan-specific N-acetylgalactosamine 4-O-sulfotransferase. *J Biol Chem.* 276: 36344–36353.

Fransson L-A, Roden L. 1967. Structure of dermatan sulfate: I. Degradation by testicular hyaluronidase. *J Biol Chem.* 242: 4161–4169.

Gingras A-C, Caballero M, Zarske M, Sanchez A, Hazbun TR, Fields S, Sonenberg N, Hafen E, Raught B, Aebersold R. 2005. A novel, evolutionarily conserved protein phosphatase complex involved in cisplatin sensitivity. *Mol Cell Proteomics.* 4: 1725–1740.

Greve H, Cully Z, Blumberg P, Kresse H. 1988. Influence of chlorate on proteoglycan biosynthesis by cultured human fibroblasts. *J Biol Chem.* 263: 12886–12892.

Hacker U, Nybakken K, Perrimon N. 2005. Heparan sulphate proteoglycans: The sweet side of development. *Nat Rev Mol Cell Biol.* 6: 530–541.

Halldorsdottir AM, Zhang L, Tollefsen DM. 2006. N-Acetylgalactosamine 4,6-O-sulfate residues mediate binding and activation of heparin cofactor II by porcine mucosal dermatan sulfate. *Glycobiology.* 16: 693–701.

Hannesson HH, Hagner-McWhirter A, Tiedemann K, Lindahl U, Malmstrom A. 1996. Biosynthesis of dermatan sulphate. Defructosylated *Escherichia coli* K4 capsular polysaccharide as a substrate for the D-glucuronoyl C-5 epimerase, and an indication of a two-base reaction mechanism. *Biochem J.* 313: 589–596.

Hiraoka N, Misra A, Belot F, Hindsgaul O, Fukuda M. 2001. Molecular cloning and expression of two distinct human N-acetylgalactosamine 4-O-sulfotransferases that transfer sulfate to GalNAcβ1 → 4GlcNAcβ1 → R in both N- and O-glycans. *Glycobiology.* 11: 495–504.

Iozzo RV. 1998. Matrix proteoglycans: From molecular design to cellular function. *Ann Rev Biochem.* 67: 609–652.

Kae T, Hiromi S, Hiroshi K, Kazuyuki S. 1998. Functional expression and genomic structure of human chondroitin 6-sulfotransferase1. The nucleotide sequence reported in this paper has been submitted to the DDBJ/GenBank/EBI data banks with accession number AB017915.1. *FEBS Lett.* 441: 235–241.

Karamanos NK, Vanky P, Syrokou A, Hjerpe A. 1995. Identity of dermatan and chondroitin sequences in dermatan sulfate chains determined by using fragmentation with chondroitinases and ion-pair high-performance liquid chromatography. *Anal Biochem.* 225: 220–230.

Kitagawa H, Fujita M, Ito N, Sugahara K. 2000. Molecular cloning and expression of a novel chondroitin 6-O-sulfotransferase. *J Biol Chem.* 275: 21075–21080.

Kluppel M, Wight TN, Chan C, Hinek A, Wrana JL. 2005. Maintenance of chondroitin sulfation balance by chondroitin-4-sulfotransferase 1 is required for chondrocyte development and growth factor signaling during cartilage morphogenesis. *Development.* 132: 3989–4003.

Kobayashi M, Sugumaran G, Liu J, Shworak NW, Silbert JE, Rosenberg RD. 1999. Molecular cloning and characterization of a human uronyl 2-sulfotransferase that sulfates iduronyl and glucuronyl residues in dermatan/chondroitin sulfate. *J Biol Chem.* 274: 10474–10480.

Lyon M, Deakin JA, Rahmoune H, Fernig DG, Nakamura T, Gallagher JT. 1998. Hepatocyte growth factor/scatter factor binds with high affinity to dermatan sulfate. *J Biol Chem.* 273: 271–278.

Maccarana M, Olander B, Malmstrom J, Tiedemann K, Aebersold R, Lindahl U, Li J-p, Malmstrom A. 2006. Biosynthesis of dermatan sulfate: Chondroitin-glucuronate C5-epimerase is identical to SART2. *J Biol Chem.* 281: 11560–11568.

Maimone MM, Tollefsen DM. 1990. Structure of a dermatan sulfate hexasaccharide that binds to heparin cofactor II with high affinity [published erratum appears in J Biol Chem 1991 Aug 5; 266(22): 14830]. *J Biol Chem.* 265: 18263–18271.

Malmstrom A. 1984. Biosynthesis of dermatan sulfate: II. Substrate specificity of the C-5 uronosyl epimerase. *J Biol Chem.* 259: 161–165.

Malmstrom A, Fransson LA. 1975. Biosynthesis of dermatan sulfate: I. Formation of L-iduronic acid residues. *J Biol Chem.* 250: 3419–3425.

Midura RJ, Salustri A, Calabro A, Yanagishita M, Hascall VC. 1994. High-resolution separation of disaccharide and oligosaccharide alditols from chondroitin sulfate, dermatan sulfate and hyaluronan using CarboPac PA1 chromatography. *Glycobiology.* 4: 333–342.

Mikami T, Mizumoto S, Kago N, Kitagawa H, Sugahara K. 2003. Specificities of three distinct human chondroitin/dermatan N-acetylgalactosamine 4-O-sulfotransferases demonstrated using partially desulfated dermatan sulfate as an acceptor: Implication of differential roles in dermatan sulfate biosynthesis. *J Biol Chem.* 278: 36115–36127.

Pacheco B, Maccarana M, Goodlett DR, Malmstrom A, Malmstrom L. 2009a. Identification of the active site of DS-epimerase 1 and requirement of N-glycosylation for enzyme function. *J Biol Chem.* 284: 1741–1747.

Pacheco B, Malmstrom A, Maccarana M. 2009b. Two dermatan sulfate epimerases form iduronic acid domains in dermatan sulfate. *J Biol Chem.* 284: 9788–9795.

Pinhal MAS, Smith B, Olson S, Aikawa J-i, Kimata K, Esko JD. 2001. Enzyme interactions in heparan sulfate biosynthesis: Uronosyl 5-epimerase and 2-O-sulfotransferase interact in vivo. *Proc Natl Acad Sci.* 98: 12984–12989.

Silbert JE, Palmer ME, Humphries DE, Silbert CK. 1986. Formation of dermatan sulfate by cultured human skin fibroblasts. Effects of sulfate concentration on proportions of dermatan/chondroitin. *J Biol Chem.* 261: 13397–13400.

Taylor KR, Rudisill JA, Gallo RL. 2005. Structural and sequence motifs in dermatan sulfate for promoting fibroblast growth factor-2 (FGF-2) and FGF-7 activity. *J Biol Chem.* 280: 5300–5306.

Tiedemann K, Larsson T, Heinegard D, Malmstrom A. 2001. The glucuronoyl C5-epimerase activity is the limiting factor in the dermatan sulfate biosynthesis. *Arch Biochem Biophys.* 391: 65–71.

Tiedemann K, Olander B, Eklund E, Todorova L, Bengtsson M, Maccarana M, Westergren-Thorsson G, Malmstrom A. 2005. Regulation of the chondroitin/dermatan fine structure by transforming growth factor-β1 through effects on polymer-modifying enzymes. *Glycobiology.* 15: 1277–1285.

Trowbridge JM, Gallo RL. 2002. Dermatan sulfate: New functions from an old glycosaminoglycan. *Glycobiology.* 12: 117R–125R.

Uyama T, Ishida M, Izumikawa T, Trybala E, Tufaro F, Bergstrom T, Sugahara K, Kitagawa H. 2006. Chondroitin 4-O-sulfotransferase-1 regulates E disaccharide expression of chondroitin sulfate required for herpes simplex virus infectivity. *J Biol Chem.* 281: 38668–38674.

Xu D, Song D, Pedersen LC, Liu J. 2007. Mutational study of heparan sulfate 2-O-sulfotransferase and chondroitin sulfate 2-O-sulfotransferase. *J Biol Chem.* 282: 8356–8367.

Copyright of Glycobiology is the property of Oxford University Press / UK and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.
