
OPEN ACCESS Freely available online

The QseC Adrenergic Signaling Cascade in Enterohemorrhagic *E. coli* (EHEC)

David T. Hughes${}^{1,2}$, Marcie B. Clarke${}^{1,2}$, Kaneyoshi Yamamoto${}^{3}$, David A. Rasko${}^{1,4*}$, Vanessa Sperandio${}^{1,2*}$

1 Department of Microbiology, University of Texas Southwestern Medical Center, Dallas, Texas, United States of America, 2 Department of Biochemistry, University of Texas Southwestern Medical Center, Dallas, Texas, United States of America, 3 Department of Agricultural Chemistry, Kinki University, Nakamachi, Nara, Japan, 4 Institute for Genome Sciences & Department of Microbiology and Immunology, University of Maryland School of Medicine, Baltimore, Maryland, United States of America

---

### Abstract

The ability to respond to stress is at the core of an organism’s survival. The hormones epinephrine and norepinephrine play a central role in stress responses in mammals, which require the synchronized interaction of the whole neuroendocrine system. Mammalian adrenergic receptors are G-coupled protein receptors (GPCRs); bacteria, however, sense these hormones through histidine sensor kinases (HKs). HKs autophosphorylate in response to signals and transfer this phosphate to response regulators (RRs). Two bacterial adrenergic receptors have been identified in EHEC, QseC and QseE, with QseE being downstream of QseC in this signaling cascade. Here we mapped the QseC signaling cascade in the deadly pathogen enterohemorrhagic *E. coli* (EHEC), which exploits this signaling system to promote disease. Through QseC, EHEC activates expression of metabolic, virulence and stress response genes, synchronizing the cell response to these stress hormones. Coordination of these responses is achieved by QseC phosphorylating three of the thirty-two EHEC RRs. The QseB RR, which is QseC’s cognate RR, activates the flagella regulon which controls bacteria motility and chemotaxis. The QseF RR, which is also phosphorylated by the QseE adrenergic sensor, coordinates expression of virulence genes involved in formation of lesions in the intestinal epithelia by EHEC, and the bacterial SOS stress response. The third RR, KdpE, controls potassium uptake, osmolarity, and also the formation of lesions in the intestine. Adrenergic regulation of bacterial gene expression shares several parallels with mammalian adrenergic signaling having profound effects in the whole organism. Understanding adrenergic regulation of a bacterial cell is a powerful approach for studying the underlying mechanisms of stress and cellular survival.

---

### Citation: Hughes DT, Clarke MB, Yamamoto K, Rasko DA, Sperandio V (2009) The QseC Adrenergic Signaling Cascade in Enterohemorrhagic *E. coli* (EHEC). PLoS Pathog 5(8): e1000553. doi:10.1371/journal.ppat.1000553

### Editor: C. Erec Stebbins, The Rockefeller University, United States of America

### Received March 25, 2009; Accepted July 23, 2009; Published August 21, 2009

### Copyright: © 2009 Hughes et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.

### Funding: This work was supported by NIH Grants AI053067 and the Burroughs Wellcome Fund. D.T.H. was supported through NIH Training Grant 5-T32-AI007520-07. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

### Competing Interests: The authors have declared that no competing interests exist.

* E-mail: drasko@som.umaryland.edu (DAR); vanessa.sperandio@utsouthwestern.edu (VS)

---

### Introduction

The survival of an organism lies within its intrinsic ability to detect and efficiently respond to stress cues. Stress responses play a key role in adaptation to environmental, psychosocial, and physical insults. Hence it comes as no surprise that stress responses require synchronization and coordination of an organism’s resources to ensure that metabolic substrates are available to meet the increasing energy demands of an effective stress response. Stress responses are generally termed “fight or flight” responses in higher animals, because they rely in the ability of an organism’s to assess whether its better chance of survival relies on facing or avoiding an environmental insult. The hormones epinephrine and norepinephrine are at the core of stress responses [1].

In mammalian cells epinephrine and norepinephrine are recognized by GPCRs, which are membrane receptors coupled to heterotrimeric guanine-binding proteins (G-proteins). These proteins consist of three subunits α, β and γ. The binding of these signals to GPCRs result in a conformational change that activates the G-protein through the exchange of GDP for GTP. The activated G-protein dissociates from the receptor, the α, β, and γ subunits then dissociate and activate their intracellular targets. The GPCR specificity is controlled by the type of G-protein associated with the receptor. G-proteins are divided in four families according to their association with effector proteins. Three of these signaling pathways, Gαs, Gαi and Gαq, have been extensively studied, with Gαs activating adenylate cyclase, Gαi inhibiting adenylate cyclase, and Gαq activating phospholipase C [1].

Most of the knowledge of epinephrine/norepinephrine-mediated signaling has been derived from studies in mammalian systems. However, although bacterial cells sense and respond to epinephrine and norepinephrine, the signaling pathways regulated by these mammalian hormones in bacteria have not been mapped [2,3]. Bacteria do not express homologues of mammalian adrenergic receptors. These signals are sensed through histidine sensor kinases (HKs) [4,5]. HKs constitute the predominant family of signaling proteins in bacteria. HKs usually act in concert with a response regulator (RR) protein constituting a two-component system. Upon sensing a defined environmental cue the HK autophosphorylates a conserved histidine residue, and then transfers this phosphate to an aspartate residue in the receiver domain of a cognate RR. The majority of the RRs are transcription factors, which are activated upon phosphorylation [6].

Two HKs, QseC and QseE, characterized in *E. coli* have been reported to sense epinephrine and norepinephrine [4,5]. QseC binds to and increases its autophosphorylation in response to epinephrine, norepinephrine, and a bacterial signaling molecule termed autoinducer-3 (AI-3) [4]. QseE increases its autopho-

Author Summary

Bacterial cells respond to the human stress hormones epinephrine (adrenaline) and norepinephrine (noradrenaline). These hormones are sensed by a bacterial receptor named QseC, which is a sensor kinase in the membrane that increases its autophosphorylation upon binding to these host signals. In addition to recognizing these signals, QseC also responds to a bacterial hormone-like molecule named autoinducer-3 (AI-3) that is produced by the human intestinal microbial flora. In this manuscript we have mapped genetically and biochemically the QseC signaling cascade in the deadly pathogen enterohemorrhagic *E. coli* (EHEC) O157:H7. EHEC uses this signaling system to activate expression of virulence genes. We show that the QseC signaling cascade is very complex so it can precisely modulate when different virulence traits are expressed. Because these sensor kinases are being evaluated as drug targets, a profound understanding of this signaling pathway is important for the development of novel therapeutic strategies to combat bacterial infections.

cells. Many of these effectors mimic mammalian signaling proteins having profound effects in the host cell signal transduction culminating in diarrheal disease [25]. Seven of these effectors are encoded within the LEE region [25], while many others are scattered throughout the genome [26, 27]. The first secreted effector discovered outside of the LEE was NleA [28]. NleA is known to inhibit cellular protein secretion by disrupting mammalian COPII function and mutation of the *nleA* gene resulted in attenuation in mouse model of infection [28, 29]. EHEC also produces a potent Shiga toxin (Stx) that is responsible for the major symptoms of hemorrhagic colitis and HUS [30].

Expression of LEE, Shiga toxin and the flagella and motility genes in EHEC are regulated by the signals AI-3, epinephrine and norepinephrine through QseC [4, 10]. This regulation is important for EHEC virulence, given that *qseC* mutants are attenuated for infection in animal models of disease [4, 10]. QseC activates transcription of the *flhDC* genes, which encode the master regulators of the flagellar regulon, directly through QseB binding to the *flhDC* promoter. Importantly, this interaction is dependent on QseB’s phosphorylation state [31], whereas, expression of the LEE and Shiga toxin genes are not regulated by QseB. Here we report a global analysis of EHEC gene expression in response to adrenergic signals, and map the QseC signaling cascade. In this study we unravel the adrenergic response of a bacterial cell at the genetic and biochemical levels, and demonstrate that adrenergic signaling has a profound effect on cell homeostasis, cell-to-cell signaling, and bacterial pathogenesis.

Results

Global assessment of QseC gene regulation in EHEC

We had previously reported that inactivation of the *qseC* gene results in reduced flagella expression and motility, and reduced auto-activation [8, 31]. To further characterize the role of QseC in EHEC, Affymetrix *E. coli* 2.0 microarrays were used to compare expression profiles of the WT and Δ*qseC* strains in the presence and absence of the signals AI-3 and epinephrine in Dulbecco’s modified eagle media (DMEM), which is optimal for expression of the LEE genes, and LB, which is optimal for expression of the flagella regulon. These arrays contain ~10,000 probe sets (array genes), covering all genes in the genomes of the two sequenced EHEC strains (EDL933 and Sakai), K-12 strain MG1655, uropathogenic *E. coli* (UPEC) strain CFT073, and 700 probes to intergenic regions (which can encode non-annotated small ORFs, or small regulatory RNAs). Expression data can be accessed using accession number (GSE15050) at the NCBI GEO database. During growth in LB, 126 probe sets were down-regulated (28 specific to EHEC), and 708 were up-regulated (232 EHEC specific) in the *qseC* mutant (Table 1). The majority of the genes with an altered profile were derived from the *E. coli* K-12 strain MG1655 (68%), which represent a common *E. coli* backbone conserved among all *E. coli* pathovars [32]. Many of these genes are associated with metabolism, and they also include the flagella regulon (Figure 1B and Figure 2D and 2E). The EHEC specific genes (32%) include several prophage-encoded genes and *stxAB* encoding Shiga toxin. These studies revealed that QseC not only activates transcription of the flagella regulon, but also of the genes encoding Shiga toxin.

Transcriptome comparisons between WT and the *qseC* mutant grown in DMEM, a condition conducive to LEE and virulence gene expression, in the presence of AI-3 alone (both WT and the *qseC* mutant produce AI-3 when grown to late exponential phase in DMEM) or AI-3 plus epinephrine also revealed a global role for QseC regulation of virulence genes (Table 2). In the presence of

The Adrenergic Signaling Cascade in Bacteria

Table 1. Comparison of 86-24 (WT) and the qseC mutant under different growth conditions.

| Increased | Marginal Increased | Decreased | Marginal Decreased | No Change |
|-----------|--------------------|-----------|--------------------|-----------|
| qseC-LB   | 708                | 130       | 126                | 9132      |
| qseC-AI-3 | 106                | 562       | 273                | 9061      |
| qseC-Epi  | 70                 | 432       | 311                | 9171      |

Increased and decreased are at least two fold changes in the expression levels. Marginally increased or decreased are changes that are either less than two fold or designated as "marginally increased or decreased" by the Affymetrix analysis software GCOSv1.4. Comparisons of qseC-AI3 to WT-AI3 were performed from RNA harvested from strains grown in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3). Comparisons of qseC-Epi to WT-Epi were performed from RNA harvested from strains grown in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3) with 10 μM epinephrine. doi:10.1371/journal.ppat.1000553.t001

AI-3 alone, expression of 106 genes was increased and 273 decreased in the qseC mutant compared to WT. In the presence of AI-3 plus epinephrine expression of 70 genes was increased and 311 decreased in the qseC mutant compared to WT. AI-3 and epinephrine have been reported to act as agonistic signals [33]. This agonistic relationship in signaling can be further illustrated by

Figure 1. QseC regulates multiple virulence factors. (A) Schematic representation of QseC responding to epinephrine/norepinephrine and AI-3 and regulating multiple virulence factors (B) Heat maps from microarray analysis representing the effects of epinephrine and AI-3 on WT EHEC and ΔqseC, differential regulation of the LEE genes, the flagellar genes and the non-LEE encoded secreted effectors are shown. Both WT and ΔqseC produce AI-3 in DMEM (OD<sub>600</sub> 1.0), hence these data reflect the transcriptome in the presence of AI-3 alone (WT+AI-3 and ΔqseC+AI-3) or AI-3 plus epinephrine (WT+Epi, ΔqseC+Epi) (C) QPCR of ler in wt EHEC, ΔqseC, and ΔqseC complement strain grown in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3) (D) QPCR of stx2a and recA in wt EHEC, ΔqseC, and ΔqseC complement strain grown in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3) (E) QPCR of nleA in WT EHEC and ΔqseC in LB and DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3) (F) QPCR of nleA in wt EHEC, ΔqseC, and ΔqseC complement strain grown in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3). doi:10.1371/journal.ppat.1000553.g001

The Adrenergic Signaling Cascade in Bacteria

**Figure 2. ΔqseC and ΔqseB do not have the same phenotype.** (A) Schematic representation of QseC responding to the signals epinephrine/ norepinephrine and AI-3 and transferring its phosphate onto its cognate response regulator QseB. (B) Motility plate of wt EHEC, ΔqseB, and the ΔqseB complement strain (complemented with plasmid pVS178, qseBC in pBAD33 [35]) (in the presence of self-produced AI-3) (C) Western blot of FliC in wt EHEC, ΔqseB, and the ΔqseB complement strain (complemented with plasmid pVS178, qseBC in pBAD33) (in the presence of self-produced AI-3) (D) QPCR of flhD in wt EHEC, ΔqseC, and ΔqseB in LB (OD<sub>600</sub> 1.0) (in the presence of self-produced AI-3) (E) β-galactosidase assay of the flhDC promoter controlling lacZ expression in wt EHEC, ΔqseC, the ΔqseC complement strain, ΔqseB, and the ΔqseB complement strain (complemented with plasmid pVS178, qseBC in pBAD33) in LB (OD<sub>600</sub> 1.0) (in the presence of self-produced AI-3).

doi:10.1371/journal.ppat.1000553.g002

the observation that while AI-3 is only sensed through QseC, epinephrine is sensed by both QseC and QseE [4,5]. However, it is worth mentioning that QseC acts upstream of QseE, given that transcription of qseE is activated by QseC [7]. These data suggest that both signals tend to activate global gene expression in a qseC-dependent fashion more frequently than repress expression. Among the genes activated in a qseC-dependent manner are the LEE (through activation of ler transcription, within the LEE1 operon, encoding the Ler activator of all other LEE genes) and stxAB (Shiga toxin) genes (Figure 1B, 1C and 1D). The genes encoding Stx are located within the late genes of a λ-bacteriophage and are transcribed when the phage enters its lytic cycle upon induction of an SOS response in the bacterial cell [34]. Upon the induction of an SOS response, recA is upregulated and cleaves the λ cI repressor allowing transcription of the middle and late genes to proceed, and together with them the stxAB genes. QseC-induction of stxAB transcription occurs through induction of recA expression (Figure 1D), suggesting that QseC mediates SOS induction in bacterial cells. In addition to activating expression of the LEE-encoded TTSS, the majority of the genes encoding effectors translocated through this TTSS are also regulated by QseC (Figure 1B). Of note, transcription of the gene encoding the NleA effector is strongly repressed by QseC in LB, while its expression is slightly (non-statistically significant) decreased in the

**The QseC signaling transduction pathway**

Through QseC, EHEC senses AI-3, epinephrine and norepinephrine to activate flagella and motility, AE lesion formation and Shiga toxin expression. Given that these are expensive biological processes that have to occur in concert, the kinetics of expression of these genes has to be exquisitely fine-tuned. We have previously reported that a ΔqseC EHEC had reduced motility, expressed less flagella, and presented reduced transcription of the flagella regulon [35]. The cognate RR of the QseC HK is QseB, which is phosphorylated at a conserved aspartate residue by QseC [4] (Figure 2A). In this study we deleted the cognate response regulator qseB. Since we had previously shown that QseC regulated the flagellar genes through a direct interaction of QseB and the flhDC promoter (FlhDC are the master activators of the flagella regulon) [31], we hypothesized that mutation of qseB would result in decreased motility. However, a ΔqseB mutant has no

Table 2. Pathovar distribution under different growth conditions.

|           | MG1655 | EDL933 | Sakai | CFT073 | Intergenic |
|-----------|--------|--------|-------|--------|------------|
| **qseC-LB** |        |        |       |        |            |
| Decreased  | 56     | 25     | 3     | 34     | 3          |
| Marg_decreased | 45     | 35     | 2     | 24     | 3          |
| Increased  | 266    | 194    | 38    | 121    | 83         |
| Marg_increased | 56     | 37     | 7     | 14     | 16         |
| No Change  | 3647   | 1496   | 323   | 2293   | 1192       |
| **qseC-DMEM-Epi** |        |        |       |        |            |
| Decreased  | 75     | 144    | 23    | 44     | 25         |
| Marg_decreased | 115    | 62     | 10    | 12     | 5          |
| Increased  | 36     | 7      | 5     | 14     | 2          |
| Marg_increased | 239    | 62     | 14    | 65     | 28         |
| No Change  | 3605   | 1512   | 321   | 2351   | 1237       |
| **qseC-DMEM-AI3** |        |        |       |        |            |
| Decreased  | 118    | 109    | 13    | 22     | 11         |
| Marg_decreased | 135    | 42     | 8     | 6      | 15         |
| Increased  | 46     | 18     | 5     | 23     | 13         |
| Marg_increased | 245    | 156    | 30    | 59     | 49         |
| No Change  | 3526   | 1462   | 317   | 2376   | 1209       |

Increased and decreased are at least two fold changes in the expression levels. Marginally increased or decreased are changes that are either less than two fold or designated as "marginally increased or decreased" by the Affymetrix analysis software GCOSv1.4.

Comparisons of qseC-AI3 to WT-AI3 were performed from RNA harvested from strains grown in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3). Comparisons of qseC-Epi to WT-Epi were performed from RNA harvested from strains grown in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3) with 10 μM epinephrine.
doi:10.1371/journal.ppat.1000553.t002

QseB can act as a repressor of the flagellar gene expression (Figure 3A). We also complemented the ΔqseB strain with a plasmid expressing QseB, and observed that the complemented strain had decreased motility; again suggesting that overabundance of unphosphorylated QseB has a repressive role in motility (Figure 3B). However, when we complemented the ΔqseB strain with a plasmid expressing qseBC (Figure 2), we did not observe any differences in motility, probably because the levels of QseB and QseC were balanced in this strain. Next, we overexpressed qseB, in a strain containing the -900 to +50bp region of the flhDC promoter upstream of a promoterless lacZ. We found that in the strain overexpressing qseB there was a five-fold decrease in β-galactosidase activity (Figure 3C). We also observed decreased flhDC transcription in a strain overexpressing a QseB site-directed mutant (QseB D51A) that cannot be phosphorylated (the conserved aspartate phosphorylated residue has been changed to an alanine) (Figure 3C), further indicating that an abundance of unphosphorylated QseB represses expression of flhDC.

We had previously shown that QseB can bind to two regions of the flhDC promoter, -300 to +50bp and -900 to -650bp [31]. We demonstrated that this binding required QseB to be phosphorylated [31] (Figure 3C), which can be achieved by providing a small phosphate donor, acetyl phosphate, to QseB in vitro. QseB will only bind to the -300 to +50bp flhDC region in the presence of acetyl phosphate (Figure 3D), and the QseBD51A mutant is also unable to bind to this region of flhDC (Figure 3D). We have discovered a new QseB binding site in the flhDC promoter from -650 to -300bp to which QseB can bind in the absence of phosphorylation. QseB binds to this -650 to -300bp site in the absence of acetyl phosphate, and QseBD51A can also bind to this site (Figure 3E and 3F). The presence of this new binding site provides further evidence for a dual role of QseB in the regulation of the flhDC promoter. At low signal concentration there is low QseC activation and thus low QseB phosphorylation. In this case only the -650 to -300bp site of the flhDC promoter will be occupied by non-phosphorylated-QseB and this binding may lead to repression. When the signal is high the opposite is true. The -300 to +50bp and -900 to -650bp sites will be occupied by phosphorylated QseB and flhDC will be activated (Figure 3H). In further support of this model, a nested deletion analyses of the flhDC promoter fused to lacZ shows that the full length fusion (-900 to +50bp) is activated by QseC (Figure 3G). This fusion contains all three QseB binding sites, and in the presence of QseC, phosphorylated QseB will occupy the activating sites from -950 to -650bp and -300 to +50 bp, increasing transcription. In the -650 to +50bp fusion, transcription of flhDC is repressed in the absence or presence of QseC, probably because of non-phosphorylated QseB binding to the -650 to -300bp site, which represses flhDC transcription. Non-phospho-QseB binding to the -650 to -300bp region is probably “locked” in the absence of the upstream (-900 to -650) site. When both upstream sites are removed (-300 to +50bp fusion), phospho-QseB bound to this proximal site will activate flhDC transcription (Figure 3F). In the complete absence of QseB, as in a qseB null strain, there will be QseC-independent expression of flhDC transcription, without any repression or activation (de-repression) by QseB (Figure 2). These data indicate that regulation of flhDC transcription by QseC occurs through its cognate RR QseB, and that QseB plays a dual role in this regulation according to its phosphorylated state.

QseB, however, does not seem to play a role in QseC-dependent activation of LEE and stxAB transcription (Figure 4), suggesting that this regulation may occur through phosphorylation of other RRs. In addition to QseB there are at least 31 other RR in E. coli that could be activated via QseC [9]. There is minimal

The Adrenergic Signaling Cascade in Bacteria

**Figure 3. QseB differentially regulates flhDC based on its phosphorylation state.**

(A) Motility plate of wt EHEC, ΔqseC, and ΔqseC overexpressing qseB (in the presence of self-produced AI-3)

(B) Motility plate of wt EHEC, ΔqseB, and ΔqseB overexpressing qseB (in the presence of self-produced AI-3)

(C) β-galactosidase assay of the flhDC promoter controlling lacZ expression in wt EHEC and in wt EHEC overexpressing qseB and the QseB D51A mutant in LB (OD<sub>600</sub> 1.0) (in the presence of self-produced AI-3)

(D) EMSA of the –300 bp to +50 bp region of the flhDC promoter with QseB and the phosphor-donor acetyl phosphate, QseB, and the nonphosphorylatable QseB D51A with the phosphor-donor acetyl phosphate

(E) EMSA of the –50 bp to +50 bp region of the flhDC promoter with QseB in the presence and absence of the phosphor-donor acetyl phosphate

(F) EMSA of the –650 bp to +50 bp region of the flhDC promoter with the nonphosphorylatable QseB D51A

(G) Nested deletion β-galactosidase analysis of the flhDC promoter (–900 bp to +50 bp, –650 bp to +50 bp, and –300 bp to +50 bp) controlling lacZ expression in wt EHEC, ΔqseC, the ΔqseC complement strain in LB (OD<sub>600</sub> 1.0) (in the presence of self-produced AI-3)

(H) QseB binding sites on the flhDC promoter.

doi:10.1371/journal.ppat.1000553.g003

cross-talk (cross-phosphorylation) between different two-component systems ensuring faithful transmission of information through distinct signaling pathways [36,37]. Indeed, the incidence of cross-phosphorylation between non-cognate HKs and RRs is low in *E. coli*. Yamamoto et al. showed that phosphorylation of non-cognate response regulators by HKs is rare and occurs in only 22 of 692 possible combinations [9]. However, in this same study, Yamamoto noticed that a distinct few HKs are more prone to also signal through non-congante RRs.

We have previously reported that QseC autophosphorylates in response to AI-3, epinephrine and norepinephrine in an *in vitro* liposome assay and can phototransfer onto its cognate RR, QseB [4]. In order to test QseC’s ability to phototransfer onto non-cognate RRs, we purified 31 *E. coli* RRs and performed phototransfer assays with QseC in liposomes. Of note all of these RRs were soluble and correctly folded upon purification, and have been previously shown by Yamamoto et al. to be active in phototransfer reactions with their cognate HKs [9]. Through this assay, we found only two additional QseC phosphorylation targets: KdpE and QseF (Table 3, Figure 5A and 5B). KdpE has been shown to regulate potassium uptake and medium osmolarity [38]. We found that *kdpA*, one of the genes regulated by KdpE, is also down-regulated in the ΔqseC (Figure 6A), indicating that cross-

**Figure 4. qPCR of ler and stx2a in wt EHEC and ΔqseB in DMEM (OD<sub>600</sub> 1.0) (in the presence of self-produced AI-3).**

doi:10.1371/journal.ppat.1000553.g004

PLoS Pathogens | www.plospathogens.org 6 August 2009 | Volume 5 | Issue 8 | e1000553

Table 3. QseC response regulator phosphotransfer table.

| Response regulator | Phosphorylated by QseC |
|--------------------|-----------------------|
| ArcA               | No                    |
| AtoC               | No                    |
| BaeR               | No                    |
| BasR               | No                    |
| CheB               | No                    |
| CheY               | No                    |
| CitB               | No                    |
| CpxR               | No                    |
| CreB               | No                    |
| CusR               | No                    |
| DcuR               | No                    |
| EvgA               | No                    |
| HydG               | No                    |
| KdpE               | Yes                   |
| NarL               | No                    |
| NarP               | No                    |
| NtrC               | No                    |
| PhoB               | No                    |
| PhoP               | No                    |
| QseB               | Yes                   |
| QseF               | Yes                   |
| RcsB               | No                    |
| RssB               | No                    |
| RstA               | No                    |
| TorR               | No                    |
| UhpA               | No                    |
| UvrY               | No                    |
| YedW               | No                    |
| YehT               | No                    |
| YfhK               | No                    |
| YhjB               | No                    |
| YpdB               | No                    |
| UvrY               | No                    |

Phosphotranfer studies were performed with purified QseC inserted in a liposome in the presence of 50 μM epinephrine and each purified response regulator. Yes indicates that QseC phosphotransfers to that response regulator (positive responses are bolded), No indicates the absence of phosphotransfer.
doi:10.1371/journal.ppat.1000553.t003

The Adrenergic Signaling Cascade in Bacteria

polymerization for AE lesion formation [39, 40]. QseF, however is not involved in regulation of LEE gene expression (Figure 6E) [7], nor in flagella and motility regulation [7]. However, a qseF knockout presented diminished expression of the stx gene (Figure 6E), suggesting that QseC activation of Shiga toxin expression occurs through the QseF RR. The QseF cognate HK is QseE [9], which is a second bacterial adrenergic receptor that senses epinephrine, phosphate and sulfate [11]. The addition of epinephrine to EHEC activates expression of qseEF, and this regulation is eliminated in the ΔqseC mutant, indicating that QseC activates transcription of qseEF [7]. Transcriptional regulation of qseEF by QseC, in addition to cross-phosphorylation of QseF by QseC and QseE may fine tune the timing for switching from motility, to AE lesion formation to Shiga toxin production during infection.

QseC phosphorylates three RRs: QseB, KdpE and QseF (Figure 7A). Through QseB the flagella regulon is regulated. KdpE activates expression of ler, and consequently of all LEE genes. QseF plays a role in inducing an SOS response and Shiga toxin production, as well as activating expression of espFu [7], which encodes an effector essential for AE lesion formation. To search globally which sets of QseC-dependent genes are regulated through each RR we performed transcriptome assays (GEO series GSE15050). These comparisons were performed with gene arrays hybridized with cDNA from RNA extracted from WT, ΔqseC, ΔqseB, ΔkdpE and ΔqseF strains grown in DMEM to an OD₆₀₀ of 1.0, conditions known to yield maximal endogenous AI-3 production in these strains [41]. Given that AI-3 is only sensed through QseC, and QseC will phosphorylate in the presence of either AI-3 or epinephrine [4, 11], by working under these conditions we would detect only QseC-dependent genes. We avoided using epinephrine in these comparisons, because epinephrine is also sensed by the QseE HK [4, 11]. Transcription of 324 genes was increased, and 344 decreased in the ΔqseC mutant compared to WT (Figure 7B). Of the 324 genes increased in the ΔqseC, 15 were also increased in ΔqseB, 13 in ΔqseF, and 63 in ΔkdpE (Figure 7B). These data suggest that 91 of these 324 genes repressed by QseC are under the control of the QseB, KdpE and QseF RRs. These leaves 233 genes repressed through QseC unaccounted for. A possible explanation could be that these genes may be activated and repressed by QseB in a similar fashion to flhDC (Figure 3), and these genes would not appear as transcriptionally regulated through QseB using gene arrays. QseC activates transcription of 344 genes, with 205 being activated through QseB, 44 through QseF and 87 through KdpE (Figure 7B). These three RRs activate transcription of 336 of the 344 QseC-dependent genes, giving almost 100% coverage of QseC-activated genes.

Discussion

Chemical signaling between cells underlies the basis of multicellularity. Although bacteria are unicellular, bacterial populations also utilize chemical signaling, through hormone-like compounds named autoinducers, to achieve cell-cell communication and coordination of behavior [42]. Chemical signaling is also essential for an organism to survive, successfully adapt to ever changing environments and protect themselves from insults, which can be collectively considered stress. Successful stress responses require energy input, and the coordination of many complex signaling pathways within the cell. Co-evolution of prokaryotic species and their respective eukaryotic host have exposed bacteria to hormones and eukaryotic cells to autoinducers. Therefore, it is not surprising that bacteria can respond to host hormones, and

The Adrenergic Signaling Cascade in Bacteria

**Figure 5. QseC phosphotransfers to the response regulators QseB, QseF and KdpE. (A) Autoradiograph of QseC autophosphorylation in lipid vesicles in the presence of 10 μM epinephrine and phosphotransfer onto QseB and KdpE (B) Autoradiograph of QseC autophosphorylation in lipid vesicles in the presence of 10 μM epinephrine and phosphotransfer onto QseB and QseF.**

doi:10.1371/journal.ppat.1000553.g005

that some pathogenic species have high-jacked these signaling systems to promote disease states [43].

One example of a pathogen that senses host hormones to regulate virulence is EHEC [2]. Upon reaching the human colon, EHEC senses the autoinducer-3 (AI-3) produced by the microbial gastrointestinal flora, and epinephrine and norepinephrine produced by the host through the HK QseC [2,4]. This signal transduction activates transcription of virulence genes in a coordinated fashion leading to the formation of AE lesions on intestinal cells by the locus of enterocyte effacement (LEE) genes,

the flagella regulon for enhanced motility, and Shiga toxin production which is responsible for HUS. EHEC probably first encounters the AI-3 signal produced by the microbial flora that inhabits the intestinal lumen [2]. Because the infectious dose of EHEC is very low (estimated to be 50 CFUs) [21], it is unlikely that it responds to self-produced signal to initiate infection. Upon sensing AI-3, QseC initiates the signaling cascade that will activate the flagella regulon leading to swimming motility, which may aid EHEC to come closer to the intestinal epithelial layer. As EHEC approaches the epithelium and starts forming AE lesions it is

**Figure 6. QseC, KdpE and QseF regulatory targets. (A) QPCR of kdpA in wt EHEC, ΔqseC, and ΔqseC complement strain in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3) (B) QPCR of flhD in wt EHEC, ΔqseC, and ΔkdpE in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3) (C) Motility plate of wt EHEC, ΔqseB, ΔkdpE, and ΔqseC (D) QPCR of ler and stx2a in wt EHEC and ΔkdpE in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3) (E) QPCR of ler and stx2a in wt EHEC and ΔqseF in DMEM (OD<sub>600</sub> 1.0) (in the presence of self produced AI-3).**

doi:10.1371/journal.ppat.1000553.g006

PLoS Pathogens | www.plospathogens.org August 2009 | Volume 5 | Issue 8 | e1000553

The Adrenergic Signaling Cascade in Bacteria

A

B

INCREASE

DECREASE

qseC vs qseB

309

15

43

139

205

548

qseC vs qseF

311

13

421

300

44

690

qseC vs kdpE

261

63

1002

257

87

688

Figure 7. Regulatory overlap of QseC and its phosphorylation targets. (A) Schematic representation of QseC responding to the signals epinephrine/norepinephrine and AI-3 and transferring its phosphate onto QseB, KdpE, and QseF (B) Microarray analysis comparing ΔqseC to ΔqseB, ΔkdpE, and ΔqseF in DMEM (OD<sub>600</sub> 1.0) (in the presence of self-produced AI-3).

doi:10.1371/journal.ppat.1000553.g007

probably then exposed to epinephrine and/or norepinephrine. Norepinephrine is synthesized within the adrenergic neurons of the enteric nervous system (ENS) that innervates the basolateral layer of the intestine [16]. Epinephrine is synthesized in the central nervous system (CNS) and in the adrenal medulla; it acts systemically after being released into the bloodstream, when it can reach the intestine [17]. AE lesion formation and the commencement of bloody diarrhea may increase EHEC exposure to epinephrine and norepinephrine, further upregulating expression of virulence genes in EHEC. This coordinated regulation involves a number of two-component regulatory systems composed of HKs and RRs that result in cascades of gene expression.

Recognition of AI-3/epinephrine/NE by QseC can be specifically blocked by the administration of the α-adrenergic antagonist phentolamine [4], and a synthetic compound called LED209 [10]. Using two different rabbit infection models, it has been demonstrated that QseC plays an important role in pathogenesis *in vivo*, since *qseC* mutants were attenuated for virulence in these animals [4,10]. Recently, a novel two-component system, the QseEF system [7], where QseE is the HK and QseF is the RR was shown to also regulate virulence in EHEC. QseE can also respond to the host hormone epinephrine like QseC, but in contrast, does not sense the bacterial signal AI-3. QseE is downstream from QseC in this signaling cascade, given that *qseEF* transcription is activated by epinephrine via QseC. The QseEF system is not involved in regulation of flagella and motility, but plays an important role in activating genes necessary for AE lesion formation [7] and also activates expression of Shiga toxin (Figure 6).

The AI-3/epinephrine/NE signaling system is not restricted to EHEC. *In silico* analysis showed homologues of QseC in other bacterial species such as *Salmonella* sp., *Shigella flexneri*, *Francisella tularensis*, *Haemophilus influenzae*, *Erwinia carotovora*, and many others [10]. *In vivo* studies provided evidence that the QseC HK is important in *Salmonella typhimurium* [10,44] and *Francisella tularensis* [45] pathogenesis, since *qseC* mutants of these strains are attenuated

in animal models of infection and *in vivo* inhibition of QseC by LED209 results in attenuation of infection by these organisms [10].

Because QseC is central for sensing adrenergic signals, and the effect these signals have in basic biological processes, a complete understanding of the QseC signaling transduction pathway in bacteria will offer clues on how eukaryotic stress responses affect a prokaryotic cell. We demonstrate that QseC acts promiscuously through three RRs (Figures 5 and 7) to initiate a complex signaling cascade that affects both metabolism and pathogenesis (Figure 8). QseC controls the expression of all of these features, either directly or indirectly and must be considered to be at or near the top of the signaling cascade. The fact that more than one kinase can activate multiple response regulators suggests that there is a hierarchy of signaling, beginning with QseC. It is currently unclear if the regulation by the associated HK and RR overrides the signal employed by a non-cognate HK or if they work in synergy to amplify the initial signal. This additional level of control may be the fine-tuning that is observed in EHEC where the motility, formation of lesions and secretion of toxin must be exquisitely choreographed to have an effective infection occur.

An additional level of complexity included in this signaling cascade is that QseB binds to different sites in the target promoters according to its phosphorylation state (Figure 3). This allows further modulation of gene expression by the spatial arrangement of these sites in the regulatory region of genes, allowing the same RR to both repress and activate transcription of the same gene. In the non-activated form (non-phosphorylated) QseB forms an additional regulatory barrier to the expression of *flhDC*. Only under conditions where QseB is both phosphorylated and in sufficient concentration is there full activation of the flagella regulon. Thus this two-step process provides additional levels of control for this energetically expensive appendage. These types of mechanisms ensure that only under conditions which are favorable the resources are devoted to this response. The DNA binding domain of QseB shares similarities with the DNA binding domain

The Adrenergic Signaling Cascade in Bacteria

![Diagram](attachment:diagram.png)

Figure 8. Model of the QseC and QseE signaling cascades in EHEC.
doi:10.1371/journal.ppat.1000553.g008

of the OmpR RR, which also recognizes different sites on DNA according to its phosphorylation state [46,47].

Because epinephrine and norepinephrine exert a profound effect in the host physiology and immune system, the ability to sense these hormones by bacteria may facilitate gauging the fitness of the host. Inter-kingdom chemical signaling plays an important role in the relationships forged between bacteria and animals. Chemical communication within kingdoms has been studied for many decades, however, the interception of these languages between different kingdoms has been appreciated only more recently. As this field expands, more and more examples will be described, and many questions answered.

### Materials and Methods

#### Strains and plasmids
All bacterial strains and plasmids utilized in this study are listed in Table S1. *E. coli* strains were grown aerobically in LB or DMEM (Invitrogen) medium at 37°C unless otherwise stated. Antibiotics were added at the following concentrations: 100 μg ml⁻¹ ampicillin and 30 μg ml⁻¹ chloramphenicol.

#### Recombinant DNA techniques
Standard methods were used to perform plasmid purification, PCR, ligation, restriction digests, transformation and gel electrophoresis [48].

#### Isogenic mutant construction
Construction of isogenic *kdpE* (DH11) and *qseB* (MC474) mutants was carried out as previously described [49]. Briefly,

86-24 cells containing pKD46 were prepared for electroporation. A *kdpE* PCR product was generated using primers kdpEλRed-F and kdpEλRed-R (Table S2) and pKD3 as a template and PCR-purified (Qiagen). A *qseB* PCR product was generated using primers qseBλRed-F and qseBλRed-R (Table S2) and pKD3 as a template and PCR-purified (Qiagen). Electroporation of the PCR products into these cells was performed; cells were incubated at 22°C for 16 h in SOC, and plated on media containing 30 μg ml⁻¹ chloramphenicol overnight at 42°C. Resulting colonies were patched for chloramphenicol resistance and ampicillin sensitivity, and PCR-verified for the absence of the gene. The chloramphenicol cassette was then resolved from the mutants in order to create non-polar, isogenic *kdpE* and *qseB* mutants. Plasmid pCP20, encoding a resolvase, was electroporated into the mutant strains, and resulting colonies were patched for chloramphenicol sensitivity. Construction of *qseC* and *qseF* mutants has been previously published [7,35].

#### Site-directed mutagenesis
Site-directed mutagenesis was carried out using the Quick Change II site-directed mutagenesis kit (Stratagene). Mutagenesis PCR primers were constructed using the Primer X software (http://www.bioinformatics.org/primerx/) and are listed in Table 1 (qseBD51AF and qseBD51AR). The plasmid pVS154 was PCR amplified with the mutagenesis primers according to Stratagene’s PCR protocol, generating the plasmid pDH12 (86-24 *qseBD51A* in pBADMycHis). The PCR product was digested with DpnI for 3 h at 37°C in order to remove the template plasmid. After digestion, the PCR product was transformed into XL-1 Blue supercompetent cells (Stratagene) and plated on selective media.
The next day, plasmid DNA was isolated and sequenced to determine if the mutation was present.

### RNA extraction and real-time RT-PCR studies

Cultures were grown aerobically in LB medium at $37^\circ$C overnight, diluted 1:100 in LB or DMEM (in the presence of self-produced AI-3 and in the absence or presence of $10 \mu$M epinephrine) and grown aerobically at $37^\circ$C. $0.2\%$ arabinose was added to the media when induction was required. RNA from three biological replicate cultures of each strain was extracted at the late exponential growth phase ($OD_{600}$ of 1.0) using the RiboPure Bacteria RNA isolation kit (Ambion) according to the manufacturer's guidelines. The primers used in the real-time assays were designed using Primer Express v1.5 (Applied Biosystems) (Table S2). Real-time reverse transcription-PCR (RT-PCR) was performed in a one-step reaction using an ABI 7500 sequence detection system (Applied Biosystems). For each $20-\mu$l reaction mixture, $10 \mu$l $2 \times$ SYBR master mix, $0.1 \mu$l MultiScribe reverse transcriptase (Applied Biosystems), and $0.1 \mu$l RNase inhibitor (Applied Biosystems) were added. Amplification efficiency of each of the primer pairs was verified using standard curves of known RNA concentrations. Melting-curve analysis was used to ensure template specificity by heating products to $95^\circ$C for $15 \mathrm{~s}$, followed by cooling to $60^{\circ} \mathrm{C}$ and heating to $95^{\circ} \mathrm{C}$ while monitoring fluorescence. Once the amplification efficiency and template specificity were determined for each primer pair, relative quantification analysis was used to analyze the unknown samples using the following conditions for cDNA generation and amplification: 1 cycle at $48^{\circ} \mathrm{C}$ for $30 \mathrm{~min}$, 1 cycle at $95^{\circ} \mathrm{C}$ for $10 \mathrm{~min}$, and 40 cycles at $95^{\circ} \mathrm{C}$ for $15 \mathrm{~s}$ and $60^{\circ} \mathrm{C}$ for $1 \mathrm{~min}$. The $rpoA$ (RNA polymerase subunit A) gene was used as the endogenous control. Real-time RT-PCR primers for the LEE genes and $rpoA$ have been previously described [33].

#### Electrophoretic mobility shift assay (EMSA)

In order to study the binding of QseB to the $flhDC$ promoter EMSAs were performed using the purified QseB protein and the $flhDC$ promoter. DNA probes were then end-labeled with [$\gamma$-32P]-ATP (NEB) using T4 polynucleotide kinase using standard procedures [48]. End-labeled fragments were run on a 5% polyacrylamide gel, excised and purified using the Qiagen PCR purification kit. Electrophoretic mobility shift assays were performed by adding increasing amounts of purified QseB or QseBD51A protein ($0-20 \mu \mathrm{M}$) to end-labeled probe ($10 \mathrm{ng}$) in binding buffer [$500 \mu \mathrm{g} \mathrm{ml}^{-1}$ BSA (NEB), $50 \mathrm{ng} \mu \mathrm{l}^{-1}$ poly-dIdC, $60 \mathrm{mM}$ HEPES pH 7.5, $5 \mathrm{mM}$ EDTA, $3 \mathrm{mM}$ DTT, $300 \mathrm{mM}$ KCl, $25 \mathrm{mM}$ MgCl2] with or without $0.1 \mathrm{M}$ acetyl phosphate for $20 \mathrm{~min}$ at $4^{\circ} \mathrm{C}$. Immediately before loading, a 5% ficol solution was added to the mixtures. The reactions were electrophoresed for approximately $14 \mathrm{~h}$ at $65 \mathrm{~V}$ on a 5% polyacrylamide gel, dried and exposed to KODAK X-OMAT film.

#### Microarrays.

Microarrays and analysis were performed as previously described [50]. The GeneChip $E$. coli Genome 2.0 array system of the Affymetrix system was used to compare the gene expression in strain 86-24 to that in strains VS138, MC474, and DH11. The GeneChip $E$. coli Genome 2.0 array includes approximately 10,208 probe sets for all 20,366 genes present in the following four strains of $E$. coli: K-12 lab strain MG1655, uropathogenic strain CFT073, O157:H7 enterohemorrhagic strain EDL933, and O157:H7 enterohemorrhagic strain Sakai (http://www.affymetrix.com/products/arrays/specific/ecoli2.affx). The RNA-processing, labeling, hybridization, and slide-scanning procedures were preformed as described in the Affymetrix Gene Expression Technical Manual (http://www.affymetrix.com/support/technical/manual/expression_manual.affx).

The output from scanning a single replicate of the Affymetrix GeneChip $E$. coli Genome 2.0 array for each of the biological conditions was obtained using GCOS v1.4 according to the manufacturer’s instructions. Data were normalized using Robust Multiarray analysis at the RMAExpress website (http://rmaexpress.bmbolstad.com/). The resulting data were compared to determine features whose expression was increased or decreased in response to inactivation of the $qseC$, $qseB$, $qseF$ and $kdpE$ genes. Custom analysis scripts were written in Perl to complete multiple array analyses. The results of the array analyses were further confirmed using real-time RT-PCR as described. We note that the isolate used in these studies has not been sequenced and thus is not fully contained on the array and that differences in genome content are evident. Expression data can be accessed using accession number (GSE15050) at the NCBI GEO database.

#### Motility assays

Assays were performed as previously described [31]. Briefly, motility assays were performed at $37^{\circ} \mathrm{C}$ on $0.3 \%$ agar plates containing Tryptone media ($1 \%$ tryptone and $0.25 \% \mathrm{NaCl}$). The motility halos were measured at $4 \mathrm{~h}$ and $8 \mathrm{~h}$.

#### Protein purification

One liter of LB media was inoculated at 1:100 and grown to O.D. 0.6 at $30^{\circ} \mathrm{C}$. The culture temperatures were reduced to $25^{\circ} \mathrm{C}$, induced with $400 \mu \mathrm{M}$ IPTG (Sigma) or $0.2 \%$ arabinose, and grown for either $3 \mathrm{~h}$ or $18 \mathrm{~h}$. Cells were harvested, suspended in lysis buffer ($50 \mathrm{mM}$ phosphate buffer $\mathrm{pH} 8,300 \mathrm{mM} \mathrm{NaCl}$, and $20 \mathrm{mM}$ imidazole) and lysed by homogenization. The lysed cells were centrifuged and the lysates were loaded onto to a Ni$^{2+}$-NTA-agarose gravity column (Qiagen). The column was washed with lysis buffer and protein was eluted with elution buffer ($50 \mathrm{mM}$ phosphate buffer $\mathrm{pH} 8,300 \mathrm{mM} \mathrm{NaCl}, 250 \mathrm{mM}$ imidazole). Fractions containing purified protein were confirmed by SDS-PAGE and concentrated for further use.

#### Reconstitution of QseC-His into liposomes

Liposomes were reconstituted as described previously [4,51]. Briefly, $50 \mathrm{mg}$ of $E$. coli phospholipids ($20 \mathrm{mg} / \mathrm{ml}$ in chloroform; Avanti Polar Lipids) were evaporated and then dissolved into $5 \mathrm{ml}$ of potassium phosphate buffer containing $80 \mathrm{mg}$ of N-octyl-$\beta$-d-glucopyranoside. The solution was dialyzed overnight against potassium phosphate buffer. The resulting liposome suspension was subjected to freeze-thaw in liquid N2. Liposomes were then destabilized by the addition of $26.1 \mathrm{mg}$ of dodecylmaltoside, and $0.625 \mathrm{mg}$ of QseC-MycHis was added, followed by stirring at room temperature for $10 \mathrm{~min}$. Two hundred-sixty milligrams of Biobeads (Biorad) were then added to remove the detergent, and the resulting solution was allowed to incubate at $4^{\circ} \mathrm{C}$ for $16 \mathrm{~h}$. The supernatant was then incubated with fresh Biobeads for $1 \mathrm{~h}$ at $22^{\circ} \mathrm{C}$ the next day. The resulting liposomes containing reconstituted QseC-MycHis were frozen in liquid N2 and stored at $-80^{\circ} \mathrm{C}$ until used.

#### Autophosphorylation and phosphotransfer assays

Assays were performed as previously described [4]. Briefly, twenty microliters of the liposomes containing QseC-MycHis were adjusted to $10 \mathrm{mM} \mathrm{MgCl}_{2}$ and $1 \mathrm{mM}$ DTT, and $10 \mu \mathrm{M}$ epinephrine, frozen and thawed rapidly in liquid N2, and kept at room temperature for $1 \mathrm{~h}$ (this allows for the signals to be loaded within the liposomes). [$\gamma 32 \mathrm{P}] \mathrm{dATP}(0.625 \mu \mathrm{l})(110 \mathrm{TBq}$
mmol) was added to each reaction. To some reactions, 12.5 μg of response regulator was added. At each time point (0, 10, 30 min), 10 μl of SDS loading buffer (with 20% SDS, to completely denature the liposome) was added. For all experiments involving QseC alone, a time point of 10 min was used. The samples were run on SDS/PAGE without boiling and visualized via PhosphorImager. The bands were quantitated by using imagequant version 5.0 software (Amersham Pharmacia).

### β-galactosidase assays

Assays were performed as previously described [31]. Briefly, bacteria containing *lacZ* fusions were grown overnight at 37°C in LB containing the appropriate selective antibiotic. Cultures were diluted 1:100 and grown in LB, and when necessary supplemented with 0.2% arabinose, to an OD~600~ of 1.0 at 37°C. These cultures were then assayed for β-galactosidase activity using o-nitrophenyl-β-d-galactopyranoside (ONPG) as a substrate as described previously [52].

## References

1. Molina PE (2006) Endocrine Physiology: The McGraw Hill Companies Inc.
2. Sperandio V, Torres AG, Jarvis B, Nataro JP, Kaper JB (2003) Bacteria-host communication: the language of hormones. Proc Natl Acad Sci USA 100: 8951–8956.
3. Lyte M, Ernst S (1992) Catecholamine induced growth of gram negative bacteria. Life Sci 50: 203–212.
4. Clarke MB, Hughes DT, Zhu C, Boedeker EC, Sperandio V (2006) The QseC sensor kinase: a bacterial adrenergic receptor. Proc Natl Acad Sci USA 103: 10420–10425.
5. Reading NC, Rasko DA, Torres AG, Sperandio V (2009) The two-component system QseEF and the membrane protein QseG link adrenergic and stress sensing to bacterial pathogenesis. Proc Natl Acad Sci USA 106: 5889–5894.
6. Stock AM, Robinson VL, Goudreau PN (2000) Two-component signal transduction. Annu Rev Biochem 69: 183–215.
7. Reading NC, Torres AG, Kendall MM, Hughes DT, Yamamoto K, et al. (2007) A novel two-component signaling system that activates transcription of an enterohemorrhagic Escherichia coli effector involved in remodeling of host actin. J Bacteriol 189: 2468–2476.
8. Clarke MB, Sperandio V (2005) Transcriptional autoregulation by quorum sensing Escherichia coli regulators B and C (QseBC) in enterohaemorrhagic E. coli (EHEC). Mol Microbiol 58: 441–455.
9. Yamamoto K, Hirao K, Oshima T, Aiba H, Utsumi R, et al. (2005) Functional characterization in vitro of all two-component signal transduction systems from Escherichia coli. J Biol Chem 280: 1448–1456.
10. Rasko DA, Moreira CG, Lide R, Reading NC, Ritchie JM, et al. (2008) Targeting QseC signaling and virulence for antibiotic development. Science 321: 1078–1080.
11. Reading NC, Rasko D, Torres AG, Sperandio V (2008) The two-component system QseEF and the membrane protein QseG link adrenergic and stress sensing to bacterial pathogenesis. PNAS In Press.
12. Bearson BL, Bearson SM (2007) The role of the QseC quorum-sensing sensor kinase in colonization and norepinephrine-enhanced motility of Salmonella enterica serovar Typhimurium. Microb Pathog 44: 271–278.
13. Lyte M, Arulanandam BP, Frank CD (1996) Production of Shiga-like toxins by Escherichia coli O157:H7 can be influenced by the neuroendocrine hormone norepinephrine. J Lab Clin Med 128: 392–398.
14. Lyte M, Erickson AK, Arulanandam BP, Frank CD, Crawford MA, et al. (1997) Norepinephrine-induced expression of the K99 pilus adhesin of enterotoxigenic Escherichia coli. Biochem Biophys Res Commun 232: 682–686.
15. Bailey MT, Karaszewski JW, Lubach GR, Coe CL, Lyte M (1999) In vivo adaptation of attenuated Salmonella typhimurium results in increased growth upon exposure to norepinephrine. Physiol Behav 67: 359–364.
16. Furness JB (2000) Types of neurons in the enteric nervous system. Journal of the autonomic nervous system 81: 87–96.
17. Purves D, Fitzpatrick D, Williams SM, McNamara JO, Augustine GJ, et al. (2001) Neuroscience: Sinauer Associates, Inc.
18. Eldrup E, Richter EA (2000) DOPA, dopamine, and DOPAC concentrations in the rat gastrointestinal tract decrease during fasting. Am J Physiol Endocrinol Metab 279: E815–822.
19. Horger S, Schultheiss G, Diener M (1998) Segment-specific effects of epinephrine on ion transport in the colon of the rat. Am J Physiol 275: G1367–1376.
20. Freddolino PL, Kalani MY, Vaidihi N, Floriano WB, Hall SE, et al. (2004) Predicted 3D structure for the human beta 2 adrenergic receptor and its binding site for agonists and antagonists. Proc Natl Acad Sci USA 101: 2736–2741.
21. Kaper JB, Nataro JP, Mobley HL (2004) Pathogenic Escherichia coli. Nat Rev Microbiol 2: 123–140.

---

The Adrenergic Signaling Cascade in Bacteria

## Supporting Information

**Table S1** Strains and Plasmids  
Found at: doi:10.1371/journal.ppat.1000553.s001 (0.07 MB DOC)

**Table S2** Oligonucleotide Primers  
Found at: doi:10.1371/journal.ppat.1000553.s002 (0.04 MB DOC)

## Author Contributions

Conceived and designed the experiments: DTH MBC DR VS. Performed the experiments: DTH MBC DR VS. Analyzed the data: DTH MBC DR VS. Contributed reagents/materials/analysis tools: DTH KY DR VS. Wrote the paper: DTH DR VS.

---

22. McDaniel TK, Jarvis KG, Donnenberg MS, Kaper JB (1995) A genetic locus of enterocyte effacement conserved among diverse enterobacterial pathogens. Proc Natl Acad Sci USA 92: 1664–1668.
23. Mellies JL, Elliott SJ, Sperandio V, Donnenberg MS, Kaper JB (1999) The Per regulon of enteropathogenic Escherichia coli: identification of a regulatory cascade and a novel transcriptional activator, the locus of enterocyte effacement (LEE)-encoded regulator (Ler). Mol Microbiol 33: 296–306.
24. Jarvis KG, Giron JA, Jerse AE, McDaniel TK, Donnenberg MS, et al. (1995) Enteropathogenic Escherichia coli contains a putative type III secretion system necessary for the export of proteins involved in attaching and effacing lesion formation. Proc Natl Acad Sci USA 92: 7996–8000.
25. Gar mendia J, Frankel G, Crepin VF (2005) Enteropathogenic and enterohemorrhagic Escherichia coli infections: translocation, translocation, translocation. Infect Immun 73: 2573–2585.
26. Deng W, Puente JL, Gruenheid S, Li Y, Vallance BA, et al. (2004) Dissecting virulence: systematic and functional analyses of a pathogenicity island. Proc Natl Acad Sci USA 101: 3597–3602.
27. Tobe T, Beatson SA, Taniguchi H, Abe H, Bailey CM, et al. (2006) An extensive repertoire of type III secretion effectors in Escherichia coli O157 and the role of lambdoid phages in their dissemination. Proc Natl Acad Sci USA 103: 14941–14946.
28. Gruenheid S, Sekirov I, Thomas NA, Deng W, O'Donnell P, et al. (2004) Identification and characterization of NleA, a non-LEE-encoded type III translocated virulence factor of enterohaemorrhagic Escherichia coli O157:H7. Mol Microbiol 51: 1233–1249.
29. Mundy R, Petrovska L, Smollett K, Simpson N, Wilson RK, et al. (2004) Identification of a novel Citrobacter rodentium type III secreted protein, EspI, and roles of this and other secreted proteins in infection. Infect Immun 72: 2288–2302.
30. Karmali MA, Petric M, Lim C, Fleming PC, Steele BT (1983) Escherichia coli cytotoxin, haemolytic-uraemic syndrome, and haemorrhagic colitis. Lancet 2: 1299–1300.
31. Clarke MB, Sperandio V (2005) Transcriptional regulation of flhDC by QseBC and sigma (FliA) in enterohaemorrhagic Escherichia coli. Mol Microbiol 57: 1734–1749.
32. Rasko DA, Rosovitz MJ, Myers GS, Mongodin EF, Fricke WF, et al. (2008) The pangenome structure of Escherichia coli: comparative genomic analysis of E. coli commensal and pathogenic isolates. J Bacteriol 190: 6881–6893.
33. Walters M, Sperandio V (2006) Autoinducer 3 and epinephrine signaling in the kinetics of locus of enterocyte effacement gene expression in enterohemorrhagic Escherichia coli. Infect Immun 74: 5445–5455.
34. Neely MN, Friedman DI (1998) Functional and genetic analysis of regulatory regions of coliphage H-19B: location of shiga-like toxin and lysis genes suggest a role for phage functions in toxin release. Mol Microbiol 28: 1255–1267.
35. Sperandio V, Torres AG, Kaper JB (2002) Quorum sensing Escherichia coli regulators B and C (QseBC): a novel two-component regulatory system involved in the regulation of flagella and motility by quorum sensing in E. coli. Mol Microbiol 43: 809–821.
36. Laub MT, Goulian M (2007) Specificity in two-component signal transduction pathways. Annu Rev Genet 41: 121–145.
37. Skerker JM, Perchuk BS, Siryaporn A, Lubin EA, Ashenberg O, et al. (2008) Rewiring the specificity of two-component signal transduction systems. Cell 133: 1043–1054.
38. Nakashima K, Sugiura A, Momoi H, Mizuno T (1992) Phosphotransfer signal transduction between two regulatory factors involved in the osmoregulated kdp operon in Escherichia coli. Mol Microbiol 6: 1777–1784.

39. Garmendia J, Phillips AD, Carlier MF, Chong Y, Schuller S, et al. (2004) TccP is an enterohaemorrhagic Escherichia coli O157:H7 type III effector protein that couples Tir to the actin-cytoskeleton. Cell Microbiol 6: 1167–1183.

40. Campellone KG, Robbins D, Leong JM (2004) EspFU is a translocated EHEC effector that interacts with Tir and N-WASP and promotes Nck-independent actin assembly. Dev Cell 7: 217–228.

41. Walters M, Sircili MP, Sperandio V (2006) AI-3 synthesis is not dependent on luxS in Escherichia coli. J Bacteriol 188: 5668–5681.

42. Fuqua WC, Winans SC, Greenberg EP (1994) Quorum sensing in bacteria: the LuxR-LuxI family of cell density-responsive transcriptional regulators. J Bacteriol 176: 269–275.

43. Hughes DT, Sperandio V (2008) Inter-kingdom signaling: communication between bacteria and host. Nature Reviews Microbiology 6: 111–120.

44. Bearson BL, Bearson SM (2008) The role of the QseC quorum-sensing sensor kinase in colonization and norepinephrine-enhanced motility of Salmonella enterica serovar Typhimurium. Microb Pathog 44: 271–278.

45. Weiss DS, Brotcke A, Henry T, Margolis JJ, Chan K, et al. (2007) In vivo negative selection screen identifies genes required for Francisella virulence. Proc Natl Acad Sci USA 104: 6037–6042.

46. Harlocker SL, Bergstrom L, Inouye M (1995) Tandem binding of six OmpR proteins to the ompF upstream regulatory sequence of Escherichia coli. J Biol Chem 270: 26849–26856.

47. Bergstrom LC, Qin L, Harlocker SL, Egger LA, Inouye M (1998) Hierarchical and co-operative binding of OmpR to a fusion construct containing the ompC and ompF upstream regulatory sequences of Escherichia coli. Genes Cells 3: 777–788.

48. Sambrook J, Fritsch EF, Maniatis T (1989) Molecular cloning: a laboratory manual. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

49. Datsenko KA, Wanner BL (2000) One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci USA 97: 6640–6645.

50. Kendall MM, Rasko DA, Sperandio V (2007) Global effects of the cell-to-cell signaling molecules autoinducer-2, autoinducer-3, and epinephrine in a luxS mutant of enterohemorrhagic Escherichia coli. Infect Immun 75: 4875–4884.

51. Janausch IG, Garcia-Moreno I, Lehnert D, Zeuner Y, Unden G (2004) Phosphorylation and DNA binding of the regulator DcuR of the fumarate-responsive two-component system DcuSR of Escherichia coli. Microbiology 150: 877–883.

52. Miller JH (1972) Experiments in molecular genetics. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.
Copyright of PLoS Pathogens is the property of Public Library of Science and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.