
# THERAPEUTIC RESISTANCE

## Biological determinants of endocrine resistance in breast cancer

**Elizabeth A. Musgrove**\*† **and Robert L. Sutherland**\*‡

### Abstract | Endocrine therapies targeting oestrogen action (anti-oestrogens, such as tamoxifen, and aromatase inhibitors) decrease mortality from breast cancer, but their efficacy is limited by intrinsic and acquired therapeutic resistance. Candidate molecular biomarkers and gene expression signatures of tamoxifen response emphasize the importance of deregulation of proliferation and survival signalling in endocrine resistance. However, definition of the specific genetic lesions and molecular processes that determine clinical endocrine resistance is incomplete. The development of large-scale computational and genetic approaches offers the promise of identifying the mediators of endocrine resistance that may be exploited as potential therapeutic targets and biomarkers of response in the clinic.

---

#### Aromatase inhibitors
Drugs that function by blocking aromatase, the enzyme that converts androgens to oestrogens in tissues including the breast and adipose tissue. Examples include anastrazole, letrozole and exemestane.

#### ER-positive breast cancers
In current clinical practice, ER-positive breast cancers are those with immunohistochemically detectable ERα levels.

The steroid hormone oestrogen is central to normal female physiology, reproduction and behaviour, through its effects on cellular processes including cell proliferation and cell survival. These effects are mediated by nuclear oestrogen receptors (ERα and ERβ; BOX 1). ERα is responsible for many of the effects of oestrogen on normal and cancerous breast tissue, through ligand-activated transcriptional regulation (genomic actions) and by acting as a component of membrane and cytoplasmic signalling cascades (non-genomic actions)\(^{1}\) (FIG. 1).

Sustained exposure to endogenous or exogenous oestrogen is a well-established cause of breast cancer\(^{2,3}\), underpinning the use of anti-oestrogens and aromatase inhibitors in breast cancer prevention\(^{4-6}\). At least 70% of breast cancers are classified as ER-positive breast cancers\(^{7}\), and interfering with oestrogen action has been a mainstay of breast cancer treatment for more than a century. Early therapies included surgical removal of the ovaries, but the synthesis of competitive inhibitors of oestrogen–ER binding during the 1970s led to the first, and to date most successful, targeted cancer therapy: the selective oestrogen receptor modulator (SERM) tamoxifen\(^{8}\). Adjuvant therapy with tamoxifen almost halves the rate of disease recurrence and reduces the annual breast cancer death rate by one-third, making a significant contribution to the 25–30% decrease in breast cancer mortality in the past two decades\(^{9}\). Subsequently, other new, effective endocrine therapies have been developed that target oestrogen synthesis (such as aromatase inhibitors\(^{10}\)) or ER signalling (such as other SERMs and ‘pure’ anti-oestrogens\(^{11}\)).

One-third of women treated with tamoxifen for 5 years will have recurrent disease within 15 years\(^{9}\), and so endocrine-resistant disease may represent up to one-quarter of all breast cancers. Therefore, two major challenges for the successful treatment of breast cancer are the development of more specific biomarkers that predict therapeutic response to endocrine therapy and the identification of new therapeutic targets for endocrine-resistant disease. This Review summarizes and evaluates the recent insights into the mechanisms of endocrine resistance that have been made through candidate gene approaches, as well as more global gene expression profiling and functional genetic screens. We necessarily focus on tamoxifen resistance, as the experience with this drug is more extensive and the clinical data more mature than for other drugs. Many of the broad concepts discussed will probably also apply to resistance to aromatase inhibitors and other anti-oestrogens, although the lack of clinical cross-resistance\(^{10-12}\) indicates that some resistance mechanisms are independent.

### Molecular mechanisms of resistance
The primary mechanism of *de novo* or intrinsic resistance to tamoxifen is lack of expression of ERα. Recently, a second intrinsic mechanism has been documented in which patients carrying inactive alleles of cytochrome P450 2D6 (CYP2D6) (approximately 8% of Caucasian women) fail to convert tamoxifen to its active metabolite, endoxifen, and are consequently less responsive to tamoxifen\(^{13}\). By contrast, a plethora of mechanisms

REVIEWs

At a glance

- Endocrine therapies that target oestrogen action (anti-oestrogens and aromatase inhibitors) are widely used and successful breast cancer therapies, but many women treated with these therapies will relapse with endocrine-resistant disease.
- Mechanisms of endocrine resistance in oestrogen receptor (ER)-positive breast cancers include loss of ERα expression and expression of truncated isoforms of ERα and ERβ, post-translational modifications of ERα, increased AP1 activity and deregulation of ER co-activators, increased receptor tyrosine kinase signalling leading to the activation of the Erk and PI3K pathways, and deregulation of the cell cycle and apoptotic machinery.
- Gene expression signatures that are predictive of poor outcome in women treated with tamoxifen commonly contain ER target genes, as well as genes involved in proliferation, apoptosis, and invasion and metastasis. Many of these signatures are also predictive of outcome in women who have not been treated with tamoxifen and so are markers of intrinsic biology rather than specific to tamoxifen responsiveness.
- Gene expression signatures representing particular biological processes (for example, cell cycle progression, cell death and invasion) or pathways (for example, RB deregulation, MYC overexpression and E2f activation) can also predict outcome in women treated with tamoxifen and point towards possible mechanisms for endocrine resistance.
- Functional genetic screens have successfully identified several genes, the loss or overexpression of which can reduce anti-oestrogen sensitivity in cell lines and is associated with clinical endocrine resistance.
- Insights into the mechanisms of resistance have suggested possible therapeutic approaches for endocrine-resistant ER-positive breast cancer, for example tyrosine kinase inhibitors. Further potential therapeutic targets may emerge from combining large-scale genomic and transcriptomic data with large-scale functional analyses.

As detailed information on the biology of the many molecules implicated in tamoxifen resistance *in vitro* and the means by which they cause resistance is summarized in a series of excellent recent reviews ${}^{11,12,14-21}$, we focus here on recent developments that shed light on potential mechanisms of prognostic or predictive importance. As shown by the examples in TABLE 1, most of the molecules that modulate tamoxifen sensitivity in experimental models are correlated with disease outcome not only in women treated with tamoxifen but also in a wider population of patients with breast cancer. Furthermore, because adjuvant tamoxifen has been the ‘therapy of choice’ in ER-positive breast cancer for more than 25 years, most available patient cohorts do not allow the comparison of therapeutic responsiveness and outcome in well-matched control populations that differ only with respect to tamoxifen therapy. Therefore, it is difficult to distinguish specific differences in tumour response to tamoxifen from the broader effects of the underlying biology of the disease.

**ER and co-regulators.** Response to tamoxifen is rare in ER-negative breast cancer, and so ERα expression is currently the principal biomarker of response to endocrine therapy. Early studies implicated the loss of ERα expression or ERα mutations as potential mechanisms of acquired resistance. However, loss of ERα expression occurs in only a minority (15–20%) of resistant breast cancers ${}^{22}$ and $<$1% of ER-positive tumours have ERα mutations ${}^{14,20,23}$. More recently, expression of a new truncated variant of ERα, ERα36, in the presence of full-length ERα has been associated with reduced responsiveness ${}^{24}$. The development of antibodies that can distinguish between ERα, ERβ and naturally occurring ERβ variants (BOX 1) has led to the identification of responses in ERβ-positive but ERα-negative cancers and a potential role for the carboxy-terminally truncated variants of ERβ (ERβ2/cx and ERβ5) in tamoxifen responsiveness ${}^{25,26}$. In addition, the oestrogen-related receptor ERRγ is overexpressed and mediates tamoxifen resistance in lobular invasive breast cancer models ${}^{27}$.

One mechanism by which ERα regulates gene expression is through protein–protein interactions with other transcription factors — for example, activator protein 1 (AP1), specificity protein 1 (SP1) and nuclear factor-κB (NF-κB) (FIG. 1). Increased AP1 and NF-κB transcriptional activity are also associated with endocrine resistance ${}^{28-30}$. ERα function is regulated by post-translational modifications (phosphorylation, methylation and sumoylation) that influence interactions with other proteins, including transcriptional co-regulators ${}^{19}$ and cytoplasmic signalling molecules (FIG. 1). There is significant evidence to show that effects on these end points contribute to endocrine resistance ${}^{18,20,31}$. Overexpression and increased phosphorylation of ERα co-activators, particularly nuclear receptor co-activator 3 (NCOA3; also known as AIB1 or SRC3), leads to constitutive ERα-mediated transcription, which confers resistance *in vitro* and in xenograft models ${}^{12,16}$ and is associated with reduced responsiveness to tamoxifen in patients ${}^{32}$. Transient methylation of ERα at R260 by protein arginine N-methyltransferase 1 (PRMT1) results

Adjuvant therapy  
A drug treatment (for example, chemotherapy or endocrine therapy) that is given after the primary therapy (for example, surgery and/or radiotherapy), with the aim of increasing the overall effectiveness of treatment.

SERMs  
Drugs such as tamoxifen that bind the oestrogen receptor and thereby block the effects of oestrogen on tissues such as the breast but that function similarly to oestrogen in other tissues such as bone. Unlike oestrogen, these drugs are not steroidal in structure.

‘Pure’ anti-oestrogens  
Drugs that bind the oestrogen receptor, thereby blocking the effect of oestrogen, but have no detectable oestrogen-like effects. Most have a steroidal structure.

Intrinsic resistance  
The failure to respond to initial drug therapy.

in the formation of cytoplasmic complexes that contain ERα, PI3K, the tyrosine kinase SRC and focal adhesion kinase (FAK; also known as PTK2) and that activate Akt (FIG. 1). However, it is not known whether this methylation event, which is frequent in breast cancer³³, is associated with the endocrine response. Another example is the ER co-activator PELP1, which in many breast cancers localizes to the cytoplasm, where it can confer tamoxifen resistance³¹. PELP1 functions as a scaffold that modulates ER interaction with SRC, leading to activation of SRC and the Erk family kinases and also promotes oestrogen activation of PI3K³¹ (FIG. 1).

Receptor tyrosine kinase signalling. The bidirectional crosstalk between ER and receptor tyrosine kinase signalling is evidenced by the early observations of reciprocal expression of ER and members of the epidermal growth factor receptor (Egfr) family such as EGFR and ERBB2 (also known as HER2)³⁴. Growth factors of the Egf and insulin-like growth factor (Igf) families can modulate tamoxifen sensitivity *in vitro*³⁵; although breast cancer cells are quiescent and insensitive to growth factor stimulation following treatment with the pure anti-oestrogen ICI 182780 (fulvestrant), tamoxifen treatment does not lead to growth factor insensitivity³⁶. This has focused attention on receptor tyrosine kinase expression and function as potential mediators of endocrine resistance. Increased expression of EGFR, ERBB2 and IGF1 receptor (IGF1R) can elicit tamoxifen resistance³⁷⁻⁴⁰, as can activation of the components of their downstream signalling pathways, particularly the Erk and PI3K pathways⁴¹⁻⁴³. In some cases, deregulation of these signalling pathways occurs as a result of genetic or epigenetic modifications, such as amplification of ERBB2, activating mutations in PIK3CA, which encodes a catalytic subunit of type I PI3Ks, and loss of heterozygosity or methylation of PTEN, a tumour suppressor that inhibits the PI3K pathway²⁰,²¹. In other cases, however, deregulation of these pathways reflects aberrations in upstream regulators, such as the activation of Akt in association

with the loss of PTEN expression or overexpression of ERBB2 (REFS 20,21) and activation of IGF1R and ERBB3 following the loss of PTEN⁴⁰. How these events mediate tamoxifen resistance has not been fully elucidated, but several potential contributing factors have been suggested (FIGS 1,3): decreased ERα expression mediated by ERK activation; loss of ER-mediated repression of EGFR and ERBB2 and consequent activation of mitogenic signalling cascades; ligand-independent activation of ER or its co-activators through phosphorylation; upregulation of key cell cycle regulators, for example MYC and the D-type and E-type cyclins, through constitutive activation of mitogenic signalling pathways; and the inhibition of apoptosis through constitutive activation of survival signalling.

Overexpression of ERBB2 is one of the best-characterized mechanisms of endocrine resistance²¹. Recent evidence implicates the loss of transcriptional repressors and amplification of ERBB2 as mechanisms that are responsible for increased expression of this receptor. The X-linked tumour suppressor forkhead box P3 (FOXP3) and the zinc finger transcription factor GATA4 can repress ERBB2 expression, even in a cell line with an approximately tenfold amplification of ERBB2, and their expression is negatively correlated with ERBB2 expression in breast cancer⁴⁴,⁴⁵. In addition, a recent pivotal study showed that ERα-mediated repression of ERBB2 is dependent on competition between the paired-domain transcription factor PAX2 and the ERα co-activator NCOA3 for binding and regulation of ERBB2 transcription and, in turn, tamoxifen responsiveness⁴⁶. A direct relationship between FOXP3 or GATA4 expression and tamoxifen responsiveness has not been established. However, increased PAX2 expression and consequent repression of ERBB2 was associated with increased survival following tamoxifen treatment, and loss of PAX2 expression in the presence of increased NCOA3 expression predicted a poor outcome⁴⁶, indicating that this mechanism is of direct clinical relevance.

Members of the Src family of tyrosine kinases, particularly SRC itself, and their downstream targets are also commonly overexpressed in breast cancer and have been implicated in resistance. The Src substrate BCAR1 (also known as CAS) is a focal adhesion adaptor protein that activates proliferative, survival and invasion pathways. It can induce tamoxifen resistance when overexpressed *in vitro*⁴⁷, and BCAR1-overexpressing breast cancers are less responsive to tamoxifen⁴⁸. BCAR1 binds and activates SRC with consequent phosphorylation of the Src substrates EGFR and signal transducer and activator of transcription 5B (STAT5B) and effects on downstream signalling pathways²⁰. However, recent data suggest that the ability of BCAR1 to confer anti-oestrogen resistance may not require interaction with SRC⁴⁹. The putative guanine nucleotide-exchange factor BCAR3, which synergizes with BCAR1 to activate SRC⁵⁰, also causes tamoxifen resistance *in vitro*⁵¹. In addition, BCAR3 activates Rac and p21-activated kinase 1 (PAK1)⁵²; the latter is itself implicated in tamoxifen resistance through ERα phosphorylation⁵³.

Box 1 | Oestrogen receptors

The oestrogen receptors ERα and ERβ are encoded by separate genes located on different chromosomes. They have a similar overall domain structure, consisting of a central DNA-binding domain flanked by two autonomous transcriptional activation domains, one of which, AF-2, is positioned in the ligand-binding domain and is ligand dependent (reviewed in REF. 11). The AF-2 domain also mediates interactions with co-activators that increase ER transcriptional activity¹⁹,¹²⁷. Isoforms of both ERα and ERβ have been described, but the most relevant in the context of endocrine resistance are a truncated variant of ERα, ERα36 (REF. 24), and two isoforms of ERβ, ERβ2/cx and ERβ5, in which amino-terminal truncations remove some of the ligand-binding domain²⁶.

In the breast, ERα-positive epithelial cells located in the ductal lumen facilitate the development of a branched ductal tree, which functions as a scaffold for milk-producing alveoli. Not all luminal cells express ERα, but ERα-null mice develop only a rudimentary mammary ductal tree and are unable to lactate¹²⁸, indicating that the ERα-positive cells make an essential contribution to mammary development. By contrast, the mammary glands of ERβ-null mice develop normally¹²⁸. When the two receptors are co-expressed in breast cancer cell lines, ERβ functions as an antagonist of ERα, impairing the ability of oestrogen to stimulate proliferation¹²⁹.

REVIEW S

**Neoadjuvant**
A drug treatment that is given weeks to months before surgery, often to reduce the size of tumours before surgery.

**Cyclin E1**
Cyclin E1 and cyclin E2 are regulatory subunits of kinase complexes that contain CDK2 as their catalytic subunit and regulate the G1 to S phase cell cycle transition.

Cell cycle regulators. Data from experimental model systems, supported by clinical correlations, indicate that anti-oestrogens are both cytostatic and cytotoxic. Neoadjuvant endocrine therapy leads to decreased proliferation 54, and in cell culture anti-oestrogen treatment leads to a G1 phase-specific cell cycle arrest and a consequent reduction in growth rate 55. Not unexpectedly, the molecules pivotal to the anti-oestrogen effects on cell cycle progression (FIG. 2a) have central roles in the control of G1 phase progression downstream of polypeptide growth factor mitogens, as well as oestrogen. Aberrant expression of several such oestrogen and

![Diagram](attachment:diagram.png)

Figure 1 | Oestrogen action at the molecular level. Three distinct pathways of oestrogen regulation of gene expression (reviewed in REFS 1,11,31) are shown. First, in classic oestrogen signalling, ligand-bound oestrogen receptor (ER) activates gene expression — either through direct binding of dimeric ER to specific DNA response elements, EREs, in complexes including co-activators (CoAs) and histone acetyl transferases (HATs), or through protein–protein interactions with other transcription factors, particularly members of the activation protein 1 (Ap1) and specificity protein 1 (Sp1) families — to facilitate binding to serum response elements (SREs) and activation of transcription (a). Second, ER can also be activated as a consequence of signalling events downstream of receptor tyrosine kinases (RTKs) such as the epidermal growth factor receptor (EGFR), ERBB2 (also known as HER2) and the insulin-like growth factor receptor (IGFR) (b). Phosphorylation (P) by the Erk or Akt serine/threonine kinases leads to ligand-independent activation of the ER. Third, signalling can be mediated through non-genomic mechanisms by ER that is localized at the cell membrane or in the cytoplasm (c). Ligand binding induces the assembly of functional protein complexes that involve other signalling molecules and that activate intracellular signalling cascades, resulting in transcription factor (TF) activation. Two recently characterized mechanisms that ultimately activate transcription independently of ER binding to DNA are illustrated: ligand-induced methylation (M) of ER and formation of an ER–PI3K–Src–focal adhesion kinase (FAK) complex that activates Akt (d), and activation of Erk by ER–Src–PELP1 complexes (e).

anti-oestrogen targets confers resistance *in vitro* and is associated with reduced tamoxifen responsiveness in patients. Overexpression of MYC, cyclin E1, cyclin D1 or the cyclin D1 splice variant cyclin D1b, or the inactivation of the RB tumour suppressor — an important substrate for cyclin-dependent kinases (CDKs) that are active in G1 phase — and the decreased expression of the CDK inhibitors p21 or p27, results in decreased anti-oestrogen sensitivity *in vitro* 56–64. MYC overexpression and consequent tamoxifen resistance is accompanied by transcriptional repression of *CDKN1A* (which encodes p21) 65, relieving the inhibitory effect of p21 on cyclin E1–CDK2 complexes. Cyclin D1 overexpression leads to an increased abundance of cyclin D1–CDK4 complexes (which indirectly activate cyclin E1–CDK2 (REF. 58) by sequestering p21 and p27) and to the activation of cyclin E2–CDK2 by increased transcription of *CCNE2* (which encodes cyclin E2) 66. In addition to its cell cycle regulatory role, cyclin D1 interacts with several transcription factors, including ERα and STAT3 (REF. 67). Tamoxifen induces cyclin D1 binding to ERα at the expense of cyclin D1–STAT3 binding, activating both STAT3 and ERα — this is an additional mechanism by which cyclin D1 overexpression can affect the tumour’s response to tamoxifen 68. Therefore, MYC and cyclin D1 overexpression can potentially affect anti-oestrogen sensitivity at several levels.

Supporting the clinical relevance of the aberrant expression of these molecules, there is accumulating evidence that overexpression of MYC or cyclin D1 is associated with tamoxifen resistance in patients 69,70. There is also more limited evidence for a relationship between the overexpression of cyclin E1, RB inactivation and reduced expression of p27 and clinical response 62,69–71. In breast cancer, overexpression of MYC, cyclin D1 and cyclin E1 is at least two to three times more common than amplification of the corresponding genes 69, and RB inactivation is also more common than *RB1* deletion or mutation 62. The reasons for this probably include the activation of upstream mitogenic signalling pathways and the deregulation of transcriptional regulators, including the E2f family. The gene encoding p27, *CDKN1B*, is rarely mutated or deleted in breast cancer, but p27 expression is frequently reduced by oncogenic activation of mitogenic signalling (for example, by ERBB2 overexpression or Src activation) and increased p27 degradation 71. The microRNAs (miRNAs) miR-221 and miR-222 reduce p27 expression and confer resistance to tamoxifen, although the precise mechanism of resistance is unclear, as these miRNAs also reduce ERα expression and are overexpressed in ERBB2-overexpressing breast cancers 72,73. There are conflicting data on the relationship between p21 expression and outcome in breast cancer 69, and the role of p21 in tamoxifen response in breast cancer has not been studied extensively, although the ERBB2 repressor FOXP3 is essential for p21 expression 74 and there is some evidence for p21 deregulation in cancers with ERBB2 overexpression or Akt activation 70. Akt activation results in the mislocalization of p21 to the cytoplasm; predominantly cytoplasmic localization of p21 has been associated with poor response to tamoxifen 75.

REVIEW

messenger ceramide¹⁷,⁷⁶ (FIG. 2b). Crosstalk between the apoptotic effects of anti-oestrogens and the tumour necrosis factor (TNF) pathway, as well as anti-oestrogen effects on survival signalling through the PI3K–Akt, NF-κB and interferon pathways, is also likely to contribute to anti-oestrogen-mediated apoptosis¹⁷ (FIG. 2b). Finally, intriguing recent observations indicate that autophagy is a mechanism of cell survival in breast cancer cells that are resistant to apoptotic concentrations of tamoxifen⁷⁷.

It has been difficult to establish the role of apoptosis in the clinical setting. Neoadjuvant studies have yielded conflicting data and have been limited by small patient numbers and the methodological challenges of measuring apoptosis *in vivo*⁷⁸. Nevertheless, many signatures of response to endocrine therapy include genes with roles in apoptosis, as discussed below. As tumour growth reflects the balance between cell proliferation and cell death, disruption of this balance by effects on survival signalling and apoptosis are expected to affect clinical response. There is accumulating evidence for the increased expression of anti-apoptotic molecules, for example BCL-2 and BCL-Xₗ, and decreased expression of pro-apoptotic molecules, for example BAK, BIK and caspase 9, in attenuated responses to tamoxifen¹⁷. Although many of these responses are probably consequences of the activation of survival signalling through the PI3K–Akt pathway, as a consequence of overexpression of receptor tyrosine kinases and increased ‘non-genomic’ signalling from cytoplasmic ER, other pathways have been documented. For example, increased DNA-binding and transcriptional activity of NF-κB are features of tamoxifen-resistant cells³⁰, and tamoxifen sensitivity can be restored by parthenolide, a specific NF-κB inhibitor³⁰,⁷⁹. Tamoxifen insensitivity *in vitro* is also associated with the downregulation of IRF1, an interferon-responsive putative tumour suppressor that binds NF-κB and is essential for apoptosis. Furthermore, overexpression of a splice variant of human X-box-binding protein 1 (XBP1), a transcription factor that controls the unfolded protein response, is also associated with tamoxifen resistance *in vitro* and poor survival in patients with breast cancer treated with tamoxifen⁸⁰⁻⁸². NF-κB, XBP1 and IRF1 expression are correlated in patients with breast cancer⁸³, which may indicate that these molecules function in a common pathway¹⁴.

**Signatures of tamoxifen responsiveness**

The advent of genome-wide gene expression analysis allowed clinical material from patients of known responsiveness to tamoxifen to be used as a means of gaining broad insights into the potential mechanisms of endocrine resistance. It also helped in the development of clinically relevant markers of response and potential mechanisms of resistance. This is not without its own limitations, for example the difficulty of obtaining tumour tissue at the time when resistance has developed, rather than before therapy.

**Selection on the basis of disease outcome.** The identification of women who are unlikely to respond to endocrine therapy, but who may benefit from chemotherapy, is a

Figure 2 | **Anti-oestrogen action on the cell cycle and apoptotic pathways.**

a | Anti-oestrogen (AE) treatment of cultured breast cancer cells leads to oestrogen receptor (ER) binding and subsequent rapid decreases in the expression of MYC, followed by decreased expression of cyclin D1. Downregulation of MYC leads to de-repression of CDKN1A (which encodes p21) transcription. In addition, because cyclin D1–cyclin-dependent kinase 4 (CDK4) complexes function as a cellular ‘sink’ for the CDK inhibitors p21 and p27, the reduction in cyclin D1–CDK4 abundance makes p21 and p27 available for cyclin E1–CDK2 binding, and so indirectly contributes to the inhibition of cyclin E1–CDK2 activity. The decrease in activity of both CDK2 and CDK4 prevents RB phosphorylation (P) and therefore impedes transition into S phase. Treatment with the pure anti-oestrogen ICI 182780, but not tamoxifen, leads to an increase in the expression of p27 and molecular markers that are characteristic of quiescence (G0), that is, the formation of p130–E2F4 complexes and the accumulation of hyperphosphorylated E2F4. These effects are reviewed in REF. 55.

b | Proteins and processes that are upregulated during anti-oestrogen-induced apoptosis are indicated in green (reviewed in REF. 76); red crosses indicate proteins that are downregulated. Apoptotic concentrations of tamoxifen elicit caspase activation downstream of responses such as activation of the stress kinases Jun N-terminal kinase (JNK) and p38 MAPK, activation of the intracellular second messenger ceramide, transcriptional downregulation of anti-apoptotic molecules including BCL-2, and upregulation of pro-apoptotic molecules such as IRF1, BIK and possibly BAK. In addition, anti-oestrogens have effects on the interferon (IFN) and nuclear factor-κB (NF-κB) pathways, and on survival signalling through Akt downstream of receptor tyrosine kinases (RTKs), as well as synergistic effects on tumour necrosis factor (TNF)-mediated apoptosis. These pro-apoptotic effects of tamoxifen are opposed by autophagy⁷⁷. JAK, janus kinase; Stat, signal transducer and transcription activator.

---

**Cell survival signalling and apoptosis.** Treatment with high (micromolar) concentrations of anti-oestrogen, oestrogen withdrawal (mimicking the effects of aromatase inhibitors) or aromatase inhibitor treatment of cells transfected with aromatase leads to the activation of the cellular stress response and apoptosis in breast cancer cells¹⁷,⁷⁶. The molecular mechanisms are not well defined, but several molecular consequences that promote apoptosis have been documented, including the regulation of Bcl-2 family members and increases in the apoptotic second

**Cyclin D1**

The regulatory subunit of a kinase complex that functions as a growth factor sensor to regulate G1 phase cell cycle progression. The catalytic subunits of cyclin D1-dependent kinases are CDK4 and CDK6.

Table 1 | Pathways associated with tamoxifen resistance in vitro

| Pathway | Molecular aberration | Clinical correlates | Tamoxifen resistance |
| --- | --- | --- | --- |
|  |  | Poor outcome* |  |
| **ER signalling** |  |  |  |
| ERs and ERRs | ERα loss (methylation) | Yes<sup>14</sup> | Yes<sup>14</sup> |
|  | ERα36 (truncated variant) | No<sup>24</sup> | Yes<sup>24</sup> |
|  | ERα phosphorylation | Yes<sup>130</sup> | Yes<sup>130</sup> |
|  | ERα methylation (by PRMT1) | ND | ND |
|  | ERβ, ERβ2/cx and ERβ5 | Yes<sup>23,25</sup> | Yes<sup>23,25</sup> |
|  | ERRγ | ND | ND |
| **ER-associated transcription factors and co-activators** | AP1 overexpression | Yes<sup>28–30</sup> | Yes<sup>28–30</sup> |
|  | NF-κB activation | Yes<sup>131</sup> | Yes<sup>131</sup> |
|  | NCOA1 overexpression | Yes<sup>132</sup> | Yes<sup>132</sup> |
|  | NCOA3 amplification | Yes<sup>32,133</sup> | Yes<sup>32,132</sup> |
|  | PELP1 | ND | ND |
|  | CBP and p300 overexpression | Yes<sup>134</sup> | Yes<sup>134</sup> |
| **Growth factor receptors and cytoplasmic signalling** |  |  |  |
| Receptors | EGFR overexpression and ERBB2 amplification | Yes<sup>21,135</sup> | Yes<sup>21,135</sup> |
|  | PAX2 loss leading to ERBB2 de-repression | Yes<sup>46</sup> | Yes<sup>46</sup> |
|  | IGF1R overexpression | Yes<sup>136</sup> | ND |
|  | FGFR overexpression | ND | Yes (only FGFR4)<sup>137</sup> |
| MAPK signalling | Mek and Erk activation | Yes<sup>138</sup> | Yes<sup>138</sup> |
|  | CDK10 methylation | ND | Yes<sup>114</sup> |
| PI3K signalling | Akt activation and overexpression | Yes<sup>139,140</sup> | Yes<sup>139,140</sup> |
|  | PTEN loss | Yes<sup>141</sup> | Yes<sup>141</sup> |
| SRC and SRC-interacting proteins | SRC activation | ND | ND |
|  | BCAR1 | No<sup>111</sup> | Yes<sup>48</sup> |
|  | BCAR3 overexpression | No<sup>111</sup> | No — associated with favourable response<sup>111</sup> |

pressing clinical need that has driven much of the work aimed at identifying clinically useful markers of tamoxifen response. Gene selection on the basis of correlations between expression and patient outcome, with the goal of identifying a minimum gene set that retains robust predictive ability in women treated with tamoxifen, has led to the development of several gene signatures (TABLE 2), some of which are the subject of testing in prospective clinical trials (reviewed in REF. 84). Other signatures derived using a broader group of patients with breast cancer can also distinguish prognostic groups within ER-positive cancers treated with tamoxifen (TABLE 2).

Consistent with the idea that deregulation of ER signalling and upregulation of alternative mitogenic pathways is a common mechanism of resistance, essentially all these signatures contain a substantial proportion of genes that are ER targets, are involved in ER action or have roles in cell proliferation and survival (TABLE 2). Genes involved in apoptosis, invasion and cell motility are also consistently included<sup>85</sup> (TABLE 2). Perhaps surprisingly, MYC, CCND1 (which encodes cyclin D1) and RB1 are rarely included in the signatures, despite the predominance of proliferation markers and the importance of these molecules in anti-oestrogen effects on proliferation. However, signatures predominantly composed of MYC-responsive or RB- and E2f-responsive genes are associated with poor outcome in women treated with tamoxifen<sup>62,86</sup>, suggesting that the absence of these genes reflects the limitations of assessing their activity on the basis of their expression alone.

Because of their shared biological features, many of the signatures selected on the basis of correlations with outcome are largely concordant, both with each other and with more generally derived prognostic signatures that have not been tested for association with response to tamoxifen, for example the ‘Mammaprint’ signature<sup>87–89</sup>. In a recent meta-analysis that compared nine prognostic signatures, the subset of genes related to proliferation within each signature was at least as good a predictor as the whole signature<sup>90</sup>. However, the remaining genes were also often significantly correlated with outcome<sup>90</sup>, indicating that biological processes other than proliferation may also be important. These probably include apoptosis and invasion or metastasis, as signatures predominantly composed of genes with roles in these cellular processes are predictive of poor outcome after tamoxifen treatment<sup>86,91,92</sup>.

**Bcl-2 family**
A protein family of up to 25 members that are classified according to their structure and function as anti-apoptotic (BCL2-like) or pro-apoptotic (multidomain BAX-like and ‘BH3-only’) proteins.

**Autophagy**
A cellular response in which the cell metabolizes its own contents and organelles to maintain energy production, often in response to stressful stimuli. Although such a process can eventually result in cell death, it can also be used to maintain cell survival.

Table 1 (cont.) | Pathways associated with tamoxifen resistance in vitro

| Pathway                     | Molecular aberration                          | Clinical correlates | Tamoxifen resistance† |
|-----------------------------|---------------------------------------------|---------------------|-----------------------|
| **Cell cycle**              |                                             |                     |                       |
| Cyclins                     | Cyclin D1 amplification or overexpression   | Yes⁶⁹               | Yes⁶⁹                |
|                             | Cyclin D1b overexpression                   | Yes¹²²              | ND                    |
|                             | Cyclin E1 overexpression                    | Yes⁶⁹               | Yes¹⁴³                |
| MYC                         | MYC amplification and overexpression        | Yes⁶⁹,¹⁴⁴          | Yes⁶⁹                |
| CDK inhibitors              | Cytoplasmic p21 expression                  | ND                  | Yes⁷⁵                |
|                             | Low p27 expression                          | Yes¹²¹,¹⁴⁵         | Yes¹²¹,¹⁴⁵           |
| RB                          | RB inactivation                             | ND                  | Yes⁶²                 |
| **Apoptosis and cell survival signalling** |                               |                     |                       |
| Bcl-2 family members       | Low BCL-2 expression                        | Yes¹⁴⁶             | Yes¹⁴⁷                |
|                             | BIK                                        | ND                  | ND                    |
|                             | Low BAD expression                         | Yes¹⁴⁸             | Yes¹⁴⁸                |
| Survival signalling         | (IRF1)                                     | ND                  | ND                    |
|                             | XBP1 (spliced variant)                      | ND                  | Yes⁸²                 |

*Yes indicates poor outcome in women with breast cancer, whether or not treated with tamoxifen. †Yes indicates poor outcome in women treated with tamoxifen. AP1, activator protein 1; CBP, CREB-binding protein; CDK, cyclin-dependent kinase; EGFR, epidermal growth factor receptor; ER, oestrogen receptor; ERRγ, oestrogen-related receptor-γ; FGFR, fibroblast growth factor receptor; IGF1R, insulin-like growth factor receptor; IRF1, interferon regulatory factor 1; ND, not done; NF-κB, nuclear factor-κB; NCOA, nuclear receptor co-activator; PRMT1, protein arginine N-methyltransferase 1; XBP1, X-box-binding protein 1.

---

### Biology as a selection criterion.
Biomarkers of therapeutic response are expected to include genes that are mechanistically involved in conferring drug resistance, but signatures of tamoxifen response derived solely from clinical outcome data have provided only limited mechanistic insights. There is little overlap between their component genes and, consequently, there are few obvious starting points for functional studies. Potential contributing factors include dependence of the signature composition on the patient set used to derive it, and minimization of the signature size to increase clinical practicality, with the result that multiple non-overlapping signatures of equal predictive power could potentially be derived from the same analysis⁹³. Therefore, any single prognostic or predictive signature contains a somewhat arbitrary set of genes, and this selectivity can confound attempts to understand the underlying biology.

A potentially more mechanistically revealing approach is the inclusion of facets of the underlying biology in the selection criteria for genes that constitute a signature. Because of the importance of oestrogen action in breast cancer biology and the evidence for deregulation of oestrogen signalling as a major mechanism of resistance, oestrogen-regulated genes are common starting points for the derivation of signatures that are predictive of outcome in women treated with tamoxifen⁸⁶,⁹⁴–⁹⁶ (TABLE 2). The progesterone receptor (PR) is a well-established marker of response to endocrine therapy⁹⁷; PR expression occurs rarely in the absence of functional ER, implying that PR-positive cancers, and cancers expressing high levels of other oestrogen-induced genes, are likely to be more dependent on ER signalling. However, although a signature

---

### Unfolded protein response
A cellular response to stress that senses the misfolding of proteins in the endoplasmic reticulum. It activates a series of pathways that help the cells survive proteotoxicity that is caused by unfolded proteins or activate mechanisms of cell death.

### Concordant
Clinical biomarkers and signatures are concordant if they classify the same patients as ‘high risk’.

---

© 2009 Macmillan Publishers Limited. All rights reserved

Multivariate analysis  
A statistical analysis of the relationship between multiple parameters (variables) to identify those that have a dominant effect on outcome (termed independent predictors of outcome) and those that are dependent or redundant.

Biological concepts analysis  
A bioinformatic approach in which related information is grouped together into a 'biological concept', and associations between different 'concepts' are sought.

response to tamoxifen may arise through differences in intrinsic tumour biology rather than through differences in tamoxifen response *per se*.

Differences in response also arise from differences in the rate of cell proliferation. Although proliferation is reduced in almost all ER-positive breast cancers following neoadjuvant endocrine therapy, the level of proliferation following short-term therapy is a better predictor of outcome than its pretreatment level or the magnitude of the decrease in proliferation following treatment [102,103]. This suggests that poor treatment response does not result from intrinsically high proliferation but rather from an ability to maintain high levels of proliferation in the presence of tamoxifen. Increased proliferation that occurs as a consequence of deregulated mitogenic signalling pathways may be more readily inhibited by tamoxifen than is increased proliferation that results from genetic or epigenetic events targeting individual genes directly involved in cell cycle progression (for example, MYC or CCND1 amplification). In addition, the effects of the deregulation of cell cycle regulatory genes are likely to be gene specific. Consequently, to better understand the mechanisms of tamoxifen resistance

it might be informative to distinguish different mechanisms of deregulation of proliferation or different types of proliferative defects.

### Insights from computational approaches

#### Dissection of biological processes  
A limitation of many clustering algorithms used to derive gene signatures is their dependence on global behaviour across the entire test set of profiles, at the expense of identifying more specific features that may be strongly related to only a small group of patients or conditions. In addition, an individual gene is often allocated to a particular cluster, although genes commonly contribute to different biological processes or are regulated by multiple stimuli. However, some algorithms do allow the identification of partially overlapping gene sets that show coordinate expression in only particular contexts, and so these tend to be functionally coherent. One such algorithm yielded eight biologically consistent gene set ‘modules’ in breast cancer expression profiles, including one module that was largely composed of genes with roles in apoptosis [91]. These genes were downregulated when MCF-7 breast cancer cells were treated with tamoxifen, suggesting that their expression may reflect active ER signalling [91]. Consistent with this idea, high expression of this module was associated with longer survival in several patient populations, including in women treated with tamoxifen.

A complementary approach is to identify biologically coherent signatures on the basis of a candidate gene approach or a functional annotation. For example, the high expression of a gene signature consisting of genes that are differentially expressed following RB loss, or the expression of constitutively active (growth-inhibitory) RB mutants in rodent fibroblasts, was associated with shorter disease-free survival in women treated with tamoxifen [62]. In another study, a combination of gene ontology and pathway analysis that used a curated database of functional annotations led to the subdivision of oestrogen-regulated genes into networks that represented specific biological processes, for example two distinct aspects of cell proliferation (cell cycle progression and increased cell size (cell growth)), apoptosis and survival signalling, and transcriptional regulation [86]. The networks representing cell proliferation and apoptosis were predictive of poor outcome in women treated with tamoxifen [86]. Previous analyses had pointed to both proliferation and apoptosis genes as potential markers of outcome in women treated with tamoxifen. This study provides evidence that these may identify distinct subpopulations with different mechanisms of resistance, as the cell proliferation-related signatures and the apoptosis signature were independent predictors of outcome in multivariate analysis [86].

### Molecular mechanisms  
Although data from large-scale genomic studies of breast cancer are increasingly available, few studies have adopted global approaches to identify specific molecular aberrations that might be associated with response to endocrine therapy. However, a global biological concept analysis identified associations

---

**Figure 3 | Molecular mechanisms of endocrine resistance.** Acquired tamoxifen resistance in breast cancer cells is mediated by either modulation of the oestrogen receptor (ER) pathway or aberrant or compensatory cellular signalling pathways controlled by growth factor receptors that negate the anti-proliferative and pro-apoptotic effects of tamoxifen [11,12,14–21]. In tamoxifen-resistant cells, these endpoints are regulated by nuclear receptors (ER and EERα) and their co-activators, for example nuclear receptor co-activator 3 (NCOA3; also known as AIB1 or SRC3); receptor tyrosine kinases (RTKs), including the epidermal growth factor receptor (EGFR), ERBB2, insulin-like growth factor receptor (IGFR) and fibroblast growth factor receptor (FGFR); RTK ligands, including transforming growth factor-α (TGFα) and heregulin (HRG), and downstream signalling pathways; and cellular stress responses, including those downstream of reactive oxygen species (ROS). CDK10, cyclin-dependent kinase 10; JNK, Jun N-terminal kinase; P, phosphorylation; PAK1, p21-activated kinase 1; PAX2, paired box 2 transcription factor; NF-κB, nuclear factor-κB; XBP1, X-box-binding protein 1.

Table 2 | Signatures predicting response to endocrine therapies

| Signature | Selection criteria | Clinical correlates | Biological processes |
|-----------|--------------------|---------------------|----------------------|
| Breast cancer subtypes<sup>98,100</sup> | Hierarchical clustering of breast cancers | Poor outcome* | Yes | Tamoxifen resistance† | Yes (luminal A compared with luminal B) | Proliferation (luminal A compared with luminal B) |
| HOXB13/IL17RB expression ratio<sup>149</sup> | Differential expression in recurrent compared with non-recurrent breast cancers treated with adjuvant tamoxifen; association with recurrence | Yes | Yes | ER targets and invasion (HOXB13) |
| 21-gene signature: 16 cancer-related and 5 controls (Oncotype Dx)<sup>150</sup> | 250 manually selected genes tested for association with outcome following adjuvant tamoxifen | Yes | Yes | ER targets, proliferation<sup>85</sup>, apoptosis<sup>85</sup>, invasion and motility<sup>85</sup>, signalling<sup>85</sup> and ERBB2 amplification |
| 81-gene discriminatory signature and 44-gene predictive signature<sup>151</sup> | Differential expression in progressive disease compared with objective response in recurrent disease treated with tamoxifen | No<sup>89</sup> | Yes | ER action and ER targets, apoptosis, extracellular matrix formation, immune response and 17q21–22 amplification |
| 76-gene signature<sup>152</sup> | Correlation with outcome in breast cancer | Yes | Possibly | Proliferation and cell cycle<sup>85</sup>, and apoptosis<sup>85</sup> |
| REF. 94 | Oestrogen-induced genes used to cluster ER-positive breast cancer: groups have differing outcomes | Yes | Yes | ER targets, proliferation and cell cycle, and apoptosis |
| Genomic grade signature<sup>153</sup> | Distinguishes grade 1 and grade 3 breast cancer | Yes | Yes<sup>88</sup> | Proliferation and cell cycle<sup>85</sup>, apoptosis<sup>85</sup>, motility<sup>85</sup> and immune response<sup>85</sup> |
| ‘TuM1’ 33 genes<sup>91</sup> | Modular analysis linked to biological function | No | Yes | Apoptosis |
| 59-gene RB-deficiency signature<sup>62</sup> | Genes deregulated by RB1 loss or repressed by RB activation | ND | Yes | Proliferation and E2f targets |
| 36-gene signature<sup>154</sup> | Segregation of relapse and relapse-free breast cancers treated with adjuvant tamoxifen | ND | Yes | ER action and ER targets, proliferation, apoptosis, cell survival, adhesion and motility, signalling, metabolism and immune response |
| REFS 155,156 | Segregation of responsive and non-responsive breast cancers treated with neoadjuvant aromatase inhibitor with or without tamoxifen: oestrogen-responsive gene subset identified | ND | Yes | ER targets, cell cycle, cell movement, apoptosis, and TGFβ and ERBB2 signalling |
| 36 genes<sup>96</sup> | Oestrogen-induced, underexpressed in ER-negative breast cancer and correlated with PR | No | Yes | ER targets |
| 181 genes in 13 biological clusters<sup>157</sup> | Clusters of co-expressed genes in breast cancers treated with adjuvant tamoxifen; 13 were combined to predict outcome after adjuvant tamoxifen | ND | Yes — adjuvant tamoxifen; no — tamoxifen at relapse except for one cluster alone | Cell cycle and proliferation, apoptosis, motility, and signalling of TGFβ, EGF, IGF and PDGF |
| 256 genes in 4 signatures<sup>86</sup> | Functional annotation of oestrogen-regulated genes | ND | Yes | ER targets, Myc targets, cell cycle, proliferation, cell growth, apoptosis, survival signalling, and invasion and motility |
| 47 genes<sup>158</sup> | Differential expression at the time of progression on tamoxifen therapy compared with non-recurrent cancers | ND | Yes — some individual genes | ER targets, immune response, transcriptional regulation, proliferation, invasion and adhesion, and apoptosis |

*Yes indicates poor outcome in women with breast cancer, whether or not treated with tamoxifen. †Yes indicates poor outcome in women treated with tamoxifen. EGF, epidermal growth factor; ER, oestrogen receptor; HOXB13, homeobox B13; IGF, insulin-like growth factor; IL17RB, interleukin 17 receptor B; ND, not done; PDGF, platelet-derived growth factor; PR, progesterone receptor; TGFβ, transforming growth factor-β.

between a signature of early relapse in ER-positive breast cancer and the activation of MYC- and E2f-responsive pathways, together with chromosomal aberrations at loci including 8q24 (the location of the MYC gene)<sup>104</sup>.

The E2f and MYC biological concepts seemed to make independent contributions to the likelihood of disease progression<sup>104</sup>, despite the considerable overlap in the known functions of these genes. This may be because

aberrations in these pathways occur independently and characterize biologically distinct phenotypes, as a subset of ER-positive breast cancers is characterized by a high probability of MYC pathway deregulation but a low probability of E2f pathway deregulation$^{105,106}$.

### Functional genetic screens

In parallel with the emergence of increasingly sophisticated bioinformatic and genomic tools, there has been an increase in the feasibility of global functional screens. Gain-of-function screens in breast cancer cells have identified >40 candidate tamoxifen resistance genes$^{47,51,107-110}$. These include genes encoding cell surface receptors (EGFR, ERBB2, platelet-derived growth factor receptor-β (PDGFRB) and colony-stimulating factor 1 receptor (CSF1R)) and their ligands (neuregulin, which is a ligand for some Erbb receptors, and FGF17), intracellular signalling molecules (BCAR1, BCAR3, AKT1, AKT2, SRC and GRB7) and transcriptional regulators (BCAR2 (also known as TRERF1) and the ER co-repressor NCOR2 (also known as SMRT)), as well as genes of poorly understood function (for example, BCAR4 and TLE3). Many have roles in signalling pathways that are implicated in endocrine resistance, such as the Erk and PI3K–Akt pathways (as summarized above and in TABLE 1), illustrating the potential for identifying functionally relevant pathways using this approach. Genes derived from unbiased functional screens will not necessarily be deregulated in cancer, and even if they do show differential expression there is no guarantee that this will correlate with a therapeutic response. However, some of the genes that confer tamoxifen resistance in functional genetic screens are overexpressed in breast cancer and are independent predictors of response in women treated with tamoxifen at relapse$^{48,111-113}$.

An RNA interference screen to identify kinases for which decreased expression can reduce tamoxifen sensitivity yielded 20 candidates, including multiple components of the Erk signalling pathway, other intracellular signalling molecules and several kinases related to CDC2, the prototypical CDK$^{114}$. One of these, CDK10, was examined in more detail. The CDK10 gene was silenced by methylation in 7 of the 34 breast cancers examined, and low CDK10 expression was associated with poor outcome in women treated with adjuvant tamoxifen$^{114}$. Downregulation of CDK10 in cultured breast cancer cells led to an ETS2-dependent increase in RAF1 transcription and activation of the Erk pathway. Therefore, downregulation of CDK10 seems to modulate tamoxifen sensitivity through effects on intracellular signalling, rather than through more direct effects on cell cycle progression. This screen also identified sensitizers of tamoxifen response, many of which were in the PI3K pathway, including PIK3C2B, PDK1 (which is activated by PI3K) and the PDK1 substrates PRKCZ and AKT1, as well as several other activators of Akt$^{115}$. The sensitizers identified in a parallel small-molecule screen also included many inhibitors of Akt$^{115}$, again emphasizing the importance of the PI3K pathway as a determinant of the response to tamoxifen.

### Implications for therapy

Endocrine-resistant ER-positive disease accounts for approximately one in four breast cancers but, similarly to the ‘triple-negative’ phenotype that does not express ER, PR or ERBB2, it is not well served by current targeted therapies. The possibility of identifying new targets for therapy in resistant disease, or patients who may benefit from additional treatment with existing therapies, provides a strong impetus to identify markers and mediators of therapeutic resistance. Some candidate endocrine resistance genes can also affect response to other therapies; for example, BCAR1 confers resistance to adriamycin$^{116}$, and cyclin D1 confers resistance to the EGFR-targeted therapy gefitinib$^{117}$. This adds to the challenge of identifying alternative therapies in endocrine-resistant disease. Recent data suggest that integrative bioinformatics and/or targeted large-scale screening may assist in meeting this challenge. One bioinformatic study identified highly significant links between genes repressed by inhibition of PI3K and genes activated by MYC, suggesting that PI3K inhibition may be an effective therapy for MYC-dependent cancers$^{104}$. Together with evidence from the combined small-molecule and RNA interference screen discussed above that highlighted inhibition of Akt as a means of potentiating tamoxifen responses *in vitro*$^{115}$, this observation argues that therapies targeting the PI3K–Akt pathway may be useful in endocrine-resistant breast cancer. Similar integrative analyses including data relating specifically to endocrine resistance may yield new targets for further analysis and testing. Synthetic lethal genetic and/or small-molecule screens using cell lines that are tamoxifen resistant, either because of adaptation in culture or because of the introduction of specific genetic lesions, may also identify pathways specifically required in tamoxifen-resistant cells and the means of targeting them.

Current models of endocrine resistance (FIG. 3) identify many individual molecules that can affect anti-oestrogen sensitivity *in vitro* and are therefore potential targets for therapy in endocrine-resistant disease. The convergence of many potential resistance genes on the Erk and PI3K pathways suggests that inhibitors of these pathways that are currently in development may be useful in this setting. Data from cell line models indicate that inhibiting Src$^{118}$, BCAR1 (REF. 119), Mek–Erk$^{120}$, Akt$^{38,115,121,122}$, mTOR$^{39,123}$ or NF-κB$^{30,79}$ can restore or potentiate tamoxifen sensitivity. However, early clinical trials of EGFR- and ERBB2-targeted agents that reduce Erk signalling (gefitinib, erlotinib, trastuzumab and lapatinib) or mTOR inhibitors (everolimus and temsirolimus) in combination with endocrine therapies have yielded mixed results, which may be partly due to difficulties in identifying tumours that are dependent on these pathways and so are most likely to benefit from the additional therapy$^{124}$. In addition, crosstalk and negative feedback loops between different signal transduction pathways may lead to cellular resistance to individual inhibitors. One approach to this problem is the use of inhibitors that target more than one kinase or the use of combinations of therapies to simultaneously target multiple pathways. Alternatively, end points shared
by multiple pathways — for example MYC and cyclin D1–CDK4 — could be targeted, which would have the benefit of more directly targeting proliferation, a central component in essentially all gene signatures that are predictive of endocrine resistance. Recent evidence indicates that targeting MYC may be an effective cancer therapy<sup>125</sup>, and some small-molecule CDK inhibitors have entered clinical trials<sup>126</sup>.

### Concluding remarks

To date, much of the information on the mechanisms of endocrine resistance has come from studies that have considered relatively few genes. The application of integrative approaches that examine gene expression in the context of global genomic, proteomic or functional data to understand mechanisms of endocrine resistance is in its infancy and has relied on small-scale genomic and transcriptional data sets. Nevertheless, as summarized above, this approach has provided some pointers towards areas for further study. The next generation of high-throughput technologies allows genome-scale genetic and synthetic lethal screens, large-scale proteomic and phospho-proteomic profiling, and parallel transcriptome, epigenome and genome sequencing of both experimental models and well-annotated clinical samples. The further use of integrative bioinformatic approaches combined with these new technologies offers the potential for a ‘systems biology’ approach, in which cellular responses and pathways are considered as a whole. This may identify previously unconsidered mechanisms for the modulation of therapeutic response, as suggested by recent data implicating miRNAs in tamoxifen resistance<sup>72</sup>, and thereby greatly increase the depth of our understanding of the mechanisms of endocrine-resistant disease. This, in turn, should aid in the identification of the specific genetic events that cause resistance and the means by which these events can be targeted therapeutically.

---

1. McDonnell, D. P. & Norris, J. D. Connections and regulation of the human estrogen receptor. *Science* **296**, 1642–1644 (2002).
2. Colditz, G. A. Relationship between estrogen levels, use of hormone replacement therapy, and breast cancer. *J. Natl Cancer Inst.* **90**, 814–823 (1998).
3. Hankinson, S. E., Colditz, G. A. & Willett, W. C. Towards an integrated model for breast cancer etiology: the lifelong interplay of genes, lifestyle, and hormones. *Breast Cancer Res.* **6**, 213–218 (2004).
4. Cuzick, J. Aromatase inhibitors for breast cancer prevention. *J. Clin. Oncol.* **23**, 1636–1643 (2005).
5. Jordan, V. C. Chemoprevention of breast cancer with selective oestrogen-receptor modulators. *Nature Rev. Cancer* **7**, 46–53 (2007).
6. Howell, A. The endocrine prevention of breast cancer. *Best Pract. Res. Clin. Endocrinol. Metab.* **22**, 615–623 (2008).
7. Harvey, J. M., Clark, G. M., Osborne, C. K. & Allred, D. C. Estrogen receptor status by immunohistochemistry is superior to the ligand-binding assay for predicting response to adjuvant endocrine therapy in breast cancer. *J. Clin. Oncol.* **17**, 1474–1481 (1999).
8. Jensen, E. V. & Jordan, V. C. The estrogen receptor: a model for molecular medicine. *Clin. Cancer Res.* **9**, 1980–1989 (2003).
9. Early Breast Cancer Trialists’ Collaborative Group (EBCTCG). Effects of chemotherapy and hormonal therapy for early breast cancer on recurrence and 15-year survival: an overview of the randomised trials. *Lancet* **365**, 1687–1717 (2005).
10. Johnston, S. R. & Dowsett, M. Aromatase inhibitors for breast cancer: lessons from the laboratory. *Nature Rev. Cancer* **3**, 821–831 (2003).
11. MacGregor, J. I. & Jordan, V. C. Basic guide to the mechanisms of antiestrogen action. *Pharmacol. Rev.* **50**, 151–196 (1998).
12. Ali, S. & Coombes, R. C. Endocrine-responsive breast cancer and strategies for combating resistance. *Nature Rev. Cancer* **2**, 101–112 (2002).
13. Hoskins, J. M., Carey, L. A. & McLeod, H. L. CYP2D6 and tamoxifen: DNA matters in breast cancer. *Nature Rev. Cancer* **9**, 576–586 (2009).
14. Clarke, R. *et al*. Antiestrogen resistance in breast cancer and the role of estrogen receptor signaling. *Oncogene* **22**, 7316–7339 (2003).
15. Jordan, V. C. Selective estrogen receptor modulation: concept and consequences in cancer. *Cancer Cell* **5**, 207–213 (2004).
16. Ring, A. & Dowsett, M. Mechanisms of tamoxifen resistance. *Endocr. Relat. Cancer* **11**, 643–658 (2004).
17. Riggins, R. B., Bouton, A. H., Liu, M. C. & Clarke, R. Antiestrogens, aromatase inhibitors, and apoptosis in breast cancer. *Vitam. Horm.* **71**, 201–237 (2005).
18. Nicholson, R. I. *et al*. Growth factor signalling in endocrine and anti-growth factor resistant breast cancer. *Rev. Endocr. Metab. Disord.* **8**, 241–253 (2007).

19. Jordan, V. C. & O'Malley, B. W. Selective estrogen-receptor modulators and antihormonal resistance in breast cancer. *J. Clin. Oncol.* **25**, 5815–5824 (2007).
20. Riggins, R. B., Schrecengost, R. S., Guerrero, M. S. & Bouton, A. H. Pathways to tamoxifen resistance. *Cancer Lett.* **256**, 1–24 (2007).
21. Arpino, G., Wiechmann, L., Osborne, C. K. & Schiff, R. Crosstalk between the estrogen receptor and the HER tyrosine kinase receptor family: molecular mechanism and clinical implications for endocrine therapy resistance. *Endocr. Rev.* **29**, 217–233 (2008).
22. Gutierrez, M. C. *et al*. Molecular changes in tamoxifen-resistant breast cancer: relationship between estrogen receptor, HER-2, and p38 mitogen-activated protein kinase. *J. Clin. Oncol.* **23**, 2469–2476 (2005).
23. Herynk, M. H. & Fuqua, S. A. Estrogen receptor mutations in human disease. *Endocr. Rev.* **25**, 869–898 (2004).
24. Shi, L. *et al*. Expression of ER-α36, a novel variant of estrogen receptor α, and resistance to tamoxifen treatment in breast cancer. *J. Clin. Oncol.* **27**, 3423–3429 (2009).
25. Honma, N. *et al*. Clinical importance of estrogen receptor-β evaluation in breast cancer patients treated with adjuvant tamoxifen therapy. *J. Clin. Oncol.* **26**, 3727–3734 (2008).
26. Murphy, L. C. & Watson, P. H. Is oestrogen receptor-β a predictor of endocrine therapy responsiveness in human breast cancer? *Endocr. Relat. Cancer* **13**, 327–334 (2006).
27. Riggins, R. B. *et al*. ERRγ mediates tamoxifen resistance in novel models of invasive lobular breast cancer. *Cancer Res.* **68**, 8908–8917 (2008).
28. Johnston, S. R. *et al*. Increased activator protein-1 DNA binding and c-Jun NH2-terminal kinase activity in human breast tumors with acquired tamoxifen resistance. *Clin. Cancer Res.* **5**, 251–256 (1999).
29. Schiff, R. *et al*. Oxidative stress and AP-1 activity in tamoxifen-resistant breast tumors *in vivo*. *J. Natl Cancer Inst.* **92**, 1926–1934 (2000).
30. Zhou, Y. *et al*. Enhanced NFκB and AP-1 transcriptional activity associated with antiestrogen resistant breast cancer. *BMC Cancer* **7**, 59 (2007).
31. Gururaj, A. E., Rayala, S. K., Vadlamudi, R. K. & Kumar, R. Novel mechanisms of resistance to endocrine therapy: genomic and nongenomic considerations. *Clin. Cancer Res.* **12**, 1001s–1007s (2006).
32. Osborne, C. K. *et al*. Role of the estrogen receptor coactivator AIB1 (SRC-3) and HER-2/neu in tamoxifen resistance in breast cancer. *J. Natl Cancer Inst.* **95**, 353–361 (2003).
33. Le Romancer, M. *et al*. Regulation of estrogen rapid signaling through arginine methylation by PRMT1. *Mol. Cell* **31**, 212–221 (2008).
34. Sainsbury, J. R., Farndon, J. R., Sherbet, G. V. & Harris, A. L. Epidermal-growth-factor receptors and estrogen receptors in human breast cancer. *Lancet* **1**, 364–366 (1985).

35. Koga, M., Musgrove, E. A. & Sutherland, R. L. Modulation of the growth-inhibitory effects of progestins and the antiestrogen hydroxyclomiphene on human breast cancer cells by epidermal growth factor and insulin. *Cancer Res.* **49**, 112–116 (1989).
36. Carroll, J. S. *et al*. p27<sup>Kip1</sup> induces quiescence and growth factor insensitivity in tamoxifen-treated breast cancer cells. *Cancer Res.* **63**, 4322–4326 (2003). This paper shows that although cells treated with tamoxifen and other SERMs remain sensitive to growth factor stimulation, cells treated with ‘pure’ anti-oestrogens do not, implicating the CDK inhibitor p27 in modulating growth factor sensitivity in this context.
37. Faridi, J., Wang, L., Endemann, G. & Roth, R. A. Expression of constitutively active Akt-3 in MCF-7 breast cancer cells reverses the estrogen and tamoxifen responsiveness of these cells *in vivo*. *Clin. Cancer Res.* **9**, 2933–2939 (2003).
38. deGraffenried, L. A. *et al*. Eicosapentaenoic acid restores tamoxifen sensitivity in breast cancer cells with high Akt activity. *Ann. Oncol.* **14**, 1051–1056 (2003).
39. deGraffenried, L. A. *et al*. Inhibition of mTOR activity restores tamoxifen response in breast cancer cells with aberrant Akt activity. *Clin. Cancer Res.* **10**, 8059–8067 (2004).
40. Miller, T. W. *et al*. Loss of phosphatase and tensin homologue deleted on chromosome 10 engages ErbB3 and insulin-like growth factor-I receptor signaling to promote antiestrogen resistance in breast cancer. *Cancer Res.* **69**, 4192–4201 (2009).
41. McClelland, R. A. *et al*. Enhanced epidermal growth factor receptor signaling in MCF7 breast cancer cells after long-term culture in the presence of the pure antiestrogen ICI 182,780 (Faslodex). *Endocrinology* **142**, 2776–2788 (2001).
42. Knowlden, J. M. *et al*. Elevated levels of epidermal growth factor receptor/c-erbB2 heterodimers mediate an autocrine growth regulatory pathway in tamoxifen-resistant MCF-7 cells. *Endocrinology* **144**, 1032–1044 (2003).
43. Hutcheson, I. R. *et al*. Oestrogen receptor-mediated modulation of the EGFR/MAPK pathway in tamoxifen-resistant MCF-7 cells. *Breast Cancer Res. Treat.* **81**, 81–93 (2003). References 41–43 provide the first detailed mechanistic evidence that activation of EGFR and ERBB2 signalling leads to anti-oestrogen resistance.
44. Hua, G. *et al*. A negative feedback regulatory loop associates the tyrosine kinase receptor ERBB2 and the transcription factor GATA4 in breast cancer cells. *Mol. Cancer Res.* **7**, 402–414 (2009).
45. Zuo, T. *et al*. FOXP3 is an X-linked breast cancer suppressor gene and an important repressor of the HER-2/ErbB2 oncogene. *Cell* **129**, 1275–1286 (2007).
REVIEW S

46. Hurtado, A. *et al.* Regulation of ERBB2 by oestrogen receptor-PAX2 determines response to tamoxifen. *Nature* 456, 663–666 (2008). This paper shows that the transcription factor PAX2 competes with the ERα co-activator AIB1, thereby modulating ER-mediated repression of ERBB2 and tamoxifen responsiveness, and that this mechanism of ERBB2 repression is of clinical significance.

47. Dorssers, L. C., van Agthoven, T., Dekker, A., van Agthoven, T. L. & Kok, E. M. Induction of antiestrogen resistance in human breast cancer cells by random insertion mutagenesis using defective retroviruses: identification of bcar-1, a common integration site. *Mol. Endocrinol.* 7, 870–878 (1993).

48. van der Flier, S. *et al.* Bcar1/p130Cas protein and primary breast cancer: prognosis and response to tamoxifen treatment. *J. Natl Cancer Inst.* 92, 120–127 (2000).

49. Brinkman, A. *et al.* The substrate domain of BCAR1 is essential for anti-estrogen-resistant proliferation of human breast cancer cells. *Breast Cancer Res. Treat.* 3 May 2009 (doi:10.1007/s10549-009-0403-0404).

50. Riggins, R. B., Quilliam, L. A. & Bouton, A. H. Synergistic promotion of c-Src activation and cell migration by Cas and AND-34/BCAR3. *J. Biol. Chem.* 278, 28264–28273 (2003).

51. van Agthoven, T. *et al.* Identification of BCAR3 by a random search for genes involved in antiestrogen resistance of human breast cancer cells. *EMBO J.* 17, 2799–2808 (1998).

52. Cai, D. *et al.* AND-34/BCAR3, a GDP exchange factor whose overexpression confers antiestrogen resistance, activates Rac, PAK1, and the cyclin D1 promoter. *Cancer Res.* 63, 6802–6808 (2003).

53. Rayala, S. K., Molli, P. R. & Kumar, R. Nuclear p21-activated kinase 1 in breast cancer packs off tamoxifen sensitivity. *Cancer Res.* 66, 5985–5988 (2006).

54. Dowsett, M. *et al.* Proliferation and apoptosis as markers of benefit in neoadjuvant endocrine therapy of breast cancer. *Clin. Cancer Res.* 12, 1024s–1030s (2006).

55. Doisneau-Sixou, S. F. *et al.* Estrogen and antiestrogen regulation of cell cycle progression in breast cancer cells. *Endocr. Relat. Cancer* 10, 179–186 (2003).

56. Prall, O. W. J., Rogan, E. M., Musgrove, E. A., Watts, C. K. W. & Sutherland, R. L. c-Myc or cyclin D1 mimics estrogen effects on cyclin E-Cdk2 activation and cell cycle reentry. *Mol. Cell. Biol.* 18, 4499–4508 (1998).

57. Venditti, M., Iwasiow, B., Orr, F. W. & Shiu, R. P. C-myc gene expression alone is sufficient to confer resistance to antiestrogen in human breast cancer cells. *Int. J. Cancer* 99, 35–42 (2002).

58. Hui, R. *et al.* Constitutive overexpression of cyclin D1 but not cyclin E confers acute resistance to antiestrogens in T-47D breast cancer cells. *Cancer Res.* 62, 6916–6923 (2002).

59. Dhillon, N. K. & Mudryj, M. Ectopic expression of cyclin E in estrogen responsive cells abrogates antiestrogen mediated growth arrest. *Oncogene* 21, 4626–4634 (2002).

60. Cariou, S. *et al.* Down-regulation of p21WAF1/CIP1 or p27Kip1 abrogates antiestrogen-mediated cell cycle arrest in human breast cancer cells. *Proc. Natl Acad. Sci. USA* 97, 9042–9046 (2000).

61. Carroll, J. S., Prall, O. W. J., Musgrove, E. A. & Sutherland, R. L. A pure estrogen antagonist inhibits cyclin E-Cdk2 activity in MCF-7 breast cancer cells and induces accumulation of p130-E2F4 complexes characteristic of quiescence. *J. Biol. Chem.* 275, 38221–38229 (2000).

62. Bosco, E. E. *et al.* The retinoblastoma tumor suppressor modifies the therapeutic response of breast cancer. *J. Clin. Invest.* 117, 218–228 (2007).

63. Varma, H., Skildum, A. J. & Conrad, S. E. Functional ablation of pRb activates Cdk2 and causes antiestrogen resistance in human breast cancer cells. *PLoS ONE* 2, e1256 (2007).

64. Wang, Y. *et al.* Cyclin D1b is aberrantly regulated in response to therapeutic challenge and promotes resistance to estrogen antagonists. *Cancer Res.* 68, 5628–5638 (2008).

65. Mukherjee, S. & Conrad, S. E. c-Myc suppresses p21WAF1/CIP1 expression during estrogen signaling and antiestrogen resistance in human breast cancer cells. *J. Biol. Chem.* 280, 17617–17625 (2005).

66. Caldon, C. E. *et al.* Estrogen regulation of cyclin E2 requires cyclin D1, but not c-Myc. *Mol. Cell. Biol.* 29 4623–4639 (2009).

67. Coqueret, O. Linking cyclins to transcriptional control. *Gene* 299, 35–55 (2002).

68. Ishii, Y., Waxman, S. & Germain, D. Tamoxifen stimulates the growth of cyclin D1-overexpressing breast cancer cells by promoting the activation of signal transducer and activator of transcription 3. *Cancer Res.* 68, 852–860 (2008).

69. Butt, A. J., McNeil, C. M., Musgrove, E. A. & Sutherland, R. L. Downstream targets of growth factor and oestrogen signalling and endocrine resistance: the potential roles of c-Myc, cyclin D1 and cyclin E. *Endocr. Relat. Cancer* 12, S47–S59 (2005).

70. Caldon, C. E., Daly, R. J., Sutherland, R. L. & Musgrove, E. A. Cell cycle control in breast cancer cells. *J. Cell. Biochem.* 97, 261–274 (2006).

71. Chu, I. M., Hengst, L. & Slingerland, J. M. The Cdk inhibitor p27 in human cancer: prognostic potential and relevance to anticancer therapy. *Nature Rev. Cancer* 8, 253–267 (2008).

72. Miller, T. E. *et al.* MicroRNA-221/222 confers tamoxifen resistance in breast cancer by targeting p27Kip1. *J. Biol. Chem.* 283, 29897–29903 (2008).

73. Zhao, J. J. *et al.* MicroRNA-221/222 negatively regulates estrogen receptor α and is associated with tamoxifen resistance in breast cancer. *J. Biol. Chem.* 283, 31079–31086 (2008).

74. Liu, R. *et al.* FOXP3 up-regulates p21 expression by site-specific inhibition of histone deacetylase 2/histone deacetylase 4 association to the locus. *Cancer Res.* 69, 2252–2259 (2009).

75. Perez-Tenorio, G. *et al.* Cytoplasmic p21WAF1/CIP1 correlates with Akt activation and poor response to tamoxifen in breast cancer. *Int. J. Oncol.* 28, 1031–1042 (2006).

76. Mandlekar, S. & Kong, A. N. Mechanisms of tamoxifen-induced apoptosis. *Apoptosis* 6, 469–477 (2001).

77. Schoenlein, P. V., Periyasamy-Thandavan, S., Samaddar, J. S., Jackson, W. H. & Barrett, J. T. Autophagy facilitates the progression of ERα-positive breast cancer cells to antiestrogen resistance. *Autophagy* 5, 400–403 (2009).

78. Butt, A. J., Sutherland, R. L. & Musgrove, E. A. Live or let die: oestrogen regulation of survival signalling in endocrine response. *Breast Cancer Res.* 9, 306 (2007).

79. Riggins, R. B., Zwart, A., Nehra, R. & Clarke, R. The nuclear factor κB inhibitor parthenolide restores ICI 182,780 (Faslodex; fulvestrant)-induced apoptosis in antiestrogen-resistant breast cancer cells. *Mol. Cancer Ther.* 4, 33–41 (2005).

80. Gu, Z. *et al.* Association of interferon regulatory factor-1, nucleophosmin, nuclear factor-κB, and cyclic AMP response element binding with acquired resistance to Faslodex (ICI 182,780). *Cancer Res.* 62, 3428–3437 (2002).

81. Gomez, B. P. *et al.* Human X-box binding protein-1 confers both estrogen independence and antiestrogen resistance in breast cancer cell lines. *FASEB J.* 21, 4013–4027 (2007).

82. Davies, M. P. *et al.* Expression and splicing of the unfolded protein response gene XBP-1 are significantly associated with clinical outcome of endocrine-treated breast cancer. *Int. J. Cancer* 123, 85–88 (2008).

83. Zhu, Y. *et al.* Expression patterns among interferon regulatory factor-1, human X-box binding protein-1, nuclear factor kappa B, nucleophosmin, estrogen receptor-alpha and progesterone receptor proteins in breast cancer tissue microarrays. *Int. J. Oncol.* 28, 67–76 (2006).

84. Sotiriou, C. & Pusztai, L. Gene-expression signatures in breast cancer. *N. Engl. J. Med.* 360, 790–800 (2009).

85. Yu, J. X. *et al.* Pathway analysis of gene signatures predicting metastasis of node-negative primary breast cancer. *BMC Cancer* 7, 182 (2007).

86. Musgrove, E. A. *et al.* Identification of functional networks of estrogen- and c-Myc-responsive genes and their relationship to response to tamoxifen therapy in breast cancer. *PLoS ONE* 3, e2987 (2008). This paper and reference 62 derive gene networks on the basis of biological function that are correlated with outcome in women treated with tamoxifen.

87. Fan, C. *et al.* Concordance among gene-expression-based predictors for breast cancer. *N. Engl. J. Med.* 355, 560–569 (2006).

88. Loi, S. *et al.* Definition of clinically distinct molecular subtypes in estrogen receptor-positive breast carcinomas through genomic grade. *J. Clin. Oncol.* 25, 1239–1246 (2007).

89. Kok, M. *et al.* Comparison of gene expression profiles predicting progression in breast cancer patients treated with tamoxifen. *Breast Cancer Res. Treat.* 113, 275–283 (2009).

90. Wirapati, P. *et al.* Meta-analysis of gene expression profiles in breast cancer: toward a unified understanding of breast cancer subtyping and prognosis signatures. *Breast Cancer Res.* 10, R65 (2008).

91. Yu, K., Ganesan, K., Miller, L. D. & Tan, P. A modular analysis of breast cancer reveals a novel low-grade molecular signature in estrogen receptor-positive tumors. *Clin. Cancer Res.* 12, 3288–3296 (2006).

92. Helleman, J. *et al.* Association of an extracellular matrix gene cluster with breast cancer prognosis and endocrine therapy response. *Clin. Cancer Res.* 14, 5555–5564 (2008).

93. Ein-Dor, L., Kela, I., Getz, G., Givol, D. & Domany, E. Outcome signature genes in breast cancer: is there a unique set? *Bioinformatics* 21, 171–178 (2005).

94. Oh, D. S. *et al.* Estrogen-regulated genes predict survival in hormone receptor-positive breast cancers. *J. Clin. Oncol.* 24, 1656–1664 (2006).

95. Yu, J. *et al.* A transcriptional fingerprint of estrogen in human breast cancer predicts patient survival. *Neoplasia* 10, 79–88 (2008).

96. Lippman, M. E., Rae, J. M. & Chinnaiyan, A. M. An expression signature of estrogen-regulated genes predicts disease-free survival in tamoxifen-treated patients better than progesterone receptor status. *Trans. Am. Clin. Climatol. Assoc.* 119, 77–90 (2008).

References 86 and 94–96 examine the relationship between networks of estrogen-regulated genes and response to tamoxifen therapy.

97. Clark, G. M., McGuire, W. L., Hubay, C. A., Pearson, O. H. & Marshall, J. S. Progesterone receptors as a prognostic factor in Stage II breast cancer. *N. Engl. J. Med.* 309, 1343–1347 (1983).

98. Sorlie, T. *et al.* Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications. *Proc. Natl Acad. Sci. USA* 98, 10869–10874 (2001).

99. Sotiriou, C. *et al.* Breast cancer classification and prognosis based on gene expression profiles from a population-based study. *Proc. Natl Acad. Sci. USA* 100, 10393–10398 (2003).

100. Sorlie, T. *et al.* Repeated observation of breast tumor subtypes in independent gene expression data sets. *Proc. Natl Acad. Sci. USA* 100, 8418–8423 (2003).

101. Chin, K. *et al.* Genomic and transcriptional aberrations linked to breast cancer pathophysiologies. *Cancer Cell* 10, 529–541 (2006).

102. Dowsett, M. *et al.* Prognostic value of Ki67 expression after short-term presurgical endocrine therapy for primary breast cancer. *J. Natl Cancer Inst.* 99, 167–170 (2007). A neoadjuvant study providing the first definitive evidence that a marker of proliferation, Ki67, predicts responsiveness to endocrine therapy in patients.

103. Ellis, M. J. *et al.* Outcome prediction for estrogen receptor-positive breast cancer based on postneoadjuvant endocrine therapy tumor characteristics. *J. Natl Cancer Inst.* 100, 1380–1388 (2008).

104. Rhodes, D. R. *et al.* Molecular concepts analysis links tumors, pathways, mechanisms, and drugs. *Neoplasia* 9, 443–454 (2007).

105. Acharya, C. R. *et al.* Gene expression signatures, clinicopathological features, and individualized therapy in breast cancer. *JAMA* 299, 1574–1587 (2008).

106. Bild, A. H., Potti, A. & Nevins, J. R. Linking oncogenic pathways with therapeutic opportunities. *Nature Rev. Cancer* 6, 735–471 (2006).

107. van Agthoven, T. *et al.* Functional identification of genes causing estrogen independence of human breast cancer cells. *Breast Cancer Res. Treat.* 114, 23–30 (2009).

108. Dorssers, L. C. & Veldscholte, J. Identification of a novel breast-cancer-anti-estrogen-resistance (BCAR2) locus by cell-fusion-mediated gene transfer in human breast-cancer cells. *Int. J. Cancer* 72, 700–705 (1997).

109. Brinkman, A., van der Flier, S., Kok, E. M. & Dorssers, L. C. BCAR1, a human homologue of the adapter protein p130Cas, and antiestrogen resistance in breast cancer cells. *J. Natl Cancer Inst.* 92, 112–120 (2000).

110. Meijer, D., van Agthoven, T., Bosma, P. T., Nooter, K. & Dorssers, L. C. Functional screen for genes responsible for tamoxifen resistance in human breast cancer cells. Mol. Cancer Res. 4, 379–386 (2006). This paper and references 47, 51 and 108 document the first genetic screens aimed at identifying genes that can confer resistance to anti-oestrogens in culture.
111. van Agthoven, T. *et al.* Relevance of breast cancer antiestrogen resistance genes in human breast cancer progression and tamoxifen resistance. J. Clin. Oncol. 27, 542–529 (2009).
112. Meijer, D. *et al.* TSC22D1 and PSAP predict clinical outcome of tamoxifen treatment in patients with recurrent breast cancer. Breast Cancer Res. Treat. 113, 253–260 (2009).
113. Dorssers, L. C. *et al.* The prognostic value of BCAR1 in patients with primary breast cancer. Clin. Cancer Res. 10, 6194–6202 (2004).
114. Iorns, E. *et al.* Identification of CDK10 as an important determinant of resistance to endocrine therapy for breast cancer. Cancer Cell 13, 91–104 (2008). This publication describes the first RNA interference screen for anti-oestrogen resistance. It shows that decreased expression of CDK10 is associated with anti-oestrogen resistance in culture and in women treated with tamoxifen.
115. Iorns, E., Lord, C. J. & Ashworth, A. Parallel RNAi and compound screens identify the PDK1 pathway as a target for tamoxifen sensitization. Biochem. J. 417, 361–370 (2009).
116. Ta, H. O., Thomas, K. S., Schrecengost, R. S. & Bouton, A. H. A novel association between p130Cas and resistance to the chemotherapeutic drug adriamycin in human breast cancer cells. Cancer Res. 68, 8796–8804 (2008).
117. Kalish, L. H. *et al.* Deregulated cyclin D1 expression is associated with decreased efficacy of the selective epidermal growth factor receptor tyrosine kinase inhibitor gefitinib in head and neck squamous cell carcinoma cell lines. Clin. Cancer Res. 10, 7764–7774 (2004).
118. Planas-Silva, M. D. & Hamilton, K. N. Targeting c-Src kinase enhances tamoxifen’s inhibitory effect on cell growth by modulating expression of cell cycle and survival proteins. Cancer Chemother. Pharmacol. 60, 535–543 (2007).
119. Soni, S., Lin, B. T., August, A., Nicholson, R. I. & Kirsch, K. H. Expression of a phosphorylated p130(Cas) substrate domain attenuates the phosphatidylinositol 3-kinase/Akt survival pathway in tamoxifen resistant breast cancer cells. J. Cell. Biochem. 107, 364–375 (2009).
120. Kurokawa, H. *et al.* Inhibition of HER2/neu (erbB-2) and mitogen-activated protein kinases enhances tamoxifen action against HER2-overexpressing, tamoxifen-resistant breast cancer cells. Cancer Res. 60, 5887–5894 (2000).
121. Clark, A. S., West, K., Streicher, S. & Dennis, P. A. Constitutive and inducible Akt activity promotes resistance to chemotherapy, trastuzumab, or tamoxifen in breast cancer cells. Mol. Cancer Ther. 1, 707–717 (2002).
122. deGraffenried, L. A. *et al.* NF-kappa B inhibition markedly enhances sensitivity of resistant breast cancer tumor cells to tamoxifen. Ann. Oncol. 15, 885–890 (2004).
123. Boulay, A. *et al.* Dual inhibition of mTOR and estrogen receptor signaling in vitro induces cell death in models of breast cancer. Clin. Cancer Res. 11, 5319–5328 (2005).
124. Leary, A. F., Sirohi, B. & Johnston, S. R. Clinical trials update: endocrine and biological therapy combinations in the treatment of breast cancer. Breast Cancer Res. 9, 112 (2007).
125. Soucek, L. *et al.* Modelling Myc inhibition as a cancer therapy. Nature 455, 679–683 (2008).
126. Shapiro, G. I. Cyclin-dependent kinase pathways as targets for cancer treatment. J. Clin. Oncol. 24, 1770–1783 (2006).
127. Perissi, V. & Rosenfeld, M. G. Controlling nuclear receptors: the circular logic of cofactor cycles. Nature Rev. Mol. Cell Biol. 6, 542–554 (2005).
128. Couse, J. F. & Korach, K. S. Estrogen receptor null mice: what have we learned and where will they lead us. Endocr. Rev. 20, 358–417 (1999).
129. Strom, A. *et al.* Estrogen receptor β inhibits 17β-estradiol-stimulated proliferation of the breast cancer cell line T47D. Proc. Natl Acad. Sci. USA 101, 1566–1571 (2004).
130. Murphy, L. C., Niu, Y., Snell, L. & Watson, P. Phospho-serine-118 estrogen receptor-α expression is associated with better disease outcome in women treated with tamoxifen. Clin. Cancer Res. 10, 5902–5906 (2004).
131. Zhou, Y., Eppeberger-Castori, S., Eppeberger, U. & Benz, C. C. The NFκB pathway and endocrine-resistant breast cancer. Endocr. Relat. Cancer 12 (Suppl. 1), S37–S46 (2005).
132. Redmond, A. M. *et al.* Coassociation of estrogen receptor and p160 proteins predicts resistance to endocrine treatment; SRC-1 is an independent predictor of breast cancer recurrence. Clin. Cancer Res. 15, 2098–2106 (2009).
133. Harigopal, M. *et al.* Estrogen receptor co-activator (AIB1) protein expression by automated quantitative analysis (AQUA) in a breast cancer tissue microarray and association with patient outcome. Breast Cancer Res. Treat. 115, 77–85 (2009).
134. Green, A. R. *et al.* The prognostic significance of steroid receptor co-regulators in breast cancer: co-repressor NCOR2/SMRT is an independent indicator of poor outcome. Breast Cancer Res. Treat. 110, 427–437 (2008).
135. Arpino, G. *et al.* HER-2 amplification, HER-1 expression, and tamoxifen response in estrogen receptor-positive metastatic breast cancer: a southwest oncology group study. Clin. Cancer Res. 10, 5670–5676 (2004).
136. Creighton, C. J. *et al.* Insulin-like growth factor-I activates gene transcription programs strongly associated with poor breast cancer prognosis. J. Clin. Oncol. 26, 4078–4085 (2008).
137. Meijer, D. *et al.* Fibroblast growth factor receptor 4 predicts failure on tamoxifen therapy in patients with recurrent breast cancer. Endocr. Relat. Cancer 15, 101–111 (2008).
138. Bergqvist, J. *et al.* Activated ERK1/2 and phosphorylated oestrogen receptor α are associated with improved breast cancer survival in women treated with tamoxifen. Eur. J. Cancer 42, 1104–1112 (2006).
139. Kirkegaard, T. *et al.* AKT activation predicts outcome in breast cancer patients treated with tamoxifen. J. Pathol. 207, 139–146 (2005).
140. Tokunaga, E. *et al.* Activation of PI3K/Akt signaling and hormone resistance in breast cancer. Breast Cancer 13, 137–144 (2006).
141. Shoman, N. *et al.* Reduced PTEN expression predicts relapse in patients with breast carcinoma treated by tamoxifen. Mod. Pathol. 18, 250–259 (2005).
142. Millar, E. K. *et al.* Cyclin D1b protein expression in breast cancer is independent of cyclin D1a and associated with poor disease outcome. Oncogene 28, 1812–1820 (2009).
143. Span, P. N., Tjan-Heijnen, V. C., Manders, P., Beex, L. V. & Sweep, C. G. Cyclin-E is a strong predictor of endocrine therapy failure in human breast cancer. Oncogene 22, 4898–4904 (2003).
144. Denning, S. L., Nass, S. J., Dickson, R. B. & Trock, B. J. C-myc amplification in breast cancer: a meta-analysis of its occurrence and prognostic relevance. Br. J. Cancer 83, 1688–1695 (2000).
145. Pohl, G. *et al.* High p27Kip1 expression predicts superior relapse-free and overall survival for premenopausal women with early-stage breast cancer receiving adjuvant treatment with tamoxifen plus goserelin. J. Clin. Oncol. 21, 3594–3600 (2003).
146. Callagy, G. M. *et al.* Bcl-2 is a prognostic marker in breast cancer independently of the Nottingham Prognostic Index. Clin. Cancer Res. 12, 2468–2475 (2006).
147. Gasparini, G. *et al.* Expression of bcl-2 protein predicts efficacy of adjuvant treatments in operable node-positive breast cancer. Clin. Cancer Res. 1, 189–198 (1995).
148. Cannings, E. *et al.* Bad expression predicts outcome in patients treated with tamoxifen. Breast Cancer Res. Treat. 102, 173–179 (2007).
149. Ma, X. J. *et al.* A two-gene expression ratio predicts clinical outcome in breast cancer patients treated with tamoxifen. Cancer Cell 5, 607–616 (2004).
150. Paik, S. *et al.* A multigene assay to predict recurrence of tamoxifen-treated, node-negative breast cancer. N. Engl. J. Med. 351, 2817–2826 (2004).
151. Jansen, M. P. *et al.* Molecular classification of tamoxifen-resistant breast carcinomas by gene expression profiling. J. Clin. Oncol. 23, 732–740 (2005).

References 149–151 were the first studies to use clinical material to derive gene expression signatures that were specifically aimed at predicting response to endocrine therapy.

152. Wang, Y. *et al.* Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer. Lancet 365, 671–679 (2005).
153. Sotiriou, C. *et al.* Gene expression profiling in breast cancer: understanding the molecular basis of histologic grade to improve prognosis. J. Natl Cancer Inst. 98, 262–272 (2006).
154. Chanrion, M. *et al.* A gene expression signature that can predict the recurrence of tamoxifen-treated primary breast cancer. Clin. Cancer Res. 14, 1744–1752 (2008).
155. Harvell, D. M. *et al.* Estrogen regulated gene expression in response to neoadjuvant endocrine therapy of breast cancers: tamoxifen agonist effects dominate in the presence of an aromatase inhibitor. Breast Cancer Res. Treat. 112, 489–501 (2008).
156. Harvell, D. M. *et al.* Molecular signatures of neoadjuvant endocrine therapy for breast cancer: characteristics of response or intrinsic resistance. Breast Cancer Res. Treat. 112, 475–488 (2008).
157. Loi, S. *et al.* Predicting prognosis using molecular profiling in estrogen receptor-positive breast cancer treated with tamoxifen. BMC Genomics 9, 239 (2008).
158. Vendrell, J. A. *et al.* A candidate molecular signature associated with tamoxifen failure in primary breast cancer. Breast Cancer Res. 10, R88 (2008).

**Acknowledgements**

We are grateful to R. J. Daly, C. E. Caldon and A. J. Butt for helpful discussions and comments, and C. E. Caldon for expert assistance with the figures. Research in the authors’ laboratories is supported by the National Health and Medical Research Council of Australia, the Cancer Institute New South Wales, the Australian Cancer Research Foundation, the Petre Foundation and the RT Hall Trust.

**DATABASES**

National Cancer Institute Drug Dictionary: http://www.cancer.gov/drugdictionary/ adriamycin | erlotinib | everolimus | gefitinib | lapatinib | tamoxifen | temsirolimus

UniProtKB: http://www.uniprot.org AP1|BCAR1|EGFR|ERα|ERβ|ERBB2|FOXP3|GATA4|IRF1|NCOA3|PAK1|PAX2|PELP1|PR|PRMT1|SP1|STAT5B

**FURTHER INFORMATION**

Elizabeth A. Musgrove’s homepage: http://www.garvan.org.au/about-us/our-people/associate-professor-liz-musgrove

Robert L. Sutherland’s homepage: http://www.garvan.org.au/about-us/our-people/professor-rob-sutherland

ALL LINKS ARE ACTIVE IN THE ONLINE PDF
