
Innate Immunity  
http://ini.sagepub.com/  

---

Review: Lipopolysaccharide biosynthesis in *Pseudomonas aeruginosa*  
Jerry D. King, Dana Kocínková, Erin L. Westman and Joseph S. Lam  
*Innate Immunity* 2009 15: 261 originally published online 26 August 2009  
DOI: 10.1177/1753425909106436  

The online version of this article can be found at:  
http://ini.sagepub.com/content/15/5/261  

---

Published by:  
SAGE  
http://www.sagepublications.com  

On behalf of:  
International Endotoxin & Innate Immunity Society  

Additional services and information for *Innate Immunity* can be found at:  

- Email Alerts: http://ini.sagepub.com/cgi/alerts  
- Subscriptions: http://ini.sagepub.com/subscriptions  
- Reprints: http://www.sagepub.com/journalsReprints.nav  
- Permissions: http://www.sagepub.com/journalsPermissions.nav  
- Citations: http://ini.sagepub.com/content/15/5/261.refs.html  

>> Version of Record - Oct 6, 2009  
OnlineFirst Version of Record - Aug 26, 2009  
What is This?  

Downloaded from ini.sagepub.com at Scientific library of Moscow State University on February 10, 2014

Review

Lipopolysaccharide biosynthesis in *Pseudomonas aeruginosa*

Jerry D. King, Dana Kocínková, Erin L. Westman, Joseph S. Lam

*Department of Molecular and Cellular Biology, University of Guelph, Guelph, Ontario, Canada*

---

*Pseudomonas aeruginosa* causes serious nosocomial infections, and an important virulence factor produced by this organism is lipopolysaccharide (LPS). This review summarizes knowledge about biosynthesis of all three structural domains of LPS – lipid A, core oligosaccharide, and O polysaccharides. In addition, based on similarities with other bacterial species, this review proposes new hypothetical pathways for unstudied steps in the biosynthesis of *P. aeruginosa* LPS. Lipid A biosynthesis is discussed in relation to *Escherichia coli* and *Salmonella*, and the biosyntheses of core sugar precursors and core oligosaccharide are summarized. *Pseudomonas aeruginosa* attaches a Common Polysaccharide Antigen and O-Specific Antigen polysaccharides to lipid A-core. Both forms of O polysaccharide are discussed with respect to their independent synthesis mechanisms. Recent advances in understanding O-polysaccharide biosynthesis since the last major review on this subject, published nearly a decade ago, are highlighted. Since *P. aeruginosa* O polysaccharides contain unusual sugars, sugar-nucleotide biosynthesis pathways are reviewed in detail. Knowledge derived from detailed studies in the O5, O6 and O11 serotypes is applied to predict biosynthesis pathways of sugars in poorly-studied serotypes, especially O1, O4, and O13/O14. Although further work is required, a full understanding of LPS biosynthesis in *P. aeruginosa* is almost within reach.

**Keywords:** lipid A, lipopolysaccharide, O antigen, core, *Pseudomonas aeruginosa*

---

### PSEUDOMONAS AERUGINOSA

*Pseudomonas aeruginosa* is a Gram-negative, motile, rod-shaped bacterium, which is widely distributed in the environment. In the 1890s, it was isolated from the wounds of two patients in France; because of the characteristic blue-green pus associated with these infections, it was called *Bacillus pyocyaneus* ('blue pus bacillus'). Early reports from the 19th century had already associated this organism with hospital epidemics<sup>1</sup> and today it remains one of the major causes of nosocomial infections.<sup>2</sup> *P. aeruginosa* is an opportunistic pathogen and causes disease by exploiting defects in normal host defences. In humans, this pathogen can cause life-threatening infections in individuals who are compromised because of immune deficiencies, and those suffering from burn wounds, cancer or cystic fibrosis.<sup>3–5</sup>

The success of *P. aeruginosa* as a pathogen is due partly to its metabolic diversity but also to its arsenal of virulence factors (reviewed by Lyczak *et al.*<sup>6</sup>). Furthermore, once a *P. aeruginosa* infection is established, treatment can be difficult due to its high intrinsic antibiotic resistance. This innate resistance is due, in part, to a large number of multidrug efflux systems,<sup>7</sup> and to the low permeability of the *P. aeruginosa* outer membrane, which is estimated to be two orders of magnitude lower than the outer membrane of *Escherichia coli* for small hydrophilic compounds.<sup>8,9</sup> Further resistance to multiple drugs can be acquired in *P. aeruginosa* by step-by-step accumulation of multiple mutations that reduce drug uptake, increase efflux, or alter targets within the cell. Alone, these individual mutations may have modest effects on antibiotic resistance, but the cumulative effect can significantly reduce the efficacy of clinically important drugs without

---

Received 4 March 2009; Revised 21 April 2009; Accepted 24 April 2009

Correspondence to: Dr Joseph S. Lam, Department of Molecular and Cellular Biology, University of Guelph, Ontario N1G 2W1, Canada.  
Tel: +1 519 824 4120 ext 53823; Fax: +1 519 837 1802; E-mail: jlam@uoguelph.ca

Downloaded from [ini.sagepub.com](http://ini.sagepub.com) at Scientific library of Moscow State University on February 10, 2014

262 King, Kocínková, Westman, Lam

any need for the strain to acquire exogenous antibiotic-  
resistance enzymes.¹⁰ The emergence of multiple-drug  
resistant strains is a contributing factor to the poor  
prognoses associated with *P. aeruginosa* infections once  
they are established.¹¹ The biofilm mode of growth also  
contributes to *P. aeruginosa* resistance to antibiotics.  
Biofilms, defined as complex bacterial communities  
within an extracellular matrix, are more resistant than  
non-biofilm grown bacteria, because antibiotics may  
bind to the extracellular matrix, and have reduced  
activity in areas of biofilm that are characterised by  
poor oxygenation or low pH.¹²

### LIPOPOLYSACCHARIDE

An important virulence factor for *P. aeruginosa*  
is lipopolysaccharide (LPS), a complex glycolipid  
which is the major component of the outer leaflet  
of the outer membrane in Gram-negative bacteria.  
Lipopolysaccharide constitutes a physical barrier protecting  
the bacterium from host defences, mediates direct interactions  
with host cell receptors and antibiotics and, as endotoxin,  
is itself a potent signalling molecule which initiates some  
of the events leading to host tissue damage and much of the  
pathology associated with bacteraemia.

Due to the biological significance of this molecule, it  
has been the subject of intense study and a great deal is  
known about its biosynthesis. It is now nearly a decade  
since the last major review on *P. aeruginosa* LPS  
biosynthesis,¹³ which focused specifically on O polysaccharides;  
since that time, many significant advances have been made  
in our understanding of enzymes and pathways involved in  
the biosynthesis of this molecule in *P. aeruginosa*.  
The purpose of this review is to reflect the current knowledge  
of the biology and chemistry of LPS biosynthesis in *P. aeruginosa*.

#### Lipopolysaccharide has three domains

Lipopolysaccharide is often described as a molecule  
with three domains. The first domain is called lipid A: it  
contains a disaccharide backbone, to which are attached  
several fatty acid chains that anchor LPS into the outer  
membrane. In all wild-type strains, the lipid A is  
attached to a nine or ten-sugar, branched oligosaccharide  
known as the core. A proportion of the LPS molecules on  
the surface of any given cell has only these two domains.  
Such molecules are sometimes referred to as lipid  
A-core. The third LPS domain consists of a repetitive  
carbohydrate polymer, which is covalently attached to  
the core, and can be referred to as the O antigen, O  
polysaccharide or O chain. *Pseudomonas aeruginosa*

cells can simultaneously produce two types of O polysaccharide in the same cell, which are distinct structurally, serologically and according to the mechanisms of their biosynthesis. Historically, these have been called A-band and B-band O polysaccharides; however, because these are not intuitively meaningful terms, we propose the following nomenclature for these structures. As the B-band polysaccharides are the basis for the *P. aeruginosa* O-serotyping scheme, these structures will be called ‘O-Specific Antigens’, or ‘O Antigens’. The A-band O polysaccharide has previously been called ‘Common Polysaccharide Antigen’, or ‘Common O Polysaccharide’ and these terms usefully distinguish this structure from the O-Specific Antigens and also, for example, from common outer-membrane protein antigens.¹⁴

The Common Polysaccharide Antigen is a homopolymer of D-rhamnose (D-Rha). It is produced by the majority of, but not all, *P. aeruginosa* strains, and elicits a relatively weak antibody response. There is some heterogeneity in the rhamnan polymer lengths produced in one cell, but Common O-Polysaccharide chains are usually approximately 70 sugars long.

The O-Specific Antigens are heteropolymers in which different sugars are organised into repetitive O units. In contrast to the Common O Polysaccharide, the O-Specific Antigen polysaccharides are highly immunogenic, eliciting strong antibody responses. The nature and number of sugars in the O-Antigen repeating unit varies from strain to strain, and these are the structures recognised by O-specific antibodies. The International Antigenic Typing Scheme (IATS) for *P. aeruginosa* classifies strains into 20 major serotypes, O1 to O20, reflecting the diversity of O-Antigen structures within the species. In comparison to the Common O Polysaccharides, the O-Specific Antigen polymers are added to lipid A-core with a wide range of different chain lengths, some considerably longer than Common O-Polysaccharide chains, and some containing a single repeat unit.

Not all *P. aeruginosa* strains are O-serotypable, however, and it is common, especially in clinical isolates, for the ability to produce O-Antigen to be lost. Bacteria devoid of O Antigen are known as rough strains because, in some genera, mutations which abrogated O Antigen production result in ‘rough’ colony morphology, compared with the ‘smooth’ colonies of O Antigen-producing strains. The rough and smooth terminology is now used to refer to O-polysaccharide⁻ or O-polysaccharide⁺ strains regardless of colony morphology, and is sometimes even used to describe LPS molecules which either are or are not substituted with O-polysaccharides. For example, lipid A-core can be called ‘rough-type’ (R) LPS. O-Antigen mutants have also been described which only decorate lipid A-core with a single O unit, but do not produce O-Antigen polymers. Such strains are called

‘semi-rough’ (SR), and it is also common to refer to semi-rough LPS molecules (or core + 1 LPS), which normally make up a proportion of the LPS present on the surface of smooth strains.

The three-domain notion of LPS has been very successful because different properties of the molecule segregate more-or-less along the domain boundaries. For example, O polysaccharides and lipid A-core are synthesised by independent biosynthetic pathways, and the lipid A domain is responsible for the biological properties of LPS associated with its alternative name of endotoxin. In the following section, we briefly outline the biological functions of each domain; then, we shall describe the chemical structures and consider the biosynthesis of LPS, taking each domain in turn.

## The biological significance of specific LPS domains

### Role of lipid A in pathogenesis

Lipid A is the domain of LPS which mediates inflammatory response-induced endotoxicity; however, the extent of that toxicity varies widely between different bacterial species. *E. coli* and *Neisseria meningitidis* lipid-A structures each contain six 14- or 12-carbon lipids, and are generally considered to be optimal for eliciting the maximal inflammatory response via human Toll-like receptor 4 (TLR4). As described in detail below, the major lipid A produced by laboratory strains of *P. aeruginosa* differs from these optimal structures both in the number and length of acyl chains. Therefore, the inflammatory response elicited by *P. aeruginosa* lipid A would be expected to be less than that of enterobacterial LPS, and this is indeed the case. This being said, some clinical isolates of *P. aeruginosa*, from cystic fibrosis patients with severe pulmonary disease, synthesise a hyperacylated lipid A which induces an enhanced inflammatory response. In these cases, lipid A structures produced by the cystic fibrosis airway-adapted *P. aeruginosa* probably contribute to the severity of lung damage.

Many bacteria make modifications of their lipid A structures, in response to environmental stimuli, which increase resistance to the bactericidal effects of cationic antimicrobial peptides. These peptides are a diverse, wide-spread component of the innate immune system (reviewed by Hancock and Diamond). They are abundant both inside neutrophils and on mucosal surfaces and other sites of potential infection; while they contribute to host defences through a range of different mechanisms, some (for example, polymyxin B, human defensin, and cecropin) function directly by interaction with bacterial membranes. By modification of the patterns of lipid A acylation, or addition of positively-charged substituents to the phosphate groups, bacteria reduce the permeability of the membrane or the affinity of lipid A for these cationic peptides. *Pseudomonas aeruginosa* also modifies lipid A in response to environmental signals, resulting in greater resistance to some antimicrobial peptides and, in part, these modifications can be induced by exposure to such peptides.

#### Role of core in pathogenesis

Binding and internalisation of respiratory antigens by epithelial cells, followed by desquamation may be an important mechanism for clearing bacteria from the lung. Internalisation of *P. aeruginosa* by epithelial cells correlates with membrane expression of the cystic fibrosis transmembrane conductance regulator (CFTR) that is defective (for instance, the ΔF508 allele) in cystic fibrosis patients. When an airway epithelial cell line expressing a functional wild-type human CFTR was tested for ingestion of *P. aeruginosa*, the internalisation of *P. aeruginosa* cells could be inhibited by addition of exogenous LPS core oligosaccharides from a strain producing a complete core, but oligosaccharides from a strain with a truncated core did not inhibit. This indicated that the bacterial ligand for the wild-type CFTR receptor which mediates epithelial-cell internalisation of *P. aeruginosa* is core oligosaccharide.

*Pseudomonas aeruginosa* is the most common cause of bacterial keratitis associated with the use of contact lenses. It is able to enter and survive in corneal cells in an *in vitro* cell line and mouse or rabbit models. As with the epithelial cell line experiments described above, it appears that the bacterial ligand which mediates uptake into corneal cells is the core oligosaccharide. Strains with core structure defects were ingested more slowly than the control with complete core. Ingestion could again be inhibited by addition of exogenous core oligosaccharides, but only if the core was complete. Since the core oligosaccharide structures in these experiments were only characterised using sugar composition analyses, the core structure equating to the ligand responsible for interaction with the host cells is unknown.

Later, Zaidi and co-workers showed, in an animal model, that homozygous ΔF508 *cftr* mice were almost completely resistant to *P. aeruginosa* corneal infection. Thus, wild-type CFTR-mediated internalisation of *P. aeruginosa* by corneal epithelial cells is a mechanism which causes disease in experimental eye infection, while in the airways of humans or animals, CFTR-mediated *P. aeruginosa* uptake by epithelial cells appears to be protective, probably by enhancing clearance of bacteria.

#### Role of O polysaccharides in pathogenesis

O polysaccharides extend outward from the outer cell membrane, and are thus involved in many of the

264 King, Kocínková, Westman, Lam

interactions between the bacterium and the environment or host. During infections caused by *P. aeruginosa*, the O-Specific Antigen protects the bacteria from phagocytosis and confers resistance to complement-mediated killing.³² Whole cell agglutination tests suggest that long-chain O-Specific Antigen conceals the shorter Common O Polysaccharide on cells producing both forms of O polysaccharide,³³ and strains that express only the Common O Polysaccharide were not protected from lysis by complement.³²

When present in a host, *P. aeruginosa* is faced with immune defence systems producing a variety of reactive oxygen species, such as H₂O₂. A recent study showed that O polysaccharide protects against oxidative stress. The sensitivity to hydrogen peroxide of a *P. aeruginosa* *waaL* mutant, which cannot ligate O polysaccharide to the LPS core, was tested. This strain had a 4-fold reduction in survival when challenged with H₂O₂, compared to the wild type.³⁴ *Pseudomonas aeruginosa* *waaL* mutants in PAO1 and PA14 have also been shown to be defective in twitching and swimming motility.³⁵ Since these forms of motility are important for initial surface colonisation, O polysaccharides may play a role in surface attachment.

In the context of chronic *P. aeruginosa* infections, such as those common in cystic fibrosis patients, the Common Polysaccharide Antigen becomes the major LPS antigen over time, because there are apparently long-term selective pressures for loss of high molecular mass O-Specific Antigen.³⁶ The production of O Antigen has been restored to clinical isolates by transformation with a plasmid encoding all or select genes from the O-Antigen biosynthesis locus, suggesting that loss of this structure was due to mutations in the biosynthetic genes rather than due to regulatory changes.³⁷ O Antigen is highly immunogenic, and elicits a strong antibody response from the infected host. In chronic *P. aeruginosa* infections, it is common for strains to reduce O-Antigen production and then to lose it altogether. These changes may be driven by the selective pressures on the bacteria to evade the anti-O-Antigen immune response. Another selective pressure for loss of O Antigen may be exposure of the bacteria to repeated antibiotic therapy, since defects in synthesis of either O polysaccharide reduce ionic binding of the aminoglycoside antibiotic gentamicin to the cell surface and the strains are, therefore, less prone to antibiotic-induced killing.³⁸ This being said, there have been no reports of selective pressure for loss of Common Polysaccharide Antigen in chronic infections.

A longitudinal study of lung infections in cystic fibrosis patients showed that the appearance of Common Polysaccharide Antigen-specific antibodies in host serum correlates with decreased pulmonary function and extended duration of *P. aeruginosa* infection.³³

The exopolysaccharide alginate can also protect against anti-Common Polysaccharide Antigen antibody-mediated opsonophagocytosis.³⁹ *In vitro* assays with monolayers of cultured human bronchial epithelial cells showed that a defined Common Polysaccharide Antigen-deficient *P. aeruginosa* mutant had a defect in adherence. The numbers of mutant cells attached to the epithelial cells were lower than those of wild type, and the mutant was attached as isolated bacteria rather than microcolonies like the wild-type. This suggests that, despite being apparently masked by the long-chain O-Antigen, the Common O polysaccharide plays a role in adherence of *P. aeruginosa* to epithelial cells under certain conditions.⁴⁰

Infection studies using animal models have shown that O polysaccharides are required for virulence. In a burned-mouse model, the LD₅₀ for an O Antigen-deficient mutant was at least 1000-fold higher than that of the wild-type strain. This was attributed to the inability of the mutant to colonise burned skin effectively and to avoid serum killing.⁴¹ The status of Common O Polysaccharide production in this strain was not tested, but visualisation of the results from LPS analysis by SDS-PAGE and silver staining in this report indicated that the strain used in this study did produce Common Polysaccharide Antigen.

In several studies, mutants producing truncated core structures, that are defective as acceptors for either O polysaccharide, have been used to assess the role of O polysaccharides in virulence. Some of the mutations have pleiotropic effects, but the loss of O-polysaccharide production likely contributed significantly to the decrease in virulence observed. For example, a *galU* mutant that cannot produce UDP-D-Glc was significantly attenuated in a corneal infection model. The same mutant had higher LD₅₀ than wild-type cells in an acute pneumonia model, but still caused mortality and severe pneumonia despite minimal systemic spread.⁴² A separate study⁴³ used a neonatal mouse model to compare the virulence of *P. aeruginosa* PAO1 to that of a PAO1 *algC* mutant, defective in phosphomannomutase (PMM) and phosphoglucomutase (PGM) activity. The *algC* gene is required for the synthesis of UDP-D-Glc, GDP-D-Man, and dTDP-L-Rha, which are precursors for LPS core, Common Polysaccharide Antigen, or the two exopolysaccharides important for biofilm formation, alginate and Psl. Due to the defect in LPS core, the *algC* mutant is unable to produce LPS core with either O polysaccharide.⁴⁴ In an *in vitro* binding assay, the *algC* mutant was significantly less adherent to human respiratory epithelial cells, transformed tracheal epithelial cells, or to mouse adenoma cells. When mice were challenged with the *algC* mutant, the bacteria were readily cleared from the respiratory tract and the bacteraemia and mortality rates were significantly lower than those with PAO1.⁴³

LIPID A

**Lipid A structures in P. aeruginosa**

Lipoplysaccharide is present on the bacterial cell surface as a heterogeneous mixture of related structures, and much of this heterogeneity is contributed by variations in the lipid A moiety. While relatively little work has been done to characterise the biosynthetic steps for lipid A in *P. aeruginosa*, the structures themselves have been extensively studied (reviewed by Knirel *et al.*⁴⁵). The backbone of *P. aeruginosa* lipid A consists of doubly-phosphorylated diglucosamine (phosphate→4′-β-d-GlcNⅡ-(1 → 6)-α-d-GlcNⅠ-1 → phosphate) and the lipid A produced by most laboratory strains in standard media, has a basic penta- or hexa-acylated form (Fig. 1A).⁴⁶ In the hexa-acyl form, which comprises about 25% of lipid A in these strains, the diglucosamine backbone is symmetrically acylated, each sugar having an N-linked 12:0(3-OH) [(R)-3-hydroxylauroyl] group at the 2 position, an O-linked 10:0(3-OH) [(R)-3-hydroxydecanoyl] substituent at the 3 position and a secondary 12:0 (lauroyl) chain. The secondary lauroyl chains are non-stoichiometrically 2-hydroxylated (Fig. 1, green). In about 75% of lipid A prepared from laboratory strains, the 3 position of GlcNⅠ is unsubstituted, and this constitutes the penta-acyl form (Fig. 1A, red).

When lipid A from clinical isolates has been characterised, novel structures were found, arising from the loss and/or addition of particular structural elements (Fig. 1B). Some of these modifications are regulated in response to environmental stimuli, allowing *P. aeruginosa* to modulate its interactions with a host organism, or to resist the bactericidal effects of antimicrobial agents.²² In isolates of *P. aeruginosa* from non-cystic fibrosis infections (such as blood infection), a penta-acyl lipid A was found lacking the secondary acyl chain on GlcNI (Fig. 1B, red secondary chain).¹⁹,²² *Pseudomonas aeruginosa* isolated from cystic fibrosis infections exhibits lipid A with six or seven acyl chains, similar to the laboratory penta- and hexa-acyl structures but with an additional secondary palmitoyl group (16:0) in an ester linkage to the 3-hydroxylauroyl at C3 on GlcNⅡ (Fig. 1B, blue).¹⁹,²²,⁴⁷ Non-stoichiometric addition of 4-amino-4-deoxy-L-arabinose (L-Ara4N) at either or both lipid-A phosphates has also been observed in clinical isolates as well as in laboratory strains grown under particular conditions, such as low magnesium (Fig. 1B, blue).⁴⁷–⁴⁹

**Biosynthesis of lipid A in P. aeruginosa**

The biosynthesis of lipid A has been studied in detail for enteric bacteria, particularly *E. coli*, and this pathway is assumed to be generally conserved in *P. aeruginosa*.

---

Fig. 1. Lipid A structures. (A) Typical lipid A produced by laboratory-adapted strains in standard laboratory media. The 3-hydroxydecanoyl chain at the 3-position is absent in approximately 75% of LPS molecules. Secondary acyl chains can exhibit 2-hydroxylation. (B) Clinical isolates of *P. aeruginosa* commonly produce lipid A with various modifications of the basic structure shown in (A). These include loss of secondary acylation at the 2-position, secondary palmitoylation at position 3′, and addition of L-Ara4N to either of the phosphates. (C) The hexa-acyl lipid A structure of *E. coli*.

266 King, Kocínková, Westman, Lam

This assumption is based largely on the identification of homologues of the *E. coli* genes in the *P. aeruginosa* genome (Table 1), but the majority of lipid A biosynthetic steps have not been directly investigated. In the following section, we concentrate on steps in *P. aeruginosa* lipid A biosynthesis for which experimental data exist. The reader is referred to two excellent reviews ${ }^{50,51}$ for the general scheme of lipid A biosynthesis.

Transfer of primary acyl chains: *lpxA* and *lpxD*

The lengths of the primary *O*- or *N*-linked lipid chains differ between bacterial species (in *E. coli* they are all 14:0(3-OH), whereas *P. aeruginosa* has 10:0(3-OH) and 12:0(3-OH) groups). The distribution of these fatty acids on lipid A is determined by the species-specific hydrocarbon-length preferences of their respective *O*-acyltransferase (*LpxA*) ${ }^{52}$ and the *N*-acyltransferase (*LpxD*) ${ }^{53}$ enzymes.

The first step in lipid A biosynthesis is the reversible 3-*O*-acylation of UDP-*d*-GlcNAc (1)

catalysed by LpxA (Fig. 2). By making use of the anticipated preference of *P. aeruginosa* LpxA for the shorter chain lengths, compared with the *E. coli* enzyme, the *P. aeruginosa lpxA* gene was identified in a 3-hydroxydecanoyl acyltransferase screen of a PAO1 library expressed in *E. coli*. ${ }^{52}$ The *P. aeruginosa* gene was then cloned and shown to complement an *E. coli lpxA* temperature-sensitive mutation, restoring viability to the cell when grown at the non-permissive temperature, but it caused incorporation of 3-hydroxydecanoic acid [10:0(3-OH)] rather than the 3-hydroxymyristoic acid [14:0(3-OH)] usually found in *E. coli* lipid A. LpxA enzymes use acyl carrier protein (ACP)-linked fatty acids as substrates; in a transferase assay, the *P. aeruginosa* enzyme showed a >1000-fold preference for 3-hydroxydecanoyl-ACP, which is the reverse of the *E. coli* LpxA preference. ${ }^{52}$ These enzymes exquisitely discriminate between different fatty chain lengths, using what have been called 'precise hydrocarbon rulers', and their chain-length preferences can be interchanged by single amino acid substitutions. ${ }^{54}$

Table 1. Homologues of lipid-A biosynthesis genes found in the PAO1 genome. Sequence conservation relates to alignments with the *E. coli* K-12, or *B. bronchiseptica* RB 50 sequences

| *E. coli/P.* | Related proteins | Proposed function | Key |
| --- | --- | --- | --- |
| *aeruginosa* gene | (% identity) |  | reference* |
| *lpxA/PA3644* | 54% *E. coli* LpxA | 3-Hydroxydecanoyl-[acyl carrier protein] → UDP-GlcNAc 3-*O*-acyltransferase | 52 |
| *lpxC/PA4406* | 57% *E. coli* LpxC | UDP-3-*O*-acyl-GlcNAc 2-*N*-deacetylase | 53 |
| *lpxD/PA3646* | 51% *E. coli* LpxD | 3-Hydroxylauroyl-[acyl carrier protein] → UDP-3-*O*-acyl-GlcN 2-*N*-acyltransferase | 53 |
| *lpxH/PA1792* | 47% *E. coli* LpxH | UDP-diacyl-GlcN pyrophosphatase |  |
| *lpxB/PA3643* | 53% *E. coli* LpxB | Lipid A precursor disaccharide synthase |  |
| *lpxK/PA2981* | 45% *E. coli* LpxK | Lipid A precursor disaccharide kinase |  |
| *lpxL PA3242* | 47% *E. coli* LpxL | Lauroyl-[acyl carrier protein] → lipid IV${}_{\mathrm{A}}$ secondary lauroyl chain transferase | 59 |
| *lpxL PA0011* | 29% *E. coli* LpxL | Secondary lauroyl chain 2-hydroxylase | 59 |
| *lpxO PA4512* | 60% LpxO *S. enterica* sv. Typhimurium | Secondary lauroyl chain 2-hydroxylase |  |
| *lpxO PA1936* | 50% LpxO *S. enterica* sv. Typhimurium | Lipid A GlcN${}^{\mathrm{I}}$ 3-*O*-deacylase |  |
| *pagL/PA4661* | 29% *B. bronchiseptica* PagL | Lipid A GlcN${}^{\mathrm{I}}$ 3-*O*-deacylase | 65 |
| *arnA/PA3554* | 69% *E. coli* ArnA (K-12) | Undecaprenyl-phosphate-L-Ara4N synthesis | 23 |
| *arnB/PA3552* | 62% *E. coli* ArnB | Undecaprenyl-phosphate-L-Ara4N synthesis |  |
| *arnC/PA3553* | 64% *E. coli* ArnC | Undecaprenyl-phosphate-L-Ara4N synthesis |  |
| *arnD/PA3555* | 60% *E. coli* ArnD | Undecaprenyl-phosphate-L-Ara4N synthesis |  |
| *arnE/PA3557* | 54% *E. coli* ArnE | Undecaprenyl-phosphate-L-Ara4N export |  |
| *arnF/PA3558* | 36% *E. coli* ArnF | Undecaprenyl-phosphate-L-Ara4N export |  |
| *arnT/PA3556* | 41% *E. coli* ArnT | Undecaprenyl-phosphate-L-Ara4N → lipid A L-Ara4N transferase |  |
| *ugd/PA3559* | 28% *E. coli* Ugd | UDP-Glc 6-dehydrogenase (undecaprenyl-phosphate-linked L-Ara4N synthesis) | 69 |

*References are cited if they describe experimental investigation of the *P. aeruginosa* gene.
The assay data described above, from ectopically expressed *P. aeruginosa* LpxA, mimicked the substrate preference observed in an assay previously conducted to assess the abilities of *P. aeruginosa* crude extracts to modify UDP-GlcNAc with ACP-linked acyl chains of different lengths.⁵³ The same approach was used to judge the specificity of the reaction in which the *N*-linked acyl chain of lipid A is attached, by incubating crude cell extracts with *in situ*-generated UDP-3-*O*-[(R)-3-hydroxymyristoyl]-GlcN as the acyl acceptor. The compound added into these incubations was UDP-3-*O*-[(R)-3-hydroxy-myristoyl]-GlcNAc, and the authors relied on the ability of crude extracts to de-acetylate this compound (*via* the LpxC-catalysed reaction) to generate the free amine substrate. In this case, *P. aeruginosa* extracts showed a preference for 3-hydroxylauroyl-ACP, in contrast to *E. coli* extracts, which used 3-hydroxy-myristoyl-ACP most efficiently.⁵³ The enzyme catalysing this reaction in *P. aeruginosa* is presumably the *P. aeruginosa* homologue of *E. coli* LpxD, encoded by PA3646.

### Addition of secondary lauroyl chains: *lpxL1* and *lpxL2*

In *E. coli* and *S. enterica* sv. Typhimurium, lipid A biosynthesis cannot be completed without addition of the first core sugar, 3-deoxy-D-manno-oct-2-ulosonic acid (Kdo). The *E. coli* acyl transferase enzymes are called LpxL (HtrB) and LpxM (MsbB), and they catalyse addition of the secondary lauroyl group and then the myristoyl group, respectively, to complete synthesis of Kdo₂-lipid A.⁵⁵,⁵⁶ The *E. coli* enzymes have a Kdo-containing substrate specificity.⁵⁵–⁵⁷ The enzymes which catalyse transfer of the two secondary lauroyl chains in *P. aeruginosa* apparently do not share the same specificity because even if Kdo biosynthesis is inhibited (by an antibacterial agent), *P. aeruginosa* produces lipid A without Kdo, but fully acylated.⁵⁸ The lack of an absolute Kdo-requirement was confirmed by *in vitro* assays using dilute cytoplasmic extracts or membrane preparations from *P. aeruginosa* with lauroyl-ACP (12:0-ACP) and *E. coli* lipid IVₐ as the donor and acceptor substrates, respectively. In a similar experiment, soluble *E. coli* extracts are unable to lauroylate lipid IVₐ, but added two lauroyl groups to Kdo₂-lipid IVₐ. In contrast, each of the *P. aeruginosa* extracts catalysed the addition of a single lauroyl group, and was able to do so more efficiently without Kdo present in the substrate.⁵⁹

The *P. aeruginosa* gene or genes which encode this activity have not been determined, but there are two candidates in the PAO1 genome, PA3242 (*lpxL1*) and PA0011 (*lpxL2*). These genes encode putative acyltransferases homologous to LpxL and LpxM from *E. coli*, but both are more closely related to the lauroyltransferase LpxL. Presumably, these enzymes each transfer one of the secondary lauroyl groups found in most mature *P. aeruginosa* lipid A molecules (8) (Table 1). It is not known why only one secondary lauroyl group was transferred by the *P. aeruginosa* extracts. It could be that the lauroyltransferase to act second has a preference for its native *P. aeruginosa*

### Biosynthesis of the lipid IVₐ analogue in *P. aeruginosa*: *lpxH*, *lpxB* and *lpxK*

Lipid IVₐ is an important intermediate lipid structure in the biosynthesis of *E. coli* and *Salmonella enterica* sv. Typhimurium lipid A.⁵⁰ To our knowledge, none of the three steps leading to biosynthesis of the lipid IVₐ analogue have been investigated experimentally in *P. aeruginosa*, but the nucleotidase (LpxH), disaccharide synthase (LpxB) and kinase (LpxK) reactions are presumed to be catalysed by the *P. aeruginosa* homologues of the *E. coli* enzymes in a conserved manner (Fig. 3; Table 1). The end-product of these reactions is the *P. aeruginosa* analogue of lipid IVₐ (7) (Fig. 3).

![Chemical structures](attachment://chemical_structures.png)

**Fig. 2.** Early steps in the lipid A biosynthesis pathway. The starting material UDP-D-GlcNAc (1) is a common precursor for synthesis of LPS sugars, lipid A and peptidoglycan. Activity relating to all three steps has been detected in *P. aeruginosa* lysates.⁵²,⁵³ ACP, acyl carrier protein; 3-OH-C₁₀ (*R*)-3-hydroxydecanoate [10:0(3-OH)]; 3-OH-C₁₂, (*R*)-3-hydroxylaurate [12:0(3-OH)].
268 King, Kocínková, Westman, Lam

![Chemical Structure](chemical_structure.png)

Fig. 3. Proposed pathway for the biosynthesis of the *P. aeruginosa* lipid IV<sub>A</sub> analogue. This scheme is predicted by analogy to the well-described pathway for lipid IV<sub>A</sub> biosynthesis in *E. coli*.<sup>50</sup> Well-conserved homologues for the *E. coli* genes are present in the PAO1 genome. P<sub>i</sub>, phosphate.

substrate in which the primary acyl chains are shorter than those in *E. coli*-derived substrate used in these experiments.<sup>59</sup>

Some non-cystic fibrosis clinical isolates of *P. aeruginosa*, for example from blood infections, lack one of the secondary acyl chains usually present in isolates from other environments (Fig. 1B, red secondary chain).<sup>19</sup> It is unknown whether this structure arises because the function of one of the lauroyltransferase enzymes is down-regulated or lost and the acyl chain was never added to these molecules, or whether this acyl chain is perhaps removed after completion of lipid A synthesis by a novel deacylase enzyme similar to PagL (see below). Both scenarios are conceptually possible.

**Hydroxylation of secondary lauroyl chains: *lpxO1* and *lpxO2***

Analyses of *P. aeruginosa* lipid A structures reveal that both secondary lauroyl chains can be 2-hydroxylated. This is a feature of *P. aeruginosa* lipid A which is not found in *E. coli*, but 2-hydroxylation of one secondary myristoyl chain is observed in *S. enterica* sv. Typhimurium lipid A.<sup>60</sup> The enzyme responsible for this in *S. enterica* sv. Typhimurium is LpxO.<sup>61,62</sup>

When *lpxO* is inactivated in this organism, the (*S*)-2-hydroxymyristoyl [14:0(2-OH)] group is lost<sup>63</sup> and when *lpxO* is introduced to *E. coli* on a plasmid, its lipid A gains 2-hydroxylation of myristate.<sup>61</sup> *Salmonella enterica* sv. Typhimurium LpxO is a membrane protein which incorporates <sup>18</sup>O from <sup>18</sup>O<sub>2</sub> into lipid A<sup>61</sup> and, *in vitro*, membranes containing LpxO catalysed the hydroxylation of Kdo<sub>2</sub>-hexa-acyl-lipid A in the presence of Fe<sup>2+</sup>, O<sub>2</sub>, and α-ketoglutarate.<sup>62</sup>

Genes encoding LpxO homologues are found in bacteria which produce 2-hydroxylacyl-containing lipid A and two homologues are present in the PAO1 genome – PA4512 and PA1936.<sup>61</sup> Probably the enzymes encoded by these genes each hydroxylate one of the two acyl chains exhibiting this modification, but neither gene has yet been investigated experimentally.

**Modifications of lipid A in *P. aeruginosa***

In this section, we describe alterations to lipid A structures which occur after transport of LPS across the inner membrane. For a general description of lipid-A modification systems, the reader is referred to an excellent review by Trent *et al.*<sup>21</sup>

Lipoplysaccharide biosynthesis in Pseudomonas aeruginosa 269

Deacylation: $pagL$

As described above, the majority of lipid A molecules produced by laboratory strains of *P. aeruginosa* lack acylation at the 3 position of GlcN${}^{\mathrm{I}}$ (Fig. 1, red primary acyl chain). From work in *S. enterica* sv. Typhimurium, it was known that such species can be synthesised by removal of the 3-acyl chain in a reaction catalysed by an outer membrane enzyme known as PagL. ${ }^{64}$ The $pagL$ gene in *P. aeruginosa* shares little sequence identity with the *S. enterica* sv. Typhimurium $pagL$ but was identified using an iterative BLAST methodology. ${ }^{65}$ As judged by mass spectrometry (MS) and gas chromatography (GC), a *P. aeruginosa* $pagL$ mutant lost the predominant penta-acylated lipid A of its parental strain (PAO1) and expressed instead a hexa-acylated lipid A. ${ }^{66}$ When over-expressed in *E. coli*, the *P. aeruginosa* PagL caused lipid A to be deacylated, indicating that *P. aeruginosa* PagL can remove the longer 14:0(3-OH) chains of *E. coli* lipid A as well as the 10:0(3-OH) chains of its native species. PagL from *P. aeruginosa* has been crystallised and the crystal structure revealed that this enzyme has a serine hydrolase-type catalytic triad consisting of His126, Ser128 and Glu140 (amino acid numbering is for the mature protein). ${ }^{67}$

Regulation of PagL activity

Mass spectrometrical and GC analysis of lipid-A structures indicated that, in acute infection and environmental isolates of *P. aeruginosa*, the 3-$O$-deacylase activity attributed to PagL was regulated by magnesium concentration and temperature, being less active at low [Mg${}^{2+}$] and low temperatures. ${ }^{66}$ In contrast to these observations, when PagL protein expression levels were analysed in laboratory-adapted *P. aeruginosa* stains, a lack of response to Mg${}^{2+}$ concentration was observed. ${ }^{65}$ This may indicate that *P. aeruginosa* strains lose Mg${}^{2+}$-regulation of PagL activity as a part of their adaptation to the laboratory, or that these researchers did not test a wide-enough range of Mg${}^{2+}$ concentrations. A third possibility is that PagL activity is regulated by some mechanism other than control of protein expression levels.

Ernst and co-workers ${ }^{66}$ showed that a third of *P. aeruginosa* isolates that they examined from cystic fibrosis patients with severe lung disease produced hexa- and hepta-acylated lipid A, which appeared to have arisen by loss of 3-$O$-deacylase activity. The hypothesis that these lipid A structures were due to loss of PagL function was confirmed by complementation of this phenotype by introduction of $pagL$ on a plasmid.

Addition of L-Ara4N: the $arn$ locus

Partial masking of the negative charges of the lipid A phosphate substituents by addition of positively charged-moieties such as L-Ara4N is one mechanism by which bacteria reduce their susceptibility to the bactericidal properties of cationic antimicrobial peptides. The pathway for addition of L-Ara4N to lipid A is well understood from studies in *E. coli* and *S. enterica* sv. Typhimurium. It requires the products of the $ugd$ gene and the $arnBCADTEF$ gene cluster; the pathway has been reviewed. ${ }^{68}$ Briefly, L-Ara4N is transferred to lipid A from an undecaprenyl-phosphate lipid carrier on the periplasmic face of the inner membrane, in a reaction catalysed by ArnT. UDP-Glc 6-dehydrogenase (Ugd), ArnA, ArnB, ArnC, and ArnD catalyse biosynthesis of the undecaprenol-phosphate-linked L-Ara4N in the cytoplasm, and ArnE and ArnF are required for its efficient transport across the inner membrane. Homologues of the *E. coli* genes are all found in a single cluster in *P. aeruginosa* – PA3552-PA3559 (or $arnB$-$ugd$) (Table 1, Fig. S1). To our knowledge, none of the *P. aeruginosa* genes have been investigated experimentally except for $ugd$ which was confirmed to encode a UDP-glucose 6-dehydrogenase. PA3552 is one of two $ugd$ genes in PAO1 but the polymyxin B-susceptible phenotype specific to the PA3552 mutant was characteristic of strains defective for aminoarabinoylation of lipid A suggesting that the $ugd$ in the $arn$ operon has a specific role in lipid A modification. ${ }^{69}$

Regulation of L-Ara4N addition to lipid A: $phoP$, $phoQ$, $pmrA$ and $pmrB$

The expression of $arn$ genes in *P. aeruginosa* is controlled by two two-component regulatory systems, PmrA–PmrB and PhoP–PhoQ. Both of these systems respond to Mg${}^{2+}$-limiting conditions and gene transcript analysis shows that both also regulate expression of the $arn$ operon. ${ }^{70}$ With use of quantitative PCR and an $arnB$:$lux$ reporter fusion, McPhee and co-workers ${ }^{70}$ showed that $arn$ gene expression is induced by low Mg${}^{2+}$ in a manner which is partially dependent on PhoP. $arn$ genes are also induced by a number of cationic antimicrobial peptides, including polymyxin B, in a manner which was partially dependent on PmrA–PmrB system. ${ }^{23}$ The role of the PmrA–PmrB system in resistance to cationic antimicrobial peptides has also been examined at the level of lipid A structures: single nucleotide mutations in $pmrB$ can cause constitutive activation of this two-component system resulting in resistance to polymyxin B and other cationic peptides. MS analysis of purified lipid A from these mutants showed that they added L-Ara4N to lipid A in conditions where the parental strain does not do so. ${ }^{49}$ While it has not been directly demonstrated in *P. aeruginosa*, the addition of L-Ara4N is very likely an important mechanism through which PmrA–PmrB mediates cationic peptide resistance as this modification correlates well with resistance across a broad range of bacterial species. ${ }^{71-75}$

270 King, Kocínková, Westman, Lam

Transfer of secondary palmitoyl chain: PagP-like activity

Laboratory adapted *P. aeruginosa* strains add a secondary palmitoyl (16:0) fatty acid to lipid A at the 3′-position (Fig. 1B, blue) when they are grown in low Mg²⁺ concentrations, a condition which activates PhoP–PhoQ.²² However, in a survey of *P. aeruginosa* strains obtained from different sources, there was a perfect correlation between isolation of strains from a cystic fibrosis patient lung infection and their stable palmitoylation of lipid A, irrespective of Mg²⁺ concentration.⁴⁷ In other bacteria, palmitoylation of lipid A is catalysed by homologues of the *E. coli* and *S. enterica* sv. Typhimurium enzyme PagP.⁶⁰,⁷⁶,⁷⁷ These are outer membrane proteins which catalyse the transfer of a palmitoyl chain to lipid A using acyl groups derived from phospholipids.⁷⁸,⁷⁹

PagP activity is generally regulated in response to environmental stimuli⁷⁶,⁷⁷ and confers greater resistance to cationic antimicrobial peptides in some bacteria,⁶⁰,⁷⁶ but not all.⁷⁷ It is required for full virulence in certain infection models.⁷⁶,⁷⁷ Differences in regulation and expression of PagP, even between the closely related Bordetellae (*B. pertussis*, *B. parapertussis* and *B. bronchiseptica*), suggest that specific bacteria-host relationships apply diverse selective pressures on the function of this enzyme.⁸⁰,⁸¹ This point is underlined by the contrast between complete inactivation of *pagP* in the acute, human respiratory tract pathogen *B. pertussis*, with apparently rapid selection for constitutive PagP-like activity in *P. aeruginosa* which is typically associated with chronic lung infections in cystic fibrosis.

Thus far, no PagP homologue has been identified in *P. aeruginosa*, but its existence is inferred from the regulated palmitoylation of *P. aeruginosa* lipid A (which takes place at the 3′-position, just as in PagP-mediated palmitoylation in *B. bronchiseptica*) and also indicated by reports of an unpublished observation of PagP-like acyltransferase activity detected in *P. aeruginosa* membranes.⁵¹

The inner core structure is similar to that of *E. coli* and *S. enterica* sv. Typhimurium, composed of two residues of Kdo (Kdoᴵ and Kdoᴵᴵ) and two residues of L-glycero-D-manno-heptose (Hepᴵ and Hepᴵᴵ). Both heptose residues serve as phosphorylation sites (Fig. 4). Among Gram-negative bacteria, *P. aeruginosa* has the most phosphorylated core. Phosphorylation of LPS has been associated with intrinsic resistance to antibiotics, such as novobiocin.⁸² Ionic interactions between negatively-charged phosphate substituents on heptoses and divalent cations may stabilise the outer membrane by cross-linking LPS molecules,⁸³ and this mechanism may be particularly important for *P. aeruginosa* because it is hypersusceptible to lysis by agents that chelate divalent cations (such as EDTA).⁸⁴,⁸⁵ This hypersusceptibility to metal chelators appears to be somehow related to the high phosphate content of *P. aeruginosa* LPS.⁸⁶ *Pseudomonas aeruginosa* core has three phosphorylation sites: positions 2 and 4 of Hepᴵ and position 6 of Hepᴵᴵ (Fig. 4).⁸⁷⁻⁹⁰ In addition, the phosphorylation position 2 of Hepᴵ has been found to be non-stoichiometrically occupied by diphosphoethanolamine (Fig. 4).⁹⁰ Another modification in the inner core structure is the stoichiometric presence of an O-carbamoyl substituent at position 7 of Hepᴵᴵ.⁹¹

The *P. aeruginosa* outer core is composed of three sugar types: D-glucose (Glc), L-rhamnose (L-Rha), and 2-amino-2-deoxy-D-galactose (D-galactosamine, GalN). The GalN is further modified by alanine (Ala). *Pseudomonas aeruginosa* cells simultaneously produce two structurally similar outer-core glycoforms, which are present in comparable amounts. These are glycoform 1₍O₋₎, which is never substituted with O polysaccharide and glycoform 2₍O₊₎ which is the core glycoform which can be further substituted by O polysaccharides. Glycoform 1₍O₋₎ has sometimes been called the ‘uncapped’ core glycoform, and glycoform 2₍O₊₎ is also known as the ‘capped’ core glycoform. Both glycoforms share the same inner core structure and four sugars, GalN and Glcᴵ–Glcᴵᴵᴵ, of the outer core (Fig. 4). The glycoforms differ in the position of L-Rha and only glycoform 1₍O₋₎ has a fourth glucose, Glcᴵⱽ.⁸⁸,⁸⁹,⁹²⁻⁹⁵ Glycoform 2₍O₊₎ has L-Rhaᴮ (1 → 3)-linked to Glcᴵ, whereas the glycoform 1₍O₋₎ has L-Rhaᴬ (1 → 6)-linked to Glcᴵᴵ (Fig. 4). No core glycoform with two Rha residues has been found, suggesting that the attachment of either L-Rha residue precludes attachment of the other.

## CORE OLIGOSACCHARIDE

### Structure of core oligosaccharide

The central domain of LPS that is interposed between lipid A and O polysaccharide is the core oligosaccharide.

### Core oligosaccharide structure of wild-type *P. aeruginosa* PAO1

The core oligosaccharide of PAO1 is the most-studied among *P. aeruginosa* strains. As with other species, the *P. aeruginosa* core structure is commonly divided, conceptually, into two parts – inner and outer core.

### Strain-dependent variations in core structures

Core structures have been characterised for a number of *P. aeruginosa* strains; while the basic structure is conserved, variation is seen among the peripheral structural features, including the terminal Glcᴵⱽ and many of the non-carbohydrate substituents.

Glycoform 1

$\beta$-D-Glc${}^{\text{IV}}$ 1 $\cdots \rightarrow$ 2 L-Rha${}^{A}$ 1 $\rightarrow$ 6 D-Glc${}^{\text{II}}$

$\begin{array}{c}
\uparrow \\
\text { migA* }
\end{array}$

$\begin{array}{c}
\downarrow \\
4
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { wapH* }
\end{array}$

$\begin{array}{c}
\downarrow \\
6
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { waaP* }
\end{array}$

$\begin{array}{c}
\downarrow \\
4
\end{array}$

D-Glc${}^{\text{III}}$ 1 $\rightarrow$ 6 $\beta$-D-Glc${}^{\text{I}}$ 1 $\rightarrow$ 3 D-GalN 1 $\rightarrow$ 3 Hep${}^{\text{II}}$ 1 $\rightarrow$ 3 Hep${}^{\text{I}}$ 1 $\rightarrow$ 5 Kdo${}^{\text{I}}$ 2 $\rightarrow$

$\begin{array}{c}
\uparrow \\
2
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { wapG }
\end{array}$

$\begin{array}{c}
\uparrow \\
7
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { waaF* }
\end{array}$

$\begin{array}{c}
\uparrow \\
2
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { waaC* }
\end{array}$

$\begin{array}{c}
\uparrow \\
2
\end{array}$

L-Ala

Cm

Kdo${}^{\text{II}}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

EtnP${}_{i}$

lipid A

---

Glycoform 2

O polysaccharide $\cdots \rightarrow$ 3 L-Rha${}^{B}$ D-Glc${}^{\text{II}}$

$\begin{array}{c}
\downarrow \\
1
\end{array}$

$\begin{array}{c}
\downarrow \\
3
\end{array}$

$\begin{array}{c}
\downarrow \\
1
\end{array}$

$\begin{array}{c}
\downarrow \\
4
\end{array}$

$\begin{array}{c}
\downarrow \\
6
\end{array}$

$\begin{array}{c}
\downarrow \\
4
\end{array}$

$\begin{array}{c}
\leftarrow \text { wapR* } \\
\leftarrow \text { wapH* } \\
\leftarrow \text { waaP* }
\end{array}$

D-Glc${}^{\text{III}}$ 1 $\rightarrow$ 6 $\beta$-D-Glc${}^{\text{I}}$ 1 $\rightarrow$ 3 D-GalN 1 $\rightarrow$ 3 Hep${}^{\text{II}}$ 1 $\rightarrow$ 3 Hep${}^{\text{I}}$ 1 $\rightarrow$ 5 Kdo${}^{\text{I}}$ 2 $\rightarrow$

$\begin{array}{c}
\uparrow \\
2
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { wapG }
\end{array}$

$\begin{array}{c}
\uparrow \\
7
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { waaF* }
\end{array}$

$\begin{array}{c}
\uparrow \\
2
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { waaC* }
\end{array}$

$\begin{array}{c}
\uparrow \\
2
\end{array}$

L-Ala

Cm

Kdo${}^{\text{II}}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

$\begin{array}{c}
\uparrow \\
\text { P}_{i}
\end{array}$

EtnP${}_{i}$

lipid A

---

Fig. 4. Structures of core glycoform 1 and glycoform 2. Sugars have the $\alpha$ configuration unless otherwise indicated. Dotted lines indicate non-stoichiometric substitutions, or substitutions not present in all strains. Genes involved or putatively involved in enzymatic steps are indicated and an asterisk denotes the existence of some experimental evidence for the functions of these genes. Ala, alanine; Cm, carbamoyl; Etn, ethanolamine; GalN, 2-amino-2-deoxy-galactose; Glc, glucose; Hep, $L$-glycero-$D$-manno-heptose; Kdo, 3-deoxy-$D$-manno-oct-2-ulosonic acid; Rha, rhamnose.

---

### Phosphorylation

It is very commonly observed that core phosphorylation patterns differ between *P. aeruginosa* strains. Some of this variation is probably due to genuine strain-to-strain differences, but it should be borne in mind that chemical treatment of LPS prior to analysis may cause underestimation of the true phosphorylation state. Acid hydrolysis is a common procedure used prior to structural analysis of LPS oligosaccharide domains, because it removes lipids and makes the molecules readily water-soluble. However, some pyrophosphate bonds may be hydrolysed in the process. ${}^{94}$ Each phosphorylation site of the inner-core heptose residues may be occupied by monophosphate, diphosphate or even triphosphate substitutions. ${}^{90,94,96}$ A rare phosphate substitution was found at position 4 of Hep${}^{\text{II}}$ in a *P. aeruginosa* cystic fibrosis isolate (the strain 2192). ${}^{88}$ Most of the analyzed inner cores of *P. aeruginosa* also contain phosphoethanolamine or diphosphoethanolamine, present in non-stoichiometric amounts. ${}^{94}$ Early studies on phosphorylation of LPS showed that even ethanolamine triphosphate might be present in LPS. ${}^{97,98}$ In some strains, no ethanolamine could be detected in the core; these include strain 170021 (serotype O4), an *algC* mutant (derived from serotype O3) which has a truncation within the outer core, a *wbjE* mutant (derived from

---

### Addition of Glc${}^{\text{IV}}$

Not all strains add the fourth glucose (Glc${}^{\text{IV}}$) to glycoform 1${}_{(O-)}$. In studies which determined outer core structures of different *P. aeruginosa* strains by NMR and MS, most of the analyzed serotypes had three Glc residues in the glycoform 1${}_{(O-)}$ core (serotypes: O1, O3, O4, O6, O9, O11, O12, O13, O15, O17, O19). ${}^{88,89,94,100}$ The O10 serotype contained only the four-Glc glycoform 1${}_{(O-)}$ core and in the serotype O8 and O2 strains, the major glycoform contained four Glc residues and a three-Glc core was present as a minor species. ${}^{94}$ These data from full structural determination of core structures correlate well with earlier immunochemical characterisation of core oligosaccharides by ELISA (enzyme-linked immunosorbent assay) and Western blotting using an anti-glycoform 1${}_{(O-)}$ core monoclonal antibody (5C-101) that recognizes the terminal Glc${}^{\text{IV}}$. ${}^{101}$ Several IATS reference strains reacted with mAb 5C-101, including O2, O5, O7, O8, O10, O16, O18, O19, and O20; while LPS from O1, O3, O4, O6, O9, O11, O12, O13, O14, O15, O17 reference strains were not recognised by this antibody. ${}^{101}$

272 King, Kocínková, Westman, Lam

### Acetylation

Acetylation of outer-core sugars is relatively common in *P. aeruginosa* strains and up to five *O*-acetyl groups have been found. Similar to phosphate and ethanolamine pyrophosphates, *O*-acetyl groups might be lost during mild acid delipidation of LPS prior to structural analysis; thus, the *O*-acetylation of core may be underestimated. The extent of core acetylation detected in different IATS reference strains was as follows: no acetylation was found in O4, O5, and O10; one acetylation site in O8, O12, and O13; at least one acetylation site is present in O6; one or two acetylation sites in O1 and O19; three acetylation sites in O3; four acetylation sites in a cystic fibrosis isolate; and up to 5 acetylation sites in O9, O11, O15, and O17. Although the *O*-acetyl groups are randomly distributed among sugars of the outer core, the most frequently acetylated residue is the terminal Rha<sup>A</sup> of glycoform 1<sub>(O-)</sub>. *O*-Acetylation occurs to a much higher extent when Rha<sup>A</sup> is not substituted with Glc<sup>IV</sup> or if Rha<sup>B</sup> in glycoform 2 is not substituted with O Antigen, for example in clinical isolates that have lost O Antigen expression. Unlike the wild type, PAO1, some strains that have truncated outer core such as *wapR* and *rmlC* mutants or a *migA rmd* double mutant, were found to have an *O*-acetyl group on terminal Glc<sup>II</sup> residue. The functions of these genes with respect to core biosynthesis are discussed below.

The *wbjE* mutant of strain PA103 (serotype O11) produced various truncated forms of outer core, some of which had acetylated GalN, instead of the *N*-linked alanyl group. Similarly, the *P. aeruginosa* H4 strain that produces a truncated core, also had the terminal GalN acetylated rather than modified with the alanyl group.

### Biosynthesis of core sugars

#### Biosynthesis of D-glucose (D-Glc): *glk*, *algC* and *galU*

The predominant sugar in the outer core oligosaccharide is D-Glc. In *E. coli* and other bacteria, glucose is transported as glucose-6-phosphate (10) (Fig. 5). Otherwise, glucose can be converted to glucose-6-phosphate by Glk, a glucokinase. The *glk* gene function was demonstrated by measurements of Glk enzyme activity in cell lysates from a *glk* mutant and from complemented strains harbouring *glk* on plasmids. The next step in this pathway is conversion of glucose-6-phosphate to glucose-1-phosphate (11) catalyzed by AlgC (Fig. 5). As mentioned earlier, the *algC* gene encodes a bifunctional enzyme that has phosphoglucomutase (PGM) and phosphomannomutase (PMM) activities. The PGM activity converts glucose-6-phosphate to glucose-1-phosphate, which is a common intermediate between the biosynthetic pathways for D-Glc and L-Rha. Phosphoglucomutase activity could not be detected in crude extracts of the *P. aeruginosa algC* mutant, and the *algC* gene cloned from *P. aeruginosa* complemented a *pgm* mutant of *E. coli*, restoring its ability to ferment galactose, which is dependent on the presence of PGM activity. The *algC* mutants of *P. aeruginosa* PAO1 (serotype O5) and PAC1R (serotype O3) produced truncated core structures missing all Glc and Rha residues as determined by NMR and MS.

Conversion of Glc-1-phosphate to UDP-D-Glc, the proposed nucleotide-activated precursor for core Glc

![Diagram](https://i.imgur.com/yourimageurl.png)

Fig. 5. Proposed UDP-D-glucose and dTDP-L-rhamnose biosynthesis pathways. The phosphoglucomutase activity of AlgC is required for the biosynthesis of both UDP-D-Glc (12) and dTDP-L-Rha (16). Pi, phosphate. 10, Glc-6-P<sub>i</sub>; 11, Glc-1-P<sub>i</sub>; 13, dTDP-D-Glc; 14, dTDP-6-deoxy-D-xylo-4-hexulose; 15, dTDP-6-deoxy-L-lyxo-4-hexose.

residues, is catalyzed by GalU, a UDP-D-glucose pyrophosphorylase (Fig. 5). A *galU* mutant of PA103 (serogroup O11) was constructed and showed truncation in the core oligosaccharide and absence of O polysaccharide as judged by silver staining. By NMR analysis, the *galU* mutant core oligosaccharide was shown to lack all Glc and Rha residues. Transforming the mutants with the *galU* gene in trans restored O-polysaccharide production, but did not restore the wild-type mobility of lipid A-core bands in SDS-PAGE for reasons that are unknown.

Biosynthesis of L-rhamnose (L-Rha): *rmlA*, *rmlB*, *rmlC* and *rmlD*

Synthesis of dTDP-L-Rha, the core L-Rha precursor, has been characterised at the biochemical level in other bacteria, such as *S.* enterica sv. Typhimurium. Accordingly, conversion of Glc-1-phosphate and dTTP to dTDP-L-Rha requires four reaction steps catalyzed by RmlA (glucose-1-phosphate thymidylyltransferase), RmlB (dTDP-D-glucose 4,6-dehydratase), RmlC (dTDP-6-deoxy-D-xylo-4-hexulose 3′,5′-epimerase) and RmlD (dTDP-6-deoxy-L-lyxo-4-hexulose reductase; Fig. 5). The *P.* aeruginosa genome contains *rmlB*, *rmlD*, *rmlA* and *rmlC* in this order in the PA5161–PA5164 gene cluster (Table 2, Fig. S1). The three-dimensional structure of RmlA from *P.* aeruginosa as well as the enzymatic mechanism of this protein has been determined. There are clear structural relationships between RmlA and SpsA, a glycosyltransferase from *Bacillus subtilis* and the N-terminal domain of GlmU, N-acetylglucosamine-1-phosphate uridyltransferase. Based on the contacts with substrates and absolute sequence conservation amongst sugar nucleotidyltransferases, several catalytically important residues have been identified (Arg15, Lys25, Asp110, Lys162 and Asp225). *Pseudomonas aeruginosa* RmlA exhibits feedback inhibition by dTDP-L-Rha, the final product of the RmlABCD pathway.

The structure of *P.* aeruginosa RmlC in complex with product and product mimics has been recently solved. Although RmlC enzymes from different organisms have considerable diversity in sequence, they seem to share a common enzyme mechanism. Dong and co-workers suggested that the order of the double epimerisation (C3 and C5) of the substrate (dTDP-6-deoxy-D-xylo-4-hexulose; 14) catalysed by RmlC of *P.* aeruginosa is first C5 followed by C3. The structural and biochemical data were consistent with a mechanism in which His65 and Tyr134 of *P.* aeruginosa RmlC are catalytic residues involved in the epimerisation steps. The role of *rmlBDAC* in LPS biosynthesis in *P.* aeruginosa was confirmed by genetic experiments.

**Table 2. Genes involved or putatively involved in the biosynthesis of core oligosaccharide sugars**

| Gene          | Related proteins (% identity)           | Proposed/demonstrated function                          | Key reference* |
|---------------|-----------------------------------------|--------------------------------------------------------|----------------|
| *eno/PA3635*  | 76% *E. coli* Eno                       | Enolase                                                |                |
| *pyrG/PA3637* | –                                       | CTP synthase                                           | 115            |
| *kdsA/PA3636* | –                                       | Kdo-8-phosphate synthase                               | 115            |
| *kdsB/PA2979* | 54% *E. coli* KdsB                      | CMP-Kdo synthetase                                     |                |
| *kdsC/PA4458* | 48% *E. coli* KdsC                      | Kdo-8-phosphate phosphatase                            |                |
| *kdsD/PA4457* | 54% *E. coli* KdsD                      | D-Arabinose 5-phosphate isomerase                      |                |
| *gmhA/PA4425* | 41% *E. coli* GmhA                      | Sedoheptulose 7-phosphate isomerase                    |                |
| *gmhB/PA0006* | 35% *E. coli* GmhB                      | D-α,β-D-heptose 1,7-bisphosphate phosphatase           |                |
| *hldE/PA4996* | 57% *E. coli* HldE                      | D-β-D-heptose 7-phosphate kinase/                      |                |
|               |                                          | D-β-D-heptose 1-phosphate adenylyltransferase          |                |
| *hldD/PA3337* | 53% *E. coli* HldD                      | ADP-D-β-D-heptose epimerase                            |                |
| *glk/PA3193*  | –                                       | Glucokinase                                            | 104            |
| *algC/PA5322* | –                                       | Phosphoglucomutase/phosphomannomutase                  | 44             |
| *galU/PA2023* | –                                       | UDP-D-glucose pyrophosphorylase                        | 105            |
| *rmlA/PA5163* | –                                       | Glucose-1-phosphate thymidylyltransferase              | 108            |
| *rmlB/PA5161* | 61% *S. enterica* RmlB                  | dTDP-D-glucose 4,6-dehydratase                         |                |
| *rmlC/PA5164* | –                                       | dTDP-6-deoxy-D-xylo-4-hexulose 3′,5′-epimerase         | 112            |
| *rmlD/PA5162* | 52% *S. enterica* RmlD                  | dTDP-6-deoxy-L-lyxo-4-hexulose reductase               |                |

*References are cited if they describe experimental investigation of the *P.* aeruginosa gene.

A non-polar mutation of *rmlC* in *P. aeruginosa* abrogates the production of complete glycoform 1\(_{(O-)}\) core (not recognised by the 5C-101 antibody) as well as the addition of O polysaccharides to glycoform 2 (not recognised by the anti-O polysaccharide antibodies).<sup>113</sup> NMR and MS analyses of *rmlC* mutants showed complete absence of Rha residues within the outer core.<sup>102,113</sup>

### Biosynthesis of Kdo: *kdsD*, *kdsA*, *kdsC* and *kdsB*

In *E. coli*, the biosynthesis of CMP-Kdo, the precursor for Kdo residues in LPS, involves four sequentially-acting enzymes (Fig. 6): KdsD, a D-arabinose 5-phosphate isomerase; KdsA, a 3-deoxy-D-manno-octulosonate 8-phosphate (Kdo 8-P) synthase that utilises D-arabinose 5-phosphate (A5P; 17) and phosphoenolpyruvate (PEP) as substrates; KdsC, a Kdo 8-phosphate phosphatase; and KdsB, a cytidine 5'-monophosphate-Kdo synthetase (reviewed by Raetz<sup>114</sup>).

Our group showed that in *P. aeruginosa*, PA3636 encodes a functional homologue of KdsA.<sup>115</sup> This *kdsA* gene (PA3636) was able to rescue growth of *S. enterica* sv. Typhimurium strain with a temperature-sensitive mutation in *kdsA* at 42°C. Moreover, the crude cell lysate from an *E. coli* strain overexpressing *kdsA* from *P. aeruginosa* catalysed the formation of Kdo-8-phosphate from arabinose-5-phosphate and phosphoenolpyruvate (PEP).<sup>115</sup> The *kdsA* gene is located within a gene cluster containing *pyrG* and *eno*. Enolase (Eno) is structurally conserved in organisms which metabolise sugars and catalyses the formation of PEP from phosphoglycerate during glycolysis.<sup>116</sup> PyrG (CTP synthase) catalyzes the transfer of ammonia to UTP to form cytidine triphosphate (CTP)<sup>117</sup> that is required for the CMP-Kdo synthesis step.<sup>118</sup> The *pyrG* gene of *P. aeruginosa* was able to complement growth of the *E. coli pyrG* mutant that was auxotrophic for cytidine, on a minimal medium lacking cytidine.

A KdsB homologue is present in *P. aeruginosa*, encoded by PA2979. Its function has not been proven, but it shares 54% identity with characterized *E. coli* KdsB. KdsC and KdsD homologues are encoded by adjacent ORFs in the *P. aeruginosa* genome, PA4458 and PA4457, respectively. These two genes from *P. aeruginosa* have not been characterised, but their products share 48% and 54% identity to the characterised *E. coli* homologues.

### Biosynthesis of L-glycero-D-manno-heptose (Hep): *gmhA*, *hldE*, *gmhB* and *hldD*

In *E. coli*, ADP-L-glycero-β-D-manno-heptose (ADP-L-D-Hep) is the substrate for heptosyltransferase reactions.<sup>119</sup> The initial substrate for ADP-L-D-Hep biosynthesis is D-sedoheptulose-7-phosphate (Fig. 7; 21) that is an intermediate in the pentose phosphate pathway.<sup>120</sup> Four enzymes catalyse the biosynthesis of ADP-L-D-Hep from D-sedoheptulose-7-phosphate (Fig. 7): (i) GmhA, a sedoheptulose-7-phosphate isomerase; (ii) HldE (formerly RfaE), a bifunctional enzyme that functions as a D-D-heptose-7-phosphate kinase and a D-D-heptose 1-phosphate adenyltransferase; (iii) GmhB, a D-D-heptose 1,7-bisphosphate phosphatase; and (iv) HldD (formerly RfaD), an ADP-D-β-D-heptose epimerase (Fig. 7).<sup>121,122</sup>

![Figure 6](https://i.imgur.com/yourimageurl.png)

**Fig. 6.** Proposed CMP-Kdo biosynthesis pathway. P\(_i\), phosphate. PEP, phosphoenolpyruvate. 17, D-arabinose-5-P\(_i\); 18, 3-deoxy-D-manno-oct-2-ulosonate 8-P\(_i\).

None of the enzymes of the ADP-L-D-Hep pathway has been characterised in *P. aeruginosa*, but homologues of the ADP-L-D-heptose synthesis genes from *E. coli* are scattered though the *P. aeruginosa* genome (Table 2).

### Location of transferase genes involved in the core oligosaccharide biosynthesis

Most of the transferase genes involved in assembly of the core are located in a large gene cluster that contains 17 ORFs (PA4996–PA5012) (Fig. S1). The 5′-end of the cluster is terminated by PA4996/*hldE*, which is likely involved in the biosynthesis of heptose (as mentioned in the previous biosynthetic section). The locus also contains two genes (*msbA* and *waaL*) involved in LPS transport and ligation of O polysaccharide to core (see below). Organisation and sequence (77–100% amino acid identity) of the core gene cluster is well conserved among different, sequenced *P. aeruginosa* strains. This is not unexpected, since the composition of the core is also conserved among various *P. aeruginosa* strains.

*Pseudomonas aeruginosa* mutants lacking inner core Hep or phosphates have never been isolated or constructed, suggesting that Hep-linked phosphates are essential for *P. aeruginosa* viability. Thus, transferase genes involved in the assembly of inner core biosynthesis have been identified by cross-complementation experiments of *Salmonella* or *E. coli* mutants instead of creating knock-outs in *P. aeruginosa*.

### Glycosyltransferase-encoding genes

Numerous glycosyltransferases are required for core biosynthesis (Table 3). Glycosyltransferases have been classified into 91 different families on the basis of BLAST analysis and they are listed in the regularly updated CAZy database (<http://www.cazy.org>),<sup>123</sup> and the same three-dimensional fold is expected to occur within each of the families. Glycosyltransferases use either an inverting or a retaining mechanism, in which the stereochemistry of the substrate sugar's anomeric bond is inverted or retained in the transfer reaction. Frequently, members of a given GT family have the same type of mechanism so that, in theory, the CAZy family classification is useful for discriminating the possible functions of uncharacterised glycosyltransferases. However, there can be exceptions and, in at least one example, a glycosyltransferase that possessed the closest homology to inverting GT-2 family was found to be mechanistically a retaining glycosyltransferase,<sup>124</sup> showing that GT family classification does not perfectly predict glycosyltransferase mechanism.

#### Kdo transferase: *waaA*

The biochemical activity of the Kdo transferase has not been characterised in *P. aeruginosa*, but PA4988 encodes a homologue of WaaA, which catalyzes the sequential addition of two Kdo sugars to a molecule of lipid A in *E. coli*.<sup>125</sup> PA4988 is close to, but outside of, the large core biosynthesis gene cluster and the PA4988 gene product shares 55% identity to *E. coli* WaaA. While the *E. coli* WaaA contains a transmembrane domain,<sup>126</sup> no transmembrane domain was predicted for the *P. aeruginosa* homologue, by topology prediction programs such as TMHMM or SOSUI.<sup>127,128</sup> Whether this reflects a genuine difference between the *E. coli* and *P. aeruginosa* WaaA proteins is unknown. PA4988 is classified as a member of GT-30 family, members of which are predicted to use an inverting mechanism. This is consistent with the glycosidic linkages found in the core structure.

#### Heptosyltransferases: *waaC* and *waaF*

*Pseudomonas aeruginosa* core oligosaccharide contains heptose residues that have identical linkages to Hep<sup>I</sup> and

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

**Fig. 7.** Proposed ADP-L-glycero-β-D-manno-heptose biosynthesis pathway. P<sub>i</sub>, phosphate. 22, d-glycero-d-manno-heptose-7-P<sub>i</sub>; 23, d-glycero-d-manno-heptose-1,7-diphosphate; 24, d-glycero-β-d-manno-heptose-1-P<sub>i</sub>; 25, ADP-d-glycero-β-d-manno-heptose.

Table 3. Genes involved or putatively involved in core biosynthesis

| Gene           | Related proteins/domains (% identity)                          | Proposed/demonstrated function                                      | Key reference* |
|----------------|-------------------------------------------------------------|--------------------------------------------------------------------|----------------|
| waaA/PA4988    | 55% *E. coli* WaaA                                          | Glycosyltransferase (GT-30) of Kdo                                 |                |
| waaC/PA5011    | –                                                         | Glycosyltransferase (GT-9) of Hep<sup>I</sup>                      | 131            |
| waaF/PA5012    | –                                                         | Glycosyltransferase (GT-9) of Hep<sup>II</sup>                     | 131            |
| migA/PA0705    | –                                                         | Glycosyltransferase (GT-2) of Rha<sup>A</sup>                       | 102            |
| wapR/PA5000    | –                                                         | Glycosyltransferase (GT-2) of Rha<sup>B</sup>                       | 102            |
| wapG/PA5010    | 51% *E. coli* WaaG                                         | Glycosyltransferase (GT-4) possibly of GalN                        |                |
| wapH/PA5004    | 32% *E. coli* WaaG                                         | Glycosyltransferase (GT-4) possibly of Glc<sup>II</sup>             | 40             |
| PA5001         | 24% WaaG GT domain                                         | Glycosyltransferase (GT-4)                                         |                |
| wapQ/PA5007    | –                                                         | Heptose kinase                                                     | 82             |
| wapP/PA5008    | –                                                         | Heptose kinase                                                     | 82             |
| waaP/PA5009    | –                                                         | Heptose kinase: position 4 of Hep<sup>I</sup>                      | 141            |
| PA5006         | 30% APH/ChoK kinase domain                                 | Kinase                                                              |                |
|                | 28% LPS kinase domain                                      |                                                                    |                |
| PA4998         | 25% APH/ChoK kinase domain                                 | Kinase                                                              |                |
|                | 55% *Pseudomonas putida* Ttg8                              |                                                                    |                |
| wapO/PA5005    | 31% *Streptomyces clavuligerus* CmcH                       | Carbamoyltransferase                                               |                |
| PA5002         | 24% PIG-L domain                                           | Unknown                                                            |                |
| PA5003         | 26% *S. enterica* sv. Typhimurium                         | Unknown                                                            |                |
|                | Mig-14                                                     | Unknown                                                            |                |

Glycosyltransferase (GT) family classification is taken from the CAZy database.<sup>123</sup>

*References are cited if they describe experimental investigation of the *P. aeruginosa* gene.

Hep<sup>II</sup> present in *Salmonella* core oligosaccharide. In *Salmonella*, these residues are transferred by two enzymes, heptosyltransferase I and heptosyltransferase II, encoded by the *waaC* and *waaF* genes, respectively.<sup>129,130</sup> The *P. aeruginosa* *waaC* (PA5011) and *waaF* (PA5012) possess 56% and 54% amino acid identity, respectively, to their *Salmonella* homologues and complemented *S. enterica* sv. Typhimurium *waaC* and *waaF* mutants.<sup>131</sup> The complemented strains regained the ability to produce O polysaccharide and lost sensitivity to phage FFM phage, which is specific for deep-rough *Salmonella* strains (having LPS with severe defect in core oligosaccharide). Attempts to construct knockout mutants of *waaC* and *waaF* in *P. aeruginosa* were unsuccessful, despite using a method that has proven to be effective for making mutants in other *P. aeruginosa* genes. This indicates that these genes may be essential for cellular viability.<sup>131</sup> Despite carrying out similar functions, *P. aeruginosa* WaaC and WaaF have a limited level of identity to each other (24%). Both enzymes are classified as GT-9 family members and are, therefore, predicted to have an inverting mechanism of catalysis. This is consistent with the core oligosaccharide structure.

Possible GalN-transferase: *wapG*

WapG, encoded by PA5010, shares 51% identity with WaaG, which transfers the first sugar of the outer core in *E. coli*, the D-Glc which is (1 → 3)-linked to Hep<sup>II</sup>.<sup>132</sup> In *P. aeruginosa*, the first outer core sugar is the (1 → 3)-linked N-(L-alanyl)-D-GalN. Because *wapG* is the closest *waaG* homologue in the *P. aeruginosa* genome, it has been suggested that WapG is responsible for addition of the GalN residue. There are also conserved *wapG* homologues in *P. fluorescens*, *P. stutzeri* and *P. syringae* genomes, supporting its hypothetical role as a GalN transferase, since the GalN residues are also present in the outer cores of these three *Pseudomonas* species.<sup>133–135</sup> Although many different strategies were tried, a *wapG* mutant in *P. aeruginosa* was not obtained (M.J. Matewish and J.S. Lam, unpublished observations). Therefore, there is so far no biochemical or genetic evidence for the proposed function of WapG in outer core biosynthesis.

The structure of the sugar-nucleotide precursor of GalN in the core is not known. In *Klebsiella pneumoniae*, the core contains a similar aminohexose, 2-amino-2-deoxy-D-glucose, (GlcN). Since UDP-D-GlcN was not found in cell lysates, it is believed that the *K. pneumoniae* GlcN residue is derived from an initial transfer of D-GlcNAc catalysed by WabH which uses UDP-D-GlcNAc as substrate.<sup>136</sup> This sugar is then subsequently deacetylated, on the nascent core oligosaccharide, in a reaction catalysed by the WabN enzyme.<sup>137</sup> A similar scheme is conceptually possible for the first outer core sugar in *P. aeruginosa*; however, it should be noted that, although WapG is a homologue of *K. pneumoniae* WabH (23% identity), this is much lower sequence conservation than that shared with *E. coli* WaaG (51%) and thus far we have been unable

to identify a *wabN* homologue in the *P. aeruginosa* genome. In any case, WapG is classified as a member of CAZy family GT-4 and the predicted retaining mechanism would be consistent with the proposed role of adding the α-D-GalN residue from a likely nucleotide-α-D-sugar precursor.

### Possible Glc<sup>II</sup>-transferase: *wapH*

WapH, encoded by PA5004, is another core glycosyltransferase. It shares 32% identity with WaaG of *E. coli* and is a member of the GT-4 family of retaining glycosyltransferases. A non-polar knockout mutant of *wapH* produced incomplete core oligosaccharide showing that WapH participates in LPS core assembly.<sup>40</sup> The mutant did not synthesize glycoform 1<sub>(O−)</sub>, because the LPS core was not recognized by the 5C-101 antibody. Inactivation of *wapH* also influenced synthesis of the core glycoform 2<sub>(O+)</sub>, which was shown by the complete loss of Common Polysaccharide Antigen and low molecular mass forms of the O-Specific Antigen from LPS, including the lipid A-core + 1 form. However, the mutant was not completely devoid of O polysaccharide and still produced some high molecular mass O Antigen. Based upon the SDS-PAGE migration of *wapH* LPS bands in comparison with LPS from wild-type and other truncated core mutants, it was suggested that the *wapH* gene might code for a glycosyltransferase that adds Glc<sup>II</sup> in (1 → 4)-linkage to *N*-(L-alanyl)-galactosamine. Absence of Glc<sup>II</sup> would preclude addition of L-Rha<sup>A</sup> and Glc<sup>IV</sup>, and may make the growing oligosaccharide a weak acceptor for addition of L-Rha<sup>B</sup> which is necessary for attachment of O polysaccharides. The predicted retaining mechanism of WapH would be in agreement with transfer of Glc<sup>II</sup> to the core by WapH. Another piece of evidence that indirectly supports the proposed role of WapH in *P. aeruginosa* core biosynthesis is the absence of close *wapH* homologues in the *P. stutzeri* genome, since the core of a *P. stutzeri* strain does not contain (1 → 4)-linked D-glucose.<sup>134</sup> The core structure of the *wapH* mutant has not been determined by NMR and MS, so the proposed WapH function remains hypothetical.

### L-Rhamnosyltransferases: *migA* and *wapR*

Two genes encoding putative rhamnosyltransferases, *wapR*, and *migA*, are involved in the outer core oligosaccharide biosynthesis. They share 35% identity with each other and are both members of the CAZy family GT-2. The predicted inverting mechanism based on this CAZy classification is consistent with the use of dTDP-β-L-Rha as sugar donor to form either the α-L-Rha<sup>A</sup> or α-L-Rha<sup>B</sup> in *P. aeruginosa* core. MigA is thought to catalyse formation of the (1 → 6)-linkage of L-Rha<sup>A</sup> in core glycoform 1<sub>(O−)</sub> and WapR to catalyse formation of the (1 → 3)-linkage of L-Rha<sup>B</sup> in glycoform 2<sub>(O+)</sub>. Their function was characterized by genetic experiments and substantiated by elucidation of the chemical structure of core prepared from mutant strains. Interruption of the *migA* gene led to the truncation of core glycoform 1<sub>(O−)</sub> and the core was not recognized by mAb 5C-101.<sup>102,138</sup> NMR and MS analyses of LPS from *migA* mutants showed the presence of core glycoform 2<sub>(O+)</sub> (with Rha<sup>B</sup>), ruling out the possibility that MigA catalyses formation of the (1 → 3)-bond of Rha<sup>B</sup> or addition of any other core glycoform 2<sub>(O+)</sub> sugars (GalN, Glc<sup>I</sup>, Glc<sup>II</sup>, Glc<sup>III</sup>). Neither anti-O-polysaccharide or anti core + 1 monoclonal antibodies reacted with LPS from the *wapR* mutant, showing that the WapR function is required for O-polysaccharide attachment. Data from NMR and MS analyses of LPS from the *wapR* mutant showed the presence of glycoform 1<sub>(O−)</sub> core structure (with L-Rha<sup>A</sup>), ruling out the possibility that WapR is responsible for the formation of the (1 → 6)-L-Rha<sup>A</sup> linkage or addition of any other core glycoform 1<sub>(O−)</sub> sugars (GalN, Glc<sup>I</sup>, Glc<sup>II</sup>, Glc<sup>III</sup>, Glc<sup>IV</sup>).<sup>102</sup>

Since both enzymes are suggested to utilize the same dTDP-L-Rha donor and nascent lipid A-core acceptor, a certain level of competition for substrates might be expected to exist between these two proteins, which could determine the ratio between glycoform 1<sub>(O−)</sub> and glycoform 2<sub>(O+)</sub> LPS molecules. Complementation of the *migA* mutant phenotype by introducing a functional gene carried on a multicopy plasmid also led to loss of core + 1 LPS, as assayed by Western blotting. This observation might support the MigA/WapR competition hypothesis, since overproduction of MigA due to the multicopy plasmid likely causes rapid consumption of substrates for the WapR transferase reaction, which is critical for the attachment of one repeat unit of O-Antigen. This provides evidence that regulation of *migA* or *wapR* expression may be a mechanism used by *P. aeruginosa* to determine the ratio of core glycoform 1<sub>(O−)</sub> and glycoform 2<sub>(O+)</sub> produced. At present, nothing is known about regulation of *wapR* expression, but the *migA* gene was initially described as a mucus-inducible gene, up-regulated in the presence of cystic fibrosis mucus.<sup>139</sup> In a separate study, it was shown by Northern blotting with RNA collected from cystic fibrosis sputum that *migA* is expressed in this environment.<sup>138</sup> The transcription start of *migA* was identified by primer extension, and it did not change in the presence or absence of cystic fibrosis sputum. A *lacZ* promoter fusion showed that the expression of *migA* is cell-density dependent, which is characteristic of genes under the control of quorum sensing. Furthermore, *las* box-like sequences, (CT-(N<sub>11</sub>)-AG), recognition sites for quorum sensing regulation were found upstream of the *migA* gene. Two quorum sensing systems in *P. aeruginosa* were known at the time of these experiments, RhlI–RhlR

and LasI–LasR. Each is composed of two components the first of which is an inducer locus, *lasI* or *rhlI*, that controls the synthesis of the diffusible auto-inducer molecules *N*-3-oxo-dodecanoyl-homoserine lactone (3OC12-HSL) or *N*-butanoyl homoserine lactone (C4-HSL), respectively. The second is a response locus, *lasR* or *rhlR*, that encode transcription factors which become active upon binding to the respective auto-inducers. At high cell density, *migA* expression levels were much lower in either the *rhlI* or *lasI rhlI* double mutant background than in wild-type strain, suggesting that *migA* is regulated by the RhlI–RhlR system. Furthermore, *migA* expression in a *lasI rhlI* double mutant background was increased by the addition of C4-HSL, an auto-inducer that positively stimulates the RhlI/RhlR system. ${ }^{138}$

### Putative glycosyl/glucosyltransferase: PA5001

PA5001 encodes a putative glycosyltransferase sharing 24% identity with a conserved glycosyltransferase domain of WaaG proteins, which are classified into the GT-4 family. It has not been previously suggested which enzyme might be responsible for the transfer of $\alpha$-Glc${}^{\text {III }}$, $\beta$-Glc${}^{\text {I }}$ or $\beta$-Glc${}^{\text {IV }}$. The GT-4 family contains retaining glycosyltransferases and thus, theoretically, PA5001 might be responsible for transfer of $\alpha$-Glc${}^{\text {III }}$, but this prediction must be tested experimentally.

### Addition of phosphates to core: *waaP*, *wapP*, and *wapQ*

Three genes encoding the inner core kinases and a putative kinase have been identified – *waaP* (PA5009), *wapP* (PA5008) and *wapQ* (PA5007). ${ }^{82}$ WaaP was recognised as a putative heptose kinase due to a high (53%) sequence identity to the inner core heptose kinase WaaP of *E. coli* (Table 3). Although WapP and WapQ are not homologues of WaaP from *E. coli* or *S. enterica* sv. Typhimurium, they contain conserved kinase domains and were hypothesised to be involved in the inner core oligosaccharide heptose phosphorylation. ${ }^{82}$ Mutants in *waaP* and *wapP* genes could not be constructed, indicating that the genes may be essential, but the authors were able to create the *wapQ* mutant. The *wapQ* mutant did not exhibit any particular change in LPS phenotype based on silver-stained SDS-PAGE gels and Western immunoblotting analyses, and results from NMR analysis of the *wapQ* core structure were inconclusive. However, this mutant exhibited a reduced MIC (minimal inhibition concentration) for sodium dodecylsulphate (SDS), when compared with wild-type. In general, mutants affected in phosphorylation of inner core have a destabilized outer membrane that leads to a higher susceptibility to SDS and a higher permeability to hydrophobic compounds such as novobiocin. Thus, the SDS susceptibility data suggested that the

*wapQ* mutant might have a subtle core phosphorylation defect, which was not readily apparent in other analyses. However, the *wapQ* function as a kinase has not been proven.

*waaP* from *P. aeruginosa* (*waaP*${}_{Pa}$) complemented a *S. enterica* sv. Typhimurium *waaP* mutant as assayed by NMR analysis; the complemented mutant restored phosphate and ethanolamine diphosphate substitutions at the 4 position of Hep${}^{I}$. Mutation of the chromosomal *waaP* was only achieved in *P. aeruginosa* when a copy of the wild-type gene was present *in trans*, indicating that the mutation is probably lethal. The viability of the *P. aeruginosa* mutant could also be maintained by a plasmid containing the *E. coli waaP* gene. These data suggest that WaaP of *P. aeruginosa*, like its homologues from *E. coli* and *S. enterica* sv. Typhimurium, catalyses transfer of phosphate to the 4 position of Hep${}^{I}$. Alignment of WaaP${}_{Pa}$ with eukaryotic protein kinases, such as the $\alpha$-form of bovine brain protein kinase C (PKC-$\alpha$), reveals potential conserved residues, among them, Lys69 being a putative catalytic residue involved in the proton transfer in the phosphotransfer reaction and Asp163 being putatively involved in the ATP-binding. The site-directed mutants K69A and D163A were constructed, cloned into a vector and tested for ability to complement the *E. coli waaP* mutant in terms of its susceptibility to SDS and novobiocin. Neither of these mutant constructs was able to restore the wild-type phenotype, indicating that the residues might be important for the function of WaaP${}_{Pa}$. In addition, WaaP${}_{Pa}$ was shown by Western blotting to contain phosphorylated amino acids and data from MS analysis indicated that all eight tyrosine residues in this protein were phosphorylated. By developing a chemiluminescence-based ELISA method to determine the activity of WaaP, our group showed that this protein was able to autophosphorylate and also that WaaP has a sugar-kinase activity when given dephosphorylated LPS as a substrate. ${ }^{140,141}$ WaaP was localized to the cytoplasmic cell fraction indicating that core phosphorylation occurs before transport across the inner membrane and therefore before O-Antigen units are attached to the core in the periplasm.

The function of WapP as a heptose kinase was first assigned indirectly, by showing that a *S. enterica* sv. Typhimurium *waaP* mutant complemented with *wapP* and *waaP*${}_{Pa}$ resulted in increased MICs for novobiocin and SDS above the *Salmonella* mutant complemented with *waaP*${}_{Pa}$ alone. Immunchemical evidence was obtained using a similar chemiluminescence-based ELISA method as that used to study WaaP, which showed that WapP could phosphorylate dephosphorylated LPS. ${ }^{142}$ Further evidence from MS and Western blotting analyses revealed that WapP contains phosphorylated serine consistent with a kinase function. ${ }^{142}$

Lipopolysaccharide biosynthesis in Pseudomonas aeruginosa 279

**Addition of carbamoyl: wapO**

Another putative transferase-encoding gene present in the core biosynthesis cluster is *wapO* (PA5005). WapO shares 31% amino acid sequence identity with CmcH (Table 3), an *O*-carbamoyltransferase from *Streptomyces clavuligerus*<sup>143</sup> suggesting that WapO may catalyse transfer of the carbamoyl group found at the 7 position on Hep<sup>II</sup>.

**Core locus genes encoding proteins with unknown functions**

The product of the PA5006 gene has not yet been characterised, but the sequence contains a LPS kinase family conserved domain (Pfam06293), and an APH/ChoK kinase family conserved domain (cd05120).<sup>144</sup> The PA4998 gene product also contains a APH/ChoK conserved domain, and shares 55% identity with the toluene tolerance protein of *P. putida*, whose mechanism and function remain unclear. In general, substrates for the members of the APH/ChoK family are small molecule substrates such as ethanolamine.<sup>145</sup> Therefore, it is tempting to speculate that the PA5006 or PA4998 gene product is an ethanolamine kinase involved in synthesis of the ethanolamine-phosphate moieties found in the inner core.

The PA5002 protein contains a putative transmembrane domain and shares 24% identity with the PIG-L protein domain (Pfam02585) and the PA5003 gene product shares 26% identity with Mig-14 of *S. enterica* sv. Typhimurium, an inner-membrane associated virulence factor with uncharacterised function.<sup>146</sup>

The PA5006, PA5003, PA5002 genes with unknown functions are well conserved in all the genomes of *P. aeruginosa* strains that have been sequenced as well as in the genomes of all other *Pseudomonas* species that have been sequenced (*P. entomophila*, *P. fluorescens*, *P. mendocina*, *P. putida*, *P. stutzeri*, *P. syringae*) suggesting an important role in *Pseudomonas* biology.

**Unknown mechanisms of core modification**

**Addition of diphosphoethanolamine**

In *E. coli*, kinetic experiments indicated that phosphatidylethanolamine (PE) is the precursor of both the phosphate and the ethanolamine of phosphoethanolamine. It has been proposed that the diphosphoethanolamine substituent of LPS arises by transfer of the phosphoethanolamine moiety from PE to phosphate on Hep<sup>I</sup>.<sup>147</sup> As mentioned before, in *P. aeruginosa* ethanolamine can be part of phosphoethanolamine, diphosphoethanolamine or even triphosphoethanolamine. The steps leading to tri-, di- or monophosphoethanolamine on the *P. aeruginosa* core are presently unclear. There is an indication that WaaP may contribute to the substitution

of heptose with diphosphoethanolamine, since complementation of the *S. enterica* sv. Typhimurium *waaP* mutant by *P. aeruginosa waaP* restored both phosphate and diphosphoethanolamine on the inner core heptose.<sup>82</sup> However, data from this experiment did not directly demonstrate the ability of WaaP to transfer di- or monophosphoethanolamine, since the diphosphoethanolamine in *Salmonella* is found at position 4 of Hep<sup>I</sup>, whereas the diphosphoethanolamine position in *P. aeruginosa* core oligosaccharide is at the 2 position of Hep<sup>I</sup>. In *S. enterica* sv. Typhimurium, the *cptA* gene is necessary for phosphoethanolamine addition to the inner core as demonstrated by analysis of mutant LPS by NMR and MS.<sup>148</sup> However, there is no direct biochemical evidence of the CptA phosphoethanolamine transferase function. The *P. aeruginosa* genome contains three homologues of *cptA* – PA4517, PA1972 and PA3310 – which all encode proteins with four or five predicted transmembrane helices and a long hydrophilic loop at the C terminus, similar to CtpA. PA4517 is the closest homologue, and shares 59% amino acid identity with *Salmonella* CtpA.

**Transfer of Ala residue to GalN**

The enzyme that is responsible for attachment of Ala to the GalN-residue has not been identified. Substitution of core oligosaccharide with an amino acid is not rare. Some bacteria, such as *N. meningitidis*, *Haemophilus influenzae* and *Shigella flexneri*,<sup>149–151</sup> have core substituted with a glycine residue. The mechanism of the transfer is not known. A specific tRNA was identified that is a glycine donor and might be the substrate for an unknown transferase.<sup>152</sup> However, no further study identifying the potential transferase or confirming role of a tRNA in the transfer has been published.

**O-Acetylation of core**

The gene(s) responsible for *O*-acetylation of core have not been identified in *P. aeruginosa*. In *H. influenzae*, the HI0392 gene is required for *O*-acetylation of the core and was originally found as a homologue of *S. enterica* sv. Typhimurium O Antigen acetylase encoding gene *oafA*.<sup>153</sup> In PAO1, the PA5238 gene is a *H. influenzae* HI0392 homologue encoding a protein with 30% identity.

**Transport of lipid A-core: msbA**

Compelling genetic data from studies in *E. coli* indicate that MsbA is an essential ABC (ATP-binding cassette) transporter which is required for export of lipid A across the inner membrane. When MsbA is disabled, newly synthesised lipid A accumulates on the cytoplasmic face of this membrane.<sup>154</sup> Furthermore, single amino acid

280 King, Kocínková, Westman, Lam

substitutions in *E. coli* MsbA allow export of lipid A precursor molecules to the cell surface.¹⁵⁵

In PAO1, *msbA* (PA4997) is localised to the large core cluster and encodes a homologue of the *E. coli* MsbA, sharing 40% sequence identity. Characterisation of purified *P. aeruginosa* MsbA, reconstituted into membranes, showed that this protein has intrinsic ATPase activity, which is stimulated by lipid A-core. The stimulatory effect was diminished when the experiment was performed with *S. enterica* sv. Typhimurium LPS, a *P. aeruginosa* lipid A-truncated core lacking L-Rha and D-GlcIV (see Fig. 4) or with a *P. aeruginosa* lipid A-core from which the phosphates had been removed by hydrofluoric acid solvolysis.¹⁵⁶ These data are consistent with assignment of this protein as the LPS transporter, which catalyses ATP-dependent transport of lipid-A-core across the inner membrane. The specificity observed for complete lipid-A core would prevent MsbA from prematurely exporting unfinished LPS molecules.

---

### O POLYSACCHARIDES

#### O Antigen serotyping schemes

In the International Antigenic Typing System (IATS), *P. aeruginosa* strains are classified into 20 serotypes on the basis of serological reactivity of major heat-stable antigens.¹⁵⁷,¹⁵⁸ Other schemes exist, and may have certain advantages. In particular, the Lànyi–Bergan classification scheme emphasises serological relatedness of strains based on analysis using the classical precipitin method in gel, which detects identity, and partial identity (that is, common cross-reactivities and minor antigens) of antigenic epitopes between any two LPS types.¹⁵⁹,¹⁶⁰ The Lànyi–Bergan classification involves a smaller number (13) of serogroups, but most are subdivided, which results in a total of more than 31 distinct subtypes. The Lànyi–Bergan system is favoured by researchers analysing the chemical structures of the O polysaccharides, because this system exquisitely reflects structural relationships between O-Antigen repeat units.⁴⁵ In this review, however, we will mainly use the IATS classification because the genetics of O-Antigen biosynthesis has been investigated exclusively with reference to this scheme.

Some of the 20 IATS serotypes are sometimes collected into groups, which are related serologically,¹⁵⁷,¹⁵⁸ structurally⁴⁵ and genetically.¹⁶¹ These are the O2/O5/O16/O18/O20, O7/O8, O10/O19 and O13/O14 serogroups. Structures of the O-Antigen repeat units produced by the 20 IATS type strains are shown in Table 4.

---

### Common Polysaccharide Antigen and O-Specific Antigen O polysaccharides

The terms ‘A band’ and ‘B band’, which have been used to describe these structures, were coined by Rivera and co-workers¹⁶² to describe LPS subpopulations which they identified in LPS from *P. aeruginosa* PAO1 derivatives. Gel filtration chromatography in a deoxycholate-containing buffer and extremely slow flow rate was able to resolve LPS fractions which when subjected to SDS-PAGE revealed two subpopulations of silver-stained LPS molecules. The upper (A) cluster of bands contained a polymer rich in neutral sugars which was not recognised by the O serotype-specific monoclonal antibody (anti-503). The lower (B) cluster of bands contained the serotype-specific epitope, and its higher electrophoretic mobility toward the anode is probably due to the negative charge of the PAO1 O-Antigen repeat.

As mentioned above, the O-Specific Antigen and Common Antigen O polysaccharides are distinct structurally, serologically and according to the mechanisms of their biosynthesis. The Common O Polysaccharide, which is produced by most *P. aeruginosa* isolates, is a homopolymer of D-Rha. In contrast, the O-Specific Antigens contain different sugars that are organised into repetitive O units.

---

### Models of O-polysaccharide biosynthesis pathways

All the available data suggest that the Common and O-Specific polysaccharides are assembled by different mechanisms according to the two most common models for O-polysaccharide assembly. Several studies of LPS biosynthesis in *P. aeruginosa*, for example de Kievit and Lam on Wzy,¹⁶³ and Rochetta and Lam on Wzm and Wzt,¹⁶⁴ have provided evidence to substantiate the models, both of which were proposed earlier by Whitfield.¹⁶⁵ In brief, Common O-Polysaccharide synthesis follows the ‘ABC transporter-dependent’ pathway and O-Specific Antigen synthesis follows the ‘Wzy-dependent’ pathway. These models have been reviewed by Raetz and Whitfield.⁵⁰

According to both models, O-polysaccharide sugars are assembled on an isoprenyl lipid carrier by cytoplasmic glycosyltransferases, and the completed O-polysaccharide chains are ligated to the lipid-A core in the periplasm. However, the two pathways use distinct mechanisms for the steps in-between, such as polymerisation of the O chain, and export of lipid-linked glycans across the inner membrane. In the ABC transporter-dependent pathway, which is apparently followed by Common Polysaccharide Antigen biosynthesis, the O chain is fully assembled on the cytoplasmic side of the

Table 4. O-Antigen repeating units of the IATS reference strains

| IATS serotype | O-Antigen repeat |
|---------------|------------------|
| O1            | →4)-D-GalNAc-(1→4)-β-D-GlcNAc3NAcA-D-(1→3)-D-FucNAc-(1→3)-D-QuiNAc-(1→ |
| O2            | →4)-β-D-ManNAc3NAmA-(1→4)-L-GulNAc3NAcA-(1→3)-β-D-FucNAc-(1→        |
| O5            | →4)-β-D-ManNAc3NAmA-(1→4)-β-D-ManNAc3NAcA-(1→3)-β-D-FucNAc-(1→       |
| O16           | →4)-β-D-ManNAc3NAmA-(1→4)-β-D-ManNAc3NAcA-(1→3)-β-D-FucNAc-(1→      |
| O18           | →4)-L-GulNAc3NAmA-(1→4)-β-D-ManNAc3NAcA-(1→3)-β-D-FucNAc-(1→        |
| O20           | →4)-L-GulNAc3NAmA-(1→4)-β-D-ManNAc3NAcA-(1→3)-β-D-FucNAc4OAc-(1→     |
| O3            | →2)-L-Rha3OAc-(1→6)-D-GlcNAc-(1→4)-L-GalNAc4OAcA-(1→3)-β-D-QuiNAc4NSHb-(1→ |
| O4            | →2)-L-Rha-(1→3)-L-FucNAc-(1→3)-L-FucNAc-(1→3)-D-QuiNAc-(1→          |
| O6            | →3)-L-Rha-(1→4)-D-GalNAc3OAcAN-(1→4)-D-GalNFOA-(1→3)-D-QuiNAc-(1→   |
| O7            | →4)-Pse4OAc5NRHb7NFo-(2→4)-β-D-Xyl-(1→3)-D-FucNAc-(1→               |
| O8            | →4)-Pse4OAc5NAc7NFo-(2→4)-β-D-Xyl-(1→3)-D-FucNAc-(1→                 |
| O9            | →3′)-Pse4OAc5NAc7NRHb-(2→4)-D-FucNAc-(1→3)-β-D-QuiNAc-(1→*          |
| O10           | →3)-L-Rha2OAc-(1→4)-L-GalNAcA-(1→3)-D-QuiNAc-(1→                     |
| O19           | →3)-L-Rha-(1→4)-L-GalNAcA-(1→3)-D-QuiNAc-(1→                        |
| O11           | →2)-β-D-Glc-(1→3)-L-FucNAc-(1→3)-D-FucNAc-(1→                       |
| O12           | →8)-8eLeg5NAc7NAc-(2→3)-L-FucNAm-(1→3)-D-QuiNAc-(1→                 |
| O13           | →2)-L-Rha-(1→3)-L-Rha-(1→4)-D-GalNAc3OAcA-(1→3)-β-D-QuiNAc-(1→      |
| O14           | →2)-[D-Glc-1→3)]-L-Rha-(1→3)-L-Rha-(1→4)-D-GalNAc3A-(1→3)-β-D-QuiNAc-(1→ |
| O15           | →2)-β-D-Ribf-(1→4)-D-GalNAc-(1→                                    |
| O17           | →4)-β-L-Rha-(1→3)-D-ManNAc-(1→                                     |

Adapted from Knirel *et al.*⁴⁵ O-Antigen structures are clustered to reflect structural relationships. Anomeric conformations are α unless marked β. Sugars have the pyranose form except ribose in O15. Ac, acetyl; 8eLeg, 5,7-diamino-3,5,7,9-tetradeoxy-L-glycero-D-galacto-non-2-ulosonic (8-epilegionaminic) acid; Fo, formyl; FucN, 2-amino-2,6-dideoxy-galactose; GalN, 2-amino-2-deoxy-galactose; GlcNA, 2-amino-2-deoxy-glucuronic acid; GulNA, 2-amino-2-deoxy-guluronic acid; ManN 2-amino-2-deoxy-mannose; ManNA, 2-amino-2-deoxy-mannuronic acid; N, amino; NAc, acetamido; NAm, acetamidino; OAc, O-acetyl; Pse, 5,7-diamino-3,5,7,9-tetradeoxy-L-glycero-L-manno-non-2-ulosonic (pseudaminic) acid; QuiN, 2-amino-2,6-dideoxy-glucose; Rha, rhamnose; RHb (R)-3-hydroxybutanoyl; Rib, ribose; SHb, (S)-3-hydroxybutanoyl. In the polymer, QuiNAc is linked to the pseudaminic acid via the 3-hydroxybutanoyl group.

inner membrane and, after it is complete, is exported by an ABC transporter to the periplasmic face of the inner membrane. In the Wzy-dependent pathway, which is followed by O-Specific Antigen biosynthesis, the O-repeating units are individually flipped across the inner membrane and polymerised in the periplasm in a reaction which requires the product of the wzy gene.

### Initiation of O Antigen biosynthesis: wbpL

For both the Wzy-dependent and ABC transporter-dependent pathways, initiation of O-polysaccharide synthesis is accomplished by transfer of a sugar-1-phosphate to undecaprenyl-phosphate. This reaction generates an undecaprenyl-pyrophosphoryl-linked glycan which can be extended by the action of downstream glycosyltransferases. The initiating sugar-1-phosphate transferases have been identified in S. enterica (WbaP, formerly RfbP) and E. coli (WecA, formerly Rfe). WbaP is a transferase that catalyses the transfer of galactose-1-phosphate from UDP-galactose to undecaprenyl-phosphate.¹⁶⁶ WecA transfers either 2-acetamido-2-deoxy-glucose-1-phosphate (GlcNAc-1-Pᵢ)¹⁶⁷ or 2-acetamido-2-deoxy-galactose-1-phosphate (GalNAc-1-Pᵢ).¹⁶⁸

The initiating sugar-1-phosphate transferase in *P. aeruginosa* is encoded by the *wbpL* gene which is in the *wecA* family of 2-acetamido-2-deoxyhexose-1-phosphate transferases. *wbpL* is located in the O-Specific Antigen biosynthesis gene cluster (described below), but is necessary for synthesis of both Common Polysaccharide Antigen and O-Specific Antigen. *wbpL* mutants made in O2, O5, O6 and O16 strains were all completely devoid of O polysaccharides.¹⁶⁹,¹⁷⁰ The structures of the biological repeating units have been elucidated for most of the *P. aeruginosa* IATS serotype O-Specific Antigens and each has a 2-acetamido-2-deoxy-D-fucose (D-FucNAc), a 2-acetamido-2-deoxy-D-quinovose (D-QuiNAc), or a derivative of these at the reducing terminus.⁹⁴ The presence of these different sugars at the reducing termini raised the question of the specificity of WbpL enzymes from different serotypes for the sugar substrate. Therefore, the ability of WbpL enzymes from

282 King, Kocínková, Westman, Lam

O5 and O6 strains to initiate O-Antigen synthesis using either of these two sugars was tested in cross-complementation experiments. Serotype O5 has D-FucNAc at the reducing terminus and O6 has D-QuiNAc. The $wbpL_{O5}$ gene could fully complement O-Antigen biosynthesis in an O6 serotype $wbpL$ mutant, showing that the WbpL$_{O5}$ enzyme is capable of transferring either sugar-1-phosphate. On the other hand, the $wbpL_{O6}$ gene restored O-Antigen biosynthesis in an O5 serotype $wbpL$ mutant, but only at reduced levels. ${}^{169}$ This suggests that the WbpL$_{O6}$ enzyme has a preference for transferring QuiNAc-1-phosphate as opposed to FucNAc-1-phosphate. In two *P. aeruginosa* O serotypes (O1 and O9) both D-FucNAc and D-QuiNAc are present in the O-Antigen repeating unit, but the initiating sugar in both cases is D-QuiNAc suggesting that WbpL$_{O1}$ and WbpL$_{O9}$ also have a strong specificity for this sugar. ${}^{94}$

Although the repeating unit of the Common Polysaccharide Antigen has been well characterised, ${}^{171}$ the full structure of the Common O polysaccharide has not yet been determined (see below), hence, the identity of the initiating sugar involved in Common O-polysaccharide biosynthesis is unknown. In attempts to compare the substrate specificities of WbpL and *E. coli* WecA, a series of cross-complementation experiments was performed. Expression of $wecA$ in trans restored Common O Polysaccharide, but was unable to restore O-Specific Antigen synthesis in O5- or O6-background $wbpL$ mutants. ${}^{169,170}$ In the converse experiment, $wbpL$ was able to restore synthesis of the *K. pneumoniae* O1 O-Antigen expression in an *E. coli* K-12 $wecA$ mutant strain, but only at very reduced levels. ${}^{170}$ These data indicate that $wecA$ is not functional as a FucNAc-1-phosphate or QuiNAc-1-phosphate transferase, and that Common O-Polysaccharide and O Specific-Antigen synthesis are initiated with different sugars. *Klebsiella pneumoniae* O polysaccharides are synthesized with a GlcNAc residue at the reducing terminus, ${}^{172}$ but it is not possible to discern from these experiments whether this is the same for *P. aeruginosa* Common Polysaccharide Antigen. Indeed, without knowing the substrate specificity of the downstream glycosyltransferase it is not possible to know from the available data whether polysaccharide synthesis is initiated using the same sugar in the cross-complementation experiments as is usually used in the wild-type. As an example to illustrate this problem, broad substrate specificity in both WecA and in the subsequent mannosyltransferase reaction led to the original assignment of WecA as a Glc-1-phosphate transferase on the basis of in vitro reactions. ${}^{173,174}$ Later, however, WecA was shown to transfer GlcNAc-1-phosphate to undecaprenyl phosphate in vivo ${}^{175,176}$ and to transfer GalNAc-1-phosphate to initiate synthesis of Yersinia enterocolitica O8 O Antigen synthesis in *E. coli* K-12. ${}^{168}$

To establish the in vivo function of WbpL in initiation of Common O-polysaccharide biosynthesis it will probably be necessary to characterise the linkage of the D-rhamnan to LPS core, as has been done for *K. pneumoniae* LPS, ${}^{172}$ or to elucidate the full structure of the undecaprenol linked precursor. Composition and glycosyl linkage analysis of a Common Polysaccharide Antigen-containing preparation from a *P. aeruginosa* $galU$ mutant (which is unable to add O polysaccharides to LPS due to truncation of the outer core) indicated the presence of 4-linked GlcNAc suggesting this may be the sugar initiating Common O polysaccharide. ${}^{96}$ Other analyses of Common O-Polysaccharide preparations failed, however, to detect this sugar. ${}^{171,177}$

# COMMON O POLYSACCHARIDE

The Common Polysaccharide Antigen is produced by the majority of *P. aeruginosa* isolates. Of the IATS type strains the serotype O7, O12, O13, O14, O15 and O16 strains lack Common O Polysaccharide in their LPS. ${}^{33}$

## Structure of Common O Polysaccharide

The Common O Polysaccharide contains a trisaccharide chemical repeating unit: [→3)-α-D-Rha-(1→3)-α-D-Rha-(1→2)-α-D-Rha-(1→]$_n$. This rhamnan structure has been characterised several times with good agreement between different studies. ${}^{96,171,177}$ However, monosaccharide composition analyses have identified small amounts of other sugars, with less consistency between reports. As well as rhamnan repeating unit and probable lipid A-core sugars, the following have been identified: 3-O-methyl-6-deoxyhexose and xylose; ${}^{177}$ 3-O-methylrhamnose, ribose, mannose and 3-O-methylhexose; ${}^{171}$ or mannose, GlcNAc and small amounts of O- and N-acetylation. ${}^{96}$ The disagreement between these reports may be due to genuine strain differences, the details of preparation, or to the sensitivity and accuracy of the methods employed.

## The Common Polysaccharide Antigen biosynthetic locus

Functions required specifically for biosynthesis of the *P. aeruginosa* Common O Polysaccharide are encoded in the Common Polysaccharide Antigen biosynthesis gene cluster. Attention has previously focused exclusively on eight contiguous genes from $rmd$ (PA5454) to $wbpZ$ (PA5447) which are probably co-transcribed from a promoter upstream of $rmd$ (Fig. S2). ${}^{178}$ A cosmid clone (pFV3) which contained these genes was able to restore Common Polysaccharide Antigen expression to five of the six IATS type strains which otherwise lack

this structure. While preparing this review, we analysed the region upstream of *rmd* and identified five more coding sequences which were not all present on pFV3 but are related to polysaccharide biosynthesis/ modification genes in other species and may, therefore, also function in the synthesis and processing of the Common O Polysaccharide. These five open reading frames, PA5455–PA5459, are apparently transcribed divergently from *rmd* (Table 5). It is also known that Common Polysaccharide Antigen production requires functions encoded outside of this 13-gene cluster. All of these genes are described below.

### Biosynthesis of D-rhamnose (D-Rha)

The Common Polysaccharide Antigen biosynthetic cluster contains three genes encoding functions in the biosynthesis of GDP-D-Rha, the sugar-nucleotide precursor for the Common O polysaccharide homopolymer. These genes code for WbpW, GMD and RMD. A fourth enzyme, AlgC which is encoded outside of this locus, also participates in this pathway (Fig. 8).

#### wbpW

In *E. coli*, the biosynthesis of GDP-D-Man from D-fructose-6-phosphate (27) is encoded by three genes *manA*, *manB*, and *manC*. In *P. aeruginosa*, *wbpW*

encodes a bifunctional enzyme which catalyses the first and third (ManA- and ManC-catalysed) of these steps (Fig. 8). *WbpW* catalyses both D-mannose-6-phosphate isomerase (phosphomannoisomerase, PMI) and GDP-D-Man pyrophosphorylase (GMP) reactions, which was demonstrated by the ability of *wbpW* to complement *E. coli manA* (PMI) and *manC* (GMP) mutants in K30 capsule biosynthesis. *wbpW* mutants express reduced quantities of Common Polysaccharide Antigen-containing LPS but, when cultures are grown for extended periods of time, it is possible to detect Common Polysaccharide Antigen production. Two *wbpW* functional homologues are present in the PAO1 genome and the presence of these genes probably accounts for the production of the Common O Polysaccharide in the *wbpW* mutant. The first homologue, *algA* (PA3551, 46% amino acid identity with *WbpW*) is located in the locus responsible for alginate biosynthesis and *AlgA* is also bifunctional. Mature alginate is derived from a homopolymer of mannuronic acid and GDP-D-Man is an intermediate in the alginate biosynthetic pathway (reviewed by Remminghorst and Rehm). The provision of *algA* in *trans* is able to boost Common Polysaccharide Antigen production in an *algA wbpW* double mutant, a greater effect being seen with a multicopy *algA*-containing plasmid as opposed to a single copy *algA* chromosomal insertion. This indicates that, in these strains, Common Polysaccharide

**Table 5. Genes involved or potentially involved in Common Polysaccharide Antigen biosynthesis**

| Gene          | Related proteins (% identity)                                                                 | Proposed/demonstrated function                                      | Key reference* |
|---------------|-----------------------------------------------------------------------------------------------|---------------------------------------------------------------------|----------------|
| *wbpZ/PA5447* | 52% *E. coli O9aWbdC*                                                                         | Glycosyltransferase (GT-4)                                          | 170            |
| *wbpY/PA5448* | 34% *E. coli O9aWbdB*                                                                         | Glycosyltransferase (GT-4)                                          | 170            |
| *wbpX/PA5449* | 33% *E. coli O9aWbdA* C-terminal domain (over 301 amino acids)<br>25% *E. coli O9aWbdA* N-terminal domain (over 262 amino acids) | Glycosyltransferase (GT-4)                                          | 170            |
| *wzt/PA5450*  | 61% *E. coli O8Wzt*                                                                           | ABC transporter                                                     | 164            |
| *wzm/PA5451*  | 56% *E. coli O8Wzm*                                                                           | ABC transporter                                                     | 164            |
| *wbpW/PA5452* | 46% *P. aeruginosa AlgA*<br>60% *P. aeruginosa PslB*                                           | D-Man-6-phosphate isomerase /<br>GDP-D-Man pyrophosphorylase        | 178            |
| *gmd/PA5453*  | 47% *E. coli GMD*                                                                             | GDP-D-Man 4,6-dehydratase                                           | 187            |
| *rmd/PA5454*  | 33% *Aneurinibacillus thermoaerophilus RMD*                                                    | GDP-D-Rha synthase                                                  | 187            |
| PA5455        |                                                                                               | Glycosyltransferase (GT-4)                                          |                |
| PA5456        |                                                                                               | Glycosyltransferase (GT-4)                                          |                |
| PA5457        | 20% *E. coli O8WbdD* (over 149 amino acids)                                                   | Methyltransferase                                                   |                |
| PA5458        | 24% *Staphylococcus aureus OatA*                                                              | Acetyltransferase                                                   |                |
| PA5459        | 23% *E. coli O8WbdD* (over 139 amino acids)                                                   | Methyltransferase                                                   |                |
| *algC/PA5322* | 31% *E. coli ManB*                                                                            | Phosphomannomutase/phosphoglucomutase                              | 184            |

Glycosyltransferase (GT) family classification is taken from the CAZy database.
*References are cited if they describe experimental investigation of the *P. aeruginosa* gene.

Downloaded from [ini.sagepub.com](http://ini.sagepub.com) at Scientific library of Moscow State University on February 10, 2014

284 King, Kocínková, Westman, Lam

![Diagram](attachment:diagram.png)

Fig. 8. The GDP-D-rhamnose biosynthesis pathway. Except for the second step in this pathway, all of the enzymes are encoded in the Common Polysaccharide Antigen biosynthesis gene cluster. The gene encoding AlgC is in the alginate locus. P<sub>i</sub>, phosphate; 28, mannose-6-P<sub>i</sub>; 29, α-mannose-1-P<sub>i</sub>; 31, GDP-4-keto-D-Rha.

Antigen synthesis is limited, in part, by the availability of the product of the reactions catalysed by these enzymes. The second *wbpW* homologue is *pslB* (PA2232). *pslB*, which was originally called ORF488, encodes a bifunctional enzyme that shares 60% sequence identity with WbpW,<sup>182</sup> and is in the locus responsible for production of a cell surface polysaccharide known as Psl. Psl is a galactose- and mannose-rich exopolysaccharide;<sup>183</sup> therefore, GDP-D-Man is also presumably a precursor for synthesis of this polymer.

**algC**

The step interposed between the two reactions catalysed by WbpW/AlgA/PslB is encoded by an alginate-locus gene, *algC*. This appears to be the only gene in *P. aeruginosa* encoding a phosphomannomutase (PMM)<sup>184</sup> as *algC* mutation results in loss of detectable cell-lysate PMM activity. As mentioned above in the core biosynthesis section, AlgC also functions as a phosphoglucomutase (PGM) as shown by loss of this activity in the lysates of *algC P. aeruginosa* mutants, and the ability of *algC* to complement a defined *E. coli* PGM mutant.<sup>44</sup> In contrast to the degeneracy in *wbpW/algA/pslB* where each polysaccharide biosynthesis cluster possesses its own PMI/GMP, AlgC catalyses the mutase steps required for synthesis of multiple cell surface glycans. Common Polysaccharide Antigen,<sup>44</sup> alginate,<sup>184</sup> and presumably Psl synthesis require the AlgC PMM function, and LPS core,<sup>44,90</sup> and rhamnolipids<sup>185</sup> require its PGM function.

**gmd and rmd**

*gmd* encodes a GDP-D-mannose 4,6-dehydratase, which catalyses the conversion of GDP-D-mannose (30) to GDP-6-deoxy-D-lyxo-hexos-4-ulose (GDP-4-keto-D-Rha; 31). Expression of this gene in trans restored Common Polysaccharide Antigen expression to the Common O Polysaccharide strain rd7513.<sup>186</sup> The function of the purified protein has also been characterised *in vitro* by capillary electrophoresis and NMR analysis of enzyme-catalysed reaction products<sup>187</sup> and the X-ray crystal structure has been solved.<sup>188</sup>

The protein product of *rmd* catalyses the final step in the GDP-D-Rha biosynthesis pathway, the dinucleotide cofactor-dependent reduction of the GMD reaction product, GDP-4-keto-D-Rha (31) producing GDP-D-Rha (32). Disruption of the *rmd* gene on the *P. aeruginosa* chromosome abrogates Common O Polysaccharide synthesis.<sup>178</sup> RMD has also been studied in purified form, using *in vitro* assays.<sup>187</sup>

*In vitro* assays showed that, like some homologues from other species, *P. aeruginosa* GMD is bifunctional, and capable of catalysing the same reaction as RMD.<sup>187</sup> While this second GMD function may be important in regulation of enzyme function or allowing GMD to change the oxidation state of its bound co-factor, it is unlikely to be significant for *in vivo* GDP-D-Man synthesis. The fact that *rmd* mutations prevent Common Polysaccharide Antigen production indicates that GMD alone cannot complement a defect in the specific reductase (RMD) function.

Co-incubation of GMD, RMD and GDP-D-Man inhibits the GMD reaction,<sup>187</sup> indicating that this pathway may be subject to feedback inhibition. This could be a mechanism by which the cell regulates consumption of GDP-D-Man, which as noted above, is a precursor for several cell surface polysaccharides.

Glycosyltransferases, *wbpX, wbpY* and *wbpZ*

*wbpX, wbpY,* and *wbpZ* encode putative glycosyltransferases required for biosynthesis of the Common Polysaccharide Antigen. All encode proteins with the

Pfam00534 glycosyltransferase conserved motif and generation of non-polar mutants showed that each is necessary for Common O-Polysaccharide production. The IATS O15 *P. aeruginosa* type strain appears to lack Common Polysaccharide Antigen as a result of a mutation in *wbpX* since introduction of *wbpX* in trans restores Common Polysaccharide Antigen synthesis in this strain.¹⁷⁰ The closest, characterised homologues of these *P. aeruginosa* genes are annotated as α-mannosyltransferases from the *E. coli* O9a O Antigen biosynthesis (*wbd*) locus.¹⁸⁹ The *E. coli* O9a antigen is a homopolymer of α-(1 → 2)- and α-(1 → 3)-linked mannose which differs from rhamnose only in the presence of the C-6 hydroxyl. WbpX, WbpY and WbpZ are homologous with WbdA, WbdB and WbdC, respectively. The *E. coli* genes were cloned from what was originally reported to be an O9 strain, and when they were introduced to *E. coli* K-12 cells on plasmids, membranes prepared from those cells were able to incorporate mannose from GDP-[¹⁴C]-mannose *in vitro*. On the basis of these data, a model was proposed for the function of WbdA, WbdB and WbdC in biosynthesis of the *E. coli* O9 mannan.¹⁸⁹ Based upon this model, precise functions were also assigned for WbpX, WbpY and WbpZ from *P. aeruginosa*.¹⁷⁰ For example, WbpZ was proposed as the transferase which adds the first rhamnose to the lipid carrier, because it is 52% identical with WbdC, the enzyme required for initiation of the O9a mannan.¹⁸⁹ This is a remarkably high level of sequence conservation for glycosyltransferase enzymes and it is interesting to speculate that since small amounts of mannose have twice been identified in Common O-Polysaccharide preparations,⁹⁶,¹⁷¹ WbpZ may add mannose to initiate Common O-Polysaccharide synthesis. Since the original report, however, Kido and colleagues¹⁹⁰ have reported further characterisation of the *E. coli* system and it is rather difficult to reconcile some of the newer data with the original model, particularly with respect to the roles of the two mannosyltransferases proposed to elongate the chain: WbdB being suggested to create α-(1 → 3) glycosyl linkages and WbdA responsible for the α-(1 → 2) linkages. First, *E. coli* strain F719, from which these mannosyltransferases were cloned has been reclassified as serotype O9a with a shorter repeating unit than originally thought.¹⁹⁰ Second, WbdA has two domains and truncation of WbdA resulted in a polymer with exclusively α-(1 → 2) linkages.¹⁹¹ In other words, the C-terminal domain of WbdA was found to be necessary for formation of α-(1 → 3) linkages. Further work is required to establish the precise functions of WbdA and WbdB. It is also important to note that while *wbpX* and *wbdA* are homologous to each other, *wbpX* encodes a much shorter protein of only 460 amino acids with only one predicted glycosyltransferase domain compared with the 815 amino acids and two domains

of WbdA. BLAST analysis aligns WbpX with both the WbdA C-terminal and N-terminal domains (Table 5). At this stage, it is unclear whether the *P. aeruginosa* proteins function in a manner precisely analogous to their homologues in the *E. coli* model system.

### Export and processing of Common O polysaccharide

#### ABC transporter: *wzm* and *wzt*

The Common Polysaccharide Antigen biosynthesis gene cluster contains two genes, *wzm* and *wzt*, which encode the integral membrane protein and ATP-binding protein components of the ABC O-polysaccharide exporter, respectively. Allele replacement mutagenesis of either coding sequence prevents export of Common O Polysaccharide to the cell surface. Western blot analysis of whole cell lysates and immunoelectron microscopy indicated that these mutants still produced the Common O Polysaccharide, but that it had a higher electrophoretic mobility in SDS-PAGE and was located in the cytoplasm.¹⁶⁴ These data are consistent with the hypothesis that the ABC transporter mutants were defective in translocation of the O polymer across the inner membrane, so that the polymer cannot be ligated to lipid-A core (which can only occur in the periplasm), but remains attached to an isoprenyl-pyrophosphate lipid carrier.

Cuthbertson and colleagues¹⁹²,¹⁹³ identified a subclass of ABC transporters with an extended Wzt C-terminal domain which specifically recognises modifications of the non-reducing terminus of their polysaccharide chain transport substrates. In *E. coli* O8, the terminal mannose is methylated, whereas in *E. coli* O9a, the terminal sugar is both phosphorylated and methylated. While the Wzm ABC transporter components are interchangeable between the two systems, the C-terminal domains of the Wzt proteins are not, as these recognise the serotype-specific modifications of the non-reducing terminal residue.¹⁹²

The *P. aeruginosa* Common Polysaccharide Antigen locus Wzt has an extended C-terminus like that of *E. coli* O8 and O9a Wzt proteins. Furthermore, the *E. coli* O8 Wzt is the closest characterised homologue, sharing 61% sequence identity overall, with 69% identity over the N-terminal 254 amino acids and, perhaps most meaningfully, 50% identity over the extended C-terminal domain. The *E. coli* O9a Wzt is also a homologue of the *P. aeruginosa* transporter protein with 64% identity in the N-terminus, but only 21% in the C-terminal domain. This is comparable to the low conservation between the *E. coli* O8 and O9a Wzt C-termini.¹⁹³ On the basis of this sequence analysis, we tentatively predict that the Common O-polysaccharide translocation across the inner membrane is dependent on recognition of specific

modification of the non-reducing terminal residue, and that this modification is methylation, and not phosphorylation.

**Determination of chain length and other modifications: PA5455–PA5459**

In the *E. coli* O8 and O9a systems, the non-reducing terminus of the O polysaccharide is modified by a protein called WbdD. As well as introducing the specific terminal modifications for recognition of the polysaccharide by the ABC transporter as described above, the WbdD-catalysed reaction terminates chain extension thereby regulating the O-Antigen chain length. In *E. coli* O9a, WbdD has both kinase and methyltransferase domains, and chain extension is terminated by transfer of a phosphate group (followed by transfer of a methyl group) prior to translocation of the complete polymer across the inner membrane. The *E. coli* O8 WbdD lacks the kinase domain and terminates chain extension by transfer of a methyl group to the non-reducing terminus.¹⁹⁴

In the *P. aeruginosa* eight-gene Common Polysaccharide Antigen cluster previously described, no *wbdD* is present, but BLAST interrogation of the PAO1 genome identifies two homologues of WbdD<sub>E. coliO8</sub>: PA5457 and PA5459 encoded in the putative operon immediately upstream of *rmd* (*rmd* is PA5454; Table 5). The same analysis using the *E. coli* O9a WbdD sequence as probe fails to identify any significant homologues in PAO1. PA5457 and PA5459 share 46% identity with each other and both contain an S-adenosylmethionine (SAM)-dependent methyltransferase domain (Pfam08242). Thus, sequence analysis suggests a role for one or both of these encoded proteins in regulation of O-chain length. The mechanisms by which these systems regulate methylation may differ, however, since the *P. aeruginosa* cluster contains two putative methyltransferases and the PA5457 and PA5459 proteins are both shorter than WbdD from *E. coli* O8. WbdD has an extended C-terminus containing a predicted coiled-coil domain.¹⁹⁴

The methyltransferase domain of PA5459 is also homologous to the N-terminus of WsaE from *Geobacillus stearothermophilus*. *G. stearothermophilus* NRS 2004/3a produces an L-Rha homopolymer¹⁹⁵ by a pathway believed to be related to the biosynthesis of ABC transporter-dependent O polysaccharides.¹⁹⁶ WsaE methylates the non-reducing terminus of L-Rha oligomers, preventing further elongation of the chain. In this system, the chain-terminating methyltransferase domain is fused to two glycosyltransferase domains which function in elongation of the L-rhamnan.¹⁹⁶

Of the other genes in this locus, PA5455 and PA5456 both encode putative glycosyltransferases having partial glycosyltransferase motifs (Pfam00534). They are also homologous to each other. The detection of small quantities of non-D-Rha sugars in Common O-Polysaccharide preparations (as described above) suggests the possibility that there may be an oligosaccharide linker or adapter present at the reducing end of the D-Rha homopolymer. The homopolymeric O polysaccharide of *Bordetella bronchiseptica* appears to be assembled on a five sugar oligosaccharide which links the O homopolymer to the lipid carrier.¹⁹⁷,¹⁹⁸ PA5455 and PA5456 could function in the assembly of such a linker.

Finally, PA5458 has an acetyltransferase-3-family conserved domain (Pfam01757) and shares 24% amino acid sequence identity with OatA, the peptidoglycan O-acetylase which O-acetylates C-6 of N-acetyl muramic acid (MurNAc).¹⁹⁹ The acetyltransferase domain and homology with a polysaccharide acetylase support the hypothesis that the PA5458 enzyme is responsible for the small amounts of O-acetylation detected in one compositional analysis of the Common O polysaccharide.⁹⁶

None of the genes in the PA5455–PA5459 locus have been previously investigated experimentally, though this work is now under way in our laboratory.

**Conservation of the Common Polysaccharide Antigen biosynthetic locus in *P. aeruginosa* strains**

Southern hybridisation identified each of the eight genes from *rmd* to *wbpZ* in all 20 IATS serotype strains’ genomic DNA.¹⁶⁴,¹⁷⁰,¹⁷⁸,²⁰⁰ The majority of these strains showed the same hybridisation patterns, with a few exceptions. In particular, the IATS-type strains for O12, O13, O15 and O16 showed altered Southern hybridisation patterns with probes containing more than one of these genes. These are four of the IATS strains which do not produce Common Polysaccharide Antigen. IATS strains O7 and O14 also do not produce Common Polysaccharide Antigen,³³ but showed normal hybridisation patterns. In each of the Common-Polysaccharide Antigen⁻ IATS reference strains except for the O12 type strain, the recombinant cosmid pFV3 (which contains the eight-gene Common O-Polysaccharide biosynthesis operon) was able to restore Common Polysaccharide Antigen LPS production. This indicates that even in the O7 and O14 reference strains which have normal Southern hybridisation profiles, the defect in Common Polysaccharide Antigen is due to a mutation in this region of the chromosome, potentially frameshift mutations or point mutations affecting catalytic residues. As mentioned above, in the IATS O15 reference strain the defect was localised to a mutation in *wbpX* by complementation experiments.¹⁷⁰ Not all O12 strains lack Common Polysaccharide Antigen.²⁰⁰

Of the *P. aeruginosa* strains whose genome sequences are currently available at Genbank,²⁰¹ PAO1, PA7, PA14, 2192, C3719, PACS2 (and we also checked LESB58) all have the Common Polysaccharide Antigen biosynthesis locus including the five genes which we identified upstream of *rmd*. The organisation of these genes is also conserved in all of these strains except for PA7, in which the five-gene subcluster has been apparently transposed and flipped in a re-arrangement event and now lies on the same strand as the other eight genes and 9.8-kb upstream of the *rmd* homologue.

---

### O-SPECIFIC ANTIGEN

#### O Antigen gene clusters

Early work on O-Antigen biosynthesis in *P. aeruginosa* demonstrated that, in each of IATS serotype O5, O6 and O11 strains, a large cluster of O-Antigen biosynthesis genes was located in the same region of the genome (Fig. S3).¹⁶⁹,²⁰²,²⁰³ Each serotype has a different set of genes located between the highly conserved flanking sequences of the *himD/ihfB* gene upstream of the locus and *wbpm* at the downstream end of the locus. High stringency Southern blotting analysis with a *wbpm* probe indicated that this gene is common to all *P. aeruginosa* serotypes.²⁰² Taking advantage of the conserved flanking sequences on either side of the O-Antigen locus, Maynard Olson’s group used a yeast-recombinational cloning method and cloned the *himD/ihfB–wbpm* region from all 20 IATS serotypes and sequenced the intervening DNA.¹⁶¹ They identified 11 divergent O-Antigen clusters. Four of these clusters were found in more than one IATS strain, with a very high degree of sequence conservation. If strains are clustered according to shared sets of genes in this locus, there is excellent agreement with the grouping of strains suggested by serological¹⁵⁷⁻¹⁶⁰ and structural⁹⁴ properties of O-Antigen. In other words, almost identical sets of genes are present within the O2/O5/O16/O18/O20, O7/O8, O10/O19 and O13/O14 serogroups. The two exceptions to this rule are the IATS O15 and O17 strains. The IATS O17 *himD/ihfB–wbpm* cluster is closely related to the O11 O-Antigen locus but contains two insertion-sequence elements and a major deletion, and the locus does not function to produce the O11 O-Specific polysaccharide.²⁰⁴ Of the two IATS O15 strains examined by Raymond and co-workers,¹⁶¹ one had a cluster resembling the O3 locus, but containing three insertion-sequence elements, and the other was completely devoid of an O-Antigen cluster in this region. Furthermore, *wbpm* can be mutated in O15 or O17 backgrounds without affecting O-Antigen production.²⁰⁵ This observation was later explained by the observations made by Bystrova *et al.*⁹⁴ that the IATS O15 and O17 O-Antigen structures (Lányi–Bergan serogroups O15 and O14, respectively) are exceptional because they do not have the usual initiating sugar (a FucNAc, or a QuiNAc derivative) in the O repeat and their lipopolysaccharides lack the semi-rough (core + 1 O repeat) forms. It is believed, therefore, that biosynthesis of the O15 and O17 O polysaccharides is encoded elsewhere in the genome.

---

### Biosynthesis of O-Antigen repeat units

**wbpm – a common gene in all serotypes**

*wbpm* is conserved in all 20 serotypes of *P. aeruginosa* and is found at the distal end of the O-Antigen biosynthetic locus. Its presence in all serotypes suggested that it may be involved in synthesis of the 6-deoxy sugars D-QuiNAc and D-FucNAc, and their derivatives, found in the O repeat in almost all serotypes. Consistent with this hypothesis, knockout mutation of *wbpm* abrogated O-Antigen synthesis in O5,²⁰² O6,¹⁶⁹ O7,²⁰⁶ O10, O3,²⁰⁵ and O11,¹⁰⁵ but is not required for synthesis of the Common Polysaccharide Antigen, or the O15 and O17 O-Antigens. O15 and O17 O repeats do not contain D-QuiNAc or D-FucNAc.²⁰⁵ Furthermore, subsequent biochemical analysis indicated that WbpM is a UDP-D-GlcNAc 4,6-dehydratase (Fig. 9).²⁰⁷ Reduction of the product of the WbpM-catalysed reaction (33) produces UDP-D-QuiNAc (35) or UDP-D-FucNAc (34) depending on which face of the keto sugar accepts the hydride.

WbpM is a short-chain dehydrogenase/reductase (SDR) family protein, and it has an unusual catalytic triad composed of Thr428, Met438 and Lys442, in which a methionine takes the place and apparently performs the function of the usual SDR catalytic triad tyrosine.²⁰⁸ Note that Ser427 was originally reported as the conserved Ser/Thr member of the triad, but Reverse Position-Specific (RPS)-BLAST alignment of the WbpM sequence with Pfam01370 identified Thr428 as the conserved residue. WbpM is a member of a subfamily of SDR enzymes which have an extended N-terminus that includes several transmembrane helices. Topological analysis by generation of PhoA and LacZ fusions showed that WbpM has four transmembrane helices, and the catalytic C-terminal domain is located cytoplasmically. The N-terminal membrane-spanning domains are not required for enzyme function, but appear to play a role in LPS biosynthesis nevertheless: a construct called S262 is a 786-bp 5′-truncation of *wbpm* and it encodes a soluble enzyme that can be expressed and restore O-Antigen expression in a *wbpm* mutant. S262 is, therefore, catalytically active *in vivo*. However, the LPS produced by this complemented mutant lacks the usual cluster of high molecular mass O chains

suggesting that the N-terminal region of WbpM is necessary for proper regulation of O-Antigen chain length, possibly through interactions with other components of the hypothetical O-Antigen biosynthetic complex.²⁰⁹

WbpM has homologues in many medically important bacterial strains. Of the homologues of comparable length (approximately 600 amino acids) and with similar predicted membrane-spanning domains, the genes encoding WbgZ from *Plesiomonas shigelloides* (47% identical), WlbL from the LPS trisaccharide biosynthesis cluster of *Bordetella pertussis* (44% identical), and Cap8D from *Staphylococcus aureus* (34% identical) can all complement a *wbpM* mutation in *P. aeruginosa*.²⁰⁵ PglF (Cj1120c) from *Campylobacter jejuni* also has the same catalytic function.²¹⁰ There are also a number of homologues of *wbpM* which encode shorter,

soluble enzymes, notably *flaA1* from *Helicobacter pylori* (HP0840, recently renamed *pseB*), *pseB* from *C. jejuni* (Cj1293),²¹¹ and *wbjB* from the *P. aeruginosa* O11 O-Antigen cluster.²¹² For many years, WbpM was misannotated as a bifunctional UDP-D-GlcNAc 4,6-dehydratase/C4-reductase. Part of the confusion was due to early cross-complementation experiments that showed that in multicopy, *flaA1* (which was also misannotated) could restore O-Antigen production in a *wbpM* mutant²⁰⁵ and the implication was drawn that the two enzymes were functionally equivalent.²⁰⁸ The confusion now appears to have been resolved, however, thanks in part to developments in NMR methodology and instrumentation.²¹² WbpM and all currently characterised members of the longer, membrane-associated subfamily of WbpM homologues are simple (retaining) UDP-GlcNAc 4,6-dehydratases and the purified product


Fig. 9. Sugar-nucleotide biosynthesis pathways initiated by UDP-GlcNAc 4,6-dehydratase enzymes. UDP-D-GlcNAc 4,6-dehydratase enzymes such as WbpM catalyse the synthesis of only one product, UDP-4-keto-D-QuiNAc (33). UDP-GlcNAc 4,6-dehydratase-5-epimerases such as WbjB catalyse the production of UDP-6-deoxy-4-keto-L-IdoNAc (36), which is slowly racemised to produce (33). Conserved domains present in LfnA suggest that it may catalyse amidotransfer to the 2-acetamido group of L-FucNAc to form the L-FucNAm residue in *P. aeruginosa* O11,²²⁶ but there is no direct evidence that the amidotransfer occurs onto the sugar-nucleotide as shown here. 38, UDP-L-PneNAc.

of a reaction catalysed by S262, the soluble truncated form of WbpM, was identified by NMR analysis to be UDP-2-acetamido-2,6-dideoxy-D-xylo-4-hexulose (UDP-4-keto-D-QuiNAc, ${}^{33}$; Fig. 9). ${}^{207}$ An analogous experiment with PglF detected the same product. On the other hand, current data suggest that the subfamily of short WbpM homologues, *i.e.* FlaA1, PseB and WbjB, catalyse a 4,6-dehydration of UDP-D-GlcNAc, by a mechanism involving inversion of the stereochemistry at C5. The X-ray crystal structure of FlaA1 ${}^{213}$ and mechanistic studies on PseB from *C. jejuni* ${}^{214}$ have elucidated details of the inverting dehydratase reaction catalysed by these enzymes. The initial product of the inverting dehydratase reaction is UDP-2-acetamido-2,6-dideoxy-L-arabino-4-hexulose (UDP-6-deoxy-4-keto-L-IdoNAc), which is a very labile compound and so was not detected in early experiments. Subsequently, a second product appears in these enzyme-substrate incubations, which is identical to the WbpM-catalysed reaction product, UDP-4-keto-D-QuiNAc (33). This second product appears as a result of racemisation of the C5 stereocentre. ${}^{210,212}$ Because of the slow kinetics for the C5 racemisation, it has been suggested that it is probably not physiologically relevant. ${}^{213}$ Slow racemisation of the FlaA1 product may account, however, for the ability of *flaA1* to complement a *wbpM* mutant when expressed from a multi-copy plasmid (discussed by Schoenhofen *et al*. ${}^{210}$).

---

### Biosynthesis of the *P. aeruginosa* O5 O Antigen

*Pseudomonas aeruginosa* PAO1 is the type-strain for the species, ${}^{215}$ hence its genome was the first in this species to be sequenced and annotated. ${}^{216}$ *Pseudomonas* O Antigen is one of the best characterised (Table 6). The O Antigen of *P. aeruginosa* serotype O5 contains 2-acetamido-3-acetamidino-2,3-dideoxy-D-mannuronic acid (D-ManNAc3NAmA), 2,3-diacetamido-2,3-dideoxy-D-mannuronic acid (D-ManNAc3NAcA), and 2-acetamido-2,6-dideoxy-D-galactose (N-acetyl-D-fucosamine; D-FucNAc); (Table 4). ${}^{217}$ Based on similar serology and O Antigen structures, serotype O5 is in the O2/O5/O16/O18/O20 serogroup. Members of this serogroup have nearly identical O Antigen biosynthesis clusters containing 20 ORFs (Fig. S3). ${}^{161}$

---

### Biosynthesis of 2-acetamido-2-deoxy-D-fucose (D-FucNAc): *wbpM*, and *wbpK*

The biosynthesis of UDP-D-FucNAc (34) is thought to use UDP-D-GlcNAc (1) as the initial substrate and require *wbpM* and *wbpK* (Fig. 9). Currently, WbpK is postulated to be a dinucleotide co-factor-dependent 4-reductase enzyme which catalyses reduction of UDP-4-keto-D-QuiNAc (33, product of the WbpM-catalysed reaction) to generate UDP-D-FucNAc (Fig. 9). A non-polar knockout mutation of *wbpK* abrogated

---

### Table 6. Genes in the O-Antigen biosynthesis cluster of *P. aeruginosa* IATS O5

| Gene           | Proposed/demonstrated function                          | Key reference* |
|----------------|---------------------------------------------------------|----------------|
| *wzz/PA3160*   | O-Antigen chain length regulator                        | 268            |
| *wbpA/PA3159*  | UDP-D-GlcNAc 6-dehydrogenase                           | 221            |
| *wbpB/PA3158*  | UDP-D-GlcNAcA 3-oxidase                                | 223            |
| *wbpC/PA3157*  | Possible O-acetyltransferase                           |                |
| *wbpD/PA3156*  | UDP-D-GlcNAc3NA 3-acetyltransferase                    | 223            |
| *wbpE/PA3155*  | UDP-3-keto-D-GlcNAcA 3-transaminase                    | 223            |
| *wzy/PA3154*   | O Antigen α-polymerase                                 | 259            |
| *wzx/PA3153*   | O unit flippase                                        | 261            |
| *hisH2/PA3152* | Amidotransferase glutamine domain?                      |                |
| *hisF2/PA3151* | Amidotransferase                                       | 13             |
| *wbpG/PA3150*  | Glycosyltransferase (GT-4)                             |                |
| *wbpH/PA3149*  | UDP-D-GlcNAc3NAcA 2-epimerase                         | 225            |
| *wbpI/PA3148*  | Glycosyltransferase (GT-4)                             |                |
| *wbpJ/PA3147*  | UDP-4-keto-D-QuiNAc 4-reductase                       | 169            |
|                | (UDP-D-FucNAc forming)                                 |                |
| *wbpL/PA3145*  | Initiating glycosyl-1-phosphate transferase            |                |
| *PA3144*       | Insertion sequence                                     |                |
| *PA3143*       | Insertion sequence                                     |                |
| *PA3142*       | Insertion sequence                                     |                |
| *wbpM/PA3141*  | UDP-D-GlcNAc 4,6-dehydratase                           | 207            |

---

Glycosyltransferase (GT) family classification is taken from the CAZy database. ${}^{123}$

*References are cited if they describe experimental investigation of the *P. aeruginosa* gene.

O-Antigen synthesis<sup>169</sup> confirming that it has a role in O-Antigen biosynthesis, but there is no experimental evidence to support, specifically, annotation of the enzyme as a 4-reductase. That being said, WbpK is a member of the SDR family, with 36% amino acid identity with the UDP-4-keto-D-QuiNAc 4-reductase enzyme from the O6 locus, WbpV (see below) and these bioinformatic data are consistent with its proposed role.

D-FucNAc residues, or closely related sugars, are also present in the O units of serotypes O1, O9, and O11 and the serogroups O2/O5/O16/O18/O20 and O7/O8. Except for serotypes O1 and O9, the O-Antigen biosynthesis clusters in all of these serotypes possess well-conserved homologues of WbpK with greater than 50% amino acid identity (Table 7). This strongly suggests that for most D-FucNAc-producing *P. aeruginosa* serotypes, the WbpM–WbpK pathway is conserved. The O units in serotypes O1 and O9 are the only ones to contain both D-FucNAc and its 4-epimer, D-QuiNAc. Together with the lack of a well-conserved WbpK homologue, this leads us to speculate that D-FucNAc synthesis may follow a different pathway in these serotypes, possibly involving stereo-non-specific reduction of UDP-4-keto-D-QuiNAc, or 4-epimerisation of UDP-D-QuiNAc (35).

Biosynthesis of 2,3-diacetamido-2,3-dideoxy-D-mannuronic acid (D-ManNAc3NAcA): wbpA, wbpB, wbpE, wbpD, and wbpI

Biosynthesis of UDP-D-ManNAc3NAcA begins with the common precursor, UDP-D-GlcNAc, and requires five genes – wbpA, wbpB, wbpE, wbpD, and wbpI (Fig. 10). A knockout mutant lacking any one of these genes is unable to produce O Antigen.<sup>218–220</sup> The first enzyme of the pathway, WbpA, is a 6-dehydrogenase that catalyzes the conversion of UDP-D-GlcNAc to UDP-2-acetamido-2-deoxy-D-glucuronic acid (UDP-D-GlcNAcA, 40) using NAD<sup>+</sup> as a co-enzyme.<sup>221</sup>

The next steps are thought to be an oxidation reaction catalyzed by WbpB, forming UDP-2-acetamido-2-deoxy-D-ribo-hex-3-uluronic acid (UDP-3-keto-D-GlcNAcA, 41), followed by a WbpE-catalysed transamination reaction creating UDP-2-acetamido-3-amino-2,3-dideoxy-D-glucuronic acid (UDP-D-GlcNAc3NA, 42). A similar two-step oxidation and transamination of UDP-D-GlcNAc to UDP-D-GlcNAc3N has been demonstrated in *Acidithiobacillus ferrooxidans* lipid A biosynthesis.<sup>222</sup> WbpB contains putative oxidoreductase motifs and a Rossman-fold region for binding NAD(P). A *wbpE* knockout is complemented by the *B. pertussis* *wlbC* gene, indicating that they have the same

---

**Table 7. Homologues of WbpK<sub>O5</sub> in other D-FucNAc-producing serotypes of *P. aeruginosa***

| O-Antigen cluster | O1<sup>†</sup> | O1<sup>†</sup> | O2/O16/O18/O20 | O7/O8 | O9<sup>†</sup> | O9<sup>†</sup> | O11 |
|-------------------|---------------|---------------|----------------|-------|---------------|---------------|------|
| ORF               | *orf14*       | *orf16*       | *orf18*        | *orf18* | *orf14*       | *orf16*       | *orf12* |
| Identity (%)      | 21            | 38            | 100            | 50     | 21            | 38            | 51    |

<sup>†</sup>Serotypes that produce both D-FucNAc and D-QuiNAc.

---

![Figure 10](https://i.imgur.com/yourimageurl.png)

**Fig. 10.** UDP-D-ManNAc3NAcA and proposed UDP-D-ManNAc3NAmA biosynthesis pathways. No direct evidence exists for the precise role proposed here for WbpG. 40, UDP-GlcNAcA; 41, UDP-3-keto-GlcNAcA; 42, UDP-GlcNAc3NA; 43, UDP-GlcNAc3NAcA.

function, ${ }^{220}$ and WlbC is a typical type-2 aminotransferase based on structural and spectroscopic data. ${ }^{223}$

The reactions catalysed by WbpB and WbpE were characterised by incubation of both purified proteins with UDP-GlcNAcA (40). The anticipated product of the dual-enzyme reaction, UDP-GlcNAc3NA (42), was purified and its structure confirmed by NMR. ${ }^{223}$ WbpB produces UDP-3-keto-GlcNAcA and uses $\alpha$-ketoglutarate to regenerate the NAD $^{+}$ co-factor bound in the enzyme's active site. $\alpha$-ketoglutarate is a product of the downstream, WbpE-catalysed aminotransfer reaction, which generates UDP-GlcNAc3NA. ${ }^{224}$

UDP-D-GlcNAc3NA (42) was long suspected to be the substrate for WbpD, which was proposed to be an $N$-acetyltransferase. ${ }^{219}$ Early studies showed that WbpD belongs to the superfamily of hexapeptide acyltransferases, forms trimers, and utilizes acetyl-CoA as a co-factor. ${ }^{219}$ After advances in our understanding of WbpB and WbpE, a four-step synthesis of UDP-2,3-diacetamido-2,3-dideoxy-D-glucuronic acid (UDP-D-GlcNAc3NAcA, 43) was accomplished using WbpA, WbpB, WbpE, and WbpD in a single reaction and the final product was verified by NMR. ${ }^{223}$ This demonstrated that WbpD is a 3-$N$-acetyltransferase that converts UDP-D-GlcNAc3NA (42) into UDP-D-GlcNAc3NAcA (43) using acetyl-CoA as the co-factor. ${ }^{223}$

The final step in this pathway requires wbpI, which encodes a 2-epimerase that interconverts UDP-D-GlcNAc3NAcA (43) and UDP-D-ManNAc3NAcA (44). ${ }^{225}$ This reaction was studied using a live-reaction NMR approach that allowed the conversion to be followed in real-time. ${ }^{225}$

---

**Biosynthesis of 2-acetamido-3-acetamidino-2,3-dideoxy-D-mannuronic acid (D-ManNAc3NAmA): wbpG**

UDP-D-ManNAc3NAcA (44) is thought to undergo an amidotransfer reaction catalysed by WbpG to produce UDP-D-ManNAc3NAmA (45) (Fig. 10). The amino acid sequence of WbpG has conserved domains common to ATP-binding enzymes and amidotransferases; this prediction is supported by genetic evidence since a WbpG knockout lacks O-Antigen polymers, but does appear to produce the semi-rough LPS (lipid A — core + 1 O unit). ${ }^{13,202}$ The LPS phenotype of the wbpG knockout, therefore, resembles that of an O-Antigen polymerase (wzy) knockout strain (Wzy is described below). The wbpG phenotype is consistent with its proposed function if it is supposed that three undecaprenyl-pyrophosphate-linked O-Antigen sugars are processed and transferred to lipid-A-core as normal, with a second ManNAc3NAcA in the normal place of ManNAc3NAmA; but that the presence of the acetamidino (NAm)-containing non-reducing terminal sugar is required by the O-Antigen polymerase in order to polymerise the O units. ${ }^{13}$

Homologues of WbpG are found in other *P. aeruginosa* serotypes and other bacterial species that synthesise sugars with acetamidino groups. *P. aeruginosa* O12 and *E. coli* O145 synthesise 2,6-dideoxy-2-acetamidino-L-galactose (L-FucNAm) as part of the O Antigen and the wbpG homologue *lfnA* is required to synthesise the acetamidino moiety in this sugar in serotype O12 (Fig. 9). ${ }^{226}$ *Campylobacter jejuni* produces 5-acetamido-7-acetamidino-3,5,7,9-tetradeoxy-L-glycero-$\alpha$-L-manno-nonulosonic acid (Pse5NAc7Am) for the flagellin glycan, and synthesis of the acetamidino moiety in this sugar requires the wbpG homologue, *pseA*. ${ }^{227}$ Thus, there is an inductive suggestion that wbpG is an amidotransferase responsible for generating D-ManNAc3NAmA.

---

**Glycosyltransferases: wbpH and wbpJ**

The O-Antigen biosynthesis cluster of *P. aeruginosa* PAO1 contains two putative glycosyltransferases, wbpH and wbpJ. Both are assigned by the CAZy database into the GT-4 family. Due to the similarity of their predicted substrates (UDP-D-ManNAc3NAcA and UDP-D-ManNAc3NAmA), sequence analysis is not sufficient to predict which glycosyltransferase adds each residue.

---

**Biosynthesis of the P. aeruginosa O6 O Antigen**

The *P. aeruginosa* O6 O Antigen contains D-QuiNAc, two sugars related to 2-amino-2-deoxy-D-galacturonic acid (GalNA), and L-Rha (Table 4). The O6 O-Antigen biosynthesis cluster contains 12 ORFs (Fig. S3), some of which have been demonstrated to encode biosynthetic enzymes required to form these sugars (Table 8).

---

**Biosynthesis of 2-acetamido-2-deoxy-D-quinovose (D-QuiNAc): wbpM and wbpV**

The UDP-4-keto-D-QuiNAc (33) product of the WbpM-catalysed 4,6-dehydration of UDP-D-GlcNAc (1), is thought to be reduced to UDP-D-QuiNAc (35) in a reaction catalysed by WbpV (Fig. 9). WbpV is predicted to have a NAD(P)-binding Rossmann fold, and is in the SDR family of oxidoreductase enzymes. This reaction will therefore use a NADH or NADPH co-factor as the hydride donor in reduction of the keto-sugar. wbpV is essential for O-Antigen synthesis, as a knockout mutant in *P. aeruginosa* lost O-Antigen expression. ${ }^{169}$ In *Rhizobium etli*, a homologue of wbpV, *lpsQ,* (the two proteins share 46% identity) is required for synthesis of D-QuiNAc in the R. etli LPS core. *lpsQ* mutants synthesise LPS with 4-keto-D-QuiNAc in the usual place of D-QuiNAc, ${ }^{228}$ and expression of wbpV in trans complemented the *R. etli lpsQ* mutation. ${ }^{207}$ This suggests that LpsQ and WbpV are functionally homologous and catalyse the 4-reduction of UDP-4-keto-D-QuiNAc (33)

Table 8. Genes present in the O-Antigen biosynthesis cluster of *P. aeruginosa* IATS serotype O6 strain

| Gene    | Related proteins (% identity)                                                                 | Proposed/demonstrated function                                                                                          | Key reference* |
|---------|-----------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|----------------|
| wzz     | 53% *P. aeruginosa* O5 Wzz                                                                    | Chain length regulator                                                                                                  |                |
| wbpO    | 71% *E. coli* WbqA O121<br>71% *S. dysenteriae* 7 Gna                                        | UDP-GlcNAc 6-dehydrogenase                                                                                             | 230            |
| wbpP    | 68% *E. coli* O121 WbqB<br>68% *S. dysenteriae* 7 Gne                                        | UDP-GlcNAc A 4-epimerase                                                                                               | 230            |
| (wbpQ)  | No significant homologues (by BLAST analysis)                                                |                                                                                                                        |                |
| wzx     |                                                                                               | Und-PP-O unit flippase                                                                                                 |                |
| wbpR    | 32% *P. aeruginosa* O13/O14 Orf10                                                             | Glycosyltransferase (GT-4)                                                                                             |                |
| wbpS    | 54% *E. coli* O121 WbqG<br>54% *S. dysenteriae* 7 WbpS<br>30% *B. subtilis* AsnB (over 631 amino acids) | Glutamine-dependent amidotransferase –synthesis of UDP-GalNAcAN                                                      |                |
| wbpT    | 46% *E. coli* O121 WbqH<br>46% *S. dysenteriae* 7 WbnK                                       | Glycosyltransferase (GT-4) probably of a GalNA sugar derivative                                                          |                |
| wbpU    | 52% *E. coli* O121 WbqI<br>52% *S. dysenteriae* 7 WbnL<br>88/87% *P. aeruginosa* O13/O14 Orf11 | Glycosyltransferase (GT-4), probably of a GalNA sugar derivative                                                         |                |
| wbpV    | 46% *R. etli* LpsQ                                                                            | UDP-4-keto-D-QuiNAc 4-reductase (UDP-QuiNAc synthesising)                                                                | 207            |
| wbpL    | 61% *P. aeruginosa* O5 WbpL                                                                  | Initiating glycosyl-1-phosphate transferase                                                                             | 169            |
| wbpM    | 98% *P. aeruginosa* O5 WbpM                                                                  | UDP-D-GlcNAc 4,6-dehydratase                                                                                           | 169            |

Glycosyltransferase (GT) family classification is taken from the CAZy database.¹²³  
*References are cited if they describe experimental investigation of the *P. aeruginosa* gene.

Lipopolysaccharide biosynthesis in Pseudomonas aeruginosa 293

Table 9. Homologues of WbpV<sup>O6</sup> in other D-QuiNAc-producing serotypes of *P. aeruginosa*

| O-Antigen cluster | O1<sup>†</sup> | O1<sup>†</sup> | O4 | O9<sup>†</sup> | O9<sup>†</sup> | O10/O19 | O12 | O13 | O14 |
|-------------------|---------------|---------------|----|---------------|---------------|---------|-----|-----|-----|
| ORF               | orf14         | orf16         | orf13 | orf14        | orf16         | orf13   | orf24 | orf12 | orf12 |
| Identity (%)      | 39%           | 60%           | 55%  | 39%          | 60%           | 58%     | 55%  | 82%  | 83%  |

<sup>†</sup>Serotypes that produce both D-FucNAc and D-QuiNAc.

---

Table 10. Lányi–Bergan O6 serogroup O-Antigen repeat structures

| Other typing schemes | Lányi- Bergan serogroup | O-Antigen repeat                                                                                          | GalNFoAN amidation |
|----------------------|------------------------|----------------------------------------------------------------------------------------------------------|--------------------|
| IATS O6              | 6a                     | →3)-L-Rha-(1→4)-D-GalNAc3OAcAN-(1→4)-D-GalNFOA-(1→3)-D-QuiNAc-(1→                                              | –                  |
|                      | 6a,6b                  | →2)-L-Rha-(1→4)-D-GalNAcAN-(1→4)-D-GalNFO3OAcA(N)-(1→3)-D-QuiNAc-(1→                                            | 10%                |
|                      | 6a,6c                  | →3)-L-Rha-(1→4)-D-GalNAcAN-(1→4)-D-GalNFOA-(1→3)-D-QuiNAc-(1→                                                  | –                  |
|                      | 6a,6d                  | →3)-L-Rha-(1→4)-D-GalNAc3OAcAN-(1→4)-D-GalNFOA(N)-(1→3)-β-D-QuiNAc-(1→                                          | 20%                |
| Fisher 1             |                        | →2)-L-Rha-(1→4)-D-GalNAc3OAcAN-(1→4)-D-GalNFOAN-(1→3)-D-QuiNAc-(1→                                             | 100%               |
| Homma G              |                        | →3)-L-Rha-(1→4)-D-GalNAcAN-(1→4)-D-GalNFOA(N)-(1→3)-D-QuiNAc-(1→                                               | 50%                |

Adapted from Knirel *et al.*<sup>45</sup>

is a dehydrogenase which catalyses the NAD(P)<sup>+</sup>-dependent 2-fold oxidation of hexoses in sugar-nucleotide substrates to generate uronic acids.<sup>229,230</sup>

WbpP is a NAD<sup>+</sup>-binding SDR enzyme.<sup>231</sup> On the basis of capillary electrophoresis, and spectrophotometric assays, WbpP is a sugar-nucleotide 4-epimerase<sup>230,232</sup> which received detailed biochemical and structural scrutiny because it was the first sugar-nucleotide 4-epimerase enzyme shown to have a substrate preference for UDP-D-GlcNAc/UDP-D-GalNAc over UDP-D-Glc/UDP-D-Gal.<sup>231,233</sup>

Due to somewhat relaxed substrate specificities in both enzymes, WbpO and WbpP can operate in either order to catalyse the overall conversion of UDP-D-GlcNAc (1) to UDP-D-GalNAcA (47; Fig. 11). However, measurement of the kinetic and equilibrium parameters for all of the possible reactions catalysed by these enzymes indicated that in the major pathway *in vivo*, WbpO acts first, producing UDP-D-GlcNAcA (40). This is then followed by subsequent WbpP-catalysed 4-epimerisation to produce UDP-D-GalNAcA (47; Fig. 11).<sup>234</sup>

**Biosynthesis of uronamides: wbpS**

Uronamides are uncommon in nature, and the mechanism for biosynthesis of these sugars is unknown. They are present in the O polysaccharides of *Shigella dysenteriae* type 7,<sup>234</sup> *Francisella tularensis* ssp. *tularensis* Schu S4,<sup>235,236</sup> *Bordetella* spp.<sup>198</sup> and *E. coli* O121.<sup>237</sup> In three of these species, the O polysaccharide contains D-GalNAcAN, and the gene cluster contains a close homologue of the *P. aeruginosa* O6 O-Antigen locus gene *wbpS* (at the amino acid level, *S. dysenteriae*

WbpS, 54%; *E. coli* WbqG, 54%; *F. tularensis* WbtH, 48%). As WbpS has conserved glutamine-dependent amidotransferase domains, Feng and co-workers<sup>234</sup> proposed that WbpS catalyses the transfer of ammonia to D-GalNAcA, producing D-GalNAcAN. Our laboratory has confirmed the role of WbpS in O-Antigen biosynthesis: a non-polar knockout mutation of *wbpS* abrogates O-Antigen production (J.D. King, V. Tran and J.S. Lam, unpublished data). Furthermore, the N-terminus of WbpS is homologous to the glutamine-hydrolysing domain of asparagine synthase B.<sup>238</sup> Point mutation of conserved amino acids in WbpS showed that this region functions to provide ammonia for O-Antigen synthesis. The point-mutant *wbpS* genes could fully complement the *P. aeruginosa wbpS* mutant when it was grown in ammonia-rich medium, but could not do so in minimal medium without ammonia (J.D. King and J.S. Lam, unpublished data).

**Biosynthesis of D-GalNAcA-like sugars in other serotypes**

The O units of serotypes O13 and O14 contain D-GalNAc(3OAc)A. Since the O13/O14 O-Antigen cluster contains close homologues of WbpO and WbpP, sharing 81% and 69% identity with the O6 genes, respectively; the pathway for production of UDP-D-GalNAcA appears to be a conserved strategy in these serotypes. Characterisation of the O Antigen of a Homma K serogroup strain indicated that a proportion of O13 D-GalNAc3OAcA residues can be present as the uronamide (Homma is the name of a serotyping scheme commonly used in Japan, Homma K corresponds to IATS O13).<sup>239</sup> However, no WbpS homologue was identified in the IATS O13 reference strain’s

294 King, Kocínková, Westman, Lam

![Diagram](https://i.imgur.com/yourimage.png)

Fig. 11. UDP-D-GalNAcA and proposed UDP-D-GalNAcAN biosynthesis pathways. WbpO can catalyze the dehydrogenation of UDP-D-GlcNAc (1) or UDP-D-GalNAc (46); and WbpP is an epimerase that can catalyze the interconversion of UDP-D-GlcNAc (1) and UDP-D-GalNAc (46) or UDP-D-GlcNAcA (40) and UDP-D-GalNAcA (47). The physiologically most relevant of the alternative pathways is first oxidation, then epimerisation, with compound 40 as intermediate.²³⁰ It has been proposed WbpS catalyses an amidotransfer reaction to generate the uronamide, GalNAcAN ²³⁴, but it is unknown whether this transformation occurs at the sugar nucleotide level, forming UDP-d-GalNAcAN (48) as shown here.

O-Antigen cluster. This may indicate that this O13 strain has an amidotransferase gene outside of this locus, or it may be that the sequenced O13 O-Antigen cluster was from a strain which does not produce uronamides.

### Biosynthesis of L-rhamnose (L-Rha)

L-Rha is a component of the LPS core oligosaccharide, and the L-Rha found in the O6 O Antigen is, therefore, presumably synthesised by products of the *rml* gene cluster as described in the section on core biosynthesis. This is probably the case for all serotypes which have L-Rha in their O units (O3, O4, O6, O10, O13, O14, O17 and O19).

### Glycosyltransferases: *wbpr*, *wbpT* and *wbpU*

In addition to the initiating glycosyl-1-phosphate transferase gene, *wbpL* (described above), there are three glycosyltransferase-encoding genes present in the O6 O-Antigen cluster, *wbpr*, *wbpT* and *wbpU*, and none of these have been investigated experimentally. The CAZy database¹²³ classifies the products of all three of these genes in glycosyltransferase family GT-4. Based on the properties of characterised members of this family, GT-4 enzymes are believed to share the GT-B type fold and to catalyse the sugar transfer using a retaining mechanism, i.e. one in which the donor sugar stereochemistry at the anomeric centre is conserved in the product glycan.²⁴⁰ The O6 O-repeat structure contains three internal α-glycosyl linkages. The enzyme which catalyses the addition of L-Rha in an α-(1 → 4) linkage must be an inverting glycosyltransferase (L-Rha is derived from the activated precursor dTDP-β-L-Rha). The other internal linkages will be formed by retaining glycosyltransferases. Either the inverting L-Rha transferase is encoded outside of this gene cluster or this structure illustrates the limitations in the use of CAZy classifications to predict

glycosyltransferase properties. WbpT and WbpU both have relatively well-conserved homologues encoded in the *E. coli* O121 O-Antigen locus, WbqH (46% identical to WbpT) and WbqI (52% identical to WbpU). The *E. coli* O121 O-Antigen repeat has the structure →4)-D-GalNAc(3OAc)AN-(1 → 4)-D-GalNAcA-(1 → 3)-D-GlcNAc-(1 → 3)-β-D-Qui4NGlyAc-(1→,²³⁷ which contains two very similar glycosyl bonds to those in the *P.* aeruginosa O6 O Antigen (in bold). On this basis, it is likely that WbpT and WbpU are responsible for catalyzing addition of the D-GalNAc3OAcAN-(1 → 4)-D-GalNFOA-(1 → 3) disaccharide. However, this prediction must be verified by experimental investigation.

### Biosynthesis of the *P. aeruginosa* O11 O Antigen

O11 was the second *P. aeruginosa* serotype for which the O-Antigen biosynthesis cluster was sequenced (Table 11).²⁰³ It is the shortest of the *P. aeruginosa* O-Antigen loci (Fig. S3), reflecting the relative simplicity of the O repeat. The repeating unit contains D-Glc and both stereoisomers of FucNAc. As D-Glc is a component of LPS core, the biosynthesis of the sugar-nucleotide precursor for this sugar is discussed above. Therefore in this section we focus on the biosynthesis of D- and L-FucNAc.

### Biosynthesis of 2-acetamido-2-deoxy-D-fucose (D-FucNAc): *wbpm* and *wbjF*

The *wbpm* gene in the O11 O-Antigen locus encodes a protein which shares 96% sequence identity with the PAO1 sequence, and is required for O11 O-Antigen production. Mutation of *wbpm* in this serotype appears to place the cells under stress, because they apparently pick up second-site mutations with high frequency.¹⁰⁵

Lipopolysaccharide biosynthesis in Pseudomonas aeruginosa 295

Table 11. Genes present in the O-Antigen biosynthesis cluster of *P. aeruginosa* IATS serotype O11 strain

| Gene   | Related proteins (% identity)                                                                 | Proposed/demonstrated function                                                                                   | Key reference* |
|--------|-----------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|----------------|
| wzz    | -                                                                                             | Chain-length regulator                                                                                          |                |
| wzx    | -                                                                                             | O-Antigen repeat unit flippase                                                                                  |                |
| wbjA   | -                                                                                             | Glycosyltransferase (GT-2)                                                                                     |                |
| wzy    | -                                                                                             | O-Antigen polymerase                                                                                            | 203            |
| wbjB   | 78% *E. coli* O26 FnlA<br>66% *S. aureus* Cap5E<br>37% *H. pylori* FlaA1<br>40% *C. jejuni* 11168 PseB | UDP-D-GlcNAc 4,6-dehydratase/5-epimerase                                                                       | 212            |
| wbjC   | 54% *E. coli* O26 FnlB<br>40% *S. aureus* Cap5F                                               | UDP-6-deoxy-4-keto-L-IdoNAc 3-epimerase/4-reductase                                                             | 249            |
| wbjD   | 54% *E. coli* O26 FnlC<br>53% *S. aureus* Cap5G                                                | UDP-PneNAc/UDP-FucNAc 2-epimerase                                                                               | 249            |
| wbjE   | 20% *S. aureus* Cap5L                                                                         | Glycosyltransferase (GT-4)                                                                                     |                |
| wbjF   | 52% *P. aeruginosa* O5 WbpK                                                                   | UDP-4-keto-D-FucNAc 4-reductase (UDP-FucNAc forming)                                                            |                |
| wbpL   | 60% *P. aeruginosa* O5 WbpL                                                                   | Initiating glycosyl-1-phosphate transferase                                                                     |                |
| wbpM   | 96% *P. aeruginosa* O5 WbpM                                                                   | UDP-D-GlcNAc 4,6-dehydratase                                                                                    | 105            |

Glycosyltransferase (GT) family classification is taken from the CAZy database. ${}^{123}$  
*References are cited if they describe experimental investigation of the *P. aeruginosa* gene.

The O11 *wbjF* gene is believed to encode a functional homologue of WbpK from serotype O5. This is hypothesised because *P. aeruginosa* O11 O-unit biosynthesis is initiated with D-FucNAc ${}^{94}$ and because the WbjF sequence is closer to WbpK ${}_{O5}$ (52% identity) than WbpV ${}_{O6}$ (40%). If this is the case, then UDP-D-FucNAc (34) is synthesised in serotype O11 in a scheme like that of serotype O5 (Fig. 9). The function of the product encoded by *wbjF* has not, however, been investigated experimentally.

**Biosynthesis of 2-acetamido-2-deoxy-L-fucose (L-FucNAc): *wbjB*, *wbjC* and *wbjD***

The biosynthesis of L-FucNAc has been studied in detail, because it is a constituent of O-Antigen or capsule polysaccharides in a number of pathogenic bacteria including *P. aeruginosa* serotypes O4, O11, ${}^{241}$ *S. aureus* serotypes 5 ${}^{242}$ and 8, ${}^{243}$ *Streptococcus pneumoniae* capsule type 4, ${}^{244}$ and *E. coli* O4, ${}^{245}$ O26 ${}^{246}$ and O172. ${}^{247}$ Specifically, the pathways in *P. aeruginosa* O11 and *S. aureus* type 5, have been investigated both using genetics and biochemistry (Fig. 9). ${}^{210,212,248,249}$ The O11 genes *wbjB*, *wbjC* and *wbjD* constitute a conserved UDP-L-FucNAc (39) biosynthesis cassette, with homologues present in other L-FucNAc-producing bacteria. In *E. coli*, these genes are annotated as *fnlA*, *fnlB* and *fnlC*, or *fnl1*, *fnl2* and *fnl3*; and in *S. aureus* as *capE*, *capF* and *capG*, often with the capsular serotype of the originating strain indicated; for example, *cap5E* is *capE* from *S. aureus* type 5. The *E. coli* genes are the closest to their *P. aeruginosa* homologues, FnlA, FnlB, and FnlC sharing 78%, 52–54% and 52–54% sequence identity with WbjB, WbjC and WbjD, respectively.

wbjB is essential for O-Antigen production in *P. aeruginosa* O11. ${}^{248}$ *cap5E* encodes an enzyme which shares 66% sequence identity with WbjB, and is functionally equivalent, as demonstrated by both cross-complementation of the *wbjB* mutation with *cap5E*, and by capillary electrophoresis (CE), and CE-MS analysis of *in vitro*, enzyme-substrate incubation products. ${}^{248}$ The WbjB-catalysed reaction is a C5-inverting 4,6-dehydration in which UDP-D-GlcNAc is converted to UDP-2-acetamido-2,6-dideoxy-L-arabino-4-hexulose (UDP-6-deoxy-4-keto-L-IdoNAc; 36); (Fig. 9). As discussed above, in the section on *wbpM*, this is the same reaction as is catalysed by *H. pylori* FlaA1 (37% identity with WbjB) and *C. jejuni* PseB (40% identity with WbjB) in the pseudaminic acid (Pse, 5,7-diacetamido-3,5,7,9-tetradeoxy-L-glycero-L-manno-nonulosonic acid) biosynthesis pathway. ${}^{211}$ In a side reaction, all of these functional homologues also catalyse racemisation of the C5 stereocentre in the product hexulose moiety producing UDP-4-keto-D-QuiNAc (33), ${}^{210,212}$ but this reaction is not relevant to UDP-L-FucNAc (39) biosynthesis. WbjB is a member of the SDR family and, like WbpM, has an unusual TMK catalytic triad (the WbjB triad was originally reported as SMK, but re-alignment with the Pfam01370 motif using BLAST identifies Thr124 rather than Ser123 as the conserved active site residue). WbjB requires NADP ${}^{+}$ to catalyse the dehydratase reaction. ${}^{248}$

WbjC is a bifunctional enzyme catalysing C3-epimerisation of the WbjB-catalysed reaction product (36), and then reduction of the C4 ketone to an alcohol (38). The product of the second step is UDP-2-acetamino-2,6-dideoxy-L-talose (UDP-L-PneNAc, 38) as determined by

296 King, Kocínková, Westman, Lam

NMR analysis of the purified reaction product (Fig. 9).²⁴⁹ WbjC is another SDR family protein, and requires a reduced-form dinucleotide co-factor to catalyse the ketone reduction without any specific preference for either NADH or NADPH.²⁴⁸ WbjC catalyses the same reaction as Cap5F of *S. aureus* (40% sequence identity) *in vitro*.²⁴⁸

The final step in the pathway is catalysed by WbjD. Cap5G (53% identity with WbjD) is functionally equivalent *in vitro* but, in this case, the *S. aureus* enzyme was inactive at 37°C.²⁴⁸ These enzymes have the sugar-nucleotide 2-epimerase conserved domain (Pfam02350), and catalyse the epimerisation of UDP-L-PneNAc (38) to UDP-L-FucNAc (39; Fig. 9). As is typical of epimerisation reactions, total conversion of the starting material does not occur, and the two isomers attain equilibrium.²⁴⁸ *In vitro*, the proportion of UDP-L-FucNAc at equilibrium can be maximised by performing the reaction at pH 9, an observation which facilitated purification and NMR characterisation of the final product of the pathway by NMR.²⁴⁹

L-FucNAc is also found in the O-unit of serotype O4. The O-Antigen biosynthesis cluster of serotype O4 has three genes *orf9*, *orf10* and *orf11*, which encode homologues of the O11 UDP-L-FucNAc biosynthesis enzymes WbjB, WbjC, and WbjD with very high identity (88%, 74%, and 83%, respectively).

### Glycosyltransferases: *wbjA* and *wbjE*

In addition to WbpL<sup>O11</sup>, which probably initiates O-Antigen synthesis by transfer of D-FucNAc-1-phosphate to undecaprenyl-phosphate, the O11 O-Antigen gene cluster encodes two putative glycosyltransferases. *wbjA* encodes a protein classified in the CAZy family GT-2. To date, characterised members of this family use an inverting mechanism and the GT-A three-dimensional fold.²⁴⁰ *wbjE* encodes a CAZy GT-4 family protein which would suggest that WbjE has the GT-B fold and retains the anomeric configuration of the transferred sugar. As formation of the two internal glycosyl bonds in the O11 repeat (namely the L-FucNAc-α-(1→ linkage from a UDP-β-L-FucNAc donor, and D-Glc-β-(1→ linkage from a presumed UDP-α-D-Glc donor) requires two inverting glycosyltransferases, the CAZy classification of these enzymes does not help distinguish the functions of WbjA and WbjE. WbjE and Cap5L, a putative glycosyltransferase from the *S. aureus* type 5 capsule locus, share rather limited sequence conservation (20% identity); on this basis, we very tentatively suggest that these two enzymes may transfer the L-FucNAc present in each of their respective polysaccharides.

---

### Biosynthesis of sugars in other serotypes

While the O5, O6 and O11 O-Antigen clusters have been studied in some detail, very little direct experimental investigation has been conducted into the functions of genes in the remaining O-Antigen loci. To our knowledge, only one report has been published, which concerns the function of *lfnA* in serotype O12.²²⁶ Based on comparison of O-Antigen cluster gene sequences and the repeat unit structures between serotypes, it seems that where the same sugar appears in more than one serotype, the biosynthetic apparatus is also conserved. The identification of homologues of characterised O5, O6 and O11 genes in the other serotypes, therefore, allows us to annotate the other O-Antigen clusters intelligently. In this section, we illustrate how our understanding of the biosynthetic pathways for O-Antigen sugars in O5, O6, and O11 provides a useful basis on which to generate hypotheses regarding the biosynthesis of some sugars required for the other serotypes.

---

### Biosynthesis of 2-acetamidino-2-deoxy-L-fucose (L-FucNAm): *lfnA*

The *P. aeruginosa* serotype O12 O repeat contains a 2,6-dideoxy-2-acetamidino-L-galactose (L-FucNAm) sugar. Since the O12 O-Antigen biosynthesis cluster contains close homologues of the serotype O11 genes *wbjB*, *wbjC* and *wbjD* (sharing 87%, 71% and 83% amino acid identity, respectively), it is likely that the L-FucNAm biosynthetic pathway involves initial synthesis of UDP-L-FucNAc (39), and subsequent modification of the acetamido group by an amidotransfer reaction. Our group has investigated the genetics of this modification reaction and shown that, without a functional copy of the *lfnA* gene, *P. aeruginosa* O12 produces an O Antigen which contains L-FucNAc rather than L-FucNAm. The *lfnA* mutant can also be complemented by *in trans* expression of the *E. coli* O145 gene *wbuX* (*LfnA* and *WbuX* share 71% identity).²²⁶ Furthermore, *LfnA* contains conserved domains consistent with a role activating its substrate to accept the ammonia nucleophile in an amidotransfer reaction, including a partial alignment with Pfam00733 (C-terminal domain of asparagine synthase). While these experiments demonstrated that *lfnA* is necessary for synthesis of the acetamidino moiety, they do not prove that *LfnA* is sufficient to catalyse this reaction by itself. Indeed, introduction of *lfnA* alone into other serotypes of *P. aeruginosa* which have L-FucNAc as a component of the O repeat (O4 and O11) apparently did not result in changes to the O-Antigen structure.²²⁶ One of the possible reasons for this result may be that other proteins encoded in the large O12 O-Antigen locus are required

for acetamidino group synthesis. The O12 locus orf10 is a homologue of *Campylobacter coli ptmE* gene (40% identity at amino acid level). It has been postulated that *ptmE* was required for acetamidino-modification of one of the *C. coli* flagellar glycan, though it should be noted that the sugars in question were only tentatively identified by MS.²⁵⁰ Furthermore, many amidotransferase enzymes have a second domain which hydrolyses glutamine to provide ammonia, but sequence analysis predicts that LfnA has no such domain. However, a glutamine hydrolysis domain (Pfam00117, glutamine amidotransferase class I) is apparently encoded, by *orf18* which is immediately downstream of *lfnA*. *orf10 and orf18* may be required, in addition to *lfnA*, for acetamidino group synthesis. Since mutation in the *lfnA* homologue, *PseA*, in *C. jejuni* results in loss of acetamidino-containing species from the sugar-nucleotide pool,²²⁷ the current working hypothesis is that the amidotransfer reaction which requires *lfnA* also occurs at the sugar-nucleotide level and produces UDP-L-FucNAm (49) as the precursor to the L-FucNAm found in the O repeat (Fig. 6). Further work is required to verify (or refute) this hypothesis.

### Biosynthesis of nonulosonic acids

Serotypes O7, O8, O9 and O12 contain nonulosonic acids at the non-reducing termini of their O-Antigen repeats. In O7, O8 and O9 these residues are derivatives of pseudaminic acid (Pse); in O12 this is a derivative of 5,7-diamino-3,5,7,9-tetradeoxy-L-glycero-D-galacto-non-2-ulosonic acid (8-epilegionaminic acid, 8eLeg). The biosyntheses of these sugars have not been investigated in the context of *P. aeruginosa* LPS but probably follow pathways analogous to the Pse pathway in *C. jejuni*²¹¹ and the Leg pathway in *Legionella pneumophila*.²⁵¹ In these pathways, after biosynthesis of a UDP-activated aminoheose sugar, the distinctive enzymatic steps are:

1. Removal of the UDP. In the pseudaminic acid pathway this is catalysed by a nucleotidase *PseG*,²¹¹ in the neuraminic acid and legionaminic acid pathway this is accomplished by a homologue of NeuC which catalyses both inversion of the stereochemistry at C2 and also removal of the UDP moiety.²⁵¹,²⁵²
2. Condensation of the aminoheose derived from UDP-GlcNAc, with phosphoenolpyruvate (PEP), catalysed by a NeuB homologue (*PseI* in pseudaminic acid biosynthesis).
3. CMP-activation of the resulting nonulosonic acid with consumption of CTP, which is catalysed by NeuA in *E. coli* K1, or its homologue *PseF* in the pseudaminic acid pathway.

BLAST analysis of *P. aeruginosa* sequences using the sequences of characterised enzymes identifies candidates for these genes in the O7, O8, O9 and O12 O-Antigen biosynthesis clusters (Table 12).

#### Biosynthesis of 2-acetamido-4-O-acetyl-2-deoxy-D-fucose (D-FucNAc4OAc): *wbpC*

Modification of the D-FucNAc residue with a 4-O-acetyl substituent is observed in *P. aeruginosa* serotype O20. The gene which encodes this modification is currently unknown; a candidate is the *wbpC* gene within the O-Antigen biosynthesis cluster, which shows similarity to acyltransferases. The WbpC sequence has an acyltransferase 3 superfamily conserved domain (Pfam01757).²⁵³ *wbpC* is also present in the O-Antigen clusters of other members of the O2/O5/O16/O18/O20 serogroup. Knockout mutation of *wbpC* in *P. aeruginosa* serotype O5 made no noticeable change to the LPS phenotype; however, because O20 is the only serotype which produces D-FucNAc4OAc, no phenotype would be predicted (C. Wenzel and J.S. Lam, unpublished data). Reverse-transcriptase PCR detected *wbpC* mRNA in both serotypes O5 and O20 indicating that, if WbpC is indeed responsible for 4-O-acetylation of D-FucNAc in O20, the reason this activity is not seen in O5 is not because the *wbpC* gene is not transcribed (E.L. Westman and J.S. Lam, unpublished data). Additional research is required to clarify the role of *wbpC*; construction of a *wbpC* knockout in *P. aeruginosa* serotype O20 would be the next logical experiment.

---

**Table 12. Homologues of distinctive nonulosonic acid biosynthesis enzymes in *P. aeruginosa* O-Antigen gene clusters**

| BLAST query          | Homologues from O7/O8 cluster | Homologues from O9 cluster | Homologues from O12 cluster |
|----------------------|-------------------------------|----------------------------|-----------------------------|
| *C. jejuni* PseG     | Orf11 (25%)                  | Orf 9 (29%)                | Orf7 (32%)                 |
| *E. coli* K1 NeuC    |                               |                            |                             |
| *E. coli* K1 NeuB    | Orf13 (35%)                  | Orf11 (31%)                | Orf 8 (39%)                |
| *E. coli* K1 NeuA    | Orf10 (24%)                  | Orf8 (27%)                 | Orf12 (29%)                |

Amino acid sequence identities are listed in parentheses. Alignments were produced using the PSI-BLAST algorithm, interrogating *P. aeruginosa* sequences at Genbank.

298 King, Kocínková, Westman, Lam

Biosynthesis of 2,3-diacetamido-2,3-dideoxy-L-guluronic acid (L-GulNAc3NAcA) and derivatives: hypothetical AlgG-like activity

The O-Antigen repeats of *P. aeruginosa* serotypes O2, O18, and O20 contain a guluronic acid derivative, 2,3-acetamido-2,3-dideoxy-L-guluronic acid or 2-acetamido-3-acetamidino-2,3-dideoxy-L-guluronic acid (L-GulNAc3NAcA or L-GulNAc3NAmA). The L-gulo configuration can be produced by 5-epimerisation of the corresponding D-manno-hexose. For example, alginate, an exopolysaccharide of *P. aeruginosa*, is synthesised as a polymer of D-mannuronic acid before epimerisation of some of the residues to L-guluronic acid by the mannuronan 5-epimerase, AlgG.²⁵⁴ Thus, a UDP-D-ManNAc3NAcA 5-epimerase could produce UDP-L-GulNAc3NAcA, or a putative AlgG-like 5-epimerase could act on D-ManNAc3NAcA residues in the assembled O-Antigen polymer. No candidate 5-epimerase genes are present in the O-Antigen biosynthesis cluster, so this function may be encoded outside of this locus.

Biosynthesis of D-xylose (D-Xyl): serogroup O7/O8 orf16 (*arnA* homologue)

Serotypes O7 and O8 can be treated together from the perspective of O-Antigen sugar biosynthesis, since their O-Antigen biosynthesis clusters contain highly conserved sets of genes.¹⁶¹ The O-Antigen repeating units of O7 and O8 both contain D-Xyl.⁴⁵ UDP-D-Xyl could be synthesised from UDP-D-GlcA in a pathway involving a decarboxylase reaction. The C-terminal domain of ArnA from *E. coli* catalyzes the oxidation and decarboxylation of UDP-GlcA to UDP-4-keto-arabinose. *P. aeruginosa* PAO1 also encodes an ArnA homologue in the *arn* locus (see lipid A section). *orf16* of the serogroup O7/O8 O-Antigen locus encodes an SDR enzyme with 29% amino acid identity with the C-terminal region of ArnAPA01. Possibly, *orf16* encodes the necessary decarboxylase, but the functions of SDR enzymes cannot be predicted from percentage identity alone.²⁵⁵

Biosynthesis of 2,3-diacetamido-2,3-dideoxy-D-glucuronic acid (D-GlcNAc3NAcA): *wbpA*, *wbpB*, *wbpE*, and *wbpD*

The O Antigen of serotype O1 contains D-GlcNAc3NAcA. UDP-D-GlcNAc3NAcA (43) is produced as an intermediate in *P. aeruginosa* serotype O5; the O Antigen of serotype O5 contains the 2-epimer, D-ManNAc3NAcA, for which the pathway has been described (Fig. 10). The predicted protein sequences of Orf6, Orf7, Orf8, and Orf9 from the serotype O1 O-Antigen cluster show >75% identity to WbpA, WbpB, WbpD, and WbpE of serotype O5 (Table 13). Thus, these genes are expected to be involved in the biosynthesis of UDP-D-GlcNAc3NAcA (43), using the same scheme that has been described in *P. aeruginosa* PAO1.

New annotation of selected O-Antigen biosynthesis clusters

While the majority of *P. aeruginosa* O-Antigen biosynthesis genes has not been directly investigated, sufficient information has been provided by the characterisation of genes and their products from O5, O6 and O11 (see previous sections) to allow rational prediction of the pathways for synthesis of most sugars found in the O repeats of other serotypes. With this notion, we outline below, a preliminary annotation of the O-Antigen biosynthesis clusters of O1, O4 and O13/O14. These annotations will form a basis for understanding O-Antigen biosynthesis in these serotypes.

Serotype O1

This serotype is clinically prevalent, constituting, for example, 21% of *P. aeruginosa* clinical isolates identified in a recent study.²⁵⁶ The O-Antigen cluster of serotype O1 (Fig. S4) contains 15 ORFs with similarity to genes from the well-studied serotypes O5 and O6 (Table 13). Our assignment of genes is based on homology to characterised genes from serotypes O5 and O6, conserved motifs and predicted transmembrane helices (Table 13). For example, there are two ORFs of serotype O1 that have 19–20% identity to Wzx of serotype O6 (WzxO6, 474 amino acids); however, *orf5* encodes only 91 amino acids and so, if it encodes any function at all, it is very unlikely to encode the same function as WzxO6. Therefore the gene encoding a full-length homologue of WzxO6 was annotated as *wzx*. The *wzy* annotation was based on 30% amino acid identity to Wzy of *E. coli* O42 and multiple predicted transmembrane helices.

One challenge in annotation of the serotype O1 gene cluster arises because the O-Antigen repeat unit contains D-GalNAc, D-FucNAc, and D-QuiNAc. In other serotypes, all of these sugars are synthesized from UDP-D-GlcNAc in reactions catalysed by specific SDR enzymes in addition to WbpM (Figs 9 and 11). Enzymes are identifiable as SDRs through the conservation of several motifs.²⁵⁷ However, members of this large protein family catalyse such a wide range of chemical reactions, including oxidation, reduction, epimerisation, dehydration, and decarboxylation, and typically share 15–25% identity with other SDR enzymes, including those that share the same catalytic chemistry, and those that do not. Enzyme chemistry is determined by the presence or absence of a small number of amino acid side chains, which are often not readily identifiable in sequence alignments.²⁵⁵ This means that, where well-conserved

Table 13. Genes present in the O-Antigen biosynthesis cluster of *P. aeruginosa* IATS serotype O1 strain

| Gene/ORF | Related proteins (% identity) | Proposed function |
|----------|-------------------------------|--------------------|
| wzz/orf4 | 69% *P. aeruginosa* O5 Wzz     | O-Antigen chain length regulator |
| orf5*    | 19% *P. aeruginosa* O6 Wzz    |                    |
| wbpA/orf6 | 85% *P. aeruginosa* O5 WbpA   | UDP-D-GlcNAc 6-dehydrogenase |
| wbpB/orf7 | 76% *P. aeruginosa* O5 WbpB   | UDP-D-GlcNAcA 3-oxidase |
| wbpD/orf8 | 82% *P. aeruginosa* O5 WbpD   | UDP-D-GlcNAc3NA 3-acetyltransferase |
| wbpE/orf9 | 76% *P. aeruginosa* O5 WbpE   | UDP-3-keto-D-GlcNAcA 3-transaminase |
| orf10    |                               | glycosyltransferase (GT-4) probably of D-GalNAc or D-FucNAc |
| wzx/orf11 | 20% *P. aeruginosa* O6 WzxO6  | O-unit flippase |
| wzy/orf12 | 30% *E. coli* O42 Wzy         | O-Antigen polymerase |
| orf13    | 37% *P. aeruginosa* O5 MigA    | Glycosyltransferase (GT-2) probably of D-GlcNAc3NAcA (rhamnosyltransferase) |
| orf14    | 98% *P. aeruginosa* O9 Orf14    | SDR probably involved in production of D-GalNAc or D-FucNAc |
|          | 23% *P. aeruginosa* O11 WbjF   |                    |
|          | 23% *P. aeruginosa* O6 WbpV    |                    |
|          | 21% *P. aeruginosa* O5 WbpK    |                    |
| orf15    | 28% *P. aeruginosa* O5 WbpZ    | Glycosyltransferase (GT-4) probably of D-GalNAc or D-FucNAc |
|          | 27% *P. aeruginosa* O6 WbpU    |                    |
|          | 26% *P. aeruginosa* O6 WbpT    |                    |
| wbpV/orf16 | 98% *P. aeruginosa* O9 Orf16  | UDP-4-keto-D-QuiNAc 4-reductase (UDP-D-QuiNAc synthesising) |
|          | 60% *P. aeruginosa* O6 WbpV    |                    |
|          | 37% *P. aeruginosa* O5 WbpK    |                    |
|          | 25% *P. aeruginosa* O6 WbpP    |                    |
| wbpL/orf17 | 63% *P. aeruginosa* O5 WbpL   | Initiating glycosyl-1-phosphate transferase |
| wbpM/orf18 | 97% *P. aeruginosa* O5 WbpM   | UDP-D-GlcNAc 4,6-dehydratase |

*Truncated sequence (91 vs 474 amino acids in *P. aeruginosa* O6 Wzx) Glycosyltransferase (GT) family classification is taken from the CAZy database.*

homologues do not exist, experimental characterisation is the only option for annotation of SDR enzymes.

Despite the hypothetical requirement for three SDR enzymes to produce UDP-D-GalNAc, UDP-D-FucNAc, and UDP-D-QuiNAc, the O-Antigen cluster of serotype O1 encodes only two putative SDRs in addition to WbpM: *orf14* and *orf16*. *orf16* encodes a close homologue of WbpV (60% identity), which is the 4-reductase of serotype O6 involved in producing UDP-D-QuiNAc, suggesting that UDP-D-QuiNAc could be produced in the common manner (Fig. 9). However, the other SDR enzyme has limited similarity (<25% identity) to any other *P. aeruginosa* gene except for *orf14* of serotype O9, which has an unknown function (Table 13). Therefore, precise prediction of the enzyme function encoded by *orf14* is premature given the available information, but it is likely involved in the production of UDP-D-GalNAc and/or UDP-D-FucNAc.

Identification of conserved genes putatively involved in the biosynthesis of common sugars among *P. aeruginosa* serotypes has allowed nearly complete annotation of the sugar-nucleotide pathways of serotype O1 (Table 13). Experimentation is required to clearly delineate the roles of the putative SDR enzymes in this serotype, particularly with respect to *orf14*.

Serotype O4

Serotype O4 strains have not been well-studied, but have recently been identified in Hungary with multidrug resistance and country-wide distribution. The serotype O4 O-Antigen cluster *orf6* did not share significant similarity with any proteins in BLAST analysis. However, since *orf6* encodes a protein with multiple predicted transmembrane helices and other encoded membrane proteins in the O4 cluster were all assigned other functions in our analysis, we annotated *orf6* as a putative *wzy* gene (Table 14). The serotype O4 cluster (Fig. S4) contains several genes with homology (>25% identity) to genes of serotype O11 (Table 14), which encode creation of L-FucNAc residues (Fig. 9); therefore, the functions of these genes are very likely conserved in the serotype O4 homologues. Based on the presence of these conserved genes and the production of conserved sugars, we have been able to present a complete annotation of the serotype O4 O-Antigen locus (Table 14).

Serotypes O13/O14

Serotypes O13 and O14 can be treated together from the perspective of O-Antigen sugar biosynthesis, since their O-Antigen biosynthesis clusters contain highly

conserved sets of genes.¹⁶¹ Serotypes O13/O14 produce an O repeat unit that is very similar to that of serotype O6: all three serotypes produce O Antigens that contain L-Rha, D-QuiNAc, and D-GalNAc(3OAc)A (Table 4). Accordingly, many ORFs in the serotype O13/O14 gene cluster have a good degree of identity (>32%) to genes of serotype O6 (Table 15).

As with serotype O4, a putative wzy gene was identified based on the presence of multiple predicted transmembrane helices and by eliminating other candidates for this function within the cluster. As is often the case with Wzy sequences, no significant similarity to

other proteins in *P. aeruginosa* or other species was found by BLAST analysis of the protein encoded by this gene (Table 15). In serotype O14, a frameshift mutation has apparently occurred which likely renders the putative wzy non-functional; but polymerized O Antigens are still produced. These observations are consistent with the findings of Kaluzny *et al.*,²⁵⁹ that O2 and O16 O-specific polysaccharide polymerisation is encoded outside of the O-Antigen cluster, which must also be the case in O6, where there is no Wzy gene within the O-Antigen cluster.¹⁶⁹ The putative wzy with frameshift in the O-Antigen cluster of serotype O14 thus

### Table 14. Genes present in the O-Antigen biosynthesis cluster of *P. aeruginosa* IATS serotype O4 strain

| Gene/ORF | Related proteins (% identity) | Proposed function |
|----------|-------------------------------|--------------------|
| wzz/orf4 | 48% *P. aeruginosa* O5 Wzz     | Chain length regulator |
| wzx/orf5 | 20% *P. aeruginosa* O6 Wzx     | Und-PP-O unit flippase |
| wzy/orf6 | Contains predicted            | O-Antigen polymerase |
|          | transmembrane domains         |                    |
| orf7     |                               | Glycosyltransferase (GT-2) probably |
|          |                               | of D-QuiNAc or L-FucNAc |
| orf8     |                               | Glycosyltransferase (no CAZy family assigned) |
|          |                               | probably of D-QuiNAc or L-FucNAc |
| wbjB/orf9 | 88% *P. aeruginosa* O11 WbjB | UDP-D-GlcNAc 4,6-dehydratase and 5-epimerase |
| wbjC/orf10 | 74% *P. aeruginosa* O11 WbjC | UDP-2-acetamido-2,6-dideoxy-β-L-arabino-4-hexulose |
|          |                               | 3-epimerase and 4-reductase |
| wbjD/orf11 | 83% *P. aeruginosa* O11 WbjD | UDP-L-PneNAc 2-epimerase |
| wbjE/orf12 | 25% *P. aeruginosa* O11 WbjE | Glycosyltransferase (GT-4) probably of L-FucNAc |
| wbpV/orf13 | 55% *P. aeruginosa* O6 WbpV | UDP-4-keto-D-QuiNAc 4-reductase |
|          |                               | (UDP-D-QuiNAc synthesising) |
| wbpL/orf14 | 62% *P. aeruginosa* O5 WbpL | Initiating glycosyl-1-phosphate transferase |
| wbpM/orf15 | 93% *P. aeruginosa* O5 WbpM | UDP-D-GlcNAc 4,6-dehydratase |

Glycosyltransferase (GT) family classification is taken from the CAZy database.¹²³

### Table 15. Genes present in the O-Antigen biosynthesis cluster of *P. aeruginosa* IATS serotype O13/O14 strain

| Gene/ORF | Related proteins (% identity) | Proposed function |
|----------|-------------------------------|--------------------|
| wzz/orf4 | 54% *P. aeruginosa* O5 Wzz     | Chain length regulator |
| wbpO/orf5 | 81% *P. aeruginosa* O6 WbpO   | UDP-D-GlcNAc/UDP-D-GalNAc 6-dehydrogenase |
| wbpP/orf6 | 69% *P. aeruginosa* O6 WbpP   | UDP-D-GlcNAcA/UDP-D-GalNAcA 4-epimerase |
| wzx/orf7 | 25% *P. aeruginosa* O5 Wzx     | Und-PP-O unit flippase |
| wzy/orf8 | Contains predicted            | O-Antigen polymerase |
|          | transmembrane domains         |                    |
| migA/orf9 | 26% *P. aeruginosa* O5 MigA   | Glycosyltransferase (GT-2), probably of L-Rha |
| wbpR/orf10 | 32% *P. aeruginosa* O6 WbpR  | Glycosyltransferase (GT-4), probably of L-Rha |
| wbpU/orf11 | 88% (O13) or 87% (O14)       | Glycosyltransferase (GT-4), |
|          | *P. aeruginosa* O6 WbpU        | probably of D-GalNAc(3OAc)A |
| wbpV/orf12 | 82% *P. aeruginosa* O6 WbpV  | UDP-4-keto-D-QuiNAc 4-reductase |
|          |                               | (UDP-D-QuiNAc synthesising) |
| wbpL/orf14 | 62% *P. aeruginosa* O5 WbpL  | Initiating glycosyl-1-phosphate transferase |
| wbpM/orf15 | 97% *P. aeruginosa* O5 WbpM  | UDP-D-GlcNAc 4,6-dehydratase |

Glycosyltransferase (GT) family classification is taken from the CAZy database.¹²³

is likely complemented by another *wzy* encoded elsewhere in the genome.

Unlike serotype O13, serotype O14 produces O-Antigen with a lateral D-Glc residue (Table 4), which is probably derived from UDP-D-Glc and produced by enzymes involved in core biosynthesis. Collectively, these predictions have allowed us to present a complete annotation of the genes putatively required for sugar-nucleotide biosynthesis in the serotype O13/O14 O-Antigen locus (Table 15).

**Glycosyltransferases encoded in the O-Antigen clusters**

Assignment of the putative glycosyltransferases is problematic in all serotypes. Their classification into glycosyltransferase families provides some clues, but annotation of their specific substrates will require additional study. Little diversity of glycosyltransferase families is observed within the *P. aeruginosa* O-Antigen biosynthesis clusters, all belonging to either CAZy family GT-2 or GT-4. According to the CAZy database, ${}^{123}$ GT-2 enzymes are expected to use the inverting mechanism and have the GT-A fold, while GT-4 proteins are predicted to use retaining chemistry and have the GT-B fold.

The ability to predict accurately the inverting or retaining mechanism of glycosyltransferases based on their families would be very useful in assigning roles to individual gene products. For example, in serotype O1, the chemistry of the sugar linkages (Table 4) suggests that UDP-D-GlcNAc3NAcA should be transferred by an inverting glycosyltransferase to form the β-(1→3) linkage, while the others are expected to be transferred by retaining glycosyltransferases. Orf10, Orf13, and Orf15 are candidate glycosyltransferases in serotype O1 (Table 13), but only Orf13 belongs to an inverting family (GT-2). Orf10 and Orf15 belong to a retaining family, GT-4. Thus, this analysis suggests that Orf13 transfers D-GlcNAc3NAcA, and either Orf10 or Orf15 transfer D-GalNAc or D-FucNAc. Based on its role as the initiating glycosyltransferase, WbpL${}_{O_{1}}$ is expected to transfer D-QuiNAc, the first sugar of the repeating O-unit in serotype O1.

However, in other serotypes, the glycosyltransferase family assignments appear to be inconsistent with the O-repeat structures, casting some general doubt on the utility of the CAZy group assignments. For example, serotype O4 appears to require three inverting glycosyltransferases to add the three L-sugars in α-linkages (Table 4). However, of the putative glycosyltransferases Orf7, Orf8, and $wbjE_{04}$ (Table 14), only Orf7 belongs to an inverting glycosyltransferase family (GT-2). Orf8 has not been placed in any CAZy family, and $wbjE_{04}$ belongs to the retaining glycosyltransferase family, GT-4. Despite the CAZy assignment of $wbjE$ to a retaining

glycosyltransferase family, its 25% sequence identity to $WbjE_{O_{11}}$ (a proposed L-FucNAc transferase of serotype O11) and its position downstream of genes required for UDP-L-FucNAc biosynthesis ($wbjB$, $wbjC$, and $wbjD$; Fig. S4) lead us to propose that $wbjE$ is an inverting L-FucNAc transferase.

In general, each of the O-Antigen clusters contains the correct number of putative glycosyltransferases as would be predicted from the number of sugar residues in the O-repeat. Serotype O14 is an exception, which produces a pentasaccharide repeat unit (Table 4) but contains only four putative glycosyltransferases in the O-Antigen biosynthesis cluster (Table 15). This is an unusual case because the fifth sugar, the lateral Glc, is not present in the core + 1 O-unit (semi-rough LPS); this observation has not yet been explained. However, in all serotypes except O14, the agreement between the number of putative glycosyltransferase-encoding genes and the number of required glycosyltransferases indicates that assignment of the function as glycosyltransfer is probably correct, even if bioinformatics alone cannot determine the substrate or mechanism. This highlights one of the shortcomings of sequence analysis and underlines the importance of functional studies.

**Export, polymerisation and chain-length regulation of O-Specific Antigen**

After synthesis of the O-repeat units on their undecaprenyl pyrophosphate carriers, they must be polymerised and transported before ligation to lipid A-core. The proteins encoded by three genes, *wzx wzy* and *wzz*, are required for these processes.

**Transport of O-Antigen units across the inner membrane: *wzx***

The first studies of *wzx* were conducted in *S. enterica* and demonstrated that a strain carrying a *wzx* mutation in a plasmid-borne copy of the O-Antigen biosynthesis locus accumulated undecaprenol-pyrophosphate linked to a single O-Antigen repeat unit on the cytoplasmic side of the inner membrane. ${}^{260}$ Due to this phenotype, Wzx is proposed to catalyze transmembrane transport of undecaprenol-pyrophosphate linked O units, and has been called the O-unit flippase. Burrows and Lam ${}^{261}$ identified a putative *wzx* gene in *P. aeruginosa* PAO1 (serotype O5) based on the high hydrophobicity and large number of expected transmembrane spanning segments in the predicted protein sequence. Analysis of chromosomal knockout mutants of *wzx* showed that the mutants did not produce O Antigen. Strangely, the *wzx* knockouts also demonstrated a marked delay in the production of the Common Polysaccharide Antigen, which may be due to effects on WbpL since providing

an additional copy of *wbpL* in trans could complement the delay.²⁶¹

Wzx proteins from some O-Antigen systems in the Enterobacteriaceae appear to have specificity for the first sugar attached to undecaprenyl-phosphate,²⁶² regardless of the structure of the remainder of the subunit.²⁶³ *E. coli* K-12/O16 produces LPS in which the first sugar is D-GlcNAc; in these experiments, *wzx* from *S. enterica* LT2 (which has D-Gal as the initial sugar) could not complement a knockout in *wzx* of *E. coli* K-12/O16.²⁶² However, *wzx* from *S. flexneri* 2a or *E. coli* serotypes that use D-GlcNAc or D-GalNAc as the initial sugar could complement the K-12/O16 *wzx* knockout. Of particular interest is the finding that *wzx* of *P. aeruginosa* PAO1, which uses D-FucNAc as the first sugar residue, could complement the *wzx* knockout in *E. coli* K-12/O16.²⁶² This suggests that *P. aeruginosa* PAO1 Wzx belongs to the proposed subfamily of Wzx proteins that recognizes O units initiated by N-acetylhexosamines.²⁶²

Since *P. aeruginosa* serotypes mostly use either D-FucNAc or D-QuiNAc as the initiating sugar, we compared protein sequences to see if any correlation between Wzx sequence and the initiating sugar could be detected. If any correlation were detected, it would suggest that closely related Wzx proteins may share an adaptation for transport of substrates with the same initiating sugar. PSI-BLAST analysis was used to identify homologues of Wzx from serotype O6 (*Wzx*₀₆), which was selected as an example of D-QuiNAc-initiated repeat units (Table 16). Almost all of the *P. aeruginosa* serotypes that produce D-QuiNAc-initiated repeat units were found to be homologous to *Wzx*₀₆. Only serotypes O10/O19 (which have identical *Wzx* sequences) were not aligned with *Wzx*₀₆ and instead were shown through separate searches to share 27% identity to *Wzx* of serotype O11, which has D-FucNAc as the initial sugar. When the same strategy was used to identify homologues of Wzx from PAO1 (serotype O5; *Wzx*₀₅) as an example of D-FucNAc-initiated repeat units, only the closely related serogroup

O2/O5/O16/O18/O20 *Wzx* sequences were identified (identity 99–100%). Serotypes O7/O8 *Wzx* sequences are identical to each other, but produced no significant hits to any other *Wzx* of *P. aeruginosa*. Thus, the predicted *Wzx* amino acid sequences cluster into four groups: O5-like (serogroup O2/O5/O16/O18/O20), O6-like (all strains with D-QuiNAc initiated O units except O10/O19), O11-like (O11 and O10/O19), and O7/O8. Of these, only the O11-like group contains members that use different initial sugars. Experimentation is required to determine whether the conservation of primary sequence between *P. aeruginosa* *Wzx* protein sequences genuinely indicates any substrate specificity.

### O-Antigen polymerase: *wzy*

A knockout of *wzy* produces semi-rough LPS (LPS with only one O-repeat unit). This phenotype has been described in knockout studies in *P. aeruginosa* PAO1 (serotype O5)¹⁶³ and *P. aeruginosa* PA103 (serotype O11).²⁰³ Since *wzy* is necessary for polymerization of O-Antigen, it has been called the O-Antigen polymerase gene despite the lack of biochemical evidence for polymerase activity of the encoded protein from any bacterial species. The *wzy* (formerly *rfc*) genes from PAO1 and PA103 show no significant similarity at the nucleotide or protein level, but are predicted to share very high hydrophobicity and topology with 9–12 transmembrane spanning segments. Both characteristics are typical of other bacterial O-Antigen polymerases.²⁶⁴,²⁶⁵ Of all the well-studied serotypes, only O6 does not have a putative *wzy* gene within the O-Antigen cluster;¹⁶⁹ presumably, the gene is located elsewhere in the genome.

The closely related serogroup O2/O5/O16/O18/O20 have nearly identical O-Antigen biosynthesis clusters that contain a single *wzy* gene (Table 6).¹⁶¹ This serogroup produces O units that are similar in sugar composition, but with different linkages between the repeat units: serotypes O5, O18, and O20 have α-linked O units, whereas serotypes O2 and O16 have β-linked O units.²¹⁷ The *wzy* gene of the O-Antigen biosynthesis

---

**Table 16. Amino acid sequence identities to *Wzx*₀₆ among *P. aeruginosa* serotypes with D-QuiNAc (or derivatives) as the initial sugar**

| Serotype | ORF in O-antigen biosynthesis cluster | Initial sugar | Sequence identity with *Wzx*₀₆ |
|----------|---------------------------------------|---------------|--------------------------------|
| O1       | Orf5                                  | D-QuiNAc      | 21% over 147 amino acids      |
| O3       | Orf7                                  | D-QuiNAc4NSHb | 19% over 204 amino acids      |
| O4       | Orf5                                  | D-QuiNAc      | 20% over 240 amino acids      |
| O9       | Orf5                                  | D-QuiNAc      | 19% over 393 amino acids      |
| O10/O19  | Orf7                                  | D-QuiNAc      | –, (27% with *Wzx*₁₁)         |
| O12      | Orf15                                 | D-QuiNAc      | 13% over 375 amino acids      |
| O13/O14  | Orf7                                  | D-QuiNAc      | 25% over 312 amino acids      |

Downloaded from [ini.sagepub.com](http://ini.sagepub.com) at Scientific library of Moscow State University on February 10, 2014
cluster (PA3154) has, therefore, sometimes been designated $wzy_{\alpha}$ to emphasise the proposed role of the encoded protein in polymerizing $\alpha$-linked O-repeat units. An alternate form of Wzy, encoded by $wzy_{\beta}$ elsewhere in the genome, has been proposed to be responsible for the polymerization of O units with $\beta$-linkages.²⁵⁹

Reverse-transcriptase-PCR showed that the $wzy_{\alpha}$ gene in the O-Antigen biosynthesis cluster of *P. aeruginosa* serotypes O2 and O16 is transcribed, despite the fact that these two serotypes have $\beta$-linked O-units. The predicted protein sequence for $wzy_{\alpha}$ of serotypes O2 and O16 is 100% identical to that of O5 (which has $\alpha$-linked O-units) and any of the three genes are sufficient to complement a $wzy_{\alpha}$ knockout in serotype O5.²⁵⁹ Serotypes O2 and O16 were each found to contain a $wzy_{\beta}$ gene encoding identical Wzy$_{\beta}$ proteins that is located outside of the O-Antigen biosynthesis cluster. The O16 $wzy_{\beta}$ gene was sufficient to cause the production of $\beta$-linked O-Antigen repeat units when expressed in an O5 $wzy_{\alpha}$ knockout strain.²⁵⁹ The expressed and functional Wzy$_{\alpha}$ of serotype O2 and O16 does not synthesize $\alpha$-linked repeat units in these strains because of the presence of an inhibitor. This inhibitor of the $\alpha$-polymerase, *iap*, was initially described as part of a three-component cassette from bacteriophage D3 that was required for serotype conversion of *P. aeruginosa* from O5 to O16 and O-acetylation of the D-FucNAc moiety.²⁶⁶ Chromosomal copies of *Iap* were located in serotypes O2 and O16, suggesting that *iap* inactivates Wzy$\alpha$ while leaving Wzy$_{\beta}$ functional.²⁵⁹

### O Antigen chain-length regulator: *wzz*

The O Antigen could theoretically contain any number of repeat units if polymerisation were not regulated. In wild-type *P. aeruginosa* serotype O5, a characteristic modal distribution of O-Antigen lengths exists, with O chains of approximately 12–16, 22–30, or 40–50 repeat units being the most common. The *wzz* genes of *P. aeruginosa* were initially studied in serotypes O5 and O16; knockouts of the O-Antigen cluster *wzz* synthesize O Antigens with a different distribution of chain lengths than the wild-type strains.²⁶⁷ The *wzz* mutant in serotype O5 produced proportionally more short O chains of approximately 1–12 repeat units, and *wzz* mutants in both O5 and O16 produced less mid-length O chains of roughly 12–30 repeat units. Thus, the average O-Antigen chain length was altered in these *wzz* knockouts, but it seemed that some chain length modulation remained. The predicted Wzz protein of O5 has roughly 20% identity with Wzz of *E. coli*, *S. enterica*, and *S. flexneri*, and was able to function in *E. coli* to alter polysaccharide chain lengths when expressed in trans.²⁶⁷ The mode of action of Wzz proteins is unknown but they all share similar topology and are required for regulation of the chain lengths of both O Antigens and capsules.

*Pseudomonas aeruginosa* PAO1 possesses *wzz₁* (PA3160) and *wzz₂* (PA0938) that encode two distinct functional Wzz proteins.²⁶⁸ Each Wzz protein governs the production of LPS with different modal lengths; Wzz₁ imparts LPS with a bimodal distribution of 12–16 or 22–30 repeat units, while Wzz₂ imparts LPS with longer modal lengths (~40–50 repeat units).²⁶⁸ A strain lacking *wzz₂* and unable to produce normal core oligosaccharide (*wzz₂ rmlC* double mutant) was shown to lose wild-type modality and produce only undecaprenol-linked O Antigen.²⁶⁸ When this strain was complemented with *wzz₂*, the O Antigen was still undecaprenol-linked due to the absence of core oligosaccharide acceptors for O polysaccharide caused by the *rmlC* knockout, but wild-type modality of the undecaprenol-linked bands was restored. Thus, Wzz directs the preferential polymerization of modal chain lengths before the O Antigen is ligated to the core oligosaccharide.²⁶⁸ Wzz proteins were also found to co-precipitate with undecaprenol-linked O Antigens, indicating a direct or indirect interaction with the O-Antigen chain.²⁶⁸

A *wzz₁* homologue was identified using the GENEMARK algorithm;¹⁶¹ in all of the functional O-Antigen biosynthesis clusters. Using polyclonal antisera raised against Wzz₂ of PAO1, which is located outside the O-Antigen cluster, Wzz₂ was detected in all serotypes.²⁶⁸ The proposed *wzz₁* homologues share >40% identity to the *wzz₁* from PAO1, with the poorest conservation (41% identity) from serotype O11. Recent studies of *P. aeruginosa* PA103 (serotype O11) revealed the sequence of *wzz₂* of PA103, which shares 92% amino acid similarity with *wzz₂* of PAO1.²⁶⁹ Knockout mutants lacking *wzz₁* or *wzz₂* were constructed in PA103, and were shown to lack wild-type chain length distributions. Specifically, *wzz₁* was determined to produce O Antigens of ‘long’ chain length, and *wzz₂* was determined to produce O Antigens of ‘very long’ chain length,²⁶⁹ which is consistent with the observations made in an earlier study of these genes in PAO1.²⁶⁸ The PA103 *wzz₁* mutant demonstrated reduced virulence than either the wild-type PA103 or the *wzz₂* mutant, based on serum sensitivity assays and a mouse pneumonia model study, suggesting that the ‘long’ chain length of the O Antigen is particularly important for pathogenesis.²⁶⁹

### Ligation of O Antigen to lipid A-core: *waaL*

By different mechanisms, the Common Polysaccharide Antigen and O-specific polysaccharides are assembled and delivered as complete polymers to the periplasmic side of the inner membrane. The final step in
304 King, Kocínková, Westman, Lam

O-polysaccharide biosynthesis is the ligation of these O chains to lipid A-core, in a reaction which requires the product of the *waaL* gene. WaaL is known as the O-polysaccharide ligase (reviewed by Raetz and Whitfield⁵⁰). There is little primary sequence conservation between the enzymes which perform this function, but identification of the *P. aeruginosa waaL* was possible because these proteins do share very similar hydrophathy profiles, which are characterised by the presence of approximately 11 predicted transmembrane helices and a 50–100-amino acid loop near the C terminus, normally predicted to be in the periplasm. In PAO1, the *waaL* gene (PA4999) is located in the core cluster described above. *Pseudomonas aeruginosa waaL* mutants do not produce Common Polysaccharide Antigen-containing or O-Specific Antigen-containing LPS, or the semi-rough form. In Western blotting analysis of whole cell lysates of these mutants, polymers are present which are recognised by Common Polysaccharide Antigen-specific and O-specific antibodies. However, in aqueous phenol LPS extractions these polymers are not detected, suggesting that these bands are O-polysaccharide chains linked to an isoprenyl-pyrophosphate lipid carrier, that are probably hydrolysed in hot aqueous phenol.³⁵ Site-directed mutagenesis of conserved amino acids in the probable periplasmic loop of *P. aeruginosa* WaaL showed that His303 is important for ligase function in complementation experiments, but that another positively-charged amino acid side chain can be placed in this position without loss of function. The *P. aeruginosa* WaaL enzyme was purified and shown to hydrolyse ATP *in vitro* suggesting that the O-Antigen ligation reaction may be an ATP-dependent process. Site-directed mutagenesis studies showed that conserved amino acids in what had been proposed as potential ATP binding/hydrolysis motifs,²⁷⁰ were not essential for ligation by the *E. coli* WaaL.²⁷¹ Therefore, further work is required to establish the link between ATP hydrolysis and the ligation reaction. *Escherichia coli* K-12 *waaL* cannot cross-complement a *P. aeruginosa waaL* mutation.²⁷⁰ Since the outer core structures of *E. coli* and *P. aeruginosa* are different, the failure to cross-complement is probably due to specificity of WaaL enzymes for structural features of their cognate core oligosaccharide acceptors.²⁷²

very detailed characterisation of LPS structures from a large number of wild-type and mutant *P. aeruginosa* strains have provided a valuable opportunity to advance our understanding of LPS genetics. Many features of *P. aeruginosa* LPS structures and pathways are conserved in surface polysaccharides of pathogenic bacteria, so the work summarised in this review also illuminates the biosynthesis of virulence factors in other medically important organisms.

### ACKNOWLEDGEMENTS

Research in our laboratory is supported by a grant from the Canadian Institutes of Health Research (CIHR; MOP 14687) and an operating grant from the Canadian Cystic Fibrosis Foundation (CCFF). ELW holds a CIHR Canada Graduate Scholarship Doctoral Research Award, DK holds a CCFF Postdoctoral Fellowship, and JSL holds a Canada Research Chair in Cystic Fibrosis and Microbial Glycobiology, funded jointly by the Canadian Foundation for Innovation and the Ontario Innovation Trust.

### REFERENCES

1. Villavicencio RT. The history of blue pus. *J Am Coll Surg* 1998; **187**: 212–216.
2. Cardo D, Horan T, Andrus M *et al.* National Nosocomial Infections Surveillance (NNIS) System Report, data summary from January 1992 through June 2004, issued October 2004. *Am J Infect Control* 2004; **32**: 470–485.
3. Govan JR, Deretic V. Microbial pathogenesis in cystic fibrosis: mucoid *Pseudomonas aeruginosa* and *Burkholderia cepacia*. *Microbiol Rev* 1996; **60**: 539–574.
4. Lang AB, Horn MP, Imboden MA, Zuercher AW. Prophylaxis and therapy of *Pseudomonas aeruginosa* infection in cystic fibrosis and immunocompromised patients. *Vaccine* 2004; **22** (Suppl 1): S44–S48.
5. Taneja N, Emmanuel R, Chari PS, Sharma M. A prospective study of hospital-acquired infections in burn patients at a tertiary care referral centre in North India. *Burns* 2004; **30**: 665–669.
6. Lyczak JB, Cannon CL, Pier GB. Establishment of *Pseudomonas aeruginosa* infection: lessons from a versatile opportunist. *Microbes Infect* 2000; **2**: 1051–1060.
7. Poole K. Efflux-mediated multiresistance in Gram-negative bacteria. *Clin Microbiol Infect* 2004; **10**: 12–26.
8. Yoshimura F, Nikaido H. Permeability of *Pseudomonas aeruginosa* outer membrane to hydrophilic solutes. *J Bacteriol* 1982; **152**: 636–642.
9. Nikaido H, Hancock REW. Outer membrane permeability of *Pseudomonas aeruginosa*. In: Sokatch JR. (ed). *The Bacteria, a Treatise on Structure and Function*. Orlando, FL: Academic Press, 1986; 145–193.
10. El’Garch F, Jeannot K, Hocquet D, Llanes-Barakat C, Plesiat P. Cumulative effects of several nonenzymatic mechanisms on the resistance of *Pseudomonas aeruginosa* to aminoglycosides. *Antimicrob Agents Chemother* 2007; **51**: 1016–1021.

### CONCLUDING REMARKS

*Pseudomonas aeruginosa* LPS biosynthesis has proven to be an excellent biological system in which to study polysaccharide biosynthesis and many details are now known about the biogenesis of this macromolecule. In the last decade, the publication of the O-Antigen cluster sequences, the annotation of the PAO1 genome, and the

11. Giamarellou H. Prescribing guidelines for severe *Pseudomonas* infections. *J Antimicrob Chemother* 2002; **49**: 229–233.
12. Foweraker J. Recent advances in the microbiology of respiratory tract infection in cystic fibrosis. *Br Med Bull* 2009; **89**: 93–110.
13. Rocchetta HL, Burrows LL, Lam JS. Genetics of O-antigen biosynthesis in *Pseudomonas aeruginosa*. *Microbiol Mol Biol Rev* 1999; **63**: 523–553.
14. Stanislavsky ES, Lam JS. *Pseudomonas aeruginosa* antigens as potential vaccines. *FEMS Microbiol Rev* 1997; **21**: 243–277.
15. Galanos C, Lüderitz O, Rietschel ET *et al.* Synthetic and natural *Escherichia coli* free lipid A express identical endotoxic activities. *Eur J Biochem* 1985; **148**: 1–5.
16. Alexander C, Rietschel ET. Bacterial lipopolysaccharides and innate immunity. *J Endotoxin Res* 2001; **7**: 167–202.
17. Backhed F, Normark S, Schweda EK, Oscarson S, Richter-Dahlfors A. Structural requirements for TLR4-mediated LPS signalling: a biological role for LPS modifications. *Microbes Infect* 2003; **5**: 1057–1063.
18. Hajjar AM, Ernst RK, Tsai JH, Wilson CB, Miller SI. Human Toll-like receptor 4 recognizes host-specific LPS modifications. *Nat Immunol* 2002; **3**: 354–359.
19. Ernst RK, Hajjar AM, Tsai JH, Moskowitz SM, Wilson CB, Miller SI. *Pseudomonas aeruginosa* lipid A diversity and its recognition by Toll-like receptor 4. *J Endotoxin Res* 2003; **9**: 395–400.
20. Hancock RE, Diamond G. The role of cationic antimicrobial peptides in innate host defences. *Trends Microbiol* 2000; **8**: 402–410.
21. Trent MS, Stead CM, Tran AX, Hankins JV. Diversity of endotoxin and its impact on pathogenesis. *J Endotoxin Res* 2006; **12**: 205–223.
22. Ernst RK, Yi EC, Guo L *et al.* Specific lipopolysaccharide found in cystic fibrosis airway *Pseudomonas aeruginosa*. *Science* 1999; **286**: 1561–1565.
23. McPhee JB, Lewenza S, Hancock RE. Cationic antimicrobial peptides activate a two-component regulatory system, PmrA-PmrB, that regulates resistance to polymyxin B and cationic antimicrobial peptides in *Pseudomonas aeruginosa*. *Mol Microbiol* 2003; **50**: 205–217.
24. Pier GB, Grout M, Zaidi TS, Goldberg JB. How mutant CFTR may contribute to *Pseudomonas aeruginosa* infection in cystic fibrosis. *Am J Respir Crit Care Med* 1996; **154**: S175–S182.
25. Willcox MD. *Pseudomonas aeruginosa* infection and inflammation during contact lens wear: a review. *Optom Vis Sci* 2007; **84**: 273–278.
26. Fleiszig SM, Zaidi TS, Fletcher EL, Preston MJ, Pier GB. *Pseudomonas aeruginosa* invades corneal epithelial cells during experimental infection. *Infect Immun* 1994; **62**: 3485–3493.
27. Fleiszig SM, Zaidi TS, Pier GB. *Pseudomonas aeruginosa* invasion of and multiplication within corneal epithelial cells in vitro. *Infect Immun* 1995; **63**: 4072–4077.
28. Zaidi TS, Fleiszig SM, Preston MJ, Goldberg JB, Pier GB. Lipopolysaccharide outer core is a ligand for corneal cell binding and ingestion of *Pseudomonas aeruginosa*. *Invest Ophthalmol Vis Sci* 1996; **37**: 976–986.
29. Zaidi TS, Lyczak J, Preston M, Pier GB. Cystic fibrosis transmembrane conductance regulator-mediated corneal epithelial cell ingestion of *Pseudomonas aeruginosa* is a key component in the pathogenesis of experimental murine keratitis. *Infect Immun* 1999; **67**: 1481–1492.
30. Pier GB, Grout M, Zaidi TS. Cystic fibrosis transmembrane conductance regulator is an epithelial cell receptor for clearance of *Pseudomonas aeruginosa* from the lung. *Proc Natl Acad Sci USA* 1997; **94**: 12088–12093.
31. Schroeder TH, Reiniger N, Meluleni G, Grout M, Coleman FT, Pier GB. Transgenic cystic fibrosis mice exhibit reduced early clearance of *Pseudomonas aeruginosa* from the respiratory tract. *J Immunol* 2001; **166**: 7410–7418.
32. Dasgupta T, de Kievit TR, Masoud H *et al.* Characterization of lipopolysaccharide-deficient mutants of *Pseudomonas aeruginosa* derived from serotypes O3, O5, and O6. *Infect Immun* 1994; **62**: 809–817.
33. Lam MY, McGroarty EJ, Kropinski AM *et al.* Occurrence of a common lipopolysaccharide antigen in standard and clinical strains of *Pseudomonas aeruginosa*. *J Clin Microbiol* 1989; **27**: 962–967.
34. Berry MC, McGhee GC, Zhao Y, Sundin GW. Effect of a *waaL* mutation on lipopolysaccharide composition, oxidative stress survival, and virulence in *Erwinia amylovora*. *FEMS Microbiol Lett* 2009; **291**: 80–87.
35. Abeyrathne PD, Daniels C, Poon KK, Matewish MJ, Lam JS. Functional characterization of WaaL, a ligase associated with linking O-antigen polysaccharide to the core of *Pseudomonas aeruginosa* lipopolysaccharide. *J Bacteriol* 2005; **187**: 3002–3012.
36. Hancock RE, Mutharia LM, Chan L, Darveau RP, Speert DP, Pier GB. *Pseudomonas aeruginosa* isolates from patients with cystic fibrosis: a class of serum-sensitive, nontypable strains deficient in lipopolysaccharide O side chains. *Infect Immun* 1983; **42**: 170–177.
37. Evans DJ, Pier GB, Coyne MJR, Goldberg JB. The *rfb* locus from *Pseudomonas aeruginosa* strain PA103 promotes the expression of O antigen by both LPS-rough and LPS-smooth isolates from cystic fibrosis patients. *Mol Microbiol* 1994; **13**: 427–434.
38. Kadurugamuwa JL, Lam JS, Beveridge TJ. Interaction of gentamicin with the A band and B band lipopolysaccharides of *Pseudomonas aeruginosa* and its possible lethal effect. *Antimicrob Agents Chemother* 1993; **37**: 715–721.
39. Hatano K, Goldberg JB, Pier GB. Biologic activities of antibodies to the neutral-polysaccharide component of the *Pseudomonas aeruginosa* lipopolysaccharide are blocked by O side chains and mucoid exopolysaccharide (alginate). *Infect Immun* 1995; **63**: 21–26.
40. Matewish MJ. *The functional role of lipopolysaccharide in the cell envelope and surface proteins of Pseudomonas aeruginosa*. PhD thesis. University of Guelph, Ontario, Canada, 2004.
41. Cryz SJr, Pitt TL, Furer E, Germanier R. Role of lipopolysaccharide in virulence of *Pseudomonas aeruginosa*. *Infect Immun* 1984; **44**: 508–513.
42. Priebe GP, Dean CR, Zaidi T *et al.* The *galU* gene of *Pseudomonas aeruginosa* is required for corneal infection and efficient systemic spread following pneumonia but not for infection confined to the lung. *Infect Immun* 2004; **72**: 4224–4232.
43. Tang HB, DiMango E, Bryan R *et al.* Contribution of specific *Pseudomonas aeruginosa* virulence factors to pathogenesis of pneumonia in a neonatal mouse model of infection. *Infect Immun* 1996; **64**: 37–43.
44. Coyne MJR, Russell KS, Coyle CL, Goldberg JB. The *Pseudomonas aeruginosa algC* gene encodes phosphoglucomutase, required for the synthesis of a complete lipopolysaccharide core. *J Bacteriol* 1994; **176**: 3500–3507.
45. Knirel YA, Bystrova OV, Kocharova NA, Zähringer U, Pier GB. Conserved and variable structural features in the lipopolysaccharide of *Pseudomonas aeruginosa*. *J Endotoxin Res* 2006; **12**: 324–336.
46. Kulshin VA, Zähringer U, Lindner B, Jäger KE, Dmitriev BA, Rietschel ET. Structural characterization of the lipid A component

306 King, Kocínková, Westman, Lam

of *Pseudomonas aeruginosa* wild-type and rough mutant lipopolysaccharides. *Eur J Biochem* 1991; **198**: 697–704.

47. Ernst RK, Moskowitz SM, Emerson JC *et al.* Unique lipid A modifications in *Pseudomonas aeruginosa* isolated from the airways of patients with cystic fibrosis. *J Infect Dis* 2007; **196**: 1088–1092.

48. Bhat R, Marx A, Galanos C, Conrad RS. Structural studies of lipid A from *Pseudomonas aeruginosa* PAO1: occurrence of 4-amino-4-deoxyarabinose. *J Bacteriol* 1990; **172**: 6631–6636.

49. Moskowitz SM, Ernst RK, Miller SI. PmrAB, a two-component regulatory system of *Pseudomonas aeruginosa* that modulates resistance to cationic antimicrobial peptides and addition of aminoarabinose to lipid A. *J Bacteriol* 2004; **186**: 575–579.

50. Raetz CR, Whitfield C. Lipopolysaccharide endotoxins. *Annu Rev Biochem* 2002; **71**: 635–700.

51. Trent MS. Biosynthesis, transport, and modification of lipid A. *Biochem Cell Biol* 2004; **82**: 71–86.

52. Dotson GD, Kaltashov IA, Cotter RJ, Raetz CR. Expression cloning of a *Pseudomonas* gene encoding a hydroxydecanoyl-acyl carrier protein-dependent UDP-GlcNAc acyltransferase. *J Bacteriol* 1998; **180**: 330–337.

53. Williamson JM, Anderson MS, Raetz CR. Acyl-acyl carrier protein specificity of UDP-GlcNAc acyltransferases from Gram-negative bacteria: relationship to lipid A structure. *J Bacteriol* 1991; **173**: 3591–3596.

54. Wyckoff TJ, Lin S, Cotter RJ, Dotson GD, Raetz CR. Hydrocarbon rulers in UDP-N-acetylglucosamine acyltransferases. *J Biol Chem* 1998; **273**: 32369–32372.

55. Clementz T, Bednarski JJ, Raetz CR. Function of the *htrB* high temperature requirement gene of *Escherichia coli* in the acylation of lipid A: HtrB catalyzed incorporation of laurate. *J Biol Chem* 1996; **271**: 12095–12102.

56. Clementz T, Zhou Z, Raetz CR. Function of the *Escherichia coli msbB* gene, a multicopy suppressor of *htrB* knockouts, in the acylation of lipid A. Acylation by MsbB follows laurate incorporation by HtrB. *J Biol Chem* 1997; **272**: 10353–10360.

57. Brozek KA, Raetz CR. Biosynthesis of lipid A in *Escherichia coli*. Acyl carrier protein-dependent incorporation of laurate and myristate. *J Biol Chem* 1990; **265**: 15410–15417.

58. Goldman RC, Doran CC, Kadam SK, Capobianco JO. Lipid A precursor from *Pseudomonas aeruginosa* is completely acylated prior to addition of 3-deoxy-D-manno-octulosonate. *J Biol Chem* 1988; **263**: 5217–5223.

59. Mohan S, Raetz CR. Endotoxin biosynthesis in *Pseudomonas aeruginosa*: enzymatic incorporation of laurate before 3-deoxy-D-manno-octulosonate. *J Bacteriol* 1994; **176**: 6944–6951.

60. Guo L, Lim KB, Poduje CM *et al.* Lipid A acylation and bacterial resistance against vertebrate antimicrobial peptides. *Cell* 1998; **95**: 189–198.

61. Gibbons HS, Lin S, Cotter RJ, Raetz CR. Oxygen requirement for the S-2-hydroxymyristate moiety in *Salmonella typhimurium* lipid A. Function of LpxO, A new Fe²⁺/alpha-ketoglutarate-dependent dioxygenase homologue. *J Biol Chem* 2000; **275**: 32940–32949.

62. Gibbons HS, Reynolds CM, Guan Z, Raetz CR. An inner membrane dioxygenase that generates the 2-hydroxymyristate moiety of *Salmonella* lipid A. *Biochemistry* 2008; **47**: 2814–2825.

63. Gibbons HS, Kalb SR, Cotter RJ, Raetz CR. Role of Mg²⁺ and pH in the modification of *Salmonella* lipid A after endocytosis by macrophage tumour cells. *Mol Microbiol* 2005; **55**: 425–440.

64. Trent MS, Pabich W, Raetz CR, Miller SI. A PhoP/PhoQ-induced lipase (PagL) that catalyzes 3-O-deacylation of lipid A precursors in membranes of *Salmonella typhimurium*. *J Biol Chem* 2001; **276**: 9083–9092.

65. Geurtsen J, Steeghs L, Hove JT, van der Ley P, Tommassen J. Dissemination of lipid A deacylases (PagL) among Gram-negative bacteria: identification of active-site histidine and serine residues. *J Biol Chem* 2005; **280**: 8248–8259.

66. Ernst RK, Adams KN, Moskowitz SM *et al.* The *Pseudomonas aeruginosa* lipid A deacylase: selection for expression and loss within the cystic fibrosis airway. *J Bacteriol* 2006; **188**: 191–201.

67. Rutten L, Geurtsen J, Lambert W *et al.* Crystal structure and catalytic mechanism of the LPS 3-O-deacylase PagL from *Pseudomonas aeruginosa*. *Proc Natl Acad Sci USA* 2006; **103**: 7071–7076.

68. Yan A, Guan Z, Raetz CR. An undecaprenyl phosphate-aminoarabinose flippase required for polymyxin resistance in *Escherichia coli*. *J Biol Chem* 2007; **282**: 36077–36089.

69. Hung RJ, Chien HS, Lin RZ *et al.* Comparative analysis of two UDP-glucose dehydrogenases in *Pseudomonas aeruginosa* PAO1. *J Biol Chem* 2007; **282**: 17738–17748.

70. McPhee JB, Bains M, Winsor G *et al.* Contribution of the PhoP-PhoQ and PmrA-PmrB two-component regulatory systems to Mg²⁺-induced gene regulation in *Pseudomonas aeruginosa*. *J Bacteriol* 2006; **188**: 3995–4006.

71. Nummiola K, Kilpeläinen I, Zähringer U, Vaara M, Helander IM. Lipopolysaccharides of polymyxin B-resistant mutants of *Escherichia coli* are extensively substituted by 2-aminoethyl pyrophosphate and contain aminoarabinose in lipid A. *Mol Microbiol* 1995; **16**: 271–278.

72. Helander IM, Kato Y, Kilpeläinen I *et al.* Characterization of lipopolysaccharides of polymyxin-resistant and polymyxin-sensitive *Klebsiella pneumoniae* O3. *Eur J Biochem* 1996; **237**: 272–278.

73. Gunn JS, Lim KB, Krueger J *et al.* PmrA-PmrB-regulated genes necessary for 4-aminoarabinose lipid A modification and polymyxin resistance. *Mol Microbiol* 1998; **27**: 1171–1182.

74. McCoy AJ, Liu H, Falla TJ, Gunn JS. Identification of *Proteus mirabilis* mutants with increased sensitivity to antimicrobial peptides. *Antimicrob Agents Chemother* 2001; **45**: 2030–2037.

75. Derzelle S, Turlin E, Duchaud E *et al.* The PhoP-PhoQ two-component regulatory system of *Photorhabdus luminescens* is essential for virulence in insects. *J Bacteriol* 2004; **186**: 1270–1279.

76. Robey M, O’Connell W, Cianciotto NP. Identification of *Legionella pneumophila rcp*, a *pagP*-like gene that confers resistance to cationic antimicrobial peptides and promotes intracellular infection. *Infect Immun* 2001; **69**: 4276–4286.

77. Preston A, Maxim E, Toland E *et al.* *Bordetella bronchiseptica* PagP is a Bvg-regulated lipid A palmitoyl transferase that is required for persistent colonization of the mouse respiratory tract. *Mol Microbiol* 2003; **48**: 725–736.

78. Brozek KA, Bulawa CE, Raetz CR. Biosynthesis of lipid A precursors in *Escherichia coli*. A membrane-bound enzyme that transfers a palmitoyl residue from a glycerophospholipid to lipid X. *J Biol Chem* 1987; **262**: 5170–5179.

79. Bishop RE, Gibbons HS, Guina T, Trent MS, Miller SI, Raetz CR. Transfer of palmitate from phospholipids to lipid A in outer membranes of Gram-negative bacteria. *EMBO J* 2000; **19**: 5071–5080.

80. El Hamidi A, Novikov A, Karibian D, Perry MB, Caroff M. Structural characterization of *Bordetella parapertussis* lipid A. *J Lipid Res* 2009; **50**: 854–859.

81. MacArthur I, Mann PB, Harvill ET, Preston A. IEIIS Meeting minireview: *Bordetella* evolution: lipid A and Toll-like receptor 4. *J Endotoxin Res* 2007; **13**: 243–247.

82. Walsh AG, Matewish MJ, Burrows LL, Monteiro MA, Perry MB, Lam JS. Lipopolysaccharide core phosphates are required for

viability and intrinsic drug resistance in *Pseudomonas aeruginosa*. Mol Microbiol 2000; **35**: 718–727.

83. Schnaitman CA, Klena JD. Genetics of lipopolysaccharide biosynthesis in enteric bacteria. Microbiol Rev 1993; **57**: 655–682.

84. Eagon R, Carson K. Lysis of cell walls and intact cells of *Pseudomonas aeruginosa* by ethylenediamine tetraacetic acid and by lysosozyme. Can J Microbiol 1965; **11**: 193–201.

85. Gray GW, Wilkinson SG. The effect of ethylenediaminetetraacetic acid on the cell walls of some Gram-negative bacteria. J Gen Microbiol 1965; **39**: 385–399.

86. Wilkinson SG. Cell walls of *Pseudomonas* species sensitive to ethylenediaminetetraacetic acid. J Bacteriol 1970; **104**: 1035–1044.

87. Sanchez Carballo PM, Rietschel ET, Kosma P, Zähringer U. Elucidation of the structure of an alanine-lacking core tetrasaccharide trisphosphate from the lipopolysaccharide of *Pseudomonas aeruginosa* mutant H4. Eur J Biochem 1999; **261**: 500–508.

88. Knirel YA, Bystrova OV, Shashkov AS *et al.* Structural analysis of the lipopolysaccharide core of a rough, cystic fibrosis isolate of *Pseudomonas aeruginosa*. Eur J Biochem 2001; **268**: 4708–4719.

89. Bystrova OV, Shashkov AS, Kocharova NA *et al.* Structural studies on the core and the O-polysaccharide repeating unit of *Pseudomonas aeruginosa* immunotype 1 lipopolysaccharide. Eur J Biochem 2002; **269**: 2194–2203.

90. Kooistra O, Bedoux G, Brecker L *et al.* Structure of a highly phosphorylated lipopolysaccharide core in the Delta *algC* mutants derived from *Pseudomonas aeruginosa* wild-type strains PAO1 (serogroup O5) and PAC1R (serogroup O3). Carbohydr Res 2003; **338**: 2667–2677.

91. Beckmann F, Moll H, Jäger KE, Zähringer U. Preliminary communication 7-O-carbamoyl-L-glycero-D-manno-heptose: a new core constituent in the lipopolysaccharide of *Pseudomonas aeruginosa*. Carbohydr Res 1995; **267**: C3–C7.

92. Sadovskaya I, Brisson JR, Lam JS, Richards JC, Altman E. Structural elucidation of the lipopolysaccharide core regions of the wild-type strain PAO1 and O-chain-deficient mutant strains AK1401 and AK1012 from *Pseudomonas aeruginosa* serotype O5. Eur J Biochem 1998; **255**: 673–684.

93. Sadovskaya I, Brisson JR, Thibault P, Richards JC, Lam JS, Altman E. Structural characterization of the outer core and the O-chain linkage region of lipopolysaccharide from *Pseudomonas aeruginosa* serotype O5. Eur J Biochem 2000; **267**: 1640–1650.

94. Bystrova OV, Knirel YA, Lindner B *et al.* Structures of the core oligosaccharide and O-units in the R- and SR-type lipopolysaccharides of reference strains of *Pseudomonas aeruginosa* O-serogroups. FEMS Immunol Med Microbiol 2006; **46**: 85–99.

95. Bystrova OV, Lindner B, Moll H *et al.* Structure of the lipopolysaccharide of *Pseudomonas aeruginosa* O-12 with a randomly O-acetylated core region. Carbohydr Res 2003; **338**: 1895–1905.

96. Choudhury B, Carlson RW, Goldberg JB. The structure of the lipopolysaccharide from a *galU* mutant of *Pseudomonas aeruginosa* serogroup-O11. Carbohydr Res 2005; **340**: 2761–2772.

97. Drewry DT, Gray GW, Wilkinson SG. Low-molecular-weight solutes released during mild acid hydrolysis of the lipopolysaccharide of *Pseudomonas aeruginosa*. Biochem J 1972; **130**: 289–295.

98. Wilkinson SG. 31P N.M.R. evidence for the presence of triphosphate residues in lipopolysaccharides from *Pseudomonas aeruginosa*. Biochem J 1981; **199**: 833–835.

99. Choudhury B, Carlson RW, Goldberg JB. Characterization of the lipopolysaccharide from a *wbjE* mutant of the serogroup O11 *Pseudomonas aeruginosa* strain, PA103. Carbohydr Res 2008; **343**: 238–248.

100. Bystrova OV, Lindner B, Moll H *et al.* Structure of the biological repeating unit of the O-antigen of *Pseudomonas aeruginosa* immunotype 4 containing both 2-acetamido-2,6-dideoxy-D-glucose and 2-acetamido-2,6-dideoxy-D-galactose. Carbohydr Res 2003; **338**: 1801–1806.

101. de Kievit TR, Lam JS. Monoclonal antibodies that distinguish inner core, outer core, and lipid A regions of *Pseudomonas aeruginosa* lipopolysaccharide. J Bacteriol 1994; **176**: 7129–7139.

102. Poon KK, Westman EL, Vinogradov E, Jin S, Lam JS. Functional characterization of MigA and WapR: putative rhamnosyltransferases involved in outer core oligosaccharide biosynthesis of *Pseudomonas aeruginosa*. J Bacteriol 2008; **190**: 1857–1865.

103. Postma PW, Lengeler JW, Jacobson GR. Phosphoenolpyruvate:carbohydrate phosphotransferase systems of bacteria. Microbiol Rev 1993; **57**: 543–594.

104. Temple L, Cuskey SM, Perkins RE *et al.* Analysis of cloned structural and regulatory genes for carbohydrate utilization in *Pseudomonas aeruginosa* PAO. J Bacteriol 1990; **172**: 6396–6402.

105. Dean CR, Goldberg JB. *Pseudomonas aeruginosa galU* is required for a complete lipopolysaccharide core and repairs a secondary mutation in a PA103 (serogroup O11) *wbpM* mutant. FEMS Microbiol Lett 2002; **210**: 277–283.

106. Giraud MF, Naismith JH. The rhamnose pathway. Curr Opin Struct Biol 2000; **10**: 687–696.

107. Graninger M, Nidetzky B, Heinrichs DE, Whitfield C, Messner P. Characterization of dTDP-4-dehydrorhamnose 3,5-epimerase and dTDP-4-dehydrorhamnose reductase, required for dTDP-L-rhamnose biosynthesis in *Salmonella enterica* serovar Typhimurium LT2. J Biol Chem 1999; **274**: 25069–25077.

108. Blankenfeldt W, Asuncion M, Lam JS, Naismith JH. The structural basis of the catalytic mechanism and regulation of glucose-1-phosphate thymidylyltransferase (RmlA). EMBO J 2000; **19**: 6652–6663.

109. Blankenfeldt W, Giraud MF, Leonard G *et al.* The purification, crystallization and preliminary structural characterization of glucose-1-phosphate thymidylyltransferase (RmlA), the first enzyme of the dTDP-L-rhamnose synthesis pathway from *Pseudomonas aeruginosa*. Acta Crystallogr D Biol Crystallogr 2000; **56**: 1501–1504.

110. Charnock SJ, Davies GJ. Structure of the nucleotide-diphospho-sugar transferase, SpsA from *Bacillus subtilis*, in native and nucleotide-complexed forms. Biochemistry 1999; **38**: 6380–6385.

111. Melo A, Glaser L. The nucleotide specificity and feedback control of thymidine diphosphate D-glucose pyrophosphorylase. J Biol Chem 1965; **240**: 398–405.

112. Dong C, Major LL, Srikannathan V *et al.* RmlC, a C3′ and C5′ carbohydrate epimerase, appears to operate via an intermediate with an unusual twist boat conformation. J Mol Biol 2007; **365**: 146–159.

113. Rahim R, Burrows LL, Monteiro MA, Perry MB, Lam JS. Involvement of the *rml* locus in core oligosaccharide and O polysaccharide assembly in *Pseudomonas aeruginosa*. Microbiology 2000; **146** (Pt 11): 2803–2814.

114. Raetz CR. Biochemistry of endotoxins. Annu Rev Biochem 1990; **59**: 129–170.

115. Walsh AG, Burrows LL, Lam JS. Genetic and biochemical characterization of an operon involved in the biosynthesis of 3-deoxy-D-manno-octulosonic acid in *Pseudomonas aeruginosa*. FEMS Microbiol Lett 1999; **173**: 27–33.

116. Lebioda L, Stec B. Crystal structure of enolase indicates that enolase and pyruvate kinase evolved from a common ancestor. Nature 1988; **333**: 683–686.

308 King, Kocínková, Westman, Lam

117. Weng M, Makaroff CA, Zalkin H. Nucleotide sequence of *Escherichia coli pyrG* encoding CTP synthetase. *J Biol Chem* 1986; **261**: 5568–5574.

118. Ray PH, Benedict CD, Grasmuk H. Purification and characterization of cytidine 5′-triphosphate:cytidine 5′-monophosphate-3-deoxy-D-manno-octulosonate cytidylyltransferase. *J Bacteriol* 1981; **145**: 1273–1280.

119. Gronow S, Oertelt C, Ervela E *et al.* Characterization of the physiological substrate for lipopolysaccharide heptosyltransferases I and II. *J Endotoxin Res* 2001; **7**: 263–270.

120. Eidels L, Osborn MJ. Lipopolysaccharide and aldohexose biosynthesis in transketolase mutants of *Salmonella typhimurium*. *Proc Natl Acad Sci USA* 1971; **68**: 1673–1677.

121. Valvano MA, Marolda CL, Bittner M, Glaskin-Clay M, Simon TL, Klena JD. The *rfae* gene from *Escherichia coli* encodes a bifunctional protein involved in biosynthesis of the lipopolysaccharide core precursor ADP-L-glycerol-D-manno-heptose. *J Bacteriol* 2000; **182**: 488–497.

122. Valvano MA, Messner P, Kosma P. Novel pathways for biosynthesis of nucleotide-activated glycerol-manno-heptose precursors of bacterial glycoproteins and cell surface polysaccharides. *Microbiology* 2002; **148**: 1979–1989.

123. Cantarel BL, Coutinho PM, Rancurel C, Bernard T, Lombard V, Henrissat B. The Carbohydrate-Active EnZymes database (CAZy): an expert resource for glycogenomics. *Nucleic Acids Res* 2009; **37**: D233–D238.

124. Flint J, Taylor E, Yang M *et al.* Structural dissection and high-throughput screening of mannosylglycerate synthase. *Nat Struct Mol Biol* 2005; **12**: 608–614.

125. Belunis CJ, Raetz CR. Biosynthesis of endotoxins. Purification and catalytic properties of 3-deoxy-D-manno-octulosonic acid transferase from *Escherichia coli*. *J Biol Chem* 1992; **267**: 9988–9997.

126. Pradel E, Parker CT, Schnaitman CA. Structures of the *rfab*, *rfal*, *rfaj*, and *rfas* genes of *Escherichia coli* K-12 and their roles in assembly of the lipopolysaccharide core. *J Bacteriol* 1992; **174**: 4736–4745.

127. Hirokawa T, Boon-Chieng S, Mitaku S. SOSUI: classification and secondary structure prediction system for membrane proteins. *Bioinformatics* 1998; **14**: 378–379.

128. Krogh A, Larsson B, von Heijne G, Sonnhammer EL. Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. *J Mol Biol* 2001; **305**: 567–580.

129. Roantree RJ, Kuo TT, MacPhee DG. The effect of defined lipopolysaccharide core defects upon antibiotic resistances of *Salmonella typhimurium*. *J Gen Microbiol* 1977; **103**: 223–234.

130. Chatterjee AK, Sanderson KE, Ross H. Influence of temperature on growth of lipopolysaccharide-deficient (rough) mutants of *Salmonella typhimurium* and *Salmonella minnesota*. *Can J Microbiol* 1976; **22**: 1540–1548.

131. de Kievit TR, Lam JS. Isolation and characterization of two genes, *waac* (*rfac*) and *waaF* (*rfaf*), involved in *Pseudomonas aeruginosa* serotype O5 inner-core biosynthesis. *J Bacteriol* 1997; **179**: 3451–3457.

132. Yethon JA, Vinogradov E, Perry MB, Whitfield C. Mutation of the lipopolysaccharide core glycosyltransferase encoded by *waaG* destabilizes the outer membrane of *Escherichia coli* by interfering with core phosphorylation. *J Bacteriol* 2000; **182**: 5620–5623.

133. Knirel YA, Helbig JH, Zähringer U. Structure of a decasaccharide isolated by mild acid degradation and dephosphorylation of the lipopolysaccharide of *Pseudomonas fluorescens* strain ATCC 49271. *Carbohydr Res* 1996; **283**: 129–139.

134. Leone S, Izzo V, Silipo A *et al.* A novel type of highly negatively charged lipo oligosaccharide from *Pseudomonas stutzeri* OX1 possessing two 4,6-O-(1-carboxy)-ethylidene residues in the outer core region. *Eur J Biochem* 2004; **271**: 2691–2704.

135. Zdorovenko EL, Vinogradov E, Zdorovenko GM *et al.* Structure of the core oligosaccharide of a rough-type lipopolysaccharide of *Pseudomonas syringae pv. phaseolicola*. *Eur J Biochem* 2004; **271**: 4968–4977.

136. Frirdich E, Vinogradov E, Whitfield C. Biosynthesis of a novel 3-deoxy-D-manno-oct-2-ulosonic acid-containing outer core oligosaccharide in the lipopolysaccharide of *Klebsiella pneumoniae*. *J Biol Chem* 2004; **279**: 27928–27940.

137. Regué M, Izquierdo L, Fresno S *et al.* The incorporation of glucosamine into enterobacterial core lipopolysaccharide: two enzymatic steps are required. *J Biol Chem* 2005; **280**: 36648–36656.

138. Yang H, Matewish M, Loubens I, Storey DG, Lam JS, Jin S. *migA*, a quorum-responsive gene of *Pseudomonas aeruginosa*, is highly expressed in the cystic fibrosis lung environment and modifies low-molecular-mass lipopolysaccharide. *Microbiology* 2000; **146**: 2509–2519.

139. Wang J, Lory S, Ramphal R, Jin S. Isolation and characterization of *Pseudomonas aeruginosa* genes inducible by respiratory mucus derived from cystic fibrosis patients. *Mol Microbiol* 1996; **22**: 1005–1012.

140. Zhao X, Lam JS. WaaP of *Pseudomonas aeruginosa* is a novel eukaryotic type protein-tyrosine kinase as well as a sugar kinase essential for the biosynthesis of core lipopolysaccharide. *J Biol Chem* 2002; **277**: 4722–4730.

141. Zhao X, Wenzel CQ, Lam JS. Nonradiolabeling assay for WaaP, an essential sugar kinase involved in biosynthesis of core lipopolysaccharide of *Pseudomonas aeruginosa*. *Antimicrob Agents Chemother* 2002; **46**: 2035–2037.

142. To T. *Purification and characterization of WapP of Pseudomonas aeruginosa, a putative lipopolysaccharide kinase*. MSc thesis. University of Guelph, Ontario, Canada, 2006.

143. Coque JJ, Perez-Llarena FJ, Enguita FJ, Fuente JL, Martin JF, Liras P. Characterization of the *cmch* genes of *Nocardia lactamdurans* and *Streptomyces clavuligerus* encoding a functional 3-hydroxymethylcephem O-carbamoyltransferase for cephamycin biosynthesis. *Gene* 1995; **162**: 21–27.

144. Scheeff ED, Bourne PE. Structural evolution of the protein kinase-like superfamily. *PLoS Comput Biol* 2005; **1**: e49.

145. Tian Y, Jackson P, Gunter C, Wang J, Rock CO, Jackowski S. Placental thrombosis and spontaneous fetal death in mice deficient in ethanolamine kinase 2. *J Biol Chem* 2006; **281**: 28438–28449.

146. Brodsky IE, Ghori N, Falkow S, Monack D. Mig-14 is an inner membrane-associated protein that promotes *Salmonella typhimurium* resistance to CRAMP, survival within activated macrophages and persistent infection. *Mol Microbiol* 2005; **55**: 954–972.

147. Hasin M, Kennedy EP. Role of phosphatidylethanolamine in the biosynthesis of pyrophosphoethanolamine residues in the lipopolysaccharide of *Escherichia coli*. *J Biol Chem* 1982; **257**: 12475–12477.

148. Tamayo R, Choudhury B, Septer A, Merighi M, Carlson R, Gunn JS. Identification of *cptA*, a PmrA-regulated locus required for phosphoethanolamine modification of the *Salmonella enterica* serovar Typhimurium lipopolysaccharide core. *J Bacteriol* 2005; **187**: 3391–3399.

149. Cox AD, Li J, Richards JC. Identification and localization of glycine in the inner core lipopolysaccharide of *Neisseria meningitidis*. *Eur J Biochem* 2002; **269**: 4169–4175.

150. Lundström SL, Li J, Deadman ME, Hood DW, Moxon ER, Schweda EK. Structural analysis of the lipopolysaccharide from nontypeable *Haemophilus influenzae* strain R2846. *Biochemistry* 2008; **47**: 6025–6038.

151. Molinaro A, Silipo A, Castro CD *et al.* Full structural characterization of *Shigella flexneri* M90T serotype 5 wild-type R-LPS and its delta *galU* mutant: glycine residue location in the inner core of the lipopolysaccharide. *Glycobiology* 2008; **18**: 260–269.

152. Gamian A, Krzyzaniak A, Barczewska MZ, Gawronska I, Barczewski J. Specific incorporation of glycine into bacterial lipopolysaccharide. Novel function of specific transfer ribonucleic acids. *Nucleic Acids Res* 1991; **19**: 6021–6025.

153. Fox KL, Yildirim HH, Deadman ME, Schweda EK, Moxon ER, Hood DW. Novel lipopolysaccharide biosynthetic genes containing tetranucleotide repeats in *Haemophilus influenzae*, identification of a gene for adding O-acetyl groups. *Mol Microbiol* 2005; **58**: 207–216.

154. Doerrler WT, Gibbons HS, Raetz CR. MsbA-dependent translocation of lipids across the inner membrane of *Escherichia coli*. *J Biol Chem* 2004; **279**: 45102–45109.

155. Mamat U, Meredith TC, Aggarwal P *et al.* Single amino acid substitutions in either YhjD or MsbA confer viability to 3-deoxy-D-manno-oct-2-ulosonic acid-depleted *Escherichia coli*. *Mol Microbiol* 2008; **67**: 633–648.

156. Ghanei H, Abeyrathne PD, Lam JS. Biochemical characterization of MsbA from *Pseudomonas aeruginosa*. *J Biol Chem* 2007; **282**: 26939–26947.

157. Liu PV, Matsumoto H, Kusama H, Bergan TOM. Survey of heat-stable, major somatic antigens of *Pseudomonas aeruginosa*. *Int J Syst Bacteriol* 1983; **33**: 256–264.

158. Liu PV, Wang S. Three new major somatic antigens of *Pseudomonas aeruginosa*. *J Clin Microbiol* 1990; **28**: 922–925.

159. Akatova NS, Smirnova NE. [Serological classification of *Pseudomonas aeruginosa*]. *Zh Mikrobiol Epidemiol Immunobiol* 1982; 87–91.

160. Lànyi B, Bergan T. Serological characterization of *Pseudomonas aeruginosa*. *Methods Microbiol* 1978; **10**: 93–168.

161. Raymond CK, Sims EH, Kas A *et al.* Genetic variation at the O-antigen biosynthetic locus in *Pseudomonas aeruginosa*. *J Bacteriol* 2002; **184**: 3614–3622.

162. Rivera M, Bryan LE, Hancock RE, McGroarty EJ. Heterogeneity of lipopolysaccharides from *Pseudomonas aeruginosa*: analysis of lipopolysaccharide chain length. *J Bacteriol* 1988; **170**: 512–521.

163. de Kievit TR, Dasgupta T, Schweizer H, Lam JS. Molecular cloning and characterization of the *rfc* gene of *Pseudomonas aeruginosa* (serotype O5). *Mol Microbiol* 1995; **16**: 565–574.

164. Rocchetta HL, Lam JS. Identification and functional characterization of an ABC transport system involved in polysaccharide export of A-band lipopolysaccharide in *Pseudomonas aeruginosa*. *J Bacteriol* 1997; **179**: 4713–4724.

165. Whitfield C. Biosynthesis of lipopolysaccharide O antigens. *Trends Microbiol* 1995; **3**: 178–185.

166. Jiang XM, Neal B, Santiago F, Lee SJ, Romana LK, Reeves PR. Structure and sequence of the *rfb* (O antigen) gene cluster of *Salmonella* serovar Typhimurium (strain LT2). *Mol Microbiol* 1991; **5**: 695–713.

167. Alexander DC, Valvano MA. Role of the *rfe* gene in the biosynthesis of the *Escherichia coli* O7-specific lipopolysaccharide and other O-specific polysaccharides containing N-acetylglucosamine. *J Bacteriol* 1994; **176**: 7079–7084.

168. Zhang L, Radziejewska-Lebrecht J, Krajewska-Pietrasik D, Toivanen P, Skurnik M. Molecular and chemical characterization of the lipopolysaccharide O antigen and its role in the

virulence of *Yersinia enterocolitica* serotype O:8. *Mol Microbiol* 1997; **23**: 63–76.

169. Bélanger M, Burrows LL, Lam JS. Functional analysis of genes responsible for the synthesis of the B-band O antigen of *Pseudomonas aeruginosa* serotype O6 lipopolysaccharide. *Microbiology* 1999; **145**: 3505–3521.

170. Rocchetta HL, Burrows LL, Pacan JC, Lam JS. Three rhamnosyltransferases responsible for assembly of the A-band D-rhamnan polysaccharide in *Pseudomonas aeruginosa*: a fourth transferase, WbpL, is required for the initiation of both A-band and B-band lipopolysaccharide synthesis. *Mol Microbiol* 1998; **28**: 1103–1119.

171. Arsenault TL, Hughes DW, MacLean DB, Szarek WA, Kropinski AMB, Lam JS. Structural studies on the polysaccharide portion of ‘A-band’ lipopolysaccharide from a mutant (AK14O1) of *Pseudomonas aeruginosa* PAO1. *Can J Chem* 1991; **69**: 1273–1280.

172. Vinogradov E, Frirdich E, MacLean LL *et al.* Structures of lipopolysaccharides from *Klebsiella pneumoniae*. Elucidation of the structure of the linkage region between core and polysaccharide O chain and identification of the residues at the non-reducing termini of the O chains. *J Biol Chem* 2002; **277**: 25070–25081.

173. Jann K, Goldemann G, Weisgerber C, Wolf-Ullisch C, Kanegasaki S. Biosynthesis of the O9 antigen of *Escherichia coli*. Initial reaction and overall mechanism. *Eur J Biochem* 1982; **127**: 157–164.

174. Weisgerber C, Jann K. Glucosyldiphosphoundecaprenol, the mannose acceptor in the synthesis of the O9 antigen of *Escherichia coli*. Biosynthesis and characterization. *Eur J Biochem* 1982; **127**: 165–168.

175. Meier-Dieter U, Barr K, Starman R, Hatch L, Rick PD. Nucleotide sequence of the *Escherichia coli rfe* gene involved in the synthesis of enterobacterial common antigen. Molecular cloning of the *rfe-rff* gene cluster. *J Biol Chem* 1992; **267**: 746–753.

176. Rick PD, Hubbard GL, Barr K. Role of the *rfe* gene in the synthesis of the O8 antigen in *Escherichia coli* K-12. *J Bacteriol* 1994; **176**: 2877–2884.

177. Yokota S, Kaya S, Kawamura T, Araki Y, Ito E. The structure of the O-specific chain of lipopolysaccharide from *Pseudomonas aeruginosa* IID 1008 (ATCC 27584). *J Biochem* 1986; **99**: 1551–1561.

178. Rocchetta HL, Pacan JC, Lam JS. Synthesis of the A-band polysaccharide sugar D-rhamnose requires Rmd and WbpW: identification of multiple AlgA homologues, WbpW and ORF488, in *Pseudomonas aeruginosa*. *Mol Microbiol* 1998; **29**: 1419–1434.

179. Lightfoot J, Lam JS. Molecular cloning of genes involved with expression of A-band lipopolysaccharide, an antigenically conserved form, in *Pseudomonas aeruginosa*. *J Bacteriol* 1991; **173**: 5624–5630.

180. Shinabarger D, Berry A, May TB, Rothmel R, Fialho A, Chakrabarty AM. Purification and characterization of phosphomannose isomerase-guanosine diphospho-D-mannose pyrophosphorylase. A bifunctional enzyme in the alginate biosynthetic pathway of *Pseudomonas aeruginosa*. *J Biol Chem* 1991; **266**: 2080–2088.

181. Remminghorst U, Rehm BH. Bacterial alginate: from biosynthesis to applications. *Biotechnol Lett* 2006; **28**: 1701–1712.

182. Lee HJ, Chang HY, Venkatesan N, Peng HL. Identification of amino acid residues important for the phosphomannose isomerase activity of PslB in *Pseudomonas aeruginosa* PAO1. *FEBS Lett* 2008; **582**: 3479–3483.

310 King, Kocínková, Westman, Lam

183. Ma L, Lu H, Sprinkle A, Parsek MR, Wozniak DJ. *Pseudomonas aeruginosa* Psl is a galactose- and mannose-rich exopolysaccharide. *J Bacteriol* 2007; **189**: 8353–8356.

184. Zielinski NA, Chakrabarty AM, Berry A. Characterization and regulation of the *Pseudomonas aeruginosa algC* gene encoding phosphomannomutase. *J Biol Chem* 1991; **266**: 9754–9763.

185. Olvera C, Goldberg JB, Sanchez R, Soberon-Chavez G. The *Pseudomonas aeruginosa algC* gene product participates in rhamnolipid biosynthesis. *FEMS Microbiol Lett* 1999; **179**: 85–90.

186. Lightfoot J, Lam JS. Chromosomal mapping, expression and synthesis of lipopolysaccharide in *Pseudomonas aeruginosa*: a role for guanosine diphospho (GDP)-D-mannose. *Mol Microbiol* 1993; **8**: 771–782.

187. King JD, Poon KK, Webb NA *et al.* The structural basis for catalytic function of GMD and RMD, two closely related enzymes from the GDP-D-rhamnose biosynthesis pathway. *FEBS J* 2009; **276**: 2688–2700.

188. Webb NA, Mulichak AM, Lam JS, Rocchetta HL, Garavito RM. Crystal structure of a tetrameric GDP-D-mannose 4,6-dehydratase from a bacterial GDP-D-rhamnose biosynthetic pathway. *Protein Sci* 2004; **13**: 529–539.

189. Kido N, Torgov VI, Sugiyama T *et al.* Expression of the O9 polysaccharide of *Escherichia coli*: sequencing of the *E. coli* O9 *rfb* gene cluster, characterization of mannosyl transferases, and evidence for an ATP-binding cassette transport system. *J Bacteriol* 1995; **177**: 2178–2187.

190. Kido N, Morooka N, Paeng N *et al.* Production of monoclonal antibody discriminating serological difference in *Escherichia coli* O9 and O9a polysaccharides. *Microbiol Immunol* 1997; **41**: 519–525.

191. Kido N, Sugiyama T, Yokochi T, Kobayashi H, Okawa Y. Synthesis of *Escherichia coli* O9a polysaccharide requires the participation of two domains of WbdA, a mannosyltransferase encoded within the *wb** gene cluster. *Mol Microbiol* 1998; **27**: 1213–1221.

192. Cuthbertson L, Kimber MS, Whitfield C. Substrate binding by a bacterial ABC transporter involved in polysaccharide export. *Proc Natl Acad Sci USA* 2007; **104**: 19529–19534.

193. Cuthbertson L, Powers J, Whitfield C. The C-terminal domain of the nucleotide-binding domain protein Wzt determines substrate specificity in the ATP-binding cassette transporter for the lipopolysaccharide O-antigens in *Escherichia coli* serotypes O8 and O9a. *J Biol Chem* 2005; **280**: 30310–30319.

194. Clarke BR, Cuthbertson L, Whitfield C. Nonreducing terminal modifications determine the chain length of polymannose O antigens of *Escherichia coli* and couple chain termination to polymer export via an ATP-binding cassette transporter. *J Biol Chem* 2004; **279**: 35709–35718.

195. Schäffer C, Wugeditsch T, Kählig H, Scheberl A, Zayni S, Messner P. The surface layer (S-layer) glycoprotein of *Geobacillus stearothermophilus* NRS 2004/3a. Analysis of its glycosylation. *J Biol Chem* 2002; **277**: 6230–6239.

196. Steiner K, Novotny R, Werz DB *et al.* Molecular basis of S-layer glycoprotein glycan biosynthesis in *Geobacillus stearothermophilus*. *J Biol Chem* 2008; **283**: 21120–21133.

197. King JD. Characterisation of polysaccharide biosynthesis genes in *Bordetella bronchiseptica*. PhD thesis. University of Cambridge, UK, 2006.

198. Preston A, Petersen BO, Duus JO *et al.* Complete structures of *Bordetella bronchiseptica* and *Bordetella parapertussis* lipopolysaccharides. *J Biol Chem* 2006; **281**: 18135–18144.

199. Bera A, Herbert S, Jakob A, Vollmer W, Götz F. Why are pathogenic *Staphylococci* so lysozyme resistant? The peptidoglycan O-acetyltransferase OatA is the major determinant for lysozyme resistance of *Staphylococcus aureus*. *Mol Microbiol* 2005; **55**: 778–787.

200. Currie HL, Lightfoot J, Lam JS. Prevalence of *gca*, a gene involved in synthesis of A-band common antigen polysaccharide in *Pseudomonas aeruginosa*. *Clin Diagn Lab Immunol* 1995; **2**: 554–562.

201. Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL. GenBank. *Nucleic Acids Res* 2005; **33**: D34–D38.

202. Burrows LL, Charter DF, Lam JS. Molecular characterization of the *Pseudomonas aeruginosa* serotype O5 (PAO1) B-band lipopolysaccharide gene cluster. *Mol Microbiol* 1996; **22**: 481–495.

203. Dean CR, Franklund CV, Retief JD *et al.* Characterization of the serogroup O11 O-antigen locus of *Pseudomonas aeruginosa* PA103. *J Bacteriol* 1999; **181**: 4275–4284.

204. Dean CR, Goldberg JB. The *wbpM* gene in *Pseudomonas aeruginosa* serogroup O17 resides on a cryptic copy of the serogroup O11 O antigen gene locus. *FEMS Microbiol Lett* 2000; **187**: 59–63.

205. Burrows LL, Urbanic RV, Lam JS. Functional conservation of the polysaccharide biosynthetic protein WbpM and its homologues in *Pseudomonas aeruginosa* and other medically significant bacteria. *Infect Immun* 2000; **68**: 931–936.

206. DiGiandomenico A, Matewish MJ, Bisaillon A, Stehle JR, Lam JS, Castric P. Glycosylation of *Pseudomonas aeruginosa* 1244 pilin: glycan substrate specificity. *Mol Microbiol* 2002; **46**: 519–530.

207. Miller WL, Lam JS. Molecular biology of cell-surface polysaccharides in *Pseudomonas aeruginosa*: from gene to protein function. In: Cornelis P. (ed) *Pseudomonas: Genomics and Molecular Biology*. Horizon Scientific Press, 2007, 87–128.

208. Creuzenet C, Urbanic RV, Lam JS. Structure-function studies of two novel UDP-GlcNAc C6 dehydratases/C4 reductases. Variation from the SYK dogma. *J Biol Chem* 2002; **277**: 26769–26778.

209. Creuzenet C, Lam JS. Topological and functional characterization of WbpM, an inner membrane UDP-GlcNAc C6 dehydratase essential for lipopolysaccharide biosynthesis in *Pseudomonas aeruginosa*. *Mol Microbiol* 2001; **41**: 1295–1310.

210. Schoenhofen IC, McNally DJ, Vinogradov E *et al.* Functional characterization of dehydratase/aminotransferase pairs from *Helicobacter* and *Campylobacter*: enzymes distinguishing the pseudaminic acid and bacillosamine biosynthetic pathways. *J Biol Chem* 2006; **281**: 723–732.

211. Schoenhofen IC, McNally DJ, Brisson JR, Logan SM. Elucidation of the CMP-pseudaminic acid pathway in *Helicobacter pylori*: synthesis from UDP-N-acetylglucosamine by a single enzymatic reaction. *Glycobiology* 2006; **16**: 8C–14C.

212. McNally DJ, Schoenhofen IC, Mulrooney EF *et al.* Identification of labile UDP-ketosugars in *Helicobacter pylori*, *Campylobacter jejuni* and *Pseudomonas aeruginosa*: key metabolites used to make glycan virulence factors. *Chembiochem* 2006; **7**: 1865–1868.

213. Ishiyama N, Creuzenet C, Miller WL *et al.* Structural studies of FlaA1 from *Helicobacter pylori* reveal the mechanism for inverting 4,6-dehydratase activity. *J Biol Chem* 2006; **281**: 24489–24495.

214. Morrison JP, Schoenhofen IC, Tanner ME. Mechanistic studies on PseB of pseudaminic acid biosynthesis: a UDP-N-acetylglicosamine 5-inverting 4,6-dehydratase. *Bioorg Chem* 2008; **36**: 312–320.

215. Holloway BW, Cooper GN. Lysogenic conversion in *Pseudomonas aeruginosa*. *J Bacteriol* 1962; **84**: 1321–1324.

216. Stover CK, Pham XQ, Erwin AL *et al.* Complete genome sequence of *Pseudomonas aeruginosa* PAO1, an opportunistic pathogen. *Nature* 2000; **406**: 959–964.

217. Knirel YA, Kochetkov NK. The structure of lipopolysaccharides of Gram-negative bacteria. III. The structure of O-antigens: a review. *Biochemistry* 1994; **59**: 1325–1382.

218. Burrows LL, Pigeon KE, Lam JS. *Pseudomonas aeruginosa* B-band lipopolysaccharide genes *wbpA* and *wbpI* and their *Escherichia coli* homologues *wecC* and *wecB* are not functionally interchangeable. *FEMS Microbiol Lett* 2000; **189**: 135–141.

219. Wenzel CQ, Daniels C, Keates RA, Brewer D, Lam JS. Evidence that WbpD is an N-acetyltransferase belonging to the hexapeptide acyltransferase superfamily and an important protein for O-antigen biosynthesis in *Pseudomonas aeruginosa* PAO1. *Mol Microbiol* 2005; **57**: 1288–1303.

220. Westman EL, Preston A, Field RA, Lam JS. Biosynthesis of a rare di-N-acetylated sugar in the lipopolysaccharides of both *Pseudomonas aeruginosa* and *Bordetella pertussis* occurs via an identical scheme despite different gene clusters. *J Bacteriol* 2008; **190**: 6060–6069.

221. Miller WL, Wenzel CQ, Daniels C, Larocque S, Brisson JR, Lam JS. Biochemical characterization of WbpA, a UDP-N-acetyl-D-glucosamine 6-dehydrogenase involved in O-antigen biosynthesis in *Pseudomonas aeruginosa* PAO1. *J Biol Chem* 2004; **279**: 37551–37558.

222. Sweet CR, Ribeiro AA, Raetz CR. Oxidation and transamination of the 3′-position of UDP-N-acetylglucosamine by enzymes from *Acidithiobacillus ferrooxidans*. Role in the formation of lipid A molecules with four amide-linked acyl chains. *J Biol Chem* 2004; **279**: 25400–25410.

223. Westman EL, McNally DJ, Charchoglyan A, Brewer D, Field RA, Lam JS. Characterization of WbpB, WbpE, and WbpD, and reconstitution of a pathway for the biosynthesis of UDP-2,3-diacetamido-2,3-dideoxy-D-mannuronic acid in *Pseudomonas aeruginosa*. *J Biol Chem* 2009; **284**: 11854–11862.

224. Larkin A, Imperiali B. Biosynthesis of UDP-GlcNAc(3NAc)A by WbpB, WbpE, and WbpD: Enzymes in the Wbp Pathway Responsible for O-antigen Assembly in *Pseudomonas aeruginosa* PAO1. *Biochemistry* 2009: In press.

225. Westman EL, McNally DJ, Rejzek M *et al.* Identification and biochemical characterization of two novel UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-glucuronic acid 2-epimerases from respiratory pathogens. *Biochem J* 2007; **405**: 123–130.

226. King JD, Mulrooney EF, Vinogradov E, Kneidinger B, Mead K, Lam JS. *lfnA* from *Pseudomonas aeruginosa* O12 and *wbuX* from *Escherichia coli* O145 encode membrane-associated proteins and are required for expression of 2,6-dideoxy-2-acetamidino-L-galactose in lipopolysaccharide O antigen. *J Bacteriol* 2008; **190**: 1671–1679.

227. McNally DJ, Hui JP, Aubry AJ *et al.* Functional characterization of the flagellar glycosylation locus in *Campylobacter jejuni* 81–176 using a focused metabolomics approach. *J Biol Chem* 2006; **281**: 18489–18498.

228. Forsberg LS, Noel KD, Box J, Carlson RW. Genetic locus and structural characterization of the biochemical defect in the O-antigenic polysaccharide of the symbiotically deficient *Rhizobium etli* mutant, CE166. Replacement of N-acetylquinosamine with its hexosyl-4-ulose precursor. *J Biol Chem* 2003; **278**: 51347–51359.

229. Zhao X, Creuzenet C, Belanger M, Egbosimba E, Li J, Lam JS. WbpO, a UDP-N-acetyl-D-galactosamine dehydrogenase from *Pseudomonas aeruginosa* serotype O6. *J Biol Chem* 2000; **275**: 33252–33259.

230. Miller WL, Matewish MJ, McNally DJ *et al.* Flagellin glycosylation in *Pseudomonas aeruginosa* PAK requires the O-antigen biosynthesis enzyme WbpO. *J Biol Chem* 2008; **283**: 3507–3518.

231. Ishiyama N, Creuzenet C, Lam JS, Berghuis AM. Crystal structure of WbpP, a genuine UDP-N-acetylglucosamine 4-epimerase from *Pseudomonas aeruginosa*: substrate specificity in UDP-hexose 4-epimerases. *J Biol Chem* 2004; **279**: 22635–22642.

232. Creuzenet C, Bélanger M, Wakarchuk WW, Lam JS. Expression, purification, and biochemical characterization of WbpP, a new UDP-GlcNAc C4 epimerase from *Pseudomonas aeruginosa* serotype O6. *J Biol Chem* 2000; **275**: 19060–19067.

233. Demendi M, Ishiyama N, Lam JS, Berghuis AM, Creuzenet C. Towards a better understanding of the substrate specificity of the UDP-N-acetylglucosamine C4 epimerase WbpP. *Biochem J* 2005; **389**: 173–180.

234. Feng L, Tao J, Guo H *et al.* Structure of the *Shigella dysenteriae* 7 O antigen gene cluster and identification of its antigen specific genes. *Microb Pathog* 2004; **36**: 109–115.

235. Vinogradov EV, Shashkov AS, Knirel YA *et al.* Structure of the O-antigen of *Francisella tularensis* strain 15. *Carbohydr Res* 1991; **214**: 289–297.

236. Prior JL, Prior RG, Hitchen PG *et al.* Characterization of the O antigen gene cluster and structural analysis of the O antigen of *Francisella tularensis* subsp. *tularensis*. *J Med Microbiol* 2003; **52**: 845–851.

237. Parolis H, Parolis LA, Olivieri G. Structural studies on the *Shigella*-like *Escherichia coli* O121 O-specific polysaccharide. *Carbohydr Res* 1997; **303**: 319–325.

238. Larsen TM, Boehlein SK, Schuster SM *et al.* Three-dimensional structure of *Escherichia coli* asparagine synthetase B: a short journey from substrate to product. *Biochemistry* 1999; **38**: 16146–16157.

239. Kaya S, Araki Y, Ito E. The structure of the O-specific chain of lipopolysaccharide from *Pseudomonas aeruginosa* IID 1012 (ATCC 27588). *J Biochem* 1989; **105**: 29–34.

240. Schuman B, Alfaro JA, Evans SV. Glycosyltransferase structure and function. In: Peters T. (ed.). *Bioactive Conformation I*. Berlin: Springer, 2007; 217–257.

241. Knirel YA, Kochetkov NK. [Structure of lipopolysaccharides from Gram-negative bacteria. III. Structure of O-specific polysaccharides]. *Biokhimiia* 1994; **59**: 1784–1851.

242. Moreau M, Richards JC, Fournier JM, Byrd RA, Karakawa WW, Vann WF. Structure of the type 5 capsular polysaccharide of *Staphylococcus aureus*. *Carbohydr Res* 1990; **201**: 285–297.

243. Fournier JM, Vann WF, Karakawa WW. Purification and characterization of *Staphylococcus aureus* type 8 capsular polysaccharide. *Infect Immun* 1984; **45**: 87–93.

244. Jones C, Currie F, Forster MJ. N.M.R. and conformational analysis of the capsular polysaccharide from *Streptococcus pneumoniae* type 4. *Carbohydr Res* 1991; **221**: 95–121.

245. Schmidt MA, Jann B, Jann K. Cell-wall lipopolysaccharide of the urinary-tract-infective *Escherichia coli* O4:K12:H−. Structure of the polysaccharide chain. *Eur J Biochem* 1983; **137**: 163–171.

246. Manca MC, Weintraub A, Widmalm G. Structural studies of the *Escherichia coli* O26 O-antigen polysaccharide. *Carbohydr Res* 1996; **281**: 155–160.

247. Landersjö C, Weintraub A, Widmalm G. Structural analysis of the O-antigen polysaccharide from the Shiga toxin-producing *Escherichia coli* O172. *Eur J Biochem* 2001; **268**: 2239–2245.

248. Kneidinger B, O’Riordan K, Li J, Brisson JR, Lee JC, Lam JS. Three highly conserved proteins catalyze the conversion of UDP-N-acetyl-D-glucosamine to precursors for the biosynthesis of O antigen in *Pseudomonas aeruginosa* O11 and capsule in *Staphylococcus aureus* type 5. Implications for the UDP-N-acetyl-L-fucosamine biosynthetic pathway. *J Biol Chem* 2003; **278**: 3615–3627.

312 King, Kocínková, Westman, Lam

249. Mulrooney EF, Poon KK, McNally DJ, Brisson JR, Lam JS. Biosynthesis of UDP-N-acetyl-L-fucosamine, a precursor to the biosynthesis of lipopolysaccharide in *Pseudomonas aeruginosa* serotype O11. *J Biol Chem* 2005; **280**: 19535–19542.

250. Guerry P, Doig P, Alm RA, Burr DH, Kinsella N, Trust TJ. Identification and characterization of genes required for post-translational modification of *Campylobacter coli* VC167 flagelin. *Mol Microbiol* 1996; **19**: 369–378.

251. Glaze PA, Watson DC, Young NM, Tanner ME. Biosynthesis of CMP-N,N′-diacetylllegionaminic acid from UDP-N,N′-diacetyl-bacillosamine in *Legionella pneumophila*. *Biochemistry* 2008; **47**: 3272–3282.

252. Vann WF, Daines DA, Murkin AS *et al.* The NeuC protein of *Escherichia coli* K1 is a UDP N-acetylglucosamine 2-epimerase. *J Bacteriol* 2004; **186**: 706–712.

253. Marchler-Bauer A, Anderson JB, Derbyshire MK *et al.* CDD: a conserved domain database for interactive domain family analysis. *Nucleic Acids Res* 2007; **35**: D237–D240.

254. Franklin MJ, Chitnis CE, Gacesa P, Sonesson A, White DC, Ohman DE. *Pseudomonas aeruginosa* AlgG is a polymer level alginate C5-mannuronan epimerase. *J Bacteriol* 1994; **176**: 1821–1830.

255. King JD, Harmer NJ, Preston A *et al.* Predicting protein function from structure - the roles of short-chain dehydrogenase/reductase enzymes in *Bordetella* O-antigen biosynthesis. *J Mol Biol* 2007; **374**: 749–763.

256. Faure K, Shimabukuro D, Ajayi T, Allmond LR, Sawa T, Wiener-Kronish JP. O-antigen serotypes and type III secretory toxins in clinical isolates of *Pseudomonas aeruginosa*. *J Clin Microbiol* 2003; **41**: 2158–2160.

257. Persson B, Kallberg Y, Oppermann U, Jornvall H. Coenzyme-based functional assignments of short-chain dehydrogenases/reductases (SDRs). *Chem Biol Interact* 2003; **143/144**: 271–278.

258. Libisch B, Balogh B, Fuzi M. Identification of two multidrug-resistant *Pseudomonas aeruginosa* clonal lineages with a countrywide distribution in Hungary. *Curr Microbiol* 2009; **58**: 111–116.

259. Kaluzny K, Abeyrathne PD, Lam JS. Coexistence of two distinct versions of O-antigen polymerase, Wzy-alpha and Wzy-beta, in *Pseudomonas aeruginosa* serogroup O2 and their contributions to cell surface diversity. *J Bacteriol* 2007; **189**: 4141–4152.

260. Liu D, Cole RA, Reeves PR. An O-antigen processing function for Wzx (RfbX): a promising candidate for O-unit flippase. *J Bacteriol* 1996; **178**: 2102–2107.

261. Burrows LL, Lam JS. Effect of *wzx* (*rfbX*) mutations on A-band and B-band lipopolysaccharide biosynthesis in *Pseudomonas aeruginosa* O5. *J Bacteriol* 1999; **181**: 973–980.

262. Marolda CL, Vicarioli J, Valvano MA. Wzx proteins involved in biosynthesis of O antigen function in association with the first sugar of the O-specific lipopolysaccharide subunit. *Microbiology* 2004; **150**: 4095–4105.

263. Feldman MF, Marolda CL, Monteiro MA, Perry MB, Parodi AJ, Valvano MA. The activity of a putative polyisoprenol-linked sugar translocase (Wzx) involved in *Escherichia coli* O antigen assembly is independent of the chemical structure of the O repeat. *J Biol Chem* 1999; **274**: 35129–35138.

264. Daniels C, Vindurampulle C, Morona R. Overexpression and topology of the *Shigella flexneri* O-antigen polymerase (Rfc/Wzy). *Mol Microbiol* 1998; **28**: 1211–1222.

265. Jiang SM, Wang L, Reeves PR. Molecular characterization of *Streptococcus pneumoniae* type 4, 6B, 8, and 18C capsular polysaccharide gene clusters. *Infect Immun* 2001; **69**: 1244–1255.

266. Newton GJ, Daniels C, Burrows LL, Kropinski AM, Clarke AJ, Lam JS. Three-component-mediated serotype conversion in *Pseudomonas aeruginosa* by bacteriophage D3. *Mol Microbiol* 2001; **39**: 1237–1247.

267. Burrows LL, Chow D, Lam JS. *Pseudomonas aeruginosa* B-band O-antigen chain length is modulated by Wzz (Rol). *J Bacteriol* 1997; **179**: 1482–1489.

268. Daniels C, Griffiths C, Cowles B, Lam JS. *Pseudomonas aeruginosa* O-antigen chain length is determined before ligation to lipid A core. *Environ Microbiol* 2002; **4**: 883–897.

269. Kintz E, Scarff JM, DiGiandomenico A, Goldberg JB. Lipopolysaccharide O-antigen chain length regulation in *Pseudomonas aeruginosa* serogroup O11 strain PA103. *J Bacteriol* 2008; **190**: 2709–2716.

270. Abeyrathne PD, Lam JS. WaaL of *Pseudomonas aeruginosa* utilizes ATP in *in vitro* ligation of O antigen onto lipid A-core. *Mol Microbiol* 2007; **65**: 1345–1359.

271. Pérez JM, McGarry MA, Marolda CL, Valvano MA. Functional analysis of the large periplasmic loop of the *Escherichia coli* K-12 WaaL O-antigen ligase. *Mol Microbiol* 2008; **70**: 1424–1440.

272. Heinrichs DE, Monteiro MA, Perry MB, Whitfield C. The assembly system for the lipopolysaccharide R2 core-type of *Escherichia coli* is a hybrid of those found in *Escherichia coli* K-12 and *Salmonella enterica*. Structure and function of the R2 WaaK and WaaL homologs. *J Biol Chem* 1998; **273**: 8849–8859.
