
LETTERS

Brassinosteroid signal transduction from cell-surface receptor kinases to nuclear transcription factors

Tae-Wuk Kim¹, Shenheng Guan², Yu Sun¹, Zhiping Deng¹, Wenqiang Tang¹, Jian-Xiu Shang³, Ying Sun³, Alma L. Burlingame² and Zhi-Yong Wang¹,⁴

Brassinosteroid (BR) regulates gene expression and plant development through a receptor kinase-mediated signal transduction pathway¹. Despite the identification of many components of this pathway, it remains unclear how the BR signal is transduced from the cell surface to the nucleus². Here we describe a complete BR signalling pathway by elucidating key missing steps. We show that phosphorylation of BSK1 (BR-signalling kinase 1) by the BR receptor kinase BRI1 (BR-insensitive 1) promotes BSK1 binding to the BSU1 (BRI1 suppressor 1) phosphatase, and BSU1 inactivates the GSK3-like kinase BIN2 (BR-insensitive 2) by dephosphorylating a conserved phospho-tyrosine residue (pTyr 200). Mutations that affect phosphorylation/dephosphorylation of BIN2 pTyr200 (*bin2-1*, *bin2-Y200F* and quadruple loss-of-function of BSU1-related phosphatases) support an essential role for BSU1-mediated BIN2 dephosphorylation in BR-dependent plant growth. These results demonstrate direct sequential BR activation of BRI1, BSK1 and BSU1, and inactivation of BIN2, leading to accumulation of unphosphorylated BZR (brassinazole resistant) transcription factors in the nucleus. This study establishes a fully connected BR signalling pathway and provides new insights into the mechanism of GSK3 regulation.

Steroid hormones are critical for development of all multicellular organisms and in plants BRs have essential roles in a wide range of developmental and physiological processes³. Unlike animal steroid hormones, which function through nuclear receptors, BRs bind to a receptor kinase (BRI1) at the cell surface to activate the BR response transcription factors BZR1 and BZR2 (also known as BES1) through a signal transduction pathway¹⁴. Although many components have been identified and studied in detail, our understanding of the BR signalling pathway remains incomplete, with major gaps between the receptor kinases at the cell surface and downstream components in the cytoplasm and nucleus (Supplementary Information, Fig. S1a)¹,².

The upstream BR-signalling components at the plasma membrane include BRI1 (refs 5, 6) and BAK1 (BR-insensitive 1-associated receptor kinase 1; refs 7, 8) receptor kinases, a new protein, BKI1, that inhibits BRI1 (ref. 9) and the plasma membrane-associated BR-signalling kinases (BSKs)¹⁰. BR binding to the extracellular domain of BRI1 causes dissociation of BKI1 from BRI1 (ref. 9, 11) and induces association and trans-phosphorylation between BRI1 and its co-receptor BAK1 (ref. 12), leading to activation of BRI1 kinase and phosphorylation of its BSK substrates¹⁰. Proteomic and genetic studies have demonstrated an essential role for BSKs in transducing the signal to the downstream components, but their direct target remains unknown¹⁰.

Downstream BR signalling involves the GSK3-like kinase BIN2 (ref. 13), the Kelch-repeats-containing phosphatase BSU1 (ref. 14), the 14-3-3 family of phosphopeptide-binding proteins¹⁵, and BZR1 and BZR2, which directly bind to DNA and regulate BR-responsive gene expression¹⁶–¹⁹. As a negative regulator of BR signalling, BIN2 phosphorylates BZR1 and BZR2 at numerous sites to inhibit their activities through several mechanisms². These include accelerating proteasome-mediated degradation²⁰, promoting nuclear export and cytoplasmic retention by the 14-3-3 proteins¹⁵,²¹, and inhibiting DNA binding and transcriptional activity²,¹⁵,²². By contrast, the BSU1 phosphatase is a positive regulator of BR signalling¹⁴. Overexpression of BSU1 increases dephosphorylated BZR2 and activates BR responses¹⁴. However, BSU1 does not interact with or effectively dephosphorylate BZR2 *in vitro* and the biochemical function of BSU1 remains unknown¹⁴. It is believed that BR induces rapid dephosphorylation of BZR1 and BZR2 by inhibiting BIN2 and/or activating BSU1. However, the mechanisms by which upstream BR signalling regulates BIN2 and BSU1 remain unclear (Supplementary Information, Fig. S1a)²,²³.

To understand how BR signalling regulates BIN2, we analysed BR-induced changes in BIN2 phosphorylation using immunoblotting of two-dimensional gel electrophoresis and found that treatment with brassinolide (the most active form of BR) caused disappearance of the acidic forms and an increase in the basic forms of epitope-tagged BIN2 (Fig. 1a), suggesting that BR induced dephosphorylation of BIN2. This result led us to investigate the role of BSU1 phosphatase in BR regulation of BIN2. Both BSU1 and its closest homologue BSL1 promote BR signalling *in vivo* (Supplementary Information, Fig. S2a)¹⁴ and show manganese-dependent

---

¹ Department of Plant Biology, Carnegie Institution for Science, Stanford, CA 94305, USA.  
² Department of Pharmaceutical Chemistry, University of California, San Francisco, CA 94143, USA.  
³ Institute of Molecular Cell Biology, Hebei Normal University, Shijiazhuang, Hebei, 050016, China.  
⁴ Correspondence should be addressed to Z.-Y.W. (e-mail: zywang24@stanford.edu).

Received 24 April 2009; accepted 4 June 2009; published online 6 September 2009; DOI: 10.1038/ncb1970

1254 | NATURE CELL BIOLOGY VOLUME 11 | NUMBER 10 | OCTOBER 2009

© 2009 Macmillan Publishers Limited. All rights reserved.

LETTERS

Figure 1 BSU1 directly inhibits BIN2 phosphorylation of BZR1. (a) BR induces dephosphorylation of BIN2. Total proteins of TAP–BIN2 transgenic plants treated with brassinolide (+ BL; 0.25 μM) or mock solution (− BL) for 2 h were analysed by two-dimensional gel electrophoresis before immunoblotting using an anti-peroxidase antibody that detects TAP–BIN2. (b) BSU1 does not dephosphorylate pBZR1 in vitro. BIN2-phosphorylated MBP–BZR1 (BIN2 removed) was incubated with GST, GST–BSU1 or GST–BSL1 for 12 h and analysed by immunoblotting using an anti-MBP antibody. (c–e) BSU1 inhibits BIN2 but not bin2-1. (c) GST–BIN2 was pre-incubated with GST–BSU1 or GST for the indicated times before MBP–BZR1 and [γ-32P]ATP were added. (d) Partially phosphorylated 32P-MBP–pBZR1 was further incubated with GST–BIN2, GST–BSU1 or both, in the presence of non-radioactive ATP, and analysed by autoradiography. GST–BIN2M115A is a BIN2 kinase-inactive mutant. (e) GST–BIN2 or GST–bin2-1 was first treated with BSU1–YFP immunoprecipitated from BR-treated (+ BL; 0.25 μM for 30 min) or untreated 35S::BSU1-YFP plants, and then incubated with MBP–BZR1 and [γ-32P]ATP. Samples were run on the same gel. Col-0, immunoprecipitation from non-transgenic plant as a control. (f–i) BSU1

directly interacts with BIN2 and bin2-1. (f) A gel blot containing GST, GST–BIN2 and GST–bin2-1 was probed sequentially with MBP–BSU1 and an anti-MBP antibody (top) and then stained with Ponceau S (bottom). (g) BiFC assay of interactions between BSU1 or BSL1, and BIN2. The indicated constructs were transformed into tobacco leaf cells. Bright spots in BIN2–nYFP + cYFP are chloroplast autofluorescence. Scale bars, 10 μm. (h) The proteins of tobacco leaves transiently transformed with the indicated constructs were immunoprecipitated with an anti-GFP antibody, and the immunoblot was probed with anti-Myc and anti-GFP antibodies. Samples were run on two different gels, but were part of the same experiment. (i) *Arabidopsis* plants (F1) expressing BSU1–YFP or co-expressing BSU1–YFP and BIN2–Myc, grown on medium containing BR biosynthetic inhibitor BRZ for 10 days, were treated with MG-132 (10 μM) for 1 h and then with brassinolide (BL; 0.2 μM) or mock solution for 15 min. Total protein extracts were immunoprecipitated with anti-Myc antibodies, and the immunoblot was probed with anti-GFP and anti-Myc antibodies. Full scans of immunoblots and in vitro kinase/phosphatase assays are shown in Supplementary Information, Fig. S12. CBB, Coomassie brilliant blue-stained gels. IP, immunoprecipitate.

phosphatase activity (Supplementary Information, Fig. S2b, c). BSU1 only partially reduced the phosphorylation of BZR1 or BZR2 when co-incubated with BIN2 and BZR1 or BZR2 (Supplementary Information, Fig. S3a, b)14, and failed to dephosphorylate BZR1 and BZR2 when added after BIN2 and ATP had been removed from the kinase reaction (Fig. 1b; Supplementary Information, Fig. S3c, d). In contrast, BSU1 most effectively reduced BZR1 phosphorylation when pre-incubated with BIN2 before adding BZR1 (Fig. 1c). Partially phosphorylated BZR1, by BIN2 using radioactive [γ-32P] ATP, was further phosphorylated by BIN2 using non-radioactive ATP, causing a mobility shift of the pre-labelled BZR1. Addition of BSU1 did not reduce the radioactivity of 32P-labelled BZR1, but abolished the mobility shift of the BZR1 band caused by BIN2 (Fig. 1d). These results indicate that BSU1 inhibits BIN2 kinase activity, but does not dephosphorylate pre-phosphorylated BZR1 in vitro. The phosphatase domain of BSU1 reduced BIN2 phosphorylation of BZR1 whereas the Kelch repeat domain showed no effect (Supplementary Information, Fig. S3e)

We next examined whether BR and the bin2-1 mutation affect BSU1 inhibition of BIN2. A BSU1–YFP (yellow fluorescent protein) fusion protein was immunoprecipitated from transgenic *Arabidopsis thaliana*. Similarly to recombinant GST (glutathione S-transferase)–BSU1,

LETTERS

a
- BZR1-YFP
- BZR1-YFP + BIN2-Myc
- BZR1-YFP + BIN2-Myc + BSU1-Myc
- BZR1-YFP + BIN2-Myc + bin2-1-Myc
- BZR1-YFP + bin2-1-Myc + BSU1-Myc

b
|  | BIN2-Myc | bin2-1-Myc |
| --- | --- | --- |
| pBZR1- |  |  |
| BZR1- |  |  |
| BSU1- | - | - | + | - | - | + | - |
| BSU1<sup>D510N</sup>- | - | - | + | - | - | - | + |

c
|  | BL | - | - | + | + |
| --- | --- | --- | --- | --- | --- |
| BSU1-YFP | - | + | - | + |
| BIN2-Myc |  |  |  |  | -55 |
| BSU1-YFP |  |  |  |  | -130 |
| Nonspecific |  |  |  |  | -43 |

d
|  | BIN2-Myc | bin2-1-Myc |
| --- | --- | --- |
| BSU1-Myc | - | + | - | - | + | - |
| BSU1<sup>D510N</sup>-Myc | - | - | + | - | - | + |
| BIN2/bin2-1-Myc |  |  |  |  | -55 |
| BSU1/BSU1<sup>D510N</sup>-Myc |  |  |  |  | -130 |
| Nonspecific |  |  |  |  | -34 |

e
- -BSU1
- +BSU1

f

g
![Graph](attachment://graph_image.png)

Figure 2 BSU1 regulates BIN2 but not bin2-1 in vivo. (a) Subcellular localization of BZR1–YFP in the cells co-transformed with the indicated constructs. Scale bars, 10 μm. (b) Immunoblots of BZR1–YFP proteins obtained from the tobacco leaves co-transformed with constructs indicated. The upper band is phosphorylated BZR1 and the lower one, unphosphorylated BZR1. (c) Overexpression of BSU1–YFP reduces the accumulation of BIN2–Myc in a transgenic Arabidopsis line. Heterozygous 35S::BIN2-myc and 35S::BIN2-myc/35S::BSU1-YFP plants (F<sub>1</sub>) were treated with brassinolide (BL; 0.25 μM) or mock solution for 30 min. The immunoblot was probed with anti-Myc or anti-GFP antibodies, with a nonspecific band serving as a loading control. (d) BSU1 reduces the accumulation of BIN2 but not that of bin2-1. The BIN2–Myc or bin2-1–Myc level was analysed by an anti-Myc antibody in tobacco cells co-expressing Myc-tagged BSU1 or the BSU1<sup>D510N</sup> mutant. A nonspecific band served as a loading control. (e) Overexpression of BSU1–YFP (+ BSU1) partially rescues the bri1-116 mutant, but not the bin2-1 mutant. (f) Hypocotyl phenotypes of seedlings (genotype shown) grown in the dark on MS medium for 5 days. Bottom two panels show confocal images of BSU1–YFP in the plants indicated. Scale bars, 10 μm. (g) Quantitative real time PCR analysis of SAUR-AC1 RNA expression in wild-type (bri1-116 (+)), bri1-116 (-/-) and BSU1–YFP/bri1-116 plants. Error bars indicate mean ± s.e.m., n = 3. Full scans of immunoblots are shown in Supplementary Information, Fig. S12.

BSU1–YFP from plants did not dephosphorylate the pre-phosphorylated BZR1 (Supplementary Information, Fig. S4a), but reduced BZR1 phosphorylation when co-incubated with BIN2 and BZR1 (Supplementary Information, Fig. S4b) or pre-incubated with BIN2 before adding to BZR1 (Fig. 1e). Moreover, BSU1–YFP from plants treated with brassinolide more effectively inhibited BIN2 phosphorylation of BZR1 than that from untreated plants (Fig. 1e; Supplementary Information, Fig. S4b), suggesting that BR activates BSU1. The gain-of-function bin2-1 mutation that causes BR-insensitive phenotypes<sup>23</sup> blocked regulation by BSU1–YFP in vitro (Fig. 1e).

Several experiments demonstrated that BSU1 directly interacts with BIN2. First, GST-BIN2 was detected on a gel blot by MBP (maltose-binding protein) – BSU1 and an anti-MBP antibody (Fig. 1f). Second, in vivo interaction was demonstrated by bimolecular fluorescence complementation (BiFC) assays<sup>24</sup> in which tobacco cells cotransformed with BIN2 fused to the amino-terminal half (nYFP) and BSU1 fused to carboxy-terminal half (cYFP) of YFP showed strong fluorescence signals (Fig. 1g). Furthermore, epitope-tagged BIN2 and BSU1 were co-immunoprecipitated and the amount of co-immunoprecipitation was increased by BR treatment (Fig. 1h, i; Supplementary Information, Fig. S5a), indicating that upstream BR signalling induces BSU1 binding to BIN2. Similarly, BSL1 also interacted with BIN2 in BiFC and co-immunoprecipitation assays (Fig. 1g, h). The bin2-1 mutant protein also interacted with BSU1 and BSL1 in these assays (Fig. 1f; Supplementary Information, Fig. S5a, b), suggesting that the bin2-1 mutation blocks BSU1 regulation of BIN2 without abolishing their physical interaction.

A BSU1–GFP (green fluorescent protein) fusion protein was previously observed only in the nucleus<sup>14</sup>. In this study, the BSU1–YFP protein was detected predominantly in the nucleus and weakly in the cytoplasm. Interestingly, BSL1–YFP was excluded from the nucleus and localized exclusively in the cytoplasm and plasma membrane (Fig. 1g; Supplementary Information, Fig. S5c). BSL1 and its two other homologues have all been identified as plasma membrane proteins by recent proteomics studies<sup>25</sup>, suggesting that members of the BSU family can mediate upstream BR signalling at the plasma membrane as well as function in the cytoplasm and nucleus.

We further examined whether BSU1 inhibits BIN2 activity in vivo. We reported previously that BIN2 phosphorylation of BZR1 promotes BZR1 cytoplasmic retention by the 14-3-3 proteins whereas unphosphorylated

Figure 3 BSU1 dephosphorylates pTyr 200 of BIN2 but not bin2-1.

(a) Tyr 200 phosphorylation is required for BIN2 kinase activity. GST–BIN2 or GST–BIN2<sup>Y200F</sup> was incubated with MBP–BZR1 and [γ-<sup>32</sup>P]ATP. CBB, Coomassie brilliant blue-staining. (b, c) BSU1 dephosphorylates pTyr 200 of BIN2 but not of bin2-1. Immunoblots of GST–BIN2, GST–BIN2<sup>Y200A</sup> and GST–bin2-1 incubated with MBP or MBP–BSU1 (b), or with BSU1–YFP immunoprecipitated from transgenic *Arabidopsis* (c), were probed with an anti-pTyr antibody and then with an anti-GST antibody. Samples were run on the same gel. Percentage indicates the relative signal level of pTyr 200 normalized to total GST–BIN2 or GST–bin2-1 protein level. (d, e) BR induces dephosphorylation of BIN2 pTyr 200 but not bin2-1. (d) The *det2* mutant was treated with MG132 (10 μM) for 1 h before treatment with brassinolide (BL; 0.2 μM) for the indicated times. BIN2 protein was immunoprecipitated with a polyclonal anti-serum and immunoblotted with anti-pTyr, anti-BIN2 serum, and an anti-GSK3 α/β antibody. (e) Transgenic plants expressing BIN2–Myc or bin2-1–Myc was pretreated with MG132 (10 μM) and then treated with brassinolide (+ BL; 0.25 μM) or mock solution (− BL). BIN2–Myc and bin2-1–Myc were immunoprecipitated by an anti-Myc antibody and gel

blots were probed with the antibodies indicated. (f, g) Phosphorylation of Tyr 200 is required for BIN2 inhibition of plant growth. (f) Overexpression of *BIN2–YFP* but not *BIN2<sup>Y200F</sup>–YFP* causes severe dwarf phenotypes in the T<sub>1</sub> generation. Upper left panel shows a zoom view. Lower panel shows BIN2–YFP and BIN2<sup>Y200F</sup> protein levels detected by anti-GFP antibodies. A nonspecific band serves as a loading control. (g) Overexpression of *bin2-1–myc* but not *bin2-1<sup>Y200F</sup>–myc* causes dwarf phenotypes. Seventy-six out of 281 35S::bin2-1–myc transgenic T<sub>1</sub> seedlings and none out of 412 35S::bin2-1-Y200F–myc transgenic plants showed a dwarf phenotype. (h–j) BSU1 family members have an essential role in BR signalling. (h) Eight out of 27 *bsu1;bsl1* double-mutant plants transformed with an artificial microRNA construct targeting *BSL2* and *BSL3* (*BSL2,3-amiRNA*) showed dwarf phenotypes. Right panel shows zoom view of the quadruple mutant. (i) Phenotypes of 5-day-old dark-grown seedlings of *bsu1;bsl1/BSL2,3-amiRNA*. (j) Quantitative real time PCR analysis of SAUR-AC1 RNA expression. Error bar indicates mean ± s.e.m., *n* = 3. Full scans of immunoblots and in vitro kinase/phosphatase assays are shown in Supplementary Information, Fig. S12. Col-0, non-transgenic plant used as a control.

BSU1–YFP/*bril-116* plants (Fig. 2g). These results demonstrate that BSU1 functions between BRI1 and BIN2 in the BR signal transduction pathway.

The direct interaction and the requirement of phosphatase activity of BSU1 suggest that BSU1 inhibits BIN2 by dephosphorylating BIN2. We analysed the autophosphorylation sites of BIN2 in vitro using mass spectrometry, and identified pTyr 200 of BIN2 as a major phosphorylation site (Supplementary Information, Fig. S8a). The same residue was recently detected as an in vivo phosphorylation site of BIN2 by a phosphoproteome analysis of *Arabidopsis*<sup>26</sup>. Mutation of Tyr 200 to Phe (Y200F) in BIN2 greatly reduced its substrate phosphorylation (Fig. 3a), indicating an essential role of phosphorylation of this residue for full BIN2 activity. The amino-acid sequence flanking Tyr 200 of BIN2 is highly conserved in mammalian GSK3s (Supplementary Information, Fig. S8b), and a monoclonal antibody for pTyr 216 of human GSK3β<sup>27</sup> specifically detected wild-type GST–BIN2 but not the GST–BIN2 containing a

in vivo phosphorylation of Tyr 200 of BIN2 but not that of bin2-1 (Fig. 3d, e). These results demonstrate that BR signalling inhibits BIN2 through BSU1-mediated dephosphorylation of pTyr 200, and that the *bin2-1* mutation causes BR insensitivity by blocking this dephosphorylation.

To further confirm the role of Tyr 200 phosphorylation in BIN2 regulation, a Y200F mutation was generated. Whereas overexpression of wild-type BIN2 or mutant *bin2-1* caused BR-insensitive dwarf phenotypes in transgenic *Arabidopsis* plants, overexpression of BIN2 or *bin2-1* containing the Y200F mutation did not (Fig. 3f, g), indicating that Tyr 200 phosphorylation is essential for BIN2 to inhibit BR-dependent plant growth and that dephosphorylation of pTyr 200 is sufficient to inactivate BIN2. Consistent with an essential role of BSU1 and its homologues in inhibiting BIN2, suppressing BSL2 and BSL3 expression in the *bsu1; bsl1* double mutant caused a severe dwarf phenotype and reduced expression of the BES1-target gene SAUR-AC1 (Fig. 3h–j). Taken together, these results demonstrate that dephosphorylation by the BSU1-related phosphatases is the primary mechanism of BIN2 inactivation and is an essential step in BR signal transduction.

The *Arabidopsis* genome encodes ten GSK3/shaggy-like kinases (AtSKs), which are classified into four subgroups (Fig. 4a). A triple-knockout mutant for group II AtSKs, including BIN2, shows increased cell elongation but still accumulates phosphorylated BES1 and responds to brassinolide, indicating that other GSK3-like kinases also function in BR signalling²². We performed an interaction study between BZR1 and nine AtSKs representing four subgroups, and found that all six AtSKs of subgroups I and II interact with BZR1 in yeast two-hybrid assays (Fig. 4b). We further examined the function of AtSK12 as a representative of subgroup I AtSKs in BR signalling. BiFC assays showed that, in *Arabidopsis*, AtSK12 interacts with BZR1 as does BIN2, and that deletion of the C-terminal 29 amino acids of AtSK12 abolished the interaction with BZR1 (Fig. 4c; Supplementary Information, Fig. S9a). Transgenic plants overexpressing AtSK12 or AtSK12<sup>E297K</sup> (corresponding to the *bin2-1* gain-of-function mutation) displayed similar dwarf phenotypes to plants overexpressing BIN2 or *bin2-1* (Fig. 4d)¹³. Similarly to BIN2, AtSK12 strongly phosphorylates BZR1 *in vitro* (Fig. 4e), is localized in both the cytoplasm and nucleus independently of BR treatment (Supplementary Information, Fig. S9b), is stabilized by the BR biosynthetic inhibitor brassinazole (BRZ; Supplementary Information, Fig. S9c) and is destabilized by brassinolide or by overexpression of BSU1-YFP (Fig. 4f, g). Mass spectrometry analysis indicated that Tyr 233 of AtSK12 (corresponding to Tyr 200 of BIN2) is also phosphorylated (Supplementary Information, Fig. S9d). BR treatment greatly reduced phosphorylation of AtSK12 Tyr 233, indicating a similar mechanism of its regulation by BR to that of BIN2 (Fig. 4h). These results indicate that BSU1-mediated tyrosine dephosphorylation is a common mechanism shared by at least two of the six GSK3-like kinases that are probably involved in BR signalling.

The functioning of BSU1 upstream of BIN2 suggests that it might be directly regulated by upstream components on the plasma membrane. We tested for direct interaction of BSU1 with BRI1, BAK1 and BSK1 *in vitro* and found that the MBP-BSU1 protein interacted with BSK1 but not with BRI1 or BAK1 (Fig. 5a), which is consistent with BSK1 being downstream of BRI1 in the signalling pathway. BiFC assays showed that BSK1 interacts with both BSU1 and BSL1 *in vivo* (Fig. 5b), and the interaction was further confirmed by co-immunoprecipitation assays (Fig. 5c). We have shown previously that BRI1 phosphorylates BSK1 at Ser 230. We found that phosphorylation of BSK1 by BRI1 increased, whereas the mutation BSK1 S230A abolished, the binding of BSK1 to BSU1 (Fig. 5d), indicating that BRI1 phosphorylation of BSK1 at Ser 230 increases the interaction of

Y200A mutation or the kinase-inactivating M115A mutation (Fig. 4b; Supplementary Information, Fig. S8c), indicating specificity for the pTyr 200 residue of BIN2. Immunoblot experiments using this antibody showed that BSU1 greatly reduced Tyr 200 phosphorylation of wild-type BIN2, but not that of the mutant *bin2-1* (Fig. 3b, c). Brassinolide treatment also reduced

Figure 5 BSK1 directly interacts with BSU1.

(a) The GST fusion proteins of the kinase domains of BRI1 (GST–BRI1-K) and BAK1 (GST–BAK1-K) and full-length BSK1 (GST–BSK1) were separated by SDS–PAGE and blotted onto a nitrocellulose membrane. The blot was probed sequentially with MBP–BSU1 and an anti-MBP antibody (top) and then stained with Ponceau S (bottom).

(b) BiFC assays show the in vivo interaction between BSU1 or BSL1, and BSK1. Tobacco leaf epidermal cells were transformed with indicated constructs. At5g49760 is a receptor kinase unrelated to BR signalling, used here as a negative control. Bright spots in nYFP + BSK1–cYFP and At5g49760–nYFP + BSK1–cYFP cells are chloroplast autofluorescence. Scale bars, 10 μm.

(c) Total protein extracts obtained from Arabidopsis plants expressing BSU1–YFP or co-expressing BSU1–YFP and BSK1–Myc were immunoprecipitated with anti-Myc, and the immunoblot was probed with anti-GFP and anti-Myc antibodies.

(d) Phosphorylation of BSK1 Ser 230 by BR1 enhances BSK1 binding to BSU1. GST–BSK1 or GST–BSK1<sup>S230A</sup> was incubated with GST–BRI1-K or GST for 2 h. An overlay assay was performed

BSK1 with BSU1. These results demonstrate that BRI1 phosphorylation of BSK1 Ser 230 promotes BSK1 binding to BSU1. Such interaction with BSK1 is likely to mediate BR activation of BSU1 in vivo, although we did not detect an effect of BSK1 on BSU1 activity in vitro (data not shown).

Together, our results bridge the last major gaps and elucidate a complete BR signalling pathway that involves several steps of sequential phosphorylation/dephosphorylation, transducing the signal from the BRI1–BAK1 receptor kinase complex to BSK1, BSU1, BIN2, and BZR1 and BZR2 (Fig. 5e). In the absence of BR, BZR1 and BZR2 are inhibited by BIN2-catalysed phosphorylation and are consequently excluded from the nucleus due to binding by the 14-3-3 proteins, loss of DNA binding activity and degradation by the proteasome. BR binding to the extracellular domain of BRI1 activates BRI1 kinase through ligand-induced association

as described in a. Full scans of immunoblots are shown in Supplementary Information, Fig. S12. (e) The BR signal transduction pathway. Filled objects indicate components in active states and open objects, inactive states. In the absence of BR (−BR), BRI1 is kept in an inactive form with help from its inhibitor BKI1; consequently BAK1, BSK1 and BSU1 are inactive, while BIN2 is active and phosphorylates BZR1 and BZR2 (BZR1/2), leading to loss of their DNA-binding activity, exclusion from the nucleus by the 14-3-3 proteins and degradation by the proteasome. In the presence of BR (+BR), BR binding to the extracellular domain of BRI1 induces dissociation of BKI1 and association and inter-activation between BRI1 and BAK1. Activated BRI1 then phosphorylates BSK1, which in turn dissociates from the receptor complex and interacts with, and presumably activates, BSU1. BSU1 inactivates BIN2 by dephosphorylating it at pTyr 200, allowing accumulation of unphosphorylated BZR1/2, probably with help from an unknown phosphatase. Unphosphorylated BZR1/2 accumulates in the nucleus and binds to promoters to regulate the expression of BR-target genes, leading to cellular and developmental responses.

and trans-phosphorylation with its co-receptor kinase BAK1 (ref. 12). BRI1 then phosphorylates BSK1 kinase at Ser 230 (ref. 10), and this phosphorylation promotes BSK1 interaction with, and activation of, BSU1. On activation, BSU1 dephosphorylates BIN2 at its pTyr 200 residue to inhibit BIN2 kinase activity, allowing accumulation of unphosphorylated BZR1 and BZR2 in the nucleus where they regulate BR-responsive gene expression and plant growth (Fig. 5e; Supplementary Information, Fig. S1b).

Signal transduction through cell surface receptor kinases is a fundamental mechanism for cellular regulation in living organisms. BRI1 is a member of the large family of leucine-rich-repeat receptor-like kinases (LRR-RLK), with over 220 members in *Arabidopsis* and 400 in rice<sup>28</sup>. Only a handful of these RLKs have been studied<sup>29</sup>. The BR signalling pathway described in this study represents the first complete RLK-mediated signalling pathway
in plants and provides a model for understanding other signal transduction pathways in plants.

Each BR signalling component in *Arabidopsis* is encoded by a small gene family with three to six members, which have similar biochemical functions. *BRI1* is the only essential gene for BR signalling that has been identified by recessive mutations, yet two BRI1 homologues, BRL1 and BRL3, seem to mediate BR signalling in a tissue-specific manner<sup>11,30,31</sup>. All the other components have been identified either by gain-of-function mutations or by proteomic/biochemical approaches<sup>10</sup>. Single knockout of *BIN2*, *BZR1*, *BES1*, *BSU1* or *BSK1* caused no obvious phenotype or a very subtle growth phenotype, suggesting genetic redundancy among the members of each gene family. Our transgenic and biochemical studies provide strong evidence that six *Arabidopsis* GSK3s (groups I and II) are involved in BR signalling (Fig. 4), and our loss-of-function data show that all four members of the BSU1 family contribute to BR signalling (Fig. 3h–j). As such, each step of BR signal transduction seems to be carried out by several members of the gene family, although only the founding member of each family is presented in the conceptual model of BR signal transduction (Fig. 5e). In contrast, microarray data indicates very similar ubiquitous expression patterns for BRI1, BSK1, BSU1, BSL1 and BZR1 (Supplementary Information, Fig. S10)<sup>14,31</sup>, supporting their major roles in BR responses.

Our study reveals BSU1-mediated pTyr 200 dephosphorylation as the primary mechanism for regulating plant GSK3s in the BR signalling pathway. This tyrosine residue is absolutely conserved in all GSK3s identified so far, and its phosphorylation is required for kinase activity in *Dictyostelium discoideum* and mammals<sup>27,32,33</sup>. However, the phosphatase required for this regulation has not been identified in these systems<sup>27,33–35</sup>. BSU1 represents the first phosphatase that mediates dephosphorylation of this conserved tyrosine residue of GSK3s. BSU1 contains an N-terminal Kelch-repeat domain and a C-terminal phosphatase domain<sup>14</sup>, which dephosphorylates both pSer/Thr and pTyr residues (Supplementary Information, Figs S2b, c and S11). The phosphatase domain of BSU1 shares about 45% sequence identity with mammalian PP1 (protein phosphatase-1). Interestingly, PP1 expressed in *Escherichia coli* shows both Tyr and Ser/Thr phosphatase activity, although native PP1 expressed in mammalian cells is inactive on pTyr due to inhibition by inhibitor-2 (ref. 36). It will be interesting to see if BSU1-related phosphatases mediate tyrosine dephosphorylation of GSK3s in mammals and other species.

### METHODS

Methods and any associated references are available in the online version of the paper at http://www.nature.com/naturecellbiology/.

Note: Supplementary Information is available on the Nature Cell Biology website.

### ACKNOWLEDGEMENTS

We thank J. Chory for providing the *bsu1-D* seeds and BSU1 cDNA clone and D. Bhaya and K. Barton for comments on the manuscript. Research was supported by grants from NIH (R01GM066258), the National Science Foundation (0724688), the US Department of Energy (DE-FG02-08ER15973) and the Herman Frasch Foundation. The UCSF Mass Spectrometry Facility is supported by the Biomedical Research Technology Program of the National Center for Research Resources, NIH NCRR (RR01614, RR012961 and RR019934).

### AUTHOR CONTRIBUTIONS

S.G. and A.L.B. carried out LC-MS/MS analysis. Y.S.<sup>1</sup> and Z.D. were involved in 2D PAGE immunoblotting. J.X.S. and Y.S.<sup>3</sup> developed BIN2 antiserum. W.T. generated GST-BSK1 proteins. T.W.K. performed all other experiments. T.W.K and Z.W. designed the experiments, analysed data and wrote the manuscript.

---

COMPETING FINANCIAL INTERESTS

The authors declare no competing financial interests.

Published online at http://www.nature.com/naturecellbiology/.

Reprints and permissions information is available online at http://npg.nature.com/reprintsandpermissions/.

1. Vert, G., Nemhauser, J. L., Geldner, N., Hong, F. & Chory, J. Molecular mechanisms of steroid hormone signaling in plants. *Annu. Rev. Cell Dev. Biol.* **21**, 177–201 (2005).
2. Gendron, J. M. & Wang, Z. Y. Multiple mechanisms modulate brassinosteroid signaling. *Curr. Opin. Plant Biol.* **10**, 436–441 (2007).
3. Clouse, S. D. & Sasse, J. M. BRASSINOSTEROIDS: Essential Regulators of Plant Growth and Development. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **49**, 427–451 (1998).
4. Belkhadir, Y., Wang, X. & Chory, J. *Arabidopsis* brassinosteroid signaling pathway. *Sci. STKE* **2006**, cm5 (2006).
5. Li, J. & Chory, J. A putative leucine-rich repeat receptor kinase involved in brassinosteroid signal transduction. *Cell* **90**, 929–938 (1997).
6. Wang, Z. Y., Seto, H., Fujioka, S., Yoshida, S. & Chory, J. BRI1 is a critical component of a plasma-membrane receptor for plant steroids. *Nature* **410**, 380–383 (2001).
7. Li, J. *et al*. BAK1, an *Arabidopsis* LRR receptor-like protein kinase, interacts with BRI1 and modulates brassinosteroid signaling. *Cell* **110**, 213–222 (2002).
8. Nam, K. H. & Li, J. BRI1/BAK1, a receptor kinase pair mediating brassinosteroid signaling. *Cell* **110**, 203–212 (2002).
9. Wang, X. & Chory, J. Brassinosteroids regulate dissociation of BKI1, a negative regulator of BRI1 signaling, from the plasma membrane. *Science* **313**, 1118–1122 (2006).
10. Tang, W. *et al*. BSKs mediate signal transduction from the receptor kinase BRI1 in *Arabidopsis*. *Science* **321**, 557–560 (2008).
11. Kinoshita, T. *et al*. Binding of brassinosteroids to the extracellular domain of plant receptor kinase BRI1. *Nature* **433**, 167–171 (2005).
12. Wang, X. *et al*. Sequential transphosphorylation of the BRI1/BAK1 receptor kinase complex impacts early events in brassinosteroid signaling. *Dev. Cell* **15**, 220–235 (2008).
13. Li, J. & Nam, K. H. Regulation of brassinosteroid signaling by a GSK3/SHAGGY-like kinase. *Science* **295**, 1299–1301 (2002).
14. Mora-Garcia, S. *et al*. Nuclear protein phosphatases with Kelch-repeat domains modulate the response to brassinosteroids in *Arabidopsis*. *Genes Dev.* **18**, 448–460 (2004).
15. Gampala, S. S. *et al*. An essential role for 14-3-3 proteins in brassinosteroid signal transduction in *Arabidopsis*. *Dev. Cell* **13**, 177–189 (2007).
16. Wang, Z. Y. *et al*. Nuclear-localized BZR1 mediates brassinosteroid-induced growth and feedback suppression of brassinosteroid biosynthesis. *Dev. Cell* **2**, 505–513 (2002).
17. Yin, Y. *et al*. A crucial role for the putative *Arabidopsis* topoisomerase VI in plant growth and development. *Proc. Natl Acad. Sci. U S A* **99**, 10191–10196 (2002).
18. He, J.-X. *et al*. BZR1 is a transcriptional repressor with dual roles in brassinosteroid homeostasis and growth responses. *Science* **307**, 1634–1638 (2005).
19. Yin, Y. *et al*. A new class of transcription factors mediates brassinosteroid-regulated gene expression in *Arabidopsis*. *Cell* **120**, 249–259 (2005).
20. He, J. X., Gendron, J. M., Yang, Y., Li, J. & Wang, Z. Y. The GSK3-like kinase BIN2 phosphorylates and destabilizes BZR1, a positive regulator of the brassinosteroid signaling pathway in *Arabidopsis*. *Proc. Natl Acad. Sci. USA* **99**, 10185–10190 (2002).
21. Ryu, H. *et al*. Nucleocytoplasmic shuttling of BZR1 mediated by phosphorylation is essential in *Arabidopsis* brassinosteroid signaling. *Plant Cell* **19**, 2749–2762 (2007).
22. Vert, G. & Chory, J. Downstream nuclear events in brassinosteroid signalling. *Nature* **441**, 96–100 (2006).
23. Peng, P., Yan, Z., Zhu, Y. & Li, J. Regulation of the *Arabidopsis* GSK3-like Kinase Brassinosteroid-insenstivie 2 through proteasome-mediated protein degradation. *Mol. Plant* **1**, 338–346 (2008).
24. Hu, C. D., Chinenov, Y. & Kerppola, T. K. Visualization of interactions among bZIP and Rel family proteins in living cells using bimolecular fluorescence complementation. *Mol. Cell* **9**, 789–798 (2002).
25. Benschop, J. J. *et al*. Quantitative phosphoproteomics of early elicitor signaling in *Arabidopsis*. *Mol. Cell Proteomics* **6**, 1198–1214 (2007).
26. Sugiyama, N. *et al*. Large-scale phosphorylation mapping reveals the extent of tyrosine phosphorylation in *Arabidopsis*. *Mol. Syst. Biol.* **4**, 193 (2008).
27. Kim, L., Harwood, A. & Kimmel, A. R. Receptor-dependent and tyrosine phosphatase-mediated inhibition of GSK3 regulates cell fate choice. *Dev. Cell* **3**, 523–532 (2002).
28. Shiu, S. H. *et al*. Comparative analysis of the receptor-like kinase family in *Arabidopsis* and rice. *Plant Cell* **16**, 1220–1234 (2004).
29. Johnson, K. L. & Ingram, G. C. Sending the right signals: regulating receptor kinase activity. *Curr. Opin. Plant Biol.* **8**, 648–656 (2005).
30. Zhou, A., Wang, H., Walker, J. C. & Li, J. BRL1, a leucine-rich repeat receptor-like protein kinase, is functionally redundant with BRI1 in regulating *Arabidopsis* brassinosteroid signaling. *Plant J.* **40**, 399–409 (2004).
31. Cano-Delgado, A. *et al*. BRL1 and BRL3 are novel brassinosteroid receptors that function in vascular differentiation in *Arabidopsis*. *Development* **131**, 5341–5351 (2004).
32. Hughes, K., Nikolakaki, E., Plyte, S. E., Totty, N. F. & Woodgett, J. R. Modulation of the glycogen synthase kinase-3 family by tyrosine phosphorylation. *EMBO J.* **12**, 803–808 (1993).
33. Bhat, R. V. *et al*. Regulation and localization of tyrosine216 phosphorylation of glycogen synthase kinase-3β in cellular and animal models of neuronal degeneration. *Proc. Natl Acad. Sci. USA* **97**, 11074–11079 (2000).
34. Kim, L. & Kimmel, A. R. GSK3 at the edge: regulation of developmental specification and cell polarization. *Curr. Drug Targets.* **7**, 1411–1419 (2006).
35. Cole, A., Frame, S. & Cohen, P. Further evidence that the tyrosine phosphorylation of glycogen synthase kinase-3 (GSK3) in mammalian cells is an autophosphorylation event. *Biochem. J.* **377**, 249–255 (2004).
36. MacKintosh, C. *et al*. Further evidence that inhibitor-2 acts like a chaperone to fold PP1 into its native conformation. *FEBS Lett.* **397**, 235–238 (1996).
DOI: 10.1038/ncb1970

# METHODS

Materials. The *bri1-5* mutant is in Wassilewskija (Ws) ecotype background, and all other *Arabidopsis thaliana* plants are in Columbia ecotype background. The *det2*, *BIN2–Myc*, *bin2-1–Myc*, *AtSK12–Myc* and *BSU1–YFP* plants for western blotting or *in vitro* kinase and phosphatase assays were sterilized with bleach and grown in agar plates containing half strength (× 0.5) Murashige-Skoog (MS) medium under continuous light for 10 days. Tobacco (*Nicotiana benthamiana*) plants were grown in a greenhouse under 16 h light/8 h dark cycles. All fusion proteins were expressed by the 35S promoter, unless indicated otherwise, in transient assays or in stable plant transformation experiments.

Phenotypic analysis of hypocotyls. Sterilized *Arabidopsis* seeds were planted on × 0.5 MS agar plates. Cold-treated agar plates were kept under white light for 6 h and plants were vertically grown in the dark for 5 days. The seedlings were photocopied by digital camera.

*In vitro* kinase and phosphatase assays. MBP–BZR1 and GST–BIN2 proteins were expressed and purified from *E. coli*, and maltose or glutathione was removed from the proteins by ultrafiltration using Centricon 50 (Amicon Ultra, Millipore). To prepare fully phosphorylated BZR1 proteins, MBP–BZR1 was incubated with GST–BIN2 at a 1:1 ratio in the kinase buffer (20 mM Tris at pH 7.5, 1 mM MgCl₂, 100 mM NaCl and 1 mM DTT) containing ATP (100 μM) at 30°C overnight. The protein mixture was incubated with glutathione Sepharose beads to remove GST–BIN2, then with amylose beads to purify MBP–pBZR1. Partially phosphorylated ³²P-labelled pBZR1 and pBZR2 were prepared using the same method but MBP–BZR1 or MBP–BZR2 was incubated with GST–BIN2 at a 15:1 ratio for 3 h in the presence of [γ-³²P]ATP (20 μCi). For dephosphorylation, GST–BSU1 was incubated with fully phosphorylated MBP–pBZR1 and ³²P-MBP–pBZR1 or ³²P-MBP–pBZR2 for 12 or 16 h.

*In vitro* BIN2 inhibition assays were performed by 3 h co-incubation of MBP–BZR1, GST–BIN2, GST–BSU1 and [γ-³²P]ATP or pre-incubation of GST–BIN2 with GST–BSU1 for various times before adding MBP–BZR1 and [γ-³²P]ATP. To examine activities of different domains of BSU1, the N-terminal Kelch (amino-acids 1–363) and C-terminal phosphatase (amino-acids 364–793) regions were used. GST, GST–BSU1, GST–BSU1-Kelch and GST–BSU1-phosphatase were pre-incubated with GST–BIN2 for 1 h, and further incubated with MBP–BZR1 and [γ-³²P]ATP for 3 h.

To test the activity of BSU1–YFP, anti-GFP antibody-Protein A beads were used to immunoprecipitate BSU1–YFP from extracts of BSU1–YFP-transgenic plants. Non-transgenic wild-type plants were used as a control. The beads were incubated with GST–BIN2 or GST–bin2-1 for 1 h, and then the beads were removed. The BSU1-treated GST–BIN2 or GST–bin2-1 was further incubated with MBP–BZR1 and [γ-³²P]ATP for 3 h.

An *in vitro* phosphatase assay using phospho-MBP was performed according to the manufacturer’s protocol (New England Biolab). To examine tyrosine phosphatase activity of BSU1, *p*-nitrophenyl phosphate (20 mM) was incubated with MBP–BSU1 in 50 ul of reaction buffer (50 mM Tris at pH 7.2, 20 mM NaCl, 5 mM DTT and 10 mM MgCl₂). The reaction was quenched by the addition of NaOH (100 ul of 0.5 M) after incubation at 30 °C for 1 h. *p*-nitrophenol production was determined by measuring A₄₀₅ (extinction coefficient, ε = 1.78 × 10⁴ M⁻¹cm⁻¹).

Immunoprecipitation and co-immunoprecipitation. Plant materials were ground with liquid nitrogen and resuspended in IP buffer (50 mM Tris at pH 7.5, 150 mM NaCl, 5% Glycerol, 1% Triton X-100, 1 mM PMSF and 1 × protease inhibitor cocktail from Sigma). Filtered protein extracts were centrifuged at 20,000g for 10 min and the resulting supernatant was incubated with an anti-GFP antibody bound Protein A beads or anti-Myc agarose beads for 1 h. Beads were washed five times with washing buffer (50 mM Tris at pH 7.5, 150 mM NaCl, 0.2% Triton X-100, 1 mM PMSF and 1 × Protease inhibitor cocktail). The beads were resuspended with a small volume of kinase buffer and used for *in vitro* phosphatase assays, or immunoprecipitated proteins were eluted with buffer containing 2% SDS and analysed by SDS–PAGE and immunoblotting.

Dephosphorylation of BIN2 at pTyr 200. GST–BIN2 or GST–bin2-1 was incubated with MBP–BSU1 or BSU1–YFP beads for 3 h and subjected to immunoblotting. The pTyr 200 residue of BIN2 was detected by an anti-phospho-GSK3α/β (Tyr 279/216) monoclonal antibody (5G-2F, Millipore; 1:2,000) and re-probed

with an HRP-conjugated anti-GST antibody (Santa Cruz Biotechnology; 1:2,500). The *det2* plants were treated with brassinolide (0.2 μM) after 1 h incubation with MG132 (10 μM). Anti-BIN2 serum was developed in rabbits using GST–BIN2 as an immunogen. A monoclonal anti-GSK3α/β antibody was purchased from Invitrogen (1:2,000).

Site-directed mutagenesis. Point mutations were generated by site-directed mutagenesis PCR according to the manufacturer’s protocol (Stratagene). The primers used for different mutagenesis were: *BIN2*²⁰⁰ᶠ forward, 5′-GAAGCCAACATTCTTTTCATCT GCTCACGATT-3′; *BIN2*²⁰⁰ᶠ reverse, 5′-AAGCCAAACATTCTTTCATCTGCTCACGATT C-3′; *BIN2*²⁰⁰ᵃ forward, 5′-GAAGCCAACATTCTGCCATCTGCTCACGATT-3′; *BIN2*²⁰⁰ᵃ reverse, 5′-GAATCGTGAGCAGATGGCAGAAATGTTGGCTTC TGAGA-3′; *BIN2*¹¹⁵ᵃ forward, 5′-CTTTTCTTGAACTTGGTTGCGGAGTATGTCCC TGAGA-3′; *BIN2*¹¹⁵ᵃ reverse, 5′-TCTC AGGGACATACTCCGCAACCAA GTTCAAGAAAAG-3′; *AtSKI2*²⁹⁷ᴷ forward, 5′-GAACA CCAACAAGGG AAAAAATCAAATGCATGAAACC-3′; *AtSKI2*²⁹⁷ᴷ reverse, 5′-GGGTTC ATGCATTTGATTTTTCCCTTGTTGGTGTTC-3′; *BSU1*⁵¹⁰ᴺ forward, 5′-CAATCAAAGT CTTCGGCAATATCCCATGGACAATAC-3′; *BSU1*⁵¹⁰ᴺ reverse, 5′-GTATTGTCCATGGAT ATTGCCGAAGACTTTGATTG-3′.

Overexpression and knockout/down of BSU1-related phosphatases. Full-length cDNAs of *BSU1* and *BSL1* without stop codons were amplified by PCR using gene-specific primers (*BSU1* forward, 5′-caccATGGCTCCTGATCAATCTTATCAATA T-3′; *BSU1* reverse, 5′-TTCACTTGACTCCCCCTCGAGCTGGAGTAG-3′; *BSL1* forward, 5′-caccATGGGCTCGA AGCCTTGGCTACATCCA-3′; *BSL1* reverse, 5′-GATGTATGCAAGCGAGCTTCTGTCAAAA TC-3′) from reverse transcription of *Arabidopsis* mRNA and cDNA clone (RIKEN, RAFL09-11-J01), respectively. The cDNAs were cloned into pENTR/SD/D-TOPO vectors (Invitrogen) and subcloned into gateway compatible pEarleyGate 101, pGWB17, pGWB20 or BiFC vectors¹⁵ by using LR reaction kit (Invitrogen). To test phenotypic suppression of *bri1-116* and *bin2-1* by *BSU1*, 35S::*BSU1–YFP* single plant was crossed into *bri1-116* and *bin2-1*. The phenotype of F₃ double homozygous plants was analysed. To generate the quadruple loss-of-function mutant of *bsu1-1, bsl1/BSL2,3-amiRNA*, the double mutant of *bsu1-1* (SALK_030721) and *bsl1-1* (SALK_051383)³⁷ was transformed with an artificial microRNA construct targeting both *BSL2* and *BSL3* genes (*BSL2,3-amiRNA*), which was designed by the Web MicroRNA Designer 2 (ref. 38), using the oligonucleotide 5′-TATTCATCAAAAAGGCGCGTG-3′ and plasmid pRS300 (ref. 38). The DNA fragment of amiRNA was cloned into pEarleyGate 100 (pEG100) by using the Gateway cloning kit (Invitrogen), yielding *BSL2,3-amiRNA*/pEG100. The binary vector constructs were introduced into *Agrobacterium* strain GV3101 by electroporation and transformed into *Arabidopsis* by using the floral dipping method.

Quantitative real time PCR. SAUR-AC1 mRNA was analysed using quantitative real-time PCR as described by Gampala *et al.*¹⁵ using gene-specific primers (SAUR-AC1 forward, 5′-AAGAGGATTCATGGCGGTCTATG-3′; SAUR-AC1 reverse, 5′-GTATTGTTAAGCCGCCC ATTGG-3′). UBC (UBC forward, 5 ′-CAAATCCAAAAACCCTAGAAACCGAA-3′; UBC reverse, 5′-ATCTC CCGTAGGACCTGCACTG-3′) was used to normalize the loading.

Yeast two-hybrid assays of AtSKs. The cDNA clones of *AtSKs* were obtained from ABRC (http://www.biosci.ohio-state.edu/pcmb/Facilities/abrc/abrchrome.htm)³⁹. All *AtSK* cDNAs were subcloned into gateway compatible pGADT7 vector (Clontech). Nine *AtSKs*-pGADT7 constructs and empty pGADT7 vector were transformed into cells containing BZR1-pGBK7. Yeast clones were grown on synthetic dropout (SD) or SD-histidine containing 3-amino-1, 2, 4-triazole (2.5 ~ 10 mM).

*In vitro* kinase assay of AtSK12. GST–AtSK12 (1 μg) was incubated with MBP–BZR1 (2 μg), ATP (100 μM) and [γ-³²P]ATP (10 μCi) in the kinase buffer for 2 h. The reaction was terminated by addition of 2 × SDS loading buffer and separated by 7.5% SDS–PAGE. Gel was stained with Coomassie brilliant blue followed by drying. The radioactivity was analysed by Phospho-image screen using Typhoon 8600 Scanner (GE Healthcare).

Determination of *in vitro* phosphorylation sites of BIN2 and AtSK12. GST–BIN2 or GST–AtSK12 protein (25 μg) purified from *E.coli* was incubated with

METHODOLOGY

ATP (100 μM) in the kinase buffer for 16 h at 30 °C. Autophosphorylated GST–BIN2 or GST–AtSK12 was subjected to in-solution alkylation/tryptic digestion followed by LC-MS/MS analysis as described by Gampala *et al.*¹⁵.

**Overlay western blot.** To test the interaction of BSU1 with BIN2 or bin2-1 *in vitro*, a gel blot separating GST, GST–BIN2 and GST–bin2-1 was incubated with MBP–BSU1 (20 μg) in 5% non-fat dry milk/PBS buffer and washed four times. The blot was then probed with a polyclonal anti-MBP antibody. In the case of BSU1 overlay with BSK1, GST–BRI1-K (K, kinase domain), GST–BAK1-K and GST–BSK1 were separated by SDS-PAGE. To prepare phosphorylated BSK1, GST–BSK1 was incubated with GST–BRI1-K and ATP (100 μM) in the kinase buffer for 2 h before SDS–PAGE. The blot was sequentially probed with MBP–BSU1 and a monoclonal anti-MBP antibody (New England Biolab; 1:2,000).

**Immunoblotting of 2D gel electrophoresis.** Total proteins were extracted from brassinolide-treated or untreated 35S::TAP-BIN2 plants for 2D gel electrophoresis as described previously⁴⁰. The amount of brassinolide-treated and untreated

TAP-BIN2 proteins was normalized with western blotting. Equal amounts of TAP-BIN2 proteins were separated by 2D gel electrophoresis using an immobilized pH gradient gel strip (7 cm, pH 3–10 non-linear) and 7.5% SDS–PAGE gel. The blots were probed with an anti-PAP antibody (Sigma; 1:2,000).

**Transient transformation and confocal microscopy.** Transformation by *Agrobacterium* infiltration and observations of subcellular localization and BiFC signal in tobacco or *Arabidopsis* were performed as described previously¹⁵. Fluorescence of YFP was visualized by using a spinning-disk confocal microscope (Leica Microsystems).

37. Alonso, J. M. *et al.* Genome-wide insertional mutagenesis of *Arabidopsis thaliana*. *Science* **301**, 653–657 (2003).

38. Schwab, R., Ossowski, S., Riester, M., Warthmann, N. & Weigel, D. Highly specific gene silencing by artificial microRNAs in *Arabidopsis*. *Plant Cell* **18**, 1121–1133 (2006).

39. Yamada, K. *et al.* Empirical analysis of transcriptional activity in the *Arabidopsis* genome. *Science* **302**, 842–846 (2003).

40. Deng, Z. *et al.* A proteomics study of brassinosteroid response in *Arabidopsis*. *Mol. Cell Proteomics* **6**, 2058–2071 (2007).

SUPPLEMENTARY INFORMATION

**DOI: 10.1038/ncb1970**

---

**Figure S1** The model of the BR signal transduction pathway before (a) and after (b) this study. In the absence of BR, the GSK3-like kinase BIN2 phosphorylates two transcription factors, BZR1 and BZR2 (pBZR1/2), to inhibit BR-responsive gene expression. Upon activation by BR binding, BRI1 receptor kinase phosphorylates BSKs, and this leads to accumulation of dephosphorylated BZR1 and BZR2, most likely by inhibiting BIN2 or activating BSU1. (a) In previous models of BR signaling, BSU1 was proposed to mediate dephosphorylation of BZR1 and BZR2, and the mechanism for inhibiting BIN2 kinase remains unknown. (b) Results of this study demonstrate that BSU1 does not directly dephosphorylate BZR1 or BZR2. Instead, it dephosphorylates BIN2 at tyrosine 200 to inactivate BIN2 kinase activity and inhibit BIN2 phosphorylation of BZR1 and BZR2. BR-activated BRI1 phosphorylates BSKs to promote its binding and activation of BSU1. Arrows show promotion actions and bar ends show inhibitory actions. Solid lines show direct regulation, and dotted lines indicate hypothetical regulation.

---

WWW.NATURE.COM/NATURECELLBIOLOGY

© 2009 Macmillan Publishers Limited. All rights reserved.

SUPPLEMENTARY INFORMATION

a

b

c

Figure S2 Overexpression phenotype of BSU1 homolog, BSL1 and phosphatase activities of BSU1 and BSL1 purified from E.coli. (a) The bri1-5 overexpressing BSL1-YFP (BSL1-YFP/bri1-5, left) and untransformed bri1-5 (right) were grown in soil for six weeks. (b) Both GST-BSU1 and GST-BSL1 dephosphorylate phospho-myelin basic protein. (c) GST-BSU1 requires manganese ion for its activity. All metal ions were added to the phosphatase reactions as 1 mM final concentration.

SUPPLEMENTARY INFORMATION

a
| GST | + | - | - |
| --- | --- | --- | --- |
| GST-BSU1 | - | + | - |
| GST-BSL1 | - | - | + |
| GST-BIN2 | + | + | + |
| MBP-BZR1 | + | + | + |

32P-pBZR1

CBB

b
| GST | + | - | - |
| --- | --- | --- | --- |
| GST-BSU1 | - | + | - |
| GST-BSL1 | - | - | + |
| GST-BIN2 | + | + | + |
| MBP-BZR2 | + | + | + |

32P-pBZR2

CBB

c
| 32P-pBZR1 |  |
| --- | --- |
| CBB |  |
| 32P-MBP-pBZR1 | + | + | + |
| GST | + | - | - |
| GST-BSU1 | - | + | - |
| GST-BSL1 | - | - | + |

d
| 32P-pBZR2 |  |
| --- | --- |
| CBB |  |
| 32P-MBP-pBZR2 | + | + | + |
| GST | + | - | - |
| GST-BSU1 | - | + | - |
| GST-BSL1 | - | - | + |

e
| 32P-pBZR1 |  |
| --- | --- |
| CBB |  |
| GST-BSU1 | + | - | - | - |
| GST-BSU1-P | - | + | - | - |
| GST-BSU1-KL | - | - | + | - |
| GST | - | - | - | + |
| GST-BIN2 | + | + | + | + |
| MBP-BZR1 | + | + | + | + |

Figure S3 In vitro kinase/phosphatase assay of BSU1 and BSL1. (a, b) BSU1 and BSL1 inhibit BIN2 phosphorylation of BZR1 and BZR2. GST-BIN2 and GST-BSU1 or GST-BSL1 were co-incubated with MBP-BZR1 (a) or MBP-BZR2 (b) and 32P-γATP for 3 hrs at 30°C. (c, d) BSU1 and BSL1 do not dephosphorylate phosphorylated BZR1 and BZR2 in vitro. 32P-pBZR1 and 32P-pBZR2 were prepared by incubation with GST-BIN2 and 32P-γATP followed by removal of GST-BIN2 and 32P-γATP by sequential purification using glutathione and amylose beads. Pre-labeled 32P-pBZR1 (c) and 32P-pBZR2 (d) were then incubated with GST, GST-BSU1 and GST-BSL1, respectively, for 16 hrs at 30°C. (e) BSU1 and BSU1 phosphatase domain inhibit BIN2 phosphorylation of BZR1. GST-BIN2 and GST-BSU1 or GST-BSU1-P (C-terminal phosphatase domain) or GST-BSU1-KL (N-terminal Kelch domain) were pre-incubated for 1 hr, and then incubated with MBP-BZR1 and 32P-γATP for 3 hrs at 30°C. CBB indicates Coomassie brilliant blue stained-gel.

SUPPLEMENTARY INFORMATION

a
| Col-0 | + | - | - |
| --- | --- | --- | --- |
| BSU1-YFP | - | + | + |
| BL | - | - | + |
| 32P-MBP-pBZR1 | + | + | + |

Anti-GFP IP

32P-pBZR1 | Autoradiograph
BSU1-YFP | CBB

b
| + MBP-BZR1 /GST-BIN2 |  |  |
| --- | --- | --- |
| Col-0 | + | - | - |
| BSU1-YFP | - | + | + |
| BL | - | - | + |

Anti-GFP IP

32P-pBZR1 | Autoradiograph
| CBB |

Figure S4 BSU1-YFP inhibits BIN2 activity but does not dephosphorylate phosphorylated BZR1. BSU1-YFP protein was immunoprecipitated (IP) from 35S-BSU1-YFP transgenic Arabidopsis plants. (a) BSU1-YFP was incubated for 3 hrs with pre-phosphorylated 32P-MBP-BZR1 after removal of GST-BIN2. (b) BSU1-YFP immunoprecipitated from plants treated with 0.5 μM BL or mock solution was incubated with GST-BIN2, MBP-BZR1 and 32P-γATP for 3 hrs. CBB indicates Coomassie brilliant blue stained-gel.

SUPPLEMENTARY INFORMATION

a
![Image](image_a.png)

b
| BIN2-nYFP | bin2-1-nYFP |
|-----------|-------------|
| cYFP-BSU1 |             |
| BSL1-cYFP |             |
| cYFP      |             |

c
![Image](image_c.png)

Figure S5 *In vivo* interaction between BSU1 or BSL1 and BIN2 or bin2-1 in BiFC assays, and distinct subcellular localization patterns of BSU1 and BSL1 in transgenic *Arabidopsis* plants. (a) BIN2-myc and bin2-1-myc proteins expressed in transgenic *Arabidopsis* were immuno-precipitated (IP) by anti-myc antibody, and the beads were then incubated with extracts of BSU1-YFP overexpressing plants. Immunoblot was probed with anti-myc and anti-GFP antibody. Col-0, wild type plants expressing no BIN2-myc. (b) Cells co-transformed with BIN2 or bin2-1 fused N-terminal half (nYFP) and BSU1 or BSL1 fused C-terminal half (cYFP) of yellow fluorescence protein (YFP) showed good fluorescence signal consistent with their subcellular localization patterns, whereas cells co-expressing BIN2 or bin2-1-nYFP and non-fusion cYFP showed only auto-fluorescence of chloroplast. (c) Confocal images show BSU1-YFP (right) and BSL1-YFP (left) signal in hypocotyls of *Arabidopsis* seedlings grown in the dark for 5 days. The scale bar is 10 μm.

SUPPLEMENTARY INFORMATION

Figure S6 The substitution of BSU1 Asp510 to Asn abolishes its activity for decrease of BIN2 level and BR treatment reduces the level of BIN2 but not bin2-1 proteins. (a) Phosphatase assay using phospho-myelin basic proteins as a substrate showed that BSU1-D510N mutant has about 15% phosphatase activity of the wild type protein. GST and GST-BSU1-Kelch domain (KL) were used as a negative control. (b) BSU1-D510N-YFP (left) shows same subcellular localization pattern as wild type BSU1-YFP (right) in Arabidopsis leaf epidermal cells. The scale bar is 10 μm. (c) BSU1-D510N overexpression cannot decrease the BIN2-myc protein amount in Arabidopsis. Immunoblot of total proteins was probed with anti-myc and anti-GFP antibody. (d) BIN2-myc mRNA level in BSU-YFPxBIN2-myc is similar to Col-0xBIN2-myc. Semi-quantitative RT-PCR analysis was performed to compare BIN2-myc mRNA expression level. PP2A (At1g13320) was used as normalization control. (e) Tobacco leaves transformed with 35S-BIN2-myc or 35S-bin2-1-myc constructs were treated with 1 μM BL for 1 hr. Immunoblot of total proteins was probed with anti-myc antibody. (f) Transgenic plants expressing BIN2-myc or bin2-1-myc were treated with 0.25 μM BL for 30 min. Proteins immunoprecipitated by anti-myc agarose were blotted onto nitrocellulose membrane and probed by anti-myc antibody.

SUPPLEMENTARY INFORMATION

a  
![](https://example.com/image1.png)

b  
![](https://example.com/image2.png)  
bin2-1(-/-)  
BSU1-YFP  
/bin2-1(-/-)  
Col-0  
bin2-1(+/-)

c  
![](https://example.com/image3.png)  
bin2-1(-/-)  
BSU1-YFP  
/bin2-1(-/-)  
plasmid  

BSU1-YFP

Figure S7 The bin2-1 mutation suppresses the BSU1-overexpression phenotypes. (a) Homozygous bin2-1 (left) and bin2-1;bsu1-D (right) plants. (b, c) Genotyping of plants shown in Fig. 2e. (b) The DNA fragments containing bin2-1 mutation site amplified by PCR were digested with Xhol restriction enzyme. (c) BSU1-YFP DNA fragments were amplified with PCR using 35S promoter- and BSU1-specific primers.

SUPPLEMENTARY INFORMATION

a

GEANIS**pY**ICSR  
Theoretical m/z = 675.282  
Experimental m/z = 675.285  
+4.4ppm  

b

Human/mouse GSK3α 267 --- KQLVRGE**ENVSYICSRYY** --- 286  
Human/mouse GSK3β 204 --- KQLVRGE**ENVSYICSRYY** --- 223  
BIN2 188 --- KQLVKGE**ANISYICSRFY** --- 207  

c

GST-BIN2 M115A  
GST-BIN2  
GST-BIN2 Y200A  

Anti-pTyr 279/216GSK3α/β  
---  

Anti-GST  
---  

Figure S8 Tyrosine 200 residue of BIN2 is autophosphorylated and specifically detected by human anti-phospho-Tyr 297/216GSK3α/β antibody.  
(a) Mass spectrometry analysis of BIN2 auto-phosphorylation site. GST-BIN2 protein purified from E.coli was subjected to in vitro kinase reaction. The protein was digested by trypsin and analyzed by LC-MS/MS using LTQ/FT mass spectrometry. The CID mass spectrum and sequence of the peptide containing phospho-tyrosine 200 residue of BIN2 are shown. (b) Amino acids alignment of the immunogen peptide of phospho-tyrosine 279/216 GSK3α/β antibody and the same region of BIN2. The phospho-tyrosine residue is marked by asterisk. (c) Anti-phosoho-Tyr 279/216GSK3α/β antibody specifically detects phospho-tyrosine 200 of BIN2. Immunoblot of the wild type, the kinase inactive M115A, and the Y200A mutant GST-BIN2 proteins were probed with the anti-phosoho-Tyr 279/216GSK3α/β antibody. The blot was re-probed with anti-GST antibody.

SUPPLEMENTARY INFORMATION

a  
+ BZR1-cYFP  

nYFP-BIN2 nYFP nYFP  
-AtSK12 -AtSK12-cd  

b  
Mock +BRZ +BL  

c  
MS BRZ  
Anti-myc  
Ponceau S  

d  

GEPNISpYICSR  
Theoretical m/z = 688.289  
Experimental m/z = 688.300  
+16.0ppm  

200 400 600 800 1000 1200  
m/z  

Figure S9 In vivo interaction between AtSK12 and BZR1, effect of brassinazole (BRZ) and brassinolide (BL), and identification of autophosphorylation site of AtSK12. (a) Both AtSK12 and BIN2 interact with BZR1 in BiFC assay. Transgenic Arabidopsis plants expressing nYFP-BIN2, nYFP-AtSK12 and nYFP-AtSK12-cd (C-terminal 29 amino acid deletion) were crossed into BZR1-cYFP plants, respectively. The seedlings of F1 generation were grown in the dark for 4 days and YFP signals of hypocotyls were observed. The scale bar is 10 μm. (b) Confocal images of hypocotyls cells of transgenic Arabidopsis plants expressing YFP-AtSK12 grown on MS medium, or MS containing 2 μM BRZ or 0.1 μM BL in the dark for 4 days. The scale bar is 10 μm. (c) BRZ induces the accumulation of AtSK12. AtSK12-myc plants were grown on MS or 2 μM BRZ medium for 5 days. Total protein extracts were blotted onto nitrocellulose membrane and probed by anti-myc antibody. (d) Mass spectrometry analysis of AtSK12 autophosphorylation site. GST-AtSK12 protein purified from E.coli was subjected to in vitro kinase reaction. The protein was digested by trypsin and analyzed by LC-MS/MS using LTQ/FT mass spectrometry. The CID mass spectrum and sequence of the peptide containing phospho-tyrosine 233 residue of AtSK12 are shown.

WWW.NATURE.COM/NATURECELLBIOLOGY

© 2009 Macmillan Publishers Limited. All rights reserved.

SUPPLEMENTARY INFORMATION

BIN2 (red) and BSU1 (blue)

BIN2 (red) and BRI1 (blue)

BIN2 (red) and BSL1 (blue)

BIN2 (red) and BSK1 (blue)

BIN2 (red) and BZR1 (blue)

Figure S10 Similar gene expression patterns of the BR signaling components. (a) Absolute expression level of BIN2. (b-f) Comparison of gene expression between BIN2 and BRI1, BSK1, BSU1, BSL1 or BZR1. The color scale at the left bottom of each image shows color coding of expression level

(a) or ratio of expression levels of BIN2 to the other genes (b-f, yellow color indicates similar expression level). Results were from the Arabidopsis eFP browser (http://bbc.botany.utoronto.ca/efp/cgi-bin/efpWeb.cgi) (Winter et al., 2007. PLoS One 2(8): 2718).

SUPPLEMENTARY INFORMATION

![Bar chart showing enzyme activity](#)

Figure S11 BSU1 shows tyrosine phosphatase activity. MBP, MBP-Kelch (N-terminal domain of BSU1), or MBP-BSU1 was incubated with $p$-nitrophenyl phosphate as a substrate. The enzyme activity was determined by production of $p$-nitrophenol.

WWW.NATURE.COM/NATURECELLBIOLOGY

© 2009 Macmillan Publishers Limited. All rights reserved.

SUPPLEMENTARY INFORMATION

Fig. 1a  
IB: PAP  

Fig. 1b  
IB: MBP  

Fig. 1c  
Autoradiography  

Fig. 1d  
Autoradiography  

Fig. 1e  
Autoradiography  

Fig. 1h  
IB: myc  

Fig. 1i  
IB: myc  

Fig. 2b  
IB: GFP  

Fig. 2c  
IB: myc  

Fig. 2d  
BIN2  
bin2-1  
IB: myc  

Fig. 3a  
Autoradiography  

Fig. 3b  
BIN2  
BIN2  
IB: pTyr  
IB: GST  

Fig. 3c  
bin2-1  
bin2-1  
BIN2  
BIN2  
bin2-1  
bin2-1  
IB: pTyr  
IB: GST  
IB: pTyr  
IB: GST  
IB: pTyr  
IB: GST  

Fig. 3d  
55  
43  
34  
26  
BIN2  
bin2-1  
IB: pTyr  

Fig. 3e  
BIN2  
bin2-1  
IB: pTyr  

Fig. 4e  
Autoradiography  

Fig. 4f  
IB: myc  

Fig. 4g  
IB: myc  

Fig. 5a  
IB: MBP-BSU1  

Fig. 5c  
Input  
IP  
IB: GFP  

Fig. 5d  
170  
130  
95  
72  
55  
43  
Marker  
(kDa)  
IB: MBP-BSU1  

Figure S12 Full scan data of immunoblot and in vitro kinase/phosphatase assays. Correct bands are indicated by red asterisk.
