
BIOCHEMISTRY
including biophysical chemistry & molecular biology
Article

The ATP-Dependent Amide Ligases DdaG and DdaF Assemble the Fumaramoyl-
Dipeptide Scaffold of the Dapdiamide Antibiotics†

Marie A. Hollenhorst, Jon Clardy, and Christopher T. Walsh*

Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115

Received July 29, 2009; Revised Manuscript Received September 18, 2009

ABSTRACT: The enzymes DdaG and DdaF, encoded in the *Pantoea agglomerans* dapdiamide antibiotic biosynthetic gene cluster, when expressed in *Escherichia coli*, form the tandem amide bonds of the dapdiamide scaffold at the expense of ATP cleavage. DdaG uses fumarate, 2,3-diaminopropionate (DAP), and ATP to make fumaryl-AMP transiently on the way to the $N_{\beta}$-fumaryl-DAP regioisomer. Then DdaF acts as a second ATP-dependent amide ligase, but this enzyme cleaves ATP to ADP and P$_i$ during amide bond formation. However, DdaF will not accept $N_{\beta}$-fumaryl-DAP; the enzyme requires the fumaryl moiety to be first converted to the fumaramoyl half-amide in $N_{\beta}$-fumaramoyl-DAP. DdaF adds Val, Ile, or Leu to the carboxylate of fumaramoyl-DAP to make dapdiamide A, B, or C, respectively. Thus, to build the dapdiamide antibiotic scaffold, amidation must occur on the fumaryl-DAP scaffold, after DdaG action but before DdaF catalysis. This is an unusual instance of two ligases acting sequentially in untemplated amide bond formations using attack of substrate carboxylates at P$_{\alpha}$ (AMP-forming) and then at P$_{\gamma}$ (ADP-forming) of ATP cosubstrates.

---

Pantoea agglomerans strains produce a variety of antibiotics, as evidenced by the discoveries of the pantocins (1, 2), andrimid (3), and most recently dapdiamides A–E (4) (Figure 1A). This family of fumaramoyl and epoxysuccinamoyl dipeptides is named for the presence of DAP and the two amide linkages of the antibiotic scaffold. The compounds are similar in structure to synthetic methoxyfumaryl-DAP dipeptides which show broad spectrum activity by targeting bacterial and fungal glucosamine-6-phosphate (GlcN-6-P) synthase, the enzyme that catalyzes the rate-limiting step in the formation of the essential cell-wall building block UDP-N-acetylglucosamine (5, 6). An analogous mode of action has been proposed for three previously isolated antibiotics with structures similar to those of the dapdiamides, CB-25-I from *Serratia plymuthica* (7) and Sch37137 (8) and A19009 (9, 10) from actinomycete isolates (Figure 1B) (5). These compounds are likely cleaved intracellularly to generate acyl-DAP warheads which can inhibit GlcN-6-P synthase, as demonstrated for $N_{\beta}$-fumaramoyl-DAP (11) (Figure 1A). The fumaramoyl-DAP warheads derived from A19009 and dapdiamides A–D would provide a conjugated eneamide system that is likely attacked by the thiolate side chain of the active site Cys in the glutaminase domain of GlcN-6-P synthase in analogy to the mechanism of inactivation observed for $N_{\beta}$-methoxyfumaryl-DAP (5) (Figure 1C). Alkylation of this residue blocks generation of the ammonia required to convert fructose-6-P into glucosamine-6-P. The epoxysuccinamoyl moiety in CB-25-I, Sch37137, and dapdiamide E could likewise function as an inactivating electrophile for the catalytic Cys in the target enzyme.

Peptide antibiotics are typically formed by two biological strategies. The first involves ribosomal synthesis of protein precursors that undergo posttranslational proteolysis and/or modification and is exemplified by the Microcin B17, C7, and E492 molecules as well as the lantibiotics such as nisin, and the thiazolyl peptides such as thiostrepton and thiocillins (12). The second strategy uses nonribosomal peptide synthetase (NRPS) assembly lines to specify the order and composition of amino acid monomers incorporated and accounts for the biogenesis of penicillin, vancomycin, and daptomycin scaffolds among others (12). However, there is another way that peptides can be assembled, in a manner independent of ribosomal-based mRNA or NRPS machinery, via ATP-dependent coupling of amino acid monomers and elongating peptides by soluble enzymes. This route has precedents in glutathione biosynthesis (13) and bacterial cell wall pentapeptide elongation (14) where ATP is cleaved to ADP and intermediate acyl phosphates are formed as reactive species. A variant of this route occurs in the ATP-dependent elongation and cyclization of the siderophore ferrioxamine E (15) by cleavage of ATP to AMP and PP$_i$ and involves acyl-AMP intermediates. We have found that the acyl dipeptides of the dapdiamide antibiotic family are constructed by this third strategy, via tandem action of two soluble ligases, one (DdaG) that cleaves ATP to AMP and another (DdaF) that cleaves ATP to ADP.

While no biosynthetic gene information is available for CB-25-I, Sch37137, or A19009, a single gene cluster from *P. agglomerans* that makes all five dapdiamides has recently been cloned into *Escherichia coli* (4) (Figure S1 of the Supporting Information). This metabolic capacity from one gene cluster suggests both pathway enzyme promiscuity and the prospect of increased scaffold diversity from combinatorial biosynthesis once the catalysts have been identified. Inspection of the fumaramoyl/epoxysuccinamoyl-dipeptide scaffold of

†This work was supported in part by National Institutes of Health Grants GM 20011 (C.T.W.) and GM 086258 (J.C.) and Medical Scientist Training Program GM 07753 (M.A.H.).

*To whom correspondence should be addressed. E-mail: christopher_walsh@hms.harvard.edu. Phone: (617) 432-1715. Fax: (617) 432-0483.

MATERIALS AND METHODS

**Bacterial Strains, Plasmids, Materials, and Instrumentation.** Oligonucleotide primers were synthesized by Integrated DNA Technologies (Coralville, IA). PCR was performed with Phusion High-Fidelity PCR Mastermix (New England Biolabs). Cloning was performed using the Gateway System (Invitrogen). One Shot Chemically Competent TOP10 *E. coli* (Invitrogen) and NovaBlue(DE3) (Novagen) were used for routine cloning and propagation of DNA vectors. Recombinant plasmid DNA was purified with a Qiaprep kit (Qiagen). DNA sequencing was performed at the Molecular Biology Core Facilities of the Dana Farber Cancer Institute (Boston, MA). Nickel-nitrilotriacetic acid agarose (Ni-NTA) superflow resin and SDS-PAGE gels were purchased from Qiagen. Protein samples were concentrated using 30 kDa MMCO Amicon Ultra filters (Millipore). Protein concentrations were determined by Bradford assay with BSA as a standard. Chemicals were purchased from Sigma-Aldrich. NMR solvents were purchased from Cambridge Isotopes.

A pyruvate kinase/lactate dehydrogenase (PK/LDH) enzyme mix from rabbit muscle was purchased from Sigma as a buffered aqueous glycerol solution. Myokinase from chicken muscle was purchased from Sigma as a lyophilized powder and resuspended in 10 mM HEPES (pH 8). Synthetic dapdiamide A and the plasmid containing the dapdiamide gene cluster, pUC19A10A, were provided by J. Dawlaty. Fumaramic acid was prepared from monomethyl fumarate as described previously (17) or from monoethyl fumarate via an analogous procedure.

¹H NMR spectra were recorded on a Varian 400 or 600 MHz spectrometer. MS analysis was performed on an Agilent Technologies 6520 Accurate-Mass Q-TOF LC-MS instrument, an Agilent Technologies 6210 Accurate-Mass TOF LC-MS instrument, or by staff at the Harvard University Mass Spectrometry Laboratory (Cambridge, MA). HPLC data were collected on a Beckman Coulter System Gold (126 solvent module, 168 detector). An Alltech Alltima C18 (250 mm × 4.6 mm) column was used for routine analytical HPLC. A Chiralcel OD-RH (150 mm × 4.6 mm) chiral column was used for separation of FMOC-derivatized *N*-fumaryl-DAP regioisomers. A Phenomenex Luna C18 (250 mm × 21.2 mm) column was used for preparative HPLC.

**Cloning, Overexpression, and Purification of DdaG and DdaF.** *ddaG* and *ddaF* were PCR amplified from pUC19A10A. For *ddaG*, the forward primer was 5′-CACCAATAAGGGA-GAAAACATGAATCTGGAAATG-3′ and the reverse primer was 5′-TTAATAGCGGGCGTTTACTGACGTTGCCG-3′. For *ddaF*, the forward primer was 5′-CACCTCGATTTTGAACAA-TAAAGAGTCATCGTAATC-3′ and the reverse primer was 5′-TTATTCATCAATTAGGCCCAAAGTGATAAAAATCCG-3′. Unpurified PCR products were mixed with the pENTR/D-TOPO vector, and the TOPO construct was then transformed into TOP10 *E. coli*. The purified vector was analyzed by PCR to confirm the presence of the insert. The insert was recombined into pDEST17, and the product of this reaction was transformed into TOP10 *E. coli*. The purified vector was sequenced to confirm the insert. BL21(DE3) *E. coli* cells were transformed with pDEST17-*ddaF* or pDEST17-*ddaG*. Cultures (25 mL) were grown overnight at 37°C in Luria-Bertani (LB) medium supplemented with 50 μg/mL carbenicillin. Two liters of LB with 50 μg/mL carbenicillin was inoculated with 10 mL of this overnight culture. The cultures were incubated at 37°C for 3.5–5 h and then at 15°C for 0.5–1 h. Cultures were induced with 0.5 mM IPTG. The cultures were

incubated at 15 °C for 20 h, and then the cells were harvested by centrifugation at 4200 rpm for 12 min. Cell pellets were resuspended in 10 mL of lysis buffer [10 mM Tris-HCl (pH 8) with 200 mM NaCl or 10 mM HEPES (pH 8) with 200 mM NaCl] and lysed at 5000–10000 psi in an Avestin EmulsiFlex-C5 high-pressure homogenizer. The cell debris was removed by ultracentrifugation at 35000 rpm for 35 min. Supernatants were applied to a low-pressure chromatography column containing Ni-NTA resin pre-equilibrated in lysis buffer. The resin was washed with lysis buffer containing 20 mM imidazole. The protein was eluted with lysis buffer containing 80 and 200 mM imidazole. Protein was concentrated and imidazole removed using 30 kDa MMCO tubes. In some cases, additional purification was achieved by gel filtration chromatography on an Amersham Pharmacia Biotech AKTA FPLC system using a Superdex 75 26/60 HiLoad column. Aliquots of the proteins were flash-frozen in liquid nitrogen and stored at –80 °C.

**DdaG and DdaF HPLC Activity Assays.** Reaction mixtures (50 μL) were incubated at room temperature and contained 9.5 μM enzyme, 2.5 or 5 mM amine substrate, 2.5 or 5 mM carboxylate substrate, 6 mM ATP, 10 mM MgCl₂, and 100 mM HEPES (pH 8). The protein was removed by centrifugation in 5 kDa MMCO filter tubes at 12000g for 15 min; 100 μL of water was added, and then filter tubes were spun again at 12000g for 15 min. Eighty microliters of filtered reaction mixture was combined with 80 μL of CH₃CN, 50 μL of water, and 40 μL of 10 mM FMOC-Cl (freshly prepared solution in CH₃CN) and incubated at room temperature for at least 5 min; 80 μL of 100 mM aminoadamantane (in a 1:1 CH₃CN/H₂O mixture) was added, and reaction mixtures were incubated at room temperature for at least 5 min before HPLC analysis using a gradient from 20 to 100% solvent B (0.1% TFA in CH₃CN) in solvent A (0.1% TFA in water) over 25 min. For resolution of N-fumaroyl-DAP regioisomers, FMOC-derivatized samples were analyzed by chiral HPLC using a gradient from 20 to 70% solvent B (0.1% TFA in CH₃CN) in solvent A (0.1% TFA in water) over 35 min. The absorbance was monitored at 263 nm.

**Coupled Spectrophotometric Assays for AMP or ADP Production.** For all experiments, 200 μL reaction mixtures were incubated at room temperature and contained 10 mM ATP, 12 mM MgCl₂, 200 μM NADH, 500 μM phosphoenolpyruvate, and 100 mM HEPES (pH 8). For characterization of DdaG, reaction mixtures in addition contained 43 units/mL LDH, 30 units/mL PK, 21 units/mL myokinase, 10 mM DAP, and 1.9 nM DdaG for fumarate assays or 9.5 nM DdaG for succinate assays. For comparison of Nα- and Nβ-fumar(am)oyl-DAP substrates for DdaF, reaction mixtures in addition contained 30 units/mL LDH, 15 units/mL PK, 2 mM Val, and 240 nM DdaF. For determination of the DdaF reaction velocity with varying concentrations of Nβ-fumaramoyl-DAP, reaction mixtures in addition contained 60 units/mL LDH, 42 units/mL PK, 240 nM DdaF, and 5 mM Val. For characterization of DdaF with respect to different amino acids, reaction mixtures in addition contained 60 units/mL LDH, 42 units/mL PK, 125 μM Nβ-fumaramoyl-DAP, and 240 nM DdaF. The consumption of NADH was monitored continuously in a Molecular Devices SPECTRAmax PLUS 96-well plate reader by measuring the absorbance at 340 nm every 20 s. Kinetic constants were derived from velocity versus substrate concentration data using a nonlinear, least-squares fitting method with GraphPad Prism.

**ATP-[³²P]PPi Exchange Assays.** Reaction mixtures (1.2 mL) contained 1.9 μM DdaG, 5 mM substrate, 1 mM

ATP, 1 mM MgCl₂, 40 mM KCl, 1 mM DTT, 5 mM Na[³²P]PPi (3.3 × 10⁴ cpm/mL), and 50 mM Tris-HCl (pH 8). Mixtures were incubated at 37 °C for 30 min, and then 350 μL aliquots were removed and quenched with 750 μL of a charcoal suspension (100 mM NaPPi, 350 mM HClO₄, and 16 g/L charcoal). The mixtures were vortexed and then centrifuged at 13000 rpm for 3 min. The pellets were washed twice with 750 μL of wash solution (100 mM NaPPi and 350 mM HClO₄). Charcoal-bound radioactivity was measured on a Beckman LS 6500 scintillation counter.

---

**RESULTS**

*Expression and Purification of DdaG and DdaF in E. coli.* ddaG and ddaF were PCR amplified from pUC19 A10A, a plasmid containing the dapdiamide gene cluster, and cloned into an expression vector encoding an N-terminal His₆ tag. The proteins were overexpressed in *E. coli* BL21(DE3) and purified by Ni-NTA affinity chromatography (Figure S2 of the Supporting Information). Yields ranged from 12 to 16 mg/L for DdaG and from 6 to 11 mg/L for DdaF.

*DdaG Is a Regioselective ATP-Dependent Fumaroyl-DAP Ligase.* A small amount of Nβ-fumaramoyl-DAP was isolated with the dapdiamides from fermentation (4), suggesting that acylation of DAP precedes the formation of the DAP-amino acid amide bond. To determine which enzyme is responsible for this acylating activity, purified DdaG and DdaF were incubated with DAP and potential carboxylate substrates, and reaction mixtures were derivatized with FMOC and analyzed via HPLC.

DdaG ligates fumarate and DAP to form N-fumaroyl-DAP in an ATP-dependent manner (Figure 2A and Table S1 of the Supporting Information). The enzyme also forms N-succinoyl-DAP from succinate and DAP (Figure S3 and Table S1 of the Supporting Information). In contrast, the ligase showed no activity in these assays with the corresponding half-amides, fumarate and succinate, or with the two enantiomers of trans-epoxysuccinate (Figure S4 of the Supporting Information). A coupled spectrophotometric assay for AMP formation was used to kinetically characterize DdaG, and these experiments demonstrated that the enzyme is specific for fumarate over succinate, with a $k_{cat}/K_m$ of 21 mM⁻¹ s⁻¹ for fumarate and 0.19 mM⁻¹ s⁻¹ for succinate (Table S2 of the Supporting Information).

ATP-[³²P]PPi exchange assays, which test for the reversible formation of adenylated intermediates, demonstrated that DdaG adenylates its carboxylate substrate as predicted on the basis of its homology to AMP-forming enzymes (Figure 2B). As expected, activation of both fumarate and succinate was observed, whereas there was no activation of either the epoxysuccinates or succinate. Exchange was seen with fumarate as a result of contamination of this substrate with fumarate.²

In dapdiamides A–C, fumarate and DAP are linked through the DAP β-amino group, while in dapdiamides D and E, it is the α-amino group that is acylated. This suggested that DdaG could act in a regiopromiscuous manner, accepting either the DAP α- or β-amine as a nucleophile. To test this hypothesis,

---

²Fumaramate used in this assay was prepared from commercial monoethyl fumarate that is contaminated with a small amount of fumarate. Kinetic analysis of the contaminated fumaramate is consistent with the hypothesis that DdaG activity with this substrate results from contamination with 2–5% fumarate.

10470 Biochemistry, Vol. 48, No. 43, 2009

Hollenhorst et al.

**Figure 2: Characterization of DdaG activity.**

(A) HPLC analysis showing ATP-dependent ligation of fumarate and DAP by DdaG.

(B) ATP–\[^{32}P\]PPi exchange data. The graph shows counts per minute arising from \[^{32}P\]ATP that is adsorbed to charcoal after incubation of DdaG, \[^{32}P\]PPi, unlabeled ATP, and carboxylate substrate. *Activity with fumaramate results from 2–5% contamination with fumarate.*² (C) Co-elution of the FMOC-derivatized DdaG product with FMOC-Nβ-fumaroyl-DAP.

both fumaroyl-DAP isomers were synthesized (Schemes S1 and S2 of the Supporting Information) and their FMOC derivatives

**Figure 3: Relative rates of ADP formation on incubation of DdaF with potential carboxylate substrates and 2 mM Val.**

**Figure 4: HPLC analysis showing ATP-dependent ligation of Nβ-fumaroyl-DAP and Val by DdaF to form dapdiamide A.**

separated by chiral HPLC. The FMOC-derivatized product of DdaG catalysis co-eluted with the β-isomer, demonstrating that DdaG acts in a regioselective manner to form Nβ-fumaroyl-DAP exclusively (Figure 2C).

Article

**Scheme 1: Biosynthetic Route to Dapdiamides A–C**

![Chemical Scheme](https://i.imgur.com/yourimageurl.png)

"The grayed-out step has not been biochemically validated."

**DdaF Is an ATP-Dependent Nβ-Fumaramoyl-DAP-Amino Acid Ligase.** DdaF was characterized using both an ADP production assay and the same HPLC assay used to study DdaG. These experiments showed that DdaF catalyzes the second subunit condensation reaction in this pathway. The enzyme is not active with either the product of DdaG catalysis, Nβ-fumaroyl-DAP, or the corresponding α-isomer (Figure 3 and Figure S5 of the Supporting Information). DdaF accepts only the corresponding amide Nβ-fumaramoyl-DAP, which it can ligate to Val, Ile, or Leu to form dapdiamides A–C, respectively (Figures 3 and 4 and Figure S6 and Table S3 of the Supporting Information). DdaF is specific for the Nβ-isomer of fumaryl-DAP; it will not ligate Nα-fumaramoyl-DAP to Val (Figure 3 and Figure S7 of the Supporting Information).

Apparent substrate inhibition was observed at Nβ-fumaramoyl-DAP concentrations of >500 μM (Figure S8 of the Supporting Information), so kinetics with respect to the three amino acid substrates were determined at a fixed Nβ-fumaramoyl-DAP concentration of 125 μM. While the apparent \( k_{\text{cat}} \) values for each amino acid were similar, the apparent \( K_{\text{m}} \) values were 260 μM for Val, 830 μM for Ile, and 7.82 mM for Leu (Table S4 of the Supporting Information).

**Stability of the Nβ-Acyl-DAP Compounds.** The lack of DdaF activity with Nα-fumaramoyl-DAP makes the origin of the α-linked dapdiamides D and E mysterious. We suspected that these compounds could arise from nonenzymatic isomerization of the corresponding β-linked regioisomers, so we tested the stability of dapdiamide A in 500 mM NH₄OH at room temperature overnight. However, we did not observe any isomerization (Figure S9 of the Supporting Information). We also tested for possible isomerization of the pathway intermediates. There was no conversion of Nβ-fumaroyl-DAP to the α-regioisomer when it was incubated at pH 8 for 3 days (Figure S10 of the Supporting Information). Preliminary results indicate that isomerization of the corresponding amide, Nβ-fumaramoyl-DAP, also does not happen readily at pH 9.5. On the other hand, some frozen samples of Nα-fumaramoyl-DAP (but not the Nβ-isomer) decomposed, but the process has not been examined thoroughly.

**DISCUSSION**

We observe that DdaG uses fumarate, DAP, and ATP to generate Nβ-fumaroyl-DAP as ATP is cleaved to AMP and PP₁. DAP can in principle be used as a nucleophile either at the β-NH₂ group (on the way to dapdiamides A–C) or at the α-NH₂ group (on the way to dapdiamides D and E). In these studies, only the β-coupling regiochemistry was seen with purified DdaG. In turn, DdaF cleaves ATP to ADP and Pᵢ as Nβ-fumaramoyl-DAP is coupled to Val, Ile, or Leu. Most notably, while DdaG will not accept fumaramate in place of fumarate as the coupling substrate, DdaF requires fumaramoyl-DAP rather than fumaroyl-DAP.

The characterization of DdaF and DdaG allows us to propose the timing of other catalytic events in this biosynthetic pathway. Bioinformatic analysis suggested that DdaH is an amidotransferase (4), and our results imply that this enzyme acts after DdaG, converting Nβ-fumaroyl-DAP to Nβ-fumaramoyl-DAP. DdaF can then ligate this compound to three branched chain amino acids (Scheme 1). DdaC is a putative Fe(II)/α-ketoglutarate-dependent dioxygenase and is likely responsible for the formation of the dapdiamide E epoxide. Because DdaG shows no activity with epoxysuccinate, it is probable that DdaC acts on a double bond of either fumar(am)oyl-DAP or fumaramoyl-DAP-Val. This epoxidation may occur while the substrate is tethered to the adenylation-thiolation enzyme DdaD, and the oxidized substrate could be released by the thioesterase homologue DdaE. The origin of the Nα-acyl-DAP linkage in dapdiamides D and E remains an open question as we have been unable to observe any Nα-acyl-DAP formation from either enzymatic or nonenzymatic conversion. These compounds could potentially be formed via the action of an enzyme that catalyzes an α/β-isomerization or via the action of a DAP α-NH₂-specific ligase; however, these activities do not appear to be encoded in the dapdiamide gene cluster.

The dapdiamides constitute the first example of a family of small molecules targeted against GlcN-6-P synthase that are produced by a single biosynthetic gene cluster. The production of a suite of compounds instead of a single antibiotic may allow the producing organism to target a broader range of microbes and/or blunt development of target resistance. It has been shown that the identity of the amino acid ligated to DAP in the methoxyfumaryl-oyl-DAP dipeptides influences the rate of target-cell uptake (6), and this is likely also true for the dapdiamides. The promiscuity of DdaF with respect to its amino acid substrate allows the formation of compounds with three different C-terminal amino acids, and each may interact differently with the nonspecific peptide permeases that import these types of compounds. Additionally, the two different electrophilic moieties (Michael acceptor or epoxide) may have different levels of reactivity with the target enzyme. It remains to be explored whether production

of a family of structurally distinct compounds with the same target confers an advantage on the producer, but this may be one explanation for the evolution of substrate promiscuity in the dapdiamide pathway.

Here we focused on the two amide bond-forming enzymes that are required to generate the fumaramoyl-DAP-dipeptide scaffold of the dapdiamides. These catalysts are new members of an expanding class of ligases that assemble antibiotic peptides in a distinct manner from ribosomal and traditional NRPS systems. Such enzymes typically use one of two modes of carboxylate substrate activation: adenylation or phosphorylation. Like DdaG, the amide ligases CouL, NovL, and CloL use the first strategy to activate carboxylates for nucleophilic attack by donor amino groups to bring together monomers in the assembly of the aminocoumarin antibiotics coumermycin, novobiocin, and clorobiocin, respectively (18–20). In contrast, ATP grasp family members in the bacilysin (21, 22) and tabtoxin (23) pathways likely activate carboxylate substrates via acyl-phosphate formation, like DdaF. The dapdiamide pathway is an atypical example of the recruitment of two amide ligases with distinct ATP cleavage patterns to form tandem peptide bonds.

Recent studies of *P. agglomerans* antibiotics have led to the identification of several novel condensation catalysts (24) that are homologues of enzymes from primary metabolism, suggesting that nature’s repertoire of condensation catalysts for natural product biosynthesis may be broader than once suspected. A detailed understanding of the function of these catalysts may open the door to the study of new classes of compounds that are not produced by canonical biosynthetic enzymes and that may have structural features which set them apart from previously identified natural products.

### ACKNOWLEDGMENT

We thank Jessica Dawlaty for providing a sample of synthetic dapdiamide A and the pUC19 A10A plasmid containing the dapdiamide gene cluster. We thank Elizabeth Sattely, Christopher Neumann, Emily Balskus, and Michael Fischbach for helpful discussions.

### SUPPORTING INFORMATION AVAILABLE

Figures S1–S10, Tables S1–S4, Schemes S1–S4, and supplemental methods. This material is available free of charge via the Internet at http://pubs.acs.org.

### REFERENCES

1. Brady, S. F., Wright, S. A., Lee, J. C., Suton, A. E., Zumoff, C. H., Wodzinski, R. S., Beer, S. V., and Clardy, J. (1999) Pantocin B, an antibiotic from *Erwinia herbicola* discovered by heterologous expression of cloned genes. *J. Am. Chem. Soc.* 121, 11912–11913.
2. Jin, M., Liu, L., Wright, S. A. I., Beer, S. V., and Clardy, J. (2003) Structural and functional analysis of Pantocin A: An antibiotic from *Pantoea agglomerans* discovered by heterologous expression of cloned genes. *Angew. Chem.*, Int. Ed. 42, 2898–2901.
3. Jin, M., Fischbach, M. A., and Clardy, J. (2006) A biosynthetic gene cluster for the acetyl-CoA carboxylase inhibitor andrimid. *J. Am. Chem. Soc.* 128, 10660–10661.
4. (a) Dawlaty, J. A. (2008) Bioactive natural products from *Pantoea agglomerans*, *Janithinobacterium lividum*, and *Conocarpus erectus*. Ph.D. Thesis, Cornell University, Ithaca, NY. (b) Dawlaty, J., Fischbach, M. A., and Clardy, J. Personal communications.
5. Kucharczyk, N., Denisot, M. A., Le Goffic, F., and Badet, B. (1990) Glucosamine-6-phosphate synthase from *Escherichia coli*: Determination of the mechanism of inactivation by N³-fumaroyl-L-2,3-diaminopropionic derivatives. *Biochemistry (Moscow, Russ. Fed.)* 29, 3668–3676.
6. Milewski, S., Andruszkiewicz, R., Kasprzak, L., Mazerski, J., Mignini, F., and Borowski, E. (1991) Mechanism of action of antifungal dipeptides containing inhibitors of glucosamine-6-phosphate synthase. *Antimicrob. Agents Chemother.* 35, 36–43.
7. Shoji, J., Hinoo, H., Sakazaki, R., Kato, T., Hattori, T., Matsumoto, K., Tawara, K., Kikuchi, J., and Terui, Y. (1989) Isolation of CB-25-I, an antifungal antibiotic, from *Serratia plymuthica*. *J. Antibiot.* 42, 869–874.
8. Cooper, R., Horan, A. C., Gentile, F., Gullo, V., Loebenberg, D., Marquez, J., Patel, M., Puar, M. S., and Truumees, I. (1988) Sch 37137, a novel antifungal compound produced by a *Micromonospora sp.* Taxonomy, fermentation, isolation, structure and biological properties. *J. Antibiot.* 41, 13–19.
9. Molloy, B. B., Lively, D. H., Gale, R. M., Forman, M., Boeck, L. D., Higgins, C. E., Kastner, R. E., Huckstep, L. L., and Neuss, N. (1972) A new dipeptide antibiotic from *Streptomyces collinus*, lindenbein. *J. Antibiot.* 25, 137–140.
10. van der Baan, J. L., Barnick, J. W. F. K., and Bickelhaupt, F. (1983) Antibiotic A 19009 Structural Investigation and Synthesis. *J. Antibiot.* 36, 784–792.
11. Chmara, H., Andruszkiewicz, R., and Borowski, E. (1985) Inactivation of glucosamine-6-phosphate synthetase from *Salmonella typhimurium* LT2 by fumaroyl diamino-propanoic acid derivatives, a novel group of glutamine analogs. *Biochim. Biophys. Acta* 870, 357–366.
12. Nolan, E. M., and Walsh, C. T. (2009) How Nature Morphs Peptide Scaffolds into Antibiotics. *ChemBioChem* 10, 34–53.
13. Mooz, E. D., and Meister, A. (1967) Tripeptide (glutathione) synthetase. Purification, properties, and mechanism of action. *Biochemistry (Moscow, Russ. Fed.)* 6, 1722–1734.
14. Walsh, C. T. (1989) Enzymes in the d-alanine branch of bacterial cell wall peptidoglycan assembly. *J. Biol. Chem.* 264, 2393–2396.
15. Kadi, N., Oves-Costales, D., Barona-Gomez, F., and Challis, G. L. (2007) A new family of ATP-dependent oligomerization-macrocyclization biocatalysts. *Nat. Chem. Biol.* 3, 652–656.
16. Galperin, M. I., and Koonin, E. V. (1997) A diverse superfamily of enzymes with ATP-dependent carboxylate-amine/thiol ligase activity. *Protein Sci.* 6, 2639–2643.
17. Talley, E. A., Fitzpatrick, T. J., and Porter, W. L. (1959) Formation of fumaramic acid from asparagine in phosphate buffer. *J. Am. Chem. Soc.* 81, 174–175.
18. Schmutz, E., Steffensky, M., Schmidt, J., Porzel, A., Li, S.-M., and Heide, L. (2003) An unusual amide synthetase (CouL) from the coumermycin A₁ biosynthetic gene cluster from *Streptomyces rishiriensis* DSM 40489. *Eur. J. Biochem.* 270, 4413–4419.
19. Steffensky, M., Li, S.-M., and Heide, L. (2000) Cloning, overexpression, and purification of novobiocin acid synthetase from *Streptomyces spheroides* NCIMB 11891. *J. Biol. Chem.* 275, 21754–21760.
20. Galm, U., Dessoy, M. A., Schmidt, J., Wessjohann, L. A., and Heide, L. (2004) *In vitro* and *in vivo* production of new aminocoumarins by a combined biochemical, genetic, and synthetic approach. *Chem. Biol.* 11, 173–183.
21. Steinborn, G., Hajirezaei, M.-R., and Hofemeister, J. (2005) *bac* genes for recombinant bacilysin and anticapsin production in *Bacillus* host strains. *Arch. Microbiol.* 183, 71–79.
22. Tabata, K., Ikeda, H., and Hashimoto, S.-I. (2005) *ywfE* in *Bacillus subtilis* codes for a novel enzyme, L-amino acid ligase. *J. Bacteriol.* 187, 5195–5202.
23. Kinscherf, T. G., and Willis, D. K. (2005) The biosynthetic gene cluster for the β-lactam antibiotic tabtoxin in *Pseudomonas syringae*. *J. Antibiot.* 58, 817–821.
24. Fortin, P. D., Walsh, C. T., and Magarvey, N. A. (2007) A transglutaminase homologue as a condensation catalyst in antibiotic assembly lines. *Nature* 448, 824–827.
